
REVIEW

# Powering through ribosome assembly

**BETHANY S. STRUNK**${}^{1}$ **and KATRIN KARBSTEIN**${}^{1,2,3}$

${}^{1}$Chemical Biology Doctoral Program, University of Michigan, Ann Arbor, Michigan 48109-1055, USA  
${}^{2}$Department of Chemistry, University of Michigan, Ann Arbor, Michigan 48109-1055, USA  
${}^{3}$Department of Biological Chemistry, University of Michigan, Ann Arbor, Michigan 48109-1055, USA  

## ABSTRACT

Ribosome assembly is required for cell growth in all organisms. Classic in vitro work in bacteria has led to a detailed understanding of the biophysical, thermodynamic, and structural basis for the ordered and correct assembly of ribosomal proteins on ribosomal RNA. Furthermore, it has enabled reconstitution of active subunits from ribosomal RNA and proteins in vitro. Nevertheless, recent work has shown that eukaryotic ribosome assembly requires a large macromolecular machinery in vivo. Many of these assembly factors such as ATPases, GTPases, and kinases hydrolyze nucleotide triphosphates. Because these enzymes are likely regulatory proteins, much work to date has focused on understanding their role in the assembly process. Here, we review these factors, as well as other sources of energy, and their roles in the ribosome assembly process. In addition, we propose roles of energy-releasing enzymes in the assembly process, to explain why energy is used for a process that occurs largely spontaneously in bacteria. Finally, we use literature data to suggest testable models for how these enzymes could be used as targets for regulation of ribosome assembly.

Keywords: ATPase; GTPase; quality control; rRNA processing; ribosome assembly

## INTRODUCTION

In order to catalyze protein synthesis, ribosomes must be assembled from ribosomal RNA (rRNA) and proteins (Rps). Mature ribosomes are composed of two subunits in all organisms, the large subunit (LSU) and the small subunit (SSU). The LSU comprises three rRNAs in eukaryotes (25S, 5.8S, 5S in *Saccharomyces cerevisiae*) and two in prokaryotes (23S, 5S in *Escherichia coli*), while the SSU has one rRNA in all kingdoms (18S in *S. cerevisiae* and 16S in *E. coli*). All three bacterial rRNAs and three of the four eukaryotic rRNAs are synthesized as a single transcript that must be processed and cleaved at specific sites during the assembly process to produce the mature rRNA species (Fig. 1; for review, see Hage and Tollervey 2004; Connolly and Culver 2009).

Although eukaryotic ribosome subunits are in general larger than their prokaryotic counterparts (5500 total nucleotides and 78 Rps in *S. cerevisiae* versus 4567 total nucleotides and 58 Rps in *E. coli*), the high-resolution crystal structures of prokaryotic ribosomes fit the electron density from cryo-EM of eukaryotic ribosomes (Beckmann et al. 2001; Spahn et al. 2001; Chandramouli et al. 2008). This is because ribosomal proteins and rRNA structures unique to eukaryotes are largely restricted to surface “expansion segments,” while the secondary and tertiary structures of the core, including the active site, are well conserved across all domains of life (Woese et al. 1980; Winker and Woese 1991; Schnare et al. 1996; Cannone et al. 2002; Mears et al. 2002). Despite these structural similarities, genetic and proteomic studies paint very different pictures of the assembly processes in *E. coli* and *S. cerevisiae*. In contrast to the small number of nonessential assembly factors that have been identified in prokaryotes (for review, see Hage and Tollervey 2004; Connolly and Culver 2009), yeast ribosome assembly involves association with ~200 mostly essential accessory proteins that are not part of the mature ribosome structure (Fromont-Racine et al. 2003). In the absence of just one of these proteins, ribosome biogenesis is stalled, and cell growth is terminated even under optimal growth conditions.

Accessory factors required for eukaryotic ribosome assembly include NTP-dependent enzymes such as GTPases, ATPases, and kinases (Fig. 1). As such, these enzymes could stabilize or destabilize pre-ribosomal complexes at specific stages in the maturation through energetic manipulations during complex association and/or dissociation. They may also couple unfavorable maturation events to their favorable enzymatic activities. In the first part of this review, we

Reprint requests to: Katrin Karbstein, Department of Chemistry, University of Michigan, 930 N. University, Ann Arbor, MI 48109-1055, USA; e-mail: kkarbst@umich.edu; fax: (734) 647-1179.

Article published online ahead of print. Article and publication date are at http://www.rnajournal.org/cgi/doi/10.1261/rna.1792109.

Downloaded from majournal.cshlp.org on May 10, 2015 - Published by Cold Spring Harbor Laboratory Press

**Strunk and Karbstein**

---

have the ability to perturb free-energy landscapes. Finally, we discuss possible commonalities to explore why so much energy is released during ribosome assembly.

### CLASSES OF NTP-HYDROLYZING ENZYMES

Enzymes can change the free energy of a pre-ribosomal intermediate either by using chemical energy to perform mechanical work and remodel the RNA-protein complex (RNP), or by covalent modification of the intermediates. Of the ~200 ribosome assembly factors in *S. cerevisiae*, ~20% have GO terms that identify them as nucleoside triphosphate hydrolyzing enzymes such as GTPases, ATPases, and kinases, either through direct biochemical evidence or sequence homology with enzymes of known function. In addition, there are enzymes that utilize energy from non-nucleotide sources, at least in their direct interaction with the pre-ribosome. For example, endo- and exonucleases release energy stored in the phosphodiester bonds of the rRNA backbone. Additionally, methylases release the energy stored in S-adenosyl methionine (SAM) through methylation of rRNA. Finally, ubiquitin-conjugating enzymes and pseudouridylases likewise use high-energy bonds to modify components of pre-ribosomal complexes. Thus, the number of accessory factors with the potential to change the free-energy landscape of ribosome is higher than GO terms imply. The following subsections review the classes of energy-releasing factors playing essential roles in eukaryotic ribosome assembly.

#### GTPases

GTPases have been studied extensively as their activity is central to cellular signaling, transport, cytoskeleton organization, and translation (for reviews, see Bourne et al. 1991; Leipe et al. 2002). GTPases, like the other NTP-dependent enzymes discussed below, are P-loop NTPases. As such they contain a conserved nucleotide-binding domain, defined by the Walker A motif, responsible for binding of the α- and

---

**FIGURE 1.** rRNA cleavage involves many steps and energy-consuming factors. (Red arrows) Endonucleolytic steps; (blue arrows) exonucleolytic steps. The steps are labeled with the name of the nuclease if known or suggested, or the name of the cleavage site. Cleavage at site A₂ (Venema and Tollervey 1999; Karbstein 2009), cleavage at site D requires prior cleavage at site A₂ (Karbstein 2009), and processing to site B₁₅ requires prior cleavage at site A₃ (Karbstein 2009). For simplicity, only the major 60S processing pathway is shown. (Cyan) Putative ATPases (for simplicity, DEAD-box proteins are omitted); (green) GTPases; (orange) kinases.

provide an overview of the function of nucleotide-hydrolyzing enzymes in ribosome assembly. Next, we expand the commonly discussed classes of energy-releasing factors, to include RNA and protein modifying enzymes, which also

**2084 RNA, Vol. 15, No. 12**

β-phosphates, and the Walker B motif, responsible for coordination of a Mg²⁺ ion that binds to the β- and γ-phosphates and participates in GTP hydrolysis. The GTP- and GDP-bound forms of these proteins differ in two exposed loops (the switch I and II regions) located on the surface of the molecule. These loops form part of the interface with effector molecules rendering their binding sensitive to the nucleotide state. Because effectors bind more strongly in the GTP-bound state than the GDP-bound state, the GTP-bound form is considered the “active” state. Therefore, GTP hydrolysis “deactivates” the GTPase. GTP hydrolysis is stimulated by interaction with a GTPase activating protein (GAP). Unlike prototypical GTPases, the characterized GTPases in ribosome assembly nucleotides weakly and are thus not expected to be dependent on guanine nucleotide exchange factors to exchange GDP for GTP (Karbstein 2007).

Yeast ribosome assembly requires the action of six GTPases: Bms1 for assembly of the 40S subunit; Nog1, Nog2, Nug1, Lsg1, and Ria1 for assembly of the 60S subunit (Fig. 1, shown in green; Becam et al. 2001; Gelperin et al. 2001; Saveanu et al. 2001; Senger et al. 2001; Wegierski et al. 2001; Jensen et al. 2003; Kallstrom et al. 2003; Hedges et al. 2005; Karbstein et al. 2005; Bassler et al. 2006). Interestingly, GTPases form the largest class of essential assembly factors in bacteria (although their essential function is not thought to arise from ribosome assembly), suggesting that their role is conserved. Roles of GTPases in ribosome assembly have been recently reviewed in depth (Karbstein 2007) and will not be further discussed here.

### AAA⁺-ATPases

ATPases associated with diverse cellular activities (AAA⁺-ATPases) perform a wide range of functions including protein unfolding, dissociation of protein–protein interactions, and unidirectional translocation along a track. These proteins are characterized by their nucleotide-dependent oligomerization into (usually hexameric) ring structures and the extensive conformational change they undergo during the ATPase cycle (Erzberger and Berger 2006). It appears that these conformational changes are transduced within the protein and to its substrate via the function of two sensor elements (which correspond to the switch I and II in GTPases and are also found in ABC transporters and some helicases), although the exact mechanism of mechano-chemical coupling is still debated. Rix7, Rea1, and Drg1 are AAA⁺-ATPases essential for assembly of the large ribosomal subunit in yeast (Fig. 1, shown in cyan; Gadal et al. 2001; Galani et al. 2004; Pertschy et al. 2007). Depletion of Rix7 inhibits cleavage at the 3′-end of 5.8S rRNA, transport of pre-60S particles to the nucleus, and destabilizes 27SB pre-rRNA (Gadal et al. 2001). A synthetic lethal screen with the temperature-sensitive (ts) Rix7-31 identified a point mutation in Nsa1 (Fig. 2; Kressler et al. 2008). These two proteins also interact in a yeast two-hybrid screen, although direct in vitro interactions could not be detected. Furthermore, while the majority of Rix7 is free in the nucleus, a small portion can be found associated (substoichiometrically) with Nsa1-containing pre-ribosomes, but not with other pre-ribosomal particles, indicating that this interaction is transient. Whereas in wild-type strains Nsa1 dissociates from pre-ribosomal particles in the nucleus, in cells containing a ts N-terminal truncation (Rix7Δ14N), Nsa1 is retained on cytoplasmic ribosomes and is even detected in polysomes, suggesting that removal of Nsa1 from pre-ribosomes is disrupted at the nonpermissive temperature (Kressler et al. 2008). Additionally, pre-ribosomal particles purified via TAP-tagging of assembly factors that act after Nsa1 are missing proteins when

![Figure](attachment://image.png)

**FIGURE 2.** Model for Rix7-catalyzed remodeling of pre-60S subunits. Nsa1 binds to pre-60S subunits containing 27SA rRNA; other early 60S assembly factors including Rrp5, Noc1, and Nop4 are also bound to this intermediate. Rix7 interacts with Nsa1, potentially only in the ubiquitinated/sumoylated form, to remove it from pre-ribosomes (directly or indirectly). Nsa1 removal coincides with loss of the early 60S assembly factors Rrp5, Noc1, and Nop4, and allows binding of the later 60S assembly factors Rsa4, Nop53, Spb1, Nog2, Sda1, Arx1, and Nug1. (Orange) Energy-consuming assembly factors; (yellow) other assembly factors; (blue) Nsa1; (red) Rix7.

Downloaded from majournal.cshlp.org on May 10, 2015 - Published by Cold Spring Harbor Laboratory Press

**Strunk and Karbstein**

purified from the Rix7Δ14N background relative to wild-type strains, while pre-ribosomal particles purified via factors that act prior to Nsa1 accumulate additional proteins compared to the same particles from wild-type cells. Thus, recycling of Nsa1 coincides with a shift in the composition of nuclear pre-60S particles. Together, these data indicate that Rix7 interacts with a nuclear pre-60S particle via Nsa1 to facilitate recycling of Nsa1 and remodeling of the pre-ribosomal particle (Fig. 2; Kressler et al. 2008). It remains to be seen whether the shift in pre-ribosome composition is the cause or a consequence of Nsa1 removal.

Real1 is also required for 3′-maturation of 5.8S rRNA, but is a component of a later nuclear pre-60S particle that precedes association of the nuclear export adaptor Nmd3 (Galani et al. 2004; Nissan et al. 2004). Addition of ATP, but not AMPPNP, ADP, or GTP, to pre-ribosomes results in dissociation of Real1p and Nog2p from this particle and may reflect an essential pre-export remodeling step coupled to ATP hydrolysis by Real1p (Galani et al. 2004).

Drg1 associates with 60S pre-ribosomes in the cytoplasm (Pertschy et al. 2007). Inactivation of Drg1 leads to accumulation of Rlp24, Nog1, and Tif6 in the cytoplasm, and depletion from the nucleus (Pertschy et al. 2007). Accordingly, the rRNA processing defects of these cells resemble Nog1, Rlp24, or Tif6 depleted cells.

A common theme for all three AAA⁺-ATPases is their proposed role in shifting the composition of pre-ribosomal assembly intermediates. It is tempting to speculate that these proteins are involved in breaking a number of very strong protein–protein or RNA–protein interactions (as in protein unfolding or SNARE dissociation), which could require hydrolysis of multiple ATP molecules.

Although they share sequence similarity with helicases that couple ATP hydrolysis to unwinding of long duplexes, DExH/D proteins are not processive, consistent with the observation that RNA duplexes are typically short (Yang et al. 2007). Furthermore, DEAD-box proteins do not translocate through a substrate duplex. Rather, they use single-stranded regions as binding platforms from which they can separate a nearby duplex by binding anywhere within that duplex, destabilizing it locally (Yang et al. 2007; Chen et al. 2008). This leads to unwinding of no more than two helical turns of a duplex in a single binding event. The lack of processivity is also consistent with a new view of the mechanism by which DEAD-box helicases unwind duplexes. For at least a subset of DEAD-box proteins, ATP hydrolysis is neither coupled to nor required for duplex unwinding (Chen et al. 2008; Liu et al. 2008). Instead, duplex separation is induced by ATP-dependent binding to single-stranded RNA. ATP hydrolysis is required only for dissociation of the DEAD-box protein after unwinding. While the universality of this mechanism is still under investigation (especially for DExH proteins), it reconciles the finding that some DEAD-box proteins are merely ATP-dependent RNA-binding proteins (Shibuya et al. 2006). In this new model, all DEAD-box proteins are ATP-dependent RNA-binding proteins, some hydrolyzing ATP to release single-stranded RNA. This new mechanistic insight also is consistent with the observation that nucleotide hydrolysis itself is not irreversible for these and other NTP-hydrolyzing enzymes, as that step is barely energetically favorable (Hassett et al. 1982). Instead, it appears that dissociation of one (or more) of the products is what renders these processes irreversible (e.g., Baird et al. 2001; Algire et al. 2005).

Nineteen out of 37 DExH/D proteins encoded in the S. cerevisiae genome are involved in ribosome biogenesis, and 17 of these are essential (see Table 1; for review, see Bleichert and Baserga 2007). Seven (Dhr1, Dhr2, Dbp8, Rok1, Fap1, Rrp3, Dbp4) are required for SSU biogenesis (Song et al. 1995; O’Day et al. 1996; Liang et al. 1997; Venema et al. 1997; Colley et al. 2000; Daugeron and Linder 2001; Granneman et al. 2006a), while eight (Dbp2, Dbp6, Dbp9, Mak5, Drs1, Dbp10, Spb4, Mtr4) are required for LSU biogenesis (Sachs and Davis 1990; Ripmaster et al. 1993; de la Cruz et al. 1998a,b, 2004; Kressler et al. 1998; Burger et al. 2000; Bond et al. 2001; Bernstein et al. 2006). Prp43 and Has1 associate with, and are required for, both SSU and LSU biogenesis at stages after these subunits have entered separate processing pathways, suggesting that these proteins may have more general functions in the ribosome assembly process (Emery et al. 2004; Lebaron et al. 2005; Bernstein et al. 2006; Combs et al. 2006; Leeds et al. 2006; Liang and Fournier 2006). Interestingly, DEAD-box proteins are also among the identified bacterial assembly factors, pointing to conserved roles. Potential substrates in ribosome assembly include “misfolded” rRNA–rRNA duplexes (see

**DExH/D proteins**

DExH/D ATPases are named for a conserved amino acid sequence in the Walker B motif and are further subclassified into DEAD, DEAH, and DExH branches. These families demonstrate different nucleotide specificities and different levels of basal ATPase activity and are thus believed to have distinct modes of action (Jankowsky and Fairman 2007). Because the ATPase activity of many DExH/D proteins is enhanced in the presence of RNA, they are also referred to as RNA-dependent ATPases. Several DExH/D proteins unwind RNA duplexes or disrupt RNA–protein interactions. In some cases, they facilitate duplex annealing and even disruption of protein–protein interactions, and some are simply ATP-dependent RNA-binding proteins (for review, see Cordin et al. 2006; Jankowsky and Fairman 2007). All DExH/D proteins share a common fold created by two RecA-like helicase domains joined by a flexible linker (Tanner and Linder 2001; Cordin et al. 2006). Functional specificity is conferred by variable extensions flanking the helicase domains (Tanner and Linder 2001; Jankowsky and Fairman 2007).

2086 RNA, Vol. 15, No. 12

TABLE 1. DExH/D helicases involved in S. cerevisiae ribosome assembly

| Protein | Classification | Null phenotype | Target ribosomal subunit | Ribosome biogenesis/rRNA processing defect | Suggested function | Reference |
|---------|----------------|----------------|--------------------------|-------------------------------------------|--------------------|-----------|
| Dbp8p   | DEAD-box       | Inviable       | SSU                      | A0, A1, A2 inhibited$^a$                 |                    | Daugeron and Linder (2001) |
| Dbp4p   | DEAD-box       | Inviable       | SSU                      | A0, A1, A2 inhibited$^b$                 | U14 removal        | Liang et al. (1997); Kos and Tollervey (2005) |
| Dhr1p   | DEAH           | Inviable       | SSU                      | A1, A2 inhibited$^a$                     |                    | Colley et al. (2000) |
| Dhr2p   | DEAH           | Inviable       | SSU                      | A0, A1, A2 inhibited$^a$                 |                    | Colley et al. (2000) |
| Rrp3p   | DEAD-box       | Inviable       | SSU                      | A0, A1, A2 inhibited$^a$                 |                    | O'Day et al. (1996) |
| Rok1p   | DEAD-box       | Inviable       | SSU                      | A0, A1, A2 inhibited$^b$                 | snR30 removal      | Song et al. (1995); Venema et al. (1997); Bohnsack et al. (2008) |
| Fal1p   | DEAD-box       | Inviable       | SSU                      | A0, A1, A2 inhibited$^a$                 |                    | Kressler et al. (1997) |
| Prp43p  | DEAH           | Inviable       | SSU and LSU              | A0, A1, A2 inhibited, reduced 20S and 7S$^c$ | snoRNA removal     | Lebaron et al. (2005); Combs et al. (2006); Leeds et al. (2006) |
| Has1p   | DEAD-box       | Inviable       | SSU and LSU              | A0, A1, A2 inhibited, 27SA3 and 27SB accumulation$^b$ |                  | Emery et al. (2004) |
| Dbp3p   | DEAD-box       | Viable; synthetic slow growth with Dbp7p | LSU                                      | A3 inhibited or delayed, 27SA2 accumulation$^b$ |                   | Weaver et al. (1997) |
| Dbp6p   | DEAD-box       | Inviable       | LSU                      | Decreased 27SA2, 27SB, 7S$^b$            |                    | Kressler et al. (1998) |
| Dbp7p   | DEAD-box       | Slow growth, synthetic enhancement with Dbp3p | LSU                                      | Decreased 27SA2, 27SB, 7S$^b$ |                    | Daugeron and Linder (1998) |
| Dbp9p   | DEAD-box       | Inviable       | LSU                      | Decreased 27SA2, 27SB, 7S$^b$            |                    | Daugeron et al. (2001) |
| Mak5p   | DEAD-box       | Inviable       | LSU                      | 60S subunit deficit$^c$                  |                    | Zagulski et al. (2003) |
| Drs1p   | DEAD-box       | Inviable       | LSU                      | 60S subunit deficit$^c$                  |                    | Ripmaster et al. (1993) |
| Dbp2p   | DEAD-box       | Inviable       | LSU                      | 60S subunit deficit, suspected A3 cleavage defect$^c$ |                | Bond et al. (2001) |
| Spb4p   | DEAD-box       | Inviable       | LSU                      | 27SB accumulation$^c$                    |                    | Sachs and Davis (1990); de la Cruz et al. (1998a) |
| Dbp10p  | DEAD-box       | Inviable       | LSU                      | 27SB accumulation, C1 and C2 cleavage defect$^a$ |                  | Burger et al. (2000) |
| Mtr4p   | DExH           | Inviable       | LSU                      | 7S accumulation, required for exosome-mediated 3'-end processing of 5.8SrRNA | Part of the TRAMP complex | de la Cruz et al. (1998b) |

$a$ Dominant phenotype from point mutations after 48 h.  
$b$ Deletion phenotype.  
$c$ Point mutation or conditional mutant phenotype.

in yeast ribosome assembly. The three essential snoRNAs—U3, U14, and snR30—are required for early rRNA cleavage events (Tollervey 1987; Bally et al. 1988; Li et al. 1990; Morrissey and Tollervey 1993), while the other 72 are nonessential and specify rRNA residues for methylation or pseudouridylation (Decatur and Fournier 2003). Because the duplexes formed between snoRNAs and rRNAs are extensive and, based on nearest neighbor rules, are expected to be stable for years, it is anticipated that in vivo duplex dissociation is accelerated by RNA helicases. Thus, much effort has concentrated on identifying DExH/D/snoRNA pairs. An in vivo screen by Tollervey and colleagues examined accumulation of snoRNAs in pre-ribosomal particles when DExH/D proteins required for maturation of the SSU were individually depleted (Bohnsack et al. 2008). While none of the 72 nonessential snoRNAs accumulated significantly in high-molecular-weight fractions in this screen, U3 and U14 accumulated in the absence of multiple DExH/D proteins. This finding could indicate that multiple helicases cooperate to dissociate these essential snoRNAs, or it could indicate that indirect effects can complicate such screens, for example, if dissociation of an upstream inhibitory structure is required for downstream dissociation of a snoRNA. One of the helicases required for proper distribution of U14, Dbp4, has been previously implicated in modulating U14’s essential function through a genetic interaction with the U14 snoRNA (Liang et al. 1997), bolstering the argument that Dbp4 is required for U14 dissociation from pre-ribosomes (Kos and Tollervey 2005). Nevertheless, this example points to the importance of additional experiments to test and verify models suggested from such screens.

One such set of experiments includes the purification of native complexes associated with individual DExH/D proteins to narrow the field of potential substrates and cofactors of these proteins. This approach has been successfully used in the splicing field to assign substrates to RNA helicases. Interestingly, Prp43p, which was not included in the screen for snoRNA accumulation, coprecipitates many box-C/D snoRNAs and a few box-H/ACA snoRNAs. The efficiency of the co-IP is increased in a cold sensitive (cs) mutant at low temperature (Lebaron et al. 2005; Combs et al. 2006; Leeds et al. 2006). At least one rRNA residue targeted by Prp43-bound snoRNAs is also undermethylated when the cs strain is grown at low temperature (Leeds et al. 2006). These results suggest that Prp43 may play a general role in modulating snoRNP dissociation, perhaps specifically box-C/D snoRNA dissociation, from ribosome precursors. Interestingly, Prp43 (Lebaron et al. 2005), as well as the snoRNA components Nop1, Cbf5, Gar1, Nhp2, and Nop56/58, interact with many Pol I transcription factors (Cadwell et al. 1997; Faith et al. 2000; Krogan et al. 2004, 2006; Collins et al. 2007; Tarassov et al. 2008), indicating that snoRNA-directed rRNA modification (and snoRNA removal) could occur

cotranscriptionally at some sites, akin to cleavage at site A₂ (Osheim et al. 2004).

A final class of experiments to delineate helicase/substrate pairs utilizes in vitro studies that evaluate ATPase and helicase stimulation upon binding to specific RNA substrates. These experiments are based on the assumption that the RNA substrate will bind more strongly to the helicase and/or better stimulate its ATPase activity. They are inspired by early experiments from the Fuller-Pace laboratory, which demonstrated that the ATPase activity of the bacterial DEAD-box protein DbpA is highly stimulated by a very specific hairpin structure in the 23S rRNA (Fuller-Pace et al. 1993). The advantage of such experiments lies in the rigor by which unwanted secondary effects can be excluded in a controlled system. However, the length of yeast pre-rRNA (~7000 nucleotides), its potential for alternative interactions, and the abundance of snoRNAs suggest that identifying the “right” substrate might be akin to looking for a needle in a haystack. Consistent with this suggestion, a recent tour-de-force manuscript from the Uhlenbeck lab indicates that a specific RNA substrate was not found for any of the four tested helicases, despite screening of a large number of RNAs under a wide variety of conditions (Garcia and Uhlenbeck 2008). This finding shows that prior information will be required to fine-tune biochemical experiments in reconstituted systems. Interestingly, protein cofactors can increase the rate of ATP hydrolysis. For example, Ntr1 and Esf2 moderately stimulate the ATPase rates of Prp43 and Dbp8, respectively (Granneman et al. 2006b; Tanaka et al. 2007). These proteins may impart specificity, and knowledge of their binding sites may help in identifying helicase function.

While much progress has been made in developing tools to understand the roles that DExH/D proteins play in ribosome biogenesis, definitive assignments of substrates for these RNA-dependent ATPases will require combined approaches: Further in vivo investigations will be required to zone in on substrates and effectors for individual DExH/D proteins, which can then be included in biochemical experiments to extend and verify potential models.

### ABC-ATPases

ATP-binding cassette (ABC)-ATPases are characterized by ATP-dependent dimerization of two nucleotide-binding sites harbored within one or two polypeptides, such that two ATP molecules are sandwiched between the two domains (for review, see Hopfner and Tainer 2003; Higgins and Linton 2004). Each ATP is coordinated by residues from each opposing nucleotide-binding domain, and the two nucleotides make up approximately half of the area dimer interface (Hopfner and Tainer 2003). Conformational rearrangements induced by ATP-dependent dimerization are transmitted to an effector domain. Classic examples of ABC-ATPases are membrane-associated transporters.

However, there are soluble ABC-ATPases, two of which, Arbl and Rli1, are essential for ribosome assembly (Fig. 1, shown in cyan; Dong et al. 2004, 2005; Kispal et al. 2005; Yarunin et al. 2005). Mutation of conserved residues predicted to be involved in ATP binding and hydrolysis is lethal for both Rli1 and Arbl, suggesting that ATPase activity is central to the essential roles of both proteins (Dong et al. 2004, 2005). Rli1 was initially identified as a factor involved in translation initiation and copurifies with the multi-subunit initiation factor eIF3 via direct interaction with the Hcr1 subunit (Dong et al. 2004; Kispal et al. 2005; Yarunin et al. 2005; Andersen and Leavers 2007). Not incidentally, the phenotype from deletion of Hcr1 resembles that from deletion of Rli1 (Valasek et al. 2001; Dong et al. 2004).

In a highly unusual manner, Arbl and Rli1 are required for LSU and SSU rRNA processing and maturation; they are both shuttling proteins that mainly reside in the cytoplasm and are both required for export of nascent subunits (Dong et al. 2005; Kispal et al. 2005; Yarunin et al. 2005). Furthermore, while Arbl binds to pre-60S ribosomes, its depletion reduces the amount of 40S subunits relative to 60S. Similarly, Rli1 binds to pre-40S subunits, but its deletion reduces the amount of 60S subunits, indicating that these proteins may be involved in fine-tuning the relative levels of ribosomal subunits. Furthermore, the parallels in the observed phenotypes might reflect the similar functions of these factors.

While Arbl is an essential protein, its depletion only leads to moderate defects in SSU and LSU rRNA processing (Dong et al. 2005), suggesting either that it acts after rRNA processing is complete, or that its essential role is in a different cellular process (or both). Consistent with both of these possibilities, Arbl interacts with the very-late cytoplasmic 60S assembly factors Tif6 (see Fig. 5, below) and Lsg1, as well as with several translation initiation factors, including the eIF3 complex, eIF2A and eIF4A, as well as Zuo1 and Ded1, both of which are typically involved in translation (Dong et al. 2005). Rli1 depletion also has very moderate effects on pre-rRNA processing (Yarunin et al. 2005). In contrast, the effects on translation initiation are much stronger and even observed in extracts (Dong et al. 2004). Furthermore, it has been shown that deletion of Rli1 abolishes cell growth after only 4 h (less than two cell divisions), much faster than typically observed for ribosome assembly proteins. Because ribosomes are stable and effects on cell growth are only observed once existing pools of ribosomes are diluted via cell division, effects on cell growth are slow to emerge after depletion of ribosome assembly factors. The rapid manifestation of Rli1's growth defect further indicates that the essential function from Rli1 may be in translation initiation and not ribosome assembly.

Because both factors have such unusual pleiotropic effects on multiple steps in ribosome assembly and because they are both mainly cytoplasmic proteins that interact

with very late cytoplasmic pre-ribosomes, we speculate that both proteins function in an as-yet-undiscovered “handoff” between the ribosome assembly machinery and the translation initiation machinery, perhaps removing a final set of “blocking” assembly factors to allow for binding of initiation factors (see Fig. 5, below). This model would suggest the presence of a final check before ribosomal subunits are released into the pool of translating ribosomes. The modest effects on ribosome assembly could then be indirect and due to reduced translation of mRNAs, 30% of which encode ribosomal proteins (Warner 1999).

### Kinases

Kinases transfer a phosphate from ATP to a serine, threonine, or tyrosine (or, in rare cases, histidine) within a protein. This modification can affect a protein’s activity, ability to interact with other proteins, and cellular localization. Controlled protein phosphorylation by vast networks of kinases is central to the regulation and coordination of cellular processes including metabolism, growth, differentiation, and response to stress (Cohen 2000; Manning et al. 2002). Although many proteins involved in ribosome assembly are phosphorylated in vitro (Ptacek et al. 2005) and in vivo (Ficarro et al. 2002; Chi et al. 2007), the significance of these modifications and the kinases responsible for them remain largely unknown. Three essential kinases (shown in orange in Fig. 1) are involved in ribosome assembly in S. cerevisiae: Rio1 and Rio2 facilitate SSU assembly, and Hrr25 is required for maturation of both subunits.

The right open (Rio) reading frame kinases are conserved from archaea to humans (for review, see LaRonde-LeBlanc and Wlodawer 2005). Crystal structures of the archaeal Rio proteins indicate that these proteins possess a kinase-like fold, consistent with the conservation of amino acids required in all eukaryotic protein kinases (ePKs) (LaRonde-LeBlanc and Wlodawer 2004; LaRonde-LeBlanc et al. 2005b). However, there are several differences between Rio kinases and ePKs: The Rio kinase domain is exceptionally short, lacking loops and helices highly conserved in most ePKs. Among the missing features is the “activation loop,” which is regarded as essential for substrate binding and specificity in ePKs. Consistent with this finding, the structures of Rio kinases in the presence of nucleotide indicate that these proteins bind ATP in a more extended conformation than typical ePKs. Despite this divergence, it has been shown that Rio kinases can autophosphorylate (LaRonde-LeBlanc et al. 2005a,b), although the rates for these phosphorylation events relative to conventional ePKs have not been determined. The Rio1 and Rio2 subfamilies are distinguished by conserved amino acids in their kinase motifs, indicating perhaps a different mode of activity, as well as a conserved N-terminal winged helix domain, a nucleic-acid-binding motif, found only

in the Rio2 subfamily (LaRonde-LeBlanc and Wlodawer 2004).

Rio1 and Rio2 are both required for processing of the 20S rRNA. Although both bind to pre-ribosomal particles containing the 20S rRNA precursor (Geerlings et al. 2003; Vanrobays et al. 2003), only Rio2 is a stable component of these particles (Gavin et al. 2002; Schafer et al. 2003; Collins et al. 2007). Rio1 is not recovered with pre-ribosomes obtained by affinity purification of TAP-tagged accessory factors, indicating a transient association with pre-ribosomes, as expected if Rio1 phosphorylates a pre-ribosomal component and then dissociates. Consistent with such a role, mutations in Rio1's kinase domain that abolish in vitro kinase activity do not support growth (Angermayr et al. 2002). However, these data were obtained prior to knowledge of Rio1's essential role in ribosome assembly, and the effect of these mutations on ribosome assembly was not determined. As Rio1 is required for cell cycle progression (Angermayr et al. 2002), it cannot be ruled out that inactivation of Rio1's kinase activity disrupts an essential process other than ribosome assembly. In contrast, Rio2's kinase activity may not be integral to its essential role in ribosome assembly. A Rio2 mutant in which in vitro kinase activity is largely abolished can support growth, at least when overexpressed (Geerlings et al. 2003). Even though there are conflicting data concerning the ability of very low levels of wild-type but not kinase-deficient Rio2 to support growth (Geerlings et al. 2003; Vanrobays et al. 2003), it is likely that Rio2 is essential, but its kinase activity is not. This would be consistent with an inhibitory/regulatory role for this activity rather than an essential role. Persistent binding of Rio2 to the pre-40S complex may allow maintenance of a phosphorylated state even in the presence of competing phosphatase activity. Removal of Rio2 or inhibition of its kinase activity might then be required for assembly progression. Alternatively, Rio2 may simply be an ATP-dependent RNA-binding protein, perhaps modulated by its own phosphorylation.

Hrr25 is one of four yeast homologs of mammalian casein kinase I and has documented roles in the DNA damage response, vesicle budding, meiosis, and cell survival during stress (Hoekstra et al. 1991; Ho et al. 1997; Kafadar et al. 2003; Petronczki et al. 2006). Hrr25 binds to 90S and 40S, as well as to 66S pre-ribosomes, and its depletion leads to 18S and 25S processing defects (Schafer et al. 2006; Ray et al. 2008). Recent data from the Maitra and Tschocher laboratories suggest that cycles of phosphorylation and dephosphorylation, mediated by Hrr25, are required for export of both ribosomal subunits (Basu et al. 2003; Schafer et al. 2006; Ray et al. 2008). Hrr25 phosphorylates Tif6 at serine 174 in vitro and in vivo (Basu et al. 2003; Ray et al. 2008). Mutation of this residue to alanine prevents phosphorylation, traps Tif6 in the nucleus, and leads to defective 60S assembly (Basu et al. 2003). The S174D mutation, mimicking a constitutively phosphorylated protein, is also inviable, indicating that dephosphorylation of Tif6 is also

required for ribosome assembly (Basu et al. 2003; Ray et al. 2008). Similarly, phosphorylation and subsequent dephosphorylation of Enpl and Rps3 are required for stable incorporation of Rps3 into the small subunit (Schafer et al. 2006). This phosphorylation is abolished in Hrr25 knockout cells, indicating that Hrr25 is either directly or indirectly responsible. Because Rps3 is linked to a stabilization of the beak, which protrudes from the SSU, its stable incorporation could impede passage of this particle through the nuclear pore. Phosphorylation prior to and dephosphorylation following nuclear export might promote a conformational rearrangement that renders nuclear export irreversible (Schafer et al. 2006). Together, these data indicate that Hrr25 regulates (and perhaps synchronizes) the export of both subunits (Fig. 5, below).

### Other ATPases

Fap7, Utp14, and Kre33 (Fig. 1, shown in cyan) have sequence homology with P-loop-type ATPases but do not fall into one of the classes described above. Utp14 and Kre33 are associated with early nucleolar pre-ribosomes and are required for assembly of the 40S subunit. Utp14 is required for the early nucleolar rRNA processing steps A₀, A₁, and A₂, and its depletion leads to accumulation of the dead-end 23S rRNA, as found for most Utps (Gallagher et al. 2004). A mutant form of Kre33 is defective in pre-40S export to the cytoplasm (Grandi et al. 2002).

Fap7 is conserved from archaea to humans and bears weak sequence homology with adenylate kinases and AAA⁺-ATPases. Fap7 is required for cytoplasmic cleavage at site D to produce the mature 3′-end of the 18S rRNA (Figs. 1, 5, below; Granneman et al. 2005). Fap7 is believed to act through direct interaction with the small ribosomal protein Rps14, as these two proteins interact in vivo (Gavin et al. 2006) and in vitro (Granneman et al. 2005). Rio2, an essential kinase discussed above, accumulates in pre-40S complexes in Fap7-depleted cells (Granneman et al. 2005). Fap7 may thus be involved in removing Rio2 from the pre-40S particle.

### ENZYMES USING NON-NUCLEOTIDE ENERGY SOURCES

The enzymes discussed above promote ribosome assembly by binding and hydrolyzing nucleoside triphosphates. However, modification of any component in a pre-ribosomal complex alters the free energy of that complex. Furthermore, principles of metabolism demonstrate that thermodynamically unfavorable reactions can be driven by coupling them to thermodynamically favorable, irreversible reactions, as long as the net process is favorable. Likewise, unfavorable manipulations of the pre-ribosome could be coupled to favorable or irreversible modifications. Any such modification, therefore, has the potential to destabilize or

stabilize the formation of specific structures in the pre-
ribosome, enforcing directionality of the maturation pro-
cess. Outlined below are examples of such modifications in
ribosome biogenesis.

# Cleavage of polycistronic rRNA precursors

Synthesis of three rRNAs in a single transcript may be
a strategy that evolved to coordinate synthesis of these
molecules, reflecting the need for tight co-regulation of
ribosomal components. In addition, however, phospho-
diester cleavages during rRNA processing release energy
stored in these bonds and are thus irreversible modifica-
tions (especially if one of the released pieces is degraded via
the exosome or Rat1/Xrn1 nucleases). If rRNA cleavage
steps are coupled to unfavorable steps in the assembly
process, they could provide directionality to the maturation
process. Perhaps this is why rRNA cleavage is performed in
a stepwise fashion (Fig. 1), as it allows the energy to be
harnessed at multiple points. Incidentally, these cleavage
steps should then be regarded as potential regulatory
points, as shown for the favorable steps in metabolic
pathways. Consistent with the idea of cleavage steps as
energy-releasing control points, it has been shown that
Rcl1, the candidate for cleavage at site A₂, requires the
GTPase Bms1 for delivery to pre-ribosomes (Karbstein
et al. 2005). Furthermore, two Rio kinases and the ATPase
Fap7 regulate cytoplasmic cleavage at site D (Geerlings
et al. 2003; Vanrobays et al. 2003; Granneman et al. 2005).
The strict ordering of cleavage in polycistronic rRNA
precursors may be imposed to take advantage of energy
stored in these bonds at specific stages in the ribosome
assembly process. Additionally, or alternatively, it has been
suggested that ordering could help “proofread” the assem-
bly of ribosomes (Karbstein 2009). Finally, in the even
more complex mammalian system, there are more cleavage
steps, perhaps to allow for more irreversible steps that
could provide for control points.

# rRNA modifications

Modifications of specific rRNA residues, including pseu-
douridylation, 2′-O methylation, and base modifications
(usually methylation), occur in all domains of life (for
review, see Decatur and Fournier 2003). Many of these
modifications are clustered around conserved positions in
the active site of the mature ribosome, and without them
ribosome assembly and function are compromised (King
et al. 2003; Liang et al. 2007; Piekna-Przybylska et al. 2008).
Modest amounts of energy are released during each of
these essentially irreversible modifications: methylation ex-
changes the high-energy bond in the SAM cofactor (gen-
erated in an ATP-consuming process) for a more stable
bond to a rRNA base or ribose; pseudouridylation is an
isomerization that converts uridine bases to pseudouridine.
Pseudouridylation requires no energy input or cofactor but
releases energy through the exchange of the N-glycosidic
bond between the uracil and ribose for a lower-energy
carbon–carbon glycoside bond (Charette and Gray 2000).
Both modifications locally increase the structural stability
of the rRNA through enhanced intramolecular interactions
such as hydrogen-bonding potential and base-stacking
(Arnez and Steitz 1994; Charette and Gray 2000). Thus,
strategically placed modifications may help shift the ener-
getic landscape of the nascent ribosome to help enforce
directionality, thereby favoring maturation. Although no
single rRNA modification has been identified as essential,
global disruption of rRNA methylation by point mutations
in the modifying enzymes results in defects in ribosome
assembly and function (Tollervey et al. 1993). Furthermore,
it has recently been shown that loss of the hypermodifica-
tion of ψ1191 delays the conversion of 20S to 18S rRNA
(Liang et al. 2009).

Further supporting the potentially important regulatory
role of methylating enzymes are recent results obtained
with Dim1/KsgA and Emg1/Nep1. Dim1 is responsible for
the almost universally conserved dimethylation of two ade-
nines near the 3′-end of the mature 18S rRNA (Lafontaine
et al. 1995, 1998). Although this methylation event is not
strictly required, Dim1’s association with the complex is
essential (Lafontaine et al. 1995, 1998), and recent work
with the bacterial homolog KsgA indicates that its binding
prevents premature translation initiation (Xu et al. 2008),
pointing to an important regulatory role. Interestingly,
methylase activity is required to release KsgA (Connolly
et al. 2008). The role of the putative methylase Emg1/Nep1
is more nebulous. A lethal mutation in Nep1 can be rescued
by deletion of the snoRNA snR57, or by overexpression of
the ribosomal protein Rps19 (Buchhaupt et al. 2006). snR57
and Rps19 are predicted to bind near each other in the 3′
domain of 18S rRNA, which also contains an RNA sequence
identified in a three-hybrid screen with Nep1 (Fig. 3; Buchhaupt
et al. 2006). Together, these results suggest a model in which
Nep1 competes with snR57, to facilitate the incorporation
of Rps19, possibly by effecting a conformational switch in
the rRNA structure. In this model, snR57 negatively regulates
this conformational switch, while Nep1 is a positive regulator.
Rps19 is mutated in 25% of cases of the human blood disorder
Diamond Blackfan Anemia (Draptchinskaia et al. 1999), and
mutations in Nep1 cause Bowan Conradi Syndrome, a disease
with similar phenotypes (Armistead et al. 2009), further
supporting the importance of this step.

## Post-translational modifications of ribosomal proteins
and accessory factors

The principles that might allow rRNA modifications to
assist in driving ribosome assembly also apply to post-
translational modifications of proteins. Protein modifications

Downloaded from majournal.cshlp.org on May 10, 2015 - Published by Cold Spring Harbor Laboratory Press

**Strunk and Karbstein**

FIGURE 3. Model for a conformational change in the 3′-major domain of 18S rRNA induced by the methylase Nep1. snR57 binds to pre-18S rRNA (complex I) to methylate G1572, marked by an asterisk. This interaction prevents the formation of H33 in mature 18S ribosomes (shown in green in complex IV). Nep1 binds to the GCAACUU sequence upstream of H47 could displace snR57, consistent with suppression of the Nep1 deletion by deletion of snR57. The ensuing formation of H33 (shown in green) allows binding of Rps19 to its suggested site in H45, prior to dissociation of Nep1.

in many cases trigger or abolish interactions with complex signaling networks. A number of ribosomal proteins and ribosome assembly factors are post-translationally modified. These modifications include phosphorylation, methylation, acetylation, ubiquitination, and sumoylation. Of these, phosphorylation (discussed above), ubiquitination, and sumoylation are perhaps the most intriguing because of potential relationships with well-characterized pathways in which these modifications play regulatory roles. Ubiquitin is an essential protein of 76 amino acids that can be covalently conjugated to substrate proteins (Ciechanover and Schwartz 1994; Glickman and Ciechanover 2002). Ubiquitin conjugation begins with ATP-dependent activation of the ubiquitin polypeptide and culminates in ligation of ubiquitin’s C-terminal carboxylate to a lysine in the target protein. As there are lysines within ubiquitin,

polyubiquitin chains can be generated. In a similar process, sumoylation is the conjugation of a small ubiquitin-related modifier (SUMO) to lysines. Both of these modifications can be reversed enzymatically by deconjugating enzymes (Ciechanover and Schwartz 1994; Schwartz and Hochstrasser 2003). Polyubiquitination is frequently a signal directing proteasome-mediated degradation of the modified protein, but sumoylation, monoubiquitination, and polyubiquitination can also modulate protein function by altering conformation and/or ability to interact with other factors (Gill 2004; Kerscher et al. 2006; Ulrich 2008).

Ribosomal proteins and assembly factors have been identified in proteome-wide screens for ubiquitinated or sumoylated proteins (Peng et al. 2003; Panse et al. 2004, 2006). Accordingly, rRNA transcription, early and late processing steps, and nuclear export of both subunits are

2092 RNA, Vol. 15, No. 12

influenced by the ubiquitin–proteasome system (Panse et al. 2006; Stavreva et al. 2006). Below we explore examples of how ubiquitination or sumoylation affect ribosome assembly.

Three of the four genes for ubiquitin in yeast are fusion proteins with ribosomal proteins (Finley et al. 1989). Deleting the ubiquitin moiety from the Rps31-ubiquitin fusion leads to slow growth and accumulation of the 20S precursor to 18S rRNA, which is also incorporated into polysomes. Together, these results indicate that ubiquitination of Rps31 is important to ensure that only mature 40S subunits enter the pool of translating ribosomes (Lacombe et al. 2009).

Nsa1 exists in both ubiquitinated and sumoylated forms (Panse et al. 2006; Kressler et al. 2008). Interestingly, the AAA⁺-ATPase Rix7, which directly or indirectly removes Nsa1 from the pre-60S particle (Fig. 2; Kressler et al. 2008), is closely related to Cdc48, which powers the dissociation of ubiquitinated substrates from unmodified binding partners (DeLaBarre and Brunger 2005; Pye et al. 2006). It is speculated that ubiquitination and/or sumoylation of Nsa1 regulates its removal from the pre-ribosome by Rix7 (Kressler et al. 2008).

Sumoylation of the 60S assembly factor Ebp2 weakens its association with the ribosome assembly factors Loc1 and Nop12, leading to preferential binding of factors involved in nonribosomal processes, including a mating type switch (Shirai and Mizuta 2008). This finding suggests that sumoylation regulates Ebp2 binding to pre-60S ribosomes to allow cellular resources to be directed to other pathways under certain conditions.

F-box proteins provide target specificity to the Skp1-Cullin-F-box (SCF)-complex-mediated ubiquitination of proteins that are destined for degradation by the proteasome. The nucleolar F-box protein Dia2 interacts physically or genetically with 15 ribosome assembly factors (Ho et al. 2002; Blake et al. 2006; Pan et al. 2006), and its partners Cdc53 and Hrt1 bind three additional ones (Ho et al. 2002; Davierwala et al. 2005). In addition, Dia2 interacts with the cullin Rtt101 and Mms1. Both of these proteins have been implicated in the destruction of nonfunctional ribosomes (Fujii et al. 2009). We speculate that SCF complexes containing Dia2 and Rtt101/Mms1 or Cdc53/Hrt1 target pre-ribosomes for destruction, either as part of a passive, constant degradation pathway (Karbstein 2009), or as part of an active, targeted mechanism (Dez et al. 2006).

Together, these examples suggest that pre-ribosomes are polyubiquitinated and thereby targeted for degradation as part of quality-control pathways. In addition, ubiquitination and sumoylation are also used as reversible regulatory modifications, akin to phosphorylation.

(Schmid and Baldwin 1978; Schmid 1995). As enzymes, PPIs do not change the free energy of a system. However, because they accelerate the rate of interconversion and may thus render assembly processes irreversible via kinetic means (by speeding by a branch point and avoiding misfolding), we will consider them here. Interestingly, the nucleolar yeast PPI Fpr3 and the nuclear Fpr4 together interact with 10 different ribosome assembly factors in large-scale studies (Ho et al. 2002; Saveanu et al. 2003; Lebaron et al. 2005; Sydorskyy et al. 2005; Gavin et al. 2006; Collins et al. 2007; Oeffinger et al. 2007; Tarassov et al. 2008). This suggests that they could play a substantial role in facilitating ribosome assembly. Furthermore, knock-down of the human PPI parvulin affects assembly of both ribosomal subunits (Fujiyama-Nakamura et al. 2009), indicating that peptide bond isomerization could be important for ribosome assembly kinetics.

### WHY USE SO MUCH ENERGY?

In prokaryotes, in vitro reconstitution of functional ribosomal subunits from purified or in vitro transcribed rRNAs and recombinant or purified Rps has been demonstrated for both subunits with variable efficiency, and assembly maps for both subunits have been constructed (Traub and Nomura 1968; Mizushima and Nomura 1970; Held et al. 1974; Nierhaus and Dohme 1974; Rohl and Nierhaus 1982; Herold and Nierhaus 1987; Green and Noller 1999; Culver and Noller 2000). In these systems an energy barrier can be overcome by heating or raising salt concentrations (Traub and Nomura 1968). Thus, assembly of the prokaryotic ribosome (at least to the RI assembly intermediate) is determined and driven by the intrinsic properties of its component rRNAs and proteins. Consistent with this finding, only few assembly factors (other than rRNA-modifying enzymes and the nucleases required for cleavage of the polycistronic rRNA transcript) have been identified in prokaryotes (Maki et al. 2002; Iost and Dreyfus 2006; Jiang et al. 2007; Karbstein 2007). These assembly factors include DEAD-box ATPases and GTPases. While these cofactors enhance ribosome assembly in vivo, they are nonessential under normal growth conditions, which is perhaps surprising given the requirement for energy input into the in vitro system. Either these factors are redundant, or aspects of the in vivo assembly process eliminate the need for energy input. For instance, cotranscriptional assembly could simplify the assembly process by limiting potential interactions of the growing oligonucleotide: Correct rRNA structures can form before additional sequences that lead to off-pathway interactions are transcribed (Dammel and Noller 1993; Adilakshmi et al. 2008; Ramaswamy and Woodson 2009). Additionally, each cleavage step of the polycistronic rRNA transcript in vivo produces irreversible steps as discussed above. Stages of structural maturation may be coupled to these cleavage

Protein folding: Protein-prolyl-isomerases (PPIs)

PPIs catalyze the cis-trans-isomerization of X-Pro peptide bonds, the rate-limiting step in folding of many proteins

events imposing directionality on this process. Finally, subcellular compartmentalization, for example, tethering of the nascent pre-ribosomal complex to the rDNA from which it is being transcribed, may allow early maturation steps to be completed before components localized elsewhere are encountered, again minimizing the potential for off-pathway conformations (Fromont-Racine et al. 2003; Hage and Tollervey 2004).

If prokaryotic and eukaryotic ribosomes do not differ extensively in complexity and prokaryotic ribosomes can largely self-assemble, why are so many energy-releasing enzymes required in eukaryotic ribosome assembly? The large number of essential assembly factors in eukaryotes may explain the need for energy input during the assembly process. By definition, ribosome assembly factors are temporarily bound to pre-ribosomes and not found in mature ribosomes. Numerous studies indicate that individual proteins associate with and dissociate from pre-ribosomal complexes at specific stages throughout assembly, often as part of smaller subcomplexes (for review, see Henras et al. 2008). If ribosome binding by an assembly factor is thermodynamically stable at one point but unfavorable at another, the relative free energies of the bound and dissociated states must be altered in the meantime. Energy-releasing assembly factors could accomplish this. But then we are left with the question: Why do eukaryotes power an essentially spontaneous process? Here we consider three (not mutually exclusive) answers: (1) intracellular export, (2) quality control, and (3) integration with other cellular processes. It should be noted that some of these have been outlined previously by Tollervey and coworkers (Hage and Tollervey 2004).

**Intracellular transport**

Eukaryotes bear the added burden of exporting maturing ribosomal subunits from the nucleus to the cytoplasm. While this certainly accounts for some of the additional energy input as exemplified by Hrr25, it does not explain all of it. In fact, the separation of early and late stages of ribosome assembly by the nuclear envelope should simplify the assembly process by preventing encounter of the nascent rRNA with late-binding, cytoplasmic, ribosomal proteins (Hage and Tollervey 2004; Karbstein 2009), and by conferring directionality to the process.

**Quality control**

One explanation for the great expansion of ribosome assembly factors in eukaryotes is that the greater complexity of eukaryotic cellular processes and tighter regulation of gene expression demand greater fidelity of translation. Mistakes in translation may have far more catastrophic implications for a multicellular organism than for a bacterium. In addition, 80% of mRNAs in yeast are present in less than two copies/cell (Wang et al. 2002), suggesting that sequestering of a transcript could have profound consequences on the proteome of a cell (Karbstein 2009). Eukaryotic accessory factors may facilitate more stringent quality control in ribosome biogenesis or, minimally, provide alternative mechanisms for this control (Hage and Tollervey 2004). Outlined below are potential quality-control mechanisms that require energy input into ribosome assembly.

**Alternative RNA structures**

rRNA sequences may have evolved to prevent aberrant or premature rRNA duplex formation through sequestering of key rRNA segments in alternative duplexes either within the nascent rRNA or with specific snoRNAs. Such a role has been proposed for the essential snoRNA U3, whose binding prevents the formation of the central pseudoknot, an essential tertiary structure motif in the SSU (Hughes 1996). Recent data suggest such a similar role for another essential snoRNA, snR30 (Fig. 4). Kiss and coworkers recently showed that snR30 binds two elements in ES6, an expansion segment present in eukaryotic rRNA (Fayet-Lebaron et al. 2009). Intriguingly, one of the two snR30 base-pairing sites in ES6 has previously been suggested to form a long-range tertiary interaction with ES3, which would tie together the central and 5′-domains via an interaction on the “foot” of the small ribosomal subunit (Alkemar and Nygard 2003, 2004, 2006). Binding of snR30 would prevent this interaction, and the timing of its release

![Figure 4](https://i.imgur.com/yourimageurl.png)

**FIGURE 4.** Dissociation of snR30 leads to a conformational switch in pre-ribosomes. Interaction of snR30 (red) with expansion segment 6 (ES6) in 18S rRNA (Fayet-Lebaron et al. 2009) prevents the proposed interaction between ES6 and ES3 (green), as well as a suggested pseudoknot in ES6 (blue) (Alkemar and Nygard 2003, 2006).

is likely to coincide with a major folding event, perhaps even the release of U3 snoRNA, as formation of the central pseudoknot is also expected to position the subdomains of 18S rRNA with respect to each other. The activity of specialized helicases would be required to disrupt these duplexes and might explain the function of at least a subset of the 17 DExH/D proteins in ribosome assembly. Analogously, during spliceosome assembly, the helicase Brr2 disrupts the stable U4/U6 duplex to allow for formation of the alternative U2/U6 duplex, which forms the active site (Raghunathan and Guthrie 1998). Interestingly, the Brr2 helicase is controlled in multiple layers via Prp8 (Small et al. 2006), a long-enigmatic protein at the heart of the spliceosome, as well as by ubiquitination (Bellare et al. 2008). Analogous regulatory switches may exist in ribosome assembly. Each helicase may require “specificity” factors, thus explaining the existence of a further subset of the nearly 200 eukaryotic accessory factors.

### Checkpoints

Binding of accessory factors may also prevent downstream events by physically blocking the binding of ribosomal proteins or other assembly factors. This could be used as a quality-control mechanism if correct completion of a certain assembly step is required for removal of the roadblock. Such a mechanism could account for the presence of assembly factors as well as energy-releasing enzymes to remove them. As described above, the binding sites for RNA methylase KsgA/Dim1 and translation initiation factor IF3 overlap, suggesting that KsgA/Dim1 binding prevents translation initiation by immature ribosomes. This roadblock has a built-in switch, as methylation is required for KsgA release (Connolly et al. 2008). It is tempting to speculate that similar roles are played by Rlp7, Rlp24, and Mrt4, ribosome assembly factors with high sequence similarity to the ribosomal proteins Rpl7, Rpl24, and P0, respectively. In this model, removal of assembly factors is required for incorporation of essential ribosomal proteins and could thus provide a checkpoint for correct assembly.

### Cooperativity in binding of assembly factors

Literature data suggest that the ribosome assembly machinery is organized in a highly cooperative manner. While the enzymatic function of several proteins is dispensable (at least for a while), deletion of these proteins leads to severe assembly defects (Tollervey et al. 1991; Lafontaine et al. 1995; Venema and Tollervey 1996; Lafontaine et al. 1998; Torchet et al. 1998; Eppens et al. 1999; Jakovljevic et al. 2004; Ferreira-Cerca et al. 2005; Bleichert et al. 2006). For example, it has been shown that inactivating Dim1 has no effects on ribosome assembly, while deletion is lethal (Lafontaine et al. 1995, 1998). Furthermore, electron microscopy has shown in several cases that the SSU processosome at the 5′-end of nascent transcripts is disrupted when just one protein is missing (Dragon et al. 2002; Osheim et al. 2004; Hoang et al. 2005; Segerstolpe et al. 2008; Vanrobays et al. 2008). This suggests that there is substantial cross talk between factors. This observation could account for the many “glue” proteins, which have no function detectable by bioinformatics other than to bind to other proteins. One possible reason for this remarkable cooperativity is the heightened potential for quality control, as all aspects of the nascent ribosome can be checked simultaneously and forward progress can be stalled if mistakes were made. A less cooperative machinery might allow assembly to progress even if one factor is missing. Interestingly, this observation could also indicate that removal of just one protein would destabilize the entire complex, explaining, perhaps, why the use of the powerful hexameric AAA⁺-ATPases is restricted to later 60S assembly intermediates, which may be assembled less cooperatively.

### Degradation of misassembled ribosomes

A passive kinetic quality control system has been proposed, wherein each step of ribosome assembly competes with degradation (Karbstein 2009). In that model, the longer a ribosomal intermediate takes to transition from one stage to another, the more likely it is to be degraded. This mechanism would lead to quality control, as assembly mistakes or missing factors slow down conversion steps, sending a larger fraction of assembly intermediates to the decay pathway. Consistent with such a view, literature data show 10 physical interactions between energy-consuming regulatory ribosome assembly factors and the TRAMP complex or the exosome, including three with the small subunit assembly factors Kre33, Dhr1, and Rli1 (Davierwala et al. 2005; Krogan et al. 2006; Collins et al. 2007; Tarassov et al. 2008; Wilmes et al. 2008). A passive quality-control system may be enhanced by energy input according to the classic principle of kinetic proofreading (Hopfield 1978). Binding of specialized “assembly” factors to undesired structures may also trigger active degradation (Dez et al. 2006, 2007; Wery et al. 2009).

### Multilayered control mechanisms for late cytoplasmic assembly intermediates

The cytoplasmic assembly steps of both ribosomal subunits are especially highly regulated (Fig. 5). Cytoplasmic maturation of 40S subunits involves the action of two ATPases, Fap7 and Rli1, and all three kinases in ribosome assembly, Rio1, Rio2, and Hrr25. Similarly, cytoplasmic maturation of 60S subunits requires the activity of two GTPases (Ria1 and Lsg1), two ATPases (Drg1 and Arb1), and the kinase Hrr25. The necessity for the intricate regulation of these late steps may arise because cytoplasmic pre-ribosomes are largely assembled and likely to be at least partially functional.

Downloaded from majournal.cshlp.org on May 10, 2015 - Published by Cold Spring Harbor Laboratory Press

**Strunk and Karbstein**

ribosomes. Mutations in human Sdol cause Shwachman–Diamond Syndrome (Shammas et al. 2005), a rare blood disease that predisposes to cancer and is caused by bone marrow failure, providing further evidence for the importance of regulation in Tif6 removal.

### Integration with other cellular processes

Another reason for the expanded repertoire of eukaryotic ribosome assembly factors may be the increased need to coordinate ribosome assembly with other cellular processes such as stress response, growth state, cell cycle progression, etc. These points of cross talk are most likely to occur at the energy-releasing steps, which impart the ability to temporarily stall this process at specific stages under adverse conditions. Some examples of such cross talk have even been documented in bacteria, where GTPases involved in ribosome assembly also regulate cell cycle progression and sporulation (Karbstein 2007). In principle, such cross talk could occur via two means. First, components shared between ribosome assembly and other cellular processes could provide for direct cross talk between such processes. Second, signaling networks could affect ribosome assembly and other processes. Such signaling networks may involve kinases or other post-transcriptional modifications such as ubiquitination. Data in the literature indicate that both mechanisms occur, as outlined below.

#### Cross talk of ribosome biogenesis through shared components

Examples of cross talk between ribosome assembly and other cellular processes mediated through shared components include replication, rRNA transcription, tRNA export, cell cycle control, and stress response (e.g., Angermayr and Bandlow 2002; Du and Stillman 2002; Steiner-Mosonyi et al. 2003; Killian et al. 2004; Bernstein et al. 2007; Rudra et al. 2007; Strub et al. 2007; Jwa et al. 2008), and this cross talk is more evolved in higher organisms. Because the connection to the cell cycle has been recently and expertly reviewed (Dez and Tollervey 2004; Granneman and Baserga 2005), it will not be covered here.

Fap7, Yar1, and Ltv1 have been implicated in late 40S assembly and the response to oxidative and osmotic stress (Loar et al. 2004; Granneman et al. 2005; Schafer et al. 2006). Deletion of the nonessential Yar1 and Ltv1 render yeast hypersensitive to oxidative stress, while mutations in Fap7 were identified in a screen for mutants defective in activation of the Skn7 transcription factor, which regulates both the osmotic and oxidative stress response (Juhnke et al. 2000). Furthermore, oxidative stress also leads to nuclear accumulation of Dim2 (Vanrobays et al. 2008). These results suggest that oxidative stress inhibits 40S assembly, perhaps not unexpected, given the observation that oxidative stress downregulates transcription of Rps and ribosome assembly factors (Gasch et al. 2000).

**FIGURE 5.** Late cytoplasmic assembly intermediates are regulated in multiple layers. (Blue) 40S subunits; (green) 60S subunits; (orange) energy-consuming assembly factors; (yellow) select other assembly factors. Phosphorylation of Enp1, Rps3, and Tif6, catalyzed by Hrr25, and subsequent dephosphorylation, are required for maturation and export of the 40S and 60S subunits, respectively. Nob1-dependent cleavage at the 3′-end of 18S rRNA is regulated by the kinases Rio1 and Rio2, and the ATPase Fap7. The binding site for the bacterial homolog of Dim1, KsgA, overlaps with the binding site for IF3, indicating that Dim1-containing ribosomes cannot initiate translation. Rli1 binds to both pre-ribosomes and eIF3, indicating a link between assembling and initiating ribosomes. Release of Tif6 requires binding of the Shwachman-Diamond protein Sdo1, perhaps to recruit the GTPase Ria1, which can remove Tif6 from ribosomes in vitro. In addition, Drg1 is required to release Tif6 from pre-ribosomes. Arbl binds to pre-ribosomes containing Tif6, but also interacts with initiation factors, linking the ribosome assembly and translation machineries.

Furthermore, they can encounter mature subunits to join for translation, if not properly safeguarded. In this context, the number of regulatory interactions with Tif6 is especially intriguing, because Tif6 binding to 60S ribosomes prevents the binding of 40S subunits (Si and Maitra 1999). Thus, removal of Tif6 should only occur if ribosomes are fully matured. Tif6 removal is likely catalyzed directly by the GTPase Ria1 (Senger et al. 2001); however, the ATPase Drg1 appears to be upstream of this process as mutations also lead to Tif6 accumulation in cytoplasmic pre-ribosomes (Pertschy et al. 2007). Tif6 is also the direct target of the export-regulating kinase Hrr25 (Basu et al. 2003; Ray et al. 2008). Finally, the severe growth defect caused by deletion of Sdol can be rescued by mutations in Tif6p that weaken its association with the pre-60S particle (Menne et al. 2007). Sdo1 interacts with Ria1 (Krogan et al. 2006), suggesting that it may help recruit Ria1 to pre-

2096 RNA, Vol. 15, No. 12

Co-regulation of pathways

Co-regulation of biochemical pathways can also occur via simultaneous phosphorylation of components of multiple pathways via the same kinase. While there is good evidence that ribosome assembly is regulated by phosphorylation, the exact kinase cascades and, indeed, the targets of that phosphorylation have not been determined in most cases.

Nutrient availability affects transcription rates of rRNA and r-protein mRNAs (Ju and Warner 1994; Gasch et al. 2000). Furthermore, splicing of ribosomal proteins is also sensitive to nutrient availability (Pleiss et al. 2007). In addition to the production of ribosomal components, ribosome assembly is directly regulated by the Tor-pathway (Powers and Walter 1999), although the exact mechanisms remain obscure. Localization of the RNA-binding proteins Dim2, Rrp12, and the GTPase Nog1 are affected by inhibition of this pathway (Honma et al. 2006; Vanrobays et al. 2008). However, it is not clear that either protein is a target of the Tor kinase (or downstream effectors), and the observed effects might be due to phosphorylation and inactivation of other ribosome assembly factors. Nevertheless, these data provide strong evidence that assembly of both ribosomal subunits is directly responsive to nitrogen deprivation and inhibition of the Tor pathway.

Intriguingly, Atg1, another Tor-target and a regulator of the autophagy pathway (a starvation response), phosphorylates 11 ribosome assembly factors in vitro (although eight of them are nuclear proteins and Atg1 is thought to be a cytosolic protein) (Table 2; Ptacek et al. 2005). Furthermore, Gcn2, a kinase activated by amino acid deprivation, and its cofactors Gcn1 and Gcn20 interact genetically as well as physically with more than 20 ribosome assembly factors, including four nucleotide-hydrolyzing enzymes, indicating that these proteins may regulate ribosome assembly (Table 2; Gavin et al. 2002, 2006; Schafer et al. 2003; Krogan et al.

2006; Collins et al. 2007; Wilmes et al. 2008). We speculate that nutrient stress may result in phosphorylation and inactivation of ribosome assembly proteins.

Perhaps most suggestive is the potential co-regulation of ribosome assembly and invasive growth, a response to nutrient stress, during which yeast cells fail to separate after division, creating long multicellular chains that invade their growth medium. Recent work has shown that the non-essential kinase Ksp1 is required for the invasive growth pathway (Bharucha et al. 2008). Ksp1 translocates to the nucleus during invasive growth, and nuclear localization is required for invasive growth, suggesting that at least a subset of its substrates are nuclear proteins (Bharucha et al. 2008). Ksp1 phosphorylates eight 40S and four 60S assembly factors in vitro (Ptacek et al. 2005) and interacts with the 60S assembly factors Sdo1 and Dbp7 in vivo (Table 2; Ho et al. 2002; Krogan et al. 2006). Furthermore, reduction of Rps26 and deletion of the 40S component Ascl both abolish the invasive growth pathway (Strittmatter et al. 2006; Valerius et al. 2007). Finally, a large-scale mutagenesis approach has identified genes required for invasive growth. While ribosome assembly factors are under-screened (because they are essential), mutations in Utp5, Utp7, Utp10, Brix1, and Nog2, as well as in 10 r-proteins with two orthologs (including Rps26), abolish invasive growth, indicating that ribosome assembly is essential for invasive growth (Jin et al. 2008). This is perhaps surprising given that invasive growth is a response to nitrogen limitation. We speculate that invasive growth and ribosome assembly are somehow co-regulated via the action of Ksp1. Alternatively, it is possible that Ksp1 up-regulates ribosome assembly under conditions that induce invasive growth, and that the reduction in ribosomal subunits is responsible for the observed invasive growth deficiency in Ksp1 mutant strains.


TABLE 2. Interactions between energy-consuming assembly factors and protein kinases

| Kinase | Process | 40S Assembly Factor |
|--------|---------|---------------------|
|        |         | Prp43 | Utp14 | Dbp4 | Has1 | Dhr1 | Dhr2 | Kre33 | Fal1 | Dbp8 | Rrp3 | Bms1 | Rok1 | Fap7 | Rio1 | Rio2 | Hrr25 | Ril1 |
| Gcn2   | Starvation | Gcn1 |  |  |  |  |  | Gcn1 |  |  |  |  |  |  | Gcn2 |  |  |  |
| Atg1   | Autophagy |  |  |  | Atg1 | Atg1 |  |  |  |  |  |  |  |  |  |  |  |  |
| Ksp1   | Filam. gro. |  |  |  |  |  |  |  |  |  |  |  |  | Ksp1 | Ksp1 | Ksp1 |  |  |
| PKC    | Growth | Bcy1 |  | Tpk1 | Tpk1 |  |  |  |  |  |  |  |  |  | Cka1,2 |  |  |  |
| CK2    | Growth |  |  |  |  |  |  | Cka1 |  |  |  |  |  |  |  |  | Cka2 |  |

| Kinase | 60S Assembly Factor |
|--------|---------------------|
|        | Mak5 | Dbp3 | Dbp6 | Dbp7 | Dbp9 | Drs1 | Dbp10 | Dbp2 | Nog1 | Nog2 | Nug1 | Spb4 | Rix7 | Rea1 | Drg1 | Lsg1 | Ria1 | Arbl |
| Gcn2   | Gcn1 |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Atg1   |  |  |  | Atg1 | Atg1 |  |  |  |  |  |  |  |  |  |  |  |  |  |
| Ksp1   |  |  |  | Ksp1 |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| PKC    | Tpk3 |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
| CK2    |  |  | Ckb1 |  |  |  |  |  | Tpk1 | Tpk1 | Cka1 | Cka1 | Cka1 | Cka1 |  |  |  |  |

Physical interactions are shown in red, in vitro phosphorylation is shown in yellow, genetic interactions are shown in purple, and interactions in protein complementation assay are shown in blue. NTP-hydrolyzing assembly factors are color-coded dark blue for DExH/D ATPases, light blue for other ATPases, green for GTPases, and orange for kinases. Filamentous growth is abbreviated as “Filam. gro.”

Downloaded from majournal.cshlp.org on May 10, 2015 - Published by Cold Spring Harbor Laboratory Press

**Strunk and Karbstein**

Table 2 also illustrates interactions between growth-promoting pathways and ribosome assembly, including the protein kinase C (PKC) and casein kinase 2 (CK2) pathways. PKC binds or phosphorylates six energy-consuming ribosome assembly factors (Table 2), and PKC phosphorylates 16 additional assembly factors in vitro (Ptacek et al. 2005). In addition, the RNase III Rnt1 interacts with the PKC regulatory subunit Bcy1 (Tarassov et al. 2008). While most of these interactions are not validated, they are consistent with the known influence of the PKC signaling pathway on transcription of ribosomal components.

Data in the literature document physical interactions between CK2 and eight energy-consuming assembly factors (Table 2), and 17 other assembly factors. Interactions with Utp18 and Utp22 are found with each one of CK2's four subunits, and Nop7 and Nop12 show genetic or direct physical interactions with three subunits. The interaction between Utp22, Rrp7, CK2, and the r-protein transcription factor Ifh1 has been validated, and it has been suggested to provide an interface for co-regulation of rP transcription and ribosome assembly (Rudra et al. 2007). Furthermore, interactions between Rio1 and CK2 depend on Rio1's C-terminal tail, and CK2 phosphorylates six serines in that tail in vivo and in vitro (Angermayr et al. 2007). Serine-to-alanine mutation produces a slow-growth phenotype and leads to cell cycle phenotypes similar to those seen in the Rio1 deletion (Angermayr et al. 2007). Finally, phosphorylation activates Rio1's kinase activity, indicating that the CK2 pathway could activate 40S assembly by modulating Rio1's activity in response to optimal growth conditions.

ribosome assembly is regulated in living cells. This will be a prerequisite for targeting this pathway with drugs, a potentially fruitful area, due to the importance of ribosome assembly for cell growth. Furthermore, the link between defects in ribosome assembly and cancer marks this pathway as a new target for anti-cancer drugs. As described above, the literature points to several pathways as potential regulators. It will be exciting to learn which assembly factors are targeted, if multiple targets exist, and what the composition of these signaling networks is.

## ACKNOWLEDGMENTS

B.S.S. is supported by an NSF pre-doctoral fellowship. Research on energy-consuming ribosome assembly factors in our laboratory is supported by an NSF-CAREER award (MCB0845156) and a young investigator award from the American Heart Association (0835141N). We thank members of our laboratory for comments on the manuscript.

## REFERENCES

Adilakshmi T, Bellur DL, Woodson SA. 2008. Concurrent nucleation of 16S folding and induced fit in 30S ribosome assembly. *Nature* **455**: 1268–1272.

Algire MA, Maag D, Lorsch JR. 2005. π Release from eIF2, not GTP hydrolysis, is the step controlled by start-site selection during eukaryotic translation initiation. *Mol Cell* **20**: 251–262.

Alkemar G, Nygard O. 2003. A possible tertiary rRNA interaction between expansion segments ES3 and ES6 in eukaryotic 40S ribosomal subunits. *RNA* **9**: 20–24.

Alkemar G, Nygard O. 2004. Secondary structure of two regions in expansion segments ES3 and ES6 with the potential of forming a tertiary interaction in eukaryotic 40S ribosomal subunits. *RNA* **10**: 403–411.

Alkemar G, Nygard O. 2006. Probing the secondary structure of expansion segment ES6 in 18S ribosomal RNA. *Biochemistry* **45**: 8067–8078.

Andersen DS, Leivers SJ. 2007. The essential *Drosophila* ATP-binding cassette domain protein, pixie, binds the 40 S ribosome in an ATP-dependent manner and is required for translation initiation. *J Biol Chem* **282**: 14752–14760.

Angermayr M, Bandlow W. 2002. Rio1, an extraordinary novel protein kinase. *FEBS Lett* **524**: 31–36.

Angermayr M, Roidl A, Bandlow W. 2002. Yeast Rio1p is the founding member of a novel subfamily of protein serine kinases involved in the control of cell cycle progression. *Mol Microbiol* **44**: 309–324.

Angermayr M, Hochleitner E, Lottspeich F, Bandlow W. 2007. Protein kinase CK2 activates the atypical Rio1p kinase and promotes its cell-cycle phase-dependent degradation in yeast. *FEBS J* **274**: 4654–4667.

Armistead J, Khatkars Meyer B, Mark BL, Patel N, Coghlan G, Lamont RE, Liu S, Wiechert J, Cattini PA, et al. 2009. Mutation of a gene essential for ribosome biogenesis, Emgl, causes Bowen-Conradi syndrome. *Am J Hum Genet* **84**: 728–739.

Arnez JG, Steitz TA. 1994. Crystal structure of unmodified tRNA<sup>Gln</sup> complexed with glutaminyl-tRNA synthetase and ATP suggests a possible role for pseudo-uridines in stabilization of RNA structure. *Biochemistry* **33**: 7560–7567.

Baird CL, Gordon MS, Andrenyak DM, Marecek JF, Lindsley JE. 2001. The ATPase reaction cycle of yeast DNA topoisomerase II. Slow rates of ATP resynthesis and P<sub>i</sub> release. *J Biol Chem* **276**: 27893–27898.

Bally M, Hughes J, Cesareni G. 1988. SnR30: A new, essential small nuclear RNA from *Saccharomyces cerevisiae*. *Nucleic Acids Res* **16**: 5291–5303.

## PERSPECTIVE

Work over the last decade has re-created the ribosome assembly field, demonstrating that in eukaryotes, ribosome assembly is much more complex than indicated by the facile reconstitution of ribosomal subunits from bacteria. Almost 200 protein and many RNA cofactors for ribosome assembly have been identified, most of which are essential in vivo, and it is likely that the identification of ribosome assembly factors is close to complete. In addition, for many assembly factors, it is known whether they are required for assembly of the small or large ribosomal subunit and roughly at which step in the respective assembly pathway they act. More recently, work has started to identify subcomplexes, functional and physical units within assembling ribosomes. This work is the first step in attacking the next frontier: establishing the function for these assembly factors. Much work to date has focused on the NTP-dependent factors discussed herein, simply because of their potential importance as regulatory enzymes, and much progress has been made. Nevertheless, the molecular details of the function of many proteins and how their interplay is managed remains elusive and will be exciting to follow in the future. A second frontier is to understand how

2098 RNA, Vol. 15, No. 12

Bassler J, Kallas M, Hurt E. 2006. The Nug1 GTPase reveals and N-terminal RNA-binding domain that is essential for association with 60S pre-ribosomal particles. *J Biol Chem* **281**: 24737–24744.

Basu U, Si K, Deng H, Maitra U. 2003. Phosphorylation of mammalian eukaryotic translation initiation factor 6 and its *Saccharomyces cerevisiae* homologue Tif6p: Evidence that phosphorylation of Tif6p regulates its nucleocytoplasmic distribution and is required for yeast cell growth. *Mol Cell Biol* **23**: 6187–6199.

Becam AM, Nasr F, Racki WJ, Zagulski M, Herbert CJ. 2001. Rial1p (Ynl163c), a protein similar to elongation factors 2, is involved in the biogenesis of the 60S subunit of the ribosome in *Saccharomyces cerevisiae*. *Mol Genet Genomics* **266**: 454–462.

Beckmann R, Spahn CM, Eswar N, Helmers J, Penczek PA, Sali A, Frank J, Blobel G. 2001. Architecture of the protein-conducting channel associated with the translating 80S ribosome. *Cell* **107**: 361–372.

Bellare P, Small EC, Huang X, Wohlschlegel JA, Staley JP, Sontheimer EJ. 2008. A role for ubiquitin in the spliceosome assembly pathway. *Nat Struct Mol Biol* **15**: 444–451.

Bernstein KA, Granneman S, Lee AV, Manickam S, Baserga SJ. 2006. Comprehensive mutational analysis of yeast DExD/H box RNA helicases involved in large ribosomal subunit biogenesis. *Mol Cell Biol* **26**: 1195–1208.

Bernstein KA, Bleichert F, Bean J, Cross F, Baserga SJ. 2007. Ribosome biogenesis is sensed at the start cell cycle checkpoint. *Mol Biol Cell* **18**: 953–964.

Bharucha N, Ma J, Dobry CJ, Lawson SK, Yang Z, Kumar A. 2008. Analysis of the yeast kinome reveals a network of regulated protein localization during filamentous growth. *Mol Biol Cell* **19**: 2708–2717.

Blake D, Luke B, Kanellis P, Jorgensen P, Goh T, Penfold S, Breitkreutz BJ, Durocher D, Peter M, Tyers M. 2006. The F-box protein Dia2 overcomes replication impedance to promote genome stability in *Saccharomyces cerevisiae*. *Genetics* **174**: 1709–1727.

Bleichert F, Baserga SJ. 2007. The long unwinding road of RNA helicases. *Mol Cell* **27**: 339–352.

Bleichert F, Granneman S, Osheim YN, Beyer AL, Baserga SJ. 2006. The PINc domain protein Utp24, a putative nuclease, is required for the early cleavage steps in 18S rRNA maturation. *Proc Natl Acad Sci* **103**: 9464–9469.

Bohnsack MT, Kos M, Tollervey D. 2008. Quantitative analysis of snoRNA association with pre-ribosomes and release of snR30 by Rok1 helicase. *EMBO Rep* **9**: 1230–1236.

Bond AT, Mangus DA, He F, Jacobson A. 2001. Absence of Dbp2p alters both nonsense-mediated mRNA decay and rRNA processing. *Mol Cell Biol* **21**: 7366–7379.

Bourne HR, Sanders DA, McCormick F. 1991. The GTPase superfamily: Conserved structure and molecular mechanism. *Nature* **349**: 117–127.

Buchhaupt M, Meyer B, Kotter P, Entian KD. 2006. Genetic evidence for 18S rRNA binding and an Rps19p assembly function of yeast nucleolar protein Nep1p. *Mol Genet Genomics* **276**: 273–284.

Burger F, Daugeron MC, Linder P. 2000. Dbp10p, a putative RNA helicase from *Saccharomyces cerevisiae*, is required for ribosome biogenesis. *Nucleic Acids Res* **28**: 2315–2323.

Cadwell C, Yoon HJ, Zebarjadian Y, Carbon J. 1997. The yeast nucleolar protein Cbf5p is involved in rRNA biosynthesis and interacts genetically with the RNA polymerase I transcription factor RRN3. *Mol Cell Biol* **17**: 6175–6183.

Cannone JJ, Subramanian S, Schnare MN, Collett JR, D’Souza LM, Du Y, Feng B, Lin N, Madabusi LV, Muller KM, et al. 2002. The Comparative RNA Web (CRW) site: An online database of comparative sequence and structure information for ribosomal, intron, and other RNAs. *BMC Bioinformatics* **3**: 2. doi: 10.1186/1471-2105-3-2.

Chandramouli P, Topf M, Menetret JF, Eswar N, Cannone JJ, Gutell RR, Sali A, Akey CW. 2008. Structure of the mammalian 80S ribosome at 8.7 Å resolution. *Structure* **16**: 535–548.


Charette M, Gray MW. 2000. Pseudouridine in RNA: What, where, how, and why. *IUBMB Life* **49**: 341–351.

Chen Y, Potratz JP, Tijerina P, Del Campo M, Lambowitz AM, Russell R. 2008. DEAD-box proteins can completely separate an RNA duplex using a single ATP. *Proc Natl Acad Sci* **105**: 20203–20208.

Chi A, Huttenhower C, Geer LY, Coon JJ, Syka JE, Bai DL, Shabanowitz J, Burke DJ, Troyanskaya OG, Hunt DF. 2007. Analysis of phosphorylation sites on proteins from *Saccharomyces cerevisiae* by electron transfer dissociation (ETD) mass spectrometry. *Proc Natl Acad Sci* **104**: 2193–2198.

Ciechanover A, Schwartz AL. 1994. The ubiquitin-mediated proteolytic pathway: Mechanisms of recognition of the proteolytic substrate and involvement in the degradation of native cellular proteins. *FASEB J* **8**: 182–191.

Cohen P. 2000. The regulation of protein function by multisite phosphorylation—a 25 year update. *Trends Biochem Sci* **25**: 596–601.

Colley A, Beggs JD, Tollervey D, Lafontaine DLJ. 2000. Dhr1p, a putative DEAH-Box RNA helicase, is associated with the box C+D snoRNP U3. *Mol Cell Biol* **20**: 7238–7246.

Collins SR, Kemmeren P, Zhao XC, Greenblatt JF, Spencer F, Holstege FC, Weissman JS, Krogan NJ. 2007. Toward a comprehensive atlas of the physical interactome of *Saccharomyces cerevisiae*. *Mol Cell Proteomics* **6**: 439–450.

Combs DJ, Nagel RJ, Ares MJr, Stevens SW. 2006. Prp43p is a DEAH-box spliceosome disassembly factor essential for ribosome biogenesis. *Mol Cell Biol* **26**: 523–534.

Connolly K, Culver G. 2009. Deconstructing ribosome construction. *Trends Biochem Sci* **34**: 256–263.

Connolly K, Rife JP, Culver G. 2008. Mechanistic insight into the ribosome biogenesis functions of the ancient protein KsgA. *Mol Microbiol* **70**: 1062–1075.

Cordin O, Banroques J, Tanner NK, Linder P. 2006. The DEAD-box protein family of RNA helicases. *Gene* **367**: 17–37.

Culver GM, Noller HF. 2000. In vitro reconstitution of 30S ribosomal subunits using complete set of recombinant proteins. *Methods Enzymol* **318**: 446–460.

Dammel CS, Noller HF. 1993. A cold-sensitive mutation in 16S rRNA provides evidence for helical switching in ribosome assembly. *Genes & Dev* **7**: 660–670.

Daugeron MC, Linder P. 1998. Dbp7p, a putative ATP-dependent RNA helicase from *Saccharomyces cerevisiae*, is required for 60S ribosomal subunit assembly. *RNA* **4**: 566–581.

Daugeron ML, Linder P. 2001. Characterization and mutational analysis of yeast Dbp8p, a putative RNA helicase involved in ribosome biogenesis. *Nucleic Acids Res* **29**: 1144–1155.

Daugeron MC, Kressler D, Linder P. 2001. Dbp9p, a putative ATP-dependent RNA helicase involved in 60S-ribosomal-subunit biogenesis, functionally interacts with Dbp6p. *RNA* **7**: 1317–1334.

Davierwala AP, Haynes J, Li Z, Brost RL, Robinson MD, Yu L, Mnaimneh S, Ding H, Zhu H, Chen Y, et al. 2005. The synthetic genetic interaction spectrum of essential genes. *Nat Genet* **37**: 1147–1152.

Decatur WA, Fournier MJ. 2003. RNA-guided nucleotide modification of ribosomal and other RNAs. *J Biol Chem* **278**: 695–698.

DeLaBarre B, Brunger AT. 2005. Nucleotide dependent motion and mechanism of action of p97/VCP. *J Mol Biol* **347**: 437–452.

de la Cruz J, Kressler D, Rojo M, Tollervey D, Linder P. 1998a. Spb4p, an essential putative RNA helicase, is required for a late step in the assembly of 60S ribosomal subunits in *Saccharomyces cerevisiae*. *RNA* **4**: 1268–1281.

de la Cruz J, Kressler D, Tollervey D, Linder P. 1998b. Dob1p (Mtr4p) is a putative ATP-dependent RNA helicase required for the 3′ end formation of 5.8S rRNA in *Saccharomyces cerevisiae*. *EMBO J* **17**: 1128–1140.

de la Cruz J, Lacombe T, Deloche O, Linder P, Kressler D. 2004. The putative RNA helicase Dbp6p functionally interacts with Rpl3p,

Nop8p and the novel trans-acting factor Rsa3p during biogenesis of 60S ribosomal subunits in *Saccharomyces cerevisiae*. *Genetics* **166**: 1687–1699.

Dez C, Tollervey D. 2004. Ribosome synthesis meets the cell cycle. *Curr Opin Microbiol* **7**: 631–637.

Dez C, Houseley J, Tollervey D. 2006. Surveillance of nuclear-restricted pre-ribosomes within a subnuclear region of *Saccharomyces cerevisiae*. *EMBO J* **25**: 1534–1546.

Dez C, Dlakic M, Tollervey D. 2007. Roles of the HEAT repeat proteins Utp10 and Utp20 in 40S ribosome maturation. *RNA* **13**: 1516–1527.

Dong J, Lai R, Nielsen K, Fekete CA, Qiu H, Hinnebusch AG. 2004. The essential ATP-binding cassette protein RLI1 functions in translation by promoting preinitiation complex assembly. *J Biol Chem* **279**: 42157–42168.

Dong J, Lai R, Jennings JL, Link AJ, Hinnebusch AG. 2005. The novel ATP-binding cassette protein ARB1 is a shuttling factor that stimulates 40S and 60S ribosome biogenesis. *Mol Cell Biol* **25**: 9859–9873.

Dragon F, Gallagher JE, Compagnone-Post PA, Mitchell BM, Porwancher KA, Wehner KA, Wormsley S, Settlage RE, Shabanowitz J, Osheim Y, et al. 2002. A large nucleolar U3 ribonucleoprotein required for 18S ribosomal RNA biogenesis. *Nature* **417**: 967–970.

Draptchinskaia N, Gustavsson P, Andersson B, Pettersson M, Willig TN, Dianzani I, Ball S, Tchernia G, Klar J, Matsson H, et al. 1999. The gene encoding ribosomal protein S19 is mutated in Diamond-Blackfan anaemia. *Nat Genet* **21**: 169–175.

Du YC, Stillman B. 2002. Yph1p, an ORC-interacting protein: Potential links between cell proliferation control, DNA replication, and ribosome biogenesis. *Cell* **109**: 835–848.

Emery B, de la Cruz J, Rocak S, Deloche O, Linder P. 2004. Has1p, a member of the DEAD-box family, is required for 40S ribosomal subunit biogenesis in *Saccharomyces cerevisiae*. *Mol Microbiol* **52**: 141–158.

Eppens NA, Rensen S, Granneman S, Raue HA, Venema J. 1999. The roles of Rrp5p in the synthesis of yeast 18S and 5.8S rRNA can be functionally and physically separated. *RNA* **5**: 779–793.

Erzberger JP, Berger JM. 2006. Evolutionary relationships and structural mechanisms of AAA+ proteins. *Annu Rev Biophys Biomol Struct* **35**: 93–114.

Fath S, Milkereit P, Podtelejnikov AV, Bischler N, Schultz P, Bier M, Mann M, Tschochner H. 2000. Association of yeast RNA polymerase I with a nucleolar substructure active in rRNA synthesis and processing. *J Cell Biol* **149**: 575–589.

Fayet-Lebaron E, Atzorn V, Henry Y, Kiss T. 2009. 18S rRNA processing requires base pairings of snR30H/ACA snoRNA to eukaryote-specific 18S sequences. *EMBO J* **28**: 1260–1270.

Ferreira-Cerca S, Poll G, Gleizes PE, Tschochner H, Milkereit P. 2005. Roles of eukaryotic ribosomal proteins in maturation and transport of pre-18S rRNA and ribosome function. *Mol Cell* **20**: 263–275.

Ficarro SB, McCleland ML, Stukenberg PT, Burke DJ, Ross MM, Shabanowitz J, Hunt DF, White FM. 2002. Phosphoproteome analysis by mass spectrometry and its application to *Saccharomyces cerevisiae*. *Nat Biotechnol* **20**: 301–305.

Finley D, Bartel B, Varshavsky A. 1989. The tails of ubiquitin precursors are ribosomal proteins whose fusion to ubiquitin facilitates ribosome biogenesis. *Nature* **338**: 394–401.

Fromont-Racine M, Senger B, Saveanu C, Fasiolo F. 2003. Ribosome assembly in eukaryotes. *Gene* **313**: 17–42.

Fujii K, Kitabatake M, Sakata T, Miyata A, Ohno M. 2009. A role for ubiquitin in the clearance of nonfunctional rRNAs. *Genes & Dev* **23**: 963–974.

Fujiyama-Nakamura S, Yoshikawa H, Homma K, Hayano T, Tsujimura-Takahashi T, Izumikawa K, Ishikawa H, Miyazawa N, Yanagida M, Miura Y, et al. 2009. Parvulin (Parl4), a peptidyl prolyl cis-trans isomerase, is a novel rRNA-processing factor evolved in the metazoan lineage. *Mol Cell Proteomics* **8**: 1552–1565.


Fuller-Pace FV, Nicol SM, Reid AD, Lane DP. 1993. DbpA: A DEAD box protein specifically activated by 23S rRNA. *EMBO J* **12**: 3619–3626.

Gadal O, Strauss D, Braspennning J, Hoepfner D, Petfalski E, Philippsen P, Tollervey D, Hurt E. 2001. A nuclear AAA-type ATPase (Rix7p) is required for biogenesis and nuclear export of 60S ribosomal subunits. *EMBO J* **20**: 3695–3704.

Galani K, Nissan TA, Petfalski E, Tollervey D, Hurt E. 2004. Real, a dynein-related nuclear AAA-ATPase, is involved in late rRNA processing and nuclear export of 60S subunits. *J Biol Chem* **279**: 55411–55418.

Gallagher JE, Dunbar DA, Granneman S, Mitchell BM, Osheim Y, Beyer AL, Baserga SJ. 2004. RNA polymerase I transcription and pre-rRNA processing are linked by specific SSU processome components. *Genes & Dev* **18**: 2506–2517.

Garcia I, Uhlenbeck OC. 2008. Differential RNA-dependent ATPase activities of four rRNA processing yeast DEAD-box proteins. *Biochemistry* **47**: 12562–12573.

Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, Storz G, Botstein D, Brown PO. 2000. Genomic expression programs in the response of yeast cells to environmental changes. *Mol Cell Biol* **11**: 4241–4257.

Gavin AC, Bosche M, Krause R, Grandi P, Marzioch M, Bauer A, Schultz J, Rick JM, Michon AM, Cruciat CM, et al. 2002. Functional organization of the yeast proteome by systematic analysis of protein complexes. *Nature* **415**: 141–147.

Gavin AC, Aloy P, Grandi P, Krause R, Boesche M, Marzioch M, Rau C, Jensen LJ, Bastuck S, Dumpelfeld B, et al. 2006. Proteome survey reveals modularity of the yeast cell machinery. *Nature* **440**: 631–636.

Geerlings TH, Faber AW, Bister MD, Vos JC, Raue HA. 2003. Rio2p, an evolutionarily conserved, low abundant protein kinase essential for processing of 20S Pre-rRNA in *Saccharomyces cerevisiae*. *J Biol Chem* **278**: 22537–22545.

Gelperin D, Horton L, Beckman J, Hensold J, Lemmon SK. 2001. Bms1p, a novel GTP-binding protein, and the related Tsr1p are required for distinct steps of 40S ribosome biogenesis in yeast. *RNA* **7**: 1268–1283.

Gill G. 2004. SUMO and ubiquitin in the nucleus: Different functions, similar mechanisms? *Genes & Dev* **18**: 2046–2059.

Glickman MH, Ciechanover A. 2002. The ubiquitin-proteasome proteolytic pathway: Destruction for the sake of construction. *Physiol Rev* **82**: 373–428.

Grandi P, Rybin V, Bassler J, Petfalski E, Strauss D, Marzioch M, Schafer T, Kuster B, Tschochner H, Tollervey D, et al. 2002. 90S pre-ribosomes include the 35S pre-rRNA, the U3 snoRNP, and 40S subunit processing factors but predominantly lack 60S synthesis factors. *Mol Cell* **10**: 105–115.

Granneman S, Baserga SJ. 2005. Cross talk in gene expression: Coupling and co-regulation of rDNA transcription, pre-ribosome assembly and pre-rRNA processing. *Curr Opin Cell Biol* **17**: 281–286.

Granneman S, Nandineni MR, Baserga SJ. 2005. The putative NTPase Fap7 mediates cytoplasmic 20S pre-rRNA processing through a direct interaction with Rps14. *Mol Cell Biol* **25**: 10352–10364.

Granneman S, Bernstein KA, Bleichert F, Baserga SJ. 2006a. Comprehensive mutational analysis of yeast DEXD/H box RNA helicases required for small ribosomal subunit synthesis. *Mol Cell Biol* **26**: 1183–1194.

Granneman S, Lin C, Champion EA, Nandineni MR, Zorca C, Baserga SJ. 2006b. The nuclear protein Esf2 interacts directly with the DExD/H box RNA helicase, Dbp8, to stimulate ATP hydrolysis. *Nucleic Acids Res* **34**: 3189–3199.

Green R, Noller HF. 1999. Reconstitution of functional 50S ribosomes from in vitro transcripts of *Bacillus stearothermophilus* 23S rRNA. *Biochemistry* **38**: 1772–1779.

Hage AE, Tollervey D. 2004. A surfeit of factors: Why is ribosome assembly so much more complicated in eukaryotes than bacteria? *RNA Biol* **1**: 10–15.

Hassett A, Blattler W, Knowles JR. 1982. Pyruvate kinase: Is the mechanism of phospho transfer associative or dissociative? *Biochemistry* **21**: 6335–6340.

Hedges J, West M, Johnson AW. 2005. Release of the export adapter, Nmd3p, from the 60S ribosomal subunit requires Rpl10p and the cytoplasmic GTPase Lsg1p. *EMBO J* **24**: 567–579.

Held WA, Ballou B, Mizushima S, Nomura M. 1974. Assembly mapping of 30S ribosomal proteins from *Escherichia coli*. Further studies. *J Biol Chem* **249**: 3103–3111.

Henras AK, Soudet J, Gerus M, Lebaron S, Caizergues-Ferrer M, Mougin A, Henry Y. 2008. The post-transcriptional steps of eukaryotic ribosome biogenesis. *Cell Mol Life Sci* **65**: 2334–2359.

Herold M, Nierhaus KH. 1987. Incorporation of six additional proteins to complete the assembly map of the 50S subunit from *Escherichia coli* ribosomes. *J Biol Chem* **262**: 8826–8833.

Higgins CF, Linton KJ. 2004. The ATP switch model for ABC transporters. *Nat Struct Mol Biol* **11**: 918–926.

Ho HY, Lee HH, Lai MZ. 1997. Overexpression of mitogen-activated protein kinase kinase reversed cAMP inhibition of NF-κB in T cells. *Eur J Immunol* **27**: 222–226.

Ho Y, Gruhler A, Heilbut A, Bader GD, Moore L, Adams SL, Millar A, Taylor P, Bennett K, Boutilier K, et al. 2002. Systematic identification of protein complexes in *Saccharomyces cerevisiae* by mass spectrometry. *Nature* **415**: 180–183.

Hoang T, Peng WT, Vanrobays E, Krogan N, Hiley S, Beyer AL, Osheim YN, Greenblatt J, Hughes TR, Lafontaine DL. 2005. Esf2p, a U3-associated factor required for small-subunit processome assembly and compaction. *Mol Cell Biol* **25**: 5523–5534.

Hoekstra MF, Liskay RM, Ou AC, DeMaggio AJ, Burbee DG, Heffron F. 1991. HRR25, a putative protein kinase from budding yeast: Association with repair of damaged DNA. *Science* **253**: 1031–1034.

Honma Y, Kitamura A, Shioda R, Maruyama H, Ozaki K, Oda Y, Mini T, Jeno P, Maki Y, Yonezawa K, et al. 2006. TOR regulates late steps of ribosome maturation in the nucleoplasm via Nog1 in response to nutrients. *EMBO J* **25**: 3832–3842.

Hopfield JJ. 1978. Origin of the genetic code: A testable hypothesis based on tRNA structure, sequence, and kinetic proofreading. *Proc Natl Acad Sci* **75**: 4334–4338.

Hopfner KP, Tainer JA. 2003. Rad50/SMC proteins and ABC transporters: Unifying concepts from high-resolution structures. *Curr Opin Struct Biol* **13**: 249–255.

Hughes JMX. 1996. Functional base-pairing interaction between highly conserved elements of U3 small nucleolar RNA and the small ribosomal subunit RNA. *J Mol Biol* **259**: 645–654.

Iost I, Dreyfus M. 2006. DEAD-box RNA helicases in *Escherichia coli*. *Nucleic Acids Res* **34**: 4189–4197.

Jakovljevic J, de Mayolo PA, Miles TD, Nguyen TM, Leger-Silvestre I, Gas N, Woolford JL Jr. 2004. The carboxy-terminal extension of yeast ribosomal protein S14 is necessary for maturation of 43S pre-ribosomes. *Mol Cell* **14**: 331–342.

Jankowsky E, Fairman ME. 2007. RNA helicases—one fold for many functions. *Curr Opin Struct Biol* **17**: 316–324.

Jensen BC, Wang Q, Kifer CT, Parsons M. 2003. The Nog1 GTP-binding protein is required for biogenesis of the 60S ribosomal subunit. *J Biol Chem* **278**: 32204–32211.

Jiang M, Sullivan SM, Walker AK, Strahler JR, Andrews PC, Maddock JR. 2007. Identification of novel *Escherichia coli* ribosome-associated proteins using isobaric tags and multidimensional protein identification techniques. *J Bacteriol* **189**: 3434–3444.

Jin R, Dobry CJ, McCown PJ, Kumar A. 2008. Large-scale analysis of yeast filamentous growth by systematic gene disruption and overexpression. *Mol Biol Cell* **19**: 284–296.

Ju Q, Warner JR. 1994. Ribosome synthesis during the growth cycle of *Saccharomyces cerevisiae*. *Yeast* **10**: 151–157.

Juhnke H, Charizanis C, Latifi F, Krems B, Entian KD. 2000. The essential protein Fap7 is involved in the oxidative stress response of *Saccharomyces cerevisiae*. *Mol Microbiol* **35**: 936–948.


Jwa M, Kim JH, Chan CS. 2008. Regulation of Slu15/INCENP, kinetochore, and Cdc14 phosphatase functions by the ribosome biogenesis protein Utp7. *J Cell Biol* **182**: 1099–1111.

Kafadar KA, Zhu H, Snyder M, Cyert MS. 2003. Negative regulation of calcineurin signaling by Hrr25p, a yeast homolog of casein kinase I. *Genes & Dev* **17**: 2698–2708.

Kallstrom G, Hedges J, Johnson A. 2003. The putative GTPases Nog1p and Lsg1p are required for 60S ribosomal subunit biogenesis and are localized to the nucleus and cytoplasm, respectively. *Mol Cell Biol* **23**: 4344–4355.

Karbstein K. 2007. The role of GTPases in ribosome assembly. *Biopolymers* **87**: 1–11.

Karbstein K. 2009. Ribosome assembly, eukaryotic. In *Wiley encyclopedia of chemical biology* (ed. T Begley), Vol. 4, pp. 222–230. Wiley, Hoboken, NJ.

Karbstein K, Jonas S, Doudna JA. 2005. An essential GTPase promotes assembly of preribosomal RNA processing complexes. *Mol Cell* **20**: 633–643.

Kerscher O, Felberbaum R, Hochstrasser M. 2006. Modification of proteins by ubiquitin and ubiquitin-like proteins. *Annu Rev Cell Dev Biol* **22**: 159–180.

Killian A, Le Meur N, Sesboue R, Bourguignon J, Bougeard G, Gautherot J, Bastard C, Frebourg T, Flaman JM. 2004. Inactivation of the RRB1-Pescadillo pathway involved in ribosome biogenesis induces chromosomal instability. *Oncogene* **23**: 8597–8602.

King TH, Liu B, McCully RR, Fournier MJ. 2003. Ribosome structure and activity are altered in cells lacking snoRNPs that form pseudouridines in the peptidyl transferase center. *Mol Cell* **11**: 425–435.

Kispal G, Sipos K, Lange H, Fekete Z, Bedekovics T, Janaky T, Bassler J, Aguilar Netz DJ, Balk J, Rotte C, et al. 2005. Biogenesis of cytosolic ribosomes requires the essential iron-sulphur protein Rli1p and mitochondria. *EMBO J* **24**: 589–598.

Kos M, Tollervey D. 2005. The putative RNA helicase Dbp4p is required for release of U14 snoRNA from pre-ribosomes in *Saccharomyces cerevisiae*. *Mol Cell* **20**: 53–64.

Kressler D, delaCruz J, Rojo M, Linder P. 1997. Fal1p is an essential DEAD-box protein involved in 40S-ribosomal-subunit biogenesis in *Saccharomyces cerevisiae*. *Mol Cell Biol* **17**: 7283–7294.

Kressler D, DelaCruz J, Rojo M, Linder P. 1998. Dbp6p is an essential putative ATP-dependent RNA helicase required for SOS-ribosomal-subunit assembly in *Saccharomyces cerevisiae*. *Mol Cell Biol* **18**: 1855–1865.

Kressler D, Roser D, Pertschy B, Hurt E. 2008. The AAA ATPase Rix7 powers progression of ribosome biogenesis by stripping Nsa1 from pre-60S particles. *J Cell Biol* **181**: 935–944.

Krogan NJ, Peng WT, Cagney G, Robinson MD, Haw R, Zhong G, Guo X, Zhang X, Canadien V, Richards DP, et al. 2004. High-definition macromolecular composition of yeast RNA-processing complexes. *Mol Cell* **13**: 225–239.

Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, Ignatchenko A, Li J, Pu S, Datta N, Tikuisis AP, et al. 2006. Global landscape of protein complexes in the yeast *Saccharomyces cerevisiae*. *Nature* **440**: 637–643.

Lacombe T, Garcia-Gomez JJ, de la Cruz J, Roser D, Hurt E, Linder P, Kressler D. 2009. Linear ubiquitin fusion to Rps31 and its subsequent cleavage are required for the efficient production and functional integrity of 40S ribosomal subunits. *Mol Microbiol* **72**: 69–84.

Lafontaine D, Vandenaute J, Tollervey D. 1995. The 18S ribosomal RNA dimethylase Dim1p is required for pre-ribosomal-RNA processing in yeast. *Genes & Dev* **9**: 2470–2481.

Lafontaine DLJ, Preiss T, Tollervey D. 1998. Yeast 18S rRNA dimethylase Dim1p: A quality control mechanism in ribosome synthesis? *Mol Cell Biol* **18**: 2360–2370.

LaRonde-LeBlanc N, Wlodawer A. 2004. Crystal structure of *A. fulgidus* Rio2 defines a new family of serine protein kinases. *Structure* **12**: 1585–1594.

LaRonde-LeBlanc N, Wlodawer A. 2005. A family portrait of the Rio kinases. *J Biol Chem* **280**: 37297–37300.

LaRonde-LeBlanc N, Guszczynski T, Copeland T, Wlodawer A. 2005a. Autophosphorylation of *Archaeoglobus fulgidus* Rio2 and crystal structures of its nucleotide-metal ion complexes. *FEBS J* **272**: 2800–2810.

LaRonde-LeBlanc N, Guszczynski T, Copeland T, Wlodawer A. 2005b. Structure and activity of the atypical serine kinase Rio1. *FEBS J* **272**: 3698–3713.

Lebaron S, Froment C, Fromont-Racine M, Rain JC, Monsarrat B, Caizergues-Ferrer M, Henry Y. 2005. The splicing ATPase Prp43p is a component of multiple preribosomal particles. *Mol Cell Biol* **25**: 9269–9282.

Leeds NB, Small EC, Hiley SL, Hughes TR, Staley JP. 2006. The splicing factor Prp43p, a DEAH box ATPase, functions in ribosome biogenesis. *Mol Cell Biol* **26**: 513–522.

Leipe DD, Wolf YI, Koonin EV, Aravind L. 2002. Classification and evolution of P-loop GTPases and related ATPases. *J Mol Biol* **317**: 41–72.

Li HD, Zagorski J, Fournier MJ. 1990. Depletion of U14 small nuclear RNA (snR128) disrupts production of 18S rRNA in *Saccharomyces cerevisiae*. *Mol Cell Biol* **10**: 1145–1152.

Liang XH, Fournier MJ. 2006. The helicase Has1p is required for snoRNA release from pre-rRNA. *Mol Cell Biol* **26**: 7437–7450.

Liang WQ, Clark JA, Fournier MJ. 1997. The rRNA-processing function of the yeast U14 small nuclear RNA can be rescued by a conserved RNA helicase-like protein. *Mol Cell Biol* **17**: 4124–4132.

Liang XH, Liu Q, Fournier MJ. 2007. rRNA modifications in an intersubunit bridge of the ribosome strongly affect both ribosome biogenesis and activity. *Mol Cell* **28**: 965–977.

Liang XH, Liu Q, Fournier MJ. 2009. Loss of rRNA modifications in the decoding center of the ribosome impairs translation and strongly delays pre-rRNA processing. *RNA* **15**: 1716–1728.

Liu F, Putnam A, Jankowsky E. 2008. ATP hydrolysis is required for DEAD-box protein recycling but not for duplex unwinding. *Proc Natl Acad Sci* **105**: 20209–20214.

Loar JW, Seiser RM, Sundberg AE, Sagerson HJ, Ilias N, Zobel-Thropp P, Craig EA, Lycan DE. 2004. Genetic and biochemical interactions among Yar1, Ltv1, and Rps3 define novel links between environmental stress and ribosome biogenesis in *Saccharomyces cerevisiae*. *Genetics* **168**: 1877–1889.

Maki JA, Schnobrich DJ, Culver GM. 2002. The DnaK chaperone system facilitates 30S ribosomal subunit assembly. *Mol Cell* **10**: 129–138.

Manning G, Plowman GD, Hunter T, Sudarsanam S. 2002. Evolution of protein kinase signaling from yeast to man. *Trends Biochem Sci* **27**: 514–520.

Mears JA, Canone JJ, Stagg SM, Gutell RR, Agrawal RK, Harvey SC. 2002. Modeling a minimal ribosome based on comparative sequence analysis. *J Mol Biol* **321**: 215–234.

Menne TF, Goyenechea B, Sanchez-Puig N, Wong CC, Tonkin LM, Ancliff PJ, Brost RL, Costanzo M, Boone C, Warren AJ. 2007. The Shwachman–Bodian–Diamond syndrome protein mediates translational activation of ribosomes in yeast. *Nat Genet* **39**: 486–495.

Mizushima S, Nomura M. 1970. Assembly mapping of 30S ribosomal proteins from *E. coli*. *Nature* **226**: 1214–1218.

Morrissey JP, Tollervey D. 1993. Yeast SnR30 is a small nucleolar RNA required for 18S ribosomal-RNA synthesis. *Mol Cell Biol* **13**: 2469–2477.

Nierhaus KH, Dohme F. 1974. Total reconstitution of functionally active 50S ribosomal subunits from *Escherichia coli*. *Proc Natl Acad Sci* **71**: 4713–4717.

Nissan TA, Galani K, Maco B, Tollervey D, Aebi U, Hurt E. 2004. A pre-ribosome with a tadpole-like structure functions in ATP-dependent maturation of 60S subunits. *Mol Cell* **15**: 295–301.

O’Day CL, Chavanikamannil F, Abelson J. 1996. 18S rRNA processing requires the RNA helicase-like protein Rrp3. *Nucleic Acids Res* **24**: 3201–3207.


Oeffinger M, Wei KE, Rogers R, DeGrasse JA, Chait BT, Aitchison JD, Rout MP. 2007. Comprehensive analysis of diverse ribonucleoprotein complexes. *Nat Methods* **4**: 951–956.

Osheim YN, French SL, Keck KM, Champion EA, Spasov K, Dragon F, Baserga SJ, Beyer AL. 2004. Pre-18S ribosomal RNA is structurally compacted into the SSU processome prior to being cleaved from nascent transcripts in *Saccharomyces cerevisiae*. *Mol Cell* **16**: 943–954.

Pan X, Ye P, Yuan DS, Wang X, Bader JS, Boeke JD. 2006. A DNA integrity network in the yeast *Saccharomyces cerevisiae*. *Cell* **124**: 1069–1081.

Panse VG, Hardeland U, Werner T, Kuster B, Hurt E. 2004. A proteome-wide approach identifies sumoylated substrate proteins in yeast. *J Biol Chem* **279**: 41346–41351.

Panse VG, Kressler D, Pauli A, Petfalski E, Gnadig M, Tollervey D, Hurt E. 2006. Formation and nuclear export of preribosomes are functionally linked to the small-ubiquitin-related modifier pathway. *Traffic* **7**: 1311–1321.

Peng J, Schwartz D, Elias JE, Thoreen CC, Cheng D, Marsischky G, Roelofs J, Finley D, Gygi SP. 2003. A proteomics approach to understanding protein ubiquitination. *Nat Biotechnol* **21**: 921–926.

Pertschy B, Saveanu C, Zisser G, Lebreton A, Tengg M, Jacquier A, Liebmingere Nobis B, Kappel L, van der Klei I, et al. 2007. Cytoplasmic recycling of 60S preribosomal factors depends on the AAA protein Drg1. *Mol Cell Biol* **27**: 6581–6592.

Petronczki M, Matos J, Mori S, Gregan J, Bogdanova A, Schwickart M, Mechtlere K, Shirahige K, Zachariae W, Nasmyth K. 2006. Monopolar attachment of sister kinetochores at meiosis I requires casein kinase 1. *Cell* **126**: 1049–1064.

Piekna-Przybylska D, Przybylski P, Baudin-Baillieu A, Rousset JP, Fournier MJ. 2008. Ribosome performance is enhanced by a rich cluster of pseudouridines in the A-site finger region of the large subunit. *J Biol Chem* **283**: 26026–26036.

Pleiss JA, Whitworth GB, Bergkessel M, Guthrie C. 2007. Rapid, transcript-specific changes in splicing in response to environmental stress. *Mol Cell* **27**: 928–937.

Powers T, Walter P. 1999. Regulation of ribosome biogenesis by the rapamycin-sensitive TOR-signaling pathway in *Saccharomyces cerevisiae*. *Mol Biol Cell* **10**: 987–1000.

Ptacek J, Devgan G, Michaud G, Zhu H, Zhu X, Fasolo J, Guo H, Jona G, Breitkreutz A, Sopko R, et al. 2005. Global analysis of protein phosphorylation in yeast. *Nature* **438**: 679–684.

Pye VE, Dreveny I, Briggs LC, Sands C, Beuron F, Zhang X, Freemont PS. 2006. Going through the motions: The ATPase cycle of p97. *J Struct Biol* **156**: 12–28.

Raghunathan PL, Guthrie C. 1998. RNA unwinding in U4/U6 snRNPs requires ATP hydrolysis and the DEIH-box splicing factor Brr2. *Curr Biol* **8**: 847–855.

Ramaswamy P, Woodson SA. 2009. S16 throws a conformational switch during assembly of 30S 5′ domain. *Nat Struct Mol Biol* **16**: 438–445.

Ray P, Basu U, Ray A, Majumdar R, Deng H, Maitra U. 2008. The *Saccharomyces cerevisiae* 60S ribosome biogenesis factor Tif6p is regulated by Hrr25p-mediated phosphorylation. *J Biol Chem* **283**: 9681–9691.

Ripmaster TL, Vaughn GP, Woolford JL Jr. 1993. Drs1 to Drs7, novel genes required for ribosome assembly and function in *Saccharomyces cerevisiae*. *Mol Cell Biol* **13**: 7901–7912.

Rohl R, Nierhaus KH. 1982. Assembly map of the large subunit (50S) of *Escherichia coli* ribosomes. *Proc Natl Acad Sci* **79**: 729–733.

Rudra D, Mallick J, Zhao Y, Warner JR. 2007. Potential interface between ribosomal protein production and pre-rRNA processing. *Mol Cell Biol* **27**: 4815–4824.

Sachs AB, Davis RW. 1990. Translation initiation and ribosomal biogenesis: Involvement of a putative rRNA helicase and Rpl46. *Science* **247**: 1077–1079.

Saveanu C, Bienvenu D, Namane A, Gleizes PE, Gas N, Jacquier A, Fromont-Racine M. 2001. Nog2p, a putative GTPase associated
with pre-60S subunits and required for late 60S maturation steps. *EMBO J* **20**: 6475–6484.

Saveanu C, Namane A, Gleizes PE, Lebreton A, Rousselle JC, Noaillac-Depeyre J, Gas N, Jacquier A, Fromont-Racine M. 2003. Sequential protein association with nascent 60S ribosomal particles. *Mol Cell Biol* **23**: 4449–4460.

Schafer T, Strauss D, Petfalski E, Tollervey D, Hurt E. 2003. The path from nuclear 90S to cytoplasmic 40S pre-ribosomes. *EMBO J* **22**: 1370–1380.

Schafer T, Maco B, Petfalski E, Tollervey D, Bottcher B, Aebi U, Hurt E. 2006. Hrr25-dependent phosphorylation state regulates organization of the pre-40S subunit. *Nature* **441**: 651–655.

Schmid FX. 1995. Protein folding. Prolyl isomerases join the fold. *Curr Biol* **5**: 993–994.

Schmid FX, Baldwin RL. 1978. Acid catalysis of the formation of the slow-folding species of RNase A: Evidence that the reaction is proline isomerization. *Proc Natl Acad Sci* **75**: 4764–4768.

Schnare MN, Damberger SH, Gray MW, Gutell RR. 1996. Comprehensive comparison of structural characteristics in eukaryotic cytoplasmic large subunit (23 S-like) ribosomal RNA. *J Mol Biol* **256**: 701–719.

Schwartz DC, Hochstrasser M. 2003. A superfamily of protein tags: Ubiquitin, SUMO and related modifiers. *Trends Biochem Sci* **28**: 321–328.

Segerstolpe A, Lundkvist P, Osheim YN, Beyer AL, Wieslander L. 2008. Mrd1p binds to pre-rRNA early during transcription independent of U3 snoRNA and is required for compaction of the pre-rRNA into small subunit processomes. *Nucleic Acids Res* **36**: 4364–4380.

Senger B, Lafontaine DL, Graindorge JS, Gadal O, Camasses A, Sanni A, Garnier JM, Breitenbach M, Hurt E, Fasiolo F. 2001. The nucle(ol)ar Tif6p and Efl1p are required for a late cytoplasmic step of ribosome synthesis. *Mol Cell* **8**: 1363–1373.

Shammas C, Menne TF, Hilcenko C, Michell SR, Goyenechea B, Boocock GR, Durie PR, Rommens JM, Warren AJ. 2005. Structural and mutational analysis of the SBDS protein family. Insight into the leukemia-associated Shwachman–Diamond Syndrome. *J Biol Chem* **280**: 19221–19229.

Shibuya T, Tange TO, Stroupe ME, Moore MJ. 2006. Mutational analysis of human eIF4AIII identifies regions necessary for exon junction complex formation and nonsense-mediated mRNA decay. *RNA* **12**: 360–374.

Shirai C, Mizuta K. 2008. SUMO mediates interaction of Ebp2p, the yeast homolog of Epstein-Barr virus nuclear antigen 1-binding protein 2, with a RING finger protein Ris1p. *Biosci Biotechnol Biochem* **72**: 1881–1886.

Si K, Maitra U. 1999. The *Saccharomyces cerevisiae* homologue of mammalian translation initiation factor 6 does not function as a translation initiation factor. *Mol Cell Biol* **19**: 1416–1426.

Small EC, Leggett SR, Winans AA, Staley JP. 2006. The EF-G-like GTPase Snu114p regulates spliceosome dynamics mediated by Brr2p, a DExD/H box ATPase. *Mol Cell* **23**: 389–399.

Song Y, Kim S, Kim J. 1995. Rok1, a high-copy-number plasmid suppressor of kem1, encodes a putative ATP-dependent RNA helicase in *Saccharomyces cerevisiae*. *Gene* **166**: 151–154.

Spahn CM, Beckmann R, Eswar N, Penczek PA, Sali A, Blobel G, Frank J. 2001. Structure of the 80S ribosome from *Saccharomyces cerevisiae*—tRNA-ribosome and subunit–subunit interactions. *Cell* **107**: 373–386.

Stavreva DA, Kawasaki M, Dundr M, Koberna K, Muller WG, Tsujimura-Takahashi T, Komatsu W, Hayano T, Isobe T, Raska I, et al. 2006. Potential roles for ubiquitin and the proteasome during ribosome biogenesis. *Mol Cell Biol* **26**: 5131–5145.

Steiner-Mosonyi M, Leslie DM, Dehghani H, Aitchison JD, Mangroo D. 2003. Utp8p is an essential intranuclear component of the nuclear tRNA export machinery of *Saccharomyces cerevisiae*. *J Biol Chem* **278**: 32236–32245.

Strittmatter AW, Fischer C, Kleinschmidt M, Braus GH. 2006. Flo11 mediated filamentous growth of the yeast *Saccharomyces cerevisiae*

depends on the expression of the ribosomal Rps26 genes. *Mol Genet Genomics* **276**: 113–125.

Strub BR, Eswara MB, Pierce JB, Mangroo D. 2007. Utp8p is a nuclear tRNA-binding protein that forms a complex with components of the nuclear tRNA export machinery in *Saccharomyces cerevisiae*. *Mol Biol Cell* **18**: 3845–3859.

Sydorskyy Y, Dilworth DJ, Halloran B, Yi EC, Makhnevych T, Wozniak RW, Aitchison JD. 2005. Nop53p is a novel nuclear 60S ribosomal subunit biogenesis protein. *Biochem J* **388**: 819–826.

Tanaka N, Aronova A, Schwer B. 2007. Ntr1 activates the Prp43 helicase to trigger release of lariat-intron from the spliceosome. *Genes & Dev* **21**: 2312–2325.

Tanner NK, Linder P. 2001. DExD/H box RNA helicases: From generic motors to specific dissociation functions. *Mol Cell* **8**: 251–262.

Tarassov K, Messier V, Landry CR, Radinovic S, Serna Molina MM, Shames I, Malitskaya Y, Vogel J, Bussey H, Michnick SW. 2008. An in vivo map of the yeast protein interactome. *Science* **320**: 1465–1470.

Tollervey D. 1987. A yeast small nuclear RNA is required for normal processing of pre-ribosomal RNA. *EMBO J* **6**: 4169–4175.

Tollervey D, Lehtonen H, Carmofonseca M, Hurt EC. 1991. The small nuclear RNP protein Nop1 (Fibrillarin) is required for pre-ribosomal-RNA processing in yeast. *EMBO J* **10**: 573–583.

Tollervey D, Lehtonen H, Jansen R, Kern H, Hurt EC. 1993. Temperature-sensitive mutations demonstrate roles for yeast fibrillarin in pre-ribosomal-RNA processing, pre-ribosomal-RNA methylation and ribosome assembly. *Cell* **72**: 443–457.

Torchet C, Jacq C, Hermann-Le Denmat S. 1998. Two mutant forms of the S1/TPR-containing protein Rrp5p affect the 18S rRNA synthesis in *Saccharomyces cerevisiae*. *RNA* **4**: 1636–1652.

Traub P, Nomura M. 1968. Structure and function of *E. coli* ribosomes. V. Reconstitution of functionally active 30S ribosomal particles from RNA and proteins. *Proc Natl Acad Sci* **59**: 777–784.

Ulrich HD. 2008. The fast-growing business of SUMO chains. *Mol Cell* **32**: 301–305.

Valasek L, Hasek J, Nielsen KH, Hinnebusch AG. 2001. Dual function of eIF3j/Hcr1p in processing 20S pre-rRNA and translation initiation. *J Biol Chem* **276**: 43351–43360.

Valerius O, Kleinschmidt M, Rachfall N, Schulze F, Lopez Marin S, Hoppert M, Streckfuss-Bomeke K, Fischer C, Braus GH. 2007. The *Saccharomyces* homolog of mammalian RACK1, Cpc2/Ascl1p, is required for Flo11-dependent adhesive growth and dimorphism. *Mol Cell Proteomics* **6**: 1968–1979.

Vanrobays E, Gelugne JP, Gleizes PE, Caizergues-Ferrer M. 2003. Late cytoplasmic maturation of the small ribosomal subunit requires Rio proteins in *Saccharomyces cerevisiae*. *Mol Cell Biol* **23**: 2083–2095.

Vanrobays E, Leplus A, Osheim YN, Beyer AL, Wacheul L, Lafontaine DL. 2008. TOR regulates the subcellular distribution of Dim2, a KH domain protein required for cotranscriptional ribosome assembly and pre-40S ribosome export. *RNA* **14**: 2061–2073.

Venema J, Tollervey D. 1996. RRP5 is required for formation of both 18S and 5.85 rRNA in yeast. *EMBO J* **15**: 5701–5714.

Venema J, Tollervey D. 1999. Ribosome synthesis in *Saccharomyces cerevisiae*. *Annu Rev Genet* **33**: 261–311.

Venema J, BousquetAntonelli C, Gelugne JP, CaizerguesFerrer M, Tollervey D. 1997. Rok1p is a putative RNA helicase required for rRNA processing. *Mol Cell Biol* **17**: 3398–3407.

Wang Y, Liu CL, Storey JD, Tibshirani RJ, Herschlag D, Brown PO. 2002. Precision and functional specificity in mRNA decay. *Proc Natl Acad Sci* **99**: 5860–5865.

Warner JR. 1999. The economics of ribosome biosynthesis in yeast. *Trends Biochem Sci* **24**: 437–440.

Weaver PL, Sun C, Chang TH. 1997. Dbp3p, a putative RNA helicase in *Saccharomyces cerevisiae*, is required for efficient pre-rRNA processing predominantly at site A3. *Mol Cell Biol* **17**: 1354–1365.
Wegierski T, Billy E, Nasr F, Filipowicz W. 2001. Bms1p, a G-domain- containing protein, associates with Rcl1p and is required for 18S rRNA biogenesis in yeast. RNA 7: 1254–1267.

Wery M, Ruidant S, Schillewaert S, Lepore N, Lafontaine DL. 2009. The nuclear poly(A) polymerase and exosome cofactor Trf5 is recruited cotranscriptionally to nucleolar surveillance. RNA 15: 406–419.

Wilmes GM, Bergkessel M, Bandyopadhyay S, Shales M, Braberg H, Cagney G, Collins SR, Whitworth GB, Kress TL, Weissman JS, et al. 2008. A genetic interaction map of RNA-processing factors reveals links between Sem1/Dss1-containing complexes and mRNA export and splicing. Mol Cell 32: 735–746.

Winker S, Woese CR. 1991. A definition of the domains Archaea, Bacteria and Eucarya in terms of small subunit ribosomal RNA characteristics. Syst Appl Microbiol 14: 305–310.

Woese CR, Magrum LJ, Gupta R, Siegel RB, Stahl DA, Kop J, Crawford N, Brosius J, Gutell R, Hogan JJ, et al. 1980. Secondary

structure model for bacterial 16S ribosomal RNA: Phylogenetic, enzymatic, and chemical evidence. Nucleic Acids Res 8: 2275–2293.

Xu Z, O'Farrell HC, Rife JP, Culver GM. 2008. A conserved rRNA methyltransferase regulates ribosome biogenesis. Nat Struct Mol Biol 15: 534–536.

Yang Q, Del Campo M, Lambowitz AM, Jankowsky E. 2007. DEAD-box proteins unwind duplexes by local strand separation. Mol Cell 28: 253–263.

Yarunin A, Panse VG, Petfalski E, Dez C, Tollervey D, Hurt EC. 2005. Functional link between ribosome formation and biogenesis of iron-sulfur proteins. EMBO J 24: 580–588.

Zagulski M, Kressler D, Becam AM, Rytka J, Herbert CJ. 2003. Mak5p, which is required for the maintenance of the M1 dsRNA virus, is encoded by the yeast ORF YBR142w and is involved in the biogenesis of the 60S subunit of the ribosome. Mol Genet Genomics 270: 216–224.

Downloaded from majournal.cshlp.org on May 10, 2015 - Published by Cold Spring Harbor Laboratory Press

![RNA Logo](https://example.com/rna-logo.png)

**Powering through ribosome assembly**

Bethany S. Strunk and Katrin Karbstein

RNA 2009 15: 2083-2104 originally published online October 22, 2009  
Access the most recent version at doi:[10.1261/rna.1792109](http://10.1261/rna.1792109)

---

**References**  
This article cites 226 articles, 109 of which can be accessed free at:  
[http://rnajournal.cshlp.org/content/15/12/2083.full.html#ref-list-1](http://rnajournal.cshlp.org/content/15/12/2083.full.html#ref-list-1)

**Email Alerting Service**  
Receive free email alerts when new articles cite this article - sign up in the box at the top right corner of the article or [click here](http://click.here).

---

**Unique RNA sequencing in serum and plasma**  
EXIQON

---

To subscribe to RNA go to:  
[http://rnajournal.cshlp.org/subscriptions](http://rnajournal.cshlp.org/subscriptions)

---

Copyright © 2009 RNA Society
