
Biochimica et Biophysica Acta 1803 (2010) 673–683

Contents lists available at ScienceDirect

Biochimica et Biophysica Acta

journal homepage: www.elsevier.com/locate/bbamcr

Review

# Driving ribosome assembly

Dieter Kressler ${ }^{a, b}$, Ed Hurt ${ }^{a, *}$, Jochen Baβler ${ }^{a, *}$

${ }^{a}$ Biochemie-Zentrum der Universität Heidelberg, Im Neuenheimer Feld 328, 69120 Heidelberg, Germany  
${ }^{b}$ University of Fribourg, Department of Medicine, Unit of Biochemistry, Chemin du Musée 5, 1700 Fribourg, Switzerland

---

## Article Info

**Article history:**  
Received 8 September 2009  
Received in revised form 13 October 2009  
Accepted 26 October 2009  
Available online 30 October 2009  

**Keywords:**  
Ribosome assembly  
Nuclear export of ribosomes  
Diamond-Blackfan anemia  
Shwachman-Diamond syndrome  
Dyskeratosis congenita  
Cartilage-hair hypoplasia  
Bowen-Conradi syndrome  
Cancer  
Quality control of ribosomes

---

## Abstract

Ribosome biogenesis is a fundamental process that provides cells with the molecular factories for cellular protein production. Accordingly, its misregulation lies at the heart of several hereditary diseases (e.g., Diamond-Blackfan anemia). The process of ribosome assembly comprises the processing and folding of the pre-rRNA and its concomitant assembly with the ribosomal proteins. Eukaryotic ribosome biogenesis relies on a large number (>200) of non-ribosomal factors, which confer directionality and accuracy to this process. Many of these non-ribosomal factors fall into different families of energy-consuming enzymes, notably including ATP-dependent RNA helicases, AAA-ATPases, GTPases, and kinases. Ribosome biogenesis is highly conserved within eukaryotic organisms; however, due to the combination of powerful genetic and biochemical methods, it is best studied in the yeast Saccharomyces cerevisiae. This review summarizes our current knowledge on eukaryotic ribosome assembly, with particular focus on the molecular role of the involved energy-consuming enzymes.

© 2009 Elsevier B.V. All rights reserved.

---

## 1. Introduction

The ribosome is a complex molecular machine that is composed of a small 40S and large 60S subunit. Despite their conserved molecular function, eukaryotic and prokaryotic ribosomal subunits differ significantly in size and complexity (Saccharomyces cerevisiae: 40S [18S rRNA, 33 RPs]; 60S [25S, 5.8S, 5S rRNA, 46 RPs]—Escherichia coli: 30S [16S rRNA, 21 RPs]; 50S [23S, 5S rRNA, 34 RPs]). These differences may reflect an additional regulation of translation and is also the foundation of several antibiotics that block specifically the function of the prokaryotic subunits [1]. Despite our detailed knowledge of the structure and function of ribosomes (reviewed in references [2,3]), the molecular mechanisms driving ribosome assembly remain largely elusive. Ribosome biogenesis faces the challenge to coordinate the processing and modification of ribosomal RNA (rRNA) with its correct structural assembly with ribosomal proteins (RP). Furthermore, this process has to be regulated according to the cellular environment ([4], see below), hence ribosome biogenesis is tightly coupled to growth rate: actively dividing cells, including cancer cells, depend on active ribosome biogenesis, whereas arrested or starving cells halt the production of new ribosomal subunits.

Due to its easy experimental accessibility by genetic, biochemical, and cell biological methods, S. cerevisiae represents a suitable eukaryotic model organism to study ribosome assembly and the function of non-ribosomal factors. Over the past 20 years, it has been shown that a large number of non-ribosomal factors (>200) and snoRNAs (75) are involved in ribosome assembly [5,6]. Moreover, the tandem affinity purification method (TAP) enabled the isolation and characterization of several assembly intermediates, which correspond to snap shots of pre-ribosomal particles along their maturation path [5,7–10]. The current challenge is to identify direct interaction partners of individual proteins and obtain structural information of single proteins and pre-ribosomal particles.

## 2. Dynamics in ribosome assembly

### 2.1. Birth of pre-ribosomal particles

The biogenesis of both subunits starts with the transcription of the common precursor, the 35S pre-rRNA, by RNA polymerase I (Fig. 3). As would be expected for a tightly regulated process, the rate of ribosome synthesis is under strict transcriptional control (reviewed in references [4,11–14]). The nascent rRNA is modified by about 75 different small nucleolar ribonucleoprotein particles (snoRNPs), which mediate 2′-O-ribose methylation of nucleotides and the formation of pseudouridines. These snoRNP complexes are targeted to their substrate via base pairing between rRNA and snoRNA, whereas associated proteins catalyze the modification reaction (reviewed in references [5,6,15,16]). A subset of small subunit ribosomal proteins (Rps) and non-ribosomal factors assemble co-transcriptionally with the pre-RNA to form a terminal knob, the first pre-ribosomal particle on the path to the small ribosomal subunit (see

* Corresponding authors. Tel.: +49 6221 546757; fax: +49 6221 544369.
E-mail addresses: ed.hurt@bzh.uni-heidelberg.de (E. Hurt), jochen.bassler@bzh.uni-heidelberg.de (J. Baβler).

0167-4889/$ – see front matter © 2009 Elsevier B.V. All rights reserved.  
doi:10.1016/j.bbamcr.2009.10.009

phosphorylation/dephosphorylation event, involving the Enp1–Ltv1–Rps3 complex and the protein kinase Hrr25 (see section 4.2) [22]. The cytoplasmic cleavage of the 20S pre-rRNA at site D, which yields the mature 18S rRNA, depends on several non-ribosomal factors (e.g., Nob1, Rio1, Rio2, Tsr1 and Fap7), as evidenced by the strong cytoplasmic accumulation of the 20S pre-rRNA upon mutation of these factors [23–26]. Recent evidence suggests that Nob1, which contains a PIN domain typical of endonucleases [27], catalyzes 20S cleavage [28,29]. This 20S>18S rRNA processing step completes the assembly of 40S subunits.

### 2.3. Assembly of 60S subunits

The first pre-ribosomal particles that could be isolated by the TAP method were nucleolar/nuclear 60S particles [30–33]. The co-purified proteins were then used in a reverse tagging approach to isolate additional intermediates ranging from early nucleolar to cytoplasmic pre-60S particles [8]. Since some factors are associated with several pre-ribosomal particles (i.e., Nop7, Nog1, Nsa2, Nsa3, Nug1, and Tif6) and only a selected few can be found on specific intermediates (i.e., Noc1, Noc3, Nsa1, Rsa4, and Rix1) [34,35], it became evident that only a subset of bait proteins appears to be suitable for the isolation of distinct particles (Fig. 3). The earliest rather distinct pre-60S particle can be purified by Ssf1, which contains a mixture of 27SA and 27SB pre-rRNA, ribosomal proteins, and about 30 non-ribosomal proteins, including diagnostic early factors like Noc1 and Rrp5 [30,35]. However, within this particle, no snoRNPs components could be identified, suggesting that an earlier particle exists. An attractive candidate is the Npa1-defined particle, which is composed of the 27SA₂ pre-rRNA and nearly 40 different non-ribosomal factors. In addition to typical early pre-60S factors (i.e., Noc1 and Nop4), eight RNA helicases, several snoRNP components, and even some 90S-associated factors could be co-purified with Npa1 [36]. Moreover, Npa1 was not identified within the Ssf1 particle [30].

The next distinct intermediate is defined by the nucleolar Nsa1 particle [35]. Besides the 5S rRNA, this particle almost exclusively contains the 27SB rRNA and has already made the exchange of the Noc1–Noc2 to the Noc2–Noc3 module [35,37]. A further module in these nucleolar particles is the Ytm1–Erb1–Nop7 subcomplex [34,38,39], which contributes to 5′ trimming of the 27SA₃ pre-rRNA by exonucleases Rat1 and Xrn1 [40]. Interestingly, both the Ssf1 and Nsa1 particles contain Rpf2, which together with Rrs1 mediates the incorporation of the 5S RNP complex (5S rRNA and Rpl5) and Rpl11 into pre-ribosomes [41]. The transition from the nucleolus (Nsa1 particle) to the nucleoplasm (Rix1 particle) is accompanied by major compositional changes ([35], J.B. and E.H. unpublished data).

Compared to the Nsa1 particle, the nuclear Rix1 particle has lost many factors including Spb1, Erb1, Nop2, Puf6, Ebp2, Ytm1, the Noc2–Noc3 subcomplex and the DExD/H-ATPases Dbp10, Drs1, Spb4, Dbp9, and Has1 (Fig. 4) [35]. On the other hand, the Rix1 particle has acquired new factors, such as Rea1, the Rix1–Ipi3–Ipi1 subcomplex, Rsa4, the Arx1–Alb1 subcomplex, Sda1, and Nug2. Moreover, the 27SB pre-rRNA has been almost completely processed into 25S and 7S/5.8S rRNAs ([8], J.B. and E.H. unpublished data). Interestingly, electron microscopy (EM) revealed that the Rix1-defined pre-ribosome exhibits a tadpole-like structure [34,42]. The tail of the particle is composed of the huge Rea1 AAA-ATPase that promotes the release of Rsa4 and the Rix1–Ipi3–Ipi1 subcomplex, thereby priming the pre-60S particle for export [34].

Such an export-competent particle (see section 3), which has already recruited Nmd3 and the Mex67-Mtr2 heterodimer, can be purified via Arx1 [43]. However, one has to consider that this purification still contains a minor pool of the Rix1 particle since Arx1 is already recruited at the level of this intermediate. Significantly, Arx1 can be found at both sides of the nuclear pore complex (NPC;

D. Kressler et al. / Biochimica et Biophysica Acta 1803 (2010) 673–683

40S Biogenesis Pathway

Nucleolus
Nucleoplasm
Cytoplasm

![Diagram](attachment:diagram.png)

Fig. 2. Pre-ribosomal particles along the 40S assembly pathway. The major intermediates of 40S pre-ribosomes, their rRNA content (dark green) and the presence of ATP/GTP-consuming enzymes are depicted (DExD/H-box ATPases in green, kinases in light blue, GTPase in orange). The nascent 35S pre-rRNA is modified and folded to form a first precursor of the 40S subunit. The cleavages at site A0, A1, A2, which generate the 20S pre-rRNA, are accompanied by a major exchange of non-ribosomal factors. The formation of the beak and final 20S processing occurs in the cytoplasm.

i.e., nucleoplasm and cytoplasm), indicating that the Arx1 particle represents a transiting particle. Once pre-60S ribosomes have reached the cytoplasm, the AAA-ATPase Drg1 binds to the Arx1 particle and promotes the release of Nog1 and Rlp24 [44]. Similarly, the cytoplasmic phosphatase Yvh1 facilitates the exchange of Mrt4 for the ribosomal protein Rpp0 (Fig. 4) [45,46]. Compared to the Rix1 particle, the Arx1 purification contains more mature 5.8S and less 7S rRNA (B. Bradatsch, B. Pertschy and E.H., unpublished data [8]). Moreover, there is evidence that the final 5.8S processing occurs in the cytoplasm [47–49].

The latest pre-60S particle that has been purified is represented by the Lsg1/Kre35 particle [8], which still contains Nmd3 and Tif6 [34], but already slightly diminished amounts of Arx1 (B. Bradatsch and E.H. unpublished data). As depicted in Fig. 4, release of Arx1 from cytoplasmic pre-60S ribosomes depends on Rei1 and Jjj1 and furthermore requires the prior action of Drg1 [44,50–53]. The recycling of the export adaptor Nmd3 involves the GTPase Lsg1 and is coupled to the incorporation of Rpl10 into pre-60S subunits (see section 4.5). Moreover, the release of Tif6 by Efl1 and Sdo1 from 60S subunits is a prerequisite for their association with 40S ribosomal subunits [54,55].

### 3. Nuclear export of ribosomal subunits

Remarkably, the export of pre-60S ribosomes relies on several nuclear export receptors (Fig. 4). This may be due to the enormous size of this particle, since the kinetics by which a shuttling transport receptor passes through the NPC are considerably slowed down with increasing cargo size [56]. Initial studies revealed that the nuclear export of both subunits depends on the general export factor Xpo1/Crm1 and the regulatory GTPase Ran [57–61]. The export factor Xpo1, which belongs to the family of karyopherins, binds to a short nuclear export sequence (NES) of its cargo and facilitates the translocation process by transiently interacting with the interior FG meshwork of the NPC (reviewed in references [62,63]).

In the case of 60S subunits, the conserved Nmd3 has been identified as an essential export adaptor that acts as a bridge between Xpo1 and the pre-60S particle. It is docked via its N-terminal domain on the export competent particle (e.g., Arx1 particle), whereas its C-terminal NES sequences are recognized by Xpo1 [57,61,64–66]. The binding site of Nmd3 lies in close proximity to the position of Rpl10 on mature 60S subunits [66], but its direct binding partner (rRNA or protein) remains to be identified. However, this binding site seems to be masked until the Rix1-Ipi3-Ipi1 complex has been released by the AAA-ATPase Rea1 [34]. After export the Xpo1-RanGTP-Nmd3 complex is dissociated by GTP hydrolysis and the subsequent release of Nmd3 is coupled to the incorporation of Rpl10 into pre-60S ribosomes (see section 4.5 and Hedges et al. [67] and West et al. [68]).

The second receptor mediating export of pre-60S subunits is the heterodimer Mex67-Mtr2, originally described as a general mRNA export factor ([43] and references therein). It was found that certain Mtr2 and Mex67 mutants are specifically impaired in 60S export and functionally linked to *nmd3* alleles [31,43]. Moreover, in vitro binding assays showed that the Mex67-Mtr2 heterodimer directly binds to nucleoporins and 5S rRNA [43,69], thus meeting all requirements of an export receptor.

The third factor contributing to pre-60S export is the unorthodox export receptor Arx1 whose binding site on 60S subunits is in proximity to Rpl25 [50,70]. By virtue of its interaction with the FG repeats of nucleoporins, Arx1 could shield the surface of the pre-60S

D. Kressler et al. / Biochimica et Biophysica Acta 1803 (2010) 673–683

# 60S Biogenesis Pathway

| Nucleolus | Nucleoplasm | Cytoplasm |
|-----------|-------------|-----------|
| Ssf1      | Rix1        | Arx1      |
|           |             | Lsg1      |

Fig. 3. Pre-ribosomal particles along the 60S assembly pathway. The different 60S pre-ribosomes are depicted together with their rRNA content (blue). The presence of ATP/GTP-consuming enzymes (GTPases in orange, DExD/H-box ATPases in green, AAA-type ATPases in pink), prominent subcomplexes (purple/yellow), and export factors (red) is shown. Bait proteins purifying the corresponding, distinct particles are indicated on top.

particle against the non-polar meshwork of the interior of NPCs, thereby facilitating translocation of pre-60S subunits [70]. In agreement with its accessory role, the arx1Δ null mutation confers synthetic lethality to nmd3, mex67, and mtr2 alleles [70].

Finally, Ecm1 is also functionally linked to the 60S export process. Initially, Ecm1 was identified in a synthetic lethal screen with the mtr2-33 mutant [31] and was further shown to be genetically linked to Mex67, Arx1, Nmd3, and a number of nucleoporins ([70,71], B. Bradatsch and E.H. unpublished data). Ecm1 is weakly associated with late pre-60S particles that also contain Nmd3, Mex67, and Arx1 [71]. However, its precise role during 60S export still needs to be elucidated.

In contrast to the large subunit, the export mechanism of the small subunit is still unclear. Despite the role of Xpo1 in the export of pre-40S ribosomes, no NES-adaptor has been identified to date. However, depletion of few ribosomal proteins, namely, Rps15, Rps10, Rps26,

D. Kressler et al. / Biochimica et Biophysica Acta 1803 (2010) 673–683

Fig. 4. Nuclear export of the pre-60S particle and cytoplasmic release of non-ribosomal factors. Export factors are depicted in red (green dot indicates interaction with FG repeats of nucleoporins); GTPases, in orange; Drg1 AAA-ATPase, in pink; and further non-ribosomal factors, in yellow. Ribosomal proteins, incorporated into cytoplasmic pre-60S particles, are displayed in blue.

involvement of Sda1 and Rrp12 in export of ribosomal subunits remains elusive.

## 4. Energy-consuming enzymes in ribosome biogenesis

Among the non-ribosomal factors, there are several essential ATP- or GTP-consuming enzymes, including DExD/H-box ATPases (19), GTPases (6), AAA-ATPases (3), kinases (3), and ABC proteins (2). These enzymes are believed to provide the energy that is required to confer directionality to the assembly and maturation process. Moreover, there are additional factors whose enzymatic activity is less clear, for example, Utp14 was found to associate with early 90S particle and in silico analysis suggested the presence of a P-loop, a motif participating in nucleotide binding [21]. While it is evident that the DExD/H-box proteins are almost exclusively engaged in early, nuclear assembly events, AAA-ATPases or GTPases predominantly act at later stages during 60S biogenesis (see below, Fig. 3). This clearly highlights the particular requirement for the different kinds of NTPases at distinct steps during the maturation of pre-ribosomal particles.

### 4.1. DExD/H-ATPases — modulators of RNP structures

#### 4.1.1. General aspects of DExD/H-ATPases

ATPases (or so-called RNA helicases) of the DEAD-, DEAH- and DExH-box families (collectively referred to as DExD/H-box proteins) belong to the SF2 superfamily of helicases [79]. They constitute the largest class of NTPases involved in ribosome biogenesis [80]. DExD/ H-box proteins are encoded by viral, prokaryotic, and eukaryotic genomes and they are involved in virtually all aspects of cellular RNA metabolism, including ribosome biogenesis, pre-mRNA splicing, mRNA export, translation initiation, and RNA turnover [81,82]. Their in vitro activities comprise RNA-dependent ATP hydrolysis, ATP-dependent RNA binding, ATP-dependent unwinding or strand separation of short double-stranded RNA or RNA/DNA duplexes, ATP-dependent dissociation of RNA-bound proteins, and ATP-independent annealing of complementary single-stranded RNA [81,83–85]. Due to the poor unwinding efficiency of long duplex substrates and in agreement with their anticipated in vivo substrates (see also below), it is unlikely that DExD/H-box proteins generally act as processive enzymes. Hence the original term RNA helicase, which was derived from their homology to the processive DNA helicases, does not reflect their actual mechanistic activity [81,84].

The ‘helicase’ core (around 350 to 400 amino acids) is earmarked by eight conserved sequence motifs, including the well-known Walker A [P-loop, A(x)₄GKT] and Walker B [DEAD, DEAH or DExH] motifs [81,86,87]. The available crystal structures reveal that the ‘helicase’ core of DExD/H-box proteins is built up of two similar domains that can adopt an open or closed conformation. The conserved motifs, located on both domains, mediate ATP binding, ATP hydrolysis, RNA binding, and/or joining of the two domains. RNA binding occurs on a surface that is created by both domains, opposite the ATP-binding site [81,84].

A combination of structural and mechanistic studies has led to the following general model [88–92]. Interdependent binding of RNA and ATP would promote an inter-domain movement from the open to the closed conformation. According to structural data, the closed conformation only allows binding of a bent single-stranded RNA and therefore causes local strand separation [88]. This bending should be sufficient for the destabilization of a short RNA duplex and thus cause unwinding, local remodelling of RNP structure, or release of RNA binding proteins. Apparently, these steps are energy-independent and ATP hydrolysis might be solely required to recycle the DExD/H-box ATPase [91].

Furthermore, most of these proteins contain additional N- and C-terminal domains of variable length and composition that flank the ‘helicase’ core. It has been suggested that these domains may contribute, either directly or indirectly via accessory proteins, to the recognition of the specific RNA substrates or regulate the ATPase activity [81,82]. In conclusion, DExD/H-box proteins can be viewed as energy-consuming chaperones/modulators of RNA or RNP structures [83,85,93].

#### 4.1.2. DExD/H-ATPases in ribosome biogenesis

Given that pre-ribosomal particles represent very sophisticated RNPs with high rRNA and snoRNA content, it is not surprising that the largest number of eukaryotic DExD/H-box proteins is dedicated to the ribosome biogenesis process [82,94]. Specifically, seven DExD/H-box proteins (Dbp4, Dbp8, Dhr1, Dhr2, Fal1, Rok1, and Rrp3) are required for 40S synthesis, ten (Dbp2, Dbp3, Dbp6, Dbp7, Dbp9, Dbp10, Drs1, Mak5, Mtr4/Dob1, and Spb4) are involved in the biogenesis of 60S subunits, while Has1 and Prp43 contribute to the assembly of both subunits (see Table S1). Almost all of these proteins are evolutionary conserved and likely fulfill specific non-redundant functions [80,82,83,95]. Comprehensive mutational and biochemical analyses indicate that these DExD/H-box proteins act indeed as ATPases within an RNP context [96–103], but their molecular function remains largely

elusive. We have inferred the presence of the DExD/H-box proteins on the different distinct pre-ribosomal particles from their appearance in biochemical purifications of pre-ribosomal factors or their co-precipitation of snoRNAs (Figs. 2, 3). Overall, our ‘association map’ fits well with the pre-rRNA processing phenotypes and their genetic and functional environments (Table S1).

Since DExD/H-box ATPases act very early on partially structured pre-RNPs decorated with snoRNPs, the following roles can be proposed: (i) local RNP remodeling and dissociation of RNA:RNA, RNA:protein or protein:protein interactions; (ii) rendering the pre-rRNA accessible for endo- or exonucleases; and (iii) snoRNP release by unwinding of pre-rRNA:snoRNA duplexes. While a general role of DExD/H-box ATPases in the release of modification guide snoRNAs can be likely discarded [104], three DEAD-box ATPases may contribute to the release of the essential U3, U14, and snR30 snoRNAs ([101,104,105] discussed in Bleichert and Baserga [82]), which are required for the early cleavages of the 35S pre-rRNA [16]. Mtr4/Dob1 (DExH-box) provides an example of a helicase preparing the pre-rRNA for nuclease attack as it assists the nuclear exosome in the 3′ processing of the 7S to 5.8S rRNA [106,107], likely by unwinding a duplex with a 3′-overhang [97,103]. However, the most common function of DExD/H-box ATPases appears to be the promotion of structural rearrangements in order to confer directionality to the assembly process. This energy-requiring stabilization of productive folding intermediates is most likely achieved by rejecting unfavorable substrates and/or promoting correct products [108,109], which may then be further stabilized upon recruitment of downstream factors. It will be an enormous challenge for future studies to identify the substrate(s) of DExD/H ATPases and to unravel the regulation and timing of their enzymatic activity.

### 4.2. Kinases

Surprisingly, all kinases implicated in ribosome biogenesis so far have a predominant role in 40S formation. However, due to their ubiquitous substrates, it is likely that all kinases have additional cellular functions. Hrr25, an isoform of casein kinase I, is implicated in multiple cellular processes including kinetochore attachment by microtubules, DNA repair, cell cycle, Ca²⁺ signaling, and modification of the NPC ([110,111] and references therein). Additionally, Hrr25 is a component of late pre-40S particles and phosphorylates members of the Ltv1–Enp1–Rps3 subcomplex [9,22]. A combination of in vitro maturation assays and EM studies revealed that phosphorylation and subsequent dephosphorylation of the Ltv1–Enp1–Rps3 complex is required for the formation of the 40S ‘beak’ structure and stable association of Rps3 with mature 40S subunits [22]. Recently, Hrr25 was also implicated in 60S biogenesis by phosphorylation of Tif6 (see section 4.5), a modification that is essential for its function [112]. Consistently, Hrr25 depletion shows 40S and 60S processing defects [22,113].

The homologous Rio1 and Rio2 belong to a family of atypical serine protein kinases. Besides its role in 40S biogenesis, Rio1 is required for cell cycle progression and chromosome maintenance [114]. Both Rio1 and Rio2 can shuttle between the nucleus and cytoplasm, are associated with pre-40S particles, and are involved in cytoplasmic processing of the 20S pre-rRNA [25,115,116]. However, Rio1 and Rio2 are both essential and fulfill non-redundant roles [25]. Moreover, only Rio2 is efficiently co-purified in TAP purifications of late 40S pre-ribosomes [9,22], suggesting that Rio1 is less stably associated with pre-40S particles. Despite variations within the conserved kinase motifs, Rio1 and Rio2 from *Archaeoglobus fulgidus* have a similar overall structure [117]. Interestingly, autophosphorylation occurs in a flexible loop that is in proximity of the ATP-binding pocket [115,118]. However, it remains to be determined whether autophosphorylation serves a regulatory role in vivo and whether pre-40S associated factors represent additional substrates.

In silico analysis indicated that Fap7 belongs to a novel family of predicted P-loop kinases [119]. Its depletion specifically affected processing of the 20S pre-rRNA and, moreover, Fap7 is a direct binding partner of Rps14 [24]. While mutations in the predicted Walker A or Walker B motifs affect Fap7 function [24], there is no experimental evidence that Fap7 displays kinase activity.

#### 4.3. Release of non-ribosomal factors by AAA-ATPases

AAA-type ATPases (ATPases associated with various cellular activities) are found in all organisms and they are involved in a variety of cellular processes. Common to all these proteins is a structurally conserved ATPase domain (~250 amino acids) that contains, besides other characteristic features, Walker A (P-loop) and Walker B (DExx-box) motifs [120]. Independent of whether these ATPases harbor one (type I), two (type II), or six AAA domains, they assemble into mostly hexameric rings that undergo structural changes during the ATPase cycle [121,122]. Hence, AAA-type ATPases utilize ATP hydrolysis to apply force on their substrates, which can trigger structural rearrangements or substrate release [120,121]. Three AAA-ATPases, namely, Rix7, Rea1/Mdn1, and Drg1/Afg2, were found to be essential for 60S ribosome biogenesis, and each removes specific non-ribosomal factors from pre-60S particles.

Drg1 contains two AAA domains (termed D1 and D2), with the D2 domain mediating its oligomerization into hexameric rings [123]. Drg1 is an exclusively cytoplasmic factor that is associated with the Arx1 pre-60S particle (Fig. 3) [44]. Interestingly, it can bind FG repeats of cytoplasmically located nucleoporins [124]. Importantly, depletion or mutation of Drg1 causes Nog1, Rlp24, and Arx1 to mislocalize to the cytoplasm. Consistently, both Nog1 and Rlp24 accumulate on Arx1 particles under these conditions [44]. In summary, these data strongly indicate a role for Drg1 in the release of Nog1, Rlp24, and Arx1 directly after the export of the pre-60S ribosomes (Fig. 4).

Rix7, another type II AAA-ATPase, was initially isolated in a screen for mutants that were defective in 60S export [125]. Further genetic screens identified Nsa1 as a functional interaction partner of Rix7. In addition, biochemical and cell biological data revealed that Rix7 is required for the release of Nsa1 from nucleolar pre-60S particles. Most notably, in a *rix7* mutant Nsa1 escapes to the cytoplasm, where it remains associated with translating 60S subunits [35]. However, it is not clear whether Nsa1 is a direct substrate of Rix7, and if so, how the release of Nsa1 is timed and triggered. Since Rix7 is the closest homologue of Cdc48/p97, which recognizes ubiquitinated substrates via its N-terminal domain [126], it is appealing to speculate that Rix7 may act on a modified substrate. In line with such a possibility, Rix7 and Nsa1 have previously been connected to the SUMO pathway [127] and, in addition, Nsa1 was found to be polyubiquitinated (D. K. and E.H., unpublished data).

The third AAA-ATPase involved in ribosome formation is Rea1. Rea1 was identified as a component of the Nug1 and Rix1 particle [8,31] and is required for 60S subunit formation and ITS2 processing [128]. Like dynein heavy chain, this huge ATPase (~550 kDa) has six AAA domains that form a ring structure [34,129]; however, sequence analysis suggested that only protomer 2, 3, 4, and 5 are active [129]. Notably, Rea1 contains a MIDAS (Metal Ion Dependent Adhesion Site) domain at the end of the long C-terminal tail that protrudes from the AAA domain ring structure. EM studies of the Rix1 pre-60S particle, which reveal a tadpole-like structure, indicate that Rea1 is attached to the pre-ribosome via its ring domain, in close proximity to the Rix1–Ipi3–Ipi1 subcomplex [34]. Significantly, Rea1's flexible tail contacts via its MIDAS domain the N-terminal domain of Rsa4, which is located on the pre-ribosome at a distinct site from the Rix1–Ipi3–Ipi1 subcomplex [34]. Being attached at two different sites on the Rix1 particle, ATP hydrolysis within the Rea1 ring domain may create a tensile force, which accounts for the observed release of Rsa4 and the Rix1–Ipi3–Ipi1 subcomplex from
this pre-60S intermediate [34] (Fig.3). Interestingly, Rea1's MIDAS domain also interacts with a nucleolar non-ribosomal factor, suggesting that Rea1 may mediate the release of further substrate proteins (J.B. and E.H., unpublished data). Future work should unravel the structural rearrangements that are brought about by the Rea1-mediated release of non-ribosomal factors and how these changes power progression of 60S biogenesis.

### 4.4. ATP-binding cassette (ABC) superfamily

Most ABC proteins are membrane transporters that utilize ATP hydrolysis to transport their cargo against a concentration gradient. However, two members of this family, Rli1 and Arb1, are both located within the nucleoplasm and cytoplasm and fulfill a role in ribosome assembly [130,131]. Preliminary mutational analyses suggested that ATP hydrolysis by Arb1 and Rli1 is essential [131,132]. Depletion of Arb1 caused a 40S biogenesis defect, but Arb1 appeared to be associated with 60S pre-ribosomes [131]. In case of Rli1, depletion leads to a 40S and 60S export defect [130,133], whereas the protein sediments with 40S and 80S particles. Importantly, purification of Rli1 yields the multi-protein translation initiation factor eIF3; moreover, Rli1 is genetically linked to Hcr1, a non-essential eIF3 subunit [130]. Altogether, the precise molecular roles of Arb1 and Rli1 in ribosome biogenesis remain to be explored.

### 4.5. GTPases in ribosome assembly

GTPases are key regulators of many cellular processes, and they are characterized by the presence of the so-called G-domain, which contains five conserved sequence motifs (G1–G5), including the characteristic Walker A motif (P-loop/G1) [134]. To date, six GTPases (Bms1, Efl1/Ria1, Lsg1/Kre35, Nog1, Nug1, and Nug2/Nog2) have been implicated in ribosome biogenesis. Bms1 is essential for 40S biogenesis by mediating the incorporation of Rcl1 into 90S particles [26,135,136]. Formation of a trimeric Rcl1–Bms1–GTP complex is a prerequisite for its recruitment to the U3 snoRNA within 90S particles (Fig. 2) [135,137]. Remarkably, the C-terminal domain of Bms1 seems to act as an intramolecular GTPase activator (GAP), which may trigger the release of Bms1 from pre-ribosomal particles [135,137].

The other five GTPases are involved in different aspects of 60S biogenesis. The cytoplasmic Efl1/Ria1 stimulates the release of Tif6, which is already recruited to nucleolar pre-60S particles (Figs. 3, 4) [34,55]. The GTPase activity of Efl1 is stimulated upon binding to cytoplasmic pre-60S subunits [55]. In addition, Tif6 dissociation is stimulated by Sdo1, the yeast homolog of the SBDS protein mutated in patients with the Shwachman-Diamond syndrome [54,138]. Since the presence of Tif6 on 60S subunits prevents subunit joining [139], its release appears to be a final step in 60S biogenesis and may occur only once all prior biogenesis steps (e.g., release of export factors) have been completed (Fig. 4).

Nog1 is an evolutionarily conserved, essential GTPase that associates with pre-60S particles from the nucleolar to the cytoplasmic stage [34,35,44,140]. Nog1 interacts genetically and physically with the ribosomal like protein Rlp24 [140]. Directly after the nuclear export, both proteins are released from the pre-60S particle in a Drg1-dependent manner and rapidly re-imported [44]. However, it is still not clear what the function of Nog1 is and which events are triggered by its GTP hydrolysis.

Nug1, Nug2/Nog2, and Lsg1/Kre35 belong to an exceptional group of circularly permuted GTPases (cpGTPases). In cpGTPases, the characteristic GTPase motifs (G1–G2–G3–G4–G5) are circularly permuted to G4–G1–G2–G3 without a clearly conserved G5 motif [134,141]. The cpGTPase domain of Nug1, Nug2, and Lsg1 is highly homologous to the prokaryotic YlqF/RbgA cpGTPase, for which structural information is available. Notably, these proteins are characterized by a unique G5* motif (DAR) that is not present in small Ras-like GTPases. Likely, these cpGTPases have a different activation mechanism since the switch II region D(S/T)PG is followed by a hydrophobic residue that cannot stimulate GTP hydrolysis [142]. Furthermore, GTP hydrolysis causes a mild structural change in the relative orientations between the GTPase and C-terminal domain [142]. Interestingly, YlqF can bind directly to L25 and its inactivation results in 45S intermediates lacking L16 and L27. Moreover, its GTPase activity could be stimulated by the free 50S subunit and 45S intermediates. Thus, this GTPase appears to be important for the incorporation of ribosomal proteins during the 50S assembly process [143–145].

The eukaryotic Nug1, Nug2, and Lsg1 cpGTPases possess extended N-terminal and C-terminal domains. However, only the GTPase activity of Nug1 has been shown directly [146]. Nevertheless, mutations inside the cpGTP-domain of Nug2 and Lsg1 disrupt ribosome biogenesis [33,67]. Similar to YlqF, there is significant evidence that Lsg1 is involved in the cytoplasmic incorporation of Rpl10 into the pre-60S ribosome [67,68]. According to the current model (Fig. 4), Rpl10, in complex with its chaperone Sqt1, binds to the exported pre-60S subunit in proximity to Nmd3. Subsequently, Lsg1 promotes the incorporation of Rpl10 through an exchange with Nmd3, thereby releasing Sqt1 and Nmd3 from the pre-ribosome [67,68]. However, the molecular details of this process remain unclear.

Similar to YlqF, which apparently binds to 23S rRNA, Nug1 can interact with 5S rRNA in vitro, and mutants of Nug1 are genetically linked to Rpl5 [146]. However, it remains unclear whether Nug1 or Nug2 are important for incorporation of any ribosomal proteins. Altogether, cpGTPases in general may have RNA binding activity and could stimulate incorporation of ribosomal proteins or trigger structural rearrangements within the pre-ribosome. Consistent with such a central function, cpGTPases appear to exclusively act in ribosome assembly, indicative of a specialized molecular role in rRNA metabolism.

### 5. Quality control of ribosomal subunits

Due to their complex nature, it is expected that a certain percentage of ribosomes may get incorrectly assembled or become damaged by, e.g., UV radiation or oxidative stress. These defective ribosomes may waste cellular energy by synthesizing non-functional or even harmful proteins. Such a negative effect can be avoided by the degradation of non-functional ribosomes. Accordingly, a surveillance mechanism needs to display three basic features: (i) detection of non-functional ribosomes/pre-ribosomes, (ii) recruitment of the degradation machinery, and (iii) degradation of the non-functional ribosome.

It was found in several studies that rRNA mutations in the decoding or peptidyl transferase center (PTC) of mature ribosomes are recognized and subjected to non-functional rRNA decay (NRD) [147,148]. These mutations did not affect ribosome assembly, but ultimately these non-functional, mature subunits were not present in translating ribosomes and had a decreased half-life. Pioneering work by Fujii et al. identified components of an E3 ubiquitin ligase complex (Mms1, Rtt101) that are involved in active ubiquitin-dependent degradation of non-functional 60S, but not 40S subunits [148].

In contrast, the NRD of 40S is similar to no-go decay (NGD) of mRNAs [147]. Cole et al. (2009) could demonstrate that the 18S decay depends on Dom34, Hbs1, the exonuclease Xrn1, and the exosome recruitment factor Ski7 [147]. Since 18S NRD depends on the active, ongoing translation process [147], it is suggested that non-functional 40S subunits are detected by an impaired or delayed translation process.

In addition to the surveillance of mature ribosomal subunits, it has been observed that mutants in ribosome biogenesis fail toaccumulate high levels of rRNA precursors [16]. This observation could be due to surveillance of pre-ribosomal particles or due to a general block in ribosome assembly. Since different intermediates are coupled to up- and downstream events, even a specific block will cause depletion of non-ribosomal factors that are needed for the progression of up- and downstream particles. Accordingly, it has been found that defects in the cytoplasmic recycling process (e.g., Efl1, Drg1, or Lsg1 mutants) affect nucleolar/nuclear stages of ribosome assembly (by nuclear depletion of Tif6, Nog1, or Nmd3, respectively). However, specialized surveillance mechanisms may still exist that control the quality of certain pre-ribosomal intermediates. There are several reports showing that the exosome, a complex of 3′–5′ exonucleases, is involved in the degradation of defective rRNAs ([108,149] and references therein). In addition, the TRAMP (Trf4/5, Air1, Mtr4 polyadenylation complex) complex was implicated in polyadenylation of rRNAs that could lead to exosome-mediated degradation [77]. However, it remains unclear whether specific surveillance mechanisms exist or whether assembly intermediates are degraded in general if they failed to be further processed within a certain time frame.

### 6. Ribosome assembly and human diseases

In agreement with the pivotal cellular role of ribosome biogenesis, several mutations within this process have been linked to human genetic diseases. To date most of these mutations are associated with the group of inherited bone marrow failure syndromes, which are characterized by reduced numbers of blood cells and predisposition to cancer. Apparently, precursor red blood cells have an enormous demand for ribosomes, due to the high number of cell divisions and the need to synthesize large numbers of ribosomes (dedicated to the synthesis of hemoglobin) before the loss of the nucleus. Accordingly, mutations in ribosomal proteins or non-ribosomal factors were linked to Diamond-Blackfan anemia (DBA), dyskeratosis congenita (DC), Shwachman-Diamond syndrome (SDS), and cartilage–hair hypoplasia (CHH) ([150] and references therein).

Diamond-Blackfan anemia (DBA) is characterized by red blood cell aplasia, macrocytic anemia, and growth retardation or congenital anomalies in approximately 30–50% of patients. For DBA, mutations in RPL5, RPL11, RPS17, RPS19, and/or RPS24, respectively, were found in about 30–40% of the patients ([150–153] and references therein). Moreover, mutations in RPL36, RPS15, and RPS27A may also contribute to DBA [151], which indicates that DBA is caused by a general delay in ribosome biogenesis [153].

The Shwachman-Diamond syndrome (SDS) is characterized by impaired hematopoiesis, exocrine pancreatic insufficiency, and increased leukemia risk [150]. SDS has been associated with mutations in the SBDS protein, which is homologous to the Sdo1 protein from yeast. Sdo1 and Efl1 are both non-essential cytoplasmic proteins, which are involved in the cytoplasmic dissociation of Tif6 from the almost mature pre-60S subunits (see section 4.5 [54,55]).

Dyskeratosis congenita (DC) is marked by abnormal skin pigmentation, mucosal leucoplakia, and nail dystrophy. The underlying cause of one form of DC is mutations in the protein dyskerin, which is the catalytic component of H/ACA snoRNPs responsible for pseudouridylation of rRNA [154,155]. However, since the snoRNP components are also associated with telomerase RNA and some patients have mutations in the telomerase and further associated factors, it remains open to which extent a defect in ribosome biogenesis contributes to DC ([156,157] and references therein).

Cartilage–hair hypoplasia (CHH), characterized by clinical features such as short limbed dwarfism, skeletal abnormalities, hypoplastic hair, immunodeficiency, and gastrointestinal dysfunction, is associated with mutations in the RNA component of the ribonuclease RNase MRP, which is involved in rRNA processing ([158,159] and references therein). However, RNase MRP has additional roles in RNA metabolism (replication in the mitochondria and degradation of the cyclin B2 mRNA), thus further studies will be required to understand the molecular basis of CHH.

Furthermore, the Bowen-Conradi syndrome, characterized by growth retardation, microcephaly, nose malformation, micrognathia, joint abnormalities, rocker bottom feet, and psychomotor delay, is caused by mutations in the human orthologue of yeast Emg1/Nep1, which is involved in 40S biogenesis ([160] and references therein).

All the aforementioned diseases have an increased risk for cancer development. Furthermore, various cancer cell lines show an increased expression of ribosomal and non-ribosomal factors involved in ribosome assembly (e.g., Nucleostemin/GNL3-L, a homologue of the Nug1 cpGTPase ([161] and references therein)). This is expected since (hyper-)actively dividing cells have a high demand for ribosomes [162]. Indeed, it has been found that the proto-oncogenic transcription factor myc can stimulate ribosome biogenesis ([163] and reference therein). However, in the converse situation, impaired ribosome assembly generates a feedback signal to cell cycle regulators causing cell cycle arrest or apoptosis. Specifically, RPL11 could inhibit myc function and RPL23, RPL11, RPL5, and RPS7 impair MDM2 activity, leading to stabilization of p53 [162,164]. At first glance it seems contradictory that a reduced biosynthesis of ribosomes causes unregulated cell growth. However, a delayed progression in cell cycle may increase the selective pressure for mutations to overcome the problems of a reduced ribosome biogenesis and the negative feedback on the cell cycle. Thus, bypassing of the regulation may finally contribute to cancer development.

### 7. Concluding remarks and perspective

The proteomic era revealed the complex nature of ribosome biogenesis and led to the identification of a huge number of non-ribosomal proteins. Now we face the challenge to determine their binding partners and molecular function. To succeed in this challenge, we have to apply novel approaches that allow us to identify protein or RNA binding partners, determine their function in vitro, and gain structural insight. A first step has been achieved through the development of the CRAC method that allows the identification of the rRNA fragments bound to non-ribosomal biogenesis factors [165]. Moreover, it is already possible to reconstitute distinct maturation steps during 60S and 40S subunit biogenesis in vitro [22,34,55] and to determine the EM structure of pre-ribosomal intermediates [22,34]. Altogether, the increasing structural information in combination with classical genetic and biochemical approaches will certainly unveil many exciting molecular details along the maturation path of eukaryotic ribosomal subunits.

### Acknowledgements

We thank H. Bergler, B. Bradatsch, A. Johnson, V.G. Panse, B. Pertschy, M. Fromont-Racine, D. Tollervey, E. Thomson, and J. Woolford for communicating unpublished data. We are grateful to B. Bradatsch, S. Ferreira-Cerca, M. Kos, and E. Thomson for helpful discussion and comments on the article. E.H. and J.B. are recipients of grants from the Deutsche Forschungsgemeinschaft (Hu363/10 and Gottfried Wilhelm Leibniz Program) and Fonds der Chemischen Industrie. We further apologize to scientists whose publications could not be cited due to space limitations.

### Appendix A. Supplementary data

Supplementary data associated with this article can be found, in the online version, at doi:10.1016/j.bbamcr.2009.10.009.
References

[1] D.L. Lafontaine, D. Tollervey, The function and synthesis of ribosomes, Nat. Rev. Mol. Cell. Biol. 2 (2001) 514–520.

[2] H.S. Zaher, R. Green, Fidelity at the molecular level: lessons from protein synthesis, Cell 136 (2009) 746–762.

[3] T.A. Steitz, A structural understanding of the dynamic ribosome machine, Nat. Rev. Mol. Cell Biol. 9 (2008) 242–253.

[4] J.R. Warner, The economics of ribosome biosynthesis in yeast, Trends Biochem. Sci. 24 (1999) 437–440.

[5] M. Fromont-Racine, B. Senger, C. Saveanu, F. Fasiolo, Ribosome assembly in eukaryotes, Gene 313 (2003) 17–42.

[6] A.K. Henras, J. Soudet, M. Gerus, S. Lebaron, M. Caizergues-Ferrer, A. Mougin, Y. Henry, The post-transcriptional steps of eukaryotic ribosome biogenesis, Cell. Mol. Life Sci. 65 (2008) 2334–2359.

[7] H. Tschochner, E. Hurt, Pre-ribosomes on the road from the nucleolus to the cytoplasm, Trends Cell Biol. 13 (2003) 255–263.

[8] T.A. Nissan, J. Bassler, E. Petfalski, D. Tollervey, E.C. Hurt, 60S pre-ribosome formation viewed from assembly in the nucleolus until export to the cytoplasm, EMBO J. 21 (2002) 5539–5547.

[9] T. Schäfer, D. Strau, E. Petfalski, D. Tollervey, E.C. Hurt, The path from nucleolar 90S to cytoplasmic 40S pre-ribosomes, EMBO J. 22 (2003) 1370–1380.

[10] S. Granneman, S.J. Baserga, Ribosome biogenesis: of knobs and RNA processing, Exp. Cell. Res. 296 (2004) 43–50.

[11] C. Mayer, I. Grummt, Ribosome biogenesis and cell growth: mTOR coordinates transcription by all three classes of nuclear RNA polymerases, Oncogene 25 (2006) 6384–6391.

[12] T. Moss, F. Langlois, T. Gagnon-Kugler, V. Stefanovsky, A housekeeper with power of attorney: the rRNA genes in ribosome biogenesis, Cell. Mol. Life Sci. 64 (2007) 29–49.

[13] D. Rudra, J.R. Warner, What better measure than ribosome synthesis? Genes Dev. 18 (2004) 2431–2436.

[14] D.E. Martin, T. Powers, M.N. Hall, Regulation of ribosome biogenesis: where is TOR? Cell. Metab. 4 (2006) 259–260.

[15] A.K. Henras, C. Dez, Y. Henry, RNA structure and function in C/D and H/ACA s (no)RNPs, Curr. Opin. Struct. Biol. 14 (2004) 335–343.

[16] J. Venema, D. Tollervey, Ribosome synthesis in Saccharomyces cerevisiae, Annu. Rev. Genet. 33 (1999) 261–311.

[17] Y.N. Osheim, S.L. French, K.M. Keck, E.A. Champion, K. Spasov, F. Dragon, S.J. Baserga, A.L. Beyer, Pre-18S ribosomal RNA is structurally compacted into the SSU processome prior to being cleaved from nascent transcripts in Saccharomyces cerevisiae, Mol. Cell 16 (2004) 943–954.

[18] J. Perez-Fernandez, A. Roman, J. De Las Rivas, X.R. Bustelo, M. Dosil, The 90S preribosome is a multimodular structure that is assembled through a hierarchical mechanism, Mol. Cell. Biol. 27 (2007) 5414–5429.

[19] N.J. Krogan, W.T. Peng, G. Cagney, M.D. Robinson, R. Haw, G. Zhong, X. Guo, X. Zhang, V. Canadien, D.P. Richards, B.K. Beattie, A. Lalev, W. Zhang, A.P. Davierwala, S. Mnaimneh, A. Starostine, A.P. Tikuisis, J. Grigull, N. Datta, J.E. Bray, T.R. Hughes, A. Emili, J.F. Greenblatt, High-definition macromolecular composition of yeast RNA-processing complexes, Mol. Cell 13 (2004) 225–239.

[20] P. Grandi, V. Rybin, J. Bassler, E. Petfalski, D. Strauss, M. Marzioch, T. Schäfer, B. Kuster, H. Tschochner, D. Tollervey, A.C. Gavin, E. Hurt, 90S pre-ribosomes include the 35S pre-rRNA, the U3 snoRNP, and 40S subunit processing factors but predominantly lack 60S synthesis factors, Mol. Cell 10 (2002) 105–115.

[21] F. Dragon, J.E. Gallagher, P.A. Compagnone-Post, B.M. Mitchell, K.A. Porwancher, K.A. Wehner, S. Wormsley, R.E. Settlage, J. Shabanowitz, Y. Osheim, A.L. Beyer, D.F. Hunt, S.J. Baserga, A large nuclear U3 ribonucleoprotein required for 18S ribosomal RNA biogenesis, Nature 417 (2002) 967–970.

[22] T. Schäfer, B. Maco, E. Petfalski, D. Tollervey, B. Bottcher, U. Aebi, E. Hurt, Hrr25-dependent phosphorylation state regulates organization of the pre-40S subunit, Nature 441 (2006) 651–655.

[23] A. Fatica, M. Oeffinger, M. Dlakic, D. Tollervey, Nob1p is required for cleavage of the 3′ end of 18S rRNA, Mol. Cell. Biol. 23 (2003) 1798–1807.

[24] S. Granneman, M.R. Nandineni, S.J. Baserga, The putative NTPase Fap7 mediates cytoplasmic 20S pre-rRNA processing through a direct interaction with Rps14, Mol. Cell. Biol. 25 (2005) 10352–10364.

[25] E. Vanrobays, J.P. Gelugne, P.E. Gleizes, M. Caizergues-Ferrer, Late cytoplasmic maturation of the small ribosomal subunit requires RIO proteins in Saccharomyces cerevisiae, Mol. Cell Biol. 23 (2003) 2083–2095.

[26] D. Gelperin, L. Horton, J. Beckman, J. Hensold, S.K. Lemmon, Bms1p, a novel GTP-binding protein, and the related Tsr1p are required for distinct steps of 40S ribosome biogenesis in yeast, RNA 7 (2001) 1268–1283.

[27] A. Fatica, D. Tollervey, M. Dlakic, PIN domain of Nob1p is required for D-site cleavage in 20S pre-rRNA, RNA 10 (2004) 1698–1701.

[28] B. Pertschy, C. Schneider, M. Gnädig, T. Schäfer, D. Tollervey, E. Hurt, RNA helicase Prp43 and its co-factor Pfa1 promote 20S to 18S rRNA processing catalyzed by the endonuclease Nob1, J. Biol. Chem. 284 (2009) 35079–35091, http://www.jbc.org/content/early/2009/09/29/jbc.M109.040774.long.

[29] A.C. Lamanna, K. Karbstein, Nob1 binds the single-stranded cleavage site D at the 3′-end of 18S rRNA with its PIN domain, Pric. Natl. Acad. Sci. U.S.A. 106 (2009) 14259–14264.

[30] A. Fatica, A.D. Cronshaw, D.M.D. Tollervey, Ssf1p prevents premature processing of an early pre-60S ribosomal particle, Mol. Cell 9 (2002) 341–351.

[31] J. Bassler, P. Grandi, O. Gadal, T. Le mann, D. Tollervey, J. Lechner, E.C. Hurt,

Identification of a 60S pre-ribosomal particle that is closely linked to nuclear export, Mol. Cell 8 (2001) 517–529.

[32] P. Harnpicharnchai, J. Jakovljevic, E. Horsey, T. Miles, J. Roman, M. Rout, D. Meagher, B. Imai, Y. Guo, C.J. Brame, J. Shabanowitz, D.F. Hunt, J.L. Woolford, Composition and functional characterization of yeast 66 s ribosome assembly intermediates, Mol. Cell 8 (2001) 505–515.

[33] C. Saveanu, D. Bienvenu, A. Namane, P.E. Gleizes, N. Gas, A. Jacquier, M. Fromont-Racine, Nog2p, a putative GTPase associated with pre-60S subunits and required for late 60S maturation steps, EMBO J. 20 (2001) 6475–6484.

[34] C. Ulbrich, M. Diepholz, J. Bassler, D. Kressler, B. Pertschy, G.K.B. Böttcher, E. Hurt, Mechanochemical removal of ribosome biogenesis factors from nascent 60S ribosomal subunit, Cell 138 (2009) 911–922.

[35] D. Kressler, D. Roser, B. Pertschy, E. Hurt, The AAA ATPase Rix7 powers progression of ribosome biogenesis by stripping Nsa1 from pre-60S particles, J. Cell Biol. 181 (2008) 935–944.

[36] C. Dez, C. Froment, J. Noaillac-Depeyre, B. Monsarrat, M. Caizergues-Ferrer, Y. Henry, Npa1p, a component of very early pre-60S ribosomal particles, associates with a subset of small nucleolar RNPs required for peptidyl transferase center modification, Mol. Cell. Biol. 24 (2004) 6324–6337.

[37] P. Milkereit, O. Gadal, A. Podtelejnikov, S. Trumtel, N. Gas, E. Petfalski, D. Tollervey, M. Mann, E. Hurt, H. Tschochner, Maturation of pre-ribosomes requires Noc-proteins and is coupled to transport from the nucleolus to the nucleoplasm, Cell 105 (2001) 499–509.

[38] L. Tang, A. Sahasranaman, J. Jakovljevic, E. Schleifman, J.L. Woolford Jr., Interactions among Ytm1, Erb1, and Nop7 required for assembly of the Nop7-subcomplex in yeast pre-ribosomes, Mol. Biol. Cell 19 (2008) 2844–2856.

[39] . M. Hölzel, M. Rohrmoser, M. Schlee, T. Grimm, T. Harasim, A. Malamoussi, A. Gruber-Eber, E. Kremmer, W. Hiddemann, G.W. Bornkamm, D. Eick, Mammalian WDR12 is a novel member of the Pes1-Bop1 complex and is required for ribosome biogenesis and cell proliferation, J. Cell Biol. 170 (2005) 367–378.

[40] A. Sahasranaman, J. Jakovljevic, J. Strahler, P. Andrews, J.R. Maddock, J.L. Woolford, A functional cluster of assembly factors regulate the exonuclease Rat1 to create precise 5′ends of 5.8S rRNA and is necessary for construction of the polypeptide tunnel, Oral presentation at the 8th Conference on Ribosome Synthesis, Germany, Regensburg, 2009.

[41] J. Zhang, P. Harnpicharnchai, J. Jakovljevic, L. Tang, Y. Guo, M. Oeffinger, M.P. Rout, S.L. Hiley, T. Hughes, J.L. Woolford Jr., Assembly factors Rpf2 and Rrs1 recruit 5S rRNA and ribosomal proteins rpL5 and rpL11 into nascent ribosomes, Genes Dev. 21 (2007) 2580–2592.

[42] T.A. Nissan, K. Galani, B. Maco, D. Tollervey, U. Aebi, E. Hurt, A pre-ribosome with a tadpole-like structure functions in ATP-dependent maturation of 60S subunits, Mol. Cell 15 (2004) 295–301.

[43] W. Yao, D. Roser, A. Köhler, B. Bradatsch, J. Bassler, E. Hurt, Nuclear export of ribosomal 60S subunits by the general mRNA export receptor Mex67-Mtr2, Mol. Cell 26 (2007) 51–62.

[44] B. Pertschy, C. Saveanu, G. Zisser, A. Lebreton, M. Tengg, A. Jacquier, E. Liebminger, B. Nobis, L. Kappel, I. van der Klei, G. Hogenauer, M. Fromont-Racine, H. Bergler, Cytoplasmic recycling of 60S pre-ribosomal factors depends on the AAA protein Drf1, Mol. Cell. Biol. 27 (2007) 6581–6592.

[45] K.-Y. Lo, Z. Li, F. Wang, E. Marcotte, A.W. Johnson, Ribosome stalk assembly requires the dual specificity phosphatase Yvh1 for the exchange of Mrt4 with P0, J. Cell Biol. (2009) 186.

[46] S. Kemmler, L. Occhipinti, M. Veisu, G.V. Panse, Yvh1 is required for a late maturation step in the 60S biogenesis pathway, J. Cell Biol. (2009) 186.

[47] K.M. Ansel, W.A. Pastor, N. Rath, A.D. Lapan, E. Glasmacher, C. Wolf, L.C. Smith, N. Papadopoulou, E.D. Lamperti, M. Tahlilani, J.W. Ellwart, Y. Shi, E. Kremmer, A. Rao, V. Heissmeyer, Mouse Eri1 interacts with the ribosome and catalyzes 5.8S rRNA processing, Nat. Struct. Mol. Biol. 15 (2008) 523–530.

[48] H.W. Gabel, G. Ruvkun, The exonuclease ERI-1 has a conserved dual role in 5.8S rRNA processing and RNAi, Nat. Struct. Mol. Biol. 15 (2008) 531–533.

[49] E. Thomson, D. Tollervey, The final step in 5.8S rRNA processing is cytoplasmic in Saccharomyces cerevisiae, Poster presentation at the 8th Conference on Ribosome Synthesis, Germany, Regensburg, 2009.

[50] N.J. Hung, A.W. Johnson, Nuclear recycling of the pre-60S ribosomal subunit-associated factor Arx1 depends on Rei1 in Saccharomyces cerevisiae, Mol. Cell. Biol. 26 (2006) 3718–3727.

[51] A. Lebreton, C. Saveanu, L. Decourty, J.C. Rain, A. Jacquier, M. Fromont-Racine, A functional network involved in the recycling of nucleocytoplasmic pre-60S factors, J. Cell Biol. 173 (2006) 349–360.

[52] E. Demoinet, A. Jacquier, G. Lutfalla, M. Fromont-Racine, The Hsp40 chaperone Jjj1 is required for the nucleo-cytoplasmic recycling of pre-ribosomal factors in Saccharomyces cerevisiae, RNA 13 (2007) 1570–1581.

[53] A.E. Meyer, N.J. Hung, P. Yang, A.W. Johnson, E.A. Craig, The specialized cytosolic J-protein, Jjj1, functions in 60S ribosomal subunit biogenesis, Proc. Natl. Acad. Sci. U. S. A. 104 (2007) 1558–1563.

[54] T.F. Menne, B. Goyenechea, N. Sanchez-Puig, C.C. Wong, L.M. Tonkin, P.J. Ancliff, R.L. Brost, M. Costanzo, C. Boone, A.J. Warren, The Shwachman-Bodian-Diamond syndrome protein mediates translational activation of ribosomes in yeast, Nat. Genet. 39 (2007) 486–495.

[55] B. Senger, D.L.J. Lafontaine, J.-S. Graindorge, O. Gadal, A. Camasses, A. Sanni, J.-M. Garnier, M. Breitenbach, E.C. Hurt, F. Fasiolo, The nucle(ol)ar Tif6p and Efl1p, a novel EF-2 like GTPase, are required for a late cytoplasmic step of ribosome synthesis, Mol. Cell 8 (2001) 1363–1373.

[56] K. Ribbeck, D. Görlich, The permeability barrier of nuclear pore complexes appears to operate via hydrophobic exclusion, EMBO J. 21 (2002) 2664–2671.

[57] C.R. Trotta, E. Lund, L. Kahan, A.W. Johnson, J.E. Dahlberg, Coordinated nuclear export of 60S ribosomal subunits and NMD3 in vertebrates, EMBO J. 22 (2003) 2841–2851.

[58] E. Hurt, S. Hannus, B. Schmelzl, D. Lau, D. Tollervey, G. Simos, A novel in vivo assay reveals inhibition of ribosomal nuclear export in Ran-cycle and nucleoporin mutants, J. Cell Biol. 144 (1999) 389–401.

[59] T.I. Moy, P.A. Silver, Requirements for the nuclear export of the small ribosomal subunit, J. Cell. Sci. 115 (2002) 2985–2995.

[60] T.I. Moy, P.A. Silver, Nuclear export of the small ribosomal subunit requires the Ran-GTPase cycle and certain nucleoporins, Genes Dev. 13 (1999) 2118–2133.

[61] F. Thomas, U. Kutay, Biogenesis and nuclear export of ribosomal subunits in higher eukaryotes depend on the CRM1 export pathway, J. Cell. Sci. 116 (2003) 2409–2419.

[62] H. Fried, U. Kutay, Nucleocytoplasmic transport: taking an inventory, Cell. Mol. Life Sci. 60 (2003) 1659–1688.

[63] Y.M. Chook, G. Blobel, Karyopherins and nuclear import, Curr. Opin. Struct. Biol. 11 (2001) 703–715.

[64] O. Gadal, D. Strauss, J. Kessl, B. Trumpower, D. Tollervey, E. Hurt, Nuclear export of 60S ribosomal subunits depends on Xpo1p and requires a NES-containing factor Nmd3p that associates with the large subunit protein Rpl10p, Mol. Cell. Biol. 21 (2001) 3405–3415.

[65] J.H.N. Ho, G. Kallstrom, A.W. Johnson, Nmd3p is a Crm1p-dependent adapter protein for nuclear export of the large ribosomal subunit, J. Cell Biol. 151 (2000) 1057–1066.

[66] J. Hedges, Y.I. Chen, M. West, C. Bussiere, A.W. Johnson, Mapping the functional domains of yeast NMD3, the nuclear export adapter for the 60 S ribosomal subunit, J. Biol. Chem. 281 (2006) 36579–36587.

[67] J. Hedges, M. West, A.W. Johnson, Release of the export adapter, Nmd3p, from the 60S ribosomal subunit requires Rpl10p and the cytoplasmic GTPase Lsg1p, EMBO J. 24 (2005) 567–579.

[68] M. West, J.B. Hedges, A. Chen, A.W. Johnson, Defining the order in which Nmd3p and Rpl10p load onto nascent 60S ribosomal subunits, Mol. Cell. Biol. 25 (2005) 3802–3813.

[69] K. Strässer, J. Bassler, E.C. Hurt, Binding of the Mex67p/Mtr2p heterodimer to FXFG, GLFG, and FG repeat nucleoporins is essential for nuclear mRNA export, J. Cell Biol. 150 (2000) 695–706.

[70] B. Bradatsch, J. Katahira, E. Kowalinski, G. Bange, W. Yao, T. Sekimoto, V. Baumgartel, G. Boese, J. Bassler, K. Wild, R. Peters, Y. Yoneda, I. Sinning, E. Hurt, Arx1 functions as an unorthodox nuclear export receptor for the 60S preribosomal subunit, Mol. Cell 27 (2007) 767–779.

[71] Y. Yao, E. Demoinet, C. Saveanu, P. Lenormand, A. Jacquier, M. Fromont-Racine, Ecm1, a new pre-60S nuclear export factor, Poster presentation at the 8th Conference on Ribosome Synthesis, Germany, Regensburg, 2009.

[72] S. Ferreira-Cerca, G. Poll, P.E. Gleizes, H. Tschochner, P. Milkerait, Roles of eukaryotic ribosomal proteins in maturation and transport of pre-18S rRNA and ribosome function, Mol. Cell 20 (2005) 263–275.

[73] I. Leger-Silvestre, P. Milkereit, S. Ferreira-Cerca, C. Saveanu, J.C. Rousselle, V. Choesmel, C. Guinefoleau, N. Gas, P.E. Gleizes, The ribosomal protein Rps15p is required for nuclear exit of the 40S subunit precursors in yeast, EMBO J. 23 (2004) 2336–2347.

[74] E. Vanrobays, A. Leplus, Y.N. Osheim, A.L. Beyer, L. Wacheul, D.L. Lafontaine, TOR regulates the subcellular distribution of DIM2, a KH domain protein required for cotranscriptional ribosome assembly and pre-40S ribosome export, RNA 14 (2008) 2061–2073.

[75] R.M. Seiser, A.E. Sundberg, B.J. Wollam, P. Zobel-Thropp, K. Baldwin, M.D. Spector, D.E. Lycan, Ltv1 is required for efficient nuclear export of the ribosomal small subunit in Saccharomyces cerevisiae, Genetics 174 (2006) 679–691.

[76] I. Zemp, T. Wild, M.F. O'Donohue, F. Wandrey, B. Widmann, P.E. Gleizes, U. Kutay, Distinct cytoplasmic maturation steps of 40S ribosomal subunit precursors require hRio2, J. Cell Biol. 185 (2009) 1167–1180.

[77] C. Dez, J. Houseley, D. Tollervey, Surveillance of nuclear-restricted pre-ribosomes within a subnuclear region of Saccharomyces cerevisiae, EMBO J. 25 (2006) 1534–1546.

[78] M. Oeffinger, M. Dlakic, D. Tollervey, A pre-ribosome-associated HEAT-repeat protein is required for export of both ribosomal subunits, Genes Dev. 18 (2004) 196–209.

[79] A.E. Gorbalenya, E.V. Koonin, Helicases: amino acid sequence comparisons and structure-function relationships, Curr. Opin. Struct. Biol. 3 (1993) 419–429.

[80] J. de la Cruz, D. Kressler, P. Linder, Ribosomal subunit assembly, in: M.O.J. Olson (Ed.), The Nucleolus, Landes Bioscience/Eurekah.com, Georgetown, Texas, 2004, pp. 258–285.

[81] O. Cordin, J. Banroques, N.K. Tanner, P. Linder, The DEAD-box protein family of RNA helicases, Gene 367 (2006) 17–37.

[82] F. Bleichert, S.J. Baserga, The long unwinding road of RNA helicases, Mol. Cell 27 (2007) 339–352.

[83] P. Linder, Dead-box proteins: a family affair-active and passive players in RNP-remodeling, Nucleic Acids Res. 34 (2006) 4168–4180.

[84] E. Jankowsky, M.E. Fairman, RNA helicases—one fold for many functions, Curr. Opin. Struct. Biol. 17 (2007) 316–324.

[85] E. Jankowsky, H. Bowers, Remodeling of ribonucleoprotein complexes with DExH/D RNA helicases, Nucleic Acids Res. 34 (2006) 4181–4188.

[86] S. Rocak, P. Linder, DEAD-box proteins: the driving forces behind RNA metabolism, Nat. Rev. Mol. Cell Biol. 5 (2004) 232–241.

[87] J.E. Walker, M. Saraste, M.J. Runswick, N.J. Gay, Distantly related sequences in the alpha- and beta-subunits of ATP synthase, myosin, kinases and other ATP-

requiring enzymes and a common nucleotide binding fold, EMBO J. 1 (1982) 945–951.

[88] T. Sengoku, O. Nureki, A. Nakamura, S. Kobayashi, S. Yokoyama, Structural basis for RNA unwinding by the DEAD-box protein *Drosophila* vasa, Cell 125 (2006) 287–300.

[89] Q. Yang, E. Jankowsky, The DEAD-box protein Ded1 unwinds RNA duplexes by a mode distinct from translocating helicases, Nat. Struct. Mol. Biol. 13 (2006) 981–986.

[90] Q. Yang, M. Del Campo, A.M. Lambowitz, E. Jankowsky, DEAD-box proteins unwind duplexes by local strand separation, Mol. Cell 28 (2007) 253–263.

[91] F. Liu, A. Putnam, E. Jankowsky, ATP hydrolysis is required for DEAD-box protein recycling but not for duplex unwinding, Proc. Natl. Acad. Sci. U. S. A. 105 (2008) 20209–20214.

[92] M. Del Campo, S. Mohr, Y. Jiang, H. Jia, E. Jankowsky, A.M. Lambowitz, Unwinding by local strand separation is critical for the function of DEAD-box proteins as RNA chaperones, J. Mol. Biol. 389 (2009) 674–693.

[93] P. Linder, P. Lasko, Bent out of shape: RNA unwinding by the DEAD-box helicase Vasa, Cell 125 (2006) 219–221.

[94] J. De la Cruz, D. Kressler, P. Linder, Unwinding RNA in Saccharomyces cerevisiae: DEAD-box proteins and related families, Trends Biochem. Sci. 24 (1999) 192–198.

[95] M. Abdelhaleem, L. Maltais, H. Wain, The human DDX and DHX gene families of putative RNA helicases, Genomics 81 (2003) 618–622.

[96] S. Granneman, C. Lin, E.A. Champion, M.R. Nandineni, C. Zorca, S.J. Baserga, The nucleolar protein Esf2 interacts directly with the DExD/H box RNA helicase Dbp8, to stimulate ATP hydrolysis, Nucleic Acids Res. 34 (2006) 3189–3199.

[97] X. Wang, H. Jia, E. Jankowsky, J.T. Anderson, Degradation of hypomodified tRNA (iMet) in vivo involves RNA-dependent ATPase activity of the DExH helicase Mtr4p, RNA 14 (2008) 107–116.

[98] S. Granneman, K.A. Bernstein, F. Bleichert, S.J. Baserga, Comprehensive mutational analysis of yeast DEXD/H box RNA helicases required for small ribosomal subunit synthesis, Mol. Cell. Biol. 26 (2006) 1183–1194.

[99] S. Rocak, B. Emery, N.K. Tanner, P. Linder, Characterization of the ATPase and unwinding activities of the yeast DEAD-box protein Has1p and the analysis of the roles of the conserved motifs, Nucleic Acids Res. 33 (2005) 999–1009.

[100] K.A. Bernstein, S. Granneman, A.V. Lee, S. Manickam, S.J. Baserga, Comprehensive mutational analysis of yeast DEXD/H box RNA helicases involved in large ribosomal subunit biogenesis, Mol. Cell. Biol. 26 (2006) 1195–1208.

[101] M. Kos, D. Tollervey, The Putative RNA Helicase Dbp4p Is Required for Release of the U14 snoRNA from Pre-ribosomes in Saccharomyces cerevisiae, Mol. Cell 20 (2005) 53–64.

[102] I. Garcia, O.C. Uhlenbeck, Differential RNA-dependent ATPase activities of four rRNA processing yeast DEAD-box proteins, Biochemistry 47 (2008) 12562–12573.

[103] J. Bernstein, D.N. Patterson, G.M. Wilson, E.A. Toth, Characterization of the essential activities of Saccharomyces cerevisiae Mtr4p, a 3′→5′ helicase partner of the nuclear exosome, J. Biol. Chem. 283 (2008) 4930–4942.

[104] M.T. Bohnsack, M. Kos, D. Tollervey, Quantitative analysis of snoRNA association with pre-ribosomes and release of snR30 by Rok1 helicase, EMBO Rep. 9 (2008) 1230–1236.

[105] X.H. Liang, M.J. Fournier, The helicase Has1p is required for snoRNA release from pre-rRNA, Mol. Cell. Biol. 26 (2006) 7437–7450.

[106] J. De la Cruz, D. Kressler, D. Tollervey, P. Linder, Dob1p (Mtr4p) is a putative ATP-dependent RNA helicase required for the 3′ end formation of 5.8S rRNA in Saccharomyces cerevisiae, EMBO J. 17 (1998) 1128–1140.

[107] J. Houseley, J. LaCava, D. Tollervey, RNA-quality control by the exosome, Nat. Rev. Mol. Cell Biol. 7 (2006) 529–539.

[108] J. Houseley, D. Tollervey, The many pathways of RNA degradation, Cell 136 (2009) 763–776.

[109] J.P. Staley, J.L. Woolford Jr., Assembly of ribosomes and spliceosomes: complex ribonucleoprotein machines, Curr. Opin. Cell Biol. 21 (2009) 109–118.

[110] M. Petronczki, J. Matos, S. Mori, J. Gregan, A. Bogdanova, M. Schwickart, K. Mechtler, K. Shirahige, W. Zachariae, K. Nasmyth, Monopolar attachment of sister kinetochores at meiosis I requires casein kinase 1, Cell 126 (2006) 1049–1064.

[111] C.P. Lusk, D.D. Waller, T. Makhnevych, A. Dienemann, M. Whiteway, D.Y. Thomas, R.W. Wozniak, Nup53p is a target of two mitotic kinases, Cdk1p and Hrr25p, Traffic 8 (2007) 647–660.

[112] U. Basu, K. Si, H. Deng, U. Maitra, Phosphorylation of mammalian eukaryotic translation initiation factor 6 and its Saccharomyces cerevisiae homologue Tif6p: evidence that phosphorylation of Tif6p regulates its nucleocytoplasmic distribution and is required for yeast cell growth, Mol. Cell. Biol. 23 (2003) 6187–6199.

[113] P. Ray, U. Basu, A. Ray, R. Majumdar, H. Deng, U. Maitra, The Saccharomyces cerevisiae 60 S ribosome biogenesis factor Tif6p is regulated by Hrr25p-mediated phosphorylation, J. Biol. Chem. 283 (2008) 9681–9691.

[114] M. Angermayr, A. Roidl, W. Bandlow, Yeast Rio1p is the founding member of a novel subfamily of protein serine kinases involved in the control of cell cycle progression, Mol. Microbiol. 44 (2002) 309–324.

[115] T.H. Geerlings, A.W. Faber, M.D. Bister, J.C. Vos, H.A. Raue, Rio2p, an evolutionarily conserved, low abundant protein kinase essential for processing of 20 S Pre-rRNA in Saccharomyces cerevisiae, J. Biol. Chem. 278 (2003) 22537–22545.

[116] E. Vanrobays, P.E. Gleizes, C. Bousquet-Antonelli, J. Noaillac-Depeyre, M. Caizergues-Ferrer, J.P. Gelugne, Processing of 20S pre-rRNA to 18S ribosomal RNA in yeast requires Rrp10p, an essential non-ribosomal cytoplasmic protein, EMBO J. 20 (2001) 4204–4213.

[117] N. LaRonde-LeBlanc, A. Wlodawer, A family portrait of the RIO kinases, J. Biol. Chem. 280 (2005) 37297–37300.

[118] N. LaRonde-LeBlanc, A. Wlodawer, The RIO kinases: an atypical protein kinase family required for ribosome biogenesis and cell cycle progression, Biochem. Biophys. Acta 1754 (2005) 14–24.

[119] D.D. Leipe, E.V. Koonin, L. Aravind, Evolution and classification of P-loop kinases and related proteins, J. Mol. Biol. 333 (2003) 781–815.

[120] J.P. Erzberger, J.M. Berger, Evolutionary relationships and structural mechanisms of AAA+ proteins, Annu. Rev. Biophys. Biomol. Struct. 35 (2006) 93–114.

[121] R.D. Vale, AAA proteins: lords of the ring, J. Cell Biol. 150 (2000) 13–19.

[122] P.I. Hanson, S.W. Whiteheart, AAA+ proteins: have engine, will work, Nat. Rev. Mol. Cell Biol. 6 (2005) 519–529.

[123] A. Zakalskiy, G. Hogenauer, T. Ishikawa, E. Wehrschutz-Sigl, F. Wendler, D. Teis, G. Zisser, A.C. Steven, H. Bergler, Structural and enzymatic properties of the AAA protein Drg1p from Saccharomyces cerevisiae. Decoupling of intracellular function from ATPase activity and hexamerization, J. Biol. Chem. 277 (2002) 26788–26795.

[124] L. Kappel, M. Loibl, I. Klein, G. Zisser, B. Pertschy, H. Bergler, The AAA-protein Drg1 interacts with the nuclear pore complex and is essential for early steps in cytoplasmic pre-60S formation, Oral and Poster presentation at the 8th Conference on Ribosome Synthesis, Germany, Regensburg, 2009.

[125] O. Gadal, D. Strau, J. Braspenninc, D. Hoepfner, E. Petfalski, P. Philippsen, D. Tollervey, E.C. Hurt, A nuclear AAA-type ATPase (Rix7p) is required for biogenesis and nuclear export of 60S ribosomal subunits, EMBO J. 20 (2001) 3695–3704.

[126] S. Jentsch, S. Rumpf, Cdc48 (p97): a “molecular gearbox” in the ubiquitin pathway? Trends Biochem. Sci. 32 (2007) 6–11.

[127] V.G. Panse, D. Kressler, A. Pauli, E. Petfalski, M. Gnadig, D. Tollervey, E. Hurt, Formation and nuclear export of preribosomes are functionally linked to the small-ubiquitin-related modifier pathway, Traffic 7 (2006) 1311–1321.

[128] K. Galani, T.A. Nissan, E. Petfalski, D. Tollervey, E. Hurt, Rea1, a Dynein-related nuclear AAA-ATPase, is involved in late rRNA processing and nuclear export of 60 S subunits, J. Biol. Chem. 279 (2004) 55411–55418.

[129] J.E. Garbarino, I.R. Gibbons, Expression and genomic analysis of midasin, a novel and highly conserved AAA protein distantly related to dynein, BMC Genomics 3 (2002) 18–28.

[130] A. Yarunin, V.G. Panse, E. Petfalski, C. Dez, D. Tollervey, E.C. Hurt, Functional link between ribosome formation and biogenesis of iron-sulfur proteins, EMBO J. 24 (2005) 580–588.

[131] J. Dong, R. Lai, J.L. Jennings, A.J. Link, A.G. Hinnebusch, The novel ATP-binding cassette protein ARB1 is a shuttling factor that stimulates 40S and 60S ribosome biogenesis, Mol. Cell. Biol. 25 (2005) 9859–9873.

[132] J. Dong, R. Lai, K. Nielsen, C.A. Fekete, H. Qiu, A.G. Hinnebusch, The essential ATP-binding cassette protein RLI1 functions in translation by promoting preinitiation complex assembly, J. Biol. Chem. 279 (2004) 42157–42168.

[133] G. Kispal, K. Sipos, H. Lange, Z. Fekete, T. Bedekovics, T. Janaky, J. Bassler, D.J. Aguilar Netz, J. Balk, C. Rotte, R. Lill, Biogenesis of cytosolic ribosomes requires the essential iron-sulphur protein Rli1p and mitochondria, EMBO J. 24 (2005) 589–598.

[134] D.D. Leipe, Y.I. Wolf, E.V. Koonin, L. Aravind, Classification and evolution of P-loop GTPases and related ATPases, J. Mol. Biol. 317 (2002) 41–72.

[135] K. Karbstein, S. Jonas, J.A. Doudna, An essential GTPase promotes assembly of preribosomal RNA processing complexes, Mol. Cell 20 (2005) 633–643.

[136] T. Wegierski, E. Billy, F. Nasr, W. Filipowicz, Bms1p, a G-domain-containing protein, associates with Rcl1p and is required for 18S rRNA biogenesis in yeast, RNA 7 (2001) 1254–1267.

[137] K. Karbstein, J.A. Doudna, GTP-dependent formation of a ribonucleoprotein subcomplex required for ribosome biogenesis, J. Mol. Biol. 356 (2006) 432–443.

[138] J.S. Luz, R.C. Georg, C.H. Gomes, G.M. Machado-Santelli, C.C. Oliveira, Sdo1p, the yeast orthologue of Shwachman-Bodian-Diamond syndrome protein, binds RNA and interacts with nuclear rRNA-processing factors, Yeast 26 (2009) 287–298.

[139] P. Raychaudhuri, E.A. Stringer, D.M. Valenzuela, U. Maitra, Ribosomal subunit antiassociation activity in rabbit reticulocyte lysates. Evidence for a low molecular weight ribosomal subunit antiassociation protein factor (Mr=25,000), J. Biol. Chem. 259 (1984) 11930–11935.

[140] C. Saveanu, A. Namane, P.E. Gleizes, A. Lebreton, J.C. Rousselle, J. Noaillac-Depeyre, N. Gas, A. Jacquier, M. Fromont-Racine, Sequential protein association with nascent 60S ribosomal particles, Mol. Cell Biol. 23 (2003) 4449–4460.

[141] K. Karbstein, Role of GTPases in ribosome assembly, Biopolymers 87 (2007) 1–11.

[142] J. Kim do, J.Y. Jang, H.J. Yoon, S.W. Suh, Crystal structure of YlqF, a circularly permuted GTPase: implications for its GTPase activation in 50 S ribosomal subunit assembly, Proteins 72 (2008) 1363–1370.

[143] Y. Matsuo, T. Oshima, P.C. Loh, T. Morimoto, N. Ogasawara, Isolation and characterization of a dominant negative mutant of Bacillus subtilis GTP-binding

protein, YlqF, essential for biogenesis and maintenance of the 50 S ribosomal subunit, J. Biol. Chem. 282 (2007) 25270–25277.

[144] Y. Matsuo, T. Morimoto, M. Kuwano, P.C. Loh, T. Oshima, N. Ogasawara, The GTP-binding protein YlqF participates in the late step of 50 S ribosomal subunit assembly in Bacillus subtilis, J. Biol. Chem. 281 (2006) 8110–8117.

[145] W.C. Uicker, L. Schaefer, R.A. Britton, The essential GTPase RbgA (YlqF) is required for 50S ribosome assembly in Bacillus subtilis, Mol. Microbiol. 59 (2006) 528–540.

[146] J. Bassler, M. Kallas, E. Hurt, The NUG1 GTPase reveals and N-terminal RNA-binding domain that is essential for association with 60 S pre-ribosomal particles, J. Biol. Chem. 281 (2006) 24737–24744.

[147] S.E. Cole, F.J. LaRiviere, C.N. Merrikh, M.J. Moore, A convergence of rRNA and mRNA quality control pathways revealed by mechanistic analysis of nonfunctional rRNA decay, Mol. Cell 34 (2009) 440–450.

[148] K. Fujii, M. Kitabatake, T. Sakata, A. Miyata, M. Ohno, A role for ubiquitin in the clearance of nonfunctional rRNAs, Genes Dev. 23 (2009) 963–974.

[149] S. Vanacova, R. Stefl, The exosome and RNA quality control in the nucleus, EMBO Rep. 8 (2007) 651–657.

[150] K.A. Ganapathi, A. Shimamura, Ribosomal dysfunction and inherited marrow failure, Br. J. Haematol. 141 (2008) 376–387.

[151] H.T. Gazda, M.R. Sheen, A. Vlachos, V. Choesmel, M.F. O'Donohue, H. Schneider, N. Darras, C. Hasman, C.A. Sieff, P.E. Newburger, S.E. Ball, E. Niewiadomska, M. Matysiak, J.M. Zaucha, B. Glader, C. Niemeyer, J.J. Meerpohl, E. Atsidaftos, J.M. Lipton, P.E. Gleizes, A.H. Beggs, Ribosomal protein L5 and L11 mutations are associated with cleft palate and abnormal thumbs in Diamond-Blackfan anemia patients, Am. J. Hum. Genet. 83 (2008) 769–780.

[152] V. Choesmel, S. Fribourg, A.H. Aguissa-Toure, N. Pinaud, P. Legrand, H.T. Gazda, P.E. Gleizes, Mutation of ribosomal protein RPS24 in Diamond-Blackfan anemia results in a ribosome biogenesis disorder, Hum. Mol. Genet. 17 (2008) 1253–1263.

[153] S. Robledo, R.A. Idol, D.L. Crimmins, J.H. Ladenson, P.J. Mason, M. Bessler, The role of human ribosomal proteins in the maturation of rRNA and ribosome production, RNA 14 (2008) 1918–1929.

[154] T. Kiss, E. Fayet, B.E. Jady, P. Richard, M. Weber, Biogenesis and intranuclear trafficking of human box C/D and H/ACA RNPs, Cold Spring Harb. Symp. Quant. Biol. 71 (2006) 407–417.

[155] D. Ruggero, S. Grisendi, F. Piazza, E. Rego, F. Mari, P.H. Rao, C. Cordon-Cardo, P.P. Pandolfi, Dyskeratosis congenita and cancer in mice deficient in ribosomal RNA modification, Science 299 (2003) 259–262.

[156] B. Gu, M. Bessler, P.J. Mason, Dyskerin, telomerase and the DNA damage response, Cell Cycle 8 (2009) 6–10.

[157] M. Kirwan, I. Dokal, Dyskeratosis congenita: a genetic disorder of many faces, Clin. Genet. 73 (2008) 103–112.

[158] L.D. Notarangelo, C.M. Roifman, S. Giliani, Cartilage-hair hypoplasia: molecular basis and heterogeneity of the immunological phenotype, Curr. Opin. Allergy Clin. Immunol. 8 (2008) 534–539.

[159] A.N. Martin, Y. Li, RNase MRP RNA and human genetic diseases, Cell Res. 17 (2007) 219–226.

[160] J. Armistead, S. Khatkar, B. Meyer, B.L. Mark, N. Patel, G. Coghlan, R.E. Lamont, S. Liu, J. Wiechert, P.A. Cattini, P. Koetter, K. Wrogemann, C.R. Greenberg, K.D. Entian, T. Zelinski, B. Triggs-Raine, Mutation of a gene essential for ribosome biogenesis, EMG1, causes Bowen-Conradi syndrome, Am. J. Hum. Genet. 84 (2009) 728–739.

[161] L. Romanova, A. Grand, L. Zhang, S. Rayner, N. Katoku-Kikyo, S. Kellner, N. Kikyo, Critical role of nucleostemin in pre-rRNA processing, J. Biol. Chem. 284 (2009) 4968–4977.

[162] M.S. Dai, H. Lu, Crosstalk between c-Myc and ribosome in ribosomal biogenesis and cancer, J. Cell Biochem. 105 (2008) 670–677.

[163] N. Gomez-Roman, Z.A. Felton-Edkins, N.S. Kenneth, S.J. Goodfellow, D. Athineos, J. Zhang, B.A. Ramsbottom, F. Innes, T. Kantidakis, E.R. Kerr, J. Brodie, C. Grandori, R.J. White, Activation by c-Myc of transcription by RNA polymerases I, II and III, Biochem. Soc. Symp. (2006) 141–154.

[164] S. Fumagalli, A. Di Cara, A. Neb-Gulati, F. Natt, S. Schwemberger, J. Hall, G.F. Babcock, R. Bernardi, P.P. Pandolfi, G. Thomas, Absence of nuclear disruption after impairment of 40S ribosome biogenesis reveals an rpL11-translation-dependent mechanism of p53 induction, Nat. Cell Biol. 11 (2009) 501–508.

[165] S. Granneman, G. Kudla, E. Petfalski, D. Tollervey, Identification of protein binding sites on U3 snoRNA and pre-rRNA by UV cross-linking and high-throughput analysis of cDNAs, Proc. Natl. Acad. Sci. U. S. A. 106 (2009) 9613–9618.
