
Apoptosis (2010) 15:269–278  
DOI 10.1007/s10495-009-0442-7  

---

**UNUSUAL MODEL SYSTEMS FOR CELL DEATH RESEARCH**

# Apoptosis in pre-Bilaterians: Hydra as a model

**Margherita Lasi · Charles N. David · Angelika Böttger**

Published online: 30 December 2009  
© Springer Science+Business Media, LLC 2009  

---

## Abstract

Hydra is a member of the ancient metazoan phylum Cnidaria and is an especially well investigated model organism for questions of the evolutionary origin of metazoan processes. Apoptosis in Hydra is important for the regulation of cellular homeostasis under different conditions of nutrient supply. The molecular mechanisms leading to apoptosis in Hydra are surprisingly extensive and comparable to those in mammals. Genome-wide sequence analysis has revealed the presence of large caspase and Bcl-2 families, the apoptotic protease activating factor (APAF-1), inhibitors of apoptotic proteases (IAPs), and components of a putative death receptor pathway. Regulation of apoptosis in Hydra may involve BH-3 only proteins and survival pathways, possibly including insulin signaling.

## Keywords

Apoptosis · Hydra · Evolution · BH-3 only · Bcl-2 · Caspase

---

## Introduction

Cnidaria, Porifera, Ctenophora, and Placozoa are the four ancient phyla that diverged from the metazoan lineage before the evolution of Bilaterians. Studying model organisms from these phyla is expected to shed light on the evolution of multicellularity in the animal kingdom. Thus, a number of new model organisms have emerged from the pre-bilateria metazoan phyla in recent years. EST and whole genome sequencing projects for these models have revealed an intriguingly high complexity of molecular developmental pathways in early metazoans.

The transition from single-celled to multi-cellular organisms required cooperation between different somatic cell types. Cells had to divide tasks requiring some to proliferate, others to differentiate, and, as we know now, a number of them to die. Forms of programmed cell death are found in all organisms, including the slime mold Dictyostelium, protozoans (e.g., Tetrahymena), algae (e.g., Volvox), fungi (Saccharomyces), and plants (reviewed in [1]). The term apoptosis is usually used for those forms of programmed cell death that are based on the action of caspases and Bcl-2 proteins and involve phagocytosis of dying cells. These appear to have evolved only in the animal lineage. It is therefore informative to look at model organisms from ancient animal phyla to understand how apoptotic pathways may have evolved.

### The cnidarian model organism Hydra

The fresh water polyp Hydra clearly is the most thoroughly studied non-bilaterian animal. The first experiments on the Cnidarian Hydra were done in the mid-eighteenth century immediately after the polyps had been discovered. They revealed an astonishing capacity of these animals to regenerate, which extends to complete regeneration of polyps from dissociated cells [2].

Hydra has a simple body plan (Fig. 1). It has one body axis with hypostome and tentacles at the apical end and a peduncle and basal disc at the basal end. It develops from the two germ layers after gastrulation of the embryo. The endodermal layer develops into the inner gastrodermis and

---

M. Lasi · C. N. David · A. Böttger (✉)  
Department Biology II, Biozentrum, Ludwig-Maximilians-University Munich, Großhaderner Str. 2, 82152 Planegg-Martinsried, Germany  
e-mail: boettger@zi.biologie.uni-muenchen.de  

C. N. David  
e-mail: david@zi.biologie.uni-muenchen.de

junctons including adhering belt desmosomes, septate junctions and gap junctions [6–8]. Epithelial cells in the Hydra body column proliferate continuously and are displaced towards the apical end into tentacles and towards the basal end into the peduncle and basal disk and into buds (Fig. 1a). Cells in the basal disk and in the tentacles stop proliferating and differentiate into specialised cell types, e.g. battery cells in the tentacles and foot cells in the peduncle. They are continuously sloughed off at the ends of the animals and replaced by newly arriving cells from the proliferation zone in the body column. Cells at the apical tip of the animal constitute a second proliferation zone and differentiate into hypostome cells. Thus, both epithelia are constantly renewed [9–11].

In addition to the two epithelial cell layers Hydra has specialised cells. These include nerve cells, which form a nerve net extending throughout the entire animal but not forming a brain or central ganglia. Gland cells are mostly present between the epithelial cells of the gastrodermis where they secrete digestive enzymes into the gastric cavity. The name-giving cell type for the phylum cnidaria are nematocytes, also called cnidocytes. These cells contain a nematocyst capsule, which is used to capture prey. They are incorporated into battery cells of the tentacles where they remain until used for stinging and poisoning of prey animals. Finally, Hydra produces male and female gametes from germ cells for sexual reproduction. All these specialised cell types are products of the pluripotent interstitial stem cell lineage. Interstitial cells are located in the spaces between the epithelial cells, mostly in the epidermis. They constitute a self-renewing proliferating stem cell population, part of which differentiates in each cell generation leading to the formation of the above mentioned cell types [9, 12–14].

## Apoptosis in Hydra

The permanent self-renewal of all Hydra tissues implies the need for tight regulation of cellular homeostasis, especially under conditions of changing growth rate. Hydra responds to limiting food supply by slowing down growth. The budding rate decreases and the animals become smaller [15]. However, corresponding changes in the epithelial cell cycle are not observed [15, 16] i.e. cell production continues although cell numbers do not increase. In this situation a large number of newly produced cells are phagocytosed by neighbouring cells [16]. This was first shown in Feulgen-DNA stained preparations of starving animals where significant numbers of pyknotic nuclei were found in round phagocytic vacuoles mainly in gastrodermal epithelial cells. Later re-examination of the cells in phagocytic vacuoles clearly indicated that they were apoptotic.

Fig. 1 a Schematic drawing of an adult Hydra with bud. Hypostome, tentacles, body column, peduncle and basal disk are indicated; b and c ectodermal and endodermal epithelial cell from macerates of Hydra cells; left hand panels show phase contrast; right hand panels show DNA stain with DAPI. In these images nuclei are stained as indicated by arrows. Apoptotic cells (apo) in phagocytic vacuoles show pyknotic chromatin (arrows). Some food granules in the endodermal cell are also stained with DAPI

the ectodermal layer into the outer epidermis. Epidermis and gastrodermis are separated by the acellular mesoglea, an extracellular matrix containing laminin and collagen [3–5]. Epidermis and gastrodermis are polarised and sealed epithelia containing cells that are joined by intercellular
They showed the typical apoptotic DNA distribution when stained with DAPI (Fig. 1 b,c) and were TUNEL-positive [17, 18]. Colchicine treatment, which disrupts the cell cycle, induces massive cell death in Hydra and the dying cells are phagocytosed by epithelial cells of the gastrodermis [17]. The degradation of DNA in colchicine-treated animals was demonstrated later to result in DNA laddering, confirming that cell death in *Hydra* is apoptotic [17]. Furthermore, apoptotic vacuoles could be stained with acridine orange (AO) [17, 18] due to the fact that AO accumulates at the acidic pH in the vacuoles, as shown before in *Tetrahymena* and in *Drosophila* [19, 20].

Regulation of cell numbers is an important physiological function of apoptosis in *Hydra*. However, there are several other situations where apoptosis has been shown to occur in *Hydra* including interspecies grafts and regeneration [21–26] and in response to cytostatic agents and irradiation [27–29].

A very intriguing special case of apoptosis is observed during differentiation of germ cells in *Hydra*. This applies to both, the male and the female germ line [30–32]. Here we will only discuss oogenesis. Germ cells differentiate from the interstitial stem cell lineage. As a first step in oogenesis a population of interstitial cells proliferates and forms a so-called egg-patch at the side of the mother animal. Cells in this egg patch later exit the mitotic cycle and start meiotic S-phase. Their appearance changes, they accumulate granules in their cytoplasm and they enlarge. A few of these cells then enter meiotic diplotene and grow even further. The others start to transfer their cytoplasm into these diplotene cells. At some point the few developing oocytes fuse, so that only one oocyte remains, continually increasing in size through cytoplasm transfer from all the other germ cells. The latter cells shrink and finally, their remains are phagocytosed by the oocyte. The morphological features of their nuclei suggest apoptosis. They are positive with TUNEL and their chromatin appears pyknotic [31]. The fascinating thing is that they are not further degraded after phagocytosis by the oocyte. The oocyte instead keeps them enclosed in phagocytic vacuoles even after fertilisation. During embryonic development they are distributed to the endodermal cells of the embryo and only get degraded after the new polyp has hatched [32]. How this arrest in apoptosis is achieved is not known at present.

### Molecular mechanisms for apoptosis are conserved in all animals

Molecular mechanisms of cell death are conserved throughout animal evolution. They involve caspases, Bcl-2 proteins, adaptor proteins, death receptors and inhibitors of apoptotic proteases (IAPs) (see Table 1).

Caspases are proteases with active-site cysteine residues. Their activation from inactive pro-caspases is the critical step in apoptosis and occurs through an amplifying cascade from upstream initiator caspases to downstream executioner caspases. The autocatalytic activation of initiator caspases requires a local increase in their concentration. This is achieved by adaptor proteins, which recruit pro-caspases through homotypic protein-protein interactions via DED-domains or CARD-domains. These domains are present in both the caspase pro-domains and the adaptor proteins [33, 34].

---

**Table 1 Comparison of apoptotic proteins present in *Hydra*, *Caenorhabditis*, *Drosophila* and mammals**

| Apoptotic proteins in different model organisms | *Hydra* | *Caenorhabditis* | *Drosophila* | *Homo* |
| --- | --- | --- | --- | --- |
| Caspases | HyCaspA, B, C, D, E, F, G, H, I, L, M | Ced 3, Csp-1, Csp-2 | Dredd, Dronc, Dream, Dcp-1, DRICE, Decay, Daydream | Caspase 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 12, 13, 14 |
| HyCARD 1, 2 |  |  |  |  |
| HyDED-Casp |  |  |  |  |
| HyDD-Casp |  |  |  |  |
| APAF-1 | HyAPAF-1 | Ced-4 | DARK | APAF-1 |
| Bcl-2 proteins: multi-domain | HyBcl-2-like 1, 2, 3, 4, 5, 6, 7HyBak-like 1, 2 | Ced-9 | Debcl, Buffy | Bcl-2, Bcl-xL, Bcl-W, Mcl-1, Bcl-B, Bcl-2A1, Bax, Bak, Bok |
| Bcl-2 proteins: BH3-only | HyBH3-only 1, 2, 3, 4 | Egl-1 | – | Bid, Bmf, NOXA, PUMA, Bad, Bim, Bik, Hrk |
| IAP | HyIAP | – | DIAP1, 2 | XIAP, NAIP, c-IAP1, 2 |
| Death receptor | HyTNFR-like receptor | – | Wengen | TNF-R1, 2CD95, CD40, p75 and many others |
| FADD | HyFADD | – | dFADD | FADD |

BH3-only proteins of the BNip family are present in all three organisms and have been omitted from the table.
Extrinsic apoptosis induction requires death receptors represented by homologs of the TNF-receptor family. Binding of ligands such as TNFα or FasL leads to receptor trimerisation and clustering. The adaptor proteins TRADD and/or FADD and pro-caspases with DED domains (e.g. pro-caspase 8) are recruited to these clusters and interact via DED-domains forming a so-called death inducing signalling complex (DISC) at the membrane. This leads to autoactivation of pro-caspases with DED domains and subsequent activation of executioner caspases [35, 36].

Intrinsic apoptosis induction is mediated by the apoptosome, a protein complex including APAF-1 and pro-caspases with CARD-domains. In mammals formation of the apoptosome requires cytochrome C, which is released from mitochondria. Mitochondrial integrity in turn is regulated by the Bcl-2 protein family with its multi and single domain members exhibiting pro- and anti-apoptotic functions. Anti-apoptotic members of this family are integrated into the outer mitochondrial membrane and inhibit pro-apoptotic family members. Apoptosis is induced by single domain (or BH3 only) Bcl-2 family members, which respond to stimuli such as DNA damage or the absence of survival signals. This changes the equilibrium between the multi-domain pro- and anti-apoptotic Bcl-2 proteins and leads to mitochondrial membrane permeabilisation and cytochrome C release [37].

Further components of apoptotic pathways are IAPs, inhibitors of apoptotic proteases. They contain characteristic zinc-finger motifs (BIR-domains: baculoviral IAP repeats) and directly inhibit the catalytic activity of caspases. In addition, many IAPs have a RING-domain and thus E3 ubiquitin ligase activity and mediate the proteasomal degradation of caspases (reviewed in [38]).

Caspases and Bcl-2 family proteins have been found in sponges, cnidarians and all bilaterian clades [39–41]. Their function in apoptosis has been best studied in mammals and in the major invertebrate model organisms *Caenorhabditis elegans* and *Drosophila melanogaster* [42].

*C. elegans* has only one member of the Bcl-2 family (ced-9), one caspase, which has a CARD-domain and is involved in apoptosis (ced-3), the APAF-1 homolog Ced-4 and one BH3-only protein (egl-1), which initiates somatic cell death. Cytochrome C can not bind to Ced-4 and thus cytochrome C release from mitochondria is probably not necessary for apoptosis in *Caenorhabditis*. Death receptors are not known to participate in apoptosis in *C. elegans*, and IAPs are not found encoded in the genome [43] (with the exception of survivin, whose primary function in *C. elegans* is not in apoptosis [44]).

*Drosophila* has seven caspases, five are effector caspases, one has a CARD-domain and one has a DED-domain, suggesting a more complex network for apoptosis in comparison with *Caenorhabditis*. Two Bcl-2 family members, Debcl and Buffy are also present in *Drosophila*, although their precise function in apoptotic pathways is not clear. Genetic studies have shown, that apoptosis in *Drosophila* is primarily regulated via IAPs. Death stimuli are relayed through expression of inhibitors of IAPs, namely RPR, GRIM and HID [45]. A death receptor has been described in *Drosophila*, but its activation does not lead to caspase recruitment (it lacks the DD domain, which is required for binding to FADD; [46–48]).

Mammals, by comparison with *Caenorhabditis* and *Drosophila*, exhibit highly complex extrinsic and intrinsic pathways for apoptosis induction [49]. They have a large caspase family with 15 members, which are divided into several groups including initiator and executioner caspases (reviewed in [50]). Mammals also have an extended Bcl-2 family with pro and antiapoptotic members ([39] and reviewed in [49]). In addition a large number of BH3-only proteins is known, only one of which, BNip, is conserved through evolution. Finally, mammals have four IAP genes involved in apoptosis regulation.

This brief summary indicates that caspases and Bcl-2 proteins first evolved in porifera and that these gene families increased in complexity from worms to mammals. However, recent analyses of whole genome sequences from cnidarians suggest that the caspase and Bcl-2 families were already highly complex in cnidarians and that *Caenorhabditis* and *Drosophila* lost many of the genes involved in apoptosis. Ten genes encoding putative caspases, 11 genes encoding putative Bcl-2 family members and several genes encoding APAF-1 homologs and IAPs have been found in genome searches of the sea anemones *Aiptasia* and *Nematostella* [51, 52]. An even larger number of caspases and Bcl-2 family proteins, including BH3-only proteins was found in the hydrozoan *Hydra magnipapillata*. As described above, apoptosis in *Hydra* has been investigated for many years and now, with the complete set of apoptosis genes available, studies elucidating their function are beginning. The first results are reviewed in the following.

## Hydra caspases and potential caspase activation mechanisms

Lysates from *Hydra* in which massive apoptosis has been induced by colchicine treatment exhibit a caspase 3 specific enzymatic activity detectable with fluorogenic substrates [17]. Genome and EST-library searches have now revealed 19 caspase genes. Two are incompletely covered by ESTs, these have been omitted from the following analysis. Eight have been cloned from cDNA (manuscript in preparation; for HyCARDCaspl Lange and Bosch, personal communication). Nine more are completely represented in the EST-library (Fig. 2a, b). Analysis of these 17 sequences

a

b

Group 1

| THGE | QACQQ | GSWFM |
| --- | --- | --- |
| HyCaspE |  |  |

| THGK | QACQQ | GSWYI |
| --- | --- | --- |
| HyCaspG |  |  |

| THGK | QACRG | GSWFV |
| --- | --- | --- |
| HyCaspB |  |  |

| SHGE | QACRG | GSWFI |
| --- | --- | --- |
| HyCaspC |  |  |

| THGE | QACRG | GSKFI |
| --- | --- | --- |
| HyCaspF |  |  |

| SHGL | QACQQ | GSWFI |
| --- | --- | --- |
| HyCaspH |  |  |

| SHGN | QACRG | GSWFI |
| --- | --- | --- |
| HyCaspD |  |  |

| SHGS | QACRG | GSWFI |
| --- | --- | --- |
| HyCaspA |  |  |

| CARD | SHGV | QACRV | GSYFI |
| --- | --- | --- | --- |
| HyCARD1 |  |  |  |

| CARD | SHGQ | QACQQ | GSWFI |
| --- | --- | --- | --- |
| HyCARD2 |  |  |  |

| SYGE | ESCLG | GSWFI |
| --- | --- | --- |
| HyPseudocasp1 |  |  |

| SIGD | QTCTG | NSWFI |
| --- | --- | --- |
| HyPseudocasp3 |  |  |

Group 2

| SHGF | QACQQ | GSWYI |
| --- | --- | --- |
| HyCaspl |  |  |

| DED | SHGY | QACQT | GSWYI |
| --- | --- | --- | --- |
| HyDEDCaspl |  |  |  |

| SHGF | QACQQ | GSYYI |
| --- | --- | --- |
| HyCaspl |  |  |

Group 3

| DD | AHGQ ESCRG | SPWVI |
| --- | --- | --- |
| HyDDCaspl |  |  |

| AHGE | DSCRG | SPWVK |
| --- | --- | --- |
| HyCaspm |  |  |

100 amino acids

scale bar

Fig. 2a Phylogenetic tree on the basis of a ClustalW alignment of caspase sequences. All sequences are represented by gene models (hmas) in the Hydra genome (http://hydrazone.metazome.net) and are covered by EST-sequences. Boxed caspase sequences have been cloned from Hydra cDNA. TreeTop-graphic; (http://www.genebee.msu.su/services/phtree_reduced.html); bootstraps are indicated; branches with bootstraps < 50 have dotted lines. Caspases are in 3 groups as indicated: group 1 contains two caspases with CARD domains, group 2 contains one caspase with a DED domain and group 3 one caspase with a DD domain; b Schematic representation of

Hydra caspase sequences in groups 1, 2 and 3. Sequences divided into prodomains, large subunits and small subunits are shown as bars according to scale; sequences surrounding active site histidine and cysteine residues and conserved substrate binding site in the small subunits are indicated; CARD-, DED-and DD-prodomain signatures are also shown where present. Accession numbers: Hydra caspase 3B: AF155128; Hydra caspase 3A: AF155127; Hydra caspase 3C: AY924233; Hydra caspase D: GU128636; Hydra caspase DD: GU128637; Hydra CARD caspase 1: GU128638; Hydra CARD caspase 2: GU128639; Hydra DED caspase: GU208869

shows that 15 have conserved active site residues, including a cysteine and a histidine residue. Moreover, the substrate pocket in the small subunit is also conserved in these sequences as GSWF/Y. In addition aspartate residues representing potential caspase cleavage sites between the large and small subunits are conserved (Fig. 2b). Two of the 17 sequences with high homology to caspases lack the important active site histidine residue. We thus call them pseudocaspases (Fig. 2b).

Phylogenetic sequence comparison of all Hydra caspases reveals that they fall into three groups (Fig. 2a). The largest group contains two caspases with CARD-domains

and two pseudocaspases. All the other caspases in this group have prodomains of varying length. These prodomains, however, do not show signatures for CARD domains although we have demonstrated experimentally that HyCaspA is capable of autoprocessing in contrast to HyCaspB, C and D, which are not (manuscript in preparation). The second group contains 3 caspases, one of which has a DED domain and could thus represent a Hydra caspase 8 homolog (although mammalian caspase 8 homologs have two DED-domains). The other members in the group have prodomains with no recognisable signature. The third group is represented by two unusual sequences.
HyDDCasp has a DD domain in its prodomain and is to our knowledge the only caspase with such a prodomain.

The presence of potential initiator caspases with CARD, DED and DD domains in Hydra suggests similar caspase activation mechanisms to those known from mammals. This includes both extrinsic and intrinsic caspase activation mechanisms. Based on the occurrence of additional proteins involved in both pathways it appears that both could be operating in Hydra. For the intrinsic pathway, in addition to two CARD-domain caspases, Hydra has an APAF-1 homolog shown schematically in Fig. 3a [53]. APAF-1 is a central component of the human apoptosome. It has an N-terminal CARD-domain for recruitment of CARD-domains from initiator caspases, e.g. caspase 9. C-terminally it has a wheel-like structure with two propellers, one formed by the first seven WD40 repeats and the other by the remaining six WD40 repeats [54, 55]. Both propellers are thought to interact with cytochrome C. HyAPAF-1 also has an N-terminal CARD domain that could potentially interact with the Hydra CARD-domain caspases. In Hydra APAF-1 only seven WD40 repeats are present (see Fig. 3a). These could potentially form a wheel, but only one propeller. It is therefore hard to predict whether the Hydra protein would bind to cytochrome C. Experiments are underway to show whether HyAPAF-1 is involved in caspase activation and if cytochrome C plays a role in this.

For the extrinsic pathway in Hydra, in addition to a DED-domain containing caspase, a FADD-like adaptor protein and a potential death receptor have been identified. The receptor has been cloned from Hydra cDNA (Fig. 3b,c). Whether these proteins engage in a stimulus induced death activation pathway remains to be analysed. Moreover, it is tempting to speculate that the unusual HyDDCasp could participate directly in the extrinsic apoptosis induction pathway by interacting with the DD-domain of the Hydra death receptor.

**Fig. 3**

a Schematic drawing of the domain structure of the Hydra APAF-1 (accession number: GU121227) homolog in comparison with human APAF-1 and C. elegans CED-4; CARD domains, NOD-domains and WD40-repeats are indicated; single WD40 domains as black boxes;

b schematic drawing of the domains of the Hydra TNF-receptor like protein (accession number: GU121226) sequence as deduced from cloned cDNA; TNF-R1 and R2 and transmembrane (TM) domains are indicated; c domain structure of the Hydra FADD homolog as deduced from the gene model hma1.131332 (http://hydrazome.metazome.net); DD and DED-domains are indicated; d domain structure of Hydra IAP (accession number: GU121225). Sequence was deduced from cloned cDNA; 3 BIR-domains and RING-domain are shown in comparison with human XIAP

| **a** | **APAF-1** |
| --- | --- |
| CARD | NOD | WD 40 repeats |
| HsAPAF-1 |  | β-propeller 1 | β-propeller 2 |
| CeCed-4 |  |  |
| HyAPAF-1 |  |  |

| **b** | **TNF-R** |
| --- | --- |
| TNF-R1 | TNF-R2 | TM | DD |
| HyTNF-R-like |  |  |  |

| **c** | **FADD** |
| --- | --- |
| DED | DD |
| HyFADD |  |  |

| **d** | **IAP** |
| --- | --- |
| BIR1 | BIR2 | BIR3 | RING |
| HsXIAP |  |  |  |
| HyIAP |  |  |  |
Caspase activity in *Hydra* could potentially also be regulated by IAPs or by pseudocaspases. One *bona fide* IAP gene was cloned from *Hydra* cDNA. It encodes a protein with 3 BIR domains and a C-terminal RING domain (Fig. 3d, manuscript in preparation). Moreover, the pseudocaspases in our first caspase group (Fig. 2a) could potentially act as inhibitors of “real” caspases by competing for substrate binding. Recent work in *C. elegans* has hinted at such a possibility. The *C. elegans* caspase-like protein Csp-3 does not contain an active-site cysteine [56]. Loss of the *csp-3* gene, however, caused increased cell death. Moreover, Csp-3 interacted with Ced-3 and inhibited its autoactivation (but not the activity of the processed caspase 3) [43].

### Hydra Bcl-2 protein family

The second large gene family we identified in the *Hydra* genome are the *bcl-2* genes. Nine such genes were found and cloned from cDNA (Fig. 4). Protein sequence conservation is very strong in the BH1 and BH2 domains but not in the BH3 and BH4 domains ([53] and manuscript in preparation). Similar observations have been made by

| BH4 | BH3 | BH1 | BH2 | TM |
| --- | --- | --- | --- | --- |
| HyBcl-2-like 1 | ![Diagram](https://via.placeholder.com/100x20) |
| HyBcl-2-like 2 | ![Diagram](https://via.placeholder.com/100x20) |
| HyBcl-2-like 3 | ![Diagram](https://via.placeholder.com/100x20) |
| HyBcl-2-like 4 | ![Diagram](https://via.placeholder.com/100x20) |
| HyBcl-2-like 5 | ![Diagram](https://via.placeholder.com/100x20) |
| HyBcl-2-like 6 | ![Diagram](https://via.placeholder.com/100x20) |
| HyBcl-2-like 7 | ![Diagram](https://via.placeholder.com/100x20) |
| HyBak-like 1 | ![Diagram](https://via.placeholder.com/100x20) |
| HyBak-like 2 | ![Diagram](https://via.placeholder.com/100x20) |

**Fig. 4** Schematic drawing of conserved BH domains of the *Hydra* Bcl-2 family proteins as deduced from cloned cDNAs. Dark grey coding region, small squared boxes BH4 domains, longitudinal stripes BH3 domains, diagonal stripes BH1 domain, dotted boxes BH2 domains, black box membrane anchoring domain. Accession numbers: Hydra Bak-like protein 1: EF104645; Hydra Bak-like protein 2: EU035760; Hydra Bcl-2-like 1: EF104646; Hydra Bcl-2-like 2: EF104647; Hydra Bcl-2-like 3: EU035765; Hydra Bcl-2-like 4: EU035764; Hydra Bcl-2-like 5: EU035763; Hydra Bcl-2-like 6: EU035762; Hydra Bcl-2-like 7: EU035761

analysing sequences from the sea anemone *Aiptasia* [51]. The incomplete conservation of *bcl-2* sequences in *Hydra* and sea anemone made it questionable whether these cnidarian Bcl-2 proteins were involved in programmed cell death. However, our recent experiments have now shown that six *Hydra* Bcl-2-like proteins inhibited apoptosis when expressed in mammalian cells. One Bcl-2-like protein mildly induced apoptosis. Two more sequences are very similar to mammalian Bak-proteins (they were termed HyBak-like 1 and 2) and strongly induced apoptosis in human cells. We also showed that most of the *Hydra* Bcl-2 proteins are localised to mitochondria and that this localisation is mediated via a C-terminal hydrophobic domain [57]. Two, however, were localised to the ER. We therefore suggest that *Hydra* Bcl-2 proteins are involved in apoptosis regulation and that apoptotic pathways involve both mitochondria and the ER [53].

In mammals, a large family of BH3-only proteins is involved in apoptosis induction. These include Bad, PUMA, NOXA and many more [49]. Most of these proteins are not related in primary sequence except for the presence of the BH3 domain. BH3-only proteins of this group have not been found in Drosophila. However, the BH3-only protein Egl-1 in *C. elegans* is upregulated in all somatic cells that are destined to die [58]. In addition, it is upregulated after genotoxic insult in the *C. elegans* germ line as a response to transcriptional activation by the *C. elegans* p53 homolog [59]. In pre-bilaterian animals no information about BH3-only proteins is currently available. Our searches of the *Hydra* genome have now revealed the presence of several genes encoding putative BH3-only proteins. Investigations are underway to show where these BH3-only proteins are expressed in *Hydra* and whether they participate in apoptosis regulation. These BH3-only proteins may hold an important key to understanding how apoptosis is induced in *Hydra*.

### Hydra insulin signalling in apoptosis regulation

Results discussed at the beginning of this review suggested that apoptosis is used to ensure cellular homeostasis in *Hydra*, especially under conditions of varying nutrition. Sensing of metabolic activity and processing of this information in order to adjust growth rate is achieved by insulin signalling in many organisms. In mammals insulin regulates glucose uptake, glycogen production and fat storage by acting as an endocrine hormone. In invertebrates, insulin signalling also regulates growth [60]. The picture here is very complex because *Drosophila* and *Caenorhabditis* have a large number of insulin-like peptides. *C. elegans* responds to food limitation by dauer formation and this is regulated via insulin pathways

Hydra proinsulin 1 rescues cells from apoptosis

![Graph](https://i.imgur.com/empty.png)

Fig. 5 HydraInsulin 1 (HyIns1) rescues cells expressing HyFOXO-GFP from apoptosis. The HyFOXO-GFP sequence was cloned into the vector HoTG. The HyProinsulin 1 sequence was cloned into the HoTG vector after removal of the GFP encoding sequence; this leads to expression of Hyproinsulin 1 without the GFP-tag; its expression in hydra cells was confirmed with an anti-proinsulin 1 antibody. Plasmids were transfected into hydra cells with the particle gun. GFP-expressing cells were scored as normal or apoptotic based on nuclear morphology. As a control, HyFOXO-GFP was expressed together with RFP. The plasmid containing the HyProinsulin 1 cDNA (Accession number: GU219979) was a kind gift from Rob Steele (Irvine).

including PI(3) kinase, PKB and the transcription factor FoxO, which translocates to the nucleus in the absence of insulin signals and induces gene expression that leads to dauer formation [61]. *Drosophila* growth is also dependent on food availability and is also regulated by insulin signalling through the insulin receptor, IRS, PKB and FoxO [62]. Prerequisites for insulin signalling are also known for *Hydra*. An insulin receptor homolog was cloned [63] and three insulin genes have now been identified (Steele, personal communication). It was shown that protein kinase B in *Hydra* has all the characteristics to be regulated by PI(3) kinase signalling [64] and that the PI(3) kinase inhibitor wortmannin induces massive apoptosis in *Hydra* [65]. When a *Hydra* FoxO-GFP fusion protein is expressed ectopically in *Hydra* epithelial cells, it is partially translocated into the nucleus due to the GFP-tag [57, 66]. As a consequence, apoptosis is induced. Interestingly, when we co-expressed the *Hydra* proinsulin-1 gene in such cells, apoptosis was inhibited (Fig. 5). These preliminary results constitute the first direct evidence that insulin signalling in *Hydra* is involved in apoptosis regulation.

### Perspectives

Based on the elaborate genetic machinery available for apoptosis in *Hydra* and our initial experimental data on

apoptosis in *Hydra* it appears that the network for the regulation of apoptosis in pre-bilaterian animals was very complex. Essentially, all pathways that we know to be involved in apoptosis regulation in mammals had already evolved in the common ancestor of bilaterians and cnidarians (Table 1). Hence, the simple cnidarian *Hydra* is emerging as a useful model organism to study new aspects of apoptosis as a regulator of cellular homeostasis. The completion of the *Hydra* genome sequencing in addition to new techniques to create transgenic *Hydra* provide a substantial experimental platform which will stimulate research in this area [67].

### References

1. Böttger A, David CN (2003) Evolution of Cell death: caspase mediated mechanisms in early metazoans; noncaspase mechanisms in single-celled eukaryotes. In: Grimm S (ed) Genetics of apoptosis. BIOS Scientific Publishers Ltd, Oxford, pp 145–153
2. Gierer A, Berking S, Bode H et al (1972) Regeneration of hydra from reaggregated cells. Nat New Biol 239:98–101
3. Fowler SJ, Jose S, Zhang X, Deutzmann R, Sarras MP Jr, Boot-Handford RP (2000) Characterization of hydra type IV collagen. Type IV collagen is essential for head regeneration and its expression is up-regulated upon exposure to glucose. J Biol Chem 275:39589–39599
4. Sarras MP Jr, Yan L, Grens A et al (1994) Cloning and biological function of laminin in *Hydra vulgaris*. Dev Biol 164:312–324
5. Zhang X, Boot-Handford RP, Huxley-Jones J et al (2007) The collagens of hydra provide insight into the evolution of metazoan extracellular matrices. J Biol Chem 282:6792–6802
6. Alexopoulos H, Bottger A, Fischer S et al (2004) Evolution of gap junctions: the missing link? Curr Biol 14:R879–R880
7. Filshie BK, Flower NE (1977) Junctional structures in hydra. J Cell Sci 23:151–172
8. Hand AR, Gobel S (1972) The structural organization of the septate and gap junctions of *Hydra*. J Cell Biol 52:397–408
9. Campbell RD (1967) Tissue dynamics of steady state growth in *Hydra littoralis*. II. Patterns of tissue movement. J Morphol 121:19–28
10. David CN, Gierer A (1974) Cell cycle kinetics and development of *Hydra attenuata*. III. Nerve and nematocyte differentiation. J Cell Sci 16:359–375
11. David CN, Campbell RD (1972) Cell cycle kinetics and development of *Hydra attenuata*. I. Epithelial cells. J Cell Sci 11:557–568
12. Bode HR (1996) The interstitial cell lineage of hydra: a stem cell system that arose early in evolution. J Cell Sci 109:1155–1164
13. David CN, Murphy S (1977) Characterization of interstitial stem cells in hydra by cloning. Dev Biol 58:372–383
14. Bosch TCG, David CN (1987) Stem cells of *Hydra magnipapillata* can differentiate into somatic and germ line cells. Dev Biol 121:182–191
15. Otto JJ, Campbell RD (1977) Tissue economics of hydra: regulation of cell cycle, animal size and development by controlled feeding rates. J Cell Sci 28:117–132
16. Bosch TC, David CN (1984) Growth regulation in *Hydra*: relationship between epithelial cell cycle length and growth rate. Dev Biol 104:161–171
17. Cikala M, Wilm B, Hobmayer E, Böttger A, David CN (1999) Identification of caspases and apoptosis in the simple metazoan *Hydra*. Curr Biol 9:959–962

18. Böttger A, Alexandrova O (2007) Programmed cell death in Hydra. Semin Cancer Biol 17:134–146
19. Abrams JM, White K, Fessler LI, Steller H (1993) Programmed cell death during Drosophila embryogenesis. Development 117:29–43
20. Mpoke SS, Wolfe J (1997) Differential staining of apoptotic nuclei in living cells: application to macronuclear elimination in Tetrahymena. J Histochem Cytochem 45:675–683
21. Bosch TCG, David CN (1986) Immunocompetence in Hydra: epithelial cells recognize self-nonself and react against it. J Exp Zool 238:225–234
22. Kuznetsov SG, Bosch TC (2003) Self/nonself recognition in Cnidaria: contact to allogeneic tissue does not result in elimination of nonself cells in Hydra vulgaris. Zoology (Jena) 106:109–116
23. Fujisawa T, David CN (1984) Loss of differentiating nematocytes induced by regeneration and wound healing in Hydra. J Cell Sci 68:243–255
24. Yaross MS, Bode HR (1978) Regulation of interstitial cell differentiation in Hydra attenuata. IV. Nerve cell commitment in head regeneration is position-dependent. J Cell Sci 34:27–38
25. Yaross MS, Bode HR (1978) Regulation of interstitial cell differentiation in Hydra attenuata. V. Inability of regenerating head to support nematocyte differentiation. J Cell Sci 34:39–52
26. Chera S, Ghila L, Dobretz K et al (2009) Apoptotic cells provide an unexpected source of Wnt3 signaling to drive hydra head regeneration. Dev Cell 17:279–289
27. Campbell RD (1976) Elimination by Hydra interstitial and nerve cells by means of colchicine. J Cell Sci 21:1–13
28. Fujisawa T, Sugiyama T (1978) Genetic analysis of developmental mechanisms in Hydra. IV. Characterization of a nematocyst-deficient strain. J Cell Sci 30:175–185
29. Marcum BA, Campbell RD (1978) Development of Hydra lacking nerve and interstitial cells. J Cell Sci 29:17–33
30. Aizenshtadt TB (1978) Oogenesis in Hydra. III. The growth and fusion of the oocytes. Ontogenez 9:115–123
31. Alexandrova O, Schade M, Böttger A, David CN (2005) Oogenesis in Hydra: nurse cells transfer cytoplasm directly to the growing oocyte. Dev Biol 281:91–101
32. Technau U, Miller MA, Bridge D, Steele RE (2003) Arrested apoptosis of nurse cells during Hydra oogenesis and embryogenesis. Dev Biol 260:191–206
33. Carrington PE, Sandu C, Wei Y et al (2006) The structure of FADD and its mode of interaction with procaspase-8. Mol Cell 22:599–610
34. Festjens N, Cornelis S, Lamkanfi M, Vandenabeele P (2006) Caspase-containing complexes in the regulation of cell death and inflammation. Biol Chem 387:1005–1016
35. Chowdhury I, Tharakan B, Bhat GK (2006) Current concepts in apoptosis: the physiological suicide program revisited. Cell Mol Biol Lett 11:506–525
36. Dempsey PW, Doyle SE, He JQ, Cheng G (2003) The signaling adaptors and pathways activated by TNF superfamily. Cytokine Growth Factor Rev 14:193–209
37. Giam M, Huang DC, Bouillet P (2008) BH3-only proteins and their roles in programmed cell death. Oncogene 27(Suppl 1):S128–S136
38. Vaux DL, Silke J (2005) IAPs, RINGs and ubiquitylation. Nat Rev Mol Cell Biol 6:287–297
39. Aouacheria A, Brunet F, Gouy M (2005) Phylogenomics of life-or-death switches in multicellular animals: Bcl-2, BH3-Only, and BNip families of apoptotic regulators. Mol Biol Evol 22:2395–2416
40. Wiens M, Diehl-Seifert B, Müller WE (2001) Sponge Bcl-2 homologous protein (BHP2-GC) confers distinct stress resistance to human HEK-293 cells. Cell Death Differ 8:887–898
41. Wiens M, Krasko A, Müller CI, Müller WE (2000) Molecular evolution of apoptotic pathways: cloning of key domains from sponges (Bcl-2 homology domains and death domains) and their phylogenetic relationships. J Mol Evol 50:520–531
42. Wang C, Youle RJ (2009) The role of mitochondria in apoptosis. Annu Rev Genet
43. Geng X, Shi Y, Nakagawa A et al (2008) Inhibition of CED-3 zymogen activation and apoptosis in Caenorhabditis elegans by caspase homolog CSP-3. Nat Struct Mol Biol 15:1094–1101
44. Silke J, Vaux DL (2001) Two kinds of BIR-containing protein— inhibitors of apoptosis, or required for mitosis. J Cell Sci 114:1821–1827
45. Lin N, Zhang C, Pang J, Zhou L (2009) By design or by chance: cell death during Drosophila embryogenesis. Apoptosis 14:935–942
46. Igaki T, Kanda H, Yamamoto-Goto Y et al (2002) Eiger, a TNF superfamily ligand that triggers the Drosophila JNK pathway. EMBO J 21:3009–3018
47. Kanda H, Igaki T, Kanuka H, Yagi T, Miura M (2002) Wengen, a member of the Drosophila tumor necrosis factor receptor superfamily, is required for Eiger signaling. J Biol Chem 277:28372–28375
48. Kauppila S, Maaty WS, Chen P et al (2003) Eiger and its receptor, Wengen, comprise a TNF-like system in Drosophila. Oncogene 22:4860–4867
49. Taylor RC, Cullen SP, Martin SJ (2008) Apoptosis: controlled demolition at the cellular level. Nat Rev Mol Cell Biol 9:231–241
50. Chowdhury I, Tharakan B, Bhat GK (2008) Caspases—an update. Comp Biochem Physiol B Biochem Mol Biol 151:10–27
51. Dunn SR, Phillips WS, Spatafora JW, Green DR, Weis VM (2006) Highly conserved caspase and Bcl-2 homologues from the sea anemone Aiptasia pallida: lower metazoans as models for the study of apoptosis evolution. J Mol Evol 63:95–107
52. Zmasek CM, Zhang Q, Ye Y, Godzik A (2007) Surprising complexity of the ancestral apoptosis network. Genome Biol 8:R226
53. Lasi M (2009) Das komplexe Netzwerk Apoptose regulierender Proteine in Hydra. Dissertation, Ludwig-Maximilians-University Munich
54. Yu X, Acehan D, Menetret JF et al (2005) A structure of the human apoptosome at 12.8. A resolution provides insights into this cell death platform. Structure 13:1725–1735
55. Acehan D, Jiang X, Morgan DG, Heuser JE, Wang X, Akey CW (2002) Three-dimensional structure of the apoptosome: implications for assembly, procaspase-9 binding, and activation. Mol Cell 9:423–432
56. Shaham S (1998) Identification of multiple Caenorhabditis elegans caspases and their potential roles in proteolytic cascades. J Biol Chem 273:35109–35117
57. Müller-Taubenberger A, Vos MJ, Böttger A et al. (2006) Monomeric red fluorescent protein variants used for imaging studies in different species. Eur J Cell Biol
58. Conradt B, Horvitz HR (1998) The C. elegans protein EGL-1 is required for programmed cell death and interacts with the Bcl-2-like protein CED-9. Cell 93:519–529
59. Schumacher B, Schertel C, Wittenburg N et al (2005) C. elegans ced-13 can promote apoptosis and is induced in response to DNA damage. Cell Death Differ 12:153–161
60. Edgar BA (2006) How flies get their size: genetics meets physiology. Nat Rev Genet 7:907–916
61. Burgering BM, Kops GJ (2002) Cell cycle and death control: long live Forkheads. Trends Biochem Sci 27:352–360
62. Puig O, Tjian R (2006) Nutrient availability and growth: regulation of insulin signaling by dFOXO/FOXO1. Cell Cycle 5:503–505
63. Steele R, Lieu P, Shenk M, Sarras M (1996) Response to insulin and the expression pattern of a gene encoding an insulin receptor

homologue suggests a role for an insulin-like molecule in regulating growth and patterning in Hydra. Dev Genes Evol 206:247–259

64. Herold M, Cikala M, MacWilliams H, David CN, Böttger A (2002) Cloning and characterisation of PKB and PRK homologs from Hydra and the evolution of the protein kinase family. Dev Genes Evol 212:513–519

65. David CN, Schmidt N, Schade M, Pauly B, Alexandrova O, Böttger A (2005) Hydra and the evolution of apoptosis. Integr Comp Biol 45:631–638

66. Böttger A, Alexandrova O, Cikala M, Schade M, Herold M, David CN (2002) GFP expression in Hydra: lessons from the particle gun. Dev Genes Evol 212:302–305

67. Wittlieb J, Khalturin K, Lohmann JU, Anton-Erxleben F, Bosch TC (2006) Transgenic Hydra allow in vivo tracking of individual stem cells during morphogenesis. Proc Natl Acad Sci USA 103:6208–6211
