
Photosynth Res (2010) 104:163–176  
DOI 10.1007/s11120-010-9531-2  

---

**REVIEW**

# Inorganic sulfur oxidizing system in green sulfur bacteria

Hidehiro Sakurai · Takuro Ogawa · Michiko Shiga · Kazuhito Inoue

Received: 14 September 2009 / Accepted: 16 January 2010 / Published online: 9 February 2010  
© Springer Science+Business Media B.V. 2010  

---

**Abstract** Green sulfur bacteria use various reduced sulfur compounds such as sulfide, elemental sulfur, and thiosulfate as electron donors for photoautotrophic growth. This article briefly summarizes what is known about the inorganic sulfur oxidizing systems of these bacteria with emphasis on the biochemical aspects. Enzymes that oxidize sulfide in green sulfur bacteria are membrane-bound sulfide-quinone reductase, periplasmic (sometimes membrane-bound) flavocytochrome *c* sulfide dehydrogenase, and monomeric flavocytochrome *c* (SoxF). Some green sulfur bacteria oxidize thiosulfate by the multi-enzyme system called either the TOMES (thiosulfate oxidizing multi-enzyme system) or Sox (sulfur oxidizing system) composed of the three periplasmic proteins: SoxB, SoxYZ, and SoxAXK with a soluble small molecule cytochrome *c* as the electron acceptor. The oxidation of sulfide and thiosulfate by these enzymes in vitro is assumed to yield two electrons and result in the transfer of a sulfur atom to persulfides, which are subsequently transformed to elemental sulfur. The elemental sulfur is temporarily stored in the form of globules attached to the extracellular surface of the outer membranes. The oxidation pathway of elemental sulfur to sulfate is currently unclear, although the participation of several proteins including those of the dissimilatory sulfite reductase system etc. is suggested from comparative genomic analyses.

**Keywords** Cytochrome · Green sulfur bacteria · Sox · Sulfide dehydrogenase · Thiosulfate

---

**Abbreviations**

- APS: Adenosine phosphosulfate
- cyt: Cytochrome
- FCSD: Flavo-cytochrome *c* sulfide dehydrogenase
- GSB: Green sulfur bacteria
- ISP: Iron-sulfur protein
- RC: Reaction center
- $S^0$: Zero-valence sulfur
- Sox: Sulfur oxidizing enzyme system
- SQR: Sulfide-quinone reductase
- TOMES: Thiosulfate oxidizing multi-enzyme system

---

H. Sakurai · K. Inoue  
Research Institute for Photosynthetic Hydrogen Production,  
Kanagawa University, Hiratsuka, Kanagawa 259-1293, Japan  

M. Shiga · K. Inoue (✉)  
Department of Biological Sciences, Kanagawa University,  
Hiratsuka, Kanagawa 259-1293, Japan  
e-mail: inouek01@kanagawa-u.ac.jp  

H. Sakurai  
Department of Biology, School of Education, Waseda University, Nishiwaseda, Shinjuku, Tokyo 169-8050, Japan  

T. Ogawa · K. Inoue  
Graduate School of Science, University of Tokyo,  
Bunkyo, Tokyo 113-0033, Japan  

---

**Utilization of reduced sulfur compounds by prokaryotes**

In the global sulfur oxidation-reduction cycle, in addition to non-biological chemical reactions, biological oxidation to sulfate of inorganic sulfur compounds such as sulfide, persulfides, elemental sulfur, thiosulfate, polythionate etc. constitutes an important part of the half biogeochemical cycle, with biological reduction of sulfate to sulfide or elemental sulfur (sulfate respiration) contributing to the other half.
Monographs describing the sulfur metabolism in prokaryotes and phototrophic organisms including eukaryotes, respectively, have recently appeared (Dahl and Friedrich (2008) and Hell et al. (2008), respectively). Recent advances in various fields of research on phototrophic sulfur bacteria (phylogeny, ecology, genomics, metabolism etc.) were extensively reviewed (Frigaard and Dahl 2009). The organisms that use reduced sulfur compounds as electron donors and for respiration or photosynthesis are confined to prokaryotes. Some of the representative families are: Chlorobiaceae (Genus Chlorobaculum, Chlorobium, An calochloris, Chloroherpeton, Pelodictyon, Prosthecochloris), Chromatiaceae (Chromatium, Allochromatium, Thiorhodococcus), Rhodobacteraceae (Paracoccus, Rhodobacter, Roseovarius), Bradyrhizobiaceae (Bradyrhizobium), Hyphomicrobiaceae (Starkeya), Burkholderiaceae (Ralstonia), Cyanobacteriaceae (Oscillatoria) in the domain Bacteria, and Sulfolobaceae (Genus Sulfolobus, Sulfuricoccus) in the domain Archaea. The metabolic pathways of Archaea seem to be largely different from those of Bacteria although they have several components in common (Friedrich et al. 2005). Sulfur metabolism in the Archaea will not be discussed in this review.

Green sulfur bacteria (GSB) carry out anoxygenic photosynthesis with reduced sulfur compounds such as sulfide and elemental sulfur, and for some species thiosulfate as the electron donor for photoautotrophic growth (Brune 1989, 1995b; Overmann 2001; Imhoff 2008; Frigaard and Dahl 2009). The one known exception is Chlorobium ferrooxidans, assigned to the genus Chlorobium based on 16S-rRNA encoding DNA sequence analysis, which is unable to use reduced sulfur compounds as electron donors for growth and uses ferrous iron instead (Heising et al. 1999). Some GSB also use hydrogen and ferrous iron as the electron donors (Frigaard and Bryant 2008a, b).

In heliobacteria, photoautotrophic growth on reduced sulfur compounds and CO₂ has not been observed, although some of them use sulfide and/or thiosulfate as the sole source of sulfur in the biosynthesis of cysteine and methionine (Madigan 2001). Heliobacteria will not be discussed further in this review.

involved in sulfur metabolism had been identified by the mid-1990s. These proteins include soluble flavocytochrome sulfide dehydrogenase (FCSD) (Kusai and Yamanaka 1973b), membrane-bound sulfide-quinone reductase (SQR) (Shahak et al. 1992), some components of the thiosulfate oxidizing multi-enzyme system (TOMES), etc. (extensively reviewed by Brune 1989, 1995b). Although there have subsequently been significant advances in studies of the inorganic sulfur metabolism by phototrophs, his reviews are still useful in providing a general view of relevant portions of the inorganic sulfur metabolism including the redox potentials of sulfur compounds.

Significant studies that have recently advanced our understanding of inorganic sulfur metabolism in GSB can be summarized as follows:

(1) Availability of the whole genome sequence of Chlorobaculum tepidum (formerly Chlorobium tepidum) (Eisen et al. 2002), followed by an ever-increasing number of additional sequences (currently, whole genome sequences from a total of 12 strains (11 complete sequences and one obtained by a whole genome shotgun sequencing project) are publicly available) ([http://www.ncbi.nlm.nih.gov/](http://www.ncbi.nlm.nih.gov/), accessed January 14th, 2010).

(2) Studies advancing the understanding of the biochemical pathways as well as characterization of the genes of the proteins involved in sulfur metabolism in various inorganic sulfur-utilizing prokaryotes (Kelly et al. 1997; Friedrich et al. 2001, 2005).

(3) Comparative genomic studies and proposed models of putative metabolic pathways of inorganic sulfur compounds in GSB (Eisen et al. 2002; Frigaard and Bryant 2008a, b; Frigaard and Dahl 2009).

(4) Creation of mutants by targeted inactivation of the genes assumed to be involved in inorganic sulfur metabolism followed by analyses of the characteristics of the mutants (Chan et al. 2008c, 2009; Azai et al. 2009).

(5) Biochemical characterization of the proteins involved in thiosulfate oxidation in GSB (Ogawa et al. 2008, in press).

Outline of the electron transport pathways leading to the reduction of the oxidized primary donor P840 in GSB

The RC of GSB is similar to photosystem I of oxygenic photosynthetic organisms, called Type I or iron-sulfur type RC with ferredoxin and flavodoxin as immediate electron acceptors (Sakurai et al. 1996; Hauska et al. 2001). The primary donor of the RC is a special pair of bacteriochlorophylls called P840 residing on the core homodimeric protein PscA, and its immediate electron donor is the RC-bound cytochrome (cyt) c-551 (PscC, CTI639 gene product)

Studies that have led to a greater understanding of inorganic sulfur metabolism

The pathways of inorganic sulfur metabolism show significant similarities among various prokaryotic groups, with some notable differences among them. In studies where inorganic sulfur compounds were supplied as electron donors to phototrophic bacteria, several proteins
(Fig. 1). In the moderately thermophilic *C. tepidum* TLS, based on in vitro studies there seem to be at least two pathways for the reduction of the bound cyt $c$-551 (PscC). Itoh et al. (2002) showed that a soluble mono-heme cyt $c$-554 of about 10 kDa (CT0075) donates electrons to the bound cyt $c$-551 rather than directly to photooxidized P840. A similar small, mono-heme cyt $c$-555 is a good electron acceptor in thiosulfate oxidation in *C. limicola f. thiosulfatophilum* (Kusai and Yamanaka 1973a) as in *C. tepidum* (Ogawa et al. 2008). There is evidence that *C. tepidum* has an alternative electron transfer pathway, as a membrane preparation from this bacterium catalyzes efficient electron transfer from quinol to photooxidized P840 without addition of soluble cyt $c$-554 (Oh-oka et al. 1998). Based on these results, the authors proposed the electron transport pathway as: sulfide → membrane-bound SQR → membrane-bound quinol oxidoreductase (cyt $b$/Rieske-type iron–sulfur protein (ISP) complex) (Klughammer et al. 1995) → cyt $c$-556 bound to the complex → cyt $c$-551 bound to the RC → P840. The plausibility

of this reaction scheme is supported by studies of mutant cells disrupted in the soluble cyt $c$-554 gene (CT0075). These mutant cells could still grow photoautotrophically in a medium containing sulfide and thiosulfate, although at a lower rate than the wild-type (Tsukatani et al. 2006).

### Enzymes of inorganic sulfur oxidation

#### Enzymes of sulfide oxidation in GSB

There are four kinds of sulfide-oxidizing enzymes in GSB that have been characterized so far by in vitro biochemical assays (Table 1).

##### Sulfide-quinone reductase (SQR)

SQR (also called sulfide-quinone oxidoreductase) is distributed over various prokaryotes as well as eukaryotic

![Diagram](attachment:diagram.png)

Fig. 1 Overview of known and hypothesized pathways of electron and inorganic sulfur compounds in *Chlorobaculum tepidum*. A part of the figure is derived from Eisen et al. (2002), Frigaard and Bryant (2008a, b), Dahl (2008), and Cort et al. (2008). Thin arrows indicate pathways of $e^-$, and thick ones, metabolic or transport pathways of substrates/products. ? hypothetical, Apr adenosine 5′-phosphosulfate reductase, APS adenosine 5′-phosphosulfate, $c$ cytochrome $c$ with the wavelength (nm) of the $\alpha$-band of the reduced form, cyt $b$/ISP cyt $b$/Rieske-type ISP complex, Dsb homologues encoded by genes in the $dsb$ gene cluster encoding thiol:disulfide interchange proteins, $Dsr$

proteins encoded by genes in the $dsr$ gene cluster encoding dissimilatory sulfite reductase proteins, FCSD flavocytochrome $c$ sulfide dehydrogenase, $Fd$ ferredoxin, $FNR$ ferrodoxin-NAD(P)$^+$/NAD(P)H oxidoreductase, $IM$ inner membrane, $MQ$ menaquinone, $OM$ outer membrane, $Qmo$ quinone oxidizing membrane protein, $R^{1,2,3}$ hypothetical thiol, $RC$ reaction center, $[S^0]$ the zero-valence sulfur or the equivalent (sulfur atom in organic and inorganic polysulfide, elemental sulfur), Sox proteins encoded by genes in the $sox$ gene cluster, SQR sulfide-quinone reductase, TOMES thiosulfate oxidizing multi-enzyme system. For details, see text

Table 1 Some kinetic properties of inorganic sulfur oxidizing enzymes

| $K_{\mathrm{m}}$ ($\mu \mathrm{M}$) | $V_{\max }$ ($\mu \mathrm{mol} \mathrm{e}^{-} / \mu \mathrm{mol} / \mathrm{min}$) | pH optimum | $\mathrm{e}^{-}$ acceptor used |
|------------------------------------|-------------------------------------------------------------|------------|---------------------------|
| H₂S (p$K_{1}=7.06$, p$K_{2}=12.44$) | SQR | $Oscillatoria \, limnetica$ | 8 | 3.3–5 | plastoquinone-1${}^{b}$ |
|                                   | SQR | Rhodobacter capsulatus | 2 | 3.5 | 6.3 | dodecyl ubiquinone${}^{c}$ |
|                                   | FCSD | Thiobacillus sp. | 1.7 | 107${}^{a}$ | 8.6 | horse heart cyt $c^{d}$ |
|                                   | SoxF | P. pantotrophus | 2.3 | 5.5 | 6 | horse heart cyt $c^{e}$ |
|                                   | SoxF | C. tepidum | 2.1 | 100 | >8 | $C. t \, cyt \, c-554^{f}$ |
|                                   | SoxF | C. tepidum | 2.1 | 0.2 | >8 | Horse heart cyt $c^{f}$ |
|                                   | Core TOMES | C. tepidum | 41 | 16 | 6–6.5 | $C. t \, cyt \, c-554^{f}$ |
| H₂S₂O₃ (p$K_{1}=1.56$, p$K_{2}=3.73$) | Core TOMES | C. tepidum | 150 | 7.2 | 6–6.5 | $C. t \, cyt \, c-554^{f}$ |
|                                   | Core TOMES + SoxF | C. tepidum | 150 | 13.8 | 6–6.5 | $C. t \, cyt \, c-554^{f}$ |
| H₂SO₃ (p$K_{1}=1.78$, p$K_{2}=6.99$) | Core TOMES | C. tepidum | 23 | 3.5 | $C. t \, cyt \, c-554^{f}$ |

${}^{a}$ Calculated from :FCSD 51 kDa, $\nu=2.1 \, \mu \mathrm{mol} \, \mathrm{cyt} \, c$ reduced mg protein${}^{-1} \, \mathrm{min}^{-1}$

${}^{b}$ Arieli et al. 1994

${}^{c}$ Schütz et al. 1997

${}^{d}$ Visser et al. 1997

${}^{e}$ Quentmeier et al. 2004

${}^{f}$ Ogawa et al. in press

mitochondria (see Shahak and Hauska 2008). SQR from the cyanobacterium *Oscillatoria limnetica* is a membrane-bound protein that could be solubilized by mild detergent treatment. The purified protein was monomeric and shown to bind FAD as the prosthetic group (Arieli et al. 1994). SQR is assumed to supply electrons to the quinone pool presumably with persulfide, but not elemental sulfur as the product (Griesbeck et al. 2002) (see “Transient elemental sulfur formation” section, for further metabolism of persulfide).

The presence of SQR activity in GSB was first demonstrated in *Chlorobium limicola* f. *thiosulfatophilum* (Shahak et al. 1992). A gene encoding SQR has been identified in all of the genomes of GSB sequenced thus far (Frigaard and Bryant 2008a,b). In GSB, the reduced menaquinone is then oxidized by cyt b/ISP complex accompanied by the generation of a proton-motive force (Hauska et al. 2001).

### Flavocytochrome c-sulfide dehydrogenase (FCSD or FCC)

Flavocytochrome c is a relatively abundant soluble protein in *Chlorobium thiosulfatophilum* (Meyer et al. (1968). The protein was subsequently found to have sulfide dehydrogenase activity in vitro with soluble small molecule cyt c as the electron acceptor, and named flavocytochrome c-sulfide dehydrogenase (FCSD) (Kusai and Yamanaka 1973b). The protein is a tightly bound heterodimer composed of a c-type cytochrome subunit and a flavoprotein subunit encoded by the $fccA$ and $fccB$ genes respectively, in *Allochromatium*

### Monomeric flavoprotein (SoxF)

Subsequent studies revealed that homologous genes encoding proteins with significant amino acid sequence identity with FCSD were frequently found in the thiosulfate oxidizing gene (sox) cluster of various bacteria (Friedrich et al. 2008), where the homologues of cyt c and flavoprotein genes are referred to as $soxE$ and $soxF$ (instead of $fccA$ and $fccB$), respectively. From the facultative lithotrophic bacterium *Paracoccus pantotrophus*, a monomeric flavoprotein

encoded by the *soxF* gene in the sulfur oxidizing *sox* gene cluster was prepared (Rother et al. 2001), and subsequently found to have sulfide dehydrogenase activity in vitro with externally added cyt *c* (from horse heart) as the electron acceptor (Quentmeier et al. 2004). Although the *sox* gene cluster of this bacterium contains *soxE* in tandem with *soxF* encoding the cyt *c* homologue of FCSD (Friedrich et al. 2000), SoxF was obtained as a monomeric flavoprotein devoid of bound cytochrome. For discussion of other possible physiological functions of SoxF, see below.

### Sulfide dehydrogenase activity of thiosulfate oxidizing multi-enzyme system

The thiosulfate oxidizing enzyme system from *C. tepidum* (the core TOMES, see below) has low but measurable sulfide dehydrogenase activity (Ogawa et al. in press) although the affinity for sulfide is lower than the above three kinds of enzymes (Table 1).

### Thiosulfate oxidizing activity of various Bacteria

### Different pathways of thiosulfate oxidation: S4I pathway and TOMES (or Sox) pathway

Two different biochemical pathways for thiosulfate oxidation are distinguishable in the domain *Bacteria* (Kelly et al. 1997; Friedrich et al. 2001, 2005). In one pathway found in bacteria such as the lithoautotrophic proteobacterium *Acidiphilium acidophilum* (formerly *Thiobacillus acidophilus*), thiosulfate is first oxidized by a periplasmic thiosulfate dehydrogenase to tetrathionate (Okuzumi and Kita 1965; Meulenberg et al. 1993) either as the final or intermediary product (the tetrathionate (S4) intermediate pathway or S4I pathway (Kelly et al. 1997):

2S₂O₃²⁻ (thiosulfate) → O₃SSSSO₃⁻ (tetrathionate) + 2e⁻.

Briefly, in *A. acidophilum*, tetrathionate appears to be hydrolyzed by tetrathionate hydrolase to thiosulfate, sulfur, and sulfate (De Jong et al. 1997), and in *Acidithiobacillus ferrooxidans*, the electron acceptor of tetrathionate hydrolase seems to be ferric iron (Sugio et al. 2009). Although *Chlorobium vibrioforme f.* sp. *thiosulfatophilum* cells utilized tetrathionate as the electron donor for CO₂ assimilation, the thiosulfate oxidation in this bacterium did not seem to proceed through S4I pathway, because tetrathionate was not the oxidation product of thiosulfate (Khanna and Nicholas 1982).

In the other thiosulfate oxidation pathway found in bacteria such as the facultative lithotrophic *Paracoccus versutus* (formerly *Thiobacillus versutus*) (Kelly et al. 1997) and *P. pantotrophus* (Friedrich et al. 2001, 2005),

the purple sulfur bacterium *Allochromatium vinosum* (Grimm et al. 2008) and the GSB *Chlorobaculum tepidum* (Ogawa et al. 2008), thiosulfate oxidation is catalyzed by the collaboration of several periplasmic proteins, referred to either as the TOMES (Kelly et al. 1997) or as the sulfur oxidizing system (Sox (Friedrich et al. 2000)).

Some bacteria such as the acidophilic sulfur oxidizing *Starkeya novella* (Kappler et al. 2001) and the purple sulfur bacterium *Allochromatium vinosum* (Hensen et al. 2006) appear to have both pathways.

#### Thiosulfate oxidizing multi-enzyme system (TOMES or Sox)

In bacteria that oxidize thiosulfate by TOMES, its components are found to be largely similar among different phyla, with some variations notably with respect to the presence or absence of sulfur dehydrogenase SoxCD. The TOMES components have been intensively studied in the facultative lithotrophic bacterium *P. versutus* (Kelly et al. 1997), *P. pantotrophus* (Friedrich et al. 2001, 2005), and in purple sulfur bacteria such as *Allochromatium vinosum* (Welte et al. 2009). At a minimum, three proteins (SoxAX, SoxB and SoxYZ) are indispensable to thiosulfate oxidation (Friedrich et al. 2001).

SoxAX is a heterodimeric (heterotrimeric in some bacteria, see below) protein, sometimes called cyt *c*-551 (Kusai and Yamanaka 1973c) which mediates electron transfer to external cytochrome and/or high-potential ISP (Meyer and Cusanovich 2003). In *P. pantotrophus*, SoxA is a di-heme (mono-heme in some bacteria, see below) and SoxX is a mono-heme subunit (Friedrich et al. 2000). SoxYZ is a colorless heterodimeric protein that binds the oxidized product of thiosulfate on the cysteinyl-SH group of SoxY as the intermediate to form a cysteinyl S-thiosulfonate (Quentmeier and Friedrich 2001). SoxB (also called Enzyme B (Kelly et al. 1997)) is a monomeric dimanganese protein (Cammack et al. 1989; Epel et al. 2005) that hydrolyzes the cysteinyl S-thiosulfonate to cysteinyl persulfide and sulfate (thiosulfohydrolase). Recently, the SoxB 3-D structure was determined by X-ray crystallography (Saue et al. 2009). SoxCD is an α2β2 heterotetramer (SoxC: binding molybdenum cofactor, SoxD: di-heme cyt *c*), which oxidatively hydrolyzes the cysteinyl persulfide.

In bacteria such as *P. pantotrophus* that have SoxCD, the following reaction scheme was proposed (Friedrich et al. 2001):

(1)
SoxY(–SH) + SSO₃⁻ → SoxY(–S) – SSO₃⁻ + 2e⁻ + H⁺,

catalyzed by SoxAX;

(2)
SoxY(–S) – SSO₃⁻ + H₂O → SoxY(–S) – SH + SO₄²⁻ + H⁺,

catalyzed by SoxB;
SoxY(−S) − SH + 3H₂O → SoxY(−S)SO₃⁻ + 6e⁻ + 7H⁺,
(3)
catalyzed by SoxCD; and
SoxY(−S)SO₃⁻ + H₂O → SoxY(−SH) + SO₄²⁻ + H⁺
(4)
catalyzed by SoxB.

Experimental systems using reconstituted TOMES containing SoxCD yield eight electrons and two sulfate per thiosulfate.

In bacteria lacking SoxCD such as GSB, the reaction proceeds according to Eqs. 1 and 2 to yield SoxY(−S)−SH and sulfate yielding two electrons and one sulfate, and this reaction cycle is believed to be repeated multiple times to yield a poly-sulfide group (the S⁰ equivalent) on SoxY.

**Thiosulfate oxidizing multi-enzyme system (TOMES) of GSB**

In 2002, the whole genome sequence of *C. tepidum* was determined (Eisen et al. 2002) and the sequence of the sox gene cluster of *C. limicola* was reported (Verté et al. 2002). It became apparent that the thiosulfate oxidizing proteins of GSB would be largely similar to those of *P. pantotrophus* (Friedrich et al. 2001). Genome sequencing of GSB continued, and Frigaard and Bryant (2008a, b) deduced the inorganic sulfur metabolic pathways from genome data of 10 strains available from NCBI GenBank and the two draft genome sequences. They concluded that most strains are able to oxidize sulfide, while oxidation of thiosulfate is less commonly encountered in cultivated strains.

In GSB, the biochemical studies of thiosulfate oxidation revealed several proteins by the mid 1970s. These include a multi-heme cyt c-551 (Meyer et al. 1968; Kusai and Yamanaka 1973c) and “thiosulfate-multiheme cyt c reductase” (Kusai and Yamanaka 1973c). Although these studies elucidated various important components of the thiosulfate oxidizing pathway of GSB (reviewed in Yamana 1996; Meyer and Cusanovich 2003), there was evidence that the biochemical pathway was far from complete.

The proteins essential for thiosulfate oxidation in *C. tepidum* were recently established by biochemical studies (Table 2) (Ogawa et al. 2008). These were composed of three periplasmic proteins (see below) very similar to those of *P. pantotrophus* except that SoxCD was absent. The three proteins will be referred to as the core TOMES, in order to distinguish it from the *P. pantotrophus*-type TOMES that contains SoxCD. Among the new findings was evidence of the presence of an additional subunit SoxK (SAXB) in the SoxAX complex. The presence of *soxK* in the *sox* gene cluster of various GSB had been noted in comparative genomic analyses (Frigaard and Bryant 2008a, b), but the function of the encoded protein was unknown (see below).

SoxB is an almost colorless monomeric protein with significant deduced amino acid sequence identity to *P. pantotrophus* SoxB and appears to be a di-manganese protein (Cammack et al. 1989).

SoxYZ is a colorless dimeric protein, and appears to be able to bind the thiosulfonyl residue as the oxidized product of thiosulfate on the conserved cysteinyl SH of SoxY (Quentmeier and Friedrich 2001).

SoxAXK is a heterotrimeric multi-heme protein, reddish brown in color. SoxA is an extremely low potential mono-heme cyt c-551 (*E₀′ < −550 mV*) in *C. tepidum*. SoxA is a mono-heme protein in *S. novella* (Kappler et al. 2004, 2005) as in *C. tepidum*, but a di-heme protein in *P. pantotrophus* (Friedrich et al. 2000; Dambe et al. 2005) and *Rhodovulum sulfidophilum* (Appia-Ayme et al. 2001; Bamford et al. 2002). *C. tepidum* SoxX is a mono-heme cyt c-551 with a much higher *E₀′* value of +161 mV (Ogawa et al. 2008). SoxK is a newly identified subunit apparently devoid of prosthetic groups. The recombinant SoxK (rSoxK, over-expressed in *Escherichia coli* cells and purified) enhanced the binding of rSoxA and rSoxx as evidenced by gel-permeation chromatography, hence it is referred to as SAXB (SoxAX binding protein). In the presence of SoxB and SoxYZ, the mixture of rSoxA and rSoxx showed a low but detectable thiosulfate oxidation activity, which was significantly enhanced by the further addition of rSoxx to a level almost equal to the level of the

**Table 2 Sox proteins of *C. tepidum***

| Sox protein                     | Subunit | Prosthetic group etc. | Mrᵇ (kDa) | Signal peptide |
|----------------------------------|---------|------------------------|------------|----------------|
| SoxAXK (cyt c-551)              | SoxA    | Mono-heme cyt c       | 30         | +              |
|                                  | SoxX    | Mono-heme cyt c       | 11         | +              |
|                                  | SoxK    |                        | 9          | +              |
| SoxB (thiosulfhydrolase)        | Monomer | Di-manganese          | 62         | +              |
| SoxYZ (thiosulfonate and polysulfane-binding protein) | SoxY |            | 13            | +              |
|                                  | SoxZ    |                        | 9          | no             |
| SoxFᵃ                          | Monomer | FAD                   | 44         | +              |

ᵃ From Ogawa et al. in press.  
All other data are from Ogawa et al. (2008)  
ᵇ Calculated from deduced mature polypeptides and prosthetic groups

mixture containing the native SoxAXK. The *soxK* homologues are found in the *sox* gene clusters of not only GSB, but also in those of a number of other families of bacteria including Allochromatium vinosum, Bradyrhizobium japonicum, etc. (in about one-third of the *sox* gene clusters of the bacteria selected by Friedrich et al. (2008) for comparative studies) (Fig. 2). The unrooted phylogenetic tree indicated that the SoxKs of the GSB are congruent (Ogawa et al. 2008).

In the reconstituted in vitro reaction, the core TOMES of *C. tepidum* oxidized thiosulfate with various cyt *c* (including horse heart cyt *c*) as the electron acceptors (Ogawa et al. 2008) as reported with a partially purified enzyme system from Chlorobium limicola (Kusai and Yamanaka 1973c). The redox potentials of the core TOMES components and some inorganic sulfur compounds are listed in Table 3.

A reaction model of thiosulfate oxidation by TOMES was proposed by Friedrich et al. (2001) with the initial steps outlined as follows: in the presence of SoxYZ,

SoxAX (the protein of this bacterium does not bind SoxK) initiates the oxidation resulting in covalent attachment of oxidized thiosulfate to a conserved cysteine of SoxY (Eq. 1 in “Thiosulfate oxidizing multienzyme system (TOMES or SOX)” section). SoxB would then hydrolytically release sulfate leaving the sulfane sulfur atom of thiosulfate on SoxY to yield persulfide (Eq. 2). In contrast to this scheme, we found that all three core TOMES components namely SoxAXK, SoxB, and SoxYZ were absolutely necessary for initiation of thiosulfate oxidation, and that no reduction of cytochromes (even as low as the stoichiometric amount of SoxYZ and SoxAXK) occurred in the absence of SoxB, indicating that participation of SoxB would also be required in the initial reaction step (Ogawa et al. in press).

The core TOMES also has sulfide dehydrogenase activity (see “Sulfide dehydrogenase activity of thiosulfate oxidizing multi-enzyme system” section), and the participation of all three core TOMES components was required for this activity (Ogawa et al. in press).


### Maps of the *sox* gene cluster


CT1015                     CT1020
F   X   Y   Z   A          K       B       W      F-like        fccA    fccB
                                                                                   C. tepidum (AE006470)
F   X   Y   Z   A     orf106       B       W
                                                                                   fccA    fccB
                                                                                   C. limicola
                                                                                   f. thiosulfatophilum (AY074395)
F-like
F                           X   Y   Z   A       B       W      fccB    fccA
                                                                                   C. chlorochromatii (CP000108)
F   X   Y   Z   A          B       W      F-like
                                                                                   P. vibrioformis (CP000607)
PphaDRAFT_0456
F   X   Y   Z   A          B       W      Y   fccA    fccB
                                                                                   P. phaeoclathratiforme (CP000607)
MED92_13493
C   D   X   Y   Z   A      B
                                                                                   N. caesariensis (AAOW00000000)
B   X   A   ORF9
Z   Y
                                                                                   A. vinosum (DQ441405)
F   F   E   D   C          B       Z Y2    A       X       W       V
                                                                                   R. palustris (BX571963)
F   D   C                      B       Z   Y       A       X       W   V
                                                                                   S. novella (AF139113)
R   S   V   W   X   Y   Z   A       B       C       D       E       F       G       H
                                                                                   P. pantotrophus (X79242)
V   W   X   Y   Z   A       B       C       D       E       F
                                                                                   R. sulfidophilum (AY005800)


---

Fig. 2 Map of the *sox* gene cluster. Black arrows: SAXB gene (*soxK*) homologs. Striped arrows and genes are; vertically: *soxA*, obliquely: *soxB*, horizontally: *soxF*, both vertically and horizontally: *soxE*. Other hypothetical *sox* genes are shown in light gray. The sources used were (organism (GenBank nucleotide sequence accession number), accessed September 1, 2009): Allochromatium vinosum (DQ441405), Chlorobium chlorochromatii CaD3 (CP000108), Chlorobium limicola f. thiosulfatophilum (AY074395), Chlorobium tepidum TLS (AE006470), *N. caesariensis* (AAOW00000000), *Paracoccus pantotrophus* GB17 (X79242PDSOX) (EMBL data library accession number), accessed 1 September 2009), *Pelodictyon phaeoclathratiforme* BU-1 (AAIK00000000), *Prosthecochloris vibrioformis* DSM 265 (CP000607), *Rhodopseudomonas palustris* CGA 009 (BX571963), *Rhodovulum sulfidophilum* (AY005800), *Starkeya novella* DSMZ 506T (AF139113)

Table 3 Redox potentials of sulfur compounds and C. tepidum proteins

| $E_{0}^{\prime}$ (mV) |
| --- |
| Inorganic sulfur compounds |
| $\mathrm{SO}_{4}^{2-} / \mathrm{SO}_{3}^{-}$ | $-515^{\mathrm{d}}$ |
| $\mathrm{S}+2 \mathrm{H}^{+} / \mathrm{H}_{2} \mathrm{~S}$ | $-274^{\mathrm{e}}$ |
| $2 \mathrm{SO}_{4}^{2-} / \mathrm{S}_{2} \mathrm{O}_{3}^{2-}$ | $-245^{\mathrm{d}}$ |
| $\mathrm{SO}_{4}^{2-} / \mathrm{S}^{0}$ | $-200^{\mathrm{d}}$ |
| Bacillus PS3 membrane |
| Menaquinone | $-60^{\mathrm{f}}$ |
| C. tepidum |
| SoxAXK cyt c-551 | $+161$ and lower than $-550^{\mathrm{g}}$ |
| rSoxX${ }^{\mathrm{a}}$ | $+153^{\mathrm{g}}$ |
| rSoxA${ }^{\mathrm{b}}$ | lower than $-550^{\mathrm{g}}$ |
| cyt c-554 | $+148^{\mathrm{h}}$ |
| cyt c-551 (PsaC) | $+180^{\mathrm{i}}$ |
| cyt c-551 (PsaC) | $+177^{\mathrm{c}, \mathrm{j}}$ |
| P840 | $+230^{\mathrm{j}}$ |

---

a Recombinant SoxX  
b Recombinant SoxA  
c 53 mV lower than P840 $(230-53=177 \mathrm{mV})$  
d Brune 1995b  
e Japanese Biochemical Society 1984  
f Liebl et al. 1992  
g Ogawa et al. 2008  
h Itoh et al. 2002  
i Okumura et al. 1994  
j Kusumoto et al. 1999  

The monomeric flavoprotein encoded by the soxF

In many thiosulfate oxidizing bacteria, the genes encoding TOMES proteins generally occur in either a single or multiple cluster(s) (the sox gene cluster(s)). The genes encoding SoxAX, SoxB, and SoxYZ (the core TOMES) are invariably present in these bacteria, and the encoded proteins share relatively high amino acid sequence identity with each other irrespective of the presence or absence of soxCD (Friedrich et al. 2008). However, there are some differences among thiosulfate oxidizing bacteria with respect to the presence or absence of the genes soxE and soxF in addition to soxK, soxC, and soxD as described above. In P. pantotrophus, the soxF and soxE genes (the homologues of Allochromatium vinosum fccB and fccA genes, respectively (Reinartz et al. 1998)), are found in the sox gene cluster (Friedrich et al. 2000), where they encode a flavoprotein and cytochrome homologue, respectively, of the dimeric flavocytochrome c (FCSD) complex that acts as a sulfide dehydrogenase (see “Flavocytochrome c-sulfide dehydrogenase (FCSD or FCC)” section). In the thiosulfate oxidizing bacterial species selected by Friedrich et al. (2008) for comparative studies, the majority of species have one or sometimes two soxF homolog(s) within the sox gene cluster with roughly half of them devoid of soxE. In a few species, the soxF gene is absent from the sox gene cluster.

The bacterium P. pantotrophus has both soxF and soxE genes in the sox gene cluster, but SoxF protein “as isolated” from the cells occurred as a monomer (Rother et al. 2001) and catalyzed sulfide oxidation with horse heart cyt c in the absence of SoxE (Quentmeier et al. 2004). A soxF deletion mutant of P. pantotrophus was shown to have a slower growth rate than wild-type cells grown on thiosulfate and mutant cells oxidized thiosulfate at a rate of 40% of the wild-type cells (Bardischewsky et al. 2006). The rate of oxidation of thiosulfate in crude cell-free extracts prepared from the mutant cells using horse heart cyt c as the electron acceptor, was significantly increased when supplemented with SoxF isolated from wild-type cells. However, when SoxF was added to the reconstituted system composed of four purified sox proteins (the three core TOMES (SoxAX, SoxB, SoxYZ), and SoxCD), the thiosulfate oxidizing activity was not significantly increased. From these results, the authors speculated that SoxF acts on some component or under certain conditions present only in whole cells and crude cell-free extracts, but not in the purified reconstituted system. The biochemical basis of the stimulating effects of SoxF in the crude extract and the possible functions of SoxF in thiosulfate oxidation in P. pantotrophus remain to be investigated.

The sox gene cluster of C. tepidum contains soxF (also called soxJ; Frigaard and Bryant 2008a, b), soxF2 in C. limicola f. thiosulfatophilum (Verté et al. 2002), and fccB in Allochromatium vinosum Reinartz et al. 1998), but soxE is absent (Fig. 2). In C. tepidum, fccA and fccB encoding FCSD are found in tandem about 1 Mb away from the sox gene cluster. From C. tepidum cells, a monomeric flavoprotein SoxF that has sulfide dehydrogenase activity was purified to homogeneity (Ogawa et al. in press). In the presence of the core TOMES, SoxF stimulated the reduction of cyt c-554 from this bacterium by about two-fold, indicating a possible physiological function of SoxF. The degree of stimulation by SoxF was low when horse heart cyt c and yeast cyt c were used as the electron acceptors (Ogawa et al. in press), indicating the importance of the species of electron acceptor used for the in vitro assays and a need for a reexamination of the SoxF assay system in other bacteria including P. pantotrophus.

In P. pantotrophus, SoxYZ has been isolated mostly in an inactivated state (Quentmeier et al. 2007; Quentmeier et al. 2008) leading to recent speculation that it is activated by SoxF in vivo suggesting an additional possible physiological function of SoxF (Friedrich et al. 2008).

Sulfite dehydrogenase activity

Although GSB are known to be unable to grow on sulfite as the electron donor (Brune 1989, 1995b), the purified periplasmic core TOMES from *C. tepidum* catalyzed thiosulfate oxidation with externally added cyt c-554 as the electron acceptor. However, this activity was rather quickly inactivated in a few minutes at sulfite concentrations higher than 0.5 mM in vitro (Ogawa et al. in press), and this may be the reason why GSB apparently cannot grow on externally added sulfite as the sole electron donor. Theoretical calculations show that sulfite readily condenses with some thiols (RSH) to form RSSO₂⁻ (Steudel and Steudel 2010), and this may be the cause of the observed inactivation of TOMES by sulfite.

As will be described below (“Disimilatory sulfite reductase (DSR)” section), elemental sulfur appears to be oxidized to sulfite in the cytoplasm by the dissimilatory sulfite reductase (DSR) system (Dahl 2008).

### Utilization of elemental sulfur

#### Transient elemental sulfur formation

Many GSB can use elemental sulfur, and they transiently deposit elemental sulfur outside the cells in the course of oxidation of sulfide and thiosulfate, which is finally oxidized to sulfate. Oxidation of the latters in the periplasmic space would yield polysulfides (including persulfide HSSH, the S⁰ equivalent, see Fig. 1), which are then converted to elemental sulfur. In contrast, purple sulfur bacteria transiently deposit elemental sulfur (or organic polysulfides) in the periplasmic space as globules surrounded by the proteins called Spgs (Pattaragulwanit et al. 1998). Three Spg proteins were identified from *Allochromatium vinosum* and two from *T. roseopersicina* (Brune 1995a; Grimm et al. 2008).

Results of in vitro experiments with the reconstituted core TOMES from *C. tepidum* showed that the stoichiometric amounts of cytochrome reduced were about two for each molecule of thiosulfate, sulfide or sulfite oxidized (Ogawa et al. in press). The ratio was also about two with SoxF-catalyzed sulfide oxidation. In these experiments, the amount of cytochrome reduced usually far exceeded (by at least eight-fold) the concentrations of the core TOMES components or SoxF2. This was also true with the sulfide dehydrogenase catalyzed by FCSD (Kusai and Yamanaka 1973b).

These results indicate that in GSB, the reconstituted TOMES yields two electrons and one sulfate per thiosulfate leaving a sulfane sulfur on SoxYZ in the periplasmic space, and this reaction cycle is repeated multiple times to yield the S⁰ equivalent in a polysulfide group. The sulfide oxidation catalyzed by the monomeric SoxF and the dimeric FCSD appears to initially yield the S⁰ equivalent in a polysulfide, which appears to be subsequently converted to elemental sulfur (S⁰).

In inorganic chemistry, sulfanes (H₂Sₙ) (polysulfide) are known to be rather unstable compounds, and freshly prepared sulfane oil is a mixture of H₂S₄, H₂S₅, H₂S₆, H₂S₃ (in this order with decreasing concentrations) with the higher sulfanes up to H₂S₂₅ and trace amounts of di- (H₂S₂), mono-sulfane (H₂S), etc. (Steudel 2003). Upon aging of the sulfane oil, in addition to the interchange of S⁰, the following reactions take place:

2H₂Sₙ → H₂Sₙ₊ₓ + H₂Sₙ₋ₓ,

and

H₂Sₙ → H₂Sₙ₋₈ + S₈ (elemental sulfur).

Because of this property of inorganic sulfane, it is conceivable that the polysulfide group bound to SoxY(SH) (or subsequently transferred to some thiol, RSH) will spontaneously liberate elemental sulfur in GSB:

SoxY(S) – SₙH → SoxY(S) – Sₙ₋₈H + S₈

or

SoxY(S) – SₙH + RSH → SoxY(S) – Sₙ₋ₘH + RSₘ₊₁H.

Alternatively, an as of yet unidentified factor(s) (either protein(s) or small molecule thiol(s)) may participate in the reaction.

The reactions of subsequent sulfur globule utilization

As in other bacteria, the biochemistry of the metabolic pathways of elemental sulfur is one of the least understood parts of inorganic sulfur metabolism in GSB, although several models with involvement of possible protein(s) have been proposed based on comparative genomics and mutant analyses (in GSB: Eisen et al. 2002; Chan et al. 2008a, c; Frigaard and Bryant 2008a, b, and in purple sulfur bacteria: Sander and Dahl 2009).

In the following discussion, we consider some of the possible candidate proteins in GSB.

### Dissimilatory sulfite reductase (DSR)

The reverse dissimilatory sulfite reductase (DsrAB) of the purple sulfur bacterium *Allochromatium vinosum* (formerly *Chromatium vinosum*) is encoded together with 13 other proteins in the *dsr* operon (Dahl et al. 2005; Sander and Dahl 2009). Polar insertional mutations immediately upstream of *dsrA*, and in *dsrB*, *dsrH* and *dsrM*, were found to lead to an inability of the mutants to oxidize stored

sulfur S⁰ (Pott and Dahl 1998). These genes were found to be not essential for sulfide or thiosulfate oxidation, implying that the enzyme works in the reverse direction in the oxidation of S⁰ to sulfite. The Dsr proteins are either cytoplasmic or inner membrane-bound, indicating that the extracellular or periplasmic S⁰ equivalent has to be transported across the cytoplasmic membrane. Reaction models of cytoplasmic sulfide oxidation involving a dissimilatory sulfite reductase DsrAB, a membrane-bound quinol oxidase-like DsrM, and ISP DsrK were presented (Pott and Dahl 1998), followed by more refined models (Grimm et al. 2008; Dahl 2008; Sander and Dahl 2009).

Similar *dsr* genes were found in the genome of *C. tepidum* (Eisen et al. 2002), and subsequently in all the genomes of GSB opened to the public thus far except *C. ferrooxidans* (Frigaard and Bryant 2008a, b), strongly suggesting that the Dsr system also works in GSB by oxidizing the S⁰ equivalent to sulfite. A model of complete S⁰ oxidation in GSB with participation of several Dsr proteins and several other proteins including some hypothetical ones has been proposed (Frigaard and Bryant 2008a).

The proposed biochemical pathway suggested by comparative genomic analyses leading to complete oxidation of intracellular sulfite to sulfate (in the reverse direction for those of Archaea) for *C. tepidum* is: adenosine phosphosulfate (APS) formation by APS reductase coupled to quinone reduction by Qmo (see below), and formation of sulfate from APS (possibly: APS + pyrophosphate → ATP + sulfate). For a further discussion of the possible metabolic pathways of sulfur oxidation in the cytoplasm involving Dsr, see Sander and Dahl (2009) (see also Fig. 1). The cytoplasmic sulfite oxidation pathway appears to slightly differ among GSB strains because the *qmo* genes are apparently absent in several GSB strains (Frigaard and Bryant 2008a; Frigaard and Dahl 2009).

### Transmembrane transfers of the S⁰ equivalents: several candidates and SoxVW homologues

Given that the Dsr system also functions in GSB, it follows that for intracellular oxidation of the extracellular elemental sulfur, the S⁰ equivalent must be supplied to the cytoplasmic space by some mechanism. It is unclear in what form(s) the elemental sulfur is transformed, and how it crosses the outer membrane, the periplasmic space, and the inner membrane in GSB. From comparative genomic analyses, Frigaard and Bryant (2008a, b) proposed the following candidates for the transmembrane transport of S⁰: polysulfide-reductase-like complexes PsrABC (a molybdopterin-containing PsrA, a [4Fe-4S]-cluster-binding PsrB, and a membrane-anchoring PsrC), sulfhydrogenase-like complex (CT0494, CT0495, and CT0496) and heterodisulfide-reductase-like complexes (encoded by CT1891-CY1894).

In the following, we discuss some additional candidates for the transmembrane transport of S⁰. In *E. coli*, the protein DsbD (thiol:disulfide interchange protein) transfers disulfide across the inner membrane to the periplasmic space and is required for maturation of periplasmic cytochromes (thioether bond formation) (Missiakas et al. 1995; for a review, see Messens and Collet 2006).

The *sox* gene cluster of *P. pantotrophus* contains *soxVW*, which if mutated renders the cells unable to grow on thiosulfate (Bardischewsky et al. 2006). SoxW is a periplasmic protein containing a thioredoxin motif. SoxV is a homologue of DsbD, and is deduced to be a cytoplasmic membrane protein that has a cysteinyI group which transfers reducing equivalents from cytoplasmic thioredoxin to the periplasm, reducing target proteins such as SoxW in the periplasm (heterodisulfide reductase). From analyses of the *soxV* and *soxW* mutants of the purple bacterium *R. sulpiphilum*, it was found that SoxV, but not SoxW was essential to lithoautotrophic growth of this bacterium with thiosulfate (Appia-Ayme and Berks 2002). Furthermore, it was found that SoxV was required for maintaining SoxW in the reduced state in the presence of thiosulfate. Similar results were obtained with *sox* mutants of *P. pantotrophus*, and in addition, the TOMES protein levels of the mutants were found to be unaffected by the *soxV* mutation, indicating that the *soxV-minus* phenotype is not due to a defect in the maturation of cytochromes (Bardischewsky et al. 2006).

In GSB that use thiosulfate, *soxW* is invariably present in the *sox* gene cluster, but *soxV* is apparently absent from the cluster. With *E. coli* DsbD [AP_004637] as the query protein sequence, we found CT1075 from the *C. tepidum* genome with an E-value of 9e−66 from a BLASTP search result and that the gene was annotated as “thiol:disulfide interchange protein DsbD” by the KEGG internet site ([http://www.kegg.jp/ja/](http://www.kegg.jp/ja/), accessed January 14, 2010). All 12 of the publicly available whole genome sequences from GSB strains have a gene encoding a CT1075 homologue (at a minimum of 57% amino acid sequence identity). In GSB, the involvement of SoxW in the periplasm and DsbD in transferring the S⁰ equivalent across the inner membrane from periplasm to cytosol (in the reverse direction that occurs in *E. coli* cells) seems to be a possibility (Fig. 1). Another possibility is that CT1075 carries S⁰ (in the form of di-thiol) outside as in *E. coli*, because *Chlorobium ferrooxidans* has a CT1075 homologue in its genome and is unable to grow when inorganic sulfur compounds are the electron donors.

In the vicinity of CT1075, there are several genes encoding putative proteins related to sulfur metabolism; CT1072; thiol-disulfide interchange protein DsbE (also called CcmG), CT1074; ApbE family protein, and CT1078 (the gene, in the reverse orientation); lipoic acid synthetase

LipA. In *P. pantotrophus*, CcmG (DsbE) is a periplasmic thioredoxin required for *c*- and *aa₃*-type cytochrome biogenesis (Page and Ferguson 1997; Fabianek et al. 1998). In *Salmonella*, ApbE is reported to be a lipoprotein involved in thiamine synthesis, and the lack of ApbC or ApbE results in a defect in the Fe–S cluster metabolism (Beck and Downs 1998). LipA is assumed to be involved in sulfide donation in lipoate biosynthesis (Miller et al. 2000).

**Qmo, a putative quinone-interacting membrane-bound heterodisulfide reductase complex**

The outcome of S⁰ oxidation seems to differ among GSB. In several strains including *C. tepidum*, the genes encoding the homolog of Qmo of the sulfate reducing Archaea were found from comparative genome analyses (Frigaard and Bryant 2008a, b). Chan et al. (2008c) created a *C. tepidum* mutant by deleting the region CT868-876 including one of the putative *qmo* genes, and found that the mutant was completely defective for growth on thiosulfate as the sole electron donor, but only slightly defective for growth on sulfide or thiosulfate plus sulfide. Based on their results, they suggested that CT0872 of previously unknown function is the most likely candidate for the thiosulfate oxidation phenotype observed in their mutant strain. Lending credence to this hypothesis is the conservation of CT0872 in the chemolithotrophic sulfur oxidizing bacteria and Archaea.

### Miscellaneous proteins possibly involved in inorganic sulfur metabolism

From comparative genome sequence analyses, in addition to the above described proteins, potential involvement of the following proteins in some part of sulfur metabolism have been discussed: rhodanese-like protein, SoyYZ (SoxYZ-related protein), Rubisco-like protein, etc. (for details, see Frigaard and Bryant 2008a, b; Frigaard and Dahl 2009).

### Creation of mutants by targeted inactivation of genes involved in sulfur metabolism

Chan et al. (2008a, b) pointed out the value of targeted mutant studies, particularly when there appears to be genetically redundant enzymes encoded within the genome. Chan et al. (2009) created a *C. tepidum* mutant by disrupting two putative SQR genes, and found that its SQR activity was completely abolished and that the mutant could still grow, although more slowly than the wild-type on sulfide as the sole electron donor. They found that transcripts of *CT208l* (*fccB*) encoding the flavoprotein of

FCSD and of *soxF* (*CT1015* or *soxJ*) were detected at all stages of growth on sulfide, but failed to detect any sulfide dehydrogenase activity in vitro in the soluble or membrane fraction with horse heart cyt *c* as the electron acceptor. They speculated that the failure to detect any sulfide dehydrogenase activity might be ascribed to the fact that horse heart cyt *c* is not a suitable acceptor for FCSD of this bacterium. This might be the case as monomeric SoxF has sulfide dehydrogenase activity with a high affinity for sulfide, and cyt *c*-554 from this bacterium is by far a better electron acceptor than horse heart cyt *c* (Ogawa et al. in press).

Azai et al. (2009) reported that a mutant of *C. tepidum* devoid of the gene encoding cyt *c*-554 (a good electron acceptor in the reconstituted TOMES (Ogawa et al. 2008)) could grow on thiosulfate at slower rates than the wild type and that a double mutant devoid of both cyt *c*-554 and SoxB could not grow on thiosulfate. Their results indicate that SoxB is indispensable to in vivo oxidation of thiosulfate, and that in the absence of cyt *c*-554, some factor(s) mediates electron transport, although at much slower rate, between the core TOMES and RC-bound cyt *c*-551 that serves as an immediate electron donor to the oxidized primary donor P840⁺. Because a variety of *c*-type cytochromes can accept electrons from the core TOMES (Kusai and Yamanaka 1973a, c, Ogawa et al. 2008, in press), it seems likely that some cytochrome(s) could mediate the electron transport in the mutant cells.

### Future studies

There are various observations that have yet to be elucidated biochemically, and some of them are as follows.

- The reviews of Brune (1989, 1995b) inferred from reports that some GCB could use tetrathionate and perform a photochemical disproportionation of S⁰ into H₂S and S₂O₃²⁻ probably by the reaction scheme:  
  3S + 3H₂O → H₂S + S₂O₃²⁻ + 4H⁺.
- Photosynthetically growing cells of *Chlorobium limicola f. thiosulfatophylum* (DSM 249) were reported to transiently form thiosulfate from sulfide oxidation and to have disproportionation activity of S⁰ into H₂S and S₂O₃²⁻ (Trüper et al. 1988).
- The mutants of *C. tepidum* created by disrupting one or both of the two putative SQR genes showed differences in susceptibility to high concentrations of sulfide (Chan et al. 2009).

### Acknowledgment

We thank M. Kitashima for comparative genomic analyses and preparation of the figures. This work was supported in part by the Global COE Program (Integrative Life Science Based

on the Study of Biosignaling Mechanisms) and by the High-tech
Research Center project, MEXT, Japan to KI.

**References**

Appia-Ayme C, Berks BC (2002) SoxV, an orthogolue of the CcdA
disulfide transporter, is involved in thiosulfate oxidation in
*Rhodovulum sulfidophilum* and reduces the periplasmic thiore-
doxin SoxW. Biochem Biophys Res Commun 296:737–741

Appia-Ayme C, Little PJ, Matsumoto Y, Leech AP, Berks BC (2001)
Cytochrome complex essential for photosynthetic oxidation of
both thiosulfate and sulfide in *Rhodovulum sulfidophilum*.
J Bacteriol 183:6107–6118

Arieli B, Shahak Y, Taglicht D, Hauska G, Padan E (1994)
Purification and characterization of sulfide-quinone reductase, a
novel enzyme driving anoxygenic photosynthesis in *Oscillatoria
limnetica*. J Biol Chem 269:5705–5711

Azai C, Tsukatani Y, Harada J, Oh-oka H (2009) Sulfur oxidation in
mutants of the photosynthetic green sulfur bacterium *Chlorobi-
um tepidum* devoid of cytochrome c-554 and SoxB. Photosynth
Res 100:57–65

Bamford VA, Bruno A, Rasmussen T, Appia-Ayme C, Cheesman
MR, Berks BC, Hemmings AM (2002) Structural basis for the
oxidation of thiosulfate by a sulfur cycle enzyme. EMBO J
21:5599–5610

Bardischewsky F, Quentmeier A, Friedrich CG (2006) The flavo-
protein SoxF functions in chemotrophic thiosulfate oxidation of
*Paracoccus pantotrophus* in vivo and in vitro. FEMS Microbiol
Lett 258:121–126

Beck BJ, Downs DM (1998) The *apbE* gene encodes a lipoprotein
involved in thiamine synthesis in *Salmonella typhimurium*.
J Bacteriol 180:885–891

Brune DC (1989) Sulfur oxidation by phototrophic bacteria. Biochim
Biophys Acta 975:189–221

Brune DC (1995a) Isolation and characterization of sulfur globule
proteins from *Chromatium vinosum* and *Thiocapsa roseopersi-
cina*. Arch Microbiol 163:391–399

Brune DC (1995b) Sulfur compounds as photosynthetic electron
donors. In: Blankenship RE, Madigan MT, Bauer CE (eds)
Anoxygenic photosynthetic bacteria. Kluwer, Amsterdam, pp
847–870

Cammack R, Chapman A, Lu WP, Karagouni A, Kelly DP (1989)
Evidence that protein B of the thiosulphate-oxidizing system of
*Thiobacillus versutus* contains a binuclear manganese cluster.
FEBS Lett 253:239–243

Chan LK, Morgan-Kiss RM, Hanson TE (2008a) Sulfur oxidation in
*Chlorobium tepidum* (syn. *Chlorobaculum tepidum*): genetic and
proteomic analyses. In: Dahl C, Friedrich CG (eds) Microbial
sulfur metabolism. Springer, Berlin, pp 117–126

Chan LK, Morgan-Kiss RM, Hanson TE (2008b) Genetic and
proteomic studies of sulfur oxidation in *Chlorobium tepidum*
(syn. *Chlorobaculum tepidum*). In: Hell R, Dahl C, Knaff DB,
Leustek T (eds) Sulfur metabolism in phototrophic organisms.
Springer, Berlin, pp 357–373

Chan LK, Weber TS, Morgan-Kiss RM, Hanson TE (2008c) A
genomic region required for phototrophic thiosulfate oxidation in
the green sulfur bacterium *Chlorobium tepidum* (syn. *Chloro-
baculum tepidum*). Microbiology 154:818–829

Chan LK, Morgan-Kiss RM, Hanson TE (2009) Functional analysis
of three sulfide:quinone oxidoreductase homologs in *Chloro-
baculum tepidum*. J Bacteriol 191:1026–1034

Cort JR, Selan U, Schulte A, Grimm F, Kennedy MA, Dahl C (2008)
Allochromatium vinosum DsrC: Solution-state NMR structure,
redox properties, and interaction with DsrEFH, a protein

essential for purple sulfur bacterial sulfur oxidation. J Mol Biol
382:692–707

Dahl C (2008) Inorganic sulfur compounds as electron donors in
purple sulfur bacteria. In: Hell R, Dahl C, Knaff DB, Leustek T
(eds) Sulfur metabolism in phototrophic organisms. Springer,
Berlin, pp 289–317

Dahl C, Friedrich CG (eds) (2008) Microbial sulfur metabolism.
Springer, Heidelberg

Dahl C, Engels S, Pott-Sperling AS, Schulte A, Sander J, Lübbe Y,
Deuster O, Brune DC (2005) Novel genes of the *dsr* gene cluster
and evidence for close interaction of Dsr proteins during sulfur
oxidation in the phototrophic sulfur bacterium *Allochromatium
vinosum*. J Bacteriol 187:1392–1404

Dambe T, Quentmeier A, Rother D, Friedrich C, Scheidig AJ (2005)
Structure of the cytochrome complex SoxXA of *Paracoccus
pantotrophus*, a heme enzyme initiating chemotrophic sulfur
oxidation. J Struct Biol 152:229–234

Davidson MW, Gray GO, Knaff DB (1985) Interaction of *Chroma-
tium vinosum* flavocytochrome c-552 with cytochromes studied
by affinity-chromatography. FEBS Lett 187:155–159

De Jong GAH, Hazeu W, Bos P, Kuenen G (1997) Isolation of the
tetrathionate hydrolase from *Thiobacillus acidophilus*. Eur J
Biochem 243:678–683

Eisen JA, Nelson KE, Paulsen IT, Heidelberg JF, Wu M, Dodson RJ,
Deboy R, Gwinn ML, Nelson WC, Haft DH, Hickey EK, Peterson
JD, Durkin AS, Kolonay JL, Yang F, Holt I, Umayam LA, Mason
T, Brenner M, Shea TP, Parksey D, Nierman WC, Feldblyum TV,
Hansen CL, Craven MB, Radune D, Vamathevan J, Khouri H,
White O, Gruber TM, Ketchum KA, Venter JC, Tettelin H, Bryant
DA, Fraser CM (2002) The complete genome sequence of
*Chlorobium tepidum* TLS, a photosynthetic, anaerobic, green-
sulfur bacterium. Proc Natl Acad Sci USA 99:9509–9514

Epel B, Schäfer KO, Quentmeier A, Friedrich C, Lubitz W (2005)
Multifrequency EPR analysis of the dimanganese cluster of the
putative sulfate thiohydrolase SoxB of *Paracoccus pantotro-
phus*. J Biol Inorg Chem 10:636–642

Fabianek RA, Hennecke H, Thony-Meyer L (1998) The active-site
cysteines of the periplasmic thioredoxin-like protein CcmG of
*Escherichia coli* are important but not essential for cytochrome c
maturation in vivo. J Bacteriol 180:1947–1950

Friedrich CG, Quentmeier A, Bardischewsky F, Rother D, Kraft R,
Kostka S, Prinz H (2000) Novel genes for lithotrophic sulfur
oxidation of *Paracoccus pantotrophus* GB17. J Bacteriol
182:4677–4687

Friedrich CG, Rother D, Bardischewsky F, Quentmeier A, Fischer J
(2001) Oxidation of reduced inorganic sulfur compounds by
bacteria: emergence of a common mechanism? Appl Environ
Microbiol 67:2873–2882

Friedrich CG, Bardischewsky F, Rother D, Quentmeier A, Fischer J
(2005) Prokaryotic sulfur oxidation. Curr Opin Microbiol 8:253–
259

Friedrich CG, Quentmeier A, Bardischewsky F, Rother D, Orawski G,
Hellwig P, Fischer J (2008) Redox control of chemotrophic
sulfur oxidation of *Paracoccus pantotrophus*. In: Dahl C,
Friedrich CG (eds) Microbial sulfur metabolism. Springer,
Berlin, pp 139–150

Frigaard NU, Bryant DA (2008a) Genomic and evolutionary
perspectives on sulfur metabolism in green sulfur bacteria. In:
Dahl C, Friedrich CG (eds) Microbial sulfur metabolism.
Springer, Berlin, pp 60–76

Frigaard NU, Bryant DA (2008b) Genomic insights into the sulfur
metabolism of phototrophic green sulfur bacteria. In: Hell R,
Dahl C, Knaff DB, Leustek T (eds) Sulfur metabolism in
phototrophic organisms. Springer, Berlin, pp 337–355

Frigaard NU, Dahl C (2009) Sulfur metabolism in phototrophic sulfur
bacteria. Adv Microbiol Physiol 54:103–200

Griesbeck C, Schütz M, Schödl T, Bathe S, Nausch L, Mederer N, Vielreicher M, Hauska G (2002) Mechanism of sulfide-quinone reductase investigated using site-directed mutagenesis and sulfur analysis. Biochemistry 41:11552–11565

Grimm F, Bettina F, Dahl C (2008) Thiosulfate and sulfur oxidation in purple sulfur bacteria. In: Dahl C, Friedrich CG (eds) Microbial sulfur metabolism. Springer, Berlin, pp 101–116

Hauska G, Schoedl T, Remigy H, Tsiotis G (2001) The reaction center of green sulfur bacteria. Biochim Biophys Acta 1507:260–277

Heising S, Richter L, Ludwig W, Schink B (1999) Chlorobium ferrooxidans sp nov., a phototrophic green sulfur bacterium that oxidizes ferrous iron in coculture with a “Geospirillum” sp strain. Arch Microbiol 172:116–124

Hell R, Dahl C, Knaff DB, Leustek T (eds) (2008) Sulfur metabolism in phototrophic organisms. Advances in photosynthesis and respiration, vol 27. Springer, Berlin

Hansen D, Sperling D, Trüper HG, Brune DC, Dahl C (2006) Thiosulphate oxidation in the phototrophic sulphur bacterium Allochromatium vinosum. Mol Microbiol 62:794–810

Imhoff JF (2008) Systematics of anoxygenic phototrophic bacteria. In: Hell R, Dahl C, Knaff DB, Leustek T (eds) Sulfur metabolism in phototrophic organisms. Springer, Berlin, pp 269–287

Itoh M, Seo D, Sakurai H, Sétif P (2002) Kinetics of electron transfer between soluble cytochrome c-554 and purified reaction center complex from the green sulfur bacterium Chlorobium tepidum. Photosynth Res 71:125–135

Japanese Biochemical Society (ed) (1984) In: Seikagaku handbook (in Japanese). Maruzen, Tokyo

Kappler U, Friedrich CG, Trüper HG, Dahl C (2001) Evidence for two pathways of thiosulfate oxidation in Starkeya novella (formerly Thiobacillus novellus). Arch Microbiol 175:102–111

Kappler U, Aguey-Zinsou KF, Hanson GR, Bernhardt PV, McEwan AG (2004) Cytochrome c<sub>551</sub> from Starkeya novella. Characterization, spectroscopic properties, and phylogeny of a diheme protein of the SoxAX family. J Biol Chem 279:6252–6260

Kappler U, Hanson GR, Jones A, McEwan AG (2005) A recombinant diheme SoxAX cytochrome—implications for the relationship between EPR signals and modified heme-ligands. FEBS Lett 579:2491–2498

Kelly DP, Shergill JK, Lu WP, Wood AP (1997) Oxidative metabolism of inorganic sulfur compounds by bacteria. Antonie van Leeuwenhoek 71:95–107

Khanna S, Nicholas DJD (1982) Utilization of tetrathionate and S<sup>35</sup>-labeled thiosulfate by washed cells of Chlorobium vibrioforme f. sp. thiofulvatophilum. J Gen Microbiol 128:1027–1034

Klughammer C, Hager C, Padan E, Schitz M, Schreiber U, Shahak Y, Hauska G (1995) Reduction of cytochromes with menaquinol and sulfide in membranes from green sulfur bacteria. Photosynth Res 43:27–34

Kostanjevecki V, Brige A, Meyer TE, Cusanovich MA, Guisez Y, Van Beeumen J (2000) A membrane-bound flavocytochrome c-sulfide dehydrogenase from the purple phototrophic sulfur bacterium Ectothiorhodospira vacuolata. J Bacteriol 182:3097–3103

Kusai A, Yamanaka T (1973a) A novel function of cytochrome c (555, Chlorobium thiosulfatophilum) in oxidation of thiosulfate. Biochem Biophys Res Comm 51:107–112

Kusai A, Yamanaka T (1973b) Cytochrome c (553, Chlorobium thiosulfatophilum) is a sulphide-cytochrome c reductase. FEBS Lett 34:235–237

Kusai A, Yamanaka T (1973c) The oxidation mechanisms of thiosulphate and sulphide in Chlorobium thiosulphatophilum: roles of cytochrome c-551 and cytochrome c-553. Biochim Biophys Acta 325:304–314

Kusumoto N, Setif P, Brettel K, Seo D, Sakurai H (1999) Electron transfer kinetics in purified reaction centers from the green sulfur bacterium Chlorobium tepidum studied by multiple-flash excitation. Biochemistry 38:12124–12137

Liebl U, Pezennec S, Riedel A, Kellner E, Nitschke W (1992) The Rieske FeS center from the gram-positive bacterium PS3 and its interaction with the menaquinone pool studied by EPR. J Biol Chem 267:14068–14072

Madigan MT (2001) Family VI. “Heliobacteriaceae”. In: Boone DR, Castenholz RW, Garrity GM (eds) Bergey’s manual of systematic bacteriology, 2nd edn. Springer, New York, pp 625–630

Messens J, Collet JF (2006) Pathways of disulfide bond formation in Escherichia coli. Int J Biochem Cell Physiol 38:1050–1062

Meulenberg R, Pronk JT, Hazeu W, Vandijken JP, Frank J, Bos P, Kuenen JG (1993) Purification and partial characterization of thiosulfate dehydrogenase from Thiobacillus acidophilus. J Gen Microbiol 139:2033–2039

Meyer TE, Cusanovich MA (2003) Discovery and characterization of electron transfer proteins in the photosynthetic bacteria. Photosynth Res 76:111–126

Meyer TE, Bartsch RG, Cusanovich MA, Mathewson JH (1968) Cytochromes of Chlorobium thiosulfatophilum. Biochim Biophys Acta 153:854–861

Miller JR, Busby RW, Jordan SW, Cheek J, Henshaw TF, Ashley GW, Broderick JB, Cronan JE, Marletta MA (2000) Escherichia coli LipA is a lipoyl synthase: In vitro biosynthesis of lipoylated pyruvate dehydrogenase complex from octanoyl-acyl carrier protein. Biochemistry 39:15166–15178

Missiakas D, Schwager F, Raina S (1995) Identification and characterization of a new disulfide isomerase-like protein (DsbD) in Escherichia coli. EMBO J 14:3415–3424

Ogawa T, Furusawa T, Nomura R, Seo D, Hosoya-Matsuda N, Sakurai H, Inoue K (2008) SoxAX binding protein, a novel component of the thiosulfate-oxidizing multienzyme system in the green sulfur bacterium Chlorobium tepidum. J Bacteriol 190:6097–6110

Ogawa T, Furusawa T, Shiga M, Seo D, Sakurai H, Inoue K (in press) Biochemical studies of the soxF-encoded monomeric flavoprotein purified from the green sulfur bacterium Chlorobaculum tepidum that stimulates in vitro thiosulfate oxidation. Biosci Biotechnol Biochem

Oh-Oka H, Iwaki M, Itoh S (1998) Membrane-bound cytochrome c<sub>z</sub> couples quinol oxidoreductase to the P840 reaction center complex in isolated membranes of the green sulfur bacterium Chlorobium tepidum. Biochemistry 37:12293–12300

Okumura N, Shimada K, Matsuura K (1994) Photooxidation of membrane-bound and soluble cytochrome c in the green sulfur bacterium Chlorobium tepidum. Photosynth Res 41:125–134

Okuzumi M, Kita Y (1965) Studies on biochemistry of Thiobacilli. 6. Oxidation of thiosulfate to tetrathionate by T. thiooxidans. Agr Biol Chem 29:1063–1068

Overmann J (2001) Green sulfur bacteria. In: Boone DR, Castenholz RW, Garrity GM (eds) Bergey’s manual of systematic bacteriology, 2nd edn. Springer, Berlin, pp 601–623

Page MD, Ferguson SJ (1997) Paracoccus denitrificans CcmG is a periplasmic protein-disulphide oxidoreductase required for c-and aa(3)-type cytochrome biogenesis; Evidence for a reductase role in vivo. Mol Microbiol 24:977–990

Pattaragulwanit K, Brune DC, Trüper HG, Dahl C (1998) Molecular genetic evidence for extracytoplasmic localization of sulfur globules in Chromatium vinosum. Arch Microbiol 169:434–444

Pott AS, Dahl C (1998) Sirohaem sulfite reductase and other proteins encoded by genes at the dsr locus of Chromatium vinosum are involved in the oxidation of intracellular sulfur. Microbiology-SGM 144:1881–1894

Quentmeier A, Friedrich CG (2001) The cysteine residue of the SoxY protein as the active site of protein-bound sulfur oxidation of *Paracoccus pantotrophus* GB17. FEBS Lett 503:168–172

Quentmeier A, Hellwig P, Bardischewsky F, Wichmann R, Friedrich CG (2004) Sulfide dehydrogenase activity of the monomeric flavoprotein SoxF of *Paracoccus pantotrophus*. Biochemistry 43:14696–14703

Quentmeier A, Janning P, Hellwig P, Friedrich CG (2007) Activation of the heterodimeric central complex SoxYZ of chemotrophic sulfur oxidation is linked to a conformational change and SoxY-Y interprotein disulfide formation. Biochemistry 46:10990–10998

Quentmeier A, Li L, Friedrich CG (2008) Identification of two inactive forms of the central sulfur cycle protein SoxYZ of *Paracoccus pantotrophus*. FEBS Lett 582:3701–3704

Reinartz M, Tschape J, Bruser T, Trüper HG, Dahl C (1998) Sulfide oxidation in the phototrophic sulfur bacterium *Chromatium vinosum*. Arch Microbiol 170:59–68

Rother D, Henrich HJ, Quentmeier A, Bardischewsky F, Friedrich CG (2001) Novel genes of the *sox* gene cluster, mutagenesis of the flavoprotein SoxF, and evidence for a general sulfur-oxidizing system in *Paracoccus pantotrophus* GB17. J Bacteriol 183:4499–4508

Sakurai H, Kusumoto N, Inoue K (1996) Function of the reaction center of green sulfur bacteria. Photochem Photobiol 64:5–13

Sander J, Dahl C (2009) Metabolism of inorganic sulfur compounds in purple bacteria. In: Hunter N, Daldal F, Thurnauer MC, Beaty JT (eds) The purple phototrophic bacteria. Berlin, Springer, pp 60–76

Sauve V, Roversi P, Leath KJ, Garman EF, Antrobus R, Lea SM, Berks BC (2009) Mechanism for the hydrolysis of a sulfur-sulfur bond based on the crystal structure of the thiosulfate hydrolase SoxB. J Biol Chem 284:21707–21718

Schütz M, Shahak Y, Padan E, Hauska G (1997) Sulfide-quinone reductase from *Rhodobacter capsulatus*—purification, cloning, and expression. J Biol Chem 272:9890–9894

Shahak Y, Hauska G (2008) Sulfide oxidation from cyanobacteria to humans: sulfide-quinone oxidoreductase (SQR). In: Hell R, Dahl

C, Knaff DB, Leustek T (eds) Sulfur metabolism in phototrophic organisms. Springer, Berlin, pp 337–355

Shahak Y, Arieli B, Padan E, Hauska G (1992) Sulfide quinone reductase (SQR) activity in *Chlorobium*. FEBS Lett 299:127–130

Steudel R (2003) Inorganic polysulfane H₂Sₙ with n > 1. In: Steudel R (ed) Elemental sulfur and sulfur-rich compounds II. Topics in current chemistry, vol 231. Springer, Berlin

Steudel R, Steudel Y (2010) Derivatives of cysteine related to the thiosulfate metabolism of sulfur bacteria by the multi-enzyme complex “Sox’’-studied by B3LYP-PCM and G3X(MP2) calculations. Phys Chem Chem Phys 12:630–644

Sugio T, Taha TM, Takeuchi F (2009) Ferrous iron production mediated by tetrathionate hydrolase in tetrathionate-, sulfur-, and iron-grown *Acidithiobacillus ferrooxidans* ATCC 23270 cells. Biosci Biotechnol Biochem 73:1381–1386

Trüper HG, Lorenz C, Schedel M, Steinmetz M (1988) Metabolism of thiosulfate in *Chlorobium*. In: Olson M, Ormerod JG, Ames J, Stackebrandt E, Trüper HG (eds) Green photosynthetic bacteria. Plenum Press, New York, pp 189–200

Tsukatani Y, Miyamoto R, Itoh S, Oh-oka H (2006) Soluble cytochrome c-554, CycA, is not essential for photosynthetic electron transfer in *Chlorobium tepidum*. FEBS Lett 580:2191–2194

Verté F, Kostanjevecki V, Smet LD, Meyer TE, Cusanovich MA, Van Beeumen JJ (2002) Identification of a thiosulfate utilization gene cluster from the green phototrophic bacterium *Chlorobium limicola*. Biochemistry 41:2932–2945

Visser JM, de Jong GAH, Robertson LA, Kuenen JG (1997) A novel membrane-bound flavocytochrome c sulfide dehydrogenase from the colourless sulfur bacterium *Thiobacillus* sp. W5. Arch Microbiol 167:295–301

Welte C, Hafner S, Kratzer C, Quentmeier A, Friedrich CG, Dahl C (2009) Interaction between Sox proteins of two physiologically distinct bacteria and a new protein involved in thiosulfate oxidation. FEBS Lett 583:1281–1286

Yamanaka T (1996) Mechanisms of oxidation of inorganic electron donors in autotrophic bacteria. Plant Cell Physiol 37:569–574
