
Brassinosteroid Signal Transduction from Receptor Kinases to Transcription Factors

Tae-Wuk Kim and Zhi-Yong Wang

Department of Plant Biology, Carnegie Institution for Science, Stanford, California 94305; email: zywang24@stanford.edu

Annu. Rev. Plant Biol. 2010. 61:681–704

First published online as a Review in Advance on February 5, 2010

The Annual Review of Plant Biology is online at plant.annualreviews.org

This article's doi: 10.1146/annurev.arplant.043008.092057

Copyright © 2010 by Annual Reviews. All rights reserved

1543-5008/10/0602-0681$20.00

Key Words

Arabidopsis, GSK3 kinase, plant hormone, receptor kinase, signal transduction, steroid hormone

Abstract

Brassinosteroids (BRs) are growth-promoting steroid hormones in plants. Genetic studies in *Arabidopsis* illustrated the essential roles of BRs in a wide range of developmental processes and helped identify many genes involved in BR biosynthesis and signal transduction. Recently, proteomic studies identified missing links. Together, these approaches established the BR signal transduction cascade, which includes BR perception by the BRI1 receptor kinase at the cell surface, activation of the BRI1/BAK1 kinase complex by transphosphorylation, subsequent phosphorylation of the BSK kinases, activation of the BSU1 phosphatase, dephosphorylation and inactivation of the BIN2 kinase, and accumulation of unphosphorylated BZR transcription factors in the nucleus. Mass spectrometric analyses are providing detailed information on the phosphorylation events involved in each step of signal relay. Thus, the BR signaling pathway provides a paradigm for understanding receptor kinase-mediated signal transduction as well as tools for the genetic improvement of the productivity of crop plants.

Contents
INTRODUCTION ..... 682
RECEPTOR ACTIVATION ..... 684
BRI1 Perceives BRs at the Cell Surface ..... 684
BR-Induced Receptor Kinase Dimerization and Oligomerization ..... 684
Team Play and Multitasking of the Receptor Kinases ..... 686
Regulation by Auto- and Transphosphorylation ..... 687
Endocytosis of BRI1 and BAK1 ..... 690
Regulation by BKI1 ..... 690
SIGNALING OUTPUT FROM THE RECEPTOR KINASES ..... 691
BZR1 AND BZR2/BES1: KEY TRANSCRIPTION FACTORS FOR BR-REGULATED GENE EXPRESSION ..... 692
REGULATION OF BZR1 AND BZR2/BES1 BY BIN2-MEDIATED PHOSPHORYLATION ..... 694
REGULATION OF BIN2 BY BSU1-MEDIATED TYROSINE DEPHOSPHORYLATION AND PROTEASOME-MEDIATED DEGRADATION ..... 696
CONCLUDING REMARKS ..... 699

INTRODUCTION

Steroids have long been known as hormones in metazoans (85). Their hormonal activity in plants was only discovered about 30 years ago, when a growth-promoting activity in pollen extract of oilseed rape (*Brassica napus*) was attributed to a steroid compound named brassinolide (BL) (24). Since then, steroids with similar structure and activities as BL have been identified in a variety of plants and are collectively called brassinosteroids (BRs) (13). While hormone treatments demonstrated many physiological and developmental responses induced by exogenous BR, the essential function of BRs in plant development was not widely appreciated until BR-deficient and BR-insensitive mutants were identified in a relative of *B. napus*, the model organism *Arabidopsis thaliana* (12, 49, 81). These mutants display many growth defects, including dwarfism, dark green leaves, delayed flowering, male sterility, and photomorphogenesis in complete darkness. Now BRs are considered essential hormones that regulate a wide range of developmental and physiological processes, such as cell elongation, vascular differentiation, root growth, responses to light, resistance to stresses, and senescence.

*Arabidopsis* mutants identified not only BR biosynthetic enzymes but also signaling proteins that perceive and transduce the BR signal. Forward genetic screens in *Arabidopsis* identified multiple alleles of two loci that cause BR insensitivity, *bri1* and *bin2*. *BR INSENSITIVE 1 (BRI1)* encodes one of over 200 members of the leucine-rich repeat-receptor-like kinase (LRR-RLK) family in *Arabidopsis* (79), and *BR INSENSITIVE 2 (BIN2)* encodes one of the 10 glycogen synthase kinase 3 (GSK3)-like kinases of *Arabidopsis* (50). Sensitized genetic screens using the BR biosynthetic inhibitor brassinazole (BRZ) or weak alleles of *bri1* as genetic background identified additional components of the pathway, including the BRASSINAZOLE RESISTANT 1 (BZR1) and *bri1*-EMS-SUPPRESSOR 1 (BES1) transcription factors (93, 97), BRI1-ASSOCIATED RECEPTOR KINASE 1 (BAK1) (52, 60), and *bri1*-SUPPRESSOR 1 (BSU1) phosphatase (57) (Figure 1). BES1 is 89% identical to BZR1 and was also named BZR2 (93). BAK1 was also identified by yeast two-hybrid screening as a BRI1-interacting protein (60).

Additional components of the BR signal transduction pathway have been identified as proteins that interact with these genetically defined components. The BRI1-interacting proteins include BAK1 (60), BRI1 KINASE INHIBITOR 1 (BKI1) (89), and TRANSTHYRETIN-LIKE (TTL) (61). The 14-3-3 proteins were identified as BZR1- and BZR2/BES1-interacting proteins (21, 70). In addition, there is evidence that BES1/BZR2

The structures of BRI1 and BAK1. BRI1 is composed of 24 leucine-rich repeats (LRR) and a 70-amino-acid island domain (ID), single-pass transmembrane region (TM), juxtamembrane region (JM), kinase domain (KD), and C-terminal (CT) region. BAK1 contains 4 leucine zippers (LZ), 5 LRRs, and a proline-rich region (pro-rich), single-pass TM, KD, and CT. The putative signal peptide region is shown as a black box and unassigned regions are shown as gray boxes. The activation loop of kinases is designated AL. The confirmed phosphorylation sites are marked with circles, and putative phosphorylation sites are marked with squares containing the letter P. The activation phosphorylation sites are shown in red, inhibitory sites in blue, and residues without significant effect on the kinase activity or not examined experimentally in yellow.
interacts with other transcription factors, including BIM1, ELF6, and Myb30 (53, 96, 98). Proteomic studies identified additional components that filled the last gap in the pathway (84), and biochemical studies assembled these components into a phosphorylation cascade that connects BR perception at the cell surface to activation of transcription factors in the nucleus (42). With the process in hand, detailed biochemical studies are revealing molecular mechanisms of receptor kinase activation (91, 92), signal relay, and regulation of the BZR transcription factors by phosphorylation (21, 70, 86). BR regulation of gene expression mediated by the BZR transcription factors has been covered in recent reviews (87). This review focuses on how the BR signal is transduced from the cell surface receptor kinases to the nuclear transcription factors.

flanking LRR21 is sufficient for BR binding (43). These studies demonstrated that BRI1 is the receptor of BRs and defined a new steroid-binding motif. BR binding to the extracellular domain activates the cytoplasmic kinase, which transduces the signal to other intracellular proteins.

Most of the mutations identified in the 28 *bri1* alleles are clustered in either the ID-LRR21 or cytoplasmic kinase domain, consistent with an essential role of the BR binding and kinase activity for BRI1 function. Interestingly, some of the mutations in the extracellular domain, such as *bri1-5* and *bri1-9* alleles, cause retention of the mutant protein in the endoplasmic reticulum (ER) due to protein misfolding. Genetic screens for suppressors of these alleles have identified components of the ER quality-control system that mediate retention and processing of the misfolded *bri1* proteins (34, 37).

A domain-structure analysis also uncovered important roles for the JM and CT domains. Deletion of the JM domain abolished the signaling function of BRI1, suggesting a positive role for the JM in BRI1 function. In contrast, a truncated BRI1, with the C-terminal 49 amino acid residues deleted, displayed higher kinase activity in vitro and more complete rescue of the *bri1* mutant phenotype in transgenic plants, suggesting an autoinhibitory role of the CT (92). The CT region contains multiple phosphorylation sites, and mutation of these Ser/Thr residues to phosphomimetic acidic amino acids had effects similar to the deletion of this region, suggesting that phosphorylation of the CT region helps to activate the kinase. This may occur through a conformational change that releases the autoinhibition of the kinase by the C-terminal tail (92).

### RECEPTOR ACTIVATION

#### BRI1 Perceives BRs at the Cell Surface

The *brassinosteroid insensitive 1* (*bri1*) mutant was identified as a dwarf insensitive to BR treatment (12, 40). Additional *bri1* alleles helped to clone the *BRI1* gene, which encodes an LRR-RLK (47). The extracellular domain of BRI1 contains 24 leucine-rich repeats (LRRs) and an island domain (ID) between the twentieth and twenty-first LRRs; BRI1's cytoplasmic domain contains a serine/threonine (Ser/Thr) kinase domain, a juxtamembrane region (JM), and the C-terminal extension (CT) (47, 87) (Figure 1).

The function of BRI1 as the BR receptor has been demonstrated by several lines of experimental evidence. These include a chimeric receptor of BRI1-XA21 conferring BR-induced defense responses (29), binding of BRI1 immunoprecipitated from *Arabidopsis* plants (94) or expressed and purified in *Escherichia coli* (43) to tritium-labeled BL (^3^H-BL), and BR-induced BRI1 auto-phosphorylation (90, 91, 94). It was further shown that the 94-amino-acid ID-LRR21 domain containing ID and its

### BR-Induced Receptor Kinase Dimerization and Oligomerization

Studies on animals have shown that receptor kinase activation usually involves ligand-induced homo- or hetero-dimerization orhetero-oligomerization (67, 74). For example, perception of TGFβ involves a homo-dimeric type I TGFβ receptor and a homo-dimeric type II TGFβ receptor. TGFβ binds to the type II receptor to promote its association with and phosphorylation of the type I receptor, which transduces the signal to downstream components. Similarly, BR signaling also involves ligand-induced BRI1 homodimerization or homo-oligomerization and hetero-oligomerization with a coreceptor kinase named BRI1-associated kinase 1 (BAK1).

Homo-dimerization of BRI1 was first detected using fluorescence resonance energy transfer (FRET) analysis in the plasma membrane (PM) of cowpea protoplasts (69). Furthermore, using single molecule detection methods, it has been demonstrated that about 20% of BRI1 molecules in the PM are present in homo-dimerized form without higher-order oligomerization (31). BRI1 homo-dimerization has also been detected by coimmunoprecipitation of BRI1-GFP and BRI1-FLAG fusion proteins from transgenic *Arabidopsis* plants (92). Although BR had no effect on BRI1 homo-dimerization in protoplasts (31), BR treatment of *Arabidopsis* plants increased the coimmunoprecipitation of BRI1-GFP and BRI1-FLAG, indicating that BR promotes or stabilizes BRI1 homo-dimerization (92). It is yet unclear whether BRs function as a molecular glue, in a manner similar to auxin-induced TIR1-IAA protein dimerization (82), to help stabilize the BRI1 homo-dimer, or whether BR binding causes a conformational change that favors homo-dimerization. The latter possibility is supported by the estimated one BL molecule that is bound per BRI1-GFP molecule (43). It is also unclear whether the kinase activity is required for BR-induced homo-dimerization of BRI1. Nevertheless, BR-induced homo-dimerization of BRI1 seems not sufficient to fully activate the BRI1 kinase, and further association with the coreceptor kinase BAK1 is required.

BAK1 was identified independently by two research groups using a yeast two-hybrid screen for BRI1 interacting proteins and an acti-

vation tagging screen for *brl1-5* suppressors (52, 60). BAK1 is also named SERK3 and is one of the five SOMATIC EMBRYOGENESIS RECEPTOR-LIKE KINASES (SERK1 to SERK5) (75), which are LRR-RLKs that contain a small extracellular domain with five LRRs (Figure 1). Overexpression of BAK1 suppresses the phenotype of a weak *brl1* allele (52, 60). The *bak1* loss-of-function mutant resembles weak *brl1* mutants (52), and overexpression of a kinase-inactive *bak1* mutant protein causes severe dwarf phenotypes similar to strong *brl1* mutants, presumably due to a dominant negative effect (52). Recent genetic studies of loss-of-function mutations of BAK1/SERK3 and its homologs (SERK1 and SERK4) confirmed their essential roles in BR signaling, as well as in additional pathways (3, 28).

The kinase domains of BRI1 and BAK1 interact with and trans-phosphorylate each other in vitro and in yeast (52, 60), and their in vivo interaction has been detected by coimmunoprecipitation using transgenic *Arabidopsis* and FRET analysis in protoplasts (52, 60, 69). Both BRI1 and BAK1 show basal levels of kinase activities in vitro, which are increased by the corresponding partner (52). In yeast, kinase activity is detected only when wild type BRI1 and BAK1 are coexpressed but is not detected when either BRI1 or BAK1 is mutated or absent (60). These results suggest that full activation of either receptor kinase requires transphosphorylation by its partner.

In *Arabidopsis*, BR treatment promotes the interaction between BRI1 and BAK1 and increases their phosphorylation (90, 91). The BR-induced BRI1–BAK1 association seems to be mediated by their kinase domains rather than cooperative binding of the extracellular domains. First, BR binding to BRI1 appears to be independent of BAK1 and BAK1 does not participate in BR binding, because the BL binding activity of BRI1 was not altered in a *bak1* null mutant or by overexpression of BAK1 (43, 92). Second, the interaction between BRI1 and BAK1 requires their kinase activities. In vitro interaction is reduced by mutations of the BAK1 kinase domain and almost completelyabolished by kinase-inactive mutations of the  
BRI1 kinase domain (52). A kinase-dead mutant  
BRI1 did not show BR-induced association with  
BAK1, indicating that intact extracellular do-  
mains are not sufficient and BRI1 kinase activ-  
ity is required for BR-induced association with  
BAK1 (91). However, the L46E mutation in the  
leucine zipper of BAK1 extracellular domain  
appeared to abolish BAK1 binding to BRI1 in  
yeast (99). While this result supported a pos-  
sible role of the extracellular domain in BRI1–  
BAK1 association, the lack of BRI1 interaction  
could also be due to a possible mislocalization  
(such as retention in the ER) or instability of the  
mutant BAK1 in the cell. On the other hand, a  
kinase-dead mutant BAK1 still associates with  
wild type BRI1 upon BR treatment, suggest-  
ing that BR first activates BRI1 kinase, and the  
association with BAK1 is a result of initial acti-  
vation of the BRI1 kinase (91). A requirement  
of BRI1 kinase activity for BAK1 interaction  
and activation is consistent with the observa-  
tion that BAK1 overexpression suppresses only  
weak alleles of bri1 but not the bri1-5 det2 dou-  
ble mutant or bril null mutants (52, 60). These  
results suggest that BAK1 is involved in BRI1  
activation rather than transducing the BR signal  
to downstream substrates.

### Team Play and Multitasking  
of the Receptor Kinases  

BRI1 is the major BR receptor in higher plants.  
Mutations of the BRI1 orthologs in tomato,  
rice, and pea cause BR-insensitive phenotypes  
(55, 59, 62). However, BRI1 is not the only  
BR receptor. There are three close homologs  
of BRI1 in *Arabidopsis* and at least two of them,  
BRL1 and BRL3, bind to BR with high affinity  
and can rescue the bri1 mutant when expressed  
from a BRI1 promoter (8, 43, 101). These BRI1  
homologs apparently mediate cell-type-specific  
BR response in vascular tissues (8). Such func-  
tional specification of BRI1 homologs obvi-  
ously occurred during evolution before the sep-  
aration of monocots from dicots, since OsBRL1  
and OsBRL3 also seem to mediate tissue- or  
cell-type-specific BR responses in rice (59).

BAK1 also belongs to a small subfamily of  
RLKs called SOMATIC EMBRYOGENESIS  
RECEPTOR KINASES (SERK1 to SERK5).  
In addition to BAK1 (SERK3), two additional  
members of the SERK family, SERK1 and  
BKK1 (BAK1-like 1, SERK4), appear to play  
redundant roles with BAK1 in BR signal-  
ing. As with BAK1, overexpression of SERK1  
or BKK1 partially suppresses the bri1-5 mu-  
tant, and they interact with BRI1 in vivo (3,  
28, 38, 69). Whereas the bak1 single loss-  
of-function mutant shows a very weak dwarf  
phenotype, the bak1 serk1-1 double mutant  
shows a significantly enhanced BR-insensitive  
dwarf phenotype, and the bak1-4 bkk1-1 dou-  
ble mutant shows an only slightly enhanced  
de-etiolation phenotype compared to bak1 it-  
self. It was recently proposed that SERK1 and  
BAK1/SERK3, but not BKK1/SERK4, medi-  
ate BR response (3). However, the phenotype  
of the bak1 serk1 double mutant is still much  
weaker than the strong bri1 alleles or than trans-  
genic plants expressing a dominant negative  
form of bak1 (52). The analysis might have been  
complicated by the additional roles of BAK1  
and BKK1 in pathways that suppress cell death.  

Genetic studies have revealed additional  
roles of BAK1 and other members of the SERK  
family in multiple signaling pathways. First,  
BAK1 and BKK1/SERK4 are involved in con-  
trolling a light-dependent cell death process  
(27). The bak1 bkk1 double mutant displays a  
seedling-lethal phenotype in the light but not  
in the dark. Many aspects of cell death pheno-  
types, including the upregulation of defense-  
related genes and the accumulation of callose  
and H₂O₂, were observed in the bak1 bkk1 dou-  
ble mutant but not in each single mutant or in  
the bri1 null mutant, indicating that BAK1 and  
BKK1 play a redundant role in a pathway that  
suppresses cell death independently of BR sig-  
naling (28). Second, BAK1 plays a key role in  
pathogenesis. The bak1 mutant displayed en-  
hanced susceptibility to necrotrophic pathogen  
infection (41). The application of BR suppresses  
only growth defects but not the disease phe-  
notype of the bak1 mutant, supporting the  
hypothesis that BAK1 has a BR-independent  
role in defense (41). In addition, the *bak1* mutant shows reduced sensitivity to pathogen-associated molecular patterns (PAMPs) such as flagellin, EF-Tu, INF1, and CSP22. It was further revealed that BAK1 interacts with the flagellin receptor FLS2, another LRR-RLK protein, upon flagellin treatment (10). These results indicate that BAK1 also functions as a coreceptor of FLS2 for flagellin signaling. Shan and coworkers (78) reported that virulence factors of bacterial pathogen target BAK1 to overcome the host defense system and to inhibit BR signaling as well. The transgenic plants overexpressing the bacterial effector AvrPto show dwarf phenotypes that mimic weak *bril* mutants, and AvrPto and AvrPtoB interact with BAK1, leading to disruption of FLS2-BAK1 association (78). Third, SERK1 and SERK2 play a redundant role in male sporogenesis. The *serk1 serk2* double mutant is male sterile due to a defect in tapetum development in the anther (2, 14).

Taken together, it has been shown that the four members of the SERK family play overlapping roles in at least four distinct pathways: BAK1, SERK1, and BKK1/SERK4 in BR signaling; BAK1 and BKK1/SERK4 in cell death control; BAK1 in flagellin signaling and innate immunity; and SERK1 and SERK2 in male sporogenesis (3). Members of a gene family were likely generated by gene duplication, which initially created copies of genes that were fully redundant. How different SERK members gain or retain specific functions is an interesting evolutionary question. Functional specificity can be generated by changes in the promoter or coding sequences, resulting in diverse tissue-specific gene expression or protein activity. Expression of SERK3 from the SERK1 or SERK2 promoter failed to rescue the male-sterility phenotype of *serk1 serk2* double mutant, suggesting that the protein specifies the function in this case (3). In contrast, the BRI1 homologs appear to have gained functional specificity by changes in the promoters and gene expression patterns, as expression of BRL1 from the BRI1 promoter rescued the *bri1* mutant (8).

It should be noted that dual function was also reported for tomato BRI1, which was shown to bind systemin, a peptide hormone for wound-ing response (72, 73); however, this finding was later disputed (32, 33, 55). The more recent studies concluded that tomato BRI1 is required for BR signaling but not for systemin signaling (33, 55). Similarly, progesterone was shown to bind to the same G-protein-coupled receptor as the peptide oxytocin in certain animal cells (23), but this finding was also later disputed (4, 7).

**Regulation by Auto- and Transphosphorylation**

To understand the signaling mechanisms of BRI1 and BAK1, several recent studies have identified phosphorylation sites in their kinase domains, mostly by using mass spectrometry (MS) (39, 64, 90, 91, 99) (Figure 1). The in vitro phosphorylation sites were identified by analyzing the recombinant intracellular domains expressed and purified from the bacterium *Escherichia coli*, and in vivo phosphorylation sites were identified by analyzing epitope-tagged BRI1 or BAK1 proteins immunoprecipitated from transgenic *Arabidopsis* plants. These studies identified eleven unambiguous phosphorylation sites in BRI1 (six identified in vivo and five identified only in vitro). MS data also supported the presence of additional phosphorylation sites that could not be assigned to specific amino acid residues, including at least three sites in the activation loop between domains VII and VIII. In addition, site-directed mutagenesis followed by phosphopeptide mapping provided the evidence for phosphorylation on additional residues (S1162) (92). Although initial MS analyses identified only phosphorylation of Ser/Thr residues, a recent study using phosphopeptide-specific antibodies and MS analysis provided convincing evidence that BRI1 is also phosphorylated at tyrosine residues in vitro and in vivo, indicating that BRI1 is a dual-specificity kinase (64). Interestingly, many other LRR-RLKs, e.g., BAK1 and BKK1, also contain phosphorylation at tyrosine residues (64).
The functions of many identified or putative phosphorylation sites in BRI1 have been studied using site-directed mutagenesis followed by in vitro kinase assays and in planta rescue of the *bri1* mutants (64, 90, 91). These studies confirmed an important role of phosphorylation in the activation loop for the kinase activity of BRI1. Mutations of S1044/T1045, T1049, Y1052, or Y1057 nearly abolished BRI1's kinase activity in vitro and its ability to rescue the *bri1-5* mutant. Mutations of two additional Tyr residues (Y956 or Y1072) outside the activation loop also abolished the kinase activity, though phosphorylation of Y1072 has not been detected in vivo.

Important function was also demonstrated for phosphorylation in the JM and CT domains of BRI1. Deletion of the JM domain from BRI1 abolished its signaling function and ability to rescue the *bri1-5* mutant phenotype, without affecting its localization at the plasma membrane (92). Seven phosphorylation sites in the JM domain were identified as autophosphorylation sites in vitro and in vivo by MS analysis (63, 90). Of these, the phosphomimetic substitutions of four phosphorylation sites (S838D, T842D, T846D, and S858D) enhanced the phosphorylation of a BRI1 synthetic peptide substrate in the absence of BAK1, indicating that phosphorylation of the JM domain activates BRI1 (91). However, the action mechanism by which the JM domain modulates BRI1 activity remains unclear.

By contrast, deletion of the CT domain increases the kinase activity of BRI1 in vitro and in vivo and promotes hypocotyl growth of *bri1-5* and *det2* mutant more effectively than wild type BRI1 (92). Multiple Ser/Thr residues of the CT domain are phosphorylated. At least four Ser/Thr residues (S-1162, S-1166, S-1168, and T-1180) among eight possible sites in the CT domain were experimentally confirmed (90–92). Multiple substitutions of Ser/Thr residues to Asp in the CT domain increase BRI1 kinase activity independent of BAK1 activity (91). Moreover, mutant BRI1 containing phosphomimetic mutations in the CT domain more effectively suppressed the dwarf phenotype of

det2 (91, 92). These results demonstrated that the CT domain of BRI1 is autoinhibitory of BRI1 kinase and this inhibitory effect is alleviated by phosphorylation (92). The functions of the JM and CT domains of BRI1 demonstrate that noncatalytic regions of the cytoplasmic domain play important roles in specifically modulating kinase activity. It is well known that nonkinase intracellular domains of mammalian receptor kinases play important roles not only in autoregulation but also in creating binding sites for substrates in a phosphorylation-dependent manner (35, 36). Thus, it is possible that JM and CT of BRI1 are involved in regulating kinase activity as well as creating binding sites for BRI1 downstream substrates.

A total of nine phosphorylation sites have been identified in the intracellular region of BAK1; five of them were identified in vivo (S290, T312, T446, T449, and T455) and four in vitro (S286, T450, S604, and S612) (39, 91, 92, 99). Unlike BRI1, BAK1 is mainly phosphorylated in the activation loop (T446, T449, T450, and T455) (91), although two phosphorylation sites (S-604 and S-612) in the CT domain were recently detected in vitro (39). The phosphorylation of the T455 residue, corresponding to T1049 of BRI1 in the activation loop of BAK1, is essential for BAK1 function, since mutation of T455A abolished the kinase activity (91). Although single mutation of the other phosphorylation sites in the activation loop had little effect on BAK1 activity, changing all three Thr to Ala (T446A/T449A/T450A) abolished the kinase activity (91, 99), indicating that phosphorylation in the activation loop is critical for BAK1 kinase activity.

While analyses of the in vitro autophosphorylation and in vivo phosphorylation sites elucidated important residues at which phosphorylation affects kinase activity and signaling function, the mechanisms of BR regulation of these phosphorylation events remain unclear. Recently, it has been proposed that BR-induced association between BRI1 and BAK1 is a result of initial activation of the BRI1 kinase (91), and sequential transphosphorylation between BRI1 and BAK1 activates both receptor kinases.BR-induced association between BRI1 and BAK1 requires kinase activity of BRI1 but not that of BAK1, since BR increases the interaction between wild type BRI1 and kinase-dead mutant BAK1 but not between mutant BRI1 and wild type BAK1. Although the mutant BRI1 still shows basal level of interaction with BAK1, this interaction is not increased by BR treatment (91). In addition, BR-induced BAK1 phosphorylation is enhanced by BRI1 overexpression but abolished in the *bri1* null mutant. By contrast, BRI1 phosphorylation is still induced by BR treatment even in the *bak1 bkk1 (serk3 serk4)* double mutant background, though at a reduced degree compared with the wild type or BAK1-overexpressing plants. Prephosphorylation of BRI1 by BAK1 in vitro increases BRI1 phosphorylation of a BRI1-specific substrate, indicating that BRI1 has basal activity that is further activated by BAK1 transphosphorylation (91). Furthermore, overexpression of BRI1 rescues hypocotyl elongation of the *bak1 bkk1* double mutant, whereas overexpression of BAK1 fails to suppress null *bri1* mutants (91). However, SERK1 might still provide a function in the *bak1 bkk1* double mutant, because SERK1 also associates with BRI1 and contributes to BR signaling (3, 38). As such, analysis of a *bak1 bkk1 serk1* triple null mutant is still required to determine if BAK1 and related proteins play essential or supporting roles in BR activation of BRI1.

Transactivation between BRI1 and BAK1 is mediated by transphosphorylation. The transphosphorylation sites have been identified by using in vitro kinase assays performed with kinase-inactive mutant protein as the substrate for its wild type partner. MS analyses showed that BRI1 mainly phosphorylates the kinase domain of BAK1. Of six BAK1 residues (S290, T312, T446, T449, T450, and T455) transphosphorylated by BRI1, four Thr residues (T446, T449, T450, and T455) are located in the activation loop and required for the kinase activity and signaling function of BAK1 (91). Mutation to Ala either of T455 alone or of T446, T449, and T450 together abolished the kinase activity of BAK1 (91, 99). Therefore,

phosphorylation of these residues by BRI1 is likely to activate BAK1.

In contrast to BRI1 phosphorylation of the BAK1 activation loop, BAK1 mainly transphosphorylates the JM and CT domains of mutant BRI1 (91). Three phospho-residues in the JM domain (S838, T846, and S858) and two in the CT domain (S1166 and T1180) of BRI1 were identified as BAK1 transphosphorylation sites. In addition, phosphomimetic mutants of JM and CT domains showed increased BRI1 kinase activity (91), and *bri1* mutants carrying the Ser/Thr substitution to Ala in the JM domain were less effective than wild-type BRI1 in suppressing the short hypocotyl phenotype of the *bri1-5* mutant. These results support the hypothesis that BAK1 activates BRI1 by phosphorylating the JM and CT domains (91). Together, the results support a sequential phosphorylation model for BR-activation of the BRI1–BAK1 receptor kinases: BR binding induces a basal activation of BRI1 kinase, which then binds to BAK1 and phosphorylates BAK1 in the activation loop to activate BAK1; activated BAK1 then phosphorylates BRI1 in the JM and CT region to fully activate BRI1’s kinase activity and signaling function through enhanced phosphorylation of downstream substrates.

While phosphorylation at most sites positively activates the kinases, mutagenesis studies also revealed phosphorylation sites that possibly inhibit kinase activity. The T872A mutation of BRI1 increased its kinase activity by over tenfold, due to an increased \(V_{\text{max}}\) and reduced \(K_{\text{m}}\). The results suggest that phosphorylation at T872 has a negative regulatory effect on the kinase activity, although it is also possible that the substitution of Ala at this specific Thr residue changes the protein conformation to a more active state (90). In addition, the S286D mutation abolished BAK1 activity in vitro and in vivo, although S286A mutation had little effect (91). Phosphorylation of S286 was detected in vitro as a BAK1 autophosphorylation site. It remains unclear whether S286 is phosphorylated in vivo by BAK1 autophosphorylation as a desensitizing mechanism or by other kinases that modulate the BAK1 activity (91).
In addition to affecting the kinase activity, some phosphorylation sites appear to mediate specific signaling output. Mutation of Y831F in the JM domain of BRI1 had no obvious effect on the kinase activity. Expression of mutant BRI1 Y831F rescued the growth phenotype of *bril-5* but caused larger leaves and early flowering, suggesting that phosphorylation of Y831 might provide signal output to leaf development and flowering regulation. In addition, BAK1 containing the T450A mutation in the activation loop could rescue the cell-death and dwarf phenotypes of the *bak1 bkk1* double mutant but failed to rescue the flagellin-insensitive phenotype of the mutant (91). These results suggest that individual phosphorylation sites might affect specific signaling output.

**Endocytosis of BRI1 and BAK1**

In mammals and yeast, endocytosis is a common mechanism for delivering activated receptors from the plasma membrane (PM) to its downstream cellular targets or for clearing the receptors from the cell surface to terminate the signaling and desensitize the system (30, 56). Endocytosis of BRI1 and BAK1 was first observed in cowpea protoplasts (69), where endocytosis was accelerated by coexpression of BRI1 and BAK1. Recently, BRI1-GFP was observed in the endosomes in *Arabidopsis* root cells independently of BR treatment (22). Brefeldin A (BFA), an inhibitor blocking transition from early to late endosomal compartments, induced the accumulation of BRI1-GFP in endosomal compartments. Interestingly, BFA-treatment activated BR signaling and caused dephosphorylation of BZR2/BES1 and inhibition of DWF4 expression (22). In contrast, Endosidin1, a chemical affecting PM/endosome trafficking, resulted in the accumulation of BRI1-GFP in large intracellular bodies (68) but caused a dwarf phenotype similar to the *bril1* mutant. BRI1 endocytosis appears not to be regulated by BR but is increased by overexpression of BAK1 in the protoplasts.

SERK1 is also internalized, and its internalization involves the KINASE-ASSOCIATED

PROTEIN PHOSPHATASE (KAPP) (77). KAPP contains a phosphopeptide-binding domain (FHA) and interacts with SERK1 in a kinase-activity-dependent manner. Coexpression of KAPP increased SERK1 internalization. Although both SERK1 and KAPP colocalize at the plasma membrane, a FRET signal that indicates their physical interaction was detected only in the intracellular vesicles. Thus, it has been proposed that KAPP controls SERK1 internalization (77). KAPP also interacts with both BRI1 and BAK1 in a phosphorylation-dependent manner (17), since it did not interact with kinase-inactive mutant BRI1 (K911E) and BAK1 (T546A). The *kapp bril-5* double mutant showed increased BL sensitivity over the *bril-5* single mutant, suggesting that KAPP might be a negative regulator of BR signaling (17). Whether KAPP inhibits BR signaling by dephosphorylating the receptor kinases or promoting their endocytosis remains to be studied.

While these results support the importance of correct localization of BRI1 for its signaling function, the function of BRI1 endocytosis remains unclear. Based on positive signaling from BFA-sensitive endosomal BRI1, it has been proposed that endocytosis provides additional surface space or proximity for interaction with cytoplasmic proteins, or traps ligand-receptor complexes in small vesicle enclosures to maintain BR signaling (22). The latter scenario is likely to be important in the roots, where water can dilute extracellular signal molecules. In *Arabidopsis*, BRI1 is localized in the PM in most cells and endosomal BRI1 is observed primarily in root cells (20, 22). On the other hand, we still do not know how BRI1 is removed from the cell surface, e.g., in mature tissues that stop growing (20), and whether internalization could be involved in receptor degradation as shown for some receptors in animal systems (30).

**Regulation by BKI1**

The activity of the BRI1/BAK1 receptor kinases is also regulated by additional interacting proteins. In addition to BAK1, several novel proteins have been identified
as BRI1-interacting proteins by yeast two-
hybrid screens (61, 89). One of these, the
BRI1 KINASE INHIBITOR 1 (BKI1), was
shown to negatively regulate BRI1 function.
RNAi knockdown of BKI1 caused enhanced
hypocotyl elongation, whereas overexpression
of BKI1 resulted in a weak dwarf phenotype.
In addition, BKI1 overexpression reduces the
BZR2/BES1 dephosphorylation induced by BR
treatment and alters BR-regulated gene expres-
sion, indicating that BKI1 is a negative regula-
tor of BR signaling. BKI1 is localized at the
plasma membrane in the absence of BR, but
is rapidly relocalized into the cytosol upon BR
treatment (89). These observations suggest that
BKI1 interacts with BRI1 at the plasma mem-
brane to keep BRI1 in an inactive form. BKI1
inhibits BRI1 at least partly by blocking its in-
teraction with BAK1, because incubation with
BKI1 reduced BRI1 interaction with BAK1 in
vitro. Upon initial BR activation of BRI1 kinase,
BKI1 is phosphorylated by BRI1 and rapidly
dissociated from BRI1, allowing BAK1 associ-
ation and full activation of BRI1 (89). In sup-
port of this hypothesis, forced constitutive lo-
calization of BKI1 to the plasma membrane by
an artificial N-terminal myristoylation site en-
hanced the dwarf phenotype caused by overex-
pression of BKI1. These results demonstrated
that BKI1 provides another level of control that
keeps BRI1 at a basal level when BR levels are
low.

**SIGNALING OUTPUT FROM**
**THE RECEPTOR KINASES**

Signaling output from the BRI1–BAK1 recep-
tor complex should be mediated by intracellular
substrates of these kinases. In addition to BAK1
and BKI1, additional BRI1-interacting proteins
have been identified and shown to be phospho-
rylated by BRI1, at least in vitro. These include
the PM-localized TRANSTHYRETIN-LIKE
(TTL) protein, the homolog of TGFβ-
RECEPTOR-INTERACTING PROTEIN
1 (TRIP-1), and the BR SIGNALING
KINASES (BSKs). While the mechanism of
function remains to be established for TTL

and TRIP-1, the BSKs have been shown to
mediate signaling from BRI1 to downstream
gene expression responses.
TTL interacts with BRI1 in yeast and is
phosphorylated by BRI1 in vitro (61). Over-
expression of TTL caused dwarf phenotypes
similar to weak *bril* mutants, and a *ttl* knock-
out mutant displayed enhanced BR sensitivity
and plant growth, suggesting that TTL neg-
atively regulates BR signaling via interaction
with BRI1 kinase. TTL binds more strongly to
active BRI1 kinase than to a mutant BRI1 lack-
ing kinase activity (61). This is directly opposite
to BKI1, which binds more strongly to inactive
forms of BRI1 kinase (89). Based on the local-
ization of TTL at the plasma membrane, it was
proposed that TTL might mediate direct BR
regulation of membrane activities that control
cell elongation (61). Whether TTL interaction
with BRI1 affects downstream BR-responsive
gene expression remains unknown.
TRIP-1 shares sequence similarity with the
mammalian TGFβ receptor-interacting pro-
tein, which plays a role in TBFβ signaling but is
also an essential subunit of the eIF3 eukaryotic
translation initiation factor. TRIP-1 interacts
with BRI1 in vitro and in vivo and is phospho-
rylated by BRI1 on at least three residues (T14,
T89, and either T197 or S198) in vitro, sug-
gesting that TRIP-1 is a cytoplasmic substrate
of the BRI1 kinase. It was proposed that TRIP-
1 might play a role in BR-mediated transla-
tional regulation (18); however, direct evidence
for BR-regulation of protein translation is still
lacking. Recently, quantitative proteomics anal-
yses identified a large number of BR-regulated
proteins, but a change at the protein level with-
out a change in the level of the coding RNA has
not been found (16), except for the BR signaling
components, which are regulated by phospho-
rylation at the posttranslational level.
Using two-dimensional difference gel elec-
trophoresis, proteomic analyses of total pro-
tein samples identified only late response
proteins, but analyses of phosphoprotein or
plasma membrane fractions identified early BR-
response proteins that mediate BR signal trans-
duction (83). These include BAK1 and the

www.annualreviews.org • Brassinosteroid Signal Transduction
PM-localized BR signaling kinases (BSK1 and BSK2), which are phosphorylated upon BR treatment (84). BSKs belong to the cytoplasmic receptor-like kinase (RLCK) subfamily XII (80), which includes 12 members, all containing an N-terminal kinase domain and a C-terminal tetra trico peptide motif. In *Arabidopsis*, RLKs are a large protein family with more than 610 members, and 25% of them contain no extracellular domain or transmembrane domain and are named RLCK (80). BSK1 was phosphorylated by BRI1 but not by BAK1 in vitro and BRI1 phosphorylated a specific Ser residue (S-230) of BSK1. In vivo interaction between BRI1 and BSKs was detected by coimmunoprecipitation, and the amount of coimmunoprecipitation was reduced by BR treatment, suggesting that BSKs disassociate from BRI1 upon phosphorylation (84). Although most of the knockout mutants of BSKs did not show an obvious phenotype, the *bsk3* knockout plant showed reduced BR sensitivity and hypersensitivity to the BR biosynthetic inhibitor, BRZ. Overexpression of BSKs strongly suppressed the dwarf phenotype and restored BR-regulated gene expression in the *bri1-5* mutant. Furthermore, overexpression of BSKs partially suppressed a *bri1* null mutant but did not suppress homozygous *bin2-1*, indicating that BSKs are positive regulators acting downstream of BRI1 and upstream of BIN2 (84).

**BZR1 AND BZR2/BES1: KEY TRANSCRIPTION FACTORS FOR BR-REGULATED GENE EXPRESSION**

Genetic studies identified two homologous transcription factors as key components of the BR signaling pathway and demonstrated that BR regulates plant development by modulating the expression of nuclear genes. The *brassinazole resistant 1* (*bzr1-1D*) mutant looked like wild type grown without BRZ when grown on the BR biosynthetic inhibitor, BRZ, in contrast to the de-etiolation phenotype of wild type seedlings (93). The *bzr1-1D* mutant fully suppresses the de-etiolation phenotype and partly suppresses the dwarf phenotype of the *bri1* mutant. BZR1 is a member of a plant-specific gene family with five additional members in *Arabidopsis* (Figure 2). The Pro234 to Leu mutation in *bzr1-1D* stabilizes the protein, which causes constitutive BR responsive growth in the dark (93). The exact amino acid change P to L mutation in BZR1's closest homolog BZR2 (88% identical amino acid sequence) was found in a *bri1* suppressor mutant named *bes1-D* (*bri1-EMS-suppressor 1*), and thus BZR2 was also named BES1 (97). Overexpression of BZR1 increased hypocotyl length and a T-DNA knockout mutant *bzr2-1* shows a slight reduction in hypocotyl length (25, 93). Furthermore, RNAi

![Diagram](attachment://image.png)

**Figure 2**

The structure of the transcription factor BZR1. BZR1 contains an alanine-rich (AR) domain, nuclear localization signal (NLS), DNA binding (DB) domain, BIN2 phosphorylation domain, 14-3-3 binding motif, and PEST (proline, glutamic acid, serine, and threonine-rich) domain. Putative BIN2 phosphorylation sites are indicated by asterisks. Yellow circles containing the letter P indicate sites phosphorylated by BIN2 in vitro, and red circles indicate in vivo phosphorylation sites. (Adapted from References 16, 21; T-W. Kim and Z. Wang, unpublished data).
suppression of BZR2/BES1 and its close homologs caused a dwarf phenotype (96), indicating that BZR1 and BZR2/BES1 are positive regulators of the BR pathway with redundant roles.

The BZR1 and BZR2/BES1 proteins were recently shown to be transcription factors that directly regulate BR-responsive gene expression (25, 96). In one set of experiments, BZR1 fused to the DNA-binding domain of GAL4 was able to repress the transcription from a promoter-reporter gene containing GAL4 binding sites, demonstrating BZR1's transcription-repressor capacity. Chromatin-immunoprecipitation experiments demonstrated that BZR1 binds to the promoters of the BR-biosynthetic genes CPD and DWF4, which are rapidly suppressed by BR treatment as a mechanism for feedback regulation of BR biosynthesis. A series of in vitro DNA-binding experiments demonstrated that BZR1, through its N-terminal domain, directly bound optimally to the CGTGT/CG sequence, which is required for the CPD promoter to respond to BR and is thus named BR-response element (BRRE). Consistent with BZR1 repressing gene expression through BRRE, BRRE was found to be enriched in a large number of BR-repressed genes, including additional BR biosynthetic genes. Interestingly, BRRE is also slightly enriched in promoters of BR-induced genes. Because BZR1 plays dual roles in feedback inhibition of BR biosynthesis and BR promotion of plant growth, it is likely that BZR1 also activates certain promoters, possibly by interacting with different partners (25).

In contrast to the transcription-repressing activity of BZR1, BZR2/BES1 was shown to activate a BR-induced gene SAUR-AC1 (96). BZR2/BES1 interacts with a bHLH transcription factor named BIM1, and together they bind to the E-box (CANNTG) of the SAUR-AC1 promoter. Transient co-overexpression of BZR2/BES1 and BIM1 activates a SAUR-AC1 promoter–GUS reporter gene, indicating that BZR2/BES1 and BIM1 positively regulate SAUR-AC1 expression. While both BIM1

overexpression and knockdown experiments support a role in plant growth regulation, the phenotypes of these plants are subtle (96). It is possible that additional bHLH proteins play redundant roles or that BIMs control only a subset of BR-regulated genes and developmental processes. In fact, a recent study identified two Jumonji domain-containing proteins, ELF6 and REF6, as additional BZR2/BES1-interacting proteins. Both proteins were previously found to regulate flowering (98). Furthermore, BIM1 was shown to interact with two AP2/ERF transcription factors, DORNROSCHEN and DORNROSCHEN-LIKE, involved in embryo patterning, and the *bim1* mutant also shows embryo-patterning defects at low penetrance (9). These results suggest that BZR2/BES1 recruits other transcriptional regulators to modulate the expression of subsets of target genes and specific developmental processes.

The opposite transcription activities of BZR1 and BZR2/BES1 provide a possible explanation for the differences in the light-grown phenotypes of the *bzr1-1D* and *bes1-D* mutants. While both mutants were insensitive to BRZ and suppress *bril* mutants in the dark, they showed opposite phenotypes when grown in the light, with *bzr1-1D* being a semi-dwarf and *bes1-D* having long petioles and pale leaves, reminiscent of BR-treated plants (93, 97). The dwarf phenotype of *bzr1-1D* can be rescued by BR treatment or overexpression of a BR-biosynthetic gene and is thus at least partly caused by excessive feedback inhibition of BR synthesis (25). Opposite transcription activities would explain such opposite effects on cell elongation of the light-grown mutants. However, this would be inconsistent with the similar overall constitutive BR-response phenotype in the dark and under BR-deficient conditions. In fact, it has been reported that BZR2/BES1 also mediates feedback inhibition of CPD and DWF4 expression (57, 87) and that BZR1 also mediates activation of BR-induced genes (25). Given the almost identical amino acid sequences of BZR1 and BZR2/BES1, especially in the DNA-binding domain, it is possible that BZR1 andBZR2/BES1 have similar DNA-binding and transcription activities but are expressed in different tissues and thus have different effects on BR production in BR source tissues and on cell elongation in BR target tissues. A detailed analysis of tissue-specific expression of BZR family members will shed light on their functional specification. Moreover, identification of all the direct target genes of BZR1 and BZR2/BES1 will be important for understanding the function of the BR signaling pathway.

**REGULATION OF BZR1 AND BZR2/BES1 BY BIN2-MEDIATED PHOSPHORYLATION**

As key transcription factors of the BR signaling pathway, BZR1 and BZR2/BES1 are tightly controlled by upstream BR signaling. BR treatment induces rapid dephosphorylation of BZR1 and BZR2/BES1, which can be detected as band shift in immunoblotting (26, 93, 97). As such, BZR1 and BZR2/BES1 should be phosphorylated by a kinase that negatively regulates BR response. Such a kinase has been identified by the second BR-insensitive mutant *bin2*.

In addition to *bri1*, *bin2* (also named *dwf12* or *ucu1*) is another locus identified based on BR-insensitive dwarf phenotypes. Unlike *bri1* mutants, *bin2* mutants are due to semi-dominant gain-of-function mutations (11, 51, 66). BIN2 encodes a protein kinase that shares 70% sequence similarity to the mammalian Glycogen Synthase Kinase-3 (GSK3). GSK3s are conserved in all eukaryotes from yeast to humans and play important roles in a wide range of signaling pathways, usually as a negative regulator. Overexpression of BIN2 caused a dwarf phenotype similar to that of *bin2-1*, and loss-of-function of BIN2 and its homologs increases cell elongation and partially suppresses the *bri1-5* mutant, indicating an inhibitory role for BIN2 in BR signaling (50, 95).

Several lines of evidence demonstrate that BIN2 negatively regulates BR signaling by phosphorylating BZR1 and BZR2/BES1. First, the *bzr1-1D* and *bes1-D* mutations fully suppress the *bin2-1* mutant phenotype,

supporting the idea that BZR1 and BZR2/BES1 are downstream of BIN2 in the BR signaling pathway (93, 97). Second, BIN2 interacts directly with BZR1 and BZR2/BES1 in yeast and in vitro (93, 97, 100). Third, BIN2 effectively phosphorylates BZR1 and BZR2/BES1 in vitro, causing a mobility shift that is opposite to that caused by BR treatment of plants. Fourth, BZR1 and BZR2/BES1 are more phosphorylated and accumulate at a reduced level in the *bin2-1* mutant (93, 97). Furthermore, inhibition of BIN2 by chemical inhibitors, such as LiCl and bikinin, causes dephosphorylation of BZR1 and BZR2/BES1 (15, 95). There are about 25 putative GSK3 phosphorylation sites conserved in BZR1, BZR2/BES1, and the rice homolog OsBZR1, and these phosphorylation sites appear to provide multiple mechanisms for regulating the transcription factors.

BIN2 phosphorylation has multiple inhibitory effects on the function of BZR1 and BZR2/BES1. First, phosphorylated BZR1 undergoes 26S proteasome-mediated degradation, since treatment with the proteasome inhibitor MG132 causes accumulation of the phosphorylated BZR1 but not the dephosphorylated form. While there have been conflicting observations of whether BR treatment affects the accumulation of BZR2/BES1 (86, 97), BZR2/BES1 level is much reduced in the *bri1* and *bin2-1* mutants (97). Second, phosphorylation of BZR1 and BZR2/BES1 inhibits their DNA-binding activity. The BIN2-phosphorylated BZR1 and BZR2/BES1 fail to bind to DNA fragments derived from *CPD* or *SAUR-AC1* promoter in vitro, and the transcriptional activity of BZR2/BES1 in yeast was suppressed in the presence of BIN2 (21, 86). Third, phosphorylation of S173 causes binding to 14-3-3 proteins, which reduce nuclear localization of BZR1 and BZR2/BES1 (21).

There have been controversial observations of whether BR induces nuclear localization of BZR1 and BZR2/BES1 (21, 86, 93, 97, 100). The discrepancy is likely to reflect nonuniform responses of different cells or cells under different growth conditions (21). For example, BR rapidly increases the ratio of nuclear tocytoplasmic BZR1 in hypocotyls of young seedlings, but the effect is less pronounced in mature seedlings. Subcellular fractionation experiments revealed association of phosphorylated BZR1 with plasma membranes, suggesting retention of phospho-BZR1 in the cytoplasm (21). Furthermore, studies in both *Arabidopsis* and rice confirmed the important role of the 14-3-3 proteins in cytoplasmic localization of phosphorylated BZR factors (5, 21, 70).

Yeast two-hybrid screens identified five of the fourteen *Arabidopsis* 14-3-3 proteins as BZR1-interacting proteins and all eight rice 14-3-3 proteins as OsBZR1-interacting proteins (5, 21). 14-3-3s are phosphopeptide-binding proteins that are highly conserved in all eukaryotes (6). They are involved in numerous signaling pathways and regulate many cellular processes in yeast and mammals (6). In plants, 14-3-3 proteins are known to regulate metabolic enzymes, ion channels, and response to hormones such as abscisic acid and gibberellins (76). 14-3-3s interact with target proteins in a sequence-specific and phosphorylation-dependent manner. They bind to two types of specific sequences: mode I, RXXpSXP, or mode II, RXXXpSXP (X = any amino acid, R = arginine, pS = phosphoserine, and P = proline), in which the S residue needs to be phosphorylated (1, 58). BZR1, BZR2/BES1, and OsBZR1 all contain a conserved 14-3-3 binding motif (amino acids 169–175 of BZR1: RISNSCP), in which the required S173 of BZR1 is phosphorylated by BIN2 in vitro (5, 21). Mutations of S173A of BZR1 and S156A of OsBZR1 abolish 14-3-3 binding in vitro and in vivo. Expression of BZR1-S173A and OsBZR1-S156A in transgenic *Arabidopsis* caused constitutive BR response phenotypes similar to the *bZR1-1D* mutant, indicating that binding by the 14-3-3 proteins inhibits BZR1 function (5, 21, 70). These mutant proteins seem to have normal DNA-binding activity and stability in plant cells but show increased accumulation in the nucleus and reduced levels in the cytoplasm (5, 21, 70). Furthermore, treatment with the

14-3-3 inhibitor AICAR (5-aminoimidazole-4-carboxamide-1-b-D-ribofuranoside) also increases the nuclear accumulation of BZR1-YFP protein and reduces the expression level of BZR1-repressed target genes *CPD* and *DWF4* (21). Another study using the protoplast system also demonstrated that BIN2-mediated phosphorylation regulates nucleocytoplasmic shuttling of BZR1 and the 14-3-3 binding site and another phosphorylation domain are involved in the regulation (70). Taken together, these studies provide convincing evidence for an essential role of the phosphorylation-dependent and 14-3-3 protein-mediated cytoplasmic localization of the BZR factors in BR signal transduction. The 14-3-3 proteins are likely to increase cytoplasmic localization of phospho-BZR1 by promoting both nuclear exporting and cytoplasmic retention (21, 70).

The *Arabidopsis* genome contains 10 GSK3-like kinases, also named AtSKs (*Arabidopsis thaliana* Shaggy-like Kinases), which can be divided into four subgroups. Recent studies show that three AtSKs (AtSK21, AtSK22, and AtSK23/BIN2) belonging to subgroup II function redundantly in BR signaling. Their triple loss-of-function mutant shows elongated petiole and BRZ-resistant phenotype. However, considerable amounts of phosphorylated BZR2/BES1 are still present in the triple mutant, indicating that other AtSKs might be involved in BR signaling (86, 95). Indeed, yeast two-hybrid assay using nine AtSKs covering all four subgroups shows that all six AtSKs belonging to subgroup I and II bind to BZR1 (42). Further study of AtSK12, a member of subgroup I, demonstrated that AtSK12 interacts with BZR1 in vivo and phosphorylates BZR1 in vitro and is inactivated by BR treatment (42). Thus, it is believed that at least six AtSKs can play negative roles in BR signaling by phosphorylating BZR1.

In addition to BZR1 and BZR2/BES1, BIN2 also phosphorylates the ARF2 transcription factor (88). ARF2 is a transcriptional repressor that negatively regulates BR and auxin responses, possibly by competing with activator ARFs that promote auxin-induced gene expression.
Loss-of-function *arf2* mutant shows reduced sensitivity to BR inhibitor, BRZ, suggesting a negative role of ARF2 in BR-mediated growth. BIN2 alleviated ARF2 transcriptional repression activity and inhibited its DNA-binding activity in vitro (88). It was proposed that BIN2 inactivates ARF2 to promote BR and auxin (88). However, as a negative regulator of BR signaling, BIN2 is inactivated by BR (42, 65), which would lead to BR activation of ARF2. This seems contradictory to the negative role of ARF2 in hypocotyl elongation but would be consistent with ARF2's positive role in promoting stamen elongation and leaf senescence (19). The mechanism of ARF2 regulation by BR signaling and the function of ARF2 in BR response require further study.

**REGULATION OF BIN2 BY BSU1-MEDIATED TYROSINE DEPHOSPHORYLATION AND PROTEASOME-MEDIATED DEGRADATION**

It is believed that BR induces dephosphorylation of BZR factors by inhibiting BIN2, but how BIN2 is inactivated by upstream BR signaling has remained an outstanding question. In animal systems, GSK3β plays a role in Wnt signaling similar to BIN2's role in BR signaling. GSK3β phosphorylates β-catenin to promote its degradation and cytoplasmic retention, similar to BIN2 regulation of BZR1/2. In mammals, GSK3β action is modulated by phosphorylation in its N-terminal domain, protein complex formation, and by priming phosphorylation of its substrate (54). However, these mechanisms seem not to be used in BIN2 regulation of the BR signaling pathway (48). A 12-amino-acid BIN2-binding motif in BZR1/2 functions as a docking site for BIN2, though it remains unclear whether BR regulates BIN2-BZR1/2 binding (48). Meanwhile, Peng and coworkers (65) have reported that BR induces rapid degradation of BIN2, which is blocked by MG132, the inhibitor of the 26S proteasome. Interestingly, the *bin2-1* mutant protein accumulates in plants and is not degraded upon BR treatment, providing genetic evidence for an important role for BIN2 degradation by the proteasome in BR signaling. However, the primary mechanism that triggers BIN2 degradation was not identified (65).

Recently, Kim and coworkers (42) detected BR-induced dephosphorylation of BIN2 using immuno-blotting following two-dimensional gel electrophoresis. This result indicated that a protein phosphatase might be involved in the regulation of BIN2. This phosphatase turned out to be BSU1 (bri1 SUPPRESSOR 1), which was initially thought to mediate dephosphorylation of BZR2/BES1.

BSU1 was identified by activation tagging screening for suppressors of a weak *bri1-5* allele (57). BSU1 contains an N-terminal kelch-repeat domain and a C-terminal phosphatase domain homologous to protein phosphatase 1 (PP1). Overexpression of BSU1 suppresses a *bri1* mutant and causes accumulation of dephosphorylated BZR2/BES1. RNAi suppression of BSU1 homologs caused a weak dwarf phenotype (57). Quadruple loss-of-function of BSU1 family members, by T-DNA knockout of *bsu1 bsl1*, and RNAi suppression of *BSL2* and *BSL3* caused an extreme dwarf phenotype similar to the *bri1* null mutant, confirming a positive role of BSU1 in BR signaling (42).

When BZR1 or BZR2/BES1 and BIN2 were mixed with BSU1 in vitro, BZR1 or BZR2/BES1 phosphorylation was reduced by BSU1. However, the effect of BSU1 in the in vitro assay was mild, and it was proposed that BSU1 might dephosphorylate BZR1 and BZR2/BES1 to activate BR responses in vivo but need to be activated by posttranslational modification or by a partner protein (46, 57). However, Kim and coworkers recently found that recombinant BSU1 or the immunopurified plant BSU1 does not dephosphorylate BZR1 that was prephosphorylated by BIN2, if BIN2 was removed before adding BSU1 (42). Reduced phosphorylation of BZR1 and BZR2/BES1 was observed only when BIN2 was present with BSU1, as reported

originally (42, 57). Using BZR1 incompletely phosphorylated by \[^{32}\]P-ATP to distinguish dephosphorylation from further phosphorylation by cold ATP, it was shown that BSU1 inhibits BIN2 phosphorylation but does not dephosphorylate BZR1 directly. Furthermore, BSU1 inhibition of wild type BIN2 is increased by BR treatment of the plants from which BSU1 was immunopurified, suggesting that BR signaling activates BSU1. BSU1 fails to inhibit phosphorylation of BZR1 by the mutant bin2-1, which causes BR-insensitive phenotypes in planta. Similar to BR treatment, overexpression of BSU1 reduces the level of BIN2 protein but not of bin2-1 (42). BSU1 overexpression suppresses *bri1* null allele but not homozygous *bin2-1*, although it partly suppresses the dwarf phenotype of heterozygous *bin2-1*. These results demonstrate that BSU1 functions downstream of BRI1 but upstream of BIN2 (42). It was further shown that BSU1 interacts with BIN2 in vitro and in vivo (42).

Mass spectrometry analysis of recombinant BIN2 identified Tyr200 as an autophosphorylation site (42). Tyr200 is absolutely conserved in all GSK3s and phosphorylation at this site is required for kinase activity of BIN2 as it is for mammalian GSK3s. The Tyr200-Phe substitution abolished the ability of BIN2 and *bin2-1* to cause a dwarf phenotype when overexpressed in transgenic *Arabidopsis*, suggesting that phosphorylation of Tyr200 is essential for BIN2 function in vivo. The level of Tyr200 phosphorylation of wild type BIN2 protein was reduced by incubation with BSU1 in vitro and by BR treatment of plants in vivo (42). Interestingly, Tyr200 phosphorylation of mutant *bin2-1* protein was not affected by BSU1 in vitro or by BR treatment in vivo, indicating that the primary effect of the *bin2-1* mutation is to block BSU1 dephosphorylation of Tyr200. These results further demonstrate that BSU1-mediated dephosphorylation of Tyr200 is the primary mechanism by which BR signaling inactivates BIN2 (42). It was also shown that Tyr233 of AtSK12 (corresponding to Tyr200 of BIN2) is also dephosphorylated upon BR treatment.

Thus, BSU1-mediated Tyr200 dephosphorylation appears to be a general mechanism for inactivating GSK3s in BR signaling (42).

Placing BSU1 upstream of BIN2 raised a possibility that BSU1 might directly interact with BSKs at the plasma membrane, which would close the last gap in the BR signaling pathway. Although BSU1 is primarily localized in the nucleus, small amounts of BSU1 can be observed in the cytoplasm (42, 57). Microscopic observation and proteomic studies revealed that the BSU1 homologs (BSL1, BSL2, and BSL3) are localized at the plasma membrane and in the cytoplasm, raising the possibility of direct interaction with components localized at the plasma membrane (42). Indeed, in vitro interaction assays showed that BSU1 specifically interacts with BSK1 but not with BRI1 and BAK1 (42). This interaction was increased by BRI1 phosphorylation of BSK1 and abolished by mutation of the BRI1 phosphorylation site in BSK1 (S230A). In vivo interaction between BSU1 and BSK1 was also confirmed by coimmunoprecipitation and BiFC assays (42). These results show that BRI1 phosphorylation of S230 of BSK1 promotes interaction with BSU1 (42). As such, all major BR signaling components from cell surface receptor BRI1 kinase to nuclear transcription factors are now physically connected. The completed BR signaling pathway illustrates the molecular details of sequential signaling events: BR binding to BRI1 at the cell surface, BRI1 kinase activation by ligand-induced association and transphosphorylation with the BAK1 coreceptor and dissociation of the BKI1 inhibitor, BRI1 phosphorylation of BSKs, activation of BSU1, inactivation of BIN2 by BSU1-catalyzed tyrosine dephosphorylation, and nuclear accumulation of dephosphorylated BZR factors and BR-responsive gene expression (42).

As the first complete signal transduction pathway mediated by a receptor kinase in plants (Figure 3), the BR signaling pathway provides a paradigm for understanding plant signal transduction. Many signaling mechanisms revealed in the BR pathway are likely used

a

BAK1
BR1
Plasma membrane
BSU1
14-3-3
Proteasomal degradation
Nucleus
BZR1/2-target genes
Cytoplasm

b

BAK1
BRI1
BL
HO
HO
Plasma membrane
BK1
BSKS
BSU1
14-3-3
BZR1/2
Nucleus
Unknown factor
BZR1/2-target genes
Cytoplasm
Gene expression and cell growth
Proteasomal degradation
BIN2
BIN2
BIN2
BIN2
broadly in other pathways. For example, the mechanism of BRI1-BAK1 kinase activation by ligand induced oligomerization and transphosphorylation is likely shared by other RLKs. The regulation of GSK3-like kinases through tyrosine dephosphorylation by BSU1 (a PP1-related phosphatase) is likely a conserved mechanism used in mammals, because phosphorylation of the same tyrosine residue also affects the activity of mammalian GSK3s and PP1 also plays a role in the Wnt signaling pathway upstream of GSK3β. The regulation of BIN2 and BRI1 by tyrosine phosphorylation clearly establishes the role of tyrosine phosphorylation as a common mechanism of signal transduction in plants. Although homologs of animal tyrosine kinases have not been found in plants, many plant kinases seem to have dual specificity for Ser/Thr and Tyr.

**CONCLUDING REMARKS**

The combination of genetics, biochemistry, and proteomics approaches has contributed to rapid progress toward a deep understanding of the BR signaling mechanism. The completed BR signaling pathway illustrates the molecular details of sequential signaling events: BR binding to BRI1 at the cell surface, BRI1 kinase activation by ligand-induced association and transphosphorylation with the BAK1 coreceptor and dissociation of the BKI1 inhibitor, BRI1 phosphorylation of BSKs, activation of BSU1, inactivation of BIN2 by BSU1-catalyzed tyrosine dephosphorylation, and nuclear accumulation of dephosphorylated BZR factors and BR-responsive gene expression. However, many more questions are yet to be answered. For example, how is BSU1 activity regulated by BSKs? What kind of phosphatase dephosphorylates BZR1/BES1 and how is it regulated by BR signaling? Do any BR signaling kinases and phosphatases directly regulate any cellular processes independent of the transcriptional pathway? Yet the more exciting and challenging question is how such a linear signaling pathway regulates diverse developmental and physiological responses. Identification of direct target genes of BZR1 and BZR2/BES1 is key for identifying downstream branch pathways leading to specific responses and for dissecting the transcriptional network that integrates with other signaling pathways. Ultimately, we will need to understand the operation of the BR signaling pathway in the context of tissue, organ, and whole plant development (44, 45, 71). Most important though, the knowledge of BR function needs to be applied to genetic engineering that improves food and bio-energy production.

---

Figure 3

The brassinosteroid signal transduction pathway. The signaling components in active and inactive status are shown in pink and blue, respectively. Phosphorylation confers positive effects (*yellow circles*) and negative effects (*red circles*). (*a*) In the absence of BR, BKI1 interacts with inactive forms of BRI1, leading to interruption of BRI1 binding to BAK1. BRI1-bound BSKs are kept in inactive status. Consequently, BSU1 is inactive. Active BIN2 (harboring phospho-tyrosine at the 200th amino acid residue) constitutively phosphorylates BZR1 and BZR2/BES1, leading to nuclear exporting and cytoplasmic retention by the 14-3-3 proteins, loss of DNA-binding activity, and proteasomal degradation of BZR1 and BZR2/BES1. (*b*) In the presence of BR, BR binds to BRI1, which induces association with BAK1 and disassociation of BKI1. By sequential transphosphorylation between BRI1 and BAK1, BRI1 is fully activated and phosphorylates BSKs. Then the phosphorylated BSKs are released from the receptor complex and bind to BSU1, presumably to enhance BSU1 activity. Activated BSU1 inhibits BIN2 through dephosphorylation of the phospho-tyrosine residue of BIN2, which allows accumulation of unphosphorylated BZR1 and BZR2/BES1 in the nucleus. Active BZR1 and BZR2/BES1 bind to genomic DNA to regulate BR-target gene expression, thereby modulating growth and development of plants.
DISCLOSURE STATEMENT

The authors are not aware of any affiliations, memberships, funding, or financial holdings that might be perceived as affecting the objectivity of this review.

LITERATURE CITED

1. Aitken A. 2006. 14-3-3 proteins: a historic overview. *Semin. Cancer Biol.* 16:162–72
2. Albrecht C, Russinova E, Hecht V, Baaijens E, de Vries S. 2005. The *Arabidopsis thaliana* SOMATIC EMBRYOGENESIS RECEPTOR-LIKE KINASES1 and 2 control male sporogenesis. *Plant Cell* 17: 3337–49
3. Albrecht C, Russinova E, Kemmerling B, Kwaaitaal M, de Vries SC. 2008. *Arabidopsis* SOMATIC EMBRYOGENESIS RECEPTOR KINASE proteins serve brassinosteroid-dependent and -independent signaling pathways. *Plant Physiol.* 148:611–19
4. Astle S, Khan RN, Thornton S. 2003. The effects of a progesterone metabolite, 5 beta-dihydroprogesterone, on oxytocin receptor binding in human myometrial membranes. *BJOG* 110:589–92
5. Bai MY, Zhang LY, Gampala SS, Zhu SW, Song WY, et al. 2007. Functions of OsBZR1 and 14-3-3 proteins in brassinosteroid signaling in rice. *Proc. Natl. Acad. Sci. USA* 104:13839–44
6. Bridges D, Moorhead GB. 2005. 14-3-3 proteins: a number of functions for a numbered protein. *Sci. STKE* 2005:re10
7. Burger K, Fahrenholz F, Gimpl G. 1999. Non-genomic effects of progesterone on the signaling function of G protein-coupled receptors. *FEBS Lett.* 464:25–29
8. Cano-Delgado A, Yin Y, Yu C, Vafeados D, Mora-Garcia S, et al. 2004. BRL1 and BRL3 are novel brassinosteroid receptors that function in vascular differentiation in *Arabidopsis*. *Development* 131:5341–51
9. Chandler JW, Cole M, Flier A, Werr W. 2009. BIM1, a bHLH protein involved in brassinosteroid signaling, controls *Arabidopsis* embryonic patterning via interaction with DORNROSCHEN and DORNROSCHEN-LIKE. *Plant Mol. Biol.* 69:57–68
10. Chinchilla D, Zipfel C, Robatzek S, Kemmerling B, Nurnberger T, et al. 2007. A flagellin-induced complex of the receptor FLS2 and BAK1 initiates plant defense. *Nature* 448:497–500
11. Choe S, Schmitz RJ, Fujioka S, Takatsuto S, Lee MO, et al. 2002. *Arabidopsis* brassinosteroid-insensitive dwarf12 mutants are semidominant and defective in a glycogen synthase kinase 3beta-like kinase. *Plant Physiol.* 130:1506–15
12. Clouse SD, Langford M, McMorris TC. 1996. A brassinosteroid-insensitive mutant in *Arabidopsis thaliana* exhibits multiple defects in growth and development. *Plant Physiol.* 111:671–78
13. Clouse SD, Sasse JM. 1998. BRASSINOSTEROIDS: essential regulators of plant growth and development. *Annu. Rev. Plant Physiol. Plant. Mol. Biol.* 49:427–51
14. Colcombet J, Boisson-Dernier A, Ros-Palau R, Vera CE, Schroeder JI. 2005. *Arabidopsis* SOMATIC EMBRYOGENESIS RECEPTOR KINASES1 and 2 are essential for tapetum development and microspore maturation. *Plant Cell* 17:3350–61
15. De Rybel B, Audenaert D, Vert G, Rozhon W, Mayerhofer J, et al. 2009. Chemical inhibition of a subset of *Arabidopsis thaliana* GSK3-like kinases activates brassinosteroid signaling. *Chem. Biol.* 16:594–604
16. Deng Z, Zhang X, Tang W, Oses-Prieto JA, Suzuki N, et al. 2007. A proteomics study of brassinosteroid response in *Arabidopsis*. *Mol. Cell Proteomics* 6:2058–71
17. Ding ZWH, Liang X, Morris ER, Gallazzzi F, Pandit S, et al. 2007. Phosphoprotein and phosphopeptide interactions with the FHA domain from *Arabidopsis* kinase-associated protein phosphatase. *Biochemistry* 46:2684–96
18. Ehsan H, Ray WK, Phinney B, Wang X, Huber SC, Clouse SD. 2005. Interaction of *Arabidopsis* BRASSINOSTEROID-INSENSITIVE 1 receptor kinase with a homolog of mammalian TGF-beta receptor interacting protein. *Plant J.* 43:251–61
19. Ellis CM, Nagpal P, Young JC, Hagen G, Guilfoyle TJ, Reed JW. 2005. AUXIN RESPONSE FACTOR1 and AUXIN RESPONSE FACTOR2 regulate senescence and floral organ abscission in *Arabidopsis thaliana*. *Development* 132:4563–74

20. Friedrichsen DM, Joazeiro CA, Li J, Hunter T, Chory J. 2000. Brassinosteroid-insensitive-1 is a ubiquitously expressed leucine-rich repeat receptor serine/threonine kinase. *Plant Physiol.* 123:1247–56

21. Gampala SS, Kim TW, He JX, Tang W, Deng Z, et al. 2007. An essential role for 14-3-3 proteins in brassinosteroid signal transduction in *Arabidopsis*. *Dev. Cell* 13:177–89

22. Geldner N, Hyman DL, Wang X, Schumacher K, Chory J. 2007. Endosomal signaling of plant steroid receptor kinase BRI1. *Genes Dev.* 21:1598–602

23. Grazzini E, Guillot G, Mouillac B, Zingg HH. 1998. Inhibition of oxytocin receptor function by direct binding of progesterone. *Nature* 392:509–12

24. Grove MD, Spencer GF, Rohwedder WK, Mandava NB, Worley JF, et al. 1979. Brassinolide, a plant growth-promoting steroid isolated from *Brassica napus* pollen. *Nature* 281:216–17

25. He J-X, Gendron JM, Sun Y, Gampala SS, Gendron N, et al. 2005. BZR1 is a transcriptional repressor with dual roles in brassinosteroid homeostasis and growth responses. *Science* 307:1634–38

26. He JX, Gendron JM, Yang Y, Li J, Wang ZY. 2002. The GSK3-like kinase BIN2 phosphorylates and destabilizes BZR1, a positive regulator of the brassinosteroid signaling pathway in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 99:10185–90

27. He K, Gou X, Powell RA, Yang H, Yuan T, et al. 2008. Receptor-like protein kinases, BAK1 and BKK1, regulate a light-dependent cell-death control pathway. *Plant Signal Behav.* 3:813–15

28. He K, Gou X, Yuan T, Lin H, Asami T, et al. 2007. BAK1 and BKK1 Regulate Brassinosteroid-Dependent Growth and Brassinosteroid-Independent Cell-Death Pathways. *Curr. Biol.* 17:1109–15

29. He Z, Wang ZY, Li J, Zhu Q, Lamb C, et al. 2000. Perception of brassinosteroids by the extracellular domain of the receptor kinase BRI1. *Science* 288:2360–63

30. Hicke L, Zanolari B, Riezman H. 1998. Cytoplasmic tail phosphorylation of the alpha-factor receptor is required for its ubiquitination and internalization. *J. Cell Biol.* 141:349–58

31. Hink MA, Shah K, Russinova E, de Vries SC, Visser AJ. 2008. Fluorescence fluctuation analysis of *Arabidopsis thaliana* somatic embryogenesis receptor-like kinase and brassinosteroid insensitive 1 receptor oligomerization. *Biophys. J.* 94:1052–62

32. Holton N, Cano-Delgado A, Harrison K, Montoya T, Chory J, Bishop GJ. 2007. Tomato BRASSINOSTEROID INSENSITIVE1 is required for systemin-induced root elongation in *Solanum pimpinellifolium* but is not essential for wound signaling. *Plant Cell* 19:1709–17

33. Holton N, Harrison K, Yokota T, Bishop GJ. 2008. Tomato BRI1 and systemin wound signaling. *Plant Signal Behav.* 3:54–55

34. Hong Z, Jin H, Tzfira T, Li J. 2008. Multiple mechanism-mediated retention of a defective brassinosteroid receptor in the endoplasmic reticulum of *Arabidopsis*. *Plant Cell* 20:3418–29

35. Hubbard SR. 2004. Juxtamembrane autoinhibition in receptor tyrosine kinases. *Nat. Rev. Mol. Cell Biol.* 5:464–71

36. Huse M, Muir TW, Xu L, Chen YG, Kuriyan J, Massague J. 2001. The TGF beta receptor activation process: an inhibitor-to substrate-binding switch. *Mol. Cell* 8:671–82

37. Jin H, Yan Z, Nam KH, Li J. 2007. Allele-specific suppression of a defective brassinosteroid receptor reveals a physiological role of UGGT in ER quality control. *Mol. Cell* 26:821–30

38. Karlova R, Boeren S, Russinova E, Aker J, Vervoort J, de Vries S. 2006. The *Arabidopsis* SOMATIC EMBRYOGENESIS RECEPTOR-LIKE KINASE1 Protein Complex Includes BRASSINOSTEROID-INSENSITIVE1. *Plant Cell* 18:626–38

39. Karlova R, Boeren S, van Dongen W, Kwaaitaal M, Aker J, et al. 2009. Identification of in vitro phosphorylation sites in the *Arabidopsis thaliana* somatic embryogenesis receptor-like kinases. *Proteomics* 9:368–79

40. Kauschmann A, Jessop A, Koncz C, Szekeres M, Willmitzer L, Altmann T. 1996. Genetic evidence for an essential role of brassinosteroids in plant development. *Plant J.* 9:701–13

41. Kemmerling B, Schwedt A, Rodriguez P, Mazzotta S, Frank M, et al. 2007. The BRI1-Associated Kinase 1, BAK1, has a brassinolide-independent role in plant cell-death control. *Curr. Biol.* 17:1116–22

42. Kim T-W, Guan S, Sun Y, Deng Z, Tang W, et al. 2009. Brassinosteroid signal transduction from cell surface receptor kinases to nuclear transcription factors. *Nat. Cell Biol.* 11:1254–60

43. Kinoshita T, Cano-Delgado A, Seto H, Hiranuma S, Fujioka S, et al. 2005. Binding of brassinosteroids to the extracellular domain of plant receptor kinase BRI1. *Nature* 433:167–71

44. Kuppusamy KT, Chen AY, Nemhauser JL. 2009. Steroids are required for epidermal cell fate establishment in *Arabidopsis* roots. *Proc. Natl. Acad. Sci. USA* 106:8073–76

45. Kutschera U, Niklas KJ. 2007. The epidermal-growth-control theory of stem elongation: an old and a new perspective. *J. Plant Physiol.* 164:1395–409

46. Li J. 2005. Brassinosteroid signaling: from receptor kinases to transcription factors. *Curr. Opin. Plant Biol.* 8:526–31

47. Li J, Chory J. 1997. A putative leucine-rich repeat receptor kinase involved in brassinosteroid signal transduction. *Cell* 90:929–38

48. Li J, Jin H. 2007. Regulation of brassinosteroid signaling. *Trends Plant Sci.* 12:37–41

49. Li J, Nagpal P, Vitart V, McMorris TC, Chory J. 1996. A role for brassinosteroids in light-dependent development of *Arabidopsis*. *Science* 272:398–401

50. Li J, Nam KH. 2002. Regulation of brassinosteroid signaling by a GSK3/SHAGGY-like kinase. *Science* 295:1299–301

51. Li J, Nam KH, Vafeados D, Chory J. 2001. BIN2, a new brassinosteroid-insensitive locus in *Arabidopsis*. *Plant Physiol.* 127:14–22

52. Li J, Wen J, Lease KA, Doke JT, Tax FE, Walker JC. 2002. BAK1, an *Arabidopsis* LRR receptor-like protein kinase, interacts with BRI1 and modulates brassinosteroid signaling. *Cell* 110:213–22

53. Li L, Yu X, Thompson A, Guo M, Yoshida S, et al. 2009. *Arabidopsis* MYB30 is a direct target of BES1 and cooperates with BES1 to regulate brassinosteroid-induced gene expression. *Plant J.* 58:275–86

54. MacDonald BT, Tamai K, He X. 2009. Wnt/beta-catenin signaling: components, mechanisms, and diseases. *Dev. Cell* 17:9–26

55. Malinowski R, Higgins R, Luo Y, Piper L, Nazir A, et al. 2009. The tomato brassinosteroid receptor BRI1 increases binding of systemin to tobacco plasma membranes, but is not involved in systemin signaling. *Plant Mol. Biol.* 70:603–16

56. Miaczynska M, Pelkmans L, Zerial M. 2004. Not just a sink: endosomes in control of signal transduction. *Curr. Opin. Cell Biol.* 16:400–6

57. Mora-Garcia S, Vert G, Yin Y, Cano-Delgado A, Cheong H, Chory J. 2004. Nuclear protein phosphatases with Kelch-repeat domains modulate the response to brassinosteroids in *Arabidopsis*. *Genes Dev.* 18:448–60

58. Muslin AJ, Tanner JW, Allen PM, Shaw AS. 1996. Interaction of 14-3-3 with signaling proteins is mediated by the recognition of phosphoserine. *Cell* 84:889–97

59. Nakamura A, Fujioka S, Sunohara H, Kamiya N, Hong Z, et al. 2006. The role of OsBRI1 and its homologous genes, OsBRL1 and OsBRL3 in rice. *Plant Physiol.* 140:580–90

60. Nam KH, Li J. 2002. BRI1/BAK1, a receptor kinase pair mediating brassinosteroid signaling. *Cell* 110:203–12

61. Nam KH, Li J. 2004. The *Arabidopsis* transthyretin-like protein is a potential substrate of BRASSINOSTEROID-INSENSITIVE 1. *Plant Cell* 16:2406–17

62. Nomura T, Bishop GJ, Kaneta T, Reid JB, Chory J, Yokota T. 2003. The LKA gene is a BRASSINOSTEROID INSENSITIVE 1 homolog of pea. *Plant J.* 36:291–300

63. Oh MH, Ray WK, Huber SC, Asara JM, Gauge DA, Clouse SD. 2000. Recombinant brassinosteroid insensitive 1 receptor-like kinase autophosphorylates on serine and threonine residues and phosphorylates a conserved peptide motif in vitro. *Plant Physiol.* 124:751–66

64. Oh MH, Wang X, Kota U, Goshe MB, Clouse SD, Huber SC. 2009. Tyrosine phosphorylation of the BRI1 receptor kinase emerges as a component of brassinosteroid signaling in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 106:658–63

65. Peng P, Yan Z, Zhu Y, Li J. 2008. Regulation of the *Arabidopsis* GSK3-like kinase BRASSINOSTEROID-INSENSITIVE 2 through proteasome-mediated protein degradation. *Mol. Plant* 1:338–46

66. Perez-Perez JM, Ponce MR, Micol JL. 2002. The UCU1 *Arabidopsis* gene encodes a SHAGGY/GSK3-like kinase required for cell expansion along the proximodistal axis. *Dev. Biol.* 242:161–73

67. Rahimi RA, Leof EB. 2007. TGF-beta signaling: a tale of two responses. *J. Cell Biochem.* 102:593–608

68. Robert S, Chary SN, Drakakaki G, Li S, Yang Z, et al. 2008. Endosidin1 defines a compartment involved in endocytosis of the brassinosteroid receptor BRI1 and the auxin transporters PIN2 and AUX1. *Proc. Natl. Acad. Sci. USA* 105:8464–69

69. Russinova E, Borst JW, Kwaaitaal M, Cano-Delgado A, Yin Y, et al. 2004. Heterodimerization and endocytosis of *Arabidopsis* brassinosteroid receptors BRI1 and AtSERK3 (BAK1). *Plant Cell* 16:3216–29

70. Ryu H, Kim K, Cho H, Park J, Choe S, Hwang I. 2007. Nucleocytoplasmic shuttling of BZR1 mediated by phosphorylation is essential in *Arabidopsis* brassinosteroid signaling. *Plant Cell* 19:2749–62

71. Savaldi-Goldstein S, Peto C, Chory J. 2007. The epidermis both drives and restricts plant shoot growth. *Nature* 446:199–202

72. Scheer JM, Pearce G, Ryan CA. 2003. Generation of systemin signaling in tobacco by transformation with the tomato systemin receptor kinase gene. *Proc. Natl. Acad. Sci. USA* 100:10114–17

73. Scheer JM, Ryan CA Jr. 2002. The systemin receptor SR160 from *Lycopersicon peruvianum* is a member of the LRR receptor kinase family. *Proc. Natl. Acad. Sci. USA* 99:9585–90

74. Schlessinger J. 2002. Ligand-induced, receptor-mediated dimerization and activation of EGF receptor. *Cell* 110:669–72

75. Schmidt ED, Guzzo F, Toonen MA, de Vries SC. 1997. A leucine-rich repeat containing receptor-like kinase marks somatic plant cells competent to form embryos. *Development* 124:2049–62

76. Schoonheim PJ, Veiga H, da Costa Pereira D, Friso G, van Wijk KJ, de Boer AH. 2007. A comprehensive analysis of the 14-3-3 interactome in barley leaves using a complementary proteomics and two-hybrid approach. *Plant Physiol.* 143:670–83

77. Shah K, Russinova E, Gadella TW Jr, Willemse J, de Vries SC. 2002. The *Arabidopsis* kinase-associated protein phosphatase controls internalization of the somatic embryogenesis receptor kinase 1. *Genes Dev.* 16:1707–20

78. Shan L, He P, Li J, Heese A, Peck SC, et al. 2008. Bacterial effectors target the common signaling partner BAK1 to disrupt multiple MAMP receptor-signaling complexes and impede plant immunity. *Cell Host Microbe* 4:17–27

79. Shiu SH, Bleecker AB. 2001. Plant receptor-like kinase gene family: diversity, function, and signaling. *Sci. STKE* 113:re22

80. Shiu SH, Karlowski WM, Pan R, Tzeng YH, Mayer KF, Li WH. 2004. Comparative analysis of the receptor-like kinase family in *Arabidopsis* and rice. *Plant Cell* 16:1220–34

81. Szekeres M, Nemeth K, Koncz-Kalman Z, Mathur J, Kauschmann A, et al. 1996. Brassinosteroids rescue the deficiency of CYP90, a cytochrome P450, controlling cell elongation and de-etiolation in *Arabidopsis*. *Cell* 85:171–82

82. Tan X, Calderon-Villalobos LI, Sharon M, Zheng C, Robinson CV, et al. 2007. Mechanism of auxin perception by the TIR1 ubiquitin ligase. *Nature* 446:640–45

83. Tang W, Deng Z, Oses-Prieto JA, Suzuki N, Zhu SW, et al. 2008. Proteomic studies of brassinosteroid signal transduction using prefractionation and two-dimensional DIGE. *Mol. Cell Proteomics* 7:728–38

84. Tang W, Kim TW, Oses-Prieto JA, Sun Y, Deng Z, et al. 2008. BSks mediate signal transduction from the receptor kinase BRI1 in *Arabidopsis*. *Science* 321:557–60

85. Thummel CS, Chory J. 2002. Steroid signaling in plants and insects—common themes, different pathways. *Genes Dev.* 16:3113–29

86. Vert G, Chory J. 2006. Downstream nuclear events in brassinosteroid signaling. *Nature* 441:96–100

87. Vert G, Nemhauser JL, Geldner N, Hong F, Chory J. 2005. Molecular mechanisms of steroid hormone signaling in plants. *Annu. Rev. Cell Dev. Biol.* 21:177–201

88. Vert G, Walcher CL, Chory J, Nemhauser JL. 2008. Integration of auxin and brassinosteroid pathways by Auxin Response Factor 2. *Proc. Natl. Acad. Sci. USA* 105:9829–34

89. Wang X, Chory J. 2006. Brassinosteroids regulate dissociation of BKI1, a negative regulator of BRI1 signaling, from the plasma membrane. *Science* 313:1118–22

90. Wang X, Goshe MB, Soderblom EJ, Pinney BS, Kuchar JA, et al. 2005. Identification and functional analysis of in vivo phosphorylation sites of the *Arabidopsis* BRASSINOSTEROID-INSENSITIVE 1 receptor kinase. *Plant Cell* 17:1685–703

91. Wang X, Kota U, He K, Blackburn K, Li J, et al. 2008. Sequential transphosphorylation of the BRI1/BAK1 receptor kinase complex impacts early events in brassinosteroid signaling. *Dev. Cell* 15:220–35

92. Wang X, Li X, Meisenhelder J, Hunter T, Yoshida S, et al. 2005. Autoregulation and homodimerization are involved in the activation of the plant steroid receptor BRI1. *Dev. Cell* 8:855–65

93. Wang ZY, Nakano T, Gendron J, He J, Chen M, et al. 2002. Nuclear-localized BZR1 mediates brassinosteroid-induced growth and feedback suppression of brassinosteroid biosynthesis. *Dev. Cell* 2:505–13

94. Wang ZY, Seto H, Fujioka S, Yoshida S, Chory J. 2001. BRI1 is a critical component of a plasma-membrane receptor for plant steroids. *Nature* 410:380–83

95. Yan Z, Zhao J, Peng P, Chihara RK, Li J. 2009. BIN2 functions redundantly with other *Arabidopsis* GSK3-like kinases to regulate brassinosteroid signaling. *Plant Physiol.* 150:710–21

96. Yin Y, Vafeados D, Tao Y, Yoshida S, Asami T, Chory J. 2005. A new class of transcription factors mediates brassinosteroid-regulated gene expression in *Arabidopsis*. *Cell* 120:249–59

97. Yin Y, Wang ZY, Mora-Garcia S, Li J, Yoshida S, et al. 2002. BES1 accumulates in the nucleus in response to brassinosteroids to regulate gene expression and promote stem elongation. *Cell* 109:181–91

98. Yu X, Li L, Guo M, Chory J, Yin Y. 2008. Modulation of brassinosteroid-regulated gene expression by Jumonji domain-containing proteins ELF6 and REF6 in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA* 105:7618–23

99. Yun HS, Bae YH, Lee YJ, Chang SC, Kim SK, et al. 2009. Analysis of phosphorylation of the BRI1/BAK1 complex in *Arabidopsis* reveals amino acid residues critical for receptor formation and activation of BR signaling. *Mol. Cell* 27:183–90

100. Zhao J, Peng P, Schmitz RJ, Decker AD, Tax FE, Li J. 2002. Two putative BIN2 substrates are nuclear components of brassinosteroid signaling. *Plant Physiol.* 130:1221–29

101. Zhou A, Wang H, Walker JC, Li J. 2004. BRL1, a leucine-rich repeat receptor-like protein kinase, is functionally redundant with BRI1 in regulating *Arabidopsis* brassinosteroid signaling. *Plant J.* 40:399–409

AR410-FM ARI 6 April 2010 15:25

Annual Review of  
Plant Biology  
Volume 61, 2010

Contents

A Wandering Pathway in Plant Biology: From Wildflowers to  
Phototropins to Bacterial Virulence  
Winslow R. Briggs . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Evolution in Action: Plants Resistant to Herbicides  
Stephen B. Powles and Qin Yu ………………………………………… 317  

Insights from the Comparison of Plant Genome Sequences  
Andrew H. Paterson, Michael Freeling, Haibao Tang, and Xiyin Wang ………… 349  

High-Throughput Characterization of Plant Gene Functions by Using  
Gain-of-Function Technology  
Youichi Kondou, Mieko Higuchi, and Minami Matsui ………………………… 373  

Histone Methylation in Higher Plants  
Chunyan Liu, Falong Lu, Xia Cui, and Xiaofeng Cao ………………………… 395  

Genetic and Molecular Basis of Rice Yield  
Yongzhong Xing and Qifa Zhang ………………………………………… 421  

Genetic Engineering for Modern Agriculture: Challenges and  
Perspectives  
Ron Mittler and Eduardo Blumwald ………………………………………… 443  

Metabolomics for Functional Genomics, Systems Biology, and  
Biotechnology  
Kazuki Saito and Fumio Matsuda ………………………………………… 463  

Quantitation in Mass-Spectrometry-Based Proteomics  
Waltraud X. Schulze and Björn Usadel …………………………………… 491  

Metal Hyperaccumulation in Plants  
Ute Krämer ……………………………………………………………… 517  

Arsenic as a Food Chain Contaminant: Mechanisms of Plant Uptake  
and Metabolism and Mitigation Strategies  
Fang-Jie Zhao, Steve P. McGrath, and Andrew A. Meharg …………………… 535  

Guard Cell Signal Transduction Network: Advances in Understanding  
Abscisic Acid, CO₂, and Ca²⁺ Signaling  
Tae-Houn Kim, Maik Böhmer, Honghong Hu, Noriyuki Nishimura,  
and Julian I. Schroeder ………………………………………………… 561  

The Language of Calcium Signaling  
Antony N. Dodd, Jörg Kudla, and Dale Sanders …………………………… 593  

Mitogen-Activated Protein Kinase Signaling in Plants  
Maria Cristina Suarez Rodriguez, Morten Petersen, and John Mundy ………… 621  

Abscisic Acid: Emergence of a Core Signaling Network  
Sean R. Cutler, Pedro L. Rodriguez, Ruth R. Finkelstein, and Suzanne R. Abrams … 651  

Brassinosteroid Signal Transduction from Receptor Kinases to  
Transcription Factors  
Tae-Wuk Kim and Zhi-Yong Wang ………………………………………… 681  

Directional Gravity Sensing in Gravitropism  
Miyo Terao Morita 705  

Indexes  
Cumulative Index of Contributing Authors, Volumes 51–61 721  
Cumulative Index of Chapter Titles, Volumes 51–61 726  

Errata  
An online log of corrections to *Annual Review of Plant Biology* articles may be found at  
http://plant.annualreviews.org
