
Biochimica et Biophysica Acta 1803 (2010) 694–705

Contents lists available at ScienceDirect

Biochimica et Biophysica Acta

journal homepage: www.elsevier.com/locate/bbamcr

---

Review

Endoplasmic reticulum associated protein degradation: A chaperone assisted journey to hell

Alexandra Stolz, Dieter H. Wolf*

Institut für Biochemie, Universität Stuttgart, Germany

---

**ARTICLE INFO**

Article history:

Received 19 November 2009

Received in revised form 11 February 2010

Accepted 18 February 2010

Available online 25 February 2010

Keywords:

Chaperone

ERAD

Lectin

Protein disulfide isomerase

Hsp70

Hsp40

---

**ABSTRACT**

Recognition and elimination of misfolded proteins are essential cellular processes. More than thirty percent of the cellular proteins are proteins of the secretory pathway. They fold in the lumen or membrane of the endoplasmic reticulum from where they are sorted to their site of action. The folding process, as well as any refolding after cell stress, depends on chaperone activity. In case proteins are unable to acquire their native conformation, chaperones with different substrate specificity and activity guide them to elimination. For most misfolded proteins of the endoplasmic reticulum this requires retro-translocation to the cytosol and polyubiquitylation of the misfolded protein by an endoplasmic reticulum associated machinery. Thereafter ubiquitylated proteins are guided to the proteasome for degradation. This review summarizes our up to date knowledge of chaperone classes and chaperone function in endoplasmic reticulum associated degradation of protein waste.

© 2010 Elsevier B.V. All rights reserved.

---

Life depends on the proper function of every protein in a cell. The function relies on the three dimensional conformation, which the amino acid chain of the protein acquires during and after its synthesis by the ribosome. Reaching the state of an energy minimum in the landscape of folding intermediates seems rather easy in diluted physiological solutions. However, conditions in cells are quite different. The cellular environment is crowded with a multitude of macromolecules. Folding mistakes and trapped intermediates caused by genetic mutations or cellular stresses occur constantly. To minimize these mistakes, helper proteins – chaperones – bind and accompany the folding event. The cell possesses a highly active quality control system that recognizes the folding state of proteins. The system scans the folding process, detects terminally misfolded proteins and specifically eliminates them by proteolysis. Failure of the protein quality control system finally leads to the formation of protein aggregates that in turn can cause severe diseases like Alzheimer-, Parkinson- or Creutzfeldt-Jakob disease. Chaperones have been thought to be solely responsible for the folding process of newly synthesized proteins or the refolding of proteins that had been partially unfolded by stresses, e.g. heat (“heat shock proteins”) [1–3]. Nowadays chaperones are known to accompany proteins in almost all periods of their life.

This review focuses on chaperones that accompany the folding and degradation process of proteins of the secretory pathway. Proteins of the secretory pathway represent around one third of the proteins synthesized in eukaryotic cells. They integrate into the membrane of the endoplasmic reticulum (ER) or pass this membrane barrier to reach the ER lumen, where they are folded and modified. Import of proteins into the ER can occur in a ribosome-coupled fashion (co-translationally) or in an uncoupled fashion (post-translationally). Upon folding failures chaperones keep proteins soluble and guide them to the ubiquitin proteasome system (UPS), the major cellular protein elimination machinery in eukaryotic cells [4–7]. As the degradation machinery of misfolded ER proteins has turned out to be the UPS, located in the cytosol [8,9], it was obvious that these false ER-imported proteins have to be recognized and retro-translocated across the ER membrane back into the cytosol, where they are polyubiquitylated and degraded [9–11]. The ER associated degradation (ERAD) machinery retro-translocates the misfolded proteins by a yet not well-defined mechanism. Subsequently, they are modified with polyubiquitin chains and by this designated for proteasomal degradation. For misfolded ER luminal proteins the retro-translocation complex (RTC) comprises the Hrd-Der ubiquitin ligase complex of the ER membrane, connected to the Derlin Der1 and the Sec61 translocon. On the cytoplasmic side of the ER a Cdc48-Ufd1-Npl4 segregation machinery, linked to the RTC, delivers the polyubiquitylated proteins to the proteasome for degradation. Misfolded membrane proteins with lesions in the cytosol find their way to the proteasome via the trimeric Cdc48 complex after polyubiquitylation by the ER membrane integrated ubiquitin ligase

* Corresponding author.
E-mail address: dieter.wolf@ibc.uni-stuttgart.de (D.H. Wolf).

0167-4889/$ – see front matter © 2010 Elsevier B.V. All rights reserved.
doi:10.1016/j.bbamcr.2010.02.005

Doa10 (Fig. 1) [12–16]. Overload of misfolded proteins leads to the induction of the unfolded protein response (UPR), a mechanism, which upregulates proteins necessary for dealing with this unfavorable situation. These proteins include chaperones and components of the ERAD machinery [17,18]. In the case of such an overload, misfolded proteins may aggregate. Chaperones show the capacity to re-solubilize these aggregates [5,19,20]. Thus, they not only “chaperone” protein folding but also act as “scavengers” for intracellular protein waste. Polyubiquitylation prior to proteasomal degradation is not required for elimination of a mutated secretory polypeptide, pro-α-factor [21]. It is still not clear what distinguishes this misfolded polypeptide from other misfolded ER proteins analyzed, which all require a fully active polyubiquitylation system for degradation. Chaperones have been found in all folding compartments of the eukaryotic cell: the cytoplasm, the endoplasmic reticulum and mitochondria. In addition, the nucleus and peroxisomes are sites of chaperone activity [22].

### 1. The cellular chaperone equipment

Chaperones belong to a protein class of great variety. They show remarkable differences concerning their mechanistic behaviour, the need of co-factors and ATP. Chaperone sizes start around 25 kDa, but can reach over 100 kDa. This chapter introduces the different chaperone families and describes their modes of action.

#### 1.1. The Hsp70 family

Hsp70s are most probably the best-studied class of chaperones. They have a molecular mass of approximately 70 kDa and are highly

![Diagram](attachment://image.png)

**Fig. 1.** The ERAD machinery for delivery, ubiquitylation and degradation of misfolded ERAD-L and ERAD-C proteins. ERAD-L and ERAD-C substrates can be distinguished by their requirement of two different ER machineries. After carbohydrate trimming (Fig. 3), Pdi1 action and binding to Hrd3 as well as Yos9 scanning, soluble ERAD-L substrates are delivered to a retro-translocation complex (RTC) composed of different ER membrane proteins including Sec61 [89], Hrd3 itself, Usa1, Der1 and the ubiquitin ligase Hrd1/Der3. During retrotranslocation the misfolded proteins are polyubiquitylated on the cytosolic side of the ER via the Hrd1/Der3 ligase [12,13,16]. The following steps are similar for ERAD-L and ERAD-C: the trimeric AAA-ATPase complex composed of Cdc48-Ufd1-Npl4, which is linked to the ER membrane by Ubx2 [12,13,16,183,184] pulls the polyubiquitylated protein from the ER membrane and, via the ubiquitin receptors Dsk2 and Rad23 [185] delivers the polyubiquitylated protein to the proteasome for degradation. Ubx4 may release the Cdc48 complex from the ER membrane [186]. Png1 [187] cleaves off the carbohydrate chains prior to degradation. For ERAD-C substrates the RTC is in part less well-defined but simpler: after or during retro-translocation from the ER membrane the ERAD-C substrates are polyubiquitylated by virtue of a second ubiquitin ligase, Doa10. The pathway merges with the ERAD-L pathway at the Cdc48-Ufd1-Npl4 machinery.

conserved among all organisms. Members of this family act e.g. in protein folding, in preventing protein aggregation and in membrane translocation. Classical chaperone activity of Hsp70s is ATP dependent and includes cycles of protein binding and ATP hydrolysis, affected by co-chaperones (Hsp40s) and nucleotide exchange factors (NEFs) (Fig. 2). Hsp70s typically contain three domains: an N-terminal ATPase domain, a central peptide binding domain and a C-terminal lid [23–26]. In an ATP bound state Hsp70s show low affinity for their substrates and cycles of substrate association and dissociation are fast. During ATP hydrolysis the tertiary structure changes and a ‘closing’ of the C-terminal lid is stimulated. Thereby substrates get tightly attached to the chaperone [25,27–29]. In the ADP bound state, the affinity of the peptide binding domain for substrates strongly increases [30,31]. Due to a subsequent nucleotide exchange of ADP and ATP the lid opens and substrates are released. Rates of ATP hydrolysis and nucleotide exchange are very low, but can be stimulated by binding of the substrate, co-chaperones and NEFs, respectively [30,32–35]. However, the precise mechanism of exchange events could not be ascertained till now.

### 1.2. The Hsp40 family

Hsp40s, acting as co-chaperones, induce the ATPase activity of Hsp70s. All Hsp40s contain a J-domain and are therefore often referred to as J-proteins. J-domains were named after a conserved motif in the *E. coli* protein DnaJ [36]. Hsp40s have a conserved HPD tripeptide in the J-domain that is necessary for the stimulation of Hsp70 ATPase activity [37–39]. One classification divides members of the Hsp40 family into three subgroups [40]. While Type III Hsp40s just contain the J-domain, additional domains were found in Hsp40s of type I and II. Type I Hsp40s, like DnaJ, possess the J-domain and a glycine/phenylalanine-rich domain (G/F) that seems to function as a spacer and is required for efficient stimulation. In addition they contain a cysteine-rich Zn²⁺ binding domain (CR) which functions as a peptide binding region of non-native substrates [41,42]. Type II Hsp40s lack this CR-domain but contain the G/F-domain. Even though Hsp40s and Hsp70s mainly bind the same types of peptides, there is specificity for some peptides. It is not yet clear whether Hsp70s replace Hsp40s at the substrate, or whether there exists a ‘handover’ mechanism between them, which would postulate different substrate binding sites and simultaneous binding of the Hsp40 and Hsp70 to the substrate. The second mechanism indicates active substrate guidance by Hsp40s to their Hsp70 partner [43].

### 1.3. Nucleotide exchange factors

Nucleotide exchange factors help to promote Hsp70 triggered folding cycles by accelerating the exchange of ADP against ATP. NEFs do not feature a special domain but are highly diverse in structure and mechanism. Bacterial GrpE, the first identified NEF, forms dimers, interacts with the ATPase domain of the Hsp70 homologue DnaK and can stimulate the exchange of both, ADP and ATP [44,45]. The BAG domain, which was found in BAG-1 (*Bcl-2* associated athanogene; mammals) and Snl1 (yeast), shows a different mechanism of stimulation. BAG-1 discriminates between ATP and ADP exchange and the efficiency of nucleotide exchange is dependent on Hsp70/BAG-1 stoichiometry [46]. Another unique mechanism was found for HspBP1 (*Hsp Binding Protein 1*). This mechanism does not comprise the conserved conformational change in the ATPase domain caused by the binding of GrpE or BAG-1 [47,48]. Quite recently Hsp70 (Lhs1) and Hsp70 related Hsp110 chaperones were shown to act as NEFs on other Hsp70s [49–51]. These examples make clear, how diverse the mechanisms of nucleotide exchange can be and thus how many types of regulation and specification are possible.

### 1.4. The Hsp90 family

Hsp90 chaperone activity is linked to ATP hydrolysis. Therefore, these chaperones contain an ATP binding domain in the N-terminal region of the protein [52]. A single ATP binding domain is not sufficient for ATP binding but requires transient dimerization of two N-terminal domains. A helical motif was identified in the C-terminal region of Hsp90, which enable dimerization [53,54]. Substrate binding is attributed to a large hydrophobic patch in the middle region of Hsp90 [55]. Prodromou et al. [54] proposed a molecular ‘clamp’ mechanism. Clients are bound to the middle region of C-terminally dimerized Hsp90s. ATP binding to the N-terminal ATP binding domains induces closing of a ‘lid’ above the clients and

![Figure 2](attachment://image.png)

Fig. 2. The folding cycle of Hsp70 chaperones. Hsp70s in an ATP bound state exhibit low affinity for their substrates and cycles of substrate association and dissociation occur fast. Interaction of an appropriate J-protein induces the ATPase activity of the Hsp70. Some Hsp40s are able to actively deliver substrates to their Hsp70 partner. During ATP hydrolysis the tertiary structure of the Hsp 70 changes, causing a ‘closing’ of the C-terminal lid. By this substrates get tightly attached to the chaperone. Subsequently a nucleotide exchange factor initiates the exchange of ADP against ATP, resulting in the opening of the lid and the release of the substrate. Please note that this figure is a schematic representation of a working model. It is not yet clear in which precise order substrates and co-factors bind and leave the Hsp70.
ATPase activity. After ATP hydrolysis ADP dissociates and the lid opens again. However, ATPase activity is low and regulated via co-chaperones including Hsp70s and Hsp40s [52,56–60]. As for Hsp70s, interaction partners influence whether Hsp90s promote folding or proteolytic degradation of clients.

### 1.5. Small heat shock proteins

Small heat shock proteins (sHsp) represent a class of chaperones, which bind substrates rather unspecifically and prevent proteins from irreversible aggregation. Clients are partially denatured proteins in an unstable state [61,62], which are bound until general cell conditions are suitable for proper folding or final degradation. Chaperone activity is considered to be energy-independent, however substrate affinity binding sites are potentially affected by ATP binding [63]. sHsp-substrate interaction is quite stable and disassembly may require interaction of ATP dependent chaperones like Hsp70s or Hsp90s, which facilitate substrate refolding [64,65]. Small heat shock proteins contain a conserved α-crystallin domain [66–68] and are enriched in β-strands. Crystallization revealed sHsps to form dimers and oligomers. Dynamic disassembly of these complexes seems to be required for effective chaperon function [69]; for review see [20].

### 1.6. Thiol oxidoreductases

Thiol oxidoreductases contain a characteristic CXXC motif that is located in the active site of the enzyme and are referred to as members of the PDI (protein disulfide isomerase) family. They take part in disulfide bond formation, protein folding and ERAD [70,71]. The precise sequence and number of the CXXC motifs and protein structures varies among family members. Because the environment in the ER is highly optimized for oxidative protein folding, PDI family members are mainly ER localized. Five members in yeast and over 19 in human have been identified so far. At least some of them were shown to possess chaperone-like functions [72–74]. Except for single species, oxidative activities of the PDI family members are very similar [75,76]. It is not yet clear if and how substrate specificity is achieved.

### 1.7. Lectin-like chaperones

Lectin-like chaperones of the ER predominantly bind substrates with N-glycan modification. Dependent on the glycan structure they either facilitate efficient folding and ER retention of immature glycoproteins or identify aberrant proteins, prevent aggregation and facilitate destruction [77–79]. Members of this class contain a sugar binding domain like a mannosidase-like domain or a mannosidase receptor homology (MRH) domain [80–82]. However, they are able to bind at least some non-glycosylated ERAD substrates, too, suggesting a chaperone-like activity [77,83].

## 2. Chaperone function in ERAD

With respect to the localization of the misfolded domain in a protein two major pathways have been defined, ERAD-L and ERAD-C, which are used when the misfolded protein domain resides in the ER lumen and the cytosol, respectively [84]. A third, not yet precisely defined ERAD-M (membrane) pathway was proposed [85]. It was a surprising discovery in the beginning of ERAD research, that misfolded, fully glycosylated ER lumenal proteins are finally eliminated in the cytoplasm [9]. Specific misfolded substrates have been discovered or designed as useful tools for the elucidation of how ERAD in general and ERAD-L specifically works. The mutated vacuolar/lysosomal yeast enzyme carboxypeptidase yscY (CPY*) has become a model for an ER lumenal misfolded protein. The enzyme is mutated two amino acids distant from the active site serine (G255R). This glycine residue is highly conserved among all serine peptidases and the mutation leads to misfolding and loss of activity [86]. CPY* is imported into the ER lumen, fully glycosylated, recognized as being misfolded, retrograde transported out of the ER in glycosylated form, fully ubiquitylated and degraded by the proteasome [9].

### 2.1. Kar2 keeps proteins soluble during import into the ER

During post-translational import of CPY* into the ER via the Sec61 translocation channel, the protein is N-glycosylated at its four glycosylation sites. The spatial gap between the point of modification by the oligosaccharyl transferase (OST) complex and the Sec61 import channel was identified to be 15 amino acid residues [87]. Glycosylation at an engineered fifth glycosylation site at the very C-terminus of CPY* proved complete import of the misfolded protein into the ER lumen [88,89]. The first chaperone that contacts post-translationally imported proteins like CPY* in yeast is the Hsp70 Kar2, the orthologue of mammalian BiP. Kar2 binds short hydrophobic patches of incoming proteins and is tethered to the ER membrane by virtue of Sec63, an integral membrane partner of Sec61. In a concerted action of Kar2 (BiP) and the J-domain of Sec63, nascent proteins are imported. In this process Kar2 (BiP) works as a Brownian ratchet. Interaction with Kar2 prevents the polypeptide chain to slide back through the Sec61 channel into the cytosol, by this enabling only forward movement [90]. In general Kar2 (BiP) supports folding of post- and co-translationally imported proteins and keeps them soluble. At a certain stage along the folding process the quality control system marks proteins like CPY* as misfolded. At this point Kar2 (BiP) becomes an important component of the machinery that directs misfolded substrates towards the degradation pathway. Mutations in KAR2 significantly slow down CPY* degradation [91,92]. In addition, degradation of the mutated pro-α-factor was shown to be Kar2 (BiP) dependent [70,93]. Failure to assemble with the immunoglobulin (Ig) heavy chain makes the Ig light chain an ERAD substrate. Dissociation of BiP from unassembled Ig light chains correlates with the degradation kinetics of the substrate, indicating the importance of this ER lumenal Hsp70 chaperone for the ERAD process [94].

### 2.2. J-proteins enhancing the ATPase activity of Kar2 vary in protein import and ERAD

In addition to the ER membrane embedded Sec63, two well studied J-proteins, Jem1 and Scj1, reside in the ER lumen and enhance the ATPase activity of Kar2. While Sec63 was shown to exhibit a rather minor role during ERAD of CPY* [91,92], Jem1 and Scj1 seem to act as redundant enhancers of the ATPase activity of Kar2. Deletion of both Hsp40s resulted in prolonged degradation and led to aggregation of the soluble ERAD substrate CPY* [91,95]. Another J-protein called Erj5 was recently identified. Deletion of ERJ5 was found to increase growth defects associated with deletion of JEM1 or SCJ1. Cells with a triple deletion (∆jem1∆scj1∆erj5) show a constitutive induction of the unfolded protein response (UPR) and increased levels of KAR2 mRNA, compared to cells deleted only in SCJ1 and JEM1. This points to an Erj5 requirement for the maintenance of an optimal folding environment in the ER [96]. However, direct involvement of Erj5 in ERAD has not been observed yet.

### 2.3. Kar2 and its NEF Lhs1 show coupled ATPase cycles

Lhs1 (lumenal hsp seventy) could be identified in the ER lumen and is conserved in higher eukaryotic cells (human: Grp170) [97]. Deletion of LHS1 is not essential but shows synthetic lethality with KAR2 mutations or deletion of IRE1, the kinase triggering the UPR [98]. The protein acts as a nucleotide exchange factor on Kar2 and together
with an appropriate J-protein increases Kar2 ATPase activity up to 30-fold [50]. The structure of Lhs1 contains a nucleotide binding domain (NBD), which is essential for NEF activity. Besides NEF activity, Lhs1 was shown to act as a holdase and to prevent protein aggregation in vitro, independent of its nucleotide binding domain [49]. ATPase cycles of Kar2 and Lhs1 seem to be coupled for efficient chaperone function. In addition to Lhs1, Sil1 (“suppressor of the Δire1 Δlhs1 double mutant number 1”) has been identified as a nucleotide exchange factor of Kar2 (BiP). Sil1 is a GrpE-like non-essential protein, which shows some functional redundancy with Lhs1. Lack of both proteins causes synthetic lethality [99]. However, function of Lhs1 and Sil1 is not completely redundant. LHS1 mutated cells are defective in protein translocation into the ER whereas cells containing a SIL1 mutation are not [50,100].

### 2.4. Mutation of the chaperone Rot1 leads to rapid protein degradation via ERAD

Rot1 was identified as a chaperone responsible for folding of nascent proteins. It is an ER localized glycosylated type I membrane protein [101,102]. Initially, the essential gene ROT1 was reported to encode a protein that is required for 1,6-β-glucan synthesis [103,104]. From the current point of view this finding might be attributed to a secondary effect. The viable mutant rot1-2 triggers UPR and shows synthetic lethality with kar2-1 indicating a partially redundant function. Rot1 and Kar2 interact in vivo [102]. In vitro Rot1 is able to prevent protein aggregation and specifically interacts with nascent proteins. In rot1-2 mutated cells several proteins are unable to reach their native conformation. As a consequence they are rapidly degraded via ERAD [105]. Based on this finding general participation of Rot1 in ERAD can be excluded. However, involvement in the degradation of specific proteins is still possible.

### 2.5. The glycan structure of glycoproteins reflects their folding status

Upon entrance of secretory proteins into the ER, two modifications are introduced during the folding process: carbohydrates are added and disulfide bonds between cysteine residues are formed. As mentioned above, most proteins become N-glycosylated upon their entry into the ER. A glycan consisting of Glc₃-Man₉-GlcNAc₂ is linked to asparagine residues within the consensus sequence Asn-X-Ser/Thr of the imported proteins. A trimmed N-glycosyl structure (Fig. 3) has turned out to be an important attribute of misfolded proteins. It acts as an accelerator of degradation. While the three terminal glucose residues are cleaved off by two glucosidases, glucosidase I and II, the protein folds into its native structure. After cleavage of the third glucose residue, properly folded proteins are ready for travelling to the Golgi apparatus for further delivery to their final destination [106]. However, if the last glucose residue is cleaved off before the protein has folded properly, mannose units are removed from the carbohydrate structure in order to mark it as misfolded [107,108].

### 2.6. The calnexin/calreticulin cycle promotes protein folding

In higher eukaryotic cells chaperones are part of a cycle for efficient protein folding [106]. After the removal of the two terminal glucose residues from the glycan structure, the molecular chaperones calnexin and calreticulin interact with the Glc₁-Man₉-GlcNAc₂ charged protein and promote its folding [109] (Fig. 4). After dissociation, the remaining glucose residue can be removed by glucosidase II and properly folded proteins are able to exit the ER. In the case of incomplete folding, the folding sensor UDP-glucose/glycoprotein glucosyltransferase (UGGT) reloads proteins with a terminal glucose and thereby initiates another cycle of folding [110–112]. Proteins that do not fold within several times of chaperone cycling are released

Fig. 3. During the folding process of glycoproteins in the ER N-linked glycan structures are processed. In yeast the three terminal glucose residues of the initial Glc₃-Man₉-GlcNAc₂ structure are removed by glucosidase I (Gls1) and glucosidase II (Gls2). Prolonged ER lumen retention of the protein due to not yet completed folding recruits an α-1,2-mannosidase (Mns1) and an α-mannosidase-like protein (Htm1/Mnl1) to remove two mannose residues resulting in a Man₇-GlcNAc₂ glycan structure. This structure, containing a terminal 1,6-linked mannose, is an attribute of misfolded glycoproteins. In mammalian cells the orthologues of Htm1/Mnl1, EDEM1-3 are able to trim oligosaccharides to Man₆-GlcNAc₂ or Man₅-GlcNAc₂ glycan structures.

from the cycle due to mannosidase action and become determined to ERAD [107,108,113–116] (Figs. 3 and 4).

In Saccharomyces cerevisiae only the calnexin/calreticulin orthologue Cne1 has been found. An UGGT orthologue remained elusive [117,118]. Cne1 exhibits efficient chaperone activity for glycosylated and non-glycosylated substrates [118,119]. It was found to participate in the quality control of some under-glycosylated variants of CPY*. Here the mannosidase Htm1/Mnl1 and Cne1 play antagonistic roles in targeting the substrate for degradation: loss of Cne1 accelerates degradation, loss of Htm1/Mnl1 stabilizes the substrate [120]. A crosstalk of Cne1 with Kar2 (BiP) and Pdi1 (protein disulfide isomerase) under heat shock conditions was discovered [93,121,122]. Cne1 was shown to interact with the Pdi1 homologue Mpd1, enhancing its reductive activity, while at the same time the chaperone activity of Cne1 is inhibited [123]. These interactions are conserved in higher eukaryotes. In mammalian cells an interaction between PDI family members and calnexin could be shown [124]. Due to the relatively slow action of α-mannosidase I (Mns1) and Htm1/Mnl1, a time window for the folding of ER-imported glycoproteins is kept open. The mannose timer hypothesis was put forward to exemplify this behaviour [125]. However, not all ERAD substrates are glycosylated. Thus, there has to be an additional mechanism, which distinguishes unglycosylated proteins in the process of folding from terminally misfolded proteins.

### 2.7. α-Mannosidases mark glycoproteins as misfolded

In yeast the α-1,2-mannosidase Mns1 removes a mannose residue of the middle branch of the Man₉-GlcNAc₂ carbohydrate tree, resulting in a terminal 1,3-linked mannose (Fig. 3). This leads to an acceleration of ERAD of N-glycosylated substrates [126]. Lectin-like proteins sharing a mannosidase-homology domain (MHD) had been discovered in the ER lumen of yeast and mammalian cells: Htm1/Mnl1 and EDEM (ER Degradation Enhancing α-Mannosidase-like protein) [127–129]. They considerably enhance the degradation of N-glycosylated misfolded proteins. While the MHD domain of Htm1/Mnl1 is necessary for substrate binding in yeast, mammalian EDEM1 was shown to bind misfolded proteins independently of the MHD domain and therefore independently of their glycan structure. The MHD domain of EDEM1 was required for the interaction with the ubiquitin ligase adaptor protein SEL1, the equivalent of the yeast Hrd3 protein [77]. Recently, Htm1/Mnl1 and the EDEM proteins have

A. Stolz, D.H. Wolf / Biochimica et Biophysica Acta 1803 (2010) 694–705

![Diagram](attachment:diagram.png)

Fig. 4. ER import, folding cycle and quality control of a secretory protein. Nascent ER lumenal proteins enter the ER via the Sec61 translocon. The oligosaccharyl transferase (OST) complex charges the protein with a core oligosaccharide (Glc₃-Man₉-GlcNAc₂) at appropriate consensus sequences. Together with its co-chaperone Sec63 the Hsp70 Kar2 drives import of post-translationally imported proteins and in general promotes the folding of the imported proteins. Protein disulfide isomerase (Pdi1) facilitates the formation of intramolecular disulfide bonds. Yeast Cne1, an orthologue of mammalian calnexin, may support Kar2 in the folding process. In mammalian cells (denoted by asterisk) calnexin and UDP-glucose/glycoprotein glucosyltransferase (UGGT) support a folding cycle. Turns of deglucosylation and reloading of the glycan structure with a single glucose residue provide further time for the folding process. Properly folded proteins exit from the ER and travel to the Golgi apparatus for further distribution to their final secretory destination. Terminally misfolded proteins become substrates of the α-mannosidases Mns1 and Htm1/Mnl1, which further trim the glycan structures (Fig. 3). By this the protein becomes marked as being misfolded and is recognized by the lectin Yos9. Prior to retro-translocation into the cytoplasm Pdi1 reduces disulfide bonds present in a protein. In the case of ER membrane proteins exposing misfolded cytosolic domains, the cytosolic Hsp70 machinery, in yeast consisting of the Hsp70 Ssa1 and its co-factors Sse1, Ydj1 and Hlj1, is necessary for efficient protein degradation via the UPS.

been classified as mannosidases. Htm1/Mnl1 cleaves off a mannose residue from the Man₈-GlcNAc₂ structure generating a Man₇ structure with an α-1,6-mannose linkage [130,131] (Fig. 3). Interestingly, not all carbohydrate chains of a protein are equivalent in their potential to signal misfolding. A misfolded protein structure around at least one of the glycan chains is equally important for recognition [120,132,133]. In mammalian cells the EDEM proteins, similar to yeast Mns1 and Htm1/Mnl1, cause mannose trimming, resulting in the formation of Man₆ and Man₅ structures [134–137].

2.8. PDI and its orthologues interact with ERAD components and reduce disulfide bonds for retro-translocation

The formation of disulfide bonds within proteins is dependent on a family of enzymes called protein disulfide isomerases (PDI). This ER localized class of enzymes with thiol oxido-reductase activity also exhibits chaperone-like activity [138,139]. The best-known member is PDI (Pdi1 in yeast), which is essential and conserved in all eukaryotic cells [140]. Four additional members of the PDI family are found in yeast: Mpd1 [141], Mpd2 [142], Eug1 [143] and Eps1 [144]. Even though they are not essential like Pdi1, they can suppress inviability of a Δpdi1 mutant if overexpressed. In human 19 orthologues have been identified so far [72]. The oxidative activities within the human enzymes are not necessarily different [75]. However, their specificity with respect to their chaperone-like activity may differ for particular protein substrates [73,74,124,144,145]. In yeast chaperone function of Pdi1 is required for the retro-translocation of the thiol-free misfolded polypeptide mutated pro-α-factor. In contrast, the enzymatic activity of Pdi1 is necessary for the export of disulfide bond containing CPY* to the cytosol, indicating the requirement of disulfide

bond reduction for retro-translocation [146]. The ER membrane located Eps1 specifically binds a mutant form of the plasma membrane ATPase Pma1, Pma1D378N, but not the wild type enzyme. Eps1 prevents escape of the mutant protein from the ER membrane and facilitates its degradation [144]. In recent studies Pdi1 was shown to form a stable (>60 min) intermolecular disulfide bond with the lectin-like chaperone Htm1/Mnl1 [131,147]. It is possible that this interaction facilitates Pdi1 recruitment to ERAD substrates with the aim to reduce disulfide bonds prior to transport out of the ER. The PDI orthologue of mammals, ERdj5, a J-domain containing oxidoreductase with the highest reducing equilibrium constant found so far, interacts with EDEM1, the mammalian orthologue of Htm1/Mnl1. At the same time ERdj5 interacts with BiP via its His-Pro-Asp containing J-domain. The degradation of two ERAD substrates containing disulfide bonds was dependent on ERdj5. ERdj5 reductase activity and interaction between ERdj5, BiP and EDEM1 were required for efficient degradation [76].

Based on the experiments of Ushioda et al. [76] the following sequence of events can be hypothesized: EDEM, ERdj5 and BiP have distinct but concerted roles in the degradation of misfolded proteins. After the release of terminally misfolded proteins from the calnexin-calreticulin cycle, EDEM further trims the Man₈-GlcNAc₂ structure. Concomitantly the disulfide bonds of the substrate are reduced by ERdj5 to allow retrograde translocation into the cytosol. At the same time ERdj5 activates the ATPase activity of BiP via its J-domain. The ADP-form of BiP in turn strongly binds the substrate and holds it in a retro-translocation competent form until it gets transferred to the RTC.

2.9. Yos9 releases misfolded proteins for retrograde transport and subsequent ubiquitylation

A crucial protein factor in the delivery pathway of misfolded lumenal proteins out of the ER to the cytosol is Yos9 and its mammalian orthologue OS-9. Yos9 is a lectin-like protein, which shares a mannosidase receptor homology (MRH) domain with members of the mannose-6-phosphate receptor (MPR) family [82,148]. The protein carries a HDEL ER retention signal at its C-terminus. Analyses with CPY* and CPY* derivatives proved Yos9 to be required for the degradation of N-glycosylated substrates [83,149–151]. The function of its MRH domain was shown to reside in glycan structure dependent substrate binding [83,151]. Bhamidipati et al. [83] demonstrated also an interaction between Yos9 and the unglycosylated ERAD-L substrate CPY*0000. However, accelerated degradation of CPY* depends on recognition of its glycan structure by Yos9 [149]. Recent studies in yeast strongly support the requirement of glycan dependent interaction between Yos9 and lumenal ERAD substrates for efficient degradation [78,130,131]. Yos9 has been proposed to scan proteins for their mannose-trimmed glycan structure and by this acting as a gatekeeper. It is thought that only terminally misfolded proteins carrying a glycan with a terminal 1,6-linked mannose are allowed to leave the ER for proteasomal degradation [78,83,130,152]. One important interaction partner of Yos9 is Hrd3, a type I transmembrane protein, which together with Hrd1/Der3 is part of an ER membrane embedded ubiquitin ligase complex. Hrd3 is able to bind misfolded proteins independently of Yos9 [78]. Most likely, only the scanning function is left to Yos9. Thus, a Hrd3–Yos9 complex recognizes a bipartite signal in the substrate protein, hydrophobic protein patches and the properly mannose-trimmed glycan. The Yos9 MRH domain is not necessarily required for Yos9 Hrd3 interaction. Gauss et al. [78] found evidence for an additional ligase complex without Yos9. It is likely that this complex is specific for the recognition and cytosolic delivery of non-glycosylated and therefore Yos9 independent misfolded lumenal ERAD substrates. Yos9 has two human orthologues, XTP3-B and OS-9. These lectin-like proteins contain MRH domains that are required for interaction

with the human Hrd3 orthologue SEL1 but not with the substrate [153,154]. In addition, OS-9 associates with the ER chaperone GRP94, which together with SEL1 and the HRD1 ligase is required for degradation of the ERAD substrate mutant α1-antitrypsin. Both proteins, XTP3-B and OS-9 are components of distinct, partially redundant quality control pathways [153].

2.10. A model for the degradation of the ERAD-L substrate CPY*

CPY* remains bound to the Hsp70 Kar2 and its Hsp40 co-factors Jem1 and Scj1 during import, glycosylation and the formation of disulfide bonds within the molecule. After unsuccessful attempts to fold properly and removal of the three glucose residues of the N-linked carbohydrate structure, the α-mannosidases Mns1 and Htm1/Mnl1 process the glycan chains of the protein. At the same time Htm1/Mnl1 bound Pdi1 reduces the disulfide bonds. CPY* and the Kar2–Jem1–Scj1 complex bind to Hrd3, which is part of the ER membrane located RTC. Yos9, bound to Hrd3, recognizes the processed carbohydrate structure at position 4 of the protein, which is surrounded by exposed hydrophobic residues. CPY* is then delivered to the RTC, which contains Sec61. The RTC relocates CPY* from the ER lumen to the cytoplasm where it is polyubiquitylated and delivered to the proteasome for degradation (Figs.1 and 4).

3. The cytosolic face of ERAD

The degradation of misfolded ER membrane proteins requires a different set of chaperones as compared to ER luminal proteins, due to the exposure of protein domains to the cytosol. Model substrates with cytosolic domains studied in yeast comprise the fusion protein CTG* that contains the luminal misfolded CPY* domain and a transmembrane domain fused to the green fluorescent protein (GFP) on the cytoplasmic side of the ER [155]. Others are: the mutated ATP binding cassette transporter Ste6*, in its wildtype form exporting the mating pheromone α-factor through the plasma membrane of cells [156,157], a mutated form of the plasma membrane ATPase Pma1 [158] and the mutated yeast ABC transporter Pdr5* [159]. Furthermore, mammalian apolipoprotein B (apoB) [160] and cystic fibrosis transmembrane conductance regulator (CFTR) expressed in yeast are intensely studied ERAD substrates [161–163]. In mammalian cells the best-studied ERAD substrate is a mutant form of the CFTR channel, CFTRΔF508. The protein carries a deletion of phenylalanine at position 508 and causes the disease cystic fibrosis in humans [164,165]. The prominent role of the cytosolic Hsp70 machinery in the degradation of membrane proteins becomes visible by analyzing these latter substrates, exposing large misfolded domains or hydrophobic patches to the cytosol (ERAD-C substrates). ERAD-C either fully or in part requires the ubiquitin ligase Doa10 for polyubiquitylation (Fig. 1). In some instances the Hrd1/Der3 ligase has some overlapping specificity with Doa10 [166]. The degradation pathway of ERAD-L and ERAD-C substrates merges at the Cdc48-Ufd1-Npl4 machinery, which further delivers the misfolded polyubiquitylated proteins to the proteasome for degradation (Fig. 1).

3.1. The Hsp70 Ssa1 is required for the degradation of ERAD-C substrates

Degradation of Ste6*, Pma1D378S as well as yeast expressed CFTR and apoB relies on the Hsp70 family member Ssa1 [156,158,163,167]. Most likely, the additional cytosolic Hsp70s of the Ssa family (Ssa2, Ssa3, Ssa4), which share 85–99% amino acid identity with Ssa1, have overlapping activities in the elimination process of these proteins. Given the large lipophilic N-terminal domains of apoB that reside in the ER, it is not surprising that the degradation of yeast expressed apoB in addition requires the ER luminal Kar2 (BiP) [160]. Kar2 (BiP) is not required for the degradation of the polytopic membrane proteins Ste6*, yeast expressed CFTR and the mutated yeast ABC

transporter Pdr5* [156,159,163]. Obviously, these substrates do not contain large unfolded domains in the ER lumen. Both ER membrane-localized cytosolic Hsp40 chaperones Ydj1 and Hlj1 have been found to be important for ERAD of Ste6* and yeast-expressed CFTR [156,161] (Fig. 4). In authentic surrounding—mammalian cells—the degradation of CFTR and CFTRΔF508 is dependent on the chaperone pair Hsp70–Hsp40 (Hsc70–Hdj2) [168].

### 3.2. The ERAD-L substrate CTG* needs cytosolic chaperones for efficient degradation

In contrast to soluble CPY* and membrane-bound CT* elimination of CTG* is dependent on the cytosolic Hsp70 chaperone Ssa1 [155]. This requirement was unexpected and must be due to the tightly folded GFP moiety of CTG*. Among all tested cytosolic Hsp40 species, only the DnaJ-family member Hlj1 had a noticeable modulating effect on the degradation of CTG*. Interestingly, Ydj1, which like Hlj1 is localized at the ER membrane, the J-domain facing the cytosol, is not involved in CTG* degradation. In contrast, membrane proteins that expose misfolded domains into the cytosol do need Ydj1 for efficient degradation (see above). Hsp104, known to act in concert with Hsp70 and Hsp40 in reactivation of proteins [169], is to some extent involved in the degradation of CTG*. The two Hsp90 family members Hsc82 and Hsp82 are not active in the degradation process of this protein [155].

### 3.3. The diverse effects of the nucleotide exchange factor Sse1

One nucleotide exchange factor (NEF) of Ssa1 was shown to be important for ERAD: Sse1. Sse1 is a homologue of Hsp110, a divergent relative of Hsp70 chaperones. It was found to act very efficiently on Ssa1 [51,170–172]. Like for Lhs1, a NEF of Kar2 (BiP) in the ER lumen of yeast, a chaperone-like activity was proposed for Sse1. Studies on the ERAD substrate apoB expressed in yeast revealed a stabilizing effect of Sse1 on this substrate [160,173]. In particular, Sse1 stabilizes apoB in an ATP-dependent manner and thus acts contrary to Ssa1, which facilitates degradation of apoB [167,174]. The stabilizing effect of Sse1 for apoB matches the reported interaction of Sse1 and Ssa1 with nascent proteins [175]. Substrate specificity turns the balance of Sse1 activity: Sse1 facilitates degradation (Ste6*), prevents degradation (apoB) or does not show any influence on the degradation process at all (CFTR) [160,161]. A recent study specifies some important differences between Hsp110 and Hsp70. For instance, peptide binding to Hsp70s stimulates ATP hydrolysis, however, it does not in the case of Sse1. In addition, distinct peptide preferences have been shown for Sse1 [176].

### 3.4. Hsp90 is involved in the ERAD-C pathway

Analyses of the Hsp90 machinery Hsc82 and Hsp82 revealed functions that are dependent on the tested substrate: analysis of CFTR degradation in a Hsp90 mutant strain revealed acceleration of CFTR disappearance, indicating that these chaperones preserve CFTR in a folded state [161]. Assistance in folding by Hsp90 has also been found for CFTR in mammalian cells [177]. In contrast, Hsp90 facilitates ERAD of apoB [167]. Obviously, some substrates utilize Hsp90 for protection and folding, while others engage Hsp90 en route to degradation. In some cases Hsp90 is involved in both events. It first helps to fold and, if this cannot successfully be accomplished, targets the substrate for degradation [178].

### 3.5. Small heat shock proteins facilitate re-solubilization of protein aggregates

Small heat shock proteins prevent irreversible protein aggregation, for instance after heat shock. Two sHsps, Hsp26 and Hsp42, have been identified in the yeast *S. cerevisiae*. While Hsp42 seems to be a general chaperone, which is permanently active under all conditions, Hsp26 is activated upon heat shock temperatures. Above 37 °C it changes its structure by dissociating from an oligomeric into a dimeric state [179–182]. Deletion of both sHsps leads to full stabilization of the yeast-expressed ERAD substrate CFTRΔF508. The sHsps seem to work in a redundant fashion as single mutations showed only a minor stabilization of CFTRΔF508. Quantification revealed the same amount of polyubiquitylated CFTRΔF508 in wild-type and Δhsp26Δhsp42 mutant cells, indicating that the sHsps act downstream of the ubiquitylation of the substrate [162]. Degradation of Ste6*, a topologically comparable ERAD-C substrate, was not influenced by deletion of both sHsps [162].

What can be extracted from the studies on chaperone requirement for ERAD? It has become clear in recent years that chaperones play a central role in the elimination process of proteins that are either damaged, terminally misfolded or do not fold in a certain time frame. In addition, orphan proteins, which cannot assemble with their interaction partners to form native complexes, are targets for chaperones. All these species have in common that they expose hydrophobic amino acid patches at their surface, which constitute the interaction motifs of chaperones. Usually, this interaction keeps unfolded proteins soluble and helps to acquire a native structure. The clearest picture is presently available for the chaperones of the Hsp70 family members with their Hsp40 co-chaperones. In case, a protein exposes large hydrophobic areas in the ER lumen the Kar2/BiP machinery takes over the binding. Exposure of large hydrophobic patches in the cytosol leads to the recruitment of the cytosolic Hsp70 machinery, in yeast the Ssa family. Many questions remain. When do chaperones “decide” that proteins are terminally misfolded or remain an orphan? Is it an extended time window, kept open by chaperones or chaperone complexes that allow the ubiquitylation machinery to bind and polyubiquitinate the protein? Or are additional factors recruited, which finally signal the decision for entering the degradation pathway? What is the exact function and mechanism of the many different chaperones, which either stabilize or target a protein for degradation? How do chaperone family members, protein disulfide isomerases, lectins and other factors interact and finally hand over a misfolded protein to the RTC and the ubiquitylation machinery for ERAD? Many more experiments have to be performed in the future to allow a detailed understanding of how the chaperone machinery assists a misfolded protein's journey to hell.

## Acknowledgements

We thank Wolfgang Hilt, Frederik Eisele and Elena Martinez Benitez for critically reading of the manuscript. The work of the authors was funded by the Deutsche Forschungsgemeinschaft, the EU network of excellence RUBICON and the LGF Baden-Württemberg.

## References

[1] R.J. Ellis, S.M. Hemmingsen, Molecular chaperones: proteins essential for the biogenesis of some macromolecular structures, Trends Biochem. Sci. 14 (1989) 339–342.

[2] F.U. Hartl, M. Hayer-Hartl, Molecular chaperones in the cytosol: from nascent chain to folded protein, Science 295 (2002) 1852–1858.

[3] R.I. Morimoto, M.P. Kline, D.N. Bimston, J.J. Cotto, The heat-shock response: regulation and function of heat-shock proteins and molecular chaperones, Essays Biochem. 32 (1997) 17–29.

[4] C. Esser, S. Alberti, J. Hohfeld, Cooperation of molecular chaperones with the ubiquitin/proteasome system, Biochim. Biophys. Acta 1695 (2004) 171–188.

[5] S.H. Park, N. Bolender, F. Eisele, Z. Kostova, J. Takeuchi, P. Coffino, D.H. Wolf, The cytoplasmic Hsp70 chaperone machinery subjects misfolded and endoplasmic reticulum import-incompetent proteins to degradation via the ubiquitin-proteasome system, Mol. Biol. Cell 18 (2007) 153–165.

[6] A.J. McClellan, M.D. Scott, J. Frydman, Folding and quality control of the VHL tumor suppressor proceed through distinct chaperone pathways, Cell 121 (2005) 739–748.

[7] A.J. McClellan, S. Tam, D. Kaganovich, J. Frydman, Protein quality control: chaperones culling corrupt conformations, Nat. Cell Biol. 7 (2005) 736–741.

[8] T. Sommer, S. Jentsch, A protein translocation defect linked to ubiquitin conjugation at the endoplasmic reticulum, Nature 365 (1993) 176–179.

[9] M.M. Hiller, A. Finger, M. Schweiger, D.H. Wolf, ER degradation of a misfolded luminal protein by the cytosolic ubiquitin–proteasome pathway, Science 273 (1996) 1725–1728.

[10] C.E. Shamu, C.M. Story, T.A. Rapoport, H.L. Ploegh, The pathway of US11-dependent degradation of MHC class I heavy chains involves a ubiquitin-conjugated intermediate, J. Cell Biol. 147 (1999) 45–58.

[11] E.J. Wiertz, T.R. Jones, L. Sun, M. Bogyo, H.J. Geuze, H.L. Ploegh, The human cytomegalovirus US11 gene product dislocates MHC class I heavy chains from the endoplasmic reticulum to the cytosol, Cell 84 (1996) 769–779.

[12] Z. Kostova, D.H. Wolf, For whom the bell tolls: protein quality control of the endoplasmic reticulum and the ubiquitin–proteasome connection, EMBO J. 22 (2003) 2309–2317.

[13] S.S. Vembar, J.L. Brodsky, One step at a time: endoplasmic reticulum-associated degradation, Nat. Rev. Mol. Cell Biol. 9 (2008) 944–957.

[14] T. Sommer, D.H. Wolf, Endoplasmic reticulum degradation: reverse protein flow of no return, FASEB J. 11 (1997) 1227–1233.

[15] R.K. Plemper, D.H. Wolf, Retrograde protein translocation: ERADication of secretory proteins in health and disease, Trends Biochem. Sci. 24 (1999) 266–270.

[16] C. Hirsch, R. Gauss, S.C. Horn, O. Neuber, T. Sommer, The ubiquitylation machinery of the endoplasmic reticulum, Nature 458 (2009) 453–460.

[17] K.J. Travers, C.K. Patil, L. Wodicka, D.J. Lockhart, J.S. Weissman, P. Walter, Functional and genomic analyses reveal an essential coordination between the unfolded protein response and ER-associated degradation, Cell 101 (2000) 249–258.

[18] R. Friedländer, E. Jarosch, J. Urban, C. Volkwein, T. Sommer, A regulatory link between ER-associated protein degradation and the unfolded-protein response, Nat. Cell Biol. 2 (2000) 379–384.

[19] D.A. Parsell, A.S. Kowal, M.A. Singer, S. Lindquist, Protein disaggregation mediated by heat-shock protein Hsp104, Nature 372 (1994) 475–478.

[20] Y. Sun, T.H. MacRae, Small heat shock proteins: molecular structure and chaperone function, Cell. Mol. Life Sci. 62 (2005) 2460–2476.

[21] E.D. Werner, J.L. Brodsky, A.A. McCracken, Proteasome-dependent endoplasmic reticulum-associated protein degradation: an unconventional route to a familiar fate, Proc. Natl. Acad. Sci. U. S. A. 93 (1996) 13797–13801.

[22] J. Hageman, M.J. Vos, M.A. van Waarde, H.H. Kampina, Comparison of intra-organellar chaperone capacity for dealing with stress-induced protein unfolding, J. Biol. Chem. 282 (2007) 34334–34345.

[23] K.M. Flaherty, C. DeLuca-Flaherty, D.B. McKay, Three-dimensional structure of the ATPase fragment of a 70K heat-shock cognate protein, Nature 346 (1990) 623–628.

[24] R. Sousa, E.M. Lafer, Keep the traffic moving: mechanism of the Hsp70 motor, Traffic 7 (2006) 1596–1603.

[25] X. Zhu, X. Zhao, W.F. Burkholder, A. Gragerov, C.M. Ogata, M.E. Gottesman, W.A. Hendrickson, Structural analysis of substrate binding by the molecular chaperone DnaK, Science 272 (1996) 1606–1614.

[26] B. Bukau, A.L. Horwich, The Hsp70 and Hsp60 chaperone machines, Cell 92 (1998) 351–366.

[27] G.C. Flynn, T.G. Chappell, J.E. Rothman, Peptide binding and release by proteins implicated as catalysts of protein assembly, Science 245 (1989) 385–390.

[28] R. Jordan, R. McMacken, Modulation of the ATPase activity of the molecular chaperone DnaK by peptides and the DnaJ and GrpE heat shock proteins, J. Biol. Chem. 270 (1995) 4563–4569.

[29] P. D'Silva, J. Marszalek, E.A. Craig, An essential connection: link between Hsp70's domains at last, Mol. Cell 20 (2005) 493–494.

[30] J.S. McCarty, A. Buchberger, J. Reinstein, B. Bukau, The role of ATP in the functional cycle of the DnaK chaperone system, J. Mol. Biol. 249 (1995) 126–137.

[31] R. Russell, A. Wali Karzai, A.F. Mehl, R. McMacken, DnaJ dramatically stimulates ATP hydrolysis by DnaK: insight into targeting of Hsp70 proteins to polypeptide substrates, Biochemistry 38 (1999) 4165–4176.

[32] S. Alberti, C. Esser, J. Hohfeld, BAG-1—a nucleotide exchange factor of Hsc70 with multiple cellular functions, Cell Stress Chaperones 8 (2003) 225–231.

[33] J.H. Ha, D.B. McKay, ATPase kinetics of recombinant bovine 70 kDa heat shock cognate protein and its amino-terminal ATPase domain, Biochemistry 33 (1994) 14625–14635.

[34] C. Harrison, GrpE, a nucleotide exchange factor for DnaK, Cell Stress Chaperones 8 (2003) 218–224.

[35] H. Theyssen, H.P. Schuster, L. Packschies, B. Bukau, J. Reinstein, The second step of ATP binding to DnaK induces peptide release, J. Mol. Biol. 263 (1996) 657–670.

[36] C.P. Georgopoulos, A. Lundquist-Heil, J. Yochem, M. Feiss, Identification of the *E. coli* dnaJ gene product, Mol. Gen. Genet. 178 (1980) 583–588.

[37] Z. Lu, D.M. Cyr, The conserved carboxyl terminus and zinc finger-like domain of the co-chaperone Ydj1 assist Hsp70 in protein folding, J. Biol. Chem. 273 (1998) 5970–5978.

[38] A.A. Michels, B. Kanon, O. Bensaude, H.H. Kampina, Heat shock protein (Hsp) 40 mutants inhibit Hsp70 in mammalian cells, J. Biol. Chem. 274 (1999) 36757–36763.

[39] D. Wall, M. Zylicz, C. Georgopoulos, The NH2-terminal 108 amino acids of the *Escherichia coli* DnaJ protein stimulate the ATPase activity of DnaK and are sufficient for lambda replication, J. Biol. Chem. 269 (1994) 5446–5451.

[40] M.E. Cheetham, A.J. Caplan, Structure, function and evolution of DnaJ: conservation and adaptation of chaperone function, Cell Stress Chaperones 3 (1998) 28–36.

[41] B. Banecki, K. Liberek, D. Wall, A. Wawrzynow, C. Georgopoulos, E. Bertoli, F. Tanfani, M. Zylicz, Structure–function analysis of the zinc finger region of the DnaJ molecular chaperone, J. Biol. Chem. 271 (1996) 14840–14848.

[42] A. Szabo, R. Korszun, F.U. Hartl, J. Flanagan, A zinc finger-like domain of the molecular chaperone DnaJ is involved in binding to denatured protein substrates, EMBO J. 15 (1996) 408–417.

[43] S. Rüdiger, J. Schneider-Mergener, B. Bukau, Its substrate specificity characterizes the DnaJ co-chaperone as a scanning factor for the DnaK chaperone, EMBO J. 20 (2001) 1042–1050.

[44] C.J. Harrison, M. Hayer-Hartl, M. Di Liberto, F. Hartl, J. Kuriyan, Crystal structure of the nucleotide exchange factor GrpE bound to the ATPase domain of the molecular chaperone DnaK, Science 276 (1997) 431–435.

[45] H.J. Schönfeld, D. Schmidt, H. Schroder, B. Bukau, The DnaK chaperone system of *Escherichia coli*: quaternary structures and interactions of the DnaK and GrpE components, J. Biol. Chem. 270 (1995) 2183–2189.

[46] C.S. Gassler, T. Wiederkehr, D. Brehmer, B. Bukau, M.P. Mayer, Bag-1M accelerates nucleotide release for human Hsc70 and Hsp70 and can act concentration-dependent as positive and negative cofactor, J. Biol. Chem. 276 (2001) 32538–32544.

[47] D.A. Raynes, V. Guerrierio Jr., Inhibition of Hsp70 ATPase activity and protein renaturation by a novel Hsp70-binding protein, J. Biol. Chem. 273 (1998) 32883–32888.

[48] Y. Shomura, Z. Dragovic, H.C. Chang, N. Tzvetkov, J.C. Young, J.L. Brodsky, V. Guerrierio, F.U. Hartl, A. Bracher, Regulation of Hsp70 function by HspBP1: structural analysis reveals an alternate mechanism for Hsp70 nucleotide exchange, Mol. Cell 17 (2005) 367–379.

[49] J. de Keyzer, G.J. Steel, S.J. Hale, D. Humphreys, C.J. Stirling, Nucleotide binding by LHS1P is essential for its nucleotide exchange activity and for function in vivo, J. Biol. Chem. 284 (2009) 31564–31571.

[50] G.J. Steel, D.M. Fullerton, J.R. Tyson, C.J. Stirling, Coordinated activation of Hsp70 chaperones, Science 303 (2004) 98–101.

[51] Z. Dragovic, S.A. Broadley, Y. Shomura, A. Bracher, F.U. Hartl, Molecular chaperones of the Hsp110 family act as nucleotide exchange factors of Hsp70s, EMBO J. 25 (2006) 2519–2528.

[52] W.M. Obermann, H. Sondermann, A.A. Russo, N.P. Pavletich, F.U. Hartl, In vivo function of Hsp90 is dependent on ATP binding and ATP hydrolysis, J. Cell Biol. 143 (1998) 901–910.

[53] P. Meyer, C. Prodromou, B. Hu, C. Vaughan, S.M. Roe, B. Panaretou, P.W. Piper, L.H. Pearl, Structural and functional analysis of the middle segment of hsp90: implications for ATP hydrolysis and client protein and cochaperone interactions, Mol. Cell 11 (2003) 647–658.

[54] C. Prodromou, B. Panaretou, S. Chohan, G. Siligardi, R. O'Brien, J.E. Ladbury, S.M. Roe, P.W. Piper, L.H. Pearl, The ATPase cycle of Hsp90 drives a molecular ‘clamp’ via transient dimerization of the N-terminal domains, EMBO J. 19 (2000) 4383–4392.

[55] S.F. Harris, A.K. Shiau, D.A. Agard, The crystal structure of the carboxy-terminal dimerization domain of htpG, the *Escherichia coli* Hsp90, reveals a potential substrate binding site, Structure 12 (2004) 1087–1097.

[56] J.P. Grenert, B.D. Johnson, D.O. Toft, The importance of ATP binding and hydrolysis by hsp90 in formation and function of protein heterocomplexes, J. Biol. Chem. 274 (1999) 17525–17533.

[57] B. Panaretou, C. Prodromou, S.M. Roe, R. O'Brien, J.E. Ladbury, P.W. Piper, L.H. Pearl, ATP binding and hydrolysis are essential to the function of the Hsp90 molecular chaperone in vivo, EMBO J. 17 (1998) 4829–4836.

[58] M. Mickler, M. Hessling, C. Ratzke, J. Buchner, T. Hugel, The large conformational changes of Hsp90 are only weakly coupled to ATP hydrolysis, Nat. Struct. Mol. Biol. 16 (2009) 281–286.

[59] M. Hessling, K. Richter, J. Buchner, Dissection of the ATP-induced conformational cycle of the molecular chaperone Hsp90, Nat. Struct. Mol. Biol. 16 (2009) 287–293.

[60] S.K. Wandinger, K. Richter, J. Buchner, The Hsp90 chaperone machinery, J. Biol. Chem. 283 (2008) 18473–18477.

[61] J.A. Carver, R.A. Lindner, C. Lyon, D. Canet, H. Hernandez, C.M. Dobson, C. Redfield, The interaction of the molecular chaperone alpha-crystallin with unfolding alpha-lactalbumin: a structural and kinetic spectroscopic study, J. Mol. Biol. 318 (2002) 815–827.

[62] R.A. Lindner, T.M. Treweek, J.A. Carver, The molecular chaperone alpha-crystallin is in kinetic competition with aggregation to stabilize a monomeric molten-globule form of alpha-lactalbumin, Biochem. J. 354 (2001) 79–87.

[63] A. Biswas, K.P. Das, Role of ATP on the interaction of alpha-crystallin with its substrates and its implications for the molecular chaperone function, J. Biol. Chem. 279 (2004) 42648–42657.

[64] E. Basha, G.J. Lee, L.A. Breci, A.C. Hausrath, N.R. Buan, K.C. Giese, E. Vierling, The identity of proteins associated with a small heat shock protein during heat stress in vivo indicates that these chaperones protect a wide range of cellular functions, J. Biol. Chem. 279 (2004) 7566–7575.

[65] K.L. Friedrich, K.C. Giese, N.R. Buan, E. Vierling, Interactions between small heat shock protein subunits and substrate in small heat shock protein–substrate complexes, J. Biol. Chem. 279 (2004) 1080–1089.

[66] B.K. Derham, M.A. van Boekel, P.J. Muchowski, J.I. Clark, J. Horwitz, H.W. Hepburne-Scott, W.W. de Jong, M.J. Crabbe, J.J. Harding, Chaperone function of mutant versions of alpha-A- and alpha-B-crystallin prepared to pinpoint chaperone binding sites, Eur. J. Biochem. 268 (2001) 713–721.

[67] N. Lentze, S. Studer, F. Narberhaus, Structural and functional defects caused by point mutations in the alpha-crystallin domain of a bacterial alpha-heat shock protein, J. Mol. Biol. 328 (2003) 927–937.

[68] P.J. Muchowski, G.J. Wu, J.J. Liang, E.T. Adman, J.I. Clark, Site-directed mutations within the core “alpha-crystallin” domain of the small heat-shock protein, human alphaB-crystallin, decrease molecular chaperone functions, J. Mol. Biol. 289 (1999) 397–411.

[69] P.L. Wintrode, K.L. Friedrich, E. Vierling, J.B. Smith, D.L. Smith, Solution structure and dynamics of a heat shock protein assembly probed by hydrogen exchange and mass spectrometry, Biochemistry 42 (2003) 10667–10673.

[70] S. Nishikawa, J.L. Brodsky, K. Nakatsukasa, Roles of molecular chaperones in endoplasmic reticulum (ER) quality control and ER-associated degradation (ERAD), J. Biochem. 137 (2005) 551–555.

[71] B.P. Tu, J.S. Weissman, Oxidative protein folding in eukaryotes: mechanisms and consequences, J. Cell Biol. 164 (2004) 341–346.

[72] C. Appenzeller-Herzog, L. Ellgaard, The human PDI family: versatility packed into a single fold, Biochim. Biophys. Acta 1783 (2008) 535–548.

[73] H. Cai, C.C. Wang, C.L. Tsou, Chaperone-like activity of protein disulfide isomerase in the refolding of a protein with no disulfide bonds, J. Biol. Chem. 269 (1994) 24550–24552.

[74] J.L. Song, C.C. Wang, Chaperone-like activity of protein disulfide-isomerase in the refolding of rhodanese, Eur. J. Biochem. 231 (1995) 312–316.

[75] H.I. Alanen, K.E. Salo, A. Pirneskoski, L.W. Ruddock, pH dependence of the peptide thiol-disulfide oxidase activity of six members of the human protein disulfide isomerase family, Antioxid. Redox Signal. 8 (2006) 283–291.

[76] R. Ushioda, J. Hoseki, K. Araki, G. Jansen, D.Y. Thomas, K. Nagata, ERdj5 is required as a disulfide reductase for degradation of misfolded proteins in the ER, Science 321 (2008) 569–572.

[77] J.H. Cormier, T. Tamura, J.C. Sunryd, D.N. Hebert, EDEM1 recognition and delivery of misfolded proteins to the SEL1L-containing ERAD complex, Mol. Cell 34 (2009) 627–633.

[78] R. Gauss, E. Jarosch, T. Sommer, C. Hirsch, A complex of Yos9p and the HRD ligase integrates endoplasmic reticulum quality control into the degradation machinery, Nat. Cell Biol. 8 (2006) 849–854.

[79] S. Svedine, T. Wang, R. Halaban, D.N. Hebert, Carbohydrates act as sorting determinants in ER-associated degradation of tyrosinase, J. Cell Sci. 117 (2004) 2937–2949.

[80] F. Lipari, A. Herscovics, Calcium binding to the class I alpha-1, 2-mannosidase from Saccharomyces cerevisiae occurs outside the EF hand motif, Biochemistry 38 (1999) 1111–1118.

[81] F. Valleé, F. Lipari, P. Yip, B. Sleno, A. Herscovics, P.L. Howell, Crystal structure of a class I alpha1, 2-mannosidase involved in N-glycan processing and endoplasmic reticulum quality control, EMBO J. 19 (2000) 581–588.

[82] S. Munro, The MRH domain suggests a shared ancestry for the mannose 6-phosphate receptors and other N-glycan-recognising proteins, Curr. Biol. 11 (2001) R499–R501.

[83] A. Bhamidipati, V. Denic, E.M. Quan, J.S. Weissman, Exploration of the topological requirements of ERAD identifies Yos9p as a lectin sensor of misfolded glycoproteins in the ER lumen, Mol. Cell 19 (2005) 741–751.

[84] S. Vashist, D.T. Ng, Misfolded proteins are sorted by a sequential checkpoint mechanism of ER quality control, J. Cell Biol. 165 (2004) 41–52.

[85] P. Carvalho, V. Goder, T.A. Rapoport, Distinct ubiquitin-ligase complexes define convergent pathways for the degradation of ER proteins, Cell 126 (2006) 361–373.

[86] A. Finger, M. Knop, D.H. Wolf, Analysis of two mutated vacuolar proteins reveals a degradation pathway in the endoplasmic reticulum or a related compartment of yeast, Eur. J. Biochem. 218 (1993) 565–574.

[87] I.M. Nilsson, G. von Heijne, Determination of the distance between the oligosaccharyltransferase active site and the endoplasmic reticulum membrane, J. Biol. Chem. 268 (1993) 5798–5801.

[88] R.K. Plemper, P.M. Deak, R.T. Otto, D.H. Wolf, Re-entering the translocon from the lumenal side of the endoplasmic reticulum. Studies on mutated carboxypeptidase yscY species, FEBS Lett. 443 (1999) 241–245.

[89] A. Schäfer, D.H. Wolf, Sec61p is part of the endoplasmic reticulum-associated degradation machinery, EMBO J. 28 (2009) 2874–2884.

[90] K.E. Matlack, B. Misselwitz, K. Plath, T.A. Rapoport, BiP acts as a molecular ratchet during posttranslational transport of prepro-alpha factor across the ER membrane, Cell 97 (1999) 553–564.

[91] S. Nishikawa, S.W. Fewell, Y. Kato, J.L. Brodsky, T. Endo, Molecular chaperones in the yeast endoplasmic reticulum maintain the solubility of proteins for retrotranslocation and degradation, J. Cell Biol. 153 (2001) 1061–1070.

[92] R.K. Plemper, S. Böhmler, J. Bordallo, T. Sommer, D.H. Wolf, Mutant analysis links the translocon and BiP to retrograde protein transport for ER degradation, Nature 388 (1997) 891–895.

[93] J.L. Brodsky, E.D. Werner, M.E. Dubas, J.L. Goeckeler, K.B. Kruse, A.A. McCracken, The requirement for molecular chaperones during endoplasmic reticulum-associated protein degradation demonstrates that protein export and import are mechanistically distinct, J. Biol. Chem. 274 (1999) 3453–3460.

[94] M.R. Knittler, S. Dirks, I.G. Haas, Molecular chaperones involved in protein degradation in the endoplasmic reticulum: quantitative interaction of the heat shock cognate protein BiP with partially folded immunoglobulin light chains that are degraded in the endoplasmic reticulum, Proc. Natl. Acad. Sci. U. S. A. 92 (1995) 1764–1768.

[95] S. Silberstein, G. Schlenstedt, P.A. Silver, R. Gilmore, A role for the DnaJ homologue Scj1p in protein folding in the yeast endoplasmic reticulum, J. Cell Biol. 143 (1998) 921–933.

[96] M. Carla Fama, D. Raden, N. Zacchi, D.R. Lemos, A.S. Robinson, S. Silberstein, The Saccharomyces cerevisiae YFR041C/ERJ5 gene encoding a type I membrane protein with a J domain is required to preserve the folding capacity of the endoplasmic reticulum, Biochim. Biophys. Acta 1773 (2007) 232–242.

[97] A. Weitzmann, J. Volkmer, R. Zimmermann, The nucleotide exchange factor activity of Grp170 may explain the non-lethal phenotype of loss of Sil1 function in man and mouse, FEBS Lett. 580 (2006) 5237–5240.

[98] R.A. Craven, M. Egerton, C.J. Stirling, A novel Hsp70 of the yeast ER lumen is required for the efficient translocation of a number of protein precursors, EMBO J. 15 (1996) 2640–2650.

[99] J.R. Tyson, C.J. Stirling, LHS1 and SIL1 provide a luminal function that is essential for protein translocation into the endoplasmic reticulum, EMBO J. 19 (2000) 6440–6452.

[100] M. Kabani, J.M. Beckerich, C. Gaillard, Sls1p stimulates Sec63p-mediated activation of Kar2p in a conformation-dependent manner in the yeast endoplasmic reticulum, Mol. Cell. Biol. 20 (2000) 6923–6934.

[101] M. Angeles Juanes, J. Carlos Igual, M. Carmen Bano, Membrane topology and post-translational modification of the Saccharomyces cerevisiae essential protein Rot1, Yeast 25 (2008) 93–106.

[102] M. Takeuchi, Y. Kimata, A. Hirata, M. Oka, K. Kohno, Saccharomyces cerevisiae Rot1p is an ER-localized membrane protein that may function with BiP/Kar2p in protein folding, J. Biochem. 139 (2006) 597–605.

[103] M. Bickle, P.A. Delley, A. Schmidt, M.N. Hall, Cell wall integrity modulates RHO1 activity via the exchange factor ROM2, EMBO J. 17 (1998) 2235–2245.

[104] K. Machi, M. Azuma, K. Igarashi, T. Matsumoto, H. Fukuda, A. Kondo, H. Oshima, Rot1p of Saccharomyces cerevisiae is a putative membrane protein required for normal levels of the cell wall 1, 6-beta-glucan, Microbiology 150 (2004) 3163–3173.

[105] M. Takeuchi, Y. Kimata, K. Kohno, Saccharomyces cerevisiae Rot1 is an essential molecular chaperone in the endoplasmic reticulum, Mol. Biol. Cell 19 (2008) 3514–3525.

[106] A. Helenius, How N-linked oligosaccharides affect glycoprotein folding in the endoplasmic reticulum, Mol. Biol. Cell 5 (1994) 253–265.

[107] M. Aebi, R. Bernasconi, S. Clerc, M. Molinari, N-glycan structures: recognition and processing in the ER, Trends Biochem. Sci. 35 (2010) 74–82.

[108] A. Helenius, M. Aebi, Roles of N-linked glycans in the endoplasmic reticulum, Annu. Rev. Biochem. 73 (2004) 1019–1049.

[109] C. Hammond, I. Braakman, A. Helenius, Role of N-linked oligosaccharide recognition, glucose trimming, and calnexin in glycoprotein folding and quality control, Proc. Natl. Acad. Sci. U. S. A. 91 (1994) 913–917.

[110] J.J. Caramelo, O.A. Castro, L.G. Alonso, G. De Prat-Gay, A.J. Parodi, UDP-Glc: glycoprotein glucosyltransferase recognizes structured and solvent accessible hydrophobic patches in molten globule-like folding intermediates, Proc. Natl. Acad. Sci. U. S. A. 100 (2003) 86–91.

[111] S.C. Taylor, P. Thibault, D.C. Tessier, J.J. Bergeron, D.Y. Thomas, Glycopeptide specificity of the secretory protein folding sensor UDP-glucose glycoprotein glucosyltransferase, EMBO Rep. 4 (2003) 405–411.

[112] C. Ritter, K. Quirin, M. Kowarik, A. Helenius, Minor folding defects trigger local modification of glycoproteins by the ER folding sensor GT, EMBO J. 24 (2005) 1730–1738.

[113] Y. Oda, N. Hosokawa, I. Wada, K. Nagata, EDEM as an acceptor of terminally misfolded glycoproteins released from calnexin, Science 299 (2003) 1394–1397.

[114] J. Chillaron, C. Adan, I.G. Haas, Mannosidase action, independent of glucose trimming, is essential for proteasome-mediated degradation of unassembled glycosylated Ig light chains, Biol. Chem. 381 (2000) 1155–1164.

[115] C. Fagioli, R. Sitia, Glycoprotein quality control in the endoplasmic reticulum. Mannose trimming by endoplasmic reticulum mannosidase I times the proteasomal degradation of unassembled immunoglobulin subunits, J. Biol. Chem. 276 (2001) 12885–12892.

[116] M. Molinari, V. Calanca, C. Galli, P. Lucca, P. Paganetti, Role of EDEM in the release of misfolded glycoproteins from the calnexin cycle, Science 299 (2003) 1397–1400.

[117] F. Parlati, M. Dominguez, J.J. Bergeron, D.Y. Thomas, Saccharomyces cerevisiae CNE1 encodes an endoplasmic reticulum (ER) membrane protein with sequence similarity to calnexin and calreticulin and functions as a constituent of the ER quality control apparatus, J. Biol. Chem. 270 (1995) 244–253.

[118] X. Xu, K. Kanbara, H. Azakami, A. Kato, Expression and characterization of Saccharomyces cerevisiae Cne1p, a calnexin homologue, J. Biochem. 135 (2004) 615–618.

[119] X. Xu, H. Azakami, A. Kato, P-domain and lectin site are involved in the chaperone function of Saccharomyces cerevisiae calnexin homologue, FEBS Lett. 570 (2004) 155–160.

[120] Z. Kostova, D.H. Wolf, Importance of carbohydrate positioning in the recognition of mutated CPY for ER-associated degradation, J. Cell Sci. 118 (2005) 1485–1492.

[121] H. Zhang, J. He, Y. Ji, A. Kato, Y. Song, The effect of calnexin deletion on the expression level of PDI in Saccharomyces cerevisiae under heat stress conditions, Cell. Mol. Biol. Lett. 13 (2008) 38–48.

[122] H. Zhang, B. Hu, Y. Ji, A. Kato, Y. Song, The effect of calnexin deletion on the expression level of binding protein (BiP) under heat stress conditions in Saccharomyces cerevisiae, Cell. Mol. Biol. Lett. 13 (2008) 621–631.

[123] T. Kimura, Y. Hosoda, Y. Sato, Y. Kitamura, T. Ikeda, T. Horibe, M. Kikuchi, Interactions among yeast protein-disulfide isomerase proteins influence their activities, J. Biol. Chem. 280 (2005) 31438–31441.

[124] J.D. Oliver, H.L. Roderick, D.H. Llewellyn, S. High, ERp57 functions as a subunit of specific complexes formed with the ER lectins calreticulin and calnexin, Mol. Biol. Cell 10 (1999) 2573–2582.

[125] A. Helenius, E.S. Trombetta, D.N. Hebert, J.F. Simons, Calnexin, calreticulin and the folding of glycoproteins, Trends Cell Biol. 7 (1997) 193–200.

[126] M. Knop, N. Hauser, D.H. Wolf, N-Glycosylation affects endoplasmic reticulum degradation of a mutated derivative of carboxypeptidase yscY in yeast, Yeast 12 (1996) 1229–1238.

[127] N. Hosokawa, I. Wada, K. Hasegawa, T. Yorihuzi, L.O. Tremblay, A. Herscovics, K. Nagata, A novel ER alpha-mannosidase-like protein accelerates ER-associated degradation, EMBO Rep. 2 (2001) 415–422.

[128] C.A. Jakob, D. Bodmer, U. Spirig, P. Battig, A. Marcil, D. Dignard, J.J. Bergeron, D.Y. Thomas, M. Aebi, Htm1p, a mannosidase-like protein, is involved in glycoprotein degradation in yeast, EMBO Rep. 2 (2001) 423–430.

[129] K. Nakatsukasa, S. Nishikawa, N. Hosokawa, K. Nagata, T. Endo, Mnl1p, an alpha-mannosidase-like protein in yeast Saccharomyces cerevisiae, is required for endoplasmic reticulum-associated degradation of glycoproteins, J. Biol. Chem. 276 (2001) 8635–8638.

[130] E.M. Quan, Y. Kamiya, D. Kamiya, V. Denic, J. Weibezahn, K. Kato, J.S. Weissman, Defining the glycan destruction signal for endoplasmic reticulum-associated degradation, Mol. Cell 32 (2008) 870–877.

[131] S. Clerc, C. Hirsch, D.M. Oggier, P. Deprez, C. Jakob, T. Sommer, M. Aebi, Htm1 protein generates the N-glycan signal for glycoprotein degradation in the endoplasmic reticulum, J. Cell Biol. 184 (2009) 159–172.

[132] E.D. Spear, D.T. Ng, Single, context-specific glycans can target misfolded glycoproteins for ER-associated degradation, J. Cell Biol. 169 (2005) 73–82.

[133] W. Xie, K. Kanehara, A. Sayeed, D.T. Ng, Intrinsic conformational determinants signal protein misfolding to the Hrd1/Htm1 endoplasmic reticulum-associated degradation system, Mol. Biol. Cell 20 (2009) 3317–3329.

[134] F. Foulquier, S. Duvet, A. Klein, A.M. Mir, F. Chirat, R. Cacan, Endoplasmic reticulum-associated degradation of glycoproteins bearing Man5GlcNAc2 and Man9GlcNAc2 species in the MI8-5 CHO cell line, Eur. J. Biochem. 271 (2004) 398–404.

[135] Z. Frenkel, W. Gregory, S. Kornfeld, G.Z. Lederkremer, Endoplasmic reticulum-associated degradation of mammalian glycoproteins involves sugar chain trimming to Man6-5GlcNAc2, J. Biol. Chem. 278 (2003) 34119–34124.

[136] N. Hosokawa, L.O. Tremblay, Z. You, A. Herscovics, I. Wada, K. Nagata, Enhancement of endoplasmic reticulum (ER) degradation of misfolded Null Hong Kong alpha1-antitrypsin by human ER mannosidase I, J. Biol. Chem. 278 (2003) 26287–26294.

[137] C. Kitzmuller, A. Caprini, S.E. Moore, J.P. Frenoy, E. Schwaiger, O. Kellermann, N.E. Ivessa, M. Ermonval, Processing of N-linked glycans during endoplasmic reticulum-associated degradation of a short-lived variant of ribophorin I, Biochem. J. 376 (2003) 687–696.

[138] R.B. Freedman, T.R. Hirst, M.F. Tuite, Protein disulphide isomerase: building bridges in protein folding, Trends Biochem. Sci. 19 (1994) 331–336.

[139] T.P. Primm, K.W. Walker, H.F. Gilbert, Facilitated protein aggregation. Effects of calcium on the chaperone and anti-chaperone activity of protein disulfide-isomerase, J. Biol. Chem. 271 (1996) 33664–33669.

[140] R. Farquhar, N. Honey, S.J. Murant, P. Bossier, L. Schultz, D. Montgomery, R.W. Ellis, R.B. Freedman, M.F. Tuite, Protein disulfide isomerase is essential for viability in Saccharomyces cerevisiae, Gene 108 (1991) 81–89.

[141] H. Tachikawa, Y. Takeuchi, W. Funahashi, T. Miura, X.D. Gao, D. Fujimoto, T. Mizunaga, K. Onodera, Isolation and characterization of a yeast gene, MPD1, the overexpression of which suppresses inviability caused by protein disulfide isomerase depletion, FEBS Lett. 369 (1995) 212–216.

[142] H. Tachikawa, W. Funahashi, Y. Takeuchi, H. Nakanishi, R. Nishihara, S. Katoh, X.D. Gao, T. Mizunaga, D. Fujimoto, Overproduction of Mpd2p suppresses the lethality of protein disulfide isomerase depletion in a CXXC sequence dependent manner, Biochem. Biophys. Res. Commun. 239 (1997) 710–714.

[143] C. Tachibana, T.H. Stevens, The yeast EUG1 gene encodes an endoplasmic reticulum protein that is functionally related to protein disulfide isomerase, Mol. Cell. Biol. 12 (1992) 4601–4611.

[144] Q. Wang, A. Chang, Eps1, a novel PDI-related protein involved in ER quality control in yeast, EMBO J. 18 (1999) 5972–5982.

[145] Q. Wang, A. Chang, Substrate recognition in ER-associated degradation mediated by Eps1, a member of the protein disulfide isomerase family, EMBO J. 22 (2003) 3792–3802.

[146] P. Gillece, J.M. Luz, W.J. Lennarz, F.J. de La Cruz, K. Römisch, Export of a cysteine-free misfolded secretory protein from the endoplasmic reticulum for degradation requires interaction with protein disulfide isomerase, J. Cell Biol. 147 (1999) 1443–1456.

[147] M. Sakoh-Nakatogawa, S. Nishikawa, T. Endo, Roles of protein-disulfide isomerase-mediated disulfide bond formation of yeast Mnl1p in endoplasmic reticulum-associated degradation, J. Biol. Chem. 284 (2009) 11815–11825.

[148] Y. Kimura, M. Nakazawa, M. Yamada, Cloning and characterization of three isoforms of OS-9 cDNA and expression of the OS-9 gene in various human tumor cell lines, J. Biochem. (Tokyo) 123 (1998) 876–882.

[149] B. Buschhorn, Z. Kostova, B. Medicherla, D.H. Wolf, A genome wide screen identifies Yos9p as a new lectin essential for ER-associated degradation (ERAD) of glycoproteins, FEBS Lett. 577 (2004) 422–426.

[150] W. Kim, E.D. Spear, D.T. Ng, Yos9p detects and targets misfolded glycoproteins for ER-associated degradation, Mol. Cell 19 (2005) 753–764.

[151] R. Szathmary, R. Bielmann, M. Nita-Lazar, P. Burda, C.A. Jakob, Yos9 protein is essential for degradation of misfolded glycoproteins and may function as lectin in ERAD, Mol. Cell 19 (2005) 765–775.

[152] V. Denic, E.M. Quan, J.S. Weissman, A luminal surveillance complex that selects misfolded glycoproteins for ER-associated degradation, Cell 126 (2006) 349–359.

[153] J.C. Christianson, T.A. Shaler, R.E. Tyler, R.R. Kopito, OS-9 and GRP94 deliver mutant alpha1-antitrypsin to the Hrd1-SEL1L ubiquitin ligase complex for ERAD, Nat. Cell Biol. 10 (2008) 272–282.

[154] B. Müller, B.N. Lilley, H.L. Ploegh, SEL1L, the homologue of yeast Hrd3p, is involved in protein dislocation from the mammalian ER, J. Cell Biol. 175 (2006) 261–270.

[155] C. Taxis, R. Hitt, S.H. Park, P.M. Deak, Z. Kostova, D.H. Wolf, Use of modular substrates demonstrates mechanistic diversity and reveals differences in chaperone requirement of ERAD, J. Biol. Chem. 278 (2003) 35903–35913.

[156] G. Huyer, W.F. Piluek, Z. Fansler, S.G. Kreft, M. Hochstrasser, J.L. Brodsky, S. Michaelis, Distinct machinery is required in Saccharomyces cerevisiae for the endoplasmic reticulum-associated degradation of a multispanning membrane protein and a soluble luminal protein, J. Biol. Chem. 279 (2004) 38369–38378.

[157] K. Nakatsukasa, G. Huyer, S. Michaelis, J.L. Brodsky, Dissecting the ER-associated degradation of a misfolded polytopic membrane protein, Cell 132 (2008) 101–112.

[158] S. Han, Y. Liu, A. Chang, Cytoplasmic Hsp70 promotes ubiquitination for endoplasmic reticulum-associated degradation of a misfolded mutant of the yeast plasma membrane ATPase, PMA1, J. Biol. Chem. 282 (2007) 26140–26149.

[159] R.K. Plemper, R. Egner, K. Kuchler, D.H. Wolf, Endoplasmic reticulum degradation of a mutated ATP-binding cassette transporter Pdr5 proceeds in a concerted action of Sec61 and the proteasome, J. Biol. Chem. 273 (1998) 32848–32856.

[160] S.L. Hrizo, V. Gusarova, D.M. Habiel, J.L. Goeckeler, E.A. Fisher, J.L. Brodsky, The Hsp110 molecular chaperone stabilizes apolipoprotein B from endoplasmic reticulum-associated degradation (ERAD), J. Biol. Chem. 282 (2007) 32665–32675.

[161] R.T. Youker, P. Walsh, T. Beilharz, T. Lithgow, J.L. Brodsky, Distinct roles for the Hsp40 and Hsp90 molecular chaperones during cystic fibrosis transmembrane conductance regulator degradation in yeast, Mol. Biol. Cell 15 (2004) 4787–4797.

[162] A. Ahner, K. Nakatsukasa, H. Zhang, R.A. Frizzell, J.L. Brodsky, Small heat-shock proteins select deltaF508-CFTR for endoplasmic reticulum-associated degradation, Mol. Biol. Cell 18 (2007) 806–814.

[163] Y. Zhang, G. Nijbroek, M.L. Sullivan, A.A. McCracken, S.C. Watkins, S. Michaelis, J.L. Brodsky, Hsp70 molecular chaperone facilitates endoplasmic reticulum-associated protein degradation of cystic fibrosis transmembrane conductance regulator in yeast, Mol. Biol. Cell 12 (2001) 1303–1314.

[164] T.J. Jensen, M.A. Loo, S. Pind, D.B. Williams, A.L. Goldberg, J.R. Riordan, Multiple proteolytic systems, including the proteasome, contribute to CFTR processing, Cell 83 (1995) 129–135.

[165] C.L. Ward, S. Omura, R.R. Kopito, Degradation of CFTR by the ubiquitin-proteasome pathway, Cell 83 (1995) 121–127.

[166] A. Gnann, J.R. Riordan, D.H. Wolf, Cystic fibrosis transmembrane conductance regulator degradation depends on the lectins Htm1p/EDEM and the Cdc48 protein complex in yeast, Mol. Biol. Cell 15 (2004) 4125–4135.

[167] V. Gusarova, A.J. Caplan, J.L. Brodsky, E.A. Fisher, Apoprotein B degradation is promoted by the molecular chaperones hsp90 and hsp70, J. Biol. Chem. 276 (2001) 24891–24900.

[168] G.C. Meacham, Z. Lu, S. King, E. Sorscher, A. Tousson, D.M. Cyr, The Hdj-2/Hsc70 chaperone pair facilitates early steps in CFTR biogenesis, EMBO J. 18 (1999) 1492–1505.

[169] J.R. Glover, S. Lindquist, Hsp104, Hsp70, and Hsp40: a novel chaperone system that rescues previously aggregated proteins, Cell 94 (1998) 73–82.

[170] H. Raviol, H. Sadlish, F. Rodriguez, M.P. Mayer, B. Bukau, Chaperone network in the yeast cytosol: Hsp110 is revealed as an Hsp70 nucleotide exchange factor, EMBO J. 25 (2006) 2510–2518.

[171] L. Shaner, H. Wegele, J. Buchner, K.A. Morano, The yeast Hsp110 Sse1 functionally interacts with the Hsp70 chaperones Ssa and Ssb, J. Biol. Chem. 280 (2005) 41262–41269.

[172] J.P. Schuermann, J. Jiang, J. Cuellar, O. Llorca, L. Wang, L.E. Gimenez, S. Jin, A.B. Taylor, B. Demeler, K.A. Morano, P.J. Hart, J.M. Valpuesta, E.M. Lafer, R. Sousa, Structure of the Hsp110:Hsc70 nucleotide exchange machine, Mol. Cell 31 (2008) 232–243.

[173] H. Sadlish, H. Rampelt, J. Shorter, R.D. Wegrzyn, C. Andreasson, S. Lindquist, B. Bukau, Hsp110 chaperones regulate prion formation and propagation in S. cerevisiae by two discrete activities, PLoS One 3 (2008) e1763.

[174] E.A. Fisher, M. Zhou, D.M. Mitchell, X. Wu, S. Omura, H. Wang, A.L. Goldberg, H.N. Ginsberg, The degradation of apolipoprotein B100 is mediated by the ubiquitin-proteasome pathway and involves heat shock protein 70, J. Biol. Chem. 272 (1997) 20427–20434.

[175] V. Albanese, A.Y. Yam, J. Baughman, C. Parnot, J. Frydman, Systems analyses reveal two chaperone networks with distinct functions in eukaryotic cells, Cell 124 (2006) 75–88.

[176] J.L. Goeckeler, A.P. Petruso, J. Aguirre, C.C. Clement, G. Chiosis, J.L. Brodsky, The yeast Hsp110, Sse1p, exhibits high-affinity peptide binding, FEBS Lett. 582 (2008) 2393–2396.

[177] M.A. Loo, T.J. Jensen, L. Cui, Y. Hou, X.B. Chang, J.R. Riordan, Perturbation of Hsp90 interaction with nascent CFTR prevents its maturation and accelerates its degradation by the proteasome, EMBO J. 17 (1998) 6879–6887.

[178] C. Schneider, L. Sepp-Lorenzino, E. Nimmesgern, O. Ouerfelli, S. Danishefsky, N. Rosen, F.U. Hartl, Pharmacologic shifting of a balance between protein refolding and degradation mediated by Hsp90, Proc. Natl. Acad. Sci. U. S. A. 93 (1996) 14536–14541.

[179] R.M. Ferreira, L.R. de Andrade, M.B. Dutra, M.F. de Souza, V.M. Floisi Paschoalin, J.T. Silva, Purification and characterization of the chaperone-like Hsp26 from Saccharomyces cerevisiae, Protein Expr. Purif. 47 (2006) 384–392.

[180] M. Haslbeck, N. Braun, T. Stromer, B. Richter, N. Model, S. Weinkauf, J. Buchner, Hsp42 is the general small heat shock protein in the cytosol of Saccharomyces cerevisiae, EMBO J. 23 (2004) 638–649.

[181] M. Haslbeck, S. Walke, T. Stromer, M. Ehrnsperger, H.E. White, S. Chen, H.R. Saibil, J. Buchner, Hsp26: a temperature-regulated chaperone, EMBO J. 18 (1999) 6744–6751.

[182] H.E. White, E.V. Orlova, S. Chen, L. Wang, A. Ignatiou, B. Gowen, T. Stromer, T.M. Franzmann, M. Haslbeck, J. Buchner, H.R. Saibil, Multiple distinct assemblies reveal conformational flexibility in the small heat shock protein Hsp26, Structure 14 (2006) 1197–1204.

[183] C. Schuberth, A. Buchberger, Membrane-bound Ubx2 recruits Cdc48 to ubiquitin ligases and their substrates to ensure efficient ER-associated protein degradation, Nat. Cell Biol. 7 (2005) 999–1006.

[184] O. Neuber, E. Jarosch, C. Volkwein, J. Walter, T. Sommer, Ubx2 links the Cdc48 complex to ER-associated protein degradation, Nat. Cell Biol. 7 (2005) 993–998.

[185] B. Medicherla, Z. Kostova, A. Schäfer, D.H. Wolf, A genomic screen identifies Dsk2p and Rad23p as essential components of ER-associated degradation, EMBO Rep. 5 (2004) 692–697.

[186] S.M. Alberts, C. Sonntag, A. Schäfer, D.H. Wolf, Ubx4 modulates cdc48 activity and influences degradation of misfolded proteins of the endoplasmic reticulum, J. Biol. Chem. 284 (2009) 16082–16089.

[187] C. Hirsch, D. Blom, H.L. Ploegh, A role for N-glycanase in the cytosolic turnover of glycoproteins, EMBO J. 22 (2003) 1036–1046.
