
REVIEW

**Complexity and simplicity in the biosynthesis of enediyne natural products**

Zhao-Xun Liang*

Received 15th December 2009  
First published as an Advance Article on the web 18th February 2010  
DOI: 10.1039/b908165h  

Covering: up to the end of 2009  

The review highlights the recent progress on the elucidation of the biosynthetic mechanism of enediyne natural products. A large body of knowledge has been gathered from the genetic, biochemical and structural studies since the sequencing of the first two enediyne gene clusters in 2002. Primary attention is devoted to the novel pathways and enzymes in the biosynthesis of the enediyne warhead as well as the peripheral moieties.

---

### Table of Contents
1. Introduction  
2. Study of enediyne biosynthesis in the pre-genomic era  
3. Enediyne gene clusters  
4. Biosynthesis of the enediyne warhead  
   - 4.1 Iterative PKS  
     - 4.1.1 Domain organization of the iterative PKS  
     - 4.1.2 Products of the iterative PKS  
   - 4.2 Structure and catalytic mechanism of the thioesterase  
   - 4.3 The putative acetylase  
   - 4.4 Two other gene products encoded by the warhead gene cassette  
   - 4.5 Maturation of the enediyne moiety  
5. Apoprotein of 9-membered enediynes  
6. Peripheral moieties of the 9-membered enediyne chromophores  
   - 6.1 C-1027  
     - 6.1.1 β-Tyrosine moiety  
     - 6.1.2 Benzoxazoline moiety  
     - 6.1.3 Deoxyaminosugar  
     - 6.1.4 Production of C-1027 analogs  
   - 6.2 Neocarzinostatin  
     - 6.2.1 Naphthoic acid  
     - 6.2.2 Deoxyaminosugar  
   - 6.3 Maduropeptin  
     - 6.3.1 Dimethylsalicyclic acid  
     - 6.3.2 Aryl hydroxypropionic acid  
     - 6.3.3 Aminosugar  
   - 6.4 Kedarcidin  
   - 6.5 N1999A2  
   - 6.6 Sporolides  
   - 6.7 Cyanosporasides  
7. Peripheral moieties of the 10-membered enediynes  
   - 7.1 Calicheamicin γ₁  
     - 7.1.1 Orsellenic acid  
     - 7.1.2 The sugar moieties  
   - 7.2 Esperamicin A₁  

7.3 Namenamicin and shishijimicins  
7.4 Dynemicins and uncialamycin  
8. Regulation of enediyne biosynthesis  
9. Self-resistance mechanisms  
10. Summary and perspective  
11. Acknowledgements  
12. References  

---

### 1. Introduction

Enediyne natural products are a family of structurally distinctive secondary metabolites produced mainly by soil and marine microorganisms. The structure of the first enediyne natural product, neocarzinostatin, was reported by Edo and co-workers in 1985.¹² The structure has a highly unusual bicyclo[7.3.0]dodecadienediyne core that contains an epoxide-masked 1,5-diyne-3-ene (or ene-diyne) unit embedded in a 9-membered ring. The structures of the calicheamicins and esperamicins determined in 1987 disclosed a different bicyclo[7.3.1]-tridecadiynene core that contains the 1,5-diyne-3-ene unit in a 10-membered ring.³⁻⁹ Since then, several other subfamilies of enediyne natural products that contain either the bicyclo[7.3.0]dodecadienediyne or bicyclo[7.3.1]-tridecadiynene core have been characterized. Based on the presence of the bicyclo[7.3.0]dodecadienediyne or bicyclo[7.3.1]-tridecadiynene core, naturally occurring enediyne chromophores are conveniently categorized into 9-membered or 10-membered families. The 9-membered family currently encompasses neocarzinostatin (1), kedarcidin (2), C-1027 (3), maduropeptin (4), N1999A2 (5) and the two putative enediyne precursors 6 and 7 (Fig. 1). Isolated from the marine bacterium *Salinospora*, 6 and 7 could only be obtained as the stable forms known as sporolides and cyanosporasides.¹⁰⁻¹² The 10-membered family includes several subfamilies. The structures of the representative calicheamicin γ₁¹ (8), esperamicin A₁ (9), dynemicin A (12) and shishijimicin A (10), as well as namenamicin (11) and uncialamycin (13), are shown in Fig. 2. The 10-membered enediynes are further divided into the calicheamicin-like enediynes (8, 9, 10 and 11) and dynemicin-like enediynes (12, 13), which are

---

Division of Chemical Biology & Biotechnology, School of Biological Sciences, Nanyang Technological University, 60 Nanyang Drive, Singapore 637551. E-mail: zxliang@ntu.edu.sg; Fax: +65 67913856; Tel: +65 63167866

This journal is © The Royal Society of Chemistry 2010  
Nat. Prod. Rep., 2010, 27, 499–528 | 499

Neocarzinostatin (1)
Kedarcidin (2)
C-1027 (3)

Maduropeptin (4)
N1999A2 (5)
Presumed precursor of sporolides (6)
Presumed precursor of cyanosporasides (7)

Fig. 1 9-Membered enediyne chromophores with the bicyclo[7.3.0]dodecadienediyne core highlighted.

characterized by a slightly different bicyclic enediyne core and an anthraquinone moiety.

Enediyne natural products have received considerable attention in the last two decades, not only for their unprecedented molecular architectures, but also because of their remarkable

antibiotic and antitumor activities. ${}^{13,14}$ In fact, the enediynes are some of the most potent antitumor natural products ever discovered. For instance, calicheamicin $\gamma_{1}{ }^{\prime}$ and shishijimicin A are approximately 4000 and 13 000-times more potent than the widely used antitumor agent adriamycin. The mode of action for the enediynes in inducing cell apoptosis and tumor cell killing has been intensively investigated. It was found that the enediynes intercalate into chromosomal DNA in a sequence-specific manner, and that the enediynes exert their antibiotic and anti-tumor activities largely by damaging chromosomal DNA. ${}^{14-23}$ Studies with synthetic and naturally occurring enediynes have established the mechanism of action, which involves the formation of a reactive enediyne diradical through the Bergman or Myers–Saito cyclization mechanism (Fig. 3). ${}^{24-26}$ For C-1027 and calicheamicin-like enediynes, the nucleophilic addition to the bicyclic enediyne “warhead” triggers the cycloaromatization and concomitant formation of the diradical intermediate. For neocarzinostatin, the free-standing chromophore is highly labile and undergoes spontaneous cycloaromatization at ambient temperature. Once locked in the minor groove of DNA, the diradical readily abstracts hydrogen atoms from DNA and generates a carbon-centered radical on the ribose. In the presence of O₂, the DNA will undergo facile double- or single-stranded cleavage through an oxidative radical mechanism.

Despite the remarkable antitumor activity, clinical usage of enediyne natural products was initially hampered by the lack of tumor cell specificity and delayed cytotoxicity. To circumvent the

Zhao-Xun Liang studied organic chemistry at Lanzhou University in China, where he received B.S./M.S. degrees in 1997 under the guidance of Prof. Qing-Xiang Guo. He did his Ph.D. study on heme proteins with Prof. Brian Hoffman at Northwestern University from 1997–2002. As a postdoctoral fellow, he worked with Prof. Judith Klinman on enzyme-catalyzed hydrogen transfer at the University of California at Berkeley. He then joined the School of Biological Sciences of Nanyang Technological University in Singapore as an assistant professor in September 2005. His group studies the structure and function of certain bacterial proteins involved in host-pathogen interactions and secondary metabolite biosynthesis.

Calicheamicin γ1I (8)

Namenamicin (11)

Esperamicin A1 (9)

Dynemicin A (12)

Shishijimicin A (10)

Uncialamycin (13)

Fig. 2 Representative 10-membered enediyne chromophores with the bicyclo[7.3.1]-tridecadiynene core highlighted.

cell-specific problem, polymer-based and antibody-conjugated derivatives of enediyne natural products have been developed.27–29 A polymer-conjugated form of neocarzinostatin was developed in 1994 for the treatment of hepatocellular carcinoma in Japan under the name of zinostatin stimalamer (SMANCS).30 Antibody-conjugated calicheamicins have also been developed to target tumor cells through specific antibody-antigen interactions. One conjugate that has been approved by the US FDA for the treatment of refractory acute myeloid leukemia is gemtuzumab ozogamicin (Mylotarg®), which consists of a chemically modified calicheamicin γ1I and a recombinant humanized IgG4 κ antibody.28 Another promising calicheamicin–antibody conjugate (inotuzumab ozogamicin (CMC-544)) developed for the treatment of non-Hodgkin’s lymphoma is currently in clinical trial.31

Total synthesis of enediyne chromophores was actively pursued in the 1980s and 1990s by organic chemists inspired by the unprecedented structure and biological activity of these natural products. The total synthesis of almost every enediyne subfamily, including the daunting calicheamicin γ1I and dynemicin A, has been accomplished.32–45 In addition, a large number of artificial or designed enediynes have been synthesized.46–55 These elaborate synthetic studies not only assisted the elucidation of the complex structures, but also shed light on the mechanism of action in DNA binding and cleavage. Given the scope of the current review, the readers are referred to existing reviews

or articles for the detailed discussion of the synthesis of the enediynes.46,47,56–58

The topic of enediyne biosynthesis has been covered by several reviews over the years with different scopes.13,57,59–62 Given the recent explosion of knowledge on the biosynthetic mechanisms, this review will strive to provide a broad and updated view of the fast-moving field. After a brief introduction to the early studies, the landmark sequencing of the enediyne gene clusters and its impact will be discussed. Current knowledge on the biosynthesis of the enediyne warhead as well as the periphery sugar and aromatic moieties will be summarized with major attention devoted to the novel pathways and enzymes. The regulation of the biosynthesis and the self-resistance mechanism will be briefly discussed towards the end of the review.

## Study of enediyne biosynthesis in the pre-genomic era

The highly unusual molecular architecture and reactivity have sparked great interest in the biosynthetic origin of the enediyne cores. As early as 1988, with the concise synthetic routes devised for the enediyne cores of calicheamicins and neocarzinostatin, Schreiber and Kiessling demonstrated that a series of steps that include electrocyclic ring closure, proton transfer and oxidation could transform 14 into a 9-membered enediyne (pathway a, Fig. 4); whereas an intra-molecular Diels–Alder reaction

Myers-Saito mechanism

![Mechanism diagram](image1.png)

Bergman mechanism

![Mechanism diagram](image2.png)

Fig. 3 Mechanisms of diradical formation for 9- and 10-membered enediynes.

![Folding pattern diagram](image3.png)

Fig. 5 Folding pattern for the 9-membered enediyne of neocarzinostatin.

![Folding patterns diagram](image4.png)

Fig. 6 Folding patterns for the 10-membered enediyne of dynemicin A.

the bicyclo[7.3.0]dodecadienediyne moiety is likely to derive from a linear precursor that consists of seven acetate units (Fig. 5). Culturing with doubly- and mixed-labeled acetate further indicated that the acetates were assembled in a head-to-tail fashion. Importantly, the two carbons of the -yne group seem to originate from the same acetate unit, which marks a striking difference between the 9- and 10-membered enediynes as discussed below. The authors further proposed that the linear precursor, probably derived from oleate or crepenynate, is truncated on both ends prior to cyclization. Subsequent oxidation, cyclization and oxygenation would furnish the fully functionalized 14-carbon enediyne core.

A few years later, another isotopic labeling study conducted by Tokiwa and coworkers revealed a very different \({}^{13}\)C-incorporation pattern for the bicyclo[7.3.1]-tridecadiynene core of dynemicin A from *Micromonospora chersina* M956-1. The authors suggested that the bicyclic enediyne core and the anthraquinone moiety of dynemicin A are derived from two heptaketide chains that consist of two sets of seven acetate units. There are two possible pathways for the linear heptaketide to fold into the final bicyclic structure (Fig. 6). Identification of the incorporated acetate units from the [1,2-\({}^{13}\)C\(_2\)] acetate feeding experiment suggested that the cyclization follows pathway a rather than b. From the results, it is also evident that the two carbons comprising the -yne group in dynemicin are derived from different acetate units, in sharp contrast to the early finding that the two carbons of the -yne group in neocarzinostatin originate from the same acetate unit. Although uncialamycin (13), the other member of the dynemicin-type enediyne subfamily, appears to contain a different enediyne core, it was postulated that 13 is derived from a dynemicin-like precursor.

A similar isotopic labeling experiment with the esperamicin-producing *Actinomadura verrucospora* suggested that the enediyne of esperamicin A\(_1\) is also assembled from seven acetate

Fig. 4 An early biosynthetic mechanism postulated for enediynes.

following the addition of one carbon unit at the acetylene terminus could convert 14 into a 10-membered enediyne (pathway b). The authors postulated that the branched molecule 14 might be the common biosynthetic precursor for both the 9- and 10-membered enediynes.

In 1989, Hensens and coworkers produced \({}^{13}\)C-enriched neocarzinostatin by feeding \({}^{13}\)C-labeled acetate to *Streptomyces carzinostaticus*. \({}^{13}\)C NMR studies not only confirmed the structures proposed by Edo and Myers, but also yielded the first clue about the biosynthetic origins of the molecular building blocks. The experiment with singly labeled acetate suggested that

Fig. 7 Folding patterns for the 10-membered enediyne of esperamicin A₁.

units in a head-to-tail fashion.⁶⁷ In this case, there are four possible pathways for the linear heptaketide to fold into the bicyclic structure (Fig. 7). Identification of the incorporated acetate units, particularly the starting acetate unit C11–C12, ruled out the pathways a, b and d. The production of esperamicin A₁ by *A. verrucospora* was significantly reduced by administering cerulenin, an inhibitor that targets the β-ketosynthase (KS) domain of both fatty acid synthase (FAS) and polyketide synthase (PKS), whereas supplementation of the culture with the fatty acid oleate did not restore the biosynthesis. These observations provided one of the early clues that the enediyne warhead is probably derived from a polyketide precursor, rather than a fatty acid one.

Together, the three isotopic labeling experiments suggested that the enediyne cores are derived from linear, probably polyketide, precursors that consist of head-to-tail coupled acetate units. The differences in isotope incorporation pattern among neocarzinostatin, calicheamicin and dynemicin, in particular the different origins of the -yne carbons, also revealed intriguing differences among the biosynthetic pathways.

## 3 Enediyne gene clusters

In the several years following the isotopic labeling experiments, little new insight was gained on the biosynthesis of the enediynes. Initial efforts to locate the putative type I or type II PKS gene on the genomic DNA of the enediyne producers by DNA hybridization were unsuccessful. Instead, by searching for the *cagA* gene that encodes the chromophore-associated apoprotein CagA (see Section 5) and two putatively conserved deoxysugar-synthesizing genes, Shen and coworkers finally succeeded in locating the DNA fragment that encompasses the enediyne gene clusters for C-1027.⁶⁸ At the same time, Thorson and co-workers identified the gene locus for calicheamicin biosynthesis by screening a genomic library for the cosmids that confer calicheamicin resistance to *M. echinospora*.¹³,⁵⁷,⁶⁹ In 2002, the two groups reached a milestone by announcing the completion of the sequencing and partial annotation of the C-1027 and calicheamicin gene clusters.⁷⁰,⁷¹ With the genetic blueprints in hand, the study of enediyne biosynthesis finally entered the genomic era.

Fig. 8 Organization of the highly conserved warhead gene cassette.

Each of the two gene clusters, with a stunning size of ~80 kb, contains 70 to 80 genes predicted to encode a variety of enzymes, transporters, transcriptional regulators and proteins with unknown functions.⁷⁰,⁷¹ Importantly, the two gene clusters contain a pair of PKS genes (*calE8* and *sgcE*) that encode two single-module PKSs sharing high sequence homology and identical domain organization. Disruption of either one of the PKS genes completely abolished the production of the metabolite, whereas gene complementation restored the biosynthesis in the mutant strains. Four more genes in the vicinity of the PKS gene were also found to be conserved between the two gene clusters. Since C-1027 and calicheamicin γ₁' do not share any common peripheral moiety, the five conserved genes are therefore most likely involved in the assembly of the enediyne warhead. The four conserved genes and the PKS gene were suggested to constitute the so-called ‘minimal enediyne PKS gene cassette’ or ‘warhead gene cassette’ (Fig. 8).⁷¹,⁷² Moreover, the functions of the gene products for the biosynthesis of peripheral building blocks, self-resistance, regulation and uptake were also tentatively assigned based on sequence homology.

Sequencing of the gene clusters for neocarzinostatin, maduropeptin and dynemicin in the following years further confirmed the conservation of the warhead gene cassette (Fig. 8).⁷³⁻⁷⁵ Thorson and coworkers also uncovered a dozen gene loci that contain the gene cassette in soil-dwelling actinomycetes strains by using a high-throughput genome scanning method.⁷² The result indicates that the distribution of enediynes is likely to be more widespread than previously realized. Moreover, genomic sequencing of the marine bacterium *Salinispora tropica* has unveiled the biosynthetic gene clusters of a large number of secondary metabolites.⁷⁶⁻⁷⁸ Two gene clusters that contain the minimal enediyne PKS gene cassettes were identified among the gene clusters. The first gene cluster was confirmed to be responsible for the biosynthesis of the enediyne precursor of sporolides, two related macrocyclic metabolites isolated from *Salinispora tropica* by Fellica in 2005.¹¹ The second gene cluster is predicted to encode a 10-membered enediyne based on its phylogenetic relationship with other minimal PKS cassette genes.⁷⁶,⁷⁹
The sequencing of the enediyne gene clusters not only confirmed the polyketide origin of the enediyne core, but also heralded the beginning of a new era for detailed elucidation of the biosynthetic pathways and mechanisms. The foundation has also been laid for exploring the possibility of generating enediyne analogs by metabolic engineering.

4 Biosynthesis of the enediyne warhead

4.1 Iterative PKS

4.1.1 Domain organization of the iterative PKS. Many known bacterial PKSs belong to type I modular PKSs or type II PKSs. In contrast, the enediyne PKSs from the biosynthetic pathways of calicheamicin γ<sup>1</sup> and C-1027 seem to resemble a family of fungal iterative type I PKSs that use a single set of catalytic domains for polyketide synthesis. The enediyne PKSs encompass several domains that were readily identified as ketosynthase (KS), acyltransferase (AT), ketoreductase (KR) and dehydratase (DH) domains based on sequence homology (Fig. 9). An acyl carrier protein (ACP) domain and a phosphopantetheinyl transferase (PPTase) domain were also postulated initially despite low sequence homology shared with other ACP and PPTase proteins. The significant sequence identity and similarity shared by CalE8 and SgcE, the two PKSs for calicheamicin and C-1027 synthesis, led to the postulation that the 9- and 10-membered enediyne cores may share a common PKS progenitor.

The identities of the ACP and PPTase domains were confirmed recently. Shen and coworkers showed that the ACP domain of SgcE is phosphopantetheinylated even in the absence of any exogenous PPTase. Meanwhile, our group has expressed the putative ACP domain of CalE8 as a free-standing protein and demonstrated that the protein could be phosphopantetheinylated by Sfp, a widely used PPTase with broad specificity. Hence, enediyne PKSs indeed contain a functional ACP domain, despite the lack of the signature GX(H/D)S(L/I) motif that can be found in many CPs. We also demonstrated that the C-terminal region of CalE8 is a functional PPTase domain capable of modifying heterogenous ACPs. Since most Sfp-like PPTases identified so far are discrete proteins, the annexation of a PPTase domain into the enediyne PKSs as well as several other PKSs is uncommon.

Although the yeast FAS also contains an internal PPTase domain, it instead resembles the AcpS-type PPTases.

In addition to the initially assigned KS, AT, KR, DH, ACP and PPTase domains, the enediyne PKSs also contain two stretches of sequence located downstream of the ACP and DH (labeled as DH<sub>1</sub> in Fig. 9) domains. No homolog can be found for the ~100 and ~150 amino acid fragments in the protein data bank (PDB). Nonetheless, our *de novo* structural modeling study suggested that the ~100 amino acid segment before the KR domain (labeled as KR<sub>2</sub> in Fig. 9) adopts an α/β fold with six β strands flanked by α-helices. The protein fold bears great similarity to the structural subdomain of the KR domains of the modular PKS DEBS and the mammalian fatty acid synthase. It is thus likely that the unknown domain KR<sub>1</sub> is in fact part of the intact KR domain (~410 aa). Moreover, we previously speculated that the ~150 amino acid sequence after the DH<sub>1</sub> domain is an evolutionary relic of an AcpS-like PPTase domain. Giving the recent surprising finding that the DH domains of the mammalian FAS and modular PKS are actually comprised of two hotdog subdomains, the sequence between the DH<sub>1</sub> and PPTase domains might be the second half of the intact DH domain. Our *de novo* structural modeling indeed suggested that the domain adopts a hotdog fold. The domain organization of the enediyne PKSs, with the two unknown regions tentatively assigned as the KR<sub>1</sub> and DH<sub>2</sub> subdomains, is depicted in Fig. 9.

4.1.2 Products of the iterative PKS. Although *E. coli* is generally not a suitable host for the expression of large PKS proteins, Shen and coworkers showed that the enediyne PKSs for C-1027 and neocarzinostatin biosynthesis could be expressed in *E. coli* as soluble proteins. Co-expression of the PKS (SgcE or NcsE) and its cognate thioesterase TE (SgcE10 or NcsE10) in *E. coli* produced a discrete isolable compound that was identified as 1,3,5,7,9,11,13-pentadecaheptaene (21) (Fig. 10). The production of the 15-carbon polyene 21 suggests that the recombinant PKSs are able to catalyze the decarboxylative condensation, ketoreduction, and dehydration steps iteratively for assembling the polyene chain. Compound 21 was postulated to derive from the ACP domain-anchored 3-hydroxy-4,6,8,10,12,14-hexadecahexaene, and to be off-loaded from the PKS by the thioesterase SgcE10 or NcsE10. *In vivo* experiments further showed that SgcE and NcsE are interchangeable within the 9-membered families of enediynes, supporting the idea that the 9-membered enediyne cores are derived from a common polyketide precursor, presumably 21.

Meanwhile, the PKS (CalE8) in calicheamicin biosynthesis was expressed in *E. coli* by our lab. The *E. coli* cells that harbor the recombinant CalE8 appeared bright yellow, in contrast to the orange coloration for the SgcE-harboring *E. coli* cells (Fig. 11). The purified CalE8 and SgcE8 also exhibit notable differences in coloration and absorption spectra at similar protein concentration. The difference in absorption spectrum, which arises from the PKS-tethered product, suggests that CalE8 and SgcE generate different products in the cytoplasmic environment of *E. coli*. Enzymatic assay further showed that CalE8 produced a linear carbonyl-conjugated polyene (3,5,7,9,11,13-pentadecen-2-one (20)) and three pyrone derivatives 15, 16 and 17 with the assistance of the cognate thioesterase CalE7. (Fig. 10)

Fig. 9 Proposed domain organization of the iterative PKS in enediyne biosynthesis. The structural models of the unknown domains are shown in the frames.
Fig. 10 In vitro products of the iterative PKS.

Comparison of 20 with a synthetic standard unequivocally established that 20 contains trans double bonds. The 15 carbon-containing 20 was proposed to derive from a labile β-keto carboxylic intermediate (18), which was detected recently by Townsend and coworkers using mass spectrometry. Interestingly, unlike most PKSs characterized so far, CalE8 does not require acetyl CoA as starter, indicating that CalE8 can generate acetyl CoA from malonyl CoA via an intrinsic decarboxylative mechanism.

More recently, it was revealed that CalE8 and CalE7 not only produce 20, 16, 17 and 18, but also 21. The yield of 21 was underestimated in the previous study due to the decomposition of 21 in trifluoroacetic acid (TFA). The landmark study of the iterative PKS (LNKS) in lovastatin biosynthesis has revealed that the production of the correct product requires a free-standing ER domain protein (LovC), and that LNKS only produces aberrant products that include three pyrone derivatives in the absence of LovC. The assembly of the correct PKS product by tenellin synthetase also requires a trans-acting ER domain protein. Based on the observation that CalE8 also produced multiple products, which include three pyrone derivatives, Townsend and coworkers proposed that neither 20 nor 21 is the authentic biosynthetic intermediate. This implies that CalE8 may need the assistance of a trans-acting regulatory factor for producing the correct product. The latest enzymatic assays with CalE8, SgcE and the PKS (DynE8) for dynemicin biosynthesis showed that although multiple products are produced at high catalytic rates and low [NADPH], 21 was produced exclusively when the reaction proceeds at a moderate catalytic rate with sufficient NADPH present. The exclusive production of 21 by the PKSs seems to suggest that 21 cannot be entirely excluded as a physiologically relevant product at this moment. Nonetheless, it also cannot be ruled out that 21 is merely an aberrant product and the bona fide product may only be produced in the presence of a regulatory factor.

4.2 Structure and catalytic mechanism of the thioesterase

Polyketide and nonribosomal peptide synthesis generally involves type I and type II thioesterases for the release of final product or removal of aberrant products. Type I thioesterases (TE I) are cis-acting domains fused to the C-terminus of the most downstream module of PKS or nonribosomal peptide synthase (NRPS) for the release and cyclization of the final product. By contrast, type II thioesterases (TE II) are discrete proteins responsible for the trans hydrolytic release of

Fig. 11 Heterologous expression of the iterative PKS SgcE and CalE8 in E. coli.
misloaded substrates or aberrant products.¹⁰³⁻¹⁰⁵ TE II proteins are structurally and evolutionarily related to a family of well-known α/β hydrolases that contain 240–260 residues.¹⁰⁶ A common serine esterase motif GXSXG and another downstream motif GXH are conserved in TE II proteins.¹⁰⁷,¹⁰⁸ The stand-alone CalE 7 and its homolog do not belong to the TE II family because they are neither α/β fold hydrolase nor proteins containing the two conserved motifs for TEII. Instead, CalE 7 and its homolog share moderate sequence homology with a family of hotdog fold proteins characterized by a long central α-helix packed against a five-stranded antiparallel β-sheet. Such hotdog fold thioesterases are mainly known to utilize acyl-CoAs as substrates, with only a few known examples that act on peptidyl carrier protein (PCP) or ACP-tethered acyl substrates,¹⁰⁹⁻¹¹¹ including the EntH protein (or YbdB) involved in the biosynthesis of the nonribosomal peptide-derived enterobactin.¹¹⁰,¹¹²,¹¹³

The crystal structure of CalE 7 determined by Kotaka *et al.* revealed a hotdog fold thioesterases structure with an εγ oligomeric arrangement and a kinked substrate-binding channel located at the subunit interface.⁹⁷ Based on the observation that Arg³⁷ is the only essential residue for catalysis, a catalytic mechanism has been put forward.⁹⁷ The nucleophilic attack of the thioester carbonyl by a water or hydroxide anion is assisted by Arg³⁷, which acts as an oxyanion hole to stabilize the transition state. Determination of the R37K mutant structure confirmed that the mutation does not cause major structural perturbation (Liew *et al.*, unpublished results), further supporting the catalytic role of Arg³⁷. The collapse of the penta-coordinated transition state would break the thioester bond to generate the labile 16-carbon β-keto carboxylic intermediate (18). Townsend and coworkers have shown that the decarboxylation of 18 can occur gradually in solution.⁹⁹ The decarboxylation of the β-keto carboxylic acid 18, which requires a protonation step, may be promoted by the water molecules observed in the shielded substrate-binding channel.⁹⁷ Meanwhile, the β-hydroxyl carboxylic acid 19 is considered to be a relatively stable intermediate that does not undergo spontaneous decarboxylation and dehydration. It is currently not clear whether and how the thioesterase catalyzes the conversion of 19 to 21.

### 4.3 The putative acetylase

One of the gene products encoded by the enediyne warhead gene cassette, as represented by CalU15, shares minimum sequence homology with a family of diiron cluster containing desaturases or acetylenases for fatty acid modification.¹¹⁴⁻¹¹⁷ Desaturases and acetylenases catalyze the oxidation of single bonds to furnish double bonds, or triple bonds in the presence of O₂ and electron donors. Therefore, it was postulated that CalU15 and its homologs may be the enzymes for catalyzing the formation of the acetylenic bonds of the 1,5-diyne-3-ene unit. Preliminary studies showed that the recombinant CalU15 indeed exhibits a signature absorbance around 340 nm for diiron proteins.⁹⁶ However, the identity of the cofactor remains to be fully established and the enzymatic activity of the protein needs to be demonstrated.

---

### 4.4 Two other gene products encoded by the warhead gene cassette

Another conserved gene product encoded by the gene cassette, as represented by CalU14, encodes a ~60 kDa protein with an N-terminal signaling peptide. Secondary structure analysis suggested that the protein is a large β-barrel protein with GAP-FG repeats and an ASPIC/UnbV motif. The ASPIC/UnbV motif can be found in the α-chain of integrin and a dozen proteins from *Rhodopirellula baltica* SH 1. The function of CalU14 and its homologs is unknown, though these secreted proteins may function as carriers or surface-adhesive proteins. The second gene product, as represented by CalT3, is predicted to be a helical membrane protein that contains very short cytoplasmic segments. Although it has been annotated as a transporter,⁷¹ the precise role of CalT3 and its homologs remains to be established.

---

### 4.5 Maturation of the enediyne moiety

The enediyne warheads are decorated with various functional groups during the maturation stage. For example, some of the hydroxyl groups for the attachment of peripheral moieties are presumably installed at this stage. The hydroxylation of the enediyne core is most likely to be catalyzed by the cytochrome P450 and FAD-dependent hydroxylases encoded by the gene clusters. In addition, the highly unusual allylic trisulfide group in calicheamicin-like enediynes acts as the trigger unit during diradical formation (Fig. 3). The enzymes responsible for the installation of the trisulfide unit have not been identified. Isotope labeling has revealed that the sulfurs and the S-methyl group of esperamicin A₁ are likely to derive from L-methionine or L-cysteine.⁶⁷ The calicheamicin gene cluster contains several ORFs encoding proteins that share sequence homology with cysteine lyase, cystathionine γ-lyase and homocysteine S-methyltransferase.⁷¹ These ORFs are absent from the other sequenced enediyne gene clusters. Since calicheamicin is the only trisulfide-containing chromophore among all the enediynes with gene clusters sequenced, these gene products could be involved in the biosynthesis of the trisulfide moiety.

---

### 5 Apoprotein of 9-membered enediynes

All known 10-membered enediyne natural products were isolated as free-standing chromophores, whereas the 9-membered enediyne chromophores were isolated in association with a protective apoprotein with the exception of N1999A2. The requirement for the apoprotein is due to the intrinsic reactivity of the 9-membered enediyne core. For instance, while neocarzinostatin is extremely unstable in aqueous solution in the absence of the apoprotein, sequestration by the apoprotein greatly enhances the stability of the labile chromophore.¹¹⁸,¹¹⁹ Interestingly, the maduropeptin apoprotein (MdpA) does not share similarity with the apoproteins of neocarzinostatin, C-1027 and kedarcidin. In fact, the 133 amino acid long MdpA represents a new protein class that does not share significant sequence homology with any protein deposited in the NCBI database.⁷⁵,¹²⁰

The structures of neocarzinostatin and the aromatized C-1027 in complex with their apoproteins (NcsA and CagA) have been determined.¹¹⁹,¹²¹⁻¹²⁶ Overall, the apoproteins share an immunoglobulin-like fold that consists of a seven-stranded antiparallel
In the aftermath of the sequencing and annotation of the enediyne gene clusters, validation of the biosynthetic pathways has uncovered a plethora of novel enzymes and chemical transformations in the biosynthesis of the aromatic and sugar moieties. These pathways and enzymatic transformations will be summarized in Sections 6 and 7. Highlighted in the two sections are also the successful stories of exploiting the biosynthetic pathways for producing C-1027 and calicheamicin γ₁ variants.

### 6.1 C-1027

C-1027 was isolated from *Streptomyces globisporus* as an apoprotein-chromophore complex.¹³¹⁻¹³³ In addition to the 9-membered enediyne core, C-1027 contains three distinct structural units that include a deoxy aminosugar, a β-amino acid, and a benzoxazoline group (Fig. 14). Initial annotation of the gene products of the C-1027 gene cluster suggested that the deoxy amino sugar and the β-amino acids are derived from glucose-1-phosphate and L-tyrosine respectively,⁷⁰ which were later confirmed by biochemical studies. The biosynthesis of the benzoxazoline moiety was originally proposed to start from anthranilate, a common intermediate from the shikimate pathway.⁷⁰ Subsequent characterization of the enzymes from the putative pathway revealed an unexpected pathway with chorismic acid as the starting material.¹³⁴

#### 6.1.1 β-Tyrosine moiety

C-1027 contains a (S)-3-chloro-5-hydroxy-β-tyrosine moiety that derives from L-α-tyrosine. Characterization of the enzymes in the seemingly simple β-amino acid biosynthetic pathway has nonetheless revealed some novel chemistry and enzymology. A biosynthetic mechanism that involves six proteins (SgcC-C6) has been proposed by Shen and coworkers, with the functions of all the participating proteins confirmed by enzymatic assays (Fig. 15).

The first step of the pathway is catalyzed by the aminomutase SgcC4 (also SgcTAM) that converts L-tyrosine to (S)-β-tyrosine. SgcC4 belongs to a small family of enzymes that use the prosthetic group 4-methylideneimidazole-5-one (MIO) for substrate activation. Although closely related to ammonia lyases, SgcC4 catalyzes a chemically challenging and stereospecific 1,2-amino shift to generate (S)-β-tyrosine.¹³⁵⁻¹³⁸ Comparison of the crystal structure of SgcC4 determined by Bruner and coworkers with the L-tyrosine ammonia lyase revealed that SgcC4 has a closed active site that favors ammonia sequestration and disfavors the formation of lyase elimination products.¹³⁹ Mechanistic studies further support a catalytic mechanism that involves the formation of a covalent enzyme-substrate adduct and amine addition to the cinnamate intermediate via 1,4-Michael addition (Fig. 16).¹³⁷ The use of epoxide analogs as mechanism-based inhibitors also provided support for the formation of an amine-MIO covalent adduct during catalysis.¹⁴⁰

Another distinctive feature of the pathway is the involvement of the peptidyl carrier protein (PCP) SgcC2. Shen and coworkers established that the β-tyrosine moiety undergoes modification with the aromatic moiety anchored to SgcC2, which to some extent resembles the modification of the PCP domain-tethered intermediate in NRPSs.¹⁴¹ *In vitro* enzymatic assay also established that SgcCl, a free-standing NRPS adenylation domain, is responsible for the activation and loading of the β-tyrosine onto

---

**This journal is © The Royal Society of Chemistry 2010**

**Nat. Prod. Rep., 2010, 27, 499–528 | 507**

Fig. 13 Building blocks of enediyne chromophores. The sites on the enediyne cores for peripheral moiety attachment are indicated by the dots.

SgcC2.¹⁴¹ Anchoring of the substrate to the carrier protein provides a means for controlling substrate specificity through protein–protein interactions. A similar strategy is apparently employed in the synthesis of other secondary metabolites such as andrimid.¹⁴²

Once the β-tyrosine is anchored to Sgc2, the aromatic ring is chlorinated and hydroxylated by two FAD-dependent enzymes with strict regioselectivity. Several structurally and mechanistically distinctive families of halogenases have been discovered in recent years.¹⁴³,¹⁴⁴ SgcC3 shares sequence homology with a family of O₂- and FAD-dependent halogenases that act on phenolic or indole group-containing substrates.¹⁴⁴ Biochemical characterization of SgcC3 revealed that the enzyme catalyzes both chlorination and bromination of the SgcC2-anchored substrate in the presence of the FAD reductase SgcE6.¹⁴⁵ Meanwhile, SgcC is an FAD-dependent monooxygenase that installs a hydroxyl group at the 5-position of the aromatic ring.¹⁴⁶ Similar to SgcC3, SgcC functions in conjunction with the FAD reductase SgcE6 as a two-component enzyme system. Similar two-component systems that employ a FAD reductase for electron shuttling have been identified in other biosynthetic pathways.¹⁴⁷⁻¹⁵¹ The SgcC3/SgcE6 system, nonetheless, represents the first such system to act on a carrier-protein-anchored substrate.

The substrate specificity of some of the enzymes in the β-tyrosine biosynthetic pathway was examined. It was shown that the halogenase SgcC3 catalyzes not only the chlorination,

in vivo from the SgcC3-catalyzed bromination. The relaxed substrate specificity of SgcC and SgcC3 provides a window of opportunity for generating C-1027 analogs, as discussed in Section 6.1.4.

In the final stages of C-1027 biosynthesis, the (S)-3-chloro-5-hydroxy-β-tyrosine is tethered to the enediyne core through ester and ether linkages. Shen and coworkers discovered that the formation of the ester bond is catalyzed by a novel free-standing NRPS condensation enzyme (SgcC5). By using 1-phenyl-1,2-ethanediol (22) or 2-amino-1-phenyl-1-ethanol as mimics of the aromatized enediyne core, the authors demonstrated that SgcC5 catalyzes the formation of the ester bond between the substrate mimic and SgcC2-tethered (S)-3-chloro-5-hydroxy-β-tyrosine (Fig. 15). The finding that SgcC5 can also catalyze the formation of amide bond, albeit with low efficiency, indicated that SgcC5 is evolutionarily and functionally related to the canonical NRPS condensation domains that catalyze amide bond formation.

### 6.1.2 Benzoxazoline moiety. The benzoxazoline moiety of C-1027 not only confers the chromophore its DNA binding specificity but also provides stability to the enediyne core by

Fig. 14 Building blocks of C-1027 (3).

but the bromination of the substrate with a slightly lower efficiency. Exploration of the substrate specificity of SgcC also revealed that the monooxygenase was able to hydroxylate several 3-substituted β-tyrosyl-S-SgcC2. Notably, SgcC can catalyze the hydroxylation of the brominated substrate that can be produced

Fig. 15 Proposed biosynthetic pathway for the (S)-3-chloro-5-hydroxy-β-tyrosine moiety of C-1027.

Fig. 16 Proposed catalytic mechanism for the aminomutase SgcC4.

interacting with the apoprotein.²¹ It was initially suggested that the benzoxazoline moiety derives from anthranilate, a common intermediate from the shikimic acid pathway.⁷⁰ However, biochemical characterization of the enzymes SgcD and SgcG revealed a novel biosynthetic pathway that starts with chorismate.¹³⁴ Chorismate is known to be an important branch-point metabolite in the biosynthesis of aromatic amino acids (Trp, Tyr and Phe), menaquinone, folic acid and other metabolites.¹⁵³,¹⁵⁴ The discovery of the conversion of chorismate to 3-enolpyruvylanthranilate (OPA, 24) in C-1027 biosynthesis established a new branching point in chorismate metabolism (Fig. 17).

The biosynthetic pathway proposed by Shen and coworkers starts with the conversion of chorismate to 2-amino-2-deoxy-isochorismate (ADIC, 23). Although the first enzyme SgcD shares significant sequence homology with the anthranilate synthase subunit I (ASI) in tryptophan biosynthesis,¹⁵⁵,¹⁵⁶ it exhibited unexpected catalytic activity by acting on chorismate and converting it to ADIC with glutamate as the amine donor. Two related ADIC synthases in the biosynthesis of *p*-amino-benzoic acid and phenazine have been characterized previously.¹⁵⁴,¹⁵⁷ A likely catalytic mechanism involves the nucleophilic addition of the ammonia at the C2 position and

the substitution of the hydroxyl group at C4 position (Fig. 17, inset box).

The finding of chorismate as starting material also led to authors to identify the enzyme SgcG as the oxidoreductase for catalyzing the formation of the aromatic moiety in 3-enolpyr-uvoylanthranilate (OPA, 24). SgcG is a novel enzyme that employs FMN and an iron–sulfur cluster to catalyze the oxidative aromatization. The detailed catalytic mechanism for this step remains to be delineated. The following steps have yet to be experimentally validated, although the benzoxazoline moiety is likely to be formed after the ring closure, hydroxylation and methylation steps. The benzoxazoline moiety is presumably converted to its CoA derivative for the final assembly of C-1027 by the acyltransferase SgcD6.

### 6.1.3 Deoxy aminosugar

Many enediyne chromophores are decorated with highly modified mono-, di- or trisaccharides. Validation of the proposed biosynthetic pathways has shed light on the formation of these unusual sugars and the glycosylation reactions. The biosynthesis of the sugar moieties in neo-carzinostatin and calicheamicin has been examined in great detail, as will be discussed in sections 6.2.2 and 7.1.2.2. Seven gene products (SgcA to SgcA6) were proposed by Shen and

![Figure 17](image.png)
*Fig. 17 Proposed biosynthetic mechanism for the benzoxazoline moiety.*

![Figure 18](image.png)
*Fig. 18 Proposed biosynthetic pathway for the deoxyamino sugar moiety of C-1027.*
coworkers for building the deoxy aminosugar of C-1027 based on sequence homology and the established biosynthetic routes for deoxy aminosugars (Fig. 18).^{70,158} As a common theme in sugar biosynthesis, the biosynthetic pathway is very likely to rely on the activation of the monosaccharide as its nucleoside diphosphate (NDP) derivative by the nucleotidyltransferase (SgcA1). The TDP-tethered sugar is further modified by processing enzymes SgcA, A2, A3, A4 and A5 to yield the deoxy-amino sugar, which is finally transferred onto the enediyne core by the putative glycosyltransferase SgcA6.

### 6.1.4 Production of C-1027 analogs.
The biological activity of C-1027 or any other enediyne is highly dependent on the peripheral moieties. Given the cytotoxicity and other undesired pharmacological properties of the naturally occurring enediynes, the idea of generating structural analogs by modifying the peripheral moieties through metabolic engineering is appealing. Generating analogs at large scale by chemical synthesis is impractical given the complex structure of the chromophore. Elucidation of the pathways in C-127 biosynthesis has opened the door for incorporating modified building blocks through metabolic engineering.

Knowing the relaxed substrate specificity of SgcC3 and SgcC, Shen and coworkers demonstrated that the disruption of the halogenase *sgcC3* and monooxygenase *sgcC* gene in *S. globisporus* led to the production of deschloro-C-1027 (27) and deshydroxyl-C-1027 (28) respectively. In addition, disruption of the methyltransferase *sgcD4* gene in the benzoxazoline pathway resulted in the production of desmethyl-C-1027 (26).^{54} C-1027 (3) is known as a cytotoxin that causes both double-stranded breaks (DSB) and interstrand cross-links (ICLs) in DNA. The two types of DNA damage trigger different cellular responses for damage control. Remarkably, the minor modification of C-1027 seems be able to alter the mechanism of DNA damage and elicit a different cellular response.^{159} In contrast to 3, 27 and 28, 26 seemed to cause ICL, but not DSB damage, to chromosomal DNA. The dominance of the oxygen-independent ICL mechanism for 26 could be exploited for cancer treatment in view of the fact that the some tumor cells exhibit a high level of hypoxia.

The resulted alteration in biological activity by the minor modification and the manipulation of metabolic pathway by rational design are equally remarkable. This serves as an encouraging example that accentuates the viability of the metabolic engineering approach. All three pathways for the biosynthesis of the peripheral moieties seem to contain enzymes with relaxed substrate specificity, and thus may present more opportunities for the manipulation of the metabolic pathways. In particular, recent studies have shown that some glycosyltransferases are promiscuous with respect to their sugar nucleotide donors.^{158,160–163} Hence, it may also be possible to exploit the glycosyltransferase (SgcA6) for incorporating sugar variants into C-1027.

### 6.2 Neocarzinostatin

Neocarzinostatin (1) was isolated from the culture filtrate of *Streptomyces carzinostaticus* var. E-41 as a 1:1 complex with an 11 kDa apoprotein.^{1,2,164} Although 1 is the first characterized enediyne chromophore, the free-standing chromophore is highly unstable upon exposure to heat, high pH or UV-light irradiation. In comparison to C-1027, neocarzinostatin consists of a distinct

![Chemical Structures](https://i.imgur.com/1234567.png)

This journal is © The Royal Society of Chemistry 2010  
Nat. Prod. Rep., 2010, 27, 499–528 | 511
deoxy aminosugar and a naphthoic acid moiety in addition to the epoxide-derivatized 9-membered enediyne. The peripheral moieties of C-1027 and neocarzinostatin are also anchored to the enediyne core at different positions, indicating that the two enediyne cores are hydroxylated at different sites during the maturation of the enediyne core.

### 6.2.1 Naphthoic acid.
Isotopic labeling experiments have established that the naphthoic acid moiety originates from a single polyketide chain of six head-to-tail acetate units. Based on the neocarzinostatin gene cluster, a biosynthetic pathway for the naphthoic acid moiety that involves an iterative PKS (NcsB), a CoA ligase and several ancillary enzymes was proposed. Furthermore, the 5.45 kb NcsB was cloned and expressed in *S. lividans* TK24 and *S. coelicolor* YU105 by Sohng and coworkers. Observation of the major product 2-hydroxy-5-methyl naphthoic acid in the fermentation culture confirmed the polyketide origin of the naphthoic acid.

The biosynthesis of the naphthoic acid moiety starts with NcsB, an iterative PKS consisting of KS, AT, DH, KR, ACP and a core domain with unknown function. Similar PKSs with identical domain organization were also found in the biosynthesis of 6-methylsalicylic acid (6-MSA) and the 5-methyl naphthoate moiety of azinomycin B. NcsB is likely to use acetyl-CoA as starter and malonyl-CoA as extender to assemble a nascent hexaketide with the selective reduction and dehydration of the keto groups at C5 and C9 (Fig. 19). The hexaketide intermediate presumably undergoes aromatization to furnish the naphthoic acid via intramolecular aldol condensation. Intriguingly, the homologous PKS AziB, which shares the same domain organization with NcsB, produces 5-methyl naphthoic acid (30) instead of 2-hydroxy-5-methyl naphthoic acid (29).

The formation of the different products is due to the reduction and dehydration at C3 for AziB but not NcsB. The structural determinants and molecular mechanism of the programmed reduction remain unknown.

From a broad perspective, NcsB, AziB and the enediyne PKSs represent a distinct class of iterative type I PKSs from the well-known modular PKSs and type II aromatic PKSs. Many type I iterative PKSs can be found in fungi, including the renowned lovastatin nonaketide synthase (LNKS) for the biosynthesis of the cholesterol-lowering drug lovastatin. Type I iterative PKSs bear some resemblance to the modular PKSs since they share similar domain architecture and are able to generate chemically diverse products by selectively reducing and dehydrating the keto groups during chain elongation. However, the iterative PKSs differ from the modular PKSs in that they use only one single set of domains repetitively for each round of chain extension. The detailed molecular mechanisms by which the type I iterative PKSs carry out the chain elongation, termination and selective reduction largely remain to be elucidated. Though there has been some recent progress for the nonreducing iterative PKS systems, principally by Townsend and co-workers.

The post-PKS modification of the naphthoic acid moiety starts with the installation of a hydroxyl group at C8 catalyzed by the cytochrome P450 hydroxylase NcsB3. (Fig. 20) An S-adenosylmethionine (SAM)-dependent O-methyltransferase (NcsB1) catalyzes the methylation of the hydroxyl group, in full agreement with the observation that the methyl group originates from L-methionine. Furthermore, the substrate tolerance of NcsB1 was examined by testing structural analogs of naphthoic acid. Apart from the cognate substrate, NcsB1 was able to turnover three dihydroxynaphthoic acids to their methylated products

---

**Formation of 5-methyl naphthoic acid (29) in neocarzinostatin biosynthesis**

![Diagram](https://i.imgur.com/1234567.png)

**Formation of 2-hydroxy 5-methyl naphthoic acid (30) in azinomycin B biosynthesis**

![Diagram](https://i.imgur.com/7654321.png)

**Fig. 19** Possible biosynthesis mechanism for the naphthoic acid moiety of neocarzinostatin and azinomycin B. The incorporated acetate units are represented by the thick bonds.

Fig. 20 Modification of the PKS product in the biosynthesis of the naphthoic acid moiety. The substrate analogs used for exploring the substrate specificity of NcsB1 and NcsB2 are shown in the boxes with the methylation sites shaded.

(Fig. 20). The crystal structures of NcsB1 determined by Cooke *et al.* not only revealed important residues for substrate binding, but also significant conformational changes during the course of catalysis.¹⁷⁰

The Shen group further confirmed that NcsB2 is a ligase that catalyzes the adenylolation of 2-hydroxy-7-methoxy-5-methyl-1-naphthoic acid (31) to form its CoA derivative 32.¹⁷¹ The discovery of NcsB2 as a promiscuous naphthoic acid/CoA ligase with relaxed substrate specificity provided a great opportunity for producing structural analogs of neocarzinostatin by metabolic engineering (Fig. 20). As the last step, the putative acyl transferase (NcsB4) is likely to be responsible for the transfer of the naphthoic group onto the enediyne scaffold.

### 6.2.2 Deoxy aminosugar. The deoxyamino sugar moieties of neocarzinostatin and C-1027 differ in the positions and numbers of the amino, hydroxyl, and methyl groups. The biosynthetic pathway is likely to start with the activation of the monosaccharide as its NDP derivative by the nucleotidyltransferase NcsC (Fig. 21). Based on sequence homology and established biosynthetic routes,⁷³,¹⁵⁸ several gene products have been proposed for the activation of the sugar ring via the formation of a 4-keto intermediate, the deoxygenation at C6, and the installation of an amino group at C2. Previous labeling experiments with [methyl-³H] methionine have suggested that the *N*-methyl of the deoxyamino sugar originates from the methionine of *S*-adenosylmethionine (SAM).⁶⁴ The methylation is presumably catalyzed by the methyltransferase NcsC5, whereas the predicted glycosyltransferase NcsC6 may transfer the sugar moiety onto the enediyne core.

Together, the characterization of the enzymes in the naphthoic acid and deoxyamino sugar pathways by Shen and coworkers provides an entry point for the exploration of the possibility of producing neocarzinostatin analogs by metabolic engineering and fermentation. The relaxed substrate specificity of NcsB1 and NcsB2 can be potentially exploited. Likewise, the final step catalyzed by the glycosyltransferases NcsC6 could represent another opportunity for generating analogs by glycodiversification.¹⁶³

### 6.3 Maduropeptin

Maduropeptin (4) was first isolated as a 1:1 apoprotein-chromophore complex from the *Actinomadura madurae* strain H710-49 in 1991 by a Japanese team.¹²⁰ The maduropeptin chromophore is tightly bound by a new class of apoprotein that

Fig. 21 Proposed biosynthetic pathway for the deoxyamino sugar moiety of neocarzinostatin.⁷³

is distinct from the apoproteins of neocarzinostatin, C-1027 and kedarcidin. The first determined structures of maduropeptin are in fact the solvent adducts of the labile maduropeptin chromophore (Fig. 22).<sup>172</sup> The structure of 4 was therefore deduced based on the structure of the adducts. The adduct formation is a reversible process, which explains why the adducts still exhibit antitumor and antibiotic activity, though the potency is ~100-fold lower than 4.<sup>120</sup> The recent synthesis of the methanol adduct by Komano *et al.* has led to the proposal that 4 possesses

a slightly different conformation from the originally proposed structure with an antipodal madurosamine moiety.<sup>173</sup>

Maduropeptin contains a 3,6-dimethylsalicyclic acid, a deoxy aminosugar and an aryl hydroxypropionic acid moiety.<sup>172</sup> The recent sequencing and annotation of the maduropeptin gene clusters (which contain 42 ORFs) by Shen and coworkers has suggested that the biosynthetic pathways for the three peripheral moieties are likely to be evolutionarily related to those of C-1027 (3) and neocarzinostatin (1).<sup>75</sup>

### 6.3.1 Dimethylsalicyclic acid.

It has been known that 6-methylsalicylic acid (6-MSA) produced by some bacteria and fungi has a polyketide origin.<sup>81,82,94,166,174–178</sup> The presence of a second PKS gene (*mdpB*) in the maduropeptin gene cluster hinted that the 3,6-dimethylsalicyclic acid moiety may also originate from a polyketide precursor. In addition, MdpB is highly homologous to a group of iterative fungal PKSs that include the 6-methylsalicylic acid synthases (MSAS) from *Penicillium patulum*, *Aspergillus terreus* and *Glarea lozoyensis*.<sup>81,94,177,178</sup> MdpB also shares the same domain composition with NcsB, the PKS for the biosynthesis of the naphthoic acid moiety in neocarzinostatin (see Section 6.2.1). The catalytic mechanism probably involves a partially reduced polyketide intermediate before aromatization (Fig. 23). As discussed in Section 6.2.1, the molecular basis underlying the selective reduction of the keto group remains unexplained for these iterative PKSs.

The PKS product may undergo methylation and adenylation after it is off-loaded from the PKS (pathway a, Fig. 23), though it is unclear how the PKS product is off-loaded. Alternatively, the PKS product may be directly transferred to the sugar moiety with the assistance of a transferase without off-loading (pathway b), as exemplified by the incorporation of the substituted 6-MSA in chlorothricin.<sup>179</sup> Interestingly, the neighboring gene *mdpB3* encodes a hydrolase that may actually act as a transferase, resembling the role of ChlB6 in the chlorothricin pathway.<sup>179</sup>

### 6.3.2 Aryl hydroxypropionic acid.

Annotation of the maduropeptin gene cluster indicated that the pathway for the (*S*)-3-(2-chloro-3-hydroxy-4-methoxyphenyl)-3-hydroxypropionic acid (or

**Fig. 22** Proposed mechanism for the conversion between maduropeptin and its adducts.

**Fig. 23** Possible biosynthetic mechanisms for the 3,6-dimethylsalicyclic acid of maduropeptin.
aryl hydroxypropionic acid moiety) is closely related to the biosynthetic pathway for the β-tyrosine of C-1027.⁷⁰,⁷⁵,¹³⁹,¹⁴⁵,¹⁴⁶ The functions of the enzymes were assigned with reasonably high confidence given the high sequence homology shared by the enzymes from the two pathways. In the proposed pathway, the first two enzymes MdpC4 and MdpCl are highly homologous to the aminomutase SgcC4 and NRPS adenylation domain protein SgcCl, whereas MdpC2 is predicted to be a SgcC2-like PCP to which the substrate is anchored (Fig. 24). MdpC7, a predicted pyridoxal phosphate (PLP)-dependent transaminase that is absent from the C-1027 pathway, converts the β-amino into a keto group. Further modification of the aromatic moiety is likely to be carried out by the putative hydrolase MdpC, methyl-transferase MdpC6 and halogenase MdpC3, although the precise timing of these steps remains to be determined. The last enzyme (MdpC8), a predicted alcohol dehydrogenase, furnishes the β-hydroxyl group. In contrast to the ester linkage in C-1027 between the enediyne core and the β-tyrosine moiety,¹⁵² the aryl hydroxypropionic acid moiety is connected to the enediyne core through an amide linkage in 4. It will be of great interest to identify the enzyme that catalyzes the formation of the amide bond in the future.

### 6.3.3 Aminosugar. Maduropeptin has a different aminosugar compared to C-1027 and neocarzinostatin, with the major

difference being the absence of two methyl groups on C5. Once again, the synthesis of the aminosugar was proposed to begin with D-glucose-1-phosphate.⁷⁵ The gene product (MdpA1) that is highly homologous to the nucleotidyltransferase (SgcA1) is likely to install the NDP group on the D-glucose-1-phosphate. (Fig. 25) The putative dehydrogenase (MdpA2) and decarboxylase (MdpA3) were proposed for the removal of the CH₂OH group at the C5 position. The sugar nucleotide moiety is further modified by four more putative enzymes including a dehydrogenase, a decarboxylase, a SAM-dependent methyltransferase and a transaminase to furnish the final aminosugar. The predicted glycotransferases (SgcA6) is likely to be responsible for the final transfer of the sugar moiety onto the enediyne core. The assignment of the protein function is based on sequence homology with characterized enzymes and has not been validated experimentally.

#### 6.4 Kedarcidin

The highly toxic and unstable kedarcidin chromophore (2) was isolated from an actinomycete strain in 1991.¹⁸⁰–¹⁸⁴ The structure of kedarcidin has been further revised since the report of the first structure in 1992.⁴⁴,⁴⁵,¹⁸⁴ The current structure and stereochemistry are based on the total synthesis by Myers and coworkers in 2007.⁴⁵ It is of interest to note that the binding of DNA by

![Fig. 24 Proposed biosynthetic pathway for the aryl hydroxypropionic acid moiety of maduropeptin.](attachment://image1.png)

![Fig. 25 Proposed biosynthetic pathway for the aminosugar of maduropeptin.](attachment://image2.png)

This journal is © The Royal Society of Chemistry 2010  
Nat. Prod. Rep., 2010, 27, 499–528 | 515kedarcidin (and maduropeptin) is subject to $\mathrm{Ca}^{2+}$ and $\mathrm{Mg}^{2+}$ inhibition. ${ }^{18,19}$ This is because the divalent ions can coordinate the 2-hydroxynaphthoyl moiety of the kedarcidin chromophore to prevent DNA binding. Although the gene cluster has not been sequenced, the enediyne core, naphthoic acid and sugar moieties of kedarcidin are likely to share similar biosynthetic pathways with those of C-127 and neocarzinostatin. The biosynthetic origin of the highly unusual $\beta$-2-chloroazatyrosine moiety, however, remains to be revealed.

### 6.5 N1999A2

N1999A2 (5) was isolated from the broth filtrate of *Streptomyces* sp. AJ9493 by Sugiura and coworkers. ${ }^{185}$ In contrast to other 9-membered enediynes, the N1999A2 chromophore was not associated with an apoprotein and seems to be stable during isolation. N1999A2 consists of a 9-membered enediyne core and substituted naphthoic acid moiety. ${ }^{17,32,185}$ Given the structural similarity between neocarzinostatin and N1999A2, the naphthoic acid moiety is likely to share the same biosynthetic pathway with that of neocarzinostatin as discussed in Section 6.2.1. Apart from the anticipated iterative PKS, the accessory halogenase, hydroxylase and methyltransferase are expected to be involved in post-PKS modifications.

### 6.6 Sporolides

Sporolide A (33) and B (34) are polycyclic macrolides isolated from the marine bacterium *Salinispora tropica* CNB-440 by Fenical and coworkers in 2005. ${ }^{11}$ The chromophores are comprised of a chlorinated cyclopenta[$a$]indene unit and a cyclohexenone epoxide moiety. ${ }^{11,186}$ The chlorinated cyclopenta[$a$]indene unit bears striking similarity to the cycloaromatized products of 9-membered enediynes, which led the authors to postulate that the sporolides may be derived from the enediyne precursor 6 (Fig. 26). Sequencing of the *Salinispora tropica* genome by Moore and coworkers indeed revealed two enediyne gene clusters among 17 gene loci for secondary metabolite biosynthesis. ${ }^{76,187}$ One of the enediyne gene clusters was predicted to be involved in the biosynthesis of a 9-membered enediyne based on phylogenetic analysis, whereas the other for the synthesis of an unknown 10-membered enediyne. The validation of the cyclohexenone pathway described below confirms that the former gene cluster indeed encodes proteins for synthesizing the sporolides.

Several biosynthetic pathways with different starting materials could be envisioned for the cyclohexenone unit. ${ }^{188}$ Annotation of the *S.* tropica genomes by Moore and coworkers led the authors to postulate a pathway with L-tyrosine as the starting material. ${ }^{76}$ Such a hypothesis is in consistent with the observed incorporation of $^{13} \mathrm{C}$ from labeled tyrosine and L-methionine. Tyrosine is also a limiting nutrient in sporolide biosynthesis since the supplementation of tyrosine led to a ~10-fold increase in metabolite production. ${ }^{189}$ To validate the biosynthetic pathway of the cyclohexenone unit, Moore and coworkers examined the enzymatic activity of SpoT1, a putative mononuclear iron dioxygenase encoded by the 9-membered enediyne gene cluster. The authors confirmed that SpoT1 functions as a $p$-hydroxy-mandelate ($p$-HMA) oxygenase (HMAS) that converts $p$-HPPA

---

Fig. 26 Generation of sporolides from the enediyne precursor by cycloaromatization.

to $(S)$-$p$-HMA. ${ }^{189}$ (Fig. 27) Confirmation of this key step led the authors to propose a biosynthetic pathway for the cyclohexenone moiety. Among the ten enzymes in the pathway (Fig. 27), SpoT2 is a didomain protein composed of a NRPS adenylation domain and a PCP domain. The cyclohexenone moiety is assembled with the intermediates anchored to SpoT2. The advantage of using a carrier protein has been discussed in Sections 6.1.1 and 6.3.2 for the synthesis of the $\beta$-tyrosine moiety of C-1027 and maduropetin. The cyclohexenone pathway differs slightly with SpoT2 composed of an adenylation and a CP domain, while the other two pathways use discrete adenylation and PCP proteins. After the substrate is tethered to SpoT2, four oxidation and two methylation reactions follow to furnish the cyclohexenone structure. An additional putative epimerase (SpoT8) is likely to invert the stereochemistry at the benzyllic position.

The cyclohexenone unit and the cyclopenta[$a$]indene moiety are joined through an ester and an ether linkage. For the final assembly of 6, Moore and coworkers suggested that a type II NRPS condensation enzyme (SpoT10) catalyzes the ester bond formation; ${ }^{189}$ the enzymes for the formation of the two ether bonds remain to be identified.

### 6.7 Cyanosporasides

Fenical and coworkers reported the isolation of two novel cyclopet[$a$]indene glycosides from the marine bacterium *Salinispora pacifica*. ${ }^{12}$ The compounds, named cyanosporasides A (35) and B (36), contain a 3,3a-dihydrocyclopet[$a$]indene aglycone and a rare 3′-oxo-4′-methyl-$\beta$-fucopyranose moiety. The resemblance of the 3,3a-dihydrocyclopet[$a$]indene moiety to the cycloaromatized product of 9-membered enediynes led the authors to suggest that 35 and 36 are derived from the enediyne precursor 7 (Fig. 28). Such postulation is supported by the
Fig. 27 Proposed biosynthetic pathway for the cyclohexenone moiety of sporolides (A: adenylation domain).

observed ratio of cyanosporaside A to cyanosporaside B of 1 : 1 in the crude fermentation broth. In addition, the chlorination pattern of the aromatic substitution in 35 and 36 resembles that of the enediyne-originated sporolides 33 and 34, further supporting the enediyne origin of the metabolites. The sugar moiety is presumably derived from the oxidation and methylation of 6′-β-deoxygalactose,¹² whereas the biosynthetic origin of the nitrile moiety remains unknown.

## Peripheral moieties of the 10-membered enediynes

The 10-membered enediyne chromophores can be further divided into the calicheamicin-like and dynemicin-like enediynes based on the structure and folding pattern of the enediyne core. Calicheamicin-like enediynes, which also include esperamycins, namenamicin and shishijimicins, are decorated with a variety of sugar and aromatic peripheral moieties (Fig. 2). In comparison, the dynemicin-like chromophores (dynemicin and uncialamycin) are characterized by an anthraquinone peripheral moiety. The calicheamicin and dynemicin gene clusters, along with an unknown 10-membered enediyne chromophore, have been sequenced and partially annotated.⁷¹,⁷⁴,⁷⁶ Some of the enzymes involved in the biosynthesis of the peripheral moieties of calicheamicin γ₁¹, such as the sugar-modifying enzymes and glycotransferases, have been characterized by Thorson and coworkers. The studies have uncovered some novel enzymes for sugar modification and glycosyltransferases with previously unknown properties.

7.1 Calicheamicin $\gamma_{1}^{I}$

Calicheamicins were isolated from *Micromonospora echinospora* spp. *calichensis* from caliche soils in Texas. ${ }^{190}$ Calicheamicins represent a subfamily of enediyne chromophores, and their structure, mode of action and pharmacological properties have been examined in great detail. ${ }^{3-5,57}$ Calicheamicin $\gamma_{1}^{I}$, the best characterized member of the calicheamicins, contains a complex aryltetrasaccharide moiety that confers the chromophore its DNA-binding specificity. The aryltetrasaccharide is composed of an iodized orsellenic acid moiety and four monosaccharides, and characterized by an unusual thiosugar and a hydroxylamino glycosidic linkage (Fig. 29).

### 7.1.1 Orsellenic acid. In addition to $calE8$, a second iterative PKS ($calO5$) gene is present in the calicheamicin $\gamma_{1}^{I}$ gene cluster. ${ }^{71}$ The predicted gene product (CalO5) shares significant sequence homology with AviM, the iterative PKS for the biosynthesis of the oligosaccharide antibiotics avilamycins (37) produced by *Streptomyces viridochromogenes*. ${ }^{191}$ Notably, AviM is also the first type I polyketide synthase isolated from bacteria to produce an aromatic compound. The fact that avilamycins and the related evernimicin (38) also contain a chlorinated orsellenic acid group hints that CalO5 could be involved in the synthesis of the iodized orsellenic moiety in calicheamicin.

Sequence analysis suggested that CalO5 contains at least four domains (KS, AT, DH and ACP) with a 'core' domain apparently located between the DH and ACP domains. The lack of KR domain suggests that the keto groups remain unreduced in the nascent polyketide chain. A possible mechanism for the formation of the aromatic ring has been suggested (Fig. 30). ${ }^{82,166}$ Four predicted enzymes that include a hydroxylase, a halogenase and two methyltransferases putatively modify the aromatic ring to furnish the final product. The predicted cytochrome P450 (CalO2) and FADH $_{2}$-dependent halogenase (CalO3) are likely to catalyze the hydroxylation and iodination respectively, whereas the $O$-methyltransferases CalO1 and CalO6 install the two methyl groups. Although the precise timing of the post-PKS steps and the stringency of the substrate specificity of the enzymes remain to be fully established, biochemical and structural characterization of CalO2 indicated that the hydroxylation may occur after the halogenations. ${ }^{192}$ Based on the crystal structure of CalO2, ${ }^{192}$ MoCoy and coworkers also suggested that CalO2 may act on a CoA-derivatized or ACP domain-tethered substrate. Thorson and coworkers have also confirmed that sugar C is installed on the orsellenic group by glycosyltransferase CalG1 (Section 7.1.2.2). ${ }^{193}$ Meanwhile, the FabH/KS domain-like CalO4 is likely to catalyze the formation of the thioester linkage between the orsellenic acid and sugar B.

![Fig. 29](https://via.placeholder.com/600x400.png?text=Building+blocks+of+calicheamicin+%CE%B3%5E1+(8).)

![Fig. 30](https://via.placeholder.com/600x400.png?text=Putative+biosynthetic+mechanism+for+the+orsellenic+acid+moiety+of+calicheamicins.)

518 | Nat. Prod. Rep., 2010, 27, 499–528
This journal is © The Royal Society of Chemistry 2010

7.1.2 The sugar moieties.

7.1.2.1 Biosynthesis of monosaccharides. The aryltetrasaccharide of calicheamicin γ<sup>1</sup><sup>I</sup> contains four highly modified monosaccharides with an unusual hydroxylamino glycosidic linkage between two of them. All four sugars are deoxysugars that exhibit greater hydrophobicity compared to common sugars. The deoxysugars, along with the orsellinic acid group, define the DNA binding specificity of calicheamicin γ<sup>1</sup><sup>I</sup>.<sup>194,195</sup>

Similar to the sugar pathways for C-1027, neocarzinostatin, maduropeptin and other glycosides, the biosynthesis of the four monosaccharides begins with the conversion of glucose-1-phosphate to NDP-glucose.<sup>196</sup> Sohng and coworkers confirmed that

the sole nucleotidytransferase protein (CalS7) encoded by the gene cluster possesses both thymidylyltransferase and uridylyl-transferase activity, suggesting that CalS7 could catalyze the formation of NDP-glucose in the early stage of sugars A–D synthesis.<sup>197</sup> As discussed below, the pathways for the four sugars would diverge after the formation of NDP-glucose.

The 4-hydroxyamino-6-deoxy-α-D-glucose (sugar A) is directly tethered to the enediyne core. The deoxygenation at C6 position and installation of the amino group are likely to follow similar routes that involve the NDP-4-keto-6-deoxy-α-glucose intermediate, as discussed in Sections 6.1.3 and 6.2.2 (Fig. 31).<sup>158</sup> The hydroxylamino glycosidic bond between sugar

![Figure 31](attachment:Figure_31.png)

Fig. 31 Possible biosynthetic pathways for the monosaccharides of calicheamicin γ<sup>1</sup><sup>I</sup>.

This journal is © The Royal Society of Chemistry 2010
Nat. Prod. Rep., 2010, 27, 499–528 | 519
A and sugar B has been suggested to be critical for maintaining the correct conformation and rigidity of the oligosaccharide for effective binding to the minor groove of DNA. ${ }^{195,198}$ The search for an enzyme that catalyzes the hydroxylation of the amine group led to the discovery of a novel aminosugar oxidase (CalE10). The $N$-oxidase CalE10 was shown to be a heme protein that exhibits stringent regiospecificity with limited over- oxidation *in vitro*. ${ }^{199}$

Calicheamicin $\gamma_{I}^{\prime}$ contains a 4-thio-2,4,6-trideoxy-$\alpha$-glucose (sugar B) that is connected to the orsellenic acid moiety through a thioester linkage. The sulfur atom of sugar B contributes to the binding mode of calicheamicin $\gamma_{I}^{\prime}$ by forming hydrogen bonds with an exposed amino proton from 3'-guanine in the minor groove of DNA. ${ }^{195,198}$ Although it remains to be validated, the biosynthetic pathway is also likely to involve the NDP-4-keto-6-deoxy-$\alpha$-glucose intermediate to provide the necessary ring activation for the deoxygenation at C2 position. The mechanism by which the sulfur atom is installed is currently unknown. The closely related esperamicin A${}_{1}$ also contains a similar thiosugar. From the isotope feeding experiment, it is known that the sulfur is derived from L-methionine or L-cysteine. ${ }^{67}$

Although the enzymes for the methylated 3-methoxy-L-rhamnose (sugar C) synthesis have not been confirmed, the biosynthesis is likely to follow a common pathway for rhamnose synthesis with the deoxygenation at C6 facilitated by the formation of the NDP-4-keto-6-deoxy-$\alpha$-D-glucose intermediate (Fig. 31). ${ }^{196,200}$ A putative epimerase and a methyltransferase would catalyze the epimerization at C2 and methylation at C3 respectively.

Calicheamicin $\gamma_{I}^{\prime}$ shares a common aminodideoxypentose (4-amino-3-O-methoxy-2,4,5-trideoxypentose, sugar D) with esperamicin A${}_{1}$ and the indolocarbazole antitumor antibiotic AT2433 (39). ${ }^{201}$ Comparison of the calicheamicin and AT2433 gene clusters uncovered a set of common genes that could be involved in the biosynthesis of the 4-amino-3-O-methoxy-2,4,5-trideoxypentose. ${ }^{202}$ Thorson and coworkers confirmed that the synthesis of the modified pentose proceeds *via* a TDP-sugar pathway. ${ }^{203}$ The pathway involves a decarboxylation step at C6 with a UDP-glucuronate decarboxylase-like protein (CalS9); whereas the deoxygenation of C2 and installation of amino group at C4 probably follow the established mechanisms (Fig. 31). ${ }^{158}$ A similar pathway has also been proposed for the synthesis of the sugar moiety in maduropeptin (Section 6.3.3). ${ }^{75}$

Overall, the biosynthesis of the highly modified monosaccharides of calicheamicin $\gamma_{I}^{\prime}$ and other glycosides seem to share some common themes. ${ }^{158}$ First, the modification of the monosaccharides usually requires the activation of the sugar by nucleotidyltransferase through the formation of a nucleoside monophosphate or diphosphate. Second, the sugar pathways usually involve an NDP-4-keto-6-deoxy-$\alpha$-D-glucose intermediate with the sugar ring activated by the 4-keto group for deoxygenation. Third, the deoxygenation reaction proceeds by different catalytic mechanisms, depending on whether the deoxygenation occurs at the C2, C3, C5 or C6 position.

### 7.1.2.2 Assembly of calicheamicin $\gamma_{I}^{\prime}$ by glycosyltransferases.
The orsellenic acid and sugar moieties are assembled onto the enediyne aglycone in the final stages of calicheamicin biosynthesis. A set of O-glycosyltransferases are utilized to catalyze the transfer of the sugars from the nucleotide diphosphate sugar donors to the aglycone acceptor. By *in vitro* enzymatic assays, Thorson and coworkers confirmed that the glycosyltransferases CalG3 and CalG2 catalyze the first two glycosylation steps in a sequential fashion (Fig. 32). ${ }^{193}$ The crystal structure of CalG3 determined by the same group revealed a typical UDP-glycosyltransferase/glycogen phosphorylase fold found in many glycosyltransferases. ${ }^{193}$ In addition, the authors demonstrated that CalG3 catalyzes the formation of the unusual hydroxylamino glycosidic bond by using TDP-4,6-dideoxy-$\alpha$-D-glucose as a substrate surrogate. The other two putative glycosyltransferases (CalG1 and CalG4) encoded by the gene clusters were also confirmed to be the rhamnosyltransferase and amino-pentosyltransferase for the installation of sugars C and D respectively. ${ }^{161}$ In contrast to the glycosylation reactions catalyzed by CalG2 and G3 that must proceed sequentially, the enzymatic assays suggested that the last two steps may occur at a random order (Fig. 32).

#### 7.1.2.3 Generation of calicheamicin $\gamma_{I}^{\prime}$ derivatives by sugar exchange.
Biochemical validation of the biosynthetic pathways led to the discovery that some of the glycosyltransferases not only possess relaxed substrate specificity, but also exhibit unexpected activity in catalyzing the reverse reaction of glycosylation. ${ }^{161,193}$ These findings inspired Thorson and coworkers to utilize the glycosyltransferases for generating unnatural glycosylated enediyne chromophores. As a proof of principle, by using a 'sugar exchange' strategy developed based on the reversible glycosylation catalyzed by CalG1, the authors created a library of 70 calicheamicin analogs with sugar C substituted by various sugars. (Fig. 33) ${ }^{161}$ The application of the sugar exchange strategy to CalG3 also allowed the substitution of sugar A with ten other monosaccharides. ${ }^{193}$ The glycosyltransferase-catalyzed reversible glycosylation is potentially a very powerful tool for altering the properties of natural product-based therapeutics through glycodiversification. ${ }^{158,161,204,205}$

### 7.2 Esperamicin A${}_{1}$

Esperamicins are highly related to calicheamicins in terms of structure and mode of action. ${ }^{8,9,22}$ However, the enediyne warhead in calicheamicin $\gamma_{I}^{\prime}$ is linked to an aryltetrasaccharide in only one direction and is likely to be anchored less rigidly in the minor groove, whereas the enediyne core of esperamicin A${}_{1}$ is bridged by a trisaccharide and arylmonosaccharide peripheral moieties (Fig. 34). Intercalation of the peripheral moieties locks the enediyne core deep in the minor groove. The distinct binding configuration is likely to account for the predominant single-strand cleavage chemistry observed for esperamicin A${}_{1}$.

Despite the different anchoring site for sugar C, esperamicin A${}_{1}$ contains essentially the same set of monosaccharides (sugars A–D) as calicheamicin $\gamma_{I}^{\prime}$. Although the esperamicin A1 gene cluster has not been sequenced, it is reasonable to believe that some homologous enzymes are involved in the biosynthesis of the monosaccharides. Esperamicin A${}_{1}$ also contains an aryl group with an unknown origin. Isotopic feeding experiments rule out a polyketide origin since the aromatic moiety is not derived from acetate. ${ }^{67}$
Enediyne aglycone

Sugar A
CalG3

Sugar B
CalG2

ACP or CoA-tethered
orsellenic acid
CalO4

Sugar C
CalG1

Sugar D
CalG4

Sugar D
CalG4

Sugar C
CalG1

Fig. 32 Assembly of calicheamicin γ₁ˡ by glycosyltransferases.

### 7.3 Namenicin and shishijimicins

The structurally related namenicin (11) and shishijimicins were the first enediyne chromophores isolated from marine organisms. Namenicin was first isolated from the Japanese marine invertebrate tunicate *Polysyncraton lithostrotum*.²⁰⁶ A follow-up study by Oku and coworkers uncovered shishijimicins A (10), B and C from the thin encrusting orange ascidian *Didemnum proliferum*.²⁰⁷ Given the striking similarity of the enediyne aglycone to those of the calicheamicins and esperamicins, it was suspected that the real producers of namenicin and the shishijimicins might be a symbiotic microbe instead.²⁰⁶,²⁰⁸

Namenicin contains three monosaccharides that resemble sugar A, B and D of calicheamicin γ₁ˡ and esperamicin A₁, except that the N–O glycosidic linkage between sugar A and B is replaced by an unusual C–O linkage. Meanwhile, the shishijimicins contain a disaccharide that may share a similar biosynthetic origin with sugars A and D of calicheamicin γ₁ˡ and esperamicin A₁.

A major difference between the shishijimicins and namenicin is the presence of the tricyclic β-carboline moiety in shishijimicins. The fully aromatized β-carboline or tricyclic prido[3,4-b]indole moiety is in fact not seen in other enediyne natural products. β-Carboline derivatives are known to alter DNA replication by binding DNA or inhibiting DNA repair proteins.²⁰⁹ Shishijimicins exhibit higher cytotoxicity than namenicin, presumably due to the greater DNA binding properties of the β-carboline moiety. Although the biosynthetic origin of the β-carboline in shishijimicins has not been elucidated, the heterocyclic structure is probably derived from tryptophan or tryptamine. The tryptamine origin has been established for a few β-carboline alkaloids from *Alangium lamarcii*, and a possible pathway for the formation of the

Fig. 33 Generation of calicheamicin derivatives by exploiting reversible glycosylation.

Fig. 34 Building blocks of esperamicin A₁ (9).

Fig. 35 A possible biosynthetic mechanism for the formation of the β-carboline in the shishijimicins.

tricyclic ring is through the condensation with an aldehyde by the Pictet-Spengler mechanism (Fig. 35).²¹⁰

### 7.4 Dynemicins and uncialamycin

Dynemicins (as represented by dynemycin A (12)), and uncialamycin (13) are potent antitumor antibiotics isolated from *Micromonospora chersina* M956-1 and a strain related to *Streptomyces cyanogenus*, respectively.²¹¹ The highly rigid structures of dynemicins and uncialamycin consist of a bicyclo[7.3.1]enediyne core and an anthraquinone moiety. The two subfamilies of enediynes are most likely to share the same biosynthetic origin given the striking structural similarity. An isotopic incorporation experiment by Tokiwa *et al.* established

metabolites. Notably, Thorson and coworkers uncovered an unprecedented self-resistance mechanism by which the microbes destroy the reactive calicheamicins by employing a self-sacrificing protein (CalC).<sup>69,213,214</sup>

The protein CalC was discovered to confer calicheamicin resistance to *Micromonospora* and *E. coli* by inhibiting DNA cleavage, during the initial search for enediyne gene locus.<sup>69</sup> The resistance mechanism involves the sequestration of calicheamicin by the small protein CalC. Once bound by CalC, the reactive 1,4-benzenoid diradical resulting from cycloaromatization would abstract hydrogen atoms from the Gly<sup>113</sup> residue, rather than the DNA ribose (Fig. 37). The interception by CalC deactivates the reactive diradical and transfers the radical onto CalC, which undergoes site-specific proteolytic cleavage in the presence of O<sub>2</sub>.<sup>213</sup> The solution structure of CalC revealed a START domain-like fold with a putative binding pocket for calicheamicin.<sup>214</sup> A model of the CalC-calicheamicin γ<sub>1</sub> complex based on NMR titration supports the self-resistance mechanism by revealing the juxtaposition of the key residue Gly<sup>113</sup> and the -yne group of the enediyne core (Fig. 37). The discovery of the self-sacrifice mechanism in protection against calicheamicins established a new paradigm that differs from previously known resistance mechanisms.<sup>215</sup>

## Summary and perspective

This review summarizes the remarkable advances in our understanding of enediyne biosynthesis made from 2002 to the end of 2009. This period of time, beginning with the sequencing of the first two enediyne gene clusters, has proven to be extremely fruitful for the field of enediyne biosynthesis. The genetic, biochemical, structural studies highlighted above have revealed a cornucopia of novel chemical transformations and enzymes that underscore the highly unusual enediyne structure as well as the diversity of the peripheral moieties.

The biosynthetic origin of the enediyne warheads has been at the center of speculation and discussion since the unveiling of the first enediyne structure. Sequencing of the gene clusters not only confirms the postulated polyketide origin of the enediyne warheads, but also indicates that the bicyclo[7.3.1]tridecadiyene and bicyclo[7.3.0]dodecadienediyne cores share similar biosynthetic pathways. Recombinant enediyne PKSs produce multiple *in vitro* products, with the polyene 21 as the exclusive product under some conditions. Whether 21 or 20 is the biosynthetic precursor of the enediyne core remains controversial at this moment. If 21 and 20 are indeed aberrant products, then what might be the physiological product of the PKS? A conceivable scenario is that the function of the thioesterase is to remove the derailed products (*e.g.* 15–19, Fig. 11), and that the authentic intermediate remains anchored to the PKS for downstream oxidation and cyclization to furnish the enediyne core. As a corollary question, is there a *trans*-acting factor that regulates the KR and DH domain for controlling the reduction and dehydration? From the gene clusters, it is not apparent which gene product may play the role of the *trans*-acting factor. There are several other major questions remaining to be answered in order to fully understand the biogenesis of the enediyne core. These highly intertwined questions are: (*i*) What is the function of the CalU 15 homologs? If they are indeed diiron-containing

## Regulation of enediyne biosynthesis

Given the cytotoxicity of the enediyne chromophores, it was reasonable to assume that the biosynthesis is tightly regulated in the microbial cells. This assumption is supported by the presence of the ORFs that encode putative regulatory proteins in the gene clusters. The most apparent ones are the transcriptional factors with predicted DNA binding motifs. In the C-1027 biosynthetic gene cluster, three ORFs (*sgcR1*, *sgcR2*, *sgcR3*) have been tentatively annotated as regulatory genes.<sup>70</sup> Hong and coworkers demonstrated that the disruption of *sgcR3* abolished C-1027 production, indicating that SgcR3 is a positive transcriptional regulator in *S. globisporus*.<sup>212</sup> It was further shown that SgcR3 binds directly to the promoter region of *sgcR1* and *sgcR2*, suggesting that SgcR3 regulates the biosynthesis indirectly through a hierarchy of regulatory networks. Apart from the limited knowledge on SgcR3, the mechanisms by which the regulatory proteins control enediyne biosynthesis largely remain to be elucidated. A better understanding of the regulatory mechanisms should facilitate the discovery and identification of new enediyne natural products.

## Self-resistance mechanisms

In addition to the regulation on the transcriptional level, enediyne-producing microbes have further evolved self-resistance mechanisms to protect themselves from the highly toxic

CalC

![Chemical Structure](https://via.placeholder.com/300)

Enediyne  
core  

Gly113

Chromosomal  
DNA

![Chemical Structure](https://via.placeholder.com/300)

Enediyne  
core

CalC degradation

O₂

Oxidative cleavage  
of DNA

Fig. 37 Proposed resistance mechanism by the self-sacrifice of CalC. Models of the CalC–calicheamicin complex²¹⁴ and the solution structure of the DNA–calicheamicin γ₁' complex¹⁹⁸ are also included.

acetylenases as inferred from sequence homology, how do they achieve the regioselectivity during oxidation? (ii) When and how does the cyclization occur, *i.e.* does it occur before or after the formation of the acetylenic bonds? If the enediynes indeed share the same polyketide precursor, what are the factors that determine the different cyclization patterns for the calicheamicin, dynemicin and C-1027-type enediynes? (iii) When does the chain truncation, and probably a rearrangement of the double bonds, take place for the 9-membered enediyne? (iv) What are the functions of the highly conserved CalT3 and CalU14 homologs? Do they play any active roles in biosynthesis?

The structural complexity and diversity exhibited by enediyne chromophores arise from the variation of the number, structure, and anchoring site of the sugar and aromatic moieties. Underlying the structural diversity is the complex consortium of enzymes for aromatic moiety synthesis and sugar modification. The aromatic moieties are derived from either polyketides or primary metabolic intermediates, whereas the unusual sugars are generated from the modification of common sugars by the enzymes encoded in the gene clusters. Established sugar-modifying mechanisms for deoxygenation, decarboxylation and installation of heterologous atoms in other glycosides seem to be conserved in enediyne pathways. Despite the structural complexity and diversity, a simple synthetic logic has emerged from recent studies: the assembly of the enediyne chromophores involves the convergence of parallel biosynthetic pathways in the final stage. The building blocks are assembled separately and joined together to furnish the final product by using glycosyltransferases, ligases and condensation enzymes with stringent regioselectivity. Another layer of complexity that has yet to be fully understood is how the microbes synchronize and orchestrate the converging pathways. Although the biosynthetic pathways are apparently regulated on the transcriptional level, the details of the regulatory circuits have yet to be revealed.

Despite the unsurpassed antitumor activity, the direct use of naturally occurring enediyne chromophores as therapeutics has been hampered by the lack of tumour cell specificity and other less than ideal pharmacological properties. The characterization of the enzymes and pathways for enediyne biosynthesis has opened numerous windows of opportunity for producing structural analogs with improved pharmacological property by metabolic engineering. To this end, the proof-of-principle

experiments with calicheamicin γ1 and C-1027 have provided illustrative examples of obtaining structural analogs through rational design and manipulation of the biosynthetic pathway. And finally, with the rapid advances in genomic sequencing methods and the understanding of the regulation of secondary metabolite biosynthesis, more enediyne natural products are bound to be discovered. It is reasonable to expect that new aromatic and sugar building blocks produced by novel biosynthetic pathways will be revealed. It will be of great interest to see – although with less certainty – whether any novel enediyne core can be discovered from nature's immense molecular archive.

### 11 Acknowledgements

The work on polyketide synthase in the author’s laboratory is supported by the Ministry of Education of Singapore through the Academic Research Council (ARC) Tier II grants (R60/06 and RG17/09). The author would like to thank Xun-Ming Zhao for his assistance in preparing some of the figures.

### 12 References

1. K. Edo, S. Katamine, F. Kitame, N. Ishida, Y. Koide, G. Kusano and S. Nozoe, *J. Antibiot.*, 1980, **33**, 347–351.
2. K. Edo, M. Mizugaki, Y. Koide, H. Seto, K. Furihata, N. Otake and N. Ishida, *Tetrahedron Lett.*, 1985, **26**, 331–334.
3. M. D. Lee, T. S. Dunne, C. C. Chang, G. A. Ellestad, M. M. Siegel, G. O. Morton, W. J. McGahren and D. B. Borders, *J. Am. Chem. Soc.*, 1987, **109**, 3466–3468.
4. M. D. Lee, T. S. Dunne, C. C. Chang, M. M. Siegel, G. O. Morton, G. A. Ellestad, W. J. McGahren and D. B. Borders, *J. Am. Chem. Soc.*, 1992, **114**, 985–997.
5. M. D. Lee, T. S. Dunne, M. M. Siegel, C. C. Chang, G. O. Morton and D. B. Borders, *J. Am. Chem. Soc.*, 1987, **109**, 3464–3466.
6. J. Golik, J. Clardy, G. Dubay, G. Groenewold, H. Kawaguchi, M. Konishi, B. Krishnan, H. Ohkuma, K. Saitoh and T. W. Doyle, *J. Am. Chem. Soc.*, 1987, **109**, 3461–3462.
7. J. Golik, T. W. Doyle, B. Krishnan, G. Dubay and J. A. Matson, *J. Antibiot.*, 1989, **42**, 1784–1789.
8. J. Golik, G. Dubay, G. Groenewold, H. Kawaguchi, M. Konishi, B. Krishnan, H. Ohkuma, K. Saitoh and T. W. Doyle, *J. Am. Chem. Soc.*, 1987, **109**, 3462–3464.
9. M. Konishi, H. Ohkuma, K. I. Saitoh, H. Kawaguchi, J. Golik, G. Dubay, G. Groenewold, B. Krishnan and T. W. Doyle, *J. Antibiot.*, 1985, **38**, 1605–1609.
10. R. H. Feling, G. O. Buchanan, T. J. Mincer, C. A. Kauffman, P. R. Jensen and W. Fenical, *Angew. Chem., Int. Ed.*, 2003, **42**, 355–357.
11. G. O. Buchanan, P. G. Williams, R. H. Feling, C. A. Kauffman, P. R. Jensen and W. Fenical, *Org. Lett.*, 2005, **7**, 2731–2734.
12. D.-C. Oh, P. G. Williams, C. A. Kauffman, P. R. Jensen and W. Fenical, *Org. Lett.*, 2006, **8**, 1021–1024.
13. J. S. Thorson, E. L. Sievers, J. Ahlert, E. Shepard, R. E. Whitwam, K. C. Onwueme and M. Ruppen, *Curr. Pharm. Des.*, 2000, **6**, 1841–1879.
14. A. L. Smith and K. C. Nicolaou, *J. Med. Chem.*, 1996, **39**, 2103–2117.
15. Y. Sugiura and T. Matsumoto, *Biochemistry*, 1993, **32**, 5548–5553.
16. Y. Sugiura, T. Shiraki, M. Konishi and T. Oki, *Proc. Natl. Acad. Sci. U. S. A.*, 1990, **87**, 3831–3835.
17. N. Miyagawa, D. Sasaki, M. Matsuoka, M. Imanishi, T. Ando and Y. Sugiura, *Biochem. Biophys. Res. Commun.*, 2003, **306**, 87–92.
18. N. Zein, K. L. Colson, J. E. Leet, D. R. Schroeder, W. Solomon, T. W. Doyle and A. M. Casazza, *Proc. Natl. Acad. Sci. U. S. A.*, 1993, **90**, 2822–2826.
19. N. Zein, A. M. Casazza, T. W. Doyle, J. E. Leet, D. R. Schroeder, W. Solomon and S. G. Nadler, *Proc. Natl. Acad. Sci. U. S. A.*, 1993, **90**, 8009–8012.
20. Y. Okuno, T. Iwashita, T. Otani and Y. Sugiura, *J. Am. Chem. Soc.*, 1996, **118**, 4729–4730.
21. L. Yu, S. Mah, T. Otani and P. Dedon, *J. Am. Chem. Soc.*, 1995, **117**, 8877–8878.
22. J. Xu, J. Wu and P. C. Dedon, *Biochemistry*, 1998, **37**, 1890–1897.
23. J. B. Biggins, J. R. Prudent, D. J. Marshall, M. Ruppen and J. S. Thorson, *Proc. Natl. Acad. Sci. U. S. A.*, 2000, **97**, 13537–13542.
24. R. R. Jones and R. G. Bergman, *J. Am. Chem. Soc.*, 1972, **94**, 660–661.
25. A. G. Myers, *Tetrahedron Lett.*, 1987, **28**, 4493–4494.
26. M. Hirama, K. Akiyama, T. Tanaka, T. Noda, K.-i. Ida, I. Sato, R. Hanaishi, S. Fukuda-Ishisaka, M. Ishiguro, T. Otani and J. E. Leet, *J. Am. Chem. Soc.*, 2000, **122**, 720–721.
27. E. R. Boghaert, L. Sridharan, D. C. Armellino, K. M. Khandke, J. F. DiJoseph, A. Kunz, M. M. Dougher, F. Jiang, L. B. Kalyandrug, P. R. Hamann, P. Frost and N. K. Damle, *Clin. Cancer Res.*, 2004, **10**, 4538–4549.
28. N. K. Damle and P. Frost, *Curr. Opin. Pharmacol.*, 2003, **3**, 386–390.
29. J. F. DiJoseph, D. C. Armellino, E. R. Boghaert, K. Khandke, M. M. Dougher, L. Sridharan, A. Kunz, P. R. Hamann, B. Gorovits, C. Udata, J. K. Moran, A. G. Popplewell, S. Stephens, P. Frost and N. K. Damle, *Blood*, 2004, **103**, 1807–1814.
30. H. Maeda, *Adv. Drug Delivery Rev.*, 2001, **46**, 169–185.
31. A. Takeshita, N. Yamakage, K. Shinjo, T. Ono, I. Hirano, S. Nakamura, K. Shigeno, T. Tobita, M. Maekawa, H. Kiyoi, T. Naoe, K. Ohnishi, Y. Sugimoto and R. Ohno, *Leukemia*, 2009, **23**, 1329–1336.
32. N. Ji, H. O’Dowd, B. M. Rosen and A. G. Myers, *J. Am. Chem. Soc.*, 2006, **128**, 14825–14827.
33. S. Kobayashi, M. Hori, G. X. Wang and M. Hirama, *J. Org. Chem.*, 2006, **71**, 636–644.
34. R. D. Groneberg, T. Miyazaki, N. A. Stylianides, T. J. Schulze, W. Stahl, E. P. Schreiner, T. Suzuki, Y. Iwabuchi, A. L. Smith and K. C. Nicolaou, *J. Am. Chem. Soc.*, 1993, **115**, 7593–7611.
35. S. A. Hitchcock, S. H. Boyer, M. Y. Chumoyer, S. H. Olson and S. J. Danishefsky, *Angew. Chem., Int. Ed. Engl.*, 1994, **33**, 858–862.
36. S. A. Hitchcock, M. Y. Chumoyer, S. H. Boyer, S. H. Olson and S. J. Danishefsky, *J. Am. Chem. Soc.*, 1995, **117**, 5750–5756.
37. K. C. Nicolaou, C. W. Hummel, M. Nakada, K. Shibayama, E. N. Pitsinos, H. Saimoto, Y. Mizuno, K. U. Baldenius and A. L. Smith, *J. Am. Chem. Soc.*, 1993, **115**, 7625–7635.
38. K. C. Nicolaou, C. W. Hummel, E. N. Pitsinos, M. Nakada, A. L. Smith, K. Shibayama and H. Saimoto, *J. Am. Chem. Soc.*, 1992, **114**, 10082–10084.
39. K. C. Nicolaou, H. Zhang, J. S. Chen, J. J. Crawford and L. Pasunoori, *Angew. Chem., Int. Ed.*, 2007, **46**, 4704–4707.
40. M. D. Shair, T. Y. Yoon and S. J. Danishefsky, *Angew. Chem., Int. Ed. Engl.*, 1995, **34**, 1721–1723.
41. M. D. Shair, T. Y. Yoon, K. K. Mosny, T. C. Chou and S. J. Danishefsky, *J. Am. Chem. Soc.*, 1996, **118**, 9509–9525.
42. A. L. Smith, E. N. Pitsinos, C. K. Hwang, Y. Mizuno, H. Saimoto, G. R. Scarlato, T. Suzuki and K. C. Nicolaou, *J. Am. Chem. Soc.*, 1993, **115**, 7612–7624.
43. K. C. Nicolaou, J. S. Chen, H. J. Zhang and A. Montero, *Angew. Chem., Int. Ed.*, 2008, **47**, 185–189.
44. S. Kawata, S. Ashizawa and M. Hirama, *J. Am. Chem. Soc.*, 1997, **119**, 12012–12013.
45. F. Ren, P. C. Hogan, A. J. Anderson and A. G. Myers, *J. Am. Chem. Soc.*, 2007, **129**, 5381–5383.
46. K. C. Nicolaou, A. L. Smith and E. W. Yue, *Proc. Natl. Acad. Sci. U. S. A.*, 1993, **90**, 5881–5888.
47. G. B. Jones, J. M. Wright, G. Plourde, A. D. Purohit, J. K. Wyatt, G. Hynd and F. Fouad, *J. Am. Chem. Soc.*, 2000, **122**, 9872–9873.
48. E. Kraka and D. Cremer, *J. Am. Chem. Soc.*, 2000, **122**, 8245–8264.
49. K.-i. Ida and M. Hirama, *J. Am. Chem. Soc.*, 1995, **117**, 8875–8876.
50. J. M. Nuss, R. A. Rennels and B. H. Levine, *J. Am. Chem. Soc.*, 1993, **115**, 6991–6992.
51. K. Toshima, K. Ohta, A. Ohashi, T. Nakamura, M. Nakata, K. Tatsuta and S. Matsumura, *J. Am. Chem. Soc.*, 1995, **117**, 4822–4831.
52. K. Toshima, K. Ohta, K. Yanagawa, T. Kano, M. Nakata, M. Kinoshita and S. Matsumura, *J. Am. Chem. Soc.*, 1995, **117**, 10825–10831.
53. F. S. Fouad, J. M. Wright, A. D. Purohit, J. K. Wyatt, A. El-Shafey, G. Hynd, C. F. Crasto, Y. Lin and G. B. Jones, *J. Org. Chem.*, 2005, **70**, 9789–9797.

54 D. R. Kennedy, J. Ju, B. Shen and T. A. Beerman, *Proc. Natl. Acad. Sci. U. S. A.*, 2007, **104**, 17632–17637.

55 D. R. Pandithavidana, A. Poloukhine and V. V. Popik, *J. Am. Chem. Soc.*, 2009, **131**, 351–356.

56 K. C. Nicolaou and W.-M. Dai, *Angew. Chem., Int. Ed.*, 1994, **30**, 1387–1530.

57 J. S. Thorson, B. Shen, R. E. Whitwam, W. Liu, Y. Li and J. Ahlert, *Bioorg. Chem.*, 1999, **27**, 172–188.

58 S. J. Danishefsky and M. D. Shair, *J. Org. Chem.*, 1996, **61**, 16–44.

59 B. Shen, *Curr. Opin. Chem. Biol.*, 2003, **7**, 285–295.

60 B. Shen, W. Liu and K. Nonaka, *Curr. Med. Chem.*, 2003, **10**, 2317–2325.

61 U. Galm, M. H. Hager, S. G. Van Lanen, J. H. Ju, J. S. Thorson and B. Shen, *Chem. Rev.*, 2005, **105**, 739–758.

62 S. G. Van Lanen and B. Shen, *Curr. Top. Med. Chem.*, 2008, **8**, 448–459.

63 S. L. Schreiber and L. L. Kiessling, *J. Am. Chem. Soc.*, 1988, **110**, 631–633.

64 O. D. Hensens, J. L. Giner and I. H. Goldberg, *J. Am. Chem. Soc.*, 1989, **111**, 3295–3299.

65 Y. Tokiwa, M. Miyoshi-Saitoh, H. Kobayashi, R. Sunaga, M. Konishi, T. Oki and S. Iwasaki, *J. Am. Chem. Soc.*, 1992, **114**, 4107–4110.

66 J. Davies, H. Wang, T. Taylor, K. Warabi, X. H. Huang and R. J. Andersen, *Org. Lett.*, 2005, **7**, 5233–5236.

67 K. S. Lam, J. A. Veitch, J. Golik, B. Krishnan, S. E. Klohr, K. J. Volk, S. Forenza and T. W. Doyle, *J. Am. Chem. Soc.*, 1993, **115**, 12340–12345.

68 W. Liu and B. Shen, *Antimicrob. Agents Chemother.*, 2000, **44**, 382–392.

69 R. E. Whitwam, J. Ahlert, T. R. Holman, M. Ruppen and J. S. Thorson, *J. Am. Chem. Soc.*, 2000, **122**, 1556–1557.

70 W. Liu, S. D. Christenson, S. Standage and B. Shen, *Science*, 2002, **297**, 1170–1173.

71 J. Ahlert, E. Shepard, N. Lomovskaya, E. Zazopoulos, A. Staffa, B. O. Bachmann, K. X. Huang, L. Fonstein, A. Czisny, R. E. Whitwam, C. M. Farnet and J. S. Thorson, *Science*, 2002, **297**, 1173–1176.

72 E. Zazopoulos, K. X. Huang, A. Staffa, W. Liu, B. O. Bachmann, K. Nonaka, J. Ahlert, J. S. Thorson, B. Shen and C. M. Farnet, *Nat. Biotechnol.*, 2003, **21**, 187–190.

73 W. Liu, K. Nonaka, L. P. Nie, J. Zhang, S. D. Christenson, J. Bae, S. G. Van Lanen, E. Zazopoulos, C. M. Farnet, C. F. Yang and B. Shen, *Chem. Biol.*, 2005, **12**, 293–302.

74 Q. Gao and J. S. Thorson, *FEMS Microbiol. Lett.*, 2008, **282**, 105–114.

75 S. G. Van Lanen, T. J. Oh, W. Liu, E. Wendt-Pienkowski and B. Shen, *J. Am. Chem. Soc.*, 2007, **129**, 13082–13094.

76 D. W. Uduary, L. Zeigler, R. N. Asolkar, V. Singan, A. Lapidus, W. Fenical, P. R. Jensen and B. S. Moore, *Proc. Natl. Acad. Sci. U. S. A.*, 2007, **104**, 10376–10381.

77 P. G. Williams, G. O. Buchanan, R. H. Feling, C. A. Kauffman, P. R. Jensen and W. Fenical, *J. Org. Chem.*, 2005, **70**, 6196–6203.

78 K. A. Reed, R. R. Manam, S. S. Mitchell, J. Xu, S. Teisan, T. H. Chao, G. Deyanat-Yazdi, S. T. C. Neuteboom, K. S. Lam and B. C. M. Potts, *J. Nat. Prod.*, 2007, **70**, 269–276.

79 W. Liu, J. Ahlert, Q. J. Gao, E. Wendt-Pienkowski, B. Shen and J. S. Thorson, *Proc. Natl. Acad. Sci. U. S. A.*, 2003, **100**, 11959–11963.

80 J. Staunton and K. J. Weissman, *Nat. Prod. Rep.*, 2001, **18**, 380–416.

81 R. J. Cox, *Org. Biomol. Chem.*, 2007, **5**, 2010–2026.

82 L. Shao, X.-D. Qu, X.-Y. Jia, Q.-F. Zhao, Z.-H. Tian, M. Wang, G.-L. Tang and W. Liu, *Biochem. Biophys. Res. Commun.*, 2006, **345**, 133–139.

83 S. M. Ma, J. W. H. Li, J. W. Choi, H. Zhou, K. K. M. Lee, V. A. Moorthie, X. Xie, J. T. Kealey, N. A. Da Silva, J. C. Vederas and Y. Tang, *Science*, 2009, **326**, 589–592.

84 J. Kennedy, K. Auclair, S. G. Kendrew, C. Park, J. C. Vederas and C. R. Hutchinson, *Science*, 1999, **284**, 1368–1372.

85 J. Zhang, S. G. Van Lanen, J. Ju, W. Liu, P. C. Dorrestein, W. Li, N. L. Kelleher and B. Shen, *Proc. Natl. Acad. Sci. USA*, 2008, **105**, 1461–1465.

86 E. Murugan and Z.-X. Liang, *FEBS Lett.*, 2008, **582**, 1097–1103.

87 L. E. N. Quadri, P. H. Weinreb, M. Lei, M. M. Nakano, P. Zuber and C. T. Walsh, *Biochemistry*, 1998, **37**, 1585–1595.

88 J. W. Kira, H. Hui, O. Markiyan, P. S. Alexis and F. L. Peter, *ChemBioChem*, 2004, **5**, 116–125.

89 I. B. Lomakin, Y. Xiong and T. A. Steitz, *Cell*, 2007, **129**, 319–332.

90 T. Maier, M. Leibundgut and N. Ban, *Science*, 2008, **321**, 1315–1322.

91 A. T. Keatinge-Clay and R. M. Stroud, *Structure*, 2006, **14**, 737–748.

92 A. Keatinge-Clay, *J. Mol. Biol.*, 2008, **384**, 941–953.

93 I. Fujii, *Nat. Prod. Rep.*, 2009, **26**, 155–169.

94 I. Fujii, Y. Ono, H. Tada, K. Gomi, Y. Ebizuka and U. Sankawa, *Mol. Gen. Genet.*, 1996, **253**, 1–10.

95 D. J. Bedford, E. Schweizer, D. A. Hopwood and C. Khosla, *J. Bacteriol.*, 1995, **177**, 4544–4548.

96 R. Kong, L. P. Goh, C. W. Liew, Q. S. Ho, E. Murugan, B. Li, K. Tang and Z.-X. Liang, *J. Am. Chem. Soc.*, 2008, **130**, 8142–8143.

97 M. Kotada, R. Kong, I. Qureshi, Q. Ho, H. Sun, C. W. Liew, L. P. Goh, P. Cheung, Y. Mu, J. Lescar and Z.-X. Liang, *J. Biol. Chem.*, 2008, **284**, 15739–15749.

98 H. Sun, R. Kong, D. Zhu, W. Lu, Q. Ji, C. W. Liew, J. Lescar, G. Zhong and Z.-X. Liang, *Chem. Commun.*, 2009, 7399–7401.

99 K. Belecki, J. M. Crawford and C. A. Townsend, *J. Am. Chem. Soc.*, 2009, **131**, 12564–12566.

100 L. M. Halo, J. W. Marshall, A. A. Yakasai, Z. Song, C. P. Butts, M. P. Crump, M. Heneghan, A. M. Bailey, T. J. Simpson, C. M. Lazarus and R. J. Cox, *ChemBioChem*, 2008, **9**, 585–594.

101 D. L. Akey, J. D. Kittendorf, J. W. Giraldes, R. A. Fecik, D. H. Sherman and J. L. Smith, *Nat. Chem. Biol.*, 2006, **2**, 537–542.

102 J. W. Trauger, R. M. Kohli, H. D. Mootz, M. A. Marahiel and C. T. Walsh, *Nature*, 2000, **407**, 215–218.

103 B. S. Kim, T. A. Cropp, B. J. Beck, D. H. Sherman and K. A. Reynolds, *J. Biol. Chem.*, 2002, **277**, 48028–48034.

104 Z. Hu, B. A. Pfeifer, E. Chao, S. Murli, J. Kealey, J. R. Carney, G. Ashley, C. Khosla and C. R. Hutchinson, *Microbiology*, 2003, **149**, 2213–2225.

105 M. L. Heathcote, J. Staunton and P. F. Leadlay, *Chem. Biol.*, 2001, **8**, 207–220.

106 A. Koglin, F. Lohr, F. Bernhard, V. V. Rogov, D. P. Frueh, E. R. Strieter, M. R. Mofid, P. Guntert, G. Wagner, C. T. Walsh, M. A. Marahiel and V. Dotsch, *Nature*, 2008, **454**, 907–911.

107 U. Linne, D. Schwarzer, G. N. Schroeder and M. A. Marahiel, *Eur. J. Biochem.*, 2004, **271**, 1536–1545.

108 M. Kotowska, K. Pawlik, A. R. Butler, E. Cundliffe, E. Takano and K. Kuczek, *Microbiology*, 2002, **148**, 1777–1783.

109 S. C. Dillon and A. Bateman, *BMC Bioinformatics*, 2004, **5**, 109.

110 D. Leduc, A. Battesti and E. Bouveret, *J. Bacteriol.*, 2007, **189**, 7112–7126.

111 J. J. Salas and J. B. Ohlrogge, *Arch. Biochem. Biophys.*, 2002, **403**, 25–34.

112 D. Chen, R. Wu, T. L. Bryan and D. Dunaway-Mariano, *Biochemistry*, 2009, **48**, 511–513.

113 Z. F. Guo, Y. Sun, S. Zheng and Z. Guo, *Biochemistry*, 2009.

114 B. G. Fox, K. S. Lyle and C. E. Rogge, *Acc. Chem. Res.*, 2004, **37**, 421–429.

115 A. S. Carlsson, S. Thomaeus, M. Hamberg and S. Stymne, *Eur. J. Biochem.*, 2004, **271**, 2991–2997.

116 B. G. Fox, J. Shanklin, C. Somerville and E. Munck, *Proc. Natl. Acad. Sci. U. S. A.*, 1993, **90**, 2486–2490.

117 D. W. Reed, D. R. Polichuk, P. H. Buist, S. J. Ambrose, R. J. Sasata, C. K. Savile, A. R. S. Ross and P. S. Covello, *J. Am. Chem. Soc.*, 2003, **125**, 10635–10640.

118 L. S. Kappen and I. H. Goldberg, *Biochemistry*, 1980, **19**, 4786–4790.

119 L. S. Kappen, M. A. Napier and I. H. Goldberg, *Proc. Natl. Acad. Sci. U. S. A.*, 1980, **77**, 1970–1974.

120 M. Hanada, H. Ohkuma, T. Yonemoto, K. Tomita, M. Ohbayashi, H. Kamei, T. Miyaki, M. Konishi, H. Kawaguchi and S. Forenza, *J. Antibiot.*, 1991, **44**, 403–414.

121 Z. H. Cui, L. F. Wang, S. M. Wang, G. W. Li and B. Hong, *FEMS Microbiol. Lett.*, 2009, **301**, 57–68.

122 K. H. Kim, B. M. Kwon, A. G. Myers and D. C. Rees, *Science*, 1993, **262**, 1042–1046.

123 E. Adjadj, E. Quiniou, J. Mispelter, V. Favaudon and J. M. Lhoste, *Eur. J. Biochem.*, 1992, **203**, 505–511.

124 H. Takashima, T. Yoshida, T. Ishino, K. Hasuda, T. Ohkubo and Y. Kobayashi, *J. Biol. Chem.*, 2005, **280**, 11340–11346.

125 T. Tanaka, S. Fukuda-Ishisaka, M. Hirama and T. Otani, *J. Mol. Biol.*, 2001, **309**, 267–283.

126 G. C. P. Sudhahar, K. Balamurugan and D. H. Chin, *J. Biol. Chem.*, 2000, **275**, 39900–39906.

127 J. R. Baker, D. N. Woolfson, F. W. Muskett, R. G. Stoneman, M. D. Urbaniak and S. Caddick, *ChemBioChem*, 2007, **8**, 704–717.

128 S. Caddick, F. W. Muskett, R. G. Stoneman and D. N. Woolfson, *J. Am. Chem. Soc.*, 2006, **128**, 4204–4205.

129 M. D. Urbaniak, J. P. Bingham, J. A. Hartley, D. N. Woolfson and S. Caddick, *J. Med. Chem.*, 2004, **47**, 4710–4715.

130 A. Drevelle, M. Graille, B. Heyd, I. Sorel, N. Ulryck, F. Pecorari, M. Desmadril, H. van Tilburg and P. Minard, *J. Mol. Biol.*, 2006, **358**, 455–471.

131 T. Otani, Y. M. Minami, T. R. Zhang and M.-Y. Xie, *J. Antibiot.*, 1988, **41**, 1580–1585.

132 K. Ida, T. Ishii, M. Hirama, T. Otani, Y. Minami and K. Yoshida, *Tetrahedron Lett.*, 1993, **34**, 4079–4082.

133 Y. Minami, K. Yoshida, R. Azuma, M. Saeki and T. Otani, *Tetrahedron Lett.*, 1993, **34**, 2633–2636.

134 S. G. Van Lanen, S. Lin and B. Shen, *Proc. Natl. Acad. Sci. U. S. A.*, 2008, **105**, 494–499.

135 S. D. Christenson, W. Liu, M. D. Toney and B. Shen, *J. Am. Chem. Soc.*, 2003, **125**, 6062–6063.

136 S. D. Christenson, W. M. Wu, M. A. Spies, B. Shen and M. D. Toney, *Biochemistry*, 2003, **42**, 12708–12718.

137 C. V. Christianson, T. J. Montavon, G. M. Festin, H. A. Cooke, B. Shen and S. D. Bruner, *J. Am. Chem. Soc.*, 2007, **129**, 15744–15745.

138 W. Mutatu, K. L. Klettke, C. Foster and K. D. Walker, *Biochemistry*, 2007, **46**, 9785–9794.

139 C. V. Christianson, T. J. Montavon, S. G. Van Lanen, B. Shen and S. D. Bruner, *Biochemistry*, 2007, **46**, 7205–7214.

140 T. J. Montavon, C. V. Christianson, G. M. Festin, B. Shen and S. D. Bruner, *Bioorg. Med. Chem. Lett.*, 2008, **18**, 3099–3102.

141 S. G. Van Lanen, P. C. Dorrestein, S. D. Christenson, W. Liu, J. H. Ju, N. L. Kelleher and B. Shen, *J. Am. Chem. Soc.*, 2005, **127**, 11594–11595.

142 N. A. Magarvey, P. D. Fortin, P. M. Thomas, N. L. Kelleher and C. T. Walsh, *ACS Chem. Biol.*, 2008, **3**, 542–554.

143 F. H. Vaillancourt, E. Yeh, D. A. Vosburg, S. Garneau-Tsodikova and C. T. Walsh, *Chem. Rev.*, 2006, **106**, 3364–3378.

144 C. Wagner, M. El Omari and G. M. König, *J. Nat. Prod.*, 2009, **72**, 540–553.

145 S. Lin, S. G. Van Lanen and B. Shen, *J. Am. Chem. Soc.*, 2007, **129**, 12432–12438.

146 S. Lin, S. G. Van Lanen and B. Shen, *J. Am. Chem. Soc.*, 2008, **130**, 6616–6623.

147 E. Eichhorn, J. R. van der Ploeg and T. Leisinger, *J. Biol. Chem.*, 1999, **274**, 26639–26646.

148 U. Kirchner, A. H. Westphal, R. Müller and W. J. H. van Berkel, *J. Biol. Chem.*, 2003, **278**, 47545–47553.

149 R. J. Parry and W. Li, *J. Biol. Chem.*, 1997, **272**, 23303–23311.

150 J. Valton, L. Filisetti, M. Fontecave and V. Niviere, *J. Biol. Chem.*, 2004, **279**, 44362–44369.

151 J. Valton, C. Mathevon, M. Fontecave, V. Niviere and D. P. Ballou, *J. Biol. Chem.*, 2008, **283**, 10287–10296.

152 S. Lin, S. G. Van Lanen and B. Shen, *Proc. Natl. Acad. Sci. U. S. A.*, 2009, **106**, 4183–4188.

153 J. Vanderkeyden, *Crit. Rev. Microbiol.*, 2001, **27**, 75–131.

154 H. S. Schadt, S. Schadt, F. Oldach and R. D. Süßmuth, *J. Am. Chem. Soc.*, 2009, **131**, 3481–3483.

155 T. Knochel, A. Ivens, G. Hester, A. Gonzalez, R. Bauerle, M. Wilmanns, K. Kirschner and J. N. Jansonius, *Proc. Natl. Acad. Sci. U. S. A.*, 1999, **96**, 9479–9484.

156 A. A. Morollo and M. J. Eck, *Nat. Struct. Biol.*, 2001, **8**, 243–247.

157 M. McDonald, D. V. Mavrodi, L. S. Thomashow and H. G. Floss, *J. Am. Chem. Soc.*, 2001, **123**, 9459–9460.

158 C. J. Thibodeaux, C. E. Melancon and H.-w. Liu, *Nature*, 2007, **446**, 1008–1016.

159 T. A. Beerman, L. S. Gawron, S. Shin, B. Shen and M. M. McHught, *Cancer Res.*, 2009, **69**, 593–598.

160 J. S. Thorson, T. J. Hosted, J. Jiang, J. B. Biggins and J. Ahlert, *Curr. Org. Chem.*, 2001, **5**, 139–167.

161 C. Zhang, B. R. Griffith, Q. Fu, C. Albermann, X. Fu, I.-K. Lee, L. Li and J. S. Thorson, *Science*, 2006, **313**, 1291–1294.

162 A. Minami, K. Kakinuma and T. Eguchi, *Tetrahedron Lett.*, 2005, **46**, 6187–6190.

163 X. Fu, *Nat. Biotechnol.*, 2003, **21**, 1467–1469.

164 K. Kudo, T. Suto, Y. Koide, K. Edo and N. Ishida, *J. Antibiot.*, 1982, **35**, 1111–1115.

165 B. Sthapit, T. J. Oh, R. Lamichhane, K. Liou, H. C. Lee, C. G. Kim and J. K. Sohng, *FEBS Lett.*, 2004, **566**, 201–206.

166 Q. F. Zhao, Q. L. He, W. Ding, M. C. Tang, Q. J. Kang, Y. Yu, W. Deng, Q. Zhang, J. Fang, G. L. Tang and W. Liu, *Chem. Biol.*, 2008, **15**, 693–705.

167 J. M. Crawford, P. M. Thomas, J. R. Scheerer, A. L. Vagstad, N. L. Kelleher and C. A. Townsend, *Science*, 2008, **320**, 243–246.

168 J. M. Crawford, T. P. Korman, J. W. Labonte, A. L. Vagstad, E. A. Hill, O. Kamari-Bidkorpeh, S.-C. Tsai and C. A. Townsend, *Nature*, 2009, **461**, 1139–1143.

169 Y. G. Luo, S. J. Lin, J. Zhang, H. A. Cooke, S. D. Bruner and B. Shen, *J. Biol. Chem.*, 2008, **283**, 14694–14702.

170 H. A. Cooke, E. L. Guenther, Y. Luo, B. Shen and S. D. Bruner, *Biochemistry*, 2009, **48**, 9590–9598.

171 H. A. Cooke, J. Zhang, M. A. Griffin, K. Nonaka, S. G. Van Lanen, B. Shen and S. D. Bruner, *J. Am. Chem. Soc.*, 2007, **129**, 7728–7729.

172 D. R. Schroeder, K. L. Colson, S. E. Klohr, N. Zein, D. R. Langley, M. S. Lee, J. A. Matson and T. W. Doyle, *J. Am. Chem. Soc.*, 1994, **116**, 9351–9352.

173 K. Komano, S. Shimamura, Y. Norizuki, D. Zhao, C. Kabuto, I. Sato and M. Hirama, *J. Am. Chem. Soc.*, 2009, **131**, 12072–12073.

174 A. Birch, P. Massy-Westropp and c. Moye, *Aust. J. Chem.*, 1955, **8**, 539–544.

175 I. Schmitt, S. Kautz and H. T. Lumbsch, *Mycol. Res.*, 2008, **112**, 289–296.

176 S. Wattalachaisaereekul, A. E. Lantz, M. L. Nielsen, O. S. Andresson and J. Nielsen, *Biotechnol. Bioeng.*, 2007, **97**, 893–900.

177 J. Beck, S. Ripka, A. Siegner, E. Schiltz and E. Schweizer, *Eur. J. Biochem.*, 1990, **192**, 487–498.

178 P. Lu, A. Zhang, L. M. Dennis, A. M. Dahl-Roshak, Y. Q. Xia, B. Arison, Z. An and J. S. Tkacz, *Mol. Genet. Genomics*, 2005, **273**, 207–216.

179 H. Qing-Li, J. Xin-Ying, T. Man-Cheng, T. Zhen-Hua, T. Gong-Li and L. Wen, *ChemBioChem*, 2009, **10**, 813–819.

180 S. J. Hofstead, J. A. Matson, A. R. Malacko and H. Marquardt, *J. Antibiot.*, 1992, **45**, 1250–1254.

181 K. S. Lam, G. A. Hesler, D. R. Gustavson, A. R. Crosswell, J. M. Veitch, S. Forenza and K. Tomita, *J. Antibiot.*, 1991, **44**, 472–478.

182 J. E. Leet, J. Golik, S. J. Hofstead, J. A. Matson, A. Y. Lee and J. Clardy, *Tetrahedron Lett.*, 1992, **33**, 6107–6110.

183 J. E. Leet, D. R. Schroeder, S. J. Hofstead, J. Golik, K. L. Colson, S. Huang, S. E. Klohr, M. S. Lee, J. Golik, S. J. Hofstead, T. W. Doyle and J. A. Matson, *J. Am. Chem. Soc.*, 1992, **114**, 7946–7948.

184 J. E. Leet, D. R. Schroeder, D. R. Langley, K. L. Colson, S. Huang, S. E. Klohr, M. S. Lee, J. Golik, S. J. Hofstead, T. W. Doyle and J. A. Matson, *J. Am. Chem. Soc.*, 1993, **115**, 8432–8443.

185 T. Ando, M. Ishii, T. Kajiura, T. Kameyama, K. Miwa and Y. Sugiura, *Tetrahedron Lett.*, 1998, **39**, 6495–6498.

186 K. C. Nicolaou, Y. F. Tang and J. H. Wang, *Angew. Chem., Int. Ed.*, 2009, **48**, 3449–3453.

187 M. Nett, H. Ikeda and B.S. Moore, *Nat. Prod. Rep.*, 2009, **26**, 1362–1384.

188 J. Marco-Contelles, M. T. Molina and S. Anjum, *Chem. Rev.*, 2004, **104**, 2857–2899.

189 R. P. McGlinchey, M. Nett and B. S. Moore, *J. Am. Chem. Soc.*, 2008, **130**, 2406–2407.

190 M. D. Lee, T. S. Dunne, C. C. Chang, M. M. Siegel, G. O. Morton, G. A. Ellestad, W. J. McGahren and D. B. Borders, *J. Am. Chem. Soc.*, 1992, **114**, 985–997.

191 S. Gaisser, A. Trefzer, S. Stockert, A. Kirschning and A. Bechthold, *J. Bacteriol.*, 1997, **179**, 6271–6278.

192 J. G. McCoy, H. D. Johnson, S. Singh, C. A. Bingman, I.-K. Lei, J. S. Thorson and G. N. Phillips, *Proteins*, 2009, **74**.

193 C. S. Zhang, E. Bitto, R. D. Goff, S. Singh, C. A. Bingman, B. R. Griffith, C. Albermann, G. N. Phillips and J. S. Thorson, *Chem. Biol.*, 2008, **15**, 842–853.

194 S. Walker, K. G. Valentine and D. Kahne, *J. Am. Chem. Soc.*, 1990, **112**, 6428–6429.

195 N. Ikemoto, R. A. Kumar, T. T. Ling, G. A. Ellestad, S. J. Danishefsky and D. J. Patel, *Proc. Natl. Acad. Sci. U. S. A.*, 1995, **92**, 10506–10510.

196 M. Tonetti, L. Sturla, A. Bisso, D. Zanardi, U. Benatti and A. De Flora, *Biochimie*, 1998, **80**, 923–931.

197 D. Simkhada, T. J. Oh, E. M. Kim, J. C. Yoo and J. K. Sohng, *Biotechnol. Lett.*, 2009, **31**, 147–153.

198 R. A. Kumar, N. Ikemoto and D. J. Patel, *J. Mol. Biol.*, 1997, **265**, 187–201.

199 H. D. Johnson and J. S. Thorson, *J. Am. Chem. Soc.*, 2008, **130**, 17662–17663.

200 C. Dong, K. Beis, M. F. Giraud, W. Blankenfeldt, S. Allard, L. L. Major, I. D. Kerr, C. Whitfield and J. H. Naismith, *Biochem. Soc. Trans.*, 2003, **31**, 532–536.

201 G. J. Doyle, K. B. Dubay and J. A. Matson, *J. Antibiot.*, 1989, **42**, 1784–1789.

202 Q. J. Gao, C. S. Zhang, S. Blanchard and J. S. Thorson, *Chem. Biol.*, 2006, **13**, 733–743.

203 Q. Gao, C. Zhang, S. Blanchard and J. S. Thorson, *Chem. Biol.*, 2006, **13**, 733–743.

204 C. E. Melancon, C. J. Thibodeaux and H. w. Liu, *ACS Chem. Biol.*, 2006, **1**, 499–504.

205 C. Olano, C. Gómez, M. Pérez, M. Palomino, A. Pineda-Lucena, R. J. Carbajo, A. F. Braña, C. Méndez and J. A. Salas, *Chem. Biol.*, 2009, **16**, 1031–1044.

206 L. A. McDonald, T. L. Capson, G. Krishnamurthy, W.-D. Ding, G. A. Ellestad, V. S. Bernan, W. M. Maiese, P. Lassota, R. A. Kramer and C. M. Ireland, *J. Am. Chem. Soc.*, 1996, **118**, 10898–10899.

207 N. Oku, S. Matsunaga and N. Fusetani, *J. Am. Chem. Soc.*, 2003, **125**, 2044–2045.

208 H. He, W.-D. Ding, V. S. Bernan, A. D. Richardson, C. M. Ireland, M. Greenstein, G. A. Ellestad and G. T. Carter, *J. Am. Chem. Soc.*, 2001, **123**, 5362–5363.

209 R. Cao, W. Peng, Z. Wang and A. Xu, *Curr. Med. Chem.*, 2007, **14**, 479–500.

210 S. Jain, A. Sinha and D. S. Bhakuni, *Phytochemistry*, 2002, **60**, 853–859.

211 M. Konishi, H. Ohkuma, T. Tsuno, T. Oki, G. D. VanDuyne and J. Clardy, *J. Am. Chem. Soc.*, 1990, **112**, 3715–3716.

212 L. Wang, Y. Hu, Y. Zhang, S. Wang, Z. Cui, Y. Bao, W. Jiang and B. Hong, *BMC Biochem.*, 2009, **9**, 14.

213 J. B. Biggins, K. C. Onwueme and J. S. Thorson, *Science*, 2003, **301**, 1537–1541.

214 S. Singh, M. H. Hager, C. S. Zhang, B. R. Griffith, M. S. Lee, K. Hallenga, J. L. Markley and J. S. Thorson, *ACS Chem. Biol.*, 2006, **1**, 451–460.

215 C. Walsh, *Nature*, 2000, **406**, 775–781.
