
Review | Nuclear Receptor Signaling | The Open Access Journal of the Nuclear Receptor Signaling Atlas

PPARα: energy combustion, hypolipidemia, inflammation and cancer

Sean R. Pyper, Navin Viswakarma, Songtao Yu and Janardan K. Reddy ✉

Corresponding Author: jkreddy@northwestern.edu

Feinberg School of Medicine, Northwestern University, Chicago, Illinois, USA

The peroxisome proliferator-activated receptor α (PPARα, or NR1C1) is a nuclear hormone receptor activated by a structurally diverse array of synthetic chemicals known as peroxisome proliferators. Endogenous activation of PPARα in liver has also been observed in certain gene knockout mouse models of lipid metabolism, implying the existence of enzymes that either generate (synthesize) or degrade endogenous PPARα agonists. For example, substrates involved in fatty acid oxidation can function as PPARα ligands. PPARα serves as a xenobiotic and lipid sensor to regulate energy combustion, hepatic steatosis, lipoprotein synthesis, inflammation and liver cancer. Mainly, PPARα modulates the activities of all three fatty acid oxidation systems, namely mitochondrial and peroxisomal β-oxidation and microsomal ω-oxidation, and thus plays a key role in energy expenditure. Sustained activation of PPARα by either exogenous or endogenous agonists leads to the development of hepatocellular carcinoma resulting from sustained oxidative and possibly endoplasmic reticulum stress and liver cell proliferation. PPARα requires transcription coactivator PPAR-binding protein (PBP)/mediator subunit 1(MED1) for its transcriptional activity.

Received November 11th, 2009; Accepted March 4th, 2010; Published April 16th, 2010 | Abbreviations: ACOX1: peroxisomal fatty acyl-CoA oxidase 1; CBP: cAMP response element-binding protein-binding protein; CYP: cytochrome P450; D-PBE: D-3-hydroxyacyl-CoA dehydratase/D-3-hydroxyacyl-CoA dehydrogenase (D-bifunctional enzyme); DRIP: vitamin D receptor-interacting protein(s); FACS: fatty acyl-CoA synthetase; FAS: fatty acid synthase; L-PBE: enoyl-CoA hydratase/L-3-hydroxyacyl-CoA dehydrogenase (L-bifunctional enzyme); MED1: mediator subunit 1; PBP: PPAR-binding protein; PIMT: PRIP-interacting protein with methyltransferase activity; PPAR: peroxisome proliferator-activated receptor; PPRE: PPAR response element; PRIC: PPARα-interacting coregulator complex; PRIP: PPAR-interacting protein; PTL: peroxisomal thiolase; RXR: retinoid X receptor; SRC-1: steroid receptor coactivator-1; TRAP: thyroid hormone receptor-associated protein | Copyright © 2010, Pyper et al. This is an open-access article distributed under the terms of the Creative Commons Non-Commercial Attribution License, which permits unrestricted non-commercial use distribution and reproduction in any medium, provided the original work is properly cited.

Cite this article: Nuclear Receptor Signaling (2010) 8, e002

## Introduction

Peroxisome proliferator-activated receptor (PPAR) α (NR1C1), β/δ (NR1C2), and γ (NR1C3), are ligand-activated transcription factors that are members of the nuclear-hormone receptor (NR) superfamily [Bookout et al., 2006; Issemann and Green, 1990; Michalik et al., 2006]. The foundation for the discovery and designation of the PPAR subfamily of nuclear receptors in the 1990s [Dreyer et al., 1992; Issemann and Green, 1990], is the cumulative work over the preceding 25 years with peroxisome proliferators (PPs), a group of structurally diverse chemicals that lower serum lipids, and induce massive proliferation of peroxisomes in liver cells, with associated coordinated transcriptional activation of peroxisomal fatty acid β-oxidation system genes [Reddy et al., 1980; Reddy and Krishnakantha, 1975; Reddy et al., 1976; Reddy et al., 1986b]. A receptor for these chemicals was originally postulated in the mid 1980s [Lalwani et al., 1987; Lalwani et al., 1983; Rao and Reddy, 1987; Reddy and Lalwani, 1983] to account for the cell-specific highly reproducible pleiotropic responses [Reddy and Chu, 1996]. Chronic exposure to PPs results in the development of liver cancer in rats and mice, although these agents are nonmutagenic [Rao and Reddy, 1987; Reddy et al., 1980; Reddy et al., 1976]. It was therefore envisaged that a receptor-mediated mechanism is responsible for both the immediate and long-term effects of PPs, including the development of liver cancer [Rao and Reddy, 1987; Reddy and Chu, 1996]. A search for a receptor for these ligands was underway [Lalwani et al., 1987], and this culminated in the cloning of a receptor activated by PPs by Issemann and Green [Issemann and Green, 1990]. This receptor was appropriately designated PP-Activated Receptor or PPAR [Issemann and Green, 1990]. Subsequent identification of two other members of this family, namely PPARβ/δ and PPARγ, resulted in designating the original receptor activated by PPs as PPARα [Dreyer et al., 1992]. Gene knockout studies in mice have unequivocally established that PPARα is indeed responsible for the PP-induced array of pleiotropic responses including liver cancer in rodents [Lee et al., 1995].

All three members of the PPAR subfamily function as sensors for fatty acids and fatty acid derivatives, and thus control important metabolic pathways involved in lipid and energy metabolism [Chawla et al., 2001; Krey et al., 1997]. In the liver, PPARα is activated by synthetic PPs and by both saturated and polyunsaturated fatty acids and their derivatives, and regulates fatty acid oxidation systems to increase energy combustion. PPARα also plays a role in lipoprotein synthesis, inflammatory responses and the development of cancer in the rodent liver [Lee et al., 1995; Mei et al., 2009; Mukherjee et al., 2008]. PPARβ/δ also enhances fatty acid catabolism and energy uncoupling in skeletal muscle and adipose tissue as well as recently shown in the liver where it may participate in the hepatic response to starvation [Sanderson et al., 2009; Wang et al., 2003]. Additionally, PPARβ/δ may play a protective role in the liver by

Review
Pleiotropic responses of PPARα

downregulating inflammatory signals in circumstances of liver damage [Nagasawa et al., 2006; Shan et al., 2008a; Shan et al., 2008b]. Together, PPARα and PPARβ/δ are active participants in energy burning, whereas PPARγ, which has two isoforms, PPARγ1 and PPARγ2, is critical in regulating adipocyte differentiation and energy storage by adipocytes [Kliewer et al., 2001; Tontonoz et al., 1994; Zhu et al., 1995]. PPARγ can also induce lipid accumulation in other cell types, such as hepatocytes, when overexpressed [Yu et al., 2003]. In essence, PPARγ regulates anabolic lipid metabolism, while PPARα and PPARβ/δ function as catabolic regulators of energy [Chawla et al., 1994; Tontonoz et al., 1994; Wang et al., 2003].

The PPARα gene and protein structure

The human PPARα (NR1C1) gene, which spans ~93.2 kb, is located on chromosome 22 at position 22q12-q13.1 and it encodes a protein of 468 amino acids [Sher et al., 1993]. The mouse PPARα gene, located on chromosome 15E2, also encodes a protein of 468 amino acids [Issemann and Green, 1990]. As in the mouse, the human PPARα gene encoded mRNA is derived from 8 exons with a 5' untranslated region encoded by exons 1, 2, and part of exon 3 [Desvergne and Wahli, 1999; Sher et al., 1993]. The remainder of exon 3 and exons 4-8 contribute to the coding region of PPARα [Desvergne and Wahli, 1999; Sher et al., 1993]. The last 232 bp of exon 8 contribute to the 3'-untranslated region [Vohl et al., 2000].

PPARα shares common structural characteristics with the other members of the NR family [Desvergne and Wahli, 1999]. It displays five distinguishable domains designated the A/B, C, D, E and F domains, respectively (Figure 1). The PPARα N-terminal A/B domain contains an activation function-1 (AF-1) region. This region has a low level of basal transcriptional activity and functions independent of ligand-binding [Desvergne and Wahli, 1999]. The specificity of gene transcription is granted by the isoform-specific sequence of the A/B domain of the receptor [Hummasti and Tontonoz, 2006]. Chimeric proteins generated by fusion with the A/B domains of other receptor proteins attenuate the specificity of target gene activation [Hummasti and Tontonoz, 2006]. The A/B domain is followed by a DNA-binding domain (C) encompassing amino acids 101-166 in the human that contains two very highly conserved zinc finger motifs and architectural elements capable of sequence-specific binding to DNA [Desvergne and Wahli, 1999].

Next to the DNA-binding domain is a flexible hinge domain (D), which connects the DNA-binding domain and the ligand-binding domain (E) [Desvergne and Wahli, 1999]. This hinge region binds corepressor proteins, with the characteristic LXXXIXXXL repressor motif, to the receptor in its quiescent, unliganded state [Dowell et al., 1999]. The corepressor-binding site partly overlaps with the coactivator-binding site. The ligand binding domain in the human PPARα protein extends from amino acids 280-468 and contains an activation function-2 (AF-2) region composed of two α-helices flanking one four-stranded β-sheet [Desvergne and Wahli, 1999]. The AF-2 domain

is repressed until ligand-binding occurs. Following ligand-binding, the AF-2 domain undergoes a conformational shift, which allows the formation of hydrogen bonds between Tyr-314 and Tyr-464, as well as the formation of a charge clamp between Glu-462 and Lys-292 [Xu et al., 2001]. This conformational change in the protein allows interaction of the receptor with the LXXLL (L, leucine; X, any amino acid) motifs located in coactivator proteins [Xu et al., 2001]. PPARα, like other nuclear receptors, will undergo conformational adjustment upon binding to a ligand to achieve the coregulator exchanges and activation of the target genes [Xu et al., 2001].

The PPAR ligand binding cavity, with a total volume of 1300 to 1400 Å³, is larger than that found in other NRs [Desvergne and Wahli, 1999; Xu et al., 2001]. It is defined by 34 amino acid residues and 80% of these residues are highly conserved across the three PPAR subtypes [Dreyer et al., 1992]. PPARα and PPARγ ligand binding cavities are similar in size, but the PPARβ/δ pocket is significantly narrower in the region adjacent to the AF-2 helix and does not readily accommodate bulky polar heads [Cronet et al., 2001]. Similarity between PPARα and PPARγ ligand binding pockets may account for some degree of flexibility in ligand-receptor interaction between these two isoforms and their ability to bind a wide range of synthetic and natural lipophilic compounds [Xu et al., 2001]. Comparison of the crystal structures of the ligand binding domains showed that a single amino acid, tyrosine 314 in PPARα and histidine 323 in PPARγ, accounts for the subtype selectivity for classes of ligands [Xu et al., 2001]. The PPARα pocket is more lipophilic than that of PPARγ and this may account for the greater affinity of PPARα to bind the more lipophilic-saturated fatty acids [Xu et al., 2001].

Expression of PPARα

PPARα belongs to the type II nuclear receptor family and like the other two PPAR isoforms, is localized in the nucleus. This differs from type I nuclear receptors, which are cytosolic and translocate to the nucleus upon ligand-binding. High levels of PPARα expression are found in tissues with active fatty acid catabolism, such as liver, heart, kidney, brown adipose tissue, muscle, small intestine and the large intestine [Bookout et al., 2006; Braissant et al., 1996; Issemann and Green, 1990]. Similar tissue expression profiles of PPARα have been found in rodents and human [Bookout et al., 2006]. PPARα exerts a dominant role in fatty acid catabolism and ketone body synthesis in the liver [Kersten et al., 1999]. PPARβ/δ also participates in energy catabolism, but its effects are more prominent in muscle and heart, though it also regulates some genes in the liver [Sanderson et al., 2009]. In contrast, PPARγ is expressed abundantly in white adipose tissue, where it is involved in lipid storage and in brown adipose tissue, where it participates in energy dissipation [Tontonoz et al., 1994].

While PPARα regulates many target genes, the expression of the PPARα gene itself is under the control of other transcription factors [Pineda Torra et al., 2002].

Review
Pleiotropic responses of PPARα

Domain Structure

| 1 | A/B | 101 | C | 166 | D | 244 | E/F | 468 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| AF1 | DBD | Hinge | LBD/AF2 |

Phosphorylation and Cofactor binding sites

| 6 | 12 | 21 | 179 | 230 |
| --- | --- | --- | --- | --- |
| NCoR | SMRT | CBP/p300 | SRC-1 | SRC-3 |
| PBP | PRIP | PRIC285 | PRIC295 | PRIC320 |
| PGC-1α | PGC-1β |

✨ Phosphorylated Serine or Threonine
🔴 Corepressor binding site
🔵 Coactivator binding site

Figure 1. Schematic view of the PPARα structure. Domain structure of the PPARα protein (A) with the ligand-independent activation function 1 (AF1) domain or A/B domain shown in blue, the DNA-binding domain (DBD) or C domain shown in yellow, the hinge or D domain shown in orange, and the ligand-binding domain (LBD) or E domain together with the activation function 2 (AF2) or F domain shown in green. (B) Schematic representation of the PPARα protein with phosphorylation sites labeled with yellow stars, the corepressor site labeled with a red circle, and the coactivator binding site shown with a blue hexagon.

In rodents, PPARα is regulated by various physiological conditions such as stress, hormones (including growth hormone), glucocorticoids, insulin, and leptin [Inoue et al., 2008; Lee et al., 2002]. PPARα expression is also reported to be related to aging [Poynter and Daynes, 1998], and is induced during brown adipocyte differentiation [Kelly et al., 1998]. Expression levels of PPARα in mouse liver are increased during starvation and this leads to enhanced expression of PPARα target genes in liver to catabolize influxed fatty acids [Kersten et al., 1999]. Human PPARα is also regulated at the transcriptional level by nuclear receptors, such as hepatocyte nuclear factor 4 (HNF4), and chicken ovalbumin upstream promoter-transcription factor II (COUP-TFII) [Pineda Torra et al., 2002]. HNF4 positively regulates PPARα expression via a direct repeat 1 (DR1) element, which is composed of the consensus sequence AGG(A/T)CA separated with a single nucleotide spacing between two repeats [Pineda Torra et al., 2002]. On the other hand, the DR1 element in the human PPARα promoter is antagonized by the orphan receptor COUP-TFII [Pineda Torra et al., 2002]. Furthermore, PPARα appears to modulate its own expression [Corton et al., 2000]. In human liver, PPARα mRNA levels may vary significantly among individuals and are considered lower than the levels noted in rodent livers, but this may in part be related to variations in liver sample procurement for the frozen archival liver bank [Palmer et al., 1998; Walgren et al., 2000]. PPARα transcript levels are also induced during macrophage differentiation by high glucose levels [Rigamonti et al., 2008]. PPARα is also regulated by the ubiquitin proteasomal degradation system [Genini et al., 2008].

### Posttranslational modifications

A truncated PPARα isoform resulting from a posttranslational exon skipping mechanism has been detected in human, swine, and jerboas [El Kebbai et al., 2009; Gervois et al., 1999]. This truncated form harbors dominant negative transcriptional repressive activity on PPARα. In addition, PPARα activity is regulated by posttranslational modifications such as phosphorylation (Figure 1). PPARα was first shown to be phosphorylated in primary rat adipocytes in culture [Shalev et al., 1996]. Insulin treatment increases PPARα phosphorylation and nearly doubles its transcriptional activity in CV-1 and HepG2 cells in culture [Shalev et al., 1996]. This insulin-mediated phosphorylation of PPARα involves amino acids Ser12 and Ser21 [Juge-Aubry et al., 1999]. Stress stimuli exert an increase in PPARα

Review

phosphorylation in rat neonatal cardiac myocytes by the p38 mitogen-activated protein kinase (MAPK) pathway [Barger et al., 2001]. p38 MAPK phosphorylates Ser 6, 12 and 21 residues located within the N-terminal A/B domain of the protein and this phosphorylation significantly enhanced ligand-dependent transactivation of PPARα [Barger et al., 2001]. PPARα is also reported to be phosphorylated by protein kinase C (PKC). Inhibition of PKC activity impairs ligand-activated PPARα transactivation activity, but enhances PPARα transrepression activity. Purified PPARα protein is phosphorylated *in vitro* by recombinant PKCα and -βII. These data demonstrate that the PKC signaling pathway may act as a molecular switch for the transactivation and transrepression properties of PPARα, which involve phosphorylation of Ser179 and Ser230 [Blanquart et al., 2004]. This PKC phosphorylation of PPARα is believed to play an important role in statin-mediated acute anti-inflammatory effects, though the importance of phosphorylation in PPARα signaling has never been clearly demonstrated *in vivo* [Paumelle et al., 2006].

PPARα ligands

PPARα ligands fall into two general categories: synthetic xenobiotics (exogenous) and biological molecules (endogenous). Synthetic ligands, referred to as PPs, include hypolipidemic drugs such as clofibrate, fenofibrate, gemfibrozil, bezafibrate, ciprofibrate, nafenopin, methyl clofenapate, tibric acid, and Wy-14,643 (pirinixic acid) used in the treatment of dyslipidemias, and industrial phthalate-monoester plasticizers, such as di-(2-ethylhexyl)-phthalate (DEHP), and di-(2-ethylhexyl) adipate (DEHA) used in the manufacture of polyvinyl chloride plastics [Lalwani et al., 1983; Maloney and Waxman, 1999]. Certain herbicides, pesticides, industrial solvents, food flavoring agents, and leukotriene D4 receptor antagonists also function as PPARα ligands [Maloney and Waxman, 1999]. These structurally-diverse chemicals induce highly reproducible and similar pleiotropic responses in the rat and mouse liver. Proliferation of peroxisomes in liver parenchymal cells and profound transcriptional activation of fatty acid oxidation system genes are the hallmarks of PP-induced pleiotropic responses in the rat and mouse liver.

Endogenous or naturally-occurring biological molecules that serve as PPARα ligands include fatty acids and fatty acid derivatives (Figure 2) [Devchand et al., 1996; Hostetler et al., 2005]. Endogenous PPARα activators, in most part, have been identified using *in vitro* approaches [Forman et al., 1997; Gottlicher et al., 1992; Hostetler et al., 2005; Krey et al., 1997]. Both dietary saturated and unsaturated fatty acids function as direct ligands for PPARα [Hostetler et al., 2005]. Alternately, the ultimate ligands for this receptor may be generated during fatty acid catabolism and new fatty acid synthesis *in vivo* [Chakravarthy et al., 2009] (Figure 2). Several enzymes such as the 8-, 12-, 15- and 5-lipoxygenases, the cyclooxygenases and cytochrome P450s utilize fatty acids as substrates to produce putative PPARα ligands [Brash, 1999; Crisafulli and Cuzzocrea, 2009; Ng et al.,

Pleiotropic responses of PPARα

2007]. Although many synthetic ligands activate PPARα efficiently, it appears that PPARα has evolved primarily for lipid sensing to regulate energy combustion [Reddy, 2004; Reddy and Rao, 2006].

Evidence suggests that PPARα senses certain endogenous lipid metabolic intermediates as ligands and participates in their metabolism (degradation) by inducing downstream lipid metabolism-related genes [Fan et al., 1998]. The first *in vivo* evidence came from studies with fatty acyl-CoA oxidase 1 (ACOX1) gene disrupted mice, in that mice deficient in this first and rate-limiting enzyme of the fatty acid β-oxidation system reveal profound activation of PPARα in liver [Fan et al., 1996; Fan et al., 1998; Yeldandi et al., 2000]. Animals deficient for the ACOX1 have high levels of the very long chain fatty acids and since very-long chain fatty acyl-CoAs are incapable of entering the fatty acid oxidation pathway due to ACOX1 deficiency, these unmetabolized substrates act as ligands to hyperactivate PPARα [Fan et al., 1998; Yeldandi et al., 2000]. Recent studies with other gene knockout mouse models suggest that enzymes such as ACOX1, enoyl-CoA hydratase/L-3-hydroxyacyl-CoA dehydrogenase (L-PBE/MFP), D-3-hydroxyacyl-CoA dehydratase/D-3-hydroxyacyl-CoA dehydrogenase (D-PBE/MFP2) and sterol carrier protein x (SCPx) are necessary for the degradative metabolism of endogenously-generated PPARα ligands [Ellinghaus et al., 1999; Fan et al., 1998; Jia et al., 2003; Martens et al., 2008; Qi et al., 1999a; Seedorf et al., 1998], while other enzymes such as fatty-acid synthase (FAS), fatty acyl-CoA synthetase (FACS) and certain lipoxygenases (LO), are required for PPARα ligand generation [Chakravarthy et al., 2009; Martin et al., 2000; Yu and Reddy, 2007]. It is implied that substrates of ACOX1, such as very long-chain fatty acyl-CoAs, when left unmetabolized in the absence of this enzyme, function as potent endogenous ligands of PPARα [Fan et al., 1998; Yeldandi et al., 2000]. It is known that long-chain acyl-CoA esters can act as regulatory molecules *in vivo* [Faergeman and Knudsen, 1997; Fan et al., 1996; Fan et al., 1998; Hostetler et al., 2005]. PPARα exhibits high affinity for the CoA thioesters of the common (C20-C24) very long chain fatty acids and these CoA thioesters of very-long chain and branched-chain fatty acids are more potent PPARα ligands than the free fatty acids [Hostetler et al., 2005]. Further evidence that fatty acids are not the direct *in vivo* ligands for PPARα comes from mice with disrupted fatty acyl-CoA synthetase gene [Li et al., 2009]. Fatty acyl-CoA synthetases activate long-chain fatty acids to acyl-CoAs, thereby providing substrates for downstream fatty acid β-oxidation pathways [Li et al., 2009]. Mice with a disrupted fatty acyl-CoA synthetase 1 gene [Li et al., 2009], or deficient for the adrenoleukodystrophy protein gene (X-ALD) [Heinzer et al., 2003; Reddy and Hashimoto, 2001], fail to convert fatty acids to acyl-CoAs to serve as substrates for fatty acid β-oxidation vis à vis ligands for PPARα [Heinzer et al., 2003; Li et al., 2009]. If long-chain fatty acids function as PPARα ligands *in vivo*, one would have expected increased PPARα target gene activation in X-ALD and fatty acyl-CoA synthetase null mice, but that does not appear to be the case in these

Review                                                                                       Pleiotropic responses of PPARα

![Diagram](#)

Figure 2. Biological ligands of PPARα. Diagram illustrating different known biological ligands of PPARα. Metabolic sources for ligands are grouped in the boxes together with ligand generating enzymes marked with an asterisk, ligands marked with a square and ligand degrading enzymes marked with a triangle. The PPARα and RXRα heterodimer are shown bound to a PPRE sequence in the promoter of a target gene with associated coactivator proteins forming a complex with the cellular transcription machinery.

null livers [Heinzer et al., 2003; Li et al., 2009]. Failure to induce peroxisome proliferation in liver by dietary lipid overload also indicates that fatty acids per se are not effective PPARα ligands [Fan et al., 1998].

Experiments with liver-specific inactivation of fatty-acid synthase (FAS) suggest that products of FAS serve as endogenous activators of PPARα in adult liver [Chakravarthy et al., 2007]. FAS-deficient mice reveal a phenotype resembling PPARα deficiency [Chakravarthy et al., 2007]. Recently, it has been demonstrated that FAS generates 1-palmityl-2-oleoyl-sn-glycerol-3-phosphocholine (16:0/18:1-GPC), which appears to be a physiologically relevant PPARα ligand [Chakravarthy et al., 2009].

Leukotriene B4 (LTB4) is another well known putative PPARα ligand, which connects the receptor to inflammation and immune responses. 19- and 20-hydroxyeicosatetraenoic acid (19- and 20-HETE) are the major products of cytochrome P450 catalyzed conversion of arachidonic acid and play roles in the cardiovascular system, kidney function, cellular proliferation and inflammation. HETEs bind to and activate PPARα in HepG2 cells transfected with human PPARα, but do not show any transactivation activity when the cells are transfected with the unrelated nuclear receptor TRβ [Ng et al., 2007]. Oleylethanolamide (OEA), a

naturally-occurring lipid has been identified as a high affinity PPARα ligand regulating PPARα activity and lipid metabolism. Finding a way of regulating PPARα activity through modulating endogenous ligand, and therefore achieving control of energy metabolism and body weight, may have the potential to generate new therapies.

A high degree of interest presently exists in discovering new partial agonists of PPARα and of other PPAR isoforms that are selective for beneficial effects of PPAR activation and avoid problems with dose-related toxicity and other negative side effects of PPAR activation [Chang et al., 2007]. Direct glucose binding with the receptor has also been shown to occur and may play an important regulatory role in PPARα activity, particularly in the setting of diabetes [Hostetler et al., 2008].

PPARα agonists, including fibrates such as fenofibrate, gemfibrozil and less commonly clofibrate, have been in use for several decades as therapeutic agents to decrease plasma levels of triglycerides and VLDL by enhancing lipid uptake and catabolism and to increase plasma levels of HDL-C [Bays and Stein, 2003; Lefebvre et al., 2006; Marx et al., 2004]. The fibrate class of PPARα ligands increase the production in the liver of the apolipoproteins A-I and A-II, which are major components of HDL-C [Bays and Stein, 2003]. The search for agents that display potent and selective binding to the PPARα

Review

isotype that support robust recruitment of transcription coactivators is expected to result in the identification of new classes of ligands [Kane et al., 2009]. These novel ligands, including LY518674, a novel PPARα ligand recently reported by Nissen et al., have the traditional benefits of reducing serum triglycerides and increasing serum HDL-C, but may confer additional benefits such as reduction of serum LDL-C, especially when paired with other treatments. Novel therapeutics such as this may hold the key to better treatments for lipid disorders such as atherogenic dyslipidemia, which lead to other metabolic and cardiac problems [Nissen et al., 2007]. Other current therapeutic benefits of potential PPARα agonists are under investigation for use in the prevention of atherosclerosis, cardiomyopathies, ischemia-reperfusion injuries, prevention of angiogenesis, anti-proliferation of tumorigenic cells and others [Bulhak et al., 2009; Crisafulli and Cuzzocrea, 2009; Gizard et al., 2008; Massip-Salcedo et al., 2008; Panigrahay et al., 2008; Patel et al., 2009; Ramanan et al., 2008; Staels et al., 2008; Wang et al., 2009].

Other PPARα ligands with environmental impact, in particular phthalate ester plasticizers widely used in the manufacture of polyvinyl chloride plastics, and industrial solvents such as trichloroethylene have been in use for over 50 years. These are known contaminants of ground water and estuaries, but the impact on human health is unclear. More recently, certain herbicides have been shown to function as PPARα ligands, implying that these agents also constitute a major category of environmental contaminants that can activate this receptor, and thus raising issues about long-term or lifelong exposure.

PPARα heterodimerization with RXR, binding to DNA elements and PPARα-regulated genes

PPARα, like many non-steroid members of the NR family, functions as an obligate heterodimer with another nuclear receptor, retinoid X receptor (RXR; NR2B) [Kliwer et al., 1992]. The PPARα/RXR heterodimers can form independent of the PPARα ligand and these unliganded heterodimers recruit corepressor protein complex and inhibit target gene transcription [DiRenzo et al., 1997]. Upon ligand binding, the corepressor complex is released from the PPARα/RXR heterodimer, and the coactivator complexes will be recruited to the promoter region of target genes to initiate transcription [Dowell et al., 1999]. The PPARα/RXR heterodimers bind to a specific DNA sequence element called a peroxisome proliferator response element (PPRE), which was initially characterized using the PPAR responsive promoter of the rat Acox1 gene [Chandra et al., 2008; Kliewer et al., 1992; Osumi et al., 1991; Tugwood et al., 1992; van der Meer et al., 2010]. This PPRE element is located in the promoter region of target genes [Tugwood et al., 1992; van der Meer et al., 2010; Zhang et al., 1992]. The PPRE is of the direct repeat 1 (DR1) category, which is composed of two copies of the consensus hexamer sequence AGG(A/T)CA separated by a single base pair (Figure 2). Neither PPARα homodimers nor monomers

Pleiotropic responses of PPARα

bind to the PPRE. PPARα target genes usually contain one or more PPRE in the promoter and preferentially bind PPARα/RXR heterodimers [Chu et al., 1995; Tugwood et al., 1992]. Occasionally, PPRE may be in the proximal transcribed regions of certain target genes [Di-Poi et al., 2002]. The PPRE element contains an additional AACT motif positioned 5′ to the DR-1 [Juge-Aubry et al., 1997]. This upstream extended hexamer of the DR1 interacts with PPARs, whereas the downstream hexamer is known to interact with RXR of the PPAR/RXR heterodimer [Desvergne and Wahli, 1999; Juge-Aubry et al., 1999]. Of the three PPARs, PPARγ binds more strongly than do PPARα and PPARβ/δ, implying that conservation of the 5′-flanking extension is more important for the binding of PPARα and PPARβ/δ [Juge-Aubry et al., 1997]. The nature of this upstream extension may confer isotype specificity [IJpenberg et al., 1997].

The presence of PPRE upstream of the fatty acyl-CoA oxidase 1 (ACOX1) gene [Tugwood et al., 1992], and peroxisomal enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase (L-PBE) gene [Zhang et al., 1992] were the first described. Interestingly, these were the first two genes identified to be transcriptionally activated by PPs [Reddy et al., 1986a; Reddy and Hashimoto, 2001; Tugwood et al., 1992; Zhang et al., 1992]. Over the years, many other genes that possess PPREs have been characterized as PPARα target genes (Supplementary File 1 and Supplementary File 2). cDNA microarray profiling and comparative proteomic data have revealed several known PPARα target genes and some new genes that are upregulated in liver by the PPARα ligand, Wy-14,643 [Cherkaoui-Malki et al., 2001; Chu et al., 2004; Kane et al., 2009; Nielsen et al., 2006]. These fall into several functional categories including lipid metabolism, xenobiotic metabolism, inflammation and others.

RXR, the heterodimerization partner of PPARs, is also a partner of many other NRs [Bugge et al., 1992; Juge-Aubry et al., 1995; Mangelsdorf et al., 1995]. This promiscuous centrality of RXR in NR functioning, raises the possibility of competition between NRs for this common binding partner [Juge-Aubry et al., 1995]. Reciprocal negative interaction between the PPARα and thyroid hormone receptor signaling pathways is indirect, in that it is mostly due to sequestration of RXR, the common heterodimeric partner [Chu et al., 1995; Juge-Aubry et al., 1995]. A similar competition can also occur among PPAR isotypes with RXR in different cell types, but the *in vivo* effects of this competition and limiting amounts of RXR protein remain unclear [Jow and Mukherjee, 1995].

PPARα knockout phenotype

PPARα knockout mice were generated by the Gonzalez group [Lee et al., 1995]. These mice have demonstrated unequivocally that PPARα is indeed the bona fide receptor responsible for mediating the pleiotropic responses induced by synthetic peroxisome proliferators [Lee et al., 1995]. In this knockout mouse model, the ligand binding domain was disrupted by homologous recombination. Mice homozygous for the mutation did

Review

not express PPARα protein and yet are viable and fertile, with no detectable gross phenotypic defects [Lee et al., 1995]. PPARα null mice display normal basal levels of the inducible peroxisomal β-oxidation enzymes in liver, but generally lower levels of mitochondrial fatty acid metabolizing enzymes [Aoyama et al., 1998]. PPARα knockout mice do not reveal PP-induced pleiotropic responses when challenged with the classical peroxisome proliferators, clofibrate and Wy-14,643 [Lee et al., 1995]. Subsequently, it was shown that PPARα knockout mice fail to develop liver tumors when chronically exposed to these PPARα ligands, indicating that PP-induced hepatocarcinogenesis is receptor mediated, as previously proposed [Gonzalez and Shah, 2008; Hays et al., 2005; Peters et al., 1997; Rao and Reddy, 1987]. In essence, these observations clearly established that PPARα is the central player in mediating the pleiotropic responses of peroxisome proliferators [Lee et al., 1995]. Numerous studies with PPARα null mice during the past 15 years have demonstrated the critical roles played by this receptor in energy metabolism, hepatic steatosis, inflammation, cardiac pathophysiology, cell cycle alterations and hepatocarcinogenesis [Lefebvre et al., 2006].

### PPARα repressors and coactivators

Transcriptional activation of genes is a complex process, which involves the participation of many transcription coregulators [Glass and Rosenfeld, 2000]. PPARs, like other nuclear receptors, interact with coactivators such as SRC-1 (steroid receptor coactivator-1) [Onate et al., 1995; Zhu et al., 1996], or corepressors such as N-CoR (nuclear corepressor) [Chen and Evans, 1995; Dowell et al., 1999], and SMRT (silencing mediator for retinoid and thyroid hormone receptors) [Chen and Evans, 1995; Dowell et al., 1999; Hu and Lazar, 1999]. PPARα-interacting coactivators and corepressors augment or repress, respectively, the PPARα transactivation activity [Feige et al., 2006; Yu and Reddy, 2007; Zoete et al., 2007]. Since the cloning of SRC-1 [Onate et al., 1995] 15 years ago, over 300 coactivators/coregulators have been identified, with new members still being added to this expanding spectrum [Lonard and O'Malley, 2006]. PPARα is known to interact with some of these coregulators (Table 1). These include CBP/p300, three members of the SRC/p160 family, PBP/MED1 (PPAR-binding protein/TRAP220/DRIP205/mediator subunit 1), PRIP/NCoA6 (ASC2/RAP250/TRBP/NRC) [Zhu et al., 2000a], PRIC285, PRIC295, PRIC320, PGC-1α, and PGC-1β, as well as coactivator-associated proteins PIMT (NCoA6IP) and CARM-1 (Table 1). The PPARα-interacting coregulator (PRIC) complex isolated from rat liver nuclear extracts reveals the presence of many of these coregulators, presumably forming one mega complex [Surapureddi et al., 2002]. This diversity raises several issues about the evolutionary importance of the versatility and complexity of coregulatory molecules, their relative abundance in various cell types, and their affinity for a given nuclear receptor in orchestrating

---

Pleiotropic responses of PPARα

transcription in gene-, cell-, and developmental stage-specific transcription.

In the absence of a specific ligand, PPARα interacts with corepressors NCoR and SMRT, but the importance of this interaction for PPARα action is not well understood [Feige et al., 2006]. Homozygous deletion of NCoR or SMRT in mice is embryonic lethal, indicating that they cannot fully compensate for each other during development [Ghisletti et al., 2009; Jepsen et al., 2008; Jepsen et al., 2000]. Furthermore, another corepressor, the Receptor Interacting Protein 140 (RIP140), which can interact with PPARα, is known to repress the activity of NRs by competing with coactivators and by recruiting downstream effectors such as histone deacetylases (HDACs) [White et al., 2004]. Interestingly, the phenotype of RIP140 knockout mice suggests a role for this corepressor in PPARα signaling, as these mice exhibit resistance to high-fat diet-induced obesity resulting from upregulation of genes involved in energy dissipation [Leonardsson et al., 2004].

The binding of ligand to a nuclear receptor influences first, the release of corepressors and beginning recruitment of coactivator complexes, such as members of the SRC-1/p160 family, which exhibit histone acetyltransferase activity required to facilitate chromatin remodeling. Subsequent docking of other coregulators, either singly, or as preassembled multisubunit protein complexes, including mediator complex (MED), facilitates interaction of ligand-activated receptor with RNA polymerase and the general basal transcription machinery to enhance the transcription of a specific set of genes [Dotson et al., 2000; Glass and Rosenfeld, 2000; Shang et al., 2000]. As indicated above, coactivators contain an LXXLL motif that forms two turns of α-helix and binds to a hydrophobic cleft on the surface of the nuclear receptor [Voegel et al., 1998]. Data on the identification and characterization of coactivators have been derived mostly from *in vitro* observations, but there is a paucity of information about the *in vivo* cell- and gene-specific functional roles of individual coactivators [Yu and Reddy, 2007].

### Coactivator gene-knockout mice: PPARα signaling requires PBP/MED1

Gene knockout mouse models have established that some of the coactivators are essential for embryonic growth and survival, while others are not critical for mouse embryogenesis [Reddy et al., 2006; Yu and Reddy, 2007]. For example, disruption of the PBP/MED1 gene results in embryonic lethality in the mouse around gestational day 11.5 [Crawford et al., 2002; Ito et al., 2000; Landles et al., 2003; Zhu et al., 2000b]. Likewise, germ-line deletion of the PRIP/ASC2/NRC/RAP250/NCoA6 gene [Antonson et al., 2003; Qi et al., 2004], and the PRIP-interacting protein PIMT/TGS1 (NCoA6IP) gene in the mouse also result in embryonic lethality [Jia et al., 2009b; Monecke et al., 2009]. In contrast, null mutation of coactivators SRC-1 [Qi et al., 1999b], SRC-2 [Chopra et al., 2008], and SRC-3, PGC-1, PRIC285 [Viswakarma

Review

| Coactivators | Enzyme activity | Function | References |
|--------------|-----------------|----------|------------|
| CARM-1       | MT              | Histone methyltransferase | Chen et al., 1999 |
| CBP/p300     | HAT             | Histone acetylation; recruit p160/SRCs | Kwok et al., 1994 |
| SRC-1        | HAT             | Chromatin acetylation | Onate et al., 1995 |
| SRC-3        | HAT             | Chromatin acetylation | Suen et al., 1998 |
| PBP          | None            | Anchor TRAP complex | Zhu et al., 1997 |
| PIMT/NCoA6IP | MT              | RNA Methyl Transferase | Zhu et al., 2001 |
| PRIP         | None            | Recruit ASC complex | Zhu et al., 2000 |
| PRIC 285     | Helicase        | Chromatin remodeling | Surapureddi et al., 2002 |
| PRIC 320     | Helicase        | Chromatin remodeling | Surapureddi et al., 2006 |
| PGC-1α       | None            | Recruit HAT activities | Puigserver et al., 1998 |
| PGC-1β       | None            | Recruit HAT activities | Lin et al., 2002 |

Table 1. Some known PPARα coactivator proteins. Coactivator proteins known to interact with activated PPARα, with their enzymatic activities and functions. Acronyms used: (MT) Methyltransferase, (HAT) Histone acetyltransferase. References identifying each protein as a coregulator of PPARα are provided in the reference section.

et al., 2009], and CARM1 result in a viable phenotype with or without overt functional changes.

Studies on the role of germ-line deletion of coactivators SRC-1, SRC-2, and SRC-3 in the mouse revealed that none of these are essential for PPARα ligand-induced pleiotropic responses in liver ([Qi et al., 1999b] and Jia, Y., Xu, J., and Reddy, J.K., unpublished observations). These p160 family members are among the coactivators considered important for nuclear receptor function, but the in vivo results indicate their redundancy at least for PPARα function in liver [Qi et al., 1999b]. It is likely that other coactivators with histone acetyltransferase activity may be adequate to facilitate chromatin remodeling in the absence of SRC-1 and other members of the p160 family [Qi et al., 1999b]. Further studies are needed to ascertain the effects of loss of two or more members of this p160 family in PPARα function. Combined loss of p/CIP/SRC-3 and SRC-1 function has been shown to lead to a specific block in brown fat development, in part due to a failure of induction of PPARγ target genes [Wang et al., 2006]. Loss of both p/CIP/SRC-3 and SRC-1 also leads to increased food intake on both regular chow and a high-fat diet, indicating excess energy burning [Wang et al., 2006]. Whether loss of these two coactivators results in strongly increased energy expenditure by regulating PPARα- [Reddy and Hashimoto, 2001; Reddy and Rao, 2006] and PPARβ/δ-dependent transcriptional activities [Wang et al., 2003] remains to be investigated.

To study the cell- and gene-specific functions of coactivator genes such as PBP/MED1 and PRIP/NCoA6, which when germ-line disrupted, lead to embryonic lethality in the mouse, it becomes necessary to generate conditional null mice using the Cre-loxP strategy [Jia et al., 2005; Jia et al., 2004; Jia et al., 2009a; Matsumoto et al., 2007; Sarkar et al., 2007]. Conditional deletion of PBP/MED1 gene in liver results in the abrogation of PPARα ligand-induced pleiotropic effects, indicating that PBP/MED1 is essential for PPARα signaling [Jia et al., 2005; Matsumoto et al., 2007]. When fed a diet containing the PPARα ligands, Wy-14, 643 or ciprofibrate, PBP/MED1 liver null mice revealed no peroxisome proliferation in PBP/MED1-deficient hepatocytes [Jia et al., 2004]. The presence of an occasional PBP/MED1-positive hepatocyte in these livers that escaped Cre-excision provided a characteristic visual contrast, in that such cells displayed the expected response to PPARα ligands, whereas the response of PBP/MED1-deficient hepatocytes was similar to that observed in cells that lacked PPARα [Jia et al., 2004]. PPARα ligand-induced liver cell proliferation was abolished in PBP/MED1 null hepatocytes and in these livers, residual PBP/MED1-expressing cells displayed a proliferative advantage [Jia et al., 2004; Matsumoto et al., 2007]. Furthermore, PBP/MED1 is essential for partial hepatectomy-related liver regeneration and PPARα ligand Wy-14, 643-induced receptor-mediated hepatocarcinogenesis [Matsumoto et al., 2007]. PBP/MED1-deficient hepatocytes also fail to give rise to

Review

liver tumors following the administration of the genotoxic carcinogen diethylnitrosamine [Matsumoto et al., 2010].

Deletion of the PBP/MED1 gene in liver also abolished the responses induced by constitutive androstane receptor (CAR) activators, phenobarbital or 1,4-bis[2-(3,5-dichloropyridyloxy)]benzene (TCPOBOP) and of acetaminophen-induced hepatotoxicity [Jia et al., 2005; Jia et al., 2009a]. In contrast, liver conditional knockout of the PRIP/NCoA6 gene had no effect on PPARα and CAR signaling in liver [Sarkar et al., 2007]. Mice with floxed PBP/MED1 [Jia et al., 2005; Jia et al., 2009a; Matsumoto et al., 2007], PRIP/NCoA6 [Antonson et al., 2003; Qi et al., 2004], PIMT/TGS1/NCoA6IP [Jia et al., 2009b], PRIC285 [Viswakarma et al., 2009]; and PRIC320 (Viswakarma, N. and Reddy, J.K., unpublished observation) alleles should serve as valuable tools for conditional gene disruption in cells of interest to further evaluate the functional roles of PPARα.

PPARα functions

PPARα and energy burning

Obesity, resulting from increased consumption of energy-rich foods high in saturated fats and sugars, and reduced physical activity, has reached epidemic proportions globally, with more than 1 billion adults overweight [WHO, 2006]. When consumption of energy far exceeds the combustion of calories, there is increased deposition of long-chain fatty acids, principally in the form of triglycerides in adipose and other tissues leading to obesity [Evans et al., 2004]. All three members of the PPAR subfamily of nuclear receptors have been shown to participate in energy metabolism in that PPARα and PPARβ/δ function mostly as catabolic regulators of energy expenditure, while PPARγ regulates anabolic metabolism in that it plays a role in energy storage [Chawla et al., 2001; Chawla et al., 1994; Cook et al., 2000; Evans et al., 2004; Krey et al., 1997; Tontonoz et al., 1994].

The liver is a central player in the whole body energy homeostasis by its ability to orchestrate fatty acid and glucose metabolism. Hepatic lipid metabolism principally involves three aspects: lipogenesis, oxidation of fatty acids and secretion of lipids. Lipogenesis consists of de novo fatty acid synthesis and subsequent conversion of these fatty acids into triglycerides. In the liver, lipogenesis is regulated by transcription factors, sterol regulatory element-binding protein (SREBP-1c), carbohydrate response-element binding protein (ChREBP) and PPARγ [Browning and Horton, 2004; Horton et al., 2002; Wang et al., 2003; Yu et al., 2003]. SREBP-1c regulates the expression of constellations of glycolytic and lipogenic genes, including stearoyl CoA desaturase (Scd-1) and fatty acid synthase (Fas) [Hebbachi et al., 2008; Horton et al., 2002; Reddy and Rao, 2006]. Emerging evidence suggests that PPARα also influences lipogenesis by increasing the transcription of Scd-1 and other lipogenic genes by regulating the primary transcription factors SREBP-1c and liver X receptor α (LXRα) [Hebbachi et al., 2008]. It is suggested that PPARα participates in the generation of an endogenous LXRα ligand, since a

Pleiotropic responses of PPARα

synthetic, non-steroidal ligand of LXRα has been found effective in the induction of lipogenic genes in mice deficient in PPARα [Hebbachi et al., 2008]. However, the role of PPARα in lipogenesis, albeit modest, is intriguing, as it appears paradoxical to the well-known role of this transcription factor in the regulation of fatty acid oxidation [Reddy and Hashimoto, 2001]. The involvement of PPARα in lipogenesis may suggest a fail-safe compensatory mechanism for the removal of important fatty acids, for example, during starvation [Hebbachi et al., 2008].

Liver plays an important role in fatty acid oxidation and this catabolic energy burning is regulated by PPARα [Reddy and Hashimoto, 2001; Reddy and Rao, 2006]. Oxidation of fatty acids occurs in three subcellular organelles, with the bulk of β-oxidation confined to mitochondria and peroxisomes, while CYP4A-catalyzed ω-oxidation takes place in the endoplasmic reticulum [Rao and Reddy, 2004; Reddy and Hashimoto, 2001]. Some of the key enzymes involved in these three fatty acid oxidation systems possess PPRE elements and are regulated by PPARα, though PPARβ/δ has also been shown to regulate some of these genes [Desvergne and Wahli, 1999; Reddy and Chu, 1996; Reddy and Hashimoto, 2001; Sanderson et al., 2009; van der Meer et al., 2010; Varanasi et al., 1996]. Fatty acids are converted into acyl-CoAs prior to oxidation.

Mitochondrial β-oxidation is primarily involved in the oxidation of the major portion of the short-(<C₈), medium-(C₈-C₁₂) and long-(C₁₂-C₂₀) chain fatty acids, and in the process, constitutes the primary source of energy derived from fatty acids [Reddy and Hashimoto, 2001]. The PPARα-regulated enzymes, long chain acyl-CoA synthetases and carnitine palmitoyl transferase-1, are essential, respectively, for generating fatty acyl-CoA, and facilitating the entry of fatty acyl carnitine into mitochondria [Coleman et al., 2002]. The first step in mitochondrial β-oxidation is comprised of α-β-dehydrogenation of the fatty acyl-CoA ester by a family of four chain length specific straight chain acyl-CoA dehydrogenases [Hashimoto, 1999; Reddy and Hashimoto, 2001]. These include very-long chain, medium chain and short chain enzymes. A heterotrimeric protein, exhibiting 2-enoyl-CoA hydratase, 3-hydroxyacyl-CoA dehydrogenase, and 3-keto acyl-CoA thiolase activities, participates in the second, third and fourth steps in the mitochondrial β-oxidation pathway [Hashimoto, 1999; Reddy and Rao, 2006]. Mitochondrial β-oxidation results in the progressive shortening of fatty acids into acetyl-CoA subunits. These either condense into ketone bodies to serve as oxidizable energy substrates for extrahepatic tissues, especially during starvation, or enter into the tricarboxylic acid cycle for further oxidation to water and carbon dioxide [Reddy and Hashimoto, 2001; Reddy and Rao, 2006].

Peroxisomal β-oxidation is streamlined toward the metabolism, almost exclusively, of very long straight-chain fatty acids (>C₂₀), 2-methyl branched fatty acids (e.g., pristanic acid that is generated by α-oxidation of the 3-methyl-branched fatty acid phytanic acid), prostanoids,

Review

dicarboxylic acids, and the C₂₇ bile acid intermediates di-
and trihydroxycoprostanoic acids [Ferdinandusse et al., 
2009; Jia et al., 2003]. Substrates for peroxisomal 
β-oxidation are less abundant and relatively more toxic, 
and they are not processed by the mitochondrial 
β-oxidation system [Reddy and Rao, 2006]. However, 
peroxisomal β-oxidation is required to shorten the chain 
length of very long chain fatty acids and chain-shortened 
acyl-CoAs are shunted to mitochondria for the completion 
of β-oxidation [Reddy and Hashimoto, 2001]. The 
coordinate inducibility of peroxisomal β-oxidation system 
enzymes by peroxisome proliferators in now well 
established [Reddy and Hashimoto, 2001; Reddy et al., 
1986b] and this induction is regulated by PPARα 
[Issemann and Green, 1990; Kliewer et al., 2001; Lee et 
al., 1995].

Peroxisomal β-oxidation consists of four metabolic steps 
and each metabolic conversion can be performed by at 
least two different enzymes [Hashimoto, 1999; Reddy 
and Hashimoto, 2001]. Some of these enzymes are 
inducible by peroxisome proliferators, while others seem 
refractive to induction by these PPARα ligands [Reddy 
and Hashimoto, 2001]. Straight-chain fatty acyl-CoAs 
serve as substrates for the classical inducible pathway, 
whereas the second non-inducible β-oxidation enzyme 
system acts on 2-methyl-branched fatty acyl-CoAs 
[Hashimoto, 1999]. In the inducible PPARα-regulated 
peroxisomal β-oxidation pathway, fatty acyl-CoA oxidase 
1 (ACOX1) is responsible for the initial oxidation of 
very-long-chain fatty acyl-CoAs to their corresponding 
trans-2-enoyl-CoAs. On the other hand, the non-inducible 
fatty acyl-CoA oxidase 2 (ACOX2) serves as the first step 
in the metabolism of branched-chain fatty acyl-CoAs. The 
second and third reactions in the inducible β-oxidation 
system, hydration and dehydrogenation of enoyl-CoA 
esters to 3-ketoacyl-CoA are catalyzed by a single 
bifunctional enzyme enoyl-CoA 
hydratase/L-3-hydroxyacyl-CoA dehydrogenase 
(L-PBE/MFP1). Ketoacyl-CoAs generated by 
L-PBE/MFP1 are then converted by 3-ketoacyl-CoA 
thiolase (PTL) to acetyl-CoA and to an acyl-CoA that is 
two carbon atoms shorter than the original molecule. The 
shortened acyl-CoA reenters the β-oxidation cycle, and 
this process repeats for about 5 cycles, resulting in the 
removal of ten carbon atoms. The appropriately 
chain-shortened acyl-CoAs are then transported to 
mitochondria for the completion of β-oxidation [Hashimoto, 
1999].

All three genes of this classical inducible peroxisomal 
β-oxidation system (ACOX1, L-PBE/MFP1, and PTL) are 
tightly regulated by PPARα. Of interest is that 
L-PBE/MFP1 protein is induced abundantly in the livers 
of rats and mice treated with peroxisome proliferators 
and accounts in part for the increase in peroxisome 
population [Jia et al., 2003; Reddy, 2004]. Although, 
L-PBE/MFP1 is highly inducible by both synthetic and 
natural PPARα ligands in liver, the functions carried out 
by this enzyme can be also performed highly effectively 
by the D-3-hydroxyacyl-CoA 
dehydratase/D-3-hydroxyacyl-CoA dehydrogenase


Pleiotropic responses of PPARα

(D-bi/multifunctional enzyme 2; D-PBE/MFP2)
[Hashimoto, 1999; Jia et al., 2003]. 3-ketoacyl-CoAs 
generated by D-PBE/MFP2 of the non-inducible pathway 
are cleaved by the third enzyme of this system, sterol 
carrier protein X, which possesses thiolase activity 
[Hashimoto, 1999].

In addition to mitochondrial and peroxisomal β-oxidative 
processing, fatty acids are also oxidized by microsomal 
ω-oxidation carried out by CYP4A enzymes that are 
regulated by PPARα [Reddy and Hashimoto, 2001]. The 
first step of ω-hydroxylation of saturated and unsaturated 
fatty acids occurs in the endoplasmic reticulum and the 
resulting ω-hydroxy fatty acid is then dehydrogenated to 
yield highly toxic dicarboxylic acids in the cytosol. 
Dicarboxylic acids are converted to dicarboxylyl-CoAs 
and enter peroxisomes for further metabolism by the 
classical inducible β-oxidative enzymes [Ferdinandusse 
et al., 2004; Reddy and Rao, 2006]. Dicarboxylic acids 
are unique in that they serve as substrates for the 
peroxisomal β-oxidation system and also as ligands for 
PPARα [Hashimoto, 1999]. It would appear that 
dicarboxylic fatty acids streamline their own metabolism 
by inducing PPARα activation and thus regulating all 
three fatty acid oxidation enzyme pathways [Reddy and 
Hashimoto, 2001].

In summary, the PPARα regulates all three fatty acid 
oxidation systems. Gene knockout mouse models have 
provided evidence to indicate that substrates for 
peroxisomal β-oxidation function as PPARα ligands to 
enhance energy burning. In this regard, hyperactivation 
of PPARα by pharmacological intervention might prove 
to be useful as an adjuvant to exercise in combating 
obesity in individuals who are refractory to modulating 
energy consumption. PPARα ligands do not appear to 
cause weight loss independent of PPARα activation.

**PPARα and hepatic steatosis**

Fatty liver disease is the most common liver disease, as 
it encompasses a morphological spectrum of hepatic 
steatosis and steatohepatitis that progresses to cirrhosis 
and hepatocellular carcinoma [Reddy, 2001; Reddy and 
Rao, 2006; Zafrani, 2004]. Fatty liver disease occurs 
worldwide in those with excess alcohol consumption 
(alcoholic fatty liver disease), and those who are obese 
with or without added insulin resistance (non-alcoholic 
fatty liver disease) [Crabb et al., 2004; Evans et al., 2004; 
Hamaguchi et al., 2005; Reddy and Rao, 2006; Sozio 
and Crabb, 2008]. Several metabolic and genetic 
diseases that influence fatty acid metabolism also develop 
fatty liver disease [Browning and Horton, 2004; Reddy 
and Rao, 2006]. Fatty liver disease is the culmination of 
increased energy uptake, increased hepatic lipogenesis, 
decreased energy combustion and decreased hepatic 
secretion of liver triglycerides. PPARα gene knockout 
mice have provided valuable clues regarding the role of 
this transcription factor in energy metabolism by liver.

By virtue of its unique ability to orchestrate fatty acid 
oxidation, PPARα appears to play a significant role in the 
pathogenesis of hepatic steatosis. First, as indicated

Review

above, PPARα influences the expression of hepatic lipogenic genes by regulating the primary transcription factors SREBP-1c and liver X receptor α (LXRα) [Browning and Horton, 2004; Hebbachi et al., 2008]. Second, in conditions for increased demand for fatty acid oxidation, such as starvation, PPARα is essential for the upregulation of some of the enzymes necessary for this process, though PPARβ/δ has recently been shown to be important in regulating some of these genes independently of PPARα [Sanderson et al., 2009]. Under fasted conditions, PPARα senses the lipid influx into the liver and upregulates all three fatty acid oxidation systems to burn the energy and minimize hepatic steatosis [Hashimoto et al., 2000; Kersten et al., 1999; Leone et al., 1999]. PPARα null mice are unable to efficiently induce the expression of genes involved in the oxidation of fatty acids released from adipose tissue. Accordingly, PPARα-deficient mice exhibit hepatic steatosis, hyperlipidemia and hypothermia [Hashimoto et al., 2000; Kersten et al., 1999; Leone et al., 1999]. PPARα null mice also develop severe steatohepatitis compared to wild-type mice when maintained on a diet deficient in methionine and choline [Ip et al., 2003]. Furthermore, upon treatment with PPARα ligands there is an attenuation of methionine-choline deficient diet-induced steatohepatitis [Rao and Reddy, 2004].

PPARα null mice fed ethanol develop marked hepatomegaly, steatohepatitis, and liver cell death and proliferation, implying a role for decreased fatty acid oxidation in mice deficient in this transcription factor [Nakajima et al., 2004]. Furthermore, ethanol is known to inhibit fatty acid oxidation and this is attributed to ethanol inhibition of PPARα transcription [Sozio and Crabb, 2008]. Hypo-activity of PPARα might play a role in the severity of alcohol liver disease in the human. On the other hand, it has been shown that PPARα activation is essential for hepatitis C virus (HCV) core protein-induced hepatic steatosis and hepatocellular carcinoma in mice [Tanaka et al., 2008].

### PPARα and hypolipidemic effects

PPARα ligands reduce VLDL production and enhance the catabolism of TG-rich particles, which indirectly decreases small dense LDL (sdLDL) particles, enhancing the formation of HDL particles and hepatic elimination of excess cholesterol [Lefebvre et al., 2006]. PPARα upregulates lipoprotein lipase transcription in liver and muscle and this leads to increased TG hydrolysis.

The activity of ApoC-III, which is an inhibitor of both LPL activity and remnant clearance, is lowered by PPARα agonists [Peters et al., 1997]. PPARα agonists induce ABCA1 and SR-BI expression in macrophages. HDL apolipoprotein genes apoA-I and apoA-II are activated by PPARα by direct transcriptional control [Desvergne and Wahli, 1999; Lefebvre et al., 2006; Peters et al., 1997]. Thus, PPARα activation by fibrates and other compounds, elicits a global normolipidemic response, by reducing TG-rich particle production, increasing their lipolysis, and promoting HDL metabolism [Lefebvre et al., 2006].

---

Pleiotropic responses of PPARα

### PPARα and inflammation

Evidence indicates that PPARα plays a beneficial role in reducing inflammation. Activation of this receptor appears to influence both acute and chronic inflammatory disorders involving neutrophils and macrophages. Leukotriene B₄ (LTB4), a powerful chemotactic inflammatory eicosanoid, is an endogenous PPARα ligand (Figure 2). Like other PPARα ligands, it induces transcription of genes of the β- and ω-oxidation pathways that neutralize and degrade LTB4 itself to regulate the inflammatory response [Devchand et al., 1996; Ford-Hutchinson, 1990]. Absence of PPARα prolongs the LTB4-induced inflammatory response [Devchand et al., 1996]. Agents containing LTB4 or its precursor arachidonic acid, when applied to the ears of PPARα knockout and wild-type mice, showed that the inflammatory response was significantly prolonged in PPARα null mice, compared to the wild-type controls [Devchand et al., 1996]. These data established that PPARα regulates the duration of the inflammatory response, possibly through limiting cytokine expression and by inducing genes that metabolize LTB4 [Devchand et al., 1996]. Based on these observations, it is logical to surmise that PPARα ligands exert potential anti-inflammatory effects in modulating various inflammatory processes such as atherogenesis and hepatitis [Devchand et al., 1996; Ricote and Glass, 2007; Staels et al., 2008; Zandbergen and Plutzky, 2007].

Macrophages are mediators of inflammation in the vasculature and participate in the development of atherosclerotic plaques [Schwartz et al., 1985]. In the pathogenesis of atherosclerosis, PPARα activation could result initially in reduced leukocyte adhesion to activated endothelial cells of the arterial lumen, and subsequently in inhibiting the formation of macrophage foam cells by regulating expression of genes involved in reverse cholesterol transport and reactive oxygen species output [Chinetti et al., 2001; Zandbergen and Plutzky, 2007]. Accordingly, activation of PPARα may be beneficial in curtailing the inflammatory response and, in particular, the formation and progression of atherosclerotic plaques by minimizing lipoprotein oxidative modifications.

PPARα ligands significantly reduce the levels of pro-inflammatory cytokines such as interleukin-1 (IL-1), tumor necrosis factor-α (TNF-α), cyclooxygenase-2 (COX-2), and inducible nitric oxide synthase (iNOS) by inhibiting the translocation of the p65 subunit of nuclear factor κ-B (NF-κB) and decreasing phosphorylation of the c-jun subunit of AP-1 [Ramanan et al., 2008]. Loss of PPARα-mediated gene transcription in PPARα null macrophages resulted in enhanced MAPK phosphorylation, leading to increased NF-κB [Crisafulli and Cuzzocrea, 2009]. Several other studies have also demonstrated a similar, important role for PPARα in modulating inflammation in vascular endothelial cells, cartilage and bone tissue, kidney, adipose tissue, and glial cells in the central nervous system (CNS) [Kim et al., 2009; Kono et al., 2009; Murakami et al., 2007; Poleni et al., 2007; Still et al., 2008]. These studies may point toward potential new roles for PPARα and PPARα target

www.nursa.org | NRS | 2010 | Vol. 8 | DOI: 10.1621/nrs.08002 | Page 11 of 21

Review
---

genes as therapeutic targets in disorders involving inflammation such as atherosclerosis, joint disease, and autoimmune disorders such as multiple sclerosis [Arzuaga et al., 2007; Dushkin et al., 2007].

**PPARα and liver cancer**

The development of HCC in mice fed a diet containing nafenopin, a potent peroxisome proliferator, was first reported in 1976 [Reddy et al., 1976]. Subsequently, several hypolipidemic compounds and plasticizers such as DEHP and DEHA have also been shown to induce liver tumors in rats and mice [Kluwe et al., 1983; Rao and Reddy, 1987; Reddy et al., 1980]. Rodents exposed to peroxisome proliferators develop hepatomegaly, largely due to the early hyperplastic and hypertrophic growth of hepatocytes [Reddy et al., 1980; Reddy and Krishnakantha, 1975; Reddy et al., 1979]. With chronic exposure, hepatic adenomas and hepatocellular carcinomas develop in these rodents [Rao and Reddy, 1996; Reddy et al., 1980; Reddy et al., 1976]. The latency period and the incidence of liver tumors appear to correlate well with the effectiveness of a given compound to induce peroxisome proliferation-associated pleiotropic responses [Rao and Reddy, 1996]. Phenotypic features of peroxisome proliferator-induced liver tumors differ from those induced by classic genotoxic hepatocarcinogens [Rao and Reddy, 1996]. Since peroxisome proliferators are neither DNA damaging nor mutagenic, it was proposed that these compounds constitute a novel class of nongenotoxic hepatocarcinogens and this concept laid the foundation for the receptor-mediated hepatocarcinogenesis [Rao and Reddy, 1987; Reddy et al., 1980; Reddy and Lalwani, 1983; Yeldandi et al., 2000].

Peroxisome proliferators do not directly cause genetic damage. It became increasingly evident that hepatocarcinogenicity is due to metabolic alterations resulting from sustained receptor activation that contribute to oxidative stress induced DNA damage in liver [Peters et al., 2005; Rao and Reddy, 1987; Reddy and Lalwani, 1983; Rusyn et al., 2004; Yeldandi et al., 2000]. In normal liver, hydrogen peroxide is produced as a byproduct of many oxidative reactions [Yeldandi et al., 2000]. Of these, the catabolism of very long chain fatty acids, resulting from massive induction of the peroxisomal fatty acid oxidation system by peroxisome proliferators, has received considerable attention in peroxisome proliferator-induced liver carcinogenesis [Nemali et al., 1988; Reddy et al., 1982]. Peroxisomal catalase degrades hydrogen peroxide in normal liver, but in livers with peroxisome proliferation catalase expression increases ~2- to 5-fold in contrast to disproportionately large increases in hydrogen peroxide producing acyl-CoA oxidase and microsomal ω-oxidation enzymes [Chu et al., 2004; Nemali et al., 1988; Yeldandi et al., 2000]. This imbalance between the expression of enzymes capable of producing- and degrading- hydrogen peroxide and other reactive oxygen species in hepatocytes contribute to oxidative stress, lipid peroxidation and oxidative DNA damage [Rao and Reddy, 1987]. These changes influence lipofuscin accumulation, a product of the oxidation of

---

Pleiotropic responses of PPARα
---

unsaturated fatty acids, as well as hepatocellular proliferation [Peters et al., 2005; Rao and Reddy, 1996; Reddy et al., 1982]. DNA damage through oxidative stress and hepatocellular proliferation, together, are considered as possible mechanisms responsible for the development of hepatocellular carcinomas in rodents chronically exposed to peroxisome proliferators [Peters et al., 2005; Rao and Reddy, 1987; Reddy, 2004; Reddy et al., 1980]. Consistent with this concept is a concordant marked induction in the liver of genes specific for the long-patch base excision DNA repair following exposure to PPARα ligand, Wy-14,643 [Rusyn et al., 2004]. Long-patch base excision DNA repair is a predominant pathway that removes oxidized DNA lesions [Rusyn et al., 2004]. It is evident that DNA-damaging oxidants are generated by enzymes induced after activation of PPARα, such as those involved in fatty acid oxidation [Rusyn et al., 2004]. Based on these observations, it is concluded that chronic exposure to synthetic peroxisome proliferators results in sustained activation of PPARα and transcriptional activation of PPARα responsive genes that affect intermediary metabolism in liver [Reddy and Chu, 1996; Reddy and Hashimoto, 2001]. These metabolic changes, along with the anti-apoptotic effects of PPARα activation, contribute to oxidative DNA damage and increased hepatocellular proliferation leading to liver cancer development [Chen et al., 2009; Peters et al., 2005; Reddy and Chu, 1996].

**Levels of regulation of PPARα transcriptional activity**

Although it has been well-demonstrated that peroxisome proliferation and liver cancer develop in rodent models, it is not clear that chronic administration of PPARα ligands leads to tumorigenesis in humans. However, many factors may contribute to this variability seen in species- and cell-specific responses to PPARα ligands. The levels of regulation of PPARα-induced responses (Figure 3) include: the potency of specific exogenous and endogenous PPARα ligands to activate PPARα [Fan et al., 1998; Reddy et al., 1976], differential expression of the PPARα and its heterodimerizing partner RXR [Palmer et al., 1998], species differences in the promoter regions of PPARα target genes [Reddy and Chu, 1996], differential expression of certain coactivator proteins necessary for PPARα-mediated transactivation and differing expression levels and activity of PPARα target genes [Jia et al., 2004; Yu and Reddy, 2007].

PPARα ligands show an intrinsic difference in their capability to induce maximal PPARα activation and peroxisome proliferation. Male F344 rats require feeding of DEHP at a 2% (w/w) dietary level for 30 days in order to achieve a significant degree of hepatomegaly and peroxisome proliferation, while rats fed for the same period on a diet containing only 0.01% ciprofibrate show a significant increase in liver to body weight ratio [Reddy et al., 1986a]. Ciprofibrate is more effective at inducing the classical peroxisome proliferative response when compared to phthalate ester plasticizers such as DEHP. Furthermore, it should be noted that hepatic peroxisome

Review                                                                                       Pleiotropic responses of PPARα

![Diagram](#)

Figure 3. Levels of regulation of PPARα-mediated transcription. Diagram illustrating the different factors which regulate the ability of PPARα to activate or repress transcription of target genes. Five major levels of regulation are: (1) the ligand, (2) the receptor expression, (3) the promoter, (4) the coactivator proteins and (5) the expression of target genes.

proliferation is inducible in the livers of several species including cats, chickens, pigeons and the nonhuman primates such as rhesus and cynomologus monkeys by ciprofibrate at dose levels that exceed therapeutic levels [Lalwani et al., 1985; Reddy et al., 1984]. It would appear that induction of peroxisome proliferation, in part, can be dose-dependent, but not a species-specific phenomenon.

Tissue levels of expression of the PPARα receptor may help explain the differences seen in response to treatment with PPARα agonists. In the mouse, liver, kidney and heart use fatty acid β-oxidation as a source of fuel. In these tissues, the basal levels of mitochondrial β-oxidation enzymes are similar. However, liver shows a much higher degree of induction of PPARα-regulated fatty acid oxidation enzymes than kidney or heart [Cook et al., 2000]. PPARα levels appear to be lower in human livers, as compared to rodent livers, and have been proposed to account for reduced response of human liver to peroxisome proliferation [Palmer et al., 1998]. Since these estimates were from post mortem livers, data on freshly obtained liver biopsy samples will be required for reliable comparison [Palmer et al., 1998; Reddy, 2004]. Adenoviral overexpression of human PPARα in PPARα-null mouse liver restored peroxisome proliferator-induced pleiotropic responses in liver,

including peroxisome proliferation, induction of PPARα target genes and liver cell proliferation [Yu et al., 2001].

The PPARα transactivational response is also regulated by the nature of the PPRE sequence present in the promoter region of responsive genes. PPARα may regulate gene activity in conjunction with other transcription factors such as CCAAT/enhancer-binding protein (C/EBP), sterol regulatory element binding protein 1 (SREBP1), TATA-binding protein (TBP) and members of the signal transducer and regulator of transcription (STAT) family [van der Meer et al., 2010]. The presence or absence of motifs which bind these other transcription factors in the promoter regions of PPARα target genes had a significant influence on the ability of PPARα to transactivate or transrepress expression of those genes. Species differences in the promoter regions of PPARα target genes may account for some of the differences observed in response to treatment with PPARα ligands.

As previously discussed, coactivator proteins are required for effective transactivation of target genes by PPARα. The protein, PBP/MED1, when disrupted in mouse liver, results in the abrogation of peroxisome proliferation and other pleiotropic effects of treatment with PPARα ligands [Jia et al., 2004; Matsumoto et al., 2007]. The degree of

www.nursa.org                                                                                   NRS | 2010 | Vol. 8 | DOI: 10.1621/nrs.08002 | Page 13 of 21

Review

expression of different coactivator proteins such as PBP/MED1, and the particular milieu of coactivators expressed in a particular tissue, organism, or species may therefore also contribute to the variable species response to treatment with PPARα ligands.

Recently, it has been shown that activation of PPARα represses the expression of the let-7C miRNA, which in turn releases the repression of c-myc expression [Shah et al., 2007]. Interestingly, mice expressing the human PPARα protein do not show this repression of the let-7C miRNA [Yang et al., 2008]. Accordingly, these mice also do not exhibit hepatocellular proliferation or development of tumors [Gonzalez and Shah, 2008; Morimura et al., 2006; Shah et al., 2007]. This induced expression of c-myc protein is proposed to be the mechanism contributing to PPARα ligand-induced hepatocellular proliferation [Shah et al., 2007]. Increased cell proliferative stimulus mediated by PPARα ligands in an intense oxidative stress inducing environment in livers with peroxisome proliferation, could act in concert and contribute to the development of liver tumors. Nonetheless, the level of human PPARα expression may be an issue for consideration in humanized PPARα mice, since adenoviral overexpression of human PPARα in PPARα-null mice showed robust liver cell proliferation in response to a synthetic PPARα ligand [Yu et al., 2001]. Accordingly, sustained activation of PPARα by synthetic ligands is hepatocarcinogenic, as observed in rodent models [Reddy and Chu, 1996].

Additionally, PPARα activity is influenced by the ability of target genes, such as ACOX1 to efficiently metabolize and thus remove endogenous ligands in hepatocytes. In ACOX1 null mouse livers, PPARα activity is dramatically increased as a result of the accumulation of unmetabolized, endogenous ligands [Cook et al., 2000; Fan et al., 1998; Yeldandi et al., 2000]. As a result, species differences in the expression of PPARα target genes may also contribute to species-specific responses to treatment with PPARα ligands. ACOX1 null mouse livers display the hallmarks of PPARα activation such as hepatocellular regeneration accompanied by profound generalized spontaneous peroxisome proliferation, and increased mRNA levels of genes that are regulated by PPARα, which ultimately lead to liver tumor development [Fan et al., 1998]. Although ACOX1, a major hydrogen peroxide generating enzyme in peroxisomes, is absent in these livers, there appear to be other sources that may contribute to oxidative stress in liver such as cytochrome P450 family proteins induced by activated PPARα [Fan et al., 1998; Reddy and Hashimoto, 2001; Yeldandi et al., 2000]. These CYP4A proteins typically catalyze the addition of oxygen to a variety of substrate molecules and may produce reactive oxygen as a by-product [Reddy, 2004; Reddy and Hashimoto, 2001; Yeldandi et al., 2000]. Thus, it would be essential to consider that ACOX1 is not the only source of oxidative stress in PPARα-activated livers. Expression of long-patch base excision DNA repair genes was found to be upregulated significantly in livers of naïve ACOX1-null mice and was increased similarly to the effect of WY-14,643 [Rusyn et

Pleiotropic responses of PPARα

al., 2004]. These observations strongly support that sustained activation of PPARα in liver, either by synthetic or natural ligands, leads to endoplasmic reticulum and oxidative stress. Liver tumors that develop in ACOX1 null livers exhibit similar cDNA profiles as tumors resulting from long-term treatment with PPARα ligands, but not similar to tumors resulting from treatment with diethylnitrosamine, a genotoxic mutagen [Fan et al., 1998; Meyer et al., 2003]. These observations further support the novel nature of PPARα ligand-induced hepatocarcinogenesis.

PPARα polymorphism

Genetic variation and polymorphism of PPARα in the human have been under intensive study [Carlberg and Dunlop, 2006]. To date, at least 14 polymorphisms of PPARα have been reported (Figure 1) [Naito et al., 2006] and include P22R, D140Y [Hara et al., 2001], R127Q (rs1800204), R131Q [Sapone et al., 2000], L162V (rs1800206), R178G [Nielsen et al., 2003], V227A (rs1800234), A268V (rs1042311), D304N (rs1800242), G395A (rs2229245), D409T (rs1800243), Q413L (rs9615759), D140N and G395E [Naito et al., 2006]. Some of these variations were reportedly associated with human lipid metabolic changes. For example, PPARα-V227A is a major polymorphism in the Japanese population [Naito et al., 2006]. Its activity may be greater than that of wild-type PPARα, but is decreased by increased alcohol consumption, suggesting that alcoholic liver injury may affect the activity of this receptor [Naito et al., 2006]. Among the 4,248 subjects from 3 ethnic groups (2,899 Chinese, 761 Malay and 588 Asian Indians) genotyped for polymorphisms at the PPARA locus, allele frequencies for the V227A mutation were 0.04 in Chinese, 0.006 in Malays and 0.003 in Asian Indians [Chan et al., 2006]. This polymorphism is associated with lower serum concentrations of total cholesterol and triglycerides in Chinese women. The V227A polymorphism is believed to modulate the association between dietary polyunsaturated fatty acid intake and serum high density lipoprotein concentration. These observations suggest that genetic variation of PPARα may determine the response to changes in dietary PUFA intake [Chan et al., 2006]. Furthermore, L162V polymorphism in the PPARα gene, first identified in the Caucasian population [Sapone et al., 2000; Vohl et al., 2000], was associated with increased levels of serum apolipoprotein B and LDL cholesterol. The L162V polymorphism in PPARα may determine the response to PPARα ligands [Bosse et al., 2002]. The PPARα-L162V polymorphism alone or in interaction with dietary fat intake is associated with components of the metabolic syndrome and may contribute to variability in plasma lipoprotein and lipid response after modification of the dietary polyunsaturated to saturated fatty acids ratio [Paradis et al., 2005; Robitaille et al., 2004; Sapone et al., 2000; Vohl et al., 2000].

Conclusions

Studies with a structurally diverse group of chemicals designated as peroxisome proliferators laid the foundation

Review

for the discovery of a subfamily of nuclear receptors named peroxisome proliferator-activated receptor (PPAR) with three members: PPARα, PPARβ/δ and PPARγ. Of these, PPARα plays a critical role in fatty acid oxidation and is thus responsible for energy expenditure. Sustained activation of PPAR leads to the development of liver tumors in rats and mice, whereas PPARα deficiency leads to impaired response to fatty acid influx into liver and contributes to hepatic steatosis. Studies with gene knockout mouse models have yielded information on putative endogenous PPARα ligands that are either generated or degraded by certain enzymes during intermediary metabolism. Several nuclear receptor coactivators have been noted to interact with PPARα, but studies to-date indicate that the coactivator PBP/MED1 is essential for PPAR target gene activation, liver cell proliferation and liver tumorigenesis. As for future challenges, it remains to be determined the extent and nature of other endogenous PPAR agonists and other coactivators that influence PPARα function. Such information may provide clues for therapeutic intervention to enhance energy utilization, minimize the adverse effects of both alcoholic and nonalcoholic fatty liver disease and modulate liver cell proliferation.

Supplementary Material

Supplementary File 1: Known target genes of PPARα.

Supplementary File 2: Bibliography for the known target genes of PPARα table above.

References

Antonson, P., Schuster, G. U., Wang, L., Rozell, B., Holter, E., Flodby, P., Treuter, E., Holmgren, L. and Gustafsson, J. A. (2003) Inactivation of the nuclear receptor coactivator RAP250 in mice results in placental vascular dysfunction Mol Cell Biol 23, 1260-8.

Aoyama, T., Peters, J. M., Iritani, N., Nakajima, T., Furihata, K., Hashimoto, T. and Gonzalez, F. J. (1998) Altered constitutive expression of fatty acid-metabolizing enzymes in mice lacking the peroxisome proliferator-activated receptor α (PPARalpha) J Biol Chem 273, 5678-84.

Arzuaga, X., Reiterer, G., Majkova, Z., Kilgore, M. W., Toborek, M. and Hennig, B. (2007) PPARAlpha ligands reduce PCB-induced endothelial activation: possible interactions in inflammation and atherosclerosis Cardiovasc Toxicol 7, 264-72.

Barger, P. M., Browning, A. C., Garner, A. N. and Kelly, D. P. (2001) p38 mitogen-activated protein kinase activates peroxisome proliferator-activated receptor α: a potential role in the cardiac metabolic stress response J Biol Chem 276, 44495-501.

Bays, H. and Stein, E. A. (2003) Pharmacotherapy for dyslipidaemia--current therapies and future agents Expert Opin Pharmacother 4, 1901-38.

Blanquart, C., Mansouri, R., Paumelle, R., Fruchart, J. C., Staels, B. and Glineur, C. (2004) The protein kinase C signaling pathway regulates a molecular switch between transactivation and transrepression activity of the peroxisome proliferator-activated receptor α Mol Endocrinol 18, 1906-18.

Bookout, A. L., Jeong, Y., Downes, M., Yu, R. T., Evans, R. M. and Mangelsdorf, D. J. (2006) Anatomical profiling of nuclear receptor expression reveals a hierarchical transcriptional network Cell 126, 789-99.

Bosse, Y., Pascot, A., Dumont, M., Brochu, M., Prud'homme, D., Bergeron, J., Despres, J. P. and Vohl, M. C. (2002) Influences of the PPAR α-L162V polymorphism on plasma HDL(2)-cholesterol response of abdominally obese men treated with gemfibrozil Genet Med 4, 311-5.

Braissant, O., Foufelle, F., Scotto, C., Dauca, M. and Wahli, W. (1996) Differential expression of peroxisome proliferator-activated receptors (PPARs): tissue distribution of PPAR-α, -β, and -γ in the adult rat Endocrinology 137, 354-66.

Brash, A. R. (1999) Lipoxygenases: occurrence, functions, catalysis, and acquisition of substrate J Biol Chem 274, 23679-82.

Browning, J. D. and Horton, J. D. (2004) Molecular mediators of hepatic steatosis and liver injury J Clin Invest 114, 147-52.

Bugge, T. H., Pohl, J., Lonnoy, O. and Stunnenberg, H. G. (1992) RXR α, a promiscuous partner of retinoic acid and thyroid hormone receptors Embo J 11, 1409-18.

Bulhak, A. A., Jung, C., Ostenson, C. G., Lundberg, J. O., Sjoquist, P. O. and Pernow, J. (2009) PPAR-α activation protects the type 2 diabetic myocardium against ischemia-reperfusion injury: involvement of the PI3-Kinase/Akt and NO pathway Am J Physiol Heart Circ Physiol 296, H719-27.

Carlberg, C. and Dunlop, T. W. (2006) An integrated biological approach to nuclear receptor signaling in physiological control and disease Crit Rev Eukaryot Gene Expr 16, 1-22.

Chakravarthy, M. V., Zhu, Y., Lopez, M., Yin, L., Wozniak, D. F., Coleman, T., Hu, Z., Wolfgang, M., Vidal-Puig, A., Lane, M. D. and Semenkovich, C. F. (2007) Brain fatty acid synthase activates PPARalpha to maintain energy homeostasis J Clin Invest 117, 2539-52.

Chakravarthy, M. V., Lodhi, I. J., Yin, L., Malapaka, R. R., Xu, H. E., Turk, J. and Semenkovich, C. F. (2009) Identification of a physiologically relevant endogenous ligand for PPARalpha in liver Cell 138, 476-88.

Chandra, V., Huang, P., Hamuro, Y., Raghuram, S., Wang, Y., Burris, T. P. and Rastinejad, F. (2008) Structure of the intact PPAR-γ-RXR- nuclear receptor complex on DNA Nature 456, 350-6.

Chang, F., Jaber, L. A., Berlie, H. D. and O'Connell, M. B. (2007) Evolution of peroxisome proliferator-activated receptor agonists Ann Pharmacother 41, 973-83.

Chan, E., Tan, C. S., Deurenberg-Yap, M., Chia, K. S., Chew, S. K. and Tai, E. S. (2006) The V227A polymorphism at the PPARA locus is associated with serum lipid concentrations and modulates the association between dietary polyunsaturated fatty acid intake and serum high density lipoprotein concentrations in Chinese women Atherosclerosis 187, 309-15.

Chawla, A., Repa, J. J., Evans, R. M. and Mangelsdorf, D. J. (2001) Nuclear receptors and lipid physiology: opening the X-files Science 294, 1866-70.

Chawla, A., Schwarz, E. J., Dimaculangan, D. D. and Lazar, M. A. (1994) Peroxisome proliferator-activated receptor (PPAR) γ: adipose-predominant expression and induction early in adipocyte differentiation Endocrinology 135, 798-800.

Chen, J. D. and Evans, R. M. (1995) A transcriptional co-repressor that interacts with nuclear hormone receptors Nature 377, 454-7.

Chen, H. H., Sue, Y. M., Chen, C. H., Hsu, Y. H., Hou, C. C., Cheng, C. Y., Lin, S. L., Tsai, W. L., Chen, T. W. and Chen, T. H. (2009) Peroxisome proliferator-activated receptor α plays a crucial role in L-carnitine anti-apoptosis effect in renal tubular cells Nephrol Dial Transplant 24, 3042-9.

Cherkaoui-Malki, M., Meyer, K., Cao, W. Q., Latruffe, N., Yeldandi, A. V., Rao, M. S., Bradfield, C. A. and Reddy, J. K. (2001) Identification of novel peroxisome proliferator-activated receptor α (PPARalpha) target genes in mouse liver using cDNA microarray analysis Gene Expr 9, 291-304.

Chinetti, G., Lestavel, S., Bocher, V., Remaley, A. T., Neve, B., Torra, I. P., Teissier, E., Minnich, A., Jaye, M., Duverger, N., Brewer, H. B., Fruchart, J. C., Clavey, V. and Staels, B. (2001) PPAR-α and PPAR-γ

www.nursa.org
NRS | 2010 | Vol. 8 | DOI: 10.1621/nrs.08002 | Page 15 of 21

activators induce cholesterol removal from human macrophage foam cells through stimulation of the ABCA1 pathway Nat Med 7, 53-8.

Chopra, A. R., Louet, J. F., Saha, P., An, J., Demayo, F., Xu, J., York, B., Karpen, S., Finegold, M., Moore, D., Chan, L., Newgard, C. B. and O'Malley, B. W. (2008) Absence of the SRC-2 coactivator results in a glycogenopathy resembling Von Gierke's disease Science 322, 1395-9.

Chu, R., Lin, Y., Rao, M. S. and Reddy, J. K. (1995) Cooperative formation of higher order peroxisome proliferator-activated receptor and retinoid X receptor complexes on the peroxisome proliferator responsive element of the rat hydratase-dehydrogenase gene J Biol Chem 270, 29636-9.

Chu, R., Lim, H., Brumfield, L., Liu, H., Herring, C., Ulintz, P., Reddy, J. K. and Davison, M. (2004) Protein profiling of mouse livers with peroxisome proliferator-activated receptor α activation Mol Cell Biol 24, 6288-97.

Coleman, R. A., Lewin, T. M., Van Horn, C. G. and Gonzalez-Baro, M. R. (2002) Do long-chain acyl-CoA synthetases regulate fatty acid entry into synthetic versus degradative pathways? J Nutr 132, 2123-6.

Cook, W. S., Yeldandi, A. V., Rao, M. S., Hashimoto, T. and Reddy, J. K. (2000) Less extrahepatic induction of fatty acid β-oxidation enzymes by PPAR α Biochem Biophys Res Commun 278, 250-7.

Corton, J. C., Anderson, S. P. and Stauber, A. (2000) Central role of peroxisome proliferator-activated receptors in the actions of peroxisome proliferators Annu Rev Pharmacol Toxicol 40, 491-518.

Crabb, D. W., Galli, A., Fischer, M. and You, M. (2004) Molecular mechanisms of alcoholic fatty liver: role of peroxisome proliferator-activated receptor α Alcohol 34, 35-8.

Crawford, S. E., Qi, C., Misra, P., Stellmach, V., Rao, M. S., Engel, J. D., Zhu, Y. and Reddy, J. K. (2002) Defects of the heart, eye, and megakaryocytes in peroxisome proliferator activator receptor-binding protein (PBP) null embryos implicate GATA family of transcription factors J Biol Chem 277, 3585-92.

Crisafulli, C. and Cuzzocrea, S. (2009) The role of endogenous and exogenous ligands for the peroxisome proliferator-activated receptor α (PPAR-α) in the regulation of inflammation in macrophages Shock 32, 62-73.

Cronet, P., Petersen, J. F., Folmer, R., Blomberg, N., Sjoblom, K., Karlsson, U., Lindstedt, E. L. and Bamberg, K. (2001) Structure of the PPARAlpha and -γ ligand binding domain in complex with AZ 242; ligand selectivity and agonist activation in the PPAR family Structure 9, 699-706.

Desvergne, B. and Wahli, W. (1999) Peroxisome proliferator-activated receptors: nuclear control of metabolism Endocr Rev 20, 649-88.

Devchand, P. R., Keller, H., Peters, J. M., Vazquez, M., Gonzalez, F. J. and Wahli, W. (1996) The PPARalpha-leukotriene B4 pathway to inflammation control Nature 384, 39-43.

Di-Poi, N., Tan, N. S., Michalik, L., Wahli, W. and Desvergne, B. (2002) Antiapoptotic role of PPARbeta in keratinocytes via transcriptional control of the Akt1 signaling pathway Mol Cell 10, 721-33.

DiRenzo, J., Soderstrom, M., Kurokawa, R., Ogliastro, M. H., Ricote, M., Ingrey, S., Horlein, A., Rosenfeld, M. G. and Glass, C. K. (1997) Peroxisome proliferator-activated receptors and retinoic acid receptors differentially control the interactions of retinoid X receptor heterodimers with ligands, coactivators, and corepressors Mol Cell Biol 17, 2166-76.

Dotson, M. R., Yuan, C. X., Roeder, R. G., Myers, L. C., Gustafsson, C. M., Jiang, Y. W., Li, Y., Kornberg, R. D. and Asturias, F. J. (2000) Structural organization of yeast and mammalian mediator complexes Proc Natl Acad Sci U S A 97, 14307-10.

Dowell, P., Ishmael, J. E., Avram, D., Peterson, V. J., Nevriv, D. J. and Leid, M. (1999) Identification of nuclear receptor corepressor as a peroxisome proliferator-activated receptor α interacting protein J Biol Chem 274, 15901-7.


Review

Pleiotropic responses of PPARα

Dreyer, C., Krey, G., Keller, H., Givel, F., Helftenbein, G. and Wahli, W. (1992) Control of the peroxisomal β-oxidation pathway by a novel family of nuclear hormone receptors Cell 68, 879-87.

Dushkin, M. I., Khoshchenko, O. M., Posokhova, E. N. and Schwarts, YSh (2007) Agonists of PPAR-α, PPAR-γ, and RXR inhibit the formation of foam cells from macrophages in mice with inflammation Bull Exp Biol Med 144, 713-6.

El Kebbaj, Z., Andreoletti, P., Mountassif, D., Kabine, M., Schohn, H., Dauca, M., Latruffe, N., El Kebbaj, M. S. and Cherkaoui-Malki, M. (2009) Differential regulation of peroxisome proliferator-activated receptor (PPAR)-alpha1 and truncated PPARalpha2 as an adaptive response to fasting in the control of hepatic peroxisomal fatty acid β-oxidation in the hibernating mammal Endocrinology 150, 1192-201.

Ellinghaus, P., Wolfrum, C., Assmann, G., Spener, F. and Seedorf, U. (1999) Phytanic acid activates the peroxisome proliferator-activated receptor α (PPARAlpha) in sterol carrier protein 2-/ sterol carrier protein x-deficient mice J Biol Chem 274, 2766-72.

Evans, R. M., Barish, G. D. and Wang, Y. X. (2004) PPARs and the complex journey to obesity Nat Med 10, 355-61.

Faergeman, N. J. and Knudsen, J. (1997) Role of long-chain fatty acyl-CoA esters in the regulation of metabolism and in cell signalling Biochem J 323 (Pt 1), 1-12.

Fan, C. Y., Pan, J., Chu, R., Lee, D., Kluckman, K. D., Usuda, N., Singh, I., Yeldandi, A. V., Rao, M. S., Maeda, N. and Reddy, J. K. (1996) Hepatocellular and hepatic peroxisomal alterations in mice with a disrupted peroxisomal fatty acyl-coenzyme A oxidase gene J Biol Chem 271, 24698-710.

Fan, C.Y., Pan, J., Usuda, N., Yeldandi, A. V., Rao, M. S. and Reddy, J. K. (1998) Steatohepatitis, spontaneous peroxisome proliferation and liver tumors in mice lacking peroxisomal fatty acyl-CoA oxidase. Implications for peroxisome proliferator-activated receptor α natural ligand metabolism J Biol Chem 273, 15639-45.

Feige, J. N., Gelman, L., Michalik, L., Desvergne, B. and Wahli, W. (2006) From molecular action to physiological outputs: peroxisome proliferator-activated receptors are nuclear receptors at the crossroads of key cellular functions Prog Lipid Res 45, 120-59.

Ferdinandusse, S., Denis, S., Faust, P. L. and Wanders, R. J. (2009) Bile acids: the role of peroxisomes J Lipid Res 50, 2139-47.

Ferdinandusse, S., Denis, S., Van Roermund, C. W., Wanders, R. J. and Dacremont, G. (2004) Identification of the peroxisomal β-oxidation enzymes involved in the degradation of long-chain dicarboxylic acids J Lipid Res 45, 1104-11.

Ford-Hutchinson, A. W. (1990) Leukotriene B4 in inflammation Crit Rev Immunol 10, 1-12.

Forman, B. M., Chen, J. and Evans, R. M. (1997) Hypolipidemic drugs, polyunsaturated fatty acids, and eicosanoids are ligands for peroxisome proliferator-activated receptors α and δ Proc Natl Acad Sci U S A 94, 4312-7.

Genini, D., Carbone, G. M. and Catapano, C. V. (2008) Multiple Interactions between Peroxisome Proliferators-Activated Receptors and the Ubiquitin-Proteasome System and Implications for Cancer Pathogenesis PPAR Res 2008, 195065.

Gervois, P., Torra, I. P., Chinetti, G., Grotzinger, T., Dubois, G., Fruchart, J. C., Fruchart-Najib, J., Leitersdorf, E. and Staels, B. (1999) A truncated human peroxisome proliferator-activated receptor α splice variant with dominant negative activity Mol Endocrinol 13, 1535-49.

Ghisletti, S., Huang, W., Jepsen, K., Benner, C., Hardiman, G., Rosenfeld, M. G. and Glass, C. K. (2009) Cooperative NCoR/SMRT interactions establish a corepressor-based strategy for integration of inflammatory and anti-inflammatory signaling pathways Genes Dev 23, 681-93.

Gizard, F., Nomiyama, T., Zhao, Y., Findeisen, H. M., Heywood, E. B., Jones, K. L., Staels, B. and Bruemmer, D. (2008) The PPARalpha/p16INK4a pathway inhibits vascular smooth muscle cell


www.nursa.org

NRS | 2010 | Vol. 8 | DOI: 10.1621/nrs.08002 | Page 16 of 21

Review                                                                                      Pleiotropic responses of PPARα

proliferation by repressing cell cycle-dependent telomerase activation
Circ Res 103, 1155-63.

Glass, C.K. and Rosenfeld, M.G. (2000) The coregulator exchange in
transcriptional functions of nuclear receptors Genes Dev 14, 121-41.

Gonzalez, F.J. and Shah, Y.M. (2008) PPARAlpha: mechanism of species
differences and hepatocarcinogenesis of peroxisome proliferators
Toxicology 246, 2-8.

Gottlicher, M., Widmark, E., Li, Q. and Gustafsson, J.A. (1992) Fatty
acids activate a chimera of the clofibril acid-activated receptor and the
glucocorticoid receptor Proc Natl Acad Sci U S A 89, 4653-7.

Hamaguchi, M., Kojima, T., Takeda, N., Nakagawa, T., Taniguchi, H.,
Fujii, K., Omatsu, T., Nakajima, T., Sarui, H., Shimazaki, M., Kato, T.,
Okuda, J. and Ida, K. (2005) The metabolic syndrome as a predictor of
nonalcoholic fatty liver disease Ann Intern Med 143, 722-8.

Hara, M., Wang, X., Paz, V.P., Iwasaki, N., Honda, M., Iwamoto, Y. and
Bell, G.I. (2001) Identification of three missense mutations in the
peroxisome proliferator-activated receptor α gene in Japanese subjects
with maturity-onset diabetes of the young J Hum Genet 46, 285-8.

Hashimoto, T., Cook, W.S., Qi, C., Yeldandi, A.V., Reddy, J.K. and Rao,
M.S. (2000) Defect in peroxisome proliferator-activated receptor
α-inducible fatty acid oxidation determines the severity of hepatic steatosis
in response to fasting J Biol Chem 275, 28918-28.

Hashimoto, T. (1999) Peroxisomal β-oxidation enzymes Neurochem Res
24, 551-63.

Hays, T., Rusyn, I., Burns, A.M., Kennett, M.J., Ward, J.M., Gonzalez,
F.J. and Peters, J.M. (2005) Role of peroxisome proliferator-activated
receptor-α (PPARAlpha) in bezafibrate-induced hepatocarcinogenesis
and cholestasis Carcinogenesis 26, 219-27.

Hebbachi, A.M., Knight, B.L., Wiggins, D., Patel, D.D. and Gibbons, G.
F. (2008) Peroxisome proliferator-activated receptor α deficiency abolishes
the response of lipogenic gene expression to re-feeding: restoration of
the normal response by activation of liver X receptor α J Biol Chem 283,
4866-76.

Heinzer, A.K., Watkins, P.A., Lu, J.F., Kemp, S., Moser, A.B., Li, Y.Y.,
Mihalik, S., Powers, J.M. and Smith, K.D. (2003) A very long-chain
acyl-CoA synthetase-deficient mouse and its relevance to X-linked
adrenoleukodystrophy Hum Mol Genet 12, 1145-54.

Horton, J.D., Goldstein, J.L. and Brown, M.S. (2002) SREBPs: activators
of the complete program of cholesterol and fatty acid synthesis in the liver
J Clin Invest 109, 1125-31.

Hostetler, H.A., Huang, H., Kier, A.B. and Schroeder, F. (2008) Glucose
directly links to lipid metabolism through high affinity interaction with
peroxisome proliferator-activated receptor α J Biol Chem 283, 2246-54.

Hostetler, H.A., Petrescu, A.D., Kier, A.B. and Schroeder, F. (2005)
Peroxisome proliferator-activated receptor α interacts with high affinity
and is conformationally responsive to endogenous ligands J Biol Chem
280, 18667-82.

Hummasti, S. and Tontonoz, P. (2006) The peroxisome
proliferator-activated receptor N-terminal domain controls isotype-selective
gene expression and adipogenesis Mol Endocrinol 20, 1261-75.

Hu, X. and Lazar, M.A. (1999) The CoRNR motif controls the recruitment
of corepressors by nuclear hormone receptors Nature 402, 93-6.

Ijpenberg, A., Jeannin, E., Wahli, W. and Desvergne, B. (1997) Polarity
and specific sequence requirements of peroxisome proliferator-activated
receptor (PPAR)/retinoid X receptor heterodimer binding to DNA. A
functional analysis of the malic enzyme gene PPAR response element J
Biol Chem 272, 20108-17.

Inoue, J., Satoh, S., Kita, M., Nakahara, M., Hachimura, S., Miyata, M.,
Nishimaki-Mogami, T. and Sato, R. (2008) PPARAlpha gene expression
is up-regulated by LXR and PXR activators in the small intestine Biochem
Biophys Res Commun 371, 675-8.


Ip, E., Farrell, G.C., Robertson, G., Hall, P., Kirsch, R. and Leclercq, I.
(2003) Central role of PPARAlpha-dependent hepatic lipid turnover in
dietary steatohepatitis in mice Hepatology 38, 123-32.

Issemann, I. and Green, S. (1990) Activation of a member of the steroid
hormone receptor superfamily by peroxisome proliferators Nature 347,
645-50.

Ito, M., Yuan, C.X., Okano, H.J., Darnell, R.B. and Roeder, R.G. (2000)
Involvement of the TRAP220 component of the TRAP/SMCC coactivator
complex in embryonic development and thyroid hormone action Mol Cell
5, 683-93.

Jepsen, K., Hermanson, O., Onami, T.M., Gleiberman, A.S., Lunyak,
V., McEvilly, R.J., Kurokawa, R., Kumar, V., Liu, F., Seto, E., Hedrick,
S.M., Mandel, G., Glass, C.K., Rose, D.W. and Rosenfeld, M.G. (2000)
Combinatorial roles of the nuclear receptor corepressor in transcription
and development Cell 102, 753-63.

Jepsen, K., Gleiberman, A.S., Shi, C., Simon, D.I. and Rosenfeld, M.
G. (2008) Cooperative regulation in development by SMRT and FOXP1
Genes Dev 22, 740-5.

Jia, Y., Viswakarma, N., Fu, T., Yu, S., Rao, M.S., Borensztajn, J. and
Reddy, J.K. (2009b) Conditional ablation of mediator subunit MED1
(MED1/PPARBP) gene in mouse liver attenuates glucocorticoid receptor
agonist dexamethasone-induced hepatic steatosis Gene Expr 14, 291-306.

Jia, Y., Viswakarma, N., Matsumoto, K., Pyper, S., Rao, M.S. and Reddy,
J.K. (2009a) Early Embryonic Lethality of Mice with Disrupted
Transcription Cofactor PIMT/NCoA6IP Gene FASEB J. 23, 739.

Jia, Y., Qi, C., Zhang, Z., Hashimoto, T., Rao, M.S., Huyghe, S., Suzuki,
Y., Van Veldhoven, P.P., Baes, M. and Reddy, J.K. (2003)
Overexpression of peroxisome proliferator-activated receptor-α
(PPARalpha)-regulated genes in liver in the absence of peroxisome
proliferation in mice deficient in both L- and D-forms of enoyl-CoA
hydratase/dehydrogenase enzymes of peroxisomal β-oxidation system
J Biol Chem 278, 47232-9.

Jia, Y., Qi, C., Kashireddi, P., Surapureddi, S., Zhu, Y.J., Rao, M.S., Le
Roith, D., Chambon, P., Gonzalez, F.J. and Reddy, J.K. (2004)
Transcription coactivator PBP, the peroxisome proliferator-activated
receptor (PPAR)-binding protein, is required for PPARAlpha-regulated
gene expression in liver J Biol Chem 279, 24427-34.

Jia, Y., Guo, G.L., Surapureddi, S., Sarkar, J., Qi, C., Guo, D., Xia, J.,
Kashireddi, P., Yu, S., Cho, Y.W., Rao, M.S., Kemper, B., Ge, K.,
Gonzalez, F.J. and Reddy, J.K. (2005) Transcription coactivator
peroxisome proliferator-activated receptor-binding protein/mediator 1
deficiency abrogates acetaminophen hepatotoxicity Proc Natl Acad Sci
U S A 102, 12531-6.

Jow, L. and Mukherjee, R. (1995) The human peroxisome
proliferator-activated receptor (PPAR) subtype NUC1 represses the
activation of hPPAR α and thyroid hormone receptors J Biol Chem 270,
3836-40.

Juge-Aubry, C., Pernin, A., Favez, T., Burger, A.G., Wahli, W., Meier, C.
A. and Desvergne, B. (1997) DNA binding properties of peroxisome
proliferator-activated receptor subtypes on various natural peroxisome
proliferator response elements. Importance of the 5'-flanking region J Biol
Chem 272, 25252-9.

Juge-Aubry, C.E., Gorla-Bajszczak, A., Pernin, A., Lemberger, T., Wahli,
W., Burger, A.G. and Meier, C.A. (1995) Peroxisome proliferator-activated
receptor mediates cross-talk with thyroid hormone receptor by competition
for retinoid X receptor. Possible role of a leucine zipper-like heptad repeat
J Biol Chem 270, 18117-22.

Juge-Aubry, C.E., Hammar, E., Siegrist-Kaiser, C., Pernin, A., Takeshita,
A., Chin, W.W., Burger, A.G. and Meier, C.A. (1999) Regulation of the
transcriptional activity of the peroxisome proliferator-activated receptor
α by phosphorylation of a ligand-independent trans-activating domain J
Biol Chem 274, 10505-10.

Kane, C.D., Stevens, K.A., Fischer, J.E., Haghighpassand, M., Royer, L.
J., Aldinger, C., Landschulz, K.T., Zagouras, P., Bagley, S.W., Hada,
W., Dullea, R., Hayward, C.M. and Francone, O.L. (2009) Molecular


www.nursa.org                                                                                   NRS | 2010 | Vol. 8 | DOI: 10.1621/nrs.08002 | Page 17 of 21

Review

characterization of novel and selective peroxisome proliferator-activated receptor α agonists with robust hypolipidemic activity in vivo Mol Pharmacol 75, 296-306.

Kelly, L. J., Vicario, P. P., Thompson, G. M., Candelore, M. R., Doebber, T. W., Ventre, J., Wu, M. S., Meurer, R., Forrest, M. J., Conner, M. W., Cascieri, M. A. and Moller, D. E. (1998) Peroxisome proliferator-activated receptors γ and α mediate in vivo regulation of uncoupling protein (UCP-1, UCP-2, UCP-3) gene expression Endocrinology 139, 4920-7.

Kersten, S., Seydoux, J., Peters, J. M., Gonzalez, F. J., Desvergne, B. and Wahli, W. (1999) Peroxisome proliferator-activated receptor α mediates the adaptive response to fasting J Clin Invest 103, 1489-98.

Kim, B. H., Won, Y. S., Kim, D. Y., Kim, B., Kim, E. Y., Yoon, M. and Oh, G. T. (2009) Signal crosstalk between estrogen and peroxisome proliferator-activated receptor α on adiposity BMB Rep 42, 91-5.

Kliewer, S. A., Umesono, K., Noonan, D. J., Heyman, R. A. and Evans, R. M. (1992) Convergence of 9-cis retinoic acid and peroxisome proliferator signalling pathways through heterodimer formation of their receptors Nature 358, 771-4.

Kliewer, S. A., Xu, H. E., Lambert, M. H. and Willson, T. M. (2001) Peroxisome proliferator-activated receptors: from genes to physiology Recent Prog Horm Res 56, 239-63.

Kluwe, W. M., Haseman, J. K. and Huff, J. E. (1983) The carcinogenicity of di(2-ethylhexyl) phthalate (DEHP) in perspective J Toxicol Environ Health 12, 159-69.

Kono, K., Kamijo, Y., Hora, K., Takahashi, K., Higuchi, M., Kiyosawa, K., Shigematsu, H., Gonzalez, F. J. and Aoyama, T. (2009) PPAR{α} attenuates the proinflammatory response in activated mesangial cells Am J Physiol Renal Physiol 296, F328-36.

Krey, G., Braissant, O., L'Horset, F., Kalkhoven, E., Perroud, M., Parker, M. G. and Wahli, W. (1997) Fatty acids, eicosanoids, and hypolipidemic agents identified as ligands of peroxisome proliferator-activated receptors by coactivator-dependent receptor ligand assay Mol Endocrinol 11, 779-91.

Lalwani, N. D., Fahl, W. E. and Reddy, J. K. (1983) Detection of a nafenopin-binding protein in rat liver cytosol associated with the induction of peroxisome proliferation by hypolipidemic compounds Biochem Biophys Res Commun 116, 388-93.

Lalwani, N. D., Reddy, M. K., Ghosh, S., Barnard, S. D., Molello, J. A. and Reddy, J. K. (1985) Induction of fatty acid β-oxidation and peroxisome proliferation in the liver of rhesus monkeys by DL-040, a new hypolipidemic agent Biochem Pharmacol 34, 3473-82.

Lalwani, N. D., Alvares, K., Reddy, M. K., Reddy, M. N., Parikh, I. and Reddy, J. K. (1987) Peroxisome proliferator-binding protein: identification and partial characterization of nafenopin-, clofibrac acid-, and ciprofibrate-binding proteins from rat liver Proc Natl Acad Sci U S A 84, 5242-6.

Landles, C., Chalk, S., Steel, J. H., Rosewell, I., Spencer-Dene, B., Lalani el, N. and Parker, M. G. (2003) The thyroid hormone receptor-associated protein TRAP220 is required at distinct embryonic stages in placental, cardiac, and hepatic development Mol Endocrinol 17, 2418-35.

Lee, Y., Yu, X., Gonzales, F., Mangelsdorf, D. J., Wang, M. Y., Richardson, C., Witters, L. A. and Unger, R. H. (2002) PPAR α is necessary for the lipopenic action of hyperleptinemia on white adipose and liver tissue Proc Natl Acad Sci U S A 99, 11848-53.

Lee, S. S., Pineau, T., Drago, J., Lee, E. J., Owens, J. W., Kroetz, D. L., Fernandez-Salguero, P. M., Westphal, H. and Gonzalez, F. J. (1995) Targeted disruption of the α isoform of the peroxisome proliferator-activated receptor gene in mice results in abolishment of the pleiotropic effects of peroxisome proliferators Mol Cell Biol 15, 3012-22.

Lefebvre, P., Chinetti, G., Fruchart, J. C. and Staels, B. (2006) Sorting out the roles of PPAR α in energy metabolism and vascular homeostasis J Clin Invest 116, 571-80.

Leonardsson, G., Steel, J. H., Christian, M., Pocock, V., Milligan, S., Bell, J., So, P. W., Medina-Gomez, G., Vidal-Puig, A., White, R. and Parker,

www.nursa.org

Pleiotropic responses of PPARα

M. G. (2004) Nuclear receptor corepressor RIP140 regulates fat accumulation Proc Natl Acad Sci U S A 101, 8437-42.

Leone, T. C., Weinheimer, C. J. and Kelly, D. P. (1999) A critical role for the peroxisome proliferator-activated receptor α (PPARAlpha) in the cellular fasting response: the PPARalpha-null mouse as a model of fatty acid oxidation disorders Proc Natl Acad Sci U S A 96, 7473-8.

Li, L. O., Ellis, J. M., Paich, H. A., Wang, S., Gong, N., Altshuller, G., Thresher, R. J., Koves, T. R., Watkins, S. M., Muoio, D. M., Cline, G. W., Shulman, G. I. and Coleman, R. A. (2009) Liver-specific loss of long chain acyl-CoA synthetase-1 decreases triacylglycerol synthesis and β-oxidation and alters phospholipid fatty acid composition J Biol Chem 284, 27816-26.

Lonard, D. M. and O'Malley, B. W. (2006) The expanding cosmos of nuclear receptor coactivators Cell 125, 411-4.

Maloney, E. K. and Waxman, D. J. (1999) trans-Activation of PPARAlpha and PPARgamma by structurally diverse environmental chemicals Toxicol Appl Pharmacol 161, 209-18.

Mangelsdorf, D. J., Thummel, C., Beato, M., Herrlich, P., Schutz, G., Umesono, K., Blumberg, B., Kastner, P., Mark, M., Chambon, P. and Evans, R. M. (1995) The nuclear receptor superfamily: the second decade Cell 83, 835-9.

Martens, K., Ver Loren van Themaat, E., van Batenburg, M. F., Heinaniemi, M., Huyghe, S., Van Hummelen, P., Carlberg, C., Van Veldhoven, P. P., Van Kampen, A. and Baes, M. (2008) Coordinate induction of PPAR α and SREBP2 in multifunctional protein 2 deficient mice Biochim Biophys Acta 1781, 694-702.

Martin, G., Poirier, H., Hennuyer, N., Crombie, D., Fruchart, J. C., Heyman, R. A., Besnard, P. and Auwerx, J. (2000) Induction of the fatty acid transport protein 1 and acyl-CoA synthase genes by dimer-selective rexinoids suggests that the peroxisome proliferator-activated receptor-retinoid X receptor heterodimer is their molecular target J Biol Chem 275, 12612-8.

Marx, N., Duez, H., Fruchart, J. C. and Staels, B. (2004) Peroxisome proliferator-activated receptors and atherogenesis: regulators of gene expression in vascular cells Circ Res 94, 1168-78.

Massip-Salcedo, M., Zaouali, M. A., Padrissa-Altes, S., Casillas-Ramirez, A., Rodes, J., Rosello-Catafau, J. and Peralta, C. (2008) Activation of peroxisome proliferator-activated receptor-α inhibits the injurious effects of adiponectin in rat steatotic liver undergoing ischemia-reperfusion Hepatology 47, 461-72.

Matsumoto, K., Yu, S., Jia, Y., Ahmed, M. R., Viswakarma, N., Sarkar, J., Kashireddy, P. V., Rao, M. S., Karpus, W., Gonzalez, F. J. and Reddy, J. K. (2007) Critical role for transcription coactivator peroxisome proliferator-activated receptor (PPAR)-binding protein/TRAP220 in liver regeneration and PPARAlpha ligand-induced liver tumor development J Biol Chem 282, 17053-60.

Matsumoto, K., Huang, J., Viswakarma, N., Bai, L., Jia, Y., Zhu, Y. T., Yang, G., Borensztajn, J., Rao, M. S., Zhu, Y. J. and Reddy, J. K. (2010) Transcription coactivator PBP/MED1-deficient hepatocytes are not susceptible to diethylnitrosamine-induced hepatocarcinogenesis in the mouse Carcinogenesis 31, 318-25.

Mei, C. L., He, P., Cheng, B., Liu, W., Wang, Y. F. and Wan, J. J. (2009) Chlamydia pneumoniae induces macrophage-derived foam cell formation via PPAR α and PPAR γ-dependent pathways Cell Biol Int 33, 301-8.

Meyer, K., Lee, J. S., Dyck, P. A., Cao, W. Q., Rao, M. S., Thorgeirsson, S. S. and Reddy, J. K. (2003) Molecular profiling of hepatocellular carcinomas developing spontaneously in acyl-CoA oxidase deficient mice: comparison with liver tumors induced in wild-type mice by a peroxisome proliferator and a genotoxic carcinogen Carcinogenesis 24, 975-84.

Michalik, L., Auwerx, J., Berger, J. P., Chatterjee, V. K., Glass, C. K., Gonzalez, F. J., Grimaldi, P. A., Kadowaki, T., Lazar, M. A., O'Rahilly, S., Palmer, C. N., Plutzky, J., Reddy, J. K., Spiegelman, B. M., Staels, B. and Wahli, W. (2006) International Union of Pharmacology. LXI. Peroxisome proliferator-activated receptors Pharmacol Rev 58, 726-41.

Review                                                                                      Pleiotropic responses of PPARα

Monecke, T., Dickmanns, A. and Ficner, R. (2009) Structural basis for m7G-cap hypermethylation of small nuclear, small nucleolar and telomerase RNA by the dimethyltransferase TGS1 Nucleic Acids Res 37, 3865-77.

Morimura, K., Cheung, C., Ward, J. M., Reddy, J. K. and Gonzalez, F. J. (2006) Differential susceptibility of mice humanized for peroxisome proliferator-activated receptor α to Wy-14,643-induced liver tumorigenesis Carcinogenesis 27, 1074-80.

Mukherjee, R., Locke, K. T., Miao, B., Meyers, D., Monshizadegan, H., Zhang, R., Search, D., Grimm, D., Flynn, M., O'Malley, K. M., Zhang, L., Li, J., Shi, Y., Kennedy, L. J., Blanar, M., Cheng, P. T., Tino, J. and Srivastava, R. A. (2008) Novel peroxisome proliferator-activated receptor α agonists lower low-density lipoprotein and triglycerides, raise high-density lipoprotein, and synergistically increase cholesterol excretion with a liver X receptor agonist J Pharmacol Exp Ther 327, 716-26.

Murakami, K., Bujo, H., Unoki, H. and Saito, Y. (2007) Effect of PPARAlpha activation of macrophages on the secretion of inflammatory cytokines in cultured adipocytes Eur J Pharmacol 561, 206-13.

Nagasawa, T., Inada, Y., Nakano, S., Tamura, T., Takahashi, T., Maruyama, K., Yamazaki, Y., Kuroda, J. and Shibata, N. (2006) Effects of bezafibrate, PPAR pan-agonist, and GW501516, PPARdelta agonist, on development of steatohepatitis in mice fed a methionine- and choline-deficient diet Eur J Pharmacol 536, 182-91.

Naito, H., Yamanoshita, O., Kamijima, M., Katoh, T., Matsunaga, T., Lee, C. H., Kim, H., Aoyama, T., Gonzalez, F. J. and Nakajima, T. (2006) Association of V227A PPARAlpha polymorphism with altered serum biochemistry and alcohol drinking in Japanese men Pharmacogenet Genomics 16, 569-77.

Nakajima, T., Kamijo, Y., Tanaka, N., Sugiyama, E., Tanaka, E., Kiyosawa, K., Fukushima, Y., Peters, J. M., Gonzalez, F. J. and Aoyama, T. (2004) Peroxisome proliferator-activated receptor α protects against alcohol-induced liver damage Hepatology 40, 972-80.

Nemali, M. R., Usuda, N., Reddy, M. K., Oyasu, K., Hashimoto, T., Osumi, T., Rao, M. S. and Reddy, J. K. (1988) Comparison of constitutive and inducible levels of expression of peroxisomal β-oxidation and catalase genes in liver and extrahepatic tissues of rat Cancer Res 48, 5316-24.

Ng, V. Y., Huang, Y., Reddy, L. M., Falck, J. R., Lin, E. T. and Kroetz, D. L. (2007) Cytochrome P450 eicosanoids are activators of peroxisome proliferator-activated receptor α Drug Metab Dispos 35, 1126-34.

Nielsen, E. M., Hansen, L., Echwald, S. M., Drivsholm, T., Borch-Johnsen, K., Ekstrom, C. T., Hansen, T. and Pedersen, O. (2003) Evidence for an association between the Leu162Val polymorphism of the PPARAlpha gene and decreased fasting serum triglyceride levels in glucose tolerant subjects Pharmacogenetics 13, 417-23.

Nielsen, R., Grontved, L., Stunnenberg, H. G. and Mandrup, S. (2006) Peroxisome proliferator-activated receptor subtype- and cell-type-specific activation of genomic target genes upon adenoviral transgene delivery Mol Cell Biol 26, 5698-714.

Nissen, S. E., Nicholls, S. J., Wolski, K., Howey, D. C., McErlean, E., Wang, M. D., Gomez, E. V. and Russo, J. M. (2007) Effects of a potent and selective PPAR-α agonist in patients with atherogenic dyslipidemia or hypercholesterolemia: two randomized controlled trials Jama 297, 1362-73.

Onate, S. A., Tsai, S. Y., Tsai, M. J. and O'Malley, B. W. (1995) Sequence and characterization of a coactivator for the steroid hormone receptor superfamily Science 270, 1354-7.

Osumi, T., Wen, J. K. and Hashimoto, T. (1991) Two cis-acting regulatory sequences in the peroxisome proliferator-responsive enhancer region of rat acyl-CoA oxidase gene Biochem Biophys Res Commun 175, 866-71.

Palmer, C. N., Hsu, M. H., Griffin, K. J., Raucy, J. L. and Johnson, E. F. (1998) Peroxisome proliferator activated receptor-α expression in human liver Mol Pharmacol 53, 14-22.

Panigrahy, D., Kaipainen, A., Huang, S., Butterfield, C. E., Barnes, C. M., Fannon, M., Laforme, A. M., Chaponis, D. M., Folkman, J. and Kieran,

M.W. (2008) PPARAlpha agonist fenofibrate suppresses tumor growth through direct and indirect angiogenesis inhibition Proc Natl Acad Sci U S A 105, 985-90.

Paradis, A. M., Fontaine-Bisson, B., Bosse, Y., Robitaille, J., Lemieux, S., Jacques, H., Lamarche, B., Tchernof, A., Couture, P. and Vohl, M. C. (2005) The peroxisome proliferator-activated receptor α Leu162Val polymorphism influences the metabolic response to a dietary intervention altering fatty acid proportions in healthy men Am J Clin Nutr 81, 523-30.

Patel, N. S., di Paola, R., Mazzon, E., Britti, D., Thiemermann, C. and Cuzzocrea, S. (2009) Peroxisome proliferator-activated receptor-α contributes to the resolution of inflammation after renal ischemia/reperfusion injury J Pharmacol Exp Ther 328, 635-43.

Paumelle, R., Blanquart, C., Briand, O., Barbier, O., Duhem, C., Woerly, G., Percevault, F., Fruchart, J. C., Dombrowicz, D., Glineur, C. and Staels, B. (2006) Acute antiinflammatory properties of statins involve peroxisome proliferator-activated receptor-α via inhibition of the protein kinase C signaling pathway Circ Res 98, 361-9.

Peters, J. M., Cheung, C. and Gonzalez, F. J. (2005) Peroxisome proliferator-activated receptor-α and liver cancer: where do we stand? J Mol Med 83, 774-85.

Peters, J. M., Cattley, R. C. and Gonzalez, F. J. (1997) Role of PPAR α in the mechanism of action of the nongenotoxic carcinogen and peroxisome proliferator Wy-14,643 Carcinogenesis 18, 2029-33.

Pineda Torra, I., Jamshidi, Y., Flavell, D. M., Fruchart, J. C. and Staels, B. (2002) Characterization of the human PPARAlpha promoter: identification of a functional nuclear receptor response element Mol Endocrinol 16, 1013-28.

Poleni, P. E., Bianchi, A., Etienne, S., Kufany, M., Sebillaud, S., Netter, P., Terlain, B. and Jouzeau, J. Y. (2007) Agonists of peroxisome proliferators-activated receptors (PPAR) α, β/δ or γ reduce transforming growth factor (TGF)-β-induced proteoglycans' production in chondrocytes Osteoarthritis Cartilage 15, 493-505.

Poynter, M. E. and Daynes, R. A. (1998) Peroxisome proliferator-activated receptor α activation modulates cellular redox status, represses nuclear factor-kappaB signaling, and reduces inflammatory cytokine production in aging J Biol Chem 273, 32833-41.

Qi, C., Zhu, Y., Pan, J., Usuda, N., Maeda, N., Yeldandi, A. V., Rao, M. S., Hashimoto, T. and Reddy, J. K. (1999a) Absence of spontaneous peroxisome proliferation in enoyl-CoA Hydratase/L-3-hydroxyacyl-CoA dehydrogenase-deficient mouse liver. Further support for the role of fatty acyl CoA oxidase in PPARAlpha ligand metabolism J Biol Chem 274, 15775-80.

Qi, C., Zhu, Y., Pan, J., Yeldandi, A. V., Rao, M. S., Maeda, N., Subbarao, V., Pulikuri, S., Hashimoto, T. and Reddy, J. K. (1999b) Mouse steroid receptor coactivator-1 is not essential for peroxisome proliferator-activated receptor α-regulated gene expression Proc Natl Acad Sci U S A 96, 1585-90.

Qi, C., Kashireddy, P., Zhu, Y. T., Rao, S. M. and Zhu, Y. J. (2004) Null mutation of peroxisome proliferator-activated receptor-interacting protein in mammary glands causes defective mammapoiesis J Biol Chem 279, 33696-701.

Ramanan, S., Kooshki, M., Zhao, W., Hsu, F. C. and Robbins, M. E. (2008) PPARAlpha ligands inhibit radiation-induced microglial inflammatory responses by negatively regulating NF-kappaB and AP-1 pathways Free Radic Biol Med 45, 1695-704.

Rao, M. S. and Reddy, J. K. (1996) Hepatocarcinogenesis of peroxisome proliferators Ann N Y Acad Sci 804, 573-87.

Rao, M. S. and Reddy, J. K. (1987) Peroxisome proliferation and hepatocarcinogenesis Carcinogenesis 8, 631-6.

Rao, M. S. and Reddy, J. K. (2004) PPARAlpha in the pathogenesis of fatty liver disease Hepatology 40, 783-6.

Review                                                                                       Pleiotropic responses of PPARα

Reddy, J. K. and Lalwani, N. D. (1983) Carcinogenesis by hepatic peroxisome proliferators: evaluation of the risk of hypolipidemic drugs and industrial plasticizers to humans *Crit Rev Toxicol* 12, 1-58.

Reddy, J. K., Reddy, M. K., Usman, M. I., Lalwani, N. D. and Rao, M. S. (1986a) Comparison of hepatic peroxisome proliferative effect and its implication for hepatocarcinogenicity of phthalate esters, di(2-ethylhexyl) phthalate, and di(2-ethylhexyl) adipate with a hypolipidemic drug *Environ Health Perspect* 65, 317-27.

Reddy, J. K., Lalwani, N. D., Reddy, M. K. and Qureshi, S. A. (1982) Excessive accumulation of autofluorescent lipofuscin in the liver during hepatocarcinogenesis by methyl clofenapate and other hypolipidemic peroxisome proliferators *Cancer Res* 42, 259-66.

Reddy, J. K. and Krishnakantha, T. P. (1975) Hepatic peroxisome proliferation: induction by two novel compounds structurally unrelated to clofibrate *Science* 190, 787-9.

Reddy, J. K., Rao, S. and Moody, D. E. (1976) Hepatocellular carcinomas in acatalasemic mice treated with nafenopin, a hypolipidemic peroxisome proliferator *Cancer Res* 36, 1211-7.

Reddy, J. K., Azarnoff, D. L. and Hignite, C. E. (1980) Hypolipidaemic hepatic peroxisome proliferators form a novel class of chemical carcinogens *Nature* 283, 397-8.

Reddy, J. K., Lalwani, N. D., Qureshi, S. A., Reddy, M. K. and Moehle, C. M. (1984) Induction of hepatic peroxisome proliferation in nonrodent species, including primates *Am J Pathol* 114, 171-83.

Reddy, J. K. and Rao, M. S. (2006a) Lipid metabolism and liver inflammation. II. Fatty liver disease and fatty acid oxidation *Am J Physiol Gastrointest Liver Physiol* 290, G852-8.

Reddy, J. K., Rao, M. S., Azarnoff, D. L. and Sell, S. (1979) Mitogenic and carcinogenic effects of a hypolipidemic peroxisome proliferator, [4-chloro-6-(2,3-xylidino)-2-pyrimidinylthio]acetic acid (Wy-14, 643), in rat and mouse liver *Cancer Res* 39, 152-61.

Reddy, J. K. (2001b) Nonalcoholic steatosis and steatohepatitis. III. Peroxisomal β-oxidation, PPAR α, and steatohepatitis *Am J Physiol Gastrointest Liver Physiol* 281, G1333-9.

Reddy, J. K., Guo, D., Jia, Y., Yu, S. and Rao, M.S. (2006b) Nuclear receptor transcriptional coactivators in development and metabolism *Advances in Developmental Biology* 16, 389-420.

Reddy, J. K. and Hashimoto, T. (2001a) Peroxisomal β-oxidation and peroxisome proliferator-activated receptor α: an adaptive metabolic system *Annu Rev Nutr* 21, 193-230.

Reddy, J. K. and Chu, R. (1996) Peroxisome proliferator-induced pleiotropic responses: pursuit of a phenomenon *Ann N Y Acad Sci* 804, 176-201.

Reddy, J. K. (2004) Peroxisome proliferators and peroxisome proliferator-activated receptor α: biotic and xenobiotic sensing *Am J Pathol* 164, 2305-21.

Reddy, J. K., Goel, S. K., Nemali, M. R., Carrino, J. J., Laffler, T. G., Reddy, M. K., Sperbeck, S. J., Osumi, T., Hashimoto, T., Lalwani, N. D. and , (1986b) Transcription regulation of peroxisomal fatty acyl-CoA oxidase and enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase in rat liver by peroxisome proliferators *Proc Natl Acad Sci USA* 83, 1747-51.

Ricote, M. and Glass, C. K. (2007) PPARs and molecular mechanisms of transrepression *Biochim Biophys Acta* 1771, 926-35.

Rigamonti, E., Chinetti-Gbaguidi, G. and Staels, B. (2008) Regulation of macrophage functions by PPAR-α, PPAR-γ, and LXRs in mice and men *Arterioscler Thromb Vasc Biol* 28, 1050-9.

Robitaille, J., Brouillette, C., Houde, A., Lemieux, S., Perusse, L., Tchernof, A., Gaudet, D. and Vohl, M. C. (2004) Association between the PPARAlpha-L162V polymorphism and components of the metabolic syndrome *J Hum Genet* 49, 482-9.

Rusyn, I., Asakura, S., Pachkowski, B., Bradford, B. U., Denissenko, M. F., Peters, J. M., Holland, S. M., Reddy, J. K., Cunningham, M. L. and Swenberg, J. A. (2004) Expression of base excision DNA repair genes is a sensitive biomarker for in vivo detection of chemical-induced chronic oxidative stress: identification of the molecular source of radicals responsible for DNA damage by peroxisome proliferators *Cancer Res* 64, 1050-7.

Sanderson, L. M., Degenhardt, T., Koppen, A., Kalkhoven, E., Desvergne, B., Muller, M. and Kersten, S. (2009) Peroxisome proliferator-activated receptor β/δ (PPARbeta/δ) but not PPARalpha serves as a plasma free fatty acid sensor in liver *Mol Cell Biol* 29, 6257-67.

Sapone, A., Peters, J. M., Sakai, S., Tomita, S., Papiha, S. S., Dai, R., Friedman, F. K. and Gonzalez, F. J. (2000) The human peroxisome proliferator-activated receptor α gene: identification and functional characterization of two natural allelic variants *Pharmacogenetics* 10, 321-33.

Sarkar, J., Qi, C., Guo, D., Ahmed, M. R., Jia, Y., Usuda, N., Viswakarma, N., Rao, M. S. and Reddy, J. K. (2007) Transcription coactivator PRIP, the peroxisome proliferator-activated receptor (PPAR)-interacting protein, is redundant for the function of nuclear receptors PParalpha and CAR, the constitutive androstane receptor, in mouse liver *Gene Expr* 13, 255-69.

Schwartz, C. J., Valente, A. J., Sprague, E. A., Kelley, J. L., Suenram, C. A. and Rozek, M. M. (1985) Atherosclerosis as an inflammatory process. The roles of the monocyte-macrophage *Ann N Y Acad Sci* 454, 115-20.

Seedorf, U., Raabe, M., Ellinghaus, P., Kannenberg, F., Fobker, M., Engel, T., Denis, S., Wouters, F., Wirtz, K. W., Wanders, R. J., Maeda, N. and Assmann, G. (1998) Defective peroxisomal catabolism of branched fatty acyl coenzyme A in mice lacking the sterol carrier protein-2/sterol carrier protein-x gene function *Genes Dev* 12, 1189-201.

Shah, Y. M., Morimura, K., Yang, Q., Tanabe, T., Takagi, M. and Gonzalez, F. J. (2007) Peroxisome proliferator-activated receptor α regulates a microRNA-mediated signaling cascade responsible for hepatocellular proliferation *Mol Cell Biol* 27, 4238-47.

Shalev, A., Siegrist-Kaiser, C. A., Yen, P. M., Wahli, W., Burger, A. G., Chin, W. W. and Meier, C. A. (1996) The peroxisome proliferator-activated receptor α is a phosphoprotein: regulation by insulin *Endocrinology* 137, 4499-502.

Shang, Y., Hu, X., DiRenzo, J., Lazar, M. A. and Brown, M. (2000) Cofactor dynamics and sufficiency in estrogen receptor-regulated transcription *Cell* 103, 843-52.

Shan, W., Palkar, P. S., Murray, I. A., McDevitt, E. I., Kennett, M. J., Kang, B. H., Isom, H. C., Perdew, G. H., Gonzalez, F. J. and Peters, J. M. (2008b) Ligand activation of peroxisome proliferator-activated receptor β/δ (PPARbeta/δ) attenuates carbon tetrachloride hepatotoxicity by downregulating proinflammatory gene expression *Toxicol Sci* 105, 418-28.

Shan, W., Nicol, C. J., Ito, S., Bility, M. T., Kennett, M. J., Ward, J. M., Gonzalez, F.J. and Peters, J. M. (2008a) Peroxisome proliferator-activated receptor-β/δ protects against chemically induced liver toxicity in mice *Hepatology* 47, 225-35.

Sher, T., Yi, H. F., McBride, O. W. and Gonzalez, F. J. (1993) cDNA cloning, chromosomal mapping, and functional characterization of the human peroxisome proliferator activated receptor *Biochemistry* 32, 5598-604.

Sozio, M. and Crabb, D. W. (2008) Alcohol and lipid metabolism *Am J Physiol Endocrinol Metab* 295, E10-6.

Staels, B., Maes, M. and Zambon, A. (2008) Fibrates and future PPARAlpha agonists in the treatment of cardiovascular disease *Nat Clin Pract Cardiovasc Med* 5, 542-53.

Still, K., Grabowski, P., Mackie, I., Perry, M. and Bishop, N. (2008) The peroxisome proliferator activator receptor α/δ agonists linoleic acid and bezafibrate upregulate osteoblast differentiation and induce periosteal bone formation in vivo *Calcif Tissue Int* 83, 285-92.

Surapureddi, S., Yu, S., Bu, H., Hashimoto, T., Yeldandi, A. V., Kashireddy, P., Cherkaoui-Malki, M., Qi, C., Zhu, Y. J., Rao, M. S. and 

Review                                                                                      Pleiotropic responses of PPARα

Reddy, J. K. (2002) Identification of a transcriptionally active peroxisome proliferator-activated receptor α-interacting cofactor complex in rat liver and characterization of PRIC285 as a coactivator Proc Natl Acad Sci U S A 99, 11836-41.

Tanaka, N., Moriya, K., Kiyosawa, K., Koike, K., Gonzalez, F. J. and Aoyama, T. (2008) PPARalpha activation is essential for HCV core protein-induced hepatic steatosis and hepatocellular carcinoma in mice J Clin Invest 118, 683-94.

Tontonoz, P., Hu, E. and Spiegelman, B. M. (1994) Stimulation of adipogenesis in fibroblasts by PPAR γ 2, a lipid-activated transcription factor Cell 79, 1147-56.

Tugwood, J. D., Issemann, I., Anderson, R. G., Bundell, K. R., McPheat, W. L. and Green, S. (1992) The mouse peroxisome proliferator activated receptor recognizes a response element in the 5' flanking sequence of the rat acyl CoA oxidase gene Embo J 11, 433-9.

van der Meer, D. L., Degenhardt, T., Vaisanen, S., de Groot, P. J., Heinaniemi, M., de Vries, S. C., Muller, M., Carlberg, C. and Kersten, S. (2010) Profiling of promoter occupancy by PPAR{α} in human hepatoma cells via ChIP-chip analysis Nucleic Acids Res, (in press).

Varanasi, U., Chu, R., Huang, Q., Castellon, R., Yeldandi, A. V. and Reddy, J. K. (1996) Identification of a peroxisome proliferator-responsive element upstream of the human peroxisomal fatty acyl coenzyme A oxidase gene J Biol Chem 271, 2147-55.

Viswakarma, N., Matsumoto, K., Jia, Y., Rao, M. S. and Reddy, J. K. (2009) Mice lacking transcription cofactor PRIC285 reveal attenuation of liver regeneration but are viable and develop normally FASEB J 23, 117.

Voegel, J. J., Heine, M. J., Tini, M., Vivat, V., Chambon, P. and Gronemeyer, H. (1998) The coactivator TIF2 contains three nuclear receptor-binding motifs and mediates transactivation through CBP binding-dependent and -independent pathways Embo J 17, 507-19.

Vohl, M. C., Lepage, P., Gaudet, D., Brewer, C. G., Betard, C., Perron, P., Houde, G., Cellier, C., Faith, J. M., Despres, J. P., Morgan, K. and Hudson, T. J. (2000) Molecular scanning of the human PPARα gene: association of the L162v mutation with hyperapobetalipoproteinemia J Lipid Res 41, 945-52.

Walgren, J. E., Kurtz, D. T. and McMillan, J. M. (2000) Expression of PPAR(α) in human hepatocytes and activation by trichloroacetate and dichloroacetate Res Commun Mol Pathol Pharmacol 108, 116-32.

Wang, Z., Qi, C., Krones, A., Woodring, P., Zhu, X., Reddy, J. K., Evans, R. M., Rosenfeld, M. G. and Hunter, T. (2006) Critical roles of the p160 transcriptional coactivators p/CIP and SRC-1 in energy balance Cell Metab 3, 111-22.

Wang, Y., Feng, W., Xue, W., Tan, Y., Hein, D. W., Li, X. K. and Cai, L. (2009) Inactivation of GSK-3beta by metallothionein prevents diabetes-related changes in cardiac energy metabolism, inflammation, nitrosative damage, and remodeling Diabetes 58, 1391-402.

Wang, Y. X., Lee, C. H., Tiep, S., Yu, R. T., Ham, J., Kang, H. and Evans, R. M. (2003) Peroxisome-proliferator-activated receptor δ activates fat metabolism to prevent obesity Cell 113, 159-70.

White, J. H., Fernandes, I., Mader, S. and Yang, X. J. (2004) Corepressor recruitment by agonist-bound nuclear receptors Vitam Horm 68, 123-43.

World Health Organization. (2006) Obesity and Overweight http://www.who.int/mediacentre/factsheets/fs311/en/index.html http://www.who.int/mediacentre/factsheets/fs311/en/index.html

Xu, H. E., Lambert, M. H., Montana, V. G., Plunket, K. D., Moore, L. B., Collins, J. L., Oplinger, J. A., Kliewer, S. A., Gampe, R. T., Jr., McKee, D. D., Moore, J. T. and Willson, T. M. (2001) Structural determinants of ligand binding selectivity between the peroxisome proliferator-activated receptors Proc Natl Acad Sci U S A 98, 13919-24.

Yang, Q., Nagano, T., Shah, Y., Cheung, C., Ito, S. and Gonzalez, F. J. (2008) The PPAR α-humanized mouse: a model to investigate species differences in liver toxicity mediated by PPAR α Toxicol Sci 101, 132-9.

Yeldandi, A. V., Rao, M. S. and Reddy, J. K. (2000) Hydrogen peroxide generation in peroxisome proliferator-induced oncogenesis Mutat Res 448, 159-77.

Yu, S., Matsusue, K., Kashireddy, P., Cao, W. Q., Yeldandi, V., Yeldandi, A. V., Rao, M. S., Gonzalez, F. J. and Reddy, J. K. (2003) Adipocyte-specific gene expression and adipogenic steatosis in the mouse liver due to peroxisome proliferator-activated receptor gamma1 (PPARgamma1) overexpression J Biol Chem 278, 498-505.

Yu, S., Cao, W. Q., Kashireddy, P., Meyer, K., Jia, Y., Hughes, D. E., Tan, Y., Feng, J., Yeldandi, A. V., Rao, M. S., Costa, R. H., Gonzalez, F. J. and Reddy, J. K. (2001) Human peroxisome proliferator-activated receptor α (PPARAlpha) supports the induction of peroxisome proliferation in PPARAlpha-deficient mouse liver J Biol Chem 276, 42485-91.

Yu, S. and Reddy, J. K. (2007) Transcription coactivators for peroxisome proliferator-activated receptors Biochim Biophys Acta 1771, 936-51.

Zafrani, E. S. (2004) Non-alcoholic fatty liver disease: an emerging pathological spectrum Virchows Arch 444, 3-12.

Zandbergen, F. and Plutzky, J. (2007) PPARAlpha in atherosclerosis and inflammation Biochim Biophys Acta 1771, 972-82.

Zhang, B., Marcus, S. L., Sajjadi, F. G., Alvares, K., Reddy, J. K., Subramani, S., Rachubinski, R. A. and Capone, J. P. (1992) Identification of a peroxisome proliferator-responsive element upstream of the gene encoding rat peroxisomal enoyl-CoA hydratase/3-hydroxyacyl-CoA dehydrogenase Proc Natl Acad Sci U S A 89, 7541-5.

Zhu, Y., Qi, C., Calandra, C., Rao, M. S. and Reddy, J. K. (1996) Cloning and identification of mouse steroid receptor coactivator-1 (mSRC-1), as a coactivator of peroxisome proliferator-activated receptor γ Gene Expr 6, 185-95.

Zhu, Y., Qi, C., Jia, Y., Nye, J. S., Rao, M. S. and Reddy, J. K. (2000b) Deletion of PBP/PPARBP, the gene for nuclear receptor coactivator peroxisome proliferator-activated receptor-binding protein, results in embryonic lethality J Biol Chem 275, 14779-82.

Zhu, Y., Kan, L., Qi, C., Kanwar, Y. S., Yeldandi, A. V., Rao, M. S. and Reddy, J. K. (2000a) Isolation and characterization of peroxisome proliferator-activated receptor (PPAR) interacting protein (PRIP) as a coactivator for PPAR J Biol Chem 275, 13510-6.

Zhu, Y., Qi, C., Korenberg, J. R., Chen, X. N., Noya, D., Rao, M. S. and Reddy, J. K. (1995) Structural organization of mouse peroxisome proliferator-activated receptor γ (mPPAR γ) gene: alternative promoter use and different splicing yield two mPPAR γ isoforms Proc Natl Acad Sci U S A 92, 7921-5.

Zoete, V., Grosdidier, A. and Michielin, O. (2007) Peroxisome proliferator-activated receptor structures: ligand specificity, molecular switch and interactions with regulators Biochim Biophys Acta 1771, 915-25.
