
APPLIED AND ENVIRONMENTAL MICROBIOLOGY, June 2010, p. 3869–3877
0099-2240/10/$12.00 doi:10.1128/AEM.03083-09
Copyright © 2010, American Society for Microbiology. All Rights Reserved.

**TDP-L-Megosamine Biosynthesis Pathway Elucidation and Megalomicin A Production in *Escherichia coli***

Mariana Useglio,$^{1\dagger}$ Salvador Peirú,$^{1\dagger}$ Eduardo Rodríguez,$^{1*}$ Guillermo R. Labadie,$^{2}$ John R. Carney,$^{3\#}$ and Hugo Gramajo$^{1}$

Microbiology Division, IBR (Instituto de Biología Molecular y Celular de Rosario), Consejo Nacional de Investigaciones Científicas y Técnicas, Facultad de Ciencias Bioquímicas y Farmacéuticas, Universidad Nacional de Rosario, Suipacha 531, S2002LRK Rosario, Argentina$^{1}$; IQUIOS (Instituto de Química Orgánica de Síntesis, Departamento de Química Orgánica), Facultad de Ciencias Bioquímicas y Farmacéuticas, Universidad Nacional de Rosario, Suipacha 531, S2002LRK Rosario, Argentina$^{2}$; and Kosan Biosciences, Inc., 3832 Bay Center Place, Hayward, California 94545$^{3}$

Received 21 December 2009/Accepted 16 April 2010

*In vivo* reconstitution of the TDP-L-megosamine pathway from the megalomicin gene cluster of *Micromonospora megalomicaea* was accomplished by the heterologous expression of its biosynthetic genes in *Escherichia coli*. Mass spectrometric analysis of the TDP-sugar intermediates produced from operons containing different sets of genes showed that the production of TDP-L-megosamine from TDP-4-keto-6-deoxy-D-glucose requires only five biosynthetic steps, catalyzed by MegBVI, MegDII, MegDIII, MegDIV, and MegDV. Bioconversion studies demonstrated that the sugar transferase MegDI, along with the helper protein MegDVI, catalyzes the transfer of L-megosamine to either erythromycin C or erythromycin D, suggesting two possible routes for the production of megalomicin A. Analysis *in vivo* of the hydroxylation step by MegK indicated that erythromycin C is the intermediate of megalomicin A biosynthesis.

Most of the deoxy sugars found in natural products belong to the 6-deoxyhexose (6DOH) family (21). Since many of these 6DOHs are essential for the bioactivity of natural compounds, extensive efforts have been made to investigate the relevant genetics, enzymology, and mechanistic features of the biosynthetic pathways leading to these sugars. The amino sugar L-megosamine is found within a family of macrolide compounds produced by the actinomycete *Micromonospora megalomicaea*, named megalomicins A (MegA) (structure 1), B, C1, and C2 (Fig. 1A) (27). These compounds consist of a 14-membered macrolactone ring carrying three deoxy sugar residues, L-mycorose, D-desosamine, and L-megosamine. The megalomicin congeners differ from each other in the specific acetyl or propionyl groups attached at the 3‴ or 4‴ hydroxyls of the mycarose moiety. These macrolides were originally discovered as antibacterial agents which inhibit protein synthesis through selective binding to the bacterial 50S ribosomal subunit in a mode similar to that of erythromycins and other macrolides (25). Due to the similarities with erythromycin in terms of structure, antibacterial activity, and pharmacological properties, megalomicins did not receive much attention until antiviral and antiparasitic activities of these compounds were reported (1, 3). These studies demonstrated that megalomicins interfere with protein trafficking, resulting in an anomalous protein glycosylation (4, 5) that affects the maturation of enveloped viruses, including herpes simplex virus, Semliki Forest virus, vesicular stomatitis virus, and more importantly the human immunodeficiency virus type 1 (HIV-1) (1, 22). In HIV replication, inhibition of gp160 protein processing to gp120 and gp41 resulted in noninfectious virions (22). In addition, megalomicins also showed antiparasitic activity against the epimastigote stage of *Trypanosoma cruzi*, *Leishmania* spp., and *Plasmodium falciparum*, although in this case the mechanism of action still remains unclear (3).

The main structural difference between megalomicins and erythromycins is the presence of the L-megosamine sugar moiety at C-6 (Fig. 1A). Since erythromycin does not exhibit antiparasitic and antiviral activities, the presence of this additional amino sugar in megalomicins could be associated with the differential properties of these compounds (1, 3). Due to the potential pharmacological relevance of megalomicins and the lack of a detailed characterization of the L-megosamine biosynthetic pathway from the megalomicin (*meg*) gene cluster, an in-depth metabolic route study was deemed warranted.

Analysis of the overall organization of the *meg* gene cluster revealed that L-megosamine biosynthesis genes are grouped together within this gene cluster (Fig. 1B) (25). This was demonstrated by the heterologous expression of a 12-kb DNA fragment that included the putative megosamine biosynthesis genes in the erythromycin producer strain *Saccharopolyspora erythraea*, which allowed the production of megalomicins in this host (25). Six biosynthetic steps were proposed for the biosynthesis of TDP-L-megosamine (L-Meg) (structure 2) from the intermediate TDP-4-keto-6-deoxy-D-glucose (TKDG) (structure 3). Neither the biosynthesis pathway nor the enzymes involved in each catalytic step have been confirmed.

Herein, the investigation focused on the biosynthesis of L-Meg.

3870 USEGLIO ET AL.

APPL. ENVIRON. MICROBIOL.

A

![Diagram](https://i.imgur.com/1234567.png)

Megalomycin A (R₁=R₂=H) (1)
B (R₁=COCH₃, R₂=H)
C1 (R₁=COCH₃, R₂=COCH₃)
C2 (R₁=COCH₂CH₃, R₂=COCH₃)

Erythromycin A (R₃=OH, R₄=CH₃)
B (R₃=H, R₄=CH₃)
C (R₃=OH, R₄=H) (11)
D (R₃=H, R₄=H) (12)

B

![Diagram](https://i.imgur.com/8901234.png)

FIG. 1. (A) Structures of megalomicins and erythromycins. (B) Genetic organization of the meg gene cluster from M. megalomicea. A 12-kb fragment, including putative L-megosamine biosynthesis genes, is indicated.

from M. megalomicea by the heterologous expression of meg genes in Escherichia coli. The sequence of enzymatic reactions implicated in this pathway was confirmed by analyzing the TDP-sugar intermediates generated from the expression of operons containing different sets of genes. This methodology allowed the validation of a new pathway for the biosynthesis of L-Meg from the precursor TKDG through the use of five enzymatic steps. Bioconversion experiments furthermore demonstrated that the attachment of L-megosamine to the macrolide intermediate required both a specific glycosyltransferase and a helper protein.

MATERIALS AND METHODS

Bacterial strains, plasmids, and growth media. The bacterial strains and plasmids used in this study are shown in Table 1. Luria-Bertani (LB) medium was used for the growth of E. coli strains. The following antibiotics were added to the medium when necessary: kanamycin (50 μg/ml) and chloramphenicol (20 μg/ml).

TABLE 1. Strains and plasmids used in this study

| Strain or plasmid | Relevant genotype or description$^a$ | Source or reference |
|-------------------|---------------------------------------|---------------------|
| Strains           |                                       |                     |
| DH5α              | lacZΔM15 recA1                       | Promega             |
| LB19b             | K207-3 ΔrmlCwecDEvioABwzxacrAB        | 18                  |
| Plasmids          |                                       |                     |
| pET24b            | E. coli expression vector, ColE1 ori, kan | Novagen             |
| pLB29             | megBVI in pET28a, kan                | 17                  |
| pLB 132           | megK in pET28a, kan                  | 17                  |
| pKOS506-72B       | megCIV-megCV-megDIII-megCIII-ermE in PCDF1b, str | 20                  |
| pM1               | megBVI-megDII-megDIII in pET28a, kan | This work           |
| pM3               | megDVI in pET28a, kan                | This work           |
| pM9               | megBVI-megDII-megDIII-megDIVmegDVI-megDV-megDI in pET28a, kan | This work           |
| pM19              | megBVI-megDII in pET28akan           | This work           |
| pM20              | megBVI-megDII-megDIII-megDIV in pET28a, kan | This work           |
| pM21              | megBVI-megDII-megDIII-megDIV-megDV in pET28a, kan | This work           |
| pM22              | megBVI-megDII-megDIII-megDIV-megBIV in pET28a, kan | This work           |
| pM30              | megBVI-megDII-megDIII-megDIV-megDV-megDVI-ermE in pET28a, kan | This work           |
| pM31              | megBVI-megDII-megDIII-megDIV-megDV-megDI-ermE in pET28a, kan | This work           |
| pM32              | megBVI-megDII-megDIII-megDIV-megBIV-megDI-megCII-ermE in pET28a, kan | This work           |
| pM34              | megBVI-megDII-megDIII-megDIV-megDV-megCIII-megDVI-ermE in pET28a, kan | This work           |
| pGro7             | pBAD groES-groEL in pACYC184, cat     | Takara              |

$^a$ Abbreviations: kan, kanamycin resistance gene; str, streptomycin resistance gene; cat, chloramphenicol acetyltransferase gene.

DNA manipulation. DNA restriction enzymes were used as recommended by the manufacturer (New England Biolabs). Standard protocols were used for the recombinant DNA techniques (21a). DNA fragments were purified from agarose gels using GFX PCR DNA and gel band purification kits (GE Healthcare). Plasmids were prepared using a QIAprep Spin Miniprep kit (Qiagen). Deep Vent DNA polymerase was used in all PCRs according to the supplier's instructions (New England Biolabs).

Plasmid constructions. Three specific genes from the L-megosamine biosynthetic pathway were amplified by PCR using *M. megalomicea* genomic DNA as a template. Each PCR product was cloned into PCR-BluntII-TOPO (Invitrogen) and sequenced to confirm that it was free of errors. The 5′ primers used were designed to have an NdeI site overlapping the translational initiation codon, changing GTG start codons to ATG when required. The 3′ primers contained EcoRI and SpeI sites downstream of the stop codon. The following oligonucleotides were used to clone the different *meg* genes: 5′ GCATATGGTGGGTGC CGGCGCGTCGGGTTTC 3′ (upper) and 5′ CGAATTCACTAGTCAGGAG GGCTCGGACGGGGCGGC 3′ (lower) for *megDV*; 5′ ACATATGCGCGGTC GTTTTTCATCGATGGC 3′ (upper) and 5′ TGAATTCACTAGTCATCGC GGCAGGTGCGGCTCGGC 3′ (lower) for *megDI*; and 5′ TCATATGGCAG TTGGCGATCGAAGGCGGCT 3′ (upper) and 5′ TGAATTCACTAGTCTA CAGCTCGACGGGCAACGGCT 3′ (lower) for *megDV*. PCRs were carried out using a DNA thermal cycler 480 (Perkin-Elmer) with the following cycling parameters: 30 cycles of 30 s at 94°C, 30 s at 56°C, and 80 s at 72°C. The PCR products were digested with NdeI and EcoRI and cloned into identical sites of the pET24b or pET28a vector in order to express proteins with their natural N termini or as His tag fusions, respectively.

Megosamine operons were constructed by recursive ligation of XbaI/HindIII fragments carrying downstream *meg* genes into the SpeI and HindIII sites of vectors carrying one or more of the upstream genes of the operon being constructed. Ligation of the XbaI and SpeI sites inserted a 43-bp sequence with an appropriately positioned ribosome binding site and destroyed both sites to allow the next round of ligation. The process was repeated in a recursive fashion until the entire operon containing all genes in tandem was constructed. Plasmids containing the different sets of genes are shown in Table 1.

To incorporate the erythromycin resistance gene, *ermE*, into megosamine operons, an XbaI/EcoRI DNA fragment from the vector pKOS342-96 was cloned into the acceptor megosamine operons digested with SpeI/EcoRI to give the final plasmids listed in Table 1.

TDP-sugar analysis of cell extracts. *E. coli* strains were grown at 37°C in shake flasks in LB medium and in the presence of the corresponding antibiotics for plasmid maintenance if required. Overnight cultures were diluted 1:100 in fresh medium and grown to an optical density at 600 nm (*OD*~600~) of 0.6 before the addition of 2 mg/ml L-arabinose and 0.5 mM isopropyl-β-D-thiogalactopyranoside (IPTG) when needed. Induction was allowed to proceed for 24 h at 23°C. The cells were harvested, resuspended in 20 mM Tris buffer (pH 7.6), and disrupted by sonication. After centrifugation at 15,000 × *g* for 20 min, the supernatants were analyzed by liquid chromatography-tandem mass spectrometry (LC/MS/MS) for detection of TDP-sugars, as described previously (20).

Bioconversion experiments. *E. coli* strains harboring pGro7 and the different expression plasmids were cultured overnight at 37°C in LB with appropriate antibiotics and then subcultured by 1:100 dilutions in the same medium and grown to an *OD*~600~ of 0.6. Chaperones and sugar gene expression were induced by addition of 2 mg/ml L-arabinose and 0.5 mM IPTG, respectively, and cultures were supplemented with 20 mg/ml of 3-α-mycarosyl-erythronolide B, erythromycin C, or erythromycin D and incubated at 23°C for 72 h. Cultures were clarified by centrifugation at 5,000 × *g* for 10 min, their pH values were adjusted to 9.5, and they were extracted with an equal volume of ethyl acetate. The organic layer was separated and concentrated under vacuum, and the presence of bioconversion products was further analyzed by thin-layer chromatography (TLC) in a solvent system consisting of ethyl acetate, ethanol, and ammonia in a ratio of 80:15:5.

Thin-layer chromatography. All bioconversions were monitored by TLC performed on silica gel 60 F~254~ precoated aluminum sheets (Merck), visualized by a 254-nm UV lamp, and stained with an ethanolic solution of para-anisaldehyde.

Product purification. Compounds were purified by column flash chromatography using silica gel 60 (230 to 400 mesh; Merck) by isocratic elution with a solvent system consisting of hexane, ethyl acetate, and triethylamine in a ratio of 10:10:1.

NMR experiments. Nuclear magnetic resonance (NMR) spectra were acquired at 300 MHz for ^1^H and at 75 MHz for ^13^C on samples dissolved on CDCl~3~ with tetramethylsilane (TMS) for ^1^H and chloroform *d* for ^13^C as the internal reference.

---

LC-HRMS. Liquid chromatography high-resolution mass spectrometry (LC-HRMS) was recorded at UMYMFOR (University of Buenos Aires) on a Bruker micrOTOF-Q II mass spectrometer.

---

**RESULTS**

Analysis of TDP–L-megosamine biosynthesis pathway. Like most 6DOHs, L-Meg is proposed to be synthesized from glucose-1-phosphate via the common intermediate TKDG (Fig. 2A). This intermediate is also the biosynthetic precursor of both TDP–L-mycarose (L-Myc) (structure 4) and TDP–D-desosamine (D-Des) (structure 5), and its synthesis in *M. megalomicea* is catalyzed by the enzymes MegL and MegM (17). Based on previous knowledge of the biosynthesis of L-Myc and D-Des (17, 19, 20) and on the amino acid sequence similarity of the putative *meg* biosynthesis enzymes to other well-characterized 6DOH biosynthetic proteins, the following pathway is proposed for the biosynthesis of L-Meg in *M. megalomicea* from the common intermediate TKDG (Fig. 2B). This biosynthetic route for L-Meg presents several differences from that previously suggested by Volchegursky and coworkers (25).

The proposed biosynthetic route of L-Meg involves a 2,3-dehydration of TKDG, followed by a 3-transamination, in a sequence of reactions catalyzed by MegBVI and MegDII, respectively. MegBVI is the only 2,3-dehydratase present in the *meg* cluster, and it was demonstrated that this enzyme is involved in the C-2 deoxygenation step during the biosynthesis of L-Myc, generating the intermediate TDP–2,6-dideoxy-3,4-diketo-D-glucose (structure 6) (19) (Fig. 2A). On the other hand, MegDII is a 3-aminotransferase that catalyzes the transfer of an amino group to TDP–3-keto-4,6-dideoxy-D-glucose (structure 7) during the biosynthesis of D-Des (20) (Fig. 2A). MegDII also shares a high protein sequence similarity to EvaB from *Amycolatopsis orientalis* (80% similar and 70% identical), which is a 3-aminotransferase that converts TDP–2,6-dideoxy-3,4-diketo-D-glucose into TDP–3-amino-2,3,6-trideoxy-4-keto-D-glucose (structure 8) during the biosynthesis of TDP–L-epivancosamine in this strain (8). Based on these observations, a 2,3-dehydration/3-transamination sequential mechanism catalyzed by MegBVI/MegDII is proposed to be similar to that described for EvaA/EvaB.

After the transamination mediated by MegDII, the next biosynthetic step proposed is an N,N dimethylation of TDP–3-amino-2,3,6-trideoxy-4-keto-D-glucose catalyzed by MegDIII, the only protein encoded in the *meg* gene cluster with significant similarity to *N*-methyltransferases. This enzyme has also been demonstrated to be involved in the biosynthesis of D-Des, although using a different substrate (17) (Fig. 2A).

The final steps proposed for the biosynthesis of L-Meg involve a 5-epimerization reaction, followed by a 4-ketoreduction (Fig. 2B). This epimerization step is believed to be catalyzed by MegDIV, the only 5-epimerase present in the *meg* cluster, which also participates in the biosynthesis of L-Myc (Fig. 2A) (17). Two proteins sharing similarity to 4-ketoreductases, MegBIV and MegDV, were also found to be encoded in the *meg* gene cluster. Since MegBIV was previously demonstrated to take part in the biosynthesis of L-Myc (Fig. 2A) (17) and *megDV* is located in the megosamine “island” of the *meg* cluster (Fig. 1B), the final reductive step in the biosynthesis of L-Meg is expected to be mediated by MegDV.

A Glu-1-P

$\begin{aligned}
& \text{HOH}_{2} \text{C} \\
& \text{HOH}_{2} \text{C} \\
& \text{OH} \\
& \text{OPO}_{3}^{-2}
\end{aligned}$

$\xrightarrow{\text{MegL}}$

$\begin{aligned}
& \text{HOH}_{2} \text{C} \\
& \text{HOH}_{2} \text{C} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegM}}$

$\begin{aligned}
& \text{O} \\
& \text{H}_{3} \text{C} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegBVI}}$

$\left[\begin{array}{c}
\text{O} \\
\text{H}_{3} \text{C} \\
\text{OH} \\
\text{OTDP}
\end{array}\right.$

$\leftrightarrow$

$\left.\begin{array}{c}
\text{O} \\
\text{H}_{3} \text{C} \\
\text{O} \\
\text{OTDP}
\end{array}\right]$

6

$\xrightarrow{\text{MegCIV}}$

$\begin{aligned}
& \text{H}_{2} \text{N} \\
& \text{CH}_{3} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegCV}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{O} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

7

$\xrightarrow{\text{MegDII}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{O} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegDIII}}$

$\begin{aligned}
& (\text{H}_{3} \text{C})_{2} \text{N} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

D-Des (5)

$\xrightarrow{\text{MegBIa}}$

$\begin{aligned}
& \text{O} \\
& \text{H}_{3} \text{C} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegBIII}}$

$\begin{aligned}
& \text{O} \\
& \text{H}_{3} \text{C} \\
& \text{CH}_{3} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegDIV}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegBIV}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{OH} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

L-Myc (4)

$\xrightarrow{\text{MegBIb}}$

$\left[\begin{array}{c}
\text{H}_{3} \text{C} \\
\text{O} \\
\text{HO} \\
\text{OTDP}
\end{array}\right.$

$\leftrightarrow$

$\left.\begin{array}{c}
\text{H}_{3} \text{C} \\
\text{O} \\
\text{O} \\
\text{OTDP}
\end{array}\right]$

$\xrightarrow{\text{MegDV}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{O} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegDVI}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{HO} \\
& \text{O} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegDII}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{NH}_{2} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

10

$\xrightarrow{\text{MegDIII}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{N}(\text{CH}_{3})_{2} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

L-Meg (2)

B

$\begin{aligned}
& \text{O} \\
& \text{H}_{3} \text{C} \\
& \text{OH} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xrightarrow{\text{MegBVI}}$

$\left[\begin{array}{c}
\text{O} \\
\text{H}_{3} \text{C} \\
\text{OH} \\
\text{OTDP}
\end{array}\right.$

$\leftrightarrow$

$\left.\begin{array}{c}
\text{O} \\
\text{H}_{3} \text{C} \\
\text{O} \\
\text{OTDP}
\end{array}\right]$

$\xrightarrow{\text{MegDII}}$

$\begin{aligned}
& \text{H}_{2} \text{N} \\
& \text{H}_{3} \text{C} \\
& \text{OTDP}
\end{aligned}$

8

TKDG (3)

6

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{N}(\text{CH}_{3})_{2} \\
& \text{OH} \\
& \text{OTDP}
\end{aligned}$

$\xleftarrow{\text{MegDV}}$

$\begin{aligned}
& \text{H}_{3} \text{C} \\
& \text{N}(\text{CH}_{3})_{2} \\
& \text{O} \\
& \text{OTDP}
\end{aligned}$

10

$\xleftarrow{\text{MegDIV}}$

$\begin{aligned}
& (\text{H}_{3} \text{C})_{2} \text{N} \\
& \text{O} \\
& \text{OTDP}
\end{aligned}$

9

$\xrightarrow{\text{MegDIII}}$

L-Meg (2)

FIG. 2. (A) Schematic representation of the validated biosynthesis pathways of D-Des (structure 5) (20) and L-Myc (structure 4) (19) and reaction steps for the biosynthesis of L-Meg (structure 2), as previously proposed by Volchegursky and coworkers (25). (B) Reformulated biosynthesis pathway of L-Meg (structure 2) from TKDG (structure 3) (see the text for details). The enzymes involved during biosynthesis are shown. MegBIIb was originally named MegDVII (25). Glu-1-P, glucose-1-phosphate.

In vivo characterization of TDP-L-megosamine pathway. In order to validate the proposed pathway for the biosynthesis of L-Meg, several T7-derived expression vectors were constructed to contain the genes required for each biosynthetic step. megDI, megDV, and megDVI were then amplified and cloned from the meg gene cluster of M. megalomicea. These genes and the previously characterized genes megBVI, megDII, megDIII, and megBIV (17) were assembled in different combinations to generate alternative operons for the analysis of each biosynthesis step and transformed into the E. coli strain LB19b (18). This strain is a BL21(DE3) derivative strain that contains several modifications resulting in increased intracellular levels of the precursor TKDG. In this way, the L-Meg biosynthesis intermediates accumulated by the expression of each operon were analyzed by LC/MS/MS as previously described (20).

The first step proposed for the generation of L-Meg from TKDG, catalyzed by MegBVI, is common to the L-Myc pathway and was already analyzed in a previous work (19). In that study, the expression of MegBVI in E. coli resulted in the consumption of TKDG, with no apparent production of a detectable TDP-sugar, a phenomenon explained by the spontaneous decomposition of the expected TDP-2,6-dideoxy-3,4-diketo-D-glucose into maltol (7, 10). Therefore, to demonstrate the proposed dehydration/transamination sequential steps catalyzed by MegBVI/MegDII during L-Meg biosynthesis, the coexpression of these proteins was analyzed using plasmid pM19 in LB19b (Table 1). The LC/MS/MS analyses of cell extracts obtained from these cultures did not show the presence of any new TDP-sugar (Table 2). The absence of any detectable intermediate could be due to the spontaneous decomposition of TDP-2,6-dideoxy-3,4-diketo-D-glucose, which could shift the equilibrium of the reversible transamination reaction catalyzed by MegDII, leading to the net consumption of the expected intermediate TDP-3-amino-2,3,6-trideoxy-4-keto-D-glucose (Fig. 2B). This degradation has been previously observed in the in vitro characterization of the TDP-L-epivan-cosamine biosynthesis pathway using EvaB (8).

Additionally, the LC/MS/MS analysis of cell extract obtained

TABLE 2. Analysis of masses expected for each set of proteins expressed

| Plasmid used | Expected compound | Parent/daughter $(m/z)$ | Detection$^a$ |
|--------------|-------------------|------------------------|---------------|
| pLB132       | 6                 | 527/321               | ND            |
| pM19         | 8                 | 528/321               | ND            |
| pM1          | 9                 | 556/321               | D             |
| pM20         | 10                | 556/321               | D             |
| pM21         | 2                 | 558/321               | D             |
| pM22         | 2                 | 558/321               | ND            |

$^a$ D, detected; ND, not detected.

from cultures coexpressing MegBVI, MegDII, and MegDIII (plasmid pM1) showed the presence of an intermediate with a parent/daughter pair of $m/z$ 556/321 (Table 2 and Fig. 3A). These data correspond to the expected intermediate TDP–3-N,N-dimethylamino-2,3,6-trideoxy-4-keto-D-glucose (structure 9), thus confirming the three enzymatic steps proposed. The addition of the 5-epimerase encoded by $megDIV$ in this operon (plasmid pM20) showed the presence of a compound with the same parent/daughter ion pair and retention time as TDP–3-N,N-dimethylamino-2,3,6-trideoxy-4-keto-D-glucose. This result indicates that both D and L epimers have the same retention time under the high-pressure liquid chromatography (HPLC) condition assayed.

The final step proposed for the L-Meg biosynthesis route is the reduction of TDP–3-N,N-dimethylamino-2,3,6-trideoxy-4-keto-L-glucose (structure 10) by MegDV. Although it was previously demonstrated that MegBIV is involved in the biosynthesis of L-Myc (19), the participation of either MegDV or MegBIV as 4-ketoreductases in L-Meg biosynthesis still needs to be addressed. For this purpose, two different expression vectors were constructed (pM21 and pM22), adding either $megDV$ or $megBIV$ to pM1, respectively. The expression of each plasmid was tested within the LB19b strain, and the production of L-Meg was evaluated by LC/MS/MS. Analysis of cell extracts obtained from cells expressing pM21 showed the presence of a TDP-sugar with a parent/daughter pair of $m/z$ 558/321, consistent with the expected mass for L-Meg (Fig. 3B). This TDP-sugar was not detected in cultures expressing pM22 (data not shown), indicating that MegBIV acts specifically on the L-Myc route and MegDV on the L-Meg pathway.

Overall, these results confirm our predicted scheme of reactions for the biosynthesis of L-Meg and the meg genes involved.

Production of megalomicin A in *E. coli*. Once L-Meg is synthesized, the sugar moiety has to be attached to its macrolide substrate by a dedicated glycosyltransferase in order to generate megalomicin (Fig. 4). Three putative glycosyltransferases were previously identified in the megalomicin gene cluster, $megBV$, $megCIII$, and $megDI$ (25). MegBV is the glycosyltransferase required for the attachment of L-Myc to erythronolide B (17), and MegCIII is the desosaminyltransferase that, together with the helper protein MegCII, catalyzes the transfer of D-Des to 3-$\alpha$-mycarosyl-erythronolide B (MEB) (20). These data suggest that MegDI should be the glycosyltransferase that attaches L-Meg to the macrolide. Moreover, the presence of $megDVI$, located immediately upstream of $megDI$ (Fig. 1B), which encodes a protein with homology to TDP-amino-sugar

FIG. 3. LC/MS/MS analysis of TDP-sugars from cell extracts of strain LB19b expressing different biosynthesis operons: pM1 (A), resulting in the accumulation of TDP–3-N,N-dimethylamino-2,3,6-trideoxy-4-keto-D-glucose (structure 9), or pM21 (B), resulting in the accumulation of L-Meg (structure 2).

FIG. 4. Schematic representation of two alternative biosynthesis pathways for the conversion of EryD (structure 12) into MegA (structure 1).

two pairs of amino-sugar transferases (MegDI/MegDVI and MegCIII/MegCII) present in the meg cluster, bioconversion experiments were carried out, combining the expression of these protein pairs, or combinations of their components, with different sugar donors and macrolide acceptors. Replacement of *megDVI* with *megCII* in the megosamine operon (pM32) allowed the transfer of L-megosamine to EryC, producing MegA in bioconversion experiments with an efficiency similar to that when using pM9. Similarly, desosamination of MEB was observed when *megCII* was replaced by *megDVI* in the desosamine operon (plasmid pKOS506-72B plus pM3). However, despite the high amino acid sequence similarity within MegDI and MegCIII (77% similar and 65% identical) (Table 2), replacement of *megCIII* with *megDI* in the desosamine

operon or *megDI* with *megCIII* in the megosamine operon did not result in any bioconversion product (Table 3). These results confirm that the auxiliary proteins do not contribute to the specificity of the glycosyltransferase, although their presence is essential for the reaction, as previously shown with other helper proteins (6, 11, 15, 16, 30).

Final steps in megalomicin A biosynthesis and generation of a new megalomicin analog. Although it was demonstrated that EryC is a macrolide acceptor for the megosaminyltransferase pair MegDI/MegDVI, there is no experimental evidence that the attachment of D-desosamine or the C-12 oxidation mediated by MegK occurs before the attachment of the L-megosamine residue during megalomicin synthesis (Fig. 4). To assess whether this glycosylation step could also

FIG. 5. MS/MS analysis of MegA and 12dMegA. (A) MegA, calculated for C₄₄H₈₀N₂O₁₅ (M+H⁺) = 877.56315, in agreement with previously reported data (14). (B) 12dMegA, calculated for C₄₄H₈₀NaN₂O₁₄ (M+Na⁺) = 883.55018.

TABLE 3. Production of EryD or MegA using different combinations of glycosyltransferases and helper proteins

| Substrates | Protein expressed |
|------------|-------------------|
|            | MegDI             | MegCIII        |
|            | MegDVI            | MegDVI         |
|            | MegCII            | MegCII         |
| D-Des + MEB | MegA              | EryD           |
| L-Meg + EryC | MegA              | ND$^a$         |

$a$, not detected.

take place with MEB or erythromycin D (EryD) (structure 12) as substrates, bioconversion experiments were carried out by supplementing culture broths of strain LB19b carrying the plasmid pM9 with 20 mg/liter of MEB or EryD. Cultures were incubated at 23°C for 72 h, and fermentation broths were analyzed by TLC and MS/MS. These analyses showed that only EryD is mesosaminylated by MegDI/MegDVI, and this glycosylation results in the production of a novel megalomicin analog, 12-deoxy-megalomicin A (12dMegA) (structure 13) (Fig. 5B). High-resolution MS/MS analysis of pure 12dMegA gave a molecular ion of 861.56987, which is in agreement with a molecular formula of C₄₄H₈₁N₂O₁₄ (M+H), which has a calculated mass of 861.56878. This result indicates some degree of flexibility of MegDI/MegDVI toward the aglycon substrate.

If the synthesis of megalomicin takes place through the attachment of L-mesosamine prior to C-12 hydroxylation, the P450 hydroxylase MegK should also be flexible toward the macrolide substrate (Fig. 4, pathway B). In previous work, it was demonstrated that MegK catalyzes the hydroxylation of EryD in vivo (17). To evaluate if this enzyme can also catalyze the conversion of 12dMegA into MegA, bioconversion experiments were performed with strain LB19b expressing MegK from plasmid pM132. Cultures were supplemented with 20 mg/liter of 12dMegA after the induction of megK expression and incubated at 23°C for 72 h. No detectable amounts of MegA could be observed in culture extracts analyzed by TLC and MS/MS, indicating that no hydroxylation of 12dMegA took place under the experimental conditions assayed (data not shown). This result confirms that mesosaminylation takes place after MegK hydroxylates EryD during the final steps of megalomicin biosynthesis (Fig. 4, pathway A).

## DISCUSSION

The characterization of deoxy sugar biosynthetic pathways is an essential task for the development of novel therapeutic drugs through the modification of their glycosylation pattern. The present work elucidates the enzymatic steps and the enzymes required for the biosynthesis of the deoxy sugar L-Meg and its transfer to the acceptor macrolide, a crucial step in the synthesis of the antibacterial, antiviral, and antiparasitic polyketides megalomicins. Additionally, the biosynthesis pathway of L-Meg is reformulated on the basis of more-recent data reported for the biosynthesis of L-Myc and D-Des for *M. megalomicea* (17, 19, 20). Several reaction steps were experimentally established and modified with respect to those previously suggested by Volchegursky and coworkers (25). The enzymes involved in L-Meg biosynthesis were mostly reassigned or, in the case of MegBIIb (originally named MegDVII), finally excluded. Validation of this new L-Meg biosynthetic pathway was accomplished through its reconstitution in the heterologous host *E. coli*.

By comparing the gene organization of the left arm of the *meg* and *ery* clusters, Volchegursky and coworkers suggested that a mesosamine biosynthesis “island” was formed via an insertion of the *megY* and *megD* genes into an existing erythromycin or common ancestral gene cluster (Fig. 1B) (25). As a result, several genes in the mesosamine “island” might encode proteins with redundant activities in relation to others present in the cluster. This work proved that only the ketoreductase MegDV and the glycosyltransferase MegDI are exclusively dedicated to L-mesosamine biosynthesis.

Current and previous studies (17, 19, 20) confirm that four enzymes of the L-Meg route, MegDII, MegDIII, MegDIV, and MegBVI, also participate in one of the two other TDP-sugar pathways involved in the biosynthesis of megalomicins. Remarkably, these enzymes (with the exception of MegBVI) recognize different sugar substrates in each pathway (Fig. 2), indicating an inherent tolerance for the sugar substrate. The relaxed substrate specificity of these TDP-sugar enzymes should facilitate combinatorial biosynthesis and provide several new models for exploring the structural basis of substrate recognition.

MegDII is a pyridoxal 5′-phosphate-dependent aminotransferase that belongs to the subgroup DegT/DbrJ/EryCl/StrS of the aminotransferase family. This enzyme acts on two different TDP-3-keto-sugar substrates. During L-Meg biosynthesis, MegDII utilizes 2,6-dideoxyhexose as a substrate (Fig. 2B), while it uses a 2,4,6-trideoxyhexose substrate during D-Des biosynthesis (Fig. 2A) (20). Previous phylogenetic analysis using amino acid sequence alignments of this subgroup of the aminotransferase family allowed the classification of this subfamily of proteins into three subgroups based on the position of the amino receptor (12, 26). One subgroup of enzymes acts on NDP-3-keto sugars (VIα), another on NDP-4-keto sugars (VIβ), and the last one on a scyllo-inosose substrate (VIγ) (12). Figure 6 shows a new phylogenetic tree, where newly characterized SAT enzymes were included. The phylogenetic analysis of the VIα subgroup reveals two distinct branches in this subgroup. This subdivision is also supported by the different catalytic roles of these enzymes, where one branch utilizes NDP-3-keto-2,4,6-trideoxy sugar substrates (i.e., EvaB and DnrJ) while the other uses NDP-3-keto-2,6-dideoxy or NDP-3-keto-6-deoxy sugar substrates (i.e., EryCI, DesV, and TylB). Although MegDII clearly has a common ancestor from the NDP-3-keto-2,4,6-trideoxy subgroup, it has evolved to additionally catalyze the transamination reaction using an NDP-3-keto-2,6-dideoxy sugar. This is the first in vivo report of a relaxed specificity of this subgroup of SATs toward its amino acceptor.

MegDIII is an S-adenosylmethionine-dependent N,N-dimethyltransferase acting on two different substrates during megalomicin biosynthesis. The in vivo experiments conducted during this investigation demonstrated that MegDIII catalyzes the N,N dimethylation of TDP-3-amino-2,3,6-trideoxy-4-keto-D-glucose in L-Meg biosynthesis, while it uses TDP-3-amino-2,3,4,6-tetradeoxy-D-glucose for the biosynthesis of D-Des (20). So far, two methyltransferases have been biochemically characterized, DesVI from the pikromycin gene cluster of Strepto-

VIαA
- WxcK
- QdtB
- TyIB
- OleN2
- DesV

VIαB
- MegDI
- TcaB8
- AngB
- Med20
- KijD2
- EvaB
- DnrJ
- AknZ
- AclZ

VIβ
- RfbE
- Per
- ColD
- CanA
- WecE
- Cj1121c
- Cj1294
- DesI
- OleN1
- MegCIV
- EryCI
- LmbS
- StrS Sgl
- StsC
- TbmB
- WecE
- Med20
- NbmG
- KijD2
- EvaB
- AclZ
- TcaB8
- QdtB
- OleN2
- SpnR
- DesI
- EryCIV
- PseC
- PglE

VIγ
- BlmS
- StrS
- SpcS2
- StsC
- TbmB
- LmbS
- SpnR
- ArnB
- VioA
- DesI
- OleN1
- MegCIV
- Gssd
- Cj1294
- Cj1121c

FIG. 6. Phylogenetic tree of SAT family. Multiple alignments were performed using the Clustal W software program, and the tree was constructed using the MEGA 4 software program (23). Proteins (GenBank accession numbers) are as follows: VioA (AAD44154), AknZ (AAF73462), ArnB (AAM92146), AngB (AAG33854), CanA (CAC22113), Cj1121c (CAL35238), Cj1294 (AAT12282), DesV (AAC68680), DnrJ (B43306), EryCI (S06725), LmbS (CAA55764), MegDI (CAC3737809), Per (O07849), RfbE (CAA42137), StrS (CAA68523), SpcS2 (AAD28492), StrS Sgl (CAA07383), StsC (CAA70012), TbmB (Q2MF17), TyIB (S49052), WecE (AAC76796), WxcK (AAK53470), Med20 (BAC79028), NbmG (AAM88356), KijD2 (ACB46490), EvaB (CAA11782), AclZ (BAB72037), TcaB8 (ACB37737), QdtB (AAR85519), OleN1 (AAD55456), OleN2 (AAD55458), SpnR (AAG23279), DesI (AAC68684), EryCIV (YP_001102983), MegCIV (World patent WO2004/003169), PseC (NP_207164), PglE (YP_001000799), and ColD (NP_419828).

myces venezuelae and TylM1 from the tylosin gene cluster of Streptomyces fradiae (9). In vitro and in vivo studies revealed that these enzymes also exhibit some tolerance for their sugar substrates, where TylM1 was able to N,N dimethylate the DesVI substrate and vice versa.

The 5′ epimerase step catalyzed by MegDIV is another interesting example of bifunctional enzymes encoded in the meg cluster. During L-Meg biosynthesis, this enzyme epimerizes TDP-3-N,N-dimethylamino-2,3,6-trideoxy-4-ke-to-D-glucose (Fig. 2), while it was previously demonstrated that MegDIV is also responsible for the 5′ epimerization of TDP-3-methyl-4-keto-2,6-dideoxy-D-glucose in the L-Myc pathway (17).

Finally, this work conclusively determined that the attachment of L-megosamine to its macrolide substrate is mediated by the glycosyltransferase pair MegDI/MegDVI. Similar glycosyltransferase/auxiliary protein systems have already been described for several amino sugars, including the desosaminyltransferase pair MegCIII/MegCII, encoded in the meg cluster (6, 15, 20, 30). Bioconversion experiments confirmed EryC as its substrate during the biosynthesis of MegA. However, the MegDI/MegDVI pair was also capable of incorporating L-megosamine to EryD.

to produce a novel polyketide, 12dMegA. This substrate tolerance is an important feature for the development of novel megalomicin analogs, and it has already been successfully used to generate new megosamine-containing macrolide derivatives through bioconversion experiments in *E. coli* (S. Peiru, unpublished results).

Although MegDI/MegDVI is able to megosaminylate both EryC and EryD, opening the possibility of two alternative megalomicin biosynthesis routes (Fig. 4), no hydroxylation of 12dMegA by the P450 hydroxylase MegK was observed under the conditions tested. Because neither 12dMegA nor any acyl derivative of this compound could be detected in culture broths obtained from the natural producer *M. megalomicea* (27), further investigation needs to be done to shed light on these results.

Since 6DOHs are vital components of biologically active glycoconjugates defining properties like solubility, efficacy, and specificity (28), several *in vivo* and *in vitro* strategies, referred to as glycodiversification, have been established for the development of new therapeutic agents by modifying and/or exchanging the sugar structures to enhance or alter the biological activities of their parent molecules (2). However, the success of these approaches eventually depends on the tolerance of the biosynthetic enzymes acting on unusual substrates, especially the glycosyltransferases, which are the enzymes responsible for the addition of the sugar moieties to a scaffold (29). Enzymes from the *meg* cluster are a unique example of enzymes exhibiting an important degree of natural substrate tolerance. The broad substrate specificity of these enzymes could make them a new tool for glycodiversification approaches toward the generation of novel chemotherapeutic agents.

## ACKNOWLEDGMENTS

This work was supported by ANPCyT grants 2006-01978 (to E. Rodríguez) and 15-31969 (to H. Gramajo and E. Rodríguez) and by grant PIP 6436 from CONICET (to H. Gramajo).

## REFERENCES

1. Alarcon, B., M. E. Gonzalez, and L. Carrasco. 1988. Megalomicin C, a macrolide antibiotic that blocks protein glycosylation and shows antiviral activity. FEBS Lett. 231:207–211.
2. Blanchard, S., and J. S. Thorson. 2006. Enzymatic tools for engineering natural product glycosylation. Curr. Opin. Chem. Biol. 10:263–271.
3. Bonay, P., I. Duran-Chica, M. Fresno, B. Alarcon, and A. Alcina. 1998. Antiparasitic effects of the intra-Golgi transport inhibitor megalomicin. Antimicrob. Agents Chemother. 42:2668–2673.
4. Bonay, P., M. Fresno, and B. Alarcon. 1997. Megalomicin disrupts lysosomal functions. J. Cell Sci. 110(Pt. 16):1839–1849.
5. Bonay, P., S. Munro, M. Fresno, and B. Alarcon. 1996. Intra-Golgi transport inhibition by megalomicin. J. Biol. Chem. 271:3719–3726.
6. Borisova, S. A., L. Zhao, I. C. Melancon, C. L. Kao, and H. W. Liu. 2004. Characterization of the glycosyltransferase activity of desVII: analysis of and implications for the biosynthesis of macrolide antibiotics. J. Am. Chem. Soc. 126:6534–6535.
7. Chen, H., G. Agnihotri, Z. Guo, N. L. S. Que, X. H. Chen, and H.-w. Liu. 1999. Biosynthesis of mycarose: isolation and characterization of enzymes involved in the C-2 deoxygenation. J. Am. Chem. Soc. 121:8124–8125.
8. Chen, H., M. G. Thomas, B. K. Hubbard, H. C. Losey, C. T. Walsh, and M. D. Burkart. 2000. Deoxysugars in glycopeptide antibiotics: enzymatic synthesis of TDP-L-epivancosamine in chloroeremomycin biosynthesis. Proc. Natl. Acad. Sci. U. S. A. 97:11942–11947.
9. Chen, H., H. Yamase, K. Murakami, C. W. Chang, L. Zhao, Z. Zhao, and H. W. Liu. 2002. Expression, purification, and characterization of two N,N-dimethyltransferases, tylM1 and desVI, involved in the biosynthesis of mycaminose and desosamine. Biochemistry 41:9165–9183.
10. Draeger, G., S.-H. Park, and H. G. Floss. 1999. Mechanism of the 2-deoxygenation step in the biosynthesis of the deoxyhexose moieties of the antibiotics granaticin and oleandomycin. J. Am. Chem. Soc. 121:2611–2612.
11. Hong, J. S., S. J. Park, N. Parajuli, S. R. Park, H. S. Koh, W. S. Jung, C. Y. Choi, and Y. J. Yoon. 2007. Functional analysis of desVIII homologues involved in glycosylation of macrolide antibiotics by interspecies complementation. Gene 386:123–130.
12. Hwang, B. Y., H. J. Lee, Y. H. Yang, H. S. Joo, and B. G. Kim. 2004. Characterization and investigation of substrate specificity of the sugar amidotransferase WecE from *E. coli* K12. Chem. Biol. 11:915–925.
13. Jaret, R. S., A. K. Mallams, and H. Reimann. 1973. The megalomics. IV. The structures of megalomics A, B, C1, and C2. J. Chem. Soc. Perkin 1 13:1374–1388.
14. Jaret, R. S., A. K. Mallams, and H. F. Vernay. 1973. The megalomics. V. Mass spectral studies. J. Chem. Soc. Perkin 1 13:1389–1400.
15. Lu, W., C. Leimkuhler, G. J. Gatto, Jr., R. G. Kruger, M. Oberthur, D. Kahne, and C. T. Walsh. 2005. AknT is an activating protein for the glycosyltransferase AknS in L-aminodeoxysugar transfer to the aglycone of aclacinomycin A. Chem. Biol. 12:527–534.
16. Melancon, C. E., III, H. Takahashi, and H. W. Liu. 2004. Characterization of tylM3/tylM2 and mydC/mycB pairs required for efficient glycosyltransfer in macrolide antibiotic biosynthesis. J. Am. Chem. Soc. 126:16726–16727.
17. Peiru, S., H. G. Menzella, E. Rodriguez, J. Carney, and H. Gramajo. 2005. Production of the potent antibacterial polyketide erythromycin C in Escherichia coli. Appl. Environ. Microbiol. 71:2539–2547.
18. Peiru, S., E. Rodriguez, H. Menzella, J. Carney, and H. Gramajo. 2008. Metabolically engineered *Escherichia coli* for efficient production of glycosylated natural products. Microb. Biotechnol. 1:476–486.
19. Peiru, S., E. Rodriguez, C. Q. Tran, J. R. Carney, and H. Gramajo. 2007. Characterization of the heterodimeric MegBIIa:MegBIIb aldo-keto reductase involved in the biosynthesis of L-mycarose from *Micromonospora megalomicea*. Biochemistry 46:8100–8109.
20. Rodriguez, E., S. Peiru, J. R. Carney, and H. Gramajo. 2006. In vivo characterization of the dTDP-D-desosamine pathway of the megalomicin gene cluster from *Micromonospora megalomicea*. Microbiology 152:667–673.
21. Salas, J. A., and C. Mendez. 2007. Engineering the glycosylation of natural products in actinomycetes. Trends Microbiol. 15:219–232.
21a. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.
22. San Jose, E., M. A. Munoz-Fernandez, and B. Alarcon. 1997. Megalomicin inhibits HIV-1 replication and interferes with gp160 processing. Virology 239:303–314.
23. Tamura, K., J. Dudley, M. Nei, and S. Kumar. 2007. MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0. Mol. Biol. Evol. 24:1596–1599.
24. Ton That, T., G. Lukacs, S. Omura, P. Bartner, D. L. Boxler, R. Brambilla, A. K. Mallams, J. B. Morton, and P. Reichert. 1978. Megalomics. 6. Tertiary glycosidic macrolide antibiotics. A structural revision by carbon-13 nuclear magnetic resonance and x-ray crystallography. J. Am. Chem. Soc. 100:663–666.
25. Volchegursky, Y., Z. Hu, L. Katz, and R. McDaniel. 2000. Biosynthesis of the anti-parasitic agent megalomicin: transformation of erythromycin to megalomicin in *Saccharopolyspora erythraea*. Mol. Microbiol. 37:752–762.
26. Wang, Y., Y. Xu, A. V. Perepelov, Y. Qi, Y. A. Knirel, L. Wang, and L. Feng. 2007. Biochemical characterization of dTDP-D-Qui4N and dTDP-D-Qui4NAc biosynthetic pathways in *Shigella dysenteriae* type 7 and *Escherichia coli* O7. J. Bacteriol. 189:8626–8635.
27. Weinstein, M. J., G. H. Wagman, J. A. Marquez, R. T. Testa, E. Oden, and J. A. Waitz. 1969. Megalomicin, a new macrolide antibiotic complex produced by *Micromonospora*. J. Antibiot. (Tokyo) 22:253–258.
28. Weymouth-Wilson, A. C. 1997. The role of carbohydrates in biologically active natural products. Nat. Prod. Rep. 14:99–110.
29. Williams, G. J., R. W. Gantt, and J. S. Thorson. 2008. The impact of enzyme engineering upon natural product glycodiversification. Curr. Opin. Chem. Biol. 12:556–564.
30. Yuan, Y., H. S. Chung, C. Leimkuhler, C. T. Walsh, D. Kahne, and S. Walker. 2005. In vitro reconstitution of EryCIII activity for the preparation of unnatural macrolides. J. Am. Chem. Soc. 127:14128–14129.
