
Journal of Microbiological Methods 82 (2010) 71–77

Contents lists available at ScienceDirect

Journal of Microbiological Methods

journal homepage: www.elsevier.com/locate/jmicmeth

# A multiplex PCR method to detect 14 Escherichia coli serogroups associated with urinary tract infections

Dan Li<sup>a,b</sup>, Bin Liu<sup>a,b</sup>, Min Chen<sup>e</sup>, Dan Guo<sup>a,b</sup>, Xi Guo<sup>a,b</sup>, Fenxia Liu<sup>a,b</sup>, Lu Feng<sup>a,b,c</sup>, Lei Wang<sup>a,b,c,d,∗</sup>

<sup>a</sup> TEDA School of Biological Sciences and Biotechnology, Nankai University, 23 Hongda Street, TEDA, Tianjin 300457, PR China  
<sup>b</sup> Tianjin Key Laboratory of Microbial Functional Genomics, Tianjin 300457, PR China  
<sup>c</sup> Tianjin Research Center for Functional Genomics and Biochip, Tianjin 300457, PR China  
<sup>d</sup> The Key Laboratory of Molecular Microbiology and Technology, Ministry of Education, PR China  
<sup>e</sup> Shanghai Municipal Center For Disease Control & Prevention, No.1380, Zhong Shan Xi Rd., Shanghai, 200336, PR China

## Article Info

**Article history:**  
Received 18 December 2009  
Received in revised form 16 April 2010  
Accepted 22 April 2010  
Available online 28 April 2010  

**Keywords:**  
Multiplex PCR  
O-antigen  
Urinary tract infection  
Uropathogenic Escherichia coli

## Abstract

Urinary tract infections (UTIs) are one of the most common bacterial infections and are predominantly caused by uropathogenic *Escherichia coli* (UPEC). *E. coli* strains belonging to 14 serogroups, including O1, O2, O4, O6, O7, O8, O15, O16, O18, O21, O22, O25, O75 and O83, are the most frequently detected UPEC strains in a diverse range of clinical urine specimens. In the current study, the O-antigen gene clusters of *E. coli* serogroups O1, O2, O18 and O75 were characterized. A multiplex PCR method based on O-antigen-specific genes was developed for the simultaneous detection of all 14 *E. coli* serogroups. The multiplex PCR method was shown to be highly specific and reproducible when tested against 186 *E. coli* and *Shigella* O-serogroup reference strains, 47 *E. coli* clinical isolates and 10 strains of other bacterial species. The sensitivity of the multiplex PCR method was analyzed and shown to detect O-antigen-specific genes in samples containing 25 ng of genomic DNA or in mock urine specimens containing 40 colony-forming units (CFUs) per ml. Five urine specimens from hospital were examined using this multiplex PCR method, and the result for one sample was verified by the conventional serotyping methods. The multiplex PCR method developed herein can be used for the detection of relevant *E. coli* strains from clinical and/or environmental samples, and it is particularly useful for epidemiologic analysis of urine specimens from patients with UTIs.

© 2010 Elsevier B.V. All rights reserved.

## 1. Introduction

Urinary tract infections (UTIs) are one of the most common infectious diseases in humans and are a major cause of morbidity and mortality. UTIs comprise a range of disorders, including cystitis (infection of the bladder) and pyelonephritis (infection of the kidney), which are defined by the presence of microorganisms in the otherwise sterile urinary tract ([Kulkarni et al., 2009](https://doi.org/Kulkarni%20et%20al.,%202009)). It is estimated that between 40 and 50% of women experience a UTI at least once during their lifetime, and that 33% of women in the United States suffering from a UTI require antimicrobial therapy by the age of 24 ([Foxman, 2003](https://doi.org/Foxman,%202003); [Foxman and Brown, 2003](https://doi.org/Foxman%20and%20Brown,%202003)). UTIs are also a common cause of febrile illness in young children. It has been estimated that UTIs are diagnosed in 1% of boys and in 3–8% of girls ([WHO, 2005](https://doi.org/WHO,%202005)).

There are many known microbial causes of UTIs, including *Candida albicans*, *Pseudomonas aeruginosa*, *Staphylococcus saprophyticus* and *Escherichia coli* (*E. coli*) ([Kulkarni et al., 2009](https://doi.org/Kulkarni%20et%20al.,%202009)). Extraintestinal pathogenic *E. coli* (ExPEC) strains that cause UTIs are called uropathogenic *E. coli* (UPEC). UPECs are responsible for between 70 and 90% of community-acquired UTIs and approximately 40% of all nosocomial UTIs in the United States ([Struelens et al., 2004](https://doi.org/Struelens%20et%20al.,%202004)). *E. coli* is a clonal species that includes both commensal and pathogenic clones of strains, which are normally identified by serological typing of their O (lipopolysaccharide), H (flagellar), and, in some cases, K (capsular) surface antigens. Recently, 174 O-serogroups had been described for *E. coli* ([Liu et al., 2008](https://doi.org/Liu%20et%20al.,%202008)), and some O-serogroups forms are disproportionately represented in pathogenic clones ([Reeves and Wang, 2002](https://doi.org/Reeves%20and%20Wang,%202002)). For UPEC, the virulence factor (VF) profile of each strain is related with their O-serogroups. It has been widely accepted that serotyping can be used as an epidemiological marker for pathogenicity for *E. coli* ([Bertschinger, 1999](https://doi.org/Bertschinger,%201999)). Several O-serogroups, including O1, O2, O4, O6, O7, O8, O15, O16, O18, O21, O22, O25, O75 and O83, are preferentially associated with UPEC strains ([Abe et al., 2008](https://doi.org/Abe%20et%20al.,%202008); [Ananias and Yano, 2008](https://doi.org/Ananias%20and%20Yano,%202008); [Bidet et al., 2007](https://doi.org/Bidet%20et%20al.,%202007); [Blanco et al., 1996](https://doi.org/Blanco%20et%20al.,%201996); [Johnson et al., 1994](https://doi.org/Johnson%20et%20al.,%201994); [Lloyd et al., 2007](https://doi.org/Lloyd%20et%20al.,%202007); [Orskov et al., 1982](https://doi.org/Orskov%20et%20al.,%201982); [Yamamoto, 2007](https://doi.org/Yamamoto,%202007)). Certain O-serogroups were found to occur with a variable frequency in the individual patient groups with UTI ([Budde et al., 1980](https://doi.org/Budde%20et%20al.,%201980)).

The O-antigen is part of the lipopolysaccharide (LPS) present in the outer membrane of Gram-negative bacteria and is comprised of

---

* Corresponding author. TEDA School of Biological Sciences and Biotechnology, Nankai University, 23 Hongda Street, TEDA, Tianjin 300457, PR China. Tel.: +86 22 6622 9588; fax: +86 22 6622 9596.  
E-mail address: wanglei@nankai.edu.cn (L. Wang).

0167-7012/$ – see front matter © 2010 Elsevier B.V. All rights reserved.  
doi:10.1016/j.jmicmeth.2010.04.008

repeat oligosaccharide units (O-units), which typically comprise three to six sugar residues. The O-antigen contributes the major antigenic variability to the cell surface and is subject to intense selection by the host immune system and other environmental factors, such as bacteriophages, which may account for the maintenance of diverse O-antigen forms within a species (Reeves, 1995). Screening for the E. coli O-serogroup is the conventional method for identifying E. coli clones.

Characteristically, all genes specific to O-antigen synthesis are normally grouped together on the chromosome in a gene cluster that maps between the *galF* and *gnd* genes in *E. coli* (Reeves et al., 1996; Shepherd et al., 2000). Genes involved in O-antigen synthesis are normally classified into three main groups: 1) nucleotide sugar biosynthesis genes; 2) glycosyltransferase genes; and 3) O-unit processing genes including the flippase gene (*wzx*) and the polymerase gene (*wzy*) (Reeves and Wang, 2002). Genes in the last two classes are normally specific to individual O-antigens and can be used as targets in molecular typing. Recently, PCR assays based on O-antigen-specific genes for several different serogroups of *E. coli* have been developed (Ren et al., 2008; Wang et al., 2005; Wang and Reeves, 1998).

In the current study, the O-antigen gene clusters of *E. coli* serogroups O1, O2, O18 and O75 were sequenced and analyzed. When combined with the published O-antigen-specific gene data for other O-serogroups, the current study provides a multiplex PCR method for the identification and detection of the 14 *E. coli* O-serogroups associated with UPEC. The multiplex PCR method was tested for its specificity and sensitivity, and was applied to mock urine specimen samples.

## 2. Materials and methods

### 2.1. Bacterial strains used

The bacterial strains used in this study included 186 *E. coli* and Shigella O-standard reference strains (Feng et al., 2004a), 47 clinical isolates and 10 other bacterial species (Table 1). The 47 clinical isolates were characterized for their respective O-serogroup using specific antisera (data not shown).

### 2.2. Construction of DNasel shot gun bank

Chromosomal DNA was prepared as described previously (Bastin and Reeves, 1995). Long-range PCR was performed with the Expand Long Template PCR system (Roche Applied Science) using primers wl-14082 (5′-CGTTGG TCG CTA TGT GCT TTC TGC CGA TAT-3′) and wl-14083 (5′-AAA TAG AGA CGG TAT AAC CAC GGCTTT CGA TG-3′), based on sequences of the *galF* and *gnd* genes, respectively (Wang et al., 1998). PCR was performed as follows: 95 °C for 5 min, 30 cycles of 95 °C for 30 s, 55 °C for 45 min, and 68 °C for 15 min, and a final extension at 68 °C for 7 min. The PCR products were digested with DNase I, and the resulting DNA fragments were cloned into the pGEM-T Easy vector (Promega) to produce a shotgun bank as described previously (Wang et al., 1998).

### 2.3. DNA sequencing and analysis

DNA sequencing was carried out by the Tianjin Biochip Corporation using an ABI 3730 automated DNA sequencer. Sequence data were assembled using the Staden Package (Staden, 1996). The program Artemis (Rutherford et al., 2000) was used for annotation. Blast and PSI-BLAST (Altschul et al., 1997) were used for searching databases including GenBank, COG and Pfam protein motif databases (Bateman et al., 2002). The program TMHMM 2.0 ([http://www.cbs.dtu.dk/services/TMHMM-2.0/](http://www.cbs.dtu.dk/services/TMHMM-2.0/)) was used to identify potential transmembrane segments.

### 2.4. Development of a multiplex PCR method

The multiplex PCR primer pairs were designed based on the *wzx* gene of *E. coli* serogroups O1, O4, O7, O16, O18, O21, O22, and O83, the *wzy* gene of *E. coli* serogroups O2, O6, O15, O25 and O75 and the *orf486* gene of *E. coli* serogroup O8 (Table 2). The fourteen primer pairs were divided into two groups to generate target DNA by multiplex PCR (Table 2).

Each multiplex PCR reaction was performed using 30 μl of a reaction mixture consisting of 50–100 ng of DNA, 3 μl of 10× PCR buffer (50 mM KCl, 10 mM Tris–HCl [pH 8.3]), 2.5 mM MgCl₂, 0.3 mM deoxynucleotide triphosphates, 0.05–0.13 μM of the respective primers and 2 U of Taq DNA polymerase (TaKaRa Taq™). All of the reagents were thawed completely on ice and all of the reactions were assembled on ice. The reaction parameters were 95 °C for 5 min, 30 cycles of 95 °C for 30 s, 55 °C for 1 min, and 72 °C for 1 min and a final extension at 72 °C for 5 min. In order to reduce non-specific amplification, the thermal cycler was pre-warmed to 80 °C before the reaction tubes were inserted. PCR amplification of the DNA was confirmed by running 2 μl of the PCR product on a 2% agarose gel. Gel images have been captured digitally and analyzed using the Quantity One system (Bio-Rad).

The chromosomal DNA of each O-standard strain of 14 serogroups was used as reference control for primers accuracy and specificity. Meanwhile, in a separate tube the 16S rRNA gene was used as the positive control to generate the 919 bp PCR fragment for all *E. coli* stains, the sequences of primer pair of wl-3110 and wl-3111 were listed in Table 2.

Table 2  
Primers and their concentrations in multiplex PCR$^a$.

| Serogroup | Specific gene | Gene bank accession no or reference | Primer name | Primer sequence (5′–3′)$^b$ | Amplicon size (bp) | Concn. in multiplex PCR (µM) |
|-----------|---------------|------------------------------------|-------------|-----------------------------|--------------------|-------------------------------|
| Group 1   |               |                                    |             |                             |                    |                               |
| E.coli O1 | wzx           | GU299791                          | wl-14632    | (F) GTGAGC AAAA AGT GAA ATA AGGAACG | 1098              | 0.05                          |
|           |               |                                    | wl-14633    | (R) CGCTGATACGA ATACC ATCCTAC |                    |                               |
| 06        | wzy           | AJ426423                          | wl-14646    | (F) GGATGACGATGTGATTTTGGCTAAC | 783               | 0.07                          |
|           |               |                                    | wl-14647    | (R) TCTGGGTTTGCTGTGTATGAGGC |                    |                               |
| 07        | wzx           | AF125322                          | wl-14648    | (F) CTATCAA AAT ACC TCTGCTGGAATC | 610               | 0.12                          |
|           |               |                                    | wl-14649    | (R) TGGCTTCGAGATTA AACCTATTCCT |                    |                               |
| 08        | orf 469       | AB010150                          | wl-14652    | (F) CCAGAGGCATAATCAGAAAATAACAG | 448               | 0.12                          |
|           |               |                                    | wl-14653    | (R) GCAGAGTTAGTCAACAAAAAGGTCAG |                    |                               |
| 016       | wzx           | AAC31631                          | wl-14654    | (F) GGTITCAATCTCACAGCAACTCAG | 302               | 0.13                          |
|           |               |                                    | wl-14655    | (R) GTTAGAGGGATAATAGCCAAAGCGG |                    |                               |
| 021       | wzx           | EU694098                          | wl-14676    | (F) CTGCTGATGTCGCTATTATTGCTG | 209               | 0.12                          |
|           |               |                                    | wl-14677    | (R) TGAAAAAAAAAGGAAACAGAAGAGCC |                    |                               |
| 075       | wzy           | GU299795                          | wl-17413    | (F) GAGATATACATGGGGAGGTTAGGCT | 511               | 0.07                          |
|           |               |                                    | wl-17414    | (R) ACCCGATAATCATATTCTTCCCAAC |                    |                               |
| Group 2   |               |                                    |             |                             |                    |                               |
| E.coli O2 | wzy           | GU299792                          | wl-14636    | (F) AGTGAGTTACTTTTAGCGATGGAC | 770               | 0.07                          |
|           |               |                                    | wl-14637    | (R) AGTTTAGTATGCC CTCGACTTTGAA |                    |                               |
| 04        | wzx           | AY568960                          | wl-14642    | (F) TTGTTGCGATAATGTGCATGTTCC | 664               | 0.07                          |
|           |               |                                    | wl-14643    | (R) AATAATTTGCTATACCCACACCCTC |                    |                               |
| 015       | wzy           | AY647261                          | wl-14672    | (F) TCTTGT TAGAGT CATTTGGTGTATCG | 183               | 0.08                          |
|           |               |                                    | wl-14673    | (R) ATAAAAACGAGCAAGCACCCACACC |                    |                               |
| 018       | wzx           | GU299793                          | wl-14656    | (F) GTTCGGTGGTTGGATTACAGTTAG | 551               | 0.08                          |
|           |               |                                    | wl-14657    | (R) CTACTATCATCCTC ACTGACCACG |                    |                               |
| 022       | wzx           | DQ851855                          | wl-14660    | (F) TTCATTGTCGCCACTACTTTCCG | 468               | 0.08                          |
|           |               |                                    | wl-14661    | (R) GAAACAGCCCATGACATTACTACG |                    |                               |
| 025       | wzy           | GU299796                          | wl-14666    | (F) AGAGATCCGTCTTTTATTTGTT CGC | 230               | 0.08                          |
|           |               |                                    | wl-14667    | (R) GTTCTGGATACCTAAGCAA TACCC |                    |                               |
| 083       | wzx           | GU299797                          | wl-14668    | (F) GTACACCAGGCAAACCTCGAAAG | 362               | 0.08                          |
|           |               |                                    | wl-14669    | (R) TTCTGTAAGCTAATGAATAGGCACC |                    |                               |
| Positive control |           |                                    |             |                             |                    |                               |
|           | 16S rRNA gene | War sen et al. (2004) Lane et al. (1985) | wl-3110$^c$ | (F) AGAGTTTGATCMTGGCTCAG | 919               |                               |
|           |               |                                    | wl-3111     | (R) CCGTCAATTCATTTGAGTTT |                    |                               |

$a$ The multiplex PCR primer sets were divided into two groups.  
$b$ F, forward primer; R, reverse primer.  
$c$ M = A/C.

D. Li et al. / Journal of Microbiological Methods 82 (2010) 71–77

E.coli O1  
Gene rmlB rmlDrmlArmlC wzx mnaA wekM wzy wekN wekO  

E.coli O2  
Gene rmlB rmlDrmlAfdtAfdtCfdtB wzx wekPrmlCwzywekQwekR wekS  

E.coli O18  
Gene rmlB rmlDrmlArmlCwzx wzy wekTwekUwekV wekW  

E.coli O75  
Gene rmlB rmlDrmlArmlCwekX wzy wekYwekZ manC manB wzx  

Fig. 1. The O-antigen gene cluster organization of E. coli serogroups O1, O2, O18 and O75. The H-repeat elements in E. coli serogroups O2 are indicated by grey rectangles. (See Supplementary Tables 1, 2, 3 and 4, respectively, for a description of the function of these genes).

four-step synthesis of dTDP-L-Rha from glucose 1-phosphate ([Graninger et al., 1999]), and the gene (mnaA) responsible for the synthesis of UDP-D-ManNAc from UDP-D-GlcNAc ([Samuel and Reeves, 2003]). Common sugars, including D-GlcNAc, D-glucose (D-Glc) and D-galactose (D-Gal), are synthesized by housekeeping genes located outside the O-antigen gene cluster. Three glycosyltransferase genes (wekM, wekN and wekO) and two O-unit processing genes (wzx and wzy) are also located in the O-antigen gene cluster.

The O-unit of E. coli serogroup O2 contains five sugar residues: three L-Rha residues, a D-GlcNAc residue and a D-Fuc3NAc residue ([Jansson et al., 1987]). The E. coli serogroup O2 O-antigen gene cluster contains the rmlABCD genes, which are responsible for the synthesis of dTDP-L-Rha, and the fdtA, fdtB and fdtC genes, which are responsible for the synthesis of dTDP-D-Fuc3NAc together with the rmlA and rmlB genes ([Pfoestl et al., 2003]). The O-antigen gene cluster also contains four glycosyltransferase genes (wekP, wekQ, wekR and wekS), and the wzx and wzy genes.

The O-unit of E. coli serogroup O18 contains five sugar residues: an L-Rha residue, a D-Glc residue, a D-Gal residue and two D-GlcNAc residues ([Jansson et al., 1989]). The O-antigen gene cluster of E. coli serogroup O18 contains the rmlABCD genes for the synthesis of dTDP-L-Rha, four glycosyltransferase genes (wekT, wekU, wekV and wekW), and the wzx and wzy genes.

The O-unit of E. coli serogroup O75 contains four sugar residues: a D-Man residue, an L-Rha residue, a D-Gal residue and a D-GlcNAc residue ([Erbing et al., 1975]). With the exception of the rmlABCD genes, the O-antigen gene cluster of E. coli serogroup O75 contains the manB and manC genes for the synthesis of GDP-D-Man, four glycosyltransferase genes (wekX, wekY and wekZ) and the wzx and wzy genes.

### 3.2. Development of a multiplex PCR method for identification of 14 E. coli serogroups associated with UPEC

The O-unit processing genes (wzx and wzy) are highly specific to individual O-antigens, and are typically used as the target in PCR-based typing methods. Therefore, in the current study, the specific primer pairs were designed based on the wzx gene (for E. coli serogroups O1, O4, O7, O16, O18, O21, O22 and O83) and the wzy gene (for E. coli serogroups O2, O6, O15, O25 and O75). The only exception was E. coli serogroup O8, whose O-antigen gene cluster shares a high level of similarity with that of Klebsiella pneumoniae serogroup O5 ([Merino et al., 2000]). In the current study, we selected orf469, which is the most heterogeneous gene between the O-antigen gene cluster of E. coli serogroup O8 and K. pneumoniae serogroup O5, as the E. coli serogroup O8 target gene. The 14 primer pairs were divided into two groups to generate target DNA ([Table 2]).

We initially used the same concentration (0.2 μM) for all 14 primer pairs, however we did not consistently achieve the expected results for some of the serogroups (probably due to competition/interference between different primer pairs in the multiplex PCR reaction). Therefore, we tested a number of different primer concentrations and combinations, and the optimal conditions are shown in [Table 2].

The multiplex PCR method was tested against 186 E. coli and Shigella O-standard reference strains and 10 strains of other bacterial species ([Table 1]). In the presence of each target strain, only the corresponding O-serogroup-specific primer pairs were used, and only one band of the expected size was generated ([Fig. 2]). The amplicons ranged in size from 183 bp to 1098 bp in length ([Table 2]). The representative E. coli strains belonging to other serogroups or other bacterial strains did not generate PCR products of the correct size. The results showed that all 14 primer pairs were specific and compatible in multiplex PCR reactions.

A double-blind test was performed to verify the reliability and specificity of the multiplex PCR method. A total of 47 clinical isolates of E. coli ([Table 1]) were selected for detection using the multiplex PCR method without disclosure of their identities at the time of testing. All of the results were consistent with those generated by conventional methods.

#### 3.3. Sensitivity of the multiplex PCR method

The sensitivity of the multiplex PCR method was determined using a dilution series of chromosomal DNA from E. coli strains corresponding to each of the 14 serogroups, as described. In all multiplex PCR reactions, 25 ng of chromosomal DNA for each serogroup was sufficient to amplify specific fragments, which were detected easily by eye on an agarose gel ([Fig. 2]). All of the 47 clinical isolates belonging to the 14 serogroups could be detected successfully at this level.

The sensitivity of each of the 14 single-plex PCR reactions was also tested. 0.5 ng of chromosomal DNA for E. coli serogroups O1, O2, O7, O15, O16, O18, O21, O22, O25, O75 and O83 is sufficient to amplify specific fragments. 0.1 ng of chromosomal DNA for E. coli serogroups O6 and O8, and 0.01 ng of chromosomal DNA for E. coli serogroup O4 is sufficient. Obviously, the sensitivities of each of the 14 single-plex PCR reactions are much higher than the multiplex PCR assay. It is possible that the interaction among the primers in the multiplex PCR method decreased the reaction efficiency.

We tested the sensitivity of multiplex PCR method using mock urine specimens containing E. coli strains corresponding to each of the

D. Li et al. / Journal of Microbiological Methods 82 (2010) 71–77

![Image Description](image.png)

Fig. 2. Agarose gel electrophoresis of multiplex PCR reaction products. Lanes: Mr, molecular weight standards (100 bp + 1.5 kb DNA Marker-K); (A) Group 1: A, *E. coli* serogroup O1; B, *E. coli* serogroup O6; C, *E. coli* serogroup O7; D, *E. coli* serogroup O75; E, *E. coli* serogroup O8; F, *E. coli* serogroup O16; and G, *E. coli* serogroup O21. (B) Group 2: H, *E. coli* serogroup O2; I, *E. coli* serogroup O4; J, *E. coli* serogroup O18; K, *E. coli* serogroup O22; L, *E. coli* serogroup O83; M, *E. coli* serogroup O25; and N, *E. coli* serogroup O15.

14 serogroups. As little as 40 CFU/ml could be detected after the samples were grown at 37 °C for 10 h. Some strains, such as G1201 (*E. coli* serogroup O15) and G1720 (*E. coli* serogroup O75), could be detected successfully at 8 CFU/ml. The sensitivity of our multiplex PCR method is sensitive enough for clinical diagnosis of UTI.

### 3.4. Test in clinical specimens

The multiplex PCR method was applied for examination of five urine specimens obtained from patients with pyelonephritis and concomitant with symptoms of UTI in Tianjin medical university general hospital. Sample I gave positive results to *E. coli* serogroups O83 and O25 strains; Sample II, serogroups O75 and O21; Sample III, serogroup O21; Sample IV gave negative results; while Sample V had the same result with Sample III. In order to confirm the results, the culture of Sample I was selected and plated for single colonies. A total of 96 single bacterial colonies were picked and screened by PCR using *E. coli* serogroup O83-, O25-specific primers, respectively. Two O25-positive colonies were found after 64 colonies were screened, and their identities were confirmed by using the conventional O-serogrouping methods with commercial antisera (Institute of Veterinary Drug Control, Beijing, China) and sequencing 16S rRNA gene of the strains (data not shown). We did not find any *E. coli* serogroup O83 strains after all 96 colonies were screened, and it is possible that the corresponding colonies were missed. This indicates that the multiplex PCR method is more sensitive than conventional serotyping methods.

## 4. Discussion

Systematic O-serogrouping of *E. coli* began in the early 1930s (Nataro and Kaper, 1998), and it became an important tool for the classification of *E. coli* strains in clinical settings. There are many experimental evidences showing a close connection among certain serogroups and certain markers of pathogenicity in pathogens, such as UPEC (Blanco et al., 1996; Yamamoto, 2007; Terai et al., 1997; Rizvi and Kumar, 2003; Hughes et al., 1982). For example, the majority of uropathogenic O4, O6, O14, O22, O75 and O83 *E. coli* strains were Hly⁺CNF1⁺ and expressed P-fimbriae or MRHA type III, whereas the strains of serogroup O18 were Hly⁺CNF1⁻ and P-fimbriated (Blanco et al., 1996). Moreover, O-serogrouping of *E. coli* has been an invaluable typing method for epidemiological investigations (Bertschinger, 1999). It has been reported that O16 strains are specifically associated with febrile UTI in infants (Marild et al., 1988) and asymptomatic bacteriuria (ABU) in girls (Lidin-Janson et al., 1977), O18 strains are associated with ABU in infants (Marild et al., 1988), O8, O25, and O83 strains are associated with cystitis in girls (Lindberg, 1975), whereas O4 occurs with relatively greater frequency in males (Bourke et al., 1973).

Conventional O-serogrouping of *E. coli* strains in clinical specimens, food and environmental samples is laborious, time-consuming and impractical for the analysis of large numbers of specimens. Moreover, the serological assay cannot be performed on strains with a rough O-antigen, which are frequently isolated from clinical and environmental samples (Feng et al., 2004c). A PCR-based method is a rapid analytical technique for the detection of a variety of bacterial strains (Wright and Wynford-Thomas, 1990) and has the potential to reduce the time required to identify bacterial strains. Recently, PCR assays based on O-antigen-specific genes were developed for the identification of a number of *E. coli* serogroups, including *E. coli* serogroups O114, O21 and O3 (Feng et al., 2004b; Feng et al., 2004c; Ren et al., 2008). However, conventional PCR methods that amplify individual target genes are laborious when many specific genes must be detected. Taqman probe-based real-time PCR is limited by its capacity to detect only four common used fluorophores in a single reaction. Therefore, it will be very difficult to detect multiple targets simultaneously. Real-time PCR using SYBR green fluorescence followed by melting temperature determination is less accurate and stable than multiplex PCR method when detecting multiple targets. Multiplex PCR can simultaneously amplify multiple target genes in the same reaction, and has the potential for increased accuracy while saving time and effort. Because of these advantages, multiplex PCR has been applied successfully for the identification of many bacterial strains (Song et al., 2000; Tran and Rudney, 1999).

UTIs are among the world's most common bacterial infections and cause significant morbidity (Foxman, 2002). Based on the traditional diagnostic criterion, a UTI is defined as bacteriuria greater than or equal to 10⁵ CFU/ml of midstream urine (Sheffield and Cunningham, 2005; Stamm et al., 1982). However, Stamm studied women with presumptively lower numbers of CFUs and discovered that up to 50% of UTIs in symptomatic women with coliforms in their urine were not detected when using this criterion (Lloyd et al., 2007; Stamm et al.,

1982). For this reason, many laboratories have opted to use lower colony counts as a criterion for interpreting and reporting results. One common criterion is a colony count of 10⁴ CFU/ml, which would be expected to increase the sensitivity of the test without making it impractical for clinician and laboratory use (Wilson and Gaido, 2004). Considering the high sensitivity of detection with mock urine specimens, the multiplex PCR-based method developed herein can achieve the requirement in the clinic. In summary, the multiplex PCR method developed in this study, which is able to detect 14 UPEC-related *E. coli* serogroups with high specificity and sensitivity, is an efficient and convenient strategy for the detection of relevant UTI-associated *E. coli* stains, especially in large-scale epidemiologic investigations.

### Acknowledgements

This work was supported by the Tianjin Municipal Special Fund for Science and Technology Innovation Grant 05FZZDSH00800, the National Natural Science Foundation of China (NSFC) Key Programs Grants 30530010 and 20536040, the Chinese National Science Fund for Distinguished Young Scholars (30788001), NSFC General Program Grant 30670038, 30870078, 30771175 and 30900041, the National 863 Program of China grants 2006AA020703 and 2006AA06Z409, the National 973 Program of China grants 2009CB522603, and National Key Programs for Infectious Diseases of China 2008ZX10004-002, 2008ZX10004-009 and 2009ZX10004-108.

### Appendix A. Supplementary data

Supplementary data associated with this article can be found, in the online version, at [doi:10.1016/j.mimet.2010.04.008](https://doi.org/10.1016/j.mimet.2010.04.008).

### References

Abe, C.M., Salvador, F.A., Falsetti, I.N., Vieira, M.A., Blanco, J., Blanco, J.E., Blanco, M., Machado, A.M., Elias, W.P., Hernandes, R.T., Gomes, T.A., 2008. Uropathogenic *Escherichia coli* (UPEC) strains may carry virulence properties of diarrhoeagenic *E. coli*. FEMS Immunol. Med. Microbiol. 52, 397–406.

Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., Lipman, D.J., 1997. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucl. Acids Res. 25, 3389–3402.

Ananias, M., Yano, T., 2008. Serogroups and virulence genotypes of *Escherichia coli* isolated from patients with sepsis. Braz. J. Med. Biol. Res. 41, 877–883.

Bastin, D.A., Reeves, P.R., 1995. Sequence and analysis of the O antigen gene (*rfb*) cluster of *Escherichia coli* O111. Gene 164, 17–23.

Bateman, A., Birney, E., Cerruti, L., Durbin, R., Eddy, S.R., Griffiths-Jones, S., Howe, K.L., Marshall, M., Sonnhammer, E.L., 2002. The Pfam protein families database. Nucl. Acids Res. 30, 276–280.

Bertschinger, H.U., 1999. Postweaning *E. coli* diarrhea and edema disease. In: Straw, B.E., D'Allaire, S., Mengeling, W.L., Taylor, D.J. (Eds.), Diseases of Swine. Iowa State University Press, Ames, pp. 441–454.

Bidet, P., Mahjoub-Messai, F., Blanco, J., Dehem, M., Aujard, Y., Bingen, E., Bonacorsi, S., 2007. Combined multilocus sequence typing and O serogrouping distinguishes *Escherichia coli* subtypes associated with infant urosepsis and/or meningitis. J. Infect. Dis. 196, 297–303.

Blanco, M., Blanco, J.E., Alonso, M.P., Blanco, J., 1996. Virulence factors and O groups of *Escherichia coli* isolates from patients with acute pyelonephritis, cystitis and asymptomatic bacteriuria. Eur. J. Epidemiol. 12, 191–198.

Bourke, M., Carroll, R., Deasy, P.F., 1973. *Escherichia coli* serotypes in childhood urinary tract infection. Ir. J. Med. Sci. 142, 91–97.

Budde, E., Naumann, G., Nimmich, W., Straube, E., 1980. *Escherichia coli* O Serotyping in different patients groups with urinary tract infections (author's transl). Infection 8, 22–26.

Erbing, C., Svensson, S., Hammarstrom, S., 1975. Structural studies on the O-specific side-chains of the cell-wall lipopolysaccharide from *Escherichia coli* O75. Carbohydr. Res. 44, 259–265.

Feng, L., Senchenkova, S.N., Yang, J., Shashkov, A.S., Tao, J., Guo, H., Zhao, G., Knirel, Y.A., Reeves, P., Wang, L., 2004a. Structural and genetic characterization of the *Shigella boydii* Type 13 O antigen. J. Bacteriol. 182, 383–392.

Feng, L., Tao, J., Guo, H., Xu, J., Li, Y., Rezwan, F., Reeves, P.R., Wang, L., 2004b. Structure of the *Shigella dysenteriae* 7 O antigen gene cluster and identification of its antigen specific genes. Microb. Pathog. 36, 109–115.

Feng, L., Wang, W., Tao, J., Guo, H., Krause, G., Beutin, L., Wang, L., 2004c. Identification of *Escherichia coli* O114 O-antigen gene cluster and development of an O114 serogroup-specific PCR assay. J. Clin. Microbiol. 42, 3799–3804.

Foxman, B., 2002. Epidemiology of urinary tract infections: incidence, morbidity, and economic costs. Am. J. Med. 113 (Suppl 1A), 5S–13S.

Foxman, B., 2003. Epidemiology of urinary tract infections: incidence, morbidity, and economic costs. Dis. Mon. 49, 53–70.

Foxman, B., Brown, P., 2003. Epidemiology of urinary tract infections: transmission and risk factors, incidence, and costs. Infect. Dis. Clin. North Am. 17, 227–241.

Graninger, M., Nidetzky, B., Heinrichs, D.E., Whitfield, C., Messner, P., 1999. Characterization of dTDP-4-dehydrorhamnose 3,5-epimerase and dTDP-4-dehydrorhamnose reductase, required for dTDP-L-rhamnose biosynthesis in *Salmonella enterica* serovar Typhimurium LT2. J. Biol. Chem. 274, 25069–25077.

Hughes, C., Phillips, R., Roberts, A.P., 1982. Serum resistance among *Escherichia coli* strains causing urinary tract infection in relation to O type and the carriage of hemolysin, colicin, and antibiotic resistance determinants. Infect. Immun. 35, 270–275.

Jansson, P.E., Lennholm, H., Lindberg, B., Lindquist, U., Svenson, S.B., 1987. Structural studies of the O-specific side-chains of the *Escherichia coli* O2 lipopolysaccharide. Carbohydr. Res. 161, 273–279.

Jansson, P.E., Kenne, L., Widmalm, G., 1989. Structure of the O-antigen polysaccharide from *Escherichia coli* O18ac: a revision using computer-assisted structural analysis with the program CASPER. Carbohydr. Res. 193, 322–325.

Johnson, J.R., Orskov, I., Orskov, F., Goullet, P., Picard, B., Moseley, S.L., Roberts, P.L., Stamm, W.E., 1994. O, K, and H antigens predict virulence factors, carboxylesterase B pattern, antimicrobial resistance, and host compromise among *Escherichia coli* strains causing urosepsis. J. Infect. Dis. 169, 119–126.

Knirel, Y.A., Kochetkov, N.K., 1994. The structure of lipopolysaccharides of Gram-negative bacteria. III. The structure of O antigens: a review. Biochemistry (Moscow) 59, 1325–1383.

Kulkarni, R., Dhakal, B.K., Slechta, E.S., Kurtz, Z., Mulvey, M.A., Thanassi, D.G., 2009. Roles of putative type II secretion and type IV pilus systems in the virulence of uropathogenic *Escherichia coli*. PLoS ONE 4, e4752.

Lane, D.J., Pace, B., Olsen, G.J., Stahl, D.A., Sogin, M.L., Pace, N.R., 1985. Rapid determination of 16S ribosomal RNA sequences for phylogenetic analyses. Proc. Natl. Acad. Sci. U. S. A. 82, 6955–6959.

Lidin-Janson, G., Hanson, L.A., Kaijser, B., Lincoln, K., Lindberg, U., Olling, S., Wedel, H., 1977. Comparison of *Escherichia coli* from bacteriuric patients with those from feces of healthy schoolchildren. J. Infect. Dis. 136, 346–353.

Lindberg, U., 1975. Asymptomatic bacteriuria in school girls. V. The clinical course and response to treatment. Acta. Paediatr. Scand. 64, 718–724.

Liu, B., Knirel, Y.A., Feng, L., Perepelov, A.V., Senchenkova, S.N., Wang, Q., Reeves, P.R., Wang, L., 2008. Structure and genetics of *Shigella* O antigens. FEMS Microbiol. Rev. 32, 627–653.

Lloyd, A.L., Rasko, D.A., Mobley, H.L., 2007. Defining genomic islands and uropathogen-specific genes in uropathogenic *Escherichia coli*. J. Bacteriol. 189, 3532–3546.

Marild, S., Wettergren, B., Hellstrom, M., Jodal, U., Lincoln, K., Orskov, I., Orskov, F., Svanborg Eden, C., 1988. Bacterial virulence and inflammatory response in infants with febrile urinary tract infection or screening bacteriuria. J. Pediatr. 112, 348–354.

Merino, S., Altarriba, M., Izquierdo, L., Nogueras, M.M., Regue, M., Tomas, J.M., 2000. Cloning and sequencing of the *Klebsiella pneumoniae* O5 wb gene cluster and its role in pathogenesis. Infect. Immun. 68, 2435–2440.

Nataro, J.P., Kaper, J.B., 1998. Diarrheagenic *Escherichia coli*. Clin. Microbiol. Rev. 11, 142–201.

Orskov, I., Orskov, F., Birch-Andersen, A., Kanamori, M., Svanborg-Eden, C., 1982. O, K, H and fimbrial antigens in *Escherichia coli* serotypes associated with pyelonephritis and cystitis. Scand. J. Infect. Dis. Suppl. 33, 18–25.

Pfoestl, A., Hofinger, A., Kosma, P., Messner, P., 2003. Biosynthesis of dTDP-3-acetamido-3, 6-dideoxy-alpha-D-galactose in *Aneurinibacillus thermoaeophilus* L420-91T. J. Biol. Chem. 278, 26410–26417.

Reeves, P.R., 1995. Role of O-antigen variation in the immune response. Trends Microbiol. 3, 381–386.

Reeves, P.P., Wang, L., 2002. Genomic organization of LPS-specific loci. Curr. Top. Microbiol. Immunol. 264, 109–135.

Reeves, P.R., Hobbs, M., Valvano, M., Skurnik, M., Whitfield, C., Coplin, D., Kido, N., Klena, J., Maskell, D., Raetz, C., Rick, P., 1996. Bacterial polysaccharide synthesis and gene nomenclature. Trends Microbiol. 4, 495–503.

Ren, Y., Liu, B., Cheng, J., Liu, F., Feng, L., Wang, L., 2008. Characterization of *Escherichia coli* O3 and O21 O antigen gene clusters and development of serogroup-specific PCR assays. J. Microbiol. Methods 75, 329–334.

Rizvi, M., Kumar, S., 2003. Serum resistance of *Escherichia coli* strains causing urinary tract infection and diarrhoea in relation to alpha haemolysin production and O type. Indian J. Pathol. Microbiol. 46, 504–506.

Rutherford, K., Parkhill, J., Crook, J., Horsnell, T., Rice, P., Rajandream, M.A., Barrell, B., 2000. Artemis: sequence visualisation and annotation. Bioinformatics 16, 944–945.

Samuel, G., Reeves, P., 2003. Biosynthesis of O-antigens: genes and pathways involved in nucleotide sugar precursor synthesis and O-antigen assembly. Carbohydr. Res. 338, 2503–2519.

Sheffield, J.S., Cunningham, F.G., 2005. Urinary tract infection in women. Obstet. Gynecol. 106, 1085–1092.

Shepherd, J.G., Wang, L., Reeves, P.R., 2000. Comparison of O-antigen gene clusters of *Escherichia coli* (*Shigella*) sonnei and *Plesiomonas shigelloides* O17: Sonnei gained its current plasmid-borne O-antigen genes from *P. shigelloides* in a recent event. Infect. Immun. 68, 6056–6061.

Song, Y., Kato, N., Liu, C., Matsumiya, Y., Kato, H., Watanabe, K., 2000. Rapid identification of 11 human intestinal *Lactobacillus* species by multiplex PCR assays using group- and species-specific primers derived from the 16S–23S rRNA intergenic spacer region and its flanking 23S rRNA. FEMS Microbiol. Lett. 187, 167–173.

Staden, R., 1996. The Staden sequence analysis package. Mol. Biotech. 5, 233–241.

Stamm, W.E., Counts, G.W., Running, K.R., Fihn, S., Turck, M., Holmes, K.K., 1982. Diagnosis of coliform infection in acutely dysuric women. N. Engl. J. Med. 307, 463–468.

Struelens, M.J., Denis, O., Rodriguez-Villalobos, H., 2004. Microbiology of nosocomial infections: progress and challenges. Microbes. Infect. 6, 1043–1048.

Terai, A., Yamamoto, S., Mitsumori, K., Okada, Y., Kurazono, H., Takeda, Y., Yoshida, O., 1997. *Escherichia coli* virulence factors and serotypes in acute bacterial prostatitis. Int. J. Urol. 4, 289–294.

Tran, S.D., Rudney, J.D., 1999. Improved multiplex PCR using conserved and species-specific 16S rRNA gene primers for simultaneous detection of *Actinobacillus actinomycetemcomitans*, *Bacteroides forsythus*, and *Porphyromonas gingivalis*. J. Clin. Microbiol. 37, 3504–3508.

Wang, L., Reeves, P.R., 1998. Organization of *Escherichia coli* O157 O antigen gene cluster and identification of its specific genes. Infect. Immun. 66, 3545–3551.

Wang, L., Jensen, S., Hallman, R., Reeves, P.R., 1998. Expression of the O antigen gene cluster is regulated by RfaH through the JUMPstart sequence. FEMS Microbiol. Lett. 165, 201–206.

Wang, L., Liu, B., Kong, Q., Steinruck, H., Krause, G., Beutin, L., Feng, L., 2005. Molecular markers for detection of pathogenic *Escherichia coli* strains belonging to serogroups O138 and O139. Vet. Microbiol. 111, 181–190.

Warsen, A.E., Krug, M.J., LaFrentz, S., Stanek, D.R., Loge, F.J., Call, D.R., 2004. Simultaneous discrimination between 15 fish pathogens by using 16S ribosomal DNA PCR and DNA microarrays. Appl. Environ. Microb. 70, 4216–4221.

WHO, 2005. Urinary Tract Infections in Infants and Children in Developing Countries in the Context of IMCI. World Health Organization, Geneva, Switzerland. 2009.

Wilson, M.L., Gaido, L., 2004. Laboratory diagnosis of urinary tract infections in adult patients. Clin. Infect. Dis. 38, 1150–1158.

Wright, P.A., Wynford-Thomas, D., 1990. The polymerase chain reaction: miracle or mirage? A critical review of its uses and limitations in diagnosis and research. J. Pathol. 162, 99–117.

Yamamoto, S., 2007. Molecular epidemiology of uropathogenic *Escherichia coli*. J. Infect. Chemother. 13, 68–73.
