
Microbiology (2010), 156, 2757–2769

DOI 10.1099/mic.0.040196-0

Role of Vibrio polysaccharide (vps) genes in VPS production, biofilm formation and Vibrio cholerae pathogenesis

Jiunn C. N. Fong,$^{1}$ Khalid A. Syed,$^{2}$ Karl E. Klose$^{2}$ and Fitnat H. Yildiz$^{1}$

$^{1}$Department of Microbiology and Environmental Toxicology, University of California, Santa Cruz, Santa Cruz, CA 95064, USA

$^{2}$Department of Biology, The University of Texas at San Antonio, South Texas Center for Emerging Infectious Diseases, San Antonio, TX 78249, USA

Biofilm formation enhances the survival and persistence of the facultative human pathogen *Vibrio cholerae* in natural ecosystems and its transmission during seasonal cholera outbreaks. A major component of the *V. cholerae* biofilm matrix is the *Vibrio* polysaccharide (VPS), which is essential for development of three-dimensional biofilm structures. The *vps* genes are clustered in two regions, the *vps-I* cluster (*vpsU*, *vpsA-K*, VC0916–27) and the *vps-II* cluster (*vpsL-Q*, VC0934–39), separated by an intergenic region containing the *rbm* gene cluster that encodes biofilm matrix proteins. In-frame deletions of the *vps* clusters and genes encoding matrix proteins drastically altered biofilm formation phenotypes. To determine which genes within the *vps* gene clusters are required for biofilm formation and VPS synthesis, we generated in-frame deletion mutants for all the *vps* genes. Many of these mutants exhibited reduced capacity to produce VPS and biofilms. Infant mouse colonization assays revealed that mutants lacking either *vps* clusters or *rbmA* (encoding secreted matrix protein RbmA) exhibited a defect in intestinal colonization compared to the wild-type. Understanding the roles of the various *vps* gene products will aid in the biochemical characterization of the VPS biosynthetic pathway and elucidate how *vps* gene products contribute to VPS biosynthesis, biofilm formation and virulence in *V. cholerae*.

---

INTRODUCTION

*Vibrio cholerae* causes the diarrhoeal disease cholera (Kaper *et al.*, 1995) and is a natural inhabitant of aquatic ecosystems (Faruque *et al.*, 1998). *V. cholerae*'s ability to cause epidemics is linked to its ability to survive in its natural habitats, which is enhanced by its ability to form matrix-enclosed, surface-associated communities, known as biofilms (Alam *et al.*, 2006; Faruque *et al.*, 1998). In aquatic habitats, *V. cholerae* can be isolated from surfaces of phytoplankton, zooplankton, aquatic plants, crustaceans and insects, as well as sediments (Broza *et al.*, 2005; Halpern *et al.*, 2004; Huq *et al.*, 1983, 1986, 1995), and biofilm formation facilitates environmental survival of the pathogen. Biofilms are also critical for the transmission and infectivity of *V. cholerae*. Removal of particles >20 μm in diameter from water can reduce cholera incidence (Colwell *et al.*, 2003; Huo *et al.*, 1996). Furthermore, stool samples of cholera patients contain both biofilm-like aggregates and planktonic forms of *V. cholerae* (Faruque *et al.*, 2006), and the average infectivity of the aggregate form is significantly higher than that of planktonic cells (Faruque *et al.*, 2006). Altogether, these studies indicate the importance of the biofilm growth mode in both the intestinal and aquatic phases of *V. cholerae*'s life cycle.

Formation of mature biofilms requires the production of extracellular matrix components. A major component of the *V. cholerae* biofilm matrix is VPS (*Vibrio* polysaccharide exopolysaccharide), which is required for the formation of mature biofilm structures (Yildiz & Schoolnik, 1999), while matrix proteins, in particular RbmA, RbmC and Bap1, are required for maintaining the structural integrity of the wild-type biofilm (Fong & Yildiz, 2007; Fong *et al.*, 2006). VPS was first isolated from a rugose wild-type strain of *V. cholerae* that exhibited enhanced capacity to produce VPS, and consequently enhanced biofilm formation compared to the smooth wild-type strain. VPS of *V. cholerae* O1 El Tor strain A1552 contains glucose, galactose, N-acetylglucosamine and mannose, and the genes required for VPS synthesis were identified through transposon mutagenesis (Yildiz & Schoolnik, 1999). The *vps* genes are clustered in two regions on the large chromosome of *V. cholerae* O1 El Tor [*vpsU* (VC0916), *vpsA-K* (VC0917–27) (*vps-I* cluster); and *vpsL-Q* (VC0934–9) (*vps-II* cluster)].

Abbreviations: CSLM, confocal scanning laser microscopy; VPS, *Vibrio* polysaccharide.

A supplementary table of primers is available with the online version of this paper.

J. C. N. Fong and others

However, it is not known if all the genes within the *vps-I* and *vps-II* clusters are required for biofilm formation and VPS production.

Several studies suggest that VPS is produced during infection and could contribute to *in vivo* colonization and survival. It has been shown that *vpsH* (VC0924), involved in VPS production, is induced during *in vivo* growth in an infant mouse model system (Lee *et al.*, 2001). Furthermore, an *in vivo* expression technology (IVET) screen performed in human volunteers to identify genes expressed during infection revealed that several *vps* genes, namely *vpsA* (VC0917), *vpsB* (VC0918), *vpsC* (VC0919) and *vpsN* (VC0936), are expressed during human infection (Lombardo *et al.*, 2007). *In vivo*-induced antigen technology (IVIAT) has also shown that sera from cholera patients recognize VpsH (VC0924), which is required for VPS production (Hang *et al.*, 2003). Hung *et al.* (2006) reported that bile acids stimulate *vps* transcription and *in vitro* biofilm formation. A recent study has also shown that both planktonic and aggregated forms (which are predicted to result from *in vivo* biofilms) of *V. cholerae* are shed in human stools (Faruque *et al.*, 2006). In addition, it was recently shown that VpsR, the master regulator of biofilm formation and *vps* expression, directly activates transcription of *aphA*, which encodes a positive transcriptional regulator of virulence gene expression in *V. cholerae* (Lin *et al.*, 2007). Taken together, these studies strongly suggest that *vps* genes are expressed and probably important *in vivo*.

In this study, we generated in-frame deletions of all the *vps* genes located in the *vps-I* and *vps-II* clusters and determined the role of these VPS proteins in biofilm formation and VPS biosynthesis. We also analysed the contribution of VPS and matrix proteins to *in vivo* fitness of *V. cholerae*.

### METHODS

#### Colony morphology and pellicle formation analyses.
For analysis of corrugated colony morphology development, cultures grown overnight at 30 °C with shaking (200 r.p.m.) were serially diluted with LB medium and 100 µl aliquots of the diluted cultures were plated onto LB agar medium. The cultures were incubated at 30 °C for 2 days. For analysis of pellicle formation, glass culture tubes (18 × 150 mm) containing 5 ml LB medium were inoculated with overnight-grown cultures, resulting in a 200-fold dilution. The tubes were incubated at 30 °C under non-shaking conditions for 2 days. Assays were repeated with at least two different biological replicates.

#### VPS immunoblot assays.
Isolation of crude VPS from wild-type and mutant strains and immunoblot analyses of the extracted VPS were carried out according to a protocol similar to those previously published (Enos-Berlage & McCarter, 2000; Fong & Yildiz, 2007; Yildiz & Schoolnik, 1999). Briefly, overnight-grown cultures on LB agar medium were harvested and resuspended in PBS. Normalization was carried out by adjusting each culture to the same OD~600~, measured with a Beckman Du Series 500 spectrophotometer. Equal volumes of the cultures were pipetted into Erlenmeyer flasks and the suspension was incubated overnight at 4 °C, shaking at 100 r.p.m. Crude VPS supernatant was separated from the bacterial cells and debris by centrifugation twice at 20 000 g for 30 min and precipitated with 3 volumes of ethanol at –20 °C overnight. Crude VPS was pelleted by centrifugation at 20 000 g for 30 min, washed with 70% ethanol, air-dried and resuspended in 500 µl water. Purified VPS from the wild-type rugose variant was used as a positive control and for quantification. Purified VPS was isolated as described above with additional purification steps published previously (Fong & Yildiz, 2007). Briefly, crude VPS pellet was resuspended in nuclease buffer (40 mM Tris/HCl pH 8.0, 10 mM MgCl₂, 2 mM CaCl₂, 0.05% NaN₃). DNase I and RNase A were added to the VPS suspension at final concentrations of 2 units ml⁻¹ and 50 µg ml⁻¹, respectively, followed by incubation at 37 °C shaking for 8 h. Proteinase K was then added at a final concentration of 200 µg ml⁻¹, and the suspension was further incubated with shaking overnight at 37 °C. Phenol/chloroform extractions (equal volumes) were carried out, followed by precipitation with 3 volumes of ethanol, washing with 70% ethanol, and solubilization in water. Detoxi-Gel endotoxin-removing gel (Pierce) was used to remove contaminating lipopolysaccharides (LPS) according to the manufacturer’s instructions and the purified VPS suspension was dialysed with water at 4 °C overnight using a Slide-A-Lyser 3.5K MWCO dialysis cassette (Pierce). The VPS suspension was then dried using a Micro Modulyo freeze-drier (Thermo Savant), the dry weight of the VPS sample was determined and the dried purified VPS was resolubilized in water. Crude and purified VPS (3 µl) was spotted onto nitrocellulose membranes and immunoblot analyses were carried out using anti-VPS antiserum and goat anti-rabbit horseradish-peroxidase-conjugated antibody. The immunoblots were developed with the SuperSignal West Pico chemiluminescent kit (Pierce) and quantified using ImageQuant 5.2 software (Molecular Dynamics). VPS immunoblot analyses were carried out with two different biological replicates and at least three technical replicates. It should be noted that the VPS antibody used in this study is polyclonal, with undefined epitopes. As such, the immunoblot assay was used as a phenotypic screen to identify mutants that cannot produce ‘wild-type’ VPS and not for determining the effect of a given mutation on the VPS structure.

#### Quantitative biofilm assays.
Biofilm formation assays were carried out in PVC microtitre plates (BD Falcon) with 100 µl of overnight-grown cultures diluted to an OD~600~ of 0.04. The microtitre plates were incubated at 30 °C for 8 h. Crystal violet staining and ethanol solubilization were carried out as previously described (Fong *et al.*, 2006; Yildiz *et al.*, 2001). Absorbance of the solubilized crystal violet stain was measured at 595 nm using a VersMax Tunable Microplate

#### Bacterial strains, plasmids and culture conditions.
The bacterial strains and plasmids used in this study are listed in Table 1. All *V. cholerae* and *Escherichia coli* strains were routinely grown aerobically in Luria–Bertani (LB) medium (1% tryptone, 0.5% yeast extract, 1% NaCl) at 30 °C and 37 °C respectively, unless otherwise noted. Agar medium contained 1.5% granulated agar (Difco). Concentrations of antibiotics used were as follows: ampicillin, 100 µg ml⁻¹; rifampicin, 100 µg ml⁻¹; gentamicin, 50 µg ml⁻¹.

#### Recombinant DNA techniques.
Restriction enzymes, DNA modification enzymes and Phusion High-Fidelity DNA polymerase were purchased from New England Biolabs. PCRs were carried out with primers purchased from Bioneer USA Corporation, listed in Supplementary Table S1, available with the online version of this paper. DNA sequencing was carried out by the Sequetech Corporation.

#### Generation of in-frame deletion mutants.
Deletion mutants were generated according to the protocol previously published (Fong *et al.*, 2006; Fullner & Mekalanos, 1999). The DNA sequences of the constructed deletion plasmids were verified by DNA sequencing.

Table 1. Bacterial strains and plasmids used in this study

| Strain or plasmid | Relevant genotype and phenotype | Source |
|-------------------|---------------------------------|--------|
| **E. coli strains** |  |  |
| CC118λpir | Δ(ara-leu) araD ΔlacX74 galE galK phoA20 thi-1 rpsE rpoB argE(Am) recA1 λpir | Herrero et al. (1990) |
| S17-1λpir | Tp<sup>r</sup> Sm<sup>r</sup> recA thi pro r<sub>K</sub><sup>-</sup>m<sub>K</sub><sup>+</sup> RP4:::2-Tc:::MuKmTn7λpir | de Lorenzo & Timmis (1994) |
| **V. cholerae strains** |  |  |
| FY_Vc_1 | V. cholerae O1 El Tor A1552, smooth wild-type variant, Rif<sup>r</sup> | Yildiz & Schoolnik (1999) |
| FY_Vc_2 | V. cholerae O1 El Tor A1552, rugose wild-type variant, Rif<sup>r</sup> | Yildiz & Schoolnik (1999) |
| FY_Vc_234 | RΔvps-I, rugose variant with deletion of vpsA-K, Rif<sup>r</sup> | Beyhan & Yildiz (2007) |
| FY_Vc_4324 | RΔvps-II, rugose variant with deletion of vpsL-Q, Rif<sup>r</sup> | This study |
| FY_Vc_4327 | RΔvps-IΔvps-II, rugose variant with deletion of vpsA-K and vpsL-Q, Rif<sup>r</sup> | This study |
| FY_Vc_3679 | RΔvpsU, Rif<sup>r</sup> | This study |
| FY_Vc_2784 | RΔvpsA, Rif<sup>r</sup> | This study |
| FY_Vc_3682 | RΔvpsB, Rif<sup>r</sup> | This study |
| FY_Vc_4949 | RΔvpsC, Rif<sup>r</sup> | This study |
| FY_Vc_2787 | RΔvpsD, Rif<sup>r</sup> | This study |
| FY_Vc_4974 | RΔvpsE, Rif<sup>r</sup> | This study |
| FY_Vc_4951 | RΔvpsF, Rif<sup>r</sup> | This study |
| FY_Vc_3326 | RΔvpsG, Rif<sup>r</sup> | This study |
| FY_Vc_4954 | RΔvpsCΔvpsG, Rif<sup>r</sup> | This study |
| FY_Vc_4976 | RΔvpsH, Rif<sup>r</sup> | This study |
| FY_Vc_3374 | RΔvpsI, Rif<sup>r</sup> | This study |
| FY_Vc_3685 | RΔvpsJ, Rif<sup>r</sup> | This study |
| FY_Vc_3377 | RΔvpsK, Rif<sup>r</sup> | This study |
| FY_Vc_3380 | RΔvpsL, Rif<sup>r</sup> | This study |
| FY_Vc_3688 | RΔvpsM, Rif<sup>r</sup> | This study |
| FY_Vc_3382 | RΔvpsN, Rif<sup>r</sup> | This study |
| FY_Vc_3384 | RΔvpsO, Rif<sup>r</sup> | This study |
| FY_Vc_3691 | RΔvpsP, Rif<sup>r</sup> | This study |
| FY_Vc_4979 | RΔvpsQ, Rif<sup>r</sup> | This study |
| FY_Vc_240 | Rugose mTn7-gfp, Rif<sup>r</sup>Gm<sup>r</sup> | Beyhan et al. (2007) |
| FY_Vc_6226 | RΔvps-IΔvps-II mTn7-gfp, Rif<sup>r</sup>Gm<sup>r</sup> | This study |
| **Plasmids** |  |  |
| pGP704-sacB28 | pGP704 derivative, mob/oriT sacB, Ap<sup>r</sup> | G. Schoolnik |
| pAJH9 | pWM91::Δvps-I (ΔvpsA-K), Ap<sup>r</sup> | Kierek & Watnick (2003) |
| pFY-659 | pGP704-sacB28::Δvps-II (ΔvpsL-Q), Ap<sup>r</sup> | Fong & Yildiz (2008) |
| pFY-631 | pGP704-sacB28::ΔvpsU | This study |
| pFY-530 | pGP704-sacB28::ΔvpsA | This study |
| pFY-632 | pGP704-sacB28::ΔvpsB | This study |
| pFY-930 | pGP704-sacB28::ΔvpsC | This study |
| pFY-534 | pGP704-sacB28::ΔvpsD | This study |
| pFY-934 | pGP704-sacB28::ΔvpsE | This study |
| pFY-736 | pGP704-sacB28::ΔvpsF | This study |
| pFY-633 | pGP704-sacB28::ΔvpsG | This study |
| pFY-938 | pGP704-sacB28::ΔvpsH | This study |
| pFY-918 | pGP704-sacB28::ΔvpsI | This study |
| pFY-634 | pGP704-sacB28::ΔvpsJ | This study |
| pFY-920 | pGP704-sacB28::ΔvpsK | This study |
| pFY-922 | pGP704-sacB28::ΔvpsL | This study |
| pFY-635 | pGP704-sacB28::ΔvpsM | This study |
| pFY-924 | pGP704-sacB28::ΔvpsN | This study |
| pFY-925 | pGP704-sacB28::ΔvpsO | This study |
| pFY-636 | pGP704-sacB28::ΔvpsP | This study |
| pFY-942 | pGP704-sacB28::ΔvpsQ | This study |
| pMCM11 | pGP704::mTn7-gfp, Gm<sup>r</sup>Ap<sup>r</sup> | M. Miller and G. Schoolnik |
| pUX-BF13 | oriR6K helper plasmid, mob/oriT, provides the Tn7 transposition function in trans, Ap<sup>r</sup> | Bao et al. (1991) |
J. C. N. Fong and others

The assays were repeated with at least two different biological replicates and at least five technical replicates.

### Generation of GFP-tagged strains and confocal scanning laser microscopy (CSLM).
*V. cholerae* wild-type and deletion strains were tagged with green fluorescent protein (GFP) according to the procedure previously described (Fong *et al.*, 2006). Flow-cell experiments were carried out at room temperature as described previously (Beyhan *et al.*, 2007). Images were acquired using a Zeiss Axiocert 200 M laser scanning microscope. 3D images of the biofilms were reconstructed using IMARIS software (Bitplane) and quantified using COMSTAT (Heydorn *et al.*, 2000). Experiments were carried out with at least two biological replicates.

### Infection studies.
The infant mouse intestinal colonization model system was used in the single-strain infection experiment. Oral administration of \(10^6\) exponential-phase *V. cholerae* wild-type variants or deletion mutants (unable to produce biofilm matrix components) to 4–5-day-old CD-1 suckling mice was carried out. At 20 h post-infection, the mice were sacrificed and their small intestines were removed and homogenized. Intestinal homogenates were serially diluted and plated onto selective agar. The number of bacteria per intestine was quantified by colony counting and used to determine intestinal colonization of the mutant strain. One-way ANOVA (Kruskal–Wallis test) was performed to determine statistically significant differences. Individual groups were then compared using the Mann–Whitney non-parametric *t*-test; a *P*-value of ≤0.05 was considered significant (Prism 5, GraphPad Software).

---

## RESULTS AND DISCUSSION

### Molecular analysis of *vps* gene clusters
A major component of *V. cholerae* biofilm matrix is the VPS exopolysaccharide. VPS production is mediated by proteins encoded by the *vps* genes, which are organized into *vps-I* and *vps-II* clusters on the large chromosome, separated by an 8.3 kb intergenic region containing six genes (Fong & Yildiz, 2007; Fong *et al.*, 2006; Yildiz & Schoolnik, 1999). There are 18 *vps* genes, 12 of which are located in the *vps-I* cluster (*vpsU*, VC0916; *vpsA–K*, VC0917–27), while the other six are located in the *vps-II* cluster (*vpsL–Q*, VC0934–9) (Fig. 1). To determine if all the *vps* gene products are required for biofilm formation, we generated in-frame deletion mutants for each *vps* gene in the wild-type rugose genetic background (hereafter referred as RΔ). We also constructed mutant strains containing deletions of the first and/or second *vps* clusters [RΔ*vps-I* (*vpsA–K*), RΔ*vps-II* (*vpsL–Q*) and RΔ*vps-IΔvps-II*]. We then characterized the mutants for colony morphology, pellicle formation, biofilm formation and VPS production (Table 2). It is important to note that the deletion strains do not exhibit growth defects when compared to the parental wild-type strain (data not shown).

The rugose wild-type forms corrugated colonies (Fig. 2) and a robust pellicle (Fig. 3), and has increased capacity to form a biofilm (Fig. 4a) and produce VPS (Fig. 4b, c). We termed these phenotypes ‘rugosity-associated phenotypes’. Mature biofilm formed by the rugose wild-type also exhibited the characteristic elaborate 3D structures as analysed by CSLM (Fig. 5). We first characterized strains lacking the *vps* clusters (RΔ*vps-I*, RΔ*vps-II* and RΔ*vps-IΔvps-II*). These mutant strains formed flat and smooth colonies (Fig. 2), and exhibited a marked decrease in the production or secretion of VPS (Fig. 4b, c), and reduction in the ability to form pellicles (Fig. 3) and biofilms (Fig. 4a). CSLM analysis of biofilms of the rugose wild-type and RΔ*vps-IΔvps-II* revealed that the latter is unable to form the elaborate 3D mature biofilm structures (Fig. 5). COMSTAT analysis revealed that total biomass, and mean and maximum thicknesses, are lower in RΔ*vps-IΔvps-II* compared to the rugose wild-type, particularly after 24 h (Table 3). Strains carrying in-frame deletions of either the *vps-I* or *vps-II* cluster also form biofilms resembling that of RΔ*vps-IΔvps-II* (data not shown).

We then analysed the rugosity-associated phenotypes of the individual *vps* deletion mutants. Based on the predicted amino acid sequences and domains of the *vps* gene products and their predicted functions, we can group these VPS proteins into six classes (Table 2). Class I consists of proteins that are predicted to be required for production of nucleotide sugar precursors. VpsA is predicted to encode UDP-*N*-acetylglucosamine 2-epimerase while VpsB is predicted to encode UDP-*N*-acetyl-D-mannosaminuronic acid dehydrogenase. UDP-*N*-acetyl-glucosamine 2-epimerase catalyses the interconversion of UDP-*N*-acetyl-D-glucosamine to UDP-*N*-acetyl-D-mannosamine. The latter is then converted to UDP-*N*-acetyl-D-mannosaminuronic acid by UDP-*N*-acetyl-D-mannos-

![Fig. 1. Genomic organization of genes involved in VPS and matrix protein production in *V. cholerae*. Loci of the genes encoding proteins involved in VPS biosynthesis (*vps-I* and *vps-II* clusters) and genes encoding matrix proteins (*rbm* cluster and *bap1*) on the *V. cholerae* chromosome are depicted. Arrows represent genes predicted to be involved in VPS biosynthesis (black), encoding hypothetical proteins (grey) and encoding matrix proteins RbmA, RbmC and Bap1 (hatched), and other genes found in the *vps*-intergenic region (white arrows). Unlinked chromosomal DNA region is indicated (II). Illustration is not to scale.]()
Table 2. Predicted function of the vps gene products and phenotypic characteristics of vps mutants

| Annotation | Gene | Predicted peptide length | Colony corrugation* | Pellicle formation† | Biofilm formation‡ | VPS production§ | Class|| | Predicted functions/domains¶ |
|------------|------|-------------------------|---------------------|---------------------|--------------------|------------------|----------|-----------------------------|
| VC0916     | vpsU | 166                     | +#                  | +#                  | ++                 | +/-              | V        | Phosphotyrosine-protein     |
|            |      |                         |                     |                     |                    |                  |          | phosphatase                |
| VC0917     | vpsA | 372                     | -                   | -                   | +                  | -                | I        | UDP-N-acetylglucosamine    |
|            |      |                         |                     |                     |                    |                  |          | 2-epimerase                |
| VC0918     | vpsB | 413                     | -                   | -                   | +                  | -                | I        | UDP-N-acetyl-D-            |
|            |      |                         |                     |                     |                    |                  |          | mannosaminuronic acid       |
|            |      |                         |                     |                     |                    |                  |          | dehydrogenase             |
| VC0919     | vpsC | 184                     | +                   | +                   | +++                | +++              | IV       | Acetyltransferase          |
| VC0920     | vpsD | 382                     | -                   | -                   | +                  | -                | II       | Glycosyltransferase        |
| VC0921     | vpsE | 469                     | -                   | -                   | +                  | -                | III      | Polysaccharide export      |
| VC0922     | vpsF | 406                     | -                   | -                   | +                  | -                | VI       | Hypothetical protein       |
| VC0923     | vpsG | 143                     | +#                  | +#                  | ++                 | +                | IV       | Acetyltransferase          |
| VC0924     | vpsH | 446                     | +#                  | +#                  | ++                 | +                | III      | Polysaccharide polymerase  |
| VC0925     | vpsI | 365                     | -                   | -                   | +                  | -                | II       | Glycosyltransferase        |
| VC0926     | vpsJ | 390                     | -                   | -                   | +                  | -                | VI       | Hypothetical protein       |
| VC0927     | vpsK | 250                     | -                   | -                   | +                  | -                | II       | UDP-N-acetyl-D-            |
|            |      |                         |                     |                     |                    |                  |          | mannosamine transferase    |
| VC0934     | vpsL | 465                     | -                   | -                   | +                  | -                | II       | Glycosyltransferase        |
| VC0935     | vpsM | 398                     | -                   | -                   | +                  | -                | VI       | Hypothetical protein       |
| VC0936     | vpsN | 175                     | -                   | -                   | +                  | -                | III      | Polysaccharide export      |
| VC0937     | vpsO | 737                     | -                   | -                   | +                  | -                | III      | Polysaccharide polymerase  |
| VC0938     | vpsP | 235                     | +                   | +                   | +++                | ++               | VI       | Hypothetical protein       |
| VC0939     | vpsQ | 144                     | +                   | +                   | +++                | +++              | VI       | Hypothetical protein       |

* Wild-type rugose colony corrugation (+); altered colony morphology with reduced corrugation (+#); flat and smooth colony morphology (-).
† Wild-type rugose pellicle formation and structure (+); altered pellicle structure (+#); no pellicle formation (-).
‡ Wild-type rugose biofilm formation in crystal violet staining assay (+++); moderate reduction in biofilm formation (++); marked reduction in biofilm formation (+).
§ Strong rugose wild-type signal in VPS immunoblot assay (+++); moderate signal (++); weak signal (+); faint signal (+/-); very faint signal (-).
|| Classification based on domains and predicted functions.
¶ Predicted functions and domains assigned by The Institute for Genomic Research-Comprehensive Microbial Resource (TIGR-CMR) and Universal Protein Resource (UniProt) databases.

---

amine/mannosaminuronic acid dehydrogenase (Kawamura et al., 1979, 1985). Since VpsA and VpsB are predicted to be involved in the production of nucleotide sugar precursors, it is not surprising that deletion of *vpsA* and *vpsB* resulted in strains that exhibited decreased colony corrugation (Fig. 2) and reduction in the ability to form pellicle (Fig. 3), biofilm and VPS (Fig. 4).

Class II consists of four predicted glycosyltransferases encoded by *vpsD*, *vpsI*, *vpsK* and *vpsL*. VpsD and VpsI are classified as members of the glycosyltransferase family 4 (GT 4), while VpsK is classified as a member of GT 26 in the Carbohydrate Active Enzymes (CAZY) database (http://www.cazy.org) (Cantarel et al., 2009). Although VpsL is not classified as part of any GT family, the Universal Protein Resource (UniProt) database (http://www.uniprot.org/) shows that VpsL contains a bacterial sugar transferase domain (PF02397), suggesting that VpsL is probably a glycosyltransferase. Glycosyltransferases catalyse the transfer of sugar moieties to specific acceptor molecules

(Campbell et al., 1997; Whitfield, 2006), forming glycosidic bonds that are predicted to be involved in initiation and/or elongation of VPS subunits (repeat units). Thus, the mutants lacking glycosyltransferase activity are unlikely to produce mature VPS. As expected, strains containing in-frame deletion of these genes (RΔ*vpsD*, RΔ*vpsI*, RΔ*vpsK* and RΔ*vpsL*) exhibited flat and smooth colony morphology (Fig. 2). These mutants were also unable to form a pellicle (Fig. 3), and exhibited a significant reduction in biofilm formation and were unable to produce VPS (Fig. 4).

Class III consists of proteins encoded by *vpsE*, *vpsH*, *vpsN* and *vpsO* that are predicted to be involved in VPS polymerization and export. VpsE contains a predicted polysaccharide synthesis domain (PF01943), which is also found in Wzx-like proteins that are O-antigen translocases involved in LPS export in *E. coli* (Marolda et al., 2004, 2006; Whitfield, 1995). Besides the presence of the loosely conserved PF01943 domain, there is very low conservation in the protein primary sequence in these Wzx-like proteins (Marolda et al.,

J. C. N. Fong and others

| Rugose | RΔvps-I | RΔvps-II | RΔvps-IΔvps-II |
| --- | --- | --- | --- |
| ![Image](image1) | ![Image](image2) | ![Image](image3) | ![Image](image4) |

| RΔvpsU | RΔvpsA | RΔvpsB | RΔvpsC | RΔvpsD |
| --- | --- | --- | --- | --- |
| ![Image](image5) | ![Image](image6) | ![Image](image7) | ![Image](image8) | ![Image](image9) |

| RΔvpsE | RΔvpsF | RΔvpsG | RΔvpsCΔvpsG | RΔvpsH |
| --- | --- | --- | --- | --- |
| ![Image](image10) | ![Image](image11) | ![Image](image12) | ![Image](image13) | ![Image](image14) |

| RΔvpsI | RΔvpsJ | RΔvpsK | RΔvpsL | RΔvpsM |
| --- | --- | --- | --- | --- |
| ![Image](image15) | ![Image](image16) | ![Image](image17) | ![Image](image18) | ![Image](image19) |

| RΔvpsN | RΔvpsO | RΔvpsP | RΔvpsQ |
| --- | --- | --- | --- |
| ![Image](image20) | ![Image](image21) | ![Image](image22) | ![Image](image23) |

---

Fig. 2. Colony morphology of vps deletion mutants. Colony pictures of rugose wild-type and vps deletion mutants were taken after cultures had been incubated at 30 °C for 2 days. The assay was repeated with two biological replicates. Scale bars represent 0.5 mm.

2006). Interestingly, PelG from *Pseudomonas aeruginosa*, which is involved in biofilm formation, has also been described as a Wzx-like polysaccharide transporter (Vasseur *et al.*, 2005). Based on computational analysis, VpsE is likely to be a polysaccharide export protein. VpsH is annotated as a putative CapK protein in The Institute for Genomic Research Comprehensive Microbial Resource (TIGR-CMR) and UniProt databases. CapK in *Staphylococcus aureus* is reported to be a possible capsular polysaccharide polymerase (*Sau et al.*, 1997). VpsN is predicted by the UniProt database to encode a polysaccharide export-related protein that contains a polysaccharide export domain (PF02563), also associated with the Wzx-like proteins, suggesting that VpsN may also be involved in polysaccharide export. VpsO is predicted by the

V. cholerae VPS production

Rugose RΔvps-I RΔvps-II Δvps-II RΔvpsU RΔvpsA RΔvpsB RΔvpsC
RΔvpsD RΔvpsE RΔvpsF RΔvpsG ΔvpsG RΔvpsH RΔvpsI RΔvpsJ
RΔvpsK RΔvpsL RΔvpsM RΔvpsN RΔvpsO RΔvpsP RΔvpsQ

Fig. 3. Pellicle formation in vps deletion mutants. Pellicle pictures of rugose wild-type and vps deletion mutants from the top (top panels) and side (lower panels) of the cultures were taken after 2 days of incubation at 30 °C. The assay was repeated with two biological replicates.

UniProt database to contain a PF02706 domain associated with the E. coli chain length determinant protein Wzz (Franco et al., 1998).

Mutants in class III may produce lipid-linked subunits of VPS, but not full-length VPS. If VPS transport takes place after complete polymerization in the cytoplasm or periplasm, VPS could accumulate in the cytoplasm or periplasm. Alternatively, unpolymerized or partially polymerized VPS may be secreted. Mutants lacking functional VpsE, VpsN and VpsO produced flat and smooth colonies (Fig. 2), were unable to form a pellicle (Fig. 3), and exhibited drastic reduction in biofilm formation and VPS production (Fig. 4).

Although a mutant lacking functional VpsH (RΔvpsH) exhibited smooth colony morphology, it is important to note that the colony is more compact compared to the vps-clusters deletion mutants. Furthermore, RΔvpsH formed a pellicle with altered structures (Fig. 3, top panels) and exhibited a moderate decrease in biofilm formation (Fig. 4a). In addition, RΔvpsH produced much less VPS than the rugose wild-type, but not as little as the vps-cluster deletion mutants (Fig. 4b, c), suggesting that production of VPS intermediates may still occur in RΔvpsH.

Class IV consists of proteins that are encoded by vpsC and vpsG. VpsC and VpsG are predicted to be acetyltransferases

http://mic.sgmjournals.org

J. C. N. Fong and others

(a)

![Image](https://i.imgur.com/placeholder.png)

(b)

![Image](https://i.imgur.com/placeholder.png)

(c)

![Image](https://i.imgur.com/placeholder.png)

Fig. 4. Biofilm formation and VPS production in *vps* mutants. (a) Biofilm-forming capacities of the rugose wild-type and *vps* mutants were determined using the crystal violet staining assay on cultures grown at 30 °C for 8 h under static conditions. Results are means of at least five technical replicates and error bars represent standard deviations. (b) VPS production by rugose wild-type and *vps* mutants was determined by immunoblot analysis using crude VPS extract spotted on a nitrocellulose membrane and probed with an anti-VPS antiserum. (c) Quantification of VPS production in rugose wild-type and *vps* mutants. Purified VPS was used to quantify VPS production in the strains indicated. Results are means of at least three technical replicates and error bars represent standard deviations. Assays were repeated with two biological replicates.

involved in the addition of acetyl groups to polysaccharide. Mutants lacking the acetyltransferases were expected to produce VPS without acetyl modifications, which are likely to have different biophysical properties from those of the wild-type. RΔ*vpsG* exhibited a significant reduction in colony corrugation but the colonies were not completely flat and smooth (Fig. 2). RΔ*vpsG* also formed a pellicle with altered structure (Fig. 3, top panel), and showed reduced biofilm formation and VPS production (Fig. 4). In contrast, RΔ*vpsC* retained its rugose colony corrugation (Fig. 2),

Rugose RΔvps-IΔvps-II

2 h

6 h

24 h

P. aeruginosa strains, as well as for maintenance of chronic P. aeruginosa infections (Pier et al., 2001). VPS is highly viscous, and thus non-acetylated VPS may have reduced viscosity, affecting biofilm structure and stability. While the altered rugosity-associated phenotypes exhibited by RΔvpsG are expected, it is surprising that RΔvpsC formed colonies and pellicles that resemble those of the rugose parent. We also generated a strain carrying in-frame deletions of both vpsC and vpsG. This double mutant exhibited phenotypes similar to those of the single vpsG mutant (Figs 2, 3 and 4). It is possible that VpsC is not a functional acetyltransferase, or that VpsG and VpsC may be required for the modification of different portions of the VPS, or even that these different acetyltransferase may be required at different stages of biofilm formation. We are currently testing these hypotheses.

Class V consists of the phosphotyrosine-protein phosphatase encoded by vpsU, and deletion of vpsU resulted in a strain with altered colony morphology (Fig. 2), pellicle structure (Fig. 3), biofilm formation and VPS production (Fig. 4). Phosphotyrosine-protein phosphatases catalyse the dephosphorylation of tyrosine-phosphorylated proteins (Kennelly & Potts, 1999). A number of phosphotyrosine-protein phosphatases in both Gram-positive and Gram-negative bacteria have been identified (Morona et al., 2000, 2002; Soulat et al., 2002; Vincent et al., 1999), and have been shown to be involved in regulation of exopolysaccharide production (Morona et al., 2000; Vincent et al., 2000). The phosphotyrosine-protein phosphatases found in Gram-negative bacteria usually contain two conserved active-site sequence motifs (C-X₄-C-R and D-P-Y) (Kennelly & Potts, 1999; Su et al., 1994; Vincent et al., 1999), while those found in Gram-positive bacteria usually contain four conserved motifs (Aravind & Koonin, 1998; Morona et al., 2002), except for PtpA and PtpB from S. aureus, which resemble phosphotyrosine-protein phosphatases found in Gram-negative bacteria (Soulat et al., 2002). Alignment of VpsU (V. cholerae), Wzb (E. coli), PtpA (P. aeruginosa PAO1) and PtpAB (S. aureus) indeed showed that VpsU contains the two active-site motifs conserved in other similar phosphotyrosine-protein phosphatases (data not shown). The altered rugosity-associated phenotypes

formed a pellicle with structure similar to the rugose variant (Fig. 3) and did not exhibit significant defects in biofilm formation on solid surfaces or VPS production (Fig. 4). The structural modification of alginate by O-acetylation alters its physical properties, including viscosity (Nivens et al., 2001). O-Acetyl groups are required for the formation of mature biofilm structures in alginate-producing mucoid

---

**Fig. 5. Biofilm structure analysis of a vps-clusters deletion mutant. CSLM images of horizontal (xy) and vertical (xz) projections (large and side panels, respectively) of biofilm structures formed by rugose wild-type and a mutant strain unable to produce VPS (RΔvps-IΔvps-II) are shown; scale bars represent 40 μm. Assays were repeated with at least two biological replicates.**

---

**Table 3. COMSTAT analysis of biofilms formed by rugose wild-type and RΔvps-IΔvps-II**

Values are means (standard deviations) of data from at least six z-series image stacks.

| Strain         | Time (h) | Thickness (μm) | Biomass (μm³ μm⁻²) |
|----------------|----------|-----------------|---------------------|
|                |          | Mean            | Maximum             |
| Rugose        | 2        | 2.4 (1.56)      | 24.1 (9.07)         | 1.7 (1.00)       |
|                | 6        | 6.8 (1.89)      | 36.4 (9.24)         | 5.7 (1.40)       |
|                | 24       | 30.8 (3.65)     | 60.9 (7.98)         | 25.5 (2.97)      |
| RΔvps-IΔvps-II | 2        | 3.2 (1.80)      | 10.0 (1.59)         | 2.4 (1.30)       |
|                | 6        | 8.6 (1.48)      | 13.0 (1.91)         | 7.5 (1.42)       |
|                | 24       | 8.9 (2.86)      | 14.3 (3.58)         | 7.7 (2.37)       |

http://mic.sgmjournals.org

exhibited by RΔvpsU indicate a role of VpsU in VPS production, although the target for VpsU remains to be determined. In *E. coli*, Wzb (phosphotyrosine-protein phosphatase) and Wzc (protein-tyrosine kinase) function together in the production of the capsular polysaccharide colanic acid (Vincent *et al.*, 2000). Although the UniProt database suggested that VpsO is similar to the chain length determinant protein Wzz based on the presence of a PF02706 domain, BLASTP search using Wzc peptide sequence identified VpsO as the top match (*E*-value 4.6 × 10⁻²²), suggesting that VpsO may be the target of VpsU. We are currently investigating this hypothesis.

Class VI consists of hypothetical proteins encoded by *vpsF*, *vpsJ*, *vpsM*, *vpsP* and *vpsQ*. While RΔvpsP and RΔvpsQ strains produce VPS and form colonies, pellicles and biofilms similar to those of the wild-type, other mutants lacking *vpsF*, *vpsJ* and *vpsM* exhibited complete loss of colony corrugation, inability to form pellicle and drastic reduction in biofilm formation and VPS production (Figs 2, 3 and 4). Since *vpsF*, *vpsJ* and *vpsM* are predicted to encode hypothetical proteins, identifying the roles of these hypothetical proteins in building the biofilm matrix is critical.

Collectively, phenotypic analysis of the *vps* mutants shows that, under our experimental conditions, most of the VPS proteins are required for wild-type rugose colony corrugation, pellicle and biofilm formation, and VPS production.

(data not shown). We reasoned that competition assays alone may be misleading, as VPS produced by the rugose variant could complement the co-inoculated strain *in trans*, thus masking differences in *in vivo* fitness. Therefore, we performed single-strain infections in the infant mouse model. We first compared the colonization capacities of smooth and rugose wild-type variants. As shown in Fig. 6, the rugose variant colonized the intestine better (approx. 3.6-fold) than the smooth variant. These data suggest that matrix components contribute to colonization and increase *in vivo* fitness, since the rugose variant produces more biofilm matrix than the smooth variant. To test this hypothesis, we compared the colonization capacity of the rugose wild-type variant to those of a mutant that is incapable of producing VPS (RΔvps-IΔvps-II) and mutants that are unable to produce matrix proteins (RΔrbmA and RΔrbmCΔbap1). RbmA, RbmC and Bapl matrix proteins modulate rugosity-associated phenotypes (Fong & Yildiz, 2007; Fong *et al.*, 2006). RbmC and Bapl share 46.7% peptide sequence similarity and appear to be partially redundant. Deletion of *rbmA* and *bap1* in the rugose genetic background resulted in strains with altered colony morphology and biofilm formation when compared to the rugose wild-type (Fong & Yildiz, 2007; Fong *et al.*, 2006). Although deletion of *rbmC* (a homologue of *bap1*) in the rugose genetic background did not alter the colony morphology and biofilm formation significantly, the

---

**Role of biofilm matrix components in *in vivo* fitness**

*V. cholerae* biofilm formation inside the host and its contribution to pathogenesis is not well understood. While some biofilm-related factors have been studied for their roles in *V. cholerae* intestinal colonization (Lauriano *et al.*, 2004; Rashid *et al.*, 2004; Watnick *et al.*, 2001; Zhu & Mekalanos, 2003), different research groups have obtained contradictory results regarding the importance of these factors. For example, mutants lacking VpsR, a positive transcriptional regulator of biofilm matrix production, have given variable results for mouse infection, including normal and decreased colonization (Rashid *et al.*, 2004; Tischler & Camilli, 2005). In another study, a rugose variant (resulting from *flaA* deletion) was found to be defective for infant mouse intestinal colonization in a competition infection, suggesting that a high level of VPS production is detrimental (Watnick *et al.*, 2001). It is therefore evident that many factors, possibly including the presence of VPS and matrix proteins, affect *in vivo* fitness of *V. cholerae*.

To this end, we carried out experiments to investigate whether known biofilm determinants in *V. cholerae* contribute to pathogenesis. We initially performed competition experiments using the smooth (low-biofilm-forming) and rugose (high-biofilm-forming) wild-type variants in the commonly used infant mouse model, but did not see a significant difference in the competitive index

---

Fig. 6. Intestinal colonization phenotypes of *V. cholerae* wild-type and mutant strains in the infant mouse model. Wild-type smooth and rugose variants and rugose mutant strains lacking the genes required for VPS (RΔvps-IΔvps-II) and matrix protein production (RΔrbmA and RΔrbmCΔbap1) were used in single-strain infections. The data shown are pooled from two experiments, and the horizontal bars indicate the median of each dataset. The asterisks indicate significantly different medians of the recovered c.f.u. compared to the rugose variant as determined by Mann–Whitney *U* test (smooth *P*=0.0354, RΔvps-IΔvps-II *P*=0.0043 and RΔrbmA *P*=0.0426).
double deletion mutant RΔrbmCΔbap1 exhibited a marked decrease in biofilm formation (Fong & Yildiz, 2007).

We determined that RΔvps-IΔvps-II exhibited a severe defect in intestinal colonization, with a 9.4-fold lower c.f.u. recovery compared to the rugose variant. This result indicates that VPS production increases in vivo fitness. The mutant lacking the biofilm matrix protein RbmA also exhibited a defect in intestinal colonization, with a 4.1-fold lower c.f.u. recovery. This outcome indicates that VPS and RbmA contribute significantly to in vivo fitness of V. cholerae in these models, and that they contribute to biofilm formation in both in vivo and aquatic environments. In contrast, there was no significant difference in the colonization ability of the RΔrbmCΔbap1 double mutant when compared to the rugose variant, indicating that either RbmC and Bap1 do not significantly influence intestinal colonization, or their phenotype is not evident under the experimental conditions utilized, thus suggesting that the major function of these biofilm matrix proteins is in the formation of biofilms in aquatic environments. Collectively, these results indicate that the biofilm matrix components are important in in vitro biofilm formation, and that VPS and RbmA also contribute to in vivo fitness of V. cholerae.

The results presented in this study have revealed how the various vps gene products contribute to wild-type biofilm formation and VPS biosynthesis, and have laid the groundwork for future biochemical characterization of the VPS biosynthetic pathway. In vitro experiments carried out with the vps mutants in this study, and rbm mutants in previous studies, clearly demonstrate the importance of matrix components, both VPS and matrix proteins, in the building of a robust wild-type biofilm. In-frame deletion of 15 of the 18 vps genes resulted in strains that exhibited drastic altered rugosity-associated phenotypes (Figs 2, 3 and 4). VPS and RbmA also contribute to in vivo fitness of V. cholerae, as shown in in vivo mouse colonization studies, where mutants not able to produce VPS or RbmA were defective for intestinal colonization (Fig. 6).

Humans ingest V. cholerae biofilms as part of the pathogen’s normal transmission route. In this study, we have shown that VPS and biofilm matrix proteins are critical for in vitro biofilm formation and are also important for in vivo fitness. The nature of the VPS coating and matrix proteins in the biofilms may influence the progression of the disease, and/or development of an immune response against V. cholerae. A better understanding of biofilm matrix biosynthesis and function could allow us to develop inhibitors that specifically alter the matrix properties and affect either ex vivo (aquatic) survival or in vivo pathogenesis.

## ACKNOWLEDGEMENTS

This study was supported by grants AI055987 to F.H.Y and AI43486 to K.E.K. The authors also thank N. Shikuma for his help in

---

### REFERENCES

Alam, M., Sultana, M., Nair, G. B., Sack, R. B., Sack, D. A., Siddique, A. K., Ali, A., Huq, A. & Colwell, R. R. (2006). Toxigenic Vibrio cholerae in the aquatic environment of Mathbaria, Bangladesh. Appl Environ Microbiol 72, 2849–2855.

Aravind, L. & Koonin, E. V. (1998). Phosphoesterase domains associated with DNA polymerases of diverse origins. Nucleic Acids Res 26, 3746–3752.

Bao, Y., Lies, D. P., Fu, H. & Roberts, G. P. (1991). An improved Tn7-based system for the single-copy insertion of cloned genes into chromosomes of gram-negative bacteria. Gene 109, 167–168.

Beyhan, S. & Yildiz, F. H. (2007). Smooth to rugose phase variation in Vibrio cholerae can be mediated by a single nucleotide change that targets c-di-GMP signalling pathway. Mol Microbiol 63, 995–1007.

Beyhan, S., Bilecen, K., Salama, S. R., Casper-Lindley, C. & Yildiz, F. H. (2007). Regulation of rugosity and biofilm formation in Vibrio cholerae: comparison of VpsT and VpsR regulons and epistasis analysis of vpsT, vpsR, and hapR. J Bacteriol 189, 388–402.

Broza, M., Gancz, H., Halpern, M. & Kashi, Y. (2005). Adult non-biting midges: possible windborne carriers of Vibrio cholerae non-O1 non-O139. Environ Microbiol 7, 576–585.

Campbell, J. A., Davies, G. J., Bulone, V. & Henrissat, B. (1997). A classification of nucleotide-diphospho-sugar glycosyltransferases based on amino acid sequence similarities. Biochem J 326, 929–939.

Cantarel, B. L., Coutinho, P. M., Rancurel, C., Bernard, T., Lombard, V. & Henrissat, B. (2009). The Carbohydrate-Active EnZymes database (CAZy): an expert resource for glycogenomics. Nucleic Acids Res 37, D233–D238.

Colwell, R. R., Huq, A., Islam, M. S., Aziz, K. M., Yunus, M., Khan, N. H., Mahmud, A., Sack, R. B., Nair, G. B. & other authors (2003). Reduction of cholera in Bangladeshi villages by simple filtration. Proc Natl Acad Sci U S A 100, 1051–1055.

de Lorenzo, V. & Timmis, K. N. (1994). Analysis and construction of stable phenotypes in gram-negative bacteria with Tn5- and Tn10-derived minitransposons. Methods Enzymol 235, 386–405.

Enos-Berlage, J. L. & McCarter, L. L. (2000). Relation of capsular polysaccharide production and colonial cell organization to colony morphology in Vibrio parahaemolyticus. J Bacteriol 182, 5513–5520.

Faruque, S. M., Albert, M. J. & Mekalanos, J. J. (1998). Epidemiology, genetics, and ecology of toxigenic Vibrio cholerae. Microbiol Mol Biol Rev 62, 1301–1314.

Faruque, S. M., Biswas, K., Udden, S. M., Ahmad, Q. S., Sack, D. A., Nair, G. B. & Mekalanos, J. J. (2006). Transmissibility of cholera: in vivo-formed biofilms and their relationship to infectivity and persistence in the environment. Proc Natl Acad Sci U S A 103, 6350–6355.

Fong, J. C. & Yildiz, F. H. (2007). The rmbBCDEF gene cluster modulates development of rugose colony morphology and biofilm formation in Vibrio cholerae. J Bacteriol 189, 2319–2330.

Fong, J. C. & Yildiz, F. H. (2008). Interplay between cyclic AMP-cyclic AMP receptor protein and cyclic di-GMP signaling in Vibrio cholerae biofilm formation. J Bacteriol 190, 6646–6659.

Fong, J. C., Karplus, K., Schoolnik, G. K. & Yildiz, F. H. (2006). Identification and characterization of RbmA, a novel protein required for the development of rugose colony morphology and biofilm structure in Vibrio cholerae. J Bacteriol 188, 1049–1059.
J. C. N. Fong and others

Franco, A. V., Liu, D. & Reeves, P. R. (1998). The Wzz (Cld) protein in Escherichia coli: amino acid sequence variation determines O-antigen chain length specificity. *J Bacteriol* 180, 2670–2675.

Fullner, K. J. & Mekalanos, J. J. (1999). Genetic characterization of a new type IV-A pilus gene cluster found in both classical and El Tor biotypes of Vibrio cholerae. *Infect Immun* 67, 1393–1404.

Halpern, M., Broza, Y. B., Mittler, S., Arakawa, E. & Broza, M. (2004). Chironomid egg masses as a natural reservoir of Vibrio cholerae non-O1 and non-O139 in freshwater habitats. *Microb Ecol* 47, 341–349.

Hang, L., John, M., Asaduzzaman, M., Bridges, E. A., Vanderspurt, C., Kirn, T. J., Taylor, R. K., Hillman, J. D., Progulske-Fox, A. & other authors (2003). Use of in vivo-induced antigen technology (IVIAT) to identify genes uniquely expressed during human infection with Vibrio cholerae. *Proc Natl Acad Sci U S A* 100, 8508–8513.

Herrero, M., de Lorenzo, V. & Timmis, K. N. (1990). Transposon vectors containing non-antibiotic resistance selection markers for cloning and stable chromosomal insertion of foreign genes in gram-negative bacteria. *J Bacteriol* 172, 6557–6567.

Heydorn, A., Nielsen, A. T., Hentzer, M., Sternberg, C., Givskov, M., Ersboll, B. K. & Molin, S. (2000). Quantification of biofilm structures by the novel computer program COMSTAT. *Microbiology* 146, 2395–2407.

Hung, D. T., Zhu, J., Sturtevant, D. & Mekalanos, J. J. (2006). Bile acids stimulate biofilm formation in Vibrio cholerae. *Mol Microbiol* 59, 193–201.

Huo, A., Xu, B., Chowdhury, M. A., Islam, M. S., Montilla, R. & Colwell, R. R. (1996). A simple filtration method to remove plankton-associated Vibrio cholerae in raw water supplies in developing countries. *Appl Environ Microbiol* 62, 2508–2512.

Huq, A., Small, E. B., West, P. A., Huq, M. I., Rahman, R. & Colwell, R. R. (1983). Ecological relationships between Vibrio cholerae and planktonic crustacean copepods. *Appl Environ Microbiol* 45, 275–283.

Huq, A., Huq, S. A., Grimes, D. J., O’Brien, M., Chu, K. H., Capuzzo, J. M. & Colwell, R. R. (1986). Colonization of the gut of the blue crab (Callinectes sapidus) by Vibrio cholerae. *Appl Environ Microbiol* 52, 586–588.

Huq, A., Colwell, R. R., Chowdhury, M. A., Xu, B., Moniruzzaman, S. M., Islam, M. S., Yunus, M. & Albert, M. J. (1995). Coexistence of Vibrio cholerae O1 and O139 Bengal in plankton in Bangladesh. *Lancet* 345, 1249.

Kaper, J. B., Morris, J. G., Jr & Levine, M. M. (1995). Cholera. *Clin Microbiol Rev* 8, 48–86.

Kawamura, T., Ishimoto, N. & Ito, E. (1979). Enzymatic synthesis of uridine diphosphate N-acetyl-D-mannosaminuronic acid. *J Biol Chem* 254, 8457–8465.

Kawamura, T., Ichihara, N., Sugiyama, S., Yokota, H., Ishimoto, N. & Ito, E. (1985). Biosynthesis of UDP-N-acetyl-D-glucosaminuronic acid and UDP-N-acetyl-D-mannosaminuronic acid in Micrococcus luteus. *J Biochem* 98, 105–116.

Kennelly, P. J. & Potts, M. (1999). Life among the primitives: protein O-phosphatases in prokaryotes. *Front Biosci* 4, D372–D385.

Kierek, K. & Watnick, P. I. (2003). Environmental determinants of Vibrio cholerae biofilm development. *Appl Environ Microbiol* 69, 5079–5088.

Lauriano, C. M., Ghosh, C., Correa, N. E. & Klose, K. E. (2004). The sodium-driven flagellar motor controls exopolysaccharide expression in Vibrio cholerae. *J Bacteriol* 186, 4864–4874.

Lee, S. H., Butler, S. M. & Camilli, A. (2001). Selection for in vivo regulators of bacterial virulence. *Proc Natl Acad Sci U S A* 98, 6889–6894.

Lin, W., Kovacikova, G. & Skorupski, K. (2007). The quorum sensing regulator HapR downregulates the expression of the virulence gene transcription factor AphA in Vibrio cholerae by antagonizing Lrp- and VpsR-mediated activation. *Mol Microbiol* 64, 953–967.

Lombardo, M. J., Michalski, J., Martinez-Wilson, H., Morin, C., Hilton, T., Osorio, C. G., Nataro, J. P., Tacket, C. O., Camilli, A. & other authors (2007). An in vivo expression technology screen for Vibrio cholerae genes expressed in human volunteers. *Proc Natl Acad Sci U S A* 104, 18229–18234.

Marolda, C. L., Vicarioli, J. & Valvano, M. A. (2004). Wzx proteins involved in biosynthesis of O antigen function in association with the first sugar of the O-specific lipopolysaccharide subunit. *Microbiology* 150, 4095–4105.

Marolda, C. L., Tatar, L. D., Alaimo, C., Aebi, M. & Valvano, M. A. (2006). Interplay of the Wzx translocase and the corresponding polymerase and chain length regulator proteins in the translocation and periplasmic assembly of lipopolysaccharide O antigen. *J Bacteriol* 188, 5124–5135.

Morona, J. K., Paton, J. C., Miller, D. C. & Morona, R. (2000). Tyrosine phosphorylation of CpsD negatively regulates capsular polysaccharide biosynthesis in Streptococcus pneumoniae. *Mol Microbiol* 35, 1431–1442.

Morona, J. K., Morona, R., Miller, D. C. & Paton, J. C. (2002). Streptococcus pneumoniae capsule biosynthesis protein CpsB is a novel manganese-dependent phosphotyrosine-protein phosphatase. *J Bacteriol* 184, 577–583.

Nivens, D. E., Ohman, D. E., Williams, J. & Franklin, M. J. (2001). Role of alginate and its O acetylation in formation of Pseudomonas aeruginosa microcolonies and biofilms. *J Bacteriol* 183, 1047–1057.

Pier, G. B., Coleman, F., Grout, M., Franklin, M. & Ohman, D. E. (2001). Role of alginate O acetylation in resistance of mucoid Pseudomonas aeruginosa to opsonic phagocytosis. *Infect Immun* 69, 1895–1901.

Rashid, M. H., Rajanna, C., Zhang, D., Pasquale, V., Magder, L. S., Ali, A., Dumontet, S. & Karaolis, D. K. (2004). Role of exopolysaccharide, the rugose phenotype and VpsR in the pathogenesis of epidemic Vibrio cholerae. *FEMS Microbiol Lett* 230, 105–113.

Sau, S., Bhasin, N., Wann, E. R., Lee, J. C., Foster, T. J. & Lee, C. Y. (1997). The Staphylococcus aureus allelic genetic loci for serotype 5 and 8 capsule expression contain the type-specific genes flanked by common genes. *Microbiology* 143, 2395–2405.

Soulat, D., Vaganay, E., Duclos, B., Genestier, A. L., Etienne, J. & Cozzone, A. J. (2002). Staphylococcus aureus contains two low-molecular-mass phosphotyrosine protein phosphatases. *J Bacteriol* 184, 5194–5199.

Su, X. D., Taddei, N., Stefani, M., Ramponi, G. & Nordlund, P. (1994). The crystal structure of a low-molecular-weight phosphotyrosine protein phosphatase. *Nature* 370, 575–578.

Tischler, A. D. & Camilli, A. (2005). Cyclic diguanylate regulates Vibrio cholerae virulence gene expression. *Infect Immun* 73, 5873–5882.

Vasseur, P., Vallet-Gely, I., Soscia, C., Genin, S. & Filloux, A. (2005). The pel genes of the Pseudomonas aeruginosa PAK strain are involved at early and late stages of biofilm formation. *Microbiology* 151, 985–997.

Vincent, C., Doublet, P., Grangeasse, C., Vaganay, E., Cozzone, A. J. & Duclos, B. (1999). Cells of Escherichia coli contain a protein-tyrosine kinase, Wzc, and a phosphotyrosine-protein phosphatase, Wzb. *J Bacteriol* 181, 3472–3477.

Vincent, C., Duclos, B., Grangeasse, C., Vaganay, E., Ribery, M., Cozzone, A. J. & Doublet, P. (2000). Relationship between exopolysaccharide production and protein-tyrosine phosphorylation in gram-negative bacteria. *J Mol Biol* 304, 311–321.

Watnick, P. I., Lauriano, C. M., Klose, K. E., Croal, L. & Kolter, R. (2001). The absence of a flagellum leads to altered colony morphology, biofilm development and virulence in Vibrio cholerae O139. Mol Microbiol 39, 223–235.

Whitfield, C. (1995). Biosynthesis of lipopolysaccharide O antigens. Trends Microbiol 3, 178–185.

Whitfield, C. (2006). Biosynthesis and assembly of capsular polysaccharides in Escherichia coli. Annu Rev Biochem 75, 39–68.

Yildiz, F. H. & Schoolnik, G. K. (1999). Vibrio cholerae O1 El Tor: identification of a gene cluster required for the rugose colony type,

exopolysaccharide production, chlorine resistance, and biofilm formation. Proc Natl Acad Sci U S A 96, 4028–4033.

Yildiz, F. H., Dolganov, N. A. & Schoolnik, G. K. (2001). VpsR, a member of the response regulators of the two-component regulatory systems, is required for expression of vps biosynthesis genes and EPS<sup>ETr</sup>-associated phenotypes in Vibrio cholerae O1 El Tor. J Bacteriol 183, 1716–1726.

Zhu, J. & Mekalanos, J. J. (2003). Quorum sensing-dependent biofilms enhance colonization in Vibrio cholerae. Dev Cell 5, 647–656.

Edited by: V. Sperandio

http://mic.sgmjournals.org
