
Cell Research (2010) 20:812-825.
© 2010 IBCB, SIBS, CAS All rights reserved 1001-0602/10 $32.00
www.nature.com/cr

---

**ORIGINAL ARTICLE**

**The molecular cell death machinery in the simple cnidarian Hydra includes an expanded caspase family and pro- and anti-apoptotic Bcl-2 proteins**

Margherita Lasi¹, Barbara Pauly¹, Nikola Schmidt¹, Mihai Cikala¹, Beate Stiening¹, Tina Käsbauer¹, Gerhardt Zenner¹, Tanja Popp¹, Anita Wagner¹, Regina T Knapp¹, Andreas H Huber¹, Michaela Grunert¹, Johannes Söding², Charles N David¹, Angelika Böttger¹

¹Department Biology II, Ludwig-Maximilians University München, Großhaderner Str. 2, 82152 Planegg-Martinsried, Germany;  
²Gene Centre; Feodor-Lynen-Str. 25, 81377 München, Germany

---

The fresh water polyp *Hydra* belongs to the phylum Cnidaria, which diverged from the metazoan lineage before the appearance of bilaterians. In order to understand the evolution of apoptosis in metazoans, we have begun to elucidate the molecular cell death machinery in this model organism. Based on ESTs and the whole *Hydra* genome assembly, we have identified 15 caspases. We show that one is activated during apoptosis, four have characteristics of initiator caspases with N-terminal DED, CARD or DD domain and two undergo autoprocessing *in vitro*. In addition, we describe seven Bcl-2-like and two Bak-like proteins. For most of the Bcl-2 family proteins, we have observed mitochondrial localization. When expressed in mammalian cells, HyBak-like 1 and 2 strongly induced apoptosis. Six of the Bcl-2 family members inhibited apoptosis induced by camptothecin in mammalian cells with HyBcl-2-like 4 showing an especially strong protective effect. This protein also interacted with HyBak-like 1 in a yeast two-hybrid assay. Mutation of the conserved leucine in its BH3 domain abolished both the interaction with HyBak-like 1 and the anti-apoptotic effect. Moreover, we describe novel *Hydra* BH-3-only proteins. One of these interacted with Bcl-2-like 4 and induced apoptosis in mammalian cells. Our data indicate that the evolution of a complex network for cell death regulation arose at the earliest and simplest level of multicellular organization, where it exhibited a substantially higher level of complexity than in the protostome model organisms *Caenorhabditis* and *Drosophila*.

**Keywords:** apoptosis; Bcl-2; BH3-only; caspase; *Hydra*

Cell Research (2010) 20:812-825. doi:10.1038/cr.2010.66; published online 18 May 2010

---

**Introduction**

The mechanisms of programmed cell death in invertebrates have mainly been studied in *Caenorhabditis elegans* and *Drosophila melanogaster*. Comparison of the molecular pathways of apoptosis in invertebrates with the elaborate cell death machinery in vertebrates has shown that they are highly conserved throughout evolution. The major players are caspases, caspase inhibitors, members of the Bcl-2 family of pro- and anti-apoptotic proteins

---

Correspondence: Angelika Böttger  
Tel: +49 (0)89 218074 279; Fax: +49 (0)89 218074 219  
E-mail: boettger@zi.biologie.uni-muenchen.de  
Received 9 February 2010; revised 28 February 2010; accepted 3 March 2010; published online 18 May 2010

and adaptors of the Ced-4/APAF-1 type (reviewed in [1-3]). Cell death is mediated extrinsically by members of the TNF family of ligands and their receptors, which rely on adaptor proteins that link activated receptors to intracellular cell death executioners (reviewed in [4]).

Despite the high degree of conservation of these proteins, the level of complexity of the cell death pathways differs between *Drosophila* and *Caenorhabditis*. For example, extrinsic cell death pathways and caspase inhibitors of the IAP type have not been described in *Caenorhabditis* so far. Only one caspase, Ced-3, is involved in cell death in the nematode, whereas seven are present in *Drosophila*, including initiator and effector caspases. Similarly, differences exist in the Bcl-2 family of pro- and anti-apoptotic proteins. *Caenorhabditis* expresses the multidomain Bcl-2 homolog Ced-9 with
a primarily anti-apoptotic function and the BH3-only proapoptotic protein Egl-1. In *Drosophila*, no functional anti-apoptotic Bcl-2 homolog has been found, although two homologs of multidomain Bcl-2 family members, Debcl and BUFFY, are present. Debcl has been shown to harbor a pro-apoptotic function, whereas BUFFY can apparently have anti-apoptotic characteristics *in vivo* in contrast to its ability to induce cell death when overexpressed ectopically in S2 insect cells [5].

In the face of such a diversity of apoptotic pathways in *Caenorhabditis* and *Drosophila*, it appears important to look for apoptotic networks in evolutionarily older metazoans. The phylum Cnidaria branched early from the main stem leading to bilaterian animals. Its members form two germ layers from which ectodermal and endodermal tissues are derived. They possess a simple nervous system constituting a nerve net across the whole animal without clearly defined central ganglia or brain-like structures. Caspase-dependent apoptosis has been described in the fresh water polyp Hydra, where it regulates cell numbers in response to feeding and is involved in oogenesis and spermatogenesis [6–10]. In the colonial hydrozoan *Hydractinia echinata*, it is necessary for metamorphosis [11, 12]. Recent genome sequencing projects have indicated the presence of large families of apoptotic proteins, including caspases, Bcl-2 family proteins and APAF-1 in cnidarians [13–15]. However, their functional interactions and the roles they have in apoptosis are unknown.

In this study, we have focused on caspase and Bcl-2 family genes from *Hydra magnipapillata*. We have carried out functional studies to examine whether sequence homologies are paralleled by similarities in the function of apoptotic networks. Our results suggest that apoptotic pathways in cnidarians show almost the same functional complexity as in vertebrates.

### Results

#### Caspases

A search of whole genome and EST sequences from *Hydra* has revealed 17 sequences with homology to caspases. With the exception of two (designated pseudocaspases), all have putative active-site sequences with conserved histidine and cysteine residues [15]. Two have been published previously (*HvCasp3A* and *3B* [16]) and have now been renamed as *HyCaspA* and *HyCaspB* [15]. Nine of the *Hydra* caspase sequences have relatively long prodomains. A search for conserved protein-protein interaction motifs in these prodomains showed that two have CARD domains, one has a DED domain and, unusually for metazoan caspases, one has a DD domain.

To further analyze *Hydra* caspases, we isolated cDNAs for the new genes *HyCaspC* and *D* and from all caspases with CARD, DED and DD domains. Figure 1 shows a schematic representation of the eight caspases that were investigated in these experiments. A review of all *Hydra* caspase sequences that have been identified in the *Hydra* genome has been published recently [15].

In the face of the complexity of caspase sequences in *Hydra*, we first analyzed whether some of them were responsible for caspase-3 activity in hydra extracts. Previous work using fluorogenic substrates had established that a DEVD-specific proteolytic activity is present in lysates of hydra cells in which apoptosis was induced [16]. This activity could be labeled with the pan-caspase-specific active-site labeling reagent z-EK(biotin)-D-AOMK. The activity also developed spontaneously in extracts of hydra cells incubated at room temperature for 1 to 2 h (data not shown). Active-site labeling was inhibited by the pan-caspase inhibitor z-VAD-FMK and the caspase-3 specific inhibitor DEVD-FMK, but not by the ICE-inhibitor YVAD-FMK (Figure 2A).

In order to establish the molecular identity of this activity, we purified the activated caspases responsible for cleaving the specific caspase-3 substrate DEVD-AMC from *Hydra* extracts. These extracts had been left at room temperature for 1 h and exhibited DEVD-AMC cleaving activity. After ion exchange chromatography and gel filtration, DEVD-AMC cleaving fractions were

![Diagram](https://i.imgur.com/yourimage.png)

Figure 1 *Hydra* caspases. Schematic representation of eight *Hydra* caspases showing amino acids constituting the active sites (with catalytic cysteine and histidine residues in bold) and aspartate residues in putative processing sites (D followed by black bars); conserved CARD, DED and DD domains are indicated.
HyCaspA was not.

We next investigated possible caspase activation cascades. In addition to the previously published Hy-CaspA and B, we cloned HyCaspC and D (HyCaspD has a very short prodomain of only eight amino acids), the two CARD-caspases, HyCARD1 and 2, the DED-caspase, HyDEDCasp, and the caspase with the DD domain, HyDDCasp, from *Hydra vulgaris* cDNA. We then expressed myc-tagged versions of these caspases in *E. coli*. For HyCaspA and HyDDCasp, we found evidence for autoprocessing and activation, as shown in Figure 3. Western blots of soluble extracts from bacteria expressing myc-tagged wild-type HyCaspA stained with anti-HyCaspA or anti-myc antibodies yielded three bands corresponding to full-length, unprocessed HyCaspA (PLS-myc), large subunit plus prodomain (PL) and the small subunit including the myc tag (S-myc) (Figure 3A and 3B). Expression of an active-site mutant (C211S) yielded only full-length unprocessed protein (PLS-myc). Deletion of the prodomain also blocked autoprocessing and yielded only the uncleaved product (LS-myc; Figure 3E). N-terminal sequencing of the small subunit (S-myc) indicated that autoprocessing occurred after aspartate in the sequence IRKDGIK (Figure 3G).

As expected, the 30 kDa fragment containing the large subunit could be labeled with the pan-caspase active-site label, FITC-VAD-FMK, and then detected with anti-FITC antibody after SDS-PAGE and western blot (Figure 3C). The active-site mutant (C211S) could not be labeled with this active-site reagent (data not shown). Together, these results clearly demonstrate that recombinant HyCaspA undergoes self-activation by autocatalytic cleavage and that this processing requires the prodomain of HyCaspA, although this prodomain has none of the known signatures for apoptotic caspases, such as CARD or DED domains.

In lysates from bacteria expressing HyDDCasp, only a small fraction of recombinant protein underwent autoprocessing, but this could be clearly documented with the active-site label FITC-VAD-FMK. The wild-type enzyme, but not the active-site mutant (C268S), yielded a 20 kDa band corresponding to the large subunit (Figure 3F).

To determine the substrate specificity of the HyCaspA and HyDDCasp, we performed competition experiments with unlabeled inhibitors. Figure 3C shows that the activity of HyCaspA was inhibited by the pan-caspase inhibitor z-VAD-FMK, but not by the caspase 3-specific inhibitor Ac-DEVD-CHO and the ICE-inhibitor Ac-YVAD-CHO. The HyDDCasp activity, on the other hand, was inhibited by Ac-YVAD-CHO, but not by Ac-DEVD-CHO or Ac-IETD-CHO (Figure 3D). Thus neither en-

A
Full-length HyCaspA
kDa
79.0-
50.1-
34.7-
28.4-
20.8-
WT
C(211)S
PLS-myc
PL
S-myc
Anti-HyCaspA

B
Full-length HyCaspA
kDa
79.0-
50.1-
34.7-
28.4-
20.8-
WT
C(211)S
PLS-myc
S-myc
Anti-myc

C
HyCaspA
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + + + +
+ + +
enzyme had the DEVD substrate specificity that was found in apoptotic extracts of hydra cells. Moreover, they both exhibited different substrate preferences, indicating that they cleave different substrates *in vivo*.

Similar experiments with HyCaspB, C, D, HyDED-Casp and HyCARD1 and 2 did not provide any evidence for autoprocessing or catalytic activity when expressed in *E. coli* (data not shown).

### Bcl-2 family proteins

Caspase activation in metazoans involves permeabilization of the mitochondrial outer membrane, which leads to the release of pro-apoptotic factors. In mammals, these include cytochrome *c*, which participates in the formation of an apoptosome, including APAF-1 and procaspases. The permeability of the mitochondrial outer membrane is regulated by proteins of the Bcl-2 family, and hence, we searched the *Hydra* genome and EST sequences for Bcl-2 homologs. We identified nine Bcl-2 family members: two sequences show high similarity to vertebrate Bak proteins and were designated HyBak-like 1 and 2; the remaining sequences were designated HyBcl-2-like

**A**

| BH1 domain |  | BH2 domain |
| --- | --- | --- |
| Human Bcl-2 | VVEELFRDG-VNWGRIVAFFEGGV | Human Bcl-2 | TWIQDNG-GWDAFVEL |
| Human Bcl-xL | VVNELEFRDG-VNWGRIVAFFSFGGA | Human Bcl-xL | PWIQENG-GWDTFVEL |
| Human Bax | VAADMESDGNFNWGRVVVALFYFAK | Human Bax | GWIQDQG-GWDGLLSY |
| HyBcl-2-like 1 | VSQKVLTDDNLNWGRVVSPLITFGGK | HyBcl-2-like 1 | -NIKQKG-GWDDFNRT |
| HyBcl-2-like 2 | IAEEIFSDG-INWGRRIIVLYAFAGK | HyBcl-2-like 2 | -WIRDSGRGWNGFIEN |
| HyBcl-2-like 3 | VSQEVLADDNLNWGRVVSPLITFGGK | HyBcl-2-like 3 | -WIIEKG-GWDNFNRT |
| HyBcl-2-like 4 | IADEIFVSG-KNWGRVVALFLTFGGN | HyBcl-2-like 4 | YWINQQG-GWDGFLIF |
| HyBcl-2-like 5 | VLRNVLKDS-INFGRITSILAVCMI | HyBcl-2-like 5 | -WFEANG-AWDGLMKY |
| HyBcl-2-like 6 | VAKETFADGTKNWGRVLAITYLAGC | HyBcl-2-like 6 | DWIEEQG-GWDDLPKK |
| HyBcl-2-like 7 | NFSBEAMFEKCISWPLIVSFLFSAN | HyBcl-2-like 7 | GWIENG-GWEGLASF |
| HyBak-like 1 | IARRLFENG-INWGRIVALLLCFGYE | HyBak-like 1 | KWIANKG-GWLAALVG |
| HyBak-like 2 | VAKKLFPENDNIAWSHIVSLICFGVE | HyBak-like 2 | QWIAQHG-GWIKMIDF |

| BH3 domain |  | BH4 domain |
| --- | --- | --- |
| Human Bcl-2 | VVHLTLRQAGDDFSRRYRD--- | Human Bcl-2 | -DNREIVMKYTHYKLQR-GYEW |
| Human Bax | KLSECCLKRIGDEL-DSNMELQ--- | HyBcl-2-like 1 | GFIIRKLANDHHYKLF--G--- |
| HyBcl-2-like 1 | TTVVTLRRVSSELEDMHIDF--- | HyBcl-2-like 2 | --PEAKIAKEVAYQLIQKKTGK |
| HyBcl-2-like 2 | RKETELMMRLVE---EIEIKHELf | HyBcl-2-like 3 | GFIIRKLANDHHYKLF--G--- |
| HyBcl-2-like 3 | TTAVTTLRRVSSELEDMHIDF--- | HyBcl-2-like 4 | NYKRI-VKSYVGEEKLRKKGL-- |
| HyBcl-2-like 4 | EIAKTLORVGDELESANTDF---- | HyBcl-2-like 5 | --LNNEIKLSKTYIDSLLVATY |
| HyBcl-2-like 5 | EKVFECL---ELVSKHETF----- | HyBcl-2-like 6 | SIINTAFAIAMYYIPFHLRSSV |
| HyBcl-2-like 6 | TTVAKLCSNM-ELDNAFF------- | HyBcl-2-like 7 | TQAILEQLVDHEMK---------- |
| HyBcl-2-like 7 | IIIAILQLKNSLLLKLSPGLYN--- |  |  |
| HyBak-like 1 | NLGRVLASFGBEINDKYRQVF---- |  |  |
| HyBak-like 2 | KAVQILEEQGGIAIQKYHHAF---- |  |  |

**B**

|  | 5'UTR | BH4 | BH3 BH1 | BH2 TM 3'UTR |
| --- | --- | --- | --- | --- |
| Human Bcl-2 |  |  |  |  |
| Human Bax |  |  |  |  |
| Human Bak |  |  |  |  |
| C.e. Ced-9 |  |  |  |  |
| HyBak-like 1 |  |  |  |  |
| HyBak-like 2 |  |  |  |  |
| HyBcl-2-like 1 |  |  |  |  |
| HyBcl-2-like 2 |  |  |  |  |
| HyBcl-2-like 3 |  |  |  |  |
| HyBcl-2-like 4 |  |  |  |  |
| HyBcl-2-like 5 |  |  |  |  |
| HyBcl-2-like 6 |  |  |  |  |
| HyBcl-2-like 7 |  |  |  |  |

Figure 4 *Hydra* multidomain Bcl-2 family members. (A) Alignment of the BH domains of HyBcl-2 family proteins with BH domains from human homologs. Conserved amino acids are shaded in gray; (B) Exon/intron organization of *Hydra* Bcl-2 genes compared to human Bcl-2, Bax and Bak and ced-9 from *C. elegans* (information from [17]). Gray bar: 5′- and 3′-UTRs, dark gray bar: coding region; BH domains (patterned) and TM domains (black) are shown; introns are not to scale.
[15]. Figure 4A shows alignments of the BH domains of the *Hydra* Bcl-2 family members. The BH1 and BH2 domains are very well conserved and show high sequence similarity with BH1 and BH2 domains from human Bcl-2 and human Bax. The BH3 domains, in contrast, are not well conserved, except for the presence of a conserved leucine residue in all sequences. Moreover, all *Hydra* Bcl-2-like and Bak-like proteins have a C-terminal transmembrane domain (Figure 4B).

All *Hydra* *bcl-2* and *bak* genes have a conserved intron, splitting the coding region of the BH2 domain (Figure 4B). This intron is also found in human and invertebrate *bcl-2* genes [17], supporting the idea that the *Hydra* genes are orthologs of the mammalian *bcl-2* genes. Additional introns are found in the gene encoding HyBcl-2-like 7.

In order to understand the function of Bcl-2 proteins in *Hydra*, we analyzed their intracellular localization by expressing GFP-HyBcl-2-like and GFP-HyBak-like

proteins in *Hydra* cells (Figure 5). Most of the Bcl-2-like proteins and both Bak-like proteins were localized to mitochondria. Deletion of the C-terminal transmembrane domain from HyBcl-2-like 1 blocked the association with mitochondria in *Hydra* (data not shown).

HyBcl-2-like 5, in comparison, localized to the cytoplasm and the nucleus in addition to mitochondria (Figure 5B), while HyBcl-2-like 7 localized to ER-like structures in the cytoplasm and not to mitochondria. This is shown by comparison of its localization with that of an RFP-tagged *Hydra* Bax inhibitor 1, a highly conserved integral membrane protein of the ER (see Materials and Methods) (Figure 5C).

We next expressed the *Hydra* Bcl-2 proteins in human HEK293T cells in order to test their effects on apoptosis. Apoptotic cells were scored by staining with the DNA dye TO-PRO-3 (see Figure 6A) and counting the percentage of apoptotic cells, in relation to all GFP-expressing cells. The result is shown in Figure 6B. Both HyBak-like

**Figure 5 Intracellular localization of *Hydra* Bcl-2 family proteins.** Confocal images of single ectodermal cells in the *Hydra* body column expressing GFP-fusion proteins as indicated. Lower panels show larger magnifications from upper panels; animals were counterstained with anti-ATP-synthase antibody and anti-mouse-Cy3 secondary antibody (A, B, D); in merged images GFP in green, ATP-synthase in red and nuclei, stained with TO-PRO-3, in blue; (C) GFP-HyBcl-2-like 7 was co-expressed with RFP-HyBax inhibitor 1 (RFP-BI-1); in merged image GFP-HyBcl-2-like 7 in green, RFP-BI-1 in red and nuclei stained with TO-PRO-3 in blue; single optical sections; Scale bars = 10 µm.

www.cell-research.com | Cell Research
A

| GFP-HyBak-like 2 | TO-PRO-3 | Merged |
| --- | --- | --- |

| GFP-HyBcl-2-like 6 | TO-PRO-3 | Merged |
| --- | --- | --- |

B

| % of apoptotic cells |
| --- |
| HyBak1 - ETENLGRVLASFGDEINDKYRQVF - |
| HyBakmut - ETENLGRVEASFGDEINDKYRQVF - |

C

| % of apoptotic cells |
| --- |
| HyBcl2-4 - IEIAKTLQRVGDELESANTDF - |
| HyBcl2-4 mut - IEIAKTEQRVGDELESANTDF - |

Figure 6 Effect of Hydra Bcl-2 family proteins on apoptosis. (A) HEK 293T-cells 24 h after transfection of plasmids encoding GFP-HyBak-like 2 (upper panel) or GFP-HyBcl-2-like 6 (lower panel); confocal images, single sections; upper panel shows two apoptotic GFP-HyBak-like 2 expressing cells with round shape and pyknotic chromatin stained with the DNA dye TO-PRO-3; Lower panel shows three GFP-Hy-Bcl-2-like 6 expressing cells with normal interphase nuclei and GFP-HyBcl-2-like 6 on mitochondria; Scale bar = 10 μm; these images provide reference for apoptotic nuclei in comparison with normal nuclei on which quantification of apoptosis as shown in B and C was based; (B, C) Diagrams showing percentage of apoptotic GFP-positive cells in relation to all GFP-positive cells after transfection of plasmids encoding the indicated GFP-fusion proteins. For control, GFP and human caspase-8 (HsC8) were used as indicated. Inserts show the BH3 domain sequence, which was mutated in HyBak-like 1 mut and HyBcl-2-like 4 mut; (B) Induction of apoptosis by HyBcl-2 family proteins; (C) Inhibition of apoptosis by HyBcl-2 proteins treated with 10 μm camptothecin for 24 h before analysis.

proteins strongly induced apoptosis in this assay, as did human caspase-8, which was used as a control. Mutation (L265E) of the conserved leucine residue in the BH3 domain of HyBak-like 1 abolished this pro-apoptotic effect. While most of the remaining Hydra Bcl-2-like proteins had no effect when transfected into HEK293T cells, HyBcl-2-like 5 did exhibit a weak pro-apoptotic effect.

To analyze the anti-apoptotic effect of Hydra Bcl-2 family members, we transfected HEK293T cells with plasmids encoding Hydra Bcl-2 family proteins and then induced apoptosis with the topoisomerase inhibitor camptothecin. Except for HyBcl-2-like 5 (as expected because of its pro-apoptotic effect), all HyBcl-2-like proteins showed some protective effect. HyBcl-2-like 4 showed the strongest inhibition of camptothecin-induced apoptosis. Mutation (L256E) of the conserved leucine residue in the BH3 domain of HyBcl-2-like 4 abolished the protective effect (Figure 6C).

In addition, we analyzed interactions between Bcl-2 family members in a yeast two-hybrid system. The read-out for interaction of two proteins in this system was growth of yeast on minimal medium lacking leucine and expression of a GFP reporter. Table 1 shows that Bcl-2-like 4 was capable of interacting with itself, with HyBak-like 1 and with human Bcl-xL. When the leucine residue in the BH3 domain of Bcl-2-like 4 was mutated (L256E), these interactions were abolished. All the other Hydra Bcl-2 family members failed to show interactions in the yeast two-hybrid system (Table 1), although they were expressed in yeast at similar levels based on protein detection with antibodies on western blots (data not shown).

### BH3-only proteins

In view of the functional conservation of Hydra Bcl-2 family proteins in regulating apoptosis, we searched the Hydra genome for genes encoding BH3-only proteins, which might act to induce apoptosis *in vivo*. We carried out a profiled search for BH3 domains using structural

**Table 1 Interactions of HyBcl-2-like and Bak-like proteins in yeast two-hybrid assay**

| Bait/Prey | HsBcl-xL | HyBcl-2-1 | HyBcl-2-2 | HyBcl-2-4 | HyBcl-2-4<sup>Mut</sup> | HyBcl-2-5 | HyBcl-2-6 | HyBcl-2-7 | HyBak1 | HyBak2 |
|-----------|----------|-----------|-----------|-----------|------------------------|-----------|-----------|-----------|--------|--------|
| HsBcl-xL  | +        | –         | –         | +         | –                      | –         | –         | –         | –      | –      |
| HyBcl-2-1 | –        | –         | –         | n.d.      | –                      | –         | –         | –         | –      | –      |
| HyBcl-2-2 | –        | –         | –         | n.d.      | –                      | –         | –         | –         | –      | –      |
| HyBcl-2-4 | +        | –         | +         | –         | –                      | –         | –         | +         | –      | –      |
| HyBcl-2-5 | –        | –         | –         | n.d.      | –                      | –         | –         | –         | –      | –      |
| HyBcl-2-6 | –        | –         | –         | n.d.      | –                      | –         | –         | –         | –      | –      |
| HyBcl-2-7 | –        | –         | –         | n.d.      | –                      | –         | –         | –         | –      | –      |
| HyBak1    | –        | –         | +         | –         | –                      | –         | –         | –         | –      | –      |
| HyBak2    | –        | –         | –         | n.d.      | –                      | –         | –         | –         | –      | –      |

Bcl-2-like protein interactions based on yeast two-hybrid assays. Interaction was detected by growth of yeast expressing the respective Bcl-2 family proteins on minimal medium lacking leucine and by expression of a GFP reporter. Interacting pairs are marked with +.

---

BH-3 domain

| Egl-1 | EIGSKLAAMCDDFDAQMM | BH-3 | 106 | TM | 160 |
| Bik   | ALALRLACIGDEMDVSLR |      |     |    |     |
| Bad   | RYGRELRRMSDEFVDSFK |      | 168 |    |     |
| Bid   | NIARHLAQVGDSMRSIP  |      | 241 |    |     |
| Hrk-1 | LTAARLKAIIGDELHQRTM |      | 91  |    |     |
| Noxa  | ECATQLRRFGDKLNFRQK |      | 54  |    |     |
| HyBH3-only 1 (hma 2.230679) | EVATKLACVGDEIQE--- |      | 296 |    |     |
| HyBH3-only 2 (hma 2.221399) | -IGACLRAIGDDIQ--- |      | 96  |    | 231 |
| HyBH3-only 3 (hma 2.218794) | ----LQSFGDDFDDQFK |      |     |    |     |
| HyBH3-only 4 (hma 2.224514) | -----IIVMGDELNMRV- |      | 83  |    |     |

**Figure 7 Hydra BH3-only proteins. Alignment of BH-3 domain sequences and schematic representation of HyBH3-only 1, 2, 3 and 4, as predicted from gene models (hma numbers in brackets) in comparison with human Bik, Bad, Bid, Hrk and NOXA and Egl-1 from C. elegans. Names printed in bold indicate gene predictions for which cDNAs were cloned from Hydra mRNA. BH3 domains (gray) and transmembrane domains (black) are boxed in the schematic; conserved amino acids are highlighted in the alignment.**

and sequence motifs from known BH3-only proteins as
a template [18]. This yielded four *Hydra* gene models
encoding novel proteins with conserved BH3 domains.
They were designated as HyBH3-only 1, 2, 3 and 4
(Figure 7). Three of the gene models were validated by
RT-PCR and cDNA cloning (these are printed in bold in

Figure 7). Hma2.224514 codes for a short 83 amino acid
protein with the BH3 domain followed by a transmem-
brane domain (HyBH3-only4). This domain structure is
similar to the structure of human Hrk-1, although there
is no further sequence homology outside these two do-
mains [19]. In order to test whether these *Hydra* BH3-

A
GFP

DNA
DAPI

Merged

B

![image](https://via.placeholder.com/500x500)

Figure 8 HyBH3-only 2 induces apoptosis in HEK293T cells. (A) Epifluorescence imaging of HEK293T cells 24 h after trans-
fection with plasmids encoding GFP-HyBH3-only 2; left hand panels show three cells undergoing apoptosis, as indicated by
round shape (see GFP, green in merged image), and pyknotic chromatin (see DNA stain DAPI, blue in merged image); right
hand panel shows a nonapoptotic cell with GFP in cytoplasm (green in merged image); (B) Quantification of GFP-positive
cells with apoptotic morphology in percent of all GFP-positive cells after transfection with plasmids encoding GFP-BH3-only 2,
3 and 4 fusion proteins and GFP (for control).
only proteins could induce apoptosis, we expressed them in human HEK293T cells and scored apoptotic cells. Figure 8A shows four cells expressing GFP-HyBH3-only 2. Three cells in the left hand panel have rounded up and DNA staining indicates that they have apoptotic nuclei. By comparison, the cell on the right hand panel is not apoptotic. Figure 8B shows the percentage of apoptotic cells for each of the Hydra BH3-only proteins transfected into HEK293T cells. Only HyBH3-only 2 induced apoptosis in this system.

The apoptosis-inducing HyBH3-only 2 was then tested in the yeast two-hybrid system for interaction with the anti-apoptotic HyBcl-2-like 4 and the pro-apoptotic HyBak-like 1 proteins. As shown in Table 2, HyBH3-only 2 interacted strongly with HyBcl-2-like 4 but not with HyBak-like 1. By comparison, HyBH3-only 4 did not interact with either HyBcl2-like 4 or HyBak-like 1.

### Discussion

While genome searches in several lower metazoans have revealed that early nonbilaterian animals possess a large number of caspase and Bcl-2 family genes, there has been very little experimental evidence showing that these genes were involved in apoptosis. In contrast to this earlier work, our results show that these protein families, which are also expanded in *Hydra*, functionally participate in apoptotic pathways. Figure 9 summarizes the components of the apoptotic toolkit in *Hydra*, in comparison with apoptotic proteins from *Homo sapiens*. The similarity is striking and emphasizes the fact that complex apoptotic pathways evolved early in metazoan evolution and that they may have contributed significantly to stabilizing the multicellular state.

#### Caspases and caspase activation

We purified the DEVD-specific caspase activity from apoptotic extracts of hydra and showed, with the help of

**Table 2 Interactions of HyBH3-only proteins with HyBcl-2-like 4 and HyBak-like 1 in yeast two-hybrid assay**

| Bait/Prey | HyBcl-2-4 | HyBak1 |
|-----------|------------|---------|
| HyBcl-2-4 | +          | n.d.    |
| Control (vector) | –        | n.d.    |
| HyBH3-only 2 | +          | –       |
| HyBH3-only 4 | –          | –       |

Interactions were detected by growth of yeast expressing the respective *Hydra* proteins on minimal medium lacking leucine and by expression of a GFP reporter. Interacting pairs are marked with +. Bcl-2-like 4 interacting with itself was already shown in Table 1 and served as a positive control in this experiment.

---

**Figure 9 Complex apoptotic pathways in *Hydra*. Schematic comparison of apoptotic networks in *Hydra* and human cells. Question marks indicate missing evidence for interaction of components. Dotted lines indicate that links have not yet been shown experimentally.**


Hydra


![Diagram of Hydra's apoptotic pathway]


Human


![Diagram of Human's apoptotic pathway]

www.cell-research.com | Cell Research
antibodies and specific substrates, that it corresponds to HyCaspB/C (Figure 2). Based on protein structure, these caspases appear to be executioner caspases and they are not autocatalytically activated when expressed in *E. coli*. This raises the question of how they become activated. Based on sequence homology, there are several potential initiator caspases in the *Hydra* genome that have CARD, DED and DD motifs in their prodomains. Most of them did not show autoactivation when expressed in bacteria; hence, we cannot yet define them as initiator caspases. However, they may require adaptor proteins for effective autoactivation. Two such activators, an APAF1 homolog with a CARD domain and a FADD homolog with a DED domain [15], are encoded in the *Hydra* genome and could fulfill this role. However, since caspase activation in our apoptotic extracts is quite fast, we have so far been unable to define a role for APAF1 or FADD.

Although we could not demonstrate autoactivation with CARD-caspase and DED-caspase, we did observe autoactivation in bacterial extracts with HyCaspA and with HyDDCasp (Figure 3). These two *Hydra* caspases have unusual prodomains. HyCaspA has a prodomain for which no homologs can be found in other caspases. However, it enabled autoprocessing and activation of HyCaspA, giving rise to a caspase with unusual substrate specificity. HyCaspA was inhibited with the pancaspase inhibitor z-VAD-FMK, but not with any of the tetrapeptide inhibitors tested. Its autoprocessing site was C-terminal of the amino acids IRKD, consistent with the unusual substrate specificity of this caspase. The second caspase showing autoactivation has a DD-motif in its prodomain that has so far only been described in prodomains of paracaspases [20]. Its caspase activity could be inhibited with Ac-YVAD-CHO, which suggests that it has a substrate specificity similar to human ICE (caspase 1) [21]. Neither of these caspases appeared to be active in apoptotic cell lysates from hydra.

*Hydra* genome analysis has provided evidence for the presence of genes encoding components of the extrinsic pathway of apoptosis induction, which could participate in the activation of HyDEDCasps. These include a *Hydra* FADD homolog and a gene encoding a putative death receptor. This gene was cloned and shown to have two extracellular TNF-receptor domains and an intracellular DD domain, with high sequence conservation to vertebrate p75 NGF-receptors [15]. Which ligands activate this receptor and what role it plays in *Hydra* biology is presently completely unclear. However, it should be noted that during oogenesis the developing oocyte initiates massive apoptosis in surrounding nurse cells and subsequently phagocytizes the nurse cell corpses [22]. Thus the oocyte may be the source of a death ligand.

Extrinsic pathways for apoptosis induction via TNF-receptors, FADD-like adaptors and caspase-8 have so far not been documented in invertebrates. TNF-receptors have not been found in *Caenorhabditis*, nor have FADD homologs. In *Drosophila*, the TNF-receptor homolog Wengen does trigger cell death. However, it lacks an intracellular DD domain, and apoptosis is not dependent on the *Drosophila* caspase-8 homolog DREDD, but rather works through *Drosophila* DRONC (Caspase-9) and DARK (APAF1) [23–25].

### Regulation of apoptosis in Hydra

The present experiments demonstrate the occurrence of Bcl-2 and Bak homologs in *Hydra*. Although it was not possible to test the function of these proteins in *Hydra* directly, expression of the proteins in mammalian cells indicated that several are functional homologs of their mammalian counterparts. Bcl-2-like 4 strongly inhibited camptothecin-induced apoptosis and this inhibition was blocked by mutation of the BH3 domain (Figure 6C). In comparison, HyBak-like 1 strongly induced apoptosis in HEK293T cells (Figure 6B) and interacted with Bcl-2-like 4 in the yeast two-hybrid assay (Table 1). Finally, HyBH3-only 2 also induced apoptosis in mammalian cells and interacted with HyBcl-2-like 4 in the yeast two-hybrid assay. Together, these results suggest that the *Hydra* Bcl-2 proteins have functional properties, which would permit them to regulate apoptosis in *Hydra*. This could potentially occur in a similar manner to mammalian cells, where BH3-only proteins induce apoptosis by interacting with anti-apoptotic Bcl-2 proteins and thus relieve the repression of proapoptotic multidomain Bcl-2 family proteins [26]. These results constitute the first functional data that Bcl-2 family members in pre-bilateria metazoans actually have the expected biochemical and cellular properties.

The diversity of the Bcl-2 family in mammals includes various biochemical characteristics, subcellular localizations and tissue-specific expression patterns. For instance, some human BH3-only proteins interact with all members of the anti-apoptotic Bcl-2 family, while others only interact with some. Binding affinities of BH3 domains for different family members also greatly vary. Moreover, BH3-only proteins can be induced by different stimuli such as death receptor signaling, growth factor deprivation and DNA damage. To induce apoptosis some are released from the ER or from the cytoskeleton (recently reviewed in [27]). Our experimental data on the large Bcl-2 family in *Hydra* now suggest that the observed diversity could probably fulfill similar functions. The yeast two-hybrid experiments indicate that *Hydra* Bcl-2 family members have different binding affinities

Cell Research | Vol 20 No 7 | July 2010
for each other or need different conditions for their interactions. Moreover, they are localized to different cellular compartments. Future experiments will show whether they are also expressed in a tissue-specific manner.

In summary, our data indicate that complex apoptotic pathways arose early in the evolution of multicellular animals. Raff [28] has proposed that apoptosis in animals is a means for social control between cell populations fulfilling different functions in the organism. Apoptotic mechanisms therefore evolved together with the evolution of such populations in multicellular animals [29].

Strikingly, this evolution gave rise to complex regulatory pathways right from the beginning. This complexity is also reflected in the variety of different roles for apoptosis in Hydra and other cnidarians including cell number regulation, gametogenesis and metamorphosis.

## Materials and Methods

### Hydra culture
Hydra were cultured in Hydra medium (0.1 mM KCl, 1 mM NaCl, 0.1 mM MgSO₄, 1 mM Tris-HCl, 1 mM CaCl₂ (pH 7.6)) at 18 °C. They were fed daily with freshly hatched Artemia nauplii larvae and washed after 6-8 h to remove undigested material, which was expelled into the medium.

### Preparation of Hydra lysate
Animals were washed in Hydra medium and in buffer A (25 mM HEPES (pH 7.5), 5 mM MgCl₂, 1 mM EGTA, protease inhibitors leupeptin, pepstatin A and aprotinin at 10 μg/ml each and 10 mM Pefabloc, Roche), homogenized by extrusion through a fine needle in buffer A and frozen at –80 °C. After thawing, samples were sonicated three times for 10 s and centrifuged for 15 min at 12 000× g. The supernatant was used for active-site labeling with z-EK(biotin)D-AOMK or FITC-VAD-FMK.

### Purification of DEVD-specific Hydra caspase
Lysates were prepared from 10 000 animals and incubated at room temperature for 1 h to induce caspase activity, which was measured with DEVD-AMC [16]. The extract was loaded on a HiPrep16/10 DEAE-anion exchange column (Amersham Biosciences) equilibrated with 20 mM Tris, 0.1% CHAPS, 1 mM DTT (pH 7.8) (buffer A), the column was washed with buffer A and proteins were eluted with a NaCl gradient from 0 to 0.5 M NaCl. Caspase activity was high in fractions eluted at 0.2 M NaCl. Active fractions were collected, concentrated and loaded onto HiLoad 16/10 Superdex 200, prep-grade gel filtration column (Amersham Biosciences) equilibrated with buffer A with 0.2 M NaCl. The column was eluted with buffer A with 0.2 M NaCl. Active fractions were again collected, concentrated and incubated with 8 μM biotinyl-DEVD-CHO in 20 mM Tris/HCl, 0.1% CHAPS, 10 mM DTT, 5 mM MgCl₂ (pH 7.8). They were then loaded on a HiTrap-streptavidin column (Amersham Biosciences). The column was washed with buffer A with 0.15 M NaCl and then eluted with 2 mM biotin in this buffer. All eluted fractions were analyzed in SDS-PAGE, western blotted and stained with anti-HyCaspB/C antibody.

### Caspase activity assay with z-EK(biotin)D-AOMK or FITC-VAD-FMK
Bacterial soluble extract was incubated for 2 h at room temperature in assay buffer (25 mM HEPES (pH 7.5), 0.1 % (w/v) CHAPS, 10 mM DTT, 10 μg/ml pepstatin A, 10 μg/ml aprotinin, 10 μg/ml leupeptin, 10 mM pefabloc) with 1 μM of the active-site labeling reagents z-EK(biotin)D-AOMK (Peptide Institute Inc, Japan) or FITC-VAD-FMK (Alexis). To test for caspase inhibition, the sample was preincubated for half an hour with 1 μM of the indicated inhibitors before adding the active-site labeling reagent. After incubation, SDS sample buffer was added and SDS-PAGE and western blot analysis was performed. Active fragments were detected by streptavidin-peroxidase for z-EK(biotin)D-AOMK substrate or by the monoclonal mouse anti-FITC-antibody (kind gift from Burkhard Micheel, University of Potsdam, Germany) for FITC-substrate.

### Generation of antibodies
Affinity-purified denatured His₆-myc-tagged HyCaspA and HyCaspB were used for chicken immunization (Davids, Regensburg, Germany). An IgY preparation was used for the experiments. The antibodies were shown to recognize their respective bacterially expressed caspase antigens on western blots.

### Cloning
HyBcl-2-like 1-7, HyBak-like 1 and 2 and HyBH3-only 2, 3 and 4 were first amplified using Hydra cDNA as template. Primers were designed for amplification of the coding sequences, as deduced from gene models and EST sequences. The PCR products were cloned into pCR®2.1-TOPO using the TOPO TA-cloning Kit (Invitrogen), followed by sequencing. The 3′ end of HyBcl-2-like 7 was obtained by RACE reactions using the GeneRacer Kit (Invitrogen). For the fluorescent fusion proteins GFP-HyBcl-2-like 1-7 and GFP-HyBak-like 1 and 2, the corresponding sequences were cloned into the vector hoT G [30] to the 3′ end of the sequence encoding the fluorescent protein, using the EcoRI site. For yeast two-hybrid assays, the coding sequences of the HyBcl-2 family genes were cloned into the EcoRI and XhoI sites of the bait vector pEG202 or the prey vector pJG4-5 (MoBiTec). For expression in mammalian cells, HyBcl-2 family genes were cloned into the vector pEGFP (Clontech), allowing expression of N-terminal GFP-tagged HyBcl-2 family proteins.

### Transfection of hydra cells
Gold particles (1.0 μm, BioRad) were coated with plasmid DNA according to the instructions of the manufacturer. They were introduced into hydra cells with the Helios gene gun system (BioRad), as previously described [30].

### Immunofluorescence
Animals were relaxed for 2 min in 2% urethane, fixed with 2% paraformaldehyde for 1 h at room temperature, washed with PBS, permeabilized (0.5% Triton/PBS) and blocked (0.1% Triton, 1% BSA/PBS). Incubation with the primary antibody (for mitochondria: anti-OxPhos Complex V subunit α-mouse IgG2b, monoclonal Ab 7H10 (Invitrogen)) in blocking solution was carried out overnight at 4 °C. The secondary antibody (Cy3 AffiniPure sheep anti-mouse IgG (H+L); Dianova/Jackson Immuno Research) was incubated for 2 h at room temperature followed by washing and
DAPI/TO-PRO-3 staining. Finally, the animals were mounted on slides with Vectashield mounting medium (Alexis Biochemicals).

### Hydra ER-labeling

RFP-HyBax-inhibitor 1 was used as a marker for *Hydra* ER. Bax inhibitor 1 is an integral ER-membrane protein with high homology between mammals and *Hydra* [31]. By searching the *Hydra* genome (at http://hydrazome.metazome.net/cgi-bin/gbrowse/hydra), we found a gene model for the *Hydra* homolog (hmal.130444) and cloned this sequence from *Hydra* cDNA. We then inserted it into the vector hoTRed [32] for expression in *Hydra* cells. When expressed in mammalian cells, the HyBax inhibitor 1 co-localized with calnexin (data not shown) and is therefore a specific marker for ER.

### Confocal laser scanning microscopy

Optical serial sections were acquired with a confocal laser scanning microscope (TCS SP1, Leica Microsystems, Heidelberg, Germany) equipped with an oil immersion plan-apochromatic ×100/1.4 NA objective. The fluorochromes were scanned sequentially. EGFP was excited with a 488-nm argon laser line and Cy3 with a 568-nm krypton laser line using an excitation beam splitter TD 488/568/633. Emissions were recorded between 520-540 and 575-625 nm, respectively. The gray-scale single channel images were overlaid to an RGB image assigning a false color to each channel and then reassembled using Adobe Photoshop 7.0 software (Adobe Systems, USA).

### Detection and induction of apoptosis in mammalian cells

Human embryonic kidney cells (293T cells) were cultured in DMEM (Sigma) supplemented with 10% FCS and 5% pen/strep. 293T cells were transfected using PEI (pH 7.0). Cells were fixed and counterstained with DAPI 24 h later. The nuclei of GFP-expressing cells were microscopically assayed for apoptotic morphology and the percentage of apoptotic nuclei was counted. To induce apoptosis, cells were treated with 10 μm camptothecin for 24 h.

### Yeast strains, media and transformation

Standard genetic techniques were used for growth and manipulation of yeast strains.

*S. cerevisiae* strain EGY 48 (MATα, trp1, his3, ura3, leu2:::2/4/6 LECAop-LEU2) was used. Transformation of yeast was carried out with lithium acetate; cells were grown in complete minimal medium lacking uracil, tryptophan, histidine or leucine to select for the presence of various plasmids. Interaction studies, including positive and negative controls, were performed according to the manufacturers' instructions (MoBiTec (2004) Grow'n'Glow: The GFP Two-Hybrid System Product Information and Protocols Handbook).

### Software and accession numbers

ClustalW2 und Clustal X (http://www.ebi.ac.uk/Tools/clustalw2/index.html) were used to generate alignments. Search for genes was performed using the *Hydra* genome server (http://hydrazome.metazome.net/cgi-bin/gbrowse/hydra/). BH3-only proteins were searched using the software HHsearch. A sequence context-specific profile for homology searching was used.

Accession numbers for human proteins used in alignments are:

NP_000624 for human Bcl-2; NP_001179 for human Bak and NP_620116 for human Bax. Members of the Bcl-2 protein family in *Hydra* are accessible by the numbers EF104646 for HyBcl-2-like 1, EF104647 for HyBcl-2-like 2, EU035765 for HyBcl-2-like 3, EU035764 for HyBcl-2-like 4, EU035763 for HyBcl-2-like 5, EU035762 for HyBcl-2-like 6, EU035761 for HyBcl-2-like 7, EF104645 for HyBak-like 1 and EU035760 for HyBak-like 2.

## Acknowledgments

We thank Burkhard Michel (Potsdam University) for the anti-FITC antibody, Rainer Deutzmann (Regensburg University) for sequencing the HyCaspA cleavage site and the DFG for support (BO1448-2).

## References

1. Hay BA, Huh JR, Guo M. The genetics of cell death: approaches, insights and opportunities in *Drosophila*. *Nat Rev Genet* 2004; **5**:911-922.
2. Liu QA, Hengartner MO. The molecular mechanism of programmed cell death in *C. elegans*. *Ann N Y Acad Sci* 1999; **887**:92-104.
3. Twomey C, McCarthy JV. Pathways of apoptosis and importance in development. *J Cell Mol Med* 2005; **9**:345-359.
4. Jin Z, El-Deiry WS. Overview of cell death signaling pathways. *Cancer Biol Ther* 2005; **4**:139-163.
5. Igaki T, Miura M. Role of Bcl-2 family members in invertebrates. *Biochim Biophys Acta* 2004; **1644**:73-81.
6. Bosch TC, David CN. Growth regulation in *Hydra*: relationship between epithelial cell cycle length and growth rate. *Dev Biol* 1984; **104**:161-171.
7. Kuznetsov S, Lyanguzowa M, Bosch TC. Role of epithelial cells and programmed cell death in *Hydra* spermatogenesis. *Zoology (Jena)* 2001; **104**:25-31.
8. Technau U, Miller MA, Bridge D, Steele RE. Arrested apoptosis of nurse cells during *Hydra* oogenesis and embryogenesis. *Dev Biol* 2003; **260**:191-206.
9. Wager RE. The oogenesis and early development of *hydra*. *Biol Bull* 1909; **XVIII**:1-36.
10. Böttger A, Alexandrova O. Programmed cell death in *Hydra*. *Semin Cancer Biol* 2007; **17**:134-146.
11. Seipp S, Schmich J, Leitz T. Apoptosis—a death-inducing mechanism tightly linked with morphogenesis in *Hydractinia echinata* (*Cnidaria*, Hydrozoa). *Development* 2001; **128**:4891-4898.
12. Seipp S, Wittig K, Stiening B, Böttger A, Leitz T. Metamorphosis of *Hydractinia echinata* (*Cnidaria*) is caspase-dependent. *Int J Dev Biol* 2006; **50**:63-70.
13. Zmasek CM, Zhang Q, Ye Y, Godzik A. Surprising complexity of the ancestral apoptosis network. *Genome Biol* 2007; **8**:R226.
14. Dunn SR, Phillips WS, Spatafora JW, Green DR, Weis VM. Highly conserved caspase and Bcl-2 homologues from the sea anemone *Aiptasia pallida*: lower metazoans as models for the study of apoptosis evolution. *J Mol Evol* 2006; **63**:95-107.
15. Lasi M, David CN, Böttger A. Apoptosis in pre-Bilaterians: *Hydra* as a model. *Apoptosis* 2009; **15**:269-278.
16. Cikala M, Wilm B, Hobmayer E, Bottger A, David CN. Iden-
tification of caspases and apoptosis in the simple metazoan
hydra. Curr Biol 1999; 9:959-962.
17 Herberg JA, Phillips S, Beck S, et al. Genomic structure and
domain organisation of the human Bak gene. Gene 1998;
211:87-94.
18 Biegert A, Söding J. Sequence context-specific profiles for ho-
mology searching. Proc Natl Acad Sci USA 2009; 106:3770-
3775.
19 Imaizumi K, Tsuda M, Imai Y, Wanaka A, Takagi T, Tohyama
M. Molecular cloning of a novel polypeptide, DP5, induced
during programmed neuronal death. J Biol Chem 1997;
272:18842-18848.
20 Uren AG, O'Rourke K, Aravind LA, et al. Identification
of paracaspases and metacaspases: two ancient families of
caspase-like proteins, one of which plays a key role in MALT
lymphoma. Mol Cell 2000; 6:961-967.
21 Thornberry NA, Rano TA, Peterson EP, et al. A combinatorial
approach defines specificities of members of the caspase fam-
ily and granzyme B. Functional relationships established for
key mediators of apoptosis. J Biol Chem 1997; 272:17907-
17911.
22 Alexandrova O, Schade M, Böttger A, David CN. Oogenesis
in Hydra: nurse cells transfer cytoplasm directly to the grow-
ing oocyte. Dev Biol 2005; 281:91-101.
23 Igaki T, Kanda H, Yamamoto-Goto Y, et al. Eiger, a TNF su-
perfamity ligand that triggers the Drosophila JNK pathway.
EMBO J 2002; 21:3009-3018.

Kanda H, Igaki T, Kanuka H, Yagi T, Miura M. Wengen, a
member of the Drosophila tumor necrosis factor receptor su-
perfamity, is required for Eiger signaling. J Biol Chem 2002;
277:28372-28375.
25 Kauppi S, Maaty WS, Chen P, et al. Eiger and its receptor,
Wengen, comprise a TNF-like system in Drosophila. Onco-
gene 2003; 22:4860-4867.
26 Willis SN, Fletcher JI, Kaufmann T, et al. Apoptosis initiated
when BH3 ligands engage multiple Bcl-2 homologs, not Bax
or Bak. Science 2007; 315:856-859.
27 Youle RJ, Strasser A. The BCL-2 protein family: opposing
activities that mediate cell death. Nat Rev Mol Cell Biol 2008;
9:47-59.
28 Raff MC. Social controls on cell survival and cell death. Na-
ture 1992; 356:397-400.
29 David CN, Schmidt N, Schade M, Pauly B, Alexandrova O,
Böttger A. Hydra and the evolution of apoptosis. Integr Comp
Biol 2005; 45:631-638.
30 Böttger A, Alexandrova O, Cikala M, Schade M, Herold M,
David CN. GFP expression in Hydra: lessons from the par-
ticle gun. Dev Genes Evol 2002; 212:302-305.
31 Xu Q, Reed JC. Bax inhibitor-1, a mammalian apoptosis sup-
pressor identified by functional screening in yeast. Mol Cell
1998; 1:337-346.
32 Müller-Taubenberger A, Vos MJ, Böttger A, et al. Monomeric
red fluorescent protein variants used for imaging studies in
different species. Eur J Cell Biol 2006; 85:1119-1129.
