
Journal of Steroid Biochemistry & Molecular Biology 122 (2010) 253–263

Contents lists available at ScienceDirect

**Journal of Steroid Biochemistry and Molecular Biology**

journal homepage: www.elsevier.com/locate/jsbmb

---

### Steroid degradation genes in *Comamonas testosteroni* TA441: Isolation of genes encoding a Δ4(5)-isomerase and 3α- and 3β-dehydrogenases and evidence for a 100 kb steroid degradation gene hot spot

Masae Horinouchi ${}^{a,*}$, Tomokazu Kurita ${}^{a}$, Toshiaki Hayashi ${}^{a}$, Toshiaki Kudo ${}^{a,b}$

${}^{a}$ RIKEN, 2-1 Hirosawa, Wako-shi, Saitama 351-0198, Japan  
${}^{b}$ Faculty of Fisheries, Nagasaki University, 1-14 Bunkyo-machi Nagasaki-shi, 852-8521, Japan

---

#### ARTICLE INFO

**Article history:**  
Received 23 October 2009  
Received in revised form 31 May 2010  
Accepted 1 June 2010  

**Keywords:**  
*Comamonas testosteroni*  
Steroid degradation  
3β-Dehydrogenase  

---

#### ABSTRACT

In previous studies, we identified two major *Comamonas testosteroni* TA441 gene clusters involved in steroid degradation. Because most of the genes included in these clusters were revealed to be involved in degradation of basic steroidal structures and a few were suggested to be involved in the degradation of modified steroid compounds, we investigated the spectrum of steroid compounds degradable for TA441 to better identify the genes involved in steroid degradation. TA441 degraded testosterone, progesterone, epiandrosterone, dehydroepiandrosterone, cholic acid, deoxycholic acid, chenodeoxycholic acid, and lithocholic acid. The results suggested TA441 having 3α-dehydrogenase and Δ4(5)-isomerase, and 3β-, 17β-dehydrogenase gene, we isolated these genes, all of which had high homology to the corresponding genes of *C. testosteroni* ATCC11996. Results of gene-disruption experiments indicated that 3β, 17β-dehydrogenase is a unique 3β-dehydrogenase which also acts as a 17β-dehydrogenase in TA441, and there will be at least one more enzyme with 17β-dehydrogenating activity. The 3α-dehydrogenase and Δ4(5)-isomerase genes were found adjacent in the DNA region between the two main steroid degradation gene clusters together with a number of other genes that may be involved in steroid degradation, suggesting the presence of a steroid degradation gene hot spot over 100 kb in size in TA441.

© 2010 Elsevier Ltd. All rights reserved.

---

### 1. Introduction

Steroid compounds have various pharmaceutical and hormonal effects. Degradation of steroid compounds by bacteria was intensively studied during the 1950s–1970s for the purpose of obtaining materials for steroid drug synthesis, especially from two representative steroid degrading bacteria, Gram-positive *Rhodococcus* (formerly *Nocardia*) *restrictus* and Gram-negative *Comamonas* (formerly *Pseudomonas*) *testosteroni* [1–23]. The major intermediate compounds in bacterial steroid degradation were identified in these studies and a degradation pathway was proposed based on these compounds. As new functions have been discovered for several steroid compounds [24–26], bacterial steroid degradation is re-emerging as an area of research interest. However, before our study on steroid degradation in *C. testosteroni* TA441, the bacterial steroid degradation pathway remained incomplete and the identification of degradation genes for only a few early-steps had been reported.

---

* Corresponding author at: Chemical Biology Core Facility, RIKEN Advanced Science Institute, 2-1 Hirosawa, Wako-shi, Saitama 351-0198, Japan  
Tel.: +81 48 467 1111x5354; fax: +81 48 462 1353.  
E-mail address: masae@postman.riken.jp (M. Horinouchi).

0960-0760/$ – see front matter © 2010 Elsevier Ltd. All rights reserved.  
doi:10.1016/j.jsbmb.2010.06.002

Table 1  
Steroid degradation genes identified in Comamonas testosteroni TA441, ATCC11996, and ATCC17410.

| Strain | Gene/ORF (accession number) [reference] | Protein/(putative protein) |
|--------|-----------------------------------------|---------------------------|
|        | ORF18                                   | CoA-transferase at C-5 of cleaved B-ring |
|        | ORF17                                   | Reductase component of C-9 hydroxylase |
|        | tesI                                    | (PRK07121) [35]           | 3-Ketosteroid Δ4-dehydrogenase |
|        | tesH                                    | (PRK12835) [36]           | 3-Ketosteroid Δ1-dehydrogenase |
|        | tesA2                                   |                           | Hydroxylase at C-4 of aromatized A-ring |
|        | tesA1                                   |                           | Hydroxylase at C-4 of aromatized A-ring |
|        | tesE                                    |                           | 2-Hydroxyhexa-2,4-dienoic acid hydratase |
|        | tesF                                    |                           | Propionaldehyde dehydrogenase |
|        | tesG                                    |                           | 4-Hydroxy-2-oxo-hexanoate aldolase |
|        | steA                                    |                           | Dehydrogenase at C-12 from α-OH to ketone |
|        | steB                                    |                           | Hydrogenase at C-12 from ketone to β-OH |
|        | ORF7                                    |                           | (Hypothetical protein) |
|        | ORF6                                    |                           | (NADH:flavin oxidoreductase) |
|        | tesB                                    | (AAM77244) [37]           | Meta-cleavage enzyme for aromatized A-ring |
|        | ORF1                                    |                           | (CoA-transferase alpha subunit) |
|        | ORF2                                    |                           | (CoA-transferase beta subunit) |
|        | ORF3                                    |                           | (Enoyl-CoA hydratase/isomerase family) |
|        | ORF4                                    |                           | (Enoyl-ACP reductase) |
|        | ORF5                                    |                           | (Enoyl-CoA hydratase/isomerase family) |
|        | ORF21                                   |                           | (Acyl-CoA dehydrogenase) |
|        | ORF22                                   |                           | (Acyl-CoA dehydrogenase) |
|        | ORF23                                   |                           | (Thiolase) |
|        | ORF25                                   |                           | (6-Aminohexanoate-cyclic-dimer hydrolase) |
|        | ORF26                                   |                           | (6-Aminohexanoate-cyclic-dimer hydrolase) |
|        | ORF27                                   |                           | (Short-chain dehydrogenase/reductase) |
|        | ORF28                                   |                           | (Acyl-CoA dehydrogenase) |
|        | ORF30                                   |                           | (Acyl-CoA dehydrogenase) |
|        | ORF31                                   | (ACI39936)${}^{a}$        | (Short-chain dehydrogenase/reductase) |
|        | ORF32                                   | (AAV40816)${}^{a}$        | (MaoC domain protein) |
|        | ORF33                                   | (AAV40815)${}^{a}$        | (Acetyl-CoA acetyltransferase) |
|        | tesR                                    | teiR [38]                 | LuxR-type transcription factor |
|        |                                         | stdC [41]                 | Steroid binding protein |
|        | ORF61${}^{b}$                           | ORF [41]                  | Unknown |
|        | ORF62${}^{b}$                           | 3β,17β-hsd [41–43]        | Steroid 3β,17β-dehydrogenase |
|        | 3α-hsd${}^{b}$                          | 3α-hsd [44,45]            | 3-Hydroxysteroid 3α-dehydrogenase |
|        | hsdA [46]                               |                           | |
|        | ksi${}^{b}$                             | ksi [21,47]               | 3-Ketosteroid Δ4(5)-isomerase |
|        | repA [48]                               |                           | Repressor of 3-hydroxysteroid 3α-dehydrogenase |
|        | ORF64${}^{b}$                           | repB [48]                 | Repressor of 3-hydroxysteroid 3α-dehydrogenase |

Broken lines: the genes/ORFs above and below are in separated DNA region.

${}^{a}$ Available only on online database.  
${}^{b}$ The gene/ORF isolated in this study.

M. Horinouchi et al. / Journal of Steroid Biochemistry & Molecular Biology 122 (2010) 253–263

| Compounds | Growth | HPLC analysis |
| --- | --- | --- |
| testosterone | ![](https://via.placeholder.com/150x100?text=testosterone+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA+analog+(RT=9.2)+4,9-DSHA) |
| progesterone | ![](https://via.placeholder.com/150x100?text=progesterone+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA-N+4,9-DSHA) |
| dehydroepiandrosterone | ![](https://via.placeholder.com/150x100?text=dehydroepiandrosterone+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA-N+4,9-DSHA) |
| cholic acid | ![](https://via.placeholder.com/150x100?text=cholic+acid+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA+analog+(RT=3.2)) |
| chenodeoxycholic acid | ![](https://via.placeholder.com/150x100?text=chenodeoxycholic+acid+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA+analog+(RT=7.3)+4,9-DSHA) |
| deoxycholic acid | ![](https://via.placeholder.com/150x100?text=deoxycholic+acid+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA+analog+(RT=7.8)) |
| epiandrosterone | ![](https://via.placeholder.com/150x100?text=epiandrosterone+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA-N) |
| 17α-hydroxyprogesterone | ![](https://via.placeholder.com/150x100?text=17α-hydroxyprogesterone+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA-N) |
| cortisol | ![](https://via.placeholder.com/150x100?text=cortisol+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA-N) |
| cholesterol | ![](https://via.placeholder.com/150x100?text=cholesterol+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA-N) |
| none | ![](https://via.placeholder.com/150x100?text=none+Growth+Graph) | ![](https://via.placeholder.com/150x100?text=4,9-DSHA-N) |

Fig. 1. Structure of steroid compounds examined in this study, growth curves of C. testosteroni TA441 with each steroid compound serving as carbon source, and HPLC analysis of cultures of TesD-disrupted mutants incubated with each steroid compound. Compounds with strong absorbance at 320 nm were 4,9-DSHA (retention time (RT) = 10.5 min) and its analogs, and a compound detected at RT = 3.5 min with a maximum absorbance at about 280 nm is 4-aza-9,17-dioxo-9,10-secoandrost-1,3,5(10)-triene-3-oic acid (4,9-DSHA-N) non-enzymatically converted from 4,9-DSHA (closed arrowheads).
vented measurement of cell absorbance. TA441 was pre-cultured in LB-medium, washed twice, resuspended in C-medium. The same amount of the cells was inoculated to C-medium with 0.1% (w/v) each steroid and the growth was monitored by counting colonies that appeared on LB plates on which appropriate dilutions of the culture had been spread and incubated at $30^{\circ} \mathrm{C}$.

### 2.3. HPLC analysis of the culture of gene-disrupted mutants

Gene-disrupted mutants were pre-cultured in LB-medium with kanamycin (400 $\mu$g/ml), inoculated to 10 ml 1/2LB + 1/2C-medium with 0.1% (w/v) of each steroid, and incubated at $30^{\circ} \mathrm{C}$. Twice the volume of methanol was added to a small amount of the culture 24 h, 32 h, and 48 h after the start of the incubation, which was then centrifuged and the supernatant was directly injected into the HPLC. For the three-dimensional HPLC data, a Waters 2690 HPLC (Waters Corporation, MA) was used with Waters 996 Photodiode Array Detector (Waters Corporation, MA) and an Inertsil ODS-3 column ($4.6 \times 250 \mathrm{~mm}$; GL Science, Tokyo, Japan), and elution carried out using a linear gradient from 20% solution A ($\mathrm{CH}_{3} \mathrm{CN}: \mathrm{CH}_{3} \mathrm{OH}$:trifluoroacetic acid (TFA) = 95:5:0.05) and 80% solution B ($\mathrm{H}_{2} \mathrm{O}: \mathrm{CH}_{3} \mathrm{OH}$:TFA = 95:5:0.05) to 65% solution A and 35% solution B over 10 min, maintained for 13 min, and then changed to 20% solution A for 5 min. The flow rate was $1 \mathrm{ml} / \mathrm{min}$.

### 2.4. Accession number

The accession number for a DNA fragment containing 3$\beta$-dehydrogenase gene is AB474240, and for 3$\alpha$-dehydrogenase and 3-ketosteroid-$\Delta 4(5)$-isomerase genes is AB489116.

## 3. Results

### 3.1. Growth of *C. testosteroni* TA441 on steroid compounds

Previously, we elucidated the pathway and genes involved in testosterone degradation by *C. testosteroni* TA441. We utilized testosterone in those studies to reveal details about degradation of the basic skeleton of steroid compounds, and did not focus upon the degradability of modified steroid compounds. As modified steroid compounds such as the cholic acid analogs are attracting attention due to newly discovered functions (e.g., reduction of internal cholesterol level in animals and induction of defensive reactions in plants [24–26]), in this study we examined degradability of steroid compounds containing different side chains and double bonds (Fig. 1) by *C. testosteroni* TA441 in order to identify the genes involved in the degradation of modified steroid compounds. Pre-cultured *C. testosteroni* TA441 cells were inoculated into C-medium (a mineral medium used for growth of TA441 [27]) containing 0.1% (w/v) of each steroid compound and cell growth was monitored at 0, 24, 48, and 72 h by determining the number of colony forming units per unit volume (CFU/ml) (Fig. 1). The CFU reached ca. $10^{8} / \mathrm{ml}$ in control cultures without a steroid compound carbon source. This amount of growth was probably due to pre-culture in LB media; therefore a CFU of ca. $10^{8} / \mathrm{ml}$ was regarded as no growth in cultures with steroid compounds.

*C. testosteroni* TA441 showed significant growth on testosterone, progesterone, dehydroepiandrosterone, cholic acid and its analogs (deoxycholic acid, chenodeoxycholic acid, lithocholic acid), and epiandrosterone. *C. testosteroni* TA441 grew faster on cholic acid and its analogs (with the exception of lithocholic acid) than on the other steroid compounds tested, but these compounds seemed to cause cell lysis after about 48 h. We do not know why this occurred, but similar lysis was observed when *C. testosteroni* TA441 was grown on succinate as the sole carbon source. Growth on lithocholic acid was quite similar to growth on epiandrosterone so that the data is not shown in Fig. 1. The start of growth on epiandrosterone was slow. We consider this was due to the low water solubility of this steroid, because small flocks, probably consisting of bacterial cells and small particles of epiandrosterone, appeared a few hours after the start of incubation.

Growth of *C. testosteroni* TA441 on 17$\alpha$-hydroxyprogesterone was so unstable that further study will be required to confirm that the organism can grow on this compound. *C. testosteroni* TA441 showed negligible growth on cortisol and cholesterol, and also on estrone, 17$\beta$-estradiol, ergosterol, and several other steroid compounds with a large side chain at the C-17 position, such as plant sterols (data not shown). In subsequent experiments, we used only cholesterol as a representative of these undegradable steroid compounds.

### 3.2. HPLC analysis of intermediate compounds in culture media of gene-disrupted mutants of *C. testosteroni* TA441 incubated with steroid compounds

Because *C. testosteroni* TA441 degrades steroid compounds so rapidly that intermediates can be detected only transiently and in small amounts, HPLC analysis of intermediates in the culture of *C. testosteroni* TA441 is of limited value. Intermediate compounds are usually detected with greater success in gene-disrupted mutants, that gene-disrupted mutants are more suitable to confirm the ability of *C. testosteroni* TA441 to degrade steroid compounds by HPLC. tesD encodes the hydrolase for 4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandrost-1(10),2-dien-4-oic acid (4,9-DSHA), an intermediate compound in bacterial steroid degradation, and disruption of tesD causes cells to accumulate 4,9-DSHA or its analogs. These compounds have a characteristic intense yellow color under neutral condition and absorb UV strongly with a maximum at about $320 \mathrm{~nm}$ under acidic condition. In addition, although 4,9-DSHA and its analogs are potentially unstable, 4,9-DSHA and its analogs are nonenzymatically converted to 4-aza-17-oxoandrosta-1,3,5(10)-triene-3-carboxylic acid (4,9-DSHA-N) and its analogs with ammonium ion in the culture, which are quite stable and behave as dead-end compounds for *C. testosteroni* TA441.

Pre-cultured TesD-disrupted mutant cells were inoculated and incubated in 1/2LB + 1/2C-medium (mixture of equal volume of LB and C-media) with 0.1% (w/v) steroid compound. Chromatograms of culture broth analyzed by HPLC at 24 h are shown in Fig. 1. Compounds with strong absorbance at $320 \mathrm{~nm}$ were 4,9-DSHA (retention time (RT) = 10.5 min) and its analogs, and a compound detected at RT = 3.5 min with a maximum absorbance at about $280 \mathrm{~nm}$ is 4-aza-9,17-dioxo-9,10-secoandrosta-1,3,5(10)-triene-3-oic acid (4,9-DSHA-N) (closed arrowheads).

Among the steroid compounds tested, significant amounts of 4,9-DSHA and/or its analogs were detected in cultures of the TesD-disrupted mutant incubated with testosterone, progesterone, dehydroepiandrosterone, and cholic acid analogs. In the culture with epiandrosterone and 17$\alpha$-hydroxyprogesterone, a minimal amount of 4,9-DSHA-N was detected. In the cultures incubated with cortisol and cholesterol, 4,9-DSHA and its analogs, and 4,9-DSHA-N were undetectable.

To confirm whether *C. testosteroni* TA441 is capable of degrading epiandrosterone and 17$\alpha$-hydroxyprogesterone, we performed the same experiment with a TesH-disrupted mutant. TesH is a $\Delta 1$-dehydrogenase that introduces a double bond at the C-1 position of the steroid A-ring. TesH-disrupted mutants incubated with steroid compounds accumulate androsta-4-ene-3,17-dione (4-AD) and/or its analogs with compounds of the same structure as 4-AD and/or its analogs except for the presence of a hydroxyl group at the C-9 position (9-OH-4-AD and/or its analogs) when the subjected steroid compounds are degradable for *C. testos-
M. Horinouchi et al. / Journal of Steroid Biochemistry & Molecular Biology 122 (2010) 253–263

ate compounds isolated from C. testosteroni TA441 usually have a ketone group at the C-17 position and are accompanied by a small amount of the compound with a hydroxyl group at the C-17 position [34]. Based on these results, we speculate that the 4,9-DSHA analog with a RT = 9.2 min in HPLC analyses is 4,5-9,10-diseco-3,17-dihydroxy-5,9,17-trioxoandrost-1(10),2-dien-4-oic acid.

Structure of 4,9-DSHA analogs with a RT = 3.2 min and RT = 7.3 were speculated from the absorption curves, RT (under the present HPLC condition, more hydrophilic compounds are detected earlier), together with other intermediate compounds in degradation of cholic acid and chenodeoxycholic acid. We identified 7,12-dihydroxy-9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid and 7-hydroxy-9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid as the major intermediates produced by hydrolysis in cultures of ORF18-disrupted mutants incubated with cholic acid and chenodeoxycholic acid, respectively (33). This result indicates that the hydroxyl groups at positions C-7 and C-12 are present before hydrolysis, and that therefore the major 4,9-DSHA analog observed at RT = 3.2 min in cultures incubated with cholic acid is 4,5-9,10-diseco-3,7,12-trihydroxy-5,9,17-trioxoandrost-1(10),2-dien-4-oic acid. The major 4,9-DSHA analog observed at RT = 7.3 min in cultures incubated with chenodeoxycholic acid is 4,5-9,10-diseco-3,7-dihydroxy-5,9,17-trioxoandrost-1(10),2-dien-4-oic acid, while the major 4,9-DSHA analog with a RT = 7.8 min in cultures incubated with deoxycholic acid is 4,5-9,10-diseco-3,12-dihydroxy-5,9,17-trioxoandrost-1(10),2-dien-4-oic acid.

Several peaks were detected by HPLC in the culture of TesD-disrupted mutant incubated with chenodeoxycholic acid. The peak at RT = 10.5 min is considered to be 4,9-DSHA itself because the RT is in complete agreement with the RT of 4,9-DSHA run as a standard. A small amount of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid identified in the culture of ORF18-disrupted mutants incubated with chenodeoxycholic acid (33) will support that 4,9-DSHA was produced from chenodeoxycholic acid. These results imply that during degradation of chenodeoxycholic acid in C. testosteroni TA441, a double bond would be produced at Δ6(7) by dehydration, followed by conversion to a single bond by addition of 2 H. Therefore, the 4,9-DSHA analog with a RT = 10.6 is likely to be 4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandrost-1(10),2,6-trien-4-oic acid.

To further confirm our results and investigate which steroid or intermediate compound(s) induce steroid degradation genes in C. testosteroni TA441, we performed Northern hybridization analyses. Unfortunately, we were unable to obtain unequivocal results, probably because some steroid compounds tested were less soluble in water and phenol, which was used in RNA isolation, and residual steroid compounds may have prevented hybridization. With regard to the steroid compounds that were soluble, results were consistent with those of growth and HPLC analysis: steroid degradation genes were induced when the subjected steroid is degradable for C. testosteroni TA441.

### 3.3. Isolation of a putative 3β,17β-dehydrogenase, a 3α-dehydrogenase, and 3-ketosteroid-Δ4(5)-isomerase genes from C. testosteroni TA441

Our results obtained in Sections 3.1 and 3.2 indicated that in addition to genes already identified as being involved in degradation of the basic steroid skeleton, C. testosteroni TA441 produces 3β-, 17β- and 3α-dehydrogenases, a 3-ketosteroid-Δ4(5)-isomerase, and enzymes for degradation of the side chain at the C-17 position of cholic acid and progesterone. Genes encoding 3β,17β- and 3α-dehydrogenases and 3-ketosteroid-Δ4(5)-isomerase have been isolated from C. testosteroni ATCC11996 [21,41,44]. Because the DNA sequences of C. testosteroni ATCC11996 and TA441 revealed to

DNA fragment containing a putative 3α-dehydrogenase gene and a putative 3-ketosteroid-Δ5-isomerase gene

TA441

![Diagram](https://i.imgur.com/1234567.png)

ATCC11996

DNA fragment containing a putative 3β,17β-dehydrogenase gene

TA441

![Diagram](https://i.imgur.com/8901234.png)


ATCC11996

Fig. 3. The isolated DNA fragment of C. testosteroni TA441 containing putative 3α-dehydrogenase and 3-ketosteroid-Δ4(5)-isomerase genes (above) and the DNA fragment containing a putative 3β,17β-dehydrogenase gene (below) with corresponding genes of C. testosteroni ATCC11996. Numbers (%) indicate amino acid identity between the corresponding proteins.

date are quite similar, we constructed PCR primers based on DNA sequences of C. testosteroni ATCC11996 genes to isolate corresponding genes from strain TA441. PCR amplification was performed with these primers and total TA441 DNA as the template, and probes were generated based on the obtained fragments after confirming the sequence. Using these probes, DNA fragments containing both the putative 3α-dehydrogenase and 3-ketosteroid-Δ4(5)-isomerase genes, as well as a fragment containing the putative 3β,17β-dehydrogenase gene were isolated. DNA sequencing confirmed that these fragments contained the genes of interest, and the deduced amino acid sequences were highly homologous (approximately 95% identity) [Fig. 3] [21,41,44].

Negative regulators of 3α-dehydrogenase, RepA and RepB, which are encoded on the complementary strand of the 3α-dehydrogenase and 3-ketosteroid-Δ4(5)-isomerase genes was reported for C. testosteroni ATCC11996 (Fig. 3A) [39], and corresponding putative genes were found in C. testosteroni TA441. However, the deduced amino acid sequence of the product of the repA gene had only approximately 85% identity with C. testosteroni ATCC11996 RepA. While 85% identity is not low, it is conspicuously lower than the identity between the other corresponding proteins involved in steroid degradation from these two strains. The deduced amino acid sequence of the product of the DNA region correspond to repB showed high homology (around 96% identity) to C. testosteroni ATCC11996 RepB, but we were unable to find a stop codon, which resulted in a larger ORF (ORF64) containing the gene region correspond to repB. The entire deduced amino acid sequence of ORF64 showed the highest homology (92% identity) to the putative enzyme-regulator fusion protein of C. testosteroni CNB-2 [49], as well as several other putative enzyme-regulator fusion proteins belong to the GreA/GreB family of elongation factors. Other than C. testosteroni, Bordetella parapertussis showed the highest amino acid identity (73%) [50]. C. testosteroni ATCC11996 3α-dehydrogenase and 3-ketosteroid-Δ4(5)-isomerase are comprehensively studied by other research groups (3α-dehydrogenase [44–46]; 3-ketosteroid-Δ4(5)-isomerase [21,47]), and we consider that further examination of these enzymes in C. testosteroni TA441 is of less value.

The isolated DNA fragment containing a putative 3β,17β-dehydrogenase gene was composed of three putative ORFs (ORFs61–63) with a partial ORF in the upstream DNA region of ORF61, and a possible terminator immediately downstream of ORF62 (Fig. 3B). The deduced amino acid sequence of the product of ORF62 showed high homology (96% identity) to 3β,17β-dehydrogenase (3β,17β-DH) of C. testosteroni ATCC11996. The partial ORF corresponds to stdC of C. testosteroni ATCC11996, which was reported to be induced when ATCC11996 is incubated with steroid compounds [51], suggesting the possibility that the partial ORF, ORF61, and the putative 3β,17β-DH gene are involved in steroid degradation. The putative amino acid sequence of ORF61 shows high homology to putative proteins of the DUF1329 family. DUF1329 is a hypothetical protein but is found in wide range of bacteria, especially in Pseudomonas species. The deduced amino acid sequence of ORF63 showed significant homology to Sel1 domain-containing proteins and is not considered to be involved in steroid degradation.

### 3.4. Analysis of ORF61 and ORF62 (a putative 3β,17β-dehydrogenase gene) using gene-disrupted mutants

Because ORF61 might be involved in steroid degradation, and its function in steroid degradation is unclear, we constructed gene-disrupted mutants of ORF61 and ORF62 in C. testosteroni TA441. ORF61- and the ORF62-disrupted mutants were cultured and their growth was monitored in the presence of various steroid compounds as described in Section 3.1. Both mutants grew on testosterone, but the ORF62-disrupted mutant grew slower than the C. testosteroni TA441 (positive control) and the ORF61-disrupted mutant (Table 2).

The 3β,17β-dehydrogenase of C. testosteroni ATCC11996 was first reported as a 3β-dehydrogenase. We considered the main role of the enzyme encoded by ORF62 of C. testosteroni TA441 to be 3β-dehydrogenation and conducted the same growth experiment with epiandrosterone, which has a ketone group at the C-17 position and a β-oriented hydroxyl group at the C-3 position. Growth of both C. testosteroni TA441 and the ORF61-disrupted mutant on epiandrosterone was slower than on testosterone, but cell density reached more than 10⁹ CFU/ml in approximately 48 h. However, the ORF62-disrupted mutant did not show significant growth on epiandrosterone. These results suggested that the ORF62-encoded enzyme is a unique 3β-dehydrogenase in C. testosteroni TA441, and that C. testosteroni TA441 might have another dehydrogenase more suitable for the 17β-hydroxyl group.

To confirm results of the growth experiment, gene-disrupted mutants were incubated with testosterone, epiandrosterone, and dehydroepiandrosterone (we added this compound to sup-

M. Horinouchi et al. / Journal of Steroid Biochemistry & Molecular Biology 122 (2010) 253–263

**Table 2**

Growth (measured as CFU/ml) of *C. testosteroni* TA441, the ORF61-disrupted mutant, and the ORF62 (putative 3β,17β-dehydrogenase)-disrupted mutant incubated for 24 h with 0.1% (w/v) testosterone or epiandrosterone as the sole carbon source.

|  | TA441 (standard error) | ORF62⁻ (standard error) | ORF61⁻ (standard error) |
| --- | --- | --- | --- |
| (0h) | $3.27 \times 10^{7} (3.43 \times 10^{6})$ | $1.75 \times 10^{7} (3.19 \times 10^{6})$ | $1.69 \times 10^{7} (5.72 \times 10^{6})$ |
| Testosterone | $1.74 \times 10^{9} (4.82 \times 10^{7})$ | $1.21 \times 10^{9} (1.76 \times 10^{8})$ | $1.85 \times 10^{9} (1.23 \times 10^{8})$ |
| Epiandrosterone | $7.52 \times 10^{8} (1.99 \times 10^{8})$ | $2.41 \times 10^{7} (4.80 \times 10^{6})$ | $6.6 \times 10^{8} (6.00 \times 10^{7})$ |

port the data), and the cultures were analyzed by HPLC as described in Section 3.2. In this experiment, the *C. testosteroni* TA441 mutant without 17β-dehydrogenase activity and incubated with testosterone should accumulate 17-hydroxy-androsta-1,4-diene-3-one. The *C. testosteroni* TA441 mutant without 3β-dehydrogenase activity should not accumulate particular intermediate compounds because the 3β-hydroxyl group has to be converted to a ketone group before dehydrogenation on the A-ring by 3-ketosteroid-Δ1-dehydrogenase, 3-ketosteroid-Δ4-dehydrogenase, and 3-ketosteroid isomerase.

The results are shown in Fig. 4, with a higher detector sensitivity since epiandrosterone has a weaker absorbance. We were unable to detect degradation of epiandrosterone and dehy-droepiandrosterone with the ORF62-disrupted mutant, while all three steroid compounds decreased to undetectable levels with the ORF61-disrupted mutant. These results are consistent with growth results, indicating the enzyme encoded by ORF62 is a unique 3β-dehydrogenase in *C. testosteroni* TA441. Conversion of testosterone to androsta-1,4-diene 3,17-dione (ADD) was slow in the culture of the ORF62-disrupted mutant, and testosterone was still detected 10 days after the start of the incubation, indicating that the ORF62-disrupted mutant still converted a hydroxyl group at the C-17 position to a ketone group, though at a slower rate than the *C. testosteroni* TA441 control. This enzyme is considered to have 17β-

![ORF62-disrupted mutant](https://example.com/image1.png)
![ORF61-disrupted mutant](https://example.com/image2.png)

Fig. 4. HPLC analysis of the culture of the ORF62 (a putative 3β,17β-dehydrogenase gene)-disrupted mutant and ORF61-disrupted mutant incubated with testosterone (RT = 13.6), epiandrosterone (RT = 15.4), or dehydroepiandrosterone (RT = 14.3) (indicated with open arrowheads) for 1 day and 10 days.

dehydrogenase activity, but *C. testosteroni* TA441 probably has at least one more dehydrogenase that works on a hydroxyl group at the C-17 position.

# 4. Discussion

We previously elucidate a bacterial degradation pathway for the basic skeleton of steroid compounds and to determine the genes involved therein in *Comamonas testosteroni* TA441 using testosterone and ADD as representative steroid compounds due to their simple structures [27–34]. Here we investigated the spectrum of degradable steroid compounds in the purpose of understanding the whole genes involved in steroid degradation in *C. testosteroni* TA441. For the production of useful steroidal compounds, more familiar steroid compounds with modification, which includes cholic acid, cholesterol, and plant sterols, are preferable as starting materials for developing new chemicals with useful effects. These compounds are also attractive from a research perspective because a number of useful effects of these steroid compounds are known and are still under intensive investigation in searching for novel useful effects [24–26].

Among the steroid compounds we tested, *C. testosteroni* TA441 degraded testosterone, progesterone, epiandrosterone,

![Figure 5](https://i.imgur.com/yourimage.png)

Fig. 5. Proposed *Comamonas testosteroni* TA441 steroid degradation pathway. Compounds (R1, R2 = H) are: 3-oxo-5β-cholan-24-oic acid (I); androsta-4-ene-3,17-dione (II); androsta-1,4-diene-3,17-dione (ADD) (III); 9-hydroxy-1,4-androstadiene-3,17-dione (IV); 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione (V); 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione (VI); 4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandrost-1(10),2-diene-4-oic acid (4,9-DSHA) (VII); 2-hydroxyhexa-2,4-dienoic acid (VIII); 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid (IX); 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid CoA-ester (X).
indispensable for this configurational change were found in one of the two major steroid degradation gene clusters in *C. testosteroni* TA441 [34].

For the complete degradation of testosterone, progesterone, epiandrosterone, dehydroepiandrosterone, and cholic acid and its analogs, *C. testosteroni* TA441 requires 3α-dehydrogenase, 3-ketosteroid-Δ4(5)-isomerase, 3β-dehydrogenase, and 17β-dehydrogenase genes, a set of genes for degradation of the side chain at the C-17 position of cholic acid and progesterone, as well as already identified steroid degradation genes. Among the predicted genes, we isolated a putative 3α-dehydrogenase gene, a putative 3-ketosteroid-Δ4(5)-isomerase gene, and a putative 3β,17β-dehydrogenase gene, whose deduced amino acid sequences showed about 95% identity with those of the corresponding genes of *C. testosteroni* ATCC11996. Because 3α-dehydrogenase and 3-ketosteroid-Δ4(5)-isomerase are well studied in *C. testosteroni* ATCC11996 while 3β,17β-dehydrogenase was less investigated, we constructed a putative 3β,17β-dehydrogenase gene-disrupted mutant of *C. testosteroni* TA441 and found that it did not degrade epiandrosterone, which has a β-oriented hydroxyl group at the C-3 position and a ketone group at the C-17 position. The putative 3β,17β-dehydrogenase gene-disrupted mutant did degrade testosterone, which has a β-oriented hydroxyl group at the C-17 position and a ketone group at the C-3 position, but the growth was noticeably slower than that of the *C. testosteroni* TA441 control. Both of these results were supported by HPLC analysis of the intermediate compounds. From these results, this enzyme in *C. testosteroni* TA441 is considered to be a unique 3β-dehydrogenase possessing a degree of 17β-dehydrogenase activity, but *C. testosteroni* TA441 is likely to have at least one more enzyme with 17β-dehydrogenation activity. Dehydrogenation and hydrogenation at the C-17 position is thought to be a reversible reaction in *C. testosteroni* TA441 because almost all the major intermediate compounds isolated so far have a ketone group at the C-17 position and are accompanied by a small amount of a compound of the same structure with the exception of a hydroxyl group at the C-17 position. *C. testosteroni* TA441 may have more than one enzyme with 17β-dehydrogenating/hydrogenating activity to deal with intermediate compounds having considerable structural differences.

3α-Dehydrogenase [44–46,48,52] and 3-ketosteroid-Δ4(5)-isomerase [9,21,47,53–70] of *C. testosteroni* ATCC11996 were studied in detail, that we did little experimental examination of corresponding genes in *C. testosteroni* TA441. Cholic acid and its analogs have an α-oriented hydroxyl group at the C-3 position. These compounds are well-utilized by *C. testosteroni* TA441 upon induction of the putative 3α-dehydrogenase gene, suggesting that *C. testosteroni* TA441 uses 3α-dehydrogenase to produce a ketone group at the C-3 position. A TesI (Δ4-dehydrogenase)-disrupted mutant of *C. testosteroni* TA441 can grow on dehydroepiandrosterone, which has a double bond at Δ5(6), but cannot grow on epiandrosterone, which lacks a double bond at Δ5(6), indicating that *C. testosteroni* TA441 transfers the double bond at Δ5(6) to Δ4(5) using 3-ketosteroid-Δ4(5)-isomerase. Two genes encoding negative regulators of 3α-dehydrogenase, repA and repB, are found on the complementary chain of the 3α-dehydrogenase gene in *C. testosteroni* ATCC11996. The putative genes were also found in the corresponding positions in *C. testosteroni* TA441 (Fig. 3), but these putative proteins showed some differences to those of *C. testosteroni* ATCC11996. Identity between the RepA proteins of each strain was around 85%, which is obviously lower than homology between other corresponding steroid degradation enzymes, and RepB of *C. testosteroni* TA441 (303 aa) is suggested to be a fusion protein by homology search, that is longer than RepB of *C. testosteroni* ATCC11996 (78 aa).

The entire genome sequence of *C. testosteroni* CNB-2 was recently reported [49], and it has approximately 95% identity with

the sequenced DNA region of *C. testosteroni* TA441. We compared the genes/ORFs of *C. testosteroni* TA441 that are involved in steroid degradation, the two main steroid degradation gene clusters (the region from steA to tesR and from tesG to ORF18) and the two DNA fragments isolated in this study, with the corresponding DNA region of *C. testosteroni* CNB-2. Our comparison revealed that the steA to tesR region, the 3α-dehydrogenase and 3-ketosteroid-Δ4(5)-isomerase genes, and the tesG to ORF18 region are on the same DNA strand several dozen kilobases apart. Nearly 90% of these interspatial DNA regions in *C. testosteroni* TA441 have been sequenced to date. Most of the corresponding putative ORFs in *C. testosteroni* TA441 and CNB-2 are in the same order and the deduced amino acid sequences show more than 95% identity. A homology search suggested the possibility that a number of genes in this region are involved in steroid degradation, which implies that this DNA region of more than 100 kb is a hot spot of steroid degradation genes. Particularly, DNA region about 15 kb away from tesG is likely to be involved in the degradation of the cholic acid side chain at the C-17 position, because the initial step in the degradation of the cholic acid C-17 side chain is suggested to be a β-oxidation reaction and a number of putative β-oxidation genes are included in this region. Although none of the genes involved in degradation of the side chain at position C-17 in cholic acid have been isolated from *C. testosteroni* strains, the acyl-CoA-dehydrogenase for 7α,12α-dihydroxy-3-oxo-pregna-1,4-diene-20-carboxylate CoA-ester was isolated from *Pseudomonas* sp. Chol1 [71]. 7α,12α-Dihydroxy-3-oxo-pregna-1,4-diene-20-carboxylate CoA-ester is an intermediate compounds in the degradation of the side chain at C-17 position in colic acid, which is produced by removal of two carbons at C-23 and C-24 positions of cholic acid. A putative acyl-CoA-dehydrogenase gene showing about 70% amino acid identity to the acyl-CoA-dehydrogenase for 7α,12α-dihydroxy-3-oxo-pregna-1,4-diene-20-carboxylate CoA-ester of *Pseudomonas* sp. Chol1 was found in the DNA region about 15 kb away from tesG in *C. testosteroni* TA441. In addition, one of the Tn5-mutants of *C. testosteroni* TA441 constructed in our previous study accumulates 7α,12α-dihydroxy-3-oxo-pregna-1,4-diene-20-carboxylic acid when incubated with cholic acid (manuscript in preparation). Tn-5 was inserted into the putative acyl-CoA-dehydrogenase gene for 7α,12α-dihydroxy-3-oxo-pregna-1,4-diene-20-carboxylate CoA-ester in this mutant, strongly supporting speculation that this putative gene is the acyl-CoA-dehydrogenase for 7α,12α-dihydroxy-3-oxo-pregna-1,4-diene-20-carboxylate CoA-ester. These information suggest the hypothesis that putative genes in the neighboring region are also involved in degradation of the a side chain at position C-17 of cholic acid. Further analyses of the possible steroid degradation gene hot spot as well as studies on degradation of the side chain at position C-17 of cholic acid as well as steroid C- and D-rings are now underway.

Although *C. testosteroni* TA441 degraded the side chain at C-17 position of cholic acid and progesterone, it did not degrade cholesterol and other steroid compounds that have a more complicated side chain at the C-17 position, including plant sterols and estrogens. To the best of our knowledge, Gram-positive bacteria, such as *Rhodococcus* sp., are capable of degrading a wider range of steroid compounds [72]. Gram-positive bacteria have isozymes for steroid degradation [73,74] and will have more than one steroid degradation pathway [75], which may be responsible for the ability of these bacteria to degrade such a wide range of steroid compounds. *C. testosteroni* TA441 basically has only one steroid degradation pathway because gene-disruption usually lead to accumulation of intermediate compounds. It also indicates that there is only one enzyme for a single reaction, which resulted in successful investigation on the degradation pathway and the genes involved there in. Conceivably, for pharmaceutical or biomedical applications, com-
combined use of Gram-positive bacteria and gene-disrupted mutants of *C. testosteroni* TA441 might be effective for producing useful steroidal compounds, though studies on bacterial steroid degradation in more genera will be necessary to fully elucidate the genes and pathways involved.

## Acknowledgments

This work was partly supported by a grant from the Eco Molecular Sciences Research Program from RIKEN.

## References

[1] R.M. Dodson, R.D. Muir, Microbiological transformations. II. Microbiological aromatization of steroids, J. Am. Chem. Soc. 80 (1958) 5004–5005.

[2] R.M. Dodson, R.D. Muir, Microbiological transformations. III. The hydroxylation of steroids at C-9, J. Am. Chem. Soc. 80 (1958) 6148.

[3] R.M. Dodson, R.D. Muir, Microbiological transformations. VI. The microbiological aromatization of steroids, J. Am. Chem. Soc. 83 (1961) 4627–4631.

[4] C.J. Sih, Mechanisms of steroid oxidation by microorganisms, Biochim. Biophys. Acta 62 (1962) 541–547.

[5] C.J. Sih, R.E. Bennett, Steroid I-dehydrogenase of *Nocardia restrictus*, Biochim. Biophys. Acta 56 (1962) 584–592.

[6] C.J. Sih, K.C. Wang, Mechanisms of steroid oxidation by microorganisms. II. Isolation and characterization of 3α-H-4α-[3′-propionic acid]-7αβ-methylhezahydro-1,5-indanedione, J. Am. Chem. Soc. 85 (1963) 2135–2137.

[7] D.A. Shaw, L.F. Borkenhagen, P. Talalay, Enzymatic oxidation of steroids by cell-free extracts of *Pseudomonas testosteroni*: isolation of cleavage products of ring A, Proc. Natl. Acad. Sci. U.S.A. 54 (1965) 837–844.

[8] C.J. Sih, S.S. Lee, Y.K. Tsong, K.C. Wang, 3,4-Dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione. An intermediate in the microbiological degradation of ring A of androst-4-ene-3,17-dione, J. Am. Chem. Soc. 87 (1965) 1385–1386.

[9] P. Talalay, J. Boyer, Preparation of crystalline delta-5-3-ketosteroid isomerase from *Pseudomonas testosteroni*, Biochim. Biophys. Acta 105 (1965) 389–392.

[10] D.T. Gibson, K.C. Wang, C.J. Sih, H. Whitlock Jr., Mechanisms of steroid oxidation by microorganisms. IX. On the mechanism of ring A cleavage in the degradation of 9,10-seco steroids by microorganisms, J. Biol. Chem. 241 (1966) 551–559.

[11] C.J. Sih, S.S. Lee, Y.K. Tsong, K.C. Wang, Mechanisms of steroid oxidation by microorganisms. VIII. 3,4-Dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione, an intermediate in the microbiological degradation of ring A of androst-4-ene-3,17-dione, J. Biol. Chem. 241 (1966) 540–550.

[12] A.W. Coulter, P. Talalay, Studies on the microbial degradation of steroid ring A, J. Biol. Chem. 243 (1968) 3238–3247.

[13] H.H. Tai, C.J. Sih, 3,4-Dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione 4,-5-dioxygenase from *Nocardia restrictus*. I. Isolation of the enzyme and study of its physical and chemical properties, J. Biol. Chem. 245 (1970) 5062–5071.

[14] M.E. Tenneson, J.D. Baty, R.F. Bilton, A.N. Mason, The degradation of chenodeoxycholic acid by *Pseudomonas* Spp. N.C.I.B. 10590, J. Steroid Biochem. 10 (1979) 311–316.

[15] H. Groh, R. Komel, V. Deppmeyer, W. Schade, C. Horhold, Steroid transforming enzymes: the reverse reaction of the steroid-1-dehydrogenase from *Nocardia*, J. Steroid Biochem. 13 (1980) 1413–1415.

[16] M. Watanabe, D. Lefebvre, Y. Lefebvre, L.P. Sy, Membrane-bound dehydrogenases of *Pseudomonas testosteroni*, J. Steroid Biochem. 13 (1980) 821–827.

[17] A. Strijewski, The steroid-9 alpha-hydroxylation system from *Nocardia species*, Eur. J. Biochem. 128 (1982) 125–135.

[18] M.M. Francis, M. Watanabe, Purification and characterization of a membrane-associated testosterone-binding protein from *Pseudomonas testosteroni*, Can. J. Biochem. Cell Biol. 61 (1983) 307–312.

[19] E.M. Westbrook, P.B. Sigler, Enzymatic function in crystals of delta 5-3-ketosteroid isomerase. Catalytic activity and binding of competitive inhibitors, J. Biol. Chem. 259 (1984) 9090–9095.

[20] R.J. Park, N.W. Dunn, J.A. Ide, A catecholic 9,10-seco steroid as a product of aerobic catabolism of cholic acid by a *Pseudomonas* sp., Steroids 48 (1986) 439–450.

[21] K.Y. Choi, W.F. Benisek, Cloning of the gene for Δ5-3-ketosteroid isomerase from *Pseudomonas testosteroni*, Gene 58 (1987) 257–264.

[22] R.A. Leppik, D.J. Sinden, *Pseudomonas* mutant strains that accumulate androstane and seco-androstane intermediates from bile acids, Biochem. J. 243 (1987) 15–21.

[23] R.A. Leppik, Steroid catechol degradation: disecoandrostanediintermediates accumulated by *Pseudomonas* transposon mutant strains, J. Gen. Microbiol. 135 (1989) 1979–1988.

[24] J.J. Marin, How we have learned about the complexity of physiology, pathobiology and pharmacology of bile acids and biliary secretion, World J. Gastroenterol. 14 (2008) 5617–5619.

[25] J. Koga, H. Kubota, S. Gomi, K. Umemura, M. Ohnishi, T. Kono, Cholic acid, a bile acid elicitor of hypersensitive cell death, pathogenesis-related protein synthesis, and phytoalexin accumulation in rice, Plant Physiol. 140 (2006) 1475–1483.

[26] T. Shimizu, Y. Jikumaru, A. Okada, K. Okada, J. Koga, K. Umemura, E. Minami, N. Shibuya, M. Hasegawa, O. Kodama, H. Nojiri, H. Yamane, Effects of a bile acid elicitor, cholic acid, on the biosynthesis of diterpenoid phytoalexins in suspension-cultured rice cells, Phytochemistry 69 (2008) 973–981.

[27] M. Horinouchi, T. Yamamoto, K. Taguchi, H. Arai, T. Kudo, Meta-cleavage enzyme gene *tesB* is necessary for testosterone degradation in *Comamonas testosteroni* TA441, Microbiology 147 (2001) 3367–3375.

[28] M. Horinouchi, T. Hayashi, H. Koshino, T. Yamamoto, T. Kudo, Gene encoding the hydrolase for the product of the meta-cleavage reaction in testosterone degradation by *Comamonas testosteroni*, Appl. Environ. Microbiol. 69 (2003) 2139–2152.

[29] M. Horinouchi, T. Hayashi, T. Yamamoto, T. Kudo, A new bacterial steroid degradation gene cluster which consists of aromatic compound degradation genes for seco-steroids and 3-ketosteroid dehydrogenase genes in *Comamonas testosteroni* TA441, Appl. Environ. Microbiol. 69 (2003) 4421–4430.

[30] M. Horinouchi, T. Kurita, T. Yamamoto, E. Hatori, T. Hayashi, T. Kudo, Steroid degradation gene cluster of *Comamonas testosteroni* consisting of 18 putative genes from meta-cleavage enzyme gene *tesB* to regulator gene *tesR*, Biochem. Biophys. Res. Commun. 324 (2004) 597–604.

[31] M. Horinouchi, T. Hayashi, T. Kudo, The genes encoding the hydroxylase of 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione in steroid degradation in *Comamonas testosteroni* TA441, J. Steroid Biochem. Mol. Biol. 92 (2004) 143–154.

[32] M. Horinouchi, T. Hayashi, T. Kurita, T. Kudo, Identification of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid, 4-hydroxy-2-oxohexanoic acid, and 2-hydroxyhexa-2,4-dienoic acid and related enzymes involved in testosterone degradation in *Comamonas testosteroni* TA441, Appl. Environ. Microbiol. 71 (2005) 5275–5281.

[33] M. Horinouchi, T. Hayashi, H. Koshino, T. Kudo, ORF18-disrupted mutant of *Comamonas testosteroni* TA441 accumulates significant amounts of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid and its derivatives after incubation with steroids, J. Steroid Biochem. Mol. Biol. 101 (2006) 78–84.

[34] M. Horinouchi, T. Hayashi, H. Koshino, T. Yamamoto, M. Malon, T. Kudo, Identification of genes involved in inversion of stereochemistry of a C-12 hydroxyl group in the catabolism of cholic acid by *Comamonas testosteroni* TA441, J. Bacteriol. 190 (2008) 5545–5554.

[35] P. Plesiat, M. Grandguillot, S. Harayama, S. Vragar, Y. Michel-Briand, Cloning, sequencing, and expression of the *Pseudomonas testosteroni* gene encoding 3-oxosteroid Δ1-dehydrogenase, J. Bacteriol. 173 (1991) 7219–7227.

[36] C. Florin, T. Kohler, M. Grandguillot, P. Plesiat, *Comamonas testosteroni* 3-ketosteroid-delta 4(5 alpha)-dehydrogenase: gene and protein characterization, J. Bacteriol. 178 (1996) 3322–3330.

[37] D. Skowasch, E. Mobus, E. Maser, Identification of a novel *Comamonas testosteroni* gene encoding a steroid-inducible extradiol dioxygenase, Biochem. Biophys. Res. Commun. 294 (2002) 560–566.

[38] J.L. Pruneda-Paz, M. Linares, J.E. Cabrera, S. Genti-Raimondi, TeiR, a LuxR-type transcription factor required for testosterone degradation in *Comamonas testosteroni*, J. Bacteriol. 186 (2004) 1430–1437.

[39] A. Gohler, G. Xiong, S. Paulsen, G. Trentmann, E. Maser, Testosterone-inducible regulator is a kinase that drives steroid sensing and metabolism in *Comamonas testosteroni*, J. Biol. Chem. 283 (2008) 17380–17390.

[40] M. Linares, J.L. Pruneda-Paz, L. Reyna, S. Genti-Raimondi, Regulation of testosterone degradation in *Comamonas testosteroni*, J. Steroid Biochem. Mol. Biol. 112 (2008) 145–150.

[41] S. Genti-Raimondi, M.E. Tolmasky, L.C. Patrito, A. Flury, L.A. Actis, Molecular cloning and expression of the beta-hydroxysteroid dehydrogenase gene from *Pseudomonas testosteroni*, Gene 105 (1991) 43–49.

[42] S.-J. Yin, N. Vagelopoulos, G. Lundquist, H. Joernvall, *Pseudomonas* 3 beta-hydroxysteroid dehydrogenase. Primary structure and relationships to other steroid dehydrogenases, Eur. J. Biochem. 197 (1991) 359–365.

[43] J.H. Abalain, S. Di Stefano, Y. Amet, E. Quemener, M.L. Abalain-Colloc, H.H. Floch, Cloning, DNA sequencing and expression of (3–17)β hydroxysteroid dehydrogenase from *Pseudomonas testosteroni*, J. Steroid Biochem. Mol. Biol. 44 (1993) 133–139.

[44] J.H. Abalain, S. Di Stefano, M.L. Abalain-Colloc, H.H. Floch, Cloning and expression of *Pseudomonas testosteroni* gene encoding 3 alpha-hydroxysteroid dehydrogenase, J. Steroid Biochem. Mol. Biol. 55 (1995) 233–238.

[45] U.C. Oppermann, E. Maser, Characterization of a 3 alpha-hydroxysteroid dehydrogenase/carbonyl reductase from the gram-negative bacterium *Comamonas testosteroni*, Eur. J. Biochem. 241 (1996) 744–749.

[46] E. Mobus, E. Maser, Molecular cloning, overexpression, and characterization of steroid-inducible 3alpha-hydroxysteroid dehydrogenase/carbonyl reductase from *Comamonas testosteroni*. A novel member of the short-chain dehydrogenase/reductase superfamily, J. Biol. Chem. 273 (1998) 30888–30896.

[47] A. Kuliopulos, D. Shortle, P. Talalay, Isolation and sequencing of the gene encoding delta 5-3-ketosteroid isomerase of *Pseudomonas testosteroni*: over-expression of the protein, Proc. Natl. Acad. Sci. U.S.A. 84 (1987) 8893–8897.

[48] G. Xiong, H.J. Martin, E. Maser, Identification and characterization of a novel translational repressor of the steroid-inducible 3 alpha-hydroxysteroid dehydrogenase/carbonyl reductase gene in *Comamonas testosteroni*, J. Biol. Chem. 278 (2003) 47400–47407.

[49] Y.F. Ma, Y. Zhang, J.Y. Zhang, D.W. Chen, Y. Zhu, H. Zheng, S.Y. Wang, C.Y. Jiang, G.P. Zhao, S.J. Liu, The complete genome of *Comamonas testosteroni* reveals its genetic adaptations to changing environments, Appl. Environ. Microbiol. 75 (2009) 6812–6819.

[50] J. Parkhill, M. Sebaihia, A. Preston, L.D. Murphy, N. Thomson, D.E. Harris, M.T. Holden, C.M. Churcher, S.D. Bentley, K.L. Mungall, A.M. Cerdeno-Tarraga, L. Temple, K. James, B. Harris, M.A. Quail, M. Achtmann, R. Atkin, S. Baker, D. Basham, N. Bason, I. Cherevach, T. Chillingworth, M. Collins, A. Cronin, P. Davis, J. Doggett, T. Feltwell, A. Goble, N. Hamlin, H. Hauser, S. Holroyd, K. Jagels, S. Leather, S. Moule, H. Norberczak, S. O'Neil, D. Ormond, C. Price, E. Rabbinowitsch, S. Rutter, M. Sanders, D. Saunders, K. Seeger, S. Sharp, M. Simmonds, J. Skelton, R. Squares, S. Squares, K. Stevens, L. Unwin, S. Whitehead, B.G. Barrell, D.J. Maskell, Comparative analysis of the genome sequences of *Bordetella pertussis*, *Bordetella parapertussis* and *Bordetella bronchiseptica*, Nat. Genet. 35 (2003) 32–40.

[51] J.E. Cabrera, G. Panzetta-Dutari, J.L. Pruneda, S. Genti-Raimondi, A new *Comamonas testosteronei* steroid-inducible gene: Cloning and Sequence Analysis, J. Steroid Biochem. Mol. Biol. 63 (1997) 91–98.

[52] E. Maser, G. Xiong, C. Grimm, R. Ficner, K. Reuter, 3alpha-Hydroxysteroid dehydrogenase/carbonyl reductase from *Comamonas testosteronei*: biological significance, three-dimensional structure and gene regulation, Chem. Biol. Interact. 30 (2001) 707–722.

[53] M.G. Ward, L.L. Engel, Reversibility of steroid delta-isomerase. 3. The soluble enzyme of *Pseudomonas testosteronei*, J. Biol. Chem. 241 (1966) 3154–3157.

[54] F. Vincent, H. Weintraub, A. Alfsen, Conformation of native *Pseudomonas testosteronei* Delta(5)→(4) 3-oxosteroid isomerase, FEBS Lett. 22 (1972) 319–322.

[55] A.M. Benson, A.J. Suruda, R. Shaw, P. Talalay, Affinity chromatography of 3-oxosteroid delta4-delta5-isomerase of *Pseudomonas testosteronei*, Biochim. Biophys. Acta 348 (1974) 317–320.

[56] A.M. Benson, A.J. Suruda, P. Talalay, Concentration-dependent association of delta5-3-ketosteroid isomerase of *Pseudomonas testosteronei*, J. Biol. Chem. 250 (1975) 276–280.

[57] H. Weintraub, F. Vincent, E.E. Baulieu, A. Alfsen, Interaction of steroids with *Pseudomonas testosteronei* 3-oxosteroid delta4-delta5-isomerase, Biochemistry 16 (1977) 5045–5053.

[58] T.M. Penning, D.F. Covey, P. Talalay, Inactivation of delta 5-3-oxo steroid isomerase with active-site-directed acetylenic steroids, Biochem. J. 193 (1981) 217–227.

[59] A. Viger, S. Coustal, A. Marquet, A reinvestigation of the mechanism of delta 5-3-ketosteroid isomerase from bovine adrenal glands, Biochim. Biophys. Acta 660 (1981) 170–173.

[60] K.Y. Choi, W.F. Benisek, Nucleotide sequence of the gene for the delta 5-3-ketosteroid isomerase of *Pseudomonas testosteronei*, Gene 69 (1988) 121–129.

[61] B. Brooks, W.F. Benisek, Mechanism of the reaction catalyzed by delta 5-3-ketosteroid isomerase of *Comamonas* (*Pseudomonas*) *testosteronei*: kinetic properties of a modified enzyme in which tyrosine 14 is replaced by 3-fluorotyrosine, Biochemistry 33 (1994) 2682–2687.

[62] P. Wu, Y.K. Li, P. Talalay, L. Brand, Characterization of the three tyrosine residues of delta 5-3-ketosteroid isomerase by time-resolved fluorescence and circular dichroism, Biochemistry 33 (1994) 7415–7422.

[63] J.C. Austin, Q. Zhao, T. Jordan, P. Talalay, A.S. Mildvan, T.G. Spiro, Ultraviolet resonance Raman spectroscopy of delta 5-3-ketosteroid isomerase revisited: substrate polarization by active-site residues, Biochemistry 34 (1995) 4441–4447.

[64] C.M. Holman, W.F. Benisek, Insights into the catalytic mechanism and active-site environment of *Comamonas testosteronei* delta 5-3-ketosteroid isomerase as revealed by site-directed mutagenesis of the catalytic base aspartate-38, Biochemistry 34 (1995) 14245–14253.

[65] Q. Zhao, A.S. Mildvan, P. Talalay, Enzymatic and nonenzymatic polarizations of alpha,beta-unsaturated ketosteroids and phenolic steroids. Implications for the roles of hydrogen bonding in the catalytic mechanism of delta 5-3-ketosteroid isomerase, Biochemistry 34 (1995) 426–434.

[66] H.S. Cho, N.C. Ha, G. Choi, H.J. Kim, D. Lee, K.S. Oh, K.S. Kim, W. Lee, K.Y. Choi, B.H. Oh, Crystal structure of delta(5)-3-ketosteroid isomerase from *Pseudomonas testosteronei* in complex with equilenin settles the correct hydrogen bonding scheme for transition state stabilization, J. Biol. Chem. 274 (1999) 32863–32868.

[67] R.M. Pollack, L.D. Thornburg, Z.R. Wu, M.F. Summers, Mechanistic insights from the three-dimensional structure of 3-oxo-Delta(5)-steroid isomerase, Arch. Biochem. Biophys. 370 (1999) 9–15.

[68] F. Henot, R.M. Pollack, Catalytic activity of the D38A mutant of 3-oxo-Delta 5-steroid isomerase: recruitment of aspartate-99 as the base, Biochemistry 39 (2000) 3351–3359.

[69] G.H. Nam, S.S. Cha, Y.S. Yun, Y.H. Oh, B.H. Hong, H.S. Lee, K.Y. Choi, The conserved cis-Pro39 residue plays a crucial role in the proper positioning of the catalytic base Asp38 in ketosteroid isomerase from *Comamonas testosteronei*, Biochem. J. 375 (2003) 297–305.

[70] H.J. Lee, Y.J. Yoon, S. Jang do, C. Kim, H.J. Cha, B.H. Hong, K.Y. Choi, H.C. Lee, 15N NMR relaxation studies of Y14F mutant of ketosteroid isomerase: the influence of mutation on backbone mobility, J. Biochem. 144 (2008) 159–166.

[71] A. Birkenmaier, J. Holert, H. Erdbrink, H.M. Moeller, A. Friemel, R. Schoenenberger, M.J. Suter, J. Klebensberger, B. Philipp, Biochemical and genetic investigation of initial reactions in aerobic degradation of the bile acid cholate in *Pseudomonas* sp. strain Chol1, J. Bacteriol. 189 (2007) 7165–7173.

[72] R. Van der Geize, K. Yam, T. Heuser, M.H. Wilbrink, H. Hara, M.C. Anderton, E. Sim, L. Dijkhuizen, J.E. Davies, W.W. Mohn, L.D. Eltis, A gene cluster encoding cholesterol catabolism in a soil actinomycete provides insight into *Mycobacterium tuberculosis* survival in macrophages, Proc. Natl. Acad. Sci. U.S.A. 104 (2007) 1947–1952.

[73] R. van der Geize, G.I. Hessels, R. van Gerwen, P. van der Meijden, L. Dijkhuizen, Molecular and functional characterization of *kshA* and *kshB*, encoding two components of 3-ketosteroid 9alpha-hydroxylase, a class IA monooxygenase, in *Rhodococcus erythropolis* strain SQ1, Mol. Microbiol. 45 (2002) 1007–1018.

[74] R. van der Geize, G.I. Hessels, M. Nienhuis-Kuiper, L. Dijkhuizen, Characterization of a second *Rhodococcus erythropolis* SQ1 3-ketosteroid 9alpha-hydroxylase activity comprising a terminal oxygenase homologue, KshA2, active with oxygenase-reductase component KshB, Appl. Environ. Microbiol. 74 (2008) 7197–7203.

[75] S. Hayakawa, T. Fujiwara, Microbiological degradation of bile acids. Further degradation of a cholic aci1d metabolite containing the hexahydroindane nucleus by *Corynebacterium equi*, Biochem. J. 162 (1977) 387–397.
