
Cell Cycle

ISSN: 1538-4101 (Print) 1551-4005 (Online) Journal homepage: http://www.tandfonline.com/loi/kccy20

# p16Ink4A, not only a G₁ inhibitor?

## Vjekoslav Dulić

To cite this article: Vjekoslav Dulić (2010) p16Ink4A, not only a G₁ inhibitor?, Cell Cycle, 9:16, 3166-3170, DOI: 10.4161/cc.9.16.12887

To link to this article: http://dx.doi.org/10.4161/cc.9.16.12887

Copyright © 2010 Landes Bioscience

Published online: 15 Aug 2010.

Submit your article to this journal

Article views: 13

View related articles

Full Terms & Conditions of access and use can be found at
http://www.tandfonline.com/action/journalInformation?journalCode=kccy20

Download by: [80.222.53.36]
Date: 14 September 2015, At: 05:08

Cell Cycle 9:16, 3146-3150; August 15, 2010; © 2010 Landes Bioscience

# Cell Cycle News & Views

## tRNA trafficking along the TOR pathway

### Comment on: Huynh LN, et al. Cell Cycle 2010; 9:3112–18.

Thomas P. Neufeld${}^{1,*}$ and Andrew M. Arsham${}^{2,*}$; ${}^{1}$University of Minnesota, Minneapolis, MN USA; ${}^{2}$Wave 80 Biosciences, San Francisco, CA USA  
*Email: neufe003@umn.edu and andrew.arsham@wave80.com; DOI: 10.4161/cc.9.16.12781

Adaptation of metabolic activity to nutrient availability is among the most ancestral of cellular behaviors. Resource-intensive processes such as growth and protein synthesis are particularly tightly coupled to nutrient levels, ensuring that cells devote resources to these functions only under appropriate conditions. The target of rapamycin (TOR) protein kinase is an essential and conserved component of such regulation, and orchestrates a comprehensive set of cellular responses to nutrient levels. Under favorable conditions, TOR signaling promotes protein synthesis through upregulation of ribosome biogenesis and direct activation of the translation machinery. In response to nutrient withdrawal, reduction of TOR activity inhibits biosynthesis and causes activation of autophagy, in which cytoplasmic components are degraded in the lysosomal compartment. These responses promote cell survival during periods of starvation by reducing demand and generating an intracellular source of recycled nutrients. How the TOR pathway senses nutrient status is only partially understood. One upstream signal involves the Rag family of small GTPases, which regulate the association of TOR with the GTPase Rheb, a direct activator of TOR. The more proximal upstream steps in this pathway by which amino acid levels are assessed have remained elusive, with nutrient transporters, uncharged tRNAs, and intracellular levels of ATP or calcium each being proposed as potential mediators of the signal.$^{1}$

tRNAs are essential intermediates of protein synthesis, translating the mRNA ribonucleotide code into polypeptide sequence. tRNAs synthesized in the nucleus are actively exported to the cytoplasm by specific transport factors. Recent studies in yeast and mammalian cells have also identified a retrograde pathway of tRNA import into the nucleus.$^{2,3}$ Interestingly, the balance between nuclear and cytoplasmic pools of tRNA is regulated by nutrient availability, with starvation causing a rapid and reversible accumulation of tRNA in the nucleus. This response has been proposed to provide an alternative means of translational control under nutrient-poor conditions, by reducing cytoplasmic pools of charged tRNAs available for polypeptide chain elongation.

A report by Huynh et al. in a previous issue of *Cell Cycle*$^{4}$ provides further insight into this process, and identifies a new role for tRNA trafficking in transducing nutrient signals and controlling TOR-dependent responses. These authors manipulated tRNA localization by targeting the karyopherin exportin-t (Xpo-t), a tRNA-specific nuclear export receptor. Depletion of Xpo-t in human fibroblasts led to accumulation of tRNA in the nucleus, consistent with previous studies in yeast. In Xpo-t depleted cells, phosphorylation of several TOR-dependent targets (as well as TOR itself) was significantly reduced, suggesting that altered tRNA localization leads to down-regulation of TOR activity. Xpo-t depletion also caused activation of autophagy in these cells, consistent with the observed reduction in TOR signaling.

How might nuclear accumulation or cytoplasmic depletion of tRNA lead to a decrease in TOR activation? This response would not be expected to result indirectly from reduced translational capacity, as inhibitors of protein synthesis generally have a positive effect on TOR activation, presumably by increasing the intracellular concentration of free amino acids. Interestingly, recent reports have described a number of non-canonical functions of tRNAs including transcriptional regulation, mRNA degradation, translation inhibition, and suppression of apoptosis.$^{5-7}$ In addition, stimuli such as viral infection, DNA damage and oxidative stress can also lead to nuclear accumulation or specific cleavage of tRNA.$^{5,7,8}$ These studies implicate the processing and trafficking of tRNA as potential intermediate steps in a number of responses to different cellular stresses, and present a wide range of possible mechanisms by which tRNA exerts regulatory effects on cellular nutrient and energy balance. Taken together, the data suggest that, in addition to its passive role as an adaptor molecule for protein synthesis, tRNA could be yet another in the family of non-coding regulatory RNA molecules that have emerged as potent regulators of cell and developmental biology.

The study by Huynh et al. also raises the question of how nutrient conditions affect tRNA localization. In yeast, retrograde import of tRNA has been shown to be constitutive, whereas re-export of imported tRNA is responsive to nutrient levels.$^{9}$ Although TOR activity is reduced by amino acid starvation, inhibition of TOR with rapamycin does not induce nuclear accumulation of tRNA. Rather, rapamycin was unexpectedly found to block nuclear tRNA accumulation in response to deprivation of amino acids, and had no effect on nuclear tRNA accumulation in response to glucose deprivation.$^{10}$ These results indicate that different stresses can signal to the tRNA export machinery using distinct pathways. This response may be regulated in part at the level of tRNA aminoacylation by tRNA synthetases, since defects in this process can also block tRNA export.$^{11}$ Recent genetic screens in Drosophila identified mutations in aminoacyl-tRNA synthetases and nuclear transporters as causing reduction of cell size and activation of autophagy, consistent with a decrease in TOR activity.$^{12}$ Regardless of mechanism, the influence of tRNA trafficking on TOR signaling presents an interesting parallel with current models of nutrient-dependent TOR regulation, in which amino acids promote the Rag-dependent trafficking of TOR to its site of activation on the surface of the endosomal-lysosomal compartment. An important goal for future studies will be to understand the mechanisms by which nutrients affect the itineraries of these molecular journeys.

References
1. Avruch J, et al. Am J Physiol Endocrinol Metab 2009; 296:E592-602.
2. Shaheen HH, et al. Proc Natl Acad Sci USA 2007; 104:8845-50.
3. Shaheen HH, et al. Proc Natl Acad Sci USA 2005; 102:11290-5.
4. Huynh LN, et al. Cell Cycle 2010; 9:3112-18.

5. Thompson DM, et al. Cell 2009; 138:215-9.
6. Mei Y, et al. Mol Cell 2010; 37:668-78.
7. Qiu H, et al. Mol Cell Biol 2000; 20:2505-16.
8. Zaitseva L, et al. PLoS Biol 2006; 4:e332.
9. Murthi A, et al. Mol Biol Cell 2010; 21:639-49.
10. Whitney ML, et al. Mol Biol Cell 2007; 18:2678-86.
11. Grosshans H, et al. Genes Dev 2000; 14:830-40.
12. Arsham AM, et al. PLoS One. 2009; 4:e6068.

---

**A role for ATM kinase activity and Mre11 in microhomology-mediated end-joining**

**Comment on:** Rahal EA, et al. Cell Cycle 2010; 9:2866–77.

Anthony J. Davis and David J. Chen*; University of Texas Southwestern Medical Center; Dallas, TX USA;
*Email: david.chen@utsouthwestern.edu; DOI: 10.4161/cc.9.16.12814

The human genome is constantly under attack from endogenous and exogenous agents which cause DNA lesions. Of the various types of DNA damage that can occur, DNA double strand breaks (DSBs) are the most deleterious. The inability to repair DSBs can result in genomic instability, carcinogenesis, or cell death. Because of the deleterious nature of these events, cells have developed multiple mechanisms to repair DSBs. The two major pathways that are responsible for repairing DSBs are homologous recombination (HR) and non-homologous end-joining (NHEJ). The HR process mediates DSB repair by using a homologous DNA sequence as a template to guide proper restoration of the break. Because homologous templates are found on sister chromatids, HR is thought to be only active during the S and G2 cell cycle phases. NHEJ is characterized by its ability to directly ligate the two ends of the broken molecule. This process does not have the need for a homologous template and is therefore theoretically not restricted to a certain phase of the cell cycle. Recent studies have started to describe alternative pathways also responsible for repairing DSBs. One such pathway is termed microhomology-mediated end-joining (MMEJ).

MMEJ is an error-prone mechanism of repair which completes end-joining independently of the NHEJ factor Ku 70/80. Following the introduction of a DSB, a homologous stretch of 5–25 complementary base pairs on both strands is identified and used as a basis for which to align the strands with mismatches. Once there is annealing at the microhomologies, any non-complementary DNA bases, termed flaps, on the strands are removed via nucleases. The removal of the flaps may assist in stabilizing the DNA ends and provide a proper substrate for gap filling by polymerases. The DNA strands are finally ligated together, but due to the removal of the flaps and gap filling, MMEJ results in deletions, translocations, inversions, and other complex DNA rearrangements. It is due to these chromosomal abnormalities that MMEJ is frequently associated with chromosome rearrangements, cancer, and genetic variation.

In a previous issue of *Cell Cycle*, Rahal et al. continued the Dixon group's studies on the role of ataxia telangiectasia mutated (ATM) in controlling DNA end degradation via MMEJ. In this report, the authors show that a duplex DNA substrate is degraded in extracts from ATM-deficient cells (AT). In wild-type fibroblasts, very little of the duplex DNA substrate is degraded, but transient expression of kinase-dead ATM resulted in a decrease in DNA end stability similar to lysates from AT cells. This data shows that the kinase activity is required for repressing DNA degradation and correlates with the group's previous work which showed that inhibition of ATM kinase activity with the PIKK inhibitors wortmannin and caffeine prevented DNA end-protection. This data further establishes that ATM kinase activity prevents the degradation of free DNA ends possibly by suppressing nuclease activities.

Next, it was determined which nuclease ATM was possibly regulating in DNA end degradation. Previous work from other laboratories had shown a role for the nuclease activity of Mre11 for DNA end degradation and favoring alternative end-joining. Furthermore, ATM interacts with Mre11/Rad50/Nbs1 complex and phosphorylates Mre11 in response to DSBs. To test if Mre11 is the nuclease responsible for DNA end degradation in their assay, the authors first depleted Mre11 via immunodepletion. A significant amount of DNA end degradation was inhibited in the AT nuclear lysates when Mre11 was inhibited. This data was supported with experiments that showed that treatment with the Mre11 nuclease inhibitor, Mirin, and with shRNA knockdown of Mre11 also protected DNA ends from degradation in the AT nuclear lysates. Finally, using an in vivo MMEJ reporter assay, the data shows that Mre11 knock-down and inhibition results in a significant decrease in MMEJ. Taken together, the data implicates ATM's kinase activity in regulating Mre11's role in degrading DNA ends for MMEJ to occur.

Many new interesting questions are raised by this study: (a) how does ATM's kinase activity regulate Mre11 nuclease activity, is it via direct phosphorylation of Mre11, the MRN complex, or another means, (b) the data suggests that Mre11 may not be the only nuclease responsible for DNA-end degradation, is CtIP or Exo1 responsible for this degradation or another nuclease, (c) Ataxia-Telangiectasia is characterized by progressive cerebellar ataxia, immune deficiencies, radiation sensitivity, and an increased risk of cancer, it will be of interest if ATM role in MMEJ plays a role in any of these clinical phenotypes.

---

**References**
1. Khanna KK, et al. Nat Genet 2001; 27:247-54.
2. Hoeijmakers JH. Nature 2001; 411:366-74.
3. Burma S, et al. DNA Repair (Amst) 2006; 5:1042-8.
4. McVey M, et al. Trends Genet 2008; 24:529-38.
5. Li Y, et al. Environ Mol Mutagen 2001; 37:128-40.
6. Rahal EA, et al. DNA Repair (Amst) 2008; 7:464-75.
7. Paull TT, et al. Proc Natl Acad Sci USA 2000; 97:6409-14.
8. Rass E, et al. Nat Struct Mol Biol 2009; 16:819-24.
9. Xie A, et al. Nat Struct Mol Biol 2009; 16:814-8.
10. Lavin MF, et al. Cell Cycle 2007; 6:931-42.

# The flap about ATM and MRE11

## Comment on: Rahal EA, et al. Cell Cycle 2010; 9:2866–77.

Bevin P. Engelward; Massachusetts Institute of Technology; Cambridge, MA USA; Email: bevin@mit.edu; DOI: 10.4161/cc.9.16.12815

MRE11 is well established for its role in initiating the two major pathways of double strand break (DSB) repair: NHEJ and HR. ${ }^{1,2}$ What is now emerging is its importance in the lesser-known pathway of microhomology mediated end joining (MMEJ). Originally described as "illegitimate" recombination, ${ }^{3}$ MMEJ is virtually error-guaranteed and involves annealing of ~5-15 bp microhomologous regions. ${ }^{5}$ How is pathway choice regulated and what goes wrong if it is misregulated? Rahal et al. integrate biochemistry and crystallography to offer new insights into the three-pathway and three-dimensional world of DSB repair.

Solving the ATM paradox. Ataxia telangiectasia mutated (ATM) was originally discovered as the cause of severe neurological defects and radiation sensitivity, and ATM is now known to be a signal transduction hub, phosphorylating key proteins involved in DNA repair (e.g., BRCA1, NBS1, ARTEMIS, and Rad17) and cell cycle arrest (e.g., p53, E2F1, CHK2, RAD17, RAD9). ${ }^{6}$ Paradoxically, despite extreme radiation sensitivity, DSB repair is quite robust in ATM-deficient cells, ${ }^{7}$ suggesting that AT phenotypes might result from defective cell cycle arrest. However, arrest seemed an unlikely problem for non-dividing neurons.

In a previous issue of *Cell Cycle*, Dr. K. Dixon's group shows for the first time that AT mutant cells suffer from another problem: fidelity. Specifically, AT cells suffer from excessive error-prone MMEJ.

The discovery that MMEJ is overly-active in AT cells emerged from initial experiments focused on DNA degradation. Using both ATM-deficient cells and dominant negative ATM, Dixon's group showed that kinase-active wild-type ATM prevents DNA end degradation. Second, reduced MRE11 activity (both by immunodepletion and using an inhibitor), suppressed DNA degradation in AT cells. Next, they knocked down MRE11 expression and again found that DNA degradation was suppressed. A key insight linked MRE11 DNA degradation to MMEJ. Knowing that MRE11 participates in MMEJ, ${ }^{8}$ they asked if ATM might regulate MMEJ. They used an elegant plasmid-based assay in living cells, in which MMEJ at an I-SceI-induced DSB yields EGFP expression. Remarkably, MMEJ frequency rose dramatically in the absence of ATM. Furthermore, shRNA against MRE11 suppresses MMEJ in AT mutant cells. Thus, ATM is pivotal in pathway choice, rendering cells susceptible to error-prone MMEJ.

ATM and Mre11: Navigating among pathways. Just how might MRE11 impact MMEJ? The authors peered into the "crystal" ball to find answers that integrate structural and biochemical information. MRE11 is a 3'-5' exonuclease, an endonuclease, and it can tether DNA ends ${ }^{10-13}$. In an emerging model, DNA ends are bound by MRX (yeast MRN complex), and MRX mediates activation of Tel1 (ATM orthologue) (Fig. 1). After a decision to initiate MMEJ (mediated partly by Sae2 in yeast or CtlP in mammals), ends are resected by MRX and/or Exol. Microhomologous sequences are tethered by Mre11, and 3' flaps are removed by Rad1/Rad10 9 or Mre11. A handoff to Pol4 allows repair synthesis; resulting nicks are sealed by DNA ligase (ligase I, III or IV in S. cerevisiae; ligase I or III in mammals). Although most MMEJ models anneal 3' overhangs (Fig. 1), 5' overhangs created by Mre11's 3'-5' exonuclease activity could also be tethered by Mre11 (Fig. 1).

ATM's impact on MMEJ is likely mediated by CtlP. MRN-activated ATM promotes ATM-dependent phosphorylation events, including that of CtlP. Certain CtlP phosphorylation sites can dramatically impact CtlP's neighbor, NBS1, causing a wave of conformational changes

![Figure 1](attachment://figure_1.png)

*Figure 1. MMEJ models for finding microhomology. (A) Traditional MMEJ pathway with 3' overhangs with S. cerevisiae gnese, Mre11 (blue octagon), and microhomology regions (yellow bars) indicated. (B) Alternative MMEJ pathway in which microhomology is found in 5' overhangs. Regions of microhomology converge under the grip of Mre11, which simultaneously removes 3' overhangs with its exonuclease activity, and excess non-matching 5' sequences with its endonuclease activity (5' end degradation products not depicted).*
that sweeps from CtIP, through NBS1, and  
into MRN.¹⁴ Changes in MRN's conformation  
probably promote resection to initiate HR,  
whereas phosphorylation of CtIP at other sites  
allows switching between HR and MMEJ 15.  
Thus, ATM-dependent phosphorylation of CtIP  
controls communication with MRN, and in the  
absence of ATM, cells suffer from defective  
pathway choice.

Human health implications. Shifting from  
NHEJ toward MMEJ has serious implications. It  
is already know that cancer and other diseases  
can be promoted by MMEJ-driven mutations  
(e.g., BRCA2, RB, GALC and CFTR).⁵  

In terms of treating cancer, inhibiting MMEJ  
should sensitize tumors that are excessively  
dependent upon it, and synergize with treat-  
ments that inhibit NHEJ. In terms of the neu-  
rological defects, the impact of accumulated  
MMEJ-driven deletions may be significant in  
neurons, since subtle changes in cell func-  
tion can profoundly impact cognitive abilities.  
Taken together, the results from the Dixon  
and Tainer laboratories provide insights that  
support novel approaches for cancer chemo-  
therapy and novel treatments for AT patients.  

References  
1. Lavin MF, et al. Oncogene 2007; 26:7749-58.  
2. Helleday T, et al. DNA Repair (Amst) 2007; 6:923-35.  
3. Schiestl RH, et al. Proc Natl Acad Sci USA 1991;  
   88:7585-9.  
4. Boulton SJ, et al. EMBO J 1998; 17:1819-28.  
5. McVey M, et al. Trends Genet 2008; 24:529-38.  
6. Lavin MF, et al. Nat Rev Mol Cell Biol 2008; 9:759-69.  
7. Riballo E, et al. Mol Cell 2004; 16:715-24.  
8. Ma JL, et al. Mol Cell Biol 2003; 23:8820-8.  
9. Lee K, et al. Genetics 2007; 176:2003-14.  
10. Chen L, et al. Mol Cell 2001; 8:1105-15.  
11. de Jager M, et al. Mol Cell 2001; 8:1129-1135.  
12. Trujillo KM, et al. J Biol Chem 2001; 276:35458-64.  
13. Nairz K, et al. Genes Dev 1997; 11:2272-90.  
14. Williams RS, et al. Cell 2009; 139:87-99.  
15. Yun MH, et al. Nature 2009; 459:460-3.  

---

**Fat: Love it to death!**

**Comment on: Rockenfeller P, et al. Cell Cycle 2010; 9:2836–42.**

*Cristina Mazzoni; University of Rome “La Sapienza” Piazzale Aldo Moro; Rome, Italy; Email: cristina.mazzoni@uniroma1.it; DOI: 10.4161/cc.9.16.12834*

Obesity has been causally linked to an array  
of human diseases, including metabolic syn-  
drome, type II diabetes, diverse cardiovascular  
disorders, hepatosteatosis and an increased  
propensity to develop cancer.¹  

Although obesity is considered a severe  
problem, very little is known about the  
molecular mechanisms causing the associated  
degeneration of organs and finally death.  

Glucose levels have been traditionally  
considered responsible for the degeneration  
and death of insulin-producing beta-cells and,  
recently, also, free fatty acids (FFA), which are  
present in high concentration in plasma dur-  
ing obesity, have been involved in the devel-  
opment of diabetes.²  

Conversely, dietary restriction (DR) is the  
most robust environmental method to slow  
down aging in species as diverse as yeast,  
worms, fruit flies, rodents and primates.³  

It has been reported that FFA can induce  
apoptosis, albeit in tissue-specific patterns and  
by different mechanisms, probably through  
the production of elevated ceramide lev-  
els and/or reactive oxygen species. Due to  
the evolutionary conservation of the lipid  

metabolic machinery, lipotoxicity studies in  
yeast are relevant to higher eukaryotes and  
represent a suitable model to study this pro-  
cess in greater detail.⁴  

In a previous issue of *Cell Cycle*, Madeo  
and colleagues have added another piece to  
the lipotoxicity puzzle, by reporting that FFA  
stimulates necrosis through a mitochondrial-  
dependent pathway. The authors challenged  
a wild-type *S. cerevisiae* strain with various  
alimentary oils in the presence of triacylglyc-  
erol lipase from *Candida rugosa*, mimicking  
the physiological microenvironment of the  
small intestine, or a quadruple knockout strain  
(QKO), which is defective in triacylglycerol  
(TAG) synthesis and unable to esterify FFAs.⁵  

In all cases, cells accumulated ROS and  
showed high levels of PI positive cells, sug-  
gesting that FFA-mediated necrosis is pre-  
ceded and accompanied by excessive ROS  
generation.  

The necrotic nature of cell death was also  
confirmed using the release of the nuclear  
HMGB1 (a chromatin-bound non-histone pro-  
tein) into the cytosol has been established as a  
specific marker for necrotic cell death.⁶  

*S. cerevisiae* is also a powerful model  
organism to genetically dissect the path-  
ways required for maintenance of respiratory  
activity because it is capable of satisfying its  
energy requirements with ATP generated by  
fermentation.  

Yeast cells QKO devoid of mitochondrial  
DNA (rho0) were more resistant to FFA-induced  
cell death, underscoring the pivotal role of  
this organelle in lipotoxicity and in mediating  
necrosis, as also reported in mammalian cells.⁷  

Altogether, these findings point to the  
validity of yeast as a model system to identify  
the molecular mechanisms that modulate life  
span upon dietary uptake and extend our  
knowledge on lipotoxicity.  

References  
1. Kopelman PG. Nature 2000; 404:635-643.  
2. Lopez S, et al. Curr Opin Lipidol 2010; 21:15-20.  
3. Kapahi P, et al. Cell Metab 2010; 11:453-465.  
4. Garbarino J, et al. Curr Opin Clin Nutr Metab Care  
   2009; 12:110-116.  
5. Petschnigg J, et al. J Biol Chem 2009; 284:30981-30993  
6. Eisenberg T, et al. Apoptosis 2010; in press.  
7. Temkin V, et al. Mol Cell Biol 2006; 26:2215-2225.  
p16Ink4A, not only a G₁ inhibitor?

**Comment on:** Chien WW, et al. Cell Cycle 2010; 9:3286–96.

Vjekoslav Dulic; Institut de Génétique Moléculaire de Montpellier; Montpellier, France; Email: vjekoslav.dulic@igmm.cnrs.fr; DOI: 10.4161/cc.9.16.12887

Since its discovery in the early nineties, the cyclin dependent kinase inhibitor (CKI) p16INK4a (p16) remains one of the most intriguing representatives of this group of cell cycle regulators. p16 is a tumor suppressor whose inactivation, mostly by epigenetic silencing, was observed in number of malignancies, thus substantiating its crucial role in several anti-proliferative processes.¹ Unlike CKIs of the Cip/Kip family, composed of p21Waf1/Cip1, p27Kip1 and p57Kip2, which are rather promiscuous regarding their CDK targets, p16, like other members of INK4 family (p15INK4b, p18INK4c and p19INK4d), binds exclusively to Cdk4 and Cdk6, the catalytic partners of D-type cyclins and, when over-expressed, induces G₁ arrest.² Unlike the other cyclins, having roles in regulating DNA replication (E and A-type) or mitosis (A and B-type), D-type cyclins are thought to function as mitogen sensors creating a link between extracellular growth signals and the core cell cycle machinery. The most notorious, if not exclusive, Cdk4/6 substrates are tumor suppressors of the retinoblastoma (pRb) pocket protein family (p130, p107 and pRb) playing an essential role in cell cycle exit, taking place during onset of quiescence or senescence. Cdk-dependent phosphorylation, initiated by Cdk4/6 and carried out by other Cdks,³ inactivates pRb proteins and enables cell cycle progression. Consequently, the growth suppressive activity of INK4 proteins is strictly dependent on the function of pRb family members whose inactivation by viral oncproteins renders p16“useless” despite its increased expression in their absence. The regulation of p16 expression is poorly understood. Unlike p21, which is induced in response to DNA damage, or p27, which is stabilized in the absence of mitogens, the pathways controlling p16 induction are not well defined.⁴ For example, although together with p21, p16 has been considered as a key regulator of cellular senescence, its expression is induced by oncogenic Ras expression or oxidative stress, but not by telomere shortening⁵ or DNA damage. In some instances, p16 induction is subsequent to G₁ arrest induced by p21.⁶

The results advanced in the paper by Chien et al. (this issue) add another complexity to the p16 story. By exploring p16 ectopic expression experiments in several p16-deficient cancer cell lines (MCF7, U2OS, U87) the authors suggest that this CKI might have additional roles than that of an exclusive G₁ cell cycle regulator. Indeed, rather than to induce an immediate G₁ arrest, forced p16 expression resulted in a marked delay of S phase progression, entailing lengthening of the cell cycle duration prior to the ultimate cell cycle exit in G₁. This delay is not induced by the p16G101W mutant, which is unable to inhibit Cdk4/6 kinase activity. In light of the widely accepted G₁-specific role of Cdk4/6, the authors sought for other possible p16 targets responsible for this delay. While their results excluded an action of Cdk7 (a component of Cdk-activating kinase complex), suggested earlier to be a p16 target,⁷ they argue that S phase lengthening might be due to a decrease of Cdk1 (and Cdk2) protein levels and kinase activities. However, the mechanisms whereby p16 down-regulates Cdk1/2 are not understood, partly because of inconclusive immunoblot data. One possibility is that, by inhibiting Cdk4/6-dependent pRb phosphorylation, p16 blocks expression of post-G₁ E2F1 targets, as it is the case for p21.⁸ This is also consistent with the early data showing that, contrasting its presumed exclusive G₁ role, Cdk4 is active throughout the cell cycle.⁹ The only caveat is that p16 over-expression does not appear to reduce Cdk1/2 mRNAs, but this issue should be more thoroughly addressed. On the other hand, it is possible that by phosphorylating other proteins, Cdk4 (or Cdk6) might fine-tune post-G₁ cell cycle progression, a role uncovered here by p16 overexpression. It would be interesting to know whether and how, in this experimental setting, p16 over-expression affects cyclin D-Cdk4/6 complexes in all cell lines tested. Thus, while many clues regarding a possible post-G₁ role of p16 are still missing, this paper raises intriguing questions regarding a role of Cdk4/6 all around the cell cycle.

**References**

1. Gil J, et al. Nat Rev Mol Cell Biol 2006; 7:667-77.
2. Sherr CJ, et al. Genes Dev 1999; 13:1501-12.
3. Harbour JW, et al. Cell 1999; 98:859-69.
4. Pei XH, et al. Oncogene 2005; 24:2787-95.
5. Herbig U, et al. Mol Cell 2004; 14:501-13.
6. Robles SJ, et al. Oncogene 1998; 16:1113-23.
7. Nishiwaki E, et al. Mol Cell Biol 2000; 20:7726-34.
8. Jackson MW, et al. J Cell Sci 2005; 118:1821-32.
9. Matsushime H, et al. Mol Cell Biol 1994; 14:2066-76.
