
Journal of Receptors and Signal Transduction, 2010; 30(6): 403-409

REVIEW ARTICLE

# Transcriptional regulation of energy metabolism in the liver

Keiko Hirota${}^{1,2}$, and Akiyoshi Fukamizu${}^{1,2}$

${}^{1}$Center for Tsukuba Advanced Research Alliance, and ${}^{2}$Graduate School of Life and Environmental Sciences, University of Tsukuba, Tsukuba, Ibaraki, Japan

---

## Abstract

Living organisms maintain energy homeostasis by constantly adjusting internal metabolic activities in response to nutritional states. Energy metabolism is regulated by the quality and quantity of the enzymes that catalyze metabolic reactions. Recruitment and regulation of enzymes responsible for transcriptional control, among others, play an important role in this process. Located downstream from intracellular signaling cascades, transcription factors receive information signals from multiple sources. Their primary function is to integrate and interpret this information in terms of transcriptional output. It was recently suggested that signal content is converted *via* post-transcriptional modifications of the transcription factors. Many studies have shown that multiple signaling pathways converge on single transcription factors. This review discusses the post-translational modifications of transcription factors involved in the regulation of glucose metabolism, as well as the signaling networks in which they play a role.

## Keywords: Transcription; gluconeogenesis; metabolism; modification; insulin; HNF-4

---

## Introduction

The human body regulates changes in energy balance resulting from fasting, food intake, and other everyday life events and activities. It maintains energy homeostasis by controlling metabolic reactions based on interactions and coordination among a myriad of neurological and endocrine signaling pathways at the cellular and tissue levels. Remarkable progress has been made in the research of energy metabolism, along with rapid advancement in basic studies of metabolic syndrome, which is thought to develop from defective energy balance control.

Energy metabolism must proceed in the right direction at an appropriate rate. Unlike many reversible chemical reactions that occur *in vitro*, metabolic reactions in the human body move in one direction only. Most biological processes involve a chain of multiple enzyme-catalyzed reactions. The products of one reaction serve as the starting material of the next, thereby enabling unidirectional metabolic mass flow. In addition, the biological metabolic processes comprise the steps catalyzed by regulatory enzymes, which determine the overall rate of metabolism. Consequently, insight into the regulation and activation of regulatory enzymes is crucial for deepening our understanding of the mechanisms underlying energy metabolism.

Typical modulation of enzyme functions involves two dimensions, namely, qualitative, and quantitative. Qualitative changes by protein phosphorylation, allosteric regulation, and other alterations occur in a small amount of time (i.e., seconds or minutes). On the other hand, quantitative adjustment of enzyme activities takes place over a relatively long period of time *via* transcription. Accumulating evidence indicates that transmitted information regarding the energy status ultimately influences the activities of transcription factors, which in turn convert extracellular information into corresponding transcription output levels (which control the expression of enzymes) in order to achieve internal energy balance. The following sections provide the latest information on

---

Address For Correspondence: Akiyoshi Fukamizu, Ph.D, Center for Tsukuba Advanced Research Alliance, Graduate School of Life and Environmental Sciences, University of Tsukuba, Tennoudai 1-1-1, Tsukuba, Ibaraki 305-8577, Japan. E-mail: akif@tara.tsukuba.ac.jp

(Received 04 July 2010; accepted 17 July 2010)

ISSN 1079-9893 print/ISSN 1532-4281 online © 2010 Informa Healthcare USA, Inc.
DOI: 10.3109/10799893.2010.509730
http://www.informahealthcare.com/rst

the molecular mechanisms underlying the transcriptional regulation of carbohydrate metabolism.

### Regulation of glucose metabolism under fasting and feeding conditions

One of the body's most strictly controlled homeostatic mechanisms is that which keeps the blood glucose level within a narrow range. This function is primarily regulated by two pancreatic endocrine hormones, insulin, and glucagon, which act on their target tissues to adjust glucose metabolism. Insulin secretion is enhanced by high levels of blood glucose, as observed in the postprandial state, and a large proportion of glucose uptake is stored as glycogen or fat in the liver or adipose tissues. On the other hand, glucagon is secreted under low blood glucose conditions, as observed during fasting, which promotes catabolism to provide tissues with a source of fuel by way of several steps. First, glucagon acts on the liver to convert stored glycogen into glucose. However, the hepatic supply of glucose is short-lived, and glycogen stores are depleted within a day or two if fasting persists. Glucagon then activates adipocytes, causing them to release free fatty acids for consumption by peripheral tissues. However, red blood cells and nerve tissues rely on glucose as their sole source of energy, and as a result cannot take advantage of this fatty acid release. Therefore, during long-term fasting in which glycogen stores are exhausted, hepatic gluconeogenesis is activated, whereby glucose is synthesized from glycerols released by adipose tissue, muscle amino acids, and the lactate produced by red blood cells (Figure 1).

Glucose, an extremely important metabolic pathway in the regulation of blood glucose levels under fasting conditions, is activated by glucagon and suppressed by insulin. Hepatic gluconeogenesis is known to occur at an increased rate in association with insulin resistance, a major component of type-2 diabetes mellitus. Thus, gluconeogenesis has been the subject of much attention and medical research.

The rate of gluconeogenesis is modulated by the levels of rate-limiting enzymes such as glucose-6-phosphatase (G6Pase) and phosphoenolpyruvate carboxykinase (PEPCK). Hormonal stimuli (insulin, glucagon) ultimately impact transcription factors via signal transduction pathways, and the transmitted information subsequently affects the number of transcripts encoding such rate-limiting enzymes. Studies have been published regarding the ways in which hormones and other biological messengers communicate signals to transcription factors. Many signals are transferred to transcription factors in the form of post-translational modifications, such as phosphorylation, ubiquitination, and acetylation, and thereby alter transcription factor activity. A considerable number of transcription factors have been shown to undergo multiple post-translational modifications. Extensive research is underway to explore the regulatory mechanisms that underlie multiple protein modifications.

This article deals with the mechanisms by which post-transcriptional modifications of transcription factors affect transcriptional output. However, several cases have been identified in which PEPCK and other regulatory enzymes themselves underwent post-transcriptional modification to alter their activity (1). Future research on the modification of regulatory enzymes

![Metabolic pathways under fasting conditions](attachment:metabolic_pathways.png)

Figure 1. Metabolic pathways under fasting conditions. To meet the energy requirements of peripheral tissues during fasting, glycogen is degraded and free fatty acids are released from adipocytes. Prolonged fasting activates gluconeogenesis in the liver, which transforms lactate, glycerols, and amino acids into glucose. In the liver, free fatty acids are converted into ketone bodies, which are then transferred into neurons and muscles as a source of energy.

will provide new insight into the process of energy metabolism.

Regulatory mechanisms governing transcription of the rate-limiting enzymes in gluconeogenesis

Glucagon is known to induce a signaling cascade that mediates gluconeogenesis by enhancing transcription of rate-limiting enzymes. Under fasting conditions, glucagon is released from α cells in the pancreatic islets of Langerhans. It then phosphorylates cyclic adenosine monophosphate (cAMP) response element binding protein (CREB) in the liver through the cAMP/PKA signaling network. Phosphorylated CREB activates transcription of gluconeogenic genes to promote glucose production by interacting with CREB-binding protein (CBP)/p300, CREB-regulated transcription coactivator 2 (CRTC2), and other coactivators (Figure 2).

Transcriptional regulation of gluconeogenic rate-limiting enzymes

In addition to activating transcription of gluconeogenic rate-limiting enzymes, phosphorylated CREB induces the expression of peroxisome proliferator-activated receptor γ coactivator-1α (PGC-1α) by binding to its promoter (2). PGC-1α was found to perform an important function as a transcriptional coactivator for gluconeogenic rate-limiting enzymes (3). As this protein does not bind to DNA directly, but rather serves as a transcriptional coactivator, scientists have focused on identifying the DNA-binding transcriptional factor that interacts with PGC-1α. In a study using hepatocyte-specific knockout mice with a deficiency of hepatocyte nuclear factor-4 (HNF-4), Rhee et al. (2003) observed that the expression of gluconeogenic rate-limiting enzymes was suppressed under fasting conditions, and that primary hepatocytes obtained from the knockout mice did not increase the levels of gluconeogenic rate-limiting enzymes when infected with PGC-1α-expressing adenovirus (4). These findings indicated that HNF-4 was the DNA-binding transcriptional factor required for PGC-1α to regulate the transcription of rate-limiting enzymes in gluconeogenesis.

PGC-1α expression is thus induced by glucagon. In association with HNF-4 it stimulates the transcription of rate-limiting enzymes in gluconeogenesis. It is also known to regulate gluconeogenesis by undergoing post-translational modification.

Rodgers et al. (2005) found that PGC-1α was regulated by nicotinamide adenine dinucleotide (NAD)-dependent deacetylase SIRT1, and deacetylation of PGC-1α in the liver regulated gluconeogenic and glycolytic gene expression during fasting (5). More specifically, they demonstrated the following: [1] the amount of SIRT1 protein was elevated during fasting; [2] SIRT1 bound to and deacetylated PGC-1α in a NAD-dependent manner; [3] acetylation of PGC-1α promoted by knockdown of SIRT1 suppressed its concerted transcriptional activity with HNF-4; and [4] intracellular overexpression of PGC-1α activated gluconeogenic enzymes and suppressed glycolytic enzymes via SIRT1. The following year, this group used overexpression cell culture models to show that PGC-1α was directly acetylated by the histone acetyltransferase GCN5, and showed that acetylation of PGC-1α contributed to suppression of gluconeogenic enzymes and glucose production (6). These findings indicated that cells regulate gluconeogenesis through acetylation and deacetylation of PGC-1α.

SIRT1 has been found to function as an NAD-dependent deacetylase. The control of gluconeogenesis, therefore, involves modulation of PGC-1α transcriptional activity that requires the presence of the metabolite concerned with intracellular energy production (NAD), in addition to the hormone (glucagon)-induced modulation of PGC-1α expression identified previously. This illustrates a case in which multiple types of signaling pathways converge on one specific transcription factor. Of note is the finding that PGC-1α, formerly known as an 'activator' of gluconeogenic key enzyme expression, also served as a 'suppressor' of glycogenolysis. Future work addressing the roles of PGC-1α will shed more light on the regulation of conflicting metabolic processes.

CRTC2 and signal transduction pathways

In 2003, CRTC2 has been cloned (7). Subsequent studies found that CRTC2 underwent a wide variety of post-translational modifications to mediate critical transcriptional regulation of the rate-limiting enzymes of gluconeogenesis.

A series of studies conducted by the members of Montminy's laboratory revealed that modifiers of CRTC2 functions included phosphorylation, glycosylation, acetylation, and ubiquitination. Recently, phosphorylation of CRTC2 was found to take place at Ser171 in the presence of 5′AMP-activated protein kinase (AMPK), which was highly sensitive to change in intracellular AMP:ATP ratio, and phosphorylated CRTC2 were translocated into the cytoplasm (8). Subsequent research reported that CRTC2 was O-glycosylated at Ser171 under high glucose conditions. The O-glycosylation competes with phosphorylation of the same site; CRTC2 O-glycosylation was suggested as a possible factor contributing to the development of glucose intolerance that blocks CRTC2 phosphorylation and nuclear export (9). It has been reported that CRTC2 was

Fasting

Glucagon  
↓  
PKA  
↓  
CREB → CREB-P  
↓  
CRTE2  
↓  
CREB-P  
↓  
SIK1  

Feedback loop  
AMP/ATP: ↑  
↓  
AMPK  
↓  
14-3-3  
↓  
CRTC2  
Translocation  

PGC1  
↓  
NAD⁺/NADH: ↑  
↓  
GCN5  
↓  
SIRT1  
↓  
PGC1  
↓  
HNF4  
↓  
Foxo1  
↓  
CBP  
↓  
SIRT1  
↓  
Foxo1  
↓  
CRTC2  
↓  
CREB  
↓  
Gluconeogenesis  

Feeding  

COP1  
↓  
Ub  
↓  
CRTC2  
↓  
SIK2  
↓  
Insulin  
↓  
PKB/Akt  
↓  
Ub  
↓  
Foxo1  
↓  
PKCl/λ  
↓  
CBP  

Translocation and degradation  
Translocation and degradation  

Figure 2. Cascade reactions involved with transcription of gluconeogenetic rate-limiting enzymes. Glucagon stimulates phosphorylation of cyclic adenosine monophosphate (cAMP) response element binding protein (CREB) by protein kinase A (PKA). Phosphorylated CREB (CREB-P) activates proliferator-activated receptor γ coactivator 1α (PGC-1α) transcription in concert with CRTC2. The elevated level of PGC-1α activates transcription of gluconeogenetic rate-limiting enzymes via hepatocyte nuclear factor 4 (HNF-4). On the other hand, AMPK, salt-inducible kinase 1 (SIK1), salt-inducible kinase 2 (SIK2), and constitutively photomorphogenic 1 (COP1) negatively regulate CRTC2 through post-translational modifications. Sirtuin 1 (SIRT1) activates gluconeogenetic transcription through deacetylation of PGC-1α, which deactivates the transcriptional activity when acetylated by general control non-derepressible 5 (GCN5). PKA, protein kinase A; PKB, protein kinase B.

subject to modifications at Lys628 in response to changing nutritional needs. In the early phase of fasting, CRTC2 stimulated gluconeogenetic gene activation in association with p300 while the latter promoted CRTC2 acetylation (10). It was also shown that CRTC2 was deacetylated owing to a fast-induced increase in SIRT1 level (10). This indicated that gluconeogenetic gene expression was rigorously controlled by the change in CRTC2 modification that occurred in response to prolonged fasting. Moreover, CRTC2 was found to be ubiquitinated at Lys628 by COP1 in an insulin-dependent manner, and the ubiquitination induced CRTC2 degradation (10).

Overall, these results demonstrate that CRTC2 undergoes various post-translational modifications controlled by multiple input signaling pathways that involve hormones (e.g., insulin, glucagon) secreted depending on the glucose level, intracellular energy sensor systems (AMP/ATP and NAD/NADH ratios), and blood glucose levels. In other words, multiple levels of modulating signals—intercellular (e.g., hormones), intracellular (e.g., AMP/ATP and NAD/NADH ratios), and molecular (e.g., glucose level)—converge on CRTC2. Future study will benefit from exploring how multiple pathways of different kinds are ultimately integrated into single type of transcription output signal that controls energy metabolism.

Whereas the studies we have reviewed above addressed the effects of CRTC2 on the induction of gluconeogenetic rate-limiting enzymes, Wang et al. (2009) reported the linkage between CRTC2 and endoplasmic reticulum (ER) stress, which was known to promote insulin resistance (11). They found that [1] ER stress led to dephosphorylation and nuclear translocation of CRTC2; [2] upon entry to the nucleus, CRTC2 activated the expression of ER quality control genes through association with activating transcription factor 6 (ATF6); and [3] ATF6 competed with CREB for the CRTC2 N-terminus, thereby attenuating gluconeogenetic rate-limiting enzyme expression under ER stress conditions. These findings demonstrated the roles of CRTC2 not only in coordinating nutritional and metabolic cues with gluconeogenetic gene expression,

but also in balancing the crosstalk between ER stress and metabolism.

# Regulation of glucose metabolism by CBP phosphorylation

Several articles have reported on post-translational modifications of CBP, which is another coactivator of CREB. In a study using phosphomutant CBP-S436A knock-in mice (12), insulin signaling failed to cause phosphorylation of CBP at Ser 436 as well as diminished CREB activity. The mutant CBP knock-in mice produced chronically increased levels of hepatic glucose. The finding that CBP-CREB interaction was diminished in a phosphorylation-dependent manner was supported by the fact that the serine phosphorylation site was located near the CREB-interaction domain of CBP. A more recent study (13) showed that CBP was phosphorylated by insulin-induced atypical protein kinase C, and CBP phosphorylation dissociated the CREB-CBP-CRTC2 complex, leading to diminished induction of gluconeogenesis rate-limiting enzymes. Interestingly, in a study on the effect of the antidiabetic agent metformin on insulin resistance in mice fed high-fat diets, the metformin decreased blood glucose in wild-type insulin-resistant mice, whereas this was not the case with CBP-S436A mutant mice. These results suggested that CBP phosphorylation at Ser 436 was the fundamental mechanism underlying metformin's therapeutic effect.

# Insulin-dependent transcriptional regulation involving Foxo1 and HNF-4

We have briefly reviewed some of the latest reports exploring how hormonal and other signals modulate transcriptional activities via post-translational modifications of cofactors such as CRTC2, CBP, and PGC-1α. Next, we review DNA-binding transcription factors that serve as 'anchors' for these cofactors.

HNF-4, as already explained, activates the transcription of the gluconeogenic rate-limiting enzyme G6Pase (4, 14, 15). However, it is also reported to promote the expression of the glycolytic enzyme glucokinase (GK) (16). Gluconeogenic and glycolytic pathways work in opposite directions; gene transcription of the gluconeogenic rate-limiting enzyme G6Pase is enhanced in the absence of insulin (i.e., in the fasting state), whereas that of the glycolytic rate-limiting enzyme GK is elevated in the presence of insulin (i.e., under fed conditions) (17–22). It may seem quite confusing at first glance that the same transcription factor (HNF-4) promotes processes working in opposite directions. The regulatory mechanisms for these processes long remained unclear. In 2008, it has been showed that Foxo1 exerted opposite actions on HNF-4 in a gene context-dependent manner to effectively mediate the expression of two genes with opposing actions (23). In G6Pase gene regulation, Foxo1 synergizes with HNF-4 to stimulate transcription, whereas it represses HNF-4-potentiated transcription of the GK gene. These effects are abrogated by insulin-induced phosphorylation and cytoplasmic translocation of Foxo1. To investigate the gene-specific effects of Foxo1, the authors focused on the presence or absence of the transcription factor-binding sequences in the gene promoter regions. The G6Pase gene promoter contains binding elements for both Foxo1 and HNF-4. The GK promoter, however, includes binding sequences for HNF-4, but not for Foxo1. These facts suggest the possibility that the presence or absence of the Foxo1-binding sequences influence Foxo1 activity in a promoter context-dependent manner. A reporter assay using HepG2 cells showed that deletion of the Foxo1-binding elements from the G6Pase gene promoter changed the nature of Foxo1 from activator to suppressor. This suggests that the combination or arrangement of the HNF-4- and Foxo1-binding sites determines how Foxo1-mediated gene transcription proceeds. This study dealt with the effects of Foxo1 on the human GK gene; however, other research groups have shown that Foxo1 directly binds to the rat GK promoter (24). This may imply species-specific differences in the behavior and functions of Foxo1.

Next, the authors investigated the role of the G6Pase and GK promoter HNF-4-binding sequences in mice using an *in vivo* luciferase (Luc) reporter assay. First, mice were injected with the wild-type GK and G6Pase promoter constructs, and the Luc activity exhibited a pattern similar to that of the endogenous G6Pase and GK under fasting and fed conditions. This result validated this assay system as appropriate for real-time analysis of glucose-dependent gene expression. Then, cells were transfected *in vivo* with reporter gene constructs carrying G6Pase-Luc or GK-Luc with mutated Foxo1- and HNF-4-binding sites. Deletion of the HNF-4-binding sequences from the GK promoter significantly reduced the GK gene response to fasting and refeeding (Figure 3A, left). When combined with *in vitro* results, this observation suggested a novel aspect of the glucose metabolic regulation involving HNF-4, namely, it indicated that HNF-4-potentiated fasting- and refeeding-responsive gene expressions were regulated by the insulin-responsive transcription factor Foxo1. With G6Pase, deletion of HNF-4-binding sequences from the promoter region resulted in the absence of a response to fasting and refeeding, as was the case with deletion of Foxo1-binding sites reported earlier (Figure 3A, right). Worthy of note is that deletion of the Foxo1-binding elements upstream of the G6Pase gene tended to cause inversion of the fasting and refeeding responses, although the activity was high under fasting.

408 K. Hirota & A. Fukamizu

**Figure 3. Regulation of glucose metabolism by hepatocyte nuclear factor 4 (HNF-4) and Forkhead box O1 (Foxo1)**  
(A) *In vivo* luciferase assay demonstrating the role of HNF-4 and Foxo1-binding sequences. The HNF-4 binding sequences in the promoter regions of glucokinase (GK) and glucose-6-phosphatase (G6Pase) genes play a crucial role for inducing fasting- and feeding-responsive gene expression.  
(B) Schematic drawing of insulin-dependent control. During fasting, Foxo1 synergizes with HNF-4 to turn on G6Pase gene transcription while turning off GK gene transcription to promote gluconeogenesis. When insulin signaling is activated due to a shift in nutritional state, Foxo1 is phosphorylated and translocated from the nucleus to the cytoplasm, which represses gluconeogenesis and activates glycolysis. HBE, HNF4-binding element; IRS insulin response sequence.

conditions and low in fed conditions in mice with the Foxo1-binding sequences left intact. This attracts particular attention because it suggests the possibility that the effect of Foxo1 on HNF-4 changed from that of 'enhancer' to 'suppressor' along with the structural shift from the G6Pase-type factor-binding site pattern to the GK-type.

These experiments contributed to unveiling the roles of Foxo1 and HNF-4 in the regulation of glucose metabolism (Figure 3B). They shed light on the molecular mechanisms whereby these molecules are concomitantly involved in regulating the opposing pathways of gluconeogenesis and glycogenolysis. Several studies have reported that, as in the combination of HNF-4 and Foxo1, two molecules can exert the two opposing effects of gene activation and suppression in a gene context-dependent fashion (25, 26). However, little has been known about how context influences system shifts from activation to suppression or vice versa. Further exploration of the G6Pase and GK gene expression mechanisms will benefit research on genetic programs

regulating glucose metabolism and other metabolic processes.

### Conclusion

Living organisms detect shifts in the balance between energy expenditure and storage at the cellular, tissue, and whole body levels, and communicate activating signals triggered by these shifts to relevant target cells and tissues. The targets make appropriate metabolic changes to maintain energy homeostasis. Transcription factors act downstream of intracellular signaling cascades; they integrate and interpret transmitted information in terms of transcriptional output through post-translational modifications. Transcription factors, which take advantage of post-translational modifications, constitute diverse yet interrelated networks that are part of complex metabolic systems. Careful study of the individual regulatory factors of transcriptional regulation networks and gene regulation mechanisms will provide an overview of the entire

metabolic regulatory system. Such an overview will help generate useful ideas for disease prevention and control.

# Acknowledgements

The authors thank the Fukamizu laboratory members for their helpful discussion and encouragement.

# Declaration of interest

This work was supported by a grant-in-aid for Scientific Research on Priority Areas and a grant-in-aid for Young Scientists (B) from the Ministry of Education, Culture, Sports, Science and Technology of Japan.

# References

1. Zhao S, Xu W, Jiang W, Yu W, Lin Y, Zhang T, Yao J, Zhou L, Zeng Y, Li H, Li Y, Shi J, An W, Hancock SM, He F, Qin L, Chin J, Yang P, Chen X, Lei Q, Xiong Y, Guan KL. Regulation of cellular metabolism by protein lysine acetylation. Science 2010, 327, 1000-1004.
2. Herzig S, Long F, Jhala US, Hedrick S, Quinn R, Bauer A, Rudolph D, Schutz G, Yoon C, Puigserver P, Spiegelman B, Montminy M. CREB regulates hepatic gluconeogenesis through the coactivator PGC-1. Nature 2001, 413, 179-183.
3. Yoon JC, Puigserver P, Chen G, Donovan J, Wu Z, Rhee J, Adelmant G, Stafford J, Kahn CR, Granner DK, Newgard CB, Spiegelman BM. Control of hepatic gluconeogenesis through the transcriptional coactivator PGC-1. Nature 2001, 413, 131-138.
4. Rhee J, Inoue Y, Yoon JC, Puigserver P, Fan M, Gonzalez FJ, Spiegelman BM. Regulation of hepatic fasting response by PPARgamma coactivator-1alpha (PGC-1): requirement for hepatocyte nuclear factor 4 alpha in gluconeogenesis. Proc Natl Acad Sci USA 2003, 100, 4012-4017.
5. Rodgers JT, Lerin C, Haas W, Gygi SP, Spiegelman BM, Puigserver P. Nutrient control of glucose homeostasis through a complex of PGC-1alpha and SIRT1. Nature 2005, 434, 113-118.
6. Lerin C, Rodgers JT, Kalume DE, Kim SH, Pandey A, Puigserver P. GCN5 acetyltransferase complex controls glucose metabolism through transcriptional repression of PGC-1alpha. Cell Metab 2006, 3, 429-438.
7. Iourgenko V, Zhang W, Mickanin C, Daly I, Jiang C, Hexham JM, Orth AP, Miraglia L, Meltzer J, Garza D, Chirn GW, McWhinnie E, Cohen D, Skelton J, Terry R, Yu Y, Bodian D, Buxton FP, Zhu J, Song C, Labow MA. Identification of a family of cAMP response element-binding protein coactivators by genome-scale functional analysis in mammalian cells. Proc Natl Acad Sci USA 2003, 100, 12147-12152.
8. Koo SH, Flechner L, Qi L, Zhang X, Screaton RA, Jeffries S, Hedrick S, Xu W, Boussouar F, Brindle P, Takemori H, Montminy M. The CREB coactivator TORC2 is a key regulator of fasting glucose metabolism. Nature 2005, 437, 1109-1111.
9. Dentin R, Hedrick S, Xie J, Yates J 3rd, Montminy M. Hepatic glucose sensing via the CREB coactivator CRTC2. Science 2008, 319, 1402-1405.
10. Liu Y, Dentin R, Chen D, Hedrick S, Ravnskjaer K, Schenk S, Milne J, Meyers DJ, Cole P, Yates J 3rd, Olefsky J, Guarente L, Montminy M. A fasting inducible switch modulates gluconeogenesis via activator/coactivator exchange. Nature 2008, 456, 269-273.
11. Wang Y, Vera L, Fischer WH, Montminy M. The CREB coactivator CRTC2 links hepatic ER stress and fasting gluconeogenesis. Nature 2009, 460, 534-537.
12. Zhou XY, Shibusawa N, Naik K, Porras D, Temple K, Ou H, Kaihara K, Roe MW, Brady MJ, Wondisford FE. Insulin regulation of hepatic gluconeogenesis through phosphorylation of CREB-binding protein. Nat Med 2004, 10, 633-637.
13. He L, Sabet A, Djedjos S, Miller R, Sun X, Hussain MA, Radovick S, Wondisford FE. Metformin and insulin suppress hepatic gluconeogenesis through phosphorylation of CREB binding protein. Cell 2009, 137, 635-646.
14. Rajas F, Gautier A, Bady I, Montano S, Mithieux G. Polyunsaturated fatty acyl coenzyme A suppress the glucose-6-phosphatase promoter activity by modulating the DNA binding of hepatocyte nuclear factor 4 alpha. J Biol Chem 2002, 277, 15736-15744.
15. Hirota K, Aoyama H, Fukamizu A. Mutation analysis of HNF-4 binding sites in the human glucose-6-phosphatase promoter. Int J Mol Med 2005, 15, 487-490.
16. Roth U, Jungermann K, Kietzmann T. Activation of glucokinase gene expression by hepatic nuclear factor 4 alpha in primary hepatocytes. Biochem J 2002, 365, 223-228.
17. Iynedjian PB, Ucla C, Mach B. Molecular cloning of glucokinase cDNA. Developmental and dietary regulation of glucokinase mRNA in rat liver. J Biol Chem 1987, 262, 6032-6038.
18. Magnuson MA, Andreone TL, Printz RL, Koch S, Granner DK. Rat glucokinase gene: structure and regulation by insulin. Proc Natl Acad Sci USA 1989, 86, 4838-4842.
19. Iynedjian PB, Gjinovci A, Renold AE. Stimulation by insulin of glucokinase gene transcription in liver of diabetic rats. J Biol Chem 1988, 263, 740-744.
20. Iynedjian PB, Jotterand D, Nouspikel T, Asfari M, Pilot PR. Transcriptional induction of glucokinase gene by insulin in cultured liver cells and its repression by the glucagon-cAMP system. J Biol Chem 1989, 264, 21824-21829.
21. Streeter RS, Svitek CA, Chapman S, Greenbaum LE, Taub R, O'Brien RM. A multicomponent insulin response sequence mediates a strong repression of mouse glucose-6-phosphatase gene transcription by insulin. J Biol Chem 1997, 272, 11698-11701.
22. Streeter RS, Eaton EM, Ebert DH, Chapman SC, Svitek CA, O'Brien RM. Hepatocyte nuclear factor-1 acts as an accessory factor to enhance the inhibitory action of insulin on mouse glucose-6-phosphatase gene transcription. Proc Natl Acad Sci USA 1998, 95, 9208-9213.
23. Hirota K, Sakamaki J, Ishida J, Shimamoto Y, Nishihara S, Kodama N, Ohta K, Yamamoto M, Tanimoto K, Fukamizu A. A combination of HNF-4 and Foxo1 is required for reciprocal transcriptional regulation of glucokinase and glucose-6-phosphatase genes in response to fasting and feeding. J Biol Chem 2008, 283, 32432-32441.
24. Ganjam GK, Dimova EY, Unterman TG, Kietzmann T. FoxO1 and HNF-4 are involved in regulation of hepatic glucokinase gene expression by resveratrol. J Biol Chem 2009, 284, 30783-30797.
25. Scully KM, Rosenfeld MG. Pituitary development: regulatory codes in mammalian organogenesis. Science 2002, 295, 2231-2235.
26. Rausa FM, Tan Y, Costa RH. Association between hepatocyte nuclear factor 6 (HNF-6) and FoxA2 DNA binding domains stimulates FoxA2 transcriptional activity but inhibits HNF-6 DNA binding. Mol Cell Biol 2003, 23, 437-449.
