
THE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 285, NO. 47, pp. 36352–36360, November 19, 2010  
© 2010 by The American Society for Biochemistry and Molecular Biology, Inc. Printed in the U.S.A.

# Functional Redundancy of Steroid C<sub>26</sub>-monooxygenase Activity in *Mycobacterium tuberculosis* Revealed by Biochemical and Genetic Analyses*

Received for publication, July 3, 2010, and in revised form, September 10, 2010 Published, JBC Papers in Press, September 15, 2010, DOI 10.1074/jbc.M110.161117

Jonathan B. Johnston<sup>1,2</sup>, Hugues Ouellet<sup>1</sup>, and Paul R. Ortiz de Montellano<sup>3</sup>

From the Department of Pharmaceutical Chemistry, University of California, San Francisco, California 94158

---

One challenge to the development of new antitubercular drugs is the existence of multiple virulent strains that differ genetically. We and others have recently demonstrated that CYP125A1 is a steroid C<sub>26</sub>-monooxygenase that plays a key role in cholesterol catabolism in *Mycobacterium tuberculosis* CDC1551 but, unexpectedly, not in the *M. tuberculosis* H37Rv strain. This discrepancy suggests that the H37Rv strain possesses compensatory activities. Here, we examined the roles in cholesterol metabolism of two other cytochrome P450 enzymes, CYP124A1 and CYP142A1. *In vitro* analysis, including comparisons of the binding affinities and catalytic efficiencies, demonstrated that CYP142A1, but not CYP124A1, can support the growth of H37Rv cells on cholesterol in the absence of *cyp125A1*. All three enzymes can oxidize the sterol side chain to the carboxylic acid state by sequential oxidation to the alcohol, aldehyde, and acid. Interestingly, CYP125A1 generates oxidized sterols of the (25S)-26-hydroxy configuration, whereas the opposite 25R stereochemistry is obtained with CYP124A1 and CYP142A1. Western blot analysis indicated that CYP124A1 was not detectably expressed in either the H37Rv or CDC1551 strains, whereas CYP142A1 was found in H37Rv but not CDC1551. Genetic complementation of CDC1551 Δ*cyp125A1* cells with the *cyp124A1* or *cyp142A1* genes revealed that the latter can fully rescue the growth defect on cholesterol, whereas cells overexpressing CYP124A1 grow poorly and accumulate cholest-4-en-3-one. Our data clearly establish a functional redundancy in the essential C<sub>26</sub>-monooxygenase activity of *M. tuberculosis* and validate CYP125A1 and CYP142A1 as possible drug targets.

more than two million lives are claimed annually due to active *M. tuberculosis* infections (1). Among first world countries, the spread of *M. tuberculosis* has been kept mostly under control, but there has been a resurgence in developing countries in large part due to the emergence of multidrug-resistant bacterial strains that make the traditional frontline antibiotics less effective (2). Efforts continue on many fronts to understand this complex pathogen with a focus on identification of new drug targets.

To proliferate within the macrophages, *M. tuberculosis* cells undergo a shift in metabolism from using carbohydrates to primarily utilizing host lipids (3–5). Sequencing of the *M. tuberculosis* genome revealed at least 250 genes predicted to be involved in lipid metabolism (6). A cholesterol catabolism cluster of 51 genes was recently identified in the genome of the *M. tuberculosis*-related actinomycete *Rhodococcus jostii* RHA1 (7). This region corresponds to the 82-gene cluster of *M. tuberculosis* that includes genes for proteins involved in both cholesterol uptake and degradation, suggesting that *M. tuberculosis* can utilize cholesterol for growth during infection. The steroid transporter Mce4 was shown to be important for the uptake of cholesterol in both *R. jostii* RHA1 and *M. tuberculosis* (8, 9), and there is a growing body of evidence that *M. tuberculosis* can utilize cholesterol as a source of carbon and energy by degrading both the rings and the side chain during infection (10–15). The *igr* operon (*Rv3540c–Rv3545c*), which is part of this cluster and was initially identified by screening of transposon mutants that failed to grow within primary macrophages (16, 17), has been shown to be required for cholesterol metabolism (18). The six genes of the *igr* operon, consisting of a cytochrome P450 (*cyp125A1*), two acyl-CoA dehydrogenases (*fadE28* and *fadE29*), two conserved hypothetical proteins (*Rv3541-2c*), and a putative lipid carrier protein (*ltp2*), are predicted to play roles in lipid metabolism.

The participation of *cyp125A1* as a steroid C<sub>26</sub>-monooxygenase in cholesterol side chain degradation in mycobacteria and rhodococci was recently demonstrated by gene inactivation (11, 14, 19). Capyk *et al.* (14) made use of the *M. tuberculosis* H37Rv virulent strain, and Ouellet *et al.* (11) used the clinically relevant isolate CDC1551. The essential nature of the *cyp125A1* gene for growth on cholesterol and alleviation of the toxicity of the cholesterol-derived metabolite cholest-4-en-3-one was observed with *M. tuberculosis* CDC1551 but not with the H37Rv Δ*cyp125A1* mutant when similarly grown, suggesting that H37Rv possesses compensatory enzymes that allow it to cope with the absence of CYP125A1. Capyk and co-workers

---

*Mycobacterium tuberculosis* is the causative agent of human tubercular infection (tuberculosis) that, even today, poses a great threat to global human health. More than two billion people (a third of the world population) are infected latently with the bacterium, and of those individuals, ~10% will develop active tuberculosis infections during their lifetime. Currently,

---

* This work was supported, in whole or in part, by National Institutes of Health Grant RO1 AI74824 (to P. R. O. M.).

* The on-line version of this article (available at http://www.jbc.org) contains supplemental Table S1 and Figs. S1.1–S1.10 and S2.1–S2.9.

1 Both authors contributed equally to this work.

2 Recipient of a Heiser Postdoctoral Fellowship for Research in Leprosy and Tuberculosis sponsored by the New York Community Trust.

3 To whom correspondence should be addressed: Genentech Hall, N572D, 600 16th St., San Francisco, CA 94158-2280. Tel.: 415-476-2903; E-mail: ortiz@cgl.ucsf.edu.

(14) proposed that CYP130A1 and/or CYP141A1 might fulfill this role. However, very recent work in our laboratory has shown that CYP130A1 and CYP141A1 do not oxidize cholesterol or cholest-4-en-3-one (11).

Here, we examine whether two other P450<sup>4</sup> enzymes, CYP124A1 and CYP142A1, can substitute for CYP125A1 in degradation of the cholesterol side chain in *M. tuberculosis*. Comparison of the genome sequences of *M. tuberculosis* CDC1551 and H37Rv reveal key differences in cytochrome P450 loci. In CDC1551, a 639-bp deletion is found upstream of the *cyp142* (*Rv3518c*) gene. The deletion encompasses the promoter region and the sequence encoding the first 18 amino acids of the protein. The *cyp124A1* genes of H37Rv and CDC1551 only differ by two amino acids, G25D and Y75N, respectively. In contrast, the *cyp125A1* genes are identical in both strains. CYP124A1 is adjacent to a three-gene operon encoding a sulfotransferase implicated in the biosynthesis of a sulfated menaquinone called S881 (20–22). CYP124A1 was selected as a possible surrogate for CYP125A1 both because of its close sequence and structural homology to CYP125A1 and because it has been shown to catalyze the regioselective oxidation of the terminal methyl in methyl-branched lipids (21). CYP142A1 is part of a cluster of genes involved in lipid degradation that has been postulated to play a role in the metabolism of host lipids, including cholesterol (23).

We report here the results of a broad analysis of these two enzymes employing both comparative *in vitro* characterization of the three P450 enzymes and cellular analysis of their expression. This analysis included determination of the absolute stereochemistry of cholesterol hydroxylation by the three enzymes. Our results clarify the differential phenotype when grown on cholesterol of the H37Rv and CDC1551 Δ*cyp125A1* strains and highlight the basis for the strain differences that are relevant to the targeting of cholesterol degradation as a therapeutic target.

### EXPERIMENTAL PROCEDURES

#### Materials—All solvents and reagents were of the highest purity commercially available unless noted otherwise. Methyl-β-cyclodextrin, glucose 6-phosphate, Tween 20, glucose-6-phosphate dehydrogenase (*Saccharomyces cerevisiae*), ferredoxin (*Spinacea oleracea*), ferredoxin NADP<sup>+</sup> reductase (*S. oleracea*), cholesterol oxidase (*Streptomyces* sp.), cholesterol, cholest-4-en-3-one, ergosterol, and tyloxapol were purchased from Sigma. (25R)-26-Hydroxycholesterol and (25S)-26-hydroxycholesterol were purchased from Research Plus (Barnegat, NJ). Cholest-4-en-3-one-(25S)-carboxylic acid was obtained from Avanti Polar Lipids (Alabaster, AL). Polyclonal antibodies to purified *M. tuberculosis* CYP124A1, CYP125A1, and CYP142A1 were raised in rabbits (Pacific Immunology, Ramona, CA).

#### Cloning, Expression, and Purification—The CYP124A1, CYP125A1, and CYP142A1 enzymes from *M. tuberculosis* H37Rv were cloned, expressed in *Escherichia coli*, and purified as described previously (21, 24). The protein preparations used

in this study exhibited typical P450 CO-deoxy difference spectrum (data not shown). For CYP125A1 only, a small amount (20%) of P420 was observed. The concentration of P450 for the three enzymes was determined from difference spectra using the extinction coefficient ε<sub>450–490</sub> = 91,000 mM<sup>−1</sup> cm<sup>−1</sup> (25). The total heme b content was quantified by the pyridine hemochrome method (26).

#### Spectrophotometric Binding Assays—All ligand binding assays were performed by spectrophotometric titration in 50 mM potassium phosphate buffer (pH 7.4) containing 0.1 mM EDTA using a Cary UV-visible scanning spectrophotometer (Varian). Steroid stock solutions were first prepared at a concentration of 5 mm in warmed 10% (w/v) methyl-β-cyclodextrin (MβCD) dissolved in water, sonicated for 10 min, and finally diluted at 0.5 or 1.0 mM in 2% MβCD. To account for the absorbance of the tested compounds, 1 ml of protein (1.0 μM) in buffer was placed in the first chamber of two split cuvettes, and 1 ml of buffer was placed in the second chamber. After background scanning, equal volumes (0.5–1.0 μl) of ligand solution were titrated into both the second chamber of the reference cuvette containing only buffer and the first chamber containing the protein. The same volume of 2% MβCD was added into the alternate chambers to correct for solvent effects. Difference spectra were recorded from 300 to 500 nm. To determine the *K*<sub>D</sub> values, titration data points were fitted to the quadratic equation using Kaleidagraph (Synergy). In Equation 1, *A*<sub>obs</sub> is the absorption shift determined at any ligand concentration; *A*<sub>max</sub> is the maximal absorption shift obtained at saturation; *K*<sub>D</sub> is the apparent dissociation constant for the inhibitor-enzyme complex; [E<sub>t</sub>] is the total enzyme concentration used; [S] is the ligand concentration.

\[
A_{\text{obs}} = A_{\text{max}} \left( ([S] + [E_t] + K_D) - \left( ([S] + [E_t] + K_D)^2 - (4[S][E_t])^{0.5} \right) / 2[E_t] \right)
\]

#### Enzyme Assays—Enzymatic activities of CYP124A1, CYP125A1, and CYP142A1 were determined as described (11), except that MβCD was included at a final concentration of 0.45% (w/v) MβCD. To identify metabolites, reaction mixtures were incubated up to 2 h. The products were extracted three times with hexanes and derivatized by resuspending the dried organic extracts in ~70 μl of *N,O*-bis(trimethylsilyl)trifluoroacetamide (Pierce) for 2 h at room temperature. The products and control reaction mixtures were analyzed by GC-MS on an Agilent HP 6850 gas chromatography system using a DB-5MS analytical column (30 m × 0.25 mm inner diameter, 0.25-μm film thickness) equipped with an Agilent 5973 mass selective detector operating in electron impact mode at 70 eV. A 3-μl portion of each reaction was injected and subsequently separated by holding the column at 70°C for 1 min, then increasing the temperature by 10°C/min to 300°C, and finally holding at 300°C for 20 min.

#### Steady-state Kinetic Constants—To determine steady-state kinetic parameters of the three P450 isoforms toward cholesterol and cholest-4-en-3-one, the reaction conditions were optimized for the formation of the respective 26-hydroxy derivatives. The reactions were terminated after 20 min by adding an equal volume of acetonitrile, 0.1% formic acid containing 50 μM

<sup>4</sup>The abbreviations used are: P450, cytochrome P450 enzyme; CYP, cytochrome P450 enzyme; MβCD, methyl-β-cyclodextrin.

Roles of P450 Enzymes in Mycobacterial Cholesterol Utilization

ergosterol as an internal standard, mixed vigorously, and centrifuged for 5 min at 3,000 rpm at 4°C to remove any precipitated products. 100-μl aliquots of each reaction were analyzed using an Agilent Series 1200 HPLC system fitted with a reverse phase C8 column (Varian Polaris C8-A, 5 μm, 250 × 4.6 mm) and a diode array detector. Using an isocratic mobile phase system consisting of 30% H₂O (0.1% formic acid) and 70% acetonitrile (0.1% formic acid) at a flow rate of 1.0 ml/min, the products were monitored with a diode array detector set at 240 nm. Enzymatic reactions with cholest-4-en-3-one as the substrate were analyzed directly following quenching and centrifugation. For the detection of cholesterol-derived metabolites, an additional step was required to convert the 3-OH group to a 3-oxo-Δ4-enone structure. The UV-visible absorbance of the α,β-unsaturated ketone of cholest-4-en-3-one facilitates detection of the metabolites. The sterols were extracted three times with hexanes, dried under a stream of nitrogen gas, and resuspended in 50 mM KPᵢ (pH 7.6) along with 2.5 units of cholesterol oxidase and 1 unit of catalase (bovine). In all cases, the steroid enzyme incubation mixtures were compared with control reactions, and steady-state kinetic parameters were calculated by fitting to an appropriate kinetic model (21).

Stereochemistry of Hydroxylation—Assays were performed as described above but were instead quenched with an equal volume of 1 N HCl. The products were extracted with 2 volumes of ethyl acetate containing progesterone as an internal standard. The organic phase was collected and evaporated by flushing with argon. The dried metabolites were solubilized with acetonitrile and finally mixed with water containing 0.1% (v/v) formic acid. Each commercial 26-hydroxycholesterol diastereomer was individually converted and analyzed as its respective cholest-4-en-3-one counterpart by incubation at room temperature for 1 h with cholesterol oxidase (2.5 units) and bovine liver catalase (10 μg/ml) prior to quenching and extraction. The standards and enzymatic reactions were analyzed by HPLC with a Chiralcel-OD column (250 × 4.6-mm inner diameter analytical column, Chiral Technologies Europe, France) using an isocratic mobile phase consisting of 15% H₂O (0.1% formic acid) and 85% acetonitrile (0.1% formic acid) at a flow rate of 1 ml/min.

M. tuberculosis Strains, Media, and Culture Conditions—Wild-type CDC1551 strain was obtained from Colorado State University. All experiments with M. tuberculosis cells were performed in a biosafety level-3 (BSL-3) facility. The CDC1551 Δcyp125A1 mutant strain was created by homologous recombination using specialized transducing phages, as described previously (11). M. tuberculosis cells were cultured in 7H9 medium supplemented with 10% of oleic acid-albumin-dextrose complex, 0.2% glycerol, and 0.05% Tween 80. Hygromycin and Zeocin (50 mg/liter) were used where necessary. For growth on cholesterol, log-phase cells grown in 7H9 medium were used to inoculate a defined minimal medium (1 g/liter KH₂PO₄, 2.5 g/liter Na₂HPO₄, 0.5 g/liter asparagine, 50 mg/liter ferric ammonium citrate, 10 mg/liter MgSO₄·7H₂O, 0.5 mg/liter CaCl₂, and 0.1 mg/liter ZnSO₄ at an initial A₆₀₀ of 0.05 and 0.1 mM cholesterol. A stock solution of cholesterol (100 mM) was prepared in tyloxapol/ethanol (1:1) and warmed at 65°C for 30 min before addition.

Complementation of the Cyp125A1 Disrupted Mutant—The coding sequences of cyp124A1, cyp125A1, and cyp142A1 were amplified from M. tuberculosis H37Rv genomic DNA by using the forward and reverse primers listed in supplemental Table S1. The DNA sequences were confirmed by amplifying the coding region using PCR and then ligating into pCR-BluntII-TOPO vector (Invitrogen), and the NdeI-XbaI digested fragments were then subcloned downstream of the constitutive hsp60 promoter into a derivative of the pMV361 vector that confers resistance to Zeocin. The three constructs were electroporated into Δcyp125A1 cells, and transformants were selected at 37°C on 7H10 plates containing 50 mg/ml Zeocin.

Detection of Endogenous Expression of CYP124A1, CYP125A1, and CYP142A1—Western blots were carried out to detect either the native or overexpressed P450 isoforms (CYP124A1, CYP125A1, or CYP142A1) from the whole cell lysate of various M. tuberculosis strains. Cells were precultured in 7H9 medium, washed in the same medium lacking glycerol, synchronized to an A₆₀₀ of 0.2, and grown for an additional 24 h in the presence of 0.1 mM cholesterol or 0.1% (v/v) glycerol. Cells (50 ml) were harvested by centrifugation, washed twice with 25 ml of 15% isopropyl alcohol, and kept frozen at −80°C until used. The pellets were resuspended on ice with 1 ml of 10 mM Tris-Cl (pH 7.0) containing 1 mM PMSF, 1.5% SDS, and cells were disrupted by bead beating. Soluble extracts were obtained by centrifugation at 13,000 × g for 10 min at 4°C. Total protein concentration was determined using the micro-BCA assay (Pierce). Western blot analysis was performed according to standard protocols. Three identical membranes were probed with polyclonal antibodies against CYP125A1, CYP124A1, and CYP142A1. We also stained the membranes after the immunodetection procedure to ensure equal loading in the different wells.

Analysis of in Vivo Buildup of Cholest-4-en-3-one—To analyze the buildup of cholest-4-en-3-one in the various M. tuberculosis strains, cells were pre-cultured in 7H9 medium, washed in the same medium lacking glycerol, synchronized to an A₆₀₀ of 0.2, and grown for an additional 36 h in the presence of 0.1 mM cholesterol. Cells (25 ml) were harvested by centrifugation, washed twice with 25 ml of 15% isopropyl alcohol in saline, and resuspended in saline at a similar A₆₀₀ for all strains. One-milliliter samples were extracted with 5 ml of chloroform/methanol (17:1) for 24 h. The lower organic phase containing apolar lipids was collected. The lipid extracts were evaporated under a stream of argon and resuspended in 1.5 ml of methanol. Tween 80 was removed from the lipid solutions by precipitation using a cobalt/thiocyanate solution and back-extraction with hexane, according to a published procedure (27). Finally, the lipid extracts were dissolved in 1 ml of chloroform/methanol (2:1) and analyzed by HPLC at 240 nm as described previously (11).

RESULTS

CYP124A1 and CYP142A1 Specifically Bind and Catalyze the C26-Monooxygenation of Cholest-4-en-3-one and Cholesterol—The expression, purification, and general characterization of M. tuberculosis CYP124A, CYP125A1, and CYP142A1 were reported previously (11, 14, 21, 24, 28). The steroid binding and catalytic activities of CYP125A1 toward

cholesterol and cholest-4-en-3-one were also reported earlier (11, 14, 28). The specific tight binding and steroid side chain oxidizing activities of CYP124A1 and CYP142A1 reported here identify redundant catalytic activities specific for the methyl-branched side chain of cholesterol and cholest-4-en-3-one. Table 1 summarizes the apparent dissociation ($K_D$) and steady-state kinetic constants ($K_m$ and $k_{cat}$) for the oxidation of cholesterol and cholest-4-en-3-one by CYP124A1, CYP125A1, and CYP142A1.

As shown in Fig. 1A, CYP124A1 displayed the spectral properties of a ferric P450 with most of the heme iron

TABLE 1  
Steady-state kinetic constants and binding affinities of  
*M. tuberculosis* CYP124A1, CYP125A1, and CYP142A1 towards  
cholesterol and cholest-4-en-3-one  

| Enzyme          | $K_{D,app}$ | $K_{m,app}$ | $k_{cat,app}$ | $K_{D,app}/K_{m,app}$ |
|------------------|--------------|--------------|----------------|------------------------|
|                  | $nM$         | $\mu M$      | $min^{-1}$     | $\mu M^{-1} min^{-1}$  |
| Cholesterol      |              |              |                |                        |
| CYP124A1        | 11.6 ± 5     | 1.5 ± 0.2    | 0.129          |                        |
| CYP125A1        | 107 ± 64$^a$ | 10.7 ± 4     | 28 ± 3         | 2.6                    |
| CYP142A1        | 18.4 ± 4.9   | 7.7 ± 2      | 16.7 ± 1.3     | 2.2                    |
| Cholest-4-en-3-one |              |              |                |                        |
| CYP124A1        | 1056 ± 30    | 11.7 ± 1.3   | 0.56           |                        |
| CYP125A1        | 1180 ± 110$^a$ | 20.8 ± 1.6   | 175 ± 8        | 8.4                    |
| CYP142A1        | 114 ± 17     | 11.8 ± 1.5   | 84 ± 5         | 7.1                    |

$^a$ Values are from Ref. 11.

FIGURE 1. Binding of cholest-4-en-3-one and cholesterol to recombinant CYP124A1 and CYP142A1. The absolute Soret region absorption spectra of CYP124A1 (A) and CYP142A1 (C) in their resting (thick line), cholest-4-en-3-one (dashed line), cholesterol-bound (thin lines) forms are presented. In the case of CYP124A1, a partial reversal of the spectral change was recorded every 3 min for 12 min, at which point it stabilized. The concentration dependence of ligand binding was deduced from the difference absorption changes obtained from the titration of CYP124A1 (B) and CYP142A1 (D) with increasing concentrations of cholest-4-en-3-one. Representative sets of the difference spectra that were recorded are shown in the insets.

Roles of P450 Enzymes in Mycobacterial Cholesterol Utilization

Stereochemistry of Oxidation—Hydroxylation of the ω-position of the cholesterol and cholest-4-en-3-one side chains can occur at either of the two methyl groups, producing either the (25S)-or (25R)-26-hydroxy steroid derivatives. Because of the inherent chiral nature of enzymes, substrate binding within the active site should confer a stereochemical preference for hydroxylation. Based on the crystal structure of CYP124A1 bound to phytanic acid (21), CYP124A1 is predicted to produce the (25S)-26-hydroxy diastereomer. Similarly, the co-crystal structure of CYP125A1 bound with cholest-4-en-3-one in the active site predicts that the (25S)-26-hydroxy diastereomer should be favored relative to the (25R)-26-hydroxy steroid derivative (11). We cannot predict the stereochemical outcome of the CYP142A1 oxidation reaction without a co-crystal structure. We analyzed the purified (25S)-26-hydroxy cholest-4-en-3-one and (25R)-26-hydroxy cholest-4-en-3-one derivatives using a chiral HPLC column and compared these standards to enzymatic incubations of CYP124A1, CYP125A1, and CYP142A1 with cholest-4-en-3-one. We found that CYP125A1, as predicted, produces primarily the (25S)-26-hydroxycholest-4-en-3-one diastereomer (Fig. 3A), whereas both CYP124A1 and CYP142A1 generate the opposite (25R)-26-hydroxy diastereomer as the single or major product (Fig. 3, B and C, and Table 2).

Previously, our work with CYP125A1 demonstrated the formation of further side chain oxidation products, namely the carboxylic acid and its aldehyde precursor (11). The GC-MS traces presented in Fig. 2 also showed that CYP124A1 and CYP142A1 are capable of oxidizing the corresponding alcohols to the carboxylic acids. To further explore this phenomenon, we separately incubated each of the pure (25S)-and (25R)-26-hydroxycholest-4-en-3-one diastereomers with CYP124A1, CYP125A1, and CYP142A1 and analyzed the resulting products (supplemental Figs. S2.1–S2.9). From these experiments, it is clear that each of the P450 isoforms can oxidize either of the alcohol diastereomers to the corresponding acid. Both CYP125A1 and CYP142A1 catalyze the oxidation with equal preference, *in vitro*, whereas CYP124A1 oxidizes the (25R)-26-hydroxycholest-4-en-3-one diastereomer at a higher rate. Thus, both CYP124A1 and CYP142A1, like CYP125A1 (11), oxidize the cholest-4-en-3-one side chain fully to the carboxylic acid, a prerequisite for entry into the β-oxidation pathway. It should be noted that CYP124A1 also previously demonstrated the ability to sequentially oxidize the ω-position of geranylgeraniol to generate geranylgeranic acid (21).

Expression of Native Steroid C26-monooxygenase(s) in M. tuberculosis Cells—As a second step in identifying the compensatory steroid C26-monooxygenase activity lacking in CDC1551 Δ*cy*p125A1 cells, we examined the endogenous levels of CYP125A1, CYP124A1, and CYP142A1 and the inducibility of their expression by C27 steroids in both the CDC1551 and H37Rv strains. Polyclonal antibodies raised against each of the three proteins were used to detect the presence of these polypeptides by Western blot analysis (Fig. 4). Basal levels of CYP125A1 were detected in both glycerol-incubated CDC1551 and H37Rv, and those levels increased significantly after incubation with either cholesterol or cholest-4-en-3-one. This is in sharp contrast to CYP124A1, which is not expressed at detect-

FIGURE 2. Offset GC of total ion current versus time for cholest-4-en-3-one incubated with CYP142A1 (A) and CYP124A1 (B). S denotes the substrate cholest-4-en-3-one, whereas the products are marked P2 (aldehyde), P3 alcohol, and P1 acid. A scheme illustrating the reaction sequence is shown in A. Control reactions in which P450 enzyme was omitted are shown in the lower trace of each panel. The mass spectra of the substrate and product peaks are included in the supplemental material.

observed sequential oxidation of the 26-position of cholesterol to give the 26-carboxylic acid cholesterol derivative as a major product (supplemental Figs. S1–S10).

To determine the steady-state kinetic parameters (*K*~m~ and *k*~cat~) of CYP124A1, CYP125A1, and CYP142A1 toward the two steroids, the reaction conditions were optimized for the formation of a single product. Table 1 lists the steady-state kinetic constants for the oxidation of cholesterol and cholest-4-en-3-one by each of *M*.tuberculosis P450 isoforms. In general all the isoforms bind these two steroids with high affinity. However, clear differences are seen in the catalytic activities of the three P450 isoforms that can be attributed to differences in their catalytic efficiencies. Our data indicate that each of the enzymes oxidizes cholest-4-en-3-one with better catalytic efficiency than cholesterol and also that both CYP125A1 and CYP142A1 are more efficient catalysts for these biologically relevant steroids than CYP124A1. The second-order rate constants (*k*~cat~/ *K*~m~) provide parameters for comparison of the *M*. tuberculosis P450 isoforms with each other, and from the data in Table 1, it is clear that CYP125A1 is the most efficient catalyst. CYP125A1 exhibits ~20-and 15-fold better activity toward cholesterol and cholest-4-en-3-one, respectively, than CYP124A1. CYP125A1 is also more efficient than CYP142A1, which shows a slightly lower, but statistically significant (*R*² value of 0.9911), *k*~cat~/ *K*~m~ value.

Roles of P450 Enzymes in Mycobacterial Cholesterol Utilization

TABLE 2  
Stereochemistry of the 26-hydroxylated products  
The average values and standard deviations are based on nine independent determinations.

| Pure standard/reaction product                          | Relative retention time (r), 26-OH/progesterone |
|--------------------------------------------------------|-------------------------------------------------|
| (25R)-26-Hydroxycholest-4-en-3-one                      | 1.567 ± 0.002                                     |
| (25S)-26-Hydroxycholest-4-en-3-one                      | 1.551 ± 0.002                                     |
| Racemic mixture                                         | 1.560 ± 0.001                                     |
| CYP125A1                                                | 1.554 ± 0.001                                     |
| CYP142A1                                                | 1.568 ± 0.001                                     |
| CYP124A1                                                | 1.567 ± 0.001                                     |

CYP125A1  
CYP142A1  
CYP124A1  

FIGURE 4. Expression of CYP125A1, CYP124A1, and CYP142A1 in the M. tuberculosis CDC1551 and H37Rv strains. 7H9-grown cells were synchronized at A600 of 0.2 and were then incubated in the presence of 0.1% glycerol (v/v) or 0.1 mM cholesterol for 36 h before harvesting and total protein isolation. A Western blot analysis of total proteins (each lane contains 15 mg of protein extract) is shown. Detection was performed with polyclonal antibodies raised against each of the three P450 isoforms. Lanes 1, glycerol-grown CDC1551; lane 2, cholesterol-grown CDC1551; lane 3, cholest-4-en-3-one-grown CDC1551; lane 4, glycerol-grown H37Rv; lane 5, cholesterol-grown H37Rv; lane 6, cholest-4-en-3-one-grown H37Rv.

CYP125A1 in cholesterol degradation, at least under the experimental conditions examined.

Overexpression of CYP142A1 Completely Rescued the Growth Defect on Cholesterol—To determine whether any of the two other C26-monooxygenase activities can participate in the degradation of the cholesterol side chain in the absence of CYP125A1, we performed gain-of-function experiments by complementing the CDC1551Δcyp125A1 cells with cyp124A1 or cyp142A1. The constructs were integrated as a single copy into the chromosome to allow the expression of cyp124A1 or cyp142A1 under the control of the strong and constitutive promoter of the hsp60 gene. As controls, we also complemented CDC1551Δcyp125A1 with the cyp125A1 construct and the empty integrative vector. The expression levels of the three M. tuberculosis P450s in three independent clones were first examined by Western blot. As shown in Fig. 5C, the levels of CYP125A1 under the control of the hsp60 promoter are at least equivalent to those detected from wild-type CDC1551 and H37Rv cells grown in the presence of cholesterol. The expression levels of CYP124A1 and CYP142A1 not only can be detected easily but are much stronger than those displayed in the wild-type strains.

The growth of CDC1551, H37Rv, Δcyp125A1, and the complemented strains was tested in minimal media containing cholesterol as a sole source of carbon. As expected, Δcyp125A1 cells failed to grow in the presence of cholesterol, whereas all the other strains grew normally (Fig. 5A). Interestingly, the strain overexpressing CYP124A1 exhibited partial recovery, suggesting that this enzyme is less efficient in oxidizing cholesterol/cholest-4-en-3-one, consistent with our in vitro analysis. Alternatively, CYP124A1 may require specific redox partners and/or other accessory proteins and co-factors that are not present in sufficient amount under these experimental conditions. To further support the idea that CYP124A1 is not well suited as a

FIGURE 3. Representative HPLCs showing absorbance at 240 nm versus time of elution used to determine the stereochemistry of cytochrome P450-mediated hydroxylation. Comparison of the elution times of the individual diastereomer standards (25S)-26-hydroxycholest-4-en-3-one (thick dashed line) and (25R)-26-hydroxycholest-4-en-3-one (thick line) with the 26-hydroxylated products from CYP125A1 (A), CYP142A1 (B), and CYP124A1 (C) reactions (thin line). The purified diastereomers were converted into their cholest-4-en-3-one derivative and were then further processed similarly to the P450 reactions (see under "Experimental Procedures"). The relative retention time values were calculated relative to the internal standard progesterone and are listed in Table 2.

able levels in either the CDC1551orH37Rv strains. CYP142A1 was expressed at very low levels in glycerol-incubated cells, but it was induced slightly after exposure to steroids in the H37Rv strain, although, as expected, no CYP142A1 accumulation occurred in CDC1551. This result strongly suggests that CYP142A1, but not CYP124A1, participates along with

Roles of P450 Enzymes in Mycobacterial Cholesterol Utilization

A
![Graph A](graph_a.png)

B
![Graph B](graph_b.png)

C
1 2 3 4 5
CYP125A1
CYP124A1
CYP142A1

FIGURE 5. Overexpression of CYP142A1, but not CYP124A1, fully restores the growth of Δcyp125A1 cells on cholesterol. Representative growth curves (A) and HPLC analysis for the cellular accumulation of cholest-4-en-3-one (B) are shown for CDC155 (empty circle), H37Rv (empty square), CDC1551 Δcyp125A1 (empty diamond), Δcyp125A1/cyp125A1 (filled circle), Δcyp125A1/cyp124A1 (filled square), and Δcyp125A1/cyp142A1 (filled diamond) strains on minimal media containing cholesterol as the sole source of carbon. The growth was monitored by measuring the absorbance at 600 nm as a function of time. Some data points are not visible in the plot because they coincide with the baseline (C). Overexpression of CYP125A1, CYP124A1, and CYP142A1 in CDC1551 Δcyp125A1 cells was confirmed by Western blot analysis using antibodies raised against the three P450 isoforms. Lanes 1–3, total proteins extracted from three independent clones; lane 4, cholesterol-grown CDC1551; lane 5, cholesterol-grown H37Rv.

exclusively form the (25R)-26-hydroxy products. Under the experimental conditions used in this study, we detected expression of CYP142A1 in H37Rv strain, but not in CDC1551, whereas CYP124A1 protein was undetectable in extracts from both strains. Furthermore, a compensatory role in cholesterol side chain oxidation was demonstrated for CYP142A1 by complementing CDC1551 Δcyp125A1 cells.

During infection, *M. tuberculosis* is found in foamy macrophages that are rich in cholesterol (30, 31). Cholesterol has recently been identified as an important lipid for infection (9, 10, 12, 15, 18). The Mce4 transport system is required for cholesterol uptake into *M. tuberculosis* cells, and inactivation of the *mce4* locus results in an attenuated phenotype in a chronic mouse model of infection (9). There is also a growing body of evidence that *M. tuberculosis* can degrade cholesterol and utilize it as a source of carbon and energy (9–15). More specifically, we and others have provided biochemical and structural evidence that confirms the key role of the steroid C₂₆-monooxygenase CYP125A1 in initiation of the steroid side chain degradation process (11, 14, 19, 28). Carbon sources such as cholesterol provide two and three carbon atom units via β-oxidation that feed into the glyoxylate shunt pathway and serve as energy sources for the bacterium (32). The increase in propionyl-CoA from carbon sources such as the side chain of cholesterol enhance the production of sulfolipid-1 and phthiocerol dimycocerosate, two classes of virulence-related lipids, by indirectly increasing the pool of methylmalonyl-CoA (11, 33, 34). This metabolic strategy supports the bacterium during its residence in host macrophages and allows the harnessing of host cholesterol for both energy and as a source of carbon.

In a remarkable example of convergent evolution, mycobacteria and other actinomycetes have acquired the ability, like mammals (CYP27A1) (35) and *Caenorhabditis elegans* (DAF-9) (36), to oxidize the terminal position of a mammalian steroid, yet CYP125A1, CYP142A1, and CYP124A1 exhibit low sequence identity with these enzymes. Reconstitution of the enzymatic activity with surrogate redox proteins from spinach revealed that all three recombinant P450 isoforms indeed bind both cholest-4-en-3-one and cholesterol, efficiently hydroxylate both of them at C26, and then further oxidize the 26-hydroxy metabolites to their respective carboxylic acid derivatives, a prerequisite step for esterification with coenzyme A and β-oxidation. However, the *in vitro* enzyme specificities, as reflected in the $k_{cat}/K_m$ values across the enzyme and substrate series (Table 1), concur that CYP125A1 and CYP142A1 are the dominant P450 enzymes responsible for initiating steroid side chain degradation in *M. tuberculosis* and suggest that cholest-4-en-3-one rather than cholesterol is the most relevant *in vivo* substrate.

Understanding the stereochemical preferences of each of these three P450s might provide clues to the redundancy of the cholesterol utilization pathway in *M. tuberculosis*, as it is possible that one diastereomer is utilized as a source of carbon and energy, whereas the other may serve yet unknown functions. Based on proximity to the iron center, the co-crystal structures predict that CYP124A1 and CYP125A1 should catalyze oxidation to the (25S)-26-hydroxy product (11, 21). Using our reconstituted assay system, these predictions were partially confirmed.

DISCUSSION

Here, we examine the ability of two *M. tuberculosis* cytochrome P450 enzymes, CYP124A1 and CYP142A1, to compensate for loss of CYP125A1 as a steroid C₂₆-monooxygenase in two different clinical isolates, CDC1551 and H37Rv, using a combination of biochemical and genetic experiments. To better understand these redundant enzyme activities, we compared both the equilibrium binding ($K_d$) and enzyme specificity ($k_{cat}/K_m$) values and showed that of the three *M. tuberculosis* P450 isoforms, CYP125A1 is the most efficient catalyst, followed by CYP142A1 and CYP124A1. Determination of the stereochemistry of the oxidation reaction revealed that CYP125A1 generates the (25S)-26-hydroxy metabolites, as opposed to CYP142A1 and CYP124A1, which primarily or exclusively form the (25R)-26-hydroxy products.

firm, as we found that the hydroxylated product of CYP125A1 has the same retention time as (25S)-26-hydroxy cholesterol-4-en-3-one. CYP142A1 and CYP124A1, just like the mammalian CYP27A1 (35), exhibit the opposite stereochemical preference for the ω-position of the steroid side chain and produce the (25R)-26-hydroxy metabolite. In contrast, it was shown that *C. elegans* DAF-9 is a nonstereoselective cholest-4-en-3-one 26-hydroxylase (36). The observation that CYP124A1 produces oxysterols of the opposite stereoconfiguration than predicted by the co-crystal structure with phytanic acid likely stems from subtle differences in the binding orientations of fatty acids and sterols within the active site.

Bacterial degradation of branched fatty acids proceeds through an analogous route in mammalian cells. Because only the CoA ester with the 2-methyl group in the S configuration can be degraded by β-oxidation, the R stereoisomer needs to be first converted enzymatically to the S configuration by an α-methylacyl racemase. Experimental evidence confirming this inference has been obtained with a *Tn5*-derived mutant of *Mycobacterium sp.* strain P101 that is defective in the α-methylacyl-CoA activity and can grow on (S)-methylbutyric acid but not on the racemic mixture (37). These results suggest that the oxidation of methyl-branched lipids is inhibited by the R stereoisomer. Three *mcr* homologues, *mcr*, *far*, and *Rv3727*, are present in the *M. tuberculosis* genome, although the physiological role of each gene product has not been elucidated (38), suggesting that (R)-2-methylacyl-CoAs are also degraded through the same route in *M. tuberculosis*. As shown here, CDC1551Δ*cyp125A1* cells in which CYP142A1 was overexpressed grow at wild-type rates, indicating that *M. tuberculosis* is capable of dealing with the C27-steroid side chain of the opposite stereochemical configuration. This argues that an α-methylacyl-CoA racemase is present when cholesterol is used as a sole source of carbon and energy and is not rate-limiting when CYP142A1 provides the only entry of the cholesterol side chain into the β-oxidation pathway. It remains to be determined whether the (25R)-metabolite has functions that differ from those of the (25S)-product when more than one carbon source is available.

All three of the P450 enzymes examined here are able to oxidize both diastereomers of the initial hydroxylated sterol even though they stereospecifically produce only one of the alcohol diastereomers. Thus, the active sites of these three enzymes are able to bind not only the hydroxysterol that they normally produce, but also the sterol with the opposite absolute stereochemistry at the hydroxylated chain terminus. Preferential oxidation of the hydroxylated carbon rather than the unhydroxylated side chain methyl group presumably reflects the lower bond energy of a C–H bond adjacent to an oxygen atom compared with that of a primary methyl group (39, 40). Similar to CYP125A1, both CYP142A1 and CYP124A1 produce the carboxylic acid as the major product. In addition, ω-oxidation of the methyl-branched steroid side chain by CYP124A1 is consistent with our previous finding that this enzyme oxidizes chains in which one methyl of a terminal isopropyl function binds in an active site cavity and positions the other methyl of the isopropyl terminus for oxidation (21).

The differences in the expression profiles of the three P450 enzymes in the wild-type strains indicate that, despite the fact that all three *M. tuberculosis* strains studied here can oxidize cholesterol, only CYP142A1, but not CYP124A1, can substitute for CYP125A1 when it is absent. Only CYP142A1 is expressed and only in H37Rv, consistent with the presence of a 639-bp deletion upstream of the *cyp142* gene in CDC1551, a deletion that is expected to produce a truncated, unstable, and undetectable form of CYP142A1. Thus, CYP142A1 provides a functionally redundant steroid side chain oxidizing activity that can fully compensate for loss of CYP125A1, as in the H37Rv Δ*cyp125* strain. This explains the lack of a growth defect when this knock-out strain is grown on cholesterol (14). We failed to detect CYP124A1 expression *in vivo* under any of our conditions. Furthermore, the partial rescue of the growth phenotype on cholesterol when CYP124A1 is overexpressed is in good agreement with the low *in vitro* catalytic efficiency of CYP124A1 toward these C27 steroids. Altogether, these experiments provide independent and persuasive evidence that CYP124A1 is not involved in cholesterol utilization. Elucidation of the physiological function of CYP124A1 is currently underway in this laboratory.

This study presents new insights into the cholesterol side chain degradation in mycobacterial species. Further characterization of the essential early steps of cholesterol catabolism and their regulation is required to better understand the role of this process in *M. tuberculosis* pathogenesis. The physiological importance of the cholesterol-degrading pathway raises the possibility of targeting CYP125A1 and CYP142A1 for therapeutic purposes.

Acknowledgments—We thank Prof. Jeffery S. Cox and Eric D. Chow for helpful discussions.

---

**REFERENCES**

1. World Health Organization (2009) World Health Organization Facts Sheets on Tuberculosis, http://www.who.int/mediacentre/factsheet/fs104/en/index.html
2. Russell, D. G., Barry, C. E., 3rd, and Flynn, J. L. (2010) Science 328, 852–856
3. Boshoff, H. I., and Barry, C. E., 3rd. (2005) Nat. Med. 11, 599–600
4. Boshoff, H. I., and Barry, C. E., 3rd. (2005) Nat. Rev. Microbiol. 3, 70–80
5. Schnappinger, D., Ehrt, S., Voskuil, M. I., Liu, Y., Mangan, J. A., Monahan, I. M., Dolganov, G., Efron, B., Butcher, P. D., Nathan, C., and Schoolnik, G. K. (2003) J. Exp. Med. 198, 693–704
6. Cole, S. T., Brosch, R., Parkhill, J., Garnier, T., Churcher, C., Harris, D., Gordon, S. V., Eiglmeier, K., Gas, S., Barry, C. E., 3rd, Tekaia, F., Badcock, K., Basham, D., Brown, D., Chillingworth, T., Connor, R., Davies, R., Devlin, K., Feltwell, T., Gentles, S., Hamlin, N., Holroyd, S., Hornsby, T., Jagels, K., Krogh, A., McLean, J., Moule, S., Murphy, L., Oliver, K., Osborne, J., Quail, M. A., Rajandream, M. A., Rogers, J., Rutter, S., Seeger, K., Skelton, J., Squares, R., Squares, S., Sulston, J. E., Taylor, K., Whitehead, S., and Barrell, B. G. (1998) Nature 393, 537–544
7. Van der Geize, R., Yam, K., Heuser, T., Wilbrink, M. H., Hara, H., Anderson, M. C., Sim, E., Dijkhuizen, L., Davies, J. E., Mohn, W. W., and Eltis, L. D. (2007) Proc. Natl. Acad. Sci. U.S.A. 104, 1947–1952
8. Mohn, W. W., van der Geize, R., Stewart, G. R., Okamoto, S., Liu, J., Dijkhuizen, L., and Eltis, L. D. (2008) J. Biol. Chem. 283, 35368–35374
9. Pandey, A. K., and Sassetti, C. M. (2008) Proc. Natl. Acad. Sci. U.S.A. 105, 4376–4380
10. Nesbitt, N. M., Yang, X., Fontán, P., Kolesnikova, I., Smith, I., Sampson,

Roles of P450 Enzymes in Mycobacterial Cholesterol Utilization

N. S., and Dubnau, E. (2010) *Infect. Immun.* 78, 275–282

11. Ouellet, H., Guan, S., Johnston, J. B., Chow, E. D., Kells, P. M., Burlingame, A. L., Cox, J. S., Podust, L. M., and Ortiz de Montellano, P. R. (2010) *Mol. Microbiol.* 77, 730–742

12. Hu, Y., van der Geize, R., Besra, G. S., Gurcha, S. S., Liu, A., Rohde, M., Singh, M., and Coates, A. (2010) *Mol. Microbiol.* 75, 107–121

13. Brzostek, A., Pawelczyk, J., Rumijowska-Galewicz, A., Dziadek, B., and Dziadek, J. (2009) *J. Bacteriol.* 191, 6584–6591

14. Capyk, J. K., Kalscheuer, R., Stewart, G. R., Liu, J., Kwon, H., Zhao, R., Okamoto, S., Jacobs, W. R., Jr., Eltis, L. D., and Mohn, W. W. (2009) *J. Biol. Chem.* 284, 35534–35542

15. Yam, K. C., D’Angelo, I., Kalscheuer, R., Zhu, H., Wang, J. X., Snieckus, V., Ly, L. H., Converse, P. J., Jacobs, W. R., Jr., Strynadka, N., and Eltis, L. D. (2009) *PLoS Pathog.* 5, e1000344

16. Chang, J. C., Harik, N. S., Liao, R. P., and Sherman, D. R. (2007) *J. Infect. Dis.* 196, 788–795

17. Rengarajan, J., Bloom, B. R., and Rubin, E. J. (2005) *Proc. Natl. Acad. Sci. U.S.A.* 102, 8327–8332

18. Chang, J. C., Miner, M. D., Pandey, A. K., Gill, W. P., Harik, N. S., Sassetti, C. M., and Sherman, D. R. (2009) *J. Bacteriol.* 191, 5232–5239

19. Rosłoniec, K. Z., Wilbrink, M. H., Capyk, J. K., Mohn, W. W., Ostendorf, M., van der Geize, R., Dijkhuizen, L., and Eltis, L. D. (2009) *Mol. Microbiol.* 74, 1031–1043

20. Holsclaw, C. M., Sogi, K. M., Gilmore, S. A., Schelle, M. W., Leavell, M. D., Bertozzi, C. R., and Leary, J. A. (2008) *ACS Chem. Biol.* 3, 619–624

21. Johnston, J. B., Kells, P. M., Podust, L. M., and Ortiz de Montellano, P. R. (2009) *Proc. Natl. Acad. Sci. U.S.A.* 106, 20687–20692

22. Mougous, J. D., Senaratne, R. H., Petzold, C. J., Jain, M., Lee, D. H., Schelle, M. W., Leavell, M. D., Cox, J. S., Leary, J. A., Riley, L. W., and Bertozzi, C. R. (2006) *Proc. Natl. Acad. Sci. U.S.A.* 103, 4258–4263

23. Ouellet, H., Johnston, J. B., and Ortiz de Montellano, P. R. (2010) *Arch. Biochem. Biophys.* 493, 82–95

24. Ouellet, H., Lang, J., Couture, M., and Ortiz de Montellano, P. R. (2009)

25. Omura, T., and Sato, R. (1964) *J. Biol. Chem.* 239, 2379–2385

26. Appleby, C. A. (1969) *Biochim. Biophys. Acta* 172, 88–105

27. Shen, C. F., Hawari, J., and Kamen, A. (2004) *J. Chromatogr. B* 810, 119–127

28. McLean, K. J., Lafite, P., Levy, C., Cheesman, M. R., Mast, N., Pikuleva, I. A., Leys, D., and Munro, A. W. (2009) *J. Biol. Chem.* 284, 35524–35533

29. Jefcoate, C. R. (1978) *Methods Enzymol.* 52, 258–279

30. Russell, D. G., Cardona, P. J., Kim, M. J., Allain, S., and Altare, F. (2009) *Nat. Immunol.* 10, 943–948

31. Peyron, P., Vaubourgeix, J., Poquet, Y., Levillain, F., Botanch, C., Bardou, F., Daffé, M., Emile, J. F., Marchou, B., Cardona, P. J., de Chastellier, C., and Altare, F. (2008) *Plos Pathog.* 4, e1000204

32. Lorenz, M. C., and Fink, G. R. (2002) *Eukaryot. Cell* 1, 657–662

33. Jain, M., Petzold, C. J., Schelle, M. W., Leavell, M. D., Mougous, J. D., Bertozzi, C. R., Leary, J. A., and Cox, J. S. (2007) *Proc. Natl. Acad. Sci. U.S.A.* 104, 5133–5138

34. Yang, X., Nesbitt, N. M., Dubnau, E., Smith, I., and Sampson, N. S. (2009) *Biochemistry* 48, 3819–3821

35. Norlin, M., von Bahr, S., Bjorkhem, I., and Wikvall, K. (2003) *J. Lipid Res.* 44, 1515–1522

36. Motola, D. L., Cummins, C. L., Rottiers, V., Sharma, K. K., Li, T., Li, Y., Suino-Powell, K., Xu, H. E., Auchus, R. J., Antebi, A., and Mangelsdorf, D. J. (2006) *Cell* 124, 1209–1223

37. Sakai, Y., Takahashi, H., Wakasa, Y., Kotani, T., Yurimoto, H., Miyachi, N., Van Veldhoven, P. P., and Kato, N. (2004) *J. Bacteriol.* 186, 7214–7220

38. Bhaumik, P., Kursula, P., Ratas, V., Conzelmann, E., Hiltunen, J. K., Schmitz, W., and Wierenga, R. K. (2003) *Acta Crystallogr. D Biol. Crystallogr.* 59, 353–355

39. de Visser, S. P., Kumar, D., Cohen, S., Shacham, R., and Shaik, S. (2004) *J. Am. Chem. Soc.* 126, 8362–8363

40. Olsen, L., Rydberg, P., Rod, T. H., and Ryde, U. (2006) *J. Med. Chem.* 49, 6489–6499
