
Available online at www.sciencedirect.com

**ELSEVIER**

**ScienceDirect**

**Network news: prime time for systems biology of the plant circadian clock**

C Robertson McClung${}^{1}$ and Rodrigo A Gutiérrez${}^{2}$

Whole-transcriptome analyses have established that the plant circadian clock regulates virtually every plant biological process and most prominently hormonal and stress response pathways. Systems biology efforts have successfully modeled the plant central clock machinery and an iterative process of model refinement and experimental validation has contributed significantly to the current view of the central clock machinery. The challenge now is to connect this central clock to the output pathways for understanding how the plant circadian clock contributes to plant growth and fitness in a changing environment. Undoubtedly, systems approaches will be needed to integrate and model the vastly increased volume of experimental data in order to extract meaningful biological information. Thus, we have entered an era of systems modeling, experimental testing, and refinement. This approach, coupled with advances from the genetic and biochemical analyses of clock function, is accelerating our progress towards a comprehensive understanding of the plant circadian clock network.

**Addresses**

${}^{1}$ Department of Biological Sciences, Dartmouth College, Hanover, New Hampshire, 03755-3576, USA

${}^{2}$ Departamento de Genética Molecular y Microbiología, Pontificia Universidad Católica de Chile, Santiago 8331010, Chile

Corresponding author: McClung, C Robertson  
(c.robertson.mcclung@dartmouth.edu)

---

**Current Opinion in Genetics & Development 2010, 20:588–598**

This review comes from a themed issue on  
Genetics of system biology  
Edited by Jeff Hasty, Alex Hoffmann and Susan Golden  

0959-437X/$ – see front matter  
© 2010 Elsevier Ltd. All rights reserved.  

DOI **10.1016/j.gde.2010.08.010**

---

**Introduction**

Circadian clocks are nearly ubiquitous endogenous timers that play critical roles in the temporal organization of biological activities and in the coordination of those activities with daily environmental cycles. The accumulation of large quantities of experimental data is proving a challenge to all fields of biology. The growing need for interpretation, integration, and modeling these data is incentivizing biologists to move away from individual molecules towards a systems view of biological process. Circadian biology is no exception. Systems biology is a relatively new field in the

**biological sciences which aims to integrate the existing knowledge about biological components, build a model of the system *as a whole* and extract the unifying organizational principles that explain the form and function of living organisms [1]. System level models of biological processes that explain observed behaviors should be predictive and be used to derive testable hypotheses. Experimental work can then validate these hypotheses or provide new ways to refine the model. Iterating through the cycle of modeling, testing, and refinement is a hallmark of research in systems biology. Ultimately, understanding systems structure and dynamic principles will allow the design of new ones with desirable properties.**

**Circadian clocks consist of interlocked feedback loops**

Detailed analysis of plant circadian clocks reveals an intricate network of molecular components that is responsible for rhythmic behaviors. At the heart of the clock lie multiple interlocked negative feedback loops, each loop involving transcriptional activation and repression (Figure 1). The central loop consists of two MYB transcription factors, CIRCADIAN CLOCK ASSOCIATED 1 (CCA1) and LATE ELONGATED HYPOCOTYL (LHY), which repress expression of a pseudo-response regulator (PRR) gene, TIMING OF CAB EXPRESSION 1 (TOC1). TOC1 is recruited to the *CCA1* and *LHY* promoters and activates their expression. Three additional PRRs (PRR5, 7, and 9) repress their activators, *CCA1* and *LHY*, to form a second interlocked ‘morning loop’. TOC1 forms a third ‘evening loop’ through repression of a hypothetical component ‘Y’, that includes GIGANTEA (GI), a positive regulator of TOC1. In this work we emphasize new studies, including systems biology approaches, which enhance our understanding of circadian network architecture, including advances in oscillator function and in how the clock orchestrates plant biological activities. These studies add to the number and complexity of oscillator loops and improve our understanding of the multiple levels of regulation that contribute to clock function. Comprehensive reviews of the plant circadian system should be consulted for a more complete vision of the plant clock [2–4].

**In the beginning**

When does circadian clock function begin? A number of years ago it was inferred that a clock is running from the time of imbibition in etiolated seedlings, based on circadian gating of the light-mediated acute induction of clock-regulated *CATALASE2* expression [5]. Consistent with this, the clock gates the promotion of germination by

Network news: prime time for systems biology of the plant circadian clock McClung and Gutiérrez 589

Figure 1

![Diagram](https://i.imgur.com/yourimageurl.png)

A simplified working model of the Arabidopsis circadian clock. The Arabidopsis circadian clock consists of a series of interconnected feedback loops. In the central loop, CCA1 and LHY negatively regulate TOC1 through direct promoter binding. TOC1 is a positive regulator of CCA1 and LHY. This may involve an intermediate component, 'X,' which is a proposed transcriptional activator of CCA1 and LHY. 'X' may include LUX/PCL and/or ELF4. CCA1 and CHE form a reciprocally negative loop. TOC1 binds to CHE, which blocks CHE's inhibition of CCA1 expression. CCA1 and LHY also form the positive arm of the 'morning loop,' serving as positive regulators of PRR9, PRR7, and possibly PRR5. These three PRRs in turn negatively regulate CCA1 and LHY through direct promoter binding. In the evening loop, CCA1 and LHY negatively regulate a hypothetical component, 'Y,' which is a positive regulator of TOC1. GI is a likely component of 'Y.' ZTL is a cytosolic F-box protein that binds to TOC1 and PRR5, targeting them for proteasomal degradation. PRR5 stabilizes TOC1 by facilitating its nuclear accumulation, which may also contribute to TOC1 activity. GI binds to ZTL in the light and stabilizes ZTL and thus this primary interaction is indicated as a positive arrow. However, the binding of GI to ZTL may have a negative effect on ZTL degradation of TOC1 and PRR5 by blocking their interaction with ZTL. PRR3 and PRR5 also stabilize TOC1 by blocking its interaction with ZTL. Genes are coded as rectangles and proteins are coded as circles. Regulatory interactions are in black.Modified from [3,4].

far-red light pulse administered following imbibition [6]. Clock-regulated rhythmic gene expression can be detected immediately following germination, and imbibition provides a signal sufficient to synchronize clocks within a population of seedlings [7]. Clock genes play critical roles even earlier, and are necessary both for the proper establishment of dormancy and for the proper response to dormancy breaking in seeds [8].

How many clocks? Revisiting tissue and organ-specific clocks

One often speaks of 'the clock,' but considerable data have accumulated to argue persuasively for tissue-specific and

www.sciencedirect.com Current Opinion in Genetics & Development 2010, 20:588–598

this, *toc1* mutants do not show a shortened period in roots. However, when the ‘morning loop’ is disrupted in the *prr7 prr9* double mutant, *TOC1* cycling is restored [12] and apparently clock function is provided by the evening loop requiring TOC1. It would be interesting to test root clock function in the triple *prr7 prr9 toc1* mutant.

**Entrainment**

Circadian clocks are entrained to local time by a variety of input cues. The most important input cues are most obvious: light and temperature [13]. As mentioned above, imbibition also acts as a strong entraining cue [5,7]. More recently it has become apparent that the clock also monitors metabolites, such as organic nitrogen intermediates [14] or hormone levels, including cytokinin, brassinosteroid, and abscisic acid (ABA) [15,16,17], and uses their status to modulate clock phase. Because these intermediates are themselves under clock control [2–4], these inputs represent feedback loops in which the clock monitors its own outputs to modulate the status of central oscillator components.

**Mechanistic insights into oscillator function: loops within loops**

One of the most important milestones in the analysis of plant circadian clocks was the identification of the feedback loop in which TOC1 induces *CCA1* and *LHY*, which in turn represses TOC1 expression through direct binding to the TOC1 promoter [18]. However, the mechanism by which TOC1 positively regulates *CCA1* and *LHY* expression remains enigmatic.

Chromatin immunoprecipitation (ChIP) shows that TOC1 binds to the *CCA1* promoter [19]. This same study identified a second protein, CCA1 Hiking Expedition (CHE), a TCP (TEOSINTE BRANCHED1, CYCLOIDEA AND PCF) transcription factor, as a regulator of *CCA1* expression via a large-scale yeast one-hybrid screen of a library of Arabidopsis transcription factors for activated transcription from the *CCA1* promoter [19]. CHE binds to a canonical TCP binding site in the *CCA1* promoter *in vitro* and *in vivo*, although CHE does not bind to the *LHY* promoter. CHE is a negative regulator of *CCA1*; although loss of CHE function alone does not affect period, the *che lhy* double mutant has a shorter period than the *lhy* mutant, demonstrating redundancy of CHE and LHY in *CCA1* repression. *CCA1* (and also *LHY*) binds to the CHE promoter both *in vitro* and *in vivo* to repress CHE. Thus, CHE and *CCA1* form a novel reciprocally repressive feedback loop within the central oscillator.

Yeast two-hybrid experiments establish that CHE and TOC1 interact and CHE overexpression antagonizes the period lengthening resulting from TOC1 overexpression [19,20]. This would be consistent with CHE blocking a direct transcriptional induction of *CCA1* by TOC1, but no data are available to support this direct induction by TOC1.

Other components, including LUXARRHYTHMO/PHYTOCLOCK1 (LUX/PCL) [21,22] and EARLY FLOWERING 4 (ELF4) [23,24], are positive regulators of *CCA1* (and *LHY*), but this could be via indirect means. If these other players induce *CCA1*, it could be that TOC1 relieves CHE inhibition of *CCA1* expression, and thereby leads to *CCA1* induction indirectly. The details of *CCA1* transcriptional activation are slowly being revealed, but our understanding remains fragmentary.

CHE does not bind to the *LHY* promoter, emphasizing that these two close relatives are differently regulated. This difference might contribute to the differential expression of *CCA1* and *LHY* at low and high temperatures, which has been hypothesized to contribute to temperature compensation [25]. A recent study of the *LHY* promoter identified functionally important motifs, including a G-box and a CArG-like sequence to which FLC is recruited [26]. This may explain the known effects of FLC on circadian period [27]. However, the rhythmic expression of the *LHY* promoter is redundantly specified and none of the mutations tested abolished rhythmicity. As noted above, both LUX/PCL1 and ELF4 are positive regulators of *LHY* [21–24].

PRR7 and PRR9 are negative regulators of *CCA1* and *LHY* expression [28]. Recently, Nakamichi and colleagues [29] established that *CCA1* and *LHY* are direct targets of PRR5 through induction of a PRR5-GR-CFP fusion protein in the presence of the translational inhibitor, cycloheximide. Both *CCA1* and *LHY* mRNAs decreased immediately, making it clear that this regulation is direct. PRR5, PRR7 and PRR9 all have transcriptional repressor activity when targeted to a *LUC* reporter gene. This is relevant *in vivo* because PRR5, PRR7 and PRR9 can all be detected in the promoter regions of *CCA1* and *LHY* through ChIP assays. Peak binding of the three proteins occurs sequentially from PRR9 in early morning through PRR7 in the mid-light period to PRR5 in late afternoon/early night. The sequential expression patterns of the three PRR proteins extends the temporal window over which *CCA1* and *LHY* expression is repressed, offering a mechanistic explanation for their partially redundant function shown through genetic analysis of loss of function mutations [28,30–33].

The targeting of TOC1 to the *CCA1* promoter [19] and of PRR5, PRR7, and PRR9 to the *CCA1* and *LHY* promoters [29] raises the interesting question as to whether these proteins can bind directly to DNA or whether they require interaction with a known DNA-binding protein, as they have not been thought to possess a DNA-binding motif. The PRR proteins share two motifs, an N-terminal Pseudo-Receiver Domain related to the Receiver Domain found in two-component signaling response regulators, and a G-terminal CCT (named for CONSTANS [CO], CONSTANS-LIKE and TOC1) domain, which

was thought to function in protein–protein interactions. Recently it has been established that CO is recruited to a novel element in the *FT* promoter via its CCT domain [34]. This raises the hypothesis that these PRR proteins possess intrinsic DNA-binding activity through the CCT domain. Full definition both of this DNA-binding domain in the PRRs and of the DNA element(s) to which it binds requires further experimentation.

**Post-transcriptional regulation**

To date, most analyses of plant clock gene regulation have focused at the transcriptional and post-translational levels. However, examples are emerging in which mRNA stability is clock-regulated and, in the case of clock gene *CCA1*, this offers an additional mechanism with which to modulate clock function [35–37]. Alternative splicing is rapidly emerging as an important mechanism in expanding the proteomes of eukaryotes [38], and has been encountered as a mechanism to regulate expression of clock genes, including *CCA1* and *ELF3* [39, 40], as well as *GLYCINE RICH PROTEIN7 (GRP7)* [41, 42], a component of a circadian slave oscillator implicated in the promotion of flowering [43]. However, the detailed molecular mechanisms by which alternative splicing occurs within circadian networks remain largely obscure.

**Post-translational regulation**

Post-translational processes, notably phosphorylation and proteolysis, play critical roles in all clock systems [44]. It has been known for some time that casein kinase 2 phosphorylates CCA1 and that this phosphorylation is necessary for CCA1 function [45]. More recently it has been established that all five PRRs are phosphorylated with functionally significant consequences [46]. PRR5 and TOC1 are targeted for degradation through interaction with the F-box protein ZEITLUPE (ZTL) [46–48]. Phosphorylation of TOC1 and PRR3 promotes their interaction and this interaction blocks TOC1 from interaction with ZTL [46], offering mechanistic insight into how PRR3 stabilizes TOC1 [9]. A second PRR, PRR5 also binds to and stabilizes TOC1, although the PRR5–TOC1 interaction itself is independent of the phosphorylation status of both partners [49] (Figure 2). The PRR5-TOC1 interaction promotes nuclear accumulation of TOC1, which bears striking resemblance to TIMELESS–PERIOD, CYCLE–CLOCK and PERIOD2-CRYPTOCHROME interactions in Drosophila and mammals; in each case, interaction with the former promotes nuclear accumulation of the latter [49]. Interaction with PRR5 promotes TOC1 phosphorylation, which enhances the interaction of TOC1 with ZTL, but nuclear localization of the PRR5–TOC1 complex sequesters both proteins from cytoplasmic ZTL. This suggests a mechanism by which PRR5 could modulate TOC1 degradation [46]. However, it has not been shown that the PRR5-dependent phosphorylation sites on TOC1 are those that promote the TOC1/ZTL interaction. Phosphorylation of PRR5 promotes its interaction with ZTL, which leads to PRR5 degradation.

---

**Figure 2**

![Diagram](attachment:diagram.png)

A model for post-translational regulation of TOC1 by PRR5. In the cytoplasm, both PRR5 and TOC1 interact with ZTL and are targeted for proteasomal degradation. These interactions with ZTL are promoted by phosphorylation of PRR5 and TOC1. However, PRR5 and TOC1 also interact. This interaction promotes TOC1 phosphorylation and also facilitates the accumulation of TOC1 in the nucleus, where both proteins are protected from interaction with cytoplasmic ZTL [49]. Within the nucleus, TOC1 and PRR5 accumulate in nuclear foci [49]. TOC1 directly or indirectly induces transcription of target genes, such as *CCA1* (and *LHY*) [18, 19] and *ABAR* [17]. PRR5 binds to the promoters of *CCA1* (and *LHY*) to repress transcription [29]. It is not known if TOC1 and PRR5 remain complexed when recruited to target promoters.

Genetic analysis has made it clear that ZTL is the primary F-box protein responsible for degradation of TOC1 and PRR5. However, ZTL has two close relatives, LOV, KELCH PROTEIN2 (LKP2) and FLAVIN, KELCH, F-BOX1 (FKF1). Single loss of function mutations, *fkf1* or *lkp2*, have at most subtle effects on circadian function. However, compared with the *ztl* single mutant, clock defects are more pronounced in a *ztl fkf1* double mutant and even more pronounced in a *ztl fkf1 lkp2* triple mutant. This establishes that both LKP2 and FKF1 are capable of targeting TOC1 and PRR5 for proteasomal degradation [50].

The two remaining PRRs, PRR7 and PRR9, function in the ‘morning loop’ as negative regulators of CCA1 and LHY [29]. Both proteins show progressive phosphorylation as the day progresses [46]. This, in parallel with PRR5 and TOC1, suggests that phosphorylated PRR7 and PRR9 are targeted for proteasomal degradation. However, at this time mechanistic details such as the identity of a putative F-box protein (or other ubiquitin ligase complex) are lacking.

### Modeling of the clock

The power of systems approaches for gaining a deeper understanding of plant processes is probably best exemplified by efforts to model the clock machinery. Using published genetic and molecular data, a mathematical model of the original simple feedback loop containing CCA1/LHY and TOC1 was developed several years ago [51]. This model failed to explain reported circadian behaviors such as the pattern of *LHY* mRNA accumulation during the day or the short-period phenotype observed in *lhy* or *cca1* loss of function mutant plants. Adjusting the model structure by introducing interlocked feedback loops including two new components, X and Y, explained these experimental data. Subsequent analysis identified GIGANTEA (GI) as a possible candidate for Y function and later experimental work demonstrated GI as a new essential component of the central clock regulatory network that explains most of the Y functions [52]. Several models, increasingly more sophisticated, followed that added new components and regulatory feedback loops to better explain additional experimental data (reviewed in [53]). Similar approaches are now being utilized to model the integration of light signals, the photoperiod flowering pathway and the circadian clock [54]. Again, adjusting the structure of the original model to fit the experimental data predicts new components or regulatory interactions that can be tested experimentally. Namely, this modeling work predicts a novel positive regulatory interaction between FKF1 and FLOWERING LOCUS T (*FT*) controlling photoperiodism [54].

Despite scarce quantitative data, systems biology approaches are also providing novel insight into components and regulatory connections that integrate distinct biological processes such as the central circadian clock and nitrogen metabolism [14]. To identify potential ‘master’ regulators of the plant response to organic forms of nitrogen (N), transcriptomics data were analyzed in the context of gene networks [55] and several nitrogen-regulated transcription factors were identified on the basis of their regulatory potential [14]. At the top of the list, with 47 connections to targets in the N-regulated gene network, was found the central clock gene *CCA1*. ChIP assays using *CCA1* antibodies confirmed binding of *CCA1* to the promoter regions of some of its predicted targets, including central nitrogen metabolic genes. These results indicate that the circadian clock regulates N-assimilation by transcriptional regulation of N-assimilatory pathway genes by *CCA1*. In addition, the finding that *CCA1* mRNA levels are regulated by organic N-sources suggests that N signals act as an input to the circadian clock [14]. The observation that N-treatments resulted in subtle (2 h) but stable phase shifts in *pCCA1*-LUC expression, indicated that N-status serves as an input to the circadian clock [14]. The recent observation that *CCA1*, *LHY* and *PRR9* genes are differentially expressed under magnesium deficiency [56] suggests that other nutrients may have similar effects on clock function. The emerging view of the circadian clock as a key integrator of metabolic and physiologic processes is that it receives input not only from environmental stimuli but also from metabolic pathways, many of which are subject themselves to circadian regulation (Figure 3).

### Genome-wide characterization of circadian-regulated genes and processes

In order to understand the physiological significance of the circadian clock, it is important to identify the genes, pathways and processes that are circadian regulated. Several genome-wide studies have been carried out over the years to address this question using gene microarrays [57–60,61], tiling microarrays [39] and deep sequencing technologies [40]. These studies underscore the importance of clock regulation for the plant as it appears the circadian clock regulates virtually every biological process. Most prominently, hormone and stress response pathways stand out as over-represented among clock-controlled genes [61] highlighting the relevance of the clock for plant growth and development process and adaptation to changing environmental conditions.

Analysis of the transcriptome in *Arabidopsis thaliana* seedlings revealed temporal integration of hormone pathways as a mechanism to fine-tune phytohormone responses for plant growth regulation. Bioinformatic analysis of genome-wide expression data detected significant enrichment of genes involved in phytohormone metabolism or signaling at the time of day when hypocotyl growth rate is maximal [62]. The cis-acting element

Network news: prime time for systems biology of the plant circadian clock McClung and Gutiérrez 593

Figure 3

A network model of the circadian clock and its connections to entrainment and output pathways. The central clock regulatory network is composed of multiple interlocked negative feedback loops, each loop involving transcriptional activation and repression. The central loop consists of CIRCADIAN CLOCK ASSOCIATED 1 (CCA1) and LATE ELONGATED HYPOCOTYL (LHY), which repress expression of TIMING OF CAB EXPRESSION 1 (TOC1). TOC1 activates expression of CCA1 and LHY through an unknown mechanism that may include an unidentified factor X. CCA1 HIKING EXPEDITION (CHE) is a negative regulator of CCA1, which is itself repressed by CCA1. The two pseudo-response regulators, PRR7 and PRR9, repress their activators CCA1 and LHY, to form a second interlocked ‘morning loop’. PRR3, PRR5, PRR7, PRR9, and TOC1 are phosphorylated by an unknown kinase(s). PRR5 and TOC1 proteins are degraded through interaction with ZEITLUPE (ZTL) and, in the absence of ZTL, by LOV, KELCH PROTEIN2 (LKP2) and FLAVIN, KELCH, F-BOX1 (FKF1). TOC1 and PRR3 phosphorylation promotes their interaction and this blocks TOC1 from interaction with ZTL. A third ‘evening loop’ is composed of a hypothetical component ‘Y,’ that includes GIGANTEA (GI), a positive regulator of TOC1. Additional clock components have been identified but are not included as they cannot be connected with confidence to any node in this network. Green nodes represent components of the central loop; orange nodes represent components of the morning loop; purple nodes represent components of the evening loop; yellow nodes represent entrainment pathways to the central clock; gray nodes represent circadian clock outputs; thick gray edges show known interactions between the entrainment or output pathways and the central clock; green edges with an arrow head represent positive regulatory interactions (e.g., induction); red edges with a terminal perpendicular line represent negative regulatory interactions (e.g., repression); gray edges with a black circle at the end represent phosphorylation; thin dashed orange edges represent physical interactions between proteins; thick dashed orange edges with an arrow head show physical interactions for protein degradation.

(CACATG) was identified as sufficient to confer the predicted diurnal and circadian expression patterns *in vivo* of phytohormone genes [62]. Examination of the behavior of the phytohormone genes in circadian and light signaling mutants with defective hypocotyls growth showed that the circadian clock indirectly controls growth by gating light-responsive phytohormone transcript levels [62].

The circadian clock modulates (gates) the ability of a plant to respond to environmental cues such as low

www.sciencedirect.com
Current Opinion in Genetics & Development 2010, 20:588–598

Figure 4

Starch accumulation and utilization in wild type and circadian clock mutant plants during light/dark cycles. (a) In wild type, starch accumulates throughout the day and is degraded during the night (dark gray) at a constant rate such that starch is depleted at dawn, as anticipated by the circadian clock. In the event of an early dusk (light gray), starch accumulation ceases and starch is degraded at a reduced rate (dotted line) such that that starch is depleted at dawn. In both cases, carbon starvation gene expression (red line) is not induced. (b) In wild type subjected to an unexpected extension of the night, starch is depleted at predicted dawn, which precedes the real dawn, and carbon starvation gene expression (red line) is induced. (c) In wild type subjected to a short night, starch is not fully depleted at dawn. Carbon starvation gene expression (red line) is not induced. Nonetheless, growth is not maximal because of the failure to fully use accumulated starch (although it is unlikely that a single such short night would have a large effect). (d) In a short-period *cca1 lhy* mutant in 12/12 light/dark cycles, the circadian clock incorrectly predicts an early dawn and starch degradation rates are adjusted accordingly. Thus, starch is depleted before real dawn, with the consequent induction of carbon starvation genes. (e) In a long-period mutant in 12/12 light/dark cycles, the circadian clock would be expected to incorrectly predict a late dawn, with concomitant decrease of starch degradation rates. As a consequence, starch would not be fully depleted by real dawn. Carbon starvation genes would not be induced, but growth would not be maximal because of the failure to fully use accumulated starch.

temperatures [63]. A comparison of the transcriptome of wild-type and *prr9/prr7/prr5* triple mutant plants using Affymetrix technology showed that there was a significant overlap between cold-responsive genes and clock-controlled genes with a peak expression between subjective dawn and midday. These results suggested PRR9, PRR7 and PRR5 are important for anticipating diurnal stress by low temperature [64]. Cold acclimation responses require transcription factors of the CBF/DREB family and PRR5, PRR7 and PRR9 would gate the induction of DREB1 by low temperature [64]. Although the CBF/DREB transcription factors have a central role in the cold response, the regulators of the majority of the cold-responsive genes are unknown. The integration of cold-regulated and clock-regulated gene expression occurs through the interaction of regulatory proteins that bind to evening element (EE) and EE-like (EEL) elements with transcription factors acting at nearby ABA response element (ABRE)-like (ABREL) sequences; these two classes of elements are highly enriched in cold-induced genes and play a significant role in configuring the low-temperature transcriptome [65]. Microarray experiments in poplar [66] and in Arabidopsis [67] have established that the circadian clock gates the transcriptome-level response to drought; while a core set of genes responded to drought throughout the day, the magnitude of the response was strongly time-dependent. *TOC1* is a critical player linking the clock to ABA-mediated drought responses, and not only contributes to output from the clock to the ABA network but also is implicated in input to the clock from ABA; *TOC1* expression is acutely induced by ABA in a clock-gated manner [17].

Global studies have focused on plants found in temperate and sub-tropical climates. However, little is known about the circadian gene networks of plants that grow under constant day lengths and temperatures over the years. Recent genomic and computational analysis of the circadian clock would be expected to incorrectly predict a late dawn, with concomitant decrease of starch degradation rates. As a consequence, starch would not be fully depleted by real dawn. Carbon starvation genes would not be induced, but growth would not be maximal because of the failure to fully use accumulated starch.

Network news: prime time for systems biology of the plant circadian clock McClung and Gutiérrez 595

**Conclusions**

Interlocked molecular feedback loops at the heart of the clock are responsible for the rhythmic behaviors observed in plants and other systems. Determining how the central oscillators control their targets to explain biochemical, physiological, and behavioral rhythms will allow a deeper understanding of the roles of biological rhythms in enhancing growth and fitness. Genome-wide approaches to measure molecule levels such as microarray, next generation sequencing technologies, proteomics and metabolomics will continue to play important roles in the characterization of the output pathways of the circadian clocks. However, detailed information on physical and regulatory interactions is needed for a better understanding of the molecular networks underlying circadian rhythms. Moreover, integrating these data coherently in systems level models is essential for understanding biological rhythms in plants and other systems.

**Acknowledgements**

We thank D.E. Somers for Figure 2 and for helpful discussion. This work was funded by grants from the National Science Foundation (IOS 0960803, IOS 0923752, and IOS 0605736) and from the United States–Israel Binational Science Foundation (2005223) to C.R.M. and from the Fondo Nacional de Desarrollo Científico y Tecnológico (1100698), CONICYT-ANR (ANR-007), National Institutes of Health-Fogarty International Research Collaboration Award (F6414-01), and Millennium Nucleus for Plant Functional Genomics (P06-009-F) to R.A.G.

**References and recommended reading**

Papers of particular interest, published within the annual period of review, have been highlighted as:

- of special interest
- of outstanding interest

1. Gutierrez RA, Shasha DE, Coruzzi GM: Systems biology for the virtual plant. *Plant Physiol* 2005, **138**:550-554.
2. McClung CR: Plant circadian rhythms. *Plant Cell* 2006, **18**:792-803.
3. Harmer SL: The circadian system in higher plants. *Annu Rev Plant Biol* 2009, **60**:357-377.
4. Pruneda-Paz JL, Kay SA: An expanding universe of circadian networks in higher plants. *Trends Plant Sci* 2010, **15**:259-265.
5. Zhong HH, Painter JE, Salomé PA, Straume M, McClung CR: Imbibition, but not release from stratification, sets the circadian clock in Arabidopsis seedlings. *Plant Cell* 1998, **10**:2005-2017.
6. Oliverio KA, Crepy M, Martin-Tryon EL, Milich R, Harmer SL, Putterill J, Yanovsky MJ, Casal JJ: GIGANTEA regulates phytochrome A-mediated photomorphogenesis independently of its role in the circadian clock. *Plant Physiol* 2007, **144**:495-502.
7. Salomé PA, Xie Q, McClung CR: Circadian timekeeping during early Arabidopsis development. *Plant Physiol* 2008, **147**:1110-1125.
8. Penfield S, Hall A: A role for multiple circadian clock genes in the response to signals that break seed dormancy in Arabidopsis. *Plant Cell* 2009, **21**:1722-1732.
9. Para A, Farré EM, Imaizumi T, Pruneda-Paz JL, Harmon FG, Kay SA: PRR3 is a vascular regulator of TOC1 stability in the Arabidopsis circadian clock. *Plant Cell* 2007, **19**:3462-3473.
10. Michael TP, Salomé PA, McClung CR: Two Arabidopsis circadian oscillators can be distinguished by differential

www.sciencedirect.com
Current Opinion in Genetics & Development 2010, **20**:588–598

temperature sensitivity. Proc Natl Acad Sci USA 2003, 100:6878-6883.

11. Thain SC, Murtas G, Lynn JR, McGrath RB, Millar AJ: The circadian clock that controls gene expression in Arabidopsis is tissue specific. Plant Physiol 2002, 130:102-110.

12. James AB, Monreal JA, Nimmo GA, Kelly CL, Herzyk P, Jenkins GI, Nimmo HG: The circadian clock in Arabidopsis roots is a simplified slave version of the clock in shoots. Science 2008, 322:1832-1835.

13. Salomé PA, McClung CR: What makes Arabidopsis tick: light and temperature entrainment of the circadian clock. Plant Cell Environ 2005, 28:21-38.

14. Gutiérrez RA, Stokes TL, Thum K, Xu X, Obertello M, Katari MS, Tanurdzic M, Dean A, Nero DC, McClung CR et al.: Systems approach identifies an organic nitrogen-responsive gene network that is regulated by the master clock control gene CCA1. Proc Natl Acad Sci USA 2008, 105:4939-4944.

15. Hanano S, Domagalska MA, Nagy F, Davis SJ: Multiple phytohormones influence distinct parameters of the plant circadian clock. Genes to Cells 2006, 11:1381-1392.

16. Salomé PA, To JPC, Kieber JJ, McClung CR: Arabidopsis response regulators ARR3 and ARR4 play cytokinin-independent roles in the control of circadian period. Plant Cell 2006, 18:55-69.

17. Legnaioli T, Cuevas J, Mas P: TOC1 functions as a molecular switch connecting the circadian clock with plant responses to drought. EMBO J 2009, 28:3745-3757.
This study addresses the mechanistic link between the clock and ABA responses, showing that the H-subunit of magnesium-protoporphyrin IX chelatase (ABAR) and TOC1 comprise a negative feedback loop in which ABAR is necessary for ABA-mediated induction of TOC1 and TOC1 binds to the ABAR promoter to act as a transcriptional repressor.

18. Alabadi D, Oyama T, Yanovsky MJ, Harmon FG, Más P, Kay SA: Reciprocal regulation between TOC1 and LHY/CCA1 within the Arabidopsis circadian clock. Science 2001, 293:880-883.

19. Pruneda-Paz JL, Breton G, Para A, Kay SA: A functional genomics approach reveals CHE as a novel component of the Arabidopsis circadian clock. Science 2009, 323:1481-1485.
The authors screen a library of transcription factors for the ability to bind to the CCA1 promoter and identify CHE, a negative but redundantly specified negative regulator of CCA1 expression. CHE is negatively regulated by CCA1; thus, CCA1 and CHE comprise a new reciprocally repressive feedback loop in the clock.

20. Más P, Alabadi D, Yanovsky MJ, Oyama T, Kay SA: Dual role of TOC1 in the control of circadian and photomorphogenic responses in Arabidopsis. Plant Cell 2003, 15:223-236.

21. Hazen SP, Schultz TF, Pruneda-Paz JL, Borevitz JO, Ecker JR, Kay SA: LUX ARRHYTHMO encodes a Myb domain protein essential for circadian rhythms. Proc Natl Acad Sci USA 2005, 102:10387-10392.

22. Onai K, Ishiura M: PHYTOCLOCK1 encoding a novel GARP protein essential for the Arabidopsis circadian clock. Genes Cells 2005, 10:963-972.

23. Kikis EA, Khanna R, Quail PH: ELF4 is a phytochrome-regulated component of a negative-feedback loop involving the central oscillator components CCA1 and LHY. Plant J 2005, 44:300-313.

24. McWatters HG, Kolmos E, Hall A, Doyle MR, Amasino RM, Gyula P, Nagy F, Millar AJ, Davis SJ: ELF4 Is required for oscillatory properties of the circadian clock. Plant Physiol 2007, 144:391-401.

25. Gould PD, Locke JCW, Larue C, Southern MM, Davis SJ, Hanano S, Moyle R, Milich R, Putterill J, Millar AJ et al.: The molecular basis of temperature compensation in the Arabidopsis circadian clock. Plant Cell 2006, 18:1177-1187.

26. Spensley M, Kim J-Y, Picot E, Reid J, Ott S, Helliwell C, Carré IA: Evolutionarily conserved regulatory motifs in the promoter of the Arabidopsis clock gene late elongated hypocotyl. Plant Cell 2009, 21:2606-2623.

27. Salathia N, Davis SJ, Lynn JR, Michaels SD, Amasino RM, Millar AJ: FLOWERING LOCUS C-dependent and -independent regulation of the circadian clock by the autonomous and vernalization pathways. BMC Plant Biol 2006, 6:10.

28. Farré EM, Harmer SL, Harmon FG, Yanovsky MJ, Kay SA: Overlapping and distinct roles of PRR7 and PRR9 in the Arabidopsis circadian clock. Curr Biol 2005, 15:47-54.

29. Nakamichi N, Kiba T, Henriques R, Mizuno T, Chua N-H:
Sakakibara H: PSEUDO-RESPONSE REGULATORS 9, 7 and 5 are transcriptional repressors in the Arabidopsis circadian clock. Plant Cell 2010, 22:594-605.
This works shows that three PSEUDO-RESPONSE REGULATORS (PRRs) bind to the CCA1 and LHY promoters to function as transcriptional repressors. This provides important mechanistic insight into PRR function.

30. Salomé PA, McClung CR: PRR7 and PRR9 are partially redundant genes essential for the temperature responsiveness of the Arabidopsis circadian clock. Plant Cell 2005, 17:791-803.

31. Nakamichi N, Kita M, Ito S, Sato E, Yamashino T, Mizuno T: The Arabidopsis Pseudo-Response Regulators, PRR5 and PRR7, coordinately play essential roles for circadian clock function. Plant Cell Physiol 2005, 46:609-619.

32. Nakamichi N, Kita M, Ito S, Sato E, Yamashino T, Mizuno T: PSEUDO-RESPONSE REGULATORS, PRR9, PRR7 and PRR5, together play essential roles close to the circadian clock of Arabidopsis thaliana. Plant Cell Physiol 2005, 46:686-698.

33. Nakamichi N, Kita M, Niinuma K, Ito S, Yamashino T, Mizoguchi T, Mizuno T: Arabidopsis clock-associated pseudo-response regulators PRR9, PRR7 and PRR5 coordinately and positively regulate flowering time through the canonical CONSTANS-dependent photoperiodic pathway. Plant Cell Physiol 2007, 48:822-832.

34. Tiwari SB, Shen Y, Chang H-C, Hou Y, Harris A, Ma SF, McPartland M, Hymus GJ, Adam L, Marion C et al.: The flowering time regulator CONSTANS is recruited to the FLOWERING LOCUS T promoter via a unique cis-element. New Phytol 2010, 187:57-66.
CONSTANS (CO) is a critical component of the photoperiodic pathway of flowering regulation, yet until this work its biochemical function remained unclear. The authors show that the CCT domain (named for CO, CO-LIKE, and TOC1) recruits CO to a novel element in the FLOWERING LOCUS T (FT) promoter, where CO functions as a transcriptional activator.

35. Gutiérrez RA, Ewing RM, Cherry JM, Green PJ: Identification of unstable transcripts in Arabidopsis by cDNA microarray analysis: Rapid decay is associated with a group of touch- and specific clock-controlled genes. Proc Natl Acad Sci USA 2002, 99:11513-11518.

36. Lidder P, Gutiérrez RA, Salomé PA, McClung CR, Green PJ: Circadian control of mRNA stability: association with DST-mediated mRNA decay. Plant Physiol 2005, 138:2374-2385.

37. Yakir E, Hilman D, Hassidim M, Green RM: CIRCADIAN CLOCK ASSOCIATED1 transcript stability and the entrainment of the circadian clock in Arabidopsis. Plant Physiol 2007, 145:925-932.

38. Nilsen TW, Graveley BR: Expansion of the eukaryotic proteome by alternative splicing. Nature 2010, 463:457-463.

39. Hazen SP, Naef F, Quisel T, Gendron JM, Chen H, Ecker JR, Borevitz JO, Kay SA: Exploring the transcriptional landscape of plant circadian rhythms using genome tiling arrays. Genome Biol 2009, 10:R17.
This study uses tiling arrays to demonstrate widespread circadian clock regulation that extends beyond known protein-coding transcripts to include microRNAs, trans-acting short interfering RNAs, small nucleolar RNAs, antisense transcripts, and previously undescribed non-coding RNAs lacking predicted functions. Considerable clock-regulated alternative splicing was also seen.

40. Filichkin SA, Priest HD, Givan SA, Shen R, Bryant DW, Fox SE, Wong W-K, Mockler TC: Genome-wide mapping of alternative splicing in Arabidopsis thaliana. Genome Res 2010, 20:45-58.
This study uses ultrahigh throughput RNA sequencing to explore alter-

native splicing on a genomic scale. CCA1 offers one example of an alternatively spliced transcript where temporally regulated alternative isoforms containing premature termination codons are potential targets for nonsense mediated mRNA decay.

41. Staiger D, Zecca L, Kirk DAW, Apel K, Eckstein L: The circadian clock regulated RNA-binding protein AtGRP7 autoregulates its expression by influencing alternative splicing of its own pre-mRNA. *Plant J* 2003, **33**:361-371.

42. Schöning JC, Streitner C, Meyer IM, Gao Y, Staiger D: Reciprocal regulation of glycine-rich RNA-binding proteins via an interlocked feedback loop coupling alternative splicing to nonsense-mediated decay in Arabidopsis. *Nucl Acids Res* 2008, **36**:6977-6987.

43. Streitner C, Danisman S, Wehrle F, Schöning JC, Alfano JR, Staiger D: The small glycine-rich RNA binding protein AtGRP7 promotes floral transition in Arabidopsis thaliana. *Plant J* 2008, **56**:239-250.

44. Gallego M, Virshup DM: Post-translational modifications regulate the ticking of the circadian clock. *Nature Rev Mol Cell Biol* 2007, **8**:139-148.

45. Daniel X, Sugano S, Tobin EM: CK2 phosphorylation of CCA1 is necessary for its circadian oscillator function in Arabidopsis. *Proc Natl Acad Sci U S A* 2004, **101**:3292-3297.

46. Fujiwara S, Wang L, Han L, Suh SS, Salomé PA, McClung CR, Somers DE: Post-translational regulation of the circadian clock through selective proteolysis and phosphorylation of pseudo-response regulator proteins. *J Biol Chem* 2008, **283**:23073-23083.

47. Más P, Kim W-Y, Somers DE, Kay SA: Targeted degradation of TOC1 by ZTL modulates circadian function in Arabidopsis thaliana. *Nature* 2003, **426**:567-570.

48. Kiba T, Henriques R, Sakakibara H, Chua N-H: Targeted degradation of PSEUDO-RESPONSE REGULATOR5 by a SCFZTL complex regulates clock function and photomorphogenesis in Arabidopsis thaliana. *Plant Cell* 2007, **19**:2516-2530.

49. Wang L, Fujiwara S, Somers DE: PRR5 regulates phosphorylation, nuclear import and subnuclear localization of TOC1 in the Arabidopsis circadian clock. *EMBO J* 2010, **29**:1903-1915.
This study reveals an important and phylogenetically conserved molecular mechanism in the Arabidopsis clock in which PRR5 binds with TOC1 to promote the nuclear accumulation of TOC1, which is necessary for wild type clock function.

50. Baudry A, Ito S, Song YH, Strait AA, Kiba T, Lu S, Henriques R, Pruneda-Paz JL, Chua N-H, Tobin EM et al.: F-box proteins FKF1 and LKP2 act in concert with ZEITLUPE to control Arabidopsis clock progression. *Plant Cell* 2010, **22**:606-622.
Although ZEITLUPE (ZTL) is the major F-box protein required for the degradation of PRR5 and TOC1, this study establishes that the ZTL paralogs, FLAVIN BINDING, KELCH REPEAT, F-BOX1 (FKF1) and LOV KELCH PROTEIN2 (LKP2) play similar roles when ZTL is absent.

51. Locke JCW, Millar AJ, Turner MS: Modelling genetic networks with noisy and varied experimental data: the circadian clock in Arabidopsis thaliana. *J Theor Biol* 2005, **234**:383-393.

52. Locke JCW, Kozma-Bognár L, Gould PD, Fehér B, Kevei É, Nagy F, Turner MS, Hall A, Millar AJ: Experimental validation of a predicted feedback loop in the multi-oscillator clock of Arabidopsis thaliana. *Mol Syst Biol* 2006, **2**:59.

53. Hubbard KE, Robertson FC, Dalchau N, Webb AAR: Systems analyses of circadian networks. *Mol Biosyst* 2009, **5**:1502-1511.

54. Salazar JD, Saithong T, Brown PE, Foreman J, Locke JCW, Halliday KA, Carre IA, Rand DA, Millar AJ: Prediction of photoperiodic regulators from quantitative gene circuit models. *Cell* 2009, **139**:1170-1179.

55. Gutiérrez R, Lejay L, Dean A, Chiaramonte F, Shasha D, Coruzzi G: Qualitative network models and genome-wide expression data define carbon/nitrogen-responsive molecular machines in Arabidopsis. *Genome Biol* 2007, **8**:R7.

56. Hermans C, Vuylsteke M, Coppens F, Craciun A, Inzé D, Verbruggen N: Early transcriptomic changes induced by magnesium deficiency in Arabidopsis thaliana reveal the alteration of circadian clock gene expression in roots and the triggering of abscisic acid-responsive genes. *New Phytol* 2010, **187**:119-131.

57. Harmer SL, Hogenesch JB, Straume M, Chang HS, Han B, Zhu T, Wang X, Kreps JA, Kay SA: Orchestrated transcription of key pathways in Arabidopsis by the circadian clock. *Science* 2000, **290**:2110-2113.

58. Schaffer R, Landgraf J, Accerbi M, Simon V, Larson M, Wisman E: Microarray analysis of diurnal and circadian-regulated genes in Arabidopsis. *Plant Cell* 2001, **13**:113-123.

59. Edwards KD, Anderson PE, Hall A, Salathia NS, Locke JCW, Lynn JR, Straume M, Smith JQ, Millar AJ: FLOWERING LOCUS C mediates natural variation in the high-temperature response of the Arabidopsis circadian clock. *Plant Cell* 2006, **18**:639-650.

60. Michael TP, Mockler TC, Breton G, McEntee C, Byer A, Trout JD, Hazen SP, Shen R, Priest HD, Sullivan CM et al.: Network discovery pipeline elucidates time-of-day-specific cis-regulatory modules. *PLoS Genet* 2008, **4**:e14.

61. Covington MF, Maloof JN, Straume M, Kay SA, Harmer SL: Global transcriptome analysis reveals circadian regulation of key pathways in plant growth and development. *Genome Biol* 2008, **9**:R130.
The authors integrate information from multiple circadian microarray experiments in order to better estimate the fraction of the plant transcriptome that is circadian regulated, to identify regulatory elements correlated with phase-specific transcript accumulation, and to identify physiological pathways modulated by the circadian clock.

62. Michael TP, Breton G, Hazen SP, Priest H, Mockler TC, Kay SA, Chory J: A morning-specific phytohormone gene expression program underlying rhythmic plant growth. *Plos Biol* 2008, **6**:1887-1898.
The authors apply microarray analysis to assess the seedling transcriptome under multiple growth conditions and mutant backgrounds to show that the circadian clock indirectly controls growth by gating light-mediated phytohormone transcript levels to the proper time of day.

63. Fowler SG, Cook D, Thomashow MF: Low temperature induction of Arabidopsis CBF1, 2, and 3 is gated by the circadian clock. *Plant Physiol* 2005, **137**:961-968.

64. Nakamichi N, Kusano M, Fukushima A, Kita M, Ito S, Yamashino T, Saito K, Sakakibara H, Mizuno T: Transcript profiling of an Arabidopsis PSEUDO RESPONSE REGULATOR arrhythmic triple mutant reveals a role for the circadian clock in cold stress response. *Plant Cell Physiol* 2009, **50**:447-462.

65. Mikkelsen MD, Thomashow MF: A role for circadian evening elements in cold-regulated gene expression in Arabidopsis. *Plant J* 2009, **60**:328-339.

66. Wilkins O, Waldron L, Nahal H, Provart NJ, Campbell MM: Genotype and time of day shape the Populus drought response. *Plant J* 2009, **60**:703-715.
Together with the next reference [67], this study applies microarray analysis to define the transcriptomic response to water stress in poplar and Arabidopsis. In both species the transcriptional response is gated by the circadian clock, establishing clock function as crucial for proper response to this key abiotic stress, emphasizing the water stress responses as an additional mechanism for clock contribution to fitness.

67. Wilkins O, Bräutigam K, Campbell MM: Time of day shapes Arabidopsis drought transcriptomes. *Plant J.* 2010, **67**:
doi: 10.1111/j.1365-1313X.2010.04274.x.
See annotation for [66].

68. Zdepski A, Wang W, Priest HD, Ali F, Alam M, Mockler TC, Michael TP: Conserved daily transcriptional programs in *Carica papaya*. *Trop Plant Biol* 2008, **1**:236-245.

69. Green RM, Tingay S, Wang Z-Y, Tobin EM: Circadian rhythms confer a higher level of fitness to Arabidopsis plants. *Plant Physiol* 2002, **129**:576-584.

70. Dodd AN, Salathia N, Hall A, Kevei E, Toth R, Nagy F, Hibberd JM, Millar AJ, Webb AAR: Plant circadian clocks increase photosynthesis, growth, survival, and competitive advantage. *Science* 2005, **309**:630-633.

598 Genetics of system biology

71. Ni Z, Kim E-D, Ha M, Lackey E, Liu J, Zhang Y, Sun Q, Chen ZJ:
   - Altered circadian rhythms regulate growth vigour in hybrids and allopolyploids. *Nature* 2009, 457:327-331.
   The enhanced vigor associated with hybridization and polyploidization, termed hybrid vigour or heterosis, has long been exploited in agriculture, but to date there has been no understanding of the mechanisms underlying this phenomenon. This study establishes that at least part of the enhanced vigor response emerges from altered expression of a number of key components of the circadian clock with consequent alterations in the expression of a suite of genes associated with chlorophyll synthesis and starch metabolism.

72. Graf A, Schlereth A, Stitt M, Smith AM: Circadian control of carbohydrate availability for growth in *Arabidopsis* plants at night. *Proc Natl Acad Sci U S A* 2010, 107:9458-9463.
   This study shows that that the performance decrement associated with impaired clock function results from a defect in dawn anticipation, rather than from a failure of the endogenous circadian period to match the diel environmental cycle. The authors offer the physiological explanation that the performance decrement results not from decreased photosynthetic rates but rather from failure to optimally use fixed carbon stored transiently as starch.
