
Chemistry & Biology

Article

**Staphylococcus aureus and Bacillus subtilis W23 Make Polyribitol Wall Teichoic Acids Using Different Enzymatic Pathways**

Stephanie Brown,$^{1,2}$ Timothy Meredith,$^{1,2}$ Jonathan Swoboda,$^{1,3}$ and Suzanne Walker$^{1,*}$

$^{1}$Department of Microbiology and Molecular Genetics, Harvard Medical School, Boston, MA 02115, USA  
$^{2}$Present address: Merck Frosst Research Laboratories, Merck & Co., Montréal, QC H2X 3Y8, Canada  
$^{3}$Present address: The Skaggs Institute of Chemical Biology and Department of Chemistry, The Scripps Research Institute, La Jolla, CA 92037, USA  

*Correspondence: suzanne_walker@hms.harvard.edu*  
DOI 10.1016/j.chembiol.2010.07.017

---

### SUMMARY

Wall teichoic acids (WTAs) are anionic polymers that play key roles in bacterial cell shape, cell division, envelope integrity, biofilm formation, and pathogenesis. *B. subtilis* W23 and *S. aureus* both make polyribitol-phosphate (RboP) WTAs and contain similar sets of biosynthetic genes. We use in vitro reconstitution combined with genetics to show that the pathways for WTA biosynthesis in *B. subtilis* W23 and *S. aureus* are different. *S. aureus* requires a glycerol-phosphate primase called TarF in order to make RboP-WTAs; *B. subtilis* W23 contains a TarF homolog, but this enzyme makes glycerol-phosphate polymers and is not involved in RboP-WTA synthesis. Instead, *B. subtilis* TarK functions in place of TarF to prime the WTA intermediate for chain extension by TarL. This work highlights the enzymatic diversity of the poorly characterized family of phosphotransferases involved in WTA biosynthesis in Gram-positive organisms.

---

### INTRODUCTION

Teichoic acids are highly abundant anionic polymers found in Gram-positive bacteria. There are two types of teichoic acids: the lipoteichoic acids (LTAs), which are embedded in the bacterial membrane and extend into the peptidoglycan layers; and the wall teichoic acids (WTAs), which are covalently attached to the peptidoglycan layers and extend beyond them (*Figure 1A*) (*Neuhaus and Baddiley, 2003*). Teichoic acids play important but as yet poorly understood roles in cell shape determination (*D’Elia et al., 2006a; Pollack and Neuhaus, 1994; Soldo et al., 2002*), cell division (*Grundling and Schneewind, 2007; Oku et al., 2009; Schirner et al., 2009*), biofilm formation (*Fabretti et al., 2006; Fedtke et al., 2007; Vergara-Irigaray et al., 2008*), cell adhesion (*Gross et al., 2001; Weidenmaier et al., 2004*), and other aspects of Gram-positive physiology (*Swoboda et al., 2010; Xia et al., 2009*). Although neither type of teichoic acid is strictly essential for survival in vitro, organisms lacking either one display growth defects that range from mild to severe. Furthermore, it has been shown that pathogenicity is critically dependent on the expression of WTAs in *S. aureus* (*Weidenmaier et al., 2004, 2005*). A detailed understanding of WTA biosynthesis is important for exploring their roles in bacterial physiology and assessing their potential as antibacterial targets (*May et al., 2005; Swoboda et al., 2009*).

WTAs are attached via a phosphodiester linkage to the N-acetyl muramic acid sugars of peptidoglycan. WTAs typically consist of a disaccharide linkage unit followed by a polymeric main chain (*Figure 1B*). The *B. subtilis* W23 main chain is structurally identical to the main chain in *S. aureus* and contains ribitol-5-phosphate (RboP) repeats (*Swoboda et al., 2010*). As shown in *Figure 1C*, a pathway for polyribitol phosphate WTA synthesis was proposed many years ago by *Lazarevic et al. (2002)*. This model was based on comparing the genes for WTA biosynthesis in *B. subtilis* W23 to the genes in *B. subtilis* 168, which makes polyglycerol-phosphate WTAs (*Neuhaus and Baddiley, 2003; Ward, 1981*). Previous studies have confirmed the proposed functions of the first three steps in the RboP-WTA biosynthetic pathway. The first enzyme in this pathway, TagO, is an integral membrane protein that transfers phospho-GlcNAc from UDP-GlcNAc to an undecaprenyl carrier lipid embedded in the cytoplasmic surface of the bacterial membrane (*D’Elia et al., 2006b; Weidenmaier et al., 2004*). The lipid-linked monosaccharide is then converted to disaccharide 4 by the UDP-ManNAc transferase TagA (*Brown et al., 2008; D’Elia et al., 2009; Zhang et al., 2006*). A primase, TagB, then attaches a single glycerol-phosphate (GroP) unit to the nonreducing end of the disaccharide (*Brown et al., 2008*). Following assembly of the disaccharide linkage unit, the pathway for polyRboP-WTAs was proposed to require three enzymes (TarF, TarK, and TarL) to complete the polymeric main chain (*Lazarevic et al., 2002*). The proposed functions of these three enzymes are shown in *Figure 1C*. Once WTA synthesis is complete, the RboP polymers, still attached to the undecaprenyl carrier lipid, are flipped to the external surface of the membrane where they are attached to peptidoglycan (*Swoboda et al., 2010*).

Recent studies have shown that polyRboP-WTA polymer synthesis in *S. aureus* differs from the proposed pathway in *Figure 1C* in that only two enzymes are required to complete the polyRboP main chain (*Brown et al., 2008; Meredith et al., 2008; Pereira et al., 2008a*). One enzyme is TarF (TarF$_{\text{Sa}}$), which transfers a single GroP to the linkage unit. The other enzyme is TarL, which combines the proposed functions of TarK and TarL shown in *Figure 1C*. That is, *S. aureus* TarL (TarL$_{\text{Sa}}$) is a ribitol-phosphate

Chemistry & Biology

Pathway for *B. subtilis* W23 WTA Biosynthesis

Figure 1. Teichoic Acids Are a Major Component of the Gram-Positive Cell Wall, and the Pathway for Ribitol-Phosphate WTAs Has Been Proposed

(A) A schematic of the Gram-positive cell wall depicting membrane-anchored LTA and peptidoglycan-anchored WTA polymers.

(B) The chemical structure of ribitol-phosphate WTA as found in *S. aureus* and *B. subtilis* W23 (y = 1–2, z = 20–40). The tailoring modifications on the main chain hydroxyls are omitted for clarity.

(C) Lazarevic et al. (2002) proposed the biosynthetic pathway depicted above for polyribitol-phosphate WTAs. The numbers in parentheses correspond to the substrates utilized in our in vitro experiments (see Figure 2 for chemical structures).

substrates for several WTA enzymes in vitro (Brown et al., 2008; Ginsberg et al., 2006; Zhang et al., 2006). Here, we use the short chain substrate analogs shown in Figure 3 to characterize the late steps of polyribitol-phosphate WTA biosynthesis in *B. subtilis* W23. As described later, analysis of the products formed in vitro by *B. subtilis* TarF (TarF<sub>BS</sub>), combined with the in vitro substrate preferences for *B. subtilis* TarK (TarK<sub>BS</sub>), suggested a revised pathway for polyRboP-WTA biosynthesis in *B. subtilis* W23. Using genetic approaches, we have confirmed the revised *B. subtilis* W23 WTA biosynthetic pathway.

Biochemical Reconstitution of *B. subtilis* W23 TarF and Comparison to *S. aureus* TarF

The first three steps in the pathways for WTA synthesis in *B. subtilis* W23 and *S. aureus* are identical and lead to the production of lipid-pp-GlcNAc-ManNAc-GroP (compound 4) (Figure 1C). We have shown that *S. aureus* TarF is a primase that adds a single glycerol phosphate unit to 4 to produce lipid-pp-GlcNAc-ManNAc-GroP<sub>2</sub> (compound 5) (Brown et al., 2008). The function of TarF in *B. subtilis* W23 has not previously been studied, but it was proposed, like its *S. aureus* ortholog, to act as a GroP primase (Lazarevic et al., 2002), which catalyzes the reaction of 4 with CDP-glycerol (1) to form WTA intermediate 5. To test this prediction, we cloned and expressed *B. subtilis* W23 TarF (TarF<sub>BS</sub>) as a C-terminal 6-His fusion from genomic DNA and purified it via Ni<sup>2+</sup> chromatography. We incubated purified TarF<sub>BS</sub> with radiolabeled 4b in the presence of [<sup>14</sup>C]-CDP-glycerol (1) and analyzed the results by polyacrylamide gel electrophoresis. PAGE analysis showed that CDP-glycerol (1) reacted in the presence of TarF<sub>BS</sub> and lipid-pp-GlcNAc-ManNAc-GroP (4b) to generate a new radioactive spot (see Figure S1A available online); see below for product identification. TarF<sub>BS</sub> under the same conditions did not react with 1 and 3b, or with CDP-ribitol (2) and 4b. These results suggested that TarF<sub>BS</sub>, like its *S. aureus* counterpart, catalyzes the addition of GroP, but not ribitol-phosphate, to primed substrate 4, but not to the unprimed precursor 3.

We next analyzed the products formed by the reaction of radiolabeled 1 and 4b in the presence of TarF<sub>BS</sub>. For comparison

RESULTS

Description of the Approach

As described in the Introduction, WTA precursors are synthesized in the cytoplasm on an undecaprenyl phosphate-carrier lipid embedded in the membrane (Swoboda et al., 2010). The first enzyme in the WTA biosynthetic pathway, TagO, is an integral membrane protein, but many of the other enzymes involved in the assembly of the WTA polymer do not have predicted membrane-spanning regions, although it is presumed that they are membrane-associated either directly or indirectly (Bhavsar et al., 2007; Formstone et al., 2008). Membrane-associated enzymes in other lipid carrier-mediated biosynthetic pathways (e.g., peptidoglycan biosynthesis) will accept water-soluble substrates containing truncated lipid chains in the absence of biological membranes or membrane mimics (Chen et al., 2002, 2007; Faridmoayer et al., 2008; May et al., 2009; Men et al., 1998; Ye et al., 2001). Therefore, we synthesized WTA pathway intermediates containing short lipid chains and have previously shown that these intermediates are functional

Chemistry & Biology

Pathway for B. subtilis W23 WTA Biosynthesis

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

Figure 2. Structures of WTA Substrates and Intermediates

The structures of the CDP-glycerol (1) and CDP-ribitol (2) donor and lipid acceptor substrates 3, 4, 5, 6, and 7 are shown. The natural acceptor substrates contain an undecaprenyl chain. Substrates having the “a” suffix contain a neryl chain and were used for LC/MS analysis; substrates having the “b” suffix contain a farnesyl chain and were used for PAGE analysis. Synthesis of substrates and incorporation of \[^{14}C\] radiolabels for PAGE analysis were carried out as described previously (Brown et al., 2008).

the same substrates were also incubated with S. aureus TarF (TarF\_{Sa}). Products were analyzed via PAGE (Figures 3B and 3C). TarF\_{Sa} produces a single well-resolved product, identified by exact mass analysis of a nonradioactive reaction as lipid-pp-GlcNAc-ManNAc-GroP₂ (compound 5); this product corresponds to the transfer of a single GroP unit to 4. Under forcing conditions (high enzyme and substrate concentrations for prolonged reaction times), an additional product containing three GroP units (compound 6) can also be observed (Figure S2A). In marked contrast, under all conditions examined (low and high enzyme concentrations; short or long reaction times), the TarF\_{Bs} reaction produces a smear of radioactivity.

![Reaction Scheme](https://i.imgur.com/reaction_scheme.png)

Figure 3. TarF Is a Polymerase in B. subtilis W23 and a Primase in S. aureus

(A) Reaction carried out by TarF (n = 3–9 for TarF\_{Bs}; n = 2 for TarF\_{Sa}).

(B and C) PAGE analysis of TarF\_{Bs} (B) and TarF\_{Sa} (C) reactions using 1 and substrate 4b in the presence of heat-treated (−) or active enzyme (+). Tabulated masses of purified TarF reaction products using substrate 4a are shown.

without a distinct banding pattern. These results suggested the formation of multiple products. Reactions with TarF<sub>Bs</sub> and substrates 1 and 4a were carried out using nonradiolabeled substrates and subjected to Q-TOF MS analysis. Discrete products with increasing numbers of GroP units up to 9 units were detected. Thus, whereas TarF<sub>Sa</sub> is a primase that preferentially adds a single GroP unit to 4, TarF<sub>Bs</sub> behaves as a polymerase, adding GroP units to 4 in rapid succession up to a detectable length of at least 9 units.

**Reconstitution of B. subtilis W23 TarK and TarL**

Lazarevic et al. (2002) proposed that TarK (TarK<sub>Bs</sub>) and TarL (TarL<sub>Bs</sub>) from *B. subtilis* W23 form a primase-polymerase pair, with the former transferring a single ribitol-phosphate unit to the TarF<sub>Bs</sub> product and the latter adding multiple ribitol-phosphate units to build the polymer chain (Figure 1C). Using a heterologous complementation approach, we have previously provided evidence that *B. subtilis* W23 TarK and TarL function as a primase-polymerase pair because both are required to complement the deletion of *S. aureus* TarL, which combines both functions (Meredith et al., 2008; Pereira et al., 2008a). However, the genetic complementation experiments did not provide detailed information on the enzymatic reactions carried out by TarK<sub>Bs</sub> and TarL<sub>Bs</sub>. Here, we have reconstituted the in vitro activities of these enzymes to characterize their substrate preferences and verify their proposed products.

TarK<sub>Bs</sub> and TarL<sub>Bs</sub> were cloned from genomic *B. subtilis* W23 DNA, expressed as C-terminal 6-His fusion proteins and purified by nickel chromatography. WTAs extracted from *B. subtilis* W23 cells are reported to contain 1–2 GroP units, and because TarK<sub>Bs</sub> is a proposed ribitol-phosphate phosphotransferase, we first tested the activity of TarK<sub>Bs</sub> with CDP-ribitol (2) and WTA intermediate 4a, which contains 1 GroP unit. The reaction was analyzed by HPLC because products could not be resolved by polyacrylamide gel electrophoresis (Figure S3A). A decrease in the CDP-ribitol peak, accompanied by the appearance of CMP, showed that TarK<sub>Bs</sub> reacts with CDP-ribitol in the presence of 4a. We also tested the activity of TarK<sub>Bs</sub> with 4a and CDP-glycerol (1). An unchanged peak area for CDP-glycerol with no appearance of CMP showed that TarK<sub>Bs</sub> does not utilize 1 as a donor.

Exact mass analysis of a TarK<sub>Bs</sub> reaction of 2 with 4a showed that the predominant product, compound 7a, contained 1 ribitol-phosphate unit (m/z [M-H]<sup>-</sup> expected: 1087.2472, actual: 1087.2509); a small amount of product (mass abundance less than 0.05% compared with compound 7a) containing 2 ribitol-phosphate units was also detected (m/z [M-H]<sup>-</sup> expected: 1301.2715, actual: 1301.2591). The MS data and HPLC traces confirm that TarK from *B. subtilis* W23 is a ribitol-phosphate primase that catalyzes the addition of a ribitol-phosphate unit from CDP-ribitol to WTA intermediate 4 to form lipid-pp-GlcNAc-ManNAc-GroP-RboP (7).

To evaluate the substrate specificity of TarK<sub>Bs</sub> in more detail, we carried out tandem reactions of *B. subtilis* TarK and TarL. Tandem reactions were used to enable product analysis by PAGE because the TarK products cannot be resolved from starting material. TarK<sub>Bs</sub> and TarL<sub>Bs</sub> were incubated with CDP-ribitol (2) and radiolabeled lipid intermediates 4b, 5b, and 6b. These intermediates differ in the number of glycerol phosphates

appended to the ManNAc sugar. Reactions were analyzed for the appearance of polymeric product (Figure 4A). The TarK<sub>Bs</sub>/TarL<sub>Bs</sub> tandem reaction converted substrate 4b to polymeric product. Tandem reactions using substrates containing more than 1 GroP unit (e.g., 5b or 6b) resulted in little to no WTA polymer.

To confirm that the resulting polymeric products observed in the TarK<sub>Bs</sub>/TarL<sub>Bs</sub> reactions were due to a tandem reaction of TarK<sub>Bs</sub> acting as a primase and then TarL<sub>Bs</sub> acting as a polymerase, we tested the ability of *B. subtilis* W23 TarL to catalyze the addition of ribitol-phosphate units to different WTA intermediates. TarL<sub>Bs</sub> was incubated with 4b, 5b, 6b, or 7b in the presence of CDP-ribitol (2). The reaction products were analyzed by PAGE (Figure 4B; Figure S4A). No radioactive products were observed for reactions containing lipid substrates 4b, 5b, or 6b, showing that TarL<sub>Bs</sub> is unable to use any of the GroP primed intermediates to make polymer. In contrast the enzymatic reaction with compound 7b showed the disappearance of starting material and the appearance of a smear of radioactivity corresponding to higher molecular weight products. The gel analysis shows that TarL<sub>Bs</sub> reacts with the ribitol-phosphate-primed substrate and is consistent with the addition of multiple RboP subunits. These data confirm that TarL<sub>Bs</sub> is a ribitol-phosphate polymerase and that it requires for reaction a ribitol-phosphate-primed substrate produced by TarK<sub>Bs</sub>.

The TarK<sub>Bs</sub>/TarL<sub>Bs</sub> tandem reactions showed that TarK<sub>Bs</sub> has a strong preference for substrate 4, but it was not clear whether the small amount of polymeric product observed in the middle panel of Figure 4A resulted from the reaction of TarK<sub>Bs</sub> with 5b or with contaminating 4b because the starting material was estimated to be only 90% pure, and the impurity could not easily be removed. Therefore, we incubated TarK<sub>Bs</sub> with CDP-ribitol (2) and a mixture of 4a and 5a and analyzed the products by Q-TOF MS. The major peak proved to be unreacted lipid-pp-GlcNAc-ManNAc-GroP<sub>2</sub> (compound 5a) (m/z [M-H]<sup>-</sup> expected: 1027.2261, actual: 1027.2264). Another peak corresponding to compound 7a (m/z [M-H]<sup>-</sup> expected: 1087.2472, actual: 1087.2468) was detected, resulting from TarK<sub>Bs</sub> utilizing 4a as a substrate. We were unable to detect any product peak corresponding to the addition of RboP to 5a. These results imply that TarK<sub>Bs</sub> is only able to accept lipid-pp-GlcNAc-ManNAc-GroP (4), the TagB product, as a substrate and that the small amount of polymeric product observed in the middle panel of Figure 4A is due to a reaction with the small amount of substrate 4 in the starting material.

The results described in the previous paragraph confirm that TarK<sub>Bs</sub> and TarL<sub>Bs</sub> comprise a RboP primase-polymerase pair and carry out the reactions shown in Figure 4C. These reactions are not consistent with the model shown in Figure 1C (Lazarevic et al., 2002). In that model, TarF<sub>Bs</sub> acts as a primase to synthesize a lipid-GlcNAc-ManNAc-GroP<sub>2</sub> substrate for TarK<sub>Bs</sub>, but our results show that TarK<sub>Bs</sub> acts only on the TagB product, which contains a single GroP unit. Because TarK<sub>Bs</sub> is unable to accept WTA intermediates containing more than a single GroP unit, and because TarF<sub>Bs</sub> is a polymerase that adds multiple GroP units to 4, we surmised that TarF<sub>Bs</sub> could not be on the pathway to polyribitol WTA formation in *B. subtilis* W23. If this supposition is correct, then one would predict that *B. subtilis*

Chemistry & Biology

Pathway for *B. subtilis* W23 WTA Biosynthesis

A
TarK<sub>Bs</sub>
TarL<sub>Bs</sub>
TarK<sub>Bs</sub>
TarL<sub>Bs</sub>
TarK<sub>Bs</sub>
TarL<sub>Bs</sub>

RboP-WTA product —
4b—
5b—
6b—
- +
- +
- +

B
TarL<sub>Bs</sub>
TarL<sub>Bs</sub>
— RboP-WTA product
4b—
7b—
- +
- +

C
CDP-ribitol (2)
+ 
TarK<sub>Bs</sub>
4
7
+
TarL<sub>Bs</sub>
WTA
polymer
CDP-ribitol (2)

Figure 4. TarK and TarL React Sequentially on the TarB Product (4)

(A) PAGE analysis of a tandem TarK<sub>Bs</sub>/TarL<sub>Bs</sub> reaction using either heat-treated (-) or active (+) enzymes incubated with CDP-ribitol and 4b, 5b, or 6b. The polymeric material in the middle panel is due to reaction with contaminating 4b. Polymer product only forms when 4b is used as the starting material (see text).

(B) PAGE analysis of TarL<sub>Bs</sub> heat-treated (-) or active (+) enzyme incubated with 4b or 7b and CDP-ribitol (2). TarL<sub>Bs</sub> reacts only with 7b. See also Figure S4.

(C) Reactions carried out by TarK<sub>Bs</sub> and TarL<sub>Bs</sub> with CDP-ribitol (2) and substrates 4 and 7, respectively.

tarF can be deleted, and polyribitol-phosphate WTAs will still be produced in strains containing the *B. subtilis* TarK-TarL primase-polymerase pair.

**TarF Is Dispensable for Polyribitol-Phosphate WTA Synthesis in Pathways Containing a TarK Primase**

To test the predicted dispensability of *B. subtilis* TarF in strains expressing *B. subtilis* TarK and TarL, we deleted tarF from two different strains expressing these *B. subtilis* enzymes. The first strain was an *S. aureus* hybrid strain (JT215) containing *B. subtilis* TarK-TarL in place of the native bifunctional *S. aureus* enzymes (TarL or TarL') (Meredith et al., 2008). This hybrid strain contains the native *S. aureus* tarF, which is essential in the wild-type *S. aureus* background (D'Elia et al., 2006b). Despite its essentiality in the wild-type background, we were able to delete tarF from the hybrid *S. aureus* strain. In fact, the ΔtarF deletion allele was almost exclusively observed upon resolving the integrated tarF deletion vector, and the doubling time of the ΔtarF hybrid strain was faster than that of the hybrid parent strain, suggesting that the removal of TarF actually increases strain fitness. We confirmed that WTAs extracted from the ΔtarF hybrid strain are identical in length and banding pattern to those extracted from the parent strain (Figure 5C). Thus, replacing *S. aureus* TarL with Tark<sub>Bs</sub>-TarL<sub>Bs</sub> renders *S. aureus* TarF dispensable for polyRboP-WTA synthesis in an *S. aureus* background.

We next constructed a marked tarF deletion in a wild-type *B. subtilis* W23 strain containing only the native *B. subtilis* enzymes. The ΔtarF strain (DM1) had a similar growth rate as the parent strain, and the colony morphology was unchanged (Figures 5A and 5B). There were no apparent differences in the amounts of extracted WTAs in the ΔtarF and parent strains. Furthermore,

Chemistry & Biology 17, 1101–1110, October 29, 2010 ©2010 Elsevier Ltd All rights reserved 1105

Chemistry & Biology

Pathway for *B. subtilis* W23 WTA Biosynthesis

Figure 5. TarK<sub>Bs</sub> and TarL<sub>Bs</sub> Make polyRboP-WTAs In Vivo in the Absence of TarF<sub>Bs</sub>

(A) Growth curves of *B. subtilis* W23 wild-type (WT) or ΔtarF strains.

(B) Photograph of bacterial colonies of *B. subtilis* W23 wild-type (WT) or ΔtarF strains.

(C and D) WTAs of the parent wild-type strain or the ΔtarF strain extracted from the *B. subtilis*/*S. aureus* hybrid strain JT215 (C) or *B. subtilis* W23 (D) analyzed by PAGE analysis and Alcian blue/silver staining.

(E) Agarose gel analysis of PCR-amplified fragments from *B. subtilis* W23 cDNA or genomic DNA (positive control). (+) and (−) indicate the presence or absence of reverse transcriptase during cDNA synthesis. The sizes of the amplified fragments are 170 bp (*tarK*), 159 bp (*tarF*), and 68 bp (16S rRNA) (internal control).

capable of making polyGroP-WTAs in a Δ*tarK* background. All efforts to delete *tarK* were unsuccessful, suggesting that this gene, unlike *tarF*, is essential in *B. subtilis* W23. Therefore, we examined the relative levels of *tarF* and *tarK* gene expression using RT-PCR. Under normal laboratory conditions, the *tarF* transcript is barely detectable in cells harvested during late exponential growth, whereas the *tarK* transcript is abundant (Figure 5E). Increased amounts of template cDNA in the PCR reaction did not change the levels of detectable transcript. Although TarF<sub>Bs</sub> is a functional enzyme, it is not expressed under normal laboratory growth conditions. Its cellular function, if any, remains a mystery, but it is not involved in polyRboP-WTA biosynthesis.

DISCUSSION

WTAs comprise a large fraction of the Gram-positive cell wall. They play important roles in cell envelope integrity, and there is increasing evidence that they are involved in fundamental aspects of cell growth and division (Xia et al., 2009). Understanding how these polymers are made is a necessary step in dissecting their cellular functions and in exploring their potential as antimicrobial targets in pathogens such as *S. aureus*. Although pathways for polyglycerol-phosphate and polyribitol-phosphate WTA biosynthesis in *B. subtilis* were proposed many years ago (Lazarevic et al., 2002; Ward, 1981), verification and detailed characterization of the enzymatic steps were not possible until recently. The development of chemoenzymatic approaches to obtain functional WTA intermediates has made the in vitro reconstitution of both individual enzymes and entire pathways for WTA biosynthesis feasible (Brown et al., 2008; Ginsberg et al., 2006; Sewell et al., 2009; Zhang et al., 2006). In this paper we have carried out an extensive biochemical and genetic analysis of polyRboP-WTA synthesis in *B. subtilis* W23. Although some steps of the proposed model for polyRboP-WTA biosynthesis were confirmed, we have revised others.

Characterization of the enzymes proposed to be involved in the late steps of the *B. subtilis* polyribitol-phosphate WTA pathway revealed two unexpected results. First, unlike *S. aureus*

*B. subtilis tarF* Is Not Expressed during Exponential Growth

We have shown that *tarF<sub>Bs</sub>*, which is located within the WTA biosynthetic gene cluster, encodes a functional enzyme that makes short GroP-WTA polymers, but it is not involved in polyRboP WTA synthesis. Because it has been reported that strains in the W23 group (Nakamura et al., 1999), *S. xylosus*, *S. saprophyticus* (Endl et al., 1983, 1984), and an *S. aureus* biofilm-producing strain (Vinogradov et al., 2006) produce both ribitol-phosphate and GroP WTAs, we considered the possibility that *B. subtilis* W23 might contain a small proportion of polyGroP-WTAs. We were unable to confirm the presence of GroP-WTAs by MS analysis of WTAs extracted from *B. subtilis* W23 (the detection limit is 5 pmol), so we attempted to generate deletions of *tarK* in *B. subtilis* W23 to determine whether TarF<sub>Bs</sub> was

1106 Chemistry & Biology 17, 1101–1110, October 29, 2010 ©2010 Elsevier Ltd All rights reserved

Chemistry & Biology

Pathway for *B. subtilis* W23 WTA Biosynthesis

![Diagram](#)

Figure 6. Two Pathways for Polyribitol-Phosphate WTA Formation in Bacteria

A schematic of the verified late-stage intracellular steps in WTA biosynthesis in *S. aureus* and *B. subtilis* W23.

ManNAc-phosphoglycerol unit (4) (Bhavasar et al., 2005; Ginsberg et al., 2006; Soldo et al., 2002; Zhang et al., 2006). Following the third step, catalyzed by TagB, the *B. subtilis* W23 and *S. aureus* pathways diverge. *B. subtilis* W23 uses a RboP primase (TarK) followed by an RboP polymerase (TarL<sub>Bs</sub>) to complete the WTA main chain; *S. aureus* uses a GroP primase (TarF) followed by a bifunctional RboP primase/polymerase (TarL<sub>Sa</sub>) to complete the chain. The revised pathways, shown in Figure 6, are in line with previously reported data by Yokoyama et al. (1986) on the isolation of WTA intermediates from *B. subtilis* W23 and *S. aureus*.

In addition to establishing the pathways for polyribitol-phosphate WTA polymer synthesis, we note that this work highlights the enzymatic diversity of the large family of CDP-phosphotransferases involved in WTA biosynthesis. These enzymes include both GroP and RboP primases and polymerases, and it does not yet appear possible to predict their functions or substrate preferences accurately based on protein sequences (Figure S6). For example, *S. aureus* TarF is a GroP primase that adds 1 GroP subunit to the linkage unit, whereas *B. subtilis* 168 TagF is a GroP polymerase that adds 45–60 GroP units (Pereira et al., 2008b). *B. subtilis* W23 TarF makes short GroP polymers in vitro, adding about eight GroPs to the linkage unit. The RboP transferases are similarly diverse, with the primase TarK<sub>Bs</sub> having stringent specificity for singly GroP-primed substrates and the sequence-related polymerase TarL<sub>Bs</sub> having similarly stringent specificity for RboP-primed substrates, whereas *S. aureus* TarL functions both as an RboP primase and polymerase and is selective for doubly GroP-primed substrates. The first structure of a member of this diverse family of phosphotransferases was recently reported (*Staphylococcus epidermidis* TagF) (Lovering et al., 2010), and it may, when combined with accurate information about enzymatic function, facilitate the identification of the features that determine which substrates are selected or whether a particular enzyme functions as a primase or a polymerase. In closing we note that there is a strong argument to be made for a new systematic nomenclature to designate the CDP-phosphotransferases involved in WTA biosynthesis now that the diversity of enzymatic function has been recognized, and the first ascribed gene names do not accurately describe these functions.

### SIGNIFICANCE

WTAs are important cell surface polymers in many Gram-positive organisms, both pathogens and nonpathogens.

alike. They play central roles in cell growth, division, morphology, adhesion, and envelope integrity. Understanding how they are made is important for conducting detailed studies of their cellular functions (Swoboda et al., 2010). We have developed chemoenzymatic approaches to obtain WTA intermediates to enable the detailed enzymatic characterization of WTA biosynthesis. Here, we have used an extensive set of purified WTA intermediates to compare the pathways for polyribitol-phosphate WTA synthesis in *B. subtilis* W23 and *S. aureus*. We show that these organisms use different sets of enzymes to make similar WTA polymers. Hence, bacteria employ two distinct pathways to make polyribitol-phosphate WTAs as outlined in Figure 6. This work should facilitate continued efforts to predict teichoic acid gene functions and determine pathway order in sequenced genomes, will enable well-grounded biological studies to establish the roles of WTA enzymes in the growth and division of *S. aureus* and *B. subtilis*, and may facilitate antibiotic discovery.

### EXPERIMENTAL PROCEDURES

#### Materials and Methods
*B. subtilis* W23 is from ATCC (ATCC 23059). Restriction enzymes were from New England Biolabs. The plasmid used for cloning, pET-24b(+), and His-Bind Resin are available from EMD Chemicals. PCR was performed using KOD Hot-Start (EMD Chemicals). The pKM074a vector was a generous gift from David Rudner (Harvard Medical School). Radiolabeled L-[¹⁴C]-glycerol-3-phosphate was purchased from GE Healthcare. All other reagents and chemicals used were from Sigma. The WTA extraction and analysis by polyacrylamide gel electrophoresis and Alcian blue/silver staining have been described previously (Meredith et al., 2008).

#### Cloning of *B. subtilis* W23 Genes
The *S. aureus* enzymes used in this study were cloned, overexpressed, and purified as described previously (Brown et al., 2008). All primers and strains are listed in Supplemental Information. The *tarF*, *tarK*, and *tarL* genes were PCR amplified using *B. subtilis* W23 genomic DNA as a template and the primers tarFp24bF and tarFp24bR, tarKp24bF and tarKp24bR, and tarLp24bF and tarLp24bR. The PCR products were digested with the appropriate restriction enzymes and ligated into a similarly digested pET-24b(+) vector. The constructed plasmids were verified by restriction digest and DNA sequencing (Dana-Farber/Harvard Cancer Center DNA Resource Core).

#### Protein Overexpression and Purification
*S. aureus* proteins used in this study were overexpressed and purified as described previously (Brown et al., 2008). TagB from *B. subtilis* 168 was used to construct various WTA intermediates and was overexpressed and purified as described previously (Ginsberg et al., 2006). All constructed Tar protein-pET-24b plasmids were transformed into Rosetta2(DE3)pLysS cells for overexpression with IPTG. The cell pellets were lysed with rLysozyme and benzonase in 100 mM Tris HCl (pH 7.5), 500 mM NaCl, 0.6% CHAPS, and 5% glycerol. The clarified lysate was purified over charged nickel His-Bind resin. More detailed procedures can be found in Supplemental Experimental Procedures. The proteins were stored as 50% glycerol stocks at either –20°C or –80°C. The yield is approximately 5 mg/l for TarF<sub>Bs</sub>, 10 mg/l for TarK<sub>Bs</sub>, and 4 mg/l for TarL<sub>Bs</sub>.

#### Construction of WTA Intermediates
Radiolabeled [¹⁴C]-CDP-glycerol (1) was prepared as described previously using TarD (Badurina et al., 2003). CDP-ribitol (2) was prepared as described previously (Brown et al., 2008; Pereira and Brown, 2004). Due to issues with degradation upon long-term storage, CDP-ribitol was purified by HPLC after chemoenzymatic synthesis using a strong anion exchange column from Phenomenex (Phenosphere SAX80A, 250 × 4.6 mm 5 μm). For purification details, see Supplemental Experimental Procedures. WTA intermediates 3, 4, and 5 were prepared as described previously (Brown et al., 2008). To prepare radiolabeled compound 6b, an enzymatic reaction (30 μl) containing 2 μM TagB, 2 μM TarF<sub>Sa</sub>, 20 μM of purified 3b, and 80 μM [¹⁴C]-CDP-glycerol were incubated in 20 mM Tris (pH 7.5), 100 mM NaCl, and 10 mM MgCl₂ for 30 min at room temperature. This allows formation of intermediate 5b. TarF<sub>Bs</sub> (2 μM) was then added to utilize the remaining [¹⁴C]-CDP-glycerol to form compound 6b. 6b was purified using a Phenomenex Strata X 33 μm polymeric reversed phase (30 mg/1 ml) column. To prepare radiolabeled compound 7b, a 30 μl reaction containing 500 nM TagB, 500 nM TarF<sub>Sa</sub>, 5 μM TarK<sub>Bs</sub>, 50 μM of purified 3b, 100 μM of [¹⁴C]-CDP-glycerol, and 100 μM of CDP-glycerol was incubated in 20 mM Tris (pH 7.5), 100 mM NaCl, and 30 mM MgCl₂ for 2 hr at room temperature. The reaction was purified as described for compound 6b. Cold reactions performed simultaneously and analyzed by LC/MS confirmed the formation of compounds 6 and 7.

#### HPLC Analysis of TarK Reactions
For HPLC analysis a Phenomenex Phenosphere SAX 80 A, 250 × 4.6 mm 5 μm strong anion exchange column was used on an Agilent 1100 HPLC at flow rate of 1 ml/min with a linear gradient of 0%–20% B over 15 min (buffer A: 5 mM ammonium acetate [pH 3.8]; buffer B: 750 mM ammonium dihydrogen phosphate [pH 3.7]). The UV signal was monitored at 271 nm. TarK<sub>Bs</sub> (30 μl) reactions contained 200 μM 4a, 200 μM 1 or 2, and 1 μM enzyme in 20 mM Tris (pH 7.5), 100 mM NaCl, and 10mM MgCl₂. After 30 min at room temperature, the reactions were quenched with 30 μl methanol and analyzed by HPLC. Reactions containing heat-treated enzymes served as negative controls.

#### PAGE Analysis of Enzymatic Reactions
To assay TarF<sub>Bs</sub> substrate specificity, reactions (3 μl) contained 500 nM or 5 μM enzyme, 1 μM 4b, and 6 μM of either 2 or [¹⁴C]-1 in a buffer of 100 mM NaCl, 20 mM Tris (pH 7.5), and 10 mM MgCl₂. A reaction was also set up with 2 μM 3b and 2 μM [¹⁴C]-1. Heat-treated enzyme served as a negative control. Reactions were quenched after 3 hr with 3 μl DMF. Two microliters of 4× loading dye (80% glycerol, 0.02% bromophenol blue) was added, and 4 μl of the mixture was loaded on a 7 × 8 × 0.1 cm, 0.25 M TBE/20% acrylamide gel (preparation described previously [Brown et al., 2008]). The gel was electrophoresed at 100 V for 110 min, dried, exposed to a phosphor screen, and analyzed by radiometry.
TarF reactions (3 μl) were performed using 500 nM enzyme (*S. aureus* or *B. subtilis* W23) with 3 μM 4b, 18 μM [¹⁴C]-1, and 18 μM 1 in 100 mM NaCl, 20 mM Tris (pH 7.5), and 10 mM MgCl₂. Reactions were quenched with 3 μl DMF after 30 min or 12 hr at room temperature. Reactions containing the same composition, but with 2 μM enzyme, were allowed to proceed for 12 hr at room temperature. Identical reactions using heat-treated enzyme served as negative controls. Two microliters of 4× loading dye was added, and 8 μl of the mixture was loaded on a 16 × 16 × 0.1 cm 0.25 M TBE/20% acrylamide gel. After running for 260 min at 225 V, the gel was dried, exposed to phosphor screen, and analyzed by radiometry.
TarL<sub>Bs</sub> reactions (3 μl) were performed using 5 μM enzyme with 2 μM 4b, 5b, 6b, or 7b and 200 μM 2. TarK<sub>Bs</sub> reacting in tandem with TarL<sub>Bs</sub> (3 μl reactions) contained 1 μM of each enzyme with 2 μM 4b, 5b, or 6b and 200 μM 2. Reaction buffer was 100 mM NaCl, 20 mM Tris (pH 7.5), and 30 mM MgCl₂. Each substrate combination was also incubated with a heat-treated enzyme to serve as a negative control. After 2 hr at room temperature, the reactions were quenched with 3μl DMF and analyzed as described for TarF<sub>Bs</sub> substrate specificity reactions.

#### Q-TOF LC/MS Analysis
All analyses were performed on an Agilent 6520 Q-TOF LC/MS using a Phenomenex Gemini 5 micron C18 100 A column (50 × 4.6 mm) at a flow rate of 1 ml/min (solvent A: water; solvent B: methanol both with 0.1% ammonium hydroxide as a solvent modifier), with a linear gradient from 0% to 100% B over 10 min. Detailed reaction conditions and purification procedures are located in Supplemental Experimental Procedures.

#### Construction of *S. aureus* *tarF* Deletion Strain
The *tarF* deletion cassette was obtained by assembly PCR of two amplicons encoding approximately 1 kB DNA regions flanking the *tarF* open reading

Chemistry & Biology

Pathway for B. subtilis W23 WTA Biosynthesis

frame (P1tarF-P2tarF and P3tarF-P4tarF), digested with Apal, and ligated into pKOR1 (digested with Apal/EcoRV) to generate pKOR1-223. The deletion vector was introduced into strain JT215 by electroporation ([Schenk and Laddaga, 1992](https://doi.org/10.1016/0092-8674(92)90440-8)), and the *tarF* deletion strain JT230 was subsequently isolated by double crossover as described ([Meredith et al., 2008](https://doi.org/10.1128/JB.01584-07)).

### Construction of B. subtilis W23 tarF Deletion Strain

The marked *tarF* gene deletion in *B. subtilis* W23 was made using a protocol developed for *B. subtilis* 168 gene deletion ([Jarmer et al., 2002](https://doi.org/10.1093/nar/gkf456)). Assembly of the linear fragment, Z, containing the CAT cassette flanked by 1 kB DNA flanking of *B. subtilis* W23 *tarF* is described in [Supplemental Experimental Procedures](#). The *B. subtilis* W23 strain to be transformed was streaked on LB/agar and grown overnight at 37°C. A single colony was used to inoculate a 5 ml culture of *B. subtilis* W23 and grown overnight at 37°C in modified Spizizen minimal salt medium with glutamate ([Jarmer et al., 2002](https://doi.org/10.1093/nar/gkf456)). At OD<sub>450</sub> ~1, 50 μl of cells was diluted into 5 ml of fresh minimal media and grown at 37°C. At OD<sub>450</sub> ~0.6, 200 μl of cells was incubated with at least 2 μg of linear DNAZ. After 2 hr shaking at 37°C, the cells were plated on LB/agar containing 5 μg/ml chloramphenicol and incubated overnight at 37°C. A marked *tarF* gene deletion was confirmed by PCR analysis using primers that anneal outside the deletion region.

### WTA Extraction and Degradation

WTAs were extracted and broken into monomeric units following a combination of previously published protocols ([Bernal et al., 2009](https://doi.org/10.1128/JB.01584-07); [Sadovskaya et al., 2004](https://doi.org/10.1128/JB.186.17.5781-5789.2004); [Tomita et al., 2009](https://doi.org/10.1016/j.jbiotec.2009.01.010); [Wickham et al., 2009](https://doi.org/10.1016/j.jbiotec.2009.01.010)). A detailed protocol can be found in [Supplemental Experimental Procedures](#). Briefly, the cell pellet was disrupted by sonication. Noncovalently bound components were removed from the cell wall by incubation with SDS, and covalently bound protein and nucleic acids were removed by treatment with trypsin, DNase, and RNase. The WTA was extracted by 10% trichloroacetic acid. Following centrifugation to remove peptidoglycan, the WTAs were precipitated in 95% ethanol. To prepare monomeric units, the WTAs were treated with 1 N NaOH at 100°C for 3 hr. The yield was between 5 and 9 mg from 500 ml cultures. The monomeric units were analyzed by Q-TOF MS using direct inject with Solvent A (described previously) at a flow rate of 0.2 ml/min.

### RT-PCR

Three milliliters of *B. subtilis* W23 culture at an OD<sub>600</sub> of ~0.8 was harvested by centrifugation, resuspended in RNALater reagent (QIAGEN), and stored at –20°C until further use. RNA was extracted from the cell pellets using the QIAGEN RNeasy Mini Kit with an initial treatment of 0.3 μg/ml lysostaphin and 100 μg/ml Proteinase K. RQ1 DNase (Promega) was used to degrade DNA. Four hundred nanograms of RNA was reverse transcribed using SuperScript III Reverse Transcriptase (Invitrogen) using random hexamer primers. RNA without reverse transcriptase treatment was used to detect potential DNA contamination. Of this reaction, 0.3–5 μl was used as template for PCR using KOD Hot Start (Novagen). Amplification of a fragment of the 16s rRNA transcript was used as an internal reference to ensure equal amounts of cDNA template in each PCR reaction. RT-PCR was performed in both biological and analytical duplicate. PCR of *B. subtilis* W23 genomic DNA was performed to confirm that the primers were able to amplify the desired fragment of cDNA. PCR products were analyzed by electrophoresis on an ethidium bromide stained 2% agarose gel. The primers used are listed in [Supplemental Experimental Procedures](#).

### SUPPLEMENTAL INFORMATION

Supplemental Information includes Supplemental Experimental Procedures, six figures, and two tables and can be found with this article online at [doi:10.1016/j.chembiol.2010.07.017](https://doi.org/10.1016/j.chembiol.2010.07.017).

### ACKNOWLEDGMENTS

We would like to thank Yuriy Rebets and David Rudner for helpful discussions on gene deletions; Emma Doud for assistance with mass spectrometry analysis; Jennifer Campbell for allowing us to modify her WTA schematic; and Deborah Perlstein for a critical reading of this manuscript. This research was funded by the NIH (1P01AI083214 and 5R01M078477 to S.W., and F3178727 to J.S.).

Received: January 29, 2010  
Revised: May 17, 2010  
Accepted: July 16, 2010  
Published: October 28, 2010  

### REFERENCES

- Badurina, D.S., Zolli-Juran, M., and Brown, E.D. (2003). CTP:glycerol 3-phosphate cytidylyltransferase (TarD) from Staphylococcus aureus catalyzes the cytidylyl transfer via an ordered Bi-Bi reaction mechanism with micromolar K(m) values. Biochim. Biophys. Acta 1646, 196–206.
- Bernal, P., Zloh, M., and Taylor, P.W. (2009). Disruption of D-alanyl esterification of Staphylococcus aureus cell wall teichoic acid by the {beta}-lactam resistance modifier (-)-epicatechin gallate. J. Antimicrob. Chemother. 63, 1156–1162.
- Bhavsar, A.P., Truant, R., and Brown, E.D. (2005). The TagB protein in Bacillus subtilis 168 is an intracellular peripheral membrane protein that can incorporate glycerol phosphate onto a membrane-bound acceptor in vitro. J. Biol. Chem. 280, 36691–36700.
- Bhavsar, A.P., D’Elia, M.A., Sahakian, T.D., and Brown, E.D. (2007). The Amino terminus of Bacillus subtilis TagB possesses separable localization and functional properties. J. Bacteriol. 189, 6816–6823.
- Brown, S., Zhang, Y.H., and Walker, S. (2008). A revised pathway proposed for Staphylococcus aureus wall teichoic acid biosynthesis based on in vitro reconstitution of the intracellular steps. Chem. Biol. 15, 12–21.
- Chen, L., Men, H., Ha, S., Ye, X.Y., Brunner, L., Hu, Y., and Walker, S. (2002). Intrinsic lipid preferences and kinetic mechanism of Escherichia coli MurG. Biochemistry 41, 6824–6833.
- Chen, M.M., Weerapana, E., Ciepichal, E., Stupak, J., Reid, C.W., Swiezewska, E., and Imperiali, B. (2007). Polyisoprenol specificity in the Campylobacter jejuni N-linked glycosylation pathway. Biochemistry 46, 14342–14348.
- D’Elia, M.A., Millar, K.E., Beveridge, T.J., and Brown, E.D. (2006a). Wall teichoic acid polymers are dispensable for cell viability in Bacillus subtilis. J. Bacteriol. 188, 8313–8316.
- D’Elia, M.A., Pereira, M.P., Chung, Y.S., Zhao, W., Chau, A., Kenney, T.J., Sulavik, M.C., Black, T.A., and Brown, E.D. (2006b). Lesions in teichoic acid biosynthesis in Staphylococcus aureus lead to a lethal gain of function in the otherwise dispensable pathway. J. Bacteriol. 188, 4183–4189.
- D’Elia, M.A., Henderson, J.A., Beveridge, T.J., Heinrichs, D.E., and Brown, E.D. (2009). The N-acetylmannosamine transferase catalyzes the first committed step of teichoic acid assembly in Bacillus subtilis and Staphylococcus aureus. J. Bacteriol. 191, 4030–4034.
- Endl, J., Seidl, H.P., Fiedler, F., and Schleifer, K.H. (1983). Chemical composition and structure of cell wall teichoic acids of staphylococci. Arch. Microbiol. 135, 215–223.
- Endl, J., Seidl, P.H., Fiedler, F., and Schleifer, K.H. (1984). Determination of cell wall teichoic acid structure of staphylococci by rapid chemical and serological screening methods. Arch. Microbiol. 137, 272–280.
- Fabretti, F., Theilacker, C., Baldassarri, L., Kaczynski, Z., Kropec, A., Holst, O., and Huebner, J. (2006). Alanine esters of enterococcal lipoteichoic acid play a role in biofilm formation and resistance to antimicrobial peptides. Infect. Immun. 74, 4164–4171.
- Faridmoayer, A., Fentabil, M.A., Haurat, M.F., Yi, W., Woodward, R., Wang, P.G., and Feldman, M.F. (2008). Extreme substrate promiscuity of the Neisseria oligosaccharyl transferase involved in protein O-glycosylation. J. Biol. Chem. 283, 34596–34604.
- Fedtke, I., Mader, D., Kohler, T., Moll, H., Nicholson, G., Biswas, R., Henseler, K., Gotz, F., Zahringer, U., and Peschel, A. (2007). A Staphylococcus aureus ypfP mutant with strongly reduced lipoteichoic acid (LTA) content: LTA governs bacterial surface properties and autolysin activity. Mol. Microbiol. 65, 1078–1091.

Formstone, A., Carballido-Lopez, R., Noirot, P., Errington, J., and Scheffers, D.J. (2008). Localization and interactions of teichoic acid synthetic enzymes in Bacillus subtilis. J. Bacteriol. 190, 1812–1821.

Ginsberg, C., Zhang, Y.H., Yuan, Y., and Walker, S. (2006). In vitro reconstitution of two essential steps in wall teichoic acid biosynthesis. ACS Chem. Biol. 1, 25–28.

Gross, M., Cramton, S.E., Gotz, F., and Peschel, A. (2001). Key role of teichoic acid net charge in Staphylococcus aureus colonization of artificial surfaces. Infect. Immun. 69, 3423–3426.

Grundling, A., and Schneewind, O. (2007). Genes required for glycolipid synthesis and lipoteichoic acid anchoring in Staphylococcus aureus. J. Bacteriol. 189, 2521–2530.

Jarmer, H., Berka, R., Knudsen, S., and Saxild, H.H. (2002). Transcriptome analysis documents induced competence of Bacillus subtilis during nitrogen limiting conditions. FEMS Microbiol. Lett. 206, 197–200.

Lazarevic, V., Abellan, F.X., Moller, S.B., Karamata, D., and Mauel, C. (2002). Comparison of ribitol and glycerol teichoic acid genes in Bacillus subtilis W23 and 168: identical function, similar divergent organization, but different regulation. Microbiology 148, 815–824.

Lovering, A.L., Lin, L.Y., Sewell, E.W., Spreter, T., Brown, E.D., and Strynadka, N.C. (2010). Structure of the bacterial teichoic acid polymerase TagF provides insights into membrane association and catalysis. Nat. Struct. Mol. Biol. 17, 582–589.

May, J.J., Finking, R., Wiegeshoff, F., Weber, T.T., Bandur, N., Koert, U., and Marahiel, M.A. (2005). Inhibition of the D-alanine:D-alanyl carrier protein ligase from Bacillus subtilis increases the bacterium’s susceptibility to antibiotics that target the cell wall. FEBS J. 272, 2993–3003.

May, J.F., Splain, R.A., Brotschi, C., and Kiessling, L.L. (2009). A tethering mechanism for length control in a processive carbohydrate polymerization. Proc. Natl. Acad. Sci. USA 106, 11851–11856.

Men, H., Park, P., Ge, M., and Walker, S. (1998). Substrate synthesis and activity assay for MurG. J. Am. Chem. Soc. 120, 2484–2485.

Meredith, T.C., Swoboda, J.G., and Walker, S. (2008). Late-stage polyribitol phosphate wall teichoic acid biosynthesis in Staphylococcus aureus. J. Bacteriol. 190, 3046–3056.

Nakamura, L.K., Roberts, M.S., and Cohan, F.M. (1999). Relationship of Bacillus subtilis clades associated with strains 168 and W23: a proposal for Bacillus subtilis subsp. subtilis subsp. nov. and Bacillus subtilis subsp. spizizenii subsp. nov. Int. J. Syst. Bacteriol. 49, 1211–1215.

Neuhaus, F.C., and Baddiley, J. (2003). A continuum of anionic charge: structures and functions of D-alanyl-teichoic acids in gram-positive bacteria. Microbiol. Mol. Biol. Rev. 67, 686–723.

Oku, Y., Kurokawa, K., Matsuo, M., Yamada, S., Lee, B.L., and Sekimizu, K. (2009). Pleiotropic roles of polyglycerolphosphate synthase of lipoteichoic acid in growth of Staphylococcus aureus cells. J. Bacteriol. 191, 141–151.

Pereira, M.P., and Brown, E.D. (2004). Bifunctional catalysis by CDP-ribitol synthase: convergent recruitment of reductase and cytidylyltransferase activities in Haemophilus influenzae and Staphylococcus aureus. Biochemistry 43, 11802–11812.

Pereira, M.P., D’Elia, M.A., Troczynska, J., and Brown, E.D. (2008a). Duplication of teichoic acid biosynthetic genes in Staphylococcus aureus leads to functionally redundant poly(ribitol phosphate) polymerases. J. Bacteriol. 190, 5642–5649.

Pereira, M.P., Schertzer, J.W., D’Elia, M.A., Koteva, K.P., Hughes, D.W., Wright, G.D., and Brown, E.D. (2008b). The wall teichoic acid polymerase TagF efficiently synthesizes poly(glycerol phosphate) on the TagB product lipid III. Chembiochem 9, 1385–1390.

Pollack, J.H., and Neuhaus, F.C. (1994). Changes in wall teichoic acid during the rod-sphere transition of Bacillus subtilis 168. J. Bacteriol. 176, 7252–7259.

Sadovskaya, I., Vinogradov, E., Li, J., and Jabbouri, S. (2004). Structural elucidation of the extracellular and cell-wall teichoic acids of Staphylococcus epidermidis RP62A, a reference biofilm-positive strain. Carbohydr. Res. 339, 1467–1473.

Schenk, S., and Laddaga, R.A. (1992). Improved method for electroporation of Staphylococcus aureus. FEMS Microbiol. Lett. 73, 133–138.

Schirner, K., Marles-Wright, J., Lewis, R.J., and Errington, J. (2009). Distinct and essential morphogenic functions for wall- and lipo-teichoic acids in Bacillus subtilis. EMBO J. 28, 830–842.

Sewell, E.W., Pereira, M.P., and Brown, E.D. (2009). The wall teichoic acid polymerase TagF is non-processive in vitro and amenable to study using steady state kinetic analysis. J. Biol. Chem. 284, 21132–21138.

Soldo, B., Lazarevic, V., and Karamata, D. (2002). tagO is involved in the synthesis of all anionic cell-wall polymers in Bacillus subtilis 168. Microbiology 148, 2079–2087.

Swoboda, J.G., Meredith, T.C., Campbell, J., Brown, S., Suzuki, T., Bollenbach, T., Malhowski, A.J., Kishony, R., Gilmore, M.S., and Walker, S. (2009). Discovery of a small molecule that blocks wall teichoic acid biosynthesis in Staphylococcus aureus. ACS Chem. Biol. 4, 875–883.

Swoboda, J.G., Campbell, J., Meredith, T.C., and Walker, S. (2010). Wall teichoic acid function, biosynthesis, and inhibition. Chembiochem 11, 35–45.

Tomita, S., Furihata, K., Nukada, T., Satoh, E., Uchimura, T., and Okada, S. (2009). Structures of two monomeric units of teichoic acid prepared from the cell wall of Lactobacillus plantarum NRIC 1068. Biosci. Biotechnol. Biochem. 73, 530–535.

Vergara-Irigaray, M., Maira-Litran, T., Merino, N., Pier, G.B., Penades, J.R., and Lasa, I. (2008). Wall teichoic acids are dispensable for anchoring the PNAG exopolysaccharide to the Staphylococcus aureus cell surface. Microbiology 154, 865–877.

Vinogradov, E., Sadovskaya, I., Li, J., and Jabbouri, S. (2006). Structural elucidation of the extracellular and cell-wall teichoic acids of Staphylococcus aureus MN8m, a biofilm forming strain. Carbohydr. Res. 341, 738–743.

Ward, J.B. (1981). Teichoic and teichuronic acids: biosynthesis, assembly, and location. Microbiol. Rev. 45, 211–243.

Weidenmaier, C., Kokai-Kun, J.F., Kristian, S.A., Chanturiya, T., Kalbacher, H., Gross, M., Nicholson, G., Neumeister, B., Mond, J.J., and Peschel, A. (2004). Role of teichoic acids in Staphylococcus aureus nasal colonization, a major risk factor in nosocomial infections. Nat. Med. 10, 243–245.

Weidenmaier, C., Peschel, A., Xiong, Y.Q., Kristian, S.A., Dietz, K., Yeaman, M.R., and Bayer, A.S. (2005). Lack of wall teichoic acids in Staphylococcus aureus leads to reduced interactions with endothelial cells and to attenuated virulence in a rabbit model of endocarditis. J. Infect. Dis. 191, 1771–1777.

Wickham, J.R., Halye, J.L., Kashtanov, S., Khandogin, J., and Rice, C.V. (2009). Revisiting magnesium chelation by teichoic acid with phosphorus solid-state NMR and theoretical calculations. J. Phys. Chem. B 113, 2177–2183.

Xia, G., Kohler, T., and Peschel, A. (2009). The wall teichoic acid and lipoteichoic acid polymers of Staphylococcus aureus. Int. J. Med. Microbiol. 300, 148–154.

Ye, X.Y., Lo, M.C., Brunner, L., Walker, D., Kahne, D., and Walker, S. (2001). Better substrates for bacterial transglycosylases. J. Am. Chem. Soc. 123, 3155–3156.

Yokoyama, K., Miyashita, T., Araki, Y., and Ito, E. (1986). Structure and functions of linkage unit intermediates in the biosynthesis of ribitol teichoic acids in Staphylococcus aureus H and Bacillus subtilis W23. Eur. J. Biochem. 161, 479–489.

Zhang, Y.H., Ginsberg, C., Yuan, Y., and Walker, S. (2006). Acceptor substrate selectivity and kinetic mechanism of Bacillus subtilis TagA. Biochemistry 45, 10895–10904.
