
Journal of Steroid Biochemistry & Molecular Biology 129 (2012) 4–14

**Contents lists available at ScienceDirect**

**Journal of Steroid Biochemistry and Molecular Biology**

journal homepage: www.elsevier.com/locate/jsbmb

---

# Steroid degradation in *Comamonas testosteroni*

Masae Horinouchi ${}^{a,*}$, Toshiaki Hayashi ${}^{a}$, Toshiaki Kudo ${}^{a,b}$

${}^{a}$ RIKEN Advanced Science Institute, 2-1 Hirosawa, Wako-shi, Saitama 351-0198, Japan  
${}^{b}$ Faculty of Fisheries, Nagasaki University, 1-14 Bunkyo-machi, Nagasaki-shi, Nagasaki 852-8521, Japan

---

## ARTICLE INFO

**Article history:**  
Received 30 July 2010  
Received in revised form 22 October 2010  
Accepted 30 October 2010  

**Keywords:**  
*Comamonas testosteroni*  
Steroid

---

## ABSTRACT

Steroid degradation by *Comamonas testosteroni* and *Nocardia restrictus* have been intensively studied for the purpose of obtaining materials for steroid drug synthesis. *C. testosteroni* degrades side chains and converts single/double bonds of certain steroid compounds to produce androsta-1,4-diene 3,17-dione or the derivative. Following 9α-hydroxylation leads to aromatization of the A-ring accompanied by cleavage of the B-ring, and aromatized A-ring is hydroxylated at C-4 position, cleaved at Δ4 by meta-cleavage, and divided into 2-hydroxyhexa-2,4-dienoic acid (A-ring) and 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid (B,C,D-ring) by hydrolysis. Reactions and the genes involved in the cleavage and the following degradation of the A-ring are similar to those for bacterial biphenyl degradation, and 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid degradation is suggested to be mainly β-oxidation. Genes involved in A-ring aromatization and degradation form a gene cluster, and the genes involved in β-oxidation of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid also comprise a large cluster of more than 10 genes. The DNA region between these two main steroid degradation gene clusters contain 3α-hydroxysteroid dehydrogenase gene, Δ5,3-ketosteroid isomerase gene, genes for inversion of an α-oriented-hydroxyl group to a β-oriented-hydroxyl group at C-12 position of cholic acid, and genes possibly involved in the degradation of a side chain at C-17 position of cholic acid, indicating this DNA region of more than 100 kb to be a steroid degradation gene hot spot of *C. testosteroni*.  

Article from a special issue on steroids and microorganisms.

© 2010 Elsevier Ltd. All rights reserved.

---

## 1. Brief overview of steroid degradation by *Comamonas testosteroni*

### 1.1. Introduction

Steroid compounds are known for the wide variety and have long been used as medicinal chemicals for their hormonal effects and various other functions on animals. Several species of bacteria degrade certain steroid compounds to utilize as the sole carbon and energy sources, and the intermediate compounds of steroid degradation by these bacteria have been intensively studied for the purpose of obtaining materials for steroid drug synthesis. For these studies, testosterone was often used as representative steroid compound and Gram-negative *Comamonas* (formerly *Pseudomonas*) *testosteroni* and Gram-positive *Rhodococcus* (formerly *Nocardia*) *restrictus* were used as representative steroid degrading bacteria. Among *C. testosteroni*, strain ATCC 1196 has been traditionally used and some other steroid-degrading *C. testosteroni* strains such as ATCC 17410 and TA441 were reported a few decades later when genetic engineering showed explosive progress.

### 1.2. Degradation pathway of steroid compound in *C. testosteroni*

In 1958, it was already revealed that testosterone degradation by *C. testosteroni* starts with conversion of a hydroxyl group at C-17 position to a ketone group and introduction of a double bond at C-1 position of the A-ring, being followed by hydroxylation at C-9 position of the B-ring which leads to aromatization of the A-ring (Fig. 1) [1,2]. Then an intermediate compound with the aromatized A-ring, 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione (3-HSA), and an intermediate compound with the C,D-rings with cleaved B-ring, 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid, were identified in 1965 [3]. Then aromatized A-ring of 3-HSA is cleaved and the compound is divided into two compounds, A-ring and B,C,D-rings, by hydrolysis [1,2,4–12]. The degradation pathway was confirmed with detail using gene-disrupted mutants in early 2000s, with further studies revealing that A-ring is degraded with the pathway similar to a common bacterial aromatic compound degradation pathway and B,C,D-rings is degraded mainly by β-oxidation [13–18]. Investigation of the detail of degradation pathway of B,C,D-ring and other side chains of steroid compounds are still in progress.

---

* Corresponding author. Tel.: +81 48 467 5354; fax: +81 48 462 1357.  
E-mail address: masae@riken.jp (M. Horinouchi).

0960-0760/$ – see front matter © 2010 Elsevier Ltd. All rights reserved.  
doi:10.1016/j.jsbmb.2010.10.008

M. Horinouchi et al. / Journal of Steroid Biochemistry & Molecular Biology 129 (2012) 4–14

Fig. 1. Proposed steroid degradation pathway of *Comamonas testosteroni* TA441. Compound names are presented (R1 = H, R2 = H) unless others indicated: 3-oxo-5β-cholan-24-oic acid, (I); androsta-1,4-diene-3,17-dione or the derivative (R1 = α-OH or H, R2 = α-OH or H), (II); androst-4-ene-3,12,17-trione, (III); androsta-1,4-diene-3,17-dione or the derivative (R1 = β-OH or H, R2 = α-OH or H), (IV); androsta-3,17-dione, (V); androst-5-ene-3,17-dione, (VI); androsta-1,4-diene-3,17-dione or the derivative (R1 = β-OH or H, R2 = α-OH or H), (VII); 9-hydroxyandrosta-1,4-diene-3,17-dione, (VIII); 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione (3-HSA), (IX); 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione, (X); 4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandrosta-1(10),2-diene-4-oic acid, (XI); 2-hydroxyhexa-2,4-dienoic acid, (XII); 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid, (XIII); 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid CoA ester, (XIII). Enzyme names are: 3β,17β-hydroxysteroid dehydrogenase; 3β,17β-hsd, 3α-hydroxysteroid dehydrogenase; 3α-hsd, Δ5,3-ketosteroid isomerase; Δ5-ksi, 12α-hydroxysteroid dehydrogenase; 12α-hsd, 12β, hydrogenase; 12β-ksh, Δ1-dehydrogenase; Δ1-DH, Δ4-dehydrogenase; Δ4-DH. Bold numbers indicate the section numbers in the text where the detail is provided.

### 1.3. Steroid degradation enzymes in C. testosteroni

Although testosterone has been used as the representative steroid compound to reveal steroid degradation in *C. testosteroni*, *C. testosteroni* also degrades several other steroids (e.g., cholic acid and its derivatives, testosterone, progesterone, epiandrosterone, dehydroepiandrosterone) and converts them to androsta-1,4-diene 3,17-dione (ADD) or the derivative corresponding to the initial steroid compound before breaking down steroidal four rings. This suggests *C. testosteroni* has all the enzymes required for degradation of these compounds to ADD or the derivative, and among the predicted enzymes, Δ5,3-ketosteroid isomerase, 3α-dehydrogenase, and 3β-dehydrogenase have long been studied since 1950s [19–21]. These three enzymes are indispensable for degradation of steroid compounds with a double bond at Δ5 (e.g., dehydroepiandrosterone), with an α-oriente-hydroxyl group at C-3 position (e.g., cholic acid, androsterone), and with a β-oriente-hydroxyl group at C-3 position (e.g., epiandrosterone), respectively. Crystal structure analysis and genetic analysis started after around 1990.

### 1.4. Steroid degradation genes and the gene clusters in C. testosteroni

Investigation of steroid degradation genes of in *C. testosteroni* launched fully in 1990s when 3β-dehydrogenase gene, 3α-dehydrogenase gene, and Δ5,3-ketosteroid isomerase gene were isolated consistently, then degradation genes for basic steroidal structure were revealed with detailed degradation pathway in the next decade (Fig. 1, Table 1) [13–18,22–28]. The aromatized A-ring is cleaved and completely degraded by meta-cleavage pathway, a common bacterial degradation pathway for aromatic compounds, and the genes involved in the A-ring degradation show significant homology to the corresponding genes in bacterial aromatic compound degradation. Degradation pathway of intermediate compound of B,C,D-rings, 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid, is still under investigation, but the main part of the degradation is suggested to be β-oxidation. The degradation is like an aggregation that irrelevant reactions just gathered up, but the genes for the reactions are exclusively induced only when *C. testosteroni* is incubated with steroid compounds.

Table 1  
Steroid degradation genes in steroid degradation gene hotspot in Comamomas testosteroni strains.

| ATCC 11996 | ATCC 17410 | TA441 | CNB-2 |
| --- | --- | --- | --- |
| Gene or accession No. | Gene | Gene or ORF | Enzyme/Putative product | GeneID | DNA position (complementary) |
| – | – | Transcriptional regulator, GntR family | 1268 | (1432546–1433037) |
| – | – | Methylcitrate synthase/citrate synthase II | 1269 | (1433047–1434144) |
| – | – | Transcriptional regulator, XRE family | 1270 | 1434429–1435901 |
| – | – | SteA | Dehydrogenase for α-oriented hydroxyl group at C-12 | 1271 | 1436135–1436866 |
| – | – | SteB | Hydrogenase for ketone group at C-12 to β-oriented hydroxyl group | 1272 | 1436879–1437646 |
| – | – | ORF 7 | Hypothetical protein | 1273 | 1437675–1438310 |
| – | – | ORF 6 | NADH:flavin oxidoreductase | 1274 | 1438320–1439402 |
| AAM 77244[83] | – | TesB | Meta-cleavage enzyme for aromatized A-ring | 1275 | 1439744–1440598 |
| – | – | ORF 1 | CoA-transferase alpha subunit | 1276 | 1440615–1441493 |
| – | – | ORF 2 | CoA-transferase beta subunit | 1277 | 1441506–1442276 |
| – | – | ORF 3 | Enoyl-CoA hydratase/isomerase family | 1278 | 1442278–1443180 |
| – | – | ORF 4 | Enoyl-ACP reductase | 1279 | 1443177–1444265 |
| – | – | ORF 5 | Enoyl-CoA hydratase/isomerase family | 1280 | 1444288–1445046 |
| – | – | ORF 21 | Acyl-CoA dehydrogenase | 1281 | 1445075–1446229 |
| – | – | ORF 22 | Acyl-CoA dehydrogenase | 1282 | 1446247–1447386 |
| – | – | ORF 23 | Thiolase | 1283 | 1447383–1448537 |
| – | – | ORF 25 | 6-aminohexanoate-cyclic-dimer hydrolase | 1284 | 1448638–1450203 |
| – | – | ORF 26 | 6-aminohexanoate-cyclic-dimer hydrolase | 1285 | 1450204–1451793 |
| – | – | ORF 27 | Short-chain dehydrogenase/reductase family | 1286 | 1451853–1452728 |
| – | – | ORF 28 | Acyl-CoA dehydrogenase | 1287 | 1452749–1453936 |
| – | – | ORF 30 | Acyl-CoA dehydrogenase | 1288 | 1453933–1455111 |
| ACI 39936** | – | ORF 31 | Short-chain dehydrogenase/reductase family | 1289 | 1455327–1456112 |
| AAV 40816** | – | ORF 32 | MaoC domain protein | 1290 | 1456115–1456600 |
| AAV 40815** | – | ORF 33 | Acyl-CoA thiolase, acetyl-CoA acetyltransferase | 1291 | 1456668–1457939 |
| teiR[63] | – | TesR | LuxR-type transcription factor | 1292 | 1457981–1459228 |
| – | – | Diguanylate cyclase | 1293 | 1459749–1461293 |
| – | – | Carnitine dehydratase/bile acid-inducible protein | 1294 | (1461503–1462636) |
| – | – | (Short-chain dehydrogenase/reductase SDR) | 1295 | 1462821–1463564 |
| – | – | (Acetyl-CoA acetyltransferase) | 1296 | 1463926–1465059 |
| – | – | (3-Hydroxyacyl-CoA dehydrogenase) | 1297 | 1465077–1467236 |
| – | – | (Glucose-methanol-choline oxidoreductase) | 1298 | 1467325–1469031 |
| – | – | Acyl-CoA dehydrogenase domain protein | 1299 | 1469060–1471471 |
| – | – | Major facilitator superfamily MFS_1 | 1300 | 1471575–1472933 |
| – | – | AMP dependent acyl-CoA synthetase and ligase | 1301 | 1473197–1474822 |
| – | (ND) | (Hypothetical protein) | 1302 | (1474924–1475322) |
| – | – | Amidohydrolase 3 | 1303 | (1475610–1477325) |
| – | – | Major facilitator superfamily MFS_1 | 1304 | 1477575–1478918 |
| – | – | Hypothetical protein | 1305 | 1478915–1479643 |
| – | – | Rieske(2Fe-2S) domain | 1306 | (1479819–1480928) |
| – | – | Short chain dehydrogenase/reductase SDR | 1307 | (1481087–1481923) |
| – | – | Major facilitator superfamily MFS_1 | 1308 | (1481945–1483279) |
| – | – | Coniferyl aldehyde dehydrogenase | 1309 | 1483404–1484828 |
| – | – | short chain dehydrogenase/reductase SDR | 1310 | 1485097–1485867 |
| – | – | Transcriptional regulator, LysR family | 1311 | (1486107–1487042) |
| – | – | DSBA oxidoreductase | 1312 | 1487110–1487760 |
| – | (ND) | (Twin-arginine translocation pathway signal) | 1313 | 1487811–1488710 |
| – | – | Ketosteroid isomerase like protein | 1314 | 1488719–1489120 |
| – | – | Hypothetical protein | 1315 | 1489139–1489399 |
| – | – | Streptomycin-3-kinase | 1316 | (1489452–1490153) |
| – | – | Conserved hypothetical protein | 1317 | (1490567–1491058) |
| – | – | GCN 5-related N-acetyltransferase | 1318 | (1491306–1491782) |
| – | – | Protein of unknown function DUF 925 | 1319 | (1491906–1492490) |
| – | – | NADH:flavin oxidoreductase/NADH oxidase | 1320 | (1492713–1494839) |
| – | – | 2, 4-dienoyl-CoA reductase | 1321 | (1494856–1496871) |
| 3α-hsd[38,84]hsdA[85] | – | 3α-hsd | 3-hydroxysteroid 3α-dehydrogenase | 1322 | 1497027–1497800 |
| ksi[24,25] | – | ksi | 3-ketosteroid Δ4(5)-isomerase | 1323 | 1497971–1498330 |
| repA[86] | – | repA | Repressor of 3-hydroxysteroid 3α-dehydrogenase | 1324 | (1496985–1497803) |
| repB[86] | – | ORF 64 | GreA/GreB family elongation factor | 1324 | (1498493–1499410) |
| – | (ND) | (GCN 5-related N-acetyltransferase) | 1325 | (1499815–1500255) |
| – | – | Transcriptional regulator, LysR family | 1326 | (1500300–1501211) |
| – | – | Short-chain dehydrogenase/reductase SDR | 1327 | 1501320–1502087 |
| – | – | N-acyl-D-amino-acid deacylase | – | – |
| – | – | Glutathione-dependent formaldehyde-activating | 1328 | 1502236–1502637 |
| – | – | Enoyl-CoA hydratase/isomerase | 1329 | (1502710–1503522) |
| – | – | Short-chain dehydrogenase/reductase SDR | 1330 | 1503622–1504422 |
| – | – | Protein of unknown function DUF 925 | 1331 | 1504441–1505385 |
| – | – | CaiA, acyl-CoA dehydrogenase-like protein | 1332 | 1505401–1506504 |
| – | – | MaoC-like dehydratase | 1333 | 1506517–1506933 |
| – | – | Lipid-transfer protein | 1334 | 1506957–1508138 |

M. Horinouchi et al./Journal of Steroid Biochemistry & Molecular Biology 129 (2012) 4–14

Table 1 (Continued)

| ATCC 11996 | ATCC 17410 | TA441 | CNB-2 |
| --- | --- | --- | --- |
| Gene or accession No. | Gene | Gene or ORF | Enzyme/Putative product | Gene ID | DNA position (complementary) |
| – | – | CaiA, acyl-CoA dehydrogenase-like protein | 1335 | 1508156–1509337 |
| – | – | Acyl-CoA synthetase | 1336 | 1509380–1511191 |
| – | – | Beta-lactamase domain protein | 1337 | 1511184–1512251 |
| – | – | Acyl-CoA dehydrogenase-like protein | 1338 | 1512264–1513460 |
| – | – | Acyl-CoA dehydrogenase-like protein | 1339 | 1513593–1514720 |
| – | – | Long-chain-fatty-acid-CoA ligase | 1340 | 1514722–1516428 |
| – | – | Putative sigma E regulatory protein, MucB, DUF 1329 | 1341 | 1516711–1518090 |
| – | – | Protein of unknown function DUF 1302 | 1342 | 1518126–1519637 |
| – | – | Dienelactone hydrolase | 1343 | 1519751–1520812 |
| – | – | RND efflux transporter | 1344 | (1520980–1521438) |
| – | – | – | 1345 | (1521438–1523921) |
| – | – | BNR repeat containing glycosyl hydrolase | 1346 | (1523918–1524925) |
| – | – | Nonspecific lipid-transfer protein | 1347 | 1525136–1526350 |
| – | – | Acyl dehydrogenase protein of unknown function | 1348 | 1526523–1527299 |
| – | – | DUF 35 | – | – |
| – | – | (Ferric malleobactin transporter) | 1349 | 1527577–1527855 |
| – | – | TonB-dependent siderophore receptor, transporter | 1350 | 1528207–1530306 |
| – | – | TesG | 4-Hydroxy-2-oxo-hexanoate aldolase | 1351 | (1530465–1531505) |
| – | – | TesF | Propionaldehyde dehydrogenase | 1352 | (1531520–1532443) |
| – | – | TesE | 2-Hydroxyhexa-2,4-dienoic acid hydratase | 1353 | (1532440–1533255) |
| – | – | TesD | Hydratase for aromatized A-ring of steroids | 1354 | (1533260–1534099) |
| – | – | TesA1 | Hydroxylase at C-4 of aromatized A-ring | 1355 | (1534179–1535156) |
| – | – | TesA2 | Hydroxylase at C-4 of aromatized A-ring | 1356 | 1535436–1536632 |
| Δ1-DH [26] | – | TesH | 3-Ketosteroid Δ1-dehydrogenase | 1357 | 1536737–1538467 |
| Δ4-DH [27] | – | TesI | 3-Ketosteroid Δ4-dehydrogenase | 1358 | 1538483–1540219 |
| – | – | ORF 17 | Reductase component of C-9 hydroxylase | 1359 | 1540303–1541370 |
| – | – | ORF 18 | CoA-transferase at C-5 of cleaved B-ring | 1360 | 1541374–1542996 |

(ND): not detected in TA441.

–: not sequenced or sequence incomplete.

Genes of *C. testosteroni* for degradation of basic steroidal structure constitute two main steroid degradation gene clusters; one mainly consists of genes involved in aromatization and cleavage of the A-ring (named tesG-ORF18 in strain TA441) and the other consists of genes involved in β-oxidation of the B,C,D-rings (named steA to tesR in strain TA441) (Table 1). Recent study indicated these two steroid degradation gene clusters are on the same DNA strand several dozens of kilo bases apart and the 3α-dehydrogenase gene and the adjacent Δ5,3-ketosteroid isomerase gene are located in the DNA region between these clusters (Table 1), identified steroid degradation genes of *C. testosteroni* strains are shown with putative genes of *C. testosteroni* CNB-2, whose whole genome determination was completed in 2009 [29]. Quite a few possible steroid degradation genes are found in this DNA region, suggesting that this DNA region of more than 100 kb to be the steroid degradation gene hot spot of *C. testosteroni*.

2. Detail of each reaction of steroid degradation by *Comamonas testosteroni*

2.1. 3β-,17β-Dehydrogenation

In mammals, 3β-hydroxy-Δ5-steroid dehydrogenase catalyzes both dehydrogenation from 3β-hydroxyl group to ketone group as well as transposition of a double bond at Δ5 to Δ4, but in bacterial steroid degradation, the hydroxyl group at C-3 position must be converted to a ketone group before introduction of a double bond into A-ring.

3β,17β-Dehydrogenase (3β,17β-hsd) of *C. testosteroni* was purified in 1970s from strain ATCC 11996. The enzyme was initially isolated as 3β-hsd and the cloned enzyme was revealed to act on both 3β-hydroxyl group and 17β-hydroxyl group of steroid compounds (Fig. 2) [22,30–32]. In 2002, crystallographic analysis at 1.2 Å resolution revealed the enzyme to have nearly identical subunits that form a tetramer with the active site

Fig. 2. Reaction of 3β,17β-hydroxysteroid-dehydrogenase (3β,17β-hsd).

M. Horinouchi et al. / Journal of Steroid Biochemistry & Molecular Biology 129 (2012) 4–14

Fig. 3. 3β,17β-hydroxysteroid dehydrogenase (3β,17β-hsd) and surrounding genes.

Fig. 4. Reaction of 3α-hydroxysteroid-dehydrogenase (3α-hsd).

A gene-disrupted mutant of 3β,17β-hsd of strain TA441 did not show significant growth on epiandrosterone, which has a 3β-hydroxyl group and 17-ketone group, while the gene-disrupted mutant did degrade testosterone, which has a 17β-hydroxyl group and 3-ketone group, though the growth was noticeably slower than that of the strain TA441 control [accepted, Horinouchi 2010]. These results suggested that the main role of the 3β,17β-hsd in TA441 cells is 3β-dehydrogenation and that there is at least one more dehydrogenase which acts on the 17β-hydroxyl group. Dehydrogenation/hydrogenation at the C-17 position is thought to be a reversible reaction because the major intermediate compounds isolated so far from TA441 have a ketone group at the C-17 position and are accompanied by a small amount of a compound of the same structure with the exception of a hydroxyl group at the C-17 position. *C. testosteroni* may have more than one enzyme with 17β-dehydrogenating/hydrogenating activity to deal with intermediate compounds having considerable structural differences.

### 2.2. 3α-Dehydrogenation and negative regulation of 3α-dehydrogenation

Cholic acid and the derivatives, the main integrants of bile acid, and steroidal antibiotic fusidic acid are known for their anti-bacterial effect, but some steroid degrading bacteria including *C. testosteroni* are resistant with the ability of utilizing these compounds [38]. In contrast to testosterone, they have an α-oriented-hydroxyl group at C-3 position. In steroid degradation by *C. testosteroni*, α-oriente-hydroxyl group at C-3 position also has to be converted to a ketone group before introduction of double bonds in the A-ring (Fig. 4).

3α-Dehydrogenase (3α-hsd) was purified in 1965 [39]. Then cloning of 3α-hsd gene using an antibody for 3α-hsd was reported in 1995 [40], but the deduced amino acid sequence at the N-terminal end showed c.a. 35% identity to ribosomal protein L10/P0 and the deduced amino acid sequence of the rest of the putative gene showed c.a. 50% identity to ribosomal protein L7/L12.

Another gene encoding 158 aa-putative 3α-hsd was cloned from strain ATCC11996 in 1998 [23,41]. The deduced amino acid sequence of this 3α-hsd showed considerably high homology to many short-chain dehydrogenase/reductase (SDR) and the motives of 3α-hsd are conserved. The sequence of N-terminal of this 3α-hsd exactly matches to that of the enzyme purified directly from the *C. testosteroni* cells and 3-ketosteroid-Δ4(5)-isomerase (ksi) gene was found in the just downstream DNA region, both being included in steroid degradation hot spot of *C. testosteroni*.

3α-Hsd is induced by LuxR family positive regulator of *C. testosteroni* while genes encoding repressor of 3α-hsd, named RepA and RepB in ATCC11996, are found on the complementary strand of 3α-hsd and ksi (Fig. 5) [42]. In the absence of inducing steroids, RepA blocks transcription of 3α-hsd (named hsdA in ATCC11996) by preventing RNA polymerase to bind the promoter region of 3α-hsd gene. RepB blocks translation of the 3α-hsd/CR mRNA. In the presence of appropriate steroid compounds, the compounds both bind to RepA and RepB and release these proteins from the RNA polymerase and of the 3α-hsd/CR mRNA, respectively [43]. RepB of ATCC11996 is reported as a protein consists from GNAT family acetyltransferase. Homology search using deduced amino acid sequence of the putative RepB of strain TA441 and CNB-2 indicated that they are fusion proteins consisting both with GNAT family acetyltransferase and an elongation factor, but whether fusion protein or not will not affect much to the function.

### 2.3. Δ5,3-Ketosteroid isomerization

In mammals, transfer of a double bond at Δ5 to Δ4 is catalyzed by 3β-hydroxy-Δ5-steroid dehydrogenase at the same time as the dehydroxylation of 3β-hydroxyl group to ketone group, while in *C. testosteroni*, Δ5,3-ketosteroid isomerase (ksi) just transfers a double bond at Δ5 of 3-ketosteroid to Δ4 (Fig. 6). Ksi of *C. testosteroni*

Fig. 6. Reaction of Δ5,3-ketosteroid-isomerase (ksi).

M. Horinouchi et al. / Journal of Steroid Biochemistry & Molecular Biology 129 (2012) 4–14

cholic acid

$\beta$-oxidation pathway

Fig. 7. Predicted degradation pathway of the side chain at C-17 position of cholic acid.

has been under intensive studies since 1950s because of its interesting mechanism and extraordinarily high catalytic efficiency [44]. Partial crystallization of *C. testosteroni* ksi was performed in 1955 [19] and then complete crystallization and analysis was reported in 1960s [45–47]. The amino acid sequence was determined in 1971 and the gene encoding ksi was cloned in 1987 [24,25]. Complete crystal structure was revealed in 1999 [48], and recent study indicated Tyr14, Asp38 and Asp99 or the regions around them are key active site residues in ksi and Tyr14 is also significant as the backbone of ksi [49].

A Δ5,3-ketosteroid isomerase-disrupted mutant of strain TA441 can grow on dehydroepiandrosterone, which has a double bond at Δ5, but cannot grow on epiandrosterone, which lacks a double bond at Δ5, indicating that *C. testosteroni* ksi is responsible for transfer of the double bond from Δ5 to Δ4 and transfer of the double bond by hydrogenation at Δ5 and following dehydrogenation at Δ4 is not possible [37].

### 2.4. Degradation of the side chain at C-17 position of cholic acid

*C. testosteroni* is unable to degrade majority of the steroid compounds with a large side chain at C-17 position such as cholesterol and plant sterols [37], but effectively degrades cholic acid and the derivatives. Cholic acid and the derivatives are not only known for physiological activities on animals and anti-bacterial activity as the main integrants of bile acid [50–59] but also other various functions that are expected to be applied to medical and their materials. Furthermore, they are reported to induce defensive effects in plants [60,61].

Analysis of degradation pathway of the side chain at C-17 position of cholic acid in *C. testosteroni* is still in progress, but intermediate compounds identified so far from *C. testosteroni* incubated with cholic acid suggested that the degradation pathway is similar to those of *Pseudomonas* sp. NCIB 10590 and ATCC 31752 [62,63] (manuscript on *C. testosteroni* is in preparation). Degradation of the side chain at C-17 position starts with removal of carbons at C-24 and C-23 position by β-oxidation followed by oxidative removal of three carbons at C20, C21, and C22 position as a 2-oxo-propionic acid or the derivative, resulting in production of a ketone group at C-17 position (Fig. 7).

Although degradation genes for the side chain at C-17 position of cholic acid in *C. testosteroni* are yet to be isolated, a putative gene whose deduced amino acid sequence shows around 70% identity to acyl-CoA-dehydrogenase CaiA of *Pseudomonas* sp. strain Chol1 [64] was found in the hot spot of steroid degradation genes (indicated with bold letters in Table 1). CaiA is an acyl CoA-dehydrogenase for 7α,12α-dihydroxy-3-oxo-pregna-1,4-diene-20-carboxylate CoA-ester, a CoA-ester intermediate compound in colic acid degradation with two carbons at C-23 and C-24 position eliminated from cholic acid probably by β-oxidation. Moreover, a number of putative β-oxidation genes were found in the neighboring DNA region of the putative CaiA gene, suggesting that genes involved in degradation

M. Horinouchi et al. / Journal of Steroid Biochemistry & Molecular Biology 129 (2012) 4–14

Fig. 8. Chiral transformation of an α-oriented hydroxyl group to a β-oriented hydroxyl group at C-12 position of cholic acid.

![Image Description for Fig. 8]

Fig. 9. Steroid degradation gene cluster consists of the genes mainly for aromatization, cleavage, and degradation of the A-ring.

![Image Description for Fig. 9]

of the side chain at C-17 position locate in surrounding DNA region.

SteA and SteB are encoded in the DNA region upstream of a meta-cleavage enzyme gene tesB for cleavage of the aromatized A-ring (Fig. 9). These genes form a steroid degradation cluster, steA,B, ORF6,7, tesB, and the following ORFs, most of which are involved in B,C,D-ring degradation. Function of ORF6 and 7 is not clear, but they are though to be involved in steroid degradation.

Most of the α-oriented-hydroxyl group at C-7 still remains after the hydrolysis [28].

### 2.5. Inversion of an α-oriente-hydroxyl group to a β-oriente-hydroxyl group at C-12 position of cholic acid

Cholic acid has α-oriented-hydroxyl groups at C-7 position and C-12 position. After the degradation of the side chain at C-17 position to a ketone group, inversion of the α-oriented-hydroxyl group at C-12 position to the β-oriented-hydroxyl group is indispensable for aromatization of A-ring [28,65]. The detailed mechanism is not yet clear, but C-9α position may be sterically-hindered from binding of the 9α-hydroxylase by α-oriented-hydroxyl group at C-12 position. The side chain at C-17 position of cholic acid is degraded to a ketone group to produce an intermediate compound 7α,12α-dihydroxyandrost-1,4-diene-3,17-dione, which is converted to 7α-hydroxyandrost-1,4-diene-3,12,17-trione by a dehydrogenase SteA, followed by converted to 7α,12β-dihydroxyandrost-1,4-diene-3,17-dione by a hydrogenase SteB (Fig. 8). SteA is able to convert 7α-hydroxyandrost-1,4-diene-3,12,17-trione to 7α,12β-dihydroxyandrost-1,4-diene-3,17-dione a little, but the transformation of 7α,12α-dihydroxyandrost-1,4-diene-3,17-dione to 7α,12β-dihydroxyandrost-1,4-diene-3,17-dione is carried out far more effectively when both SteA and SteB are involved together.

### 2.6. Aromatization of the A-ring accompanied by the cleavage of the B-ring, and the following degradation of the aromatized A-ring (steroid degradation gene cluster I)

In bacterial steroid degradation via aromatization of the A-ring, a ketone group at C-3 position and double bonds at Δ1 and Δ4 are indispensable for the aromatization of the A-ring (Fig. 10). Δ1-Dehydrogenase (Δ1-DH) gene was cloned from *C. testosteroni* strain ATCC17410 in 1991 [26], and Δ4-dehydrogenase (Δ4-DH) gene was cloned from the same strain in 1996 [27]. Similar Δ1-DH (TesH) and Δ4-DH (TesI) genes were identified in strain TA441 with genes involved in A-ring degradation to reveal that Δ1-DH and Δ4-DH are adjacent each other and are contained in the same steroid degradation gene cluster, ORF18, ORF17, and tesIHA2A1DEFG. Most of the genes in this cluster are involved in aromatization and following degradation of the A-ring in steroid degradation [14] (Fig. 11).

Fig. 10. Aromatization, cleavage, and degradation of the A-ring in steroid degradation pathway.

![Image Description for Fig. 10]

M. Horinouchi et al. / Journal of Steroid Biochemistry & Molecular Biology 129 (2012) 4–14

![Diagram](https://i.imgur.com/yourimage.png)

Fig. 11. Steroid degradation gene cluster consists of the genes mainly for degradation of B,C,D-rings.

Testosterone degradation is initiated by dehydrogenation of the 17β-hydroxyl group to ketone group to produce 4-androstene-3,17-dione (4-AD), which then undergoes Δ1-dehydrogenation to 1,4-androstadiene-3,17-dione (ADD). Following 9α-hydroxylation results in automatic cleavage of the B-ring and aromatization of the A-ring to produce 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione (3-HSA). ORF17 (Fig. 11), which locates just downstream DNA region, is considered to encode the reductase component of 9α-hydroxylase, because an ORF17-disrupted mutant accumulates ADD when incubated with testosterone and deduced amino acid sequence of ORF17 shows more than 90% identity to flavodoxin reductase family 1 proteins though the gene encoding oxygenase component was not found in this cluster.

C-4of 3-HSA is hydroxylated by TesA1A2 to yield 3,4-dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione (3,4-HSA), which is followed by cleavage between C-4 and C-5 by meta-cleavage enzyme (TesB in strain TA441). Only the meta-cleavage gene is in another main steroid degradation gene cluster (Section 2.7). TesA1 shows the maximum over 90% amino acid sequence identity to hydroxylases and TesA2 shows significant amino acid sequence identity to many acyl-CoA dehydrogenase type 2-like proteins. Considered from the homology search results, hydroxylation at C-4 position could be possible only with TesA1, but gene-disruption experiments showed that both TesA1 and TesA2 are responsible for hydroxylation at C-4 position [16].

3,4-HSA is converted to 4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandorosta-1(10),2-dien-4-oic acid (4,9-DSHA) by the meta-cleavage enzyme, followed by hydrolysis to produce 2-hydroxyhexa-2,4-dienoic acid (A-ring) and 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid (B,C,D-ring) by the hydrolase TesD. 2-Hydroxyhexa-2,4-dienoic acid is degraded by TesEGF with a process similar to bacterial degradation of aromatic compounds such as biphenyl (Fig. 10). TesB,D,EFG, involved in the aromatized A-ring degradation, also shows 40–80% amino acid identity to corresponding enzymes in bacterial aromatic compound degradation, but the induction experiments showed that these enzymes are exclusively for steroid degradation.

Another product of hydrolysis of 4,9-DSHA, 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid (B,C,D-ring), is accumulated by ORF18-disrupted mutant incubated with testosterone and the homology search indicated ORF18-encoded enzyme to be CoA-transferase, suggesting that the degradation of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid is initiated by addition of CoA by ORF18-encoded CoA-transferase [28].

ORF18, 17, tesIHA2A1DEFG and the downstream DNA region of tesG are in more than 100 kb steroid degradation hotspot (Table 1), while putative genes in the downstream DNA region of ORF18 are not considered to be involved in steroid degradation because of the homology and little influence of the gene-disruption on steroid degradation [28].

---

![Diagram](https://i.imgur.com/yourimage2.png)

Fig. 12. Steroid degradation genes of Comamonas testosteronei TA441 (accession: BAB15810) [17] with possible steroid degradation genes on the database which were suggested by homology search using steroid degradation genes of TA441. The selected bacteria are; C. testosteronei CNB-1 [29], C. testosteronei ATCC19966 (accession: AAM77214, ACI39936, FJ215323, AAV40815) [67,87], C. testosteronei KF1 (draft sequence, accession: NZ_AAUJ02000001) [70,71], Cupriavidus necator (formally Ralstonia eutropha) JMP134 (accession: NC_007347) [72,73], R. eutropha H16 (accession: NC_008314) [74], Burkholderia cenocepacia J2315 (accession: NC_011001) [75], Burkholderia sp. 383 (accession: NC_007511) [76], Cu. taiwanensis LMG 19424 (accession: NC_010530) [77], Shewanella pealeana ATCC700345 (accession: NC_009901), S. halifaxensis HAW-EB4 (accession: NC_010334) [78,79], and Pseudoalteromonas haloplanktis TAC125 (accession: NC_007481) [80,81]. Arrows indicate genes or ORFs, and striped arrows indicate that the genes or ORFs are probably not involved in steroid degradation. Numbers (%) below the arrows indicate putative amino acid sequences identity to the corresponding ORFs of TA441. Corresponding genes and ORFs are connected with broken lines. Putative function is given below the ORFs when we thought it helpful.

2.7. β-oxidation of B,C,D-rings and positive regulator of steroid degradation genes (steroid degradation gene cluster II)

Degradation of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid starts with addition of CoA by ORF18-encoded enzyme. The detail of further degradation is not revealed yet, but the analysis of intermediate compounds accumulated by gene-disrupted mutants of strain TA441 suggested it to be mainly β-oxidation involving ORF1-33 on another main steroid degradation gene cluster steA to tesR (Fig. 9 and Table 1); steA,B,ORF7,6, tesB, ORF1–5, 21–23, 27, 28, 30–33, and tesR; tesR in strain TA441 and teiR in strain ATCC11996) (manuscript in preparation). Ongoing studies imply that degradation of C,D-rings of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid in *C. testosteroni* has significant difference to the proposed degradation pathway in *N. restrictus* [66], while degradation of testosterone to 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid are almost identical both in *C. testosteroni* and *N. restrictus*. Most of the ORFs1 to 33 are considered to be involved in 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid degradation because the gene-disrupted mutant accumulate putative intermediate compounds. ORF25 and ORF26 are not considered to be involved in the degradation due to the high homology of the deduced amino acid sequences to 6-aminohexanoate-cyclic-dimer hydrolases and the negligible influence of the gene-disruption on steroid degradation.

The gene which locates at the very end of this steroid degradation gene cluster, tesR in strain TA441 and teiR in strain ATCC11996, encodes transcriptional regulator indispensable for induction of most of the identified steroid degradation genes in *C. testosteroni* [17,67–69]. This transcriptional regulator is a LuxR family protein and possesses helix-turn-helix motif at the C-terminal to bind DNA. It is an activator which induces 3β,17β-hsd, 3α-hsd, ksi, TesA2 to ORF18-encoded enzyme, TesA1 to TesG, SteA to ORF6-encoded enzyme, the meta-cleavage enzyme and the ORFs in the downstream DNA region [17,67] when *C. testosteroni* strains are incubated with degradable steroids. Detail of induction mechanism and the induction substrate is not clear, but at least steroidal four rings are suggested to be indispensable for induction of these steroid degradation genes because steroid degradation genes are not induced with 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione, an intermediate compound with cleaved B-ring (unpublished data).

Homology search for putative steroid degradation genes in the genomic sequence data on the web using deduced amino acid sequences of tesB to ORF33 revealed the presence of putative steroid degradation gene clusters similar to tesB to ORF33 in bacteria of different genera. Among a number of candidate bacteria, ten strains of seven genera were selected for comparison (*C. testosteroni* CNB-1 [29], *C. testosteroni* KF1 (draft sequence, accession: NZ_AAUJ02000001) [70,71], *Cupriavidu necator* (formally *R. eutropha*) JMP134 (accession: NC_007347) [72,73], *Ralstonia eutropha* H16 (accession: NC_008314) [74], *Burkholderia cenocepacia* J2315 (accession: NC_011001) [75], *Burkholderia sp.* 383 (accession: NC_007511) [76], *Cu. taiwanensis* LMG 19424 (accession: NC_010530) [77], *Shewanella pealeana* ATCC700345 (accession: NC_009901), *S. halifaxensis* HAW-EB4 (accession: NC_010334) [78,79], and *Pseudoalteromonas haloplanktis* TAC125 (accession: NC_007481) [80,81]), whose deduced amino acid sequence of putative steroid degradation genes correspond to tesB to ORF33 are showing higher homology than others (Fig. 12). All the selected strains are Gram-negative bacteria. Putative meta-cleavage enzyme gene and putative ORFs correspond to ORF1 to ORF33 except for ORF25 and ORF26 of strain TA441 are found in all the candidate steroid-degradation gene clusters of these bacteria, while genes not necessary for degradation of basic steroidal four rings, putative 6-aminohexanoate-cyclic-dimer hydrolase gene

(ORF25 and ORF26), the regulator gene (tesR/teiR), and the genes correspond to steA-ORF6, are found only in *C. testosteroni* strains but are often missing in bacteria of other genera. In seven strains out of 10, the gene order of genes correspond to tesB to ORF33 are quite similar to those of TA441, while those in *S. pealeana* ATCC700345, *S. halifaxensis* HAW-EB4, and *P. haloplanktis* TAC125 are very different, implying the considerable variety of bacterial steroid degradation genes even among Gram-negative bacteria. Between steroid-degrading Gram-positive bacteria and Gram-negative bacteria, corresponding steroid degradation enzymes share homology on some level, but gene localization is completely different [82–86]. Steroid degradation pathway and the genes in *C. testosteroni* are thought to be a common type of steroid degradation in Gram-negative bacteria. Further studies are expected for more information on bacterial steroid degradation.

### References

[1] R.M. Dodson, R.D. Muir, Microbiological transformations. II. Microbiological aromatization of steroids, J. Am. Chem. Soc. 80 (1958) 5004–5005.

[2] R.M. Dodson, R.D. Muir, Microbiological transformations. III. The hydroxylation of steroids at C-9, J. Am. Chem. Soc. 80 (1958) 6148.

[3] D.A. Shaw, L.F. Borkenhagen, P. Talalay, Enzymatic oxidation of steroids by cell-free extracts of *Pseudomonas testosteroni*: isolation of cleavage products of ring A, Proc. Natl. Acad. Sci. U.S.A. 54 (1965) 837–844.

[4] R.M. Dodson, R.D. Muir, Microbiological transformations. IV. The microbiological aromatization of steroids, J. Am. Chem. Soc. 83 (1961) 4627–4631.

[5] C.J. Sih, Mechanisms of steroid oxidation by microorganisms, Biochim. Biophys. Acta 62 (1962) 541–547.

[6] C.J. Sih, K.C. Wang, Mechanisms of steroid oxidation by microorganisms. II. Isolation and characterization of 3αα-H-4α-[3′-propionic acid]-7αβ-methylhezahydro-1,5-indanedione, J. Am. Chem. Soc. 85 (1963) 2135–2137.

[7] K.C. Wang, C.J. Sih, Mechanisms of steroid oxidation by microorganisms. IV. Seco intermediates, Biochemistry 2 (1963) 1238–1243.

[8] C.J. Sih, S.S. Lee, Y.K. Tsong, K.C. Wang, 3,4-Dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione. An intermediate in the microbiological degradation of ring A of androst-4-ene-3,17-dione, J. Am. Chem. Soc. 87 (1965) 1385–1386.

[9] C.J. Sih, S.S. Lee, Y.K. Tsong, K.C. Wang, Mechanisms of steroid oxidation by microorganisms. VIII. 3,4-Dihydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione, an intermediate in the microbiological degradation of ring A of androst-4-ene-3,17-dione, J. Biol. Chem. 241 (1966) 540–550.

[10] A.W. Coulter, P. Talalay, Studies on the microbial degradation of steroid ring A, J. Biol. Chem. 243 (1968) 3238–3247.

[11] S. Hayakawa, S. Hashimoto, (+)-(5R)-Methyl-4-oxo-octane-1,8-dioic Acid, microbiological degradation product from rings C and D of cholic acid, Biochem. J. 112 (1969) 127–128.

[12] R.A. Leppik, R.J. Park, M.G. Smith, Aerobic catabolism of bile acids, Appl. Environ. Microbiol. 44 (1982) 771–776.

[13] M. Horinouchi, T. Yamamoto, K. Taguchi, H. Arai, T. Kudo, Meta-cleavage enzyme gene tesB is necessary for testosterone degradation in *Comamonas testosteroni* TA441, Microbiology 147 (2001) 3367–3375.

[14] M. Horinouchi, T. Hayashi, T. Yamamoto, T. Kudo, A new bacterial steroid degradation gene cluster which consists of aromatic compound degradation genes for seco-steroids and 3-ketosteroid dehydrogenase genes in *Comamonas testosteroni* TA441, Appl. Environ. Microbiol. 69 (2003) 4421–4430.

[15] M. Horinouchi, T. Hayashi, H. Koshino, T. Yamamoto, T. Kudo, Gene encoding the hydrolase for the product of the meta-cleavage reaction in testosterone degradation by *Comamonas testosteroni*, Appl. Environ. Microbiol. 69 (2003) 2139–2152.

[16] M. Horinouchi, T. Hayashi, T. Kudo, The genes encoding the hydroxylase of 3-hydroxy-9,10-secoandrosta-1,3,5(10)-triene-9,17-dione in steroid degradation in *Comamonas testosteroni* TA441, J. Steroid Biochem. Mol. Biol. 92 (2004) 143–154.

[17] M. Horinouchi, T. Kurita, T. Yamamoto, E. Hatori, T. Hayashi, T. Kudo, Steroid degradation gene cluster of *Comamonas testosteroni* consisting of 18 putative genes from meta-cleavage enzyme gene tesB to regulator gene tesR, Biochem. Biophys. Res. Commun. 324 (2004) 597–604.

[18] M. Horinouchi, T. Hayashi, T. Kurita, T. Kudo, Identification of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid, 4-hydroxy-2-oxohexanoic acid, and 2-hydroxyhexa-2,4-dienoic acid and related enzymes involved in testosterone degradation in *Comamonas testosteroni* TA441, Appl. Environ. Microbiol. 71 (2005) 5275–5281.

[19] P. Talalay, V.S. Wang, Enzymic isomerization of delta5-3-ketosteroids, Biochim. Biophys. Acta 18 (1955) 300–301.

[20] P. Talalay, P.I. Marcus, Enzymatic formation of 3 alpha-hydroxysteroids, Nature 173 (1954) 1189–1190.

[21] P. Talalay, M.M. Dobson, D.F. Tapley, Oxidative degradation of testosterone by adaptive enzymes, Nature 170 (1952) 620–621.

[22] S. Genti-Raimondi, M.E. Tolmasky, L.C. Patrito, A. Flury, L.A. Actis, Molecular cloning and expression of the β-hydroxysteroid dehydrogenase gene from *Pseudomonas testosteroni*, Gene 105 (1991) 43–49.

[23] E. Mobus, E. Maser, Molecular cloning, overexpression, and characterization of steroid-inducible 3α-hydroxysteroid dehydrogenase/carbonyl reductase from *Comamonas testosteroni*. A novel member of the short-chain dehydrogenase/reductase superfamily, J. Biol. Chem. 273 (1998) 30888–30896.

[24] A. Kuliopulos, D. Shortle, P. Talalay, Isolation and sequencing of the gene encoding Δ5-3-ketosteroid isomerase of *Pseudomonas testosteroni*: overexpression of the protein, Proc. Natl. Acad. Sci. U.S.A. 84 (1987) 8893–8897.

[25] K.Y. Choi, W.F. Benisek, Cloning of the gene for Δ5-3-ketosteroid isomerase from *Pseudomonas testosteroni*, Gene 58 (1987) 257–264.

[26] P. Plesiat, M. Grandguillot, S. Harayama, S. Vragar, Y. Michel-Briand, Cloning, sequencing, and expression of the *Pseudomonas testosteroni* gene encoding 3-oxosteroid Δ1-dehydrogenase, J. Bacteriol. 173 (1991) 7219–7227.

[27] C. Florin, T. Kohler, M. Grandguillot, P. Plesiat, *Comamonas testosteroni* 3-ketosteroid-Δ4(5α)-dehydrogenase: gene and protein characterization, J. Bacteriol. 178 (1996) 3322–3330.

[28] M. Horinouchi, T. Hayashi, H. Koshino, T. Kudo, ORF18-disrupted mutant of *Comamonas testosteroni* TA441 accumulates significant amounts of 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid and its derivatives after incubation with steroids, J. Steroid Biochem. Mol. Biol. 101 (2006) 78–84.

[29] Y.F. Ma, Y. Zhang, J.Y. Zhang, D.W. Chen, Y. Zhu, H. Zheng, S.Y. Wang, C.Y. Jiang, G.P. Zhao, S.J. Liu, The complete genome of *Comamonas testosteroni* reveals its genetic adaptations to changing environments, Appl. Environ. Microbiol. 75 (2009) 6812–6819.

[30] R.M. Schultz, E.V. Groman, L.L. Engel, 3(17)β-Hydroxysteroid dehydrogenase of *Pseudomonas testosteroni*. A convenient purification and demonstration of multiple molecular forms, J. Biol. Chem. 252 (1977) 3775–3783.

[31] P. Minard, M.D. Legoy, D. Thomas, 3β, 17β-Hydroxysteroid dehydrogenase of *Pseudomonas testosteroni*. Kinetic evidence for the bifunctional activity at a common catalytic site, FEBS Lett. 188 (1985) 85–90.

[32] S.J. Yin, N. Vagelopoulos, G. Lundquist, H. Joernvall, *Pseudomonas* 3β-hydroxysteroid dehydrogenase. Primary structure and relationships to other steroid dehydrogenases, Eur. J. Biochem. 197 (1991) 359–365.

[33] J. Benach, C. Filling, U.C.T. Oppermann, P. Roversi, G. Bricogne, K.D. Berndt, H. Jornvall, R. Ladenstein, Structure of bacterial 3β/17β-hydroxysteroid dehydrogenase at 1.2 Å resolution: a model for multiple steroid recognition, Biochemistry 41 (2002) 14659–14668.

[34] J.E. Cabrera, G. Panzetta-Dutari, J.L. Pruneda, S. Genti-Raimondi, A new *Comamonas testosteroni* steroid-inducible gene: cloning and sequence analysis, J. Steroid Biochem. Mol. Biol. 63 (1997) 91–98.

[35] J.E. Cabrera, J.L. Pruneda Paz, S. Genti-Raimondi, Steroid-inducible transcription of the 3β/17β-hydroxysteroid dehydrogenase gene (3β/17β-hsd) in *Comamonas testosteroni*, J. Steroid Biochem. Mol. Biol. 73 (2000) 147–152.

[36] J.L. Pruneda-Paz, M. Linares, J.E. Cabrera, S. Genti-Raimondi, Identification of a novel steroid inducible gene associated with the beta hsd locus of *Comamonas testosteroni*, J. Steroid Biochem. Mol. Biol. 88 (2004) 91–100.

[37] M. Horinouchi, T. Kurita, T. Hayashia, T. Kudo, Steroid degradation genes in *Comamonas testosteroni* TA441: Isolation of genes encoding a Δ4(5)-isomerase and 3α- and 3β-dehydrogenases and evidence for a 100 kb steroid degradation gene hot spot, J. Steroid Biochem. Mol. Biol. 122 (2010) 253–263.

[38] U.C.T. Oppermann, I. Belai, E. Maser, Antibiotic resistance and enhanced insecticide catabolism as consequences of steroid induction in the Gram-negative bacterium *Comamonas testosteroni*, J. Steroid Biochem. Mol. Biol. 58 (1996) 217–223.

[39] P.I. Marcus, P. Talalay, Induction and purification of alpha- and beta-hydroxysteroid dehydrogenases, J. Biol. Chem. 218 (1956) 661–674.

[40] J.H. Abalain, S. Di Stefano, M.L. Abalain-Colloc, H.H. Floch, Cloning, sequencing and expression of *Pseudomonas testosteroni* gene encoding 3-alpha-hydroxysteroid dehydrogenase, J. Steroid Biochem. Mol. Biol. 55 (1995) 233–238.

[41] U.C.T. Oppermann, E. Maser, Characterization of a 3α-hydroxysteroid dehydrogenase/carbonyl reductase from the gram-negative bacterium *Comamonas testosteroni*, Eur. J. Biochem. 241 (1996) 744–749.

[42] G. Xiong, E. Maser, Regulation of the steroid-inducible 3α-hydroxysteroid dehydrogenase/carbonyl reductase gene in *Comamonas testosteroni*, J. Biol. Chem. 276 (2001) 9961–9970.

[43] G. Xiong, H.J. Martin, E. Maser, Identification and characterization of a novel translational repressor of the steroid-inducible 3α-hydroxysteroid dehydrogenase/carbonyl reductase gene in *Comamonas testosteroni*, J. Biol. Chem. 278 (2003) 47400–47407.

[44] F.H. Batzold, A.M. Benson, D.F. Covey, C.H. Robinson, P. Talalay, The delta 5-3-ketosteroid isomerase reaction: catalytic mechanism, specificity and inhibition, Adv. Enzyme Regul. 14 (1976) 243–267.

[45] F.S. Kawahara, P. Talalay, Crystalline delta 5-3-ketosteroid isomerase, J. Biol. Chem. 235 (1960) 1–2.

[46] F.S. Kawahara, S.F. Wang, P. Talalay, The preparation and properties of crystalline delta5-3-ketosteroid isomerase, J. Biol. Chem. 237 (1962) 1500–1506.

[47] P. Talalay, J. Boyer, Preparation of crystalline Δ-5-3-ketosteroid isomerase from *Pseudomonas testosteroni*, Biochim. Biophys. Acta 105 (1965) 389–392.

[48] H.S. Cho, N.C. Ha, G. Choi, H.J. Kim, D. Lee, K.S. Oh, K.S. Kim, W. Lee, K.Y. Choi, B.H. Oh, Crystal structure of Δ(5)-3-ketosteroid isomerase from *Pseudomonas testosteroni* in complex with equilenin settles the correct hydrogen bonding scheme for transition state stabilization, J. Biol. Chem. 274 (1999) 32863–32868.

[49] H.J. Lee, Y.J. Yoon, S. Jang do, C. Kim, H.J. Cha, B.H. Hong, K.Y. Choi, H.C. Lee, 15N NMR relaxation studies of Y14F mutant of ketosteroid isomerase: the influence of mutation on backbone mobility, J. Biochem. 144 (2008) 159–166.

[50] A.F. Hofmann, Bile acids, diarrhea, and antibiotics: data, speculation, and a unifying hypothesis, J. Infect. Dis. 135 (Suppl.) (1977) S126–S132.

[51] M. Podda, C. Ghezzi, P.M. Battezzati, E. Bertolini, A. Crosignani, M.L. Petroni, M. Zuin, Ursodeoxycholic acid for chronic liver diseases, J. Clin. Gastroenterol. 2 (10 Suppl.) (1988) S25–S31.

[52] M. Podda, M. Zuin, P.M. Battezzati, C. Ghezzi, M.L. Dioguardi, Efficacy and safety of a combination of chenodeoxycholic acid and ursodeoxycholic acid for gallstone dissolution: a comparison with ursodeoxycholic acid alone, Gastroenterology 96 (1989) 222–229.

[53] M. Podda, C. Ghezzi, P.M. Battezzati, A. Crosignani, M. Zuin, A. Roda, Effects of ursodeoxycholic acid and taurine on serum liver enzymes and bile acids in chronic hepatitis, Gastroenterology 98 (1990) 1044–1050.

[54] G. Mazzella, P. Parini, F. Bazzoli, N. Villanova, D. Festi, R. Aldini, A. Roda, A. Cipolla, C. Polimeni, D. Tonelli, et al., Ursodeoxycholic acid administration on bile acid metabolism in patients with early stages of primary biliary cirrhosis, Dig. Dis. Sci. 38 (1993) 896–902.

[55] S. Gul dutuna, M. Leuschner, N. Wunderlich, A. Nickel, S. Bhatti, K. Hubner, U. Leuschner, Cholic acid and ursodeoxycholic acid therapy in primary biliary cirrhosis. Changes in bile acid patterns and their correlation with liver function, Eur. J. Clin. Pharmacol. 45 (1993) 221–225.

[56] U. Leuschner, Ursodeoxycholic acid therapy in primary biliary cirrhosis, Scand. J. Gastroenterol. Suppl. 204 (1994) 40–46.

[57] E. Roda, A. Cipolla, A. Salzetta, S. Marchetto, A. Pezzoli, E. Accogli, V. Novelli, C. Polimeni, C. Cerre, G. Mazzella, Influence of ursodeoxycholic acid on biliary lipids, Scand. J. Gastroenterol. Suppl. 204 (1994) 16–18.

[58] F.M. Konikoff, T. Gilat, Effects of fatty acid bile acid conjugates (FABACs) on biliary lithogenesis: potential consequences for non-surgical treatment of gallstones, Curr. Drug Targets Immune. Endocr. Metabol. Disord. 5 (2005) 171–175.

[59] M. Mikov, J.P. Fawcett, K. Kuhajda, S. Kevresan, Pharmacology of bile acids and their derivatives: absorption promoters and therapeutic agents, Eur. J. Drug Metab. Pharmacokin. 31 (2006) 237–251.

[60] J. Koga, H. Kubota, S. Gomi, K. Umemura, M. Ohnishi, T. Kono, Cholic acid, a bile acid elicitor of hypersensitive cell death, pathogenesis-related protein synthesis, and phytoalexin accumulation in rice, Plant Physiol. 140 (2006) 1475–1483.

[61] T Shimizu, Y. Jikumaru, A. Okada, K. Okada, J. Koga, K. Umemura, E. Minami, N. Shibuya, M. Hasegawa, O. Kodama, H. Nojiri, H. Yamane, Effects of a bile acid elicitor, cholic acid, on the biosynthesis of diterpenoid phytoalexins in suspension-cultured rice cells, Phytochemistry 69 (2008) 973–981.

[62] M.E. Tenneson, J.D. Baty, R.F. Bilton, A.N. Mason, The degradation of cholic acid by *Pseudomonas* sp. N.C.I.B. 10590, Biochem. J. 184 (1979) 613–618.

[63] R.J. Park, The major neutral products of the aerobic catabolism of cattle bile by *Pseudomonas* sp. ATCC 31752, Steroids 38 (1981) 383–395.

[64] A. Birkenmaier, J. Holert, H. Erdbrink, H.M. Moeller, A. Friemel, R. Schoenenberger, M.J. Suter, J. Klebensberger, B. Philipp, Biochemical and genetic investigation of initial reactions in aerobic degradation of the bile acid cholate in *Pseudomonas* sp. strain Chol1, J. Bacteriol. 189 (2007) 7165–7173.

[65] M. Horinouchi, T. Hayashi, H. Koshino, M. Malon, T. Yamamoto, T. Kudo, Identification of genes involved in inversion of stereochemistry of a C-12 hydroxyl group in the catabolism of cholic acid by *Comamonas testosteroni* TA441, J. Bacteriol. 190 (2008) 5545–5554.

[66] S.S. Lee, C.J. Sih, Mechanisms of steroid oxidation by microorganisms. XII. Metabolism of hexahydroindanpropionic acid derivatives, Biochemistry 6 (1967) 1395–1403.

[67] J.L. Pruneda-Paz, M. Linares, J.E. Cabrera, S. Genti-Raimondi, TeiR, a LuxR-type transcription factor required for testosterone degradation in *Comamonas testosteroni*, J. Bacteriol. 186 (2004) 1430–1437.

[68] A. Gohler, G. Xiong, S. Paulsen, G. Trentmann, E. Maser, Testosterone-inducible regulator is a kinase that drives steroid sensing and metabolism in *Comamonas testosteroni*, J. Biol. Chem. 283 (2008) 17380–17390.

[69] M. Linares, J.L. Pruneda-Paz, L. Reyna, S. Genti-Raimondi, Regulation of testosterone degradation in *Comamonas testosteroni*, J. Steroid Biochem. Mol. Biol. 112 (2008) 145–150.

[70] V. Rosch, K. Denger, D. Schleheck, T.H. Smits, A.M. Cook, Different bacterial strategies to degrade taurocholate, Arch. Microbiol. 190 (2008) 11–18.

[71] D. Schleheck, T.P. Knepper, K. Fischer, A.M. Cook, Mineralization of individual congeners of linear alkylbenzenesulfonate by defined pairs of heterotrophic bacteria, Appl. Environ. Microbiol. 70 (2004) 4053–4063.

[72] M.A. Sanchez, B. Gonzalez, Genetic characterization of 2,4,6-trichlorophenol degradation in *Cupriavidus necator* JMP134, Appl. Environ. Microbiol. 73 (2007) 2769–2776.

[73] .H. Leveau, J.R. van der Meer, The *tfdR* gene product can successfully take over the role of the insertion element-inactivated TfdT protein as a transcriptional activator of the *tfdCDEF* gene cluster, which encodes chlorocatechol degradation in *Ralstonia eutropha* JMP134(pJP4), J. Bacteriol. 178 (1996) 6824–6832.

[74] A. Pohlmann, W.F. Fricke, F. Reinecke, B. Kusian, H. Liesegang, R. Cramm, T. Eitinger, C. Ewering, M. Potter, E. Schwartz, A. Strittmatter, I. Voss, G. Gottschalk, A. Steinbuchel, B. Friedrich, B. Bowien, Genome sequence of the bioplastic-producing “Knallgas” bacterium *Ralstonia eutropha* H16, Nat. Biotechnol. 24 (2006) 1257–1262.

[75] M.T. Holden, H.M. Seth-Smith, L.C. Crossman, M. Sebaihia, S.D. Bentley, A.M. Cerdeno-Tarraga, N.R. Thomson, N. Bason, M.A. Quail, S. Sharp, I. Cherevach, C. Churcher, I. Goodhead, H. Hauser, N. Holroyd, K. Mungall, P. Scott, D. Walker, B. White, H. Rose, P. Iversen, D. Mil-Homens, E.P. Rocha, A.M. Fialho, A. Baldwin, C. Dowson, B.G. Barrell, J.R. Govan, P. Vandamme, C.A. Hart, E. Mahenthiralingam, J. Parkhill, The genome of *Burkholderia cenocepacia* J2315, an epidemic pathogen of cystic fibrosis patients, J. Bacteriol. 191 (2009) 261–277.

[76] J.H. Leveau, S. Gerards, Discovery of a bacterial gene cluster for catabolism of the plant hormone indole-3-acetic acid, FEMS Microbiol. Ecol. 65 (2008) 238–250.

[77] W.M. Chen, C.H. Wu, E.K. James, J.S. Chang, Metal biosorption capability of *Cupriavidus taiwanensis* and its effects on heavy metal removal by nodulated *Mimosa pudica*, J. Hazard Mater. 151 (2008) 364–371.

[78] J.S. Zhao, D. Manno, J. Hawari, Regulation of hexahydro-1,3,5-trinitro-1,3,5-triazine (RDX) metabolism in *Shewanella halifaxensis* HAW-EB4 by terminal electron acceptor and involvement of c-type cytochrome, Microbiology 154 (2008) 1026–1037.

[79] J.S. Zhao, D. Manno, C. Leggiadro, D. O'Neil, J. Hawari, *Shewanella halifaxensis* sp. nov., a novel obligately respiratory and denitrifying psychrophile, Int. J. Syst. Evol. Microbiol. 56 (2006) 205–212.

[80] R. Papa, E. Parrilli, G. Sannia, Engineered marine Antarctic bacterium *Pseudoalteromonas haloplanktis* TAC125: a promising micro-organism for the bioremediation of aromatic compounds, J. Appl. Microbiol. 106 (2009) 49–56.

[81] C. Medigue, E. Krin, G. Pascal, V. Barbe, A. Bernsel, P.N. Bertin, F. Cheung, S. Cruveiller, S. D'Amico, A. Duilio, G. Fang, G. Feller, C. Ho, S. Mangenot, G. Marino, J. Nilsson, E. Parrilli, E.P. Rocha, Z. Rouy, A. Sekowska, M.L. Tutino, D. Vallenet, G. von Heijne, A. Danchin, Coping with cold: the genome of the versatile marine Antarctica bacterium *Pseudoalteromonas haloplanktis* TAC125, Genome Res. 15 (2005) 1325–13235.

[82] R. van Der Geize, G.I. Hessels, R. van Gerwen, J.W. Vrijbloed, P. van Der Meijden, L. Dijkhuizen, Targeted disruption of the *kstD* gene encoding a 3-ketosteroid Δ(1)-dehydrogenase isoenzyme of *Rhodococcus erythropolis* strain SQ1, Appl. Environ. Microbiol. 66 (2000) 2029–2036.

[83] R. van der Geize, G.I. Hessels, R. van Gerwen, P. van der Meijden, L. Dijkhuizen, Unmarked gene deletion mutagenesis of *kstD*, encoding 3-ketosteroid Δ1-dehydrogenase, in *Rhodococcus erythropolis* SQ1 using *sacB* as counter-selectable marker, FEMS Microbiol. Lett. 205 (2001) 197–202.

[84] R. van der Geize, G.I. Hessels, R. van der Meijden, L. Dijkhuizen, Molecular and functional characterization of *kshA* and *kshB*, encoding two components of 3-ketosteroid 9α-hydroxylase, a class IA monooxygenase, in *Rhodococcus erythropolis* strain SQ1, Mol. Microbiol. 45 (2002) 1007–1018.

[85] R. van der Geize, G.I. Hessels, L. Dijkhuizen, Molecular and functional characterization of the *kstD2* gene of *Rhodococcus erythropolis* SQ1 encoding a second 3-ketosteroid Δ(1)-dehydrogenase isoenzyme, Microbiology 148 (2002) 3285–3292.

[86] R. van der Geize, G.I. Hessels, M. Nienhuis-Kuiper, L. Dijkhuizen, Characterization of a second *Rhodococcus erythropolis* SQ1 3-ketosteroid 9α-hydroxylase activity comprising a terminal oxygenase homologue, KshA2, active with oxygenase-reductase component KshB, Appl. Environ. Microbiol. 74 (2008) 7197–7203.

[87] D. Skowasch, E. Mobus, E. Maser, Identification of a novel *Comamonas testosteroni* gene encoding a steroid-inducible extradiol dioxygenase, Biochem. Biophys. Res. Commun. 294 (2002) 560–566.
