
The Journal of Antibiotics (2011) 64, 35–44  
© 2011 Japan Antibiotics Research Association All rights reserved 0021-8820/11 $32.00  
www.nature.com/ja

---

**REVIEW ARTICLE**

---

**The biosynthesis of 3-amino-5-hydroxybenzoic acid (AHBA), the precursor of mC₇N units in ansamycin and mitomycin antibiotics: a review**

Heinz G Floss¹, Tin-Wein Yu² and Kenji Arakawa³

The aminoshikimate pathway of formation of 3-amino-5-hydroxybenzoic acid (AHBA), the precursor of ansamycin and other antibiotics is reviewed. In this biosynthesis, genes for kanosamine formation have been recruited from other genomes, to provide a nitrogenous precursor. Kanosamine is then phosphorylated and converted by common cellular enzymes into 1-deoxy-1-imino-erythrose 4-phosphate, the substrate for the formation of aminoDAHP. This is converted via 5-deoxy-5-aminodehydroquinic acid and 5-deoxy-5-aminodehydroshikimic acid into AHBA. Remarkably, the pyridoxal phosphate enzyme AHBA synthase seems to have two catalytic functions: As a homodimer, it catalyzes the last reaction in the pathway, the aromatization of 5-deoxy-5-aminodehydroshikimic acid, and at the beginning of the pathway in a complex with the oxidoreductase RifL it catalyzes the transamination of UDP-3-keto-D-glucose. The AHBA synthase gene also serves as a useful tool in the genetic screening for new ansamycins and other AHBA-derived natural products.

*The Journal of Antibiotics (2011) 64, 35–44; doi:10.1038/ja.2010.139; published online 17 November 2010*

Keywords: 3-amino-5-hydroxybenzoic acid; aminoshikimate pathway; ansamitocins; ansamycins; kanosamine; mitomycin; rifamycin

---

**HISTORY**

During studies carried out on the biosynthesis of the ansamycin antibiotics,¹ the rifamycins,² a biosynthetically unique structural unit, a six-membered carbocycle carrying an extra carbon and a nitrogen in a meta arrangement, was recognized.³⁴ This so-called mC₇N unit was the only part of the structural core not derived from acetate/propionate units. Such mC₇N units were also found in ansamycins of the ‘benzenic’ type, such as geldanamycin⁵ and the maytansinoids,⁶ and in the unrelated family of the mitomycin antibiotics (Figure 1).⁷ A number of other compounds also seemed to contain mC₇N units, such as validamycin⁸ and acarbose,⁹ pactamycin,¹⁰ as well as asukamycin¹¹ and the manumycins,¹² and were proposed to have a similar biosynthetic origin.¹¹,¹³ However, subsequent work has shown their origins to be different. The mC₇N unit of validamycin and acarbose arises from 2-epi-5-epi-valiolone, a product of the cyclization of seduheptulose 7-phosphate,¹⁴ that of pactamycin from the shikimate pathway via 3-aminobenzoic acid,¹⁵ and those of asukamycin and manumycin from 3-amino-4-hydroxybenzoic acid, which arises from the condensation of a triose phosphate and a C₄-dicarboxylic acid.¹⁶

The incorporation patterns of labeled precursors into several ansamycins and into mitomycins,¹³,¹⁷⁻¹⁹ as well as mutant complementation studies²⁰ all suggested a shikimate pathway origin of the

mC₇N unit, but attempts to incorporate labeled shikimic acid, quinic acid or 3-dehydroquinic acid were unsuccessful.¹⁷,¹⁹,²¹⁻²³ The non-incorporation of shikimic acid could be because of permeability barriers. The cells of the rifamycin producer *Amycolatopsis mediterranei* are indeed impermeable for shikimic acid.²⁰ But this explanation as the general reason for the nonincorporation of shikimic acid was ruled out when in the case of ansatrienin (mycotriienin) it was shown that the producer organism, *Streptomyces collinus*, incorporated labeled shikimic acid into the cyclohexane moiety of the antibiotic, but not into its mC₇N unit.²⁴ Hence, the formation of the mC₇N unit must branch off from the shikimate pathway before dehydroquinic acid, a conclusion also reached by mutant studies on rifamycin formation.²⁰ For earlier reviews see refs Ghisalba,²⁵ Floss and Beale²⁶ and Floss.²⁷

**AHBA AS PRECURSOR OF THE MC₇N UNIT**

On the basis of biosynthetic principles, the mC₇N unit of ansamycins should be a carboxylic acid serving as the starter unit of a polyketide chain assembly. This reasoning and a comparison of all ansamycin structures led Rickards and co-workers²⁸ to identify 3-amino-5-hydroxybenzoic acid (AHBA), a naturally occurring amino acid,²⁹ as a plausible candidate precursor, which they synthesized in labeled form.³⁰ Its efficient and specific incorporation into ‘naphthalenic’

---

¹Department of Chemistry, University of Washington, Seattle, WA, USA; ²Department of Biological Science, Louisiana State University, Baton Rouge, LA, USA and ³Graduate School of Advanced Sciences of Matter, Department of Molecular Biotechnology, Hiroshima University, Higashi-Hiroshima, Hiroshima, Japan

Correspondence: Dr HG Floss, Department of Chemistry, University of Washington, Box 351700, Seattle, WA 98195-1700, USA. E-mail: floss@chem.washington.edu

This article is contributed to the special issue of the *Journal of Antibiotics* to the memory of Dr CR Hutchinson.

Received 8 September 2010; revised 17 October 2010; accepted 18 October 2010; published online 17 November 2010

Biosynthesis of 3-amino-5-hydroxybenzoic acid  
HG Floss *et al*

---

Figure 1 Antibiotics containing mC₇N units, which are shown in heavy lines, and their precursor 3-amino-5-hydroxybenzoic acid (AHBA).

and ‘benzenic’ ansamycins and into mitomycin validated their hypothesis.²⁸,³¹,³² The same conclusion was independently reached by Nüesch and Ghisalba,³³ who found that a mutant of *A. mediterranei* blocked in an early step of rifamycin biosynthesis was effectively complemented by AHBA as the only of many compounds tested. Later work has repeatedly confirmed the specific role of AHBA as precursor of the mC₇N unit of various other compounds.³⁴⁻³⁹

The biosynthetic question then raised was how AHBA was formed by a route that is related to the shikimate pathway. An important issue was whether the nitrogen of AHBA was attached to the carbon corresponding to C-3 or C-5 of shikimate. Degradations of mitomycins derived from various labeled precursors indicated that the C-5-equivalent carbon carries the nitrogen.²¹ Although this result was initially questioned when it was found that in pactamycin the C-3-equivalent carbon carries the nitrogen,¹⁵ this turned out to be an anomaly, and further work by Rinehart *et al.*⁴⁰ on geldanamycin soon confirmed Hornemann’s conclusion from the mitomycin study. Subsequent work in other systems also confirmed the site of attachment of the nitrogen.²⁶,³⁷

On the basis of the mitomycin work, Hornemann *et al.*²¹ proposed 3,4-dideoxy-4-amino-D-arabino-seduheptulosonic acid 7-phosphate (amino-DAHP), which he hypothesized to arise from DAHP, as the specific precursor of the mC₇N unit. We subsequently modified this hypothesis based on a report by the group of Jiao⁴¹ that the nitrogen of rifamycin is derived from the amide nitrogen of glutamine. Guided by the logic underlying the mechanism of anthranilate synthase and other glutamine-dependent amidotransferases,⁴² we proposed²⁶ that amino-DAHP is formed independently by reaction of erythrose 4-phosphate (E-4-P) with ammonia, generated *in situ* by hydrolysis of glutamine, to form the imine, which then is condensed with phosphoenolpyruvate (PEP). In reactions paralleling the normal shikimate pathway, aminoDAHP is then cyclized to 5-deoxy-5-aminodehydroquinate (aminoDHQ), dehydrated to 5-deoxy-5-aminodehydroshikimic acid (aminoDHS), and then aromatized to AHBA (Figure 2). The three postulated intermediates, all unknown compounds, were then synthesized⁴³,⁴⁴ and incubated with cell-free extracts of the

---

**AHBA SYNTHASE**

The only reaction in the proposed route to AHBA, which has no analogy in the normal shikimate pathway, is the aromatization of aminoDHS by a postulated AHBA synthase. This enzyme was, therefore, purified to homogeneity from *A. mediterranei* and characterized.⁴⁵ The enzyme is a pyridoxal phosphate (PLP) protein, catalyzing the aromatization of aminoDHS by Schiff’s base formation of the cofactor with the amino group of the substrate (demonstrated by reduction with ³H-NaBH₄) followed by C4/C5 dehydration and stereospecific enolization with elimination of the pro-6S hydrogen (Figure 3). Interestingly, the enzyme activity was enhanced 40–50% by DAHP; as it had been shown earlier that *A. mediterranei* DAHP synthase is inhibited by rifamycin,⁴⁶ this suggests some cross-regulation of the shikimate and aminoshikimate pathways in this organism.⁴⁵ Partial amino acid sequencing gave information that allowed the cloning of the AHBA synthase gene from *A. mediterranei* by a reverse genetics approach. The gene was overexpressed in *Escherichia coli* as a His₆-tagged fusion protein of 44 kDa subunit, 80 kDa native MW, suggesting that it exists as a dimer. The properties of the native and the recombinant enzyme were shown to be very similar. Interestingly, the enzyme showed more homology to a number of PLP-dependent aminotransferases than to PLP enzymes catalyzing mechanistically similar α,β-elimination reactions.⁴⁵ Availability of the

Biosynthesis of 3-amino-5-hydroxybenzoic acid  
HG Floss et al  

---

Shikimate Pathway  

PEP  
$\begin{aligned}
& \text{COOH} \\
& \text{CHO} \\
& \text{H-OH} \\
& \text{CH}_{2}\text{O-P}
\end{aligned}$  

+  

Glutamine  

$\begin{aligned}
& \text{COOH} \\
& \text{CHO} \\
& \text{H-OH} \\
& \text{CH}_{2}\text{O-P}
\end{aligned}$  

$\longrightarrow$  

DAHP  

$\begin{aligned}
& \text{COOH} \\
& \text{HO} \\
& \text{H-OH} \\
& \text{CH}_{2}\text{O-P}
\end{aligned}$  

$\longrightarrow$  

DHQ  

$\begin{aligned}
& \text{COOH} \\
& \text{HO} \\
& \text{H-OH} \\
& \text{H}
\end{aligned}$  

$\longrightarrow$  

DHS  

$\begin{aligned}
& \text{COOH} \\
& \text{HO} \\
& \text{H-OH} \\
& \text{H}
\end{aligned}$  

$\longrightarrow$  

Shikimate  

$\begin{aligned}
& \text{COOH} \\
& \text{HO} \\
& \text{H-OH} \\
& \text{H}
\end{aligned}$  

Aminoshikimate Pathway  

E4P  

$\begin{aligned}
& \text{CHO} \\
& \text{H-OH} \\
& \text{CH}_{2}\text{O-P}
\end{aligned}$  

$\longrightarrow$  

aminoDAHP  

$\begin{aligned}
& \text{COOH} \\
& \text{H-NH}_{2} \\
& \text{H-OH} \\
& \text{CH}_{2}\text{O-P}
\end{aligned}$  

$\longrightarrow$  

aminoDHQ  

$\begin{aligned}
& \text{COOH} \\
& \text{HO} \\
& \text{H-OH} \\
& \text{NH}_{2}
\end{aligned}$  

$\longrightarrow$  

aminoDHS  

$\begin{aligned}
& \text{COOH} \\
& \text{HO} \\
& \text{H-OH} \\
& \text{NH}_{2}
\end{aligned}$  

$\longrightarrow$  

AHBA  

$\begin{aligned}
& \text{COOH} \\
& \text{HO} \\
& \text{H-NH}_{2} \\
& \text{H}
\end{aligned}$  

Figure 2 The shikimate pathway and the parallel aminoshikimate pathway to AHBA as originally proposed.²⁶  

aminoDHS  

$\begin{aligned}
& \text{COOH} \\
& \text{H-R} \\
& \text{NH}_{2} \\
& \text{OH}
\end{aligned}$  

+  

enzyme  

$\begin{aligned}
& \text{CH}_{3} \\
& \text{N} \\
& \text{C-O-P}
\end{aligned}$  

$\longrightarrow$  

PLP-enzyme  

$\begin{aligned}
& \text{COOH} \\
& \text{H-R} \\
& \text{N-H} \\
& \text{C-O-P}
\end{aligned}$  

$\longleftrightarrow$  

$\begin{aligned}
& \text{COOH} \\
& \text{H-R} \\
& \text{N-H} \\
& \text{C-O-P}
\end{aligned}$  

$\longleftrightarrow$  

$\begin{aligned}
& \text{COOH} \\
& \text{H-R} \\
& \text{N-H} \\
& \text{C-O-P}
\end{aligned}$  

$\longrightarrow$  

stereospecific loss of H₅  

$\begin{aligned}
& \text{COOH} \\
& \text{H-R} \\
& \text{N-H} \\
& \text{C-O-P}
\end{aligned}$  

$\longrightarrow$  

PLP-enzyme  

$\begin{aligned}
& \text{COOH} \\
& \text{HO} \\
& \text{NH}_{2}
\end{aligned}$  

AHBA  

Figure 3 Mechanism of the aromatization of aminoDHS to AHBA by the PLP-enzyme AHBA synthase.  

AHBA synthase gene also allowed the disruption of this gene in the *A. mediterranei* genome by insertional inactivation, leading to loss of rifamycin production. The mutation could be complemented by AHBA, demonstrating that the AHBA synthase gene is indeed essential for rifamycin biosynthesis.⁴⁵  

The recombinant AHBA synthase protein was then crystallized and, in the hands of JC Eads et al.,⁴⁷ yielded a 2.0 Å X-ray structure of the protein with bound PLP cofactor and complexed with PLP and the inhibitor gabaculine. The overall fold of AHBA synthase is similar to that of the aspartate transaminase family of PLP enzymes, with a large domain consisting of a seven-stranded β-sheet surrounded by α-helices and a smaller C-terminal domain of a four-stranded β-sheet and four α-helices (Figure 4a). The PLP cofactor in the holoenzyme forms a Schiff's base with K188, which in the enzyme–gabaculine complex is replaced by a Schiff's base between PLP and the inhibitor. As expected, a K188A mutant was devoid of catalytic activity. Other active site amino acids (Figure 4b) are R236* and D159, forming salt bridges to the PLP phosphate group and pyridine nitrogen, respectively; H162 hydrogen bonding to the PLP phenolate oxygen; and R219* and Y226*, which interact with the carboxylate group of the inhibitor or substrate. Site-specific mutagenesis studies confirmed the essential role of most of these amino acids (Table 1) (Yu, T-W, Trinh, KQ, Müller, R and Floss, HG, unpublished data). A R219A mutant showed diminished activity, as did a H162L mutant, whereas R236A, D159A, D159E and D159K, as well as a Y226F mutant were inactive (<1% of residual activity).  

The aromatization of aminoDHS to AHBA involves several proton transfer steps; that is, (1) abstraction of the C-5 proton, (2) protonation of 4-OH to make it a better leaving group, (3) protonation of the C-3 carbonyl group as part of the enolization and (4) abstraction of the pro-6S proton. A modeled structure of AHBA synthase with bound substrate shows that the Lys-188 ε-amino group is positioned  

Biosynthesis of 3-amino-5-hydroxybenzoic acid  
HG Floss et al  

Table 1 Relative activity of recombinant wild-type and mutant AHBA synthases from *A. mediterranei* with aminoDHS as substrate in the presence of saturating concentrations of PLP

|          | $K_m (\mu M)$ | $k_{cat} (sec^{-1})$ |
|----------|----------------|-----------------------|
| WT       | $122 \pm 2$    | $6.82 \pm 0.13$      |
| H162L    | $281 \pm 9$    | $3.50 \pm 0.04$      |
| R219A    | $71 \pm 9$     | $0.64 \pm 0.02$      |
| Y291F    | $303 \pm 18$   | $0.96 \pm 0.04$      |
| F88A     | NA             | NA                    |
| D159A    | NA             | NA                    |
| D159E    | NA             | NA                    |
| D159K    | NA             | NA                    |
| Q185E    | NA             | NA                    |
| Q185H    | NA             | NA                    |
| Q185L    | NA             | NA                    |
| K188D    | NA             | NA                    |
| Y226F    | NA             | NA                    |
| R236A    | NA             | NA                    |
| Y291A    | NA             | NA                    |

Abbreviations: AHBA, 3-amino-5-hydroxybenzoic acid; aminoDHS, 5-deoxy-5-aminodehydroshikimic acid; NA, no activity detected, limit < 1%; PLP, pyridoxal phosphate; WT, wild-type.

The AHBA protein is a homodimer with two active sites, each made up of residues from both subunits; the amino acid residues contributed by one subunit are listed without asterisks and those coming from the other subunit with an asterisk (Figure 4b). This leads to the prediction that the enzyme should only be catalytically active as the dimer. This notion was confirmed by a classical chimeric dimer experiment.⁴⁸ A plasmid carrying the R236A variant of the AHBA synthase gene was expressed in a D159A mutant strain of *A. mediterranei*. Although each of the two mutations alone led to complete loss of rifamycin production, the transformant had recovered 30% of the wild-type production. Of the four different dimeric versions of the mutated protein, two are inactive 'homodimers' with the same mutation in both monomers, whereas two are mixed dimers carrying a different mutation in each monomeric unit. Each of the latter has one functional and one nonfunctional active site, leading to a theoretical 25% of activity recovery (Yu, T-W *et al.*, unpublished data). The slightly higher observed value reflects the fact that AHBA is normally not limiting for rifamycin production.⁴⁵

## THE AHBA SYNTHESIS GENE CLUSTERS

The AHBA synthase gene then served as a probe allowing the cloning of gene clusters encoding the biosynthesis of various AHBA-derived antibiotics. First of these was the rifamycin biosynthetic gene cluster, which we cloned in collaboration with the group of Hutchinson.⁴⁹ The 34 gene cluster consists of genes encoding five type I modular polyketide synthases associated with an amide synthase (*rifA–F*) and a subcluster of AHBA biosynthesis genes (*rifG–I, K–N*) as well as, separate from it, *riff*, in addition to genes controlling the post-polyketide synthase elaboration of the rifamycin structures, as well as regulatory expression and product export. Some of the genes in the AHBA subcluster (Figure 6) and their encoded products were expected based on the proposed pathway, such as *rifG* (aminoDHQ synthase), *rifH* (aminoDAHP synthase), *riff* (aminoDHQ dehydratase), *rifK* (AHBA synthase), but three of the genes were totally unexpected, *rifL, rifM* and *rifN* with homologies to oxidoreductases, phosphatases and kinases, respectively. Another unexpected gene in the subcluster, *riff*, is homologous to DHQ dehydrogenases and its role has so far remained obscure.⁴⁹

Figure 4 The 2 Å crystal structure of the AHBA synthase protein from *A. mediterranei* with the bound inhibitor gabaculine. (a) The native dimeric protein, (b) the gabaculine-PLP complex with surrounding active site amino acids (from ref. Eads *et al.*⁴⁷).

in proximity to 4-OH, H-5 and H-6S, and it is proposed that this residue mediates sequentially the steps (1), (2) and (4). However, it is too far removed from 3-O to catalyze step (3) and no other amino acid is close enough to catalyze this protonation directly. Instead, it is proposed that Q185 assists in the elimination of the protonated 4-OH as water and then the transfer of a proton from this water molecule to 3-O in a relay mechanism. Consistent with this proposal, the Q185H, Q185E and Q185L mutants are devoid of activity. On the basis of its proximity, an alternative candidate to mediate deprotonation at C-6 is Y291. However, as a Y291F mutant still shows activity, albeit greatly diminished, whereas an Y291A mutant is inactive, it seems more likely that the role of Y291 is structural. The F88 may also have an essential structural role or may stabilize the enzyme-substrate complex by hydrophobic interactions with the ring of aminoDHS/AHBA, based on the lack of activity of a F88A mutant (Yu T-W *et al.*, unpublished data). On the basis of these results, the mechanism of the enzyme can be refined to the representation shown in Figure 5.

Biosynthesis of 3-amino-5-hydroxybenzoic acid
HG Floss et al

Figure 5 Proposed stereochemical mechanism of the AHBA synthase reaction: B¹=K188, B²=Q185.

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 6 Arrangement of the AHBA biosynthetic genes in (a) the rifamycin cluster of *Amycolatopsis mediterranei* and (b) the ansamitocin cluster of *Actinosynnema pretiosum*.

![Diagram](https://i.imgur.com/yourimageurl2.png)

RifG ~ Asm47: AminoDHQ synthase  
RifH: AminoDAHP synthase  
RifI: Aminoquinol dehydrogenase  
RifK ~ Asm24 ~ Asm43: AHBA synthase  
RifL ~ Asm44: Oxidoreductase  
RifM ~ Asm45: Phosphatase  
RifN ~ Asm22: Kinase  
RifJ ~ Asm23: AminoDHQ dehydratase  

The role of these genes and their products in AHBA formation was then probed by the construction of an expression cassette containing the *rifG–N* genes under the control of the *actII-orf4* promoter and its expression in *Streptomyces coelicolor*.⁵⁰ Efficient AHBA production (350–400 mg l⁻¹) by the transformant showed that these genes are sufficient for AHBA formation in this heterologous background. Selective inactivation of *rifG* and *rifI* in the *A. mediterranei* genome had no effect on rifamycin production, showing that any function of these genes can be effectively taken over by the corresponding normal shikimate pathway genes, whereas a deletion of *rifGHI* reduced rifamycin production to 1% and inactivation of *rifJ* to 10% of wild-type levels. Remarkably, inactivation of *rifL*, *rifM* or *rifN* completely abolished rifamycin production; however, cell-free extracts of the *rifM* and *rifN* mutants were still able to convert aminoDAHP, aminoDHQ and aminoDHS into AHBA. Deletions of individual genes or sets of genes from the *S. coelicolor* expression cassette amplified these results. Deletion of *rifI* reproducibly increased AHBA production by about 20%, hinting at a possible salvage role of RifI. Conversely, Guo and Frost⁵¹ found that plasmid-based expression in *A. mediterranei* of extra copies of *rifI* under control of an amylase promoter led to a threefold increase in the specific activity of aminoshikimate dehydrogenase and to the accumulation of 0.2 gl⁻¹ of aminoshikimate, but no accumulation of aminoisquinoline. This establishes *rifI* as an *in vivo* aminoshikimate dehydrogenase. Deletion of *rifG* reduced AHBA production by 90%, whereas the absence of *rifH* led to virtually no production. No AHBA was formed when either *rifL*, *rifM* or *rifN* was deleted from the cassette, whereas deletion of *rifJ* reduced its production by about 90%. Hence, the steps of the pathway to AHBA from aminoDAHP have been confirmed by the genetic data, but the requirement for three additional genes, *rifLMN* suggests a more complicated pathway for the formation of aminoDAHP.⁵⁰ This point is amplified by the recent finding³⁹ that a transketolase gene present in the *rif* clusters of *A. mediterranei* (*rif orf15AB*) and *Salinispora arenicola* (see below) (*sare1272/1273*) is, at least in the latter organism, essential for rifamycin and saliniketal formation. An *S. arenicola sare1272/1273*-inactivated nonproducing mutant was complemented by AHBA to restore product formation, demonstrating that at least in this organism an additional, dedicated transketolase gene is required for AHBA production, although in other organisms, such as *S. coelicolor*, its function can be taken over by a housekeeping transketolase.

The AHBA synthase gene *rifK* was used by others and us to clone additional gene clusters for metabolites derived from AHBA. The cluster encoding the formation of rubradirin, an unusual naphthalenic ansamycin includes homologs of all the *rif* AHBA genes except *rifI*.⁵² The ansatrienin producer *S. collinus* gave two AHBA biosynthesis subclusters, one associated with other ansatrienin genes and containing all the genes found in the *rif* cluster except *rifI*.⁵³ The other is associated with some putative naphthomycin biosynthesis genes and contains homologs of all the *rif* AHBA genes except *rifJ* which, as in the *rif* case, may be located elsewhere among the naphthomycin biosynthesis genes.⁵³ Two AHBA biosynthesis clusters, one (AHBA-N) associated with putative naphthomycin biosynthesis and the other (AHBA-B) functionally, although not physically, associated with geldanamycin (*gdn*) biosynthesis genes, were isolated from *S. hygroscopicus*.⁵⁴⁻⁵⁶ Curiously, the AHBA-B subcluster contains two copies of an aminoDHQ synthase gene (*rifG* homolog), but lacks an aminoDAHP synthase gene (*rifH*).⁵³ A *rifH* homolog is also missing from the AHBA synthesis genes of the mitomycin cluster.⁵⁷ Evidently, the normal shikimate pathway DAHP synthase has in some cases sufficiently broad substrate specificity to catalyze also the formation of aminoDAHP, just as the next shikimate pathway enzyme, DHQ synthase can also handle the amino analog, as shown by the mutant experiments described above. A particularly interesting arrangement of the AHBA biosynthesis genes was found in the ansamitocin cluster from *Actinosynnema pretiosum* (Figure 6).⁵⁸ Two copies of AHBA synthase genes were found, both encoding functional AHBA synthases.

and essential for ansamitocin formation, one within the main cluster, associated with *riff* and *rifN* homologs, and the other 65 kb away in a second group of genes containing *rifL*, *rifM* and *rifG* homologs. There are no *rifH* and *rifI* homologs identified within these two gene groups, which are separated by a 30 kb segment carrying ansamitocin-unrelated genes.⁵⁸ Even more complex is the arrangement of these genes uncovered in the recently sequenced genome of *Actinosynnema mirum*, in which the two *rifK* homologs are 112 kb apart and each is associated with two of the four ansamitocin polyketide synthases genes.⁵⁹ These findings suggest extensive gene rearrangements in *Actinosynnema* genomes. In all the arrangements of AHBA biosynthesis genes, one or more of the early shikimate pathway gene homologs and the dedicated transketolase gene may be missing, but homologs of *rifK*, *rifL*, *rifM* and *rifN* are always present.

### EARLY STEPS OF THE AMINOSHIKIMATE PATHWAY

All the data described up to this point confirm the validity of the late steps of the proposed aminoshikimate pathway for AHBA formation from the stage of aminoDAHP on, but also suggest that the mode of formation of aminoDAHP must be more complicated than originally postulated. In particular, the fact that the *rifLMN* genes are collectively required for AHBA formation, yet have no obvious role in the proposed mode of formation of aminoDAHP, showed that unknown complexities are hidden in this biosynthesis. The solution of this puzzle came from work by Guo and Frost,⁶⁰ who demonstrated an alternative mode of formation of the postulated enzyme-bound intermediate, 1-deoxy-1-imino-erythrose 4-phosphate, by a transketolase reaction from 3-amino-3-deoxyfructose 6-phosphate (aminoF-6-P) and ribose 5-phosphate. When carried out in the presence of the *rifH* gene product, aminoDAHP synthase, or an *A. mediterranei* cell-free extract and PEP, this reaction produced amino-

DAHP in addition to DAHP (Figure 7a). An experiment with [6,6-²H, amino-¹⁵N]aminoF-6-P demonstrated that both the framework and the nitrogen of aminoF-6-P were incorporated into aminoDAHP.⁶⁰ Evidently, some of the initially formed 1-deoxy-1-imino-erythrose 4-phosphate hydrolyzes to erythrose 4-phosphate before it can be condensed with PEP by the synthase. To make this system an efficient process for aminoDAHP formation, in intact cells of *A. mediterranei* the transketolase must presumably hand its product directly over to the aminoDAHP synthase, suggesting a possible interaction between the two enzymes. This may be a reason for the presence of a dedicated transketolase gene in the *rif* and *sare* clusters.³⁹

The above findings then raised the question what the origin of aminoF-6-P is. A related compound that occurs naturally is kanosamine (3-amino-3-deoxy-d-glucose), which is produced as such by various bacteria⁶¹ and is also a component of some antibiotics, such as the kanamycins⁶² and zwittermicin A.⁶³ Its 6-phosphate could give rise to aminoF-6-P by the action of a phosphoglucose isomerase. To test this idea, Guo and Frost⁶⁴ synthesized kanosamine 6-phosphate and incubated it with ribose 5-phosphate, PEP and either *A. mediterranei* cell-free extract or yeast phosphoglucose isomerase, *E. coli* transketolase and *rifH*. In both instances, aminoDAHP was formed in addition to larger amounts of DAHP and, in the first case, small amounts of AHBA (Figure 7b). Thus, the ability of *A. mediterranei* to convert kanosamine 6-phosphate into aminoDAHP and AHBA was demonstrated.⁶⁴

The biosynthesis of kanosamine has been studied long ago by Umezawa et al.⁶⁵ in cell-free extracts of *Bacillus pumilus* (formerly *B. aminoglicosidicus*). It proceeds from UDP-glucose by 3-dehydrogenation and transamination with glutamine as the amino donor to UDP-kanosamine, which is then hydrolyzed to kanosamine. The same transformation was demonstrated by Guo and Frost⁶⁴ on incubation

![Figure 7](#fig7)

**Figure 7** Reactions demonstrated in *A. mediterranei* cell-free extracts or reconstituted enzyme systems. (a) Formation of aminoDAHP from 3-amino-3-deoxyfructose 6-phosphate shown by Guo and Frost,⁶⁰ (b) formation of aminoDAHP from kanosamine 6-phosphate shown by Guo and Frost,⁶⁴ (c) formation of kanosamine from UDP-glucose shown by Guo and Frost⁶⁴ and (d) formation of kanosamine 6-phosphate from kanosamine with recombinant RifN protein shown by Arakawa et al.⁶⁶

Biosynthesis of 3-amino-5-hydroxybenzoic acid
HG Floss et al

---

Figure 8 Demonstration of a physical association between RifL and RifK and enzymatic activity of the complex. (a) SDS-PAGE gel, stained with Coomassie Blue, of cell-free extract of *E. coli* BL21(DE3)pLysS/pHGF7711 (*rifK/apr*)/pRM030 (His<sub>6</sub>-rifL/ampR) induced with 0.1 mM IPTG. Lane M, protein molecular mass marker (sizes indicated in kDa); lane 1, total protein; lane 2, cell-free extract; lane 3, flow-through fraction from Ni-NTA column; lanes 4 and 5, fractions washed with 10 mM imidazole; lanes 6–8, fractions eluted with 200 mM imidazole. (b) Enzymatic activity of RifKL with UDP-glucose and NAD<sup>+</sup> assayed by measuring A<sub>340</sub>. Samples consist of 2 μmol XDP-glucose, 2 μmol NAD<sup>+</sup>, 10 μmol MgCl<sub>2</sub>, and 33 μg RifKL in 50 mM HEPES buffer (pH 7.6). (c) Enzymatic activity of RifKL with UDP-glucose, different nitrogen sources and NAD<sup>+</sup> assayed by measuring A<sub>340</sub>. Samples consist of 2 μmol UDP-glucose, 2 μmol NAD<sup>+</sup>, 4 μmol nitrogen source, 10 nmol pyridoxal phosphate, 10 μmol MgCl<sub>2</sub>, and 33 μg RifKL in 50 mM HEPES buffer (pH 7.6).

of UDP-glucose, NAD<sup>+</sup> and glutamine with the cell-free extract of *A. mediterranei*, and the use of UDP-[6,6-<sup>2</sup>H]glucose gave [6,6-<sup>2</sup>H]kanosamine (Figure 7c). It was proposed that the reaction did not proceed beyond that point because of a limitation of the amount of a kanosamine kinase in the cell-free extract. In parallel, our group expressed the *rifN* gene, which showed homology to kinases and demonstrated that its gene product is indeed a highly specific kanosamine kinase (Figure 7d).<sup>66</sup> Thus, the function of one of the three unassigned genes of the AHBA subcluster was revealed, leaving only *rifL* and *rifM*.

Three genes proposed to be involved in kanosamine formation were recently cloned as part of the zwittermicinA cluster of *B. cereus*.<sup>67</sup> They encode an oxidoreductase (KabC), a transaminase (KabA) and a hydrolase (KabB). KabC and KabB correspond to *rifL* and *rifM*, respectively, in the *rif* cluster, but there is no obvious candidate for the aminotransferase in the cluster. The only transaminase gene in the *rif* cluster, *orf9*, was shown by inactivation to be unrelated to rifamycin biosynthesis, and is believed to be part of a silent deoxyaminosugar pathway.<sup>50</sup> Lack of an alternative candidate and several pieces of indirect evidence led us to consider the possibility that the RiffK protein, in addition to its role as AHBA synthase, may have a second function as the transaminase recruiting nitrogen to the oxidized UDP-glucose to give UDP-kanosamine. Pointing in that direction are (1) the fact that RifK is more closely related to various glutamine-dependent aminotransferases operating in amino-glycoside biosynthesis, such as StsC, than to PLP-dependent enzymes catalyzing α,β eliminations,<sup>49</sup> (2) the close physical association of the AHBA synthase gene with *rifL* and *rifM* homologs in all the AHBA synthesis subclusters, suggesting that they may represent a transcription unit that has been recruited intact by lateral gene transfer, (3) the requirement for two separate AHBA synthase genes in ansamitocin biosynthesis, one associated with *rifL* and *rifM* homologs and the other with downstream AHBA synthesis genes and (4) the fact that *rifK* mutants do not accumulate aminoDHS or its spontaneous aromatization product, protocatechuic acid, and that despite an extensive search no other accumulation product could be found.<sup>50</sup>

The presumed function of *rifM* as the enzyme converting UDP-kanosamine into kanosamine has not yet been demonstrated, although the protein has been expressed in soluble form. No transaminase activity of AHBA synthase could be detected with a variety of pseudosubstrates, assaying both in the forward and reverse direction, and the predicted true substrates were not available (Arakawa, K, Müller, R, Yu, T-W & Floss, HG, unpublished data). Overexpression of *rifL* as a His<sub>6</sub>-fusion protein in *E. coli* gave only inclusion bodies, but in *S. lividans* produced a modest amount of soluble

protein after Ni-NTA purification. This, however, showed no enzymatic activity with NDP-glucose, Mg²⁺ and NAD⁺ or NADP⁺. Addition of RifK protein plus glutamine to the reaction mixtures did not change the outcome. It was then reasoned that the RifL protein might be unstable and may need to complex with the RifK protein for structural stability or even for both proteins to exert their functions. We, therefore, coexpressed the RifK and the His₆-tagged RifL proteins in *E.coli* from two plasmids carrying two different selection markers. The crude extract showed a strong band of AHBA synthase, most of which did not adsorb to a Ni-NTA column. The SDS gel of the eluate from the Ni-NTA column showed two weaker bands of equal intensity corresponding to the RifK and the His-tagged RifL protein (Figure 8a). As only the RifL protein carried the affinity tag, the two undenatured proteins must have bound to each other strongly enough to retain the complex on the affinity column. Preliminary enzymatic assays of this presumed RifKL complex showed NADH formation from UDP-glucose, but not dTDP-glucose, Mg²⁺ and NAD⁺ (Figure 8b). The rate of NADH formation was enhanced when glutamine was included in the incubation; NH₄⁺ or asparagine were 30% less effective as nitrogen donors and neither glutamate nor aspartate showed activity (Figure 8c) (Arakawa K, unpublished data).⁶⁶

Unfortunately, at this point the investigations came to an end because of the retirement of the senior investigator. Although the data at hand are anything but conclusive, they are consistent with the proposal that *rifKLM* represent a rudimentary kanosamine biosynthesis cluster that has been jointly recruited with other AHBA biosynthetic genes to provide a nitrogenous starting material for the formation of aminoDAHP, and that RifK has two functions in the AHBA pathway. As a homodimer it catalyzes the terminal step, aromatization of aminoDHS, and in a complex with RifL it may act as a transaminase introducing the nitrogen into the first pathway intermediate, UDP-3-keto-D-glucose. The entire pathway of AHBA formation should, therefore, be formulated as shown in Figure 9. Several of the intermediate steps in the overall AHBA biosynthetic pathway seem to be catalyzed by primary metabolism enzymes, which normally act on the non-nitrogenous analogs, with certain very specific enzymes, such as RifN, acting as gatekeepers. Evidently,

much more work is required to validate or disprove these intriguing possibilities.

# THE AHBA SYNTHASE GENE AS A PROBE

AHBA is primarily noted as the universal precursor of ansamycin antibiotics. However, it is also involved in the biosynthesis of other compound classes, as exemplified by the mitomycins and FR900482, in which it is combined with an aminosugar as the second component.¹³,³⁵,⁶⁸ Other examples are the tetrapetalones,³⁸ which were shown to incorporate an AHBA moiety, and diazepinomicin, which was also proposed to incorporate an AHBA-derived moiety, although final proof is still outstanding.⁶⁹ A particularly intriguing example are the saliniketals from *Salinispora arenicola*, which share a biosynthetic pathway with the rifamycins.³⁹ After the stage of 34a-deoxyrifamycin W, they diverge from the rifamycin pathway, undergoing an alternate cleavage of the polyketide chain, which ultimately leads to loss of the AHBA-derived moiety and the first two polyketide extension units, leaving only the AHBA nitrogen behind in the product. The saliniketal biosynthetic gene cluster, identified by genome mining of the *S. arenicola* genome, contains the same eight AHBA biosynthesis genes as the *rif* cluster, albeit in a somewhat different physical arrangement.³⁹

As mentioned earlier, the *rifK* gene has been used to identify the gene clusters encoding other known ansamycins as well as the mitomycin cluster. PCR screening for the AHBA synthase gene has also been used to explore the ansamycin biosynthesis potential of a variety of *Amycolatopsis* species⁷⁰ and the potential of 2000 newly isolated Actinomycetes for the production of AHBA-derived metabolites, identifying 33 candidates.⁷¹ Similarly, the AHBA synthase gene was used as a probe to screen plant-associated microorganisms in efforts to identify the true biosynthetic sources of the plant-derived maytansinoids and other medicinal agents.⁷²⁻⁷⁴ Perusal of the NCBI’s nucleotide database shows AHBA synthase genes from a number of microorganisms, only some of them known producers of AHBA-derived metabolites. And AHBA synthase genes have been found in a number of sequenced genomes, indicating that genome mining can be another useful approach for identifying new AHBA-derived natural products.

ACKNOWLEDGEMENTS

We thank all the co-workers, who through their dedicated efforts generated the data summarized here. Their names are shown in the publications cited. Dedicated to the late Dr C Richard ‘Dick’/‘Hutch’ Hutchinson for his exceptional contributions to natural product biosynthesis, engineering and drug discovery.

1. Rinehart, Jr K. L. & Shields, L. S. Chemistry of the ansamycin antibiotics. *Fortschr. Chem. Org. Naturst.* **33**, 273–307 (1976).
2. Oppolzer, W. & Prelog, V. Rifamycins 5. The constitution and configuration of rifamycins B, O, S and SV. *Helv. Chim. Acta.* **56**, 2287–2314 (1973).
3. Brufani, M. *et al.* Rifamycins 6. The biogenesis of rifamycin S. *Helv. Chim. Acta.* **56**, 2315–2323 (1973).
4. White, R. J., Martinelli, E., Gallo, G. G., Lancini, G. & Beynon, P. Rifamycin biosynthesis studied with $^{13}$C enriched precursors and carbon magnetic resonance. *Nature* **243**, 273–277 (1973).
5. Rinehart, K. L. Jr., Sasaki, K., Slomp, G., Grostic, M. F. & Olson, E. C. Geldanamycin. I. Structure assignment. *J. Am. Chem. Soc.* **92**, 7591–7593 (1970).
6. Asai, M., Mizuta, E., Izawa, M., Haibara, K. & Kishi, T. Isolation, chemical characterization and structure of ansamitocin, a new ansamycin antibiotic. *Tetrahedron* **35**, 1079–1085 (1979).
7. Webb, J. S. *et al.* The structures of mitomycin A, B and C and porfiromycin—part I. *J. Am. Chem. Soc.* **84**, 3185–3187 (1962).
8. Iwasa, T., Kameda, Y., Asai, M., Horii, S. & Mizuno, K. Studies on validamycins, new antibiotics. IV. Isolation and characterization of validamycins A and B. *J. Antibiot.* **24**, 119–123 (1971).
9. Truscheit, E., Frommer, W., Junge, B., Müller, L., Schmidt, D. D. & Wingender, W. Chemistry and biochemistry of microbial α-glucosidase inhibitors. *Angew. Chem. Int. Ed. Engl.* **20**, 744–761 (1981).
10. Weller, D. D. & Rinehart, Jr. K. L. Biosynthesis of the antitumor antibiotic pactamycin. A methionine-derived ethyl group and a C$_7$N unit. *J. Am. Chem. Soc.* **100**, 6757–6760 (1978).
11. Kakinuma, K., Ikekawa, N., Nakagawa, A. & Omura, S. The structure of asukamycin, a possible shunt metabolite from 3-dehydroquinic acid in the Shikimate pathway. *J. Am. Chem. Soc.* **101**, 3402–3404 (1979).
12. Sattler, I., Thiericke, R. & Zeeck, A. The manumycin-group metabolites. *Nat. Prod. Rep.* **15**, 221–240 (1998).
13. Hornemann, U., Kehrer, J. P., Nunez, C. S. & Ranieri, R. L. d-Glucosamine and L-citrulline, precursors in mitomycin biosynthesis by *Streptomyces verticillatus*. *J. Am. Chem. Soc.* **96**, 320–322 (1974).
14. Cf. Mahmud, T., Lee, S. & Floss, H. G. The biosynthesis of acarbose and validamycin. *Chem. Rec.* **1**, 300–310 (2001).
15. Rinehart, K. L. Jr., Potgieter, M., Delaware, D. L. & Seto, H. Direct evidence from multiple $^{13}$C labeling and homonuclear decoupling for the labeling pattern by glucose of the m-aminobenzoyl (C$_7$N) unit of pactamycin. *J. Am. Chem. Soc.* **103**, 2099–2101 (1981).
16. Hu, Y. & Floss, H. G. Further studies on the biosynthesis of the manumycin type antibiotic, asukamycin, and the chemical synthesis of protoasukamycin. *J. Am. Chem. Soc.* **126**, 3837–3844 (2004).
17. Karlsson, A., Sartori, G. & White, R. J. Rifamycin biosynthesis: further studies on origin of the ansa chain and chromophore. *Eur. J. Biochem.* **47**, 251–256 (1974).
18. White, R. J. & Martinelli, E. Ansamycin biogenesis: incorporation of [1-$^{13}$C]glucose and [1-$^{13}$C]glycerate into the chromophore of rifamycin S. *FEBS Lett.* **49**, 233–236 (1974).
19. Haber, A., Johnson, R. D. & Rinehart, Jr K. L. Geldanamycin. 3. Biosynthetic origin of the C$_2$ units of geldanamycin and distribution of label from D-[6-$^{13}$C]glucose. *J. Am. Chem. Soc.* **99**, 3541–3544 (1977).
20. Gygax, D., Ghisalba, O., Treichler, H. & Nüesch, J. Study to the biosynthesis of the rifamycin-chromophore in *Nocardia mediterranei*. *J. Antibiot.* **43**, 324–326 (1990) and references therein.
21. Hornemann, U., Eggert, J. H. & Honor, D. P. Role of D-[4-$^{14}$C]erythrose and [3-$^{14}$C]pyruvate in the biosynthesis of the meta-C-C$_6$-N unit of the mitomycin antibiotics in *Streptomyces verticillatus*. *J. Chem. Soc., Chem. Commun.* 11–13 (1980).
22. Bezanson, G. S. & Vining, L. C. Studies on the biosynthesis of mitomycin C by *Streptomyces verticillatus*. *Can. J. Biochem.* **49**, 911–918 (1971).
23. Meier, R.-M. & Tamm, C. Studies directed towards the biosynthesis of the C$_7$N-Unit of rifamycin B: incorporation of [$^{14}$C(G)]quinic acid and [1,2-$^{13}$C$_2$]glycerol. *J. Antibiot.* **45**, 400–410 (1992).
24. Casati, R., Beale, J. M. & Floss, H. G. Biosynthesis of ansatrienin. Nonincorporation of shikimic acid into the mC$_7$N unit and stereochemistry of its conversion to the cyclohexanecarboxylic acid moiety. *J. Am. Chem. Soc.* **109**, 8102–8104 (1987).
25. Ghisalba, O. Biosynthesis of rifamycins (ansamycins) and microbial production of shikimate pathway precursors, intermediates and metabolites. *Chimia* **39**, 79–88 (1985).
26. Floss, H. G. & Beale, J. M. Biosynthetic studies on antibiotics. *Angew. Chem. Int. Ed.* **28**, 146–177 (1989).
27. Floss, H. G. Natural products derived from unusual variants of the shikimate pathway. *Nat. Prod. Rep.* **14**, 433–452 (1997).

28. Kibby, J. J., McDonald, I. A. & Rickards, R. W. 3-amino-5-hydroxybenzoic acid as a key intermediate in ansamycin and maytansinoid biosynthesis. *J. Chem. Soc., Chem. Commun.* 768–769 (1980).
29. Kibby, J. J. & Rickards, R. W. The identification of 3-amino-5-hydroxybenzoic acid as a new natural amino acid. *J. Antibiot.* **34**, 605–607 (1981).
30. Herlt, A. J., Kibby, J. J. & Rickards, R. W. Synthesis of unlabelled and carboxyl-labelled 3-amino-5-hydroxybenzoic acid. *Aust. J. Chem.* **34**, 1319–1324 (1981).
31. Anderson, M. G., Kibby, J. J., Rickards, R. W. & Rothschild, J. M. Biosynthesis of the mitomycin antibiotics from 3-amino-5-hydroxybenzoic acid. *J. Chem. Soc., Chem. Commun.* 1277–1278 (1980).
32. Hatano, K., Akiyama, S., Asai, M. & Rickards, R. W. Biosynthetic origin of aminobenzoid nucleus (C$_7$N-Unit) of ansamitocin, a group of novel maytansinoid antibiotics. *J. Antibiot.* **35**, 1415–1417 (1982).
33. Ghisalba, O. & Nüesch, J. A Genetic approach to the biosynthesis of the rifamycin-chromophore in *Nocardia mediterranei*. IV. Identification of 3-amino-5-hydroxybenzoic acid as a direct precursor of the seven-carbon amino starter-unit. *J. Antibiot.* **34**, 64–71 (1981).
34. Wu, T. S., Duncan, J., Tsao, S. W., Chang, C. J., Keller, P. J. & Floss, H. G. Biosynthesis of the ansamycin antibiotic ansatrienin (mycotrienin) by *Streptomyces collinus*. *J. Nat. Prod.* **50**, 108–118 (1987).
35. Fujita, T., Takase, S., Otsuka, T., Terano, H. & Kohsaka, M. Precursors in the biosynthesis of FR-900482, a novel antitumor antibiotic produced by *Streptomyces sandaensis*. *J. Antibiot.* **41**, 392–394 (1988).
36. Staley, A. L. & Rinehart, Jr. K. L. Biosynthesis of the streptovaricins: 3-amino-5-hydroxybenzoic acid as a precursor to the meta-C$_7$N unit. *J. Antibiot.* **44**, 218–224 (1991).
37. Lee, J. P., Tsao, S.-W., He, X.-G., Chang, C.-J. & Floss, H. G. Biosynthesis of naphthomycin A in *Streptomyces collinus*. *Can J. Chem.* **72**, 182–187 (1994).
38. Komoda, T., Sugiyama, Y. & Hirota, A. Biosynthesis of tetrapetalones. *Org. Biomol. Chem.* **5**, 1615–1620 (2007).
39. Wilson, M. C., Gulder, T. A. M., Mahmud, T. & Moore, B. S. Shared biosynthesis of the saliniketals and rifamycins in *Salinispora arenicola* is controlled by the sare1259 encoded cytochrome P450. *J. Am. Chem. Soc.* **132**, 12757–12765 (2010).
40. Rinehart, K. L. Jr., Potgieter, M. & Wright, D. A. Use of D-[${}^{13}$C$_6$]glucose together with carbon-13-depleted glucose and homonuclear carbon-13 decoupling to identify the labeling pattern by this precursor of the ‘m-C$_7$N’ unit of geldanamycin. *J. Am. Chem. Soc.* **104**, 2649–2652 (1982).
41. Jiao, R. S., Liu, C. J., Jin, Z. K., Zhang, X. C., Ni, L. Y. & Lu, Z. M. The route of incorporation of nitrogen atom into rifamycin during its biosynthesis. *Sci. Sinica B* **27**, 380–390 (1984).
42. Massiére, F. & Badet-Denisot, M. A. The mechanism of glutamine-dependent amidotransferases. *Cell. Mol. Life Sci.* **54**, 205–222 (1998).
43. Kirschning, A., Bergon, P., Wang, J. J., Breazeale, S. A. & Floss, H. G. Synthesis of 3,4-dideoxy-4-amino-D-arabino-heptulosonic acid 7-phosphate, the biosynthetic precursor of C$_7$N units in ansamycin antibiotics. *Carbohydr. Res.* **256**, 245–256 (1994).
44. Kim, C.-G. *et al.* Biosynthesis of 3-amino-5-hydroxybenzoic acid, the precursor of mC$_7$N units in ansamycin biosynthesis. *J. Am. Chem. Soc.* **118**, 7486–7491 (1996).
45. Kim, C.-G., Yu, T.-W., Fryhle, C. B., Handa, S. & Floss, H. G. 3-amino-5-hydroxybenzoic acid synthase, the terminal enzyme in the formation of the precursor of mC$_7$N units in rifamycin and related antibiotics. *J. Biol. Chem.* **273**, 6030–6040 (1998).
46. Gygax, D., Christ, M., Ghisalba, O. & Nüesch, J. Regulation of 3-deoxy-D-arabino-heptulosonic acid 7-phosphate synthetase in *Nocardia mediterranei*. *J. FEMS Microbiol. Lett.* **15**, 169–173 (1982).
47. Eads, J. C., Beeby, M., Scapin, G., Yu, T.-W. & Floss, H. G. Crystal structure of 3-amino-5-hydroxybenzoic acid (AHBA) synthase. *Biochemistry* **38**, 9840–9849 (1999).
48. Wente, R. S. & Schachman, H. K. Shared active sites in oligomeric enzymes: model studies with defective mutants of aspartate transcarbamoylase produced by site-directed mutagenesis. *Proc. Natl Acad. Sci. USA* **84**, 31–35 (1987).
49. August, P. R. *et al.* Biosynthesis of the ansamycin antibiotic rifamycin: deductions from the molecular analysis of the *rif* biosynthetic gene cluster of *Amycolatopsis mediterranei* S699. *Chem. Biol.* **5**, 69–79 (1998).
50. Yu, T.-W. *et al.* Mutational analysis and reconstituted expression of the biosynthetic genes involved in the formation of 3-amino-5-hydroxybenzoic acid, the starter unit of rifamycin biosynthesis in *Amycolatopsis mediterranei* S699. *J. Biol. Chem.* **276**, 12546–12555 (2001).
51. Guo, J. & Frost, J. W. Synthesis of aminoshikimic acid. *Org. Lett.* **6**, 1585–1588 (2004).
52. Kim, C.-G. *et al.* Biosynthesis of rubradirin as an ansamycin antibiotic from *Streptomyces achromogenes* var. *rubradiris* NRRL3061. *Arch. Microbiol.* **189**, 463–473 (2008).
53. Chen, S. *et al.* Biosynthesis of ansatrienin (mycotrienin) and naphthomycin. Identification and analysis of two separate biosynthetic gene clusters in *Streptomyces collinus* Tu 1892. *Eur. J. Biochem.* **261**, 98–107 (1999).
54. Rascher, A. *et al.* Cloning and characterization of a gene cluster for geldanamycin production in *Streptomyces hygroscopicus* NRRL 3602. *FEMS Microbiol. Lett.* **218**, 223–230 (2003).
55. Rascher, A., Hu, Z., Buchanan, G. O., Reid, R. & Hutchinson, C. R. Insights into the biosynthesis of the benzoquinone ansamycins geldanamycin and herbimycin, obtained by gene sequencing and disruption. *Appl. Environ. Microbiol.* **71**, 4862–4871 (2005).
56. He, W., Wu, L., Gao, Q., Du, W. & Wang, Y. Identification of AHBA biosynthetic genes related to geldanamycin biosynthesis in *Streptomyces hygroscopicus* 17997. *Curr. Microbiol.* **52**, 197–203 (2006).

57 Mao, Y., Varoglu, M. & Sherman, D. H. Molecular characterization and analysis of the biosynthetic gene cluster for the antitumor antibiotic mitomycin C from *Streptomyces lavendulae* NRRL 2564. *Chem. Biol.* **6**, 251–263 (1999).

58 Yu, T.-W. *et al.* The biosynthetic gene cluster of the maytansinoid antitumor agent ansamitocin from *Actinosynnema pretiosum*. *Proc. Natl Acad. Sci USA* **99**, 7968–7973 (2002).

59 *Actinosynnema mirum* DSM 43827, complete genome sequence, GeneBank CP001630 1, June 4 (2010).

60 Guo, J. & Frost, J. W. Biosynthesis of 1-deoxy-1-imino-D-erythrose 4-phosphate: a defining metabolite in the aminoshikimate pathway. *J. Am. Chem. Soc.* **124**, 528–529 (2002).

61 Milner, J. L., Silo-Suh, L., Lee, J. C., He, H., Clardy, J. & Handelsman, J. Production of Kanosamine by *Bacillus cereus* UW85. *Appl. Environ. Microbiol.* **62**, 3061–3065 (1996) and references therein.

62 Umezawa, H. *et al.* Production and isolation of a new antibiotic, kanamycin. *J. Antibiot.* **10**, 181–188 (1957).

63 Rogers, E. W., Dalisay, D. S. & Molinski, T. F. (+)-Zwittermicin A: assignment of its complete configuration by total synthesis of the enantiomer and implication of D-serine in its biosynthesis. *Angew. Chem. Int. Ed.* **47**, 8086–8089 (2008).

64 Guo, J. & Frost, J. W. Kanosamine biosynthesis: a likely source of the aminoshikimate pathway's nitrogen atom. *J. Am. Chem. Soc.* **124**, 10642–10643 (2002).

65 Umezawa, S., Shibahara, S., Omoto, S., Takeuchi, T. & Umezawa, H. Studies on biosynthesis of 3-amino-3-deoxy-D-glucose. *J. Antibiot.* **21**, 485–491 (1968).

66 Arakawa, K., Müller, R., Mahmud, T., Yu, T.-W. & Floss, H. G. Characterization of the early stage aminoshikimate pathway in the formation of 3-amino-5-hydroxybenzoic acid: the RifN protein specifically converts kanosamine into kanosamine 6-phosphate. *J. Am. Chem. Soc.* **124**, 10644–10645 (2002).

67 Kevany, B. M., Rasko, D. A. & Thomas, M. G. Characterization of the complete zwittermicin A biosynthesis gene cluster from *Bacillus cereus*. *Appl. Environm. Microbiol.* **75**, 1144–1155 (2009).

68 Chamberland, S., Gruschow, S., Sherman, D. H. & Williams, R. M. Synthesis of potential early-stage intermediates in the biosynthesis of FR900482 and mitomycin C. *Org. Lett.* **11**, 791–794 (2009).

69 McAlpine, J. B. *et al.* Biosynthesis of diazepinomicin/ECO-4601, a *Micromonospora* secondary metabolite with a novel ring system. *J. Nat. Prod.* **71**, 1585–1590 (2008).

70 Wood, S. A., Kirby, B. M., Goodwin, C. M., Le Roes, M. & Meyers, P. R. PCR screening reveals unexpected antibiotic biosynthetic potential in *Amycolatopsis* sp. strain UM16. *J. Appl. Microbiol.* **102**, 245–253 (2007).

71 Huitu, Z. *et al.* PCR screening of 3-amino-5-hydroxybenzoic acid synthase gene leads to identification of ansamycin and AHBA-related antibiotic producers in Actinomycetes. *J. Appl. Microbiol.* **106**, 755–763 (2009).

72 Pullen, C. B. *et al.* Occurrence and non-detectability of maytansinoids in individual plants of the genera *Maytenus* and *Putterlickia*. *Phytochemistry* **62**, 377–387 (2003).

73 Zhu, N., Zhao, P. & Shen, Y. Selective isolation and ansamycin-targeted screenings of commensal actinomycetes from the ‘maytansinoid-producing’ arboreal *Trewia nudiflora*. *Curr. Microbiol.* **58**, 87–94 (2009).

74 Wu, Y., Lu, C., Qian, X., Huang, Y. & Shen, Y. Diversities within genotypes, bioactivity and biosynthetic genes of endophytic actinomycetes isolated from three pharmaceutical plants. *Curr. Microbiol.* **59**, 475–482 (2009).
