
UPDATE ON GENE COMPLETIONS AND ANNOTATIONS

# Evolutionary divergence and functions of the human interleukin (IL) gene family

**Chad Brocker,**¹ **David Thompson,**² **Akiko Matsumoto,**¹ **Daniel W. Nebert**³* and **Vasilis Vasiliou**¹

¹ Molecular Toxicology and Environmental Health Sciences Program, Department of Pharmaceutical Sciences, University of Colorado Denver, Aurora, CO 80045, USA  
² Department of Clinical Pharmacy, University of Colorado Denver, Aurora, CO 80045, USA  
³ Department of Environmental Health and Center for Environmental Genetics (CEG), University of Cincinnati Medical Center, Cincinnati, OH 45267–0056, USA  

*Correspondence to:* Tel: +1 513 821 4664; Fax: +1 513 558 0925; E-mail: dan.nebert@uc.edu; vasilis.vasiliou@ucdenver.edu  

Date received (in revised form): 22nd September 2010

## Abstract

Cytokines play a very important role in nearly all aspects of inflammation and immunity. The term ‘interleukin’ (IL) has been used to describe a group of cytokines with complex immunomodulatory functions — including cell proliferation, maturation, migration and adhesion. These cytokines also play an important role in immune cell differentiation and activation. Determining the exact function of a particular cytokine is complicated by the influence of the producing cell type, the responding cell type and the phase of the immune response. ILs can also have pro- and anti-inflammatory effects, further complicating their characterisation. These molecules are under constant pressure to evolve due to continual competition between the host’s immune system and infecting organisms; as such, ILs have undergone significant evolution. This has resulted in little amino acid conservation between orthologous proteins, which further complicates the gene family organisation. Within the literature there are a number of overlapping nomenclature and classification systems derived from biological function, receptor-binding properties and originating cell type. Determining evolutionary relationships between ILs therefore can be confusing. More recently, crystallographic data and the identification of common structural motifs have led to a more accurate classification system. To date, the known ILs can be divided into four major groups based on distinguishing structural features. These groups include the genes encoding the IL1-like cytokines, the class I helical cytokines (IL4-like, γ-chain and IL6/12-like), the class II helical cytokines (IL10-like and IL28-like) and the IL17-like cytokines. In addition, there are a number of ILs that do not fit into any of the above groups, due either to their unique structural features or lack of structural information. This suggests that the gene family organisation may be subject to further change in the near future.

## Keywords: IL, interleukin, cytokine, gene family, evolution, human genome

## Introduction

ILs are a large group of immunomodulatory proteins that elicit a wide variety of responses in cells and tissues. These cytokines comprise a large number of the known immunological ‘second-messenger’ molecules within mammals. Interleukins initiate a response by binding to high-affinity receptors located on the surface of cells; ILs act in a paracrine or autocrine fashion, rather than as an endocrine signal, which is more common with steroidal and amino acid-derived hormones. The response of a particular cell to these cytokines depends on the ligands involved, specific receptors expressed on the cell surface and the particular signalling cascades that are activated. ILs modulate growth, differentiation and activation during an

immune response.¹ This distinguishes them from
chemokines — the main function of which is to
direct immune cells to the site of inflammation via
chemotaxis — and interferons (IFNs), which pre-
dominantly mediate cellular response to viral infec-
tion.¹ Despite attempts to separate these three
groups based on function, there is a degree of
overlap. ILs can exert both inflammatory and anti-
inflammatory actions. A few members act as che-
moattractants for helper T cells, paralleling the
actions of chemokines. Others are intimately
involved in the cellular response to viral pathogens,
making them akin to IFNs. ILs are very important
mediators of the physiological response to infection
and also contribute significantly to the pathophy-
siology of a wide range of disorders. As such, they
represent a group of proteins with potential impor-
tance as therapeutic targets.

In this review, IL-containing cytokine groups
and their closest homologues are described
(Table 1). The grouping of genes is based primarily
on genomic architecture and characteristic protein
structural features (Table 2). Groups that are dis-
cussed include IL1-like cytokines, class I helical
cytokines (IL4-like, γ-chain and IL6/12-like), class
II helical cytokines (IL10-like and IL28-like) and
IL17-like cytokines. A number of ‘non-classified’
ILs that also act as immunomodulating agents are
also reviewed. All official gene and protein names
discussed in this review can be found at http://
www.genenames.org.

### Nomenclature

The term ‘interleukin’ was first coined in 1979 in
a letter to the editor of the *Journal of Immunology*
to describe a number of secreted molecules pro-
duced by leukocytes.² ILs were also described as
originating from lymphocytes; as a result, they are
sometimes referred to as lymphokines.³ A number
of groups have tried to classify members of this
gene family by a variety of characteristics, includ-
ing biological function, receptor-binding proper-
ties, sequence homology, common structural
motifs and the cell types responsible for their pro-
duction. This has resulted in a number of
nomenclature systems and has led to a great deal of
confusion when describing individual genes and
gene families.

A common interleukin nomenclature system was
approved by the International Union of
Immunological Societies (IUIS) and the World
Health Organization (WHO) Nomenclature
Subcommittee for IL1–IL10, including IL1A and
IL1B.⁴ Designations for IL11–IL13 were approved
and updated by IUIS/WHO in 1994, IL16 in
1996 and IL14–IL15 in 1997.⁵⁻⁷ Sims and col-
leagues recommended a new nomenclature system
in 2001 for IL1 family genes.⁸ This nomenclature
scheme was based on peptide sequence conserva-
tion, common structural elements, gene architec-
ture and publication date.

The designation of cytokine and IL subfamilies
further complicates IL nomenclature. Boulay and
colleagues grouped IL3, IL4, IL5 and
colony-stimulating factor 2 (CSF 2) into the ‘IL4
family’ of T cell-derived lymphokines.⁹ This classi-
fication was based on shared genetic and structural
elements, biological functions and receptor-binding
properties. Liles and Van Voorhis separated the 42
cytokine genes known at the time into six groups
(namely, haematopoietic growth factors, IFNs, lym-
phokines, monokines, chemokines and other cyto-
kines) — based primarily on biological function and
originating cell type.³ This classification system
placed the majority of interleukins (IL2–IL6, IL9–
IL10 and IL13–IL14) within the ‘lymphokine’
group. IL1A, IL1B, IL1RA, IL12 and IL15 were
classified as ‘monokines’. IL8 was considered a ‘che-
mokine’ and IL7, IL11 and the leukaemia inhibitory
factor (LIF) gene were considered ‘other cytokines’.

More recently, sequence and structural compari-
sons have come to the forefront of evolutionary
analysis. Using these methods, evolutionary
relationships can be accurately determined between
genes and gene families. Such a system has been
applied successfully to many gene families, using
conserved nucleotide and amino acid sequences to
decipher evolutionary ties between genes, which
continues to be the underlying theme of these
‘Updates on Gene Completions and Annotations’
papers in *Human Genomics*.

| Gene | Full gene name | Aliases | Chromosome | Isoforms | Refseq mRNA number | Refseq protein number | AA# |
|------|----------------|---------|------------|----------|--------------------|-----------------------|-----|
| ILIA | Interleukin 1, alpha | ILIF1, ILIA, IL-1A, ILI, ILI-ALPHA | 2q14 |  | NM_000575.3 | NP_000566.3 | 270 |
| ILIB | Interleukin 1, beta | ILIF2, ILIB, IL-I, ILI-BETA | 2q14 |  | NM_000576.2 | NP_000567.1 | 268 |
| ILIRN | Interleukin 1 receptor antagonist | DIRA, IRAP, ILIF3, ILIRA, MVCD4, ILIRN, ILIra, ILIra3, ICILIRA, MGC10430, ICIL-IRA, IL-IRN | 2q14.2 |  | NM_173842.2 | NP_776214.1 | 177 |
| ILIF5 | Interleukin 1 family, member 5 | UNQ1896/, PRO4342, FILI, FILI (DELTA), FILID, ILIHYI, ILILI, ILIRP3, MGC29840 | 2q14 |  | NM_173170.1 | NP_775262.1 | 154 |
| ILIF6 | Interleukin 1 family, member 6 | FILI, FILI (EPSILON), FILIE, IL-1F6, ILI (EPSILON), MGC129552, MGC129553 | 2q12-q14.1 |  | NM_014440.1 | NP_055255.1 | 157 |
| ILIF7 | Interleukin 1 family, member 7 | FILI, FILI (ZETA), FILIZ, IL-1F7, IL-1H4, IL-1RP1, ILIH4, ILIRPI | 2q12-q14.1 | Isoform 1 | NM_014439.3 | NP_055254.2 | 217 |
|  |  |  |  | Isoform 2 | NM_173202.1 | NP_775294.1 | 197 |
|  |  |  |  | Isoform 3 | NM_173203.1 | NP_775295.1 | 157 |
|  |  |  |  | Isoform 4 | NM_173204.1 | NP_775296.1 | 178 |
|  |  |  |  | Isoform 5 | NM_173205.1 | NP_775297.1 | 192 |
| ILIF8 | Interleukin 1 family, member 8 | FILI, FILI-(ETA), FILIH, IL-1F8, IL-1H2, ILI-ETA, ILIH2, MGC126880, MGC126882 | 2q14 |  | NM_014438.3 | NP_055253.2 | 163 |

| Gene   | Full gene name                     | Aliases                                                                 | Human                                                                 |
|--------|------------------------------------|-------------------------------------------------------------------------|-----------------------------------------------------------------------|
|        |                                    | Chromosome                                                             | Isoforms                                                              | Ref seq mRNA number | Ref seq protein number | AA# |
| IL1F9  | Interleukin 1 family, member 9     | UNQ2456/, PRO5737, IL-IF9, IL-IHI, IL-IRP2, ILIE, ILIHI, ILIRP2          | 2q12-q21                                                              | NM_019618.2         | NP_062564.1            | 168 |
| IL1F10 | Interleukin 1 family, member 10    | FKSG75, FILI-theta, IL-IHY2, ILI-theta, MGC119831, MGC119832, MGC119833 | 2q13                                                                  | NM_032556.4         | NP_115945.4            | 151 |
| IL2    | Interleukin 2                     | IL-2, TCGF, lymphokine                                                | 4q26-q27                                                              | NM_000586.3         | NP_000577.2            | 152 |
| IL3    | Interleukin 3                     | IL-3, MCGF, MGC79398, MGC79399, MULTI-CSF                             | 5q23-q31                                                              | NM_000588.3         | NP_000579.2            | 151 |
| IL4    | Interleukin 4                     | BCGF-I, BCGFI, BSFI, IL-4, MGC79402                                   | 5q23-q31                                                              | Isoform 1           | NM_000589.2            | NP_000580.1 | 152 |
|        |                                    |                                                                       |                                                                       | Isoform 2           | NM_172348.1            | NP_758858.1 | 137 |
|        |                                    |                                                                       |                                                                       | Isoform 2           | NM_172348.1            | NP_758858.1 | 137 |
| IL5    | Interleukin 5                     | EDF, IL-5, TRF                                                        | 5q23-q31                                                              | NM_000879.2         | NP_000870.1            | 133 |
| IL6    | Interleukin 6                     | BSF2, HGF, HSF, IFNB2, IL-6                                           | 7p21                                                                  | NM_000600.3         | NP_000591.1            | 211 |
| IL7    | Interleukin 7                     | IL-7                                                                  | 8q12-q13                                                              | NM_000880.2         | NP_000871.1            | 176 |
| IL8    | Interleukin 8                     | CXCL8, GCP-I, GCP1, LECT, LUCT, LYNAp, MDNCF, MONAP, NAF, NAP-I, NAPI | 4q13-q21                                                              | NM_000584.2         | NP_000575.1            | 98  |
| IL9    | Interleukin 9                     | HP40, IL-9, P40                                                       | 5q31-q35                                                              | NM_000590.1         | NP_000581.1            | 143 |
| IL10   | Interleukin 10                    | RP11-262N9.I, CSIF, IL-10, IL10A, MGC126450, MGC126451, TGIF             | 1q31-q32                                                              | NM_000572.2         | NP_000563.1            | 177 |
| IL11   | Interleukin 11                    | AGIF, IL-II                                                           | 19q13.3-q13.4                                                         | NM_000641.2         | NP_000632.1            | 198 |
|        |                                    |                                                                       |                                                                       |                                                                                 | Continued              |

| Gene | Full gene name | Aliases | Human |
| --- | --- | --- | --- |
|  |  |  | Chromosome |
|  |  |  | Isoforms |
|  |  |  | Refseq mRNA |
|  |  |  | Refseq protein |
|  |  |  | AA# |
| IL12A | Interleukin 12A | CLMF, IL-12A, NFSK, NKSFI, P35 | 3q25.33-q26 | NM_000882.2 | NP_000873.2 | 252 |
| IL13 | Interleukin 13 | ALRH, BHR1, IL-13, MGC116786, MGC116788, MGC116789, P600 | 5q31 | NM_002188.2 | NP_002179.2 | 145 |
| TXLNA | Taxilin alpha | IL14, DKFZp451J0118, MGC118870, MGC118871, RP4-622L5.4, TXLN | 1p35.1 | NM_175852.3 | NP_787048.1 | 546 |
| IL15 | Interleukin 15 | IL-15, MGC9721 | 4q31 | NM_001044731.1 | NP_000576.1 | 161 |
| IL16 | Interleukin 16 | FLJ16806, FLJ42735, FLJ44234, LCF, NIL16, PRIL16, prlL-16 | 15q26.3 | Isoform 1 | NM_004513.5 | NP_004504.3 | 631 |
|  |  |  |  | Isoform 2 | NM_172217.3 | NP_757366.2 | 1332 |
|  |  |  |  | Isoform 3 | NM_001172128.1 | NP_001165599.1 | 1331 |
| IL17A | Interleukin 17A | CTLA8, IL-17, IL-17A, IL17 | 6p12 | NM_002190.2 | NP_002181.1 | 154 |
| IL17B | Interleukin 17B | UNQ516/, PRO1031, IL-17B, IL-20, MGC138900, MGC138901, ZCYTO7 | 5q33.1 | NM_014443.2 | NP_055258.1 | 179 |
| IL17C | Interleukin 17C | UNQ561/, PRO1122, CX2, IL-17C, IL-21, MGC126884, MGC138401 | 16q24 | NM_013278.3 | NP_037410.1 | 196 |
| IL17D | Interleukin 17D | IL-22, IL-27, IL-17D, FLJ30846 | 13q11 | NM_138284.1 | NP_612141.1 | 201 |
| IL17F | Interleukin 17F | IL-17F, ML-1, ML1 | 6p12 | NM_052872.3 | NP_443104.1 | 162 |
| IL18 | Interleukin 18 | IGIF, IL1F4, IL-18, IL-1g, MGC12320 | 11q22.2-q22.3 | NM_001562.2 | NP_001553.1 | 192 |
| IL19 | Interleukin 19 | RPII-262N9.2, IL-10C, MDAl, NG.I, ZMDAl | 1q32.2 | Isoform 1 | NM_153758.1 | NP_715639.1 | 214 |
|  |  |  |  | Isoform 2 | NM_013371.2 | NP_037503.2 | 177 |

| Gene   | Full gene name     | Aliases                                                                 | Human                                                                 |
|--------|--------------------|-------------------------------------------------------------------------|-----------------------------------------------------------------------|
|        |                    | Chromosome                                                             | Isoforms                                                              | Ref seq mRNA number | Ref seq protein number | AA# |
| IL20   | Interleukin 20     | UNQ852/, PRO1801, IL-20, IL10D, MGC96907, ZCYTO10                         | 1q32                                                                  | NM_018724.3         | NP_061194.2            | 175 |
| IL21   | Interleukin 21     | Za11, IL-21                                                            | 4q26-q27                                                              | NM_021803.2         | NP_068575.1            | 161 |
| IL22   | Interleukin 22     | UNQ3099/, PRO10096, IL-21, IL-22, IL-D110, IL-TIF, IL21,ILTIF, MGC79382,    | 12q15                                                                 | NM_020525.4         | NP_065386.1            | 178 |
|        |                    | MGC79384, TIFIL-23, TIFa, zcyto18                                       |                                                                       |                       |                          |      |
| IL23A  | Interleukin23A     | UNQ2498/, PRO5798, IL23, IL-23A, IL23P19, MGC79388, P19, SGRF              | 12q13.13                                                              | NM_016584.2         | NP_001717165.2         | 205 |
| IL24   | Interleukin 24     | C49A, FISP, IL10B, MDA7, MOB5, ST16                                      | 1q32                                                                  | Isoform 1           | NM_006850.2            | NP_006841.1          | 205 |
|        |                    |                                                                       |                                                                       | Isoform 3           | NM_001185156.1         | NP_001172085.1       | 207 |
|        |                    |                                                                       |                                                                       | Isoform 4           | NM_001185157.1         | NP_001172086.1       | 154 |
|        |                    |                                                                       |                                                                       | Isoform 5           | NM_001185158.1         | NP_001172087.1       | 63  |
| IL25   | Interleukin 25     | UNQ3120/, PRO10272, IL17E                                               | 14q11.2                                                               | Isoform 1           | NM_022789.3            | NP_073626.1          | 176 |
|        |                    |                                                                       |                                                                       | Isoform 2           | NM_172314.1            | NP_758525.1          | 161 |
| IL26   | Interleukin 26     | AK155, IL-26                                                           | 12q15                                                                 | NM_018402.1         | NP_060872.1            | 170 |
| IL27A  | Interleukin 27     | IL-27, IL-27A, IL27p28, IL30, MGC71873, p28                              | 16p11                                                                 | NM_145659.3         | NP_663634.2            | 242 |
| IL28A  | Interleukin28A     | IFNL2, IL-28A                                                          | 19q13.13                                                              | NM_172138.1         | NP_742150.1            | 199 |
| IL28B  | Interleukin28B     | IFNL3, IL-28B, IL28C                                                   | 19q13.13                                                              | NM_172139.2         | NP_742151.2            | 195 |
| IL29   | Interleukin 29     | IFNLI, IL-29                                                           | 19q13.13                                                              | NM_172140.1         | NP_742152.1            | 200 |
| IL31   | Interleukin 31     | IL-31                                                                 | 12q24.31                                                              | NM_001014336.1      | NP_001014358.1         | 164 |

| Gene | Full gene name | Aliases | Human |
| --- | --- | --- | --- |
|  |  |  | Chromosome |
|  |  |  | Isoforms |
|  |  |  | Ref seq mRNA |
|  |  |  | Ref seq protein |
|  |  |  | AA# |
| IL32 | Interleukin 32 | IL-32alpha, | 16p13.3 |
|  |  | IL-32beta, |  |
|  |  | IL-32delta, |  |
|  |  | IL-32gamma, NK4, |  |
|  |  | TAIF, TAIFa, TAIFb, |  |
|  |  | TAIFc, TAIFd |  |
|  |  |  | Isoform a |
|  |  |  | NM_001012633.1 |
|  |  |  | NP_001012651.1 |
|  |  |  | 131 |
|  |  |  | Isoform b |
|  |  |  | NM_001012631.1 |
|  |  |  | NP_001012649.1 |
|  |  |  | 188 |
|  |  |  | NM_001012632.1 |
|  |  |  | NP_001012650.1 |
|  |  |  | NM_001012718.1 |
|  |  |  | NP_001012736.1 |
|  |  |  | NM_004221.4 |
|  |  |  | NP_004212.4 |
|  |  |  | Isoform c |
|  |  |  | NM_001012634.1 |
|  |  |  | NP_001012652.1 |
|  |  |  | 168 |
|  |  |  | NM_001012635.1 |
|  |  |  | NP_001012653.1 |
|  |  |  | Isoform d |
|  |  |  | NM_001012636.1 |
|  |  |  | NP_001012654.1 |
|  |  |  | 179 |
| IL33 | Interleukin 33 | ILIF11, C9orf26, | 9p24.1 |
|  |  | DKFZp586H0523, |  |
|  |  | DVS27, NF-HEV, |  |
|  |  | NFEHEV, |  |
|  |  | RP11-575C20.2 |  |
|  |  |  | NM_033439.2 |
|  |  |  | NP_254274.1 |
|  |  |  | 270 |
| IL34 | Interleukin 34 | CL6orf77, IL-34, | 16q22.1 |
|  |  | MGC34647 |  |
|  |  |  | NM_001172772.1 |
|  |  |  | NP_001166243.1 |
|  |  |  | 242 |
| CNTF | Ciliary neurotrophic factor | HCN TF | 11q12.2 |
|  |  |  | NM_000614.3 |
|  |  |  | NP_000605.1 |
|  |  |  | 200 |
| CTF1 | Cardiotrophin 1 | CT1, CT-I | 16p11.2 |
|  |  |  | Isoform 1 |
|  |  |  | NM_001330.3 |
|  |  |  | NP_001321.1 |
|  |  |  | 201 |
|  |  |  | Isoform 2 |
|  |  |  | NM_001142544.1 |
|  |  |  | NP_001136016.1 |
|  |  |  | 200 |
| CLCF1 | Cardiotrophin-like cytokine factor 1 | CLC, NR6, BSF3, | 11q13.3 |
|  |  | NNT1, BSF-3, |  |
|  |  | CISS2, NNT-I |  |
|  |  |  | Isoform 1 |
|  |  |  | NM_013246.2 |
|  |  |  | NP_037378.1 |
|  |  |  | 225 |
|  |  |  | Isoform 2 |
|  |  |  | NM_001166212.1 |
|  |  |  | NP_001159684.1 |
|  |  |  | 215 |
| CSF1 | Colony-stimulating factor 1 | MCSF, MGC31930 | 1p21-p13 |
|  |  |  | Isoform 1 |
|  |  |  | NM_000757.4 |
|  |  |  | NP_000748.3 |
|  |  |  | 554 |
|  |  |  | Isoform 2 |
|  |  |  | NM_172210.2 |
|  |  |  | NP_757349.1 |
|  |  |  | 438 |
| CSF2 | Colony-stimulating factor 2 | GMCSF, | 5q31.1 |
|  |  | MGC31935, |  |
|  |  | MGC138897, |  |
|  |  | GM-CSF |  |
|  |  |  | NM_000758.2 |
|  |  |  | NP_000749.2 |
|  |  |  | 144 |
| CSF3 | Colony-stimulating factor 3 | GCSF, CSF3OS, | 17q11.2-q12 |
|  |  | CL7orf33, |  |
|  |  | MGC45931 |  |
|  |  |  | Isoform a |
|  |  |  | NM_000759.3 |
|  |  |  | NP_000750.1 |
|  |  |  | 207 |
|  |  |  | Isoform b |
|  |  |  | NM_172219.2 |
|  |  |  | NP_757373.1 |
|  |  |  | 204 |
|  |  |  | Isoform c |
|  |  |  | NM_172220.2 |
|  |  |  | NP_757374.2 |
|  |  |  | 171 |
|  |  |  | Isoform d |
|  |  |  | NM_001178147.1 |
|  |  |  | NP_001171618.1 |
|  |  |  | 168 |
| LIF | Leukaemia inhibitory factor | CDF, DIA, HILDA | 22q12.2 |
|  |  |  | NM_002309.3 |
|  |  |  | NP_002300.1 |
|  |  |  | 202 |

Evolutionary divergence and functions of the human interleukin (IL) gene family

UPDATE ON GENE COMPLETIONS AND ANNOTATIONS

Table 1. Concluded

| Gene | Full gene name | Aliases | Human |
|------|----------------|---------|--------|
|      |                |         | Chromosome |
|      |                |         | Isoforms |
|      |                |         | Ref seq mRNA |
|      |                |         | number |
|      |                |         | Ref seq protein |
|      |                |         | number |
|      |                |         | AA# |
| OSM  | Oncostatin M   | MGC20461 | 22q12.2 | NM_020530.3 | NP_065391.1 | 252 |
| TSLP | Thymic stromal lymphopoietin | TSLP | 5q22.1 | NM_033035.4 | NP_149024.1 | 159 |

Unfortunately, sequence analysis proves difficult when analysing highly divergent gene families, and the ILs are a perfect example of such a family. In these cases, structural comparisons often provide better insight into homologous protein groups. As with many of the IL groups, it has been hypothesised that a few invariable amino acids are required to maintain the overall conserved fold structures, leading to little sequence homology between evolutionarily related group members. This review focuses on IL classification using known structural motifs. The ILs can be separated into groups and subgroups using common structural elements identified from three-dimensional protein structures. This type of classification is useful, especially when applied to cytokines that share very little sequence homology. The accuracy is, obviously, solely dependent on the amount of structural information available within existing databases and publications, however.

### Interleukin structure

Structural comparisons are a very useful way to classify evolutionarily related proteins. The IL1-like cytokines can be distinguished from other cytokines by the presence of a fold rich in β-strands. Crystallographic comparison of IL1A, IL1B and IL1RA identified a shared fold structure comprising 12 packed β-sheets, known as a β-trefoil.

A number of ILs are classified as either class I or class II helical cytokines. Class I (or type I) cytokines share a common structural fold characterised by four tightly packed α-helices arranged in an 'up-up-down-down' orientation, often referred to as a 'four-helix bundle'. Class II (or type II) proteins are distinguished by a similar 'bundled helix'

structural motif containing six or seven stacked helices.

The class I cytokines can then be further subdivided into either 'long-chain' or 'short-chain', based on the length of the core helices making up the bundle. The length of the internal bundles affects overall peptide length; consequently, long-chain cytokines are generally >165 amino acids in length and short-chain usually contain <165 residues. IL6, IL11, IL12A, IL23A, IL27A, IL31, cardiotrophin-like cytokine factor 1 (CLCF1), ciliary neurotrophic factor (CNTF), cardiotrophin 1 (CTF1), LIF, oncostatin M (OSM) and CSF3 are long-chain class I helical cytokines. The short-chain helical cytokines include IL2, IL3, IL4, IL5, IL7, IL9, IL13, IL15 and IL21. Thymic stromal lymphopoietin (TSLP) and CSF2 also fall into the short-chain class I cytokines.

The IL10-like and IL28-like (as well as some IFNs) are class II cytokines and are structurally related through a shared characteristic fold that contains six or seven stacked α-helices arranged in an anti-parallel conformation. The IL28-like genes also share similar intron-exon structure with the IL10-like cytokines. The IL10-like and IL28-like ILs, together with the type I and type II IFNs, have also been called the IL10–IFN gene family. The IL10-like proteins including IL10, IL19, IL20, IL22, IL24 and IL26. IL28A, IL28B and IL29 comprise the IL28-like group.

IL17A and its homologues are structurally unrelated to other IL groups. IL17F shares a high degree of sequence homology with IL17A. IL17F was recently crystallised and structural analysis revealed the presence of a cysteine-knot fold. This structural motif is characteristic of neurophins, which are a distinct class of secreted growth

UPDATE ON GENE COMPLETIONS AND ANNOTATIONS

Table 2. Human IL classification. The common features and structural motifs that were used for classification are listed in the right-hand column

| Group            | Genes                                                                 | Common features and structural motifs                                                                 |
|------------------|----------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
| IL1-like         | IL1A, IL1B, IL1RN, IL1F5, IL1F6, IL1F7, IL1F8, IL1F9, IL1F10, IL18, IL33 | Fold rich in β-strands known as a β-trefoil                                                          |
| γ-chain utilising ('short-chain' class I) | IL2, IL4, IL7, IL9, IL15, IL21, TSPL | Four tightly packed α-helices known as 'four-helix bundle' motif; shorter core helices; receptor complex contains γc chain subunit |
| IL4-like ('short-chain' class I) | IL3, IL4, IL5, IL13, CSF2 | Four tightly packed α-helices known as 'four-helix bundle' motif; shorter core helices |
| IL6/12-like ('long-chain' class I) | IL6, IL11, IL12A, IL23A, IL27A, IL31, CLCF1, CNTF, CTF1, LIF, OSM, CSF3 | Four tightly packed α-helices known as 'four-helix bundle' motif; longer core helices |
| IL10-like (class II) | IL10, IL19, IL20, IL22, IL24, IL26 | 'Bundled helix' structural motif containing six or seven stacked helices |
| IL28-like (class II) | IL28A, IL28B, IL29 | 'Bundled helix' structural motif containing six or seven stacked helices |
| IL17-like        | IL17A, IL17B, IL17C, IL17D, IL25 (aka IL17E), IL17F | Neurotrophin-like cysteine-knot fold |
| Non-classified   | IL8, TXLNA (aka IL14), IL16, IL32, IL34, CSF1 | Varies |

factors involved in the growth, differentiation and survival of particular cell types, including neurones.

### Evolution

Genes associated with the immune system are under constant evolutionary pressure to change as a result of host-parasite co-evolution, where advantageous mutations are heavily favoured. Consequently, cytokines are some of the most rapidly evolving genes within mammals. A study comparing the degree of evolutionary divergence between mouse and human orthologues showed that seven of the top 25 most highly divergent genes are cytokines. In spite of a high degree of divergence within mammalian cytokines, a pheromone isolated from the marine protozoan, *Euploea raikovi*, was found to share some structural similarities with IL2. Further studies revealed that the protozoan pheromone was able to bind to the mammalian IL2 receptor. The same authors showed that mammalian IL2 was able to bind to the corresponding pheromone receptor located on the ciliated cell surface of the protozoan, suggesting that ILs may be evolutionarily grounded in ancient cell-signalling systems. Despite lacking a 'true' immune system, invertebrates are capable of mounting a highly effective inflammatory-like response similar to that observed in higher eukaryotes. It appears that the invertebrate system arose independently, however, utilising analogous mechanisms and convergent evolution.

IL1 family members are clustered within 430 kilobases (kb) on human chromosome (Chr) 2q13, except for IL18 and IL33, which reside on Chr 11 and Chr 9, respectively. The close proximity of IL1 family members on Chr 2 indicates that they most likely originated from a common ancestral gene that underwent multiple gene duplications. Gene duplications are also common within the class I helical cytokines. IL2 and IL21 are separated by less than 244 kb on 4q26, suggesting another gene duplication event that led to multiple, functionally distinct proteins. The duplication is also found within the syntenic region on mouse Chr 3A3. Human IL3, IL4, IL5 and CSF2 localise to Chr 5q23-32. IL3 and CSF2 are closely

linked and separated by only 9 kb.³⁰ Although tightly linked, the genes share relatively low amino acid homology.³² IL5 and CSF2 also share a conserved regulatory element within their respective promoter regions.³² The four genes in the cluster (IL3, IL4, IL5 and CSF2) have similar gene structure, biological activities and regulatory mechanisms. This same cluster is conserved within the mouse genome on Chr 11A1.³¹,³³ The more recent discovery of the IL9 and IL13 genes places these members relatively close to one another on Chr5q31, indicating that they may also be part of the same gene cluster.³⁴ The genomic structure of the mouse orthologues is conserved on mouse Chr 11, further supporting origins grounded in more ancient duplication events.³³,³⁵ LIF and OSM are structurally similar growth factors that share many functions with IL6 and CNTF. The LIF and OSM genes are separated by <16 kb on Chr22q12.³⁶,³⁷ The close physical linkage of these two related genes suggests that they are evolutionarily related and, like many other cytokines, are likely to be the product of gene duplication. The relatively low degree of sequence homology suggests rapid divergent evolution subsequent to the ancient gene duplications.

The class II helical cytokine genes, IL10, IL19, IL20 and IL24, form a tightly linked cluster on Chr 1q32.¹⁷ IL22, IL26 and a related gene, IFN gamma (IFNG), form an additional gene cluster on Chr 12q15.³⁸ IL28A, IL28B and IL29 form a tight gene cluster on Chr19q3 similar to the mouse genes, Il28a, Il28b and Il29, which cluster on Chr7A3.³⁹

IL17A and IL17F are linked on Chr6 in humans and in the syntenic region on Chr1 in mice in a tail-to-tail configuration.⁴⁰ The remaining IL17-like cytokines are spread throughout the genome. IL17B, IL17C, IL17D and IL25 are located on human Chr 5q33, 16q24, 13q11 and 14q11, respectively.⁴¹ Interestingly, a number of groups have suggested a high degree of co-evolution between ILs and their receptors. One example is IL8, which is clustered with a number of structurally related chemokines on human Chr4.⁴² The secretory proteins in this cluster bind to a specific family of G-protein-coupled receptors found within another gene cluster located on Chr2.⁴³ A similar situation

has also been observed with IL6 ligand/receptor pairs, which appear also to have undergone co-evolution.⁴⁴ Evolutionary expansion through the tandem duplication of ligand and receptor genes located on these chromosomes, followed by divergence and co-evolution, led to the diversification of ligands and their cognate receptor gene families.⁴²

Gene duplication events and divergent evolution have played an important role in the expansion and diversity of this gene family. This expansion is fairly apparent, especially in vertebrates, where some genes underwent multiple rounds of duplication.¹⁴,⁴⁵,⁴⁶ The gene clusters for known human ILs are summarised in Table 3.

### IL1-like cytokines

There are currently 11 IL1-like proteins and each is encoded by a distinct gene—that is, IL1A, IL1B, IL1RN, IL1F5, IL1F6, IL1F7, IL1F8, IL1F9, IL1F10, IL18 and IL33 (Table 2).⁴⁷,⁴⁸ IL1-like gene products are primarily secreted by stimulated monocytes or macrophages;⁴⁹ however, other immune- and non-immune-related cell types (including neutrophils, lymphocytes, keratinocytes, chondrocytes, fibroblasts and endothelial, epithelial and smooth muscle cells) appear also to produce these proteins.⁵⁰,⁵¹ Receptor activation by IL1-like proteins plays a role in immune regulation and inflammatory processes by triggering transcriptional activation of downstream effector proteins. IL1A and IL1B rapidly induce transcription of hundreds of downstream targets, including other cytokines/chemokines, nitric oxide synthase (NOS), matrix metalloproteinases (MMPs), as well as activating their own genes.⁵²–⁵⁴ Dysregulation of IL1-like cytokine signalling is associated with many inflammatory and autoimmune disorders, including inflammatory bowel disease (IL22), rheumatoid arthritis (IL20 and IL22) and psoriasis (IL19, IL20 and IL22).⁵⁵–⁵⁸ As a result, repression of the actions of IL1-like proteins is regarded as a promising therapeutic intervention to treat these and other common inflammatory disorders involving IL1-like cytokines.⁵⁹,⁶⁰

IL1A, IL1B, IL18 and IL33 are expressed as precursor (pro-) proteins that are proteolytically

UPDATE ON GENE COMPLETIONS AND ANNOTATIONS

Table 3. Human interleukin (IL) gene clusters. The genes within each cluster are listed, as well as the chromosomal location

| Chromosome | Genes |
|------------|-------|
| 1q32       | IL10, IL19, IL20, IL24 |
| 2q13       | IL1A, IL1B, IL1RN, IL1F5, IL1F6, IL1F7, IL1F8, IL1F9, IL1F10 |
| 4q26       | IL2, IL21 |
| 5q31       | IL3, IL4, IL5, IL9, IL13, CSF2 |
| 6p12       | IL17A, IL17F |
| 12q15      | IL22, IL26, IFNG |
| 16pll      | IL27A, CTFI |
| 19q13      | IL28A, IL28B, IL29 |
| 22q12      | LIF, OSM |

co-receptor used by IL1A/B — namely, IL1RAP. It is interesting to note that five of the six IL1-like receptors form a closely linked gene cluster on Chr 2.

The IL1-like genes have been grouped on the basis of sequence homology, genomic architecture and chromosomal location (Table 1). They also share common signal transduction mechanisms. The ‘classical’ IL1-like genes include IL1A, IL1B and IL1RN. All three genes have been mapped to the same chromosomal region in both humans and mice. Further investigation led to the discovery of an additional six IL1-like genes (IL1F5, IL1F6, IL1F7, IL1F8, IL1F9 and IL1F10) located within the same gene cluster (Table 3). IL18 and IL33 are found on Chr 11 and 9, respectively. Amino acid sequence identity within the group is quite varied, with identities ranging from 10–54 percent (Table 4). IL18 and IL33 exhibit considerably less sequence homology with the other family members and are considered more distantly related. The close evolutionary relationship that exists between these proteins is exemplified in the tightly grouped branched structure formed within the IL/cytokine phylogram (Figure 1).

cleaved to form mature secretory peptides. The cleavage of pro-IL1A and pro-IL33 is mediated by calpain, whereas pro-IL1B and pro-IL18 cleavage is dependent on caspase-1. IL1A and IL1B initiate a strong pro-inflammatory signalling cascade. Both ligands bind to IL1 receptor type I (IL1R1) and signal transduction is initiated after recruitment of the IL1 receptor accessory protein (IL1RAP). They can also bind to the IL1 receptor type II (IL1R2), which is a non-signalling complex that serves as a ‘decoy receptor’. IL1 receptor antagonist (IL1RN; also known as IL1RA) directly competes with IL1A and IL1B for binding to IL1R1 but does not activate the receptor, therefore producing anti-inflammatory effects. IL18 and IL33 are both pro-inflammatory and each uses a related but distinct receptor and accessory protein pair for signal transduction. IL18 binds to a heterodimeric receptor complex containing IL18 receptor 1 (IL18R1) and the IL18R accessory protein (IL18RAP). IL33 serves as a ligand for IL1 receptor-like 1 (IL1RL1; also known as ST2) and the same

---

γ-chain cytokines

The common γ-chain (γc)-utilising cytokines include IL2, IL4, IL7, IL9, IL15, IL21 and TSLP (Table 2). Each gene encodes a protein of <180 amino acids and they are considered to be short-chain class I cytokines. All members of this group bind to a receptor complex that shares the γc chain (also known as IL2Rγ), with the exception of TSLP, which instead uses a closely related chain known as cytokine receptor-like factor 2 (CRLF2). The IL4, IL7, IL9, IL21 and TSLP proteins bind as heterodimeric receptor complexes, whereas IL2 and IL15 utilise a receptor with three subunits. In addition to γc, the heterodimeric receptors contain a cytokine-specific receptor chain (IL4Ra, IL7Ra, IL9Ra or IL21Ra). TSLP, similarly to IL7, utilises IL7Ra. The IL2 and IL15 receptors contain either IL2RB or IL15Ra, respectively, as well as both the γc and IL2RB chains. Ligand-binding and receptor multimerisation

Table 4. Amino acid sequence homology between human IL proteins, as well as closely related cytokines. Percentage identity and similarity (in parentheses) were determined using the EMBOSS Pairwise Alignment Algorithm (www.ebi.ac.uk/Tools/emboss/align/) and non-redundant human protein sequences. The predominant, mature isoform sequences were used for proteins having multiple variants

| ILI-like | ILIA | ILIB | ILIRN | ILI8 | ILIF5 | ILIF6 | ILIF7 | ILIF8 | ILIF9 | ILIFI0 |
|----------|------|------|-------|------|-------|-------|-------|-------|-------|---------|
| ILIB     |      | 23.3% (39.5%) |       |       |       |       |       |       |       |         |
| ILIRN    | 14.5% (24.8%) | 19.4% (28.5%) |       |       |       |       |       |       |       |         |
| ILI8     | 13.7% (25.8%) | 13.2% (28.6%) | 15.9% (28.8%) |       |       |       |       |       |       |         |
| ILIF5    | 12.0% (19.7%) | 15.4% (23.3%) | 40.1% (48.7%) | 11.4% (20.4%) |       |       |       |       |       |         |
| ILIF6    | 10.6% (15.2%) | 15.8% (22.6%) | 24.4% (38.1%) | 18.3% (31.7%) | 29.1% (42.3%) |       |       |       |       |         |
| ILIF7    | 14.2% (26.1%) | 21.2% (35.2%) | 21.2% (37.2%) | 17.6% (32.2%) | 24.4% (34.7%) | 25.0% (37.7%) |       |       |       |         |
| ILIF8    | 9.1% (19.3%) | 12.4% (18.6%) | 14.5% (22.8%) | 7.7% (16.5%) | 14.6% (25.9%) | 24.5% (35.5%) | 18.6% (32.5%) |       |       |         |
| ILIF9    | 14.0% (21.7%) | 11.8% (20.6%) | 22.7% (35.0%) | 21.0% (35.7%) | 26.1% (39.4%) | 51.4% (64.8%) | 26.9% (41.6%) | 23.4% (36.8%) |       |         |
| ILIFI0   | 11.7% (20.9%) | 12.4% (22.0%) | 33.3% (44.8%) | 14.4% (32.7%) | 39.3% (54.6%) | 24.9% (39.9%) | 21.2% (35.8%) | 12.3% (21.1%) | 27.4% (42.5%) |         |
| IL33     | 12.7% (21.8%) | 16.1% (29.8%) | 17.7% (28.0%) | 12.8% (24.7%) | 13.4% (20.5%) | 13.6% (23.0%) | 7.3% (12.7%) | 5.6% (11.5%) | 4.0% (6.1%) | 4.5% (8.9%) |

| γ-chain utilising | IL2 | IL7 | IL9 | ILI5 | IL2I |
|--------------------|-----|-----|-----|------|------|
| IL7                |     | 5.7% (10.4%) |       |       |       |
| IL9                | 16.9% (29.2%) | 17.6% (28.0%) |       |       |       |
| ILI5               | 18.2% (33.7%) | 16.7% (27.3%) | 17.3% (28.1%) |       |       |
| IL2I               | 19.5% (33.5%) | 15.0% (29.2%) | 17.9% (30.0%) | 21.9% (38.8%) |       |
| TSPL               | 5.8% (11.6%) | 19.5% (31.0%) | 18.6% (29.5%) | 10.5% (16.9%) | 17.7% (28.3%) |

UPDATE ON GENE COMPLETIONS AND ANNOTATIONS

Table 4. Continued

**IL4-like**

| IL3 | IL4 | IL5 | IL13 |
| --- | --- | --- | --- |
| IL4 | 22.1% |  |  |
|  | (30.4%) |  |  |
| IL5 | 23.5% | 21.4% |  |
|  | (34.2%) | (33.0%) |  |
| IL13 | 24.7% | 23.6% | 23.9% |
|  | (38.7%) | (32.5%) | (35.2%) |
| CSF 2 | 22.0% | 24.4% | 25.6% | 21.5% |
|  | (32.8%) | (35.1%) | (41.1%) | (34.9%) |

**IL6/12-like**

| IL6 | IL11 | IL12A | IL23A | IL27A | IL31 | CLCF1 | CNTF |
| --- | --- | --- | --- | --- | --- | --- | --- |
| IL11 | 20.0% |  |  |  |  |  |  |
|  | (35.1%) |  |  |  |  |  |  |
| IL12A | 16.4% | 3.5% |  |  |  |  |  |
|  | (28.9%) | (5.4%) |  |  |  |  |  |
| IL23A | 19.7% | 19.8% | 18.9% |  |  |  |  |
|  | (30.8%) | (25.5%) | (29.4%) |  |  |  |  |
| IL27A | 19.8% | 22.1% | 4.4% | 16.2% |  |  |  |
|  | (31.7%) | (32.0%) | (5.8%) | (25.6%) |  |  |  |
| IL31 | 15.7% | 10.8% | 7.6% | 21.0% | 12.7% |  |  |
|  | (28.8%) | (16.0%) | (15.2%) | (32.9%) | (18.0%) |  |  |
| CLCF1 | 17.0% | 21.5% | 15.0% | 13.3% | 20.3% | 10.2% |  |
|  | (29.3%) | (28.5%) | (23.9%) | (20.1%) | (29.7%) | (16.4%) |  |
| CNTF | 15.6% | 8.7% | 11.2% | 12.5% | 18.9% | 13.9% | 20.1% |
|  | (28.1%) | (10.7%) | (20.0%) | (24.9%) | (30.0%) | (22.3%) | (29.7%) |
| CTF1 | 3.8% | 24.4% | 3.6% | 16.3% | 5.1% | 12.3% | 21.8% | 20.7% |
|  | (8.5%) | (30.7%) | (5.6%) | (21.9%) | (8.1%) | (19.0%) | (33.3%) | (33.2%) |

**IL10-like**

| IL10 | IL19 | IL20 | IL22 | IL24 |
| --- | --- | --- | --- | --- |
| IL19 | 17.0% |  |  |  |
|  | (37.5%) |  |  |  |
| IL20 | 25.7% | 38.5% |  |  |
|  | (45.0%) | (57.7%) |  |  |
| IL22 | 21.7% | 18.9% | 20.2% |  |
|  | (36.9%) | (35.4%) | (36.0%) |  |
| IL24 | 18.3% | 23.7% | 26.8% | 23.3% |
|  | (37.2%) | (41.2%) | (45.5%) | (36.1%) |
| IL26 | 25.0% | 21.8% | 19.6% | 20.5% | 20.7% |
|  | (47.8%) | (37.9%) | (35.7%) | (35.0%) | (37.6%) |

Evolutionary divergence and functions of the human interleukin (IL) gene family

UPDATE ON GENE COMPLETIONS AND ANNOTATIONS

Table 4. Concluded

| IL28-like | IL28A | IL28B |
|-----------|-------|-------|
| IL28B     | 94.0% |       |
|           | (95.5%) |       |
| IL29      | 66.5% | 68.8% |
|           | (73.7%) | (75.1%) |

| IL17-like | IL17A | IL17B | IL17C | IL17D | IL25 |
|-----------|-------|-------|-------|-------|------|
| IL17B     | 25.9% |       |       |       |      |
|           | (37.6%) |       |       |       |      |
| IL17C     | 26.4% | 24.9% |       |       |      |
|           | (37.0%) | (38.3%) |       |       |      |
| IL17D     | 20.3% | 26.1% | 22.3% |       |      |
|           | (24.5%) | (35.2%) | (27.5%) |       |      |
| IL25      | 25.0% | 18.3% | 26.3% | 21.2% |      |
|           | (40.0%) | (27.2%) | (35.7%) | (30.3%) |      |
| IL17F     | 45.7% | 25.0% | 25.5% | 20.6% | 19.8% |
|           | (59.0%) | (40.2%) | (37.5%) | (28.8%) | (31.8%) |

results in the recruitment of Janus kinases 1 (JAK1) indicating that it is likely also to be the product of a
and 3 (JAK3), which phosphorylate a ligand- distant gene duplication event. IL4 and IL9 are
specific subset of signal transducers and activators of found within a gene cluster at Chr5q31, which also
transcription (STAT) proteins. TSLP signalling contains many related IL4-like cytokine genes (IL3,
also induces STAT-dependent transcriptional acti- IL5, IL13 and CSF2). TSLP also is located in close
vation but does not appear to involve a JAK.
Common γc cytokines are immunoregulatory mol- proximity to the 5q31 gene cluster at 5q22. The
ecules with both specific and overlapping functions. degree of amino acid homology observed between
Their production influences the proliferation, differ- these proteins is not extensive, ranging from 6–22
entiation and survival of innate and adaptive immune per cent identity (Table 4). The clustering dendro-
cells, including T cells, B cells, natural killer (NK) gram reveals the γc cytokines to be spread across the
cells, natural killer T (NKT) cells and dendritic cells entire interleukin/cytokine tree (Figure 1).
(DCs). IL7 is the predominant γc cytokine associated
with regulating immature and mature T lymphocyte
homeostasis. IL9 is known to activate B cells, eosino- IL4-like cytokines
phils, mast cells and epithelial cells. The IL2, IL15
and IL21 proteins have specific functions during T The IL4-like cytokines include IL3, IL4, IL5, IL13
cell differentiation and homeostasis. Many of the γc cytokine genes are located in and CSF2 (also known as granulocyte-macrophage
close genomic proximity with other class I helical CSF [GM-CSF]) (Table 2). These immunoregulatory
cytokine genes. IL2 and IL21 are linked on Chr molecules are generally <160 amino acids in
4q26 and believed to be the result of gene duplication. length and, like the γc ILs, are classified as short-
IL15 is located relatively close at Chr4q31, chain class I cytokines. They are structurally very
similar to the γc ILs; however, only IL4 utilises a
receptor complex containing the γc chain
co-receptor subunit. Although IL4 signals through
© HENRY STEWART PUBLICATIONS 1479–7364. HUMAN GENOMICS. VOL 5. NO 1. 30–55 OCTOBER 2010 43

UPDATE ON GENE COMPLETIONS AND ANNOTATIONS

Brocker et al.

IL8  
IL1F8  
IL1A  
IL1B  
IL1F5  
IL1RN  
IL1F10  
IL1F6  
IL1F9  
IL1F7  
IL18  
IL33  
IL12A  
IL15  
IL7  
TSLP  
IL21  
IL9  
IL4  
IL13  
IL17A  
IL17F  
IL17C  
IL17B  
IL17D  
IL25  
IL2  
IL10  
IL26  
IL19  
IL20  
IL24  
IL22  
IL6  
IL23A  
IL11  
IL27A  
CNTF  
CSF3  
CTF1  
CLCF1  
LIF  
IL3  
IL5  
IL28A  
IL28B  
IL29  
IL31  
OSM  
IL16  
CSF1  
TXLNA  
IL32  
IL34  
CSF2  

Figure 1. Phylogram of human ILs and closely related proteins. The phylogenetic tree was created using ClustalW 2 and edited using Treellustrator. IL1-like (red), γ-chain utilising (light blue), IL4-like (dark blue), IL6/12-like (purple), IL10-like (light green), IL28-like (dark green) and IL17-like (brown), as well as non-classified cytokines (black) are included.

γc, the high degree of homology with other IL4-like cytokines distinguishes it from the γc cytokines described above. All members in this group bind to heterodimeric receptor complexes.76 IL4 and IL13 share signalling mechanisms through a receptor complex containing the IL4Rα chain.77 Ligand specificity is achieved through the use of either γc or IL13Rα1 for IL4 and IL13, respectively.

Ligand binding at the receptor activates the tyrosine kinase JAK1, which leads to subsequent phosphorylation of STAT1 or STAT6. The activated STATs then translocate to the nucleus, where they regulate transcription. IL3, IL5 and CSF2 modulate transcription in a similar fashion; however, they utilise the common β-chain (βc) and either IL3Ra, IL5Ra or CSF2Ra, respectively. Subsequently, JAK2 and STAT5 mediate downstream transcriptional activation.

IL4 is associated with T helper type II (Th2) cell differentiation and plays an important role in allergic inflammation. Along with IL13, IL4 regulates immunoglobulin class switching in B cells and mast cell recruitment. IL3, IL4, IL5 and CSF2 facilitate granulocyte maturation and therefore also play an important role in regulating allergic response.

The five genes in this group are closely linked at Chr5q31, most likely originating from a series of gene duplications. The genomic organisation of these genes is also conserved within mice on Chr 11. Sequence homology between the proteins ranges from 20–25 per cent identity and 30–40 per cent similarity (Table 4). In spite of a relatively high degree of homology, the IL4-like cytokines are dispersed throughout the IL/cytokine dendrogram (Figure 1).

CSF3) or as heterodimers (IL12, IL23, IL27A and IL35). Most of the monomeric cytokines, except IL31, signal through the common co-receptor subunit IL6 signal transducer (IL6ST), previously known as glycoprotein 130 (gp130). IL31 uses the IL31RA co-receptor subunit, which is a close relative of IL6ST. IL6 initially binds to a specific high-affinity receptor, IL6RA. After activation, the IL6ST receptor chain is recruited to form the active signalling complex. IL11 and OSM also use unique receptor complexes composed of IL6ST and either IL11RA or the OSM receptor (OSMR), respectively. IL6 and IL11 are the only members of this group known to interact with IL6ST homodimers, where two IL-bound receptors (IL6RA or IL11RA) complex with an IL6ST dimer to create a hexameric complex that initiates subsequent signalling cascades. LIF and CTF1 signal through a receptor complex containing the LIF receptor (LIFR) and IL6ST. CNTF and CLCF1 bind a heterotrimeric receptor complex comprising LIFR and IL6ST, as well as the CNTF receptor (CNTFR) chain. IL31 signal transduction proceeds through OSMR and IL31RA.

The heterodimeric cytokines, composed of disulphide-linked α- and β-chains, each signal through a unique combination of dimerised receptors formed from five possible receptor subunits. The α-chain is structurally homologous to the long-chain class I cytokines (mentioned above) and contains a ‘four-helix bundle’ structural motif. The β-chains include IL12B and Epstein-Barr virus-induced-3 (EBI3), which is also known as IL27B. These proteins are structurally homologous to soluble class I helical cytokine receptors and are distinct from other class I cytokines. Biologically active IL12, a heterodimer formed between IL12A and IL12B, signals through the IL12RB1 and IL12RB2 receptor dimer. Active IL23 dimer is composed of IL12B and IL23A subunits and binds to the IL23 receptor complex formed from the association of IL12RB1 and IL23R. IL27A and EBI3 (IL27B) dimerisation forms the active IL27-signalling molecule, which is predominantly derived from macrophages as well as DCs. IL27 signals through a receptor comprising IL27RA and

### IL6/IL2-like cytokines

The biological functions of cytokines derived from IL6/IL2-like genes are diverse and comprise haematopoietic cytokines with broad physiological and pathophysiological actions. These proteins play a role in inflammation, immune response and cell differentiation and survival. ILs within this group include IL6, IL11, IL12A, IL23A, IL27A and IL31 (Table 2). There are a number of other closely related cytokines, including CLCF1, CTF1, CNTF, CTF3, LIF and OSM. IL6/IL2-like genes are categorised as long-chain class I helical cytokines and encode proteins >160 amino acids in length.

The group can be further subdivided into cytokines that are biologically active as monomers (IL6, IL11, IL31, CLCF1, CNTF, CTF1, LIF, OSM and

IL6ST, which is also used by many of the monomeric cytokines mentioned above. The IL12, IL23 and IL27A proteins are primarily expressed from antigen-presenting cells (APCs), which include DCs, macrophages and monocytes. Another novel heterodimeric cytokine formed from the dimerisation of IL12A and EBI3 (IL27B) was recently identified and designated IL35. IL35 expression from T regulatory (Treg) cells has been found to have inhibitory effects on T cell proliferation. The IL35 receptor complex has yet to be determined; however, it is hypothesised that it may use a combination of receptor subunits employed by IL12, IL23 and IL27.

Monomeric and heterodimeric long-chain class I cytokine receptor activation initiate signal transduction through similar JAK/STAT-signalling pathways. This signalling pathway is evolutionarily conserved and used by both class I and class II helical cytokines. Ligand binding activates the receptor complex and promotes JAK recruitment. Activated JAKs subsequently phosphorylate STATs, including STAT3 and STAT4, which in turn translocate to the nucleus and transactivate a number of genes associated with differentiation, growth, survival and apoptosis.

The IL6/12-like genes translate a functionally diverse group of proteins involved in the immune response to a variety of factors, including microbial and host immune stimuli. IL6 plays a very important role during the early immune response to infection. It causes B lymphocytes to differentiate into mature, immunoglobulin-secreting plasma B cells. IL6 signalling also initiates T cell activation, growth and differentiation. IL11 increases the production of proteins needed during the acute phase response and induces the differentiation of lymphocytes. IL12, IL23, IL27 and IL35 are involved in Info production and helper T cell differentiation. IL35 has been shown to have anti-inflammatory effects by expanding the population of anti-inflammatory cytokine-secreting cells. The dysregulation and aberrant expression of IL6/12-like cytokines can result in severe inflammation and is the underlying cause of several human immunological disorders,

rheumatoid arthritis, inflammatory bowel disease and asthma.

Amino acid sequence homology between group members is rather low, ranging from 12-19 percent (Table 4). Most of the IL6/12-like proteins cluster into a single branch within the IL phylogram, with the exception of IL12A, IL31 and OSM (Figure 1).

### IL10-like cytokines

IL10, IL19, IL20, IL22, IL24 and IL26 comprise the IL10-like cytokines (Table 2). IL10 was originally described as an inhibitor of T helper type I (Th1) cell-related cytokine expression. Since its discovery, the other five proteins have been added to this group. These cytokines are related through genomic organisation, intron-exon structure and a common structural fold composed of stacked α-helices. The IL10-like group and closely related IL28-like group comprise the class II helical cytokines.

Despite structural similarities, the proteins display unique expression patterns and diverse functions. T lymphocytes, monocytes and B cells are the most common source of IL10 in humans. IL10 is biologically active as a homodimer and activates the IL10R1 and IL10R2 receptor complex. Subsequent downstream signalling is mediated through JAK/STAT pathways, analogous to the signalling mechanisms used by class I cytokines (ie γ-chain, IL4-like and IL6/12-like cytokines). IL10 suppresses the expression of other inflammatory molecules and is an extremely important anti-inflammatory and immunosuppressive cytokine in humans. IL19 is an immunoregulatory cytokine primarily produced by monocytes. IL19 signalling proceeds through the IL20R1 and IL20R2 dimeric receptor complex, and subsequent receptor activation results in the release of IL6, tumour necrosis factor (TNF) and a number of reactive oxygen species (ROS) from target cells. IL20 is known for its role in haematopoiesis and inflammation. It was originally identified in a human keratinocyte library; studies suggest that it may play a role in the pathophysiology of some skin

abnormalities, such as psoriasis and palmoplantar
pustulosis. IL20, as well as IL24, can signal
through both the IL20R1/IL20R2 and IL22R1/
20R2 heterodimers. IL22 is similar to other
IL10-like cytokines in binding a heterodimeric
receptor, albeit composed of IL22R1 and IL10R2
chains. Activated T cells and NKT cells are the
primary source of IL22. IL22 facilitates defence
against microbial pathogens by initiating the release
of antimicrobial peptides associated with host
defence, such as defensin-β2 and S100A7. It is
also associated with the induction of acute phase
response
proteins. Originally named
IL10-related T-cell-derived inducible factor
(ILTIF), studies have also indicated that IL22, similarly to IL10, may be biologically active as a homo-
dimer. By contrast with IL10 and IL22, the
IL19, IL20 and IL24 proteins are biologically active
as monomers. IL24, previously known as
melanoma differentiation-associated antigen 7
(MDA-7), was identified as a cytokine that was dif-
ferentially expressed in healthy and metastatic mela-
nocyes. IL24 is predominantly secreted from
melanocytes, Th2 cells and monocytes.
Downstream inflammatory effects of IL24 include
the release of IL6, IFNG and TNF expression occurs primarily in memory Th1 cells,
Th17 cells and monocytes. Akin to IL10, IL26
functions as a homodimer and activates a unique
receptor complex composed of IL20R1 and
IL10R2 chains. The exact biological functions
of IL26 have yet to be determined — mainly due
to the lack of a mouse orthologue; this limits
experimental approaches commonly used to
characterise other ILs, such as the generation of
knockout mice. Activation of either complex is
linked to downstream JAK/STAT pathways.
IL10, IL19, IL20 and IL24 form a closely linked
gene cluster on Chr1q32. All four genes are
found within a 195kb region. By contrast with the
other IL10-like genes, IL22 and IL26 are found in
a gene cluster with IFNG on Chr12q15. As
mentioned above, a mouse homologue for IL26 has
yet to be identified; however, the IL26 gene cluster
is conserved within other mammals, as well as non-
mammals, indicating that IL26 was lost from the

murine genome. Amino acid identities for
members in this group range from 17–39 per cent
(Table 4) and they form an isolated cluster within
the IL/cytokine dendrogram (Figure 1).

### IL28-like cytokines

IL28A, IL28B and IL29 were discovered in 2003
by two independent research groups. These
cytokines exhibit IFN type I properties in response
to viral or bacterial infection. As such, they have
also been referred to as type III IFNs; however,
genomic organisation, protein structure and receptor usage for these proteins has linked them most
closely to the IL10-like cytokines. They contain the common structural motif found in
IL10-related cytokines (stacked α-helices), making
them members of the class II helical cytokines.

IL28A, IL28B and IL29, like many cytokines,
utilise a dimeric cell-surface receptor complex. The
heterodimeric receptor belongs to the class II cytokine receptor family and comprises the IL10R1
chain, which is shared with IL10, IL22 and IL26, and
a unique chain referred to as IL28R1. IL28A,
IL28B and IL29 are secreted as monomeric proteins;
receptor activation utilises JAK/STAT pathways for
downstream signalling, analogous to other class I and
class II helical cytokines. It is thought that
most, if not all, nucleated cell types are capable of
IL28-like cytokine expression following viral infection. Expression is also observed after activation
with bacterial components such as lipopolysaccharide
(LPS). The proteins exhibit direct anti-viral
activity by inhibiting viral replication. They
also have direct anti-growth activity on many cell
types, including tumour cells. Recently, a great deal
of attention has been given to these proteins due to
their potential therapeutic benefits.

IL28A and IL28B form a tight gene cluster on
Chr19q13. Gene architectures are similar to the
IL10-like genes and contain either six (IL28A and
IL28B) or five (IL29) exons. Only the mouse
Il28a and Il28b genes translate functional gene products. Il29 contains an in-frame stop codon within
the first exon and is considered a pseudogene.
The amino acid identities for the IL28-like

cytokines are very high, ranging from 66–94 per
cent (Table 4). The high degree of homology
between the proteins is further supported by the
tight cluster formed on the IL/cytokine dendrogram (Figure 1).

### IL17-like cytokines

IL17A, IL17B, IL17C, IL17D, IL17F and IL25 are
members of the IL17-like cytokine group. IL17A was
originally identified within a murine hybridoma
cDNA library in 1993. Genomic sequencing led
to the identification of five IL17A homologues,
including IL17B, IL17C, IL17D, IL25 and IL17E.
IL17A and IL17F are the most well characterised of
the six IL17-like cytokines. IL17F was recently crystallised; the results indicate that the protein adopts a
cysteine-knot fold architecture, which is a structural
motif associated with neurotrophins. IL17A and
IL17F are biologically active as homodimers but are
also capable of forming IL17A-IL17F heterodimers. IL17A and IL17F homodimers, as well as
the IL17A-IL17F heterodimer, bind with varying
affinities to a heterodimeric receptor complex composed of IL17RA and IL17RC. IL25 also activates
a dimeric receptor, comprising IL17RA and IL17RB.
IL17B and IL17C have been found to bind to the
IL17RB and IL17RE receptors, respectively. A receptor has not yet been identified for IL17D.

The functions of IL17-like cytokines are not
completely understood. IL17A expression has been
found to originate from a new subset of helper T
cells, Th17 cells. These cells are potent inducers
of tissue inflammation through the release of
IL17A, IL17F, IL21 and IL22. IL25 expression has
not been detected in Th17 cells but secretion from
Th2 cells has been identified. It is hypothesised
that the Th17 response and IL17A release is triggered when a specific pathogen does not elicit a
strong response from either Th1 or Th2 cells. In
general, the specific functions of IL17-like cytokines are not well established. Nevertheless, IL17A
appears to play an important role in the immune
response to bacterial and fungal pathogens. IL17A
is also associated with many autoimmune disorders,
including rheumatoid arthritis, inflammatory bowel

disease, multiple sclerosis, as well as many types of
cancer. The IL17-like cytokines appear to form a novel
family of signalling molecules. Not only are the
proteins structurally unlike other immunomodulatory cytokines, but the corresponding IL17 receptor family contains five members, IL17RA-E,
which are also structurally distinct from those used
by other cytokines. Perhaps not surprisingly,
these receptors are coupled to signal transduction
pathways that differ from those used by class I and II
cytokines — that is, TNF receptor-associated factor
(TRAF) proteins, TRAF3 interacting protein 2
(TRAF3IP2) and nuclear factor-κB (NF-κB).
IL17-like cytokines share 18–45 per cent amino
acid identity and are dispersed throughout the
human genome (Table 4). IL17A and IL17F are the
most closely related homologues in this group,
sharing >45 per cent sequence identity and 59 per
cent sequence similarity. The high degree of
sequence homology can be explained by the two
genes being closely linked at Chr6p12, again most
likely the product of an evolutionarily recent gene
duplication. IL17B, IL17C, IL17D and IL25 are
located on Chr5q33, 16q24, 13q11 and 14q11,
respectively. Pairwise comparisons between each of
the other members indicate that they share
between 18–26 per cent sequence identity and
24–40 per cent sequence similarity. The close
relationship between these proteins is evident in
the tight cluster they form within the IL/cytokine
dendrogram (Figure 1).

### Additional ‘non-classified’ interleukins

There are a number of ILs that cannot be definitively classified into any of the groups described
above (Table 2). The inability to classify these molecules is generally associated with their unique
structural characteristics or lack of structural information; however, their genes encode proteins with
verified immunomodulatory actions.
IL8 contains a CXC structural motif, which is a
defining characteristic of the CXC chemokine
family. This family is characterised by the presence

of three or four highly conserved cysteine residues found within the N-terminus. The CXC chemokines contain a variable residue between the first two conserved cysteines. ${ }^{1}$ IL8 has been shown to participate in leukocyte recruitment during inflammation — supporting its role as a chemokine. ${ }^{74}$ IL8 is released from many cell types, including T cells, monocytes and endothelial cells. ${ }^{69-71}$ Its expression is induced after exposure to a variety of inflammatory stimuli — including bacteria, oxidative stress, LPS, TNF and IL1B. ${ }^{155-158}$ As expected, IL8 shares little sequence homology with known ILs; however, it shares a high degree of sequence identity with other CXC chemokines. ${ }^{159}$ Chemokine (CXC motif) ligands-1 (CXCL1), -2 (CXCL2) and -3 (CXCL3), and pro-platelet basic protein (PPBP), are all well-characterised chemokines, exhibiting 36 per cent, 36 per cent, 34 per cent and 33 per cent sequence identity, respectively.

Taxilin alpha (TXLNA), also known as IL14, was originally identified as a factor produced by human B cell lymphoma cells that caused enhanced proliferation of activated B lymphocytes. ${ }^{160}$ The factor was initially called high-molecular-weight B cell growth factor. The cDNA was cloned and expression produced a 54-kilodalton (kDa) protein that was re-named IL14. Produced by B cells, T cells and DCs, IL14 enhances B cell proliferation, increases the subpopulation of memory B cells and prevents the secretion of immunoglobulins. ${ }^{161}$ Interestingly, IL14 was also identified as a novel syntaxin-binding protein involved in vesicle transport and was re-named taxilin. ${ }^{162}$ Database searches have revealed two closely related homologues, resulting in taxilin being re-named as taxilin-$\alpha$ (TXLNA), with its homologues named taxilin-$\beta$ (TXLNB) and taxilin-$\gamma$ (TXLNG). ${ }^{163}$ None of the taxilin isoforms appears to be structurally related to known ILs; their exact functions remain unknown. IL14 has been implicated in the pathophysiology of Sjögren's disease, an autoimmune disorder affecting exocrine glands. ${ }^{164}$ TXLNA, TXLNB and TXLNG are located on Chr 1p34, 6q24 and Xp22, respectively. ${ }^{163}$ Sequence analysis reveals that TXLNA shares little homology (<15 per cent) with any of the cytokines listed in Table 1.

IL16 was originally described as a T helper cell chemoattractant. The compound is also described as a chemotactic cytokine but not a chemokine because it lacks characteristic structural motifs. ${ }^{165}$ IL16 is expressed as a large precursor protein that is cleaved by caspase-3 at a serine residue within the C-terminus to produce a larger intracellular and a shorter 121-amino acid secretory protein. ${ }^{166}$ The shorter, secreted peptide assembles into homotetramers, which are believed to be the biologically active unit. ${ }^{167}$ The larger cleavage product remains within the cell, and its exact function has yet to be determined. Some evidence suggests that it is able to translocate into the nucleus and arrest the cell cycle. ${ }^{168}$ Studies have shown that many cell types are capable of IL16 secretion, including T cells, eosinophils, DCs, mast cells and neuronal cells. ${ }^{168}$ Secreted IL16 binds to the CD4 receptor to activate a downstream signalling cascade that involves STAT6 activation. ${ }^{169}$ IL16 activates many cell types, including T cells, monocytes, macrophages, eosinophils and DCs. ${ }^{170,171}$ Downstream effects include the release of inflammatory cytokines including IL1B, IL6, IL15 and TNF. ${ }^{172}$ IL16 is structurally distinct from other cytokines in that it contains multiple PDZ domains, including one that lies within the mature secretory peptide. ${ }^{173}$ PDZ domains are a common structural motif associated with protein-protein interactions. ${ }^{174}$

IL32 was originally identified in 1992 as a cytokine secreted from IL2-activated NK and T cells. ${ }^{175}$ The protein does not exhibit significant sequence similarity with other cytokines and, to date, there is little structural information in the literature — making its classification very difficult. Further studies have shown that IL32 is also released from monocytes and epithelial cells. ${ }^{176,177}$ It stimulates the release of pro-inflammatory cytokines, including IL1B, IL6, IL8, TNF and macrophage inflammatory protein 2 (MIP2). ${ }^{176}$ The signalling pathways for IL32 have yet to be fully characterised; however, studies have shown that they involve the NF-$\kappa$B and p38 mitogen-activated protein kinase pathways, which are commonly associated with inflammation. ${ }^{178}$ A number of diseases associated with IL32 expression, including inflammatory bowel

disease, myelodysplastic syndrome and HIV infection. ${}^{179-181}$ IL32 lacks significant amino acid identity ($<15$ per cent) with the other cytokines listed in Table 1. Furthermore, BLAST analyses failed to identify any significantly homologous proteins.

IL34 was identified in 2008 by Lin *et al.* ${}^{182}$ It binds to the homodimeric CSF1 receptor (CSFR1), which also binds CSF1. ${}^{182}$ CSF1 is a cytokine that controls the division, differentiation and function of monocytes, macrophages and osteoblasts. ${}^{183}$ Similar to CSF1, human IL34 has been found to function as a homodimer. ${}^{182}$ Furthermore, modelling of both IL34 and CSF1 identified a stacked ‘four-helix bundle’ structure. ${}^{184}$ The CSF1 monomer is structurally very similar to CSF2, a short-chain class I helical cytokine. ${}^{185}$ There is a high likelihood that both CSF1 and IL34 are evolutionarily related to the class I helical cytokines. BLAST analysis of the IL34 peptide sequence failed to identify any significantly homologous proteins within the human genome. Pairwise alignments revealed that IL34 shares 19 per cent sequence identity with OSM and CNTF, both of which are long-chain class I helical cytokines. Sequence homology between other ILs and cytokines was not significant (ie $<15$ per cent).

# Conclusions

To date, the human IL gene family contains 55 known IL and IL-related genes. The majority of ILs can be classified into seven distinct groups and subgroups using characteristic structural features. These groups include genes encoding the IL1-like cytokines, the class I helical cytokines (IL4-like, γ-chain and IL6/12-like), the class II helical cytokines (IL10-like and IL28-like) and the IL17-like cytokines. Because evolutionary divergence occurs so rapidly in these genes, it is impossible in many cases to assign a member to a specific ‘gene family’ or ‘gene subfamily’; therefore, structural analysis has become the principle approach for classifying. A number of ‘interleukin’ genes currently lack the structural information required for classification. It is therefore likely that these groups will expand as crystallisation data and structural analyses become available.

# Acknowledgments

We would like to thank our colleagues for valuable discussion and critical review of this manuscript. This work was supported, in part, by the following NIH grants: R01 EY17963 (V.V.), R21 AA017754 (V.V.), F31 AA018248 (C.B.) and P30 ES06096 (D.W.N.). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.

# References

1. Commins, S.P., Borish, L. and Steinke, J.W. (2010), ‘Immunologic messenger molecules: Cytokines, interferons, and chemokines’, *J. Allergy Clin. Immunol.* Vol. 125, pp. S53–S72.
2. Aarden, L.A., Brunner, T.K., Cerottini, J.C., Dayer, J.M. *et al.* (1979), ‘Revised nomenclature for antigen-non-specific T cell proliferation and helper factors’, *J. Immunol.* Vol. 123, pp. 2928–2929.
3. Liles, W.C. and Van Voorhis, W.C. (1995), ‘Review: Nomenclature and biologic significance of cytokines involved in inflammation and the host immune response’, *J. Infect. Dis.* Vol. 172, pp. 1573–1580.
4. Anon (1991), ‘Nomenclature for secreted regulatory proteins of the immune system (interleukins). WHO-IUIS Nomenclature Subcommittee on Interleukin Designation’, *Bull. World Health Organ.* Vol. 69, pp. 483–486.
5. Anon (1994), ‘Nomenclature for secreted regulatory proteins of the immune system (interleukins): Update. IUIS/WHO Nomenclature Committee’, *Bull. World Health Organ.* Vol. 72, pp. 811–812.
6. Anon (1996), ‘Interleukin 16 (IL-16). IUIS/WHO Standing Committee on Interleukin Designation’, *Bull. World Health Organ.* Vol. 74, pp. 451–452.
7. Anon (1997), ‘Nomenclature for secreted regulatory proteins of the immune system (interleukins): Update. IUIS/WHO Standing Committee on Interleukin Designation’, *Bull. World Health Organ.* Vol. 75, p. 175.
8. Sims, J.E., Nicklin, M.J., Bazan, J.F., Barton, J.L. *et al.* (2001), ‘A new nomenclature for IL-1-family genes’, *Trends Immunol.* Vol. 22, pp. 536–537.
9. Boulay, J.L. and Paul, W.E. (1992), ‘The interleukin-4 family of lymphokines’, *Curr. Opin. Immunol.* Vol. 4, pp. 294–298.
10. Veerapandian, B., Gilliland, G.L., Raag, R., Svensson, A.L. *et al.* (1992), ‘Functional implications of interleukin-1 beta based on the three-dimensional structure’, *Proteins* Vol. 12, pp. 10–23.
11. Priestle, J.P., Schar, H.P. and Grutter, M.G. (1989), ‘Crystallographic refinement of interleukin 1 beta at 2.0 Å resolution’, *Proc. Natl. Acad. Sci. USA* Vol. 86, pp. 9667–9671.
12. Vigers, G.P., Caffes, P., Evans, R.J., Thompson, R.C. *et al.* (1994), ‘X-ray structure of interleukin-1 receptor antagonist at 2.0-Å resolution’, *J. Biol. Chem.* Vol. 269, pp. 12874–12879.
13. Graves, B.J., Hatada, M.H., Hendrickson, W.A., Miller, J.K. *et al.* (1990), ‘Structure of interleukin 1 alpha at 2.7-Å resolution’, *Biochemistry* Vol. 29, pp. 2679–2684.
14. Boulay, J.L., O’Shea, J.J. and Paul, W.E. (2003), ‘Molecular phylogeny within type I cytokines and their cognate receptors’, *Immunity* Vol. 19, pp. 159–163.
15. Ealick, S.E., Cook, W.J., Vijay-Kumar, S., Carson, M. *et al.* (1991), ‘Three-dimensional structure of recombinant human interferon-gamma’, *Science* Vol. 252, pp. 698–702.
16. Zdanov, A., Schalk-Hihi, C., Gustchina, A., Tsang, M. *et al.* (1995), ‘Crystal structure of interleukin-10 reveals the functional dimer with an unexpected topological similarity to interferon gamma’, *Structure* Vol. 3, pp. 591–601.
17. Kotenko, S.V. (2002), ‘The family of IL-10-related cytokines and their receptors: Related, but to what extent?’, *Cytokine Growth Factor Rev.* Vol. 13, pp. 223–240.

18. Sheppard, P., Kindsvogel, W., Xu, W., Henderson, K. *et al.* (2003), ‘IL-28, IL-29 and their class II cytokine receptor IL-28R’, *Nat. Immunol.* Vol. 4, pp. 63–68.
19. Witte, K., Witte, E., Sabat, R. and Wolk, K. (2010), ‘IL-28A, IL-28B, and IL-29: Promising cytokines with type I interferon-like properties’, *Cytokine Growth Factor Rev.* Vol. 21, pp. 237–251.
20. Hymowitz, S.G., Filvaroff, E.H., Yin, J.P., Lee, J. *et al.* (2001), ‘IL-17s adopt a cystine knot fold: structure and activity of a novel cytokine, IL-17F, and implications for receptor binding’, *EMBO J.* Vol. 20, pp. 5332–5341.
21. McDonald, N.Q. and Hendrickson, W.A. (1993), ‘A structural superfamily of growth factors containing a cystine knot motif’, *Cell* Vol. 73, pp. 421–424.
22. Hurst, L.D. and Smith, N.G. (1999), ‘Do essential genes evolve slowly?’, *Curr. Biol.* Vol. 9, pp. 747–750.
23. Scapigliati, G., Buonocore, F. and Mazzini, M. (2006), ‘Biological activity of cytokines: An evolutionary perspective’, *Curr. Pharm. Des.* Vol. 12, pp. 3071–3081.
24. Luporini, P., Vallesi, A., Miceli, C. and Bradshaw, R.A. (1994), ‘Ciliate pheromones as early growth factors and cytokines’, *Ann. N. Y. Acad. Sci.* Vol. 712, pp. 195–205.
25. Vallesi, A., Giuli, G., Ghiara, P., Scapigliati, G. *et al.* (1998), ‘Structure-function relationships of pheromones of the ciliate *Euplotes raikovi* with mammalian growth factors: Cross-reactivity between Er-1 and interleukin-2 systems’, *Exp. Cell Res.* Vol. 241, pp. 253–259.
26. Bingulac-Popovic, J., Figueroa, F., Sato, A., Talbot, W.S. *et al.* (1997), ‘Mapping of mhc class I and class II regions to different linkage groups in the zebrafish, *Danio rerio*’, *Immunogenetics* Vol. 46, pp. 129–134.
27. Nicklin, M.J., Weith, A. and Duff, G.W. (1994), ‘A physical map of the region encompassing the human interleukin-1 alpha, interleukin-1 beta, and interleukin-1 receptor antagonist genes’, *Genomics* Vol. 19, pp. 382–384.
28. Dunn, E., Sims, J.E., Nicklin, M.J. and O’Neill, L.A. (2001), ‘Annotating genes with potential roles in the immune system: Six new members of the IL-1 family’, *Trends Immunol.* Vol. 22, pp. 533–536.
29. Takahashi, M., Yoshida, M.C., Satoh, H., Hilgers, J. *et al.* (1989), ‘Chromosomal mapping of the mouse IL-4 and human IL-5 genes’, *Genomics* Vol. 4, pp. 47–52.
30. Yang, Y.C., Kovacic, S., Kriz, R., Wolf, S. *et al.* (1988), ‘The human genes for GM-CSF and IL 3 are closely linked in tandem on chromosome 5’, *Blood* Vol. 71, pp. 958–961.
31. van Leeuwen, B.H., Martinson, M.E., Webb, G.C. and Young, I.G. (1989), ‘Molecular organization of the cytokine gene cluster, involving the human IL-3, IL-4, IL-5, and GM-CSF genes, on human chromosome 5’, *Blood* Vol. 73, pp. 1142–1148.
32. Sanderson, C.J., Campbell, H.D. and Young, I.G. (1988), ‘Molecular and cellular biology of eosinophil differentiation factor (interleukin-5) and its effects on human and mouse B cells’, *Immunol. Rev.* Vol. 102, pp. 29–50.
33. Lee, J.S., Campbell, H.D., Kozak, C.A. and Young, I.G. (1989), ‘The IL-4 and IL-5 genes are closely linked and are part of a cytokine gene cluster on mouse chromosome 11’, *Somat. Cell Mol. Genet.* Vol. 15, pp. 143–152.
34. Ryan, A.W., Thornton, J.M., Brophy, K., Daly, J.S. *et al.* (2005), ‘Chromosome 5q candidate genes in coeliac disease: Genetic variation at IL4, IL5, IL9, IL13, IL17B and NR3C1’, *Tissue Antigens* Vol. 65, pp. 150–155.
35. Webb, G.C., Lee, J.S., Campbell, H.D. and Young, I.G. (1989), ‘The genes for interleukins 3 and 5 map to the same locus on mouse chromosome 11’, *Cytogenet. Cell Genet.* Vol. 50, pp. 107–110.
36. Giovannini, M., Djabali, M., McElligott, D., Selleri, L. *et al.* (1993), ‘Tandem linkage of genes coding for leukemia inhibitory factor (LIF) and oncostatin M (OSM) on human chromosome 22’, *Cytogenet. Cell Genet.* Vol. 64, pp. 240–244.
37. Jeffery, E., Price, V. and Gearing, D.P. (1993), ‘Close proximity of the genes for leukemia inhibitory factor and oncostatin M’, *Cytokine* Vol. 5, pp. 107–111.
38. Oral, H.B., Kotenko, S.V., Yilmaz, M., Mani, O. *et al.* (2006), ‘Regulation of T cells and cytokines by the interleukin-10 (IL-10)-family cytokines IL-19, IL-20, IL-22, IL-24 and IL-26’, *Eur. J. Immunol.* Vol. 36, pp. 380–388.
39. Uze, G. and Monneron, D. (2007), ‘IL-28 and IL-29: Newcomers to the interferon family’, *Biochimie* Vol. 89, pp. 729–734.
40. Starnes, T., Robertson, M.J., Sledge, G., Kelich, S. *et al.* (2001), ‘Cutting edge: IL-17F, a novel cytokine selectively expressed in activated T cells and monocytes, regulates angiogenesis and endothelial cell cytokine production’, *J. Immunol.* Vol. 167, pp. 4137–4140.
41. Moseley, T.A., Haudenschild, D.P., Rose, L. and Reddi, A.H. (2003), ‘Interleukin-17 family and IL-17 receptors’, *Cytokine Growth Factor Rev.* Vol. 14, pp. 155–174.
42. Fryxell, K.J. (1996), ‘The coevolution of gene family trees’, *Trends Genet.* Vol. 12, pp. 364–369.
43. Ahuja, S.K., Ozcelik, T., Milatovitch, A., Francke, U. *et al.* (1992), ‘Molecular evolution of the human interleukin-8 receptor gene cluster’, *Nat. Genet.* Vol. 2, pp. 31–36.
44. Kohu, K., Yamabe, E., Matsuzawa, A., Onda, D. *et al.* (2008), ‘Comparison of 30 immunity-related genes from the common marmoset with orthologues from human and mouse’, *Tohoku J. Exp. Med.* Vol. 215, pp. 167–180.
45. Kaiser, P., Rothwell, L., Avery, S. and Balu, S. (2004), ‘Evolution of the interleukins’, *Dev. Comp. Immunol.* Vol. 28, pp. 375–394.
46. Liongue, C. and Ward, A.C. (2007), ‘Evolution of class I cytokine receptors’, *BMC Evol. Biol.* Vol. 7, p. 120.
47. Nicklin, M.J., Barton, J.L., Nguyen, M., FitzGerald, M.G. *et al.* (2002), ‘A sequence-based map of the nine genes of the human interleukin-1 cluster’, *Genomics* Vol. 79, pp. 718–725.
48. Arend, W.P., Palmer, G. and Gabay, C. (2008), ‘IL-1, IL-18, and IL-33 families of cytokines’, *Immunol. Rev.* Vol. 223, pp. 20–38.
49. Barksby, H.E., Lea, S.R., Preshaw, P.M. and Taylor, J.J. (2007), ‘The expanding family of interleukin-1 cytokines and their role in destructive inflammatory disorders’, *Clin. Exp. Immunol.* Vol. 149, pp. 217–225.
50. Gabay, C., Lamacchia, C. and Palmer, G. (2010), ‘IL-1 pathways in inflammation and human diseases’, *Nat. Rev. Rheumatol.* Vol. 6, pp. 232–241.
51. Weber, A., Wasiliew, P. and Kracht, M. (2010), ‘Interleukin-1 (IL-1) pathway’, *Sci. Signal.* Vol. 3, p. cm1.
52. Allantaz, F., Chaussabel, D., Stichweh, D., Bennett, L. *et al.* (2007), ‘Blood leukocyte microarrays to diagnose systemic onset juvenile idiopathic arthritis and follow the response to IL-1 blockade’, *J. Exp. Med.* Vol. 204, pp. 2131–2144.
53. Vincenti, M.P. and Brinckerhoff, C.E. (2001), ‘Early response genes induced in chondrocytes stimulated with the inflammatory cytokine interleukin-1beta’, *Arthritis Res.* Vol. 3, pp. 381–388.
54. Dinarello, C.A. (2002), ‘The IL-1 family and inflammatory diseases’, *Clin. Exp. Rheumatol.* Vol. 20, pp. S1–13.
55. Wolk, K., Witte, E., Hoffmann, U., Doecke, W.D. *et al.* (2007), ‘IL-22 induces lipopolysaccharide-binding protein in hepatocytes: A potential systemic role of IL-22 in Crohn’s disease’, *J. Immunol.* Vol. 178, pp. 5973–5981.
56. Hsu, Y.H., Li, H.H., Hsieh, M.Y., Liu, M.F. *et al.* (2006), ‘Function of interleukin-20 as a proinflammatory molecule in rheumatoid and experimental arthritis’, *Arthritis Rheum.* Vol. 54, pp. 2722–2733.
57. Ikeuchi, H., Kuroiwa, T., Hiramatsu, N., Kaneko, Y. *et al.* (2005), ‘Expression of interleukin-22 in rheumatoid arthritis: Potential role as a proinflammatory cytokine’, *Arthritis Rheum.* Vol. 52, pp. 1037–1046.
58. Otkjaer, K., Kragballe, K., Funding, A.T., Clausen, J.T. *et al.* (2005), ‘The dynamics of gene expression of interleukin-19 and interleukin-20 and their receptors in psoriasis’, *Br. J. Dermatol.* Vol. 153, pp. 911–918.
59. Salvi, G.E. and Lang, N.P. (2005), ‘Host response modulation in the management of periodontal diseases’, *J. Clin. Periodontol.* Vol. 32 (Suppl. 6), pp. 108–129.
60. Burger, D., Dayer, J.M., Palmer, G. and Gabay, C. (2006), ‘Is IL-1 a good therapeutic target in the treatment of arthritis?’, *Best Pract. Res. Clin. Rheumatol.* Vol. 20, pp. 879–896.

61. Thornberry, N.A., Bull, H.G., Calaycay, J.R., Chapman, K.T. *et al.* (1992), ‘A novel heterodimeric cysteine protease is required for interleukin-1 beta processing in monocytes’, *Nature* Vol. 356, pp. 768–774.
62. Hayakawa, M., Hayakawa, H., Matsuyama, Y., Tamemoto, H. *et al.* (2009), ‘Mature interleukin-33 is produced by calpain-mediated cleavage in vivo’, *Biochem. Biophys. Res. Commun.* Vol. 387, pp. 218–222.
63. Talabot-Ayer, D., Lamacchia, C., Gabay, C. and Palmer, G. (2009), ‘Interleukin-33 is biologically active independently of caspase-1 cleavage’, *J. Biol. Chem.* Vol. 284, pp. 19420–19426.
64. Thompson, R.C., Dripps, D.J. and Eisenberg, S.P. (1991), ‘IL-1ra: Properties and uses of an interleukin-1 receptor antagonist’, *Agents Actions Suppl.* Vol. 35, pp. 41–49.
65. Kakkar, R. and Lee, R.T. (2008), ‘The IL-33/ST2 pathway: Therapeutic target and novel biomarker’, *Nat. Rev. Drug Discov.* Vol. 7, pp. 827–840.
66. Dale, M. and Nicklin, M.J. (1999), ‘Interleukin-1 receptor cluster: Gene organization of IL1R2, IL1R1, IL1RL2 (IL-1Rrp2), IL1RL1 (T1/ST2), and IL18R1 (IL-1Rrp) on human chromosome 2q’, *Genomics* Vol. 57, pp. 177–179.
67. Taylor, S.L., Renshaw, B.R., Garka, K.E., Smith, D.E. *et al.* (2002), ‘Genomic organization of the interleukin-1 locus’, *Genomics* Vol. 79, pp. 726–733.
68. O’Neill, L.A. (1995), ‘Interleukin-1 signal transduction’, *Int. J. Clin. Lab. Res.* Vol. 25, pp. 169–177.
69. Steinkasserer, A., Spurr, N.K., Cox, S., Jeggo, P. *et al.* (1992), ‘The human IL-1 receptor antagonist gene (IL1RN) maps to chromosome 2q14-q21, in the region of the IL-1 alpha and IL-1 beta loci’, *Genomics* Vol. 13, pp. 654–657.
70. Gracie, J.A., Robertson, S.E. and McInnes, I.B. (2003), ‘Interleukin-18’, *J. Leukoc. Biol.* Vol. 73, pp. 213–224.
71. Park, L.S., Martin, U., Garka, K., Gliniak, B. *et al.* (2000), ‘Cloning of the murine thymic stromal lymphopoietin (TSLP) receptor: Formation of a functional heteromeric complex requires interleukin 7 receptor’, *J. Exp. Med.* Vol. 192, pp. 659–670.
72. Rochman, Y., Spolski, R. and Leonard, W.J. (2009), ‘New insights into the regulation of T cells by gamma(c) family cytokines’, *Nat. Rev. Immunol.* Vol. 9, pp. 480–490.
73. Liu, Y.J., Soumelis, V., Watanabe, N., Ito, T. *et al.* (2007), ‘TSLP: An epithelial cell cytokine that regulates T cell differentiation by conditioning dendritic cell maturation’, *Annu. Rev. Immunol.* Vol. 25, pp. 193–219.
74. Chang, H.C., Sehra, S., Goswami, R., Yao, W. *et al.* (2010), ‘The transcription factor PU.1 is required for the development of IL-9-producing T cells and allergic inflammation’, *Nat. Immunol.* Vol. 11, pp. 527–534.
75. Alves, N.L., Arosa, F.A. and van Lier, R.A. (2007), ‘Common gamma chain cytokines: Dissidence in the details’, *Immunol. Lett.* Vol. 108, pp. 113–120.
76. Nelms, K., Keegan, A.D., Zamorano, J., Ryan, J.J. *et al.* (1999), ‘The IL-4 receptor: Signaling mechanisms and biologic functions’, *Annu. Rev. Immunol.* Vol. 17, pp. 701–738.
77. Obiri, N.I., Leland, P., Murata, T., Debinski, W. *et al.* (1997), ‘The IL-13 receptor structure differs on various cell types and may share more than one component with IL-4 receptor’, *J. Immunol.* Vol. 158, pp. 756–764.
78. Chiba, Y., Todoroki, M. and Misawa, M. (2009), ‘Activation of signal transducer and activator of transcription factor 1 by interleukins-13 and -4 in cultured human bronchial smooth muscle cells’, *J. Smooth Muscle Res.* Vol. 45, pp. 279–288.
79. Kitamura, T., Sato, N., Arai, K. and Miyajima, A. (1991), ‘Expression cloning of the human IL-3 receptor cDNA reveals a shared beta subunit for the human IL-3 and GM-CSF receptors’, *Cell* Vol. 66, pp. 1165–1174.
80. Takatsu, K., Takaki, S. and Hitoshi, Y. (1994), ‘Interleukin-5 and its receptor system: Implications in the immune system and inflammation’, *Adv. Immunol.* Vol. 57, pp. 145–190.

81. Gearing, D.P., King, J.A., Gough, N.M. and Nicola, N.A. (1989), ‘Expression cloning of a receptor for human granulocyte-macrophage colony-stimulating factor’, *EMBO J.* Vol. 8, pp. 3667–3676.
82. Yang, L.P., Byun, D.G., Demeure, C.E., Vezzio, N. *et al.* (1995), ‘Default development of cloned human naive CD4 T cells into interleukin-4- and interleukin-5-producing effector cells’, *Eur. J. Immunol.* Vol. 25, pp. 3517–3520.
83. Pacor, S., Gagliardi, R., Spessotto, P., Zabucchi, G. *et al.* (1999), ‘Paracrine effects of IL-4 transfection on TS/A adenocarcinoma cells mediate reduced in vivo growth’, *Pathol. Oncol. Res.* Vol. 5, pp. 110–116.
84. Holgate, S.T. and Polosa, R. (2008), ‘Treatment strategies for allergy and asthma’, *Nat. Rev. Immunol.* Vol. 8, pp. 218–230.
85. Jazayeri, J.A., Carroll, G.J. and Vernalis, A.B. (2010), ‘Interleukin-6 subfamily cytokines and rheumatoid arthritis: Role of antagonists’, *Int. Immunopharmacol.* Vol. 10, pp. 1–8.
86. Heinrich, P.C., Behrmann, I., Haan, S., Hermanns, H.M. *et al.* (2003), ‘Principles of interleukin (IL)-6-type cytokine signalling and its regulation’, *Biochem. J.* Vol. 374, pp. 1–20.
87. Diveu, C., Lak-Hal, A.H., Froger, J., Ravon, E. *et al.* (2004), ‘Predominant expression of the long isoform of GP130-like (GPL) receptor is required for interleukin-31 signaling’, *Eur. Cytokine Netw.* Vol. 15, pp. 291–302.
88. Ward, L.D., Hammacher, A., Howlett, G.J., Matthews, J.M. *et al.* (1996), ‘Influence of interleukin-6 (IL-6) dimerization on formation of the high affinity hexameric IL-6.receptor complex’, *J. Biol. Chem.* Vol. 271, pp. 20138–20144.
89. Hilton, D.J., Hilton, A.A., Raicevic, A., Rakar, S. *et al.* (1994), ‘Cloning of a murine IL-11 receptor alpha-chain: Requirement for gp130 for high affinity binding and signal transduction’, *EMBO J.* Vol. 13, pp. 4765–4775.
90. Plun-Favreau, H., Perret, D., Diveu, C., Froger, J. *et al.* (2003), ‘Leukemia inhibitory factor (LIF), cardiotrophin-1, and oncostatin M share structural binding determinants in the immunoglobulin-like domain of LIF receptor’, *J. Biol. Chem.* Vol. 278, pp. 27169–27179.
91. Silver, J.S. and Hunter, C.A. (2010), ‘gp130 at the nexus of inflammation, autoimmunity, and cancer’, *J. Leukoc. Biol.* (Epub ahead of print).
92. Chattopadhyay, S., Tracy, E., Liang, P., Robledo, O. *et al.* (2007), ‘Interleukin-31 and oncostatin-M mediate distinct signaling reactions and response patterns in lung epithelial cells’, *J. Biol. Chem.* Vol. 282, pp. 3014–3026.
93. Trinchieri, G., Pflanz, S. and Kastelein, R.A. (2003), ‘The IL-12 family of heterodimeric cytokines: New players in the regulation of T cell responses’, *Immunity* Vol. 19, pp. 641–644.
94. Chua, A.O., Chizzonite, R., Desai, B.B., Truitt, T.P. *et al.* (1994), ‘Expression cloning of a human IL-12 receptor component. A new member of the cytokine receptor superfamily with strong homology to gp130’, *J. Immunol.* Vol. 153, pp. 128–136.
95. Parham, C., Chirica, M., Timans, J., Vaisberg, E. *et al.* (2002), ‘A receptor for the heterodimeric cytokine IL-23 is composed of IL-12Rbeta1 and a novel cytokine receptor subunit, IL-23R’, *J. Immunol.* Vol. 168, pp. 5699–5708.
96. Larousserie, F., Bardel, E., Pflanz, S., Arnulf, B. *et al.* (2005), ‘Analysis of interleukin-27 (EBI3/p28) expression in Epstein-Barr virus- and human T-cell leukemia virus type 1-associated lymphomas: Heterogeneous expression of EBI3 subunit by tumoral cells’, *Am. J. Pathol.* Vol. 166, pp. 1217–1228.
97. Pflanz, S., Hibbert, L., Mattson, J., Rosales, R. *et al.* (2004), ‘WSX-1 and glycoprotein 130 constitute a signal-transducing receptor for IL-27’, *J. Immunol.* Vol. 172, pp. 2225–2231.
98. Brunda, M.J. (1994), ‘Interleukin-12’, *J. Leukoc. Biol.* Vol. 55, pp. 280–288.
99. Oppmann, B., Lesley, R., Blom, B., Timans, J.C. *et al.* (2000), ‘Novel pp19 protein engages IL-12p40 to form a cytokine, IL-23, with biological activities similar as well as distinct from IL-12’, *Immunity* Vol. 13, pp. 715–725.

100. Niedbala, W., Wei, X.Q., Cai, B., Hueber, A.J. *et al.* (2007), ‘IL-35 is a novel cytokine with therapeutic effects against collagen-induced arthritis through the expansion of regulatory T cells and suppression of Th17 cells’, *Eur. J. Immunol.* Vol. 37, pp. 3021–3029.

101. Collison, L.W., Workman, C.J., Kuo, T.T., Boyd, K. *et al.* (2007), ‘The inhibitory cytokine IL-35 contributes to regulatory T-cell function’, *Nature* Vol. 450, pp. 566–569.

102. Hunter, C.A. (2005), ‘New IL-12-family members: IL-23 and IL-27, cytokines with divergent functions’, *Nat. Rev. Immunol.* Vol. 5, pp. 521–531.

103. Kisseleva, T., Bhattacharya, S., Braunstein, J. and Schindler, C.W. (2002), ‘Signaling through the JAK/STAT pathway, recent advances and future challenges’, *Gene* Vol. 285, pp. 1–24.

104. Heinrich, P.C., Behrmann, I., Müller-Newen, G., Schaper, F. *et al.* (1998), ‘Interleukin-6-type cytokine signalling through the gp130/Jak/STAT pathway’, *Biochem. J.* Vol. 334 (Pt. 2), pp. 297–314.

105. Gadina, M., Ferguson, P.R. and Johnston, J.A. (2003), ‘New interleukins: Are there any more?’, *Curr. Opin. Infect. Dis.* Vol. 16, pp. 211–217.

106. Goriely, S., Cavoy, R. and Goldman, M. (2009), ‘Interleukin-12 family members and type I interferons in Th17-mediated inflammatory disorders’, *Allergy* Vol. 64, pp. 702–709.

107. Fujihashi, K., Kono, Y. and Kiyono, H. (1992), ‘Effects of IL6 on B cells in mucosal immune response and inflammation’, *Res. Immunol.* Vol. 143, pp. 744–749.

108. Van, S.J., Renauld, J.C., Simpson, R.J., Uyttenhove, C. *et al.* (1989), ‘Mouse Il-6: A hybridoma growth factor with multiple effects on normal B, T cells’, *Ann. N. Y. Acad. Sci.* Vol. 557, pp. 206–13.

109. Schwertschlag, U.S., Trepicchio, W.L., Dykstra, K.H., Keith, J.C. *et al.* (1999), ‘Hematopoietic, immunomodulatory and epithelial effects of interleukin-11’, *Leukemia* Vol. 13, pp. 1307–1315.

110. Collison, L.W. and Vignali, D.A. (2008), ‘Interleukin-35: Odd one out or part of the family?’, *Immunol. Rev.* Vol. 226, pp. 248–262.

111. Castellani, M.L., Anogeianaki, A., Felaco, P., Toniato, E. *et al.* (2010), ‘IL-35, an anti-inflammatory cytokine which expands CD4+CD25+ Treg Cells’, *J. Biol. Regul. Homeost. Agents* Vol. 24, pp. 131–135.

112. Monteleone, G., Biancone, L., Marasco, R., Morrone, G. *et al.* (1997), ‘Interleukin 12 is expressed and actively released by Crohn’s disease intestinal lamina propria mononuclear cells’, *Gastroenterology* Vol. 112, pp. 1169–1178.

113. Zheng, T., Zhu, Z., Wang, J., Homer, R.J. *et al.* (2001), ‘IL-11: Insights in asthma from overexpression transgenic modeling’, *J. Allergy Clin. Immunol.* Vol. 108, pp. 489–496.

114. Fiorentino, D.F., Bond, M.W. and Mosmann, T.R. (1989), ‘Two types of mouse T helper cell. IV. Th2 clones secrete a factor that inhibits cytokine production by Th1 clones’, *J. Exp. Med.* Vol. 170, pp. 2081–2095.

115. Zdanov, A. (2004), ‘Structural features of the interleukin-10 family of cytokines’, *Curr. Pharm. Des.* Vol. 10, pp. 3873–3884.

116. Del, P.G., De, C.M., Almerigogna, F., Giudizi, M.G. *et al.* (1993), ‘Human IL-10 is produced by both type 1 helper (Th1) and type 2 helper (Th2) T cell clones and inhibits their antigen-specific proliferation and cytokine production’, *J. Immunol.* Vol. 150, pp. 353–360.

117. Murray, P.J. (2007), ‘The JAK-STAT signaling pathway: Input and output integration’, *J. Immunol.* Vol. 178, pp. 2623–2629.

118. Pletnev, S., Magracheva, E., Kozlov, S., Tobin, G. *et al.* (2003), ‘Characterization of the recombinant extracellular domains of human interleukin-20 receptors and their complexes with interleukin-19 and interleukin-20’, *Biochemistry* Vol. 42, pp. 12617–12624.

119. Liao, Y.C., Liang, W.G., Chen, F.W., Hsu, J.H. *et al.* (2002), ‘IL-19 induces production of IL-6 and TNF-alpha and results in cell apoptosis through TNF-alpha’, *J. Immunol.* Vol. 169, pp. 4288–4297.

120. Liu, L., Ding, C., Zeng, W., Heuer, J.G. *et al.* (2003), ‘Selective enhancement of multipotent hematopoietic progenitors in vitro and in vivo by IL-20’, *Blood* Vol. 102, pp. 3206–3209.

121. Sabat, R., Wallace, E., Endesfelder, S. and Wolk, K. (2007), ‘IL-19 and IL-20: Two novel cytokines with importance in inflammatory diseases’, *Expert Opin. Ther. Targets* Vol. 11, pp. 601–612.

122. Kingo, K., Mossner, R., Koks, S., Ratssep, R. *et al.* (2007), ‘Association analysis of IL19, IL20 and IL24 genes in palmoplantar pustulosis’, *Br. J. Dermatol.* Vol. 156, pp. 646–652.

123. Kingo, K., Mossner, R., Ratssep, R., Raud, K. *et al.* (2008), ‘Association analysis of IL20RA and IL20RB genes in psoriasis’, *Genes Immun.* Vol. 9, pp. 445–451.

124. Parrish-Novak, J., Xu, W., Brender, T., Yao, L. *et al.* (2002), ‘Interleukins 19, 20, and 24 signal through two distinct receptor complexes. Differences in receptor-ligand interactions mediate unique biological functions’, *J. Biol. Chem.* Vol. 277, pp. 47517–47523.

125. Xie, M.H., Aggarwal, S., Ho, W.H., Foster, J. *et al.* (2000), ‘Interleukin (IL)-22, a novel human cytokine that signals through the interferon receptor-related proteins CRF2-4 and IL-22R’, *J. Biol. Chem.* Vol. 275, pp. 31335–31339.

126. Wolk, K., Kunz, S., Asadullah, K. and Sabat, R. (2002), ‘Cutting edge: Immune cells as sources and targets of the IL-10 family members?’, *J. Immunol.* Vol. 168, pp. 5397–5402.

127. Liang, S.C., Tan, X.Y., Luxenberg, D.P., Karim, R. *et al.* (2006), ‘Interleukin (IL)-22 and IL-17 are coexpressed by Th17 cells and cooperatively enhance expression of antimicrobial peptides’, *J. Exp. Med.* Vol. 203, pp. 2271–2279.

128. Misse, D., Yssel, H., Trabattoni, D., Oblet, C. *et al.* (2007), ‘IL-22 participates in an innate anti-HIV-1 host-resistance network through acute-phase protein induction’, *J. Immunol.* Vol. 178, pp. 407–415.

129. de Oliveira, N.M., Ferreira, J.R., Jr., Colau, D., Fischer, H. *et al.* (2008), ‘Interleukin-22 forms dimers that are recognized by two interleukin-22R1 receptor chains’, *Biophys. J.* Vol. 94, pp. 1754–1765.

130. Chang, C., Magracheva, E., Kozlov, S., Fong, S. *et al.* (2003), ‘Crystal structure of interleukin-19 defines a new subfamily of helical cytokines’, *J. Biol. Chem.* Vol. 278, pp. 3308–3313.

131. Jiang, H., Lin, J.J., Su, Z.Z., Goldstein, N.I. *et al.* (1995), ‘Subtraction hybridization identifies a novel melanoma differentiation associated gene, mda-7, modulated during human melanoma differentiation, growth and progression’, *Oncogene* Vol. 11, pp. 2477–2486.

132. Commins, S., Steinke, J.W. and Borish, L. (2008), ‘The extended IL-10 superfamily: IL-10, IL-19, IL-20, IL-22, IL-24, IL-26, IL-28, and IL-29’, *J. Allergy Clin. Immunol.* Vol. 121, pp. 1108–1111.

133. Caudell, E.G., Mumm, J.B., Poinexter, N., Ekmekcioglu, S. *et al.* (2002), ‘The protein product of the tumor suppressor gene, melanoma differentiation-associated gene 7, exhibits immunostimulatory activity and is designated IL-24’, *J. Immunol.* Vol. 168, pp. 6041–6046.

134. Hor, S., Pirzer, H., Dumoutier, L., Bauer, F. *et al.* (2004), ‘The T-cell lymphokine interleukin-26 targets epithelial cells through the interleukin-20 receptor 1 and interleukin-10 receptor 2 chains’, *J. Biol. Chem.* Vol. 279, pp. 33343–33351.

135. Dambacher, J., Beigel, F., Zitzmann, K., De Toni, E.N. *et al.* (2009), ‘The role of the novel Th17 cytokine IL-26 in intestinal inflammation’, *Gut* Vol. 58, pp. 1207–1217.

136. Igawa, D., Sakai, M. and Savan, R. (2006), ‘An unexpected discovery of two interferon gamma-like genes along with interleukin (IL)-22 and -26 from teleost: IL-22 and -26 genes have been described for the first time outside mammals’, *Mol. Immunol.* Vol. 43, pp. 999–1009.

137. Kotenko, S.V., Gallagher, G., Baurin, V.V., Lewis-Antes, A. *et al.* (2003), ‘IFN-lambdas mediate antiviral protection through a distinct class II cytokine receptor complex’, *Nat. Immunol.* Vol. 4, pp. 69–77.

138. Dellgren, C., Gad, H.H., Hamming, O.J., Melchjorsen, J. *et al.* (2009), ‘Human interferon-lambda3 is a potent member of the type III interferon family’, *Genes Immun.* Vol. 10, pp. 125–131.

139. Witte, K., Gruetz, G., Volk, H.D., Looman, A.C. *et al.* (2009), ‘Despite IFN-lambda receptor expression, blood immune cells, but not keratinocytes or melanocytes, have an impaired response to type III interferons: implications for therapeutic applications of these cytokines’, *Genes Immun.* Vol. 10, pp. 702–714.

140. Dumoutier, L., Tounsi, A., Michiels, T., Sommereyns, C. *et al.* (2004), ‘Role of the interleukin (IL)-28 receptor tyrosine residues for antiviral and antiproliferative activity of IL-29/interferon-lambda 1: Similarities

with type I interferon signaling’, J. Biol. Chem. Vol. 279, pp. 32269–32274.

141. Coccia, E.M., Severa, M., Giacomini, E., Monneron, D. *et al.* (2004), ‘Viral infection and Toll-like receptor agonists induce a differential expression of type I and lambda interferons in human plasmacytoid and monocyte-derived dendritic cells’, *Eur. J. Immunol.* Vol. 34, pp. 796–805.

142. Robek, M.D., Boyd, B.S. and Chisari, F.V. (2005), ‘Lambda interferon inhibits hepatitis B and C virus replication’, *J. Virol.* Vol. 79, pp. 3851–3854.

143. Ank, N., Iversen, M.B., Bartholdy, C., Staeheli, P. *et al.* (2008), ‘An important role for type III interferon (IFN-lambda/IL-28) in TLR-induced antiviral activity’, *J. Immunol.* Vol. 180, pp. 2474–2485.

144. Lasfar, A., Lewis-Antes, A., Smirnov, S.V., Anantha, S. *et al.* (2006), ‘Characterization of the mouse IFN-lambda ligand-receptor system: IFN-lambdas exhibit antitumor activity against B16 melanoma’, *Cancer Res.* Vol. 66, pp. 4468–4477.

145. Rouvier, E., Luciani, M.F., Mattei, M.G., Denizot, F. *et al.* (1993), ‘CTLA-8, cloned from an activated T cell, bearing AU-rich messenger RNA instability sequences, and homologous to a herpesvirus saimiri gene’, *J. Immunol.* Vol. 150, pp. 5445–5456.

146. Wright, J.F., Guo, Y., Quazi, A., Luxenberg, D.P. *et al.* (2007), ‘Identification of an interleukin 17F/17A heterodimer in activated human CD4+ T cells’, *J. Biol. Chem.* Vol. 282, pp. 13447–13455.

147. Chang, S.H. and Dong, C. (2007), ‘A novel heterodimeric cytokine consisting of IL-17 and IL-17F regulates inflammatory responses’, *Cell Res.* Vol. 17, pp. 435–440.

148. Shi, Y., Ullrich, S.J., Zhang, J., Connolly, K. *et al.* (2000), ‘A novel cytokine receptor-ligand pair. Identification, molecular characterization, and in vivo immunomodulatory activity’, *J. Biol. Chem.* Vol. 275, pp. 19167–19176.

149. Gaffen, S.L. (2009), ‘Structure and signalling in the IL-17 receptor family’, *Nat. Rev. Immunol.* Vol. 9, pp. 556–567.

150. Korn, T., Bettelli, E., Oukka, M. and Kuchroo, V.K. (2009), ‘IL-17 and Th17 Cells’, *Annu. Rev. Immunol.* Vol. 27, pp. 485–517.

151. Fort, M.M., Cheung, J., Yen, D., Li, J. *et al.* (2001), ‘IL-25 induces IL-4, IL-5, and IL-13 and Th2-associated pathologies in vivo’, *Immunity* Vol. 15, pp. 985–995.

152. Ziolkowska, M., Koc, A., Luszczycykiewicz, G., Ksiezopolska-Pietrzak, K. *et al.* (2000), ‘High levels of IL-17 in rheumatoid arthritis patients: IL-15 triggers in vitro IL-17 production via cyclosporin A-sensitive mechanism’, *J. Immunol.* Vol. 164, pp. 2832–2838.

153. Liu, Z.J., Yadav, P.K., Su, J.L., Wang, J.S. *et al.* (2009), ‘Potential role of Th17 cells in the pathogenesis of inflammatory bowel disease’, *World J. Gastroenterol.* Vol. 15, pp. 5784–5788.

154. Yu, J.J. and Gaffen, S.L. (2008), ‘Interleukin-17: A novel inflammatory cytokine that bridges innate and adaptive immunity’, *Front. Biosci.* Vol. 13, pp. 170–177.

155. Shono, T., Ono, M., Izumi, H., Jimi, S.I. *et al.* (1996), ‘Involvement of the transcription factor NF-kappaB in tubular morphogenesis of human microvascular endothelial cells by oxidative stress’, *Mol. Cell Biol.* Vol. 16, pp. 4231–4239.

156. Strieter, R.M., Kunkel, S.L., Showell, H.J., Remick, D.G. *et al.* (1989), ‘Endothelial cell gene expression of a neutrophil chemotactic factor by TNF-alpha, LPS, and IL-1 beta’, *Science* Vol. 243, pp. 1467–1469.

157. Yoshimura, T., Matsushima, K., Tanaka, S., Robinson, E.A. *et al.* (1987), ‘Purification of a human monocyte-derived neutrophil chemotactic factor that has peptide sequence similarity to other host defense cytokines’, *Proc. Natl. Acad. Sci. USA* Vol. 84, pp. 9233–9237.

158. Lakshminarayanan, V., Drab-Weiss, E.A. and Roebuck, K.A. (1998), ‘H2O2 and tumor necrosis factor-alpha induce differential binding of the redox-responsive transcription factors AP-1 and NF-kappaB to the interleukin-8 promoter in endothelial and epithelial cells’, *J. Biol. Chem.* Vol. 273, pp. 32670–32678.

159. Najakshin, A.M., Mechetina, L.V., Alabyev, B.Y. and Taranin, A.V. (1999), ‘Identification of an IL-8 homolog in lamprey (*Lampetra fluviatilis*): Early evolutionary divergence of chemokines’, *Eur. J. Immunol.* Vol. 29, pp. 375–382.

160. Ambrus, J.L., Jr. and Fauci, A.S. (1985), ‘Human B lymphoma cell line producing B cell growth factor’, *J. Clin. Invest.* Vol. 75, pp. 732–739.

161. Ambrus, J.L., Jr., Chesky, L., Stephany, D., McFarland, P. *et al.* (1990), ‘Functional studies examining the subpopulation of human B lymphocytes responding to high molecular weight B cell growth factor’, *J. Immunol.* Vol. 145, pp. 3949–3955.

162. Nogami, S., Satoh, S., Nakano, M., Shimizu, H. *et al.* (2003), ‘Taxilin: A novel syntaxin-binding protein that is involved in Ca2+-dependent exocytosis in neuroendocrine cells’, *Genes Cells* Vol. 8, pp. 17–28.

163. Nogami, S., Satoh, S., Tanaka-Nakadate, S., Yoshida, K. *et al.* (2004), ‘Identification and characterization of taxilin isoforms’, *Biochem. Biophys. Res. Commun.* Vol. 319, pp. 936–943.

164. Shen, L., Suresh, L., Li, H., Zhang, C. *et al.* (2009), ‘IL-14 alpha, the nexus for primary Sjogren’s disease in mice and humans’, *Clin. Immunol.* Vol. 130, pp. 304–312.

165. Bacon, K., Baggiolini, M., Broxmeyer, H., Horuk, R. *et al.* (2002), ‘Chemokine/chemokine receptor nomenclature’, *J. Interferon Cytokine Res.* Vol. 22, pp. 1067–1068.

166. Zhang, Y., Center, D.M., Wu, D.M., Cruikshank, W.W. *et al.* (1998), ‘Processing and activation of pro-interleukin-16 by caspase-3’, *J. Biol. Chem.* Vol. 273, pp. 1144–1149.

167. Baier, M., Bannert, N., Werner, A., Lang, K. *et al.* (1997), ‘Molecular cloning, sequence, expression, and processing of the interleukin 16 precursor’, *Proc. Natl. Acad. Sci. USA* Vol. 94, pp. 5273–5277.

168. Wilson, K.C., Center, D.M. and Cruikshank, W.W. (2004), ‘The effect of interleukin-16 and its precursor on T lymphocyte activation and growth’, *Growth Factors* Vol. 22, pp. 97–104.

169. Liu, C., Mills, J., Dixon, K., Vennarini, J. *et al.* (2007), ‘IL-16 signaling specifically induces STAT6 activation through CD4’, *Cytokine* Vol. 38, pp. 145–150.

170. Cruikshank, W.W., Center, D.M., Nisar, N., Wu, M. *et al.* (1994), ‘Molecular and functional analysis of a lymphocyte chemoattractant factor: Association of biologic function with CD4 expression’, *Proc. Natl. Acad. Sci. USA* Vol. 91, pp. 5109–5113.

171. Center, D.M., Kornfeld, H. and Cruikshank, W.W. (1996), ‘Interleukin 16 and its function as a CD4 ligand’, *Immunol. Today* Vol. 17, pp. 476–481.

172. Mathy, N.L., Scheuer, W., Lanzendorfer, M., Honold, K. *et al.* (2000), ‘Interleukin-16 stimulates the expression and production of pro-inflammatory cytokines by human monocytes’, *Immunology* Vol. 100, pp. 63–69.

173. Glass, W.G., Sarisky, R.T. and Vecchio, A.M. (2006), ‘Not-so-sweet sixteen: The role of IL-16 in infectious and immune-mediated inflammatory diseases’, *J. Interferon Cytokine Res.* Vol. 26, pp. 511–520.

174. Jemth, P. and Gianni, S. (2007), ‘PDZ domains: Folding and binding’, *Biochemistry* Vol. 46, pp. 8701–8708.

175. Dahl, C.A., Schall, R.P., He, H.L. and Cairns, J.S. (1992), ‘Identification of a novel gene expressed in activated natural killer cells and T cells’, *J. Immunol.* Vol. 148, pp. 597–603.

176. Kim, S.H., Han, S.Y., Azam, T., Yoon, D.Y. *et al.* (2005), ‘Interleukin-32: A cytokine and inducer of TNFalpha’, *Immunity* Vol. 22, pp. 131–142.

177. Netea, M.G., Azam, T., Ferwerda, G., Girardin, S.E. *et al.* (2005), ‘IL-32 synergizes with nucleotide oligomerization domain (NOD) 1 and NOD2 ligands for IL-1beta and IL-6 production through a caspase 1-dependent mechanism’, *Proc. Natl. Acad. Sci. USA* Vol. 102, pp. 16309–16314.

178. Conti, P., Youinou, P. and Theoharides, T.C. (2007), ‘Modulation of autoimmunity by the latest interleukins (with special emphasis on IL-32)’, *Autoimmun. Rev.* Vol. 6, pp. 131–137.

179. Shioya, M., Nishida, A., Yagi, Y., Ogawa, A. *et al.* (2007), ‘Epithelial overexpression of interleukin-32alpha in inflammatory bowel disease’, *Clin. Exp. Immunol.* Vol. 149, pp. 480–486.

180. Marcondes, A.M., Mhyre, A.J., Stirwalt, D.L., Kim, S.H. *et al.* (2008), ‘Dysregulation of IL-32 in myelodysplastic syndrome and chronic myelomonocytic leukemia modulates apoptosis and impairs NK function’, *Proc. Natl. Sci. USA* Vol. 105, pp. 2865–2870.

181. Rasool, S.T., Tang, H., Wu, J., Li, W. *et al.* (2008), ‘Increased level of IL-32 during human immunodeficiency virus infection suppresses HIV replication’, *Immunol. Lett.* Vol. 117, pp. 161–167.

182. Lin, H., Lee, E., Hestir, K., Leo, C. *et al.* (2008), ‘Discovery of a cytokine and its receptor by functional screening of the extracellular proteome’, *Science* Vol. 320, pp. 807–811.

183. Wei, S., Nandi, S., Chitu, V., Yeung, Y.G. *et al.* (2010), ‘Functional overlap but differential expression of CSF-1 and IL-34 in their CSF-1 receptor-mediated regulation of myeloid cells’, *J. Leukoc. Biol.* Vol. 88, pp. 495–505.

184. Garceau, V., Smith, J., Paton, I.R., Davey, M. *et al.* (2010), ‘Pivotal Advance: Avian colony-stimulating factor 1 (CSF-1), interleukin-34 (IL-34), and CSF-1 receptor genes and gene products’, *J. Leukoc. Biol.* Vol. 87, pp. 753–764.

185. Pandit, J., Bohm, A., Jancarik, J., Halenbeck, R. *et al.* (1992), ‘Three-dimensional structure of dimeric human recombinant macrophage colony-stimulating factor’, *Science* Vol. 258, pp. 1358–1362.
