
Biochimica et Biophysica Acta 1815 (2011) 170–188

Contents lists available at ScienceDirect

Biochimica et Biophysica Acta

journal homepage: www.elsevier.com/locate/bbacan

Review

The RasGrf family of mammalian guanine nucleotide exchange factors

Alberto Fernández-Medarde, Eugenio Santos*

Centro de Investigación del Cáncer, IBMCC (CSIC-USAL), University of Salamanca, 37007 Salamanca, Spain

ARTICLE INFO

Article history:
Received 20 September 2010
Accepted 14 November 2010
Available online 25 November 2010

Keywords:
RasGrf1
RasGrf2
RasGEF
Guanine nucleotide exchange factor
Signal transduction
Cell growth/differentiation

ABSTRACT

RasGrf1 and RasGrf2 are highly homologous mammalian guanine nucleotide exchange factors which are able to activate specific Ras or Rho GTPases. The RasGrf genes are preferentially expressed in the central nervous system, although specific expression of either locus may also occur elsewhere. RasGrf1 is a paternally-expressed, imprinted gene that is expressed only after birth. In contrast, RasGrf2 is not imprinted and shows a wider expression pattern. A variety of isoforms for both genes are also detectable in different cellular contexts. The RasGrf proteins exhibit modular structures composed by multiple domains including CDC25H and DHPH motifs responsible for promoting GDP/GTP exchange, respectively, on Ras or Rho GTPase targets. The various domains are essential to define their intrinsic exchanger activity and to modulate the specificity of their functional activity so as to connect different upstream signals to various downstream targets and cellular responses.

Despite their homology, RasGrf1 and RasGrf2 display differing target specificities and non overlapping functional roles in a variety of signaling contexts related to cell growth and differentiation as well as neuronal excitability and response or synaptic plasticity. Whereas both RasGrfs are activatable by glutamate receptors, G-protein-coupled receptors or changes in intracellular calcium concentration, only RasGrf1 is reported to be activated by LPA, cAMP, or agonist-activated Trk and cannabinoid receptors. Analysis of various knockout mice strains has uncovered a specific functional contribution of RasGrf1 in processes of memory and learning, photoreception, control of post-natal growth and body size and pancreatic β-cell function and glucose homeostasis. For RasGrf2, specific roles in lymphocyte proliferation, T-cell signaling responses and lymphogenesis have been described.

© 2010 Elsevier B.V. All rights reserved.

Contents

1. Introduction . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
A. Fernández-Medarde, E. Santos / Biochimica et Biophysica Acta 1815 (2011) 170–188

3.1.5. PH2 domain . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
A. Fernández-Medarde, E. Santos / Biochimica et Biophysica Acta 1815 (2011) 170–188

RasGrf1

Fig. 1. Genomic structure and intron-exon distribution of the RasGrf1 locus. Genomic organization of introns and exons in the RasGrf1 locus of mouse, rat and humans. Figure assembled using data from the Ensembl genome browser database ([http://www.ensembl.org/index.html](http://www.ensembl.org/index.html)). The correspondence of coding exons to the modular domains of the full length RasGrf1 protein is indicated by the dotted lines. Chromosomal location, chromosome strand (+ or −) used for transcription and size of the genomic stretch containing the locus are also indicated for each species.

conserved CpG island [25]. A 489 bp region located immediately 5′ upstream of the first transcription initiation site and housing both AP-1 sites and Oct1 binding sites was initially shown to enhance RasGrf1 expression; this region also displays a GC-box, but in contrast to yeast CDC25, it lacks a TATA-box or Inr consensus sequences [25]. More recently, this expression-enhancing area has been expanded to a 1.3 Kbp stretch where two additional, active c-Jun/AP1 consensus sequences, able to activate the mRasGrf1 promoter, were identified [26]. The mRasGrf1 gene produces two main transcripts (see Ensemble data base for further information, [http://www.ensembl.org/Mus_musculus/Gene/Summary?g=ENSMUSG00000032356](http://www.ensembl.org/Mus_musculus/Gene/Summary?g=ENSMUSG00000032356)). The largest one (ENSMUST00000034912) is 4225 bps long and is formed by 26 separate exons, coding for a total of 1262 residues. The shortest transcript, designated GRFβ, was identified in mouse pancreas (ENSMUST00000034909). It is 1000 bps long and contains 3 exons (only the first one is shared with the full length transcript)

RasGrf2

Fig. 2. Genomic structure and intron-exon distribution of the RasGrf2 locus. Genomic organization of introns and exons in the RasGrf2 locus of mouse, rat and humans. Figure assembled using data from the Ensembl genome browser database ([http://www.ensembl.org/index.html](http://www.ensembl.org/index.html)). The correspondence of coding exons to the modular domains of the full length RasGrf2 protein is indicated by the dotted lines. Chromosomal location, chromosome strand (+ or −) used for transcription and size of the genomic stretch containing the locus are also indicated for each species. The genomic structure of the central part of the RasGrf2 mouse locus (indicated by the discontinuous line) is not yet completed in the database.

coding for 178 residues which are predicted to conform a single PH domain [27] (Fig. 1).

The human RasGrf1 gene is organized in 28 separate exons and located on chromosome 15 (Fig. 1) [25]. The nucleotide sequence of hRasGrf1 is predicted to encode for a variety of modular peptide motifs showing a similar sequential distribution than in mouse RasGrf1. Only minor, discrete sequence differences are found between human and mouse RasGrf1, which share an overall 82.7% identity. Human RasGrf1 contains an extra 13a exon whose expression does not affect any of the canonical conserved domains. In addition, the equivalent to mouse exon 18 in the hRasGrf1 gene is divided in two separate 18a and 18b exons. The last exon of hRasGrf1 contains the termination codon, a 64 bp 3′UTR and a polyadenylation signal sequence [25]. The hRasGrf1 gene (http://www.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000058335) is usually transcribed into a canonical, full length 6313 bps transcript (ENST00000419573) coding for 1273 aminoacid residues (ENSP00000405963). Studies of human brain cDNA libraries have also allowed the isolation of a shorter, 2.8 kb isoform designated H-Grf55 exhibiting a differing 5′UTR and a shorter N-terminal region than the full length, canonical isoform [28] (ENST00000394745, ENSP00000378228). H-Grf55 has an ORF of 2844 bps, corresponding to 13 distinct exons coding for 489 residues predicted to conform several peptide motifs including a REM domain and the CDC25-H domain. The H-Grf55 isoform has been shown to be able to complement CDC25 in Saccharomyces cerevisiae and to activate Ras signaling in mammalian cells [28]. Consistent with its modular structure (it lacks a DHPH domain), H-Grf55-dependent GDP releasing ability appears to be limited to Ras family proteins [28].

Screening of rat brain cDNA libraries with mRasGrf1-derived oligonucleotides resulted in the initial identification of a 5 kb ORF coding for a 142.65 kDa protein. The C-terminus of this RasGrf1 rat brain protein displays 28% homology to the same region of yeast CDC25 and is able to release GDP from Ras but not from CDC42 or RalA [29]. The rat RasGrf1 gene is located in chromosome 8, and at least three different transcripts have been identified for it in rat tissues (http://www.ensembl.org/Rattus_norvegicus/Gene/Summary?g=ENSRNOG00000014025). The larger one (ENSRNOT00000036149) is 89.9% homologous to the mouse full length transcript (its 5′UTR is bigger than that in the mouse counterpart) and has a size of 3759 bps, corresponding to 31 separate exons and 1252 residues (ENSRNOP00000030985). Its cDNA sequence is predicted to encode several peptide motifs including a PH domain, an IQ domain, a DH domain, a second PH domain, a REM domain and the CDC25-H domain, although there are no further data on the predicted protein. Another transcript (ENSRNOT00000018794) is 3732 bps long, originated from 28 separate exons and leading to the expression or a polypeptide containing 1243 residues (ENSRNOP00000018794). There is experimental evidence showing the expression of this protein in rat brain, but most studies have been centered on its human or mouse counterparts. Nevertheless, its 89.6% identity with mRasGrf1 predicts that they must have very similar signaling and physiological properties. The cDNA sequence is predicted to encode the same modular peptide motifs, with the same sequential distribution observed in the other isoform. The smallest transcript (ENSRNOT00000018702) is 1189 bps, corresponding to 4 individual exons and encoding for a protein of 177 residues. This transcript which is predicted to encode only a PH domain is expressed only in brain and pancreas and is similar to mouse GRFβ [27].

The current knowledge on the structure of the RasGrf2 gene in rodents and humans is less detailed than with RasGrf1 (Fig. 2). This is due, at least in part, to its discovery and characterization as a separate gene occurring several years later than RasGrf1.

An initial, full-length RasGrf2 cDNA sequence of 3567 bps encoding a predicted protein of 1189 amino acids was first identified by amplification of a mouse cDNA brain library [8]. The complete RasGrf2 mouse gene comprises 24 separate exons, filling an 877,413 bp stretch region of the long arm of chromosome 13 [30]. The full length murine mRNA transcript is 7595 bp long and it encodes for an 1189 amino

acids protein with a molecular weight of 135,668 Da (http://www.ncbi.nlm.nih.gov/nuccore/258613876?report=genbank). In humans, RasGrf2 (ENSG00000113319) is located in a 265,396 stretch of chromosome 5 and comprises 27 distinct exons that are transcribed into a 8100 bp transcript (ENST00000265080) which is itself translated into a 1237 amino acids protein product (ENSP00000265080), with a predicted molecular weight of 140,764 Da (Fig. 2).

### 2.2. Control of expression of the RasGrf loci

Control of RasGrf1 transcription is exerted mostly through epigenetic methylation mechanisms [31] and alternative splicing. RasGrf1 is an imprinted gene, expressed only after birth in mouse, rat and humans. Studies in the neonatal mouse brain have shown that it is expressed exclusively from the paternal allele, although bi-allelic expression has also been reported in other mouse tissues including lung, ovary and testis [25,32].

Imprinted methylation of the paternal RasGrf1 allele takes place at a differentially methylated domain (DMD) located 30 kb 5′ of the transcription initiating site [33,34]. A direct repeat sequence (40 units of a 41-mer) located immediately downstream of DMD is responsible for control of RasGrf1 methylation in the male germline and maintenance of DMD methylation in the pre-implanted embryo [35,36]. DMD is an enhancer blocker that binds to the CTCF factor (a CCCTC-binding protein) in a methylation dependent manner. The unmethylated maternal allele binds to CTCF resulting in blocked RasGrf1 expression. In contrast, CTCF binding is prevented by DMD-mediated methylation of the paternal allele, thus allowing expression of RasGrf1 [37]. It is apparent that the 41-mer repeat and the DMD sequences provide an instrumental binary switch allowing regulation of imprinting and timing of expression of the RasGrf1 locus. A recent report has demonstrated that DNA methylation and histone methylation are mutually exclusive at the RasGrf1 locus, suggesting an additional level of fine-tuned regulatory mechanisms controlling expression of the paternal or the maternal alleles of this imprinted gene (Fig. 3) [38].

Postnatal expression of the RasGrf1 gene can be influenced by a variety of signaling events. For example, psychostimulants such as cocaine, amphetamine or Δ-9-Tetrahydrocannabinol have been shown to increase in vivo RasGrf1 expression in the rat striatum and cortex as well as the mouse cerebellum [39–41]. In contrast, significant repression of RasGrf1 expression has been detected by means of microarrays in the hippocampus of transgenic mice overexpressing the Alzheimer-related amyloid precursor protein with the Swedish mutation (APPSw) [42] and in the mammary gland of bi-transgenic mice overexpressing oncogenic ErbB2/Neu and luteinizing hormone [43]. RasGrf1 was also shown to be strongly downregulated in a proteomic analysis of the cortex of transgenic mice for the NMDA Receptor subunit NR2B, suggesting a negative role of NR2B in control of RasGrf1 expression or protein stability [44]. Finally, in cultured Rat1a cells, c-Jun/AP-1 has also been shown to exert tight control of the expression of a short p75-RasGrf1 transcript, thus resulting in an increased non-adherent growth of the culture [26].

In contrast to the other family member, RasGrf2 is not an imprinted gene. However, its expression may also involve, at least in part, epigenetic regulatory mechanisms. Specifically, significant methylation of the 5′ region of RasGrf2 has been detected in pancreatic cancer cell lines, rat mammary carcinomas, and in colon and lung cancer, although further studies are needed to determine if RasGrf2 acts as a tumor suppressor in those tissues [45–47]. RasGrf2 is expressed in mouse embryonic stem cells, where it was initially isolated [8] and it has also been reported to show faint expression at P0 in mice pups, with its intracellular concentration increasing significantly during the first weeks of age [48]. Further studies are needed to fully characterize if and when RasGrf2 is expressed during prenatal stages in mice. Regarding the control of RasGrf2 expression by exogenous cellular

A. Fernández-Medarde, E. Santos / Biochimica et Biophysica Acta 1815 (2011) 170–188

Fig. 3. Regulatory mechanisms controlling brain expression of the imprinted RasGrf1 locus by methylation. RasGrf1 is an imprinted gene expressed only after birth from the paternal allele. This expression pattern is achieved through a mechanism involving tight control of methylation (star symbols ✩) of upstream regulatory elements in the male and female alleles. In the male germline (♂), during gametogenesis a 41mer repeat region is methylated by DMT (1.), leading to the methylation of the DMD and other upstream regions (2.). These methylation events inhibit the access of the CCCTC-binding factor protein (CTCF) and the transcription suppressor YY1 to the DMD region and also avoid methylation in lysine K27 of histone H3 by PRC2 (3.). The absence of CTCF and YY1 in the DMD region allows an upstream enhancer to interact with and promote transcription from the RasGrf1 promoter (4.). In the female allele (♀), the early DMT-driven methylation event does not take place in the germline (5.) and, as a result, YY1, CTCF and PRC2 have access to the RasGrf1 upstream regulatory region, resulting in YY1 and CTCF binding to the 41mer and DMD regions (6.) and PRC2 methylation in lysine (K27) of histone H3 (7.). This situation is kept during embryogenesis and in the adult brain, with the K27-methylated histone H3 inhibiting DNA methylation of the tandem repeats (8.) and the DNA-bound CTCF and YY1 factors avoiding the functional interaction of the enhancer with the promoter for RasGrf1 expression (9.). DMT, DNA Methyl Transferase. DMD, Differentially Methylated Domain. CTCF, CCCTC-binding factor. YY1, Yin and Yang 1 protein. PRC2, Polycomb repressive complex-2. MK27, Methylated Lysine 27. Schematic representation adapted from [38].

signaling events, an early report showed that methamphetamine treatment results in a significant reduction of RasGrf2 expression in the mouse striatum [49].

### 2.3. RasGrf expression variants and isoforms

As with many other RasGEFs from yeast to mammals, transcription of the RasGrf loci involves significant levels of alternative splicing resulting in the generation of a great variety of isoforms that show differential expression in specific tissues or developmental stages. However, the functional significance of many of these differentially expressed transcripts remains largely unknown.

Regarding RasGrf1, initial transcriptional analyses of specific mouse tissues allowed the detection in brain of two heavily expressed 5.0 and 5.5 kb transcripts that were also detected much more faintly in the heart. Moderate expression of a smaller, 1.7 kb mRNA species has also been reported in the brain, heart, liver, spleen and testis [5,29,50].

A variety of mRNA isoforms for RasGrf1 have also been characterized in various human tissues. Thus, specific RasGrf1 mRNA sizes of 9 kb, 6 kb, 5.5 kb, 2.4 kb and 2.2 kb have been reported in the human adult brain [6,28,51]. Transcriptional analysis of the human lung yielded mRNA species of 5 kb, 2.4 kb and 2.2 kb, whereas the human pancreas produced two mRNA forms of 5 kb and 2.2 kb [51]. Likewise, specific RasGrf1 mRNA species observed in the kidney include 9 kb and 2.4 kb bands, whereas skeletal muscle and heart expressed preferentially a 2.2 kb mRNA form [6,28,51]. Finally, expression in fetal brain and lung of 9 kb, 5 kb, 2.4 kb and 2.2 kb mRNA species has also been detected although the expression of corresponding protein isoforms has not been fully characterized [51].

Protein expression detection analysis by means of Western immunoblot identified a specific, full length 140 kDa RasGrf1 a protein product in the brain of adult mouse [52]. The full length protein form has also been detected in isolated pancreatic islets, [19] and a smaller, 20 kDa isoform designated Grf-beta has also been identified in the mouse pancreas [27]. Finally, a 58 kDa RasGrf1 protein was found in mouse embryonic brain tissue, suggesting the occurrence of differential transcriptional/translational control mechanisms at different stages of mouse brain development [50].

The predicted, full length 140 kDa RasGrf1 protein was easily identified in both the particulate and cytosolic fraction of rat brain, but not in the liver [29]. p140 RasGrf1 expression is detected in mature neurons of the brain and spinal cord, but not in glial cells. The main sites of RasGrf1 expression in rat brain include the hippocampus, thalamus, hypothalamus, cortex and cerebellum, where a significant co-expression with its purported main target, H-Ras, is also observed [18,29,53,54]. Outside the rat CNS, a small RasGrf1 protein of 54 kDa is expressed in the exocrine acinar pancreas [55].

Full length p140 RasGrf1 protein is heavily expressed in the human brain but significant levels of expression are also detected in other tissues such as the lung and pancreas. Furthermore, smaller-size RasGrf1 protein peptides were also identified by immunoblot in the human brain, lung and skeletal muscle [51].

Expression of a variety of RasGrf1 protein sizes has also been detected in different cell lines. Thus, a p75 RasGrf1 protein form has been reported in rat1a cells [26] whereas RasGrf1 peptides of 100 kDa and 64 kDa have also been detected in the human M2 melanoma cell line [56].

RasGrf2 expression is also restricted to specific tissue locations. Full length RasGrf2 protein has been preferentially identified in mouse CNS and lungs. Using specific antibodies raised against a C-terminal motif
of RasGrf2, an early study detected in a ca. 140 kDa brain tissue protein that was absent in many other tissues [7]. This protein size was confirmed after cloning and expression of the full length murine cDNA predicting a translated protein of 135 kDa [8]. Full length RasGrf2 (p135 kDa) has also been observed in mice lung and retina in later studies using a different antibody recognizing a specific region located between the REM and catalytic domains of RasGrf2 [17,20]. Smaller immunoreactive protein bands (90, 75, 56, 50 and 31 kDa) of RasGrf2 have also been described in various mouse tissues including muscle (75 and 50 kDa), liver (75 and 50 kDa), kidney (all but p135 kDa), brain (135, 75 and 50 kDa), heart (75, 56 and 50 kDa), lung (75 kDa), spleen (75 kDa), pancreas (75 and 50 kDa) and testis (75, 56 and 50 kDa). All these protein band sizes were also observed in mouse embryonic stem cells [7].

In rat, three major RasGrf2 mRNA bands are detected by Northern blot, including a 9 kb mRNA detected only in brain, as well as 12 and 2.2 kb bands also detected in brain and in other tissues like pancreas, spleen, liver and kidney [30].

Reports analyzing transcription or translation variants of human RasGrf2 are scarce. An early report dealing with cloning and expression of full-length hRasGrf2 showed expression of a 136 kDa protein [57]. Further work is needed to analyze functionally the full length hRasGrf2 and to determine the potential expression of smaller isoforms similar to those found in mice tissues.

### 2.4. Regulation of cellular RasGrf protein levels

Specific modulation of the pathways of intracellular proteolytic degradation of RasGrf1 and Rasgrf2 constitutes an additional control mechanism to regulate the cellular levels of these proteins [56,58–61]. The CDB (cyclin destruction box) and PEST domains present in the C-terminal regions of both RasGrf1 and RasGrf2 (Fig. 4) offer the structural basis for this kind of regulation, linking, respectively, to processes of ubiquitination and subsequent proteasome degradation or to degradation by other type of proteases, such as calpain.

Thus, it has been shown that the interaction between RasGrf1 and the de-ubiquitinating enzyme mUBPy in COS7 and Hek293 cells results in an increased half life for RasGrf1 [60]. Another report in M2 melanoma cells has also documented that interaction with filamin A results in downregulated levels of RasGrf1 (peptides ~100 and

64 kDa), in correlation with a decreased MMP9 expression in those cells [56].

The canonical PEST region of RasGrf1 offers also a defined target for calpain-type proteases, although the exact mechanism and functional significance of RasGrf1 cleavage by these proteases requires further clarification. Thus, early in vitro studies showed that cleavage by calpain increased the intrinsic RasGEF activity of RasGrf1 on purified H-Ras proteins [58]. These observations suggested a regulatory model by which the N-terminal moiety of RasGrf1 would be acting as a negative modulator of the catalytic activity of the CDC25-H domain located in the C-terminal portion of the full length protein. Cleavage of the N-terminus by calpain would release the CDC25H from such an inhibition [58].

In contrast to such model, other studies using CHO cells and mouse brain sections reported that RasGrf1 phosphorylation on Serine731 (located within the PEST rich region) targets RasGrf1 for proteolysis by m-Calpain and results in reduced Ras activation and subsequent AKT phosphorylation [61]. Further studies will be needed to clarify these potentially conflicting reports regarding regulatory mechanism mediated by calpain.

As for RasGrf1, various reports have described regulatory mechanisms controlling the intracellular concentration of RasGrf2 through modulation of its intracellular proteolysis. For example, it has been shown that intracellular binding of RasGrf2 to its target H-Ras protein triggers ubiquitination of its CDB domain, thus leading to rapid degradation of the RasGrf2 protein by the proteasome [59]. An alternative mechanism controlling RasGrf2 concentration has also been described showing that phosphorylation of RasGrf2 by p35/CDK5 results in significant accumulation and increased local concentration of RasGrf2 in neuronal cell bodies. This phosphorylation inhibits Rac1, but not Ras, activation by RasGrf2 [62]. Finally, although RasGrf2 contains a PEST rich region, we are unaware of any published reports directly characterizing the degradation of RasGrf2 by Calpain-type proteases.

## 3. Structure and organization of the RasGrf proteins

RasGrfs are large, multidomain proteins that act as signaling mediators transducing a variety of upstream stimuli to their Ras family GTPase targets (Fig. 4).

![Diagram](https://i.imgur.com/yourimageurl.png)

**Fig. 4.** Modular domain structure of the RasGrf proteins. Domain distribution of the full length RasGrf1 and RasGrf2 proteins and two organ-specific RasGrf1 isoforms. Protein sizes are drawn to approximate scale. Residue numbers marking boundaries of the domains are those of the respective mouse full length cDNA forms. See Section 3.1. for details. PH, Pleckstrin homology domain. CC, Coiled coil domain. IQ, Isoleucine (I)/Glutamine (Q) motif. DH, Dbl homology domain. ITIM, Immuno Tyrosine based Inhibition Motifs. REM, Ras Exchanger Motif. CDB, Cyclin Destruction Box. ND, Neurological Domain. PEST Rich Region, (P)-Proline, (E)-Glutamic acid, (S)-Serine and (T)-Threonine rich region. CDC25H, CDC25 homology domain.
RasGrf1 is the “big brother” of the family (1273 amino acids and 145 kDa in humans vs. 1262 residues and 144 kDa in the mouse, sharing 83% identity between these two species). The main difference between hRasGrf1 and mRasGrf1 lies in a 13 amino acids “insert” occurring in hRasGrf1 at a position corresponding between residues 614 and 615 of mRasGrf1.

RasGrf2, the second family member, is highly homologous to (80% homology, 63% identity), and slightly smaller than (141 kDa in humans, 136 kDa in mouse) RasGrf1. The main difference between hRasGrf2 (1237 residues) and mRasGrf2 (1189 amino acids) lies in “inserts” of 42 amino acids and 7 amino acids occurring, respectively, between residues 783–784 and 795–796 of mRasGrf2.

A wide variety of shorter versions of both RasGrf1 and RasGrf2 proteins have been described in different biological contexts. Among these, it is worth mentioning Grfβ, a ca. 20 kDa isoform identified in mouse pancreas [27] or HGRF55, a 55 kDa isoform identified in human brain [28] (Figs. 1 and 4).

### 3.1. Modular domain structure and function

Like most other GEFs, the primary structure of the mammalian RasGrfs is composed of a defined, linear sequence of functionally distinct modular protein domains. Intramolecular and/or intermolecular interactions mediated by all these domains provide a wide framework allowing fine tuning of the multiple regulatory mechanisms and signaling pathways involving the participation of either RasGrf1 or RasGrf2. The balance of the stimulatory/inhibitory interactions received and transmitted through these domains is bound to determine the overall biological outcome concerning the intrinsic GEF activity of RasGrf and its extrinsic contribution to cellular signaling pathways.

The modular domain structures of the RasGrf1 and RasGrf2 proteins are very highly homologous and conserved between both family members (Fig. 4). Up to ten different functional domains or motifs have been described and characterized in the RasGrf proteins. The following sections describe the structural and functional significance of each of these domains in both proteins. Unless otherwise specified, to facilitate functional comparisons the sequence numbers mentioned in the text and figures are those of mouse RasGrf proteins.

#### 3.1.1. PH1 domain

The RasGrfs display two differentiated PH domains in their primary structure. The amino terminal PH domain, designated PH1 appears to be functionally self-sufficient whereas the more distally located PH2 appears to form a functional unit with an adjacent DH domain [63] (Fig. 4). This PH1 domain stretches from amino acid 23 to 130 (Swiss prot, 132) in mRasGrf1 [8] and 23 to 135 in mRasGrf2 [8]. Most studies on the RasGrfs PH domains have been done with the PH1 domain of RasGrf1, although the 50% identity and a 86% homology shown in this region by the RasGrf1 and RasGrf2 PH1 domains [8] suggest a similar set of functions for this particular domain in both RasGrf proteins.

The available experimental evidence suggests a significant contribution of PH1 to the physical localization of RasGrf1 in the plasma membrane, probably through interaction of this domain with the membrane phosphatidylinositol moiety [64–66]. Thus, early transfection experiments in fibroblasts demonstrated that full length RasGrf1 is necessary for serum-induced Ras activation, as transfectants harboring shorter RasGrf1 versions lacking the PH1 domain lost the ability to trigger serum stimulation of Ras [67]. However, membrane association of RasGrf1 constructs lacking the PH1 domain is not totally lost, as some partial association to the particulate fraction is still detectable in those transfectants [63,67].

An important role of the PH1 domain as mediator of the interaction of RasGrf1 with G-protein β/γ subunits has also been proposed [68–70]. This is mostly based on reports of a high binding affinity between Gβ1γ2 subunits and the RasGrf1 PH1 domain [69,70]. Most likely, such an interaction involves the carboxy terminal region of PH1, with its N-terminal portion being responsible for phosphatidylinositol binding [64].

The PH1 domain may also play a role as negative intramolecular modulator of the activity of full length RasGrf1. Thus, it has been proposed that the N-terminal part of RasGrf1 negatively regulates the GEF activity by the catalytic domain “in vitro” [58]. This notion is further supported by the observation that Grfβ, a pancreatic mouse RasGrf1 isoform encompassing only the PH1 domain (Fig. 1) acts as a dominant negative regulator of the process of calcium signaling through RasGrf1 [27]. However, it has also been reported that PH1 deletion from RasGrf1 abolished almost completely the focus forming ability of RasGrf1 constructs transfected into NIH3T3 fibroblasts [67]. In contrast, a separate report described that PH1 elimination did not affect RasGrf1 exchanger activity, Ras GDP release or ERK activation in COS7 cells [71]. These apparent contradictions may be due to using different cell lineages or RasGrf constructs in the reports. One report studied a construct lacking amino acids 25–120 [67] whereas, in the other, the entire N-terminal region, up to amino acid 141, was deleted [71].

In addition to displaying its own, individualized functional roles, there is also significant evidence indicating that the PH1 domain cooperates functionally with neighboring domains for various specific cellular processes. For example, it has been shown that the PH1 domain cooperates with the coiled coil and the IQ domains in the process of Ras activation caused by intracellular calcium movement [72] and in LPA mediated induction of ERK activation in NIH3T3 cells [13]. In agreement with this, it has also been shown that the PH1 domain is required for normal G-protein beta/gamma induced ERK activation, although it is almost totally dispensable for JNK activation in 293 cells [14].

Another potential cellular binding partner interacting with the PH1 domain, possibly in combination with the CC and IQ domains, is IB2/JIP2, a scaffold protein for p38 MAPK. The resulting interaction between RasGrf1, Rac and p38 MAPK has been reported to result in activation of Rac proteins and their specific downstream effectors [73]. The RasGrf1 PH1, in combination with the CC and IQ domains, is also reported to be able to interact with the scaffold protein Spinophilin, although the functional outcome of such an interaction remains to be fully characterized [74]. Indeed, a recent report analyzing the interactome of the RasGrf1 PH1CCIQ module has shown the ability of this region to bind many ribosomal and RNA-binding proteins, pointing towards a still undefined role of RasGrf1 in the control of gene expression [75]. Finally, full length RasGrf1 is also known to be able to form a complex with the NGF receptor TrkA through an interaction involving its amino terminal half portion (at least up to the second PH domain) [76]. In fact, this interaction has been shown to promote RasGrf1 activation through phosphorylation of the PH1 domain by TrkA HIKE domain [77].

#### 3.1.2. Coiled coil domain

This domain stretches in mice from amino acid 151 to 178 (Ensemble database) in RasGrf1. There is 54% identity and an 80% homology between the CC domains of RasGrf1 and RasGrf2 and these domains are also nearly identical between *M. musculus* and *H. sapiens* [8]. It has been reported that the CC domain cooperates with the PH1 and IQ domains in the process of activation of RasGrf1 by intracellular calcium movements and in ERK activation by LPA [13,72] as well as in the physical interaction of RasGrf1 with the scaffold protein IB2/JIP2 [73]. No individual, specialized function has been described yet for the CC domains of RasGrf1/2. CC domains are known to be involved in protein oligomerization in some cases [78,79] but this function has been attributed to the DH domain in the RasGrf proteins [57].

#### 3.1.3. IQ domain

The IQ domains of mRasGrf1 (amino acid 208 to 233, PROSITE Profiles) and mRasGrf2 (residues 202 to 226) [8] are almost identical,

sharing 22 equal residues out of their 25 amino acid stretch. The IQ domains of *M. musculus* to *H. sapiens* are also almost identical.

IQ domains are frequently found in proteins interacting with calmodulin in calcium dependent or independent processes [80]. The original identification of the RasGrf1 IQ domain demonstrated that this domain is indeed responsible for calcium-mediated, calmodulin activation of RasGrf1 as mutations in the IQ domain of RasGrf1 abolish activation of Ras by ionomycin [81]. In addition, the IQ domain is also known to cooperate with other N-terminal domains (CC, PH1) in the process of activation of ERK by LPA and calcium [13,72] and in the interaction of RasGrf1 with the p38 MAPK scaffold IB2/JIP2 [73]. The need of additional proteins for *in vivo* calcium mediated Ras activation may account for the apparently contradictory results found *in vitro*, where calmodulin seems to inhibit RasGrf1 activity rather than inducing it [58].

The structure of the IQ domain in RasGrf2 is very similar to that of RasGrf1. Interestingly, it has been shown that the IQ domain of RasGrf2 is necessary, but not sufficient, for calmodulin binding. Regarding IQ functionality in RasGrf2, a clear dissociation has been found between the process of Ras activation (which is unaffected by IQ domain elimination) and the process of ERK activation, which is seriously impaired in 293T cells transfected with a mutant RasGrf2 construct lacking this domain [82]. This observation might be accounted for by the need to assemble a protein network to couple Ras-GTP to its downstream effectors. An alternative explanation might be a change of subcellular location of RasGrf2 caused by the IQ mutation.

### 3.1.4. DH Domain

The DH domain spans from amino acid 244 to 430 in RasGrf1 (PROSITE Profile) and residues 243 to 459 in RasGrf2 [8] (Fig. 4). This domain shares 87% identity and 99% homology between both RasGrfs. Only two amino acid changes distinguish mRasGrf1 and hRasGrf1, whereas hRasGrf2 and mRasGrf2 differ only in three amino acid residues. The very high evolutionary conservation is indicative of crucial functional roles played by this domain in the context of the functionality of full length RasGrf proteins in the cells.

The functional role of the DH domain appears to be very much intertwined with that of other neighboring domains in the N-terminal portion of the RasGrfs. Thus, an early report analyzing the activity of ectopic, transfected RasGrf1 constructs in NIH3T3 cells [67], showed that elimination of the N-terminal part of the protein (RasGrf1 construct, designated BH1, with N-terminal deletion up to amino acid 425, at the end of the DH domain) (Fig. 4), results in loss of response to serum stimulation, but higher production of transformed foci and higher rates of Ras GDP release (in guanosine nucleotide dissociation assays) than control cells transfected with full length RasGrf1 proteins. In contrast, another report eliminating the DH domain alone (deletion of amino acids 193 to 506) documented complete suppression of the transforming ability of the transfected RasGrf1 construct lacking the complete DH domain [71]. Finally, a separate study analyzing specific point mutations within the DH domain reported that these mutations completely abolished calcium mediated activation of ERK but did not affect RasGrf1 localization, calmodulin binding or Ras GEF activity *in vitro* in 293T cells transfected with the corresponding RasGrf1 construct point mutants [10]. The apparent inconsistent behavior shown by of all these deletion or point mutants affecting the DH domain may be accounted for by the need of physiological interactions between DH and the other N-terminal domains for full *in vivo* responses [10] or by subtle changes of subcellular localization of the RasGrf1 mutants assayed, dependent on the particular mutation under study.

Acting in conjunction with an adjacent PH2 domain, the DH domains are also known to play a defined functional role as the catalytically active domains responsible for facilitating GDP/GTP exchange on members of the Rho family of GTPases (reviewed in [83]). Although early reports failed to detect the ability of the RasGrfs to display GEF activity on Rho family members in addition to Ras family members [10,11], it is now clear that both RasGrf2 and RasGrf1 are able to promote GDP/GTP exchange on Rac proteins through the contribution of their DH domain [12,14]. Interestingly, the GEF activity of the DH domain of RasGrf1 toward Rac can only be observed after G protein beta/gamma induction [14] or after LPA stimulation [13].

Another likely function of the DH domain of the RasGrf is contributing to targeting the RasGrf proteins to specific membrane compartments of the cell [71]. The DH domain is also known to serve as a target for CDC42-GDP induced RasGrf1 inhibition [71]. In a separate report [84] this group has also shown that this domain is necessary for H-Ras activation in the endoplasmic reticulum (ER) but not in the plasma membrane, and that such Ras activation is better stimulated by LPA than by ionomycin [84].

A recent publication has shown that this DH region, in association with its adjacent PH2 domain is able to bind tubulin and the microtubular cytoskeleton [85], in a process coupled to RasGrf-mediated Rac1 activation that is itself inhibited by SCLIP, a protein that destabilizes microtubules and specifically binds to the DH domain of RasGrf1 [86].

The DH domain has also been reported to mediate oligomerization of the RasGrf proteins, resulting in the formation of homo- or hetero-oligomers between RasGrf1 and RasGrf2 proteins that appear to be significant for the activation of specific cellular downstream signaling pathways [57]. The possibility is not ruled out that other cellular proteins may also be involved in the formation of oligomers involving the RasGrf proteins. Dimer formation, mediated through its catalytic domain, has also been previously described for the yeast CDC25 protein [57,87].

### 3.1.5. PH2 Domain

This domain expands from amino acid 459–60 to 588 in both mRasGrf1 and mRasGrf2 [8]. As with other RasGrf domains, the PH2 domain is almost identical between RasGrf1 and RasGrf2 and between human and mouse RasGrfs.

Compared to a RasGrf1 construct lacking the N-terminal region up to the end of the PH1 domain, elimination of the DH and PH2 domains restored focus forming ability back to normal, suggesting a modulatory role of this domain on the overall RasGEF activity of RasGrf1 [67]. It is also reported that the PH2 domain is required for RasGrf1 induction of ERK activity, as mutations in this domain displayed similar deleterious effects than DH mutations on Ras activation, CaM association or subcellular localization of the RasGrf protein [10]. Consistent with this, it has also been shown that the activation of ERK (Ras dependent) and JNK (Rac dependent) induced by LPA treatment is impaired in deletion mutants lacking both DH and PH2 domains [13]. In contrast, other studies have reported that PH2 elimination show little effect on Ras GTPase activity or ERK activation, while still abolishing RasGrf1 focus forming ability in transfections [71]. The conflicting data may be accounted for by the use of different cell lines (293T, Cos7 or NIH3T3) or the comparison between point mutants and deletion mutants in those studies.

Elimination of the PH2 domain of RasGrf2 appears not to affect the resulting Ras activation [82]. The characterization of these PH2 mutants showed maximal ERK activation even in basal conditions, and a lack of activation after ionomycin treatment, suggesting that PH2 plays a role in binding to effectors but not in the catalytic activity “per se”. In agreement with this, the constructs lacking the PH2 domain had a diminished ability to bind Calmodulin [82].

### 3.1.6. ITIM Motif

Two ITIMs (immunoreceptor tyrosine-based inhibition motif) have been described in RasGrf1 (306 IFYQGL and 639 IRYASV) whereas only one is present in RasGrf2 (639 IRYASV) [88]. Because
some ITIMs are known targets of phosphatases such as SHP-1, SHP-2 or SHIP it has been suggested that a possible RasGrf1–SHP-2 interaction might have some potential functional effects [88]. In some cases, ITIMs can mediate activation signals, and the discovery of a role for RasGrf1 and RasGrf2 in T-cell receptor signaling may also suggest a possible functional role of these motifs in the RasGrf proteins [23,24].

### 3.1.7. REM motif

The Ras exchange motif (REM) spans amino acids 639 to 686, both in RasGrf1 and RasGrf2. Together with the IQ motif, it is the most conserved region in this protein family (85% identity and 94% homology). The very high homology existing between yeast CDC25 and the mammalian RasGrfs shows that this region has changed very little during evolution [89]. The REM motif was first described in S. cerevisiae CDC25 as a stretch of 48 amino acids conferring specificity to this protein towards Ras protein activation. Interestingly, this amino acid sequence was highly conserved in both the Sos and RasGrf families of Ras-specific nucleotide exchange factors [89]. Because of this, it was surprising the initial report that a minimal RasGrf construct lacking the REM motif and corresponding to 256 amino acids (1005 to 1260) of the carboxy-terminal region of RasGrf1 was fully able to catalyze GDP/GTP exchange on H-Ras [90]. These data indicated that the REM domain is involved with binding of the RasGrfs to their specific Ras GTPase targets, whereas the other C-terminal domain (CDC25H, see discussion later) constitutes the catalytic site responsible for the nucleotide exchange activity.

### 3.1.8. CDB motif

The Cyclin Destruction Box is a small amino acid region located between the REM motif and the catalytic domain. The CDB motif was initially described for RasGrf2 in a study reporting its resemblance to the CDB found in yeast CDC25 and the similarities and differences to typical type A CDBs [8]. It stretches from residue 743 to 751 in RasGrf2 [8] and has also been identified in the sequence of mouse RasGrf1 (798 to 853) and in rat RasGrf1 (792–836) [58]. Only the first half of the motif is similar between both RasGrfs. The RasGrf2 CDB is a consensus type B CDB, whereas the RasGrf1 CDB resembles more the type A CDBs. RasGrf2 is ubiquitinated in this region, and thus targeted for destruction by the proteasome, in a process that is initiated upon Ras binding and depending on an intact REM motif [59].

### 3.1.9. PEST motif rich region

PEST motifs are described as a proline (P), glutamic acid (E), proline (S) and threonine (T) — enriched regions that target some proteins for proteolysis [91]. Both RasGrf1 and RasGrf2 display a region rich in these amino acids motifs that is located between the REM and CDC25H domains [8,58] (Fig. 4). For RasGrf1, this PEST region has been described as a target for Calpain type proteases, releasing its C-terminal region from inhibition by the N-terminal portion [58]. Another possible function for the PEST region is suggested by reports showing that this portion of the protein is target for PKA-induced phosphorylation and at least Ser822 is necessary for a proper GEF activity [92] or that Ser916 is the phosphorylation site becoming activated after muscarinic receptor stimulation [103]. In addition, the amino terminal part of this region adjacent to the REM motif harbors sites for cyclin dependent kinase 5 phosphorylation of RasGrf2 [62] and RasGrf1 [61] leading, respectively, to inhibition of Rac and Ras activation.

### 3.1.10. The neuronal domain

The ND domain has been only described in RasGrf1 (mRasGrf1 residues 714–913) [94] and appears to be absent from RasGrf2. This domain is necessary and sufficient for interaction of RasGrf1 with the NR2B domain of NMDA receptors resulting in activation of the Ras–ERK pathway [94]. RasGrf1 is unable to interact with NR2A or NR1 subunits,

suggesting that NMDAR channels containing NR2B (but not NR2A or NR1) are responsible for ERK activation in neurons. In contrast, other studies have suggested that RasGrf2 is the RasGEF responsible for ERK activation in the CNS whereas RasGrf1 would activate mainly the p38MAPK [22].

#### 3.1.11. CDC25H domain

The CDC25 homologous domain (CDC25-H) encompasses the C-terminal part of the RasGrf proteins. It was initially identified in RasGrf1 as a stretch of 255 amino acids (from 1005 to 1260) that constituted the minimal, catalytic domain necessary and sufficient for Ras nucleotide exchange activity, the implication being that the rest of the RasGrf1 protein would function to modulate this intrinsic exchange activity in the cellular environment [90]. In mRasGrf2 this domains extends from residue 934 to residue 1189 at the end of the protein. This domain is common to all Ras GEFs able to activate members of the Ras subfamily and contains many conserved residues shared from yeast CDC25 to the mammalian GEFs [95].

Some reports have suggested that the N-terminal half of the RasGrf proteins may exert an inhibitory effect towards the C-terminal part, which in fact shows a higher GEF activity when split from the rest of the protein [58,82]. In contrast, other studies have reported that a shorter version of the RasGrf1 protein [67] or the catalytic domain alone [72] has a similar or even lower basal or ionomycin-induced activity than the full length protein.

The discrepancies between these reports could be due to the use of different biochemical assays and different cellular systems, and could also indicate that there is no direct correlation between the intrinsic in vitro enzymatic activity of purified RasGrf proteins and the in vivo biological activity displayed by them in the context of various biological systems [10,58]. Nevertheless, work with RasGrf2 has shown that both ERK activity and Ras exchange are increased when a construct lacking the N-terminus of the protein is transfected into 293T cells, although this deletion rendered an activation that was independent of upstream stimulation [82]. Despite the controversial aspects of all these reports, they seem to support the notion that the amino part of the RasGrf proteins is responsible for sensing interactions with external stimuli and for activity modulation of the full length proteins.

### 3.2. RasGrf protein 3D structure and oligomerization

Little is known about three-dimensional structure of full length RasGrf proteins, as most studies on this subject have been restricted to the structure and physicochemical properties of their catalytic CDC25H domain [96–98]. In contrast, much more extensive knowledge is available on the structural/functional properties of the Sos family proteins [99–102].

The earlier structural analysis of RasGrf1 involved circular dichroism (CD) studies of its isolated CDC25H domain indicating a content of 42% of alpha helices and a 15% beta sheets in this protein fragment [103]. An alpha helix-rich structure has also been described for the catalytic domain of GEFs of the Sos family [102]. Nuclear magnetic resonance shows also that the CDC25Mm HI hairpin is 85% similar to that of hSos [104]. However, subtle structural homologies and differences existing between the CDC25H domains of Sos1 and RasGrf1 proteins may account, at least in part, for some of the functional specificities of these two distinct GEF families in cellular signaling pathways. Indeed, the alpha-H helix responsible in Sos proteins for interaction with target Ras proteins and nucleotide dissociation activity on them is also present in RasGrf1 proteins and appears to be likewise important for its GEF activity on Ras proteins [97,102]. Nevertheless, further structural studies have also shown that the hairpin including both this H and the following I alpha helices in RasGrf1 is responsible only for nucleotide dissociation, whereas additional RasGrf1 regions are needed for full GEF activity requiringassociation to specific targets of the Ras protein family [96]. This particular function may lay on additional alpha helices located within the catalytic domain, such as the alpha-B and alpha-K helices, which appear to be responsible for specificity towards R-Ras. Thus, even if RasGrf1 readily activates R-Ras, both Sos1 and RasGrf2 are unable to activate this particular small GTPase target. This is probably due to the low similarity existing between the B and K alpha helices of Sos1 and RasGrf1 [105,106] and the change of a few amino acids in the sequence of RasGrf2 corresponding to the RasGrf1 region responsible for recognition of geranylgeranylated R-Ras [106].

Comparison of the crystal structure of Sos1 to that of an isolated RasGrf1 CDC25H domain suggests also that, unlike Sos, the RasGrf domain is able to maintain an active conformation in isolation because its helical hairpin has strengthened interaction with the CDC25 core [97]. These data explains why RasGrf1 lacks the allosteric activation switch that is crucial for Sos activity [97,98] and also may account for previous observations showing that the isolated RasGrf catalytic domain kept or showed increased GEF activity in comparison to full length RasGrf proteins [58,90].

The RasGrf1 and RasGrf2 proteins have also been reported to form homo- or hetero-oligomers through interactions mediated by their DH domains [57]. The oligomeric structure of the RasGrf proteins appears to be significant from the functional point of view, as oligomerization-deficient mutants are unable to induce focus formation or to mediate ionomycin dependent ERK activation in transfected NIH3T3 cells [57].

shown to display dual GEF activity, with the ability to activate both Ras and Rho family members (Table 1).

### 4. Functional aspects of the RasGrfs

#### 4.1. Dual Ras and Rho family targets

RasGrf1 and RasGrf2 were originally described as specific activators for canonical Ras GTPase. The mammalian RasGrf1 was indeed isolated because of its homology to yeast CDC25 and its ability to activate yeast Ras after the loss of CDC25 function in S. cerevisiae [6,7,67]. However, it was immediately obvious that the isolated RasGrf1 cDNA clones contained not only Ras-activating, CDC25-homologous, regions [6] but also a series of other protein domains, like the DH PH tandem responsible for the nucleotide exchange activity of GEFs able to activate Rho family members, such as Dbl or Vav [5]. Consistent with this, later studies have clearly established that RasGrf1 is able to promote GDP/GTP exchange on several members of the Ras family as well as on at least a member of the Rho family. Likewise, RasGrf2, containing the same functional domains than RasGrf1, has also been

#### 4.1.1. Ras subfamily targets

As previously mentioned, RasGrf1 was identified because it is able to complement CDC25 loss and activate Ras in yeast [9]. This isolated mammalian homologue of CDC25 was later shown to be able to promote GDP/GTP exchange in mammalian Ras proteins as well [6]. A number of reports have described that the three members of the Ras subfamily (K-Ras, H-Ras and N-Ras) are targets of RasGrf1 GEF activity in vitro [67]. However, it is not totally clear whether this is also true in in vivo, as other reports suggest that only H-Ras can be activated by RasGrf1 under physiological experimental conditions. These reports indicated that, in NIH3T3 cells transfected with full length RasGrf1, only significant increases of H-Ras-GDP/GTP exchange could be detected whereas the GTP load of N-Ras or K-Ras was not significantly increased [11,107,108]. The hyper-variable domain of these Ras family members was suggested to be responsible for the observed differences, either through targeting of H-Ras to subcellular locations accessible to RasGrf action or through specific recognition of the H Ras-bound lipid moiety by RasGrf1. The latter hypothesis is supported by a separate study documenting that the different prenylation state of H-Ras and R-Ras underlies the specificity of RasGrf1, but not RasGrf2, towards R-Ras [105].

Similar considerations can be made regarding the ability of RasGrf2 to activate Ras subfamily members. Most early studies measured total Ras activation regardless of the isoform, and only some particular reports focused on specific H-Ras activation by RasGrf2 [8,105]. A recent study with Jurkat cells overexpressing RasGrf2 has reported constitutive activation of the three members of the Ras family: H-Ras, N-Ras and K-Ras [23].

#### 4.1.2. R-Ras subfamily targets

The three members of the R-Ras subfamily (R-Ras, TC21/R-Ras2 and M-Ras/R-Ras3) are activatable by RasGrf1. Initial evidence for this ability of the RasGrf1 proteins derived from experiments showing that co-transfected RasGrf1 is able to activate the transforming activity of TC21, but not R-Ras [109] or to induce TC21-mediated transcriptional activation [110]. Later work demonstrated the direct participation of RasGrf1 in GDP/GTP exchange on R-Ras family members by showing that, although RasGrf1 does not activate R-Ras transforming ability, it does activate GDP/GTP exchange on R-Ras at similar or even higher levels than in H-Ras or N-Ras when co-transfected in Cos7 cells [107].

In sharp contrast to RasGrf1, RasGrf2 is unable to activate R-Ras. This appears to be due to sequence differences in a 30 amino acid

---

**Table 1**

Small GTPase targets for RasGrf1 and RasGrf2.

| Ras sub-family | Targets for RasGrf1 | Experimental evidence | Biological outcome of activation | References |
|----------------|--------------------|-----------------------|----------------------------------|------------|
| Ras            | H-Ras             | Both "in vitro" and "in vivo" | Neurite outgrowth, Neuron soma changes, proliferation, LTP and/or LTD | [3,11,48,108,123] |
|                | K-Ras             | Only "in vitro"      | N/D                              | [11,108]   |
|                | N-Ras             | Only "in vitro"      | N/D                              | [108]      |
| R-Ras          | R-Ras             | Both "in vitro" and "in vivo" | Increase in transforming ability | [106,107,112] |
|                | TC21              | Both "in vitro" and "in vivo" | N/D                              | [110,112,114] |
|                | M-Ras             | Both "in vitro" and "in vivo" | N/D                              | [112]      |
| Rho/Rac        | Rac1              | Only "in vivo"       | Neuron soma changes, LTD         | [13,14]    |

| Ras sub-family | Targets for RasGrf2 | Experimental evidence | Biological outcome of activation | References |
|----------------|--------------------|-----------------------|----------------------------------|------------|
| Ras            | H-Ras             | Both "in vitro" and "in vivo" | T-cell proliferation, apoptosis in the lens, cell shape and LTP. | [8,23]    |
|                | K-Ras             | Only "in vivo"       | N/D                              | [23]       |
|                | N-Ras             | Only "in vivo"       | N/D                              | [23]       |
| R-Ras          | TC21              | Only "in vivo"       | N/D                              | [114]      |
| Rho/Rac        | Rac1              | Both "in vitro" and "in vivo" | N/D                              | [12]       |

Summary of GTPase targets identified for the two members of the RasGrf family through various experimental approaches. Targets are classified according to membership in the Ras, R-Ras or Rho subfamilies. See Section 4.1. for details. Reports of in vitro activation of isolated, specific targets are not necessarily coincident with those reporting activation in vivo, due to the possible contribution or requirement of additional cellular components for the biological assays performed. GTPases reported not to be activatable by RasGrf1: Rap1A, Rap1B, Rap2, Ral, Rab3A, RhoA, RhoB, RhoC, RhoG, Cdc42 and Rac1 [10,11,14,107,112]. GTPases reported not to be activatable by RasGrf2: Cdc42, RhoA and geranylgeranylated TC21 [12].
stretch of their catalytic domains which make RasGrf2 unable to recognize the lipid moieties bound to R-Ras [105].

Separate reports have expanded the above observations by further documenting the GEF activity of RasGrf1 proteins onto both TC21 and M-Ras targets [111–113]. Furthermore, a recent report has also shown that both RasGrf1 and RasGrf2 can display its GEF activity towards TC21 in all TC21 subcellular locations but the Golgi complex, and that post-translational modifications affect the specificity of this activation as both RasGrfs are able to activate farnesylated TC21, but only RasGrf1 is able to activate geranylgeranylated TC21. In addition, TC21 activation by the RasGrfs, like H-Ras activation, is negatively controlled by GDP-Cdc42 [114].

### 4.1.3. Rac subfamily targets

After the identification, in proteins such as Vav or Dbl, of the domain responsible for their GEF activity on Rho family members [9], initial studies with RasGrf1 failed to detect the ability of this RasGEF to promote GDP/GTP exchange on Rho family members (RhoA, B, C, G, Rac1, and CDC42) [10,11]. The first direct demonstration of such an activity corresponds to a report showing that transfected RasGrf2 is able to activate Rac1 and its downstream SAPK pathways, but does not show any activity for RhoA or Cdc42 [12]. Shortly after, a later report showed that RasGrf1 is also able to activate Rac1 signaling pathways upon G-protein beta-gamma subunit activation [14].

The most likely explanation for the failing of the initial studies to detect GEF activity towards Rac1 is that those reports involved only *in vitro* assays and, unlike its GEF activity towards Ras family proteins, RasGrf1 appears to need the contribution of other proteins to activate its DH-PH2 mediated GEF activity. Another possibility is that specific upstream activation signals may be needed for RasGrf1 to induce GDP release from Rac1 [14]. Thus, it has been reported that Rac1 can be activated by RasGrf1 after Src-mediated phosphorylation of RasGrf1 [115] or after RasGrf1 interaction with the p38 scaffold IB2/JIP2 or Mapk8ip2, which localizes this GEF in close proximity to its downstream cellular effectors [73]. LPA stimulation has also been shown to activate RasGrf1 GEF activity towards Rac1, thus leading to increased Jun and c-fos promoter activities [13]. Testing of a number of other small GTPases (including Rap1A, Rap1B, Rab3A and Ral) as potential targets for activation by RasGrf1 targets has yielded negative results so far [11,107,112].

### 4.2. Signal transduction mechanisms mediated through the RasGrfs

#### 4.2.1. Activation of the RasGrfs in response to increases in intracellular calcium concentration

Both RasGrf1 and RasGrf2 have the ability to become activated by increases of cytosolic calcium concentration, thus contributing to the integration of calcium and Ras small GTPase signaling pathways [116].

Early work first showed that increased intracellular calcium concentrations resulted in association of calmodulin to the IQ domain of RasGrf1 and subsequent Ras and ERK activation in 293T cells [81]. It was also reported that the amino terminal PH1, CC and IQ domains act cooperatively in this process of calcium-mediated activation of the RasGrf1 GEF activity by facilitating the location of full length RasGrf1 protein to the cellular particulate fraction, where it is able to interact with the appropriate downstream signaling components [72]. Additional analysis has also shown a requirement of the DH and PH2 domains for proper calcium activation of RasGrf1, as mutations in those domains significantly reduce ERK activation induced by ionomycin treatment, while maintaining the ability of the RasGrf1 protein to bind calmodulin and activate Ras in an *in vitro* assay [10]. Surprisingly, a separate report has also shown that the interaction of calmodulin with the purified CDC25H domain of RasGrf1 results in inhibition of its *in vitro* Ras GEF activity. This inhibition is blocked by EGTA, demonstrating that it requires the presence of calcium [58]. The apparent contradiction with the former reports may be explained by

the “*in vitro*” nature of the assays reported here, which implies the absence of other cellular factors that may be important for proper RasGrf1 activation by calcium. In agreement with this, a recent study has shown that the stimulation of RasGrf1 by ionomycin is dependent on Cdc42, and that this calcium-mediated activation of RasGrf1 is not achieved equally at all possible RasGrf1 subcellular locations. Thus, ionomycin preferentially activates RasGrf1 at the plasma membrane, and much lower activation level is achieved at the endoplasmic reticulum [84]. The differences in activation may be due to the different scaffolding proteins that RasGrf1 can interact with in its different possible subcellular locations [117–120].

Calcium-dependent activation of RasGrf1 is known to be triggered in neuronal physiological environments after stimulation of the N-methyl-D-aspartate (NMDA) receptor (Fig. 5). The activation process requires interaction of RasGrf1 with NMDAR NR2B subunits, although the mechanistic details and functional outcome of this interaction/activation remain unclear. Initial work using cultured primary neurons showed that blocking the RasGrf1/NR2B interaction (using a NR2B dominant negative mutant or blocking peptides) abolished ERK activation after NMDAR stimulation with NMDA or bicuculline [94]. Consistently, work in mouse hippocampal slices and cultured neurons showed that NMDAR stimulation was able to induce both RasGrf1 phosphorylation and ERK activation in a signaling event requiring both calmodulin kinase kinase and calmodulin kinase I activities [121] (Fig. 5). Somewhat surprisingly, a separate study using brain hippocampal slices from RasGrf1 knockout mice has also reported that RasGrf1 is responsible for p38 MAPKinase activation, rather than ERK activation, upon NMDAR stimulation [22,122]. The mechanistic panorama is further completed by functional studies suggesting a link between RasGrf1 downregulation and improved LTP responses in the brain cortical region of NR2B transgenic mice [44] or between RasGrf1, Rac-p38MAPK and long term depression (LTD) at the hippocampus of RasGrf1 KO animals [22].

Other neurotransmitter or G-protein coupled receptors can also promote increased cytosolic calcium concentration and subsequent activation of RasGrf signaling upon ligand-induced stimulation. Thus, the α-amino 3-hydroxy-5-methyl-4-isoxazolepropionic acid (AMPA) receptor has been reported to be able to activate both RasGrf1 and RasGrf2 mediated signaling in cortical neurons of post-pubescent mice only in a calcium dependent manner [123]. Likewise, the G-protein coupled receptor for serotonin appears to require calcium/calmodulin binding to RasGrf1 in order to induce ERK1/2 phosphorylation after stimulation with its agonist. It has been reported that serotonin stimulation of HEK 293 cells overexpressing the Gs-coupled 5-HT7 receptor causes Ras activation in a process requiring PKA phosphorylation of RasGrf1 as well as increased intracellular calcium levels and Calcium/Calmodulin interaction with RasGrf1 [124].

Calcium dependent signals can also be negatively regulated through the action of RasGrf proteins. For example, the previously described, small RasGrfβ isoform has been functionally characterized as a naturally occurring, dominant negative modulator of calcium and Ras signaling in pancreatic β cells [27]. Furthermore, in CHO cells calcium activation of m-calpain proteases has also been reported to result in the downregulation of intracellular RasGrf1 protein levels. Thus, induction of phosphorylation of RasGrf1 on Ser731 by p35/CDK5 increases the sensitivity of RasGrf1 to calpain degradation. RasGrf1 phosphorylation and degradation by this mechanism leads to nuclear condensation and disorganization, thus defining an important role of RasGrf1 in control of neuronal nuclear organization [61].

The IQ domain of RasGrf2 also mediates its interaction with calmodulin, thus leading to the activation of RasGrf2 by upstream calcium signals and the subsequent stimulation of downstream Ras pathways [8]. Interestingly, a RasGrf2 construct lacking the IQ domain was still able to activate Ras-GTP formation suggesting that the IQ-calmodulin interaction is not necessary for the intrinsic RasGrf2 GEF activity but is indispensable for downstream ERK activation [82].

A. Fernández-Medarde, E. Santos / Biochimica et Biophysica Acta 1815 (2011) 170–188

Fig. 5. RasGrf-mediated integration of Ca<sup>++</sup> signals initiated by NMDA receptor activation. NMDAR activation upon glutamate binding originates an increase in the intracellular levels of calcium and subsequent downstream intracellular signaling events involving the RasGrfs. See Section 4.2.1 for details. RasGrf1 has been shown to bind directly to NMDAR-NR2B subunits, and a similar interaction (?**) is postulated for RasGrf2 with NR2A subunits based on indirect evidence. Functional activation of the RasGrfs is mediated through the action of CAM, which is previously activated by a calcium-dependent kinase cascade involving calmodulin kinase kinase (CamKK) and calmodulin kinase I (CaMKI). It has been proposed that CaMKI (?*) may also be able to activate RasGrf1 directly. Once activated, both RasGrfs are able to activate independent Rac1-p38MAPK and Ras-ERK downstream signaling cascades. In the case of RasGrf1, these downstream signaling pathways lead to activation of LTD responses in the hippocampus and possibly inhibition (?*) of LTP responses in the cortex. In contrast, RasGrf2 activation leads to activation of LTP responses in the hippocampus.

These data suggest that, in addition to directly activating Ras, RasGrf2 acts by promoting the assembly of downstream protein networks linking the activated GTPase to specific downstream effectors. The calcium/calmodulin interaction with the RasGrf2 IQ domain has also been reported to promote Rac1 and JNK activation, although the mechanistic details of this activation and its possible crosstalk with the Ras-ERK pathway remain to be totally clarified [12]. A recent report has described that calcium-induced Ras activation by RasGrf2 can be inhibited by αβ-crystallin in rat lens epithelial cells [125].

As for RasGrf1, RasGrf2 can be also activated upon ligand stimulation of the NMDAR (Fig. 5). Direct experimental evidence for physical interaction

is limited, but a mechanism has been proposed by which RasGrf2 interacts with the NR2A subunits of the receptor leading, after agonist stimulation, to activation of ERK and Long Term Potentiation in the hippocampus [22,94,122]. The respective interactions of RasGrf1 and RasGrf2 with the NR2a and NR2b subunits may provide a first level of specificity by coupling these GEFs to different calcium channels. A second level of downstream specificity may be provided by specific protein scaffolds determining the interaction of the activated RasGrf with specific downstream effectors. For example, the interaction of RasGrf1 with Spinophilin or IB2/JIP2, a scaffold for p38 MAP Kinase, may account for the preferential activation of p38 MAP Kinase by RasGrf1 upon NMDAR activation [73,74].

Fig. 6. RasGrf activation by G-protein coupled receptors. RasGrf1 is activated upon ligand stimulation of various heterotrimeric G-Protein coupled Receptors (GPCR), in a process involving interaction with split Gβγ subunits. See Section 4.2.2 for mechanistic details. For RasGrf1 activation, both serine and tyrosine phosphorylation events are needed. Intracellular second messengers such as cAMP or calcium are indispensable for full activation of RasGrf1 nucleotide exchange activity. Phosphorylation by different tyrosine kinases appears to activate different downstream signaling pathways. Thus, Src phosphorylation results in activation of Rac1 and downstream signaling pathways, whereas ACK stimulates the Ras/ERK pathway. P(Y), phosphotyrosine. P(S), phosphoserine.

A. Fernández-Medarde, E. Santos / Biochimica et Biophysica Acta 1815 (2011) 170–188

4.2.2. Activation of the RasGrfs in response to G-protein coupled receptors (GPCR)

Extracellular signals specifically interacting with transmembrane receptors coupled to heterotrimeric G proteins can activate different second messengers and channels to initiate a large variety of intracellular signaling pathways. Ligand-receptor stimulation triggers the split of the heterotrimeric G complexes into Gα-GTP and Gβγ subunits, which are then available for further specific interactions with downstream signaling components [126,127] (Fig. 6).

The Gβγ subunits have long been known to interact with PH domain containing signaling proteins such as PLCγ, spectrin or IRS [68]. Consistently, early in vitro studies showed that the N-terminal PH domain of RasGrf proteins can also interact with Gβγ proteins in a similar fashion and that a small portion of the PH1 domain of RasGrf1 (amino acids 55–134) is sufficient for such interaction in vitro [70]. Initial in vivo evidence for GPCR-mediated activation of RasGrf1 arose from studies showing that serum stimulation of NIH3T3 cells results in enhanced RasGrf1 GEF activity, accompanied by RasGrf1 protein hyperphosphorylation. This effect was inhibited by pre-treatment with pertussis toxin (PT) but not with genistein, suggesting that GPCRs, but not RTKs, are involved in serum mediated activation of RasGrf1 [128]. This notion was further confirmed by studies showing that treatment of CHO cells with lysophosphatidic acid (LPA) or serum, but not PDGF, triggered activation of RasGrf1 that was also PT-sensitive [129]. Overexpression of subtype 1 human Muscarinic receptor in NIH3T3 or COS-7 cells also leads to RasGrf phosphorylation and increased GEF activity upon carbachol stimulation. This activation was constitutive when G-protein βγ subunits were overexpressed and was prevented by treatment with phosphatases or cotransfection of G-protein α-subunit [130]. Additional work showing that LPA-induced phosphorylation involves serine residues in RasGrf1 and that calcium depletion inhibits LPA-mediated RasGrf1 activation suggested a model in which RasGrf1 may act integrating signals from various upstream messengers [131] (Fig. 6).

RasGrf1 may become phosphorylated at Ser916 after cellular stimulation by a great variety of external stimuli. Thus, in addition to carbachol or forskolin, NGF stimulation of endogenous Trk receptors also strongly induced RasGrf1 Ser916 phosphorylation in COS-7 and PC12 cultured cells as well as in rat forebrain slices and cortex [132]. Furthermore, serotonin stimulation of HEK293 cells over-expressing 5-HT4 receptors has also been reported to trigger RasGrf1 phosphorylation at Serine 916 and activation of its GEF activity [124,133].

A variety of different mechanisms are likely involved in RasGrf1 phosphorylation and activation induced by various external stimuli. Indeed, the mechanistic details and the physiological effects of such phosphorylation events are unclear and remain largely undefined in different biological contexts. For instance, although both cAMP and carbachol trigger RasGrf1 phosphorylation on the same Ser916, the cAMP-dependent phosphorylation is mediated by PKA, whereas carbachol-dependent phosphorylation involves an unknown kinase [93]. Forskolin stimulation of NIH3T3 cells expressing transfected RasGrf1 does not activate its GEF activity [93], and mutational changes in the amino acids normally phosphorylated by PKA had little or no effect on this GEF activity, either [92]. In transfected HEK293 cells, both cAMP and calcium/calmodulin signals are necessary to achieve PKA-mediated RasGrf1 phosphorylation and complete ERK activation after stimulation of surface serotonin receptors [124,133]. Analysis of differentiating PC12 cells has also shown a direct correlation between RasGrf1 phosphorylation, H-Ras expression and neurite outgrowth [132]. Likewise, in vivo neurological studies with primary cultures and hippocampal slices have shown direct correlation between RasGrf1 Ser916 phosphorylation and rises in ERK activation upon NMDA treatment [121].

Despite early failings to detect RasGrf1 GEF activity towards Rho family members, studies of overexpression of Gβγ subunits in 293 cells have clearly demonstrated an induction of RasGrf1 GEF activity

towards Rac1 [14]. Later independent studies have also shown that the Rho catalytic domain (DH-PH2) of RasGrf1 can mediate transduction of cellular signals going from external stimulation with LPA signals to activation of intracellular JNK and the c-fos promoter, in a process that can be blocked with a Rac DN construct [13]. Interestingly, further work also demonstrated that RasGrf1 needs to be tyrosine phosphorylated in order to activate its Rac1 GEF activity [14]. Non receptor tyrosine kinases, such as Src, are responsible for tyrosine phosphorylation of the RasGrf1 proteins [115] (Fig. 6).

Although RasGrf1 and RasGrf2 are very close related, most studies on the activation of the RasGrfs by various GPCRs have been done with RasGrf1. A recent study on II9 cells (chinese hamster embryo fibroblasts) has reported N-Ras activation and translocation of RasGrf2 to the plasma membrane after α-thrombin stimulation, indicating that α-thrombin and Gβ₁γ₅ subunits activate Ras through RasGrf2 in this cellular context [134]. Further experimental work will be needed to clarify the specific mechanistic details and physiological consequences of the potential activation of RasGrf2 by GPCRs in different biological contexts.

4.2.3. Activation of the RasGrfs in response to receptor and non-receptor tyrosine kinases

Although most published reports discard the activation of RasGrf1 by cellular RTKs [128], some early studies clearly showed that RasGrf1 is able to interact directly with the intracellular domain of the Trk family of tyrosine kinase receptors for nerve growth factor (NGF) [76,77] (Fig. 7). Specifically, the interaction between the N-terminal region of RasGrf1 and the HIKE domain of TrkA leads to tyrosine phosphorylation of RasGrf1 in its PH1 domain in a process where the transfected, ectopically expressed RasGrf1 potentiates the differentiation potential of NGF to induce neurite extensions in PC12 cells [77]. The process of neurite extension caused by RasGrf1 overexpression requires H-Ras and ERK activation but is independent of Rac1 or PI3K pathways [135]. On the other hand, it has also been reported that RasGrf1 coordinates both H-Ras and Rac1 pathways, in a PI3K/AKT dependent manner, to induce soma expanded morphology in PC12 cells that may be related to proper LTP achievement in a more physiological environment [135]. Somehow contradictory behavior can be observed in different cellular contexts. Thus, in HEK293 cells, the direct interaction between RasGrf1 and SCLIP has been reported to be associated with Rac1/p38 signaling and over-expression of both proteins in PC12 cells leads to the inhibition of the neurite outgrowth [86].

A number of non receptor TKs have also been reported to phosphorylate cellular RasGrf proteins. As previously mentioned, the Src tyrosine kinase has been reported to mediate GPCR signaling in vivo by phosphorylating RasGrf1 and activating its GEF activity towards Rac1 [115], although Src has also been reported to inhibit Ras signaling by phosphorylating p120GAP [136]. In vitro assays have also shown the ability of Lck, a separate member of the Src family, to phosphorylate RasGrf1 and activate its Ras GEF activity [136], but further experimental evidence is needed to demonstrate an in vivo role of this kinase in physiological contexts. The Ras GEF activity of RasGrf1 is also enhanced upon phosphorylation by ACK1, a non receptor tyrosine kinase activated by Cdc42, suggesting a link between this Rho family kinase and Ras signaling [137]. The crosstalk between Ras and Cdc42 is supported by reports showing that RasGrf1 activation of the Ras–MAPK pathway is inhibited by GDP-Cdc42, but not by GTP-Cdc42 [71,138]. A similar mechanism may be applicable for RasGrf2 activation, since the expression of a dominant negative Cdc42 construct (Cdc42N17) abolished RasGrf2 recruitment to the plasma membrane, Ras activation and ERK phosphorylation [139].

As most studies have focused on RasGrf1, there is very little data related to the possible activation of RasGrf2 by RTKs or non receptor tyrosine kinases. Nevertheless, both RasGrf2 and RasGrf1 have been reported to be activated upon T-Cell Receptor stimulation, in a process

Fig. 7. RasGrf activation by receptor tyrosine kinases. The TrkA, B and C members of this tyrosine kinase family of NGF receptors are able to interact with, and activate RasGrf1 in a process involving tyrosine phosphorylation of the PH1 domain. See Section 4.2.3. for details. Upon NGF treatment, RasGrf1 has been shown to activate Ras- and Rac-dependent signaling pathways leading to differentiation (neurite extension) and changes in morphology and size of the neuronal soma. Whereas Ras downstream signaling is sufficient for differentiation, both Ras and Rac downstream signals are required to induce the morphological changes and increased cell size.

likely requiring the tyrosine kinase activity of a Src family member. This activation engages, at least in the case of RasGrf2, Ras-dependent and PLC-γ1-mediated signaling pathways producing the activation of NF-AT, a transcriptional factor crucial for T-cell activation and differentiation [23].

### 4.3. Functional roles of the RasGrfs at the cellular level

The display of the functional activity of the RasGrf family members in different biological contexts has potential for triggering activation of different members of the Ras, R-Ras or Rac subfamilies, through a variety of specific signaling mechanisms which may themselves translate into different biological outcomes in each case. At the cellular level, RasGrf activation has been linked to diverse cellular functions including control of cell growth, neurite outgrowth/differentiation, cell internal organization, neuronal excitability or even malignant transformation.

#### 4.3.1. The RasGrfs and cell growth

The initial identification of the mammalian RasGrf1 gene was in fact based on its ability to overcome the loss of the yeast CDC25 locus, thus allowing or restoring yeast cell growth [6]. Furthermore, overexpression of RasGrf1 was shown to induce morphological transformation and increased cellular growth in transfected mouse fibroblasts [67,140]. Membrane targeting of the transfected RasGrf1 constructs significantly increased growth of the transformed NIH3T3 fibroblasts [141]. The increased growing ability of the transfected mouse fibroblasts appears to require integrity of the Ras and Rac activating domains of RasGrf1, thus demonstrating the need for crosstalk between different signaling pathways for display of full, *in vivo* biological outcome in mouse fibroblasts [71]. Independent reports have also shown the ability of a 75 kDa RasGrf1 isoform to promote anchorage-independent growth in Rat1A cells [26].

It should be noted that the above mentioned reports concern a biological outcome of RasGrf1 in rodent fibroblasts, a cellular context where it is not naturally expressed. In its more natural environment, the Central Nervous System, RasGrf1 is expressed only after birth, when cell proliferation is scarce. As Sos proteins are strongly expressed before birth, they are the more likely GEFs involved in physiologically driving embryonic neuronal proliferation events [48]. RasGrf2 has also been linked to control of cell proliferation in immunological contexts, where it has been shown to participate in T cell signaling responses. T-cells lacking RasGrf2 and Vav1 showed reduced proliferation in comparison to Vav1 KO or WT T-cells upon TCR stimulation [23].

#### 4.3.2. The RasGrfs and differentiation

The ability of RasGrf1 to promote neuronal differentiation, evaluated by measuring neurite outgrowth, has been widely analyzed. An initial report showed that ectopic expression of RasGrf1 in a SK-N-BE neuroblastoma cell line increased the rate of neurite outgrowth upon retinoic acid induction [142]. Likewise, RasGrf1 overexpression alone was not sufficient to induce differentiation of PC12 cells but, upon NGF stimulation, RasGrf1 becomes phosphorylated and neurite outgrowth was substantially increased in this cell line [77,132,135]. Direct evidence supporting a mechanistic link between RasGrf1 phosphorylation and neurite extension was provided by studies showing that mutations in the HIKE domain of TrkA led to unpaired phosphorylation and interaction of RasGrf1 with the receptor, resulting in suppression of the neurite outgrowth [77]. Other studies indicate that H-Ras over-expression alone cannot induce neurite outgrowth in PC12 cells, but its co-expression with RasGrf1 strongly induced neurite growth in a process that seemed to require RasGrf1 phosphorylation on Ser916 [132]. Furthermore, the effect of RasGrf1 on neuritogenesis appears to be independent of its Rac1 GEF activity, as deletion mutants lacking the DHPH domain retained their ability to induce neurite outgrowth [132,135]. In summary, RasGrf1 is able to favor neuronal differentiation in the presence of a neurotrophic stimulus or when co-expressed with its downstream target H-Ras, in a Rac1- and PI3K-independent manner.

A separate report indicates that the microtubule destabilizing protein SCLIP is able to inhibit RasGrf1 induced neurite outgrowth in PC12 cells. In particular, SCLIP was reported to block RasGrf1-induced differentiation mediated through activation of Rac1 but not of H-Ras [86]. As RasGrf1 is also able to interact with microtubules [85], an interesting speculation is that the mechanism of the blocking effect of SCLIP on RasGrf1-induced neuritogenesis may be more related to the functional link between RasGrf1 and the cytoskeleton than to its intrinsic GEF activity [86].

#### 4.3.3. RasGrf effects on cellular morphology and organization

RasGrf1 has also been linked to other changes in neuronal morphology. In addition to neurite extension, expression of co-transfected RasGrf1 and H-Ras in PC12 cells produced a striking increase in soma size of a significant percentage (25%) of the cell population. The mechanism for this morphological change required H-Ras expression

but was independent of its GTP loading and, in contrast to the process of neurite outgrowth, was dependent on the contribution of Rac1, PI3K and AKT [135].

Regulation of cellular RasGrf1 levels also appears to be essential for proper nuclear organization in neurons. It has been reported that the absence of the cellular mechanisms leading to p35/Cdk5 phosphorylation and subsequent degradation of RasGrf1 by the proteasome results in nuclear condensation and cellular death in rat cortical neurons and Cos7 cells [61]. RasGrf1 phosphorylation by p35/Cdk5 causes changes in Ras, but not Rac1, activation, but surprisingly affects AKT, and not ERK activation [61]. Although the mechanistic details are unclear, these observations suggest that nuclear organization in neurons may be regulated by RasGrf1/Ras/AKT signaling pathways and under p35/Cdk5 control.

RasGrf2 can also affect cell morphology. It has been reported that the overexpression of transfected RasGrf2 in HEK293 cells results in significant alterations of cell morphology [8]. The possibility of affecting the tubulin cytoskeleton and cellular shape is also raised by studies in rat cortical neurons showing that RasGrf2 phosphorylation by p35/Cdk5 produces accumulation of the cytoskeleton related protein MAP1b [62].

### 4.3.4. RasGrfs and neuronal excitability and responses

The prevalent expression of RasGrf1 in neuronal environments may account for additional cellular functions attributed to this protein. A role of RasGrf1 in neuronal processes such as excitability and synaptic plasticity is suggested by the localization of this protein in synaptic junctions [143] and the hyper-excitability observed in RasGrf1 deficient neurons, probably due to changes in their membrane ionic balance [144]. The RasGrf1 and RasGrf2 proteins are also known to exert a neuroprotective role during experimental ischemic episodes, as the absence of both RasGrfs is associated to increased rates of cell death during those incidents [48]. A role of RasGrf1 in neuroprotection can also be observed after haloperidol treatment, which induces dissociation of RasGrf from the NMDAR complexes, leading to JNK activation and increased apoptotic cell death [145]. RasGrf1 may also play a role in modulation of cellular responses to various drugs. RasGrf1 was reported to be responsible for cannabinoid CB1 receptor adaptation and drug tolerance after sub-chronical treatment of mice with Δ-9-tetrahydrocannabinol (THC) in cerebellum and striatum [146,147], and its elimination suppressed changes in cerebellar synaptic plasticity after chronic THC treatment [148]. RasGrf1 has also been implicated in cocaine responses; as RasGrf1 null animals showed reduced locomotor sensitization and conditioned place preference upon cocaine treatment [149].

The two members of the RasGrf family have also been reported to play complementing, contrasting roles regarding synaptic plasticity, based on studies suggesting a preferential link of RasGrf2 to regulation of long-term potentiation (LTP) and of RasGrf1 to regulation of long-term depression (LTD) (Fig. 5). Thus, in the hippocampus, RasGrf1 has been shown to interact functionally with NMDAR type NR2B receptors, thus leading to Rac-p38MA Pkinase activation and subsequent promotion of long-term depression (LTD). In contrast, the interaction of RasGrf2 with NMDAR type NR2A subunits in the same location has been described to induce activation of Ras–ERK signaling leading to promotion of long-term potentiation (LTP) [94,150].

An additional cellular function of the RasGrf1 proteins may be related to their ability to control expression of matrix metalloproteinases (MMP) relevant for cellular invasion and metastasis. Recent reports demonstrate a role of RasGrf1 in controlling the expression levels of MMP9 in a human melanoma cell line [56], and of MMP1 and MMP3 in fibroblast-like synoviocytes from rheumatoid arthritis patients [151].

A specific role of RasGrf2 mediating cellular apoptosis in the ocular lens has been also suggested by a report showing that RasGrf2-mediated activation of the Ras/ERK pathway leads to increases in intracellular p53

levels and Caspase-3 activation in N/N1003A rabbit lens epithelial cells [125].

#### 4.3.5. The RasGrfs and carcinogenesis

Direct experimental evidence supporting a potential link between RasGrf1 and tumor development is limited or almost absent. RasGrf1 expression has been reported in some tumor cell lines such as M2 human melanoma cells where, in addition, constitutive activation of the RAS/ERK pathway through a mechanism involving RasGrf1 has also been described [56]. A recent report has also described erasure of differentially methylated regions at the RasGrf1 imprinted locus of murine very small embryonic/epiblast like stem cells (VSELs) and it was proposed that this could potentially lead to tumor formation by these cells [152].

In contrast, the evidence suggesting a direct role of RasGrf2 in carcinogenesis is much more conclusive and extensive. A number of reports have described reduced levels of RasGrf2 expression in a variety of human and rodent tumor systems (of pancreatic, mammary, colon or lung origin), frequently in association with aberrant methylation of the RasGrf2 genomic locus [45–47,153]. Consistent with this, a wide screening for consensus coding sequences present in solid human breast and colorectal tumors described the detection of somatic missense mutations in the RasGrf2 locus in about 7% of human breast cancers [154]. In hematopoietic systems, a synergistic role of RasGrf2 and Vav1 gene deficiencies has been reported in processes of lymphomagenesis and metastasis using knockout mouse models [24]. RasGrf2 is known to be constitutively expressed in embryonic stem cells [8] and further work is needed to ascertain a potential role of RasGrf2 as a tumor suppressor in tumors where its expression is reduced or blocked.

### 4.4. Functional roles of the RasGrfs at the organism level. RasGrf1 and RasGrf2 animal models

Phenotypic characterization of genetically modified mouse strains for the RasGrf1 and RasGrf2 loci has been crucial for advancing the understanding of their respective “in vivo” functions at the organ or full body level.

Regarding RasGrf1, analysis of the various knockout mice strains described to date has defined two main groups of phenotypes allowing the identification of specific functional roles for this particular GEF. Consistent with its pattern of expression at the Central Nervous System, a first group of phenotypes initially identified defects related to processes of learning and memory consolidation [15,16]. Other reports have also described impaired light perception abilities in KO mice, suggesting a functional role of RasGrf1 in the retina, a tissue of neuroectodermal origin [20]. A second, distinct group of unrelated phenotypes has also uncovered functions played by RasGrf1 at organ locations outside the Central Nervous System. In particular, the functional relevance of RasGrf1 for pancreatic beta cells accounts, at least in part, for the growth retardation and altered glucose homeostasis observed in RasGrf1 deficient mice [17–19,155].

Regarding RasGrf2, the analysis of available, single and combined knockout strains for this particular locus has mostly identified the functional contribution of RasGrf2 to proliferation of lymphocytic cell lineages and T cell signaling responses, as well as to processes of lymphomagenesis and metastasis [23,24].

#### 4.4.1. Neural RasGrf1 phenotypes

Analysis of specific knockout mice strains has clearly revealed that RasGrf1 plays a significant role in memory and learning processes. An initial report described that the absence of RasGrf1 results in impairment of amygdala-dependent learning as a result of long term potentiation (LTP) defects, whereas hippocampus-based learning was unaffected [15]. In contrast to this, separate reports using a different knockout mouse strain have described defective long term depression
(LTD) and impairment of hippocampus-dependent learning in the defective mice [16,22]. The discrepancy between these reports may be due to the use of different gene targeting strategies or the different genetic backgrounds of the host mice [15,16]. The detection of defective retinal photoreception in RasGrf1 null animals identifies an additional functional role of the RasGrf1 proteins [20]. It is not unreasonable to speculate that this defective visual phenotype may have also contributed, at least in part, to the negative outcome of the conduct tests initially used to evaluate memory and learning defects in these animals.

Characterization of RasGrf1/RasGrf2 knockouts has also found that these animals exhibit higher sensitivity to cerebral ischemia than their WT counterparts, suggesting an important role for both RasGrfs in the mechanisms underlying neuroprotection during ischemia. This phenotype was only observed in double-knockout mice suggesting an overlapping function for both RasGrfs in neuroprotection [48].

The functional contribution of RasGrf1 to drug tolerance mechanisms is also suggested by the observation of reduced tolerance to chronically administered Δ9-tetrahydrocannabinoid, or cocaine that was observed in RasGrf1 knockout mice [146,147,149,156]. Alterations of basal cannabinoid receptor- and cAMP-mediated signaling have also been described in the cerebellum of RasGrf1-KO mice, although hippocampus and prefrontal cortex signaling appeared unaffected [146,148].

Transcriptional characterization of the hippocampus of RasGrf1 knockout mice using microarrays has identified a pattern of gene expression consistent with previously described memory and learning phenotypes that includes differentially expressed genes which may be relevant to Alzheimer and neurodegenerative diseases and suggest a possible participation of RasGrf1 in such pathological processes [157]. Separate studies have also described a defective retinal vision phenotype in RasGrf1 knockout mice. Electroretinographic analysis of these mice showed strong defects in light perception that worsened progressively with age, suggesting a functional contribution of RasGrf1 signaling to mammalian photoreception process. Transcriptional characterization of isolated retinas by means of microarrays revealed patterns of differential gene expression consistent with this notion [20]. Interestingly, RasGrf1 function may also be connected to other unrelated vision defects. Consistent with a recent GWAS study identifying RasGrf1 as part of a susceptibility locus associated to myopia and refractive error in humans, specific crystalline lens alterations have also been observed in RasGrf1 KO mice [21]. These observations suggest a potential usefulness of the RasGrf1 knockout mice as animal models for the study of defective vision syndromes in humans.

### 4.4.2. Non neural RasGrf1 phenotypes

The second set of phenotypes observed in studies of knockout animals concerns the important role played by RasGrf1 in control of postnatal growth. Indeed, adult mice devoid of RasGrf1 are 15–25% smaller than their WT counterparts [16–18]. These early observations constituted the first description of an imprinted gene affecting postnatal growth, rather than prenatal growth. Initial characterization studies reported lower levels of growth hormone in the pituitaries of RasGrf1-deficient mice suggesting a role for RasGrf1 signaling in the hypothalamus and a defect of growth hormone releasing hormone as a possible cause of the problem [18,155].

In addition to reduced body size, the characterization of a separate RasGrf1 null strain revealed glucose intolerance, reduced levels of circulating plasma insulin and reduced pancreatic β-cell mass in the knockout animals, supporting a significant role of RasGrf1 in control of β-cell proliferation and neogenesis [19]. Interestingly, there is a remarkable similarity between the growth/pancreatic phenotypes described for RasGrf1 knockout mice and those independently reported for mice deficient for different signaling molecules including Pttg1 [158], S6K1 [159,160] or Cdk4 [161], suggesting that all these

signaling molecules and RasGrf1 may share participation in the same signaling pathway(s).

### 4.4.3. RasGrf2 phenotypes

Genomic disruption of the RasGrf2 locus showed that this GEF is dispensable for mouse development and postnatal growth. The RasGrf2 null mice showed similar size, growth rate, lifespan and fertility as their wild-type control animals [17]. Analysis of an independently generated RasGrf2-knockout strain confirmed the absence of manifestly impairing phenotypes although deeper analysis of the signaling pathways in which RasGrf2 participates, pointed to lower levels of NMDA-induced ERK activation in the brains of the knockout animals [22].

The study of knockout mice harboring combinations of a disrupted RasGrf2 locus with null mutations for other GEFs, such as RasGrf1 or Vav1, has yielded further insights into the functionality of RasGrf2. Analysis of the brains of adult RasGrf1/RasGrf2 knockout mice showed a significant increase of neuronal damage in response to ischemia than the corresponding single knockouts or WT controls, suggesting an overlapping, protective role for both RasGrfs in stroke-induced neurotoxicity, probably through a reduction of CREB phosphorylation and activity [48]. Similarly, detailed functional analysis of RasGrf2/Vav1 null mice has demonstrated that RasGrf2 is an additional component of the signaling machinery involved in T-cell receptor- and NF-AT-mediated immune responses. Furthermore, the deficiency of the RasGrf2 and Vav genes exerts also a synergistic effect on processes of lymphomagenesis and metastasis going on in these knockout animals [23,24].

## Acknowledgments

This work was supported by grants from ISCIII (FIS PS09/01979) and JCyL (SA044A08 and GR93) as well as institutional support from RTICC (RD06/0020/000), ISCIII, Spain.

## References

[1] D. Broek, T. Toda, T. Michaelli, L. Levin, C. Birchmeier, M. Zoller, S. Powers, M. Wigler, The S. cerevisiae CDC25 gene product regulates the RAS/adenylate cyclase pathway, Cell 48 (1987) 789–799.

[2] L.C. Robinson, J.B. Gibbs, M.S. Marshall, I.S. Sigal, K. Tatchell, CDC25: a component of the RAS–adenylate cyclase pathway in Saccharomyces cerevisiae, Science 235 (1987) 1218–1221.

[3] A. Wolfman, I.G. Macara, A cytosolic protein catalyzes the release of GDP from p21ras, Science 248 (1990) 67–69.

[4] Y.K. Huang, H.F. Kung, T. Kamata, Purification of a factor capable of stimulating the guanine nucleotide exchange reaction of ras proteins and its effect on ras-related small molecular mass G proteins, Proc. Natl Acad. Sci. USA 87 (1990) 8008–8012.

[5] E. Martegani, M. Vanoni, R. Zippel, P. Coccetti, R. Brambilla, C. Ferrari, E. Sturani, L. Alberghina, Cloning by functional complementation of a mouse cDNA encoding a homologue of CDC25, a Saccharomyces cerevisiae RAS activator, EMBO J. 11 (1992) 2151–2157.

[6] W. Wei, R.D. Mosteller, P. Sanyal, E. Gonzales, D. McKinney, C. Dasgupta, P. Li, B.X. Liu, D. Broek, Identification of a mammalian gene structurally and functionally related to the CDC25 gene of Saccharomyces cerevisiae, Proc. Natl Acad. Sci. USA 89 (1992) 7100–7104.

[7] L. Chen, L.J. Zhang, P. Greer, P.S. Tung, M.F. Moran, A murine CDC25/ras-GRF-related protein implicated in Ras regulation, Dev. Genet. 14 (1993) 339–346.

[8] N.P. Fam, W.-T. Fan, Z. Wang, L.-J. Zhang, H. Chen, M.F. Moran, Cloning and characterization of Ras-GRF2, a novel guanine nucleotide exchange factor for Ras, Mol. Cell. Biol. 17 (1997) 1396–1406.

[9] H. Cen, A.G. Papageorge, R. Zippel, D.R. Lowy, K. Zhang, Isolation of multiple mouse cDNAs with coding homology to Saccharomyces cerevisiae CDC25: identification of a region related to Bcr, Vav, Dbl and CDC24, EMBO J. 11 (1992) 4007–4015.

[10] N.W. Freshney, S.D. Goonesekera, L.A. Feig, Activation of the exchange factor Ras-GRF by calcium requires an intact Dbl homolgy domain, FEBS Lett. 407 (1997) 111–115.

[11] S. Orita, K. Kaibuchi, S. Kuroda, K. Shimizu, H. Nakanishi, Y. Takai, Comparison of kinetic properties between two mammalian ras p21 GDP/GTP exchange proteins, ras guanine nucleotide-releasing factor and smg GDP dissociation stimulation, J. Biol. Chem. 268 (1993) 25542–25546.

[12] W. Fan, C.A. Koch, C.L. de Hoog, N.P. Fam, M.F. Moran, The exchange factor Ras-GRF2 activates Ras-dependent and Rac-dependent mitogen-activated protein kinase pathways, Curr. Biol. 8 (1998) 935–938.

[13] M. Innocenti, R. Zippel, R. Brambilla, E. Sturani, CDC25(Mm)/Ras-GRF1 regulates both Ras and Rac signaling pathways, FEBS Lett. 460 (1999) 357–362.
[14] M. Kiyono, T. Satoh, Y. Kaziro, G protein beta gamma subunit-dependent Rac-guanine nucleotide exchange activity of Ras-GRF1/CDC25(Mm), Proc. Natl Acad. Sci. USA 96 (1999) 4826–4831.

[15] R. Brambilla, N. Gnesutta, L. Minichiello, G. White, A.J. Roylance, C.E. Herron, M. Ramsey, D.P. Wolfer, V. Cestari, C. Rossi-Arnaud, S.G.N. Grant, P.F. Chapman, H.P. Lipp, E. Sturani, R. Klein, A role for the Ras signaling pathway in synaptic transmission and long-term memory, Nature 390 (1997) 281–286.

[16] K.P. Giese, E. Friedman, J.B. Telliez, N.B. Fedorov, M. Wines, L.A. Feig, A.J. Silva, Hippocampus-dependent learning and memory is impaired in mice lacking the Ras-guanine-nucleotide releasing factor 1 (Ras-GRF1), Neuropharmacology 41 (2001) 791–800.

[17] A. Fernandez-Medarde, L.M. Esteban, A. Nunez, A. Porteros, L. Tessarollo, E. Santos, Targeted disruption of Ras-Grf2 shows its dispensability for mouse growth and development, Mol. Cell. Biol. 22 (2002) 2498–2504.

[18] J.-M. Itier, G.L. Trempe, J.-F. Leonard, M.C. Multon, G. Ret, F. Schweighoffer, B. Tocque, M.-T. Bluet-Pajot, V. Cormier, F. Dautry, Imprinted gene in postnatal growth role, Nature 393 (1998) 125–126.

[19] J. Font de Mora, L.M. Esteban, D.J. Burks, A. Nunez, C. Garces, M.J. Garcia-Barrado, M.C. Iglesias-Osma, J. Moratinos, J.M. Ward, E. Santos, Ras-GRF1 signaling is required for normal beta-cell development and glucose homeostasis, EMBO J. 22 (2003) 3039–3049.

[20] A. Fernandez-Medarde, R. Barhoum, R. Riquelme, A. Porteros, A. Nunez, A. de Luis, J. de Las Rivas, P. de la Villa, I. Varela-Nieto, E. Santos, RasGRF1 disruption causes retinal photoreception defects and associated transcriptomic alterations, J. Neurochem. 110 (2009) 641–652.

[21] P.G. Hysi, T.L. Young, D.A. Mackey, T. Andrew, A. Fernández-Medarde, A.M. Solouki, A.W. Hewitt, S. Macgregor, J.R. Vingerling, Y. Li, M.K. Ikram, L.Y. Fai, P.C. Sham, L. Manyes, A. Porteros, M.C. Lopes, F. Carbonaro, S.J. Fahy, N.G. Martin, C.M. van Duyn, T.D. Spector, J.S. Rahi, E. Santos, C.C.W. Klaver, C.J. Hammond, A genome-wide association study for myopia and refractive error identifies a susceptibility locus at 15q25, Nat. Genet. 42 (10) (2010) 902–905.

[22] S. Li, X. Tian, D.M. Hartley, L.A. Feig, Distinct roles for Ras-guanine nucleotide-releasing factor 1 (Ras-GRF1) and Ras-GRF2 in the induction of long-term potentiation and long-term depression, J. Neurosci. 26 (2006) 1721–1729.

[23] S. Ruiz, E. Santos, X.R. Bustelo, RasGRF2, a guanosine nucleotide exchange factor for Ras GTPases, participates in T-cell signaling responses, Mol. Cell. Biol. 27 (2007) 8127–8142.

[24] S. Ruiz, E. Santos, X.R. Bustelo, The use of knockout mice reveals a synergistic role of the Vav1 and Rasgrf2 gene deficiencies in lymphomagenesis and metastasis, PLoS ONE 4 (2009) e8229.

[25] A. de la Puente, J. Hall, Y.Z. Wu, G. Leone, J. Peters, B.J. Yoon, P. Soloway, C. Plass, Structural characterization of Rasgrf1 and a novel linked imprinted locus, Gene 291 (2002) 287–297.

[26] V.D. Leaner, H. Donninger, C.A. Ellis, G.J. Clark, M.J. Birrer, p75-Ras-GRF1 is a c-Jun/AP-1 target protein: it's up regulation results in increased Ras activity and is necessary for c-Jun-induced nonadherent growth of Rat1a cells, Mol. Cell. Biol. 25 (2005) 3324–3337.

[27] Y. Arava, R. Seger, M.D. Walker, GRFbeta, a novel regulator of calcium signaling, is expressed in pancreatic beta cells and brain, J. Biol. Chem. 274 (1999) 24449–24452.

[28] F. Schweighoffer, M. Faure, I. Fath, M.C. Chevallier-Multon, F. Apiou, B. Dutrillaux, E. Sturani, M. Jacquet, B. Tocque, Identification of a human guanine nucleotide-releasing factor (H-GRF55) specific for Ras proteins, Oncogene 8 (1993) 1477–1485.

[29] C. Shou, C.L. Farnsworth, B.G. Neel, L.A. Feig, Molecular cloning of cDNAs encoding a guanine-nucleotide-releasing factor for Ras p21, Nature 358 (1992) 351–354.

[30] N.P. Fam, L.-J. Zhang, J.M. Rommens, B.G. Beatty, M.F. Moran, Mapping of the Ras-GRF2 gene (GRF2) to mouse chromosome 13 C3-D1 and human chromosome 5q13, near the Ras-GAP gene, Genomics 39 (1997) 118–120.

[31] M. Kaneda, M. Okano, K. Hata, T. Sado, N. Tsujimoto, E. Li, H. Sasaki, Essential role for de novo DNA methyltransferase Dnmt3a in paternal and maternal imprinting, Nature 429 (2004) 900–903.

[32] C. Plass, H. Shibata, I. Kalcheva, L. Mullins, N. Kotelevtseva, J. Mullins, R. Kato, H. Sasaki, S. Hirotsune, Y. Okazaki, W.A. Held, Y. Hayashizaki, V.M. Chapman, Identification of Grf1 on mouse chromosome 9 as an imprinted gene by RLGS-M, Nat. Genet. 14 (1996) 106–109.

[33] J.Y. Li, D.J. Lees-Murdock, G.L. Xu, C.P. Walsh, Timing of establishment of paternal methylation imprints in the mouse, Genomics 84 (2004) 952–960.

[34] H. Shibata, Y. Yoda, R. Kato, T. Ueda, M. Kamiya, N. Hiraiwa, A. Yoshiki, C. Plass, R.S. Pearsall, W.A. Held, M. Muramatsu, H. Sasaki, M. Kusakabe, Y. Hayashizaki, A methylation imprint mark in the mouse imprinted gene Grf1/Cdc25Mm locus shares a common feature with the U2afbp-rs gene: an association with a short tandem repeat and a hypermethylated region, Genomics 49 (1998) 30–37.

[35] R. Holmes, Y. Chang, P.D. Soloway, Timing and sequence requirements defined for embryonic maintenance of imprinted DNA methylation at Rasgrf1, Mol. Cell. Biol. 26 (2006) 9564–9570.

[36] B.J. Yoon, H. Herman, A. Sikora, L.T. Smith, C. Plass, P.D. Soloway, Regulation of DNA methylation of Rasgrf1, Nat. Genet. 30 (2002) 92–96.

[37] B. Yoon, H. Herman, B. Hu, Y.J. Park, A. Lindroth, A. Bell, A.G. West, Y. Chang, A. Stablewski, J.C. Piel, D.I. Loukinov, V.V. Lobanenko, P.D. Soloway, Rasgrf1 imprinting is regulated by a CTCF-dependent methylation-sensitive enhancer blocker, Mol. Cell. Biol. 25 (2005) 11184–11190.

[38] A.M. Lindroth, Y.J. Park, C.M. McLean, G.A. Dokshin, J.M. Persson, H. Herman, D. Pasini, X. Miro, M.E. Donohoe, J.T. Lee, K. Helin, P.D. Soloway, Antagonism between DNA and H3K27 methylation at the imprinted Rasgrf1 locus, PLoS Genet. 4 (2008) e1000145.

[39] G. Colombo, F. Rusconi, T. Rubino, A. Cattaneo, E. Martegani, D. Parolaro, A. Bachì, R. Zippel, Transcriptomic and proteomic analyses of mouse cerebellum reveals alterations in RasGRF1 expression following in vivo chronic treatment with delta 9-tetrahydrocannabinol, J. Mol. Neurosci. 37 (2009) 111–122.

[40] N.K. Parelkar, Q. Jiang, X.P. Chu, M.L. Guo, L.M. Mao, J.Q. Wang, Amphetamine alters Ras-guanine nucleotide-releasing factor expression in the rat striatum in vivo, Eur. J. Pharmacol. 619 (2009) 50–56.

[41] G.C. Zhang, J. Hoffmann, N.K. Parelkar, X.Y. Liu, L.M. Mao, E.E. Fibuch, J.Q. Wang, Cocaine increases Ras-guanine nucleotide-releasing factor 1 protein expression in the rat striatum in vivo, Neurosci. Lett. 427 (2007) 117–121.

[42] T.D. Stein, J.A. Johnson, Lack of neurodegeneration in transgenic mice overexpressing mutant amyloid precursor protein is associated with increased levels of transthyretin and the activation of cell survival pathways, J. Neurosci. 22 (2002) 7380–7388.

[43] M.D. Landis, D.D. Seachrist, F.W. Abdul-Karim, R.A. Keri, Sustained trophism of the mammary gland is sufficient to accelerate and synchronize development of ErbB2/Neu-induced tumors, Oncogene 25 (2006) 3325–3334.

[44] F. Gu, J. Shi, Y. Wen, H. Fan, J. Hu, Y. Hu, Z. Zhao, Translational responses of NR2B overexpression in the cerebral cortex of transgenic mice: a liquid chromatography-based proteomic approach, Brain Res. 1250 (2009) 1–13.

[45] H. Chen, M. Suzuki, Y. Nakamura, M. Ohira, S. Ando, T. Iida, T. Nakajima, A. Nakagawara, H. Kimura, Aberrant methylation of RASGRF2 and RASSF1A in human non-small cell lung cancer, Oncol. Rep. 15 (2006) 1281–1285.

[46] A. Hagihara, K. Miyamoto, J. Furuta, N. Hiraoka, K. Wakazono, S. Seki, S. Fukushima, M.S. Tsao, T. Sugimura, T. Ushijima, Identification of 27 5′ CpG islands aberrantly methylated and 13 genes silenced in human pancreatic cancers, Oncogene 23 (2004) 8705–8710.

[47] F.V. Jacinto, E. Ballestar, S. Ropero, M. Esteller, Discovery of epigenetically silenced genes by methylated DNA immunoprecipitation in colon cancer cells, Cancer Res. 67 (2007) 11481–11486.

[48] X. Tian, T. Gotoh, K. Tsuji, E.H. Lo, S. Huang, L.A. Feig, Developmentally regulated role for Ras-GRFs in coupling NMDA glutamate receptors to Ras, Erk and CREB, EMBO J. 23 (2004) 1567–1575.

[49] D.M. Thomas, D.M. Francescutti-Verbeem, X. Liu, D.M. Kuhn, Identification of differentially regulated transcripts in mouse striatum following methamphetamine treatment—an oligonucleotide microarray approach, J. Neurochem. 88 (2004) 380–393.

[50] C. Ferrari, R. Zippel, E. Martegani, N. Gnesutta, V. Carrera, E. Sturani, Expression of two different products of CDC25 Mm, a mammalian Ras activator, during development of mouse brain, Exp. Cell Res. 210 (1994) 353–357.

[51] C. Guerrero, J.M. Rojas, M. Chedid, L.M. Esteban, D.B. Zimonjic, N. Popescu, J. Font de Mora, E. Santos, Expression of alternative forms of Ras exchange factors GRF and SOS1 in different human tissues and cell lines, Oncogene 12 (1996) 1097–1107.

[52] E. Gross, I. Marbach, D. Engelberg, M. Segal, G. Simchen, A. Levitzki, Anti-Cdc25 antibodies inhibit guanyl nucleotide-dependent adenyl cyclase of Saccharomyces cerevisiae and cross-react with a 150-kilodalton mammalian protein, Mol. Cell. Biol. 12 (1992) 2653–2661.

[53] W. Wei, S.S. Schreiber, M. Baudry, G. Tocco, D. Broek, Localization of the cellular expression pattern of cdc25NEF and Ras in the juvenile rat brain, Brain Res. Mol. Brain Res. 19 (1993) 339–344.

[54] R. Zippel, N. Gnesutta, N. Matus-Leibovitch, E. Mancinelli, D. Saya, Z. Vogel, E. Sturani, Ras-GRF, the activator of Ras, is expressed preferentially in mature neurons of the Central Nervous System, Brain Res. Mol. Brain Res. 48 (1997) 140–144.

[55] P.S. Tung, N.P. Fam, L. Chen, M.F. Moran, A 54-kDa protein related to ras-guanine nucleotide release factor expressed in the rat exocrine pancreas, Cell Tissue Res. 289 (1997) 505–515.

[56] T.N. Zhu, H.J. He, S. Kole, T. D'Souza, R. Agarwal, P.J. Morin, M. Bernier, Filamin A-mediated down-regulation of the exchange factor Ras-GRF1 correlates with decreased matrix metalloproteinase-9 expression in human melanoma cells, J. Biol. Chem. 282 (2007) 14816–14826.

[57] P.H. Anborgh, X. Qian, A.G. Papageorge, W.C. Vass, J.E. DeClue, D.R. Lowy, Ras-specific exchange factor GRF: oligomerization through its dbl homology domain and calcium-dependent activation of Raf, Mol. Cell. Biol. 19 (1999) 4611–4622.

[58] S. Baouz, E. Jacquet, A. Bernardi, A. Parmeggiani, The N-terminal moiety of CDC25 (Mm), a GDP/GTP exchange factor of Ras proteins, controls the activity of the catalytic domain. Modulation by calmodulin and calpain, J. Biol. Chem. 272 (1997) 6671–6676.

[59] C.L. de Hoog, J.A. Koehler, M.D. Goldstein, P. Taylor, D. Figeys, M.F. Moran, Ras binding triggers ubiquitination for the Ras exchange factor Ras-GRF2, Mol. Cell. Biol. 21 (2001) 2107–2117.

[60] N. Gnesutta, M. Ceriani, M. Innocenti, I. Mauri, R. Zippel, E. Sturani, B. Borgonovo, G. Berruti, E. Martegani, Cloning and characterization of mouse UBPy, a deubiquitinating enzyme that interacts with the ras guanine nucleotide exchange factor CDC25(Mm)/Ras-GRF1, J. Biol. Chem. 276 (2001) 39448–39454.

[61] S. Kesavapany, T.K. Pareek, Y.L. Zheng, N. Amin, J.S. Gutkind, W. Ma, A.B. Kulkarni, P. Grant, H.C. Pant, Neuronal nuclear organization is controlled by cyclin-dependent kinase 5 phosphorylation of Ras guanine nucleotide releasing factor-1, Neurosignals 15 (2006) 157–173.

[62] S. Kesavapany, N. Amin, Y.L. Zheng, R. Nijhara, H. Jaffe, J.S. Gutkind, S. Takahashi, A. Kulkarni, P. Grant, H.C. Pant, p35/cyclin-dependent kinase 5 phosphorylation of ras guanine nucleotide releasing factor 2 (RasGRF2) mediates Rac-dependent extracellular signal-regulated kinase 1/2 activity, altering RasGRF2 and microtubule-associated protein 1b distribution in neurons, J. Neurosci. 24 (2004) 4421–4431.

[63] N. Blomberg, M. Nilges, Functional diversity of PH domains: an exhaustive modelling study, Fold. Des. 2 (1997) 343–355.

[64] J.E. Harlan, P.J. Hajduk, H.S. Yoon, S.W. Fesik, Pleckstrin homology domains bind to phosphatidylinositol-4, 5-bisphosphate, Nature 371 (1994) 168–170.

[65] M.A. Lemmon, K.M. Ferguson, Signal-dependent membrane targeting by pleckstrin homology (PH) domains, Biochem. J. 350 (Pt 1) (2000) 1–18.

[66] T. Maffucci, M. Falasca, Specificity in pleckstrin homology (PH) domain membrane targeting: a role for a phosphoinositide-protein co-operative mechanism, FEBS Lett. 506 (2001) 173–179.

[67] H. Cen, A.G. Papageorge, W.C. Vass, K.E. Zhang, D.R. Lowy, Regulated and constitutive activity by CDC25Mm (GRF), a Ras-specific exchange factor, Mol. Cell. Biol. 13 (1993) 7718–7724.

[68] J. Inglese, J. Koch, K. Touhara, R.J. Lefkowitz, Gβγ interactions with PH domains and Ras–MAPK signaling pathways, FEBS Lett. 460 (1995) 357–362.

[69] T. Sawai, T. Hirakawa, K. Yamada, Y. Nishizawa, Interaction between Pleckstrin homology domains and G protein betagamma-subunits: analyses of kinetic parameters by a biosensor-based method, Biol. Pharm. Bull. 22 (1999) 229–233.

[70] K. Touhara, J. Inglese, J.A. Pitcher, G. Shaw, R.J. Lefkowitz, Binding of G protein beta gamma-subunits to pleckstrin homology domains, J. Biol. Chem. 269 (1994) 10217–10220.

[71] I. Arozarena, D.S. Aaronson, D. Matallanas, V. Sanz, N. Ajenjo, S.P. Tenbaum, H. Teramoto, T. Ighiishi, J.C. Zabala, J.S. Gutkind, P. Crespo, The Rho family GTPase Cdc42 regulates the activation of Ras/MAP kinase by the exchange factor Ras-GRF, J. Biol. Chem. 275 (2000) 26441–26448.

[72] BuchsbaumR., TelliezJ.-B., GoonesekeraS., FeigL.A., The N-terminal pleckstrin, coiled-coil, and IQ domains of the exchange factor Ras-GRF act cooperatively to facilitate activation by calcium, Mol. Cell. Biol. 16 (1996) 4888–4896.

[73] R.J. Buchsbaum, B.A. Connolly, L.A. Feig, Interaction of Rac exchange factors Tiam1 and Ras-GRF1 with a scaffold for the p38 mitogen-activated protein kinase cascade, Mol. Cell. Biol. 22 (2002) 4073–4085.

[74] R.J. Buchsbaum, B.A. Connolly, L.A. Feig, Regulation of p70 S6 kinase by complex formation between the Rac guanine nucleotide exchange factor (Rac-GEF) Tiam1 and the scaffold spinophilin, J. Biol. Chem. 278 (2003) 18833–18841.

[75] P. Lavagni, M. Indrigo, G. Colombo, E. Martegani, K. Rosenblum, N. Gnesutta, R. Zippel, Identification of novel RasGRF1 interacting partners by large-scale proteomic analysis, J. Mol. Neurosci. 37 (2009) 212–224.

[76] J.I. MacDonald, J.M. Verdi, S.O. Meakin, Activity-dependent interaction of the intracellular domain of rat trkA with intermediate filament proteins, the beta-6 proteasomal subunit, Ras-GRF1, and the p162 subunit of eIF3, J. Mol. Neurosci. 13 (1999) 141–158.

[77] K.N. Robinson, K. Manto, R.J. Buchsbaum, J.I. MacDonald, S.O. Meakin, Neurotrophin-dependent tyrosine phosphorylation of Ras guanine-releasing factor 1 and associated neurite outgrowth is dependent on the HIKE domain of TrkA, J. Biol. Chem. 280 (2005) 225–235.

[78] J.R. McWhirter, D.L. Galasso, J.Y. Wang, A coiled-coil oligomerization domain of Bcr is essential for the transforming function of Bcr–Abl oncoproteins, Mol. Cell. Biol. 13 (1993) 7587–7595.

[79] S.R. Whitson, W.M. LeStourgeon, A.M. Krezel, Solution structure of the symmetric coiled coil tetramer formed by the oligomerization domain of hnRNP C: implications for biological function, J. Mol. Biol. 350 (2005) 319–337.

[80] M. Bahler, A. Rhoads, Calmodulin signaling via the IQ motif, FEBS Lett. 513 (2002) 107–113.

[81] C.L. Farnsworth, N.W. Freshney, L.B. Rosen, A. Ghosh, M.E. Greenberg, L.A. Feig, Calcium activation of Ras mediated by neuronal exchange factor Ras-GRF, Nature 376 (1995) 524–527.

[82] C.L. de Hoog, W.-T. Fan, M.D. Goldstein, M.F. Moran, C.A. Koch, Calmodulin-independent coordination of Ras and extracellular signal-regulated kinase activation by Ras-GRF2, Mol. Cell. Biol. 20 (2000) 2727–2733.

[83] K.L. Rossman, C.J. Der, J. Sondek, GEF means go: turning on RHO GTPases with guanine nucleotide-exchange factors, Nat. Rev. Mol. Cell Biol. 6 (2005) 167–180.

[84] I. Arozarena, D. Matallanas, M.T. Berciano, V. Sanz-Moreno, F. Calvo, M.T. Munoz, G. Egea, M. Lafarga, P. Crespo, Activation of H-Ras in the endoplasmic reticulum by the RasGRF family guanine nucleotide exchange factors, Mol. Cell. Biol. 24 (2004) 1516–1530.

[85] G. Forlani, S. Baldassa, P. Lavagni, E. Sturani, R. Zippel, The guanine nucleotide exchange factor RasGRF1 directly binds microtubules via DHPH2-mediated interaction, FEBS J. 273 (2006) 2127–2138.

[86] S. Baldassa, N. Gnesutta, U. Fascio, E. Sturani, R. Zippel, SCLIP, a microtubule-destabilizing factor, interacts with RasGRF1 and inhibits its ability to promote Rac activation and neurite outgrowth, J. Biol. Chem. 282 (2007) 2333–2345.

[87] C. Camus, M. Geymonat, H. Garreau, S. Baudet-Nessler, M. Jacquet, Dimerization of Cdc25p, the guanine-nucleotide exchange factor for Ras from Saccharomyces cerevisiae, and its interaction with Sdc25p, Eur. J. Biochem. FEBS 247 (1997) 703–708.

[88] N.R. Sinclair, Immunoreceptor tyrosine-based inhibitory motifs on activating molecules, Crit. Rev. Immunol. 20 (2000) 89–102.

[89] C.-C. Lai, M. Boguski, D. Broek, S. Powers, Influence of guanine nucleotides on complex formation between Ras and CDC25 proteins, Mol. Cell. Biol. 13 (1993) 1345–1352.

[90] P. Coccetti, I. Mauri, L. Alberghina, E. Martegani, A. Parmeggiani, The minimal active domain of the mouse ras exchange factor CDC25Mm, Biochem. Biophys. Res. Commun. 206 (1995) 253–259.

[91] S. Rogers, R. Wells, M. Rechsteiner, Amino acid sequences common to rapidly degraded proteins: the PEST hypothesis, Science 234 (1986) 364–368.

[92] S. Baouz, E. Jacquet, K. Accorsi, C. Hountondji, M. Balestrini, R. Zippel, E. Sturani, A. Parmeggiani, Sites of phosphorylation by protein kinase A in CDC25Mm/GRF1, a guanine nucleotide exchange factor for Ras, J. Biol. Chem. 276 (2001) 1742–1749.

[93] R.R. Mattingly, Phosphorylation of serine 916 of Ras-GRF1 contributes to the activation of exchange factor activity by muscarinic receptors, J. Biol. Chem. 274 (1999) 37379–37384.

[94] G. Krapivinsky, L. Krapivinsky, Y. Manasian, A. Ivanov, R. Tyzio, C. Pellegrino, Y. Ben-Ari, D.E. Clapham, I. Medina, The NMDA receptor is coupled to the ERK pathway by a direct interaction between NR2B and RasGRF1, Neuron 40 (2003) 775–784.

[95] L.A. Quilliam, J.F. Rebhun, A.F. Castro, A growing family of guanine nucleotide exchange factors is responsible for activation of Ras-family GTPases, Prog. Nucleic Acid Res. Mol. Biol. 71 (2002) 391–444.

[96] E. Sacco, S. Fantinato, R. Manzoni, D. Metalli, L. De Gioia, P. Fantucci, L. Alberghina, M. Vanoni, The isolated catalytic hairpin of the Ras-specific guanine nucleotide exchange factor Cdc25Mm retains nucleotide dissociation activity but has impaired nucleotide exchange activity, FEBS Lett. 579 (2005) 6851–6858.

[97] T.S. Freedman, H. Sondermann, G.D. Friedland, T. Kortemme, D. Bar-Sagi, S. Marqusee, J. Kuriyan, A Ras-induced conformational switch in the Ras activator Son of sevenless, Proc. Natl Acad. Sci. USA 103 (2006) 16692–16697.

[98] T.S. Freedman, H. Sondermann, O. Kuchment, G.D. Friedland, T. Kortemme, J. Kuriyan, Differences in flexibility underlie functional differences in the Ras activators son of sevenless and Ras guanine nucleotide releasing factor 1, Structure 17 (2009) 41–53.

[99] S.M. Soisson, A.S. Nimnuan, M. Uy, D. Bar-Sagi, J. Kuriyan, Crystal structure of the Dbl and pleckstrin homology domains from the human Son of sevenless protein, Cell 95 (1998) 259–268.

[100] H. Sondermann, S.M. Soisson, D. Bar-Sagi, J. Kuriyan, Tandem histone folds in the structure of the N-terminal segment of the ras activator Son of Sevenless, Structure 11 (2003) 1583–1593.

[101] H. Sondermann, S.M. Soisson, S. Boykevisch, S.S. Yang, D. Bar-Sagi, J. Kuriyan, Structural analysis of autoinhibition in the Ras activator Son of sevenless, Cell 119 (2004) 393–405.

[102] P.A. Boriack-Sjodin, S.M. Margarit, D. Bar-Sagi, J. Kuriyan, The structural basis of the activation of Ras by Sos, Nature 394 (1998) 337–343.

[103] P. Coccetti, E. Monzani, L. Alberghina, L. Casella, E. Martegani, Analysis of the secondary structure of the catalytic domain of mouse Ras exchange factor CDC25Mm, Biochim. Biophys. Acta 1383 (1998) 292–300.

[104] R. Consonni, I. Arosio, T. Recca, R. Longhi, G. Colombo, M. Vanoni, Structure determination and dynamics of peptides overlapping the catalytic hairpin of the Ras-specific GEF Cdc25(Mm), Biochemistry 42 (2003) 12154–12162.

[105] T. Gotoh, X. Tian, L.A. Feig, Prenylation of target GTPases contributes to signaling specificity of Ras-guanine nucleotide exchange factors, J. Biol. Chem. 276 (2001) 38029–38035.

[106] X. Tian, L.A. Feig, Basis for signaling specificity difference between Sos and Ras-GRF guanine nucleotide exchange factors, J. Biol. Chem. 276 (2001) 47248–47256.

[107] T. Gotoh, Y. Niino, M. Tokuda, O. Hatases, S. Nakamura, M. Matsuda, S. Hattori, Activation of R-Ras by Ras-guanine nucleotide-releasing factor, J. Biol. Chem. 272 (1997) 18602–18607.

[108] JonesM.K., JacksonJ.H., Ras-GRF activates Ha-Ras, but not N-Ras or K-Ras 4B, protein in vivo, J. Biol. Chem. 273 (1998) 1782–1787.

[109] A.F. Overbeck, T.R. Brtva, A.D. Cox, S.M. Graham, S.Y. Huffs, R. Khosravi-Far, L.A. Quilliam, P.A. Solski, C.J. Der, Guanine nucleotide exchange factors: activators of Ras superfamily proteins, Mol. Reprod. Dev. 42 (1995) 468–476.

[110] S.M. Graham, A.B. Vojtek, S.Y. Huff, A.D. Cox, G.J. Clark, J.A. Cooper, C.J. Der, TC21 causes transformation by Raf-independent signaling pathways, Mol. Cell. Biol. 16 (1996) 6132–6140.

[111] N. Movilla, P. Crespo, X.R. Bustelo, Signal transduction elements of TC21, an oncogenic member of the R-Ras subfamily of GTP-binding proteins, Oncogene 18 (1999) 5860–5869.

[112] Y. Ohba, N. Mochizuki, S. Yamashita, A.M. Chan, J.W. Schrader, S. Hattori, K. Nagashima, MatsudaM., Regulatory proteins of R-Ras, TC21/R-Ras2, and M-Ras/ R-Ras3, J. Biol. Chem. 275 (2000) 20020–20026.

[113] L.A. Quilliam, A.F. Castro, K.S. Rogers-Graham, C.B. Martin, C.J. Der, C. Bi, M-Ras/R- Ras3, a transforming Ras protein regulated by Sos1, GRF1, and p120 Ras GTPase-activating protein, interacts with the putative Ras effector AF6, J. Biol. Chem. 274 (1999) 23850–23857.

[114] F. Calvo, P. Crespo, Structural and spatial determinants regulating TC21 activation by RasGRF family nucleotide exchange factors, Mol. Biol. Cell 20 (2009) 4289–4302.

[115] M. Kiyono, Y. Kaziro, T. Satoh, Induction of rac-guanine nucleotide exchange activity of Ras-GRF1/CDC25(Mm) following phosphorylation by the nonreceptor tyrosine kinase Src, J. Biol. Chem. 275 (2000) 5441–5446.

[116] P.J. Cullen, P.J. Lockyer, Integration of calcium and Ras signalling, Nat. Rev. Mol. Cell Biol. 3 (2002) 339–348.

[117] B. Casar, I. Arozarena, V. Sanz-Moreno, A. Pinto, L. Agudo-Ibanez, R. Marais, R.E. Lewis, M.T. Berciano, P. Crespo, Ras subcellular localization defines extracellular signal-regulated kinase 1 and 2 substrate specificity through distinct utilization of scaffold proteins, Mol. Cell. Biol. 29 (2009) 1338–1353.

[118] A. Levchenko, J. Bruck, P.W. Sternberg, Scaffold proteins may biphasically affect the levels of mitogen-activated protein kinase signaling and reduce its threshold properties, Proc. Natl Acad. Sci. USA 97 (2000) 5818–5823.

[119] M.J. Marinissen, J.S. Gutkind, Scaffold proteins dictate Rho GTPase-signaling specificity, Trends Biochem. Sci. 30 (2005) 423–426.

[120] A.T. Sim, J.D. Scott, Targeting of PKA, PKC and protein phosphatases to cellular microdomains, Cell Calcium 26 (1999) 209–217.

[121] J.M. Schmitt, E.S. Guire, T. Saneyoshi, T.R. Soderling, Calmodulin-dependent kinase kinase/calmodulin kinase I activity gates extracellular-regulated kinase-dependent long-term potentiation, J. Neurosci. 25 (2005) 1281–1290.

[122] S.X. Jin, L.A. Feig, Long-term potentiation in the CA1 hippocampus induced by NR2A subunit-containing NMDA glutamate receptors is mediated by Ras-GRF2/ Erk map kinase signaling, PLoS ONE 5 (2010) e11732.

[123] X. Tian, L.A. Feig, Age-dependent participation of Ras-GRF proteins in coupling calcium-permeable AMPA glutamate receptors to Ras/Erk signaling in cortical neurons, J. Biol. Chem. 281 (2006) 7578–7582.

[124] J.H. Norum, T. Methi, R.R. Mattingly, F.O. Levy, Endogenous expression and protein kinase A-dependent phosphorylation of the guanine nucleotide exchange factor Ras-GRF1 in human embryonic kidney 293 cells, FEBS J. 272 (2005) 2304–2316.

[125] D.W. Li, J.P. Liu, Y.W. Mao, H. Xiang, J. Wang, W.Y. Ma, Z. Dong, H.M. Pike, R.E. Brown, J.C. Reed, Calcium-activated RAF/MEK/ERK signaling pathway mediates p53-dependent apoptosis and is abrogated by alpha B-crystallin through inhibition of RAS activation, Mol. Biol. Cell 16 (2005) 4437–4453.

[126] M. Bhattacharya, A.V. Babwah, S.S. Ferguson, Small GTP-binding protein-coupled receptors, Biochem. Soc. Trans. 32 (2004) 1040–1044.

[127] S.L. Ritter, R.A. Hall, Fine-tuning of GPCR activity by receptor-interacting proteins, Nat. Rev. Mol. Cell Biol. 10 (2009) 819–830.

[128] C. Shou, A. Wurmser, K. Ling, M. Barbacid, L.A. Feig, Differential response of the Ras exchange factor, Ras-GRF to tyrosine kinase and G protein mediated signals, Oncogene 10 (1995) 1887–1893.

[129] R. Zippel, S. Orecchia, E. Sturani, E. Martegani, The brain specific Ras exchange factor CDC25 Mm: modulation of its activity through Gi-protein-mediated signals, Oncogene 12 (1996) 2697–2703.

[130] R.R. Mattingly, I.G. Macara, Phosphorylation-dependent activation of the Ras-GRF/CDC25Mm exchange factor by muscarinic receptors and G-protein beta gamma subunits, Nature 382 (1996) 268–272.

[131] R.R. Mattingly, V. Saini, I.G. Macara, Activation of the Ras-GRF/CDC25Mm exchange factor by lysophosphatidic acid, Cell. Signal. 11 (1999) 603–610.

[132] H. Yang, D. Cooley, J.E. Legakis, Q. Ge, R. Andrade, R.R. Mattingly, Phosphorylation of the Ras-GRF1 exchange factor at Ser916/898 reveals activation of Ras signaling in the cerebral cortex, J. Biol. Chem. 278 (2003) 13278–13285.

[133] J.H. Norum, H. Dawood, R.R. Mattingly, D. Sandnes, F.O. Levy, Epac- and Rap-independent ERK1/2 phosphorylation induced by Gs-coupled receptor stimulation in HEK293 cells, FEBS Lett. 581 (2007) 15–20.

[134] N.H. Lents, V. Irintcheva, R. Goel, L.W. Wheeler, J.J. Baldassare, The rapid activation of N-Ras by alpha-thrombin in fibroblasts is mediated by the specific G-protein Galphai2-Gbeta1-Ggamma5 and occurs in lipid rafts, Cell. Signal. 21 (2009) 1007–1014.

[135] H. Yang, R.R. Mattingly, The Ras-GRF1 exchange factor coordinates activation of H-Ras and Rac1 to control neuronal morphology, Mol. Biol. Cell 17 (2006) 2177–2189.

[136] C. Giglione, S. Gonfloni, A. Parmeggiani, Differential actions of p60c-Src and Lck kinases on the Ras regulators p120-GAP and GDP/GTP exchange factor CDC25Mm, Eur. J. Biochem. FEBS 268 (2001) 3275–3283.

[137] M. Kiyono, J. Kato, T. Kataoka, Y. Kaziro, T. Satoh, Stimulation of Ras guanine nucleotide exchange activity of Ras-GRF1/CDC25(Mm) upon tyrosine phosphorylation by the Cdc42-regulated kinase ACK1, J. Biol. Chem. 275 (2000) 29788–29793.

[138] I. Arozarena, D. Matallanas, P. Crespo, Maintenance of CDC42 GDP-bound state by Rho-GDI inhibits MAP kinase activation by the exchange factor Ras-GRF. evidence for Ras-GRF function being inhibited by Cdc42-GDP but unaffected by CDC42-GTP, J. Biol. Chem. 276 (2001) 21878–21884.

[139] M.J. Rabiet, M. Tardif, L. Braun, F. Boulay, Inhibitory effects of a dominant-interfering form of the Rho-GTPase Cdc42 in the chemoattractant-elicited signaling pathways leading to NADPH oxidase activation in differentiated HL-60 cells, Blood 100 (2002) 1835–1844.

[140] R. Khosravi-Far, C.J. Der, The Ras signal transduction pathway, Cancer Metastasis Rev. 13 (1994) 67–89.

[141] L.A. Quilliam, S.Y. Huff, K.M. Rabun, W. Wei, W. Park, D. Broek, C.J. Der, Membrane-targeting potentiates guanine nucleotide exchange factor CDC25 and SOS1 activation of Ras transforming activity, Proc. Natl Acad. Sci. USA 91 (1994) 8512–8516.

[142] R. Tonini, E. Mancinelli, M. Balestrini, M. Mazzanti, E. Martegani, A. Ferroni, E. Sturani, R. Zippel, Expression of Ras-GRF in the SK-N-BE neuroblastoma accelerates retinoic-acid-induced neuronal differentiation and increases the functional expression of the IRK1 potassium channel, Eur. J. Neurosci. 11 (1999) 959–966.

[143] E. Sturani, A. Abbondio, P. Branduardi, C. Ferrari, R. Zippel, E. Martegani, M. Vanoni, S. Denis-Donini, The Ras guanine nucleotide exchange factor CDC25Mm is present at the synaptic junction, Exp. Cell Res. 235 (1997) 117–123.

[144] R. Tonini, S. Franceschetti, D. Parolaro, M. Sala, E. Mancinelli, S. Tininini, R. Brusetti, G. Sancini, R. Brambilla, E. Martegani, E. Sturani, R. Zippel, Involvement of

CDC25Mm/Ras-GRF1-dependent signaling in the control of neuronal excitability, Mol. Cell. Neurosci. 18 (2001) 691–701.

[145] E. Zhuravliva, T. Barbakadze, N. Natsvlishvili, D.G. Mikeladze, Haloperidol induces neurotoxicity by the NMDA receptor downstream signaling pathway, alternative from glutamate excitotoxicity, Neurochem. Int. 50 (2007) 976–982.

[146] T. Rubino, G. Forlani, D. Vigano, R. Zippel, D. Parolaro, Ras/ERK signalling in cannabinoid tolerance: from behaviour to cellular aspects, J. Neurochem. 93 (2005) 984–991.

[147] T. Rubino, D. Vigano, F. Premoli, C. Castiglioni, S. Bianchessi, R. Zippel, D. Parolaro, Changes in the expression of G protein-coupled receptor kinases and beta-arrestins in mouse brain during cannabinoid tolerance: a role for RAS-ERK cascade, Mol. Neurobiol. 33 (2006) 199–213.

[148] R. Tonini, S. Ciardo, M. Cerovic, T. Rubino, D. Parolaro, M. Mazzanti, R. Zippel, ERK-dependent modulation of cerebellar synaptic plasticity after chronic Delta9-tetrahydrocannabinol exposure, J. Neurosci. 26 (2006) 5810–5818.

[149] S. Fasano, A. D'Antoni, P.C. Orban, E. Valjent, E. Putignano, H. Vara, T. Pizzorusso, M. Giustetto, B. Yoon, P. Soloway, R. Maldonado, J. Caboche, R. Brambilla, Ras-guanine nucleotide-releasing factor 1 (Ras-GRF1) controls activation of extracellular signal-regulated kinase (ERK) signaling in the striatum and long-term behavioral responses to cocaine, Biol. Psychiatry 66 (2009) 758–768.

[150] X. Li, D.C. McFarland, S.G. Velleman, Effect of transforming growth factor-beta on decorin and beta1 integrin expression during muscle development in chickens, Poult. Sci. 85 (2006) 326–332.

[151] J.R. Abreu, D. de Launay, M.E. Sanders, A.M. Grabiec, M.G. van de Sande, P.P. Tak, K.A. Reedquist, The Ras guanine nucleotide exchange factor RasGRF1 promotes matrix metalloproteinase-3 production in rheumatoid arthritis synovial tissue, Arthritis Res. Ther. 11 (2009) R121.

[152] D.M. Shin, R. Liu, I. Klich, J. Ratajczak, M. Kucia, M.Z. Ratajczak, Molecular characterization of isolated from murine adult tissues very small embryonic/epiblast like stem cells (VSELS), Mol. Cells 29 (2010) 533–538.

[153] C. Qiu, M. Yu, L. Shan, E.G. Snyderwine, Allelic imbalance and altered expression of genes in chromosome 2q11–2q16 from rat mammary gland carcinomas induced by 2-amino-1-methyl-6-phenylimidazo[4, 5-b]pyridine, Oncogene 22 (2003) 1253–1260.

[154] T. Sjoblom, S. Jones, L.D. Wood, D.W. Parsons, J. Lin, T.D. Barber, D. Mandelker, R.J. Leary, J. Ptak, N. Silliman, S. Szabo, P. Buckhaults, C. Farrell, P. Meeh, S.D. Markowitz, J. Willis, D. Dawson, J.K. Willson, A.F. Gazdar, J. Hartigan, L. Wu, C. Liu, G. Parmigiani, B.H. Park, K.E. Bachman, N. Papadopoulos, B. Vogelstein, K.W. Kinzler, V.E. Velculescu, The consensus coding sequences of human breast and colorectal cancers, Science 314 (2006) 268–274.

[155] S.J. Clapcott, J. Peters, P.C. Orban, R. Brambilla, C.F. Graham, Two ENU-induced mutations in Rasgrf1 and early mouse growth retardation, Mamm. Genome 14 (2003) 495–505.

[156] T. Rubino, G. Forlani, D. Vigano, R. Zippel, D. Parolaro, Modulation of extracellular signal-regulated kinases cascade by chronic delta 9-tetrahydrocannabinol treatment, Mol. Cell. Neurosci. 25 (2004) 355–362.

[157] A. Fernandez-Medarde, A. Porteros, J. de Las Rivas, A. Nunez, J.J. Fuster, E. Santos, Laser microdissection and microarray analysis of the hippocampus of Ras-GRF1 knockout mice reveals gene expression changes affecting signal transduction pathways related to memory and learning, Neuroscience 146 (2007) 272–285.

[158] Z. Wang, E. Moro, K. Kovacs, R. Yu, S. Melmed, Pituitary tumor transforming gene-null male mice exhibit impaired pancreatic beta cell proliferation and diabetes, Proc. Natl Acad. Sci. USA 100 (2003) 3428–3432.

[159] M. Pende, S.C. Kozma, M. Jaquet, V. Oorschot, R. Burcelin, Y. Le Marchand-Brustel, J. Klumperman, B. Thorens, G. Thomas, Hypoinsulinaemia, glucose intolerance and diminished β-cell size in S6K1-deficient mice, Nature 408 (2000) 994–997.

[160] S.H. Um, F. Frigerio, M. Watanabe, F. Picard, M. Joaquin, M. Sticker, S. Fumagalli, P.R. Allegrini, S.C. Kozma, J. Auwerx, G. Thomas, Absence of S6K1 protects against age- and diet-induced obesity while enhancing insulin sensitivity, Nature 431 (2004) 200–205.

[161] J. Martin, S.L. Hunt, P. Dubus, R. Sotillo, F. Nehme-Pelluard, M.A. Magnuson, A.F. Parlow, M. Malumbres, S. Ortega, M. Barbacid, Genetic rescue of Cdk4 null mice restores pancreatic beta-cell proliferation but not homeostatic cell number, Oncogene 22 (2003) 5261–5269.
