
Cellular Signalling 23 (2011) 1069–1075

Contents lists available at ScienceDirect  
Cellular Signalling  
journal homepage: www.elsevier.com/locate/cellsig

---

**Review**

# Signaling of interleukin-17 family cytokines in immunity and inflammation

Seon Hee Chang, Chen Dong*

Department of Immunology and Center for Inflammation and Cancer, MD Anderson Cancer Center, Houston, TX, USA

---

## ARTICLE INFO

**Article history:**  
Received 16 November 2010  
Accepted 25 November 2010  
Available online 2 December 2010  

**Keywords:**  
IL-17 cytokine and receptor family  
Signaling pathway  
Inflammation

---

## ABSTRACT

IL-17 cytokine family, though still young since discovery, has recently emerged as critical players in immunity and inflammatory diseases. The prototype cytokine, IL-17A, plays essential roles in promoting inflammation and host defense. IL-17RA, a member of the IL-17 receptor family, forms a complex with another member, IL-17RC, to mediate effective signaling for IL-17A as well as IL-17F, which is most similar to IL-17A, via Act1 and TRAF6 factors. On the other hand, IL-17RA appears to interact with IL-17RB to regulate signaling by another cytokine IL-25. IL-25, the most distant from IL-17A in the IL-17 family, is involved in allergic disease and defense against helminthic parasites. In this review, we discuss recent advancements on signaling mechanisms and biological functions of IL-17A, IL-17F and IL-25, which will shed light on the remaining IL-17 family cytokines and help understand and treat inflammatory diseases.

Published by Elsevier Inc.

---

### Contents

1. Introduction . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .expression of IL-17A is associated with many autoimmune diseases and cancer [6–8]. IL-17A is a pleiotropic cytokine that acts on multiple cell types to enhance the production of proinflammatory molecules. Expression of IL-17A, therefore, is tightly regulated. A subset of CD4+ T cells, Th17, and innate immune cells such as γδ T cells are major producers of IL-17A although other subsets have been identified [9–13].

IL-17F, sharing the strongest amino acid sequence with IL-17A, resembles IL-17A in the cellular sources and regulation [14]. It was thought to play a redundant role with IL-17A since IL-17F also regulates proinflammatory gene expression *in vitro*. Yet, IL-17F exhibits a distinctive role, for instance, during allergic inflammation in lung and intestinal inflammation *in vivo* [15–17]. IL-17A and IL-17F proteins can form a heterodimer and behave in a similar fashion to the homodimeric forms of IL-17A and IL-17F *in vitro* [18,19].

IL-25, the most distant from IL-17A in IL-17 family, promotes Th2 cell-mediated immune responses, thereby contributing to allergic disease and defense against helminthic parasites [20–22]. Diverse cellular sources have been linked to the expression of IL-25 such as epithelial cells, eosinophils, mast cells, and basophils. Three remaining cytokines in the IL-17 family, IL-17B, IL-17C and IL-17D, currently are poorly studied with regards to their biological function and receptors.

IL-17R family is comprised of 5 members (IL-17RA, RB, RC, RD and RE). All of them share sequence homology to the earliest identified member, IL-17RA. So far, the biological roles or ligands of IL-17RD or IL-17RE are not clear. Although the expression and roles of IL-17RA have been known for some time, the recent evidence points to a much broader role of IL-17RA in signaling by IL-17A, IL-17F and IL-25.

In this review, we describe the current understanding on the receptor organization, signaling mechanisms as well as function of IL-17A, IL-17F and IL-25.

## 2. Signaling by IL-17A and IL-17F

### 2.1. Receptors

#### 2.1.1. IL-17RA

IL-17RA was discovered by cDNA library screening of murine thymoma cell line EL4 for IL-17A–Fc binding [2]. Later, IL-17F, a cytokine sharing 50% of the amino acid sequence homology with IL-17A, was discovered [23] and considered to mediate its signaling through IL-17RA. A blocking antibody against IL-17RA prevented IL-17F as well as IL-17A signaling [24]. However, soluble IL-17RA protein prevented only IL-17A signaling and had no effect on IL-17F singling [24]. Similarly, IL-17F did not bind with high affinity to the purified, monomer.ic extracellular domain of IL-17RA [1]. Wright et al. reported the Kd for IL-17RA.Fc to IL-17F is 170 nM, about 100-fold higher when compared to IL-17A [25]. Affinity of heterodimer IL-17A/F to IL-17R is intermediate between IL-17A and IL-17F.

While expression of the ligands, IL-17A and IL-17F, is tightly controlled and their cellular sources are limited, IL-17RA is expressed on a wide variety of tissues and cell types constitutively [2]. A few literature reports documented the expression of IL-17RA is regulated [26,27]. However, functional significance of such a regulation or factors that induce IL-17RA expression are not clear.

Upon the ligation of IL-17RA by IL-17A, IL-17RA initiates signaling pathways to induce proinflammatory molecules, which plays a major role in recruiting neutrophils and tissue inflammation. Depending on the nature of the cells, IL-17RA ligation is known to induce a wide variety of molecules ranging from cytokines (IL-6, GCSF, GMCSF), chemokines (CCL2, CCL7, CCL20, CXCL1, CXCL5), anti-microbial peptides (β defensin-2, S100A7, S100A8, and S100A9), mucins (MUC5B and MUC5AC) and matrix metalloproteinases (MMP1, MMP3, MMP9, MMP12 and MMP13)[29–40]. IL-17RA deficient mice are protected in various degrees in a variety of inflammatory diseases, such as

rheumatoid arthritis [41], multiple sclerosis [42], inflammatory bowel disease [43], and asthma [44]. On the other hand, IL-17RA signaling is essential during infections, since IL-17RA deficient mice are susceptible to many pathogens such as Klebsiella pneumoniae, Bacteroides fragilis, Toxoplasmosis gondii, and Candida albicans [6,45].

#### 2.1.2. IL-17RC

IL-17RC was previously identified by homology search to IL-17RA mRNA cytoplasmic domain using EST clones [46]. High mRNA levels were detected in prostate, cartilage, kidney, liver, heart, and muscle, whereas transcripts were barely detected in thymus and leukocytes, also demonstrated by IL-17RC specific antibody [47]. In contrast to the ubiquitous expression of IL-17RA, IL-17RC expression is limited or preferentially expressed in tissue resident and non-hematopoietic cells [48].

A distinctive feature of IL-17RC in comparison to IL-17RA is that it is expressed as multiple isoforms, and at least 11 RNA splice variants were found with predicted molecular weights ranging from 186 to 720 amino acids [46]. Isoforms of IL-17RC have been detected highly in certain prostate cancers although the role of various isoforms in vivo is not known [49]. Splicing variants of IL-17RC could also produce soluble decoy receptors but the affinity of IL-17A or IL-17F to soluble IL-17RC is not known and the role of soluble receptors has not been investigated. Mice deficient in IL-17RC, however, faithfully represent the role of IL-17A or IL-17F in vivo. IL-17RC-deficient cells are absent in signaling of IL-17A and IL-17F [50]. IL-17RC deficient mice are also resistant to develop EAE [67] and defective in host defense against fungal infection [51]. Although other disease models could be performed in IL-17RC deficient mice, so far, IL-17RC is not likely to have any distinctive features other than mediating IL-17A and IL-17F signaling. Since IL-17RA is involved in other IL-17 cytokine family (later in this review), the biological role of IL-17A and IL-17F is likely represented through IL-17RC, not IL-17RA.

Despite its strong *in vitro* activity of IL-17A, its binding to IL-17RA was found to be of relatively low affinity, suggesting a possible coreceptor for IL-17A and IL-17F [2]. Toy et al. indicated that IL-17RA deficient mouse embryonic fibroblasts (MEFs) were defective in IL-17A or IL-17F signaling once the cells are reconstituted with human IL-17RA, suggesting human counterpart of IL-17RA interacting protein is required for effective IL-17RA signaling [52]. When human IL-17RA was reconstituted along with human IL-17RC in IL-17RA deficient MEFs, the signaling of IL-17A and IL-17F were restored. Later, it was found that IL-17RC interacts with IL-17F [48] (*Fig.* 1). In mouse, IL-17A does not bind to IL-17RC but human IL-17RC binds to both IL-17A and IL-17F. This finding raised a question on how IL-17RA and IL-17RC are organized in the cell and undergo changes upon the ligands binding. By using the FRET method, IL-17RA was found to exist as a multimeric, preformed receptor complex [53]. However, there is no evidence of pre-assembled IL-17RA with IL-17RC complex. Ely et al. resolved the crystal structure of a complex of IL-17RA bound to IL-17F. The study indicated that the affinity of preformed IL-17RA–IL-17F and IL-17RC–IL-17F is high while the same second receptor binding is too weak to measure. Therefore, the engagement of IL-17RA or IL-17RC by IL-17A or IL-17F may promote a different receptor binding to the complex and thereby to form a heterodimeric receptor complex [54].

#### 2.1.3. Role of IL-17RA and IL-17RC in hematopoietic cells

IL-17RC expression appears to be restricted mostly in non-hematopoietic cells [48]. This may explain why IL-17A or IL-17F has functional roles predominantly in non-hemapoietic cells while effects of these cytokines in hematopoietic cells are limited. Recently, a few studies demonstrated that IL-17A directly acts on T or B cells. IL-17RA was induced during differentiation of Th1 cells, and IL-17A can suppress Th1 cell programming by downregulating Tbet and STAT1 phosphorylation [27]. IL-17A has been also reported to influence the chemotaxis of B cells and induces Rgs13 and Rgs16 expressions in B
Fig. 1. IL-17 cytokine family, their receptors and membrane proximal adaptors IL-17, IL-17F and IL-17A/F are produced by CD4 Th17 cells, γδ T cells and ILCs (Innate lymphoid cells such as iNKT, NKp46+ and LTi-like cell). IL-25 has been detected in epithelial cells and granulocytes such as eosinophils, mast cells, basophils. IL-17RA/IL-17RC is a signaling complex for IL-17, IL-17F and IL-17A/F. IL-17RA and IL-17RB mediate IL-25 signaling. IL-17R family shares a SEFIR motif that recruits Act1 and TRAF6 for signaling complex.

cells [55]. In turn, Hsu et al. concluded IL-17A induced GC development in vivo [56]. However, IL-17A or IL-17F deficient mice exhibit normal GC development [57]. IL-17A also acted in synergy with BAFF to drive the survival and proliferation of human B cells and their differentiation into immunoglobulin-secreting cells [58]. But it is not known that IL-17RC participates in IL-17A signaling in hematopoietic cells.

### 2.2. Signal transduction

#### 2.2.1. TRAF6
IL-17A activates NFκB signaling and MAP kinases through IL-17RA (Table 1). IL-17A signaling is absent in TRAF6 deficient mouse embryonic fibroblasts as IL-17A treatment fails to induce NFκB and JNK activation as well as IL-6 production [59]. When TRAF6 and IL-17RA were expressed in 293 cells together, TRAF6 was immunoprecipitated with IL-17RA. Requirement of TRAF6 in IL-17RA signaling indicates that IL-17RA signaling may resemble TLR/IL1R signaling. A protein alignment study identified IL-17R family shares the homology with TLR/IL1R family, thus proposed that IL-17R family belongs to a larger protein group called STIR (similar expression to fibroblast growth factor and IL-17R; Toll-IL-1R) [60]. Despite its similarity to the TIR domain, IL-17A does not utilize the same set of membrane-proximal adaptor molecules as Toll-like receptor signaling does such as MyD88 or IRAK4 [61]. Also, IL-17RA does not have a TRAF6 binding site, suggesting the presence of another adaptor that mediates TRAF6 recruitment to IL-17RA.

---

**Table 1**

IL-17R family and their cell specific signaling pathways. The signaling pathways of IL-17RA/IL-17RC and IL-17RA/IL-17RB are mediated by Act1/CIKS and TRAF6 although some of cell types need further investigation. While the membrane proximal adaptors, Act1/CIKS and TRAF6, are utilized in both IL-17RA/IL-17RC and IL-17RA/IL-17RB signaling pathways, the transcription factors that are involved in the signaling pathways are different between these two receptor complexes. Also, the effector molecules induced by these receptor complexes vary among the cell types (COPD; chronic obstructive pulmonary disease, LCH; Langerhans cell histiocytosis, Rgs; regulators of G-protein signaling, ZO; Zonula occludens, n/a.; not available).

| Target cell types                                      | Membrane proximal adapter | Signaling components                                                                 | Transcription factors                                                                 | Notable effector molecules                                                                                     | References       |
|---------------------------------------------------------|---------------------------|-------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|------------------|
| **IL-17RA-IL-17RC**                                    |                           |                                                                                     |                                                                                      |                                                                                                                  |                  |
| Epithelial cells or fibroblasts                         | Act1/CIKS                 | TRAF6, ERK1/2, JNK, and p38, NFκB (TRAF6-independent pathway for mRNA stabilization) | C/EBPβ, C/EBPδ, IkBζ                                                                  | IL-6, CXCL1, CCL2, CCL7, MMPs, CCL20, β defensin, mucin antimicrobial peptide                                     | [17,28,29]      |
| Primary human lung cells                               |                           |                                                                                     |                                                                                      |                                                                                                                  | [32–34]         |
| Primary human keratinocytes                            |                           |                                                                                     |                                                                                      | C reactive proteins                                                                                             | [31]             |
| Smooth muscle, hepatocytes                             |                           |                                                                                     |                                                                                      | RANKL                                                                                                          | [38]             |
| Osteoblasts, synovial fibroblasts                       |                           |                                                                                     |                                                                                      | suppress occludin and ZO-1                                                                                      | [36,37]          |
| Blood-brain barrier endothelial cells                   |                           |                                                                                     |                                                                                      | junction-forming protein claudins                                                                               | [39]             |
| Intestinal epithelial cells                            |                           |                                                                                     |                                                                                      |                                                                                                                  | [35]             |
| **IL-17RA-IL-17RB**                                    |                           |                                                                                     |                                                                                      |                                                                                                                  |                  |
| B cells                                                | n/a                       | n/a                                                                                 | n/a                                                                                  | Rgs13 and Rgs16                                                                                                 | [50,51]          |
| Dendritic cells from COPD, LCH                        | n/a                       | n/a                                                                                 | n/a                                                                                  | MMP9, MMP12                                                                                                    | [30,40]          |
| Naïve CD4                                              | n/a                       | n/a                                                                                 | JunB, NFATc1                                                                         | IL-4                                                                                                           | [82]             |
| Effector or memory Th2 cells                           | n/a                       | n/a                                                                                 | GATA3, c-maf, JunB                                                                   | IL-4, IL-5, IL-13, IL-9                                                                                         | [89,97]          |
| Innate lymphoid cells or nonlymphoid c-kit+ cell       | n/a                       | n/a                                                                                 | n/a                                                                                  | IL-13                                                                                                          | [91–94]          |
| Lung CD11C+ cells or BM derived dendritic cells        | Act1/CIKS                 | TRAF6                                                                               | n/a                                                                                  | IL-5, IL-13                                                                                                    | [95]             |
| Epithelial cells                                       | Act1/CIKS                 | TRAF6, NFκB, ERK1/2, JNK, and p38                                                   | n/a                                                                                  | IL-6, TGFβ, TARC, GCSF                                                                                          | [98,99]          |
| Brain capillary endothelial cells                      | n/a                       | n/a                                                                                 | PKCe                                                                                 | claudin-5                                                                                                      | [100]            |

2.2.2. Act1

Act1 is a cytoplasmic protein that shares homology to the cytoplasmic domain of IL-17R family. It is originally discovered through a yeast-two hybrid screening based on its interaction with IKKγ (named CIKS-connection to IKK and SAPK/JNK) [62] and screening of a constitutive activator for NF-κB using cDNA library (Act1 as an activator of NFκB) [63]. Act1 was reported to be a negative regulator for CD40 and BAFF-mediated B cell signaling [64]. Act1-deficient splenic B cells expressed stronger IκB phosphorylation, processing of NF-κB2 (p100/p52), and activation of JNK, ERK, and p38 pathways upon CD40 and BAFF treatment. Consequently, Act1-deficient mice manifested with increase in peripheral B cells, leading to lymphadenopathy, splenomegaly, hypergammaglobulinemia, inflammation in multiple tissues, and the formation of autoantibodies [65]. In another study, however, Act1-deficient mice were normal and did not have any aberrant B cell signaling leading to autoimmunity and Act1-deficient B cells exhibit normal CD40 and BAFF signaling [66]. It is not known why the two groups have a discrepancy in Act1 in mediating B cell signaling. On the other hand, mice carrying a spontaneous recessive point mutation in Act1 develop atopic dermatitis-like skin disease with hyper-IgE-emia [67].

Act1 is reported to have a direct interaction with the cytoplasmic domain of IL-17RA [61]. Since Act1 has TRAF6 binding sites and forms immunocomplex directly with IL-17RA mediated through its SEFIR domain, Act1 may recruit TRAF6 to IL-17RA. In Act1-deficient cells, IL-17A is not able to activate NFκB pathway or produce proinflammatory molecules such as IL-6 and CXCL1 [89]. Also, IL-17A is known to activate 3 transcription factors, C/EBPβ, C/EBPδ and IKBζ [68]. Induction of these transcription factors is abrogated in Act1-deficient cells. Furthermore, Act1-deficient mice showed much less inflammatory disease in vivo in both autoimmune encephalomyelitis and dextran sodium sulfate-induced colitis [69]. Since IL-17RA is required for IL-17F signaling, Act1-deficient cells are abrogated in mediating IL-17F signaling [16]. Act1 is also reported to interact with IL-17RC, and their interaction is enhanced upon IL-17A treatment [51].

2.2.3. NFκB, MAPKs and other pathways

Recruitment of Act1 and TRAF6 to IL-17RA further activates NFκB. Although some reported IL-17A as a weak NFκB activator, it may vary depending on the cell types. Also, IL-17A can synergize with other cytokines, especially with TNFα, to promote stronger immune responses [68]. In human bronchial epithelial cells, NFκB activation by IL-17RA is indispensable in inducing MUC5AC and CCL-20 [70]. JAK-associated PI3K/GSK3β signaling has also been implicated in IL-17RA signaling [71]. Inductions of hBD2 and IL-19 are partially dependent on activation of JAK-associated PI3K activation. Several laboratories have shown that IL-17A serves as a strong stimulus to induce the stabilization of constitutively unstable mRNAs [72]. mRNA stabilization is independent from the TRAF6/p38/MK2/MK3-independent pathway but requires Act1, indicating a divergent signaling pathway of IL-17RA [73]. Also, a negative regulation pathway mediated through IL-17RA has been reported. IL-17RA can be mediated through extracellular signal-regulated kinase (ERK) and glycogen synthase kinase 3β (GSK-3β)-dependent mechanisms to phosphorylate C/EBPβ, leading to suppression of the cytokines induced by IL-17RA [74].

2.2.4. Functional domains of IL-17RA and IL-17RC

SEFIR domain of IL-17RA is essential to provide homotypic interaction with Act1. SEFIR and a newly described domain, TIR-like loop (TILL) of IL-17RA both are required for the activation of NF-κB, MAPK, and the up-regulation of C/EBPβ and C/EBPδ [75]. Although some of the conserved residues within the TILL motif correspond to residues in the BB-loop in TIR that form a key salt bridge, mutations in the charged residue have no effects on IL-17RA signaling. Instead, a mutation of valine at residue 553 of IL-17RA leads to complete absence of IL-17RA signaling. This suggests that TILL motif of the cytoplasmic portion of IL-17RA also provides structural integrity of IL-17RA for signaling. Also, extended region beyond SEFIR and TILL is also involved in maintaining IL-17RA signaling [76]. Since IL-17RA pairs with IL-17RC to serve signaling receptors, a question was raised on whether the cytoplasmic domain of IL-17RC also contributes to the signaling. Ho et al. demonstrated that SEFIR domain and the extended region of cytoplasmic domain of IL-17RC are important in mediating effective IL-17 and IL-17F signaling using various forms of IL-17RC deletion mutants [51].

3. IL-25 signaling

3.1. IL-25 and IL-17RB

IL-17RB was identified by EST database homology search using IL-17RA amino acid sequences [5,77]. IL-17RB is a 502-amino acid single transmembrane protein that shares 26% amino acid identity to IL-17RA. IL-25, also known as IL-17E, is a high affinity ligand for IL-17RB [5]. IL-17B was also reported to bind to IL-17RB with an affinity of 7.6 nM [77], while IL-25 binds to IL-17RB with Kd value of 1.1–1.4 nM [1]. Binding to the same receptor raises a question of whether IL-17B and IL-25 have any overlapping or antagonistic functions. The role of IL-17B in the immune system is not understood at present and it is not known whether IL-17B functions through IL-17RB.

The biological role of IL-25 was first revealed by administration of exogenous IL-25 [78,79] or transgenic mice overexpressing IL-25 [80,81]. The distinctive features of IL-25 mediated immune responses are heavy infiltration of eosinophils and induction of Th2-related molecules such as production of IL-4, IL-5, IL-13. Enforced expression of IL-25 in lung also resulted in increased Th2 cytokine and mucus production [82,83]. Consequently, IL-25-deficient mice were unable to clear parasitic helminth such as Nippostrongylus brasiliensis [84] and Trichuris muris [85] owing to failed Th2 immune responses. Therefore, IL-25 is a critical cytokine mediating allergic inflammation and host defense against parasitic helminth. Various cellular sources have been linked with IL-25 expression. Lung epithelial cells can induce IL-25 upon encounter with allergens such as ragweed or protease from Aspergillus oryzae [82]. Also, IL-25 has been detected in Th2 cells [78], alveolar macrophages [86], mast cells [87], eosinophils [88] and basophils [89]. LacZ/IL-25 reporter mice indicated that IL-25 is expressed constitutively by CD4+ and CD8+ T cells in the cecal patch of mice resistant to Trichuris [85].

A number of cell types can respond to IL-25. Highest levels of IL-17RB expression were observed in kidney, then liver and other peripheral organs [5]. In cellular level, various subsets have been linked with to IL-17RB expression. Although IL-17RB level was not assessed directly in these studies, earlier studies concluded that one of major IL-25 responder cells in vivo is a novel accessory population of non-T/non-B cells (NBNT), characterized as Lin−, MHC classIIhigh, CD11cnull [78] and NBNT with c-kit+, FcεR1− [84]. These novel accessory cells produce IL-4, IL-5 and IL-13 upon IL-25 stimulation and were suggested to be mast cells or mast cell precursors or hematopoietic stem cells on the basis of c-Kit expression. Recently, several antibodies specific to IL-25 were developed and confirmed the expression of IL-17RB on NBNT with c-kit+, FcεR1− [90]. This population was further described to be important for the initiation of N. brasiliensis worm clearance in vivo [84]. Recently several groups identified IL-25 lead to induction of lineage negative cells, named variously as multipotent progenitor cell [91], adipose tissue-associated c-Kit(+)Sca-1(+) lymphoid cells [92] or nuocytes (IL-13 producing cells) [93] or innate Th2 cells [94]. IL-17RB was also preferentially expressed on a fraction of α-galactosylceramide (α-GalCer)/CD1d dimer + NKT cells [90]. IL-25-activated IL-17RB + NKT cells produced IL-13 predominantly and Th2 chemokines CCL17, CCL22 and eosinophil chemotactic factor-L. This population of cells is

critical in inducing airway hypersensitive reaction. In the animal model of chronic allergen responses, IL-17RB was induced in a myeloid cell population, identified as IL-4+ CD11b+ Gr1+ cells both in lung and bone marrows [95].

Activated Th2 memory cells express a higher level of IL-17RB, in particular, TSLP treated dendritic cells strongly up-regulate expression of IL-17RB in mRNA and protein level. In this setting, IL-25 appears to promote the expansion and further polarization of Th2 central memory cells [89]. IL-17RB was also detected in mouse naive CD4 T cells and induced further in differentiated Th2 cells [82]. IL-25 promotes Th2 differentiation by inducing early IL-4 production. This further promotes the production of IL-4, IL-5 and IL-13 facilitating Th2 polarization. Treatment of IL-25 on *in vitro* differentiated effector Th2 cells was also found to further enhance Th2 differentiation [82]. In addition, a newly described subset of T helper cells producing IL-9, Th9, expressed high levels of mRNA for IL-17RB [97]. TGF-β together with IL-4, a polarizing condition for the generation of Th9, induced the expression of IL-17RB. IL-25 treatment in Th9 condition enhanced IL-9 secretion. Moreover, transgenic overexpression of IL-17RB in T cells resulted in IL-25-induced IL-9 production, leading to severe allergic lung diseases, and lack of IL-25 greatly decreased IL-9 expression in vivo [97].

### 3.2. Signal transduction

Although IL-25 does not bind to IL-17RA *in vitro* [1], recent evidences point to IL-17RA and IL-17RB as a signaling complex for IL-25 signaling. Splenocytes from IL-17RB KO and IL-17RA KO mice do not produce IL-5 or IL-13 in response to IL-25 stimulation [116]. Angkasekwinai *et al.* reported that addition of IL-25 to T cells cultured in Th9-inducing conditions significantly enhanced IL-9 secretion in wild-type cells but had no effect on IL-17RA-deficient cells, indicating that IL-17RA is required for the IL-25 effect on IL-9 production in CD4 T cells [97]. The same study reported that IL-25 signaling in CD4 T cells is abrogated when the deletion mutant lacking cytoplasmic domain of IL-17RB was reconstituted. When both molecules were overexpressed in 293T cells, they were immunoprecipitated as a complex. Although IL-17RA does not directly interact with IL-25 in *in vitro* binding assay, IL-17RA bound to the IL-17RB–IL-25 complex with an affinity of 14.1 ± 2.4 μM [54]. Mice lacking expression of IL-17RA failed to exhibit IL25-elicted population expansion of the c-Kit<sup>int</sup> cells [91] similar to IL-17RB deficient mice, indicating the lineage negative cells also require IL-17RA for IL-25 signaling.

#### 3.2.1. Act1

Act1, the adaptor of IL-17RA signaling, is also involved in IL-17RB signaling [66]. The interaction between Act1 and IL-17RB was abolished when the SEFIR domain was deleted in either Act1 or IL-17RB, indicating the recruitment of Act1 to IL-17RB is through the dimerization of the SEFIR domain. Mice lacking Act1 in epithelial cells have reduced levels of Th2-associated gene expression upon IL-25 intranasal inhalation [98]. Since IL-25 does not interact with IL-17RA in *in vitro* binding assay, it is not clear the exact role of IL-17RA in IL-17RB/IL-17RA complex. Deleting mutants of IL-17RA in mediating IL-25 signaling may provide some clues to the extent of the role and mechanism of IL-17RA in mediating IL-25 signaling.

#### 3.2.2. TRAF6

As TRAF6 was implicated in IL-17RA signaling, NF-κB activation mediated through IL-17RB is abrogated in TRAF6 deficient cells, indicating IL-25 may acquire similar signaling pathways as IL-17A [99]. However, IL-25 activates ERK, JNK, and p38 by a TRAF6-independent mechanism. In contrary to IL-17RA that lacks TRAF6 binding sites, IL-17RB possesses TRAF6 binding sites. IL-17RB-mediated NF-B activation was attenuated in cells expressing in IL-17RB E338A, in which TRAF6-binding motif was mutated [99].

Therefore, the mechanisms underlying TRAF6 activation may be different in IL-17RB and IL-17RA-mediated signaling.

#### 3.2.3. Transcription factors and other pathways

Signaling events by IL-17RB also is likely to be cell-specific. IL-25 induces early IL-4 expression through JunB and NFATc1 in CD4 T cells, which then possibly activate GATA-3 and STAT6 through the IL-4 signaling pathway [82]. IL-25 induces expression of GATA-3, c-maf, and JunB by activated Th2 memory cells [89]. It is not known whether IL-25 mediates the signaling in CD4 T cells through TRAF6 and Act1 or there is independent pathway. In epithelial cells, IL-17RB is known to activate NF-κB and MAPK pathways through TRAF6 [99]. IL-25 induces PKCε phosphorylation in brain capillary endothelial cells to induce claudin-5, which is critical in maintaining the blood brain barrier function [100].

## 4. Concluding remarks

IL-17RA was identified over a decade ago as the receptor for a newly discovered cytokine, IL-17A. Additional studies revealed IL-17RA is also a receptor for IL-17F and IL-25. IL-17RA pairs with IL-17RC in tissue resident cells such as epithelial cells and fibroblasts to be the functional receptor for IL-17A and IL-17F. On the other hand, IL-17RA interacts with IL-17RB to serve as the effective receptor pairs for IL-25, demonstrated in epithelial cells and CD4 T cells. These studies support the hypothesis that IL-17RA may function as a common receptor for IL-17 cytokine family, although the biochemical and structural data are needed to substantiate this idea.

Many challenging questions are still remained. First, it is not clear how the receptors are organized. Although experiments overexpressing the receptors showed their potential to interact, coexpression of the receptor pairs alone is not sufficient to drive ligand-independent signaling. The evidence for pre-formed complex of IL-17RA–IL-17RC and IL-17RA–IL-17RB in the membrane of primary cells is lacking. IL-17RA, IL-17RC and IL-17RB may initially capture the ligands, IL-17A, IL-17F and IL-25, respectively, and bring an additional receptor, IL-17RC (for IL-17A) or IL-17RA (for IL-17F and IL-25) for the receptor complex. In this setting, what brings the other component remains unknown. Since Act1 shares the homology to these receptors, there might be a preformed complex with the each receptor and Act1. Upon the ligand binding, structural changes in the cytoplasmic domain of the receptors could allow homotypic interaction of Act1 between two receptors. In turn, Act1 may accelerate and strengthen the assembly of IL-17RA–IL-17RC or IL-17RA–IL-17RB complex. Second, it is unknown how the functional specificity of cytokines is determined once the ligands are recognized by the receptors. Since IL-17A and IL-17F bind to IL-17RA and IL-17RC with different affinity, the potency of these two cytokines in regards to their redundant functions could be mediated in part by their affinity to the receptors. On the other hand, expression level of IL-17RA and IL-17RC varies among cell types. This may influence distinct activities of IL-17A and IL-17F. In addition, the cytoplasmic domain of IL-17RA, IL-17RC and IL-17RB may recruit different signaling components to provide ligand and cell-specific signaling. While IL-17R family members share a common motif, SEFIR, each IL-17R has distinctive regions that may be involved in the ligand-specific signaling pathway. Systemic approach such as proteomics of IL-17R family or Act1 could reveal additional signaling components, which help to understand functional specificity of IL-17A, IL-17F and IL-25. In addition, future studies should reveal the receptor organization of additional members for IL-17R family, IL-17RD and IL-17RE.

## Acknowledgements

We thank the Dong laboratory members for their scientific contributions and discussion. The work was supported by research grants from NIH (to C.D.). C.D. receives a Research Trust Fellowship

and is the Olga and Harry Wiess Distinguished University Chair in Cancer Research of the University of Texas MD Anderson Cancer Center and a Leukemia and Lymphoma Society Scholar.

# References

[1] S.G. Hymowitz, E.H. Filvaroff, J.P. Yin, J. Lee, L. Cai, P. Risser, M. Maruoka, W. Mao, J. Foster, R.F. Kelley, G. Pan, A.L. Gurney, A.M. de Vos, M.A. Starovasnik, EMBO J. 20 (19) (2001) 5332.

[2] Z. Yao, W.C. Fanslow, M.F. Seldin, A.M. Rousseau, S.L. Painter, M.R. Comeau, J.I. Cohen, M.K. Spriggs, Immunity 3 (6) (1995) 811.

[3] H. Li, J. Chen, A. Huang, J. Stinson, S. Heldens, J. Foster, P. Dowd, A.L. Gurney, W.I. Wood, Proc. Natl Acad. Sci. USA 97 (2) (2000) 773.

[4] T. Starnes, H.E. Broxmeyer, M.J. Robertson, R. Hromas, J. Immunol. 169 (2) (2002) 642.

[5] J. Lee, W.H. Ho, M. Maruoka, R.T. Corpuz, D.T. Baldwin, J.S. Foster, A.D. Goddard, D.G. Yansura, R.L. Vandlen, W.I. Wood, A.L. Gurney, J. Biol. Chem. 276 (2) (2001) 1660.

[6] W. Ouyang, J.K. Kolls, Y. Zheng, Immunity 28 (4) (2008) 454.

[7] P.J. Dubin, J.K. Kolls, Immunol. Rev. 226 (2008) 160.

[8] C. Dong, Immunol. Rev. 226 (2008) 80.

[9] C. Dong, Nat. Rev. Immunol. 8 (5) (2008) 337.

[10] D.J. Cua, C.M. Tato, Nat. Rev. Immunol. 10 (7) (2010) 479.

[11] C.T. Weaver, R.D. Hatton, P.R. Mangan, L.E. Harrington, Annu. Rev. Immunol. 25 (2007) 821.

[12] L. Zhou, M.M. Chong, D.R. Littman, Immunity 30 (5) (2009) 646.

[13] T. Korn, E. Bettelli, M. Oukka, V.K. Kuchroo, Annu. Rev. Immunol. 27 (2009) 485.

[14] S.H. Chang, C. Dong, Cytokine 46 (1) (2009) 7.

[15] N. Hizawa, M. Kawaguchi, S.K. Huang, M. Nishimura, Clin. Exp. Allergy 36 (9) (2006) 1109.

[16] X.O. Yang, S.H. Chang, H. Park, R. Nurieva, B. Shah, L. Acero, Y.H. Wang, K.S. Schluns, R.R. Broaddus, Z. Zhu, C. Dong, J. Exp. Med. 205 (5) (2008) 1063.

[17] H. Ishigame, S. Kakuta, T. Nagai, M. Kadoki, A. Nambu, Y. Komiyama, N. Fujikado, Y. Tanahashi, A. Akitsu, H. Kotaki, K. Sudo, S. Nakae, C. Sasakawa, Y. Iwakura, Immunity 30 (1) (2009) 108.

[18] S.C. Liang, A.J. Long, F. Bennett, M.J. Whitters, R. Karim, M. Collins, S.J. Goldman, K. Dunussi-Joannopoulos, C.M. Williams, J.F. Wright, L.A. Fouser, J. Immunol. 179 (11) (2007) 7791.

[19] A.M. Akimzhanov, X.O. Yang, C. Dong, J. Biol. Chem. 282 (9) (2007) 5969.

[20] B.P. Pappu, P. Angkasekwinai, C. Dong, Pharmacol. Ther. 117 (3) (2008) 374.

[21] S.A. Saenz, B.C. Taylor, D. Artis, Immunol. Rev. 226 (2008) 172.

[22] J.L. Barlow, A.N. McKenzie, Biofactors 35 (2) (2009) 178.

[23] M. Kawaguchi, L.F. Onuchic, X.D. Li, D.M. Essayan, J. Schroeder, H.Q. Xiao, M.C. Liu, G. Krishnaswamy, G. Germino, S.K. Huang, J. Immunol. 167 (8) (2001) 4430.

[24] F. McAllister, A. Henry, J.L. Kreindler, P.J. Dubin, L. Ulrich, C. Steele, J.D. Finder, J.M. Pilewski, B.M. Carreno, S.J. Goldman, J. Pirhonen, J.K. Kolls, J. Immunol. 175 (1) (2005) 404.

[25] J.F. Wright, F. Bennett, B. Li, J. Brooks, D.P. Luxenberg, M.J. Whitters, K.N. Tomkinson, L.J. Fitz, N.M. Wolfman, M. Collins, K. Dunussi-Joannopoulos, M. Chatterjee-Kishore, B.M. Carreno, J. Immunol. 181 (4) (2008) 2799.

[26] M.J. Lindemann, Z. Hu, M. Benczik, K.D. Liu, S.L. Gaffen, J. Biol. Chem. 283 (20) (2008) 14100.

[27] W. O'Connor Jr., M. Kamanaka, C.J. Booth, T. Town, S. Nakae, Y. Iwakura, J.K. Kolls, R.A. Flavell, Nat. Immunol. 10 (6) (2009) 603.

[28] F. Shen, M.J. Ruddy, P. Plamondon, S.L. Gaffen, J. Leukoc. Biol. 77 (3) (2005) 388.

[29] S.L. Gaffen, Cytokine 43 (3) (2008) 402.

[30] M. Shan, H.F. Cheng, L.Z. Song, L. Roberts, L. Green, J. Hacken-Bitar, J. Huh, F. Bakaeen, H.O. Coxson, C. Storness-Bliss, M. Ramchandani, S.H. Lee, D.B. Corry, F. Kheradmand, Sci. Transl. Med. 1 (4) (2009) 4ra10.

[31] S.C. Liang, X.Y. Tan, D.P. Luxenberg, R. Karim, K. Dunussi-Joannopoulos, M. Collins, L.A. Fouser, J. Exp. Med. 203 (10) (2006) 2271.

[32] C.Y. Kao, Y. Chen, P. Thai, S. Wachi, F. Huang, C. Kim, R.W. Harper, R. Wu, J. Immunol. 173 (5) (2004) 3482.

[33] T. Fujisawa, S. Velichko, P. Thai, L.Y. Hung, F. Huang, R. Wu, J. Immunol. 183 (10) (2009) 6236.

[34] C.Y. Kao, F. Huang, Y. Chen, P. Thai, S. Wachi, C. Kim, L. Tam, R. Wu, J. Immunol. 175 (10) (2005) 6676.

[35] T. Kinugasa, T. Sakaguchi, X. Gu, H.C. Reinecker, Gastroenterology 118 (6) (2000) 1001.

[36] S. Kotake, N. Udagawa, N. Takahashi, K. Matsuzaki, K. Itoh, S. Ishiyama, S. Saito, K. Inoue, N. Kamatani, M.T. Gillespie, T.J. Martin, T. Suda, J. Clin. Invest. 103 (9) (1999) 1345.

[37] K. Sato, A. Suematsu, K. Okamoto, A. Yamaguchi, Y. Morishita, Y. Kadono, S. Tanaka, T. Kodama, S. Akira, Y. Iwakura, D.J. Cua, H. Takayanagi, J. Exp. Med. 203 (12) (2006) 2673.

[38] D.N. Patel, C.A. King, S.R. Bailey, J.W. Holt, K. Venkatachalam, A. Agrawal, A.J. Valente, B. Chandrasekar, J. Biol. Chem. 282 (37) (2007) 27229.

[39] H. Kebir, K. Kreymborg, I. Ifergan, A. Dodelet-Devillers, R. Cayrol, M. Bernard, F. Giuliani, N. Arbour, B. Becher, A. Prat, Nat. Med. 13 (10) (2007) 1173.

[40] F. Coury, N. Annels, A. Rivollier, S. Olsson, A. Santoro, C. Speziani, O. Azocar, M. Flacher, S. Djebali, J. Tebib, M. Brytting, R.M. Egeler, C. Rabourdin-Combe, J.I. Henter, M. Arico, C. Delprat, Nat. Med. 14 (1) (2008) 81.

[41] E. Lubberts, P. Schwarzenberger, W. Huang, J.R. Schurr, J.J. Peschon, W.B. van den Berg, J.K. Kolls, J. Immunol. 175 (5) (2005) 3360.

[42] I. Gonzalez-Garcia, Y. Zhao, S. Ju, Q. Gu, L. Liu, J.K. Kolls, B. Lu, J. Immunol. 182 (5) (2009) 2665.

[43] Z. Zhang, M. Zheng, J. Bindas, P. Schwarzenberger, J.K. Kolls, Inflamm. Bowel Dis. 12 (5) (2006) 382.

[44] S. Schnyder-Candrian, D. Togbe, I. Couillin, I. Mercier, F. Brombacher, V. Quesniaux, F. Fossiez, B. Ryffel, B. Schnyder, J. Exp. Med. 203 (12) (2006) 2715.

[45] J.K. Kolls, A. Linden, Immunity 21 (4) (2004) 467.

[46] D. Haudenschild, T. Moseley, L. Rose, A.H. Reddi, J. Biol. Chem. 277 (6) (2002) 4309.

[47] D. Ge, Z. You, Int. Arch. Med. 1 (1) (2008) 19.

[48] R.E. Kuestner, D.W. Taft, A. Haran, C.S. Brandt, T. Brender, K. Lum, B. Harder, S. Okada, C.D. Ostrander, J.L. Kreindler, S.J. Aujla, B. Reardon, M. Moore, P. Shea, R. Schreckhise, T.R. Bukowski, S. Presnell, P. Guerra-Lewis, J. Parrish-Novak, J.L. Ellsworth, S. Jaspers, K.E. Lewis, M. Appleby, J.K. Kolls, M. Rixon, J.W. West, Z. Gao, S.D. Levin, J. Immunol. 179 (8) (2007) 5462.

[49] Z. You, X.B. Shi, G. DuRaine, D. Haudenschild, C.G. Tepper, S.H. Lo, R. Gandour-Edwards, R.W. de Vere White, A.H. Reddi, Cancer Res. 66 (1) (2006) 175.

[50] Y. Hu, N. Ota, I. Peng, C.J. Refino, D.M. Danilenko, P. Caplazi, W. Ouyang, J. Immunol. 184 (8) (2010) 4307.

[51] A.W. Ho, F. Shen, H.R. Conti, N. Patel, E.E. Childs, A.C. Peterson, N. Hernandez-Santos, J.K. Kolls, L.P. Kane, W. Ouyang, S.L. Gaffen, J. Immunol. 185 (2) (2010) 1063.

[52] D. Toy, D. Kugler, M. Wolfson, T. Vanden Bos, J. Gurgel, J. Derry, J. Tocker, J. Peschon, J. Immunol. 177 (1) (2006) 36.

[53] J.M. Kramer, L. Yi, F. Shen, A. Maitra, X. Jiao, T. Jin, S.L. Gaffen, J. Immunol. 176 (2) (2006) 711.

[54] L.K. Ely, S. Fischer, K.C. Garcia, Nat. Immunol. 10 (12) (2009) 1245.

[55] S. Xie, J. Li, J.H. Wang, Q. Wu, P. Yang, H.C. Hsu, L.E. Smythies, J.D. Mountz, J. Immunol. 184 (5) (2010) 2289.

[56] H.C. Hsu, P. Yang, J. Wang, Q. Wu, R. Myers, J. Chen, J. Yi, T. Guentert, A. Tousson, A.L. Le, T.V. Stanus, R.G. Lorenz, H. Kolls, J.K. Xu, R.H. Carter, D.D. Chaplin, R.W. Williams, J.D. Mountz, Nat. Immunol. 9 (2) (2008) 166.

[57] R.I. Nurieva, Y. Chung, D. Hwang, X.O. Yang, H.S. Kang, L. Ma, Y.H. Wang, S.S. Watowich, A.M. Jetten, Q. Tian, C. Dong, Immunity 29 (1) (2008) 138.

[58] A. Doreau, A. Belot, J. Bastid, B. Riche, M.C. Trescol-Biemont, B. Ranchin, N. Fabien, P. Cochat, C. Pouteil-Noble, P. Trolliet, I. Durieu, J. Tebib, B. Kassai, S. Ansieau, A. Puisieux, J.F. Eliaou, N. Bonnefoy-Berard, Nat. Immunol. 10 (7) (2009) 778.

[59] R. Schwandner, K. Yamaguchi, Z. Cao, J. Exp. Med. 191 (7) (2000) 1233.

[60] M. Novatchkova, A. Leibbrandt, J. Werzowa, A. Neubuser, F. Eisenhaber, Trends Biochem. Sci. 28 (5) (2003) 226.

[61] S.H. Chang, H. Park, C. Dong, J. Biol. Chem. (2006) C600256200.

[62] A. Leonardi, A. Chariot, E. Claudio, K. Cunningham, U. Siebenlist, Proc. Natl Acad. Sci. USA 97 (19) (2000) 10494.

[63] X. Li, M. Commane, H. Nie, X. Hua, M. Chatterjee-Kishore, D. Wald, M. Haag, G.R. Stark, Proc. Natl Acad. Sci. USA 97 (19) (2000) 10489.

[64] Y. Qian, Z. Zhao, Z. Jiang, X. Li, Proc. Natl Acad. Sci. USA 99 (14) (2002) 9386.

[65] Y. Qian, J. Qin, G. Cui, M. Naramura, E.C. Snow, C.F. Ware, R.L. Fairchild, S.A. Omori, R.C. Rickert, M. Scott, B.L. Kotzin, X. Li, Immunity 21 (4) (2004) 575.

[66] E. Claudio, S.U. Sonder, S. Saret, G. Carvalho, T.R. Ramalingam, T.A. Wynn, A. Chariot, A. Garcia-Perganeda, A. Leonardi, A. Paun, A. Chen, N.Y. Ren, H. Wang, U. Siebenlist, J. Immunol. 182 (3) (2009) 1617.

[67] Y. Matsushima, Y. Kikkawa, T. Takada, K. Matsuoka, Y. Seki, H. Yoshida, Y. Minegishi, H. Karasuyama, H. Yonekawa, J. Immunol. 185 (4) (2010) 2340.

[68] M.J. Ruddy, G.C. Wong, X.K. Liu, H. Yamamoto, S. Kasayama, K.L. Kirkwood, S.L. Gaffen, J. Biol. Chem. 279 (4) (2004) 2559.

[69] Y. Qian, N. Giltiay, J. Xiao, Y. Wang, J. Tian, S. Han, M. Scott, R. Carter, T.N. Jorgensen, X. Li, Eur. J. Immunol. 38 (8) (2008) 2219.

[70] Y. Chen, P. Thai, Y.H. Zhao, Y.S. Ho, M.M. DeSouza, R. Wu, J. Biol. Chem. 278 (19) (2003) 17036.

[71] F. Huang, C.Y. Kao, S. Wachi, P. Thai, J. Ryu, R. Wu, J. Immunol. 179 (10) (2007) 6504.

[72] F. Shen, S.L. Gaffen, Cytokine 41 (2) (2008) 92.

[73] J. Hartupee, C. Liu, M. Novotny, D. Sun, X. Li, T.A. Hamilton, J. Immunol. 182 (3) (2009) 1660.

[74] F. Shen, N. Li, P. Gade, D.V. Kalvakolanu, T. Weibley, B. Doble, J.R. Woodgett, T.D. Wood, S.L. Gaffen, Sci. Signal. 2 (59) (2009) ra8.

[75] A. Maitra, F. Shen, W. Hanel, K. Mossman, J. Tocker, D. Swart, S.L. Gaffen, Proc. Natl Acad. Sci. USA 104 (18) (2007) 7506.

[76] R.M. Onishi, S.J. Park, W. Hanel, A.W. Ho, A. Maitra, S.L. Gaffen, J Biol Chem.

[77] Y. Shi, S.J. Ullrich, J. Zhang, K. Connolly, K.J. Grzegorzewski, M.C. Barber, W. Wang, K. Wathen, V. Hodge, C.L. Fisher, H. Olsen, S.M. Ruben, I. Knyazev, Y.H. Cho, V. Kao, K.A. Wilkinson, J.A. Carroll, R. Ebner, J. Biol. Chem. 275 (25) (2000) 19167.

[78] M.M. Fort, J. Cheung, D. Yen, J. Li, S.M. Zurawski, S. Lo, S. Menon, T. Clifford, B. Hunte, R. Lesley, T. Muchamuel, S.D. Hurst, G. Zurawski, M.W. Leach, D.M. Gorman, D.M. Rennick, Immunity 15 (6) (2001) 985.

[79] S.D. Hurst, T. Muchamuel, D.M. Gorman, J.M. Gilbert, T. Clifford, S. Kwan, S. Menon, B. Seymour, C. Jackson, T.T. Kung, J.K. Brieland, S.M. Zurawski, R.W. Chapman, G. Zurawski, R.L. Coffman, J. Immunol. 169 (1) (2002) 443.

[80] G. Pan, D. French, W. Mao, M. Maruoka, P. Risser, J. Lee, J. Foster, S. Aggarwal, K. Nicholes, S. Guillet, P. Schow, A.L. Gurney, J. Immunol. 167 (11) (2001) 6559.

[81] M.R. Kim, R. Manoukian, R. Yeh, S.M. Silbiger, D.M. Danilenko, S. Scully, J. Sun, M.L. DeRose, M. Stolina, D. Chang, G.Y. Van, K. Clarkin, H.Q. Nguyen, Y.B. Yu, S. Jing, G. Senaldi, G. Elliott, E.S. Medlock, Blood 100 (7) (2002) 2330.

[82] P. Angkasekwinai, H. Park, Y.H. Wang, Y.H. Wang, S.H. Chang, D.B. Corry, Y.J. Liu, Z. Zhu, C. Dong, J. Exp. Med. 204 (7) (2007) 1509.

[83] T. Tamachi, Y. Maezawa, K. Ikeda, S. Kagami, M. Hatano, Y. Seto, A. Suto, K. Suzuki, N. Watanabe, Y. Saito, T. Tokuhisa, I. Iwamoto, H. Nakajima, J. Allergy Clin
[84] P.G. Fallon, S.J. Ballantyne, N.E. Mangan, J.L. Barlow, A. Dasvarma, D.R. Hewett, A. McIlgorm, H.E. Jolin, A.N. McKenzie, J. Exp. Med. 203 (4) (2006) 1105.

[85] A.M. Owyang, C. Zaph, E.H. Wilson, K.J. Guild, T. McClanahan, H.R. Miller, D.J. Cua, M. Goldschmidt, C.A. Hunter, R.A. Kastelein, D. Artis, J. Exp. Med. 203 (4) (2006) 843.

[86] C.M. Kang, A.S. Jang, M.H. Ahn, J.A. Shin, J.H. Kim, Y.S. Choi, T.Y. Rhim, C.S. Park, Am. J. Respir. Cell Mol. Biol. 33 (3) (2005) 290.

[87] K. Ikeda, H. Nakajima, K. Suzuki, S. Kagami, K. Hirose, A. Suto, Y. Saito, I. Iwamoto, Blood 101 (9) (2003) 3594.

[88] B. Terrier, I. Bieche, T. Maisonobe, I. Laurendeau, M. Roszenzwaig, J.E. Kahn, M.C. Diemert, L. Musset, M. Vidaud, D. Sene, N. Costedoat-Chalumeau, D. Le Thi-Huong, Z. Amoura, D. Klatzmann, P. Cacoub, D. Saadoun. Blood.

[89] Y.H. Wang, P. Angkasekwinai, N. Lu, K.S. Voo, K. Arima, S. Hanabuchi, A. Hippe, C.J. Corrigan, C. Dong, B. Homey, Z. Yao, S. Ying, D.P. Huston, Y.J. Liu, J. Exp. Med. 204 (8) (2007) 1837.

[90] P. Stock, V. Lombardi, V. Kohlrautz, O. Akbari, J. Immunol. 182 (8) (2009) 5116.

[91] S.A. Saenz, M.C. Siracusa, J.G. Perrigue, S.P. Spencer, J.F. Urban, Jr., J.E. Tocker, A.L. Budelsky, M.A. Kleinschek, R.A. Kastelein, T. Kambayashi, A. Bhandoola, D. Artis, Nature 464 (7293) (2010) 1362.

[92] K. Moro, T. Yamada, M. Tanabe, T. Takeuchi, T. Ikawa, H. Kawamoto, J. Furusawa, M. Ohtani, H. Fujii, S. Koyasu, Nature 463 (7280) (2010) 540.

[93] D.R. Neill, S.H. Wong, A. Bellosi, R.J. Flynn, M. Daly, T.K. Langford, C. Bucks, C.M. Kane, P.G. Fallon, R. Pannell, H.E. Jolin, A.N. McKenzie, Nature 464 (7293) (2010) 1367.

[94] A.E. Price, H.E. Liang, B.M. Sullivan, R.L. Reinhardt, C.J. Eisley, D.J. Erle, R.M. Locksley, Proc. Natl. Acad. Sci. U. S. A. 107 (25) (2010) 11489.

[95] V. Dolgachev, B.C. Petersen, A.L. Budelsky, A.A. Berlin, N.W. Lukacs, J. Immunol. 183 (9) (2009) 5705.

[96] E.A. Rickel, L.A. Siegel, B.R. Yoon, J.B. Rottman, D.G. Kugler, D.A. Swart, P.M. Anders, J.E. Tocker, M.R. Comeau, A.L. Budelsky, J. Immunol. 181 (6) (2008) 4299.

[97] P. Angkasekwinai, S.H. Chang, M. Thapa, H. Watarai, C. Dong, Nat. Immunol. 11 (3) (2010) 250.

[98] S. Swaidani, K. Bulek, Z. Kang, C. Liu, Y. Lu, W. Yin, M. Aronica, X. Li, J. Immunol. 182 (3) (2009) 1631.

[99] Y. Maezawa, H. Nakajima, K. Suzuki, T. Tamachi, K. Ikeda, J. Inoue, Y. Saito, I. Iwamoto, J. Immunol. (176(2) (2006) 1013.

[100] Y. Sonobe, H. Takeuchi, K. Kataoka, H. Li, S. Jin, M. Mimuro, Y. Hashizume, Y. Sano, T. Kanda, T. Mizuno, A. Suzumura, J. Biol. Chem. 284 (46) (2009) 31834.
