
Antimicrobial Agents and Chemotherapy

**A Two-Component Regulatory System Interconnects Resistance to Polymyxins, Aminoglycosides, Fluoroquinolones, and β-Lactams in *Pseudomonas aeruginosa***

Cédric Muller, Patrick Plésiat and Katy Jeannot  
*Antimicrob. Agents Chemother.* 2011, 55(3):1211. DOI: 10.1128/AAC.01252-10.  
Published Ahead of Print 13 December 2010.

---

Updated information and services can be found at:  
[http://aac.asm.org/content/55/3/1211](http://aac.asm.org/content/55/3/1211)

These include:

**REFERENCES**  
This article cites 51 articles, 36 of which can be accessed free at: [http://aac.asm.org/content/55/3/1211#ref-list-1](http://aac.asm.org/content/55/3/1211#ref-list-1)

**CONTENT ALERTS**  
Receive: RSS Feeds, eTOCs, free email alerts (when new articles cite this article), [more »](more%20%E2%80%BA)

---

Information about commercial reprint orders: [http://journals.asm.org/site/misc/reprints.xhtml](http://journals.asm.org/site/misc/reprints.xhtml)  
To subscribe to to another ASM Journal go to: [http://journals.asm.org/site/subscriptions/](http://journals.asm.org/site/subscriptions/)

Journals.ASM.org

ANTIMICROBIAL AGENTS AND CHEMOTHERAPY, Mar. 2011, p. 1211–1221
0066-4804/11/$12.00 doi:10.1128/AAC.01252-10
Copyright © 2011, American Society for Microbiology. All Rights Reserved.

# A Two-Component Regulatory System Interconnects Resistance to Polymyxins, Aminoglycosides, Fluoroquinolones, and β-Lactams in *Pseudomonas aeruginosa*

Cédric Muller, Patrick Plésiat,* and Katy Jeannot

*Laboratoire de Bactériologie, Faculté de Médecine, Université de Franche-Comté, 25030 Besançon, France*

Received 10 September 2010/Returned for modification 29 October 2010/Accepted 3 December 2010

Constitutive overexpression of the active efflux system MexXY/OprM is a major cause of resistance to aminoglycosides, fluoroquinolones, and cefepime in clinical strains of *Pseudomonas aeruginosa*. Upregulation of this pump often results from mutations occurring in *mexZ*, the local repressor gene of the *mexXY* operon. In this study, analysis of MexXY-overproducing mutants selected *in vitro* from reference strain PAO1Bes on amikacin (at a concentration 1.5-fold higher than the MIC) led to identification of a new class of mutants harboring an intact *mexZ* gene and exhibiting increased resistance to colistin and imipenem in addition to aminoglycosides, fluoroquinolones, and cefepime. Reverse transcription-quantitative PCR (RT-qPCR) experiments on a selected clone named PAOW2 demonstrated that *mexXY* overexpression was independent of *mexZ* and the PA5471 gene, which is required for drug-dependent induction of *mexXY*. Furthermore, the transcript levels of the *oprD* gene, which encodes the carbapenem-selective porin OprD, were found to be reduced drastically in PAOW2. Whole-genome sequencing revealed a single mutation resulting in an M59I substitution in the ParR protein, the response regulator of the ParRS two-component regulatory system (with ParS being the sensor kinase), which is required for adaptive resistance of *P. aeruginosa* to polycationic peptides such as colistin. The multidrug resistance phenotype was suppressed in PAOW2 by deletion of the *parS* and *parRS* genes and conferred to PAO1Bes by chromosomal insertion of the mutated *parRS* locus from PAOW2. As shown by transcriptomic analysis, only a very small number of genes were expressed differentially between PAOW2 and PAO1Bes, including the lipopolysaccharide (LPS) modification operon *arnBCADTEF-ugd*, responsible for resistance to polycationic agents. Exposure of wild-type PAO1Bes to different polycationic peptides, including colistin, was shown to result in increased *mexY* and repressed *oprD* expression via ParRS, independent of PA5471. In agreement with these results, colistin antagonized activity of the MexXY/OprM substrates in PAO1Bes but not in a Δ*parRS* derivative. Finally, screening of clinical strains exhibiting the PAOW2 resistance phenotype allowed the identification of additional alterations in ParRS. Collectively, our data indicate that ParRS may promote either induced or constitutive multidrug resistance to four different classes of antibiotics through the activation of three distinct mechanisms (efflux, porin loss, and LPS modification).

---

Aminoglycosides are invaluable drugs in the management of patients with acute or chronic infections caused by *Pseudomonas aeruginosa*. Previous studies have shown that an active efflux mechanism implying a tripartite pump called MexXY/OprM modulates the activity of aminoglycosides toward this major nosocomial pathogen (1, 35, 51). The RND (resistance-nodulation-cell division family) transporter MexY interacts with the outer membrane channel OprM and the periplasmic adaptor protein MexX to actively extrude aminoglycosides and various unrelated antibiotics (fluoroquinolones, macrolides, tetracyclines, and zwitterionic β-lactams) from the intracellular compartment to the external medium (31). At the gene level, the MexX and MexY proteins are encoded by a single transcriptional unit, *mexXY*, whereas OprM is encoded by the third gene of a constitutively expressed operon, *mexAB-oprM*, coding for another efflux system (MexAB-OprM) (1, 26,

* Corresponding author. Mailing address: Laboratoire de Bactériologie, Faculté de Médecine, Université de Franche-Comté, 25030 Besançon, France. Phone: (33) 3 81 66 82 86. Fax: (33) 3 81 66 89 14. E-mail: patrick.plesiat@univ-fcomte.fr.
Published ahead of print on 13 December 2010.

TABLE 1. Bacterial strains and plasmids used in this study

| Strain or plasmid | Phenotype or genotype$^a$ | Source or reference |
|-------------------|----------------------------|---------------------|
| **Strains**       |                            |                     |
| **P. aeruginosa strains** |                        |                     |
| PAO1Bes           | Wild-type reference strain | K.Stover            |
| CMZ 091           | PAO1Bes$\Delta$mexZ        | This study          |
| CM090             | PAO1Bes$\Delta$PA5470      | This study          |
| CM093             | PAO1Bes$\Delta$PA5471      | This study          |
| CM095             | PAO1Bes$\Delta$parS        | This study          |
| CM096             | PAO1Bes$\Delta$parRS       | This study          |
| CM099             | PAO1Bes$\Delta$pmrAB       | This study          |
| CM106             | CM096cis complemented with parRS genes from PAOW2, inserted at the attB site; Tc$^r$ | This study          |
| PAOW1             | PAO1Bes spontaneous mutant; Amk$^r$ | This study          |
| PAOW2             | PAO1Bes spontaneous mutant with Met$_{59}$-to-Ile change in ParR; Amk$^r$ | This study          |
| CMZ 089           | PAOW2$\Delta$mexZ          | This study          |
| CM092             | PAOW2$\Delta$PA5470        | This study          |
| CM094             | PAOW2$\Delta$PA5471        | This study          |
| CM097             | PAOW2$\Delta$parS          | This study          |
| CM098             | PAOW2$\Delta$parRS         | This study          |
| CM100             | PAOW2$\Delta$pmrAB         | This study          |
| CMC               | Wild-type susceptible clinical strain | This study          |
| CMCW2             | CMC spontaneous mutant with Leu$_{14}$-to-Gln change in ParS; Amk$^r$ | This study          |
| CM107             | CM096cis complemented with parRS genes from CMCW2, inserted at the attB site; Tc$^r$ | This study          |
| **E. coli strains** |                          |                     |
| S17.1             | recA thi pro hsdR Tra$^+$ Sm$^r$ | 44                  |
| DH5$\alpha$       | F$^{-}$ supE44 endA1 hsdR17(rK$^{-}$ mK$^{-}$) thi-1 recA1 $\Delta$(argF-lacZYA) U169 $\phi$80dlacZ$\Delta$M15 phoA gyrA96 relA1 deoR $\lambda$$^{-}$ | Invitrogen          |
| CC118             | A(ara-leu) araD $\Delta$lacX74 galE galK phoA20 thi-1 rpsE rpoB argE(Am) recA1 | 29                  |
| CC118$\lambda$pir | CC118 lysogenized with $\lambda$pir phage | 12                  |
| HB101             | supE44 hsdS20(rB$^{-}$ mB$^{-}$) recA13 ara-14 proA2 lacY1 galK2 rpsL20 xyl-5 mtl-1 leuB6 thi-1 | 25                  |
| **Plasmids**      |                            |                     |
| pCR2.1            | Cloning vector for PCR products; lacZ$\Delta$ColE1f1ori Ap$^r$ Km$^r$ | Invitrogen          |
| pKNG101           | Suicide vector in P. aeruginosa; sacB Sm$^r$ | 21                  |
| pRK2013           | Helper plasmid; ColE1 ori Tra$^{+}$ Mob$^{+}$ Km$^r$ | 6                   |
| Mini-CTX1         | Self-proficient integration vector with tet, V-FRT-attPMCS, ori, int, and oriT; Tc$^r$ | 14                  |
| pKNG$\Delta$mexZ  | BamHI/ApaI 1.212-kb fragment composed of sequences flanking 5' and 3' ends of mexZ, cloned into pKNG101; Sm$^r$ | This study          |
| pKNG$\Delta$5470  | BamHI/ApaI 1.194-kb fragment composed of sequences flanking 5' and 3' ends of PA5470, cloned into pKNG101; Sm$^r$ | This study          |
| pKNG$\Delta$5471  | BamHI/ApaI 1.241-kb fragment composed of sequences flanking 5' and 3' ends of PA5471, cloned into pKNG101; Sm$^r$ | This study          |
| pKNG$\Delta$parS  | ApaI/ApaI 1.079-kb fragment composed of sequences flanking 5' and 3' ends of parS, cloned into pKNG101; Sm$^r$ | This study          |
| pKNG$\Delta$parRS | ApaI/ApaI 1.045-kb fragment composed of sequences flanking 5' and 3' ends of parRS, cloned into pKNG101; Sm$^r$ | This study          |
| pKNG$\Delta$pmrAB | BamHI/ApaI 1.241-kb fragment composed of sequences flanking 5' and 3' ends of pmrAB, cloned into pKNG101; Sm$^r$ | This study          |
| CTX-W2            | parRS from PAOW2 cloned into mini-CTX1 at SpeI/EcoRV sites; Tc$^r$ | This study          |
| CTX-CW2           | parRS from CMCW2 cloned into mini-CTX1 at SpeI/EcoRV sites; Tc$^r$ | This study          |

$^a$ Antibiotic resistance phenotypes; Tc$^r$, tetracycline resistance; Amk$^r$, amikacin resistance; Sm$^r$, streptomycin resistance; Ap$^r$, ampicillin resistance; Km$^r$, kanamycin resistance. MCS, multiple cloning sites.

or high (2 mM) MgSO₄ concentration (11). Isolation of one-step mutants with increased MexXY/OprM efflux activity was performed by plating 100-μl aliquots of log-phase *P. aeruginosa* cultures (*A*~600~ equal to 1) on MHA supplemented with 6 μg ml⁻¹ of amikacin (at a concentration 1.5-fold higher than the MIC for strain PAO1Bes). Recombinant plasmids were introduced into *P. aeruginosa* strains by triparental matings, using the mobilization properties of the broad-host-range helper plasmid pRK2013 (6). Transconjugants were selected on Pseudomonas isolation agar (PIA; Becton Dickinson) supplemented with appropriate antibiotic concentrations, as follows: for *Escherichia coli*, ampicillin at 100 μg/ml, tetracycline at 15 μg/ml, and streptomycin at 50 μg/ml; and for *P. aeruginosa*, ticarcillin at 150 μg/ml, tetracycline at 200 μg/ml, and streptomycin at 2,000 μg/ml.

**Antibiotic susceptibility testing.** The MICs of selected antibiotics in MHA were determined by recommended agar dilution procedures (5). Bacterial susceptibility in BM 2 glucose minimal medium was determined by the macrodilution broth method at two MgSO₄ concentrations (20 μM and 2 mM), as described previously (52). Growth was assessed after 24 h of incubation at 37°C. ParRS-dependent drug induction of the *mexY* gene was assayed by the disk diffusion method. MHA plates were inoculated with calibrated suspensions of strains PAO1Bes and CM096 as recommended by the CLSI. Disks containing meropenem (MEM; 10 μg), cefepime (FEP; 30 μg), gentamicin (GEN; 15 μg), and ciprofloxacin (CIP; 10 μg) were deposited on the agar surface 3 h after deposition of the colistin (CST; 50 μg) disk (Bio-Rad, Marnes-la-Coquette, France). The checkerboard technique for investigating antibiotic interactions has been described elsewhere (5).

**Drug killing experiments.** Overnight cultures of strain PAO1Bes and its mutant CM096 were diluted 1:50 in fresh MHB containing indolicidin or colistin (at a concentration equivalent to 0.5× MIC) and grown with constant shaking (250 rpm) at 37°C to an *A*~600~ value of 0.5 ± 0.05. The bacteria from 1 ml of culture were then collected by centrifugation (5,000 × g) and resuspended in 1 ml of drug-free MHB. This cell suspension was diluted 1:10 in prewarmed MHB prior to the addition of 16 μg/ml gentamicin (a concentration 8-fold higher than the MIC) and then was reincubated at 37°C. The survivors of gentamicin action were counted on MHA plates inoculated with serial 10-fold dilutions of culture aliquots taken at designated time points.

**Construction of deletion mutants in strain PAO1 and mutant PAOW2.** Single- or multiple-knockout mutants in the *mexZ*, PA5470, PA5471, *parS*, *parRS*, and *pmrAB* genes were constructed by using overlapping PCR and recombination events according to the method reported by Kaniga et al. (21). First, the 5′ and 3′ regions flanking each of the genes were amplified by PCR (ca. 500 bp in length) (a list of the primers used is provided in Table 2) under the following conditions: 5 min at 95°C followed by 30 cycles of 30 s at 95°C, 30 s at 60°C, and 30 s at 72°C, with a final extension step of 5 min at 72°C. The resultant amplicons were then used as templates for overlapping PCR with external pairs of primers (Table 2) to generate the mutagenic DNA fragments. The reaction mixtures contained a 200 μM concentration of each deoxynucleotide triphosphate (dNTP), 6% dimethyl sulfoxide (DMSO), 1× polymerase buffer, a 0.5 μM concentration of each primer, and 0.5 U of BioTaq Red DNA polymerase (Bioline, London, United Kingdom). The amplified products were cloned into plasmid pCR2.1 according to the manufacturer’s instructions (Invitrogen, Carlsbad, CA) and next subcloned into the suicide vector pKNG101 and E. coli CC118Δpir, as BamHI/ApaI or ApaI/ApaI fragments (21). The recombinant plasmids were transferred into *P. aeruginosa* by conjugation, and the deletion mutants were selected on PIA plates containing 5% sucrose and 2,000 μg ml⁻¹ streptomycin. The allelic exchanges were confirmed by PCR. Nucleotide sequencing experiments confirmed deletion of 627 bp, 522 bp, 957 bp, 1,037 bp, 1,815 bp, and 1,941 bp in *mexZ*, PA5470, PA5471, *parS*, *parRS*, and *pmrAB*, respectively.

**Chromosomal complementation with full-length *parRS*.** The *parRS* loci of mutants PAOW2 and CMCW2 were PCR amplified from genomic DNA by using the oligonucleotide pair CloparRSC1/CloparRSC2 (Table 2). The resulting 2,314-bp DNA fragments were cloned into pCR2.1 and next subcloned into the HindIII/SacI restriction sites of plasmid mini-CTX1 (14). The recombinant plasmids were then transferred from *E. coli* CC118 to *P. aeruginosa* strain CM096 (PAO1Δ*parRS*) by conjugation, using PIA plates supplemented with tetracycline. Flippase-promoted excision of the chromosomally integrated FRT cassettes (tetracycline resistance and integrase genes) was achieved by conjugational transfer of plasmid pFLP2 from donor *E. coli* S17.1 to strain CM096 and subsequent selection of the transconjugants on ticarcillin-MHA plates (13). Plasmid pFLP2 was subsequently cured by streaking selected recipient clones on MHA medium supplemented with 5% sucrose.

**Transcriptional profiling.** Triplicate cultures of mutant strain PAOW2 and its parent, PAO1Bes, were grown at 37°C with shaking (1:100 dilution of an overnight culture in 25 ml MHB) to an *A*~600~ value of 1 ± 0.05. The bacteria were collected by centrifugation at 15,000 × g for 1 min. Total RNA was extracted from the pellet by using an RNeasy Plus Mini kit (Qiagen SA, Courtaboeuf, France), treated with DNase (RQ1 RNase-free DNase; Promega, Charbonnières les Bains, France) for 1 h at 37°C, and purified with an RNeasy Mini Elute cleanup kit (Qiagen SA). Ten micrograms of RNA was next retrotranscribed into

---

### TABLE 2. Oligonucleotides used for gene inactivation, gene cloning, and RT-qPCR

| Oligonucleotide | Sequence (5′ → 3′) |
|------------------|--------------------|
| **Oligonucleotides for gene inactivation** |
| *mexZ* |
| PCRI*mexZC1* | CGTAGCGGCTTGAGC TTGTGC |
| PCRI*mexZC2* | GCCCTCACACTGAAC GTCC TCACAAGG |
| PCRI*mexZC3* | TTCAGTGTGAGGGCG TCAATCGTCC |
| PCRI*mexZC4* | GGTTCGTAGATCGGC ATCTC |
| PA5471 |
| PCRI5471C1 | CTCC TGCGCGAGGTCTAC |
| PCRI5471C2 | GGCGGTCGCGGACAA AGGCTTGATGTA |
| PCRI5471C3 | TGTCCGCGACC GCCAGCTGATCTACG |
| PCRI5471C4 | GATCGCCAGGCGTTT GTT |
| PA5470 |
| PCRI5470C1 | CTGGA ACTGCTCGACATCAA |
| PCRI5470C2 | GAAGCTGT CCTCGATT ACC TCGACCTC |
| PCRI5470C3 | TCGAGGACAGCTTCAC TGGGCTCTCG |
| PCRI5470C4 | CCGACCTCTTCCACCT GTT |
| *parS* |
| PCRI*parSC1* | AGGAGGACGATCTCGATCAC |
| PCRI*parSC2* | AGATCTTGGT CGACCAGAAGATCG |
| PCRI*parSC3* | GTCGACCAAGATCTT CGAAACCTTTTCG |
| PCRI*parSC4* | GCGTTGATCCTGTGCTGTC |
| *parRS* |
| PCRI*ParRSC1* | GGATTCAGCTTCTGCTCGAC |
| PCRI*ParRSC2* | AGATCTTCATA GGGTTTCATCGGCTCG |
| PCRI*ParRSC3* | CCCTATGAAGATCTTCG AAACCTTTTCG |
| PCRI*ParRSC4* | GCGTTGATCCTGTGCTGTC |
| *pmrAB* |
| PCRI*PmrABC1* | TGTCGA ACTGACCCAGCTAC |
| PCRI*PmrABC2* | CTTCCA GGTCA CCCATTCCACGGTATC |
| PCRI*PmrABC3* | GGGTGACCTGGAAGTGCAGGTGTT CCT |
| PCRI*PmrABC4* | GTGCTGAGCTCCTCGATCTT |

| **Oligonucleotides for complementation of *parRS*** |
| CloparRSC1 | GAGGGAAAAAGCAGAAGTCACC |
| CloparRSC2 | CGAGGTGTCCCATGCTAGG |

| **Oligonucleotides for RT-qPCR** |
| uvrD1 | GCAGCCTCGCCCTACAGCA |
| uvrD2 | GGATCTGGAAAGTTCTGCTCAGC |
| *mexY*1A | GCAGCCTCGCCCTACAGCA |
| *mexY*1B | GGATCTGGAAAGTTCTGCTCAGC |
| *mexZ*R TA1 | TTACCTCCTCCAGCGGC |
| *mexZ*R TA2 | GTGAGGCGGGCGTTGTG |
| PA5470C1 | CCAAAAGAGGAATCCCAGAAA |
| PA5470C2 | CAGGCAGACCTCGATCTTGT |
| PA5471C1 | TCGAGGTAATCGGAGGAGGTG |
| PA5471C2 | AGGGTCTGCAAACGGATCTC |
| oprDRTC1 | CATCTACC GCA CAAACGATG |
| oprDRTC2 | ACAGAGTTGGCGAGGAAAATC |
| parRSTC1 | CGA ACTGGAGGAAATGGTCT |
| parRSTC2 | ATGCGGATCTGTT CGACCT |
| pmrARTC1 | GATA CCGTGGAA TGGGTGAC |
| pmrARTC2 | GGCTTGGTCAGGTAGTCGTC |
| pmrBR TC1 | ACGA ACTCAACCTCCTGCTG |
| pmrBR TC2 | ATCTGCTCCATCAAGGTGCT |
| PA1559RTC1 | GCAGCAACTGGTGGACTACA |
| PA1559RTC2 | CATGCGGAAGACCAGAAGAT |
| PA1797RTC1 | GGAC CCTTTGCA GATGACTC |
| PA1797RTC2 | CGGAGT GTT CCTGAGAAGC |
| PA2358RTC1 | GTACT GTTCGCGGGAAACAAC |
| PA2358RTC2 | CCTGGGAGCAGGAATATCTGG |
| PA2655RTC1 | GTGCTGGTGTTCCTGTTGG |
| PA2655RTC2 | CGTAGGTCCCCCAGATCG |
| PA3554RTC1 | GTGGCTCGAATACCATGTGA |
| PA3554RTC2 | TGCCGTATTTCACGCAGTAG |
| PA4773RTC1 | CAGTGGATCGAGGAAAGCAT |
| PA4773RTC2 | GTACTCCGGCCAGGTATGG |
| PA4774RTC1 | CGCGGAACCACTTCTATTTTC |
| PA4774RTC2 | TCGTGGTACAGCGACTCATC |

TABLE 3. Drug susceptibilities of various mutants derived from strains PAO1Bes and CMC

| Strain       | Genotype                          | GEN | AMK | TOB | APR | FEP | CIP | IMP | MEM | CST |
|--------------|-----------------------------------|-----|-----|-----|-----|-----|-----|-----|-----|-----|
| PAO1Bes      | Wild type                         | 2   | 4   | 1   | 16  | 4   | 0.25| 2   | 1   | 1   |
| CMC          | Wild type                         | 2   | 4   | 1   | 16  | 4   | 0.25| 2   | 1   | 1   |
| PAOW1        | PAO1Bes agrW1 mutant              | 16  | 32  | 4   | 64  | 16  | 1   | 2   | 1   | 1   |
| PAOW2        | PAO1Bes agrW2 mutant              | 8   | 32  | 2   | 64  | 8   | 1   | 16  | 4   | 2   |
| CMCW2        | CMC agrW2 mutant                  | 4   | 8   | 2   | 32  | 8   | 0.5 | 16  | 4   | 2   |
| CMZ091       | PAO1 ΔmexZ                        | 8   | 16  | 2   | 32  | 8   | 1   | 2   | 1   | 1   |
| CM090        | PAO1 ΔPA5470                      | 2   | 4   | 1   | 16  | 4   | 0.25| 2   | 1   | 1   |
| CM093        | PAO1 ΔPA5471                      | ≤0.12| 2   | 0.5 | 4   | 4   | 0.12| 2   | 1   | 1   |
| CM095        | PAO1 ΔparS                        | 2   | 4   | 1   | 16  | 4   | 0.25| 2   | 1   | 0.5 |
| CM096        | PAO1 ΔparRS                       | 2   | 4   | 1   | 16  | 4   | 0.25| 2   | 1   | 0.5 |
| CM099        | PAO1 ΔpmrAB                       | 2   | 4   | 1   | 16  | 4   | 0.25| 2   | 1   | 0.5 |
| CMZ089       | PAOW2 ΔmexZ                       | 8   | 32  | 2   | 64  | 8   | 1   | 16  | 4   | 2   |
| CM092        | PAOW2 ΔPA5470                     | 8   | 32  | 2   | 64  | 8   | 1   | 16  | 4   | 2   |
| CM094        | PAOW2 ΔPA5471                     | 8   | 32  | 2   | 64  | 8   | 1   | 16  | 4   | 2   |
| CM097        | PAOW2 ΔparS                       | 2   | 4   | 1   | 16  | 4   | 0.25| 2   | 1   | 0.5 |
| CM098        | PAOW2 ΔparRS                      | 2   | 4   | 1   | 16  | 4   | 0.25| 2   | 1   | 0.5 |
| CM100        | PAOW2 ΔpmrAB                      | 8   | 32  | 2   | 64  | 8   | 1   | 16  | 4   | 1   |
| CM106        | CM096 with parRS from PAOW2       | 4   | 8   | 1   | 32  | 8   | 0.5 | 16  | 4   | 1   |
| CM107        | CM096 with parRS from CMCW2       | 4   | 8   | 1   | 32  | 8   | 0.5 | 16  | 2   | 1   |

* MICs in bold (or underlined) indicate that the bacteria were at least 2-fold more (or less) resistant than the reference strain PAO1Bes. The data are representative of four distinct experiments. Abbreviations: GEN, gentamicin; AMK, amikacin; TOB, tobramycin; APR, apramycin; FEP, cefepime; CIP, ciprofloxacin; IMP, imipenem; MEM, meropenem; and CST, colistin.

FIG. 1. Mutant strain PAOW2 upregulates the mexXY operon independently of the mexZ, PA5470, and PA5471 genes. mexY gene levels in strains PAO1Bes and PAOW2 and in their respective ΔmexZ (CMZ091 and CMZ089), ΔPA5470 (CM090 and CM092), and ΔPA5471 (CM093 and CM094) mutants were quantified by RT-qPCR. The bars represent means of three independent determinations ± standard deviations (SD).

in PAOW2. We thus constructed mexZ, PA5470, and PA5471 deletion mutants of PAO1Bes and PAOW2. As expected, inactivation of the repressor gene mexZ in PAO1Bes (mutant strain CMZ091) resulted in a strong increase in mexY expression and in more resistance to all of the MexXY/OprM substrates tested (Fig. 1 and Table 3). In PAOW2, however, deletion of mexZ (mutant strain CMZ089) upregulated mexY expression only 2-fold, without notable effects on the mutant’s susceptibility (Fig. 1 and Table 3). Concordant with data from the literature (36), inactivation of PA5471 (mutant strain CM093), but not that of the adjacent gene PA5470 (mutant strain CM090), significantly reduced mexY expression in wild-type PAO1Bes, leading to super-susceptibility to most of the antibiotics exported by MexXY/OprM (compare strains CM090, CM093, and PAO1Bes in Fig. 1 and Table 3). In contrast, suppression of PA5471 had no influence on the mexY level and on drug susceptibility in PAOW2 (compare strains CM094 and PAOW2). Altogether, these results suggested the existence of a new regulatory pathway for MexXY/OprM that is able to override the control exerted by MexZ and PA5471.

It is well established that the OprD porin is the specific uptake pathway for basic amino acids and carbapenems across the outer membrane of *P. aeruginosa* (47). Since qualitative and/or quantitative alterations of this channel are a major cause of resistance to carbapenems in this organism (24, 27, 38, 42), we sequenced and measured the expression levels of the oprD gene in PAOW2 and PAO1Bes. PAOW2 turned out to harbor an intact oprD gene whose expression was 7.1-fold ± 0.1-fold less than that in PAO1Bes (Fig. 2). Reminiscent of the case for nfxC mutants, which overproduce the efflux system MexEF-OprN while downregulating OprD (23, 39), our results provided good evidence for coordinated regulation between two complementary resistance mechanisms aimed at limiting intracellular drug accumulation (i.e., active efflux and outer membrane impermeability) in the agrW2 mutants.

Assuming that the elevated rates at which the agrW2 mutants

FIG. 2. Impact of mutations in parR and parS on expression levels of the mexY and oprD genes. The mRNA levels of the mexY (black bars) and oprD (white bars) genes were determined by RT-qPCR. The bars represent means of three independent determinations ± SD.

TABLE 4. Microarray analysis of differentially expressed genes (at least 2-fold) in the PAOW2 mutant compared with PAO1Bes

| Gene ID | Name (alias) | Fold change | Description |
|---------|--------------|-------------|-------------|
| PA0958  | oprD         | -7.5        | OprD porin   |
| PA1559  |              | 28.9        | Hypothetical protein |
| PA1560  |              | 14.3        | Hypothetical protein |
| PA1797  |              | 45.7        | Hypothetical protein |
| PA2018  | mexY         | 4.6         | RND transporter MexY |
| PA2358  |              | 25.5        | Hypothetical protein |
| PA2655  |              | 33.6        | Hypothetical protein |
| PA3552  | arnB(pmrH)   | 11.0        | Hypothetical protein |
| PA3553  | arnC(pmrF)   | 11.6        | Probable glycosyl transferase |
| PA3554  | arnA(pmrI)   | 15.3        | Hypothetical protein |
| PA3556  | arnT(pmrK)   | 8.5         | 4-Amino-4-deoxy-L-arabinose transferase |
| PA3557  | arnE(pmrL)   | 3.1         | Hypothetical protein |
| PA3558  | arnF(pmrM)   | 9.2         | Hypothetical protein |
| PA3559  | ugd(pmrE)    | 16.1        | Probable UDP-glucose dehydrogenase |
| PA4773  |              | 36.8        | Hypothetical protein |
| PA4774  |              | 12.5        | Hypothetical protein |
| PA4775  |              | 7.1         | Hypothetical protein |

---

TABLE 5. Influence of ParRS on expression of selected genes in strain PAO1Bes and mutant strain PAOW2

| Gene                | Fold change in gene expression<sup>a</sup> |
|---------------------|-------------------------------------------|
|                     | PAOW2/PAO1Bes                             | CM096/PAO1Bes                          |
| PA0958 (oprD)       | -7.1 ± 0.1                               | 1.1 ± 0.3                              |
| PA1559              | 50.1 ± 14                                | 1.6 ± 0.3                              |
| PA1797              | 315.5 ± 83                               | 1.1 ± 0.5                              |
| PA1799 (parR)       | -1.2 ± 0.4                               |                                        |
| PA2018 (mexY)       | 13.1 ± 2.8                               | -1.1 ± 0.1                             |
| PA2358              | 27.8 ± 5.6                               | 1.9 ± 0.1                              |
| PA2655              | 63.7 ± 14.1                              | 1.2 ± 0.3                              |
| PA3554 (arnA)       | 50.6 ± 8.2                               | 2.3 ± 0.4                              |
| PA4773              | 90.6 ± 15.7                              | 1.6 ± 0.2                              |
| PA4774              | 131.3 ± 25.5                             | 1.5 ± 0.5                              |
| PA4777 (pmrB)       | 15.9 ± 2.6                               | 1.4 ± 0.2                              |

<sup>a</sup> Values are means of three independent determinations ± SD.

FIG. 3. Influence of antibiotic stress on *mexY* and *oprD* expression. Changes in expression levels of the *mexY* (A) and *oprD* (B) genes were assessed by RT-qPCR for wild-type strain PAO1Bes (black bars), ΔPA5471 mutant strain CM093 (white bars), and Δ*parRS* mutant strain CM096 (gray bars), which were preincubated for 4 h in the presence of indolicidin, colistin, polymyxin B, PMBN, or gentamicin. An uninduced culture was performed in parallel as a negative control. Values are means of three independent determinations ± SD.

by mutations affecting either the response regulator ParR or the sensor kinase ParS.

ParRS mediates multidrug resistance in *P. aeruginosa*. Since the *oprD* and *mexY* genes are constitutively down- and upregulated in the PAOW2 mutant, respectively, we considered the possibility that polycationic peptides might induce increased resistance of wild-type *P. aeruginosa* strains to carbapenems, aminoglycosides, cefepime, and fluoroquinolones through their activation of ParRS. This hypothesis was tested first by measuring the transcriptional levels of the *mexY* and *oprD* genes in bacteria cultured for 4 h in the presence of a concentration equivalent to a 1:2 ratio of the MIC of colistin (0.5 μg/ml), indolicidin (4 μg/ml), polymyxin B (0.5 μg/ml), or polymyxin B nonapeptide (PMBN) (64 μg/ml) (Fig. 3A and B). The latter compound, which lacks the fatty acid tail of polymyxin B, has lost most of the antibacterial properties of its parent molecule on *P. aeruginosa* but retains a strong outer membrane-permeabilizing activity (40, 48, 49). Because the PA5471 gene is required for drug-induced expression of *mexXY* in wild-type bacteria (36), the induction experiments were carried out with mutants lacking either PA5471 (CM093) or *parRS* (CM096), in addition to PAO1Bes. Exposure of PAO1Bes to colistin, the most therapeutically relevant cationic peptide from this series, was associated with substantial changes in *mexY* and *oprD* expression compared with that in untreated cells (4.6-fold ± 0.6-fold and −2.5-fold ± 0.35-fold, respectively). Suppression of the PA5471 gene (mutant strain CM093) had no influence on these transcript levels (4.2-fold ± 0.7-fold and −2.6-fold ± 0.2-fold, respectively), indicating a PA5471-independent regulation pathway for *mexXY* by colistin (Fig. 3A and B). On the other hand, the colistin-induced responses of *mexY* and *oprD* were completely abolished in the *parRS* null mutant CM096 (−1.1-fold ± 0.15-fold and 1.29-fold ± 0.1-fold changes in expression, respectively) (Fig. 3A and B and 4). Similar results were obtained with indolicidin and polymyxin B. Notably, despite the use of a fairly high concentration (64 μg/ml), PMBN increased *mexY* activity only marginally (Fig. 3 and 4).

The possibility that gentamicin might trigger *mexXY* expression via ParRS was also examined by culturing PAO1Bes, CM093, and CM096 in the presence of a concentration of the

antibiotic equivalent to 0.5× MIC (1 μg/ml, 0.06 μg/ml, and 1 μg/ml, respectively). Confirming previous results from our laboratory (19), sub-MIC gentamicin markedly induced *mexY* gene expression (18.5-fold ± 1.7-fold) in wild-type PAO1Bes compared with untreated cells. However, this adaptive response was independent of *parRS*, as the *mexY* levels of CM096 (13.2-fold ± 2-fold) were not really different from those of PAO1Bes. In agreement with this, using a pPA3552::lux fusion, Fernandez et al. reported no effect of aminoglycosides on ParRS-dependent activation of the LPS modification operon *armBCADTEF-ugd* (9). Thus, ParRS is unlikely to respond to the presence of aminoglycosides in the external medium. Similarly, imipenem (2 μg/ml), meropenem (0.5 μg/ml), and cefepime (2 μg/ml) did not show any ParRS-dependent effects on *mexY* and *oprD* expression (data not presented).

FIG. 4. Interactions of polymyxin B and PMBN with gentamicin. Antibiograms were created for strain PAO1Bes (A) and Δ*parRS* mutant strain CM096 (B) on MHA according to CLSI recommendations. PB, polymyxin B; PMBN, polymyxin B nonapeptide. As shown in the upper part of panel A, polymyxin B antagonizes the bacteriostatic activity of gentamicin (GM) on PAO1Bes.

ria. Of greater importance, such a colistin-induced adaptation was not observed in mutant strain CM096, which exhibited a decline in living cells averaging $-3.6 \pm 0.2 \log_{10}$ after 30 min of gentamicin treatment. Similar results were obtained with amikacin and tobramycin at concentrations 8-fold higher than the MIC. If confirmed *in vivo* by clinical trials, these data would preclude the use of combined therapy with colistin and aminoglycosides in the management of CF lung infection.

Clinical strains with alterations in ParRS. To gain insight into the clinical relevance of *parRS* mutants, we screened our laboratory collection in search of CF and non-CF isolates showing a typical AgrW2 resistance phenotype. One CF (3020R) and two non-CF (4922 and 5024) isolates over 6 strains were found to fit this phenotypic profile. Subsequent determination of the colistin MIC demonstrated a modest but reproducible resistance to this agent in the three isolates (MIC of 2 μg/ml). Compared to reference strain PAO1Bes, all of the bacteria appeared to overexpress *mexY* (10.6-fold ± 1.6-fold, 4.7-fold ± 0.5-fold, and 7-fold ± 0.7-fold, respectively) and to downregulate *oprD* ($-5.2$-fold ± 0.5-fold, $-6.4$-fold ± 0.8-fold, and $-5.7$-fold ± 0.6-fold, respectively), as assayed by RT-qPCR. Interestingly, sequence analysis of the *parRS* loci from these strains showed nonsynonymous nucleotide substitutions when the sequences were aligned with the PAO1 reference genome (http://v2.pseudomonas.com). These changes were predicted to lead to single amino acid substitutions in the sensor kinase ParS (V101M in isolate 4922 and L137P in isolate 5024) or the response regulator ParR (E156K in isolate 3020R). In support of a role of these alterations in the constitutive activation of ParRS, we noted that the gene flanking the *parRS* operon, PA1797, was overexpressed in the selected strains (108.2-fold ± 23.3-fold in 3020R, 33.6-fold ± 7.5-fold in 4922, and 4.6-fold ± 0.3-fold in 5024), as observed in mutant strain PAOW2.

Conclusions. Among the numerous two-component regulatory systems (64 response regulators, 63 classical histidine kinases, and 16 atypical kinases) possessed by *P. aeruginosa* (41), only a few have been reported to provide significant antibiotic resistance in clinical strains (e.g., PmrB and PhoQ) (3, 43). To our knowledge, this study is the first showing the role played by a two-component signal transduction system (ParRS) in bacterial adaptation to no fewer than four different classes of antibiotics (polymyxins, aminoglycosides, fluoroquinolones, and β-lactams) through three distinct resistance mechanisms (i.e., LPS modification, increased drug efflux, and a reduced porin pathway). Such a multiresistance phenotype may be induced by bacterial exposure to polycationic peptides or may be inherited stably as a result of mutations affecting the sensor kinase ParS or the response regulator ParR. Adding to the recent observation that ParRS mediates the adaptive response of *P. aeruginosa* to polycations such as polymyxin B, colistin, and indolicidin at Mg²⁺ concentrations similar to those found in the human body (1 to 2 mM) (9), this work demonstrates that when it is activated, ParRS also leads to up- and down-regulation of the efflux system MexXY/OprM and the porin OprD, respectively. Some of our data strongly suggest that the transcriptional regulator ParR is activated rather than inhibited by ParS once a specific signal is detected in the bacterial environment. Indeed, deletion of *parS* or *parRS* did not affect antibiotic resistance in wild-type bacteria, except for inducers

FIG. 5. ParRS is required for colistin-induced resistance to the killing activity of gentamicin. Survivors of wild-type strain PAO1Bes (triangles) and its Δ*parRS* derivative CM096 (circles) in the presence of 16 μg/ml gentamicin were determined in triplicate after 4 h of preincubation in drug-free MHB (open symbols) or MHB supplemented with a concentration equivalent to a 1:2 ratio of the MIC of colistin.

Role of ParRS in antagonistic drug interactions. Collectively, the results mentioned above suggested that subinhibitory concentrations of polycationic peptides might promote antagonistic interactions between antipseudomonal antibiotics through distinct mechanisms, such as outer membrane impermeability and active drug efflux (Fig. 4). To demonstrate this, checkerboard assays were performed with wild-type PAO1Bes and its Δ*parRS* mutant CM096 grown in MHB. Consistent with our gene expression experiments, colistin at concentrations near the MIC strongly antagonized the bacteriostatic activities of gentamicin, meropenem, cefepime, and ciprofloxacin in PAO1Bes (fractional inhibitory concentration [FIC] indexes of 2.5, 4.5, 4.5, and 4.5, respectively) but not in CM096 (FIC indexes of 1, 1, 1.5, and 1, respectively). The FIC index reflects synergism (value of ≤0.5), additivity (value of 0.51 to 0.99), indifference (value of 1 to 2), or antagonism (value of >2) between the bacteriostatic effects of two drugs. Such an antagonistic effect of colistin toward aminoglycosides is of considerable clinical importance for cystic fibrosis. Patients suffering from this inherited genetic disease are indeed treated over long periods with aerosols of colistin and aminoglycosides, often given sequentially, when they are colonized by *P. aeruginosa* in the lungs (10). These patients are also used to receiving repeated courses of intravenous aminoglycosides during infectious exacerbations. To better evaluate the negative impact of colistin on aminoglycoside efficacy, we carried out time-kill experiments with gentamicin at 16 μg/ml (a concentration 8-fold higher than the MIC), using PAO1Bes and CM096 preincubated for 4 h with or without 0.5 μg/ml of colistin (equivalent to a 1:2 ratio of the MIC). Under noninducing conditions, PAO1Bes and its mutant CM096 exhibited similar killing profiles, with 3 orders of reduction in survivors after 30 min of gentamicin exposure (Fig. 5). As anticipated from our checkerboard assays, preincubation with colistin drastically reduced killing of PAO1Bes ($-1.15$-fold ± 0.2 log₁₀-fold), confirming the development of a refractory state in the bacter-

Vol. 55, 2011

ParRS-MEDIATED MULTIRESISTANCE IN P. AERUGINOSA 1219

FIG. 6. Schematic representation of regulation of the *mexXY* operon in *P. aeruginosa*. (A) Wild-type strain grown in drug-free medium. During bacterial growth, a translation attenuation mechanism involving a small peptide, PA5471.1, prevents transcription of PA5471, a downstream gene of unknown function whose product directly or indirectly modulates the activity of MexZ, the local repressor of the *mexXY* operon. (B) Growth of wild-type strain in the presence of *MexXY*-inducing antibiotics. Following bacterial exposure to ribosome-targeting antimicrobials, *MexXY* is produced at high levels conferring intrinsic resistance to the inducer, provided that it is a good substrate for the efflux pump (as is the case for aminoglycosides but not for chloramphenicol). This *MexZ*-dependent induction process is linked in part to increased levels of the PA5471 protein modulating *MexZ* repressor activity. A second pathway leading to drug-dependent overexpression of *mexXY* is independent of both the *MexZ* and PA5471 proteins. Our results demonstrate that polycationic antibiotics (colistin and polymyxin B) are able to promote expression of the *pmrAB*, *arnBCDTEF-ugd*, and *mexXY* operons and to coordinately downregulate the *oprD* gene through the activation of ParRS. This regulation results in multidrug resistance.

such as colistin. Overexpression of plasmid-borne wild-type *parRS* genes did not result in increased resistance either (data not shown). The nature of the signal sensed by the regulatory system remains to be identified. However, this signal is unlikely to be related to disorganization of the outer membrane architecture, since (i) some polycationic peptides (e.g., CP-28 and LL-37) were unable to activate ParRS-dependent expression of the *arnBCDTEF-ugd* operon (9) and (ii) PMBN proved to be a much weaker inducer of *mexY* expression than polymyxin B, despite their having similar outer membrane-permeabilizing capabilities (Fig. 5). Undoubtedly, identification of the molecular signal perceived by ParS is a prerequisite for the development of innovative molecules able to block the adaptive resistance of *P. aeruginosa* to polycations.

The molecular basis of ParRS-dependent regulation of *MexXY/OprM* and *OprD* is another crucial issue that should be addressed in order to understand and try to prevent the adaptation of *P. aeruginosa* to antibiotic stress. A search for motif elicitation in the promoter regions upstream of the genes responding to ParRS activation in PAOW2 by use of MEME software (2) identified a predicted region of 40 nucleotides (*P* < 10⁻¹⁰) upstream of *mexY*, *oprD*, PA1797, PA2358, and

PA2655. Thus, it is tempting to assume that *MexXY/OprM* and *OprD* act in concert to limit (i.e., by active efflux and by outer membrane impermeability) the intracellular accumulation of a toxic substrate that directly or indirectly results from the antibacterial action of polycationic peptides. Since genes coding for putative homologs of polyamine biosynthetic/degradative enzymes (e.g., PA4773 and PA4774) are overexpressed in PAOW2 or under polycation exposure, we looked at whether their inactivation would normalize the expression of *mexY* and *oprD* to baseline levels. The deletion of the PA4773 gene, which is predicted to encode an adenosylmethionine decarboxylase, failed to produce the expected results (data not shown). In the same line, plasmid-mediated overexpression or deletion of the PA2358 and PA2655 genes, whose functions are virtually unknown, had no influence on *mexY* or *oprD* expression in PAO1Bes (not presented). A search for a toxic metabolite recognized by both *MexXY/OprM* and *OprD* is currently underway in our laboratory.

Finally, this work demonstrates the existence of at least two novel types of *agrW* mutants in addition to those exhibiting alterations in the ribosomal machinery (8, 19). A preliminary study conducted on 94 non-CF, *MexXY*-overproducing clinical

strains showed that 21 (22%) were *agrW* mutants and 73 (78%) were of the *agrZ* type. The characterization of these mutants is in progress. The involvement of some of the *agrW* isolates in severe infections (e.g., bacteremia) suggests that they are fully virulent. From a physiological perspective, the active efflux system MexXY/OprM now appears to be a key element of bacterial adaptation to antibiotics targeting the ribosome (aminoglycosides, tetracyclines, macrolides, and chloramphenicol) or the cellular envelope (colistin and polymyxin B). Figure 6 summarizes our current knowledge of the complex regulation of MexXY.

# ACKNOWLEDGMENTS

We are grateful to Fabrice Poncet and Barbara Dehecq of the Faculty of Medicine, Université de Franche-Comté, Besançon, France, for their technical assistance.

Funding was obtained from the Ministère de l'Enseignement Supérieur et de la Recherche.

# REFERENCES

1. Aires, J. R., T. Köhler, H. Nikaido, and P. Plésiat. 1999. Involvement of an active efflux system in the natural resistance of *Pseudomonas aeruginosa* to aminoglycosides. Antimicrob. Agents Chemother. **43**:2624–2628.
2. Bailey, T. L., and C. Elkan. 1994. Fitting a mixture model by expectation maximization to discover motifs in biopolymers. Proc. Int. Conf. Intell. Syst. Mol. Biol. **2**:28–36.
3. Barrow, K., and D. H. Kwon. 2009. Alterations in two-component regulatory systems of PhoPQ and PmrAB are associated with polymyxin B resistance in clinical isolates of *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. **53**:5150–5154.
4. Caughlan, R. E., et al. 2009. Fmt bypass in *Pseudomonas aeruginosa* causes induction of MexXY efflux pump expression. Antimicrob. Agents Chemother. **53**:5015–5021.
5. Clinical and Laboratory Standards Institute (CLSI). 2006. Performance standards for antimicrobial susceptibility testing. CLSI document M100–S16, 16th ed. CLSI, Wayne, PA.
6. Ditta, G., S. Stanfield, D. Corbin, and D. R. Helinski. 1980. Broad-host-range DNA cloning system for gram-negative bacteria: construction of a gene bank of *Rhizobium meliloti*. Proc. Natl. Acad. Sci. U. S. A. **77**:7347–7351.
7. Dumas, J. L., C. van Delden, K. Perron, and T. Köhler. 2006. Analysis of antibiotic resistance gene expression in *Pseudomonas aeruginosa* by quantitative real-time-PCR. FEMS Microbiol. Lett. **254**:217–225.
8. El’Garch, F., K. Jeannot, D. Hocquet, C. Llanes-Barakat, and P. Plésiat. 2007. Cumulative effects of several nonenzymatic mechanisms on the resistance of *Pseudomonas aeruginosa* to aminoglycosides. Antimicrob. Agents Chemother. **51**:1016–1021.
9. Fernandez, L., et al. 2010. Adaptive resistance to the “last hope” antibiotics polymyxin B and colistin in *Pseudomonas aeruginosa* is mediated by the novel two-component regulatory system ParR-ParS. Antimicrob. Agents Chemother. **54**:3372–3382.
10. Flume, P. A., et al. 2007. Cystic fibrosis pulmonary guidelines: chronic medications for maintenance of lung health. Am. J. Respir. Crit. Care Med. **176**:957–969.
11. Gilleland, H. E., J. D. Stinnett, and R. G. Eagon. 1974. Ultrastructural and chemical alteration of the cell envelope of *Pseudomonas aeruginosa*, associated with resistance to ethylene diamine tetra acetate resulting from growth in a Mg²⁺ deficient medium. J. Bacteriol. **117**:302–311.
12. Herrero, M., V. de Lorenzo, and K. N. Timmis. 1990. Transposon vectors containing non-antibiotic resistance selection markers for cloning and stable chromosomal insertion of foreign genes in gram-negative bacteria. J. Bacteriol. **172**:6557–6567.
13. Hoang, T. T., R. R. Karkhoff-Schweizer, A. J. Kutchma, and H. P. Schweizer. 1998. A broad-host-range Flp-FRT recombination system for site-specific excision of chromosomally-located DNA sequences: application for isolation of unmarked *Pseudomonas aeruginosa* mutants. Gene **212**:77–86.
14. Hoang, T. T., A. J. Kutchma, A. Becher, and H. P. Schweizer. 2000. Integration-proficient plasmids for *Pseudomonas aeruginosa*: site-specific integration and use for engineering of reporter and expression strains. Plasmid **43**:59–72.
15. Hocquet, D., et al. 2007. *Pseudomonas aeruginosa* may accumulate drug resistance mechanisms without losing its ability to cause bloodstream infections. Antimicrob. Agents Chemother. **51**:3531–3536.
16. Irizarry, R. A., et al. 2003. Summaries of Affymetrix GeneChip probe level data. Nucleic Acids Res. **31**:15.
17. Irizarry, R. A., et al. 2003. Exploration, normalization, and summaries of high density oligonucleotide array probe level data. Biostatistics **4**:249–264.
18. Islam, S., S. Jalal, and B. Wretlind. 2004. Expression of the MexXY efflux pump in amikacin-resistant isolates of *Pseudomonas aeruginosa*. Clin. Microbiol. Infect. **10**:877–883.
19. Jeannot, K., M. L. Sobel, F. El Garch, K. Poole, and P. Plésiat. 2005. Induction of the MexXY efflux pump in *Pseudomonas aeruginosa* is dependent on drug-ribosome interaction. J. Bacteriol. **187**:5341–5346.
20. Jo, J. T., F. S. Brinkman, and R. E. Hancock. 2003. Aminoglycoside efflux in *Pseudomonas aeruginosa*: involvement of novel outer membrane proteins. Antimicrob. Agents Chemother. **47**:1101–1111.
21. Kaniga, K., I. Delor, and G. R. Cornelis. 1991. A wide-host-range suicide vector for improving reverse genetics in gram-negative bacteria: inactivation of the *blaA* gene of *Yersinia enterocolitica*. Gene **109**:137–141.
22. Klockgether, J., et al. 2010. Genome diversity of *Pseudomonas aeruginosa* PAO1 laboratory strains. J. Bacteriol. **192**:1113–1121.
23. Köhler, T., S. F. Epp, L. K. Curty, and J.-C. Pechère. 1999. Characterization of MexT, the regulator of the MexE-MexF-OprN multidrug efflux system of *Pseudomonas aeruginosa*. J. Bacteriol. **181**:6300–6305.
24. Köhler, T., M. Michéa-Hamzehpour, S. F. Epp, and J. C. Pechère. 1999. Carbapenem activities against *Pseudomonas aeruginosa*: respective contributions of OprD and efflux systems. Antimicrob. Agents Chemother. **43**:424–427.
25. Lacks, S., and B. Greenberg. 1977. Complementary specificity of restriction endonucleases of *Diplococcus pneumoniae* with respect to DNA methylation. J. Mol. Biol. **114**:153–168.
26. Li, X. Z., H. Nikaido, and K. Poole. 1995. Role of MexA-MexB-OprM in antibiotic efflux in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. **39**:1948–1953.
27. Livermore, D. M. 2002. Multiple mechanisms of antimicrobial resistance in *Pseudomonas aeruginosa*: our worst nightmare? Clin. Infect. Dis. **34**:634–640.
28. Llanes, C., et al. 2004. Clinical strains of *Pseudomonas aeruginosa* overproducing MexAB-OprM and MexXY efflux pumps simultaneously. Antimicrob. Agents Chemother. **48**:1797–1802.
29. Manoil, C., and J. Beckwith. 1985. *TnphoA*: a transposon probe for protein export signals. Proc. Natl. Acad. Sci. U. S. A. **82**:8129–8133.
30. Masuda, N., et al. 2000. Contribution of the MexX-MexY-OprM efflux system to intrinsic resistance in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. **44**:2242–2246.
31. Masuda, N., et al. 2000. Substrate specificities of MexAB-OprM, MexCD-OprJ, and MexXY-OprM efflux pumps in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. **44**:3322–3327.
32. Matsuo, Y., S. Eda, N. Gotoh, E. Yoshihara, and T. Nakae. 2004. MexZ-mediated regulation of *mexXY* multidrug efflux pump expression in *Pseudomonas aeruginosa* by binding on the *mexZ-mexX* intergenic DNA. FEMS Microbiol. Lett. **238**:23–28.
33. McPhee, J. B., et al. 2006. Contribution of the PhoP-PhoQ and PmrA-PmrB two-component regulatory systems to Mg²⁺-induced gene regulation in *Pseudomonas aeruginosa*. J. Bacteriol. **188**:3995–4006.
34. McPhee, J. B., S. Lewenza, and R. E. Hancock. 2003. Cationic antimicrobial peptides activate a two-component regulatory system, PmrA-PmrB, that regulates resistance to polymyxin B and cationic antimicrobial peptides in *Pseudomonas aeruginosa*. Mol. Microbiol. **50**:205–217.
35. Mine, T., Y. Morita, A. Kataoka, T. Mizushima, and T. Tsuchiya. 1999. Expression in *Escherichia coli* of a new multidrug efflux pump MexXY from *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. **43**:415–417.
36. Morita, Y., M. L. Sobel, and K. Poole. 2006. Antibiotic inducibility of the MexXY multidrug efflux system of *Pseudomonas aeruginosa*: involvement of the antibiotic-inducible PA5471 gene product. J. Bacteriol. **188**:1847–1855.
37. Moskowitz, S. M., R. K. Ernst, and S. I. Miller. 2004. PmrAB, a two-component regulatory system of *Pseudomonas aeruginosa* that modulates resistance to cationic antimicrobial peptides and addition of aminoarabinose to lipid A. J. Bacteriol. **186**:575–579.
38. Mushtaq, S., Y. Ge, and D. M. Livermore. 2004. Doripenem versus *Pseudomonas aeruginosa* in vitro: activity against characterized isolates, mutants, and transconjugants and resistance selection potential. Antimicrob. Agents Chemother. **48**:3086–3092.
39. Ochs, M. M., M. P. McCusker, M. Bains, and R. E. Hancock. 1999. Negative regulation of the *Pseudomonas aeruginosa* outer membrane porin OprD selective for imipenem and basic amino acids. Antimicrob. Agents Chemother. **43**:1085–1090.
40. Plésiat, P., J. R. Aires, C. Godard, and T. Köhler. 1997. Use of steroids to monitor alterations in the outer membrane of *Pseudomonas aeruginosa*. J. Bacteriol. **179**:7004–7010.
41. Rodrigue, A., Y. Quentin, A. Lazdunski, V. Mejean, and M. Foglino. 2000. Two-component systems in *Pseudomonas aeruginosa*: why so many? Trends Microbiol. **8**:498–504.
42. Satake, S., H. Yoneyama, and T. Nakae. 1991. Role of OmpD2 and chromosomal beta-lactamase in carbapenem resistance in clinical isolates of *Pseudomonas aeruginosa*. J. Antimicrob. Chemother. **28**:199–207.
43. Schurek, K. N., et al. 2009. Involvement of PmrAB and PhoPQ in polymyxin B adaptation and inducible resistance in non-cystic fibrosis clinical isolates of *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. **53**:4345–4351.

44. Simon, R., M. O'Connell, M. Labes, and A. Puhler. 1986. Plasmid vectors for the genetic analysis and manipulation of *Rhizobia* and other gram-negative bacteria. Methods Enzymol. 118:640–659.
45. Sobel, M. L., G. A. McKay, and K. Poole. 2003. Contribution of the MexXY multidrug transporter to aminoglycoside resistance in *Pseudomonas aeruginosa* clinical isolates. Antimicrob. Agents Chemother. 47:3202–3207.
46. Stover, C. K., et al. 2000. Complete genome sequence of *Pseudomonas aeruginosa* PAO1, an opportunistic pathogen. Nature 406:959–964.
47. Trias, J., and H. Nikaido. 1990. Outer membrane protein D2 catalyzes facilitated diffusion of carbapenems and penems through the outer membrane of *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 34:52–57.
48. Vaara, M., P. Viljanen, T. Vaara, and P. H. Makela. 1984. An outer membrane-disorganizing peptide PMBN sensitizes *E. coli* strains to serum bactericidal action. J. Immunol. 132:2582–2589.

49. Viljanen, P., and M. Vaara. 1984. Susceptibility of gram-negative bacteria to polymyxin B nonapeptide. Antimicrob. Agents Chemother. 25:701–705.
50. Vogne, C., J. R. Aires, C. Bailly, D. Hocquet, and P. Plésiat. 2004. Role of the multidrug efflux system MexXY in the emergence of moderate resistance to aminoglycosides among *Pseudomonas aeruginosa* isolates from patients with cystic fibrosis. Antimicrob. Agents Chemother. 48:1676–1680.
51. Westbrook-Wadman, S., et al. 1999. Characterization of a *Pseudomonas aeruginosa* efflux pump contributing to aminoglycoside impermeability. Antimicrob. Agents Chemother. 43:2975–2983.
52. Zhang, L., M. G. Scott, H. Yan, L. D. Mayer, and R. E. Hancock. 2000. Interaction of polymyxin I and structural analogs with bacterial membranes, lipopolysaccharide, and lipid monolayers. Biochemistry 39:14504–14514.
