
Chem. Rev. 2011, 111, 160–173

# Listening to a New Language: DSF-Based Quorum Sensing in Gram-Negative Bacteria

Yinyue Deng, Ji'en Wu, Fei Tao, and Lian-Hui Zhang*

Institute of Molecular and Cell Biology, Proteos, 61 Biopolis Drive, Singapore 138673

Received October 22, 2010

## Contents

1. Introduction                                                                                       160
2. Chemistry of DSF-Family Signals                                                                    161
   2.1. Detection of DSF-Family Signals                                                               161
   2.2. Purification and Structural Characterization of DSF-Family Signals                           162
   2.3. Synthesis of DSF-Family Signals                                                               163
   2.4. Nomenclature                                                                                  163
   2.5. Structural Features and Biological Activity                                                  163
3. DSF Signaling Mechanisms in *Xanthomonas campestris pv campestris*                                 164
   3.1. Enzymes Associated with DSF Biosynthesis                                                     164
   3.2. Dual Roles of the DSF Sensor Kinase RpfC                                                      164
   3.3. Response Regulator RpfG is a Novel DSF-Modulated c-di-GMP Degradation Enzyme                   165
   3.4. Clp is a Novel c-di-GMP Effector                                                              166
   3.5. Networking the DSF Signaling Pathways in Virulence Regulation                                 166
   3.6. DSF-Dependent Biofilm Dispersal                                                               167
4. DSF-Based QS Mechanisms in Other Organisms                                                         168
   4.1. *Xylella fastidiosa*                                                                          168
   4.2. *Xanthomonas axonopodis pv citri*                                                             168
   4.3. *Xanthomonas oryzae pv oryzae*                                                                168
   4.4. *Xanthomonas axonopodis pv glycines*                                                          169
   4.5. *Stenotrophomonas maltophilia*                                                                169
   4.6. *Burkholderia cepacia* complex                                                                169
   4.7. *Pseudomonas aeruginosa*                                                                      170
5. Interaction with Other Signaling Mechanisms                                                       170
6. Roles of DSF-Family Signals in Interspecies and Interkingdom Communication                         171
7. Conclusions and Future Prospective                                                                 171
8. References                                                                                        171

## 1. Introduction

It has become increasingly clear that bacterial cells can communicate with each other to facilitate their adaptation to changing environmental conditions. This cell–cell communication mechanism, known as quorum sensing (QS), relies on the production, detection, and response to diffusible signal molecules in a cell-density-dependent manner. ${}^{1-4}$ QS is of vital importance to many bacterial species in the regulation of a variety of functions such as biofilm formation, toxin production, exopolysaccharide synthesis, extracellular enzyme production, motility, and plasmid transfer. ${}^{5-9}$ At low

* To whom correspondence should be addressed. E-mail: lianhui@imcb.a-star.edu.sg.

10.1021/cr100354f © 2011 American Chemical Society  
Published on Web 12/17/2010

DSF-Based Quorum Sensing in Gram-Negative Bacteria

Fei Tao obtained his Ph.D from the Institute of Molecular and Cell Biology (IMCB) and the National University of Singapore under the mentorships of Professor Lian-Hui Zhang and Associate Professor Sanjay Swarup in 2010. Currently, he is a Research Fellow in the laboratory of Lian-Hui Zhang at IMCB. His research interest is in quorum sensing regulation of virulence and biofilm development, especially the signaling pathway and molecular mechanisms of DSF-dependent quorum sensing system in *Xanthomonas campestris*.

Lian-Hui Zhang is a Research Director/Professor at the Institute of Molecular and Cell Biology and an Adjunct Professor at the National University of Singapore (NUS). He received his Ph.D degree in 1993 under the mentorship of Professor Allen Kerr at the University of Adelaide, where he discovered an AHL-type quorum sensing signal essential for regulation of Ti plasmid conjugal transfer in *Agrobacterium tumefaciens*. He has been an Australian Research Council Postdoctoral Research Fellow and has received NUS Outstanding Researcher Award and Singapore National Science Award for his work in quorum sensing and quorum quenching. His research interests focus on cell–cell communication systems and signaling mechanisms implicated in regulation of bacterial virulence and biofilm formation and on developing quorum quenching strategies to control bacterial infections.

luminescence, plasmid transfer, virulence, biofilm formation, motility, and antibiotic production. ${}^{1,4}$ Interestingly, in addition to AHL-family signals, LuxR-type receptors can also be activated by diketopiperazines (DKPs), which are structurally unrelated to AHLs. DKPs have been identified in *Proteus mirabilis*, *Citrobacter freundii*, *Enterobacter agglomerans*, *Pseudomonas fluorescens*, and *Pseudomonas alkaligenes*. ${}^{14,15}$ Autoinducer 2 (AI-2) is another type of widely detected QS signal, which was identified originally in *Vibrio harveyi* ${}^{16}$ and subsequently in other Gram-negative bacteria species. ${}^{17}$ AI-2 type signals are involved in regulation of bioluminescence, type III secretion systems, and extracellular virulence factor production. In addition, there are other seemingly less widely distributed QS signals such as 3-hydroxypalmitic acid methyl ester (3-OH PAME), ${}^{18}$ 2-heptyl-3-hydroxy-4(1H)-quinolone (PQS), ${}^{19}$ bradyoxetin, ${}^{20}$ autoinducer in *Escherichia coli* (AI-3), ${}^{21}$ and 2-heptyl-4-quinolone (HHQ), ${}^{22}$ which have been identified and characterized within the last two decades.

Apart from the above-mentioned QS signals, the diffusible signal factor (DSF), which was originally identified in *Xanthomonas campestris* pv *campestris* (*Xcc*), ${}^{23,24}$ represents another interesting type of QS signal found in Gram-negative bacterial pathogens. Evidence is accumulating that DSF-family signals are widespread. ${}^{23-25}$ In the past few years, research has not only identified a range of DSF-family signals but also unveiled unique QS signaling pathways and mechanisms distinct from other known QS systems. This review will discuss these interesting features with focus on the conservation and diversity of various DSF-dependent QS systems in Gram-negative bacteria. For the convenience of discussion, DSF will be used as a collective term for the DSF-family signals unless otherwise specified.

## 2. Chemistry of DSF-Family Signals

### 2.1. Detection of DSF-Family Signals

Detection of DSF activity was first reported a decade ago by M. J. Daniels’ group when analyzing an *Xcc* gene cluster designated *rpf* (regulation of pathogenicity factors). It was found that the protease and endoglucanase activity of an *rpfF* mutant could be restored when cultivated in proximity to its wild-type parental strain. ${}^{23}$ This result led to speculation that wild-type *Xcc* might produce a diffusible signal factor (DSF) which induced protease and endoglucanase production and that the enzyme encoded by *rpfF* was associated with DSF biosynthesis. The protease-based bioassay of DSF activity uses an agar plate containing skimmed milk, in which DSF-induced production of extracellular proteases by *Xcc* strains is indicated by a clear zone around the bacterial colony. For detection of endoglucanase activity, an agar plate supplemented with carboxymethylcellulose (CMC) is used. The zone of CMC hydrolysis by endoglucanase becomes visible after developing with 0.1% Congo red followed by distaining with 1 M NaCl. ${}^{23}$

Second-generation DSF-activity detection methods are based on engineered DSF biosensors. Typically, the promoter of a DSF-inducible operon *engXCA* is fused with a promoterless reporter *gusA* gene that encodes β-glucuronidase (GUS); then the PengXCA-*gusA* cassette is cloned in a broad-host-range vector, and the resultant plasmid is mobilized to a DSF-minus mutant of *Xcc* to generate the DSF biosensor strains. ${}^{24,26}$ The DSF activity is indicated by a blue halo around the test colony or DSF extract in an agar plate containing 5-bromo-4-chloro-3-indolyl-β-D-glucuronic acid

Table 1. List of DSF-Family Signals

| Name                     | Structure                                                                 | Original strain                                                                                          | References | Note |
|--------------------------|---------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------|------------|------|
| $11$-Me-$C_{12}:\Delta^{2}$ (DSF) | $\ce{CH3-CH2-CH2-CH2-CH=CH-CH2-CH2-CH2-COOH}$ | $X.$ campestris, $S.$ maltophilia, $B.$ multivorans, $X.$ oryzae pv. oryzae | 24,28,29,36 |      |
| $C_{12}:\Delta^{2}$ (BDSF) | $\ce{CH3-CH2-CH2-CH2-CH2-CH=CH-CH2-CH2-COOH}$ | $B.$ cenocepacia, $B.$ lata, $B.$ multivorans, $B.$ stabilis, $B.$ vietnamiensis, $B.$ dolosa, $B.$ ambifaria, $B.$ stabilis, $B.$ anthina, $B.$ pyrrocinia, $X.$ oryzae pv. oryzae | 27-29 |      |
| $11$-Me-$C_{12}:\Delta^{2,5}$ (CDSF) | $\ce{CH3-CH2-CH2-CH=CH-CH2-CH=CH-CH2-CH2-COOH}$ | $B.$ multivorans, $B.$ stabilis, $B.$ anthina, $B.$ pyrrocinia, $X.$ oryzae pv. oryzae | 28,29 |      |
| $11$-Me-$C_{12}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-COOH}$ | $S.$ maltophilia | 36 |      |
| $10$-Me-$C_{12}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-COOH}$ | $S.$ maltophilia | 36 | $a$ |
| $12$-Me-$C_{14}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-COOH}$ | $X.$ fastidiosa | 32 | $a$ |
| $C_{10}:\Delta^{2}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH=CH-CH2-CH2-COOH}$ | $P.$ aeruginosa | 37 |      |
| $C_{13}:\Delta^{2}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH=CH-CH2-COOH}$ | $S.$ maltophilia | 36 | $a,b$ |
| $C_{14}:\Delta^{2}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH=CH-CH2-COOH}$ | $S.$ maltophilia | 36 | $a,b$ |
| $12$-Me-$C_{13}:\Delta^{2}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH=CH-CH2-COOH}$ | $S.$ maltophilia | 36 | $a,b$ |
| $13$-Me-$C_{14}:\Delta^{2}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH=CH-CH2-COOH}$ | $S.$ maltophilia | 36 | $a,b$ |
| $12$-Me-$C_{14}:\Delta^{2}$ | $\ce{CH3-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH2-CH=CH-CH2-COOH}$ | $S.$ maltophilia | 36 | $a,b$ |
| $C_{10}:\Delta^{21}$ (SDSF) | $\ce{CH3-CH2-CH2-CH2-CH2-CH=CH-CH2-CH2-COOH}$ | Streptococcus mutans | 40 |      |

$a$ These DSF signals have not been verified for their functions with the purified or synthetic molecules.  
$b$ The configuration was not determined.

DSF-Based Quorum Sensing in Gram-Negative Bacteria

Chemical Reviews, 2011, Vol. 111, No. 1 163

**Scheme 1. Synthesis of the cis-Configuration DSF-Family Signals through Favorskii Rearrangement**

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

n = 6, 8, 10, 11, 13

signals from bacterial culture supernatants. A DSF analogue, i.e., cis-2-decenoic acid, was purified from *Pseudomonas aeruginosa* culture supernatants using chloroform extraction and subsequent HPLC fractionation (Table 1). Considering the fact that fatty acids exist as anions in weak basic bacterial culture supernatants and that these anions are less soluble in organic solvent than corresponding free acids, a recent study has revised the extraction protocol by using acidified supernatants for solvent extraction. The results showed that adjusting the pH value of supernatants to ~4.0 substantially improved the extraction efficiency of DSF-family signals.

### 2.3. Synthesis of DSF-Family Signals

Biosynthesis of DSF signal in Xcc is dependent on *rpfF* and *rpfB*, which are located next to each other in the *rpf* gene cluster. These two genes encode a putative enoyl-CoA hydratase and a putative long-chain fatty acyl CoA ligase, respectively. The *rpf* gene cluster has subsequently been identified in other bacteria such as *S. maltophilia*, *X. axonopodis pv citri*, *X. oryzae pv oryzae*, and *Xy. fastidiosa*. The mechanism of DSF-family signal production appears to be widely conserved. In addition to RpfF in Xcc, the role of RpfF homologues in synthesis of DSF-family signals has also been established in other bacterial species, such as *B. cenocepacia*, *X. oryzae pv oryzae*, *Xy. fastidiosa*, *S. maltophilia*, and *X. axonopodis pv glycines*. The role of RpfB is relatively less well characterized except for in Xcc and *S. maltophilia*, in which mutation of *rpfB* seemed to abrogate the production of DSF-family signals. The putative catalytic mechanisms of these two enzymes are discussed in section 3.1.

Chemically, DSF and its cis-analogues have been synthesized by Favorskii rearrangement. Briefly, ketones prepared with relevant fatty acid substrates are used to generate the intermediates 1,3-dibromo-2-ones, which are then used to produce cis-2-enoic carboxylic acids (as shown in Scheme 1). The trans-derivatives of DSF have been synthesized through preparation of 2-bromo fatty acids followed by elimination of hydrobromic acids using a strong base such as potassium tert-butoxide. Synthetic DSF-family signals have also been used in several recent studies as standard controls, but the information regarding their synthesis has not been described.

### 2.4. Nomenclature

Like the AHL-type QS signals, the list of DSF-family signals is expanding rapidly. cis-11-Methyl-2-dodecenoic acid was identified from Xcc as the first member of DSF-family signals in 2004 and is known as DSF. For convenience, cis-2-dodecenoic acid and trans-2-decenoic acid, the DSF structural analogues identified from *B. cenocepacia* and *Streptococcus mutans*, were named BDSF and SDSF, respectively. However, the shortcoming of this nomenclature system soon becomes obvious as a range of DSF-like signals have been identified from different bacterial species (Table 1), and especially since one bacterial species may produce more than one DSF-family signal (Table 1). A new nomenclature system has recently been proposed in an attempt to cover all the possible DSF-family signals. For example, cis-11-methyl-2-dodecenoic acid (DSF) is designated as 11-Me-C₁₂:Δ² and cis-2-dodecenoic acid (BDSF) is designated as C₁₂:Δ² (Table 1). This nomenclature is based on one of the fatty acid nomenclature systems where substitution and its position are indicated first (for example, 11-Me indicates a methyl group on C-11 of the fatty acid carbon chain), followed by the length of the fatty acid carbon chain (C₁₂ represents a 12-carbon fatty acid chain), and then the position of the double bond in the fatty acid chain (Δ² indicates a double bond in cis-configuration at site 2, i.e., between C-2 and C-3 of the fatty acid carbon chain). Similarly, DSF molecules with trans-configuration can also be named following the same rules by inserting a "t" next to the position of the double bond. For example, Δ² means a double bond at site 2 with a cis-configuration, while Δ² suggests a double bond at site 2 with a trans-configuration. Following this nomenclature system, all the reported DSF-family molecules have been renamed and listed in Table 1. It should be noted that the biological activities of some DSF derivatives have not yet been verified (Table 1). At this stage, it is premature to regard them as functional QS signals. In addition, the naming of several molecules from *S. maltophilia* is not complete due to a lack of information about the double-bond configuration (Table 1); whether these double bonds are in trans- or cis-configuration requires further characterization.

### 2.5. Structural Features and Biological Activity

DSF-family signals share a fatty acid carbon chain with variations in chain length, double-bond configuration, and side-chain (Table 1). Evidence is emerging that these structural features may contribute to their biological activity and signaling specificity in intraspecies signaling and interspecies communication.

One of the key features that could affect intraspecies signaling activity is the presence or absence of a double bond and its configuration. By synthesizing a range of structural derivatives of DSF, Wang et al. analyzed the structural features of fatty acids that are important for QS-dependent regulation of virulence in Xcc. DSF (11-Me-C₁₂:Δ²), which contains a cis-double bond in α,β-position, showed about 200- and 20,000-fold stronger activity than its trans-isomer and saturated isomer, respectively, in the induction of virulence gene expression. These data have demonstrated that the α,β-double bond and cis-configuration of fatty acid are critical for QS signaling activity in Xcc. However, these findings in Xcc may not translate to other bacterial species. DSF and C₁₂:Δ² showed similar activities in the induction of xylanase activity and exopolysaccharide production when assayed using the *rpfF* mutant of *X. oryzae pv oryzae*, indicating that methyl substitution at C11 has no obvious effect on the biological activity of DSF signals in this bacterial pathogen. Similarly, cis-Δ2-11-methyldodecenoic acid (11-Me-C₁₂:Δ²) and its saturated derivative 11-methyldodecanoic acid (11-Me-C₁₂) appeared to be equally effective in the stimulation of flagella-independent translocation by *S. maltophilia*, suggesting that the double bond at α,β-position does not affect this biological activity in *S. maltophilia*.

Variations in methyl group substitution and chain length can also substantially affect the biological activity of DSF-family signals. DSF, which differs from C<sub>12</sub>:Δ<sup>2</sup> by containing a methyl group substitution at C11, showed ~120-fold higher potency in induction of virulence gene expression in *Xcc*.<sup>24</sup> The same study also showed that increasing one carbon or decreasing two carbons in the fatty acid chain could compromise signaling activity by 2- to 5-fold. In addition, comparison of DSF with 11-Me-C<sub>12</sub>:Δ<sup>2,5</sup> showed that one more double bond causes ~20% reduction of signaling activity in the induction of *X. oryzae pv oryzae* EPS and xylanase production.<sup>29</sup>

The structural features of DSF-family signals also affect their biological activity in interspecies and cross-kingdom communications. It has been reported that among DSF, C<sub>12</sub>:Δ<sup>2</sup>, and 11-Me-C<sub>12</sub>:Δ<sup>2,5</sup>, C<sub>12</sub>:Δ<sup>2</sup> is the strongest inhibitor of germ tube formation by *Candida albicans*, followed by 11-Me-C<sub>12</sub>:Δ<sup>2,5</sup> and DSF. The findings suggest that the methyl group at C11-site of DSF reduces the inhibitory activity on *C. albicans* germ tube formation, but an additional double bond at C5–C6 of 11-Me-C<sub>12</sub>:Δ<sup>2,5</sup> has the opposite effect.<sup>27,28</sup> Similar variations have also been found in bacterial interspecies communications. A recent study showed that C<sub>12</sub>:Δ<sup>2</sup> was ~10% less active than DSF and 11-Me-C<sub>12</sub>:Δ<sup>2,5</sup> in the induction of *Xcc* EPS production, whereas in *B. cenocepacia* it is DSF that was inferior to the other two signals in the induction of virulence gene expression.<sup>28</sup> Taken together, the above findings indicate that the biological activity of DSF-family signals depends on not only their structural features but also the bacterial species on which they act.

### 3. DSF Signaling Mechanisms in Xanthomonas campestris pv campestris

*X. campestris pv campestris* (*Xcc*) is the causal agent of black rot of cruciferous plants, which is possibly the most important disease of crucifers worldwide.<sup>41</sup> It belongs to the γ-subdivision of proteobacteria and is a bacterial pathogen with significant economic importance.<sup>41</sup> As a model organism of a new QS system, *Xcc* has been well characterized using an array of approaches, including genetic, genomic, biochemical, and cell biology. The research progress over the past decade has unveiled a range of enzymes and regulators that are of critical importance in DSF-dependent QS signaling pathways. The roles and molecular mechanisms of the key components in DSF signaling system will be discussed separately in this section.

#### 3.1. Enzymes Associated with DSF Biosynthesis

An early transposon mutagenesis analysis of the genes involved in the synthesis of extracellular enzymes led to identification of a gene cluster designated as *rpf* in *Xcc*.<sup>42</sup> On the basis of the findings that decreased production of extracellular enzymes of *rpfF* mutant could be restored by a diffusible extracellular factor (DSF) produced by various *Xcc* strains, except for the *rpfF* and *rpfB* mutants, it was therefore speculated that the enzymes encoded by *rpfF* and *rpfB* were involved in DSF biosynthesis.<sup>23</sup> Null mutation of *rpfF* or *rpfB* abolished DSF production, suggesting that they are essential for DSF production in *Xcc*.<sup>23,26</sup> In silico analysis predicted that *rpfF* and *rpfB* encode a putative enoyl-CoA hydratase and a putative long-chain fatty acyl CoA ligase, respectively.<sup>23,26</sup> However, their catalytic mechanisms, corresponding substrates, and reaction products require further investigation. A recent crystal structural analysis of RpfF has shown that it is structurally similar to the members of the crotonase superfamily containing an N-terminal α/β-spiral core domain and a C-terminal α-helical region.<sup>43</sup> Sequence alignment and structural analysis resulted in the identification of two putative catalytic glutamate residues (Glu141 and Glu161), which are conserved in the enoyl-CoA hydratases/dehydratases. Substitution of these two residues in RpfF completely abolished DSF production, underscoring their critical role in DSF biosynthesis.<sup>43</sup> In addition, structural analysis of RpfF also unveiled a hydrophobic pocket composed of several hydrophobic residues, including Leu136 from βB3, Gly137 and Gly138 from βB3-α3 loop, Gly85 from βB2-αB2 loop, Leu276 from α10, Met170 from βB4-αB4 loop, and Trp258 from α9.<sup>43</sup> The structural features and residue properties of this hydrophobic pocket suggest that it is likely the putative substrate-binding pocket for DSF biosynthesis.<sup>43</sup> Consistent with this speculation, a single-point mutation in any of these residues resulted in either no DSF production or dramatically reduced enzymatic activity of RpfF in DSF biosynthesis.<sup>43</sup>

Microarray analysis of the *rpfF* mutant in the presence and absence of DSF led to the identification of >165 genes that are regulated by the DSF-based QS system.<sup>44</sup> Significantly, within the DSF-regulon, 133 genes are positively regulated and 32 are negatively modulated by DSF, which is consistent with the regulation pattern for other QS systems.

### 3.2. Dual Roles of the DSF Sensor Kinase RpfC

Within the *rpf* gene cluster, the *rpfC* gene encodes a two-component-type hybrid, histidine sensor kinase consisting of multidomains, i.e., five transmembrane (TM) domains, a histidine kinase (HK) domain, a receiver (REC) domain, and a histidine phosphotransfer (HPT) domain. The findings that the *rpfC* and *rpfF* mutants displayed similar phenotypes such as decreased production of exopolysaccharides (EPS) and extracellular enzymes,<sup>26</sup> and that the *rpfF* mutant phenotype could be fully restored by exogenous addition of DSF but the *rpfC* mutant failed to respond to DSF, argued strongly that RpfC is a DSF sensor.<sup>26,45</sup> Further genetic analysis showed that substitution of the conserved residues associated with two-component-type phosphorelays, i.e., His198 in the HK domain, Asp512 in the REC domain, and His657 in the HPT domain of RpfC, abolished the DSF-dependent functions such as virulence factor production and biofilm dispersal.<sup>44,45</sup> These results have demonstrated that RpfC uses a conserved phosphorelay mechanism to transduce the DSF signal when modulating virulence factor production (Figure 1). However, the mechanism by which RpfC detects DSF and triggers the phosphorelay process remains undetermined.

Unusually, apart from functioning as a DSF sensor to positively regulate virulence factor production, RpfC also acts negatively to regulate the biosynthesis of DSF.<sup>26</sup> DSF biosensor-based diffusion plate assays revealed that the *rpfC* mutant produced a larger DSF diffusion zone than its parental wild-type strain.<sup>26</sup> Subsequent quantification analysis showed that mutation in *rpfC* caused ~16-fold increase in DSF production at the peak time.<sup>24</sup> Detailed genetic and biochemical analyses have demonstrated that RpfC regulation of DSF production is independent of the conserved phosphorelay mechanism, and presented several lines of evidence indicating that RpfC negatively controls DSF production via a novel post-translational mechanism involving domain-

DSF-Based Quorum Sensing in Gram-Negative Bacteria

Chemical Reviews, 2011, Vol. 111, No. 1 165

translational autoinduction mechanism of DSF biosynthesis (Figure 1), totally unlike the transcriptional autoinduction mechanism of AHL-type QS systems.²⁴ According to this model, at a lower cell density, the unphosphorylated RpfC maintains a conformation that facilitates formation of a complex between RpfF and RpfC, limiting DSF production at a basal level (Figure 1A). At high cell density, the accumulated DSF molecules trigger the autophosphorylation of RpfC and release of RpfF, resulting in increased DSF signal production (Figure 1B).

The above model is strongly supported by crystal structural analysis of the RpfF monomer and RpfF–REC complex,⁴³ which showed that interaction between RpfF and REC involved both hydrophilic and hydrophobic contacts. When RpfF is bound to the REC domain, the entrance of the substrate binding pocket is blocked, locking it into an inactive form and thereby negatively regulating DSF biosynthesis. Although these findings provide a structural model for the RpfC–RpfF interaction-mediated QS autoinduction mechanism, the mechanisms by which RpfC detects DSF and consequently releases RpfF remain unclear.

### 3.3. Response Regulator RpfG is a Novel DSF-Modulated c-di-GMP Degradation Enzyme

Several lines of evidence indicate that RpfC and RpfG constitute a two-component regulatory system responsible for the detection and transduction of the QS signal DSF. In *Xcc* genome, *rpfG*, *rpfH*, and *rpfC* are transcribed as an operon, suggesting that these three genes might be functionally related.²⁶ In particular, RpfC is a sensor kinase, as discussed above, and RpfG is a response regulator that contains a typical receiver domain and a HD-GYP domain. In addition, the *rpfG* and *rpfC* mutants display similar phenotypic changes, including decreased virulence factor production and, especially, insensitivity to DSF.²⁶,⁴⁶ Moreover, *in trans* expression of the RpfC–RpfG two-component system in *P. aeruginosa* enabled the pathogen to respond to exogenous addition of DSF, consequently affecting bacterial motility.⁴⁷ The role of RpfG in DSF signal transduction has been further confirmed by site-directed mutagenesis and deletion analysis. Substitution of Asp80 in the receiver domain of RpfG, which is the conserved residue implicated in two-component-type phosphorelays, decreased *Xcc* virulence factor production and abolished the bacterial response to DSF signal.⁴⁸ Importantly, *in trans* expression of the HD-GYP domain of RpfG in the *rpfG* mutant was sufficient to restore virulence factor production,⁴⁷,⁴⁸ suggesting that this domain plays a key role in DSF signal transduction.

In silico and biochemical analysis of the HD-GYP domain has revealed its novel function in c-di-GMP metabolism. c-di-GMP is a global second messenger implicated in the regulation of various biological functions in bacteria, including cellulose biosynthesis,⁴⁹ bacterial motility,⁵⁰ biofilm formation,⁴⁹,⁵¹ and virulence factor production.⁴⁷,⁵² The intracellular concentration of c-di-GMP was initially found to be modulated by the proteins containing GGDEF or EAL domains.⁵³ Typically, proteins containing a GGDEF domain are diguanylate cyclases, which catalyze the synthesis of c-di-GMP from two molecules of GTP,⁵³,⁵⁴ whereas proteins containing the EAL domain are c-di-GMP phosphodiesterases that hydrolyze c-di-GMP to generate linear nucleotide pGpG.⁵⁵,⁵⁶ HD-GYP was predicted by in silico analysis as a novel domain involved in signal transduction and nucleotide metabolism as a putative diguanylate phosphodi-

166 Chemical Reviews, 2011, Vol. 111, No. 1

esterase.⁵⁷,⁵⁸ It is a subgroup of the HD superfamily of metal-dependent phosphohydrolases containing an additional GYP motif. On the basis of the findings that the proteins consisting of the HD-GYP domain and a CheY-like two-component receiver (REC) domain are present in many bacterial species, it was predicted that this novel HD-GYP domain may have important roles in bacterial signaling and response to environmental cues.⁵⁷,⁵⁸ Subsequent biochemical analysis confirmed that the HD-GYP domain degraded c-di-GMP to generate two molecules of GMP.⁴⁷ Point mutation of the conserved residues H or D of the HD-GYP domain abrogated the RpfG enzymatic activity against c-di-GMP and regulatory activity on virulence factor production,⁴⁷ whereas substitution of the GYP motif had no effect on the activity of RpfG to degrade c-di-GMP.⁵⁹ Consistent with the predicted role of HD-GYP domain in RpfG-dependent signal transduction, *in trans* expression of the HD-GYP domain alone in the *rpfG* mutant was sufficient to restore the mutant phenotypes including virulence factor production and biofilm dispersion.⁴⁷

Taken together, these findings indicate that the HD-GYP of RpfG domain is a novel c-di-GMP degradation enzyme, which is activated by the DSF signaling system to perform its regulatory activity by enzymatic degradation of intracellular c-di-GMP (Figure 1B). Interestingly, a subsequent study found that RpfG controlled the expression of a subset of proteins with GGDEF or EAL domains, which possibly function to synthesize or degrade c-di-GMP.⁶⁰ To fully understand the RpfG signaling pathways and mechanisms, it will be essential to determine the biological roles of these GGDEF and EAL proteins and their target processes.

### 3.4. Clp is a Novel c-di-GMP Effector

The *clp* gene of *Xcc* was identified in the early 1990s through screening of *Xcc* genomic DNA library in complementation of the adenylate cyclase (*cya*)-deficient mutant of *E. coli*.⁶¹ DNA sequencing analysis of the *Xcc* DNA fragment that partially restored the ability of the *cya* mutant of *E. coli* to catabolize carbohydrate did not find a *cya* homologue, but instead revealed an open reading frame (ORF) that encodes a homologue of the *E. coli* cyclic AMP (cAMP) receptor protein (CRP). The *Xcc* homologue was accordingly named as Clp for CAP-like protein.⁶¹ Interestingly, however, null mutation of Clp in *Xcc* did not affect carbohydrate catabolism but decreased the biosynthesis of EPS, extracellular cellulase, and polygalacturonate lyase,⁶¹ which are reminiscent of the phenotypes of the DSF-deficient mutants.²³,²⁴,²⁶ The role of Clp in DSF-based QS system was not established until genetic and genomic analyses showing that Clp regulated a set of genes within the DSF regulon and that *in trans* expression of *clp* restored the phenotypes of *rpf* mutants.⁶² These findings provide confirmation that Clp is a key component of the DSF regulatory network in *Xcc* (Figure 1).

Similar to the Crp of *E. coli*, Clp contains a cNMP binding domain at N-terminal and a DNA-binding domain at C-terminal. This structural similarity prompted two studies investigating the effect of c-di-GMP on the functionality of Clp. The results showed that the Clp proteins from *Xcc* and *X. axonopodis pv citri* were allosterically inhibited by c-di-GMP.⁶³,⁶⁴ In *Xcc*, Clp acts as a global regulator that positively regulates ~260 genes and negatively regulates ~39 genes.⁶² Clp has the same DNA-binding specificity as Crp and recognizes the same conserved DNA-binding site (TGT-

GA-N6-TCACA).⁶⁵ In silico analysis of the promoters of Clp-dependent genes showed that >80 genes contain a putative Clp binding site,⁶² including the previously characterized gene *engXCA*, which encodes an extracellular endoglucanase.⁶⁶ Isothermal titration calorimetry, far-UV circular dichroism, and electrophoretic mobility shift assay analyses found that c-di-GMP specifically bound to Clp with high affinity and induced allosteric conformational changes that abolished the interaction between Clp and the promoter of *engXCA* in *Xcc*.⁶³ The same study also showed that Clp did not interact with cAMP and other nucleotides. Site-directed mutagenesis and structural analyses showed that the cNMP binding domain and the region between cNMP and DNA-binding domains of Clp in *Xcc* contain several residues implicated in c-di-GMP binding. Substitution of these residues with alanine or serine resulted in decreased sensitivity to changes in the intracellular c-di-GMP level and attenuated bacterial virulence.⁶³,⁶⁷ The results of structural analysis suggest that Clp adopts an intrinsically active conformation for DNA binding, and binding with c-di-GMP may influence the helix αD of Clp, affecting its DNA-binding activity.⁶⁷

The identification of Clp as a novel effector of c-di-GMP in *Xcc* presents a new member to the list of known c-di-GMP effectors identified in other bacterial pathogens, including PilZ domain “adaptor” protein,⁶⁸ transcriptional regulators such as FleQ⁶⁹ and Vpst,⁷⁰ proteins with inactive GGDEF and/or EAL domains such as LapD⁷¹ and FimX,⁷² and proteins with binding sites similar to GGDEF domains such as PelD.⁷³ These findings have also shown for the first time that the widely conserved cNMP domain can be grouped into two categories, with one being cAMP receptors such as the Crp of *E. coli* and the other being c-di-GMP effectors represented by the Clp of *Xcc*.⁶³

### 3.5. Networking the DSF Signaling Pathways in Virulence Regulation

The identification of the key components of the DSF-based QS system and advances in microarray and biochemical technologies have facilitated the understanding of DSF signaling pathways. Upon characterization of Clp as one of the key members in the DSF-based QS system, two Clp-modulated transcription factors, i.e., FhrR and Zur, have been identified through microarray analysis and genetic verification.⁶² Genomic analysis and phenotype characterization have demonstrated that Clp regulated the genes encoding flagellar, hypersensitive reaction and pathogenicity system, and ribosomal proteins through a TetR-family transcription factor FhrR as well as the genes encoding iron uptake, multidrug resistance, and detoxification via a Fur-family transcription factor Zur.⁶² Interestingly, a recent study showed that physical interaction between RpfG and two GGDEF-domain proteins, i.e., XC_0249 and XC_0420, regulated DSF-dependent bacterial cell motility but had no effect on other DSF-dependent functions such as extracellular enzyme production and biofilm formation.⁵⁹ Substitution of the conserved GYP motif in the HD-GYP domain of RpfG abolished this interaction, indicating that the GYP motif is essential for the interaction between RpfG and these GGDEF-domain proteins.⁵⁹ It is noteworthy that substitution of the HD motif within the HD-GYP domain of RpfG had no effect on RpfG-GGDEF interaction but affected the ability of RpfG to regulate motility and extracellular enzyme production.⁵⁹ Given that the HD motif, but not the GYP motif, is essential

DSF-Based Quorum Sensing in Gram-Negative Bacteria

Chemical Reviews, 2011, Vol. 111, No. 1 167

**Figure 2.** Schematic representation of DSF-dependent biofilm dispersal. (A) In the absence of DSF, in particular in the *rpf* mutants of *Xcc*, expression of the *xagABCD* gene cluster implicated in polysaccharide biosynthesis causes biofilm formation. (B) At high cell density, accumulation of DSF signals leads to increased expression of *manA* encoding biofilm degradation and suppression of the *xagABC* transcriptional expression. Both processes require the involvement of RpfC/RpfG two-component system. Consequently, *Xcc* grows in planktonic form.

for the enzymatic activity of RpfG,^{47,59} the above findings seem to suggest that RpfG may modulate *Xcc* motility through two mechanisms, i.e., the HD motif-dependent c-di-GMP degradation and the GYP motif-dependent RpfG-GGDEF interaction. Consistent with the former putative mechanism, deletion of the c-di-GMP effector Clp or its downstream regulator FhrR resulted in decreased *Xcc* cell motility.^{62} How the observed RpfG-GGDEF interaction influences bacterial motility remains unclear.

The above findings, together with the current understanding of the conserved molecular mechanisms of two-component regulatory systems,^{74} have outlined the DSF-based QS networks and related regulatory mechanisms in *Xcc* (Figure 1). Similar to the QS systems in Gram-positive bacteria,^{75} the DSF-dependent cell-cell communication system in *Xcc* employs a two-component system consisting of RpfC/RpfG in signal perception and transduction.^{23,26,45} Upon detection of DSF signal when *Xcc* reaches a threshold population, autophosphorylation of RpfC triggers phosphorelay through its HK to REC to HPT domains, and subsequently to the REC domain of RpfG. Phosphorylation of RpfG is believed to induce protein conformational changes and activate its phosphodiesterase activity. Degradation of c-di-GMP molecules reduces the intracellular level of this second messenger, resulting in increased intracellular level of free Clp, which regulates the target genes expression directly or indirectly through its downstream transcription factors including Zur and FhrR^{63} (Figure 1B). Additionally, RpfG could also regulate a subset of functions through interaction with certain GGDEF proteins.^{59}

### 3.6. DSF-Dependent Biofilm Dispersal

Apart from regulation of virulence, another important biological function modulated by the DSF-based QS system in *Xcc* is biofilm dispersal. Mutation of *rpfF* in both *Xcc* strains 8004 and XC1 caused the formation of bacterial cell aggregates, and addition of DSF suppressed cell aggregation,^{44,46} suggesting a critical role of DSF signal in modulation of *Xcc* switching between planktonic and biofilm forms. An early study found that a DSF-inducible enzyme, endo-β-1,4-mannanase (ManA),

168 Chemical Reviews, 2011, Vol. 111, No. 1

# 4. DSF-Based QS Mechanisms in Other Organisms

## 4.1. Xylella fastidiosa

Xy. fastidiosa is an important plant pathogen that causes disease in many important economic crops and plants, such as citrus, grape, coffee, peach, and other important horticultural species. The pathogen belongs to γ-proteobacteria, is restricted to the xylem vessels of plants, and is transmitted by xylem sap feeding insects such as the sharpshooter species. The cell–cell signaling in Xy. fastidiosa is mediated by an analogue of DSF, tentatively identified as 12-methyl-tetradecanoic acid (12-Me-C₁₄) through GC-MS analysis (Table 1). Although the chemical structure of this putative signal and its role in bacterial physiology awaits further verification and investigation, there is no doubt that Xy. fastidiosa utilizes a DSF-based QS system in the regulation of virulence. In silico analysis showed that the pathogen contains a conserved *rpfGCF* gene locus, and their peptide products share >60% identity to their corresponding counterparts from Xcc. Importantly, mutation of *rpfF* in Xy. fastidiosa abolished DSF signal production and gave rise to enhanced virulence, impaired insect transmission, and disabled biofilm formation in host insects. In contrast, the *rpfC* mutants overproduced DSF; were deficient in virulence, migration along xylem vessels, and insect transmission; and showed a hyperattachment phenotype. It was speculated that these divergent expression patterns of the genes encoding biofilm formation and virulence might indicate two possible distinct pathways for DSF perception in Xy. fastidiosa. In one pathway, RpfC, which is likely membrane-bound, acts as the sensor and DSF perception leads to activation of the genes involved in biofilm formation through RpfG. DSF perception also influences the interactions among RpfC with RpfF and a putative intracellular repressor. In the absence of RpfF, the putative repressor is sequestered by excessive RpfC, leading to expression of the downstream virulence genes, such as *tolC*, *pglA*, and *PD0279*, which are required for colonization and movement of Xy. fastidiosa in host plant. In another pathway, in the absence of RpfC, DSF is overproduced and the excessive signals may be sensed by a low-affinity intracellular DSF sensor, leading to expression of the genes required for attachment and biofilm formation. The response regulator RpfG was also proposed to be involved in this signaling pathway. Genetic analysis showed that RpfC regulated the expression of RpfG, RpfB, and a GGDEF domain protein that may be involved in intracellular signaling by modulating the levels of c-di-GMP. Much work is needed to elucidate the seemingly complex DSF-based QS signaling pathways and regulatory mechanisms in this important bacterial pathogen.

## 4.2. Xanthomonas axonopodis pv citri

X. axonopodis pv citri causes bacterial citrus canker, a serious disease affecting most citrus species. It belongs to γ-proteobacteria and shares >80% similarity with Xcc in genome sequence. X. axonopodis pv citri genome has an *rpf* locus containing *rpfF*, *rpfC*, and *rpfG*, which share ~54–96% amino acid identity with their homologues in Xcc. Like Xcc, the RpfF is the key enzyme for DSF signal generation, as deletion of *rpfF* abolished DSF biosynthesis. Interestingly, while the RpfC homologue of X. axonopodis pv citri shares only ~58% identical amino acids with the RpfC of Xcc, its putative cognate response regulator RpfG contains >96% identical amino acids with its counterpart of Xcc. Additionally, it is worth noting that the RpfC/RpfG proteins of X. axonopodis pv citri carry identical domain architecture as the RpfC/RpfG of Xcc.

As could be expected from their high levels of genome sequence similarity, the DSF signaling system of X. axonopodis pv citri appears to play similar roles as the DSF-based QS system of Xcc. Crude DSF extracts from X. axonopodis pv citri restored the phenotypes of the Xcc *rpfF* deletion mutant, and deletion of *rpfF* or *rpfC* attenuated the ability of X. axonopodis pv citri to cause canker symptoms in lemon leaves. However, the putative DSF-like signal from X. axonopodis pv citri has not yet been purified and characterized. While information is still limited and fragmented, several lines of evidence indicate that X. axonopodis pv citri and Xcc may share very similar DSF signaling mechanisms. First, yeast two-hybrid and in vitro biochemical analysis of X. axonopodis pv citri proteins show that RpfC interacted with RpfF, and deletion of *rpfC* caused DSF overproduction in X. axonopodis pv citri, reminiscent of their counterparts in Xcc. The findings suggest that DSF biosynthesis in X. axonopodis pv citri could also be regulated post-translationally through protein–protein interaction. Second, it was found that the DNA-binding activities of the Clp from X. axonopodis pv citri and that from Xcc were both inhibited by c-di-GMP at its physiological concentrations. Interestingly, the yeast two-hybrid analysis of X. axonopodis pv citri proteins revealed that RpfC can also interact with RpfG, and the HD-GYP domain of RpfG could interact with a number of GGDEF domain-containing proteins. Further investigation on the roles and mechanisms of these protein–protein interactions may provide new insight on the sophisticated DSF-dependent QS regulatory mechanisms.

## 4.3. Xanthomonas oryzae pv oryzae

X. oryzae pv oryzae is a member of γ-proteobacteria and the causal agent of rice bacterial blight disease, which is the most serious bacterial disease of rice worldwide. The DSF signaling system in X. oryzae pv oryzae positively regulates virulence and shares the same key components of DSF biosynthesis and signaling system of Xcc. Null mutation of the *rpfF* homologue of X. oryzae pv oryzae abolished the production of DSF-family molecules and reduced the bacterial virulence. Interestingly, while the *rpfF* mutant of a X. oryzae pv oryzae Korea strain KACC10331 showed the decreased production of EPS and extracellular enzymes, the same gene mutation in an India isolate BXO1 caused proficient production of these virulence factors. This different regulation pattern seems to suggest a possible strain-specific variation in the nature of DSF regulation in X. oryzae pv oryzae, which awaits further investigation. Structural characterization revealed that X. oryzae pv oryzae produced three members of DSF-family signals, i.e., DSF, cis-2-dodecenoic acid (C₁₂:Δ²), and cis,cis-11-methyldodeca-2,5-dienoic acid (11-Me-C₁₂:Δ²,⁵) (Table 1), with DSF being the major signal. These three DSF-family signals were biologically active and could effectively restore the EPS production and extracellular xylanase activity in the *rpfF* deletion mutant of X. oryzae pv oryzae. Apart from the regulation of virulence, DSF-based QS mechanisms are also implicated in regulation of other biological activities. The *rpfF* mutant of X. oryzae pv oryzae exhibited an unusual tetracycline susceptibility phenotype, overproduction of siderophores, and

growth deficiency under low iron conditions.⁸⁹ Intriguingly, while the *rpfC* gene of *X. oryzae pv oryzae* could complement the *rpfC* mutant of *Xcc*, mutation of *rpfC* in *X. oryzae pv oryzae* had no effect on production of extracellular enzymes.⁸⁸ The findings highlight functional variations of RpfC homologues in different bacterial species. Nevertheless, the role of RpfC in regulation of *X. oryzae pv oryzae* virulence is well conserved as its null mutants show reduced EPS biosynthesis and virulence.⁸⁸

### 4.4. Xanthomonas axonopodis pv glycines

*X. axonopodis pv glycines* belongs to γ-proteobacteria and is the causal agent of bacterial pustule of soybean, a serious disease that results in reduced yield and deteriorated crop quality in several major soybean-producing countries.⁹⁰ Genomic sequencing analysis revealed that *X. axonopodis pv glycines* contains an *rpfF* homologue sharing >95% identity with the *rpfF* of *Xcc*.³⁵ Such a high level of sequence similarity suggests that it could encode for the biosynthesis of putative DSF-family signals. Mutation of *rpfF* in *X. axonopodis pv glycines* resulted in decreased production of extracellular polysaccharides and extracellular enzymes, and reduced virulence on soybean.³⁵ Consistent with the phenotype analysis, transcriptional expression of the genes encoding extracellular enzymes were downregulated in the *rpfF* mutant,³⁵ suggesting that the DSF signaling system plays a key role in regulation of exoenzyme production and virulence in *X. axonopodis pv glycines*.

### 4.5. Stenotrophomonas maltophilia

*S. maltophilia* belongs to γ-proteobacteria and is widespread in natural environments such as soil, water, and plants.⁹¹ The organism has lately become widely known as an emerging opportunistic pathogen associated with nosocomial colonization and infection.⁹¹⁻⁹³ The *rpf* gene cluster is also well conserved in *S. maltophilia*, sharing ~62–82% similarity at the peptide sequence level with their counterparts in *Xcc*.³⁸ Interestingly, apart from DSF, seven other structural derivatives have also been identified in the supernatants of *S. maltophilia* with ESI-MS and GC-MS analysis³⁶ (Table 1). Two of them, 11-methyldodecanoic acid (11-Me-C₁₂) and cis-Δ2-11-methyldodecenoic acid (11-Me-C₁₂:Δ²), were verified for their biological activity to promote flagella-independent translocation with the corresponding synthetic compounds.³⁶ Genetic analysis showed that the DSF signaling system in *S. maltophilia* plays an important role in regulation of various biological functions. Mutation of *rpfF* in *S. maltophilia* led to reduced motility, decreased production of extracellular proteases, decreased tolerance to a range of antibiotics and heavy metals, altered lipopolysaccharide (LPS) structure, formation of cell aggregates, and attenuated bacterial virulence on the model organism nematode.³⁸ In another study,⁹⁴ it was shown that inactivation of *rpfF* or *rpfB* in *S. maltophilia* strain WR-C decreased the transcription of *fecA*, which encodes a ferric citrate receptor that transports exogenous siderophore-ferric citrate from environment into the bacterial periplasm. Exogenous addition of synthetic DSF to the mutant Δ*rpfF* restored *fecA* expression, establishing a link between the DSF-based QS system and ferric citrate uptake in this bacterial pathogen.

Interestingly, in contrast to *Xcc*, where DSF positively modulates the transcriptional expression of the c-di-GMP effector Clp,⁴⁴,⁶²,⁶³ the cNMP binding domain protein encoded by *crp* from *S. maltophilia* acts to positively regulate the *rpfF* expression, possibly by binding the upstream regions of the *rpfF* promoter.⁹⁴ The Crp from *S. maltophilia* and Clp shares >86% identical amino acids, and not surprisingly, both proteins show similar ligand specificity. A recent study showed that, similar to the Clp of *Xcc*, the Crp from *S. maltophilia* did not interact with cAMP but bound strongly with c-di-GMP.⁶³ These indirect lines of evidence suggest the association of Crp and c-di-GMP with the DSF signaling pathways in *S. maltophilia*.

### 4.6. Burkholderia cepacia complex

*B. cepacia* complex is a group of closely related and phenotypically diverse bacterial organisms, which currently is composed of 17 formally named bacterial species.⁹⁵,⁹⁶ Taxonomically, *B. cepacia* complex belongs to β-proteobacteria group. Strains of *B. cepacia* complex are ubiquitously distributed in nature, and some of them have emerged as problematic opportunistic pathogens in patients with cystic fibrosis or chronic granulomatous disease and immunocompromised individuals.⁹⁶⁻¹⁰¹ A DSF functional analogue designated BDSF from *B. cenocepacia* has recently been identified and characterized as cis-2-dodecenoic acid (C₁₂:Δ²).²⁷ The chemical structure of C₁₂:Δ² is similar to that of DSF of *Xcc* but differs in methyl substitution at C11-site. Genetic analysis showed that the ORF *Bcam0581*, which encodes an enzyme sharing 37% identity at the peptide sequence level with the RpfF of *Xcc*, was essential for C₁₂:Δ² production.²⁷ *Bcam0581* could genetically substitute for the DSF synthase gene *rpfF* in *Xcc*.²⁷ However, in contrast to the *rpfF* of *Xcc*, which is located within the same locus as *rpfC* and *rpfG*, no *rpfC* or *rpfG* homologue is found in the vicinity of *Bcam0581*. It was suggested that the QS system in *B. cenocepacia* could have a different origin of evolution.²⁷

While the genetic organization of the *rpf* genes differs, the biological functions of C₁₂:Δ² in *B. cenocepacia* are reminiscent of those regulated by DSF in the model organism *Xcc*. Deletion of *Bcam0581* caused retarded bacterial growth in minimal medium, reduced bacterial motility, and compromised biofilm attachment and virulence.²⁸,¹⁰²⁻¹⁰⁴ Genetic and biochemical analyses suggested that certain biological functions, including motility, biofilm formation, and virulence, were coregulated by a C₁₂:Δ² signaling system and the AHL-dependent QS system in *B. cenocepacia*.²⁸,¹⁰²⁻¹⁰⁴ (Figure 3). More recently, a mutational screen identified *Bcam0227* as a potential C₁₂:Δ² sensor, sharing 35.6% identity with RpfC of *Xcc*.¹⁰⁴ *Bcam0227* contains four domains, including a histidine kinase phosphoacceptor domain, a histidine kinase domain, a CheY-like receiver domain, and a C-terminal histidine phosphotransfer domain.¹⁰⁴ Transcriptome profile analysis revealed that the sensor controls the expression of a subset of target genes belonging to the C₁₂:Δ² regulon.¹⁰⁴ Deletion of *Bcam0227* attenuated bacterial virulence,¹⁰⁴ suggesting that the C₁₂:Δ²-based QS signaling system plays an important role in the physiology of *B. cenocepacia*.

In addition to *B. cenocepacia*, a subsequent study showed that other *B. cepacia* complex members also produced the QS signal C₁₂:Δ².²⁸ In this study, DSF signal production was determined in nine *B. cepacia* complex members. The results showed that like *B. cenocepacia*, four *B. cepacia* complex species, namely, *B. lata*, *B. vietnamiensis*, *B. dolosa*, and *B. ambifaria*, only produced C₁₂:Δ², whereas *B. multivorans*,

cation and characterization of the putative DSF signaling system in *P. aeruginosa* a tough challenge.

# 5. Interaction with Other Signaling Mechanisms

One of the interesting features of the DSF-based QS is that QS signal perception is coupled to intracellular regulatory networks through a second messenger c-di-GMP and its effector Clp.²⁵,⁶³ Bacterial pathogens may produce many c-di-GMP metabolic enzymes. For example, the genome of *Xcc* encodes at least 37 proteins with GGDEF, EAL, or HD-GYP domains, which are known to be involved in c-di-GMP biosynthesis and degradation, respectively.⁶⁰ The annotated genome of *P. aeruginosa* strain PAO1 encodes 17 proteins containing a single GGDEF domain, 5 with a single EAL domain, and 16 that carry both domains.¹⁰⁷ These findings appear to indicate that these c-di-GMP metabolic enzymes, which are commonly fused to other domains implicated in signaling sensing and receiving,²⁵ may influence DSF-based QS by changing the intracellular level of c-di-GMP. Consistent with this notion, a recent study has revealed a novel two-component system, RavS/RavR, that is involved in coregulation of *Xcc* virulence with the DSF-based QS system¹⁰⁸ (Figure 1). Null mutation of either RavS or RavR resulted in decreased virulence factor production.¹⁰⁸ Genetic and biochemical analyses showed that the functionality of RavR was dependent on its EAL domain-associated c-di-GMP phosphodiesterase activity.¹⁰⁸ RavS is the sensor kinase of this two-component system with its PAS domain being implicated in sensing low-oxygen tension. Deletion of the PAS domain or substitution of the key residues associated with hypoxia sensing abrogated the sensor activity in regulation of virulence.¹⁰⁸ Similar to DSF-dependent signaling systems, RavS/RavR also positively regulates the transcriptional expression of the global regulator Clp.¹⁰⁸ These findings present an example of how QS and other environmental cue sensing systems could interact by influencing c-di-GMP and its cognate effector Clp in modulating bacterial physiology (Figure 1).

*B. cenocepacia* may have evolved another type of interaction between two different QS signaling systems. In addition to the BDSF-based QS system as discussed in section 4.7, an AHL-type QS system is also presented in *Bcc* species including *B. cenocepacia*.¹⁰⁹,¹¹⁰ Recent studies showed that some of the BDSF-regulated functions were also controlled by the AHL-dependent QS system.¹⁰²⁻¹⁰⁴ Although the detailed mechanisms are not yet clear, the above findings suggest that the AHL- and BDSF-dependent QS systems regulate a similar set of virulence genes in parallel (Figure 3).

Adding to the diversity of interaction between signaling pathways, a physical interaction was found between the RpfC/RpfG two-component regulatory system and another two-component system consisting of NtrB/NtrC in *X. axonopodis pv citri*.³⁹ As discussed in section 4.2, RpfC and RpfG are the putative DSF sensor and cognate response regulator, respectively. The NtrB/NtrC system has been shown to regulate the σ⁵⁴-dependent promoters in some bacterial species,¹¹¹⁻¹¹³ putatively in response to a signal or environmental cue yet to be identified in *X. axonopodis pv citri*. Biochemical analysis showed that RpfC interacted with the HK domain of NtrB, whereas the HD-GYP domain of RpfG interacted with the σ⁵⁴-activating and DNA-binding domains of NtrC.³⁹ However, how these interactions influ-

ence the DSF signaling system in *X. axonopodis* pv *citri* is not yet clear.

# 6. Roles of DSF-Family Signals in Interspecies and Interkingdom Communication

It has been well established that DSF-family signals control a range of biological functions through intraspecies signaling. However, they may potentially play important roles in microbial ecology through interspecies and cross-kingdom communications.²⁴,²⁷,²⁸,¹¹⁴ Among different *Xanthomonads*, it is possible that interspecies signaling might occur frequently because these closely related bacterial pathogens produce common or similar DSF signals²⁴ (Table 1). DSF-dependent interspecies communication is not restricted to *Xanthomonads*. It was reported recently that the DSF signal produced by *S. maltophilia* influenced the biofilm formation and stress tolerance of *P. aeruginosa*.¹¹⁴ Another study showed that the DSF-family signal C₁₀:Δ² produced by *P. aeruginosa* not only dispersed its own biofilm formation but also induced dispersion of the biofilms of *E. coli*, *Klebsiella pneumoniae*, *Proteus mirabilis*, *Streptococcus pyogenes*, *Bacillus subtilis*, and *Staphylococcus aureus*.³⁷ In addition, BDSF from *B. cenocepacia* was found to be the functional homologue of DSF in the regulation of biofilm formation and virulence factor production by Xcc.²⁸ New evidence of this cross-species communication comes from the findings that a newly identified signal 11-Me-C₁₂:Δ²,⁵ from *B. multivorans* could functionally substitute DSF and C₁₂:Δ² to regulate the biofilm dispersal and virulence factor production in Xcc and *B. cenocepacia*, respectively.²⁸

Besides involvement in interspecies cross-talk, DSF-family signals, including DSF, C₁₂:Δ², and 11-Me-C₁₂:Δ²,⁵ have been shown to inhibit the morphological transition of *Candida albicans* through interkingdom interference.²⁴,²⁷,²⁸ Similarly, a fatty acid molecule trans-2-decenoic acid (C₁₀: Δ², designated SDSF) from Gram-positive bacterium *S. mutans* is also a potent inhibitor of *C. albicans* morphological transition.⁴⁰ In addition, the signal C₁₀:Δ² from *P. aeruginosa* was found to be able to induce biofilm dispersion in *C. albicans*, as well as diverse bacteria.³⁷ *C. albicans* is an eukaryotic yeast and a causal agent of opportunistic oral and genital infections in humans. This fungal pathogen is a dimorphic organism and can grow as yeast- or hyphal-form depending on environmental conditions. This ability to switch between yeast and hyphae is an important aspect of its pathogenesis. DSF signals are structurally similar to farnesoic acid, which is an autoregulatory substance in *C. albicans*. Farnesoic acid inhibits *C. albicans* germ tube formation and plays a key role in the regulation of morphological transition in *C. albicans*.¹¹⁵ These findings seem to suggest that DSF-family signals may use similar mechanisms as farnesoic acid in the modulation of *C. albicans* morphological transition.

# 7. Conclusions and Future Prospective

The DSF-based QS system, which was initially identified in a plant bacterial pathogen Xcc, has emerged as another widely conserved bacterial cell–cell communication mechanism in Gram-negative bacteria. The work on the DSF system in the model organism Xcc has unveiled several unique features that distinguish it from other known QS systems, including DSF autoinduction through RpfC/RpfF protein–protein interaction and QS coupling to bacterial intracellular regulatory network via a novel second messenger

c-di-GMP and its effector Clp. Rapid research progress over the past few years has uncovered the presence of DSF-family signals in a variety of bacterial species belonging to γ-proteobacteria and β-proteobacteria groups, including both plant and human bacterial pathogens. Similar to the well-characterized AHL-based QS systems, DSF signaling systems are known to regulate a range of important biological functions such as biofilm formation, cell growth, and virulence factor production, which are critical for bacterial survival and pathogenesis. Identification of this widely conserved DSF-based signaling mechanism presents further evidence that argues strongly that QS is an indispensable community genetic regulation mechanism essential for bacterial physiology and virulence.

DSF-based QS systems can be grouped into two categories according to their genomic evolutionary origins. The first category, represented by Xcc, is distinguished by colocalization of the genes encoding key components of the QS system such as RpfF, RpfC, and RpfG. Other DSF-producing γ-proteobacteria species, except *P. aeruginosa*, also keep a similar genome organization to Xcc and share high levels of homology among corresponding *rpf* homologues. In contrast, the *rpfF* homologue *Bcam0581* and the *rpfC* homologue *Bcam0227* are located in different places within the genome of *B. cenocepacia*, which is a β-proteobacteria species. Significantly, while the Rpf proteins of *B. cenocepacia* do not share a high level of similarity to their counterparts in Xcc, the DSF-dependent biological functions, including virulence factor production and biofilm dispersion, are conserved in both γ-proteobacteria and β-proteobacteria.

The identification of DSF-family signals as a widely conserved mechanism in Gram-negative bacteria presents numerous challenges for further investigation. Recognition of the important roles of DSF-based QS systems in bacterial physiology might motivate further screening and characterization of putative DSF-family signals and corresponding signaling systems in other bacterial pathogens. It is intriguing to determine whether, and if so, how other key factors of the DSF-based QS system identified in Xcc and other γ-proteobacteria species, such as c-di-GMP and its cognate effector Clp, play roles in the DSF signaling pathways of β-proteobacteria species. Even in the model organism Xcc, although the framework of DSF signaling pathway has been well established, little is known how the sensor RpfC may perceive DSF signals and how the sensor–ligand interaction could induce the proposed conformational changes in RpfC. In addition, the DSF biosynthetic pathway and catalytic mechanisms remain to be a mystery. Furthermore, much work is needed to elucidate how DSF controls the transcriptional and translational expression of the XagABC system associated with biofilm formation. Moreover, given that trans-2-decenoic acid is produced by *S. mutans*,⁴⁰ it is intriguing to determine whether DSF-family signals play a role as QS signals in Gram-positive bacterial pathogens.

## References

(1) Federle, M. J.; Bassler, B. L. *Clin. Invest.* **2003**, *112*, 1291.

(2) Fuqua, C.; Greenberg, E. P. *Nat. Rev. Mol. Cell. Biol.* **2002**, *3*, 685.

(3) Fuqua, W. C.; Winans, S. C.; Greenberg, E. P. *J. Bacteriol.* **1994**, *176*, 269.

(4) Whitehead, N. A.; Barnard, A. M.; Slater, H.; Simpson, N. J.; Salmond, G. P. *FEMS Microbiol. Rev.* **2001**, *25*, 365.

(5) Fuqua, C.; Winans, S. C.; Greenberg, E. P. *Annu. Rev. Microbiol.* **1996**, *50*, 727.

(6) Ohtani, K.; Hayashi, H.; Shimizu, T. *Mol. Microbiol.* **2002**, *44*, 171.

(7) Marketon, M. M.; Glenn, S. A.; Eberhard, A.; Gonzalez, J. E. J. Bacteriol. 2003, 185, 325.

(8) Quinones, B.; Dulla, G.; Lindow, S. E. Mol. Plant–Microbe Interact. 2005, 18, 682.

(9) Rice, S. A.; Koh, K. S.; Queck, S. Y.; Labbate, M.; Lam, K. W.; Kjelleberg, S. J. Bacteriol. 2005, 187, 3477.

(10) Eberhard, A.; Burlingame, A. L.; Eberhard, C.; Kenyon, G. L.; Nealson, K. H.; Oppenheimer, N. J. Biochemistry 1981, 20, 2444.

(11) Engebrecht, J.; Nealson, K.; Silverman, M. Cell 1983, 32, 773.

(12) Swartzman, A.; Kapoor, S.; Graham, A. F.; Meighen, E. A. J. Bacteriol. 1990, 172, 6797.

(13) Williams, P. Microbiology 2007, 153, 3923.

(14) Degrassi, G.; Aguilar, C.; Bosco, M.; Zahariév, S.; Pongor, S.; Venturi, V. Curr. Microbiol. 2002, 45, 250.

(15) Holden, M. T.; Ram Chhabra, S.; de Nys, R.; Stead, P.; Bainton, N. J.; Hill, P. J.; Manefield, M.; Kumar, N.; Labatte, M.; England, D.; Rice, S.; Givskov, M.; Salmond, G. P.; Stewart, G. S.; Bycroft, B. W.; Kjelleberg, S.; Williams, P. Mol. Microbiol. 1999, 33, 1254.

(16) Bassler, B. L.; Wright, M.; Showalter, R. E.; Silverman, M. R. Mol. Microbiol. 1993, 9, 773.

(17) Miller, S. T.; Xavier, K. B.; Campagna, S. R.; Taga, M. E.; Semmelhack, M. F.; Bassler, B. L.; Hughson, F. M. Mol. Cell 2004, 15, 677.

(18) Flavier, A. B.; Clough, S. J.; Schell, M. A.; Denny, T. P. Mol. Microbiol. 1997, 26, 251.

(19) Pesci, E. C.; Milbank, J. B.; Pearson, J. P.; McKnight, S.; Kende, A. S.; Greenberg, E. P.; Iglewski, B. H. Proc. Natl. Acad. Sci. U. S. A. 1999, 96, 11229.

(20) Loh, J.; Lohar, D. P.; Andersen, B.; Stacey, G. J. Bacteriol. 2002, 184, 1759.

(21) Sperandio, V.; Torres, A. G.; Jarvis, B.; Nataro, J. P.; Kaper, J. B. Proc. Natl. Acad. Sci. U. S. A. 2003, 100, 8951.

(22) Diggle, S. P.; Lumjiaktase, P.; Dipilato, F.; Winzer, K.; Kunakorn, M.; Barrett, D. A.; Chhabra, S. R.; Camara, M.; Williams, P. Chem. Biol. 2006, 13, 701.

(23) Barber, C. E.; Tang, J. L.; Feng, J. X.; Pan, M. Q.; Wilson, T. J.; Slater, H.; Dow, J. M.; Williams, P.; Daniels, M. J. Mol. Microbiol. 1997, 24, 555.

(24) Wang, L. H.; He, Y.; Gao, Y.; Wu, J. E.; Dong, Y. H.; He, C.; Wang, S. X.; Weng, L. X.; Xu, J. L.; Tay, L.; Fang, R. X.; Zhang, L. H. Mol. Microbiol. 2004, 51, 903.

(25) He, Y. W.; Zhang, L. H. FEMS Microbiol. Rev. 2008, 32, 842.

(26) Slater, H.; Alvarez-Morales, A.; Barber, C. E.; Daniels, M. J.; Dow, J. M. Mol. Microbiol. 2000, 38, 986.

(27) Boon, C.; Deng, Y.; Wang, L. H.; He, Y.; Xu, J. L.; Fan, Y.; Pan, S. Q.; Zhang, L. H. ISME J. 2008, 2, 27.

(28) Deng, Y.; Wu, J.; Eberl, L.; Zhang, L. H. Appl. Environ. Microbiol. 2010, 76, 4675.

(29) He, Y. W.; Wu, J.; Cha, J. S.; Zhang, L. H. BMC Microbiol. 2010, 10, 187.

(30) Scarpini, L. M.; Lambais, M. R.; Silva, D. S.; Carraro, D. M.; Carrer, H. FEMS Microbiol. Lett. 2003, 222, 83.

(31) Siciliano, F.; Torres, P.; Sendín, L.; Bermejo, C.; Filippone, P.; Vellice, G.; Ramallo, J.; Castagnaro, A.; Vojnov, A.; Marano, M. R. Electron. J. Biotechnol. 2006, 9, 199.

(32) Colnaghi Simionato, A. V.; da Silva, D. S.; Lambais, M. R.; Carrilho, E. J. Mass. Spectrom. 2007, 42, 490.

(33) Newman, K. L.; Almeida, R. P.; Purcell, A. H.; Lindow, S. E. Proc. Natl. Acad. Sci. U. S. A. 2004, 101, 1737.

(34) Chatterjee, S.; Wistrom, C.; Lindow, S. E. Proc. Natl. Acad. Sci. U. S. A. 2008, 105, 2670.

(35) Thowthampitak, J.; Shaffer, B. T.; Prathuangwong, S.; Loper, J. E. Phytopathology 2008, 98, 1252.

(36) Huang, T. P.; Lee Wong, A. C. Res. Microbiol. 2007, 158, 702.

(37) Davies, D. G.; Marques, C. N. J. Bacteriol. 2009, 191, 1393.

(38) Fouhy, Y.; Scanlon, K.; Schouest, K.; Spillane, C.; Crossman, L.; Avison, M. B.; Ryan, R. P.; Dow, J. M. J. Bacteriol. 2007, 189, 4964.

(39) Andrade, M. O.; Alegria, M. C.; Guzzo, C. R.; Docena, C.; Rosa, M. C.; Ramos, C. H.; Farah, C. S. Mol. Microbiol. 2006, 62, 537.

(40) Vilchez, R.; Lemme, A.; Ballhausen, B.; Thiel, V.; Schulz, S.; Jansen, R.; Sztajer, H.; Wagner-Döbler, I. ChemBioChem 2010, 11, 1552.

(41) Onsando, J. M. Diseases of Vegetables and Oil Seed Crops; Chaube, H. S., Kumar, J., Mukhopadhyay, A. N., Singh, U. S., Eds.; Prentice-Hall: Englewood Cliffs, NJ, 1992; Vol. II, pp 243–252.

(42) Tang, J. L.; Liu, Y. N.; Barber, C. E.; Dow, J. M.; Wootton, J. C.; Daniels, M. J. Mol. Gen. Genet. 1991, 226, 409.

(43) Cheng, Z.; He, Y. W.; Lim, S. C.; Qamra, R.; Walsh, M. A.; Zhang, L. H.; Song, H. Structure 2010, 18, 1199.

(44) He, Y. W.; Xu, M.; Lin, K.; Ng, Y. J.; Wen, C. M.; Wang, L. H.; Liu, Z. D.; Zhang, H. B.; Dong, Y. H.; Dow, J. M.; Zhang, L. H. Mol. Microbiol. 2006, 59, 610.

(45) He, Y. W.; Wang, C.; Zhou, L.; Song, H.; Dow, J. M.; Zhang, L. H. J. Biol. Chem. 2006, 281, 33414.

(46) Dow, J. M.; Crossman, L.; Findlay, K.; He, Y. Q.; Feng, J. X.; Tang, J. L. Proc. Natl. Acad. Sci. U. S. A. 2003, 100, 10995.

(47) Ryan, R. P.; Fouhy, Y.; Lucey, J. F.; Crossman, L. C.; Spiro, S.; He, Y. W.; Zhang, L. H.; Heeb, S.; Camara, M.; Williams, P.; Dow, J. M. Proc. Natl. Acad. Sci. U. S. A. 2006, 103, 6712.

(48) He, Y. W. Cell–cell communication in *Xanthomonas campestris* pv. *campestris*. Ph.D. Thesis, University of Singapore, Singapore, 2006.

(49) Ross, P.; Weinhouse, H.; Aloni, Y.; Michaeli, D.; Weinberger-Ohana, P.; Mayer, R.; Braun, S.; de Vroom, E.; van der Marel, G. A.; van Boom, J. H.; Benziman, M. Nature 1987, 325, 279.

(50) Simm, R.; Morr, M.; Kader, A.; Nimtz, M.; Romling, U. Mol. Microbiol. 2004, 53, 1123.

(51) Hickman, J. W.; Tifrea, D. F.; Harwood, C. S. Proc. Natl. Acad. Sci. U. S. A. 2005, 102, 14422.

(52) Tischler, A. D.; Camilli, A. Infect. Immun. 2005, 73, 5873.

(53) Tal, R.; Wong, H. C.; Calhoon, R.; Gelfand, D.; Fear, A. L.; Volman, G.; Mayer, R.; Ross, P.; Amikam, D.; Weinhouse, H.; Cohen, A.; Sapir, S.; Ohana, P.; Benziman, M. J. Bacteriol. 1998, 180, 4416.

(54) Paul, R.; Weiser, S.; Amiot, N. C.; Chan, C.; Schirmer, T.; Giese, B.; Jenal, U. Genes Dev. 2004, 18, 715.

(55) Christen, M.; Christen, B.; Folcher, M.; Schauerte, A.; Jenal, U. J. Biol. Chem. 2005, 280, 30829.

(56) Schmidt, A. J.; Ryjenkov, D. A.; Gomelsky, M. J. Bacteriol. 2005, 187, 4774.

(57) Galperin, M. Y.; Natale, D. A.; Aravind, L.; Koonin, E. V. J. Mol. Microbiol. Biotechnol. 1999, 1, 303.

(58) Galperin, M. Y.; Nikolskaya, A. N.; Koonin, E. V. FEMS Microbiol. Lett. 2001, 203, 11.

(59) Ryan, R. P.; McCarthy, Y.; Andrade, M.; Farah, C. S.; Armitage, J. P.; Dow, J. M. Proc. Natl. Acad. Sci. U. S. A. 2010, 107, 5989.

(60) Ryan, R. P.; Fouhy, Y.; Lucey, J. F.; Jiang, B. L.; He, Y. Q.; Feng, J. X.; Tang, J. L.; Dow, J. M. Mol. Microbiol. 2007, 63, 429.

(61) de Crecy-Lagard, V.; Glaser, P.; Lejeune, P.; Sismeiro, O.; Barber, C. E.; Daniels, M. J.; Danchin, A. J. Bacteriol. 1990, 172, 5877.

(62) He, Y. W.; Ng, A. Y.; Xu, M.; Lin, K.; Wang, L. H.; Dong, Y. H.; Zhang, L. H. Mol. Microbiol. 2007, 64, 281.

(63) Tao, F.; He, Y. W.; Wu, D. H.; Swarup, S.; Zhang, L. H. J. Bacteriol. 2010, 192, 1020.

(64) Leduc, J. L.; Roberts, G. P. J. Bacteriol. 2009, 191, 7121.

(65) Dong, Q.; Ebright, R. H. J. Bacteriol. 1992, 174, 5457.

(66) Hsiao, Y. M.; Liao, H. Y.; Lee, M. C.; Yang, T. C.; Tseng, Y. H. FEBS Lett. 2005, 579, 3525.

(67) Chin, K. H.; Lee, Y. C.; Tu, Z. L.; Chen, C. H.; Tseng, Y. H.; Yang, J. M.; Ryan, R. P.; McCarthy, Y.; Dow, J. M.; Wang, A. H.; Chou, S. H. J. Mol. Biol. 2010, 396, 646.

(68) Ryjenkov, D. A.; Simm, R.; Romling, U.; Gomelsky, M. J. Biol. Chem. 2006, 281, 30310.

(69) Hickman, J. W.; Harwood, C. S. Mol. Microbiol. 2008, 69, 376.

(70) Krasteva, P. V.; Fong, J. C.; Shikuma, N. J.; Beyhan, S.; Navarro, M. V.; Yildiz, F. H.; Sondermann, H. Science 2010, 327, 866.

(71) Newell, P. D.; Monds, R. D.; O’Toole, G. A. Proc. Natl. Acad. Sci. U. S. A. 2009, 106, 3461.

(72) Navarro, M. V.; De, N.; Bae, N.; Wang, Q.; Sondermann, H. Structure 2009, 17, 1104.

(73) Lee, V. T.; Matewish, J. M.; Kessler, J. L.; Hyodo, M.; Hayakawa, Y.; Lory, S. Mol. Microbiol. 2007, 65, 1474.

(74) Stock, A. M.; Robinson, V. L.; Goudreau, P. N. Annu. Rev. Biochem. 2000, 69, 183.

(75) Kleerebezem, M.; Quadri, L. E.; Kuipers, O. P.; de Vos, W. M. Mol. Microbiol. 1997, 24, 895.

(76) Tao, F.; Swarup, S.; Zhang, L. H. Environ. Microbiol. 2010, 12, 3159.

(77) Wang, X.; Dubey, A. K.; Suzuki, K.; Baker, C. S.; Babitzke, P.; Romeo, T. Mol. Microbiol. 2005, 56, 1648.

(78) Bobrov, A. G.; Kirillina, O.; Forman, S.; Mack, D.; Perry, R. D. Environ. Microbiol. 2008, 10, 1419.

(79) Hsiao, Y. M.; Liu, Y. F.; Fang, M. C.; Tseng, Y. H. J. Agric. Food Chem. 2010, 58, 1653.

(80) Hopkins, D. L.; Mollenhauer, H. H. Science 1973, 179, 298.

(81) Davis, M. J.; Purcell, A. H.; Thomson, S. V. Science 1978, 199, 75.

(82) Henderson, M.; Purcell, A. H.; Chen, D.; Smart, C.; Guilhabert, M.; Kirkpatrick, B. Appl. Environ. Microbiol. 2001, 67, 895.

(83) Purcell, A. H.; Finlay, A. H.; McLean, D. L. Science 1979, 206, 839.

(84) Purcell, A. H.; Hopkins, D. L. Annu. Rev. Phytopathol. 1996, 34, 131.

(85) Graham, J. H.; Gottwald, T. R.; Cubero, J.; Achor, D. S. Mol. Plant Pathol. 2004, 5, 1.

(86) da Silva, A. C.; Ferro, J. A.; Reinach, F. C.; Farah, C. S.; Furlan, L. R.; Quaggio, R. B.; Monteiro-Vitorello, C. B.; Van Sluys, M. A.; Almeida, N. F.; Alves, L. M.; do Amaral, A. M.; Bertolini, M. C.;

Camargo, L. E.; Camarotte, G.; Cannavan, F.; Cardozo, J.; Chamberto, F.; Ciapina, L. P.; Cicarelli, R. M.; Coutinho, L. L.; Cursino-Santos, J. R.; El-Dorry, H.; Faria, J. B.; Ferreira, A. J.; Ferreira, R. C.; Ferro, M. I.; Formighieri, E. F.; Franco, M. C.; Greggio, C. C.; Gruber, A.; Katsuyama, A. M.; Kishi, L. T.; Leite, R. P.; Lemos, E. G.; Lemos, M. V.; Locali, E. C.; Machado, M. A.; Madeira, A. M.; Martinez-Rossi, N. M.; Martins, E. C.; Meidanis, J.; Menck, C. F.; Miyaki, C. Y.; Moon, D. H.; Moreira, L. M.; Novo, M. T.; Okura, V. K.; Oliveira, M. C.; Oliveira, V. R.; Pereira, H. A.; Rossi, A.; Sena, J. A.; Silva, C.; de Souza, R. F.; Spinola, L. A.; Takita, M. A.; Tamura, R. E.; Teixeira, E. C.; Tezza, R. I.; Trindade dos Santos, M.; Truffi, D.; Tsai, S. M.; White, F. F.; Setubal, J. C.; Kitajima, J. P. *Nature* **2002**, *417*, 459.

(87) Ou, S. H. *Rice Diseases*, 2nd ed.; Commonwealth Mycological Institute: Kew, U.K., 1985; pp 380.

(88) Tang, J. L.; Feng, J. X.; Li, Q. Q.; Wen, H. X.; Zhou, D. L.; Wilson, T. J.; Dow, J. M.; Ma, Q. S.; Daniels, M. J. *Mol. Plant—Microbe Interact.* **1996**, *9*, 664.

(89) Chatterjee, S.; Sonti, R. V. *Mol. Plant—Microbe Interact.* **2002**, *15*, 463.

(90) Wrather, J. A.; Anderson, T. R.; Arsyad, D. M.; Tan, Y.; Ploper, L. D.; Porta-Puglia, A.; Ram, H.; Yorinori, J. T. *Can. J. Plant. Pathol.* **2001**, *23*, 115.

(91) Denton, M.; Kerr, K. G. *Clin. Microbiol. Rev.* **1998**, *11*, 57.

(92) Minkwitz, A.; Berg, G. *J. Clin. Microbiol.* **2001**, *39*, 139.

(93) Looney, W. J. *Br. J. Biomed. Sci.* **2005**, *62*, 145.

(94) Huang, T. P.; Wong, A. C. *Appl. Environ. Microbiol.* **2007**, *73*, 5034.

(95) Vandamme, P.; Holmes, B.; Vancanneyt, M.; Coenye, T.; Hoste, B.; Coopman, R.; Revets, H.; Lauwers, S.; Gillis, M.; Kersters, K.; Govan, J. R. *Int. J. Syst. Bacteriol.* **1997**, *47*, 1188.

(96) Mahenthiralingam, E.; Urban, T. A.; Goldberg, J. B. *Nat. Rev. Microbiol.* **2005**, *3*, 144.

(97) Govan, J. R.; Deretic, V. *Microbiol. Rev.* **1996**, *60*, 539.

(98) Mahenthiralingam, E.; Baldwin, A.; Vandamme, P. *J. Med. Microbiol.* **2002**, *51*, 533.

(99) Coenye, T.; Vandamme, P. *Environ. Microbiol.* **2003**, *5*, 719.

(100) Vanlaere, E.; Baldwin, A.; Gevers, D.; Henry, D.; De Brandt, E.; LiPuma, J. J.; Mahenthiralingam, E.; Speert, D. P.; Dowson, C.; Vandamme, P. *Int. J. Syst. Evol. Microbiol.* **2009**, *59*, 102.

(101) Vanlaere, E.; Lipuma, J. J.; Baldwin, A.; Henry, D.; De Brandt, E.; Mahenthiralingam, E.; Speert, D.; Dowson, C.; Vandamme, P. *Int. J. Syst. Evol. Microbiol.* **2008**, *58*, 1580.

(102) Ryan, R. P.; McCarthy, Y.; Watt, S. A.; Niehaus, K.; Dow, J. M. J. *Bacteriol.* **2009**, *191*, 5013.

(103) Deng, Y.; Boon, C.; Eberl, L.; Zhang, L. H. *J. Bacteriol.* **2009**, *191*, 7270.

(104) McCarthy, Y.; Yang, L.; Twomey, K. B.; Sass, A.; Tølke-Nielsen, T.; Mahenthiralingam, E.; Dow, J. M.; Ryan, R. P. *Mol. Microbiol.* **2010**, *77*, 1220.

(105) Bodey, G. P.; Bolivar, R.; Fainstein, V.; Jadeja, L. *Rev. Infect. Dis.* **1983**, *5*, 279.

(106) Richards, M. J.; Edwards, J. R.; Culver, D. H.; Gaynes, R. P. *Infect. Control Hosp. Epidemiol.* **2000**, *21*, 510.

(107) Kulasakara, H.; Lee, V.; Brencic, A.; Liberati, N.; Urbach, J.; Miyata, S.; Lee, D. G.; Neely, A. N.; Hyodo, M.; Hayakawa, Y.; Ausubel, F. M.; Lory, S. *Proc. Natl. Acad. Sci. U. S. A.* **2006**, *103*, 2839.

(108) He, Y. W.; Boon, C.; Zhou, L.; Zhang, L. H. *Mol. Microbiol.* **2009**, *71*, 1464.

(109) Eberl, L. *Int. J. Med. Microbiol.* **2006**, *296*, 103.

(110) Sokol, P. A.; Malott, R. J.; Riedel, K.; Eberl, L. *Future Microbiol.* **2007**, *2*, 555.

(111) Hirschman, J.; Wong, P. K.; Sei, K.; Keener, J.; Kustu, S. *Proc. Natl. Acad. Sci. U. S. A.* **1985**, *82*, 7525.

(112) Kustu, S.; Santero, E.; Keener, J.; Popham, D.; Weiss, D. *Microbiol. Rev.* **1989**, *53*, 367.

(113) Studholme, D. J.; Dixon, R. *J. Bacteriol.* **2003**, *185*, 1757.

(114) Ryan, R. P.; Fouhy, Y.; Garcia, B. F.; Watt, S. A.; Niehaus, K.; Yang, L.; Tølke-Nielsen, T.; Dow, J. M. *Mol. Microbiol.* **2008**, *68*, 75.

(115) Oh, K. B.; Miyazawa, H.; Naito, T.; Matsuoka, H. *Proc. Natl. Acad. Sci. U. S. A.* **2001**, *98*, 4664.
