
Molecular Microbiology (2011) 79(2), 359–374

doi:10.1111/j.1365-2958.2010.07448.x  
First published online 16 November 2010

# Unravelling the gallic acid degradation pathway in bacteria: the gal cluster from Pseudomonas putida

Juan Nogales,$^{1†}$ Ángeles Canales,$^{2}$ Jesús Jiménez-Barbero,$^{2}$ Beatriz Serra,$^{3}$ José Manuel Pingarrón,$^{3}$ José Luis García$^{1}$ and Eduardo Díaz$^{1*}$

Departments of $^{1}$Environmental Biology and $^{2}$Chemical and Physical Biology, Centro de Investigaciones Biológicas, CSIC. Ramiro de Maeztu 9, 28040 Madrid, Spain.  
$^{3}$Department of Analytical Chemistry, Faculty of Chemistry, Universidad Complutense de Madrid. 28040 Madrid, Spain.

## Summary

Gallic acid (3,4,5-trihydroxybenzoic acid, GA) is widely distributed in nature, being a major phenolic pollutant and a commonly used antioxidant and building-block for drug development. We have characterized the first complete cluster (*gal* genes) responsible for growth in GA in a derivative of the model bacterium *Pseudomonas putida* KT2440. GalT mediates specific GA uptake and chemotaxis, and highlights the critical role of GA transport in bacterial adaptation to GA consumption. The proposed GA degradation via the central intermediate 4-oxalomesaconic acid (OMA) was revisited and all enzymes involved have been identified. Thus, GalD is the prototype of a new subfamily of isomerases that catalyses a biochemical step that remained unknown, i.e. the tautomerization of the OMAketo generated by the GalA dioxygenase to OMAenol. GalB is the founding member of a new family of zinc-containing hydratases that converts OMAenol into 4-carboxy-4-hydroxy-2-oxoadipic acid (CHA). *galC* encodes the aldolase catalysing CHA cleavage to pyruvic and oxaloacetic acids. The presence of homologous *gal* clusters outside the *Pseudomonas* genus sheds light on the evolution and ecology of the *gal* genes in GA degraders. The *gal* genes were used for expanding the metabolic abilities of heterologous hosts towards GA degradation, and for engineering a GA cellular biosensor.

## Introduction

Gallic acid (3,4,5-trihydroxybenzoic acid, GA) is an common phenolic compound of plant origin that can be found in the free state or in the form of esters (e.g. tannins) or ethers (e.g. syringic acid and other lignin constituents), being a major pollutant present in the wastewaters generated in the boiling cork process and in food manufacturing industries. GA and its derivatives are used in industry as antioxidants, and they are of interest also in drug development and as antimicrobial agents (Ow and Stupans, 2003). Although GA is widely distributed in nature, it is easily oxidized at neutral or alkaline pH becoming a difficult carbon source for bacterial growth. In fact, only bacteria of the genus *Pseudomonas* have been reported to utilize free GA as sole carbon and energy source under aerobic conditions (Beveridge and Hugo, 1964; Chowdhury *et al.*, 2004). Moreover, the oxidation of GA also hinders the study of its degradation under laboratory conditions. The aerobic metabolism of GA usually starts with a direct ring-cleavage reaction and formation of the central intermediate 4-oxalomesaconic acid (OMA), which then undergoes hydration to 4-carboxy-4-hydroxy-2-oxoadipic acid (CHA) and aldolic cleavage to oxaloacetic and pyruvic acids (Fig. 1A). This pathway has been studied in some *Pseudomonas* and *Sphingomonas* strains that, surprisingly, cannot use GA as sole carbon source but are able to metabolize the GA generated when they grow at the expense of syringic acid (Tack *et al.*, 1972a,b; Sparnins and Dagley, 1975; Kersten *et al.*, 1982; Kasai *et al.*, 2005). In *Sphingomonas paucimobilis* SYK-6, the *lig* genes involved in the meta-cleavage pathway for protocatechuic acid degradation via OMA are also involved in the lower pathway for syringic acid metabolism (Kasai *et al.*, 2005). On the contrary, in most *Pseudomonas* strains protocatechuic acid is mineralized via ortho-cleavage throughout the β-ketoadipic acid pathway (Harwood and Parales, 1996) and therefore they have evolved a devoted pathway for GA degradation via OMA most of whose genetic determinants are still unknown.

Accepted 25 October, 2010. *For correspondence. E-mail ediaz@cib.csic.es; Tel. (+34) 918373112; Fax: (+34) 915360432.* †Present address: Center for Systems Biology, University of Iceland, Sturlugata 8, 101 Reykjavik, Iceland.

© 2010 Blackwell Publishing Ltd

A

![Diagram](https://i.imgur.com/yourimageurl.png)

B

![Graph](https://i.imgur.com/yourimageurl2.png)

C

![Genetic Organization Diagram](https://i.imgur.com/yourimageurl3.png)

Fig. 1. GA degradation pathway.

A. Scheme of the GA degradation pathway in *P. putida*. GalP, outer membrane porin (pink); GalT, GA transporter (dark blue); GalA, GA dioxygenase (red); GalD, OMA keto-enol tautomerase (light blue); GalB, OMAenol hydratase (green); GalC, CHA aldolase (yellow). Hatched arrows show the previously proposed ring-cleavage and hydration reactions involving the three OMA isomers (grey). The GalD- and GalB-catalysed reactions are the reversible steps of GA degradation.

B. Growth of *P. putida* KTGAL (violet), and *P. putida* KT2440 (red), *E. coli* W (green) and *Pseudomonas* sp. MT14 (blue) containing plasmids pGAL (■), pZGalT (●) or pZ1016 (□), in GA-containing MC minimal medium.

C. Genetic organization of the *gal* cluster in α-, β- and γ-Proteobacteria. Genes are functionally annotated following the colour code indicated in (A). Black, regulatory genes; striped, *ligJ*-like genes; brown, *ligAB*-like genes; white, genes of unknown function. Numbers indicate the percent amino acid sequence identity with the orthologue gene product from *P. putida* KTGAL. The abbreviations and accession codes for the gene clusters are: *P. putida*: Pseudomonas putida KT2440, PP_2513-2519 (the *galT* gene corresponds to that of *P. putida* KTGAL); Pseudomonas putida F1, Pput_3195-3201; Pseudomonas putida W619, PputW619-2026-30. A. *vinelandii*: Azotobacter vinelandii DJ, Avin_31190-31250. C. *salexigens*: Chromohalobacter salexigens DSM3043, Csal_0322-0331. S. *proteamaculans*: Serratia proteamaculans 568, Spro_2020-27. K. *pneumoniae*: Klebsiella pneumoniae subsp. pneumoniae MGH78578, KPN_04046-4051. C. *rodentium*: Citrobacter rodentium ICC168, ROC_31611-31672. B. *cepacia/B. multivorans*: Burkholderia cepacia R383, Bcep18194°C7565-7572; Burkholderia multivorans ATCC 17616, Bmul_6035-6042. R. *solanacearum*: Ralstonia solanacearum UW551, RSIPO_03319-03326. A. *excentricus*: Asticcacaulis excentricus CB 48, AstexDRAFT_3680-3687.

Pseudomonas putida KT2440 is a paradigmatic bacterium renowned for its ability to degrade a wide range of aromatic compounds (Nelson et al., 2002; Jiménez et al., 2010), and it can be also adapted to degrade GA as sole carbon and energy source (Nogales et al., 2005). In this work, we have identified and characterized in *P. putida* KT2440 the first complete gene cluster (*gal* genes) responsible for bacterial growth using GA as sole carbon and energy source. The GA uptake and the biochemistry of its degradation via OMA have been revisited, new enzymes were unravelled, and some evolutionary considerations regarding the *gal* cluster are also discussed.

Results and discussion

Transport drives bacterial adaptation to GA consumption

The *in silico* analysis of the *P. putida* KT2440 genome revealed a 8 kb *gal* cluster (Fig. 1C), which included the *galA* gene encoding an extradiol gallate dioxygenase essential for GA metabolism (Nogales et al., 2005), a putative CHA aldolase encoding gene (*galC*), two additional genes of unknown function (*galB* and *galD*), a putative transport system encoded by *galT* and *galP*, and a regulatory gene (*galR*) encoding a peculiar LysR-type transcriptional regulator with two DNA-binding domains (Table 1). We have

Table 1. gal genes and their products in P. putida KT2440.

| Distance | Gene product | Function of gene product | Accession No. or PDB code |
|----------|--------------|---------------------------|---------------------------|
| to next  | (aa/kDa)    |                           | Identity (%)              |
| gene (bp)|             |                           |                           |
| Gene     | C+G         | Name/size (aa)           | Organism                  |
|          |             | Function                 |                           |

| galD PP2513 | 65.7 | 524 | FldA/356 | OMA tautomerase | Unknown | Sphingomonas sp. | CAB87567 |
|-------------|------|-----|----------|-----------------|---------|------------------|----------|
|             |      |     | ORF1/351 | Putative transmembrane protein | S. paucimobilis SYK-6 | BAB88737 |
| galC PP2514 | 67.3 | 2   | PrpF/397 | 2-Methylcitrate cis-trans-isomerase | S. oneidensis MR-1 | 2PVZ |
|             |      |     | LigK/228 | CHA aldolase | S. paucimobilis SYK-6 | BAB88738 |
|             |      |     | ProA/227 | CHA aldolase | P. straminea NGJ1 | BAB21456 |
|             |      |     | Yer010c/234 | Putative demethylmenaquinone methyltransferase | S. cerevisiae S288C | 2C5Q |
| galB PP2515 | 62.2 | -1  | -        | -             | Sphingomonas sp. | CAB87565 |
| galR PP2516 | 67.0 | 117 | FldY/483 | Transcriptional regulator | LB126 | - |
| galP PP2517 | 63.8 | 15  | PA0240/421 | Outer membrane porin | LysR-type transcriptional regulator | S. paucimobilis SYK-6 | BAB88739 |
|             |      |     | OpdK/395 | OprD-like outer membrane porin | P. aeruginosa PAO1 | NP_248931 |
|             |      |     | LigR/393 | Vanillate outer membrane porin | P. aeruginosa PAO1 | NP_253585 |
| galA PP2518 | 64.0 | 129 | DesB/418 | Gallate dioxygenase | S. paucimobilis SYK-6 | BAD80871 |
| galT* PP2519 | 65.6 | 20  | PcaK/448 | Non-functional MFS transporter | P. putida PRS2000 | Q51955 |
|             |      |     | MphT/403 | Putative 3-hydroxyphenylpropionate transporter | E. coli K-12 | P77589 |

a. Only the first 368 amino acids of GalT* have been compared with AAHS transporters. This percentage of identity is observed along the whole amino acid sequence (449 residues) of a functional GALT transporter (Accession No. FN669140) encoded by galT in P. putida KT GAL.
previously described that a GA-adapted KT2440 strain, hereafter referred as *P. putida* KTGAL (Table S1), is able to use GA as sole carbon and energy source (Nogales et al., 2005). Interestingly, the nucleotide sequence of the *gal* cluster from *P. putida* KTGAL was identical to that of the wild-type strain unable to grow in GA, with the sole exception of a deletion of the cytosine located at position 2867088 of the KT2440 genome that caused the suppression of a frameshift mutation in *galT* and allowed the expression of a GalT protein which presented the 11th and 12th transmembrane segments and the conserved motives that characterize aromatic acid/H⁺ symporter permeases (AAHS) from the MFS superfamily of transporters (Fig. S1) (Ditty and Harwood, 2002). To confirm that the molecular basis of the genetic adaptation of *P. putida* KTGAL to grow in GA was a single point mutation in the genome that restored a functional *galT* gene, GA uptake was monitored in the KT2440 and KTGAL strains. Whereas the parental KT2440 strain was unable to transport GA, the adapted strain showed a GA inducible uptake rate (Fig. 2A), indicating the existence of an efficient GA transport system in *P. putida* KTGAL. Moreover, a disruption of the *galT* gene in *P. putida* KTGALΔ*galT* (Table S1) did not allow growth of the corresponding mutant strain in GA (data not shown). As expected, the KT2440 strain expressing the functional *galT* gene from plasmid pIZGalT recovered the ability to transport GA (Fig. 2A) and use GA as sole carbon source (Fig. 1B), demonstrating that deficient transport explains the lack of growth of *P. putida* KT2440 in GA.

The *P. putida* KTGAL cells expressing the *galT* gene from the multicopy plasmid pIZGalT showed an increased uptake not only of GA but also of its dihydroxylated analogue protocatechuate (Fig. 2A). Thus, GalT is a high-affinity permease that specifically recognizes GA \[*V*~max~ 8.4 nmol min⁻¹ (mg of protein)⁻¹, *K*~m~ 4.3 μM] and, with less efficiency, protocatechuate \[*V*~max~ 3.2 nmol min⁻¹ (mg of protein)⁻¹, *K*~m~ 9.1 μM] (Fig. 2B), but it does not transport other dihydroxylated acids, like homogentisate and caffeate, monohydroxylated acids, like 4-hydroxybenzoate, and some other GA analogues such as methyl gallate, syringate and pyrogallol. GA uptake was inhibited by agents that dissipate the electrochemical gradient across cell membranes (Fig. 2C), indicating that GalT is energized by the proton motive force. An unusual feature of some AAHS permeases is that they also mediate chemotaxis to the cognate aromatic compounds (Ditty and Harwood, 2002; Parales et al., 2008). The chemotactic response of *P. putida* KT2440 towards a range of metabolites can be analysed using agarose plug assays (Lacal et al., 2010). Whereas *P. putida* KTGAL showed a GA-inducible chemotaxis to GA, the *P. putida* KT2440 strain did not, suggesting that the GalT permease also mediates this chemotactic response (Fig. 2D). A *P. putida*

KTGALΔ*galB* mutant strain unable to degrade GA (see below) still showed a chemotactic response towards this aromatic acid (Fig. 2D), indicating that chemotaxis to GA does not require metabolism. The substrate specificity and phylogenetic analysis reveal that GalT is the first GA permease described in any organism and the founding member of a new group (TC 2.A.1.15.10) closely related to that of the archetypical 4-hydroxybenzoate/protocatechuate PcaK transporter (Ditty and Harwood, 2002; Parales et al., 2008).

The *galP* gene (Fig. 1C) encodes a protein that shows significant similarity to outer membrane porins of the OprD family (Table 1) (Tamber et al., 2006). Whereas disruption of *galP* in *P. putida* KTGALΔ*galP* did not affect growth of the mutant strain in GA, the uptake of 10 μM GA [4.2 nmol min⁻¹ (mg of protein)⁻¹] decreased to half of the value obtained with the KTGAL strain [7.4 nmol min⁻¹ (mg of protein)⁻¹]. These data suggest that GalP plays an important physiological role in the uptake of low GA concentrations that may exist in the natural habitats of *P. putida*.

### The gal cluster is responsible for GA catabolism in *P. putida* and heterologous hosts

To check whether other *gal* genes were also involved in GA catabolism, *P. putida* KTGALΔ*galA*, Δ*galB*, Δ*galC* and Δ*galR* knockout mutants were constructed (Table S1), and all of them were unable to use GA as sole carbon source (data not shown), suggesting their participation in the degradation of this aromatic compound. Growth in GA could be restored when the *P. putida* knockout mutants were complemented with a broad-host-range plasmid (pGAL) harbouring a DNA cassette containing the complete *gal* cluster (Table S1). Moreover, the pGAL plasmid also conferred the ability to use GA to other bacteria unable to degrade this aromatic compound, such as *Pseudomonas* sp. MT14 (Williams and Worsley, 1976) or *Escherichia coli* W (Davis and Mingioli, 1950), both of which lack the *gal* genes (Fig. 1B). Therefore, these results demonstrate that the *gal* cluster identified in *P. putida* constitutes the first complete set of genes reported so far in any organism that is responsible for the use of GA as carbon source.

### Transcriptional organization of the *gal* cluster

An *in silico* analysis of the gene arrangement and length of the intergenic regions within the *gal* cluster suggest the existence of three putative transcriptional units, i.e. *galBCD*, *galR* and *galTAP* (Fig. S2A). The transcriptional organization of the *gal* cluster was determined by reverse transcription-PCR (RT-PCR) gene expression experiments with RNAs obtained from *P. putida* KTGAL cells

© 2010 Blackwell Publishing Ltd, *Molecular Microbiology*, **79**, 359–374
A

| $P$. putida | Carbon | Plasmid | Uptake rate |
| --- | --- | --- | --- |
| strain | source |  | (nmol min$^{-1}$ mg protein$^{-1}$) |
|  |  | GA | PCA |
| KT2440 | Citrate/GA | pIZ1016 | Nd | - |
|  |  | pIZGalT | 9.8 ± 0.7 | - |
| KTGAL | Citrate | pIZ1016 | Nd | - |
|  |  | pIZGalT | 0.9 ± 0.1 | - |
| KTGAL | Citrate/GA | pIZ1016 | 6.9 ± 0.9 | 2.7 ± 0.8 |
|  |  | pIZGalT | 33.5 ± 2.8 | 9.1 ± 0.9 |

B

![Graph](https://i.imgur.com/1234567.png)

C

![Graph](https://i.imgur.com/2345678.png)

D

![Image](https://i.imgur.com/3456789.png)

Fig. 2. GA transport and chemotaxis in $P$. putida.

A. GA uptake in $P$. putida KT2440 and $P$. putida KTGAL strains harbouring the control plasmid pIZ1016 or the galT-expressing plasmid pIZGalT. The cells were grown in 0.2% citrate-containing MC medium (Citrate) or in 0.2% citrate-containing MC medium in the presence of 2 mM GA (Citrate/GA). The uptake rate of 10 μM GA was measured with exponentially grown cells. The uptake rate of 10 μM protocatechuate (PCA) was also measured with exponentially grown KTGAL cells. Values are the mean ± standard deviation of at least three different experiments. Nd, means not detected.

B. Uptake of GA (red) and protocatechuate (blue) by GA-induced $P$. putida KTGAL cells. The curve shows a non-linear regression fit of data to the Michaelis–Menten equation. Each point is the average value from triplicate experiments. Error bars indicate standard deviation.

C. Transport of GA by $P$. putida KTGAL (pIZGalT) cells grown in GA-containing MC medium. The uptake of 50 μM GA by the cells (100 μl of a concentrated cell culture with an $A_{600}$ of 10) was measured. The addition of the cells and that of the uncoupling agent carbonylcyanide p-trifluoromethoxyphenylhydrazone (FCCP) (30 μM) that dissipates the proton motive force or the dimethyl sulphoxide (DMSO) used to dissolve the uncoupling agent is indicated by arrows. A similar inhibition of GA uptake rate was obtained when other uncoupling agents, such as m-chlorophenylhydrazone (CCCP) (20 μM) and 2,4-dinitrophenol (1 mM), were used.

D. Chemotactic responses of $P$. putida KT2440, KTGAL and KTGALdgalB strains grown in the presence (+GA) or absence (-GA) of GA to GA-containing agarose plugs. Phase-contrast microscope images (×100) were cropped. Accumulation of cells around the GA-containing plug is shown with a yellow arrow.

grown in GA and primer pairs that were complementary to neighbouring genes for amplification of the four intergenic regions in the putative galBCD and galTAP operons (Fig. S2A). Since RT-PCR fragments of the expected sizes were obtained for genes galB-galC, galC-galD, galT-galA and galA-galP (Fig. S2B), these results suggest that the galBCD and galTAP genes are co-transcribed. To confirm the organization of the galBCD and galTAP genes

as two transcriptional units, we checked whether there are polar effects derived from the insertional disruption of the galB and galT genes in $P$. putida KTGALdgalB and $P$. putida KTGALdgalT mutant strains respectively. RT-PCR experiments revealed that the insertional disruption of the galB and galT genes cause polar effects avoiding the expression of the galD and galP genes, respectively (Fig. S2C), thus confirming the existence of the galBCD and

© 2010 Blackwell Publishing Ltd, Molecular Microbiology, 79, 359–374

Fig. 3. Identification of the GA pathway intermediates. ${}^{1}$H NMR signals of the products of the GalA- (upper panel), GalD- (middle panel) and GalB- (lower panel) mediated reactions are shown. Since the reactions were not developed to completion, ${}^{1}$H NMR signals of the corresponding substrates can be also observed. Protons detected by ${}^{1}$H NMR analysis are indicated and numbers correspond to that shown for peaks. GA, which shows a resonance signal of protons H-1 at $\delta$ 6.94 ppm, was used as substrate for the GalA dioxygenase. The conversion of GA was accomplished after incubation in 50 mM sodium phosphate buffer, pH 7.0 (prepared in 10% D$_2$O), of 200 $\mu$M GA with 2 $\mu$g of purified GalA for 1 min at 20°C. The ${}^{1}$H NMR spectrum of the reaction product shows signals at $\delta$ 6.63 ppm and 3.95 ppm that correspond to the olefinic proton H-II and the aliphatic protons H-III, respectively, of OMAketo (upper panel). GalD was then added [5 $\mu$g of $E$. coli BL21 (DE3) (pETGalDc) extract] to the previous reaction and the mixture was further incubated. The resulting ${}^{1}$H NMR spectrum indicated the conservation of the signal at $\delta$ 6.63 ppm and the appearance of a new signal at $\delta$ 6.73 ppm that correspond to protons H-V and H-IV, respectively, of the $(E)$-OMAenol (middle panel). Since no signal around $\delta$ 7.38 ppm, which would correspond to proton H-V of $(Z)$-OMAenol, was observed, this OMA isomer can be ruled out as product of the GalD tautomerase. Finally, the addition of 5 $\mu$g of purified GalB to the previous reaction showed the appearance of a new AB system within the aliphatic region ($\delta$A 2.49 ppm and $\delta$B 2.41 ppm) that corresponds to protons H-VI of the CH2 moiety of CHA (lower panel).

galTAP operons within the gal cluster. Moreover, since no RT-PCR fragments were obtained when the parental and mutant strains were grown in citrate in the absence of GA (Fig. S2B and C), the galBCD and galTAP operons were shown to be GA inducible.

### Characterization of the GA ring-cleavage product

GA ring cleavage is the first enzymatic step in the proposed GA degradation pathway (Tack et al., 1972a) (Fig. 1A), and the galA and desB genes encoding a specific extradiol gallate dioxygenase have been characterized in P. putida KT2440 (Tack et al., 1972a; Nogales et al., 2005) and S. paucimobilis SYK-6 (Kasai et al., 2005) respectively. The product of the gallate dioxygenase-mediated reaction has been reported to be a mixture of the OMAketo and the two OMAenol isomers (Kersten et al., 1982; Hara et al., 2000; Kasai et al., 2005; Nogales et al., 2005) (Fig. 1A), although OMAenol was suggested to be the product of GA ring cleavage by purified protocatechuate 4,5-dioxygenase (Tack et al.,

1972a). To determine unequivocally the chemical structure of the product generated by purified GalA acting on GA, NMR spectroscopy was performed. Since a single opening product whose ${}^{1}$H MNR spectra was consistent with the keto form rather than with the enol forms of OMA was detected (Fig. 3), we can conclude that OMAketo constitutes the real GA ring-cleavage product and therefore the controversy about the nature of the product of the GalA-mediated reaction has been finally clarified (Fig. 1A).

#### GalD catalyses a new biochemical step in the GA degradation pathway

The GA ring-cleavage product was proposed to be the substrate of the hydratase that catalyses the second enzymatic step in GA degradation (Tack et al., 1972a) (Fig. 1A). However, we were unable to detect any hydratase activity in P. putida KTGAL extracts when using as substrate a fresh solution of the GA ring-cleavage product. This result suggested the existence of an addi-

A

| 0.50 |  |
| --- | --- |
| 0.49 |  |
| 0.48 |  |
| 0.47 |  |
| 0.46 |  |
| 0.45 |  |
| 0.44 |  |
| 0.43 |  |
| 0.42 |  |
| 0.41 |  |
| 0.40 |  |

KTGAL  
KTGALdgalD  

Time (min)

B

Absorbance

210nm  
205nm  
240nm  
275nm  

λ (nm)

Fig. 4. The GalD protein is an OMA tautomerase.

A. Time-course of the conversion of the OMAketo compound. The complete conversion of GA to OMAketo was accomplished in two independent reactions after incubation in 100 mM Tris-HCl, pH 7.5, of 200 μM GA with 2 μg of purified GalA for 2 min at 30°C (time 0). The further conversion of the OMAketo to a new compound was monitored by measuring changes in \( A_{265} \). The spontaneous conversion of OMAketo was recorded until time 4 min. At time 4 min (black arrow), 5 μg of crude extract from *P. putida* KTGAL cells (black curve) or *P. putida* KTGALdgalD cells (grey curve) grown in 5 mM GA was added to the assay, and changes in \( A_{265} \) were recorded until time 10 min.

B. Spectral change of OMAketo in the GalD-catalysed isomerization. The 200 μM OMAketo solution was obtained as indicated above by incubation of 200 μM GA with GalA for 2 min at 30°C, and split into two halves in the sample and reference cuvettes to which 0.1 μg of soluble extract of *E. coli* BL21(DE3) (pETGalDc) and *E. coli* BL21(DE3) [pET-29a(+)], respectively, were added. Spectral changes were recorded at 2 min intervals for 20 min. Although the results of one experiment are shown, the spectral changes were reproducible in three different experiments. Grey arrows indicate the isosbestic points. Black arrows show the λ to which maximum absorbance changes between the substrate (OMAketo) and the product (OMAenol) of GalD were detected.

Disruption of the *galD* gene does not prevent growth of the *P. putida* KTGALdgalD strain in 5 mM GA, but the mutant strain grew poorly (\( A_{600} < 0.1 \)) at GA concentrations lower than 2 mM. This observation revealed the importance of a functional *galD* gene for an efficient GA degradation, and suggests that the spontaneous isomerization of OMA and/or the unspecific activity of additional isomerases might replace the role of GalD when the mutant cells use high GA concentrations. It is worth noting that *galD* orthologues, e.g. *orf1* and *fldA* genes found in the *lig* and *fld* clusters for protocatechuate mineralization via OMA in *Sphingomonas* strains (Table 1), were proposed to be involved in membrane-associated transport (Wattiau et al., 2001; Hara et al., 2003). In light of the data presented here, Orf1 and FldA can be tentatively reassigned as OMA tautomerases involved also in a previously unnoticed biochemical step of the well-studied protocatechuate 4,5-meta-cleavage pathway.

GalD shows moderate sequence identity with two isomerases that share a common structural fold (Table 1) (Alhapel et al., 2006; Garvey et al., 2007). Homology modelling of the three-dimensional structure of GalD reveals that the protein is composed of two structural domains that exhibit the same topology, i.e. a central α-helix surrounded by mainly antiparallel β-strands to form a pronounced β-barrel and an additional α-helix packed against the outside of β-barrel (Fig. S4). The cleft

© 2010 Blackwell Publishing Ltd, *Molecular Microbiology*, **79**, 359–374
between the two domains contains the proposed active site (Fig. S4), as deduced by its similarity with the architecture of the active sites of PrpF and Mii isomerases (Garvey *et al.*, 2007). Although the predicted overall fold of GalD with two domains that might evolve from a common ancestor via a duplication event has been also observed in other members of the diaminopimelate epimerase family (Pillai *et al.*, 2006; Velarde *et al.*, 2009), this fold has not been observed so far in other isomerases, e.g. XylH or HpcD/HpaF, involved in the catabolism of aromatic compounds (Roper *et al.*, 1994; Subramanya *et al.*, 1996; Taylor *et al.*, 1998). Therefore, GalD constitutes the prototype of a new subfamily of isomerases that are involved in the metabolism of tricarboxylic acids, e.g. OMA, generated during the degradation of some aromatic compounds.

GalB is the founding member of a new family of zinc-containing hydratases

Cell-free extracts of GA-grown *P*. putida KTGAL exhibited OMA hydratase activity on OMAenol [1.0 μmol min⁻¹ (mg of protein)⁻¹], but this activity was not detected when OMAketo was used as substrate, suggesting that hydration of OMAenol was the third step in the GA degradation pathway (Fig. 1A). Although amino acid sequence analysis of the *gal* gene products did not reveal the presence of any putative hydratase (Table 1), the *P*. putida KTGALΔ*galB* mutant strain lacked OMA hydratase activity (data not shown), pointing out that the *galB* gene might encode such activity. This was confirmed when a significant OMA hydratase activity was observed in extracts of *E*. coli BL21 (DE3) (pETGalB) cells (Table S1) overproducing the GalB protein (Fig. 5A). Since EDTA inhibited the GalB hydratase activity of cell extracts in a concentration-dependent manner (Fig. 6A), the presence of divalent cations appears to be essential for catalysis. EDTA-inactivated GalB could not be reactivated by the addition of metals, pointing that loss of the metal causes an irreversible inhibition of the protein. Cell extracts containing GalB showed a significant loss of OMA hydratase activity when diluted in the absence of metals (Fig. 6B), but the addition of Zn²⁺, and to a minor extent Co²⁺, reduced the enzyme inactivation (Fig. 6C). These findings suggested that GalB is a Zn²⁺-dependent enzyme and, accordingly, a functional GalB hydratase was purified by adding Zn²⁺ to the purification buffer. The GalB protein was purified from *E*. coli BL21 (DE3) (pETGalB) cell extracts and analysed by SDS-PAGE, showing a band whose apparent molecular mass (27 kDa) corresponded with that predicted for the *galB* gene product (27.4 kDa) (Fig. 5A and B). Analyses by inductively coupled plasma emission spectroscopy (ICP) of the metal content of purified GalB protein revealed 1 mol of this metal per mol of protein. Purified GalB was active in the pH range of 6.0–9.0, with maximum activity at pH 7.0. The optimal temperature was around 30°C and the *K*~m~ and *V*~max~ values for OMAenol were 85 μM and 76 μmol min⁻¹ (mg of protein)⁻¹ respectively. Amino acid modification studies with specific inhibitors of His, Cys, Ser and Asp/Glu residues revealed that only diethyl pyrocarbonate significantly reduced the hydratase activity to 15% of that of the untreated enzyme, suggesting the existence of His residue(s) in the active site of GalB. The GalB protein has a corrected sedimentation coefficient of 7.0 ± 0.1 S and a molar mass of 151.3 ± 12 kDa, which are compatible with the main OMA hydratase holoenzyme species being a globular homohexamer (Fig. 5C). ^1^H NMR analysis of the reaction product of OMAenol with the GalB hydratase revealed new aliphatic signals at δ values of 2.41 and 2.49 ppm (coupling constant 14.8 Hz), consistent with the outcome of two AB systems for the CH₂ groups present in CHA (Fig. 3).

Although other OMA hydratases involved in the protocatechuate 4,5-cleavage degradation pathway have been previously characterized, e.g. LigJ (Hara *et al.*, 2000) and ProH (Li *et al.*, 2007), GalB does not show amino acid sequence identity with these enzymes. Moreover, these LigJ-type enzymes are homodimers whose activity is inhibited by Cys-specific reagents but not by chelators (Hara *et al.*, 2000; Li *et al.*, 2007). Nevertheless, the three-dimensional structure of the LigJ-type OMA hydratase from *Rhodopseudomonas palustris* (PDB 2GWG) revealed the existence of one Zn ion per subunit, with a His residue located near the Zn ion that could be involved in enzyme catalysis and with a Cys residue located on a flexible loop that lies just above the Zn ion so as to protect it from chelators and accounting for the enzyme inactivation by Cys-specific reagents (Li *et al.*, 2007). Therefore, GalB constitutes the first member of a new family of OMA hydratases that have a different evolutionary origin and are predicted to use a different catalytic mechanism for the hydration of OMA than the LigJ-type enzymes. Interestingly, when the *galB* gene was transferred to *S*. paucimobilis DLJ, a derivative of *S*. paucimobilis SYK-6 unable to grow in syringic or vanillic acids since it lacks the *ligJ* gene encoding the corresponding OMA hydratase (Hara *et al.*, 2000), the recombinant strain acquired the ability to grow in both aromatic compounds as sole carbon source (Fig. S5A and B). These experiments demonstrate that GalB is able to replace the function of LigJ, thus confirming its physiological role as an OMA hydratase and suggesting that the real substrate of the equivalent hydratases acting in the protocatechuate meta-cleavage pathway is also the OMAenol rather than the previously suggested OMAketo form (Hara *et al.*, 2000).
A

| Purification step       | Total protein (mg) | Total activity ($\mu$mol·min${}^{-1}$) | Specific activity ($\mu$mol·min${}^{-1}$·mg protein${}^{-1}$) | Yield (%) | Purification (-fold) |
|-------------------------|--------------------|----------------------------------------|----------------------------------------------------------|-----------|----------------------|
| Crude extract           | 230.0              | 2392.0                                 | 10.4                                                   | 100       | 1.0                 |
| Ultracentrifugation     | 78.0               | 1747.2                                 | 22.4                                                   | 73.5      | 2.2                 |
| DEAE cellulose          | 30.8               | 1142.7                                 | 37.1                                                   | 48.4      | 3.6                 |
| Phenyl-Sepharose        | 11.7               | 504.3                                  | 43.1                                                   | 21.4      | 4.1                 |

B

C

Fig. 5. Overproduction, purification and oligomeric conformation of the GalB protein.

A. Purification of the GalB protein from *E. coli* BL21(DE3) (pETGalB) cells. The cultivation of cells, the preparation of the crude extract and the purification of the GalB protein were carried out as described in Experimental procedures.

B. SDS-PAGE analysis of the overproduction and purification of the GalB protein. Proteins were separated in a SDS-12.5% polyacrylamide gel and stained with Coomassie Brilliant Blue. Lane 1, molecular mass markers (in kDa); lane 2, soluble control extract from *E. coli* BL21(DE3) [pET-29a(+)]; lane 3, soluble crude extract from *E. coli* BL21(DE3) (pETGalB); lanes 4, 5 and 6, protein fraction after the ultracentrifugation, DEAE-cellulose and phenyl-Sepharose chromatography purification steps respectively. The purified GalB protein is indicated with an arrow. Five and three micrograms of total protein were loaded in lanes 2–4 and 5–6 respectively.

C. Determination of the oligomeric state of the GalB protein in solution by analytical ultracentrifugation experiments. The equilibrium sedimentation analysis of purified GalB is shown. Empty circles represent the data obtained, the solid line shows the theoretical equilibrium gradient that correlates with a hexamer.

Although the amino acid sequence of GalB does not show significant similarity in its entire length to any protein of known function (Table 1), a detailed analysis restricted to its N-terminal region revealed the presence of a sequence that resembles the conserved motif (V X P/A H P D D) found in the active site of deacetylases from the Pig-L family of mononuclear zinc-hydrolases (Fig. S6). The known three-dimensional structure of some of these mononuclear zinc-hydrolases revealed that the His and the second Asp residue of the conserved motif, together with a second His residue spaced approximately 100 residues from the former, constitute the Zn${}^{2+}$ binding site, acting the first Asp probably as general acid-base catalyst in the reaction (Maynes *et al.*, 2003; McCarthy *et al.*, 2004; Hernick and Fierke, 2005). Using the known three-dimensional structure of the MshB deacetylase (Maynes *et al.*, 2003), a homology modelling for the active site of GalB was accomplished. The modelled active site of GalB fits the geometry of the Zn${}^{2+}$ binding site of MshB, being H14, D17 and H127 the proposed metal co-ordination residues. However, the catalytic Asp residue conserved in all deacetylases of the Pig-L family (D15 in MshB) is replaced by an Ala residue (A16 in GalB) in GalB-like proteins, and a conserved motif of GalB-like proteins (V/I S A H S/A A D) different from that of the classical Pig-L deacetylases can be proposed (Fig. S6). Whether the

© 2010 Blackwell Publishing Ltd, Molecular Microbiology, 79, 359–374

Fig. 6. The GalB protein is a zinc-dependent OMAenol hydratase.

A. Concentration-dependent inhibition of GalB by EDTA. The GalB-containing extracts were pre-incubated for 3 h at 4°C with various concentrations (0, 5, 10, 50 and 100 mM) of EDTA. Then, the OMA hydratase activity of the mixtures was determined.

B. Dilution effect on the GalB activity. The GalB-containing extract was diluted in the purification buffer lacking Zn²⁺ to a protein concentration of 0.8, 0.4, 0.08, 0.04, 0.016 and 0.008 mg ml⁻¹, and these diluted samples were pre-incubated for 2 h at 4°C before the OMA hydratase activity was determined. In all enzyme assays the final concentration of the GalB-containing extract was 20 μg protein ml⁻¹.

C. Stimulation of GalB activity by metals. The GalB-containing extract was diluted 100-fold into the purification buffer in the absence (–) or presence of 2 mM Zn²⁺ (Zn), Ni²⁺ (Ni), Co²⁺ (Co), Cu²⁺ (Cu), Mg²⁺ (Mg) or Mn²⁺ (Mn). After incubation for 2 h at 4°C, the OMA hydratase activity was determined. In all enzyme assays the final concentration of the GalB-containing extract was 20 μg protein ml⁻¹.

Results of one experiment are shown, and values were reproducible in three separate experiments with standard deviations of < 10%.

substitution of Asp by Ala in the active site of GalB might reflect that this protein is a hydratase rather than a hydrolase is a tempting hypothesis that requires further confirmation. All these data support that GalB is the prototype of a new family of Zn²⁺-containing hydratases that are evolutionary related to the Pig-L family of zinc-hydrolases rather than to other previously characterized hydratases involved in the catabolism of aromatic compounds (Pollard and Bugg, 1998; Li et al., 2007).

The galC gene encodes a CHA aldolase

Although the enzyme that catalyses the aldolic cleavage of CHA to produce pyruvic plus oxaloacetic acids (Fig. 1A) had been biochemically characterized many years ago (Tack et al., 1972b; Sparnins and Dagley, 1975), the cognate gene has remained unknown in *P. putida*. Here we show that the *galC* gene overexpressed in *E. coli* BL21 (DE3) cells harbouring plasmid pETGalC (Table S1) encodes a 25.1 kDa protein (Fig. S7A) that exhibits a significant CHA aldolase activity [24.5 μmol min⁻¹ (mg of protein)⁻¹] in cell extracts in the presence of Mg²⁺. Moreover, whereas extracts of *P. putida* KTGAL strain showed CHA aldolase activity [2.1 μmol min⁻¹ (mg of protein)⁻¹], this activity could not be detected in *P. putida* KTGALΔ*galC*, and this strain was unable to grow in GA. These data indicate that the *galC* gene is essential for GA degradation encoding a type II (metal ion-dependent) CHA aldolase whose function cannot be replaced by that of other aldolases present in *P. putida*. GalC shows high sequence similarity to LigK and ProA, CHA aldolases from *S. paucimobilis* (Hara et al., 2003) and *Pseudomonas straminea* (Maruyama et al., 2001) respectively (Table 1).

Interestingly, GalC shows a significant amino acid sequence identity with proteins of the RraA family, e.g. the Yer010c protein from *Saccharomyces cerevisiae* (Table 1), that were originally misannotated as S-adenosylmethionine-dependent methyltransferases but whose three-dimensional structure resembles the phosphohistidine domains of phosphotransfer systems (Leulliot et al., 2005). Based on the known three-dimensional structure of the Yer010c protein (Leulliot et al., 2005), the predicted structure of the GalC monomer fits a α–β–β–α sandwich (Fig. S7B), being the last α5 and α6 helices likely involved in oligomerization of the protein and accounting for the hexameric quaternary conformation reported in native CHA aldolases from different bacteria (Tack et al., 1972b; Maruyama, 1990; Hara et al., 2003). A multiple sequence alignment among GalC, other CHA aldolases and proteins of the RraA family reveals highly conserved residues, including R123 and D124 in GalC (Fig. S7C), that have been postulated to be involved in a phosphotransfer reaction (Leulliot et al., 2005). Residues C95 and C159 in GalC are also conserved in other CHA aldolases (Fig. S7C), and they could be involved in catalysis and account for the previously described N-ethylmaleimide-dependent inhibition of the aldolase activity (Hara et al., 2003). The enzyme structure proposed here for GalC provides a new framework to advance in the study of the molecular mechanism of the CHA aldolases, and will help to elucidate the phosphate-dependent activation observed with some CHA aldolases
(Maruyama, 1990; Hara *et al.*, 2003), which might constitute an additional level of post-translational control of the cognate degradation pathway.

### Unveiling gal clusters in bacterial genomes

In silico searches predicted the existence of *gal* clusters in the genomes of several Proteobacteria (Fig. 1C), most of which are able to colonize the plant rhizosphere. The overall organization of *gal* genes present in most γ-Proteobacteria is very similar, i.e. a *galA-galT(galP)* module, encoding GA uptake and ring cleavage, separated of the *galBCD* module, encoding the OMA degradation enzymes, by a regulatory *galR* gene. The similar organization and high gene identity may reflect a common evolutionary origin for the *gal* clusters in γ-Proteobacteria. Within the *Pseudomonas* genus, the *gal* genes are present only in *P. putida* strains, e.g. KT2440, F1 or W619, where they show > 95% sequence identity. This observation is in agreement with the GA-degradation abilities detected in this work with different *P. putida* strains, e.g. GPo1, U, F1 (data not shown), and with those previously reported in *P. putida* X.1 (Beveridge and Hugo, 1964) and in closely related *Pseudomonas* species (Chowdhury *et al.*, 2004). The presence of a transposase-encoding gene (PP_2522) at the 5′ end of the *galT* gene, as well as the observation that the average GC content of the *gal* cluster (65.1%, Table 1) is higher than that of the whole genome (61.5%), and the fact that this cluster is located at the right end of genomic island 46 of strain KT2440 (Weinel *et al.*, 2002), suggest that the *gal* genes constitute an evolutionary acquisition, via horizontal gene transfer, by *P. putida* to increase its metabolic proficiency as a saprophytic omnivore. In β- and α-Proteobacteria, some of which as *Burkholderia* sp. 383 were confirmed here as efficient GA degraders, the gene arrangement of the *gal* clusters shows more heterogeneity, and reveals that a *ligJ*-like gene replaces the *galB* gene in the genome of these α in β-Proteobacteria (Fig. 1C). This replacement is in agreement with our previous finding that although possessing different structures, both types of OMA hydratases and thus, their encoding genes, are interchangeable (Fig. S5A and B), and suggests that the *ligJ*-like genes might have been recruited from protocatechuate 4,5-meta-cleavage clusters.

Interestingly, all predicted *gal* clusters harbour the *galA-galT* module, which reinforces the key role of an active GA uptake for the consumption of this aromatic compound. The proof of concept of this assumption was to observe that whereas a derivative of *S. paucimobilis* SYK-6 was unable to use GA as sole carbon source, this strain harbouring the *galT* gene from *P. putida* KTGAL acquired the ability to grow in GA-containing minimal medium (Fig. S5C). Since we have found in several bacterial genomes

gene clusters containing *galRBC* orthologues that could be involved in the metabolism of tricarboxylic acids analogous to OMA, it is tempting to speculate that the association of the latter to the *galA-galT* module might have evolved into a functional *gal* cluster. The *galD* and *galP* are the only *gal* genes that were not strictly essential for the catabolism of GA in *P. putida* (see above), and they are lacking in some *gal* clusters (Fig. 1C), which suggests that their function in GA degradation can be replaced by the unspecific activity of other genes present in the corresponding bacterial genomes.

#### Conclusions

The characterization of the *gal* cluster reveals the existence of a new and paradigmatic aromatic catabolic pathway that surprisingly remained unknown in one of the best-studied model bacteria, *P. putida* KT2440. The *gal* genes from *P. putida* became useful to identify homologous gene clusters annotated as of unknown function in bacteria whose ability to degrade GA was unexplored so far, and thus expanding outside the *Pseudomonas* genus our current view on aerobic GA degraders. The GalT protein mediates specific GA uptake and chemotaxis in *P. putida* KTGAL, and highlights the critical role of transporters in the adaptation of bacteria to aromatic carbon sources, such as GA, that can easily oxidize and become unavailable for the cells. Remarkably, proteins as the new OMA tautomerase (GalD) and the OMA hydratase (GalB) constitute the founding members of new families of enzymes, and they allowed to revisit the proposed GA biochemical pathway, as well as to assign new functions to some of the gene products involved in the well-studied protocatechuate 4,5-meta-cleavage pathway. The *gal* cluster is also a useful biotechnological tool for expanding the metabolic abilities of bacteria towards GA degradation, and it has been used also for engineering the first GA cellular biosensor able to detect GA concentrations as low as 0.5–1 μM (Fig. S8).

#### Experimental procedures

##### Bacterial strains, plasmids and growth conditions

The bacterial strains, plasmids and oligonucleotides used in this study are listed in Table S1. Bacteria were grown in LB medium (Sambrook and Russell, 2001) or MC minimal medium (Nogales *et al.*, 2005) at 37°C (*E. coli*) or 30°C (*Pseudomonas*, *Burkholderia*, *Sphingomonas*). When used as carbon sources, 0.2% citrate, and 5 mM syringate or vanillate, were added to the MC medium. Antibiotics were used at the following concentrations: ampicillin (100 μg ml⁻¹), kanamycin (50 μg ml⁻¹), gentamicin (50 μg ml⁻¹), streptomycin (50 μg ml⁻¹).

*Pseudomonas putida* KT2440 was adapted to use 5 mM GA as sole carbon source by using MC minimal medium (pH

© 2010 Blackwell Publishing Ltd, *Molecular Microbiology*, **79**, 359–3746.5) containing 2 mM L-cysteine as reducing agent. The slightly acidic pH and low amount of Fe²⁺ in the MC medium, as well as the presence of L-cysteine avoid the rapid oxidation of GA and therefore the formation of a black polyphenol that cannot be used by the cells. After a 24 h incubation of *P. putida* KT2440 cells in GA-containing MC medium, poor growth and a black coloration of the medium, due to GA oxidation, was observed. The culture was then centrifuged, and the cells were inoculated again into fresh medium. After an additional 24 h incubation, a reasonable cell growth and a slightly brown coloration of the medium was observed. This culture was then centrifuged, and the cells inoculated again into fresh medium. After an additional 12 h incubation, a significant cell growth (*A*~600~ of about 0.9) and the lack of brown coloration in the culture medium was observed. These GA-adapted cells were named *P. putida* KTGAL.

### Molecular biology techniques

Standard molecular biology techniques were performed as previously described (Sambrook and Russell, 2001). PCR products were purified with the High Pure plasmid isolation kit (Roche Applied Science). DNA fragments were purified with Gene-Clean Turbo (Q-BIOgene). Genomic DNA from *P. putida* KT2440 was isolated with GenomicPrep Cells and Tissue DNA Isolation kit (Amersham Biosciences). All cloned inserts and DNA fragments were confirmed by DNA sequencing with fluorescently labelled dideoxynucleotide terminators (Sanger *et al.*, 1977) and AmpliTaq FS DNA polymerase (Applied Biosystems) in an ABI Prism 377 automated DNA sequencer (Applied Biosystems). Transformation of *E. coli* cells was carried out by using the RbCl method (Sambrook and Russell, 2001) or by electroporation (Gene Pulser, Bio-Rad). Oligonucleotides were purchased from Sigma. The proteins were analysed by SDS-PAGE and Coomassie-stained according to standard protocols (Sambrook and Russell, 2001). The protein concentration was determined by the method of Bradford (1976) using BSA as the standard.

### Construction of *P. putida* KTGALdgal mutant strains

To construct the *P. putida* KTGALdgalB, KTGALdgalC, KTGALdgalD, KTGALdgalP, KTGALdgalR and KTGALdgalT mutant strains, an internal fragment (300–700 bp) of the target *gal* gene was PCR-amplified and cloned into the polylinker of pK18mob (Table S1), a mobilizable plasmid that does not replicate in *Pseudomonas*. To transfer the corresponding pK18gal plasmids into *P. putida* KTGAL, triparental filter matings were performed as previously described (de Lorenzo and Timmis, 1994) using *E. coli* DH10B (pK18gal) as donor strain, *E. coli* HB101 (pRK600) as helper strain and *P. putida* KTGAL as recipient strain. *P. putida* KTGAL exconjugants harbouring the *gal*-disrupted gene were isolated on MC minimal medium plates containing citrate (which selected for the *Pseudomonas* recipient cells) and kanamycin (which selected for the insertion of the suicide vector) after incubation at 30°C for 16 h (Nogales *et al.*, 2005). All mutant strains were analysed by PCR to confirm the disruption of the target gene.

### Recombinant plasmids constructions

To construct the pGAL plasmid, a 9.6 kb EcoRI fragment containing the entire *gal* cluster from PPZIA58, a Lawrirst 7-derived cosmid that contains a 41 kb DNA fragment from position 2.854.165 to 2.895.166 of the *P. putida* KT2440 chromosome (Table S1), was cloned into the EcoRI-digested broad-host-range plasmid pBBR1MCS-5 (Table S1), generating the pGALKT plasmid (14.4 kb). The AclI/XhoI fragment containing part of *galA* and the entire *galT* gene from pGALKT plasmid was replaced by the homologous AclI/XhoI fragment PCR-amplified from *P. putida* KTGAL, generating the pGAL plasmid (13.6 kb) that expresses the complete *gal* cluster from *P. putida* KTGAL (Table S1). The pIZGalT plasmid expressing the *galT* gene under the control of the LacI⁰/Ptac promoter was constructed by cloning first the 2.5 kb Ndel/blunt PCR-amplified *galT* gene from the KTGAL strain into the Ndel/EcoRV double-digested pET29-a(+) plasmid, producing plasmid pETGalT (Table S1). An Xbal/Sacl fragment containing the *galT* gene was then subcloned from pETGalT into pIZ1016 giving rise to plasmid pIZGalT (Table S1). The pETGalB plasmid expressing the *galB* gene under the *P*~T7~ promoter was constructed by cloning a 0.7 kb Ndel/Sacl PCR-amplified *galB* gene into pET29-a(+). The pETGalDc and pETGalC plasmids expressing the *galD* and *galC* genes under control of the *P*~T7~ promoter were constructed by cloning the 1.2 kb Ndel/Sall and the 0.7 kb Ndel/Sacl PCR-amplified *galD* and *galC* genes into pET29-a(+) respectively (Table S1). Plasmid pIZGalB expresses the *galB* gene under the control of the LacI⁰/Ptac promoter, and it was generated by subcloning the 0.7 kb Xbal/Sacl *galB* gene from plasmid pETGalB into the Xbal/Sacl double-digested pIZ1016 plasmid (Table S1). The pIZGalBT plasmid, which expresses both the *galB* and *galT* genes under the LacI⁰/Ptac promoter, was generated by cloning a 0.7 kb EcoRI/blunt PCR-amplified *galB* gene into the EcoRI/Smal double-digested pIZGalT plasmid (Table S1).

### RT-PCR experiments

For RT-PCR experiments, cultures of *P. putida* grown in MC medium with GA or citrate as carbon source were collected at an *A*~600~ of 0.5. Total RNAs were obtained with the RNeasy kit (Qiagen). Any contamination by DNA was eliminated by the use of a DNase treatment and removal kit (Ambion). One microgram of purified total RNA was used to prepare cDNA by the use of 200 U of SuperScript reverse transcriptase (Invitrogen), 0.5 mM dNTPs and 0.5 μM of the cognate oligonucleotide. The cDNA obtained was amplified with 1 U of AmpliTaq DNA polymerase (Biotoools) and 0.5 μM of the corresponding primer pairs (Table S1). Control reactions in which reverse transcriptase was omitted from the reaction mixture ensured that DNA products resulted from the amplification of cDNA rather than from DNA contamination.

### Phenolic compounds transport assays

Measurement of transport of GA and other phenolic compounds was accomplished by using a laccase-based electrochemical biosensor (Gamella *et al.*, 2006). Briefly, by cou-
pling the decrease of an electrochemical signal to a proportional decrease in the content of a phenolic compound in solution, a real-time estimation of cell-mediated transport of such particular compound can be monitored. The transport assay (5 ml) contains different initial concentrations of the target aromatic compound in 100 mM sodium phosphate buffer, pH 6.0. *P. putida* cells grown until mid-exponential phase were harvested, washed twice and added to the transport mixture. The decrease of the electrochemical signal, which corresponds with a proportional decrease in the aromatic compound concentration, was monitored for 5 min at 25°C. To check the substrate range of the GalT transporter, the transport assays were carried out in the presence of 50 μM of different phenolic compounds, i.e. GA, methylgal- late, syringate, protocatechuate, 4-hydroxybenzoate, pyro- gallol, homogentisate, caffeate and catechol.

### GA chemotaxis assays

Chemotaxis to GA was measured by using the agarose plug assay. Agarose plug assays were carried out as previously described (*Parales et al.*, 2000) with slight modifications. Plugs contained 2% low-melting agarose in MC medium (*Nogales et al.*, 2005) harbouring 1 mM GA. A drop (10 μl) of the melted agarose mixture was placed on a microscope slide, and a coverslip supported by two plastic strips was then placed on top to form a chamber. *P. putida* cells were grown in 0.2% citrate-containing MC medium in the absence or presence of 2 mM GA until the cultures reached mid- exponential phase, and then flooded into the chamber to surround the agarose plug. The chemotactic response was monitored under the phase-contrast microscope 5 min after addition of cells. Control plugs contained no attractant, and no response was seen.

### P. putida cell extract preparations

*Pseudomonas putida* cells were grown until the cultures (0.2 l) reached an absorbance at 600 nm of 0.5. Cells were harvested at 4°C and resuspended in 10 ml of extraction buffer: (i) 50 mM HEPES, pH 7.0, 10% glycerol (w/v), 0.2 mM ZnSO₄ and 1 mM DTT for OMA hydratase, and (ii) 50 mM sodium phosphate, pH 7.0 for OMA isomerase and CHA aldolase. Cells were then disrupted by two consecutive passages through a French press (Aminco Corp.) operated at a pressure of 20 000 p.s.i. The cell lysates (crude extracts) were centrifuged at 13 000 g for 30 min at 4°C. The clear supernatant fluid was carefully decanted and used as the soluble fraction of the crude extract.

### Overproduction of GalB, GalC and GalD proteins

*Escherichia coli* BL21(DE3) cells harbouring plasmids pET-GalDc, pETGalB or pETGalC were grown in LB medium at 37°C until the cultures (0.2 l) reached an absorbance at 600 nm of 0.6. Overexpression of the cloned genes was then induced for 3 h by the addition of 0.1 mM isopropyl-1-thio-β-D-galactopyranoside. Cells were harvested at 4°C and resuspended in 10 ml of extraction buffer (see above). Cells were then disrupted by two consecutive passages through a

French press (Aminco Corp.) operated at a pressure of 20 000 p.s.i. The cell lysates (crude extracts) were centrifuged at 13 000 g for 30 min at 4°C. The clear supernatant fluid was carefully decanted and used as the soluble fraction of the crude extract.

### Purification of the GalA dioxygenase and the GalB hydratase

The overproduction and purification of GalA was as previously described (*Nogales et al.*, 2005). The purification of the OMA hydratase (GalB) from *E. coli* BL21(DE3) (pETGalB) was performed at 4°C by a three-step procedure in the presence of 0.2 mM ZnSO₄. The *E. coli* BL21(DE3) (pETGalB) soluble extract (20 ml, 230 mg of protein) was ultracentrifuged at 80 000 g for 60 min (step 1, ultracentrifugation fraction). NaCl was added to the supernatant to a final concentration of 0.05 M and loaded onto a DEAE-cellulose column (Sigma-Aldrich) previously equilibrated with extraction buffer (see above) containing 0.05 M NaCl. Proteins were eluted from the column with a linear gradient (0.05 to 0.5 M) of NaCl in 0.1 l of extraction buffer. The OMA hydratase GalB was eluted at approximately 0.3 M NaCl (step 2, DEAE cellulose fraction). The fractions containing OMA hydratase activity were pooled, NaCl was added to a final concentration of 0.5 M, and the solution was loaded onto a phenyl-Sepharose CL-4B column (Amersham Biosciences) previously equilibrated with extraction buffer containing 0.5 M NaCl. Proteins were eluted from the column with a linear gradient (0.5 to 0 M) of NaCl. Finally, the fractions containing OMA hydratase activity, which eluted around 0.1 M NaCl, were pooled, desalted in extraction buffer and stored at –20°C.

### Analytical ultracentrifugation analyses

Sedimentation velocity experiments of purified OMA hydratase (GalB) solutions in extraction buffer [50 mM HEPES, pH 7.0, 10% (w/v) glycerol, 0.2 mM ZnSO₄ and 1 mM DTT] were carried out at 40 000 r.p.m. and 20°C in an XL-A analytical ultracentrifuge (Beckman-Coulter) equipped with UV-visible absorbance optics, using an An60Ti rotor and 12 mm double-sector centrepieces of Eponcharcoal. Absorbance scans were measured at 280 nm. Sedimentation coefficient distributions, *c(s)*, were calculated by least-squares boundary modelling of sedimentation velocity data using the program SEDFIT (*Schuck*, 2000). These coefficients were corrected to standard conditions (water and 20°C) to obtain the corresponding *s₂₀,w* values using the SEDNTERP program (*Minton*, 1994). The latter program was also used to calculate the partial specific volume of the protein from its amino acid composition. Short-column (80 μl) sedimentation equilibrium experiments were performed at two successive speeds (10 000 and 13 000 r.p.m.) on parallel protein samples. Afterward, baselines were measured at high speed (40 000 r.p.m.). The weight-average molar mass of the protein was calculated by fitting the experimental gradients to the equation that describes the radial concentration distribution of a solute at sedimentation equilibrium, as implemented in the EQASSOC program (*Minton*, 1994).

© 2010 Blackwell Publishing Ltd, *Molecular Microbiology*, **79**, 359–374

372 J. Nogales et al.

### Synthesis of OMAketo, OMAenol and CHA substrates

Since GA (Sigma-Aldrich) is the only commercially available substrate of the GA degradation pathway, enzymatic synthesis was necessary for the production of the rest of the GA degradation pathway intermediates. OMAketo (200 μM) was freshly prepared routinely by incubation for 5 min at 25°C of 200 μM GA with 5 μg of purified GalA in 50 mM sodium phosphate buffer, pH 7.0. OMAenol (200 μM) was prepared by incubation of the fresh OMAketo form preparation for 5 min at 25°C with 5 μg of *E. coli* BL21 (DE3) (pETGalDc) extracts. Since the ε₂₆₅ for OMA calculated previously did not differentiate between the keto and enol forms (Maruyama, 1985; Hara *et al.*, 2000), we have determined in this work an ε₂₆₅ = 2.000 M⁻¹ cm⁻¹ for OMAenol. CHA (200 μM) was prepared by incubating 200 μM OMAenol with 5 μg of purified OMA hydratase for 5 min at 25°C in 50 mM sodium phosphate buffer, pH 7.0.

### Enzymatic assays

Gallic acid dioxygenase (GalA) was assayed as previously described (Nogales *et al.*, 2005). The OMA tautomerase (GalD) and OMA hydratase (GalB) activities were assayed spectrophotometrically at 30°C by measuring the increase of A₂₆₅ (ε = 2.000 M⁻¹ cm⁻¹) due to the conversion of OMAketo to OMAenol, and the decrease of A₂₆₅ due to the disappearance of OMAenol, respectively, in 50 mM sodium phosphate buffer, pH 7.0. The CHA aldolase (GalC) activity was assayed at 30°C by monitoring the decrease in A₃₄₀ derived from NADH oxidation (ε = 6.6 × 10³ M⁻¹ cm⁻¹) in a coupled assay containing 200 μM CHA, 140 μM NADH, 30 U lactate dehydrogenase (Roche), MgCl₂ 1 mM, and crude extract from *E. coli* BL21 (DE3) (pETGalC) (1 μg) or *P. putida* KTGAL (5 μg), in 50 mM sodium phosphate buffer, pH 7.0, as previously described (Maruyama, 1990). The β-galactosidase activity of permeabilized *P. putida* KTGAL (pIZPb) cells was monitored as previously described (Miller, 1972).

### Chemical modifications of GalB by site-specific reagents

Stock solutions of the reagents were made up fresh daily. The cysteine-specific reagents, *N*-ethylmaleimide and iodoacetamide, were prepared at a concentration of 0.2 M by dissolving in water. The glutamic/aspartic acid-specific reagent 1-ethyl-3-(3-dimethylaminopropyl) carbodiimide (EDC) was prepared at a concentration of 0.2 M by dissolving in water. The histidine-specific reagent diethyl pyrocarbonate (DEPC) was diluted to 0.5 M in acetonitrile. The serine-specific reagent phenylmethylsulphonyl fluoride (PMSF) was diluted to 0.1 M in isopropyl alcohol. Inactivation of purified GalB enzyme was performed by incubation for 30 min (7 min when DEPC was used) at 25°C in 50 mM sodium phosphate buffer, pH 7.0 containing the inhibitor, i.e. 1 mM *N*-ethylmaleimide, 0.5 mM PMSF, 5 mM iodoacetamide, 1 mM EDC or 0.5 mM DEPC. The remaining OMA hydratase activity was assayed and compared with that of the untreated enzyme.

### Metabolite and Zn²⁺ detection

Metabolite detection was carried out by ¹H MNR and samples were prepared in 10% D₂O (pH 7.0). The spectra were acquired at 298 K on an AVANCE 500 MHz spectrometer, equipped with a broad-band z-gradient probe (Bruker). The Watergate module was employed to suppress the residual water resonance. The ¹H NMR chemical shifts are given using trimethylsilyl propionate as a reference (0 ppm). The Zn²⁺ content in purified GalB was determined by ICP using an ICP-OES Optima 2000DV equipment (PerkinElmer).

### Sequence data analyses

A homology search with finished and unfinished microbial genome databases was performed with the TBLASTN algorithm (Altschul *et al.*, 1990) at the National Center for Biotechnology Information server (http://www.ncbi.nlm.nih.gov/sutils/genom_table.cgi). Amino acid sequences retrieved from the protein databases were aligned using the CLUSTALW program (Thompson *et al.*, 1994), and the phylogenetic tree was performed by using the neighbour-joining method (Saitou and Nei, 1987) at the EMBL-EBI server (http://www.ebi.ac.uk/Tools/clustalw2/index.html), and visualized with the Tree View Program version 1.6.6. The topological model of GalT was generated using the HMMTOP program (http://www.enzim.hu/hmmtop/index.html).

### Three-dimensional modelling of GalB, GalC and GalD

The three-dimensional models for GalB, GalC and GalD from *P. putida* KT2440 were built by using the alignment mode at the SWISS-MODEL Protein Homology-Modelling Server (http://swissmodel.expasy.org/) (Arnold *et al.*, 2006). The crystal structures of the MshB deacetylase from *Mycobacterium tuberculosis* H37Rv (PDB 1Q7T), the Yer010c protein from *S. cerevisiae* (PDB 2C5Q) and the PrpF 2-methylcitrate cis-trans-isomerase from *Shewanella oneidensis* (PDB 2PVZ) were used as templates for GalB, GalC and GalD modelling respectively. The figure management was performed by using the PyMOL viewer program (DeLano, 2002).

### Acknowledgements

We thank A. Valencia for technical assistance; C. Alfonso and G. Rivas for the ultracentrifugation analyses; S.L. Secugen for DNA sequencing; D. Stjepandic and J.D. Hoheisel for the gift of the *P. putida* cosmid library; and E. Masai and E. Mahenthiralingam for strains *S. paucimobilis* DLJ and *Burkholderia* sp. 383 respectively. This work was supported by Grants GEN2001-4698-C05-02, GEN2006-27750-C5-3-E, CSD2007-00005, BIO2009-10438. J.N. was the recipient of an I3P predoctoral fellowship from the CSIC.

### References

Alhapel, A., Darley, D.J., Wagener, N., Eckel, E., Elsner, N., and Pierik, A.J. (2006) Molecular and functional analysis of nicotinate catabolism in *Eubacterium barkeri*. *Proc Natl Acad Sci USA* **103**: 12341–12346.

Altschul, S.F., Gish, W., Miller, W., Myers, E.W., and Lipman, D. (1990) Basic local alignment search tool. *J Mol Biol* **215**: 403–410.

Arnold, K., Bordoli, L., Kopp, J., and Schwede, T. (2006) The SWISS-MODEL workspace: a web-based environment for protein structure homology modelling. *Bioinformatics* 22: 195–201.

Beveridge, E.G., and Hugo, W.B. (1964) The metabolism of gallic acid by *Pseudomonas convexa* X.1. *J Appl Bacteriol* 27: 448–459.

Bradford, M.M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal Biochem* 72: 248–254.

Chowdhury, S.P., Khanna, S., Verma, S.C., and Tripathi, A.K. (2004) Molecular diversity of tannic acid degrading bacteria isolated from tannery soil. *J Appl Microbiol* 97: 1210–1219.

Davis, B.D., and Mingioli, E.S. (1950) Mutants of *Escherichia coli* requiring methionine or vitamin B12. *J Bacteriol* 60: 17–28.

DeLano, W. (2002) *The PyMOL Molecular Graphics System*. San Carlos, CA: DeLano Scientific.

Ditty, J.L., and Harwood, C.S. (2002) Charged amino acids conserved in the aromatic acid/H+ symporter family of permeases are required for 4-hydroxybenzoate transport by PcaK from *Pseudomonas putida*. *J Bacteriol* 184: 1444–1448.

Gamella, M., Campuzano, S., Reviejo, A.J., and Pingarrón, J.M. (2006) Electrochemical estimation of the polyphenol index in wines using a laccase biosensor. *J Agric Food Chem* 54: 7960–7967.

Garvey, G.S., Rocco, C.J., Escalante-Semerena, J.C., and Rayment, I. (2007) The three-dimensional crystal structure of the PrpF protein of *Shewanella oneidensis* complexed with trans-aconitate: insights into its biological function. *Protein Sci* 16: 1274–1284.

Hara, H., Masai, E., Katayama, Y., and Fukuda, M. (2000) The 4-oxalomesaconate hydratase gene, involved in the protocatechuate 4,5-cleavage pathway, is essential to vanillate and syringate degradation in *Sphingomonas paucimobilis* SYK-6. *J Bacteriol* 182: 6950–6957.

Hara, H., Masai, E., Miyauchi, K., Katayama, Y., and Fukuda, M. (2003) Characterization of the 4-carboxy-4-hydroxy-2-oxoadipate aldolase gene and operon structure of the protocatechuate 4,5-cleavage pathway genes in *Sphingomonas paucimobilis* SYK-6. *J Bacteriol* 185: 41–50.

Harayama, S., Rekik, M., Ngai, K.L., and Ornston, L.N. (1989) Physically associated enzymes produce and metabolize 2-hydroxy-2,4-dienoate, a chemically unstable intermediate formed in catechol metabolism via meta cleavage in *Pseudomonas putida*. *J Bacteriol* 171: 6251–6258.

Harwood, C.S., and Parales, R.E. (1996) The β-ketoadipate pathway and the biology of self-identity. *Annu Rev Microbiol* 50: 553–590.

Hernick, M., and Fierke, C.A. (2005) Zinc hydrolases: the mechanisms of zinc-dependent deacetylases. *Arch Biochem Biophys* 433: 71–84.

Jiménez, J.I., Nogales, J., García, J.L., and Díaz, E. (2010) A genomic view of the catabolism of aromatic compounds in *Pseudomonas*. In *Handbook of Hydrocarbon and Lipid Microbiology*. Timmis, K.N. (ed.). Berlin Heidelberg: Springer-Verlag, pp. 1297–1325.

Kasai, D., Masai, E., Miyauchi, K., Katayama, Y., and Fukuda,

M. (2005) Characterization of the gallate dioxygenase gene: three distinct ring cleavage dioxygenases are involved in syringate degradation by *Sphingomonas paucimobilis* SYK-6. *J Bacteriol* 187: 5067–5074.

Kersten, P.J., Dagley, S., Whittaker, J.W., Arciero, D.M., and Lipscomb, J.D. (1982) 2-pyrone-4,6-dicarboxylic acid, a catabolite of gallic acids in *Pseudomonas* species. *J Bacteriol* 152: 1154–1162.

Laclac, J., Alfonso, C., Liu, X., Parales, R.E., Morel, B., Conejero-Lara, F., *et al.* (2010) Identification of a chemoreceptor for tricarboxylic acid cycle intermediates: differential chemotactic response towards receptor ligands. *J Biol Chem* 285: 23126–23136.

Leulliot, N., Quevillon-Cheruel, S., Graille, M., Schiltz, M., Blondeau, K., Janin, J., and Van Tilbeurgh, H. (2005) Crystal structure of yeast YER010Cp, a notable member of the RraA protein family. *Protein Sci* 14: 2751–2758.

Li, S., Kimura, M., Takashima, T., Hayashi, K., Inoue, K., Ishiguro, R., *et al.* (2007) Role of cysteine residues in 4-oxalomesaconate hydratase from *Pseudomonas ochraceae* NGJ1. *Biosci Biotechnol Biochem* 71: 449–457.

de Lorenzo, V., and Timmis, K.N. (1994) Analysis an construction of stable phenotypes in gram-negative bacteria with Tn5- and Tn10-derived minitransposons. *Methods Enzymol* 235: 386–405.

McCarthy, A., Peterson, N., Knijff, R., and Baker, E. (2004) Crystal structure of MshB from *Mycobacterium tuberculosis*, a deacetylase involved in mycothiol biosynthesis. *J Mol Biol* 335: 1131–1141.

Maruyama, K. (1985) Purification and properties of gamma-oxalomesaconate hydratase from *Pseudomonas ochraceae* grown with phthalate. *Biochem Biophys Res Commun* 128: 271–277.

Maruyama, K. (1990) Purification and properties of 4-hydroxy-4-methyl-2-oxoglutarate aldolase from *Pseudomonas ochraceae* grown on phthalate. *J Biochem* 108: 327–333.

Maruyama, K., Miwa, M., Tsujii, N., Nagai, T., Tomita, N., Harada, T., *et al.* (2001) Cloning, sequencing, and expression of the gene encoding 4-hydroxy-4-methyl-2-oxoglutarate aldolase from *Pseudomonas ochraceae* NGJ1. *Biosci Biotechnol Biochem* 65: 2701–2709.

Maynes, J.T., Garen, C., Cherney, M.M., Newton, G., Arad, D., Av-Gay, Y., *et al.* (2003) The crystal structure of 1-D-myo-inosityl 2-acetamido-2-deoxy-α-D-glucopyranoside deacetylase (Mshb) from *Mycobacterium tuberculosis* reveals a zinc hydrolase with a lactate dehydrogenase fold. *J Biol Chem* 278: 47166–47170.

Miller, J.H. (1972) *Experiments in Molecular Genetics*. New York: Cold Spring Harbor Laboratory Press.

Minton, A.P. (1994) Conservation of signal: a new algorithm for the elimination of the reference concentration as an independently variable parameter in the analysis of sedimentation equilibrium. In *Modern Analytical Ultracentrifugation*. Schuster, T.M., and Laue, T.M. (eds). Boston, MA: Birkhäuser, pp. 81–93.

Nelson, K.E., Weinel, C., Paulsen, I.T., Dodson, R.J., Hilbert, H., Martins dos Santos, V.A.P., *et al.* (2002) Complete genome sequence and comparative analysis of the metabolically versatile *Pseudomonas putida* KT2440. *Environ Microbiol* 4: 799–808.

Nogales, J., Canales, A., Jiménez-Barbero, J., García, J.L., and Díaz, E. (2005) Molecular characterization of the gallate dioxygenase from *Pseudomonas putida* KT2440: the prototype of a new subgroup of extradiol dioxygenases. *J Biol Chem* 280: 35382–35390.

Ow, Y.Y., and Stupans, I. (2003) Gallic acid and gallic acid derivatives: effects on drug metabolizing enzymes. *Curr Drug Metab* 4: 241–248.

Parales, R.E., Ditty, J.L., and Harwood, C.S. (2000) Toluene-degrading bacteria are chemotactic towards the environmental pollutants benzene, toluene, and trichloroethylene. *Appl Environ Microbiol* 66: 4098–4104.

Parales, R.E., Ju, K.S., Rollefson, J.B., and Ditty, J. (2008) Bioavailability, chemotaxis, and transport of organic pollutants. In *Microbial Biodegradation. Genomics and Molecular Biology*. Díaz, E. (ed.). Norfolk, UK: Caister Academic Press, pp. 145–187.

Pillai, B., Cherney, M.M., Diaper, C.M., Sutherland, A., Blanchard, J.S., Vederas, J.C., and James, M.N.G. (2006) Structural insights into stereochemical inversion by diaminopimelate epimerase: an antibacterial drug target. *Proc Natl Acad Sci USA* 103: 8668–8673.

Pollard, J.R., and Bugg, T.D.H. (1998) Purification, characterisation and reaction mechanism of monofunctional 2-hydroxypentadienoic acid hydratase from *Escherichia coli*. *Eur J Biochem* 251: 98–106.

Roper, D.I., Subramanya, H.S., Shingler, V., and Wigley, D.B. (1994) Preliminary crystallographic analysis of 4-oxalocrotonate tautomerase reveals the oligomeric structure of the enzyme. *J Mol Biol* 243: 799–801.

Saitou, N., and Nei, M. (1987) The neighbor-joining method: a new method for reconstructing phylogenetic trees. *Mol Biol Evol* 4: 406–425.

Sambrook, J., and Russell, D.W. (2001) *Molecular Cloning, a Laboratory Manual*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Sanger, F., Nicklen, S., and Coulson, A.R. (1977) DNA sequencing with chain-terminating inhibitors. *Proc Natl Acad Sci USA* 74: 5463–5467.

Schuck, P. (2000) Size-distribution analysis of macromolecules by sedimentation velocity ultracentrifugation and Lamm equation modeling. *Biophys J* 78: 1606–1619.

Sparnins, V.L., and Dagley, S. (1975) Alternative routes of aromatic catabolism in *Pseudomonas acidovorans* and *Pseudomonas putida*: gallic acid as a substrate and inhibitor of dioxygenases. *J Bacteriol* 124: 1374–1381.

Subramanya, H.S., Roper, D.I., Dauter, Z., Dodson, E.J., Davies, G.J., Wilson, K.S., and Wigley, D.B. (1996) Enzymatic ketonization of 2-hydroxymuconate: specificity and mechanism investigated by the crystal structures of two isomerases. *Biochemistry* 35: 792–802.


Tack, B.F., Chapman, P.J., and Dagley, S. (1972a) Metabolism of gallic acid and syringic acid by *Pseudomonas putida*. *J Biol Chem* 247: 6438–6443.

Tack, B.F., Chapman, P.J., and Dagley, S. (1972b) Purification and properties of 4-hydroxy-4-methyl-2-oxoglutarate aldolase. *J Biol Chem* 247: 6444–6449.

Tamber, S., Ochs, M.M., and Hancock, R.E.W. (2006) Role of the novel OprD family of porins in nutrient uptake in *Pseudomonas aeruginosa*. *J Bacteriol* 188: 45–54.

Taylor, A.B., Czerwinski, R.M., Johnson, W.H., Whitman, C.P., and Hackert, M.L. (1998) Crystal structure of 4-oxalocrotonate tautomerase inactivated by 2-oxo-3-pentynoate at 2.4 Å resolution: analysis and implications for the mechanism of inactivation and catalysis. *Biochemistry* 37: 14692–14700.

Thompson, J.D., Higgins, D.G., and Gibson, T.J. (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. *Nucleic Acids Res* 22: 4673–4680.

Velarde, M., Macieira, S., Hilberg, M., Bröker, G., Tu, S.M., Holding, B.T., et al. (2009) Crystal structure and putative mechanism of 3-methylitaconate-δ-isomerase from *Eubacterium barkeri*. *J Mol Biol* 391: 609–620.

Wattiau, P., Bastiaens, L., van Herwijnen, R., Daal, L., Parsons, J.R., Renard, M.E., et al. (2001) Fluorene degradation by *Sphingomonas sp.* LB126 proceeds through protocatechuic acid: a genetic analysis. *Res Microbiol* 152: 861–872.

Weinel, C., Nelson, K.E., and Tümmler, B. (2002) Global features of the *Pseudomonas putida* KT2440 genome sequence. *Environ Microbiol* 4: 809–818.

Whitman, C.P., Aird, B.A., Gillespie, W.R., and Stolowich, N.J. (1991) Chemical and enzymic ketonization of 2-hydroxymuconate, a conjugated enol. *J Am Chem Soc* 113: 3154–3162.

Williams, P.A., and Worsley, M.J. (1976) Ubiquity of plasmids in coding for toluene and xylene metabolism in soil bacteria: evidence for the existence of new TOL plasmids. *J Bacteriol* 125: 818–828.


Supporting information

Additional supporting information may be found in the online version of this article.

Please note: Wiley-Blackwell are not responsible for the content or functionality of any supporting materials supplied by the authors. Any queries (other than missing material) should be directed to the corresponding author for the article.
