
This Review is part of a new thematic series on Cardiovascular Systems Modeling, which includes the following articles:

Integrative Systems Models of Cardiac Excitation–Contraction Coupling. [Circ Res. 2011;108:70–84]

Computational Models Reduce Complexity and Accelerate Insight Into Cardiac Signaling Networks

Alternans and Arrhythmias: From Cells to the Heart  
Whole Heart Modeling: Applications to Cardiac Electrophysiology and Electromechanics  

Raimond Winslow, Guest Editor


# Computational Models Reduce Complexity and Accelerate Insight Into Cardiac Signaling Networks

Jason H. Yang, Jeffrey J. Saucerman

**Abstract:** Cardiac signaling networks exhibit considerable complexity in size and connectivity. The intrinsic complexity of these networks complicates the interpretation of experimental findings. This motivates new methods for investigating the mechanisms regulating cardiac signaling networks and the consequences these networks have on cardiac physiology and disease. Next-generation experimental techniques are also generating a wealth of genomic and proteomic data that can be difficult to analyze or interpret. Computational models are poised to play a key role in addressing these challenges. Computational models have a long history in contributing to the understanding of cardiac physiology and are useful for identifying biological mechanisms, inferring multiscale consequences to cell signaling activities and reducing the complexity of large data sets. Models also integrate well with experimental studies to explain experimental observations and generate new hypotheses. Here, we review the contributions computational modeling approaches have made to the analysis of cardiac signaling networks and forecast opportunities for computational models to accelerate cardiac signaling research. (Circ Res. 2011;108:85-97.)

Key Words: cardiac signaling network ■ computational modeling ■ β-adrenergic signaling ■ CaMKII signaling

In the early 1990s, a major shift occurred in the understanding of heart failure as more than just a hemodynamic dysfunction. This paradigm shift (the neurohormonal hypothesis) states that persistent cell signaling by circulating factors is a key driver of pathological cardiac remodeling and helps perpetuate the heart failure phenotype.¹⁻³ This view guides the existing therapeutic strategy for managing heart failure and many other cardiac diseases: use β-blockers, angiotensin-converting enzyme inhibitors, and angiotensin receptor blockers to block the signaling pathways most strongly activated in disease first; treat other symptoms later. Indeed,

60% of all FDA-approved drugs target membrane-bound proteins; two-thirds of these are receptors for cell signaling.⁴  
This paradigm underscores the central role of cell signaling pathways in regulating cardiac physiology (eg, contractility, metabolism) and pathophysiology (eg, hypertrophy, fibrosis). These pathways comprise an intricate network of biochemical interactions between signaling proteins and exhibit considerable complexity. Tremendous effort has been exerted in the past 2 decades to understand how cardiac signaling pathways are regulated, how these pathways interface with core functions of the heart, and which pathway components are best

Original received August 8, 2010; revision received October 13, 2010; accepted October 18, 2010. In September 2010, the average time from submission to first decision for all original research papers submitted to *Circulation Research* was 13.1 days.

From the Department of Biomedical Engineering and Robert M. Berne Cardiovascular Research Center, University of Virginia, Charlottesville.

Correspondence to Dr Jeffrey J. Saucerman, Department of Biomedical Engineering, University of Virginia, Box 800759 Health System, Charlottesville, VA 22908. E-mail jsaucerman@virginia.edu

© 2011 American Heart Association, Inc.

*Circulation Research* is available at http://circres.ahajournals.org

DOI: 10.1161/CIRCRESAHA.110.223602

Downloaded from http://circres.ahajournals.org by guest on March 25, 2015

Non-standard Abbreviations and Acronyms

| $\beta_{1}$AR | $\beta_{1}$-adrenergic receptor |
| --- | --- |
| CaM | calmodulin |
| EC | excitation-contraction |
| FDA | Food and Drug Administration |
| FRET | Förster resonance energy transfer |
| IP3 | inositol 1,4,5-triphosphate |
| LCC | L-type $Ca^{2+}$ channel |
| LQT | long QT |
| LQTS | long QT syndrome |
| PK | protein kinase |
| PPI | protein-protein interaction |
| RyR | ryanodine receptor |
| SR | sarcoplasmic reticulum |
| TNF | tumor necrosis factor |

Formulate  
Biological  
Question  

Develop/  
Refine  
Hypothesis  

Develop/  
Refine  
Computational  
Model  

Generate  
Model  
Predictions  

Answer  
Biological  
Question  

Experimentally  
Test/Validate  
Model  
Predictions  

Figure 1. Typical workflow for a computational modeling study. Specific biological questions motivate the development of an appropriate modeling approach. Model prediction, validation, and refinement is iteratively cycled until an experimentally testable answer is generated to answer the original biological question. These answers can sometimes motivate new hypotheses or new biological questions.

suited for drug targeting. However, the sheer complexity of these networks often stumps experimental intuition and motivates new approaches to study cell signaling.

Still, 2 decades of experimental studies have yielded a wealth of insight into the biochemistry of signaling proteins, delineation of signaling pathways, and consequences on in vivo cardiac function. Experimental studies have also empirically explained some of the fundamental mechanisms for cell signaling control, including pathway crosstalk, transcriptional feedback, and spatial compartmentation. These studies collectively form large bodies of evidence that require new frameworks to organize and interpret data as they are generated.

Although biologists who study cardiac signaling networks use different methods to probe their respective research interests, the questions they ask are often similar. In general, these questions can be distilled into 3 fundamental questions: (1) What are the key mechanisms regulating my biological research interest? (2) What are the multiscale/multifunctional consequences of my findings? (3) How can I extract the most useful biological information from large data sets?

These 3 issues of identifying mechanism, determining consequence, and interpreting complex data are challenges well suited for computational modeling and analysis.

### Computational Modeling

By definition, a model is a simplified representation of a complex system. Every experimentalist uses conceptual models to make predictions about how 2 species may be causally related. Conceptual models help contextualize data, build intuition, generate new hypotheses, and facilitate experimental design. Computational models formalize these representations by using mathematical equations to describe the relationships between species.

Computational models take on a diversity of forms. The choice of model structure should be determined by the type of questions being asked, level of quantitative detail desired, and quality of experimental data available to constrain model equations and parameters. Mechanistic models are frequently

Downloaded from [http://circres.ahajournals.org](http://circres.ahajournals.org) by guest on March 25, 2015

insight into important behaviors such as cardiac pacemaking in the sinoatrial node,¹² excitation–contraction coupling,¹³,¹⁴ cross-bridge cycling,¹⁵ and arrhythmogenesis in the ventricles.¹⁶ Such computational modeling work in the heart played an important role in the development of systems biology,¹⁷ giving rise to global efforts such as the International Union of Physiological Science (IUPS) Physiome Project.¹⁸,¹⁹ The Physiome Project aims to understand total human cell, tissue, and whole-organism physiology via the integration of databases and computational models. Here, we specifically review the contributions computational modeling approaches have made toward understanding the regulation of cardiac signaling pathways and how these pathways interface with cardiac physiology.

### Mechanistic Models of Cell Signaling Networks

The most common forms of computational models used to study cell signaling networks are those with biochemical and biophysical mechanistic detail. These take a “bottom-up” approach, using laws of mass action and Michaelis–Menten enzyme kinetics to mechanistically represent individual biochemical reactions in a cell signaling network. These models are frequently developed to predict the time-varying relationships between the components being modeled.²⁰ This approach assumes that the individual biochemical reactions represented are sufficient to describe the overall signaling network dynamics, and its predictions can be sensitive to missing reactions. Although these models require a large number of parameters to be appropriately constrained, the detailed representation of biochemical and biophysical mechanisms enables these models to perform predictive computational experiments, which can later be validated experimentally.

Bhalla and Iyengar pioneered the use of large signaling network models, integrating many signaling pathways such as protein kinase (PK)A, PKC, mitogen-activated protein kinase, inositol 1,4,5-triphosphate (IP3), Ca²⁺/calmodulin-dependent protein kinase (CaMK)II, and Ca²⁺ signaling in neurons.²¹ In this work, the authors demonstrated how intersecting signaling pathways, as whole networks, can give rise to emergent behaviors such as signal integration across time scales, bistability, and feedback. Mechanistic models are also useful for understanding the fundamental design principles underlying biological networks. Alon and colleagues combined modeling and experimental studies to demonstrate how common network motifs found in signaling networks can give rise to a diverse spectrum of systems properties such as network robustness, signaling acceleration/deceleration, and memory.²²,²³ Taken together, biochemically and biophysically mechanistic models of cell signaling are useful for understanding the mechanisms for cell signaling regulation and the relationships between cell signaling networks and the functions they regulate.

### β-Adrenergic Signaling in the Cardiac Myocyte

β-Adrenergic signaling centrally regulates cardiac contractility and the progression of heart failure (Figure 2).²⁴⁻²⁶ Under normal sympathetic activity, catecholamines bind β-adrenergic

![Figure 2. Schematic of β-adrenergic signaling in the cardiac myocyte. Catecholamine binding to the β₁- and β₂-adrenergic receptors initiating a signaling cascade of G-protein activation, adenyl cyclase production of cAMP, and PKA activation by cAMP. Phosphorylation of PKA substrates may elicit a number of cardiac behaviors such as inotropy, lusitropy, chronotropy, hypertrophy, and apoptosis.](image.png)

G-protein-coupled receptors, signaling through Gₛ to activate PKA. PKA can then elicit enhanced contractile (inotropy), relaxation (lusitropy), growth (hypertrophy), and death (apoptosis) responses to adapt to altered circulatory demands. Although acute β-adrenergic signaling is important for the fight-or-flight response, persistent β-adrenergic signaling induces hypertrophy, fibrosis, and heart failure.²⁷,²⁸ In failing hearts, expression for multiple β-adrenergic signaling proteins decreases significantly,²⁹,³⁰ and drugs that directly inhibit β-adrenergic signaling (β-blockers) are effective first-line therapies prescribed in the management of cardiac disease,³¹⁻³⁴ although the primary mechanisms of action are still unknown.

Saucerman et al. used this mechanistic modeling approach to investigate the β-adrenergic signaling pathway in the cardiac myocyte.³⁵ Model simulations were used to compare gene therapy strategies and show how protein kinase inhibitor (PKI) can make PKA act as an ultrasensitive switch. Another key aim of this study was to understand which intracellular targets for β-adrenergic signaling were most important for regulating cardiac inotropy. Model analysis demonstrated that most of the inotropic changes attributable to β₁-adrenergic signaling could be explained by PKA-mediated phosphorylation of the L-type Ca²⁺ channel (LCC) and phospholamban. In a subsequent study,³⁶ the authors extended this model to show how PKA substrates perform specialized tasks in β₁-adrenergic receptor (β₁AR)-stimulated inotropy: phospholamban phosphorylation increases sarcoplasmic reticulum (SR) Ca²⁺ load and accelerates relaxation; LCC and phospholamban phosphorylation together contribute to increased systolic Ca²⁺. Moreover, although PKA phosphorylation of the ryanodine receptors (RyRs) increased the Ca²⁺ sensitivity for SR Ca²⁺ release,³⁷ its impact on steady-state Ca²⁺ transients was limited, consistent with the Ca²⁺ autoregulation hypothesis.³⁸ Collectively, these studies were the first to show how molecular perturbations in a cardiac signaling network could be interpreted in the context of cell physiology using computational models.

Other groups continued to explore the effects of β-adrenergic signaling on cardiac electrophysiology. Using a

stochastic model of excitation–contraction (EC) coupling in the canine ventricular myocyte, Greenstein et al showed how increases in EC coupling gain observed in β-adrenergic stimulation may be explained by increased SR Ca²⁺ load rather than alterations to LCC gating.³⁹ Moreover, the authors showed how a shift in LCC gating toward higher activity can generate stochastic early after depolarizations with implications for arrhythmogenesis under β-adrenergic stimulation. This study highlights the importance of selecting an appropriate model structure to specifically answer a biological question: by using a stochastic (instead of deterministic) approach, the authors were able to detect rare, probabilistic arrhythmic events generated by β-adrenergic signaling that are otherwise nonintuitive and difficult to observe.

Kuzumoto et al adapted the Saucerman model to study the effects of β₁AR signaling on Na⁺ regulation in a guinea pig EC coupling model, demonstrating the necessary role of phospholemman for limiting the increase in Na⁺ concentration under β₁AR stimulation.⁴⁰ This model prediction was later validated experimentally by Despa et al.⁴¹ Himeno et al adapted the Kuzumoto model to analyze the role of β₁AR stimulation in inducing positive chronotropy.⁴² They showed how increased firing frequency is driven by a combination of changes to the LCC current (I<sub>CaL</sub>), sustained inward (Na⁺ and K⁺) current (I<sub>si</sub>), and hyperpolarization-activated nonselective cation (Na⁺ and K⁺) current (I<sub>ha</sub>). Moreover, the authors made the interesting observation that whereas the slow delayed rectifier K⁺ current (I<sub>Ks</sub>) contributes weakly to overall K⁺ conductance, I<sub>Ks</sub> plays an important role in counterbalancing increases in I<sub>CaL</sub> and Na⁺/Ca²⁺ exchanger current (I<sub>NaCa</sub>) during β₁AR stimulation, which would otherwise prolong the action potential and compromise positive chronotropy. These studies demonstrate how computational models can help reduce the complexity of signaling interactions with cell physiology. Using these models, the authors were able to isolate the actions of β₁AR signaling on different ion channels to better understand how they act together in concert to regulate Na⁺ concentration or cell pacemaking (a feat that could be experimentally intractable).

Mechanistic models of cardiac signaling networks can also be integrated with experimental studies to probe mechanisms for cell signaling kinetics and localization. In a later study, Saucerman et al combined their β₁AR signaling model with live-cell imaging to examine the role of cAMP compartmentation in regulating PKA activity.⁴³ The authors integrated a spatially explicit implementation of their existing β₁AR signaling model with live cell Förster resonance energy transfer (FRET) imaging experiments in neonatal rat ventricular myocytes. Using the model, the authors showed how spatial PKA gradients detected in the FRET experiments could be explained by restricted cAMP diffusion, phosphodiesterase-mediated cAMP degradation, or PKA-mediated cAMP buffering and concluded that cAMP compartmentation is a candidate mechanism for rate-limiting PKA activation. Such compartmentation can be an important mechanism for cell signaling specificity.⁴⁴,⁴⁵

Iancu et al also took a combined modeling/FRET approach to investigate cAMP compartmentation in cardiac myocytes. The authors first developed a computational model to investigate how the M₂ muscarinic receptor can both stimulate and

![Figure 3](#fig3)

**Figure 3. Schematic of CaMKII signaling in the cardiac myocyte.** Cardiomyocyte depolarization triggers Ca²⁺ influx through the LCCs and Ca²⁺ release through the RyRs. Free Ca²⁺ binds calmodulin, which activates CaMKII and calcineurin to elicit inotropic, lusitropic, hypertrophic, and apoptotic responses.

inhibit cAMP-dependent responses to β₁AR stimulation.⁴⁶ Using their model, the authors showed how the subcellular localization of adenylyl cyclase isoforms stimulated (AC4/7) or inhibited (AC5/6) by the G<sub>i</sub> G-protein is sufficient for eliciting seemingly opposite cAMP responses to acetylcholine. Moreover, their model predicted a rebounding cAMP response following a transient acetylcholine stimulus, which the authors validated using a PKA-based FRET sensor for cAMP. In a later study, the authors combined their model simulations with experiments using an Epac2-based FRET sensor for cAMP to quantitatively estimate time-varying changes in cAMP concentration.⁴⁷ The authors showed cAMP concentrations were significantly higher in the bulk cytosol than near the PKA-based FRET sensor and suggested that these differences may explain how cAMP can differentially regulate PKA and Epac responses to β-adrenergic signaling. Together, these studies illustrate how modeling approaches can be used in concert with live-cell experimental studies to explain mechanisms for cell signaling regulation.

### CaMKII Signaling in the Cardiac Myocyte

CaMKII manages another central signaling arm in the cardiac myocyte (Figure 3).²⁴,⁴⁸ Ca²⁺-bound calmodulin (CaM) activates CaMKII, which phosphorylates many Ca²⁺, Na²⁺, and K⁺ channels to regulate EC coupling and cardiac excitability.⁴⁹,⁵⁰ CaMKII is an important integrator of many signaling pathways in the heart (Ca²⁺, IP₃, G<sub>q</sub>) and contributes to the heart failure phenotype by inducing hypertrophy, apoptosis, and aberrant Ca²⁺ handling, which can trigger arrhythmias.⁵¹,⁵² CaMKII also synergizes with PKA during β-adrenergic signaling,⁵³ and growing evidence suggests CaMKII inhibition may have a beneficial impact on the development of heart failure.⁵⁴ These observations have made CaMKII a potentially attractive target for treating cardiac diseases.⁵⁵

Hund et al implemented the first model of CaMKII signaling in an EC coupling model of the canine ventricular myocyte.⁵⁶ The authors showed that although CaMKII contributes to the positive Ca²⁺-frequency relation by increasing EC coupling gain, action potential duration adaptation at
higher frequencies is best explained by the effects of transient outward K⁺ current (Iₜₒ₁) on repolarization rather than CaMKII. Similarly, Grandi et al modeled the actions of CaMKII on Iₐₗ, Iₜₒ₁, and the fast Na⁺ current (Iₙₐ) in a rabbit EC coupling model, showing that although the individual effects of CaMKII on Iₙₐ or Iₜₒ₁ may prolong the action potential, the combined effect on all 3 contribute to action potential shortening.⁵⁷ Moreover, the authors showed how transmural variations in Iₜₒ₁ expression may enhance CaMKII-induced arrhythmia in heart failure. The findings from these 2 studies could only be achieved by separating the individual contributions of CaMKII phosphorylation to the overall action potential, a task made tractable by the use of computational models.

Hashambhoy et al also examined the consequences of CaMKII activity on cardiomyocyte electrophysiology. Using a stochastic EC coupling model of the canine ventricular myocyte, the authors showed that CaMKII-dependent shifts in LCC gating can explain LCC facilitation and the apparent faster recovery from LCC inactivation independent of changes to LCC inactivation kinetics.⁵⁸ In a subsequent study, the authors showed how LCC phosphorylation by CaMKII decreases EC coupling gain with a greater effect on RyR Ca²⁺ release than RyR phosphorylation itself.⁵⁹ Moreover, the authors show LCC hyperphosphorylation is sufficient to induce early afterdepolarizations. Soltis and Saucerman used a similar approach to show CaMKII is required for the rapid adaptation that underlies FDAR (frequency-dependent acceleration of recovery).⁶⁰ Integrating the Saucerman–McCulloch β₁AR signaling model into their work, the authors also show that CaMKII and PKA synergize to potentiate a positive-feedback loop of CaMKII-Ca²⁺-CaMKII regeneration. Finally, the authors demonstrate that CaMKII hyperphosphorylation of RyRs can trigger delayed afterdepolarizations, adding support to the hypothesis that CaMKII is responsible for inducing arrhythmias via leaky RyRs. Together, the above modeling studies demonstrate how CaMKII signaling controls the positive Ca²⁺-frequency relation, LCC facilitation, FDAR, and triggers for arrhythmogenesis, illustrating the utility of computational models for linking molecular signaling activities to emergent behaviors in the cardiac myocyte.

Computational models have also examined the role of CaMKII after myocardial infarction. Hund et al experimentally measured increased CaMKII autophosphorylation in the border zone and used a model to explain how this can abnormally decrease Ca²⁺ transients by increasing Ca²⁺ leak from the SR.⁶¹ Hyperactive CaMKII also reduced action potential upstroke velocity by altering Iₙₐ gating kinetics, a potential mechanism for slow conduction and arrhythmogenesis at the myocardial infarct. Similarly, Christensen et al measured increased CaMKII oxidation at the border zone of a canine infarct and used a model to show how oxidized CaMKII can act on Iₙₐ to prolong the action potential refractory period, slow border zone conduction, and increase the vulnerability to conduction block at a canine myocardial infarct.⁶² Taken together, these studies illustrate the use of models for interpreting the pathophysiological consequences of experimentally measured alterations to cardiac signaling in a disease condition.

Other modeling studies have focused on the biochemical mechanisms for CaMKII activation. In a combined experimental and modeling study, Song et al first showed experimentally that CaM-dependent proteins (analogous to CaMKII and calcineurin) differentially process beat-to-beat Ca²⁺ signals based on their affinity for CaM.⁶³ The authors used a simple computational model to explain how these differences in CaM affinity can give rise to qualitatively different downstream signaling by these targets. Moreover, deactivation kinetics of CaM targets was driven by Ca²⁺ dissociation from the Ca²⁺-CaM-target complex rather than dissociation of Ca²⁺-CaM. Saucerman et al extended this modeling work to evaluate local CaMKII and calcineurin dynamics in the rabbit ventricular myocyte.⁶⁴ The authors showed how low-CaM affinity CaMKII and high-CaM affinity calcineurin can have different activity profiles and sensitivities to Ca²⁺ oscillations in the cytosol and dyadic cleft (where Ca²⁺ concentrations are considerably larger). By switching the CaMKII and calcineurin affinities for CaM, the local CaMKII and calcineurin responses were switched, leading the authors to conclude that the low affinity of CaMKII for CaM is what permitted CaMKII to be highly sensitive to Ca²⁺ in the dyad, but not in the cytosol. This was in contrast to calcineurin, which is highly sensitive to Ca²⁺ in the cytosol but not in the dyad. Chiba et al evaluated the role of phosphatases in regulating CaMKII.⁶⁵ The authors found that phosphatases limited CaMKII autophosphorylation and were important for regulating frequency-dependent activation of CaMKII. Collectively, these studies exemplify how models can be used to perform in silico experiments that are not otherwise tractable (eg, manipulation of binding affinities or tracking local CaMKII activity) to identify mechanisms for the regulation of cardiac signaling pathways. These studies also highlight the need for quantitative experiments of localized kinase activities in myocytes.

In addition to activating kinase (CaMKII) and phosphatase (calcineurin) activities, CaM also has direct effects in managing other aspects of EC coupling. For instance, Ca²⁺-CaM is known to directly regulate Ca²⁺-dependent inactivation of LCCs.⁶⁶,⁶⁷ Although this is typically modeled implicitly as a property of LCC gating, Tanskanen et al explicitly represented Ca²⁺-CaM binding activities in a stochastic model of the cardiac dyad. The authors showed how protein size and arrangement in these microdomains spatially restricts Ca²⁺ movement and influences the macroscopic properties of Ca²⁺-induced Ca²⁺ release.⁶⁸ Tadross et al used a computational model to further investigate how CaM can surprisingly confer sensitivity to small global Ca²⁺ signals in the presence of large local (Ca²⁺-dependent inactivating) signals to Ca²⁺ channels in neurons.⁶⁹ The authors find these behaviors are managed by rapid Ca²⁺ dissociation from Ca²⁺-CaM and preferential binding by Ca²⁺ channels to free CaM over Ca²⁺-CaM. These studies illustrate how models can be used to identify new mechanisms for how protein complexes can contribute over many scales to regulate overall cell function.

## Other Signaling Networks

Biochemically mechanistic models have been used to study a limited number of other cardiac signaling pathways as well.
Cooling et al modeled hypertrophic IP3 transients in response to endothelin-1 and angiotensin II.⁷⁰ Using a global sensitivity analysis to comprehensively test the role of every model parameter, the authors determined IP3 transients were primarily driven by dynamics at the receptor level. In particular, the authors showed how the more transient IP3 responses to angiotensin II than those generated by endothelin-1 could be explained by differences in receptor kinetics and density. This global sensitivity analysis exemplifies the type of comprehensive in silico experiments that can bring focus on key mechanisms of cell signaling regulation and prioritize future experiments.

Shin et al explored the counterintuitive observation that MCIP can inhibit cardiac hypertrophy by blocking calcineurin but can also stimulate hypertrophy in response to isoproterenol infusion or transverse aortic constriction.⁷¹ Using a computational model, the authors showed the biphasic behavior could be explained by a transcriptional negative-feedback loop that includes a large NFAT threshold for MCIP expression. At low or moderate NFAT activity, MCIP expression is low and calcineurin inhibition is small, permitting hypertrophy. However, when NFAT activity crosses this threshold, MCIP expression increases and calcineurin inhibition is large, attenuating hypertrophy. Cooling et al used models of NFAT translocation to examine how NFAT activity can be sensitive to both the magnitude and frequency of Ca²⁺ oscillations.⁷²

Niederer and Smith investigated the role of stretch-induced nitric oxide (NO) generation on Ca²⁺ cycling and force generation in the rat ventricular myocyte.⁷³ The authors represented the effects of NO on regulating RyR function by making one of the RyR gating variables a function of cardiac myocyte strain, which is thought to increase RyR relaxation via local NO generation. This model predicted a steady-state decrease in Ca²⁺ transients in response to the actions of NO on RyRs alone. The authors conclude that this mechanism does not fully explain the slow increased force response of cardiac myocytes engaged in sustained tension. This study demonstrates how models can identify gaps in understanding and draw attention to areas which require more experimental investigation.

### Summary

Although biochemically and biophysically mechanistic models require significant data for model validation, their detailed representations can be useful for identifying the key biological mechanisms regulating cardiac signaling pathways. These approaches help reduce the complexity of a signaling network by permitting comprehensive in silico assays that dissect the simultaneous effects of multiple interacting signaling mechanisms. These approaches can also help understand how small changes to the activity of a signaling network in a disease setting can produce large changes in phenotype. Moreover, these models have significant experimental predictive value and integrate well with experimental studies to complement experimental findings with mechanistic understanding.

---

### Multiscale/Integrated Models of Cardiac Signaling Networks

Cardiac cell signaling research is motivated by the need to understand how signaling networks regulate human cardiac physiology and disease. Extrapolating molecular signaling events to organ-level phenotypes introduces inherent complexity across spatial, temporal, and functional scales. As described above, mechanistic models of cell signaling pathways have tremendous deductive value for investigating biological mechanisms. However, other approaches are required for inductive extrapolation of the consequences of cell signaling on heart function. A second class of computational models are models that integrate distinct cardiac functions into a common framework. In practice, model integration is modular in nature and usually involves linking common variables across computational models of different cardiac behaviors. For example, Cortassa et al developed an integrated model of the guinea pig cardiomyocyte, linking cell electrophysiology, force generation, and mitochondrial energy generation to investigate phenomena such as oxidative-stress induced action potential shortening.⁷⁴ By combining these different units into a cohesive framework, integrated models clarify nonintuitive relationships between subsystems without obvious mechanistic links.

---

### Multiscale Models of Cardiac Function

This integrative approach is used most extensively in modeling multiscale aspects of cardiac electromechanics and hemodynamics.⁷⁵ Some multiscale models combine EC coupling models with detailed representations of ventricular anatomies to analyze cellular mechanisms for arrhythmia.⁷⁶ Other models integrate descriptions for circulatory resistance to model cardiac hemodynamics.⁷⁷,⁷⁸ Saucerman et al used this approach to analyze the arrhythmogenic effects of a point mutation (KCNQ1-G589D), which disrupts yotiao-mediated targeting of PKA and PPI to the \(I_{\text{Ks}}\) channel.⁷⁹ The authors integrated their β-adrenergic signaling model with a rabbit EC coupling model and showed how this mutation would lead to a prolonged action potential only under β-adrenergic stimulation. Coupling this model with a three-dimensional rabbit ventricular wedge model, the authors showed how these cellular long QT (LQT) events can amplify ventricular heterogeneities in electric propagation to give rise to arrhythmias. Using this integrated approach, the authors identified mechanisms for arrhythmia that were not obvious from the cellular or molecular phenotypes.

Nakamura et al used a similar approach for investigating progesterone-dependent mechanisms for changes in LQT risk during female menstruation and pregnancy.⁸⁰ Taking a combined experimental and computational approach, the authors analyzed the effects of progesterone on the cardiomyocyte action potential and ECG. The authors first experimentally showed that the actions of progesterone were managed by increased nitric oxide (NO) production by eNOS. The authors then used a computational model to simulate shortened action potentials in conditions with elevated NO, which were consistent with their electrophysiological measurements in intact myocytes. The authors combined this model with a single-fiber representation of ventricular tissue to simulate
ECG responses to progesterone administration in a form of congenital LQT syndrome (KCNE1-D76N), showing progesterone may protect against arrhythmias by regulating cardiac repolarization. This study exemplifies a novel use of combining an integrated computational model with experimental studies to examine the molecular mechanisms behind gender differences in risk for cardiac disease.

### Summary

Multiscale, integrative modeling approaches are helpful for understanding the consequences of molecular signaling events on overall cardiac physiology and pathophysiology. These models integrate detailed descriptions of many cardiac functions into a consistent framework and can identify biological mechanisms that emerge from the coupling between cells or the heterogeneities in cardiac tissue. These models are also useful for clarifying the relationship between cell signaling and global cardiac behaviors (eg, contractility, hemodynamics, electrophysiology, metabolism). Existing multiscale models including cell signaling have not yet incorporated tissue-level heterogeneity in cell signaling itself (eg, variation in neural density, paracrine factors, expression levels); this is an important future direction.

#### Analyzing Large Data Sets

Recent advancements in high-throughput methods for characterizing genomic, transcriptomic, proteomic, and metabolomic states allow one to view the global consequences of molecular perturbations rather than just the “usual suspects.” However, this wealth of -omics data creates new challenges in data interpretation, because most of the measurements lack a biological context for interpreting the biological relevance to the experimental perturbation. Statistical modeling techniques help reduce the complexity of these data sets by identifying clusters of signaling species that may either be coregulated or that can similarly regulate other species in a signaling network.

Statistical modeling approaches draw on information theory and computer science to identify features in the data that may globally represent the entire data set (eg, principal components). One advantage to these “top-down” techniques is that they make few assumptions about the data and can provide unbiased identification of unexpected correlations. However, statistical models produce different types of information than mechanistic models. Whereas mechanistic models can predict the time-varying dynamics and spatial localization of individual species in a cell signaling network, statistical models predict correlations between species in a network. Although these correlations do not always explain the causality between correlated species, the correlations can be useful for identifying nonintuitive patterns in the data and guide future experiments. These techniques can tremendously reduce the complexity of a high-throughput data set by 3 to 4 orders of magnitude and are most useful for screening a large number of observations or generating new hypotheses to explore experimentally. In the context of cardiac signaling networks, these approaches have been used most frequently to examine changes in coregulated gene/protein expression and changes in the activity of coregulated PPIs.

---

### Statistical Analysis of High-Throughput Genomic Data

Some of the earliest applications of statistical modeling approaches to cardiac signaling networks involved efforts to interpret DNA microarray data sets. These studies drew from machine learning to identify possible mechanisms regulating the gene or protein expression changes observed between normal and diseased (or transgenic) cardiac tissue. In one group of studies, Hall et al examined gene expression profiles associated with reverse remodeling in human hearts following left ventricular assist device treatment. These studies revealed a number of important changes to cardiac vascular organization, cytoskeleton organization, and integrin and cAMP signaling, suggesting these pathways may be relevant to cardiac remodeling. Hong et al took a similar approach to analyze transcriptional profiles corresponding to 17 mouse cardiac phenotypes. In that study, the authors used spectral graph clustering and identified 31 groups of cardiac-specific genes with coregulated expression. The authors validated the differential expression of some of these genes in a transverse aortic constriction mouse by RT-PCR. These studies illustrate efforts to identify candidate genetic regulators of cardiac remodeling.

Other efforts have specifically focused on understanding experimental models of heart failure. Gao et al compared the gene expression profiles from canine tachycardia-induced heart failure against gene expression profiles from heart failure in 2 other species: (1) human idiopathic and ischemic heart failure and (2) mouse tumor necrosis factor (TNF)α overexpression and MLP knockout heart failure. The authors discovered a number of gene expression changes common between these different heart failure models, including upregulation of nucleic acid metabolism and transcription pathways and downregulation of biosynthesis/metabolism and muscle development/contraction pathways. In a later study, the authors took a novel approach, combining microarray analysis with biochemically mechanistic modeling and in vivo hemodynamic and electrophysiological measurements to examine longitudinal cardiac remodeling in canine tachycardia-induced heart failure. They found significant gene expression changes to metabolism, cell signaling and extracellular matrix pathways early in the remodeling process and coincident with left ventricular dysfunction and action potential prolongation. Focusing on genes whose expression correlated with changes in action potential duration, the authors identified a number of candidate proteins that may regulate action potential duration, including the SERCA2 gene. The change in SERCA2 expression was validated by Western blot and a computational model was used to show that SERCA2 downregulation is a sufficient mechanism for prolonging the cardiac action potential. Taken together, the authors demonstrate how these bioinformatics algorithms can be combined with experimental and computational validation to bring focus to specific molecular targets that manage the heart failure phenotype.

---

### Statistical Analysis of High-Throughput Proteomic Data

Statistical modeling methods have also been used to characterize and analyze the cardiac proteome. Differential pro-
teomic expression has significant diagnostic value in identifying patients with human heart failure.86,87 Early work by Kislinger et al combined statistical modeling methods with mass spectrometric characterization of the mouse proteome to classify the organ (brain, heart, kidney, liver, lung, placenta) and subcellular localization (cytosol, cell membrane, mitochondria, nucleus) of all detectable proteins in the mouse proteome.88 In a later study, the authors made a more comprehensive attempt to characterize the mouse cardiac proteome, classifying proteins by subcellular localization and relative abundance and validating many proteins by immunoblotting.89 Comparing this proteome with various cardiac transcriptomes, the authors showed nearly 50% of expressed proteins had a linear correlation between mRNA and protein expression (Pearson's correlation coefficient, r=0.915). Moreover, the majority of the “outlier” proteins with low mRNA/protein expression correlation (r=0.147) were involved in pathways regulating mitochondrial energy metabolism or ribosome assembly. From this body of work, the authors generated a reference profile of the wild-type mouse cardiac proteome.

These reference proteomes are useful for identifying biomarkers for cardiac disease.86,90 As an example, Gramolini et al focused on a specific heart failure model by comparing protein expression profiles from cardiomyopathic phospholamban mutant mice (PLN-R9C) against those from wild-type littermates.91 The authors identified changes in protein expression in signaling pathways related to Ca2+ signaling, ER stress, cytoskeletal remodeling, and apoptosis. These protein expression changes also included known biomarkers for heart failure (AT2A2, ANF, BNP, FABHP, and β-MHC). The authors validated these identified proteins against human cardiac PLN-R9C explants and found correlations in 27 of the 40 highest ranking candidates from the transgenic mouse tissue. Collectively, these studies illustrate how statistical learning methods can be used to simplify complex proteomic data sets to predict unique protein signatures corresponding to different cardiac phenotypes.

### Protein–Protein Interaction Networks

An alternative approach to analyzing large data sets is to use a protein–protein interaction (PPI) network to understand how changes in expression may correspond to changes in the regulation of specific signaling pathways and cardiac phenotypes. For example, Berger et al used a human PPI network to predict single-nucleotide polymorphisms and Food and Drug Administration (FDA)-approved drugs that may induce LQT syndrome (LQTS) and increase susceptibility for arrhythmias.92 The authors first curated a human PPI network and then used methods from machine learning to identify a LQTS subnetwork based on 13 genes corresponding to 12 different LQT phenotypes or reduced LQT susceptibility. The authors validated this subnetwork against genes, single-nucleotide polymorphisms, and drugs known to trigger LQTS and then used the LQTS subnetwork to predict FDA-approved drugs that were not classified as QT prolonging drugs but were associated with reports of QT prolongation. Using their PPI network, the authors hypothesized mechanisms linking the targets for these drugs to the LQT phenotype.

Using a similar approach, Lage et al recently examined the PPI subnetworks underlying cardiac morphogenesis in early development.93 The authors manually curated 255 cardiac development-related genes and computationally classified these genes into 19 functional PPI subnetworks. These 19 subnetworks were then manually annotated by their role in cardiac development, revealing recycling of functional subnetworks during heart development. The authors note increased anatomic complexity correlated with increased signaling complexity, marked by increases in PPI subnetwork activation, transcriptional activity, and protein expression. The authors experimentally validated these predictions in 19 human hearts at various stages of development and 14 embryonic human hearts, confirming the regional and temporal activation of these different subnetworks. These studies powerfully show how PPI networks can be used to give mechanistic information on proteins whose expression or activity may altered in human cardiac disease and development. More generally, these studies illustrate how statistical modeling approaches can clarify interpretation of complex data sets to gain insight into how specific signaling networks may regulate organ-level phenotypes.

#### Summary

Statistical modeling techniques are useful for reducing the dimensionality of complex data sets and identifying key changes in a disease or transgenic cardiac phenotype. These approaches can identify groups of signaling proteins that are correlated with specific phenotypes or have correlated activity. These groups can be useful for identifying biomarkers for cardiac disease or generating new experimental leads for the regulation of heart failure progression. PPI networks can help facilitate mechanistic understanding of large data sets by identifying how signaling proteins may be connected to each other. Together, these approaches can draw attention to nonobvious relationships between different parts of a signaling network and bring focus to the most important players in a complex phenotype.

#### Future Directions

Although the cardiovascular system has a rich history of using computational models to study its cellular physiology, the use of computational models to study cardiac signaling networks is still young. To accompany our growing appreciation of cell signaling complexity, there is a great need for new statistical and mechanistic modeling approaches that are scalable to larger signaling networks. At the same time, there are many areas of cardiac signaling that have not yet benefitted from computational modeling.

#### New Statistical Approaches to Characterizing Cardiac Signaling Networks

Next-generation sequencing technologies are now rapidly generating a wealth of data, providing comprehensive profiles for cardiac gene expression.94 In recent years, researchers studying other systems have developed powerful new statistical modeling techniques to deal with these growing data sets.95,96 These techniques aim to reduce the dimensionality of large-scale data sets into a more limited number of
principal components that may be more directly associated with a specific phenotype or cell behavior. For example, Janes et al examined the signaling network regulating cytokine-induced apoptosis in HT-29 cells, obtaining 7980 measurements of protein activation.⁹⁷ The authors used principal components analysis to identify groups of signaling proteins that correspond to stress-apoptosis or cell-survival behaviors. Performing regression analysis on these principal components, the authors generated a model capable of predicting apoptosis responses to TNF, epidermal growth factor, and insulin treatment, which they validated experimentally. The authors also identified certain situations that caused the model to fail in predicting experimental outcomes. By analyzing and reconciling these context-specific “model break points,” the authors identified a number of new mechanisms regulating TNF-induced apoptosis, including an unexpected role for transforming growth factor α in phosphatidylinositol 3-kinase/Akt signaling and a counterintuitive loss of extracellular signal-regulated kinase-mediated survival under interleukin-1α blockade. The authors also evaluated more general properties of cell signaling networks, demonstrating how the overall signaling network is more sensitive to the dynamic range of signaling species than the absolute strength of their signaling activation. This study exemplifies how new statistical modeling techniques can be used to help identify mechanisms for signaling network regulation. As the size of cardiac-specific genomic and proteomic data grows, similar techniques will be important for reducing the complexity of these data sets and for investigating the relationships between signaling species.

### Large-Scale Mechanistic Modeling of Cardiac Signaling Networks

The biggest challenge implementing biochemically mechanistic models is the requirement for appropriate biochemical parameters to constrain all reactions in a signaling network. Because these parameters can be difficult to estimate, detailed kinetic models of signaling pathways are typically limited to 10 to 20 protein species. However, the signaling networks for regulating some cardiac behaviors (e.g., cardiac hypertrophy) involve significantly more signaling molecules with considerably more complexity in network connectivity.⁸ Thus, the field needs to identify modeling approaches that can “do more with less.” One approach for analyzing a signaling network using its topology alone is to use Boolean or Bayesian analysis, as has been used to study signaling associated with inflammation.⁹⁸,⁹⁹ However, these approaches provide only qualitative steady-state information about a system and have difficulty with common network motifs such as feedback loops. Recent efforts have attempted to bridge the gap between network topology and signaling dynamics without requiring the full set of biochemical parameters.¹⁰⁰,¹⁰¹ As more information is known about the diverse signaling pathways that regulate complicated processes such as apoptosis, hypertrophy, and metabolism, new progress must be made in the development of computational tools that can integrate these pathways into a consistent framework and make predictions about how they crosstalk to regulate cardiac behaviors.

---

Opportunities in Cardiac Cell–Based Therapies

Cell-based therapies for cardiac diseases are an exciting new research area.¹⁰²,¹⁰³ However, the complexity of signaling pathways that regulate differentiation of cardiac progenitor cells into mature adult cardiac myocytes is a significant obstacle toward forward progress in translating these therapies to the clinic. Computational models can be useful in this context for understanding the relationship between the local environment and differentiated state of cardiac progenitor cells.¹⁰⁴ In an elegant study combining computational and experimental work, Kirouac et al showed how intercell paracrine signaling regulated the differentiation of hematopoietic stem and progenitor cells into cells that express blood lineage-associated cell surface antigens.¹⁰⁵ The authors developed a computational model and estimated parameters based on their experiments of stem and progenitor cell differentiation under different selection and media-exchange conditions. Using a sensitivity analysis, the authors determined that differentiation of these cells was primarily regulated by secreted inhibitory factors, comprising a paracrine negative feedback loop. Using their model, the authors showed how experimental heterogeneity in long-term cell cultures could be explained by stochastic variations in the secretion rates of inhibitory factors. Moreover, they showed how loss of responsiveness to these secreted inhibitory factors was sufficient to explain pathological transformation of progenitor cells into leukemic stem cells (in vitro data published by Warner et al¹⁰⁶). This study illustrates how mechanistic signaling models are currently being used to understand and guide experimental differentiation of stem and progenitor cells into desired phenotypes. As cell-based therapies become more attractive treatment options for cardiac diseases, computational models can help accelerate mechanistic understanding of the differentiation processes for cardiac progenitor cells.

---

Opportunities in Mechanotransduction

Cardiac biomechanics play a central role in shaping cardiac development and pathophysiology.¹⁰⁷,¹⁰⁸ However, the signaling pathways converting ventricular stresses and strains to signaling cues for cardiac remodeling remain poorly understood. Although it is clear that focal adhesion and integrin-mediated signaling pathways are important for regulating cardiac growth, contractility, and repair,¹⁰⁹⁻¹¹¹ these pathways have received significantly less attention than those stimulated by circulating factors. Moreover, mechanical stretch alone is sufficient for inducing hypertrophy,¹¹²⁻¹¹⁴ arrhythmia,¹¹⁵⁻¹¹⁷ and changes to G-protein signaling,¹¹⁸,¹¹⁹ and mechanical unloading of failing hearts can reverse cardiac hypertrophy.¹²⁰,¹²¹ Computational models are already being used to explore the role of mechanical stretch in regulation myocyte electrophysiology,¹²²⁻¹²⁴ electromechanics,⁷³,¹²⁵ and ventricular arrhythmogenesis.¹²⁶,¹²⁷ As cardiac mechanotransduction signaling pathways are better understood, computational models will be important for mechanistically understanding how mechanotransduction interfaces with other signaling pathways to control cardiac contractility and remodeling. Understanding these relationships will be an important step toward reconciling the strengths and weaknesses of current therapeutic strategies.
nesses of the neurohormonal and biomechanical hypotheses for human heart failure^{1–3} and may help generate new leads for better therapeutic treatment options.

# Conclusions

Computational models are important research tools that can complement experimental studies to reduce the complexity of cardiac signaling networks. Modeling approaches can accelerate mechanistic insight into how signaling networks are regulated and help extrapolate the consequences of these signaling pathways on cardiac physiology. To date, computational models have contributed significantly toward understanding β-adrenergic and CaMKII signaling. There are tremendous opportunities for these approaches to be extended to both well-studied signaling pathways (eg, α-adrenergic signaling, MAPKs) and emerging signaling-related fields (eg, cardiac stem cells, mechanotransduction). Models can help integrate different aspects of cardiac function into coherent frameworks and help understand the actions of cardiac signaling networks on both homeostatic maintenance of cardiac physiology and pathological progression into heart failure. Modeling studies can also complement experimental studies to both provide mechanistic understanding and generate new experimental leads. These approaches powerfully reduce the complexity of large data sets and bring focus to the most important signaling species or signaling mechanisms regulate cardiac behaviors. As the appreciation for cardiac signaling network complexity and the size/quantity of experimental data sets grow, computational models are becoming necessary for addressing these challenges in a quantitative, mechanistic and methodical manner. Such efforts will prove increasingly important for elucidating mechanisms underlying the neurohormonal hypothesis and understanding the pathogenesis of heart failure.

# Sources of Funding

This work was supported by the NIH (grant HL094476 to J.S. and Biotechnology Training Grant GM08715 to J.Y.) and American Heart Association Grant 0830470N (to J.S.).

# Disclosures

None.

# References

1. Packer M. The neurohormonal hypothesis: a theory to explain the mechanism of disease progression in heart failure. *J Am Coll Cardiol*. 1992;20:248–254.
2. Mann DL, Bristow MR. Mechanisms and models in heart failure: the biomechanical model and beyond. *Circulation*. 2005;111:2837–2849.
3. Katz AM. The “modern” view of heart failure: how did we get here? *Circ Heart Fail*. 2008;1:63–71.
4. Overington JP, Al-Lazikani B, Hopkins AL. How many drug targets are there? *Nat Rev Drug Discov*. 2006;5:993–996.
5. Olson EN. Gene regulatory networks in the evolution and development of the heart. *Science*. 2006;313:1922–1927.
6. Wang Y. Mitogen-activated protein kinases in heart development and diseases. *Circulation*. 2007;116:1413–1423.
7. Rockman HA, Koch WJ, Lefkowitz RJ. Seven-transmembrane-spanning receptors and heart function. *Nature*. 2002;415:206–212.
8. Heineke J, Molkentin JD. Regulation of cardiac hypertrophy by intracellular signalling pathways. *Nat Rev Mol Cell Biol*. 2006;7:589–600.
9. Zaccolo M, Movsesian MA. cAMP and cGMP signaling cross-talk: role of phosphodiesterases and implications for cardiac pathophysiology. *Circ Res*. 2007;100:1569–1578.
10. Yan C, Miller CL, Abe J. Regulation of phosphodiesterase 3 and inducible cAMP early repressor in the heart. *Circ Res*. 2007;100:489–501.
11. Steinberg SF, Brunton LL. Compartmentation of G protein-coupled signaling pathways in cardiac myocytes. *Annu Rev Pharmacol Toxicol*. 2001;41:751–773.
12. Brown HF, Kimura J, Noble D, Noble SJ, Taupignon A. The ionic currents underlying pacemaker activity in rabbit sino-atrial node: experimental results and computer simulations. *Proc R Soc Lond B Biol Sci*. 1984;222:329–347.
13. DiFrancesco D, Noble D. A model of cardiac electrical activity incorporating ionic pumps and concentration changes. *Philos Trans R Soc Lond B Biol Sci*. 1985;307:353–398.
14. Luo CH, Rudy Y. A model of the ventricular cardiac action potential. Depolarization, repolarization, and their interaction. *Circ Res*. 1991;68:1501–1526.
15. Feit TS, Bass BG. Numerical study of the behavior of an activation parameter in a sliding filament cat papillary muscle model. *J Mechanochem Cell Motil*. 1977;4:275–302.
16. Smith JM, Cohen RJ. Simple finite-element model accounts for wide range of cardiac dysrhythmias. *Proc Natl Acad Sci USA*. 1984;81:233–237.
17. Noble D. Modeling the heart—from genes to cells to the whole organ. *Science*. 2002;295:1678–1682.
18. Hunter P, Robbins P, Noble D. The IUPS human Physiome Project. *Pflugers Arch*. 2002;445:1–9.
19. Hunter PJ, Nielsen PM, Bullivant D. The IUPS Physiome Project. International Union of Physiological Sciences. *Novartis Found Symp*. 2002;247:207–217.
20. Aldridge BB, Burke JM, Lauffenburger DA, Sorger PK. Physicochemical modelling of cell signalling pathways. *Nat Cell Biol*. 2006;8:1195–1203.
21. Bhalla US, Iyengar R. Emergent properties of networks of biological signaling pathways. *Science*. 1999;283:381–387.
22. Alon U. Network motifs: theory and experimental approaches. *Nat Rev Genet*. 2007;8:450–461.
23. Milo R, Shen-Orr S, Itzkovitz S, Kashtan N, Chklovskii D, Alon U. Network motifs: simple building blocks of complex networks. *Science*. 2002;298:824–827.
24. Bers DM. Cardiac excitation-contraction coupling. *Nature*. 2002;415:198–205.
25. Saucerman JJ, McCulloch AD. Cardiac beta-adrenergic signaling: from subcellular microdomains to heart failure. *Ann NY Acad Sci*. 2006;1080:348–361.
26. Lohse MJ, Engelhardt S, Eschenhagen T. What is the role of beta-adrenergic signaling in heart failure? *Circ Res*. 2003;93:896–906.
27. Engelhardt S, Hein L, Wiesmann F, Lohse MJ. Progressive hypertrophy and heart failure in betal-adrenergic receptor transgenic mice. *Proc Natl Acad Sci USA*. 1999;96:7059–7064.
28. Bisognano JD, Weinberger HD, Bohlmeier TJ, Pende A, Reynolds MV, Sastravaha A, Roden R, Asano K, Blaxall BC, Wu SC, Communal C, Singh K, Colucci W, Bristow MR, Port DJ. Myocardial-directed overexpression of the human beta(1)-adrenergic receptor in transgenic mice. *J Mol Cell Cardiol*. 2000;32:817–830.
29. Wang X, Dhalla NS. Modification of beta-adrenoceptor signal transduction pathway by genetic manipulation and heart failure. *Mol Cell Biochem*. 2000;214:131–155.
30. DiPaola NR, Sweet WE, Stull LB, Francis GS, Schomisch Moravec C. Beta-adrenergic receptors and calcium cycling proteins in non-failing, hypertrophied and failing human hearts: transition from hypertrophy to failure. *J Mol Cell Cardiol*. 2001;33:1283–1295.
31. McMurray JJ. Clinical practice. Systolic heart failure. *N Engl J Med*. 2010;362:228–238.
32. Dickstein K, Cohen-Solal A, Filippatos G, McMurray JJ, Ponikowski P, Poole-Wilson PA, Stromberg A, van Veldhuizen DJ, Atar D, Hoes AW, Keren A, Mebazaa A, Nieminen M, Priori SG, Swedberg K, Vahanian A, Camm J, De Caterina R, Dean V, Funck-Brentano C, Hellemans I, Kristensen SD, McGregor K, Sechtem U, Silber S, Tendera M, Widimsky P, Zamorano JL, Auricchio A, Bax J, Bohm M, Corra U, della Bella P, Elliott PM, Follath F, Gheorghiade M, Hasin Y, Hernborg A, Jaarsma T, Komajda M, Kornowski R, Piepoli M, Prendergast B, Tavazzi L, Vachiery JL, Verheugt FW, Zannad F. ESC guidelines for

the diagnosis and treatment of acute and chronic heart failure 2008: the Task Force for the Diagnosis and Treatment of Acute and Chronic Heart Failure 2008 of the European Society of Cardiology. Developed in collaboration with the Heart Failure Association of the ESC (HFA) and endorsed by the European Society of Intensive Care Medicine (ESICM). *Eur J Heart Fail.* 2008;10:933–989.

33. Hunt SA, Abraham WT, Chin MH, Feldman AM, Francis GS, Ganiats TG, Jessup M, Konstam MA, Mancini DM, Michl K, Oates JA, Rahko PS, Silver MA, Stevenson LW, Yancy CW. 2009 focused update incorporated into the ACC/AHA 2005 guidelines for the diagnosis and management of heart failure in adults: a report of the American College of Cardiology Foundation/American Heart Association Task Force on Practice Guidelines: developed in collaboration with the International Society for Heart and Lung Transplantation. *Circulation.* 2009;119: e391–e479.

34. Klapholz M. Beta-blocker use for the stages of heart failure. *Mayo Clin Proc.* 2009;84:718–729.

35. Saucerman JJ, Brunton LL, Michailova AP, McCulloch AD. Modeling beta-adrenergic control of cardiac myocyte contractility in silico. *J Biol Chem.* 2003;278:47997–48003.

36. Saucerman JJ, McCulloch AD. Mechanistic systems models of cell signaling networks: a case study of myocyte adrenergic regulation. *Prog Biophys Mol Biol.* 2004;85:261–278.

37. Marx SO, Reiken S, Hisamatsu Y, Jayaraman T, Burkhoff D, Rosemblit N, Marks AR. PKA phosphorylation dissociates FKBP12.6 from the calcium release channel (ryanodine receptor): defective regulation in failing hearts. *Cell.* 2000;101:365–376.

38. Eisner DA, Choi HS, Diaz ME, O’Neill SC, Trafford AW. Integrative analysis of calcium cycling in cardiac muscle. *Circ Res.* 2000;87: 1087–1094.

39. Greenstein JL, Tanskanen AJ, Winslow RL. Modeling the actions of beta-adrenergic signaling on excitation–contraction coupling processes. *Ann N Y Acad Sci.* 2004;1015:16–27.

40. Kuzumoto M, Takeuchi A, Nakai H, Oka C, Noma A, Matsuoka S. Simulation analysis of intracellular Na+ and Cl- homeostasis during beta 1-adrenergic stimulation of cardiac myocyte. *Prog Biophys Mol Biol.* 2008;96:171–186.

41. Despa S, Tucker AL, Bers DM. Phospholemman-mediated activation of na/k-atpase limits [Na]i and inotropic state during beta-adrenergic stimulation in mouse ventricular myocytes. *Circulation.* 2008;117: 1849–1855.

42. Himeno Y, Sarai N, Matsuoka S, Noma A. Ionic mechanisms underlying the positive chronotropy induced by betal-adrenergic stimulation in guinea pig sinoatrial node cells: a simulation study. *J Physiol Sci.* 2008;58:53–65.

43. Saucerman JJ, Zhang J, Martin JC, Peng LX, Stenbit AE, Tsien RY, McCulloch AD. Systems analysis of PKA-mediated phosphorylation gradients in live cardiac myocytes. *Proc Natl Acad Sci USA.* 2006; 103:12923–12928.

44. Kholodenko BN. Cell-signalling dynamics in time and space. *Nat Rev Mol Cell Biol.* 2006;7:165–176.

45. Kholodenko BN, Hancock JF, Kolch W. Signalling ballet in space and time. *Nat Rev Mol Cell Biol.* 2010;11:414–426.

46. Iancu RV, Jones SW, Harvey RD. Compartmentation of cAMP signaling in cardiac myocytes: a computational study. *Biophys J.* 2007;92: 3317–3331.

47. Iancu RV, Ramamurthy G, Warrier S, Nikolaev VO, Lohse MJ, Jones SW, Harvey RD. Cytoplasmic cAMP concentrations in intact cardiac myocytes. *Am J Physiol Cell Physiol.* 2008;295:C414–C422.

48. Couchonnel LF, Anderson ME. The role of calmodulin kinase ii in myocardial physiology and disease. *Physiology (Bethesda).* 2008;23: 151–159.

49. Bers DM, Grandi E. Calcium/calmodulin-dependent kinase II regulation of cardiac ion channels. *J Cardiovasc Pharmacol.* 2009;54:180–187.

50. Maier LS, Bers DM. Role of Ca2+/calmodulin-dependent protein kinase (CAMK) in excitation-contraction coupling in the heart. *Cardiovasc Res.* 2007;73:631–640.

51. Zhang T, Brown JH. Role of Ca2+/calmodulin-dependent protein kinase II in cardiac hypertrophy and heart failure. *Cardiovasc Res.* 2004;63:476–486.

52. Mishra S, Ling H, Grimm M, Zhang T, Bers DM, Brown JH. Cardiac hypertrophy and heart failure development through Gq and CaM kinase II signaling. *J Cardiovasc Pharmacol.* 2010. doi: 10.1097/ FJC.0b013e3181e1d263.

53. Grimm M, Brown JH. Beta-adrenergic receptor signaling in the heart: role of CAMKII. *J Mol Cell Cardiol.* 2010;48:322–330.

54. Zhang R, Khoo MS, Wu Y, Yang Y, Grueter CE, Ni G, Price EE, Jr., Thiel W, Guatimosim S, Song LS, Madu EC, Shah AN, Vishnivetskaya TA, Atkinson JB, Gurevich VV, Salama G, Lederer WJ, Colbran RJ, Anderson ME. Calmodulin kinase II inhibition protects against structural heart disease. *Nat Med.* 2005;11:409–417.

55. Anderson ME. Calmodulin kinase signaling in heart: an intriguing candidate target for therapy of myocardial dysfunction and arrhythmias. *Pharmacol Ther.* 2005;106:39–55.

56. Hund TJ, Rudy Y. Rate dependence and regulation of action potential and calcium transient in a canine cardiac ventricular cell model. *Circulation.* 2004;110:3168–3174.

57. Grandi E, Puglisi JL, Wagner S, Maier LS, Severi S, Bers DM. Simulation of Ca-calmodulin-dependent protein kinase II on rabbit ventricular myocyte ion currents and action potentials. *Biophys J.* 2007;93: 3835–3847.

58. Hashambhoy YL, Winslow RL, Greenstein JL. CAMKII-induced shift in modal gating explains L-type Ca(2+) current facilitation: a modeling study. *Biophys J.* 2009;96:1770–1785.

59. Hashambhoy YL, Greenstein JL, Winslow RL. Role of CAMKII in RyR leak, EC coupling and action potential duration: a computational model. *J Mol Cell Cardiol.* 2010;49:617–624.

60. Soltis AR, Saucerman JJ. Synergy between CAMKII substrates and beta-adrenergic signaling in regulation of cardiac myocyte Ca(2+) handling. *Biophys J.* 2010;99:2038–2047.

61. Hund TJ, Decker KF, Kanter E, Mohler PJ, Boyden PA, Schuessler RB, Yamada KA, Rudy Y. Role of activated CAMKII in abnormal calcium homeostasis and i(Na) remodeling after myocardial infarction: insights from mathematical modeling. *J Mol Cell Cardiol.* 2008;45:420–428.

62. Christensen MD, Dun W, Boyden PA, Anderson ME, Mohler PJ, Hund TJ. Oxidized calmodulin kinase II regulates conduction following myocardial infarction: a computational analysis. *PLoS Comput Biol.* 2009; 5:e1000583.

63. Song Q, Saucerman JJ, Bossuyt J, Bers DM. Differential integration of Ca2+-calmodulin signal in intact ventricular myocytes at low and high affinity Ca2+-calmodulin targets. *J Biol Chem.* 2008;283: 31531–31540.

64. Saucerman JJ, Bers DM. Calmodulin mediates differential sensitivity of CAMKII and calcineurin to local Ca2+ in cardiac myocytes. *Biophys J.* 2008;95:4597–4612.

65. Chiba H, Schneider NS, Matsuoka S, Noma A. A simulation study on the activation of cardiac CAMKII delta-isoform and its regulation by phosphatases. *Biophys J.* 2008;95:2139–2149.

66. Peterson BZ, DeMaria CD, Adelman JP, Yue DT. Calmodulin is the Ca2+ sensor for Ca2+-dependent inactivation of L-type calcium channels. *Neuron.* 1999;22:549–558.

67. Peterson BZ, Lee JS, Mulle JG, Wang Y, de Leon M, Yue DT. Critical determinants of Ca(2+)-dependent inactivation within an EF-hand motif of L-type Ca(2+) channels. *Biophys J.* 2000;78:1906–1920.

68. Tanskanen AJ, Greenstein JL, Chen A, Sun SX, Winslow RL. Protein geometry and placement in the cardiac dyad influence macroscopic properties of calcium-induced calcium release. *Biophys J.* 2007;92: 3379–3396.

69. Tadross MR, Dick IE, Yue DT. Mechanism of local and global Ca2+ sensing by calmodulin in complex with a Ca2+ channel. *Cell.* 2008; 133:1228–1240.

70. Cooling M, Hunter P, Crampin EJ. Modeling hypertrophic IP3 transients in the cardiac myocyte. *Biophys J.* 2007;93:3421–3433.

71. Shin SY, Choo SM, Kim D, Baek SJ, Wolkenhauer O, Cho KH. Switching feedback mechanisms realize the dual role of MCIP in the regulation of calcineurin activity. *FEBS Lett.* 2006;580:5965–5973.

72. Cooling MT, Hunter P, Crampin EJ. Sensitivity of NFAT cycling to cytosolic calcium concentration: implications for hypertrophic signals in cardiac myocytes. *Biophys J.* 2009;96:2095–2104.

73. Niederer SA, Smith NP. A mathematical model of the slow force response to stretch in rat ventricular myocytes. *Biophys J.* 2007;92: 4030–4044.

74. Cortassa S, Aon MA, O’Rourke B, Jacques R, Tseng HJ, Marban E, Winslow RL. A computational model integrating electrophysiology, contraction, and mitochondrial bioenergetics in the ventricular myocyte. *Biophys J.* 2006;91:1564–1589.

75. Southern J, Pitt-Francis J, Whiteley J, Stokeley D, Kobashi H, Nobes R, Kadooka Y, Gavaghan D. Multi-scale computational modelling in biology and physiology. *Prog Biophys Mol Biol*. 2008;96:60–89.

76. Ten Tusscher KH, Hren R, Panfilov AV. Organization of ventricular fibrillation in the human heart. *Circ Res*. 2007;100:e87–e101.

77. Shim EB, Leem CH, Abe Y, Noma A. A new multi-scale simulation model of the circulation: from cells to system. *Philos Transact A Math Phys Eng Sci*. 2006;364:1483–1500.

78. Kerckhoffs RC, Neal ML, Gu Q, Bassingthwaighte JB, Omens JH, McCulloch AD. Coupling of a 3D finite element model of cardiac ventricular mechanics to lumped systems models of the systemic and pulmonic circulation. *Ann Biomed Eng*. 2007;35:1–18.

79. Saucerman JJ, Healy SN, Belik ME, Puglisi JL, McCulloch AD. Proarrhythmic consequences of a KCNQ1 AKAP-binding domain mutation: computational models of whole cells and heterogeneous tissue. *Circ Res*. 2004;95:1216–1224.

80. Nakamura H, Kurokawa J, Bai CX, Asada K, Xu J, Oren RV, Zhu ZI, Clancy CE, Isobe M, Furukawa T. Progesterone regulates cardiac repolarization through a nongenomic pathway: an in vitro patch-clamp and computational modeling study. *Circulation*. 2007;116:2913–2922.

81. Hall JL, Grindle S, Han X, Fermín D, Park S, Chen Y, Bache RJ, Mariash A, Guan Z, Ormaza S, Thompson J, Graziano J, de Sam Lazaro SE, Pan S, Simari RD, Miller LW. Genomic profiling of the human heart before and after mechanical support with a ventricular assist device reveals alterations in vascular signaling networks. *Physiol Genomics*. 2004;17:283–291.

82. Hall JL, Birks EJ, Grindle S, Cullen ME, Barton PJ, Rider JE, Lee S, Harwalker S, Mariash A, Adhikari N, Charles NJ, Felkin LE, Polster S, George RS, Miller LW, Yacoub MH. Molecular signature of recovery following combination left ventricular assist device (LVAD) support and pharmacologic therapy. *Eur Heart J*. 2007;28:613–627.

83. Hong SE, Park I, Cha H, Rho SH, Park WJ, Cho C, Kim do H. Identification of mouse heart transcriptomic network sensitive to various heart diseases. *Biotechnol J*. 2008;3:648–658.

84. Gao Z, Xu H, DiSilvestre D, Halperin VL, Tunin R, Tian Y, Yu W, Winslow RL, Tomaselli GF. Transcriptomic profiling of the canine tachycardia-induced heart failure model: global comparison to human and murine heart failure. *J Mol Cell Cardiol*. 2006;40:76–86.

85. Gao Z, Barth AS, DiSilvestre D, Akar FG, Tian Y, Tanskanen A, Kass DA, Winslow RL, Tomaselli GF. Key pathways associated with heart failure development revealed by gene networks correlated with cardiac remodeling. *Physiol Genomics*. 2008;35:222–230.

86. Van Eyk JE. Proteomics: unraveling the complexity of heart disease and striving to change cardiology. *Curr Opin Mol Ther*. 2001;3:546–553.

87. Edwards AV, White MY, Cordwell SJ. The role of proteomics in clinical cardiovascular biomarker discovery. *Mol Cell Proteomics*. 2008;7:1824–1837.

88. Kislinger T, Cox B, Kannan A, Chung C, Hu P, Ignatchenko A, Scott MS, Gramolini AO, Morris Q, Hallett MT, Rossant J, Hughes TR, Frey B, Emili A. Global survey of organ and organelle protein expression in mouse: combined proteomic and transcriptomic profiling. *Cell*. 2006;125:173–186.

89. Boussette N, Kislinger T, Fong V, Isserlin R, Hewel JA, Emil A, Gramolini AO. Large-scale characterization and analysis of the murine cardiac proteome. *J Proteome Res*. 2009;8:1887–1901.

90. Arab S, Gramolini AO, Ping P, Kislinger T, Stanley B, van Eyk J, Ouzounian M, MacLennan DH, Emili A, Liu PP. Cardiovascular proteomics: tools to develop novel biomarkers and potential applications. *J Am Coll Cardiol*. 2006;48:1733–1741.

91. Gramolini AO, Kislinger T, Alikhani-Koopaei R, Fong V, Thompson NJ, Isserlin R, Sharma P, Oudit GY, Trivieri MG, Fagan A, Kannan A, Higgins DG, Huedig H, Hess G, Arab S, Seidman JG, Seidman CE, Frey B, Perry M, Backx PH, Liu PP, MacLennan DH, Emili A. Comparative proteomics profiling of a phospholamban mutant mouse model of dilated cardiomyopathy reveals progressive intracellular stress responses. *Mol Cell Proteomics*. 2008;7:519–533.

92. Berger SI, Ma’ayan A, Iyengar R. Systems pharmacology of arrhythmias. *Sci Signal*. 2010;3:ra30.

93. Lage K, Mollgaard K, Greenway S, Wakimoto H, Gorham JM, Workman CT, Bendtsen E, Hansen NT, Rigina O, Roque FS, Wiese C, Christoffels VM, Roberts AE, Smoot LB, Pu WT, Donahoe PK, Tommerup N, Brunak S, Seidman CE, Seidman JG, Larsen LA. Dissecting spatio-temporal protein networks driving human heart development and related disorders. *Mol Syst Biol*. 2010;6:381.

94. Matkovich SJ, Zhang Y, Van Booven DJ, Dorn GW II. Deep mRNA sequencing for in vivo functional analysis of cardiac transcriptional regulators: application to Galphaq. *Circ Res*. 2010;106:1459–1467.

95. Janes KA, Yaffe MB. Data-driven modelling of signal-transduction networks. *Nat Rev Mol Cell Biol*. 2006;7:820–828.

96. Li L. Dimension reduction for high-dimensional data. *Methods Mol Biol*. 2010;620:417–434.

97. Janes KA, Reinhardt HC, Yaffe MB. Cytokine-induced signaling networks prioritize dynamic range over signal strength. *Cell*. 2008;135:343–354.

98. Morris MK, Saez-Rodriguez J, Sorger PK, Lauffenburger DA. Logic-based models for the analysis of cell signaling networks. *Biochemistry*. 2010;49:3216–3224.

99. Albert I, Thakar J, Li S, Zhang R, Albert R. Boolean network simulations for life scientists. *Source Code Biol Med*. 2008;3:16.

100. Albert R, Wang RS. Discrete dynamic modeling of cellular signaling networks. *Methods Enzymol*. 2009;467:281–306.

101. Wittmann DM, Krumiek J, Saez-Rodriguez J, Lauffenburger DA, Klamt S, Theis FJ. Transforming Boolean models to continuous models: methodology and application to t-cell receptor signaling. *BMC Syst Biol*. 2009;3:98.

102. Hansson EM, Lindsay ME, Chien KR. Regeneration next: toward heart stem cell therapeutics. *Cell Stem Cell*. 2009;5:364–377.

103. Chavakis E, Koyanagi M, Dimmeler S. Enhancing the outcome of cell therapy for cardiac repair: progress from bench to bedside and back. *Circulation*. 2010;121:325–335.

104. Viswanathan S, Zandstra PW. Towards predictive models of stem cell fate. *Cytotechnology*. 2003;41:75–92.

105. Kirouac DC, Madlambayan GJ, Yu M, Sykes EA, Ito C, Zandstra PW. Cell-cell interaction networks regulate blood stem and progenitor cell fate. *Mol Syst Biol*. 2009;5:293.

106. Warner JK, Wang JC, Takenaka K, Doulatov S, McKenzie JL, Harrington L, Dick JE. Direct evidence for cooperating genetic events in the leukemic transformation of normal human hematopoietic cells. *Leukemia*. 2005;19:1794–1805.

107. Jacot JG, Kita-Matsuo H, Wei KA, Chen HS, Omens JH, Mercola M, McCulloch AD. Cardiac myocyte force development during differentiation and maturation. *Ann N Y Acad Sci*. 2010;1188:121–127.

108. Mann DL. Left ventricular size and shape: determinants of mechanical signal transduction pathways. *Heart Fail Rev*. 2005;10:95–100.

109. Hannigan GE, Coles JG, Dedhar S. Integrin-linked kinase at the heart of cardiac contractility, repair, and disease. *Circ Res*. 2007;100:1408–1414.

110. Samarel AM. Costameres, focal adhesions, and cardiomyocyte mechanotransduction. *Am J Physiol Heart Circ Physiol*. 2005;289:H2291–H2301.

111. Marin TM, Clemente CF, Santos AM, Picardi PK, Pascoal VD, Lopes-Cendes I, Saad MJ, Franchini KG. Shp2 negatively regulates growth in cardiomyocytes by controlling focal adhesion kinase/Src and mTOR pathways. *Circ Res*. 2008;103:813–824.

112. Sussman MA, McCulloch A, Borg TK. Dance band on the titanic: biomechanical signaling in cardiac hypertrophy. *Circ Res*. 2002;91:888–898.

113. Blaauw E, van Nieuwenhoven FA, Willemse P, Delhaas T, Prinzen FW, Snoeckx LH, van Bilsen M, Van der Vusse GJ. Stretch-induced hypertrophy of isolated adult rabbit cardiomyocytes. *Am J Physiol Heart Circ Physiol*. 2010;299:H780–H787.

114. Zobel C, Rana OR, Saygili E, Bolck B, Diedrichs H, Reuter H, Frank K, Muller-Ehmsen J, Pfitzer G, Schwinger RH. Mechanisms of Ca2+-dependent calcineurin activation in mechanical stretch-induced hypertrophy. *Cardiology*. 2007;107:281–290.

115. Yamazaki M, Vaquero LM, Hou L, Campbell K, Zlochiver S, Klos M, Mironov S, Berenfeld O, Honjo H, Kodama I, Jalife J, Kalifa J. Mechanisms of stretch-induced atrial fibrillation in the presence and the absence of adrenergic stimulation: interplay between rotors and focal discharges. *Heart Rhythm*. 2009;6:1009–1017.

116. Chorro FJ, Trapero I, Such-Miquel L, Pelechano F, Mainar L, Canoves J, Tormos A, Alberola A, Hove-Madsen L, Cinca J, Such L. Pharmacological modifications of the stretch-induced effects on ventricular fibrillation in perfused rabbit hearts. *Am J Physiol Heart Circ Physiol*. 2009;297:H1860–H1869.

117. Seo K, Inagaki M, Nishimura S, Hidaka I, Sugimachi M, Hisada T, Sugiura S. Structural heterogeneity in the ventricular wall plays a significant role in the initiation of stretch-induced arrhythmias in perfused

rabbit right ventricular tissues and whole heart preparations. *Circ Res*. 2010;106:176–184.

118. Rakesh K, Yoo B, Kim IM, Salazar N, Kim KS, Rockman HA. Beta-arrestin-biased agonism of the angiotensin receptor induced by mechanical stress. *Sci Signal*. 2010;3:ra46.

119. Malhotra R, D'Souza KM, Staron ML, Birukov KG, Bodi I, Akhter SA. G alpha(q)-mediated activation of GRK2 by mechanical stretch in cardiac myocytes: the role of protein kinase C. *J Biol Chem*. 2010;285: 13748–13760.

120. Burkhoff D, Klotz S, Mancini DM. Lvad-induced reverse remodeling: basic and clinical implications for myocardial recovery. *J Card Fail*. 2006;12:227–239.

121. Klotz S, Jan Danser AH, Burkhoff D. Impact of left ventricular assist device (LVAD) support on the cardiac reverse remodeling process. *Prog Biophys Mol Biol*. 2008;97:479–496.

122. Healy SN, McCulloch AD. An ionic model of stretch-activated and stretch-modulated currents in rabbit ventricular myocytes. *Europace*. 2005;7 Suppl 2:128–134.

123. Youm JB, Han J, Kim N, Zhang YH, Kim E, Joo H, Hun Leem C, Joon Kim S, Cha KA, Earm YE. Role of stretch-activated channels on the stretch-induced changes of rat atrial myocytes. *Prog Biophys Mol Biol*. 2006;90:186–206.

124. Wang Y, Joyner RW, Wagner MB, Cheng J, Lai D, Crawford BH. Stretch-activated channel activation promotes early afterdepolarizations in rat ventricular myocytes under oxidative stress. *Am J Physiol Heart Circ Physiol*. 2009;296:H1227–H1235.

125. Kuijpers NH, ten Eikelder HM, Bovendeerd PH, Verheule S, Arts T, Hilbers PA. Mechanoelectric feedback leads to conduction slowing and block in acutely dilated atria: a modeling study of cardiac electromechanics. *Am J Physiol Heart Circ Physiol*. 2007;292: H2832–H2853.

126. Trayanova NA, Constantino J, Gurev V. Models of stretch-activated ventricular arrhythmias. *J Electrocardiol*. 2010;43:479–485.

127. Keldermann RH, Nash MP, Gelderblom H, Wang VY, Panfilov AV. Electromechanical wavebreak in a model of the human left ventricle. *Am J Physiol Heart Circ Physiol*. 2010;299:H134–H143.

Circulation
Research
JOURNAL OF THE AMERICAN HEART ASSOCIATION

Computational Models Reduce Complexity and Accelerate Insight Into Cardiac Signaling Networks
Jason H. Yang and Jeffrey J. Saucerman

*Circ Res.* 2011;108:85-97
doi: 10.1161/CIRCRESAHA.110.223602

Circulation Research is published by the American Heart Association, 7272 Greenville Avenue, Dallas, TX 75231
Copyright © 2011 American Heart Association, Inc. All rights reserved.
Print ISSN: 0009-7330. Online ISSN: 1524-4571

The online version of this article, along with updated information and services, is located on the World Wide Web at:
http://circres.ahajournals.org/content/108/1/85

---

Permissions: Requests for permissions to reproduce figures, tables, or portions of articles originally published in *Circulation Research* can be obtained via RightsLink, a service of the Copyright Clearance Center, not the Editorial Office. Once the online version of the published article for which permission is being requested is located, click Request Permissions in the middle column of the Web page under Services. Further information about this process is available in the *Permissions and Rights Question and Answer* document.

Reprints: Information about reprints can be found online at:
http://www.lww.com/reprints

Subscriptions: Information about subscribing to *Circulation Research* is online at:
http://circres.ahajournals.org/subscriptions/

Downloaded from http://circres.ahajournals.org/ by guest on March 25, 2015
