
Genes to Cells

REVIEW

Molecular mechanisms of the Keap1–Nrf2 pathway  
in stress response and cancer evolution  

Keiko Taguchi${}^{1}$, Hozumi Motohashi${}^{2}$ and Masayuki Yamamoto${}^{1*}$  

${}^{1}$Department of Medical Biochemistry, Tohoku University Graduate School of Medicine, 2-1 Seiryo-cho, Aoba-ku,  
Sendai 980-8575, Japan  

${}^{2}$Center of Radioisotope Sciences, Tohoku University Graduate School of Medicine, 2-1 Seiryo-cho, Aoba-ku,  
Sendai 980-8575, Japan  

The Keap1–Nrf2 regulatory pathway plays a central role in the protection of cells against oxidative and xenobiotic damage. Under unstressed conditions, Nrf2 is constantly ubiquitinated by the Cul3–Keap1 ubiquitin E3 ligase complex and rapidly degraded in proteasomes. Upon exposure to electrophilic and oxidative stresses, reactive cysteine residues of Keap1 become modified, leading to a decline in the E3 ligase activity, stabilization of Nrf2 and robust induction of a battery of cytoprotective genes. Biochemical and structural analyses have revealed that the intact Keap1 homodimer forms a cherry-bob structure in which one molecule of Nrf2 associates with two molecules of Keap1 by using two binding sites within the Neh2 domain of Nrf2. This two-site binding appears critical for Nrf2 ubiquitination. In many human cancers, missense mutations in KEAP1 and NRF2 genes have been identified. These mutations disrupt the Keap1–Nrf2 complex activity involved in ubiquitination and degradation of Nrf2 and result in constitutive activation of Nrf2. Elevated expression of Nrf2 target genes confers advantages in terms of stress resistance and cell proliferation in normal and cancer cells. Discovery and development of selective Nrf2 inhibitors should make a critical contribution to improved cancer therapy.

Introduction: the Keap1–Nrf2 stress  
response pathway  

One of the important contributors to the regulation of transcription factor activity is the control of the abundance of transcription factors themselves. The amount of a protein at any one time is the result of a balance between synthesis and degradation. Protein synthesis consists of multiple processes, including gene transcription, mRNA translation and protein post-translational processing. Protein degradation is also carried out by multiple pathways, including the ubiquitin–proteasome, autophagy–lysosome and calpain pathways. Relatively small changes in protein degradation significantly affect the steady-state level of a protein. Dysfunction of protein degradation results in prompt accumulation of transcription factors that regulate rapid expression of target genes involved in cell survival. Protein degradation pathways appear to be an important regulatory node in stress response, as many transcription factors involved in stress response are reported to be regulated at the level of degradation processes.  

The transcription factor Nrf2 (NF-E2-related factor 2) is a potent transcriptional activator and plays a central role in inducible expression of many cytoprotective genes in response to oxidative and electrophilic stresses (Itoh *et al.* 1997; Motohashi & Yamamoto 2004) (Fig. 1). Target genes of Nrf2 are involved in the glutathione synthesis, elimination of reactive oxygen species (ROS), xenobiotic metabolism and drug transport (Okawa *et al.* 2006; Yates *et al.* 2009). Keap1 (Kelch-like ECH-associated protein 1) is essential for the regulation of activity of Nrf2. Under normal conditions, Nrf2 is constantly  

Communicated by: Mitsuhiro Yanagida  
*Correspondence*: masiyamamoto@med.tohoku.ac.jp  

DOI: 10.1111/j.1365-2443.2010.01473.x  
© 2011 The Authors  
Journal compilation © 2011 by the Molecular Biology Society of Japan/Blackwell Publishing Ltd.

K Taguchi et al.

HS-
HS- HS-
Keap1 Keap1
Ub
Nrf2

Unstressed
condition

Nrf2 degradation

Oxidative or
electrophilic stress

Cul3 dissociation

E-S-
E-S-
E-S-
S-E
S-E
S-E
Keap1 Keap1
Hinge and latch
mechanism
Nrf2

Nrf2 stabilization

Cytoplasm

Nuclear
accumulation

1. Glutathione synthesis (Gclc, Gclm)
2. Elimination of ROS (Txnrd1, Prdx1)
3. Detoxification of xenobiotics (Gst, Nqo1)
4. Drug transport (Mrp)

Small
Maf

TGAG/CTCAGC
ARE/EpRE

Target gene

Nucleus

Figure 1 The Keap1–Nrf2 pathway. The transcription factor Nrf2 plays a central role in inducible expression of many cytoprotective genes in response to oxidative and electrophilic stresses. Keap1 is a cytoplasmic protein essential for the regulation of Nrf2 activity. Under unstressed conditions, Nrf2 is constantly degraded via the ubiquitin–proteasome pathway in a Keap1-dependent manner. When oxidative or electrophilic stress inactivates Keap1 by two possible mechanisms detailed in Fig. 3 (hinge and latch mechanism and Cul3 dissociation), Nrf2 is stabilized and *de novo* synthesized Nrf2 translocates into nuclei. Nrf2 heterodimerizes with small Maf proteins and activates target genes for cytoprotection through antioxidant/electrophile response element (ARE/EpRE). Target genes of Nrf2 are involved in 1) glutathione synthesis (Glutamate-cysteine ligase, catalytic subunit (Gclc), glutamate-cysteine ligase, modifier subunit (Gclm)), 2) elimination of ROS (Thioredoxin reductase 1 (Txnrd1), Peroxiredoxin 1 (Prdx1), 3) detoxification of xenobiotics (NAD(P)H dehydrogenase, quinone 1 (Nqo1), Glutathione S-transferase (Gst) gene family) and 4) drug transport (Multi-drug resistance-associated protein (Mrp) gene family). E, electrophile.

degraded via the ubiquitin–proteasome pathway in a
Keap1-dependent manner (Sekhar et al. 2002; McMahon et al. 2003; Kobayashi et al. 2004). In the presence of electrophiles or ROS, Nrf2 degradation ceases, stabilized Nrf2 accumulates in nuclei, heterodimerizes with small Maf proteins and activates target genes for cytoprotection through antioxidant response element (ARE)/electrophile response element (EpRE)

(Itoh et al. 2004). Thus, the level of Nrf2 protein is regulated by degradation processes, and the inducible stabilization of Nrf2 is the essence of the cellular response to oxidative and electrophilic stresses.

Elucidation of the molecular mechanisms involved in the regulation of Nrf2 stability in response to electrophiles and ROS has been a critical issue in the study of stress response. There are several important

124 Genes to Cells (2011) 16, 123–140
© 2011 The Authors
Journal compilation © 2011 by the Molecular Biology Society of Japan/Blackwell Publishing Ltd.

Molecular mechanisms of the Keap1–Nrf2 pathway

questions related to the molecular mechanisms. The first is how Nrf2 is degraded in a Keap1-dependent manner. Molecular dissection of Nrf2 identified six Neh domains (Fig. 2A). The Neh2 domain localized in the N-terminal region of Nrf2 is essential for Keap1-dependent degradation of Nrf2 (McMahon et al. 2003, 2004; Zhang et al. 2004), and this domain directly associates with the β-barrel structure of Keap1, designated as the DC domain (Tong et al. 2006a) (Fig. 2A). Keap1 also associates with Cullin 3 (Cul3) to form ubiquitin E3 ligase, in which Keap1 serves as a substrate adaptor. Keap1–Cul3 ubiquitin E3 ligase polyubiquitates Nrf2 for degradation (Fig. 2B), and Nrf2 is the most common substrate of Keap1.

The second question is how Nrf2 degradation can be stopped upon exposure to electrophiles and ROS. Keap1 is a thiol-rich protein and possesses multiple reactive cysteines whose neighboring residues are often basic amino acids. The reactive thiols are excellent targets of electrophiles, and indeed, several electrophilic reagents have been shown to modify the

(A)
DLG ETGE
motif motif
Neh2 Neh4 Neh5 Neh6 Neh1 Neh3
Keap1 binding Transactivation Small Maf binding DNA binding
Nrf2 (597 aa)
DGR CTR
Keap1 (624 aa)
NTR BTB IVR Kelch Kelch Kelch Kelch Kelch CTR
Cul3 binding dimerization
DC domain = Nrf2 binding

(B)

(C)
IκB
Hif1α
Rbx1 Rbx1
E2 Ub E2 Ub
Cul1 Cul2
Skp1 pVHL
F-box WD40 IκB EloB
Ub Ub Ub Ub
Nrf2 Hifα
Rbx1 Rbx1
E2 Ub E2 Ub
Cul3 Cul4B
Arnt AhR
Keap1 L
Nrf2 ERα
Ub Ub Ub Ub
Ub Ub

Figure 2 Degradation of Nrf2. (A) Domain structure of Nrf2 and Keap1. Nrf2 protein is divided into six highly conserved domains, Neh1 to Neh6 (NRF2-ECH homology: Neh). Neh2 domain is highly conserved among species and binds with Keap1. Neh4 and Neh5 are transactivation domains that bind to CBP. Neh1 is a basic region-leucine zipper structure for dimerization with small Maf and DNA binding. Domain structure of Keap1 protein is as follows; NTR (N-terminal region; a.a. 1-60), BTB (a.a. 61-179), intervening region (IVR; a.a. 180-314), DC domain harboring six Kelch-repeat domain (a.a. 315-359, 361-410, 412-457, 459-504, 506-551, 553-598) and C-terminal region (CTR; a.a. 599-624). BTB domain and N-terminal portion of IVR are regarded important for the association with Cul3. DC domain mediates interaction with Neh2 domain of Nrf2. (B) Ubiquitination of Cullin-based ubiquitin E3 ligase. Protein ubiquitination is a three-step process mediated by an E1-activating enzyme, an E2-conjugating enzyme and an E3 ligase. (C) Activities of transcription factors involved in environmental responses are regulated at protein degradation levels. Keap1 serves as an adaptor of Cul3-based E3 ligase for Nrf2 ubiquitination. L: a ligand for AhR such as dioxins.

© 2011 The Authors
Genes to Cells (2011) 16, 123–140 125
Journal compilation © 2011 by the Molecular Biology Society of Japan/Blackwell Publishing Ltd.

K Taguchi *et al.*

thiols directly (Holland *et al.* 2008; Satoh *et al.* 2008; Kobayashi *et al.* 2009; Sekhar *et al.* 2010). Modification of Keap1 is considered to impair the structural integrity of the Keap1-Cul3 E3 ligase complex, resulting in the decline of the ubiquitination activity, thereby facilitating accumulation of Nrf2. Thus, Keap1 acts as a sensor that transduces electrophilic signals to transcriptional activation. Currently, biochemical, biophysical and structural analyses of the Keap1-Nrf2 interaction have been conducted extensively to reveal the characteristics of the molecular sensor for electrophiles.

The regulation of Nrf2 levels by Keap1 is abrogated in some human cancers (Padmanabhan *et al.* 2006; Singh *et al.* 2006; Ohta *et al.* 2008). Mutations in *NRF2* and *KEAP1* genes have been found frequently in various cancer cases. These mutations are sufficient to lead to constitutive activation of NRF2 by disrupting the NRF2-KEAP1 interaction. NRF2 activation in cancers contributes not only to drug resistance by efflux of anticancer drugs but also to cancer cell proliferation (Ohta *et al.* 2008; Singh *et al.* 2008). In this review, we summarize the latest advances in the analysis of the Keap1-Nrf2 pathway and focus on the functional contribution of the Keap1-Nrf2 system to cancer.

### Up-regulation of transcription factor activity by derepression

Under quiescent conditions, Nrf2 is constantly degraded with a half-life of < 20 min (Katoh *et al.* 2005). This rapid turnover maintains cellular Nrf2 at a low level. Upon exposure to stresses, Keap1 is inactivated by direct modification of cysteine thiol residues, and subsequently Nrf2 is stabilized, translocates into the nucleus and activates transcription of various detoxification and antioxidant enzyme genes. As electrophilic and oxidative stresses do not induce a robust increase in *Nrf2* mRNA (Itoh *et al.* 2003; McMahon *et al.* 2003), stabilization of Nrf2 protein makes the major contribution to the elevation of Nrf2 activity in response to these stresses. Thus, the Keap1-Nrf2 pathway adopts a derepression-type stress response in which Nrf2 is liberated from repression by Keap1 and accumulates upon exposure to stress.

After BTB proteins were first found to serve as substrate-specific adaptors for Cul3 ubiquitin ligase (Pintard *et al.* 2003; Xu *et al.* 2003), it did not take long for researchers to discover that Keap1, a BTB protein, is a component of the Cul3-based ubiquitin E3 ligase complex and mediates polyubiquitination of

Nrf2 (Cullinan *et al.* 2004; Kobayashi *et al.* 2004; Zhang *et al.* 2004; Furukawa & Xiong 2005). When cells are exposed to electrophilic or oxidative stresses, thiol groups of Keap1 are modified and the E3 ligase activity declines. Derepression regulation is a common strategy taken by transcription factors responsible for stress responses. A common complex consisting of a Cullin-based ubiquitin ligase is shown in Fig. 2B. For instance, HIF1α (hypoxia-inducible factor 1α) is constantly degraded under normoxic conditions (Maynard & Ohh 2004). In this case, the Cul2-pVHL (von Hippel-Lindau) complex serves as an E3 ligase for proline-hydroxylated HIF1α. Upon exposure to hypoxia, proline hydroxylation is inhibited and polyubiquitination of Hif1α is attenuated (Fig. 2C, upper right panel). In the case of IκB (inhibitor of NFκB), which sequesters NFκB (nuclear factor-κB) in the cytoplasm, Cul1-Skp1 (S phase kinase-associated protein 1) E3 ligase is responsible for polyubiquitination of IκB (Magnani *et al.* 2000). When proinflammatory stimuli activate phosphorylation cascades, IκB is phosphorylated and polyubiquitinated for degradation (Fig. 2C, upper left panel).

A unique feature of the Keap1-Nrf2 pathway is that the substrate adaptor component of the E3 ligase (*i.e.*, Keap1), but not substrate (Nrf2), is directly modified by the stimuli, while in the other two cases, substrates (Hif1α and IκB) are modified by hydroxylation or phosphorylation. The Keap1-Cul3 complex thus comprises an ubiquitin E3 ligase with sensor function (Fig. 2C, lower left panel). A similar sensor function resides in the Cul4B-based E3 ubiquitin ligase complex. Aryl hydrocarbon receptor (AhR) associates with Cul4B upon ligand binding, consequently inducing assembly of the Cul4B-based E3 ubiquitin ligase complex (Ohtake *et al.* 2007). The Cul4B-AhR complex executes ubiquitination of estrogen receptor-α (ERα) in response to xenobiotics that bind to AhR (Fig. 2C, lower right panel).

### Mouse models for validation of the Keap1-Nrf2 system

The functional relationship between Keap1 and Nrf2 has been validated by *in vivo* experiments. *Nrf2* gene-knockout (*Nrf2*<sup>*null*</sup>) mice are fertile under normal conditions (Itoh *et al.* 1997), but they are susceptible to various chemical and physical insults (Chan & Kan 1999; Enomoto *et al.* 2001; Yoh *et al.* 2001). This is consistent with the property of Nrf2 as an inducible transcriptional activator. In contrast, *Keap1* gene-knockout (*Keap1*<sup>*null*</sup>) mice die at weaning stage
because of the hyperkeratosis of upper digestive tract leading to feeding difficulty and subsequent malnutrition (Wakabayashi *et al.* 2003). Nrf2 accumulates in the nucleus, and cytoprotective gene expression is constitutively elevated in various tissues of the Keap1-null mice. The hyperkeratosis and massive induction of Nrf2-target genes are canceled in Keap1<sup>null</sup>::Nrf2<sup>null</sup> mice. Although these double-knockout mice substantially lack inducible expression of antioxidant and detoxifying enzymes, they appear to be normal and fertile so long as they are maintained in specific pathogen-free conditions (Wakabayashi *et al.* 2003). Thus, the phenotypes of Keap1-null mice are attributable to the constitutive stabilization of Nrf2.

Keap1 function has been examined utilizing another mouse genetic approach, which is referred to as transgenic complementation rescue analysis. Transgenic mouse lines expressing Keap1 under the regulatory influence of the Keap1 gene regulatory domain (KRD) are generated and crossed with Keap1<sup>null</sup> mice. In the Keap1<sup>null</sup>::Keap1<sup>TG</sup> double-mutant mice, hyperkeratotic lesions of esophagus, elevated expression of cytoprotective genes and Nrf2 protein accumulation were all restored to normal levels, and these compound mutant mice are no longer lethal (Yamamoto *et al.* 2008).

The function of Keap1 mutant molecules has been tested employing this approach. For instance, a Keap1 mutant lacking the BTB domain (Keap1ΔBTB) was expressed in the transgenic mice. Keap1ΔBTB did not rescue the Keap1<sup>null</sup> mice from lethality (Yamamoto *et al.* 2008). As the BTB domain is required for homodimerization of Keap1 (Yamamoto *et al.* 2008) and interaction between Keap1 and Cul3 (Cullinan *et al.* 2004; Kobayashi *et al.* 2004; Zhang *et al.* 2004; Furukawa & Xiong 2005), this result suggests that Keap1 dimerization and/or Keap1–Cul3 interaction are essential for Keap1 function *in vivo*. Based on the report that the dimerization surface and Cul3-associating surface are separable (Melnick *et al.* 2000; Zhuang *et al.* 2009), the dimerization surface of Keap1 was mutated and the resultant mutant Keap1 lost the ability to repress Nrf2 (Suzuki *et al.* in press). These results thus demonstrate that Keap1 must be a homodimer to promote ubiquitination of Nrf2.

The role of the reactive cysteine residues of Keap1 has been examined by three approaches, and especially, three residues, C151, C273 and C288, have been shown critical to the Keap1 function. First, *in vitro* exposure of purified Keap1 protein to electrophilic reagents followed by mass spectrometry identified multiple cysteine residues with thiol modifications

Molecular mechanisms of the Keap1–Nrf2 pathway

(Dinkova-Kostova *et al.* 2002; Eggleter *al.* 2005; Hong *et al.* 2005a,b; Fujii *et al.* 2010) (Fig. 4). Second, these cysteine residues were further tested by reporter transfection assays, and three cysteine residues were identified to be important for Keap1 function (Zhang & Hannink 2003; Levonen *et al.* 2004; Wakabayashi *et al.* 2004). Finally, these residues were validated by means of the transgenic complementation rescue analysis (Yamamoto *et al.* 2008). Cys273 and Cys288 localized in the IVR domain were found to be indispensable for Keap1 to maintain ubiquitin E3 ligase activity and degrade Nrf2. On the other hand, another cysteine residue C151 in the BTB domain was found essential for Keap1 to diminish E3 activity in response to electrophilic stimuli.

### The Keap1 dimer forms a cherry-bob structure

Biochemical and structural analyses have provided important information for deciphering the contribution of Keap1 to the ubiquitination of Nrf2. Molecular dissection of Nrf2 showed that that two distinct motifs, DLG and ETGE (Fig. 2A), are required for the turnover of Nrf2 (McMahon *et al.* 2006). Quantitative binding analysis of full-length Keap1 and the Neh2 domain of Nrf2 using isothermal calorimetry revealed two different interactions between the Keap1 dimer and the Neh2 domain, i.e. high-affinity and low-affinity bindings (Tong *et al.* 2006a). Based on the biochemical assays showing the necessity of ETGE and DLG motifs of the Neh2 domain for Keap1 activity, mutations were introduced into the ETGE and DLG motifs (Tong *et al.* 2007). Neh2 harboring a mutation in one of the two motifs could associate with Keap1 utilizing the remaining motif. Thus, a plausible model is that one molecule of Nrf2 associates with two molecules of Keap1 (i.e. Keap1 dimer) utilizing the EGTE and DLG motifs (Fig. 3A). Thus, stoichiometry between Keap1 and Neh2 is 2:1, and the EGTE and DLG motifs of Nrf2 serve as high-affinity and low-affinity binding sites with Keap1, respectively.

Crystal structure analysis demonstrated that the Keap1 DC domain, which contains DGR composed of six Kelch repeats and CTR domains (Fig. 2A), forms a β-barrel structure composed of six propeller subdomains (Padmanabhan *et al.* 2006). Kelch-repeat β-propellers are generally involved in protein–protein interactions (Adams *et al.* 2000). Co-crystallization of the DC domain with ETGE (Padmanabhan *et al.* 2006) or DLG peptides (Tong *et al.* 2007) revealed

© 2011 The Authors  
Journal compilation © 2011 by the Molecular Biology Society of Japan/Blackwell Publishing Ltd.   Genes to Cells (2011) 16, 123–140 127
K Taguchi et al.

(A)

(B)

(C)

(D)

Figure 3 Cherry-bob structure of the Keap1 homodimer.
(A) Two-site binding model of Keap1 and Nrf2. The Keap1 homodimer binds to a single Nrf2 molecule at two sites, the DLG and ETGE motifs. Seven lysine residues, which are targets of ubiquitination, reside within the α-helical structure between the two binding sites. The two-site binding structure appears to be important for efficient ubiquitination of Nrf2. Ubiquitinated Nrf2 is degraded in the proteasome.
(B) Three-dimensional image of the Keap1 homodimer captured in single particle electron microscopy. Crystal structures of the Keap1 DC (yellow and red) are overlaid. Cyan and pink dotted lines indicate Keap1 monomers.
(C and D) Proposed mechanisms of the Keap1–Nrf2 system in response to electrophiles.
(C) A model referred to as the Hinge and Latch. Upon thiol modification with electrophiles (E), a conformational change in the Keap1 homodimer occurs, which dissociates the DLG motif from the Keap1 DC domain and stops Nrf2 ubiquitination.
(D) Interaction between Keap1 and Cul3 is disrupted upon thiol modification of Keap1. In both models, E3 ligase activity subsequently declines, and Nrf2 is stabilized.

that both peptides fit into an identical pocket surrounded by basic amino acid residues at the bottom of the DC barrel structure (Padmanabhan et al. 2008). NMR structural analysis showed that the Neh2 domain forms a rod-like structure, and especially the region flanked by the DLG and ETGE motifs forms

an α-helix conformation (Tong et al. 2006a). This α-helix contains seven lysine residues, all of which are ubiquitination targets (Zhang et al. 2004) (Fig. 3A). It is interesting to note that six lysine residues of seven in the α-helix are aligned to the same side. Based on these data, a two-site binding model was deduced (Tong et al. 2006a, 2007). One Nrf2 molecule interacts with two Keap1 molecules in the dimer using two binding sites, ETGE and DLG (Fig. 3A). The high-affinity ETGE site is engaged first, helping the low-affinity DLG site to bind. This conformation seems to stimulate Keap1-mediated ubiquitination of Nrf2 through presenting the target lysines to the ubiquitin ligase complex.

Finally, the overall structure of the Keap1 dimer was recently elucidated by means of a single particle electron microscopic analysis (Ogura et al. 2010). The emerging silhouette resembles a cherry-bob, as two round globular structures are connected with a stem-like structure (Fig. 3B). Each globular structure is a rounded cylinder with a narrow penetrating tunnel, which just fits the central channel of the DC barrel domain observed in the Keap1 crystal. The electron density is much larger than the DC domain determined by X-ray crystallography, indicating that the IVR domain is located within close proximity of the DC domain (Ogura et al. 2010). The stem-like structure is thought to be the homodimerizing BTB domains. The distance between the two binding pockets of DC domains is estimated to be approximately 80 Å, which shows good agreement with the distance of the two binding sites in the Neh2 domain.

## Two distinct mechanisms for Nrf2 stabilization

Currently, two mechanisms have been proposed for the reduction in Keap1 activity by electrophiles. One is that, upon exposure to the stimuli, Nrf2 dissociates from Keap1 and this ablates Nrf2 ubiquitination (Fig. 3C). Important observations related to this mechanism are that, even when Keap1–Cul3 E3 ligase activity is lost because of the challenge of electrophilic reagents, the overall interaction between Keap1 and Nrf2 is not disrupted (Kobayashi et al. 2006). In agreement with this, alanine substitution of C273 and C288 of Keap1 abrogates the ability to degrade Nrf2 but retains the binding affinity to Nrf2 (Kobayashi et al. 2006). It has also been found that de novo synthesized Nrf2 translocates into the nucleus rather than Nrf2 dissociated from Keap1 (Kobayashi

et al. 2006). These observations contributed to the 'hinge and latch model' (Tong et al. 2006b). As shown in Fig. 3C, in this model, electrophiles directly modify reactive cysteine residues of Keap1 (Kobayashi et al. 2009; Fishbein & Holland 2010) and alter the conformation of the Keap1 dimer. This provokes the dissociation of the low-affinity DLG motif from one of the Keap1 DC domains (latch), while the high-affinity ETGE motif remains associated with the other DC domain (hinge). The affinity of ETGE for the Keap1 DC domain is 100-fold higher than that of DLG (Tong et al. 2007).

An alternative mechanism proposes that Cul3 dissociates from Keap1 upon exposure to the stimuli (Rachakonda et al. 2008) (Fig. 3D). In particular, the Keap1–Cul3 interaction seems to be disrupted in the case of modification at C151 in BTB domain. This seems to be reasonable, as Cul3 has been suggested to associate with the Keap1 N-terminal region including the BTB domain. The loss of Cul3–Keap1 interaction results in a decline in Keap1–Cul3 E3 ligase activity.

rise to different patterns of cysteine modification of Keap1 (Dinkova-Kostova et al. 2002; Eggleter al. 2005; Hong et al. 2005a,b; Fujii et al. 2010). This implies that each electrophile prefers a specific set of cysteine residues in Keap1, including C151, C273 and C288, and displays a unique pattern of cysteine modification.

Experiments in zebrafish provide important information in support of this notion (Kobayashi et al. 2009). Zebrafish possess two Keap1 genes, zKeap1a and zKeap1b (Li et al. 2008). The cysteine residue corresponding to C151 in mouse and human is C127 and C125 in zKeap1a and zKeap1b, respectively. C151 of mammalian Keap1 and C125 of zKeap1b are highly reactive, as the neighboring lysine residue decreases the pKa of the thiol. In contrast, C127 of zKeap1a is not reactive, as the neighboring threonine does not have such an effect. While zKeap1a and zKeap1b similarly inhibit Nrf2 activity, only the latter responds to diethylmaleate (DEM), an electrophilic reagent, and stabilizes Nrf2 (Li et al. 2008). Substitution of C125 of zKeap1b with a bulky amino acid tryptophan (which seems to mimic the cysteine residue conjugated with electrophiles) abrogates the ability to suppress Nrf2 activity (Kobayashi et al. 2009). The similar result was yielded in the characterization of C151W mutant molecule of the mammalian Keap1 (Eggleter al. 2009), conforming to the notion that

The cysteine code for electrophilic compounds

As summarized in Fig. 4, mass spectrometry of Keap1 protein treated with electrophilic reagents *in vitro* demonstrates that different electrophilic reagents give

| Dexamethasone | 257 | 273 | 288 | 297 | 613 |
| --- | --- | --- | --- | --- | --- |
| 21-mesylate |  |  |  |  |  |

| Biotinylated | 151 | 257 | 273 | 288 | 297 | 319 | 613 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| iodoacetamide |  |  |  |  |  |  |  |

| IAB | 196 | 226 | 241 | 257 | 288 | 319 |  |
| --- | --- | --- | --- | --- | --- | --- | --- |

| Sulforaphane |  |  |  |  |  | 489 | 583 | 624 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |

| 8-nitro-cGMP |  |  |  |  |  | 434 |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |

![Diagram](attachment:diagram.png)

Figure 4 Modification of cysteine residues of Keap1 by electrophiles. Each electrophile displays its own favorite pattern of thiol modification (the cysteine code). The cysteine residues whose biological importance has been proved *in vivo* are shown in red. IAB, N-iodoacetyl-N-biotinylhexylenediamine.

© 2011 The Authors  
Journal compilation © 2011 by the Molecular Biology Society of Japan/Blackwell Publishing Ltd.

K Taguchi et al.

C125 of zKeap1 and C151 of mammalian Keap1 are functionally equivalent.

Substitution of C125 of zKeap1b to serine abrogates the reactivity to DEM and other electrophilic reagents, including tBHQ, ebselen, sulforaphane and 1,2-naphthoquinone, while the substitution retains the reactivity to 15d-PGJ₂ and PGA₂ (Kobayashi et al. 2009). Electrophiles similarly discriminate the cysteine residues of mammalian Keap1. Mouse embryonic fibroblasts and peritoneal macrophages obtained from the mice expressing Keap1 C151S mutant in replacing the wild-type Keap1 are sensitive to DEM and tBHQ but not to 15d-PGJ₂ (K. Takaya and M. Yamamoto, unpublished observation). 15d-PGJ₂ and PGA₂ may favor to react with the cysteine residues other than C151, most likely to be C273 and C288. Thus, each electrophile seems to exploit a distinct set of cysteine residues to trigger the response. When targets of electrophiles are expanded to proteins other than Keap1, each electrophile may attack a distinct set of thiol-bearing proteins, exerting a distinct biological output. We refer to this specific function of each cysteine residue as the cysteine code (Yamamoto et al. 2008; Kobayashi et al. 2009).

encoding detoxifying enzymes and consequently increasing the capacity to detoxify the carcinogens (Ramos-Gomez et al. 2001). A field study in China proved that Nrf2 activation is potentially effective for cancer chemoprevention (Wang et al. 1999; Kensler et al. 2005).

Surprisingly, missense mutations of the KEAP1 gene have been found in human lung adenocarcinomas (Padmanabhan et al. 2006; Singh et al. 2006; Nioi & Nguyen 2007; Ohta et al. 2008; Shibata et al. 2008a; Takahashi et al. 2010). Inactivation of KEAP1, because of somatic mutations causing amino acid substitutions, has been found in cancer cells in the lung, gallbladder and liver (Shibata et al. 2008a). The mutations found so far are distributed 3% in the NTR, 3% in the BTB, 29% in the IVR and 65% in the DC domains (Fig. 5A and Table 1A). Reduction of KEAP1 gene expression because of abnormally increased DNA methylation at its promoter region (Fig. 6A right) is also reported in cancer cells generated in lung (Wang et al. 2008) and in prostate (Zhang et al. 2010). In either case, KEAP1 function is impaired, leading to constitutive stabilization of NRF2 and increased activation of cytoprotective genes (Fig. 6B) (Okawa et al. 2006; Singh et al. 2008).

The constitutive activation of the genes encoding detoxification and antioxidant enzymes confers a great advantage on cancer cells fighting for survival against anticancer drugs, such as etoposide and carboplatin (Wang et al. 2006). The effect of KEAP1 mutation depends on NRF2 activity because NRF2 knockdown by siRNA cancels the increased resistance against the chemotherapeutic reagents (Homma et al. 2009). In addition, NRF2 knockdown inhibits cell proliferation of human lung cancer cell lines, A549 and H460, which have point mutations in the KEAP1 gene causing amino acid substitution of D236H and G333C, respectively (Singh et al. 2008). In the H292 cell line that harbors an intact KEAP1 gene, NRF2 is barely activated above the basal level (Singh et al. 2006) and NRF2 knockdown had little effect on cell proliferation or drug resistance (Homma et al. 2009).

An emerging notion is that the Keap1-Nrf2 regulatory system is a double-edged sword with regard to whole body benefits. From the viewpoint of a single cell, high-level Nrf2 activity increases cell protection and makes the cell resistant to oxidative and electrophilic stresses. Therefore, Nrf2 is protective for the whole body and preferable to suppressing cancer initiation by directing efficient elimination of ROS and detoxification of carcinogens and, consequently,

KEAP1 mutation in human cancers

The primary task of Keap1 is a sensor of electrophilic substances and also endogenous stresses. Keap1 appears to possess multiple thiol residues whose pKa values are low, as 10 μmol/L 8-nitro-cGMP, one of the endogenous electrophiles generated downstream of nitric oxide, can modify Keap1 even in the presence of 10 mmol/L glutathione (Sawa et al. 2007). This suggests that Keap1 can detect low concentrations of electrophiles within cells and serves as a sensitive sensor. Such sensitivity enables the Keap1-Nrf2 system to play a central role in electrophile counterattack response for cell protection.

Electrophilic reagents are inherently toxic because of their ability to attack nucleic acids as well as proteins possessing centers of high electron density. It is known that pretreatment of cells with low-dose electrophilic reagents improves cell protection ability, and this phenomenon is known as the electrophile counterattack response (Pretera et al. 1993). It has been shown that Nrf2 is the critical regulator that enhances cell protection against a much more severe second round attack (Higgins et al. 2009). This also explains the observation that premedication with Oltipraz, which induces Nrf2 activation, prevents chemical carcinogenesis by activating the genes

Molecular mechanisms of the Keap1–Nrf2 pathway

(A) Keap1

|  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- |
| 1/30 (3%) | 1/30 (3%) | 9/30 (29%) | 5/30 (16%) | 2/30 (7%) | 7/30 (23%) |
|  |  |  |  |  |  |
|  |  |  |  |  |  |
| NTR | BTB | IVR | DC domain |

CUL3 binding  
dimerization  

NRF2 binding  

(B) DLG ETGE  
15/35 20/35  
(43%) (57%)

|  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- |
| Neh2 | Neh4 | Neh5 | Neh6 | Neh1 | Neh3 |

KEAP1 binding  
Transactivation  
Small MAF binding  
DNA binding  

Figure 5 KEAP1 and NRF2 mutations in cancer cells. (A) Location and incidence of mutations of the KEAP1 gene identified in cancer cells. (B) Location and incidence of mutations of the NRF2 gene identified in cancer cells.

protecting DNA from oxidative and xenobiotic damage. Furthermore, recent work has shown that Nrf2 contributes to repressing cancer metastasis (Satoh et al. 2010). These observations demonstrate that Nrf2 is beneficial for a host defense against cancer.

On the other hand, elevated Nrf2 activity is also advantageous for cancer cell survival and evolution. Simple Keap1 deficiency is not enough for spontaneous cancer development (Taguchi et al. 2010). When Keap1 knockdown (Keap1KD) mice were generated, Keap1 mRNA abundance in the Keap1KD mice was approximately 5-60% of that in wild-type mice and constitutive Nrf2 activation was observed in various tissues, such as liver, lung and kidney. Over a 2-year period Keap1KD mice were examined, and the mice were found to survive until adulthood and failed to develop any spontaneous cancers during the survival. Therefore, dysregulation of the Keap1–Nrf2 system itself does not appear to trigger cancer initiation, but may confer advantages to the cancer cells in terms of proliferation and/or stress resistance. It should be noted, however, that the Keap1KD mice displayed decreased 2-year survival rates compared with wild-type mice. Whether the shortened life span observed in the Keap1KD mice is linked to the constitutive activation of Nrf2 or not remains to be elucidated.

NRF2 mutation in human cancers supports the two-site binding model

A wonderful intersection of molecular biology and cancer pathology occurred as a result of the identification of NRF2 mutations in human cancers (Fig. 5B). Mutations in NRF2 were first found in primary lung cancers (11/103, 11%) and primary head and neck cancers (3/12, 25%) (Shibata et al. 2008b). A second report identified NRF2 mutations in cancers of lung (10/125; 8%), head and neck (3/23; 13%), esophagus (8/70; 11%) and skin (1/17; 6%) (Kim et al. 2010). It should be noted that all the mutations are clustered within DLG (43%) and ETGE (57%) motifs (Fig. 5B and Table 1B), which are critical sites for the binding of Nrf2 to the Keap1 DC pocket.

These NRF2 mutations activate Nrf2; Nrf2 accumulates in the cancer cells harboring the NRF2 mutations. Mutations in the ETGE motif disrupt the high-affinity binding of Keap1 and Nrf2, so that ETGE mutant molecules are not ubiquitinated, but accumulate in nuclei (Shibata et al. 2008b). Mutations in the DLG motif disrupt the low-affinity binding of Keap1 and Nrf2 (Fig. 6A, left). The DLG mutant molecules still retain high-affinity binding through the

© 2011 The Authors  
Journal compilation © 2011 by the Molecular Biology Society of Japan/Blackwell Publishing Ltd.

K Taguchi et al.

Table 1 (A) Mutant KEAP1 found in cancer cells. (B) Mutant NRF2 found in cancer cells

| Mutation | Domain | Histology | Zygosity | Tissue origin | Reference |
|----------|--------|-----------|-----------|---------------|-----------|
| (A)      |        |           |           |               |           |
| C23Y     | NTR    |           |           | Breast        | Nioi & Nguyen (2007) |
| V167F    | BTB    |           |           | Lung          | Singh et al. (2006) |
| 543insC Frameshift (codon 181) | IVR | ADC | Homozygous | Gallbladder | Shibata et al. (2008a) |
| H200P    | IVR    | ADC       | Heterozygous | Lung          | Ohta et al. (2008) |
| D236H    | IVR    |           | Homozygous | Lung          | Singh et al. (2006) |
| 237 Frameshift (stop codon) | IVR |           |               | Lung          | Singh et al. (2006) |
| C249Y    | IVR    | CCC       | Homozygous | Liver         | Shibata et al. (2008a) |
| I264V    | IVR    |           |           | Pleural fluid | Singh et al. (2006) |
| R272C    | IVR    | ADC       | Homozygous | Lung          | Ohta et al. (2008) |
| Q284L    | IVR    |           |           | Lung          | Singh et al. (2006) |
| G332C    | IVR    | ADC       |           | Lung          | Takahashi et al. (2010) |
| 996delC Frameshift (codon 332) | 1st Kelch | AS | Homozygous | Gallbladder | Shibata et al. (2008a) |
| G333C    | 1st Kelch |           | Homozygous | Lung          | Singh et al. (2006) |
| S338L    | 1st Kelch | ADC | Heterozygous | Gallbladder   | Shibata et al. (2008a) |
| 348 Frameshift (stop codon) | 1st Kelch |           |               | Pleural fluid | Singh et al. (2006) |
| G350S    | 1st Kelch |           | Heterozygous | Lung          | Singh et al. (2006) |
| G364C    | 2nd Kelch | ADC | Homozygous | Lung          | Padmanabhan et al. (2006); Ohta et al. (2008) |
| G379D    | 2nd Kelch | ADC | Homozygous | Gallbladder   | Shibata et al. (2008a) |
| L413R    | 3rd Kelch |           | Homozygous | Lung          | Singh et al. (2006) |
| 413 Frameshift (stop codon) | 3rd Kelch |           |               | Lung          | Singh et al. (2006) |
| R415G    | 3rd Kelch | ADC | Heterozygous | Lung          | Ohta et al. (2008) |
| A427V    | 3rd Kelch | SCC | Heterozygous | Lung          | Ohta et al. (2008) |
| G430C    | 3rd Kelch | ADC | Heterozygous | Lung          | Padmanabhan et al. (2006); Ohta et al. (2008) |
| 443 Frameshift (stop codon) | 3rd Kelch |           | Homozygous | Lung          | Singh et al. (2006) |
| 457 Frameshift (stop codon) | 3rd Kelch |           |               | Lung          | Singh et al. (2006) |
| G476R    | 4th Kelch | LCC | Homozygous | Lung          | Ohta et al. (2008) |
| R483H    | 4th Kelch | SCC |           | Lung          | Takahashi et al. (2010) |
| E493Q    | 4th Kelch | ADC |           | Lung          | Takahashi et al. (2010) |
| W554X    | 6th Kelch | CCC | Heterozygous | Liver         | Shibata et al. (2008a) |
| 555-560(6aa) deletion | 6th Kelch |           |               | Pleural fluid | Singh et al. (2006) |
| G570V    | 6th Kelch | ADC |           | Lung          | Takahashi et al. (2010) |

| (B)      |        |           |           |               |           |
|----------|--------|-----------|-----------|---------------|-----------|
| W24C     | Neh2DLG | SQ        | Heterozygous | Lung          | Shibata et al. (2008b) |
| W24C     | Neh2DLG | LCNEC     | Heterozygous | Lung          | Shibata et al. (2008b) |
| D27Y     | Neh2DLG | SQ        |             | esophagus     | Kim et al. (2010) |
| I28T     | Neh2DLG | SQ        | Heterozygous | Lung          | Shibata et al. (2008b) |
| D29G     | Neh2DLG | HN        | Heterozygous | Head and neck | Shibata et al. (2008b) |
| D29H     | Neh2DLG | SQ        |             | Lung          | Kim et al. (2010) |
| D29H     | Neh2DLG | SQ        |             | Larynx        | Kim et al. (2010) |
| L30F     | Neh2DLG | SQ        | Homozygous  | Lung          | Shibata et al. (2008b) |
| G31A     | Neh2DLG | SQ        |             | Lung          | Kim et al. (2010) |
| G31A     | Neh2DLG | SQ        |             | Skin          | Kim et al. (2010) |
| V32T     | Neh2DLG | SQ        | Heterozygous | Lung          | Shibata et al. (2008b) |
| V32del   | Neh2DLG | SQ        |             | esophagus     | Kim et al. (2010) |
| S33-F37dup | Neh2DLG | SQ        |             | esophagus     | Kim et al. (2010) |
| R34Q     | Neh2DLG | SQ        | Heterozygous | Lung          | Shibata et al. (2008b) |
| R34Q     | Neh2DLG | SQ        |             | Lung          | Kim et al. (2010) |
| Q75H     | Neh2ETGE | HN        | Heterozygous | Head and neck | Shibata et al. (2008b) |
| D77V     | Neh2ETGE | NSCLC     | Heterozygous | Lung          | Shibata et al. (2008b) |

Table 1 (Continued)

| Mutation | Domain    | Histology | Zygosity     | Tissue origin | Reference           |
|----------|-----------|-----------|--------------|---------------|---------------------|
| D77A     | Neh2ETGE  | SQ        |              | Lung          | Kim et al. (2010)   |
| D77N     | Neh2ETGE  | SQ        |              | Larynx        | Kim et al. (2010)   |
| E79K     | Neh2ETGE  | SQ        | Heterozygous | Lung          | Shibata et al. (2008b) |
| E79K     | Neh2ETGE  | NSCLC     | Homozygous   | Lung          | Shibata et al. (2008b) |
| E79Q     | Neh2ETGE  | ADC       | Heterozygous | Lung          | Shibata et al. (2008b) |
| E79K     | Neh2ETGE  | SQ        |              | Lung          | Kim et al. (2010)   |
| E79Q     | Neh2ETGE  | SQ        |              | Lung          | Kim et al. (2010)   |
| E79G     | Neh2ETGE  | SQ        |              | Larynx        | Kim et al. (2010)   |
| E79dup   | Neh2ETGE  | SQ        |              | esophagus     | Kim et al. (2010)   |
| T80R     | Neh2ETGE  | SQ        | Heterozygous | Lung          | Shibata et al. (2008b) |
| T80I     | Neh2ETGE  | HN        | Heterozygous | Head and neck | Shibata et al. (2008b) |
| T80K     | Neh2ETGE  | SQ        |              | esophagus     | Kim et al. (2010)   |
| G81D     | Neh2ETGE  | ADC       |              | Lung          | Kim et al. (2010)   |
| G81V     | Neh2ETGE  | SQ        |              | esophagus     | Kim et al. (2010)   |
| E82G     | Neh2ETGE  | SQ        | Heterozygous | Lung          | Shibata et al. (2008b) |
| E82D     | Neh2ETGE  | OC        | Heterozygous | Oral          | Shibata et al. (2008b) |
| E82Q     | Neh2ETGE  | SQ        |              | Lung          | Kim et al. (2010)   |
| E82L     | Neh2ETGE  | SQ        |              | esophagus     | Kim et al. (2010)   |

ETGE motif, yet the mutants are no longer degraded. The fact that these NRF2 mutations cause constitutive stabilization of Nrf2 strongly supports the two-site binding model. In addition, the fact that DLG mutation is sufficient to cause constitutive stabilization of Nrf2 further supports the hinge and latch model of Nrf2 stabilization. Both ETGE and DLG mutations should make Nrf2 constitutively active, which is consistent with the fact that most mutations (15/17, 88%) found in NRF2 are heterozygous (Shibata et al. 2008b).

We initially surmised that the consequence of KEAP1 mutations and that of NRF2 mutations must be the same. However, cancer incidence is completely different and mutually exclusive between KEAP1 and NRF2 mutations. The former is found in adenocarcinoma, while the latter is found in squamous cell carcinoma. Several features of the patients carrying cancers with a NRF2 mutation are 1) higher incidence of smoking history (31/39, 79%), 2) higher frequency of squamous cell carcinoma (30/39, 77%), 3) lower incidence of simultaneous EGFR gene mutation and 4) poor prognosis (Shibata et al. 2008b). The last feature could be attributable to the constitutively activated Nrf2 conferring protection to the cancer cells. Supporting this notion, suppression of the NRF2 level by siRNA knockdown-sensitized cancer cells to oxidative stress and a chemotherapeutic reagent cisplatin (Homma et al. 2009; Jiang et al. 2010).

Various signals inducing Nrf2 activity

Recently, nonelectrophilic inducers of Nrf2 originating from endogenous stress have been identified. For instance, the C-terminal region of p21<sup>Cip1/WAF1p21</sup> is shown to associate with the DLG motif of Nrf2 (Chen et al. 2009), and this makes the latch (DLG site) open and turn off the E3 ligase activity of Keap1 (Fig. 6A, middle). As a target of p53-mediated cell cycle arrest and apoptosis (Gartel & Tyner 2002), p21 promotes cell survival in response to oxidative stress (O'Reilly 2005). Thus, the cytoprotective function of p21 appears to be dependent on Nrf2, and transcriptional activation of cytoprotective genes by Nrf2 seems to be potentiated in the presence of p21 through facilitated stabilization of Nrf2.

Another protein inducer is p62, a polyubiquitin-binding protein, which targets various substrates for autophagy (Komatsu & Ichimura 2010). The STGE motif within p62 interacts with Keap1 DC pockets with similar affinity to that of the DLG motif of Nrf2 (Komatsu et al. 2010) (Fig. 6A, middle). When autophagy is impaired, polyubiquitinated protein aggregates containing p62 are not cleared, and the aggregates remain within cells as inclusion bodies. Increased p62 kicks the DLG motif out of the Keap1 DC pocket, inhibits Nrf2 ubiquitination, stabilizes/accumulates Nrf2 and increases the expression of cytoprotective genes. Importantly, abnormal accumulation of p62 is often observed in certain cancers.

K Taguchi et al.

(A)

| KEAP1 or NRF2 | Accumulation of | DNA methylation of |
|---------------|-----------------|--------------------|
| somatic mutations | disruptor proteins | KEAP1 gene |

(B)

Figure 6 Nrf2 is constitutively stabilized in many cancer cells. (A) Three mechanisms have been reported for the constitutive stabilization of Nrf2 in cancer cells. The first is KEAP1 or NRF2 somatic mutations that cause amino acid substitution or truncation of Keap1 or Nrf2 (left panel). The second is the accumulation of proteins that disrupt association between Keap1 and Nrf2 (middle panel). For instance, p62, a ubiquitin-interacting protein critical for autophagy, associates with the Keap1 DC domain. p21, one of the CDK inhibitors, interacts with the Neh2 domain of Nrf2. In both cases, the two-site binding conformation of Keap1 and Nrf2 is disrupted. The third is reduced expression of KEAP1 caused by the promoter methylation of the KEAP1 gene (right). (B) Nrf2 accumulates in nuclei, heterodimerizes with small Maf, binds to antioxidant response element (ARE)/electrophile response element (EpRE) and activates many cytoprotective genes. ROS, reactive oxygen species.

such as hepatocellular carcinoma (Lu et al. 2001; Strnad et al. 2008).

Activation of Nrf2 appears to be regulated by phosphorylation-signaling pathways, including MAP-Ks (mitogen-activated protein kinases), PI3K (phosphatidylinositol 3-kinase), protein kinase C (PKC) and CK2 (casein kinase 2) (Surh et al. 2008). Ser40 and Tyr568 have been identified as phosphorylation sites of Nrf2 by PKC and tyrosine kinase Fyn, respectively (Huang et al. 2002; Bloom & Jaiswal 2003; Numazawa et al. 2003; Jain & Jaiswal 2006). In case of MAPKs, it turns out that identified Nrf2 phosphorylation sites show limited contribution to the modulation of the Nrf2-dependent antioxidant

response (Sun et al. 2009). Nonetheless, there remains a distinct possibility that phosphorylation signaling may enhance Nrf2-dependent transcription by exerting direct and/or indirect effects, such as an alteration of cofactor activity.

The unusually slow migration of Nrf2 in SDS-PAGE is a problem yet to be solved. Although the theoretical molecular weight of Nrf2 is 66 kDa, Nrf2 is detected at the size of approximately 110 kDa in SDS-PAGE-immunoblot analysis (Moi et al. 1994). The amino acid composition with abundant acidic residues may be responsible for the slow electrophoretic mobility (Van Beveren et al. 1983; Watt et al. 1985; Chan et al. 1993), considering the relatively low isoelectric point of Nrf2 (pH 4.7). In both mouse and human Nrf2, acidic residues are predominant comprising 17% of the whole protein, while basic residues (lysine, arginine, and histidine) are limited to 13%. An alternative cause may be posttranslational modification of Nrf2, such as phosphorylation (Pi et al. 2007), covalent binding of an actin monomer (43 kDa) (Kang et al. 2002) or tetra-ubiquitin conjugation (Li et al. 2005).

### p62 accumulation in human cancer

As mentioned previously, p62 has been identified as an inducer of Nrf2 (Copple et al. 2010; Jain et al. 2010; Komatsu et al. 2010; Lau et al. 2010). Recent reports have shown that p62 is commonly up-regulated in various human cancers (Zhang et al. 1999; Zatloukal et al. 2007). As p62 is a selective substrate of autophagy, there seems to be a correlation between carcinogenesis and impairment of autophagy.

Autophagy is one of the protein degradation systems, especially for organelles and cytoplasmic proteins. Proteins are sequestered into double-membrane vesicles, autophagosomes and delivered into lysosomes for breakdown and eventual recycling (Seglen et al. 1990). Besides cancer, several pathological conditions are accompanied by aberrant accumulation of p62 and possible impairment of autophagy. These pathological conditions include alcoholic/nonalcoholic steatohepatitis, Wilson's disease, idiopathic copper toxicosis and primary biliary cirrhosis (Muller et al. 2004; Zatloukal et al. 2007). As accumulated p62 occupies the Keap1 DC pocket leading to Nrf2 accumulation (Fig. 6A, middle), cytoprotective genes must be highly expressed under these pathological conditions.

Atg7 is one of the key molecules of autophagy, and disruption of the Atg7 gene completely abolishes
autophagy. Targeted deletion of *Atg7* in liver results in the accumulation of p62 as inclusion bodies in hepatocytes and leads to liver injury (Komatsu *et al.* 2007). Typical Nrf2 target genes are all elevated in the mutant liver. Intriguingly, the liver injury is rescued by simultaneous deletion of p62 or Nrf2 (Komatsu *et al.* 2007, 2010), indicating that constitutive activation of Nrf2 caused by p62 accumulation is responsible for the liver pathology.

As is the case for the hepatitis observed in *Atg7* deficient mice, cancer cells with p62 accumulation express high levels of Nrf2 target genes (Matsumoto *et al.* 2008). As the elevated expression of Nrf2 target genes confers malignant phenotypes promoting survival and proliferation, the prognosis of p62-positive cancers is expected to be poor.

### Roles of inducers and inhibitors of Nrf2 in chemoprevention and anticancer therapy

As the Keap1–Nrf2 regulatory system is a double-edged sword for our body, both inducers and inhibitors of Nrf2 are beneficial depending on the conditions. Various compounds, including naturally occurring chemicals, have been identified as Nrf2 inducers. Nrf2 inducers are effective for cancer chemoprevention. Broccoli sprouts are rich in glucosinolate that is hydrolyzed by gut microflora or by myrosinase in the cruciferous vegetables to sulforaphane [1-isothiocyanato-4-(methylsulfinyl)-butane] (Fahey *et al.* 2001). Sulforaphane is an isothiocyanate that can induce Nrf2 activity (Zhang *et al.* 1992). The effectiveness of broccoli sprouts has been examined in China, where the risk of hepatocellular carcinoma is high because of the consumption of aflatoxin-contaminated foods (Kensler *et al.* 2005). Synthetic oleanane triterpenoid 1-[2-cyano-3,12-dioxooleana-1,9(11)-dien-28-oyl] imidazole (CDDO-Im) is known as an effective reagent against aflatoxin-induced hepatic tumorigenesis (Yates *et al.* 2006), vinyl carbamate-induced lung tumors (Liby *et al.* 2007) and cigarette smoke-induced emphysema and cardiac dysfunction (Sussan *et al.* 2009). The effective dose of CDDO-Im evaluated in mice is much lower than that of sulforaphane, Oltipraz or butylated hydroxytoluene, indicating that CDDO-Im is a potent inducer of Nrf2.

Nrf2 inhibitors should be effective for anticancer therapy. In contrast to the Nrf2 inducers, specific inhibitors of Nrf2 have not yet been developed. One of the biggest problems in the development of Nrf2 inhibitors is how to achieve specificity. Nrf2 belongs to the CNC protein family containing Nrf1, Nrf3 and

p45, as activator members (Motohashi *et al.* 2002). All the members form heterodimers with small Mafs through their leucine zipper motifs and bind to the MARE consensus sequence through their basic regions, suggesting that the bZip structure of Nrf2 shares many common properties with that of other CNC members. The domains outside the bZip motif might be good targets for Nrf2 inhibitors with the required specificity. However, considering the successful synthesis of a selective inhibitor of AP1 by 3D pharmacophore modeling based on the crystal structure of the bZip domain of the AP1–DNA complex (Aikawa *et al.* 2008), bZip motifs of Nrf2-small Maf might be utilized as a target for selective Nrf2 inhibition. Discovery and development of Nrf2 inhibitors based on the molecular mechanisms of the Keap1–Nrf2 system is one of the most critical and challenging assignments for conquering cancers.

### Acknowledgements

We thank Dr Maggie Patient for her kind help of the edition of this paper. This study was supported in part by Grant-in-Aids for Creative Scientific Research, for Scientific Research on Priority Areas, for Scientific Research, and for Exploratory Research from the Ministry of Education, Science, Sports and Culture, Tohoku University Global COE Program for Conquest of Signal Transduction Diseases with ‘Network Medicine’ from JSPS, Takeda Science Foundation and Naito foundation and Research Grant of the Princess Takamatsu Cancer Research Fund 09-24118. A figure of the three-dimensional structure of the Keap1 homodimer was kindly provided by Dr Chikara Sato (National Institute of Advanced Industrial Science and Technology, Japan).

### References

Adams, J., Kelso, R. & Cooley, L. (2000) The kelch repeat superfamily of proteins: propellers of cell function. *Trends Cell Biol.* **10**, 17–24.

Aikawa, Y., Morimoto, K., Yamamoto, T., Chaki, H., Hashiramoto, A., Narita, H., Hirono, S. & Shiozawa, S. (2008) Treatment of arthritis with a selective inhibitor of c-Fos/activator protein-1. *Nat. Biotechnol.* **26**, 817–823.

Bloom, D.A. & Jaiswal, A.K. (2003) Phosphorylation of Nrf2 at Ser40 by protein kinase C in response to antioxidants leads to the release of Nrf2 from INrf2, but is not required for Nrf2 stabilization/accumulation in the nucleus and transcriptional activation of antioxidant response element-mediated NAD(P)H:quinone oxidoreductase-1 gene expression. *J. Biol. Chem.* **278**, 44675–44682.

Chan, J.Y., Han, X.L. & Kan, Y.W. (1993) Cloning of Nrf1, an NF-E2-related transcription factor, by genetic selection in yeast. *Proc. Natl Acad. Sci. USA* **90**, 11371–11375.
K Taguchi et al.

Chan, K. & Kan, Y.W. (1999) Nrf2 is essential for protection against acute pulmonary injury in mice. *Proc. Natl Acad. Sci. USA* 96, 12731–12736.

Chen, W.M., Sun, Z., Wang, X.J., Jiang, T., Huang, Z.P., Fang, D.Y. & Zhang, D.D. (2009) Direct Interaction between Nrf2 and p21(Cip1/WAF1) Upregulates the Nrf2-Mediated Antioxidant Response. *Mol. Cell* 34, 663–673.

Copple, I.M., Lister, A., Obeng, A.D., Kitteringham, N.R., Jenkins, R.E., Layfield, R., Foster, B.J., Goldring, C.E. & Park, B.K. (2010) Physical and functional interaction of sequestosome 1 with Keap1 regulates the Keap1-Nrf2 cell defense pathway. *J. Biol. Chem.* 285, 16782–16788.

Cullinan, S.B., Gordan, J.D., Jin, J., Harper, J.W. & Diehl, J.A. (2004) The Keap1-BTB protein is an adaptor that bridges Nrf2 to a Cul3-based E3 ligase: oxidative stress sensing by a Cul3-Keap1 ligase. *Mol. Cell. Biol.* 24, 8477–8486.

Dinkova-Kostova, A.T., Holtzclaw, W.D., Cole, R.N., Itoh, K., Wakabayashi, N., Katoh, Y., Yamamoto, M. & Talalay, P. (2002) Direct evidence that sulfhydryl groups of Keap1 are the sensors regulating induction of phase 2 enzymes that protect against carcinogens and oxidants. *Proc. Natl Acad. Sci. USA* 99, 11908–11913.

Eggleer, A.L., Liu, G., Pezzuto, J.M., van Breemen, R.B. & Mesecar, A.D. (2005) Modifying specific cysteines of the electrophile-sensing human Keap1 protein is insufficient to disrupt binding to the Nrf2 domain Neh2. *Proc. Natl Acad. Sci. USA* 102, 10070–10075.

Eggleer, A.L., Small, E., Hannink, M. & Mesecar, A.D. (2009) Cul3-mediated Nrf2 ubiquitination and antioxidant response element (ARE) activation are dependent on the partial molar volume at position 151 of Keap1. *Biochem. J.* 422, 171–180.

Enomoto, A., Itoh, K., Nagayoshi, E., Haruta, J., Kimura, T., O'Connor, T., Harada, T. & Yamamoto, M. (2001) High sensitivity of Nrf2 knockout mice to acetaminophen hepatotoxicity associated with decreased expression of ARE-regulated drug metabolizing enzymes and antioxidant genes. *Toxicol. Sci.* 59, 169–177.

Fahey, J.W., Zalcmann, A.T. & Talalay, P. (2001) The chemical diversity and distribution of glucosinolates and isothiocyanates among plants. *Phytochemistry* 56, 5–51.

Fishbein, J.C. & Holland, R. (2010) Chemistry of the cysteine sensors in Keap1. *Antioxid. Redox Signal.* 13, 1749–1761.

Fujii, S., Sawa, T., Ihara, H., Tong, K.I., Ida, T., Okamoto, T., Ahtesham, A.K., Ishima, Y., Motohashi, H., Yamamoto, M. & Akaike, T. (2010) The critical role of nitric oxide signaling, via protein S-guanylation and nitrated cyclic GMP, in the antioxidant adaptive response. *J. Biol. Chem.* 285, 23970–23984.

Furukawa, M. & Xiong, Y. (2005) BTB protein Keap1 targets antioxidant transcription factor Nrf2 for ubiquitination by the Cullin 3-Roc1 ligase. *Mol. Cell. Biol.* 25, 162–171.

Gartel, A.L. & Tyner, A.L. (2002) The role of the cyclin-dependent kinase inhibitor p21 in apoptosis. *Mol. Cancer Ther.* 1, 639–649.

Higgins, L.G., Kelleher, M.O., Eggleston, I.M., Itoh, K., Yamamoto, M. & Hayes, J.D. (2009) Transcription factor Nrf2 mediates an adaptive response to sulforaphane that protects fibroblasts in vitro against the cytotoxic effects of electrophiles, peroxides and redox-cycling agents. *Toxicol. Appl. Pharmacol.* 237, 267–280.

Holland, R., Hawkins, A.E., Eggleer, A.L., Mesecar, A.D., Fabris, D. & Fishbein, J.C. (2008) Prospective type 1 and type 2 disulfides of Keap1 protein. *Chem. Res. Toxicol.* 21, 2051–2060.

Homma, S., Ishii, Y., Morishima, Y., Yamadori, T., Matsuno, Y., Haraguchi, N., Kikuchi, N., Satoh, H., Sakamoto, T., Hizawa, N., Itoh, K. & Yamamoto, M. (2009) Nrf2 enhances cell proliferation and resistance to anticancer drugs in human lung cancer. *Clin. Cancer Res.* 15, 3423–3432.

Hong, F., Freeman, M.L. & Liebler, D.C. (2005a) Identification of sensor cysteines in human Keap1 modified by the cancer chemopreventive agent sulforaphane. *Chem. Res. Toxicol.* 18, 1917–1926.

Hong, F., Sekhar, K.R., Freeman, M.L. & Liebler, D.C. (2005b) Specific patterns of electrophile adduction trigger Keap1 ubiquitination and Nrf2 activation. *J. Biol. Chem.* 280, 31768–31775.

Huang, H.C., Nguyen, T. & Pickett, C.B. (2002) Phosphorylation of Nrf2 at Ser-40 by protein kinase C regulates antioxidant response element-mediated transcription. *J. Biol. Chem.* 277, 42769–42774.

Itoh, K., Chiba, T., Takahashi, S., Ishii, T., Igarashi, K., Katoh, Y., Oyake, T., Hayashi, N., Satoh, K., Hatayama, I., Yamamoto, M. & Nabeshima, Y. (1997) An Nrf2/small Maf heterodimer mediates the induction of phase II detoxifying enzyme genes through antioxidant response elements. *Biochem. Biophys. Res. Commun.* 236, 313–322.

Itoh, K., Tong, K.I. & Yamamoto, M. (2004) Molecular mechanism activating Nrf2-Keap1 pathway in regulation of adaptive response to electrophiles. *Free Radic. Biol. Med.* 36, 1208–1213.

Itoh, K., Wakabayashi, N., Katoh, Y., Ishii, T., O'Connor, T. & Yamamoto, M. (2003) Keap1 regulates both cytoplasmic-nuclear shuttling and degradation of Nrf2 in response to electrophiles. *Genes Cells* 8, 379–391.

Jain, A., Lamark, T., Sjottem, E., Larsen, K.B., Awuh, J.A., Overvatn, A., McMahon, M., Hayes, J.D. & Johansen, T. (2010) p62/SQSTM1 is a target gene for transcription factor NRF2 and creates a positive feedback loop by inducing antioxidant response element-driven gene transcription. *J. Biol. Chem.* 285, 22576–22591.

Jain, A.K. & Jaiswal, A.K. (2006) Phosphorylation of tyrosine 568 controls nuclear export of Nrf2. *J. Biol. Chem.* 281, 12132–12142.

Jiang, T., Chen, N., Zhao, F., Wang, X.J., Kong, B., Zheng, W. & Zhang, D.D. (2010) High levels of Nrf2 determine chemoresistance in type II endometrial cancer. *Cancer Res.* 70, 5486–5496.

Kang, K.W., Lee, S.J., Park, J.W. & Kim, S.G. (2002) Phosphatidylinositol 3-kinase regulates nuclear translocation of

NF-E2-related factor 2 through actin rearrangement in response to oxidative stress. Mol. Pharmacol. 62, 1001–1010.

Katoh, Y., Iida, K., Kang, M.I., Kobayashi, A., Mizukami, M., Tong, K.I., McMahon, M., Hayes, J.D., Itoh, K. & Yamamoto, M. (2005) Evolutionary conserved N-terminal domain of Nrf2 is essential for the Keap1-mediated degradation of the protein by proteasome. Arch. Biochem. Biophys. 433, 342–350.

Kensler, T.W., Chen, J.G., Egner, P.A., *et al.* (2005) Effects of glucosinolate-rich broccoli sprouts on urinary levels of aflatoxin-DNA adducts and phenanthrene tetraols in a randomized clinical trial in He Zuo township, Qidong, People’s Republic of China. Cancer Epidemiol. Biomar. 14, 2605–2613.

Kim, Y.R., Oh, J.E., Kim, M.S., Kang, M.R., Park, S.W., Han, J.Y., Eom, H.S., Yoo, N.J. & Lee, S.H. (2010) Oncogenic NRF2 mutations in squamous cell carcinomas of oesophagus and skin. J. Pathol. 220, 446–451.

Kobayashi, A., Kang, M.I., Okawa, H., Ohtsuji, M., Zenke, Y., Chiba, T., Igarashi, K. & Yamamoto, M. (2004) Oxidative stress sensor Keap1 functions as an adaptor for Cul3-based E3 ligase to regulate proteasomal degradation of Nrf2. Mol. Cell. Biol. 24, 7130–7139.

Kobayashi, A., Kang, M.I., Watai, Y., Tong, K.I., Shibata, T., Uchida, K. & Yamamoto, M. (2006) Oxidative and electrophilic stresses activate Nrf2 through inhibition of ubiquitination activity of Keap1. Mol. Cell. Biol. 26, 221–229.

Kobayashi, M., Li, L., Iwamoto, N., Nakajima-Takagi, Y., Kaneko, H., Nakayama, Y., Eguchi, M., Wada, Y., Kumagai, Y. & Yamamoto, M. (2009) The antioxidant defense system Keap1-Nrf2 comprises a multiple sensing mechanism for responding to a wide range of chemical compounds. Mol. Cell. Biol. 29, 493–502.

Komatsu, M. & Ichimura, Y. (2010) Physiological significance of selective degradation of p62 by autophagy. FEBS Lett. 584, 1374–1378.

Komatsu, M., Kurokawa, H., Waguri, S., *et al.* (2010) The selective autophagy substrate p62 activates the stress responsive transcription factor Nrf2 through inactivation of Keap1. Nat. Cell Biol. 12, 213–223.

Komatsu, M., Waguri, S., Koike, M., *et al.* (2007) Homeostatic levels of p62 control cytoplasmic inclusion body formation in autophagy-deficient mice. Cell 131, 1149–1163.

Lau, A., Wang, X.J., Zhao, F., Villeneuve, N.F., Wu, T., Jiang, T., Sun, Z., White, E. & Zhang, D.D. (2010) A noncanonical mechanism of Nrf2 activation by autophagy deficiency: direct interaction between Keap1 and p62. Mol. Cell. Biol. 30, 3275–3285.

Levonen, A.L., Landar, A., Ramachandran, A., Ceaser, E.K., Dickinson, D.A., Zanoni, G., Morrow, J.D. & Darley-Usmar, V.M. (2004) Cellular mechanisms of redox cell signalling: role of cysteine modification in controlling antioxidant defences in response to electrophilic lipid oxidation products. Biochem. J. 378, 373–382.

Li, J., Johnson, D., Calkins, M., Wright, L., Svendsen, C. & Johnson, J. (2005) Stabilization of Nrf2 by tBHQ confers

protection against oxidative stress-induced cell death in human neural stem cells. Toxicol. Sci. 83, 313–328.

Li, L., Kobayashi, M., Kaneko, H., Nakajima-Takagi, Y., Nakayama, Y. & Yamamoto, M. (2008) Molecular evolution of Keap1. Two Keap1 molecules with distinctive intervening region structures are conserved among fish. J. Biol. Chem. 283, 3248–3255.

Liby, K., Royce, D.B., Williams, C.R., Risingsong, R., Yore, M.M., Honda, T., Gribble, G.W., Dmitrovsky, E., Sporn, T.A. & Sporn, M.B. (2007) The synthetic triterpenoids CDDO-methyl ester and CDDO-ethyl amide prevent lung cancer induced by vinyl carbamate in A/J mice. Cancer Res. 67, 2414–2419.

Lu, M., Nakamura, R.M., Dent, E.D., Zhang, J.Y., Nielsen, F.C., Christiansen, J., Chan, E.K. & Tan, E.M. (2001) Aberrant expression of fetal RNA-binding protein p62 in liver cancer and liver cirrhosis. Am. J. Pathol. 159, 945–953.

Magnani, M., Crinelli, R., Bianchi, M. & Antonelli, A. (2000) The ubiquitin-dependent proteolytic system and other potential targets for the modulation of nuclear factor-kB (NF-kB). Curr. Drug Targets 1, 387–399.

Matsumoto, N., Ezaki, J., Komatsu, M., Takahashi, K., Mineki, R., Taka, H., Kikkawa, M., Fujimura, T., Takeda-Ezaki, M., Ueno, T., Tanaka, K. & Kominami, E. (2008) Comprehensive proteomics analysis of autophagy-deficient mouse liver. Biochem. Biophys. Res. Commun. 368, 643–649.

Maynard, M.A. & Ohh, M. (2004) von Hippel-Lindau tumor suppressor protein and hypoxia-inducible factor in kidney cancer. Am. J. Nephrol. 24, 1–13.

McMahon, M., Itoh, K., Yamamoto, M. & Hayes, J.D. (2003) Keap1-dependent proteasomal degradation of transcription factor Nrf2 contributes to the negative regulation of antioxidant response element-driven gene expression. J. Biol. Chem. 278, 21592–21600.

McMahon, M., Thomas, N., Itoh, K., Yamamoto, M. & Hayes, J.D. (2004) Redox-regulated turnover of Nrf2 is determined by at least two separate protein domains, the redox-sensitive Neh2 degron and the redox-insensitive Neh6 degron. J. Biol. Chem. 279, 31556–31567.

McMahon, M., Thomas, N., Itoh, K., Yamamoto, M. & Hayes, J.D. (2006) Dimerization of substrate adaptors can facilitate cullin-mediated ubiquitylation of proteins by a “tethering” mechanism: a two-site interaction model for the Nrf2-Keap1 complex. J. Biol. Chem. 281, 24756–24768.

Melnick, A., Ahmad, K.F., Arai, S., Polinger, A., Ball, H., Borden, K.L., Carlile, G.W., Prive, G.G. & Licht, J.D. (2000) In-depth mutational analysis of the promyelocytic leukemia zinc finger BTB/POZ domain reveals motifs and residues required for biological and transcriptional functions. Mol. Cell. Biol. 20, 6550–6567.

Moi, P., Chan, K., Asunis, I., Cao, A. & Kan, Y.W. (1994) Isolation of NF-E2-related factor 2 (Nrf2), a NF-E2-like basic leucine zipper transcriptional activator that binds to the tandem NF-E2/AP1 repeat of the beta-globin locus control region. Proc. Natl Acad. Sci. USA 91, 9926–9930.

K Taguchi et al.

Motohashi, H., O'Connor, T., Katsuoka, F., Engel, J.D. & Yamamoto, M. (2002) Integration and diversity of the regulatory network composed of Maf and CNC families of transcription factors. *Gene* 294, 1–12.

Motohashi, H. & Yamamoto, M. (2004) Nrf2-Keap1 defines a physiologically important stress response mechanism. *Trends Mol. Med.* 10, 549–557.

Muller, T., Langner, C., Fuchsbichler, A., *et al.* (2004) Immunohistochemical analysis of Mallory bodies in Wilsonian and non-Wilsonian hepatic copper toxicosis. *Hepatology* 39, 963–969.

Nioi, P. & Nguyen, T. (2007) A mutation of Keap1 found in breast cancer impairs its ability to repress Nrf2 activity. *Biochem. Biophys. Res. Commun.* 362, 816–821.

Numazawa, S., Ishikawa, M., Yoshida, A., Tanaka, S. & Yoshida, T. (2003) Atypical protein kinase C mediates activation of NF-E2-related factor 2 in response to oxidative stress. *Am. J. Physiol. Cell Physiol.* 285, C334–C342.

Ogura, T., Tong, K.I., Mio, K., Maruyama, Y., Kurokawa, H., Sato, C. & Yamamoto, M. (2010) Keap1 is a forked-stem dimer structure with two large spheres enclosing the intervening, double glycine repeat, and C-terminal domains. *Proc. Natl Acad. Sci. USA* 107, 2842–2847.

Ohta, T., Iijima, K., Miyamoto, M., Nakahara, I., Tanaka, H., Ohtsuji, M., Suzuki, T., Kobayashi, A., Yokota, J., Sakiyama, T., Shibata, T., Yamamoto, M. & Hirohashi, S. (2008) Loss of Keap1 function activates Nrf2 and provides advantages for lung cancer cell growth. *Cancer Res.* 68, 1303–1309.

Ohtake, F., Baba, A., Takada, I., Okada, M., Iwasaki, K., Miki, H., Takahashi, S., Kouzmenko, A., Nohara, K., Chiba, T., Fujii-Kuriyama, Y. & Kato, S. (2007) Dioxin receptor is a ligand-dependent E3 ubiquitin ligase. *Nature* 446, 562–566.

Okawa, H., Motohashi, H., Kobayashi, A., Aburatani, H., Kensler, T.W. & Yamamoto, M. (2006) Hepatocyte-specific deletion of the keap1 gene activates Nrf2 and confers potent resistance against acute drug toxicity. *Biochem. Biophys. Res. Commun.* 339, 79–88.

O'Reilly, M.A. (2005) Redox activation of p21Cip1/WAF1/Sdi1: a multifunctional regulator of cell survival and death. *Antioxid. Redox Signal.* 7, 108–118.

Padmanabhan, B., Tong, K.I., Kobayashi, A., Yamamoto, M. & Yokoyama, S. (2008) Structural insights into the similar modes of Nrf2 transcription factor recognition by the cytoplasmic repressor Keap1. *J. Synchrotron Radiat.* 15, 273–276.

Padmanabhan, B., Tong, K.I., Ohta, T., Nakamura, Y., Scharlock, M., Ohtsuji, M., Kang, M.I., Kobayashi, A., Yokoyama, S. & Yamamoto, M. (2006) Structural basis for defects of Keap1 activity provoked by its point mutations in lung cancer. *Mol. Cell* 21, 689–700.

Pi, J., Bai, Y., Reece, J.M., Williams, J., Liu, D., Freeman, M.L., Fahl, W.E., Shugar, D., Liu, J., Qu, W., Collins, S. & Waalkes, M.P. (2007) Molecular mechanism of human Nrf2 activation and degradation: role of sequential phosphorylation by protein kinase CK2. *Free Radic. Biol. Med.* 42, 1797–1806.

Pintard, L., Willis, J.H., Willem, A., Johnson, J.L., Srayko, M., Kurz, T., Glaser, S., Mains, P.E., Tyers, M., Bowerman, B. & Peter, M. (2003) The BTB protein MEL-26 is a substrate-specific adaptor of the CUL-3 ubiquitin-ligase. *Nature* 425, 311–316.

Pretera, T., Zhang, Y., Spencer, S.R., Wilczak, C.A. & Talalay, P. (1993) The electrophile counterattack response: protection against neoplasia and toxicity. *Adv. Enzyme Regul.* 33, 281–296.

Rachakonda, G., Xiong, Y., Sekhar, K.R., Stamer, S.L., Liebler, D.C. & Freeman, M.L. (2008) Covalent modification at Cys151 dissociates the electrophile sensor Keap1 from the ubiquitin ligase CUL3. *Chem. Res. Toxicol.* 21, 705–710.

Ramos-Gomez, M., Kwak, M.K., Dolan, P.M., Itoh, K., Yamamoto, M., Talalay, P. & Kensler, T.W. (2001) Sensitivity to carcinogenesis is increased and chemoprotective efficacy of enzyme inducers is lost in nrf2 transcription factor-deficient mice. *Proc. Natl Acad. Sci. USA* 98, 3410–3415.

Satoh, H., Moriguchi, T., Taguchi, K., Takai, J., Maher, J.M., Suzuki, T., Winnard, P.T. Jr, Raman, V., Ebina, M., Nukiwa, T. & Yamamoto, M. (2010) Nrf2-deficiency creates a responsive microenvironment for metastasis to the lung. *Carcinogenesis* 31, 1833–1843.

Satoh, T., Kosaka, K., Itoh, K., Kobayashi, A., Yamamoto, M., Shimojo, Y., Kitajima, C., Cui, J., Kamins, J., Okamoto, S., Izumi, M., Shirasawa, T. & Lipton, S.A. (2008) Carnosic acid, a catechol-type electrophilic compound, protects neurons both in vitro and in vivo through activation of the Keap1/Nrf2 pathway via S-alkylation of targeted cysteines on Keap1. *J. Neurochem.* 104, 1116–1131.

Sawa, T., Zaki, M.H., Okamoto, T., Akuta, T., Tokutomi, Y., Kim-Mitsuyama, S., Ihara, H., Kobayashi, A., Yamamoto, M., Fujii, S., Arimoto, H. & Akaike, T. (2007) Protein S-guanylation by the biological signal 8-nitroguanosine 3',5'-cyclic monophosphate. *Nat. Chem. Biol.* 3, 727–735.

Seglen, P.O., Gordon, P.B. & Holen, I. (1990) Non-selective autophagy. *Semin. Cell Biol.* 1, 441–448.

Sekhar, K.R., Rachakonda, G. & Freeman, M.L. (2010) Cysteine-based regulation of the CUL3 adaptor protein Keap1. *Toxicol. Appl. Pharmacol.* 244, 21–26.

Sekhar, K.R., Yan, X.X. & Freeman, M.L. (2002) Nrf2 degradation by the ubiquitin proteasome pathway is inhibited by KIAA0132, the human homolog to INrf2. *Oncogene* 21, 6829–6834.

Shibata, T., Kokubu, A., Gotoh, M., Ojima, H., Ohta, T., Yamamoto, M. & Hirohashi, S. (2008a) Genetic alteration of Keap1 confers constitutive Nrf2 activation and resistance to chemotherapy in gallbladder cancer. *Gastroenterology* 135, 1358–1368.

Shibata, T., Ohta, T., Tong, K.I., Kokubu, A., Odogawa, R., Tsuta, K., Asamura, H., Yamamoto, M. & Hirohashi, S. (2008b) Cancer related mutations in NRF2 impair its recognition by Keap1-Cul3 E3 ligase and promote malignancy. *Proc. Natl Acad. Sci. USA* 105, 13568–13573.

Singh, A., Boldin-Adamsky, S., Thimmulappa, R.K., Rath, S.K., Ashush, H., Coulter, J., Blackford, A., Goodman, S.N.,

Bunz, F., Watson, W.H., Gabrielson, E., Feinstein, E. & Biswal, S. (2008) RNAi-mediated silencing of nuclear factor erythroid-2-related factor 2 gene expression in non-small cell lung cancer inhibits tumor growth and increases efficacy of chemotherapy. *Cancer Res.* **68**, 7975–7984.

Singh, A., Misra, V., Thimmulappa, R.K., Lee, H., Ames, S., Hoque, M.O., Herman, J.G., Baylin, S.B., Sidransky, D., Gabrielson, E., Brock, M.V. & Biswal, S. (2006) Dysfunctional KEAP1-NRF2 interaction in non-small-cell lung cancer. *PloS Med.* **3**, 1865–1876.

Strnad, P., Zatloukal, K., Stumptner, C., Kulaksiz, H. & Denk, H. (2008) Mallory-Denk-bodies: lessons from keratin-containing hepatic inclusion bodies. *Biochim. Biophys. Acta* **1782**, 764–774.

Sun, Z., Huang, Z. & Zhang, D.D. (2009) Phosphorylation of Nrf2 at multiple sites by MAP kinases has a limited contribution in modulating the Nrf2-dependent antioxidant response. *PLoS ONE* **4**, e6588.

Surh, Y.J., Kundu, J.K. & Na, H.K. (2008) Nrf2 as a master redox switch in turning on the cellular signaling involved in the induction of cytoprotective genes by some chemopreventive phytochemicals. *Planta Med.* **74**, 1526–1539.

Sussan, T.E., Rangasamy, T., Blake, D.J., *et al.* (2009) Targeting Nrf2 with the triterpenoid CDDO-imidazolide attenuates cigarette smoke-induced emphysema and cardiac dysfunction in mice. *Proc. Natl Acad. Sci. USA* **106**, 250–255.

Taguchi, K., Maher, J.M., Suzuki, T., Kawatani, Y., Motohashi, H. & Yamamoto, M. (2010) Genetic analysis of cytoprotective functions supported by graded expression of Keap1. *Mol. Cell. Biol.* **30**, 3016–3026.

Takahashi, T., Sonobe, M., Menju, T., Nakayama, E., Mino, N., Iwakiri, S., Nagai, S., Sato, K., Miyahara, R., Okubo, K., Hirata, T., Date, H. & Wada, H. (2010) Mutations in Keap1 are a potential prognostic factor in resected non-small cell lung cancer. *J. Surg. Oncol.* **101**, 500–506.

Tong, K.I., Katoh, Y., Kusunoki, H., Itoh, K., Tanaka, T. & Yamamoto, M. (2006a) Keap1 recruits Neh2 through binding to ETGE and DLG motifs: characterization of the two-site molecular recognition model. *Mol. Cell. Biol.* **26**, 2887–2900.

Tong, K.I., Kobayashi, A., Katsuoka, F. & Yamamoto, M. (2006b) Two-site substrate recognition model for the Keap1-Nrf2 system: a hinge and latch mechanism. *Biol. Chem.* **387**, 1311–1320.

Tong, K.I., Padmanabhan, B., Kobayashi, A., Shang, C., Hirotsu, Y., Yokoyama, S. & Yamamoto, M. (2007) Different electrostatic potentials define ETGE and DLG motifs as hinge and latch in oxidative stress response. *Mol. Cell. Biol.* **27**, 7511–7521.

Van Beveren, C., van Straaten, F., Curran, T., Muller, R. & Verma, I.M. (1983) Analysis of FBj-MuSV provirus and c-fos (mouse) gene reveals that viral and cellular fos gene products have different carboxy termini. *Cell* **32**, 1241–1255.

Wakabayashi, N., Dinkova-Kostova, A.T., Holtzclaw, W.D., Kang, M.I., Kobayashi, A., Yamamoto, M., Kensler, T.W. & Talalay, P. (2004) Protection against electrophile and oxidant stress by induction of the phase 2 response: fate of cysteines of

the Keap1 sensor modified by inducers. *Proc. Natl Acad. Sci. USA* **101**, 2040–2045.

Wakabayashi, N., Itoh, K., Wakabayashi, J., Motohashi, H., Noda, S., Takahashi, S., Imakado, S., Kotsuji, T., Otsuka, F., Roop, D.R., Harada, T., Engel, J.D. & Yamamoto, M. (2003) Keap1-null mutation leads to postnatal lethality due to constitutive Nrf2 activation. *Nat. Genet.* **35**, 238–245.

Wang, J.S., Shen, X., He, X., Zhu, Y.R., Zhang, B.C., Wang, J.B., Qian, G.S., Kuang, S.Y., Zarba, A., Egner, P.A., Jacobson, L.P., Munoz, A., Helzlsouer, K.J., Groopman, J.D. & Kensler, T.W. (1999) Protective alterations in phase 1 and 2 metabolism of aflatoxin B1 by oltipraz in residents of Qidong, People’s Republic of China. *J. Natl Cancer Inst.* **91**, 347–354.

Wang, R., An, J., Ji, F., Jiao, H., Sun, H. & Zhou, D. (2008) Hypermethylation of the Keap1 gene in human lung cancer cell lines and lung cancer tissues. *Biochem. Biophys. Res. Commun.* **373**, 151–154.

Wang, X.J., Hayes, J.D. & Wolf, C.R. (2006) Generation of a stable antioxidant response element-driven reporter gene cell line and its use to show redox-dependent activation of nrf2 by cancer chemotherapeutic agents. *Cancer Res.* **66**, 10983–10994.

Watt, R.A., Shatzman, A.R. & Rosenberg, M. (1985) Expression and characterization of the human c-myc DNA-binding protein. *Mol. Cell. Biol.* **5**, 448–456.

Xu, L., Wei, Y., Reboul, J., Vaglio, P., Shin, T.H., Vidal, M., Elledge, S.J. & Harper, J.W. (2003) BTB proteins are substrate-specific adaptors in an SCF-like modular ubiquitin ligase containing CUL-3. *Nature* **425**, 316–321.

Yamamoto, T., Suzuki, T., Kobayashi, A., Wakabayashi, J., Maher, J., Motohashi, H. & Yamamoto, M. (2008) Physiological significance of reactive cysteine residues of Keap1 in determining Nrf2 activity. *Mol. Cell. Biol.* **28**, 2758–2770.

Yates, M.S., Kwak, M.K., Egner, P.A., Groopman, J.D., Bodreddigari, S., Sutter, T.R., Baumgartner, K.J., Roebuck, B.D., Liby, K.T., Yore, M.M., Honda, T., Gribble, G.W., Sporn, M.B. & Kensler, T.W. (2006) Potent protection against aflatoxin-induced tumorigenesis through induction of Nrf2-regulated pathways by the triterpenoid 1-[2-cyano-3-, 12-dioxooleana-1,9(11)-dien-28-oyl]imidazole. *Cancer Res.* **66**, 2488–2494.

Yates, M.S., Tran, Q.T., Dolan, P.M., Osburn, W.O., Shin, S., McCulloch, C.C., Silkworth, J.B., Taguchi, K., Yamamoto, M., Williams, C.R., Liby, K.T., Sporn, M.B., Sutter, T.R. & Kensler, T.W. (2009) Genetic versus chemoprotective activation of Nrf2 signaling: overlapping yet distinct gene expression profiles between Keap1 knockout and triterpenoid-treated mice. *Carcinogenesis* **30**, 1024–1031.

Yoh, K., Itoh, K., Enomoto, A., Hirayama, A., Yamaguchi, N., Kobayashi, M., Morito, N., Koyama, A., Yamamoto, M. & Takahashi, S. (2001) Nrf2-deficient female mice develop lupus-like autoimmune nephritis. *Kidney Int.* **60**, 1343–1353.

Zatloukal, K., French, S.W., Stumptner, C., Strnad, P., Harada, M., Toivola, D.M., Cadrin, M. & Omary, M.B. (2007) From Mallory to Mallory-Denk bodies: what, how and why? *Exp. Cell Res.* **313**, 2033–2049.

K Taguchi et al.

Zhang, D.D. & Hannink, M. (2003) Distinct cysteine residues in Keap1 are required for Keap1-dependent ubiquitination of Nrf2 and for stabilization of Nrf2 by chemopreventive agents and oxidative stress. Mol. Cell. Biol. 23, 8137–8151.

Zhang, D.D., Lo, S.C., Cross, J.V., Templeton, D.J. & Hannink, M. (2004) Keap1 is a redox-regulated substrate adaptor protein for a Cul3-dependent ubiquitin ligase complex. Mol. Cell. Biol. 24, 10941–10953.

Zhang, J.Y., Chan, E.K.L., Peng, X.X. & Tan, E.M. (1999) A novel cytoplasmic protein with RNA-binding motifs is an autoantigen in human hepatocellular carcinoma. J. Exp. Med. 189, 1101–1110.

Zhang, P., Singh, A., Yegnasubramanian, S., Esopi, D., Kombairaju, P., Bodas, M., Wu, H., Bova, S.G. & Biswal, S. (2010) Loss of Kelch-like ECH-associated protein 1 function

in prostate cancer cells causes chemoresistance and radioresistance and promotes tumor growth. Mol. Cancer Ther. 9, 336–346.

Zhang, Y., Talalay, P., Cho, C.G. & Posner, G.H. (1992) A major inducer of anticarcinogenic protective enzymes from broccoli: isolation and elucidation of structure. Proc. Natl Acad. Sci. USA 89, 2399–2403.

Zhuang, M., Calabrese, M.F., Liu, J., Waddell, M.B., Nourse, A., Hammel, M., Miller, D.J., Walden, H., Duda, D.M., Seyedin, S.N., Hoggard, T., Harper, J.W., White, K.P. & Schulman, B.A. (2009) Structures of SPOP-substrate complexes: insights into molecular architectures of BTB-Cul3 ubiquitin ligases. Mol. Cell 36, 39–50.

Received: 19 September 2010  
Accepted: 24 October 2010
