
Molecular Microbiology (2011) 80(2), 350–363

doi:10.1111/j.1365-2958.2011.07598.x  
First published online 17 March 2011

# A novel amidotransferase required for lipoic acid cofactor assembly in Bacillus subtilis

Quin H. Christensen,$^{1,3}$ Natalia Martin,$^{4}$ Maria C. Mansilla,$^{4}$ Diego de Mendoza$^{4}$ and John E. Cronan$^{1,2,3*}$

Departments of $^{1}$Microbiology and $^{2}$Biochemistry, $^{3}$Chemistry Biology Interface Training Program, University of Illinois, Urbana, IL 61801, USA. $^{4}$Instituto de Biología Molecular y Celular de Rosario (IBR-CONICET), Facultad de Ciencias Bioquímicas y Farmacéuticas, Universidad Nacional de Rosario, Suipacha 531, 2000 Rosario, Argentina.

## Summary

In the companion paper we reported that *Bacillus subtilis* requires three proteins for lipoic acid metabolism, all of which are members of the lipoate protein ligase family. Two of the proteins, LipM and LplJ, have been shown to be an octanoyltransferase and a lipoate : protein ligase respectively. The third protein, LipL, is essential for lipoic acid synthesis, but had no detectable octanoyltransferase or ligase activity either *in vitro* or *in vivo*. We report that LipM specifically modifies the glycine cleavage system protein, GcvH, and therefore another mechanism must exist for modification of other lipoic acid requiring enzymes (e.g. pyruvate dehydrogenase). We show that this function is provided by LipL, which catalyses the amidotransfer (transamidation) of the octanoyl moiety from octanoyl-GcvH to the E2 subunit of pyruvate dehydrogenase. LipL activity was demonstrated *in vitro* with purified components and proceeds via a thioester-linked acyl-enzyme intermediate. As predicted, ΔgcvH strains are lipoate auxotrophs. LipL represents a new enzyme activity. It is a GcvH:[lipoyl domain] amidotransferase that probably uses a Cys-Lys catalytic dyad. Although the active site cysteine residues of LipL and LipB are located in different positions within the polypeptide chains, alignment of their structures show these residues occupy similar positions. Thus, these two homologous enzymes have convergent architectures.

Accepted 13 February, 2011. *For correspondence. E-mail j-cronan@life.uiuc.edu; Tel. (+1) 217 333 7919; Fax (+1) 217 244 6697.*

© 2011 Blackwell Publishing Ltd

## Introduction

Comparative genomics allows for detection of incomplete metabolic models. Analysis of these discrepancies in our knowledge provides opportunities for new discoveries. Here we describe the analysis of an ‘extra’ protein required for lipoic acid metabolism in *Bacillus subtilis*, homologues of which are found in nearly all sequenced Firmicutes genomes. The Firmicutes, also known as the low G+C Gram-positive bacteria, are a phylum present throughout the environment that includes many pathogens. *B. subtilis* is the best characterized of this group and has been subject to intense genetic study since the discovery of its natural competence in 1958 (Spizizen, 1958). *B. subtilis* is a facultative spore-forming bacterium that grows in chemically defined medium lacking lipoic acid (Spizizen, 1958). In general, lipoic acid is required for aerobic, energy-conserved metabolism of 2-oxoacids and glycine; pyruvate dehydrogenase being the canonical lipoate-requiring enzyme (Perham, 2000). In *B. subtilis*, pyruvate dehydrogenase activity is required for growth under both aerobic and anaerobic conditions (Nakano *et al.*, 1997) and thus it is clear that *B. subtilis* must synthesize the cofactor.

The current model for lipoic acid metabolism is based on *Escherichia coli* (Cronan *et al.*, 2005; Rock, 2009). Enzymes that require lipoic acid for activity become modified by covalent attachment of the coenzyme to the ε-amino group of a specific lysine residue of conserved ~80 residue domains called lipoyl domains (LDs). Lipoate biosynthesis begins by transfer of octanoate from octanoyl-acyl carrier protein (ACP), a transient intermediate of fatty acid biosynthesis, to the LD by the ACP : LD octanoyltransferase LipB (Zhao *et al.*, 2003). After the LD has been octanoylated, hydrogen atoms attached to the octanoate C6 and C8 carbons atoms are replaced with two sulphur atoms by lipoyl synthase, an S-adenosyl-L-methionine radical enzyme (Booker *et al.*, 2007) to give lipoic acid. Exogenous free lipoic acid can also be scavenged by lipoyl ligase, LplA (Green *et al.*, 1995; Morris *et al.*, 1995), which is also active with octanoate (Green *et al.*, 1995; Jordan and Cronan, 1997). Octanoate can also come from intracellular sources and thereby bypass a biosynthesis requirement at the expense of ATP (Hermes and Cronan, 2009).
The LD modifying enzymes are grouped with those that modify biotinyl domains into Pfam PF03099. There are very few residues that are absolutely conserved within the family, a lysine residue being the sole conserved residue (Reche, 2000). However, there is sufficient transitive homology throughout the sequences to allow grouping (Reche, 2000). Despite this lack of sequence conservation the enzymes have considerable structural similarity including a conserved folding pattern (Ma *et al.*, 2006). Lipoyl ligases have an additional accessory domain, which has been shown to be required for adenylation but not for acyltransfer (Christensen and Cronan, 2009). Comparisons of the crystal structures of LplA obtained with and without the acyl adenylate intermediate show that the accessory domain rotates away from the active site following adenylate formation, thereby allowing interaction with the substrate LD (Fujiwara *et al.*, 2010). Interestingly, many proteins of the LplA family lack a C-terminal accessory domain. This can be due to the domain being a separate protein, as is the case in archaea (Christensen and Cronan, 2009). More recently, one of these single domain LplA homologues, *B.* subtilis LipM, has been shown to be a new ACP:GcvH octanoyltransferase isozyme (Christensen and Cronan, 2010).

GcvH is a lipoyl acceptor protein that consists of a single LD and is a key component of the glycine cleavage system. Although the glycine cleavage system is lipoic acid-dependent, its inactivation is not known to cause a growth defect in any bacterium. The glycine cleavage system is well characterized in *Arabidopsis* and other plants, where it is known as glycine decarboxylase (Bauwe and Kolukisaoglu, 2003). There it serves a vital role in the C2 pathway (dark cycle), where it concentrates the carbon dioxide released by glycine degradation. In *E.* coli the glycine cleavage system is one of two synthetic routes to serine and 5, 10-methylenetetrahydrofolate. Disruption of the glycine cleavage system in *E.* coli has no growth phenotype, although *gcv* mutants accumulate and excrete glycine (Plamann *et al.*, 1983). Disruption results in an inability to make serine from glycine, which creates a serine auxotrophy in the presence of glycine when the primary serine synthetic pathway is inactivated as well (Vanden Boom *et al.*, 1991). The well-characterized glycine cleavage systems consist of four proteins, GcvH, GcvT, GcvP and GcvL. In *B.* subtilis expression of only the T and P subunits is under control of a glycine-activated riboswitch (Mandal *et al.*, 2004). No dedicated GcvL subunit is apparent, but this function (oxidation of the lipoate moiety) may be provided by the E3 subunit that is shared among the 2-oxoacid dehydrogenases, as is the case in *E.* coli (Steiert *et al.*, 1990). Given its key role in transfer of the methylamine group of glycine from the P subunit to the T subunit, it is surprising that GcvH is not encoded together with the T and P subunits, but rather

at a remote genomic location where no recognizable riboswitch is detected (Mandal *et al.*, 2004). Recently, the glycine cleavage system H protein of yeast, Gcv3, has been shown to be required for lipoic acid biosynthesis and RNA processing (Schonauer *et al.*, 2009). In this paper we demonstrate that *B.* subtilis GcvH is both a substrate and a required component of the lipoic acid biosynthesis pathway.

Based on the *E.* coli model, the characterized LipA lipoyl synthase (Martin *et al.*, 2009) and the newly described LipM octanoyltransferase (Christensen and Cronan, 2010) should suffice for lipoic acid synthesis and attachment in *B.* subtilis. However, as reported in the companion paper (Martin *et al.*, 2011) a third gene, *lipL*, is also required for LD modification. LipL belongs to the Pfam PF03099 family and lacks an accessory domain. In this paper we describe the novel biosynthetic requirement for LipL and the unexpected reaction catalysed by this new enzyme. Based on these data we present a novel pathway for lipoic acid biosynthesis.

### Results

#### Expression, purification and modification of LipL

LipL (YwfL) was readily expressed in *E.* coli and purified as a soluble hexahistidine-tagged protein that lacked precedent catalytic activities (Martin *et al.*, 2011). *Mycobacterium tuberculosis* LipB and *B.* subtilis LipM have been shown to become modified by attachment of acyl moieties in *E.* coli (Ma *et al.*, 2006; Christensen and Cronan, 2010), so we wanted to determine if LipL also became modified. To this end the protein was subjected to trypsin digestion and the resulting peptides were analysed by liquid chromatography tandem mass spectrometry (LC-MS/MS). LipL was unusually amenable to this analysis, giving 88% coverage (Fig. 1A). Two sites of modification were found: cysteine residues C39 and C150. Lipoyl and octanoyl modifications were present at both sites (Fig. 1B) indicating that LipL accepted octanoate and lipoate from *E.* coli sources. Assuming that the reaction catalysed by this enzyme remains the same in *B.* subtilis, the substrates must be common to both organisms. The two modified cysteine residues are located on opposite sides of the LipL active site cleft (see below). Unlike LipM and *M.* tuberculosis LipB, no decanoyl Michael adduct was found when LipL was expressed in a wild-type strain of *E.* coli.

#### LipM is specific for GcvH

Although LipM readily catalysed octanoyltransfer from ACP to GcvH (Christensen and Cronan, 2010), we were puzzled by the finding that the LD of the *B.* subtilis pyru-
A

| 1 | MANQPIDLLM | QPKWRVIDQS | SLGPLFDAKQ | SFAMDDTLCM | SVGKGVSPAT |
| --- | --- | --- | --- | --- | --- |
| 51 | ARSWVHHDTI | VLGIQDTRLR | FLQDGISLLE | SEGYRVIVRN | SGGLAVVLDD |
| 101 | GVLNISLIFE | DEKKGIDIDK | GYEAMVELMR | RMLRPYNAKI | EAYEIEGSYC |
| 151 | PGSYDLSING | KK FAGISQRR | VRGGVAVQIY | LCADKSGSER | ADLIRR FYQA |
| 201 | ALKDKQN DKK | GVYPEIRPET | MASLSELLQK | DISVQDLMFA | LLTELKALST |
| 251 | HLYSAGLSID | EEMEFEKNLV | RMAERNAKVF | G |  |

B

| Peptide Sequence | Residue ΔMass | Modification | Score |
| --- | --- | --- | --- |
| QS FAMDDTLCMSVGK | C39 | 0 | None | 60 |
| QS FAMDDTLCMSVGK | C39 | 188 | Lipoylation | 34 |
| QS FAMDDTLCMSVGK | C39 | 126 | Octanoylation 41 |  |
| IEAYEIEG SYCPGSYDLSINGK | C150 | 188 | Lipoylation | 57 |
| IEAYEIEG SYCPGSYDLSINGK | C150 | 126 | Octanoylation 58 |  |

Fig. 1. A. Sequence coverage of LipL by LC-MS/MS analysis of LipL derived peptides. The LipL peptide sequences detected are underlined.

B. Modifications of LipL detected after trypsin digestion followed by LC-MS/MS analysis. The theoretical peptides are shown with the modified cysteine residues in bold and underlined. The mass differences of the modification are given. The ion score is equal to –10log(P) where P is the probability the result is random chance.

vate dehydrogenase PdhC subunit (E2<sub>PdhC</sub>) was a poor octanoyl acceptor substrate (Fig. 2A). This was first considered to be an artefact of use of overly truncated LDs. However, the full-length PdhC subunit was also a poor substrate. PdhC was purified in good yield and eluted slightly before catalase upon size exclusion chromatography consistent with an octahedral complex (data not shown). This indicates that the C-terminal hexahistidine tag does not interfere with multimerization of the protein. Moreover, *E. coli* LipB was able to modify all of the *B. subtilis* LDs tested indicating that the acceptor proteins were properly folded. However, the enigmatic biotin lipoyl acceptor protein (BLAP), a protein previously reported to be biotinylated and lipoylated when expressed in *E. coli* (Cui et al., 2006) (Fig. 2) was not modified by any enzyme tested.

These data led to the hypothesis that LipL is essential for PdhC modification and hence growth. To test this scenario octanoyltransfer assays were done with octanoyl-ACP (generated *in situ*) as the acyl donor substrate (Christensen and Cronan, 2010). Similar to previous results (Fig. 2) addition of only LipM and LipL failed to result in octanoylation of either E2<sub>PdhC</sub> or LipL (Fig. 3A, lane 3). It was only in the presence of GcvH that octanoylation of LipL and E2<sub>PdhC</sub> proceeded (Fig. 3A, lane 6).

A similar phenomenon was also seen for the LplJ, which was identified and demonstrated to be a lipoyl ligase in the companion paper (Martin et al., 2011). Although LplJ was able to ligate octanoate to the *E. coli* LD (E2<sub>AceF</sub>), no modification of either *B. subtilis* E2<sub>PdhC</sub> or of full-length *B. subtilis* PdhC was seen *in vitro* (Fig. 2B). Addition of LipL to the LplJ reaction failed to give modification of E2<sub>PdhC</sub> (Fig. 3B, lane 3). Only when GcvH, LipL and LplJ were all present did modification of E2<sub>PdhC</sub> proceed (Fig. 3B, lane 5). These observations argued that modified GcvH was an obligate intermediate in the modification of E2<sub>PdhC</sub>. It also showed that LipL was required for PdhC modification.

Fig. 2. Specificity of *B. subtilis* LipM, *B. subtilis* LplJ and *E. coli* LipB for various LDs. In panels A to C the LD tested as a substrate is indicated at the top of each lane and by an arrow. The [1-<sup>14</sup>C]octanoyl-ACP substrate is indicated by an arrow. Autoradiograms of SDS-PAGE gel separations of LD substrate specificity assays are shown. Sodium [1-<sup>14</sup>C]octanoate was used as a substrate for LplJ. Panel A: LipM octanoyltransfer to LDs from [1-<sup>14</sup>C]octanoyl-ACP generated using AasS. Panel B: Ligation of [1-<sup>14</sup>C]octanoate by LplJ to LDs. Panel C: LipB octanoyltransfer to LDs from [1-<sup>14</sup>C]octanoyl-ACP generated using *Vibrio harveyi* acyl-ACP synthetase (AasS). The two E2 domains migrate similarly and are denoted by E2s.

LipL is a novel amidotransferase

Because the octanoyl moiety of octanoyl-GcvH is attached by an amide linkage, LipL seemed to catalyse a novel amidotransfer reaction in which the active site cysteine thiol of LipL would attack the octanoyl-GcvH amide linkage to form octanoyl-S-LipL and would subsequently transfer the thioester-bound octanoyl moiety to E2<sub>PdhC</sub> to give a new amide-linked moiety (Fig. 4D). To test if LipL

was a indeed a GcvH : E2 amidotransferase, LipL activity was assayed using purified [1-¹⁴C]octanoyl-GcvH as the donor substrate. Transfer of the octanoyl moiety to each of the PDH domains tested was observed whereas BLAP was not modified (Fig. 4A). Transfer from GcvH to LipL was also observed (Fig. 4C).

To explore the LipL reaction mechanism site-directed mutagenesis of residues conserved in the PF03099 family was performed. LipM K162 is a strictly conserved residue in PF03099 whereas Y180 is conserved in the LipL amidotransferase subfamily (Fig. 5). Unfortunately, when expressed in *E. coli* both K162 substitutions (K162R and K162A) resulted in low levels of insoluble proteins whereas substitutions of alanine or phenylalanine for Y180 had no effect on catalysis. We also mutated residues C39 and C150, both of which had been found to be modified by LC-MS/MS (Fig. 1B). Replacement of C39 with alanine or serine had no effect on catalysis whereas replacement of C150 with either alanine or serine resulted in loss of transfer to E2<sub>Pdhc</sub> (Fig. 4B) and the inability to form an octanoyl-enzyme intermediate (Fig. 4C). Given the above observations, we propose a mechanism for amidotransfer by LipL (Fig. 4D).

Amidotransfer was also assayed in crude extracts of various *B. subtilis* mutant strains. Although LipM activity was observed, no amidotransfer by LipL was seen unless the extracts were supplemented with purified LipL and LipM (Fig. 6). The reason for this discrepancy is not clear. Perhaps higher concentrations of octanoyl-GcvH are required to detect LipL activity, although transfer was not seen using purified [1-¹⁴C]octanoyl-GcvH as a substrate (data not shown). Additionally, LipL may be present in low amounts under the growth conditions used or LipL may become inactivated during extract preparation.

Fig. 3. A. Component controls for a coupled AasS/LipM/LipL assay starting with [1-¹⁴C] octanoate which becomes ligated to ACP by AasS. Following the reaction the proteins were resolved by SDS-PAGE and visualized by autoradiography. The identities of the [1-¹⁴C] octanoylated proteins are indicated. B. Component controls for a coupled LplJ plus LipL assay as in Panel A. The starting substrates for LplJ-catalysed ligation reactions were [1-¹⁴C] octanoate and ATP.

Fig. 4. Autoradiograms of SDS-PAGE gels of assays for LipL-catalysed amidotransfer from purified [1-¹⁴C]octanoyl-GcvH to LDs. A. Amidotransfer of the [1-¹⁴C]octanoyl moiety from purified octanoyl-GcvH to the unmodified LD indicated. Each reaction contained purified wild-type (WT) LipL. The two E2 domains migrate similarly and are denoted by E2s. B. Amidotransfer from purified [1-¹⁴C]octanoyl-GcvH to the E2<sub>Pdhc</sub>. The purified wild-type LipL or point mutant proteins indicated were used as enzyme sources. C. Additional enzyme was added to allow detection of the octanoyl-LipL intermediate. Wild-type (WT), C150A point mutant, C150S point mutant LipL were assayed in addition to a control (NE) lacking LipL. D. Schematic of the LipL amidotransfer reaction is shown with the acyl-LipL intermediate.

© 2011 Blackwell Publishing Ltd, Molecular Microbiology, 80, 350–363

LipL_BACSU/32-277 32 FAMDD--TLCMSVGKGVSPATARSWVHHDTIVLGIQDTRLPFLQDGISLLESEGYRIVRNSG 92
LipL_BACHD/31-273 31 FAYDD--TLCTSIGKSQSPPTLRAWHHNTWVLGIQDSRLPQIKAGIEALKGFQHDVIVRNSG 91
LipL_LISMO/31-274 31 FATDD--TLCRSVGARMAPSTVRGWVHEKTWSLGIQDSKLPDIDKGIAFLQKQGYVVVRNSG 91
LipL_LACBA/31-257 31 ALLDLVA---------ELKQPIILHFWTMHPTVIMGLKDKRLPDLTAAIKAVQRYGYDYVLRNSG 85
LipL_STRPC/34-278 34 FVWTE--VFLKTINQEPNQLILHIWPMTRTVILGMLDRQLPYFELAKKEIGNNGYVPVTRNIG 94
LipL_BACSU/32-277 93 GLAVVLDDGVLNISLIFED--EKKGIDIDKGYEAMVELMRRMLRPYNAKIEAYEIEGSSYCPGS 153
LipL_BACHD/31-273 92 GLAVVLDSGILNLSLVLKE--EK-GFSIDDGYELMYELICSMFQDHREQIEAREIVGSSYCPGS 151
LipL_LISMO/31-274 92 GLAVVLDSGVLNLSMVLPD--AERGIAIERGYETMFTLIKDMFVDNEVIEAKEIEDSYCPGS 152
LipL_LACBA/31-257 86 GLAVVSDAGILNVSLFAPL--TTPPMSVDAAYAQMITALVDAAWPELA--IQHYKITHSSYCPGD 144
LipL_STRPC/34-278 95 GLAVVADDGILNFSLVIPDHFSE-SISISNAYLIMDVIRESFSDYYQRIEYHEIKNSYCPGN 156
LipL_BACSU/32-277 154 YDLSINGKKFAGISQRRVRGGVAVIYLICADKSGSERADLIIRRFYQAALKDKQNdkKGVYPEI 216
LipL_BACHD/31-273 152 YDLSIDGKKFAGISQRRIRGGVAVIYLCSVSGSGAERAKMIRTFYDKAVAGQPT--KFVYPRI 212
LipL_LISMO/31-274 153 YDLSIQGKKFAGISQRRMAKGVAVIYLAIDGDQTTRSELIRDIFYTISGKAKQT--KYTFPDV 213
LipL_LACBA/31-257 145 YDLSVNGLKIAQIAGIAQRRSPHALVTMLYLSVNGDQLARGQMIRDFYTAGLAGQEN--TWAFPDV 205
LipL_STRPC/34-278 157 FDLSIAGRKFAGIAQRRRIKKGIVVSIIYLSVCGDQAARGQLIKAFYEAGTQGEVT--KVNYPOI 217
LipL_BACSU/32-277 217 RPETMASLSELLQKDISVQDLMFALLTELKALSTHLYSAGLSIDEEMEFEEKNLVRMAERNA 277
LipL_BACHD/31-273 213 KPETMASLSELLGQPHNVSDVLLKALMTLQQHGASLLTESLSADEWLLYEQHFARISERNE 273
LipL_LISMO/31-274 214 NPNVMGSLSDLMKNDISLNGTLVRLFNSLRHYAGDLVSIGTLTSEELDFPAYYERLIARND 274
LipL_LACBA/31-257 206 DPATMTTTTAALLGQNLTIADAQQRFITAAEQAGLNVGRTTLAQAMATPEFG---------- 256
LipL_STRPC/34-278 218 DPECMATLSELLETPFTVAEVLERLRLTLRQLGFSLTEKSPDQALLTNDAVYERMQLEvv 278

Fig. 5. Alignment of members of the LipL clade. Positions having 50% or greater similarity are highlighted in grey. The catalytic cysteine residue (C150 in *B. subtilis* LipL), the other modified cysteine residue (C39 in *B. subtilis* LipL) and the conserved PF03099 lysine residue are highlighted in black. BACSU is *B. subtilis* 168, BACHD is *Bacillus halodurans*, LISMO is *Listeria monocytogenes*, LACBA is *Lactobacillus brevis* and STRPC is *Streptococcus pyogenes* serotype M12 (strain MGAS9429). This alignment is part of the larger one used to create Fig. 8A.

---

**GcvH is required for protein lipoylation in vivo**

Our *in vitro* results argued that GcvH was an essential intermediate in lipoylation of PdhC and possibly BkdB. To test the model for lipoic acid biosynthesis in *B. subtilis*, we disrupted *gcvH*. The Δ*gcvH* strain was auxotrophic for lipoic acid, the same phenotype shown by Δ*lipA* and Δ*lipM* strains. As observed in the other lipoate auxotrophs, the lipoate requirement of the Δ*gcvH* strain was bypassed by addition of acetate and branched chain fatty acid precursors. *In vivo* results showing that bypass of a Δ*lipL* mutation requires both acetate and branched chain fatty acid precursors indicates that LipL

![Figure 6](attachment:figure_6.png)

Fig. 6. A. Autoradiograph of an SDS-PAGE gel of octanoyltransfer assays using AasS generated [1-¹⁴C]octanoyl-ACP as a substrate. Extracts of the mutant strains indicated were used as a source of enzyme for transfer to GcvH and PdhC. All strains also carried a disruption in *iplJ*. When purified LipM and LipL were added to replace the missing protein(s) this is indicated with a plus sign.

Fig. 7. Growth and lipoylation phenotypes of a *B. subtilis* ΔgcvH strain. Growth of strains on minimal glucose media plates with the supplements indicated after 48 h at 37°C. Petri plates sectored by plastic walls were used to prevent cross-feeding.

A. Growth of strains JH642 (wild type) and NM20 (ΔgcvH).

B. Growth of strains JH642 (wild type) and NM21 (ΔgcvH amyE:: Pspac-gcvH).

C. Lipoylation of GcvH in *B. subtilis* crude extracts. SDS-PAGE and immunoblotting with anti-LA antibody was done as described in Experimental procedures. Cells were grown 22 h in minimal media with or without lipoate added as indicated. Crude extracts of *B. subtilis* wild type (WT) and ΔgcvH strains were used as indicated.

in a complete loss of biosynthetic lipoic acid assembly (Fig. 7C), confirming that GcvH was required in *B. subtilis* lipoic acid biosynthesis. Upon supplementation with lipoic acid the ΔgcvH strain showed strong lipoylation of the dehydrogenase E2 subunits (Fig. 7C) consistent with the robust growth seen in this medium (Fig. 7A). These results confirm the physiological relevance of the in vitro biochemical assays and show GcvH is required not only for biosynthetic lipoylation of PdhC, but other E2 subunits as well.

**LipL and LipB are mechanistically convergent**

Phylogenetic analysis of LipL compared with other members of PF03099 shows significant, albeit modest, support for *B. subtilis* LipL as part of a deeply branching clade related to lipoate protein ligases (Fig. 8A). LipL is also closely related to LipM octanoyltransferases, which branch off later from the same lipoate protein ligase clade (Christensen and Cronan, 2010). Indeed, the unpublished *Bacillus halodurans* LipL structure (PDB entry 2P5I) is the closest available structure to LipM. The positions of the catalytic cysteine residues (C150 in both proteins) are very similar in LipM and LipL. Upon structural alignment of LipL with *M. tuberculosis* LipB (PDB 1W66) the catalytic cysteine residues are located only 2.34 Å apart (Fig. 9). This was unexpected because the residues are contributed by different loops that are ~45 residues removed in sequence alignments (Christensen and Cronan, 2010). Therefore, it appears that LipL and LipB have undergone mechanistic convergence within their active sites using the same protein scaffold.

**Discussion**

We report delineation of a new pathway of lipoic acid synthesis in which octanoyl moieties are transferred from the ACP of fatty acid synthesis to the GcvH protein of the glycine cleavage system by LipM. The octanoyl moieties are then transferred by LipL, a novel GcvH : E2 amidotransferase, from GcvH to E2 subunits to form active dehydrogenase complexes (Fig. 10). Note that at this stage of our investigations it is not clear at what step(s) LipA acts. It is possible that LipA inserts the sulphur groups before LipL transfer, after LipL transfer or at either step. Note that although we characterized the *B. subtilis* pathway as a result of the sophisticated genetics available in this bacterium, paralogous genes are present in other Firmicutes proficient in lipoic acid synthesis.

The requirement for LipL is due to the specificity of the LipM for GcvH. We find significant in vitro modification of the E2<sub>AceF</sub> by LplJ, but not by LipM (Fig. 2) suggesting LplJ may be less specific than LipM. However, LipM complements an *E. coli* lipB lipoic acid auxotroph (Christensen and Cronan, 2010), which demonstrates modification of E2<sub>AceF</sub>. These data indicate that restoration of growth is a much more sensitive assay than the in vitro radiolabel assay. This is expected because growth results from protein lipoylation that in turn results in dehydrogenase catalysis with a concomitant amplification of the signal. Another difference is that lipoylation of PdhC may occur in vivo after the protein has been assembled into the PDH complex rather than as the free subunit we tested in vitro. The E2<sub>AceF</sub> and E2<sub>PdhC</sub> domains clearly form different clades indicating different conserved residues (Fig. 8B).

© 2011 Blackwell Publishing Ltd, *Molecular Microbiology*, **80**, 350–363

A


97 LipM SYNY3/1-182
65 LipM NODSP/28-206
50 LipM PROMS/20-177
90 LipM BACSU/20-208
LipM CLOD6/18-202
65 LipM THEMA/13-183
92 LplA STRPN/15-183
LplJ BACSU/16-184
76 LplA MYCGE/17-188
76 LplA BDEBA/18-187
94 LplA ECOLI/18-187
93 LipL LACBA/31-257
96 LipL STRPC/34-278
77 LipL BACHD/31-273
74 LipL LISMO/31-274
99 LipB MYCTU/27-185
99 LipB CAUCR/25-224
71 LipB ECOLI/21-178
79 LipB PSYCK/29-229
95 BirA ECOLI/10-258
85 BirA BACSU/2-266
85 BirA ENTFC/18-329


---

B


99 E2p1 ECOLI/4-81
83 E2p1 VIBHB/4-81
95 E2p HAEIN/4-81
E2p AZOVD/7-80
63 E2p1 PSEPF/4-81
96 E2p PSEAE/4-81
64 E2p LISMO/4-83
95 E2p BACAN/4-83
100 E2p ENTFA/4-83
E2p STAAW/4-83
92 E2o AZOVD/4-83
100 E2o PSEAE/4-83
E2o PSEPF/4-83
86 E2o HAEIN/4-84
E2o VIBHB/4-83
E2o ECOLI/5-84
86 E2o BACAN/3-82
64 E2o BACSU/3-82
E2o STAAW/3-82
100 GcvH BACSU/24-105
89 GcvH ECOLI/26-107
GcvH LISMO/24-105


---

Fig. 8. Minimum evolution analysis of alignments with bootstrap percentage values shown for each branch. The trees are drawn to scale, with branch lengths in the same units as those of the evolutionary distances used to infer the phylogenetic tree. The scale represents a 50% difference in compared residues per length. Branches representing *Bacillus* proteins are indicated in bold.

A. Analysis of selected catalytic domains from PF03099; the structurally related biotin ligases are used as the out-group.

B. Selected LDs involved in lipoic acid metabolism are shown.

This may explain why LplJ is able to modify E2<sub>AceF</sub> but not E2<sub>Pdhc</sub>. Identification of the residues responsible for the observed differences in substrate recognition will require further study.

The amidotransfer reaction (also called transamidation) catalysed by LipL was unexpected, but not unprecedented. The sortase transpeptidases that anchor proteins to the outer surfaces of various Gram-positive bacteria use a Cys-His-Asn triad in their amidotransfer reaction (Clancy et al., 2010). Moreover, peptide bond synthesis catalysed by cysteine proteases (Morihara and Oka, 1981) uses this same catalytic triad. The DD-transpeptidase/carboxypeptidases of bacterial cell wall biosynthesis use a Ser-Lys-Tyr catalytic triad (Silvaggi et al., 2003) and substitution of cysteine for the nucleophilic serine residue in closely related enzymes often results in active enzymes (Hadonou et al., 1992).

In our work, LipL residues C39 and C150 were found to be acylated. However, C39 was not essential for catalysis (Fig. 4B) and is not conserved in more diverse members of the LipL clade (Fig. 5). (Acylation of C39 may have resulted by trans-thioesterification from C150 during the overnight trypsin digestion). Mutagenesis of LipL C150 resulted in loss of enzymatic activity and the inability to form an acyl-enzyme intermediate (Fig. 4), which is the same result seen upon mutagenesis of the

active site LipM cysteine residue (Christensen and Cronan, 2010). Based on the *B. halodurans* LipL crystal structure K162 and Y180 are less than 3 Å from C150 (Fig. 9). We found that these residues are conserved in all members of the LipL clade (Figs 5 and 8). From these data, it seemed likely that LipL used a Cys-Lys-Tyr catalytic triad similar to those seen in other amidotransferases. However, we found that Y180 was not required for amidotransfer as the Y180F and Y180A proteins were active. Therefore, we propose that LipL amidotransferases utilize a Cys-Lys dyad as do the LipB and LipM ACP-octanoyltransferases (Ma et al., 2006; Christensen and Cronan, 2010). If so, this would be a novel active site configuration for an amidotransferase (transamidase). Note that LipL shares no detectable overall sequence or structural similarity to other known amidotransferases.

The *B. subtilis* BLAP was reported to be at least partially modified with lipoate when expressed in *E. coli* (Cui et al., 2006). In that work the concentration of lipoic acid added to the culture medium corresponded to > 60 000-fold excess over that needed for half-maximal growth of a *lipA* strain (Reed and Cronan, 1993). As previously reported (Cui et al., 2006), we found that most of the BLAP was in the unmodified form after 2 h of expression. It was thought that BLAP might be a good candi-

unknown, but it does not appear to be involved in lipoic acid metabolism.

Although the data presented in these papers provide a good working model of lipoic acid biosynthesis, a conflicting view of lipoic acid scavenging emerges. As shown in the companion paper, addition of external lipoic acid only partially restored growth of a ΔlipL mutant and modification of E2<sub>PdhC</sub> by LplJ was not seen *in vitro* (Fig. 2B) whereas growth of the ΔgcvH strain was fully restored by lipoic acid (Fig. 7A) and the dehydrogenase E2 subunits became highly modified (Fig. 7C). One possibility is LplJ activity may be lower in a *lipL* strain or higher in a *gcvH* strain. If there is an allosteric or regulatory effect, it would have to be large, as we find the phenotypes are the same over a wide range of lipoic acid concentrations (0.5 nM–0.5 mM) (data not shown). An alternative explanation would be that deletion of *lipL* results in global changes in gene expression making lipoyl scavenging insufficient to restore the required enzymatic activities. This may be why an unusual E2 lipoylation pattern was seen in the Western blot of a ΔlipL strain in the companion paper (Martin *et al.*, 2011).

A similar mechanism for lipoic acid synthesis based on genetic analyses has been proposed for the yeast Saccharomyces cerevisiae where it was found that the glycine cleavage system H protein Gcv3 is not only a target for lipoylation but also is necessary for the lipoylation of the other lipoate-modified proteins, the 2-oxoacid dehydrogenases (Schonauer *et al.*, 2009). These workers favour a model of complex formation and allosteric interactions, although a model in which Lip2, the yeast LipB homologue, transfers the octanoyl moiety from octanoyl-ACP to Gcv3, where two sulphur atoms are inserted into the acyl chain by Lip5 (the yeast LipA homologue) was subsequently proposed (Hiltunen *et al.*, 2010). The lipoyl moiety would then be transferred to the 2-oxoacid dehydrogenases by a LplA

date for intermediate carrier in lipoic acid biosynthesis. However, we assayed BLAP as an octanoyl acceptor and found it was not modified by any enzyme tested (Figs 2 and 4A). The role of BLAP in *B. subtilis* remains

---

**Fig. 9.** Structure of LipL and comparison with that of LipB.
A. The unpublished structure of *B. halodurans* LipL (PDB 2P5I). The modified cysteine residues, C39 and C150, detected by LC-MS/MS of the *B. subtilis* protein are shown in red.
B. Structural alignment of LipL from *B. halodurans* with *M.* tuberculosis LipB (PDB 1W66) (Ma *et al.*, 2006). The active site adduct is shown in purple.
C. Close-up view of the structural overlay with the LipB decanoyl adduct removed. The active site cysteine sulphur atoms are coloured orange whereas carbon atoms are white. The distance between the two sulphur atoms is 2.4 Å.
This figure is available in colour online at wileyonlinelibrary.com.

---

**Fig. 10.** Current models for lipoic acid biosynthesis in *E. coli* and *B. subtilis*. Lipoic acid synthesis in *E. coli* is accomplished by two enzymes whereas *B. subtilis* requires three enzymes. *B. subtilis* also requires GcvH as an intermediate carrier whereas *E. coli* does not. We lack data to determine if LipA acts before or after LipL. We propose that *B. subtilis*, LipA uses octanoyl-LD (both octanoyl-GcvH and octanoyl-E2) as substrates, and therefore can act before or after LipL.

© 2011 Blackwell Publishing Ltd, *Molecular Microbiology*, **80**, 350–363
homologue called Lip3. It is tempting to speculate that Lip3 might perform the same reaction as *B. subtilis* LipL.

It is not clear why *B. subtilis* routes the synthesis of the essential lipoate cofactor through GcvH. The obscure role of the glycine cleavage system in bacterial metabolism makes it amendable to other roles. It should be noted that GcvH LDs are highly divergent from the dehydrogenase E2 LDs (Fig. 8B). This divergence may explain how GcvH behaves as a distinct acceptor protein. In *B. subtilis*, strains carrying a *gcvH* disruption grow well when provided with lipoic acid or the products of the inactivated dehydrogenases (Fig. 7A), which indicates that GcvH does not play an essential role in amino acid or single carbon metabolism. In the companion paper we show that *E. coli lipB* complements both Δ*lipM* and Δ*lipL* mutant strains. Some evolutionary pressure may exist for *B. subtilis* to maintain the specificity of LipM for GcvH. However, instead of a phenomenon that can be rationalized, constructive neutral evolution (Gray *et al.*, 2010) may be involved. LipM may have coevolved with the glycine cleavage system and LipL provided a means for it to be used for other lipoate dependent enzymes. A more extensive bioinformatic analysis may reveal the ancestral states of the pathway.

*Bacillus subtilis* is predicted to encode all five known lipoic acid-dependent complexes; pyruvate dehydrogenase, 2-oxoacid dehydrogenase, branched-chain keto acid dehydrogenase, acetoin dehydrogenase and glycine cleavage system. Amidotransfer of octanoyl/lipoyl moieties may provide *B. subtilis* with the ability to quickly adapt to changing environments by transferring lipoic acid among different enzymes. Alternatively, the extra step in the biosynthetic pathway may provide additional opportunities for regulatory control. *B. subtilis* has previously been shown to utilize lipoic acid as a sulphur source (Mansilla and de Mendoza, 1997). Although it is unlikely that lipoic acid would be a significant source of sulphur in most environments, lipoylation could be used as an indirect readout of sulphur availability. No transcriptional regulation of lipoic acid biosynthesis has been found in *E. coli* (J.E. Cronan, unpublished) or *Salmonella enterica* (Smith *et al.*, 1991). However, *B. subtilis IplJ* has been shown to be induced during spore outgrowth (Keijser *et al.*, 2007) and *lipM* and *iplJ* are induced in response to valine (Ye *et al.*, 2009). Constitutive expression of GcvH would allow it to be always available to act as a lipoate transfer protein regardless of glycine supplementation. Clearly, there is much to be learned about the physiological and ecological implications of this pathway.

### Experimental procedures

#### Media and chemicals

Escherichia coli strains were grown on LB rich or M9 minimal media (Sambrook and Russell, 2001) under aerobic conditions at 37°C unless otherwise indicated. Antibiotics (Sigma Chemical) were used at the following concentrations for *E. coli* (in µg ml⁻¹): sodium ampicillin, 100; kanamycin sulphate, 50; spectinomycin sulphate, 50; chloramphenicol, 12; gentamycin sulphate, 25 and tetracycline hydrochloride, 12. Difco Vitamin-Assay Casamino Acids was obtained from Becton, Dickenson. [1-¹⁴C]Octanoic acid and [1-¹⁴C]octanoyl-CoA were purchased from Moravek. American Radiolabeled Chemicals provided [α-³²P]ATP. The pH of buffers and solutions is reported at room temperature.

#### Bacterial strains and plasmids

Strains and plasmids used are shown on Table 1. Primers used in this study are given in Table 2. Standard techniques for DNA manipulation and cloning were used unless otherwise indicated (Sambrook and Russell, 2001). PCR amplification was performed using Taq (New England Biolabs) or Pfu (Stratagene) polymerase according to the manufacturer’s recommendation, except with the addition of 5% dimethyl sulphoxide to Pfu reactions.

The genes encoding proteins containing LDs were placed under the control of a T7 promoter. The LD from *B. subtilis* PdhC (*E2*<sub>*PdhC*</sub>) was identified by alignment with the *Bacillus stearothermophilis* LD isolated by V8 protease digestion (Packman *et al.*, 1988) and this portion of the *pdhC* gene was PCR amplified using primers Q030 and Q031 (the latter primer added a termination codon). After digestion with Ncol and BamHI this product was inserted into pET28b+ cut with the same enzymes to give pQC023. The full-length pyruvate dehydrogenase E2p subunit (*pdhC*) was PCR amplified using primers Q095 and Q096 (which added a C-terminal hexahistidine tag) and inserted into pCR2.1 using the TA cloning kit. The *pdhC* insert was then moved into pRSF-1b using the Ncol and BamHI sites to give pQC047. The BLAP was PCR amplified using primers Q034 and Q042 (which added a C-terminal hexahistidine tag) and inserted into pCR2.1 using the TA cloning kit. The BLAP insert was then moved into pET28b+ using the Ncol and BamHI sites to give the pQC026.

In order to minimize proteolysis of the full-length PdhC, the protein was expressed in a strain lacking the OmpT protease, which cleaves the flexible linker region (Lessard *et al.*, 1998). The *ompT::Km* mutation was constructed as described using the primers described (Baba *et al.*, 2006) to give strain RMK98. Strain QC146 was transduced with a phage P1 stock grown on strain RMK98 to give strain QC171. The kanamycin cassette was removed (Datsenko and Wanner, 2000) to give strain QC172 and the deletion was confirmed by PCR using primers Q113 and Q114.

Mutants of LipL were produced by modification of pQC033 using QuickChange mutagenesis as per the manufacturers recommendations (Agilent). Primers were used for the corresponding mutations were: Q147/Q148 for K148A, Q149/ Q150 for K148R, Q151/Q152 for C150A, Q153/Q154 for C150R, Q171/Q172 for C39A, Q173/Q174 for C39S, Q175/ Q176 for Y180A, Q177/Q178 for Y180F. Mutant LipL proteins were expressed and purified as described in the companion paper (Martin *et al.*, 2011).

The *gcvH* deletion mutant, strain NM20, was obtained by transformation of strain JH642 with plasmid pNM67. This

© 2011 Blackwell Publishing Ltd, *Molecular Microbiology*, **80**, 350–363
Table 1. Strains and plasmids used in this study.

| Strain | Genotype | Source |
|--------|----------|--------|
| Escherichia coli: |  |  |
| MG1655 | rph-1 | CGSC |
| RMK98 | MC1061 fadK::his6-cat ΔompT::FRT | This study |
| QC146 | rph-1 ΔlplA::FRT ΔlipB::FRT | Christensen and Cronan (2009) |
| QC152 | rph-1 ΔlplA::FRT ΔlipB::FRT / pTARA, pQC026 | This study |
| QC172 | rph-1 ΔlplA::FRT ΔlipB::FRT ΔompT::FRT | This study |
| QC191 | rph-1 ΔlplA::FRT ΔlipB::FRT ΔompT::FRT / pQC047, pTARA | This study |
| QC222 | rph-1 IN(rrnD-rrnE)1 φ(fabA-lacZ)1(Hyb)cat / pQC077, pCY598 | This study |
| QC223 | rph-1 IN(rrnD-rrnE)1 φ(fabA-lacZ)1(Hyb)cat / pQC078, pCY598 | This study |
| QC224 | rph-1 IN(rrnD-rrnE)1 φ(fabA-lacZ)1(Hyb)cat / pQC079, pCY598 | This study |
| QC225 | rph-1 IN(rrnD-rrnE)1 φ(fabA-lacZ)1(Hyb)cat / pQC080, pCY598 | This study |
| QC255 | rph-1 IN(rrnD-rrnE)1 φ(fabA-lacZ)1(Hyb)cat / pQC086, pCY598 | This study |
| QC256 | rph-1 IN(rrnD-rrnE)1 φ(fabA-lacZ)1(Hyb)cat / pQC087, pCY598 | This study |
| QC257 | rph-1 IN(rrnD-rrnE)1 φ(fabA-lacZ)1(Hyb)cat / pQC088, pCY598 | This study |
| QC258 | rph-1 IN(rrnD-rrnE)1 φ(fabA-lacZ)1(Hyb)cat / pQC089, pCY598 | This study |
| Bacillus subtilis: |  |  |
| JH642 | trpC2 pheA1 | Laboratory stock |
| NM20 | JH642 gcvH:: Km | This study |
| NM21 | NM20 amyE::Pspac-gcvH | This study |

| Plasmids | Relevant properties | Source |
|----------|---------------------|--------|
| pTARA | pACYC origin, arabinose inducible T7 polymerase | Wycuff and Matthews (2000) |
| pCY598 | RSF origin, arabinose inducible T7 polymerase | Cronan (2003) |
| pCR2.1 | TOPO TA cloning vector | Invitrogen |
| pSJ120 | E. coli LipB expression vector | Jordan and Cronan (2003) |
| pQC023 | B. subtilis E2p LD expression vector | This study |
| pQC026 | C-terminal hexahistidine BLAP expression vector | This study |
| pQC033 | N-terminal hexahistidine LipL expression vector | Martin et al. (2011) |
| pQC047 | C-terminal hexahistidine PdhC expression vector | This study |
| pQC077 | N-terminal hexahistidine LipL K162R mutant expression vector | This study |
| pQC078 | N-terminal hexahistidine LipL K162A mutant expression vector | This study |
| pQC079 | N-terminal hexahistidine LipL C150A mutant expression vector | This study |
| pQC080 | N-terminal hexahistidine LipL C150S mutant expression vector | This study |
| pQC086 | N-terminal hexahistidine LipL C39A mutant expression vector | This study |
| pQC087 | N-terminal hexahistidine LipL C39S mutant expression vector | This study |
| pQC088 | N-terminal hexahistidine LipL Y180A mutant expression vector | This study |
| pQC089 | N-terminal hexahistidine LipL Y180F mutant expression vector | This study |
| pGES485 | Integrational vector; Sp | G.E. Schujman (unpublished) |
| pJM114 | Integrational vector; Km | Perego (1993) |
| pNM67 | pJM114 containing gcvH interrupted with a kanamycin cassette | This study |
| pNM70 | pGES485 containing gcvH cloned into HindIII-BamHI sites | This study |

CGSC, *E. coli* Genetic Stock Center.

Table 2. Oligonucleotides used in this study.

| Oligonucleotides | Sequence 5′-3′ |
|------------------|----------------|
| Q030             | ttaaccatggcatttgaatttaaacttc |
| Q031             | aatggatccttaaatgaagatcttcgtaacc |
| Q034             | gaaacagcacctataaggaggca |
| Q042             | ttaatgatgatgatgatgttgagttggaattgctcagttcg |
| Q095             | taaggtaccaaggagatataccatggcattttaaacttc |
| Q096             | attgatccttagtgtatggtgatggtgatgcgccctccattaaaat |
|                  | taattg |
| Q113             | aacggataagacgggcataa |
| Q114             | gggtgaaggaacgtcatt |
| Q147             | ctatgacctcagtataaatggcaaagcgttcgccggcatct |
| Q148             | agatgccggcgaaacgcttggccatttatactgaggtcatag |
| Q149             | cagctatgacctcagtataaatggcaaacgtttcgccggcatctct |
| Q150             | agagatgccggcgaaacgtttgccatttatactgaggtcatagctg |
| Q151             | ctatgaaattgaagggttctttattctccggcagcta |
| Q152             | tagctgccgggagaataagaccttcaatttcatag |
| Q153             | agcctatgaaaattgaagggtttatgctccggcagctat |
| Q154             | atagctgccggggagcataagaccctcaatttcataggct |
| Q0171            | gcaatggatgatacgcntagccatgccgtcgaaaaagg |
| Q0172            | cctttccgacggacatggctagcgtatcatccattgc |
| Q0173            | cgcaatggatgatacgcntagcatccatgtccgtcgaaa |
| Q0174            | ttccggacggagatgcgtatcatccattgcg |
| Q0175            | gcggcggagtcgctgtcaaattgctctctgtgcagata |
| Q0176            | tactgtcacagagagcaattgaacagcgactccgccgc |
| Q0177            | cggcggagtcgctgtcaaatttttctctgtgcaga |
| Q0178            | tctgcacagagaaaaattgaacagcgactccgccg |
| L                 | agtggaatccaagagcatgggaaag |
| LI                | ttgtcgactcgtgttctcctgagtaa |
| LII               | gagtcgacacaagaagagactaaaaacag |
| LIII              | aaagactgcagtgaattcaccgcca |

chromatography as described for LipM (Christensen and Cronan, 2010).

### Trypsin digestion and LC-MS/MS analysis of LipL

Trypsin digestion and LC-MS/MS analysis of LipL was performed as previously described for LipM (Christensen and Cronan, 2010). Analysis was performed on LipL from QC083 grown without triclosan and from the *fabA* strain QC143 grown with triclosan.

### Lipoic acid ligation assay

Octanoyl ligase activity was assayed to determine the LD substrate specificity of LplJ. The reaction contained 100 mM sodium phosphate (pH 7.0), 50 mM sodium chloride, 5 mM TCEP, 0.25 mM sodium [1-¹⁴C]octanoate, 1 mM magnesium chloride, 1 mM ATP, 20 μM LD and 1 μM LplJ. The reaction was analysed using a modification of the method of Laskey and Mills (1975) in which 10 μl of the reactions was subjected to SDS-PAGE on 4–20% gradient gels, which were soaked in Amplify fluorographic reagent (GE Healthcare), dried, and exposed to preflashed Biomax XAR film (Kodak) at –70°C for 24 h.

### Assay of octanoyl-[acyl carrier protein]: protein N-octanoyltransfer

Octanoyltransfer by LipM and LipB was assayed in order to determine the LD substrate specificity of these enzymes. Assays using AasS were performed as previously described (Christensen and Cronan, 2010) with different LDs substituted for GcvH. Reactions were analysed as described above with LplJ.

### Assay of amidotransfer

Amidotransfer was assayed coupled to lipoyl ligation by LplJ. The reactions contained 100 mM sodium phosphate (pH 7.0), 50 mM sodium chloride, 5 mM TCEP, 0.25 mM sodium [1-¹⁴C]octanoate, 1 mM magnesium chloride, 1 mM ATP, 20 μM GcvH, 20 μM PdhC, 1 μM LipL and 1 μM LplJ. The reactions were analysed as described above for the LplJ assays.

Amidotransfer was also assayed by coupling to octanoyl-transfer by LipM. The reactions contained 100 mM sodium phosphate (pH 7.0), 50 mM sodium chloride, 5 mM TCEP, 0.25 mM sodium [1-¹⁴C]octanoate, 1 mM magnesium chloride, 1 mM ATP, 50 μM holo-AcpP, 2 μM AasS, 20 μM GcvH, 20 μM PdhC, 10 μM LipL and 10 μM LipM. Reactions were analysed as described for the LplJ assays above.

*Bacillus subtilis* crude extracts were also assayed for LipM and LipL activities. The cultures were grown in Spizizen minimal salts (Spizizen, 1958) supplemented with 0.5% glucose, 10 mM sodium acetate, 10 mM sodium succinate, 0.1 mM sodium isovalerate, 0.1 mM sodium isobutyrate and 0.1 mM sodium 2-methylbutyrate. Cultures were grown to an OD600 of 0.6, pelleted by centrifugation, frozen at –80°C and lysed in 100 mM sodium phosphate (pH 7.0) by sonication.
The reaction conditions were the same as the LipM coupled assay with enzyme or 50 μg of extract protein added where indicated. Reactions were analysed as described for the LplJ assays above.

In order to directly assay amidotransfer \[^{14}C\]octanoyl labelled GcvH was prepared using LplJ. The reaction contained 100 mM sodium phosphate (pH 7.0), 50 mM sodium chloride, 5 mM TCEP, 1 mM sodium \[^{14}C\]octanoate, 2 mM ATP, 0.5 mM GcvH and 10 μM LplJ. The reaction was allowed to proceed at \(37^\circ\)C for 2 h. Then the reaction was diluted 20-fold in 25 mM sodium MES buffer (pH 6.1) and purified using anion exchange chromatography using an AKTA purifier10 (GE Healthcare) with a 1.8 ml POROS QE anion exchange column with a flow rate of 5 ml per minute. Protein was bound with 25 mM sodium MES buffer (pH 6.1) and eluted with a 0–1 M gradient of lithium chloride, where octanoyl GcvH eluted at 480 mM. Purified LDs were stored as above except 5 mM TCEP was the reducing agent. Amidotransfer was directly assayed from purified \[^{14}C\]-octanoyl GcvH to various LDs. The reactions contained 100 mM sodium phosphate (pH 7.0), 50 mM sodium chloride, 5 mM TCEP, 20 μM \[^{14}C\]octanoyl-GcvH, 20 μM of different LDs and 1 μM LipL. Amidotransfer reactions with mutant LipL proteins were performed with 10 μM LipL or mutant LipL plus E2\(_{\text{PdhC}}\) domain as an additional LD. The LipL acyl-enzyme intermediate was visualized using the same assay conditions lacking an acceptor domain and with 10 μM LipL or mutant LipL. The reactions were analysed as described for the LplJ assays above.

### Immunoblotting analysis

Lipoylation of crude extracts of *B. subtilis* strains was assayed with anti-lipoic acid antibody (Calbiochem) as described in the companion paper (Martin *et al.*, 2011).

### Phylogenetic analysis and bioinformatics

We determined the phylogeny of select amino acid sequences of the LipB/LplA/BirA superfamily (PF03099), the biotin_lipoyl_domain superfamily (PF00364) and the glycine cleavage system H protein family (PF01597) (Finn *et al.*, 2010). Alignment was performed using Dialign-TX with low scoring region length 4, low-quality region length 40, and unlimited sensitivity (Subramanian *et al.*, 2008). The edges of the alignment were trimmed using Jalview (Waterhouse *et al.*, 2009) so only the relevant domains remained. A minimum evolution tree was constructed with bootstrap analysis of 1000 replicates using Mega4 (Tamura *et al.*, 2007) using the default settings. All positions containing gaps and missing data were eliminated from the dataset (complete deletion option). There were a total of 63 positions in the final dataset comparing LDs. There were a total of 57 positions in the final dataset that compared catalytic domains.

Display of protein structures and structural alignments were performed using the Swiss PDB viewer (Kaplan and Littlejohn, 2001). Protein extinction coefficients and theoretical masses were calculated using Protparam at the EXPASY website (Gasteiger *et al.*, 2003) and used to determine molarity. The extinction coefficients were calculated on the assumption that all cysteine residues were in the reduced state.

---

### Acknowledgements

This investigation was supported by the National Institutes of Health under Ruth L. Kirschstein National Research Service Award 5 T32 GM070421 from the National Institute of General Medical Sciences, Grant AI15650 from the National Institute of Allergy and Infectious Disease, the Consejo Nacional de Investigaciones Científicas y Técnicas (CONICET), Agencia Nacional de Promoción Científica y Tecnológica (FONCYT) and Fundación Josefina Prats. N. Martin is a Fellow of CONICET and M. C. Mansilla and D. de Mendoza are Career Investigators of the same institution. D. de Mendoza is an International Research Scholar of the Howard Hughes Medical Institute. We thank Drs Yanfang Jiang, Sean Jordan, Gustavo Schujman and Xin Zhao for research materials. We thank the New York Structural Genomics Research Consortium and Dr Subramanyam Swaminathan for making the LipL structural co-ordinates available.

---

### References

Baba, T., Ara, T., Hasegawa, M., Takai, Y., Okumura, Y., Baba, M., *et al.* (2006) Construction of *Escherichia coli* K-12 in-frame, single-gene knockout mutants: the Keio collection. *Mol Syst Biol* **2**: 2006.0008.

Bauwe, H., and Kolukisaoglu, U. (2003) Genetic manipulation of glycine decarboxylation. *J Exp Bot* **54**: 1523–1535.

Booker, S.J., Cicchillo, R.M., and Grove, T.L. (2007) Self-sacrifice in radical S-adenosylmethionine proteins. *Curr Opin Chem Biol* **11**: 543–552.

Christensen, Q.H., and Cronan, J.E. (2009) The *Thermoplasma acidophilum* LplA-LplB complex defines a new class of bipartite lipoate-protein ligases. *J Biol Chem* **284**: 21317–21326.

Christensen, Q.H., and Cronan, J.E. (2010) Lipoic acid synthesis: a new family of octanoyltransferases generally annotated as lipoate protein ligases. *Biochemistry* **49**: 10024–10036.

Clancy, K.W., Melvin, J.A., and McCafferty, D.G. (2010) Sortase transpeptidases: insights into mechanism, substrate specificity, and inhibition. *Biopolymers* **94**: 385–396.

Cronan, J.E. (2003) Cosmid-based system for transient expression and absolute off-to-on transcriptional control of *Escherichia coli* genes. *J Bacteriol* **185**: 6522–6529.

Cronan, J.E., Zhao, X., and Jiang, Y. (2005) Function, attachment and synthesis of lipoic acid in *Escherichia coli*. *Adv Microb Physiol* **50**: 103–146.

Cui, G., Nan, B., Hu, J., Wang, Y., Jin, C., and Xia, B. (2006) Identification and solution structures of a single domain biotin/lipoyl attachment protein from *Bacillus subtilis*. *J Biol Chem* **281**: 20598–20607.

Datsenko, K.A., and Wanner, B.L. (2000) One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. *Proc Natl Acad Sci USA* **97**: 6640–6645.

Finn, R.D., Mistry, J., Tate, J., Coggill, P., Heger, A., Pollington, J.E., *et al.* (2010) The Pfam protein families database. *Nucleic Acids Res* **38**: D211–D222.

Fujiwara, K., Maita, N., Hosaka, H., Okamura-Ikeda, K., Nakagawa, A., and Taniguchi, H. (2010) Global conformational change associated with the two-step reaction cata-

© 2011 Blackwell Publishing Ltd, *Molecular Microbiology*, **80**, 350–363
lyzed by *Escherichia coli* lipoate-protein ligase A. *J Biol Chem* 285: 9971–9980.

Gasteiger, E., Gattiker, A., Hoogland, C., Ivanyi, I., Appel, R.D., and Bairoch, A. (2003) ExPASy: the proteomics server for in-depth protein knowledge and analysis. *Nucleic Acids Res* 31: 3784–3788.

Gray, M.W., Lukes, J., Archibald, J.M., Keeling, P.J., and Doolittle, W.F. (2010) Cell biology. Irremediable complexity? *Science* 330: 920–921.

Green, D., Morris, T., Green, J., Cronan, J.E., Jr, and Guest, J. (1995) Purification and properties of the lipoate protein ligase of *Escherichia coli*. *Biochem J* 309: 853.

Hadonou, A.M., Wilkin, J.M., Varetto, L., Joris, B., Lamotte-Brasseur, J., Klein, D., *et al.* (1992) Site-directed mutagenesis of the *Streptomyces* R61 DD-peptidase. Catalytic function of the conserved residues around the active site and a comparison with class-A and class-C beta-lactamases. *Eur J Biochem* 207: 97–102.

Hermes, F., and Cronan, J. (2009) Scavenging of cytosolic octanoic acid by mutant LplA lipoate ligases allows growth of *Escherichia coli* strains lacking the LipB octanoyltransferase of lipoic acid synthesis. *J Bacteriol* 191: 6796.

Hiltunen, J.K., Autio, K.J., Schonauer, M.S., Kursu, V.A., Dieckmann, C.L., and Kastaniotis, A.J. (2010) Mitochondrial fatty acid synthesis and respiration. *Biochim Biophys Acta* 1797: 1195–1202.

Jordan, S.W., and Cronan, J.E. (1997) A new metabolic link. The acyl carrier protein of lipid synthesis donates lipoic acid to the pyruvate dehydrogenase complex in *Escherichia coli* and mitochondria. *J Biol Chem* 272: 17903–17906.

Jordan, S.W., and Cronan, J.E. (2003) The *Escherichia coli lipB* gene encodes lipoyl (octanoyl)-acyl carrier protein:protein transferase. *J Bacteriol* 185: 1582–1589.

Kaplan, W., and Littlejohn, T.G. (2001) Swiss-PDB viewer (deep view). *Brief Bioinform* 2: 195–197.

Keijser, B.J., Beek, A.T., Rauwerda, H., Schuren, F., Montijn, R., van der Spek, H., and Brul, S. (2007) Analysis of temporal gene expression during *Bacillus subtilis* spore germination and outgrowth. *J Bacteriol* 189: 3624–3634.

Laskey, R.A., and Mills, A.D. (1975) Quantitative film detection of 3H and 14C in polyacrylamide gels by fluorography. *Eur J Biochem* 56: 335–341.

Lessard, I.A., Domingo, G.J., Borges, A., and Perham, R.N. (1998) Expression of genes encoding the E2 and E3 components of the *Bacillus stearothermophilus* pyruvate dehydrogenase complex and the stoichiometry of subunit interaction in assembly in vitro. *Eur J Biochem* 258: 491–501.

Ma, Q., Zhao, X., Eddine, A.N., Geerlof, A., Li, X., Cronan, J.E., *et al.* (2006) The *Mycobacterium tuberculosis* LipB enzyme functions as a cysteine/lysine dyad acyltransferase. *Proc Natl Acad Sci USA* 103: 8662–8667.

Mandal, M., Lee, M., Barrick, J.E., Weinberg, Z., Emilsson, G.M., Ruzzo, W.L., and Breaker, R.R. (2004) A glycine-dependent riboswitch that uses cooperative binding to control gene expression. *Science* 306: 275–279.

Mansilla, M.C., and de Mendoza, D. (1997) L-cysteine biosynthesis in *Bacillus subtilis*: identification, sequencing,

and functional characterization of the gene coding for phosphoadenylylsulfate sulfotransferase. *J Bacteriol* 179: 976–981.

Martin, N., Lombardia, E., Altabe, S.G., de Mendoza, D., and Mansilla, M.C. (2009) A *lipA* (*yutB*) mutant, encoding lipoic acid synthase, provides insight into the interplay between branched-chain and unsaturated fatty acid biosynthesis in *Bacillus subtilis*. *J Bacteriol* 191: 7447–7455.

Martin, N., Christensen, Q.H., Mansilla, M.C., Cronan, J.E., and de Mendoza, D. (2011) A novel two-gene requirement for the octanoyltransfer reaction of *Bacillus subtilis* lipoic acid biosynthesis. *Mol Microbiol* 80: 335–349.

Morihara, K., and Oka, T. (1981) Peptide bond synthesis catalyzed by subtilisin, papain, and pepsin. *J Biochem* 89: 385–395.

Morris, T., Reed, K., and Cronan, J. (1995) Lipoic acid metabolism in *Escherichia coli*: the *lpIA* and *lipB* genes define redundant pathways for ligation of lipoyl groups to apoprotein. *J Bacteriol* 177: 1–10.

Nakano, M.M., Dailly, Y.P., Zuber, P., and Clark, D.P. (1997) Characterization of anaerobic fermentative growth of *Bacillus subtilis*: identification of fermentation end products and genes required for growth. *J Bacteriol* 179: 6749–6755.

Packman, L.C., Borges, A., and Perham, R.N. (1988) Amino acid sequence analysis of the lipoyl and peripheral subunit-binding domains in the lipoate acetyltransferase component of the pyruvate dehydrogenase complex from *Bacillus stearothermophilus*. *Biochem J* 252: 79–86.

Perego, M. (1993) Integrational vectors for genetic manipulation in *Bacillus subtilis*. In *Bacillus Subtilis and Other Gram-Positive Bacteria: Biochemistry, Physiology, and Molecular Genetics*. Sonenshein, J.A.H.A.L., and Losick, R. (eds). Washington, DC: American Society for Microbiology, pp. 615–624.

Perham, R.N. (2000) Swinging arms and swinging domains in multifunctional enzymes: catalytic machines for multi-step reactions. *Annu Rev Biochem* 69: 961–1004.

Plamann, M.D., Rapp, W.D., and Stauffer, G.V. (1983) *Escherichia coli* K12 mutants defective in the glycine cleavage enzyme system. *Mol Gen Genet* 192: 15–20.

Reche, P.A. (2000) Lipoylating and biotinylating enzymes contain a homologous catalytic module. *Protein Sci* 9: 1922–1929.

Reed, K.E., and Cronan, J.E. (1993) Lipoic acid metabolism in *Escherichia coli*: sequencing and functional characterization of the *lipA* and *lipB* genes. *J Bacteriol* 175: 1325–1336.

Rock, C.O. (2009) Opening a new path to lipoic acid. *J Bacteriol* 191: 6782–6784.

Sambrook, J., and Russell, D. (2001) *Molecular Cloning: A Laboratory Manual*. Cold Spring Harbor, NY: CSHL Press.

Schonauer, M.S., Kastaniotis, A.J., Kursu, V.A., Hiltunen, J.K., and Dieckmann, C.L. (2009) Lipoic acid synthesis and attachment in yeast mitochondria. *J Biol Chem* 284: 23234–23242.

Silvaggi, N.R., Anderson, J.W., Brinsmade, S.R., Pratt, R.F., and Kelly, J.A. (2003) The crystal structure of phosphonate-inhibited D-Ala-D-Ala peptidase reveals an analogue of a tetrahedral transition state. *Biochemistry* 42: 1199–1208.

Smith, R.L., Pelley, J.W., and Jeter, R.M. (1991) Character-

ization of *lip* expression in Salmonella typhimurium: analysis of *lip: lac* operon fusions. J Gen Microbiol 137: 2307–2312.

Spizizen, J. (1958) Transformation of biochemically deficient strains of *Bacillus subtilis* by deoxyribonucleate. Proc Natl Acad Sci USA 44: 1072–1078.

Steiert, P.S., Stauffer, L.T., and Stauffer, G.V. (1990) The *lpd* gene product functions as the L protein in the *Escherichia coli* glycine cleavage enzyme system. J Bacteriol 172: 6142–6144.

Subramanian, A.R., Kaufmann, M., and Morgenstern, B. (2008) DIALIGN-TX: greedy and progressive approaches for segment-based multiple sequence alignment. Algorithms Mol Biol 3: 6.

Tamura, K., Dudley, J., Nei, M., and Kumar, S. (2007) MEGA4: molecular evolutionary genetics analysis (MEGA) software version 4.0. Mol Biol Evol 24: 1596–1599.

Vanden Boom, T.J., Reed, K.E., and Cronan, J.E., Jr (1991) Lipoic acid metabolism in *Escherichia coli*: isolation of null mutants defective in lipoic acid biosynthesis, molecular cloning and characterization of the *E. coli lip* locus, and identification of the lipoylated protein of the glycine cleavage system. J Bacteriol 173: 6411–6420.

Waterhouse, A.M., Procter, J.B., Martin, D.M.A., Clamp, M., and Barton, G.J. (2009) Jalview version 2 – a multiple sequence alignment editor and analysis workbench. Bioinformatics 25: 1189–1191.

Wycuff, D.R., and Matthews, K.S. (2000) Generation of an araC-araBAD promoter-regulated T7 expression system. Anal Biochem 277: 67–73.

Ye, B.C., Zhang, Y., Yu, H., Yu, W.B., Liu, B.H., Yin, B.C., et al. (2009) Time-resolved transcriptome analysis of *Bacillus subtilis* responding to valine, glutamate, and glutamine. PLoS ONE 4: e7073.

Zhao, X., Miller, J.R., Jiang, Y., Marletta, M.A., and Cronan, J.E. (2003) Assembly of the covalent linkage between lipoic acid and its cognate enzymes. Chem Biol 10: 1293–1302.
