
Cellular Signalling 23 (2011) 1257–1266

Contents lists available at ScienceDirect

Cellular Signalling

journal homepage: www.elsevier.com/locate/cellsig

Review

Rap-linked cAMP signaling Epac proteins: Compartmentation, functioning and disease implications

Magali Breckler ${ }^{a,b,c}$, Magali Berthouze ${}^{a,b}$, Anne-Coline Laurent ${}^{a,b,c}$, Bertrand Crozatier ${}^{c,d}$, Eric Morel ${}^{c,d}$, Frank Lezoualc'h ${}^{a,b,*}$

${}^{a}$ Inserm, UMR-1048, Institut des Maladies Métaboliques et Cardiovasculaires, F-31342 Toulouse, France  
${}^{b}$ Université Toulouse III, UPS, F-31342 Toulouse, France  
${}^{c}$ Université Paris-Sud 11, France  
${}^{d}$ Inserm, UMRS-769, Faculté de Pharmacie, F-92296 Châtenay-Malabry, France

---

**ARTICLE INFO**

Article history:

Received 21 December 2010  
Received in revised form 4 March 2011  
Accepted 4 March 2011  
Available online 12 March 2011  

Keywords:  
Guanine nucleotide exchange factors  
cAMP  
PKA  
CaMKII  
Phosphodiesterase  
Compartmentation  
G protein-coupled receptor  

---

**ABSTRACT**

Epac proteins respond to the second messenger cyclic AMP (cAMP) and are activated by Gs coupled receptors. They act as specific guanine nucleotide exchange factors (GEFs) for the small G proteins, Rap1 and Rap2 of the Ras family. A plethora of studies using 8-pCPT-2'-O-Me-cAMP, an Epac agonist, has revealed the importance of these multi-domain proteins in the control of key cellular functions such as cell division, migration, growth and secretion. Epac and protein kinase A (PKA) may act independently but are often associated with the same biological process, in which they fulfill either synergistic or opposite effects. In addition, compelling evidence is now accumulating about the formation of molecular complexes in distinct cellular compartments that influence Epac signaling and cellular function. Epac is spatially and temporally regulated by scaffold protein and its effectors are interconnected with other signaling pathways. Pathophysiological changes in Epac signaling may underlie certain diseases.

© 2011 Elsevier Inc. All rights reserved.

---

**Contents**

1. Introduction . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
1258

M. Breckler et al. / Cellular Signalling 23 (2011) 1257–1266

# 1. Introduction

Cyclic adenosine 3′,5′-monophosphate (cAMP) is a universal second messenger that plays a crucial role in the intracellular signal transduction of various stimuli controlling a wide variety of cellular events including secretion, cell proliferation, differentiation, migration, and apoptosis. cAMP is produced from ATP by adenyl cyclase isoforms, the majority of which are embedded in the cell surface membrane [1]. These are activated by G protein-coupled receptors (GPCRs) [2], such as the β-adrenergic receptor that, on agonist occupancy, couple to the stimulatory G protein Gs, to produce cAMP. Cyclic AMP can be extruded from the cell by members of the multidrug-resistance-associated proteins such as MRP4 [3]. Intracellular levels of cAMP are further regulated through hydrolysis to 5′AMP by a large family of cAMP phosphodiesterases (PDEs) [4]. PDEs facilitate subcellular compartmentalization and spatiotemporal cAMP dynamics and therefore participate in the specificity of the cAMP action [4,5]. In addition, A-kinase anchoring proteins (AKAPs) that function to sequester cAMP effectors and PDEs enzymes into discrete subcellular compartments maintain localized pools of cAMP and control therefore the cellular actions of this second messenger [6].

The serine- and threonine-directed protein kinase PKA (a tetramer consisting of a regulatory subunit dimer and 2 catalytic subunits) and cyclic nucleotide gated channels were thought to be the only two direct effectors of cAMP, until the identification of a novel category of cAMP mediators in 1998 [5,8]. These proteins, named Epac (exchange proteins directly activated by cAMP), function in a PKA-independent manner and therefore represent a novel mechanism for governing signaling specificity within the cAMP cascade [9]. Epac proteins act as specific guanine nucleotide exchange factors (GEFs) for the Ras GTPase family members Rap1 and Rap2. Upon binding of cAMP, Epac activates Rap by exchanging bound GDP for GTP, whereas GTPase activating proteins (GAP) return them to their inactive GDP-bound states. Rap1 was originally identified as a protein that reverted the morphological phenotype of Ras-transformed cells [10]. Initial studies showed that the Epac/Rap1 pathway was involved in the control of integrin-mediated cell adhesion [11,12]. However, with the development of 8-pCPT-2′-O-Me-cAMP a cAMP analog that can selectively activate Epac, several additional functions of Epac have recently been uncovered. Epac proteins have been shown to be involved in a large number of cellular functions such as cell division, differentiation, secretion and growth [13–15]. The multidomain structure of Epac indicates that it may have multiple binding partners. In this sense, compelling evidence is now accumulating about the formation of molecular complexes in distinct cellular compartments that influence Epac signaling and cellular function. Although this review summarizes recent findings on the biological role of Epac proteins and their hormonal regulation and compartmentation, the reader is encouraged to read other recent reviews on related issues [9,13–15].

# 2. Epac proteins

There are two isoforms of Epac, Epac1 and Epac2, which are coded by two distinct genes, RAPGEF3 and RAPGEF4 in mammals. While Epac1 is ubiquitously expressed in all tissues with high levels of expression in the kidney, Epac2 is detectable most notably in the brain, pituitary, adrenal gland and pancreas [7,8,16]. Interestingly, a shorter N-terminal splice variant of Epac2 named Epac2B has been recently identified in the adrenal gland suggesting that alternative splicing events may further add to the complexity of the functional characterization of Epac isoforms [17]. Following this discovery, the long isoform Epac2 was renamed Epac2A [17]. Epac1 and Epac2 are multi-domain proteins that share high homology sequence. Both isoforms contain an N-terminal regulatory region and a C-terminal catalytic region. The regulatory region of Epac1 and Epac2 contains a Disheveled/Egl-10/pleckstrin (DEP) domain followed by an evolutionarily conserved cyclic nucleotide binding domain (CNBD) to the CNBD of PKA and the bacterial transcriptional factor cAMP receptor protein. Direct fusion of donor and acceptor fluorescent proteins to the EPAC CNBD resulted in a probe that could image high-speed cAMP fluctuations in living cells [18]. An additional CNBD N-terminal to the DEP domain named CNBD-A is presented in Epac2A but it binds cAMP with a 20-fold lower affinity than the conserved CNBD. CNBD-A of Epac2A has been shown not to be strictly necessary for the cAMP-dependent activation of Epac [19–21]. Recent evidence indicates that CNBD-A is involved Epac2A subcellular localization [17].

The catalytic region consists of a RAS exchange motif (REM) domain that stabilizes the GEF domain, a RAS association (RA) domain that may play a role in the translocation of Epac2 to the plasma membrane [22,23], and a classic CDC25-homology domain (CDC25HD) that promotes the exchange of GDP for GTP on Rap [24]. Recent X-ray crystallography of Epac2A and NMR spectroscopy of Epac1 have provided novel insights into the molecular mechanism by which cAMP controls Epac activation [25–28]. In the absence of cAMP, the regulatory region containing the cAMP-binding domain directly interacts with the catalytic region and inhibits GEF activity. Binding of cAMP to Epac induces large conformational changes within the protein and releases the autoinhibitory effect of the N terminus of the protein, leading to Rap activation.

# 3. Epac activation

Discriminating between PKA- and Epac-dependent signaling pathways in cells has been hampered by the lack of any specific agent that selectively activates each of these two proteins. However, in 2002, few years after the cloning of Epac, rational drug design led to develop cAMP derivatives that specifically activate these cAMP-GEFs [29–31]. Indeed, Enserink et al. [29] noticed that a highly conserved glutamate residue present in the CNBD of the PKA regulatory subunits and all other known CNBD such as olfactory and pacemaker channels was not found in Epac isoforms. This glutamate residue plays a critical role in the formation of hydrogen bonds with the 2′-hydroxyl of the cAMP ribose group of PKA regulatory subunits [32]. Analogs in which the 2′-hydroxyl group has been replaced with 2′-O-Me selectively interact with Epac proteins [29]. The affinity of such analogs for Epac could be improved by the introduction of a parachlorophenylthio (pCPT) substitution at position 8 on the adenine moiety of 2′-O-Me-cAMP. One of these cAMP analogs, 8-pCPT-2′-O-Me-cAMP (8-(4-chloro-phenylthio)-2′-O-methyladenosine-3′,5′-cAMP), exhibits high affinity for Epac (Kd 2.2 μM for Epac1) as well as reduced affinity for PKA (Kd 200–300 μM) [29]. 8-pCPT-2′-O-Me-cAMP activated Rap1 with greater potency and efficacy than cAMP [29,33]. Recent findings showed that an acetoxy methyl ester (AM) of 8-pCPT-2′-O-Me-cAMP (8-pCPT-2′-O-Me-cAMP-AM) was more efficiently delivered into cells than its parent compound [34,35]. It was demonstrated that the potency of 8-pCPT-2′-O-Me-cAMP-AM was 100 to 1000-fold greater than that of 8-pCPT-2′-O-Me-cAMP in assays of Rap1 activation assay [34]. This Epac agonist does not discriminate between Epac1 and Epac2 and has been used extensively to determine the role of these cAMP-GEFs in cell function [31]. However, care should be taken in using such cAMP analog because a recent study indicates that depending upon the model or the doses used, 8-pCPT-2′-O-Me-cAMP or its non-hydrolyzable form sp8-pCPT-2′-O-Me-cAMP, can activate cAMP/PKA or cGMP/PKG pathways through the inhibition of PDE [36]. In addition, metabolites of 8-pCPT-2′-O-Me-cAMP may regulate gene expression and cell function through an unknown signaling pathway [37]. At present there is no specific pharmacological antagonist of Epac isoforms. Thus, supplemental tools such as siRNA or recombinant mutants of Epac must be used to discriminate Epac from PKA pathway and to evaluate the functional role of the GEF Epac in cellular processes. Of note, it was reported that Epac2 isoform was a direct target of the widely used antidiabetic drugs, sulfonylureas [38]. However, recent in vitro experiments using purified recombinant full-

length Epac2 and Rap1 failed to detect any binding of sulfonylureas to Epac2 nor to reveal any activation of Rap1 by these drugs suggesting that the effects of antidiabetic sulfonylurea drugs on Epac2 were most likely indirect [39].

### 4. Epac downstream effectors (Table 1)

In addition to directly regulating many important cellular processes, cAMP influences an array of intracellular signaling pathways such as phospholipase C (PLC), Ras and Ca²⁺-dependent signaling pathways. The existence of Epac proteins immediately raises many questions regarding the effectors that couple Epac to its functional effects. As discussed below, it is just the beginning of discovery of the mechanism of action of Epac and its downstream pathways (Table 1).

#### 4.1. Rap1

As mentioned above, the Ras-like GTPases Rap1 and Rap2 are the direct downstream effectors of Epac1 and Rap relays most of the biological effects of this cAMP-GEF in tissues [24,40]. Rap1 is the immediate effector of Epac. It was initially reported to antagonize Ras-induced cell transformation [10] and it was speculated that this small GTPase protein and the Rap-GEF, Epac, would be linked to inhibit cell division via the regulation of mitogen-activated protein kinases (MAPK). However, the role of Epac-Rap1 in the control of cell division is complex and dependent on the cellular context and the type of stimuli. This has been recently reviewed in detail [13]. Similarly, several reports have shown that Epac regulates negatively or positively cell migration in various cell types including tumor cell migration but the exact molecular mechanisms are currently still under investigation [41–44]. Interestingly, Epac/Rap1 has been identified as a modulator of IL-6 signaling in HUVECs making this cAMP sensor as a potential anti-inflammatory protein in vascular endothelial cells [13,45,46]. Of note, the involvement of Epac in inflammatory processes is not limited to vascular endothelial cells since this cAMP-GEF has been reported to display pro- and anti-inflammatory actions in various cell types including dendritic cells and macrophages [47–50].

One of the best known Epac-Rap1 functions is the control of cell adhesion and cell-cell junction formation [12,52]. Epac-Rap1 signaling has indeed been shown to reduce vascular permeability by altering microtubule dynamics as well as enhancing junctional molecules such as vascular-endothelial cadherin at cell-cell contacts [51–55]. Interestingly, the regulation of junctional molecules requires dynamic reorganization of the actin cytoskeleton, which is primarily orchestrated by the small GTPases of the Rho family. Epac activation indeed induces reorganization of cortical actin which supports junctional adhesion molecules thereby contributing to stabilize endothelial barrier function [51–53,56]. The effect of Epac on actin remodeling involves the activation of Rap1 which can then modulate indirectly the activity of the Rho GTPase family such as Rho and Rac through the recruitment of specific GEFs of the Rho family [56,57]. In turn, Rho proteins interact with downstream cytoskeletal and cell adhesion effectors and promote cytoskeletal remodeling. Such a cross talk between Rap and RhoGTPases regulated by cAMP and Epac exists in other cellular processes as it was reported for the shedding of the Alzheimer's disease associated protein [58–60].

---

**Table 1**

Biological role of Epac proteins and their downstream effectors. ARPKD, autosomal recessive polycystic kidney disease; CaN, calcineurin; CNS, central nervous system; COPD, chronic obstructive pulmonary disease; EC coupling, excitation-contraction coupling; MI, myocardial infarction.

| Tissue        | Function                                      | Pathology                     | Effectors                                                                 | Ref.               |
|---------------|-----------------------------------------------|-------------------------------|--------------------------------------------------------------------------|--------------------|
| CNS           | Neurotransmitter release                      | Memory disorders              | ERK, Rap, Akt                                                             | [132–134]          |
|               | Synaptic plasticity                           |                               | Rap1, Rac, STEF                                                           | [108,109,135]      |
|               | Amyloid precursor protein processing           | Alzheimer's disease           | Rit                                                                       | [58–60]            |
|               | Circadian cycle                               |                               | Bcl-2 interacting member protein                                          | [136]              |
|               | Neuronal differentiation, neurite extension   |                               | Rap1, p38MAPK                                                             | [30,88]            |
|               | Neuronal apoptosis                            |                               | PLC, PLD, PKC, Rap1, MEK/ERK, GRK2                                        | [137]              |
|               | Neuronal excitability                         |                               | Rap1, PLC, PKC, CaMKII                                                   | [138]              |
|               | Pain sensitivity                              |                               | Rap1, PKC, Connexin 43                                                    | [139,140]          |
| Heart         | Excitation-contraction coupling                | Heart failure, MI             | Rap2B, Ras, Rac, CaMKII, CaN, Rap1, ERK5                                  | [66–69]            |
|               | Cell-junctions                                |                               | Akt                                                                       | [40,70]            |
|               | Myocyte growth                                | Fibrosis                      | Rap1, PI3K                                                                | [63,72,73,107]     |
|               | Myocytes apoptosis                            |                               | Rap1                                                                      | [141]              |
| Immune cells  | Phagocytosis                                  | Inflammation                  | p38, GSK3β, PI3K, Akt                                                     | [123,124]          |
|               | Cytokines production                          | Immune diseases               | ERK1/2, Akt                                                               | [47–49,143,144]    |
|               | BCR-induced growth arrest and apoptosis       |                               | Rap1                                                                      | [145]              |
| Kidney        | Leukocyte adhesion and migration              |                               | MEK1/2, NHE3, Rap1, RafB, H⁺/K⁺ ATPase                                    | [54,146]           |
|               | Ion and water absorption                      | ARPKD                         | MEK, ERK1/2                                                               | [147,148]          |
|               | Cholangiocytes proliferation (hepatic cystogenesis) |                               | Rap1, MEK1/2, ERK1/2, UT-A1                                              | [149]              |
|               | Urea transport                                |                               | Src, PI3K                                                                 | [150]              |
| Liver         | Mesangial cells collagen synthesis            | Glomerular diseases           | Ca⁺ and Cl⁻ channel                                                      | [151]              |
|               | Hepatocyte conductance                        |                               | Rap1, SOCS3                                                               | [83]               |
|               | Gluconeogenesis                               |                               | RhoA, Rac1                                                                | [130]              |
| Lung          | Alveolar fluid clearance                      | Chronic asthma, COPD          | Rap1                                                                      | [14,152,153]       |
|               | Airway smooth muscle proliferation and relaxation |                               | Rap1                                                                      | [154]              |
|               | Fibroblast proliferation                      | Pulmonary edema               | Rap1, Tiam, Vav, Rac                                                      | [57]               |
| Pancreas      | Insulin secretion                             | Diabetes                      | Rap1, SUR1 (K-ATP channel), Rim2, Piccolo                                 | [75–80,104,155–157]|
| Prostate      | Cellular proliferation                        | Cancer                        | B-Raf, ERK                                                                | [158]              |
| Skin          | Melanoma cell migration, metastasis           |                               | Rap1, PLC, PI3K, ERK                                                     | [41,159]           |
|               | Vasculogenic mimicry                         |                               | Rap1                                                                      | [160]              |
| Sperm         | Acrosome exocytosis                           |                               | Rap1, PLC, Rab3A                                                          | [64]               |
| Thyroid       | Mitogenesis                                   | Cancer                        | Rap1B, PP2A, Akt                                                          | [105]              |
|               | TSH signaling                                 |                               | Rap1                                                                      | [161]              |
| Vascular system | Endothelium permeability                    | Edema, metastasis             | Rap1, Rac1                                                                | [56,162]           |
|               | IL-6 signaling                                | Atherosclerosis, restenosis   | Rap1, SOCS3                                                               | [45]               |
|               | Vascular remodeling                           | Angina, hypertension          | SUR2B (K-ATP channel), CaN                                                | [44,120]           |
|               | Vascular tone                                 |                               | Rap, thrombospondin-1, ERK5/MEK5, PI3K, Akt                               | [85]               |
|               | Angiogenesis                                  | Cancer                        | [163,164]                                                                 |                    |

4.2. Rap-PLC

Shortly after the discovery of Epac, Schmidt et al. [61] revealed a central role for Epac in governing cross-talk between the cAMP and PLC signaling pathways. Indeed, they showed that Epac activated the phospholipase Cε (PLCε) specifically through the activation of Rap2, resulting in the generation of inositol-1,4,5-trisphosphate and subsequent release of calcium from intracellular stores in HEK293 and neuroblastoma cell lines. This functional coupling of Epac to PLC may be explained by the binding of Rap2 to a Rap association domain located within PLCε. Epac did not exhibit any catalytic activity toward H-Ras in vitro and Epac–Rap2–PLCε signaling induced activation of H-Ras probably by a calcium-regulated Ras-GEF [7,62,63]. Further functional studies showed that Rap1 was also involved in the activation of PLC and Epac–Rap1–PLC signaling regulated important cellular functions such as the exocytosis of the acrosome reaction in sperm, melanoma cell migration and gene expression [41,64,65].

Of particular interest, recent research reveals Epac as a new actor in the regulation of calcium homeostasis in the heart. In rat ventricular cardiac myocytes, acute stimulation of Epac with 8-pCPT-2′-O-Me-cAMP induces a decrease in the (Ca²⁺)i transient amplitude and this is correlated with an increase in cell contraction via PLC, protein kinase C (PKC) and calcium/calmodulin dependent protein kinase II (CaMKII) dependent phosphorylation of sarcomeric proteins such as cardiac Myosin Binding Protein-C and Troponin I [15,66,67]. In mouse cardiac myocytes, the effect of Epac on (Ca²⁺)i transient amplitude was regulated by β-adrenergic receptor and was dependent on Rap1, PLCε and one of its principal downstream enzyme: PKCε [68,69].

The role of Epac in cardiac myocytes is not limited to the direct regulation of the contractile machinery. In this respect, Epac–PLC signaling increases intercellular permeability via a PKCε dependent phosphorylation of connexin-43 [70]. In addition, recent work has implicated Epac as a positive regulator of cardiac myocyte hypertrophy, a process which can progress to heart failure, a leading cause of morbidity and mortality in industrialized countries [71]. In response to a sustained β-adrenergic stimulation, Epac1 activation leads to morphological changes associated with an increase in cell surface area, protein synthesis and the expression of cardiac hypertrophic markers such as the atrial natriuretic factor [72,73]. Epac hypertrophic signaling involves PLC, Rac, H-Ras, the Ca²⁺ sensitive proteins ((CaMKII) and calcineurin) and their downstream transcription factors such as NFAT and [63,72]. Knock-down of Rap2B with specific siRNA partially decreased Epac-induced hypertrophic gene markers suggesting that Rap2B was involved in Epac hypertrophic signaling in primary cardiac myocytes [63]. This cAMP sensitive GEF also stimulates intracellular Ca²⁺ release channels (ryanodine receptors (RyRs)) via their phosphorylation by CaMKII and the subsequent Ca²⁺ leak may contribute to CaMKII and calcineurin activation and arrhythmia [66,74]. With these points in mind, we argue that Epac proteins are involved in the pathophysiological processes leading to cardiac disease.

One of the first roles ascertained to Epac, and in particular Epac2 was the regulation of insulin secretion suggesting that this cAMP–GEF might be a relevant pharmacological target for the treatment of diabetes. Epac2/Rap1 signaling is essential in the regulation of insulin granule dynamics by cAMP, most likely by controlling granule density near the plasma membrane [75]. PLC also participates in Epac action since a transduction cascade comprised of Epac2, Rap1, and PLCε has been reported to increase intracellular Ca²⁺ by facilitating Ca²⁺-induced Ca²⁺ release and possibly exocytosis of insulin from pancreatic β-cells [76]. However, Epac signaling in the context of insulin secretion is mediated by various downstream effectors [77] (see below).

4.3. Other effectors of Epac

Ozaki et al. [78] demonstrated that the secretagogue action of Epac2 resulted from its direct interaction with secretory granule-associated proteins, Rim2 and Piccolo. The participation of Rap in this process remains elusive. Biochemical studies also showed that Epac2 directly interacted with the sulfonylurea receptor-1 (SUR1) subunit of ATP-sensitive K⁺ (K<sub>ATP</sub>) channels in human β cells and rat INS-1 insulinoma cells. The Epac-selective agonist 8-pCPT-2′-O-Me-cAMP inhibited K<sub>ATP</sub> currents causing plasma membrane depolarization and opening of voltage-dependent Ca²⁺ channels (VDCCs), an effect that likely contributed to the release of insulin granules docked at the plasma membrane [79,80]. One other possible mechanism for Ca²⁺-induced Ca²⁺ release includes the modulation of Cav1.2 L-type VDCCs by Epac2 [81]. Thus, Epac proteins seem to regulate exocytosis of insulin through interaction with secretory-granule-associated proteins and regulation of ion channels. Interestingly, the role of Epac in the regulation of membrane channel functions extends from its role in pancreatic beta-cells and cardiac myocytes as Epac affects the activity of sodium channels in lung epithelium [82], Ca²⁺ and Cl⁻ channels in hepatocytes [83], the Ca²⁺ sensitive-potassium currents in cerebellar neurons [84], the K<sub>ATP</sub> in vascular cells [85] and in neurons of the medullary respiratory network [86].

Other studies in the literature have reported some Rap independent effects. For instance, Epac is reported to activate R-Ras, thereby stimulating phospholipase D (PLD) when HEK cells expressing M3 muscarinic acetylcholine receptors are exposed to carbachol [87]. In addition, Shi et al. [88] showed that Epac promotes activation of Rit, a close relative of Ras, in a manner that does not appear to rely upon Rap signaling or the direct regulation of Rit by Epac [88]. An additional research line represents the relation between Epac and phosphatidylinositol 3-kinase (PI 3-kinase)-dependent protein kinase B (PKB)/Akt which plays a central role in the regulation of metabolic pathways, proliferation, and cell death. For example, 8-pCPT-2′-O-Me-cAMP was found to strongly potentiate PKB activation by insulin in skeletal muscle but the participation of Rap1 in this process was uncertain [89].

5. Compartmentalization of Epac signaling

5.1. Regulation of Epac by G protein-coupled receptor (GPCR) (Fig. 1)

The concentration of cAMP required to activate purified Epac1 protein in vitro was initially estimated to be about 10-fold higher than that of PKA [29]. This result was further supported by the observation that cAMP FRET sensors based on PKA are more sensitive than those based on Epac [90]. However, recent data indicate that Epac and PKA holoenzyme have similar affinity for cAMP suggesting that both Epac and PKA respond to physiological relevant cAMP concentrations [91]. Of interest, the light chain (LC) 1 of microtubule-associated proteins (MAP) 1B and LC2 of MAP 1A have been shown to act as a molecular chaperone of Epac1, increasing the binding of cAMP to this GEF and consequently increasing Epac1 signaling in rat pheochromocytoma (PC12) cells [92,93].

Initial evidences that Epac activation was regulated by GPCRs came from the observations that 8-pCPT-2′-O-Me-cAMP reproduced some PKA independent effects of hormones or neurotransmitters as it was shown for β-adrenergic receptors, the prostaglandin (PGE2) receptor, the serotonin 5-HT₄ receptor, the prostanoid EP [2] receptor and GLP-1 receptors [12] (Fig. 2) [31,52,58,79,94]. Interestingly, fluorescent indicators based on cAMP-dependent conformational change of full length Epac1 allowed detecting Epac activation in response to β-adrenergic receptor stimulation in living cells [95]. The regulation of Epac activation by Gs-protein-coupled receptors (GsPCRs) was further confirmed by RNA interference (RNAi) studies performed in vitro in various cellular systems [56,73,88,96]. However, a complete analysis of Epac1 and Epac2 activation in vivo is still lacking and the development of Epac knock-out mice will certainly help to better understand the hormonal regulation of these cAMP–GEFs in vivo.

M. Breckler et al. / Cellular Signalling 23 (2011) 1257–1266

AVP → VP-R  
CLADO  
Adenosine → A₂-R  
Prostaglandin E₂ → Prostanoid-R  
Butaprost  
Isoproterenol → β-AR  
Exendin-4 → GLP-1-R  
GLP-1  
PACAP-38 PACAP-27 → PACAP-R  
CRF → CRF-R  
DA → DA-R  
SKF-38393 → 5-HT₄-R  
Prucalopride  

cAMP  
Epac  

Osmotic water permeability in collecting duct (165)  
Inhibition of arterial smooth muscle cell proliferation (167)  
Expression and release inhibition of pro-inflammatory chemokines in dendritic cells (94)  
Inhibition of lung fibroblast proliferation and collagen synthesis (96,154)  
Cardiac myocyte hypertrophy, EC coupling (66-69,73)  
Insulin secretion in pancreatic β-cells (76,166)  
APP processing, Neurotransmitter release, neuronal differentiation and plasticity, synaptic transmission (58,59,84,87,109,168)  

Fig. 1. Epac-mediated actions of Gs-protein coupled receptors (GsPCR). Agonist-binding to Gs-coupled receptors produces an increase of intracellular cAMP levels which induces the activation of Epac proteins. The figure illustrates the biological effects of some GsPCR involving Epac in their signaling cascade. 5-HT₄-R, serotonin type 4 receptor; β-AR, β-adrenergic receptor; A₂-R, adenosine 2 receptor; APP, amyloid precursor protein; AVP, arginine vasopressin; CLADO, 2-chloroadenosine; CRF, corticotropin releasing factor; DA, dopamine; GLP-1-R, glucagon-like peptide-1 receptor; Iso, Isoprenaline; PACAP-R, pituitary adenylate cyclase activating polypeptide receptor; Prostanoid-R, prostaglandin receptor; VP-R, vasopressin receptor. Squares indicate synthetic ligands [165–168].

The ability of distinct GPCRs to activate Epac proteins in the same cell may vary due to the generation of specific pools of cAMP by different receptors. It is now well appreciated that cyclic nucleotide phosphodiesterases (PDEs) are critical actors in limiting the spread of cAMP and in shaping and organizing intracellular signaling micro-domains [4,5]. PDEs are targeted to discrete signaling complexes and sculpt local cAMP gradients to regulate the activation of spatially localized signaling through sequestered PKA and Epac sub-populations [4]. As discussed below in Section 5.4, PDE4 isoforms contribute to the specificity of Epac signaling in a spatial and temporal manner [4,97].

### 5.2. Localization and function of Epac proteins

Epac proteins are expressed within different subcellular compartments such as the nucleus, the cytosol, nuclear and plasma membranes and their localization varies depending on the cell type and cell cycle [9,73,97–101]. As for its effector Rap, the subcellular localization of Epac

M. Breckler et al. / Cellular Signalling 23 (2011) 1257–1266

Thrombin  
receptor  

NMDA  
receptor  

Cell adhesion  

Endothelial barrier  
enhancement  

Adrenergic  
receptors  

Plasma  
membrane  

Gq  

Rap1  

Ezrin-Radixin-Moesin  

Actin  
anchoring  

MT polymerization  

Epac  

Epac  

Epac  

AKAP9  

Cytoskeleton  
( actin, MT )  

CaMKII  

CaN  

Ca²⁺  

Ca²⁺  

PKA  

PDE4D3  

ERK5  

RanBP2  

Nuclear Pore  
Complex  

NFAT, HDAC4/5  
translocation  

Gene expression  

Learning, memory  

Neuron proliferation & survival  
Synapse structure & function  

Nucleus  

Nuclear protein  
translocation ?  

Fig. 2. Multiprotein complexes localized in distinct cellular compartments and influencing Epac biological effects. Depending on its partner, Epac2 displays distinct neuronal functions at the plasma membrane. In the AKAP79/150/PKA/Epac2 complex, Epac2 activates PKB/Akt signaling which is involved in proliferation, differentiation and survival of neurons. By interacting with the anchoring proteins PSD-95 and/or neuroligins, Epac2 influences the structure and function of the synapses via Rap1. The ezrin-radixin-moesin (ERM) proteins interact with Epac1 recruiting this cAMP sensor at the plasma membrane. Depending on the cell type, ERM proteins-Epac clusters may be required for either cell adhesion or cell proliferation. AKAP9 and Epac1 functionally interact to enhance the barrier properties of the endothelium through effects on microtubules dynamics. The small G protein Ran and Ran binding protein 2 (RanBP2) anchor Epac1 to the nuclear pore, allowing cAMP to activate a local pool of Rap1 at the nuclear envelope. This complex may participate to the nuclear/cytoplasmic shuttle of NFAT and HDAC proteins observed in Epac/Ras/Rac cardiac hypertrophic signaling. In neonatal rat cardiomyocytes, Epac is also part of a multiprotein complex containing mAKAP, PKA and PDE4D3 that regulates ERK5 activity. mAKAP, muscle-A-Kinase anchoring protein; CaN, calcineurin; ERK5, extracellular Regulated Kinase 5; ERM, ezrin-radixin-moesin; PDE4D3, phosphodiesterase 4D3; RanBP2, Ran Binding Protein2; MT, microtubule.

proteins is also dynamically regulated and several studies have analyzed the molecular determinants of Epac cellular localization. Because Ras proteins are post-translationally prenylated and associated with cellular membranes it is not surprising that regulation of their upstream activators should include membrane targeting. Combining high-resolution confocal fluorescence microscopy and fluorescent resonance energy transfer assays, Ponsioen et al. [102] observed that cAMP rapidly translocated Epac1 to the plasma membrane to activate Rap1 in HEK293 cells. The cAMP-binding domain together with the DEP domain of the protein is a critical determinant of this process [102]. Further studies showed that in response to a β-adrenergic receptor stimulation, Epac1 is recruited to the plasma membrane in a multiprotein complex containing the scaffolding protein β-arrestin [103].

Despite considerable structural similarity of Epac1 with Epac2, several differences in the regulation and function of these isoforms are emerging. Epac1 contains a functional mitochondria localization signal at its N-terminus that is not present in Epac2A [22,99]. Instead, the N terminus of Epac2A is required for the binding of Rim2, a target of the small G protein Rab3, to modulate insulin secretion [104]. Recombinant Epac2A fused to GFP localized near the plasma membrane, while the shorter N-terminal splice variant Epac2B was found primarily in the cytoplasm of insulin-secreting MIN6 cells. Epac2B failed to trigger insulin secretion indicating that the N-terminus of Epac2A played an important role in cAMP-potentiated insulin secretion by determining its subcellular localization [17].

In contrast to Epac1, the molecular determinant of Epac2A translocation to the plasma membrane involves an interaction with Ras-GTP via Epac2A RA domain. Indeed, Epac2A activation is accompanied by its Ras-dependent translocation, thereby coupling cAMP to Rap1 activation solely at the plasma membrane [22,23]. Interestingly, the second cAMP-binding domain of Epac2A plays a critical role in determining its subcellular localization [17]. The targeting of Epac to a specific cellular compartment may also implicate other domains of the protein. Indeed, Borland and coworkers [92] reported that the perinuclear targeting of Epac1 was also promoted by a portion of the GEF domain located at the exchange factor's C-terminus in COS cells. Thus, the subcellular distribution of Epac is dependent on several portions of its regulatory and catalytic domains.

There are some evidences that distinct localization of different pools of Epac determines the coupling to its downstream effectors. This is supported by the observation that enforced relocalization of Epac1 from its normal perinuclear location to the plasma membrane, through the addition of a CAAX box, allows cAMP to activate ERK through Rap1 and B-Raf in PC12 cells [21]. In HEK-B2 cells, nuclear Epac coupled to Rap2, triggers the nuclear extrusion of DNA damage-
responsive kinase (DNA-PK), an enzyme that provides a key part of the DNA repair systems [97]. In addition, disruption of the DEP-dependent subcellular targeting of Epac abolished cAMP-Epac-mediated thyroid-stimulating hormone-mediated cell proliferation [105]. These data indicate that Epac-mediated proliferation of rat thyroid follicular cells depends on its proper subcellular membrane targeting via its DEP domain [105]. Together these studies show that the correct intracellular targeting of Epac may be critical in determining the response of cells to elevations in intracellular cAMP. This suggests that Epac signaling is spatially and temporally regulated by diverse anchoring mechanisms which control specific functions of the GEF by recruitment to distinct subcellular locations.

### 5.3. Interaction of Epac with scaffolding proteins (Fig. 2)

The ability of different hormones to induce diverse cellular responses through the same second messenger and effectors suggests that there are mechanisms in place to differentiate cAMP signals. Compelling evidence is now accumulating about the formation of molecular complexes in distinct cellular compartments that influence Epac signaling and cellular functions. Indeed, Epac proteins seem to be spatially and temporally regulated and exert their biological function in combination with scaffolding proteins such as AKAPs, PDEs which regulate the duration and intensity of cAMP signaling, and also PKA. AKAPs sequester discrete sets of signaling molecules to particular regions of the cell therefore allowing for the precise control of PKA-mediated phosphorylation events [106]. An important feature of AKAPs is their ability to interact with other signaling proteins such as signal-termination enzymes (i.e. phosphatases and PDEs) and signal transduction enzymes (i.e. kinases).

Several studies support the idea that AKAPs may contribute to the spatial regulation of Epac. Indeed, AKAP multiprotein complexes containing Epac have been identified in various cell types [107,108]. In neuronal cells, Epac2 has been shown to form a complex with the plasma membrane-associated AKAP79/150, PKA and PKB/Akt [108]. In contrast to PKA, direct activation of Epac2 increased phosphorylation of PKB/Akt, a protein known to be involved in neuronal regulation. AKAP150 seems therefore to coordinate PKA and Epac-mediated PKB/Akt phosphorylation but the precise biological significance of this finding remains to be elucidated. Of particular interest, AKAP9 and Epac1 functionally interact to enhance the barrier properties of the endothelium through effects on microtubule dynamics [55]. Epac has been shown to be involved in synapse morphology and function when it interacts with the anchoring proteins PSC-95 and/or Neuroligins at the plasma membrane [109,110].

Other anchoring mechanisms provide further support for the compartmentalization of Epac1 signaling. For instance, under thrombin receptor activation, the ezrin–radixin–moesin (ERM) proteins are able to interact with the N-terminal part of Epac recruiting this cAMP-sensitive GEF at the plasma membrane [111]. Depending on the cell type, ERM proteins–Epac clusters may be required for either efficient cAMP-mediated cell adhesion or cell proliferation [111,112]. In addition, cotransfection experiments showed that the Nuclear distribution element-like (Ndel1) appears to act as a scaffold protein in that it can interact with PDE4 isoforms to influence the activity status of the associated cAMP effector, Epac in HEK293 cells [113]. A particular role for the small G protein Ran and Ran binding protein 2 (RanBP2) has been suggested by Liu et al. [114] who showed that these proteins anchor Epac1 to the nuclear pore, permitting cAMP signals to activate a local pool of Rap1 at the nuclear envelope of HEK293 cells. The finding of an Epac1-dependent activation of Rap1 at the nuclear envelope suggests a potential role for this localized complex in nuclear transport. In this sense, a recent study identified Epac1 as a regulator of histone deacetylases which counteract the hypertrophic effects of Epac in cardiac myocytes [63,97].

---

### 5.4. Regulation of Epac signaling by PDEs and PKA

#### 5.4.1. Epac and PDEs

PDEs are important regulators of signal transduction processes mediated by cAMP and cGMP. Eleven PDE families (PDE1-11) have been identified with regard to their substrate affinities, biochemical and physical properties, mechanisms whereby they are regulated, and different sensitivities to inhibitors [115]. Distinctive tissue distribution, subcellular compartmentalization and differential regulation of these enzymes contribute to the establishment of local cAMP gradients by limiting the diffusion of cAMP that is generated by adenylyl cyclases [4]. As such, PDEs are able to regulate a diverse range of cellular processes that depend on spatially and temporally controlled intracellular cAMP. Among the long list of PDE, the cAMP-specific PDE3 and PDE4 phosphodiesterases invariably provide the major fraction of cAMP hydrolyzing activity in cells [116]. PDE3 and PDE4 activities have been shown to couple to various Gs-coupled receptors such as β1- and β2-adrenergic receptors and prostaglandin E1 receptors [5,117].

A number of studies have revealed the importance of interaction of PDE3 and PDE4 enzymes with Epac for regulating its activity and its downstream effectors. This is illustrated in the effects of Epac in the regulation of vascular permeability of endothelial cell monolayers. Vascular permeability is induced by various pro-inflammatory mediators including thrombin and histamine and its regulation is critical for maintaining vascular integrity in homeostasis and disease [118]. As previously noted, Epac1–Rap1 signaling has been shown to reduce vascular permeability via for example an augmentation of junctional molecules such as vascular-endothelial cadherin to cell–cell contacts [51–53]. PDE4D tethers Epac1 into the vascular endothelial cadherin-based complex and the interactions of these two enzymes are critical for coordinating the effects of cAMP on cell permeability [119,120].

Interestingly, a recent report identified that PDE3 inhibition caused an Epac-dependent reduction in the ability of insulin to stimulate lipogenesis, glucose uptake and GLUT4 translocation in rat primary adipocytes [121]. In the same cellular system, Epac and PDE4 have been shown to regulate a critical regulator of energy homeostasis, AMP-activated protein kinase [121]. In addition, the ability of Epac to trigger nuclear/cytoplasmic distribution of DNA-PK is specifically regulated by PDE4B in HEK-B2 cells [97]. Of note, interactions of PDE4 with Epac may also represent novel promising targets for therapeutic intervention. This was well illustrated in the work of Hertz et al. [50] who showed that a PDE4 inhibitor increased chemokine expression in human monocyte-derived macrophages and this might limit the efficacy of PDE4 inhibitors to decrease acute inflammatory responses in activated macrophages. As PDE4 inhibition regulates chemokine expression via Epac, it is suggested that Epac antagonists could increase the efficacy and safety of PDE4-inhibitor therapy for chronic inflammatory disorders, if those compounds can be developed [50].

#### 5.4.2. Epac and PKA

PKA and Epac are often associated with the same biological process, in which they fulfill either synergistic or opposite effects. In endothelial cells, both cAMP effector systems coordinate cell adhesion and permeability [119,120] and as such might be a good cellular system to investigate the role of PDEs in co-ordinating the activity of Epac and PKA. Consistent with the prominent expression of Epac in the brain and the importance of cAMP in learning and memory, a recent study demonstrated that both Epac and PKA produced long-lasting effects on synaptic transmission, a process involved in cognition [122]. Direct activation of Epac or PKA with cAMP analogs inhibits collagen synthesis and cardiac fibroblast proliferation [123,124]. In opossum kidney cells and murine kidney slices, activation of PKA or Epac inhibits the activity of the Na<sup>+</sup>/H<sup>+</sup> exchanger 3 which plays a major role in acid-base and extracellular volume regulation in the brush border membrane of proximal tubules[125]. Yet, Epac and PKA may also modulate intercellular communication at the level of channels called gap junctions (GJ). The predominant GJ protein in cardiomyocytes, connexin-43, is cooperatively regulated by PKA and Epac1. Whereas PKA increases the gating function of connexin-43-composed gap junctions, Epac1–Rap1 signaling enhances the accumulation of connexin-43 at cell–cell contacts [40].

cAMP is well known to regulate cell proliferation and thus, a number of studies have sought to define the potential role of Epac and PKA in this process. Epac1 acts synergistically with PKA to mediate either pro-mitogenic or cAMP-dependent cell-cycle arrest dependent on the cell type [105,126] However, other studies have indicated both pro- and anti-proliferative properties of Epac signaling independent of PKA, highlighting the complex and highly cell-type specific nature of this signaling network which also depends on the type of stimuli [42,96,127–129].

Few studies emerged to explain how Epac signaling also varied from PKA signaling to differentially regulate a biological response to cAMP. This is well illustrated with the recent finding of Gaudy et al. [130] who show that activation of the Epac pathway via the glucagon receptor leads to the induction of SOCS3, a cytokine-inducible negative regulator of cytokine receptor signaling in hepatocytes. SOCS3 in turn acting as a feedback inhibitor, then interacts with and inhibits the activity of the PKA catalytic subunit thereby decreasing the expression of gluconeogenic genes via the [130]. In rat cardiomyocytes, Epac and PKA have opposite effects on Ca²⁺-activated force and myofilament Ca²⁺ sensitivity, probably through differential phosphorylation of sarcomeric proteins [67]. As the action of Epac is specifically localized via anchoring proteins in multiprotein complexes containing several effectors of cAMP, it is not surprising that PDE, Epac, PKA and some of their downstream effectors may be interconnected to regulate cellular processes. A prime example of such an association is the control of PDE4B activity on the ability of nuclear Epac1 to drive nuclear export of DNA-PK, while cytosolic PDE4D regulates PKA-mediated nuclear import of DNA-PK [107].

The biological response to cAMP can be regulated by cross-talk between multiple intracellular signaling pathways that are sometimes activated in the same scaffolding complex. An interesting example of this mode of operation is the ability of mAKAP to simultaneously disseminate cAMP signals for Epac, PKA, PDE and the protein phosphatase PP2A at the nuclear membranes of cardiac myocytes [107]. Activation of Epac in this complex results in Rap-dependent inhibition of extracellular signal-regulated kinase 5 (ERK5) and an increase in PDE activity [107]. In contrast, stimulation of PDE4D3 activity through the activation of PKA may limit the inhibitory effect of Epac1 on ERK5 activation [107]. The presence of PP2A within this complex might contribute to regulation of local cAMP gradients by counterbalancing PKA phosphorylation events. Interestingly, in thyroid cells Epac associates in a complex with Akt and PP2A, and the associated phosphatase activity is positively modulated by cAMP in a PKA and Rap1-dependent manner [131]. By analogy of what has been described in cardiac myocytes one could speculate that AKAP-mediated complex is involved in this regulation. Clearly, it is just the beginning of the discovery of the physiological importance of AKAPs in Epac signaling and further studies are required to unravel how the cellular placement of Epac through the association with AKAPs helps to define cAMP compartmentation.

### 6. Concluding remarks

It is just the beginning of the functional characterization of Epac, and obviously the list of its downstream effectors and regulating partners is growing very fast. Yet, most of the biological effects of Epac have been associated to Rap1 and at present the contribution of Rap2 proteins in the function of Epac has to be better defined. Suffice it to say that Epac signaling pathway is complex and our understanding of the biological principles underlying its regulation and biological output is far from complete. The above mentioned findings suggest that local recruitment and/or activation of subpopulations of Epac in specific complexes contributes to cAMP specificity. There is a dynamic control of cellular expression and targeting of Epac into subcellular compartments which, coupled with spatial-temporal dynamics of cAMP, are the key mechanisms that determine Epac functional effects. So far most of the functional studies of Epac proteins were performed in vitro models and the in vivo regulation and functional roles of Epac isoforms remain to be elucidated. This pointed out the importance to develop specific Epac conditional transgenic mouse models and new Epac ligands including inhibitors to better understand the involvement of these cAMP sensors in physiology. In addition, the precise roles of Epac in embryologic development have not yet been elucidated, and this issue remains for future studies. Yet, another major challenge will be to determine whether Epac1 and Epac2 have distinct cellular functions.

Because Epac orchestrates signaling events that regulate important cellular functions, it appears reasonable that defects in Epac genes such as mutations or pathophysiological changes in Epac signaling may underlie certain diseases. Epac signaling microdomains are certainly disorganized during pathological states. Several recent reports now support the involvement of Epac in the manifestation of diseases, such as cardiac hypertrophy, tumor invasion and inflammation suggesting that Epac may represent attractive therapeutic targets for the treatment of various disorders.

#### Acknowledgments

The work of F. Lezoualc'h mentioned herein was supported by grants from Agence Nationale de la Recherche («HyperEpac» Genopath09) and Région Midi-Pyrénées. M. Breckler, M. Berthouze, and AC Laurent were recipients of grants from Région Île-de-France (CODIM), Fondation Lefoulon Delalande, and Groupe de Réflexion sur la Recherche Cardiovasculaire (G.R.R.C)/Fédération Française de Cardiologie (F. F. C).

#### References

[1] D.M. Cooper, Biochem. J. 375 (2003) 517–529.

[2] J.A. Beavo, L.L. Brunton, Nat. Rev. Mol. Cell Biol. 3 (2002) 710–718.

[3] Y. Sassi, L. Lipskaia, G. Vandecasteele, V.O. Nikolaev, S.N. Hatem, F. Cohen Aubart, F.G. Russel, N. Mougenot, C. Vrignaud, P. Lechat, A.M. Lompre, J.S. Hulot, J. Clin. Invest. 118 (2008) 2747–2757.

[4] M.D. Houslay, Trends Biochem. Sci. 35 (2010) 91–100.

[5] R. Fischmeister, L.R. Castro, A. Abi-Gerges, F. Rochais, J. Jurevicius, J. Leroy, G. Vandecasteele, Circ. Res. 99 (2006) 816–828.

[6] J.D. Scott, L.F. Santana, Circulation 121 (2010) 1264–1271.

[7] J. de Rooij, F.J. Zwartkruis, M.H. Verheijen, R.H. Cool, S.M. Nijman, A. Witteinghofer, J.L. Bos, Nature 396 (1998) 474–477.

[8] H. Kawasaki, G.M. Springett, N. Mochizuki, S. Toki, M. Nakaya, M. Matsuda, D.E. Housman, A.M. Graybiel, Science 282 (1998) 2275–2279.

[9] M. Gloerich, J.L. Bos, Annu. Rev. Pharmacol. Toxicol. 50 (2010) 355–375.

[10] H. Kitayama, Y. Sugimoto, T. Matsuzaki, Y. Ikawa, M. Noda, Cell 56 (1989) 77–84.

[11] J.L. Bos, Nat. Rev. Mol. Cell Biol. 4 (2003) 733–738.

[12] S. Rangarajan, J.M. Enserink, H.B. Kuiperij, J. de Rooij, L.S. Price, F. Schwede, J.L. Bos, J. Cell Biol. 160 (2003) 487–493.

[13] G. Borland, B.O. Smith, S.J. Yarwood, Br. J. Pharmacol. 158 (2009) 70–86.

[14] M. Grandoch, S.S. Roscioni, M. Schmidt, Br. J. Pharmacol. 159 (2010) 265–284.

[15] M. Métrich, M. Berthouze, E. Morel, B. Crozatier, A.M. Gomez, F. Lezoualc'h, Pflugers Arch. 459 (2010) 535–546.

[16] Y. Li, I.B. Konings, J. Zhao, L.S. Price, E. de Heer, P.M. Deen, Am. J. Physiol. Ren. Physiol. 295 (2008) F525–F533.

[17] M. Niimura, T. Miki, T. Shibasaki, W. Fujimoto, T. Iwanaga, S. Seino, J. Cell. Physiol. 219 (2009) 652–658.

[18] V.O. Nikolaev, M. Bunemann, L. Hein, A. Hannawacker, M.J. Lohse, J. Biol. Chem. 279 (2004) 37215–37218.

[19] H. Rehmann, A. Rueppel, J.L. Bos, A. Witteinghofer, J. Biol. Chem. 278 (2003) 23508–23514.

[20] H. Rehmann, A. Witteinghofer, J.L. Bos, Nat. Rev. Mol. Cell Biol. 8 (2007) 63–73.

[21] Z. Wang, T.J. Dillon, V. Pokala, S. Mishra, K. Labudda, B. Hunter, P.J. Stork, Mol. Cell. Biol. 26 (2006) 2130–2145.

[22] Y. Li, S. Asuri, J.F. Rebhun, A.F. Castro, N.C. Paranavitana, L.A. Quilliam, J. Biol. Chem. 281 (2006) 2506–2514.

[23] C. Liu, M. Takahashi, Y. Li, S. Song, T.J. Dillon, U. Shinde, P.J. Stork, Mol. Cell. Biol. 28 (2008) 7109–7125.
[24] J.L. Bos, Trends Biochem. Sci. 31 (2006) 680–686.
[25] H. Rehmann, J. Das, P. Knipscheer, A. Wittinghofer, J.L. Bos, Nature 439 (2006) 625–628.
[26] H. Rehmann, E. Arias-Palomo, M.A. Hadders, F. Schwede, O. Llorca, J.L. Bos, Nature 455 (2008) 124–127.
[27] R. Das, M.T. Mazhab-Jafari, S. Chowdhury, S. SilDas, R. Selvaratnam, G. Melacini, J. Biol. Chem. 283 (2008) 19691–19703.
[28] S.M. Harper, H. Wienk, R.W. Wechselberger, J.L. Bos, R. Boelens, H. Rehmann, J. Biol. Chem. 283 (2008) 6501–6508.
[29] J.M. Enserink, A.E. Christensen, J. de Rooij, M. van Triest, F. Schwede, H.G. Genieser, S.O. Doskeland, J.L. Blank, J.L. Bos, Nat. Cell Biol. 4 (2002) 901–906.
[30] A.E. Christensen, F. Selheim, J. de Rooij, S. Dremier, F. Schwede, K.K. Dao, A. Martinez, C. Maenhaut, J.L. Bos, H.G. Genieser, S.O. Doskeland, J. Biol. Chem. 278 (2003) 35394–35402.
[31] G.G. Holz, O.G. Chepurny, F. Schwede, Cell. Signal. 20 (2008) 10–20.
[32] Y. Su, W.R. Dostmann, F.W. Herberg, K. Durick, N.H. Xuong, L. Ten Eyck, S.S. Taylor, K.I. Varughese, Science 269 (1995) 807–813.
[33] H. Rehmann, F. Schwede, S.O. Doskeland, A. Wittinghofer, J.L. Bos, J. Biol. Chem. 278 (2003) 38548–38556.
[34] M.J. Vliem, B. Ponsioen, F. Schwede, W.J. Pannekoek, J. Riedl, M.R. Kooistra, K. Jalink, H.G. Genieser, J.L. Bos, H. Rehmann, Chembiochem 9 (2008) 2052–2054.
[35] O.G. Chepurny, C.A. Leech, G.G. Kelley, I. Dzhura, E. Dzhura, X. Li, M.J. Rindler, F. Schwede, H.G. Genieser, G.G. Holz, J. Biol. Chem. 284 (2009) 10728–10736.
[36] H. Poppe, S.D. Rybalkin, H. Rehmann, T.R. Hinds, X.B. Tang, A.E. Christensen, F. Schwede, H.G. Genieser, J.L. Bos, S.O. Doskeland, J.A. Beavo, E. Butt, Nat Meth. 5 (2008) 277–278.
[37] J.A. Enyeart, H. Liu, J.J. Enyeart, Mol. Pharmacol. 77 (2010) 469–482.
[38] C.L. Zhang, M. Katoh, T. Shibasaki, K. Minami, Y. Sunaga, H. Takahashi, N. Yokoi, M. Iwasaki, T. Miki, S. Seino, Science 325 (2009) 607–610.
[39] T. Tsalkova, A.V. Gribenko, X. Cheng, Assay Drug Dev. Technol. 1 (2011) 88–91.
[40] S. Somekawa, S. Fukuhara, Y. Nakaoka, H. Fujita, Y. Saito, N. Mochizuki, Circ. Res. 97 (2005) 655–662.
[41] E. Baljinnyam, M.S. De Lorenzo, L.H. Xie, M. Iwatubo, S. Chen, J.S. Goydos, M.C. Nowycky, K. Iwatubo, Cancer Res. 70 (2010) 5607–5617.
[42] M. Grandoch, A. Rose, M. ter Braak, V. Jendrossek, H. Rubben, J.W. Fischer, M. Schmidt, A.A. Weber, Br. J. Cancer 101 (2009) 2038–2042.
[43] K.S. Lyle, J.H. Raaijmakers, W. Bruinsma, J.L. Bos, J. de Rooij, Cell. Signal. 20 (2008) 1104–1116.
[44] U. Yokoyama, S. Minamisawa, H. Quan, T. Akaike, M. Jin, K. Otsu, C. Ulucan, X. Wang, E. Baljinnyam, M. Takaoka, M. Sata, Y. Ishikawa, Am. J. Physiol. Heart Circ. Physiol. 295 (2008) H1547–H1555.
[45] W.A. Sands, H.D. Woolson, G.R. Milne, C. Rutherford, T.M. Palmer, Mol. Cell. Biol. 26 (2006) 6333–6346.
[46] S.J. Yarwood, G. Borland, W.A. Sands, T.M. Palmer, J. Biol. Chem. 283 (2008) 6843–6853.
[47] D.M. Aronoff, J.K. Carstens, G.H. Chen, G.B. Toews, M. Peters-Golden, J. Interferon Cytokine Res. 26 (2006) 827–833.
[48] K.S. Tan, A.G. Nackley, K. Satterfield, W. Maxxner, L. Diatchenko, P.M. Flood, Cell. Signal. 19 (2007) 251–260.
[49] X.J. Xu, J.S. Reichner, B. Mastrofrancesco, W.L. Henry Jr., J.E. Albina, J. Immunol. 180 (2008) 2125–2131.
[50] A.L. Hertz, A.T. Bender, K.C. Smith, M. Gilchrist, P.S. Amieux, A. Aderem, J.A. Beavo, Proc. Natl Acad. Sci. U.S.A. 106 (2009) 21978–21983.
[51] X. Cullere, S.K. Shaw, L. Andersson, J. Hirahashi, F.W. Luscinskas, T.N. Mayadas, Blood 105 (2005) 1950–1955.
[52] S. Fukuhara, A. Sakurai, H. Sano, A. Yamagishi, S. Somekawa, N. Takakura, Y. Saito, K. Kangawa, N. Mochizuki, Mol. Cell. Biol. 25 (2005) 136–146.
[53] M.R. Kooistra, M. Corada, E. Dejana, J.L. Bos, FEBS Lett. 579 (2005) 4966–4972.
[54] M.J. Lorenowicz, J. van Gils, M. de Boer, P.L. Hordijk, M. Fernandez-Borja, J. Leukoc. Biol. 80 (2006) 1542–1552.
[55] S. Sehrawat, T. Ernandez, X. Cullere, M. Takahashi, Y. Ono, Y. Komarova, T.N. Mayadas, Blood 117 (2011) 708–718.
[56] A.A. Birukova, T. Zagranichnaya, P. Fu, E. Alekseeva, W. Chen, J.R. Jacobson, K.G. Birukov, Exp. Cell Res. 313 (2007) 2504–2520.
[57] A.A. Birukova, T. Zagranichnaya, E. Alekseeva, G.M. Bokoch, K.G. Birukov, J. Cell. Physiol. 215 (2008) 715–724.
[58] M. Maillet, S.J. Robert, M. Cacquevel, M. Gastineau, D. Vivien, J. Bertoglio, J.L. Zugaza, R. Fischmeister, F. Lezoualc'h, Nat. Cell Biol. 5 (2003) 633–639.
[59] S. Robert, M. Maillet, E. Morel, J.M. Launay, R. Fischmeister, L. Mercken, F. Lezoualc'h, FEBS Lett. 579 (2005) 1136–1142.
[60] N. Zaldua, M. Gastineau, M. Hoshino, F. Lezoualc'h, J.L. Zugaza, FEBS Lett. 581 (2007) 5814–5818.
[61] M. Schmidt, S. Evellin, P.A. Weernink, F. von Dorp, H. Rehmann, J.W. Lomasney, K.H. Jakobs, Nat. Cell Biol. 3 (2001) 1020–1024.
[62] M. Keiper, M.B. Stope, D. Szatkowski, A. Bohm, K. Tysack, F. Vom Dorp, O. Saur, P.A. Oude Weernink, S. Evellin, K.H. Jakobs, M. Schmidt, J. Biol. Chem. 279 (2004) 46497–46508.
[63] M. Métrich, A.C. Laurent, M. Breckler, N. Duquesnes, I. Hmitou, D. Courillau, J.P. Blondeau, B. Crozatier, F. Lezoualc'h, E. Morel, Cell. Signal. 22 (2010) 1459–1468.
[64] M.T. Branham, M.A. Bustos, G.A. De Blas, H. Rehmann, V.E. Zarelli, C.L. Trevino, A. Darszon, L.S. Mayorga, C.N. Tomes, J. Biol. Chem. 284 (2009) 24825–24839.
[65] G. Borland, R.J. Bird, T.M. Palmer, S.J. Yarwood, J. Biol. Chem. 284 (2009) 17391–17403.
[66] L. Pereira, M. Métrich, M. Fernandez-Velasco, A. Lucas, J. Leroy, R. Perrier, E. Morel, R. Fischmeister, S. Richard, J.P. Benitah, F. Lezoualc'h, A.M. Gomez, J. Physiol. 583 (2007) 685–694.
[67] O. Cazorla, A. Lucas, F. Poirier, A. Lacampagne, F. Lezoualc'h, Proc. Natl Acad. Sci. U.S.A. 106 (2009) 14144–14149.
[68] E.A. Oestreich, H. Wang, S. Malik, K.A. Kaproth-Joslin, B.C. Blaxall, G.G. Kelley, R.T. Dirksen, A.V. Smrcka, J. Biol. Chem. 282 (2007) 5488–5495.
[69] E.A. Oestreich, S. Malik, S.A. Goonasekera, B.C. Blaxall, G.G. Kelley, R.T. Dirksen, A.V. Smrcka, J. Biol. Chem. 284 (2009) 1514–1522.
[70] N. Duquesnes, M. Derangeon, M. Metrich, A. Lucas, P. Mateo, L. Li, E. Morel, F. Lezoualc'h, B. Crozatier, Pflugers Arch. 460 (2010) 731–741.
[71] D. Levy, S. Kenchaiah, M.G. Larson, E.J. Benjamin, M.J. Kupka, K.K. Ho, J.M. Murabito, R.S. Vasan, N. Engl. J. Med. 347 (2002) 1397–1402.
[72] E. Morel, A. Marcantoni, M. Gastineau, R. Birkedal, F. Rochais, A. Garnier, A.M. Lompre, G. Vandecasteele, F. Lezoualc'h, Circ. Res. 97 (2005) 1296–1304.
[73] M. Métrich, A. Lucas, M. Gastineau, J.L. Samuel, C. Heymes, E. Morel, F. Lezoualc'h, Circ. Res. 102 (2008) 959–965.
[74] M. Métrich, E. Morel, M. Berthouze, L. Pereira, P. Charron, A.M. Gomez, F. Lezoualc'h, Pharmacol. Rep. 61 (2009) 146–153.
[75] T. Shibasaki, H. Takahashi, T. Miki, Y. Sunaga, K. Matsumura, M. Yamanaka, C. Zhang, A. Tamamoto, T. Satoh, J. Miyazaki, S. Seino, Proc. Natl Acad. Sci. U.S.A. 104 (2007) 19333–19338.
[76] I. Dzhura, O.G. Chepurny, G.G. Kelley, C.A. Leech, M.W. Roe, E. Dzhura, P. Afshari, S. Malik, M.J. Rindler, X. Xu, Y. Lu, A.V. Smrcka, G.G. Holz, J. Physiol. (2010).
[77] S.A. Hinke, Sci. Signal. 2 (2009) e54.
[78] N. Ozaki, T. Shibasaki, Y. Kashima, T. Miki, K. Takahashi, H. Ueno, Y. Sunaga, H. Yano, Y. Matsuura, T. Iwanaga, Y. Takai, S. Seino, Nat. Cell Biol. 2 (2000) 805–811.
[79] G. Kang, O.G. Chepurny, B. Malester, M.J. Rindler, H. Rehmann, J.L. Bos, F. Schwede, W.A. Coetzee, G.G. Holz, J. Physiol. 573 (2006) 595–609.
[80] G. Kang, C.A. Leech, O.G. Chepurny, W.A. Coetzee, G.G. Holz, J. Physiol. 586 (2008) 1307–1319.
[81] G. Liu, S.M. Jacobo, N. Hilliard, G.H. Hockerman, J. Pharmacol. Exp. Ther. 318 (2006) 152–160.
[82] M.N. Helms, J. Self, H.F. Bao, L.C. Job, L. Jain, D.C. Eaton, Am. J. Physiol. Lung Cell. Mol. Physiol. 291 (2006) L610–L618.
[83] E.C. Aromataris, M.L. Roberts, G.J. Barritt, G.Y. Rychkov, J. Physiol. 573 (2006) 611–625.
[84] J. Ster, F. de Bock, F. Bertaso, K. Abitbol, H. Daniel, J. Bockaert, L. Fagni, J. Physiol. 587 (2009) 101–113.
[85] G.I. Purves, T. Kamishima, L.M. Davies, J.M. Quayle, C. Dart, J. Physiol. 587 (2009) 3639–3650.
[86] S.L. Mironov, E.Y. Skorova, J. Neurochem. (2011 Jan 31).
[87] M. Lopez De Jesus, M.B. Stope, P.A. Oude Weernink, Y. Mahlke, C. Borgermann, V.N. Ananaba, C. Rimmbach, D. Rosskopf, M.C. Michel, K.H. Jakobs, M. Schmidt, J. Biol. Chem. 281 (2006) 21837–21847.
[88] G.X. Shi, H. Rehmann, D.A. Andres, Mol. Cell. Biol. 26 (2006) 9136–9147.
[89] E.O. Brennesvik, C. Ktori, J. Ruzzin, E. Jebens, P.R. Shepherd, J. Jensen, Cell. Signal. 17 (2005) 1551–1559.
[90] B. Ponsioen, J. Zhao, J. Riedl, F. Zwartkruis, G. van der Krog, M. Zaccolo, W.H. Moolenaar, J.L. Bos, K. Jalink, EMBO Rep. 5 (2004) 1176–1180.
[91] K.K. Dao, K. Teigen, R. Kopperud, E. Hodneland, F. Schwede, A.E. Christensen, A. Martinez, S.O. Doskeland, J. Biol. Chem. 281 (2006) 21500–21511.
[92] G. Borland, M. Gupta, M.M. Magiera, C.J. Rundell, S. Fuld, S.J. Yarwood, Mol. Pharmacol. 69 (2006) 374–384.
[93] M. Gupta, S.J. Yarwood, J. Biol. Chem. 280 (2005) 8109–8116.
[94] H. Jing, J.H. Yen, D. Ganea, J. Biol. Chem. 279 (2004) 55176–55186.
[95] L.M. DiPilato, X. Cheng, J. Zhang, Proc. Natl Acad. Sci. U.S.A. 101 (2004) 16513–16518.
[96] S. Haag, M. Warnken, U.R. Juergens, K. Racke, Naunyn Schmiedebergs Arch. Pharmacol. 378 (2008) 617–630.
[97] E. Huston, M.J. Lynch, A. Mohamed, D.M. Collins, E.V. Hill, R. MacLeod, E. Krause, G.S. Baillie, M.D. Houslay, Proc. Natl Acad. Sci. U.S.A. 105 (2008) 12791–12796.
[98] F.C. Mei, J. Qiao, O.M. Tsygankova, J.L. Meinkoth, L.A. Quilliam, X. Cheng, J. Biol. Chem. 277 (2002) 11497–11504.
[99] J. Qiao, F.C. Mei, V.L. Popov, L.A. Vergara, X. Cheng, J. Biol. Chem. 277 (2002) 26581–26586.
[100] M.M. Magiera, M. Gupta, C.J. Rundell, N. Satish, I. Ernens, S.J. Yarwood, Biochem. J. 382 (2004) 803–810.
[101] L. Aumo, M. Rusten, G. Mellgren, M. Bakke, A.E. Lewis, Endocrinology 151 (2010) 2151–2161.
[102] B. Ponsioen, M. Gloerich, L. Ritsma, H. Rehmann, J.L. Bos, K. Jalink, Mol. Cell. Biol. 29 (2009) 2521–2531.
[103] S. Mangmool, A.K. Shukla, H.A. Rockman, J. Cell Biol. 189 (2010) 573–587.
[104] T. Shibasaki, Y. Sunaga, K. Fujimoto, Y. Kashima, S. Seino, J. Biol. Chem. 279 (2004) 7956–7961.
[105] D. Hochbaum, K. Hong, G. Barila, F. Ribeiro-Neto, D.L. Altschuler, J. Biol. Chem. 283 (2008) 4464–4468.
[106] K.L. Dodge-Kafka, L. Langeberg, J.D. Scott, Circ. Res. 98 (2006) 993–1001.
[107] K.L. Dodge-Kafka, J. Soughayer, G.C. Pare, J.J. Carlisle Michel, L.K. Langeberg, M.S. Kapiloff, J.D. Scott, Nature 437 (2005) 574–578.
[108] I.M. Nijholt, A.M. Dolga, A. Ostrovereu, P.G. Luiten, M. Schmidt, U.L. Eisel, Cell. Signal. 20 (2008) 1715–1724.
[109] K.M. Woolfrey, D.P. Srivastava, H. Photowala, M. Yamashita, M.V. Barbolina, M.E. Cahill, Z. Xie, K.A. Jones, L.A. Quilliam, M. Prakriya, P. Penzes, Nat. Neurosci. 12 (2009) 1275–
[112] D. Hochbaum, G. Barila, F. Ribeiro-Neto, D.L. Altschuler, J. Biol. Chem. 286 (2011) 859–866.

[113] D.M. Collins, H. Murdoch, A.J. Dunlop, E. Charych, G.S. Baillie, Q. Wang, F.W. Herberg, N. Brandon, A. Prinz, M.D. Houslay, Cell. Signal. 20 (2008) 2356–2369.

[114] C. Liu, M. Takahashi, Y. Li, T.J. Dillon, S. Kaech, P.J. Stork, Mol. Cell. Biol. 30 (2010) 3956–3969.

[115] M. Conti, J. Beavo, Annu. Rev. Biochem. 76 (2007) 481–511.

[116] D.R. Raymond, L.S. Wilson, R.L. Carter, D.H. Maurice, Cell. Signal. 19 (2007) 2507–2518.

[117] J. Leroy, A. Abi-Gerges, V.O. Nikolaev, W. Richter, P. Lechene, J.L. Mazet, M. Conti, R. Fischmeister, G. Vandecasteele, Circ. Res. 102 (2008) 1091–1100.

[118] V.W. van Hinsbergh, G.P. van Nieuw Amerongen, Vascul. Pharmacol. 39 (2002) 171–172.

[119] S.J. Netherton, J.A. Sutton, L.S. Wilson, R.L. Carter, D.H. Maurice, Circ. Res. 101 (2007) 768–776.

[120] S.N. Rampersad, J.D. Ovens, E. Huston, M.B. Umana, L.S. Wilson, S.J. Netherton, M.J. Lynch, G.S. Baillie, M.D. Houslay, D.H. Maurice, J. Biol. Chem. 285 (2010) 33614–33622.

[121] B. Omar, E. Zmuda-Trzebiatowska, V. Manganiello, O. Goransson, E. Degerman, Cell. Signal. 21 (2009) 760–766.

[122] M. Ouyang, L. Zhang, J.J. Zhu, F. Schwede, S.A. Thomas, Proc. Natl Acad. Sci. U.S.A. 105 (2008) 11993–11997.

[123] U. Yokoyama, H.H. Patel, N.C. Lai, N. Aroonsakool, D.M. Roth, P.A. Insel, Proc. Natl Acad. Sci. U.S.A. 105 (2008) 6386–6391.

[124] F. Villarreal, S.A. Epperson, I. Ramirez-Sanchez, K.G. Yamazaki, L.L. Brunton, Am. J. Physiol. Cell Physiol. 296 (2009) C1178–C1184.

[125] K.J. Honegger, P. Capuano, C. Winter, D. Bacic, G. Stange, C.A. Wagner, J. Biber, H. Murer, N. Hernando, Proc. Natl Acad. Sci. U.S.A. 103 (2006) 803–808.

[126] R.C. Hewer, G.B. Sala-Newby, Y.J. Wu, A.C. Newby, M. Bond, J. Mol. Cell. Cardiol. 50 (2011) 87–98.

[127] K.M. Kassel, T.A. Wyatt, R.A. Panettieri Jr., M.L. Toews, Am. J. Physiol. Lung Cell. Mol. Physiol. 294 (2008) L131–L138.

[128] F. Lezoualc'h, Am. J. Physiol. Cell Physiol. 297 (2009) C797–C799.

[129] H. Yan, D.A. Deshpande, A.M. Misior, M.C. Miles, H. Saxena, E.C. Riemer, R.M. Pascual, R.A. Panettieri, R.B. Penn, FASEB J. 25 (2011) 389–397.

[130] A.M. Gaudy, A.H. Clementi, J.S. Campbell, A.V. Smrcka, R.A. Mooney, J. Biol. Chem. 285 (2011) 41356–41365.

[131] K. Hong, L. Lou, S. Gupta, F. Ribeiro-Neto, D.L. Altschuler, J. Biol. Chem. 283 (2008) 23129–23138.

[132] I. Gekel, E. Neher, J. Neurosci. 28 (2008) 7991–8002.

[133] T. Sakaba, E. Neher, Nature 424 (2003) 775–778.

[134] N. Zhong, R.S. Zucker, J. Neurosci. 25 (2005) 208–214.

[135] J.N. Gelinas, J.L. Banko, M.M. Peters, E. Klann, E.J. Weeber, P.V. Nguyen, Learn. Mem. 15 (2008) 403–411.

[136] J.S. O'Neill, E.S. Maywood, J.E. Chesham, J.S. Takahashi, M.H. Hastings, Science 320 (2008) 949–953.

[137] S. Suzuki, U. Yokoyama, T. Abe, H. Kiyonari, N. Yamashita, Y. Kato, R. Kurotani, M. Sato, S. Okumura, Y. Ishikawa, J. Biol. Chem. 285 (2010) 24248–24259.

[138] J. Ster, F. De Bock, N.C. Guerineau, A. Janossy, S. Barrere-Lemaire, J.L. Bos, J. Bockaert, L. Fagni, Proc. Natl Acad. Sci. U.S.A. 104 (2007) 2519–2524.

[139] T.B. Hucho, O.A. Dina, J.D. Levine, J. Neurosci. 25 (2005) 6119–6126.

[140] N. Eijkelkamp, H. Wang, A. Garza-Carbajal, H.L. Willemen, F.J. Zwartkruis, J.N. Wood, R. Dantzer, K.W. Kelley, C.J. Heijnen, A. Kavelaars, J. Neurosci. 30 (2010) 12806–12815.

[141] H.J. Kwak, K.M. Park, H.E. Choi, K.S. Chung, H.J. Lim, H.Y. Park, Cell. Signal. 20 (2008) 803–814.

[142] T. Bryn, M. Mahic, J.M. Enserink, F. Schwede, E.M. Aandahl, K. Tasken, J. Immunol. 176 (2006) 7361–7370.

[143] J. Liu, X. Zhao, J. Cao, Q. Xue, X. Feng, X. Liu, F. Zhang, B. Yu, J. Mol. Neurosci. (2010).

[144] S. Gerlo, P. Verdood, R. Kooijman, J. Interferon Cytokine Res. 30 (2010) 883–891.

[145] M. Grandoch, M. Lopez de Jesus, P.A. Oude Weernink, A.A. Weber, K.H. Jakobs, M. Schmidt, Cell. Signal. 21 (2009) 609–621.

[146] C. Basoni, M. Nobles, A. Grimshaw, C. Desgranges, D. Davies, M. Perretti, I.M. Kramer, E. Genot, FASEB J. 19 (2005) 822–824.

[147] R. Murtazina, O. Kovbasnjuk, N.C. Zachos, X. Li, Y. Chen, A. Hubbard, B.M. Hogema, D. Steplock, U. Seidler, K.M. Hoque, C.M. Tse, H.R. De Jonge, E.J. Weinman, M. Donowitz, J. Biol. Chem. 282 (2007) 25141–25151.

[148] L. Balasubramanian, J.S. Sham, K.P. Yip, Pflugers Arch. 456 (2008) 747–754.

[149] J.M. Banales, T.V. Masyuk, S.A. Gradilone, A.I. Masyuk, J.F. Medina, N.F. LaRusso, Hepatology 49 (2009) 160–174.

[150] Y. Wang, J.D. Klein, M.A. Blount, C.F. Martin, K.J. Kent, V. Pech, S.M. Wall, J.M. Sands, J. Am. Soc. Nephrol. 20 (2009) 2018–2024.

[151] N. Yano, D. Suzuki, M. Endoh, T.C. Zhao, J.F. Padbury, Y.T. Tseng, J. Biol. Chem. 282 (2007) 18819–18830.

[152] S.S. Roscioni, B.G. Dekkers, A.G. Prins, M.H. Menzen, H. Meurs, M. Schmidt, H. Maarsingh, Br. J. Pharmacol. 162 (2011) 193–209.

[153] S.S. Roscioni, H. Maarsingh, C.R. Elzinga, J. Schuur, M. Menzen, A.J. Halayko, H. Meurs, M. Schmidt, J. Cell. Mol. Med. (2010).

[154] S.K. Huang, S.H. Wettlaufer, J. Chung, M. Peters-Golden, Am. J. Respir. Cell Mol. Biol. 39 (2008) 482–489.

[155] T. Shibasaki, H. Takahashi, T. Miki, Y. Sunaga, K. Matsumura, M. Yamanaka, C. Zhang, A. Tamamoto, T. Satoh, J. Miyazaki, S. Seino, Proc. Natl Acad. Sci. U.S.A. 104 (2007) 19333–19338.

[156] G. Kang, O.G. Chepurny, G.G. Holz, J. Physiol. 536 (2001) 375–385.

[157] K. Fujimoto, T. Shibasaki, N. Yokoi, Y. Kashima, M. Matsumoto, T. Sasaki, N. Tajima, T. Iwanaga, S. Seino, J. Biol. Chem. 277 (2002) 50497–50502.

[158] U.K. Misra, S.V. Pizzo, J. Cell. Biochem. 108 (2009) 998–1011.

[159] L. Gao, Y. Feng, R. Bowers, M. Becker-Hapak, J. Gardner, L. Council, G. Linette, H. Zhao, L.A. Cornelius, Cancer Res. 66 (2006) 7880–7888.

[160] J.C. Lissitzky, D. Parriaux, E. Ristorcelli, A. Verine, D. Lombardo, P. Verrando, Cancer Res. 69 (2009) 802–809.

[161] S. Dremier, F. Vandeput, F.J. Zwartkruis, J.L. Bos, J.E. Dumont, C. Maenhaut, Biochem. Biophys. Res. Commun. 267 (2000) 7–11.

[162] R.H. Adamson, J.C. Ly, R.K. Sarai, J.F. Lenz, A. Altangerel, D. Drenckhahn, F.E. Curry, Am. J. Physiol. Heart Circ. Physiol. 294 (2008) H1188–H1196.

[163] R.C. Doebele, F.T. Schulze-Hoepfner, J. Hong, A. Chlenski, B.D. Zeitlin, K. Goel, S. Gomes, Y. Liu, M.K. Abe, J.E. Nor, M.W. Lingen, M.R. Rosner, Blood 114 (2009) 4592–4600.

[164] S. Namkoong, C.K. Kim, Y.L. Cho, J.H. Kim, H. Lee, K.S. Ha, J. Choe, P.H. Kim, M.H. Won, Y.G. Kwon, E.B. Shim, Y.M. Kim, Cell. Signal. 21 (2009) 906–915.

[165] K.P. Yip, Am. J. Physiol. Ren. Physiol. 291 (2006) F882–F890.

[166] G. Kang, O.G. Chepurny, M.J. Rindler, L. Collis, Z. Chepurny, W.H. Li, M. Harbeck, M.W. Roe, G.G. Holz, J. Physiol. 566 (2005) 173–188.

[167] P. Mayer, A.V. Hinze, A. Harst, I. von Kugelgen, Cardiovasc. Res. (2010 Dec 31).

[168] S. Traver, M. Marien, E. Martin, E.C. Hirsch, P.P. Michel, Mol. Pharmacol. 70 (2006) 30–40.
