
# The Evolutionary History of YAP and the Hippo/YAP Pathway

Dror Hilman${}^{1}$ and Uri Gat${}^{*,1}$

${}^{1}$Department of Cell and Developmental Biology, The Alexander Silberman Institute of Life Sciences, Edmond Safra Campus at Givat Ram, The Hebrew University, Jerusalem, Israel

*Corresponding author: E-mail: gatu@vms.huji.ac.il.

Associate editor: Billie Swalla

## Abstract

The Hippo/YAP pathway plays an important role in animal organ size control, which it exerts by regulating tissue proliferation and apoptosis rates as a response to developmental cues, cell contact, and density. With the ever increasing advance in genome sequencing and analysis tools, our understanding of the animal world and its evolution has greatly increased in the recent years. We used bioinformatic tools to study the evolution of the Hippo/YAP pathway focusing on the transcriptional coactivator YAP, which is a pivotal effector of the pathway. The aim was to establish the origin and mode of development of YAP and its pathway in the animal world. Some pathway members can be already identified in single-celled eukaryotes like the yeast that have preceded multicellular animals. Interestingly, we can find most of the components that are present in human in the sea-anemone *Nematostella*, which belongs to a very basal group of metazoans, the cnidarians. All the major domains of YAP have been conserved between cnidarians and mammals, and YAP can be identified even in the more basal placozoan clade. We show a very high degree of conservation in regions such as the WW and the TEAD-binding domains, TEAD being the major DNA-binding partner of YAP. Remarkably, we found that the location of an intron in the WW1 genomic region has been invariant along an evolutionary span of over 700 My. We have followed the evolutionary changes in YAP and in other main components of the pathway from the first metazoans such as sponges, described the phylogenetic relationships between the YAP genes and indicated where YAP and other components have been secondarily lost. Evidence is provided that YAP and its binding partner TEAD demonstrate strong coevolution. This gives further support for the importance of the TEAD–YAP association. Beyond contributing to an understanding of the evolutionary history of this pathway, we have provided insights into the “birth” of this pathway, its functions and its mode of operation in animals with different body plans, development, and life styles.

Key words: YAP, yorkie, Hippo/YAP pathway, organ size control, evolution, coevolution.

## Introduction

A fundamental aspect of the development of multicellular organisms is the ability to control the size and shape of their tissues and organs. In recent years, the abundance and wealth of genomic data and molecular biology techniques have revealed a “toolkit” of developmental genes, which served in generating the remarkable diversity of the body plans of organisms (De Robertis 2008; King et al. 2008; Degnan et al. 2009; Srivastava et al. 2010). It appears that all the major animal groups share a subset of developmental pathways. These pathways are repeatedly employed over vast spans of evolutionary time to build ever more complex forms, using similar building blocks (De Robertis 2008). One of the most intriguing phenomena of metazoan development is their ability to precisely control organ size by coordinating cell proliferation rate during embryogenesis and maturation and in certain cases during regeneration and wound healing processes (Dong et al. 2007; Kango-Singh and Singh 2009).

In the past decade, a new pathway that controls cell proliferation and cell apoptosis in response to developmental cues has been described (reviewed in Harvey and Tapon 2007; Kango-Singh and Singh 2009; Grusche et al. 2010; Oh and Irvine 2010; Sudol and Harvey 2010; Zhao et al. 2010). This pathway is termed the Hippo or the Hippo/

YAP pathway, after some of the major components that constitute it. The pathway includes proteins involved in reception of signals, signaling mediators, a transcription co-activator, and executing transcription factors (see fig. 1 for a simplified scheme; Justice et al. 1995; Xu et al. 1995; Tapon et al. 2002; Harvey et al. 2003; Pantalacci et al. 2003; Wu et al. 2003; Huang et al. 2005; Kango-Singh and Singh 2009). It has been hypothesized that several upstream regulators are involved in producing growth arrest signals upon stimulation. These include: 1) The arthropod/mammalian planar cell polarity protocadherins Fat/Ft1-4 and dachsous/DCHS, 2) The FERM-domain proteins Expanded/FRMD6/Willin and Merlin/NF2, 3) The WW-domain protein Kibra, the Ras-familly protein dRASSF/RASSF1-10, apico-basal polarity proteins Lgl/Lgl1/2, atypical protein kinase C, and Crumbs/Crb1-3. The signals elicited are transmitted through the kinases Hpo/Mst1,2 and Warts/Lats1,2, with the help of the adaptor proteins Salvador/WW45 and Mats/Mob to the coactivators Yki/YAP and TAZ. These induce the growth program by activating several transcription factors and micro RNAs. The major YAP/TAZ transcriptional partners in the program, which provide the DNA-binding function, were shown to be Scalloped/TEAD1-4, but others like Runt/Runx and p73 have been reported as well (Strano et al. 2001, 2005; Levy et al. 2007; Vitolo et al. 2007). The phosphorylation

© The Author 2011. Published by Oxford University Press on behalf of the Society for Molecular Biology and Evolution. All rights reserved. For permissions, please e-mail: journals.permissions@oup.com

Mol. Biol. Evol. 28(8):2403–2417. 2011   doi:10.1093/molbev/msr065   Advance Access publication March 16, 2011   2403

Hilman and Gat · doi:10.1093/molbev/msr065

![Diagram](image.png)

across distant phyla. Our results shed light on the source of this pathway, its major forms and modifications along animal evolution. They also may provide novel insights into why the pathway has emerged and into the modes of its evolution.

# Methods

## Finding Orthologs for YAP Pathway Proteins

To find orthologs of Hippo/YAP pathway genes, we employed a BlastP (Altschul et al. 1990) search of the human proteins: FAT4, DCHS1, CSNK1E, RASSF1 FRMD6, NF2 (Merlin), KIBRA, MOB1, MST1/2, LATS1/2, YAP, TAZ, and TEAD1 from refseq (Pruitt et al. 2007; table 1). Other pathway associated proteins were analyzed as well. These included p73, RUNX1-3 AJUBA and c-ABL, as well as the hedgehog pathway proteins, SHH, SMO, PTCH, and GLI1-3. The later were used as controls. All human sequences were searched against the refseq protein data from different organisms, representing different metazoan and nonmetazoan groups. All blast hits were filtered, and only sequences with blast score >150 and sequence length >50 were examined. We then tested each sequence using reciprocal blast search and assigned it as an ortholog if the best hit of one blast search matched the best hit of the other (supplementary data 1, Supplementary Material online). If the sequences were not found reciprocally in two genomes, we picked the sequence with the best coverage, with score >150, relative identity >30%, and relative similarity >40%. If the parameters were lower, we assigned it as “nonortholog.” When we did not find an ortholog, we verified the lack of orthologous sequences with tBlastX against the genomic sequence and expressed sequence tags (EST) libraries of the relevant organism. The existence or absence of an ortholog was also checked by blast search of the protein sequence in the Ortho-MCL4 database of orthologs (Li et al. 2003; Chen et al. 2007). Sequence alignment of several Hippo/YAP pathway components is illustrated in supplementary data 2, Supplementary Material online.

## Sequence Alignment and Phylogenetic Analysis

Sequence alignments were performed on orthologous proteins using MUSCLE (Edgar 2004) with default parameters (supplementary data 2, Supplementary Material online), which were manually curated if necessary. Conservation of motifs and phosphorylation sites were checked by hand using data from Entrez gene (Maglott et al. 2007) and Uniprot (Consortium 2010). Phylogenetic analysis was conducted using PhyML (Guindon and Gascuel 2003; Guindon et al. 2005) and FastTree (Price et al. 2010) for maximum likelihood (ML) analysis, with default parameters. Neighbor joining (NJ) trees were built using MEGA4 (Kumar et al. 2008). Phylogeny support was verified with the bootstrap consensus tree inferred from 1,000 replicates. In NJ trees, the evolutionary distances were computed using the Poisson correction method and are given in units of the number of amino acid substitutions per site. The rate variation among sites was modeled with a gamma

Evolution of YAP and the Hippo Pathway · doi:10.1093/molbev/msr065

Table 1. The Hippo/YAP Pathway Protein Components Studied in This Article.

| Protein                     | Human Refseq Acc.                          | Drosophila Refseq Acc. |
|-----------------------------|-------------------------------------------|------------------------|
| DCHS1/ds                    | NP_003728.1                               | NP_523446.2            |
| FAT4/ft                     | NP_078858.4                               | NP_477497.1            |
| CSNK1/Dco                   | NP_001885.1                               | NP_733414.1            |
| RASSF1/Rassf                | NP_009113.3                               | NP_651126.3            |
| NF2/Merlin (Mer)            | NP_000259.1                               | NP_523413.1            |
| WWC1/Kibra                  | NP_001155133.1                           | NP_001034055.1         |
| FRMD6/expanded (ex)         | NP_001035946.1                           | NP_476840.2            |
| Mst1/2/Hippo                | NP_006273.1/NP_006272.2                   | NP_611427.1            |
| Salvador (Sav)              | NP_068590.1                               | NP_788721.1            |
| Mob1/Mats                   | NP_775739.1                              | NP_651041.3            |
| Lats1/2/Warts               | NP_004681.1/NP_055387.2                   | NP_733403.1            |
| YAP/TAZ/Yorkie              | NP_001123617.1/NP_001161750.1             | NP_001036568.2         |
| TEAD1/scalloped (sd)        | NP_068780.2                               | NP_001096989.1         |

distribution using default parameters. Positions containing alignment gaps and missing data were eliminated only in pairwise sequence comparisons so as to keep all major protein domains in the alignment.

**Testing Coevolution**

To assess the strength of coevolution between two proteins, we followed the methodology described in Pazos et al. (2005) and Sato et al. (2006). Pairwise distances between sequences in different taxa were measured in MEGA 4 with pairwise deletion and Poisson correction was used for amino acids substitution. Distances between different proteins in diverse organisms were plotted, and Pearson's correlation coefficient (product-moment r) was used to determine the overall correlation between distances of two genes (supplementary data 3, Supplementary Material online).

**Retrieving Genomic Data**

All the genomic data used in this analysis were retrieved from Entrez genome and Entrez nucleotide databases. Protein sequences were retrieved from the National Center for Biotechnology Information Refseq database. Orthologs for comparing YAP and TAZ were obtained from OrthoMCL4DB to retrieve a large group of vertebrate orthologs.

**Results and Discussion**

**YAP Proteins—A Metazoan Novelty**

Detecting the evolutionary origin of genes that belong to specific pathways may give insights into how they have contributed to the formation of novel structures and mechanisms in animals over the course of evolution. YAP orthologs have previously been explored almost exclusively in higher vertebrates and in Drosophila. We set out to find the origin of YAP and the Hippo/YAP pathway in the tree of life (fig. 2). We first explored the YAP protein, a critical connecting switch between the signaling pathway and the transcriptional program. We could not detect a bona fide YAP gene in the group that is considered to represent the most basal or “ancient” metazoans—the sponges. Searching for conserved regions of YAP in the demosponge

Amphimedon queenslandica whole-genome shotgun sequencing (Srivastava et al. 2010) and in the homoscleromorph sponge Oscarella carmela EST database (Nichols et al. 2006; Hemmrich and Bosch 2008) did not yield an obvious ortholog. Actually, proteins with two conserved WW domain proteins in sponges and in the nonmetazoan eukaryotes choanoflagellates Monosiga brevicollis (King et al. 2008) usually aligned best with the human gene WWP1-E3 ubiquitin ligase, which seems to be conserved in nearly all eukaryotes we tested. Searching further, YAP orthologs were not found in yeasts (Saccharomyces cerevisiae) or in plants (Arabidopsis thaliana). The most basal

Fig. 2. The evolutionary tree of taxa studied in this work. The gross estimated divergence time ranges of the major clades are mentioned near each group name. Divergence times are based on Otsuka and Sugaya (2003), Pisani et al. (2004), Nichols and Wörheide (2005), Hejnol et al. (2009), and Han et al. (2010).

Table 2. YAP Orthologs in Various Metazoans and Their Relation to Human YAP.

| Organism                | Ortholog (refseq)      | Length (aa) | Similarity (%) |
|-------------------------|------------------------|-------------|----------------|
| Trichoplax adhaerens    | XP_002108065.1         | 447         | 29             |
| Nematostella vectensis  | XP_001627495.1         | 412         | 58             |
| Tribolium castaneum     | XP_970492.1            | 404         | 53             |
| Apis mellifera         | XP_391844.2            | 511         | 45             |
| Drosophila melanogaster | NP_001036568.2         | 395         | 45             |
| Strongylocentrotus purpuratus | XP_789542.2        | 531         | 28             |
| Ciona intestinalis      | XP_002130260.1         | 529         | 46             |
| Branchiostoma floridae | XP_002595229.1         | 86 (partial) |                |
| Danio rerio            | NP_001132952.1         | 442         | 88             |
| Taeniopygia guttata    | XP_002198083.1         | 506         | 88             |
| Homo sapiens           | NP_001123617.1         | 504         | 100            |

YAP ortholog we found is that of the very basal metazoan, the placozoan Trichoplax adhaerens. Its existence in this organism dates the first appearance of YAP to after the emergence of the sponges in the metazoan clade, up to 1.8 billion years ago (Nichols and Wörheide 2005; table 2). We then proceeded to follow YAP's evolution in the metazoans. Interestingly, we found that when compared with human YAP, the highest percentage of blast positive positions among nonvertebrates was found in the sea-anemone Nematostella vectensis YAP (58% similarity). Nematostella belongs to the basal metazoan group of the cnidarians, estimated to have separated from the bilaterian ancestor 1.2–0.7 billion years ago (Otsuka and Sugaya 2003; Han et al. 2010). The Nematostella ortholog is more similar to human YAP than the Drosophila Yki (45% similarity) despite the much greater phylogenetic distance of cnidarians, as compared with arthropods, from humans (see fig. 2; Otsuka and Sugaya 2003; Pisani et al. 2004). The least conserved YAP genes found are the putative YAP proteins of Strongylocentrotus purpuratus and Tricho-

plax, with 28%/29% conservation, respectively (table 2). Surprisingly, the YAP protein seems to be absent in the nematodes Caenorhabditis elegans and C. briggsae because searching for YAP orthologs in their genomes retrieved sequences that aligned with the aforementioned WWP1. Since nematodes are animals that evolved after the separation of bilaterian animals from cnidarians (Hejnol et al. 2009; fig. 2), it is likely that a functional loss of YAP occurred in the nematode lineage. The TAZ proteins, which are structurally similar to YAP and are regulated by the pathway in the same fashion, are considered to be YAP paralogs and represent a late evolutionary novelty. They were not found in nonvertebrates.

**Structural Comparison of YAP Protein Orthologs**

Next, we explored changes in YAP protein structure during the course of evolution of the YAP family. The full alignment of the YAP protein sequences we retrieved is presented in supplementary figure 1, Supplementary Material online. Figure 3 shows a schematic alignment

Conservation level: 0.0 0.3 0.5 0.7 0.9

Fig. 3. Schematic alignment of YAP/TAZ orthologs. The structure of the YAP proteins in the different taxa along evolution is displayed. Conservation level between the organisms shown is tone coded as depicted below. Missing areas in the alignment are blank. Domains and phosphorylation sites are marked in the figure. Protein sizes are shown to the right of each sequence.

Evolution of YAP and the Hippo Pathway · doi:10.1093/molbev/msr065

![Diagram](https://i.imgur.com/yourimage.png)

Fig. 4. Genomic structure of YAP in representative metazoans. Major domains are marked with rectangles. The transcript coding region is drawn to scale and introns positions are illustrated but their size is not drawn to scale.

of representative YAP orthologs in a wide array of metazoans. The most noticeable YAP conserved domains are the two WW domains and the TEAD interaction domain. These are conserved from placozoans to the late bilaterians like human. The Trichoplax YAP seems to have only one WW domain, which aligned best with the second WW domain of vertebrates and insect YAPs. The WW2 domain is missing in TAZ proteins which contain only the first WW domain (WW1).

Another conserved region is the C-terminal transactivation domain. It seems to consist of several separated subdomains and its composition varies between different clades (fig. 3). As previously shown (Oka and Sudol 2009), a conspicuous conserved PDZ-binding motif can be found in the most distal sequence of the C-termini of cnidarians, sea-urchins, and chordates. However, in insects, this sequence is divergent (fig. 3). The coiled-coil (c-c) domain that is important for TAZ and YAP dimerization (Murakami et al. 2005) is also conserved in vertebrates and cnidarians. The c-c domain in the insects Tribolium castaneum and Apis mellifera is present partially and it is completely absent in Drosophila Yki. This domain is highly divergent in Ciona intestinalis (fig. 3, supplementary fig. 1, Supplementary Material online) suggesting an altered function with respect to its dimerization and interaction activities. To conclude the analysis of the domain structure, we observed that the proline rich region in the N-terminal area of mammalian and avian YAP is missing in other taxa (fig. 3). This indicates a yet unidentified function that arose late in evolution.

YAP activity was shown to be regulated by phosphorylation and indeed several phosphorylation sites found in humans are highly conserved in all metazoans tested, especially the five sites (S61, S109, S127, S164, and S397; Zhao et al. 2007; Hao et al. 2008; fig. 3). These are known to be phosphorylated by the Lats kinases in mammals and three are phosphorylated by the Warts kinase in Drosophila. The phosphoserine sites that are located prior to the WW domains are known to be important for Hpo/Wts-mediated inhibition in Yki and YAP and this is likely reflected in the high degree of conservation of the regions in which they reside (fig. 3 and supplementary fig. 1, Supplementary Material online). Mammalian YAP tyrosine phosphorylation was shown to promote apoptosis in response to DNA damage (Levy et al. 2008). However, the tyrosine that is involved, Y391, seems to be conserved only in vertebrates. This tyrosine is aligned with phenylalanine in other deuterostomes (C. intestinalis, S. purpuratus) and is absent in protostomes (see supplementary fig. 1, Supplementary Material online).

## The Genomic Composition of YAP Genes in Metazoans

A genomic analysis of the YAP gene family demonstrated the overall increased complexity of YAP gene structure over evolution (fig. 4). The basal YAP genes have 2 or 3 exons, whereas the later forms have 8–9 exons. The second WW domain appears to reside in a distinct exon in all bilaterians, which can lead to differential splicing isoforms that contain only the WW1 domain. In both human

Hilman and Gat · doi:10.1093/molbev/msr065

A YAP WW1 conserved intron:

| H. sapiens | AC TTC TTA AA | GTAAGTGAA ... TTGTCTTAG | T CAC ATC GAT |
| --- | --- | --- | --- |
| T. guttata | AC TTT CTT AA | GTAAGTAAA ... TTGTCTTAG | T CAT ATT GAT |
| D. rerio | AC TTC CTA AA | GTGAGTTGT ... TCCCCCTAG | C CAT AAT GAC |
| C. intestinalis | AT TAT ATG AA | GTAAGATAA ... TTTTTTAG | C CAT AAC GAT |
| B. floridae | AT TAT CTA AA | GTAAGATAA ... CTCCCTGCAG | C CAT AAC AAC |
| S. purpuratus | GG ATC CTG CA | GTAAGTTGA ... TCCTAACAG | T CAT AGC AAT |
| D. melanogaster | AC TAC TTG AA | GTGAGTTTG ... TACTTGCAG | T CAT ACT ACA |
| T. castaneum | AC TAT CTT AA | GTAAGAACAA ... CGTTTGCAG | C CAT ATA ACT |
| N. vectensis | AT TTT ATG AA | GTAAGTACC ... CTCGTGTAG | C CAT TAC GAA |

| H. sapiens | WEMAKTSSGQRYFLNHIDQTTT WQDPRK |
| --- | --- |
| T. guttata | WEMAKTPSGQRYFLNHIDQTTT WQDPRK |
| D. rerio | WEMAKTPSGQRYFLNHNDQTTT WQDPRK |
| C. intestinalis | WSAAKTADGQQYYMNHNDRSTT WEDPRI |
| S. purpuratus | WEMAVPTPGQKYFLDHSNQTTT WEDPRK |
| D. melanogaster | WEQAKTNDGQIYYLNHTTKSTQ WEDPRI |
| T. castaneum | WEQARTPEGQLYYLNHITRTTT WEDPRK |
| N. vectensis | WEMRTSPTGQPYFMNHYEQITT WQDPRK |

B TAZ WW conserved intron:

| H. sapiens | AC TTC CTC AA | GTAAGTCAG ... TTCTTTTAG | T CAC ATA GAA |
| --- | --- | --- | --- |
| D. rerio TAZ | WEMAFTPNGQKYFLNHIEKITTW |
| T. guttata TAZ | WEMALTQTGQRYFLNHIEKITTW |
| H. sapiens TAZ | WEMTFTATGQRYFLNHIEKITTW |

Fig. 5. A conserved intron in the WW1 domain. An intron is found in most YAP and TAZ orthologs within the WW1 domain. The intron position and phase inside the codon for "N" is highly conserved. The illustration shows the genomic sequence flanking the intron boundaries in different organisms as well as the amino acids coded by this region.

YAP and Drosophila Yki, there are isoforms documented that lack the second WW domain (fig. 4). These isoforms may mimic the ancient condition of only one WW domain in the Placozoans. The vertebrate TAZ (WWTR1) contains one WW domain, which is better aligned with the first WW domain of YAP (figs. 3 and 4, supplementary fig. 1, Supplementary Material online).

All the YAP (and TAZ) genes that we analyzed, except for the T. adhaerens gene which does not have WW1, contain a remarkably conserved intron within the WW1 coding region (figs. 4 and 5). The intron boundaries fall exactly in the same codon, which codes for a conserved asparagine (N) residue and always start in the same phase (+2) of this codon. The most diverse form of this intron boundary can be observed in the sequence of S. purpuratus. Here, the intron is contained in an aspartic acid codon and differs from the chordate and the arthropod clades sequences in several other positions around the intron boundaries (fig. 5A) The N. vectensis sequence, on the other hand, shares some features with chordate sequences and some with arthropods around the intron boundaries. This may reflect an ancient state before the separation of the major bilaterian groups. The extreme conservation of this intron strongly suggests that the same YAP structure appeared in a eumetazoan ancestor of both cnidarians and all bilaterians. Why is this intron so well conserved over such a great span of evolutionary time? The reason is not known, but we speculate that the intron harbors an important binding site for some as yet undetermined critical factor which conferred strong conservation to its proximity. Another hypothesis may be that without the presence of this intron, this WW domain may be prone to duplication by a reverse transcription-based mechanism. The result may be evolutionary unfavorable.

### Phylogeny of YAP and TAZ Proteins

In order to examine the phylogenetic relations of the YAP family orthologs, we have utilized several tree-building techniques. Both NJ and ML trees, which are considered reliable for the analysis of distant sequences, were able to distinguish between three major groups, arranging the chordates, arthropods, and early nonbilaterians in separate clades (fig. 6A and B). In the ML tree, the nonbilaterians T. adhaerens and N. vectensis form a clade of their own and do not form an outgroup to the bilaterians. The NJ analysis, however, positions the T. adhaerens YAP as the most basal, being an outgroup to N. vectensis and all bilaterians. N. vectensis YAP, on the other hand, clustered as an outgroup to the bilaterians as could be expected from its structure. Surprisingly, the deuterostome S. purpuratus tended to group closer to the protostome clade in both trees but with low bootstrap support. Because some parts of the S. purpuratus YAP (Spr-YAP) display more similarity to chordates and other parts to arthropods, the Spr-YAP sequence likely represents an early bilaterian form, different from both the chordates and the arthropods, as it has emerged before the separation of these major bilaterian clades. In both NJ and ML trees, the C. intestinalis YAP falls close to the root of the chordate clade but with a long branch. This suggests strong divergence of its sequence after the urochordate and chordate lineage separation. A similar phenomenon can be observed in insects, where the Drosophila Yki sequence seems to be more diverse (fig. 3, supplementary fig. 1, Supplementary Material online) and probably represents a more varied and novel sequence when compared with the more basal-like insect groups which were included in the analysis.

We went on and analyzed the later evolutionary radiations of the YAP family of proteins. The origin of the YAP paralogous gene TAZ was explored by aligning the multiple TAZ and YAP proteins that we retrieved from the databases and generating their NJ cladogram (fig. 6C). The resulting plot clusters the vertebrates' YAP and TAZ proteins in different clades. Both, however, have similar distance from out groups of vertebrates. This suggests that the TAZ gene probably originated from gene duplication of YAP in early vertebrates. The conserved nature of the WW1 domain in TAZ and the identical localization of the intron in it also alludes to a common origin. We did not find TAZ in chordates such as the cephalochordate B. lanceolatum or in the urochordate C. intestinalis genomes. This dates the YAP/TAZ duplication to a late stage in chordates evolution, possibly associating it with the

Evolution of YAP and the Hippo Pathway · doi:10.1093/molbev/msr065

A ML tree
C YAP and TAZ duplication

T. adhaerens

N. vectensis

D. rerio
T. guttata
H. sapiens
B. floridae
C. intestinalis

S. purpuratus
A. mellifera
T castaneum
D. melanogaster

Insects

Chordates

Vertebrates

0.2

B NJ tree

H. sapiens
T. guttata
D. rerio
B. floridae
C. intestinalis

S. purpuratus
D. melanogaster
T castaneum
A. mellifera
N. vectensis

T. adhaerens

Insects

0.2

78
P. troglodytes TAZ
98
H. sapiens TAZ
C. lupus familiaris TAZ
86
M musculus TAZ
99
R. norvegicus TAZ
99
M. domestica TAZ
G. gallus TAZ
D. rerio TAZ

vertebrates

TAZ

76

D. rerio YAP
100
G. gallus YAP
94
M musculus YAP
78
C. lupus familiaris YAP
74
P. troglodytes YAP
70
H. sapiens YAP

YAP

54

C. intestinalis YAP
N. vectensis YAP
D. melanogaster Yki
T. adhaerens YAP

0.1

Fig. 6. Phylogenetics of YAP. ML (A) and NJ (B) unrooted trees of YAP orthologs from distant metazoans. All sequences were downloaded from refseq and aligned with MUSCLE. Trees were constructed as described in Methods. The bootstrap consensus tree inferred from 1,000 replicates. The percentage of replicate trees in which the associated ortholog clustered together in the bootstrap test is shown next to the branches. The trees are drawn to scale, with branch lengths in the same units as those of the evolutionary distances used to infer the phylogenetic tree. In the NJ tree, the distances were computed using the Poisson correction method and are in the units of the number of amino acid substitutions per site. The rate variation among sites was modeled with a gamma distribution. All positions containing alignment gaps and missing data were eliminated only in pairwise sequence comparisons. There were a total of 796 positions in the final data set. (C) The TAZ proteins are paralogs of vertebrate YAPs. The cladogram demonstrates the relations between the YAP and TAZ paralogs, showing that TAZ is clustered closer to the vertebrates YAP than to the invertebrates' ortholog. The high similarity of vertebrate and Nematostella YAP (table 2 and fig. 3) relative to Drosophila Yki, puts Nematostella, closer to the vertebrates, when compared mainly with vertebrates proteins. The lack of other insects in this analysis artificially clusters Drosophila outside the bilaterians and not as presented in (A) and (B). The cladogram was inferred using the NJ method. YAP and TAZ sequences were downloaded from the orthoMCL database. The tree was built with MEGA 4 using default parameters. The optimal tree is shown and the bootstrap test (1,000 replicates) is shown next to the branches. The tree is drawn to scale, with branch lengths in the same units as those of the evolutionary distances used to infer the phylogeny.

whole-genome duplication in the ancestral vertebrate lineage (Dehal and Boore 2005). A perplexing question is the reason for the emergence of the TAZ paralogs, especially because a single WW1 harboring splicing isoform seemingly of the same structure (fig. 4), exists in both vertebrates and Drosophila. Future experiments exploring the exact function of TAZ, as opposed to YAP with its splicing forms, may shed light on the rationale for this late evolutionary development and on its benefit for vertebrates.

YAP and the Hippo/YAP Pathway—Developing Complexity

What is the evolutionary status of YAP in the context of the entire pathway? How did the pathway as we know it evolve? Did the components appear as a full set along with a novel animal group or did they emerge gradually in several steps to establish and elaborate the pathway? In order to address these questions and to learn about the evolutionary history of the pathway, we searched for the orthologs of the core pathway components as well as

the upstream members of the pathway in representative single-celled eukaryotes and in metazoan groups (table 1).

Single-Celled Eukaryotes

Figure 7 depicts the composition of the pathway along evolution from eukaryotes to mammals. We can find bona fide orthologs of TEAD and Mob in the yeast S. cerevisiae, which may represent the most ancient orthologs of the pathway. We did not find them in prokaryotes. Moreover, YAP orthologs were not found in plants, reducing the possibility of pathway loss during the early evolution of eukaryotes in general. Previously, it has been suggested that the roots of the pathway may be represented by the mitotic exit network (MEN) and the separation initiation network pathways in the budding and fission yeast correspondingly (Bardin and Amon 2001; Harvey and Tapon 2007). Our results cannot, however, categorically identify the yeast Cdc15 and Dbf2 as orthologs of Hippo and Warts, respectively, as previously suggested because they tend to align better with other paralogs of these kinases.

Hilman and Gat · doi:10.1093/molbev/msr065

![Diagram](attachment:diagram.png)

*Fig. 7.* Summary of the Hippo/YAP pathway evolution. The figure shows the conservation of the core components of the Hippo/YAP signaling pathway in representative metazoans. Orthologs between different organisms are placed in the same relative position. The colors code for the proposed antiquity of the genes in evolution. When orthologs were not found, the ortholog place remained blank with a red circle. In case the ortholog identity is not sure, a question mark (?) was added, and the putative paralog that matches this sequence was written above if found. Divergence times of major metazoa groups are presented next to the relevant branches. Our analysis shows that the nearly complete core pathway known from bilaterians is represented in the cnidarian *Nematostella vectensis*. The pathway is mostly absent in sponges and nonmetazoans and is partially present in placozoans. The nematode *Caenorhabditis elegans* and its relatives *Caenorhabditis briggsae* are deficient in several of the pathway components (represented here by the *C. elegans* conserved components). This may indicate of a functional loss of major components in nematodes. The YAP paralogs TAZ were found only in vertebrates and most likely evolved as a result of duplication in early vertebrates or prevertebrate chordates.

That said, the similarity of the genetic module and the genes involved, along with the similar interaction with Mob like protein in yeast, suggest a deep evolutionary connection that warrants further examination. Warts/Lats has been reported to be localized to the centrosome in interphase, translocate to the mitotic spindle in metaphase and anaphase, and then to localize to the midbody in telophase in human cells (Nishiyama et al. 1999; Morisaki et al. 2002). Additionally, embryonic cells from Lats2 knockout mice display multiple mitosis defects including abnormal exit from mitosis (Yabuta et al. 2007). As this expression pattern and function is reminiscent of that of Dbf2 in yeast (Bardin and Amon 2001), it may represent an ancient role for the pathway in mitotic exit that has remained in metazoans up to humans (Praskova et al. 2008).

Tec1, the putative yeast TEAD ortholog, regulates signaling of one of the mitogen-activated protein (MAP) kinase pathways in the budding yeast, a pathway that is involved in invasive growth (Wang et al. 2009). We could not find a direct connection between the function of the above MEN factors that are related to the Hippo/YAP pathway and that of TEAD/Tec1. However, the founding member of the Mst/Hippo family of kinases, Ste-20 in yeast (Radu and Chernoff 2009), is part of the MAP kinase network that modulates Tec1 activity. It is thus possible that Ste-20 is the ancestral Mst and that the origin of the pathway is related to the Tec1-regulated invasive response. In conclusion, there are several candidates in yeast for elements of the current Hippo/Yap pathway. These components may have joined to evolve the metazoan form of the pathway. In addition, the pathway may have consolidated along

metazoan evolution due to newly acquired roles of these progenitors as well as to the emergence and integration of new pathway members.

Analysis of the genome of the unicellular eukaryote choanoflagellate *M. brevicollis*, considered to be the closest relative of metazoans, did not yield any more orthologs of the core pathway. However, upstream elements such as the protocadherins fat/ds may have first emerged in choano-flagellates (King et al. 2003, 2008). This would have allowed colony formation and further on in evolution they could have been utilized for the more intricate and complex tasks of tissue growth control, which necessitated formation of the Hippo/YAP pathway by recruitment of other preexisting factors and incorporation of novel elements.

**Sponges and Placozoans**

Moving along into the metazoan clade, we can identify a true ortholog of Mst/Hippo kinase in sponges and all groups beyond, but still no Sav, Lats or YAP (fig. 7). A hallmark in the evolution of this pathway is the appearance of the dynamic effector YAP. The most basal YAP ortholog we have detected is that of the placozoan *T. adhaerens*, which is a eumetazoan with simple but organized tissues (Srivastava et al. 2008). This YAP ortholog includes the TEAD-binding domain and four out of five of the Lats/Wts phosphorylation sites, but only one WW domain, which is similar to WW2 (see fig. 3 and supplementary fig. 1, Supplementary Material online). Thus, this YAP is different in structure from all the following orthologs. Taking into account the lack of other components of the pathway in this organism, such as the Lats/Wts kinase, this suggests that the pathway in the placozoans is still in a primordial form and likely functions differently from that in bilaterians. It will be interesting to find out the mode of action of this rudimentary and minimal pathway in this little studied organism.

**The Sea-Anemone Nematostella**

The cnidarian *N. vectensis* is considered to be the closest outgroup to the bilaterians among the known extant taxa (Finnerty et al. 2004; Dunn et al. 2008; Hejnol et al. 2009; Sperling et al. 2009; Ryan et al. 2010). Our analysis clearly shows that this group of basal nonbilaterians contains most of the components of the “modern” Hippo/YAP pathway (fig. 7). This includes the complete core Hippo machinery and all the supposed upstream signaling proteins except for the ortholog of Expanded which we could not find. Notably, the *Nematostella* YAP demonstrates almost all the features recognized in many later bilaterian groups. In addition to the conserved elements that we described above for *T. adhaerens*, these include two well-conserved WW domains and a PDZ-binding C-terminal pentapeptide identical to the human form (see fig. 3 and supplementary fig. 1, Supplementary Material online). Intriguingly, when compared with different bilaterian groups, the *Nematostella* YAP shows the highest similarity to the vertebrate YAP (see table 2 and supplementary fig. 1, Supplementary Material online). This high level of resemblance between *Nematostella* and vertebrates has

been demonstrated before for many developmental genes (Ryan et al. 2007; Hejnol et al. 2009; Saina et al. 2009). It has been explained by the basal nature of both the sea-anemone genome and the vertebrate genome, which is much less derived than many of the protostome groups like insects and nematodes. The question then arises whether this similarity in protein structure is also reflected in a similar modi operandi among these very distant animal groups.

**The Bilaterian Groups—Conservation and Divergence**

When we considered the status of the pathway within the bilaterian groups for which genomic data are available, we generally found all the components that were established earlier. However, as in other major developmental pathways, the complexity increased and we were able to detect the appearances of additional factors and more paralogs added to the preexisting pathway components. Thus, the Mst, Lats, TEAD, and YAP genes underwent duplication events during deuterostome evolution (fig. 7). This may have conferred more flexibility to the pathway as the various paralogs could allow modulation of function in different organs. Our phylogenetic analysis points out that the ancient YAP sequence diverged to two main groups of proteins: the arthropod Yki and the deuterostome YAP, which includes the chordates. The sequence alignments suggest that the arthropod Yki forms are much more divergent and harbor many more changes when compared with the chordate YAP species that are more “basal” and resemble the sea-anemone ancestral YAP. Several interaction elements that are important in vertebrate YAP/TAZ, such as the SH3-binding domain (Espanel and Sudol 2001) and the Y391-like tyrosine (Levy et al. 2008), are absent in insects and early metazoans. This may point to the generation of novel functions of YAP during deuterostomes and vertebrate evolution. The new phosphorylation site that appeared on tyrosine can be associated with more elaborate apoptosis promotion abilities for which vertebrates YAPs were enlisted (Levy et al. 2008). Interestingly, although the site is a vertebrates novelty, the kinase responsible for the phosphorylation, c-Abl, is an ancient metazoan gene that is already found in the sponge.

Inspection of the *Drosophila* Yki shows that indeed, its structure is different in that the C-terminus is much shorter and is missing many sequences attributed to transcription activation. It also entirely lacks the c-c domain found in deuterostomes and in a partial form also in other insects. Moreover, the PDZ-binding domain, which was recently shown to be critical for the function of YAP in vertebrates (Oka and Sudol 2009), is highly modified in Yki. The divergent nature of Yki likely indicates that a significant change in the gene function has transpired during arthropods evolution. This change may have occurred in order to accommodate a unique form of embryonic development such as the whole-body segmentation mode typical to *Drosophila*. Unlike many other insect and arthropod groups flies lack a growth zone that gradually gives rise to posterior segments during its embryogenesis.
The apparent lack of obvious YAP orthologs in the nematodes *C. elegans* and *C. briggsae* and the lack of other pathway components (Ex and Kibra, fig. 7) is another dramatic example of a major change in the pathway in a protostome group. A similar phenomenon of the absence of crucial and conserved developmental genes in nematodes was also previously described (*Aboobaker* and *Blaxter* 2003a, 2003b), wherein the authors portrayed the disorganization and decay of the Hox gene cluster in *C. elegans*. The Hpo/Mst orthologs Cst-1/2 in *C. elegans* are Ste-20 like kinases, which were shown to increase nematode lifespan by phosphorylating Daf-16, the mammalian FoxO ortholog that is a known substrate of Mst kinases (*Lehtinen et al.* 2006). The *C. elegans* Lats/Warts, Ce-Wts-1, is important to various developmental functions in *C. elegans* including body length control by the TGF-beta Sma/Mab pathway (*Cai et al.* 2009). It is also associated with lifespan control (*Curran* and *Ruvkun* 2007), whereas the putative TEAD ortholog, egl-44, is involved in mechanosensory neuron cell fate determination (*Wu et al.* 2001). Further experimental data are required in order to establish the function of these and the remaining Hippo/YAP pathway elements in nematodes and to reveal the changes that the pathway has undergone in this derived group.

It will also be interesting to explore the status of the pathway in other protostome groups such as mollusks as more sequence data become available.

### Coevolution of Genes in the Hippo/YAP Pathway

We were interested in testing whether there is any evidence for coevolution between components of the pathway, which may indicate the evolutionary origin of a functional association between them. In recent studies, an alignment of the YAP-binding domain of TEAD1 and the TEAD-binding domain of YAP, along with the solved structure of YAP and TEAD1 (*Chen et al.* 2010; *Li et al.* 2010), showed the putative coevolution of these areas in YAP and TEAD1. Specifically, residues in the TEAD-binding domain of YAP, and in the TEAD partner protein, which physically interact in the 3D model, are also highly conserved within metazoans (*Li et al.* 2010). To test whether YAP and TEAD have coevolved, we used phylogenetic evidence to compare the calculated pairwise lengths between all YAP orthologs and TEAD orthologs across distant taxa. This method was used in earlier work to demonstrate strong protein–protein interactions between two proteins (*Pazos et al.* 2005). The principle of this analysis is that in order to assess evolutionary connection, one examines how the evolution of one sequence, which is measured by the pairwise distances of any two orthologs, is correlated to the distances of the orthologs of another gene. We expect that if two genes interacted and thus coevolved, we will observe a correlation between the pairwise distances (supplementary data 3, Supplementary Material online).

Our coevolution analysis indeed suggested a very strong correlation between YAP and TEAD pairwise distances, with a correlation coefficient $r = 0.85$, $P = 1.1 \times 10^{-16}$ (fig. 8A). This strongly supports a functional association between these two proteins. Such coevolution was also shown in the transcription factor and coactivator pair Runx-CBFβ (*Sullivan et al.* 2008). This coevolution suggests a strong selection pressure toward the interaction of YAP and TEAD and strengthens the hypothesis that TEAD is a key downstream effector of the pathway (*Ota* and *Sasaki* 2008; *Li et al.* 2010). YAP can interact with other transcriptional partners and thus we analyzed its coevolution with the aforementioned Runx and p73 cofactors. Although YAP seems to be highly correlated to p73 ($r = 0.85$, $P = 6.2 \times 10^{-7}$, supplementary data 3, Supplementary Material online), which suggests that they strongly interacted during evolution, there was no indication of coevolution with the Runx proteins. This in turn could indicate a recent interaction or that the interaction contact regions are very short.

Extending the analysis we showed that YAP/Yki exhibit a high degree of coevolution with Mob1/Mats (fig. 8B) and Sav1 (fig. 8C) and a lower but positive correlation with Lats1/Wts (fig. 8D) and Mst2/Hpo (fig. 8E). All these function in a close signaling network (*Pan* 2010). The lower correlations seen for the later (fig. 8D and E) could be due to the multiple other partners of these kinases and of YAP. Some of the other components in the pathway that are known or suspected to interact with one another also demonstrate a positive correlation such as Mob1-Mst2 (*Wei et al.* 2007; fig. 8F), Mob1-Lats1 (*Lai et al.* 2005; *Wei et al.* 2007; fig. 8G), whereas Mer-Ex display a high correlation (*McCartney et al.* 2000; *Hamaratoglu et al.* 2006; *Yu et al.* 2010; fig. 8H). As could be expected not all previously known binding partners showed coevolution in this assay, whereas other pairs of proteins, such as YAP/TAZ and Merlin, exhibited unexpected high correlation, which could indicate that they may physically interact (see supplementary data 3, Supplementary Material online). Other pairs of proteins that are not known to bind or interact show a series of distances with much lower correlation. For example, the predicted correlation between YAP and FAT4 yielded an $r = 0.27$ (fig. 8I) and with DCHS1, $r = -0.02$ (8J). The correlation between FAT4 and DCHS1, that do interact in *Drosophila* (*Cho et al.* 2006), is much higher with $r = 0.65$ (fig. 8K). β-catenin, another transcriptional coactivator that acts with the same operational logic as YAP, but in a different pathway, yields an $r = 0.10$ for YAP (fig. 8L).

To further test this technique on another signaling pathway, we analyzed the main components of the hedgehog pathway, which is also composed of an assembly of ancient eukaryotic and novel metazoan genes (*Matus et al.* 2008). We found an $r = 0.68$ between Hedgehog and its receptor, the tumor suppressor Patched1, (PTCH1), an $r = 0.32$ of PTCH1 to the smoothened receptor and an $r = 0.52$ of PTCH1 to the GLI2 transcription factor of this pathway (supplementary data 3, Supplementary Material online). These correlations point to a general trend of coevolution of interacting components in developmental pathways.

Overall, our data demonstrate the deep evolutionary connections between the components of the Hippo/YAP pathway, which are manifested in their coevolution.
Evolution of YAP and the Hippo Pathway · doi:10.1093/molbev/msr065

Fig. 8. Coevolution of pathway components. The coevolution of Hippo/YAP pathway components was demonstrated as described in Methods. Pairwise distances were calculated in MEGA 4. A correlation coefficient and P values (two-tailed) are presented for each graph on the bottom right corner. Dashed lines present linear regression lines. This analysis shows a strong correlation between the amino acids substitutions of Hippo/YAP pathway interacting proteins in different organisms. The correlation is measured by Pearson's correlation coefficient. Panels 8J and 8L show analysis of the correlation of YAP with pathway related proteins that are likely not interacting directly with YAP and thus show lower correlations. Panel 8L show the correlation of YAP with β-catenin which resembles YAP molecular function as a transcriptional coactivator, but is unrelated to the pathway. β-catenin was used as a control to examine the strength of the coevolution forces between Hippo/YAP pathway components.
and are presumably necessary to maintain the functionality of the network. In addition, we show the potential of this approach for defining possible protein–protein interactions that were previously not explored.

### Hypotheses Regarding the Origin and Evolutionary Role of YAP and the Hippo/YAP Pathway

From our evolutionary survey, we hypothesize that the pathway as we know it in human and in *Drosophila*, with the majority of its associated machinery, can be dated to the last common ancestor of cnidarians and bilaterians, estimated to exist more than 700 Ma ([Otsuka and Sugaya 2003](https://doi.org/10.1093/molbev/msr065); [Hejnol et al. 2009](https://doi.org/10.1093/molbev/msr065); [Han et al. 2010](https://doi.org/10.1093/molbev/msr065)). The finding of other components of the pathway that have much broader clusters of orthologs in distant phyla, well before the advent of metazoans estimated up to 1.8 billion years ago ([Nichols and Wörheide 2005](https://doi.org/10.1093/molbev/msr065)), indicates that the preexisting segments of this pathway likely played roles both in mitotic regulation processes like mitotic exit and in environmental signaling. Moreover, they were redirected to add multicellular tissue growth control tasks during metazoans body plan evolution. This new recruitment was necessary to perform novel regulatory functions. These, in turn, rewired or modified some existing components, eventually adding new components such as YAP. This paradigm of using preexisting proteins and incorporating components with a new combination of domains, or recruiting altogether novel domains on top of existing ones as well as the deployment of totally new proteins, is increasingly recognized as the “evolutionary workshop.” It created the new pathways (e.g., the Wnt pathway; [Adamska et al. 2010](https://doi.org/10.1093/molbev/msr065)) and the modified functional complexes (e.g., adhesion junctions; [Nichols et al. 2006](https://doi.org/10.1093/molbev/msr065)), necessary for emergence of new life forms such as the metazoans and bilaterians ([Srivastava et al. 2010](https://doi.org/10.1093/molbev/msr065)).

Although TEAD and Mob are ancient eukaryotic proteins, YAP seems to be a dynamic metazoan novelty in which an existing WW domain protein was modified and combined with novel domains such as the TEAD-binding domain and the PDZ-interacting domain. What is interesting about the evolution of YAP is that it has likely emerged to form this pathway in ancestral eumetazoans like the placozoans/cnidarians/bilaterian progenitors, which may suggest that the pathway is necessary for the growth control of more complex organized tissue as compared with sponge cells. In addition, YAP structure has been modified along the course of evolution with the putative addition of a second WW domain and the PDZ-binding domain in cnidarians. Later, more minor alterations of the scheme appeared, such as diversification of the PDZ-binding pentapeptide in insects and the formation of the TAZ paralogs in the vertebrates.

What new insights can be gleaned from our evolutionary study about the function of the Hippo/YAP pathway? The apparent lack of a YAP ortholog and several other pathway components in the ecdysozoan *C. elegans* demonstrates that the full pathway may not be essential for all types

of metazoan life forms. The variant insect forms suggest that the pathway had to be modified and diversified for some others. The absence of YAP may indicate a dramatic change in the developmental mode of nematodes toward an early “deterministic” pattern ([Schierenberg 2001](https://doi.org/10.1093/molbev/msr065)). In this mode, which is the classical alternative to the “regulative” mode of development, a signaling pathway that controls cell growth in response to environmental cues may no longer be needed. Thus, we speculate that this pathway has primarily evolved in order to regulate tissue and organ growth during metazoan “regulative” phases of development, which are flexible and rely on cell–cell interaction and environmental signals. However, the full complement of the pathway may not be essential for the “determinative (mosaic)” like type of development which is programmed in the developing organism. Exploration of the Hippo/YAP pathway’s function in developmental stages that exhibit these types of development, both of which often appear at distinct stages in the development of model organisms may show if indeed our speculation is valid.

The Hippo/YAP pathway’s evolutionary tale may thus provide insight into the evolution of tissue and organ size control in multicellular animals. It features YAP as one of the most important tools in the metazoan developmental toolbox.

## Conclusions

We explored the origin and evolution of the Hippo/YAP pathway, which controls tissue proliferation rates and organ size in animals. Some of the pathway’s elements were found already in protists like yeast and the choanoflagellates, whereas the first YAP progenitor was detected in the placozoan *T. adhaerens*. The full pathway as known in man can be first identified in the basal metazoan cnidian *N. vectensis*. Following the evolution of the pathway along the bilaterian groups shows an overall increase in its complexity, such as the emergence of additional paralogs of the core kinases and the advent of the TAZ genes, the vertebrate YAP paralogs. However, some bilaterians, such as *Drosophila* show divergence of the YAP effector. This may reflect a unique mode of their embryonic development pattern. At the same time, other groups, like the nematode *C. elegans*, show a loss of YAP and several other components, which may hint at their simplified mode of cellular growth control.

Thus, in reconstructing the origin of this pathway, we can envision the following scenario: The evolution of organized tissues and organs in metazoans required the invention of a mechanism for tissue growth regulation. For this purpose, a new genetic network that can connect intertissue signals with cellular proliferation control was necessary. This pathway has evolved in accordance with the developmental growth control programs of the various animal forms. Our findings indicate that the emergence of the “advanced” Hippo/YAP pathway is associated with the appearance of an organized body plan that incorporates distinct
organs and tissues. Even though the pathway is highly conserved, and its genes have undergone strong coevolution, it is adapted to the developmental program of the organism and shows plasticity that is reflected in divergence of some critical domains of its components.

**Supplementary Material**

Supplementary datas 1–3 and figure 1 are available at Molecular Biology and Evolution online (http://www.mbe.oxfordjournals.org/).

**Acknowledgments**

The authors would like to thank Drs. Ariel Chipman, Liran Carmel, Rachel Green, and Marshall Devor of the Hebrew University in Jerusalem for their critical reading of this manuscript and helpful suggestions. This study was supported by the Israel Science Foundation (grant 825/07).

**References**

Aboobaker A, Blaxter M. 2003a. Hox gene evolution in nematodes: novelty conserved. Curr Opin Genet Dev. 13:593–598.

Aboobaker AA, Blaxter ML. 2003b. Hox gene loss during dynamic evolution of the nematode cluster. Curr Biol. 13:37–40.

Adamska M, Larroux C, Adamski M, Green K, Lovas E, Koop D, Richards GS, Zwafink C, Degnan BM. 2010. Structure and expression of conserved Wnt pathway components in the demosponge Amphimedon queenslandica. Evol Dev. 12:494–518.

Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. 1990. Basic local alignment search tool. J Mol Biol. 215:403–410.

Bardin AJ, Amon A. 2001. MEN and SIN: what's the difference? Nat Rev Mol Cell Biol. 2:815–826.

Cai Q, Wang W, Gao Y, Yang Y, Zhu Z, Fan Q. 2009. Ce-wts-1 plays important roles in Caenorhabditis elegans development. FEBS Lett. 583:3158–3164.

Chen F, Mackey AJ, Vermunt JK, Roos DS. 2007. Assessing performance of orthology detection strategies applied to eukaryotic genomes. PLoS One. 2:e383.

Chen L, Chan SW, Zhang X, Walsh M, Lim CJ, Hong W, Song H. 2010. Structural basis of YAP recognition by TEAD4 in the hippo pathway. Gene Dev. 24:290–300.

Cho E, Feng Y, Rauskolb C, Maitra S, Fehon R, Irvine KD. 2006. Delineation of a fat tumor suppressor pathway. Nat Genet. 38:1142–1150.

Consortium TU. 2010. The universal protein resource (UniProt) in 2010. Nucleic Acids Res. 38:D142–D148.

Curran SP, Ruvkun G. 2007. Lifespan regulation by evolutionarily conserved genes essential for viability. PLoS Genet. 3:e56.

De Robertis E. 2008. Evo-devo: variations on ancestral themes. Cell 132:185–195.

Degnan BM, Vervoort M, Larroux C, Richards GS. 2009. Early evolution of metazoan transcription factors. Curr Opin Genet Dev. 19:591–599.

Dehal P, Boore JL. 2005. Two rounds of whole genome duplication in the ancestral vertebrate. PLoS Biol. 3:e314.

Dong J, Feldmann G, Huang J, Wu S, Zhang N, Comerford SA, Gayyed MF, Anders RA, Maitra A, Pan D. 2007. Elucidation of a universal size-control mechanism in Drosophila and mammals. Cell 130:1120–1133.

Dunn CW, Hejnol A, Matus DQ, et al. (18 co-authors). 2008. Broad phylogenomic sampling improves resolution of the animal tree of life. Nature 452:745–749.

Edgar RC. 2004. MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res. 32: 1792–1797.

Espanel X, Sudol M. 2001. Yes-associated protein and p53-binding protein-2 interact through their WW and SH3 domains. J Biol Chem. 276:14514–14523.

Finnerty JR, Pang K, Burton P, Paulson D, Martindale MQ. 2004. Origins of bilateral symmetry: Hox and dpp expression in a sea anemone. Science. 304:1335–1337.

Gruschke FA, Richardson HE, Harvey KF. 2010. Upstream regulation of the Hippo size control pathway. Curr Biol. 20:R574–R582.

Guindon S, Gascuel O. 2003. A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood. Syst Biol. 52:696–704.

Guindon S, Lethiec F, Duroux P, Gascuel O. 2005. PHYML online—a web server for fast maximum likelihood-based phylogenetic inference. Nucleic Acids Res. 33:W557–W559.

Hamaratoglu F, Willecke M, Kango-Singh M, Nolo R, Hyun E, Tao C, Jafar-Nejad H, Halder G. 2006. The tumour-suppressor genes NF2/Merlin and expanded act through Hippo signalling to regulate cell proliferation and apoptosis. Nat Cell Biol. 8:27–36.

Han J, Kubota S, Uchida HO, Stanley GD Jr., Yao X, Shu D, Li Y, Yasui K. 2010. Tiny sea anemone from the lower cambrian of China. PloS One. 5:e13276.

Hao Y, Chun A, Cheung K, Rashidi B, Yang X. 2008. Tumor suppressor LATS1 is a negative regulator of oncogene YAP. J Biol Chem. 283:5496–5509.

Harvey K, Tapon N. 2007. The salvador-warts-hippo pathway—an emerging tumour-suppressor network. Nat Rev Cancer. 7:182–191.

Harvey KF, Pfleger CM, Hariharan IK. 2003. The Drosophila mst ortholog, Hippo, restricts growth and cell proliferation and promotes apoptosis. Cell. 114:457–467.

Hejnol A, Obst M, Stamatakis A, et al. (17 co-authors). 2009. Assessing the root of bilaterian animals with scalable phylogenomic methods. Proc R Soc B. 276:4261–4270.

Hemmrich G, Bosch TCG. 2008. Compagen, a comparative genomics platform for early branching metazoan animals, reveals early origins of genes regulating stem-cell differentiation. BioEssays 30:1010–1018.

Hisaoaka M, Tanaka A, Hashimoto H. 2002. Molecular alterations of h-warts/LATS1 tumor suppressor in human soft tissue sarcoma. Lab Invest. 82:1427–1435.

Huang J, Wu S, Barrera J, Matthews K, Pan D. 2005. The Hippo signaling pathway coordinately regulates cell proliferation and apoptosis by inactivating Yorkie, the Drosophila homolog of YAP. Cell. 122:421–434.

Justice RW, Zilian O, Woods DF, Noll M, Bryant PJ. 1995. The Drosophila tumor suppressor gene warts encodes a homolog of human myotonic dystrophy kinase and is required for the control of cell shape and proliferation. Gene Dev. 9:534–546.

Kango-Singh M, Singh A. 2009. Regulation of organ size: insights from the Drosophila hippo signaling pathway. Dev Dyn. 238:1627–1637.

King N, Hittinger CT, Carroll SB. 2003. Evolution of key cell signaling and adhesion protein families predates animal origins. Science 301:361–363.

King N, Westbrook MJ, Young SL, et al. (36 co-authors). 2008. The genome of the choanoflagellate Monosiga brevicollis and the origin of metazoans. Nature 451:783–788.

Kumar S, Nei M, Dudley J, Tamura K. 2008. MEGA: a biologist-centric software for evolutionary analysis of DNA and protein sequences. Brief Bioinform. 9:299–306.

Hilman and Gat · doi:10.1093/molbev/msr065

Lai ZC, Wei X, Shimizu T, Ramos E, Rohrbaugh M, Nikolaidis N, Ho LL, Li Y. 2005. Control of cell proliferation and apoptosis by mob as tumor suppressor, mats. *Cell*. 120:675–685.

Lehtinen MK, Yuan Z, Boag PR, et al. (11 co-authors). 2006. A conserved MST-FOXO signaling pathway mediates oxidative-stress responses and extends life span. *Cell* 125:987–1001.

Levy D, Adamovich Y, Reuven N, Shaul Y. 2007. The Yes-associated protein 1 stabilizes p73 by preventing Itch-mediated ubiquitination of p73. *Cell Death Differ.* 14:743–751.

Levy D, Adamovich Y, Reuven N, Shaul Y. 2008. Yap1 phosphorylation by c-Abl is a critical step in selective activation of proapoptotic genes in response to DNA damage. *Mol Cell*. 29:350–361.

Li L, Stoeckert CJ, Roos DS. 2003. OrthoMCL: identification of ortholog groups for eukaryotic genomes. *Genome Res.* 13:2178–2189.

Li Z, Zhao B, Wang P, Chen F, Dong Z, Yang H, Guan KL, Xu Y. 2010. Structural insights into the YAP and TEAD complex. *Genes Dev.* 24:235–240.

Maglott D, Ostell J, Pruitt KD, Tatusova T. 2007. Entrez gene: gene-centered information at NCBI. *Nucleic Acids Res.* 35:D26–D31.

Matus DQ, Magie CR, Pang K, Martindale MQ, Thomsen GH. 2008. The Hedgehog gene family of the cnidarian, Nematostella vectensis, and implications for understanding metazoan hedgehog pathway evolution. *Dev Biol.* 313:501–518.

McCartney BM, Kulikauskas RM, LaJeunesse DR, Fehon RG. 2000. The neurofibromatosis-2 homologue, Merlin, and the tumor suppressor expanded function together in Drosophila to regulate cell proliferation and differentiation. *Development*. 127:1315–1324.

Morisaki T, Hirota T, Iida S, et al. (13 co-authors). 2002. WARTS tumor suppressor is phosphorylated by Cdc2/cyclin B at spindle poles during mitosis. *FEBS Lett.* 529:319–324.

Murakami M, Nakagawa M, Olson EN, Nakagawa O. 2005. A WW domain protein TAZ is a critical coactivator for TBX5, a transcription factor implicated in Holt-Oram syndrome. *Proc Natl Acad Sci USA*. 102:18034–18039.

Nichols S, Wörheide G. 2005. Sponges: new views of old animals. *Integr Comp Biol.* 45:333–334.

Nichols SA, Dirks W, Pearse JS, King N. 2006. Early evolution of animal cell signaling and adhesion genes. *Proc Natl Acad Sci USA*. 103:12451–12456.

Nishiyama Y, Hirota T, Morisaki T, et al. (11 co-authors). 1999. A human homolog of Drosophila warts tumor suppressor, h-warts, localized to mitotic apparatus and specifically phosphorylated during mitosis. *FEBS Lett.* 459:159–165.

Oh H, Irvine KD. 2010. Yorkie: the final destination of Hippo signaling. *Trends Cell Biol.* 20:410–417.

Oka T, Sudol M. 2009. Nuclear localization and pro-apoptotic signaling of YAP2 require intact PDZ-binding motif. *Genes Cells*. 14:607–615.

Ota M, Sasaki H. 2008. Mammalian tead proteins regulate cell proliferation and contact inhibition as transcriptional mediators of Hippo signaling. *Development* 135:4059–4069.

Otsuka J, Sugaya N. 2003. Advanced formulation of base pair changes in the stem regions of ribosomal RNAs; its application to mitochondrial rRNAs for resolving the phylogeny of animals. *J Theor Biol.* 222:447–460.

Pan D. 2010. The Hippo signaling pathway in development and cancer. *Dev Cell.* 19:491–505.

Pantalacci S, Tapon N, Leopold P. 2003. The Salvador partner Hippo promotes apoptosis and cell-cycle exit in Drosophila. *Nat Cell Biol.* 5:921–927.

Pazos F, Ranea JAG, Juan D, Sternberg MJE. 2005. Assessing protein co-evolution in the context of the tree of life assists in the prediction of the interactome. *J Mol Biol.* 352:1002–1015.

Pisani D, Poling LL, Lyons-Weiler M, Hedges SB. 2004. The colonization of land by animals: molecular phylogeny and divergence times among arthropods. *BMC Biol.* 2:1.

Praskova M, Xia F, Avruch J. 2008. MOBKL1A/MOBKL1B phosphorylation by MST1 and MST2 inhibits cell proliferation. *Curr Biol.* 18:311–321.

Price MN, Dehal PS, Arkin AP. 2010. FastTree 2—approximately maximum-likelihood trees for large alignments. *PLoS One.* 5:e9490.

Pruitt KD, Tatusova T, Maglott DR. 2007. NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. *Nucleic Acids Res.* 35:D61–D65.

Radu M, Chernoff J. 2009. The DeMSTification of mammalian Ste20 kinases. *Curr Biol.* 19:R421–R425.

Ryan JF, Mazza ME, Pang K, Matus DQ, Baxevanis AD, Martindale MQ, Finnerty JR. 2007. Pre-bilaterian origins of the hox cluster and the hox code: evidence from the sea anemone, Nematostella vectens. *PLoS One.* 2:e153.

Ryan JF, Pang K. Mullikin JC, Martindale MQ, Baxevanis AD. NISC Comparative Sequencing Program. 2010. The homeodomain complement of the ctenophore Mnemiopsis leidy suggests that Ctenophora and Porifera diverged prior to the ParaHoxozoa. *EvoDevo.* 1:9.

Saina M, Genikhovich G, Renfer E, Technau U. 2009. BMPs and chordin regulate patterning of the directive axis in a sea anemone. *Proc Natl Acad Sci USA.* 106:18592–18597.

Sato T, Yamanishi Y, Horimoto K, Kanehisa M, Toh H. 2006. Partial correlation coefficient between distance matrices as a new indicator of protein-protein interactions. *Bioinformatics* 22:2488–2492.

Schierenberg E. 2001. Three sons of fortune: early embryogenesis, evolution and ecology of nematodes. *Bioessays* 23:841–847.

Sperling EA, Peterson KJ, Pisani D. 2009. Phylogenetic-signal dissection of nuclear housekeeping genes supports the paraphyly of sponges and the monophyly of Eumetazoa. *Mol Biol Evol.* 26:2261–2274.

Srivastava M, Begovic E, Chapman J, et al. 2008. The Trichoplax genome and the nature of placozoans. *Nature* 454:955–960.

Srivastava M, Simakov O, Chapman J, et al. (33 co-authors). 2010. The Amphimedon queenslandica genome and the evolution of animal complexity. *Nature* 466:720–726.

Strano S, Monti O, Pediconi N, et al. (13 co-authors). 2005. The transcriptional coactivator Yes-associated protein drives p73 gene-target specificity in response to DNA Damage. *Mol Cell.* 18:447–459.

Strano S, Munarriz E, Rossi M, Castagnoli L, Shaul Y, Sacchi A, Oren M, Sudol M, Cesareni G, Blandino G. 2001. Physical interaction with Yes-associated protein enhances p73 transcriptional activity. *J Biol Chem.* 276:15164–15173.

Sudol M, Harvey KF. 2010. Modularity in the Hippo signaling pathway. *Trends Biochem Sci.* 35:627–633.

Sullivan JC, Sher D, Eisenstein M, Shigesada K, Reitzel AM, Marlow H, Levinon D, Groner Y, Finnerty JR, Gat U. 2008. The evolutionary origin of the Runx/CBFbeta transcription factors—studies of the most basal metazoans. *BMC Evol Biol.* 8:228–2248.

Tapon N, Harvey KF, Bell DW, Wahrer DCR, Schiripo TA, Haber DA, Hariharan IK. 2002. Salvador promotes both cell cycle exit and apoptosis in Drosophila and is mutated in human cancer cell lines. *Cell* 110:467–478.

Vitolo MI, Anglin IE, Mahoney WM, Renoud KJ, Gartenhaus RB, Bachman KE, Passaniti A. 2007. The RUNX2 transcription factor cooperates with the YES-associated protein, YAP65, to promote cell transformation. *Cancer Biol Ther.* 6:856–863.

Wang Y, Abu Irqeba A, Ayalew M, Suntay K. 2009. Sumoylation of transcription factor Tec1 regulates signaling of mitogen-activated protein kinase pathways in yeast. *PLoS One.* 4:e7456.

Wei X, Shimizu T, Lai ZC. 2007. Mob as tumor suppressor is activated by hippo kinase for growth inhibition in Drosophila. EMBO J. 26:1772–1781.

Wu J, Duggan A, Chalfie M. 2001. Inhibition of touch cell fate by egl-44 and egl-46 in C. elegans. Genes Dev. 15:789–802.

Wu S, Huang J, Dong J, Pan D. 2003. Hippo encodes a Ste-20 family protein kinase that restricts cell proliferation and promotes apoptosis in conjunction with salvador and warts. Cell 114:445–456.

Xu T, Wang W, Zhang S, Stewart R, Yu W. 1995. Identifying tumor suppressors in genetic mosaics: the Drosophila lats gene encodes a putative protein kinase. Development 121:1053–1063.

Yabuta N, Okada N, Ito A, et al. (12 co-authors). 2007. Lats2 is an essential mitotic regulator required for the coordination of cell division. J Biol Chem. 282:19259–19271.

Yu J, Zheng Y, Dong J, Klusza S, Deng WM, Pan D. 2010. Kibra functions as a tumor suppressor protein that regulates hippo signaling in conjunction with merlin and expanded. Dev Cell. 18:288–299.

Zhao B, Li L, Lei Q, Guan KL. 2010. The Hippo-YAP pathway in organ size control and tumorigenesis: an updated version. Genes Dev. 24:862–874.

Zhao B, Wei X, Li W, et al. (16 co-authors). 2007. Inactivation of YAP oncoprotein by the hippo pathway is involved in cell contact inhibition and tissue growth control. Genes Dev. 21:2747–2761.
