
Molecular Cell
Article

# A Strategy for Antagonizing Quorum Sensing

Guozhou Chen,$^{1,4}$ Lee R. Swem,$^{1,2,3,4}$ Danielle L. Swem,$^{1,2,3}$ Devin L. Stauff,$^{1,2}$ Colleen T. O’Loughlin,$^{1}$ Philip D. Jeffrey,$^{1}$ Bonnie L. Bassler,$^{1,2}$ and Frederick M. Hughson$^{1,*}$

$^{1}$Department of Molecular Biology, Princeton University, Princeton, NJ 08544, USA  
$^{2}$Howard Hughes Medical Institute  
$^{3}$Present address: Genentech, 1 DNA Way, South San Francisco, CA 94080, USA  
$^{4}$These authors contributed equally to this work  

*Correspondence: hughson@princeton.edu  
DOI 10.1016/j.molcel.2011.04.003

## SUMMARY

Quorum-sensing bacteria communicate via small molecules called autoinducers to coordinate collective behaviors. Because quorum sensing controls virulence factor expression in many clinically relevant pathogens, membrane-permeable quorum sensing antagonists that prevent population-wide expression of virulence genes offer a potential route to novel antibacterial therapeutics. Here, we report a strategy for inhibiting quorum-sensing receptors of the widespread LuxR family. Structure-function studies with natural and synthetic ligands demonstrate that the dimeric LuxR-type transcription factor CviR from Chromobacterium violaceum is potently antagonized by molecules that bind in place of the native acylated homoserine lactone autoinducer, provided that they stabilize a closed conformation. In such conformations, each of the two DNA-binding domains interacts with the ligand-binding domain of the opposing monomer. Consequently, the DNA-binding helices are held apart by ~60 Å, twice the ~30 Å separation required for operator binding. This approach may represent a general strategy for the inhibition of multidomain proteins.

## INTRODUCTION

Quorum sensing is a process of bacterial communication that relies on the production, detection, and response to extracellular signaling molecules called autoinducers (Fuqua and Greenberg, 2002; Fuqua and Winans, 1994; Waters and Bassler, 2005). Quorum sensing allows bacteria to synchronously alter their behavior in response to changes in the population density and species composition of the surrounding bacterial community. Quorum sensing controls collective behaviors including bioluminescence, virulence factor production, and biofilm formation (Fuqua et al., 2001; Miller and Bassler, 2001). Gram-negative bacteria commonly use acyl-homoserine lactone molecules (AHLs) as autoinducers (Bassler, 1999; de Kievit and Iglewski, 2000; Fuqua et al., 1996). AHLs consist of homoserine lactone rings attached via amide bonds to acyl chains containing four to 18 carbons (see Figure 1A).

Bacteria use two different mechanisms to detect AHLs that accumulate at high cell density. One mechanism, found in Vibrios, employs transmembrane histidine sensor kinases, exemplified by the LuxN receptor of Vibrio harveyi (Henke and Bassler, 2004). In these systems, AHL molecules in the periplasm bind to the membrane-bound receptor, eliciting a change in a phosphorelay cascade that impinges on downstream gene expression (Freeman and Bassler, 1999a, 1999b; Swem et al., 2008). The second mechanism, found in many other bacteria, employs cytoplasmic LuxR-type transcriptional regulators. At low cell density, in the absence of autoinducer, most LuxR-type proteins fail to fold stably and are therefore degraded by proteases (Pinto and Winans, 2009; Swem et al., 2009; Zhang et al., 2002; Zhu and Winans, 1999, 2001). By contrast, LuxR:AHL complexes are stable and bind DNA to activate transcription of quorum-sensing target genes—including virulence genes—in a wide range of pathogens (Hussain et al., 2008; Passador et al., 1993; Piper et al., 1993). Consequently, molecules that function as antagonists of LuxR-type proteins are potential leads in the development of novel antibacterial therapeutics. Indeed, small molecule quorum sensing antagonists have been identified (Hentzer and Givskov, 2003; Ni et al., 2009; Njoroge and Sperandio, 2009; Zou and Nair, 2009); however, their mechanisms of action have not been defined, in part because structural information has been unavailable.

Here, we dissect a mechanism used by quorum-sensing antagonists to modulate quorum sensing in the bacterium Chromobacterium violaceum. C. violaceum is a human pathogen that uses AHL quorum sensing to control biofilm formation, cyanide production, and synthesis of the purple pigment violacein (McClean et al., 1997). C. violaceum strain ATCC 31532 produces the autoinducer N-hexanoyl homoserine lactone (C6-HSL; Figure 1A), which is detected via the LuxR-type protein CviR (Figure 1B). The longer-chain analog C10-HSL antagonizes C. violaceum quorum sensing (McClean et al., 1997). We reported previously the discovery of a chlorolactone compound (CL) that is an even more potent antagonist than C10-HSL and is able to protect Caenorhabditis elegans from quorum-sensing-mediated killing by C. violaceum (Swem et al., 2009). In this report, we use structure-function studies to define a mechanism underlying antagonism of LuxR-type receptors. Our results also provide an explanation for the observation (Geske et al., 2005, 2007, 2008; Koch et al., 2005; Manefield et al., 2002; Müh et al., 2006) that relatively minor changes in the structure of a ligand can drastically alter its activity as a LuxR antagonist. Small molecules that

Molecular Cell

CviR Antagonist Mechanism

Figure 1. CviR-Mediated Quorum Sensing

(A) Agonist/antagonists studied in this work.

(B) At high cell density, autoinducer (C6-HSL; circles) accumulates, binding to and stabilizing CviR. CviR:C6-HSL functions as a transcriptional activator of genes, including *vioA*, controlled by quorum sensing.

at 3.3 Å resolution of full-length CviR bound to CL (Figure 2A and Table S1 available online). LuxR-type proteins are homodimers, each monomer of which consists of two domains, a ligand-binding domain (LBD) and a DNA-binding domain (DBD) (Choi and Greenberg, 1991; Hanzelka and Greenberg, 1995). Among LuxR family proteins, only TraR has previously been crystallized intact (Chen et al., 2007; Vannini et al., 2002; Zhang et al., 2002). In understanding how CL acts as an antagonist, it is informative to compare the structure of CviR:CL to that of full-length autoinducer-bound TraR, which was crystallized in complex with operator DNA (Vannini et al., 2002; Zhang et al., 2002). Despite low sequence identity (16%),

function analogously to the antagonists studied here could be broadly useful for inhibiting other LuxR-type receptors and indeed other, unrelated, multidomain proteins.

**RESULTS**

**Structure of the Antagonist Complex CviR:CL**

To begin characterizing the molecular mechanism of LuxR-type receptor antagonist function, we determined the crystal structure

Figure 2. CviR Bound to the Antagonist CL Adopts a Closed Conformation

(A) The crossed-domain conformation of CviR:CL, with the two monomers colored orange and blue (Table S1). Importantly, although a portion of one of the interdomain linkers (residues 190–196; dashed line) is disordered in the crystal structure, the conformation of the other linker is supported by clear electron density. The CL molecules (Figure 1A) are shown with a space-filling representation.

(B) Comparison of the X-ray structures of TraR bound to 3-oxo-C8-HSL and DNA (Zhang et al., 2002) and CviR bound to CL.

(C) Surface representation of the LBD of CviR:CL. In the left panel, the region that interacts with the DBD of the other monomer is highlighted in blue. In the right panel, a subset of the LBD residues that contact the DBD are highlighted in red.

(D) CviR:CL with the DNA recognition helices highlighted in red.

See also Figure S2.

Molecular Cell

CviR Antagonist Mechanism

**Figure 3. Response of CviR to AHLs**

(A) Agonist activity was evaluated by growing an *E. coli* CviR reporter strain 7 hr in the indicated AHL concentrations. Induction of *vioA-gfp* was measured by fluorimetry. Most error bars (standard deviation [SD], n = 3) are not visible because they are smaller than the symbols used.

(B) Antagonist activity was evaluated by growing the reporter strain 7 hr in 1 μM C6-HSL autoinducer plus the indicated concentrations of C8-HSL, C10-HSL, or CL. Error bars represent the SD (n = 3).

(C) Transcriptional activation by CviR was evaluated with in vitro transcription reactions containing RNA polymerase and α<sup>32</sup>P-UTP. Radiolabeled transcripts corresponding to *vioA* or to the constitutively transcribed *RNAI* (control) were analyzed by urea-PAGE and phosphorimaging.

(D) Data in (C) were quantified with ImageQuant and are displayed as mean ± SD; n = 4.

*vioA-gfp* expression is activated by the native autoinducer, C6-HSL (*Figure 3*A). This activation is completely inhibited by CL (*Figure 3*B), which itself displays no agonist activity (*Figure 3*A). C8- and C10-HSL each elicit intermediate responses: 40% and 6%, respectively, of the *vioA-gfp* activation elicited by C6-HSL (*Figure 3*A). In vitro transcription experiments verified these findings: purified CviR:ligand complexes activated transcription in the order C6-HSL > C8-HSL > C10-HSL > CL (*Figures 3C* and *3D*).

Not only do C8-HSL and C10-HSL fail to fully activate CviR-dependent transcription, but both of them function as partial antagonists in the presence of the native autoinducer, C6-HSL. Like CL, C8-HSL and C10-HSL bind CviR in the pocket normally occupied by C6-HSL (see below). As a result, high concentrations of either C8-HSL or C10HSL reduce the level of C6-HSL-mediated *vioA-gfp* activation, not to zero, but rather to the level observed for C8-HSL or C10-HSL alone (*Figures 3*A and *3B*). Thus, within this series of ligands, the size of the tail establishes both the activity level of the CviR:ligand complex and the effectiveness of the ligand as an antagonist. Specifically, lengthening the acyl tail simultaneously reduces agonism and enhances antagonism. A simple mechanism to explain these results is that CviR bound to C6-HSL is predominantly open, competent to bind DNA and activate transcription, whereas C8-HSL, C10-HSL, and CL promote the closed, inactive conformation to progressively greater extents.

Limited proteolysis provides support for this model. We used chymotrypsin cleavage to examine the conformational preferences of CviR in complex with the series of ligands discussed above. Chymotrypsin was selected because it cleaves primarily at a single site, a few residues into the DNA-binding domain at Tyr 203 (*Table 1* and *Figure S1A*). The autoinducer complex CviR:C6-HSL was cleaved with a t<sub>1/2</sub> value of 4.2 min (*Table 1* and *Figure S1B*). By contrast, the antagonist complex CviR:CL

**Functional Analysis of a CviR Ligand Series**

The CviR:CL structure described above suggests that the stability of the closed conformation could be linked to the potency of the antagonist. We examined this notion by varying the bound ligand and systematically evaluating agonist/antagonist activity. We explored autoinducer analogs with acyl tails of different sizes ranging from C6 to CL (*Figure 1*A). To assess activity, we used a heterologous system in which CviR was expressed in an *Escherichia coli* strain harboring a CviR-regulated promoter, *vioA*, fused to *gfp*. As expected, CviR-dependent

Molecular Cell

CviR Antagonist Mechanism

Table 1. Proteolytic Analysis of CviR and CviR′ with Chymotrypsin

| Protein | CviR          | CviR M89S     | CviR′         | CviR′ S89M/N77Y |
|---------|---------------|---------------|---------------|-----------------|
| Ligand  | C6-HSL        | C8-HSL        | C10-HSL       | CL              | C10-HSL       | C6-HSL        | C10-HSL       | C10-HSL       |
| $t_{1/2}$ (min) | 4.2 ± 0.2 | 5.5 ± 0.3 | 8.9 ± 0.5 | 95 ± 2 | 3.7 ± 0.1 | 5.8 ± 0.1 | 12 ± 1 | 4.1 ± 0.2 | 22 ± 0.4 |

See also Figure S1.

was significantly more resistant to digestion, displaying a half-life of 95 min. CviR:C8-HSL and CviR:C10-HSL displayed intermediate levels of protease resistance (Table 1 and Figures S1B–S1E). These results imply that the crossed-domain arrangement observed in the crystal structure of the antagonist complex is most likely present in solution, acting to stabilize the DNA-binding domains against proteolytic digestion. Moreover, our results are consistent with the model that CviR antagonists function by stabilizing the crossed-domain conformation.

To obtain additional evidence for the crossed-domain configuration of CviR:CL in solution, we used the CviR:CL structure to guide the introduction of two Cys residues—one in the LBD and one in the DBD—in locations predicted to allow the formation of a disulfide crosslink (Figure S2A). At the same time, the single endogenous Cys residue was changed to Ser. Consistent with our model, the triply mutant protein (CviR$_{Cys}$) in complex with CL rapidly formed disulfide-linked dimers under oxidizing conditions (Figure S2B). CviR$_{Cys}$:C6-HSL also formed disulfide-linked dimers, but more slowly, consistent with the idea that native autoinducer-bound CviR explores the crossed-domain conformation (Figure S2C). Finally, as expected, disulfide crosslinking reversibly eliminated DNA binding activity (Figures S2D–S2E).

In addition to promoting the crossed-domain conformation, C8-HSL, C10-HSL, and CL might also influence steps subsequent to operator DNA binding, such as the recruitment of RNA polymerase (RNAP). One indication that this is the case is the larger impact of C8-HSL and C10-HSL on transcriptional activation (2.5-fold and 17-fold, respectively; Figure 3A) than on resistance to limited proteolysis (1.3-fold and 2-fold, respectively; Table 1). Bacterial two-hybrid experiments showed that CviR:C6-HSL interacts directly with the C-terminal domain of the RNA polymerase $\alpha$-subunit ($\alpha$CTD; Figure 4B). Weak interaction was observed in the presence of C8-HSL, while no interaction was observed in the presence of C10-HSL or CL. While we cannot exclude the possibility that these differences reflect differences in the amount of fusion protein present in cells plated on medium containing different ligands, our results suggest that CviR might require an open conformation in order to interact with RNAP, possibly because this interaction involves surfaces of CviR that are concealed within the LBD-DBD interface in the closed conformation.

### Structural Analysis of C6-, C8-, and C10-HSL Complexes with CviR

We were unable to crystallize full-length CviR in complex with C6-, C8-, or C10-HSL, presumably because these ligands are less effective than CL at stabilizing the closed conformation. Instead, we determined crystal structures for the CviR LBD only (Table S1). All three LBD:ligand structures were refined to high resolution (1.6 Å) and are similar to one another (rmsd < 0.2 Å) and to the LBD domain of CviR:CL (rmsd = 0.8 Å). Furthermore, each LBD:ligand complex crystallized in the same space group, allowing us to eliminate differences in crystal packing as the cause of any differences among the structures. An excellent correlation emerges between these structures and the extent of antagonist activity. Relative to the C6-HSL agonist complex, C8-HSL elicits the subtlest change: a single side chain, Met 89, is affected, and it populates two conformations with nearly equal occupancy; one of these being the original conformation observed with C6-HSL bound (Figure 4C and Figure S3). In the C10-HSL complex, the Met 89 side chain swings away from the ligand-binding pocket to fully occupy the alternative conformation observed in the C8-HSL complex, thereby accommodating the extended acyl chain of C10-HSL. Because no other structural changes above noise are observed in the LBDs, the progressive repositioning of Met 89 likely underlies the increasing antagonist activity displayed by C8- and C10-HSL, respectively. Finally, comparison of the C6-HSL and CL complexes reveals that CL causes the side chain of Met 89 to switch to the “antagonist” position. In addition, to accommodate CL’s bulky substituted phenyl ring, helix $\alpha$3 and strand $\beta$2, which define the mouth of the ligand-binding pocket, shift approximately 1–2 Å (Figure 4D; see also Figure 2C). Many residues from these secondary structural elements, including Met 89 as well as Leu 72, Arg 74, and Asn 77, make direct contact with the DBD in the crossed-domain conformation. In summary, the ligands described here elicit structural changes of increasing severity but in all cases localized to the surface of the LBD that—in the crossed-domain conformation—is buried by the DBD.

#### Probing the LBD-DBD Interface by Mutagenesis

An implication of the antagonist mechanism we have uncovered is that it should be possible, by engineering mutations into the LBD-DBD interface, to alter the balance between the closed and open conformations of CviR and thereby modulate the antagonist activity of a given ligand. In principle, one could convert an antagonist into an agonist or convert an agonist into an antagonist. To begin exploring this idea, we tested whether mutating the key residue Met 89 affects the antagonist activity of C10-HSL. Replacing Met 89 with amino acids of similar or larger size (i.e., Leu or Phe), did not alleviate C10-HSL antagonism and converted C10-HSL into a potent agonist (Figure 5C). Indeed CviR M89S was almost fully responsive to C10-HSL, as judged by a number of criteria including interaction with DNA, interaction with the RNAP $\alpha$ subunit C-terminal domain, and *vioA-gfp* transcription in vivo and in vitro (Figures

Molecular Cell

CviR Antagonist Mechanism

![](image.png)

Figure 4. Structure-Function Analysis of a CviR Ligand Series

(A) Purified CviR or CviR M89S bound to the indicated AHL was included in DNA binding reactions with a fluorescein-labeled DNA containing the CviR′ binding site. Data are plotted as the change in fluorescence polarization resulting from CviR binding (ΔF) divided by fluorescence polarization in the absence of CviR (F₀). We note that using gel-shift assays, we previously (Swem et al., 2009) failed to detect the decreased DNA binding by the CviR:C10-HSL complex reported here. We attribute this discrepancy to our earlier use of much higher DNA concentrations, which had the effect of obscuring the relatively modest difference in affinity. Error bars represent the SD (n = 3).

(B) CviR-RNA polymerase interaction assessed by bacterial adenylate cyclase two-hybrid (BACTH) analysis. *E. coli* BTH101 was transformed with plasmids expressing the *Bordetella pertussis* adenylate cyclase T25 domain fused to CviR and the T18 domain fused to either CviR, the N-terminal domain of *Chromobacterium violaceum* RNA polymerase alpha subunit (αNTD), the C-terminal domain of *C. violaceum* RNA polymerase alpha subunit (αCTD), or region 4 of *C. violaceum* RNA polymerase sigma-70 (σ4). T25-zip and T18-zip are fusions to the constitutively interacting GCN4 leucine zipper. Strains were grown on LB agar with the addition of the indicated AHL at 10 μM final concentration. CviR is unstable and rapidly degraded in the absence of AHL, accounting for the absence of BACTH signal in plates lacking AHL.

(C) Close-up view of the ligand-binding pockets of CviR:C6-HSL and CviR:C10-HSL. The ligand and Met 89 residues in the CviR:C6-HSL and CviR:C10-HSL complexes are colored cyan and gray, respectively.

(D) Stereo view of the ligand-binding pockets of CviR bound to C6-HSL (cyan) and CL (gray), respectively.

See also Figure S3.

4A and 4B and 5C–5E). In excellent agreement with these functional results, limited proteolysis confirms that CviR M89S is in an open conformation when bound to either C10-HSL or CL (Table 1 and Figures S1F and S1G). All four CviR mutants (Ala, Ser, Leu, Phe) remained capable of activating *vioA-gfp* in response to the native autoinducer, C6-HSL (Figures 5B and 5C). Interestingly, however, the ligand concentration required for a half-maximal response (*EC₅₀*) was about 5-fold higher for the Ala and Ser replacement mutants than for wild-type CviR (Figure 3A versus Figure 3C). This result suggests that the contact between Met 89 of wild-type CviR and the native autoinducer is kinetically and/or thermodynamically advantageous.

To determine whether mutations in the LBD-DBD interface can be used to convert an agonist into an antagonist, we exploited

a *C. violaceum* strain (ATCC 12472) that produces 3-hydroxy-C10-HSL (Morohoshi et al., 2008) but responds to a range of molecules, including C10-HSL and CL (Figure 6A). This autoinducer receptor, here called CviR′, is 87% identical in amino acid sequence to CviR. Its reduced selectivity is likely attributable, at least in part, to a naturally occurring Ser (rather than Met) at position 89; based on the above results, Ser 89 likely favors the open, active conformation even when C10-HSL is bound. We therefore screened for mutant CviR′ proteins in which C10-HSL could function as an antagonist, as it does for CviR.

Simply substituting CviR′ Ser 89 with Met was not sufficient to convert C10-HSL into an antagonist; indeed, it had little impact on C10-HSL agonist activity (Figure S4A). Nonetheless, because we anticipated that C10-HSL antagonism would depend on a

Molecular Cell 42, 199–209, April 22, 2011 ©2011 Elsevier Inc. 203

Molecular Cell

CviR Antagonist Mechanism

Figure 5. Transcriptional Activation by CviR In Vivo and In Vitro

(A) E. coli vioA-gfp expressing CviR was grown in the indicated concentrations of C10-HSL or C10-HSL plus 1 μM C6-HSL. GFP fluorescence was measured and normalized to the optical density of the culture at 600 nm (OD<sub>600</sub>). Data are mean ± SD; n = 3.

(B) E. coli vioA-gfp expressing CviR M89F or CviR M89L was grown in the indicated concentrations of C10-HSL or C10-HSL plus 1 μM C6-HSL and analyzed as in (A).

(C) E. coli vioA-gfp expressing CviR M89A or CviR M89S was grown in the presence of the indicated concentrations of C6-HSL or C10-HSL and analyzed as in (A).

(D) CviR in vitro transcription. CviR or CviR M89S bound to C10-HSL were included in in vitro transcription reactions containing RNA polymerase and α<sup>32</sup>P-UTP. Radiolabeled transcripts corresponding to vioA or the constitutively transcribed RNAI were analyzed by urea-PAGE and phosphorimaging.

(E) Data in (D) were quantified with ImageQuant and are displayed as mean ± SD; n = 4.

the double mutation works by stabilizing the LBD-DBD interface, and thereby stabilizing a closed conformation, when C10-HSL or CL is bound (Figure 2C). This supposition is further supported by the diminished DNA-binding activity (Figure S4B) and RNAP interaction (Figure S4C), and enhanced proteolytic resistance (t<sub>1/2</sub> = 22 min, versus 4.1 min for wild-type; Table 1 and Figures S1H and S1I), displayed by CviR' S89M/N77Y in the presence of C10-HSL. Taken together, these results establish that CviR' mutations that map to the LBD-DBD interface (see Figure 2C) can serve to stabilize an inactive, closed conformation and, by so doing, convert an agonist to an antagonist. Interestingly, CviR' S89M/N77Y:C10-HSL displays larger defects in vioA-gfp activation (Figure 6A) and RNAP interaction (Figure S4C) than would be predicted based on its modest DNA-binding defect (Figure S4B). This finding provides additional support for the involvement of the region surrounding the mouth of the ligand-binding pocket in downstream events such as RNAP recruitment.

C6-HSL Is a Partial CviR' Antagonist, Promoting a Distinct Crossed-Domain Conformation

Additional insight into the potential of autoinducer analogs for stabilizing inactive, crossed-domain conformations comes from structural and mutagenic studies of CviR'. As mentioned above, the native autoinducer for CviR' has been reported to be 3-hydroxy-C10-HSL. C6-HSL functions as a partial antagonist (Figure S4D). Strikingly, the crystal structure of full-length CviR' bound to C6-HSL determined at 2.0 Å resolution revealed a crossed-domain conformation distinct from that observed for

steric clash between the extended acyl chain of C10-HSL and a bulky residue at position 89, we used this CviR' S89M mutant as the parent for our mutagenesis experiments. Approximately 150,000 random mutants were introduced into the E. coli vioA-gfp reporter strain. Transformants were grown in the presence of C6-HSL and FACS was used to isolate cells producing maximal vioA-gfp expression. This sorted population was subsequently grown in the presence of C10-HSL, and FACS was again used, this time to isolate transformants incapable of activating vioA-gfp expression. The growth and sorting procedures were repeated to enrich the isolation of candidate mutants. Nine CviR' mutants were isolated that retained the ability to be activated by C6-HSL but were antagonized by C10-HSL. All of the mutants had acquired the identical single amino acid change, N77Y, while six mutants carried additional, functionally silent mutations.

In the CviR' S89M/N77Y double mutant, but not in either single mutant, in vivo activation of vioA-gfp expression by C10-HSL and CL is strongly attenuated (Figure 6A and Figure S4A). Consequently both C10-HSL and CL function as antagonists for the CviR' S89M/N77Y receptor (Figure 6B). Ser 89 and Asn 77 flank the mouth of the ligand-binding pocket, reinforcing the idea that

Molecular Cell

CviR Antagonist Mechanism

**Figure 6. The Partial Antagonist C6-HSL Induces a Closed Conformation in CviR'**

(A) *E. coli* vioA-gfp expressing CviR' or CviR' S89M/N77Y was grown in the presence of the indicated concentration of C6-HSL, C10-HSL, or CL. GFP fluorescence was measured and normalized to OD<sub>600</sub>. Data are mean ± SD; n = 3.

(B) *E. coli* vioA-gfp expressing CviR' S89M/N77Y was grown in the indicated concentration of C10-HSL or CL (agonist), or in the presence of 1 μM C6-HSL plus the indicated concentration of C10-HSL or CL (antagonist). Samples were analyzed as in (A).

(C) The crossed-domain conformation of CviR':C6-HSL (Table S1), with the two monomers colored green and yellow. The C6-HSL molecules are shown in magenta using a space-filling representation.

(D) Comparison of the cross-domain conformations of CviR:CL (orange and blue) and CviR':C6-HSL (green and yellow). The N-terminal LBDs were aligned in order to emphasize the different orientations of C-terminal DBDs.

(E) The interactions between Met 253 and Met 257 side chains (cyan) and the cavity created when C6-HSL (magenta) binds in place of the native autoinducer is shown. The protein backbone is colored green and a surface representation for the LBD is gray.

See also Figure S4.

---

**DISCUSSION**

Our X-ray structural studies demonstrate that CviR and CviR' can each populate closed conformations that lock the DNA recognition helices into an arrangement incompatible with operator DNA binding (Figure 2D). The most stable of these closed conformations, by all of our criteria, is the CviR:CL complex, in which readjustments in the LBD (Figure 4D) allow it to interact favorably with the DBD in a crossed-domain interaction. A distinct crossed-domain conformation is adopted by CviR':C6-HSL, in which a cavity generated by the absence of four carbons present in the native 10-carbon autoinducer makes room for the side chain of a Met residue on the DBD (Figure 6E). The closed conformation adopted by CviR':C6-HSL provides independent validation for the strategy of interdomain stabilization. Strong additional support comes from our finding that, for CviR', mutations that convert C10-HSL from an agonist to an antagonist map to the LBD-DBD interface near the ligand-binding site (residues 77 and 89; see Figure 2C). Our results are consistent with previous reports for other LuxR-type proteins, which demonstrate that relatively minor changes in ligand structure can have drastic effects on agonist/antagonist activity (Geske et al., 2005, 2007, 2008; Müh et al., 2006). Indeed, we find that large changes in activity can result from subtle conformational rearrangements of the receptors caused, in turn, either by alterations in the ligand or by amino acid changes in the protein.

Relative to the CviR:antagonist complexes we studied, we find that CviR:agonist complexes are more susceptible to in vitro proteolytic cleavage (Table 1), possibly because—unlike the antagonist-bound complexes—they are dynamic and occupy an ensemble of conformations. Indeed, flexibility in the LBD-DBD connection may be important for DNA engagement, based on the highly asymmetric structure of the CviR homolog TraR bound to operator DNA (Vannini et al., 2002; Zhang et al., 2002). Neither the intrinsic flexibility of CviR:agonist complexes nor the extent to which they populate crossed-domain conformations has been examined directly. It nevertheless seems likely based on Cys crosslinking experiments (Figure S2) that the antagonist complex structures capture crossed-domain conformations only fleetingly present in agonist complexes, simultaneously rendering them inactive for DNA binding and suitable for crystallization.

Many proteins contain domains connected by flexible linkers. Their physical connection means that these domains are maintained at a high effective concentration with respect to one another. Here, we show that it is possible to develop antagonists that function by stabilizing transient interdomain interactions, giving rise to an inactive configuration. For quorum-sensing receptors of the LuxR family, there are two general strategies by which one might accomplish this goal. The first strategy, demonstrated here, is to identify ligands that bind in place of the autoinducer and that—to a greater extent than the autoinducer—favor an inactivating LBD-DBD interaction. A second, related approach toward antagonist development would be to identify molecules that bind to preformed LuxR-autoinducer complexes in such a way as to stabilize closed conformations. No examples of this latter type of small-molecule antagonist have to our knowledge been reported. The natural quorum-sensing inhibitor protein TraM, however, employs a related mechanism, inserting itself into TraR:autoinducer complexes between the ligand-binding and DNA-binding domains such that the latter are mispositioned for DNA binding (Chen et al., 2007). This inactive TraR configuration bears no resemblance to the crossed-domain configuration characterized here for CviR, indicating that diverse DNA-binding deficient conformations might in principle be stabilized by different natural or synthetic antagonists. In practice it seems likely that most therapeutics will stabilize transient configurations normally adopted, albeit with much lower probability, by agonist-bound LuxR proteins. Strategies aimed at stabilizing transient interdomain interactions thus appear particularly promising for antagonist design.

The canonical LuxR-type protein binds AHL, which induces protein dimerization, DNA binding, and RNA polymerase activation. A second class of LuxR-type receptor is exemplified by EsaR, which folds, dimerizes, and binds DNA in the absence of its AHL ligand (Minogue et al., 2002; von Bodman et al., 2003). DNA binding by apo-EsaR results in transcriptional repression of quorum sensing regulated genes. This repression is alleviated by the binding of the native autoinducer to EsaR, which inhibits EsaR’s DNA binding activity (Minogue et al., 2005). Based on our results, we speculate that autoinducer binding to EsaR may stabilize a crossed-domain conformation unable to bind DNA. This model could thereby explain the mechanism underlying the regulation of this second class of LuxR-type proteins.


Molecular Cell

CviR Antagonist Mechanism

EXPERIMENTAL PROCEDURES

Bacterial Strains and Media
C. violaceum strains ATCC 12472 and ATCC 31532 were obtained from the ATCC. The plasmid pET23b (Novagen) was used for expressing CviR in One Shot TOP10 (Invitrogen) or BL21(DE3) pLysS (Stratagene) E. coli strains. Plasmids were maintained with ampicillin at 100 µg ml⁻¹. Plasmid pEVS141 was used for the vioA-gfp fusion (Swem et al., 2009) and maintained with 50 µg ml⁻¹ kanamycin. E. coli strains were grown at 37°C in Luria Bertani broth (LB) for overexpression or in minimal medium (Zhu et al., 2003) for vioA-gfp assays. cviR′ was cloned from C. violaceum ATCC 12472. cviR was cloned from C. violaceum ATCC 31532. Our sequence for CviR did not agree with that in the NCBI database. Our sequence is under accession number GQ398094.

Protein Preparation
Expression plasmids were constructed by cloning cviR genes from C. violaceum ATCC 12472 and ATCC 31532 into expression vectors pET23b (for full-length CviR) and pET28b (for the ligand-binding domains of CviR′ [residues 8–187] and CviR [residues 10–187]) (Novagen). Proteins were overproduced in BL21 CodonPlus (DE3) (Novagen) grown in LB at 37°C to an optical density at 600 nm (OD₆₀₀) of 0.8–1.0. Protein expression was induced by the addition of 0.2 mM isopropyl β-D-1-thiogalactopyranoside, followed by the addition of 25 µM C6-HSL, C8-HSL, C10-HSL, or CL (Swem et al., 2009). After an additional 3 hr of growth at 25°C, cells were harvested by centrifugation and lysed in 20 mM imidazole (pH 8.0), 100 mM NaCl, 1 mM dithiothreitol (DTT), 0.5 mM EDTA, and 5% (v/v) glycerol via a Micro-fluidizer processor (MicroFluidics). Full-length CviR protein was isolated from cell lysates with heparin resin and further purified by SP cation exchange chromatography and size-exclusion chromatography (Superdex 200 16/60) (GE Healthcare). N-terminally His₆-tagged proteins were purified from cell lysates by Ni²⁺ affinity chromatography. His₆ tags were removed by thrombin digestion followed by an additional round of size exclusion chromatography (Superdex200 16/60). Purified proteins were stored at –80°C in 20 mM imidazole (pH 8.0), 150 mM NaCl, 5% (v/v) glycerol, 1 mM DTT, and 0.5 mM EDTA. The selenomethionine (SeMet) derivative of full-length CviR′ was prepared identically to wild-type protein, except that cells were grown in SeMet-containing M9 minimal medium (Doublié, 1997).

Protein Crystallization and Data Collection
All crystals were obtained using hanging-drop vapor diffusion at 23°C by combining equal volumes of protein (10–15 mg ml⁻¹) and well solutions. CL-bound full-length CviR was crystallized with a well solution containing 50 mM sodium citrate, 15%–20% (w/v) PEG 3350. C6-HSL-, C8-HSL-, and C10-HSL-bound CviR (residues 10–187) were crystallized with a well solution containing 100 mM TrisCl (pH 9.0), 200 mM MgCl₂, 25%–35% (w/v) PEG 3350. C6-HSL-bound native and SeMet-substituted full-length CviR′ were crystallized with a well solution containing 100 mM imidazole (pH 8.0), 10% (w/v) PEG monomethyl ether 5000. Crystals of C10-HSL-bound N-terminally His₆-tagged CviR′ (residues 8–187) were obtained with a well solution containing 100 mM CHES (pH 9.5), 1 M sodium citrate. Crystals were flash frozen in the well solution with a 10% increment of every ingredient plus 15%–25% (v/v) glycerol. All data were collected at NSLS beamlines X25 or X29 (Brookhaven, NY) and processed with the HKL suite (Otwinowski and Minor, 1998).

Structure Determination and Refinement
CL-bound full-length CviR was crystallized in space group H3 with two dimers in each asymmetric unit (Table S1). Molecular replacement was carried out with PHASER (McCoy et al., 2007), and the initial models for the N-terminal ligand-binding domain and C-terminal DNA-binding domain derived from CviR:C6-HSL (residues 10–185) and CviR′:C6-HSL (residues 200–255), respectively (see below). After several cycles of refinement in PHENIX (Adams et al., 2002) and model building in COOT (Emsley and Cowtan, 2004), the electron density for the interdomain linker in one of the monomers was clearly defined. C6-HSL-, C8-HSL-, and C10-HSL-bound CviR (residues 10–187) were crystallized in space group I222 with one molecule in each asymmetric unit. Molecular replacement was carried out with PHASER and the initial model

Molecular Cell

CviR Antagonist Mechanism

was derived from the structure of native CviR′ (residues 9–187). The final model comprises residues 9–187, corresponding to the complete protein sequence of this construct.

C6-HSL-bound SeMet-substituted CviR′ was crystallized in space group C2 with two molecules in the asymmetric unit, and its structure was determined to a resolution of 2.4 Å with MAD phasing methods. Specifically, the SHELX program suite ([Sheldrick, 2008](https://doi.org/10.1107/S0021889808013255)) was used to locate the SeMet sites and to calculate the initial electron density maps. The resulting phases were further improved by the program SHARP ([Bella and Rossmann, 1998](https://doi.org/10.1107/S0907444997014414)). Model building was performed with COOT and refinement was accomplished using REFMAC5 ([Adams et al., 2002](https://doi.org/10.1107/S0907444901019573); [Murshudov et al., 1997](https://doi.org/10.1107/S0907444997009205); [Winn et al., 2001](https://doi.org/10.1107/S0907444901010804)) and PHENIX. The final model contains residues 7–264 (Chain A) and 8–261 (Chain B). Next, this model was used for molecular replacement in order to determine the structure of native (i.e., not SeMet-substituted) C6-HSL-bound full-length CviR′, which crystallized in a different space group (C222₁, with one LBD and one DBD in each asymmetric unit). The final model includes residues 7–191 (chain A) and residues 192–264 (chain B). Lastly, the CviR′:C10-HSL structure was determined by molecular replacement with the CviR′:C6-HSL ligand-binding domain (residues 29–208) as the search model.

### Fluorescence Anisotropy

Fluorescence anisotropy assays were performed on a 35-mer DNA oligonucleotide (fluorescein-acgtcatgccctgacccttggaacagtattagcca) essentially as described ([Pompeani et al., 2008](https://doi.org/10.1074/jbc.M708559200)), except that the binding buffer included 150 mM NaCl and 10 μM excess AHL.

### vioA-gfp Dose-Response Curves

To perform vioA-gfp titrations, cells were grown at 37°C aerobically overnight in minimal medium supplemented with 100 μg ml⁻¹ of ampicillin and 50 μg ml⁻¹ of kanamycin and cultures were sub-cultured into fresh medium at a 1:50 dilution. C6-HSL, C8-HSL, C10-HSL, and/or CL ([Swem et al., 2009](https://doi.org/10.1073/pnas.0900667106)) were added at the specified concentrations. Cells were dispensed in triplicate into 96-well microtiter plates and allowed to grow aerobically at 37°C for 7 hr after which time vioA-gfp and OD₆₀₀ were measured with a Perkin Elmer Envision plate reader.

### In Vitro Transcription

In vitro transcription assays were performed based on a previously described protocol ([Ross and Gourse, 2009](https://doi.org/10.1016/j.jmb.2009.01.006)). Briefly, the vioA promoter was first cloned into plasmid pRLG770 ([Ross et al., 1990](https://doi.org/10.1016/0378-1119(90)90005-6)). Eight microliters in vitro transcription reactions were assembled containing 12.5 mM Tris (pH 7.8), 12.5 mM MgCl₂, 1.25 mM DTT, 125 ng/μl BSA, 150 mM KCl, 3.1 ng/μl plasmid template, 250 μM ATP, 250 μM GTP, 250 μM CTP, 12.5 μM UTP, and 1 μCi α³²P-UTP. CviR (1 μl) diluted in 20 mM imidazole (pH 8.0), 300 mM NaCl, 1 mM EDTA, 1 mM DTT, and 10 μM excess AHL was added and reactions were incubated at room temperature for 15 min. Sigma70-saturated RNA polymerase (Epicenter) diluted to 0.25 U/μl was added (1 μl), reactions were incubated at 23°C for 15 min, after which 10 μl RNA loading dye (95% formamide, 20 mM EDTA, 0.05% bromophenol blue) was added and reactions were shifted to ice. Reactions were resolved on TBE-PAGE gels containing 7 M urea, dried, and analyzed using a phosphorimager.

### Proteolytic Digestion

Purified CviR proteins were diluted to 2 mg/ml in buffer (20 mM imidazole [pH 8.0], 300 mM NaCl, 1 mM dithiothreitol, 0.5 mM EDTA and 5% [v/v] glycerol). Chymotrypsin (Sigma) was prepared in water at a concentration of approximately 0.14 mg/ml. Proteolytic digestions were carried out by mixing equal volumes of protein samples and chymotrypsin and incubating at room temperature. Proteolysis was terminated by the addition of a protease inhibitor cocktail (Sigma). Reactions were resolved on SDS-PAGE followed by Coomassie blue staining or transferring to PVDF membranes for protein sequencing (Iowa State University Protein Facility). Band intensity was quantified with ImageJ (National Institutes of Health [NIH]). The amount of intact CviR protein in samples containing or lacking chymotrypsin was compared in order to calculate percentage cleavage. t₁/₂ values were estimated using curve fitting in conjunction with a one-phase association model.

---

### Cys Crosslinking

Two hundred fifty microliters purified CviR C173S/N38C/N241C (1.5 mg/ml in 20 mM imidazole [pH 7.5], 300 mM NaCl, 0.5 mM EDTA and 5% [v/v] glycerol) bound to CL or C6-HSL was mixed with 10 μl CuPhe solution (3 mM CuSO₄ and 9 mM 1,10-phenanthroline; Sigma) and incubated at 23°C. Eighteen microliter aliquots were removed after various time intervals and the reaction was terminated by the addition 5 μl 0.5 M N-ethylmaleimide (Sigma). Reactions were resolved by SDS-PAGE in the absence of reducing agent, followed by Coomassie blue staining. The intensity of the monomer band, quantified with ImageJ (NIH), was used to calculate the percentage of CviR in which an intermolecular disulfide bond was present. Crosslinked CviR C173S/N38C/N241C:ligand complexes for DNA binding experiments were prepared by incubating 10 ml 0.1 mg/ml protein and 1 ml CuPhe solution for 1.5 hr at 23°C, after which samples were concentrated by ultrafiltration.

### Mutagenesis of cviR

Site-directed mutagenesis (QuikChange, Stratagene) was conducted on pET23:CviR and pET23:CviR′. Plasmids containing the mutated cviR gene were isolated and confirmed by sequencing. Mutant plasmids were electroporated into TOP10 *E. coli* (Invitrogen) carrying the vioA-gfp construct.

### Bacterial Two-Hybrid Assay

The bacterial adenylate cyclase two-hybrid assay (BACTH, Euromedex) was used for all two-hybrid analyses ([Karimova et al., 1998](https://doi.org/10.1074/jbc.273.25.15808)). In brief, full-length wild-type or mutant CviR or CviR′ was cloned between the Xbal and Smal sites of plasmids pUT18 and pKT25. The *C. violaceum* strain 12472 RNA polymerase α subunit N-terminal domain (amino acids 8–239) and C-terminal domain (amino acids 248–327), and σ subunit region 4 (amino acids 567–643) were also cloned between the Xbal and Smal of pKT25. Plasmids were sequenced, cotransformed into *E. coli* strain BTH101, and selected on LB supplemented with ampicillin (200 μg/ml) and kanamycin (100 μg/ml) at 30°C. For all two-hybrid assays, overnight cultures were diluted 1:1000 into fresh LB and plated on LB supplemented with ampicillin, kanamycin, and X-gal (40 μg/ml). Cells were grown at 30°C for 2 days and shifted to 4°C overnight before images were taken.

---

### ACCESSION NUMBERS

Coordinates and structure factors (RCSB accession codes 3QP1, 3QP2, 3QP4, 3QP5, 3QP6, and 3QP8) are in the Protein Data Bank.

---

### SUPPLEMENTAL INFORMATION

Supplemental Information includes four figures and one table and can be found with this article online at [doi:10.1016/j.molcel.2011.04.003](https://doi.org/10.1016/j.molcel.2011.04.003).

---

### ACKNOWLEDGMENTS

We gratefully acknowledge the staff of the National Synchrotron Light Source beamlines X25 and X29 for assistance with X-ray data collection; S. Ulrich (Ithaca College), L. Perez, and M. Semmelhack for AHLs; and members of our laboratories for discussion and advice. This work was supported by the Howard Hughes Medical Institute (HHMI), NIH (AI054442 and GM065859), and National Science Foundation (MCB-0343821). B.L.B. is an investigator of the HHMI.

Received: September 15, 2010  
Revised: January 8, 2011  
Accepted: February 22, 2011  
Published: April 21, 2011  

---

### REFERENCES

Adams, P.D., Grosse-Kunstleve, R.W., Hung, L.W., Ioerger, T.R., McCoy, A.J., Moriarty, N.W., Read, R.J., Sacchettini, J.C., Sauter, N.K., and Terwilliger, T.C.

Molecular Cell 42, 199–209, April 22, 2011 © 2011 Elsevier Inc. 207
(2002). PHENIX: building new software for automated crystallographic structure determination. Acta Crystallogr. D Biol. Crystallogr. 58, 1948–1954.

Bassler, B.L. (1999). How bacteria talk to each other: regulation of gene expression by quorum sensing. Curr. Opin. Microbiol. 2, 582–587.

Bella, J., and Rossmann, M.G. (1998). A general phasing algorithm for multiple MAD and MIR data. Acta Crystallogr. D Biol. Crystallogr. 54, 159–174.

Chen, G., Jeffrey, P.D., Fuqua, C., Shi, Y., and Chen, L. (2007). Structural basis for antiactivation in bacterial quorum sensing. Proc. Natl. Acad. Sci. USA 104, 16474–16479.

Choi, S.H., and Greenberg, E.P. (1991). The C-terminal region of the Vibrio fischeri LuxR protein contains an inducer-independent lux gene activating domain. Proc. Natl. Acad. Sci. USA 88, 11115–11119.

de Kievit, T.R., and Iglewski, B.H. (2000). Bacterial quorum sensing in pathogenic relationships. Infect. Immun. 68, 4839–4849.

Doublé, S. (1997). Preparation of selenomethionyl proteins for phase determination. Methods Enzymol. 276, 523–530.

Emsley, P., and Cowtan, K. (2004). Coot: model-building tools for molecular graphics. Acta Crystallogr. D Biol. Crystallogr. 60, 2126–2132.

Freeman, J.A., and Bassler, B.L. (1999a). A genetic analysis of the function of LuxO, a two-component response regulator involved in quorum sensing in Vibrio harveyi. Mol. Microbiol. 31, 665–677.

Freeman, J.A., and Bassler, B.L. (1999b). Sequence and function of LuxU: a two-component phosphorelay protein that regulates quorum sensing in Vibrio harveyi. J. Bacteriol. 181, 899–906.

Fuqua, C., and Greenberg, E.P. (2002). Listening in on bacteria: acyl-homoserine lactone signalling. Nat. Rev. Mol. Cell Biol. 3, 685–695.

Fuqua, W.C., and Winans, S.C. (1994). A LuxR-LuxI type regulatory system activates Agrobacterium Ti plasmid conjugal transfer in the presence of a plant tumor metabolite. J. Bacteriol. 176, 2796–2806.

Fuqua, C., Winans, S.C., and Greenberg, E.P. (1996). Census and consensus in bacterial ecosystems: the LuxR-LuxI family of quorum-sensing transcriptional regulators. Annu. Rev. Microbiol. 50, 727–751.

Fuqua, C., Parsek, M.R., and Greenberg, E.P. (2001). Regulation of gene expression by cell-to-cell communication: acyl-homoserine lactone quorum sensing. Annu. Rev. Genet. 35, 439–468.

Geske, G.D., Wezeman, R.J., Siegel, A.P., and Blackwell, H.E. (2005). Small molecule inhibitors of bacterial quorum sensing and biofilm formation. J. Am. Chem. Soc. 127, 12762–12763.

Geske, G.D., O’Neill, J.C., and Blackwell, H.E. (2007). N-phenylacetanoyl-L-homoserine lactones can strongly antagonize or superagonize quorum sensing in Vibrio fischeri. ACS Chem. Biol. 2, 315–319.

Geske, G.D., O’Neill, J.C., and Blackwell, H.E. (2008). Expanding dialogues: from natural autoinducers to non-natural analogues that modulate quorum sensing in Gram-negative bacteria. Chem. Soc. Rev. 37, 1432–1447.

Hanzelka, B.L., and Greenberg, E.P. (1995). Evidence that the N-terminal region of the Vibrio fischeri LuxR protein constitutes an autoinducer-binding domain. J. Bacteriol. 177, 815–817.

Henke, J.M., and Bassler, B.L. (2004). Three parallel quorum-sensing systems regulate gene expression in Vibrio harveyi. J. Bacteriol. 186, 6902–6914.

Hentzer, M., and Givskov, M. (2003). Pharmacological inhibition of quorum sensing for the treatment of chronic bacterial infections. J. Clin. Invest. 112, 1300–1307.

Hussain, M.B., Zhang, H.B., Xu, J.L., Liu, Q., Jiang, Z., and Zhang, L.H. (2008). The acyl-homoserine lactone-type quorum-sensing system modulates cell motility and virulence of Erwinia chrysanthemi pv. zeae. J. Bacteriol. 190, 1045–1053.

Karimova, G., Pidoux, J., Ullmann, A., and Ladant, D. (1998). A bacterial two-hybrid system based on a reconstituted signal transduction pathway. Proc. Natl. Acad. Sci. USA 95, 5752–5756.

Koch, B., Liljefors, T., Persson, T., Nielsen, J., Kjelleberg, S., and Givskov, M. (2005). The LuxR receptor: the sites of interaction with quorum-sensing signals and inhibitors. Microbiology 151, 3589–3602.

Manefield, M., Rasmussen, T.B., Henzter, M., Andersen, J.B., Steinberg, P., Kjelleberg, S., and Givskov, M. (2002). Halogenated furanones inhibit quorum sensing through accelerated LuxR turnover. Microbiology 148, 1119–1127.

McClean, K.H., Winson, M.K., Fish, L., Taylor, A., Chhabra, S.R., Camara, M., Daykin, M., Lamb, J.H., Swift, S., Bycroft, B.W., et al. (1997). Quorum sensing and Chromobacterium violaceum: exploitation of violacein production and inhibition for the detection of N-acylhomoserine lactones. Microbiology 143, 3703–3711.

McCoy, A.J., Grosse-Kunstleve, R.W., Adams, P.D., Winn, M.D., Storoni, L.C., and Read, R.J. (2007). Phaser crystallographic software. J. Appl. Cryst. 40, 658–674.

Miller, M.B., and Bassler, B.L. (2001). Quorum sensing in bacteria. Annu. Rev. Microbiol. 55, 165–199.

Minogue, T.D., Wehland-von Trebra, M., Bernhard, F., and von Bodman, S.B. (2002). The autoregulatory role of EsaR, a quorum-sensing regulator in Pantoea stewartii ssp. stewartii: evidence for a repressor function. Mol. Microbiol. 44, 1625–1635.

Minogue, T.D., Carlier, A.L., Koutsoudis, M.D., and von Bodman, S.B. (2005). The cell density-dependent expression of stewartan exopolysaccharide in Pantoea stewartii ssp. stewartii is a function of EsaR-mediated repression of the rcsA gene. Mol. Microbiol. 56, 189–203.

Morohoshi, T., Kato, M., Fukamachi, K., Kato, N., and Ikeda, T. (2008). N-acylhomoserine lactone regulates violacein production in Chromobacterium violaceum type strain ATCC 12472. FEMS Microbiol. Lett. 279, 124–130.

Müh, U., Schuster, M., Heim, R., Singh, A., Olson, E.R., and Greenberg, E.P. (2006). Novel Pseudomonas aeruginosa quorum-sensing inhibitors identified in an ultra-high-throughput screen. Antimicrob. Agents Chemother. 50, 3674–3679.

Murshudov, G.N., Vagin, A.A., and Dodson, E.J. (1997). Refinement of macromolecular structures by the maximum-likelihood method. Acta Crystallogr. D Biol. Crystallogr. 53, 240–255.

Ni, N., Li, M., Wang, J., and Wang, B. (2009). Inhibitors and antagonists of bacterial quorum sensing. Med. Res. Rev. 29, 65–124.

Njoroge, J., and Sperandio, V. (2009). Jamming bacterial communication: new approaches for the treatment of infectious diseases. EMBO Mol. Med. 1, 201–210.

Otwinowski, Z., and Minor, W. (1998). Processing of x-ray diffraction data collected in oscillation mode. Methods Enzymol. 276, 307–326.

Passador, L., Cook, J.M., Gambello, M.J., Rust, L., and Iglewski, B.H. (1993). Expression of Pseudomonas aeruginosa virulence genes requires cell-to-cell communication. Science 260, 1127–1130.

Pinto, U.M., and Winans, S.C. (2009). Dimerization of the quorum-sensing transcription factor TraR enhances resistance to cytoplasmic proteolysis. Mol. Microbiol. 73, 32–42.

Piper, K.R., Beck von Bodman, S., and Farrand, S.K. (1993). Conjugation factor of Agrobacterium tumefaciens regulates Ti plasmid transfer by autoinduction. Nature 362, 448–450.

Pompeani, A.J., Irgon, J.J., Berger, M.F., Bulyk, M.L., Wingreen, N.S., and Bassler, B.L. (2008). The Vibrio harveyi master quorum-sensing regulator, LuxR, a TetR-type protein is both an activator and a repressor: DNA recognition and binding specificity at target promoters. Mol. Microbiol. 70, 76–88.

Ross, W., and Gourse, R.L. (2009). Analysis of RNA polymerase-promoter complex formation. Methods 47, 13–24.

Ross, W., Thompson, J.F., Newlands, J.T., and Gourse, R.L. (1990). *E.coli* Fis protein activates ribosomal RNA transcription in vitro and in vivo. EMBO J. 9, 3733–3742.

Sheldrick, G.M. (2008). A short history of SHELX. Acta Crystallogr. A 64, 112–122.

Swem, L.R., Swem, D.L., Wingreen, N.S., and Bassler, B.L. (2008). Deducing receptor signaling parameters from in vivo analysis: LuxN/Al-1 quorum sensing in Vibrio harveyi. Cell 134, 461–473.
Molecular Cell

CviR Antagonist Mechanism

Swem, L.R., Swem, D.L., O’Loughlin, C.T., Gatmaitan, R., Zhao, B., Ulrich, S.M., and Bassler, B.L. (2009). A quorum-sensing antagonist targets both membrane-bound and cytoplasmic receptors and controls bacterial pathogenicity. Mol. Cell 35, 143–153.

Vannini, A., Volpari, C., Gargioli, C., Muraglia, E., Cortese, R., De Francesco, R., Neddermann, P., and Marco, S.D. (2002). The crystal structure of the quorum sensing protein TraR bound to its autoinducer and target DNA. EMBO J. 21, 4393–4401.

von Bodman, S.B., Ball, J.K., Faini, M.A., Herrera, C.M., Minogue, T.D., Urbanowski, M.L., and Stevens, A.M. (2003). The quorum sensing negative regulators EsaR and ExpR(Ecc), homologues within the LuxR family, retain the ability to function as activators of transcription. J. Bacteriol. 185, 7001–7007.

Waters, C.M., and Bassler, B.L. (2005). Quorum sensing: cell-to-cell communication in bacteria. Annu. Rev. Cell Dev. Biol. 21, 319–346.

Winn, M.D., Isupov, M.N., and Murshudov, G.N. (2001). Use of TLS parameters to model anisotropic displacements in macromolecular refinement. Acta Crystallogr. D Biol. Crystallogr. 57, 122–133.

Zhang, R.G., Pappas, T., Brace, J.L., Miller, P.C., Oulmassov, T., Molyneaux, J.M., Anderson, J.C., Bashkin, J.K., Winans, S.C., and Joachimiak, A. (2002). Structure of a bacterial quorum-sensing transcription factor complexed with pheromone and DNA. Nature 417, 971–974.

Zhu, J., and Winans, S.C. (1999). Autoinducer binding by the quorum-sensing regulator TraR increases affinity for target promoters in vitro and decreases TraR turnover rates in whole cells. Proc. Natl. Acad. Sci. USA 96, 4832–4837.

Zhu, J., and Winans, S.C. (2001). The quorum-sensing transcriptional regulator TraR requires its cognate signaling ligand for protein folding, protease resistance, and dimerization. Proc. Natl. Acad. Sci. USA 98, 1507–1512.

Zhu, J., Dizin, E., Hu, X., Wavreille, A.S., Park, J., and Pei, D. (2003). S-Ribosylhomocysteine (LuxS) is a mononuclear iron protein. Biochemistry 42, 4717–4726.

Zou, Y., and Nair, S.K. (2009). Molecular basis for the recognition of structurally distinct autoinducer mimics by the *Pseudomonas aeruginosa* LasR quorum-sensing signaling receptor. Chem. Biol. 16, 961–970.
