
REVIEW

Mechanisms and functions of Hedgehog signalling across the metazoa

Philip W. Ingham*,†, Yoshiro Nakano*§ and Claudia Seger*

Abstract | Hedgehog proteins constitute one of a small number of families of secreted signals that have a central role in the development of metazoans. Genetic analyses in flies, fish and mice have uncovered the major components of the pathway that transduces Hedgehog signals, and recent genome sequence projects have provided clues about its evolutionary origins. In this Review we provide an updated overview of the mechanisms and functions of this signalling pathway, highlighting the conserved and divergent features of the pathway, as well as some of the common themes in its deployment that have emerged from recent studies.

A defining event in the emergence of multicellularity was the harnessing and refinement of mechanisms that facilitate interaction and communication between cells. Three decades of intensive research by developmental biologists have identified the key molecular players that underpin these mechanisms and have unveiled the plethora of processes that they control. With the recent rapid growth in the availability of complete genome sequences of a wide array of organisms, we are now gaining new insights into the origins and evolution of this molecular machinery. One clear conclusion to be drawn from all these studies is that evolution has worked with a rather limited number of signalling pathways to generate the remarkable variety and complexity of form and function that we see today, both across phyla and within the individuals of a given species. Understanding how these pathways have evolved and function can thus illuminate the origin of morphological diversity, as well as the molecular and cellular basis of development and disease.

Amongst the central canon of developmental signalling pathways — transforming growth factor-β (TGFβ)–bone morphogenetic protein (BMP), the JAK–signal transducer and activator of transcription (STAT) protein kinases, epidermal growth factor receptor (EGFR), NOTCH, fibroblast growth factor (FGF), WNT and Hedgehog (HH) — the HH pathway has been and perhaps remains the most enigmatic. In contrast to the other pathways, early studies of HH signalling were exclusively based on genetic analysis in *Drosophila melanogaster*¹; by a twist of evolutionary fate the pathway has been lost from that rival workhorse of developmental geneticists, the nematode *Caenorhabditis elegans*². Moreover, unlike members of most of the other pathways, HH signalling components were not initially identified through their potential to transform cultured mammalian cells or to promote tumour formation in mice and humans. Perhaps partly for this reason, the characterization of the molecular and cellular basis of HH signalling has lagged behind that of its more mainstream counterparts. However, another impediment to progress has undoubtedly been the Byzantine biochemical nature of the pathway: the coupling of the HH ligand to cholesterol³; the role of a large multipass transmembrane transporter-like protein, Patched (PTC), as the ligand receptor⁴⁻⁶; the lack of physical interaction between HH or PTC and the signal transducer Smoothened (SMO), which is a member of the G protein-coupled receptor (GPCR) superfamily that is neither a receptor nor obviously coupled to heterotrimeric G proteins⁷. These features have all conspired to make the elucidation of the HH pathway both intriguing but apparently intractable in equal measure. Add to this its dependence in vertebrates on an ancient organelle — the primary cilium — that has seemingly been dispensed with by *Drosophila* spp. and other insects, and you have a story that continues to retain the capacity to confound and surprise even the experts in the field.

In this Review we provide an updated overview of the HH pathway in the context of its evolutionary origins. We address the origin of the HH ligands and how they are processed and distributed; describe the core elements of the signal transduction pathway, as defined by

*NATURE REVIEWS | GENETICS*  VOLUME 12 | JUNE 2011 | 393

© 2011 Macmillan Publishers Limited. All rights reserved

*Institute of Molecular and Cell Biology, 61 Biopolis Drive, Singapore 138673.
†Department of Biological Science, National University of Singapore, 14 Science Drive 4, Singapore 117543.
§Department of Genetics, Hyogo College of Medicine, 1-1 Mukogawa-cho, Nishinomiya, Hyogo 663-8501, Japan.
Correspondence to P.W.I.
e-mail: pingham@imcb.a-star.edu.sg
doi:10.1038/nrg2984
Published online 19 April 2011

REVIEW S

a
|  |  |
| --- | --- |
| SS | HH-N |
|  | HH-C |
|  |  |
| SS | Hedge |
|  | Hog |

b
|  |  |
| --- | --- |
| Monosiga brevicollis |  |
| Amphimedon queenslandica |  |
| Nematostella vectensis |  |
| Drosophila melanogaster | Atypical proto-cadherin family member Fat |

| LAG | Ig domain | EGF repeat | vWA | TNFR repeat |
| --- | --- | --- | --- | --- |
| Furin repeat | l-set domain | Nhh | CA repeat | SH2 |

Figure 1 | Structure of Hedgehog and Hedgehog-related proteins. a | The full-length Hedgehog (HH) protein is comprised of two distinct domains: the amino-terminal ‘Hedge’ domain (shown in dark green) and the carboxy-terminal ‘Hog’ domain (shown in blue), both of which are also found in proteins other than members of the HH family. The Hedge domain is preceded by a signal peptide sequence (SS; shown in brown). The Hog domain itself can be separated into two regions; the first two-thirds share similarity to self-splicing inteins, and this module has been named ‘Hint’, whereas the C-terminal one-third binds cholesterol in HH proteins and has been named the sterol-recognition region (SRR). In other Hog domain-containing proteins this region is referred to as the adduct recognition region (ARR), as the nature of the adduct is not known. The Hog domain promotes autocleavage to release the N-terminal Hedge domain (HH-N). b | Hedge domain-containing proteins predate HH and are found in protozoa and metazoa. The choanoflagellate Monosiga brevicollis has a large Hedge domain-containing transmembrane protein that also contains a von Willebrand A domain (vWA), two cadherin repeats (CA repeats), multiple tumour necrosis factor receptor repeats (TNFR repeats), as well as single immunoglobulin (lg), immunoglobulin l-set (l-set) and epidermal growth factor (EGF)-like repeats<sup>14</sup>; the Hedge domain is designated Nhh. Proteins with a similar domain organization, though with an expansion of the CA repeats at the expense of the TNFR repeats, are encoded by the so-called hedgling gene in the basal metazoan, Amphimedon queenslandica (sponge)<sup>15</sup> and the Cnidarian, Nematostella vectensis<sup>15</sup>. Interestingly, the hedgling structure resembles that of the atypical proto-cadherin family members, including the Fat and Dachsous proteins, which have important roles in planar cell polarity in Drosophila melanogaster<sup>154</sup>. LAG, Laminin A-G motif; SH2, SH2 motif.

Cadherin repeats
Extracellular calcium-binding domains originally identified in the calcium-dependent adhesion proteins, cadherins.

TNFR repeats
Cysteine-rich motifs originally defined in the extracellular domain of the tumour necrosis factor receptor but also found in laminins and other receptors.

Inteins
Protein segments that can self-excite and rejoin the remaining portions of the protein with a peptide bond.

Structure and properties of Hedgehog proteins
Hedgehog proteins are synthesized as approximately 45kDa pro-proteins that undergo autocleavage to yield two similarly sized fragments<sup>8</sup>: an amino-terminal polypeptide (HH-N), which is characterized by a signal sequence and a highly conserved ‘Hedge’ domain, and a carboxy-terminal polypeptide (HH-C), which contains the highly conserved ‘Hog’ domain<sup>9</sup> (FIG. 1a). All of the signalling activity of HH proteins is vested in the secreted HH-N fragment, which has the unusual property of being covalently coupled to cholesterol at its C-terminal end<sup>3</sup>. The only known function of the Hog domain is to promote the autocleavage reaction, a process that resembles the protein-splicing activity of inteins and requires the ‘Hint’ region, which forms part of the Hog domain<sup>10</sup>. The cleavage reaction is driven

394 | JUNE 2011 | VOLUME 12
www.nature.com/reviews/genetics

© 2011 Macmillan Publishers Limited. All rights reserved
by a nucleophilic attack on a highly conserved GCF motif in which cholesterol, recruited by the sterol recognition region (SRR) of the Hog domain, serves as the electron donor³.

**The Hedge and Hog domains have ancient origins.** Although Hedgehog proteins are the only proteins known to be conjugated to cholesterol in this way, the role of the Hog domain in processing secreted proteins seems to be an evolutionary ancient one: Hog domain proteins have been identified in a wide variety of organisms, including red algae, dinoflagellates and mosses, as well as throughout the metazoa (although not in higher plants)¹¹. In the choanoflagellate *Monosiga ovata*, the protist that is most closely related to the metazoa, the ‘Hoglet’ gene encodes a protein with a C-terminal Hog domain that is coupled to an N-terminal domain with sequence similarity to cellulose-binding proteins¹². However, in other organisms Hog domains have been found to be associated with an assortment of unrelated N-terminal domains. Only in the eumetazoa does the association of the Hedge and Hog domains in the form of Hedgehog proteins first appear¹¹.

The Hedge domain, which has a striking structural similarity to the *Streptomyces albus* metalloproteinase d,d-carboxypeptidase¹³, is also found in *Monosiga* spp., in the N-terminal extracellular domain of a large transmembrane protein¹⁴ (FIG. 1b). A protein with a similar domain organization is encoded by the so-called Hedgling gene in the basal metazoan *Amphimedon queenslandica* (Porifera)¹⁵. The Cnidarian, *Nematostella vectensis* (the sea anemone) also has a Hedgling gene, as well as two true Hedgehog genes¹⁶; by contrast, no Hedgling genes have been reported in any bilaterian species. Thus, it seems likely that Hedgehog proteins first arose in the common ancestor of the Cnidarians and the bilateria more than 650 million years ago through the combination of domains that were present in pre-existing proteins, and that the Hedgling gene was subsequently lost from the bilateria¹¹. Most bilaterians have been shown to possess at least one Hedgehog gene, with the genome expansions in vertebrates giving rise to three genes in amniotes (*Desert Hedgehog* (*Dhh*), *Indian Hedgehog* (*Ihh*) and *Sonic Hedgehog* (*Shh*)) and four or five (*Dhh*, *Ihha*, *Ihhb*, *Shha* and *Shhb*) in different teleost species¹⁷. A notable exception is *C. elegans*, which possesses multiple Hog domain-containing proteins but lacks a true Hedgehog gene². By contrast, some other nematode species have retained a Hedgehog gene, though, like *C. elegans* and its closer relatives, they have lost some of the components of the core signal transduction pathway¹¹ (described below).

Lipid modifications control the release and movement of HH. In addition to the covalent coupling of cholesterol to their C-termini, HH-N proteins also undergo palmitoylation at their N-termini, a modification that is promoted by an acyl transferase encoded in *D. melanogaster* by the skinny hedgehog (*ski*) gene¹⁸ and in vertebrates by its orthologue, *HHAT*¹⁹ (FIG. 2). This dual lipid modification of the HH signalling protein has important effects on its properties, both enhancing its membrane association²⁰ and potentiating its secretion and range of action. The secretion of lipidated HH-N requires the activity of a large multipass transmembrane protein, Dispatched (DISP)²¹. In mutant animals lacking DISP function, HH accumulates in producing cells and all HH-elicted responses are lost, except in cells that are immediately adjacent to signal producers²¹⁻²⁴. Truncated proteins that are engineered to mimic HH-N but which lack the cholesterol moiety can override this requirement for DISP; however, the range and potency of such cholesterol-free HH-N is severely compromised compared with the normal protein, suggesting that the modification is crucial for the extracellular movement of the signal following secretion²⁵,²⁶. Consistent with this, the biochemical analysis of vertebrate SHH expressed in tissue culture cells indicates that lipid modification promotes the formation of freely diffusible multimeric complexes²⁷.

In *D. melanogaster*, studies of HH signalling in imaginal discs have shown that the cholesterol adduct is essential for the incorporation of HH-N into lipoprotein particles that seem to mediate its long-range transport across epithelia²⁸ (FIG. 2). The assembly of these HH signalling entities is promoted by interaction with lipophorin, an apo-lipoprotein that circulates in the haemolymph of the fly²⁹. Lipophorin is recruited to HH-N-secretory cells by its interaction with the heparin sulphate moieties of the glypicans Dally and Dally-like²⁹. These proteoglycans, which can also interact with HH³⁰,³¹, localize to the apical surfaces of epithelial cells via GPI anchors, the cleavage of which seems to be required for effective long-range HH signalling³². Although the properties of the Dally and Dally-like mutants can be subject to differing interpretations²⁹,³²,³³, one appealing possibility is that glypicans promote the assembly of HH-N–lipophorin particles at the plasma membrane and that cleavage of their GPI anchor facilitates the release and dispersal of HH from producing cells (FIG. 2).

Notably, HH-N also localizes to the basolateral membrane of secreting and receiving cells in imaginal discs, leading to the proposal by some authors that this represents the major route of signal secretion and spread³⁴,³⁵. Experimental generation of HH-N ‘traps’ in groups of responding cells, however, suggests that the range of the basally secreted protein is fairly limited³²: this raises the interesting possibility that basal HH-N principally acts as a short-range signal, whereas apically released HH-N acts as a long-range signal. In line with this, apically localized HH-N can be observed across the imaginal disc epithelium in a graded distribution that mirrors the functionally defined gradient of signalling activity³².

Whether the secretion and movement of HH proteins is regulated in a similar way in vertebrates is currently unclear. Certainly, lipid modification has an important role in the release of SHH²⁵, and a graded distribution that mirrors its dose-dependent effects can be observed in the developing neural tube. But whether this distribution reflects the movement of the protein itself or the movement of the cells that have taken up the protein remains to be determined³⁶.

---

### Definitions

- **Choanoflagellates**: A group of protists that possess one flagellum at some stage of their life history.
- **Eumetazoa**: An animal sub-kingdom that includes Cnidarians, the ctenophorans and the bilaterians.
- **Porifera**: A phylum of multicellular animals with only two cell layers, the ectoderm and the endoderm, which are separated by an acellular mesoglea. Commonly known as sponges.
- **Cnidarians**: A simple and ancient phylum of multicellular animals, such as jelly fish or corals, found mainly in marine environments.
- **Palmitoylation**: The covalent attachment of palmitic acid or other fatty acids to cysteine residues of proteins that promotes their association with membranes.
- **Imaginal discs**: The undifferentiated epithelial sheets of cells that give rise to adult structures such as wings and legs.
- **Lipophorin**: A family of high-density lipid-transporting lipoproteins found in the haemolymph of insects.
- **Glypicans**: A family of heparan sulphate proteoglycans that are localized to the cell surface via GPI anchors.
- **GPI anchor**: A glycolipid, glycosylphosphatidylinositol, linked to the C-terminal amino acid of proteins anchoring them to the outer leaflet of the plasma membrane.
REVIEW S

![Diagram](image.png)

Figure 2 | Lipid modification and release of the Hedgehog ligand. Following its translation, full-length Hedgehog (HH) undergoes autoprolysis in the endoplasmic reticulum (ER)<sup>155</sup>, resulting in its covalent coupling to cholesterol. The amino-terminal ‘Hedge’ domain (HH-N) fragment is further modified through N-terminal palmitoylation that is promoted by the transmembrane acyl transferase Skinny hedgehog (SKI). Release of this doubly lipid-conjugated form of HH requires the activity of the multipass transmembrane protein Dispatched (DISP), which probably transports the protein across the plasma membrane. Once on the outer surface of the cell, HH remains associated with the lipid bilayer and interacts with the heparan sulphate moieties of the glypicans that are encoded by the dally and dally-like genes (in *Drosophila melanogaster*). These are also thought to recruit the apolipoprotein lipophorin, which, together with HH, becomes assembled into lipoprotein particles. Release of these particles might be mediated by the phospholipase C-like Notum, which cleaves the GPI anchors from the glypicans (indicated by scissors).

Another important parameter in regulating HH protein distribution is its sequestration by various transmembrane proteins. The best characterized of these is PTC, which also has a pivotal role in the reception and transduction of the HH signal (see below). Activation of the HH pathway leads to the upregulation of PTC expression, which in turn enhances HH sequestration and thus attenuates its signalling activity. Recent studies in *D. melanogaster* have suggested that the Brother of interference hedgehog (BOI) protein, which functions as a co-receptor for the HH signal, also regulates HH signalling by ligand sequestration<sup>37</sup>. Hedgehog-interacting protein (HHIP), which was first identified in vertebrates but which was subsequently found to be encoded by the sponge genome, has a similar role<sup>38</sup>. Expression of HHIP is also upregulated in response to HH signalling but, in contrast to PTC and BOI, HHIP seems to act exclusively to modulate HH ligand distribution and has no role in signal reception or transduction. Interestingly, *D. melanogaster* lack both an HHIP-encoding gene, as well as the zinc-binding site in HH that is crucial for its interaction with HHIP<sup>39</sup>.

The GLI transcription factors are key effectors of HH activity. Although recent studies have suggested a role for HH in modulating the cytoskeleton via SRC family kinases<sup>40</sup>, the most widespread and best-characterized response of cells to HH signalling is the upregulation of transcription of target genes. Only a few such targets have been described in detail<sup>41–44</sup>, but recent genome-wide analyses suggest that there are several hundred<sup>45–47</sup>. Some targets (for example, the gene encoding PTC) are shared across different phyla<sup>44,48–50</sup>, whereas others are almost certainly specific to particular orders, families or even species.

The principal mediators of the transcriptional response to HH are members of the zinc finger-containing GLI protein family<sup>43,44,51</sup>. Genes encoding GLI proteins are found in the most primitive metazoa, the sponges<sup>52</sup>, and so predate the probable origin of the HH pathway itself (FIG. 3). *D. melanogaster* has a single GLI protein that is encoded by the *cubitus interruptus (ci)* gene<sup>44</sup>. The CI protein and its vertebrate counterparts, GLI2 and GLI3, are bi-functional transcription factors: their

---

**GLI protein family**

Zinc finger domain-containing transcription factors mediating HH activity, so-called because the gene encoding the founding member GLI1 is frequently amplified in glioblastoma cells.

---

**The Hedgehog signal transduction pathway**

Originally defined through genetic analysis in *D. melanogaster*<sup>1</sup>, the components of the HH signalling pathway have subsequently been functionally characterized in a number of vertebrate species — principally, mouse, zebrafish and human — and have also been identified through genome sequence analyses in species from a wide range of phyla (FIG. 3). These studies have revealed a high level of conservation of the ‘core’ components of the signal transduction pathway that is likely to extend across the eumetazoa, as well as some variation in their deployment that has arisen over the course of evolution. First, we outline the highly conserved core aspects of the pathway and then consider its divergence between species.

Monosiga brevicollis

Amphimedon  
queenslandica

Trichoplax  
adherens

Nematostella  
vectensis

Drosophila  
melanogaster

Mus  
musculus

CI
CKI

DISP
FU

GLI
GSK3

HH
HHIP

HLING
PKA

IHOG (CDO in M. musculus)

KIF7 (COS in D. melanogaster)

PTC
SKI

SMO
SUFU

Ancient eukaryotes
Choanoflagellata
Porifera
Placozoa
Cnidaria
Bilateria

Ancient eukaryotes
Metazoan (animal origin)
Bilateria (vertebrate origin)
Holozoa
Eumetazoan origin

Figure 3 | Conservation of individual molecules across species with phylogenetic representation. Genomic inventory of Hedgehog (HH) pathway components in species representative of different phyla. The components are colour-coded to indicate their likely origin and are distributed between receiving cells (shaded beige, nuclei shown as ovals) and signalling cells (shaded purple, top of panels). Interactions established on the basis of genetic and/or biochemical data are indicated by the clustering of components. CKI, casein kinase 1; COS, Costal; DISP, Dispatched; FU, Fused; GSK3, glycogen synthase kinase 3; HHIP, Hedgehog-interacting protein; HLING, hedgling; IHOG, Interference hedgehog; PKA, protein kinase A; PTC, Patched; SKI, Skinny hedgehog; SMO, Smoothened; SUFU, Suppressor of fused. Left panels are modified, with permission, from REF. 38 © (2009) CSHL Press. Upper right panel is modified, with permission, from REF. 52 © (2010) Macmillan Publishers Ltd. All rights reserved.

full-length forms function as transcriptional activators, but they can be converted into lower molecular mass transcriptional repressors by removal of their C-terminal activation domains<sup>51,53</sup>. It is principally by modulating the modification, processing and nuclear trafficking of the GLI proteins that HH signals elicit their transcriptional responses (FIG. 4). In vertebrates, these responses are amplified by a third member of the family, GLI1, which lacks the N-terminal repressor domain and functions exclusively as a transcriptional activator<sup>54</sup>.

**Reception and transduction of the HH signal by PTC and SMO.** The transcriptional response to HH depends on the activity of a heptahelical transmembrane protein, SMO, which has been shown by genetic analysis to be indispensable for HH signalling both in *D. melanogaster* and in vertebrates<sup>55–59</sup>. Genome sequence analyses have identified SMO orthologues across the bilateria, as well as in Cnidaria, but not in *Amphimedon queenslandica*, which is consistent with the proposed eumetazoan origin of the HH pathway<sup>38</sup>. Despite being a member of the superfamily of GPCRs, there is no evidence that SMO acts as a receptor for HH proteins. Rather, its activity is regulated by PTC, orthologues of which are present in many bilaterian species<sup>38</sup> and in Cnidaria<sup>16</sup> but, like SMO, not in *Amphimedon queenslandica*<sup>52</sup> (FIG. 3).

Genetic analyses in *D. melanogaster* and vertebrates have shown that PTC inhibits SMO activity, so the activation of HH-target genes by the GLI proteins ultimately depends on the inhibition of PTC (FIG. 4). Inhibition is effected by HH binding to the two large extracellular domains of PTC<sup>60</sup>, an interaction that is promoted in *D. melanogaster* by the transmembrane proteins Interference hedgehog (IHOG) and BOI<sup>6</sup>, and in vertebrates by their orthologues CDO and brother of CDO (BOC)<sup>61</sup>. Genes encoding these proteins are present in the genomes of *Amphimedon queenslandica*<sup>38,52</sup> and the placazoan *Trichoplax adhaerens*<sup>62</sup> and thus predate the origins of the HH pathway — but they have been recruited into the pathway to function as co-receptors for the HH proteins. Interestingly, the fibronectin domain that mediates binding to HH differs between the orthologous *D. melanogaster* and vertebrate proteins<sup>63</sup>.

A major unresolved issue in HH signalling is the biochemical mechanism by which PTC inhibits SMO activity. Early suggestions that the two proteins form a HH receptor complex<sup>64</sup> were refuted by the demonstration that PTC inhibits SMO non-stoichiometrically<sup>65</sup>. The similarity of PTC to bacterial RND proteins<sup>66</sup> has led to the suggestion that it may act as a transporter, possibly of a lipid activator or inhibitor of SMO<sup>65</sup>. This is supported by the finding that the steroidal alkaloid cyclopamine binds to and inhibits SMO activity<sup>67</sup>. Recent studies in *D. melanogaster* have implicated the phospholipid phosphatidylinositol-4-phosphate (PI4P; also known as PtdIns4P) in the regulatory relationship between the two proteins<sup>68</sup>. These findings, based on the genetic modulation of PI4P synthesis and turnover and supported by data from experiments using mammalian fibroblasts, suggest that SMO is activated by an increase in intracellular PI4P levels and that PTC modulates these levels by inhibiting the activity of the kinase that is responsible for PI4P synthesis<sup>68</sup>. Exactly how PI4P levels could regulate the hypothetical lipid modulator of SMO remains unclear, but one suggestion is that it might mediate the intracellular trafficking of such a molecule.

**Downstream of SMO.** Despite this major gap in our knowledge, it is well established that SMO transduces the HH signal into the cell and attenuates the processing of the CI (GLI in vertebrates) proteins: but how is this accomplished? Cleavage of CI (GLI) occurs in the absence of HH stimulation — and hence of SMO activation — and is promoted by the phosphorylation of motifs within the C-terminal domain of CI (GLI)<sup>69</sup>. These motifs are phosphorylated, first by protein kinase A (PKA), which primes them for further phosphorylation at adjacent residues by two other serine/threonine kinases, glycogen synthase kinase 3 (GSK3) and casein kinase 1 (CKI)<sup>69–71</sup>. The phosphorylated motifs are recognized by the F-box protein BTRCP (known as SLMB in *D. melanogaster*), a component of the SCF complex, which in turn catalyses the ubiquitylation of the C terminus, targeting it for degradation by the proteasome to yield the truncated N-terminal repressor forms of CI (GLI)<sup>72,73</sup>. Such kinase recognition motifs are found in GLI proteins across the bilateria, which is consistent with the modulation of their phosphorylation being fundamental to the response of cells to HH signalling. Exactly how SMO activity effects this modulation, however, is still far from fully understood (BOX 1; see Supplementary information S1 (figure)).

The most complete picture that is currently available comes from studies in *D. melanogaster* in which genetic analysis has identified *costal 2* (*cos2*) as a key negative regulator of the HH pathway<sup>74</sup>. The *cos2* gene encodes the *D. melanogaster* orthologue of KIF7 (REFS 75,76), which is a highly conserved member of the kinesin family of motor proteins with roles that predate the emergence of the HH pathway; in the slime mould *Dictyostelium discoideum* for instance, KIF7 is required for the transport of cellular organelles<sup>77</sup>. Despite the integrity of the motor domain in *D. melanogaster* COS2 not being fully conserved<sup>76</sup>, studies in tissue culture cells have indicated that it retains considerable motor activity and can mediate the movement of tagged forms of CI within the cytoplasm in an ATP- and microtubule-dependent manner<sup>78</sup>. Although the importance of this movement remains unclear, many lines of evidence point to a role for COS2 in recruiting CI, PKA, GSK3 and CKI to form the HH signalling complex (HSC)<sup>75,79–81</sup>. Activation of SMO causes the partial dissociation of this HSC, thus relieving CI from phosphorylation and promoting the accumulation of its full-length form (FIG. 4b).

The HSC contains another ancient serine/threonine kinase, Fused (FU), orthologues of which control cytokinesis in higher plants<sup>82</sup> and cell polarization and chemotaxis in *Dictyostelium discoideum*<sup>83</sup>. In *D. melanogaster*, FU is a positive regulator of the HH pathway: translocation of FU to the plasma membrane induces the phosphorylation of COS2, which promotes the dissociation of CI from the HSC<sup>79,84</sup>. In addition, FU promotes

RND proteins  
A superfamily of transmembrane proteins, originally defined by bacterial proteins that function as transporters in drug resistance, legume nodulation and cell division (hence RND), but now including various eukaryotic proteins with diverse functions.

Proteasome  
Large protein complex that contains proteases that regulate the concentration of particular cellular proteins and degrade misfolded proteins by proteolysis.

Figure 4 | The Hedgehog signalling pathway in *Drosophila melanogaster*. Schematic representation of the subcellular localization and interactions of the core components of the Hedgehog (HH) signalling pathway in *Drosophila melanogaster*, in the absence (part **a**) or presence (part **b**) of HH ligand. In uninduced cells (part **a**), Patched (PTC) inhibits Smoothened (SMO) (dashed inhibitory arrow), which is located predominantly in intracellular vesicles. Cubitus interruptus (CI) is recruited to the Costal 2 (COS2) scaffold protein, which also recruits a number of serine/threonine kinases to form the HH signalling complex (HSC) (represented by the cluster of proteins; centre); phosphorylation (P) (dashed arrow) of full-length CI (CI-FL) by protein kinase A (PKA), glycogen synthase kinase 3 (GSK3) and casein kinase 1 (CKI) generates recognition signals for the F-box protein Slimb (SLMB), which catalyses the ubiquitylation of the carboxyl terminus of the protein, targeting it for degradation by the proteasome. The resulting truncated, repressor form of CI (CI-R) dissociates from the complex (solid arrow) and translocates to the nucleus where it represses transcription of HH target genes. Secreted HH binds to the transmembrane proteins Interference hedgehog (IHOG) and Brother of interference hedgehog (BOI; not shown) (part **b**), which present HH to PTC. Binding of HH to PTC blocks the activity of PTC, thereby releasing SMO from inhibition; SMO translocates to the plasma membrane where it is phosphorylated by PKA, GSK3 and CKI. This phosphorylation causes a conformational change in the SMO C-terminal domain, enhancing its interaction with COS2. Phosphorylation of COS2 by Fused (FU; dashed arrow) causes CI to be released from the HSC, abrogating its phosphorylation and so promoting the accumulation of its full-length<sup>156</sup> form. FU-dependent phosphorylation of Suppressor of fused (SUFU; dashed arrow) promotes its dissociation from CI-FL, allowing CI-FL to translocate to the nucleus where it undergoes further modification to its activated form (CI-A) and thus promotes the transcriptional activation of HH target genes.

the phosphorylation of suppressor of fused (SUFU)<sup>30</sup>, a protein that is highly conserved throughout the metazoa and shares a strong structural similarity to several bacterial proteins<sup>85</sup>. In *D. melanogaster*, SUFU binds to and stabilizes CI in the cytoplasm<sup>86,87</sup>, attenuating its nuclear import once released from the HSC<sup>88,89</sup>; phosphorylation of SUFU abrogates this interaction, thereby promoting target gene activation by CI. Thus, in unstimulated cells the absence of SUFU has no effect, but its removal is sufficient to restore pathway activity in the absence of FU activity.

Pathway divergence and the primary cilium. In contrast to the situation in *D. melanogaster*, loss-of-function mutations of the mouse SUFU orthologue cause widespread ligand-independent activation of HH target genes<sup>90,91</sup>, a phenotype which is similar to that seen in PTC loss-of-function mutants<sup>92</sup>. Interaction between SUFU and GLI proteins seems to have a key role in promoting the processing of GLIs into their truncated repressor forms in the absence of HH signalling<sup>93,94</sup>. Strikingly, functional studies have revealed that SUFU has a similarly crucial role in HH signalling in planarians<sup>95,96</sup>, suggesting that

the PTC–SMO–SUFU–GLI axis represents the core components of the HH pathway, and implying that the role of SUFU has been largely subsumed by COS2 in *D. melanogaster*.

A further notable interspecies difference, which was initially uncovered in mice, is the dependence of HH signalling in some species on the primary cilium<sup>97</sup> — an ancient organelle that is found in many eumetazoa but that is conspicuously absent from most *D. melanogaster* cells<sup>98</sup>. In mammalian cells, exposure to HH ligand causes SMO to localize to the axoneme of the primary cilium<sup>99</sup>, quickly followed by the accumulation of the SUFU–GLI complex at the cilium tip<sup>100,101</sup>. This translocation seems to facilitate the dissociation of the complex in response to SMO activity<sup>102</sup>, thereby abrogating cleavage of GLI proteins and promoting the import of their full-length forms into the nucleus where they are converted, by an as yet uncharacterized mechanism, into labile transcriptional activators<sup>93</sup>. How this dissociation is effected by SMO and how the GLI proteins move from the cilium to the nucleus is currently unclear.

Paradoxically, despite the importance of SUFU, the murine orthologue of FU has no role in HH signalling

in mice<sup>103,104</sup>, and although another kinase, CDC2L1, has been implicated as a negative regulator of mammalian SUFU<sup>105</sup>, there is currently no evidence that SUFU is phosphorylated in response to HH activity<sup>102</sup>. By contrast, a requirement for KIF7, the murine COS2 orthologue, is conserved<sup>101,106,107</sup>. Like its *D. melanogaster* counterpart,

KIF7 can physically interact with GLI proteins, and its motor activity seems to be essential for the transport of GLI3 to the tip of the primary cilium in response to HH stimulation<sup>101,107</sup>. In the absence of HH signalling, KIF7 predominantly localizes to the basal body of the primary cilium<sup>101,107</sup>, a structure that is enriched in proteasomes<sup>108,109</sup> and PKA<sup>110</sup>. Thus, like COS2 in *D. melanogaster*, KIF7 may control the processing of GLI proteins by regulating their intracellular localization.

Recent studies have shown that the primary cilium is also required for HH signalling in zebrafish<sup>111–113</sup>, suggesting that it is a conserved feature of the pathway throughout the vertebrates. By contrast, disruption of ciliogenesis has no discernible effect on HH signalling in *Planaria*<sup>95,96,113</sup>, implying that the association may be vertebrate specific. Intriguingly, although FU is dispensable for HH signalling in planarians and mice, it is essential for motile ciliogenesis in both species<sup>95,104</sup>, as is KIF27, the closest planarian homologue of COS2 (KIF7). These findings suggest that the association between HH signalling and ciliogenesis may be ancestral, with cilia-associated proteins, such as FU and KIF7, having been co-opted into the HH pathway at an early stage in its evolution. In this view, the function of cilia-associated genes has been retained by the HH pathway in *D. melanogaster* despite the loss of primary cilia; in *Planaria* their activities have somehow been dispensed with for HH pathway activity, but retained for ciliogenesis. Clearly, a comprehensive understanding of the evolution of the pathway requires functional analyses in a wider range of species than is currently available. However, several ideas about its origins have been proposed (BOX 2).

**Box 1 | Smoothened: sequence conservation and divergence across phyla**

Positioned at the nexus of the extracellular and intracellular sections of the Hedgehog (HH) pathway, Smoothened (SMO) holds the key to a comprehensive understanding of the mode of pathway action. Inhibition of SMO activity by Patched (PTC) and the modulation of its subcellular localization in response to HH activity are common themes across species, but the mechanisms and importance of these processes remain poorly understood, as does the means by which SMO engages with the intracellular HH signalling complex.

Sequence comparisons and structure–function analyses provide some clues about SMO function, but also highlight perplexing differences between species (see also Supplementary Information S1 (figure)). SMO is a divergent member of the frizzled (FZ) family of G protein-coupled receptors (GPCRs), sharing with them the amino-terminal extracellular cysteine-rich domain (CRD) that is crucial for binding of secreted WNT proteins by FZ proteins. Although no similar ligand-binding role is known for SMO, the CRD is highly conserved from Cnidaria to humans, and missense mutations in *Drosophila melanogaster* and zebrafish have shown that its integrity is crucial for SMO function<sup>143,144</sup>.

The central region of the protein, containing the seven membrane-spanning domains, also shows a high level of conservation across phyla. Notably, the seventh transmembrane domain (TM7) and flanking regions are encoded by a single exon, the boundaries of which are absolutely conserved from Cnidaria to humans. Just N-terminal to TM6 are several highly conserved residues, which in classical GPCRs are crucial for coupling to G proteins; mutation of one of these, R474, has been shown to inactivate SMO in *D. melanogaster*<sup>143</sup>. There is evidence from functional analyses in *D. melanogaster* that SMO can modulate cyclic AMP levels via interaction with Gα<sup>145</sup>.

By contrast, the intracellular carboxy-terminal domain (CTD) of SMO is highly divergent across phyla. The CTD of *Planaria*, spider and *D. melanogaster* SMO are substantially longer than those in other studied species. In *D. melanogaster*, the last 59 amino acids have been implicated in binding the serine/threonine kinase Fused (FU)<sup>146</sup>; notably there is little conservation of this sequence even between insects and none between *D. melanogaster* and zebrafish, the only other organism in which FU has been reported to function in HH signalling<sup>147</sup>. The involvement of FU in HH signalling in other species awaits further investigation.

Direct interaction between SMO and the Costal 2 (COS2) protein has also been shown to be crucial for HH signalling in *D. melanogaster*, and a COS2-binding site has been identified in the SMO CTD. This sequence shows over 50% conservation within insects and also in the crustacean, *Daphnia pulex*, but is not found in vertebrates, sea urchin, annelids or even spiders, despite the fact that the function of KIF7 (the COS2 orthologue) is conserved, at least in vertebrates. COS2 preferentially binds to phosphorylated SMO<sup>30</sup> and the CTD includes a number of K/R-K/R-X-T/S sites that are phosphorylated by protein kinase A (PKA)<sup>87</sup>, neutralizing electrostatic interactions that are mediated by adjacent clusters of positively charged Arg residues, thereby inducing a conformational change in the CTD<sup>148</sup>. The same electrostatic interactions between clusters of Arg residues have been shown to play an important part in regulating conformational changes in the CTD in vertebrate SMO. However, whereas the PKA sites are conserved in insects, crustaceans and arachnids, they are absent from vertebrate SMO, raising the question of how the conformation is regulated in these species. GPCR kinase 2 (GRK2) has been implicated in HH signalling in both *D. melanogaster* and vertebrates<sup>149,150</sup>, and the SMO CTD contains potential consensus sites for GRK2; however, the functional importance of these has not been tested in vertebrates.

An important feature of vertebrate SMO is its localization to the primary cilium in response to HH activity. A putative localization motif has been identified at the juxtamembrane region of the CTD<sup>151</sup> that is conserved in all known vertebrate SMO. Remarkably, however, this motif is also conserved in *D. melanogaster* and *Planaria*, neither of which requires primary cilia for HH signalling, but it is absent from other invertebrates, including mosquito, beetle, bee, wasp and aphid, raising questions as to its functional importance.

**Diversification of HH function and deployment**

HH signalling has a multitude of functions in embryonic and adult tissues: it acts as a key mediator of fundamental processes, such as cell fate specification, proliferation and patterning, as well tissue morphogenesis and homeostasis — roles that have been extensively reviewed elsewhere<sup>114–116</sup>. We focus on recent findings that shed light on the evolutionary origins of HH deployment and its roles in body segmentation, tissue regeneration and homeostasis.

**Hedgehog and body segmentation in arthropods and annelids.** The segmental subdivision of the *D. melanogaster* embryo along its anteroposterior axis is controlled by a cascade of transcription factor interactions that culminates in the establishment of domains of expression of *hh* and *wingless* (*wg*) (the *D. melanogaster* orthologue of vertebrate *WNT1*, which encodes another secreted signalling protein). The interface between these cell populations defines each parasegmental boundary and their mutual interaction, mediated by the HH and WNT proteins, is crucial for boundary maintenance (FIG. 5Aa). Absence of HH activity results in the loss of *wg* transcription and the breakdown of segmental boundaries; by contrast, derepression of the HH pathway causes *wg* to be misexpressed and segmental boundaries to be duplicated<sup>117,118</sup> (FIG. 5Ab). In addition, HH and WNT pattern the parasegment along its anteroposterior axis,

Box 2 | Origins and evolution of the Hedgehog pathway

The Hedgehog (HH) pathway has a number of unusual, if not unique, features that make speculation about its origins particularly intriguing. Prominent among these are the coupling of the signalling moiety of HH to cholesterol and the close homology between Patched (PTC) and proteins involved in cholesterol homeostasis. For example, PTC has a sterol-sensing domain that it shares with proteins such as HMG-CoA reductase, sterol regulatory element-binding protein cleavage-activating protein (SCAP) and niemann-pick C1 protein (NPC1)<sup>152</sup>. Consideration of these features led Hausmann and colleagues<sup>153</sup> to propose that the HH pathway evolved from a pre-existing system that controlled lipid homeostasis in single-celled eukaryotes, in which Smoothened (SMO) acted as a sensor of extracellular lipid levels and PTC functioned as a transporter of this lipid. A key element of this model is the transcriptional regulation of *Ptc* by SMO; accordingly, activation of SMO by elevated lipid levels would lead to an upregulation of *Ptc* expression, resulting in increased transport of the lipid out of the membrane<sup>153</sup>. In this view, the crucial event in the evolution of the HH pathway would have been the fortuitous fusion of DNA sequences encoding the ‘Hedge’ domain and the ‘Hog’ domain. This fusion would have facilitated the coupling of the SMO-activating lipid to the Hedge domain protein through the processing reaction that is promoted by the Hog domain, thus creating a secreted form of the lipid normally transported by PTC. Also, attachment of the lipid to the Hedge domain would have transformed it from a small molecule that was easily transported by PTC to a large potentially inhibitory molecule that is capable of blocking the membrane pore that is formed by PTC trimers. In this way a reciprocal regulatory interaction between PTC and SMO could have been established, through which inhibition of PTC activity allowed intracellular accumulation of SMO ligand, resulting in SMO activation. Subsequently, a specific protein–protein interaction interface evolved between HH and PTC that became sufficient for their binding, so that the regulation of PTC — but not the secretion and transport of HH — became independent of the lipid moiety<sup>153</sup>.

One assumption of this model is that a SMO-like protein predates the origin of multicellular organisms and, although not stated explicitly, that this SMO prototype probably already regulated GLI activity, presumably via some form of HH signalling complex (HSC). In this regard, it is notable that true GLI orthologues have not been found outside the metazoa and that Cnidaria seem to be the most basal eukaryotes to possess a SMO protein. Notably, whereas the *Amphimedon queenslandica* genome encodes the full complement of HSC proteins — namely, GLI, KIF7, Fused (FU), Suppressor of fused (SUFU), protein kinase A (PKA), glycogen synthase kinase 3 (GSK3) and casein kinase 1 (CKI) — it lacks not only SMO but also PTC and HH<sup>15</sup>. Conversely, this primitive metazoan does possess genes that encode a hedgling protein, as well as Interference hedgehog (IHOG) (and its vertebrate orthologue CDO) and Hedgehog-interacting protein (HHIP), leading to speculation that a functional interaction between these proteins might predate the origin of the HH pathway<sup>38</sup>. In this view, interaction between the cadherin-like hedgling protein and the calmodulin-like IHOG (CDO) protein may have contributed to the direct cell–cell interactions that underpinned the emergence of multicellular organization. Subsequent fusion of the Hedge and Hog domains would have created a secreted form of the IHOG (CDO) ligand, the prototype of the HH protein that subsequently evolved an interaction with PTC.

in part by regulating the expression of ligands for EGFR and NOTCH<sup>19</sup>.

Orthologues of the *hh* and *wg* genes in the flour beetle, *Tribolium castaneum*, exhibit an almost identical pattern of expression in the developing embryo, and functional studies using RNAi have confirmed a conserved role for both genes in the maintenance and patterning of segments in this insect<sup>120</sup>. Moreover, expression studies in species from two other arthropod sub-phyla, the chelicerata *Euscorpius flavicaudis* and the crustacean *Artemia franciscana*, have revealed a similar spatial deployment of HH, implying a conserved role for HH (and WNT) in segmental maintenance and patterning throughout the arthropods<sup>121</sup>.

Like the arthropods, two other bilaterian phyla — the annelids and the vertebrates — are characterized by their segmented body plans. Arthropods, annelids and vertebrates belong to divergent superclades — the Ecdysozoa, Lophotrochozoa and Deuterostomia, respectively — all three of which include phyla that lack overt body segmentation. This could mean that segmentation evolved independently within these three lineages; alternatively, it is possible that Urbilateria, the last common ancestor of bilaterian animals, was segmented and that this condition has subsequently been lost from some phyla. Although the lack of evidence of segmental expression of Hedgehog genes in vertebrates argues against the latter hypothesis, the patterns of expression of the *hh* and *wg* (*WNT1*) genes in the embryo of the annelid *Platynereis dumerilii* display a striking similarity to those of their arthropod counterparts<sup>122</sup>. These recent findings contrast with earlier studies in other annelid species that failed to identify clear homologies in the expression patterns of *wg* and *hh* orthologues<sup>123,124</sup>. However, their importance seems quite compelling in the light of functional manipulations using two small-molecule inhibitors of the HH pathway, cyclopamine and SANT1, which revealed an essential role for HH in maintaining both the expression of *wg* (*wnt1*) and the segmental boundaries in *Platynereis* spp. embryos<sup>122</sup>. Taken together, these findings suggest a common origin of segmentation, at least within the protostomes.

**Hedgehog and regeneration in planarians.** Recent studies of regeneration in the unsegmented flatworm, *Planaria* have uncovered a role for HH signalling in the anteroposterior patterning of these animals that bears striking analogies to its function in annelid and arthropod segmentation. Remarkably, derepression of the HH pathway (achieved by RNAi knock down of PTC activity) causes the suppression of head regeneration following amputation; instead, the posterior trunk fragments form double abdomens<sup>95,96</sup>. Conversely, knock down of HH activity has the opposite effect, causing anterior

REVIEW S

Aa

Ba

Bb hh siRNA

Bc ptc siRNA

D Basal cells (urethelia stem cells)

Stromal cells

Ab Wild type hh ptc

PS S PS PS S PS

P A A P P

Ca

Differentiation

Self-
renewal

Crypt

Proliferation zone

Lumen

BMP

WNT HH

T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T T Tfragment regenerates to lack tail structures and to form duplicate head structures<sup>96</sup> (FIG. 5B). Notably, earlier experiments using a similar RNAi approach had shown a requirement for WNT signalling in the regeneration of the tail: the manipulated animals in this case also formed duplicated head structures<sup>125,126</sup> and, as in the *D. melanogaster* segment, the duplications caused by HH derepression are dependent on WNT activity. Consistent with this, expression of *wntP1* transcription is upregulated in the absence of PTC activity and lost when HH activity is knocked down, implying a role for HH as an activator of *wntP1* expression<sup>96</sup>. Intriguingly, despite the lack of conservation of this HH–WNT interaction in vertebrate segmentation, the role of WNT in promoting posterior identity in *Planaria* recalls its function as a posteriorizing signal in vertebrate embryos<sup>127</sup>.

**Hedgehog and the development of the endoderm.** Extensive analyses of the role of HH signalling in patterning the neural tube has highlighted the notochord as a major source of SHH in vertebrates<sup>128</sup>, but a second prominent site of Hedgehog gene expression in vertebrate embryos is the endoderm. Notably, both *Shh* and *Ihh* are expressed in overlapping patterns along the developing gastrointestinal tract, prompting suggestions that this represents an ancestral site of HH function<sup>129</sup>. In line with this, the patterns of expression of the Hedgehog genes and their target genes *Ptc* and *Gli* in the embryo of the sea anenome, *Nematostella vectensis*, implicate the pathway in the development of the Cnidarian gut<sup>16</sup>. Expression of one of the two *Nematostella* Hedgehog genes, *NvHh1*, initiates early in gastrulation in the pharyngeal and mesenteric ectoderm, whereas *Ptc* and *Gli* are expressed in the flanking pharyngeal and oral body wall endoderm, implying that HH mediates interactions between the ectoderm and the endoderm<sup>16</sup>. The embryo of the basal deuterostome, the sea urchin *Lytechinus variegatus*, shows a similarly complementary pattern of *Hh* and *Ptc* expression in its developing gut, though in this case *Hh* is expressed in the endoderm and *Ptc* in the adjacent mesoderm<sup>130,131</sup>. Functional studies have revealed a widespread requirement for endodermally derived HH in this species; inactivation of the pathway causes defects in mesodermal derivatives, including disruption of the skeleton and disorganization of the muscles that surround the oesophagus<sup>131</sup>. In vertebrates, reciprocal interactions between the endoderm and the overlying mesoderm have essential roles in the specification and differentiation of these layers<sup>125,126</sup>. As in the sea urchin, expression of Hedgehog genes in the endodermal epithelium is accompanied by *Ptc1* expression in the adjacent mesoderm<sup>129</sup>, indicative of its response to HH signalling, which promotes its proliferation and differentiation into smooth muscle<sup>132</sup>.

**HH in endodermal tissue homeostasis and regeneration.** In vertebrates, the intestinal epithelium is renewed throughout adult life, with mature enterocytes replaced by cells that are derived from stem cells in invaginations of the epithelium, known as crypts. These cells express genes that encode several WNT proteins and their receptors, the functions of which are essential to maintain the stem cell population. Strikingly, expression of both IHH and SHH become restricted to the crypts as they develop. In contrast to the WNTs, however, these proteins signal exclusively to the surrounding mesoderm to induce the expression of BMP proteins, which signal back to the endodermal epithelium to inhibit crypt formation<sup>132,133</sup>. In this way, HH signalling restricts the stem cell population to the crypts and promotes the differentiation of their progeny as they exit the crypt<sup>134</sup> (FIG. 5Ca). Strikingly, HH and WNT signalling is also deployed in *D. melanogaster* hindgut homeostasis: as in the vertebrate crypts, expression of a WNT protein, in this case WG, is restricted to the stem cell niche at the anterior end of the hindgut, where it promotes stem cell proliferation and self-renewal<sup>135</sup>. As cells move away from the niche they undergo rapid proliferation before entering a zone in which they activate HH expression, and this promotes their exit from the cell cycle and differentiation<sup>135</sup> (FIG. 5Cb).

Recent studies have also implicated HH signalling in driving regenerative proliferation in a mitotically quiescent endodermal tissue — the bladder urothelium — in mammals<sup>136</sup>. SHH is expressed in stem cells located in the basal layer of the urothelium that proliferate and regenerate damaged tissue in response to infection. Interestingly, the proliferative response requires WNT signals that emanate from the adjacent stromal layer, expression of which is induced by the stem cell-derived SHH (FIG. 5D).

A common theme in most of these instances is the involvement of HH signalling in mediating reciprocal interactions between adjacent cell populations, either between different lineage compartments derived from the same germ layer or between tissues that originate from different germ layers. Remarkably, the positive feedback loop between HH and WNT signalling, which was first identified in the maintenance of the parasegment boundary in *D. melanogaster*, seems to be similar to the regenerative response of the mammalian bladder. It will be interesting to determine whether a similar regulatory relationship exists in Cnidaria, a phylum that diverged from the bilateria more than 650 million years ago.

### Concluding comments

A great deal has been learned about HH signalling in the past two decades, but much remains to be discovered. Key unresolved questions include the molecular basis of SMO repression by PTC and how the subcellular localization of SMO is regulated in response to PTC activity. The mechanism or mechanisms of action of SMO in promoting the dissociation of SUFU from GLI proteins, the nature of the modifications required for maximal activation of GLI proteins, and the precise role of the cilia in regulating GLI processing and activation also remain to be elucidated. Nevertheless, the analysis of the roles of HH in different model organisms has shed light on the molecular and cellular mechanisms underlying a plethora of processes, from the segmentation of the bodies of arthropods and annelids to the

---

**NATURE REVIEWS | GENETICS**

© 2011 Macmillan Publishers Limited. All rights reserved

VOLUME 12 | JUNE 2011 | 403
specification of vertebrate neuronal identity. This knowledge has already provided us with new insights into the molecular basis of morphological diversity, highlighting variation in HH signalling as a key factor in both micro-evolution and macro-evolution. Developmental analyses of the blind cave-dwelling variety of the teleost *Astyanax mexicanus*, for example, have pointed to an expansion of *Shh* expression in the ventral brain as a major driver of the anatomical adaptations that distinguish this variety from its surface river-dwelling relatives<sup>137</sup>; and at a macro-evolutionary level, the acquisition of a forebrain enhancer by the *Shh* gene seems to have played a crucial part in the emergence of the vertebrates<sup>138,139</sup>. Similarly, changes in the transcriptional regulation of *hh* in the primordial wings of butterflies have been implicated

in the development of their characteristic eyespots<sup>140</sup>, a morphological adaptation that has a key role in predator avoidance.

Concurrent with these advances, the elucidation of the pathway components and their involvement in human disease has provided a major stimulus for the discovery of small-molecule antagonists (and agonists)<sup>141</sup>, providing new tools for experimental manipulation of the pathway and new leads for the development of novel targeted therapeutic agents<sup>142</sup>. The application of these reagents to the analysis of HH function in a wider range of species, together with increasing knowledge of the interspecies variation in HH pathway gene structure, is set to provide further insights into the contribution of this fascinating pathway to metazoan evolution.

---

1. Ingham, P. W. Transducing Hedgehog: the story so far. *EMBO J.* **17**, 3505–3511 (1998).
2. Hao, L., Johnsen, R., Lauter, G., Baillie, D. & Burglin, T. R. Comprehensive analysis of gene expression patterns of hedgehog-related genes. *BMC Genomics* **7**, 280 (2006).
3. Beachy, P. A. *et al.* Multiple roles of cholesterol in hedgehog protein biogenesis and signaling. *Cold Spring Harb. Symp. Quant. Biol.* **62**, 191–204 (1997).
4. Ingham, P. W., Taylor, A. M. & Nakano, Y. Role of the *Drosophila patched* gene in positional signalling. *Nature* **353**, 184–187 (1991).
5. Marigo, V., Davey, R. A., Zuo, Y., Cunningham, J. M. & Tabin, C. J. Biochemical evidence that Patched is the Hedgehog receptor. *Nature* **384**, 176–179 (1996).
6. Zheng, X., Mann, R. K., Sever, N. & Beachy, P. A. Genetic and biochemical definition of the Hedgehog receptor. *Genes Dev.* **24**, 57–71 (2011).
7. Ayers, K. L. & Therond, P. P. Evaluating Smoothened as a G-protein-coupled receptor for Hedgehog signalling. *Trends Cell Biol.* **20**, 287–298 (2010).
8. Lee, J. J. *et al.* Autoproteolysis in hedgehog protein biogenesis. *Science* **266**, 1528–1537 (1994).
9. Bürglin, T. R. The Hedgehog protein family. *Genome Biol.* **9**, 241 (2008).
10. Koonin, E. V. A protein splice-junction motif in hedgehog family proteins. *Trends Biochem. Sci.* **20**, 141–142 (1995).
11. Bürglin, T. R. Evolution of *hedgehog* and *hedgehog*-related genes, their origin from Hog proteins in ancestral eukaryotes and discovery of a novel Hint motif. *BMC Genomics* **9**, 127 (2008).
12. Snell, E. A. *et al.* An unusual choanoflagellate protein released by Hedgehog autocatalytic processing. *Proc. Biol. Sci.* **273**, 401–407 (2006).
13. Fuse, N. *et al.* Sonic hedgehog protein signals not as a hydrolytic enzyme but as an apparent ligand for patched. *Proc. Natl Acad. Sci. USA* **96**, 10992–10999 (1999).
14. King, N. *et al.* The genome of the choanoflagellate *Monosiga brevicollis* and the origin of metazoans. *Nature* **451**, 783–788 (2008).
15. Adamska, M. *et al.* The evolutionary origin of hedgehog proteins. *Curr. Biol.* **17**, R836–R837 (2007).
16. Matus, D. Q., Magie, C. R., Pang, K., Martindale, M. Q. & Thomsen, G. H. The Hedgehog gene family of the cnidarian, *Nematostella vectensis*, and implications for understanding metazoan Hedgehog pathway evolution. *Dev. Biol.* **313**, 501–518 (2008).
17. Avaron, F., Hoffman, L., Guay, D. & Akimenko, M. A. Characterization of two new zebrafish members of the hedgehog family: atypical expression of a zebrafish *indian hedgehog* gene in skeletal elements of both endochondral and dermal origins. *Dev. Dyn.* **235**, 478–489 (2006).
18. Chamoun, Z. *et al.* Skinny hedgehog, an acyltransferase required for palmitoylation and activity of the hedgehog signal. *Science* **293**, 2080–2084 (2001).
19. Chen, M. H., Li, Y. J., Kawakami, T., Xu, S. M. & Chuang, P. T. Palmitoylation is required for the production of a soluble multimeric Hedgehog protein complex and long-range signaling in vertebrates. *Genes Dev.* **18**, 641–659 (2004).

20. Peters, C., Wolf, A., Wagner, M., Kuhlmann, J. & Waldmann, H. The cholesterol membrane anchor of the Hedgehog protein confers stable membrane association to lipid-modified proteins. *Proc. Natl Acad. Sci. USA* **101**, 8531–8536 (2004).
21. Burke, R. *et al.* Dispatched, a novel sterol-sensing domain protein dedicated to the release of cholesterol-modified hedgehog from signaling cells. *Cell* **99**, 803–815 (1999).
22. Caspari, T. *et al.* Mouse Dispatched homolog1 is required for long-range, but not juxtacrine, Hh signaling. *Curr. Biol.* **12**, 1628–1632 (2002).
23. Kawakami, T. *et al.* Mouse dispatched mutants fail to distribute hedgehog proteins and are defective in hedgehog signaling. *Development* **129**, 5753–5765 (2002).
24. Ma, Y. *et al.* Hedgehog-mediated patterning of the mammalian embryo requires transporter-like function of dispatched. *Cell* **111**, 63–75 (2002).
25. Lewis, P. M. *et al.* Cholesterol modification of sonic hedgehog is required for long-range signaling activity and effective modulation of signaling by Ptc1. *Cell* **105**, 599–612 (2001).
26. Gallet, A., Ruel, L., Staccini-Lavenant, L. & Therond, P. P. Cholesterol modification is necessary for controlled planar long-range activity of Hedgehog in *Drosophila* epithelia. *Development* **133**, 407–418 (2006).
27. Zeng, X. *et al.* A freely diffusible form of Sonic hedgehog mediates long-range signalling. *Nature* **411**, 716–720 (2001).
28. Eugster, C., Panakova, D., Mahmoud, A. & Eaton, S. Lipoprotein-heparan sulfate interactions in the Hh pathway. *Dev. Cell* **13**, 57–71 (2007).
29. Panakova, D., Sprong, H., Marois, E., Thiele, C. & Eaton, S. Lipoprotein particles are required for Hedgehog and Wingless signalling. *Nature* **435**, 58–65 (2005).
30. Lum, L. *et al.* Hedgehog signal transduction via Smoothened association with a cytoplasmic complex scaffolded by the atypical kinesin, Costal-2. *Mol. Cell* **12**, 1261–1274 (2003).
31. Vyas, N. *et al.* Nanoscale organization of hedgehog is essential for long-range signaling. *Cell* **133**, 1214–1227 (2008).
32. Ayers, K. L., Gallet, A., Staccini-Lavenant, L. & Therond, P. P. The long-range activity of Hedgehog is regulated in the apical extracellular space by the glypican Dally and the hydrolase Notum. *Dev. Cell* **18**, 605–620 (2010). Elegant genetic analysis implicating apically secreted HH in long-range signalling in the *D. melanogaster* imaginal disc and the roles of Dally and notum in promoting its release.
33. Williams, E. H. *et al.* Dally-like core protein and its mammalian homologues mediate stimulatory and inhibitory effects on Hedgehog signal response. *Proc. Natl Acad. Sci. USA* **107**, 5869–5874 (2010).
34. Su, V. F., Jones, K. A., Brodsky, M. & The, I. Quantitative analysis of Hedgehog gradient formation using an inducible expression system. *BMC Dev. Biol.* **7**, 43 (2007).
35. Callejo, A., Torroja, C., Quijada, L. & Guerrero, I. Hedgehog lipid modifications are required for Hedgehog stabilization in the extracellular matrix. *Development* **133**, 471–483 (2006).

36. Chamberlain, C. E., Jeong, J., Guo, C., Allen, B. L. & McMahon, A. P. Notochord-derived Shh concentrates in close association with the apically positioned basal body in neural target cells and forms a dynamic gradient during neural patterning. *Development* **135**, 1097–1106 (2008).
37. Hartman, T. R. *et al.* *Drosophila Boi* limits Hedgehog levels to suppress follicle stem cell proliferation. *J. Cell Biol.* **191**, 943–952 (2010).
38. Richards, G. S. & Degnan, B. M. The dawn of developmental signaling in the metazoa. *Cold Spring Harb. Symp. Quant. Biol.* **74**, 81–90 (2009).
39. Bishop, B. *et al.* Structural insights into hedgehog ligand sequestration by the human hedgehog-interacting protein HHIP. *Nature Struct. Mol. Biol.* **16**, 698–703 (2009).
40. Yam, P. T., Langlois, S. D., Morin, S. & Charron, F. Sonic hedgehog guides axons through a noncanonical, Src-family-kinase-dependent signaling pathway. *Neuron* **62**, 349–362 (2009).
41. Dai, P. *et al.* Sonic Hedgehog-induced activation of the Gli1 promoter is mediated by GLI3. *J. Biol. Chem.* **274**, 8143–8152 (1999).
42. Gustafsson, M. K. *et al.* Myf5 is a direct target of long-range Shh signaling and Gli regulation for muscle specification. *Genes Dev.* **16**, 114–126 (2002).
43. Sasaki, H., Hui, C., Nakafuku, M. & Kondoh, H. A binding site for Gli proteins is essential for *HNF-3β* floor plate enhancer activity in transgenics and can respond to Shh *in vitro*. *Development* **124**, 1313–1322 (1997).
44. Alexandre, C., Jacinto, A. & Ingham, P. W. Transcriptional activation of hedgehog target genes in *Drosophila* is mediated directly by the cubitus interruptus protein, a member of the GLI family of zinc finger DNA-binding proteins. *Genes Dev.* **10**, 2003–2013 (1996).
45. Vokes, S. A., Ji, H., Wong, W. H. & McMahon, A. P. A genome-scale analysis of the cis-regulatory circuitry underlying sonic hedgehog-mediated patterning of the mammalian limb. *Genes Dev.* **22**, 2651–2663 (2008).
46. Vokes, S. A. *et al.* The first genome-scale identification of HH and GLI target genes using chromatin immunoprecipitation-chip analysis in the mouse limb.
47. Vokes, S. A. *et al.* Genomic characterization of Gli-activator targets in sonic hedgehog-mediated neural patterning. *Development* **134**, 1977–1989 (2007).
48. Hallikas, O. *et al.* Genome-wide prediction of mammalian enhancers based on analysis of transcription-factor binding affinity. *Cell* **124**, 47–59 (2006).
49. Goodrich, L. V., Johnson, R. L., Milenkovic, L., McMahon, J. A. & Scott, M. P. Conservation of the hedgehog/patched signaling pathway from flies to mice: induction of a mouse *patched* gene by Hedgehog. *Genes Dev.* **10**, 301–312 (1996).
50. Duman-Scheel, M., Weng, L., Xin, S. & Du, W. Hedgehog regulates cell growth and proliferation by inducing Cyclin D and Cyclin E. *Nature* **417**, 299–304 (2002).
51. Kenney, A. M. & Rowitch, D. H. Sonic hedgehog promotes G<sub>1</sub> cyclin expression and sustained cell cycle progression in mammalian neuronal precursors. *Mol. Cell. Biol.* **20**, 9055–9067 (2000).

51. Sasaki, H., Nishizaki, Y., Hui, C., Nakafuku, M. & Kondoh, H. Regulation of Gli2 and Gli3 activities by an amino-terminal repression domain: implication of Gli2 and Gli3 as primary mediators of Shh signaling. *Development* **126**, 3915–3924 (1999).

52. Srivastava, M. *et al.* The *Amphimedon queenslandica* genome and the evolution of animal complexity. *Nature* **466**, 720–726 (2010).

53. Aza-Blanc, P., Ramirez-Weber, F. A., Laget, M. P., Schwartz, C. & Kornberg, T. B. Proteolysis that is inhibited by hedgehog targets Cubitus interruptus protein to the nucleus and converts it to a repressor. *Cell* **89**, 1043–1053 (1997).

54. Bai, C. B., Stephen, D. & Joyner, A. L. All mouse ventral spinal cord patterning by hedgehog is Gli dependent and involves an activator function of Gli3. *Dev. Cell* **6**, 103–115 (2004).

55. Varga, Z. M. *et al.* Zebrafish *smoothened* functions in ventral neural tube specification and axon tract formation. *Development* **128**, 3497–3509 (2001).

56. Alcedo, J., Ayzenzon, M., Von Ohlen, T., Noll, M. & Hooper, J. E. The *Drosophila smoothened* gene encodes a seven-pass membrane protein, a putative receptor for the hedgehog signal. *Cell* **86**, 221–232 (1996).

57. van den Heuvel, M. & Ingham, P. W. *smoothened* encodes a receptor-like serpentine protein required for hedgehog signalling. *Nature* **382**, 547–551 (1996).

58. Chen, W., Burgess, S. & Hopkins, N. Analysis of the zebrafish *smoothened* mutant reveals conserved and divergent functions of hedgehog activity. *Development* **128**, 2385–2396 (2001).

59. Zhang, X. M., Ramalho-Santos, M. & McMahon, A. P. *Smoothened* mutants reveal redundant roles for Shh and Ihh signaling including regulation of L/R symmetry by the mouse node. *Cell* **106**, 781–792 (2001).

60. Beachy, P. A., Hymowitz, S. G., Lazarus, R. A., Leahy, D. J. & Siebold, C. Interactions between Hedgehog proteins and their binding partners come into view. *Genes Dev.* **24**, 2001–2012 (2010).

61. Tenzen, T. *et al.* The cell surface membrane proteins Cdo and Boc are components and targets of the Hedgehog signaling pathway and feedback network in mice. *Dev. Cell* **10**, 647–656 (2006).

62. Srivastava, M. *et al.* The *Trichoplax* genome and the nature of placozoans. *Nature* **454**, 955–960 (2008).

63. McLellan, J. S. *et al.* The mode of Hedgehog binding to Ihog homologues is not conserved across different phyla. *Nature* **455**, 979–983 (2008).

64. Murone, M., Rosenthal, A. & de Sauvage, F. J. Sonic hedgehog signaling by the patched-smoothened receptor complex. *Curr. Biol.* **9**, 76–84 (1999).

65. Taipale, J., Cooper, M. K., Maiti, T. & Beachy, P. A. Patched acts catalytically to suppress the activity of Smoothened. *Nature* **418**, 892–897 (2002).

66. Tseng, T. T. *et al.* The RND permease superfamily: an ancient, ubiquitous and diverse family that includes human disease and development proteins. *J. Mol. Microbiol. Biotechnol.* **1**, 107–125 (1999).

67. Chen, J. K., Taipale, J., Cooper, M. K. & Beachy, P. A. Inhibition of Hedgehog signaling by direct binding of cyclopamine to Smoothened. *Genes Dev.* **16**, 2743–2748 (2002).

68. Yavari, A. *et al.* Role of lipid metabolism in smoothened derepression in hedgehog signaling. *Dev. Cell* **19**, 54–65 (2010).

69. Price, M. A. & Kalderon, D. Proteolysis of the Hedgehog signaling effector Cubitus interruptus requires phosphorylation by Glycogen Synthase Kinase 3 and Casein Kinase 1. *Cell* **108**, 823–835 (2002).

70. Wang, B., Fallon, J. F. & Beachy, P. A. Hedgehog-regulated processing of Gli3 produces an anterior/posterior repressor gradient in the developing vertebrate limb. *Cell* **100**, 423–434 (2000).

71. Pan, Y., Wang, C. & Wang, B. Phosphorylation of Gli2 by protein kinase A is required for Gli2 processing and degradation and the Sonic Hedgehog-regulated mouse development. *Dev. Biol.* **326**, 177–189 (2009).

72. Jia, J. *et al.* Phosphorylation by double-time/CKIepsilon and CKIalpha targets cubitus interruptus for Slimb/beta-TRCP-mediated proteolytic processing. *Dev. Cell* **9**, 819–830 (2005).

73. Tempe, D., Casas, M., Karaz, S., Blanchet-Tournier, M. F. & Concordet, J. P. Multisite protein kinase A and glycogen synthase kinase 3beta phosphorylation leads to Gli3 ubiquitination by SCFbetaTrCP. *Mol. Cell. Biol.* **26**, 4316–4326 (2006).

74. Forbes, A. J., Nakano, Y., Taylor, A. M. & Ingham, P. W. Genetic analysis of hedgehog signalling in the *Drosophila* embryo. *Dev. Suppl.* **115–124** (1993).

75. Robbins, D. J. *et al.* Hedgehog elicits signal transduction by means of a large complex containing the kinesin-related protein costal2. *Cell* **90**, 225–234 (1997).

76. Sisson, J. C., Ho, K. S., Suyama, K. & Scott, M. P. Costal2, a novel kinesin-related protein in the Hedgehog signaling pathway. *Cell* **90**, 235–245 (1997).

77. Kollmar, M. & Glockner, G. Identification and phylogenetic analysis of *Dictyostelium discoideum* kinesin proteins. *BMC Genomics* **4**, 47 (2003).

78. Farzan, S. F. *et al.* Costal2 functions as a kinesin-like protein in the hedgehog signal transduction pathway. *Curr. Biol.* **18**, 1215–1220 (2008).

79. Liu, Y., Cao, X., Jiang, J. & Jia, J. Fused-Costal2 protein complex regulates Hedgehog-induced Smo phosphorylation and cell-surface accumulation. *Genes Dev.* **21**, 1949–1963 (2007).

80. Stegman, M. A. *et al.* Identification of a tetrameric hedgehog signaling complex. *J. Biol. Chem.* **275**, 21809–21812 (2000).

81. Zhang, W. *et al.* Hedgehog-regulated Costal2-kinase complexes control phosphorylation and proteolytic processing of Cubitus interruptus. *Dev. Cell* **8**, 267–278 (2005).

82. Oh, S. A. *et al.* A divergent cellular role for the FUSED kinase family in the plant-specific cytokinetic phragmoplast. *Curr. Biol.* **15**, 2107–2111 (2005).

83. Tang, L. *et al.* *tsunami*, the *Dictyostelium* homolog of the Fused kinase, is required for polarization and chemotaxis. *Genes Dev.* **22**, 2278–2290 (2008).

84. Ruel, L. *et al.* Phosphorylation of the atypical kinesin Costal2 by the kinase Fused induces the partial disassembly of the Smoothened-Fused-Costal2-Cubitus interruptus complex in Hedgehog signalling. *Development* **134**, 3677–3689 (2007).

85. Das, D. *et al.* The crystal structure of a bacterial Sufu-like protein defines a novel group of bacterial proteins that are similar to the N-terminal domain of human Sufu. *Protein Sci.* **19**, 2131–2140 (2010).

86. Ruel, L., Rodriguez, R., Gallet, A., Lavenant-Staccini, L. & Therond, P. P. Stability and association of Smoothened, Costal2 and Fused with Cubitus interruptus are regulated by Hedgehog. *Nature Cell Biol.* **5**, 907–913 (2003).

87. Apionishev, S., Katanayeva, N. M., Marks, S. A., Kalderon, D. & Tomlinson, A. *Drosophila* Smoothened phosphorylation sites essential for Hedgehog signal transduction. *Nature Cell Biol.* **7**, 86–92 (2005).

88. Wang, G., Amanai, K., Wang, B. & Jiang, J. Interactions with Costal2 and suppressor of fused regulate nuclear translocation and activity of cubitus interruptus. *Genes Dev.* **14**, 2893–2905 (2000).

89. Methot, N. & Basler, K. Suppressor of fused opposes hedgehog signal transduction by impeding nuclear accumulation of the activator form of Cubitus interruptus. *Development* **127**, 4001–4010 (2000).

90. Svard, J. *et al.* Genetic elimination of Suppressor of fused reveals an essential repressor function in the mammalian Hedgehog signaling pathway. *Dev. Cell* **10**, 187–197 (2006).

91. Cooper, A. F. *et al.* Cardiac and CNS defects in a mouse with targeted disruption of suppressor of fused. *Development* **132**, 4407–4417 (2005). This paper, together with reference 90, describes the generation of targeted mutant alleles of the mouse *Sufu* gene and the surprising finding that SUFU has an essential role in negatively regulating the HH pathway in mammals.

92. Goodrich, L. V., Milenkovic, L., Higgins, K. M. & Scott, M. P. Altered neural cell fates and medulloblastoma in mouse patched mutants. *Science* **277**, 1109–1113 (1997).

93. Humke, E. W., Dorn, K. V., Milenkovic, L., Scott, M. P. & Rohatgi, R. The output of Hedgehog signaling is controlled by the dynamic association between Suppressor of Fused and the Gli proteins. *Genes Dev.* **24**, 670–682 (2010).

94. Kise, Y., Morinaka, A., Teglund, S. & Miki, H. Sufu recruits GSK3beta for efficient processing of Gli3. *Biochem. Biophys. Res. Commun.* **387**, 569–574 (2009).

95. Rink, J. C., Gurley, K. A., Elliott, S. A. & Sanchez Alvarado, A. Planarian Hh signaling regulates regeneration polarity and links Hh pathway evolution to cilia. *Science* **326**, 1406–1410 (2009).

96. Yazawa, S., Umesono, Y., Hayashi, T., Tarui, H. & Agata, K. Planarian Hedgehog/Patched establishes anterior-posterior polarity by regulating Wnt signaling. *Proc. Natl Acad. Sci. USA* **106**, 22329–22334 (2009). This paper and reference 95 describe the identification of HH pathway components in planarians and the first functional analysis of their role in regeneration. Reference 95 also presents evidence that HH signalling is independent of cilia in planarians.

97. Goetz, S. C. & Anderson, K. V. The primary cilium: a signalling centre during vertebrate development. *Nature Rev. Genet.* **11**, 331–344 (2010).

98. Han, Y. G., Kwok, B. H. & Kernan, M. J. Intraflagellar transport is required in *Drosophila* to differentiate sensory cilia but not sperm. *Curr. Biol.* **13**, 1679–1686 (2003).

99. Rohatgi, R., Milenkovic, L. & Scott, M. P. Patched1 regulates hedgehog signaling at the primary cilium. *Science* **317**, 372–376 (2007). An elegant study describing the generation of high-quality antibodies recognizing the mammalian SMO and PTC1 proteins and their use in studying the mutually exclusive patterns of ciliary localization of PTC1 and SMO in response to SHH activity.

100. Haycraft, C. J. *et al.* Gli2 and Gli3 localize to cilia and require the intraflagellar transport protein polaris for processing and function. *PLoS Genet.* **1**, e53 (2005).

101. Endoh-Yamagami, S. *et al.* The mammalian Cos2 homolog Kif7 plays an essential role in modulating Hh signal transduction during development. *Curr. Biol.* **19**, 1320–1326 (2009). Generation of a targeted mutation in the mouse COS2 orthologue, KIF7, demonstrating the previously disputed functional conservation of the protein in HH signalling from *D. melanogaster* to mouse and its role in regulating GLI3 trafficking in the primary cilium.

102. Tukachinsky, H., Lopez, L. V. & Salic, A. A mechanism for vertebrate Hedgehog signaling: recruitment to cilia and dissociation of SuFu-Gli protein complexes. *J. Cell Biol.* **191**, 415–428 (2010). Careful analysis of the dynamics of SUFU and GLI localization and the role of SMO in promoting their dissociation in the primary cilium in response to SHH.

103. Merchant, M. *et al.* Loss of the serine/threonine kinase fused results in postnatal growth defects and lethality due to progressive hydrocephalus. *Mol. Cell. Biol.* **25**, 7054–7068 (2005).

104. Wilson, C. W. *et al.* Fused has evolved divergent roles in vertebrate Hedgehog signalling and motile ciliogenesis. *Nature* **459**, 98–102 (2009).

105. Evangelista, M. *et al.* Kinome siRNA screen identifies regulators of ciliogenesis and hedgehog signal transduction. *Sci. Signal.* **1**, ra7 (2008).

106. Cheung, H. O. *et al.* The kinesin protein Kif7 is a critical regulator of Gli transcription factors in mammalian hedgehog signaling. *Sci. Signal.* **2**, ra29 (2009).

107. Liem, K. F., Jr, He, M., Ocbina, P. J. & Anderson, K. V. Mouse Kif7/Costal2 is a cilias-associated protein that regulates Sonic hedgehog signaling. *Proc. Natl Acad. Sci. USA* **106**, 13377–13382 (2009). Isolation of a chemically induced missense mutation in the mouse COS2 orthologue, KIF7, demonstrating the previously disputed functional conservation of the protein in HH signalling from *D. melanogaster* to mouse and the role of its motor domain in trafficking in the primary cilium.

108. Doxsey, S., Zimmerman, W. & Mikule, K. Centrosome control of the cell cycle. *Trends Cell Biol.* **15**, 303–311 (2005).

109. Wigley, W. C. *et al.* Dynamic association of proteasomal machinery with the centrosome. *J. Cell Biol.* **145**, 481–490 (1999).

110. Barzi, M., Berenguier, J., Menendez, A., Alvarez-Rodriguez, R. & Pons, S. Sonic-hedgehog-mediated proliferation requires the localization of PKA to the cilium base. *J. Cell Sci.* **123**, 62–69 (2010).

111. Kim, H. R., Richardson, J., van Eeden, F. & Ingham, P. W. Gli2a protein localization reveals a role for Iguana/DZIP1 in primary ciliogenesis and a dependence of Hedgehog signal transduction on primary cilia in the zebrafish. *BMC Biol.* **8**, 65 (2010).

112. Tay, S. Y. *et al.* The iguana/DZIP1 protein is a novel component of the ciliogenic pathway essential for axonemal biogenesis. *Dev. Dyn.* **239**, 527–534 (2010).

REVIEWS

113. Glazer, A. M. *et al.* The Zn finger protein Iguana impacts Hedgehog signaling by promoting ciliogenesis. *Dev. Biol.* **337**, 148–156 (2010).

114. Ingham, P. W. & Placzek, M. Orchestrating ontogenesis: variations on a theme by sonic hedgehog. *Nature Rev. Genet.* **7**, 841–850 (2006).

115. McMahon, A. P., Ingham, P. W. & Tabin, C. J. Developmental roles and clinical significance of hedgehog signaling. *Curr. Top. Dev. Biol.* **53**, 1–114 (2003).

116. Taipale, J. & Beachy, P. A. The Hedgehog and Wnt signalling pathways in cancer. *Nature* **411**, 349–354 (2001).

117. Ingham, P. W. & Hidalgo, A. Regulation of wingless transcription in the *Drosophila* embryo. *Development* **117**, 283–291 (1993).

118. Ingham, P. W. Localized *hedgehog* activity controls spatial limits of *wingless* transcription in the *Drosophila* embryo. *Nature* **366**, 560–562 (1993).

119. Alexandre, C., Lecourtois, M. & Vincent, J. Wingless and Hedgehog pattern *Drosophila* denticle belts by regulating the production of short-range signals. *Development* **126**, 5689–5698 (1999).

120. Farzana, L. & Brown, S. J. Hedgehog signaling pathway function conserved in *Tribolium* segmentation. *Dev. Genes Evol.* **218**, 181–192 (2008).

121. Simonnet, F., Deutsch, J. & Queinnec, E. *hedgehog* is a segment polarity gene in a crustacean and a chelicerate. *Dev. Genes Evol.* **214**, 537–545 (2004).

122. Dray, N. *et al.* Hedgehog signaling regulates segment formation in the annelid *Platynereis*. *Science* **329**, 339–342 (2010). Functional analysis of HH signalling in this annelid species providing the first evidence that the role of HH signalling in segmentation is conserved across phyla.

123. Kang, D. *et al.* A hedgehog homolog regulates gut formation in leech (*Helobdella*). *Development* **130**, 1645–1657 (2003).

124. Seaver, E. C. & Kaneshige, L. M. Expression of ‘segmentation’ genes during larval and juvenile development in the polychaetes *Capitella sp.* I and *H. elegans*. *Dev. Biol.* **289**, 179–194 (2006).

125. Petersen, C. P. & Reddien, P. W. Smed-βcatenin-1 is required for anteroposterior blastema polarity in planarian regeneration. *Science* **319**, 327–330 (2008).

126. Petersen, C. P. & Reddien, P. W. A wound-induced Wnt expression program controls planarian regeneration polarity. *Proc. Natl Acad. Sci. USA* **106**, 17061–17066 (2009).

127. Yamaguchi, T. P. Heads or tails: Wnts and anterior-posterior patterning. *Curr. Biol.* **11**, R713–R724 (2001).

128. Dessaud, E., McMahon, A. P. & Briscoe, J. Pattern formation in the vertebrate neural tube: a sonic hedgehog morphogen-regulated transcriptional network. *Development* **135**, 2489–2503 (2008).

129. Ramalho-Santos, M., Melton, D. A. & McMahon, A. P. Hedgehog signals regulate multiple aspects of gastrointestinal development. *Development* **127**, 2763–2772 (2000).

130. Walton, K. D., Croce, J. C., Glenn, T. D., Wu, S. Y. & McClay, D. R. Genomics and expression profiles of the Hedgehog and Notch signaling pathways in sea urchin development. *Dev. Biol.* **300**, 153–164 (2006).

131. Walton, K. D., Warner, J., Hertzler, P. H. & McClay, D. R. Hedgehog signaling patterns mesoderm in the sea urchin. *Dev. Biol.* **331**, 26–37 (2009).

132. Mao, J., Kim, B. M., Rajurkar, M., Shivdasani, R. A. & McMahon, A. P. Hedgehog signaling controls mesenchymal growth in the developing mammalian digestive tract. *Development* **137**, 1721–1729 (2010).

133. Madison, B. B. *et al.* Epithelial hedgehog signals pattern the intestinal crypt-villus axis. *Development* **132**, 279–289 (2005).

134. Crosnier, C., Stamataki, D. & Lewis, J. Organizing cell renewal in the intestine: stem cells, signals and combinatorial control. *Nature Rev. Genet.* **7**, 349–359 (2006).

135. Takashima, S., Mkrtchyan, M., Younossi-Hartenstein, A., Merriam, J. R. & Hartenstein, V. The behaviour of *Drosophila* adult hindgut stem cells is controlled by Wnt and Hh signalling. *Nature* **454**, 651–655 (2008).

136. Shin, K. *et al.* Hedgehog/Wnt feedback supports regenerative proliferation of epithelial stem cells bladder. *Nature* **427**, 110–114 (2011).

137. Retaux, S., Pottin, K. & Alunni, A. Shh and forebrain evolution in the blind cavefish *Astyanax mexicanus*. *Biol. Cell* **100**, 139–147 (2008).

138. Retaux, S. & Kano, S. Midline signaling and evolution of the forebrain in chordates: a focus on the lamprey hedgehog case. *Integrative Comparative Biol.* **50**, 98–109 (2010).

139. Kano, S. *et al.* Two lamprey *Hedgehog* genes share non-coding regulatory sequences and expression patterns with gnathostome Hedgehogs. *PLoS ONE* **5**, e13332.

140. Keys, D. N. *et al.* Recruitment of a hedgehog regulatory circuit in butterfly eyespot evolution. *Science* **283**, 532–534 (1999).

141. Mas, C. & Ruiz i Altaba, A. Small molecule modulation of HH-GLI signaling: current leads, trials and tribulations. *Biochem. Pharmacol.* **80**, 712–723 (2010).

142. Low, J. A. & de Sauvage, F. J. Clinical experience with Hedgehog pathway inhibitors. *J. Clin. Oncol.* **28**, 5321–5326 (2010).

143. Nakano, Y. *et al.* Functional domains and sub-cellular distribution of the Hedgehog transducing protein Smoothened in *Drosophila*. *Mech. Dev.* **121**, 507–518 (2004).

144. Aanstad, P. *et al.* The extracellular domain of Smoothened regulates ciliary localization and is required for high-level Hh signaling. *Curr. Biol.* **19**, 1034–1039 (2009).

145. Ogden, S. K. *et al.* G protein Gaq functions immediately downstream of Smoothened in Hedgehog signalling. *Nature* **456**, 967–970 (2008).

146. Malpel, S. *et al.* The last 59 amino acids of Smoothened cytoplasmic tail directly bind the protein kinase Fused and negatively regulate the Hedgehog pathway. *Dev. Biol.* **303**, 121–133 (2007).

147. Wolff, C., Roy, S. & Ingham, P. W. Multiple muscle cell identities induced by distinct levels and timing of hedgehog activity in the zebrafish embryo. *Curr. Biol.* **13**, 1169–1181 (2003).

148. Zhao, Y., Tong, C. & Jiang, J. Hedgehog regulates smoothened activity by inducing a conformational switch. *Nature* **450**, 252–258 (2007).

149. Chen, Y. *et al.* G protein-coupled receptor kinase 2 promotes high-level Hedgehog signaling by regulating the active state of Smo through kinase-dependent and kinase-independent mechanisms in *Drosophila*. *Genes Dev.* **24**, 2054–2067 (2010).

150. Philipp, M. *et al.* Smoothened signaling in vertebrates is facilitated by a G protein-coupled receptor kinase. *Mol. Biol. Cell* **19**, 5478–5489 (2008).

151. Corbit, K. C. *et al.* Vertebrate Smoothened functions at the primary cilium. *Nature* **437**, 1018–1021 (2005).

152. Kuwabara, P. E. & Labouesse, M. The sterol-sensing domain: multiple families, a unique role? *Trends Genet.* **18**, 193–201 (2002).

153. Hausmann, G., von Mering, C. & Basler, K. The hedgehog signaling pathway: where did it come from? *PLoS Biol.* **7**, e1000146 (2009).

154. Halbleib, J. M. & Nelson, W. J. Cadherins in development: cell adhesion, sorting, and tissue morphogenesis. *Genes Dev.* **20**, 3199–3214 (2006).

155. Chen, X. *et al.* Processing and turnover of the Hedgehog protein in the endoplasmic reticulum. *J. Cell Biol.* **192**, 825–838 (2011).

156. Abel, E. S. *et al.* Possible roles of protein kinase A in cell motility and excystation of the early diverging eukaryote Giardia lamblia. *J. Biol. Chem.* **276**, 10320–10329 (2001).

157. Pitsouli, C. & Perrimon, N. Developmental biology: our fly cousins’ gut. *Nature* **454**, 592–593 (2008).


**Acknowledgements**

The authors are grateful to A. McMahon for his input into earlier drafts of the manuscript and to the anonymous Reviewers for their helpful suggestions for improvements. The authors would like to apologize to those authors whose work they have failed to cite owing to journal space constraints. Y.N.’s sabbatical visit to Institute of Molecular and Cell Biology (Japan) was supported by a Naito Foundation Fellowship (Japan) and by the IMCB.


**Competing interests statement**

The authors declare competing financial interests: see Web version for details.


**FURTHER INFORMATION**

Philip W. Ingham’s homepage:  
http://www.imcb.a-star.edu.sg/php/philippingham.php


**SUPPLEMENTARY INFORMATION**

See online article: S1 (figure)

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
