
Biochimica et Biophysica Acta 1809 (2011) 678–685

Contents lists available at ScienceDirect

Biochimica et Biophysica Acta

journal homepage: www.elsevier.com/locate/bbagrm

Review

Role of microRNAs in hepatitis B virus replication and pathogenesis ⋆,☆,★

Wan-Hsin Liu<sup>a,b</sup>, Shiou-Hwei Yeh<sup>a,b</sup>, Pei-Jer Chen<sup>b,c,d,e,*</sup>

<sup>a</sup> Department of Microbiology, National Taiwan University College of Medicine, Taipei, Taiwan  
<sup>b</sup> NTU Center for Genomic Medicine, National Taiwan University College of Medicine, Taipei, Taiwan  
<sup>c</sup> Graduate Institute of Clinical Medicine, National Taiwan University College of Medicine, Taipei, Taiwan  
<sup>d</sup> Hepatitis Research Center, National Taiwan University and Hospital, Taipei, Taiwan  
<sup>e</sup> Center of Genomic, Academia Sinica, Taipei, Taiwan  

---

**ARTICLE INFO**

Article history:  
Received 5 March 2011  
Received in revised form 24 April 2011  
Accepted 25 April 2011  
Available online 4 May 2011  

Keywords:  
MicroRNA  
Hepatitis B virus  
Hepatocellular carcinoma  
Transcription  
Replication  
Pathogenesis  

---

**ABSTRACT**

The hepatitis B virus (HBV) is a widespread human pathogen and chronic HBV infection is a major risk factor for hepatocellular carcinoma (HCC). The role of microRNA (miRNA) in the replication and pathogenesis was reviewed. So far none of HBV-encoded miRNA has been identified. Cellular miRNAs have shown able to regulate HBV at the transcription level either by targeting to cellular transcriptions factors required for HBV gene expression, or by a directly binding to HBV transcripts. We also summarized the changed patterns of cellular miRNAs from hepatitis B progressing to cirrhosis and then liver cancer. The changing of a few of miRNAs, such as miR-122 or miR-21, were reproduced and worthy of further research by a deep sequencing and functional validation. These HBV-specific miRNAs should potentially become biomarkers for HBV infection and HBV-positive HCC diagnosis. The understanding of miRNA biology paved the way for applying miRNAs-based RNAi against HBV replication with minimal toxicities. This article is part of a Special Issue entitled: MicroRNAs in viral gene regulation.

© 2011 Elsevier B.V. All rights reserved.

---

1. Introduction

HBV belongs to a family of small, enveloped DNA virus called Hepadnaviridae and causes acute or persistent infection in liver. There are more than 350 million chronic carriers of HBV worldwide and chronic HBV infection has been strongly associated with an increased risk of cirrhosis and in turn, leads to hepatocellular carcinoma (HCC) [1,2]. An understanding of hepatitis B biology and pathogenesis is important for hepatitis B control. HBV is a noncytopathic virus that replicates preferentially in the hepatocytes. Although the mechanism of HBV entry to hepatocytes remains unknown, the N-terminus of the viral large envelope protein has been implicated to involve in cell attachment and entry [3]. Following fusion of viral and cellular membranes, the viral capsid is assumed transported to the nuclear pore where the HBV relaxed circular genome (RC-DNA) is released into hepatocyte nucleus. Inside the nucleus, rcDNA is converted to a covalently closed circular DNA (cccDNA) presumably by cellular enzymes [4,5]. The cccDNA serves as template for transcription of all viral RNAs. HBV genome (3.2 kb) contains four overlapping open reading frames that codes for hepatitis B virus surface antigen (S), hepatitis B virus core protein (C), viral reverse DNA polymerase (Pol) and X protein (HBx). It also transcribes viral pregenomic RNA that will undergo reverse transcription to synthesize viral DNA genome. The 3.5-kb transcript is slightly larger than the viral genome and encodes for the viral core antigen and the polymerase. The 2.1-kb transcript represents the major transcript of the S gene which encodes the viral surface antigen and a minor singly spliced viral RNA [6]. The 2.4-kb transcript encodes mainly the large envelope protein and a minor doubly-spliced RNA [6]. Finally a 0.8-kb transcript encodes the HBx protein. The viral transcripts are under the control of two master regulators, enhancer I and II, and four distinct promoters (Fig. 1 A). All HBV mRNAs terminate at a common polyadenylation signal (Fig. 2 B). As all viral life cycles depends upon the transcription of pregenomic and viral mRNA, so the step of HBV gene transcription has been the key stage for understanding HBV biology.

Viruses are a class of intracellular pathogens with well established roles in the development of many diseases. Many cellular mechanisms, such as innate immunity and cell signaling pathways may participate in controlling viral pathogenesis. Recently, there numerous studies highlighted cellular or viral miRNAs as a new class of regulators in viral pathogenesis [7]. Evidences have supported viruses, such as Herpes viruses, encode their own miRNAs which manipulate the expression of cellular proteins or viral proteins, facilitating their infection cycles [8]. Just as viruses-encoded miRNAs can alter the expression of host genes, virus-host interactions have co-evolved so that viruses also could be

---

☆ This article is part of a Special Issue entitled: MicroRNAs in viral gene regulation.  
☆☆ Potential conflict of interest: Nothing to declare.  
* Corresponding author at: Graduate Institute of Clinical Medicine, National Taiwan University College of Medicine, No. 7 Chung-Shan South Road, Taipei 100, Taiwan. Tel.: +886 2 23123456x67072; fax: +886 2 23317624.  
E-mail address: peijerchen@ntu.edu.tw (P.-J. Chen).

1874-9399/$ – see front matter © 2011 Elsevier B.V. All rights reserved.  
doi:10.1016/j.bbagrm.2011.04.008
regulated by host-encoded miRNAs. In the case of hepatitis C, cellular miR-122 has been shown regulating viral genome replication. In this review, we will discuss the effect of cellular miRNAs directly or indirectly on HBV transcription and replication, the alteration of host miRNAs profile by HBV infection, and as well as the role of these cellular miRNAs in the HBV-related pathogenesis.

## 2. Cellular miRNAs regulating HBV gene transcription and replication

Some cellular miRNAs were found to be capable of inhibiting or stimulating viral replication by directly targeting to viral RNAs [8,9]. Although HBV is a DNA virus, its transcripts might be targeted and regulated by cellular miRNAs. In the following, we will focus on some cellular miRNAs reported to affect HBV gene expression and replication.

### 2.1. MiRNAs modulating HBV-associated cellular proteins

HBV is complicatedly controlled within the host hepatocyte by a number of cellular proteins at multiple steps of the HBV replication cycle. HBV uses cccDNA to establish persistent infection by formation of nuclear minichromosome and to serve as a transcription template for generating pregenome and mRNA. Transcription of HBV cccDNA was known to be tightly regulated by epigenetic mechanisms, such as DNA methylation [10,11], acetylation, or histone modifications [12]. A number of liver-enriched transcription factors and nuclear receptors, including STAT3, HNF1/4, also have been shown to bind HBV promoter/enhancer elements or to be critical in regulation of HBV transcription [13,14]. Control of HBV at the step of transcription thus influences both HBV gene expression and replication. Here, we will briefly discuss recent findings about these critical transcription factors modulated potentially by cellular miRNAs in host hepatocytes, so affecting HBV transcription, as summarized in Fig. 1A.

CAAT enhancer-binding protein (C/EBP) α and β are co-expressed in hepatocytes and are involved in the hepatocyte metabolism and proliferation. C/EBP can bind and activate the HBV Enhancer II in a dose-dependent manner [15], and also binding to core promoter and S promoter to activate their transcription [16,17]. Wang et al. showed C/EBP-β was targeted and negatively regulated by miR-155 [18], which could down-regulate HBV transcription. A second example is miR-372. An inhibition of miR-372 results in reducing expression of its target gene, PRKACB, which induces phosphorylation of C-AMP-response element binding protein (CREB) and dissociated CREB from the promoter [19]. CREB was known to be required for expression of all HBV transcription unit for its binding to viral enhancer I [20]. Finally, a recent study by Lu's group showed miR-1 was able to enhance the HBV core promoter transcription activity by augmenting farnesoid X receptor alpha (FXRA) expression, a positive transcription factor [21].

HBV gene expression is found to be regulated by sex hormones and their nuclear receptors [22]. The ligand-stimulated androgen receptor was shown to stimulate the transcription of all HBV RNAs [23,24]. We demonstrated the mechanism as a direct binding of androgen receptor to the androgen-responsive element sites in viral enhancer I [14] (Fig. 1A), a finding subsequently confirmed by other [25]. Studies from others showed that estrogen could suppress HBV gene expression [26]. Interestingly, our group identified miR-18a over-expression in female HCC tissues, which target ER-alpha mRNA and inhibit its protein translation [27]. An elevation of miRNA-18a might lead to increasing HBV transcription. Moreover, we recently discovered that estrogen receptor alpha directly reduced the HBV transcription activity via interaction with enhancer I (unpublished data). A set of miRNAs were identified to be up-regulated after testosterone treatment in female mouse liver [28], however, none of miRNA has been shown to regulate androgen receptor levels so far.

### 2.2. Screening for cellular miRNAs targeting to HBV transcripts

In an attempt to systemically screen for cellular miRNAs affecting HBV replication, Zhang et al. employed a loss of function approach by transfecting antagomirs targeting to 328 human miRNAs into HepG2

(A)

![Diagram](https://i.imgur.com/placeholder.png)

(B)

![Diagram](https://i.imgur.com/placeholder.png)

Fig. 1. A summary of cellular miRNAs effect on HBV transcription. (A) Angled arrows indicate the HBV RNA start site for the major viral transcripts (Pre/core, PreS1, PreS, and X), and viral enhancer I and II are schematically depicted as boxes. DR I and DR II are two short repeats essential for viral replication. ARE represents two androgen response elements were identified to locate at enhancer I region. Transcription factors and nuclear receptors binding position on HBV genome are shown (C/EBP, CAAT enhancer-binding protein; CREB, C-AMP-response element binding protein; AR, androgen receptor; ER, estrogen receptor; FXRA, farnesoid X receptor alpha). The miRNAs targeting cellular factors known for regulating HBV transcription are shown. Arrow indicated an activated but the bars indicated an inhibited effect of the miRNAs on HBV transcription. (B) Positions of binding sequences in HBV transcripts proposed to be targeted by miRNAs are shown. Italic gray color represents miRNAs were predicted to probably bind HBV transcripts by four well-established target-prediction software.
2.2.15 cells which contained a dimer of the complete HBV genome able supporting full HBV replication, and then measured HBsAg expression [29]. In comparison to controls, antagonists for six miRNAs were found to cause 1.4- to 1.6-fold increase in HBsAg expression, without significantly affecting host cell proliferation. Bioinformatics analysis of six miRNAs with ViTa database suggested a putative binding site for miR-199a-3p in the HBsAg coding region and the other binding site for miR-210 in the HBV pre-S1 region (Fig. 1B). The direct effect of these two miRNAs on HBV RNA transcripts was validated by GFP reporter assay. This study suggests up-regulation of miR-199a-3p and miR-210 in HepG2 2.2.15 cells resulting in reduction of HBV replication by a direct binding to the viral RNAs. In addition, Russo's group found miR-125a-5p is able to interact with HBV surface antigen and interfere with its expression, thus reducing the amount of secreted HBsAg [30].

Consensus-target prediction approach using computer software also can be a useful and fast tool to predict potentially critical sequences of the HBV genome as human miRNAs target. Wu et al. used four well-established miRNA target-prediction software, namely miRanda, RNAhybrid, DIANA-MicroT, and MicroInspector to predict the targets for human miRNAs in HBV sequence [31]. The analysis showed that the miRNAs miR-7, miR-196b, miR-433 and miR-511 target the sequences in viral polymerase and surface gene; whereas miR-205 and miR-345 target to HBX and the pre-core genes, respectively (Fig. 1B, italic number). The target regions were highly conserved across the various clades of HBV and could be interesting for experimental validation. However, it should be noted the miRNAs, if actually binding to the proposed sites, could bind to all viral RNAs, and thus suppressing the translation of all HBV proteins and inhibiting replication from pre-genomic RNA.

### 2.3. Cellular miRNA regulating HBV replication

Other than miRNAs affecting on viral translation, there are reports on HBV replication. In Qiu's study, silence of miR-122 in HBV-transfected Huh-7 cells with antisense miR-122 significantly increase HBsAg and HBeAg secretion [32]. In contrast, miR-122 gain-of function in HBV-transfected HepG2 cells which containing low endogenous miR-122 expression showed a marked reduction of HBsAg and HBeAg expression. Interestingly, they noticed another important mechanism for the miR-122 effect. The miR-122 inhibitor also caused an increase in cellular Heme oxygenase-1 (HO-1) expression at both the mRNA and protein level [32]. HO-1 has been shown to reduce the stability of the HBV core protein, thereby decreasing HBV cccDNA levels both *in vitro* and *in vivo* [33]. Overall, miR-122 was found to suppress HBV replication indirectly through HO-1. This finding is similar with the Shan's observation on HCV previously [34]. Finally, several lines of evidence have suggested that miR-122 plays a role in stimulating the replication and expression of HCV, nevertheless, Qiu's study provided an opposite function of this liver-rich miR-122 in its ability to restrict HBV replication (Fig. 2).

DNA hypermethylation might play crucial roles not only in silencing of host genes but also in suppressing HBV cccDNA transcription. Increased protein expression of DNMTs has been significantly correlated with the malignant potential [35] as well as decrease of HBV gene expression and replication by methylating viral DNA [10,11]. To study the relationship between HBV and DNMT expression, Zhang's et al. showed the expression of miR-152 to be down-regulated in the livers of HBx transgenic mice [36]. In the further study, DNMT-1 was found to be a target for both miR-148a and miR-152 [37,38] (Fig. 2). MiR-152 expression was also shown to be inversely correlated with DNMT1 expression in HBV-related HCC. These support that miR-152 may be a factor in the regulating the methylation of host and viral cccDNA.

Recent study by Lu's group noted that miR-1 over-expression resulted in a marked increase of HBV replication, accompanied with

---

**Fig. 2.** Schematic illustration for the effects of cellular miRNA regulation on HBV replication. The miRNAs can regulate HBV replication through modulating critical cellular factors. MiR-1 increases viral replication by activating farnesoid X receptor alpha (FXAR) and suppressing histone deacetylase 4 (HDAC4). MiR-122-induced down-regulation of Heme oxygenase-1 (HO-1) negatively affects miR-122-mediated suppression of HBV. MiR-449a enhances HBV replication, whereas miR-210 reduces its replication by unknown cellular factors. MiR-152 and miR-148a inhibit DNA methyltransferase 1 (DNMT1), may affecting viral replication.

up-regulated HBV transcription, antigen expression, and progeny secretion [21]. However, miR-210 was found to decrease HBV replication. MiR-1 was known to increase HBV transcription under the control of the HBV core promoter in a farnesoid X receptor alpha (FXRA)-dependent manner. Also, HDAC4, a member of histone deacetylase, was identified as a cellular target of miR-1 and inhibited by it. However, HDAC4 was reported to suppress HBV replication. Thus, HDAC4 could attenuate the increase replication of HBV by miR-1. Currently, they newly identified another miRNA, miR-449a, had a high capacity for enhancing HBV replication (personal communication). But miR-449a did not target to HBV genome directly by bioinformatic analysis and reporter assay. However, miR-449a may promote cell differentiation by increasing expressions of hepatocyte-specific factors, which may be beneficial for HBV replication.

Collectively, host miRNAs can modulate HBV replication through regulating cellular factors that do not direct bind to viral genome (Fig. 2). This kind of indirect effect on HBV replication by cellular miRNAs is also an important way to understand host factors involved in HBV infection.

### 3. MiRNA regulated by HBV-encoded protein

Though many cellular miRNAs are reported to regulate HBV expression and replication, however, fewer studies noticed whether HBV-encoded proteins should influence cellular microRNA expression. Hepatitis B virus × protein (HBx) is a 17 kDa protein encoded by HBV and is implicated to play an important role in hepatocarcinogenesis [39]. Wang et al. studied the expression of 286 human miRNAs in HBx-expressing cells using miRNA microarrays [40]. HBx was found to significantly up-regulate the expression of 7 miRNAs but down-regulate the expression of 11 cellular miRNA, respectively. An inverse correlation was noted between the expression of HBx and that of the highly-expressed members of the let-7 family including let-7a, let-7b and let-7c in HCC patients. Let-7 is the first miRNA identified in humans and was identified to target to a number of important cellular oncogenes, such as Ras [41], HMGA2 [42] and MYC [43]. Furthermore, STAT3, known to play a critical role in liver regeneration and HCC oncogenesis, was identified as the direct cellular target of let-7 in Wang's study. Viral proteins other than HBx have not been shown directly influencing cellular miRNA profiles at present.

### 4. Alteration of miRNAs profile in tissues from chronic hepatitis B patients

Recent evidences have supported that viruses might have evolved strategies to prevent infected cells from undergoing apoptosis and

escape the innate or adaptive immune responses in host cells through regulating the expression of miRNAs [44]. HBV infection may modulate the expression of host cellular miRNAs, which participated in development of HBV-related liver diseases. Numerous studies on miRNA profiles in chronic hepatitis B patient tissues or in HBV-expressing cells are reviewed (Fig. 3).

### 4.1. MiRNAs profiling in HBV-associated clinical HCC

Several reports have examined the miRNA expression profiles in human HCC tissues and analyzed the miRNA alteration link to HBV infection by miRNA-based microarray or sequencing. They also try to find if some specific miRNA is HBV-associated or HBV-specific. Ura et al. measured the expression of 188 miRNAs in hepatic tumor and non-tumor tissues obtained from 12 HBV-related and 14 HCV-related patients. Depending upon the viral etiology, 6 miRNAs exhibited a decreased expression in the HBV group and 13 miRNAs showed a decreased expression in the HCV group. This finding suggested a difference in the miRNA expression caused by either HBV or HCV infection [45]. Murakami et al. analyzed miRNA expression in 24 HCC samples and 22 adjacent non-tumor tissues; 30 miRNAs were significantly differentially expressed in the HCC [46]. Ladeiro et al. analyzed the expression of 250 miRNAs in a series of 46 malignant and benign hepatocellular tumors, with respect to 4 normal liver tissues. They further showed miR-96 significantly over-expressed in HBV-related HCC when compared to HCC from non-HBV causes [47]. In Yeh's study, 17 miRNAs were found to be de-regulated in 80 HCC patients' profile. Elevated miR-18a expression appeared to be statistically associated with female gender, especially in HBV cases (female to male ratio: 6.65). Their finding also suggested miR-18a suppressing the translation of ER-alpha thus increasing the risk of HCC development in women [27].

Wang and co-workers examined the miRNA expression pattern, survival, and response to interferon in both men and women with liver cancer [48]. A total of 455 patients with HCC who had undergone radical tumor resection were included. Most patients in the study were chronic HBV infection that in a majority of cases had liver cirrhosis. Class-comparison analysis of paired tumors in HBV-related HCC patients revealed that 7 miRNAs were differentially expressed between men and women, and miR-26 is ubiquitously over-expressed in non-cancerous liver tissues from HBV-positive HCC patients. Analysis of gene network revealed that activation of signaling pathways between nuclear factor κB (NFκB) and interleukin-6 was generally observed in miR-26 reducing patients. Intriguingly, interleukin-6 expression was inversely correlated with miR-26 expression. Such a link is functionally relevant to liver cancer in the rodent model.

Takizawa's group using sequencing method to obtain a comprehensive profile and to characterize miRNA transcriptomes in HBV associated HCC [49]. More than 314,000 reliable reads from HCC and more than 268,000 from adjacent normal liver were obtained. Expression profiling using bioinformatics-based analysis and clone count analysis showed several miRNAs were expressed aberrantly in liver cancer, including miR-122, miR-21 and miR-34a. This technique additionally can detect miRNA modifications, for examples, adenosine-to-inosine in miR-376 family. The combination of sequencing and bioinformatics may accelerate the discovery of novel miRNAs involved in HBV related human liver cancer.

Recent studies reported that human serum/plasma contains a number of stable miRNAs, which can potentially serve as a novel noninvasive biomarker for disease diagnosis [50–52]. Li et al. examined the profiles of miRNAs in serum samples from 210 controls, 135 HBV-, 48 HCV- and 120 HCC-affected individuals by Solexa sequencing followed by validation with quantitative RT-PCR assay [53]. It successfully identified a unique expression profile for HBV-related serum miRNAs and HCC-related serum miRNAs, respectively. Compared with control serum, 13 miRNAs were obtained with differentially expressed in HBV serum (Fig. 3). This 13-miRNA-based biomarker accurately discriminated HBV serum samples from controls and HCV cases. Two of them, miR-375 and miR-92a, were further identified as HBV specific. This is the first approach to select specific serum miRNA which can be used for detecting HBV-specific cases.

### 4.2. MiRNAs profiling in HBV-related cirrhosis or dysplastic nodules

Chronic hepatitis B frequently progresses into liver cirrhosis that predisposes to HCC. A growing number of studies have focused on the expression profiles of miRNAs during cirrhotic stage for their roles in pathogenesis [54–60]. Jiang's study showed that a large number of miRNAs have increased expression in hepatitis positive and cirrhotic tissues compared with the normal liver. Their data suggest that the combination of both cirrhosis and hepatitis viral infection significantly enhances the alteration of primary miRNA expression [55]. Pineau et al. profiled miRNA expression from tissue samples of 21 normal livers, 90 adjacent cirrhotic livers, 104 HCC as well as 35 HCC cell lines and a set of 12 miRNAs was linked to disease progression from normal liver through cirrhosis to full-blown HCC [60]. Guo et al. showed changes in miRNAs associated with hepatic stellate cells (HSCs), which is the key event in liver fibrosis, revealed that 13 pathways were up-regulated

![Figure](https://i.imgur.com/yourimage.png)

**Fig. 3.** The microRNAs profiles examined to be associated with the progressive stages from chronic hepatitis to cirrhosis and to HCC after HBV infection. The stages of HCC progression are shown. Those miRNAs above represent those elevated, whereas below were miRNAs with reduced expression. The black-color group indicated changed miRNAs in HCC samples, as compared with that of adjacent normal liver in HBV carriers. The red group represents miRNA profiles in HBV serum with that in normal control serum. MiR-143 (blue) and miR-152 were screened from tumor tissues in p21-HBx mice compared with adjacent non-cancerous hepatic tissues. MiR-602 (purple) was found elevated in tissue samples collected from each stage compared with that from normal liver. Asterisk represents the number of times made from different observations for the indicated miRNA. MiR-122 was consistently down-regulated in three different studies. MiR-21 was consistently up-regulated in different studies from stage of cirrhosis to HCC.

and 22 pathways were down-regulated by miRNAs [61]. Next, we will focus on discussing the miRNA expression profile in the HBV-related cirrhosis.

Gao's study investigated the deregulation of miRNA in this multi-step process in HBV-associated liver samples [54]. A panel of seven miRNAs, which have been reported to be frequently de-regulated in liver carcinogenesis [46,47,62], were selected to investigate their expression during the early stage of HCC, associated with chronic HBV infection. The results showed an increasing trend of miRNA de-regulation from low-grade to high-grade dysplastic nodules and HCC. Low-grade dysplastic nodules were characterized by frequent down-regulation of miR-145 and miR-199b, whereas over-expression of miR-21, miR-221 and miR-224 were frequently found in the small HCCs. miR-122 was significantly under-expressed in small HCCs compared with cirrhotic/chronic hepatitis livers. This study provided a concept that miRNA de-regulation is an early event and accumulate along the multi-steps of HBV-associated hepatocarcinogenesis.

Hypersplenism is a clinical syndrome common in portal hypertension (PH) in liver cirrhosis patients. In general, cytopenia results predominantly from the increased phagocytosis and destruction of blood cells in splenic macrophages [63]. Li et al. determined whether miRNA expression is altered in splenic macrophages associated with hypersplenism due to portal hypertension in HBV-related cirrhosis [64]. Screening 292 miRNAs expressed in splenic macrophages, 99 miRNAs were identified as differentially expressed between HS-PHT and normal spleen, suggesting miRNAs may be involved in the pathogenesis of HS-PHT. Among the miRNAs identified in this study, miR-615-3p was significantly up-regulated in hypersplenism. The results in this study revealed miRNA should be a new layer of regulatory mechanisms in the pathogenesis of hypersplenism in HBV-related cirrhosis.

An important miRNA molecule which is significantly up-regulated in HBV-infected liver and HCC is miR-602 [65]. Yang et al. evaluated miRNA expression in normal, chronic HBV hepatitis, HBV-positive cirrhotic, HBV-positive HCC and corresponding adjacent non-tumor livers using miRNA microarray. Fourteen miRNAs were aberrantly expressed in HBV-related cirrhosis and HBV-related HCC, however, miR-602 was found that started to increase significantly from the stage of HBV hepatitis and reached the highest in HCC compared with normal liver. MiR-602 expression was validated to be up-regulated in HepG2 2.2.15 and HepG2-HBx about 2–3 fold of that in HepG2, which is consistent with the observation in clinical study. The cellular RASSF1A was identified as the target of miR-602. RASSF1A protein level increased while miR-602 inhibition in HepG2 cells, which resulted in increased apoptosis and decreased cell proliferation rate.

### 4.3. Specific miRNA for HBV-related HCC in animal models

Zhang's study showed up-regulation of miR-143 expression transcribed by NF-κB in HBV-HCC promotes cancer cell invasion/migration and tumor metastasis by repression of FNDC3B expression. They further used miRNA array and real-time PCR to screen specific candidate miRNA molecules in the livers of p21-HBx transgenic mice, in which hepatocellular cancer developed 18 months after birth. Among miRNAs analysis, miR-143 was under-expressed in the livers of 10-month old transgenic mice and then significantly up-regulated after HCC developed especially when accompanied by tumor metastasis to the lung. Additionally, miR-143 was dramatically up-regulated in 25 HBV-HCC samples and reached eight-fold increase among the HBV-HCC patients with lung metastasis. The ability of miR-143 in promoting cell metastasis was confirmed in HBx-HepG2 cells and in an athymic nude mouse model. Taken together, this study suggested a possible pathway mediated by miR-143 that promotes the metastasis of HCC derived from HBV infection in vitro and in vivo [36].

### 4.4. MiRNAs profiling in HBV-expressing cells

Cell culture is a simple system in which miRNAs change could be easily and conveniently assayed after HBV infection or transfection. Liu et al. compared the expression profile of cellular miRNAs of a stable HBV-expressing cell line HepG2.2.15 and its parent cell line HepG2 using miRNA microarray and Northern blot analysis [66]. Eighteen miRNAs were differentially expressed between the two cell lines defined by a threshold of three-fold difference. Among them, eleven miRNAs were found to be up-regulated and seven were down-regulated in HepG2.2.15 cells. These miRNAs screened by microarray were validated further by Northern blot analysis. The data confirmed that the expression of miR-181a, miR-181b, miR-200b and miR-146a were up-regulated, whereas the expression of miR-15a were down-regulated. However, interactions between the HBV and the host cellular miRNAs are largely unclear.

In summary, HBV infection was found to change the cellular miRNAs expression in host cells (Fig. 3). During the progression from chronic hepatitis to cirrhosis and HCC, some miRNAs were found to be aberrantly expressed from early stage to the end HCC, such as miR-21, miR-199b, miR-145 and miR-602. Detailed analysis also suggested that down-regulation of miR-145 and miR-199b is an early event frequently detected in pre-malignant dysplastic nodules and persistent throughout HCC development [54]. Among these miRNAs, miR-145 was abundantly expressed in non-tumorous livers but dramatically reduced in pre-malignant low-grade dysplastic nodules. An over-expression of miR-145 in both HepG2 and Hep3B cells significantly inhibited cell proliferation, migration and invasion.

Many studies yielded different changing profiles of miRNAs with a greater variation, however, only both miR-122 and miR-21 changes are consistently reproduced. MiR-122 was highly expressed in the normal liver and cirrhosis, but significantly down-regulated at the stage of HCC. It is a mammalian liver-specific microRNA and is expressed at a high level, accounting for around 70% of all cloned miRNAs in liver [67,68]. Several studies have reported the diversity of its roles in the hepatic function, including cell cycle progression and tumorigenesis [69,70], lipid metabolism [71], response to stress [72], and enhancing HCV replication [73]. The study by Tsou's group also showed restoration of miR-122 significantly reduced in vitro migration, invasion, and anchorage-independent growth as well as in vivo tumorigenesis, angiogenesis, and intrahepatic metastasis by direct suppression of the metastasis-related gene, ADAM17 [74]. Its role in HBV-related pathogenesis deserves a more extensive study.

Two recent studies reported miR-21 up-regulation at cirrhotic stage [54,60] and other studies showed its up-regulation was found in HCC [49,75,76]. Obviously, miR-21 expression increased through the progression into HCC. MiR-21's function in HCC was firstly suggested contributing to HCC growth and spread by reducing PTEN expression and enhancing cell growth, migration and invasion [75]. Mori's group suggested miR-21 rendering HCC cells more resistant to IFN-alpha/5-FU treatment and miR-21 expression in clinical HCC specimens was associated with the clinical response to the IFN-alpha/5FU [77].

### 5. Application of miRNAs or antagomir for HBV therapeutic

The use of RNA interference (RNAi) to inhibit gene expression is potentially applicable in the treatment of viral infections such as HBV [78]. Both synthetic and expressed sequences are being widely used to achieve RNAi-mediated HBV gene knockdown. However, a recent finding suggested that anti-HBV shRNAs may cause serious toxicity in vivo [79], raising an important concern of safety using expressed RNAi sequences for therapeutic application. In order to improve the safety of RNAi-mediated inhibition of HBV infection, other strategies were developed to take the advantages of the use of cellular miRNAs.

In order to generate a safety and effective RNAi trigger, McCaffrey group produced HBV RNAi triggers that more closely mimicked the

Table 1  
Cellular targets of miRNAs and effects on HBV biology/pathogenesis.

| miRNAs       | Known functions                                                                 | Cellular targets          | Effects on HBV biology or pathogenesis                                                                                   | References      |
|--------------|---------------------------------------------------------------------------------|--------------------------|------------------------------------------------------------------------------------------------------------------------|-----------------|
| miR-1        | Inhibit cell proliferation                                                      | HDAC4                    | Increase HBV replication by augmenting FXRA activity                                                                     | [21]            |
|              |                                                                                 | E2F5                     |                                                                                                                        |                 |
| miR-10b      | Promote metastasis in breast cancer                                             | HOXD10                   | Elevation in cirrhosis                                                                                                  | [49,54,84]      |
| miR-18a      | Increase HCC cell proliferation                                                 | ERalpha                  | Elevation in female HCC                                                                                                | [27]            |
| miR-21       | Promote proliferation and metastasis                                            | PTEN                     | Elevation in HCC                                                                                                       | [75]            |
| miR-26       | Inverse correlation with NFkB and IL-6                                          | Unknown                  | Down-regulation in HCC                                                                                                 | [48]            |
| miR-34a      | Inhibit metastasis                                                              | c-Met                    | Aberrant expression in HCC                                                                                             | [54,85,86]      |
| miR-122      | Response to nutrient or toxic stress<br>Suppress hepatocarcinogenesis<br>Enhance HCV replication<br>Regulation of lipid metabolism | CAT-1<br>Cyclin G1<br>HCV RNA<br>HO-1                      | Regulate HBV expression through inhibiting HO-1 expression<br>Down-regulation in HCC                                      | [32,68–71,87]   |
| miR-125a-5p  | Inhibit cell proliferation                                                      | ERBB2                    | Suppress HBsAg expression                                                                                              | [30,88]         |
| miR-152      | Induces aberrant DNA methylation                                                | DNMT-1                   | Down-regulation in HCC                                                                                                 | [37,38]         |
| miR-155      | Promote HCC cell growth                                                         | C/EBP                    | Elevation in HCC                                                                                                       | [18]            |
| miR-199-3p   | Promote liver fibrosis                                                          | Unknown                  | Suppress HBV replication by binding to sequences of HBsAg region                                                        | [29,89]         |
| miR-210      | Accelerate progression of cholangiocarcinoma                                    | Unknown                  | Suppress HBV replication by binding to sequences of pre-S1 region                                                       | [29,90]         |
| miR-221      | Regulate cell growth and apoptosis                                              | DDIT4, Bmf               | Elevation in HCC tumorigenesis                                                                                         | [54,60,91]      |
| miR-223      | Inhibit cell viability                                                          | STMN1                    | Aberrant expression in HCC                                                                                             | [53,92]         |
| miR-224      | Regulate cell proliferation and metastasis                                      | PAK4, MMP9               | Elevation in HCC                                                                                                       | [54,93]         |
| miR-449a     | Promote cell differentiation                                                    | Unknown                  | Enhance HBV replication by unknown mechanism                                                                           | Personal communication |
| miR-602      | Regulate cell proliferation and apoptosis                                       | RASSF1A                  | Elevation in HCC progression                                                                                           | [65]            |
| let-7        | Inhibit cell proliferation,<br>Regulation of cell death<br>Regulation of metastasis<br>Inhibit cell growth | STAT3, MYC<br>Caspase 3<br>RAS<br>HMGA2                   | Aberrant expression in HCC<br>Repression by HBx                                                                         | [40–43,53]      |

endogenous substrate of the miRNA processing pathway [78,80]. The RNAi triggers were embedded within the context of the endogenous miR-30, and in total 14 miRNA-based triggers were constructed and tested in culture. All rationally designed HBV-miRNAs triggers reduced the levels of HBV RNA by >50% and the combination of these strategies produced highly potent HBV RNAi. In HBV transgenic mice model, miRNA-based HBV RNAi expression from AAV vectors reduced about 100-fold HBV genomic DNA in serum. In notice, none of the mice displayed any sign of overt toxicity and none died.

For another example, Arbuthnot and colleagues designed pri-miRNA expression cassettes by replacing guide and complementary sequences of natural miR-31 and miR-122 with those of an effective anti-HBV shRNA [81]. The pri-miR-like sequences effectively knocked down markers of viral replication (>80%) in cultured cells. The number of circulating viral particle equivalents decreased at least 95% in the murine hydrodynamic injection model. This approach used pri-miR-122 or pri-miR-31 mimic shuttles successfully inhibit HBV replication in vitro and in vivo.

Employing a similar strategy, Jiang's group developed a lentiviral miRNA-based system that expressed siRNAs targeting the HBsAg gene of HBV. This construct significantly inhibited the HBsAg mRNA and protein level in the HepG2.2.15 cells, while HBsAg secretion was decreased by 70%. It suggests this system with lentiviral microRNA-based RNAi against the HBsAg gene is an effective tool in controlling HBV expression [82].

Another improvement is the use of linear DNA. Since linear DNA traverses nucleopores efficiently and may therefore be useful for improving delivery efficiency [83]. Chattopadhyay et al. developed an alternative DNA template containing anti-HBV miR-122 expression cassettes. Silencing of HBV replication was initially assessed in cell culture and the efficiency of >75% was found without inducing an interferon response. And a knockdown of approximately 95% HBV replication was achieved in the hydrodynamic infection model. Collectively, this kind of design is a useful alternative for silencing of HBV replication in both in vitro and in vivo.

### 6. Conclusion and perspective

In this review, we focus on the roles of miRNAs involved in HBV transcriptions (Fig. 1) and replication (Fig. 2). Indeed, only two miRNAs, miR-210 and miR-199-3p, were experimentally shown to affect HBV gene expression and replication in cell culture, maybe through a direct binding to HBV transcripts. Their activities in HBV-infection tissues await further studies. In contrast, there are many cellular miRNA indirectly regulating HBV life cycles by influencing virus-relevant cellular proteins. Such miRNAs can also play important roles in hepatitis B pathogenesis. We listed the cellular functions of these relevant miRNAs in liver disease and their biological properties associated with HBV or HBV-related HCC in Table 1. We also summarized the known changes of miRNAs involved in multiple steps of chronic hepatitis B, from hepatitis to cirrhosis and finally to HCC (Fig. 3). Most of these miRNAs do not target to HBV genomes or mRNAs, but more likely to cellular RNAs to exert their pathological effects. Obviously, certain HBV-specific host miRNAs expressions screening from serum samples were found to be changed in the early stages during HCC development, therefore, they should be good candidates for diagnostic biomarkers for liver diseases prediction. Actually, most studies addressed the effect of host cellular miRNAs on HBV infection, particularly at the level of HBV transcription. Despite the identification of numerous cellular miRNAs interacting with HBV, an in depth functional understanding is still incomplete. A combination of bioinformatics and experimental screening (such as systemic antagomirs) will let us reach this goal more effectively. The advance of the knowledge can be coupled to RNA interference for silencing HBV infection. Ideally, the development would apply a miRNA-based RNAi, not only achieving an effective inhibition of HBV replication but also remaining safe in vivo. Taken together, this understanding between miRNAs and HBV interaction will continue grow by incorporating new miRNA and their new functions, and possibly yielding new insights of therapeutic relevance.

#### Acknowledgments

This work was supported by grant from the National Research Program of Genomic Medicine, the National Science Council, (NSC99-3112-B-002-015, NSC98-3112-B-002-044 and NSC99-3112-B-002-024), the National Health Research Institutes, (NHRI-EX98-9832BI), and Center of Genomic Medicine, Academia Sinica in Taiwan.

#### References

[1] C.L. Lai, M.F. Yuen, Chronic hepatitis B-new goals, new treatment, N. Engl. J. Med. 359 (2008) 2488–2491.

[2] H.I. Yang, S.H. Yeh, P.J. Chen, U.H. Iloeje, C.L. Jen, J. Su, L.Y. Wang, S.N. Lu, S.L. You, D.S. Chen, Y.F. Liaw, C.J. Chen, Associations between hepatitis B virus genotype and mutants and the risk of hepatocellular carcinoma, J. Natl. Cancer Inst. 100 (2008) 1134–1143.

[3] C. Lepere-Douard, M. Trotard, J. Le Seyec, P. Gripon, The first transmembrane domain of the hepatitis B virus large envelope protein is crucial for infectivity, J. Virol. 83 (2009) 11819–11829.

[4] B. Weiser, D. Ganem, C. Seeger, H.E. Varmus, Closed circular viral DNA and asymmetrical heterogeneous forms in livers from animals infected with ground squirrel hepatitis virus, J. Virol. 48 (1983) 1–9.

[5] Y. Wei, C. Neveut, P. Tiollais, M.A. Buendia, Molecular biology of the hepatitis B virus and role of the X gene, Pathol. Biol. (Paris) 58 (2010) 267–272.

[6] G.H. Lee, S. Wasser, S.G. Lim, Hepatitis B pregenomic RNA splicing—the products, the regulatory mechanisms and its biological significance, Virus Res. 136 (2008) 1–7.

[7] A. Grundhoff, C.S. Sullivan, Virus-encoded microRNAs, Virology 411 (2011) 325–343.

[8] Z. Lin, E.K. Flemington, miRNAs in the pathogenesis of oncogenic human viruses, Cancer Lett. 305 (2011) 186–199.

[9] B.R. Cullen, Viral and cellular messenger RNA targets of viral microRNAs, Nature 457 (2009) 421–425.

[10] J.W. Kim, S.H. Lee, Y.S. Park, J.H. Hwang, S.H. Jeong, N. Kim, D.H. Lee, Replicative activity of hepatitis B virus is negatively associated with methylation of covalently closed circular DNA in advanced hepatitis B virus infection, Intervirology (2011) [Epub ahead of print].

[11] P. Vivekanandan, H.D. Daniel, R. Kannangai, F. Martinez-Murillo, M. Torbenson, Hepatitis B virus replication induces methylation of both host and viral DNA, J. Virol. 84 (2010) 4321–4329.

[12] L. Belloni, T. Pollicino, F. De Nicola, F. Guerrieri, G. Raffa, M. Fanciulli, G. Raimondo, M. Levrero, Nuclear HBx binds the HBV minichromosome and modifies the epigenetic regulation of cccDNA function, Proc. Natl. Acad. Sci. U. S. A. 106 (2009) 19975–19979.

[13] M. Quasdorf, U. Protzer, Control of hepatitis B virus at the level of transcription, J. Viral Hepat. 17 (2010) 527–536.

[14] S.H. Wang, S.H. Yeh, W.H. Lin, H.Y. Wang, D.S. Chen, P.J. Chen, Identification of androgen response elements in the enhancer I of hepatitis B virus: a mechanism for sex disparity in chronic hepatitis B, Hepatology 50 (2009) 1392–1402.

[15] M. Lopez-Cabrera, J. Letovsky, K.Q. Hu, A. Siddiqui, Transcriptional factor C/EBP binds to and transactivates the enhancer element II of the hepatitis B virus, Virology 183 (1991) 825–829.

[16] C.T. Bock, S. Kubicka, M.P. Manns, C. Trautwein, Two control elements in the hepatitis B virus S-promoter are important for full promoter activity mediated by CCAAT-binding factor, Hepatology 29 (1999) 1236–1247.

[17] M. Lopez-Cabrera, J. Letovsky, K.Q. Hu, A. Siddiqui, Multiple liver-specific factors bind to the hepatitis B virus core/pregenomic promoter: trans-activation and repression by CCAAT/enhancer binding protein, Proc. Natl. Acad. Sci. U. S. A. 87 (1990) 5069–5073.

[18] B. Wang, S. Majumder, G. Nuovo, H. Kutay, S. Volinia, T. Patel, T.D. Schmittgen, C. Croce, K. Ghoshal, S.T. Jacob, Role of microRNA-155 at early stages of hepatocarcinogenesis induced by choline-deficient and amino acid-defined diet in C57BL/6 mice, Hepatology 50 (2009) 1152–1161.

[19] J. Wang, X. Liu, H. Wu, P. Ni, Z. Gu, Y. Qiao, N. Chen, F. Sun, Q. Fan, CREB up-regulates long non-coding RNA, HULC expression through interaction with microRNA-372 in liver cancer, Nucleic Acids Res. 38 (2010) 5366–5383.

[20] B.K. Kim, S.O. Lim, Y.G. Park, Requirement of the cyclic adenosine monophosphate response element-binding protein for hepatitis B virus replication, Hepatology 48 (2008) 361–373.

[21] X. Zhang, E. Zhang, Z. Ma, R. Pei, M. Jiang, J.F. Schlaak, M. Roggendorf, M. Lu, Modulation of HBV replication and hepatocyte differentiation by microRNA-1, Hepatology 53 (2011) 1476–1485.

[22] S.H. Yeh, P.J. Chen, Gender disparity of hepatocellular carcinoma: the roles of sex hormones, Oncology 78 (Suppl. 1) (2010) 172–179.

[23] C.M. Chiu, S.H. Yeh, P.J. Chen, T.J. Kuo, C.J. Chang, P.J. Chen, W.J. Yang, D.S. Chen, Hepatitis B virus X protein enhances androgen receptor-responsive gene expression depending on androgen level, Proc. Natl. Acad. Sci. U. S. A. 104 (2007) 2571–2578.

[24] W.J. Yang, C.J. Chang, S.H. Yeh, W.H. Lin, S.H. Wang, T.F. Tsai, D.S. Chen, P.J. Chen, Hepatitis B virus X protein enhances the transcriptional activity of the androgen receptor through c-Src and glycogen synthase kinase-3beta kinase pathways, Hepatology 49 (2009) 1515–1524.

[25] M.H. Wu, W.L. Ma, C.L. Hsu, Y.L. Chen, J.H. Ou, C.K. Ryan, Y.C. Hung, S. Yeh, C. Chang, Androgen receptor promotes hepatitis B virus-induced hepatocarcinogenesis through modulation of hepatitis B virus RNA transcription, Sci. Transl. Med. 2 (2010) 32–35.

[26] I. Shimizu, N. Kohno, K. Tamaki, M. Shono, H.W. Huang, J.H. He, D.F. Yao, Female hepatology: favorable role of estrogen in chronic liver disease with hepatitis B virus infection, World J. Gastroenterol. 13 (2007) 4295–4305.

[27] W.H. Liu, S.H. Yeh, C.C. Lu, S.L. Yu, H.Y. Chen, C.Y. Lin, D.S. Chen, P.J. Chen, MicroRNA-18a prevents estrogen receptor-alpha expression, promoting proliferation of hepatocellular carcinoma cells, Gastroenterology 136 (2009) 683–693.

[28] D. Delic, C. Grosser, M. Dkhil, S. Al-Quraishy, F. Wunderlich, Testosterone-induced upregulation of miRNAs in the female mouse liver, Steroids 75 (2010) 998–1004.

[29] G.L. Zhang, Y.X. Li, S.Q. Zheng, M. Liu, X. Li, H. Tang, Suppression of hepatitis B virus replication by microRNA-199a-3p and microRNA-210, Antiviral Res. 88 (2010) 169–175.

[30] N. Potenza, U. Papa, N. Mosca, F. Zerbini, V. Nobile, A. Russo, Human microRNA hsa-miR-125a-5p interferes with expression of hepatitis B virus surface antigen, Nucleic Acids Res. (2011) [Epub ahead of print].

[31] F.L. Wu, W.B. Jin, J.H. Li, A.G. Guo, Targets for human encoded microRNAs in HBV genes, Virus Genes 42 (2011) 157–161.

[32] L. Qiu, H. Fan, W. Jin, B. Zhao, Y. Wang, Y. Ju, L. Chen, Y. Chen, Z. Duan, S. Meng, miR-122-induced down-regulation of HO-1 negatively affects miR-122-mediated suppression of HBV, Biochem. Biophys. Res. Commun. 398 (2010) 771–777.

[33] U. Protzer, S. Seyfried, M. Quasdorf, G. Sass, M. Svorcova, D. Webb, F. Bohne, M. Hosel, P. Schirmacher, G. Ties, Antiviral activity and hepatoprotection by heme oxygenase-1 in hepatitis B virus infection, Gastroenterology 133 (2007) 1156–1165.

[34] Y. Shan, J. Zheng, R.W. Lambrecht, H.L. Bonkovsky, Reciprocal effects of microRNA-122 on expression of heme oxygenase-1 and hepatitis C virus genes in human hepatocytes, Gastroenterology 133 (2007) 1166–1174.

[35] Y. Saito, Y. Kanai, T. Nakagawa, M. Sakamoto, H. Saito, H. Ishii, S. Hirohashi, Increased protein expression of DNA methyltransferase (DNMT) 1 is significantly correlated with the malignant potential and poor prognosis of human hepatocellular carcinomas, Int. J. Cancer 105 (2003) 527–532.

[36] X. Zhang, S. Liu, T. Hu, S. Liu, Y. He, S. Sun, Up-regulated microRNA-143 transcribed by nuclear factor kappa B enhances hepatocarcinoma metastasis by repressing fibronectin expression, Hepatology 50 (2009) 490–499.

[37] C. Braconi, N. Huang, T. Patel, MicroRNA-dependent regulation of DNA methyltransferase-1 and tumor suppressor gene expression by interleukin-6 in human malignant cholangiocytes, Hepatology 51 (2010) 881–890.

[38] J. Huang, Y. Wang, Y. Guo, S. Sun, Down-regulated microRNA-152 induces aberrant DNA methylation in hepatitis B virus-related hepatocellular carcinoma by targeting DNA methyltransferase 1, Hepatology 52 (2010) 60–70.

[39] Y. Wei, C. Neveut, P. Tiollais, M.A. Buendia, Molecular biology of the hepatitis B virus and role of the X gene, Pathol. Biol. 58 (2010) 267–272.

[40] Y. Wang, Y. Lu, S.T. Toh, W.K. Sung, P. Tan, P. Chow, A.Y. Chung, L.L. Jooi, C.G. Lee, Lethal-7 is down-regulated by the hepatitis B virus x protein and targets signal transducer and activator of transcription 3, J. Hepatol. 53 (2010) 57–66.

[41] S.M. Johnson, H. Grosshans, J. Shingara, M. Byrom, R. Jarvis, A. Cheng, E. Labourier, K.L. Reinert, D. Brown, F.J. Slack, RAS is regulated by the let-7 microRNA family, Cell 120 (2005) 635–647.

[42] C. Mayr, M.T. Hemann, D.P. Bartel, Disrupting the pairing between let-7 and Hmga2 enhances oncogenic transformation, Science 315 (2007) 1576–1579.

[43] V.B. Sampson, N.H. Rong, J. Han, Q. Yang, V. Aris, P. Soteropoulos, N.J. Petrelli, S.P. Dunn, L.J. Krueger, MicroRNA let-7a down-regulates MYC and reverts MYC-induced growth in Burkitt lymphoma cells, Cancer Res. 67 (2007) 9762–9770.

[44] P. Sarnow, C.L. Jopling, K.L. Norman, S. Schutz, K.A. Wehner, MicroRNAs: expression, avoidance and subversion by vertebrate viruses, Nat. Rev. Microbiol. 4 (2006) 651–659.

[45] S. Ura, M. Honda, T. Yamashita, T. Ueda, H. Takatori, R. Nishino, H. Sunakozaka, Y. Sakai, K. Horimoto, S. Kaneko, Differential microRNA expression between hepatitis B and hepatitis C leading disease progression to hepatocellular carcinoma, Hepatology 49 (2009) 1098–1112.

[46] Y. Murakami, T. Yasuda, K. Saigo, T. Urashima, H. Toyoda, T. Okanoue, K. Shimotohno, Comprehensive analysis of microRNA expression patterns in hepatocellular carcinoma and non-tumorous tissues, Oncogene 25 (2006) 2537–2545.

[47] Y. Ladeiro, G. Couchy, C. Balabaud, P. Bioulac-Sage, L. Pelletier, S. Rebouissou, J. Zucman-Rossi, MicroRNA profiling in hepatocellular tumors is associated with clinical features and oncogene/tumor suppressor gene mutations, Hepatology 47 (2008) 1955–1963.

[48] J. Ji, J. Shi, A. Budhu, Z. Yu, M. Forgues, S. Roessler, S. Ambs, Y. Chen, P.S. Meltzer, C.M. Croce, L.X. Qin, K. Man, C.M. Lo, J. Lee, I.O. Ng, J. Fan, Z.Y. Tang, H.C. Sun, X.W. Wang, MicroRNA expression, survival, and response to interferon in liver cancer, N. Engl. J. Med. 361 (2009) 1437–1447.

[49] Y. Mizuguchi, T. Mishima, S. Yokomuro, Y. Arima, Y. Kawahigashi, K. Shigehara, T. Kanda, H. Yoshida, E. Uchida, T. Tajiri, T. Takizawa, Sequencing and bioinformatics-based analyses of the microRNA transcriptome in hepatitis B-related hepatocellular carcinoma, PLoS One 6 (2011) e15304.

[50] X. Chen, Y. Ba, L. Ma, X. Cai, Y. Yin, K. Wang, J. Guo, Y. Zhang, J. Chen, X. Guo, Q. Li, X. Li, W. Wang, Y. Zhang, J. Wang, X. Jiang, Y. Xiang, C. Xu, P. Zheng, J. Zhang, R. Li, H. Zhang, X. Shang, T. Gong, G. Ning, J. Wang, K. Zen, J. Zhang, C.Y. Zhang, Characterization of microRNAs in serum: a novel class of biomarkers for diagnosis of cancer and other diseases, Cell Res. 18 (2008) 997–1006.

[51] S. Gilad, E. Meiri, Y. Yogev, S. Benjamin, D. Lebanony, N. Yerushalmi, H. Benjamin, M. Kushnir, H. Cholakh, N. Melamed, Z. Bentwich, M. Hod, Y. Goren, A. Chajut, Serum microRNAs are promising novel biomarkers, PLoS One 3 (2008) e3148.

[52] P.S. Mitchell, R.K. Parkin, E.M. Kroh, B.R. Fritz, S.K. Wyman, E.L. Pogosova-Agadjanyan, A. Peterson, J. Noteboom, K.C. O'Briant, A. Allen, D.W. Lin, N. Urban, C.W. Drescher, B.S. Knudsen, D.L. Stirewalt, R. Gentleman, R.L. Vessella, P.S. Nelson, D.B. Martin, M. Tewari, Circulating microRNAs as stable blood-based markers for cancer detection, Proc. Natl. Acad. Sci. U. S. A. 105 (2008) 10513–10518.

[53] L.M. Li, Z.B. Hu, Z.X. Zhou, X. Chen, F.Y. Liu, J.F. Zhang, H.B. Shen, C.Y. Zhang, K. Zen, Serum microRNA profiles serve as novel biomarkers for HBV infection and diagnosis of HBV-positive hepatocarcinoma, Cancer Res. 70 (2010) 9798–9807.

[54] P. Gao, C. Chak-Lui Wong, E. Kwok-Kwan Tung, J. Man-Fong Lee, C.M. Wong, I. Oi-Lin Ng, Deregulation of microRNA expression occurs early and accumulates in early stages of HBV-associated multistep hepatocarcinogenesis, J. Hepatol. (2010) [Epub ahead of print].

[55] J. Jiang, Y. Gusev, I. Aderca, T.A. Mettler, D.M. Nagorney, D.J. Brackett, L.R. Roberts, T.D. Schmittgen, Association of MicroRNA expression in hepatocellular carcinomas with hepatitis infection, cirrhosis, and patient survival, Clin. Cancer Res. 14 (2008) 419–427.

[56] T. Sukata, K. Sumida, M. Kushida, K. Ogata, K. Miyata, S. Yabushta, S. Uwagawa, Circulating microRNAs, possible indicators of progress of rat hepatocarcinogenesis from early stages, Toxicol. Lett. 200 (2011) 46–52.

[57] M. Shigoka, A. Tsuchida, T. Matsudo, Y. Nagakawa, H. Saito, Y. Suzuki, T. Aoki, Y. Murakami, H. Toyoda, T. Kumada, R. Bartenschlager, N. Kato, M. Ikeda, T. Takashina, M. Tanaka, R. Suzuki, K. Oikawa, M. Takanashi, M. Kuroda, Deregulation

of miR-92a expression is implicated in hepatocellular carcinoma development, Pathol. Int. 60 (2010) 351–357.

[58] S.K. Venugopal, J. Jiang, T.H. Kim, Y. Li, S.S. Wang, N.J. Torok, J. Wu, M.A. Zern, Liver fibrosis causes downregulation of miRNA-150 and miRNA-194 in hepatic stellate cells, and their overexpression causes decreased stellate cell activation, Am. J. Physiol. Gastrointest. Liver Physiol. 298 (2010) G101–G106.

[59] X. Jiang, E. Tsitsiou, S.E. Herrick, M.A. Lindsay, MicroRNAs and the regulation of fibrosis, FEBS J. 277 (2010) 2015–2021.

[60] P. Pineau, S. Volinia, K. McJunkin, A. Marchio, C. Battiston, B. Terris, V. Mazzaferro, S.W. Lowe, C.M. Croce, A. Dejean, miR-221 overexpression contributes to liver tumorigenesis, Proc. Natl. Acad. Sci. U. S. A. 107 (2010) 264–269.

[61] C.J. Guo, Q. Pan, T. Cheng, B. Jiang, G.Y. Chen, D.G. Li, Changes in microRNAs associated with hepatic stellate cell activation status identify signaling pathways, FEBS J. 276 (2009) 5163–5176.

[62] C. Braconi, T. Patel, MicroRNA expression profiling: a molecular tool for defining the phenotype of hepatocellular tumors, Hepatology 47 (2008) 1807–1809.

[63] Y. Gielchinsky, D. Elstein, I. Hadas-Halpern, A. Lahad, A. Abrahamov, A. Zimran, Is there a correlation between degree of splenomegaly, symptoms and hypersplenism? A study of 218 patients with Gaucher disease, Br. J. Haematol. 106 (1999) 812–816.

[64] Z. Li, S. Zhang, C. Huang, W. Zhang, Y. Hu, B. Wei, MicroRNAome of splenic macrophages in hypersplenism due to portal hypertension in hepatitis B virus-related cirrhosis, Exp. Biol. Med. 233 (2008) 1454–1461.

[65] L. Yang, Z. Ma, D. Wang, W. Zhao, L. Chen, G. Wang, MicroRNA-602 regulating tumor suppressive gene RASSF1A is overexpressed in hepatitis B virus-infected liver and hepatocellular carcinoma, Cancer Biol. Ther. 9 (2010) 803–808.

[66] Y. Liu, J.J. Zhao, C.M. Wang, M.Y. Li, P. Han, L. Wang, Y.Q. Cheng, F. Zoulim, X. Ma, D.P. Xu, Altered expression profiles of microRNAs in a stable hepatitis B virus-expressing cell line, Chin. Med. J. (Engl.) 122 (2009) 10–14.

[67] M. Lagos-Quintana, R. Rauhut, A. Yalcin, J. Meyer, W. Lendeckel, T. Tuschl, Identification of tissue-specific microRNAs from mouse, Curr. Biol. 12 (2002) 735–739.

[68] J. Chang, E. Nicolas, D. Marks, C. Sander, A. Lerro, M.A. Buendia, C. Xu, W.S. Mason, T. Moloshok, R. Bort, K.S. Zaret, J.M. Taylor, miR-122, a mammalian liver-specific microRNA, is processed from hcr mRNA and may downregulate the high affinity cationic amino acid transporter CAT-1, RNA Biol. 1 (2004) 106–113.

[69] S. Bai, M.W. Nasser, B. Wang, S.H. Hsu, J. Datta, H. Kutay, A. Yadav, G. Nuovo, P. Kumar, K. Ghoshal, MicroRNA-122 inhibits tumorigenic properties of hepatocellular carcinoma cells and sensitizes these cells to sorafenib, J. Biol. Chem. 284 (2009) 32015–32027.

[70] C. Coulouarn, V.M. Factor, J.B. Andersen, M.E. Durkin, S.S. Thorgeirsson, Loss of miR-122 expression in liver cancer correlates with suppression of the hepatic phenotype and gain of metastatic properties, Oncogene 28 (2009) 3526–3536.

[71] C. Esau, S. Davis, S.F. Murray, X.X. Yu, S.K. Pandey, M. Pear, L. Watts, S.L. Booten, M. Graham, R. McKay, A. Subramaniam, S. Propp, B.A. Lollo, S. Freier, C.F. Bennett, S. Bhanot, B.P. Monia, miR-122 regulation of lipid metabolism revealed by in vivo antisense targeting, Cell Metab. 3 (2006) 87–98.

[72] S.N. Bhattacharyya, R. Habermacher, U. Martine, E.I. Closs, W. Filipowicz, Relief of microRNA-mediated translational repression in human cells subjected to stress, Cell 125 (2006) 1111–1124.

[73] N. Nasheri, R. Singaravelu, M. Goodmurphy, R.K. Lyn, J.P. Pezacki, Competing roles of microRNA-122 recognition elements in hepatitis C virus RNA, Virology 410 (2010) 336–344.

[74] W.C. Tsai, P.W. Hsu, T.C. Lai, G.Y. Chau, C.W. Lin, C.M. Chen, C.D. Lin, Y.L. Liao, J.L. Wang, Y.P. Chau, M.T. Hsu, M. Hsiao, H.D. Huang, A.P. Tsou, MicroRNA-122, a tumor suppressor microRNA that regulates intrahepatic metastasis of hepatocellular carcinoma, Hepatology 49 (2009) 1571–1582.

[75] F. Meng, R. Henson, H. Wehbe-Janek, K. Ghoshal, S.T. Jacob, T. Patel, MicroRNA-21 regulates expression of the PTEN tumor suppressor gene in human hepatocellular cancer, Gastroenterology 133 (2007) 647–658.

[76] E. Connolly, M. Melegari, P. Landgraf, T. Tchaikovskaya, B.C. Tennant, B.L. Slagle, L.E. Rogler, M. Zavolan, T. Tuschl, C.E. Rogler, Elevated expression of the miR-17-92 polycistron and miR-21 in hepadnavirus-associated hepatocellular carcinoma contributes to the malignant phenotype, Am. J. Pathol. 173 (2008) 856–864.

[77] Y. Tomimaru, H. Eguchi, H. Nagano, H. Wada, A. Tomokuni, S. Kobayashi, S. Marubashi, Y. Takeda, M. Tanemura, K. Umeshita, Y. Doki, M. Mori, MicroRNA-21 induces resistance to the anti-tumour effect of interferon-alpha/5-fluorouracil in hepatocellular carcinoma cells, Br. J. Cancer 103 (2010) 1617–1626.

[78] A.P. McCaffrey, RNA interference inhibitors of hepatitis B virus, Ann. N.Y. Acad. Sci. 1175 (2009) 15–23.

[79] D. Grimm, K.L. Streetz, C.L. Jopling, T.A. Storm, K. Pandey, C.R. Davis, P. Marion, F. Salazar, M.A. Kay, Fatality in mice due to oversaturation of cellular microRNA/ short hairpin RNA pathways, Nature 441 (2006) 537–541.

[80] K. Keck, E.M. Volper, R.M. Spengler, D.D. Long, C.Y. Chan, Y. Ding, A.P. McCaffrey, Rational design leads to more potent RNA interference against hepatitis B virus: factors affecting silencing efficiency, Mol. Ther. 17 (2009) 538–547.

[81] A. Ely, T. Naidoo, S. Mufamadi, C. Crowther, P. Arbuthnot, Expressed anti-HBV primary microRNA shuttles inhibit viral replication efficiently in vitro and in vivo, Mol. Ther. 16 (2008) 1105–1112.

[82] L. Xiangji, X. Feng, C. Qingbao, T. Weifeng, J. Xiaoqing, Z. Baihe, S. Feng, W. Hongyang, W. Mengchao, Knockdown of HBV surface antigen gene expression by a lentiviral microRNA-based system inhibits HBV replication and HCC growth, J. Viral Hepat. (2010) [Epub ahead of print].

[83] P. Johansson, T. Lindgren, M. Lundstrom, A. Holmstrom, F. Elgh, G. Bucht, PCR-generated linear DNA fragments utilized as a hantavirus DNA vaccine, Vaccine 20 (2002) 3379–3388.

[84] L. Ma, J. Teruya-Feldstein, R.A. Weinberg, Tumour invasion and metastasis initiated by microRNA-10b in breast cancer, Nature 449 (2007) 682–688.

[85] J. Cheng, L. Zhou, Q.F. Xie, H.Y. Xie, X.Y. Wei, F. Gao, C.Y. Xing, X. Xu, L.J. Li, S.S. Zheng, The impact of miR-34a on protein output in hepatocellular carcinoma HepG2 cells, Proteomics 10 (2010) 1557–1572.

[86] N. Li, H. Fu, Y. Tie, Z. Hu, W. Kong, Y. Wu, X. Zheng, miR-34a inhibits migration and invasion by down-regulation of c-Met expression in human hepatocellular carcinoma cells, Cancer Lett. 275 (2009) 44–53.

[87] M. Girard, E. Jacquemin, A. Munnich, S. Lyonnet, A. Henrion-Caude, miR-122, a paradigm for the role of microRNAs in the liver, J. Hepatol. 48 (2008) 648–656.

[88] N. Nishida, K. Mimori, M. Fabbri, T. Yokobori, T. Sudo, F. Tanaka, K. Shibata, H. Ishii, Y. Doki, M. Mori, MicroRNA-125a-5p is an independent prognostic factor in gastric cancer, and inhibits the proliferation of human gastric cancer cells in combination with trastuzumab, Clin. Cancer Res. (2011) [Epub ahead of print].

[89] Y. Murakami, H. Toyoda, M. Tanaka, M. Kuroda, Y. Harada, F. Matsuda, A. Tajima, N. Kosaka, T. Ochiya, K. Shimotohno, The progression of liver fibrosis is related with overexpression of the miR-199 and 200 families, PLoS One 6 (2011) e16081.

[90] H. Yang, T.W. Li, J. Peng, X. Tang, K.S. Ko, M. Xia, M.A. Aller, A mouse model of cholestasis-associated cholangiocarcinoma and transcription factors involved in progression, Gastroenterology (2011) [Epub ahead of print].

[91] L. Gramantieri, F. Fornari, M. Ferracin, A. Veronese, S. Sabbioni, G.A. Calin, G.L. Grazi, C.M. Croce, L. Bolondi, M. Negrini, MicroRNA-221 targets Bmf in hepatocellular carcinoma and correlates with tumor multifocality, Clin. Cancer Res. 15 (2009) 5073–5081.

[92] Q.W. Wong, R.W. Lung, P.T. Law, P.B. Lai, K.Y. Chan, K.F. To, N. Wong, MicroRNA-223 is commonly repressed in hepatocellular carcinoma and potentiates expression of Stathmin1, Gastroenterology 135 (2008) 257–269.

[93] Q. Li, G. Wang, J.L. Shan, Z.X. Yang, H.Z. Wang, J. Feng, J.J. Zhen, C. Chen, Z.M. Zhang, W. Xu, X.Z. Luo, D. Wang, MicroRNA-224 is upregulated in HepG2 cells and involved in cellular migration and invasion, J. Gastroenterol. Hepatol. 25 (2010) 164–171.
