
Genetic Characterization of Measles Vaccine Strains

Bettina Bankamp,$^{1}$ Makoto Takeda,$^{2}$ Yan Zhang,$^{3}$ Wenbo Xu,$^{3}$ and Paul A. Rota$^{1}$

$^{1}$Division of Viral Diseases, Centers for Disease Control and Prevention, Atlanta; $^{2}$Department of Virology III, National Institute of Infectious Diseases, Tokyo, Japan; and $^{3}$National Institute for Viral Disease Control and Prevention, China Centers for Disease Control and Prevention, Beijing, China

The complete genomic sequences of 9 measles vaccine strains were compared with the sequence of the Edmonston wild-type virus. AIK-C, Moraten, Rubeovax, Schwarz, and Zagreb are vaccine strains of the Edmonston lineage, whereas CAM-70, Changchun-47, Leningrad-4 and Shanghai-191 were derived from 4 different wild-type isolates. Nucleotide substitutions were found in the noncoding regions of the genomes as well as in all coding regions, leading to deduced amino acid substitutions in all 8 viral proteins. Although the precise mechanisms involved in the attenuation of individual measles vaccines remain to be elucidated, in vitro assays of viral protein functions and recombinant viruses with defined genetic modifications have been used to characterize the differences between vaccine and wild-type strains. Although almost every protein contributes to an attenuated phenotype, substitutions affecting host cell tropism, virus assembly, and the ability to inhibit cellular antiviral defense mechanisms play an especially important role in attenuation.

Measles is a highly contagious disease characterized by fever, malaise, coryza, conjunctivitis, cough, and an erythematous maculopapular rash [1]. Although most patients recover from the illness, serious complications can occur, including pneumonia and invasion of the central nervous system [2, 3]. One severe complication is subacute sclerosing panencephalitis (SSPE), a rare, fatal consequence of central nervous system infection [4]. Measles caused an estimated 164,000 deaths worldwide in 2008 [5]. This number represents a 97% decrease from the estimated 6 million deaths occurring annually before the introduction of measles vaccines [6].

Measles virus (MeV) was first isolated from a child with measles in a primary culture of human kidney cells [7]. This isolate, the prototype Edmonston strain, was subsequently adapted to various types of cultured cells, giving rise to several of the currently used highly attenuated, live measles vaccines [8]. The first live, attenuated vaccine, Edmonston B, was licensed in the United States under the trade name Rubeovax in 1963 [9]. It was eventually replaced by the more attenuated Moraten strain [10]. On the basis of phylogenetic analyses, MeV isolates can be assigned to 1 of 23 recognized genotypes and 1 provisional genotype [11, 12]. All vaccine strains are grouped in genotype A, possibly because genotype A was the most widely distributed genotype at the time of isolation of the vaccines' wild-type progenitors [13].

Potential conflicts of interest: none reported.
Supplement sponsorship: This article is part of a supplement entitled "Global Progress Toward Measles Eradication and Prevention of Rubella and Congenital Rubella Syndrome," which was sponsored by the Centers for Disease Control and Prevention.
Disclaimer: The findings and conclusions in this article are those of the authors and do not necessarily represent the views of the Centers for Disease Control and Prevention.
Correspondence: Bettina Bankamp, PhD, Div of Viral Diseases, Centers for Disease Control and Prevention, 1600 Clifton Road, MS C-22, Atlanta, GA 30333 (bbankamp@cdc.gov).

MOLECULAR BIOLOGY OF MeV

MeV, an enveloped virus with a single-stranded, negative sense RNA genome, is a member of the genus *Morbillivirus* in the family *Paramyxoviridae* (for review see [2]). The genome is a 15,894-nucleotide, nonsegmented RNA molecule of negative polarity. It contains 6 transcription units which are separated by nontranscribed intergenic sequences consisting of 3 nucleotides. The 6 genes (Figure 1) encode 8 proteins. In addition to the 6

structural proteins described in Figure 1, 2 nonstructural proteins are encoded by the gene for the phosphoprotein (P). The C protein is translated from an overlapping reading frame in the P gene [14], whereas the V protein is the product of an edited mRNA transcribed from the P gene [15]. The edited message has a nontemplated G residue inserted at genomic nucleotide 2499. As a result, P and V share an amino-terminal domain of 231 amino acids but have unique carboxyl terminal domains of 276 and 69 amino acids, respectively.

## The Functions of the MeV Proteins

The most abundant structural polypeptide, the nucleoprotein (N; 525 amino acids), is essential for packaging the genome into a ribonucleoprotein complex (RNP) that serves as a template for transcription, replication and is packaged into progeny virions [2]. The amino-terminal region of the N protein (N_CORE; amino acids 1–400) forms the core of the helical nucleocapsid, whereas the carboxyl-terminal region (N_TAIL; amino acids 401–525) constitutes a disordered domain located outside of the core [16]. This region is hypervariable among wild-type viruses, and the nucleotide sequence of this region is used for genotype assignment [17, 18]. The N_TAIL region interacts with the P protein [19, 20] and the M protein [21]. Two leucine residues at amino acid positions 523 and 524 are required for the interaction with the M protein [21]. The N protein also interacts with multiple cellular proteins [22–25]. The major inducible 70-kDa heat shock protein, hsp72, promotes MeV RNA synthesis and virus propagation by interacting with the N_TAIL region [26–28]. An asparagine residue at amino acid position 522 (Asn522) in the N_TAIL region is important for the interaction with hsp72 [29].

The phosphoprotein (P; 507 amino acids) is a subunit of the viral RNA polymerase and acts as a chaperone that interacts with and regulates the cellular localization of N protein and probably assists in nucleocapsid assembly [20, 30, 31]. The P protein forms oligomers and binds the N protein and the polymerase protein. A domain in the amino-terminus of P is required for interactions with soluble N protein, whereas the unique carboxyl-terminus contains the self-assembly domain and binding sites for the nucleocapsid and the polymerase [2, 20, 30, 32]. The P, V (299 amino acids), and C (186 amino acids) proteins all play roles in antagonizing the host interferon (IFN) responses [33]. The V protein blocks the Jak/STAT signaling pathway and counteracts cellular IFN signaling [34–39]. It also blocks the MDA5-mediated IFN induction pathway [40–42]. Direct interference with IFN signaling by the P and C proteins has also been reported [36, 43, 44]. The C protein controls the levels of viral RNA synthesis [45, 46] to circumvent IFN induction [35, 42, 47]. The V protein binds RNA and can also regulate viral RNA synthesis [48, 49].

The matrix protein (M; 335 amino acids) lines the inner surface of the viral envelope and participates in virion maturation [2, 50]. It plays a crucial role in virus assembly by interacting with the cytoplasmic tails of the H and F proteins [51–53]. It binds the nucleocapsid and negatively regulates transcription, presumably by sequestering nucleocapsids to the plasma membrane [21, 54, 55].

The fusion protein (F) is translated as a precursor of 550 amino acids that is cleaved by cellular furin-like proteases into disulfide-linked F1 and F2 proteins. It is a cell surface-expressed, type I glycoprotein that mediates fusion with the host cell at neutral pH [2, 56]. Three N-linked glycosylation sites are located in the F2 protein at amino acids 29, 61, and 67 [57]. Amino acids 113–145, located at the new amino-terminus of F1, form the hydrophobic fusion peptide [58]. This domain inserts into the membrane of the host cell during the fusion process [59]. A cytoplasmic tail of 33 amino acids at the carboxyl terminus of

![Figure 1. Measles virion and genome. A, Schematic representation of a measles virion. B, Organization of the measles genome. Colored rectangles represent coding regions, and white rectangles represent noncoding regions. Vertical black bars represent intergenic trinucleotides. The size of the individual regions is not to scale.](image.png)

**Figure 1.** Measles virion and genome. *A*, Schematic representation of a measles virion. *B*, Organization of the measles genome. Colored rectangles represent coding regions, and white rectangles represent noncoding regions. Vertical black bars represent intergenic trinucleotides. The size of the individual regions is not to scale.
F1 interacts with the M protein and contains sorting signals for intracellular localization [60].

The hemagglutinin (H; 617 amino acids) protein is the second glycosylated surface-expressed protein. This type 2 glycoprotein mediates attachment to specific protein receptors on the host cell and is a required co-factor for fusion [2]. There are 4 N-linked glycosylation sites between amino acids 168 and 200 [61], and some wild-type isolates are additionally glycosylated at amino acid 416 [62]. A cytoplasmic tail of 34 amino acids at the amino-terminus interacts with the M protein and contains sorting signals for intracellular localization [63, 64]. Two receptors have been identified for MeV [65]. hSLAM (human signaling lymphocyte activation molecule, CD150) is a common receptor for all MeV strains [66], whereas CD46 (membrane cofactor protein, MCP) functions as a receptor for vaccines and some laboratory-adapted strains, but not for wild-type strains [67, 68]. hSLAM is only expressed on a subset of immune cells [69–71]. All vaccine strains have acquired the ability to use CD46 as an alternative receptor during passage in various SLAM-negative cultured cells [65]. The ubiquitous expression of CD46 explains the ability of vaccine strains to infect various human and monkey cell lines. Recent studies have revealed that another, presently unidentified receptor exists for wild-type MeV on epithelial cells that form tight junctions (polarized epithelial cells) [72–76], and this as-yet uncharacterized receptor is referred to as “EpiR” in this review. The H protein is the major target for neutralizing antibodies induced by vaccination [77].

The Large gene encodes the multifunctional catalytic subunit of the RNA dependent RNA polymerase (L; 2183 amino acids), which is responsible for both transcription and replication [2, 31]. It is thought to carry out most (if not all) enzymatic activities required for transcription and replication, including nucleotide polymerization, mRNA capping, and polyadenylation [78–80]. On the basis of amino acid sequence conservation among nonsegmented negative strand viruses, 6 domains of higher conservation have been characterized in the L protein [81]. Within these domains, several functional motifs have been identified. The conserved domains may represent concatenated functional domains that are connected by less-conserved hinge regions [81]. L forms oligomers and binds its co-factor, the P protein. Binding sites for these interactions have been identified [82, 83]. Together with the genome, the N, P, and L proteins constitute the transcriptionally active RNP.

### The Roles of Noncoding Regions

Nearly 11% of the 16-kb MeV genome is composed of noncoding RNA (Figure 1). The 6 transcription units are preceded by a 3′ leader and are followed by a 5′ trailer. All cis-acting regulatory elements such as promoters and encapsidation signals are contained within the first 107 and the last 109 nucleotides of the genome [84, 85]. Each gene contains 5′ and 3′ noncoding

regions with conserved start and end signals for transcription and polyadenylation [86]. Morbillivirus genomes are unique within the Paramyxoviridae in that they contain a very long noncoding region of ~1000 nucleotides between the M and F open-reading frame (ORF) [56, 87]. The region is GC rich and is likely to fold into complex secondary RNA structures. The function of this region is not well understood, but it may play a role in regulating transcription of the M and F gene [88–90].

### Measles Virus Replication

MeV attaches to the host cell through the interaction of the viral H protein with a cellular receptor that may be hSLAM, CD46 or the as-yet unidentified EpiR [65]. After the F protein mediates fusion of the virion with the cell membrane, the negative-stranded RNP is introduced into the cytoplasm, where it acts as a template for both primary transcription of mRNAs and replication into positive-stranded antigenomic RNA (for a review, see Lamb and Parks [91]). The polymerase complex, consisting of L and P proteins, is delivered with the RNP. mRNA transcripts are capped, polyadenylated and terminated at the gene end signals [92]. The full-length positive sense antigenomic intermediate acts as a template for the replication of full-length, negative sense genomic RNA. During replication the gene end, intergenic trinucleotide and gene start signals are ignored, and both nascent positive and negative sense genomes are concomitantly encapsidated to produce RNPs [93]. F and H proteins insert into the plasma membrane, whereas M proteins line the inside of the membrane and interact with the cytoplasmic tails of the glycoproteins and the RNP [50, 52]. The virion buds from the plasma membrane to complete the replication cycle.

### ORIGINS AND PASSAGE HISTORIES OF VACCINE STRAINS

The first complete genomic sequence of a live, attenuated measles vaccine strain was published 1993 for the strain AIK-C [94] and was later updated by Parks et al [95, 96]. Since then, the genomic sequences of 8 additional vaccine strains have become available from GenBank (for accession numbers, see Figure 3). Sequence data were reanalyzed for this review; however, many of the findings were previously published [95–98]. Figure 2 summarizes the passage histories of the measles vaccines. Several of these strains were developed from the Edmonston isolate [7], whereas others were derived from wild-type progenitors isolated independently in Russia (Leningrad-4), Japan (CAM-70), and China (Shanghai-191) [99–101]. Unfortunately, the original Edmonston isolate is no longer available for analysis. Instead, vaccine sequences are compared with the Edmonston wild-type strain, which has undergone 13 passages in cell culture [8]. The AIK-C strain is used in Japan [102], where it was developed from the Edmonston strain [103]. The Zagreb strain (also referred to

Sequences of Measles Vaccines • JID 2011:204 (Suppl 1) • S535
Expanded Programme on Immunization of the World Health Organization [107]. Edmonston B (Rubeovax) was developed from the Edmonston isolate by passaging in primary human and avian cell culture systems [108–111]. Although the vaccine induced excellent seroconversion rates, it also induced fever in 46% of vaccines [9] and was replaced with further attenuated strains [9]. The Schwarz vaccine strain was generated in 1962 by additional passaging of the Edmonston strain in chick embryo fibroblasts [112]. The strain is produced by manufacturers in Europe and Brazil [113]. Moraten was also generated by additional passaging of the attenuated Edmonston-Enders virus in chick embryo fibroblasts [9] and it is used in the United States as Attenuvax, Merck & Co., [113]. Leningrad-4 was derived from an isolate obtained in 1957 in St Petersburg (formerly Leningrad), Russia [99]. Like Edmonston B, it proved to be insufficiently attenuated [114]. Further passaging led to the development of the Chinese vaccine strain Changchun-47 [115]. Shanghai-191 was developed from an isolate obtained in Shanghai in 1960 and attenuated through passages in human and avian cell culture systems [101]. It has been used continually since 1966 [98]. CAM-70 was derived from the Japanese wild-type isolate, Tanabe, by serial passaging in the chorioallantoic membrane of chickens [100], followed by passaging in chick embryo fibroblasts [116, 117]. CAM-70 is produced in Japan and Indonesia [113].

The first study comparing the full-length genomic sequences of the Edmonston-derived vaccine strains AIK-C, Moraten, Rubeovax, Schwarz, and Zagreb with the Edmonston wild-type strain was published in 2001 [95, 96]. It was followed by the sequences of independently derived CAM-70, Shanghai-191, and Changchun-47 [97, 98]. The sequence of Leningrad-4 was submitted to GenBank in 2004 by D. Zhao but was not published. Although the propagation of the Tanabe progenitor strain and the sequence of its H gene were reported recently [118], the remainder of the sequence of the genome of the Tanabe strain remains unavailable for comparison. The sequences of several vaccine strains that are still in use have not been determined yet. Among these are Leningrad-16, a vaccine strain derived from a Russian isolate [119, 120]; Schwarz F88, a Japanese vaccine derived by further passaging of the Schwarz strain [113]; and TD97, another Japanese vaccine that was generated from the Tanabe isolate [113].

## COMPARATIVE SEQUENCE ANALYSIS

### Phylogenetic Analysis

The total number of nucleotide changes compared with Edmonston wild-type is 29 for Leningrad-4; 33 for Zagreb; 36 for Rubeovax; 42 for AIK-C, Moraten, and Schwarz; 43 for Changchun-47; 50 for Shanghai-191; and 73 for CAM-70. Of the 3 viruses with the smallest number of substitutions, 2 (Leningrad-4 and Rubeovax) were insufficiently attenuated.
Clearly, the absolute number of substitutions alone does not signify attenuation, because the fully attenuated Zagreb strain has fewer substitutions, compared with Edmonston wild-type than Rubeovax. Figure 3 demonstrates that the 2 most divergent strains are Shanghai-191 and CAM-70 (89 nucleotide differences from each other or 0.6% of the genome), which were derived from unique wild-type progenitors. These 2 strains also have the largest number of nucleotide differences from Edmonston wild-type. This degree of divergence is likely due to their generation from non-Edmonston progenitors; however, this interpretation cannot be tested until the sequences of their progenitors become available. The most closely related strains are Rubeovax and Moraten or Rubeovax and Schwarz (16 nucleotide differences, respectively). A surprising finding was that Moraten and Schwarz have identical nucleotide sequences, despite their divergent passage histories [95, 96]. Because both viruses have been passaged in CEF at reduced temperatures (Figure 2), it is possible that similar cell culture conditions may have resulted in similar nucleotide substitutions. Despite the diverse geographic origins of the progenitors and the variations in cell culture systems, incubation temperatures, and passage numbers, the genomes of vaccines demonstrate remarkable sequence similarity. Phylogenetic analysis (Figure 3) indicated that Changchun-47 could have been derived from Leningrad-4, which is consistent with the strain's reported passage history [115]. The increased number of nucleotide substitutions in CAM-70 is distributed throughout the coding sequences of the genome rather than the noncoding sequences (Figure 4). All genes—but especially the P/C/V gene—are affected, whereas the variability of the M gene is similar to that of the other vaccines. We performed an analysis of synonymous versus nonsynonymous substitution rates [121], but the results were meaningless for many ORFs that had either no synonymous or no nonsynonymous substitutions (eg, in the C and H ORFs) (Figure 4).

### Variability in Individual Proteins

Table 1 lists the nucleotide and amino acid substitutions that are specifically discussed in the text. For a complete list of substitutions in the genomes of measles vaccines, see Supplemental Table 1. With the exception of Shanghai-191, the N genes of the vaccine strains show much higher conservation than the P/V/C, M, and H genes. Comparable to the situation in wild-type isolates [122], $N_{\text{CORE}}$ is much more conserved among the vaccines (only 4 cumulative nucleotide changes in the region coding for the amino terminal 400 amino acids for all the vaccines) than $N_{\text{TAIL}}$ (12 nucleotide changes in the region coding for the carboxyl terminal 125 amino acids) (Supplemental Table 1). Although the number of substitutions in Shanghai-191 is generally similar to that of other strains (except CAM-70), the N gene of Shanghai-191 is considerably more divergent than the N genes of all other viruses (Figure 4). The reason for the variability of the Shanghai-191N gene is that it accumulated 8 T→C changes, all of which are unique to Shanghai-191 (Supplemental Table 1). All of the T→C changes are located in the hypervariable $N_{\text{TAIL}}$ region, and 2 of them predict substitution with Pro residues that may affect protein conformation. The T→C changes may be an example of hypermutation in a vaccine strain. Hypermutation has been described for sequences of M genes derived from patients with SSPE [123]. The effect has been attributed to the action of the

![Figure 4](#fig4)

**Figure 4.** Number of nucleotide and predicted amino acid changes in the coding and noncoding regions of measles vaccine strains, compared with Edmonston wild-type strain. The number of amino acids is listed in brackets behind each protein. Horizontally striped bars represent coding changes, and open bars represent noncoding or silent changes. NCR, noncoding region.

Sequences of Measles Vaccines • JID 2011:204 (Suppl 1) • S537
Table 1. Nucleotide and Amino Acid Substitutions Discussed in the Text

| Gene | Nucleotide Edmonston wild-type | Vaccine Amino acid position | Edmonston wild-type | ALK-C | Mor-Sch | Rub | Zag | CAM-70L-4S-191C-47 |
|------|-------------------------------|-----------------------------|--------------------|-------|--------|-----|-----|---------------------|
|      | 26                            | A                           | T                  | ●     | ●      | ●   | ●   | ●                   |
|      | 42                            | A                           | C                  | ●     | ●      | ●   | ●   | ●                   |
| N    | 722                           | A                           | G                  |       |        |     | ●   |                    |
| P    | 1806                          | G                           | A                  |       |        | ●   | ●   | ●                   |
|      | 2046                          | C                           | T                  | ●     | ●      | ●   | ●   | ●                   |
|      | 2063                          | G                           | T                  | 86    | S      |     |     |                     |
|      | 2101                          | A                           | G                  | 99    | N      |     |     | D                   |
|      | 2134                          | T                           | C                  | 110   | Y      |     |     | H                   |
|      | 2135                          | A                           | G                  | 110   | Y      |     |     | C                   |
|      | 2167                          | A                           | G                  | 121   | K      |     |     | E                   |
|      | 2480                          | A                           | G                  | 225   | E      | G   | G   | G G G G G G        |
|      | 3122                          | T                           | C                  | 439   | L      |     |     | P                   |
| V    | 2046                          | C                           | T                  |       |        | ●   | ●   | ● ● ● ● ● ●        |
|      | 2063                          | G                           | T                  | 86    | S      |     |     |                     |
|      | 2101                          | A                           | G                  | 99    | N      |     |     | D                   |
|      | 2134                          | T                           | C                  | 110   | Y      |     |     | H                   |
|      | 2135                          | A                           | G                  | 110   | Y      |     |     | C                   |
|      | 2167                          | A                           | G                  | 121   | K      |     |     | E                   |
|      | 2480                          | A                           | G                  | 225   | E      | G   | G   | G G G G G G        |
| C    | 2046                          | C                           | T                  | 73    | A      | V   | V   | V V V V V V        |
| M    | 3619                          | G                           | A                  | 61    | G      | D   | D   | D D D D D D        |
|      | 3627                          | C                           | T                  | 64    | P      | S   | S   | S                   |
|      | 3702                          | G                           | A                  | 89    | E      | K   | K   | K K K K K K        |
|      | 4232                          | G                           | A                  |       |        | ●   | ●   | ● ● ● ● ● ●        |
|      | 4608                          | T                           | C                  |       |        | ●   | ●   | ●                   |
| F    | 4978                          | T                           | C                  |       |        | ●   | ●   | ● ● ● ● ● ●        |
|      | 5073<sup>a</sup>               | C                           | G                  |       |        | ●   | ●   | ● ● ● ● ● ●        |
|      | 5308                          | T                           | C                  |       |        | ●   |     |                     |
|      | 5349                          | T                           | C                  |       |        | ●   | ●   | ● ● ● ● ● ●        |
|      | 5514                          | G                           | A                  |       |        | ●   | ●   | ● ● ● ● ● ●        |
|      | 6542                          | C                           | A/T<sup>b</sup>    | 362   | S      | Y(A) | Y(A) | Y(A) F(T)           |
|      | 6743                          | C                           | T                  | 429   | T      |     |     |                     |
|      | 7243                          | T                           | C                  |       |        | ●   |     |                     |
| H    | 7407                          | C                           | T                  | 46    | S      | F   | F   | F F F F F F        |
|      | 7901                          | A                           | G                  | 211   | S      | G   | G   | G G G G G G        |
|      | 8711                          | A                           | T                  | 481   | N      | Y   | Y   | Y Y Y Y Y Y        |
|      | 8721                          | C                           | A                  | 484   | T      | N   | N   | N N N               |
|      | 8906                          | G                           | A                  | 546   | G      | S   | S   | S S S               |
| L    | 9936                          | A                           | G                  | 235   | I      |     |     | V                   |
|      | 12,600                        | A                           | C                  |       |        | ●   |     |                     |
|      | 14,383                        | A                           | C                  | 1717  | D      | A   | A   | A A A A A A        |
|      | 14,579                        | A                           | G                  |       |        | ●   | ●   | ● ● ● ● ● ●        |

NOTE. For abbreviations of virus names, see Figure 3. Dashes denote noncoding region, empty fields denote no change, compared with Edmonston wild-type, and dots denote substitution in noncoding region. Nucleotides highlighted in bold indicate a substitution in at least 8 vaccine strains.

<sup>a</sup> Correction to GenBank entry: Nucleotide 5073 in the GenBank entry for Edmonston wild-type (AF266288) is a G; the nucleotide was determined to be a C, as reported elsewhere [95] (B. Bankamp, unpublished data).

<sup>b</sup> Nucleotide positions with a mix of substitutions list the nucleotide in brackets behind the amino acid. For a list of all nucleotide and amino acid substitutions, please see Supplemental Table 1.
The Leucine residues that are important for the interaction of N with the M protein [21] are conserved in all strains. The N<sub>TAIL</sub> regions of vaccine strains and Edmonston wild-type possess Asn522, whereas currently circulating wild-type strains mostly possess Asp522 [29]. Asn522 is important for interaction with heat shock protein 72 (hsp72) [29]. This difference appears to reflect differences between genotypes rather than between vaccine and wild-type strains, because other genotype A, wild-type isolates share an Asn at position 522 (GenBank accession numbers AF045218 and AY647965). Although a recombinant MeV possessing Asp522 demonstrated reduced fitness in cell culture and cotton rats [29], the fact that current patient isolates share the Asp at this position indicates that these model systems may not reproduce all fitness pressures that wild-type viruses experience in the natural host. In addition to hsp72 and the M protein, N<sub>TAIL</sub> interacts with other binding partners, including several cellular proteins; however, no specific amino acids that are important for the interactions have been identified.

Nucleotide substitutions are often observed in the P gene after passages of MeV in cultured cells, and the P genes of wild-type viruses demonstrate a higher level of variability than the corresponding N genes [126–131]. Remarkably, 35 of 42 cumulative changes in the P gene are located in the amino-terminal domain shared between the P and V proteins (Supplemental Table 1). The carboxyl termini of both P and V are well conserved. These observations agree with data from wild-type MeVs in that the shared amino terminal domain of V and P was found to be more variable than the carboxyl termini in wild-type MeV isolates [132]. The unique carboxyl termini of P and V are required for replication and for control of the innate immune response, respectively; the higher level of conservation indicates that these regions are less tolerant of amino acid substitutions. Two amino acid substitutions, Tyr110His and Cys272Arg, which are found in the common amino-terminal region of the P and V proteins and the unique carboxyl-terminal region of the V protein, respectively, disrupt the abilities of the V and P proteins to counteract the IFN signaling pathways [34, 36, 43, 133]. Adaptation of a wild-type virus to chick embryo fibroblasts introduced the Tyr110His substitution, indicating that this substitution may be important for improved growth in avian cells [127]. The P/V proteins of the Changchun-47 vaccine possess Tyr110His, and the P/V proteins of the CAM-70 vaccine possess Tyr110Cys. The latter substitution reduced the ability of the CAM-70 V protein to counteract IFN signaling [43]. That the same position was changed in 2 independently derived vaccine strains through different nucleotide substitutions may indicate the importance of impeding IFN resistance in vaccine attenuation. The P protein (and presumably also the V protein) is phosphorylated at several residues in its amino-terminal domain, including amino acids 86, 151, and 180 [134]. The P and V proteins of Shanghai-191 possess a Ser86Ile substitution, which would abolish phosphorylation of this residue. The effect

that this potential change in phosphorylation may have on the functions of P and V is currently unknown. One substitution that affects all vaccine strains is located at nucleotide 2480 in the P gene. This not only introduces a nonconservative Glu225Gly substitution at the end of the shared P/V domain—it also is located only 11 nucleotides upstream of the editing site. However, there was no significant difference in the frequency of P and V ORFs in transcripts from viruses with either nucleotide at position 2480 [132].

The P protein of CAM-70 varies from Edmonston wild-type at 11 amino acid positions and the V protein in 10 positions, while other vaccines vary at no more than 4 positions for the P protein and 3 for the V protein (Figure 4 and Supplemental Table 1). Many of these substitutions are nonconservative changes, including charge-switches at amino acids 99 and 121. Although the effect of the substitution at amino acid 110 in the P/V protein of CAM-70 on the inhibition of IFN signaling has been demonstrated [43], the contribution of the many other substitutions to attenuation remains to be investigated. The P protein of AIK-C renders the virus temperature-sensitive [135]; a Leu439Pro substitution was found to be one contributing factor [136]. None of the other vaccines share this substitution.

There are a cumulative total of 25 nucleotide substitutions and 18 amino acid changes in the small C ORF (186 amino acids), with a high proportion of coding changes (72%) (Supplemental Table 1). Such a high level of variability may indicate the importance of the C protein as a virulence factor. The C ORF of CAM-70 has 9 nucleotide changes, compared with only 0–3 in the other vaccines; however, the number of silent changes is high (Figure 4). As a result, the number of amino acid substitutions in CAM-70 C is 4, whereas the other vaccines have 0–2 substitutions. The high number of silent changes in the CAM-70 C ORF is a result of coding changes in P and V proteins. The C protein interferes with the induction of IFN and apoptosis through down-regulation of transcription [35, 42]. Using a mini-replicon assay system, Bankamp et al [45] reported that the inhibitory activity of the C protein toward viral RNA synthesis differed among 6 MeV strains and that substitutions in the region between amino acids 46 and 167 of the C protein affected its ability to down regulate viral transcription. All but 1 of the substitutions in the C proteins of the vaccine strains are located in this region (Supplemental Table 1), so it is possible that some of them may affect the ability of the C protein to control the induction of IFN. After adaptation of a wild-type virus to CEF, the C protein contained 1 amino acid change that affected its ability to down-regulate transcription [126], providing additional evidence of the importance of the C protein in cell-culture adaptation. Disturbing the delicate balance between the rate of replication and inhibition of host defenses may lead to attenuation. Because the C proteins are important virulence factors [90, 137–140], the differences among the C proteins may also contribute to the attenuated phenotypes of certain measles

Sequences of Measles Vaccines • JID 2011:204 (Suppl 1) • S539vaccines. However, the C proteins of some vaccine strains remain fully functional. Nakatsu et al [47] compared the activities of C proteins from wild-type and vaccine strains using reverse-genetics techniques and found that a C protein identical to that of Zagreb equally regulated viral RNA synthesis, compared with the wild-type C protein, and was capable of circumventing the host IFN induction. Clearly, impaired function of the C protein is not a prerequisite to attenuation. The Ala73Val substitution is shared by all vaccine strains, including those derived from non-Edmonston progenitors. However, wild-type viruses from genotypes B3 and D3 also have Val at this position [132], making it less likely that the substitution has consequences for attenuation.

The role of the P/V/C gene in attenuation is well documented with different animal models. Recombinant viruses that do not express a V or a C protein are attenuated in rhesus macaques [139, 141]. A CEF-adapted virus with mutations affecting the P, V and C proteins as well as a substitution in M did not cause rash in macaques [127]. Takeda et al [142] showed that a recombinant MeV possessing the P gene of the Edmonston-tag strain exhibited attenuated gene expression levels in both cultured cells and mice expressing the hSLAM receptor. In cotton rats, recombinant viruses carrying the temperature-sensitive P gene of AIK-C were unable to replicate after intranasal inoculation [143].

The M gene has 50 cumulative changes in only 1008 nucleotides, of which 39 (78%) are coding changes, indicating both a higher overall substitution rate and a higher proportion of coding changes than the N, F, or L genes (Supplemental Table 1). Tahara et al [144] showed that the M protein derived from the wild-type IC-B strain associated with the cytoplasmic tail of the F protein, but it associated poorly with the cytoplasmic tail of the H protein. They further indicated that Pro64Ser and Glu89Lys substitutions allow a strong interaction of the M protein with the cytoplasmic tail of the H protein [144]. All measles vaccines with published sequence data possess the Glu89Lys substitutions in the M protein. The independently derived CAM-70 vaccine shares the Pro64Ser substitution with the Edmonston-derived Rubeovax, Moraten, and Schwarz, indicating the importance of this change. Vero cells do not efficiently express entry receptors for wild-type MeV. These substitutions allow wild-type MeV to grow well in Vero cells, presumably by enhancing MeV assembly and subsequent infectious particle production [144]. On the other hand, Pro64Ser and Glu89Lys inhibit receptor-dependent cell-cell fusion, thereby reducing virus growth in hSLAM-positive cells [145]. These data indicate that the Pro64Ser and Glu89Lys substitutions modulate the mode of MeV growth for adaptation to grow in different cell types. However, the contribution of the substitution at amino acid 89 to attenuation is unclear, because a wild-type MeV with the Glu89Lys substitution grew to higher titers in cotton rats than did the wild-type. [146]. In addition to the Glu89Lys substitution, all vaccine strains share a Gly61Asp substitution that is close to the important position of

amino acid 64; however, no functional consequence is known for this substitution.

The F glycoprotein F is more conserved than the M or H genes, and the proportion of coding changes is lower (61%) (Supplemental Table 1). There are few differences between the F proteins of the prototype Edmonston strain and current wild-type strains [130, 131]. Importantly, the amino acid sequence of the F protein of the attenuated Zagreb strain is identical to the Edmonston wild-type F protein, indicating that large numbers of passages in various cell culture systems did not lead to adaptive changes. Furthermore, the Zagreb F protein is identical to the more recent wild-type IC-B isolate, a well-characterized virulent strain [131]. This demonstrates the high level of conservation in the F protein. One reported amino acid substitution that affects F protein function is Met94Val. F proteins possessing the Met94Val substitution exhibit enhanced cell-cell fusion activity [147–149]; however, Edmonston wild-type and all vaccine strains share the Met at this position. Edmonston-derived vaccines AIK-C, Schwarz, Moraten, and Rubeovax share a Ser362-Tyr substitution, whereas the independently derived CAM-70 has a Ser362Phe substitution. That the same position was changed in independently derived vaccine strains through different nucleotide substitutions may indicate the importance of this residue. Not surprisingly, important functional domains and motifs, such as the cytoplasmic tail, cleavage site, and fusion peptide, are completely conserved.

Similar to the M gene, the H gene has a very high ratio of coding changes (92%), indicating that these may be driven by selection. A large body of research is available for the study of receptor binding by measles H proteins [65]. Not surprisingly, all amino acid positions that are involved in binding to hSLAM and EpiR are conserved in Edmonston wild-type and all vaccine strains [72, 74, 150, 151]. Several amino acid positions that are involved in the ability of MeV strains to use CD46 have been reported [152–158]. Among these, the Asn481Tyr substitution is critical for use of CD46 as a receptor [152–154, 158–162]. The H proteins of all vaccine strains possess the Asn481Tyr substitution. However, this substitution alone does not confer efficient use of the CD46 receptor [163]. Tahara et al [164] showed that a Glu492Gly or Thr484Asn substitution, when present together with Asn481Tyr, enhances the ability of the wild-type H protein to use CD46. AIK-C, Zagreb, Leningrad-4, Shanghai-191, and Changchun-47 possess Thr484Asn, presumably enhancing their ability to bind CD46. Independently of Asn481Tyr, a Ser546Gly substitution can confer on MeV the ability to use CD46 as a receptor, and some Vero cell-grown MeV strains have the Ser546Gly substitution in their H proteins, instead of Asn481Tyr [129, 143, 156, 159, 160, 165–170]. The H protein of the Zagreb vaccine strain possesses Ser546Gly; however, so does the Edmonston wild-type. A recent study of the crystal structure of an MeV H protein complexed with CD46 also indicated critical roles of Asn481Tyr and Ser546Gly [151].

S540 • JID 2011:204 (Suppl 1) • Bankamp et al
Interestingly, the Asn481Tyr substitution that is critical for using CD46 also promotes MeV infection of polarized epithelial cells via a CD46-independent pathway [74]. Therefore, all vaccine strains which possess Asn481Tyr in their H proteins may be able to use EpiR more efficiently than can wild-type MeV strains. A Val451 has been reported to improve CD46 binding, and all vaccine strains and the Edmonston wild-type strain share this amino acid. The fact that the Edmonston wild-type strain possesses amino acids corresponding to the cell culture–adapted phenotype may be evidence of some degree of cell culture adaptation. A Gly211 was reported to be involved in CD46 receptor down-regulation [152]. All vaccines share Gly211, in contrast to the Edmonston wild-type strain, which has a non–down-regulating Ser211. Receptor down regulation may be associated with protection of the infected cell from complement lysis [171]. In total, there are 3 amino acid substitutions in the H protein that are shared by all vaccine strains (amino acids 46, 211, and 481) and 1 additional substitution shared by all but the Zagreb strain (amino acid 546). Three of these 4 positions may play a role in CD46 binding or down-regulation, demonstrating the importance of receptor adaptation for growth of vaccines in hSLAM-negative cell lines. The functional consequence of the nonconservative Ser46Phe, which is located in the hydrophobic transmembrane domain [172], is unknown. The H proteins of some currently circulating wild-type strains have an additional glycosylation site, owing to an Asp416Asn substitution [62, 169, 173]. Although the glycosylation site at this position is located on the opposite side to the CD46-binding region [174], it has also been shown to modulate the CD46-binding activity of the H protein [164]. However, none of the vaccine strains possess this substitution. The cytoplasmic tails of the H proteins (amino acids 1-34) are completely conserved.

Considering its large size, the L gene is considerably more conserved than the other genes, with only 75 cumulative changes in 6549 nucleotides, of which only 36 (48%) are coding changes (Supplemental Table 1). This observation is consistent with previous analyses [96, 175]. The conservation of the L gene demonstrates that substitutions in the polymerase are deleterious for the virus and therefore not tolerated. On the basis of sequence conservation, 6 domains have been characterized for the L proteins of paramyxoviruses [83]. Eleven of the 16 positions that vary among the vaccine strains are located between the conserved domains, and 10 of these affect residues that are not conserved among the polymerases of nonsegmented negative strand viruses [81]. Of the 5 substitutions that affect residues within conserved domains, 3 affect residues that are conserved among polymerases. Ile235Val lies in a highly conserved hydrophobic stretch in domain I; however, it is a substitution that conserves the hydrophobic nature of the residue. Aside from this substitution, all highly conserved stretches and motifs [81] are completely conserved. One substitution (Asp1717Ala) is shared by all vaccine strains; however, several other wild-type strains in

genotype A also have an Ala at this position [175]. Therefore, this is more likely to be a genotype-specific residue with Edmonston wild-type being the exception. Amino acids 1-408 of L are involved in L oligomerization, and hydrophobic amino acids between positions 25 and 339 are involved in binding to the P protein [82]. Four residues in this region are variable between vaccine L proteins, and 3 of these substitutions affect CAM-70 (Supplemental Table 1). Because interactions between the components of the polymerase complex are essential for virus replication, clearly these substitutions cannot inhibit L-L or L-P interactions. Whether they have subtle effects on these interactions is currently unknown. On the basis of investigations with recombinant MeV, the L gene of an attenuated laboratory strain contributed to the reduced gene expression and virus propagation in cell culture and in mice expressing hSLAM [142]. However, reduced gene expression was not observed when vaccine L proteins were used in a mini-replicon assay [142]. In addition, the L genes of measles vaccine strains showed higher activities than did the L genes of some wild-type MeV strains when analyzed by mini-replicon assays [176]. Therefore, the contribution of the L protein to attenuation remains unclear.

### Variability in Noncoding Regions

With 1 exception (nucleotide 7243, see below) all known cis-acting elements involved in transcription and replication (ie, promoter elements, intergenic regions, and gene-start and -end sequences) were conserved, as was the entire 37 nucleotide trailer sequence [2, 177]. The high level of conservation of trailer sequences of vaccine strains and wild-type viruses of different genotypes had already been reported [178]. Two nucleotide changes (26 and 42) in the leader sequence distinguish Edmonston wild-type from all vaccine strains. Although all vaccines share the A→T change at nucleotide 26, the substitution at nucleotide 42 is A→C for all vaccines except Zagreb, which has an A→T substitution. These positions affected transcription of a reporter gene in a mini-genome assay [178], and it is conceivable that they may affect replication of vaccine viruses. However, Liu et al [178] also reported that wild-type MeV isolates of genotype A had the same substitutions as the vaccine strains, so the relevance of these positions for attenuation is still unclear. In addition, no effect was observed when these substitutions were introduced into the infectious virus genome [142]. A substitution in Moraten, Schwarz, Rubeovax, and CAM-70 at nucleotide 1806 immediately precedes the AUG start codon of the P protein, possibly affecting translation efficiency [179]. It is intriguing that this substitution was selected for in Edmonston-derived and independently derived vaccines. However, the context of all MeV P start codons deviates from the optimal Kozak sequence [179], making it difficult to assess the effect of the substitution at nucleotide 1806. The suboptimal Kozak context of the P ORF possibly serves to enhance initiation at the downstream start codon for the overlapping C ORF, and it
is conceivable that substitutions that alter P protein translation could affect C protein translation.

Forty-eight nucleotide changes (affecting 19 positions) were located in the long untranslated region spanning the end of the M and the beginning of the F gene (Supplemental Table 1). The fact that MeV, like other morbilliviruses [56, 87], maintains this noncoding region of ~1 kB indicates that this region may play an important role, but its function is still unclear. It is possible that substitutions to this region may affect mRNA structure or stability. Changes to the long untranslated 3′ region of the M gene can increase M protein expression [89]. Deletion of the F nontranslated region reduced viral replication in a SCID mouse model [90] and nucleotide changes in this region decreased F protein expression [89]. Although the implications of single nucleotide changes in the F 5′ noncoding region are unclear, it is intriguing that 3 positions (4978, 5073, and 5349) in this region are changed in all vaccine strains, including those derived from non-Edmonston progenitors.

The only substitution that affects a conserved motif is a substitution in the gene-end signal of the F gene of Moraten and Schwarz (nucleotide 7243). However, although this nucleotide is part of the gene-end motif, this particular position is not conserved among the gene-end signals of MeV [95], making it less likely that the substitution changes transcription.

### Substitutions That Affect Most Vaccine Strains

Nucleotide substitutions, whether silent or not, that affect most vaccine strains are particularly interesting candidates for attenuation markers, especially if they are shared between Edmonston-derived and independently derived strains. There are 17 such changes among the vaccine strains analyzed to date (Table 1): 16 that affect all vaccine strains, and 1 (amino acid 546 in the H protein) that affects all vaccines except Zagreb. Three of these are silent nucleotide changes in coding sequences (nucleotides 4232, 5514, and 14579). Five affect noncoding regions (nucleotides 26, 42, 4978, 5073, and 5349), whereas the remaining 9 predict coding changes in P, V, C, M, H, and L proteins. Several of these substitutions have demonstrated functional consequences, at least in in vitro systems, including nucleotides 26 and 42 as well as amino acids 89 in the M protein and amino acids 211, 481, and 546 in the H protein. However, functional consequences of these substitutions could not always be confirmed in animal models [142, 146]. Because attenuation is generally assumed to be the result of an accumulation of mutations, it is possible that introduction of 1 or a few changes at a time will not be sufficient to produce measurable differences in pathogenicity in animal models.

and found complete sequence identity. Baricevic et al [180] compared a master seed lot of Zagreb with a working seed and also found complete sequence identity. However, 6 nucleotide differences were found between their sequence and the sequence of Zagreb, as published by Parks et al [96, 97], that is used in this review [180]. Passages of the vaccines in the laboratories where they were used may account for those differences. Sequences derived from vaccine-associated measles cases usually only report the carboxyl-terminal region of the N protein which was chosen for genotype analysis. Analysis of 7 such sequences available on GenBank revealed complete nucleotide identity with Moraten/Schwarz (B. Bankamp, unpublished data).

### SUBGENOMIC RNAs

Defective interfering particles (DIs) have been found in several vaccine strains, including Zagreb, AIK-C, and CAM-70 [181, 182]. DIs contain subgenomic RNAs which are generated during replication of the full-length genome [183]. Because of large deletions in the protein coding regions, they require coinfection of the cell with a full-length genome to provide viral proteins. DIs can interfere with the replication of the full-length genome, leading to reduced viral titers in cell culture [184]. Viral stocks that contain DIs induce higher amounts of IFN [185]. Whether the presence of DIs or the resulting increase in IFN induction play any role in attenuation is not known.

### IMMUNOGENICITY, PROTECTION AGAINST DIFFERENT GENOTYPES, AND ADVERSE EVENTS

Measles vaccine is one of the most efficacious vaccines presently available [113]. Seroconversion rates have exceeded 98% in children aged ≥12 months for most vaccines studied [9, 103, 114–116, 186, 187]. These data lead to the conclusion that there are no significant differences between seroconversion rates of the different vaccine strains.

On the basis of the sequences of their N and H genes, MeVs can be assigned to 1 of 23 genotypes and 1 provisional genotype [11, 12]. All vaccine strains and their wild-type progenitors are assigned to genotype A. Experiments with monoclonal antibodies have defined antigenic differences between the H proteins of genotype A vaccines and the H proteins of wild-type viruses grouped in other genotypes [62, 188, 189]. However, there is only 1 serotype for measles, and serum samples from vaccines neutralize viruses from a wide range of genotypes, albeit with different neutralization titers [188, 190] More importantly, despite the presence of different endemic genotypes, vaccination programs with standard measles vaccines have been successful in every country where they were performed adequately [191–193]. Suboptimal seroconversion after vaccination is likely the result of inadequate coverage; improper administration, transport, or storage of vaccine; or age of the vaccine recipients [194–196].

### GENETIC STABILITY OF VACCINE SEQUENCES IN CELL CULTURE AND VACCINEES

Borges et al [97] sequenced CAM-70 directly from the seed lot and after 10 passages in primary chicken embryo fibroblasts

S542 • JID 2011:204 (Suppl 1) • Bankamp et al
Measles vaccines have an outstanding safety record, with only very small numbers of adverse events reported. The most common adverse events are fever and rash beginning 8-12 days after vaccination. Most of these reactions resolve quickly without need of clinical intervention [113]. The insufficiently attenuated Leningrad-4 caused fever (temperature, >38.7°C) and rash in all 7 vaccine recipients [114], and Rubeovax caused fever (temperature, >38.2°C) in 46% of recipients and rash in 16% of cases [9]. The further-attenuated strains induced less severe and less frequent side effects, from 2% of vaccines that presented with fever (temperature, >38°C) after vaccination with Zagreb to 21% of subjects with temperatures >38.2°C after vaccination with Schwarz [9, 103, 105, 114–116, 187, 197]. The occurrence of rash ranges from 3% in Moraten-vaccines to 20% in AIK-C-vaccines [9, 103, 116, 187, 198, 199]. Although there appear to be significant differences in the rates of rash and fever, most vaccines were tested in separate studies and under different conditions, and the cutoff values for fever varied from study to study. Vaccines can cause severe adverse events in immunocompromised individuals whose immune systems are incapable of clearing the virus [200–202]. Children who are human immunodeficiency virus seropositive but do not demonstrate immune impairment can be safely vaccinated with measles vaccine, and they derive some protection from wild-type virus [203, 204]. There is no evidence that measles vaccines are responsible for SSPE [205], because only wild-type viruses have been detected in these cases.

### GENETIC BASIS FOR ATTENUATION

Attenuation is the result of adaptation of the virus to growth conditions in nonpermissive cell culture, in the case of measles especially avian cell lines. This scenario implies that many attenuating mutations affect interactions with host cell proteins. Presumably such mutations increase interaction with cellular proteins in avian cells yet reduce the ability of the protein to carry out its functions in its human host. Most experiments have relied on mammalian cells for the analysis of differences between wild-type and vaccine viruses, whereas few studies used avian cells [127, 206, 207]. Substitutions in the P, V, C, M, and H proteins have been most often associated with measurable differences between vaccine strains and wild-type strains. Specifically, receptor use and interactions with IFN induction/signaling have been shown to be important, as well as substitutions in the M protein that affect interactions with the nucleocapsid. The L proteins of vaccine viruses also appear to contribute to attenuation in ways that are not yet understood. However, no common functional change that affects all vaccines has been demonstrated. This may be because multiple, cumulative changes appear to be responsible for attenuation and because separate passage histories may have given each vaccine strain a different pathway to attenuation. Therefore, although we have made

significant progress in analyzing functional differences between wild-type and vaccine viruses, the molecular basis of attenuation remains unknown. Almost every gene contributes in some way to a cell culture-adapted phenotype; this complicates the identification of the molecular basis for attenuation in humans. Recombination, which is an efficient means for rapid genetic change for many viruses, does not occur in paramyxoviruses. No recombinant viruses have been isolated from natural infections. One constraint that may explain the absence of recombination is the rule of 6 [85, 208]. Genome lengths in the subfamily Paramyxovirinae have to be evenly divisible by 6, and genomes with insertions or deletions that violate the rule of 6 would result in nonviable virus. The complex pathway to cell culture adaptation and, ultimately, attenuation together with the absence of recombination suggest that it would be extremely unlikely for measles vaccine strains to revert to a virulent phenotype.

#### Supplementary Data

Supplementary data are available at *The Journal of Infectious Diseases* online.

#### Funding

No external funds were used at the US Centers for Disease Control and Prevention; the National Institute of Infectious Diseases, Japan; or the National Institute for Viral Disease Control and Prevention, China.

#### References

1. Hilleman MR. Current overview of the pathogenesis and prophylaxis of measles with focus on practical implications. Vaccine 2001; 20:651–65.
2. Griffin DA. Measles virus. In: Fields BN, Knipe DM, Howley PM, eds. Fields Virology. 5th ed. Vol 2. Philadelphia: Wolters Kluwer Health/Lippincott Williams & Wilkins, 2007; 1551–85.
3. Moss WJ, Griffin DE. Global measles elimination. Nat Rev Microbiol 2006; 4:900–8.
4. Schneider-Schaulies J, Meulen V, Schneider-Schaulies S. Measles infection of the central nervous system. J Neurovirol 2003; 9:247–52.
5. Measles-Initiative. WHO/UNICEF joint annual measles report 2009, 2010. http://measlesinitiative.org. Accessed 14 March 2011.
6. Wolfson LJ, Strebel PM, Gacic-Dobo M, Hoekstra EJ, McFarland JW, Hersh BS. Has the 2005 measles mortality reduction goal been achieved? A natural history modelling study. Lancet 2007; 369:191–200.
7. Enders JF, Peebles TC. Propagation in tissue cultures of cytopathogenic agents from patients with measles. Proc Soc Exp Biol Med 1954; 86:277–86.
8. Rota JS, Wang ZD, Rota PA, Bellini WJ. Comparison of sequences of the H, F, and N coding genes of measles virus vaccine strains. Virus Res 1994; 31:317–30.
9. Hilleman MR, Buynak EB, Weibel RE, Stokes JJr, Whitman JE Jr, League MB. Development and evaluation of the Moraten measles virus vaccine. JAMA 1968; 206:587–90.
10. Atkinson W, Wolfe S, Hamborsky J, McIntyre L;. Centers for Disease Control and Prevention. Epidemiology and prevention of vaccine-preventable diseases. 11th ed. Washington, DC: Public Health Foundation, 2009.

11. WHO. Nomenclature for describing the genetic characteristics of wild-type measles viruses (update). Wkly Epidemiol Rec 2001; 76:249–51.
12. Zhang Y, Ding Z, Wang H, et al. New measles virus genotype associated with outbreak, China. Emerg Infect Dis 2010; 16:943–7.
13. Riddell MA, Rota JS, Rota PA. Review of the temporal and geographical distribution of measles virus genotypes in the prevaccine and postvaccine eras. Virol J 2005; 2:87.
14. Bellini WJ, Englund G, Rozenblatt S, Arnheiter H, Richardson CD. Measles virus P gene codes for two proteins. J Virol 1985; 53:908–19.
15. Cattaneo R, Kaelin K, Baczko K, Billeter MA. Measles virus editing provides an additional cysteine-rich protein. Cell 1989; 56:759–64.
16. Longhi S, Receveur-Brechot V, Karlin D, et al. The C-terminal domain of the measles virus nucleoprotein is intrinsically disordered and folds upon binding to the C-terminal moiety of the phosphoprotein. J Biol Chem 2003; 278:18638–48.
17. Rota PA, Featherstone DA, Bellini WJ. Molecular epidemiology of measles virus. Curr Top Microbiol Immunol 2009; 330:129–50.
18. Taylor MJ, Godfrey E, Baczko K, ter Meulen V, Wild TF, Rima BK. Identification of several different lineages of measles virus. J Gen Virol 1991; 72:83–8.
19. Bankamp B, Horikami SM, Thompson PD, Huber M, Billeter M, Moyer SA. Domains of the measles virus N protein required for binding to P protein and self-assembly. Virology 1996; 216:272–7.
20. Harty RN, Palese P. Measles virus phosphoprotein (P) requires the NH2- and COOH-terminal domains for interactions with the nucleoprotein (N) but only the COOH terminus for interactions with itself. J Gen Virol 1995; 76:2863–7.
21. Iwasaki M, Takeda M, Shirogane Y, Nakatsu Y, Nakamura T, Yanagi Y. The matrix protein of measles virus regulates viral RNA synthesis and assembly by interacting with the nucleocapsid protein. J Virol 2009; 83:10374–83.
22. Laine D, Trescol-Biemont MC, Longhi S, et al. Measles virus (MV) nucleoprotein binds to a novel cell surface receptor distinct from FcγRII via its C-terminal domain: role in MV-induced immunosuppression. J Virol 2003; 77:11332–46.
23. Sato H, Masuda M, Kanai M, Tsukiyama-Kohara K, Yoneda M, Kai C. Measles virus N protein inhibits host translation by binding to eIF3-p40. J Virol 2007; 81:11569–76.
24. tenOever BR, Servant MJ, Grandvaux N, Lin R, Hiscott J. Recognition of the measles virus nucleocapsid as a mechanism of IRF-3 activation. J Virol 2002; 76:3659–69.
25. Zhang X, Glendening C, Linke H, et al. Identification and characterization of a regulatory domain on the carboxyl terminus of the measles virus nucleocapsid protein. J Virol 2002; 76:8737–46.
26. Parks CL, Lerch RA, Walpita P, Sidhu MS, Udem SA. Enhanced measles virus cDNA rescue and gene expression after heat shock. J Virol 1999; 73:3560–6.
27. Vasconcelos DY, Cai XH, Oglesbee MJ. Constitutive overexpression of the major inducible 70 kDa heat shock protein mediates large plaque formation by measles virus. J Gen Virol 1998; 79:2239–47.
28. Zhang X, Bourhis JM, Longhi S, et al. Hsp72 recognizes a P binding motif in the measles virus N protein C-terminus. Virology 2005; 337:162–74.
29. Carsillo T, Zhang X, Vasconcelos D, Niewiesk S, Oglesbee M. A single codon in the nucleocapsid protein C terminus contributes to in vitro and in vivo fitness of Edmonston measles virus. J Virol 2006; 80:2904–12.
30. Liston P, DiFlumeri C, Briedis DJ. Protein interactions entered into by the measles virus P, V, and C proteins. Virus Res 1995; 38:241–59.
31. Sedlmeier R, Neubert WJ. The replicative complex of paramyxoviruses: structure and function. Adv Virus Res 1998; 50:101–39.
32. Curran J, Marq JB, Kolakofsky D. An N-terminal domain of the Sendai paramyxovirus P protein acts as a chaperone for the NP protein during the nascent chain assembly step of genome replication. J Virol 1995; 69:849–55.
33. Fontana JM, Bankamp B, Rota PA. Inhibition of interferon induction and signaling by paramyxoviruses. Immunol Rev 2008; 225:46–67.
34. Caignard G, Guerbois M, Labernardiere JL, et al. Measles virus V protein blocks Jak1-mediated phosphorylation of STAT1 to escape IFN-alpha/beta signaling. Virology 2007; 368:351–62.
35. Nakatsu Y, Takeda M, Ohno S, Koga R, Yanagi Y. Translational inhibition and increased interferon induction in cells infected with C protein-deficient measles virus. J Virol 2006; 80:11861–7.
36. Ohno S, Ono N, Takeda M, Takeuchi K, Yanagi Y. Dissection of measles virus V protein in relation to its ability to block alpha/beta interferon signal transduction. J Gen Virol 2004; 85:2991–9.
37. Palosaari H, Parisien JP, Rodriguez JJ, Ulane CM, Horvath CM. STAT protein interference and suppression of cytokine signal transduction by measles virus V protein. J Virol 2003; 77:7635–44.
38. Ramachandran A, Parisien JP, Horvath CM. STAT2 is a primary target for measles virus V protein-mediated alpha/beta interferon signaling inhibition. J Virol 2008; 82:8330–8.
39. Takeuchi K, Kadota SI, Takeda M, Miyajima N, Nagata K. Measles virus V protein blocks interferon (IFN)-alpha/beta but not IFN-gamma signaling by inhibiting STAT1 and STAT2 phosphorylation. FEBS Lett 2003; 545:177–82.
40. Childs K, Stock N, Ross C, et al. mda-5, But not RIG-I, is a common target for paramyxovirus V proteins. Virology 2007; 359:190–200.
41. Ikegame S, Takeda M, Ohno S, Nakatsu Y, Nakanishi Y, Yanagi Y. Both RIG-I and MDA5 RNA helicases contribute to the induction of alpha/beta interferon in measles virus-infected human cells. J Virol 2010; 84:372–9.
42. Nakatsu Y, Takeda M, Ohno S, Shirogane Y, Iwasaki M, Yanagi Y. Measles virus circumvents the host interferon response by different actions of the C and V proteins. J Virol 2008; 82:8296–306.
43. Fontana JM, Bankamp B, Bellini WJ, Rota PA. Regulation of interferon signaling by the C and V proteins from attenuated and wild-type strains of measles virus. Virology 2008; 374:71–81.
44. Shaffer JA, Bellini WJ, Rota PA. The C protein of measles virus inhibits the type I interferon response. Virology 2003; 315:389–97.
45. Bankamp B, Wilson J, Bellini WJ, Rota PA. Identification of naturally occurring amino acid variations that affect the ability of the measles virus C protein to regulate genome replication and transcription. Virology 2005; 336:120–9.
46. Reutter GL, Cortese-Grogan C, Wilson J, Moyer SA. Mutations in the measles virus C protein that up regulate viral RNA synthesis. Virology 2001; 285:100–9.
47. Nakatsu Y, Takeda M, Iwasaki M, Yanagi Y. A highly attenuated measles virus vaccine strain encodes a fully functional C protein. J Virol 2009; 83:11996–2001.
48. Witko SE, Kotash C, Sidhu MS, Udem SA, Parks CL. Inhibition of measles virus minireplicon-encoded reporter gene expression by V protein. Virology 2006; 348:107–19.
49. Parks CL, Witko SE, Kotash C, Lin SL, Sidhu MS, Udem SA. Role of V protein RNA binding in inhibition of measles virus minigenome replication. Virology 2006; 348:96–106.
50. Wild TF, Buckland R. Functional aspects of envelope-associated measles virus proteins. Curr Top Microbiol Immunol 1995; 191:51–64.
51. Cathomen T, Mrkic B, Spehner D, et al. A matrix-less measles virus is infectious and elicits extensive cell fusion: consequences for propagation in the brain. EMBO J 1998; 17:3899–908.
52. Cathomen T, Naim HY, Cattaneo R. Measles viruses with altered envelope protein cytoplasmic tails gain cell fusion competence. J Virol 1998; 72:1224–34.
53. Spielhofer P, Bachí T, Fehr T, et al. Chimeric measles viruses with a foreign envelope. J Virol 1998; 72:2150–9.
54. Hirano A, Ayata M, Wang AH, Wong TC. Functional analysis of matrix proteins expressed from cloned genes of measles virus variants that cause subacute sclerosing panencephalitis reveals a common defect in nucleocapsid binding. J Virol 1993; 67:1848–53.
S544 • JID 2011:204 (Suppl 1) • Bankamp et al

55. Suryanarayana K, Baczko K, ter Meulen V, Wagner RR. Transcription inhibition and other properties of matrix proteins expressed by M genes cloned from measles viruses and diseased human brain tissue. J Virol 1994; 68:1532–43.

56. Richardson C, Hull D, Greer P, et al. The nucleotide sequence of the mRNA encoding the fusion protein of measles virus (Edmonston strain): a comparison of fusion proteins from several different paramyxoviruses. Virology 1986; 155:508–23.

57. Alkhatib G, Shen SH, Briedis D, et al. Functional analysis of N-linked glycosylation mutants of the measles virus fusion protein synthesized by recombinant vaccinia virus vectors. J Virol 1994; 68:1522–31.

58. Richardson CD, Choppin PW. Oligopeptides that specifically inhibit membrane fusion by paramyxoviruses: studies on the site of action. Virology 1983; 131:518–32.

59. Samuel O, Shai Y. Participation of two fusion peptides in measles virus-induced membrane fusion: emerging similarity with other paramyxoviruses. Biochemistry 2001; 40:1340–9.

60. Buckland R, Gerald C, Barker R, Wild TF. Fusion glycoprotein of measles virus: nucleotide sequence of the gene and comparison with other paramyxoviruses. J Gen Virol 1987; 68:1695–703.

61. Hu A, Cattaneo R, Schwartz S, Norrby E. Role of N-linked oligosaccharide chains in the processing and antigenicity of measles virus haemagglutinin protein. J Gen Virol 1994; 75:1043–52.

62. Santibanez S, Niewiesk S, Heider A, et al. Probing neutralizing-antibody responses against emerging measles viruses (MVs): immune selection of MV by H protein-specific antibodies? J Gen Virol 2005; 86:365–74.

63. Moll M, Klenk HD, Herrler G, Maisner A. A single amino acid change in the cytoplasmic domains of measles virus glycoproteins H and F alters targeting, endocytosis, and cell fusion in polarized Madin-Darby canine kidney cells. J Biol Chem 2001; 276:17887–94.

64. Naim HY, Ehler E, Billeter MA. Measles virus matrix protein specifies apical virus release and glycoprotein sorting in epithelial cells. EMBO J 2000; 19:3576–85.

65. Yanagi Y, Takeda M, Ohno S, Hashiguchi T. Measles virus receptors. Curr Top Microbiol Immunol 2009; 329:13–30.

66. Tatsuo H, Ono N, Tanaka K, Yanagi Y. SLAM. (CDw150) is a cellular receptor for measles virus. Nature 2000; 406:893–7.

67. Dorig RE, Marcil A, Chopra A, Richardson CD. The human CD46 molecule is a receptor for measles virus (Edmonston strain). Cell 1993; 75:295–305.

68. Naniche D, Varior-Krishnan G, Cervoni F, et al. Human membrane cofactor protein (CD46) acts as a cellular receptor for measles virus. J Virol 1993; 67:6025–32.

69. Aversa G, Chang CC, Carballido JM, Cocks BG, de Vries JE. Engagement of the signaling lymphocytic activation molecule (SLAM) on activated T cells results in IL-2-independent, cyclosporin A-sensitive T cell proliferation and IFN-gamma production. J Immunol 1997; 158:4036–44.

70. Cocks BG, Chang CC, Carballido JM, Yssel H, de Vries JE, Aversa G. A novel receptor involved in T-cell activation. Nature 1995; 376:260–3.

71. Sidorenko SP, Clark EA. Characterization of a cell surface glycoprotein IPO-3, expressed on activated human B and T lymphocytes. J Immunol 1993; 151:4614–24.

72. Leonard VH, Sinn PL, Hodge G, et al. Measles virus blind to its epithelial cell receptor remains virulent in rhesus monkeys but cannot cross the airway epithelium and is not shed. J Clin Invest 2008; 118:2448–58.

73. Shirogane Y, Takeda M, Tahara M, Ikegame S, Nakamura T, Yanagi Y. Epithelial-mesenchymal transition abolishes the susceptibility of polarized epithelial cell lines to measles virus. J Biol Chem 2010; 285:20882–90.

74. Tahara M, Takeda M, Shirogane Y, Hashiguchi T, Ohno S, Yanagi Y. Measles virus infects both polarized epithelial and immune cells by using distinctive receptor-binding sites on its hemagglutinin. J Virol 2008; 82:4630–7.

75. Takeda M. Measles virus breaks through epithelial cell barriers to achieve transmission. J Clin Invest 2008; 118:2386–9.

76. Takeda M, Tahara M, Hashiguchi T, et al. A human lung carcinoma cell line supports efficient measles virus growth and syncytium formation via a SLAM- and CD46-independent mechanism. J Virol 2007; 81:12091–6.

77. de Swart RL, Yuksel S, Osterhaus AD. Relative contributions of measles virus hemagglutinin- and fusion protein-specific serum antibodies to virus neutralization. J Virol 2005; 79:11547–51.

78. Grdzelishvili VZ, Smallwood S, Tower D, Hall RL, Hunt DM, Moyer SA. A single amino acid change in the L-polymerase protein of vesicular stomatitis virus completely abolishes viral mRNA cap methylation. J Virol 2005; 79:7327–37.

79. Hercyk N, Horikami SM, Moyer SA. The vesicular stomatitis virus L protein possesses the mRNA methyltransferase activities. Virology 1988; 163:222–5.

80. Ogino T, Kobayashi M, Iwama M, Mizumoto K. Sendai virus RNA-dependent RNA polymerase L protein catalyzes cap methylation of virus-specific mRNA. J Biol Chem 2005; 280:4429–35.

81. Poch O, Blumberg BM, Bougueleret L, Tordo N. Sequence comparison of five polymerases (L proteins) of unsegmented negative-strand RNA viruses: theoretical assignment of functional domains. J Gen Virol 1990; 71:1153–62.

82. Cevik B, Holmes DE, Vrotsos E, Feller JA, Smallwood S, Moyer SA. The phosphoprotein (P) and L binding sites reside in the N-terminus of the L subunit of the measles virus RNA polymerase. Virology 2004; 327:297–306.

83. Horikami SM, Smallwood S, Bankamp B, Moyer SA. An amino-proximal domain of the L protein binds to the P protein in the measles virus RNA polymerase complex. Virology 1994; 205:540–5.

84. Radecke F, Spielhofer P, Schneider H, et al. Rescue of measles viruses from cloned DNA. EMBO J 1995; 14:5773–84.

85. Sidhu MS, Chan J, Kaelin K, et al. Rescue of synthetic measles virus minireplicons: measles genomic termini direct efficient expression and propagation of a reporter gene. Virology 1995; 208:800–7.

86. Crowley JC, Dowling PC, Menonna J, et al. Sequence variability and function of measles virus 3′ and 5′ ends and intercistronic regions. Virology 1988; 164:498–506.

87. Bellini WJ, Englund G, Richardson CD, Rozenblatt S, Lazzarini RA. Matrix genes of measles virus and canine distemper virus: cloning, nucleotide sequences, and deduced amino acid sequences. J Virol 1986; 58:408–16.

88. Cathomen T, Buchholz CJ, Spielhofer P, Cattaneo R. Preferential initiation at the second AUG of the measles virus F mRNA: a role for the long untranslated region. Virology 1995; 214:628–32.

89. Takeda M, Ohno S, Seki F, Nakatsu Y, Tahara M, Yanagi Y. Long untranslated regions of the measles virus M and F genes control virus replication and cytopathogenicity. J Virol 2005; 79:14346–54.

90. Valsamakis A, Schneider H, Auwaerter PG, Kaneshima H, Billeter MA, Griffin DE. Recombinant measles viruses with mutations in the C, V, or F gene have altered growth phenotypes in vivo. J Virol 1998; 72:7754–61.

91. Lamb RA, Parks GD. Paramyxoviridae: the viruses and their replication. In: Fields BN, Knipe DM, Howley PM, eds. Fields Virology. 5th ed. Vol 1. Philadelphia: Wolters Kluwer Health/Lippincott Williams & Wilkins, 2007; 1449–96.

92. Hall WW, Meulen V. Rolyadenylic acid [poly(A)] sequences associated with measles virus intracellular ribonucleic acid (RNA) species. J Gen Virol 1977; 35:487–510.

93. Rima BK, Duprex WP. The measles virus replication cycle. Curr Top Microbiol Immunol 2009; 329:77–102.

94. Mori T, Sasaki K, Hashimoto H, Makino S. Molecular cloning and complete nucleotide sequence of genomic RNA of the AIK-C strain of attenuated measles virus. Virus Genes 1993; 7:67–81.

95. Parks CL, Lerch RA, Walpita P, Wang HP, Sidhu MS, Udem SA. Analysis of the noncoding regions of measles virus strains in the Edmonston vaccine lineage. J Virol 2001; 75:921–33.

96. Parks CL, Lerch RA, Walpita P, Wang HP, Sidhu MS, Udem SA. Comparison of predicted amino acid sequences of measles virus strains in the Edmonston vaccine lineage. J Virol 2001; 75:910–20.

97. Borges MB, Caride E, Jabor AV, et al. Study of the genetic stability of measles virus CAM-70 vaccine strain after serial passages in chicken embryo fibroblasts primary cultures. Virus Genes 2008; 36:35–44.
98. Zhang Y, Zhou J, Bellini WJ, Xu W, Rota PA. Genetic characterization of Chinese measles vaccines by analysis of complete genomic sequences. J Med Virol 2009; 81:1477–83.
99. Smorodintsev AA, Boichuk LM, Shikina ES, Batanova TB, Bystriakova LV, Peradze TV. Clinical and immunological response to live tissue culture vaccine against measles. Acta Virol 1960; 4:201–4.
100. Ueda S, Takahashi M, Minekawa Y, Ogino T, Suzuki N. Studies on further attenuated live measles vaccine. I. Adaptation of measles virus to the chorioallantoic membrane of chick embryo and clinical tests on the strain. Biken J 1970; 13:111–6.
101. Xiang JZ, Chen ZH. Measles vaccine in the People’s Republic of China. Rev Infect Dis 1983; 5:506–10.
102. Makino S. Development and characteristics of live AIK-C measles virus vaccine: a brief report. Rev Infect Dis 1983; 5:504–5.
103. Makino S, Sasaki K, Nakamura N, Nakagawa M, Nakajima S. Studies on the modification of the live AIK measles vaccine. II. Development and evaluation of the live AIK-C measles vaccine. Kitasato Arch Exp Med 1974; 47:13–21.
104. Ikic D, Beck M, Juzbasic M. Characterization of Edmonston-Zagreb measles virus. Proc. Symposium on Human Diploid Cells. Zagreb, Yugoslavia: Yugoslav Academy of Sciences and Arts, 1970; 121–9.
105. Ikic D, Juzbasic M, Beck M, Hrabar A, Cimbur-Schreiber T. Attenuation and characterisation of Edmonston-Zagreb measles virus. Ann Immunol Hung 1972; 16:175–81.
106. Tamura K, Dudley J, Nei M, Kumar S. MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0. Mol Biol Evol 2007; 24:1596–99.
107. Jayaraman KS. Biotech boom. Nature 2005; 436:480–3.
108. Hilleman MR, Stokes J Jr, Buynak EB, Weibel R, Halenda R, Goldner H. Studies of live attenuated measles virus vaccine in man. II. Appraisal of efficacy. Am J Public Health Nations Health 1962; 52:S44–56.
109. Katz SL, Enders JF, Holloway A. The development and evaluation of an attenuated measles virus vaccine. Am J Public Health Nations Health 1962; 52:S5–10.
110. Stokes J Jr, Weibel R, Halenda R, Reilly CM, Hilleman MR. Enders’ live measles-virus vaccine with human immune globulin. I. Clinical reactions. Am J Dis Child 1962; 103:366–72.
111. Stokes J Jr, Weibel R, Halenda R, Reilly CM, Hilleman MR. Studies of live attenuated measles virus vaccine in man. I. Clinical aspects. Am J Public Health Nations Health 1962; 52:S29–43.
112. Schwarz AJ. Preliminary tests of a highly attenuated measles vaccine. Am J Dis Child 1962; 103:386–9.
113. Strebel PM, Papania MJ, Dayan GH, Halsey NA. Measles vaccine. In: Plotkin SA, Orenstein WA, Offit PA, eds. Vaccines. 5th ed. Philadelphia: Saunders, 2008: 353–98.
114. Huang CH, Chia PY, Chu FT, et al. Studies on attenuated measles vaccine. I. Clinical and immunologic response to measles virus attenuated in human amnion cells. Chin Med J 1962; 81:9–14.
115. Chun H, T’Eh-chang H, Kuo-hua T, et al. Clinical and immunologic observations on two lines of attenuated measles vaccine virus upon passage in chick embryo cell culture. Chin Med J (Engl) 1975; 1:283–6.
116. Okuno Y, Ueda S, Kurimura T, Suzuki N, Yamanishi K. Studies on further attenuated liver measles vaccine. VII. Development and evaluation of CAM-70 measles virus vaccine. Biken J 1971; 14:253–8.
117. Takaku K, Sasada T, Konobe T, Onishi K, Ueda S. Studies on further attenuated live measles vaccine. 3. Selection of less reactive variants of CAM measles vaccine virus. Biken J 1970; 13:163–8.
118. Kato S, Ohgimoto S, Sharma LB, et al. Reduced ability of hemagglutinin of the CAM-70 measles virus vaccine strain to use receptors CD46 and SLAM. Vaccine 2009; 27:3838–48.
119. Smorodintsev AA, Boychuk LM, Shikina ES, et al. Prevention of measles by use of live vaccines in the U.S.S.R. Arch Gesamte Virusforsch 1965; 16:284–93.
120. Smorodintsev AA, Boychuk LM, Shikina ES, et al. Further experiences with live measles vaccines in USSR: use of virus propagated in chick fibroblasts and in guinea pig kidney cells. Am J Dis Child 1962; 103:384–6.
121. Korber B. HIV signature and sequence variation analysis. In: Rodrigo AG, Learn GH, eds. Computational analysis of HIV molecular sequences. Dordrecht, the Netherlands: Kluwer Academic Publishers, 2000: 55–72.
122. Rota PA, Bloom AE, Vanchiere JA, Bellini WJ. Evolution of the nucleoprotein and matrix genes of wild-type strains of measles virus isolated from recent epidemics. Virology 1994; 198:724–30.
123. Cattaneo R, Schmid A, Eschle D, Baczko K, ter Meulen V, Billette MA. Biased hypermutation and other genetic changes in defective measles viruses in human brain infections. Cell 1988; 55:255–65.
124. Bass BL, Weintraub H, Cattaneo R, Billette MA. Biased hypermutation of viral RNA genomes could be due to unwinding/modification of double-stranded RNA. Cell 1989; 56:331.
125. Billette MA, Cattaneo R, Spielhofer P, et al. Generation and properties of measles virus mutations typically associated with subacute sclerosing panencephalitis. Ann N Y Acad Sci 1994; 724:367–77.
126. Bankamp B, Fontana JM, Bellini WJ, Rota PA. Adaptation to cell culture induces functional differences in measles virus proteins. Virol J 2008; 5:129.
127. Bankamp B, Hodge G, McChesney MB, Bellini WJ, Rota PA. Genetic changes that affect the virulence of measles virus in a rhesus macaque model. Virology 2008; 373:39–50.
128. Druelle J, Sellin CI, Waku-Kouomou D, Horvat B, Wild FT. Wild type measles virus attenuation independent of type I IFN. Virol J 2008; 5:22.
129. Takeda M, Kato A, Kobune F, et al. Measles virus attenuation associated with transcriptional impediment and a few amino acid changes in the polymerase and accessory proteins. J Virol 1998; 72:8690–6.
130. Takeda M, Sakaguchi T, Li Y, Kobune F, Kato A, Nagai Y. The genome nucleotide sequence of a contemporary wild strain of measles virus and its comparison with the classical Edmonston strain genome. Virology 1999; 256:340–50.
131. Takeuchi K, Miyajima N, Kobune F, Tashiro M. Comparative nucleotide sequence analyses of the entire genomes of B95a cell-isolated and vero cell-isolated measles viruses from the same patient. Virus Genes 2000; 20:253–7.
132. Bankamp B, Lopareva EN, Kremer JR, et al. Genetic variability and mRNA editing frequencies of the phosphoprotein genes of wild-type measles viruses. Virus Res 2008; 135:298–306.
133. Devaux P, von Messling V, Songsungthong W, Springfield C, Cattaneo R. Tyrosine 110 in the measles virus phosphoprotein is required to block STAT1 phosphorylation. Virology 2007; 360:72–83.
134. Das T, Schuster A, Schneider-Schaulies S, Banerjee AK. Involvement of cellular casein kinase II in the phosphorylation of measles virus P protein: identification of phosphorylation sites. Virology 1995; 211:218–26.
135. Fukuda A, Sugiura A. Temperature-dependent growth restriction in measles vaccine strains. Jpn J Med Sci Biol 1983; 36:331–5.
136. Komase K, Nakayama T, Iijima M, Miki K, Kawanishi R, Uejima H. The phosphoprotein of attenuated measles AIK-C vaccine strain contributes to its temperature-sensitive phenotype. Vaccine 2006; 24:826–34.
137. Devaux P, Cattaneo R. Measles virus phosphoprotein gene products: conformational flexibility of the P/V protein amino-terminal domain and C protein infectivity factor function. J Virol 2004; 78:11632–40.
138. Patterson JB, Thomas D, Lewicki H, Billette MA, Oldstone MB. V and C proteins of measles virus function as virulence factors in vivo. Virology 2000; 267:80–9.
139. Takeuchi K, Takeda M, Miyajima N, et al. Stringent requirement for the C protein of wild-type measles virus for growth both in vitro and in macaques. J Virol 2005; 79:7838–44.
140. Tober C, Seufert M, Schneider H, et al. Expression of measles virus V protein is associated with pathogenicity and control of viral RNA synthesis. J Virol 1998; 72:8124–32.
S546 • JID 2011:204 (Suppl 1) • Bankamp et al

141. Devaux P, Hodge G, McChesney MB, Cattaneo R. Attenuation of V- or C-defective measles viruses: infection control by the inflammatory and interferon responses of rhesus monkeys. J Virol 2008; 82:5359–67.
142. Takeda M, Ohno S, Tahara M, et al. Measles viruses possessing the polymerase protein genes of the Edmonston vaccine strain exhibit attenuated gene expression and growth in cultured cells and SLAM knock-in mice. J Virol 2008; 82:11979–84.
143. Haga T, Murayama N, Shimizu Y, et al. Analysis of antibody response by temperature-sensitive measles vaccine strain in the cotton rat model. Comp Immunol Microbiol Infect Dis 2009; 32:395–406.
144. Tahara M, Takeda M, Yanagi Y. Altered interaction of the matrix protein with the cytoplasmic tail of hemagglutinin modulates measles virus growth by affecting virus assembly and cell-cell fusion. J Virol 2007; 81:6827–36.
145. Tahara M, Takeda M, Yanagi Y. Contributions of matrix and large protein genes of the measles virus edmonston strain to growth in cultured cells as revealed by recombinant viruses. J Virol 2005; 79:15218–25.
146. Dong JB, Saito A, Mine Y, et al. Adaptation of wild-type measles virus to cotton rat lung cells: E89K mutation in matrix protein contributes to its fitness. Virus Genes 2009; 39:330–4.
147. Johnston IC, ter Meulen V, Schneider-Schaulies J, Schneider-Schaulies S. A recombinant measles vaccine virus expressing wild-type glycoproteins: consequences for viral spread and cell tropism. J Virol 1999; 73:6903–15.
148. Plemper RK, Compans RW. Mutations in the putative HR-C region of the measles virus F2 glycoprotein modulate syncytium formation. J Virol 2003; 77:4181–90.
149. Plemper RK, Lakdawala AS, Gernert KM, Snyder JP, Compans RW. Structural features of paramyxovirus F protein required for fusion initiation. Biochemistry 2003; 42:6645–55.
150. Leonard VH, Hodge G, Reyes-Del Valle J, McChesney MB, Cattaneo R. Measles virus selectively blind to signaling lymphocytic activation molecule (SLAM; CD150) is attenuated and induces strong adaptive immune responses in rhesus monkeys. J Virol 2010; 84:3413–20.
151. Santiago C, Celma ML, Stehle T, Casasnovas JM. Structure of the measles virus hemagglutinin bound to the CD46 receptor. Nat Struct Mol Biol 2010; 17:124–9.
152. Bartz R, Brinckmann U, Dunster LM, Rima B, Ter Meulen V, Schneider-Schaulies J. Mapping amino acids of the measles virus hemagglutinin responsible for receptor (CD46) downregulation. Virology 1996; 224:334–7.
153. Hsu EC, Sarangi F, Iorio C, et al. A single amino acid change in the hemagglutinin protein of measles virus determines its ability to bind CD46 and reveals another receptor on marmoset B cells. J Virol 1998; 72:2905–16.
154. Lecouturier V, Fayolle J, Caballero M, et al. Identification of two amino acids in the hemagglutinin glycoprotein of measles virus (MV) that govern hemadsorption, HeLa cell fusion, and CD46 downregulation: phenotypic markers that differentiate vaccine and wild-type MV strains. J Virol 1996; 70:4200–4.
155. Masse N, Ainouze M, Neel B, Wild TF, Buckland R, Langedijk JP. Measles virus (MV) hemagglutinin: evidence that attachment sites for MV receptors SLAM and CD46 overlap on the globular head. J Virol 2004; 78:9051–63.
156. Masse N, Barrett T, Muller CP, Wild TF, Buckland R. Identification of a second major site for CD46 binding in the hemagglutinin protein from a laboratory strain of measles virus (MV): potential consequences for wild-type MV infection. J Virol 2002; 76:13034–8.
157. Patterson JB, Scheiflinger F, Manchester M, Yilma T, Oldstone MB. Structural and functional studies of the measles virus hemagglutinin: identification of a novel site required for CD46 interaction. Virology 1999; 256:142–51.
158. Vongpunsawad S, Oezgun N, Braun W, Cattaneo R. Selectively receptor-blind measles viruses: identification of residues necessary for SLAM- or CD46-induced fusion and their localization on a new hemagglutinin structural model. J Virol 2004; 78:302–13.
159. Nielsen L, Blixenkrone-Moller M, Thylstrup M, Hansen NJ, Bolt G. Adaptation of wild-type measles virus to CD46 receptor usage. Arch Virol 2001; 146:197–208.
160. Shibahara K, Hotta H, Katayama Y, Homma M. Increased binding activity of measles virus to monkey red blood cells after long-term passage in Vero cell cultures. J Gen Virol 1994; 75:3511–6.
161. Tanaka K, Xie M, Yanagi Y. The hemagglutinin of recent measles virus isolates induces cell fusion in a marmoset cell line, but not in other CD46-positive human and monkey cell lines, when expressed together with the F protein. Arch Virol 1998; 143:213–25.
162. Xie M, Tanaka K, Ono N, Minagawa H, Yanagi Y. Amino acid substitutions at position 481 differently affect the ability of the measles virus hemagglutinin to induce cell fusion in monkey and marmoset cells co-expressing the fusion protein. Arch Virol 1999; 144:1689–99.
163. Seki F, Takeda M, Minagawa H, Yanagi Y. Recombinant wild-type measles virus containing a single N481Y substitution in its haemagglutinin cannot use receptor CD46 as efficiently as that having the haemagglutinin of the Edmonston laboratory strain. J Gen Virol 2006; 87:1643–8.
164. Tahara M, Takeda M, Seki F, Hashiguchi T, Yanagi Y. Multiple amino acid substitutions in hemagglutinin are necessary for wild-type measles virus to acquire the ability to use receptor CD46 efficiently. J Virol 2007; 81:2564–72.
165. Hummel KB, Bellini WJ. Localization of monoclonal antibody epitopes and functional domains in the hemagglutinin protein of measles virus. J Virol 1995; 69:1913–6.
166. Hummel KB, Vanchiere JA, Bellini WJ. Restriction of fusion protein mRNA as a mechanism of measles virus persistence. Virology 1994; 202:665–72.
167. Li L, Qi Y. A novel amino acid position in hemagglutinin glycoprotein of measles virus is responsible for hemadsorption and CD46 binding. Arch Virol 2002; 147:775–86.
168. Rima BK, Earle JA, Baczko K, et al. Sequence divergence of measles virus haemagglutinin during natural evolution and adaptation to cell culture. J Gen Virol 1997; 78( Pt 1);97–106.
169. Rota JS, Hummel KB, Rota PA, Bellini WJ. Genetic variability of the glycoprotein genes of current wild-type measles isolates. Virology 1992; 188:135–42.
170. Woelk CH, Jin L, Holmes EC, Brown DW. Immune and artificial selection in the haemagglutinin (H) glycoprotein of measles virus. J Gen Virol 2001; 82:2463–74.
171. Schnorr JJ, Dunster LM, Nanan R, Schneider-Schaulies J, Schneider-Schaulies S, ter Meulen V. Measles virus-induced downregulation of CD46 is associated with enhanced sensitivity to complement-mediated lysis of infected cells. Eur J Immunol 1995; 25:976–84.
172. Alkhatib G, Briedis DJ. The predicted primary structure of the measles virus hemagglutinin. Virology 1986; 150:479–90.
173. Sakata H, Kobune F, Sato TA, Tanabayashi K, Yamada A, Sugiura A. Variation in field isolates of measles virus during an 8-year period in Japan. Microbiol Immunol 1993; 37:233–7.
174. Hashiguchi T, Kajikawa M, Maita N, et al. Crystal structure of measles virus hemagglutinin provides insight into effective vaccines. Proc Natl Acad Sci U S A 2007; 104:19535–40.
175. Bankamp B, Bellini WJ, Rota PA. Comparison of L proteins of vaccine and wild-type measles viruses. J Gen Virol 1999; 80:1617–25.
176. Bankamp B, Kearney SP, Liu X, Bellini WJ, Rota PA. Activity of polymerase proteins of vaccine and wild-type measles virus strains in a minigenome replication assay. J Virol 2002; 76:7073–81.
177. Rima BK, Duprex WP. The measles virus replication cycle. Curr Top Microbiol Immunol 2009; 329:77–102.
178. Liu X, Bankamp B, Xu W, Bellini WJ, Rota PA. The genomic termini of wild-type and vaccine strains of measles virus. Virus Res 2006; 122:78–84.
179. Kozak M. Point mutations define a sequence flanking the AUG initiator codon that modulates translation by eukaryotic ribosomes. Cell 1986; 44:283–92.

180. Forcic D, Gulija TK, Jug R, Mazuran R, Baricevic M. Determination of the coding and non-coding nucleotide sequences of genuine Edmonston-Zagreb master seed and current working seed lot. Vaccine 2005; 23:1072–8.
181. Bellocq C, Mottet G, Roux L. Wide occurrence of measles virus subgenomic RNAs in attenuated live-virus vaccines. Biologicals 1990; 18:337–43.
182. Calain P, Roux L. Generation of measles virus defective interfering particles and their presence in a preparation of attenuated live-virus vaccine. J Virol 1988; 62:2859–66.
183. Lazzarini RA, Keene JD, Schubert M. The origins of defective interfering particles of the negative-strand RNA viruses. Cell 1981; 26:145–54.
184. Whistler T, Bellini WJ, Rota PA. Generation of defective interfering particles by two vaccine strains of measles virus. Virology 1996; 220:480–4.
185. Shingai M, Ebihara T, Begum NA, et al. Differential type I IFN-inducing abilities of wild-type versus vaccine strains of measles virus. J Immunol 2007; 179:6123–33.
186. Banerjee K, Shaikh N, Phadke M, Bedekar SS, Rauta SK, Mehta JM. Comparative efficacy of three measles vaccines in Indian children. Indian J Public Health 1998; 42:113–9.
187. Wu T. The history and current status on measles control in China. Chin J Epidemiol 2004; 21:143–146.
188. Finsterbusch T, Wolbert A, Deitmeier I, et al. Measles viruses of genotype H1 evade recognition by vaccine-induced neutralizing antibodies targeting the linear haemagglutinin noose epitope. J Gen Virol 2009; 90:2739–45.
189. Tamin A, Rota PA, Wang ZD, Heath JL, Anderson LJ, Bellini WJ. Antigenic analysis of current wild type and vaccine strains of measles virus. J Infect Dis 1994; 170:795–801.
190. Klingele M, Hartter HK, Adu F, Ammerlaan W, Ikusika W, Müller CP. Resistance of recent measles virus wild-type isolates to antibody-mediated neutralization by vaccines with antibody. J Med Virol 2000; 62:91–8.
191. Centers for Disease Control and Prevention. Elimination of measles—South Korea, 2001–2006. MMWR Morb Mortal Wkly Rep 2007; 56:304–7.
192. Centers for Disease Control and Prevention. Progress toward the 2012 measles elimination goal—Western Pacific Region, 1990–2008. MMWR Morb Mortal Wkly Rep 2009; 58:669–73.
193. de Quadros CA, Andrus JK, Danovaro-Holliday MC, Castillo-Solorzano C. Feasibility of global measles eradication after interruption of transmission in the Americas. Expert Rev Vaccines 2008; 7:355–62.
194. Agocs MM, Markowitz LE, Straub I, Domok I. The 1988–1989 measles epidemic in Hungary: assessment of vaccine failure. Int J Epidemiol 1992; 21:1007–13.
195. Cutts FT, Markowitz LE. Successes and failures in measles control. J Infect Dis 1994; 170(Suppl 1):S32–41.
196. Hayden GF. Measles vaccine failure: a survey of causes and means of prevention. Clin Pediatr (Phila) 1979; 18:155–6161-153, 167.
197. Youwang Y, Ping W, Feng C. Serological and epidemiological effects and influence factors of primary immunization with current live attenuated measles vaccine (Hu191) among infants aged 6–15 months. Vaccine 2001; 19:1998–2005.
198. Dilraj A, Cutts FT, de Castro JF, et al. Response to different measles vaccine strains given by aerosol and subcutaneous routes to schoolchildren: a randomised trial. Lancet 2000; 355:798–803.
199. Huang KG, Spence MR, Deforest A, Bradley AT. Measles immunization in HCWs. Infect Control Hosp Epidemiol 1994; 15:4.
200. Kaplan LJ, Daum RS, Smaron M, McCarthy CA. Severe measles in immunocompromised patients. JAMA 1992; 267:1237–41.
201. Moss WJ, Cutts F, Griffin DE. Implications of the human immunodeficiency virus epidemic for control and eradication of measles. Clin Infect Dis 1999; 29:106–12.
202. Permar SR, Griffin DE, Letvin NL. Immune containment and consequences of measles virus infection in healthy and immunocompromised individuals. Clin Vaccine Immunol 2006; 13:437–43.
203. Helfand RF, Witte D, Fowlkes A, et al. Evaluation of the immune response to a 2-dose measles vaccination schedule administered at 6 and 9 months of age to HIV-infected and HIV-uninfected children in Malawi. J Infect Dis 2008; 198:1457–65.
204. Nair N, Moss WJ, Scott S, et al. HIV-1 infection in Zambian children impairs the development and avidity maturation of measles virus-specific immunoglobulin G after vaccination and infection. J Infect Dis 2009; 200:1031–8.
205. Bellini WJ, Rota JS, Lowe LE, et al. Subacute sclerosing panencephalitis: more cases of this fatal disease are prevented by measles immunization than was previously recognized. J Infect Dis 2005; 192:1686–93.
206. Escoffier C, Gerlier D. Infection of chicken embryonic fibroblasts by measles virus: adaptation at the virus entry level. J Virol 1999; 73:5220–4.
207. Sharma LB, Ohgimoto S, Kato S, et al. Contribution of matrix, fusion, hemagglutinin, and large protein genes of the CAM-70 measles virus vaccine strain to efficient growth in chicken embryonic fibroblasts. J Virol 2009; 83:11645–54.
208. Calain P, Roux L. The rule of six, a basic feature for efficient replication of Sendai virus defective interfering RNA. J Virol 1993; 67:4822–30.
