
Journal of Applied Microbiology

ORIGINAL ARTICLE

The oxidoreductases LivQ and NeoQ are responsible for the different 6′-modifications in the aminoglycosides lividomycin and neomycin

D. Clausnitzer<sup>1,2</sup>, W. Piepersberg<sup>1</sup> and U.F. Wehmeier<sup>3</sup>

1 BU Wuppertal, FB-C, Chemical Microbiology AG, Piepersberg, Wuppertal, Germany  
2 Institute of Biochemistry, Heinrich Heine University Düsseldorf, Düsseldorf, Germany  
3 BU Wuppertal, FB-G, Department of Sports Medicine, Wuppertal, Germany  

Keywords  
actinomycete(s), antibiotic(s), biochemical engineering, enzymes.

Correspondence  
Udo. F. Wehmeier, BU Wuppertal, FB-G, Department of Sports Medicine, Pauluskirchstrasse 7, D-42285 Wuppertal, Germany. E-mail: wehmeier@uni-wuppertal.de

2010/2091: received 18 November 2010, revised 3 June 2011 and accepted 13 June 2011

doi:10.1111/j.1365-2672.2011.05082.x

Abstract  
Aims: The 2-deoxystreptamine-containing aminoglycoside antibiotics (AGAs) constitute the largest subgroup of the aminoglycosides. Neomycin (NEO) and lividomycin (LIV) are both representatives of the pseudo-tetrasaccharide group among the NEO-type AGAs. While NEO contains a 6′-NH₂ group, the 6′-position remains unmodified in LIV. The aim of the study was to characterize the substrate specificities of the enzymes involved in the C-6′- and C-6″-modification in order to explain the different amination patterns.  
Methods and Results: We overproduced and purified the enzymes NeoQ (bifunctional 6′- and 6″-oxidoreductase) and NeoB (bifunctional 6′- and 6″-aminotransferase), which had been analysed before (Huang *et al.* 2007), and compared the enzymatic properties with the corresponding enzymes LivQ (postulated 6″-oxidoreductase, 72% identity to NeoQ) and LivB (postulated 6″-aminotransferase, 71% identity to NeoB) from the LIV pathway. By applying a newly established photometric assay, we proved that LivQ oxidized only pseudotetrasaccharic substrates at the 6″-position. In contrast, NeoQ accepted also the pseudodisaccharic paromamine as a substrate and oxidized the 6′- and 6″-positions on two different precursors of NEO. The aminotransferases LivB and NeoB both transfer NH₂ groups to the 6′-position in the precursor 6′-oxo-paromamine and to the 6″-position of 6″-oxo-neomycin C.  
Conclusions: The difference in the modification pattern of NEO and LIV at their 6′-positions is based only on the difference in the substrate specificities of the oxidoreductases LivQ and NeoQ, respectively. The aminotransferases LivB and NeoB share identical biochemical properties, and both are capable to transaminate the 6′ and also the 6″-position of the tested AGAs.  
Significance and Impact of the Study: Our data provide information to understand the structural variations in aminoglycosides and may be helpful to interpret variations in other natural product biosynthesis pathways.

Introduction  
Cyclitol-containing aminoglycosides (CAGs), with very few exceptions, are natural products of actinomycetes. After the discovery of the antibiotic streptomycin in 1943 (Schatz *et al.*, 1944), this group of substances became one of the major groups of natural therapeutics in medicine

and are still in focus of much molecular biological, biochemical and pharmaceutical research (Arya, 2007; Piepersberg *et al.* 2007; Wehmeier and Piepersberg 2009). The knowledge of the gene clusters for almost the entire well-studied CAGs and related cyclitol-containing metabolites (reviewed by Piepersberg *et al.* 2007) revealed that their biosynthetic pathways are all based largely on

D. Clausnitzer et al.

monofunctional enzymes catalysing single steps in the pathway. Core components in practically all natural products of this group are cyclitol units, which become decorated by side groups introduced (i) via modifying steps directly on the cyclitol and (ii) by glycosylation with various sugar moieties, which (iii) can become further modified by pathway-specific enzymes (Piepersberg and Wehmeier 2009).

The 2-deoxystreptamine (2DOS)-containing antibiotics constitute the largest subgroup of the aminoglycoside antibiotics (AGAs). Neomycin (NEO) was the first known 2DOS-containing antibiotic and was isolated first from *Streptomyces fradiae* and *S. albogriseus* (Waksman and Lechevalier 1949). The biosynthetic pathways for the NEO-related 2DOS-AGAs and their corresponding gene clusters seem to represent an ancient metabolic invention. The gene clusters for NEO-related compounds are found in taxonomically more distant bacterial producers such as in members of the families Streptomycetaceae, Micromonosporaceae (both actinobacteria) and Bacillaceae (firmicutes).

The aminoglycosides, together with other bioactive molecules of the ‘antibiotic’ type, might be molecular fossils, and thus the biosynthetic pathways for the NEO-related 2DOS-AGAs and their corresponding gene clusters could represent an ancient metabolic invention (Davies *et al.* 1993; Piepersberg *et al.* 2002, Piepersberg *et al.* 2007). The basic NEO biosynthesis cluster (*neo*-gene cluster; accession AJ629247) has been well conserved during evolution, however with considerable deviation, in gene content, organisation and sequence identity for the production of various NEO-related aminoglycosides, such as butirosin (BTR), paromomycin (PAR), ribostamycin (RIB) and lividomycin (LIV; cf. Piepersberg *et al.* 2007).

The intermediate paromamine is a common precursor in nearly all 2DOS AGA pathways, whereas its further transamination to form neamine (6′-amino-6′-deoxyparamine) seemed to be only used as a biosynthetic step in fewer of the NEO-related pathways, because the NEO-type 6′-desamino-6′-hydroxy variants like LIV (producer *Streptomyces lividus*; *liv*-gene cluster accession AJ748832) and PAR (producer *S. rimosus* ssp. *paromomycinus*; *par*-gene cluster accession AJ628955) can occur (Fig. 1). The 6′-NH₂ group in NEO is introduced via a two-step mechanism employing a 6′-oxidoreductase, NeoQ, and an aminotransferase, NeoB (Huang *et al.* 2005). The counterparts of NeoQ and NeoB in the LIV pathway, proteins LivQ and LivB, exhibit very high amino acid identities to their counterparts from the NEO pathway (NeoQ/LivQ 72% identity, NeoB/LivB 71% identity in the respective protein sequences). Moreover, the ParQ/ParB and LivQ/LivB set of enzymes does not show any indication of non-functionality in their primary structures (when compared

Oxidoreductases LivQ and NeoQ

with the 6′-dehydrogenases/6′-aminotransferases NeoQ/-NeoB), and thus the basis for the difference at C-6′ remained mysterious. As no deaminase is encoded by the *liv*-cluster, it was unclear why the 6′-OH group is maintained in LIV, although the 6″-position becomes transaminated.

The function of the NeoQ enzyme was shown only indirectly in combined assays with the aminotransferase in tests measuring the reverse reactions (Huang *et al.* 2007). NeoB represents a bifunctional enzyme that also catalyses the NH₂ group transfer at the 6″-position. The bifunctionality of the oxidoreductase NeoQ was only hypothesized so far. Therefore, we attempted to find out whether the enzymatic properties of the oxidoreductase LivQ and/or of the aminotransferase LivB are responsible for the difference in the amination patterns found at C-6′ in LIV when compared with NEO. By comparing the enzymatic properties of the enzyme pair LivQ/LivB with those of NeoQ/NeoB, we demonstrate herein that the altered substrate specificities of the oxidoreductases LivQ and NeoQ are responsible for the differences at C-6′ but that the aminotransferases LivB and NeoB both exhibit identical specificities and both are capable of transaminating the 6′ and also the 6″-position of the tested AGAs. Thus, our data show that structurally closely related enzymes may exhibit remarkably different properties.

## Materials and methods

### Enzymatic synthesis of the substrates for enzyme assays

The substrates for the enzyme assays were not commercially available and were synthesized enzymatically using overproduced enzymes.

#### N-acetyl-paromamine

The 2′-N-acetyltransferase AAC(2′) allowed the enzymatic synthesis of N-acetyl-paromamine as described by Hotta *et al.* (2000). The *aac(2′)* gene from *Streptomyces kasugaensis* was amplified by PCR applying the primers *aac2′_Start* (5′-**CATATGCCCCGGCATGACCGAG**-3′) and *aac2′_Stopp* (5′-**GGATCCCTCTACCAGAGATCCCCGT**-3′), which introduced *NdeI* and *BamHI* restriction sites (underlined), respectively. The PCR product was first ligated into pUCP21 (*NdeI/BamHI*) and then subcloned as *NdeI/BamHI* fragment into the *Escherichia coli/Streptomyces* shuttle vector pUWLHis (Table 1). The resulting plasmid pUWLHisac(2′) was transformed to *Streptomyces lividans* 66 TK64. A single transformant was inoculated in 50 ml Tryptone Soy Broth (TSB) (30 μg thiostrepton per ml) for subsequent overproduction of the 2′-N-acetyltransferase AAC(2′). After 72 h, cells were harvested by centrifugation, resuspended in 5 ml lysis buffer (50 mmol l⁻¹ Tris–HCl

© 2011 The Authors  
Journal of Applied Microbiology **111**, 642–651 © 2011 The Society for Applied Microbiology

Oxidoreductases LivQ and NeoQ

Neamine
![Structure](https://i.imgur.com/structure1.png)

Ribostamycin
![Structure](https://i.imgur.com/structure2.png)

6″-Hydroxyneomycin C
![Structure](https://i.imgur.com/structure3.png)

Neomycin C
![Structure](https://i.imgur.com/structure4.png)

Neomycin B
![Structure](https://i.imgur.com/structure5.png)

Paromamine
![Structure](https://i.imgur.com/structure6.png)

5-Ribosylparomamine
![Structure](https://i.imgur.com/structure7.png)

6‴-Hydroxyparomycin
![Structure](https://i.imgur.com/structure8.png)

Paromycin
![Structure](https://i.imgur.com/structure9.png)

Lividomycin B
![Structure](https://i.imgur.com/structure10.png)

Figure 1 Proposed biosynthetic pathways of neomycin and lividomycin (LIV). The steps starting from the common intermediate paromamine in the biosynthetic pathways for neomycin (Streptomyces fradiae) and LIV (S. lividus) are shown (cf. Piepersberg et al. 2007).

D. Clausnitzer et al.

Oxidoreductases LivQ and NeoQ

Table 1 Plasmids

| Plasmid          | Properties/product                                                                                   | Source/ref.               |
|------------------|-----------------------------------------------------------------------------------------------------|----------------------------|
| pET16b           | bla lacI P<sub>T7-Φ10</sub> lacO s10-leader His<sub>10</sub>-tag                                         | Studier et al. (1990)      |
| pJOE2775         | bla P<sub>rhaBAD</sub> rnb His-tag ColE1-ori                                                        | Volff et al. (1996)        |
| pUCPU21          | bla lacZ' ColE1-ori                                                                                | This work                 |
| pUWL201PW        | bla tsr P<sub>ermE</sub> ColE1-ori plJ101-ori                                                       | Doumith et al. (2000)      |
| pUWLHis          | bla tsr P<sub>ermE</sub> ColE1-ori plJ101-ori                                                       | This work                 |
| p201PW/neoF      | 1·1-kb Ndel/BamHI neoF fragment in pUWL201PW (Ndel/BamHI)                                              | This work                 |
| pET16/neoD       | 0·8-kb neoD-DNA-fragment from pUCPU/neoD+ (Ndel/BamHI) in pET16b (Ndel/BamHI) ligiert                    | This work                 |
| pUWL201PWlivQEFHis | 1·6-kb Ndel/HindIII livQ His-tag fusion fragment in pUWL201PW (Ndel/HindIII)                         | This work                 |
| pUWL201PWneoBEFHis | 1·4-kb Ndel/HindIII neoB His-tag fusion fragment in pUWL201PW (Ndel/HindIII)                         | This work                 |
| pUWLHisaac(2')   | 0·5-kb Ndel/BamHI aac(2') fragment in pUWLHis (Ndel/BamHI)                                            | This work                 |
| pUWLHislivB     | 1·3-kb Ndel/BamHI livB fragment in pUWLHis (Ndel/BamHI)                                               | This work                 |
| pUWLHisneoQ     | 1·7-kb Ndel/BamHI neoQ fragment in pUWLHis (Ndel/BamHI)                                               | This work                 |

pH 8, 300 mmol l<sup>−1</sup> NaCl, 10 mmol l<sup>−1</sup> imidazole) and disrupted by sonication. The Aac(2') protein was purified in a batch approach by nickel affinity gel (Sigma-Aldrich, Munich, Germany) using wash buffer (15 mmol l<sup>−1</sup> imidazole) and elution buffer (250 mmol l<sup>−1</sup> imidazole).

The enzyme assays for the enzymatic synthesis of N-acetyl-paromamine contained paromamine (3 mmol l<sup>−1</sup>), 50 mmol l<sup>−1</sup> Tris–HCl buffer (pH 7·5), N-acetyl-coenzyme A (7 mmol l<sup>−1</sup>) and purified AAC(2') protein (10 μg for 1 ml) and were incubated at 30°C overnight.

Hydroxyneomycin C

The protein NeoD was characterized as N-acetyl-hydroxyneomycin C deacetylase (Yokoyama et al. 2008) and was used for the production of hydroxyneomycin C. The neoD gene was amplified by PCR using cosmid SfrF04 (Piepersberg et al. 2007) as template and the primers Start_neoD5′-(GGGAGGAGGAGCCATATGGGGTGAGCCG-3′) and neoD_Stopp (5′-GCGCCGTGGATCCACGAGGGC-3′). The fragment was ligated to pET16b (hydrolysed with NdeI and BamHI), resulting in pET16/neoD (Table 1). Escherichia coli BL21 (DE3) was used as the host for protein overproduction. And 50 ml of Luria Broth (LB) broth (100 μg ampicillin per ml) was inoculated with an overnight culture (starting OD<sub>595</sub> 0·1) and grown at 30°C to OD<sub>595</sub> 0·6, and then isopropyl-β-thio-galactopyranoside (IPTG) (0·4 mmol l<sup>−1</sup> end concentration) was added to induce gene expression. The cultures were grown for further 4 h, and cells were collected by centrifugation and disrupted immediately in 5 ml buffer L by sonication. Cell-free extracts were obtained by centrifugation at 13 000 g for 30 min at 4°C, and the supernatant was concentrated using centrifugal concentrators (Vivaspin; Vivascience AG, Hannover, Germany) to 500 μl.

To produce hydroxyneomycin C, we added 20% concentrated NeoD extract (v/v) to enzyme assays with NeoF. After incubation overnight at 37°C, N-acetyl-hydroxyneomycin C was completely converted to the desired product hydroxyneomycin C.

N-acetyl-hydroxyneomycin C

An enzymatic approach for the production of N-acetyl-hydroxyneomycin C using the glycosyltransferase NeoF was already described (Yokoyama et al. 2008). The neoF gene was amplified using cosmid SfrF04 (Piepersberg et al. 2007) as template and the primers neoF_Start (5′-GCCGG-AGGTGCCATATGGGCTGAGGCG-3′) and neoF_Stopp (5′-CTCCGGCCGGATCCCCACGTGCGGC-3′). The PCR fragment was hydrolysed with NdeI and BamHI and then first ligated to pUPU20 and then introduced into the E. coli/Streptomyces shuttle vector pUWL201PW (Doumith et al. 2000) to give p201PW/neoF (Table 1), which was transformed to S. lividans 66 TK23. Then, 30 ml of TSB (30 μg thiostrepton per ml) were inoculated with a fresh S. lividans 66 TK23-p201PW/neoF transformant and cells were grown at 28°C for 72 h. The cells were collected by centrifugation, resuspended in 3 ml buffer L (50 mmol l<sup>−1</sup> Tris–HCl pH 7·5; 10 mmol l<sup>−1</sup> MgCl<sub>2</sub>) and disrupted by sonication. The cell debris was removed by centrifugation at 13 000 g for 30 min at 4°C, and the cell-free supernatant was used.

The enzymatic assays according to Yokoyama et al. (2008) contained ribostamycine (3 mmol l<sup>−1</sup>), UDP-N-acetylglucosamine (UDP-GlcNAc, 3 mmol l<sup>−1</sup>), MgCl<sub>2</sub> (5 mmol l<sup>−1</sup>), Tris–HCl buffer (50 mmol l<sup>−1</sup>, pH 7·5) and 20% NeoF extract (v/v) and were incubated at 30°C overnight.

Purification of the enzymatically synthesized substrates

The synthesized substrates were purified and isolated by polymeric solid-phase extraction (SPE strata-X-C; Phenomenex, Aschaffenburg, Germany) to remove by-components and to concentrate the substrates. The columns were conditioned with 1 ml methanol and then equilibrated with H<sub>2</sub>O. The samples were first acidified by

© 2011 The Authors  
Journal of Applied Microbiology **111**, 642–651 © 2011 The Society for Applied Microbiology 645

Oxidoreductases LivQ and NeoQ

adding an equal volume of formic acid and afterwards loaded on the column. Then, the columns were washed with 2 ml methanol/5% formic acid (1 : 1) and then with 1 ml H₂O. The substrates were eluted with 1 ml methanol/25% ammonia water (1 : 1) and vaporized in a speed vac. The dried substrate powders were dissolved with water to a concentration of approximately 10 mmol l⁻¹.

### Overexpression and purification of the oxidoreductases NeoQ and LivQ

The neoQ gene was amplified by PCR using cosmid SfrF04 (Piepersberg et al. 2007) as template and the primers neoQ-A (5′-GGCACCATATGAAGCGCCTTCG-3′) and neoQ-ES (5′-GCGCGGGGATCCTCAGACGTGC-3′). For PCR amplification of the livQ gene, the cosmid SliD01 (Piepersberg et al. 2007) served as template, and the primers livQ-A (5′-GCATCATATGGAGCGCCTCG-3′) and livQ-EF (5′-GTGCCGGATCCACATGGG-3′) were used to introduce NdeI and BamHI restriction sites (underlined), respectively. The PCR fragments were hydrolysed with NdeI and BamHI and ligated to pUC-PU21. The neoQ gene was directly subcloned into the expression vector pUWLHis (pUWLHisneoQ, Table 1). The livQ gene was inserted as NdeI/BamHI fragment into the vector pJOE2775 that fused a fragment encoding a His-tag to the 3′-end of livQ. Then, the 3′-end-modified livQ fragment was isolated (NdeI/HindIII) and ligated into pUWL201PW (pUWL201PWlivQEFHis, Table 1). As a result, the LivQ protein was produced with a C-terminal His-tag. Both expression plasmids, pUWLHisneoQ and pUWL201PWlivQEFHis, were transformed to *S.* livi-dans 66 TK64. Single transformants were inoculated in 50 ml TSB (30 μg thiostrepton per ml) and grown at 28°C for 72 h. Cells were harvested by centrifugation at 4000 g for 10 min and stored at −20°C. The overproduced heterologous proteins were purified by nickel affinity gel chromatography as described previously, and the protein was eluted with a buffer containing Tris-HCl (50 mmol l⁻¹, pH 8), imidazole (250 mmol l⁻¹), NaCl (500 mmol l⁻¹) and glycerol (10%). The protein solutions of the purified NeoQ and LivQ proteins appeared yellow, as already observed (Huang et al. 2007).

### Spectrophotometric assay for the measurement of oxidoreductase activity of NeoQ and LivQ

The enzymatic activities of NeoQ and LivQ were determined in coupled enzyme assays including 3-(4,5-dimethylthiazolyl-2)-5-diphenyltetrazoliumbromide (MTT; Sigma-Aldrich) and phenazine methosulfate (PMS; Sigma-Aldrich) according to tests described by Lengeler and Lin (Lengeler and Lin 1972). The oxidation of the substrates by NeoQ or LivQ correlates with the reduction in equimolar amounts of the cofactor FAD to FADH₂. FADH₂ then reduces PMS which becomes oxidized by MTT, resulting in the production of MTT formazan. MTT formazan can be monitored at 570 nm (Nakamura et al. 1986). The solutions for the tests were always prepared freshly. Each assay contained 50 μl PMS (0·1%), 100 μl substrate (2·5 mmol l⁻¹) and 300 μl of mixture A (containing 0·33 ml of 1 mol l⁻¹ Tris–HCl pH 7·5, 0·07 ml of 0·5% MTT, 0·1 ml Triton X-100 20% solution and 5·97 ml distilled H₂O). The reactions were started by adding purified protein (10 μl; 2·5 μg protein per μl), and the OD₅₇₀ was monitored continuously for 10 min. The enzyme activities are expressed in mmol l⁻¹ min⁻¹ mg⁻¹ protein.

### Overexpression and purification of the aminotransferases NeoB and LivB

The neoB gene was amplified using cosmid SfrF04 (Piepersberg et al. 2007) as template and primers (neoB-A) 5′-CCCCTGGGGGTGACATATGACGA-3′ and (neoB-EF) 5′-GGGCGGGATCCGTCGTC-3′. The livB gene was amplified by PCR using the cosmid SliD01 (Piepersberg et al. 2007) as template applying the primers (livB-A) 5′-GGACCGCATATGACGCGAAATTCC-3′ and (livB-ES) 5′-GTTGGGATCCCTGTTC-3′, which introduced unique NdeI or BamHI restriction sites (underlined). The genes were ligated into the plasmid pUCPU21 (NdeI/BamHI). For cloning the neoB gene, we followed the same strategy described for livQ. First, the fragment was cloned in pJOE2775, and then neoB was isolated as NdeI/HindIII fragment and inserted into pUWL201PW (pUWL201-PWneoBEFHis, Table 1). The livB gene was cloned as NdeI/BamHI fragment into the *E. coli*/Streptomyces shuttle vector pUWLHis (pUWLHislivB, Table 1). The overproduction and purification of both proteins were performed as described previously for the oxidoreductases NeoQ and LivQ but for NeoB SPMR-medium was used (30 μg thiostrepton/ml) (Babcock and Kendrick 1988). The purified LivB protein solution always appeared pink.

### Aminotransferase assay

According to the already-described conversion of paraom-amine to neamine by NeoQ and NeoB in coupled assays (Huang et al. 2007), the activities of NeoB and LivB were compared with a number of substrates. The assays (100 μl) contained the various substrates (0·5 mmol l⁻¹), L-glutamate (2·5 mmol l⁻¹), pyridoxal phosphate (PLP, 0·1 mmol l⁻¹), Tris–HCl (50 mmol l⁻¹ pH 8) and 25 μg of the purified dehydrogenases (NeoQ or LivQ) and of the purified aminotransferases (NeoB or LivB), respectively.

D. Clausnitzer *et al.*

After incubation overnight at 37°C, the tests were stopped, treated like the substrates using polymeric SPE and then analysed by thin layer chromatography (TLC) on silica thin-layer sheets (F₂₅₄; Merck, Darmstadt, Germany) using methanol/chloroform/12·5 % ammonium hydroxide (1 : 0·2 : 1) as solvent. The plates were sprayed with a 0·2 % ninhydrin-ethanol solution, dried and heated with a heat gun to visualize the spots.

### Results

LivQ is a monofunctional 6″′-hydroxy-neomycin C oxidoreductase

The activity of the oxidoreductase NeoQ was determined by indirectly employing the reverse reactions and by analysing the structures of the reaction products (Huang *et al.* 2007). No direct enzyme assays allowing the elucidation of the oxidoreductases like LivQ and NeoQ in AGA pathways had been described so far. The intermediates that were necessary as substrates for the enzyme tests were commercially not available. Therefore, we synthesized 2′-N-acetyl-paromamine, 2″′-N-acetyl-6″′-hydroxyneomycin C and 6″′-hydroxyneomycin C enzymatically as described in Materials and methods. The generated substrates were purified and subsequently separated on TLC for comparison with standard substances (Fig. 2). The enzymes LivQ and NeoQ were overproduced as His-tagged proteins using *S. lividans* 66 TK64/pUWL201PWlivQEFHis and *S. lividans* 66 TK23/pUWLHisneoQES and were purified (Fig. 3). We attempted to develop an enzyme assay that allowed the direct analysis of the Q-enzymes. According to tests described by Lengeler and Lin (1972), enzymatic activities of NeoQ and LivQ were determined in coupled enzyme assays including 3-(4,5-dimethylthiazolyl-2)-2, 5-diphenyltetrazoliumbromide (MTT) and PMS, resulting in the production of MTT formazan (see Materials and methods) which can be monitored at 570 nm (Nakamura *et al.* 1986). Using this test system, the enzymatic specificities can be calculated from the photometric data, and no further purification step or structural analyses will be necessary. We included sucrose as the control substrate to exclude unspecific enzyme activities or substrate-independent MTT formazan production. In all controls, we never detected an increase in the absorbance (not shown). In our assays, NeoQ oxidized paromamine (4·2 mmol l⁻¹ min⁻¹ mg⁻¹ protein), 6″′-hydroxy-neomycin C (specific activity, 6·4 mmol l⁻¹ min⁻¹ mg protein⁻¹) and also with a reduced specific activity 2″′-N-acetyl-6″′-hydroxyneomycin C (1·0 mmol l⁻¹ min⁻¹ mg⁻¹ protein). From these results, we concluded that NeoQ is a bifunctional enzyme, as it had been shown indirectly before (Huang *et al.* 2007). With all other substrates, even with the aminoglycoside

---

**Figure 2** TLC analysis of the enzymatically synthesized substrates. The synthesized substrates (lanes 2–3, 6–7) were purified using polymeric solid-phase extraction (SPE). One microlitre of each substrate solution was applied to a silica thin-layer sheet (Materials and methods). Lane 1: 10 mmol l⁻¹ paromamine (standard, Roussel Uclaf, Romainville, France); lane 2: paromamine (SPE); lane 3: 2′-N-acetyl-paromamine (SPE); lane 4: 10 mmol l⁻¹ ribostamycin (RIB) (standard, Sigma-Aldrich); lane 5: RIB (SPE); lane 6: 2″′-N-acetyl-6″′-hydroxyneomycin C (SPE); and lane 7: 6″′-hydroxyneomycin C (SPE), lane 8: 10 mmol l⁻¹ neomycin B (standard, Sigma-Aldrich).

---

(a)

| kDa |
| --- |
| 80 |
| 60 |
| 40 |
| 30 |

M 1 2 3 4 5

(b)

| kDa |
| --- |
| 80 |
| 60 |
| 40 |
| 30 |

M 1

**Figure 3** SDS-PAGE (10%) of the enzymes NeoQ (A) and LivQ (B). (a) Overproduction of NeoQ in Streptomyces *lividans* 66 TK23, lanes 1 and 2 crude extract: one pUWLHis, two pUWLHisneoQES; lanes 3 and 4 soluble protein fraction: three pUWLHis, four pUWLHisneoQES; lane 5: His-NeoQ purified using nickel affinity gel; the arrows show the NeoQ protein. (b) Lane 1: LivQ overproduced in *S. lividans* 66 TK64/pUWL201PWlivQEFHis and purified by nickel affinity gel; lanes M: molecular weight marker.

Oxidoreductases LivQ and NeoQ

(a) NeoQ
![Graph](graph_a.png)

(b) LivQ
![Graph](graph_b.png)

(c)
| Substrate¹ | Specific enzyme activity² NeoQ | Specific enzyme activity² LivQ |
|------------|--------------------------------|--------------------------------|
| ○          | 2′-N-acetyl-paromamine         | 0·0                           | 0·1                           |
| ▲          | paromamine                     | 4·2                           | 0·1                           |
| △          | ribostamycin                   | 0·0                           | 0·0                           |
| ●          | 2″′-N-acetyl-6″′-hydroxyneomycin C | 1·0                           | 0·3                           |
| ○          | 6″′-hydroxyneomycin C           | 6·4                           | 3·5                           |
| ×          | paromomycin                    | 0·1                           | 0·0                           |

Figure 4 Oxidoreductase activities of NeoQ and LivQ. The diagrams show the extinctions in 15-s intervals (OD₅₇₀ₙₘ) of the MTT/phenazine methosulfate oxidoreductase assays from NeoQ (a) and LivQ (b) using the different substrates ¹(purified and isolated by polymeric solid-phase extraction, see Materials and methods), and the corresponding symbols are explained in c. (c) The calculated specific NeoQ/LivQ enzyme activities ²(in mmol l⁻¹ (MTT) min⁻¹ mg protein⁻¹).

PAR (Fig. 1), which still contains the 6′-OH group, no activity was measured (Fig. 4). From this observation, we can conclude that our test conditions clearly differentiate among the various substrates and that no other unspecific oxidations were seen. The fact that PAR was not accepted as a substrate also clearly shows that the 6′-position is recognized by NeoB only in the pseudo-disaccharide paromamine but not in the pseudo-tetrasaccharide 6″′-hydroxy-neomycin C. The reaction product 6″′-hydroxy-neomycin C was purified and isolated by polymeric SPE from the assays (cf. Fig. 2). As seen in Fig. 2 (lane 7), 6″′-hydroxy-neomycin C was always completely oxidized by NeoB because the substrate neomycin C was never detected on TLCs.

In tests with LivQ, enzymatic activities were detected only when 6″′-hydroxy-neomycin C was applied (specific activity, 3·5 mmol l⁻¹ min⁻¹ mg protein⁻¹), and a very strongly reduced activity with 2″′-N-acetyl-6″′-hydroxy-neomycin C (0·3 mmol l⁻¹ min⁻¹ mg⁻¹ protein). No other substrate, including paromamine, was oxidized by LivQ. From these data, we inferred that LivQ, in contrast to NeoQ, is only a monofunctional enzyme and responsible for the fact that the OH group at the 6′-position in LIV remains unmodified, because the oxidation of the C-6′ OH group would be a prerequisite for the subsequent aminotransferase reaction.

LivB is a bifunctional transaminase

From the finding that LivQ is only a monofunctional oxidoreductase, the question arose whether LivB is a monofunctional enzyme too. The aminotransferases LivB and NeoB were overproduced as His-tagged proteins and purified (Fig. 5). As aminotransferases need PLP as a cofactor, the absorption spectra of the proteins were measured. For both proteins, LivB and NeoB, two typical absorption peaks for PLP-containing enzymes (at 325 nm for pyridoxal monophosphate and 415 nm for PLP; Vazquez *et al.* 1991; Stetefeld *et al.* 2006; Huang *et al.* 2007; Kudo and Eguchi 2009) were detected (not shown). When analysing NeoB, the peak at 415 nm was more intensive, which might indicate that the purified NeoB enzyme had more PLP bound, which caused the pink colour of the purified protein fraction. With the purified NeoB and LivB, coupled assays as described before for NeoQ and NeoB (Huang *et al.* 2007) were performed. In these assays, we applied paromamine as substrate. Paromamine is the first substrate accepted by NeoQ and NeoB (Fig. 1) but was not accepted by LivQ, as shown previously. In control assays using NeoQ and NeoB, we were able to monitor the formation of neamine (data not shown), as shown by Huang *et al.* (2007). In tests using the enzymes LivQ and LivB, no neamine was produced as

D. Clausnitzer et al.

Oxidoreductases LivQ and NeoQ

(a)
kDa
80
60
40
30
M 1 2 3 4 5

(b)
kDa
60
40
30
1 2 3 4 M 5

Figure 5 SDS–PAGE (10%) of the enzymes LivB (a) and NeoB (b). (a) Overproduction of LivB using *Streptomyces lividans* 66TK64-pUWLHislivBES. Lane 1: crude extract *S.* *lividans* 66 TK64-pUWLHislivBES; lane 2: crude extract *S.* *lividans* 66 TK64-pUWLHis (control); lane 3: soluble protein fraction *S.* *lividans* 66 TK64-pUWLHislivBES; lane 4: soluble protein fraction *S.* *lividans* 66 TK64-pUWLHis; and lane 5: LivB purified by nickel affinity gel. (b) Overproduction of NeoB using *S.* *lividans* 66 TK64-pUWL201PWneoBEFHis. Lane 1: crude extract *S.* *lividans* 66 TK64-pUWL201PWneoBEFHis; lane 2: crude extract *S.* *lividans* 66 TK64-pUWL201PW (control); lane 3: soluble protein fraction *S.* *lividans* 66 TK64-pUWL201PWneoBEFHis; lane 4: soluble protein fraction *S.* *lividans* 66 TK64-pUWL201PW; and lane 5: NeoB purified by nickel affinity gel. The arrows show the NeoB protein. Lanes M: molecular weight marker.

we had expected (not shown). This observation was in good agreement with the enzymatic data for LivQ shown previously. Also in tests applying LivQ together with NeoB, neamine was not produced (Fig. 6), again proving our results for LivQ. However, in coupled assays, using the oxidoreductase NeoQ in combination with the transaminase LivB, neamine was produced (Fig. 6). In these tests, NeoQ catalysed the oxidation of paromamine, and LivB, the subsequent transaminase reaction. As judged from the TLC picture, the conversion of PAR to neamine was nearly quantitative (Fig. 6).

Our data clearly showed that the aminotransferase LivB still is a bifunctional enzyme and obviously shares the same enzymatic properties as NeoB. We also demonstrate that the altered substrate specificities of the oxidoreductases NeoQ and LivQ alone are responsible for the different modification patterns at the 6′-position found in NEO and LIV.

Discussion

The gene clusters for the AGAs of the NEO family encode for many highly conserved proteins (70% and even more identical amino acids; Kudo and Eguchi 2009; Piepersberg *et al.* 2007; Llewellyn and Spencer 2006), and in all gene clusters, only the so-called Q-genes and B-genes (e.g. neoQ and neoB; cf. Piepersberg *et al.* 2007) were identified, which code for enzymes that are involved in the modifications at the 6′- and 6″-positions (Kudo and Eguchi 2009; Piepersberg *et al.* 2007; Llewellyn and Spencer 2006). The fact that some pseudo-tetrasaccharides

L-glutamate →
Tris-buffer →
Paromamine →
Neamine →

1 2 3 4 5

Figure 6 TLC analysis of the combinational aminotransferase assays. Lane 1: standard neamine, lane 2: paromamine (standard; Roussel Uclaf, Romainville, France), lane 3: paromamine with the enzymes LivQ and NeoB, lane 4: paromamine with the enzymes NeoQ and LivB, lane 5: paromamine (standard; Roussel Uclaf). The TLC was performed as described in Materials and methods.

of the NEO family of AGAs contain a 6′-hydroxy group (LIV or PAR) and others a 6′-NH₂ group (NEO) but all have a NH₂ at the 6‴-position has been discussed in different ways. It has been speculated that 6′-transamination is the last step in the biosynthesis of NEO-type

© 2011 The Authors
Journal of Applied Microbiology **111**, 642–651 © 2011 The Society for Applied Microbiology

Oxidoreductases LivQ and NeoQ

aminoglycosides and that in some producers the export of the 6′-hydroxylated precursor became more efficient during the evolution (Piepersberg *et al.* 2007). These producers then would preferably produce 6′-hydroxylated aminoglycosides (e.g. LIV or PAR), and this would suggest that 6′-transaminated product(s) would be likely to arise as minor components in the culture broth. A second hypothesis suggested that a subtle control mechanism for substrate differentiation may distinguish the two biosynthetic systems (Llewellyn and Spencer 2006).

From the data presented in this manuscript, it becomes clear that altered substrate specificities of the oxidoreductases involved in the transamination at the respective positions are responsible for the different amination patterns found in NEO or LIV. Although the enzymes LivQ and NeoQ are highly related (72% identity) and they exhibit the same conservation as the aminotransferases NeoB and LivB (71% identity), the substrate specificities of the oxidoreductases LivQ and NeoQ differ while the aminotransferases LivB and NeoB are functionally identical. This observation clearly shows that even highly conserved enzymes may strongly differ in their biochemical capacity and that one cannot predict identical biochemical properties only from protein alignments. Among the gene clusters for AGAs, more similar examples had been found, e.g. the conservation of the ParQ/ParB enzymes mentioned in the Introduction (Piepersberg *et al.* 2007). As PAR is also synthesized as a precursor during the LIV biosynthesis (cf. Fig. 1), one can predict that ParQ and ParB will share the same enzymatical properties as LivQ and LivB. From the evolutionary point of view, the observation that the oxidoreductases exhibit altered substrate specificities and not the aminotransferases is also quite evident and obviously logical. If the oxidoreductase LivQ would still be able to oxidize paromamine and the aminotransferase LivB had lost the ability to modify the 6′-position, then the intermediate 6′-oxo-paromamine, which would be rather instable, would accumulate.

LivQ and NeoQ also accepted, although with strongly reduced activities, 2″′-N-acetyl-6″′-hydroxyneomycin C as a minor substrate, while 2′-N-acetyl-paromamine was not accepted by the bifunctional enzyme NeoB. It is still unclear how the bifunctional oxidoreductases and transaminases that are found in the AGA pathways handle the different substrates. The NeoQ/B enzymes both recognize a pseudo-disaccharide (paromamine) and a pseudo-tetrosaccharide (6″′-hydroxyneomycin C). When comparing the protein sequences, it becomes obvious that a stretch of six amino acids (HRGSRH) corresponding to aa 172–178 of LivQ is present also in the sequence of ParQ (SNGARH) but deleted in NeoQ and RibQ. Whether this stretch determines that the enzmyes LivQ and ParQ are monofunctional enzymes and that the enzymes NeoQ and

RibQ are bifunctional enzymes remains to be solved. As long as crystal structures are not available, it is speculative how the structurally different substrates are interacting with the active site of the enzymes and which part of the substrate enters the enzyme. When analysing the proteins LivQ and NeoQ using the LOOPP-program (Linear Optimization of Protein Potentials, http://cbsuapps.tc.cornell.edu/loopp.aspx, Meller and Elber 2001) for the prediction of the putative protein structures, one loop in the centre of the proteins differs remarkably while the overall structure of both proteins is obviously more or less identical in all other parts. As 2″′-N-acetyl-6″′-hydroxyneomycin C is recognized by both oxidoreductases, NeoQ and LivQ, one can speculate that the acetyl residue of this pseudo-tetrosaccharide is located more distant to the active center and does not interfere as efficiently as the acetylresidue in 2′-N-acetyl-paromamine. It is also not clear whether the interactions of the enzymes with the different substrates are based on identical residues in the active centre or whether these enzymes contain two substrate-binding sites. From the enzyme’s point of view, they just have to recognize the 6-position of a glucose. As we have shown that the aminoglycoside PAR, which is also a pseudo-tetrosaccharide containing a 6′-OH group (Fig. 1), is not accepted as a substrate by the bifunctional enzyme NeoQ (Fig. 4), the recognition and differentiation of the substrates must be even more complex. It also becomes apparent that the order of the modification steps by bifunctional enzymes like NeoQ is strongly fixed, obviously by structural features of the intermediates. Counterparts of the ‘Q’ and ‘B’ enzymes are also encoded by other AGA gene clusters, e.g. the gene cluster for BTR (BtrQ and BtrB), kanamycin (KanQ and KanB; Piepersberg *et al.* 2007; Kharel *et al.* 2004a; Yanai and Murakami 2004) and tobramycin (TobB and TobQ; Piepersberg *et al.* 2007; Kharel *et al.* 2004b). The 6′-minotransferase function of BtrB and of TobB, also termed TacB, was already shown (Huang *et al.* 2007; Yu *et al.* 2008). However, the amino acid similarities of the mentioned enzymes to their counterparts from the NEO or LIV pathway are strongly reduced (*c.* 35% identical amino acid residues for the ‘B’enzymes; *c.* 40–45% for the ‘Q’-enzymes), and phylograms of the ‘Q’ and ‘B’ enzymes show that only the enzymes from pathways of NEO, LIV, PAR and RIB cluster together (Piepersberg *et al.* 2007).

In summary, we have shown that enzymes, which are obviously very closely related and originate from similar pathways, may exhibit remarkably different enzymatic properties that might explain the variations of endproducts that are not apparent from the analysis of the protein sequences. From the biotechnical point of view, the simple exchange of the genes encoding the oxidoreductases from closely related pathways can be used for the *in vivo* generation of modified endproducts with altered amination patterns.

D. Clausnitzer et al.

Acknowledgements

This work was funded by the GenoMik-plus programme by the German BMBF.

References

Arya, D.P. (ed.) (2007) Aminoglycoside Antibiotics: From Chemical Biology to Drug Discovery. Hoboken, NJ: Wiley.
Babcock, M.J. and Kendrick, K.E. (1988) Cloning of DNA involved in sporulation of Streptomyces griseus. J Bacteriol 170, 2802–2808.
Davies, J., von Ahsen, U., and Schroeder, R. (1993) Antibiotics and the RNA world: a role for low-molecular-weight effectors in biochemical evolution? In The RNA World ed. Gesteland, R.F. and Atkins, J.F. pp. 185–204. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.
Doumith, M., Weingarten, P., Wehmeier, U.F., Salah-Bey, K., Benhamou, B., Capdevila, C., Michel, J.M., Piepersberg, W. *et al.* (2000) Analysis of genes involved in 6-deoxyhexose biosynthesis and transfer in Saccharopolyspora erythraea. Mol Gen Genet 264, 477–485.
Hotta, K., Sunada, A., Ikeda, Y. and Kondo, S. (2000) Double stage activity in aminoglycoside antibiotics. J Antibiot 53, 1168–1174.
Huang, F., Haydock, S.F., Mironenko, T., Spiteller, D., Li, Y. and Spencer, J.B. (2005) The neomycin biosynthetic gene cluster of Streptomyces fradiae NCIMB 8233: characterization of an aminotransferase involved in the formation of 2-deoxystreptamine. Org Biomol Chem 3, 1410–1418.
Huang, F., Spiteller, D., Koordanally, N.A., Li, Y., Llewellyn, N.M. and Spencer, J.B. (2007) Elaboration of neosamine rings in the biosynthesis of neomycin and butirosin. Chembiochem 8, 283–288.
Kharel, M.K., Subba, B., Basnet, D.B., Woo, J.S., Lee, H.C., Liou, K. and Sohng, J.K. (2004a) A gene cluster for biosynthesis of kanamycin from Streptomyces kanamyceticus: comparison with gentamicin biosynthetic gene cluster. Arch Biochem Biophys 429, 204–214.
Kharel, M.K., Basnet, D.B., Lee, H.C., Liou, K., Woo, J.S., Kim, B.G. and Sohng, J.K. (2004b) Isolation and characterization of the tobramycin biosynthetic gene cluster from Streptomyces tenebrarius. FEMS Microbiol Lett 230, 185–190.
Kudo, F. and Eguchi, T. (2009) Biosynthetic enzymes for the aminoglycosides butirosin and neomycin. Methods Enzymol 459, 493–519.
Lengeler, J. and Lin, E.C. (1972) Reversal of the mannitol-sorbitol diauxie in Escherichia coli. J Bacteriol 112, 840–848.
Llewellyn, N.M. and Spencer, J.B. (2006) Biosynthesis of 2-deoxystreptamine-containing aminoglycoside antibiotics. Nat Prod Rep 23, 864–874.
Meller, J. and Elber, R. (2001) Linear programming optimization and a double statistical filter for protein threading protocols. Proteins 45, 241–261.


Oxidoreductases LivQ and NeoQ

Nakamura, A., Urabe, I. and Okada, H. (1986) Anchimeric assistance in the intramolecular reaction of glucose-oxidoreductase-polyethylene glycol NAD conjugate. J Biol Chem 261, 16792–16794.
Piepersberg, W., Diaz-Guardamino, P.M., Zhang, C.S., Stratmann, A., and Wehmeier, U.F. (2002) Recent advances in biosynthesis and regulation of aminoglycosides. In Microbial Secondary Metabolites ed. Fierro, F. and Martin, I.F. pp. 1–26. Kerala, India: Research Signpost.
Piepersberg, W. and Wehmeier, U.F. (2009) Aminoglycosides, bioactive bacterial metabolites. In Encyclopedia of Microbiology, 3rd edn, Chapter 38 ed. Schaechter, M. pp. 183–192. San Diego, USA: Elsevier.
Piepersberg, W., Aboshanab, K., Schmidt-Beißner, H. and Wehmeier, U.F. (2007) The biochemistry and genetics of aminoglycoside producers. In Aminoglycoside Antibiotics: From Chemical Biology to Drug Discovery ed. Arya, D.P. pp. 15–118. Hoboken, NJ: Wiley.
Schatz, A., Bugie, E. and Waksman, S. (1944) Streptomycin: a substance exhibiting antibiotic activity against gram-positive and gram-negative bacteria. Proc Soc Exp. Biol Med. 55, 66–69.
Stetefeld, J., Jenny, M. and Burkhard, P. (2006) Intersubunit signaling in glutamate-1-semialdehyde-aminomutase. Proc Natl Acad Sci USA 103, 13688–13693.
Studier, F.W., Rosenberg, A.H., Dunn, J.J. and Dubendorff, J.W. (1990) Use of T7 RNA polymerase to direct expression of cloned genes. Methods Enzymol 185, 60–89.
Vazquez, M.A., Munoz, F., Donoso, J. and Garcia Blanco, F. (1991) Spectroscopic study of the Schiff bases of dodecylamine with pyridoxal 5′-phosphate and 5′-deoxypyridoxal. A model for the Schiff bases of pyridoxal 5′-phosphate in biological systems. Biochem J 279, 759–767.
Volff, J.N., Eichenseer, C., Viell, P., Piendl, W. and Altenbuchner, J. (1996) Nucleotide sequence and role in DNA amplification of the direct repeats composing the amplifiable element AUD1 of Streptomyces lividans 66. Mol Microbiol 21, 1037–1047.
Waksman, S.A. and Lechevalier, H.A. (1949) Neomycin, a new antibiotic active against streptomycin-resistant bacteria, including tuberculosis organisms. Science 109, 305–307.
Wehmeier, U.F. and Piepersberg, W. (2009) Enzymology of aminoglycoside biosynthesis – deduction from gene clusters. Methods Enzymol 459, 457–489.
Yanai, K. and Murakami, T. (2004) The kanamycin biosynthetic gene cluster from Streptomyces kanamyceticus. J Antibiot 57, 351–354.
Yokoyama, K., Yamamoto, Y., Kudo, F. and Eguchi, T. (2008) Involvement of two distinct N-acetylglucosaminyltransferases and a dual-function deacetylase in neomycin biosynthesis. Chembiochem 9, 865–869.
Yu, Y., Hou, X., Ni, X. and Xia, H. (2008) Biosynthesis of 3′-deoxy-carbamoylkanamycin C in a Streptomyces tenebrarius mutant strain by tacB gene disruption. J Antibiot (Tokyo) 61, 63–69.

© 2011 The Authors
Journal of Applied Microbiology **111**, 642–651 © 2011 The Society for Applied Microbiology
