
MICRORNAS IN DEVELOPMENT AND DISEASE

Danish Sayed and Maha Abdellatif

Cardiovascular Research Institute, Department of Cell Biology and Molecular Medicine, University of Medicine and Dentistry of New Jersey, Newark, New Jersey

Sayed D, Abdellatif M. MicroRNAs in Development and Disease. Physiol Rev 91: 827-887, 2011; doi:10.1152/physrev.00006.2010.—MicroRNAs (miRNAs) are a class of posttranscriptional regulators that have recently introduced an additional level of intricacy to our understanding of gene regulation. There are currently over 10,000 miRNAs that have been identified in a range of species including metazoa, mycetozoa, viridiplantae, and viruses, of which 940, to date, are found in humans. It is estimated that more than 60% of human protein-coding genes harbor miRNA target sites in their 3′ untranslated region and, thus, are potentially regulated by these molecules in health and disease. This review will first briefly describe the discovery, structure, and mode of function of miRNAs in mammalian cells, before elaborating on their roles and significance during development and pathogenesis in the various mammalian organs, while attempting to reconcile their functions with our existing knowledge of their targets. Finally, we will summarize some of the advances made in utilizing miRNAs in therapeutics.

I. INTRODUCTION 827
II. MicroRNAs IN THE VASCULAR SYSTEM 831
III. MicroRNAs IN THE... 833
IV. MicroRNAs IN THE HEART 841
V. MicroRNAs IN SKELETAL MUSCLE 846
VI. miRNAs IN THE BRAIN 849
VII. MicroRNAs IN THE KIDNEY 854
VIII. MicroRNAs IN THE LUNG 855
IX. MicroRNAs IN THE LIVER 857
X. MicroRNAs IN THE PANCREAS 858
XI. MicroRNAs IN THE SKIN 860
XII. MicroRNAs IN COLON CANCER 861
XIII. MicroRNAs IN BREAST CANCER 862
XIV. MicroRNAs IN OVARIAN CANCER 866
XV. MicroRNAs IN PROSTATE CANCER 867
XVI. EXPLOITING MicroRNAs IN... 867
XVII. CONCLUSION 868

I. INTRODUCTION

A. The Discovery of MicroRNAs and Their Significance

In 1993, discovering that lin-4 is a 22-ribonucleotide-long molecule that negatively regulates lin-14, not only identified the first microRNA (miRNA), but also revealed their functional relevance (341). Lin-4 is one of the heterochronic genes that are involved in temporal regulation of “larva-to-adult switch” in *Caenorhabditis elegans* via inhibiting the expression of lin-14 and lin-28 (10). Accordingly, loss of lin-4 results in inappropriate “reiteration of cell lineages” during later developmental stages, which is a sign of perturbed developmental timing (81). After cloning the lin-4 gene, it was surprising to find that it had no substantial open reading frame, but did generate two RNA products, ~61 and 22 nucleotides (nt) long, which are now recognized as the premature and mature forms of lin-4, respectively (341). The mature form is complementary to repeat sequences in the 3′-untranslated region (UTR) of lin-14 mRNA (341) via which it inhibits its translation without affecting its stability (461). A similar mechanism was later ascribed to let-7 and its regulation of a set of genes that included lin-14, lin-28, lin-41, lin-42, and daf-12 in *C. elegans*, where if over- or underexpressed resulted in premature development or reiteration of cell lineages (506). These findings reveal two aspects of miRNA’s unique functionality: 1) precise regulation of the timing of a cellular event via 2) synchronous inhibition of a cadre of genes that are functionally interdependent, thus operating as an efficient molecular switch. This mode of function circumvents the need for transcriptional regulation of individual genes and is, thus, potentially faster and more energy efficient, consistent with what Beyer et al. referred to as translation regulation “on demand” (40). Furthermore, posttranscriptional coregulation of functionally related genes is well supported by two observations: 1) the positive correlation between mRNA and protein abundance of functionally related genes, but not those of whole cell or separate cellular compartments (40); and 2) mRNA of functionally related genes often possess similar decay rates (655). miRNAs have the necessary functional versatility and specificity to explain both these cellular attributes.

As we were reviewing the vast literature regarding the role of miRNAs in mammalian organogenesis and pathogenesis, we continually searched for patterns that could explain the functional relevance of these posttranscriptional regulators.

DANISH SAYED AND MAHA ABDELLATIF

One of the more reproducible patterns that was seen during the development of various organs was reminiscent of lin-4 in *C. elegans*, where over- or underexpression of a miRNA either accelerated or delayed differentiation of cells, indicative of its regulation of developmental timing. Some of the examples that will be encountered in this review include the role of miR-27 in myogenesis (57), the role of miR-124 (101) and miR-9 in neurogenesis (558, 711), the role of miR-30 in nephrogenesis (5), and the role of miR-17~92 in lung development (390).

### B. The Expression and Processing of miRNAs

To date, 940 human miRNAs have been identified, which are annotated and catalogued in a searchable Web-based miRNA database known as miRBase (217, 218). The first group of mammalian miRNAs was identified by means of large-scale cloning and sequencing of the mature forms (327, 334, 340). To accurately differentiate between miRNAs and other small RNA species, such as siRNAs, Ambros et al. (11) defined a set of expression and biogenesis criteria that were used for that purpose. The expression criteria included the ability to detect an ~22 nt RNA transcript by conventional Northern blotting, or an equivalent method, and in a cDNA library of size fractionated RNA. The biogenesis criteria included the presence of the ~22 nt sequence in one of the arms of a putative stem-loop secondary structure that had to include “at least 16 bp involving the first 22 nt of the miRNA and the other arm of the hairpin” (11). In addition, the sequence of the secondary structure had to be phylogenetically conserved and accumulate in cells upon loss of the processing enzyme Dicer. The structural aspects of these criteria were used in a “training set” for computer software to generate algorithms for computational prediction of miRNAs within the genome (reviewed in Ref. 33). The data from that information were then further examined in an attempt to delineate the transcriptional units of those genes (509). Out of 232 mammalian miRNAs analyzed, ~40% existed within introns of coding RNA, 10% within introns of noncoding RNA (ncRNA), and 13% within exons of ncRNA; in addition, a small portion of miRNAs exist within exons (mainly 3′ untranslated regions) of coding “host” genes, some of which are expressed from the same transcriptional unit (509, 625). The remaining miRNAs are intergenic, with mostly undefined transcriptional units as of yet. For that purpose, computational tools were developed for prediction of miRNA core promoters (271, 715). These applications revealed that miRNA genes have features common to RNA polymerase II-dependent promoters, as well as unique sequence motifs. Otherwise, there are only a handful of studies that experimentally examined the transcriptional regulation of individual miRNA promoters. Examples include the mouse miR-1–1 and miR-1–2 promoters that are regulated by serum response factor (SRF), myoD, and muscle enhancer factor 2 (Mef2c), also thought to cotranscribe miR-133a-2 and miR-133a-1, respectively, as part of bicistronic transcripts (713), and the miR-206 promoter, which binds myoD and myogenin (501), among a few others (141, 459). Additionally, a fair number of miRNA genes exist within areas of Alu repeats that are transcribed by RNA polymerase III-dependent promoters (54).

The primary transcript of a miRNA (pri-miRNA) gene encompasses a stem-loop precursor (pre-miRNA) that harbors the mature sequence. The pri-miRNA is cleaved in the nucleus at the base of the stem-loop structure by the RNase III enzyme Drosha (344) and its partner DGCR8/Pasha (139, 216, 229, 330) ([FIGURE 1](#figure-1)), which have been shown to require an intact secondary structure of the pri-miRNA for efficient processing (230). In this reaction, DGCR8 identifies the exact cleavage site in the 5′ and 3′ overhangs flanking the stem-loop, while Drosha catalyzes their excisions. Alternatively, a few intronic pri-miRNAs (mirtrons) bypass Drosha/DGCR8 and are processed by the splicing machinery (34, 516). In addition, miRNAs that are derived from small nucleolar RNA (snoRNA) in various species (163, 260, 528, 592) are processed by Dicer, but do not require Drosha (260). At this stage, similar to mRNA, pri-miRNAs are subject to posttranscriptional regulation. This was initially predicted due to the lack of correlation between the levels of mature miRNAs and their primary transcripts in various normal and cancer cells (610). In agreement, it was found that lin-28 binds the loop structure of let-7 and inhibits its processing by both Drosha in the nucleus (448, 638) and Dicer in the cytoplasm (518). Likewise, hnRNP A1 binds to miR-18a’s loop region and regulates its processing by Drosha (426). In general, little conservation was previously observed in the terminal loop sequences of miRNAs, and it was thus believed that it does not have any functional relevance. However, upon closer examination, Michlewski et al. (426) noted that 14% of pre-miRNAs have highly conserved terminal loops. This suggested that the loop sequence in these miRNAs might play a role in their processing. In support, targeting the loop sequences by antisense oligos inhibited the processing of miRNAs that harbored conserved (miR-18a, miR-101, let-7, miR-379, miR-31), but not nonconserved (mmiR-16–1 and miR-27a), loop sequences. Following completion of this nuclear processing step, the stem-loop pre-miRNA is exported out of the nucleus via exportin-5 in a RanGTP-dependent fashion (50, 394, 692) and into the cytoplasm, where it is further processed by Dicer to yield the functional single-stranded mature 20–22 ribonucleotide construct (259) ([FIGURE 1](#figure-1)).

### C. Mechanisms of Function of miRNAs

miRNAs are mainly negative regulators of mRNA translation and exert their effects through various mechanisms ([FIGURE 1](#figure-1)). As a first step, the pre-miRNAs are assembled in large protein complexes known as miRNA RNA-inducing

MicroRNAs IN DEVELOPMENT AND DISEASE

MiRNA processing and function

Pri-miRNA
Drosha/Pasha
Exportin-5
Pre-miRNA

RISC
Dicer
Ago
TRBP
GW182

Inhibition of translation initiation
m7GpppG
eIF6
Ago
PABP
Dicet
TRBP
GW182
eIF4E
eIF4G

mRNA deadenylation
m7GpppG
Dicet
Ago
TRBP
GW182
PABP
Pan2-Pan3
CCR4
CAF1

Inhibition of translation elongation
m7GpppG
PABP
Ago
Dicet
TRBP
GW182

FIGURE 1 An illustration of the steps and molecules involved in the processing and function of miRNAs. The first processing step occurs in the nucleus, where pri-miRNA is cleaved via the enzymes Drosha and Pasha. The pre-miRNA hairpin structure is then exported to the cytoplasm via exportin-5. In the cytoplasm it assembles in RISC which includes Dicer, argonaute (Ago) protein, PW182 a P-body protein, the human immunodeficiency virus transactivating response RNA-binding protein (TRBP), and fragile X mental retardation protein (FMRP1). In RISC, the loop of the pre-miRNAs is cleaved and the mature strand is used to direct the complex to its target site in the 3′-UTR of an mRNA. RISC is involved in 3 different functions: inhibition of translation initiation, inhibition of translation elongation, or mRNA deadenylation, which would result in mRNA degradation. These functions require additional recruitment of eukaryotic initiation factor 6 (eIF6). Inhibition of translation initiation is a result of disengagement of the 80S ribosomal complex in a cap (m7GpppG)-dependent manner that is partly attributed to the recruitment of the anti-association factor eIF6 by the RISC complex, or competition of Ago2 with eukaryotic initiation factor 4E (eIF4E) or eIF4G. Deadenylation occurs via interaction of Ago2 with chromatin assembly factor 1 (CAF1) and, thus, recruitment of the CCR4-NOT protein complex that harbors both chemokine (C-C motif) receptor (CCR4) and CAF1 deadenylase enzymes, in addition to a second deadenylation step involving the PABP-dependent poly(A) nuclease 2 (Pan2)-Pan3 deadenylase complex.

silencing complex (RISC or miRISC). Some of the known proteins that form RISC include the RNA-specific endonuclease Dicer, which is involved in the processing of pre-miRNA into the mature form (259); the argonaute (Ago) protein, which has four isoforms, of which only Ago2, also known as “slicer,” has the capacity to cleave the target mRNA (414, 455); PW182 a P-body protein (376); the human immunodeficiency virus transactivating response RNA-binding protein (TRBP), which recruits Ago2 to the complex (100); and fragile X mental retardation protein (FMRP1) (277), which is known to associate with polyribosomes (177); among other proteins whose functions

within the RISC have not been fully examined yet (246). Guided by partial sequence complementarity between the miRNA and its target, the complex binds to miRNAs target sites, mainly within the 3′-UTR of genes. It was first noted in *C. elegans* that miRNAs have the capacity to inhibit translation of its target mRNA at a postinitiation step, as lin-4 inhibits the translation of lin-14 without reducing mRNA abundance, polyadenylation, or polyribosomal content (461). Similar results were observed with inhibition of lin-41 by let-7 in a HeLa cell background, where the suppressed mRNA was associated with active polysomes comparable to its unsuppressed counterpart (454). In

Physiol Rev · VOL 91 · JULY 2011 · www.prv.org

support of this mechanism, cap-independent translation of mRNAs is also successfully inhibited by recombinant miRNA (483). It was suggested that this effect could be a result of premature dissociation of ribosomes during translation elongation.

A second mode of function employed by miRNAs is inhibition of translation initiation, which requires both the cap (253, 487, 608) and poly(A) tail structures (253). It has been known that both of these structures act in concert to facilitate translation. This is demonstrated by a model in which eukaryotic initiation factors 4G (eIF4G), a subunit of the 5′-cap-binding initiation factor, binds the poly(A)-binding protein (PABP) and promotes circularization of the mRNA molecule, a structure that is translationally superior to the linear form (521). A cell-free translation system shows that miRNA inhibits translation via inhibiting the assembly of the 80S ribosomal complex in a cap (m7GpppG)-dependent manner (406). This effect could be partly attributed to the recruitment of the anti-association factor eIF6 by RISC, which prevents the assembly of the 80S complex (99). Additionally, the Ago2 protein within the RISC complex contains a cap-binding domain that competes with eIF4E and inhibits binding of the initiation complex to the m7GpppG cap (302). Ago2 also has the capacity to compete with eIF4E for its interaction with eIF4G and further disrupts assembly of the initiation complex (268).

A more elaborate mechanism became evident when it was discovered that miRNAs also induce deadenylation of mRNAs (27, 641, 671), which succeeds translational inhibition but is independent of it (168). This function is mediated through the interaction of Ago protein with chromatin assembly factor 1 (CAF1), which recruits the CCR4-NOT protein complex, harboring both chemokine (C-C motif) receptor (CCR4) and CAF1 deadenylase enzymes (168). This, in addition to GW182's interaction with PABP, induces mRNA deadenylation, which may be associated with an additional step involving the PABP-dependent poly(A) nuclease 2 (Pan2)-Pan3 deadenylase complex (88). However, the results of the latter study argue against direct physical association between miRISC and any of the deadenylase enzymes and suggests an alternative model in which miRNAs renders mRNAs more liable to deadenylation via the degradation machinery, plausibly, through disrupting PABP's association with the poly(A) tail (88). In general, deadenylation leads to destabilization of mRNA. While mammalian miRNAs are commonly recognized for inhibiting translation versus inducing mRNA degradation, there is substantial evidence that supports the latter (22). Indeed, Farh et al. (170) noted that the levels of tissue-enriched miRNAs inversely correlated with levels of some of their mRNA targets compared with their concentrations in other tissues. The question of whether a miRNA induces translation inhibition versus mRNA degradation is best addressed

at the level of individual miRNAs, specific targets, and cell backgrounds.

Remarkably, miRNAs have also been observed to enhance translation in quiescent cells (631). Specifically, miR-369–3p enhances the translation of TNFα in serum-deprived, cell cycle-arrested, 293HEK cells. Interestingly, this effect switches to an inhibitory one, upon reentry of the cells into the cell cycle. A similar effect was seen with the regulation of high mobility group A2 (HMGA2) protein and its targeting miRNA, Let-7, suggesting the ubiquity of this mechanism, but not with siRNA. During miRNA-enhanced translation, it was noted that the FMRP1 protein is recruited to RISC; however, the full mechanism of this functional aspect of miRNAs awaits elucidation.

### D. Identifying the mRNA Targets of miRNAs

To appreciate the significance of a change in the level of a certain miRNA during health or disease, it is essential that we identify its mRNA targets. For that purpose, several computational target prediction tools have been developed that apply various distinct as well as overlapping algorithms, which continue to be modified as more targets are being experimentally validated. In this section, we briefly highlight some of the prominent prediction algorithms that are used by the three most widely utilized engines that generally rely on base pairing between the “seed sequence” of the miRNA and the 3′-UTR of its target, in addition to evolutionary conservation of the targeted sequence. It should be noted that while “most functional mRNA-miRNA pairing resides in the 3′-UTRs,” miRNA target sites are also found within the coding region of a gene, albeit, at a much lower frequency (352). The miRanda algorithms include the following (36, 279): 1) an alignment score between a miRNA and its target that is “the sum of match and mismatch scores” for base-pairing and gap penalties. 2) The alignment score involves an asymmetric Watson-Crick (W-C) 5′ to 3′ base-pairing, with more weight assigned to the 5′ end (seed sequence), which can, to some degree, be compensated for by stronger complementation between the 3′ end of the miRNA and its target, while allowing mismatches and G:U wobbles in the seed sequence. 3) Determining the degree of evolutionary conservation of the targeted sequence and its position in 3′-UTRs of human, mouse, and rat genes. TargetScan prediction software applies the following parameters (186, 220, 353): 1) an alignment score for W-C base-pairing of nucleotides 2–7 (seed sequence) of the miRNA to its target, in addition to base-pairings beyond the seed sequence that can compensate for mismatches in the seed; 2) it also recognizes a conserved adenosine at the first position and/or a W-C match at position 8; 3) context scoring, which evaluates the AU content 30 nt flanking both sides of the targeted site, in addition to the latter’s distance from the end of the 3′-UTR; and 4) determining the degree of conservation using three different

levels: highly conserved between human, rat, mouse, dog, and chicken; conserved between human, mouse, rat, and dog; and poorly conserved among any combination of species. A third target prediction software, PicTar, searches for (170) 1) a perfect W-C base-paired ~7 nucleotide seed at position 1–7 or 2–8; however, mismatches are allowed if the free binding energy does not increase. 2) It takes into account the binding energy of the entire miRNA-mRNA duplex and 3) the degree of conservation of targeted sites between all the species listed above in addition to chimpanzee, pufferfish, and zebrafish (316). One other important criterion that should be considered during miRNA target prediction is the accessibility of the targeted site (296, 381). It was experimentally proven that target sites that are embedded in a closed stem structure do not allow effective inhibition by the targeting miRNA (296). For that purpose, Kertesz et al. (296) designed the prediction software PITA (Probability of Interaction by Target Accessibility), which allows prediction of target sites by first searching for complementary “seed” sequences, followed by calculations of the free binding-energy after subtracting the energy used to unwind any predicted secondary structures (296). Likewise, Long et al. using Sfold (144) describe a two-step model for miRNA-target binding, in which the miRNA first hybridizes to a few accessible nucleotides (nucleation), followed by its expansion as it disrupts any secondary structures (381).

It should be noted that nonconserved miRNAs-targeted sites respond equally well to inhibition by miRNAs (170). However, these nonconserved sites appear to be present in genes that are not expressed in the same tissue as the targeting miRNAs. On the other hand, genes that preferentially coexist with miRNAs in a specific tissue are thought to have evolved through selective elimination of the targeting sites and are known as “antitargets” (170). Hence, there are also target prediction applications that do not rely on target site conservation, such as MicroTar, which may be useful in some instances (606). Note that while the three target prediction applications described above overlap in many predicted targets, they diverge in others. Thus it might be beneficial to search all the databases for potential targets of a miRNA of interest. However, one needs to take into account that not all predicted targets are genuine, as they may be subject to spatial or temporal restrictions. In addition, binding to the targeting site might be modulated by 3′-UTR cis-acting sequences or transacting factors. Moreover, a single 3′-UTR may be targeted by multiple miRNA. Thus the level of a mRNA or its translation product is governed by the combinatorial effect of its targeting miRNA.

While computational analysis predicted the structural basis for miRNA:target pairing, Brennecke et al. (61) experimentally validated some of the parameters described above. Essentially, they confirmed that nucleotides 2–8 from the 5′-end of a miRNA are the most critical in establishing base

pairing with the target. This criterion could be relaxed to 2–5 nucleotides, under conditions where the nucleotides in the 3′-end had strong complementarity with the target. Also, the position of the base pairing rather than the pairing energy determined the functional efficiency of the miRNA. It is also worth noting that a miRNA’s targeting capacity might be modified by selective posttranscriptional editing of its precursor. This process converts adenosine to inosine, catalyzed by adenosine deaminase in double-stranded RNA substrates (reviewed in Ref. 398). Luciano et al. (392) were the first to notice that premature miR-22 is edited at multiple sites. It is further estimated that ~6% of human miRNAs are subject to posttranscriptional editing (48). Significantly, some of the edited sites are within the “seed” region, which would accordingly lead to reassignment of targets (292).

Before proceeding with the role of miRNAs in organogenesis and pathogenesis, we should briefly describe the nomenclature system as defined in miRBase, the online repository for miRNA sequence data and target prediction (219). miRNAs are designated by a three-letter prefix, such as miR or let, followed by a number (e.g., miR-1). If a given miRNA is expressed from multiple genes, an additional corresponding numeric suffix is added, e.g., miR-1–1 and miR-1–2, while homologous miRNAs are identified by a common numeric suffix, followed by a distinct letter (e.g., Let-7a, Let-7b, Let-7c…, etc.). In some cases, two mature miRNAs are processed from the same stem-loop precursor, one from each arm, and are accordingly designated by an additional suffix, 5p for that released from the 5′-arm and 3p for that released from the 3′-arm (e.g., miR-199a-5p and miR-199a-3p).

II. **MicroRNAs IN THE VASCULAR SYSTEM**

A. **miRNAs in Vasculogenesis and Angiogenesis**

Early indication that miRNAs are involved in the development of the vascular system was uncovered by a generalized genomic knockout of the gene for the miRNAs processing enzyme Dicer in a mouse model. Loss of Dicer led to stunted growth and embryonic lethality that was associated with retardation in the expansion of the yolk sac’s and embryo’s vasculature (690). Although the exact nature of the defect was unclear, it appeared that progression, but not the initiation, of vasculogenesis was interrupted. This was confirmed by the knockdown of Dicer in human umbilical vascular endothelial cells (HUVEC), which exhibited defective proliferation, migration, and tubulogenesis (321, 584). Subsequently, mice models with targeted deletions of specific miRNAs enriched in endothelial or vascular smooth muscle cells (VSMC) were engineered to more precisely dissect their

DANISH SAYED AND MAHA ABDELLATIF

roles in vasculogenesis and angiogenesis. miR-126 is one that is highly expressed in the heart and vasculature of zebrafish (661). It is also highly expressed in murine lung and heart and to a lesser extent in the brain, liver, and kidney, where it is localized to endothelial (235) and epithelial cells (222). Deletion of one copy of this gene results in ~50% reduction of mature miR-126 with no apparent phenotype, while deletion of both copies results in ~50% embryonic lethality due to systemic edema and widespread hemorrhage from ruptured vessels (322, 652). Additionally, vascularization of the brain at E10.5 and the retina at P0 was severely retarded. Thus the phenotype appears to involve weakened vascular walls, as well as defective vascularization in some organs. In support of the latter, endothelial cell proliferation and tubular outgrowth formation were impaired. Those embryos that escaped the lethal phenotype developed normally with no apparent abnormalities, until they were challenged with ischemia, which demanded angiogenesis. As a result, when the left coronary artery was occluded, angiogenesis was compromised in the ischemic hearts, which resulted in precipitous cardiac failure (652). This suggests that miR-126 is required for proper formation of endothelial cell-to-cell contacts and vasculatization during early development, as well as angiogenesis during ischemic conditions.

Two of miR-126's validated targets that were upregulated in the knockout models include Spred-1 (652) and the p85β regulatory subunit of phosphatidylinositol 3-kinase (PI3K) (180, 322), although evidence for their involvement in the underlying phenotype is only circumstantial at this point. The regulatory p85 subunit is necessary for the stability and recruitment of the catalytic subunit of PI3K and, thus, the activation of the PI3K-AKT pathway (696). Indeed, knock-out of p85α-β in endothelial results in reduced AKT activity and a phenotype reminiscent of that of the miR-126 knockout mice, including embryonic lethality due to rupturing of vessels and hemorrhaging (700). Thus the upregulation of p85β in the miR-126 knockout models does not reconcile with the observed phenotype and requires further assessment. On the other hand, Wang et al. (652) show for the first time through gain-and loss-of-function experiments in an aortic ring sprouting assay that Spred-1 is sufficient for inhibiting vascular outgrowths (652). However, its role in vivo may be more complex, since inactivating mutations of spred-1 have been associated with neurofibromatosis 1-like phenotype in humans (60), while targeted deletion in mice induces a mild phenotype that exhibits a slightly lower body weight and shorter face (265). Also, spred-1 and spred-2 double knockout increased lymphatic vessels and lymphatic endothelial cells but had no effect on the number of blood vessels (598). Thus the miR-126 knockout phenotype might be attributed to a broader array of targets that remain to be identified.

Other miRNAs that may also contribute to vessel formation include miR-130a, which is upregulated in endothelial cells during tubulogenesis and facilitates the process by targeting and inhibiting the expression of antiangiogenic transcription factors GAX and HOXA5 (97). Similarly, miR-210, which is induced by hypoxia (72, 171), was sufficient for inducing tubulogenesis in HUVEC cells (171). However, proof of its direct targeting and suppression of Ephrin A3 under these conditions does not accurately explain its function and should be cautiously regarded, since eph-ephrins are generally known to be essential for angiogenesis and neovascularization (648). Additionally, let-7f and miR-27b are also necessary for tubulogenesis, although their targets have not been investigated (321).

In contrast, overexpression of miR-92a in endothelial cells blocked tubulogenesis and was associated with reduced cell migration and adhesion, but did not affect viability or proliferation (52). Indeed, knockdown of miR-92a with antagomirs during limb and myocardial ischemia enhanced angiogenesis and, thus, improved blood flow and tissue viability in the region. One of miR-92a's validated targets that may be implicated in this process is α5-integrin, although, undoubtedly, other targets must also be involved. miR-92a is a member of the miR-17~92 cluster, which is induced by Myc in colon cancer cells (140). Overexpression of this cluster with Ras in p53-null colonocytes did not influence cell proliferation, however, similar to Myc, when implanted in mice it enhanced tumor growth by increasing neovascularization (140). These results may be in part attributed to targeting and inhibiting thrombospondin and CTGF by miR-19 and miR-18, respectively. This reconciles with the fact that c-myc induces an angiogenic switch by posttranscriptionally downregulating thrombospondin-1 (270, 450, 613) and provides an explanatory mechanism for it. Thus, although overexpression of miR-92a inhibits angiogenesis, overexpression of the whole cluster enhances angiogenesis in tumors. This suggests that the function of individual members of the cluster does not necessarily reflect the function of the group.

While miR-126 is preferentially expressed in endothelial and epithelial cells, the miR-145/miR-143 cluster is enriched in vascular and visceral smooth muscle cells as early as E11.5 (678) and throughout adulthood (49, 160, 678), in addition to the embryonic heart (49, 678). Genomic deletion of this cluster resulted in viable mice with a mild vascular phenotype but no cardiac abnormalities (49, 160, 678). In particular, the smooth muscle cells were smaller and exhibited an increase in rough endoplasmic reticulum and a decrease in actin stress fibers, with a resultant thinner tunica media. In addition, there were small neointimal lesions and a megacolon phenotype, which suggested that knockout of miR-145/143 may inhibit differentiation and, subsequently, enhance proliferation and migration of smooth muscle cells (160). However, the failure to detect a

832 Physiol Rev · VOL 91 · JULY 2011 · www.prv.org

consistent increase in VSMC proliferation or apoptosis argued that the miRNA may not be directly required for differentiation but rather for the ability of VSMC to differentiate in response to the physiological contractile demand (49). In concordance, this phenotype was accompanied by lower blood pressure (49, 678) and depressed responsiveness to vasopressors (49), in spite of normal levels of smooth muscle α-actin and myosin heavy chain, although studies differed on this latter finding (160). On the other hand, the expression of caldesmon, calponin, and smoothelin was reduced in the miR-145/143-devoid VSMC (49). A separate study showed that miR-145 is also enriched in a subset of pericytes in adult brain and kidney microvessels and inhibits cell migration, plausibly by targeting Friend leukemia virus integration I (Fli1) (333). Fli1 belongs to the ETS-1 family of transcription factors that play a critical role in angiogenesis (486). Several other downstream targets have been identified and validated that could potentially mediate miR-145/143 effects; these include kruppel-like factor 4 and 5, adducin-3, slit-robo GAP-1 and -2, sling-shot 2 phosphatase, and MRTF-B, which are thought to play a role in cytoskeletal remodeling in VSMC (678). However, the functional dynamics between these molecules and how they specifically contribute to the miR-145/143 knockout phenotype remain to be examined. On the other hand, angiotensin converting enzyme (ACE), which has been validated as a miR-145 target that is derepressed in the knockout model, may be the cause of desensitization and downregulation of angiotensin I receptor, thus explaining the lower blood pressure and irresponsiveness to vasopressors (49). In support, treatment of the mice with an ACE inhibitor partially alleviated these signs, but was not sufficient for reversal of other aspects of the phenotype.

Although the results of genomic ablation of miR-145/143 in mice would argue against its involvement in cell specification or fate, cell culture studies demonstrate that increased expression of miR-145 in human embryonic stem cells is both necessary and sufficient for inducing their differentiation into mesoderm and ectoderm lineages through suppression of OCT4 and SOX2 (680). Likewise, knockdown of miR-145 in myocardin-treated fibroblast resulted in inhibition of their transformation into VSMC, while its overexpression in neural crest stem cells was sufficient for inducing their commitment to the VSMC lineage and expression of smooth muscle actin and myosin heavy chain (117). The discrepancy between the knockout models and the cell culture experiments is readily explained by the impact of a cell’s environment on its response to stimuli.

B. miRNAs in Neointimal Hyperplasia

Dedifferentiation of VSMC is an underlying mechanism in neointimal hyperplasia and, thus, vascular stenosis during atherosclerosis or vascular injury. Given the role of miR-145 in VSMC that is described above, it is not surprising to learn that this miRNA is downregulated in response to vascular injury (103). While the knockout models dispute the fact that downregulation of miR-145 is sufficient for reducing α-actin and myosin heavy chain (49, 678), they do show that abrogation of miR-145 induces limited neointimal lesion formation (49, 160). In particular, one study has shown that these lesions are only observed in the femoral artery of older knockout mice (18 mo), which suggested that although downregulation of miR-145/143 does not result in fulminant neointimal formation, it does promote it (49). In consensus, restoring miR-145 levels during vascular injury inhibits neointimal formation (160, 678), with promising therapeutic prospects. miR-145's effect in this context could be, at least partially, explained by the suppression of its target, Krupple-like 5 (Klf5). Indeed, KLF5, first identified as a transcription factor that induces embryonic smooth muscle myosin heavy chain (656), is involved in enhancing smooth muscle proliferation by upregulating cyclin D1 (588).

In contrast to miR-145, miR-221 (133, 378), but not the co-clustered miR-222 (133), positively regulates smooth muscle proliferation. It is induced by platelet-derived growth factor (PDGF), which is known to stimulate VSMC switching and proliferation during angiogenesis and neointimal formation (133, 378). Gain- and loss-of-function experiments proved that it is indeed required for mediating the effects of PDGF via targeting and suppressing the cell cycle inhibitors p27<sup>Kip1</sup> (133, 378) and p57<sup>Kip2</sup> (378), as well as c-kit (133). In concordance, knockdown of miR-221 during vascular injury reduced neointimal thickness by 40% (378). The effects of miR-221 are likely to be reinforced by the concurrent upregulation of miR-21 in neointimal lesions (275). miR-21, which is a ubiquitously expressed pro-survival miRNA, was shown to inhibit PTEN expression under these conditions (275, 416, 417). Accordingly, its abrogation increased apoptotic VSMC death and reduced neointimal thickness. Thus neointimal formation is the result of a combinatorial effect of changes in the expression of several miRNAs and their targets that regulate VSMC differentiation, proliferation, and survival.

III. MicroRNAs IN THE HEMATOLOGICAL SYSTEM

A. miRNAs in Lymphopoiesis, Lymphocytes, and Adaptive Immunity

A role for miRNA in B-cell development was demonstrated by targeted deletion of Dicer, which resulted in the arrest of pro-B to pre-B cell transition (312). In particular, some of the mature miRNAs that were depleted included miR-17, miR-142–3p, miR-181, and miR-191. The phenotype could partly be attributed to pre-B cell apoptosis as a consequence of the increase in the proapoptotic protein Bim, a

DANISH SAYED AND MAHA ABDELLATIF

miR-17~92 target (FIGURE 2). Knockout of miR-17~92 confirmed that this phenotype is indeed predominantly attributed to this miRNA cluster versus any of the others that were depleted by the loss of Dicer (633). Aside from the arrest of B-cell development, the knockout had no effect on erythropoiesis or myelopoiesis; however, the homozygous mice died at birth due to lung and cardiac ventricular septal defects. In contrast, overexpression of miR-17~92 in lymphocytes resulted in an increase in peripheral lymphocytes, especially the CD4+ T cells, which in older mice expressed the activation marker CD69 (676). In addition, there was an increase in B1 cells and the germinal centers in lymph nodes and spleen, which was associated with an enlarged spleen and lymph nodes; an increase in circulating IgG2b, IgG2a, and IgG3; infiltration of organs by lymphocytes; and deposition of immune complexes, suggestive of an autoimmune reaction in these mice. Thus miR-17~92 appears to impact B-cell number by enhancing their viability, which may be partly attributed to suppression of Bim.

Activation of CD4+ T lymphocytes results in rapid and sustained upregulation of the noncoding BIC transcript (227), whose function was ambiguous until it was discovered that it harbors miR-155 (155, 707). Similar to its host gene, miR-155 is initially very low in hematopoietic stem cells (HSC) and mature hematopoietic cells. The function of this gene was eventually uncovered by targeted deletion in a mouse model. Except for thickening of the bronchial walls (510) and reduced B cells in the germinal centers of Peyer's patches and mesenteric lymph nodes (607), loss of miR-155 had no impact on general development, including that of the myeloid and lymphoid systems (510, 607). However, the B cells' immune response was impaired in these mice, as they produced less IgM and exhibited reduced antigen-specific antibody switch. In addition, T-cell function was also impacted, producing less interleukin (IL)-2 and interferon (IFN)-γ partly due to defective antigen presentation by dendritic cells. Although miR-155 knockout did not affect differentiation of T helper (Th) cells, it did result in Th1 cells

# MiRNAs in hematopoiesis and immunity

![Diagram](https://i.imgur.com/yourimageurl.png)

**FIGURE 2** A diagram showing miRNAs and their targets that play a role in the development of the hematopoietic system. The diagram lists the various miRNAs and their targets that are involved in hematopoiesis. Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of a target gene inversely correlate with that of the targeting miRNA and are similarly represented by an up or down arrow. All listed targets have been validated. The miRNA targets listed include LIM domain only 2 (LMO2), v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog (c-kit), v-myb myeloblastosis viral oncogene homolog (c-myb), Kruppel-like factor d (klfd), GATA binding protein 2 (GATA-2), myocyte enhancer factor 2C (mef2c), acute myeloid leukemia 1 protein (AML1), Fas (TNFRSF6)-associated via death domain (FADD), tumor necrosis factor receptor (TNFR), receptor (TNFRSF)-interacting serine-threonine kinase 1 (Ripk1), SH2 containing inositol phosphatase, isoform b (SHIP), IL-1 receptor-associated kinase 1 (IRAK), TNF-receptor-associated factor 6 (TRAF6), bcl-2 interacting mediator of cell death (Bim), hematopoietic transcription factor PU.1 (Pu.1), activation-induced cytidine deaminase (AID), v-maf musculoaponeurotic fibrosarcoma oncogene homolog (c-Maf), protein-tyrosine phosphatase (SHP-1), non-receptor type tyrosine phosphatase (PTPN), and dual specificity tyrosine phosphatase (DUSP).

834 Physiol Rev · VOL 91 · JULY 2011 · www.prv.org

that produced higher levels of chemokine (C-C motif) ligand 5 (CCL5) and an increase in Th2 cells, along with elevated levels of IL-4, IL-5, and IL-10, possibly a result of an increase in c-Maf, a miR-155 target. In contrast, regulatory T cells (T-reg) numbers, but not function, were reduced in the knockout mice (309). The defective response of B cells in this model could be attributed to an increase in PU.1, which reduces IgG1 switching and is a target of miR-155 (636). On the other hand, miR-155 has also been reported to target the activation-induced cytidine deaminase (AID) gene (149, 604). In these studies, mutation of the miR-155 target site within the 3′-UTR of the endogenous AID gene resulted in higher levels of the protein in B lymphocytes undergoing class switch recombination. At a first glance, these results might appear to contradict the decrease in class switching observed in the miR-155 knockout models; however, concurrent upregulation of PU.1 might counterbalance the effects of AID. This emphasizes the need not only to identify all targets for a given miRNA in a temporal and spatial manner, but also to characterize their functional interaction.

miR-155 is regulated by Foxp3 and was indeed confirmed necessary for the optimal proliferation of thymic and peripheral Foxp3+ T-reg cells (389). This function is a consequence of its direct targeting of suppressor of cytokine signaling 1 (SOCS1), which results in enhanced IL-2-induced activation of signal transducer and activator of transcription 5 (STAT5). On the other hand, T-reg-specific Dicer knockout proved that the suppressor activity of T-reg also requires miRNAs, as these mice exhibited a lethal autoimmune response when challenged with an inflammatory condition (374). A recent report shows that, in this case, miR-146a is the miRNA responsible for T-reg cells' suppressor function (388). In contrast to miR-155 knockout, miR-146a mice had more T-reg cells, but their suppressor function was impaired. Accordingly, a miR-146a⁻/⁻/Foxop3KO mouse exhibited an autoimmune lethal phenotype similar to that observed in the T-reg-specific Dicer knockout mouse. The underlying defect appears to be resultant from an increase in miR-146a’s target STAT1, which enhances the production of IFN-γ and, thereby, exaggerates Th1 development and autoimmunity.

In contrast to miR-155, miR-150 is highly expressed in mature B and T lymphocytes ([FIGURE 2](#figure-2)). When prematurely expressed, though, it halts B-lymphocyte maturation at the pro-B cell stage, while its effect on T-cell development was inconsistent (675, 714). In contrast, miR-150 knockout mice developed normally, including a typical T-cell, follicular and marginal zone B-cell populations, except for an increase in peritoneal and splenic B1a cells, known for recognizing self-antigens (675). As a direct target of miR-150, c-myb is upregulated in this model. The phenotype of the transgenic and knockout models is consistent with the phenotype previously observed in B cell-targeted knockout of

c-myb, in which B-cell development was impaired at the pro- to pre-B conversion step with loss of the B1 subtype (609).

Similar to miR-150, miR-181 is relatively low in HSC, then specifically increases in differentiated B lymphocytes (89). In this case, when ectopically expressed in HSC, it results in ~2.5-fold increase in B lymphocytes versus an 88% decrease in T lymphocytes (89). miR-181 is also enriched in the thymus, where it is high during the early stages of T-cell differentiation and gradually declines to its lowest level in Th1 and Th2 cells (359). Its overexpression in antigen primed T cells not only increases their sensitivity to peptide antigens but also suppresses T-cell receptor antagonism. This effect is likely accomplished through downregulating multiple phosphatases that have a negative-feedback effect on T-cell receptor (TCR) stimulation. Those identified as miR-181 targets include protein tyrosine phosphate SHP-1, non-receptor type tyrosine phosphatases (PTPN), and dual-specificity phosphatases (DUSPs). The data suggest that the higher levels of miR-181 in the immature double-positive T cells versus the mature cells may be involved in heightening their sensitivity to low-affinity self-antigens and thus may play a role in establishing self-tolerance. Additionally, a recent study shows that miR-181 is necessary for negative selection of self-reactive thymocytes (153). Thus miR-17~92, miR-155, miR-150, and miR-181 play a role in fine-tuning the adaptive immune response. However, while their functions have been individually examined, their combinatorial effect remains to be elucidated.

### B. miRNA in Myelopoiesis, Granulocytes/Monocytes, and Innate Immunity

One of the bone marrow-enriched miRNAs includes miR-223, which is most abundant in the myeloid cell lineage (89). Its expression continues to increase as the cells develop into granulocytes, but conversely, it declines if they differentiate into monocytes (281) ([FIGURE 2](#figure-2)). Consistent with its developmental pattern, this gene is regulated by c/EBP (173, 190) and PU.1 (190) transcription factors, which is characteristic of myeloid-specific genes. Counter to predictions, though, knockout of miR-223 did not disrupt the development of leukocytes or monocytes, but did enhance proliferation of the granulocyte-monocyte progenitors, as well as their differentiation into granulocytes, resulting in neutrophilia (281). This effect was reversed by knockout of Mef2c that serves as a target of miR-223 and, thus, a mediator of its effects. In addition to an increase in their number, the miR-223-deficient neutrophils proved to be hyper-reactive and more efficient in eliminating target organisms. Accordingly, when challenged in vivo with sublethal doses of lipopolysaccharide (LPS), the mice exhibited a hyperinflammatory response that led to liver, spleen, and muscle necrosis, but unlike the neutrophilia, this effect was not mediated by Mef2c, suggesting that other miR-223 targets are in play.

This represents another example of miRNAs' regulation of precise developmental timing of progenitor cells.

While miR-223 is preferentially involved in differentiation of granulocytes, miR-17~92 cluster and its paralog miR-106~25 are involved in the differentiation of monocytes (181). During differentiation of unilineage monocytic cultures, downregulation of these miRNA clusters is necessary for the upregulation of the transcription factor acute myeloid leukemia 1 (AML1) and its transcriptional target monocyte-colony stimulating factor receptor (M-CSFR). Reciprocally, AML1 functions as a transcriptional inhibitor of these miRNA clusters, creating a double negative-feedback loop. On the other hand, growth factor independent 1 (Gfi1) transcription factor is required for the differentiation of both granulocytes and monocytes (287) and appears to function through suppression of miR-21 and miR-196b whose targets in this cell type remain to be identified (632).

As miRNAs are involved in the development of the myeloid cell lineage, they, in turn, impact innate immunity. miR-155 is one of the most substantially upregulated miRNA after stimulation of macrophages with IFN-$\beta$ (458) or TNF-$\alpha$ (614). Consequently, miR-155 targets and suppresses IkB kinase epsilon (IKK$\epsilon$), Fas-associated death domain protein (FADD), and receptor (TNFR superfamily)-interacting serine threonine kinase 1 (Ripk1), which are known to be involved in LPS and TNF-$\alpha$ signaling. Likewise, miR-155 is upregulated in the bone marrow in nonlymphocytic cells following LPS administration (457). Direct evidence of the involvement of miR-155 in innate immunity is provided by the fact that ectopic expression of miR-155 in HSC via retroviral delivery induces an increase in granulocytes/monocyte (GM) population and a reduction in erythroid and B-cell precursors and megakaryocytes in the bone marrow, which correlates with higher GM in the circulation, anemia, and thrombocytopenia (457). On the other hand, the spleen enlarges due to compensatory extramedullary hematopoiesis. This phenotype has been attributed to the direct suppression of the Src homology-2 domain containing inositol-5 phosphatase (SHIP) gene, a negative regulator of the AKT pathway that proved to be a direct target of miR-155 (121, 456). When knocked down with short interfering RNA, this gene induces a very similar phenotype to that observed with the overexpression of miR-155 (456). However, it partially overlaps with that of the genomic knockout of SHIP, which only exhibits reduced B cell precursors in the bone marrow (243). Overall, these results suggest that miR-155 is involved in the restoration of granulocytes and monocytes levels after their depletion by an infection. These effects appear to be counterbalanced by miR-146, which is also induced by proinflammatory stimuli in macrophages (593). This miRNA contributes to a negative feedback effect through direct suppression of the trans-

lation of IL-1 receptor associated kinase 1 (IRAK1) and TNF receptor-associated factor 6 (TRAF6).

### C. miRNAs in Thrombopoiesis

miR-150 is also more abundant in human megakaryocytes relative to their megakaryocyte-erythrocyte progenitor (MEP) or erythrocytes (385) [FIGURE 2]. Indeed, overexpression of miR-150 shifts differentiation of MEP towards megakaryocytes at the expense of erythrocytes both in cultured MEP and in vivo. This effect is mediated by its target myb, which proves to be consistent with previous studies showing that mice expressing an inactive mutant of c-myb exhibited an increase in megakaryocytes and reduced erythrocyte progenitors (422). In contrast to miR-150, overexpression of miR-155 in HSC reduced the number of megakaryocytes in the bone marrow (457), plausibly due to its suppression of the transcription factors Ets-1 and Meis1 that are involved in megakaryocyte development (512).

### D. miRNAs in Erythropoiesis

Stem cell factor (SCF) and its receptor c-kit play a critical role in the survival and proliferation of erythroid progenitor cells (452). It was noted that miR-221/222's expression gradually declines during the development of CD34+ erythroid progenitor cells, which inversely correlated with the expression of its predicted target c-kit (175) [FIGURE 2]. Further experiments confirmed that c-kit is indeed a direct target of miR-221, as ectopic expression of the miRNA depressed expression of its target, inhibited proliferation, and accelerated differentiation of erythrocyte precursors both in vitro and in vivo. Similarly, upregulation of the Lim-only protein 2 (LMO2), also required for erythroid differentiation, is a consequence of a decline in its targeting miRNA, miR-223 (650).

In contrast to miR-221/222 and miR-223, miR-451 is dramatically and specifically upregulated during the differentiation of erythroid progenitors, which suggests that it plays a positive role during that transition (405). Indeed, knockdown of miR-451 in zebrafish resulted in anemia due to defective differentiation of the progenitor cells (148, 475). Furthermore, the expression of miR-451 is driven by a GATA-1-dependent promoter (148), which reconciles with the fact that GATA-1 is also required for the differentiation of erythroid cells (189). miR-451 was also shown to be necessary, but not sufficient, for mediating GATA-1's function. Some of its targets that were proven to mediate its effect include GATA-2 (475) and c-myc, which is known to inhibit erythroid differentiation (148). Meanwhile, as miR-144 matures from the same transcript as miR-451, it targets and inhibits the $\alpha$-globin promoter activator kruppel-like factor and induces the

developmental switch from the fetal α-hemoglobin to the adult β-hemoglobin isoform (187).

**E. miRNAs in Hematological Disorders**

### 1. Lymphoma

Before the discovery of miRNA in mammalian cells, Tam et al. (596) reported that the “Bic” integration site for the avian leukosis virus, which induces B-cell lymphomas, generates a noncoding RNA. Provirus insertion into the BIC promoter increases the expression of a chimeric RNA containing Bic exon 2; however, since this transcript lacked a significant open reading frame, its role in lymphomas remained undefined until recently. After the emergence of miRNAs, it was soon discovered that this noncoding transcript harbored miR-155 (155, 707), thus potentially explaining Bic’s function. The significance of the amplification of miR-155 in lymphomas was examined by ectopically expressing it in B cells in a transgenic mouse model. This resulted in pre-B-cell expansion and bone marrow replacement, splenomegaly, and lymphopenia that preceded the development of lymphoblastic leukemia and lymphoma (122). It was noted that Bic/miR-155 transcript is upregulated in select forms of lymphomas, including Hodgkin’s, diffuse large B-cell (DLBCL), and primary mediastinal B-cell lymphomas (155, 306), but not primary Burkitt lymphoma (305). Interestingly, ectopic expression of Bic in a Burkitt lymphoma-derived cell line failed to increase mature miR-155 levels due to a possible block to its posttranscriptional processing (307), which reconciles with the lack of correlation between the levels of Bic and miR-155 in various lymphoma cells line (155). Although it is now well established that miR-155 plays a critical role in the development of lymphomas, its upstream regulatory pathway or downstream targets that mediate its malignant effects have not been fully elucidated. A few miR-155 targets have been validated in normal B cells and macrophages as described above, but it remains uncertain whether the same targets are relevant in the lymphoma cell background. A recent study by Pedersen et al. (479), though, does show that SHIP1 is a direct target of miR-155 in lymphoma cells, similar to what is observed in macrophages (456, 121); however, the extent to which it mediates the effect of miR-155 in this context remains to be examined. Meanwhile, miR-155's upstream regulators include TNF-α, which enhances its expression in DLBCL lymphoma cells.

Calin et al. (68) have reported that more than 50% of miRNAs are located at genomic sites that are disrupted or amplified in various cancers. Consistently, 13q21-qter region, which is amplified in DLBCL (500), mantle cell (136), and follicular (446) B-cell lymphomas, also encompasses miRNAs (463). The candidate gene, Chromosome 13 open reading frame 25 (C13orf25), within the 13q31 region amplified in DLBCL is preferentially upregulated in B-cell lymphoma cell lines and DLBCL patients with 13q31-q32 amplifications (463). Similar to Bic, this gene does not include a substantial open reading frame, but does harbor a cluster of seven miRNAs (miR-17~92) (463). In agreement, this miRNA cluster was found to be increased ~10-fold in 65% of B-cell lymphoma samples (240). As mentioned above, knockout of miR-17~92 inhibited B-cell maturation, whereas its overexpression induced an increase in B1 cells and germinal centers. On the other hand, when overexpressed with the myc oncogene, it greatly accelerated c-myc-induced pre-B cell lymphoma in mice and reduced apoptosis. Validated targets of this cluster that potentially mediate its function include the proapoptotic gene Bim (312, 633) and the cell cycle inhibitor CDKN1A/p21waf1/Cip1 (264). c-Myc enhances the transcription of the miR-17~92 gene (459), but suppresses that of miR-15a/16-1, miR-34a, miR-150, miR-195, Let-7, miR-26, and miR-29, suggesting that they may be tumor suppressors in this context (86). Reciprocally, let-7a directly targets and inhibits myc, creating a double-negative feedback loop (526). In addition to the above miRNAs, independent downregulation of miR-143 or miR-145 results in enhanced B-cell proliferation by derepressing the expression of ERK5 (6).

An imbalance in the steady-state stoichiometry of a specific miRNA/the number of its targeted sites is likely to perturb the translational output of those targets. Theoretically, this could be induced by fluctuations in the level of the miRNA, mRNAs, or the number of targeted sites within the mRNAs, as a result of amplifications, deletions, or mutagenesis of those sites. Most findings hitherto have focused on changes in the miRNAs’ levels that directly impact its targets. On the other hand, a substantial increase or a decrease in a mRNA will imbalance the availability of a specific miRNA that would indirectly affect other targets. With this comes the realization that in some cases the fluctuations in a miRNA’s level may be a compensatory response to changes in its targets’ levels under the same conditions. The third possibility is that a specific miRNA recognition site within a mRNA is altered through mutagenesis. An example of this is provided by the work of Chen et al. (93), where they demonstrated that the deletion in the 3'UTR of cyclin D1 (CCDN1) in mantle cell lymphoma releases that gene from the negative regulatory effect of miR-16 and, thus, enhances its translation. It is now realized that this phenomenon occurs on a much broader scale in cancer through selective cleavage and polyadenylation of mRNAs that leads to elimination of their miRNA-targeted sites and, thereby, higher translational output (410).

### 2. Chronic lymphocytic leukemia

Chronic lymphocytic leukemia (CLL) is the second most common form of leukemia in adults. The chromosomal region 13q14 is frequently deleted in CLL patients and was, thus, predicted to harbor a tumor suppressor gene (146). One of the potential candidate genes in this region is

the noncoding deleted in lymphocytic leukemia 2 (DLEU2)  
gene (379). Interestingly, this gene was subsequently found  
to be the host for the miR-15a/miR-16–1 cluster, located  
between its second and fifth exons (67). In addition, germ-  
line mutations were discovered in the miR-15a/16–1 pre-  
cursor, which reduce its expression and are also associated  
with its deletion (67). In support, an inactivating mutation  
in the same cluster was found in the New Zealand black  
CLL mouse model (502). Alternatively, a small number of  
CLL cases that did not harbor a deletion of the miR-15a/  
16–1 locus had a deletion in the host gene's promoter (350).  
Accordingly, mature miR-15a/16–1 was significantly  
downregulated in 68% of CLL patient samples tested.  
miR-15 and miR-16 are ubiquitously expressed from two  
genes, miR-15a/16–1 on chromosome 13 and miR-15b/  
16–2 on chromosome 3, being most abundant in normal  
CD5+B lymphocytes. Consistent with their being tumor  
suppressors, they directly target and inhibit the antiapop-  
totic gene BCL2 (111), the tumor suppressor immunoglob-  
ulin superfamily member 4 (IGSF4), and cyclins E1 and D1  
(93, 350), in addition to indirectly inhibiting an array of  
genes related to proliferation, oncogenesis, and suppression  
of apoptosis (66). In agreement, overexpression of miR-16  
inhibits cell proliferation (373) and suppresses tumor  
growth after in vivo implantation (66). While the data over-  
whelmingly support the fact that miR-16 is downregulated  
in carcinogenesis, a couple of studies have reported upregu-  
lation of miR-16 in CLL that did not correlate with the  
13q14 deletion status (191, 649). However, the functional  
significance of this upregulation was not examined. This  
discrepancy between studies may reflect differences in the  
clinical stage or nature of the samples that were examined  
that may exhibit differential compensatory upregulation of  
the homologous miR-15b/16–2 locus. Thus a detailed time  
course of the expression pattern of miR-16 in CLL patient  
samples may help unravel the reason behind these conflict-  
ing data. Whereas the miR-15/16 cluster plays a dominant  
role in CLL, the roles of miR-29 and miR-181 should not be  
overlooked, as they are both downregulated and target the  
oncogene T-cell leukemia/lymphoma 1 (TCL1) (481).

3. Acute myeloid leukemia

Profiling of miRNAs in cytogenetically normal acute  
myeloid leukemia (AML) samples that were associated with  
an internal tandem duplication in the fms-related tyrosine  
kinase 3 gene (FLT3-ITD), wild-type nucleophosmin  
(NPM1), or both, resulted in identification of a miRNA  
signature that predicted the disease’s outcome (404).  
Among those, miR-181 inversely correlated with event-free  
survival. This is consistent with the fact that miR-181 is  
normally present at low levels in HSC but specifically in-  
creases in differentiated B lymphocytes (89). Another dif-  
ferential display revealed a distinct pattern of miRNAs that  
differentiated between AML with mutant cytoplasmic  
NPM1c+ vs. normal NPM1 (198). This uncovered the  
preferential downregulation of miR-204 and upregulation  

of its validated targets HOXA10 and MEIS1 in NPMc+  
AML that was associated with a more favorable prognosis.  
Additionally, a separate miRNA profiling revealed that  
AML patients with higher levels of miR-191 and miR-  
199a had a lower event-free survival rate (201). In gen-  
eral, unique miRNA signatures distinguish the different  
AML karyotypes (145), although more studies are neces-  
sary to determine the functional relevance of these differ-  
ent miRNA patterns.

There is a dearth of functional studies regarding the  
role of specific miRNAs in AML. AML with a t(8;21) trans-  
location that generates the acute myeloid leukemia 1/ETO  
fusion oncogene is associated with reduced levels of miR-  
223 that inversely correlates with the severity of the pheno-  
type (dedifferentiation) (172). Interestingly, AML1/ETO  
was found to directly bind and inhibit the promoter of  
miR-223 through heterochromatic silencing. This is consis-  
tent with the fact that miR-223 is upregulated during gran-  
ulocyte differentiation (281) and induces differentiation in  
myeloid leukemic cells (172). Although knockout of miR-  
223 was not sufficient for the induction of frank AML, it  
did elicit neutrophilia (281). Another miRNA that is impli-  
cated in CLL is miR-29. Similar to CLL and other forms of  
cancer, miR-29 is downregulated in AML. The miR-29b-1/  
miR-29a cluster is located in the frequently deleted 7q32  
region, whereas a second miR-29b-2/miR-29c cluster is lo-  
cated in 1q23. Reintroducing miR-29 into leukemic cells  
decreased proliferation, increased apoptosis, and inhibited  
tumor growth in vivo (199). Three of its relevant validated  
direct targets include the Cxxc6 (tet1) oncogene, the anti-  
apoptotic Mcl1, and the cyclin-dependent kinase Cdk6.  
Ectopic expression of miR-29 also induces upregulation  
of the cell cycle inhibitors p15INK4b and ESR1 through  
demethylation of their promoters via directly targeting  
the methyltransferases DNMT3A and DNMT3B and in-  
directly DNMT1 (200).

4. Chronic myeloid leukemia

One of the characteristics of CML is a translocation of  
t(9;22) (640), which results in the production of the BCR-  
ABL fusion gene (25, 29, 135, 242). Similar to other cancer-  
related chromosomal aberrations described above, ~30%  
of Philadelphia-positive patients harboring this translo-  
cation also encompass a deletion that includes miR-199b  
and miR-219–2 (87). However, due to limited sample  
size (n = 2) in that particular study, neither the expres-  
sion pattern nor disease correlation could be established  
for these miRNAs. The function of BCR-ABL involves the  
induction of myc, which augments it transformation ca-  
pacity (531). As indicated above, myc also induces miR-  
17~92 expression, which enhances the development of  
myc-induced pre-B cell lymphoma (240). Venturini et  
al. (634) extended these findings to show that a BCR-  
ABL-myc-dependent pathway also regulates miR-17~92  
in CML CD34+ cells, in which it is similarly involved in

MicroRNAs IN DEVELOPMENT AND DISEASE

antiapoptotic and proliferative functions. While BCR-ABL regulates the expression of miR-17~92, it is itself a target of miR-203. miR-203 is located within a fragile region on chromosome 12, where its promoter is frequently hypermethylated in Philadelphia-positive tumors versus those that are free of ABL1 aberrations (62). miR-203 directly targets and inhibits the expression of ABL1 and BCR-ABL. It is plausible that downregulation of miR-29, which is implicated in aberrant hypermethylation in CLL (200), may be involved in silencing of the miR-203 promoter in CML.

5. Acute lymphocytic leukemia

Not surprisingly, ALL and CLL have distinct miRNA expression patterns (702). Moreover, ALL's unique miRNA signature could accurately distinguish it from AML (423). In particular, miR-128a, miR-128b, let-7b, and miR-223 were sufficient for discriminating between the two forms of leukemia with a 97% accuracy. miR-128a and miR-128b were higher in ALL, while Let-7a and miR-233 were higher in AML. Whereas let-7 is known for its tumor suppressive function via targeting c-myc in lymphoma cells (526), miR-223 has been shown to increase during granulocyte differentiation (281) and is sufficient for inducing differentiation of myeloid leukemia cells when ectopically expressed (172). In addition, downregulation of both miRNAs is the trend observed in many cancers, as described in other sections of this review. Accordingly, their higher expression in AML versus ALL does not reconcile well with the better overall prognosis associated with ALL. Thus stage-specific screening of more samples is needed to resolve this discrepancy.

Similar to CLL and CML, epigenetic regulation of miRNAs play a role in the underlying pathogenesis of ALL. Specifically, the promoter of miR-124a was found to be hypermethylated and silenced in ALL samples (4). Moreover, the extent of methylation correlated with higher mortality rates and proved to be an independent marker for overall, and disease-free, survival. Functionally, overexpression of miR-124a reduced cell growth in vitro and in vivo models, the effects of which could be ascribed to its targeting and inhibiting the expression of the cell cycle protein, cyclin-dependent protein 6 (Cdk6), and reducing phospho-Rb. Another unique mechanism that may be involved in the etiology of ALL involves miR-125b-1. This miRNA was found inserted in the rearranged immunoglobulin heavy chain gene locus in a single ALL patient, the frequency and reproducibility of which remains to be examined (572). Furthermore, a genome-wide approach using extensive sequencing has recently identified an array of novel miRNAs in ALL that await characterization (704).

F. Virus-Regulated miRNAs in the Immune System

1. Epstein-Barr Virus

As in other pathological conditions, viral infections can perturb a cell's miRNAs levels, where some may be involved in antiviral mechanisms, others can participate in regulating the virus's latency and lytic cycles. Viruses also express their own miRNAs (v-miRNA) that regulate their production in host cells (485). Unlike eukaryotic miRNAs, though, v-miRNAs are generally not conserved. It was first noted in Epstein-Barr virus (EBV) latently-infected B cells that this virus expresses unique v-miRNAs that originates from the Bam H1 fragment H open reading frame 1 (BHRF1) and the Bam H1-A region rightward transcript (BART) viral genes (485). Recently, deep sequencing of the EBV miRNome in nasopharyngeal carcinoma revealed 44 BART miRNAs (94). The results also uncovered 5' and 3' end v-miRNA variants, as well as nucleotide variants that resulted from posttranscriptional editing. Some of these v-miRNAs target lytic genes, suppress viral proliferation, and sustain latency. Specifically, miR-BART2 is expressed from the antisense strand of the viral DNA polymerase BALF5 lytic gene and is, thus, 100% complementary to that gene and induces degradation of its mRNA. Additionally, an EBV miRNA, v-snoRNA124PP, derived from a v-snoRNA precursor targets BALF5 (260).

EBV miRNAs are also found in various types of lymphomas that have been linked to the EBV virus (674). This includes the expression of miR-BHRF1-3 in a type III versus type I BL cell line, miR-BART2 in primary unmanipulated type I BLs and EBV+ primary effusion lymphomas (PELs), and both BART2 and BHRF1-3 in DLBCLs. By targeting and suppressing IFN-inducible T-cell attracting chemokine (C-X-C motif) ligand 11 (CXCL11), miR-BHRF1-3 may potentially be involved in evasion of the immune system (674). At the same time, EBV miRNAs can suppress certain cellular proteins for a survival advantage. One example is miR-BART5, which targets p53 upregulated modulator of apoptosis (PUMA) and, thereby, enhances resistance of the host cell to apoptotic reagents (108). In addition, EBV can expand its effects by utilizing cellular miRNAs. This includes miR-155, which is upregulated in latency type III EBV-positive Burkitt's lymphoma cell lines (305). Infection of human lymphocytes with EBV proved that this is indeed a direct effect of the viral infection and is mediated via a latent membrane protein 1 (LMP1)-NFκB-dependent mechanism (202, 384). The upregulation of miR-155 seems to be responsible for an increase in EBV copy number in latently-infected cells. Likewise, the EBV LMP1 enhances the expression of miR-146a via NFκB in Burkitt's lymphoma cells (433). Overexpression of miR-146a in Akata cells revealed that it predominantly suppresses interferon-responsive genes and is, thus, predicted to provide the virally infected cells with an advantage

Physiol Rev • VOL 91 • JULY 2011 • www.prv.org 839

against the immune system (71). Other miRNAs that are also upregulated in latency III include miR-21, miR-23a, miR-24, miR-27a, and miR-34a (70). In contrast, miR-96 and miR-128a/b were selectively reduced in lymph nodes of EBV-positive classic Hodgkin lymphoma patients (445). It is notable, though, that Godshalk et al. (209) found that 99.5% of cellular miRNAs were downregulated after initial EBV infection of human B cells that was distinct from ligand (CD40)-induced activation. Six months later, however, miR-155, miR-21, miR-17–5p, and miR-20 were highly upregulated.

### 2. Kaposi's sarcoma herpes virus

Kaposi's sarcoma herpes virus (KSHV) infection is associated with the development of malignancies, including Kaposi's sarcoma and primary effusion lymphoma. This virus expresses 12 pre-miRNAs that are clustered in the transcript of the latency kaposin gene (484, 524). Deletion of 10 of these miRNAs (K-1 through -9 and -11) results in upregulation of the lytic genes, including the immediate early transactivator Rta (383). However, in that study, the deletion did not significantly perturb latent gene expression or lytic replication, but was associated with an alteration in the virus's histone methylation and acetylation patterns and a reduction in the host's DNA methylation. Specifically, miR-K4–5p regulates DNA methylation via suppressing the host cell's retinoblastoma-like protein 2 (Rbl2), which is a negative regulator of DNMT3a/b. In contrast, Lei et al. (347) show that the same deletion did indeed enhance lytic replication, which was attributed to the deletion of miR-K1. Knockdown of miR-K1 is sufficient for inducing IkBα and, thus, inhibition of NFκB (347). miR-K1 also targets the cell cycle inhibitor p21 (210), whereas miR-K5, miR-K9, and miR-K10 target bcl2-associated factor 1 (BCLAF1) and reduce sensitivity to etoposide-induced apoptosis (718). Furthermore, KSHV miRNAs target musculoaponeurotic fibrosarcoma oncogene homolog (MAF) and induce dedifferentiation of endothelial cells (232). Interestingly, several herpes viruses miRNAs, including KSHV miR-K7, HCMV miR-UL112, and EBV miR-BART2–5p, target MHC class I polypeptide-related sequence B (MICB) at multiple sites as a means of evading natural killer cells (437). Collectively, these cellular targets may potentially contribute to the oncogenic capacity of KSHV.

One of the most intriguing KSHV miRNAs is miR-K11 (miR-K12–11), which is an ortholog of mammalian miR-155 (211, 564). Thus miR-K11 has the capacity to target the same cellular genes identified for miR-155 that are mentioned above and, accordingly, may play a role in the development of KSHV-induced B-cell lymphomas, which remains to be tested. Indeed, gene expression profiling revealed an overall overlapping pattern in cells expressing either miRNA (211, 564). In addition to miR-155 validated targets, miR-K11 was shown to target BTB and CNC homology 1, basic leucine zipper transcription factor 1

(BACH-1) (211, 564), c-Fos, XIAP associated factor 1 (BIRC4BP), angiotensin II receptor-associated protein (AGTRAP), SAM domain and HD domain 1 (SAMHD1), and riboflavin kinase (RFK) (211), while miR-K1, miR-K3–3p, miR-K6–3p, and miR-K11 target the tumor suppressor thrombospondin (525).

Similar to EBV, KSHV exploits the host cell's miRNAs to its advantage. Specifically, it induces upregulation of miR-146a via an NFκB-dependent pathway, which then targets and downregulates chemokine (C-X-C motif) receptor 4 (CXCR4). This effect is predicted to induce premature release of KSHV-infected endothelial progenitor cells into the bloodstream, thus contributing to formation of Kaposi's sarcoma. Another miRNA that is induced by herpes viruses is miR-132 (326). It is upregulated after infection of lymphatic endothelial cells with KSHV and promotes viral replication via suppressing p300 and, in turn, interferon-stimulated genes.

### 3. Human immunodeficiency virus type 1

Similar to double-stranded DNA viruses, the single-stranded RNA human immunodeficiency virus type 1 (HIV-1) was predicted to harbor five pre-miRNA sequences (32). Indeed, direct cloning from HIV IIIB-infected MT-4T cells and Northern blot verification confirmed the expression of a potential miRNA (miR-N367) from the *nef* gene (462). This gene is a conserved accessory gene that overlaps with the LTR and is expressed during viral replication. The data show that miR-N367 targets *nef* and reduces viremia, suggesting that it may play a role in reducing HIV-1 virulence. In addition, stem-loop transactivation responsive element (TAR) that is present at the 5′ end of all HIV-1 transcripts is processed by Dicer and produces two miRNAs, miR-TAR-5p and miR-TAR-3p (303, 465). These miRNAs are thought to inhibit viral transcription through recruitment of HDAC1 to the LTR promoter and, thus, plausibly promote latency. Further characterization of the function of these miRNAs and their targets in vivo is still needed.

The inhibitory effects of HIV-1 miRNAs on viral production reconciles well with the suppressive effect that both Dicer and Drosha have on viral replication (620). Bennasser et al. (30, 31) discovered that HIV-1’s Tat protein interacts with Dicer and suppresses miRNA/siRNA processing. Furthermore, recent studies show that Vpr suppresses the expression of Dicer in macrophages (113). This antagonization of Dicer inhibits the processing of viral as well as cellular miRNA. In particular, the downregulation of cellular miR-17~92 was proven necessary for enhancing the virus’s replication in Jurkat cells (620). One of the verified targets of miR-17–5p that explains its effect is the acetyltransferase p300/CBP-associated factor (PCAF) (620), which is required for Tat activity (297). Additionally, downregulation of miR-198 and upregulation of its target T cyclin is also

necessary for the virus's replication in monocytes (587). Intriguingly, several cellular miRNAs were also found to target a 1.2-kb region at the 3′end of the viral genome and inhibit all viral gene expression and replication (249). These include miR-28, -125b, -150, -223, and -382, which are more abundant in resting versus activated CD4+ T cells. Notably, inhibiting these miRNAs in resting T cells from HIV-1-positive patients induced virus production. There is also evidence that elevated levels of these miRNAs render monocytes more resistant to HIV-1 infection (654).

In contradiction to a generalized reduction in miRNAs processing, miR-29a was found to be highly upregulated in HIV-1-infected human T lymphocytes (444). In this case, miR-29a interacted with the HIV-1’s 3′-UTR and enhanced its association with RISC and P bodies, resulting in disruption of the latter. This led to enhanced virus replication and infectivity. This apparent inconsistency with the reported inhibition of Dicer can be explained by posttranscriptional regulation of miR-29a (291, 426, 618). Thus cellular miRNAs regulate viral replication indirectly through suppression of critical cellular proteins such as PCAF and T cyclin or disruption of P bodies, or directly through suppression of all viral proteins including TAT and Rev. These results demonstrate that certain cellular miRNAs play an essential role in regulating HIV-1 latency and may potentially serve as candidates for replacement therapy.

IV. MicroRNAs IN THE HEART

A. miRNAs in Cardiogenesis

Direct cloning (328) and microarray expression profiling (102, 533, 600, 626) have identified a large array of miRNAs that are expressed in the heart. In particular, miR-1 and miR-133 are highly enriched in the heart and the skeletal muscle and, thus, their roles in cardiac development have been the focus of several studies. miR-1 is highly conserved among species and the most abundant miRNA in the adult heart, where it represents ~45% of miRNA in the adult mouse heart (328) and ~24% in the adult human heart (358). Its expression coincides with the onset of cardiac myocyte differentiation in the developing heart tube and somitic myotomes in chick embryos (131) and in the looping mouse heart at embryonic day (E) 8.5, where it is regulated by SRF (713). Similarly, in *Drosophila*, dmiR-1 is expressed in the presumptive mesoderm during embryonic stage 5, and later in all its muscle cell derivatives including the dorsal vessel (the mammalian heart counterpart) (568). Its function in *Drosophila* was uncovered in a homozygous knockout model, which showed that dmiR-1 is dispensable during muscle development, but is required for normal muscle growth and survival as the larvae transition from stage 1 to stage 2 (568). In contrast, its function in the rodent heart is essential during development, as perturbation of its normal levels by overexpression of a transgene,

using a beta myosin heavy chain (βMHC) promoter, resulted in developmental arrest at E13.5. The observed defects included thin-walled ventricles and heart failure that is associated with reduced myocyte proliferation. Since miR-1 targets Hand2, it is likely that the phenotype is a consequence of an untimely reduction in endogenous Hand2 by excess miR-1 (713). Hand2, which is a transcription factor that appears in the straight heart tube of a mouse at E7.75, is critical for cardiac development (577). Targeted deletion of Hand2 is embryonic lethal at E11 and is associated with an absence of the right-sided ventricle, aortic arch arteries, trabiculae, and dilated aortic sac. The imperfect alignment between the effects of the reduction of Hand2 by overexpressing miR-1 versus genomic ablation suggests an involvement of additional miR-1 targets that modify the phenotype.

miR-1 functions were further addressed by genomic ablation of one (miR-1–2) of its two genes (miR-1–1 and miR-1–2). While this had no impact in the heterozygous animals, it was lethal between E15.5 and early neonatal life in the homozygous knockout mice (712). Since only one of two genes was deleted in this model, mature miR-1 continues to be produced by the second gene. However, because its levels were undetermined, the extent of reduction in mature miR-1 that elicited the phenotype is unknown. This is especially hard to estimate since quantitative differences in the transcriptional and posttranscriptional regulation between alleles is also unknown. The lethality observed in the homozygous mice was associated with ventricular septal defects in ~50% of embryos that died soon after birth, and pericardial edema and cardiac dysfunction in utero. In contrast to the phenotype observed in the *Drosophila*, there were no obvious skeletal muscle defects. However, since the level of miR-1 in the skeletal muscle of the knockout mice was undetermined, the possibility remains that the second gene might have compensated for any reductions. Furthermore, ~15% of the mice that escaped early lethality succumbed to ventricular dilatation and dysfunction within 2–3 mo. The remaining survivals had no major physical or contractile cardiac abnormalities, but there was frequent wall thickening due to hyperplasia that was accompanied by prolongation of the action potential and occasional sudden death. This is reminiscent of a conductivity defect and could be partly explained by the upregulation of miR-1’s target Irx5 (712). Irx5 is a negative transcriptional regulator of the potassium channel Kcnd2/Kv4.2 that is expressed in an endocardial-to-epicardial gradient, responsible for an inverse expression pattern of Kv4.2, which creates a repolarization gradient in the myocardium (120).

miR-133 is another muscle-enriched miRNA, albeit its expression level in the heart is less than that of miR-1. While deletion of one or two miR-1 genes had a lethal phenotype, deletion of one of two genes of miR-133 expressed in the heart had no effect on cardiac development. In particular,

targeted deletion of miR-133a-1 or miR-133a-2, which each resulted in ~50% reduction in mature cardiac miR-133, equivalent to the decrease associated with cardiac hypertrophy, exhibited no cardiac growth or functional abnormalities under normal or pressure overload conditions (377). On the other hand, complete ablation of miR-133 via double gene knockout resulted in increased proliferation and apoptosis of myocytes, ventricular septal defects, and prevalent embryonic lethality, whereas those that survived developed severe cardiac dilatation and failure, but no associated hypertrophy (377). In contrast, with only ~25% of residual miR-133 that is expressed from the miR133b gene, skeletal muscle had no overt abnormalities. The lack of an effect in the skeletal muscle implies that either mature miR-133 is more than fourfold higher than saturating levels relative to its targets or that the effects of its reduction are not manifest under normal conditions. The latter explanation is more consistent with the function of miRNAs in general.

Interestingly, the phenotype of a βMHC-driven, cardiac-specific, miR-133 transgenic mouse was mostly similar to that of the miR-1 transgenic model described above. Specifically, the mice died in utero with reduced myocyte proliferation, thinning of the cardiac walls, and heart failure at E13.5 (377). miR-133 validated targets in this case include cyclin D2, which partly explains its effect on myocyte proliferation. In addition, serum response factor is also a validated target that was increased in the double-knockout hearts and might have contributed to the increase in smooth muscle actin observed in this model. Previous work has shown that overexpression of SRF in the heart leads to delayed-onset cardiac hypertrophy and reexpression of the fetal gene program at 24 wk of age that progresses to failure (708). In contrast, cardiac ablation of SRF results in developmental defects and lethality, associated with increased apoptotic cells (451, 474). Although the miR-133 knockout model exhibited an increase in SRF, it was not accompanied by myocyte hypertrophy, and in contradiction to SRF ablation, exhibited an increase in apoptotic cell death (377). Thus more miR-133 targets must be involved in the underlying pathology of the knockout phenotype.

Friend of GATA 2 (Fog-2) is also an important regulator of cardiac development whose deficiency results in embryonic lethality associated with ventricular and atrial septal defects and hypoplasia (589, 605). Recently, it was found that miR-130a directly targets and inhibits Fog-2. Unlike miR-1 and miR-133, Fog-2 is more widely expressed, with higher levels in the heart and lung than observed in kidney, liver, or brain (298). Consistent with the function of its target, forced expression of miR-130a in the heart resulted in a phenotype that is reminiscent of that induced by ablation of Fog-2 (298). Thus cardiac development is regulated by at least three miRNAs, miR-1, miR-133, and miR-130a,

through an array of targets whose scope has not been fully discovered yet.

### B. miRNAs in Cardiac Hypertrophy

Early evidence that miRNAs were involved in cardiac pathology was attributed to the fact that they are differentially regulated during cardiac hypertrophy and failure in both rodent and human heart (102, 262, 407, 438, 533, 585, 600, 611, 626) [FIGURE 3]. The underlying pathogenesis in cardiac hypertrophy involves reprogramming of the gene expression profile to greatly resemble that of the fetal/neonatal heart (269, 280, 399). In agreement with a role of miRNAs in this switch, its expression pattern changes in accordance (499). One of the earliest changes observed after applying pressure overload on a mouse heart was the downregulation of miR-1, which preceded any other miRNA changes, the increase in cardiac mass, or contractile dysfunction (533). This suggested that miR-1 might be a cause rather than an effect of the underlying pathogenesis. Consistently, miR-1 is downregulated in genetic models of cardiac hypertrophy including the calcineurin (261) and AKT (76) transgenic hearts. In humans, miR-1 is decreased in the hypertrophied left ventricle of acromegalic patients (159) and in those with aortic stenosis (262), in which the ejection fraction is preserved (50–60%). In support of a general role in building muscle mass, miR-1 is also downregulated during hypertrophy of skeletal muscle (412). The significance of this reduction during myocyte hypertrophy was uncovered after normalizing its levels with exogenous miR-1, which completely suppressed growth-stimulated protein synthesis (533). On the other hand, knockdown of miR-1 in vivo in an adult mouse proved that it was also sufficient for induction of cardiac hypertrophy (76). Some of its predicted targets that have been validated include RasGAP, Cdk9, Rheb (only by Western blots) (533), insulin-like growth factor I (IGF-I) (159), calmodulin, and Mef2a (261), all of which play essential roles during cardiac hypertrophy.

In the failing human heart, the measurements of miR-1 levels were inconsistent, and accordingly, its function and targets in that context are less explained. While Matkovich et al. (407) and Thum et al. (611) report upregulation of miR-1, Ikeda et al. (262), Sucharov et al. (585), and Naga Prasad et al. (438) report a reduction in miR-1, in ischemic and nonischemic dilated cardiomyopathies accompanied by a significant reduction in percent ejection fraction. Overall, we may conclude that miR-1 functions in regulating the expression of genes that are essential for the increase in cell size and mass that characterize hypertrophy. However, as hypertrophy transitions into dilated cardiomyopathy and failure, miR-1 levels return to normal or above normal levels. This increase is also seen during ischemic heart disease, which will be discussed in the following section.

MicroRNAs IN DEVELOPMENT AND DISEASE

miRNAs in cardiac hypertrophy

Gene switch ($\alpha$MHC $\rightarrow$ $\beta$MHC)

Cellular hypertrophy

$\uparrow$ miR-208b  
$\downarrow$ Thrap1  

$\uparrow$ miR-208b  
$\downarrow$ Myostatin  

$\uparrow$ miR-23a  
$\downarrow$ MuRF1  

$\downarrow$ miR-1  
$\uparrow$ RasGAP  
$\uparrow$ Cdk-9  
$\uparrow$ Rheb  
$\uparrow$ Calmodulin  
$\uparrow$ Mef2a  
$\uparrow$ IGF1  

Fibrosis

$\uparrow$ miR-21  
$\downarrow$ SPRY1  

$\downarrow$ miR-30c  
$\uparrow$ CTGF  

$\downarrow$ miR-133  
$\uparrow$ CTGF  

Cell viability

$\downarrow$ miR-133  
$\uparrow$ HCN2  

Electrical remodeling

$\downarrow$ miR-1  
$\uparrow$ HCN2  
$\uparrow$ HNC4  

$\uparrow$ miR-208b  

FIGURE 3 A diagram showing miRNAs and their targets in cardiac hypertrophy. The diagram displays the different miRNAs and their targets that are involved in gene switching, cellular hypertrophy, fibrosis, and electrical remodeling during cardiac hypertrophy. Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of target genes inversely correlate with that of the targeting miRNA and are accordingly listed in red (downregulated) or black (upregulated) dialog boxes under the targeting miRNAs. All listed targets have been validated. The listed targets include the following: thyroid hormone receptor-associated protein 1 (Thrap1), myostatin, muscle-specific RING finger protein 1 (MuRF1), Ras GTPase-activating protein (RasGAP), cyclin-dependent kinase 9 (Cdk9), Ras homolog enriched in brain (Rheb), calmodulin, myocyte enhancer factor 2A (Mef2a), insulin-like growth factor 1 (IGF1), sprouty 2/4 (SPRY2/4), connective tissue growth factor (CTGF), hyperpolarization-activated, cyclic nucleotide-gated K⁺ 2/4 (HCN2/4), and the indirect targets $\alpha$- and $\beta$-myosin heavy chain ($\alpha$- and $\beta$MHC).

In addition to miR-1, the other muscle-enriched miRNAs, miR-133, is also downregulated during cardiac hypertrophy (76, 102, 626). Knockdown of miR-133 via antisense targeting was sufficient for inducing cardiac hypertrophy and reinduction of the fetal gene program in the adult mouse heart (76). In contrast, though, targeted deletion of miR-133a-1 or miR-133a-2, which each resulted in ~50% reduction in cardiac miR-133, equivalent to the decrease associated with cardiac hypertrophy, exhibited no cardiac growth or functional abnormalities under normal or pressure overload conditions (377). The discrepancy between the outcomes could be explained by the temporal difference of miR-133 knockdown in the heart that is elicited by these two distinct approaches. A more definitive answer was derived by using a transgenic mouse model in which the overexpressed miR-133 compensated for its downregulation during hypertrophy (408). Although cardiac weight was not normalized as a result of this intervention, other aspects of hypertrophy that included apoptosis, fibrosis, and the downregulation of $I_{Kr}$, were restored to baseline levels, while the decline in $I_{to,f}$ and prolongation of Q-T interval, were partly recovered. However, miR-133 targets that mediate its effect under these conditions remain to be identified. Thus, while knocking down a gene in a normal cell background is an invaluable tool for understanding its function, restoring its level during a pathological condition is requisite for examining its function in that specific context. However, one has to also consider that overexpression of the transgene prior to inducing the pathology may have impacted the phenotype. These results are a clear indication that the outcome of manipulating the levels of endogenous miRNAs is highly dependent on the cell's transcriptome/proteome makeup.

While miR-1 and miR-133 are enriched in both skeletal and cardiac muscle, miR-208 is cardiac restricted (37). It has two isoforms, miR-208a and -208b, that have identical seed sequences, which diverge in only three nucleotides over the length of their mature sequence, and are, thus, predicted to have common targets (217, 218). Interestingly, these two genes are contained within the introns of *Myh6* ($\alpha$MHC) and *Myh7* ($\beta$MHC) genes, respectively. In the mouse heart, $\beta$MHC is expressed at high levels during early cardiogenesis and declines after birth, while $\alpha$MHC displays an inverse

Physiol Rev · VOL 91 · JULY 2011 · www.prv.org 843
pattern. During pressure-induced cardiac hypertrophy, an isoform switch results in recapitulation of the neonatal phenotype (517). Since the miR-208a/b genes are coexpressed with their host genes (625), it is predicted that isoform switch will be paralleled by an equivalent switch in miR-208a/208b that is likely to sustain the level of total mature miR-208 and, thus, its targets under these conditions. However, while miR-208a level does not decrease within 3 wk of cardiac hypertrophy, miR-208b levels increased in parallel with βMHC (69), thus resulting in a net increase in miR-208. It was suggested that the lack of change in miR-208a levels is due to its relatively long half-life. In addition to miR-208, miR-499 is also contained within a myosin gene, Myh7b, whose transcription is constitutively high in the heart and slow-twitch skeletal muscle.

miRNAs that are enriched in skeletal and/or cardiac muscles have been appropriately dubbed myomiRs (411, 642). Their functions have been investigated by targeted genomic ablation in mice. Homozygous miR-208a mice (miR-208a⁻/⁻) are developmentally normal with the expected normal levels of α- and βMHC at birth, while the adult mice failed to upregulate βMHC in response to thyroid hormone inhibition (627). Likewise, pressure overload on the heart did not induce the expected increase in βMHC. These effects are thought to be mediated by thyroid hormone receptor-associated protein 1 (THRAP1), which is a validated target of miR-208 that is upregulated in the knockout mouse. THRAP1/MED13/TRAP240 is involved in the transcriptional regulation of a wide array of genes, including those regulated via nuclear receptors (308, 527, 699). In agreement, a transgenic mouse model overexpressing miR-208a in the heart reduced THRAP1 and induced an increase in βMHC, suggesting that endogenous miR-208a is below saturating levels (69, 627). In addition to the lack of responsiveness to thyroid hormone inhibition in the miR-208a⁻/⁻, there was absence of hypertrophic growth and fibrosis following an increase in workload, in addition to reduced levels of miR-499 (627). Interestingly, overexpression of miR-499 in the miR-208a⁻/⁻ model restored the normal response to thyroid hormone inhibition, suggesting that it is a downstream effector of miR-208 and is sufficient for mediating its effect. Paradoxically, though, ablation of miR-499 did not recapitulate the miR-208a⁻/⁻ phenotype (625).

Like miR-208a⁻/⁻, miR-208b⁻/⁻ mice were also developmentally normal. In line with the expression pattern of their host genes, miR-208a is expected to be the major source of mature miR-208 in the adult mouse heart, while miR-208b is the major source in the developing heart. Accordingly, ablation of the latter provides a more definitive conclusion regarding the lack of a role for miR-208 in cardiac development. Moreover, the heart of a double miR-499/miR-208b knockout was also normal, which contrasted with the lack of βMHC and type I myofibers in the soleus muscle. Sox6, Sp3, and Purb were identified as likely candidates in

mediating the suppression of βMHC expression in the absence of miR-208a or miR-499. Thus, while miR-208 is not necessary for the expression of βMHC during cardiac development, it is required for its upregulation during cardiac hypertrophy and hypothyroidism (FIGURE 3).

In contrast to the lack of a phenotype in the miR-208a/b knockout models, overexpression of miR-208a in mice induced cardiac hypertrophy that was only manifest after 4 mo of age (69). This phenotype, though, was not associated with a decrease in αMHC, an increase of ANF, or a change in any of the hypertrophy-associated miRNAs. In contrast, myostatin, which is a negative regulator of hypertrophy (430), is a validated target of miR-208a that was suppressed in the transgenic heart and, thus, might explain the hypertrophic growth (69). These mice also exhibited defects in conduction in the form of first- or second-degree atrioventricular bundle block, compared with the miR-208a⁻/⁻ mice, which displayed atrial fibrillation. As a possible mechanism, connexin40 (Cx40) and its transcriptional regulator Hop were dramatically reduced in the miR-208a⁻/⁻ mice, although they were unaffected in the transgenic model. Thus modulation of these proteins could potentially explain the conduction defects in the former but not the latter model. Meanwhile, GATA4, a positive regulator of Cx40 (371) and a proven target of miR-208a, is upregulated in miR-208a⁻/⁻ mice; however, this does not reconcile with the observed downregulation of Cx40.

While the focus in this section was on the roles of miR-1, miR-133, and miR-208 in cardiac hypertrophy, the differential display studies have revealed an abundance of other miRNAs that are also dysregulated that are apt to play a role in the underlying pathogenesis (FIGURE 3). However, not many have received as much attention as did the myomiRs. miR-21 is one of the most highly and consistently upregulated miRNA during cardiac hypertrophy (102, 533, 600, 626), whose role in hypertrophy has been elusive, as a transgenic mouse overexpressing the gene in the heart was completely normal and its response to pressure overload hypertrophy was comparable to wild-type mice (612). While miR-21 is upregulated in the whole heart during pathological hypertrophy, it appears that its increase in the myofibroblasts is more significant than it is in the myocytes (612). Through targeting and suppressing sprouty1, it enhances erk1/2 phosphorylation and myofibroblast survival (612). This effect indirectly contributes to the increase in fibrosis seen during cardiac hypertrophy. Accordingly, antisense targeting of miR-21 reduced fibrosis and hypertrophy. In addition, miR-21 also targets sprouty2 via which it increases connexin43-positive junctions between myocytes, facilitating the exchange of small molecules between cells (534). Since pathological hypertrophy is associated with a rise in myocyte apoptosis, it is plausible that the increase in miR-21 is also protective of cardiac myocytes under this condition. Indeed, miR-21 displays an antiapoptotic func-
MicroRNAs IN DEVELOPMENT AND DISEASE

tion in cardiac myocytes during myocardial ischemia, as will be described in the next section.

miR-23a, which is clustered with miR-27a and miR-24, is also upregulated during pressure overload (102, 533, 600, 626) and isoproterenol-induced hypertrophy (370). It is directly regulated by NFATc3 and is both necessary and sufficient for mediating the hypertrophic effect of the calcineurin-NFATc3 pathway. It exerts its effects via targeting and suppressing the negative regulator muscle ring finger 1 (MuRF1). In accordance, knockdown of miR-23a in the mouse heart using antisense antagomirs results in an increase in MuRF1 and near-complete inhibition of isoproterenol-induced cardiac hypertrophy, including changes in molecular markers and cardiac function. This reconciles with the phenotype observed of the MuRF1 knockout mice, which develop more pronounced pressure-induced hypertrophy (664).

C. miRNAs in Ischemic Heart Disease

Myocardial ischemia is a result of insufficient blood supply that leads to apoptotic, autophagic, and necrotic cell death, and subsequent fibrosis of the heart that impairs contractility, and is the leading cause of mortality in developed countries. miRNAs levels are rapidly perturbed upon exposure of cells or organs to hypoxia or ischemia (FIGURE 4). In contrast to cardiac hypertrophy, miR-1 and miR-133 are upregulated during exposure of myocytes to oxidative stress, where miR-1 was found to target heat shock proteins (HSP) HSP60 and HSP70, and miR-133 targets caspase-9 (679). Accordingly, miR-1 exerts proapoptotic effects, while miR-133 is antiapoptotic. On the other hand, miR-320 is downregulated during ischemia/reperfusion in the heart and is responsible for upregulation of HSP20 (507). Further reduction of endogenous miR-320 with an antisense construct successfully reduces infarct size after ischemia reperfusion. Meanwhile, miR-21, which is ubiquitously expressed, has emerged as a critical regulator of cell apoptosis in both myocytes and nonmyocytes in the myocardium. It is acutely reduced in the heart during ischemia, specifically within the ischemic region, where replenishing it reduces infarct size (147, 532). In contrast, it is upregulated in the periinfarct zone where the myocytes are undergoing hypertrophy. This rescue was seen in a cardiac-specific transgenic model of miR-21 (532), as well as via local viral delivery

miRNAs in cardiac ischemia

![Diagram](#)

FIGURE 4 A diagram showing miRNAs and their targets in cardiac ischemia. The diagram displays the different miRNAs and their targets that are involved in cell viability, angiogenesis, fibrosis, and electrical remodeling during cardiac ischemia. Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of target genes inversely correlate with that of the targeting miRNA and are accordingly listed in red (downregulated) or black (upregulated) dialog boxes under the targeting miRNAs. The listed targets include hypoxia-inducible factor 1α (Hif-1α); sirtuin 1 (Sirt1); heat shock proteins 20, 60, and 70 (HSP20, HSP60, HSP70); programmed cell death 4 (PDCD4); phosphatase and tensin homolog (PTEN); fas ligand (FasL); caspase 9; integrin α5 (ITGA5); sprouty-related EVH1 domain containing 1 (SPRED1); connexin 43 (Cx43); inward rectifier potassium channel 2 (Kir2.1); and cardiac voltage-gated potassium channel modulatory subunit (Kcnip2).

of that miRNA (147). These results were further confirmed in isolated cardiac myocytes, in which exogenous miR-21 was shown to inhibit apoptosis by targeting Fas ligand (FasL) (532) or programmed cell death 4 (PDCD4) (104). Interestingly, miR-21 is a downstream effector of AKT that partly mediates its antiapoptotic effect via suppression of FasL (532).

miR-21 is also enriched in the myofibroblasts and infiltrating cells in the infarct zone relative to the surrounding myocytes. In these cells, it inhibits PTEN and induces upregulation of metalloprotease-2 (515), which promotes fibrosis. As mentioned above, miR-21 is also profibrotic during cardiac hypertrophy, via enhancing myofibroblast survival (612). However, the involvement of this mechanism in myocardial ischemia has not been explored yet. In addition to miR-21, downregulation of miR-29, which targets multiple collagen isoforms (628), may potentially contribute to the development of fibrosis during ischemia. In support, antisense knockdown of miR-29b in a mouse model confirmed that it is sufficient for inducing an increase in collagen in the heart. Thus precise replenishment of miR-29 in the ischemic heart may be a potential therapeutic advantage. On the other hand, since miR-21 has both antiapoptotic and profibrotic effects, its therapeutic targeting has to be carefully evaluated.

The response of a cell to hypoxia or ischemia is bimodal, in the form of an initial, adaptive, conditioning reaction that transitions into cell death upon persistence of the insult. Accordingly, brief exposures to hypoxia or ischemia elicit an antiapoptotic response that involves activation of the AKT pathway (237), which subsequently protects the cells against protracted periods of hypoxia or ischemia that may immediately follow. This is known as early hypoxia preconditioning (HPC), or ischemia preconditioning (IPC) (436). miRNAs are also involved in regulating this process, which requires translational but not transcriptional events (514), including upregulation of hypoxia-inducible factor 1α (Hif-1α) (65, 154). Specifically, miR-199a-5p is acutely downregulated through a posttranscriptional mechanism, in cardiac myocytes and the heart, following HPC or IPC, respectively. In myocytes, this downregulation proved both necessary and sufficient for the rapid upregulation of its target Hif-1α (497). Sirt1 is also a direct and validated target of miR-199a-5p that complements its function by downregulating prolyl hydroxylase 2 and, thereby, stabilizing Hif-1α (497). These effects are mediated by AKT, which induces upregulation of Hif-1α and Sirt1 via downregulation of miR-199a-5p (498). Thus miR-199a-5p is an example in which a miRNA functions as an immediate early switch that enhances the translation of a gene whose expression is required for an acute response. In contrast, AKT induces upregulation of miR-21 (532), which is also increased following ischemia preconditioning of the heart (105, 694). Through targeting major antiapoptotic genes such as PTEN, FasL, and PDCD4, it

assumes a powerful antiapoptotic function during ischemic conditions. Thus the AKT pathway utilizes miRNAs to both increase prosurvival proteins and suppress proapoptotic molecules during early ischemia preconditioning.

In addition to miRNAs-regulated apoptotic and survival pathways involved in myocardial ischemia, miRNAs also regulate angiogenesis and, thus, influence the rate of tissue healing after ischemia. Please refer to the section on angiogenesis for a description on the role of miRNAs in this vital process.

### D. miRNAs in Electrical Remodeling of the Heart

Arrhythmias are characteristic of coronary/ischemic heart disease, where they are elicited by abnormalities in electrical conduction and/or repolarization. As miRNAs regulate a cell's response to ischemia, they also regulate the expression of molecules involved in mounting an action potential, as well as the cell's conductivity. miR-1 is increased in the human hearts that have suffered coronary artery disease (CAD) and in rat ischemic hearts, which are associated with an increase in arrhythmogenesis (686). Increasing or decreasing miR-1 in the ischemic rat heart results in an equivalent effect on the frequency of arrhythmias. miR-1 targets that could explain its effects include the gap junction protein Cx43 and Kir2.1, which is a subunit of the K⁺ channel that mediates I<sub>K1</sub>. Inhibition of arrhythmias by knockdown of endogenous miR-1 is reversed by concomitant knockdown of both these molecules. In support, ablation or missense mutation of Cx43 (349) or I<sub>K1</sub> (489), respectively, are associated with increased incidence of arrhythmias. In contrast to their regulation in ischemic conditions, miR-1 and miR-133 are downregulated during cardiac hypertrophy, where miR-1 targets HCN2 and HCN4, and miR-133 targets HCN2 (395). These proteins are components of the pacemaker channel responsible for the hyperpolarization-activated current (I<sub>f</sub>). They are also upregulated during cardiac hypertrophy and failure and have been implicated in arrhythmogenesis (425). The downregulation of miR-133 during cardiac hypertrophy is also associated with a decrease in the I<sub>to,f</sub> accessory subunit Kcnip2, albeit via an indirect mechanism, and is, accordingly, responsible for prolongation of the QT interval (408). Thus miR-1 and miR-133 are major regulators of various disease aspects in both the hypertrophic and ischemic heart through a broad spectrum of targets (Figures 3 and 4).

### V. MicroRNAs IN SKELETAL MUSCLE

#### A. miRNAs in Myogenesis

miRNAs are intricately involved in self-renewal and differentiation of myogenic stem cells (MSC) (FIGURE 5). MSC

MicroRNAs IN DEVELOPMENT AND DISEASE

miRNAs in myogenesis

$\uparrow$ miR-181  
$\downarrow$ HoxA11  
$\uparrow$ MyoD  

$\uparrow$ miR-27  
$\downarrow$ Pax3  
MSC  

$\uparrow$ p27  
$\downarrow$ miR-221/222  

$\uparrow$ miR-206  
$\downarrow$ DNA polα  
cell cycle exit  

$\downarrow$ SRF  
$\uparrow$ miR-133  

$\uparrow$ miR-29  
$\downarrow$ YY1 $\downarrow$ Ezh2  
differentiation  

$\uparrow$ miR-214  
$\downarrow$ miR-26  

$\uparrow$ MEF2  
$\downarrow$ UCP2  

$\downarrow$ HDAC4  
$\uparrow$ miR-133  

$\uparrow$ miR-1  
$\uparrow$ MyoD  

myotube  

FIGURE 5 A diagram showing miRNAs and their targets in myogenesis. The diagram displays the different miRNAs and their targets that are involved in cell cycle exit of myogenic stem cells (MSC) and their differentiation into myotubes. The muscle-specific transcription factor MyoD plays a central role through regulating the expression of the muscle-enriched miR-1, miR-133, and miR-206. Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of a target gene inversely correlate with that of the targeting miRNA and are similarly represented by an up or down arrow. Only the miRNA targets that have been specifically verified in skeletal muscle are listed. All listed targets have been validated. The listed targets include homeobox A11 (HoxA11), paired box gene 3 (Pax3), DNA polymerase alpha catalytic subunit (DNA polα), serum response factor (SRF), zeste homolog 2 (Ezh2), ying and yang 1 (YY1), histone deacetylase 4 (HDAC4), and uncoupling protein 2 (UCP2).

maintain their self-renewal capacity in part through the transcription factor Pax3 (13). Thus downregulation of Pax3 is necessary for exiting the cell cycle and myogenic differentiation. miR-27 was identified as a targeting miRNA of this molecule and is accordingly expressed in differentiating skeletal muscle in both the myotome and adult satellite cells (127). A transgenic mouse model confirmed that ectopic expression of miR-27 downregulates Pax3 and induces premature differentiation of progenitor cells. Conversely, knockdown of miR-27 delays differentiation, which is consistent with the effect of sustained expression of Pax3 (57). While Pax promotes proliferation, the polycomb group (PcG) suppresses differentiation by occupying the MyoD enhancer site and inhibiting MyoD-dependent transcription (343). The zeste homolog 2 (Ezh2) is a histone lysine methyltransferase subunit of this complex that mediates promoter silencing through H3K4 and H3K27 methylation. In accordance, Ezh2 is present in myotomes and is downregulated upon differentiation, allowing MyoD to bind to and activate muscle genes (77). Interestingly, PcG was found to occupy a MyoD enhancer site upstream of miR-214 in undifferentiated muscle cells (285). Upon differentiation, PcG is released and MyoD induces upregulation of miR-214, which creates a negative feedback loop by targeting and inhibiting Ezh2 (285). Accordingly, overexpression of miR-214 accelerates C2C12 differentiation. Similar results were observed with miR-26a, which also targets Ezh2, although in this case it is not known whether MyoD regulates its expression (666). Notably, Ezh2 requires the transcriptional regulator YY1 for binding to, and blocking, MyoD enhancers (343). miR-29 targets and suppresses YY1, which in turn inhibits miR-29 in a negative feedback loop (666). Thus upregulation of miR-214, -26, and -29 is collaboratively involved in suppressing PcG expression and function in muscle stem cells and, thereby, promoting muscle-specific gene expression and differentiation. In addition, upregulation of miR-181 promotes differentiation by suppressing a MyoD negative regulator, HoxA11 (439), while downregulation of miR-221/222 derepresses p27kip1, inhibiting proliferation (75).

Upon initiation of differentiation, MSC become enriched in miR-206, miR-1, and miR-133, plausibly via myoD-dependent transcriptional regulation (501). While both miR-1 and miR-133 simultaneously increase during differentiation of C2C12, they were reported to have antagonistic effects (91). miR-1's expression is necessary and sufficient for inducing differentiation (91, 441), whereas miR-133 expression suspends differentiation via sustaining proliferation (91). Their relevant targets in this context are HDAC4 and SRF, respectively, where HDAC inhibits the transcriptional activity of the muscle-specific Mef2c, and SRF inhibits its proliferation of myoblasts. The relevance of having these two opposing but overlapping functions during muscle differentiation is not readily explainable, since the accelerated differentiation induced by overexpression of miR-1 has not been examined in the absence of miR-133, or vice versa. In contrast, though, another study showed that upregulation of miR-133 is essential for downregulation of uncoupling protein 2 (UCP2), which is an inhibitor of muscle differentiation (96). UCP1 is a mitochondrial transporter that uncouples ATP synthesis. Although UCP2 is highly homologous to UCP1, its primary function remains unknown. The UCP2 is an example of a gene whose mRNA but not its translational product is found in the heart, which is consistent with the fact that it is a target of the muscle-enriched miR-133. Its inhibition, but not that of UCP3, proved necessary for C2C12 differentiation.

In contrast to the effect of miR-1 and miR-133 in cultured cells, genomic knockout of both miR-133a-1 and miR-133a-2 (377) or miR-1–2 (712) had no apparent impact on skeletal muscle development in the surviving mice. In contrast, disruption of miR-133 or miR-1 in zebrafish resulted in reduced fiber size, while knockdown of both disrupted sarcomeric actin organization (428). The phenotype of the miR-1–2 knockout mouse model could be explained by compensatory upregulation of the second allele of miR-1 (miR-1–1) or miR-206, which have the same seed sequence. On the other hand, genomic ablation of miR-133a resulted in more than 70% reduction of endogenous mature miR-133, with no apparent compensation from miR-133b, suggesting that miR-133 had no role in muscle differentiation or growth in vivo. Thus the role of miR-1 and miR-133 in muscle development in vivo needs a more detailed examination. On the other hand, there is evidence that downregulation of miR-1/206 is associated with muscle hypertrophy, as the levels of miR-1 and miR-133 are ~50% lower in hypertrophied skeletal muscle in mice (412), similar to what is observed in cardiac hypertrophy (76, 533).

While miR-1 and miR-133 are also expressed in the heart, miR-206 is a skeletal muscle-restricted miRNA, which, in consensus, is induced by MyoD (513). Its mature sequence differs in only three nucleotides from that of miR-1, which are outside the seed sequence. Thus the predicted targets of miR-1 and miR-206 are the same. Accordingly, it is not surprising to find that like miR-1, overexpression of miR-206 enhances differentiation of C2C12 cells (299). Moreover, like miR-1 and miR-133, ablation of miR-206 in a mouse model had no effect on normal muscle development (663). Additional direct targets that were validated for miR-206 include the p180 subunit of the DNA polymerase α-subunit (299), whose downregulation coincides with cell cycle exit and differentiation of various tissues (252). Although Cx43 is a predicted and validated target that is suppressed by miR-1/206 during C2C12 differentiation, this finding does not agree with many studies that show that Cx43 is also critical for myoblast differentiation (14, 15, 494). It could be envisioned, though, that the ratio of miR-206 to Cx43 may fine-tune the expression of Cx43 during different stages of muscle development; thus more detailed dose- and time-dependent experiments are required for clarifying the role of miR-206 in regulating Cx43. miR-206 also directly targets and suppresses the expression of utrophin and follistatin (513), which are required in formation of the postsynaptic fold (138) and stimulation of angiogenesis (464), respectively, but have no known direct roles in myogenesis yet.

Thus, at least in cultured skeletal myotubes, miR-1, -133, and -206 are positively involved in myogenic differentiation through the regulation of various aspects of this process. Ishikawa et al. (442) exploited this functionality by locally applying a mixture of these miRNAs to an injured rat tibialis anterior muscle, where they observed enhanced muscle regeneration and less fibrosis. Indeed, this could provide future therapeutic utility during muscle injury.

### B. miRNAs in Muscle Pathologies

#### 1. Muscular dystrophies

Expression profiling of miRNAs revealed 185 differentially expressed miRNA in 10 major muscular disorders in humans including Duchenne muscular dystrophy (DMD), Becker muscular dystrophy (BMD), facioscapulohumeral muscular dystrophy (FHMD), limb-girdle muscular dystrophies types 2A and 2B (LGMD2A and LGMD2B), Miyoshi myopathy (MM), nemaline myopathy (NM), polymyositis (PM), dermatomyositis (DM), and inclusion body myositis (IBM). Interestingly, five miRNA only (miR-146b, miR-221, miR-222, miR-155, and miR-214) were consistently upregulated in all forms of MD except BMD, which did not include any of the muscle-enriched myomiRs (miR-206, miR-1, miR-133). Another interesting profiling was one that compared DMD samples or dystrophin-deficient mice (mdx) with acute ischemic muscle degeneration and regeneration phases (214). The miRNAs were accordingly classified into three categories that included the following: regeneration miRNA (miR-31, -34c, -206, -335, and 449) that were upregulated in DMD/mdx and postischemic regeneration, degeneration miRNA (miR-1, -29c, and -135a)

that were downregulated in DMD/mdx and postischemic degenerative phase, and inflammatory miRNA (miR-222 and -223) that were increased in the damaged, infiltrated, regions. However, caution has to be exercised in interpreting these data, as McCarthy et al. (413) have previously shown that miR-206 is not uniformly upregulated in the different skeletal muscles of mdx mice. Also, the functional relevance of these changes during muscle regeneration awaits investigation.

### 2. Rhabdomyosarcoma

This form of sarcoma originates from a muscle cell lineage. One of its characteristic is the increased expression of c-Met, a tyrosine kinase receptor activated by hepatocyte growth factor (178, 505). miR-1/miR-206 are reduced to undetectable levels in rhabdomyosarcoma (RMS) and have been shown to directly target c-Met, suggesting that they contribute to the upregulation of this protein (601, 684). Indeed, overexpression of either miRNA in RMS cells resulted in reduced proliferation, migration, and increased apoptosis. In addition, it induced a shift in the gene expression profile of RMS cells towards that of a differentiated muscle (601). More conclusively, ectopic expression of either miRNA in implanted tumor cell inhibited tumor growth (601, 684). Similarly, YY1, a cancer-associated transcription factor (79), is also increased in RMS and is involved with miR-29 in a double negative regulatory loop (647). In accordance, overexpression of miR-29 in implanted RMS cells resulted in retardation of tumor growth.

### 3. Neurodegenerative diseases

Denervation induces differential dysregulation of muscle miRNA in the form of downregulation of miR-1 and miR-133 within 1 wk in rat soleus muscle, which returns to normal levels during the reinnervation phase (272). On the other hand, miR-206 is initially unchanged but increases during reinnervation. Although genomic ablation of miR-206 had no impact on normal muscle or neuromuscular junction development, it did provide evidence that miR-206 is involved in accelerating the reinnervation process (663). This was further substantiated by the fact that the absence of miR-206 accelerated muscle atrophy in an amyotrophic lateral sclerosis (ALS) mouse model (expressing an activated mutant of superoxide dismutase). miR-206 targets and inhibits the expression of HDAC4, which is accordingly increased in miR-206 knockout (KO) relative to denervated wild type muscle. Thus, as predicted, muscle-specific knockout of HDAC4 enhanced muscle reinnervation. It was suggested that these effects might be mediated by fibroblast growth factor binding protein 1, which is downregulated in the miR-206 KO and upregulated in the HDAC4 KO mice, although its role in this process remains to be elucidated.

---

### VI. miRNAs IN THE BRAIN

#### A. miRNAs in Neurogenesis

Knockout of Dicer provided unequivocal evidence that miRNAs play a role in brain development. Using an *Emx1* gene promoter that is specifically expressed in neuroepithelial cells in the dorsal telencephalon starting at embryonic day 9.5, or a CAMKII promoter that is expressed in excitatory neurons of the cortex, hippocampus, and caudate-putamen starting at E15.5, Cre-lox-dependent ablation of dicer in both models resulted in a smaller cortex and premature death around P21 (134, 137). This could be partly explained by an increase in programmed cell death that was observed in the cortex. In addition, the former model was associated with a 30% reduction in the neuronal layer and number of neurons at E13.5 (137), while the latter model was associated with a reduction in dendritic branching, an increase in dendritic spine length, and ataxia (134). Intriguingly, though, the thickness of the progenitor layers, progenitor cell proliferation, development, and differentiation were normal during early neurogenesis (137). The subsequent reduction in mitotic cells in the ventricular zone (VZ) and subventricular zone (SVZ) at E14.5 was attributed to an increase in cell death. Apoptosis is also a feature of postnatal knockout of dicer in differentiated Purkinje cells, using Pcp2 promoter-Cre recombinase, which is associated with cerebellar degeneration and ataxia (535), plausibly due to downregulation of ataxin1-targeting miR-19, miR-101, and miR-130 (345). The results suggest that while miRNAs are not required for cell differentiation, they are involved in cell homeostasis and survival.

miR-124 is a brain-enriched miRNA (328), where it starts increasing at E13 and remains elevated throughout adulthood (317, 567). In particular, it is localized to differentiated neurons (567) and is suppressed in non-neuronal cells by the RE1-silencing transcription factor (REST) (114). In agreement with the phenotype of Dicer ablation, ectopic expression or knockdown of miR-124 in the chick neural cord, including the neuronal progenitor cells, had no impact on neuronal differentiation or neurogenesis (74). However, its overexpression was associated with some cell death and disruption of the basal lamina that encloses the neural tube, via suppression of laminin γ1 and integrin β1. The latter is consistent with the fact that in some cases miRNAs and their predicted targets have evolved to avoid coexistence (170). Thus ectopic expression of miR-124 inhibits expression of its targets laminin γ1 and integrin β1, which are highly expressed in the ventricular zone progenitor cells but minimally in the high miR-124-expressing mantle zone. Controversially, another study showed that overexpression of miR-124 enhanced neurogenesis in the same model via suppression of synaptopetal complex protein 1 (SCP1), an anti-neural factor that is recruited by REST (637). The difference between studies could be explained by the fact that

miR-124 in the subventricular zone stem cells niche regulates the timing of differentiation of transit amplifying cells into neuroblasts and neuroblasts into neurons, where it is upregulated at each of these steps (101). Furthermore, knockdown of miR-124 delayed, but did not inhibit, differentiation of progenitor cells during neuronal regeneration.

While miR-124 had no independent effect on stem cell differentiation, it did collaborate with miR-9 to preferentially inhibit the differentiation of progenitors cells into glial cells, while increasing the ratio of neuron to glial cells (318). miR-124 was found to target and inhibit distal-less homeobox 2 (Dlx2) and Jagged 1 (Jag1), which play roles in interneuron formation and self-renewal of SVZ, respectively (101). Another highly relevant target is Sox9, which is known to be involved in promoting glial cell specification (581). In agreement, overexpression of miR-124-resistant Sox9 in SVZ cells delayed neurogenesis as it maintained the astrocyte population. miR-124 also exerts its function by inducing an alternative splicing pattern of genes that is characteristic of neurons (401). It exerts this effect through targeting and inhibiting the splicing suppressor polypyrimidine tract binding protein 1 (PTBP1), which results in alternative splicing of exon 10 of PTBP2 and an increase in its protein. In addition to its role in neurogenesis, miR-124 may be required for neurite outgrowth through suppression of Cdc42 and relocalization of Rac1, as seen during the differentiation of P19 cell (697).

miR-9 is also enriched in the brain (328), as it gradually increases during the development of the mouse brain, but unlike miR-124, it declines after birth (317, 567) and exists in both neurons and astrocytes (567). Except for subtle abnormalities in the sensory organs in *Drosophila*, it is dispensable for this species' normal development and survival (367). In particular, its ablation resulted in extra ddaE and ddaF neurons in the third instar larvae and extra sensory bristles on the anterior wing margins of adult flies, suggesting that it is required for controlling the precise number of sensory organs generated. This is consistent with a role of miRNAs in fine tuning cellular processes versus essentiality. On the other hand, in zebrafish, miR-9 is necessary for establishing the midbrain-hindbrain boundary (351). In the developing mouse brain, at E10.5, miR-9's expression is dorsomedially graded in the cortex, and at E12.5 it becomes enriched in the most medial pallium (558). Intriguingly, Foxg1, which suppresses differentiation of neuroblasts into Cajal-Retzius cells (231), is absent in this region and has been validated as a miR-9 target. Cajal-Retzius cells are the earliest born neurons in the neocortex that arise from the cortical hem. In vivo gain- and loss-of-function of miR-9 in the developing mouse brain ascertained that it was critical for the differentiation of progenitor cells into Cajal-Retzius cells (558). Although overexpression of miR-9 in neuronal stem cells was not sufficient for inducing differentiation, it did accelerate the process in

the presence of differentiating medium, leading to an increase in both glial and neuronal cells in vitro and in vivo through targeting tailess (TLX) (711), a nuclear receptor involved in adult neural stem cell self-renewal (329). miR-125 is also one of the miRNAs that is highly expressed in the brain and has a similar expression pattern as miR-9 during the development of the mouse brain (567). However, in contrast, it was reported to be both necessary and sufficient for inducing differentiation of a human neuroblastoma cell line as well as a human neuronal progenitor cell line (335).

miR-134 is another brain-restricted miRNA, increasing gradually in the hippocampus, where it plateaus at P13 after synaptic maturation and becomes localized to dendrites near synaptic sites (537). Overexpressing it in hippocampal cells decreases the volume of dendritic spines and vice versa, an effect that is mediated by Lim domain-containing kinase-1 (LIMK-1). In agreement, a knockout model of this protein exhibits a similar phenotype (418). The dendritic spines are known to increase in size during synaptic excitation and, thereby, establish sites of synaptic contact. It is proposed that, upon brain-derived neurotrophic factor (BDNF) stimulation, miR-134 is released from LIMK-1 3'-UTR, resulting in upregulation of the protein, which increases spine volume and, thus, establishment of synaptic contact (537). In addition, a differential display revealed other miRNAs that are enriched in synaptosomes versus forebrain, which include miR-9 and miR-138 (561). Similar to miR-134, miR-138 is a negative regulator of spine volume and synaptic transmission, in this case through targeting acyl-protein thioesterase 1, known to regulate synaptic strength (157). On the other hand, BDNF stimulation of cortical neurons induced upregulation of miR-132, but not miR-134 or miR-138, which was required for upregulation of the glutamate receptors through an indirect mechanism (293). One of its previously validated targets include methyl CpG-binding protein 2 (MeCP2), which is the protein mutated in Rett syndrome (304); however, its role in this context remains unknown.

### B. miRNAs in Neurodegenerative Diseases

#### 1. Alzheimer’s disease

Mutations in amyloid precursor protein (APP), presenilin 1, and presenilin 2 are associated with familial Alzheimer's disease (AD). APP is cleaved by the β-amyloid cleavage enzyme (BACE1), generating the C99 product, which is then cleaved by the γ-secretase complex (includes presenilins) into Aβ40 and Aβ42 (reviewed in Ref. 545). Accumulation of β-amyloid plaques is a hallmark of AD; thus there has been much focus on miRNAs that regulate the proteins involved in this process. AD has a distinct miRNA expression pattern, where miR-9, miR-125b, and miR-146a are upregulated in the temporal lobe neocortex of AD patients but not in those with Parkinson's, schizophrenia, or ALS

(549). In contrast, miR-29b is downregulated in sporadic AD patients and inversely correlates with BACE1 expression (241). It directly targets BACE1 and suppresses its expression and, thereby, proteolytic cleavage of APP. Similarly, miR-107 is downregulated during early stages of AD and also directly targets BACE1 (653). Although only confirmed in a mouse model of AD, downregulation of miR-298 and miR-328 also plays a role in regulating BACE1 through direct targeting (51). On the other hand, APP is a direct target of miR-106a and -520c (477), as miR-106b is downregulated in AD (241). It should be noted though, that there is little overlap in the expression pattern of miRNAs in AD samples reported in the different studies. This might be attributed to the unstable nature of the miRNAs as demonstrated by Sethi et al. (549), or differences in the stage of the disease, or sampling sites. Thus more studies are required to confirm these findings.

### 2. Parkinson's disease

Progressive neurodegeneration of dopaminergic neurons (DN) in the substantia nigra is the underlying cause of tremors, rigidity, and bradykinesia in Parkinson's disease (PD). Two different Dicer knockout models reveal a role for miRNAs in the function and survival of DN and dopaminoreceptive neurons. Dicer knockout in ES cells that coincides with the appearance of postmitotic DN (using the dopamine transporter promoter) results in complete loss of these neurons but not GABAnergic neurons (300). Similarly, knockout of Dicer in the postmitotic midbrain DN results in progressive loss of these neurons via programmed cell death and bradykinesia (300). One of the miRNAs that was suggested to be involved in the survival of DN is the muscle-enriched miR-133, which was also found in the midbrain of normal subjects but is absent in PD patients. The results show that miR-133b inhibits DN differentiation by targeting Pitx3; however, this finding disagrees with the loss of DN in the Dicer knockout. On the other hand, miR-133 has been shown to inhibit apoptosis in cardiac myocytes by targeting caspase-9 (679) and, thus, its loss in PD patients may trigger apoptosis of DN.

In contrast to those results, knocking out Dicer in postmitotic dopaminoreceptive neurons in the striatum and deep cortical layers (using dopamine receptor-1 promoter-driven Cre) did not result in neurodegeneration, but was associated with ataxia, reduced brain size due to smaller cells, astrogliosis, behavioral changes that impacted feeding habit, loss of weight, and premature death at 10–12 wk (128). α-Synuclein is also an underlying cause of DN dysfunction and degeneration in PD. Thus a reduction in miR-7, which targets its 3′-UTR, might underlie an increase in α-synuclein protein, which forms the characteristic fibrillar aggregates in nigral PD (286). Notably, α-synuclein is also induced by FGF10, which was identified as a risk factor at 8p21.3–22 with a SNP in the miR-433-binding site within its 3UTR (646).

---

### 3. Huntington's disease

Huntington's disease (HD) is characterized by the gradual atrophy of the striatum that involves loss of neurons (reviewed in Ref. 225). Some of the underlying causes include dopamine toxicity, apoptosis, and autophagy. The mechanism has been partly attributed to increased activity of the REST as it dissociates from the mutant huntingtin protein (719), which is known to suppress the transcription of neuronal genes. Interestingly, these genes include the brain-enriched miR-124a (114), which, as described above, is involved in the timing of neuronal differentiation. Conversely, miR-124 suppresses the expression of non-neuronal genes (114). HD is associated with a decrease in mature miR-9/9*, miR-124, miR-29b, and upregulation of mature miR-132 (467), of which, at least, miR-124, miR-9, and miR-132 genes have RE1 binding sites in their vicinity (670). miR-9/9* is involved in a double negative feedback loop in which miR-9 targets and suppresses REST, while miR-9* targets and suppresses CoREST (467). These findings suggest that downregulation of miR-9 and miR-124, and upregulation of REST in HD, may be involved in suppressing any de novo differentiation of neural stem cells, with eventual atrophy of the striatum.

---

### C. miRNAs in Psychiatric Disorders

#### 1. Schizophrenia

Schizophrenia is a psychiatric disorder with a genetic predisposition that is also associated with differential expression of miRNAs (39, 482). One of the highest genetic risk factors known is a hemizygous microdeletion of the 22q11.2 locus, which is found in 1–2% of cases. To characterize the effects of this deletion, Stark et al. (579) generated a mouse model with an equivalent deletion that spans a 1.3-Mb region and encompasses 27 genes [Df(16)A+/− mice]. As predicted, this mouse exhibited some cognitive and behavioral changes that were reminiscent of schizophrenic traits. Most interestingly, gene expression profiling revealed that several probe sets in close proximity to miRNA genes, including the miRNAs-containing gene (Mirg), were among the top scoring upregulated probes in the prefrontal cortex (PFC) and hippocampus. Notably, the Mirg is abundant in the adult brain and encompasses the largest known cluster of miRNAs, including miR-134, -136, -127, -154, -342, and -345, among several others (543). This finding led to the prediction that haploinsufficiency of Dgcr8, which lies within the 22q11.2 syntenic mouse locus, may be responsible for impaired pri-miRNA processing. In support, 59 miRNAs in the PFC and 30 in the hippocampus were downregulated. These included miR-185 and miR-134 that were present in the microdeletion. In accordance, the targets of these miRNAs were preferentially upregulated. As proof of a role for Dgcr8 in schizophrenia.

phrenia, heterozygous *Dgcr8* knockout mice exhibited a phenotype similar to that of the *Df(16)A+/−* mice. In contrast, though, others have shown that in brain samples of schizophrenic patients, there is a generalized increase in miRNAs' expression that was associated with an increase in DGCR8 (38). Thus a more detailed sample assessment is necessary to determine the role of DGCR8 in schizophrenic patients.

Several SNPs are associated with this disease, such as those in the genes for miR-206, miR-198 (233), and miR-30e (682). In addition, by sequencing 59 miRNAs genes on the X-chromosome of male schizophrenic patients, Feng et al. (176) found eight ultra-rare SNPs in mature or precursor miRNA sequences. These included nucleotide variations in the mature sequence of let-7f-2, miR-188, -325, -660, -509-3, and -510, which altered the degree of their inhibition of target genes, and in the precursor of miR-18b, -505, and -502, which impaired their processing. SNPs are also found in the miRNA-binding sites in genes, such as that found in complexin 2 (CPLX2). Complexins 1–4 (CPLX1–4) are a family of presynaptic regulatory protein, of which CPLX2 was recently tested for its association with schizophrenia, using a phenotype-based genetic association study in a relatively large cohort (26). This led to the identification of a C-to-T SNP in the 3′-UTR binding site for miR-498 seed sequence, which abrogated the corresponding miRNA's inhibitory effect on this gene. In concordance, this SNP was associated with an increase in the expression of the CPLX2 mRNA in peripheral blood mononuclear cells of patients.

### 2. Tourette’s syndrome

Noteworthy, a G-to-A nucleotide variant (var321) was specifically found in 2 of 174 unrelated patients with Tourette’s syndrome, in the binding site of miR-189 within the 3′-UTR of Slit and Trk-like (*Slitrk1*) (1, 460). The mutation enhances the repression of *Slitrk1* by miR-189, as both molecules colocalize in the neocortex, hippocampus, thalamus, and cerebellum. *Slitrk1* is a brain-restricted transmembrane protein with an extracellular region that has homology to Slit, an axonal growth factor and an intracellular domain with homology to trk neurotrophin receptors (17). The protein was shown to induce unipolar neurites in cultured neuronal cells (17). Moreover, a mouse model lacking *Slitrk1* exhibits an anxiety-like behavior that is suggestive of a Tourette’s syndrome trait (288).

### D. miRNAs in Brain Tumors

#### 1. Gliomas

Gliomas are the most common form of adult brain tumors, of which glioblastoma multiforme is the most aggressive, grade IV glioma, and has received the most attention in miRNA studies. miR-21 is one of the most commonly upregulated miRNA in cancers, including glioblastoma, breast, ovarian, colorectal, pancreatic, lymphomas, lung, hepatocellular, prostate, head and neck, gastric, thyroid, cervical, and cholangiocarcinoma. It has an antiapoptotic function in glioblastoma cells, where its knockdown induces activation of caspase-3 and apoptosis (82). Interestingly, gene ontology analysis of miR-21 predicted targets revealed a significant bias in cell growth, proliferation, cell death, cancer, and cell cycle genes (469). With the use of this set of annotated genes, three interacting networks could be predicted that included genes from the transforming growth factor (TGF)-β, p53, and mitochondrial apoptotic pathways. In agreement, knockdown of miR-21 results in activation of these pathways, cell cycle arrest, and apoptosis, plausibly through its validated targets, which include heterogeneous nuclear ribonucleoprotein K (HNRK), p53-related TAp63 (469), and PDCD4 ([FIGURE 6](#figure-6)), in glioblastoma cell lines (98). In addition, miR-21 is also involved in cell invasion and metastasis. It targets the metalloproteinase (MMP) inhibitors, reversion-inducing-cysteine-rich protein with kazal motifs (RECK) and tissue inhibitor of metalloproteinase-3 (TIMP3) and, thus, activates MMPs and facilitates cell motility and metastasis both in vitro and in vivo (192). While miR-21 has also been reported to target PTEN in other cell types, this has not been shown in glioblastomas. On the other hand, miR-26a is another PTEN-targeting miRNA that is upregulated in glioblastomas through amplification of its locus (257). It is plausible that both miRNAs collaborate to effect a more robust inhibition of PTEN and activation of AKT. While miR-21 is evenly upregulated in all glioma tumors, miR-221 is exceptionally higher in glioblastoma multiforme (grade IV) (116). miR-221 and miR-222, which are coexpressed, were found to target and inhibit the cell cycle inhibitor p27kip1 in glioblastoma cells (206). Recently, the AKT pathway through inhibition of FOXO3a was also found to inhibit p27kip1 expression (555). Thus miRNAs may form an integrative network with other signaling pathways to potentiate their effects.

miR-21, miR-26a, and miR-221/222 function as oncogenes by suppressing genes that negatively regulate cell growth, survival, and invasion. In contrast, miR-7, miR-34a, miR-124, miR-137, miR-146b, miR-15b, miR-128, and miR-326 function as tumor suppressors and are, thus, downregulated in glioblastomas. Downregulation of miR-7 complements the suppression of the AKT inhibitor PTEN by miR-26a, by directly derepressing the translation of epidermal growth factor receptor (EGFR) and insulin receptor substrate-1, both of which contribute to activation of the AKT pathway (295). In accordance, overexpressing miR-7 reduced cell survival and invasiveness of glioblastomas. Meanwhile, miR-34a (365) and miR-326 (294) are also downregulated and result in an increase in their common target Notch-1, which is known to enhance the expression of EGFR (496). In a negative feedback loop, Notch-1 was

852 Physiol Rev · VOL 91 · JULY 2011 · [www.prv.org](http://www.prv.org)

MicroRNAs IN DEVELOPMENT AND DISEASE

miRNAs in glioblastomas

| Glial Cell | tumorigenesis | survival | invasion/metastasis |
| --- | --- | --- | --- |
| miR-21 ↑ | hnPRK ↓ | miR-26a ↑ | RECK ↓ |
| miR-221/222 ↑ | TAp63 ↓ | PTEN ↓ | miR-21 ↑ |
|  | p27kip1 ↓ | miR-21 ↑ | TIMP3 ↓ |
|  |  | PDCD4 ↓ |  |
| miR-124 ↓ |  | miR-7 ↓ |  |
| miR-137 ↓ | Cdk6 ↑ | EGFR ↑ |  |
| miR-34a ↓ |  | IRS-2 ↑ |  |
| miR-128 ↓ | E2F3a ↑ | miR-326 ↓ | miR-146a ↓ |
| miR-15 ↓ | cycE1 ↑ | Notch ↑ | MMP16 ↑ |
|  |  | miR-34a ↓ |  |
|  |  | miR-128 ↓ |  |
|  |  | Bmi-1 ↑ |  |

FIGURE 6 A diagram showing miRNAs and their targets in glioblastomas. The diagram displays the different miRNAs and their targets that are involved in transformation, survival, and invasion/metastasis of glioblastomas. Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of a target gene inversely correlate with that of the targeting miRNA and are similarly represented by an up or down arrow. All listed targets have been validated. These include the following: heterogeneous nuclear ribonucleoprotein K (hnRPK), tumor protein p63 (TAp63), cyclin-dependent kinase 6 (Cdk6), E2F transcription factor 3 (E2F3a), phosphatase and tensin homolog (PTEN), programmed cell death 4 (PDCD4), epidermal growth factor receptor (EGFR), insulin receptor substrate 2 (IRS-2), BMI1 polycomb ring finger oncogene (Bmi-1), reversion-inducing-cysteine-rich protein with kazal motifs (RECK), tissue inhibitor of metalloproteinase 3 (TIMP3), and matrix metallopeptidase 16 (MMP16).

shown to inhibit miR-326 (294). Moreover, downregulation of miR-128 enhances the expression of its target Bmi-1 (208), which suppresses PTEN and increases AKT activity (570). Similarly, downregulation of miR-146b complements miR-21's suppression of MMP inhibitors, by relieving the inhibition of MMP16, and thus, enhancing migration and invasion of glioblastoma cells (673). More direct effects on the cell cycle proteins are seen through downregulation of miR-15b, which targets cyclin E1 (672), miR-128, which targets E2F3a (710), and miR-124 and miR-137, which target cdk6 (562), augmenting the effect of miR-34a on the same target (365). [FIGURE 6].

### 2. Medulloblastoma

Medulloblastoma (MB) is the most common form of childhood brain tumors, which may originate from granular cell progenitors (GCP) that fail to differentiate. Sonic Hedgehog (Shh) is a key regulator of cerebellar development, where it is responsible for the expansion of the GCP (623). miRNA, including miR-125b, miR-326, and miR-324-5p, enhance GCP differentiation by targeting the Hedgehog pathway regulator Smoothened (Smo), the latter also targeting the downstream transcription factor Gli (179). These progenitor cells are the most proliferating in the developed brain, where mutations in Shh pathway can result in MB. Indeed, downregulation of miR-125b, miR-326, and miR-325-p is observed in MB with high Shh signaling. In combination, these miRNAs are both necessary and sufficient for cell proliferation of MB cells. In addition, the miR-17~92a cluster was specifically upregulated in tumors with increased Shh activity (622). Overexpression of this cluster accelerated tumor formation and induced up-

regulation of Shh pathway mediators Math1 and Gli1. It is important to note, though, that it did not downregulate any of the targets reported in other cell types, including PTEN, p27kip, p130, or E2F-1, emphasizing the importance of the cell background in defining a miRNA's targets. In addition, the Notch receptor pathway also plays a role in MB where it is regulated by miR-199b-5p, which targets its effector HES1 (197). Lower levels of miR-199b-5p positively correlate with metastasis of MB.

#### E. miRNAs in the Inner Ear

The mouse inner ear hair cells are particularly enriched in miR-182, -183, and -96 (miR-183 family), which are expressed early during the postnatal period and persist into adulthood (660). Conditional Dicer knockout in the otic placode at E8.5 demonstrates the importance of miRNA in inner ear development and, in particular, a role for miR-183 in the formation of the stereocilia of hair cells (573). Specific Dicer knockout in the hair cells furthers confirms the role of miRNAs in the formation of stereocilia (185). Likewise, studies in zebrafish show that knockdown of the miR-182, -183, and -96 results in a reduction in the number of inner ear hair cells (356). These data are suggestive of a role of the miR-183 family in hearing via regulating the proper development and number of the hair cells. Certainly, this was confirmed when two mutations in miR-96's seed sequence were found to segregate with nonsyndromic hearing loss in two affected families (415). These mutations not only impaired miR-96's inhibition of its target genes but also compromised processing of its precursor. Moreover, one of the mutations converted miR-96's seed sequence into that of miR-514. Thus these mutants have potentially acquired

Physiol Rev · VOL 91 · JULY 2011 · www.prv.org

new targets, of which some were experimentally confirmed. Excitingly, this is the first pathogenic miRNA mutation to be confirmed in humans. Moreover, this finding was corroborated in a mouse model, in which a hemizygous N-ethyl-N-nitrosurea-induced mutation in miR-96 resulted in progressive hearing loss (354). This was associated with degeneration of all hair cells in the homozygous mice by 4–6 wk and the outer, but not inner, hair cells in the heterozygous mice. While the miR-96-binding site was enriched in the 3′-UTR of many genes that were upregulated in this model, the direct targets that mediate the effect of miR-96 remain to be identified.

**VII. MicroRNAs IN THE KIDNEY**

### A. miRNAs in Nephrogenesis

Dicer or Drosha knockout in *Xenopus* (2- to 4-cell stage) results in gross malformation and various organ defects, including a reduction in the pronephric tubules, in addition to what appears to be delayed differentiation of the epithelia of the pronephric duct (5). Closer examination of the pronephros revealed that the proximal tubule marker Na⁺/glucose cotransporter (SGLT1-K) was reduced, the intermediate tubule and part of the distal tubule were less convoluted, and the distal tubule was shorter. This implicates miRNAs in later versus earlier stages of pronephros development, including the timing of pronephric duct differentiation and tubule morphogenesis. Compared with other miRNAs expressed in the kidney, the miR-30 family was most restricted to, and highly abundant in, the pronephros. Thus, not surprisingly, when abrogated it induced a phenotype very similar to that of Dicer’s ablation (5). Although miR-30 knockdown reduced proliferation, that function was not responsible for the phenotype. On the other hand, its direct inhibition of the LIM-class homeobox factor Xlim1 may be accountable, as its downregulation in the developing pronephros was delayed.

In a mouse model, podocyte-specific knockout of Dicer results in proteinuria within 2–3 wk, associated with glomerular hypertrophy and tubular dilatation, followed by death due to kidney failure (236, 245, 556). The histological findings include foot process effacement, accumulation of lipid droplets, and cytoplasmic vacuoles. Additionally, the basement membrane exhibits focal splitting, inclusions, and subepithelial projections. At the molecular level, the mutant podocytes had lower levels of the cytoskeletal proteins synaptopodin, ezrin, moesin, and podocalyxin. In addition, they had lower podocyte-specific nephrin, podocin, and CD2-associated protein (CD2AP) proteins, albeit levels of the Wilm’s tumor (WT) transcription factor were normal. Aberrant proliferation and apoptosis are also observed at the later stages, which suggest that they are an effect rather than a cause of the phenotype. mRNAs profiling in the Dicer-deficient podocytes uncovered 190 genes that were

upregulated whose 3′-UTRs were enriched in target sites for 16 miRNAs, which included the 4 miR-30 family members (556). Two of those targets are vimentin and HSP20 that may be involved in the underlying cytoskeletal changes observed in the mutant podocytes.

On the other hand, Dicer knockout in the ureteric bud derivatives results in unilateral or bilateral hydronephrosis in 47% of the mice, with occasional cystic tubules (476). Although the roles of specific miRNAs in the mammalian kidneys have not been identified yet, the Dicer knockout model proved their requirement for normal podocyte differentiation and ureteric development. In addition, the knockout of Dicer in *Xenopus* proved a role for miRNAs in the accurate timing of epithelial differentiation in the pronephric duct.

### B. miRNAs in Polycystic Kidney

Adult-onset autosomal dominant polycystic kidney disease (ADPKD) is characterized by formation of liver and kidney cysts. While the mechanism is not fully understood, it has been partly attributed to enhanced proliferation of the epithelium of renal and biliary ducts (35). As miRNAs play a major role in proliferation of cancer cells, they were predicted to similarly play a role in the pathogenesis of ADPKD. Differential display of cholangiocytes from a rat PKD model revealed downregulation of miR-15a; a similar finding was observed in the liver of ADPKD patients (342). This is consistent with other findings that implicate reduced miR-15 levels in the mechanisms underlying a wide range of cancers (24, 43, 53, 55, 66, 111). In colangioocytes, miR15a targets the cell cycle regulator Cdc25A and, thus, its downregulation increases the protein (342). Moreover, knockdown of miR-15a in normal colangioocytes enhances their potential to form cysts in a three-dimensional culture system, but the role of Cdc25A in mediating this effect was not examined. It is not clear, however, whether miR-15 plays a role in renal cystogenesis, since differential profiling did not reveal any changes in its level (468). Otherwise, there is a plethora of miRNAs that are downregulated in PKD, whereas miR-21 is the only one that is upregulated. Although not specifically interrogated in this tissue, miR-21 is known to play a major role in cell survival and invasion during carcinogenesis and may, thus, play a similar role in kidney cystogenesis.

### C. miRNAs in Diabetic Nephropathy

Diabetic nephropathy is mainly due to the effect of hyperglycemia and cytokines on renal mesangial cells and is characterized by mesangial cell hypertrophy and apoptosis and excessive deposition of extracellular matrix proteins. To identify the role of miRNAs in this process, differential profiling was performed on mesangial cells cultured in high-

glucose medium (651). This revealed a high expression level of miR-377 that was upregulated in a mouse diabetic nephropathy model and via stimulation with TGF-$\beta$. Overexpression of this miRNA in mesangial cells results in an increase in fibronectin, most likely through an indirect mechanism. Indeed, miR-377 directly targets and inhibits p21 protein (Cdc42/Rac)-activated kinase 1 (PAK1) and superoxide dismutase (SOD1) in mesangial cells, where knockdown of either protein increases the production of fibronectin. On the other hand, TGF-$\beta$ plays a major role in the production of collagen in the diabetic kidney through inhibiting the expression of the E-box repressors, ZEB1 and ZEB2, which potentially suppress E-box-dependent expression of collagen (290). miR-192, which is induced by TGF-$\beta$, mediates its inhibition of ZEB2 via targeting that gene's 3'-UTR (290). Interestingly, suppression of ZEB2 also results in directly derepressing the expression of the gene encoding miR-216a and miR-217, both of which directly target PTEN, a negative regulator of the AKT pathway (289). This finding provides a unique mechanism for TGF-$\beta$-mediated activation of AKT. Indeed, TGF-$\beta$ stimulates AKT activity in a miR-21-PTEN-dependent fashion and increases phosphorylation of its downstream effectors, mTOR, FOXO3a, and GSK3$\beta$. This pathway, which is well established in mediating cell hypertrophy in various cell types, may play a role in the pathogenesis of diabetic nephropathy.

D. miRNAs in Renal Cancer

Renal cell carcinoma (RCC) is the third most frequent urological cancer, of which clear cell carcinoma (CCC) constitutes ~70% and is associated with a high mortality rate compared with the less frequent chromophobe cell carcinoma (ChCC). A differential display revealed that miR-200c and miR-141 are preferentially downregulated in CCC (440). Both these miRNAs were found to target and inhibit the E-box repressor ZEB2, which is involved in endothelial-to-mesenchymal transition (EMT). This finding is common to many other cancer forms (2, 63, 215, 310, 313, 472) and is consistent with the fact that the milder form of RCC is not associated with downregulation of the miR-200 family (440). WT, on the other hand, has a unique pattern of gene expression that differentiates it from all forms of RCC (314). This pattern represents an E2F3 signature, a transcription factor frequently amplified in many forms of cancer, including WT. Similar to its effect in Burkitt’s lymphoma (668), E2F3 induces upregulation of the miR-17~92 cluster.

VIII. MicroRNAs IN THE LUNG

A. miRNAs in Lung Development

The first indication that miRNAs were involved in lung development was provided by a knockout mouse model in which the Dicer gene was ablated in sonic hedgehog-expressing cells, which include lung epithelial cells (234). This disruption resulted in the cessation of lung branching at day 12.5, while the epithelial cells continued to proliferate and form large dilated distal ends of the existing branches. Although there was increased apoptosis in the mutant lungs, it was mostly observed at a later time point (E13.0), excluding it as a possible mechanism for the defective branching. The phenotype could be partly explained by upregulation of fibroblast growth factor 10 (FGF10) and its aberrant distribution within the mesenchyme. Since Dicer was deleted in the epithelium, whereas FGF10 is in the mesenchyme, its upregulation could be due to abnormal levels of other, secreted, regulatory factors. Specific miRNAs that play a role in lung development include miR-127. During lung development it is expressed in the mesenchyme and later in the epithelium, where its untimely overexpression disrupts branching of a fetal lung organ culture (41). Similarly, disrupting the normal expression pattern of miR-17~92, which is high during early development and low in the adult lung, via introducing a transgene in epithelial cells, results in delayed differentiation and epithelial hyperplasia at the expense of the alveolar cavity (390). Although retinoblastoma-like 2 (RBL2) was confirmed as target of miR-17–5p and is reduced in the transgenic model, its contribution to the phenotype has not been determined. This phenotype is consistent with the miR-17~92 knockout model, which is embryonic lethal and is associated with a hypoplastic lung but no branching or other obvious defects (633). Thus once more a miRNA (miR-17~92) regulates lung development by regulating the timing of epithelial cell development and exit from the cell cycle.

B. miRNAs in Lung Cancer

Downregulation of let-7 has been one of the more consistent findings in lung cancer and, thus, the most investigated (FIGURE 7). Furthermore, it is associated with poor prognosis after curative resection, independent of disease stage (595). Functionally, overexpression of let-7 inhibits growth of a lung adenocarcinoma cell line (323, 595). It also suppresses growth of tumor xenografts (166, 323) and primary tumors in a K-Ras<sup>G12D</sup> lung cancer model in vivo, when delivered intranasally via a lentivirus vector (166). Furthermore, let-7 does not only retard tumor growth, but also induces regression of established tumors through an apoptosis-independent mechanism (619). Conversely, inhibition of let-7 is sufficient for enhancing cell proliferation (282). Consistent with its targeting and suppression of K-Ras (283) and high mobility group AT-hook 2 (HMGA2) (346), let-7 was more effective in inhibiting growth of K-Ras mutant-positive versus -negative non-small cell lung cancer (NSCLC) cell lines, and its effect was reversed by reintroducing Ras, but less effectively by HMGA2 (323). The tumor suppressor effect of let-7 is further substantiated by its inhibition of the cell cycle proteins Cdk6 and cell division

DANISH SAYED AND MAHA ABDELLATIF

miRNAs in lung cancer

![Diagram](#)

FIGURE 7 A diagram showing miRNAs and their targets in lung cancer. The diagram displays the different miRNAs and their targets that are involved in transformation, survival, angiogenesis, and invasion/metastasis of lung cancer. Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of a target gene inversely correlate with that of the targeting miRNA and are similarly represented by an up or down arrow. All listed targets have been validated. These include the following: cell division cycle 25 (Cdc25), cyclin-dependent kinase 6 (Cdk6), high mobility group AT-hook 2 (HMGA2), K-Ras, v-crk sarcoma virus CT10 oncogene homolog (CRK), histone deacetylase 4 (HDAC4), forkhead box P1 (FoxP1), proviral integration site 2 (Pim-1), hepatocyte growth factor receptor (c-Met), DNA methyltransferase 3A/B (DNMT3A/B), myeloid cell leukemia sequence 1 (MCL1), Bcl2-like 2 (BCL2L2), epidermal growth factor receptor (EGFR), vascular endothelial growth factor A (VEGF-A), zinc finger E-box binding homeobox 1 (ZEB1), and transforming growth factor beta 2 (TGFb2).

cycle 25 (Cdc25) (282). Moreover, the existence of a single-nucleotide polymorphism within the let-7 target site in the 3'-UTR of K-Ras is associated with increased risk of NSCLC, underscoring its tumor suppressive role (106). It should be noted, however, that other miRNAs have essential and complementary roles in lung cancer. Specifically, miR-15/16, which is frequently deleted in leukemia, was also frequently reduced in lung cancer, where it directly influences the cell cycle by targeting cyclin D1 (cycD1), cyclin D2 (cycD2), and cyclin E1 (cycE1) (24). However, since these cyclins function with their kinase partners to phosphorylate and inhibit RB, the effect of miR-15/16 on proliferation is not observed in cancer cells deficient in RB. Additional tumor suppressors that are downregulated in lung cancer include miR-126 (126, 375), miR-107, and miR-185 (594). While miR-126 was found to target v-crk sarcoma virus CT10 oncogene homolog (CRK) and vascular endothelial growth factor A (VEGF-A), miR-107 targets Cdk6. On the other hand, downregulation of miR-29 contributes to tumorigenesis by derepressing the methyltransferase genes DNA (cytosine-5-)-methyltransferase 3 alpha (DNMT3A) and 3B, which invoke aberrant methylation and gene silencing (167).

Interestingly, the two muscle-enriched miRNAs, miR-1 and miR-133b, which are minimally expressed in normal bronchial epithelium, were found dramatically reduced in lung tumors (125, 443). Reexpressing miR-1 in cancer cell lines suppressed proliferation, migration, and tumorigenicity (443). It also induced apoptosis in response to anticancer agents. Some of miR-1's targets that were upregulated in this context included hepatocyte growth factor receptor (c-Met), proviral integration site 2 (Pim-1), forkhead box P1 (FoxP1), and HDAC4. In particular, c-Met has been previously implicated in lung tumorigenesis (495, 519, 578). In addition, Pim-1 has proliferative and prosurvival functions (677), HDAC4 inhibits the cell cycle inhibitor p21 (434, 665), while FoxP1's role is dependent on the type of cancer (311) and remains to be investigated in lung cancer. On the other hand, miR-133 overexpression was mainly shown to induce apoptosis in the presence of anticancer drugs, plausibly by targeting the established prosurvival genes Bcl2-like 2 (BCL2L2) and myeloid cell leukemia sequence 1 (BCL2-related) (MLC1) (125).

In contrast to miRNAs that are downregulated, several were upregulated and are involved in lung cancer cell proliferation, survival, and resistance to chemotherapy (FIGURE 7). This includes the miR-17~92 cluster that is increased in cancer, sometimes through gene amplification, and is involved in enhancing cell proliferation (238). In addition, its abrogation by antisense knockdown proved that it has an antiapoptotic function that is mostly mediated through the miR-17~5p and miR-20a members (409). However, its targets in this particular tissue have yet to be identified. On the other hand, c-Met via AP-1 induces upregulation of miR-221/222, which increases lung cancer resistance to the major extrinsic apoptotic pathway, stimulated by TNF-α-related apoptosis inducing-ligand (TRAIL) (195, 196). This is accomplished by its direct targeting of PTEN, AKT's negative modulator, which has been previously shown to be responsible for resistance to TRAIL through preventing BID cleavage (447). Moreover, miR-221/222 targets TIMP3, a tumor suppressor gene and a negative regulator of cell invasion that is frequently methylated in cancer cells (20). miR-221/222-expressing cells also have an advantage in cell proliferation that could be attributed to the direct suppression of p27kip1 (196). In addition, miR-21 plays an antiapoptotic role in

this context, where it is induced by activated EGFR (542), which itself is induced as a result of a reduction in miR-145 (107). In contrast to the above listed antiapoptotic miRNAs, p53 induces apoptosis via upregulation of miR-34a, following its reactivation by PRIMA-1 (150) [a small molecule that restores mutant p53's transcriptional activity and apoptotic effects in lung cancer cell lines (64)].

In addition to regulating proliferation and viability of lung cancer cells, miRNAs are also involved in migration and metastasis (FIGURE 7). One of the established mechanisms in cancer cell metastasis is the EMT of cells, which is associated with upregulation of vimentin and downregulation of cell border uvomorulin, thus increasing cell motility and invasiveness (21). EMT could be stimulated by TGF-β, TNF-α, or HGF, which induce upregulation of the zinc-finger E-box binding homeobox (ZEB) protein that suppresses E-cadherin. The miR-200 family plays a major role in metastasis of lung, colon, and breast cancers. Using K-Ras/p53 double mutant cells that develop metastatic lung adenocarcinoma, Gibbons et al. (205) found that EMT required downregulation of the miR-200 family. In specific, miR-141 and miR-200c were >90% reduced in metastasized cells relative to the tumor of origin. Consequently, overexpression of miR-200b dramatically reduced TGF-β-induced EMT via suppression of its target ZEB1, but not ZEB2, and rescued the downregulation of E-cadherin. In contrast, stimulation of lung cancer cells with the tumor-promoting EGF did not significantly reduce miR-200c but did induce ~50% downregulation of miR-125–5p, as well as several members of the let-7 family. Furthermore, ectopic expression of miR-125–5p inhibited cell migration and invasion, but its targets here remain unidentified (645). Another miRNA that is involved in metastasis of lung cancer is miR-183. It is preferentially underexpressed in metastatic versus nonmetastatic lung cancer cells of the same origin and targeted the cytoskeletal protein Ezrin (644), which plays an established role in cell invasion and metastasis.

went apoptosis and were replaced by normal cells within 6–12 wk. Interestingly, though, the few that did survive contributed to high frequency of hepatocellular carcinoma in 6-mo-old mice.

Because the phenotype of the Dicer knockout mice was gradually reversed, the effect of miRNAs, including miR-122, on the adult liver function was not fully evaluated. Another study, using antisense knockdown of miR-122 in the adult liver, confirms its essential role in lipid metabolism (165). This was demonstrated by lower plasma cholesterol (26–28% lower) and triglyceride levels. Indeed, ex vivo experiments proved that the rate of sterol and fatty acid synthesis was decreased, while fatty acid oxidation was increased approximately twofold. This phenotype could be partly explained by a reduction in acetyl-coenzyme A carboxylase 2 (ACC2) and stearoyl-coenzyme A desaturase 1 (SCD1), and an increase in phosphorylated AMP-activated kinase alpha 1 subunit (AMPKa1), although miR-122 direct targets that elicit these effects were not identified in this study. In a third study, Elmen et al. (162) showed that antisense knockdown of miR-122 in vivo induced upregulation of some of its predicted targets that include aldolase A, branched-chain α-ketoacid dehydrogenase kinase, Cd320 antigen, putative very-low-density lipoprotein receptor, and N-myc downstream regulated gene 3 (162). In addition, miR-122 is also responsible for almost complete suppression of the cationic amino acid transporter in the adult liver (84).

B. miRNAs in Hepatitis

Hepatitis C virus (HCV) replicates specifically in hepatocytes, indicating the requirement for liver-specific factors. With the advent of miRNAs, it was also noted that HCV only replicates in the hepatoma cell lines that have detectable miR-122 (Huh7) but not those that are deficient in this miRNA (HepG2) (284). Depletion of miR-122 in Huh7 cells reduced the viral RNA replicon by ~80%, while increasing miR-122 increased its production, suggesting a role for miR-122 in HCV replication. Moreover, ectopic expression of miR-122 in human embryonic kidney HEK-293 cells facilitated HCV replication (83). Interestingly, two putative miR-122 targeting sites were discovered in the RNA genome of HCV; two in the 5′ noncoding region (NCR) between the stem-loops I and II and one in the 3′ NCR, which were highly conserved among six HCV genotypes (244, 284). Mutation analysis revealed that only the former was necessary for miR-122-dependent viral replication, which appeared to require direct interaction with miR-122. This interaction, however, had no effect on the stability of the viral RNA, whereas reports on its role in gene translation are controversial (244, 284). While Jopling et al. (284) noted no effect on viral translation, Henke et al. (244) report that miR-122 enhances HCV replicon translation by stimulating the association of the small ribosomal subunit

IX. MicroRNAs IN THE LIVER

A. miRNAs in Liver Development and Function

Consistent with a more subtle regulatory function, Dicer deletion did not affect early liver development even though the liver-specific miR-122 was almost completely lost for up to 3 wk postnatally (544). However, 3-wk-old livers were paler than normal and exhibited an increase in proliferation and apoptosis. In addition, cells had more lipid droplets but lacked glycogen granules, indicative of metabolic abnormalities. At the molecular levels, markers of terminally differentiated hepatocytes were normal in Dicer-deficient cells, but were accompanied by reiteration of the fetal gene program. Eventually, most Dicer-negative hepatocytes under-
with the viral replicon at the initiation stages. Consistent with the latter observation is the fact that the 5′NCR regulates translation through stem-loop II-IV, which is the internal ribosome entry site (563) and where proximal binding of miR-122 has been shown to induce an “open” conformation that may favor translation (142). These findings suggest that miR-122 could serve as a major antiviral therapeutic target for treatment of hepatitis C. Indeed, a recent study shows that antisense antagonism of miR-122 in primates resulted in efficient reduction in serum (2.6 logs) and liver (2.3 logs) HCV, with no evidence of viral resistance or side effects (331). In addition to downregulation of miR-122, other antiviral miRNAs are induced by IFN-β in Huh7 cells that include miR-1, miR-30, miR-128, miR-196, miR-296, miR-351, miR-431, and miR-448, all with HCV-complementary seed sequences that could contribute to a robust anti-HCV effect (478). Notably, though, in chronic hepatitis C (HC) patients, miR-122 levels did not correlate with HCV RNA levels and did not change in response to interferon treatment (529). However, the status of miR-122 in acute cases of HC remains to be examined.

### C. miRNAs in Hepatocellular Carcinoma

Downregulation of the liver-enriched miR-122 has been a consistent finding in both rodent and human hepatocellular carcinoma (HCC). This finding was observed in ~50% of primary human tumors (324) as it appeared to correlate with poor prognosis, larger tumor size, c-Met positivity, and loss of the hepatocyte differentiation (123). Moreover, overexpression of miR-122 suppressed HCC growth and migration both in vitro and in vivo, and vice versa (23, 123). Some of the targets that are likely to mediate its effects and whose expression inversely correlate with that of miR-122 in tumors include disintegrin and metalloprotease 17 (ADAM17) (621), serum response factor (SRF), and IGF-IR (23). Another miR-122 validated target that is significant in hepatocellular carcinogenesis is cyclin G1 (212). In accordance, overexpression of miR-122 in hepatocellular carcinoma cells suppressed cyclin G1 and enhanced phosphorylation of p53 (183). Indeed, cyclin G1 null mice have a higher content of p53 and lower incidence of hepatic tumors (273). Moreover, overexpression of miR-122 or knockdown of cyclin G1 enhanced E-cadherin expression and reduced cell invasion and increased apoptosis. However, whether cyclin G1 is necessary or sufficient for mediating miR-122’s effects remains unexamined. In addition, a proapoptotic function of miR-122 is possibly mediated by its direct targeting of Bcl-w (a Bcl-2 family member) (368).

Other growth suppressor miRNAs include miR-195, which is depressed in HCC and where its replenishment inhibits growth, possibly by targeting cyclin D1, Cdk6, and E2F3 (681); and miR-1, by targeting FoxP1 and HDAC4 (132). Downregulated miRNAs in HCC that were shown to specifically inhibit cell invasion and migration include miR-34a

(357), miR-23b (523), and miR-1 (132), which all target c-Met; and miR-101, which inhibits AP-1-dependent transcription through targeting v-fos (360). In addition, other miRNAs were identified as mediators of apoptosis and are also reduced in HCC, these include miR-101, which targets the antiapoptotic protein Mcl-1 (583), and miR-233, a suppressor of stathmin (667), a microtubule-stabilizing and tumor-promoting protein (129, 204, 429). Considering the preponderance of miRNAs that are downregulated in HCC, Kato et al. (315) examined the efficacy of miRNA replacement therapy for its treatment. For that purpose, they selected miR-26a, which they found downregulated in a mouse model of Myc-induced HCC, as well as human HCC samples. Another criterion they sought was a miRNA, normally highly expressed in all organs, that was reduced only in the tumor, which they predicted would be generally well tolerated if modestly overexpressed. In this case, they delivered miR-26a intravenously via an adenoassociated virus. miR-26a, which they found to target cyclins D2 and E2, impressively reduced tumor burden in the liver. This effect was a result of reduced proliferation and increased apoptosis of the tumor cell.

Furthermore, PPARα plays a role in HCC by activating a miRNA-dependent pathway that is initiated by downregulation of the tumor suppressor let-7 and an increase in its target c-myc (550). This, in turn, induces upregulation of the oncogenic miR-17~92 cluster (550). The latter was found upregulated in 100% of human HCC and is involved in promoting proliferation and anchorage-independent growth of cancer cells (115). Although the targets of miR-17~92 have not been specifically identified in HCC, that of its paralog, miR-106b~25, were further investigated. Data show that in HCC miR-25 targets the proapoptotic protein Bim, while miR-106b targets E2F1 (366). It was predicted that suppression of E2F1 might be necessary for regulating any excessive increase of the protein that may elicit an adverse apoptotic function. Other miRNAs with oncogenic functions (oncomirs) in HCC include the following: miR-21, which suppresses PTEN and is associated with an increase in phospho-AKT, phospho-focal adhesion kinase (p-FAK), and metalloproteinase 2 (MMP2) (417); miR-221, which inhibits the cell cycle inhibitors p27 and p57 (183), and BH3-only protein Bmf (213); miR-155, which targets c/EBPβ (643); and miR-145, which enhances cell invasion by targeting fibronectin type III domain containing 3B (709).

### X. MicroRNAs IN THE PANCREAS

#### A. miRNAs in Pancreas Development and Function

Conditional knockout of Dicer in the pancreas uncovers the requirement of miRNAs during its development in mice
(396). The development of all the endocrine cell lineages, especially β-cells, was dramatically impaired in this model, which was lethal before P3. The phenotype was mostly due to a reduction in progenitor cells as a result of an increase in the notch-signaling target Hes1 and cellular apoptosis rather than a reduction in cell proliferation, as evidenced by reduced neurogenin3-positive progenitors. This is also supported by the fact that postdevelopment β-cell-specific deletion of Dicer had no effect on β-cell proliferation or maintenance. On the other hand, a generalized Dicer-hypomorphic knockout model with only ~20% residual dicer in all tissues, displayed abnormalities in the adult pancreas in the form of irregular distribution of islet cells, an increase in the number of ductal insulin+/glucagon+/Pdx-1 epithelial cells, and abnormal multinucleated cells, which argued that miRNAs might be regulating progenitor cell proliferation and differentiation in this organ (431).

Cloning of miRNAs from pancreatic islet cells revealed the expression of the tissue-enriched miR-375 (492). In vivo promoter expression studies revealed that it is specifically transcribed in α-, β-, and δ-islet cells of the adult pancreas via the helix-loop-helix Id2 and Id3 factors (19). miR-375's function proved inhibitory to glucose-induced insulin secretion by suppressing exocytosis, but otherwise had no effect on insulin synthesis, Ca²⁺ signaling, availability of granules, or submembrane actin network (492). One of its identified targets includes V1/myotrophin, whose knockdown mimicked miR-375's effect in β-cells. A second study, while confirming that miR-375 inhibits insulin secretion, shows that it does so through PDK1-dependent inhibition of insulin synthesis (156). Accordingly, glucose enhances insulin expression via inducing downregulation of miR-375 and upregulation of its target PDK1. However, glucose's effect is not limited to miR-375; it also increases the expression of several miRNAs including miR-30d, which plays a role in enhancing the expression of insulin in a β-cell line (597). Conversely, miR-9, a brain-enriched miRNA, is expressed in islet cells and inhibits glucose-induced insulin secretion through suppression of One-cut-2, which, in turn, enhances the expression of the negative regulator granuphilin/Slp4 (488). Thus synthesis and secretion of insulin are a function of at least three miRNA (miR-375, miR-30d, and miR-9) that might operate as a molecular switch during hyperglycemia.

Targeted deletion of miR-375 in a mouse model showed that it is also required for regulation of β to α cell ratio during development (493). This phenotype was evident at 3 wk of age, when the pancreas was found to have 38% less β-cells versus 70% more α-cells. The increase in α-cells was responsible for an increase in plasma glucagon, hepatic gluconeogenesis, and hyperglycemia in these mice. In contrast, the obese mouse model exhibits an increase in β-cells due to insulin resistance, which is associated with a 30% increase in miR-375. Cross-breeding the two models resulted in decreasing the levels of β-cells in the obese mouse, reducing insulin production, and exacerbating the diabetic phenotype. At least 10 of miR-375's targets were upregulated in pancreatic islets from the knockout model. Those included the following: caveolin1 (Cav1), inhibitor of DNA binding 3 (Id3), SWI/SNF related, matrix associated, actin dependent regulator of chromatin, subfamily a, member 2 (Smarca2), Ras-dexamethasone-induced-1 (Rasd1), regulator of G protein signaling 16 (Rgs16), eukaryotic elongation factor 1 epsilon 1 (Eef1e1), apoptosis-inducing factor, mitochondrion-associated 1 (Aifm1), cell adhesion molecule 1 (Cadm1), HuD antigen (HuD), and complement component 1 q subcomponent binding protein (C1qbp). This set of genes is known to negatively regulate cell growth and proliferation that partly explains the reduction in β-cells. In addition to miR-375, other islet-enriched miRNAs include miR-7 (59, 119, 278), whose function remains to be explored.

### B. miRNAs in Pancreatic Cancer

Expression profiling of miRNAs in pancreatic cancer (PaCa) versus chronic pancreatitis identified upregulation of 15 and downregulation of 8 miRNAs that accurately classified 93% of the samples (47). Moreover, increased expression of miR-196a and miR-219 emerged as predictors of poor survival rate. miR-196a/b's role was further emphasized as it was strictly detected in cancer tissue and cell lines but not in normal tissue or in pancreatitis (591). Other notable miRNAs that were identified in this screen included miR-21, miR-221, and miR-155 that were upregulated, and miR-375, which was downregulated (47). miR-21 is upregulated in ~79% of pancreatic tumors, and its levels are predictive of survival in patients that are negative for lymph node metastasis; however, it does not correlate with tumor size, differentiation, or stage (143). Knocking down miR-21 in PaCa cells reduced proliferation and invasion and sensitized them to chemotherapeutic agents as two of its targets, PTEN and reversion-inducing cysteine rich protein with Kazal motifs (RECK), were upregulated (432, 471). A similar effect is exerted by miR-221 via targeting the cell cycle inhibitor p27kip1 (471). In addition, miR-21 and miR-155 are two of the most highly expressed in early noninvasive intraductal papillary mucinous neoplasms of the pancreas (228), the latter targeting tumor protein p53-induced target gene (TP53INP1) and inhibiting p53-induced apoptosis (207).

Consistent with other forms of cancer, the miR-200 family of miRNAs is downregulated, promoting EMT and cell invasion (63, 659). A double negative feedback loop is created as ZEB1 directly suppresses the expression of miR-200c and miR-141, while miR-200c represses ZEB1 expression and miR-141 represses TGFβ2, an inducer of EMT. In addition to the enhanced invasiveness associated with EMT, recent studies show that cells that undergo EMT also

acquire stem cell markers (403). Accordingly, ZEB1-deficient tumors were not only impaired in their metastatic capacity, but were also smaller in size, which supports the idea of reduced numbers of cancer stem cells (659). In agreement, ZEB1 knockdown in MiaPaCa2 reduced the number of stem cells and their sphere-forming capacity and resistance to chemotherapy. This was confirmed by lower expression of stem cell factors including Sox2, bmi1, and p63. In addition to the miR-200 cluster, ZEB1 was shown to directly suppress miR-203 and had putative binding sites in miR-183's promoter region. Indeed, overexpression of either in PaCa cells reduced the number of cancer stem cells. miR-200c, miR-203, and miR-183 all targeted Bmi1, a protein known for its function in stem cell renewal (470), while miR-200c additionally targets Sox2 and klf4. Furthermore, overexpression of any these miRNAs reproduced the effects of ZEB1 knockdown in cancer cells. Similarly, miR-34a suppresses cancer stem cell proliferation and is usually downregulated, deleted (85), or methylated in PaCa (380). Overexpression of miR-34 in a pancreatic cancer cell line reduced the number of cancer stem cells and, thus, its capacity for tumor formation in nude mice (274). This was associated with downregulation of its targets Bcl-2, notch-1, and notch-2 proteins, apoptosis, cell cycle arrest, and reduced invasion (274). Thus these results reveal an intricate network between ZEB1, miR-200, miR-203, miR-183, and miR-34a, that promotes EMT and stemness in PaCa. However, EMT is not the only mechanism that promotes metastasis, as miR-10a was found to promote metastasis through suppression of HOXB1 and HOXB3 (658).

XI. MicroRNAs IN THE SKIN

A. miRNAs in Skin Morphogenesis

Like other major organs, the skin harbors at least one predominant, tissue-enriched miRNA, namely, miR-203, which is highly conserved and specifically enriched in keratinocyte (571). miR-203 is upregulated in differentiating keratinocytes (348, 691) and is, thus, enriched in the suprabasal epithelial cells and in differentiated epidermal cells, while being absent in the proliferating basal layer in both mouse (348) and human (657) skin. Ectopic expression of miR-203 in the basal layer of stratified epithelium results in flattened basal cells and only a single layer of suprabasal cells and, thus, thin skin, attributed to premature differentiation and depletion of the basal progenitor cells (348). In accordance, these mice die shortly after birth from apparent dehydration. Conversely, knockdown of miR-203 using locally applied antisense constructs, delayed cell cycle exit of the suprabasal epithelial cells. In agreement, tissue-specific knockout of Dicer1 led to thickening of the epidermis. The levels of miR-203 inversely correlated with that of the transcription factor p63, which is known for maintaining “stemness” of the progenitor cells (547). Indeed, p63 proved to be a direct target of miR-203 (348,

691). In accordance, the phenotype of the miR-203 transgenic mouse may be ascribed to suppression of p63, as it recapitulates the phenotype of the p63 null mice (547). The data suggest that miR-203 regulates the timing of cell cycle exit and differentiation of skin stem cells.

B. miRNAs in Psoriasis

Psoriasis is a chronic inflammatory skin disease in which both the keratinocyte and infiltrating immune cells are involved in the formation of erythematous scaling plaques (336). miR-203 and miR-146a are specifically upregulated in cases of psoriasis, but not in eczema (571). This is in contrast to miR-21, which is upregulated in both lesions. Conversely, miR-125b is downregulated in both. Moreover, the cell-type distribution of these molecules is significantly different. Whereas miR-203 is restricted to keratinocytes, miR-146a is virtually undetectable in these cells or in dermal fibroblasts. On the other hand, miR-146a is relatively high in the infiltrating immune cells, consistent with its function in immune reactions, while miR-21 is generally ubiquitous.

In psoriasis, keratinocytes’ proliferation is enhanced as their differentiation is retarded (382), rendering it hard to reconcile the function of miR-203 in psoriasis with its role in skin morphogenesis. The data suggest that miR-203 may have different targets in normal versus psoriatic keratinocytes, where it exists in supernormal levels. Surely, only in psoriasis, miR-203 targets SOCS-3, which is a negative regulator in cytokine signaling pathways and keratinocyte proliferation (372) and is downregulated in the psoriatic lesions (571).

C. miRNAs in Malignant Melanoma

miRNA genes are frequently amplified or deleted in malignant melanoma (MM), breast cancer, and ovarian cancer (705). Among those frequently amplified in melanoma include miR-182 and miR-15/16, and those deleted include Let-7a/b. High levels of miR-15b were shown to correlate with poor recurrence-free and overall survival, possibly through promoting tumor survival (530). Meanwhile, miR-182 is found in the chromosomal region 7q31–34 that is frequently amplified and is, accordingly, upregulated in MM samples (541). In particular, its expression correlates with metastasis of human MM. It was also proven to enhance migration and survival of melanoma cells both in vitro and in a mouse model through targeting microphthalmia-associated transcription factor-M (MITF-M) and FOXO3. MITF is a regulator of melanocyte growth and differentiation and is frequently downregulated in melanomas (546). It is cleaved by caspases, rendering the COOH-terminal cleavage product proapoptotic (332). Promyelocytic leukemia zinc finger (PLZF) is another transcription

factor that is reduced in MM, unblocking the transcription of miR-221/222 (174). This miRNA suppresses p27kip1, c-kit receptor, and its downstream effector MITF, thus enhancing cell proliferation and promoting dedifferentiation. In contrast to the aforementioned tumor-promoting miRNAs, Let-7a is a tumor suppressor that is frequently downregulated in cancer (705). Let-7a inhibits MM cell migration, which could be explained by its suppression of β3 integrin (435), while Let-7b inhibits cell proliferation via targeting cycD1, D3, and Cdk4 (538). The former is also a target of miR-193b, which is similarly downregulated in MM (90).

**XII. MicroRNAs IN COLON CANCER**

Sequencing of miRNAs in colorectal cancer (CRC) revealed consistent downregulation of mature miR-145/143 but not their stem-loop precursor, indicating a posttranscriptional mechanism in the regulation of this cluster (424) [FIGURE 8]. miR-145/143, which are widely expressed in all normal tissues examined, were virtually undetectable in cancer cell lines, including colon, prostate, neuroblastoma, cervical, hepatic, and leukemia (8). Ectopically expressing either miRNA in colon cancer cell lines inhibited their growth (8, 554) and was associated with downregulation of miR-143's predicted target, Erk5 (8). miR-145 has been confirmed to target insulin receptor substrate 1 (IRS-1) (554). This adaptor protein mediates signaling of insulin and IGF receptors to their downstream AKT and Erk1/2 signaling pathways, while polymorphism of its gene is associated with a 70% increased risk of colon cancer (566). Thus, derepression of IRS-1 by downregulation of miR-145 is one mechanism that enhances the expression of IRS-1 in cancer cells. Furthermore, miR-145 targets IGF-1 receptor, which emphasizes its role in regulating this pathway (325). However, while a miR-145-resistant IRS-1 gene was sufficient for re-

versing miR-145-induced growth inhibition of colon cancer cells, IGF-1R was not. The results are supported by the fact that miR-145 failed to inhibit the growth of a mammary cancer cell line that does not express IRS-1 (325). miR-143, on the other hand, was shown to target the protooncogene K-Ras (95) and DNA methyltransferase 3A (DNMT3A) (449), which is linked to hypermethylation and silencing of genes by the polycomb groups during carcinogenesis (276). K-Ras is also a validated target of let-7, which is also downregulated in 30% of CRC cases (7). It should be noted here that another study has identified miR-145 as an oncomiR rather than a tumor suppressor in metastatic colon cancer cells (16). However, this finding is hard to reconcile with the fact that miR-145 is consistently downregulated in primary tumors and most cell lines and that its levels inversely correlate with tumor size (565).

Gene silencing by CpG island hypermethylation is a common feature of cancer cells and is mainly achieved by the DNA methyltransferase (DNMT) family of proteins that are upregulated in cancer cells (reviewed in Ref. 158). Knockout of DNMT1 and DNMT3b in colon cancer HCT-116 cell line results in the upregulation of 18 miRNAs, 5 of which are embedded in CpG islands (393). In particular, the CpG islands harboring the three miR-124a genes are selectively hypermethylated in the cancer cells. Cdk6 was identified as a target of miR-124 in CRC, where its expression positively correlated with DNA hypermethylation and negatively with miR-124a levels. Another gene that is silenced in CRC tumors and cell lines by GpC hypermethylation is miR-34a/b/c (380, 617). Notably, transcription of the miR-34a gene is also dependent on p53 and is accordingly depressed in cancer cells lacking p53 (85, 118, 239, 503, 599). Moreover, a p53 enhancer that is in close proximity to the 5'-end of the miR-34a primary transcript is located in a CpG island (503). In support, hypermethylation of the gene

---

### miRNAs in colorectal cancer

| **tumorigenesis** | **survival** | **angiogenesis** | **invasion/metastasis** |
|-------------------|--------------|------------------|--------------------------|
| ↑ miR-17~92 → ↓ E2F1 | ↑ miR-21 → ↓ PDCD4 | ↑ miR-21 → ↓ SPRY2 |
| ↓ Let-7 → ↑ K-Ras | ↓ miR-145 → ↑ EGFR, IRS-1 | ↓ miR-126 → ↑ P85β | ↓ miR-200c → ↑ ZEB1 |
| ↓ miR-143 → ↑ DNMT3A, Erk5 | ↓ miR-34 → ↑ Sirt1 | ↓ miR-141 → ↑ TGFβ2 |
| ↓ miR-34 → ↑ Cdk4, E2F3, SFRS2 | ↓ miR-126 → ↑ p85β | ↓ miR-34 → ↑ c-Met |
| ↓ miR-101 → ↑ Cox-2 | ↓ miR-192 → ↑ DHFR |

---

**FIGURE 8** A diagram showing miRNAs and their targets in colorectal carcinoma. The diagram displays the different miRNAs and their targets that are involved in transformation, survival, angiogenesis, and invasion/metastasis of colorectal cancer. Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of a target gene inversely correlate with that of the targeting miRNA and are similarly represented by an up or down arrow. All listed targets have been validated. These include the following: E2F transcription factor 1 (E2F1), K-Ras, DNA methyltransferase 3A (DNMT3A), mitogen-activated protein kinase 7 (Erk5), cyclin-dependent kinase 4 (Cdk4), E2F transcription factor 3 (E2F3), splicing factor, arginine/serine-rich 2 (SFRS2), Cox-2, cyclin-dependent kinase 6 (Cdk6), epidermal growth factor receptor (EGFR), insulin receptor substrate 1 (IRS-1), Sirtuin 1 (Sirt1), phosphoinositide-3-kinase, regulatory subunit 2 (beta) (p85β), dihydrofolate reductase (DHFR), zinc finger E-box binding homeobox 1 (ZEB1), transforming growth factor beta 2 (TGFβ2), and hepatocyte growth factor receptor (c-Met).

abrogated its activation by p53 (380). Thus hypermethylation and loss of p53 in cancer cell ensure silencing of miR-34, which underscores its tumor suppressive function. Some of its predicted targets that have been validated in CRC include Cdk 4, c-MET, splicing factor, arginine/serine-rich 2 (SFRS2) (617), and E2F3 (603). Also, by targeting and suppressing Sirt1, it increases acetylation and activation of p53, and thereby induces apoptosis in p53-positive CRC cells (683). In accordance, overexpression of miR-34 inhibits tumor growth in vivo (603). Likewise, a p53-binding site was identified in the promoter of miR-192 (569). Similar to miR-34, its expression positively correlates with that of p53, and it inhibits cell proliferation in a p53-dependent fashion, albeit to a lesser extent than that offered by miR-34 (58, 569). It was also shown to increase sensitivity to chemotherapeutic agent, plausibly by targeting dihydrofolate reductase (DHFR) (569).

Expression profiling of tumors identified 37 miRNAs that are differentially expressed in CRC (536). Specifically, high levels of miR-21 correlated with poor survival and resistance to therapy. miR-21 was also the only miRNA that was consistently upregulated in all solid tumors examined (639). Among the miR-21 targets that were validated in CRC is the tumor suppressor programmed cell death 4 (PDCD4) (18). This protein is known to interact with eIF4A and eIF4G and inhibit protein translation (687, 701), a mechanism which may explain its inhibition of neoplastic transformation (112). In addition, knockdown of miR-21 reduces invasion, extravasation, and metastasis of CRC cells (18), plausibly via enhancing the expression of its target Sprouty 2 (534), a known inhibitor of cell migration (337, 693) (703). Another mechanism that initiates cancer cell metastasis is EMT. It is induced by TGF-β, TNF-α, or HGF and is associated with an increase of ZEB, which suppresses the expression of basement membrane proteins (574) and, thus, enhances metastasis in CRC (575). ZEB1 also suppresses the transcription of four miRNAs in CRC and breast cancer cells lines that include three members of the miR-200 family (miR-141, -200b, -200c) (63). Conversely, miR-141 and miR-200c inhibit EMT by directly targeting TGF-β2 and ZEB1 and function as a switch for EMT [FIGURE 8].

Finally, it is worth noting that cyclooxygenase-2 (Cox-2), which is expressed in 86% of colorectal adenocarcinomas and 43% of adenomas (151) and plays a critical role in tumor formation (553), is also subject to regulation by miRNAs. Specifically, miR-101 is downregulated in CRC and directly targets and inhibits Cox-2 translation (582). Additionally, downregulation of miR-126 may indirectly participate in the regulation of Cox-2 via increasing the expression of p85β subunit of the PI3-kinase and enhancing the activity of AKT (222), a positive regulator of Cox-2 (551). On the other hand, the role of p85β in angiogenesis (700) suggests

that downregulation of miR-126 in CRC may also participate in that process [FIGURE 8].

### XIII. MicroRNAs IN BREAST CANCER

#### 1. Tumor initiation (Figure 9)

Investigations of the tumorigenic potentially of mammary cancer cells led to the finding that only a minor subpopulation, with stem cell characteristic, had tumor initiating capacity (9). These breast cancer stem cells (BCSC) are characterized by a CD44⁺CD24⁻/lineage⁻ phenotype that is utilized as a selection marker. miRNAs profiling of this subpopulation revealed reduced levels of let-7, which was gradually reversed upon their adherence and differentiation in culture (695). Accordingly, let-7 proved to be a negative regulator of the self-renewal, tumorigenic, and metastatic capacity of BCSC. Its expression inversely correlates with the expression of two of its targets, Ras and HMG2A. Knockdown of either protein separately revealed that Ras was responsible for enhancing cell proliferation, while HMGA2 suppressed differentiation of BCSC. In this context let-7 is regulated by an inflammatory pathway that is involved in epigenetic switching of cells into a cancerous phenotype (263). This pathway is triggered by Src, which, via activation of NFκB, induces the expression of Lin-28, an inhibitor of pri-let-7 processing (448). Subsequently, as lower levels of let-7 enhance the expression of IL-6, which also activates NFκB, the pathway becomes perpetually activated. Other miRNAs that were discovered by differential profiling of BCSC included the following: miR-200c-miR-141, miR-200b-miR-200a-miR-429, and miR-183-miR-92-miR-1182, which were downregulated in both BCSC and embryonic stem cell carcinoma (559). This group of miRNAs was also lower in normal breast stem cells relative to the more differentiated progenitor cells. miR-200c/miR-200b/miR429, which all have the same seed sequence, target and suppress Bim1, a protein known for its function in stem cell renewal (470). Thus, as predicted, overexpression of miR-200c inhibited colony formation by BCSC, through suppression of Bim1, and completely inhibited their tumorigenic capacity in vivo (559).

#### 2. Tumor growth (Figure 9)

Early studies showing the differential expression pattern of miRNAs in breast cancer tissue revealed that some of the more significantly upregulated included miR-21 and miR-155, and those downregulated included miR-125b and miR-145 (267). Consistent with a tumor-promoting function, miR-21 was found to target and inhibit TPM1 (716), PDCD4 (184, 391), and maspin (717). While PDCD and TPM1 are generally known for their tumor suppressive properties, maspin has been implicated in the inhibition of tumor invasion by inducing upregulation of α3 and α5 integrins in breast cancer cells (540). In agreement, ectopic

MicroRNAs IN DEVELOPMENT AND DISEASE

miRNAs in tumor initiation and growth of breast cancer

src  
↓  
NFκB  
↓  
↑ Lin-28  
↓  
↓ Let-7  
↓  
↑ IL-6 ↑ HMGA2 ↑ Ras  

Epigenetic switching  
and BCSC self-renewal  

miR-200c/200b/429  
↓  
↑ Bmi1  

miR-21  
↓  
↓ PDCD4 ↓ TPM1  

miR-17-20  
↓  
↑ CycD1 ↑ AIB1  

miR-125  
↓  
↑ HuR ↑ ErbB2 ↑ ErbB3 ↑ BMPR1  
↓  
↓ miR-205  

↑ miR-27a  
↓  
ZBTB10 ↓ Myt1 ↓ Foxo1 ← ↑ miR-182  
↓  
↑ Sp1/2/1 ↑ Cdc2 ↑ miR-96  
↓  
↑ survivin  

tumorigenesis  

FIGURE 9 A diagram showing miRNAs and their targets in initiation and growth of breast cancer. The diagram displays the different miRNAs and their targets that are involved in breast cancer stem cells (BCSC) self-renewal and tumor growth. This also includes some of the identified upstream molecules that regulate their expression. Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of a target gene inversely correlate with that of the targeting miRNA and are similarly represented by an up or down arrow. All listed targets have been validated. The targets listed include the following: v-src sarcoma (Schmidt-Ruppin A-2) viral oncogene homolog (src), nuclear factor of kappa light polypeptide gene enhancer in B-cells 1 (NFκB), interleukin-6 (IL-6), high mobility group AT-hook 2 (HMGA2), Ras oncogene, signal transducer and activator of transcription 3 (STAT3), programmed cell death 4 (PDCD4), tropomyosin 1 (TPM1), zinc finger and BTB domain containing 10 (ZBTB10), myelin transcription factor 1 (Myt1), forkhead box O1 (Foxo1), Sp1 transcription factor, cell division cycle 2 protein (Cdc2), Hu antigen R (HuR), v-erb-b2 erythroblastic leukemia viral oncogene homolog 2 (ErbB2) and homolog 3 (ErbB3), bone morphogenetic protein receptor type 1B (BMPR1B), cyclin D1 (CycD1), and amplified in breast cancer 1 protein (AIB1).

expression of miR-21 enhanced breast cancer cell growth both in vitro and in vivo (184, 391, 560) and HER2-mediated cell invasion (251), and promoted metastasis (717). Conversely, its knockdown in an invasive breast cancer line inhibited metastasis, although it had no effect on tumor growth (717). In addition to its pro-growth and -metastasis effects, miR-21 also indirectly induces upregulation of Bcl-2 and, thus, exerts an antiapoptotic function (560). In concordance, miR-21 is one of the few miRNAs that correlates with advanced clinical stage, lymph node metastasis, and poor survival in breast cancer patients (685). In addition, miR-27a is another oncomir that is highly expressed in breast cancer cells (226, 421) and targets the zinc finger protein ZBTB10 (421) and the proapoptotic protein Foxo1 (226). Via suppressing ZBTB10, it induces upregulation of the transcriptional enhancers Sp1, 3, 4, which, in turn, increase expression of the antiapoptotic protein survivin (421). Moreover, by targeting the cell cycle inhibitor

Myt-1 (421), miR-27a may promote tumor growth. These effects may be enhanced by the upregulation of miR-96 and miR-182, which also target Foxo1 (226).

In contrast to the upregulation of miR-21 and miR-27 and their oncogenic functions, the miR-17~20 cluster (247, 698) and miR-125a (224) are reduced in breast cancer. miR-17~20 cluster inversely correlates with the expression of its target cyclin D1 (698) and amplified in breast cancer 1 (AIB1) (247) and, thereby, inhibits estrogen-dependent and -independent cell proliferation of breast cancer cells (247, 698). Meanwhile, miR-125a inhibits cell proliferation and migration via targeting the RNA-binding protein HuR (224), and ErbB2 and ErbB3 family members of the EGFR (539). Moreover, the discovery of a germ line mutation in miR-125a (362) and a SNP in the miR-125b-targeted site of the 3'-UTR of bone morphogenetic receptor type 1B

DANISH SAYED AND MAHA ABDELLATIF

(BMPR1B) (522) that are associated with breast cancer, confirms its role in the underlying pathogenesis.

3. Metastasis (Figure 10)

EMT of cancer cells is one of the mechanisms that increases the cell's motility and invasiveness and is characterized by loss of proteins involved in cell-to-cell attachments (21).

E-cadherin is one of these proteins that is suppressed by the transcriptional regulators snail, zinc-finger E-box-binding homeobox 1 (ZEB1/TCF8/δEF1), and ZEB2/SIP1; however, the mechanism of their regulation is not fully understood yet (reviewed in Ref. 480). miRNAs provide one mode of regulation for these critical factors as seen in many cancers. Downregulation of miR-200c is implicated in EMT and cell invasion by releasing ZEB1 from its inhibitory effect and,

miRNAs in breast cancer cell invasion and metastasis

![Diagram](#)

FIGURE 10 A diagram showing miRNAs and their targets in breast cancer metastasis. The diagram displays the different miRNAs and their targets that are involved in breast cancer cell invasion and metastasis. This also includes some of the identified upstream molecules that regulate their expression and the convergence of multiple miRNAs on endothelial-to-mesenchymal transition (EMT). Upregulation or downregulation of a specific miRNA is represented by an upward (black) or a downward (red) arrow, respectively. The changes in the expression levels of a target gene inversely correlate with that of the targeting miRNA and are similarly represented by an up or down arrow. All listed targets have been validated. The targets listed include the following: Raf oncogene, mitogen-activated protein kinase (MAPK), myelocytomatosis oncogene (myc), high mobility group AT-hook 2 (HMGA2), E-cadherin, ZEB1/2, TTP, TGFB1, ras homolog gene family, member A/B (RhoA/C), homeo box D10 (HoxD10), breast cancer metastasis-suppressor 1 (BRMS1), plasminogen activator, urokinase (uPA), TNF-receptor-associated factor 6 (TRAF6), interleukin-1 receptor-associated kinase 1 (IRAK), epidermal growth factor receptor (EGFR), nuclear factor of kappa light polypeptide gene enhancer in B-cells 1 (NFkB), mucin 1 (Muc-1), tropomyosin 1 (TPM1), programmed cell death 4 (PDCD4), maspin, CD44 antigen, metastasis associated 1 (MTA1), SRY-box containing gene 4 (Sox4), Tenascin C, vascular endothelial growth factor (VEGF), and v-erb-b2 erythroblastic leukemia viral oncogene homolog 3 (ErbB3).

thus, suppressing the expression of E-cadherin in breast
cancer cells (256). Indeed, overexpression of miR-200c induces differentiation of breast cancer cells into epithelial
cells and reduces cell migration and invasion equivalent to
the effect seen on knockdown of ZEB1 (63). Meanwhile,
miR-141, another miR-200 family member, targets TGF-
β2, which is an inducer of EMT. Conversely, ZEB1 directly
suppresses the promoter of miR-141/-200b/-200c (63), creating a double negative feedback loop. Upon closer examination, it was discovered that miR-200a/141 and miR-
200b/200c/429 (each group having common seed sequences) target both ZEB1 and ZEB2 through multiple
target sites for each within their 3′-UTR (215). As expected,
miR-200 family members are specifically lost in metastatic
breast cancer cells that lack E-cadherin. Thus the miR-200
family is involved in both cancer initiation and invasion.

To further identify genes that are involved in breast
cancer metastasis, a differential profiling study was performed on a human breast cancer cell line and its highly
metastatic derivatives (602). This uncovered two miRNAs,
miR-335 and miR-206, whose expression is selectively reduced during metastasis and that positively correlate with a
low metastasis free survival in patients. Indeed, replenishing
these miRNAs reduced metastasis in vivo but did not inhibit
tumor growth or cell proliferation, which is in contrast to
miR-126. miR-126, which was discovered in the same
screen, also inhibits metastasis; however, it is unclear if its
effect is direct or secondary to inhibition of the tumor's
growth. It was also determined that the function of miR-
335 and miR-206 are not due to inhibition of EMT, but
rather a consequence of an independent decrease in cell
migration. miR-335 targets included the transcription factor Sox4 and the extracellular matrix protein tenascin, both
of which are required for cell migration.

Another miRNA that was identified in metastasis,
based on its consistent downregulation that was reported
by separate differential expression studies, is miR-31. While
exhibiting ~4-fold downregulation in nonmetastatic cancer
cells relative to noncancerous cells, it is >100-fold reduced
in metastatic cells (624). It is also lower in metastatic cells
relative to their tumor of origin, and its expression inversely
correlates with metastasis in human samples. Although
overexpression of miR-31 in implanted tumors resulted in
~1.5-fold larger tumors, they were well encapsulated with
reduced metastatic capacity. In addition to that, miR-31-
expressing cancer cells had reduced capacity for colonization and proliferation in metastatic sites. A reverse effect
was elicited by knockdown of miR-31 in noninvasive cells.
In this context, miR-31 was shown to target and inhibit a
consortium of genes that included frizzled (FZD3), integrin
α5 (ITGA5), matrix metalloproteinase 16 (MMP16), radixin (RDX), and RhoA. In particular, replenishing RhoA
alone was sufficient for completely reversing miR-31-induced inhibition of invasion or metastasis both in vitro and

in vivo. RhoA is also a downstream effector of TGF-β-
induced EMT (45) and, thus, may be involved in mediating
its effect on the miR-200 family.

Let-7 has been shown to function as a tumor suppressor, at least partly through suppression of Ras (282). In
breast cancer, let-7 suppresses proliferation, tumorigenicity,
and metastatic potential of breast cancer stem cell (695). In
addition to the inflammatory signaling pathway described
above, let-7 is also downregulated by the Raf-MAPK-Myc-
Lin-28 pathway that is suppressed by Ras kinase inhibitory
protein (130). By targeting HMGA2, a chromatin remodeling protein, it suppresses the expression of Snail, which is
expressed in metastatic tumors and has been implicated in
the mechanism for EMT by directly suppressing E-cadherin
(73). This further explains how downregulation of let-7
may suppress breast cancer cell metastasis (695).

Breast cancer metastasis suppressor 1 (BRMS1) is a
known suppressor of breast cancer metastasis (548), functioning via inhibiting the activities of various genes including NFκB (110), EGFR (255), and urokinase-type plasminogen activator (uPA) (109). Recently, BRMS1 was shown
to induce miR-146, which directly suppresses expression of
EGFR and has the capacity to independently inhibit metastasis, suggesting that it may mediate BRMS1's inhibitory
function (254). In consensus, miR-146 also inhibits NFκB
activity by suppressing its regulators TNF receptor-associated protein 6 (TRAF6) and interleukin-1 receptor associated kinase (IRAK1) (44). In parallel to its regulation by
miR-146, EGFR is indirectly induced as a result of down-
regulation of the tumor suppressor miR-145 and the increase in its target mucin 1 (520), which prevents its degradation (490). Mucin 1 proved sufficient for enhancing cell
invasion, and its effect is overcome by overexpression of
miR-145 (520), although whether EGFR mediates its effect
has not been determined. miR-193b is also implicated in
this network as it is reduced in breast cancer cells, thus
releasing uPA from its inhibition (364). Accordingly, over-
expression of miR-193b inhibits breast cancer cell growth
and metastasis, but there is no evidence yet if it is also
induced by BRMS1.

In contrast to miRNAs that inhibit metastasis, others
were found to promote it. miR-10b is specifically upregulated in metastatic breast cancer cells (397). It is directly
induced by Twist, which itself plays an essential role in
EMT-induced metastasis (688). Both gain- and loss-of-
function experiments proved that miR-10b is both necessary and sufficient for specifically inducing cell invasion and
metastasis in vitro and in vivo (397). HoxD10 was validated as a direct target of miR-10b. This protein belongs to
a family of homeobox genes that are involved in maintenance of the differentiated cell status and are, accordingly,
generally downregulated in invasive breast cancer tissue
(402). In particular, HoxD10 was shown to inhibit breast

cancer cell invasion (78). Suppression of this gene's expression and derepression of its transcriptional target RhoC are necessary events in mediating miR-10's metastatic effect (397). This pathway potentially converges on EMT via RhoC, which induces EMT in colon cancer cells (28).

In addition to miR10b, upregulation of miR-373 and miR-520c also specifically promote metastasis (250). These miRNAs were discovered through a gain-of-function screen that involved 450 miRNAs. Introducing miR-373 or miR-520c into MCF-7 resulted in enhancing cell migration and invasion and in vivo metastasis, but not proliferation. Moreover, miR-373/520c were higher in metastatic lymph node cells than in primary breast cancer tumors from which they originated. A main target that is suppressed by both these miRNAs, as they both have the same seed sequence, is the metastasis-related hyaluronan receptor CD44. However, studies on this molecule have overwhelmingly demonstrated a prometastatic role of CD44 in various cancer cells and has been accordingly a target of therapeutic interventions [reviewed by Toole et al. (616)]. Thus the fact that miR-373/520c promotes metastasis by suppressing CD44 does not reconcile well with the established function of this protein. It should be noted, though, that there are a few studies that demonstrated an anti-metastatic role for CD44, including Huang et al. (250), who show that knockdown of CD44 in breast cancer cells enhances invasiveness, whereas overexpressing it reduces miR-373-induced invasion. However, a role for other miR-373/520c targets cannot be excluded.

Several other miRNAs are also involved in breast cancer metastasis through regulating crucial molecules that are involved in the process. For instance, miR-205 is downregulated in human breast cancer resulting in increased expression of its target ErbB3, thus activating the AKT pathway (266, 669). Consequently, reintroducing miR-205 in cancer cells reversed these effects and reduced colony formation. Moreover, it suppressed lung metastasis in vivo (669). Although another study has shown that miR-205 targets the EMT genes ZEB1 and ZEB2, its expression levels in invasive ductal versus metaplastic breast cancer tissues was variable and did not correlate with E-cadherin levels (215). miR-661 also participates in metastasis by declining during breast cancer progression as a consequence of the downregulation of its transcriptional activator c/EBPα (504). It targets metastatic tumor antigen 1 (MTA1), which is a transcriptional corepressor that induces deacetylation of histones, p53, and Hif-1α, and promotes a more aggressive phenotype of cancer cells (615). Thus, as predicted, overexpression of miR-661 in breast cancer cells inhibits migration, invasion, and tumor size in vivo (504). miR-29, on the other hand, is upregulated in the invasive mesenchymal breast cancer cells but not in epithelial cell lines (203). It enhances EMT-dependent metastasis and directly suppresses tristetraprolin (TTP). TTP is an RNA-binding protein that destabilizes AU-rich mRNAs, which has been implicated in tumor suppression by targeting genes such as IL-3 (580), Hif-1α (301), and Cox2 (56). Knockdown of this protein was shown to enhance EMT; however, direct proof that it indeed mediates miR-29a effects has not been presented (203).

### 4. Estrogen receptors (ER) in breast cancer

Most breast tumors express ERα and ERβ receptors, which have opposing effects on mammary development and differentiation; ERα enhances proliferation while ERβ inhibits it (355). Several studies have identified miRNAs that are regulated by estrogen in cancer cells. These included miR-17~92 and its paralog miR-106a-363 that are upregulated by E2 in MCF-7 cells and function as negative regulators of the ER receptor and its coactivator AIB1 (80), miR-21, and let-7 (42). However, upregulation of miR-17~92 and let-7 contradict other findings that show that they are downregulated in breast cancer (247, 695, 698). Controversially, Maillot et al. (400) reported that estrogen induces only downregulation of miRNAs in MCF-7 cells (400). In particular, miR-26a and miR-181a strongly inhibit estrogen-induced proliferation and directly target and suppress the progesterone receptor. On the other hand, miRNAs that regulate the ERα include miR-206 (3) and miR-221/222 (3), which is consistent with the finding that miR-221/222 is upregulated in tamoxifen-resistant cancer cells (427).

### XIV. MicroRNAs IN OVARIAN CANCER

In general, ovarian cancer (OC) is associated with a larger fraction of downregulated (88.6%) than upregulated miRNAs (706). Indeed, it was found that ~37% of miRNA genes are encompassed in genomic regions that are subject to DNA copy abnormalities in OC (705), while ~36% exhibit epigenetic silencing (706). However, studies have differed on the role of Dicer and Drosha in this preferential downregulation of miRNAs. The consensus, though, is that low Dicer expression in epithelial and serous ovarian cancer is associated with advanced tumor stage and low survival (169, 420). Some specific prognostic markers that have also been reported include low levels of the miR-200 family that predicted poor survival (248) and low level of let-7i that was associated with shorter progression-free survival of end-stage patients (689). Moreover, HMGA2 was identified as a let-7 target that was elevated in OC, and together their reciprocal pattern of expression was associated with a <5-yr progression-free survival (552). It should be noted though that the expression of let-7 and HMGA2 did not differ in primary versus metastatic tumors (473). Consistent results were observed with lin-28b, a negative regulator of let-7 processing, where its expression positively correlated with low progression-free survival (387). In addition, let-7 is also silenced by DNA methylation (386).

MicroRNAs IN DEVELOPMENT AND DISEASE

Specific miRNAs that are downregulated in OC have been identified as inhibitors of cell growth; those include miR-34c, miR-15/16, miR-9, and miR-199a. P53-deficient OC cells are low in miR-34b and miR-34c, which are transcriptional targets of p53 (118). Moreover, low levels of miR34c correlate with decreased survival in serous OC (338). Replenishment of these miRNAs reduces cell proliferation; however, predicted targets remain to be validated (118). Reduced miR-15 and miR-16's expression levels inversely correlate with their target Bmi-1 and, thus, inhibit proliferation when they are ectopically expressed (43). A similar function is observed via downregulation of miR-9 and derepression of its target NFκB1 (223), whose activity may be further enhanced by downregulation of miR-199a and upregulation of its target IKKB (92). On the other hand, downregulation of the miR-200 family members directly correlates with the expression of E-cadherin in primary serous papillary OC, as it has been shown to directly inhibit the expression of E-cadherin’s repressors ZEB1 and ZEB2 (472). The latter cluster of miRNAs, in addition to an increase in AT-rich interactive domain 3B protein (ARID3B) as a result of reduced miR-125 levels (124), are likely to contribute to EMT and enhance OC cell migration, although this has not been confirmed yet.

Another array of miRNAs that have proapoptotic functions are also dysregulated in CaP and enhance cell survival. These include downregulation of miR-34, which is especially reduced in p53-deficient CaP cells, as it is induced by p53 and mediates its apoptotic effects (188, 511); downregulation of miR-330, which suppresses E2F-1 and E2F-1-mediated AKT phosphorylation (339); upregulation of miR-20a, which suppresses E2F2 and E2F3 and prevents their excessive accumulation and, thus, apoptosis (590); upregulation of miR-125b, which inhibits the expression of the proapoptotic protein Bak1 (557), in addition, to the upregulation of miR-21, which is an established suppressor of apoptosis (361). In CaP cells, miR-21 is induced by stimulation of androgen receptors and mediates hormone-dependent and -independent cell growth (508). In contrast, though, miR-21 enhances cell migration and invasion but not proliferation in androgen-independent cell lines, where it targets myristoylated alanine-rich C kinase substrate (MARKCS) (361). However, the set of targets that distinguish miR-21’s functions in androgen-dependent vs. -independent cells has not been determined. In contrast to miR-21, miR-221 increases in androgen-independent tumors (586) and positively regulates cell proliferation, colony formation, and tumor growth through directly suppressing p27kip (193, 419). In contrast, it is downregulated in 98% of metastatic CaP tumors and is associated with poor prognosis (576). However, the set of targets that distinguish these two effects of miR-221 have not been determined. Other miRNAs involved in cell invasion of CaP include miR-146a, which is reduced in androgen-independent CaP cells (369). It inhibits proliferation and hyaluronan-induced cell invasion through preventing upregulation of Rho-activated protein kinase 1 (ROCK1). On the other hand, PDGF induces EMT and cell invasion through downregulation of miR-200 family member, which targets ZEB1 and ZEB2 (310), while downregulation of miR-205 participates in this process by derepressing a panel of targets that include ErbB3, E2F1, E2F5, ZEB2, and PKC-ε (194).

XV. MicroRNAs IN PROSTATE CANCER

miRNAs profiling of prostate tumors uncovered the dysregulation of some of the usual culprits, including downregulation of let-7, miR-145, miR-141, miR-125, miR-1, miR-133, miR-106b, and miR-16 (12, 466, 491). The miR-15/16 cluster, which is present in the 13q14 chromosomal region, is increasingly lost during the progression of cancer of the prostate (CaP). Reduced levels of miR-15/16 are associated with an increase in the protein levels of their targets Bcl2, cyclin D1, and WNT3A (53) that is known to enhance CaP growth and stem cell self-renewal (46, 363, 635). Indeed, knockdown of this cluster in a mouse model results in prostate hyperplasia that is associated with an increase in cyclin D1 and Wnt3A (53). Likewise, its knockdown in normal prostate cells increases survival, proliferation, migration, and tumorigenic capacity and invasion in vivo. Conversely, overexpression of miR-15/16 suppresses tumor growth and induces regression. Another miRNA that is frequently lost in CaP is miR-101; it is absent in 37% of local CaP tumors and 66% of metastatic cells (629). This inversely correlates with upregulation of its target, enhancer of zeste homolog 2 (Ezh2), a methyltransferase subunit of the polycomb repressive complex 2. This protein is highly expressed in hormone-refractory, metastatic, prostate cancer and promotes cell proliferation (630). Accordingly, suppression of Ezh2 expression by miR-101 inhibits cell proliferation and invasion and reduces global genomic trimethyl H3k27 levels (629). Additional tumor suppressors that are downregulated in CaP include miR-331–3p and miR-449a, which contribute to tumor growth by releasing their targets ErbB2 (164) and HDAC1 (453), respectively, from their inhibitory effect.

XVI. EXPLOITING MicroRNAs IN THERAPEUTICS

miRNAs are characterized by targeting multiple, functionally related, versus single, genes, which renders them a potentially powerful therapeutic tool. As an example, it proved more functionally effective to simultaneously, rather than individually, inhibit the cell cycle-related miR-16 targets (373). One of the tools that has been developed for reducing a miRNA’s levels in vivo is a specific antisense oligonucleotide. Indeed, various forms of modified antisense miRNAs (antimiRs), with (antagomirs) or without a cholesterol conjugate, have been applied in vivo with great success. Hutvagner et al. (258) first reported that 2′-O-methyl-modified oligonucleotides successfully inhibited let-7 function in *Drosophila*, whereas Esau et al. (165) observed inhibition of liver miR-122 in mice by using 2′-O-methoxyethyl phosphorothioate-modified oligodeoxy-

nucleotides. Likewise, injection of 2′-O-methyl, phosphorothioate, 5′-cholesterol-conjugated antisense miR-122 modified oligonucleotides “antagomirs,” effectively inhibited miR-122 in the liver (320). These antisense oligos appear to function by binding to the corresponding mature miRNA within the cytoplasmic compartment and induces its degradation (319). There are several examples that have been mentioned throughout this review that demonstrate the successful use of antisense miRNA constructs in treating animal disease models. This includes alleviation of cardiac fibrosis and hypertrophy by intravenous injection of a miR-21 antagomir, as a result of derepressing the expression of its target SPRY1 and increasing myofibroblast apoptosis (612). Alternatively, Elmen et al. (161) used intravenous injection of a 16-nt-long antisense, unconjugated, locked-nucleic-acid-modified oligonucleotide (LNA-antimiR), to deplete miR-122 in primates’ liver. This strategy proved effective in reducing serum (2.6 logs) and liver (2.3 logs) HCV, with no evidence of viral resistance or side effects (331).

On the other hand, miRNAs are also frequently downregulated in disease, in which case a therapeutic approach would require replacement of specific miRNAs. According to our knowledge of a miRNA’s processing and function, this would require the double-stranded form of the molecule for proper assembly of the “driver” strand into the miRISC. This kind of construct was dubbed miRNA mimic or mimetic. The double-stranded design, versus the single-stranded mature miRNA, did indeed prove necessary for efficient incorporation of the mature miRNA into the miRISC (182, 221). As proof of concept, Wiggins et al. (662) showed that intravenous or intratumoral injection of a miR-34a mimic, in a lipid-based vehicle, in a mouse model of non-small-cell lung cancer, induced apoptosis and significantly inhibited tumor growth. Similarly, direct injection of Let-7 mimics into a lung cancer xenograft reduced tumor burden by 66% (619). As an alternative, the primary or precursor sequence of miRNAs under the control of a constitutive or tissue-specific promoter can be delivered in vivo using viral vectors. This has been successfully demonstrated by intranasal delivery of a Let-7-expressing lentivirus in mice, which resulted in a 75% decrease in size of K-ras-induced tumors (619). On the other hand, Kato et al. delivered miR-26a intravenously via an adenoassociated virus (315). One of the criteria they used was that the miRNA they chose would be normally highly expressed in all organs and reduced only in the tumor, as they predicted it would be generally well tolerated. Promisingly, they found that miR-26a, which targets cyclins D2 and E2, reduced tumor burden in the liver, as a result of reduced proliferation and increased apoptosis of the tumor cell. Conversely, one can envision the delivery of viral vectors that express specific antisense miRNA-targeting constructs, such as the “eraser” (497, 534) or the “sponge” (152) for reducing the levels of upregulated miRNA in diseased organs, which awaits testing.

---

**XVII. CONCLUSION**

In conclusion, while much has been done to elucidate the functions of individual miRNAs during health and disease, much more remains to be discovered. Specifically, the full spectrum of genes that a single miRNA targets in a given context, how these targets functionally interact, and the extent to which each contributes to the miRNA’s observed functionality. In addition, since a single mRNA could be simultaneously regulated by multiple miRNAs, the collaborative effects of these miRNAs in a given function need to also be explored. These details will further explain the functional significance of miRNAs and aid in planning for future therapeutic targeting of a miRNA.

---

**ACKNOWLEDGMENTS**

Address for reprint requests and other correspondence: M. Abdellatif, Cardiovascular Research Institute, Dept. of Cell Biology and Molecular Medicine, Univ. of Medicine and Dentistry of New Jersey, Newark, NJ 07103 (e-mail: abdellma@umdnj.edu).

---

**DISCLOSURES**

No conflicts of interest, financial or otherwise, are declared by the authors.

---

**REFERENCES**

1. Abelson JF, Kwan KY, O’Roak BJ, Baek DY, Stillman AA, Morgan TM, Mathews CA, Pauls DL, Rasin MR, Gunel M, Davis NR, Ercan-Sencicek AG, Guez DH, Spertus JA, Leckman JF, Dure LSt Kurlan R, Singer HS, Gilbert DL, Farhi A, Louvi A, Lifton RP, Sestan N, State MW. Sequence variants in SLITRK1 are associated with Tourette’s syndrome. *Science* 310: 317–320, 2005.
2. Adam L, Zhong M, Choi W, Qi W, Nicoloso M, Arora A, Calin G, Wang H, Siefker-Radtke A, McConkey D, Bar-Eli M, Dinney C. miR-200 expression regulates epithelial-to-mesenchymal transition in bladder cancer cells and reverses resistance to epidermal growth factor receptor therapy. *Clin Cancer Res* 15: 5060–5072, 2009.
3. Adams BD, Furneaux H, White BA. The micro-ribonucleic acid (miRNA) miR-206 targets the human estrogen receptor-alpha (ERalpha) and represses ERalpha messenger RNA and protein expression in breast cancer cell lines. *Mol Endocrinol* 21: 1132–1147, 2007.
4. Agirre X, Jimenez-Velasco A, San Jose-Eneriz E, Garate L, Bandres E, Cordeu L, Aparicio O, Saez B, Navarro G, Vilas-Zornoza A, Perez-Roger I, Garcia-Foncillas J, Torres A, Heiniger A, Calasanz MJ, Fortes P, Roman-Gomez J, Prosper F. Downregulation of hsa-miR-10a in chronic myeloid leukemia CD34+ cells increases USF2-mediated cell growth. *Mol Cancer Res* 6: 1830–1840, 2008.
5. Agrawal R, Tran U, Wessely O. The miR-30 miRNA family regulates Xenopus pronephros development and targets the transcription factor Xlim1/Lhx1. *Development* 136: 3927–3936, 2009.
6. Akao Y, Nakagawa Y, Kitade Y, Kinoshita T, Naoe T. Downregulation of microRNAs-143 and -145 in B-cell malignancies. *Cancer Sci* 98: 1914–1920, 2007.
7. Akao Y, Nakagawa Y, Naoe T. let-7 microRNA functions as a potential growth suppressor in human colon cancer cells. *Biol Pharm Bull* 29: 903–906, 2006.
8. Akao Y, Nakagawa Y, Naoe T. MicroRNAs 143 and 145 are possible common oncogenic microRNAs in human cancers. *Oncol Rep* 16: 845–850, 2006.

MicroRNAs IN DEVELOPMENT AND DISEASE

9. Al-Hajj M, Wicha MS, Benito-Hernandez A, Morrison SJ, Clarke MF. Prospective identification of tumorigenic breast cancer cells. Proc Natl Acad Sci USA 100: 3983–3988, 2003.

10. Ambros V. A hierarchy of regulatory genes controls a larva-to-adult developmental switch in C. elegans. Cell 57: 49–57, 1989.

11. Ambros V, Bartel B, Bartel DP, Burge CB, Carrington JC, Chen X, Dreyfuss G, Eddy SR, Griffiths-Jones S, Marshall M, Matzke M, Ruvkun G, Tuschl T. A uniform system for microRNA annotation. RNA 9: 277–279, 2003.

12. Ambs S, Prueitt RL, Yi M, Hudson RS, Howe TM, Petrocca F, Wallace TA, Liu CG, Volinia S, Calin GA, Yfantis HG, Stephens RM, Croce CM. Genomic profiling of microRNA and messenger RNA reveals deregulated microRNA expression in prostate cancer. Cancer Res 68: 6162–6170, 2008.

13. Amthor H, Christ B, Patel K. A molecular mechanism enabling continuous embryonic muscle growth: a balance between proliferation and differentiation. Development 126: 1041–1053, 1999.

14. Araya R, Eckardt D, Maxeiner S, Kruger O, Theis M, Willecke K, Saez JC. Expression of connexins during differentiation and regeneration of skeletal muscle: functional relevance of connexin 43. J Cell Sci 118: 27–37, 2005.

15. Araya R, Eckardt D, Riquelme MA, Willecke K, Saez JC. Presence and importance of connexin 43 during myogenesis. Cell Commun Adhes 10: 451–456, 2003.

16. Arndt GM, Dossey L, Cullen LM, Lai A, Druker R, Eisbacher M, Zhang C, Tran N, Fan H, Retzlaff K, Bittner A, Raponi M. Characterization of global microRNA expression reveals oncogenic potential of miR-145 in metastatic colorectal cancer. BMC Cancer 9: 374, 2009.

17. Aruga J, Mikoshiba K. Identification and characterization of Slitrk, a novel neuronal transmembrane protein family controlling neurite outgrowth. Mol Cell Neurosci 24: 117–129, 2003.

18. Asangani IA, Rasheed SA, Nikolova DA, Leupold JH, Colburn NH, Post S, Allgayer H. MicroRNA-21 (miR-21) post-transcriptionally downregulates tumor suppressor Pdcd4 and stimulates invasion, intravasation and metastasis in colorectal cancer. Oncogene 29: 29, 2007.

19. Avnit-Sagi T, Kantorovich L, Kredo-Russo S, Hornstein E, Walker MD. The promoter of the pri-miR-375 gene directs expression selectively to the endocrine pancreas. PLoS One 4: 3, 2009.

20. Bachman KE, Herman JG, Corn PG, Merlo A, Costello JF, Cavenee WK, Baylin SB, Graff JR. Methylation-associated silencing of the tissue inhibitor of metalloproteinase-3 gene suggests a suppressor role in kidney, brain, and other human cancers. Cancer Res 59: 798–802, 1999.

21. Bae SN, Arand G, Azzam H, Pavasant P, Torri J, Frandsen TL, Thompson EW. Molecular and cellular analysis of basement membrane invasion by human breast cancer cells in Matrigel-based in vitro assays. Breast Cancer Res Treat 24: 241–255, 1993.

22. Bagga S, Bracht J, Hunter S, Massirer K, Holtz J, Eachus R, Pasquinelli AE. Regulation by let-7 and lin-4 miRNAs results in target mRNA degradation. Cell 122: 553–563, 2005.

23. Bai S, Nasser MW, Wang B, Hsu SH, Datta J, Kutay H, Yadav A, Nuovo G, Kumar P, Ghoshal K. MicroRNA-122 inhibits tumorigenic properties of hepatocellular carcinoma cells and sensitizes these cells to sorafenib. J Biol Chem 284: 32015–32027, 2009.

24. Bandi N, Zbinden S, Gugger M, Arnold M, Kocher V, Hasan L, Kappeler A, Brunner T, Vassella E. miR-15a and miR-16 are implicated in cell cycle regulation in a Rb-dependent manner and are frequently deleted or down-regulated in non-small cell lung cancer. Cancer Res 69: 5553–5559, 2009.

25. Bartram CR, de Klein A, Hagemeijer A, van Agthoven T, Geurts van Kessel A, Bootsma D, Grosveld G, Ferguson-Smith MA, Davies T, Stone M. Translocation of c-abl oncogene correlates with the presence of a Philadelphia chromosome in chronic myelocytic leukaemia. Nature 306: 277–280, 1983.

26. Begemann M, Grube S, Papiol S, Malzahn D, Krampe H, Ribbe K, Friedrichs H, Radyushkin KA, El-Kordi A, Benseler F, Hannke K, Sperling S, Schwerdtfeger D, Thanhauser I, Gerchen MF, Ghorbani M, Gutwinski S, Hilmes C, Leppert R, Ronnenberg A, Sowislo J, Stawicki S, Stoldtke M, Szuszies C, Reim K, Riggert J, Eckstein F, Falkai P, Bickeboller H, Nave KA, Brose N, Ehrenreich H. Modification of cognitive performance in schizophrenia by complexin 2 gene polymorphisms. Arch Gen Psychiatry 67: 879–888, 2010.

27. Behm-Ansmant I, Rehwinkel J, Doerks T, Stark A, Bork P, Izaurralde E. mRNA degradation by miRNAs and GW182 requires both CCR4-NOT deadenylase and DCP1-DCP2 decapping complexes. Genes Dev 20: 1885–1898, 2006.

28. Bellovin DI, Simpson KJ, Danilov T, Maynard E, Rimm DL, Oettgen P, Mercurio AM. Reciprocal regulation of RhoA and RhoC characterizes the EMT and identifies RhoC as a prognostic marker of colon carcinoma. Oncogene 25: 6959–6967, 2006.

29. Ben-Neriah Y, Daley GQ, Mes-Masson AM, Witte ON, Baltimore D. The chronic myelogenous leukemia-specific P210 protein is the product of the bcr/abl hybrid gene. Science 233: 212–214, 1986.

30. Bennasser Y, Jeang KT. HIV-1 Tat interaction with Dicer: requirement for RNA. Retrovirology 3: 95, 2006.

31. Bennasser Y, Le SY, Benkirane M, Jeang KT. Evidence that HIV-1 encodes an siRNA and a suppressor of RNA silencing. Immunity 22: 607–619, 2005.

32. Bennasser Y, Le SY, Yeung ML, Jeang KT. HIV-1 encoded candidate micro-RNAs and their cellular targets. Retrovirology 1: 43, 2004.

33. Bentwich I. Prediction and validation of microRNAs and their targets. FEBS Lett 579: 5904–5910, 2005.

34. Berezikov E, Chung WJ, Willis J, Cuppen E, Lai EC. Mammalian mirtron genes. Mol Cell 28: 328–336, 2007.

35. Bernstein J, Evan AP, Gardner KD Jr. Epithelial hyperplasia in human polycystic kidney diseases. Its role in pathogenesis and risk of neoplasia. Am J Pathol 129: 92–101, 1987.

36. Betel D, Wilson M, Gabow A, Marks DS, Sander C. The microRNA org resource: targets and expression. Nucleic Acids Res 36: D149–153, 2008.

37. Beuvink I, Kolb FA, Budach W, Garnier A, Lange J, Natt F, Dengler U, Hall J, Filipowicz W, Weiler J. A novel microarray approach reveals new tissue-specific signatures of known and predicted mammalian microRNAs. Nucleic Acids Res 13: 13, 2007.

38. Beveridge NJ, Gardiner E, Carroll AP, Tooney PA, Cairns MJ. Schizophrenia is associated with an increase in cortical microRNA biogenesis. Mol Psychiatry 1: 1, 2009.

39. Beveridge NJ, Tooney PA, Carroll AP, Gardiner E, Bowden N, Scott RJ, Tran N, Dedova I, Cairns MJ. Dysregulation of miRNA 181b in the temporal cortex in schizophrenia. Hum Mol Genet 17: 1156–1168, 2008.

40. Beyer A, Hollunder J, Nasheuer HP, Wilhelm T. Post-transcriptional expression regulation in the yeast Saccharomyces cerevisiae on a genomic scale. Mol Cell Proteomics 3: 1083–1092, 2004.

41. Bhaskaran M, Wang Y, Zhang H, Weng T, Baviskar P, Guo Y, Gou D, Liu L. MicroRNA-127 modulates fetal lung development. Physiol Genomics 37: 268–278, 2009.

42. Bhat-Nakshatri P, Wang G, Collins NR, Thomson MJ, Geistlinger TR, Carroll JS, Brown M, Hammond S, Srour EF, Liu Y, Nakshatri H. Estradiol-regulated microRNAs control estradiol response in breast cancer cells. Nucleic Acids Res 37: 4850–4861, 2009.

43. Bhattacharya R, Nicoloso M, Arvizo R, Wang E, Cortez A, Rossi S, Calin GA, Mukherjee P. MiR-15a and MiR-16 control Bmi-1 expression in ovarian cancer. Cancer Res 69: 9090–9095, 2009.

44. Bhaumik D, Scott GK, Schokrpur S, Patil CK, Campisi J, Benz CC. Expression of microRNA-146 suppresses NF-kappaB activity with reduction of metastatic potential in breast cancer cells. Oncogene 5643–5647, 2008.

45. Bhowmick NA, Ghiassi M, Bakin A, Aakre M, Lundquist CA, Engel ME, Arteaga CL, Moses HL. Transforming growth factor-beta mediates epithelial to mesenchymal transdifferentiation through a RhoA-dependent mechanism. Mol Biol Cell 12: 27–36, 2001.

46. Bisson I, Prowse DM. WNT signaling regulates self-renewal and differentiation of prostate cancer cells with stem cell characteristics. Cell Res 19: 683–697, 2009.

47. Bloomston M, Frankel WL, Petrocca F, Volinia S, Alder H, Hagan JP, Liu CG, Bhatt D, Taccioli C, Croce CM. MicroRNA expression patterns to differentiate pancreatic adenocarcinoma from normal pancreas and chronic pancreatitis. JAMA 297: 1901–1908, 2007.

48. Blow MJ, Grocock RJ, van Dongen S, Enright AJ, Dicks E, Futreal PA, Wooster R, Stratton MR. RNA editing of human microRNAs. Genome Biol 7: 4, 2006.

49. Boettger T, Beetz N, Kostin S, Schneider J, Kruger M, Hein L, Braun T. Acquisition of the contractile phenotype by murine arterial smooth muscle cells depends on the Mir143/145 gene cluster. J Clin Invest 119: 2634–2647, 2009.

50. Bohnsack MT, Czaplinski K, Gorlich D. Exportin 5 is a RanGTP-dependent dsRNA-binding protein that mediates nuclear export of pre-miRNAs. RNA 10: 185–191, 2004.

51. Boissonneault V, Plante I, Rivest S, Provost P. MicroRNA-298 and microRNA-328 regulate expression of mouse beta-amyloid precursor protein-converting enzyme I. J Biol Chem 284: 1971–1981, 2009.

52. Bonauer A, Carmona G, Iwasaki M, Mione M, Koyanagi M, Fischer A, Burchfield J, Fox H, Doebele C, Ohtani K, Chavakis E, Potente M, Tjwa M, Urbich C, Zeiher AM, Dimmeler S. MicroRNA-92a controls angiogenesis and functional recovery of ischemic tissues in mice. Science 324: 1710–1713, 2009.

53. Bonci D, Coppola V, Musumeci M, Addario A, Giuffrida R, Memeo L, D'Urso L, Pagliuca A, Biffoni M, Labbaye C, Bartucci M, Muto G, Peschle C, De Maria R. The miR-15a-miR-16-1 cluster controls prostate cancer by targeting multiple oncogenic activities. Nat Med 14: 1271–1277, 2008.

54. Borchert GM, Lanier W, Davidson BL. RNA polymerase III transcribes human microRNAs. Nat Struct Mol Biol 13: 1097–1101, 2006.

55. Bottoni A, Piccin D, Tagliati F, Luchin A, Zatelli MC, degli Uberti EC. miR-15a and miR-16-1 down-regulation in pituitary adenomas. J Cell Physiol 204: 280–285, 2005.

56. Boutaud O, Dixon DA, Oates JA, Sawaoka H. Tristetraprolin binds to the COX-2 mRNA 3′ untranslated region in cancer cells. Adv Exp Med Biol 525: 157–160, 2003.

57. Boutet SC, Disatnik MH, Chan LS, Iori K, Rando TA. Regulation of Pax3 by proteasomal degradation of monoubiquitinated protein in skeletal muscle progenitors. Cell 130: 349–362, 2007.

58. Braun CJ, Zhang X, Savelyeva I, Wolff S, Moll UM, Schepeler T, Orntoft TF, Andersen CL, Dobbelstein M. p53-Responsive microRNAs 192 and 215 are capable of inducing cell cycle arrest. Cancer Res 68: 10094–10104, 2008.

59. Bravo-Egana V, Rosero S, Molano RD, Pileggi A, Ricordi C, Dominguez-Bendala J, Pastori RL. Quantitative differential expression analysis reveals miR-7 as major islet microRNA. Biochem Biophys Res Commun 366: 922–926, 2008.

60. Brems H, Chmara M, Sahbatou M, Denayer E, Taniguchi K, Kato R, Somers R, Messiaen L, De Schepper S, Fryns JP, Cools J, Marynen P, Thomas G, Yoshimura A, Legius E. Germline loss-of-function mutations in SPRED1 cause a neurofibromatosis I-like phenotype. Nat Genet 39: 1120–1126, 2007.

61. Brennecke J, Stark A, Russell RB, Cohen SM. Principles of microRNA-target recognition. PLoS Biol 3: 2005.

62. Bueno MJ, Perez de Castro I, Gomez de Cedron M, Santos J, Calin GA, Cigudosa JC, Croce CM, Fernandez-Piqueras J, Malumbres M. Genetic and epigenetic silencing of microRNA-203 enhances ABL1 and BCR-ABL1 oncogene expression. Cancer Cell 13: 496–506, 2008.

63. Burk U, Schubert J, Wellner U, Schmalhofer O, Vincan E, Spaderna S, Brabletz T. A reciprocal repression between ZEB1 and members of the miR-200 family promotes EMT and invasion in cancer cells. EMBO Rep 9: 582–589, 2008.

64. Bykov VJ, Issaeva N, Shilov A, Hultcrantz M, Pugacheva E, Chumakov P, Bergman J, Wiman KG, Selivanova G. Restoration of the tumor suppressor function to mutant p53 by a low-molecular-weight compound. Nat Med 8: 282–288, 2002.

65. Cai Z, Zhong H, Bosch-Marce M, Fox-Talbot K, Wang L, Wei C, Trush MA, Semenza GL. Complete loss of ischaemic preconditioning-induced cardioprotection in mice with partial deficiency of HIF-1α. Cardiovasc Res 77: 463–470, 2008.

66. Calin GA, Cimmino A, Fabbri M, Ferracin M, Wojcik SE, Shimizu M, Taccioli C, Zanesi N, Garzon R, Aqeilan RI, Alder H, Volinia S, Rassenti L, Liu X, Liu CG, Kipps TJ, Negrini M, Croce CM. MiR-15a and miR-16-1 cluster functions in human leukemia. Proc Natl Acad Sci USA 105: 5166–5171, 2008.

67. Calin GA, Dumitru CD, Shimizu M, Bichi R, Zupo S, Noch E, Adler H, Rattan S, Keating M, Rai K, Rassenti L, Kipps T, Negrini M, Bullrich F, Croce CM. Frequent deletions and down-regulation of micro-RNA genes miR15 and miR16 at 13q14 in chronic lymphocytic leukemia. Proc Natl Acad Sci USA 99: 15524–15529, 2002.

68. Calin GA, Sevignani C, Dumitru CD, Hyslop T, Noch E, Yendamuri S, Shimizu M, Rattan S, Bullrich F, Negrini M, Croce CM. Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers. Proc Natl Acad Sci USA 101: 2999–3004, 2004.

69. Callis TE, Pandya K, Seok HY, Tang RH, Tatsuguchi M, Huang ZP, Chen JF, Deng Z, Gunn B, Shumate J, Willis MS, Selzman CH, Wang DZ. MicroRNA-208a is a regulator of cardiac hypertrophy and conduction in mice. J Clin Invest 119: 2772–2786, 2009.

70. Cameron JE, Fewell C, Yin Q, McBride J, Wang X, Lin Z, Flemington EK. Epstein-Barr virus growth/latency III program alters cellular microRNA expression. Virology 382: 257–266, 2008.

71. Cameron JE, Yin Q, Fewell C, Lacey M, McBride J, Wang X, Lin Z, Schaefer BC, Flemington EK. Epstein-Barr virus latent membrane protein I induces cellular MicroRNA miR-146a, a modulator of lymphocyte signaling pathways. J Virol 82: 1946–1958, 2008.

72. Camps C, Buffa FM, Colella S, Moore J, Sotiriou C, Sheldon H, Harris AL, Gleadle JM, Ragoussis J. hsa-miR-210 is induced by hypoxia and is an independent prognostic factor in breast cancer. Clin Cancer Res 14: 1340–1348, 2008.

73. Cano A, Perez-Moreno MA, Rodrigo I, Locascio A, Blanco MJ, del Barrio MG, Portillo F, Nieto MA. The transcription factor snail controls epithelial-mesenchymal transitions by repressing E-cadherin expression. Nat Cell Biol 2: 76–83, 2000.

74. Cao X, Pfaff SL, Gage FH. A functional study of miR-124 in the developing neural tube. Genes Dev 21: 531–536, 2007.

75. Cardinali B, Castellani L, Fasanaro P, Basso A, Alema S, Martelli F, Falcone G. MicroRNA-221 and microRNA-222 modulate differentiation and maturation of skeletal muscle cells. PLoS One 4: 2009.

76. Care A, Catalucci D, Felicetti F, Bonci D, Addario A, Gallo P, Bang ML, Segnalini P, Gu Y, Dalton ND, Elia L, Latronico MV, Hoydal M, Autore C, Russo MA, Dorn GW ,2nd, Ellingsen O, Ruiz-Lozano P, Peterson KL, Croce CM, Peschle C, Condorelli G. MicroRNA-133 controls cardiac hypertrophy. Nat Med 13: 613–618, 2007.

77. Caretti G, Di Padova M, Micales B, Lyons GE, Sartorelli V. The Polycomb Ezh2 methyltransferase regulates muscle gene expression and skeletal muscle differentiation. Genes Dev 18: 2627–2638, 2004.

78. Carrio M, Arderiu G, Myers C, Boudreau NJ. Homeobox D10 induces phenotypic reversion of breast tumor cells in a three-dimensional culture model. Cancer Res 65: 7177–7185, 2005.

79. Castellano G, Torrisi E, Ligresti G, Malaponte G, Militello L, Russo AE, McCubrey JA, Canevari S, Libra M. The involvement of the transcription factor Yin Yang 1 in cancer development and progression. Cell Cycle 8: 1367–1372, 2009.

80. Castellano L, Giamas G, Jacob J, Coombes RC, Lucchesi W, Thiruchelvam P, Barton G, Jiao LR, Wait R, Waxman J, Hannon GJ, Stebbing J. The estrogen receptor-alpha-induced microRNA signature regulates itself and its transcriptional response. Proc Natl Acad Sci USA 106: 15732–15737, 2009.

81. Chalfie M, Horvitz HR, Sulston JE. Mutations that lead to reiterations in the cell lineages of C. elegans. Cell 24: 59–69, 1981.

82. Chan JA, Krichevsky AM, Kosik KS. MicroRNA-21 is an antiapoptotic factor in human glioblastoma cells. Cancer Res 65: 6029–6033, 2005.

83. Chang J, Guo JT, Jiang D, Guo H, Taylor JM, Block TM. Liver-specific microRNA miR-122 enhances the replication of hepatitis C virus in nonhepatic cells. J Virol 82: 8215–8223, 2008.

84. Chang J, Nicolas E, Marks D, Sander C, Lerro A, Buendia MA, Xu C, Mason WS, Moloshok T, Bort R, Zaret KS, Taylor JM. miR-122, a mammalian liver-specific microRNA, is processed from hcr mRNA and may downregulate the high affinity cationic amino acid transporter CAT-1. RNA Biol 1: 106–113, 2004.

85. Chang TC, Wentzel EA, Kent OA, Ramachandran K, Mullendore M, Lee KH, Feldmann G, Yamakuchi M, Ferlito M, Lowenstein CJ, Arking DE, Beer MA, Maitra A, Mendell JT. Transactivation of miR-34a by p53 broadly influences gene expression and promotes apoptosis. Mol Cell 26: 745–752, 2007.

86. Chang TC, Yu D, Lee YS, Wentzel EA, Arking DE, West KM, Dang CV, Thomas-Tikhonenko A, Mendell JT. Widespread microRNA repression by Myc contributes to tumorigenesis. Nat Genet 40: 43–50, 2008.

MicroRNAs IN DEVELOPMENT AND DISEASE

87. Chaubey A, Karanti S, Rai D, Oh T, Adhvaryu SG, Aguiar RC. MicroRNAs and deletion of the derivative chromosome 9 in chronic myeloid leukemia. *Leukemia* 23: 186–188, 2009.

88. Chen CY, Zheng D, Xia Z, Shyu AB. Ago-TNRC6 triggers microRNA-mediated decay by promoting two deadenylation steps. *Nat Struct Mol Biol* 16: 1160–1166, 2009.

89. Chen CZ, Li L, Lodish HF, Bartel DP. MicroRNAs modulate hematopoietic lineage differentiation. *Science* 303: 83–86, 2004.

90. Chen J, Feilotter HE, Pare GC, Zhang X, Pemberton JG, Garady C, Lai D, Yang X, Tron VA. MicroRNA-193b represses cell proliferation and regulates cyclin D1 in melanoma. *Am J Pathol* 176: 2520–2529, 2010.

91. Chen JF, Mandel EM, Thomson JM, Wu Q, Callis TE, Hammond SM, Conlon FL, Wang DZ. The role of microRNA-1 and microRNA-133 in skeletal muscle proliferation and differentiation. *Nat Genet* 38: 228–233, 2006.

92. Chen R, Alvero AB, Silasi DA, Kelly MG, Fest S, Visintin I, Leiser A, Schwartz PE, Rutherford T, Mor G. Regulation of IKKbeta by miR-199a affects NF-kappaB activity in ovarian cancer cells. *Oncogene* 27: 4712–4723, 2008.

93. Chen RW, Bemis LT, Amato CM, Myint H, Tran H, Birks DK, Eckhardt SG, Robinson WA. Truncation in CCND1 mRNA alters miR-16–1 regulation in mantle cell lymphoma. *Blood* 112: 822–829, 2008.

94. Chen SJ, Chen GH, Chen YH, Liu CY, Chang KP, Chang YS, Chen HC. Characterization of Epstein-Barr virus miRNAome in nasopharyngeal carcinoma by deep sequencing. *PLoS One* 5: 2010.

95. Chen X, Guo X, Zhang H, Xiang Y, Chen J, Yin Y, Cai X, Wang K, Wang G, Ba Y, Zhu L, Wang J, Yang R, Zhang Y, Ren Z, Zen K, Zhang J, Zhang CY. Role of miR-143 targeting KRAS in colorectal tumorigenesis. *Oncogene* 28: 1385–1392, 2009.

96. Chen X, Wang K, Chen J, Guo J, Yin Y, Cai X, Guo X, Wang G, Yang R, Zhu L, Zhang Y, Wang J, Xiang Y, Weng C, Zen K, Zhang J, Zhang CY. In vitro evidence suggests that miR-133a-mediated regulation of uncoupling protein 2 (UCP2) is an indispensable step in myogenic differentiation. *J Biol Chem* 284: 5362–5369, 2009.

97. Chen Y, Gorski DH. Regulation of angiogenesis through a microRNA (miR-130a) that down-regulates antiangiogenic homeobox genes GAX and HOXA5. *Blood* 111: 1217–1226, 2008.

98. Chen Y, Liu W, Chao T, Zhang Y, Yan X, Gong Y, Qiang B, Yuan J, Sun M, Peng X. MicroRNA-21 down-regulates the expression of tumor suppressor PDCD4 in human glioblastoma cell T98G. *Cancer Lett* 272: 197–205, 2008.

99. Chendrimada TP, Finn KJ, Ji X, Baillat D, Gregory RI, Liebhaber SA, Pasquinelli AE, Shiekhattar R. MicroRNA silencing through RISC recruitment of eIF6. *Nature* 447: 823–828, 2007.

100. Chendrimada TP, Gregory RI, Kumaraswamy E, Norman J, Cooch N, Nishikura K, Shiekhattar R. TRBP recruits the Dicer complex to Ago2 for microRNA processing and gene silencing. *Nature* 436: 740–744, 2005.

101. Cheng LC, Pastrana E, Tavazoie M, Doetsch F. miR-124 regulates adult neurogenesis in the subventricular zone stem cell niche. *Nat Neurosci* 12: 399–408, 2009.

102. Cheng Y, Ji R, Yue J, Yang J, Liu X, Chen H, Dean DB, Zhang C. MicroRNAs are aberrantly expressed in hypertrophic heart: do they play a role in cardiac hypertrophy? *Am J Pathol* 170: 1831–1840, 2007.

103. Cheng Y, Liu X, Yang J, Lin Y, Xu DZ, Lu Q, Deitch EA, Huo Y, Delphin ES, Zhang C. MicroRNA-145, a novel smooth muscle cell phenotypic marker and modulator, controls vascular neointimal lesion formation. *Circ Res* 105: 158–166, 2009.

104. Cheng Y, Liu X, Zhang S, Lin Y, Yang J, Zhang C. MicroRNA-21 protects against the H₂O₂-induced injury on cardiac myocytes via its target gene PDCD4. *J Mol Cell Cardiol* 47: 5–14, 2009.

105. Cheng Y, Zhu P, Yang J, Liu X, Dong S, Wang X, Chun B, Zhuang J, Zhang C. Ischaemic preconditioning-regulated miR-21 protects heart against ischaemia/reperfusion injury via anti-apoptosis through its target PDCD4. *Cardiovasc Res* 87: 431–439, 2010.

106. Chin LJ, Ratner E, Leng S, Zhai R, Nallur S, Babar I, Muller RU, Straka E, Su L, Burki EA, Crowell RE, Patel R, Kulkarni T, Homer R, Zelterman D, Kidd KK, Zhu Y, Christiani DC, Belinsky SA, Slack FJ, Weidhaas JB. A SNP in a let-7 microRNA complementary site in the KRAS 3′ untranslated region increases non-small cell lung cancer risk. *Cancer Res* 68: 8535–8540, 2008.

107. Cho WC, Chow AS, Au JS. Restoration of tumour suppressor hsa-miR-145 inhibits cancer cell growth in lung adenocarcinoma patients with epidermal growth factor receptor mutation. *Eur J Cancer* 45: 2197–2206, 2009.

108. Choy EY, Siu KL, Kok KH, Lung RW, Tsang CM, To KF, Kwong DL, Tsao SW, Jin DY. An Epstein-Barr virus-encoded microRNA targets PUMA to promote host cell survival. *J Exp Med* 205: 2551–2560, 2008.

109. Cicek M, Fukuyama R, Cicek MS, Sizemore S, Welch DR, Sizemore N, Casey G. BRMS1 contributes to the negative regulation of uPA gene expression through recruitment of HDAC1 to the NF-kappaB binding site of the uPA promoter. *Clin Exp Metastasis* 26: 229–237, 2009.

110. Cicek M, Fukuyama R, Welch DR, Sizemore N, Casey G. Breast cancer metastasis suppressor 1 inhibits gene expression by targeting nuclear factor-kappaB activity. *Cancer Res* 65: 3586–3595, 2005.

111. Cimmino A, Calin GA, Fabbri M, Iorio MV, Ferracin M, Shimizu M, Wojcik SE, Aqeilan RI, Zupo S, Dono M, Rassenti L, Alder H, Volinia S, Liu CG, Kipps TJ, Negrini M, Croce CM. miR-15 and miR-16 induce apoptosis by targeting BCL2. *Proc Natl Acad Sci USA* 102: 13944–13949, 2005.

112. Cmarik JL, Min H, Hegamyer G, Zhan S, Kulesz-Martin M, Yoshinaga H, Matsuhashi S, Colburn NH. Differentially expressed protein Pdcd4 inhibits tumor promoter-induced neoplastic transformation. *Proc Natl Acad Sci USA* 96: 14037–14042, 1999.

113. Coley W, Van Duyne R, Carpio L, Guendel I, Kehn-Hall K, Chevalier S, Narayanan A, Luu T, Lee N, Klase Z, Kashanchi F. Absence of Dicer in monocytes and its regulation by HIV-1. *J Biol Chem* 28: 28, 2010.

114. Conaco C, Otto S, Han JJ, Mandel G. Reciprocal actions of REST and a microRNA promote neuronal identity. *Proc Natl Acad Sci USA* 103: 2422–2427, 2006.

115. Connolly E, Melegari M, Landgraf P, Tchaikovskaya T, Tennant BC, Slagle BL, Rogler LE, Zavolan M, Tuschl T, Rogler CE. Elevated expression of the miR-17–92 polycistron and miR-21 in hepatitis-associated hepatocellular carcinoma contributes to the malignant phenotype. *Am J Pathol* 173: 856–864, 2008.

116. Conti A, Aguennouz M, La Torre D, Tomasello C, Cardali S, Angileri FF, Maio F, Cama A, Germano A, Vita G, Tomasello F. miR-21 and 221 upregulation and miR-181b downregulation in human grade II–IV astrocytic tumors. *J Neurooncol* 93: 325–332, 2009.

117. Cordes KR, Sheehy NT, White MP, Berry EC, Morton SU, Muth AN, Lee TH, Miano JM, Ivey KN, Srivastava D. miR-145 and miR-143 regulate smooth muscle cell fate and plasticity. *Nature* 460: 705–710, 2009.

118. Corney DC, Flecken-Nikitin A, Godwin AK, Wang W, Nikitin AY. MicroRNA-34b and MicroRNA-34c are targets of p53 and cooperate in control of cell proliferation and adhesion-independent growth. *Cancer Res* 67: 8433–8438, 2007.

119. Correa-Medina M, Bravo-Egana V, Rosero S, Ricordi C, Edlund H, Diez J, Pastori RL. MicroRNA miR-7 is preferentially expressed in endocrine cells of the developing and adult human pancreas. *Gene Expr Patterns* 9: 193–199, 2009.

120. Costantini DL, Arruda EP, Agarwal P, Kim KH, Zhu Y, Zhu W, Lebel M, Cheng CW, Park CY, Pierce SA, Guerchicoff A, Pollevick GD, Chan TY, Kabir MG, Cheng SH, Husain M, Antzelevitch C, Srivastava D, Gross GJ, Hui CC, Backx PH, Bruneau BG. The homeodomain transcription factor Irx5 establishes the mouse cardiac ventricular repolarization gradient. *Cell* 123: 347–358, 2005.

121. Costinean S, Sandhu SK, Pedersen IM, Tili E, Trotta R, Perrotti D, Ciarlariello D, Neviani P, Harb J, Kauffman LR, Shidham A, Croce CM. Src homology 2 domain-containing inositol-5-phosphatase and CCAAT enhancer-binding protein beta are targeted by miR-155 in B cells of Emicro-MiR-155 transgenic mice. *Blood* 114: 1374–1382, 2009.

122. Costinean S, Zanesi N, Pekarsky Y, Tili E, Volinia S, Heerema N, Croce CM. Pre-B cell proliferation and lymphoblastic leukemia/high-grade lymphoma in E(mu)-miR155 transgenic mice. *Proc Natl Acad Sci USA* 103: 7024–7029, 2006.

123. Coulouarn C, Factor VM, Andersen JB, Durkin ME, Thorgeirsson SS. Loss of miR-122 expression in liver cancer correlates with suppression of the hepatic phenotype and gain of metastatic properties. *Oncogene* 28: 3526–3536, 2009.

124. Cowden Dahl KD, Dahl R, Kruijchak JN, Hudson LG. The epidermal growth factor receptor responsive miR-125a represses mesenchymal morphology in ovarian cancer cells. *Neoplasia* 11: 1208–1215, 2009.

125. Crawford M, Batte K, Yu L, Wu X, Nuovo GJ, Marsh CB, Otterson GA, Nana-Sinkam SP. MicroRNA 133B targets pro-survival molecules MCL-1 and BCL2L2 in lung cancer. Biochem Biophys Res Commun 388: 483–489, 2009.

126. Crawford M, Brawner E, Batte K, Yu L, Hunter MG, Otterson GA, Nuovo G, Marsh CB, Nana-Sinkam SP. MicroRNA-126 inhibits invasion in non-small cell lung carcinoma cell lines. Biochem Biophys Res Commun 373: 607–612, 2008.

127. Crist CG, Montarras D, Pallafacchina G, Rocancourt D, Cumano A, Conway SJ, Buckingham M. Muscle stem cell behavior is modified by microRNA-27 regulation of Pax3 expression. Proc Natl Acad Sci USA 106: 13383–13387, 2009.

128. Cuellar TL, Davis TH, Nelson PT, Loeb GB, Harfe BD, Ullian E, McManus MT. Dicer loss in striatal neurons produces behavioral and neuroanatomical phenotypes in the absence of neurodegeneration. Proc Natl Acad Sci USA 105: 5614–5619, 2008.

129. Curmi PA, Nogues C, Lachkar S, Carelle N, Gonthier MP, Sobel A, Lidereau R, Bieche I. Overexpression of stathmin in breast carcinomas points out to highly proliferative tumours. Br J Cancer 82: 142–150, 2000.

130. Dangi-Garimella S, Yun J, Eves EM, Newman M, Erkeland SJ, Hammond SM, Minn AJ, Rosner MR. Raf kinase inhibitory protein suppresses a metastasis signalling cascade involving LIN28 and let-7. EMBO J 28: 347–358, 2009.

131. Darnell DK, Kaur S, Stanislaw S, Konieczka JH, Yatskievych TA, Antin PB. MicroRNA expression during chick embryo development. Dev Dyn 235: 3156–3165, 2006.

132. Datta J, Kutay H, Nasser MW, Nuovo GJ, Wang B, Majumder S, Liu CG, Volinia S, Croce CM, Schmittgen TD, Ghoshal K, Jacob ST. Methylation mediated silencing of MicroRNA-I gene and its role in hepatocellular carcinogenesis. Cancer Res 68: 5049–5058, 2008.

133. Davis BN, Hilyard AC, Nguyen PH, Lagna G, Hata A. Induction of microRNA-221 by platelet-derived growth factor signaling is critical for modulation of vascular smooth muscle phenotype. J Biol Chem 284: 3728–3738, 2009.

134. Davis TH, Cuellar TL, Koch SM, Barker AJ, Harfe BD, McManus MT, Ullian EM. Conditional loss of Dicer disrupts cellular and tissue morphogenesis in the cortex and hippocampus. J Neurosci 28: 4322–4330, 2008.

135. De Klein A, van Kessel AG, Grosveld G, Bartram CR, Hagemeijer A, Bootsma D, Spurr NK, Heisterkamp N, Groffen J, Stephenson JR. A cellular oncogene is translocated to the Philadelphia chromosome in chronic myelocytic leukaemia. Nature 300: 765–767, 1982.

136. De Leeuw RJ, Davies JJ, Rosenwald A, Bebb G, Gascoyne RD, Dyer MJ, Staudt LM, Martinez-Climent JA, Lam WL. Comprehensive whole genome array CGH profiling of mantle cell lymphoma model genomes. Hum Mol Genet 13: 1827–1837, 2004.

137. De Pietri Tonelli D, Pulvers JN, Haffner C, Murchison EP, Hannon GJ, Huttner WB. miRNAs are essential for survival and differentiation of newborn neurons but not for expansion of neural progenitors during early neurogenesis in the mouse embryonic neocortex. Development 135: 3911–3921, 2008.

138. Deconinck AE, Potter AC, Tinsley JM, Wood SJ, Vater R, Young C, Metzinger L, Vincent A, Slater CR, Davies KE. Postsynaptic abnormalities at the neuromuscular junctions of utrophin-deficient mice. J Cell Biol 136: 883–894, 1997.

139. Denli AM, Tops BB, Plasterk RH, Ketting RF, Hannon GJ. Processing of primary microRNAs by the Microprocessor complex. Nature 432: 231–235, 2004.

140. Dews M, Homayouni A, Yu D, Murphy D, Sevignani C, Wentzel E, Furth EE, Lee WM, Enders GH, Mendell JT, Thomas-Tikhonenko A. Augmentation of tumor angiogenesis by a Myc-activated microRNA cluster. Nat Genet 38: 1060–1065, 2006.

141. Dews M, Homayouni A, Yu D, Murphy D, Sevignani C, Wentzel E, Furth EE, Lee WM, Enders GH, Mendell JT, Thomas-Tikhonenko A. Augmentation of tumor angiogenesis by a Myc-activated microRNA cluster. Nat Genet 30: 30, 2006.

142. Diaz-Toledano R, Ariza-Mateos A, Birk A, Martinez-Garcia B, Gomez J. In vitro characterization of a miR-122-sensitive double-helical switch element in the 5′ region of hepatitis C virus RNA. Nucleic Acids Res 37: 5498–5510, 2009.

143. Dillhoff M, Liu J, Frankel W, Croce C, Bloomston M. MicroRNA-21 is overexpressed in pancreatic cancer and a potential predictor of survival. J Gastrointest Surg 12: 2171–2176, 2008.

144. Ding Y, Chan CY, Lawrence CE. Sfold web server for statistical folding and rational design of nucleic acids. Nucleic Acids Res 32: W135–W141, 2004.

145. Dixon-McIver A, East P, Mein CA, Cazier JB, Molloy G, Chaplin T, Andrew Lister T, Young BD, Debernardi S. Distinctive patterns of microRNA expression associated with karyotype in acute myeloid leukaemia. PLoS One 3: 2008.

146. Dohner H, Stilgenbauer S, Benner A, Leupolt E, Krober A, Bullinger L, Dohner K, Bentz M, Lichter P. Genomic aberrations and survival in chronic lymphocytic leukemia. N Engl J Med 343: 1910–1916, 2000.

147. Dong S, Cheng Y, Yang J, Li J, Liu X, Wang X, Wang D, Krall TJ, Delphin ES, Zhang C. MicroRNA expression signature and the role of microRNA-21 in the early phase of acute myocardial infarction. J Biol Chem 25: 25, 2009.

148. Dore LC, Amigo JD, Dos Santos CO, Zhang Z, Gai X, Tobias JW, Yu D, Klein AM, Dorman C, Wu W, Hardison RC, Paw BH, Weiss MJ. A GATA-1-regulated microRNA locus essential for erythropoiesis. Proc Natl Acad Sci USA 105: 3333–3338, 2008.

149. Dorsett Y, McBride KM, Jankovic M, Gazumyan A, Thai TH, Robbiani DF, Di Virgilio M, San-Martin BR, Heidkamp G, Schwicker T, Eisenreich T, Rajewsky K, Nussen-zweig MC. MicroRNA-155 suppresses activation-induced cytidine deaminase-mediated Myc-Igh translocation. Immunity 28: 630–638, 2008.

150. Duan W, Gao L, Wu X, Wang L, Nana-Sinkam SP, Otterson GA, Villalona-Calero MA. MicroRNA-34a is an important component of PRIMA-1 induced apoptotic network in human lung cancer cells. Int J Cancer 17: 17, 2009.

151. Eberhart CE, Coffey RJ, Radhika A, Giardiello FM, Ferrenbach S, DuBois RN. Upregulation of cyclooxygenase 2 gene expression in human colorectal adenomas and adenocarcinomas. Gastroenterology 107: 1183–1188, 1994.

152. Ebert MS, Neilson JR, Sharp PA. MicroRNA sponges: competitive inhibitors of small RNAs in mammalian cells. Nat Methods 4: 721–726, 2007.

153. Ebert PJ, Jiang S, Xie J, Li QJ, Davis MM. An endogenous positively selecting peptide enhances mature T cell responses and becomes an autoantigen in the absence of microRNA miR-181a. Nat Immunol 10: 1162–1169, 2009.

154. Eckle T, Kohler D, Lehmann R, El Kasmi K, Eltzschig HK. Hypoxia-inducible factor-1 is central to cardioprotection: a new paradigm for ischemic preconditioning. Circulation 118: 166–175, 2008.

155. Eis PS, Tam W, Sun L, Chadburn A, Li Z, Gomez MF, Lund E, Dahlberg JE. Accumulation of miR-155 and BIC RNA in human B cell lymphomas. Proc Natl Acad Sci USA 102: 3627–3632, 2005.

156. El Ouaamari A, Baroukh N, Martens GA, Lebrun P, Pipeleers D, van Obberghen E. miR-375 targets 3′-phosphoinositide-dependent protein kinase-1 and regulates glucose-induced biological responses in pancreatic beta-cells. Diabetes 57: 2708–2717, 2008.

157. El-Husseini-Ael D, Schnell E, Dakoji S, Sweeney N, Zhou Q, Prange O, Gauthier-Campbell C, Aguilera-Moreno A, Nicoll RA, Bredt DS. Synaptic strength regulated by palmitate cycling on PSD-95. Cell 108: 849–863, 2002.

158. El-Osta A. DNMT cooperativity: the developing links between methylation, chromatin structure and cancer. Bioessays 25: 1071–1084, 2003.

159. Elia L, Contu R, Quintavalle M, Varrone F, Chimenti C, Russo MA, Cimino V, De Marinis L, Frustaci A, Catalucci D, Condorelli G. Reciprocal regulation of microRNA-1 and insulin-like growth factor-1 signal transduction cascade in cardiac and skeletal muscle in physiological and pathological conditions. Circulation 120: 2377–2385, 2009.

160. Elia L, Quintavalle M, Zhang J, Contu R, Cossu L, Latronico MV, Peterson KL, Indolfi C, Catalucci D, Chen J, Courtneidge SA, Condorelli G. The knockout of miR-143 and -145 alters smooth muscle cell maintenance and vascular homeostasis in mice: correlates with human disease. Cell Death Differ 16: 1590–1598, 2009.

161. Elmen J, Lindow M, Schutz S, Lawrence M, Petri A, Obad S, Lindholm M, Hedtjarn M, Hansen HF, Berger U, Gullans S, Kearney P, Sarnow P, Straarup EM, Kauppinen S. LNA-mediated microRNA silencing in non-human primates. Nature 452: 896–899, 2008.

162. Elmen J, Lindow M, Silahtaroglu A, Bak M, Christensen M, Lind-Thomsen A, Hedtjarn M, Hansen JB, Hansen HF, Straarup EM, McCullagh K, Kearney P, Kauppinen S. Antagonism of microRNA-122 in mice by systemically administered LNA-antimiR leads to up-regulation of a large set of predicted target mRNAs in the liver. Nucleic Acids Res 36: 1153–1162, 2008.

MicroRNAs IN DEVELOPMENT AND DISEASE

163. Ender C, Krek A, Friedlander MR, Beitzinger M, Weinmann L, Chen W, Pfeffer S, Rajewsky N, Meister G. A human snoRNA with microRNA-like functions. Mol Cell 32: 519–528, 2008.

164. Epis MR, Giles KM, Barker A, Kendrick TS, Leedman PJ. miR-331–3p regulates ERBB-2 expression and androgen receptor signaling in prostate cancer. J Biol Chem 284: 24696–24704, 2009.

165. Esau C, Davis S, Murray SF, Yu XX, Pandey SK, Pear M, Watts L, Booten SL, Graham M, McKay R, Subramaniam A, Propp S, Lollo BA, Freier S, Bennett CF, Bhanot S, Monia BP. miR-122 regulation of lipid metabolism revealed by in vivo antisense targeting. Cell Metab 3: 87–98, 2006.

166. Esquela-Kerscher A, Trang P, Wiggins JF, Patrawala L, Cheng A, Ford L, Weidhaas JB, Brown D, Bader AG, Slack FJ. The let-7 microRNA reduces tumor growth in mouse models of lung cancer. Cell Cycle 7: 759–764, 2008.

167. Fabbri M, Garzon R, Cimmino A, Liu Z, Zanesi N, Callegari E, Liu S, Alder H, Costinean S, Fernandez-Cymering C, Volinia S, Guler G, Morrison CD, Chan KK, Marcucci G, Calin GA, Huebner K, Croce CM. MicroRNA-29 family reverts aberrant methylation in lung cancer by targeting DNA methyltransferases 3A and 3B. Proc Natl Acad Sci USA 104: 15805–15810, 2007.

168. Fabian MR, Mathonnet G, Sundermeier T, Mathys H, Zipprich JT, Svitkin YV, Rivas F, Jinek M, Wohlschlegel J, Doudna JA, Chen CY, Shyu AB, Yates JR ,3rd, Hannon GJ, Filipowicz W, Duchaine TF, Sonenberg N. Mammalian miRNA RISC recruits CAFI and PABP to affect PABP-dependent deadenylation. Mol Cell 35: 868–880, 2009.

169. Faggad A, Budczies J, Tchernitsa O, Darb-Esfahani S, Sehouli J, Muller BM, Wirtz R, Chekerov R, Weichert W, Sinn B, Mucha C, Elwali NE, Schafer R, Dietel M, Denkert C. Prognostic significance of Dicer expression in ovarian cancer-link to global microRNA changes and oestrogen receptor expression. J Pathol 220: 382–391, 2010.

170. Farh KK, Grimson A, Jan C, Lewis BP, Johnston WK, Lim LP, Burge CB, Bartel DP. The widespread impact of mammalian MicroRNAs on mRNA repression and evolution. Science 310: 1817–1821, 2005.

171. Fasanaro P, D'Alessandra Y, Di Stefano V, Melchionna R, Romani S, Pompilio G, Capogrossi MC, Martelli F. MicroRNA-210 modulates endothelial cell response to hypoxia and inhibits the receptor tyrosine kinase ligand Ephrin-A3. J Biol Chem 283: 15878–15883, 2008.

172. Fazi F, Racanicchi S, Zardo G, Starnes LM, Mancini M, Travaglini L, Diverio D, Ammatuna E, Cimino G, Lo-Coco F, Grignani F, Nervi C. Epigenetic silencing of the myelopoiesis regulator microRNA-223 by the AML1/ETO oncoprotein. Cancer Cell 12: 457–466, 2007.

173. Fazi F, Rosa A, Fatica A, Gelmetti V, De Marchis ML, Nervi C, Bozzoni I. A minicircuit comprised of microRNA-223 and transcription factors NFI-A and C/EBPalpha regulates human granulopoiesis. Cell 123: 819–831, 2005.

174. Felicetti F, Errico MC, Bottero L, Segnalini P, Stoppacciaro A, Biffoni M, Felli N, Mattia G, Petrini M, Colombo MP, Peschle C, Care A. The promyelocytic leukemia zinc finger-microRNA-221/-222 pathway controls melanoma progression through multiple oncogenic mechanisms. Cancer Res 68: 2745–2754, 2008.

175. Felli N, Fontana L, Pelosi E, Botta R, Bonci D, Facchiano F, Liuzzi F, Lulli V, Morsilli O, Santoro S, Valtieri M, Calin GA, Liu CG, Sorrentino A, Croce CM, Peschle C. MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation. Proc Natl Acad Sci USA 102: 18081–18086, 2005.

176. Feng J, Sun G, Yan J, Noltner K, Li W, Buzin CH, Longmate J, Heston LL, Rossi J, Sommer SS. Evidence for X-chromosomal schizophrenia associated with microRNA alterations. PLoS One 4: 2009.

177. Feng Y, Gutekunst CA, Eberhart DE, Yi H, Warren ST, Hersch SM. Fragile X mental retardation protein: nucleocytoplasmic shuttling and association with somatodendritic ribosomes. J Neurosci 17: 1539–1547, 1997.

178. Ferracini R, Olivero M, Di Renzo MF, Martano M, De Giovanni C, Nanni P, Basso G, Scotlandi K, Lollini PL, Comoglio PM. Retrogenic expression of the MET proto-oncogene correlates with the invasive phenotype of human rhabdomyosarcomas. Oncogene 12: 1697–1705, 1996.

179. Ferretti E, De Smaele E, Miele E, Laneve P, Po A, Pelloni M, Paganelli A, Di Marcotullio L, Caffarelli E, Screpanti I, Bozzoni I, Gulino A. Concerted microRNA control of Hedgehog signalling in cerebellar neuronal progenitor and tumour cells. EMBO J 27: 2616–2627, 2008.

180. Fish JE, Santoro MM, Morton SU, Yu S, Yeh RF, Wythe JD, Ivey KN, Bruneau BG, Stainier DY, Srivastava D. miR-126 regulates angiogenic signaling and vascular integrity. Dev Cell 15: 272–284, 2008.

181. Fontana L, Pelosi E, Greco P, Racanicchi S, Testa U, Liuzzi F, Croce CM, Brunetti E, Grignani F, Peschle C. MicroRNAs 17–5p-20a-106a control monocytopenies through AML1 targeting and M-CSF receptor upregulation. Nat Cell Biol 9: 775–787, 2007.

182. Ford LP. Using synthetic miRNA mimics for diverting cell fate: a possibility of miRNA-based therapeutics? Leuk Res 30: 511–513, 2006.

183. Fornari F, Gramantieri L, Ferracin M, Veronese A, Sabbioni S, Calin GA, Grazi GL, Giovannini C, Croce CM, Bolondi L, Negrini M. MiR-221 controls CDKN1C/p57 and CDKN1B/p27 expression in human hepatocellular carcinoma. Oncogene 27: 5651–5661, 2008.

184. Frankel LB, Christoffersen NR, Jacobsen A, Lindow M, Krogh A, Lund AH. Programmed cell death 4 (PDCD4) is an important functional target of the microRNA miR-21 in breast cancer cells. J Biol Chem 283: 1026–1033, 2008.

185. Friedman LM, Dror AA, Mor E, Tenne T, Toren G, Satoh T, Biesemeier DJ, Shomron N, Fekete DM, Hornstein E, Avraham KB. MicroRNAs are essential for development and function of inner ear hair cells in vertebrates. Proc Natl Acad Sci USA 106: 7915–7920, 2009.

186. Friedman RC, Farh KK, Burge CB, Bartel DP. Most mammalian mRNAs are conserved targets of microRNAs. Genome Res 19: 92–105, 2009.

187. Fu YF, Du TT, Dong M, Zhu KY, Jing CB, Zhang Y, Wang L, Fan HB, Chen Y, Jin Y, Yue GP, Chen SJ, Chen Z, Huang QH, Jing Q, Deng M, Liu TX. Mir-144 selectively regulates embryonic alpha-hemoglobin synthesis during primitive erythropoiesis. Blood 113: 1340–1349, 2009.

188. Fujita Y, Kojima K, Hamada N, Ohhashi R, Akao Y, Nozawa Y, Deguchi T, Ito M. Effects of miR-34a on cell growth and chemoresistance in prostate cancer PC3 cells. Biochem Biophys Res Commun 377: 114–119, 2008.

189. Fujiwara Y, Browne CP, Cunniff K, Goff SC, Orkin SH. Arrested development of embryonic red cell precursors in mouse embryos lacking transcription factor GATA-1. Proc Natl Acad Sci USA 93: 12355–12358, 1996.

190. Fukao T, Fukuda Y, Kiga K, Sharif J, Hino K, Enomoto Y, Kawamura A, Nakamura K, Takeuchi T, Tanabe M. An evolutionarily conserved mechanism for microRNA-223 expression revealed by microRNA gene profiling. Cell 129: 617–631, 2007.

191. Fulci V, Chiaretti S, Goldoni M, Azzalin G, Carucci N, Tavolaro S, Castellano L, Magrelli A, Citarella F, Messina M, Maggio R, Peragine N, Santangelo S, Mauro FR, Landgraf P, Tuschl T, Weir DB, Chien M, Russo JJ, Ju J, Sheridan R, Sander C, Zavolan M, Guarini A, Foa R, Macino G. Quantitative technologies establish a novel microRNA profile of chronic lymphocytic leukemia. Blood 109: 4944–4951, 2007.

192. Gabriely G, Wurdinger T, Kesari S, Esau CC, Burchard J, Linsley PS, Krichevsky AM. MicroRNA 21 promotes glioma invasion by targeting matrix metalloproteinase regulators. Mol Cell Biol 28: 5369–5380, 2008.

193. Galardi S, Mercatelli N, Giorda E, Massalini S, Frajese GV, Ciafre SA, Farace MG. miR-221 and miR-222 expression affects the proliferation potential of human prostate carcinoma cell lines by targeting p27Kip1. J Biol Chem 282: 23716–23724, 2007.

194. Gandellini P, Folini M, Longoni N, Pennati M, Binda M, Colecchia M, Salvioni R, Supino R, Moretti R, Limonta P, Valdagni R, Daidone MG, Zaffaroni N. miR-205 exerts tumor-suppressive functions in human prostate through down-regulation of protein kinase Cε. Cancer Res 69: 2287–2295, 2009.

195. Garofalo M, Di Leva G, Romano G, Nuovo G, Suh SS, Ngankeu A, Taccioli C, Pichiorri F, Alder H, Secchiero P, Gasparini P, Gonelli A, Costinean S, Acunzo M, Condorelli G, Croce CM, Condorelli G. MicroRNA signatures of TRAIL resistance in human non-small cell lung cancer. Oncogene 27: 3845–3855, 2008.

196. Garofalo M, Quintavalle C, Di Leva G, Zanca C, Romano G, Taccioli C, Liu CG, Croce CM, Condorelli G. MicroRNA signatures of TRAIL resistance in human non-small cell lung cancer. Oncogene 27: 3845–3855, 2008.

197. Garzia L, Andolfi I, Cusanelli E, Marino N, Petrosino G, De Martino D, Esposito V, Galeone A, Navas L, Esposito S, Gargiulo S, Fattet S, Donofrio V, Cinalli G, Brunetti A, Vecchio LD, Northcott PA, Delattre O, Taylor MD, Iolascon A, Zollo M. MicroRNA-199b-5p impairs cancer stem cells through negative regulation of HES1 in medulloblastoma. PLoS One 4: e4998, 2009.

DANISH SAYED AND MAHA ABDELLATIF

198. Garzon R, Garofalo M, Martelli MP, Briesewitz R, Wang L, Fernandez-Cymering C, Volinia S, Liu CG, Schnittger S, Haferlach T, Liso A, Diverio D, Mancini M, Meloni G, Foa R, Martelli MF, Mecucci C, Croce CM, Falini B. Distinctive microRNA signature of acute myeloid leukemia bearing cytoplasmic mutated nucleophosmin. *Proc Natl Acad Sci USA* 105: 3945–3950, 2008.

199. Garzon R, Heaphy CE, Havelange V, Fabbri M, Volinia S, Tsao T, Zanesi N, Kornblau SM, Marcucci G, Calin GA, Andreeff M, Croce CM. MicroRNA 29b functions in acute myeloid leukemia. *Blood* 22: 22, 2009.

200. Garzon R, Liu S, Fabbri M, Liu Z, Heaphy CE, Callegari E, Schwind S, Pang J, Yu J, Muthusamy N, Havelange V, Volinia S, Blum W, Rush LJ, Perrotti D, Andreeff M, Bloomfield CD, Byrd JC, Chan K, Wu LC, Croce CM, Marcucci G. MicroRNA-29b induces global DNA hypomethylation and tumor suppressor gene reexpression in acute myeloid leukemia by targeting directly DNMT3A and 3B and indirectly DNMT1. *Blood* 113: 6411–6418, 2009.

201. Garzon R, Volinia S, Liu CG, Fernandez-Cymering C, Palumbo T, Pichiorri F, Fabbri M, Coombes K, Alder H, Nakamura T, Flomenberg N, Marcucci G, Calin GA, Kornblau SM, Kantarjian H, Bloomfield CD, Andreeff M, Croce CM. MicroRNA signatures associated with cytogenetics and prognosis in acute myeloid leukemia. *Blood* 111: 3183–3189, 2008.

202. Gatto G, Rossi A, Rossi D, Kroening S, Bonatti S, Mallardo M. Epstein-Barr virus latent membrane protein I trans-activates miR-155 transcription through the NF-kappaB pathway. *Nucleic Acids Res* 36: 6608–6619, 2008.

203. Gebeshuber CA, Zatloukal K, Martinez J. miR-29a suppresses tristetraprolin, which is a regulator of epithelial polarity and metastasis. *EMBO Rep* 10: 400–405, 2009.

204. Ghosh R, Gu G, Tillman E, Yuan J, Wang Y, Fazli L, Rennie PS, Kasper S. Increased expression and differential phosphorylation of stathmin may promote prostate cancer progression. *Prostate* 67: 1038–1052, 2007.

205. Gibbons DL, Lin W, Creighton CJ, Rizvi ZH, Gregory PA, Goodall GJ, Thilaganathan N, Du L, Zhang Y, Pertsemlidis A, Kurie JM. Contextual extracellular cues promote tumor cell EMT and metastasis by regulating miR-200 family expression. *Genes Dev* 23: 2140–2151, 2009.

206. Gillies JK, Lorimer IA. Regulation of p27Kip1 by miRNA 221/222 in glioblastoma. *Cell Cycle* 6: 2005–2009, 2007.

207. Gironella M, Seux M, Xie MJ, Cano C, Tomasini R, Gommeaux J, Garcia S, Nowak J, Yeung ML, Jeang KT, Chaix A, Fazli L, Motoo Y, Wang Q, Rocchi P, Russo A, Gleave M, Dagorn JC, Iovanna JL, Carrier A, Pebusque MJ, Dusetti NJ. Tumor protein 53-induced nuclear protein I expression is repressed by miR-155, its restoration inhibits pancreatic tumor development. *Proc Natl Acad Sci USA* 104: 16170–16175, 2007.

208. Godlewski J, Nowicki MO, Bronisz A, Williams S, Otsuki A, Nuovo G, Raychaudhury A, Newton HB, Chiocca EA, Lawler S. Targeting of the Bmi-1 oncogene/stem cell renewal factor by microRNA-128 inhibits glioma proliferation and self-renewal. *Cancer Res* 68: 9125–9130, 2008.

209. Godshalk SE, Bhaduri-McIntosh S, Slack FJ. Epstein-Barr virus-mediated dysregulation of human microRNA expression. *Cell Cycle* 7: 3595–3600, 2008.

210. Gottwein E, Cullen BR. A human herpesvirus microRNA inhibits p21 expression and attenuates p21-mediated cell cycle arrest. *J Virol* 84: 5229–5237, 2010.

211. Gottwein E, Mukherjee N, Sachse C, Frenzel C, Majoros WH, Chi JT, Braich R, Manoharan M, Soutschek J, Ohler U, Cullen BR. A viral microRNA functions as an orthologue of cellular miR-155. *Nature* 450: 1096–1099, 2007.

212. Gramantieri L, Ferracin M, Fornari F, Veronese A, Sabbioni S, Liu CG, Calin GA, Giovannini C, Ferrazzi E, Grazi GL, Croce CM, Bolondi L, Negrini M. Cyclin G1 is a target of miR-122a, a microRNA frequently down-regulated in human hepatocellular carcinoma. *Cancer Res* 67: 6092–6099, 2007.

213. Gramantieri L, Fornari F, Ferracin M, Veronese A, Sabbioni S, Calin GA, Grazi GL, Croce CM, Bolondi L, Negrini M. MicroRNA-221 targets Bmf in hepatocellular carcinoma and correlates with tumor multifocality. *Clin Cancer Res* 15: 5073–5081, 2009.

214. Greco S, De Simone M, Colussi C, Zaccagnini G, Fasanaro P, Pescatori M, Cardani R, Perbellini R, Isaia E, Sale P, Meola G, Capogrossi MC, Gaetano C, Martelli F. Common micro-RNA signature in skeletal muscle damage and regeneration induced by Duchenne muscular dystrophy and acute ischemia. *FASEB J* 23: 3335–3346, 2009.

215. Gregory PA, Bert AG, Paterson EL, Barry SC, Tsykin A, Farshid G, Vadas MA, Khew-Goodall Y, Goodall GJ. The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1. *Nat Cell Biol* 10: 593–601, 2008.

216. Gregory RI, Yan KP, Amuthan G, Chendrimada T, Doratotaj B, Cooch N, Shiekhattar R. The microprocessor complex mediates the genesis of microRNAs. *Nature* 432: 235–240, 2004.

217. Griffiths-Jones S. The microRNA registry. *Nucleic Acids Res* 32: D109–D111, 2004.

218. Griffiths-Jones S, Grocock RJ, van Dongen S, Bateman A, Enright AJ. miRBase: microRNA sequences, targets and gene nomenclature. *Nucleic Acids Res* 34: D140–D144, 2006.

219. Griffiths-Jones S, Saini HK, van Dongen S, Enright AJ. miRBase: tools for microRNA genomics. *Nucleic Acids Res* 36: 8, 2008.

220. Grimson A, Farh KK, Johnston WK, Garrett-Engele P, Lim LP, Bartel DP. MicroRNA targeting specificity in mammals: determinants beyond seed pairing. *Mol Cell* 27: 91–105, 2007.

221. Guimaraes-Sternberg C, Meerson A, Shaked I, Soreq H. MicroRNA modulation of megakaryoblast fate involves cholinergic signaling. *Leuk Res* 30: 583–595, 2006.

222. Guo C, Sah JF, Beard L, Willson JK, Markowitz SD, Guda K. The noncoding RNA, miR-126, suppresses the growth of neoplastic cells by targeting phosphatidylinositol 3-kinase signaling and is frequently lost in colon cancers. *Genes Chromosomes Cancer* 47: 939–946, 2008.

223. Guo LM, Pu Y, Han Z, Liu T, Li YX, Liu M, Li X, Tang H. MicroRNA-9 inhibits ovarian cancer cell growth through regulation of NF-kappaBI. *FEBS Lett* 276: 5537–5546, 2009.

224. Guo X, Wu Y, Hartley RS. MicroRNA-125a represses cell growth by targeting HuR in breast cancer. *RNA Biol* 6: 2009.

225. Gusella JF, MacDonald ME. Huntington’s disease. *Semin Cell Biol* 6: 21–28, 1995.

226. Guttila IK, White BA. Coordinate regulation of FOXO1 by miR-27a, miR-96, and miR-182 in breast cancer cells. *J Biol Chem* 284: 23204–23216, 2009.

227. Haasch D, Chen YW, Reilly RM, Chiou XG, Koterski S, Smith ML, Kroeger P, McWeeny K, Halbert DN, Mollison KW, Djuric SW, Trevillyan JM. T cell activation induces a noncoding RNA transcript sensitive to inhibition by immunosuppressant drugs and encoded by the proto-oncogene, BIC. *Cell Immunol* 217: 78–86, 2002.

228. Habbe N, Koorstra JB, Mendell JT, Offerhaus GJ, Ryu JK, Feldmann G, Mullendore ME, Goggins MG, Hong SM, Maitra A. MicroRNA miR-155 is a biomarker of early pancreatic neoplasia. *Cancer Biol Ther* 8: 340–346, 2009.

229. Han J, Lee Y, Yeom KH, Kim YK, Jin H, Kim VN. The Drosha-DGCR8 complex in primary microRNA processing. *Genes Dev* 18: 3016–3027, 2004.

230. Han J, Lee Y, Yeom KH, Nam JW, Heo I, Rhee JK, Sohn SY, Cho Y, Zhang BT, Kim VN. Molecular basis for the recognition of primary microRNAs by the Drosha-DGCR8 complex. *Cell* 125: 887–901, 2006.

231. Hanashima C, Li SC, Shen L, Lai E, Fishell G. Foxgl1 suppresses early cortical cell fate. *Science* 303: 56–59, 2004.

232. Hansen A, Henderson S, Lagos D, Nikitenko L, Coulter E, Roberts S, Gratrix F, Plaisance K, Renne R, Bower M, Kellam P, Boshoff C. KSHV-encoded miRNAs target MAF to induce endothelial cell reprogramming. *Genes Dev* 24: 195–205, 2010.

233. Hansen T, Olsen L, Lindow M, Jakobsen KD, Ullum H, Jonsson E, Andreassen OA, Djurovic S, Melle I, Agartz I, Hall H, Timm S, Wang AG, Werge T. Brain expressed microRNAs implicated in schizophrenia etiology. *PLoS One* 2: 2007.

234. Harris KS, Zhang Z, McManus MT, Harfe BD, Sun X. Dicer function is essential for lung epithelium morphogenesis. *Proc Natl Acad Sci USA* 103: 2208–2213, 2006.

235. Harris TA, Yamakuchi M, Ferlito M, Mendell JT, Lowenstein CJ. MicroRNA-126 regulates endothelial expression of vascular cell adhesion molecule I. *Proc Natl Acad Sci USA* 105: 1516–1521, 2008.

236. Harvey SJ, Jarad G, Cunningham J, Goldberg S, Schermer B, Harfe BD, McManus MT, Benzing T, Miner JH. Podocyte-specific deletion of dicer alters cytoskeletal dynamics and causes glomerular disease. *J Am Soc Nephrol* 19: 2150–2158, 2008.

237. Hausenloy DJ, Tsang A, Mocanu MM, Yellon DM. Ischemic preconditioning protects by activating prosurvival kinases at reperfusion. Am J Physiol Heart Circ Physiol 288: H9, 2005.

238. Hayashita Y, Osada H, Tatematsu Y, Yamada H, Yanagisawa K, Tomida S, Yatabe Y, Kawahara K, Sekido Y, Takahashi T. A polycistronic microRNA cluster, miR-17–92, is overexpressed in human lung cancers and enhances cell proliferation. Cancer Res 65: 9628–9632, 2005.

239. He L, He X, Lim LP, de Stanchina E, Xuan Z, Liang Y, Xue W, Zender L, Magnus J, Ridzon D, Jackson AL, Linsley PS, Chen C, Lowe SW, Cleary MA, Hannon GJ. A microRNA component of the p53 tumour suppressor network. Nature 447: 1130–1134, 2007.

240. He L, Thomson JM, Hemann MT, Hernando-Monge E, Mu D, Goodson S, Powers S, Cordon-Cardo C, Lowe SW, Hannon GJ, Hammond SM. A microRNA polycistron as a potential human oncogene. Nature 435: 828–833, 2005.

241. Hebert SS, Horre K, Nicolai L, Papadopoulou AS, Mandemakers W, Silva taroglu AN, Kauppinen S, Delacourte A, De Strooper B. Loss of microRNA cluster miR-29a/b-1 in sporadic Alzheimer’s disease correlates with increased BACE1/beta-secretase expression. Proc Natl Acad Sci USA 105: 6415–6420, 2008.

242. Heisterkamp N, Stephenson JR, Groffen J, Hansen PF, de Klein A, Bartram CR, Grosfeld G. Localization of the c-abl oncogene adjacent to a translocation break point in chronic myelocytic leukaemia. Nature 306: 239–242, 1983.

243. Helgason CD, Kalberer CP, Damen JE, Chappel SM, Pineault N, Krystal G, Humphries RK. A dual role for Src homology 2 domain-containing inositol-5-phosphatase (SHIP) in immunity: aberrant development and enhanced function of b lymphocytes in ship –/– mice. J Exp Med 191: 781–794, 2000.

244. Henke JL, Goergen D, Zheng J, Song Y, Schuttler CG, Fehr C, Junemann C, Niepmann M. microRNA-122 stimulates translation of hepatitis C virus RNA. EMBO J 27: 3300–3310, 2008.

245. Ho J, Ng KH, Rosen S, Dostal A, Gregory RI, Kreidberg JA. Podocyte-specific loss of functional microRNAs leads to rapid glomerular and tubular injury. J Am Soc Nephrol 19: 2069–2075, 2008.

246. Hock J, Weinmann L, Ender C, Rudel S, Kremmer E, Raabe M, Urlaub H, Meister G. Proteomic and functional analysis of Argonaute-containing mRNA-protein complexes in human cells. EMBO Rep 8: 1052–1060, 2007.

247. Hossain A, Kuo MT, Saunders GF. Mir-17–5p regulates breast cancer cell proliferation by inhibiting translation of AIB1 mRNA. Mol Cell Biol 28: 28, 2006.

248. Hu X, Macdonald DM, Huettner PC, Feng Z, El Naqa IM, Schwarz JK, Mutch DG, Grigsby PW, Powell SN, Wang X. A miR-200 microRNA cluster as prognostic marker in advanced ovarian cancer. Gynecol Oncol 114: 457–464, 2009.

249. Huang J, Wang F, Argyris E, Chen K, Liang Z, Tian H, Huang W, Squires K, Verlinghieri G, Zhang H. Cellular microRNAs contribute to HIV-1 latency in resting primary CD4+T lymphocytes. Nat Med 13: 1241–1247, 2007.

250. Huang Q, Gumireddy K, Schrier M, le Sage C, Nagel R, Nair S, Egan DA, Li A, Huang G, Klein-Szanto AJ, Gimotty PA, Katsaros D, Coukos G, Zhang L, Pure E, Agami R. The microRNAs miR-373 and miR-520c promote tumour invasion and metastasis. Nat Cell Biol 10: 202–210, 2008.

251. Huang TH, Wu F, Loeb GB, Hsu R, Heidersbach A, Brincat A, Horiuchi D, Lebbink RJ, Mo YY, Goga A, McManus MT. Up-regulation of miR-21 by HER2/neu signaling promotes cell invasion. J Biol Chem 284: 18515–18524, 2009.

252. Hubscher U, Kuenzle CC, Spadari S. Variation of DNA polymerases-alpha, -beta, and -gamma during perinatal tissue growth and differentiation. Nucleic Acids Res 4: 2917–2929, 1977.

253. Humphreys DT, Westman BJ, Martin DI, Preiss T. MicroRNAs control translation initiation by inhibiting eukaryotic initiation factor 4E/cap and poly(A) tail function. Proc Natl Acad Sci USA 102: 16961–16966, 2005.

254. Hurst DR, Edmonds MD, Scott GK, Benz CC, Vaidya KS, Welch DR. Breast cancer metastasis suppressor 1 up-regulates miR-146, which suppresses breast cancer metastasis. Cancer Res 69: 1279–1283, 2009.

255. Hurst DR, Xie Y, Vaidya KS, Mehta A, Moore BP, Accavitti-Loper MA, Samant RS, Saxena R, Silveira AC, Welch DR. Alterations of BRMS1-ARID4A interaction modify

gene expression but still suppress metastasis in human breast cancer cells. J Biol Chem 283: 7438–7444, 2008.

256. Hurteau GJ, Carlson JA, Spivack SD, Brock GJ. Overexpression of the microRNA hsa-miR-200c leads to reduced expression of transcription factor 8 and increased expression of E-cadherin. Cancer Res 67: 7972–7976, 2007.

257. Huse JT, Brennan C, Hambardzumyan D, Wee B, Pena J, Rouhanifard SH, Sohn-Lee C, le Sage C, Agami R, Tuschl T, Holland EC. The PTEN-regulating microRNA miR-26a is amplified in high-grade glioma and facilitates gliomagenesis in vivo. Genes Dev 23: 1327–1337, 2009.

258. Hutvagner G, Simard MJ, Mello CC, Zamore PD. Sequence-specific inhibition of small RNA function. PLoS Biol 2: 24, 2004.

259. Hutvagner G, Zamore PD. A microRNA in a multiple-turnover RNAi enzyme complex. Science 297: 2056–2060, 2002.

260. Hutzinger R, Feederle R, Mrazek J, Schiefermeier N, Balwierz PJ, Zavolan M, Polacek N, Delecluse HJ, Huttenhofer A. Expression and processing of a small nucleolar RNA from the Epstein-Barr virus genome. PLoS Pathog 5: 14, 2009.

261. Ikeda S, He A, Kong SW, Lu J, Bejar R, Bodyak N, Lee KH, Ma Q, Kang PM, Golub TR, Pu WT. MicroRNA-1 negatively regulates expression of the hypertrophy-associated calmodulin and Mef2a genes. Mol Cell Biol 29: 2193–2204, 2009.

262. Ikeda S, Kong SW, Lu J, Bisping E, Zhang H, Allen PD, Golub TR, Pieske B, Pu WT. Altered microRNA expression in human heart disease. Physiol Genomics 31: 367–373, 2007.

263. Iliopoulos D, Hirsch HA, Struhl K. An epigenetic switch involving NF-kappaB, Lin28, Let-7 MicroRNA, and IL6 links inflammation to cell transformation. Cell 139: 693–706, 2009.

264. Inomata M, Tagawa H, Guo YM, Kameoka Y, Takahashi N, Sawada K. MicroRNA-17–92 down-regulates expression of distinct targets in different B-cell lymphoma subtypes. Blood 113: 396–402, 2009.

265. Inoue H, Kato R, Fukuyama S, Nonami A, Taniguchi K, Matsumoto K, Nakano T, Tsuda M, Matsumura M, Kubo M, Ishikawa F, Moon BG, Takatsu K, Nakanishi Y, Yoshimura A. Sprd-1 negatively regulates allergen-induced airway eosinophilia and hyperresponsiveness. J Exp Med 201: 73–82, 2005.

266. Iorio MV, Casalini P, Piovan C, Di Leva G, Merlo A, Triulzi T, Menard S, Croce CM, Tagliabue E. microRNA-205 regulates HER3 in human breast cancer. Cancer Res 69: 2195–2200, 2009.

267. Iorio MV, Ferracin M, Liu CG, Veronese A, Spizzo R, Sabbioni S, Magri E, Pedriali M, Fabbri M, Campiglio M, Menard S, Palazzo JP, Rosenberg A, Musiani P, Volinia S, Nenci I, Calin GA, Querzoli P, Negrini M, Croce CM. MicroRNA gene expression deregulation in human breast cancer. Cancer Res 65: 7065–7070, 2005.

268. Iwasaki S, Kawamata T, Tomari Y. Drosophila argonaute1 and argonaute2 employ distinct mechanisms for translational repression. Mol Cell 34: 58–67, 2009.

269. Izumo S, Nadal-Ginard B, Mahdavi V. Protooncogene induction and reprogramming of cardiac gene expression produced in pressure overload. Proc Natl Acad Sci USA 85: 339–343, 1988.

270. Janz A, Sevignani C, Kenyon K, Ngo CV, Thomas-Tikhonenko A. Activation of the myc oncoprotein leads to increased turnover of thrombospondin-1 mRNA. Nucleic Acids Res 28: 2268–2275, 2000.

271. Jegga AG, Chen J, Gowrisankar S, Deshmukh MA, Gudivada R, Kong S, Kaimal V, Aronow BJ. GenomeTrafac: a whole genome resource for the detection of transcription factor binding site clusters associated with conventional and microRNA encoding genes conserved between mouse and human gene orthologs. Nucleic Acids Res 35: 18, 2007.

272. Jeng SF, Rau CS, Liliang PC, Wu CJ, Lu TH, Chen YC, Lin CJ, Hsieh CH. Profiling muscle-specific MicroRNA expression after peripheral denervation and re-innervation in a rat model. J Neurotrauma 8: 8, 2009.

273. Jensen MR, Factor VM, Fantozzi A, Helin K, Huh CG, Thorgeirsson SS. Reduced hepatic tumor incidence in cyclin G1-deficient mice. Hepatology 37: 862–870, 2003.

274. Ji Q, Hao X, Zhang M, Tang W, Yang M, Li L, Xiang D, Desano JT, Bommer GT, Fan D, Fearon ER, Lawrence TS, Xu L. MicroRNA miR-34 inhibits human pancreatic cancer tumor-initiating cells. PLoS One 4: 2009.

DANISH SAYED AND MAHA ABDELLATIF

275. Ji R, Cheng Y, Yue J, Yang J, Liu X, Chen H, Dean DB, Zhang C. MicroRNA expression signature and antisense-mediated depletion reveal an essential role of MicroRNA in vascular neointimal lesion formation. *Circ Res* 100: 1579–1588, 2007.

276. Jin B, Yao B, Li JL, Fields CR, Delmas AL, Liu C, Robertson KD. DNMT1 and DNMT3B modulate distinct polycomb-mediated histone modifications in colon cancer. *Cancer Res* 69: 7412–7421, 2009.

277. Jin P, Zarnescu DC, Ceman S, Nakamoto M, Mowrey J, Jongens TA, Nelson DL, Moses K, Warren ST. Biochemical and genetic interaction between the fragile X mental retardation protein and the microRNA pathway. *Nat Neurosci* 7: 113–117, 2004.

278. Joglekar MV, Joglekar VM, Hardikar AA. Expression of islet-specific microRNAs during human pancreatic development. *Gene Expr Patterns* 9: 109–113, 2009.

279. John B, Enright AJ, Aravin A, Tuschl T, Sander C, Marks DS. Human MicroRNA targets. *PLoS Biol* 2: 1862–1879, 2004.

280. Johnatty SE, Dyck JR, Michael LH, Olson EN, Abdellatif M. Identification of genes regulated during mechanical load-induced cardiac hypertrophy. *J Mol Cell Cardiol* 32: 805–815, 2000.

281. Johnnidis JB, Harris MH, Wheeler RT, Stehling-Sun S, Lam MH, Kirak O, Brummelkamp TR, Fleming MD, Camargo FD. Regulation of progenitor cell proliferation and granulocyte function by microRNA-223. *Nature* 451: 1125–1129, 2008.

282. Johnson CD, Esquela-Kerscher A, Stefani G, Byrom M, Kelnar K, Ovcharenko D, Wilson M, Wang X, Shelton J, Shingara J, Chin L, Brown D, Slack FJ. The let-7 microRNA represses cell proliferation pathways in human cells. *Cancer Res* 67: 7713–7722, 2007.

283. Johnson SM, Grosshans H, Shingara J, Byrom M, Jarvis R, Cheng A, Labourier E, Reinert KL, Brown D, Slack FJ. RAS is regulated by the let-7 microRNA family. *Cell* 120: 635–647, 2005.

284. Jopling CL, Yi M, Lancaster AM, Lemon SM, Sarnow P. Modulation of hepatitis C virus RNA abundance by a liver-specific MicroRNA. *Science* 309: 1577–1581, 2005.

285. Juan AH, Kumar RM, Marx JG, Young RA, Sartorelli V. Mir-214-dependent regulation of the polycomb protein Ezh2 in skeletal muscle and embryonic stem cells. *Mol Cell* 36: 61–74, 2009.

286. Junn E, Lee KW, Jeong BS, Chan TW, Im JY, Mouradian MM. Repression of alpha-synuclein expression and toxicity by microRNA-7. *Proc Natl Acad Sci USA* 106: 13052–13057, 2009.

287. Karsunky H, Zeng H, Schmidt T, Zevnik B, Kluge R, Schmid KW, Duhrsen U, Moroy T. Inflammatory reactions and severe neutropenia in mice lacking the transcriptional repressor Gfi1. *Nat Genet* 30: 295–300, 2002.

288. Katayama K, Yamada K, Ornthalalai VG, Inoue T, Ota M, Murphy NP, Aruga J. Slitrk1-deficient mice display elevated anxiety-like behavior and noradrenergic abnormalities. *Mol Psychiatry* 15: 177–184, 2010.

289. Kato M, Putta S, Wang M, Yuan H, Lan ting L, Nair I, Gunn A, Nakagawa Y, Shimano H, Todorov I, Rossi JJ, Natarajan R. TGF-beta activates Akt kinase through a microRNA-dependent amplifying circuit targeting PTEN. *Nat Cell Biol* 11: 881–889, 2009.

290. Kato M, Zhang J, Wang M, Lan ting L, Yuan H, Rossi JJ, Natarajan R. MicroRNA-192 in diabetic kidney glomeruli and its function in TGF-beta-induced collagen expression via inhibition of E-box repressors. *Proc Natl Acad Sci USA* 104: 3432–3437, 2007.

291. Katoh T, Sakaguchi Y, Miyauchi K, Suzuki T, Kashiwabara S, Baba T. Selective stabilization of mammalian microRNAs by 3′ adenylation mediated by the cytoplasmic poly(A) polymerase GLD-2. *Genes Dev* 23: 433–438, 2009.

292. Kawahara Y, Zinshteyn B, Sethupathy P, Izasa H, Hatzigeorgiou AG, Nishikura K. Redirection of silencing targets by adenosine-to-inosine editing of miRNAs. *Science* 315: 1137–1140, 2007.

293. Kawashima H, Numakawa T, Kumamaru E, Adachi N, Mizuno H, Ninomiya M, Kunugi H, Hashido K. Glucocorticoid attenuates brain-derived neurotrophic factor-dependent upregulation of glutamate receptors via the suppression of microRNA-132 expression. *Neuroscience* 1: 1, 2009.

294. Kefas B, Comeau L, Floyd DH, Seleverstov O, Godlewski J, Schmittgen T, Jiang J, diPierro CG, Li Y, Chiocca EA, Lee J, Fine H, Abounader R, Lawler S, Purow B. The neuronal microRNA miR-326 acts in a feedback loop with notch and has therapeutic potential against brain tumors. *J Neurosci* 29: 15161–15168, 2009.

295. Kefas B, Godlewski J, Comeau L, Li Y, Abounader R, Hawkinson M, Lee J, Fine H, Chiocca EA, Lawler S, Purow B. MicroRNA-7 inhibits the epidermal growth factor receptor and the Akt pathway and is down-regulated in glioblastoma. *Cancer Res* 68: 3566–3572, 2008.

296. Kertesz M, Lovino N, Unnerstall U, Gaul U, Segal E. The role of site accessibility in microRNA target recognition. *Nat Genet* 39: 1278–1284, 2007.

297. Kiernan RE, Vanhulle C, Schiltz L, Adam E, Xiao H, Maudoux F, Calomme C, Burny A, Nakatani Y, Jeang KT, Benkirane M, Van Lint C. HIV-1 tat transcriptional activity is regulated by acetylation. *EMBO J* 18: 6106–6118, 1999.

298. Kim GH, Samant SA, Earley JU, Svensson EC. Translational control of FOG-2 expression in cardiomyocytes by microRNA-130a. *PLoS One* 4: e6161, 2009.

299. Kim HK, Lee YS, Sivaprasad U, Malhotra A, Dutta A. Muscle-specific microRNA miR-206 promotes muscle differentiation. *J Cell Biol* 21: 21, 2006.

300. Kim J, Inoue K, Ishii J, Vanti WB, Voronov SV, Murchison E, Hannon G, Abeliovich A. A MicroRNA feedback circuit in midbrain dopamine neurons. *Science* 317: 1220–1224, 2007.

301. Kim TW, Yim S, Choi BJ, Jang Y, Lee JJ, Sohn BH, Yoo HS, Yeom YI, Park KC. Tristetraprolin regulates the stability of HIF-1alpha mRNA during prolonged hypoxia. *Biochem Biophys Res Commun* 391: 963–968, 2010.

302. Kiriakidou M, Tan GS, Lamprinaki S, De Planell-Saguer M, Nelson PT, Mourelatos Z. An mRNA m7G cap binding-like motif within human Ago2 represses translation. *Cell* 129: 1141–1151, 2007.

303. Klase Z, Kale P, Winograd R, Gupta MV, Heydarian M, Berro R, McCaffrey T, Kashanchi F. HIV-1 TAR element is processed by Dicer to yield a viral micro-RNA involved in chromatin remodeling of the viral LTR. *BMC Mol Biol* 8: 63, 2007.

304. Klein ME, Lioy DT, Ma L, Impey S, Mandel G, Goodman RH. Homeostatic regulation of MeCP2 expression by a CREB-induced microRNA. *Nat Neurosci* 10: 1513–1514, 2007.

305. Kluiver J, Haralambieva E, de Jong D, Blokzijl T, Jacobs S, Kroesen BJ, Poppema S, van den Berg A. Lack of BIC and microRNA miR-155 expression in primary cases of Burkitt lymphoma. *Genes Chromosomes Cancer* 45: 147–153, 2006.

306. Kluiver J, Poppema S, de Jong D, Blokzijl T, Harms G, Jacobs S, Kroesen BJ, van den Berg A. BIC and miR-155 are highly expressed in Hodgkin, primary mediastinal and diffuse large B cell lymphomas. *J Pathol* 207: 243–249, 2005.

307. Kluiver J, van den Berg A, de Jong D, Blokzijl T, Harms G, Bouwman E, Jacobs S, Poppema S, Kroesen BJ. Regulation of pri-microRNA BIC transcription and processing in Burkitt lymphoma. *Oncogene* 26: 3769–3776, 2007.

308. Knuesel MT, Meyer KD, Berneicky C, Taatjes DJ. The human CDK8 subcomplex is a molecular switch that controls Mediator coactivator function. *Genes Dev* 23: 439–451, 2009.

309. Kohlhaas S, Garden OA, Scudamore C, Turner M, Okkenhaug K, Vigorito E. Cutting edge: the Foxp3 target miR-155 contributes to the development of regulatory T cells. *J Immunol* 182: 2578–2582, 2009.

310. Kong D, Li Y, Wang Z, Banerjee S, Ahmad A, Kim HR, Sarkar FH. miR-200 regulates PDGF-D-mediated epithelial-mesenchymal transition, adhesion, and invasion of prostate cancer cells. *Stem Cells* 27: 1712–1721, 2009.

311. Koon HB, Ippolito GC, Banham AH, Tucker PW. FOXPI: a potential therapeutic target in cancer. *Expert Opin Ther Targets* 11: 955–965, 2007.

312. Koralov SB, Muljo SA, Galler GR, Krek A, Chakraborty T, Kanellopoulou C, Jensen K, Cobb BS, Merkenschlager M, Rajewsky N, Rajewsky K. Dicer ablation affects antibody diversity and cell survival in the B lymphocyte lineage. *Cell* 132: 860–874, 2008.

313. Korpel M, Lee ES, Hu G, Kang Y. The miR-200 family inhibits epithelial-mesenchymal transition and cancer cell migration by direct targeting of E-cadherin transcriptional repressors ZEB1 and ZEB2. *J Biol Chem* 283: 14910–14914, 2008.

314. Kort EJ, Farber L, Tretiakova M, Petillo D, Forge KA, Yang XJ, Cornelius A, Teh BT. The E2F3-Oncomir-1 axis is activated in Wilms’ tumor. *Cancer Res* 68: 4034–4038, 2008.

315. Kota J, Chivukula RR, O'Donnell KA, Wentzel EA, Montgomery CL, Hwang HW, Chang TC, Vivekanandan P, Torbenson M, Clark KR, Mendell JR, Mendell JT. Ther-

MicroRNAs IN DEVELOPMENT AND DISEASE

316. Krek A, Grun D, Poy MN, Wolf R, Rosenberg L, Epstein EJ, MacMenamin P, da Piedade I, Gunsalus KC, Stoffel M, Rajewsky N. Combinatorial microRNA target predictions. Nat Genet 37: 495–500, 2005.

317. Krichevsky AM, King KS, Donahue CP, Khrapko K, Kosik KS. A microRNA array reveals extensive regulation of microRNAs during brain development. RNA 9: 1274–1281, 2003.

318. Krichevsky AM, Sonntag KC, Isacson O, Kosik KS. Specific microRNAs modulate embryonic stem cell-derived neurogenesis. Stem Cells 24: 857–864, 2006.

319. Krutzfeldt J, Kuwajima S, Braich R, Rajeev KG, Pena J, Tuschl T, Manoharan M, Stoffel M. Specificity, duplex degradation and subcellular localization of antagomirs. Nucleic Acids Res 16: 16, 2007.

320. Krutzfeldt J, Rajewsky N, Braich R, Rajeev KG, Tuschl T, Manoharan M, Stoffel M. Silencing of microRNAs in vivo with “antagomirs.” Nature 438: 685–689, 2005.

321. Kuehbacher A, Urbich C, Zeiher AM, Dimmeler S. Role of Dicer and Drosha for endothelial microRNA expression and angiogenesis. Circ Res 101: 59–68, 2007.

322. Kuhnert F, Mancuso MR, Hampton J, Stankunas K, Asano T, Chen CZ, Kuo CJ. Attribution of vascular phenotypes of the murine Egfl7 locus to the microRNA miR-126. Development 135: 3989–3993, 2008.

323. Kumar MS, Erkeland SJ, Pester RE, Chen CY, Ebert MS, Sharp PA, Jacks T. Suppression of non-small cell lung tumor development by the let-7 microRNA family. Proc Natl Acad Sci USA 105: 3903–3908, 2008.

324. Kutay H, Bai S, Datta J, Motiwala T, Pogribny I, Frankel W, Jacob ST, Ghoshal K. Downregulation of miR-122 in the rodent and human hepatocellular carcinomas. J Cell Biochem 99: 671–678, 2006.

325. La Rocca G, Badin M, Shi B, Xu SQ, Deangelis T, Sepp-Lorenzinoi L, Baserga R. Mechanism of growth inhibition by MicroRNA 145: the role of the IGF-I receptor signaling pathway. J Cell Physiol 220: 485–491, 2009.

326. Lagos D, Pollara G, Henderson S, Gratrix F, Fabani M, Milne RS, Gotch F, Boshoff C. miR-132 regulates antiviral innate immunity through suppression of the p300 transcriptional co-activator. Nat Cell Biol 12: 513–519, 2010.

327. Lagos-Quintana M, Rauhut R, Lendeckel W, Tuschl T. Identification of novel genes coding for small expressed RNAs. Science 294: 853–858, 2001.

328. Lagos-Quintana M, Rauhut R, Yalcin A, Meyer J, Lendeckel W, Tuschl T. Identification of tissue-specific microRNAs from mouse. Curr Biol 12: 735–739, 2002.

329. Land PW, Monaghan AP. Expression of the transcription factor, tailless, is required for formation of superficial cortical layers. Cereb Cortex 13: 921–931, 2003.

330. Landthaler M, Yalcin A, Tuschl T. The human DiGeorge syndrome critical region gene 8 and its D. melanogaster homolog are required for miRNA biogenesis. Curr Biol 14: 2162–2167, 2004.

331. Lanford RE, Hildebrandt-Eriksen ES, Petri A, Persson R, Lindow M, Munk ME, Kauppinen S, Orum H. Therapeutic silencing of MicroRNA-122 in primates with chronic hepatitis C virus infection. Science 327: 198–201, 2010.

332. Larribere L, Hilmi C, Khaled M, Gaggioli C, Bille K, Auberger P, Ortonne JP, Ballotti R, Bertolotto C. The cleavage of microphthalmia-associated transcription factor, MITF, by caspases plays an essential role in melanocyte and melanoma cell apoptosis. Genes Dev 19: 1980–1985, 2005.

333. Larsson E, Fredlund Fuchs P, Heldin J, Barkefors I, Bondjers C, Genove G, Arrondel C, Gerwins P, Kurschat C, Schermer B, Benzing T, Harvey SJ, Kreuger J, Lindahl P. Discovery of microvascular miRNAs using public gene expression data: miR-145 is expressed in pericytes and is a regulator of Flt1. Genome Med 1: 108, 2009.

334. Lau NC, Lim LP, Weinstein EG, Bartel DP. An abundant class of tiny RNAs with probable regulatory roles in Caenorhabditis elegans. Science 294: 858–862, 2001.

335. Le MT, Xie H, Zhou B, Chia PH, Rizk P, Um M, Udolph G, Yang H, Lim B, Lodish HF. MicroRNA-125b promotes neuronal differentiation in human cells by repressing multiple targets. Mol Cell Biol 29: 5290–5305, 2009.

336. Lebwohl M. Psoriasis. Lancet 361: 1197–1204, 2003.

337. Lee CC, Putnam AJ, Miranti CK, Gustafson M, Wang LM, Vande Woude GF, Gao CF. Overexpression of sprouty 2 inhibits HGF/SF-mediated cell growth, invasion, migration, and cytokinesis. Oncogene 23: 5193–5202, 2004.

338. Lee CH, Subramanian S, Beck AH, Espinosa I, Senz J, Zhu SX, Huntsman D, van de Rijn M, Gilks CB. MicroRNA profiling of BRCA1/2 mutation-carrying and non-mutation-carrying high-grade serous carcinomas of ovary. PLoS One 4: 2009.

339. Lee KH, Chen YL, Yeh SD, Hsiao M, Lin JT, Goan YG, Lu PJ. MicroRNA-330 acts as tumor suppressor and induces apoptosis of prostate cancer cells through E2F1-mediated suppression of Akt phosphorylation. Oncogene 28: 3360–3370, 2009.

340. Lee RC, Ambros V. An extensive class of small RNAs in Caenorhabditis elegans. Science 294: 862–864, 2001.

341. Lee RC, Feinbaum RL, Ambros V. The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell 75: 843–854, 1993.

342. Lee SO, Masyuk T, Splinter P, Banales JM, Masyuk A, Stroope A, Larusso N. MicroRNA15a modulates expression of the cell-cycle regulator Cdc25A and affects hepatic cystogenesis in a rat model of polycystic kidney disease. J Clin Invest 118: 3714–3724, 2008.

343. Lee TI, Jenner RG, Boyer LA, Guenther MG, Levine SS, Kumar RM, Chevalier B, Johnstone SE, Cole MF, Isono K, Koseki H, Fuchikami T, Abe K, Murray HL, Zucker JP, Yuan B, Bell GW, Herbstheimer E, Hannett NM, Sun K, Odom DT, Otte AP, Volkert TL, Bartel DP, Melton DA, Gifford DK, Jaenisch R, Young RA. Control of developmental regulators by Polycomb in human embryonic stem cells. Cell 125: 301–313, 2006.

344. Lee Y, Ahn C, Han J, Choi H, Kim J, Yim J, Lee J, Provost P, Radmark O, Kim S, Kim VN. The nuclear RNase III Drosha initiates microRNA processing. Nature 425: 415–419, 2003.

345. Lee Y, Samaco RC, Gatchel JR, Thaller C, Orr HT, Zoghbi HY. miR-19, miR-101 and miR-130 co-regulate ATXN1 levels to potentially modulate SCA1 pathogenesis. Nat Neurosci 11: 1137–1139, 2008.

346. Lee YS, Dutta A. The tumor suppressor microRNA let-7 represses the HMGA2 oncogene. Genes Dev 16: 16, 2007.

347. Lei X, Bai Z, Ye F, Xie J, Kim CG, Huang Y, Gao SJ. Regulation of NF-kappaB inhibitor IkappaBalpha and viral replication by a KSHV microRNA. Nat Cell Biol 12: 193–199, 2010.

348. Lena AM, Shalom-Feuerstein R, Rivetti di Val Cervo P, Aberdam D, Knight RA, Melino G, Candi E. miR-203 represses “stemness” by repressing DeltaNp63. Cell Death Differ 15: 1187–1195, 2008.

349. Lerner DL, Yamada KA, Schuessler RB, Saffitz JE. Accelerated onset and increased incidence of ventricular arrhythmias induced by ischemia in Cx43-deficient mice. Circulation 101: 547–552, 2000.

350. Lerner M, Harada M, Loven J, Castro J, Davis Z, Oscier D, Henriksson M, Sangfelt O, Grander D, Corcoran MM. DLEU2, frequently deleted in malignancy, functions as a critical host gene of the cell cycle inhibitory microRNAs miR-15a and miR-16–1. Exp Cell Res 315: 2941–2952, 2009.

351. Leucht C, Stigloher C, Wizenmann A, Klafke R, Folchert A, Bally-Cuif L. MicroRNA-9 directs late organizer activity of the midbrain-hindbrain boundary. Nat Neurosci 11: 641–648, 2008.

352. Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are microRNA targets. Cell 120: 15–20, 2005.

353. Lewis BP, Shih IH, Jones-Rhoades MW, Bartel DP, Burge CB. Prediction of mammalian microRNA targets. Cell 115: 787–798, 2003.

354. Lewis MA, Quint E, Glazier AM, Fuchs H, De Angelis MH, Langford C, van Dongen S, Abreu-Goodger C, Piipari M, Redshaw N, Dalmay T, Moreno-Pelayo MA, Enright AJ, Steel KP. An ENU-induced mutation of miR-96 associated with progressive hearing loss in mice. Nat Genet 41: 614–618, 2009.

355. Lewis-Wambi JS, Jordan VC. Estrogen regulation of apoptosis: how can one hormone stimulate and inhibit? Breast Cancer Res 11: 206, 2009.

356. Li H, Kloosterman W, Fekete DM. MicroRNA-183 family members regulate sensory fates in the inner ear. J Neurosci 30: 3254–3263, 2010.

357. Li N, Fu H, Tie Y, Hu Z, Kong W, Wu Y, Zheng X. miR-34a inhibits migration and invasion by down-regulation of c-Met expression in human hepatocellular carcinoma cells. *Cancer Lett* 275: 44–53, 2009.

358. Li Q, Song XW, Zou J, Wang GK, Kremneva E, Li XQ, Zhu N, Sun T, Lappalainen P, Yuan WJ, Qin YW, Jing Q. Attenuation of microRNA-I derepresses the cytoskeleton regulatory protein twinfilin-I to provoke cardiac hypertrophy. *J Cell Sci* 123: 2444–2452.

359. Li QJ, Chau J, Ebert PJ, Sylvester G, Min H, Liu G, Braich R, Manoharan M, Soutschek J, Skare P, Klein LO, Davis MM, Chen CZ. miR-18Ia is an intrinsic modulator of T cell sensitivity and selection. *Cell* 129: 147–161, 2007.

360. Li S, Fu H, Wang Y, Tie Y, Xing R, Zhu J, Sun Z, Wei L, Zheng X. MicroRNA-10I regulates expression of the v-fos FBJ murine osteosarcoma viral oncogene homolog (FOS) oncogene in human hepatocellular carcinoma. *Hepatology* 49: 1194–1202, 2009.

361. Li T, Li D, Sha J, Sun P, Huang Y. MicroRNA-2I directly targets MARCKS and promotes apoptosis resistance and invasion in prostate cancer cells. *Biochem Biophys Res Commun* 383: 280–285, 2009.

362. Li W, Duan R, Kooy F, Sherman SL, Zhou W, Jin P. Germline mutation of microRNA-I25a is associated with breast cancer. *J Med Genet* 46: 358–360, 2009.

363. Li X, Placencio V, Iturregui JM, Uwamariya C, Sharif-Afshar AR, Koyama T, Hayward SW, Bhowmick NA. Prostate tumor progression is mediated by a paracrine TGF-beta/Wnt3a signaling axis. *Oncogene* 27: 7118–7130, 2008.

364. Li XF, Yan PJ, Shao ZM. Downregulation of miR-193b contributes to enhance urokinase-type plasminogen activator (uPA) expression and tumor progression and invasion in human breast cancer. *Oncogene* 28: 3937–3948, 2009.

365. Li Y, Guessous F, Zhang Y, Dipierro C, Kefas B, Johnson E, Marcinkiewicz L, Jiang J, Yang Y, Schmittgen TD, Lopes B, Schiff D, Purow B, Abounader R. MicroRNA-34a inhibits glioblastoma growth by targeting multiple oncogenes. *Cancer Res* 69: 7569–7576, 2009.

366. Li Y, Tan W, Neo TW, Aung MO, Wasser S, Lim SG, Tan TM. Role of the miR-106b-25 microRNA cluster in hepatocellular carcinoma. *Cancer Sci* 100: 1234–1242, 2009.

367. Li Y, Wang F, Lee JA, Gao FB. MicroRNA-9a ensures the precise specification of sensory organ precursors in *Drosophila*. *Genes Dev* 20: 2793–2805, 2006.

368. Lin CJ, Gong HY, Tseng HC, Wang WL, Wu JL. miR-122 targets an anti-apoptotic gene, Bcl-w, in human hepatocellular carcinoma cell lines. *Biochem Biophys Res Commun* 375: 315–320, 2008.

369. Lin SL, Chiang A, Chang D, Ying SY. Loss of mir-146a function in hormone-refractory prostate cancer. *RNA* 14: 417–424, 2008.

370. Lin Z, Murtaza I, Wang K, Jiao J, Gao J, Li PF. miR-23a functions downstream of NFATc3 to regulate cardiac hypertrophy. *Proc Natl Acad Sci USA* 106: 12103–12108, 2009.

371. Linhares VL, Almeida NA, Menezes DC, Elliott DA, Lai D, Beyer EC, Campos de Carvalho AC, Costa MW. Transcriptional regulation of the murine connexin 40 promoter by cardiac factors Nkx2–5, GATA4 and Tbx5. *Cardiovasc Res* 64: 402–411, 2004.

372. Linke A, Goren I, Bosl MR, Pfeilschifter J, Frank S. The suppressor of cytokine signaling (SOCS)-3 determines keratinocyte proliferative and migratory potential during skin repair. *J Invest Dermatol* 130: 876–885, 2010.

373. Linsley PS, Schelter J, Burchard J, Kibukawa M, Martin MM, Bartz SR, Johnson JM, Cummins JM, Raymond CK, Dai H, Chau N, Cleary M, Jackson AL, Carleton M, Lim L. Transcripts targeted by the microRNA-16 family cooperatively regulate cell cycle progression. *Mol Cell Biol* 27: 2240–2252, 2007.

374. Liston A, Lu LF, O’Carroll D, Tarakhovsky A, Rudensky AY. Dicer-dependent microRNA pathway safeguards regulatory T cell function. *J Exp Med* 205: 1993–2004, 2008.

375. Liu B, Peng XC, Zheng XL, Wang J, Qin YW. MiR-126 restoration down-regulate VEGF and inhibit the growth of lung cancer cell lines in vitro and in vivo. *Lung Cancer* 66: 169–175, 2009.

376. Liu J, Rivas FV, Woehschlegel J, Yates JR ,3rd, Parker R, Hannon GJ. A role for the P-body component GW182 in microRNA function. *Nat Cell Biol* 7: 1261–1266, 2005.

377. Liu N, Bezprozvannaya S, Williams AH, Qi X, Richardson JA, Bassel-Duby R, Olson EN. microRNA-133a regulates cardiomyocyte proliferation and suppresses smooth muscle gene expression in the heart. *Genes Dev* 22: 3242–3254, 2008.

378. Liu X, Cheng Y, Zhang S, Lin Y, Yang J, Zhang C. A necessary role of miR-221 and miR-222 in vascular smooth muscle cell proliferation and neointimal hyperplasia. *Circ Res* 104: 476–487, 2009.

379. Liu Y, Corcoran M, Rasool O, Ivanova G, Ibbotson R, Grander D, Iyengar A, Baranova A, Kashuba V, Merup M, Wu X, Gardiner A, Mullenbach R, Poltaraus A, Hultstrom AL, Juliusson G, Chapman R, Tiller M, Cotter F, Gahrton G, Yankovsky N, Zabarovsky E, Einhorn S, Oscier D. Cloning of two candidate tumor suppressor genes within a 10 kb region on chromosome 13q14, frequently deleted in chronic lymphocytic leukemia. *Oncogene* 15: 2463–2473, 1997.

380. Lodygin D, Tarasov V, Epanchintsev A, Berking C, Knyazeva T, Korner H, Knyazev P, Diebold J, Hermeking H. Inactivation of miR-34a by aberrant CpG methylation in multiple types of cancer. *Cell Cycle* 7: 2591–2600, 2008.

381. Long D, Lee R, Williams P, Chan CY, Ambros V, Ding Y. Potent effect of target structure on microRNA function. *Nat Struct Mol Biol* 14: 287–294, 2007.

382. Lowes MA, Bowcock AM, Krueger JG. Pathogenesis and therapy of psoriasis. *Nature* 445: 866–873, 2007.

383. Lu F, Stedman W, Yousef M, Renne R, Lieberman PM. Epigenetic regulation of Kaposis sarcoma-associated herpesvirus latency by virus-encoded microRNAs that target Rta and the cellular Rbl2-DNMT pathway. *J Virol* 84: 2697–2706, 2010.

384. Lu F, Weidmer A, Liu CG, Volinia S, Croce CM, Lieberman PM. Epstein-Barr virus-induced miR-155 attenuates NF-kappaB signaling and stabilizes latent virus persistence. *J Virol* 82: 10436–10443, 2008.

385. Lu J, Guo S, Ebert BL, Zhang H, Peng X, Bosco J, Pretz J, Schlanger R, Wang JY, Mak RH, Dombkowski DM, Preffer FI, Scadden DT, Golub TR. MicroRNA-mediated control of cell fate in megakaryocyte-erythrocyte progenitors. *Dev Cell* 14: 843–853, 2008.

386. Lu L, Katsaros D, de la Longrais IA, Sochirca O, Yu H. Hypermethylation of let-7a-3 in epithelial ovarian cancer is associated with low insulin-like growth factor-II expression and favorable prognosis. *Cancer Res* 67: 10117–10122, 2007.

387. Lu L, Katsaros D, Shaverdashvili K, Qian B, Wu Y, de la Longrais IA, Preti M, Menato G, Yu H. Pluripotent factor lin-28 and its homologue lin-28b in epithelial ovarian cancer and their associations with disease outcomes and expression of let-7a and IGF-II. *Eur J Cancer* 45: 2212–2218, 2009.

388. Lu LF, Boldin MP, Chaudhry A, Lin LL, Taganov KD, Hanada T, Yoshimura A, Baltimore D, Rudensky AY. Function of miR-146a in controlling Treg cell-mediated regulation of Th1 responses. *Cell* 142: 914–929, 2010.

389. Lu LF, Thai TH, Calado DP, Chaudhry A, Kubo M, Tanaka K, Loeb GB, Lee H, Yoshimura A, Rajewsky K, Rudensky AY. Foxp3-dependent microRNA155 confers competitive fitness to regulatory T cells by targeting SOCS1 protein. *Immunity* 30: 80–91, 2009.

390. Lu Y, Thomson JM, Wong HY, Hammond SM, Hogan BL. Transgenic over-expression of the microRNA miR-17–92 cluster promotes proliferation and inhibits differentiation of lung epithelial progenitor cells. *Dev Biol* 310: 442–453, 2007.

391. Lu Z, Liu M, Stribinskis V, Klinge CM, Ramos KS, Colburn NH, Li Y. MicroRNA-2I promotes cell transformation by targeting the programmed cell death 4 gene. *Oncogene* 27: 4373–4379, 2008.

392. Luciano DJ, Mirsky H, Vendetti NJ, Maas S. RNA editing of a miRNA precursor. *RNA* 10: 1174–1177, 2004.

393. Lujambio A, Ropero S, Ballestar E, Fraga MF, Cerrato C, Setien F, Casado S, Suarez-Gauthier A, Sanchez-Cespedes M, Git A, Spiteri I, Das PP, Caldas C, Miska E, Esteller M. Genetic unmasking of an epigenetically silenced microRNA in human cancer cells. *Cancer Res* 67: 1424–1429, 2007.

394. Lund E, Guttinger S, Calado A, Dahlberg JE, Kutay U. Nuclear export of microRNA precursors. *Science* 303: 95–98, 2004.

395. Luo X, Lin H, Pan Z, Xiao J, Zhang Y, Lu Y, Yang B, Wang Z. Down-regulation of miR-I/miR-133 contributes to re-expression of pacemaker channel genes HCN2 and HCN4 in hypertrophic heart. *J Biol Chem* 283: 20045–20052, 2008.

MicroRNAs IN DEVELOPMENT AND DISEASE

396. Lynn FC, Skewes-Cox P, Kosaka Y, McManus MT, Harfe BD, German MS. MicroRNA expression is required for pancreatic islet cell genesis in the mouse. *Diabetes* 56: 2938–2945, 2007.

397. Ma L, Teruya-Feldstein J, Weinberg RA. Tumour invasion and metastasis initiated by microRNA-10b in breast cancer. *Nature* 449: 682–688, 2007.

398. Maas S, Rich A. Changing genetic information through RNA editing. *Bioessays* 22: 790–802, 2000.

399. Mahdavi V, Lompre AM, Chambers AP, Nadal-Ginard B. Cardiac myosin heavy chain isozymic transitions during development and under pathological conditions are regulated at the level of mRNA availability. *Eur Heart J* 5 Suppl F: , 1984181–191.

400. Maillet G, Lacroix-Triki M, Pierredon S, Gratadou L, Schmidt S, Benes V, Roche H, Dalenc F, Auboeuf D, Millevoi S, Vagner S. Widespread estrogen-dependent repression of microRNAs involved in breast tumor cell growth. *Cancer Res* 69: 8332–8340, 2009.

401. Makeyev EV, Zhang J, Carrasco MA, Maniatis T. The MicroRNA miR-124 promotes neuronal differentiation by triggering brain-specific alternative pre-mRNA splicing. *Mol Cell* 27: 435–448, 2007.

402. Makiyama K, Hamada J, Takada M, Murakawa K, Takahashi Y, Tada M, Tamoto E, Shindo G, Matsunaga A, Teramoto K, Komuro K, Kondo S, Katoh H, Koike T, Moriuchi T. Aberrant expression of HOX genes in human invasive breast carcinoma. *Oncol Rep* 13: 673–679, 2005.

403. Mani SA, Guo W, Liao MJ, Eaton EN, Ayyanan A, Zhou AY, Brooks M, Reinhard F, Zhang CC, Shipitsin M, Campbell LL, Polyak K, Brisken C, Yang J, Weinberg RA. The epithelial-mesenchymal transition generates cells with properties of stem cells. *Cell* 133: 704–715, 2008.

404. Marcucci G, Radmacher MD, Maharry K, Mrózek K, Ruppert AS, Paschka P, Vuko-savljevic T, Whitman SP, Baldus CD, Langer C, Liu CG, Carroll AJ, Powell BL, Garzon R, Croce CM, Kolitz JE, Caligiuri MA, Larson RA, Bloomfield CD. MicroRNA expression in cytogenetically normal acute myeloid leukemia. *N Engl J Med* 358: 1919–1928, 2008.

405. Masaki S, Ohtsuka R, Abe Y, Muta K, Umemura T. Expression patterns of microRNAs I55 and 451 during normal human erythropoiesis. *Biochem Biophys Res Commun* 364: 509–514, 2007.

406. Mathonnet G, Fabian MR, Svitkin YV, Parsyan A, Huck L, Murata T, Biffo S, Merrick WC, Darzynkiewicz E, Pillai RS, Filipowicz W, Duchaine TF, Sonenberg N. MicroRNA inhibition of translation initiation in vitro by targeting the cap-binding complex eIF4F. *Science* 317: 1764–1767, 2007.

407. Matkovich SJ, Van Booven DJ, Youker KA, Torre-Amione G, Diwan A, Eschenbacher WH, Dorn LE, Watson MA, Margulies KB, Dorn GW ,2nd. Reciprocal regulation of myocardial microRNAs and messenger RNA in human cardiomyopathy and reversal of the microRNA signature by biomechanical support. *Circulation* 119: 1263–1271, 2009.

408. Matkovich SJ, Wang W, Tu Y, Eschenbacher WH, Dorn LE, Condorelli G, Diwan A, Nerbonne JM, Dorn GW ,2nd. MicroRNA-133a protects against myocardial fibrosis and modulates electrical repolarization without affecting hypertrophy in pressure-overloaded adult hearts. *Circ Res* 106: 166–175, 2010.

409. Matsubara H, Takeuchi T, Nishikawa E, Yanagisawa K, Hayashita Y, Ebi H, Yamada H, Suzuki M, Nagino M, Nimura Y, Osada H, Takahashi T. Apoptosis induction by antisense oligonucleotides against miR-17–5p and miR-20a in lung cancers overexpressing miR-17–92. *Oncogene* 26: 26, 2007.

410. Mayr C, Bartel DP. Widespread shortening of 3′UTRs by alternative cleavage and polyadenylation activates oncogenes in cancer cells. *Cell* 138: 673–684, 2009.

411. McCarthy JJ. MicroRNA-206: the skeletal muscle-specific myomiR. *Biochim Biophys Acta* 11: 682–691, 2008.

412. McCarthy JJ, Esser KA. MicroRNA-1 and microRNA-133a expression are decreased during skeletal muscle hypertrophy. *J Appl Physiol* 102: 306–313, 2007.

413. McCarthy JJ, Esser KA, Andrade FH. MicroRNA-206 is overexpressed in the diaphragm but not the hindlimb muscle of mdx mouse. *Am J Physiol Cell Physiol* 293: C25, 2007.

414. Meister G, Landthaler M, Patkaniowska A, Dorsett Y, Teng G, Tuschl T. Human Argonaute2 mediates RNA cleavage targeted by miRNAs and siRNAs. *Mol Cell* 15: 185–197, 2004.

415. Mencia A, Modamio-Hoybjor S, Redshaw N, Morin M, Mayo-Merino F, Olavarrieta L, Aguirre LA, del Castillo I, Steel KP, Dalmay T, Moreno F, Moreno-Pelayo MA. Mutations in the seed region of human miR-96 are responsible for nonsyndromic progressive hearing loss. *Nat Genet* 41: 609–613, 2009.

416. Meng F, Henson R, Lang M, Wehbe H, Maheshwari S, Mendell JT, Jiang J, Schmittgen TD, Patel T. Involvement of human micro-RNA in growth and response to chemotherapy in human cholangiocarcinoma cell lines. *Gastroenterology* 130: 2113–2129, 2006.

417. Meng F, Henson R, Wehbe-Janek H, Ghoshal K, Jacob ST, Patel T. MicroRNA-21 regulates expression of the PTEN tumor suppressor gene in human hepatocellular cancer. *Gastroenterology* 133: 647–658, 2007.

418. Meng Y, Zhang Y, Tregoubov V, Janus C, Cruz L, Jackson M, Lu WY, MacDonald JF, Wang JY, Falls DL, Jia Z. Abnormal spine morphology and enhanced LTP in LIMK-1 knockout mice. *Neuron* 35: 121–133, 2002.

419. Mercatelli N, Coppola V, Bonci D, Miele F, Costantini A, Guadagnoli M, Bonanno E, Muto G, Frajese GV, De Maria R, Spagnoli LG, Farace MG, Ciafre SA. The inhibition of the highly expressed miR-221 and miR-222 impairs the growth of prostate carcinoma xenografts in mice. *PLoS One* 3: e4029, 2008.

420. Merritt WM, Lin YG, Han LY, Kamat AA, Spannuth WA, Schmandt R, Urbauer D, Pennacchio LA, Cheng JF, Nick AM, Deavers MT, Mourad-Zeidan A, Wang H, Mueller P, Lenburg ME, Gray JW, Mok S, Birrer MJ, Lopez-Berestein G, Coleman RL, Bar-Eli M, Sood AK. Dicer, Drosha, and outcomes in patients with ovarian cancer. *N Engl J Med* 359: 2641–2650, 2008.

421. Mertens-Talcott SU, Chinthalapalli S, Li X, Safe S. The oncogenic microRNA-27a targets genes that regulate specificity protein transcription factors and the G2-M checkpoint in MDA-MB-231 breast cancer cells. *Cancer Res* 67: 11001–11011, 2007.

422. Metcalf D, Carpinelli MR, Hyland C, Mifsud S, Dirago L, Nicola NA, Hilton DJ, Alexander WS. Anomalous megakaryocytopenia in mice with mutations in the c-Myb gene. *Blood* 105: 3480–3487, 2005.

423. Mi S, Lu J, Sun M, Li Z, Zhang H, Neilly MB, Wang Y, Qian Z, Jin J, Zhang Y, Bohlander SK, Le Beau MM, Larson RA, Golub TR, Rowley JD, Chen J. MicroRNA expression signatures accurately discriminate acute lymphoblastic leukemia from acute myeloid leukemia. *Proc Natl Acad Sci USA* 104: 19971–19976, 2007.

424. Michael MZ, SMOC, van Holst Pellekaan NG, Young GP, James RJ. Reduced accumulation of specific microRNAs in colorectal neoplasia. *Mol Cancer Res* 1: 882–891, 2003.

425. Michels G, Er F, Khan I, Sudkamp M, Herzig S, Hoppe UC. Single-channel properties support a potential contribution of hyperpolarization-activated cyclic nucleotide-gated channels and If to cardiac arrhythmias. *Circulation* 111: 399–404, 2005.

426. Michlewski G, Guil S, Semple CA, Cáceres JF. Posttranscriptional regulation of miR-NAs harboring conserved terminal loops. *Mol Cell* 32: 383–393, 2008.

427. Miller TE, Ghoshal K, Ramaswamy B, Roy S, Datta J, Shapiro CL, Jacob S, Majumder S. MicroRNA-221/222 confers tamoxifen resistance in breast cancer by targeting p27Kip1. *J Biol Chem* 283: 29897–29903, 2008.

428. Mishima Y, Abreu-Goodger C, Staton AA, Stahlhut C, Shou C, Cheng C, Gerstein M, Enright AJ, Giraldez AJ. Zebrafish miR-1 and miR-133 shape muscle gene expression and regulate sarcomeric actin organization. *Genes Dev* 23: 619–632, 2009.

429. Mistry SJ, Bank A, Atweh GF. Targeting stathmin in prostate cancer. *Mol Cancer Ther* 4: 1821–1829, 2005.

430. Morissette MR, Cook SA, Foo S, McKay G, Ashida N, Novikov M, Scherrer-Crosbie M, Li L, Matsui T, Brooks G, Rosenzweig A. Myostatin regulates cardiomyocyte growth through modulation of Akt signaling. *Circ Res* 99: 15–24, 2006.

431. Morita S, Hara A, Kojima I, Horii T, Kimura M, Kitamura T, Ochiya T, Nakanishi K, Matoba R, Matsubara K, Hatada I. Dicer is required for maintaining adult pancreas. *PLoS One* 4: 16, 2009.

432. Moriyama T, Ohuchida K, Mizumoto K, Yu J, Sato N, Nabae T, Takahata S, Toma H, Nagai E, Tanaka M. MicroRNA-21 modulates biological functions of pancreatic cancer cells including their proliferation, invasion, and chemoresistance. *Mol Cancer Ther* 12: 12, 2009.

DANISH SAYED AND MAHA ABDELLATIF

433. Motsch N, Pfuhl T, Mrazek J, Barth S, Grasser FA. Epstein-Barr virus-encoded latent membrane protein I (LMP1) induces the expression of the cellular microRNA miR-146a. RNA Biol 4: 131–137, 2007.

434. Mottet D, Pirotte S, Lamour V, Hagedorn M, Javerzat S, Bikfalvi A, Bellahcene A, Verdin E, Castronovo V. HDAC4 represses p21(WAF1/Cip1) expression in human cancer cells through a Sp1-dependent, p53-independent mechanism. Oncogene 28: 243–256, 2009.

435. Muller DW, Bosserhoff AK. Integrin beta 3 expression is regulated by let-7a miRNA in malignant melanoma. Oncogene 27: 6698–6706, 2008.

436. Murray CE, Jennings RB, Reimer KA. Preconditioning with ischemia: a delay of lethal cell injury in ischemic myocardium. Circulation 74: 1124–1136, 1986.

437. Nachmani D, Stern-Ginossar N, Sarid R, Mandelboim O. Diverse herpesvirus microRNAs target the stress-induced immune ligand MICB to escape recognition by natural killer cells. Cell Host Microbe 5: 376–385, 2009.

438. Naga Prasad SV, Duan ZH, Gupta MK, Surampudi VS, Volinia S, Calin GA, Liu CG, Kotwal A, Moravec CS, Starling RC, Perez DM, Sen S, Wu Q, Plow EF, Croce CM, Karnik S. Unique microRNA profile in end-stage heart failure indicates alterations in specific cardiovascular signaling networks. J Biol Chem 284: 27487–27499, 2009.

439. Naguibneva I, Ameyar-Zazoua M, Polesskaya A, Ait-Si-Ali S, Groisman R, Souidi M, Cuvellier S, Harel-Bellan A. The microRNA miR-181 targets the homeobox protein Hox-A11 during mammalian myoblast differentiation. Nat Cell Biol 8: 278–284, 2006.

440. Nakada C, Matsuura K, Tsukamoto Y, Tanigawa M, Yoshimoto T, Narimatsu T, Nguyen LT, Hijiya N, Uchida T, Sato F, Mimata H, Seto M, Moriyama M. Genome-wide microRNA expression profiling in renal cell carcinoma: significant down-regulation of miR-141 and miR-200c. J Pathol 216: 418–427, 2008.

441. Nakajima N, Takahashi T, Kitamura R, Isodono K, Asada S, Ueyama T, Matsubara H, Oh H. MicroRNA-1 facilitates skeletal myogenic differentiation without affecting osteoblastic and adipogenic differentiation. Biochem Biophys Res Commun 350: 1006–1012, 2006.

442. Nakasa T, Ishikawa M, Shi M, Shibuya H, Adachi N, Ochi M. Acceleration of muscle regeneration by local injection of muscle-specific microRNAs in rat skeletal muscle injury model. J Cell Mol Med 14: 14, 2009.

443. Nasser MW, Datta J, Nuovo G, Kutay H, Motiwala T, Majumder S, Wang B, Suster S, Jacob ST, Ghoshal K. Downregulation of micro-RNA-1 (miR-1) in lung cancer. Suppression of tumorigenic property of lung cancer cells and their sensitization to doxorubicin-induced apoptosis by miR-1. J Biol Chem 283: 33394–33405, 2008.

444. Nathans R, Chu CY, Serquina AK, Lu CC, Cao H, Rana TM. Cellular microRNA and P bodies modulate host-HIV-1 interactions. Mol Cell 34: 696–709, 2009.

445. Navarro A, Gaya A, Martinez A, Urbano-Ispizua A, Pons A, Balague O, Gel B, Abrisqueta P, Lopez-Guillermo A, Artells R, Montserrat E, Monzo M. MicroRNA expression profiling in classic Hodgkin lymphoma. Blood 111: 2825–2832, 2008.

446. Neat MJ, Foot N, Jenner M, Goff L, Ashcroft K, Burford D, Dunham A, Norton A, Lister TA, Fitzgibbon J. Localisation of a novel region of recurrent amplification in follicular lymphoma to an ~6.8 Mb region of 13q32–33. Genes Chromosomes Cancer 32: 236–243, 2001.

447. Nesterov A, Lu X, Johnson M, Miller GJ, Ivashchenko Y, Kraft AS. Elevated AKT activity protects the prostate cancer cell line LNCaP from TRAIL-induced apoptosis. J Biol Chem 276: 10767–10774, 2001.

448. Newman MA, Thomson JM, Hammond SM. Lin-28 interaction with the Let-7 precursor loop mediates microRNA processing. RNA 14: 1539–1549, 2008.

449. Ng EK, Tsang WP, Ng SS, Jin HC, Yu J, Li JJ, Rocken C, Ebert MP, Kwok TT, Sung JJ. MicroRNA-143 targets DNA methyltransferases 3A in colorectal cancer. Br J Cancer 101: 699–706, 2009.

450. Ngo CV, Gee M, Akhtar N, Yu D, Volpert O, Auerbach R, Thomas-Tikhonenko A. An in vivo function for the transforming Myc protein: elicitation of the angiogenic phenotype. Cell Growth Differ 11: 201–210, 2000.

451. Niu Z, Yu W, Zhang SX, Barron M, Belaguli NS, Schneider MD, Parmacek M, Nordheim A, Schwartz RJ. Conditional mutagenesis of the murine serum response factor gene blocks cardiogenesis and the transcription of downstream gene targets. J Biol Chem 280: 32531–32538, 2005.

452. Nocka K, Majumder S, Chabot B, Ray P, Cervone M, Bernstein A, Besmer P. Expression of c-kit gene products in known cellular targets of W mutations in normal and W mutant mice: evidence for an impaired c-kit kinase in mutant mice. Genes Dev 3: 816–826, 1989.

453. Noonan EJ, Place RF, Pookot D, Basak S, Whitson JM, Hirata H, Giardina C, Dahiya R. miR-449a targets HDAC-1 and induces growth arrest in prostate cancer. Oncogene 28: 1714–1724, 2009.

454. Nottrott S, Simard MJ, Richter JD. Human let-7a miRNA blocks protein production on actively translating polyribosomes. Nat Struct Mol Biol 13: 1108–1114, 2006.

455. O'Carroll D, Mecklenbrauker I, Das PP, Santana A, Koenig U, Enright AJ, Miska EA, Tarakhovsky A. A Slicer-independent role for Argonaute 2 in hematopoiesis and the microRNA pathway. Genes Dev 21: 1999–2004, 2007.

456. O'Connell RM, Chaudhuri AA, Rao DS, Baltimore D. Inositol phosphatase SHIP1 is a primary target of miR-155. Proc Natl Acad Sci USA 106: 7113–7118, 2009.

457. O'Connell RM, Rao DS, Chaudhuri AA, Boldin MP, Taganov KD, Nicoll J, Paquette RL, Baltimore D. Sustained expression of microRNA-155 in hematopoietic stem cells causes a myeloproliferative disorder. J Exp Med 205: 585–594, 2008.

458. O'Connell RM, Taganov KD, Boldin MP, Cheng G, Baltimore D. MicroRNA-155 is induced during the macrophage inflammatory response. Proc Natl Acad Sci USA 104: 1604–1609, 2007.

459. O'Donnell KA, Wentzel EA, Zeller KI, Dang CV, Mendell JT. c-Myc-regulated microRNAs modulate E2F1 expression. Nature 435: 839–843, 2005.

460. O'Roak BJ, Morgan TM, Fishman DO, Saus E, Alonso P, Gratacos M, Estivill X, Teltsh O, Kohn Y, Kidd KK, Cho J, Lifton RP, State MW. Additional support for the association of SLITRK1 var321 and Tourette syndrome. Mol Psychiatry 15: 447–450, 2010.

461. Olsen PH, Ambros V. The lin-4 regulatory RNA controls developmental timing in Caenorhabditis elegans by blocking LIN-14 protein synthesis after the initiation of translation. Dev Biol 216: 671–680, 1999.

462. Omoto S, Fujii YR. Regulation of human immunodeficiency virus 1 transcription by nef microRNA. J Gen Virol 86: 751–755, 2005.

463. Ota A, Tagawa H, Karnan S, Tsuzuki S, Karpas A, Kira S, Yoshida Y, Seto M. Identification and characterization of a novel gene, C13orf25, as a target for 13q31-q32 amplification in malignant lymphoma. Cancer Res 64: 3087–3095, 2004.

464. Ouchi N, Oshima Y, Ohashi K, Higuchi A, Ikegami C, Izumiya Y, Walsh K. Folliculin-like 1, a secreted muscle protein, promotes endothelial cell function and revascularization in ischemic tissue through a nitric-oxide synthase-dependent mechanism. J Biol Chem 283: 32802–32811, 2008.

465. Ouellet DL, Plante I, Landry P, Barat C, Janelle ME, Flamand L, Tremblay MJ, Provost P. Identification of functional microRNAs released through asymmetrical processing of HIV-1 TAR element. Nucleic Acids Res 36: 2353–2365, 2008.

466. Ozen M, Creighton CJ, Ozdemir M, Ittmann M. Widespread deregulation of microRNA expression in human prostate cancer. Oncogene 27: 1788–1793, 2008.

467. Packer AN, Xing Y, Harper SQ, Jones L, Davidson BL. The bifunctional microRNA miR-9/miR-9* regulates REST and CoREST and is downregulated in Huntington’s disease. J Neurosci 28: 14341–14346, 2008.

468. Pandey P, Brors B, Srivastava PK, Bott A, Boehn SN, Groene HJ, Gretz N. Microarray-based approach identifies microRNAs and their target functional patterns in polycystic kidney disease. BMC Genomics 9: 624, 2008.

469. Papagiannakopoulos T, Shapiro A, Kosik KS. MicroRNA-21 targets a network of key tumor-suppressive pathways in glioblastoma cells. Cancer Res 68: 8164–8172, 2008.

470. Pardal R, Molofsky AV, He S, Morrison SJ. Stem cell self-renewal and cancer cell proliferation are regulated by common networks that balance the activation of proto-oncogenes and tumor suppressors. Cold Spring Harb Symp Quant Biol 70: 177–185, 2005.

471. Park JK, Lee EJ, Esau C, Schmittgen TD. Antisense inhibition of microRNA-21 or -221 arrests cell cycle, induces apoptosis, and sensitizes the effects of gemcitabine in pancreatic adenocarcinoma. Pancreas 38: e190–199, 2009.

472. Park SM, Gaur AB, Lengyel E, Peter ME. The miR-200 family determines the epithelial phenotype of cancer cells by targeting the E-cadherin repressors ZEB1 and ZEB2. Genes Dev 22: 894–907, 2008.

MicroRNAs IN DEVELOPMENT AND DISEASE

473. Park SM, Shell S, Radjabi AR, Schickel R, Feig C, Boyerinas B, Dinulescu DM, Lengyel E, Peter ME. Let-7 prevents early cancer progression by suppressing expression of the embryonic gene HMGA2. *Cell Cycle* 6: 2585–2590, 2007.

474. Parlakian A, Tuil D, Hamard G, Tavernier G, Hentzen D, Concordet JP, Paulin D, Li Z, Daegelen D. Targeted inactivation of serum response factor in the developing heart results in myocardial defects and embryonic lethality. *Mol Cell Biol* 24: 5281–5289, 2004.

475. Pase L, Layton JE, Kloosterman WP, Carradice D, Waterhouse PM, Lieschke GJ. miR-451 regulates zebrafish erythroid maturation in vivo via its target gata2. *Blood* 113: 1794–1804, 2009.

476. Pastorelli LM, Wells S, Fray M, Smith A, Hough T, Harfe BD, McManus MT, Smith L, Woolf AS, Cheeseman M, Greenfield A. Genetic analyses reveal a requirement for Dicer1 in the mouse urogenital tract. *Mamm Genome* 20: 140–151, 2009.

477. Patel N, Hoang D, Miller N, Ansaloni S, Huang Q, Rogers JT, Lee JC, Saunders AJ. MicroRNAs can regulate human APP levels. *Mol Neurodegener* 3: 10, 2008.

478. Pedersen IM, Cheng G, Wieland S, Volinia S, Croce CM, Chisari FV, David M. Interferon modulation of cellular microRNAs as an antiviral mechanism. *Nature* 449: 919–922, 2007.

479. Pedersen IM, Otero D, Kao E, Miletic AV, Hother C, Ralfkiaer E, Rickert RC, Gronbaek K, David M. Onco-miR-155 targets SHIP1 to promote TNFα-dependent growth of B cell lymphomas. *EMBO Mol Med* 1: 288–295, 2009.

480. Peinado H, Olmeda D, Cano A. Snail, Zeb and bHLH factors in tumour progression: an alliance against the epithelial phenotype? *Nat Rev Cancer* 7: 415–428, 2007.

481. Pekarsky Y, Santanam U, Cimmino A, Palamarchuk A, Efanov A, Maximov V, Volinia S, Alder H, Liu CG, Rassenti L, Calin GA, Hagan JP, Kipps T, Croce CM. Tcl1 expression in chronic lymphocytic leukemia is regulated by miR-29 and miR-181. *Cancer Res* 66: 11590–11593, 2006.

482. Perkins DO, Jeffries CD, Jarskog LF, Thomson JM, Woods K, Newman MA, Parker JS, Jin J, Hammond SM. microRNA expression in the prefrontal cortex of individuals with schizophrenia and schizoaffective disorder. *Genome Biol* 8: 2007.

483. Petersen CP, Bordeleau ME, Pelletier J, Sharp PA. Short RNAs repress translation after initiation in mammalian cells. *Mol Cell* 21: 533–542, 2006.

484. Pfeffer S, Sewer A, Lagos-Quintana M, Sheridan R, Sander C, Grasser FA, van Dyk LF, Ho CK, Shuman S, Chien M, Russo JJ, Ju J, Randall G, Lindenbach BD, Rice CM, Simon V, Ho DD, Zavolan M, Tuschl T. Identification of microRNAs of the herpesvirus family. *Nat Methods* 2: 269–276, 2005.

485. Pfeffer S, Zavolan M, Grasser FA, Chien M, Russo JJ, Ju J, John B, Enright AJ, Marks D, Sander C, Tuschl T. Identification of virus-encoded microRNAs. *Science* 304: 734–736, 2004.

486. Pham VN, Lawson ND, Mugford JW, Dye L, Castranova D, Lo B, Weinstein BM. Combinatorial function of ETS transcription factors in the developing vasculature. *Dev Biol* 303: 772–783, 2007.

487. Pillai RS, Bhattacharyya SN, Artus CG, Zoller T, Cougot N, Basyuk E, Bertrand E, Filipowicz W. Inhibition of translational initiation by Let-7 MicroRNA in human cells. *Science* 309: 1573–1576, 2005.

488. Plaisance V, Abderrahmani A, Perret-Menoud V, Jacquemin P, Lemaigre F, Regazzi R. MicroRNA-9 controls the expression of Granuphilin/Slp4 and the secretory response of insulin-producing cells. *J Biol Chem* 281: 26932–26942, 2006.

489. Plaster NM, Tawil R, Tristani-Firouzi M, Canun S, Bendahhou S, Tsunoda A, Donaldson MR, Iannaccone ST, Brunt E, Barohn R, Clark J, Deymeer F, George AL Jr, Fish FA, Hahn A, Nitu A, Ozdemir C, Serdaroglu P, Subramony SH, Wolfe G, Fu YH, Ptacek LJ. Mutations in Kir2.1 cause the developmental and episodic electrical phenotypes of Andersen’s syndrome. *Cell* 105: 511–519, 2001.

490. Pochampalli MR, el Bejjani RM, Schroeder JA. MUC1 is a novel regulator of ErbB1 receptor trafficking. *Oncogene* 26: 1693–1701, 2007.

491. Porkka KP, Pfeiffer MJ, Waltering KK, Vessella RL, Tammela TL, Visakorpi T. MicroRNA expression profiling in prostate cancer. *Cancer Res* 67: 6130–6135, 2007.

492. Poy MN, Eliasson L, Krutzfeldt J, Kuwajima S, Ma X, Macdonald PE, Pfeffer S, Tuschl T, Rajewsky N, Rorsman P, Stoffel M. A pancreatic islet-specific microRNA regulates insulin secretion. *Nature* 432: 226–230, 2004.

493. Poy MN, Hausser J, Trajkovski M, Braun M, Collins S, Rorsman P, Zavolan M, Stoffel M. miR-375 maintains normal pancreatic alpha- and beta-cell mass. *Proc Natl Acad Sci USA* 106: 5813–5818, 2009.

494. Proulx AA, Lin ZX, Naus CC. Transfection of rhabdomyosarcoma cells with connexin43 induces myogenic differentiation. *Cell Growth Differ* 8: 533–540, 1997.

495. Puri N, Khramtsov A, Ahmed S, Nallasura V, Hetzel JT, Jagadeeswaran R, Karczmar G, Salgia R. A selective small molecule inhibitor of c-Met, PHA665752, inhibits tumorigenicity and angiogenesis in mouse lung cancer xenografts. *Cancer Res* 67: 3529–3534, 2007.

496. Purow BW, Sundaresan TK, Burdick MJ, Kefas BA, Comeau LD, Hawkinson MP, Su Q, Kotliarov Y, Lee J, Zhang W, Fine HA. Notch-1 regulates transcription of the epidermal growth factor receptor through p53. *Carcinogenesis* 29: 918–925, 2008.

497. Rane S, He M, Sayed D, Vashistha H, Malhotra A, Sadoshima J, Vatner DE, Vatner SF, Abdellatif M. Downregulation of MiR-199a derepresses hypoxia-inducible factor-1α and sirtuin 1 and recapitulates hypoxia preconditioning in cardiac myocytes. *Circ Res* 104: 879–886, 2009.

498. Rane S, He M, Sayed D, Yan L, Vatner D, Abdellatif M. An antagonism between the AKT and beta-adrenergic signaling pathways mediated through their reciprocal effects on miR-199a-5p. *Cell Signal* 22: 1054–1062, 2010.

499. Rane S, Sayed D, Abdellatif M. MicroRNA with a MacroFunction. *Cell Cycle* 6: 1850–1855, 2007.

500. Rao PH, Houldsworth J, Dyomina K, Parsa NZ, Cigudosa JC, Louie DC, Popplewell L, Offit K, Jhanwar SC, Chaganti RS. Chromosomal and gene amplification in diffuse large B-cell lymphoma. *Blood* 92: 234–240, 1998.

501. Rao PK, Kumar RM, Farkhondeh M, Baskerville S, Lodish HF. Myogenic factors that regulate expression of muscle-specific microRNAs. *Proc Natl Acad Sci USA* 103: 8721–8726, 2006.

502. Raveche ES, Salerno E, Scaglione BJ, Manohar V, Abbasi F, Lin YC, Fredrickson T, Landgraf P, Ramachandra S, Huppi K, Toro JR, Zenger VE, Metcalf RA, Marti GE. Abnormal microRNA-16 locus with synteny to human 13q14 linked to CLL in NZB mice. *Blood* 109: 5079–5086, 2007.

503. Raver-Shapira N, Marciano E, Meiri E, Spector Y, Rosenfeld N, Moskovits N, Bentwich Z, Oren M. Transcriptional activation of miR-34a contributes to p53-mediated apoptosis. *Mol Cell* 26: 731–743, 2007.

504. Reddy SD, Pakala SB, Ohshiro K, Rayala SK, Kumar R. MicroRNA-661, a c/EBPalpha target, inhibits metastatic tumor antigen 1 and regulates its functions. *Cancer Res* 69: 5639–5642, 2009.

505. Rees H, Williamson D, Papanastasiou A, Jina N, Nabarro S, Shipley J, Anderson J. The MET receptor tyrosine kinase contributes to invasive tumour growth in rhabdomyosarcomas. *Growth Factors* 24: 197–208, 2006.

506. Reinhart BJ, Slack FJ, Basson M, Pasquinelli AE, Bettinger JC, Rougvie AE, Horvitz HR, Ruvkun G. The 21-nucleotide let-7 RNA regulates developmental timing in Caenorhabditis elegans. *Nature* 403: 901–906, 2000.

507. Ren XP, Wu J, Wang X, Sartor MA, Qian J, Jones K, Nicolaou P, Pritchard TJ, Fan GC. MicroRNA-320 is involved in the regulation of cardiac ischemia/reperfusion injury by targeting heat-shock protein 20. *Circulation* 119: 2357–2366, 2009.

508. Ribas J, Ni X, Haffner M, Wentzel EA, Salmasi AH, Chowdhury WH, Kudrolli TA, Yegnasubramanian S, Luo J, Rodriguez R, Mendell JT, Lupold SE. miR-21: an androgen receptor-regulated microRNA that promotes hormone-dependent and hormone-independent prostate cancer growth. *Cancer Res* 69: 7165–7169, 2009.

509. Rodriguez A, Griffiths-Jones S, Ashurst JL, Bradley A. Identification of mammalian microRNA host genes and transcription units. *Genome Res* 14: 1902–1910, 2004.

510. Rodriguez A, Vigorito E, Clare S, Warren MV, Couttet P, Soond DR, van Dongen S, Grocock RJ, Das PP, Miska EA, Vetrie D, Okkenhaug K, Enright AJ, Dougan G, Turner M, Bradley A. Requirement of bic/microRNA-155 for normal immune function. *Science* 316: 608–611, 2007.

511. Rokhlin OW, Scheinker VS, Taghiyev AF, Bumcrot D, Glover RA, Cohen MB. MicroRNA-34 mediates AR-dependent p53-induced apoptosis in prostate cancer. *Cancer Biol Ther* 7: 1288–1296, 2008.

512. Romania P, Lulli V, Pelosi E, Biffoni M, Peschle C, Marziali G. MicroRNA 155 modulates megakaryopoiesis at progenitor and precursor level by targeting Ets-1 and Meis1 transcription factors. *Br J Haematol* 143: 570–580, 2008.

513. Rosenberg MI, Georges SA, Asawachaicharn A, Analau E, Tapscott SJ. MyoD inhibits Fstl1 and Utrn expression by inducing transcription of miR-206. *J Cell Biol* 175: 77–85, 2006.

514. Rowland RT, Meng X, Cleveland JC, Meldrum DR, Harken AH, Brown JM. Cardio-adaptation induced by cyclic ischemic preconditioning is mediated by translational regulation of de novo protein synthesis. *J Surg Res* 71: 155–160, 1997.

515. Roy S, Khanna S, Hussain SR, Biswas S, Azad A, Rink C, Gnyawali S, Shilo S, Nuovo GJ, Sen CK. MicroRNA expression in response to murine myocardial infarction: miR-21 regulates fibroblast metalloprotease-2 via phosphatase and tensin homologue. *Cardiovasc Res* 82: 21–29, 2009.

516. Ruby JG, Jan CH, Bartel DP. Intronic microRNA precursors that bypass Drosha processing. *Nature* 448: 83–86, 2007.

517. Rupp H. The adaptive changes in the isoenzyme pattern of myosin from hypertrophied rat myocardium as a result of pressure overload and physical training. *Basic Res Cardiol* 76: 79–88, 1981.

518. Rybak A, Fuchs H, Smirnova L, Brandt C, Pohl EE, Nitsch R, Wulczyn FG. A feedback loop comprising lin-28 and let-7 controls pre-let-7 maturation during neural stem-cell commitment. *Nat Cell Biol* 10: 987–993, 2008.

519. Rygaard K, Nakamura T, Spang-Thomsen M. Expression of the proto-oncogenes c-met and c-kit and their ligands, hepatocyte growth factor/scatter factor and stem cell factor, in SCLC cell lines and xenografts. *Br J Cancer* 67: 37–46, 1993.

520. Sachdeva M, Mo YY. MicroRNA-145 suppresses cell invasion and metastasis by directly targeting mucin 1. *Cancer Res* 2009.

521. Sachs A. *Translational Control of Gene Expression*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press, 2000, p. 447–466.

522. Saetrom P, Biesinger J, Li SM, Smith D, Thomas LF, Majzoub K, Rivas GE, Alluin J, Rossi JJ, Krontiris TG, Weitzel J, Daly MB, Benson AB, Kirkwood JM, O’Dwyer PJ, Sutphen R, Stewart JA, Johnson D, Larson GP. A risk variant in an miR-125b binding site in BMPR1B is associated with breast cancer pathogenesis. *Cancer Res* 69: 7459–7465, 2009.

523. Salvi A, Sabelli C, Moncini S, Venturin M, Arici B, Riva P, Portolani N, Giulini SM, De Petro G, Barlati S. MicroRNA-23b mediates urokinase and c-met downmodulation and a decreased migration of human hepatocellular carcinoma cells. *FEBS Lett* 276: 2966–2982, 2009.

524. Samols MA, Hu J, Skalsky RL, Renne R. Cloning and identification of a microRNA cluster within the latency-associated region of Kaposi’s sarcoma-associated herpesvirus. *J Virol* 79: 9301–9305, 2005.

525. Samols MA, Skalsky RL, Maldonado AM, Riva A, Lopez MC, Baker HV, Renne R. Identification of cellular genes targeted by KSHV-encoded microRNAs. *PLoS Pathog* 3: 2007.

526. Sampson VB, Rong NH, Han J, Yang Q, Aris V, Soteropoulos P, Petrelli NJ, Dunn SP, Krueger LJ. MicroRNA let-7a down-regulates MYC and reverts MYC-induced growth in Burkitt lymphoma cells. *Cancer Res* 67: 9762–9770, 2007.

527. Samuelsen CO, Baraznenok V, Khorosjutina O, Spahr H, Kieselbach T, Holmberg S, Gustafsson CM. TRAP230/ARC240 and TRAP240/ARC250 mediator subunits are functionally conserved through evolution. *Proc Natl Acad Sci USA* 100: 6422–6427, 2003.

528. Saraiya AA, Wang CC. snoRNA, a novel precursor of microRNA in Giardia lamblia. *PLoS Pathog* 4: 28, 2008.

529. Sarasin-Filipowicz M, Krol J, Markiewicz I, Heim MH, Filipowicz W. Decreased levels of microRNA miR-122 in individuals with hepatitis C responding poorly to interferon therapy. *Nat Med* 15: 31–33, 2009.

530. Satzger I, Mattern A, Kuettler U, Weinspach D, Voelker B, Kapp A, Gutzmer R. MicroRNA-15b represents an independent prognostic parameter and is correlated with tumor cell proliferation and apoptosis in malignant melanoma. *Int J Cancer* 126: 2553–2562, 2010.

531. Sawyers CL, Callahan W, Witte ON. Dominant negative MYC blocks transformation by ABL oncogenes. *Cell* 70: 901–910, 1992.

532. Sayed D, He M, Hong C, Gao S, Rane S, Yang Z, Abdellatif M. MicroRNA-21 is a downstream effector of AKT that mediates its antiapoptotic effects via suppression of Fas ligand. *J Biol Chem* 285: 20281–20290, 2010.

533. Sayed D, Hong C, Chen IY, Lypowy J, Abdellatif M. MicroRNAs play an essential role in the development of cardiac hypertrophy. *Circ Res* 100: 416–424, 2007.

534. Sayed D, Rane S, Lypowy J, He M, Chen IY, Vashistha H, Yan L, Malhotra A, Vatner D, Abdellatif M. MicroRNA-21 targets Sprouty2 and promotes cellular outgrowths. *Mol Biol Cell* 18: 3272–3282, 2008.

535. Schaefer A, O’Carroll D, Tan CL, Hillman D, Sugimori M, Llinas R, Greengard P. Cerebellar neurodegeneration in the absence of microRNAs. *J Exp Med* 204: 1553–1558, 2007.

536. Schetter AJ, Leung SY, Sohn JJ, Zanetti KA, Bowman ED, Yanaihara N, Yuen ST, Chan TL, Kwong DL, Au GK, Liu CG, Calin GA, Croce CM, Harris CC. MicroRNA expression profiles associated with prognosis and therapeutic outcome in colon adenocarcinoma. *JAMA* 299: 425–436, 2008.

537. Schratt GM, Tuebing F, Nigh EA, Kane CG, Sabatini ME, Kiebler M, Greenberg ME. A brain-specific microRNA regulates dendritic spine development. *Nature* 439: 283–289, 2006.

538. Schultz J, Lorenz P, Gross G, Ibrahim S, Kunz M. MicroRNA let-7b targets important cell cycle molecules in malignant melanoma cells and interferes with anchorage-independent growth. *Cell Res* 18: 549–557, 2008.

539. Scott GK, Goga A, Bhaumik D, Berger CE, Sullivan CS, Benz CC. Coordinate suppression of ERBB2 and ERBB3 by enforced expression of micro-RNA miR-125a or miR-125b. *J Biol Chem* 282: 1479–1486, 2007.

540. Seftor RE, Seftor EA, Sheng S, Pemberton PA, Sager R, Hendrix MJ. Maspin suppresses the invasive phenotype of human breast carcinoma. *Cancer Res* 58: 5681–5685, 1998.

541. Segura MF, Belitskaya-Levy I, Rose AE, Zakrzewski J, Gaziel A, Hanniford D, Darvishian F, Berman RS, Shapiro RL, Pavlick AC, Osman I, Hernando E. Melanoma MicroRNA signature predicts post-recurrence survival. *Clin Cancer Res* 16: 1577–1586, 2010.

542. Seike M, Goto A, Okano T, Bowman ED, Schetter AJ, Horikawa I, Mathe EA, Jen J, Yang P, Sugimura H, Gemma A, Kudoh S, Croce CM, Harris CC. MiR-21 is an EGFR-regulated anti-apoptotic factor in lung cancer in never-smokers. *Proc Natl Acad Sci USA* 106: 12085–12090, 2009.

543. Seitz H, Royo H, Bortolin ML, Lin SP, Ferguson-Smith AC, Cavaille J. A large imprinted microRNA gene cluster at the mouse Dlk1-Gtl2 domain. *Genome Res* 14: 1741–1748, 2004.

544. Sekine S, Ogawa R, Ito R, Hiraoka N, McManus MT, Kanai Y, Hebrok M. Disruption of Dicer1 induces dysregulated fetal gene expression and promotes hepatocarcinogenesis. *Gastroenterology* 136: 2304–2315, 2009.

545. Selkoe DJ. The cell biology of beta-amyloid precursor protein and presenilin in Alzheimer’s disease. *Trends Cell Biol* 8: 447–453, 1998.

546. Selzer E, Wacheck V, Lucas T, Heere-Ress E, Wu M, Weilbaecher KN, Schlegel W, Valent P, Wrba F, Pehamberger H, Fisher D, Jansen B. The melanocyte-specific isoform of the microphthalmia transcription factor affects the phenotype of human melanoma. *Cancer Res* 62: 2098–2103, 2002.

547. Senoo M, Pinto F, Crum CP, McKeon F. p63 is essential for the proliferative potential of stem cells in stratified epithelia. *Cell* 129: 523–536, 2007.

548. Seraj MJ, Samant RS, Verderame MF, Welch DR. Functional evidence for a novel human breast carcinoma metastasis suppressor, BRMS1, encoded at chromosome 11q13. *Cancer Res* 60: 2764–2769, 2000.

549. Sethi P, Lukiw WJ. Micro-RNA abundance and stability in human brain: specific alterations in Alzheimer’s disease temporal lobe neocortex. *Neurosci Lett* 459: 100–104, 2009.

550. Shah YM, Morimura K, Yang Q, Tanabe T, Takagi M, Gonzalez FJ. Peroxisome proliferator-activated receptor alpha regulates a microRNA-mediated signaling cascade responsible for hepatocellular proliferation. *Mol Cell Biol* 27: 4238–4247, 2007.

551. Shao J, Sheng H, Inoue H, Morrow JD, DuBois RN. Regulation of constitutive cyclooxygenase-2 expression in colon carcinoma cells. *J Biol Chem* 275: 33951–33956, 2000.

MicroRNAs IN DEVELOPMENT AND DISEASE

552. Shell S, Park SM, Radjabi AR, Schickel R, Kistner EO, Jewell DA, Feig C, Lengyel E, Peter ME. Let-7 expression defines two differentiation stages of cancer. *Proc Natl Acad Sci USA* 104: 11400–11405, 2007.

553. Sheng H, Shao J, Kirkland SC, Isakson P, Coffey RJ, Morrow J, Beauchamp RD, DuBois RN. Inhibition of human colon cancer cell growth by selective inhibition of cyclooxygenase-2. *J Clin Invest* 99: 2254–2259, 1997.

554. Shi B, Sepp-Lorenzino L, Prisco M, Linsley P, deAngelis T, Baserga R. MicroRNA-145 targets the insulin receptor substrate-1 and inhibits the growth of colon cancer cells. *J Biol Chem* 282: 32582–32590, 2007.

555. Shi J, Zhang L, Shen A, Zhang J, Wang Y, Zhao Y, Zou L, Ke Q, He F, Wang P, Cheng C, Shi G. Clinical and biological significance of forkhead class box O 3a expression in glioma: mediation of glioma malignancy by transcriptional regulation of p27(kip1). *J Neurooncol* 98: 57–69, 2010.

556. Shi S, Yu L, Chiu C, Sun Y, Chen J, Khitrov G, Merkenschlager M, Holzman LB, Zhang W, Mundel P, Bottinger EP. Podocyte-selective deletion of dicer induces proteinuria and glomerulosclerosis. *J Am Soc Nephrol* 19: 2159–2169, 2008.

557. Shi XB, Xue L, Yang J, Ma AH, Zhao J, Xu M, Tepper CG, Evans CP, Kung HJ, deVere White RW. An androgen-regulated miRNA suppresses Bak1 expression and induces androgen-independent growth of prostate cancer cells. *Proc Natl Acad Sci USA* 104: 19983–19988, 2007.

558. Shibata M, Kurokawa D, Nakao H, Ohmura T, Aizawa S. MicroRNA-9 modulates Cajal-Retzius cell differentiation by suppressing Foxgl expression in mouse medial pallium. *J Neurosci* 28: 10415–10421, 2008.

559. Shimono Y, Zabala M, Cho RW, Lobo N, Dalerba P, Qian D, Diehn M, Liu H, Panula SP, Chiao E, Dirbas FM, Somlo G, Pera RA, Lao K, Clarke MF. Downregulation of miRNA-200c links breast cancer stem cells with normal stem cells. *Cell* 138: 592–603, 2009.

560. Si ML, Zhu S, Wu H, Lu Z, Wu F, Mo YY. miR-21-mediated tumor growth. *Oncogene* 26: 2799–2803, 2007.

561. Siegel G, Obernosterer G, Fiore R, Oehmen M, Bicker S, Christensen M, Khudyayberdiev S, Leuschner PF, Busch CJ, Kane C, Hubel K, Dekker F, Hedberg C, Rengan B, Drepper C, Waldmann H, Kauppinen S, Greenberg ME, Draguhn A, Rehmsmeier M, Martinez J, Schratt GM. A functional screen implicates microRNA-138-dependent regulation of the depalmitoylation enzyme APT1 in dendritic spine morphogenesis. *Nat Cell Biol* 11: 705–716, 2009.

562. Silber J, Lim DA, Petritsch C, Persson AI, Maunakea AK, Yu M, Vandenberg SR, Ginzinger DG, James CD, Costello JF, Bergers G, Weiss WA, Alvarez-Buylla A, Hodgson JG. miR-124 and miR-137 inhibit proliferation of glioblastoma multiforme cells and induce differentiation of brain tumor stem cells. *BMC Med* 6: 14, 2008.

563. Sizova DV, Kolupaeva VG, Pestova TV, Shatsky IN, Hellen CU. Specific interaction of eukaryotic translation initiation factor 3 with the 5′ nontranslated regions of hepatitis C virus and classical swine fever virus RNAs. *J Virol* 72: 4775–4782, 1998.

564. Skalsky RL, Samols MA, Plaisance KB, Boss IW, Riva A, Lopez MC, Baker HV, Renne R. Kaposi’s sarcoma-associated herpesvirus encodes an ortholog of miR-155. *J Virol* 81: 12836–12845, 2007.

565. Slaby O, Svoboda M, Fabian P, Smerdova T, Knoflickova D, Bednarikova M, Nenutil R, Vyzula R. Altered expression of miR-21, miR-31, miR-143 and miR-145 is related to clinicopathologic features of colorectal cancer. *Oncology* 72: 397–402, 2007.

566. Slattery ML, Samowitz W, Curtin K, Ma KN, Hoffman M, Caan B, Neuhausen S. Associations among IRS1, IRS2, IGF1, IGFBP3 genetic polymorphisms and colorectal cancer. *Cancer Epidemiol Biomarkers Prev* 13: 1206–1214, 2004.

567. Smirnova L, Grafe A, Seiler A, Schumacher S, Nitsch R, Wulczyn FG. Regulation of miRNA expression during neural cell specification. *Eur J Neurosci* 21: 1469–1477, 2005.

568. Sokol NS, Ambros V. Mesodermally expressed Drosophila microRNA-1 is regulated by Twist and is required in muscles during larval growth. *Genes Dev* 19: 2343–2354, 2005.

569. Song B, Wang Y, Kudo K, Gavin EJ, Xi Y, Ju J. miR-192 regulates dihydrofolate reductase and cellular proliferation through the p53-microRNA circuit. *Clin Cancer Res* 14: 8080–8086, 2008.

570. Song LB, Li J, Liao WT, Feng Y, Yu CP, Hu LJ, Kong QL, Xu LH, Zhang X, Liu WL, Li MZ, Zhang L, Kang TB, Fu LW, Huang WL, Xia YF, Tsao SW, Li M, Band V, Band H, Shi QH, Zeng YX, Zeng MS. The polycomb group protein Bmi-1 represses the tumor suppressor PTEN and induces epithelial-mesenchymal transition in human nasopharyngeal epithelial cells. *J Clin Invest* 119: 3626–3636, 2009.

571. Sonkoly E, Wei T, Janson PC, Saaf A, Lundeberg L, Tengvall-Linder M, Norstedt G, Alenius H, Homey B, Scheynius A, Stahle M, Pivarcsi A. MicroRNAs: novel regulators involved in the pathogenesis of psoriasis? *PLoS One* 2: 2007.

572. Sonoki T, Iwanaga E, Mitsuya H, Asou N. Insertion of microRNA-125b-1, a human homologue of lin-4, into a rearranged immunoglobulin heavy chain gene locus in a patient with precursor B-cell acute lymphoblastic leukemia. *Leukemia* 19: 2009–2010, 2005.

573. Soukup GA, Fritzsch B, Pierce ML, Weston MD, Jahan I, McManus MT, Harfe BD. Residual microRNA expression dictates the extent of inner ear development in conditional Dicer knockout mice. *Dev Biol* 328: 328–341, 2009.

574. Spaderna S, Schmalhofer O, Hlubek F, Berx G, Eger A, Merkel S, Jung A, Kirchner T, Brabletz T. A transient, EMT-linked loss of basement membranes indicates metastasis and poor survival in colorectal cancer. *Gastroenterology* 131: 830–840, 2006.

575. Spaderna S, Schmalhofer O, Wahlbuhl M, Dimmler A, Bauer K, Sultan A, Hlubek F, Jung A, Strand D, Eger A, Kirchner T, Behrens J, Brabletz T. The transcriptional repressor ZEB1 promotes metastasis and loss of cell polarity in cancer. *Cancer Res* 68: 537–544, 2008.

576. Spahn M, Kneitz S, Scholz CJ, Nico S, Rudiger T, Strobel P, Redmilller H, Kneitz B. Expression of microRNA-221 is progressively reduced in aggressive prostate cancer and metastasis and predicts clinical recurrence. *Int J Cancer* 127: 394–403, 2010.

577. Srivastava D, Thomas T, Lin Q, Kirby ML, Brown D, Olson EN. Regulation of cardiac mesodermal and neural crest development by the bHLH transcription factor, dHAND. *Nat Genet* 16: 154–160, 1997.

578. Stabile LP, Lyker JS, Land SR, Dacic S, Zamboni BA, Siegfried JM. Transgenic mice overexpressing hepatocyte growth factor in the airways show increased susceptibility to lung cancer. *Carcinogenesis* 27: 1547–1555, 2006.

579. Stark KL, Xu B, Bagchi A, Lai WS, Liu H, Hsu R, Wan X, Pavlidis P, Mills AA, Karayiorgou M, Gogos JA. Altered brain microRNA biogenesis contributes to phenotypic deficits in a 22q11-deletion mouse model. *Nat Genet* 40: 751–760, 2008.

580. Stoecklin G, Gross B, Ming XF, Moroni C. A novel mechanism of tumor suppression by destabilizing AU-rich growth factor mRNA. *Oncogene* 22: 3554–3561, 2003.

581. Stolt CC, Lommes P, Sock E, Chaboissier MC, Schedl A, Wegner M. The Sox9 transcription factor determines glial fate choice in the developing spinal cord. *Genes Dev* 17: 1677–1689, 2003.

582. Strillacci A, Griffoni C, Sansone P, Paterini P, Piazzi G, Lazzarini G, Spisni E, Pantaleo MA, Biasco G, Tomasi V. MiR-101 downregulation is involved in cyclooxygenase-2 overexpression in human colon cancer cells. *Exp Cell Res* 315: 1439–1447, 2009.

583. Su H, Yang JR, Xu T, Huang J, Xu L, Yuan Y, Zhuang SM. MicroRNA-101, downregulated in hepatocellular carcinoma, promotes apoptosis and suppresses tumorigenicity. *Cancer Res* 69: 1135–1142, 2009.

584. Suarez Y, Fernandez-Hernando C, Pober JS, Sessa WC. Dicer dependent microRNAs regulate gene expression and functions in human endothelial cells. *Circ Res* 100: 1164–1173, 2007.

585. Sucharov C, Bristow MR, Port JD. miRNA expression in the failing human heart: functional correlates. *J Mol Cell Cardiol* 45: 185–192, 2008.

586. Sun T, Wang Q, Balk S, Brown M, Lee GS, Kantoff P. The role of microRNA-221 and microRNA-222 in androgen-independent prostate cancer cell lines. *Cancer Res* 69: 3356–3363, 2009.

587. Sung TL, Rice AP. miR-198 inhibits HIV-1 gene expression and replication in monocytes and its mechanism of action appears to involve repression of cyclin T1. *PLoS Pathog* 5: 16, 2009.

588. Suzuki T, Sawaki D, Aizawa K, Munemasa Y, Matsumura T, Ishida J, Nagai R. Krüppel-like factor 5 shows proliferation-specific roles in vascular remodeling, direct stimulation of cell growth, and inhibition of apoptosis. *J Biol Chem* 284: 9549–9557, 2009.

589. Svensson EC, Huggins GS, Lin H, Clendenin C, Jiang F, Tufts R, Dardik FB, Leiden JM. A syndrome of tricuspid atresia in mice with a targeted mutation of the gene encoding Fog-2. Nat Genet 25: 353–356, 2000.

590. Sylvestre Y, De Guire V, Querido E, Mukhopadhyay UK, Bourdeau V, Major F, Ferbeyre G, Chartrand P. An E2F/miR-20a autoregulatory feedback loop. J Biol Chem 282: 2135–2143, 2007.

591. Szafranska AE, Davison TS, John J, Cannon T, Sipos B, Maghnouj A, Labourier E, Hahn SA. MicroRNA expression alterations are linked to tumorigenesis and non-neoplastic processes in pancreatic ductal adenocarcinoma. Oncogene 26: 4442–4452, 2007.

592. Taft RJ, Glazov EA, Lassmann T, Hayashizaki Y, Carninci P, Mattick JS. Small RNAs derived from snoRNAs. RNA 15: 1233–1240, 2009.

593. Taganov KD, Boldin MP, Chang KJ, Baltimore D. NF-κB-dependent induction of microRNA miR-146, an inhibitor targeted to signaling proteins of innate immune responses. Proc Natl Acad Sci USA 2: 2, 2006.

594. Takahashi Y, Forrest AR, Maeno E, Hashimoto T, Daub CO, Yasuda J. MiR-107 and MiR-185 can induce cell cycle arrest in human non small cell lung cancer cell lines. PLoS One 4: 2009.

595. Takamizawa J, Konishi H, Yanagisawa K, Tomida S, Osada H, Endoh H, Harano T, Yatabe Y, Nagino M, Nimura Y, Mitsudomi T, Takahashi T. Reduced expression of the let-7 microRNAs in human lung cancers in association with shortened postoperative survival. Cancer Res 64: 3753–3756, 2004.

596. Tam W, Ben-Yehuda D, Hayward WS. bic, a novel gene activated by proviral insertions in avian leukosis virus-induced lymphomas, is likely to function through its noncoding RNA. Mol Cell Biol 17: 1490–1502, 1997.

597. Tang X, Muniappan L, Tang G, Ozcan S. Identification of glucose-regulated miRNAs from pancreatic β cells reveals a role for miR-30d in insulin transcription. RNA 15: 287–293, 2009.

598. Taniguchi K, Kohno R, Ayada T, Kato R, Ichiyama K, Morisada T, Oike Y, Yonemitsu Y, Maehara Y, Yoshimura A. Spreds are essential for embryonic lymphangiogenesis by regulating vascular endothelial growth factor receptor 3 signaling. Mol Cell Biol 27: 4541–4550, 2007.

599. Tarasov V, Jung P, Verdoodt B, Lodygin D, Epanchintsev A, Menssen A, Meister G, Hermeking H. Differential regulation of microRNAs by p53 revealed by massively parallel sequencing: miR-34a is a p53 target that induces apoptosis and G1-arrest. Cell Cycle 6: 1586–1593, 2007.

600. Tatsuguchi M, Seok HY, Callis TE, Thomson JM, Chen JF, Newman M, Rojas M, Hammond SM, Wang DZ. Expression of microRNAs is dynamically regulated during cardiomyocyte hypertrophy. J Mol Cell Cardiol 42: 1137–1141, 2007.

601. Taulli R, Bersani F, Foglizzo V, Linari A, Vigna E, Ladanyi M, Tuschl T, Ponzetto C. The muscle-specific microRNA miR-206 blocks human rhabdomyosarcoma growth in xenotransplanted mice by promoting myogenic differentiation. J Clin Invest 119: 2366–2378, 2009.

602. Tavazoie SF, Alarcon C, Oskarsson T, Padua D, Wang Q, Bos PD, Gerald WL, Massague J. Endogenous human microRNAs that suppress breast cancer metastasis. Nature 451: 147–152, 2008.

603. Tazawa H, Tsuchiya N, Izumiya M, Nakagama H. Tumor-suppressive miR-34a induces senescence-like growth arrest through modulation of the E2F pathway in human colon cancer cells. Proc Natl Acad Sci USA 104: 15472–15477, 2007.

604. Teng G, Hakimpour P, Landgraf P, Rice A, Tuschl T, Casellas R, Papavasiliou FN. MicroRNA-155 is a negative regulator of activation-induced cytidine deaminase. Immunity 28: 621–629, 2008.

605. Tevosian SG, Deconinck AE, Tanaka M, Schinke M, Litovsky SH, Izumo S, Fujiwara Y, Orkin SH. FOG-2, a cofactor for GATA transcription factors, is essential for heart morphogenesis and development of coronary vessels from epicardium. Cell 101: 729–739, 2000.

606. Thadani R, Tammi MT. MicroTar: predicting microRNA targets from RNA duplexes. BMC Bioinformatics 7 Suppl 5: S20, 2006.

607. Thai TH, Calado DP, Casola S, Ansel KM, Xiao C, Xue Y, Murphy A, Frendewey D, Valenzuela D, Kutok JL, Schmidt-Supprian M, Rajewsky N, Yancopoulos G, Rao A, Rajewsky K. Regulation of the germinal center response by microRNA-155. Science 316: 604–608, 2007.

608. Thermann R, Hentze MW. Drosophila miR2 induces pseudo-polysomes and inhibits translation initiation. Nature 447: 875–878, 2007.

609. Thomas MD, Kremer CS, Ravichandran KS, Rajewsky K, Bender TP. c-Myb is critical for B cell development and maintenance of follicular B cells. Immunity 23: 275–286, 2005.

610. Thomson JM, Newman M, Parker JS, Morin-Kensicki EM, Wright T, Hammond SM. Extensive post-transcriptional regulation of microRNAs and its implications for cancer. Genes Dev 1: 1, 2006.

611. Thum T, Galuppo P, Wolf C, Fiedler J, Kneitz S, van Laake LW, Doevendans PA, Mummery CL, Borlak J, Haverich A, Gross C, Engelhardt S, Ertl G, Bauersachs J. MicroRNAs in the human heart: a clue to fetal gene reprogramming in heart failure. Circulation 116: 258–267, 2007.

612. Thum T, Gross C, Fiedler J, Fischer T, Kissler S, Bussen M, Galuppo P, Just S, Rottbauer W, Frantz S, Castoldi M, Soutschek J, Koteliansky V, Rosenwald A, Basson MA, Licht JD, Pena JT, Rouhanifard SH, Muckenthaler MU, Tuschl T, Martin GR, Bauersachs J, Engelhardt S. MicroRNA-21 contributes to myocardial disease by stimulating MAP kinase signalling in fibroblasts. Nature 456: 980–984, 2008.

613. Tikhonenko AT, Black DJ, Linial ML. Viral Myc oncoproteins in infected fibroblasts down-modulate thrombospondin-1, a possible tumor suppressor gene. J Biol Chem 271: 30741–30747, 1996.

614. Tili E, Michaille JJ, Cimino A, Costinean S, Dumitru CD, Adair B, Fabbri M, Alder H, Liu CG, Calin GA, Croce CM. Modulation of miR-155 and miR-125b levels following lipopolysaccharide/TNF-alpha stimulation and their possible roles in regulating the response to endotoxin shock. J Immunol 179: 5082–5089, 2007.

615. Toh Y, Nicolson GL. The role of the MTA family and their encoded proteins in human cancers: molecular functions and clinical implications. Clin Exp Metastasis 26: 215–227, 2009.

616. Toole BP. Hyaluronan-CD44 interactions in cancer: paradoxes and possibilities. Clin Cancer Res 15: 7462–7468, 2009.

617. Toyota M, Suzuki H, Sasaki Y, Maruyama R, Imai K, Shinomura Y, Tokino T. Epigenetic silencing of microRNA-34b/c and B-cell translocation gene 4 is associated with CpG island methylation in colorectal cancer. Cancer Res 68: 4123–4132, 2008.

618. Trabucchi M, Briata P, Garcia-Mayoral M, Haase AD, Filipowicz W, Ramos A, Gherzi R, Rosenfeld MG. The RNA-binding protein KSRP promotes the biogenesis of a subset of microRNAs. Nature 459: 1010–1014, 2009.

619. Trang P, Medina PP, Wiggins JF, Ruffino L, Kelnar K, Omotola M, Homer R, Brown D, Bader AG, Weidhaas JB, Slack FJ. Regression of murine lung tumors by the let-7 microRNA. Oncogene 7: 7, 2009.

620. Triboulet R, Mari B, Lin YL, Chable-Bessia C, Bennasser Y, Lebrigand K, Cardinaud B, Maurin T, Barbry P, Baillat V, Reynes J, Corbeau P, Jeang KT, Benkirane M. Suppression of microRNA-silencing pathway by HIV-1 during virus replication. Science 315: 1579–1582, 2007.

621. Tsai WC, Hsu PW, Lai TC, Chau GY, Lin CW, Chen CM, Lin CD, Liao YL, Wang JL, Chau YP, Hsu MT, Hsiao M, Huang HD, Tsou AP. MicroRNA-122, a tumor suppressor microRNA that regulates intrahepatic metastasis of hepatocellular carcinoma. Hepatology 49: 1571–1582, 2009.

622. Uziel T, Karginov FV, Xie S, Parker JS, Wang YD, Gajjar A, He L, Ellison D, Gilbertson RJ, Hannon G, Roussel MF. The miR-17~92 cluster collaborates with the Sonic Hedgehog pathway in medulloblastoma. Proc Natl Acad Sci USA 106: 2812–2817, 2009.

623. Vaillant C, Monard D. SHH pathway and cerebellar development. Cerebellum 8: 291–301, 2009.

624. Valastyan S, Reinhardt F, Benaich N, Calogrias D, Szasz AM, Wang ZC, Brock JE, Richardson AL, Weinberg RA. A pleiotropically acting microRNA, miR-31, inhibits breast cancer metastasis. Cell 137: 1032–1046, 2009.

625. Van Rooij E, Quiat D, Johnson BA, Sutherland LB, Qi X, Richardson JA, Kelm RJ Jr, Olson EN. A family of microRNAs encoded by myosin genes governs myosin expression and muscle performance. Dev Cell 17: 662–673, 2009.

626. Van Rooij E, Sutherland LB, Liu N, Williams AH, McAnally J, Gerard RD, Richardson JA, Olson EN. A signature pattern of stress-responsive microRNAs that can

evoke cardiac hypertrophy and heart failure. Proc Natl Acad Sci USA 103: 18255–18260, 2006.

627. Van Rooij E, Sutherland LB, Qi X, Richardson JA, Hill J, Olson EN. Control of stress-dependent cardiac growth and gene expression by a MicroRNA. Science 316: 575–579, 2007.

628. Van Rooij E, Sutherland LB, Thatcher JE, DiMaio JM, Naseem RH, Marshall WS, Hill JA, Olson EN. Dysregulation of microRNAs after myocardial infarction reveals a role of miR-29 in cardiac fibrosis. Proc Natl Acad Sci USA 105: 13027–13032, 2008.

629. Varambally S, Cao Q, Mani RS, Shankar S, Wang X, Ateeq B, Laxman B, Cao X, Jing X, Ramnarayanan K, Brenner JC, Yu J, Kim JH, Han B, Tan P, Kumar-Sinha C, Lonigro RJ, Palanisamy N, Maher CA, Chinnaiyan AM. Genomic loss of microRNA-101 leads to overexpression of histone methyltransferase EZH2 in cancer. Science 322: 1695–1699, 2008.

630. Varambally S, Dhanasekaran SM, Zhou M, Barrette TR, Kumar-Sinha C, Sanda MG, Ghosh D, Pienta KJ, Sewalt RG, Otte AP, Rubin MA, Chinnaiyan AM. The polycomb group protein EZH2 is involved in progression of prostate cancer. Nature 419: 624–629, 2002.

631. Vasudevan S, Tong Y, Steitz JA. Switching from repression to activation: microRNAs can up-regulate translation. Science 318: 1931–1934, 2007.

632. Velu CS, Baktula AM, Grimes HL. Gfil regulates miR-21 and miR-196b to control myelopoiesis. Blood 113: 4720–4728, 2009.

633. Ventura A, Young AG, Winslow MM, Lintault L, Meissner A, Erkeland SJ, Newman J, Bronson RT, Crowley D, Stone JR, Jaenisch R, Sharp PA, Jacks T. Targeted deletion reveals essential and overlapping functions of the miR-17 through 92 family of miRNA clusters. Cell 132: 875–886, 2008.

634. Venturini L, Battmer K, Castoldi M, Schulteis B, Hochhaus A, Muckenthaler MU, Ganser A, Eder M, Scherr M. Expression of the miR-17–92 polycistron in chronic myeloid leukemia (CML) CD34+ cells. Blood 109: 4399–4405, 2007.

635. Verras M, Brown J, Li X, Nusse R, Sun Z. Wnt3a growth factor induces androgen receptor-mediated transcription and enhances cell growth in human prostate cancer cells. Cancer Res 64: 8860–8866, 2004.

636. Vigorito E, Perks KL, Abreu-Goodger C, Bunting S, Xiang Z, Kohlhaas S, Das PP, Miska EA, Rodriguez A, Bradley A, Smith KG, Rada C, Enright AJ, Toellner KM, Maclennan IC, Turner M. microRNA-155 regulates the generation of immunoglobulin class-switched plasma cells. Immunity 27: 847–859, 2007.

637. Visvanathan J, Lee S, Lee B, Lee JW, Lee SK. The microRNA miR-124 antagonizes the anti-neural REST/SCP1 pathway during embryonic CNS development. Genes Dev 21: 744–749, 2007.

638. Viswanathan SR, Daley GQ, Gregory RI. Selective blockade of microRNA processing by Lin28. Science 320: 97–100, 2008.

639. Volinia S, Calin GA, Liu CG, Ambs S, Croce CM, Petrocca F, Visone R, Iorio M, Roldo C, Ferracin M, Prueitt RL, Yanaihara N, Lanza G, Scarpa A, Vecchione A, Negrini M, Harris CC. A microRNA expression signature of human solid tumors defines cancer gene targets. Proc Natl Acad Sci USA 103: 2257–2261, 2006.

640. Wahrman J, Voss R, Shapiro T, Ashkenazi A. The Philadelphia chromosome in two children with chronic myeloid leukemia. Isr J Med Sci 3: 380–391, 1967.

641. Wakiyama M, Takimoto K, Ohara O, Yokoyama S. Let-7 microRNA-mediated mRNA deadenylation and translational repression in a mammalian cell-free system. Genes Dev 21: 1857–1862, 2007.

642. Walden TB, Timmons JA, Keller P, Nedergaard J, Cannon B. Distinct expression of muscle-specific microRNAs (myomirs) in brown adipocytes. J Cell Physiol 218: 444–449, 2009.

643. Wang B, Majumder S, Nuovo G, Kutay H, Volinia S, Patel T, Schmittgen TD, Croce C, Ghoshal K, Jacob ST. Role of microRNA-155 at early stages of hepatocarcinogenesis induced by choline-deficient and amino acid-defined diet in C57BL/6 mice. Hepatology 50: 1152–1161, 2009.

644. Wang G, Mao W, Zheng S. MicroRNA-183 regulates Ezrin expression in lung cancer cells. FEBS Lett 582: 3663–3668, 2008.

645. Wang G, Mao W, Zheng S, Ye J. Epidermal growth factor receptor-regulated miR-125a-5p—a metastatic inhibitor of lung cancer. FEBS Lett 276: 5571–5578, 2009.

646. Wang G, van der Walt JM, Mayhew G, Li YJ, Zuchner S, Scott WK, Martin ER, Vance JM. Variation in the miRNA-433 binding site of FGF20 confers risk for Parkinson disease by overexpression of alpha-synuclein. Am J Hum Genet 82: 283–289, 2008.

647. Wang H, Garzon R, Sun H, Ladner KJ, Singh R, Dahlman J, Cheng A, Hall BM, Qualman SJ, Chandler DS, Croce CM, Guttridge DC. NF-kappaB-YY1-miR-29 regulatory circuitry in skeletal myogenesis and rhabdomyosarcoma. Cancer Cell 14: 369–381, 2008.

648. Wang HU, Chen ZF, Anderson DJ. Molecular distinction and angiogenic interaction between embryonic arteries and veins revealed by ephrin-B2 and its receptor Eph-B4. Cell 93: 741–753, 1998.

649. Wang M, Tan LP, Dijkstra MK, van Lom K, Robertus JL, Harms G, Blokzijl T, Kooistra K, van T’veer MB, Rosati S, Visser L, Jongen-Lavrencic M, Kluijn PM, van den Berg A. miRNA analysis in B-cell chronic lymphocytic leukaemia: proliferation centres characterized by low miR-150 and high BIC/miR-155 expression. J Pathol 215: 13–20, 2008.

650. Wang Q, Huang Z, Xue H, Jin C, Ju XL, Han JD, Chen YG. MicroRNA miR-24 inhibits erythropoiesis by targeting activin type I receptor ALK4. Blood 111: 588–595, 2008.

651. Wang Q, Wang Y, Minto AW, Wang J, Shi Q, Li X, Quigg RJ. MicroRNA-377 is up-regulated and can lead to increased fibronectin production in diabetic nephropathy. FASEB J 22: 4126–4135, 2008.

652. Wang S, Aurora AB, Johnson BA, Qi X, McAnally J, Hill JA, Richardson JA, Bassel-Duby R, Olson EN. The endothelial-specific microRNA miR-126 governs vascular integrity and angiogenesis. Dev Cell 15: 261–271, 2008.

653. Wang WX, Rajeev BW, Stromberg AJ, Ren N, Tang G, Huang Q, Rigoutsos I, Nelson PT. The expression of microRNA miR-107 decreases early in Alzheimer’s disease and may accelerate disease progression through regulation of beta-site amyloid precursor protein-cleaving enzyme 1. J Neurosci 28: 1213–1223, 2008.

654. Wang X, Ye L, Hou W, Zhou Y, Wang YJ, Metzger DS, Ho WZ. Cellular microRNA expression correlates with susceptibility of monocytes/macrophages to HIV-1 infection. Blood 113: 671–674, 2009.

655. Wang Y, Liu CL, Storey JD, Tibshirani RJ, Herschlag D, Brown PO. Precision and functional specificity in mRNA decay. Proc Natl Acad Sci USA 99: 5860–5865, 2002.

656. Watanabe N, Kurabayashi M, Shimomura Y, Kawai-Kowase K, Hoshino Y, Manabe I, Watanabe M, Aikawa M, Kuro-o M, Suzuki T, Yazaki Y, Nagai R. BTEB2, a Kruppel-like transcription factor, regulates expression of the SMemb/Nonmuscle myosin heavy chain B (SMemb/NMHC-B) gene. Circ Res 85: 182–191, 1999.

657. Wei T, Orfanidis K, Xu N, Janson P, Stahle M, Pivarcsi A, Sonkoly E. The expression of microRNA-203 during human skin morphogenesis. Exp Dermatol 19: 854–856, 2010.

658. Weiss FU, Marques IJ, Woltering JM, Vlecken DH, Aghdassi A, Partecke LI, Heidecke CD, Lerch MM, Bagowski CP. Retinoic acid receptor antagonists inhibit miR-10a expression and block metastatic behavior of pancreatic cancer. Gastroenterology 137: 2136–2145, 2009.

659. Wellner U, Schubert J, Burk UC, Schmalhofer O, Zhu F, Sonntag A, Waldvogel B, Vannier C, Darling D, zur Hausen A, Brunton VG, Morton J, Sansom O, Schuler J, Stemmler MP, Herzberger C, Hopt U, Keck T, Brabletz S, Brabletz T. The EMT-activator ZEB1 promotes tumorigenicity by repressing stemness-inhibiting microRNAs. Nat Cell Biol 11: 1487–1495, 2009.

660. Weston MD, Pierce ML, Rocha-Sanchez S, Beisel KW, Soukup GA. MicroRNA gene expression in the mouse inner ear. Brain Res 21: 95–104, 2006.

661. Wienholds E, Kloosterman WP, Miska E, Alvarez-Saavedra E, Berezikov E, de Bruijn E, Horvitz HR, Kauppinen S, Plasterk RH. MicroRNA expression in zebrafish embryonic development. Science 309: 310–311, 2005.

662. Wiggins JF, Ruffino L, Kelnar K, Omotola M, Patrawala L, Brown D, Bader AG. Development of a lung cancer therapeutic based on the tumor suppressor microRNA-34. Cancer Res 70: 5923–5930, 2010.

663. Williams AH, Valdez G, Moresi V, Qi X, McAnally J, Elliott JL, Bassel-Duby R, Sanes JR, Olson EN. MicroRNA-206 delays ALS progression and promotes regeneration of neuromuscular synapses in mice. Science 326: 1549–1554, 2009.

664. Willis MS, Ike C, Li L, Wang DZ, Glass DJ, Patterson C. Muscle ring finger 1, but not muscle ring finger 2, regulates cardiac hypertrophy in vivo. Circ Res 100: 456–459, 2007.

DANISH SAYED AND MAHA ABDELLATIF

665. Wilson AJ, Byun DS, Nasser S, Murray LB, Ayyanar K, Arango D, Figueroa M, Melnick A, Kao GD, Augenlicht LH, Mariadason JM. HDAC4 promotes growth of colon cancer cells via repression of p21. Mol Biol Cell 19: 4062–4075, 2008.

666. Wong CF, Tellam RL. MicroRNA-26a targets the histone methyltransferase Enhancer of Zeste homolog 2 during myogenesis. J Biol Chem 283: 9836–9843, 2008.

667. Wong QW, Lung RW, Law PT, Lai PB, Chan KY, To KF, Wong N. MicroRNA-223 is commonly repressed in hepatocellular carcinoma and potentiates expression of Stathmin1. Gastroenterology 135: 257–269, 2008.

668. Woods K, Thomson JM, Hammond SM. Direct regulation of an oncogenic micro-RNA cluster by E2F transcription factors. J Biol Chem 282: 2130–2134, 2007.

669. Wu H, Zhu S, Mo YY. Suppression of cell growth and invasion by miR-205 in breast cancer. Cell Res 19: 439–448, 2009.

670. Wu J, Xie X. Comparative sequence analysis reveals an intricate network among REST, CREB and miRNA in mediating neuronal gene expression. Genome Biol 7: R85, 2006.

671. Wu L, Fan J, Belasco JG. MicroRNAs direct rapid deadenylation of mRNA. Proc Natl Acad Sci USA 103: 4034–4039, 2006.

672. Xia H, Qi Y, Ng SS, Chen X, Chen S, Fang M, Li D, Zhao Y, Ge R, Li G, Chen Y, He ML, Kung HF, Lai L, Lin MC. MicroRNA-15b regulates cell cycle progression by targeting cyclins in glioma cells. Biochem Biophys Res Commun 380: 205–210, 2009.

673. Xia H, Qi Y, Ng SS, Chen X, Li D, Chen S, Ge R, Jiang S, Li G, Chen Y, He ML, Kung HF, Lai L, Lin MC. microRNA-146b inhibits glioma cell migration and invasion by targeting MMPs. Brain Res 1269: 158–165, 2009.

674. Xia T, O'Hara A, Araujo I, Barreto J, Carvalho E, Sapucaia JB, Ramos JC, Luz E, Pedroso C, Manrique M, Toomey NL, Brites C, Dittmer DP, Harrington WJ Jr. EBV microRNAs in primary lymphomas and targeting of CXCL-11 by ebv-mir-BHRF1-3. Cancer Res 68: 1436–1442, 2008.

675. Xiao C, Calado DP, Galler G, Thai TH, Patterson HC, Wang J, Rajewsky N, Bender TP, Rajewsky K. MiR-150 controls B cell differentiation by targeting the transcription factor c-Myb. Cell 131: 146–159, 2007.

676. Xiao C, Srinivasan L, Calado DP, Patterson HC, Zhang B, Wang J, Henderson JM, Kutok JL, Rajewsky K. Lymphoproliferative disease and autoimmunity in mice with increased miR-17–92 expression in lymphocytes. Nat Immunol 9: 405–414, 2008.

677. Xie Y, Xu K, Dai B, Guo Z, Jiang T, Chen H, Qiu Y. The 44 kDa Pim-1 kinase directly interacts with tyrosine kinase Etk/BMX and protects human prostate cancer cells from apoptosis induced by chemotherapeutic drugs. Oncogene 25: 70–78, 2006.

678. Xin M, Small EM, Sutherland LB, Qi X, McNally J, Plato CF, Richardson JA, Bassel-Duby R, Olson EN. MicroRNAs miR-143 and miR-145 modulate cytoskeletal dynamics and responsiveness of smooth muscle cells to injury. Genes Dev 23: 2166–2178, 2009.

679. Xu C, Lu Y, Pan Z, Chu W, Luo X, Lin H, Xiao J, Shan H, Wang Z, Yang B. The muscle-specific microRNAs miR-1 and miR-133 produce opposing effects on apoptosis by targeting HSP60, HSP70 and caspase-9 in cardiomyocytes. J Cell Sci 120: 3045–3052, 2007.

680. Xu N, Papagiannakopoulos T, Pan G, Thomson JA, Kosik KS. MicroRNA-145 regulates OCT4, SOX2, KLF4 and represses pluripotency in human embryonic stem cells. Cell 137: 647–658, 2009.

681. Xu T, Zhu Y, Xiong Y, Ge YY, Yun JP, Zhuang SM. MicroRNA-195 suppresses tumorigenicity and regulates G1/S transition of human hepatocellular carcinoma cells. Hepatology 50: 113–121, 2009.

682. Xu Y, Li F, Zhang B, Zhang K, Zhang F, Huang X, Sun N, Ren Y, Sui M, Liu P. MicroRNAs and target site screening reveals a pre-microRNA-30e variant associated with schizophrenia. Schizophr Res 119: 219–227, 2010.

683. Yamakuchi M, Ferlito M, Lowenstein CJ. miR-34a repression of SIRT1 regulates apoptosis. Proc Natl Acad Sci USA 28: 28, 2008.

684. Yan D, Dong Xda E, Chen X, Wang L, Lu C, Wang J, Qu J, Tu L. MicroRNA-1/206 targets c-Met and inhibits rhabdomyosarcoma development. J Biol Chem 284: 29596–29604, 2009.

685. Yan LX, Huang XF, Shao Q, Huang MY, Deng L, Wu QL, Zeng YX, Shao JY. MicroRNA miR-21 overexpression in human breast cancer is associated with advanced clinical stage, lymph node metastasis and patient poor prognosis. RNA 14: 2348–2360, 2008.

686. Yang B, Lin H, Xiao J, Lu Y, Luo X, Li B, Zhang Y, Xu C, Bai Y, Wang H, Chen G, Wang Z. The muscle-specific microRNA miR-1 regulates cardiac arrhythmogenic potential by targeting GJA1 and KCNJ2. Nat Med 13: 486–491, 2007.

687. Yang HS, Cho MH, Zakowicz H, Hegamyer G, Sonenberg N, Colburn NH. A novel function of the MA-3 domains in transformation and translation suppressor Pdcd4 is essential for its binding to eukaryotic translation initiation factor 4A. Mol Cell Biol 24: 3894–3906, 2004.

688. Yang J, Mani SA, Donaher JL, Ramaswamy S, Itzykson RA, Come C, Savagner P, Gitelman I, Richardson A, Weinberg RA. Twist, a master regulator of morphogenesis, plays an essential role in tumor metastasis. Cell 117: 927–939, 2004.

689. Yang N, Kaur S, Volinia S, Greshock J, Lassus H, Hasegawa K, Liang S, Leminen A, Deng S, Smith L, Johnstone CN, Chen XM, Liu CG, Huang Q, Katsaros D, Calin GA, Weber BL, Butzow R, Croce CM, Coukos G, Zhang L. MicroRNA microarray identifies Let-7i as a novel biomarker and therapeutic target in human epithelial ovarian cancer. Cancer Res 68: 10307–10314, 2008.

690. Yang WJ, Yang DD, Na S, Sandusky GE, Zhang Q, Zhao G. Dicer is required for embryonic angiogenesis during mouse development. J Biol Chem 280: 9330–9335, 2005.

691. Yi R, Poy MN, Stoffel M, Fuchs E. A skin microRNA promotes differentiation by repressing “stemness.” Nature 452: 225–229, 2008.

692. Yi R, Qin Y, Macara IG, Cullen BR. Exportin-5 mediates the nuclear export of pre-microRNAs and short hairpin RNAs. Genes Dev 17: 3011–3016, 2003.

693. Yigzaw Y, Cartin L, Pierre S, Scholich K, Patel TB. The C terminus of sprouty is important for modulation of cellular migration and proliferation. J Biol Chem 276: 22742–22747, 2001.

694. Yin C, Salloum FN, Kukreja RC. A novel role of MicroRNA in late preconditioning upregulation of endothelial nitric oxide synthase and heat shock protein 70. Circ Res 12: 12, 2009.

695. Yu F, Yao H, Zhu P, Zhang X, Pan Q, Gong C, Huang Y, Hu X, Su F, Lieberman J, Song E. let-7 regulates self renewal and tumorigenicity of breast cancer cells. Cell 131: 1109–1123, 2007.

696. Yu J, Zhang Y, McIlroy J, Rordorf-Nikolic T, Orr GA, Backer JM. Regulation of the p85/p110 phosphatidylinositol 3′-kinase: stabilization and inhibition of the p110alpha catalytic subunit by the p85 regulatory subunit. Mol Cell Biol 18: 1379–1387, 1998.

697. Yu JY, Chung KH, Deo M, Thompson RC, Turner DL. MicroRNA miR-124 regulates neurite outgrowth during neuronal differentiation. Exp Cell Res 314: 2618–2633, 2008.

698. Yu Z, Wang C, Wang M, Li Z, Casimiro MC, Liu M, Wu K, Whittle J, Ju X, Hyslop T, McCue P, Pestell RG. A cyclin D1/microRNA 17/20 regulatory feedback loop in control of breast cancer cell proliferation. J Cell Biol 182: 509–517, 2008.

699. Yuan CX, Ito M, Fondell JD, Fu ZY, Roeder RG. The TRAP220 component of a thyroid hormone receptor-associated protein (TRAP) coactivator complex interacts directly with nuclear receptors in a ligand-dependent fashion. Proc Natl Acad Sci USA 95: 7939–7944, 1998.

700. Yuan TL, Choi HS, Matsui A, Benes C, Lifshits E, Luo J, Frangioni JV, Cantley LC. Class IA PI3K regulates vessel integrity during development and tumorigenesis. Proc Natl Acad Sci USA 105: 9739–9744, 2008.

701. Zakowicz H, Yang HS, Stark C, Wlodawer A, Laronde-Leblanc N, Colburn NH. Mutational analysis of the DEAD-box RNA helicase eIF4AII characterizes its interaction with transformation suppressor Pdcd4 and eIF4GI. RNA 11: 261–274, 2005.

702. Zanette DL, Rivadavia F, Molfetta GA, Barbuzano FG, Proto-Siqueira R, Silva-Jr WA, Falcao RP, Zago MA. miRNA expression profiles in chronic lymphocytic and acute lymphocytic leukemia. Braz J Med Biol Res 40: 1435–1440, 2007.

703. Zhang C, Chaturvedi D, Jaggar L, Magnuson D, Lee JM, Patel TB. Regulation of vascular smooth muscle cell proliferation and migration by human sprouty 2. Arterioscler Thromb Vasc Biol 25: 533–538, 2005.

704. Zhang H, Yang JH, Zheng YS, Zhang P, Chen X, Wu J, Xu L, Luo XQ, Ke ZY, Zhou H, Qu LH, Chen YQ. Genome-wide analysis of small RNA and novel MicroRNA discovery.

ery in human acute lymphoblastic leukemia based on extensive sequencing approach. PLoS One 4: 2009.

705. Zhang L, Huang J, Yang N, Greshock J, Megraw MS, Giannakakis A, Liang S, Naylor TL, Barchetti A, Ward MR, Yao G, Medina A, O'Brien-Jenkins A, Katsaros D, Hatzigeorgiou A, Gimotty PA, Weber BL, Coukos G. microRNAs exhibit high frequency genomic alterations in human cancer. Proc Natl Acad Sci USA 103: 9136–9141, 2006.

706. Zhang L, Volinia S, Bonome T, Calin GA, Greshock J, Yang N, Liu CG, Giannakakis A, Alexiou P, Hasegawa K, Johnstone CN, Megraw MS, Adams S, Lassus H, Huang J, Kaur S, Liang S, Sethupathy P, Leminen A, Simossis VA, Sandaltzopoulos R, Naomoto Y, Katsaros D, Gimotty PA, DeMichele A, Huang Q, Butzow R, Rustgi AK, Weber BL, Birrer MJ, Hatzigeorgiou AG, Croce CM, Coukos G. Genomic and epigenetic alterations deregulate microRNA expression in human epithelial ovarian cancer. Proc Natl Acad Sci USA 105: 7004–7009, 2008.

707. Zhang T, Nie K, Tam W. BIC is processed efficiently to microRNA-155 in Burkitt lymphoma cells. Leukemia 22: 1795–1797, 2008.

708. Zhang X, Azhar G, Chai J, Sheridan P, Nagano K, Brown T, Yang J, Khrapko K, Borras AM, Lawitts J, Misra RP, Wei JY. Cardiomyopathy in transgenic mice with cardiac-specific overexpression of serum response factor. Am J Physiol Heart Circ Physiol 280: H1782–H1792, 2001.

709. Zhang X, Liu S, Hu T, He Y, Sun S. Up-regulated microRNA-143 transcribed by nuclear factor kappa B enhances hepatocarcinoma metastasis by repressing fibronectin expression. Hepatology 50: 490–499, 2009.

710. Zhang Y, Chao T, Li R, Liu W, Chen Y, Yan X, Gong Y, Yin B, Qiang B, Zhao J, Yuan J, Peng X. MicroRNA-128 inhibits glioma cells proliferation by targeting transcription factor E2F3a. J Mol Med 87: 43–51, 2009.

711. Zhao C, Sun G, Li S, Shi Y. A feedback regulatory loop involving microRNA-9 and nuclear receptor TLX in neural stem cell fate determination. Nat Struct Mol Biol 16: 365–371, 2009.

712. Zhao Y, Ransom JF, Li A, Vedantham V, von Drehle M, Muth AN, Tsuchihashi T, McManus MT, Schwartz RJ, Srivastava D. Dysregulation of cardiogenesis, cardiac conduction, and cell cycle in mice lacking miRNA-1–2. Cell 129: 303–317, 2007.

713. Zhao Y, Samal E, Srivastava D. Serum response factor regulates a muscle-specific microRNA that targets Hand2 during cardiogenesis. Nature 436: 214–220, 2005.

714. Zhou B, Wang S, Mayr C, Bartel DP, Lodish HF. miR-150, a microRNA expressed in mature B and T cells, blocks early B cell development when expressed prematurely. Proc Natl Acad Sci USA 104: 7080–7085, 2007.

715. Zhou X, Ruan J, Wang G, Zhang W. Characterization and identification of MicroRNA core promoters in four model species. PLoS Comput Biol 3: 0412–0423, 2007.

716. Zhu S, Si ML, Wu H, Mo YY. MicroRNA-21 targets the tumor suppressor gene tropomyosin I (TPMI). J Biol Chem 282: 14328–14336, 2007.

717. Zhu S, Wu H, Wu F, Nie D, Sheng S, Mo YY. MicroRNA-21 targets tumor suppressor genes in invasion and metastasis. Cell Res 18: 350–359, 2008.

718. Ziegelbauer JM, Sullivan CS, Ganem D. Tandem array-based expression screens identify host mRNA targets of virus-encoded microRNAs. Nat Genet 41: 130–134, 2009.

719. Zuccato C, Tartari M, Crotti A, Goffredo D, Valenza M, Conti L, Cataudella T, Leavitt BR, Hayden MR, Timmusk T, Rigamonti D, Cattaneo E. Huntingtin interacts with REST/NRSF to modulate the transcription of NRSE-controlled neuronal genes. Nat Genet 35: 76–83, 2003.

MicroRNAs in Development and Disease
Danish Sayed and Maha Abdellatif
*Physiol Rev* 91:827-887, 2011. doi:10.1152/physrev.00006.2010

---

**You might find this additional info useful...**

This article cites 700 articles, 330 of which can be accessed free at:
/content/91/3/827.full.html#ref-list-1

This article has been cited by 50 other HighWire hosted articles, the first 5 are:

- AntagomiR directed against miR-20a restores functional BMPR2 signalling and prevents vascular remodelling in hypoxia-induced pulmonary hypertension  
  Matthias Brock, Victor J. Samillan, Michelle Trenkmann, Colin Schwarzwald, Silvia Ulrich, Renate E. Gay, Max Gassmann, Louise Ostergaard, Steffen Gay, Rudolf Speich and Lars C. Huber  
  *Eur Heart J*, December 1, 2014; 35 (45): 3203-3211.  
  [Abstract] [Full Text] [PDF]

- MicroRNA-200b targets protein kinase Cα and suppresses triple-negative breast cancer metastasis  
  Brock Humphries, Zhishan Wang, Aaron L. Oom, Theresa Fisher, Dongfeng Tan, Yuehua Cui, Yiguo Jiang and Chengfeng Yang  
  *Carcinogenesis*, October , 2014; 35 (10): 2254-2263.  
  [Abstract] [Full Text] [PDF]

- MicroRNA-130a Is Up-regulated in Mouse Liver by Iron Deficiency and Targets the Bone Morphogenetic Protein (BMP) Receptor ALK2 to Attenuate BMP Signaling and Hepcidin Transcription  
  Kimberly B. Zumbrennen-Bullough, Qifang Wu, Amanda B. Core, Susanna Canali, Wenjie Chen, Igor Theurl, Delphine Meynard and Jodie L. Babitt  
  *J. Biol. Chem.*, August 22, 2014; 289 (34): 23796-23808.  
  [Abstract] [Full Text] [PDF]

- MicroRNA-200b targets protein kinase Cα and suppresses triple-negative breast cancer metastasis  
  Brock Humphries, Zhishan Wang, Aaron L. Oom, Theresa Fisher, Dongfeng Tan, Yuehua Cui, Yiguo Jiang and Chengfeng Yang  
  *Carcinogenesis*, June 12, 2014;.  
  [Abstract] [Full Text] [PDF]

- MicroRNAs: History, Biogenesis, and Their Evolving Role in Animal Development and Disease  
  M. Bhaskaran and M. Mohan  
  *Vet Pathol.*, July , 2014; 51 (4): 759-774.  
  [Abstract] [Full Text] [PDF]

Updated information and services including high resolution figures, can be found at:
/content/91/3/827.full.html

Additional material and information about *Physiological Reviews* can be found at:
http://www.the-aps.org/publications/prv

---

*Physiological Reviews* provides state of the art coverage of timely issues in the physiological and biomedical sciences. It is published quarterly in January, April, July, and October by the American Physiological Society, 9650 Rockville Pike, Bethesda MD 20814-3991. Copyright © 2011 by the American Physiological Society. ISSN: 0031-9333, ESSN: 1522-1210. Visit our website at http://www.the-aps.org/.

This information is current as of January 21, 2015.

---

*Physiological Reviews* provides state of the art coverage of timely issues in the physiological and biomedical sciences. It is published quarterly in January, April, July, and October by the American Physiological Society, 9650 Rockville Pike, Bethesda MD 20814-3991. Copyright © 2011 by the American Physiological Society. ISSN: 0031-9333, ESSN: 1522-1210. Visit our website at http://www.the-aps.org/.
