
DOI: 10.1002/cbic.201100366

# Definition of the Common and Divergent Steps in Carbapenem β-Lactam Antibiotic Biosynthesis

Micah J. Bodner, Rongfeng Li, Ryan M. Phelan, Michael F. Freeman, Kristos A. Moshos, Evan P. Lloyd, and Craig A. Townsend*[a]

---

Approximately 50 naturally occurring carbapenem β-lactam antibiotics are known. All but one of these have been isolated from Streptomyces species and are disubstituted structural variants of a simple core that is synthesized by *Pectobacterium carotovorum* (*Erwinia carotovora*), a phylogenetically distant plant pathogen. While the biosynthesis of the simple carbapenem, (5R)-carbapen-2-em-3-carboxylic acid, is impressively efficient requiring only three enzymes, CarA, CarB and CarC, the formation of thienamycin, one of the former group of metabolites from Streptomyces, is markedly more complex. Despite their phylogenetic separation, bioinformatic analysis of the enzyme coding gene clusters suggests that the two pathways could be related. Here we demonstrate with gene swapping, stereochemical and kinetics experiments that CarB and CarA and their *S. cattleya* orthologues, ThnE and ThnM, respectively, are functionally and stereochemically equivalent, although their catalytic efficiencies differ. The biosynthetic pathways, therefore, to thienamycin, and likely to the other disubstituted carbapenems, and to the simplest carbapenem, (5R)-carbapen-2-em-3-carboxylic acid, are initiated in the same manner, but share only two common steps before diverging.

## Introduction

While semisynthetic penicillins and cephalosporins remain frequently prescribed drugs in human medicine, the carbapenem β-lactam antibiotics have assumed growing importance for their broad-spectrum activity, potency and effectiveness against resistant infections.[1] The family of naturally-occurring carbapenems consists of the simplest core metabolite, carbapen-2-em-3-carboxylic acid (1) on the one hand, and approximately 50 more highly elaborated structures that differ by the nature of the sulfur substituent at C2 and an alkyl group at C6, often present at an elevated oxidation state, for example, 2 (Scheme 1 A).[2] Prominent among the latter is thienamycin (3) the amide derivative of which, imipenem, is marketed as Primaxin.[3] The two carbons of the hydroxyethyl side chain of 3 are known to be derived from methionine by presumably successive C1-transfers.[4–6] Recent work has established that the C2 side chain is derived by stepwise truncation of coenzyme A (CoA) rather than by direct incorporation of cysteamine or cysteine, as previously thought.[7] The timing, however, of C2 and C6 side-chain attachment among the overall biosynthetic events leading to thienamycin is unknown.

The unsubstituted carbapenem 1 is created in three highly efficient steps (Scheme 1 B). Carboxymethylproline synthase (CarB), a member of the crotonase superfamily, mediates the decarboxylation of malonyl-CoA (MalCoA) and the stereospecific addition of the resulting enzyme-bound enolate to L-glutamate γ-semialdehyde (4) or L-pyrrolidine-5-carboxylic acid (L-P5C, 5). The resulting (2S,5S)-carboxymethylproline coenzyme A thioester (CMP-CoA, 6) partitions between hydrolysis and diffusion from the active site to give the acid 9. CarB is also capable of catalyzing the same series of reactions with methylmalonyl-CoA (MeMalCoA), albeit less efficiently, to give the 6-methyl (2S,5S)-CMP disasteriomers 10 and 11 in a 1.2:1 ratio.[8] Adenylation of 9 by carbapenam synthetase (CarA) is coupled to β-lactam formation by the coordinated participation of an active site Tyr-Glu dyad and a lysine residue to deftly catalyze formation of this more highly strained bicyclic intermediate.[9] In contrast to the strict stereospecificity of CarB, CarA will accept and process epimers of (2S,5S)-CMP (9).[8a,9b] Finally, carbapenem synthase (CarC), a member of the nonheme iron α-ketoglutarate (α-KG)-dependent oxygenase superfamily, carries out the bridgehead epimerization of 12 to 13 and ring desaturation to give 1.[10]

Identification of the thienamycin biosynthetic gene cluster in *Streptomyces cattleya* opened the way to investigation of the more structurally complex members of the carbapenem antibiotic family.[6] It was immediately apparent that many more proteins were involved in the biosynthesis of 3 than of 1 and, while orthologues of CarA and CarB were encoded by the cluster, ThnE and ThnM bore only 37 and 25% amino acid sequence identity, respectively. Second, there was no orthologue of CarC, suggesting that different biochemical solutions had evolved to accomplish C2/3 desaturation and, by analogy to the formation of 1, to carry out bridgehead inversion—both essential to antibiotic activity. Support for at least one step in a common pathway to both 3 and the simple carbapenem 1

[a] Dr. M. J. Bodner, Dr. R. Li, R. M. Phelan, Dr. M. F. Freeman, Dr. K. A. Moshos, E. P. Lloyd, Prof. Dr. C. A. Townsend  
Department of Chemistry, The Johns Hopkins University  
Baltimore, Maryland 21218 (USA)  
E-mail: ctownsend@jhu.edu  

Supporting information for this article is available on the WWW under http://dx.doi.org/10.1002/cbic.201100366.

CHEMBIOCHEM

was recently reported when a truncated ThnE (Δ2–46) indeed catalyzed the formation of (2S,5S)-CMP (9, Scheme 1B) from MalCoA, like CarB. A 4:1 diastereomeric mixture of 6-methyl-(2S,5S)-CMPs 10 and 11 was also observed from MeMalCoA.[11] We have gathered compelling gene swapping, stereochemical and kinetic evidence that 3 and 1 share two biosynthetic steps, despite substantial phylogenetic separation of the producing organisms, before diverging, and that introduction of the C6 side chain in thienamycin biosynthesis occurs after carbapenam 12 formation.

Results and Discussion

In vivo analysis of ThnE and ThnM

Despite the low overall identity between ThnE and CarB, or ThnM and CarA, many residues in their respective active sites appeared to be conserved, indicating that ThnE and ThnM could potentially carry out corresponding reactions in the simple carbapenem pathway. Their biosynthetic activities were first probed globally, therefore, by analyzing their abilities to substitute for their counterparts CarB and CarA and support the biosynthesis of (5R)-carbapen-2-em-3-carboxylic acid (1), in vivo (Figure 1). Genetic replacements by *thnE* and *thnM* were made individually and pairwise in the previously described plasmid, pET24a(+)/carABC.[12] In this construct, expression of *carA*, *carB* and *carC* were controlled by a T7 promoter and T7 terminator as a single operon. It had been shown that the coexpression of *carB*, *carA* and *carC* in *E. coli* BL21(DE3)(pLysS) reconstituted the synthesis of carbapenem 1 at a titer comparable to that of wild-type *P. carotovorum*, which could be readily detected by two sensitive bioassays by using a β-lactam super-sensitive *E. coli*

and a β-lactamase induction assay.[9b,13] To obtain higher expression and improved solubility of ThnE, the codons of the 21

Scheme 1. A) Representative carbapenems. B) The biosynthesis of carbapen-2-em-3-carboxylic acid (1). C) Carboxymethylprolines relevant to thienamycin (3) biosynthesis.

Figure 1. CarA and CarB orthologue replacement experiments. A) Inhibition of the growth of β-lactam super-sensitive *E. coli* SC12155. B) Coexpression of biosynthetic enzymes; U: uninduced cellular fraction, I: induced cellular fraction.

Common Steps in Carbapenem Biosynthesis

N-terminal amino acids were optimized and the engineered ThnE (ThnE*) showed elevated expression profiles (data not shown). As illustrated in Figure 1, when substitutions were made in this construct by either or both *thnE* or *thnM*, approximately equivalent levels of antibiotic production were achieved in the optimized medium, suggesting overlap of their catalytic activities with both CarB and CarA, respectively.

The ability, however, of CarB and truncated ThnE (∆2–46) to carry out reactions with both MalCoA and MeMalCoA led us to address more rigorously the reactions catalyzed by ThnE. Similarly, the modest sequence identity between CarA and ThnM raised questions about the substrate preferences of both ThnE and ThnM, particularly with respect to C2 and C6 side-chain attachments and their timing in the biosynthetic pathway. The *thnE* gene was cloned from *S. cattleya* genomic DNA and the first 21 codons were optimized to favor expression in *E. coli*. The partially codon-optimized *thnE** was inserted into pET28b encoding an N-terminal His₆-tag. The *thnM* gene was similarly cloned and inserted into pET29b bearing a C-terminal His₆-tag. The recombinant proteins were over-produced in *E. coli* Rosetta2(DE3) and purified by Ni-NTA affinity chromatography.

### Preparation of substrates

The substrates of CarB and CarA, L-P5C (5) and (2S,5S)-CMP (9), as well as (2S,5R)-CMP (14, Scheme 1C) were synthesized by using established methods. In addition, (6R)-methyl and (6R)-ethyl CMPs (10 and 15) were synthesized by extension of a recently described route to carbapenems (Scheme 2). 3-Methyl and 3-ethyl (3R,4R)-azetidinones 16 and 17 were prepared in catalytic, asymmetric reactions to set the absolute configuration at C4 that is produced by ThnE and the configuration at C3 that is observed in thienamycin (3) and all known carbapenem metabolites of *S. cattleya*. The *cis* relation of the C3/C4 substituents was preserved by Arndt-Eistert homologation of 24 and 25 in the synthesis of C2/3 unsubstituted carbapenems. The α,β-unsaturated esters 30 and 31 thus obtained underwent reduction by selective conjugate hydride addition mediated by Stryker's reagent to give the saturated carbapenams 32 and 33. The newly created carbapenam C3 stereocenter was directed exclusively exo to the β-lactam ring, matching the orientation present in 9. The C3 acids could be deblocked by hydrogenolysis prior to hydrolysis of the β-lactam ring. Alternatively, the β-lactam ring and ester could be cleaved concurrently to give the C6-substituted carboxymethylprolines 10 and 15. Epimerization of the thermodynamically less stable *cis*-C5/C6 configuration to the more stable trans-C5/C6 configuration was not detected by ¹H NMR spectroscopy.

The known (5S)-carbapenem 35 was treated with *N*-acetyl-cysteamine to give a diastereotopic mixture of 36 and 37 in a 4:1 ratio, which was separated by chromatography on silica gel and deprotected by hydrogenolysis. The carboxylic acids 38 and 39 were purified by HPLC. The low concentration of acid present during this step catalyzed β-lactam hydrolysis. The individual stereoisomers upon lyophilization yielded the

---

### Scheme 2. Synthesis of carboxymethylprolines.
a) SmI₂, *i*PrOH, Sm(0), THF, 80%; b) CH₂Cl₂, TBSOTf, Et₃N, 92 %; c) THF, H₂, Pd/C, 99 %; d) 1) CH₂Cl₂, cat. DMF, oxalyl chloride, 2) Et₂O, diazomethane, 85 %; e) THF, 10 % H₂O, hv, 96 %; f) 1) CH₃CN, carbonyl diimidazole, 2) Mg(mono-PNB malonate)₂, 3) MeOH, 10 % 1 M HCl, 4) CH₃CN, mesyl azide, Et₃N, 50 %; g) 1) benzene, Rh₂(OAc)₄, 80 °C, 2) THF/MeOH, NaBH₄ —78 °C, 3) CH₂Cl₂, mesyl chloride, Et₃N, 65 %; h) benzene, MeOH, [(PPh₃)CuH]₆, polymethylhydrosiloxane, 50 %; i) 2:1 THF/H₂O, 2 equiv KOH, 99 %; j) 2:1 THF/H₂O, 1 equiv KHCO₃, H₂, Pd/C, 90 %; k) H₂O, 1 equiv KOH, 99 %.

corresponding cleaved (2S,3R,5S)- and (2R,3R,5S)-2-cysteaminyl-5-carboxy-ethylprolines 40 and 41 (Scheme 3 and the Supporting Information).

---

### Alternate substrate studies and kinetics

A more detailed analysis of ThnE function began with a comparison of its substrate specificity for MalCoA, the native substrate of CarB, or MeMalCoA, a potential precursor of the thi-

CHEMBIOCHEM

C.A. Townsend et al.

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

**Scheme 3. Synthesis of 2-N-acetylcysteaminyl-CMP diastereomers.**  
a) N-acetylcysteamine, DBU, CH₃CN; b) H₂, 10% Pd/C, KHCO₃, 2:1 THF/H₂O; c) HPLC separation/lyophilization.

enamycin (3) hydroxyethyl group. The derivation of (R)- or (S)-MeMalCoA or ethylmalonyl-CoA by C₁-transfers to MalCoA by S-adenosylmethionine is unprecedented, but is chemically feasible.[18] However remote, this possibility was examined by incubation of L-P5C (5) with MalCoA or MeMalCoA and monitoring the course of reaction by HPLC. The ThnE-catalyzed reaction between MalCoA and 5 was confirmed by Dowex purification and characterization of (2S,5S)-CMP (9) from the enzymatic reaction. ThnE also condensed MeMalCoA with 5 to give a 3:2 diastereomeric mixture of 6-methyl CMPs 10 and 11, respectively, an unselective product ratio similar to that seen previously with CarB.[8a] No reaction was observed with ethylmalonyl-CoA as previously with a truncated ThnE.[11]

With MeMalCoA as a substrate, ThnE also accumulated two other products observable by HPLC. HPLC purification and ESI mass spectrometric analysis (*m/z* 935.14 \[M−H⁺\]) identified them as CMP-CoA esters 7 and 8. The appearance of these CMP-CoA ester intermediates was also observed in the CarB-catalyzed reaction and could indicate a high degree of catalytic similarity between the enzymes. To further probe the extent of functional similarity, the kinetic constants for the ThnE-catalyzed reaction between MalCoA or MeMalCoA and 5 were determined (Table 1). ThnE is selective for MalCoA with a 66-fold higher specificity constant (*k*~cat~/*K*~M~) for the reaction with 5 and MalCoA than with MeMalCoA.

| **Substrate** | **K<sub>M</sub> [mM]** | **k<sub>cat</sub> [s⁻¹]** | **k<sub>cat</sub>/K<sub>M</sub> [mM⁻¹s⁻¹]** |
|---------------|-----------------------|--------------------------|-----------------------------------------|
| MalCoA        | 0.017 ± 0.002         | 0.30 ± 0.01              | 17.4 ± 2.1                              |
| MeMalCoA      | 0.203 ± 0.008         | 0.05 ± 0.001             | 0.26 ± 0.01                             |

9, 6-methyl (2S,5S,6R)-CMP (10) and 6-ethyl (2S,5S,6R)-CMP (15), the latter two bearing alkyl substituents matching the thienamycin (3) stereochemistry, were examined. As a further probe of pathway timing and C5 stereoisomerization, ThnM-catalyzed β-lactam formation with the epimeric (2S,5R)-CMP (14) as substrate was also investigated. Of 9, 10, 15, and 14, ThnM showed reaction only with 9 and 10. In an analogous manner the 2-cystaminyl CMPs, 40 and 41 were assayed and also found to be inactive with ThnM. It is apparent that, while a methyl substituent is tolerated, more sterically demanding substitutions at C2 or C6 are not.

To investigate the reactions of ThnM with 9 and 10 more precisely, their kinetic constants were determined under optimized conditions from a pH/rate profile by using a continuous coupled-enzyme assay that linked production of the terminal byproduct AMP to a decrease in the concentration of NADH detectable at 340 nm (Table 2 and the Supporting Information).[19] Three notable observations emerged from this analysis. First, based on the selectivity constants (*k*~cat~/*K*~M~), the preferred substrate for ThnM, like CarA, is 9. Second, while the *K*~M~ of 9 is nearly identical for ThnM and CarA, the *k*~cat~ for the former is an order of magnitude lower, signaling a much less efficient enzyme.[9b] This lower *k*~cat~ might result from selective pressure on the host to survive a more potent carbapenem product, or from downstream enzyme(s) that are overall rate-limiting to the flux of the biosynthesis, hence reducing evolutionary pressure on ThnM. Third, the *k*~cat~ for 10 is approximately 50% greater than the unsubstituted substrate. We propose that it is the eclipsed relationship of the (6R)-methyl and the five-membered ring of the adenylated CMP imposed by the active site that engenders bond angle compression in the classic Thorpe–Ingold sense. This steric compression raises the ground state energy leading to β-lactam closure, hence accelerating the rate of this chemical step.[20] It is known from detailed kinetic analysis of CarA that both formation of the four-membered ring and a protein conformational change limit the overall rate of this enzyme.[9d] By analogy to ThnM, the intrinsic magnitude of the Thorpe–Ingold effect favoring β-lactam synthesis could be partially masked by a corresponding protein conformational change.

### Conclusions

The combined results for ThnE and ThnM, particularly the kinetic discrimination of each enzyme for variously methyl- and

2162 www.chembiochem.org © 2011 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim ChemBioChem 2011, 12, 2159–2165

Common Steps in Carbapenem Biosynthesis

ethyl-substituted potential substrates, reveal unambiguous preferences for the natural substrates of CarB and CarA involved in the synthesis of the structurally simpler (5R)-carbap- en-2-em-3-carboxylic acid (1). The low sequence identity evident for each pair of homologous enzymes raised the question whether C₁-alkylations or thioether side-chain introduction could take place at this early stage in the biosynthesis of thi- enamycin (3). One can envision that the active site(s) of CarB and ThnE or CarA and ThnM could have easily been altered by mutation to accommodate such charge-neutral steric changes in their substrates. The experiments strongly suggest, however, that this evolutionary alternative was not chosen despite the large phylogenetic separation of the respective hosts, and that thienamycin and simple carbapenem biosynthesis are functionally and stereochemically identical up to the formation of (3S,5S)-carbapenam (12). At this point the pathways to the simple 1 and 3 diverge (Scheme 4). In *P. carotovorum* the Fe²⁺/ α-ketoglutarate-dependent oxygenase CarC both inverts the bicyclic ring junction and desaturates C2/3 to give 1 (Scheme 1B). By contrast a more complex series of events takes place to accomplish these steps and C6 side- chain modification in 3 formation.[2] Recently described insertional inactivation of *thnL* and *thnP*, two apparent radical SAM methyltransferases in the thienamycin biosynthetic gene cluster, gives rise to a detectable product the nominal mass of which is at least consistent with the accumulation of 12.[21] While this experiment does not strictly prove the identity or intermediacy of 12 in thienamycin biosynthesis, it is potentially supportive of the conclusions drawn here. C6 alkylation and C2 sulfur side-chain attachment each temper the reactivity of the simple carbapenem-3-carboxylic acid in ways desirable for antibiotic activity, a property that could account for the additional metabolic burden they entail. The timing and mechanism of these events, and of bicyclic ring inversion and C2/3 desaturation, remain to be determined.

### Experimental Section

#### Construction of co-overexpression vector pET24a(+)/carABC:
RBS-carB generated as a Xbal-HindIII fragment from pET24a(+) / carB was blunted with T4 DNA polymerase and ligated into the blunt-ended NotI site located downstream of carA in pET24a(+) / carA. The resulting plasmid with the RBS-carB at the same orientation as the carA was named pET24a(+)/carA-carB. The RBS-carC was excised from pET24a(+)/carC as a Xbal-Xhol fragment. It was blunt-ended and inserted into the blunt-ended Xhol site downstream of the carB in pET24a(+)/carA-carB to form the final vector pET24a(+)/carABC. The co-overexpression of carA, carB and carC in pET24a(+)/carABC was controlled by the T7 promoter and T7 terminator as a single operon.

Cloning and partial codon optimization of *thnE*: Wild-type *thnE* was amplified from genomic DNA (gDNA) of *S. cattleya* by using the forward primer: GCATATGGGC GCGGCCGCCC GCGAG, and reverse primer: CAAGCTTCAG CTCCGCCCCGA TGACGCG. Ndel and HindIII restriction sites were introduced to facilitate downstream cloning experiments. The PCR reaction (100 μL) contained 10× cloned Pfu DNA polymerase buffer (10 μL), dNTPs (2.5 μL, 10 mM), each primer (2 μL, 20 nmol/mL⁻¹), DMSO (5 μL), gDNA (85 ng/μL⁻¹, 1 μL), and Pfu DNA polymerase (2.5 U/μL⁻¹, 1 μL). The reaction was preheated to 98°C for 2 min before the addition of Pfu DNA polymerase, then followed by 30 cycles of 45 s at 98°C, 45 s at 69°C, and 1 min at 72°C with the final extension time of 10 min at 72°C. The PCR product was purified with the GeneClean III kit (Q-BioGene, Solon, OH, USA) and ligated into pBluescript II SK+ (Stratagene, La Jolla, CA, USA) to obtain pBS/*thnE*. Once confirmed by DNA sequencing analysis, *thnE* was excised from pBS/*thnE* with Ndel/HindIII and ligated into pET24a(+) to generate pET24a(+)/thnE.

WT  
ATGGCGCGCGCCGCGCGAGCGGAGAACCC  
thnE*  
ATGGCGCGCGCCGCGCAAACGTGCTACC  
M G A A A G E R R T  
CTCTCACGAAAAAGAGGCACCGTGGATGTCC  
L S R K R G T T V D V  
CTGTCTCGTAAACGTGGCACCACCGTGGATGTCC  

Ecod-F  
TATGGCGCGCGCCGCGGAACGTGCTACCCTGTC  
T CGTAAACGTGGCACCACCGTGGATGTCC  

Ecod-C  
ACCCGCGCCGCGGCGGCTTGCAAGATGGGACAGAG  
CATTTGCACCACGTGGTGGCACCTACA  

To optimize the codons of the first 21 amino acids of *thnE*, two oligonucleotides, Ecod-F and Ecod-R, harboring the optimized codons and Ndel and BsrI restriction ends were mixed in equal concentrations. After being heated at 100°C for 10 min, the mixture was left to cool slowly at room temperature to allow the oligonucleotides to anneal. Plasmid pET24a(+)/thnE was digested with HindIII/BsrI to generate a 821 bp fragment of *thnE* absent the 64 bp 5′ region. A three-way ligation reaction containing Ndel/HindIII digested pET24a(+), the 821 bp BsrI-HindIII *thnE* fragment and the 64 bp Ndel-BsrI annealed oligonucleotides was carried out to generate plasmid pET24a(+)/thnE.

---

**Scheme 4. Common biosynthetic steps to the carbapenem β-lactam antibiotics.**

![Chemical Scheme](https://i.imgur.com/yourimage.png)

---

ChemBioChem 2011, 12, 2159–2165 © 2011 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim www.chembiochem.org 2163

CHEMBIOCHEM

thnE*. Gene thnE* was excised with Ndel/HindIII from pET24a(+)/thnE* and inserted into pET28b(+) to give pET28b(+)/thnE* for expression of ThnE as N-His₆ protein.

Construction of co-overexpression vector pET24a(+)/thnE*M-carC: RBS-carC was excised from pET24a(+)/carC with Xbal/HindIII, blunt-ended with Klenow DNA polymerase, and ligated into the blunt ended NotI site of pET24a(+)/thnM to give plasmid pET24a(+)/thnM-carC. The RBS-thnM-RBS-carC fragment was generated with Xbal/Xhol, blunt-ended and ligated downstream of thnE* at the blunt-ended Xhol site to obtain the final expression vector pET24a(+)/thnE*M-carC.

Construction of co-overexpression vector pET24a(+)/thnE*-carAC and pET28b(+)/thnE*-carAC: The RBS-carC fragment was generated by digesting pET24a(+)/carC with Xbal/Xhol. It was blunt-ended with T4 DNA polymerase and inserted into the blunt-ended NotI site of pET24a(+)/carA to give the co-overexpression vector pET24a(+)/carAC. RBS-carA-RBS-carC was excised as a XbaI-XhoI fragment from pET24a(+)/carAC and blunt-ended with Klenow DNA polymerase. Plasmids pET24a(+)/thnE* and pET28b(+)/thnE* were digested with NotI, blunt-ended with Klenow DNA polymerase and ligated with the RBS-carA-RBS-carC fragment to generate co-overexpression vectors pET24a(+)/thnE*-carAC and pET28b(+)/thnE*-carAC, respectively.

Construction of co-overexpression vector pET24a(+)/thnM-carBC: pET24a(+)/thnM-carC was digested with XbaI/XhoI to give a fragment containing RBS-thnM-RBS-carC. It was then blunt-ended with Klenow DNA polymerase and ligated into the blunt-ended XhoI site of pET24a(+)/carB to give the final plasmid pET24a(+)/thnM-carBC.

Small-scale co-overexpression of biosynthetic genes and heterologous production of carbapenem in E. coli: Seed medium (3 mL) containing kanamycin (50 mg mL⁻¹) and chloramphenicol (25 mg mL⁻¹) was inoculated with a single colony of freshly transformed Rosetta2(De3) or BL21(DE3)(pLysS) harboring recombinant plasmid and grown, overnight, at 37 °C. Seed culture (100 mL) was transferred into LB⁺ medium (50 mL; g L⁻¹: Bacto-tryptone 10; Bacto-yeast extract 5; NaCl 10; glutamate 10; NaOAc 1; FeSO₄·7H₂O 0.25; CoCl₂·6H₂O 0.01, pH 7.5) or modified carbapenem production medium (g L⁻¹: glutamate 5; NH₄Cl 0.75; K₂HPO₄ 2; NaCl 0.5; CaCO₃ 0.25; glucose 10, pH 7.6; 75 mg FeSO₄·7H₂O and 12.5 mL of 1 M MgSO₄ were added after autoclaving). The secondary culture was grown at 37 °C to OD₆₀₀ = 0.3–0.4. Expression of proteins was induced with IPTG (1 mM) at 28 °C for 5 h.

Detection of carbapenems: Samples were withdrawn from cultures every 60 min after co-overexpression was induced, and centrifuged at 16,000 g for 10 min. Supernatant (250 mL) was added to paper discs placed on a bioassay plate of Bacto-Nutrient agar seeded with β-lactam supersensitive E. coli SC12155.[13b] The bioassay plates were incubated at 37 °C for 20 h. The production of carbapenems in the engineered E. coli was indicated by the inhibition of super-sensitive E. coli growth in zones around the paper discs. The production of carbapenem was also observed by Nitrocefin colorimetric assay.[13a] The recombinant E. coli supernatant (300 mL) was loaded on to paper discs sitting on BA₂ agar (gL⁻¹: BBL seed agar 30.5; NaCl 5) seeded with Bacillus licheniformis ATCC14580. The plates were incubated at 37 °C for 3 h and overlaid with Nitrocefin solution (1.5 mL of 300 mg mL⁻¹). The production of carbapenems was indicated by the formation of red zones around the paper discs.

HPLC analysis of ThnE-catalyzed reactions: A previously reported method was modified.[8a] HPLC conditions: Phenomenex Prodigy 5 μ ODS-3 analytical column, λ = 260 nm, 1 mL min⁻¹; buffer A = 100 mm NH₄H₂PO₄, 75 mm ammonium acetate, pH 4.65 with acetic acid, buffer B = 70% solvent A and 30% methanol. Method 1, t = 0 min 40% B, t = 28 min 85% B, t = 29 min 100% B, t = 39 min 100% B, t = 42 min 40% B, t = 52 min 40% B. Method 2, t = 0 min 40% B, t = 20 min 80% B, t = 21 min 40% B, t = 30 min 40% B.

Relative rates of ThnE-catalyzed reactions with L-P5C (5) and MalCoA or MeMalCoA: Reactions containing L-P5C (5, 0.5 mm), 0.3 mm MalCoA or MeMalCoA and ThnE (2.4 mg mL⁻¹) as well as the corresponding reactions without 5 and no-enzyme reactions were assembled in potassium phosphate (100 mm, pH 7.8).[8a] At 0, 15, 30, 60, 120 min and overnight, samples (200 μL) were quenched in HCl (200 μL, 0.2 M) then vortexed with CCl₄ (400 μL), the top layer (300 μL) was removed from the CCl₄ for HPLC analysis.

Purification and ESI-MS of CMP-CoA esters from ThnE reactions: New products observable in the HPLC trace were collected from multiple runs, lyophilized and desalted with a STRATA X column (33 μm, 30 mg mL⁻¹).[8a] The column was prewashed with methanol (10 mL) and 0.1% TFA (10 mL). The lyophilized sample was taken up in water (1 mL) and 0.1% TFA (2 mL) was added. The sample was applied to the column and washed with 0.1% TFA (2 mL). It was then eluted with methanol/0.1% TFA 4:1 (5 mL). Fractions (1 mL) containing the desired CoA ester were identified by their characteristic absorbance at 260 nm and lyophilized. The resulting powder was taken up in 1:1 CH₃CN/water with 0.1% NH₄OH or 0.1% TFA and analyzed by ESI-MS.

HPLC analysis of ThnM-catalyzed reactions: Reactions for the determination of product formation (200 μL) were run in a buffer containing HEPES (100 mm) and piperazine (80 mm) at pH 8.2 with μ = 100 mm (KCl).[9b] Each reaction contained ATP (2 mm), DTT (1 mm), MgCl₂ (12.5 mm), substrate (10 mm) and ThnM (15.0 μm). The reactions were run at 25 °C for 2 h and immediately frozen in liquid N₂ until analysis. Analysis was performed on an Agilent 1100 HPLC by using a Phenomenex Luna 5 μ phenyl-hexyl analytical column. Injections (50 μL) were run in an isocratic mobile phase consisting of a phosphate buffer (50 mm) at pH 6.5 with a flow rate of 2 mL min⁻¹. Reactions were monitored at λ = 210 and 230 nm. Compounds produced in ThnM-catalyzed reactions were identified by comparison to authentic product standards.

Acknowledgements

We are grateful to Dr. Anthony Stapon for preparation of CMPs, and to Prof. Barbara Gerratana and Dr. Samantha O. Arnett for guidance and early insightful discussions. We thank Dr. Mary L. Raber for critical review of the kinetic data and the NIH (AI014937) for financial support.

Keywords: beta-lactam antibiotics · biosynthesis · carbapenems · enzyme catalysis · thienamycin

[1] a) A. Dalhoff, N. Janjic, R. Echols, Biochem. Pharmacol. 2006, 71, 1085–1095; b) S. T. Kumagai, S. Tamai, T. Abe, M. Hikda, Curr. Med. Chem. Anti-Infect. Agents 2002, 1, 1–14.
[2] M. J. Bodner, R. M. Phelan, M. F. Freeman, R. Li, C. A. Townsend, J. Am. Chem. Soc. 2010, 132, 12–13.

2164 www.chembiochem.org © 2011 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim ChemBioChem 2011, 12, 2159–2165

[3] J. Birnbaum, F. M. Kahan, H. Kropp, J. S. MacDonald, *Am. J. Med.* 1985, 78, 3–21.

[4] J. M. Williamson, E. Inamine, K. E. Wilson, A. W. Douglas, J. M. Liesch, G. Albers-Schönberg, *J. Biol. Chem.* 1985, 260, 4637–4647.

[5] a) D. R. Houck, K. Kobayashi, J. M. Williamson, H. G. Floss, *J. Am. Chem. Soc.* 1986, 108, 5365–5366; b) T. M. Zydowsky, L. F. Courtney, V. Frasca, K. Kobayashi, H. Shimizu, L. D. Yuen, R. G. Matthews, S. J. Benkovic, H. G. Floss, *J. Am. Chem. Soc.* 1986, 108, 3152–3153; c) D. Arigoni, *Ciba Found. Symp.* 1978, 243–261; d) H. G. Floss, M. D. Tsai, *Adv. Enzymol. Relat. Areas Mol. Biol.* 1979, 50, 243–302.

[6] L. E. Nunez, C. Mendez, A. F. Brana, G. Blanco, J. A. Salas, *Chem. Biol.* 2003, 10, 301–311.

[7] M. F. Freeman, K. A. Moshos, M. J. Bodner, R. F. Li, C. A. Townsend, *Proc. Natl. Acad. Sci. USA* 2008, 105, 11128–11133.

[8] a) B. Gerratana, S. O. Arnett, A. Stapon, C. A. Townsend, *Biochemistry* 2004, 43, 15936–15945; b) E. T. Batchelar, R. B. Hamed, C. Ducho, T. D. W. Claridge, M. J. Edelmann, B. Kessler, C. J. Schofield, *Angew. Chem.* 2008, 120, 9462–9465; *Angew. Chem. Int. Ed.* 2008, 47, 9322–9325; c) R. B. Hamed, E. T. Batchelar, I. J. Clifton, C. J. Schofield, *Cell. Mol. Life Sci.* 2008, 65, 2507–2527; d) M. C. Sleeman, J. L. Sorensen, E. T. Batchelar, M. A. McDonough, C. J. Schofield, *J. Biol. Chem.* 2005, 280, 34956–34965.

[9] a) M. T. Miller, B. Gerratana, A. Stapon, C. A. Townsend, A. C. Rosenzweig, *J. Biol. Chem.* 2003, 278, 40996–41002; b) B. Gerratana, A. Stapon, C. A. Townsend, *Biochemistry* 2003, 42, 7836–7847; c) M. L. Raber, A. Castillo, A. Greer, C. A. Townsend, *ChemBioChem* 2009, 10, 2904–2912; d) M. L. Raber, S. O. Arnett, C. A. Townsend, *Biochemistry* 2009, 48, 4959–4971; e) M. L. Raber, M. F. Freeman, C. A. Townsend, *J. Biol. Chem.* 2009, 284, 207–217; f) S. O. Arnett, B. Gerratana, C. A. Townsend, *Biochemistry* 2007, 46, 9337–9345.

[10] a) A. Stapon, R. F. Li, C. A. Townsend, *J. Am. Chem. Soc.* 2003, 125, 8486–8493; b) A. Stapon, R. F. Li, C. A. Townsend, *J. Am. Chem. Soc.* 2003, 125, 15746–15747; c) M. C. Sleeman, P. Smith, B. Kellam, S. R. Chhabra, B. W. Bycroft, C. J. Schofield, *ChemBioChem* 2004, 5, 879–882; d) I. J. Clifton, L. X. Doan, M. C. Sleeman, M. Topf, H. Suzuki, R. C. Wilmouth, C. J. Schofield, *J. Biol. Chem.* 2003, 278, 20843–20850.

[11] R. B. Hamed, E. T. Batchelar, J. Mecinovic, T. D. W. Claridge, C. J. Schofield, *ChemBioChem* 2009, 10, 246–250.

[12] R. F. Li, A. Stapon, J. T. Blanchfield, C. A. Townsend, *J. Am. Chem. Soc.* 2000, 122, 9296–9297.

[13] a) R. B. Sykes, J. S. Wells, *J. Antibiot.* 1985, 38, 119–121; b) H. Aoki, H. Sakai, M. Kohsaka, T. Konomi, J. Hosoda, Y. Kubochi, E. Iguchi, H. Imanaka, *J. Antibiot.* 1976, 29, 492–500.

[14] M. J. Bodner, R. M. Phelan, C. A. Townsend, *Org. Lett.* 2009, 11, 3606–3609.

[15] J. Fetter, K. Lempert, T. Gizur, J. Nyitrai, M. Kajtanperedy, G. Simig, G. Hornyak, G. Doleschall, *J. Chem. Soc. Perkin Trans. 1* 1986, 221–227.

[16] Y. Ueda, C. E. Damas, V. Vinet, *Can. J. Chem.* 1983, 61, 2257–2263.

[17] a) W. S. Mahoney, D. M. Brestensky, J. M. Stryker, *J. Am. Chem. Soc.* 1988, 110, 291–293; b) B. H. Lipshutz, W. Chrisman, K. Noson, P. Papa, J. A. Sclafani, R. W. Vivian, J. M. Keith, *Tetrahedron* 2000, 56, 2779–2788.

[18] J. D. Erfle, *Biochim. Biophys. Acta* 1973, 316, 143–155.

[19] B. O. Bachmann, C. A. Townsend, *Biochemistry* 2000, 39, 11187–11193.

[20] M. E. Jung, G. Piizzi, *Chem. Rev.* 2005, 105, 1735–1766.

[21] M. Rodriguez, L. E. Nunez, A. F. Brana, C. Mendez, J. A. Salas, G. Blanco, *Antimicrob. Agents Chemother.* 2011, 55, 1638–1649.

Received: June 11, 2011  
Published online on August 24, 2011

ChemBioChem 2011, 12, 2159–2165  
© 2011 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim  
www.chembiochem.org 2165
