
Review

Cholesterol catabolism as a therapeutic target in *Mycobacterium tuberculosis*

Hugues Ouellet, Jonathan B. Johnston and Paul R. Ortiz de Montellano

Department of Pharmaceutical Chemistry, University of California at San Francisco, Genentech Hall, N572D, 600 16th Street, San Francisco, CA 94158-2517, USA

*Mycobacterium tuberculosis* (*Mtb*) is an intracellular pathogen that infects 10 million people worldwide and kills 2 million people every year. The uptake and utilization of nutrients by *Mtb* within the host cell is still poorly understood, although lipids play an important role in *Mtb* persistence. The recent identification of a large regulon of cholesterol catabolic genes suggests that *Mtb* can use host sterol for infection and persistence. In this review, we report on recent progress in elucidation of the *Mtb* cholesterol catabolic reactions and their potential utility as targets for tuberculosis therapeutic agents.

**Tuberculosis: an old disease but still a continuing threat**

Tuberculosis, which has been a scourge of mankind for thousands of years, continues to be a major cause of death in developing countries and in environments where drug resistance can develop rapidly. The threat of tuberculosis has become greater over the past decades with the expansion from single drug resistance to multiple drug resistant strains and, most recently, the emergence of extreme drug resistance that threatens to overwhelm all available drugs. The importance of tuberculosis in developing countries, the potential of its renewal as a major threat in the developed world and the determination of the genome of *Mycobacterium tuberculosis* (*Mtb*) [1] have intensified the search for new avenues in the design of therapeutic agents against *Mtb*. Many of the available drugs for tuberculosis target its lipid biosynthetic pathways. *Mtb* has highly unusual cell wall lipids (Figure 1) and a relatively large fraction of its genes are devoted to the synthesis of these lipids. Although *Mtb*, similar to most bacteria, does not make sterols, it has been shown that cholesterol is both required for infection of macrophages and that the mycobacteria can employ cholesterol as a sole source of carbon. In this review, we summarize the available information on the involvement of cholesterol in the virulence and growth of *Mtb*, as well as recent research suggesting that the utilization of cholesterol could provide new targets for the development of novel therapeutic agents.

**Mtb does not synthesize cholesterol**

Cholesterol is a major structural component of animal cell membranes, where it is required to maintain proper membrane permeability and fluidity. In addition, cholesterol is an important anabolic precursor for the biosynthesis of bile acids, vitamin D and steroid hormones. Although cholesterol is mainly synthesized in animals, small quantities are also produced in plants and fungi. Sterol biosynthesis in bacteria has been controversial, with the only unambiguously equivalent enzymes found in the proteobacterium, *Methyloroccus capsulatus*, and in the planctomycete, *Gemmata obscuriglobus* [2,3]. By contrast, *Mtb* apparently lacks the squalene monooxygenase and oxidosqualene cyclase that are absolutely essential for sterol biosynthesis. Interestingly, the *Mtb* genome encodes the gene for CYP51B1, a cytochrome P450 enzyme that catalyzes the 14α-demethylation of lanosterol to give the 8,14-diene, a key step in cholesterol biosynthesis [4]. Although bacterial CYP51 enzymes are well conserved across actinomycetes species, it is relevant that CYP51 from *Streptomyces coelicolor* A3(2), unlike its eukaryotic counterparts, is not essential for cell viability under *in vitro* growth conditions [5]. The physiological function of CYP51B1 enzymes in *Mtb* and other bacteria remains obscure [5].

**Cholesterol is important for *Mtb* infection and persistence**

*Mtb* can infect, grow and survive in the harsh environment of the macrophage and other host cells using mechanisms that are not yet well understood [6,7]. Host cholesterol levels are thought to be involved in the development of *Mtb* infection [8], with high levels of cholesterol in the diet significantly enhancing the bacterial burden in the lung [9] and impairing immunity to *Mtb* [10]. Specifically, cholesterol is required for the phagocytosis of mycobacteria into macrophages [11,12]. In fact, mycobacteria enter phagocytes through cholesterol-rich membrane microdomains [13]. Additionally, cholesterol is required to hold the host protein coronin 1 (TACO) on *Mtb*-infected phagosomes, leading to the inhibition of phagosome–lysozyme fusion [14]. This experimental evidence suggests an important role for cholesterol during *Mtb* infection and persistence.

**Mtb imports and utilizes cholesterol**

The acquisition and utilization of nutrients during *Mtb* infection is poorly understood, although it has been proposed that host lipids play an important role in *Mtb* survival [15]. Several lines of evidence suggest that pathogenic

Corresponding author: Ortiz de Montellano, P.R. (ortiz@cgl.ucsf.edu).

0966-842X/$ – see front matter © 2011 Elsevier Ltd. All rights reserved. doi:10.1016/j.tim.2011.07.009 Trends in Microbiology, November 2011, Vol. 19, No. 11

Review

Trends in Microbiology November 2011, Vol. 19, No. 11

Figure 1. Schematic representation of the cell envelope of *Mtb* with lipidomic analysis of the surface lipids and PDIM. The cell envelope of *Mtb* cells, from the inside to the outside, consists of a plasma membrane; a cell wall composed of three covalently linked macromolecules (peptidoglycan, arabinogalactan and mycolic acids) and noncovalently linked lipids and proteins; and a capsule consisting of polysaccharides, proteins and lipids. For mass spectrometric analysis, total lipids were extracted from cells grown in the presence of glycerol, cholesterol or heptadeuterated cholesterol. As shown, cholesterol metabolism in *Mtb* cells increases the average mass of the methyl-branched lipid virulence factor PDIM, presumably due to a higher metabolic flux of propionate derived from cholesterol side chain catabolism [22]. Abbreviations: PIM, phosphatidylinositol mannosides; PDIM, phthiocerol dimycocerosate; PAT, polyacyltrehaloses; DAT, diacyltrehaloses; SL-1, sulfolipid-1s; TDM, trehalose-6,6'-dimycolates; TMM, trehalose-6-monomycolates.

mycobacteria primarily use fatty acids rather than carbohydrates as carbon substrates during infection [16]. Respiration of *Mtb* in mouse lungs is strongly stimulated by fatty acids but is unresponsive to carbohydrates [16,17]. An ABC-like transport system, *mce4*, was recently identified in *Mtb* that is involved in cholesterol import into *Mtb* [18]. Deletion of the *mce4* operon in *Mtb* resulted in a growth defect when cholesterol was used as the primary source of carbon [18]. *Mce4* loci are also found in many other actinomycetes species, including *Rhodococcus jostii* [19,20], *Mycobacterium smegmatis* [21] and *Mycobacterium bovis* BCG [19], and these can also utilize cholesterol for growth. Using ^14^C-labeled cholesterol derivatives, it has been elegantly demonstrated that cholesterol is degraded by *Mtb*, with the carbon atoms from the sterol framework and the aliphatic side chain going to energy production and lipid synthesis, respectively [18]. Using a lipidomic approach, Yang *et al.* showed that cholesterol metabolism in *Mtb* cells increases the average mass of the lipid virulence factor phthiocerol dimycocerosate (PDIM), presumably due to a higher metabolic flux of propionate derived from cholesterol catabolism [22]. Subsequently, heptadeuterated 25, 26, 26, 26, 27, 27, 27-D7-cholesterol was used to establish the role of the cytochrome P450 encoded by the *Mtb cyp125* gene in the degradation of cholesterol and incorporation of the cholesterol side chain into the virulence factor PDIM [23] (Figure 1). Additionally, the block of cholesterol import in Δ*mce4* cells markedly attenuated infection in both activated macrophages and the mouse model of infection, confirming that cholesterol metabolism is important for the chronic phase of infection [18]. Screening of a transposon mutant library allowed the identification of another locus, *igr*, which is essential for growth in unactivated macrophages [24]. The *igr* operon consists of six genes, including a cytochrome P450 (*cyp125*), two probable acyl-CoA dehydrogenases (*fadE28* and *fadE29*), two conserved hypothetical proteins (*Rv3541-2c*) and a putative lipid carrier protein (*ltp2*) (Figure 2). In a follow-up study, inactivation of the whole *igr* operon was shown to result in a marked defect in growth on cholesterol alone or in combination with glycerol, suggesting some form of cell intoxication by cholesterol or cholesterol-derived metabolites [25]. Moreover, Δ*igr* cells exhibited a growth defect in the early phase of the mouse model of infection, an effect that is suppressed by mutating the sterol uptake *Mce4* system. Together, these results indicate that cholesterol is important throughout infection and is crucial for *Mtb* persistence, but additionally show that ineffective breakdown of this sterol in the early phase of disease gives rise to a different form of attenuation.

*Mtb* cholesterol degradation pathway

A growing body of evidence suggests that *Mtb* uses host cholesterol during infection as a source of carbon and energy. In their breakthrough study, Van der Geize *et al.* identified a complete suite of genes required for steroid degradation in *R. jostii* [19]. Interestingly, their bioinformatic analysis found that the 51 genes specifically expressed during growth on cholesterol, including those predicted to break down the sterol A and B rings, are likewise found within an 82-gene cluster in the *Mtb* and *M. bovis* BCG genomes. One of the *Mtb* genes thus identified, *kstR* (*Rv3574*), encodes a TetR-like transcriptional repressor. Inactivation of its homolog in the related saprophyte *M. smegmatis* derepressed the expression of 83 genes, as assayed by quantitative reverse transcriptase (RT)-PCR and microarrays, with 74 orthologs predicted to

KstR-regulated genes (Rv3494c-Rv3547...Rv3566-Rv3574)

mce4F (Mce4 transport system) #
mce4E (Mce4 transport system) #
mce4D (Mce4 transport system) #
mce4C (Mce4 transport system) #
mce4B (Mce4 transport system) #
mce4A (Mce4 transport system) #
yrb4A (possible ABC transporter) #
yrb4B (possible ABC transporter) #
hsd4A (possible 17β-HSD) #
fdxD (probable ferredoxin)
fadE27 (probable acyl-CoA dehydrogenase)
PE PGRS53 (PE PGRS family)
PE PGRS54 (PE PGRS family)
ilvX (probable acetyl-CoA ligase)
RV3510c (CHP)
PE PGRS55 (PE PGRS family)
PE PGRS56 (PE PGRS family)
fadD17 (possible fatty-aid-CoA hydrolase)
fadD19 (probable fatty-aid-CoA hydrolase)
RV3517 (CHP)
cyp142 (cytochrome P450)
RV3519 (CHP) #
RV3520c (coenzyme F420-dependent oxidoreductase)
ltp4 (probable ketoacyl-CoA thiolase)
ltp3 (probable ketoacyl-CoA thiolase)

KstR-regulated genes (Rv3526-Rv3547...Rv3566-Rv3523)

kshA (ketosteroid-9α-hydroxylase) #
RV3527 (HP) #
RV3528c (CHP)
RV3529c (possible oxidereductase)
RV3530c (hypothetical protein)
PPE61 (PPE family)
PPE62 (PE family)
hsaF (probable 2-oxovalerate aldolase) #
hsaG (probable dehydrogenase)
hsaE (probable aldehyde hydratase)
kstD (∆-KSTD) *
hsd4B (probable enoyl-CoA thiolase) #
PPE63 (PPE family)
ltp2 (probable ketoacyl-CoA thiolase) #
RV3541c (CHP) #
fadE29 (probable acyl-CoA dehydrogenase)
RV3542c (CHP) #
fadA5 (cytochrome P450) #
cyp125 (acetoacetyl-CoA thiolase) #, *
RV3547 (CHP)
nat (arylamine N-acetyltransferase)
RV3566A (CHP)
hsaB (3-hydroxy-9,10-seconandrost-1,3,5(10)-triene-9,17-dione hydroxylase) #
hsaC (3α-DHSA dioxygenase) #
hsaD (4,9-DHSA hydroxylase) #
1,3,5(10)-triene-9,17-dione hydroxylase) #, *
kshB (ketosteroid-9α-hydroxylase) #
RV3572 (HP)
fadE34 (probable acyl-CoA dehydrogenase)
ksfR (Tet-R transcriptional repressor) #

igr operon#

Rv1106c (3β-HSD)

dehydrogenase
dehydrogenase
echA20 (possible enoyl-CoA hydratase)
Rv3551 (possible CoA transferase (α subunit)) #
Rv3552 (possible CoA transferase (β subunit)) #
fdxB (possible protein)
Rv3555c (CHP)
fadA6 (acetoacetyl-CoA thiolase) #, *
ksfR2 (Tet-R transcriptional repressor)
PP64 (PPE family)
Rv3559c (probable oxidereductase)
fadE30 (probable acyl-CoA dehydrogenase) #, *
fadE31 (probable acyl-CoA dehydrogenase)
fadE32 (probable acyl-CoA dehydrogenase)
fadE33 (probable acyl-CoA dehydrogenase)
aspB (possible aspartate aminotransferase)

KstR-regulated

KstR2-regulated genes (Rv3548c-Rv3565)

Figure 2. The cholesterol catabolic genes of Mtb. The transcriptional repressor KstR controls the expression of the majority of the genes clustered in two chromosomal segments (Rv3494c–Rv3547 and Rv3566c–Rv3574). Many other KstR-regulated genes are scattered throughout the Mtb genome, including Rv1106c. KstR2 controls the expression of a second cholesterol regulon in M. smegmatis [28], with the corresponding orthologs in the Mtb genome (Rv3548c–Rv3565). Genes in the map are color-coded according to their assigned or proposed catalytic activity: red, uptake; side chain degradation, magenta; degradation of rings A and B, green; β-oxidation, blue; transcriptional regulator, orange; unassigned function, black; not regulated by KstR or KstR2, gray. The genes encoding for enzymes specifically involved in the degradation of rings C and D are not unambiguously assigned yet. The catalytic functions of HsaEFG (Rv3534c–Rv3536c) in Mtb have not been investigated yet. Genes predicted [30,31] or proven [18,24,27,46,57] to be essential for survival in macrophage cells and in the mouse model of infection are indicated by * and #, respectively. Abbreviations: ABC transporter, ATP-binding cassette transporter; PE, protein containing Pro-Glu motifs; PE-PGRS, protein consisting of the PE domain followed by a C-terminal extension with multiple tandem repetitions of Gly-Gly-Ala or Gly-Gly-Asn; CHP, conserved hypothetical protein; PPE, protein containing Pro-Pro-Glu motifs.

be similarly regulated in Mtb (Figure 2) [26]. The role of KstR as a cholesterol-responsive regulator in Mtb was later demonstrated by performing transcriptional profiling of cholesterol gene inactivation and in a kstR mutant of Mtb, corroborating the results obtained from the other actinomycetes [27]. More recently, a second transcriptional repressor, kstR2 (Rv3557c), was identified and shown to control the expression of a subset of 15 genes (Rv3548– Rv3565) within the kstR regulon (Figure 2) [28]. Moreover, the observation that several genes in the kstR/kstR2 and cholesterol regulons are induced in macrophages [29] or are essential for infection [30,31] strongly suggests that cholesterol catabolism is central to Mtb survival in vivo. In the following sections we summarize the cholesterol cata- bolic pathway in Mtb, which can be divided into two major phases: (i) initial degradation of the aliphatic side chain and (ii) subsequent degradation of the A-D rings (Figure 3). It remains unclear whether there is an obliga- tory order of the degradation reactions in Mtb. The evi- dence obtained from the literature on rhodococcal sterol

catabolism suggests that intermediates of ring and side chain degradation can be exchanged between the two pathway branches [32]. However, the situation seems to differ in Mtb, as blockage of the side chain degradation resulted in the accumulation of cholest-4-en-3-one as a major metabolite [23], suggesting that the ring-degrading enzymes (e.g. KsaAB and HsaA-C) optimally act after the side chain is removed.

Side chain degradation

Less is known about the degradation of the aliphatic side chain of cholesterol than about the degradation of the sterol framework. The side chain, as also found for the biosynthe- sis of bile acids in mammals, is generally accepted to be shortened by β-oxidation reactions (Figure 3). This is con- sistent with the identification of genes encoding putative β-oxidation enzymes in the cholesterol regulons of Mtb and Rhodococcus rhodochrous DSM 43269. These genes include Mtb fadA5. Indeed, the thiolase enzyme encoded by fadA5 catalyzes the thiolysis of acetoacetyl-CoA in vitro and is

Review

Trends in Microbiology November 2011, Vol. 19, No. 11

Sterol side chain degradation

![Diagram of sterol side chain degradation]

Sterol ring degradation

![Diagram of sterol ring degradation]

Figure 3. Proposed degradation pathways and flux of metabolites derived from catabolism of the cholesterol aliphatic side chain and ring nucleus. All the enzymes for which a catalytic activity has been identified are labeled in red, whereas the putative β-oxidation enzymes are labeled in magenta. The carbon atoms derived from the ring nucleus are converted to CO₂ via the tricarboxylic acid cycle (TCA), whereas the propionyl-CoA produced from the degradation of the side chain is assimilated into mycobacterial lipids, including the virulence factor PDIM. The enzymes and the cholesterol metabolites are described in the text. The methylmalonyl pathway enzymes include a propionyl-CoA carboxylase (PCC) and a vitamin B₁₂-dependent methylmalonyl-CoA mutase (MCM). The fate of DOHNAA is still unknown.

required for growth on cholesterol [27]. This thiolase activity, which is consistent with removal of the side chain by β-oxidation to yield androsterone metabolites [e.g. 4-androstenedione (AD) and 1,4-androstenedione (ADD)] [27], is required for virulence, especially during the late stage of mouse infection. Very recently, the roles of several genes [fadD17, fadD19, fadE26, fadE27 and ro04690 (DSM43269)] putatively involved in cholesterol side chain degradation were investigated in the actinomycete *R. rhodochrous* DSM43269 [33]. Among those genes, *fadD19* encodes a steroid-coenzyme A (CoA) ligase with an essential in vivo role in degradation of the side chains of C-24 branched chain sterols, but interestingly not cholesterol. The high similarity (67%) between the *Rhodococcus* FadD19 (DSM43269) and *Mtb* FadD19 enzymes suggests that FadD19 also has an in vivo role in sterol metabolism in *Mtb* [33].

Sterol ring degradation

The first step of sterol ring degradation is the conversion cholesterol to cholest-4-en-3-one catalyzed by either a 3β-hydroxysteroid dehydrogenase (3β-HSD) or a cholesterol oxidase (ChoD) (Figure 3). *Mtb Rv1106c* encodes for a 3β-HSD enzyme that is a member of the short chain dehydrogenase superfamily, uses NAD⁺ as a cofactor and oxidizes cholesterol, pregnenolone and dehydroepiandrosterone to their respective 3-keto-4-ene products [34]. *Mtb* ChoD, encoded by *Rv3409c*, which shares only 24% amino acid identity with the well-characterized cholesterol oxidases from *Streptomyces* and *Rhodococcus*, is required for *Mtb* virulence [35]. A significant amount of cholest-4-en-3-one was shown to temporarily accumulate in cholesterol-fed *M. smegmatis* cells overexpressing *Mtb* ChoD [35], but the disruption of *Rv3409c* in *Mtb* cells did not result in a marked in vitro defect of growth on

Review

Trends in Microbiology November 2011, Vol. 19, No. 11

cholesterol [36], suggesting that ChoD does not play a major role in cholesterol oxidation. By contrast, *Mtb* Δ*Rv1106c* cells cannot grow on cholesterol as a primary source of carbon, but they did not show an attenuated phenotype during infection of macrophages and guinea pigs [36]. The authors therefore concluded that cholesterol is not an essential nutrient during infection, but the existence of low levels of a redundant 3β-HSD or cholesterol oxidase activity expressed during infection and/or the presence of host cholest-4-en-3-one [37] cannot be definitively ruled out. Redundant 3β-HSD activities were also proposed for the initiation of cholesterol degradation in *M.* smegmatis, although the enzyme similar to *Mtb* Rv1106c and encoded by MSMEG_5228 also appears to constitute the major activity [38].

KstD is a flavoprotein that catalyzes the trans-axial elimination of the C1(α) and C2(β) hydrogen atoms of the 3-ketosteroid A ring (Figure 3) [39,40]. The *Rv3537* gene encodes a putative and unique Δ<sup>1</sup>KstD enzyme that is part of the cholesterol regulon [19] and is essential for survival in macrophages [31]. Phylogenetic analysis indicates that the *Rv3537* gene product is an ortholog of Δ<sup>1</sup>KstD3 of *Rhodococcus erythropolis* SQ1 [41]. Targeted disruption of the *Rv3537* (Δ<sup>1</sup>KstD) gene inhibited growth on cholesterol and resulted in accumulation of 9-hydroxy-4-androstene-3,17-dione [42]. Interestingly, heterologously expressed *Mtb* Δ<sup>1</sup>KstD displayed a clear preference for 3-ketosteroids with a saturated A ring, with little (e.g. progesterone) or no (e.g. 4-androstenedione and cholest-4-en-3-one) detectable activity for 3-keto-Δ<sup>4</sup> steroids [41]. By contrast, biochemical characterization of the rhodococcal Δ<sup>1</sup>KstD enzymes revealed relatively high levels of activity with 3-keto-Δ<sup>4</sup> steroids [41,43,44]. This data suggests that *Mtb* may possess an unidentified reductase activity that saturates 3-keto-Δ<sup>4</sup> steroids.

9-Hydroxylation of the 3-ketosteroid together with the C1-C2 dehydrogenation catalyzed by Δ<sup>1</sup>KstD are key steps leading to opening of the B ring and aromatization of the A ring via the unstable intermediate, 9-hydroxy-1,4,androsten-3-17-dione (9OHADD) (Figure 3). In actinomycetes, the 9-hydroxylation of 3-ketosteroids is catalyzed by KshAB, a two-component Rieske oxygenase [45–48]. KshA (*Rv3526*), the oxygenase component, is a homotrimer containing a Rieske [2Fe-2S] cluster and mononuclear ferrous iron, whereas the reductase component KshB (*Rv3571*) is a monomeric protein containing a plant-type [2Fe-2S] cluster and flavin adenine dinucleotide [45]. Consistent with a role in cholesterol utilization, a transposon mutant of *kshA* exhibited a strongly attenuated phenotype in the infection of interferon-γ-activated macrophages [31]. Moreover, Hu *et al.* reported that targeted disruption of either *kshA* or *kshB* resulted in mycobacterial cell death in the mouse model of infection [46]. Interestingly, inactivation of *kshB*, but not *kshA*, alters the biosynthesis of penta-acylated trehalose (PAT), a multimethyl-branched lipid class, suggesting that the reductase of KshB is involved in several processes.

In *R. jostii* RHA1 and *Mtb*, the *hsaACDB* genes are part of a single operon within the cholesterol regulon [19]. The *hsaA* and *hsaB* genes encode an oxygenase and a reductase, respectively [49]. The two proteins act together as a flavin-dependent monooxygenase that hydroxylates 3-hydroxy-9,10-seconandrost-1,3,5(10)-triene-9,17-dione (3-HSA) to the catechol 3,4-dihydroxy-9,10-seconandrost-1,3,5(10)-triene-9,17-dione (3,4-DHSA) (Figure 3). Analysis of the crystal structure of ligand-free HsaA at 2.5 Å resolution revealed that this enzyme displays the same fold, flavin-binding site and catalytic residues as *p*-hydroxyphenyl acetate hydroxylase. Additionally, deletion of *hsaA* in *R. jostii* RHA1 caused a growth defect on cholesterol, and the screening of a transposon mutant library identified *hsaA* as one of the *Mtb* genes required for growth in activated macrophages [31].

HsaC, the next enzyme in the pathway, is an iron-dependent extradiol dioxygenase that efficiently oxygenates and cleaves the catecholic cholesterol metabolite 3,4-DHSA to 4,5-9,10-diseco-3-hydroxy-5,9,17-trioxoandrosta-1(10),2-diene-4-oic acid (4,9-DSHA) (Figure 3). The structures of HsaC:DHSA complexes at 2.1 Å revealed two catechol-binding modes with the position of the bicycloalkanone moiety of DHSA being very similar in the two binding models, suggesting that this interaction is a determinant of the initial substrate-binding step. The inactivation of *Mtb hsaC* resulted in cell death when the cells were grown in the presence of cholesterol, presumably due to cell poisoning by the accumulated catechol metabolites. The infection of immunocompromised mice and wild-type guinea pigs with Δ*hsaC* cells also revealed a significant attenuation phenotype. From these observations, the authors concluded that cholesterol metabolism is most important during the chronic stage of infection, but also contributes to pathogen dissemination throughout the infection.

The catabolic gene, *hsaD*, was identified as one of the *Mtb* genes required for survival in macrophages [31]. HsaD is a member of the α/β hydrolase family involved in the aerobic degradation of aromatic compounds in microbes. This enzyme, for which a crystal structure at 2.35 Å is available [50,51], catalyzes hydrolytic carbon-carbon bond cleavage of 4,9-DSHA to yield 9,17-dioxo-1,2,3,4,10,19-hexanorandrostan-5-oic acid (DOHNAA) and 2-hydroxy-hexa-2,4-dienoic acid (HHD) (Figure 3) [51]. HHD is finally metabolized to tricarboxylic acid cycle intermediates and propionyl-CoA, whereas it is unknown if *Mtb* further degrades DOHNAA, which corresponds to the C ring and D ring fragment. Although much is known about the steps involved in sterol ring degradation, substantial gaps exist in our understanding of this pathway. Nevertheless, the structural data obtained for HsaA, HsaC and HsaD provides insights into the binding of steroid substrates that should facilitate inhibitor design.

Role of *Mtb* P450 in cholesterol catabolism

The saturated side chain of cholesterol must be chemically functionalized at the ω-position before it can enter into the *Mtb* β-oxidation pathway (Figure 3). A minimum of four chemical steps are necessary to prepare the side chain for β-oxidation. The first three steps, sequential oxidations of the cholesterol side chain to the terminal alcohol, aldehyde and acid, are catalyzed by cytochrome P450 enzymes. Three *Mtb* P450 enzymes, CYP125 (*Rv3545c*), CYP142 (*Rv3518c*) and CYP124 (*Rv2266*), are capable of oxidizing the side chains of cholesterol and cholest-4-en-3-one.

final ATP-dependent step is catalyzed by a sterol-CoA ligase [52].

Cytochrome P450 enzymes catalyzing steroid side chain oxidations

The first oxidation introduces a hydroxy group onto the steroid side chain, the second oxidizes the alcohol to the aldehyde and the third converts the aldehyde to a carboxyllic acid. In principle, the initial oxidation could occur at a primary, secondary or tertiary position, as all three are found in the cholesterol side chain, although the ease of hydroxylation depends on the relative C-H bond dissociation energies and thus decreases in the order 3° > 2° >> 1°. Nevertheless, only oxidation of the primary C-H bond to give the ω-oxidation product is observed [53]. These P450 enzymes, which have evolved to selectively oxidize the relatively unreactive terminal methyl group, enforce a strict regiospecificity by preventing the more reactive carbon of the side chain from being properly positioned relative to the heme iron atom [53]. However, as outlined below, there are distinct differences in the gene induction, protein expression, stereochemical preferences and catalytic efficiencies of the three enzymes able to oxidize cholesterol and cholest-4-en-3-one.

Which P450 enzymes are involved?

Comparison of the genome sequences of Mtb CDC1551 and H37Rv reveals key differences in the cytochrome P450 loci. In CDC1551, a 639-bp deletion encompassing the promoter region and the sequence encoding the first 18 amino acids of the protein is found upstream of the cyp142 (Rv3518c) gene, indicating that this strain does not produce CYP142. The cyp124 genes of H37Rv and CDC1551 only differ by two amino acids, G25D and Y75N, respectively. However, no notable differences in catalytic activity are observed between the CYP124 variants. By contrast, the cyp125 genes are identical in both strains. In vitro, CYP125, CYP142 and CYP124 catalyze the sequential oxidation of the hydrocarbon side chain of cholesterol to the carboxyllic acid. The second-order rate constants (kcat/KM) for terminal hydroxylation of the cholesterol and cholest-4-en-3-one side chains establish that CYP125 is a slightly more efficient catalyst than CYP142 [54], and both are much more efficient than CYP124 [54]. The characterization of CYP125, reported independently by four groups, includes X-ray crystal structures [23,55], biochemical studies [23,32,55,56] and in vivo genetic analyses that probe the loss and gain of this P450 function in different virulent clinical strains and R. jostii RHA1 [23,32,56]. Two other Mtb P450 enzymes, CYP141 and CYP130, were found not to catalyze the oxidation of these sterols [54].

Oxidation can occur at either of the two terminal methyl groups, producing either the (25S)-and/or (25R)-26-hydroxy sterols. The stereochemical preferences of these enzymes differ: CYP125 preferentially forms the (25S)-26-hydroxy product, whereas CYP142 and CYP124 produce the (25R)-26-hydroxy configuration [54]. It is unclear whether there are any physiological consequences of these different stereochemical preferences.

Several lines of evidence indicate that CYP125 and CYP142 are induced when Mtb is grown on cholesterol

as the sole carbon source [23,27,54]. Immunoblots confirmed that the CYP125 and CYP142 proteins are expressed and induced in cells grown on cholesterol, whereas CYP124 was not detectably expressed under these conditions [54]. Interestingly, cyp142 is able to fully complement the loss of cyp125 in CDC1551Mtb cells when a chromosomal copy is introduced into the Δcyp125 knock-out cells. CYP125 is the primary P450 responsible for the initiation of cholesterol side chain degradation in Mtb. CYP142 can substitute for CYP125 in H37Rv but not CDC1551 cells, as the cyp142 gene in the latter is inactivated by a mutation. Similar to CYP125 and CYP142, CYP124 can catalyze sequential oxidation of the cholesterol and cholest-4-en-3-one side chains, but CYP124 expression is not induced by cholesterol [54]. In agreement with the in vitro kinetic data, CYP124 can only partially reverse the abnormal growth phenotype of Δcyp125 cells grown on cholesterol. Concomitant with this partial growth restoration, cholest-4-en-3-one was found to accumulate in the Δcyp125 CYP124 overexpressing cells [54]. The presence of multiple cytochrome P450 isoforms capable of oxidizing the sterol side chain and differences in their expression highlight some of the potential challenges of drug-targeting critical pathways of Mtb and the need for understanding the biochemical and genetic basis for the differences in clinically relevant strains.

Toxicity of cholesterol metabolites

Inhibition of the cholesterol catabolic pathways not only can cause carbon starvation but blockage at certain steps has also been shown to result in cell death or bacteriostasis. For instance, the Mtb ΔhsaC mutant not only failed to grow on cholesterol but the cells developed a pink color, indicating an accumulation of catechols and their conversion to colored o-benzoquinones and condensation products [19,57]. Additionally, a loss of cell viability was observed when ΔhsaC cells were grown in the presence of cholesterol, suggesting that the catechol derivatives are toxic to Mtb cells [57]. A growth inhibitory effect of cholesterol metabolism that depends on cholesterol import was observed when Δigr cells were incubated with both cholesterol and glycerol [25]. The same inhibitory effect was observed when cyp125, which is part of the igr operon, was deleted in similarly cultured Mtb CDC1551 cells. Importantly, cholest-4-en-3-one, which was shown to accumulate in high amounts in cyp125 mutant cells [23,56], was toxic to wild-type Mtb CDC1551, Erdman and H37Rv strains grown in media containing a second carbon source [23].

A highly speculative mechanism for the toxicity associated with the accumulation of cholest-4-en-3-one is an alteration of the cell membrane integrity. In eukaryotic plasma membrane models, it was shown that cholesterol can participate in strong and close packing with saturated lipids, a crucial feature required for the formation of lipid domains, with the latter implicated in numerous biological processes, including signal transduction events, and toxin entry into cells [58]. It was shown that cholest-4-en-3-one, for which the 3-OH group is replaced by a keto group, inhibits lipid domain formation, thus altering the plasma membrane properties [58,59]. When cultivated in the presence of cholesterol and a second carbon source, Mtb is able to

Review

Trends in Microbiology November 2011, Vol. 19, No. 11

accumulate cholesterol in the free-lipid zone of its cell wall, resulting in a decreased permeability for the primary anti-tubercular drug, rifampin [42]. From these observations, it can be speculated that high levels of cholest-4-en-3-one might render the cell wall more permeable to drugs and xenobiotics. Whatever the mechanism, the identification of cholest-4-en-3-one as a toxic agent in *Mtb* opens up the intriguing possibility that elucidation of its site(s) of action could reveal new targets for antitubercular drug design.

Cholesterol catabolism inhibitors

In view of the importance of cholesterol catabolism in *Mtb* infection and persistence, targeting this pathway could represent a fruitful avenue for the development of new therapeutic agents. Some of the important questions that require attention are outlined in Box 1. The antifungal action of azole drugs has been shown to result from inhibition of sterol biosynthesis at the level of CYP51. Azole drugs, most notably econazole, exhibit antimycobacterial activities against both latent *Mtb* and multidrug-resistant strains [60,61], a finding that implicates one or more cytochrome P450 enzymes as targets (Table 1). In this regard, all the *Mtb* cytochrome P450 enzymes characterized to date, including CYP125 and CYP142, bind azole drugs with low-to submicromolar affinities (Table 1) [62–67]. The crystal structure of CYP125 in complex with

**Table 1. Apparent dissociation constant (*K*<sub>D</sub>) for select azole drugs against *Mtb* sterol-oxidizing cytochrome P450 enzymes and minimum inhibitor concentration (MIC) values**

| Inhibitor | *K*<sub>D</sub><sup>a</sup> (µM) | MIC<sup>b</sup> (µg/ml) |
| --- | --- | --- |
|  | CYP51B1 | CYP124 | CYP125 | CYP142 |  |
| ![](https://via.placeholder.com/150) Econazole | 0.77 ± 0.04 | 2.1 ± 0.1 | 11.7 ± 0.7 | 4.6 ± 0.2 | 0.25 |
|  |  |  |  |  | 16<sup>b</sup> |
| ![](https://via.placeholder.com/150) Miconazole | 0.59 ± 0.03 | 1.9 ± 0.2 | 4.6 ± 0.4 | n.a. | 16.6<sup>b</sup> |
| ![](https://via.placeholder.com/150) Clotrimazole | < 1.0 | 2.5 ± 0.1 | 5.3 ± 0.6 | 3.8 ± 0.9 | 0.38 |
|  |  |  |  |  | 33<sup>b</sup> |
| ![](https://via.placeholder.com/150) Ketoconazole | 19 ± 1.9 | n.a. | 27.1 ± 0.9 | 21 ± 4 | 30<sup>b</sup> |

<sup>a</sup>The *K*<sub>D</sub> values were obtained from: [4,56,63,64,66,67].

<sup>b</sup>The MIC values in bold were obtained using BACTEC7H12B, which is an enriched 7H9 medium containing radiolabeled palmitic acid as a source of carbon [72], as opposed to the use of Youman's minimal medium [60].

Review

Box 1. Outstanding questions
- Are the C and D rings and the side chain of cholesterol completely degraded, and if so how?
- In addition to leading to an increased mass of the methyl-branched lipid virulence factor, PDIM, does the utilization of host cholesterol result in the biosynthesis of unidentified but important lipids *in vivo*?
- Does *Mtb* possess functionally redundant and as yet unidentified cholesterol oxidase or dehydrogenase activities? The apparent dispensability of the 3β-HSD encoded by *Rv1106c* for infection and the lack of detectable cholesterol oxidase activity catalyzed by ChoD (*Rv3409c*) *in vitro* suggest that cholesterol is not an essential nutrient during infection. Could cholest-4-en-3-one also be produced by the host at the site of infection?
- Is cholesterol simply metabolized as a source of carbon and energy for growth or do cholesterol-derived metabolites secreted by *Mtb* cells have any physiological effects through host-pathogen interactions?
- What is the target of cholest-4-en-3-one that results in a bacteriostatic effect? The elucidation of this inhibition effect could potentially lead to the identification of essential activities that could also be useful for the development of new therapeutics.
- Is it possible to design agents that specifically target *Mtb* enzymes and not the host isoforms? Although antifungal azole drugs are known to target fungal cytochrome P450 enzymes, most of them cannot be administered orally as it could result in serious side effects due to inhibition of the human cytochrome P450 complement.

Concluding remarks
Despite the fact that *Mtb* does not make any sterols, it retains one cytochrome P450 enzyme (CYP51) with high-sequence identity and the ability to catalyze the same specific reaction as lanosterol 14α-demethylase, a key enzyme in sterol biosynthesis. Furthermore, roles for host cholesterol and its metabolism throughout the infection, particularly during the chronic phase of infection, are now well established. Cholesterol is required for the *Mtb* infection of macrophages and serves as a carbon source for growth of the mycobacteria. Degradation of the cholesterol molecule is initiated by hydroxylation of the sterol side chain either by CYP125 or CYP142. In the absence of these enzymes, the cholesterol metabolite cholest-4-en-3-one accumulates and has a bacteriostatic effect. Catechol intermediates that are also toxic accumulate if sterol degradation is interrupted at the level of HsaC, the enzyme that fragments ring A of the sterol. The cholesterol degradation pathway thus provides targets for the endogenous generation of toxic molecules. Furthermore, azole drugs that normally target cytochrome P450 enzymes have been shown to inhibit the growth of *Mtb* in both the proliferating and resting states. These collective results suggest that the cholesterol degradation pathway could prove to be a fertile ground for the development of antituberculosis agents.

Acknowledgments
The preparation of this review and the work from the author’s laboratory were supported by National Institutes of Health Grant AI07824. J.B.J. was supported by a Heiser Postdoctoral Fellowship for Research in Leprosy and Tuberculosis sponsored by the New York Community Trust.

References
1. Cole, S.T. *et al.* (1998) Deciphering the biology of *Mycobacterium tuberculosis* from the complete genome sequence. *Nature* 393, 537–544
2. Lamb, D.C. *et al.* (2007) Lanosterol biosynthesis in the prokaryote *Methylococcus capsulatus*: insight into the evolution of sterol biosynthesis. *Mol. Biol. Evol.* 24, 1714–1721
3. Pearson, A. *et al.* (2003) Phylogenetic and biochemical evidence for sterol synthesis in the bacterium *Gemmatia obscuriglobus*. *Proc. Natl. Acad. Sci. U.S.A.* 100, 15352–15357
4. Bellamine, A. *et al.* (1999) Characterization and catalytic properties of the sterol 14alpha-demethylase from *Mycobacterium tuberculosis*. *Proc. Natl. Acad. Sci. U.S.A.* 96, 8937–8942
5. Lamb, D.C. *et al.* (2002) Sterol 14alpha-demethylase activity in *Streptomyces coelicolor* A3(2) is associated with an unusual member of the CYP51 gene family. *Biochem. J.* 364, 555–562
6. Clark-Curtiss, J.E. and Haydel, S.E. (2003) Molecular genetics of *Mycobacterium tuberculosis* pathogenesis. *Annu. Rev. Microbiol.* 57, 517–549
7. Koul, A. *et al.* (2004) Interplay between mycobacteria and host signalling pathways. *Nat. Rev. Microbiol.* 2, 189–202
8. Kim, M.J. *et al.* (2010) Caseation of human tuberculosis granulomas correlates with elevated host lipid metabolism. *EMBO Mol. Med.* 2, 258–274
9. Schafer, G. *et al.* (2009) The role of scavenger receptor B1 in infection with *Mycobacterium tuberculosis* in a murine model. *PLoS ONE* 4, e8448
10. Martens, G.W. *et al.* (2008) Hypercholesterolemia impairs immunity to tuberculosis. *Infect. Immun.* 76, 3464–3472
11. Gatfield, J. and Pieters, J. (2000) Essential role for cholesterol in entry of mycobacteria into macrophages. *Science* 288, 1647–1650
12. Peyron, P. *et al.* (2000) Nonopsonic phagocytosis of *Mycobacterium kansasii* by human neutrophils depends on cholesterol and is mediated by CR3 associated with glycosylphosphatidylinositol-anchored proteins. *J. Immunol.* 165, 5186–5191
13. Munoz, S. *et al.* (2009) *Mycobacterium tuberculosis* entry into mast cells through cholesterol-rich membrane microdomains. *Scand. J. Immunol.* 70, 256–263

Review

14 Nguyen, L. and Pieters, J. (2005) The Trojan horse: survival tactics of pathogenic mycobacteria in macrophages. *Trends Cell Biol.* 15, 269–276

15 Savvi, S. *et al.* (2008) Functional characterization of a vitamin B12-dependent methylmalonyl pathway in *Mycobacterium tuberculosis*: implications for propionate metabolism during growth on fatty acids. *J. Bacteriol.* 190, 3886–3895

16 Munoz-Elias, E.J. and McKinney, J.D. (2005) *Mycobacterium tuberculosis* isocitrate lyases 1 and 2 are jointly required for in vivo growth and virulence. *Nat. Med.* 11, 638–644

17 Bloch, H. and Segal, W. (1956) Biochemical differentiation of *Mycobacterium tuberculosis* grown in vivo and in vitro. *J. Bacteriol.* 72, 132–141

18 Pandey, A.K. and Sassetti, C.M. (2008) Mycobacterial persistence requires the utilization of host cholesterol. *Proc. Natl. Acad. Sci. U.S.A.* 105, 4376–4380

19 Van der Geize, R. *et al.* (2007) A gene cluster encoding cholesterol catabolism in a soil actinomycete provides insight into *Mycobacterium tuberculosis* survival in macrophages. *Proc. Natl. Acad. Sci. U.S.A.* 104, 1947–1952

20 Mohn, W.W. *et al.* (2008) The actinobacterial mce4 locus encodes a steroid transporter. *J. Biol. Chem.* 283, 35368–35374

21 Av-Gay, Y. and Sobouti, R. (2000) Cholesterol is accumulated by mycobacteria but its degradation is limited to non-pathogenic fast-growing mycobacteria. *Can. J. Microbiol.* 46, 826–831

22 Yang, X. *et al.* (2009) Cholesterol metabolism increases the metabolic pool of propionate in *Mycobacterium tuberculosis*. *Biochemistry* 48, 3819–3821

23 Ouellet, H. *et al.* (2010) *Mycobacterium tuberculosis* CYP125A1, a steroid C27 monooxygenase that detoxifies intracellularly generated cholest-4-en-3-one. *Mol. Microbiol.* 77, 730–742

24 Chang, J.C. *et al.* (2007) Identification of mycobacterial genes that alter growth and pathology in macrophages and in mice. *J. Infect. Dis.* 196, 788–795

25 Chang, J.C. *et al.* (2009) *igr* genes and *Mycobacterium tuberculosis* cholesterol metabolism. *J. Bacteriol.* 191, 5232–5239

26 Kendall, S.L. *et al.* (2007) A highly conserved transcriptional repressor controls a large regulon involved in lipid degradation in *Mycobacterium smegmatis* and *Mycobacterium tuberculosis*. *Mol. Microbiol.* 65, 684–699

27 Nesbitt, N.M. *et al.* (2010) A thiolase of *Mycobacterium tuberculosis* is required for virulence and production of androstenedione and androstadienedione from cholesterol. *Infect. Immun.* 78, 275–282

28 Kendall, S.L. *et al.* (2010) Cholesterol utilization in mycobacteria is controlled by two TetR-type transcriptional regulators: *kstR* and *kstR2*. *Microbiology* 156, 1362–1371

29 Schnappinger, D. *et al.* (2003) Transcriptional adaptation of *Mycobacterium tuberculosis* within macrophages: insights into the phagosomal environment. *J. Exp. Med.* 198, 693–704

30 Sassetti, C.M. and Rubin, E.J. (2003) Genetic requirements for mycobacterial survival during infection. *Proc. Natl. Acad. Sci. U.S.A.* 100, 12989–12994

31 Rengarajan, J. *et al.* (2005) Genome-wide requirements for *Mycobacterium tuberculosis* adaptation and survival in macrophages. *Proc. Natl. Acad. Sci. U.S.A.* 102, 8327–8332

32 Rosloniec, K.Z. *et al.* (2009) Cytochrome P450 125 (CYP125) catalyses C26-hydroxylation to initiate sterol side-chain degradation in *Rhodococcus jostii* RHA1. *Mol. Microbiol.* 74, 1031–1043

33 Wilbrink, M.H. *et al.* (2011) FadD19 of *Rhodococcus rhodochrous* DSM43269, a steroid-coenzyme A ligase essential for degradation of C-24 branched sterol side chains. *Appl. Environ. Microbiol.* 77, 4455–4464

34 Yang, X. *et al.* (2007) Rv1106c from *Mycobacterium tuberculosis* is a 3beta-hydroxysteroid dehydrogenase. *Biochemistry* 46, 9058–9067

35 Brzostek, A. *et al.* (2007) Cholesterol oxidase is required for virulence of *Mycobacterium tuberculosis*. *FEMS Microbiol. Lett.* 275, 106–112

36 Yang, X. *et al.* (2011) Cholesterol is not an essential source of nutrition for *Mycobacterium tuberculosis* during infection. *J. Bacteriol.* 193, 1473–1476

37 Tabas, I. *et al.* (1988) Acyl coenzyme A:cholesterol acyl transferase in macrophages utilizes a cellular pool of cholesterol oxidase-accessible cholesterol as substrate. *J. Biol. Chem.* 263, 1266–1272

38 Uhia, I. *et al.* (2011) Initial step in the catabolism of cholesterol by *Mycobacterium smegmatis* mc(2)155. *Environ. Microbiol.* 4, 943–959

39 Itagaki, E. *et al.* (1990) Spectral properties of 3-ketosteroid-delta 1-dehydrogenase from *Nocardia corallina*. *Biochim. Biophys. Acta* 1040, 281–286

40 Itagaki, E. *et al.* (1990) Purification and characterization of 3-ketosteroid-delta 1-dehydrogenase from *Nocardia corallina*. *Biochim. Biophys. Acta* 1038, 60–67

41 Knol, J. *et al.* (2008) 3-Keto-5alpha-steroid Delta(1)-dehydrogenase from *Rhodococcus erythropolis* SQ1 and its orthologue in *Mycobacterium tuberculosis* H37Rv are highly specific enzymes that function in cholesterol catabolism. *Biochem. J.* 410, 339–346

42 Brzostek, A. *et al.* (2009) *Mycobacterium tuberculosis* is able to accumulate and utilize cholesterol. *J. Bacteriol.* 191, 6584–6591

43 van der Geize, R. *et al.* (2002) Molecular and functional characterization of the *kstD2* gene of *Rhodococcus erythropolis* SQ1 encoding a second 3-ketosteroid Delta(1)-dehydrogenase isoenzyme. *Microbiology* 148, 3285–3292

44 van der Geize, R. *et al.* (2001) Unmarked gene deletion mutagenesis of *kstD*, encoding 3-ketosteroid Delta1-dehydrogenase, in *Rhodococcus erythropolis* SQ1 using *sacB* as counter-selectable marker. *FEMS Microbiol. Lett.* 205, 197–202

45 Capyk, J.K. *et al.* (2009) Characterization of 3-ketosteroid 9-alpha-hydroxylase, a Rieske oxygenase in the cholesterol degradation pathway of *Mycobacterium tuberculosis*. *J. Biol. Chem.* 284, 9937–9946

46 Hu, Y. *et al.* (2010) 3-Ketosteroid 9alpha-hydroxylase is an essential factor in the pathogenesis of *Mycobacterium tuberculosis*. *Mol. Microbiol.* 75, 107–121

47 van der Geize, R. *et al.* (2008) Characterization of a second *Rhodococcus erythropolis* SQ1 3-ketosteroid 9alpha-hydroxylase activity comprising a terminal oxygenase homologue, KshA2, active with oxygenase-reductase component KshB. *Appl. Environ. Microbiol.* 74, 7197–7203

48 Andor, A. *et al.* (2006) Generation of useful insertionally blocked steroid degradation mutants of fast-growing mycobacteria and cloning, characterization, and expression of the terminal oxygenase of the 3-ketosteroid 9 alpha-hydroxylase in *Mycobacterium smegmatis* mc(2)155. *Appl. Environ. Microbiol.* 72, 6554–6559

49 Dresen, C. *et al.* (2010) A flavin-dependent monooxygenase from *Mycobacterium tuberculosis* involved in cholesterol catabolism. *J. Biol. Chem.* 285, 22264–22275

50 Lack, N. *et al.* (2008) Structure of HsaD, a steroid-degrading hydrolase, from *Mycobacterium tuberculosis*. *Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.* 64, 2–7

51 Lack, N.A. *et al.* (2010) Characterization of a carbon-carbon hydrolase from *Mycobacterium tuberculosis* involved in cholesterol metabolism. *J. Biol. Chem.* 285, 434–443

52 Kunau, W.H. *et al.* (1995) Beta-oxidation of fatty acids in mitochondria, peroxisomes, and bacteria: a century of continued progress. *Prog. Lipid Res.* 34, 267–342

53 Johnston, J.B. *et al.* (2011) Structural control of cytochrome P450-catalyzed omega-hydroxylation. *Arch. Biochem. Biophys.* 507, 86–94

54 Johnston, J.B. *et al.* (2010) Functional redundancy of steroid C26-monooxygenase activity in *Mycobacterium tuberculosis* revealed by biochemical and genetic analyses. *J. Biol. Chem.* 285, 36352–36360

55 McLean, K.J. *et al.* (2009) The structure of *Mycobacterium tuberculosis* CYP125: molecular basis for cholesterol binding in a P450 needed for host infection. *J. Biol. Chem.* 284, 35524–35533

56 Capyk, J.K. *et al.* (2009) Mycobacterial cytochrome p450 125 (*cyp125*) catalyzes the terminal hydroxylation of C27 steroids. *J. Biol. Chem.* 284, 35534–35542

57 Yam, K.C. *et al.* (2009) Studies of a ring-cleaving dioxygenase illuminate the role of cholesterol metabolism in the pathogenesis of *Mycobacterium tuberculosis*. *PLoS Pathog.* 5, e1000344

58 Xu, X. and London, E. (2000) The effect of sterol structure on membrane lipid domains reveals how cholesterol can induce lipid domain formation. *Biochemistry* 39, 843–849

59 Beattie, M.E. *et al.* (2005) Sterol structure determines miscibility versus melting transitions in lipid vesicles. *Biophys. J.* 89, 1760–1768

60 Ahmad, Z. *et al.* (2006) The potential of azole antifungals against latent/persistent tuberculosis. *FEMS Microbiol. Lett.* 258, 200–203

61 Ahmad, Z. *et al.* (2006) Antimycobacterial activity of econazole against multidrug-resistant strains of *Mycobacterium tuberculosis*. *Int. J. Antimicrob. Agents* 28, 543–544

62 Driscoll, M.D. *et al.* (2011) Expression and characterization of Mycobacterium tuberculosis CYP144: common themes and lessons learned in the Mycobacterium tuberculosis P450 enzyme family. *Biochim. Biophys. Acta* 1814, 76–87

63 Driscoll, M.D. *et al.* (2010) Structural and biochemical characterization of Mycobacterium tuberculosis CYP142: evidence for multiple cholesterol 27-hydroxylase activities in a human pathogen. *J. Biol. Chem.* 285, 38270–38282

64 Johnston, J.B. *et al.* (2009) Biochemical and structural characterization of CYP124: a methyl-branched lipid omega- hydroxylase from Mycobacterium tuberculosis. *Proc. Natl. Acad. Sci. U.S.A.* 106, 20687–20692

65 McLean, K.J. *et al.* (2002) Expression, purification and spectroscopic characterization of the cytochrome P450 CYP121 from Mycobacterium tuberculosis. *J. Inorg. Biochem.* 91, 527–541

66 McLean, K.J. *et al.* (2002) Azole antifungals are potent inhibitors of cytochrome P450 mono-oxygenases and bacterial growth in mycobacteria and streptomycetes. *Microbiology* 148, 2937–2949

67 Ouellet, H. *et al.* (2008) Mycobacterium tuberculosis CYP130: crystal structure, biophysical characterization, and interactions with antifungal azole drugs. *J. Biol. Chem.* 283, 5069–5080

68 Ouellet, H. *et al.* (2011) Reverse type I inhibitor of Mycobacterium tuberculosis CYP125A1. *Bioorg. Med. Chem. Lett.* 21, 332–337

69 Thomas, S.T. *et al.* (2011) Inhibition of the Mycobacterium tuberculosis 3beta-hydroxysteroid dehydrogenase by azasteroids. *Bioorg. Med. Chem. Lett.* 21, 2216–2219

70 Frye, S.V. *et al.* (1993) 6-Azasteroids: potent dual inhibitors of human type 1 and 2 steroid 5 alpha-reductase. *J. Med. Chem.* 36, 4313–4315

71 Frye, S.V. *et al.* (1994) 6-Azasteroids: structure-activity relationships for inhibition of type 1 and 2 human 5 alpha-reductase and human adrenal 3 beta-hydroxy-delta 5-steroid dehydrogenase/3-keto-delta 5-steroid isomerase. *J. Med. Chem.* 37, 2352–2360

72 McLean, K.J. *et al.* (2008) Characterization of active site structure in CYP121. A cytochrome P450 essential for viability of Mycobacterium tuberculosis H37Rv. *J. Biol. Chem.* 283, 33406–33416
