
Rajagopala et al. BMC Microbiology 2011, 11:213  
http://www.biomedcentral.com/1471-2180/11/213  

---

**RESEARCH ARTICLE**  
**Open Access**

# The protein interaction map of bacteriophage lambda

Seesandra V Rajagopala¹*, Sherwood Casjens² and Peter Uetz³,⁴*

---

## Abstract

### Background:
Bacteriophage lambda is a model phage for most other dsDNA phages and has been studied for over 60 years. Although it is probably the best-characterized phage there are still about 20 poorly understood open reading frames in its 48-kb genome. For a complete understanding we need to know all interactions among its proteins. We have manually curated the lambda literature and compiled a total of 33 interactions that have been found among lambda proteins. We set out to find out how many protein-protein interactions remain to be found in this phage.

### Results:
In order to map lambda's interactions, we have cloned 68 out of 73 lambda open reading frames (the "ORFeome") into Gateway vectors and systematically tested all proteins for interactions using exhaustive array-based yeast two-hybrid screens. These screens identified 97 interactions. We found 16 out of 30 previously published interactions (53%). We have also found at least 18 new plausible interactions among functionally related proteins. All previously found and new interactions are combined into structural and network models of phage lambda.

### Conclusions:
Phage lambda serves as a benchmark for future studies of protein interactions among phage, viruses in general, or large protein assemblies. We conclude that we could not find all the known interactions because they require chaperones, post-translational modifications, or multiple proteins for their interactions. The lambda protein network connects 12 proteins of unknown function with well characterized proteins, which should shed light on the functional associations of these uncharacterized proteins.

---

## Background

Sixty years ago, in 1951, Esther Lederberg discovered phage lambda [1]. Since this seminal discovery lambda has become a model organism in which many foundational studies lead to our current understanding of how genes work and how they are regulated, as well as how proteins perform such functions as DNA replication, homologous and site-specific recombination, and virion assembly. In addition, tailed phages are the most abundant life form on earth [2], and so deserve to be studied in their own right and in the context of global ecology. Nevertheless, phage lambda is not completely understood. There are still a number of genes in its 48.5 kb genome whose function remains only vaguely defined, if at all. For instance, many of the genes in the b2 and nin regions have no known function (Figure 1). And 14 of the 73 predicted lambda proteins have unknown functions.

Two of the best-characterized aspects of lambda biology are the genetic switch that determines whether a phage reproduces and lyses the cell or whether it integrates into its host genomes to become a prophage [3,4] and the mechanisms through which transcription anti-termination controls its gene expression cascade. Nevertheless, lambda continues to yield new insights into its gene regulatory circuits [4,5], and recent studies of its DNA packaging motor are in the vanguard of nanomotor research [6].

Surprisingly, even the structure of the lambda virion is incompletely known: the structures of only 5 of the ~14 proteins in the virus particle have been solved, and it is unknown whether several proteins that are required for tail assembly are in the completed virion, even though

* Correspondence: raja@jcvi.org; peter@uetz.us  
¹ J Craig Venter Institute, Rockville, MD 20850, USA  
² Center for the Study of Biological Complexity, Virginia Commonwealth University, PO Box 842030, 1015 Floyd Ave., Richmond, VA 23284, USA  
Full list of author information is available at the end of the article

© 2011 Rajagopala et al; licensee BioMed Central Ltd. This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

**A**
Head Tail Tail tip Tail fibers b2 region
terminase W Nu3 D FII V tape measure L J central tail fiber stf Ea31
cos A B C E Z T H M K I lom tfa Ea47
Nu1
0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25

Site-spec. general
b2 region recombin. Ea region recombin.
DNA
replication nin region lysis
Ea31 Ea59 att XIS Ea22 exo bet kil clll ral sieB rexB rexA cro P L ren Q Rz1 cos
sib int P1 gam Earθ N cl P RE O P 146 290 204 221 S R Rz bor
genes?
Ea47
24 25 26 27 28 29 30 31 32 33 34 35 36 37 39 40 41 42 43 44 45 46 47 48 kbp
Early left operon Early right operon late operon

**B** C (protease, ~10) **C**
D (405)
E (405)
DNA (1)
W (~6) B' (portal, 12) head
FII (~6) U (~6) tail
V (32x6=192)
tfa (12)
stf (12)
M ?
J (~3) H (~6)

Figure 1 The Lambda genome and virion. (A) Genome of phage lambda. Colored ORFs correspond to colored proteins in (B). Main transcripts are shown as arrows. (B) A model of phage lambda, indicating protein-protein interactions. Proteins in bold font have known structures (Table 1). Numbers indicate the number of protein copies in the particle. It is unclear whether M and L proteins are in the final particle or only required for assembly. (C) Electron micrograph of phage lambda. (A) and (C) modified after [24].

the overall structure is well known from electron microscopy [7].

Key to the understanding of lambda biology is a detailed understanding of protein function, including their interactions. We have curated more than 30 protein-protein interactions (PPIs) from the literature, identified over the past 60 years. Such interactions are reasonably well known within the virus particle and during the life cycle of lambda, i.e. during replication and recombination. However, the molecular details of virion assembly, obviously highly dependent on coordinated interactions of structural and accessory proteins, are still largely mysterious.

The structures of at least 17 lambda proteins have been solved (Table 1). In addition, the lambda head has been studied in some detail by cryo-electron microscopy, X-ray crystallography, and NMR (Figure 1). The tail is much less well known. While we do have

structures of the head-tail junction proteins W, FII, and U individually, their connections to the head via the portal protein (B) and to each other are not very clear. Similarly, while we do have a structure of the major tail tube protein V, the remaining tail is structurally largely uncharacterized.

Our motivation for this study was three-fold: first, in our continuous attempts to improve the yeast two-hybrid system further, we thought that phage lambda would be an excellent “gold-standard” to benchmark our experimental system by demonstrating how many previously known interactions (Table 2) we are able to identify in such a well-studied system. Second, we believe that interaction data can help to solve the structures of protein complexes, since binary interactions as described here may facilitate the crystallization of co-complexes. Despite its well-understood biology, phage lambda is not well understood structurally; especially

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

Table 1 Lambda proteins of known structure

| Protein | PDB | reference |
|---------|-----|-----------|
| CI      | 3BDN | [77]      |
| CII     | 1ZS4, 1XWR | [78,79] |
| Cro     | 2ECS, 2OVG, 2A63 | [80,81] |
| D       | 1VD0, 1C5E, 1TCZ | [50,82,83] |
| Exo     | 1AVQ | [84]      |
| FII     | 2KX4, 1K0H | [85,86] |
| Gam     | 2UUZ, 2UV1 | [87]      |
| Int     | 2WCC, 1P7D, 1Z19, 1Z1B, 1Z1G | [88-90] |
| N       | 1QFQ | [91]      |
| NinB    | 1PC6 | [26]      |
| Nu1     | 1J9I | [33]      |
| R       | 3D3D | [92]      |
| Nini*   | 1G5B | [93]      |
| U       | 3FZ2, 3FZB, 1Z1Z | [19,94] |
| V       | 2L04, 2K4Q | [94-96] |
| W       | 1HYW | [39]      |
| Xis     | 2OG0, 2IEF, 1RH6, 1LX8 | [69,97-99] |

*Ser/Thr protein phosphatase

the assembly of its tail remains poorly understood. Third, and possibly most important, we wondered if we could contribute to the understanding of lambda biology, either by discovering new interactions or by verifying questionable or poorly supported interactions.

To achieve these goals, we cloned almost all lambda open reading frames (ORFs) and tested them for all pair-wise interactions, using a novel yeast two-hybrid strategy [8]. We identified a total of 97 unique interactions, most of which have not been previously described. About half of all published interactions were identified, and we will discuss why the other half has been missed and how these interactions might be detected by future two-hybrid studies.

### Results

#### Approach
In order to find as many interactions as possible, we cloned 68 lambda ORFs into six different Y2H vectors (see Table 3 and Methods). In fact, each vector pair results in very different subsets of interactions as we have shown previously [8-10]. For example, the pGADT7g/pGBK7g vectors yielded 44 interactions while the pGBKCg/pGADCg vectors yielded only 18. The main difference between these two pairs is the way the fusion proteins are constructed: in the former two vectors the Gal4 DNA-binding (DBD) and activation domains (AD) are fused to the N-terminus of the lambda proteins (Figure 2). In the latter two the DBD and AD are fused to the C-terminus of the lambda proteins. It is thus reasonable to assume that structural constraints cause many of the observed differences.

---

Assay sensitivity and false positives
As we have observed before in other contexts [10], the pGADT7g/pGBK7g vectors yielded almost half of all interactions discovered in this study and almost three times as many as the pDEST series of vectors (which uses similar N-terminal fusions). The pDEST system may detect fewer interactions but they probably also detect fewer false positives (see discussion).

In a previous study we benchmarked the false positive rate for each Y2H vector systems under different screening (stringency) conditions [9]. To evaluate the accuracy of the vector system we used specificity estimates from this study [9] (i.e., the experimental proportions of negative interactions among negative reference interactions). The sensitivity was estimated using known lambda interactions (i.e., the experimental proportion of positive interactions among positive reference interactions). Specificity ranged from most specific, namely 98.9% for GADT7g/pGBK7g and pGBK7g/pGADCg to 95.7% for pGBKcg/pGADT7g (least specific). Sensitivity ranged from 33.3% for pGBK7g/pGADCg to 17% for pGBKcg/pGADCg and pDEST22/32. For each method, we estimated the probability of being a true interaction using Bayes theorem: pDEST22/32 (83.3%), pGADT7g/pGBK7g (80.0%), pGBK7g/pGADCg and pGBKcg/pGADCg (71.4%), and pGBKcg/pGADT7g (40.0%) (Figure 2C).

### Verification and quality scores
If an interaction is found in more than one vector combination, the reliability is higher than when it is found in only one. Twenty-four interactions (out of 97) were found in 2 or more vector combinations (Table 4). This number of combinations can be used as a score, and the 3 interactions with the highest score have all been described in the literature before. Of the 24 high-scoring interactions, six (25%) have been described before (Figure 2D). To test if the difference of the proportions of detected literature interactions is greater for the more than one vector combination group, we carried out a one-sided test for difference of proportions. The null hypothesis can be rejected for alpha = 0.1 indicating a moderately significant difference (P-Value = 0.098) (Additional file 1: Table S6). We conclude that the number of supporting vector combinations can be used as a confidence score. This suggests that the 18 novel high-scoring interactions are possibly physiologically relevant interactions and thus good candidates for further studies (see discussion).

Of the 73 interactions that were found in only one combination, 10 have been published previously, demonstrating that they are useful too. In fact, 16 out of 30 previously found interactions were also found in our screen, i.e. 53%. Note that three previously found interactions (Xis-Xis, Xis-Int, and SieB-Esc) could not be

Table 2 Previously published interactions among lambda proteins

| interacting λ proteins | notes | ref# |
|------------------------|-------|------|
| head                  |       |      |
| 1                     | A     | Nu1   | A(N-term) - Nu1(C-term) | [32-34] |
| 2                     | A     | B     | A(C-term) - B(=portal) | [32,35] |
| 3                     | A     | FI    | Genetic evidence | [21] |
| 4                     | FI    | E     | Genetic evidence | [22] |
| 5                     | Nu3   | B     | Nu3 required for B incorporation into procapsid | [36] |
| 6                     | W     | B     | W required for FII binding, FII connects head to tail | [37,38] |
| 7                     | W     | FII   | W required for FII binding, FII connects head to tail | [37,39] |
| 8                     | B     | B     | 12-mer(22aa removed from BN-term) | [40,41] |
| 9                     | C     | E     | Covalent PPI(in virion?) | [42,43] |
| 10                    | C     | B     | copurify in procapsid | [44] |
| 11                    | B     | E     | copurify in procapsid | [45] |
| 12                    | C     | Nu3   | C may degrade Nu3 (before DNA packaging) | [45-47] |
| 13                    | D     | D     | Capsid vertices, D forms trimers | [48-50] |
| 14                    | E     | E     | Main capsid protein | [20,51,52] |
| 15                    | D     | E     | Nu3 multimer | [20,51,52] |
|                       | Nu3   | Nu3   | Nu3 multimer | unpublished* |

| tail                  |       |      |
|-----------------------|-------|------|
| 16                    | U     | U     | "probably a hexamer", interact in crystal | [53] |
| 17                    | V     | V     | the T domain binds soluble V | [51,54-56] |
| 18                    | V     | GT    | the T domain binds soluble V | [24] |
| 19                    | H     | G/GT  | G/GTholdHin an extended fashion | [24] |
| 20                    | H     | V     | V probably assembles around H, displacing G/GT | [57] |

| replication           |       |      |
|-----------------------|-------|------|
| 21                    | O     | O     | O-O interactions when bound to oriDNA | [58] |
| 22                    | O     | P     | O-O interactions when bound to oriDNA | [59-62] |

| transcription         |       |      |
|-----------------------|-------|------|
| 23                    | CI    | CI    | Forms octamer that links OR to OL | [63,64] |
| 24                    | CII   | CII   | homotetramers | [65] |
| 25                    | CIII  | CIII  | dimer | [66] |
| 26                    | Cro   | Cro   | dimer; x-ray structure | [67] |

| Recombination         |       |      |
|-----------------------|-------|------|
| 27                    | Exo   | Bet   | [68] |
| 28                    | Xis   | Int   | [69]# |
| 29                    | Xis   | Xis   | Xis-Xis binding mediates cooperative DNA-binding | [69]# |
| 30                    | Int   | Int   | Dimer | [70] |

| lysis                 |       |      |
|-----------------------|-------|------|
| 31                    | Rz    | Rz1   | heteromultimer that is supposed to span the periplasm | [71] |
| 32                    | S     | S     | large ring in inner membrane | [72] |
|                       | S     | S'    | S'inhibits S ring formation(S:105aa, S':107aa) | [73] |

| lysogenic conversion  |       |      |
|-----------------------|-------|------|
| 33                    | SieB  | Esc   | Esc is encoded in frame in sieB+inhibits sieB | [74,75]# |

bold: found in this study.*unpublished (C.Catalano, pers.comm., by permission), #interactions not tested in Y2H assays(one or both clones not available).

tested since we were unable to obtain ORF clones of J,
Xis, NinH, and Esc (which is encoded within SieB).
Prey counts
There are other criteria that can be used to score inter-
actions.One of them is the number of times a prey
protein is found.This “prey count” indicates whether a
protein interacts very specifically (low prey count) or
more unspecifically and thus promiscuously.Proteins
with high prey counts are more likely false positives,
and hence we removed these interactions with prey
count > 5 from further analysis (see Additional file 1:

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

Table 3 Vectors and interaction summary

| Vector pair(s)          | Fusions proteins                          | Interactions* |
|-------------------------|------------------------------------------|---------------|
| pDEST22/pDEST32         | N/N (N-terminal fusions)                 | 8             |
| pGADT7g/pGBK7g          | N/N (N-terminal fusions)                 | 44            |
| pGBK7g/pGADCg           | N/C (N-terminal/C-terminal fusions)      | 39            |
| pGBKcg/pGADCg           | C/C (C-terminal/C-terminal fusions)      | 18            |
| pGBKcg/pGADT7g          | C/N (C-terminal/N-terminal fusions)      | 26            |

* Redundant, i.e., some interactions are found with multiple vectors.
Fusion proteins indicate the location of the DNA-binding (DBD) and activation domains (AD), respectively, of each vector pair. For instance, the pDEST vectors both have the DBD and AD fused at the N-terminus of the bait and prey protein. Vectors are listed as bait/prey pairs.

Tables S2 and S3). However, this was not generally true in our study: of the preys that were found 1 to 3 times, 12 were found among the “gold-standard” literature interactions. Of the preys that were found 4 to 5 times,

A Bait: pGBK7g-A; preys in pGADT7g+pGADCg*

|   | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 |
|---|---|---|---|---|---|---|---|---|---|----|----|----|
| A |   |   | A |   |   |   |   |   | FI* |    |    |    |
| B |   | Nu1* |   |   |   |   |   |   |    |    |    |    |
| C |   | V |   |   |   |   |   |   |    |    |    |    |
| D |   |   |   |   |   |   |   |   |    |    |    |    |
| E |   |   |   |   |   |   |   |   |    |    |    |    |
| F |   |   | Orf48 |   |   |   |   |   |    |    |    |    |
| G |   |   |   |   |   |   |   |   |    |    |    |    |
| H |   |   | NinF |   |   |   |   |   |    |    |    |    |
| I |   |   |   |   |   |   |   |   |    |    |    |    |
| J |   |   |   |   |   |   |   |   |    |    |    |    |
| K |   | Ea47 |   |   | Bet |   |   |   |    |    |    |    |
| L |   |   |   |   |   |   |   |   |    |    |    |    |
| M |   |   |   |   |   |   |   |   |    |    |    |    |
| N |   |   |   |   |   |   |   |   |    |    |    |    |

B Bait: pGBKcg-A; preys in pGADT7g+pGADCg*

|   | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 |
|---|---|---|---|---|---|---|---|---|---|----|----|----|
| A |   |   |   |   |   |   |   |   |    |    |    |    |
| B |   | Nu1* |   |   |   |   |   |   |    |    |    |    |
| C |   | V |   |   |   |   |   |   |    |    |    |    |
| D |   |   |   |   |   |   |   |   |    |    |    |    |
| E |   |   |   |   |   |   |   |   |    |    |    |    |
| F |   |   |   |   |   |   |   |   |    |    |    |    |
| G |   |   |   |   |   |   |   |   |    |    |    |    |
| H |   |   |   |   |   |   |   |   |    |    |    |    |
| I |   |   |   |   |   |   |   |   |    |    |    |    |
| J |   |   |   |   |   |   |   |   |    |    |    |    |
| K |   |   |   |   |   |   |   |   |    |    |    |    |
| L |   |   |   |   |   |   |   |   |    |    |    |    |
| M |   |   |   |   |   |   |   |   |    |    |    |    |
| N |   |   |   |   |   |   |   |   |    |    |    |    |

Figure 2 Yeast two-hybrid array screens and vectors. Shown are two Y2H screens with four different vector combinations. Each interaction is represented by two colonies to ensure reproducibility. (A) Lambda bait protein A (DNA packaging protein) was fused to an N-terminal DNA-binding domain ("DBD", in pGBK7g) and was tested against prey constructs in both N- and C-terminal configurations (activation domains in pGADT7g, and pGADCg). (B) The C-terminal DBD fusion (in pGBKcg) as tested against prey constructs in both N- and C-terminal configurations (in pGADT7g, and pGADCg). The interactions of C-terminal preys are labeled with an asterisk (*), all remaining interactions use N-terminal fusions. All the interactions obtained from the array screening were subjected to Y2H retests: we were able to retest all the interactions shown in Figure 2 except A-Ea47, which has thus been removed from the final interaction list. Technical details of the screening procedure have been described in [8, 10]. (C) Interaction quality assessment. Using the experimental derived false positive rate from [9] and Bayes theorem, we estimated the probability of an interaction to be true. This estimate depends on the vector system, being highest (83%) for pDEST22/32, and lowest (40%) for pGBKcg/pGADT7g. (D) Detection of known PPIs with different vector systems. Known PPIs are enriched in the subset of PPIs detected by >= 2 vector systems compared to PPIs detected by 1 vector combination.

as the poorly understood proteins NinG, NinH, Orf 221 (NinI), Orf 290 (NinC), and SieB (see discussion).

Functional specificity

We grouped all lambda proteins in 9 groups, namely virion head, virion tail, transcription, replication, recombination, lysis, lysogenic conversion, others with known function, and unknown (Table 4). A statistical analysis of interactions shows that proteins involved in head assembly have the highest specificity (Figure 4): when interactions among different functional classes are considered, the proteins involved in capsid assembly tend to interact with themselves more frequently compared to other functional classes. Interestingly, the proteins of unknown function show interactions with proteins involved in several functional classes, including tail assembly, transcription and recombination (Figure 4).

Overall, the 97 protein-protein interactions (PPIs) of our screens correspond to ~4.2% of the lambda search space (=97/68*0.5), i.e. all possible protein pairs of the lambda proteome (here: 68*68). This is significantly less than we found in Streptococcus phage Dp1, namely 156 interactions among 72 ORFs [11] even though in the latter case only 2 vector pairs were used. A possible explanation is that we used a more rigorous retesting scheme herein which only interactions were counted that were found in multiple rounds of retesting.

Discussion

Lambda protein interaction network

This is only the second study that has applied multiple two-hybrid vector systems to characterize the protein-protein interactions at a genome scale, the first being our analysis of the Varicella Zoster Virus [8]. The lambda protein network connects 12 proteins of unknown function with well characterized proteins, which should shed light on the functional associations of these uncharacterized proteins (Figure 3). For example, NinI interacts with two proteins N and Q which are involved in transcription antitermination. The scaffolding protein Nu3 forms dimers, and interacts with the tail proteins Z and M as well as the capsid protein E. Thus, Nu3 may play an accessory role in the assembly of both head and tail, even though Nu3 is not absolutely required for tail assembly.

False negatives

This study discovered more than 53% of all published interactions among lambda proteins. However, it failed to discover the remaining 47%. We can only speculate why this is the case. Some of the early steps in virion assembly depend on chaperones [12]. For instance, the portal protein B requires GroES/EL, most likely for folding [13]. These chaperones are not present in the yeast

Table 4 All PPIs discovered in this study

| Bait | Prey | B fun | P fun | NN | NC | CC | CN | Vectors | Notes |
|------|------|-------|-------|----|----|----|----|---------|-------|
| 1.    | A    | head  | head  | NC |    | CC | CN | 3       | Possible |
| 2.    | A    | Bet   | head  | rec| G  |    |    | 1       |        |
| 3.    | A    | Fl    | head  | head |    | NC | CC'CN' | 3       | Known |
| 4.    | A    | NinF  | head  | ukn | G  |    |    | 1       |        |
| 5.    | A    | Nu1   | head  | head | G' | NC'CC |    | 3       | Known |
| 6.    | A    | Orf79 | head  | unk | G  |    |    | 1       |        |
| 7.    | A    | V     | head  | tail | G  |    |    | 1       |        |
| 8.    | Cl   | Cl    | trx   | trx  |    | CC |    | 1       | Known |
| 9.    | Cl   | Kil   | trx   | other |    | CC |    | 1       |        |
| 10.   | Cl   | Cl    | trx   | trx  |    | NC |    | 1       | known |
| 11.   | C    | C     | head  | head |    | NC |    | 1       |        |
| 12.   | C    | Nu3   | head  | head | G' | NC' |    | 2       | Known |
| 13.   | C    | Orf79 | head  | unk | G  |    |    | 1       |        |
| 14.   | D    | D     | head  | head |    | NC |    | 1       | Known |
| 15.   | D    | E     | head  | head | D  |    |    | 1       | Known |
| 16.   | E    | E     | head  | head | D  |    |    | 1       | Known |
| 17.   | E    | Fi    | head  | head | G  | NC | CC'CN' | 4       | Known |
| 18.   | E    | Nu3   | head  | head | DG' |    |    | 2       | 2v     |
| 19.   | Ea8.5 | Ea8.5 | ihr   | unk  |    | NC |    | 1       | Possible |
| 20.   | Ea8.5 | Int   | ihr   | rec  | G  | NC |    | 2       | 2v     |
| 21.   | Ea8.5 | Tfa   | ihr   | tail | G  |    |    | 1       |        |
| 22.   | Ea8.5 | Stf   | ihr   | tail | G  |    | CN | 2       |        |
| 23.   | Ea8.5 | Q     | ihr   | trx  | G  |    |    | 1       |        |
| 24.   | Ea8.5 | Ren   | ihr   | unk  |    | NC |    | 1       |        |
| 25.   | Fl    | NinB  | head  | rec  |    | CN |    | 1       |        |
| 26.   | G     | G     | tail  | tail | G  |    | CC CN | 3       | Possible |
| 27.   | G     | H     | tail  | tail | D' |    |    | 1       | Known |
| 28.   | G     | S'    | tail  | lysis | G  |    | CN | 2       | 2v     |
| 29.   | G     | T     | tail  | tail |    | CC |    | 1       | Likely |
| 30.   | H     | Cl    | tail  | trx  |    | NC |    | 1       |        |
| 31.   | H     | Ren   | tail  | unk  |    | NC |    | 1       |        |
| 32.   | H     | V     | tail  | tail |    | NC |    | 1       | Known |
| 33.   | Int   | A     | rec   | head | G  |    |    | 1       |        |
| 34.   | Int   | Bet   | rec   | rec  | G  |    |    | 1       | Possible |
| 35.   | Int   | Int   | rec   | rec  | G  | NC |    | 2       | known |
| 36.   | Int   | Orf48 | rec   | unk  | G  |    |    | 1       |        |
| 37.   | Int   | Tfa   | rec   | tail |    | CN |    | 1       |        |
| 38.   | Int   | V     | rec   | tail | G  |    |    | 1       |        |
| 39.   | M     | Fi    | tail  | head |    | CC'CN' | 2       | 2v     |
| 40.   | M     | G     | tail  | tail | G  |    | CC CN | 3       | Possible |
| 41.   | M     | NinF  | tail  | unk  | G  |    | CN | 2       | 2v     |
| 42.   | M     | Nu3   | tail  | head |    | CN |    | 1       |        |
| 43.   | M     | Orf35 | tail  | unk  |    | NC CC | 2       | 2v     |
| 44.   | N     | Bet   | trx   | rec  | G  |    |    | 1       |        |
| 45.   | N     | Ea47  | trx   | unk  | G  |    |    | 1       |        |
| 46.   | N     | L     | trx   | tail | G  |    |    | 1       |        |
| 47.   | N     | Nu1   | trx   | head |    | NC |    | 1       |        |
| 48.   | N     | V     | trx   | tail | G  |    |    | 1       |        |
| 49.   | NinD  | Cro   | unk   | trx  | G  |    |    | 1       |        |
| 50.   | NinD  | K     | unk   | tail | G  | NC |    | 2       | 2v     |
| 51.   | NinD  | Q     | unk   | trx  | G  |    |    | 1       |        |
| 52.   | Ninl  | N     | unk   | trx  | G  |    |    | 1       |        |

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

Table 4 All PPIs discovered in this study (Continued)

| 53. | Ninl | Q | unk | trx | G | 1 |
| --- | --- | --- | --- | --- | --- | --- |
| 54. | Nu1 | Nu1 | head | head | NC CC | 2 | 2v |
| 55. | Nu1 | Tfa | head | tail | G | 1 |
| 56. | Nu1 | Orf64 | head | unk | CC | 1 |
| 57. | Nu1 | R | head | lysis | D | 1 |
| 58. | Nu1 | V | head | tail | G | 1 |
| 59. | Nu3 | Nu3 | head | head | G | 1 |
| 60. | Nu3 | Z | head | tail | G | 1 |
| 61. | O | P | repl | repl | D | 1 | Known |
| 62. | Orf35 | Cl | unk | trx | NC | 1 |
| 63. | Orf35 | Int | unk | rec | G NC | 2 | 2v |
| 64. | Orf35 | K | unk | tail | G NC | 2 | 2v |
| 65. | Orf35 | Orf78 | unk | unk | NC | 1 |
| 66. | Orf35 | Ren | unk | unk | NC | 1 |
| 67. | Orf48 | Orf48 | unk | unk | NC | 1 | Possible |
| 68. | Orf79 | Orf79 | unk | unk | CC CN | 2 | Possible |
| 69. | Orf63 | N | rec | trx | G | 1 |
| 70. | Orf63 | Orf78 | rec | unk | NC | 1 |
| 71. | Orf63 | P | rec | repl | NC | 1 |
| 72. | Orf63 | Q | rec | trx | G | 1 |
| 73. | Orf63 | Ren | rec | unk | NC | 1 |
| 74. | Orf63 | Rz1 | rec | lysis | G | 1 |
| 75. | P | Bet | repl | rec | G | 1 |
| 76. | P | Q | repl | trx | G | 1 |
| 77. | RexB | A | conv | head | NC | 1 |
| 78. | RexB | Orf48 | conv | unk | NC | 1 |
| 79. | RexB | Orf78 | conv | unk | NC | 1 |
| 80. | RexB | Ren | conv | unk | NC | 1 |
| 81. | S' | S' | lysis | lysis | G | 1 |
| 82. | U | Ea47 | tail | unk | CC CN | 2 | 2v |
| 83. | U | NinB | tail | rec | CN | 1 |
| 84. | U | NinE | tail | unk | CN | 1 |
| 85. | U | NinF | tail | unk | CN | 1 |
| 86. | U | Orf78 | tail | unk | NC | 1 |
| 87. | U | U | tail | tail | CC | 1 | known |
| 88. | U | Xis | tail | rec | NC | 1 |
| 89. | V | G | tail | tail | D | 2 | Known |
| 90. | W | B | head | head | NC | 1 | Known |
| 91. | U | Cl | tail | trx | CN | 1 |
| 92. | M | Rz1 | tail | lysis | CC CN | 2 | 2v |
| 93. | Orf79 | NinB | unk | rec | CN | 1 |
| 94. | Int | G | rec | tail | G | CN | 2 | 2v |
| 95. | Ea.85 | NinB | unk | rec | CN | 1 |
| 96. | S' | NinB | lysis | rec | CN | 1 |
| 97. | S' | Rz1 | lysis | lysis | CN | 1 |

Bfun = bait protein function, Pfun = prey protein function group (rec = recombination, repl = replication, trx = transcription, conv = lysogenic conversion, ihr - inhibition of host replication [76]). NN, CN, NC, CC indicated the fusion type of the bait and prey proteins (see text). The two NN vectors are indicated by G (pGBK/pGAD) and D (pDEST22/32). Interaction that have been found in inverted prey-bait combinations are indicated by a prime sign (). Interactions that have been found in both bait-prey and prey-bait orientations are indicated by bold and primes (e.g. NC'), respectively. Interactions without any note are unexpected and may not be physiologically relevant. 2v = interactions found with 2 vector pairs. Stf = Orf314.

cells which we used for our interaction screens. We found only one of five known interactions of B (namely W-B) and aberrant folding in yeast may be the reason for not detecting the other four known interactions. In addition, several lambda proteins are processed during assembly. For instance, the C protease is processed and covalently linked to the capsid protein E. This fusion protein is then further processed to yield products named "X1" and "X2" even though recent attempts to identify X1 and X2 were unsuccessful and thus X1 and X2 may be artifacts [14]. A 21 amino acid peptide is also proteolytically removed from the portal protein B but it is not known how this affects its interaction properties. Finally, protein S, which forms a membrane protein involved in lysis, is made in two variants that use different start codons. In fact, we do find that the shorter variant, S' (105 amino acids) has a slightly different interaction pattern compared to the full-length variant, S (107 amino acids) (Figure 3). We have not investigated the detailed mechanism of these differences but it has been shown in several studies that fragments of proteins show different interaction patterns than their full-length proteins [15, 16] even though this is an extreme case given the small difference between S and S'. While steric hindrance may be an obvious reason for this behavior, little is known about the mechanistic details in most other published cases.

False negatives may also be a result of the obligate stepwise assembly of large protein structures in lambda and other phage, e.g. when a conformational change due to interaction between two proteins creates a new binding site for a third protein. For instance, in phage T7 only the heterodimer of gp5 and the host thioredoxin provides a binding site for the single-stranded-binding protein (SSB=gp2.5) and the primase-helicase gp4 [17]. Such cases can only be detected if all three proteins were expressed simultaneously and the constructs involved allowed the formation of complex oligomers.

False positives

While we found only 53% of all previously known interactions of lambda, we also found many new ones (Table 4). However, many of the new interactions have only been found once and hence are lower confidence interactions (34%) take place within their functional group, they are likely physiological. 34 of the 97 interactions (34%) take place within their functional group, including the 16 known ones. Some of the remaining

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

Page 8 of 15

![Diagram](attachment:diagram.png)

Functional classes | Interactions
--- | ---
Head | Y2H & literature
Hypothetical | Y2H
Transcription | Literature
Other |
Tail |
Replication |
Recombination |

Figure 3 The protein interaction network of phage lambda. Interactions from this study have been integrated with previously published interactions ("literature"). Nodes in the network represent proteins and are colored according to their functional class (see color key). The protein-protein interactions are indicated by lines ("edges"). The edge color represents the source of the interactions, e.g., all red edges are previously reported interactions, all blue interactions were identified in our two-hybrid study, and all green interactions are previously known and are reproduced in our study.

interactions are discussed below in the context of their functional group.

Some proteins appear to be particularly “sticky”. For example, G, a tail protein, is involved in 8 different two-hybrid interactions. The specificity of such interactions is inversely proportional to the number of such interactions; thus, G likely interacts rather unspecifically, and its interactions have to be interpreted cautiously. Similarly, Int, A, Nul, and U are involved in 8 or more two-hybrid interactions each, and thus each interaction has to be evaluated individually keeping in mind its promiscuity. We have attempted a careful manual evaluation in Table 4.

The reason for interaction promiscuity and thus false positives remains unclear. Several hypotheses have been proposed to explain such cases. For example, a protein may have hydrophobic patches that interact unspecifically. Some authors have suggested that simply an increase in abundance might cause a promiscuous gain of interactions [18] but such theories remain to be tested rigorously.

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

Figure 4 Interactions among functional groups of proteins. Each row and column of the shown profile corresponds to a protein-protein interaction (two-hybrid) count with different functional classes (see matrix). The interactions within certain functional classes are enriched compared to other functions groups, e.g. head assembly proteins show 15 interactions among each other, 8 interactions are detected between tail assembly proteins and 3 interactions among proteins of unknown function (see Additional file 1: Tables S4 and S5 for details).

The Y2H assay appears to be sensitive enough to detect weak interactions that are not detectable in NMR experiments, e.g. the interaction between U monomers [19]. The high sensitivity may also explain a significant number of false positives which may have been detected in our screen but which do not have any physiological significance. Future quantitative measurements are thus required to clarify the relationship between affinity and physiological relevance.

Head assembly and structure
The structure of the lambda protein shell is known in great detail [20]. However, its assembly is much less well understood as are the locations and functions of the “minor” proteins that are present in only a few molecules/virion (Figure 5). The portal protein B is believed to be the nucleator or initiator of head assembly, which first assembles with the C protease and with the scaffolding protein Nu3 into an ill-defined initiator structure. B, C, and Nu3 are known to form a complex in which several interactions have been previously reported (C′-B, C-Nu3, Nu3-Nu3, and Nu3-B, Table 2). We could not detect B in any interaction although we did find Nu3-C, Nu3-Nu3 and Nu3 interactions with E and Z. This is noteworthy because Nu3-E and Nu3-Z are new interactions. It is known that E (the major capsid protein) assembles onto or around the initiator structure to form the procapsid [12], and it is conceivable that B joins such an assembly. If Nu3 and C proteins are both required for B to join, we would have missed this interaction, given that we tested only pairs of proteins. Nu3 also appears to form dimers by the Y2H analysis, and this has been confirmed independently (C. Catalano, pers. comm.).

The head shell is bound by the D protein which stabilizes the coat protein shell. However, if Nu1, A, or FI are

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

![Diagram](attachment:diagram.png)

Figure 5 Head assembly. Head assembly has been subdivided in five steps although most steps are not very well understood in mechanistic terms. The tail is assembled independently. The C protease, the scaffolding protein Nu3, and the portal protein (B) form an ill-defined initiator structure. Protein E joins this complex but the chaperonins GroES and GroEL are required for that step. Within the prohead C and E are processed to form covalently joined X1 and X2 proteins although this is controversial (see text). Proteins Nu1, A, and F1 are required for DNA packaging. Protein D joins and stabilizes the capsid as a structural protein. FII and W are connecting the head to the tail that joins once the head is completed. Modified after [12] and [20].

missing, DNA is not packaged and as a consequence, the coat shell does not expand, and D can only add after expansion. We could confirm the A-Nu1 interaction as well as the interactions between F1 and A and F1 and E which were previously known only from genetic experiments [21,22]. We also confirmed the D-E and E-E interactions.

The terminase and the portal proteins are the largest proteins of the lambda head. Using fragments of these proteins as baits - as opposed to full-length proteins - may result in additional interactions, especially since we were not able to detect most of the B interactions reported in the literature (Tables 2 and 4).

Tail assembly and structure

Tail assembly is even less well understood than head assembly (Figure 6). From genetic analyses it is known that the host receptor protein J initiates the process with I, L, K, and G (including its fusion protein G-T) successively joining the process [23]. Older studies suggest a slightly different order of action, namely J > I > K > L [24]. In fact, it is not known if I, L and M are components of the finished virion or are assembly factors that are not present in virions. It is thus difficult to reconstruct the detailed molecular events during tail assembly. In any case, J eventually associates with the tape measure protein H, and the major tail protein V forms a tube around this central rod. U finally joins the head-proximal part of the tail. Similarly, W and FII join

to the portal protein in the head to form the binding site for the tail. The main tail proteins are connected by known direct protein-protein interactions (Table 2) but the interactions during the initiation of tail assembly have eluded previous studies. In fact, we failed to detect any interaction involving J and I, and the only interactions of L and K did not involve other tail proteins (Table 4). However, we did find several new interactions that are potentially relevant for tail assembly. For instance, G, a fairly promiscuous protein with a total of 8 interactions, was found to bind to V, G, T, H, and M. It is thus possible that it acts as a scaffold organizing the assembly of the tail. By contrast, the interactions of H and V with G were their sole tail-related interactions. We did not find the tail fiber proteins Stf and Tfa to interact with other tail proteins in our screens. Stf has been speculated to assume a trimeric structure, similar to the tail fiber protein of phage T4 [25] although there is no specific evidence for oligomerization in lambda.

In summary, it is surprising that we found so many virion protein interactions, given that virion assembly is an obligately ordered pathway and most binding sites may be only present in the growing virion and not on individual unassembled proteins.

Transcription

The genetic switch leading to a decision between lysogeny and lysis has made lambda a prime model system

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

![Diagram](attachment:diagram.png)

Figure 6 Tail assembly. The lambda tail is made of at least 6 proteins (U, V, J, H, Tfa, Stf) with another 7 required for assembly (I, M, L, K, G/T, Z). Assembly starts with protein J, which then, in a poorly characterized fashion, recruits proteins I, L, K, and G/T to add the tape measure protein H. G and G/T then leave the complex so that the main tail protein (V) can assemble on the J/H scaffold. Finally, U is added to the head-proximal end of the tail. Protein Z is required to connect the tail to the pre-assembled head. Protein H is cleaved between the action of U and Z [31]. It remains unclear if proteins M and L are part of the final particle [24]. Modified after [23].

for transcriptional regulation. A significant fraction of lambda literature has been devoted to this question [3]. Here, we ignore the interactions of transcription factors with DNA and concentrate on their interactions among each other and the transcriptional machinery. Several factors form dimers (Cro, CI, CII, CIII). Of these, we could only confirm the CII self-interaction. CI, CII, and CIII all interact with various components of the virion in our two-hybrid studies, especially of the tail. However, whether these interactions are physiologically relevant is questionable. Notably, the antiterminators N and Q also show a number of interactions in our tests although none of these involve any other transcriptional regulators. Also, all of these interactions were found in a single vector combination, so they are not as well supported as other interactions.

Recombination, integration, and excision
Integration of the lambda genome into the host chromosome is part of the establishment of the lysogenic state. Integrase (Int), assisted by the integration host factor (IHF) catalyzes this reaction. Similarly, integrase (Int), this time assisted by excisionase (Xis) and the host Fis protein, catalyzes the excision of the lambda prophage. Three other lambda proteins are known to be involved in homologous recombination: Exo (exonuclease), Bet (= β, strand annealing protein), Gam (an anti-recBCD protein), and NinB (which can replace the recFOR complex which can load RecA onto ssDNA covered with single-stranded DNA-binding (SSB) protein [26]). We did not find the known interaction between Bet and Exo. In fact, we found Int and Bet to both homodimerize, and Bet and

Int to interact. This indicates that these proteins may assist Int. A number of other interactions involving these recombination proteins and unrelated gene products are difficult to explain and require further analysis. However, they may implicate several uncharacterized small ORFs in the process of recombination (Table 4).

Host interactions
At least 15 lambda proteins interact with host proteins (S. Blasche, S.V. Rajagopala & P. Uetz, unpublished data). Lambda critically depends on host factors for integration, transcription, excision and virion assembly. Hence, a detailed understanding of lambda biology depends on information about such host-phage interactions. These interactions are beyond the scope of this study. We will address this issue in a forthcoming paper.

Protein networks and functional genomics of phage lambda
Phage lambda has been studied almost exclusively by detailed and directed functional studies for the past 60 years. Systematic or large-scale studies have been initiated only recently. For instance, Maynard et al. [27] have screened the KEIO collection of *E. coli* deletion mutants for genes that affect lambda reproduction. This study found 57 *E. coli* genes of which more than half had not been associated with lambda biology before. Similarly, Osterhout et al. [28] investigated *E. coli* gene expression as a result of prophage induction and found 728 genes to change their expression patterns when lambda lysogens are induced. We expect to finish our own screens of lambda-host interactions

soon and integrate the resulting protein-protein interactions into a systems biology model of lambda biology.

### Conclusions

Using phage lambda as a benchmark we showed that we can find about 50% of the interactions among its proteins using Y2H screens. No other technology has been able to detect such a large fraction of interactions in a single macromolecular assembly (except crystallization of whole complexes, which is not possible with phage particles). We thus predict that our strategy can find roughly half of all interactions in other phage and protein complexes. However, other methods will be required to find interactions that require chaperones, post-translational modifications, or other additional factors that could not be provided in our assay.

### Methods

#### Cloning the phage lambda ORFs into Gateway entry vector

The DNA sequence of phage lambda was obtained from the NCBI genomes database (NC_001416) and primers were designed, using the Primer Design Tool [29]. The primers were designed without endogenous stop codons. In addition to the 20- to 30-nucleotide-long ORF-specific sequence the attB1 segment (5'-aaaaagcaggctta-3') was added to each forward primer, followed by ORF-specific bases. The attB2 segment (5'-agaaagctgggtg-3') was added at the 5' end of each reverse primer, which was complementary to the end of the ORF, without the last nucleotides of the stop codon.

#### PCR amplification and cloning of lambda ORFs into gateway entry vector

All the ORFs of phage lambda were PCR amplified using KOD DNA polymerase (Novagen), and phage lambda genomic DNA (NEB:N3011L). The complete sequences of attB1 (5'-GGGGACAAGTTTGTACAAAAAAAGCAGGCT-3') and attB2 (5'-GGGGACCACTTTGTACAAGAAAGCTGGGT-3') were added in the secondary round PCR, where the first round PCR product was used as a template, to generate the full-length attB1 and attB2 sites flanking the ORFs. The PCR cycles were used as recommended by the KOD DNA polymerase manufacturer (Novagen, Cat. No.710853).

The PCR-amplified ORFs with attB1 and attB2 sites were recombined into the entry vector pDONR™/Zeo (Invitrogen) by using the BP Clonase™ II Enzyme Mix (Invitrogen). The products resulting from site-specific recombination were transformed into chemically competent *E. coli* (DH5-α) and plated onto solid LB medium containing Zeocin. Two isolated colonies were selected for each reaction and the clones were verified by colony-PCR with pDONR™/Zeo-specific primers. The clones that had an insert of the expected size were picked for plasmid isolation and the plasmid preparations were sequenced with a pDONR™/Zeo-specific forward and reverse primers to verify the insert from both N-terminal and C-terminal ends of the ORFs. All the sequencing reads were analyzed using NCBI standalone BLAST against the phage lambda genome to confirm the identity of each ORF. We obtained 68 entry clones out of 73 targeted lambda ORFs (see Additional file 1: Table S1).

#### Yeast two-hybrid clones

All the lambda phage ORFs in the entry vectors are subcloned into yeast two-hybrid expression vectors (Table 3), by using the LR Clonase™II Enzyme Mix (Invitrogen). The destination vectors used were pDEST22, pDEST32 (Invitrogen), pGADT7g, pGBK7g and pGADCg, pGBKcg vectors [8].

#### Yeast two-hybrid screening

We carried out comprehensive Y2H interaction screening with the following Y2H vector pairs: pDEST32-pDEST22, pGBK7g-pGADT7g, pGBK7g-pGADCg, pGBKcg-pGADCg and pGBKcg-pGADT7g (listed as bait-prey vector pair). In the array screening we tested each protein both as activation (prey) and DNA-binding domain fusion (bait), including C-terminal fusions in pGBKcg and pGADCg. This way, we tested each protein pair in ten different configurations (Figure 2). The yeast two-hybrid assays were conducted as described in detail by Rajagopala et al. [10,30].

#### Data availability

The protein interactions from this publication have been submitted to the IMEx http://www.imexconsortium.org consortium through IntAct http://www.ebi.ac.uk/intact/ and assigned the identifier IM-15871.

### Additional material

Additional file 1: Tables S1-S7(Excel spreadsheet with tables in individual sheets). S1. Lambda pDONR clones. S2. Lambda protein-protein interactions from Y2H screening. S3. Lambda protein-protein interactions with high prey count (unspecific interactions). S4. Phage Lambda Genome Annotation (Uniprot). S5. Protein interaction with different functional groups. S6. Protein interaction confidence assessment. S7. Layout of Y2H preys pGADT7g and pGADC on screening plates.

### Acknowledgements

Svetlana Shtivelband and Kenny Huang helped in an early phase of this project with cloning lambda ORFs. We thank Johannes Goll for the PPIs statistical analysis. PU was funded by NIH grant R01GM79710 and the

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

European Union (grant HEALTH-F3-2009-223101). SC acknowledges supported by the NIH (grant AI074825).

**Author details**
1. J Craig Venter Institute, Rockville, MD 20850, USA.
2. Division of Microbiology and Immunology, Pathology Department, University of Utah School of Medicine, Salt Lake City, UT 84112, USA.
3. Center for the Study of Biological Complexity, Virginia Commonwealth University, PO Box 842030, 1015 Floyd Ave., Richmond, VA 23284, USA.
4. Proteros Biostructures, Am Klopferspitz 19, D - 82152 Martinsried, Germany.

**Authors’ contributions**
SVR conducted all experiments and analyzed data. SRC analyzed data and wrote part of the paper. PU conceived this study, analyzed data, and wrote part of the paper. All authors contributed in writing the manuscript and approved its final content.

Received: 12 May 2011 Accepted: 26 September 2011 Published: 26 September 2011

**References**

1. Lederberg E: Lysogenicity in E. coli K-12. *Genetics* 1951, **36**:560.
2. Wommack KE, Colwell RR: Virioplankton: viruses in aquatic ecosystems. *Microbiol Mol Biol Rev* 2000, **64**(1):69-114.
3. Ptashne M: A Genetic Switch - Phage Lambda Revisited. Cold Spring Harbor, NY: CSHL Press; Third 2004.
4. Court DL, Oppenheim AB, Adhya SL: A new look at bacteriophage lambda genetic networks. *J Bacteriol* 2007, **189**(2):298-304.
5. Cao Y, Lu HM, Liang J: Probability landscape of heritable and robust epigenetic state of lysogeny in phage lambda. *Proceedings of the National Academy of Sciences of the United States of America* 2010, **107**(43):18445-18450.
6. Tsay JM, Sippy J, Feiss M, Smith DE: The Q motif of a viral packaging motor governs its force generation and communicates ATP recognition to DNA interaction. *Proc Natl Acad Sci USA* 2009, **106**(34):14355-14360.
7. Hendrix R, Roberts J, Stahl FW, Weisberg R, eds: *Lambda II*. Cold Spring Harbor, NY: CSHL Press; 1983.
8. Stellberger T, Hauser R, Baiker A, Pothineni VR, Haas J, Uetz P: Improving the yeast two-hybrid system with permuted fusions proteins: the Varicella Zoster Virus interactome. *Proteome Sci* 2010, **8**:8.
9. Chen YC, Rajagopala SV, Stellberger T, Uetz P: Exhaustive benchmarking of the yeast two-hybrid system. *Nature Methods* 2010, **7**(9):667-668.
10. Rajagopala SV, Hughes KT, Uetz P: Benchmarking yeast two-hybrid systems using the interactions of bacterial motility proteins. *Proteomics* 2009, **9**(23):5296-5302.
11. Sabri M, Häuser R, Ouellette M, Liu J, Dehbi M, Moeck G, García E, Titz B, Uetz P, Moineau S: Genome annotation and intra-viral interactome of the Streptococcus pneumoniae virulent phage Dp-1. *J Bacteriol* 2011, **193**(2):551-562.
12. Georgopoulos C, Tilly K, Casjens S: Lambdoid Phage Head Assembly. In *Lambda II*. Edited by: Hendrix R, Roberts J, Stahl FW, Weisberg R. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory; 1983:279-304.
13. Ang D, Keppel F, Klein G, Richardson A, Georgopoulos C: Genetic analysis of bacteriophage-encoded chaperonins. *Annu Rev Genet* 2000, **34**:439-456.
14. Medina E, Wieczorek D, Medina EM, Yang Q, Feiss M, Catalano CE: Assembly and maturation of the bacteriophage lambda procapsid: gpC is the viral protease. *J Mol Biol* 2010, **401**(5):813-830.
15. Flajolet M, Rotondo G, Daviet L, Bergametti F, Inchauspe G, Tiollais P, Transy C, Legrain P: A genomic approach of the hepatitis C virus generates a protein interaction map. *Gene* 2000, **242**(1-2):369-379.
16. Boxem M, Maliga Z, Klitgord N, Li N, Lemmens I, Mana M, de Lichtervelde L, Mul JD, van de Peut D, Devos M, et al: A protein domain-based interactome network for C-elegans early embryogenesis. *Cell* 2008, **134**(3):534-545.
17. Hamdan SM, Richardson CC: Motors, switches, and contacts in the replisome. *Annual review of biochemistry* 2009, **78**:205-243.
18. Wilkins MR, Kurnmerfeld SK: Sticking together? Failing apart? Exploring the dynamics of the interactome. *Trends in Biochemical Sciences* 2008, **33**(5):195-200.
19. Edmonds L, Liu A, Kwan JJ, Avanessy A, Caracoglia M, Yang J, Maxwell KL, Rubenstein J, Davidson AR, Donaldson LW: The NMR structure of the gpU tail-terminator protein from bacteriophage lambda: identification of sites contributing to Mg(II)-mediated oligomerization and biological function. *J Mol Biol* 2007, **365**(1):175-186.
20. Lander GC, Evilevitch A, Jeembaeva M, Potter CS, Carragher B, Johnson JE: Bacteriophage lambda stabilization by auxiliary protein gpD: timing, location, and mechanism of attachment determined by cryo-EM. *Structure* 2008, **16**(9):1399-1406.
21. Catalano CE, Tomka MA: Role of gpFl protein in DNA packaging by bacteriophage lambda. *Biochemistry* 1995, **34**(31):10036-10042.
22. Murialdo H, Tzamtzis D: Mutations of the coat protein gene of bacteriophage lambda that overcome the necessity for the Fl gene; the EF1 domain. *Mol Microbiol* 1997, **24**(2):341-353.
23. Bacteriophage lambda tail assembly pathway. [http://www.pitt.edu/~duda/lambdatail.html].
24. Hendrix R, Casjens S: Chapter 27: Bacteriophage Lambda and its Genetic Neighborhood. In *The Bacteriophages*. Edited by: Calendar R. Oxford: Oxford University Press; 2006:409-445.
25. Makhov AM, Trus BL, Conway JF, Simon MN, Zurabishvili TG, Mesyanzhinov W, Steven AC: The short tail-fiber of bacteriophage T4: molecular structure and a mechanism for its conformational transition. *Virology* 1993, **194**(1):117-127.
26. Maxwell KL, Reed P, Zhang RG, Beasley S, Walmsley AR, Curtis FA, Joachimiak A, Edwards AM, Sharples GJ: Functional similarities between phage lambda Orf and Escherichia coli RecFOR in initiation of genetic exchange. *Proc Natl Acad Sci USA* 2005, **102**(32):11260-11265.
27. Maynard ND, Birch EW, Sanghvi JC, Chen L, Gutschow MV, Covert MW: A forward-genetic screen and dynamic analysis of lambda phage host-dependencies reveals an extensive interaction network and a new anti-viral strategy. *PLoS Genet* 2010, **6**(7):e1001017.
28. Osterhout RE, Figueroa IA, Keasling JD, Arkin AP: Global analysis of host response to induction of a latent bacteriophage. *BMC microbiology* 2007, **7**:82.
29. Express Primer Tool for High Throughput Gene Cloning and Expression. [http://tools.bio.anl.gov/bioJAVA/jsp/ExpressPrimerTool/].
30. Rajagopala SV, Titz B, Uetz P: Array-based yeast two-hybrid screening for protein-protein interactions. *Yeast Gene Analysis*, Second 2007, **36**:139-163.
31. Tsui LC, Hendrix RW: Proteolytic processing of phage lambda tail protein gpH: timing of the cleavage. *Virology* 1983, **125**(2):257-264.
32. Catalano CE: The terminase enzyme from bacteriophage lambda: a DNA-packaging machine. *Cell Mol Life Sci* 2000, **57**(1):128-148.
33. de Beer T, Fang J, Ortega M, Yang Q, Maes L, Duffy C, Berton N, Sippy J, Overduin M, Feiss M, et al: Insights into specific DNA recognition during the assembly of a viral genome packaging machine. *Mol Cell* 2002, **9**(5):981-991.
34. Hang Q, Woods L, Feiss M, Catalano CE: Cloning, expression, and biochemical characterization of hexahistidine-tagged terminase proteins. *J Biol Chem* 1999, **274**(22):15305-15314.
35. Duffy C, Feiss M: The large subunit of bacteriophage lambda’s terminase plays a role in DNA translocation and packaging termination. *J Mol Biol* 2002, **316**(3):547-561.
36. Ray P, Murialdo H: The role of gene Nu3 in bacteriophage lambda head morphogenesis. *Virology* 1975, **64**(1):247-263.
37. Casjens S: Bacteriophage lambda FlI gene protein: role in head assembly. *J Mol Biol* 1974, **90**(1):1-20.
38. Casjens S, Horn T, Kaiser AD: Head assembly steps controlled by genes F and W in bacteriophage lambda. *J Mol Biol* 1972, **64**(3):551-563.
39. Maxwell KL, Yee AA, Booth V, Arrowsmith CH, Gold M, Davidson AR: The solution structure of bacteriophage lambda protein W, a small morphogenetic protein possessing a novel fold. *J Mol Biol* 2001, **308**(1):9-14.
40. Kochan J, Carrascosa JL, Murialdo H: Bacteriophage lambda preconnectors. Purification and structure. *J Mol Biol* 1984, **174**(3):433-447.
41. Tsui L, Hendrix RW: Head-tail connector of bacteriophage lambda. *J Mol Biol* 1980, **142**(3):419-438.
42. Hendrix RW, Casjens SR: Protein fusion: a novel reaction in bacteriophage lambda head assembly. *Proc Natl Acad Sci USA* 1974, **71**(4):1451-1455.
43. Hendrix RW, Duda RL: Bacteriophage HK97 head assembly: a protein ballet. *Adv Virus Res* 1998, **50**:235-288.

Rajagopala et al. BMC Microbiology 2011, 11:213
http://www.biomedcentral.com/1471-2180/11/213

44. Murialdo H: Early intermediates in bacteriophage lambda prohead assembly. Virology 1979, 96(2):341-367.
45. Hendrix RW, Casjens SR: Assembly of bacteriophage lambda heads: protein processing and its genetic control in petit lambda assembly. J Mol Biol 1975, 91(2):187-199.
46. Hohn T, Flick H, Hohn B: Petit lambda, a family of particles from coliphage lambda infected cells. J Mol Biol 1975, 98(1):107-120.
47. Murialdo H, Becker A: Head morphogenesis of complex double-stranded deoxyribonucleic acid bacteriophages. Microbiol Rev 1978, 42(3):529-576.
48. Mikawa YG, Maruyama IN, Brenner S: Surface display of proteins on bacteriophage lambda heads. J Mol Biol 1996, 262(1):21-30.
49. Sternberg N, Hoess RH: Display of peptides and proteins on the surface of bacteriophage lambda. Proc Natl Acad Sci USA 1995, 92(5):1609-1613.
50. Yang F, Forrer P, Dauter Z, Conway JF, Cheng N, Cerritelli ME, Steven AC, Pluckthun A, Wlodawer A: Novel fold and capsid-binding properties of the lambda-phage display platform protein gpD. Nat Struct Biol 2000, 7(3):230-237.
51. Casjens SR, Hendrix RW: Locations and amounts of major structural proteins in bacteriophage lambda. J Mol Biol 1974, 88(2):535-545.
52. Dokland T, Murialdo H: Structural transitions during maturation of bacteriophage lambda capsids. J Mol Biol 1993, 233(4):682-694.
53. Katsura I, Tsugita A: Purification and characterization of the major protein and the terminator protein of the bacteriophage lambda tail. Virology 1977, 76(1):129-145.
54. Buchwald M, Murialdo H, Siminovitch L: The morphogenesis of bacteriophage lambda. II. Identification of the principal structural proteins. Virology 1970, 42(2):390-400.
55. Buchwald M, Steed-Glaister P, Siminovitch L: The morphogenesis of bacteriophage lambda. I. Purification and characterization of lambda heads and lambda tails. Virology 1970, 42(2):375-389.
56. Katsura I: Tail assembly and injection. In Lambda II. Edited by: Hendrix R, Roberts J, Stahl FW, Weisberg R. Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory; 1983:331-346.
57. Xu J, Hendrix RW, Duda RL: Conserved translational frameshift in dsDNA bacteriophage tail assembly genes. Mol Cell 2004, 16(1):11-21.
58. Alfano C, McMacken R: Ordered assembly of nucleoprotein structures at the bacteriophage lambda replication origin during the initiation of DNA replication. J Biol Chem 1989, 264(18):10699-10708.
59. Wickner SH, Zahn K: Characterization of the DNA binding domain of bacteriophage lambda O protein. J Biol Chem 1986, 261(16):7537-7543.
60. Zahn K, Blattner FR: Binding and bending of the lambda replication origin by the phage O protein. Embo J 1985, 4(13A):3605-3616.
61. Zahn K, Blattner FR: Direct evidence for DNA bending at the lambda replication origin. Science 1987, 236(4800):416-422.
62. Zahn K, Landy A: Modulation of lambda integrase synthesis by rare arginine tRNA. Mol Microbiol 1996, 21(1):69-76.
63. Bell CE, Lewis M: Crystal structure of the lambda repressor C-terminal domain octamer. J Mol Biol 2001, 314(5):1127-1136.
64. Dodd IB, Perkins AJ, Tsemitidis D, Egan JB: Octamerization of lambda CI repressor is needed for effective repression of P(RM) and efficient switching from lysogeny. Genes Dev 2001, 15(22):3013-3022.
65. Ho Y, Lewis M, Rosenberg M: Purification and properties of a transcriptional activator. The cI protein of phage lambda. The Journal of biological chemistry 1982, 257(15):9128-9134.
66. Halder S, Datta AB, Parrack P: Probing the antiprotease activity of lambdaCIll, an inhibitor of the Escherichia coli metalloprotease HflB (FtsH). J Bacteriol 2007, 189(22):8130-8138.
67. Anderson WF, Takeda Y, Echols H, Matthews BW: The structure of a repressor: crystallographic data for the Cro regulatory protein of bacteriophage lambda. J Mol Biol 1979, 130(4):507-510.
68. Radding CM, Rosenzweig J, Richards J, Cassuto E: Appendix: Separation and characterization of exonuclease, β protein and a complex of both. J Biol Chem 1971, 146:2510-2512.
69. Sam MD, Cascio D, Johnson RC, Clubb RT: Crystal structure of the excisionase-DNA complex from bacteriophage lambda. J Mol Biol 2004, 338(2):229-240.
70. Warren D, Sam MD, Manley K, Sarkar D, Lee SY, Abbani M, Wojciak JM, Clubb RT, Landy A: Identification of the lambda integrase surface that interacts with Xis reveals a residue that is also critical for Int dimer formation. Proc Natl Acad Sci USA 2003, 100(14):8176-8181.

71. Summer E, Berry J, Tran T, Niu L, Struck D, Young R: Rz/Rz1 lysis gene equivalents in pahges of Gram-negative hosts. J Mol Biol 2007.
72. Savva CG, Dewey JS, Deaton J, White RL, Struck DK, Holzenburg A, Young R: The holin of bacteriophage lambda forms rings with large diameter. Mol Microbiol 2008, 69(4):784-793.
73. Grundling A, Smith DL, Blasi U, Young R: Dimerization between the holin and holin inhibitor of phage lambda. Journal of bacteriology 2000, 182(21):6075-6081.
74. Ranade K, Poteete AR: Superinfection exclusion (sieB) genes of bacteriophages P22 and lambda. J Bacteriol 1993, 175(15):4712-4718.
75. Ranade K, Poteete AR: A switch in translation mediated by an antisense RNA. Genes Dev 1993, 7(8):1498-1507.
76. Sergueev K, Court D, Reaves L, Austin S: E.coli cell-cycle regulation by bacteriophage lambda. J Mol Biol 2002, 324(2):297-307.
77. Stayrook S, Jaru-Ampornpan P, Ni J, Hochschild A, Lewis M: Crystal structure of the lambda repressor and a model for pairwise cooperative operator binding. Nature 2008, 452(7190):1022-1025.
78. Jain D, Kim Y, Maxwell KL, Beasley S, Zhang R, Gussin GN, Edwards AM, Darst SA: Crystal structure of bacteriophage lambda cII and its DNA complex. Mol Cell 2005, 19(2):259-269.
79. Datta AB, Roy S, Parrack P: Role of C-terminal residues in oligomerization and stability of lambda CI: implications for lysis-lysogeny decision of the phage. J Mol Biol 2005, 345(2):315-324.
80. Hall BM, Roberts SA, Heroux A, Cordes MH: Two structures of a lambda Cro variant highlight dimer flexibility but disfavor major dimer distortions upon specific binding of cognate DNA. J Mol Biol 2008, 375(3):802-811.
81. Newlove T, Atkinson KR, Van Dorn LO, Cordes MH: A trade between similar but nonequivalent intrasubunit and intersubunit contacts in Cro dimer evolution. Biochemistry 2006, 45(20):6379-6391.
82. Iwai H, Forrer P, Pluckthun A, Guntert P: NMR solution structure of the monomeric form of the bacteriophage lambda capsid stabilizing protein gpD. J Biomol NMR 2005, 31(4):351-356.
83. Chang C, Pluckthun A, Wlodawer A: Crystal structure of a truncated version of the phage lambda protein gpD. Proteins 2004, 57(4):866-868.
84. Kovall R, Matthews BW: Toroidal structure of lambda-exonuclease. Science 1997, 277(5333):1824-1827.
85. Maxwell KL, Yee AA, Arrowsmith CH, Gold M, Davidson AR: The solution structure of the bacteriophage lambda head-tail joining protein, gpFI. J Mol Biol 2002, 318(5):1395-1404.
86. Cardarelli L, Pell LG, Neudecker P, Pirani N, Liu A, Baker LA, Rubinstein JL, Maxwell KL, Davidson AR: Phages have adapted the same protein fold to fulfill multiple functions in virion assembly. Proceedings of the National Academy of Sciences of the United States of America 2010, 107(32):14384-14389.
87. Court R, Cook N, Saikrishnan K, Wigley D: The crystal structure of lambda-Gam protein suggests a model for RecBCD inhibition. J Mol Biol 2007, 371(1):25-33.
88. Fadeev EA, Sam MD, Clubb RT: NMR structure of the amino-terminal domain of the lambda integrase protein in complex with DNA: immobilization of a flexible tail facilitates beta-sheet recognition of the major groove. J Mol Biol 2009, 388(4):682-690.
89. Aihara H, Kwon HJ, Nunes-Duby SE, Landy A, Ellenberger T: A conformational switch controls the DNA cleavage activity of lambda integrase. Mol Cell 2003, 12(1):187-198.
90. Biswas T, Aihara H, Radman-Livaja M, Filman D, Landy A, Ellenberger T: A structural basis for allosteric control of DNA recombination by lambda integrase. Nature 2005, 435(7045):1059-1066.
91. Scharpf M, Sticht H, Schweimer K, Boehm M, Hoffmann S, Rosch P: Antitermination in bacteriophage lambda. The structure of the N36 peptide-boxB RNA complex. Eur J Biochem 2000, 267(8):2397-2408.
92. Leung AK, Duewel HS, Honek JF, Berghuis AM: Crystal structure of the lytic transglycosylase from bacteriophage lambda in complex with hexa-N-acetylchitohexaose. Biochemistry 2001, 40(19):5665-5673.
93. Voegtli WC, White DJ, Reiter NJ, Rusnak F, Rosenzweig AC: Structure of the bacteriophage lambda Ser/Thr protein phosphatase with sulfate ion bound in two coordination modes. Biochemistry 2000, 39(50):15365-15374.
94. Pell LG, Liu A, Edmonds L, Donaldson LW, Howell PL, Davidson AR: The X-ray crystal structure of the phage lambda tail terminator protein reveals the biologically relevant hexameric ring structure and demonstrates a

Rajagopala et al. BMC Microbiology 2011, 11:213  
http://www.biomedcentral.com/1471-2180/11/213  

conserved mechanism of tail termination among diverse long-tailed  
phages. J Mol Biol 2009, 389(5):938-951.  

95. Pell LG, Gasmi-Seabrook GM, Morais M, Neudecker P, Kanelis V, Bona D,  
Donaldson LW, Edwards AM, Howell PL, Davidson AR, *et al*: The solution  
structure of the C-terminal Ig-like domain of the bacteriophage lambda  
tail tube protein. J Mol Biol 2010, 403(3):468-479.  

96. Pell LG, Kanelis V, Donaldson LW, Howell PL, Davidson AR: The phage  
lambda major tail protein structure reveals a common evolution for  
long-tailed phages and the type VI bacterial secretion system. Proc Natl  
Acad Sci USA 2009, 106(11):4160-4165.  

97. Papagiannis CV, Sam MD, Abbani MA, Yoo D, Cascio D, Clubb RT,  
Johnson RC: Fis targets assembly of the Xis nucleoprotein filament to  
promote excisive recombination by phage lambda. J Mol Biol 2007,  
367(2):328-343.  

98. Abbani MA, Papagiannis CV, Sam MD, Cascio D, Johnson RC, Clubb RT:  
Structure of the cooperative Xis-DNA complex reveals a  
micronucleoprotein filament that regulates phage lambda intasome  
assembly. Proc Natl Acad Sci USA 2007, 104(7):2109-2114.  

99. Sam MD, Papagiannis CV, Connolly KM, Corselli L, Iwahara J, Lee J,  
Phillips M, Wojciak JM, Johnson RC, Clubb RT: Regulation of directionality  
in bacteriophage lambda site-specific recombination: structure of the Xis  
protein. J Mol Biol 2002, 324(4):791-805.  

doi:10.1186/1471-2180-11-213  
Cite this article as: Rajagopala *et al.*: The protein interaction map of  
bacteriophage lambda. BMC Microbiology 2011 11:213.

---

Submit your next manuscript to BioMed Central  
and take full advantage of:

- Convenient online submission  
- Thorough peer review  
- No space constraints or color figure charges  
- Immediate publication on acceptance  
- Inclusion in PubMed, CAS, Scopus and Google Scholar  
- Research which is freely available for redistribution  

Submit your manuscript at  
www.biomedcentral.com/submit
