
FEBS Letters 585 (2011) 3337–3341

---

**Biochemical characterization of the very long-chain fatty acid elongase ELOVL7**

Tatsuro Naganuma ${}^{1}$, Yuichiro Sato ${}^{1}$, Takayuki Sassa, Yusuke Ohno, Akio Kihara ${}^{\ast}$

*Laboratory of Biochemistry, Faculty of Pharmaceutical Sciences, Hokkaido University, Japan*

---

**ARTICLE INFO**

Article history:  
Received 26 July 2011  
Revised 14 September 2011  
Accepted 16 September 2011  
Available online 22 September 2011  

Edited by Stuart Ferguson  

Keywords:  
Very long-chain fatty acid  
Elongase  
Lipid  
ELOVL  
Membrane  

---

**ABSTRACT**

Very long-chain fatty acids (VLCFAs) have a variety of physiological functions and are related to numerous disorders. The key step of VLCFA elongation is catalyzed by members of the elongase family, ELOVLs. Mammals have seven ELOVLs (ELOVL1-7), yet none of them has been purified and analyzed. In the presented study we purified ELOVL7 and measured its activity by reconstituting it into proteoliposomes. Purified ELOVL7 exhibited high activity toward acyl-CoAs with C18 carbon chain length. The calculated $K_{\mathrm{m}}$ values toward C18:3(n-3)-CoA and malonyl-CoA were both in the $\mu \mathrm{M}$ range. We also found that progression of the VLCFA cycle enhances ELOVL7 activity.

© 2011 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.

---

### 1. Introduction

Most cellular lipid molecules include as a major constituent at least one fatty acid (FA). FAs are diverse in carbon chain length and degree of saturation, which contributes greatly to the variety of functions exhibited by the lipids. The most abundant cellular FAs are long-chain fatty acids (LCFAs) with 16–18 carbons. Very long-chain fatty acids (VLCFAs), those having a chain length >20 carbons, are less abundant than LCFAs but possess unique functions that cannot be substituted for by LCFAs. LCFAs such as palmitic acid (C16:0), oleic acid (C18:1), and linoleic acid (C18:2) are mainly utilized as components of glycerophospholipids and triacylglycerol [1], the latter of which is relevant to obesity and hyperlipidemia. In contrast, most saturated or monounsaturated VLCFAs are incorporated into sphingolipids and play important functions in physiological processes such as skin barrier and myelin formation [2,3]. Moreover, polyunsaturated VLCFAs such as docosahexaenoic acid (C22:6) exhibit anti-inflammatory and anti-atherogenic effects [4].

VLCFAs are synthesized in the endoplasmic reticulum from LCFAs, which are food-derived or synthesized in the cytosol de novo by FA synthase. VLCFA elongation occurs by cycling through a 4-step reaction (condensation, reduction, dehydration, and reduction), with 2 carbons being added through each cycle [5]. In the first reaction acyl-CoA and malonyl-CoA are condensed to 3-ketoacyl-CoA by one of the FA elongases named the ELOVLs [5]. The second step is an NADPH-dependent reduction catalyzed by the 3-ketoacyl-CoA reductase KAR to produce 3-hydroxyacyl-CoA [6], and the third, a dehydration step by one of the 3-hydroxyacyl-CoA dehydratases HACD1-4 [7]. The resulting 2,3-trans-enoyl-CoA is finally reduced to acyl-CoA by the trans-enoyl-CoA reductase TER, which is NADPH-dependent [6].

The ELOVLs catalyze the first, rate-limiting step of the VLCFA elongation cycle. In mammals, seven ELOVL isozymes (ELOVL1-7) exist, and each has a specific substrate specificity and physiological functions [5]. ELOVLs are also associated with known pathologies. Dominant *ELOVL4* mutations cause a juvenile macular dystrophy (Stargardt disease-3) [8], and *Elolv4* knockout mice die soon after birth due to a deficiency in skin barrier formation [9]. Studies in *Elolv6*-null mice have revealed that ELOVL6 is involved in an obesity-induced insulin resistance [10], and ELOVL7 is reportedly involved in prostate cancer growth [11].

We recently investigated the enzyme activities of all ELOVLs under identical conditions, and determined the substrate specificity of each [12]. We demonstrated that ELOVL1 exhibits high activity toward C20–C22 saturated and monounsaturated acyl-CoAs, and ELOVL7 elongates C16–C20 acyl-CoAs, with its highest activity toward C18-CoAs. In that study we determined the activities only at a fixed concentration (50 $\mu \mathrm{M}$) of each substrate acyl-CoA, and

---

**Abbreviations:** VLCFA, very long-chain fatty acid; FA, fatty acid; LCFA, long-chain fatty acid; SC, synthetic complete; HMF, His${}_{6}$, Myc, and 3xFLAG

* Corresponding author. Address: Laboratory of Biochemistry, Faculty of Pharmaceutical Sciences, Hokkaido University, Kita 12-jo, Nishi 6-choume, Kita-ku, Sapporo 060-0812, Japan. Fax: +81 11 706 4900.  
E-mail address: kihara@pharm.hokudai.ac.jp (A. Kihara).

1 These two authors have equally contributed to this work.

---

0014-5793/$36.00 © 2011 Federation of European Biochemical Societies. Published by Elsevier B.V. All rights reserved.  
doi:10.1016/j.febslet.2011.09.024

used as the enzyme source total membranes from cells overproducing each ELOVL. It is also important, though, to determine both the $K_{\mathrm{m}}$ and $V_{\max }$ values of purified enzymes, so as to evaluate the contribution of each ELOVL to the VLCFA elongation reaction. Although there have been reports on the enzymatic characterization of yeast ELOVL homologs using purified enzymes [13], there have been no such reports regarding the mammalian ELOVLs. In the presented study, we subjected ELOVL7, the least studied of the mammalian ELOVLs, to purification and determination of its $K_{\mathrm{m}}$ and $V_{\max }$ values. Although ELOVL7 lost its activity when solubilized in Triton X-100, we successfully measured its activity by reconstituting it into proteoliposomes. Moreover, by comparing the activity of purified ELOVL7 with activity in a membrane fraction, we determined that progression of the VLCFA cycle is required for maximal activity of ELOVL7.

2. Materials and methods

2.1. Cell culture and transfection

HEK 293T cells were grown in 0.3% collagen-coated dishes, in Dulbecco's modified Eagle's medium (Sigma, St. Louis, MO) containing 10% fetal bovine serum and supplemented with 100 units/ml penicillin and 100 $\mu \mathrm{g} / \mathrm{ml}$ streptomycin. Transfections were performed using Lipofectamine Plus™ Reagent (Invitrogen, Carlsbad, CA) according to the manufacturer's instructions.

The Saccharomyces cerevisiae strain DEY113 ($\Delta p r b 1:: K a n M X 4$ $\Delta p e p 4:: M E T 15)$ is a derivative of BY4741 ($M A T a h i s 3 \Delta 1 l e u 2 \Delta 0$ $m e t 15 \Delta 0 u r a 3 \Delta 0)$ [14]. DEY113 cells carrying the pYS15 or pYS19 plasmid were grown in synthetic complete (SC) medium (0.67% yeast nitrogen base and $2 \% D$ -glucose) containing nutritional supplements.

2.2. Plasmids

The pCE-puro 3xFLAG-1 plasmid is a mammalian expression vector designed to produce an N-terminal 3xFLAG-tagged protein. The pCE-puro 3xFLAG-ELOVL7 plasmid is a derivative of the pCE-puro 3xFLAG-1 plasmid and encodes 3xFLAG-ELOVL7, as described previously [12]. The pAK1018 plasmid (URA3 marker) is a yeast expression vector designed to produce an N-terminal, tandemly oriented His$_6$, Myc, and 3xFLAG (HMF)-tagged protein under the control of the TDH3 (glyceraldehyde 3-phosphate dehydrogenase) promoter. The pYS15 plasmid encoding HMF-ELOVL7 was constructed by cloning ELOVL7 cDNA from the pCE-puro 3xFLAG-ELOVL7 plasmid into the pAK1018 vector. The pYS19 plasmid encoding HMF-ELOVL7 (AA) was created by site-directed mutagenesis using a QuikChange kit (Stratagene, Agilent Technologies, La Jolla, CA) and primers 5'-TCCCTTCATGTATTCGCTGCTACCAT-CATGCCGTGATGGTAGCAGCGAATACATGAA GGAA-3'.

2.3. Purification of HMF-ELOVL7

Yeast cells expressing HMF-ELOVL7 or HMF-ELOVL7 (AA) were suspended in buffer A (50 mM HEPES-NaOH (pH 6.8), 500 mM NaCl, 10% glycerol, 1 mM dithiothreitol, 1 mM phenylmethylsulfonyl fluoride, and a 1× protease inhibitor mixture (Complete™ EDTA free; Roche Diagnostics, Indianapolis, IN)) and lysed by vigorously mixing with glass beads using a BeadBeater (Biospec Products, Bartlesville, OK). After removal of cell debris by centrifugation (2000×g, 3 min, $4^{\circ} \mathrm{C}$ ), the supernatant was centrifuged at high speed (100000×g, 60 min, $4^{\circ} \mathrm{C}$ ). The resulting pellet (membrane fraction) was suspended in buffer A containing 2% Triton X-100. After an additional high-speed centrifugation (100000×g,

60 min, $4^{\circ} \mathrm{C}$ ), the supernatant (solubilized fraction) was incubated overnight at $4^{\circ} \mathrm{C}$ with anti-FLAG M2 agarose (Sigma) while rotating. The beads were washed six times in decreasing concentrations of Triton X-100 in buffer A. The bound proteins were then eluted with buffer A containing 0.1% Triton X-100 and 100 $\mu \mathrm{g} / \mathrm{ml}$ 3xFLAG peptide.

2.4. In vitro FA elongase assay

Membrane fractions used for in vitro elongase assays were prepared as described previously [12]. Reconstitution of proteins into proteoliposomes was performed essentially as described elsewhere [13], using phosphatidylcholine (Avanti Polar Lipids, Alabaster, AL) as the liposomal lipid and Bio-beads SM-2 adsorbents (Bio-Rad, Hercules, CA) for adsorption of Triton X-100. In vitro FA elongation assays were performed as described previously [12].

2.5. Immunoblotting

Immunoblotting was performed as described previously [15], using the anti-FLAG antibody M2 (1 $\mu \mathrm{g} / \mathrm{ml}$; Stratagene, Agilent Technologies, La Jolla, CA) as the primary antibody, and HRP-conjugated anti-mouse IgG F(ab')$_2$ fragment (1:7500 dilution; GE Healthcare Bio-Sciences) as the secondary antibody. Labeling was detected using Pierce Western Blotting Substrate (Thermo Fisher Scientific, Waltham, MA).

3. Results

3.1. Progression of the VLCFA elongation cycle enhances ELOVL7 activity

In our previous report, we determined the substrate specificity of ELOVL7 using total membrane proteins prepared from HEK 293T cells overproducing 3xFLAG-ELOVL7 [12]. In that experiment we included the reducing agent NADPH, which is required for the second and fourth reactions of the VLCFA elongation cycle, but not for the ELOVL7-catalyzed first reaction. Under such conditions, the ELOVL7 product 3-ketoacyl-CoA was rapidly converted to acyl-CoA via 3-hydroxyacyl-CoA and 2,3-trans-enoyl-CoA [12]. Here, we compared the ELOVL7 activity in the presence or absence of NADPH. Total membrane proteins prepared from HEK 293T cells transfected with the vector or with a plasmid encoding 3xFLAG-ELOVL7, were incubated with [$^{14} \mathrm{C}]$ malonyl-CoA and C18:3(n-3)-CoA. After the reaction, the acyl-CoAs produced were converted to FAs by saponification, then separated by TLC. In the presence of NADPH, the levels of the acyl-CoA generated by the ELOVL7 membrane preparations were much higher than those produced by control membranes (Fig. 1A, FA), as reported previously [12]. Low levels of 3-hydroxyacyl-CoA were also detected in the reaction products generated by the ELOVL7 membrane preparations. In contrast, when the reactions were performed in the absence of NADPH, 3-ketoacyl-CoA, but no acyl-CoA or 3-hydroxyacyl-CoA, was produced by the ELOVL7 membrane preparations (Fig. 1A). An additional band (Fig. 1A, asterisk) was also generated by the ELOVL7 membrane preparations, but this may represent a by-product of 3-ketoacyl-CoA, perhaps a decarboxylated form, formed during the saponification reaction. Little 3-ketoacyl-CoA was generated by control membranes in the absence of NADPH (Fig. 1A). Although NADPH itself is not involved in the ELOVL7-catalyzed reaction, the reaction products were more abundant in the presence of NADPH (Fig. 1A, acyl-CoA plus 3-hydroxyacyl-CoA) than in its absence (Fig. 1A, 3-ketoacyl-CoA plus 3-ketoacyl-CoA*). These results suggest that for ELOVL7 to exert its maximal activity, participation of the entire VLCFA elongation cycle is required.

T. Naganuma et al./FEBS Letters 585 (2011) 3337–3341

### 3.2. Enzyme activity of purified ELOVL7

To evaluate the activity of purified ELOVL7 utilizing the proteoliposome-reconstitution system, we first overproduced His<sub>6</sub>-Myc-3xFLAG (HMF)-tagged wild type ELOVL7 in yeast. We also constructed a mutant of the conserved HXXHH motif and used it as a negative control. The HXXHH motif is conserved among ELOVL family members, and its mutation in the yeast ELOVL family protein Fen1 reportedly results in a loss of activity [13]. In wild type ELOVL7, the sequence of this motif is HVFHH, so we created an HVFAA mutant. Both wild type HMF-ELOVL7 proteins and HMF-ELOVL7 (AA) mutant proteins were overproduced in yeast, then solubilized with Triton X-100 and purified. SDS-PAGE and subsequent Coomassie brilliant blue staining revealed that both proteins migrated at around 32 kDa (Fig. 2A). Additional bands, including bands near 70 and 30 kDa were also observed. The 70 kDa band may represent the SDS-resistant dimer, and the 30 kDa band may be a degradation product of HMF-ELOVL7. These bands were all detected by immunoblotting with anti-FLAG antibodies (Fig. 2B).

We next performed in vitro elongase assays using the purified enzymes and the substrate C18:0-CoA. As expected, wild type HMF-ELOVL7 had no activity when solubilized in Triton X-100, but it did exhibit activity when reconstituted into proteoliposomes (Fig. 2C). Inclusion of NADPH did not affect the enzyme activity (Fig. 2C), confirming that ELOVL7 itself does not require NADPH for its catalytic reaction. We observed low levels of 3-hydroxyacyl-CoA in the presence of NADPH, suggesting slight contamination of the enzyme preparation by endogenously expressed yeast.

---

**Fig. 1. NADPH- and membrane-dependent ELOVL7 activity.** Total membrane proteins were prepared from HEK293T cells transfected with a vector (pCE-puro 3xFLAG-1) or a plasmid encoding the 3xFLAG-ELOVL7 protein (pCE-puro 3xFLAG-ELOVL7). (A) Total membrane proteins (10 μg protein) were incubated with 50 μM C18:3(n-6)-CoA and 0.075 μCi [¹⁴C]malonyl-CoA for 30 min at 37 °C in the presence or absence of 1 mM NADPH. After termination of the reactions, lipids were saponified, acidified, extracted, and separated by normal-phase TLC, followed by detection with a bioimaging analyzer BAS-2500 (Fuji Photo Film, Tokyo, Japan). keto, 3-keto-FA; *keto, a by-product of 3-keto-FA; hydroxy, 3-hydroxy-FA. (B and C) Total membrane proteins were prepared from HEK293T cells transfected with the pCE-puro 3xFLAG-ELOVL7 plasmid were solubilized with 1% Triton X-100 then reconstituted into proteoliposomes. (B) In vitro FA elongation assays were performed with 10 μg protein preparations of total membrane proteins (TM) or solubilized membrane proteins (SM), or with proteoliposomes prepared from 10 μg (liposome x1) or 40 μg total membrane proteins (liposome x4). (C) Preparations in (B) (1/20 amounts) were subjected to immunoblotting with an anti-FLAG antibody.

Since ELOVL family members are multi-span membrane proteins [13], solubilization by a non-ionic detergent such as Triton X-100 is necessary for purification. However, when the ELOVL7 membrane preparations were solubilized with Triton X-100, ELOVL7 exhibited no enzymatic activity (Fig. 1B). We speculated that it was necessary for ELOVL7 to be embedded in a lipid bilayer to exert its activity. We therefore reconstituted solubilized proteins from the ELOVL7 membranes into proteoliposomes by removing Triton X-100 in the presence of phosphatidylcholine. Immunoblotting revealed that the reconstitution efficiency was approximately 25%, since a 4-fold (x4) amount of proteoliposomes contained a similar amount of ELOVL7 protein as compared to the total membrane and solubilized membrane fraction (Fig. 1C). Proteoliposomes (x4) having an equivalent amount of ELOVL7 exhibited activity similar to that of total membrane preparations (Fig. 1B). Thus, reconstituting ELOVL7 in proteoliposomes solved the problem of no activity in solubilized ELOVL7.

---

**Fig. 2. Enzyme assays of purified ELOVL7 reconstituted in proteoliposomes.** (A) DEY113 cells harboring the pYS15 plasmid encoding HMF-ELOVL7 or the pYS19 plasmid encoding HMF-ELOVL7 (AA) were cultured in SC medium lacking uracil. Cells were harvested and lysed by glass beads, and total membrane proteins were prepared. Proteins were solubilized with Triton X-100 then subjected to purification using anti-FLAG M2 agarose affinity beads. Fixed amounts of purified proteins (0.75 μg) were separated by SDS-PAGE and stained with Coomassie brilliant blue. (B) Purified proteins (30 ng) were subjected to immunoblotting with an anti-FLAG antibody. (C) Purified proteins (1 μg) in solution or reconstituted into proteoliposomes were incubated with 50 μM C18:0-CoA and 0.075 μCi [¹⁴C]malonyl-CoA for 30 min at 37 °C, in the presence or absence of 1 mM NADPH. After termination of the reactions, lipids were saponified, acidified, extracted, and separated by normal-phase TLC, followed by detection with a bioimaging analyzer BAS-2500. WT, wild type ELOVL7; MT, mutant ELOVL7; keto, 3-keto-FA; *keto, by-product of 3-keto-FA.

3-ketoacyl-CoA reductase Ybr159w/Ifa38. Enzymes responsible for each step of the FA elongation cycle interact with each other and form an elongase complex both in yeast and mammals [12, 13, 16, 17], so it is probable that ELOVL7 is able to form an elongase complex with yeast components including Ybr159w. However, inclusion of the 3-ketoacyl-CoA reductase or others, if any, apparently do not affect the results and conclusion, since the similarly purified mutant HMF-ELOVL7 (AA) had no enzyme activity when reconstituted in proteoliposomes (Fig. 2C).

### 3.3. Substrate specificity of purified ELOVL7

We next investigated the substrate specificity of ELOVL7 using 11 different acyl-CoAs and purified wild type HMF-ELOVL7 proteins reconstituted in proteoliposomes. ELOVL7 exhibited activity toward acyl-CoAs with C16–C20 chain length, irrespective of the number of double bonds (Fig. 3A and B). The highest activity of ELOVL7 observed was toward C18:3(n-3)-CoA and C18:3(n-6)-CoA. ELOVL7 was also active toward C20:4-, C18:0-, C18:1-, C18:2-, and C16:0-CoAs, and weakly toward C20:0-CoA. In contrast, little or no activity was observed toward C22:0-, C24:0-, or C26:0-CoA.

ELOVL7 substrate specificity identified here using purified enzyme correlated well with that in our recent report, determined using crude membrane fractions from cells overproducing ELOVL7 [12].

#### 3.4. Determination of enzyme kinetic parameters of ELOVL7

Determining the kinetic parameters (\(K_m\) and \(V_{max}\)) of ELOVL7 and comparing them with parameters of other ELOVLs will be important to elucidate the actual involvement of ELOVL7 in each elongation reaction. We examined ELOVL7 activity toward C18:3(n-3)-CoA, the FA to which ELOVL7 exhibited the highest activity, at various concentrations. A Lineweaver-Burk plot of the obtained data revealed a \(K_m\) value of 2.6 μM and a \(V_{max}\) value of 0.33 pmol/min/μg protein (Fig. 4A). We also determined the kinetic parameters of ELOVL7 toward another substrate, malonyl-CoA (\(K_m\), 11.7 μM; \(V_{max}\), 0.31 pmol/min/μg) (Fig. 4B). Thus, the \(K_m\) values determined are in the μM range for both acyl-CoA and malonyl-CoA substrates. Moreover, both plots presented as straight lines, indicating that ELOVL7 is not regulated by any homotropic, allosteric effect.

### 4. Discussion

To date, all in vitro assays for the mammalian VLCFA elongases known as ELOVLs were performed using total membrane or microsomal fractions from cells overproducing the studied ELOVL.

![Figure 3](#fig3)

**Fig. 3.** Substrate specificity of purified ELOVL7 enzyme. (A and B) Purified wild type HMF-ELOVL7 protein (1 μg) was reconstituted into proteoliposomes and incubated with the indicated acyl-CoA (50 μM) and 0.075 μCi \[^{14}C\]malonyl-CoA for 30 min at 37 °C. After termination of the reactions, lipids were saponified, acidified, extracted, and separated by normal-phase TLC, followed by detection on X-ray film (A) and quantification by a bioimaging analyzer BAS-2500 (B). Values presented are FA levels generated for each μg protein after 30 min, and represent the mean ± S.D. from three independent experiments.

![Figure 4](#fig4)

**Fig. 4.** Kinetic parameters of ELOVL7. Purified wild type HMF-ELOVL7 protein (0.5 μg) was reconstituted into proteoliposomes and incubated with the indicated concentrations of C18:3(n-3)-CoA and 0.075 μCi (27.3 μM) \[^{14}C\]malonyl-CoA for 30 min at 37 °C (A). The reconstituted HMF-ELOVL7 protein (0.5 μg) was incubated with 50 μM C18:3(n-3)-CoA and indicated concentrations of \[^{14}C\]malonyl-CoA for 30 min at 37 °C (B). After termination of the reactions, lipids were saponified, acidified, extracted, and separated by normal-phase TLC. The radioactivities associated with the reaction product FAs were quantified using a bioimaging analyzer BAS-2500 and are expressed in a Lineweaver-Burk plot.

However, such fractions contain other proteins that can potentially affect the results or the activity of the ELOVL protein under investigation. For example, unidentified regulatory proteins can exist in such fractions, generating differences in the activity of the ELOVL in crude membranes and that of purified protein. Therefore, an in vitro FA elongation assay using purified enzymes is particularly important.

Although membrane proteins must be solubilized by detergent for purification, they sometimes exhibit a loss of activity in such a solubilized state. One possible reason may be structural changes (denaturation). Alternatively, accessibility between an enzyme and a substrate, each existing in different micelles, may be compromised, especially in the case of hydrophobic substrates. ELOVL7 also lost its activity under solubilized conditions (Figs. 1B and 2C). This loss is not restricted to ELOVL7, and may be applicable to ELOVLs in general, since ELOVL1 and ELOVL6 also have no activity in the solubilized state (our unpublished data). We successfully addressed this problem by reconstituting ELOVL7 into proteoliposomes, in which ELOVL7 is embedded in a lipid bilayer (Figs. 1B and 2C). The reconstituted ELOVL7 exhibited activity toward several acyl-CoAs of C16–C20 chain length, with the highest activity toward C18:3(n-3)-CoA and C18:3(n-6)-CoA (Fig. 3A and B). This substrate specificity, determined using purified enzyme, is nearly identical to our previous results using total membrane fractions [12], confirming the validity of the analyses. The determined \( K_m \) values for ELOVL7 toward C18:3(n-3)-CoA and malonyl-CoA were both low, in the μM range. These values may be within the range of actual cellular concentrations, although precise cellular levels are not known.

The activity of ELOVL7 in total membrane fractions was ~12-fold lower in the absence of NADPH than in its presence (Fig. 1A), although purified, reconstituted ELOVL7 exhibited similar activity regardless of the presence of NADPH (Fig. 2C). ELOVL7 itself does not need NADPH for its activity, but other components of the VLCFA elongation machinery, KAR and TER, which catalyze the second and forth step, respectively, do use NADPH as a cofactor. Thus, progression of the entire VLCFA elongation cycle seems to enhance the activity of ELOVL7. Considering that the components of the VLCFA elongation machinery are known to constitute an elongase complex [7,12,13,16,17], we speculate that removal of the reaction product and transfer to the next subunit is required for the subsequent round of reactions. For example, the reaction product of ELOVL7, 3-ketoacyl-CoA, must be removed by KAR to resume the next condensation reaction of ELOVL7. Without this removal step, 3-ketoacyl-CoA may continue to occupy the reaction pocket of ELOVL7.

Our analysis presented here opens the way to analyzing other purified ELOVLs. Determination of the kinetic parameters of all ELOVLs will be required to evaluate the actual involvement of each ELOVL in FA elongation reactions. Moreover, the in vitro assay system described here may be useful in developing drugs for certain ELOVL-associated disorders, for example by screening for inhibitors.

## Acknowledgements

We are grateful to Dr. E.A. Sweeney for scientific editing of the manuscript. This work was supported by a Grant-in-Aid for Scientific Research (B) (23370057) from the Japan Society for the Promotion of Science (JSPS).

## References

[1] Hodson, L., Skeaff, C.M. and Fielding, B.A. (2008) Fatty acid composition of adipose tissue and blood in humans and its use as a biomarker of dietary intake. Prog. Lipid Res. 47, 348–380.

[2] Kihara, A., Mitsutake, S., Mizutani, Y. and Igarashi, Y. (2007) Metabolism and biological functions of two phosphorylated sphingolipids, sphingosine 1-phosphate and ceramide 1-phosphate. Prog. Lipid Res. 46, 126–144.

[3] Mizutani, Y., Mitsutake, S., Tsuji, K., Kihara, A. and Igarashi, Y. (2009) Ceramide biosynthesis in keratinocyte and its role in skin function. Biochimie 91, 784–790.

[4] Torrejon, C., Jung, U.J. and Deckelbaum, R.J. (2007) n-3 Fatty acids and cardiovascular disease: actions and molecular mechanisms. Prostaglandins Leukot. Essent. Fatty Acids 77, 319–326.

[5] Leonard, A.E., Pereira, S.L., Sprecher, H. and Huang, Y.S. (2004) Elongation of long-chain fatty acids. Prog. Lipid Res. 43, 36–54.

[6] Moon, Y.A. and Horton, J.D. (2003) Identification of two mammalian reductases involved in the two-carbon fatty acyl elongation cascade. J. Biol. Chem. 278, 7335–7343.

[7] Ikeda, M., Kanao, Y., Yamanaka, M., Sakuraba, H., Mizutani, Y., Igarashi, Y. and Kihara, A. (2008) Characterization of four mammalian 3-hydroxyacyl-CoA dehydratases involved in very long-chain fatty acid synthesis. FEBS Lett. 582, 2435–2440.

[8] Zhang, K. et al. (2001) A 5-bp deletion in ELOVL4 is associated with two related forms of autosomal dominant macular dystrophy. Nat. Genet. 27, 89–93.

[9] Li, W., Sandhoff, R., Kono, M., Zerfas, P., Hoffmann, V., Ding, B.C., Proia, R.L. and Deng, C.X. (2007) Depletion of ceramides with very long chain fatty acids causes defective skin permeability barrier function, and neonatal lethality in ELOVL4 deficient mice. Int. J. Biol. Sci. 3, 120–128.

[10] Matsuzaka, T. et al. (2007) Crucial role of a long-chain fatty acid elongase, Elovl6, in obesity-induced insulin resistance. Nat. Med. 13, 1193–1202.

[11] Tamura, K. et al. (2009) Novel lipogenic enzyme ELOVL7 is involved in prostate cancer growth through saturated long-chain fatty acid metabolism. Cancer Res. 69, 8133–8140.

[12] Ohno, Y., Suto, S., Yamanaka, M., Mizutani, Y., Mitsutake, S., Igarashi, Y., Sassa, T. and Kihara, A. (2010) ELOVL1 production of C24 acyl-CoAs is linked to C24 sphingolipid synthesis. Proc. Natl. Acad. Sci. USA 107, 18439–18444.

[13] Denic, V. and Weissman, J.S. (2007) A molecular caliper mechanism for determining very long-chain fatty acid length. Cell 130, 663–677.

[14] Brachmann, C.B., Davies, A., Cost, G.J., Caputo, E., Li, J., Hieter, P. and Boeke, J.D. (1998) Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications. Yeast 14, 115–132.

[15] Kihara, A., Ikeda, M., Kariya, Y., Lee, E.Y., Lee, Y.M. and Igarashi, Y. (2003) Sphingosine-1-phosphate lyase is involved in the differentiation of F9 embryonal carcinoma cells to primitive endoderm. J. Biol. Chem. 278, 14578–14585.

[16] Okuda, A., Naganuma, T., Ohno, Y., Abe, K., Yamagata, M., Igarashi, Y. and Kihara, A. (2010) Hetero-oligomeric interactions of an ELOVL4 mutant protein: implications in the molecular mechanism of Stargardt-3 macular dystrophy. Mol. Vis. 16, 2438–2445.

[17] Konishi, H., Okuda, A., Ohno, Y. and Kihara, A. (2010) Characterization of HACD1 K64Q mutant found in arrhythmogenic right ventricular dysplasia patients. J. Biochem. 148, 617–622.
