
Park, J.W. et al.

SUPPLEMENTARY INFORMATION  
for  

Discovery of parallel pathways of kanamycin biosynthesis allows antibiotic manipulation  

Je Won Park${}^{1,2,5}$, Sung Ryeol Park${}^{1,5}$, Keshav K Nepal${}^{2}$, Ah Reum Han${}^{3}$, Yeon Hee Ban${}^{1}$, Young Ji Yoo${}^{1}$, Eun Ji Kim${}^{1}$, Eui Min Kim${}^{2}$, Dooil Kim${}^{4}$, Jae Kyung Sohng${}^{2*}$ & Yeo Joon Yoon${}^{1*}$  

${}^{1}$Department of Chemistry and Nano Science, Ewha Womans University, Seoul 120-750, Republic of Korea. ${}^{2}$Department of Pharmaceutical Engineering, Institute of Biomolecule Reconstruction, Sun Moon University, Chungnam 336-708, Republic of Korea. ${}^{3}$Interdisciplinary Program of Biochemical Engineering and Biotechnology, Seoul National University, Seoul 151-742, Republic of Korea. ${}^{4}$Systems Microbiology Research Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea. ${}^{5}$These authors contributed equally to this work. ${}^{*}$Correspondence should be addressed to J.K.S. (sohng@sunmoon.ac.kr) or Y.J.Y. (joonyoon@ewha.ac.kr).

Nature Chemical Biology: doi:10.1038/nchembio.671

Contents

Supplementary Methods ………… 4
Supplementary Table 1 ………… 12
Supplementary Table 2 ………… 13
Supplementary Results ………… 15
Supplementary Figure 1 ………… 15
Supplementary Figure 2 ………… 16
Supplementary Table 3 ………… 17
Supplementary Figure 3 ………… 18
Supplementary Table 4 ………… 19
Supplementary Figure 4 ………… 20
Supplementary Table 5 ………… 21
Supplementary Figure 5 ………… 22
Supplementary Table 6 ………… 23
Supplementary Figure 6 ………… 24
Supplementary Table 7 ………… 25
Supplementary Figure 7 ………… 26
Supplementary Table 8 ………… 27
Supplementary Figure 8 ………… 28
Supplementary Table 9 ………… 29
Supplementary Figure 9 ………… 30
Supplementary Table 10 ………… 31
Supplementary Figure 10 ………… 32
Supplementary Table 11 ………… 33
Supplementary Figure 11 ………… 34
Supplementary Table 12 ………… 35
Supplementary Figure 12 ………… 36
Supplementary Table 13 ………… 37
Supplementary Figure 13 ………… 38
Supplementary Table 14 ………… 39
Supplementary Figure 14 ………… 40
Supplementary Table 15 ………… 41
Supplementary Figure 15 ………… 42
Supplementary Table 16 ………… 43
Supplementary Figure 16 ………… 44

Page 2 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 17 ………………………………………… 45  
Supplementary Figure 17 ……………………………………… 46  
Supplementary Figure 18 ……………………………………… 47  
Supplementary Table 18 ……………………………………… 48  
Supplementary Figure 19 ……………………………………… 49  
Supplementary Figure 20 ……………………………………… 50  
Supplementary Figure 21 ……………………………………… 51  
Supplementary Figure 22 ……………………………………… 52  
Supplementary Figure 23 ……………………………………… 53  
Supplementary Figure 24 ……………………………………… 54  
Supplementary Figure 25 ……………………………………… 55  
Supplementary Figure 26 ……………………………………… 56  
Supplementary Figure 27 ……………………………………… 57  
Supplementary Table 19 ……………………………………… 58  
Supplementary Figure 28 ……………………………………… 59  
Supplementary Figure 29 ……………………………………… 60  
Supplementary Table 20 ……………………………………… 61  
Supplementary References …………………………………… 62  

Page 3 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

# Supplementary Methods

Chemicals, bacterial strains, and culture conditions. Kanamycin, kanamycin B, neomycin, paromomycin, tobramycin, and amikacin authentic standards were obtained from Sigma. 2-Mercaptoethanol, phenylmethylsulfonyl fluoride (PMSF), phenol/chloroform/isoamyl alcohol (25:24:1), uridine 5′-diphospho-D-glucose (UDP-Glc), and glass beads (150 to 212 μm) were also obtained from Sigma. Heptafluorobutyric acid (HFBA) was purchased from Fluka, and HPLC-grade acetonitrile, methanol, and water were supplied by J.T. Baker. 2-Deoxystreptamine (2-DOS, **1**) and UDP-2-N-acetyl-D-glucosamine (UDP-GlcNAc) were obtained from GeneChem (Republic of Korea), and the cationic solid-phase exchanger (OASIS MCX SPE, 3 mL/60 mg) and vacuum manifold were obtained from Waters. Paromamine (**3**) and neamine (**4**) were prepared from paromomycin and neomycin, respectively, by methanolysis¹. UDP-2-D-glucosamine (UDP-GlcN) was prepared by the enzymatic reaction of UDP-D-glucose pyrophosphorylase with glucosamine-1-phosphate and uridine 5′-triphosphate (UTP) as previously described². UDP-kanosamine (Kns) was synthesized from UDP-Glc by using the cell-free extract (CFE) of the Streptomyces venezuelae expressing *kanC-kanD*. A couple of reactions was carried out by incubating the CFE (4.5 ml) supplemented with UDP-Glc (final 1.5 mM) at 37 °C for 12 h, and then quenched as described in Methods. UDP-Kns from the reactants was purified by HPLC using an XTerra MS C₁₈ reversed-phase column (Waters, 3.5 μm, 250 x 2.1 mm) eluted with 10 mM aqueous HFBA at a flow rate of 0.2 ml/min and subsequent ESI-MS/MS analyses of each peak collected, giving as a white amorphous powder (0.8 mg, 11% yield); HR-ESI-MS: *m/z* 564.0713 in negative ion mode (calculated for C₁₅H₂₅N₃O₁₆P₂, 565.0710) using LCT-premier XE mass spectrometer (Waters); characteristic fragment ion generated by a cleavage of phosphate group at 323.2; ¹H NMR (900 MHz, D₂O using Shigemi advanced NMR microtube from Sigma, Bruker Avance II 900 spectrometer at 298 K, processed using the Mnova software from Mestrelab Research S.L.): δ 10.06 (s, 1H), 9.64 (d, 1H) 5.95 (d, 1H), 5.71 (d, 1H), 5.41 (t, 1H), 5.14 (s, 2H), 4.58 (t, 1H), 4.41 (q, 1H), 4.28 (m, 1H), 4.01 (m, 2H), 3.90 (dq, 1H), 3.78 (q, 2H), 3.67 (m, 1H), 3.61 (m, 1H), 3.54 (m, 4H), 3.49 (m, 1H), 2.82 (t, 1H).

Escherichia coli DH10B and plasmid Litmus 28 (New England Biolabs) were used for routine subcloning. Expression vectors pLVE2.3d and pET series such as pET21c and pET28a were purchased from Roche and Novagen, respectively. The high-copy-number *E. coli*-Streptomyces shuttle vector pSE34 containing the constitutive *ermE* promoter plus a thiostrepton resistance marker was used for expression³. Recombinant strains of *S. venezuelae* were grown in liquid R2YE⁴ at 30 °C for preparation of protoplasts, which were regenerated on R2YE agar medium supplemented with thiostrepton (30 μg/ml). The *E. coli* strains used for

Page 4 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

subcloning were grown in LB medium supplemented with ampicillin (50 μg/ml) to select for plasmids. For production of kanamycin intermediates and their analogs, S. venezuelae strains expressing candidate biosynthetic genes were cultivated at 30 °C for 4 days in one-liter baffled Erlenmeyer flasks containing 300 ml of R2YE medium supplemented with thiostrepton (25 μg/ml). S. kanamyceticus ATCC 12853 was grown for 5 days at 30 °C in liquid ISP2 (0.4% yeast extract, 1.0% malt extract, and 0.4% glucose), and S. tenebrarius ATCC 17920 was grown for 5 days at 30 °C in fermentation medium (2.0% glucose, 2.0% soluble starch, 4.0% soybean meal, 0.5% yeast extract, 0.5% CaCO₃, and 0.4% MgSO₄·7H₂O, pH 7.0). The culture medium components, soybean meal, yeast extract, and malt extract were acquired from BD Biosciences (USA). A total of six Gram-negative test strains were employed to determine the antibacterial spectra of kanamycin biosynthetic intermediates, and analogs. Two of these strains were type strains: kanamycin-sensitive (Kan<sup>S</sup>) E. coli ATCC 25922, kanamycin-resistant (Kan<sup>R</sup>) Pseudomonas aeruginosa ATCC 27853. The remaining four strains were clinical isolates obtained from the Culture Collection of Antimicrobial Resistant Microbes (CCARM, Republic of Korea): Kan<sup>S</sup> E. coli CCARM 1A020, Kan<sup>R</sup> E. coli CCARM 1A023, amikacin-sensitive (Amk<sup>S</sup>) P. aeruginosa CCARM 2206, and amikacin-resistant (Amk<sup>R</sup>) P. aeruginosa CCARM 2178.

Cloning and construction of expression plasmids and recombinant S. venezuelae strains. An engineered strain of S. venezuelae deficient in the biosynthesis of the endogenous deoxysugar thymidine 5′-diphospho (TDP)-D-desosamine was used as a heterologous host. A gene replacement plasmid, pYJ188<sup>5</sup>, was introduced into protoplasts of the S. venezuelae YJ003 mutant<sup>6</sup> for deletion of the kanamycin modifying gene *aphII*<sup>7</sup> by a replicative plasmid-mediated homologous recombination as described previously<sup>8</sup>. Several double crossover mutants were identified on the basis of kanamycin sensitivity phenotype as well as genotype, which was determined by Southern hybridization.

DNA fragments containing a variety of kanamycin biosynthetic gene sequences were amplified from pSKC2<sup>9</sup> by PCR with specific deoxyoligonucleotide primers (Supplementary Table 1). Fragments encoding NemD, GtmG, and AprD3, AprD4, TobM1, and TobM2 were obtained by PCR-amplification using the genomic DNA of S. fradiae ATCC 10745, Micromonospora echinospora ATCC 15385, and S. tenebrarius ATCC 17920, respectively, as templates. DNA fragments containing the genes *btrl*, *btrJ*, *btrK*, *btrO*, *btrV*, *btrG*, and *btrH* required for the biosynthesis and transfer of S-4-amino-2-hydroxybutyric acid (AHBA) were amplified using the genomic DNA of Bacillus circulans NR3312 as a template. Each primer sequence contained a restriction site to facilitate subcloning of the resulting DNA fragment. PCR was performed using *Pfu* polymerase (Fermentas) under the conditions recommended by the manufacturer. All PCR products were cloned into Litmus 28 and sequenced to confirm their

Page 5 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

authenticity.

The gene constructs made in this study are summarized in Supplementary Table 2. For expression of 1-biosynthetic genes, PCR-amplified fragments containing kanA-kanB and kanK were digested with BamHI/Spel and Xbal/HindIII, respectively, and simultaneously ligated with pSE34, which had been cut with BamHI and HindIII, thus generating pDOS. Plasmid pDOSf (kanA-kanB-kanK-kanF-gtmF-gtmK-gtmL) was constructed to express kanF along with 1-biosynthetic genes by ligating the Spel/HindIII-digested kanF DNA fragment with the Xbal/HindIII-digested fragments of the resistance gene set gtmF-gtmK-gtmL isolated from pYJ489<sup>10</sup>, then cloning the resulting Xbal/HindIII fragment into pDOS. All subsequent cloning steps for construction of the plasmids pPAR, pPARcd, pPARil, pKCXΔcΔd, pKCX, pKABΔcΔd, pKAB, pDOSn, pDOS<sub>t1</sub>, pDOSg, pKCXΔfn, pKCXΔft<sub>1</sub>, pKCXΔfg, and pKABΔfn were carried out using the compatible cohesive ends of Spel and Xbal sites as described above. For the direct fermentative production of 1-N-AHBA-conjugated kanamycins, Litmus 28 containing btrG-btrH was digested with Spel/SnaBI and ligated with a btrl-btrJ DNA fragment that had been cut by Xbal/SnaBI. The resulting plasmid carrying btrG-btrH-btrl-btrJ was subsequently ligated with the DNA fragment containing btrK-btrO-btrV using the same procedure described above. Then, the plasmid containing btrG-btrH-btrl-btrJ-btrK-btrO-btrV was digested with Xbal/Spel and transplanted into the the Xbal site of pKCX and pKAB (see Supplementary Table 2) to generate pKCXb and pKABb, respectively. The correct orientations of the DNA fragments containing btrG-btrH-btrl-btrJ-btrK-btrO-btrV in pKCXb and pKABb were confirmed by restriction fragment analysis. Next, the plasmids pKABΔfna and pKABΔfnΔet<sub>2</sub>a were constructed in an attempt to produce 3′-deoxykanamycins *in vivo*. Litmus 28 containing aprD3 was digested with Spel/HindIII and combined with the Xbal/HindIII DNA fragment containing aprD4. The plasmid containing aprD3-aprD4 was ligated with Xbal/HindIII DNA fragments containing nemD-kanE-kanC-kanD-kanI-kacA-kacL-gtmF-gtmK-gtmL. The insert of resulting plasmid was moved into the Xbal/HindIII sites of pSE34 containing kanA-kanB-kanK (pDOS) to generate pKABΔfna. Plasmid pKABΔfnΔet<sub>2</sub>a was equivalent to pKABΔfna with tobM2 substituted for kanE. To prepare CFEs of S. venezuelae for functional assignment of each gene product, DNA fragments containing kanF, kanE, kanI-kacL, kanC-kanD, and aprD3-aprD4 were transplanted into pSE34 as Xbal/HindIII fragments, generating pKanF, pKanE, pKanI-KacL, pKanC-KanD, and pAprD3-AprD4, respectively.

To obtain the recombinant KanF and KanE, PCR-amplified DNA fragments containing kanF<sub>His6</sub> and kanE<sub>His6</sub> were digested Ndel/Xhol and ligated into the same sites of pVEX2.3d (Roche). The Ndel/BamHI-digested kanF<sub>His6</sub> fragment from pVEX2.3d was ligated into the same sites of pET28a (Novagen) to encode N,C-terminal His<sub>6</sub>-tagged construct. The Ndel/EcoRI-digested kanE<sub>His6</sub> fragment from pVEX2.3d was ligated into the same sites of

Page 6 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

pET21c vector to encode C-terminal His₆-tagged construct. The resulting plasmids were transplanted into pSE34 as XbaI/HindIII fragments, generating pKanF<sub>His6</sub> and pKanE<sub>His6</sub>.

All resulting plasmids were then transformed into an engineered strain of *S. venezuelae*, yielding the corresponding recombinants (see Supplementary Table 2). *S. venezuelae* protoplast formation and transformation procedures were performed as described [11].

### Isolation and identification of kanamycin biosynthetic intermediates and their analogs.
Preparative HPLC was performed with a Spherisorb S5 ODS2 semi-prep column (250 × 20 mm; Waters) eluted with the same mobile phase used in HPLC-ESI-MS/MS analyses (see Methods), at a flow rate of 12 ml/min over a period of 150 min. The eluent was fractionated into 3-mL portions that were monitored by HPLC-ESI-MS/MS to detect and characterize each kanamycin-related biosynthetic intermediate and analog. Fractions containing products of interest were pooled and extracted using OASIS MCX SPE cleanup [12] followed by freeze-drying. <sup>1</sup>H, <sup>13</sup>C, and 2D <sup>1</sup>H-<sup>1</sup>H COSY NMR spectra were acquired with a Varian INOVA 500 spectrometer at 298 K. Chemical shifts were reported in ppm using trimethylsilyl-2,2,3,3-tetradeuteropropionic acid (TSP) as an internal reference. The assignment of each compound was carried out by comparison with previously assigned <sup>1</sup>H and <sup>13</sup>C NMR spectra [13–17], and NMR data processing was performed using MESTREC (Magnetic Resonance Companion) software. Spectral data are described in Supplementary Figures 2 to 21 and Supplementary Tables 3 to 18.

### Expression and purification of recombinant KanF and KanE.
Recombinant strains of *S.* venezuelae harboring pKanF<sub>His6</sub> and pKanE<sub>His6</sub> were grown in R2YE medium at 30 °C for 3 days. The cells were harvested and re-suspended in Tris buffer (75 mM Tris·HCl, 300 mM NaCl, 10 mM MgCl₂, 10% glycerol, 10 mM imidazole, pH 7.5). Lysozyme (Sigma; final 1 mg/ml) and PMSF (Sigma, final 1 mM) were added, and the resulting suspension was incubated on ice for 1 h. The cells were disrupted by sonication in 10 s bursts (30 times) with 30 s cooling periods between bursts, and then the lysate was clarified by centrifugation. Soluble protein was collected and applied to Ni-nitrilotriacetic acid (NTA) agarose (Qiagen) equilibrated with Tris buffer as described above. The unbound proteins were removed by washing with Tris buffer containing 40 mM imidazole, and bound proteins were eluted with the same buffer containing 500 mM imidazole. Sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) analysis was employed to ascertain the presence and purity of protein in each fraction. The desired fractions were pooled and desalted, and then the purified KanF<sub>His6</sub> and KanE<sub>His6</sub> proteins were concentrated using an Amicon Ultracel 30K molecular weight cut-off spin filter (Millipore). Protein concentration was determined by the Bradford protein assay using bovine serum albumin as a standard. Typically, approximately 1.2 and 1.0 mg of KanF<sub>His6</sub> and KanE<sub>His6</sub> were obtained.

respectively, from 1 litre of culture (see Supplementary Figures 23,24).

**In vitro** reactions using cell-free extracts obtained from the wild-type strains of *S. kanamyceticus* and *S. tenebrarius*. Reactions to determine the conversion of kanamycin C (6), kanamycin B (7), or kanamycin X (12) into kanamycin A (8), were performed by supplementing *S. kanamyceticus* (SK CFE) derived by homogenization with glass beads were further treated using OASIS MCX SPE pass-through to minimize the presence of the kanamycin-related congeners as follows. First, the CFE was loaded onto an SPE cartridge, which was conditioned with methanol and water. Next, the pass-through from the cartridge was pooled, and the same step was repeated. These procedures were performed under ice-cold conditions. The resulting CFE was incubated with 100 μM 6, 7, or 12 for 2 h at 30 °C, then purified by OASIS MCX SPE cleanup<sup>12</sup> and analyzed by HPLC-ESI-MS/MS. Independent experiments were carried out in duplicate, and the results are illustrated in Supplementary Figure 25.

Reactions to determine the glycosyltransfer activities of KanE and TobM2 on 3′-deoxyneamine, namely nebramine (18), were performed by supplementing the CFEs obtained from *S. kanamyceticus* (SK CFE) and *S. tenebrarius* (ST CFE), respectively, with 100 μM 18. To minimize the presence of the kanamycin- or tobramycin-related congeners, which might interfere with the reactions, the CFEs of both wild-type strains derived by homogenization with glass beads were further purified as described above. Reactions were carried out and the products of interest were purified and analyzed as described above. The results are illustrated in Supplementary Figure 28.

**In vitro** reactions using cell-free extracts containing AprD3-AprD4. To determine the activity of AprD3-AprD4 in the 3′-deoxygenation of pseudodi- or trisaccharides, reactions were carried out by supplementing the CFE from the recombinant host expressing *aprD3-aprD4* with 100 μM neamine (4) or 7. After incubation for 2 h at 30 °C, the reaction was quenched. The resulting supernatant containing the product of interest was extracted using OASIS MCX SPE cleanup<sup>12</sup>, then subjected to HPLC-ESI-MS/MS analysis. Independent experiments were carried out in duplicate, and the results are illustrated in Supplementary Figure 29.

3-Dimensional (3D) homology modeling, docking and molecular dynamics simulation of glycosyltransferases KanF and NemD. Homology modeling of the glycosyltransferases (GTs) KanF and NemD was performed with MODELLER<sup>18</sup> and optimized by FoldX<sup>19</sup> using the atomic coordinates of MshA (PDB code: 3C48<sup>20</sup>) as a template. The 3D structural models were assessed by the program VADAR<sup>21</sup>. Molecular docking was used to determine the binding conformation of the glycosyl donors UDP-Glc and UDP-GlcNAc and the acceptor 2-DOS (1).
and the resulting model was energy-minimized and structure optimized with Gaussian03${}^{22}$ using the HF/6-31G(d) basis set for molecular dynamics simulation.

The docking of the glycosyl donors and 1 to GTs was initially carried out according to the topological binding sites predicted by several algorithms${}^{23}$. The automated docking was carried out by the CDOCKER program (Accelrys, Inc.)${}^{24}$ based on the Merck Molecular Force Field (MMFF)${}^{25}$ and the AutoDock 4.0 program suite${}^{26}$. The active site was defined as a sphere with a radius of 6 Å centered on the putative catalytic center of KanF or NemD. Each complex model was solvated with TIP3 water molecules in a cubical box ensuring that the entire surface of the KanF or NemD protein bound to the glycosyl donors/1 was covered by a water layer with a thickness of more than 12 Å. The energy minimization for each complex was performed using the steepest descent algorithm, followed by the conjugate gradient in the CHARMM package${}^{27}$. Then, a 500-ps position restrained molecular dynamics simulation was performed with the proteins and glycosyl donors/1 fixed using CHARMM. Finally, a 3-ns molecular dynamics simulation was started by taking initial velocities from a Maxwellian distribution at 300 K. Solvent and glycosyl donors/1 were independently weakly coupled to a temperature bath with a relaxation time of 0.1 ps. The system was also isotropically weakly coupled to a pressure bath at 1.0 atm with a relaxation time of 0.5 ps and an isothermal compressibility of $0.5 \times 10^{-3}$/bar. Long-range electrostatics were calculated using the particle-mesh Ewald method${}^{28}$. Short-range van der Waals and coulombic interactions were cut off at 0.1 Å. All bond lengths were constrained using the SHAKE algorithm${}^{29}$, and the time step was set to 0.002 ps. The binding free energies between the glycosyl donors and KanF or NemD were calculated by the linear interaction energy method${}^{30,31}$ using default parameters.

### Building and evaluation of 3D KanF and NemD models

3D models of GTs KanF and NemD were generated by homology modeling using the crystal structure of MshA${}^{20}$ as a template. These enzymes have an overall sequence similarity of over 40%. The identities and interactions of critical residues are the same in both enzymes, strongly suggesting that their mechanisms of action are conserved.

To gain insight into the catalytic mechanism, we repeated simulations aimed at the preparation of a KanF/NemD complex with glycosyl donors/acceptor. First, we performed a docking of the glycosyl donors UDP-Glc and UDP-GlcNAc to the putative substrate binding site in KanF and NemD, respectively, using AutoDock. Secondly, we found that the acceptor 2-DOS (1) docked well into the putative glycosyl-acceptor binding site of each GT/glycosyl donors complex using CDOCKER and deeply into the Glc or GlcNAc area, resulting in ideal interactions with the glycosyl donors and the surrounding residues of each GT. The distance between the C1 atom of the glycosyl donors and the O3 atom of 1 fluctuates within only very narrow limits (~3.0

Page 9 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671
Å). The total energy of the KanF or NemD/glycosyl donors/1 complex was stabilized and remained stable throughout the simulation. The radius of gyration for the complexes was found to be stable at roughly 20 Å throughout the simulation, suggesting that the presence of 1 in the complexes induces a more compact protein structure than in its absence.

There is a large hydrophilic pocket between the two domains of the GT, which is formed by the glycosyl donor-neighboring hydrophilic KanF (NemD) residues Glu14 (Glu41), Gln19 (Gln46), Gln20 (Gln47), His88 (His115), Tyr110 (Tyr137), Thr111 (Thr138), Asp117 (Asp204), Lys218 (Lys245), Asn272 (Asn299), Glu292 (Glu319), Glu293 (Glu320), Ser297 (Ser324), and Glu300 (Glu327). Analysis of hydrophilic-hydrophilic contact in the docked structure showed that these residues are playing a crucial role in the interaction. The sugar groups of the glycosyl donors/1 complex would interact with hydrophilic residues of the binding pocket. The uracil moiety of both glycosyl donors forms hydrophobic interactions with Gly243 (Gly270), Phe269 (Phe296), and Ile275 (Ile302), whereas their ribose moieties generate the hydrophilic interactions with Ser297 (Ser324), and Glu300 (Glu327). The hydrophilic contact between 1 and Tyr110 (Tyr137) is strong; the phenolic group of Tyr is moved somewhat inward or outward from its original position as 1, which is located above the phenolic group, tries to get closer to Tyr110 (Tyr137). However, Leu89 (Leu116) makes a strong hydrophobic interaction with 1, which does not move much at all. The amino group at the C3 position in 1 interacts with Asp177 (Asp204), which allows for the dissipation of the delocalized negative charge of 1. The hydroxyl group at the C6 position in 1 makes a hydrophilic interaction with His88 (His115), whereas the ring moiety of 1 makes hydrophobic interactions with Gly16 (Gly43) and Val17 (Val44). The Glc or GlcNAc of the glycosyl donor is anchored to a pocket formed by Glu292 (Glu319), Glu293 (Glu320), and Leu294 (Leu321), suggesting that these residues play a critical role in the stability of those complexes. The sugar moieties of the glycosyl donors are stacked in between the carboxyl moieties of Glu292 (Glu319) and Glu293 (Glu320). Leu294 (Leu321) is located almost at the center of the hydrophobic and hydrophilic interaction with the sugar moiety.

Molecular dynamics and binding free energy analysis. The glycosyl donors have several hydrophilic groups, as described above; thus, hydrogen bonding with GTs as well as solvent molecules at the active site plays an important role in their ability to bind tightly with both KanF and NemD. The time dependence of the number of hydrogen bonds is illustrated in Supplementary Figure 26. The number of hydrogen bonds between KanF and UDP-Glc (~5) is larger than that between KanF and UDP-GlcNAc (~2) (Supplementary Fig. 26a,b), whereas the number between NemD and UDP-Glc (~3) is smaller than that between NemD and UDP-GlcNAc (~6), indicating that UDP-Glc binds more tightly to KanF than does UDP-GlcNAc to NemD (Supplementary Fig. 26c,d). Snapshots of the active site structure of all of the above

Page 10 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

simulations at 3 ns, which are shown in Supplementary Figure 27, provide more detailed information regarding the interaction between the glycosyl donors and both GTs. The calculated free energies of the binding of UDP-Glc and UDP-GlcNAc to KanF and NemD is -42.2, -47.3, -41.4, and -46.7 kJ/mol, respectively, and are listed in Supplementary Table 19.

Page 11 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

| Gene         | Primer sequence (5' to 3', restriction site underlined)                                                                 | Portion of gene | Restriction enzyme |
|--------------|-----------------------------------------------------------------------------------------------------------------|-----------------|--------------------|
| kanA-kanB    | CCCGGATCCGAATCCCCCTTCGTGACG                                                                                   | 5'              | BamHI              |
|              | GTGACTAGTGTTCGTGACCACCGCGTCGA                                                                                 | 3'              | SpeI               |
| kanK         | GCTTCTAGAACCTCCGGAGCACCCGTGCA                                                                                 | 5'              | XbaI               |
|              | GTGAAGCTTCGTGGTGCCGGACAGGCCCTA                                                                                 | 3'              | HindIII            |
| kanC         | GGGACCTCTAGAACGGGTGGTCGACGAAAC                                                                                | 5'              | XbaI               |
|              | TCTTCCAAGCTTACTAGTTGTCCGGCGGTCGCCCCGA                                                                           | 3'              | SpeI/HindIII       |
| KanD         | GACCGCTCTAGACACCTCCGAGGGTCCCTCTC                                                                              | 5'              | XbaI               |
|              | TGAAAAAGCTTACTAGTGGGTGACGAGACGCCGGG                                                                            | 3'              | SpeI/HindIII       |
| kanE         | ACATCTAGAGGCTCCGGAAGACCGCCGACGCCA                                                                             | 5'              | XbaI               |
|              | TCGAAGCTTACTAGTGGAGACGAGGAGGACCCT                                                                             | 3'              | SpeI/HindIII       |
| kanF         | CGAGGCTCTAGAGCCGACCAGAAACCCATT                                                                                | 5'              | XbaI               |
|              | TAGAACAAAGCTTACTAGTACGTCGGGGTGTCGTACGG                                                                         | 3'              | SpeI/HindIII       |
| kanL-kacL    | GATCGCTCTAGACACCTGGTTCTGGTTCCC                                                                                | 5'              | XbaI               |
|              | AGGGGAAAGCTTACTAGTTGCAGGAGATGCCGACCG                                                                           | 3'              | SpeI/HindIII       |
| kacA         | GGCGTCTAGATACCGGGAGATCGGGGCTGTG                                                                               | 5'              | XbaI               |
|              | ACAAAAGCTTACTAGTTGCTCATAGCGACTCCTGT                                                                            | 3'              | SpeI/HindIII       |
| nemD         | CACTCTAGACATCGCCGTCTCTCCCCGT                                                                                  | 5'              | XbaI               |
|              | TCAAAGCTTACTAGTGGCGCAGATACGGCGCAC                                                                              | 3'              | SpeI/HindIII       |
| tobM1        | CCGTCTAGACCGCCCTTTCCCCGCAAC                                                                                   | 5'              | XbaI               |
|              | GATAAGCTTACTAGTCACACCGTCTCTATTCCTG                                                                            | 3'              | SpeI/HindIII       |
| tobM2        | CAATCTAGACGGGAGGCGACCGTGT                                                                                     | 5'              | XbaI               |
|              | GCGAAGCTTACTAGTTTCAGACGCATACGCCCA                                                                             | 3'              | SpeI/HindIII       |
| gtmG         | TGTCCTCTAGAGCTGCCCGGTCACTTCCCCGC                                                                               | 5'              | XbaI               |
|              | AAAAAGCTTACTAGTCACTTCCGGAAGAATC                                                                               | 3'              | SpeI/HindIII       |
| btrG-btrH    | TTCTCTAGACCTGCAGGTTCTAGGAAAACGGCATGCC                                                                          | 5'              | XbaI/SbfI          |
|              | GGCTACGTA AAAACTAGTGGTTTATCCGCTTTTGCT                                                                          | 3'              | SpeI/SnaBI         |
| btrI-btrJ    | ACCTCTAGACGGAACC GCCATCCCCAT                                                                                   | 5'              | XbaI               |
|              | TCCTACGTA AAAACTAGTGAGTTAATGAACAGCCGT                                                                          | 3'              | SpeI/SnaBI         |
| btrK-btrO    | AGCTCTAGAAAAGCCC GAAGCCTGCTTG                                                                                 | 5'              | XbaI               |
|              | GTTTACGTA AAAACTAGTTCCCTAGACGGATT CGA                                                                           | 3'              | SpeI/SnaBI         |
| btrV         | TATTCTAGAATTGACTTATAACTCAAT                                                                                   | 5'              | XbaI               |
|              | CATTA CGTA AAAACTAGTATCCGAACGTCACATAAG                                                                         | 3'              | SpeI/SnaBI         |
| aprD3        | GGTCTAGATGATGGACGTGGCCGCGA                                                                                    | 5'              | XbaI               |
|              | GTTAAGCTTACTAGTGGGCCGTCGGTCGT CCTG                                                                            | 3'              | SpeI/HindIII       |
| aprD4        | GGCTCTAGAAACCCCACCCTCACCATCCAG                                                                                | 5'              | XbaI               |
|              | CGCAAGCTTACTAGTATCGTTGGTCTCCGGCT                                                                              | 3'              | SpeI/HindIII       |
| kanF_His6    | TACCATATGCGCGTACTCCGGCTCACC                                                                                   | 5'              | NdeI               |
|              | GTACTCGAGCCGTTCTTTGCCGAGTAC                                                                                   | 3'              | XhoI               |
| kanE_His6    | GGGCATATGACCGAGCCTGCCAAGGGT                                                                                   | 5'              | NdeI               |
|              | GGACTCGAGCCC CGATCTCCCGGTA                                                                                    | 3'              | XhoI               |

Supplementary Table 2 | Constructed gene combinations used for in vivo heterologous expression and in vitro reactions using the cell-free extracts or His-tagged proteins obtained from recombinants

| 2DOS   | Resistance         | GT1    | 2'DA     | GT2    | Kns    | 6'AM       | AHBA        | 3'DO      |
|--------|--------------------|--------|----------|--------|--------|------------|-------------|-----------|
| **In vivo** |
| pDOS   | kanA-kanB-kanK     |        |          |        |        |            |             |           |
| pDOSf  | kanA-kanB-kanK     | gtmF-gtmK-gtmL | kanF |        |        |            |             |           |
| pPAR   | kanA-kanB-kanK     | gtmF-gtmK-gtmL | kanF | kacA   |        |            |             |           |
| pPARcd | kanA-kanB-kanK     | gtmF-gtmK-gtmL | kanF | kacA   | —      | kanC-kanD  |             |           |
| pPARIl | kanA-kanB-kanK     | gtmF-gtmK-gtmL | kanF | kacA   | —      |            | kanI-kacL   |           |
| pKCXΔcΔd | kanA-kanB-kanK | gtmF-gtmK-gtmL | kanF | kacA   | kanE   |            |             |           |
| pKCX   | kanA-kanB-kanK     | gtmF-gtmK-gtmL | kanF | kacA   | kanE   | kanC-kanD  |             |           |
| pKABΔcΔd | kanA-kanB-kanK | gtmF-gtmK-gtmL | kanF | kacA   | kanE   | —          | kanI-kacL   |           |
| pKAB   | kanA-kanB-kanK     | gtmF-gtmK-gtmL | kanF | kacA   | kanE   | kanC-kanD  | kanI-kacL   |           |
| pDOSn  | kanA-kanB-kanK     | gtmF-gtmK-gtmL | nemD |        |        |            |             |           |
| pDOS${}_{t_{1}}$ | kanA-kanB-kanK | gtmF-gtmK-gtmL | tobM1 |        |        |            |             |           |
| pDOSg  | kanA-kanB-kanK     | gtmF-gtmK-gtmL | gtmG |        |        |            |             |           |
| pKCXΔfn | kanA-kanB-kanK | gtmF-gtmK-gtmL | nemD | kacA   | kanE   | kanC-kanD  |             |           |
| pKCXΔft${}_{1}$ | kanA-kanB-kanK | gtmF-gtmK-gtmL | tobM1 | kacA   | kanE   | kanC-kanD  |             |           |
| pKCXΔfg | kanA-kanB-kanK | gtmF-gtmK-gtmL | gtmG | kacA   | kanE   | kanC-kanD  |             |           |
| pKABΔfn | kanA-kanB-kanK | gtmF-gtmK-gtmL | nemD | kacA   | kanE   | kanC-kanD  | kanI-kacL   |           |
| pKCXb  | kanA-kanB-kanK     | gtmF-gtmK-gtmL | kanF | kacA   | kanE   | kanC-kanD  | —           | btrGHIJKOV |
| pKAb   | kanA-kanB-kanK     | gtmF-gtmK-gtmL | kanF | kacA   | kanE   | kanC-kanD  | kanI-kacL   | btrGHIJKOV |

Supplementary Table 2. (Contd.)

| 2DOS           | Resistance            | GT1          | 2'DA   | GT2   | Kns         | 6'AM        | AHBA       | 3'DO     |
|-----------------|-----------------------|--------------|--------|-------|-------------|-------------|------------|----------|
| pKABΔfna        | kanA-kanB-kanK        | gtmF-gtmK-gtmL | nemD   | kacA  | kanE        | kanC-kanD   | kanI-kacL  | aprD3D4  |
| pKABΔfnΔet2a    | kanA-kanB-kanK        | gtmF-gtmK-gtmL | nemD   | kacA  | tobM2       | kanC-kanD   | kanI-kacL  | aprD3D4  |
| **In vitro**    |                       |              |        |       |             |             |            |          |
| pKanF           | —                     | —            | kanF   |       |             |             |            |          |
| pKanI-KacL      | —                     | —            | —      | —     | —           | kanI-kacL   |            |          |
| pKanE           | —                     | —            | —      | —     | kanE        |             |            |          |
| pKanC-KanD      | —                     | —            | —      | —     | —           | kanC-kanD   |            |          |
| pAprD3-AprD4    | —                     | —            | —      | —     | —           | —           | —          | aprD3D4  |
| pKanFHis6       | —                     | —            | kanFHis6 | —     | —           |             |            |          |
| pKanEHis6       | —                     | —            | —      | —     | —           | kanEHis6    |            |          |

2DOS: 1-biosynthetic genes; Resistance: genes responsible for conferring resistance to kanamycin in S. venezuelae<sup>10</sup>; GT1: putative gene encoding first glycosyltransferase (kanF, nemD, tobM1, and gtmG from S. kanamyceticus, S. fradiae, S. tenebrarius, and M. echinospora, respectively); 2'DA: gene encoding 2'-deacetylase<sup>10</sup>; GT2: putative gene encoding second glycosyltransferase (kanE and tobM2 from S. kanamyceticus and S. tenebrarius, respectively); Kns: putative genes responsible for UDP-kanosamine biosynthesis; 6'AM: putative genes responsible for 6'-amination of pseudodi- and trisaccharides; AHBA: seven genes from B. circulans responsible for the biosynthesis and incorporation of the AHBA side chain; 3'DO: putative gene pair responsible for 3'-deoxygenation of pseudodi- or trisaccharides. “—” indicates that the specific gene was not added to the construct. His6 represents the expression vectors to obtain the recombinant His-tagged proteins.

Supplementary Results

4,6-disubstituted                                                                 4,5-disubstituted

![Chemical Structure 1](image1.png)                                                ![Chemical Structure 2](image2.png)

| Naturally-occurring | $R_1$ | $R_2$ | $R_3$ | $R_4$ | $R_5$ |
|---------------------|-------|-------|-------|-------|-------|
| Kanamycin A         | OH    | OH    | NH₂   | H     |       |
| Kanamycin B         | NH₂   | OH    | NH₂   | H     |       |
| Kanamycin C         | NH₂   | OH    | OH    | H     |       |
| Tobramycin          | NH₂   | H     | OH    | NH₂   | H     |

| $R_1$ | $R_2$ |
|-------|-------|
| Gentamicin C₁ | CH₃ | NHCH₃ |
| Gentamicin C₂ | CH₃ | NH₂   |
| Gentamicin C₁a | H   | NH₂   |

| $R_1$ | $R_2$ |
|-------|-------|
| Butirosin | AHBA | H      |
| Neomycin | H     | ring A |

| Semi-synthetic | $R_1$ | $R_2$ | $R_3$ | $R_4$ | $R_5$ |
|-----------------|-------|-------|-------|-------|-------|
| Amikacin        | OH    | OH    | OH    | NH₂   | AHBA  |
| Dibekacin       | NH₂   | H     | H     | NH₂   | H     |
| Arbekacin       | NH₂   | H     | H     | NH₂   | AHBA  |

---

**Supplementary Figure 1 | Structure of 2-deoxystreptamine (1)-containing aminoglycosides.** Chemical structure of the aminoglycosides. Kanamycins, tobramycin, and gentamicins are 4,6-disubstituted aminoglycosides, whereas butirosin and neomycin are 4,5-disubstituted aminoglycosides. They are all natural products, whereas amikacin (15), dibekacin, and arbekacin are semi-synthetic aminoglycosides derived from kanamycins. AHBA: S-4-amino-2-hydroxybutyric acid.

Page 15 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

a

b

c

Heterologously produced 1

Authentic 1

Supplementary Figure 2 | Structural elucidation of 2-deoxystreptamine (1). (a) ESI-MS/MS fragmentation pattern of 1, which was heterologously produced by an S. venezuelae strain (DOS) expressing kanA-kanB-kanK. (b) ESI-MS/MS spectra of 1, which was heterologously produced. (c) ¹H NMR spectra obtained from both heterologously produced and authentic 1. One liter of culture broth from a DOS strain was treated using a preparative HPLC column followed by OASIS MCX SPE cleanup and freeze-drying, as described in Supplementary Methods, yielding 1 as a white solid (2.5 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 1.1 min; HR-ESI-MS: $m/z$ 163.1882 [M+H]⁺ (calculated for C₆H₁₄N₂O₃, 163.1870); characteristic fragment ion generated by a cross-ring cleavage of C₁₆ at 84. The structure 1, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on multiple reaction monitoring (MRM) mode (163 > 84), ESI-MS/MS and NMR spectra with those of authentic 1.

Page 16 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 3 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for 2-deoxystreptamine (1) isolated from an S. venezuelae strain (DOS) expressing kanA-kanB-kanK (see Supplementary Fig. 2)

| Position | $\delta_{\text{H}}$ (m) | $\delta_{\text{C}}$ |
|----------|------------------------|---------------------|
| 1        | 2.58 (q)              | 52.21               |
| 2        | 1.68 (dt)             | 37.50               |
|          | 1.88 (dt)             |                     |
| 3        | 2.57 (q)              | 52.30               |
| 4        | 3.40 (t)              | 79.15               |
| 5        | 3.32 (t)              | 74.39               |
| 6        | 3.36 (t)              | 79.62               |

Supplementary Figure 3 | Structural elucidation of 2′-N-acetylparomamine (2). (a) ESI-MS/MS fragmentation pattern of 2, which was heterologously produced by an S. venezuelae strain (DOSf) expressing kanA-kanB-kanK-kanF. (b) ESI-MS/MS spectra of 2, which was heterologously produced. (c) 1H NMR spectra obtained from both heterologously produced and authentic 2. Two liters of culture broth from a DOSf strain were treated as described in Supplementary Methods, yielding 2 as a pale yellow solid (0.7 mg). Tretention on HPLC-ESI-MS/MS: 9.6 min; HR-ESI-MS: m/z 366.1807 [M+H]+ (calculated for C14H27N3O8, 366.1795); characteristic fragment ions generated by a glycosidic link cleavage at 163 and 204. The structure of 2, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (366 > 163), ESI-MS/MS and NMR spectra with those of authentic 2, which was previously characterized^10.

Page 18 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 4 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for 2′-N-acetylparomamine (2) isolated from an S. venezuelae strain (DOSf) expressing kanA-kanB-kanK-kanF (see Supplementary Fig. 3)

| Position | $\delta_{\text{H}}$ (m) | $\delta_{\text{C}}$ |
|----------|-------------------------|--------------------|
| 1        | 2.67 (q)               | 51.20              |
| 2        | 1.57 (dt)              | 35.43              |
|          | 1.88 (dt)              |                    |
| 3        | 2.76 (q)               | 49.32              |
| 4        | 3.12 (t)               | 86.13              |
| 5        | 3.50 (t)               | 72.38              |
| 6        | 3.43 (t)               | 78.14              |
| 1'       | 5.53 (d)               | 101.41             |
| 2'       | 4.11 (t)               | 64.13              |
| 3'       | 4.03 (t)               | 71.82              |
| 4'       | 3.41 (t)               | 73.12              |
| 5'       | 3.73 (m)               | 77.96              |
| 6'       | 3.47 (m)               | 62.23              |
|          | 3.72 (m)               |                    |
| 7'       | —                      | 169.93             |
| 8'       | 2.22 (s)               | 23.23              |

Supplementary Figure 4 | Structural elucidation of paromamine (3). (a) ESI-MS/MS fragmentation pattern of 3, which was heterologously produced by an S. venezuelae strain (PAR) expressing kanA-kanB-kanK-kanF-kacA. (b) ESI-MS/MS spectra of 3, which was heterologously produced. (c) 1H NMR spectra obtained from both heterologously produced and authentic 3. One liter of culture broth from a PAR strain was treated as described in Supplementary Methods, yielding 3 as a white solid (0.8 mg). Tretention on HPLC-ESI-MS/MS: 10.6 min; HR-ESI-MS: m/z 324.3440 [M+H]+ (calculated for C12H25N3O7, 324.3428); characteristic fragment ions generated by a glycosidic link cleavage at 162 and 163. The structure of 3, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (324 > 163), ESI-MS/MS and NMR spectra with those of authentic 3, which was previously characterized<sup>10</sup>.

Page 20 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 5 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for paromamine (3) isolated from an S. venezuelae strain (PAR) expressing kanA-kanB-kanK-kanF-kacA (see Supplementary Fig. 4)

| Position | $\delta_{\text{H}}$ (m) | $\delta_{\text{C}}$ |
|----------|------------------------|--------------------|
| 1        | 2.58 (q)               | 51.21              |
| 2        | 1.28 (dt)              | 35.50              |
|          | 1.68 (dt)              |                    |
| 3        | 2.57 (q)               | 49.30              |
| 4        | 3.43 (m)               | 85.95              |
| 5        | 3.32 (t)               | 72.39              |
| 6        | 3.49 (t)               | 78.12              |
| 1'       | 5.18 (d)               | 102.08             |
| 2'       | 3.13 (m)               | 55.47              |
| 3'       | 3.62 (t)               | 74.00              |
| 4'       | 3.41 (t)               | 72.55              |
| 5'       | 3.75 (m)               | 77.37              |
| 6'       | 3.52 (m)               | 62.21              |
|          | 3.78 (m)               |                    |

Supplementary Figure 5 | Structural elucidation of neamine (4). (a) ESI-MS/MS fragmentation pattern of 4, which was heterologously produced by an S. venezuelae strain (PARil) expressing kanA-kanB-kanK-kanF-kacA-kanL-kacL. (b) ESI-MS/MS spectra of 4, which was heterologously produced. (c) ${}^{1}$H NMR spectra obtained from both heterologously produced and authentic 4. Two liters of culture broth from a PARil strain were treated as described in Supplementary Methods, yielding 4 as a white solid (0.9 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 17.2 min; HR-ESI-MS: $m/z$ 323.3664 [M+H]${}^{+}$ (calculated for C${}_{12}$H${}_{26}$N${}_{4}$O${}_{6}$, 323.3646); characteristic fragment ions generated by a glycosidic link cleavage at 161 and 163. The structure of 4, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (323 > 163), ESI-MS/MS and NMR spectra with those of authentic 4, which was prepared from neomycin by methanolysis${}^{1}$.

Page 22 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 6 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for neamine (4) isolated from an S. venezuelae strain (PARil) expressing kanA-kanB-kanK-kanF-kacA-kanI-kacL (see Supplementary Fig. 5)

| Position | $\delta_{\text{H}}$ (m) | $\delta_{\text{C}}$ |
|----------|-------------------------|--------------------|
| 1        | 2.57 (q)               | 51.21              |
| 2        | 1.29 (dt)              | 35.53              |
|          | 1.69 (dt)              |                    |
| 3        | 2.57 (q)               | 49.25              |
| 4        | 3.40 (m)               | 86.01              |
| 5        | 3.32 (t)               | 72.40              |
| 6        | 3.42 (t)               | 78.12              |
| 1'       | 5.18 (d)               | 101.76             |
| 2'       | 3.13 (m)               | 55.22              |
| 3'       | 3.64 (t)               | 73.73              |
| 4'       | 3.40 (t)               | 73.10              |
| 5'       | 3.91 (m)               | 72.21              |
| 6'       | 2.64 (m)               | 41.86              |
|          | 2.89 (m)               |                    |

Supplementary Figure 6 | Structural elucidation of 3″-deamino-3″-hydroxykanamycin C (5). (a) ESI-MS/MS fragmentation pattern of 5, which was heterologously produced by an S. venezuelae strain (KCXΔcΔd) expressing kanA-kanB-kanK-kanF-kacA-kanE. (b) ESI-MS/MS spectra of 5, which was heterologously produced. (c) ¹H NMR spectra obtained from both heterologously produced and wild-type produced 5. One liter of culture broth from a KCXΔcΔd strain was treated as described in Supplementary Methods, yielding 5 as a pale yellow solid (1.2 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 14.6 min; HR-ESI-MS: $m/z$ 486.4887 [M+H]⁺ (calculated for C₁₈H₃₅N₃O₁₂, 486.4870); characteristic fragment ions generated by 4,6-glycosidic link cleavages at 162, 163 and 324. The structure of 5, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (486 > 163), ESI-MS/MS and NMR spectra with those of wild-type produced 5, which was isolated from the wild-type S. kanamyceticus ATCC 12853 and recently characterized³².

Page 24 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 7 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for 3″-deamino-3″-hydroxykanamycin C (5) isolated from an S. venezuelae strain (KCXΔcΔd) expressing kanA-kanB-kanK-kanF-kacA-kanE (see Supplementary Fig. 6)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.57 (q)               | 49.37              |
| 2        | 1.26 (dt)              | 35.72              |
|          | 1.66 (dt)              |                    |
| 3        | 2.56 (q)               | 49.19              |
| 4        | 3.42 (m)               | 86.52              |
| 5        | 3.32 (t)               | 70.44              |
| 6        | 3.44 (t)               | 86.44              |
| 1′       | 5.18 (d)               | 102.07             |
| 2′       | 3.12 (m)               | 55.26              |
| 3′       | 3.64 (t)               | 74.35              |
| 4′       | 3.42 (t)               | 72.57              |
| 5′       | 3.76 (m)               | 77.43              |
| 6′       | 3.59 (m)               | 62.20              |
|          | 3.84 (m)               |                    |
| 1″       | 5.03 (d)               | 102.57             |
| 2″       | 3.77 (t)               | 74.23              |
| 3″       | 3.48 (m)               | 73.41              |
| 4″       | 3.40 (t)               | 71.40              |
| 5″       | 3.71 (m)               | 77.59              |
| 6″       | 3.53 (m)               | 62.35              |
|          | 3.78 (m)               |                    |

a

b

c

Heterologously produced 6

Authentic 6

Supplementary Figure 7 | Structural elucidation of kanamycin C (6). (a) ESI-MS/MS fragmentation pattern of 6, which was heterologously produced by an S. venezuelae strain (KCX) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD. (b) ESI-MS/MS spectra of 6, which was heterologously produced. (c) ${}^{1}$H NMR spectra obtained from both heterologously produced and authentic 6. A liter and five hundred milliliters of culture broth from a KCX strain were treated as described in Supplementary Methods, yielding 6 as a pale yellow solid (1.1 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 19.2 min; HR-ESI-MS: $m/z$ 485.5073 [M+H]${}^{+}$ (calculated for $\mathrm{C}_{18} \mathrm{H}_{36} \mathrm{~N}_{4} \mathrm{O}_{11}$, 485.5052); characteristic fragment ions generated by both 4,6-glycosidic link cleavages and a cross-ring cleavage of the kanosamine moiety at 162, 163, 205 and 324. The structure of 6, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (485 > 163), ESI-MS/MS and NMR spectra with those of authentic 6, which was isolated from authentic kanamycin.

Page 26 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 8 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for kanamycin C (6) isolated from an S. venezuelae strain (KCX) expressing kanA-kanB-kank-kanF-kacA-kanE-kanC-kanD (see Supplementary Fig. 7)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.57 (q)               | 49.30              |
| 2        | 1.27 (dt)              | 35.71              |
|          | 1.67 (dt)              |                    |
| 3        | 2.53 (q)               | 49.23              |
| 4        | 3.42 (m)               | 86.57              |
| 5        | 3.33 (t)               | 70.41              |
| 6        | 3.47 (t)               | 86.49              |
| 1'       | 5.18 (d)               | 101.98             |
| 2'       | 3.13 (m)               | 55.52              |
| 3'       | 3.64 (t)               | 74.12              |
| 4'       | 3.42 (t)               | 72.56              |
| 5'       | 3.76 (m)               | 77.39              |
| 6'       | 3.52 (m)               | 62.18              |
|          | 3.78 (m)               |                    |
| 1"       | 5.03 (d)               | 103.70             |
| 2"       | 3.86 (m)               | 72.11              |
| 3"       | 2.88 (m)               | 54.48              |
| 4"       | 3.54 (t)               | 72.47              |
| 5"       | 3.76 (m)               | 78.82              |
| 6"       | 3.53 (m)               | 61.88              |
|          | 3.78 (m)               |                    |

a

b

163.0

323.1

160.9

205.1

143.6

365.3

483.9

100 200 300 400 500 $m/z$

Amine

c

4,6

3', 4'

8' 6"

Alcohol

5'

2", 5", 6"

5

2'

6'

3"

1, 3

6'

2

Heterologously produced 7

Authentic 7

5 4 δ 3 2 1

Supplementary Figure 8 | Structural elucidation of kanamycin B (7). (a) ESI-MS/MS fragmentation pattern of 7, which was heterologously produced by an S. venezuelae strain (KAB) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD-kanI-kacL. (b) ESI-MS/MS spectra of 7, which was heterologously produced. (c) ${}^{1}$H NMR spectra obtained from both heterologously produced and authentic 7. A liter of culture broth from a KAB strain was treated as described in Supplementary Methods, yielding 7 as a pale yellow solid (0.7 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 26.9 min; HR-ESI-MS: $m/z$ 484.5159 [M+H]${}^{+}$ (calculated for C${}_{18}$H${}_{37}$N${}_{5}$O${}_{10}$, 484.5139); characteristic fragment ions generated by both 4,6-glycosidic link cleavages and a cross-ring cleavage of the kanosamine moiety at 161, 163, 205 and 323. The structure of 7, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (484 > 163), ESI-MS/MS and NMR spectra with those of authentic 7 (kanamycin B).

Page 28 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 9 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for kanamycin B (7) isolated from an S. venezuelae strain (KAB) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD-kanI-kacL (see Supplementary Fig. 8)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.57 (q)               | 49.37              |
| 2        | 1.29 (dt)              | 35.70              |
|          | 1.69 (dt)              |                    |
| 3        | 2.54 (q)               | 49.19              |
| 4        | 3.40 (m)               | 86.49              |
| 5        | 3.32 (t)               | 70.49              |
| 6        | 3.42 (t)               | 86.40              |
| 1'       | 5.17 (d)               | 101.41             |
| 2'       | 3.13 (m)               | 55.43              |
| 3'       | 3.64 (t)               | 73.76              |
| 4'       | 3.40 (t)               | 73.24              |
| 5'       | 3.90 (m)               | 72.15              |
| 6'       | 2.65 (m)               | 41.90              |
|          | 2.90 (m)               |                    |
| 1"       | 5.03 (d)               | 103.73             |
| 2"       | 3.88 (m)               | 72.11              |
| 3"       | 2.88 (m)               | 54.42              |
| 4"       | 3.55 (t)               | 72.40              |
| 5"       | 3.76 (m)               | 78.78              |
| 6"       | 3.53 (m)               | 61.80              |
|          | 3.79 (m)               |                    |

Heterologously produced 8

Authentic 8

Supplementary Figure 9 | Structural elucidation of kanamycin A (8). (a) ESI-MS/MS fragmentation pattern of 8, which was heterologously produced by an S. venezuelae strain (KAB) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD-kanI-kacL. (b) ESI-MS/MS spectra of 8, which was heterologously produced. (c) 1H NMR spectra obtained from both heterologously produced and authentic 8. A liter of culture broth from a KAB strain was treated as described in Supplementary Methods, yielding 8 as a pale yellow solid (2.0 mg). Tretention on HPLC-ESI-MS/MS: 15.6 min; HR-ESI-MS: m/z 485.4999 [M+H]+ (calculated for C18H36N4O11, 485.4986); characteristic fragment ions generated by both 4,6-glycosidic link cleavages and a cross-ring cleavage of the kanosamine moiety at 162, 163, 205 and 324. The structure of 8, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (485 > 163), ESI-MS/MS and NMR spectra with those of authentic 8, which was isolated from the authentic kanamycin.

Page 30 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 10 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for kanamycin A (8) isolated from an S. venezuelae strain (KAB) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD-kanI-kacL (see Supplementary Fig. 9)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.57 (q)               | 49.21               |
| 2        | 1.27 (dt)              | 35.78               |
|          | 1.68 (dt)              |                     |
| 3        | 2.56 (q)               | 49.26               |
| 4        | 3.40 (t)               | 86.59               |
| 5        | 3.32 (t)               | 70.40               |
| 6        | 3.44 (t)               | 86.50               |
| 1'       | 5.07 (d)               | 102.12              |
| 2'       | 3.73 (t)               | 74.05               |
| 3'       | 3.49 (t)               | 73.12               |
| 4'       | 3.40 (t)               | 72.76               |
| 5'       | 3.91 (m)               | 72.11               |
| 6'       | 2.65 (m)               | 41.83               |
|          | 2.90 (m)               |                     |
| 1"       | 5.07 (d)               | 103.61              |
| 2"       | 3.86 (m)               | 72.11               |
| 3"       | 2.85 (m)               | 54.39               |
| 4"       | 3.51 (t)               | 72.50               |
| 5"       | 3.75 (m)               | 78.87               |
| 6"       | 3.54 (m)               | 61.59               |
|          | 3.79 (m)               |                     |

a

b

162.9

144.8

126.9

187.0

205.1

288.9

325.2

$m/z$

c

Alcohol

Heterologously produced 9

Wild-type produced 9

Amine

Supplementary Figure 10 | Structural elucidation of 2′-deamino-2′-hydroxyparomamine (9).

(a) ESI-MS/MS fragmentation pattern of 9, which was heterologously produced by an S.

venezuelae strain (DOSf) expressing kanA-kanB-kanK-kanF. (b) ESI-MS/MS spectra of 9, which

was heterologously produced. (c) ${}^{1}$H NMR spectra obtained from both heterologously produced

and wild-type produced 9. A liter of culture broth from a DOSf strain was treated as described in

Supplementary Methods, yielding 9 as a white solid (1.8 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS:

7.0 min; HR-ESI-MS: $m/z$ 325.3294 [M+H]${}^{+}$ (calculated for C${}_{12}$H${}_{24}$N${}_{2}$O${}_{8}$, 325.3276); characteristic

fragment ion generated by a glycosidic link cleavage at 163. The structure of 9, which was

heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-

MS/MS captured on MRM (325 > 163), ESI-MS/MS and NMR spectra with those of wild-type

produced 9, which was previously characterized ${}^{14}$ and isolated herein from S. kanamyceticus

ATCC 12853.

Page 32 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 11 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for 2′-deamino-2′-hydroxyparomamine (9) isolated from an S. venezuelae strain (DOSf) expressing kanA-kanB-kanK-kanF (see Supplementary Fig. 10)

| Position | $\delta_{\text{H}}$ (m) | $\delta_{\text{C}}$ |
|----------|------------------------|--------------------|
| 1        | 2.57 (q)              | 51.20              |
| 2        | 1.27 (dt)             | 35.49              |
|          | 1.68 (dt)             |                    |
| 3        | 2.56 (q)              | 49.32              |
| 4        | 3.42 (t)              | 86.23              |
| 5        | 3.30 (t)              | 72.38              |
| 6        | 3.49 (t)              | 78.18              |
| 1'       | 5.08 (d)              | 102.71             |
| 2'       | 3.75 (t)              | 74.13              |
| 3'       | 3.59 (t)              | 73.42              |
| 4'       | 3.40 (t)              | 71.62              |
| 5'       | 3.76 (m)              | 77.69              |
| 6'       | 3.51 (m)              | 62.30              |
|          | 3.77 (m)              |                    |

Supplementary Figure 11 | Structural elucidation of 2′-deamino-2′-hydroxyneamine (10).

(a) ESI-MS/MS fragmentation pattern of 10, which was heterologously produced by an S. venezuelae strain (PARil) expressing kanA-kanB-kanK-kanF-kacA-kanL-kacL. (b) ESI-MS/MS spectra of 10, which was heterologously produced. (c) ¹H NMR spectra obtained from both heterologously produced and wild-type produced 10. A liter of culture broth from a PARil strain was treated as described in Supplementary Methods, yielding 10 as a pale yellow solid (1.0 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 10.6 min; HR-ESI-MS: $m/z$ 325.3290 [M+H]⁺ (calculated for C₁₂H₂₅N₃O₇, 325.3276); characteristic fragment ions generated by a glycosidic link cleavage at 162 and 163. The structure of 10, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (325 > 163), ESI-MS/MS and NMR spectra with those of wild-type produced 10, which was previously characterized¹³ and isolated herein from S. kanamyceticus ATCC 12853.

Supplementary Table 12 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for 2′-deamino-2′-hydroxyneamine (10) isolated from an S. venezuelae strain (PARil) expressing kanA-kanB-kanK-kanF-kacA-kanL-kacL (see Supplementary Fig. 11)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.58 (q)               | 51.18              |
| 2        | 1.27 (dt)              | 35.50              |
|          | 1.69 (dt)              |                    |
| 3        | 2.55 (q)               | 49.25              |
| 4        | 3.42 (t)               | 86.27              |
| 5        | 3.31 (t)               | 72.40              |
| 6        | 3.44 (t)               | 78.18              |
| 1'       | 5.08 (d)               | 102.44             |
| 2'       | 3.73 (t)               | 74.25              |
| 3'       | 3.49 (t)               | 73.19              |
| 4'       | 3.40 (t)               | 72.64              |
| 5'       | 3.89 (m)               | 72.43              |
| 6'       | 2.62 (m)               | 41.81              |
|          | 2.87 (m)               |                    |

![](image.png)

Supplementary Figure 12 | Structural elucidation of 3″-deamino-3″-hydroxykanamycin X (11). (a) ESI-MS/MS fragmentation pattern of 1, which was heterologously produced by an S. venezuelae strain (KCXΔcΔd) expressing kanA-kanB-kanK-kanF-kacA-kanE. (b) ESI-MS/MS spectra of 11, which was heterologously produced. (c) ¹H NMR spectra obtained from both heterologously produced and wild-type produced 11. A liter of culture broth from a KCXΔcΔd strain was treated as described in Supplementary Methods, yielding 11 as a white solid (1.4 mg). T<sub>retention</sub> on HPLC-ESI-MS/MS: 9.4 min; HR-ESI-MS: m/z 487.4701 [M+H]<sup>+</sup> (calculated for C<sub>18</sub>H<sub>34</sub>N<sub>2</sub>O<sub>13</sub>, 487.4682); characteristic fragment ions generated by 4,6-glycosidic link cleavages at 163 and 325. The structure of 11, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (487 > 163), ESI-MS/MS and NMR spectra with those of wild-type produced 11, which was previously characterized<sup>14</sup> and isolated herein from S. kanamyceticus ATCC 12853.

Page 36 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 13 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for 3″-deamino-3″-hydroxykanamycin X (11) isolated from an S. venezuelae strain (KCXΔcΔd) expressing kanA-kanB-kanK-kanF-kacA-kanE (see Supplementary Fig. 12)

| Position | $\delta_{\text{H}}$ (m) | $\delta_{\text{C}}$ |
|----------|-------------------------|--------------------|
| 1        | 2.58 (q)               | 49.30              |
| 2        | 1.27 (dt)              | 35.81              |
|          | 1.67 (dt)              |                    |
| 3        | 2.57 (q)               | 49.19              |
| 4        | 3.42 (t)               | 86.59              |
| 5        | 3.31 (t)               | 70.48              |
| 6        | 3.42 (t)               | 86.45              |
| 1′       | 5.03 (d)               | 102.37             |
| 2′       | 3.73 (t)               | 73.99              |
| 3′       | 3.47 (t)               | 73.12              |
| 4′       | 3.38 (t)               | 71.39              |
| 5′       | 3.75 (m)               | 77.68              |
| 6′       | 3.54 (m)               | 62.17              |
|          | 3.80 (m)               |                    |
| 1″       | 5.03 (d)               | 102.52             |
| 2″       | 3.77 (t)               | 74.20              |
| 3″       | 3.49 (m)               | 73.45              |
| 4″       | 3.40 (t)               | 71.46              |
| 5″       | 3.71 (m)               | 71.78              |
| 6″       | 3.52 (m)               | 62.29              |
|          | 3.76 (m)               |                    |

Heterologously produced 12  
Wild-type produced 12  

Supplementary Figure 13 | Structural elucidation of kanamycin X (12). (a) ESI-MS/MS fragmentation pattern of 12, which was heterologously produced by an S. venezuelae strain (KCX) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD. (b) ESI-MS/MS spectra of 12, which was heterologously produced. (c) ${}^{1}$H NMR spectra obtained from both heterologously produced and wild-type produced 12. A liter of culture broth from a KCX strain was treated as described in Supplementary Methods, yielding 12 as a pale yellow solid (2.0 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 11.3 min; HR-ESI-MS: $m/z$ 486.4850 [M+H]${}^{+}$ (calculated for C${}_{18}$H${}_{35}$N${}_{3}$O${}_{12}$, 486.4834); characteristic fragment ions generated by both 4,6-glycosidic link cleavages and a cross-ring cleavage of the kanosamine moiety at 163, 205 and 325. The structure of 12, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (486 > 163), ESI-MS/MS and NMR spectra with those of wild-type produced 12, which was previously characterized ${}^{15}$ and isolated herein from S. kanamyceticus ATCC 12853.

Page 38 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 14 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for kanamycin X (12) isolated from an S. venezuelae strain (KCX) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD (see Supplementary Fig. 13)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.57 (q)               | 49.31              |
| 2        | 1.29 (dt)              | 35.78              |
|          | 1.69 (dt)              |                    |
| 3        | 2.54 (q)               | 49.27              |
| 4        | 3.42 (t)               | 86.57              |
| 5        | 3.30 (t)               | 70.44              |
| 6        | 3.47 (t)               | 86.45              |
| 1'       | 5.03 (d)               | 102.54             |
| 2'       | 3.73 (t)               | 74.06              |
| 3'       | 3.51 (t)               | 73.41              |
| 4'       | 3.39 (t)               | 71.39              |
| 5'       | 3.75 (m)               | 77.73              |
| 6'       | 3.51 (m)               | 62.20              |
|          | 3.78 (m)               |                    |
| 1"       | 5.03 (d)               | 103.64             |
| 2"       | 3.88 (m)               | 72.08              |
| 3"       | 2.89 (m)               | 54.44              |
| 4"       | 3.55 (t)               | 72.37              |
| 5"       | 3.76 (m)               | 78.82              |
| 6"       | 3.54 (m)               | 61.69              |
|          | 3.79 (m)               |                    |

a

b

MH⁺ = 485

c

Heterologously produced **13**

Authentic **13**

Supplementary Figure 14 | Structural elucidation of 3″-deamino-3″-hydroxykanamycin B (**13**). (a) ESI-MS/MS fragmentation pattern of **13**, which was heterologously produced by an *S*. venezuelae strain (KABΔcΔd) expressing *kanA-kanB-kanK-kanF-kacA-kanE-kanI-kacL*. (b) ESI-MS/MS spectra of **13**, which was heterologously produced. (c) ¹H NMR spectra obtained from both heterologously produced and authentic **13**. A liter of culture broth from a KABΔcΔd strain was treated as described in Supplementary Methods, yielding **13** as a pale yellow solid (0.3 mg). *T*<sub>retention</sub> on HPLC-ESI-MS/MS: 18.2 min; HR-ESI-MS: *m/z* 485.5002 [M+H]<sup>+</sup> (calculated for C<sub>18</sub>H<sub>36</sub>N<sub>4</sub>O<sub>11</sub>, 485.4986); characteristic fragment ions generated by 4,6-glycosidic link cleavages at 161, 163 and 323. The structure of **13**, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (485 > 163), ESI-MS/MS and NMR spectra with those of authentic **13**, which was isolated from wild-type *S*. tenebrarius ATCC 17920 and previously characterized as nebramycin factor 3<sup>33</sup>.

Page 40 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 15 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for 3″-deamino-3″-hydroxykanamycin B (13) isolated from an S. venezuelae strain (KABΔcΔd) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanI-kacL (see Supplementary Fig. 14)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.57 (q)               | 49.32              |
| 2        | 1.29 (dt)              | 35.72              |
|          | 1.68 (dt)              |                    |
| 3        | 2.55-58 (u)*           | 49.22              |
| 4        | 3.40 (m)               | 86.50              |
| 5        | 3.33 (t)               | 70.41              |
| 6        | 3.42 (t)               | 86.44              |
| 1′       | 5.18 (d)               | 101.32             |
| 2′       | 3.13 (m)               | 55.43              |
| 3′       | 3.65 (t)               | 73.67              |
| 4′       | 3.41 (t)               | 73.14              |
| 5′       | 3.91 (m)               | 72.35              |
| 6′       | 2.64 (m)               | 41.85              |
|          | 2.89 (m)               |                    |
| 1″       | 5.02 (d)               | 102.63             |
| 2″       | 3.69-72 (u)*           | 74.33              |
| 3″       | 3.49 (m)               | 73.41              |
| 4″       | 3.39 (t)               | 71.50              |
| 5″       | 3.69-73 (u)*           | 77.46              |
| 6″       | 3.54 (m)               | 62.28              |
|          | 3.79 (m)               |                    |

* Undistinguishable signals due to the poor S/N ratio.

a

b

MH⁺ = 486

c

Heterologously produced **14**

Wild-type produced **14**

Supplementary Figure 15 | Structural elucidation of kanamycin D (**14**). (a) ESI-MS/MS fragmentation pattern of **14**, which was heterologously produced by an *S*. venezuelae strain (KABΔcΔd) expressing *kanA-kanB-kanK-kanF-kacA-kanE-kanL-kacL*. (b) ESI-MS/MS spectra of **14**, which was heterologously produced. (c) ${}^{1}$H NMR spectra obtained from both heterologously produced and wild-type produced **14**. A liter of culture broth from a KABΔcΔd strain was treated as described in Supplementary Methods, yielding **14** as a pale yellow solid (0.6 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 11.8 min; HR-ESI-MS: $m/z$ 486.4855 [M+H]⁺ (calculated for C₁₈H₃₅N₃O₁₂, 486.4834); characteristic fragment ions generated by 4,6-glycosidic link cleavage at 162, 163 and 324. The structure of **14**, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (486 > 163), ESI-MS/MS and NMR spectra with those of wild-type produced **14**, which was previously characterized¹⁷ and isolated herein from *S*. kanamyceticus ATCC 12853.

Page 42 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 16 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for kanamycin D (14) isolated from an S. venezuelae strain (KAB$\Delta$c$\Delta$d) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanI-kacL (see Supplementary Fig. 15)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.57 (q)               | 49.28              |
| 2        | 1.26 (dt)              | 35.80              |
|          | 1.67 (dt)              |                    |
| 3        | 2.55 (q)               | 49.17              |
| 4        | 3.40 (t)               | 86.52              |
| 5        | 3.32 (t)               | 70.40              |
| 6        | 3.46 (t)               | 86.45              |
| 1'       | 5.06 (d)               | 102.19             |
| 2'       | 3.72 (t)               | 74.06              |
| 3'       | 3.47 (t)               | 73.39              |
| 4'       | 3.40 (t)               | 72.87              |
| 5'       | 3.90 (m)               | 72.17              |
| 6'       | 2.63 (m)               | 41.77              |
|          | 2.88 (m)               |                    |
| 1"       | 5.06 (d)               | 102.50             |
| 2"       | 3.73 (t)               | 74.32              |
| 3"       | 3.49 (m)               | 73.78              |
| 4"       | 3.39 (t)               | 71.46              |
| 5"       | 3.76 (m)               | 71.83              |
| 6"       | 3.52 (m)               | 62.20              |
|          | 3.77 (m)               |                    |

Heterologously produced **15**

Authentic **15**

Supplementary Figure 16 | Structural elucidation of amikacin (**15**). (a) ESI-MS/MS fragmentation pattern of **15**, which was heterologously produced by an *S. venezuelae* strain (KABb) expressing *kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD-kanI-kacL* together with *btrG-btrH-btrI-btrJ-btrK-btrO-btrV*. (b) ESI-MS/MS spectra of **15**, which was heterologously produced. (c) ${}^{1}$H NMR spectra obtained from both heterologously produced and authentic **15**. Two liters of culture broth from a KABb strain were treated as described in Supplementary Methods, yielding **15** as a pale yellow solid (0.7 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 13.8 min; HR-ESI-MS: $m/z$ 586.6044 [M+H]${}^{+}$ (calculated for C${}_{22}$H${}_{43}$N${}_{5}$O${}_{13}$, 586.6025); characteristic fragment ions generated by both 4,6-glycosidic link cleavages and a cross-ring cleavage of the kanosamine moiety at 162, 163, 264 and 306. The structure of **15**, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (586 > 264), ESI-MS/MS and NMR spectra with those of authentic **15** (amikacin).

Page 44 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Table 17 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for amikacin (15) isolated from an S. venezuelae strain (KABb) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD-kanI-kacL together with btG-btrH-btrl-btrJ-btrK-btrO-btrV (see Supplementary Fig. 16)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.55 (q)               | 49.02              |
| 2        | 1.29 (dt)              | 35.67              |
|          | 1.68 (dt)              |                    |
| 3        | 2.54 (q)               | 49.26              |
| 4        | 3.40 (t)               | 86.54              |
| 5        | 3.32 (t)               | 70.40              |
| 6        | 3.44 (t)               | 86.44              |
| 1'       | 5.07 (d)               | 102.12             |
| 2'       | 3.70 (t)               | 74.05              |
| 3'       | 3.49 (t)               | 73.40              |
| 4'       | 3.40 (t)               | 72.76              |
| 5'       | 3.91 (m)               | 72.11              |
| 6'       | 2.63 (m)               | 41.83              |
|          | 2.89 (m)               |                    |
| 1"       | 5.03 (d)               | 103.61             |
| 2"       | 3.84 (m)               | 71.99              |
| 3"       | 2.87 (m)               | 54.19              |
| 4"       | 3.51 (t)               | 72.47              |
| 5"       | 3.75 (m)               | 78.87              |
| 6"       | 3.54 (m)               | 61.64              |
|          | 3.78 (m)               |                    |
| A        | —                      | 177.10             |
| B        | 4.14 (t)               | 69.88              |
| C        | 1.96 (q)               | 37.02              |
| D        | 2.70 (m)               | 34.26              |

MH⁺ = 468

Supplementary Figure 17 | Structural elucidation of tobramycin (16). (a) ESI-MS/MS fragmentation pattern of 16, which was heterologously produced by an S. venezuelae strain (KABΔfnΔet₂a) expressing kanA-kanB-kanK-nemD-kacA-tobM2-kanC-kanD-kanI-kacL together with aprD3-aprD4. (b) ESI-MS/MS spectra of 16, which was heterologously produced. $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 27.4 min; HR-ESI-MS: $m/z$ 468.5167 [M+H]⁺ (calculated for C₁₈H₃₇N₅O₉, 468.5145); characteristic fragment ions generated by both 4,6-glycosidic link cleavage and a cross-ring cleavage of the kanosamine moiety at 145, 162, 163, 205, 307 and 324. The structure of 16, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (468 > 163) and ESI-MS/MS spectra with those of authentic 16 (tobramycin).

Heterologously produced 17

Authentic 15

Supplementary Figure 18 | Structural elucidation of 1-N-AHBA-kanamycin X (17). (a) ESI-MS/MS fragmentation pattern of 17, which was heterologously produced by an S. venezuelae strain (KCXb) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD together with btrG-btrH-btrl-btrJ-btrK-btrO-btrV. (b) ESI-MS/MS spectra of 17, which was heterologously produced. (c) ¹H NMR spectra obtained from both heterologously produced and authentic 15. Two liters of culture broth from a KCXb strain were treated as described in Supplementary Methods, yielding 17 as a pale yellow solid (0.8 mg). $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 9.6 min; HR-ESI-MS: $m/z$ 587.5894 [M+H]⁺ (calculated for C₂₂H₄₂N₄O₁₄, 587.5873); characteristic fragment ions generated by both 4,6-glycosidic link cleavages and a cross-ring cleavage of the kanosamine moiety at 163, 264, 306 and 425. The structure of 17, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (587 > 264), ESI-MS/MS and NMR spectra with those of authentic 15 (amikacin).

Supplementary Table 18 | $^1$H and $^{13}$C NMR data (500 MHz, D$_2$O) for 1-N-AHBA-kanamycin X (17) isolated from an S. venezuelae strain (KCXb) expressing kanA-kanB-kanK-kanF-kacA-kanE-kanC-kanD together with btrG-btrH-btrl-btrJ-btrK-btrO-btrV (see Supplementary Fig. 18)

| Position | $\delta_{\mathrm{H}}$ (m) | $\delta_{\mathrm{C}}$ |
|----------|-------------------------|---------------------|
| 1        | 2.57 (q)               | 49.11              |
| 2        | 1.28 (dt)              | 35.58              |
|          | 1.69 (dt)              |                    |
| 3        | 2.54 (q)               | 49.27              |
| 4        | 3.40 (t)               | 86.57              |
| 5        | 3.30 (t)               | 70.41              |
| 6        | 3.43 (t)               | 86.44              |
| 1'       | 5.03 (d)               | 102.54             |
| 2'       | 3.73 (t)               | 74.06              |
| 3'       | 3.51 (t)               | 73.41              |
| 4'       | 3.39 (t)               | 71.39              |
| 5'       | 3.75 (m)               | 77.73              |
| 6'       | 3.51 (m)               | 62.20              |
|          | 3.78 (m)               |                    |
| 1"       | 5.01 (d)               | 103.03             |
| 2"       | 3.84 (m)               | 71.97              |
| 3"       | 2.89 (m)               | 54.04              |
| 4"       | 3.55 (t)               | 72.37              |
| 5"       | 3.76 (m)               | 78.82              |
| 6"       | 3.54 (m)               | 61.69              |
|          | 3.79 (m)               |                    |
| A        | —                      | 176.90             |
| B        | 4.14 (t)               | 69.88              |
| C        | 1.97 (q)               | 37.10              |
| D        | 2.65 (m)               | 34.31              |

Supplementary Figure 19 | Structural elucidation of nebramine (18). (a) ESI-MS/MS fragmentation pattern of 18, which was heterologously produced by an S. venezuelae strain (KABΔfna) expressing kanA-kanB-kanK-nemD-kacA-kanE-kanC-kanD-kanI-kacL together with aprD3-aprD4. (b) ESI-MS/MS spectra of 18, which was heterologously produced. $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 29.0 min; HR-ESI-MS: $m/z$ 307.3605 [M+H]$^+$ (calculated for C$_{12}$H$_{26}$N$_4$O$_5$, 307.3586); characteristic fragment ion generated by glycosidic link cleavage at 163. The structure of 18, which was heterologously produced was assigned by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (307 > 163) and ESI-MS/MS spectra with those of authentic 18, which was isolated from wild-type S. tenebrarius ATCC 17920 and previously characterized as nebramycin factor 8${}^{33}$. The previously reported characterization data was not sufficient to serve as a reference for direct comparison of retention time and MS/MS fragmentation pattern.

$\text{a}$

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

$\text{b}$

![MS/MS Spectra](https://i.imgur.com/ms_ms_spectra.png)

$\text{MH}^{+}=469$

**Supplementary Figure 20 | Structural estimation of 3′-deoxykanamycin C (19).** (a) ESI-MS/MS fragmentation pattern of **19**, which was heterologously produced by an *S. venezuelae* strain (KABΔfna) expressing *kanA-kanB-kanK-nemD-kacA-kanE-kanC-kanD-kanI-kacL* together with *aprD3-aprD4*. (b) ESI-MS/MS spectra of **19**, which was heterologously produced. $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 20.8 min; HR-ESI-MS: $m/z$ 469.5012 [M+H]$^{+}$ (calculated for $\mathrm{C}_{18}\mathrm{H}_{36}\mathrm{N}_{4}\mathrm{O}_{10}$, 469.4992); characteristic fragment ions generated by both 4,6-glycosidic link cleavages and a cross-ring cleavage of the kanosamine moiety at 145, 162, 163, 205 and 324. The structure of **19**, which was heterologously produced was proposed by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (469 > 163) and ESI-MS/MS spectra with those of one of the 3′-deoxykanamycin analogs, namely authentic **16** (tobramycin). The previously reported characterization data was not sufficient to serve as a reference for direct comparison of retention time and MS/MS fragmentation pattern.

Page 50 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

MH⁺ = 469

Supplementary Figure 21 | Structural estimation of 3′-deoxykanamycin A (20). (a) ESI-MS/MS fragmentation pattern of 20, which was heterologously produced by an S. venezuelae strain (KABΔfna) expressing kanA-kanB-kanK-nemD-kacA-kanE-kanC-kanD-kanI-kacL together with aprD3-aprD4. (b) ESI-MS/MS spectra of 20, which was heterologously produced. $T_{\text{retention}}$ on HPLC-ESI-MS/MS: 17.8 min; HR-ESI-MS: $m/z$ 469.5012 [M+H]⁺ (calculated for C₁₈H₃₆N₄O₁₀, 469.4992); characteristic fragment ions generated by both 4,6-glycosidic link cleavages and a cross-ring cleavage of the kanosamine moiety at 145, 162, 163, 205 and 324. The structure of 20, which was heterologously produced was proposed by comparing its retention behavior on HPLC-ESI-MS/MS captured on MRM (469 > 163) and ESI-MS/MS spectra with those of one of the 3′-deoxykanamycin analogs, namely authentic 16 (tobramycin). The previously reported characterization data was not sufficient to serve as a reference for direct comparison of retention time and MS/MS fragmentation pattern.

a

![Diagram](https://i.imgur.com/1234567.png)

b

![Diagram](https://i.imgur.com/8901234.png)

c

![Diagram](https://i.imgur.com/5678901.png)

Supplementary Figure 22 | HPLC-ESI-MS/MS analysis of kanamycin biosynthetic intermediates obtained from in vitro reactions using cell-free extracts of recombinant S. venezuelae or wild-type S. kanamyceticus strain. (a) Chromatograms of the S. venezuelae- (SV CFE) or S. kanamyceticus ATCC 12853 (SK CFE)-derived production of **9**, supplemented with **1** (100 μM) and UDP-Glc (200 μM), respectively. Each box represents the SV or SK CFE and black-framed boxes indicate boiled CFEs used as controls. CFEs were prepared by homogenization with glass beads<sup>10</sup>, followed by pass-through with an OASIS MCX SPE column (see Methods). **2** and **9** were detected as remnants in the SK CFE control. (b) Chromatograms of the KanE-catalyzed production of 3′-deamino-3′-hydroxykanamycin X (**11**) and kanamycin X (**12**), supplemented with 2′-deamino-2′-hydroxyparomamine (**9**) and UDP-Glc. (c) Chromatograms of the KanE-catalyzed production of kanamycin A (**8**) and kanamycin D (**14**), supplemented with 2′-deamino-2′-hydroxyneamine (**10**) and UDP-Glc.

Page 52 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Figure 23 | SDS-PAGE of purified KanF<sub>His6</sub> and its kinetic data. (a) SDS-PAGE of purified KanF<sub>His6</sub> (Lane 1). Lane M shows the molecular weight marker with the indicated masses. (b) Michaelis-Menten kinetics for KanF<sub>His6</sub>-catalyzed production of 2′-deamino-2′-hydroxyparomamine (9) using the fixed donor concentration (UDP-Glc, 3.0 mM) with the varied acceptor concentrations (2-DOS (1), ranging from 0.05 to 1.5 mM). (c) Michaelis-Menten kinetics for KanF<sub>His6</sub>-catalyzed production of 2′-N-acetylparomamine (2) using the fixed donor concentration (UDP-GlcNAc, 3.0 mM) with the varied acceptor concentrations (1, ranging from 0.05 to 1.5 mM). (d) Michaelis-Menten kinetics for KanF<sub>His6</sub>-catalyzed production of 9 using the fixed acceptor concentration (1, 1.5 mM) with the varied donor concentrations (UDP-Glc, ranging from 1.0 to 6.0 mM). (e) Michaelis-Menten kinetics for KanF<sub>His6</sub>-catalyzed production of 2 using the fixed acceptor concentration (1, 1.5 mM) with the varied donor concentrations (UDP-GlcNAc, ranging from 1.0 to 6.0 mM). Black circle in the curves represents 9, whereas red one for 2. See Methods in the text for details of reaction conditions and assays performed. The curves represent the Michaelis-Menten equation fitted using GraphPad Prism (GraphPad Inc., version 5, San Diego, CA).

Page 53 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Figure 24 | SDS-PAGE of purified KanE<sub>His6</sub> and its kinetic data. (a) SDS-PAGE of purified KanE<sub>His6</sub> (Lane 1). Lane M shows the molecular weight marker with the indicated masses. (b) Michaelis-Menten kinetics for KanE<sub>His6</sub>-catalyzed production of 3″-deamino-3″-hydroxykanamycin C (5) using the fixed donor concentration (UDP-Glc, 1.0 mM) with the varied acceptor concentrations (paromamine (3), ranging from 0.05 to 1.0 mM). (c) Michaelis-Menten kinetics for KanE<sub>His6</sub>-catalyzed production of kanamycin C (6) using the fixed donor concentration (UDP-kanosamine [Kns], 1.0 mM) with the varied acceptor concentrations (3, ranging from 0.05 to 1.0 mM). (d) Michaelis-Menten kinetics for KanE<sub>His6</sub>-catalyzed production of 5 using the fixed acceptor concentration (3, 1.0 mM) with the varied donor concentrations (UDP-Glc, ranging from 0.2 to 2.0 mM). (e) Michaelis-Menten kinetics for KanE<sub>His6</sub>-catalyzed production of 6 using the fixed acceptor concentration (3, 1.0 mM) with the varied donor concentrations (UDP-Kns, ranging from 0.2 to 2.0 mM). Open circle in the curves represents 5, whereas red open one for 6. See Methods in the text for details of reaction conditions and assays performed. Details of UDP-Kns synthesis were described in Supplementary Methods. The curves represent the Michaelis-Menten equation fitted using GraphPad Prism (GraphPad Inc., version 5, San Diego, CA)

Page 54 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671

Supplementary Figure 25 | HPLC-ESI-MS/MS analysis of kanamycin pseudotrisaccharides obtained from in vitro reactions using cell-free extracts of a wild-type S. kanamyceticus strain. Chromatograms of the S. kanamyceticus ATCC 12853 cell-free extracts (SK CFE)-derived production of kanamycin B (7) or kanamycin A (8), supplemented with kanamycin C (6) or kanamycin X (12), respectively. Each box represents the SK CFE and black-framed boxes indicate boiled CFEs used as controls. CFEs were prepared by homogenization with glass beads$^{10}$, followed by pass-through with an OASIS MCX SPE column (see Supplementary Methods). Kanamycin A (8) was detected as a remnant in the SK CFE controls.

Supplementary Figure 26 | Time dependence of the number of hydrogen bonds. (a) KanF/UDP-Glc/2-DOS (1), (b) KanF/UDP-GlcNAc/1, (c) NemD/UDP-Glc/1, and (d) NemD/UDP- GlcNAc/1. All curves were obtained by taking the 10 ps average.

Supplementary Figure 27 | Snapshots of the average structure of the active sites in 3-dimensional docking models. (a) KanF/UDP-Glc/2-DOS (1), (b) KanF/UDP-GlcNAc/1, (c) NemD/UDP-Glc/1, and (d) NemD/UDP-GlcNAc/1 during 3-ns molecular dynamics simulation.

In the simulation of KanF/UDP-Glc complex, the residues Glu292, Glu293, Leu294, Ser297, and Glu300 form hydrogen bonds directly with UDP-Glc. In the simulation of KanF/UDP-GlcNAc complex, Glu292, Glu293, Ser297, and Glu300 form hydrogen bonds directly with UDP-GlcNAc; Ser297 forms a Glu292-mediated hydrogen bond with UDP-GlcNAc, while Glu293 only forms a hydrogen bond with the 2-hydroxyl and N-acetyl group of UDP-GlcNAc. In the simulation of NemD/UDP-Glc complex, the residues Glu319, Ser324, and Glu327 form hydrogen bonds directly with UDP-Glc. In the simulation of NemD/UDP-GlcNAc complex, the residues Glu319, Glu320, Ser324, and Glu327 form hydrogen bonds directly with UDP-GlcNAc.

Supplementary Table 19 | Average binding free energies (kJ/mol) between glycosyl donors and surrounding residues of KanF and NemD during molecular dynamics simulation.

|  | KanF + UDP-GlcNAc | KanF + UDP-Glc | NemD + UDP-Glc | NemD + UDP-GlcNAc |
| --- | --- | --- | --- | --- |
| $\left\langle E_{enz}^{vdW} \right\rangle$ | -7.98 ± 0.59 | -7.13 ± 0.17 | -8.07 ± 0.24 | -7.45 ± 0.40 |
| $\left\langle E_{enz+sub}^{vdW} \right\rangle$ | -42.09 ± 0.39 | -35.04 ± 0.84 | -43.90 ± 0.38 | -41.95 ± 0.08 |
| $\left\langle E_{enz}^{Elec.} \right\rangle$ | -258.33 ± 1.65 | -284.48 ± 1.47 | -253.13 ± 1.34 | -277.01 ± 1.83 |
| $\left\langle E_{enz+sub}^{Elec.} \right\rangle$ | -329.18 ± 0.47 | -367.88 ± 0.48 | -321.53 ± 0.74 | -356.63 ± 0.18 |
| ΔG | -42.25 ± 1.15 | -47.28 ± 1.82 | -41.37 ± 1.18 | -46.71 ± 1.09 |

*The binding free energy (ΔG) calculation was based on the molecular dynamics simulation;*

$\Delta G = 0.2 \left( \left\langle E_{enz+sub}^{vdW} \right\rangle - \left\langle E_{enz}^{vdW} \right\rangle \right) + 0.5 \left( \left\langle E_{enz+sub}^{Elec.} \right\rangle - \left\langle E_{enz}^{Elec.} \right\rangle \right)$, where $\left\langle \right\rangle$ denotes averages of the van der Waals (vdW) and electrostatic (Elec) interactions between the substrates (sub) UDP-Glc and UDP-GlcNAc and the surrounding residues of the enzymes (enz) KanF and NemD. 0.2 and 0.5 are scaling factors for vdW and Elec, respectively.

Supplementary Figure 28 | HPLC-ESI-MS/MS analysis of tobramycin (16) produced by in vitro reactions using cell-free extracts of wild-type S. kanamyceticus and S. tenebrarius strains. Chromatograms of the second glycosyltransferases (GTs) KanE- or TobM2-catalyzed production of tobramycin (16), supplemented with nebramine (18). Each box represents the cell-free extract (CFE) obtained from the kanamycin-producing S. kanamyceticus ATCC 12853 (SK CFE) and apramycin (including tobramycin)-producing S. tenebrarius ATCC 17920 (ST CFE). Black-framed boxes indicate boiled CFEs used as controls. CFEs were prepared by homogenization with glass beads<sup>10</sup>, followed by pass-through with an OASIS MCX SPE column (see Supplementary Methods). Kanamycin A (8) and apramycin were detected as remnants in the SK CFE and ST CFE controls, respectively.

Supplementary Figure 29 | HPLC-ESI-MS/MS analysis of tobramycin (16) or nebramine (18) produced by in vitro reactions using cell-free extracts of a recombinant S. venezuelae strain expressing aprD3-aprD4. Chromatograms of the AprD3-AprD4-catalyzed production of tobramycin (16) or nebramine (18), supplemented with kanamycin B (7) or neamine (4), respectively (see Supplementary Methods). Each box represents the cell-free extract (CFE) obtained from a recombinant strain expressing aprD3-aprD4. Black-framed boxes indicate boiled CFEs used as controls.

Kanamycin-related aminoglycoside

| Type strain | Clinical isolate |
|-------------|------------------|
| E. coli      | P. aeruginosa    | E. coli       | P. aeruginosa   | P. aeruginosa   |
| ATCC 25922   | ATCC 27853       | CCARM1A020    | CCARM1A023      | CCARM2206       | CCARM2178      |
| (Kan<sup>S</sup>) | (Kan<sup>R</sup>) | (Kan<sup>S</sup>) | (Kan<sup>R</sup>) | (Amk<sup>S</sup>) | (Amk<sup>R</sup>) |

| Pseudodisaccharides | 3 | 9 | 4 | 10 |
|---------------------|---|---|---|----|
|                     | 128 | 128 | 64 | 64 |
|                     | NA | NA | NA | NA |
|                     | 128 | 128 | 64 | 64 |
|                     | NA | NA | NA | NA |
|                     | NA | NA | 128 | 128 |
|                     | NA | NA | NA | NA |

| Pseudotrisaccharides | 6 | 12 | 5 | 11 | 7 | 8 | 13 | 14 |
|----------------------|---|----|---|----|---|---|----|----|
|                      | 16 | 16 | 128 | NA | 2 | 2 | 128 | 128 |
|                      | NA | NA | NA | NA | NA | NA | NA | NA |
|                      | 16 | 16 | NA | NA | 4 | 4 | 128 | 128 |
|                      | NA | NA | NA | NA | NA | NA | NA | NA |
|                      | NA | NA | NA | NA | 32 | 16 | NA | NA |
|                      | NA | NA | NA | NA | NA | NA | NA | NA |

| AHBA-pseudotrisaccharides | 15 | 17 |
|---------------------------|----|----|
|                           | 128 | 16 |
|                           | 64 | 16 |
|                           | 128 | 32 |
|                           | 32 | 16 |
|                           | ≤0.25 | 0.25 |
|                           | NA | 64 |

MIC: minimal inhibitory concentration. Type strains and clinical isolates were obtained from the American Type Culture Collection (ATCC, USA) and the Culture Collection of Antimicrobial Resistant Microbes (CCARM, Republic of Korea), respectively. Kan<sup>S</sup> and Kan<sup>R</sup> indicate kanamycin-sensitive and -resistant bacterial strains, respectively. Amk<sup>S</sup> and Amk<sup>R</sup> indicate amikacin(15)-sensitive and -resistant strains, respectively. NA indicates no activity at 128 μg/ml.

Supplementary References

1. Haskell, T.H., French, J.C. & Bartz, Q.R. Paromomycin. I. paromamine, a glycoside of D-glucosamine. *J. Am. Chem. Soc.* 81, 3480–3481 (1959).
2. Lee, H.C., Lee, S.D., Sohng, J.K. & Liou, K. One-pot enzymatic synthesis of UDP-D-glucose from UMP and glucose-1-phosphate using an ATP regeneration system. *J. Biochem. Mol. Biol.* 37, 503–506 (2004).
3. Smirnova, N. & Reynolds, K.A. Engineered fatty acid biosynthesis in Streptomyces by altered catalytic function of beta-ketoacyl-acyl carrier protein synthase III. *J. Bacteriol.* 183, 2335–2342 (2001).
4. Hopwood, D.A. & Wright, H.M. Bacterial protoplast fusion: recombination in fused protoplasts of Streptomyces coelicolor. *Mol. Gen. Genet.* 162, 307–317 (1978).
5. Jung, W.S. *et al.* Heterologous expression of tylosin polyketide synthase and production of a hybrid bioactive macrolide in Streptomyces venezuelae. *Appl. Microbiol. Biotechnol.* 72, 763–769 (2006).
6. Hong, J.S.J. *et al.* New olivosyl derivatives of methymycin/pikromycin from an engineered strain of Streptomyces venezuelae. *FEMS Microbiol. Lett.* 238, 291–299 (2004).
7. Ward, J.M. *et al.* Construction and characterization of a series of multi-copy promoter-probe plasmid vectors for Streptomyces using the aminoglycoside phosphotransferase gene from Tn5 as indicator. *Mol. Gen. Genet.* 203, 468–478 (1986).
8. Yoon, Y.J. *et al.* Generation of multiple bioactive macrolides by hybrid modular polyketide synthases in Streptomyces venezuelae. *Chem. Biol.* 9, 203–214 (2002).
9. Thapa, L.P. *et al.* Heterologous expression of the kanamycin biosynthetic gene cluster (pSKC2) in Streptomyces venezuelae YJ003. *Appl. Microbiol. Biotechnol.* 76, 1357–1364 (2007).
10. Park, J.W. *et al.* Genetic dissection of the biosynthetic route to gentamicin A₂ by heterologous expression of its minimal gene set. *Proc. Natl. Acad. Sci. USA* 105, 8399–8404 (2008).
11. Kieser, T. *et al.* Practical Streptomyces Genetics Ch. 10 (John Innes Foundation, Norwich, England, 2000).
12. Park, J.W. *et al.* Analytical profiling of biosynthetic intermediates involved in the gentamicin pathway of Micromonospora echinospora by high-performance liquid chromatography using electrospray ionization mass spectrometric detection. *Anal. Chem.* 79, 4860–4869 (2007).
13. Yamada, Y., Kojima, M. & Umezawa, H. Structure and activity of kanamycins and kanamycin-related compounds. *Jpn. J. Med. Sci. Biol.* 21, 223–224 (1968).
14. Lemieux, R.U., Nagabhushan, T.L., Clemetson, K.J. & Tucker, L.C.N. The synthesis of
kanamycin analogs. I. α-D-glucopyranosyl derivatives of deoxystreptamine. Can. J. Chem. 51, 53–66 (1973).

15. Chang, L.T., Behr, D.A. & Elander, D.A. The effects of intercalating agents on kanamycin production and other phenotypic characteristics in *Streptomyces kanamyceticus*. Dev. Ind. Microbiol. Series 21, 233–243 (1980).

16. Shitara, T., Umemura, E., Tsuchiya, T. & Matsuno, T. Synthesis of 5-deoxy-5-epifluoro derivatives of arbekacin, amikacin, and 1-*N*-[(S)-4-amino-2-hydroxybutanoyl]tobramycin (study on structure--toxicity relationships). Carbohydr. Res. 276, 75–89 (1995).

17. Adams, E., De Bie, E., Roets, E. & Hoogmartens, J. Isolation and identification of a new kanamycin component. Eur. J. Pharm. Sci. 4 (Suppl. 1), 110 (1996).

18. Sali, A. & Blundell, T.L. Comparative protein modeling by satisfaction of spatial restraints. J. Mol. Biol. 234, 779–815 (1993).

19. Schymkowitz, J. *et al.* The FoldX web server: an online force field. Nucleic Acids Res. 33, W382–388 (2005).

20. Vetting, M.W., Frantom, P.A. & Blanchard, J.S. Structural and enzymatic analysis of MshA from *Corynebacterium glutamicum*: substrate-assisted catalysis. J. Biol. Chem. 283, 15834–15844 (2008).

21. Willard, L. *et al.* VADAR: a web server for quantitative evaluation of protein structure quality. Nucleic Acids Res. 13, 3316–3319 (2003).

22. Frisch, M.J. *et al.* Gaussian 03, Revision C.02, Gaussian, Inc., Wallingford, CT (2004).

23. Huang, B. & Schroeder, M. Using protein binding site prediction to improve protein docking. Gene 422, 14–21 (2008).

24. Wu, G., Robertson, D.H., Brooks, C.L. & Vieth, M. Detailed analysis of grid-based molecular docking: a case study of CDOCKER - a CHARMM-based MD docking algorithm. J. Comput. Chem. 24, 1549–1562 (2003).

25. Halgren, T.A. MMFF VII. Characterization of MMFF94, MMFF94s, and other widely available force fields for conformational energies and for intermolecular-interaction energies and geometries. J. Comput. Chem. 20, 730–748 (1999).

26. Huey, R., Morris, G.M., Olson, A.J., Goodsell, D.S. A semiempirical free energy force field with charge-based desolvation. J. Comput. Chem. 28, 1145–1152 (2007).

27. Brooks, B.R. *et al.* CHARMM: a program for macromolecular energy, minimization, and dynamics calculations. J. Comput. Chem. 4, 187–217 (1983).

28. Essmann, D. *et al.* A smooth particle mesh Ewald potential. J. Chem. Phys. 103, 8577–8592 (1995).

29. Berendsen, H.J.C., Postma, J.P.M., DiNola, A. & Haak, J.R. Molecular dynamics with coupling to an external bath. J. Chem. Phys. 81, 3684–3690 (1984).

Page 63 of 64 in Supplementary Information

Nature Chemical Biology: doi:10.1038/nchembio.671
30. Aqvist, J., Luzhkov, V.B. & Brandsdal, B.O. Ligand binding affinities from MD simulations. Acc. Chem. Res. **35**, 358–365 (2002).

31. Mareljus, J., Graffner-Nordberg, M., Hansson, T., Hallberg, A. & Aqvist, J. Computation of affinity and selectivity: binding of 2,4-diaminopteridine and 2,4-diaminoquinazoline inhibitors to dihydrofolate reductases. *J. Comput. Aided Mol. Des.* **12**, 119-131 (1998).

32. Kudo, F., Sucipto, H. & Eguchi, T. Enzymatic activity of a glycosyltransferase KanM2 encoded in the kanamycin biosynthetic gene cluster. *J. Antibiot.* (Tokyo) **62**, 707–710 (2009).

33. Park, J.W. *et al.* The nebramycin aminoglycoside profiles of *Streptomyces tenebrarius* and their characterization using an integrated liquid chromatography-electrospray ionization-tandem mass spectrometric analysis. *Anal. Chim. Acta.* **661**, 76–84 (2010).
