
Viruses 2009, 1, 1240-1264; doi:10.3390/v1031240

OPEN ACCESS

**viruses**

ISSN 1999-4915  
www.mdpi.com/journal/viruses

---

Review

# Interplay between Herpesvirus Infection and Host Defense by PML Nuclear Bodies

**Nina Tavalai and Thomas Stamminger\***

Institute for Clinical and Molecular Virology, University of Erlangen-Nuremberg, Schlossgarten 4,  
91054 Erlangen, Germany; E-Mail: nina.tavalai@viro.med.uni-erlangen.de  

* Author to whom correspondence should be addressed;  
E-Mail: thomas.stamminger@viro.med.uni-erlangen.de; Tel.: +49-9131-8526783;  
Fax: +49-9131-8522101.

Received: 18 November 2009; in revised form: 10 December 2009 / Accepted: 14 December 2009 / Published: 15 December 2009

---

## Abstract: In recent studies we and others have identified the cellular proteins PML, hDaxx, and Sp100, which form a subnuclear structure known as nuclear domain 10 (ND10) or PML nuclear bodies (PML-NBs), as host restriction factors that counteract herpesviral infections by inhibiting viral replication at different stages. The antiviral function of ND10, however, is antagonized by viral regulatory proteins (e.g., ICP0 of herpes simplex virus; IE1 of human cytomegalovirus) which induce either a modification or disruption of ND10. This review will summarize the current knowledge on how viral replication is inhibited by ND10 proteins. Furthermore, herpesviral strategies to defeat this host defense mechanism are discussed.

## Keywords: herpesvirus; nuclear domain 10; PML nuclear bodies; PML; Sp100; hDaxx; antiviral defense; intrinsic immunity; interferon

---

## 1. Introduction

In general, viruses depend in many aspects on the cellular machinery in order to replicate efficiently. Consequently, viruses have evolved in tight association to the host cell to be able to hijack the cellular apparatus that is necessary for their replication. During co-evolution, however, the cell has developed various antiviral defense mechanisms in order to limit viral replication and spread, which

viruses have to cope with. In particular, it could be shown that a set of constitutively expressed cellular proteins, collectively termed intrinsic immunity, represents one of the first lines of antiviral defense in naive hosts [1]. Approximately 15 years ago a subnuclear structure known as nuclear domain 10 (ND10; alternatively termed PML-NBs for PML nuclear bodies or PODs for PML oncogenic domains) has been discovered that is targeted by a variety of viruses belonging to different viral families. Although there is now increasing evidence that ND10 proteins mediate intrinsic immunity against viruses, there has always been a controversial debate about the functional significance of ND10 for viral replication. The main objective of this article is to critically discuss the very recent literature on the role of ND10 for viral infection. It will focus on herpesviruses, since these have been most extensively studied with respect to the functional consequences of ND10 association. The studies highlighted in this review provide accumulating evidence for an important role of ND10 in host antiviral defenses against herpesviruses. Finally, the diverse strategies herpesviruses have evolved to overcome this ND10-based host defense are outlined.

2. Structural Aspects of ND10

Nuclear domains 10 (ND10), also referred to as nuclear dots, PML nuclear bodies (PML-NBs), or PML oncogenic domains (PODs), can be defined as dynamic, spherical, macromolecular structures which represent accumulations of multiple cellular proteins that assemble in distinct foci within the interchromosomal space of the nucleus [2]. Like many transcription or RNA processing factors, ND10 bodies are associated with the nuclear matrix, since treatment with RNase or DNase does not alter their morphology [3,4]. The apparent size of PML-NBs ranges from 0.2 to 1 μm, and their frequency depends on cell type and status, varying from two or three to as many as 30 per cell [4]. Presently, the nuclear protein database lists 70 proteins that have been reported to be present in or associated with ND10 structures (http://npd.hgu.mrc.ac.uk/). These proteins can be subdivided into two classes, based on whether they are constitutively or transiently present at ND10. Permanently localized at ND10 are the major components PML (promyelocytic leukemia protein), hDaxx, Sp100 (speckled protein of 100 kDa), SUMO-1 (small ubiquitin-related modifier 1), and the Bloom syndrome helicase BLM [2]. On the contrary, quite a variety of factors can be found at ND10 only under certain conditions (e.g., components of the DNA repair machinery) [5–7] or upon overexpression (e.g., BRCA1-breast cancer protein 1) [2,8]. However, in the latter case, it has to be questioned, whether such polypeptides constitute natural ND10 components, as more and more data suggest a role of ND10 in the degradation of mis- or unfolded protein aggregates [9,10]. Thus, as the composition of ND10 varies both within and between cells, it is evident that ND10 represent functionally heterogeneous protein accumulations. The Sp100 protein was the first ND10 constituent identified using sera of patients suffering from the autoimmune disease primary biliary cirrhosis (PBC) [11]. Thereafter, PML was discovered as the defining factor of ND10. It functions as a kind of scaffold protein that is responsible for the assembly and maintenance of PML-NBs and recruits other ND10-associated proteins like hDaxx to this subnuclear structure [12,13]. Since PML constitutes the key component of ND10 integrity, loss of PML consequently leads to a dispersal of ND10-resident proteins as observed in mouse PML-null fibroblasts [12,13] or human PML-knockdown (PML-kd) fibroblasts [14,15].

Viruses 2009, 1

3. The Major ND10 Constituents and Their Functions

3.1. PML

The PML protein was originally discovered in patients suffering from acute promyelocytic leukemia (APL), where a reciprocal chromosomal translocation resulting in a fusion of the PML protein to the retinoic acid receptor α turned out to be responsible for this hematopoietic malignancy [16]. PML, also known as TRIM19, belongs to the RBCC or tri-partite motif family (TRIM) of proteins that are comprised of a zinc-finger RING domain, one or two B-boxes (cysteine/histidine-rich motifs) and a predicted α-helical coiled-coil domain [17]. This motif, which allows PML to interact with other proteins as well as to homo-oligomerize, is essential for ND10 formation and for the function of PML as a growth- and transformation-suppressor [18]. Due to differential splicing of the PML gene transcript which consists of nine exons, at least seven different PML isoforms (I to VII) are expressed within cells, all sharing a common N-terminus but varying in their C-termini [17]. Since not all PML isoforms retain the nuclear localization signal in exon 6, beside nuclear ND10-localized PML also cytoplasmic species of PML exist [17]. So far, the functions of the PML splice variants are not fully understood. However, in this context it is of note that recent publications demonstrate an influence of herpesvirus infections on the splicing pattern of PML suggesting diverse roles for the individual PML isoforms with regard to viral replication [19, 20]. Besides determining the subnuclear distribution of PML, alternative splicing could add new functional domains to the protein or may be an important mechanism for generating differential PML-binding interfaces for a variety of factors.

Moreover, all isoforms having molecular weights ranging from 48 to 97 kDa (kilo Dalton), are subject to posttranslational modifications like phosphorylation [21] or conjugation to the ubiquitin-homologous protein SUMO (SUMOylation) [22, 23]. In addition to three SUMO modification sites at lysine residues 65 (in the RING finger domain), 160 (in the B1 box), and 490 (in the nuclear localization signal) [24], PML also contains a SUMO binding motif that enables it to interact noncovalently with SUMO [25]. Covalent as well as noncovalent SUMO modification is extremely important for the function of PML to orchestrate ND10 formation as clearly illustrated in case of the recruitment of hDaxx to this subnuclear structure [13, 25, 26].

3.2. hDaxx

HDaxx is a highly conserved nuclear protein that contains a serine/proline/threonine-rich domain, an acidic domain, a coiled-coil region, and two paired amphipathic helices [27–30]. It has been recognized as a regulator of both apoptosis and gene expression [31]. In gene regulation, hDaxx has been shown to function as a transcriptional corepressor. It negatively affects gene expression by suppressing the activity of several transcription factors, including Ets-1 (E twenty-six 1) [32], NF-κB (nuclear factor κB) [33], Pax3 [27], E2F-1 [34], Smad4 [35], p53 family members [36], and glucocorticoid, mineralocorticoid and androgen receptors [37]. Recent findings indicate that hDaxx associates via its newly identified SUMO-interacting motif (SIM) with SUMOylated DNA-binding transcription factors [26], thereby recruiting proteins involved in transcriptional repression, such as histone deacetylase 1 (HDAC1) [29], HDAC2 [38], DNA methyltransferase 1 (DNMT1) [39], or

ATRX (α-thalassaemia/mental retardation syndrome X-linked) [40, 41], a member of the SNF2 family of chromatin remodeling enzymes, to targeted promoters. The transrepressive effect of hDaxx, in turn, is modulated by its subnuclear compartmentalization. The SIM enables hDaxx to also noncovalently interact with SUMOylated PML, resulting in the sequestration of nucleoplasmic hDaxx to ND10 [25, 26], which is accompanied by attenuation of its inhibitory function [29]. Beside ND10, hDaxx can also be targeted to the nucleolus, centromeres, or heterochromatin through its interactions with either the nuclear protein MSP58 (microspherule protein of 58 kDa) [42], the centromeric protein CENP-C [30], or the heterochromatin-associated factor ATRX [40]. Taken together, these observations imply that, depending on its subnuclear localization, hDaxx seems to fulfill distinct functions.

### 3.3. Sp100

Like PML, the Sp100 protein represents a permanent ND10 constituent which is expressed via alternative splicing from a single gene transcript giving rise to four different isoforms designated as Sp100A [11], Sp100B [43], Sp100C [44], and Sp100-HMG [45, 46]. All Sp100 splice variants share a common N-terminus harboring an HSR (homogeneously staining region) motif responsible for homo-oligomerization of Sp100 and its targeting to ND10 as well as a binding site for the nonhistone chromosomal DNA-binding protein HP1 (heterochromatin protein 1). The larger splice variants Sp100B, -C, and -HMG encode additional functional motifs at the C-terminus such as a SAND domain (common to all three longer isoforms), a PHD finger-bromodomain (Sp100-C), or an HMG box (Sp100-HMG). All three motifs represent potential DNA-binding domains which can frequently be found in proteins affecting chromatin structure [47]. Together with the observation that Sp100 interacts with heterochromatin protein HP1 [45], which plays a central role in establishing a stable heterochromatic network, this suggests a role of Sp100 in transcriptional regulation. Indeed, the isoform Sp100B has been shown to function as a transcriptional repressor of both cellular and viral promoters in transient expression experiments [48]. In addition, as recently reported, Sp100B preferentially associates via its SAND domain with DNA sequences containing unmethylated CpG dinucleotides, since methylation of cytosines in the CpG context abrogates DNA binding of Sp100B [49]. This finding prompted the authors to speculate that the preference of Sp100B for nonmethylated CpGs could provide a mechanism to specifically target this isoform to foreign DNA (including viral genomes), which is predominantly hypomethylated [49].

Like many ND10 constituents, including PML and hDaxx [23, 50, 51], Sp100 is also subject to posttranslational modification by SUMO [51]. While SUMOylation of Sp100 is not required for its ND10 localization [52], it has been shown to enhance the interaction with HP1 and thus to stabilize Sp100-HP1 complexes suggesting that the functional interplay of Sp100 between ND10 and heterochromatin could be regulated in this way [44]. Recent data indicate, that the SUMO modification status as well as the relative expression level of Sp100 are to some extent regulated, either directly or indirectly, by PML [14]. SiRNA-mediated knockdown of PML resulted in an apparent reduction in the abundance of Sp100-HMG and the SUMO-modified form of Sp100A, with a concomitant increase in unmodified Sp100A [14]. Thus, Sp100 metabolism seems to be closely linked to that of PML.

Viruses 2009, 1

4. Functions of ND10

Unlike other, more specialized subnuclear structures such as Cajal and Polycomb group bodies, PML-NBs are functionally promiscuous as they represent cellular organizing centers for the coordinated regulation of diverse cellular processes. The research in a variety of fields has produced a substantial literature that links ND10 to oncogenesis [53], DNA damage repair [6], apoptosis [54, 55], stress response [56], senescence [57], the ubiquitin pathway [58] as well as to the regulation of gene expression [59]. Despite these various cellular responses, the functions of ND10 are still not fully understood.

In the absence of a clear understanding of the biochemical function of ND10, the following, not mutually exclusive, models have been proposed to explain how this subnuclear structure could exert its biological functions [2, 59–62]: (i) PML-NBs might operate as nuclear depots or storage sites for the accumulation of proteins both under pathological conditions to sequester foreign or misfolded proteins as well as under normal conditions to accumulate proteins that can be released or relocated elsewhere as required in order to control their availability at nucleoplasmic sites other than ND10; (ii) PML-NBs may illustrate “catalytic surfaces” where the post-translational modification of proteins takes place (e.g., SUMOylation); (iii) ND10 domains could be active sites for defined nuclear functions such as transcriptional and chromatin regulation.

With respect to the latter activity, evidence continues to accumulate that ND10 play a role for transcriptional regulation since numerous transcription factors and transcriptional regulators dynamically localize to these domains [59] and nascent RNA has been detected in the immediate vicinity of PML-NBs [63]. Nevertheless, the exact molecular mechanisms of ND10-mediated transcriptional regulation remain elusive. Several studies indicate that PML-NBs might regulate transcription by modulating the nucleoplasmic availability or the activity status of transcription factors [59, 64]. While not contradicting this model, another emerging idea is that ND10 could also control transcriptional activities indirectly on an epigenetic level by participating in chromatin-remodeling processes. For the ND10 components PML and hDaxx, an association with histone deacetylases (HDACs) [29, 38, 65] or DNA methyltransferases [66, 67], which all exhibit a transcriptionally repressive function, has been demonstrated. Certain PML isoforms, in addition, have been shown to form complexes with multiple corepressors like c-Ski, N-CoR, mSin3A [68], or the novel KRAB-zinc finger repressor PAROT (PML-associated repressor of transcription) [69] as well as to silence transcription by recruiting the histone methyltransferase SUV39H1 [70] or the polycomb group (PcG) protein EZH2 [71], one of the core components of the Polycomb repressive complexes (PRC) 2/3/4. Similarly, Sp100 also behaves as a transcriptional repressor by interacting with the heterochromatin protein HP1 [45, 48]. Thus, the fact that several ND10 components associate with potent repressors of gene expression gave rise to the idea of ND10 acting as sites of transcriptional repression. In contrast, however, the presence of transcriptional activators like the acetyltransferase CBP or p53 at ND10 has likewise been described [72, 73], reflecting the still controversial debate about the role of this subnuclear structure in regulating gene expression.

Finally, as will be discussed in this review, ND10 have been implicated to play an important role during viral infection [5, 62, 74–76]. For instance, there are several indications that ND10 are linked to the interferon-mediated antiviral response of the cell [77, 78]. Many ND10 proteins, including the

major components PML and Sp100, are interferon (IFN) inducible [77, 79, 80]. Although PML and Sp100 are expressed in the absence of IFN, their expression is greatly increased and directly induced by type I (IFN-α and IFN-β) as well as type II (IFN-γ) interferons through an “IFN-stimulated response element” (ISRE) and an “IFN-gamma activation site” (GAS) which are located within the promoters of their genes [81, 82]. In addition, PML transcription can also directly be activated by the IFN-regulatory factor 3 (IRF3), which is also capable of binding to the PML promoter [83]. As a consequence of this, in response to interferon treatment of cells, the number, size, and intensity of PML-NBs increases [84]. The study of ND10 structures in the context of viral infection provides further evidence to implicate ND10 in the IFN pathway. As specified in detail in the sections below, many viruses have evolved polypeptides in order to compromise ND10 integrity to variable extents. Such kind of structural modifications (e.g. dispersal) of ND10 have been shown to frequently correlate with the efficiency of viral infection and thus could be viewed as part of a viral strategy to evade an antiviral function of ND10 [78]. The following chapters will summarize the current state of knowledge concerning the potential role of ND10 during infection with different herpesviruses.

5. ND10 during Herpesvirus Infections

Interest in the interaction between viruses and ND10 was first sparked by the observation that infection by herpes simplex virus type 1 (HSV-1) caused a rapid destruction of this subnuclear structure [85]. Subsequently, it was found that the parental genomes and replication complexes of HSV-1 were preferentially located in close association with ND10 [86, 87]. This observation has been extended to include members of all subfamilies of the herpesviridae [74, 76]. Thus, it appears to be a general feature of nuclear-replicating herpesviruses that their parental genomes preferentially become associated with ND10, and that their initial sites of transcription and the development of DNA replication centers are frequently juxtaposed to these domains or their remnants. The intimate spatial association between herpesviruses and ND10 has raised the important question about the functional consequences of this tight interaction. Some progress has been made within the last years towards answering this question.

6. Alpha-Herpesviruses and ND10

6.1. Herpes Simplex Virus Type 1 (HSV-1)

HSV-1, a member of the neurotropic alpha sub-family of herpesviruses, is a common human pathogen that causes recurrent infections through its ability to establish a latent state in sensory ganglia after a primary infection of epithelial cells. It was the first virus to be shown to affect ND10 morphology during infection. The initial observation that HSV-1 disrupts ND10 but an ICP0-deficient mutant does not [85], rapidly led to the identification of ICP0 being necessary and sufficient for this effect [88, 89]. ICP0 is a RING finger protein that is very important in certain cell types for initiating viral lytic infection and contributes to the reactivation of quiescent viral genomes in cultured cells as well as of latent virus from neurons in mouse models [90]. ICP0 initially precisely colocalizes with ND10 at early times upon infection and subsequently mediates the disaggregation of PML-NBs by inducing the degradation of the SUMO-1 modified forms of PML and Sp100 [91–94], leading to the

release and dispersal of other ND10 proteins. This effect of ICP0 on ND10 is dependent upon ICP0 having an intact RING finger motif [88] and reflects its ability as an E3 ubiquitin ligase to recruit the E2 ubiquitin-conjugating enzymes UbcH5a and UbcH6 in order to target specific proteins for degradation by the proteasome [95, 96]. However, it is likely that the apparent loss of the low molecular weight isoforms of Sp100 (except Sp100-A) during wt HSV-1 infection is a consequence of ICP0-induced degradation of PML, rather than being a direct effect of ICP0 on Sp100 itself (since the electrophoretic pattern of Sp100 mimics what is observed in PML-depleted cells) [14]. Thus, it is tempting to speculate that by downregulating PML, ICP0 indirectly targets both major ND10 components.

Disruption of ND10 by ICP0 correlates with its role in stimulating HSV-1 lytic infection and reactivation from quiescence or latency [90]. Furthermore, it has been shown that an ICP0-negative HSV-1, which fails to disrupt ND10 is hypersensitive to the effects of IFN in certain cultured cell lines [97, 98] and exhibits low pathogenicity in normal mice in vivo, while infection of mice that are unable to mount an interferon response restores its replication capacity [99]. Although neither infection with wt nor ICP0-deficient HSV-1 is enhanced in the absence of PML in PML−/− murine embryonic fibroblasts (MEFs) compared to control PML+/+ MEFs, the replication of the ICP0-deletion virus is substantially compromised by IFN treatment of these cells in the presence but not absence of PML [100]. These results suggest that PML contributes to the cellular IFN-mediated restriction of HSV-1 infection and that one function of ICP0 is to efficiently counteract the repressive effect of PML. However, neither exogenous expression of PML isoforms III, IV, nor VI turned out to affect virus yield, even though PML overexpression blocked the dispersal of ND10 in response to HSV-1 infection [91, 101, 102]. In light of the increasing evidence that different PML isoforms possess distinct properties, one possible explanation could be that the PML variants used in those studies may play a minor role in the host IFN-mediated antiviral response. In addition, in contrast to the prior fixed cell analyses, live-cell imaging experiments clearly demonstrated that large ND10 aggregates as a consequence of high-level expression of PML are nonetheless subject to extensive modification in terms of both content and morphology during HSV-1 infection [101].

In principle, it was not until the generation of physiologically more relevant knockdown cells being derived from the natural human host, in which individual ND10 components were depleted, that substantial advances in understanding the biological relevance of ND10 for HSV-1 replication were made. Although infection of cells with an shRNA-mediated down-regulation of either PML or Sp100 had no effect on wt HSV-1 replication, it resulted in a significant increase in the efficacy of gene expression and plaque formation of an ICP0-null mutant virus [14, 103]. On the same theme, Negorev et al. previously reported that the restrictive activity of Sp100 is based on isoforms B, C, and HMG having an intact SAND domain with which they are capable of repressing viral IE transcription at the promoter level [104, 105]. In accordance with this, only overexpression of Sp100B but not Sp100A suppressed HSV-1 gene expression [48, 105]. Moreover, by specifically depleting the repressive Sp100 variants B, C, and HMG using RNA interference, these proteins could be confirmed as an essential part of the IFN-mediated restriction of HSV-1 replication [105]. In continuative studies Negorev et al. [104], in addition, demonstrated that IFN treatment of cells leads to a change in the splicing pattern of Sp100 transcripts in favor of the repressive isoform Sp100C

Interestingly, simultaneous knockdown of both ND10 proteins, PML as well as Sp100, further stimulated gene expression and plaque-forming ability of the ICP0-knockout virus but did not completely eliminate its replication defect indicating the involvement of additional repressive factors [103]. Indeed, unpublished data presented by the Everett group at the 34th International Herpesvirus Workshop suggest repressive roles for further ND10 constituents such as the transcriptional repressor hDaxx or the chromatin-remodeling enzyme ATRX which cooperatively contribute to the ND10-based inhibition of HSV-1 infection by forming a chromatin-remodeling complex (Everett, R., personal communication). Taken together, these findings add more weight to the concept of ND10 as an intrinsic antiviral defense mechanism of the cell as individual ND10 components participate independently in the silencing of viral gene expression. Furthermore, a recent publication suggests that not only the nuclear ND10-associated PML variants but also the cytoplasmic isoforms could potentially contribute to an antiviral response against HSV-1 as they are capable of sequestering the viral transactivator protein ICP0 to the cytoplasm thereby limiting viral protein accumulation and replication [20]. Interestingly, HSV-1 infection has been shown to induce changes in the splicing of PML pre-mRNA resulting in a selective enrichment of the suppressive cytoplasmic PML variants [20]. In this regard, it is noteworthy that splicing of PML pre-mRNA is also affected by the closely related herpes simplex virus type 2 (HSV-2) [19]. Nojima and coworkers recently identified the viral effector protein ICP27 as an alternative splicing regulator of the PML transcripts which has the ability to actively alter the PML expression profile during HSV-2 infection [19].

### 6.2. Varicella Zoster Virus (VZV)

Like HSV-1, varicella zoster virus (VZV), the causative agent of chicken pox (varicella) in children or shingles (zoster) upon reactivation in elderly or immunocompromised individuals, is characterized as a human alpha-herpesvirus [106]. Comparable to the situation with HSV-1, new findings indicate that VZV infection is also negatively regulated by components of ND10 [107]. The analysis of VZV propagation in cells lacking the major ND10 factors revealed that replication of wt VZV is accelerated in case PML or hDaxx expression is silenced, while depletion of Sp100 had no effect on viral growth [107]. Like all members of the alpha-herpesvirus subfamily, VZV codes for an ortholog of HSV-1 ICP0 termed ORF61p which, similar to ICP0, transcriptionally activates viral promoters and enhances the infectivity of viral DNA [108, 109]. Since accumulating evidence suggests that the transactivator function of ICP0, at least in part, is based on the ability of ICP0 to inactivate the cellular intrinsic defense instituted by ND10, a similar role was assumed for ORF61p. However, in contrast to ICP0 which is known to downregulate both restriction factors, PML as well as Sp100, ORF61p was found to specifically reduce Sp100 protein levels only in order to allow efficient virus replication [107, 110]. Consequently, a chimeric HSV-1 mutant, which expressed ORF61p in place of ICP0 only partially rescued the phenotype of an ICP0-null virus as ORF61p fails to target PML, which remains available to restrict virus replication [110]. These findings suggest that ICP0 and ORF61p have evolved separately to provide different functions during virus infection. Thus, although alpha-herpesviruses like HSV and VZV are considered to be very similar, the studies presented by the Silverstein group demonstrate that they have evolved unique and specialized ways to interfere with the ND10-mediated host cell response to ensure optimal viral replication.

Viruses 2009, 1

7. Beta-Herpesviruses and ND10

Also in case of HCMV, the prototype of the β-subgroup of herpesviruses, parental viral genomes associate with ND10 followed by the targeting of newly synthesized viral IE transactivators IE1 and IE2, which are essential for initiating the lytic replication program of HCMV [111]. The interplay between HCMV nucleoprotein complexes and ND10 structures, however, can only be observed within a narrow time window due to the action of the IE1 protein, which is responsible for the disruption of ND10 during infection with HCMV [112, 114, 115]. As a mechanism for this, it has been proposed that IE1 abrogates the SUMOylation of PML [116, 117], but in contrast to ICP0 of HSV-1 (see 6.1.), IE1 does not require proteasome activity for this effect [118], nor does it possess any intrinsic desumoylation activity in vitro [119]. Thus, although the biochemical basis for ND10 disruption by IE1 remains unclear, structural modification of ND10 has been shown to correlate with the functional activities of IE1 in transcriptional regulation, resulting in increased efficacy of viral replication. In the same context, the HCMV infectious cycle, on the contrary, is significantly attenuated in cells expressing high levels of exogenous PML due to a delay in IE1-mediated ND10 dispersal leading to impaired establishment of replication centers and reduced production of early and late proteins [120] (our own unpublished observations). Consequently, these data already implied a repressive function of PML on HCMV replication which is counteracted by the IE1-induced reorganization of ND10. Conclusive evidence for this assumption was finally obtained from infection studies using cells being devoid of genuine ND10: extensive, siRNA-mediated depletion of PML in primary human fibroblasts significantly increased the plaque-forming efficiency of HCMV as a result of an augmented IE gene expression [15]. This effect was considerably enhanced after infection of PML-kd cells with an IE1-deficient HCMV, since loss of PML complemented the growth defect of this mutant virus [15]. Thus, the clear demonstration of an intrinsic antiviral activity of the main ND10 component PML was an important step forward in understanding the functional significance of the intimate relationship between HCMV and ND10.

The interaction of HCMV proteins with ND10 is not exclusive to the IE proteins IE1 and IE2. The viral tegument protein and transactivator pp71 likewise accumulates at ND10 immediately upon HCMV infection and before the production of IE proteins [121–123]. Direct binding to the ND10 component hDaxx has been shown to be responsible for the targeting of pp71 to PML-NBs [121, 122]. Interestingly, abolishing pp71's ability to interact with hDaxx blocks pp71's localization at ND10 and inhibits pp71's ability to transactivate the major immediate promoter (MIEP) of HCMV [121, 122, 124], indicating that the association of pp71 with hDaxx in ND10 is critical for its function as a facilitator of IE gene expression at the very start of a lytic infection. Although initial studies implied that pp71 and hDaxx associate at ND10 to cooperatively activate the MIEP [121], it is now clear from multiple subsequent work that hDaxx actually silences the MIEP, and that pp71 relieves this repression [125–129]. Consistent with this assumption, overexpression of the cellular restriction factor hDaxx abolishes HCMV infection while downregulation of hDaxx by usage of small interfering RNA (siRNA) technology, on the contrary, results in increased gene expression and virus replication [125–129]. The hDaxx-mediated repression of viral IE gene expression correlates with changes of the chromatin structure around the MIEP as knockdown of hDaxx results in loss of transcriptionally repressive and gain of transcriptionally active chromatin at the MIEP [125].

regulation appears to involve histone deacetylases, since treatment of infected cells with HDAC-inhibitors relieves the repression of viral IE gene expression [125, 126].

In order to successfully antagonize hDaxx-mediated intrinsic immune defense, tegument-delivered pp71 induces the degradation of hDaxx at the very start of a lytic infection [126, 129], which has been postulated to occur in a proteasome-dependent [126] and ubiquitin-independent manner [130]. Inhibiting the proteasome at the time of infection, however, still suppresses viral IE gene expression even in the absence of hDaxx, suggesting the existence of additional antiviral targets for proteasomal degradation [131]. Additionally, pp71 has been shown to promote the SUMOylation of hDaxx thereby increasing the basal level of SUMO-conjugated forms of the transcriptional repressor during infection [132]. However, the role this posttranslational modification plays in regulating hDaxx activity remains enigmatic. Interestingly, recent data from the Preston group revealed that prior to the removal of hDaxx, pp71 first stimulates the release of the chromatin-remodeling protein ATRX from ND10 [133]. ATRX has been shown to be sequestered to PML-NBs in an hDaxx-dependent manner where both proteins associate to form a chromatin-remodeling complex. The displacement of ATRX from ND10 contributes to the role of pp71 in alleviating the repression of viral IE gene expression [133].

Consistent with a function of pp71 in counteracting the antiviral effects of hDaxx or ATRX, HCMV inefficiently enters productive infection in the absence of pp71 [124, 134], unless hDaxx or ATRX proteins are depleted prior to infection, thus annihilating the impaired growth phenotype associated with a pp71-deficient mutant [127–129]. Failure of pp71 to overcome hDaxx repression blocks viral IE gene expression and may promote the establishment of latent HCMV infections [135]. However, whether hDaxx is a major factor contributing to the control of HCMV latency is still controversially discussed [135, 136].

In summary, these data clearly identified the ND10 proteins PML, hDaxx and ATRX as cellular restriction factors responsible for silencing of HCMV IE gene expression directly upon infection. Moreover, since knockdown of hDaxx in combination with PML led to a further increase in the replication efficacy of HCMV [129], this strongly argues for an independent involvement of individual ND10 constituents in the restriction of viral infection and gives rise to the following working model: Immediately after infection, incoming viral genomes are targeted by ND10. It is assumed that ND10 proteins contribute to the formation of a repressive chromatin structure on intranuclear viral DNA via epigenetic processes resulting in transcriptionally inactive viral genomes [137], as many of the ND10 factors have been implicated in epigenetic regulation [138]. In fact, evidence for a “pre-IE” inhibition of viral gene expression instituted by histone posttranslational modifications has recently been provided by Groves et al. (2009) [139]. As a first line of defense, the imported HCMV structural protein pp71 antagonizes ATRX-mediated gene silencing by inducing the dissociation of ATRX from ND10. Thereafter, pp71 inactivates hDaxx's antagonistic function by promoting its proteasomal degradation. The pp71-dependent relieve of ATRX- as well as hDaxx-related repression, then, allows the initiation of viral IE gene expression. The synthesis and accumulation of high levels of IE1 at ND10, in a next step, efficiently overcomes PML-mediated repression by inducing the disruption of this subnuclear domain. This results in modification of the chromatin structure around early and late gene promoters driving a regulated cascade of viral lytic gene expression thereby ensuring an efficient virus production [137, 139, 140]. In case pp71 is sequestered in the cytoplasm of infected cells, as postulated for cells in which latent-like HCMV infections are established [135], input viral genomes

are likewise targeted by ND10. However, the repressive effect of hDaxx on viral gene expression is not relieved as pp71 fails to enter the nucleus and to degrade hDaxx. As a consequence of this, the viral genomes remain in a transcriptionally repressed quiescent state.

Interestingly, recent unpublished observations of our group indicate that, comparable to the situation in PML-, hDaxx- or ATRX-kd cells, ablation of Sp100 also leads to a considerable increase in the number of IE protein-positive cells, suggesting that Sp100 likewise contributes to ND10-mediated viral repression (Adler, M., Tavalai, N., Stamminger, T., in preparation). Overall, these data support a model in which, comparable to HSV-1 infection, individual ND10 components function in counteracting the initiation of an efficient productive viral replication. This further underlines the notion of ND10 as mediators of an intrinsic immune defense against herpesvirus infections in general.

## 8. Gamma-Herpesviruses and ND10

### 8.1. Epstein-Barr Virus (EBV)

In common with all herpesviruses, EBV from the gamma-herpesvirus subfamily, possesses a biphasic life cycle consisting of a lytic and a latent phase. However, in contrast to HSV-1, its latent state is characterized by the expression of a set of proteins that are necessary for establishment and maintenance of latency [111]. Latent EBV infection, in addition, is strongly associated with several types of cancer, including nasopharyngeal carcinoma (NPC) [141]. While during latency, EBV genomes can be found in close association with interphase chromosomes, activation of the lytic replication cycle results in a spatial redistribution of viral DNA to ND10 [142]. Using in situ hybridization the authors also showed the development of replication centers starting at ND10 as replicating EBV genomes were frequently found beside this nuclear domain [142]. Subsequent studies identified the ori<sub>lyt</sub> sequence as being required for the appearance of replicating episomes in association with PML-NBs [143]. However, the integrity of ND10 is no longer maintained as soon as productive EBV infection is initiated. The expression of lytic proteins is accompanied by a sequential redistribution of ND10 proteins starting with the dispersal of Sp100, hDaxx, and NDP55 which is then finally followed by PML relocalization [142–144]. Induction of productive EBV replication goes along with the expression of BZLF-1, the principal inducer of the lytic gene expression program [111]. Indeed, experiments carried out by Adamson and Kenney [144] illustrated, that BZLF-1 expression is sufficient to induce ND10 disruption by reducing the level of SUMOylated PML. BZLF-1, which is itself SUMO-1 modified, has been shown to compete with PML for limiting amounts of SUMO-1 [144]. In accordance with this, mutation of the SUMO-modification site of BZLF-1 abrogates desumoylation of PML [144]. However, this does not annihilate the capability of BZLF-1 to disperse ND10 after transient expression suggesting the existence of additional mechanisms for BZLF-1-mediated ND10 disruption. Further mutational analysis of BZLF-1 revealed a correlation between the capability of this protein to disperse Sp100 from ND10 and its efficiency to activate transcription via interaction with the ND10-located factor CBP [145].

Although initial studies suggested that EBV infection has no influence on ND10 during latency [142], recent evidence indicates that the latently expressed protein EBNA-LP (also designated as EBNA5) is likewise modifying the composition of ND10 accumulations. Already over ten years

ago, EBNA-LP, which is important for EBV-mediated B-cell immortalization by functioning as a potent gene-specific coactivator of the viral transcriptional activator EBNA2 [146], was reported to colocalize with ND10 in EBV-positive lymphoblastoid cell lines [147]. Novel findings imply that EBNA-LP coactivates EBNA2 through binding of the ND10 factor Sp100, thereby selectively displacing it together with HP1α from PML-NBs [148]. Since both HP1α and Sp100 have been shown to possess transcriptional repressor activity [44, 45, 149], and expression of a mutant form of Sp100, that was unable to associate with ND10, resulted in coactivation of EBNA2 even in the absence of EBNA-LP [148], it has been speculated that the EBNA-LP-induced ND10 rearrangements may mitigate transcriptional barriers that prevent efficient expression of viral latent genes important for establishing latency. Furthermore, as IFN-αβ treatment of cells did not compromise EBNA-LP's coactivator function, it was suggested that EBNA-LP might play a role in EBV-evasion of IFN-mediated antiviral responses [150]. Interestingly, novel findings from Sivachandran et al. demonstrate that EBV latent infection in NPC cells is also associated with a modification of ND10 integrity and that EBNA1 is responsible for this effect by decreasing PML protein levels in a proteasome-dependent manner as well as through interaction with the cellular ubiquitin-specific protease USP7/HAUSP [151]. In addition, the data support a model in which EBNA1-mediated ND10 disruption promotes the survival of cells with DNA damage [151]. Thus, EBNA1 is considered as a viral factor that contributes to the establishment of NPC. In summary, these results are regarded as the first indication that modulation of ND10 might also be required for the establishment of nonproductive or latent herpesvirus infections as well as for the development of virus-induced cancers.

### 8.2. Kaposi's Sarcoma-Associated Herpesvirus (KSHV)

Human herpesvirus 8 (HHV8), a gamma-herpesvirus also known as Kaposi's sarcoma-associated herpesvirus (KSHV), is a lymphotropic virus involved in the pathogenesis of Kaposi's sarcoma (KS), primary effusion lymphoma (PEL), and a subset of multicentric Castleman's disease (MCDs). KSHV also encodes an early protein, K8, which is a distant evolutionary homologue of the EBV regulatory factor BZLF-1, that has been found to localize to ND10 and to establish replication compartments in association with this nuclear substructure [152, 153]. However, unlike BZLF-1, K8 does not induce any changes to ND10 composition [153]. Consequently, since the genome replication of KSHV also occurs in close contact to PML-NBs, it seems to be a general feature of herpesviruses to initiate their replication in the near vicinity of ND10. Nevertheless, for a long time, KSHV was considered to be unique among herpesviruses in that it might not target ND10 for destruction. However, very recently, Marcos-Villa et al. identified the LANA2 protein of KSHV as a viral regulatory factor that is capable of inducing the disruption of ND10 [154]. As a mechanism for this, it was proposed that LANA2 mediates the proteasome-dependent degradation of PML by promoting SUMO2-based ubiquitylation of PML [154]. Moreover, the authors presented evidence that depletion of PML caused by LANA2 relieves PML-mediated transcriptional repression of cellular survivin, a host protein that directly contributes to the malignant progression of KSHV-infected PEL cells [154]. Taken together, these data strongly support the notion that, similar to EBV (see 8.1.), KSHV has developed strategies to neutralize ND10 function in order to promote cellular transformation and carcinogenesis.

Viruses 2009, 1

8.3. Murine Gamma-Herpesvirus 68 (MHV-68)

Like KSHV, MHV-68 (also known as murid herpesvirus 4) belongs to the herpesvirus genus rhadinovirus. It is emerging as a suitable model to study basic biological questions of gamma-herpesvirus host interactions due to the fact that it naturally infects mice and establishes chronic infections in them.

Intriguingly, recent data suggested that the MHV-68 infection cycle is also negatively influenced by ND10 as viral replication results in the disassembly of this subnuclear structure [155, 156]. The virion tegument protein ORF75c of MHV-68, a polypeptide with homology to the cellular formylglycinamide ribotide amidotransferase (FGARAT) enzyme, was identified as being responsible for this effect by inducing the proteasomal degradation of the main structural ND10-organizer PML [155]. Consequently, an ORF75c deletion mutant failed to destroy ND10 accumulations [156]. In accordance with the assumption that the ORF75c-induced depletion of PML is a means to annihilate the antiviral activity of ND10, MHV-68 replication turned out to be more robust in murine fibroblasts being deficient of PML [155]. However, although Ling and colleagues claimed that the degradation of PML by ORF75c is limited to murine fibroblasts and cannot be observed with human PML [155], we could also detect a complete elimination of the human PML variants upon infection of primary human fibroblast with MHV-68 (Tavalai, Full, Ensser and Stamminger, unpublished). Furthermore, the entire loss of PML as a consequence of MHV-68 infection presumably also accounted for a simultaneously observed disappearance of the low mobility isoforms of Sp100, comparable to the situation in HSV-1 infected cells (see 6.1.) or after depletion of the defining ND10 factor in PML-kd HFFs [14]. At the same time, MHV-68 also induced a significant reduction in hDaxx protein levels when compared to mock-infected cells. Although this effect on hDaxx is much less pronounced in comparison to MHV-68-mediated PML downregulation, it nevertheless mimics the findings for HCMV, where we were likewise never able to detect a total depletion of the transcriptional repressor hDaxx during the HCMV replicative cycle [129]. Thus, MHV-68 negatively affects the abundance of all three major ND10 constituents directly upon infection in order to efficiently circumvent the intrinsic antiviral response instituted by ND10. Given the fact, that MHV-68 naturally infects mice, it could represent a useful model system for investigation of ND10-herpesvirus interactions *in vivo*.

9. Concluding Remarks

ND10 have been identified as preferential replication sites of virtually all members of the herpesvirus family. However, the reason why herpesviruses interact with and commonly alter ND10 accumulations during their replicative cycle was for a long time quite controversially discussed. In principle, two conflicting theories had been established over the years that tried to explain the reason for the intimate virus-ND10 interaction during infection. Firstly, ND10 could harbor components or provide functions that are advantageous for the virus and support viral replication. Alternatively, ND10 structures could illustrate aggregations of proteins that compromise virus growth and which are therefore targeted by the virus for destruction to be inactivated.

Virus       | Viral protein   | ND 10 target | Effects
----------- | --------------- | ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
HSV-1       | ICP0            | PML          | Proteasomal degradation of all isoforms
            |                 | Sp100        | Loss of low mobility isoforms (indirect effect: linkage to PML metabolism)
VZV         | ORF61p          | Sp100        | Downregulation of all isoforms
HCMV        | IE1             | PML          | Loss of SUMOylated variants
            |                 | Sp100        | Loss of low mobility isoforms (indirect effect: linkage to PML metabolism)
            | pp71            | hDaxx        | Proteasomal degradation
            |                 | ATRX         | Release from ND 10
EBV         | BZLF-1          | PML          | Loss of SUMOylated variants during lytic replication
            | EBNA-LP         | Sp100        | Release from ND 10
            | EBNA1           | PML          | Proteasomal degradation during latent infection
KSHV        | LANA2           | PML          | Proteasomal degradation during latent infection
MHV-68      | ORF75c          | PML          | Proteasomal degradation of all isoforms
            |                 | Sp100        | Loss of low mobility isoforms (indirect effect: linkage to PML metabolism)
            |                 | hDaxx        | Downregulation

components, PML, Sp100 as well as hDaxx constitute host factors with antiviral activities. This assumption is most clearly supported by work on herpesviruses like HSV-1 (see 6.1.) or HCMV (see 7.1.), which show an enhanced infectivity in the absence of either of these basic ND10 constituents. In addition, recent findings indicate that also factors which only transiently localize to PML-NBs, like the SWI/SNF protein ATRX, contribute to the antiviral response instituted by ND10. Thus, it is tempting to speculate that, besides the structure defining components, still more ND10-related factors exist with antiviral properties, which await to be identified, yet.

Another characteristic aspect of cellular antiviral defense mechanisms is the fact that they are subject to viral countermeasures. Indeed, almost all herpesviruses for which replication has been linked to ND10, have evolved regulatory proteins that are capable of inactivating single ND10 components or disturbing the integrity of the whole subnuclear structure (see Figure 1 and Table 1). In many cases, these virus-induced alterations to ND10 composition have already been demonstrated to be important for the outcome of infection and correlate with the efficiency of viral propagation. Furthermore, it has been suggested that defeating ND10 function is not only a prerequisite for induction of an efficient productive replication but is also required for the establishment of latent infections as proposed in case of EBV (see 8.1.). Interestingly, recent data obtained from the herpesviruses EBV (see 8.1.) and KSHV (see 8.2.), in addition, imply that inactivation of ND10 may also contribute to the development of the tumors associated with these viruses. Consequently, ND10 seem to act at multiple steps of the virus life cycle in order to compromise the detrimental outcome of viral infections. Intriguingly, however, even closely related members of the herpesvirus family appear to have evolved their own specific strategies during virus-host co-evolution to cope with the antiviral aspects of ND10.

In summary, the future challenge will be to get a more detailed understanding of the underlying molecular mechanisms of ND10 as well as of the viral factors antagonizing its repressive function since elucidating the exact mode of action may help to develop novel therapeutic approaches against herpesviruses.

## Acknowledgements

We thank F. Full and A. Ensser (Erlangen) for collaboration on ND10 modification by MHV-68. Work presented in this review was supported by the Deutsche Forschungsgemeinschaft (SFBs 473 and 796), the IZKF Erlangen, the elite graduate school BIGSS, the GRK1071 and the Wilhelm Sander Stiftung.

## References

1. Bieniasz, P.D. Intrinsic immunity: a front-line defense against viral attack. *Nat. Immunol.* 2004, **5**, 1109–1115.
2. Negorev, D.; Maul, G.G. Cellular proteins localized at and interacting within ND10/PML nuclear bodies/PODs suggest functions of a nuclear depot. *Oncogene* 2001, **20**, 7234–7242.
3. Stuurman, N.; de Graaf, A.; Floore, A.; Josso, A.; Humbel, B.; de, J.L.; van Driel, R.A. monoclonal antibody recognizing nuclear matrix-associated nuclear bodies. *J. Cell Sci.* 1992, **101**, 773–784.

4. Ascoli, C.A.; Maul, G.G. Identification of a novel nuclear domain. J. Cell Biol. 1991, 112, 785–795.
5. Everett, R.D. Interactions between DNA viruses, ND10 and the DNA damage response. Cell Microbiol. 2006, 8, 365–374.
6. Dellaire, G.; Bazett-Jones, D.P. PML nuclear bodies: dynamic sensors of DNA damage and cellular stress. Bioessays 2004, 26, 963–977.
7. Dellaire, G.; Bazett-Jones, D.P. Beyond repair foci: subnuclear domains and the cellular response to DNA damage. Cell Cycle 2007, 6, 1864–1872.
8. Maul, G.G.; Jensen, D.E.; Ishov, A.M.; Herlyn, M.; Rauscher, F.J., III. Nuclear redistribution of BRCA1 during viral infection. Cell Growth Differ. 1998, 9, 743–755.
9. Wileman, T. Aggresomes and pericentriolar sites of virus assembly: cellular defense or viral design? Annu. Rev. Microbiol. 2007, 61, 149–167.
10. Fu, L.; Gao, Y.S.; Tousson, A.; Shah, A.; Chen, T.L.; Vertel, B.M.; Sztul, E. Nuclear aggresomes form by fusion of PML-associated aggregates. Mol. Biol. Cell 2005, 16, 4905–4917.
11. Szostecki, C.; Guldner, H.H.; Netter, H.J.; Will, H. Isolation and characterization of cDNA encoding a human nuclear antigen predominantly recognized by autoantibodies from patients with primary biliary cirrhosis. J. Immunol. 1990, 145, 4338–4347.
12. Zhong, S.; Muller, S.; Ronchetti, S.; Freemont, P.S.; Dejean, A.; Pandolfi, P.P. Role of SUMO-1-modified PML in nuclear body formation. Blood 2000, 95, 2748–2753.
13. Ishov, A.M.; Sotnikov, A.G.; Negorev, D.; Vladimirova, O.V.; Neff, N.; Kamitani, T.; Yeh, E.T.H.; Strauss, J.F.; Maul, G.G. PML is critical for ND10 formation and recruits the PML-interacting protein Daxx to this nuclear structure when modified by SUMO-1. J. Cell Biol. 1999, 147, 221–233.
14. Everett, R.D.; Rechter, S.; Papior, P.; Tavalai, N.; Stamminger, T.; Orr, A. PML contributes to a cellular mechanism of repression of herpes simplex virus type 1 infection that is inactivated by ICP0. J. Virol. 2006, 80, 7995–8005.
15. Tavalai, N.; Papior, P.; Rechter, S.; Leis, M.; Stamminger, T. Evidence for a role of the cellular ND10 protein PML in mediating intrinsic immunity against human cytomegalovirus infections. J. Virol. 2006, 80, 8006–8018.
16. Kakizuka, A.; Miller, W.H., Jr.; Umesono, K.; Warrell, R.P., Jr.; Frankel, S.R.; Murty, V.V.; Dmitrovsky, E.; Evans, R.M. Chromosomal translocation t(15;17) in human acute promyelocytic leukemia fuses RAR alpha with a novel putative transcription factor, PML. Cell 1991, 66, 663–674.
17. Jensen, K.; Shiels, C.; Freemont, P.S. PML protein isoforms and the RBCC/TRIM motif. Oncogene 2001, 20, 7223–7233.
18. Le, X.F.; Yang, P.; Chang, K.S. Analysis of the growth and transformation suppressor domains of promyelocytic leukemia gene, PML. J. Biol. Chem. 1996, 271, 130–135.
19. Nojima, T.; Oshiro-Ideue, T.; Nakanoya, H.; Kawamura, H.; Morimoto, T.; Kawaguchi, Y.; Kataoka, N.; Hagiwara, M. Herpesvirus protein ICP27 switches PML isoform by altering mRNA splicing. Nucleic Acids Res. 2009, 37, 6515–6527.
20. McNally, B.A.; Trgovcich, J.; Maul, G.G.; Liu, Y.; Zheng, P. A role for cytoplasmic PML in cellular resistance to viral infection. PLoS ONE. 2008, 3, e2277.

21. Everett, R.D.; Lomonte, P.; Sternsdorf, T.; van, D.R.; Orr, A. Cell cycle regulation of PML modification and ND10 composition. *J. Cell Sci.* 1999, **112**, 4581–4588.
22. Kamitani, T.; Nguyen, H.P.; Kito, K.; Fukuda-Kamitani, T.; Yeh, E.T. Covalent modification of PML by the sentrin family of ubiquitin-like proteins. *J. Biol. Chem.* 1998, **273**, 3117–3120.
23. Muller, S.; Matunis, M.J.; Dejean, A. Conjugation with the ubiquitin-related modifier SUMO-1 regulates the partitioning of PML within the nucleus. *EMBO J.* 1998, **17**, 61–70.
24. Kamitani, T.; Kito, K.; Nguyen, H.P.; Wada, H.; Fukuda-Kamitani, T.; Yeh, E.T. Identification of three major sentrinization sites in PML. *J. Biol. Chem.* 1998, **273**, 26675–26682.
25. Shen, T.H.; Lin, H.K.; Scaglioni, P.P.; Yung, T.M.; Pandolfi, P.P. The mechanisms of PML-nuclear body formation. *Mol. Cell* 2006, **24**, 331–339.
26. Lin, D.Y.; Huang, Y.S.; Jeng, J.C.; Kuo, H.Y.; Chang, C.C.; Chao, T.T.; Ho, C.C.; Chen, Y.C.; Lin, T.P.; Fang, H.I. Role of SUMO-interacting motif in Daxx SUMO modification, subnuclear localization, and repression of sumoylated transcription factors. *Mol. Cell* 2006, **24**, 341–354.
27. Hollenbach, A.D.; Sublett, J.E.; McPherson, C.J.; Grosveld, G. The Pax3-FKHR oncoprotein is unresponsive to the Pax3-associated repressor hDaxx. *EMBO J.* 1999, **18**, 3702–3711.
28. Kiriakidou, M.; Driscoll, D.A.; Lopez-Guisa, J.M.; Strauss, J.F., III Cloning and expression of primate Daxx cDNAs and mapping of the human gene to chromosome 6p21.3 in the MHC region. *DNA Cell Biol.* 1997, **16**, 1289–1298.
29. Li, H.; Leo, C.; Zhu, J.; Wu, X.; O'Neil, J.; Park, E.J.; Chen, J.D. Sequestration and inhibition of Daxx-mediated transcriptional repression by PML. *Mol. Cell Biol.* 2000, **20**, 1784–1796.
30. Pluta, A.F.; Earnshaw, W.C.; Goldberg, I.G. Interphase-specific association of intrinsic centromere protein CENP-C with HDaxx, a death domain-binding protein implicated in Fas-mediated cell death. *J. Cell Sci.* 1998, **111**, 2029–2041.
31. Salomoni, P.; Khelifi, A.F. Daxx: death or survival protein? *Trends Cell Biol.* 2006, **16**, 97–104.
32. Li, R.; Pei, H.; Watson, D.K.; Papas, T.S. EAP1/Daxx interacts with ETS1 and represses transcriptional activation of ETS1 target genes. *Oncogene* 2000, **19**, 745–753.
33. Park, J.; Lee, J.H.; La, M.; Jang, M.J.; Chae, G.W.; Kim, S.B.; Tak, H.; Jung, Y.; Byun, B.; Ahn, J.K. Inhibition of NF-kappaB acetylation and its transcriptional activity by Daxx. *J. Mol. Biol.* 2007, **368**, 388–397.
34. Michaelson, J.S.; Leder, P. RNAi reveals anti-apoptotic and transcriptionally repressive activities of DAXX. *J. Cell Sci.* 2003, **116**, 345–352.
35. Chang, C.C.; Lin, D.Y.; Fang, H.I.; Chen, R.H.; Shih, H.M. Daxx mediates the small ubiquitin-like modifier-dependent transcriptional repression of Smad4. *J. Biol. Chem.* 2005, **280**, 10164–10173.
36. Kim, E.J.; Park, J.S.; Um, S.J. Identification of Daxx interacting with p73, one of the p53 family, and its regulation of p53 activity by competitive interaction with PML. *Nucleic Acids Res.* 2003, **31**, 5356–5367.
37. Lin, D.Y.; Fang, H.I.; Ma, A.H.; Huang, Y.S.; Pu, Y.S.; Jenster, G.; Kung, H.J.; Shih, H.M. Negative modulation of androgen receptor transcriptional activity by Daxx. *Mol. Cell Biol.* 2004, **24**, 10529–10541.

Viruses 2009, 1

38. Hollenbach, A.D.; McPherson, C.J.; Mientjes, E.J.; Iyengar, R.; Grosveld, G. Daxx and histone deacetylase II associate with chromatin through an interaction with core histones and the chromatin-associated protein Dek. *J. Cell Sci.* 2002, **115**, 3319–3330.

39. Muromoto, R.; Sugiyama, K.; Takachi, A.; Imoto, S.; Sato, N.; Yamamoto, T.; Oritani, K.; Shimoda, K.; Matsuda, T. Physical and functional interactions between Daxx and DNA methyltransferase 1-associated protein, DMAP1. *J. Immunol.* 2004, **172**, 2985–2993.

40. Ishov, A.M.; Vladimirova, O.V.; Maul, G.G. Heterochromatin and ND10 are cell-cycle regulated and phosphorylation-dependent alternate nuclear sites of the transcription repressor Daxx and SWI/SNF protein ATRX. *J. Cell Sci.* 2004, **117**, 3807–3820.

41. Xue, Y.; Gibbons, R.; Yan, Z.; Yang, D.; McDowell, T.L.; Sechi, S.; Qin, J.; Zhou, S.; Higgs, D.; Wang, W. The ATRX syndrome protein forms a chromatin-remodeling complex with Daxx and localizes in promyelocytic leukemia nuclear bodies. *Proc. Natl. Acad. Sci. USA* 2003, **100**, 10635–10640.

42. Lin, D.Y.; Shih, H.M. Essential role of the 58-kDa microspherule protein in the modulation of Daxx-dependent transcriptional repression as revealed by nucleolar sequestration. *J. Biol. Chem.* 2002, **277**, 25446–25456.

43. Dent, A.L.; Yewdell, J.; Puvion-Dutilleul, F.; Koken, M.H.; de Thé, H.; Staudt, L.M. LYSP100-associated nuclear domains (LANDs): description of a new class of subnuclear structures and their relationship to PML nuclear bodies. *Blood* 1996, **88**, 1423–1426.

44. Seeler, J.S.; Marchio, A.; Losson, R.; Desterro, J.M.; Hay, R.T.; Chambon, P.; Dejean, A. Common properties of nuclear body protein SP100 and TIF1alpha chromatin factor: role of SUMO modification. *Mol. Cell Biol.* 2001, **21**, 3314–3324.

45. Seeler, J.S.; Marchio, A.; Sitterlin, D.; Transy, C.; Dejean, A. Interaction of SP100 with HP1 proteins: a link between the promyelocytic leukemia-associated nuclear bodies and the chromatin compartment. *Proc. Natl. Acad. Sci. USA* 1998, **95**, 7316–7321.

46. Guldner, H.H.; Szostecki, C.; Schroder, P.; Matschl, U.; Jensen, K.; Luders, C.; Will, H.; Sternsdorf, T. Splice variants of the nuclear dot-associated Sp100 protein contain homologies to HMG-1 and a human nuclear phosphoprotein-box motif. *J. Cell Sci.* 1999, **112**, 733–747.

47. Bottomley, M.J.; Collard, M.W.; Huggenvik, J.I.; Liu, Z.; Gibson, T.J.; Sattler, M. The SAND domain structure defines a novel DNA-binding fold in transcriptional regulation. *Nat. Struct. Biol.* 2001, **8**, 626–633.

48. Wilcox, K.W.; Sheriff, S.; Isaac, A.; Taylor, J.L. SP100B is a repressor of gene expression. *J. Cell Biochem.* 2005, **95**, 352–365.

49. Isaac, A.; Wilcox, K.W.; Taylor, J.L. SP100B, a repressor of gene expression preferentially binds to DNA with unmethylated CpGs. *J. Cell Biochem.* 2006, **98**, 1106–1122.

50. Jang, M.S.; Ryu, S.W.; Kim, E. Modification of Daxx by small ubiquitin-related modifier-1. *Biochem. Biophys. Res. Commun.* 2002, **295**, 495–500.

51. Sternsdorf, T.; Jensen, K.; Will, H. Evidence for covalent modification of the nuclear dot-associated proteins PML and Sp100 by PIC1/SUMO-1. *J. Cell Biol.* 1997, **139**, 1621–1634.

52. Sternsdorf, T.; Jensen, K.; Reich, B.; Will, H. The nuclear dot protein sp100, characterization of domains necessary for dimerization, subcellular localization, and modification by small ubiquitin-like modifiers. *J. Biol. Chem.* 1999, **274**, 12555–12566.

53. Salomoni, P.; Pandolfi, P.P. The role of PML in tumor suppression. *Cell* 2002, 108, 165–170.

54. Takahashi, Y.; Lallemand-Breitenbach, V.; Zhu, J.; de Thé, H. PML nuclear bodies and apoptosis. *Oncogene* 2004, 23, 2819–2824.

55. Bernardi, R.; Pandolfi, P.P. Role of PML and the PML-nuclear body in the control of programmed cell death. *Oncogene* 2003, 22, 9048–9057.

56. Maul, G.G.; Yu, E.; Ishov, A.M.; Epstein, A.L. Nuclear domain 10 (ND10) associated proteins are also present in nuclear bodies and redistribute to hundreds of nuclear sites after stress. *J. Cell Biochem.* 1995, 59, 498–513.

57. Bischof, O.; Kirsh, O.; Pearson, M.; Itahana, K.; Pelicci, P.G.; Dejean, A. Deconstructing PML-induced premature senescence. *EMBO J.* 2002, 21, 3358–3369.

58. Lallemand-Breitenbach, V.; Zhu, J.; Puvion, F.; Koken, M.; Honore, N.; Doubeikovsky, A.; Duprez, E.; Pandolfi, P.P.; Puvion, E.; Freemont, P. *et al.* Role of promyelocytic leukemia (PML) sumolation in nuclear body formation, 11S proteasome recruitment, and As2O3-induced PML or PML/retinoic acid receptor alpha degradation. *J. Exp. Med.* 2001, 193, 1361–1371.

59. Zhong, S.; Salomoni, P.; Pandolfi, P.P. The transcriptional role of PML and the nuclear body. *Nat. Cell Biol.* 2000, 2, E85–E90.

60. Borden, K.L. Pondering the promyelocytic leukemia protein (PML) puzzle: possible functions for PML nuclear bodies. *Mol. Cell Biol.* 2002, 22, 5259–5269.

61. Bernardi, R.; Pandolfi, P.P. Structure, dynamics and functions of promyelocytic leukaemia nuclear bodies. *Nat. Rev. Mol. Cell Biol.* 2007, 8, 1006–1016.

62. Eskiw, C.H.; Bazett-Jones, D.P. The promyelocytic leukemia nuclear body: sites of activity? *Biochem. Cell Biol.* 2002, 80, 301–310.

63. Boisvert, F.M.; Hendzel, M.J.; Bazett-Jones, D.P. Promyelocytic leukemia (PML) nuclear bodies are protein structures that do not accumulate RNA. *J. Cell Biol.* 2000, 148, 283–292.

64. Shih, H.M.; Chang, C.C.; Kuo, H.Y.; Lin, D.Y. Daxx mediates SUMO-dependent transcriptional control and subnuclear compartmentalization. *Biochem. Soc. Trans.* 2007, 35, 1397–1400.

65. Wu, W.S.; Vallian, S.; Seto, E.; Yang, W.M.; Edmondson, D.; Roth, S.; Chang, K.S. The growth suppressor PML represses transcription by functionally and physically interacting with histone deacetylases. *Mol. Cell Biol.* 2001, 21, 2259–2268.

66. Puto, L.A.; Reed, J.C. Daxx represses RelB target promoters via DNA methyltransferase recruitment and DNA hypermethylation. *Genes Dev.* 2008, 22, 998–1010.

67. Di Croce, L.; Raker, V.A.; Corsaro, M.; Fazi, F.; Fanelli, M.; Faretta, M.; Fuks, F.; Lo, C.F.; Kouzarides, T.; Nervi, C. Methyltransferase recruitment and DNA hypermethylation of target promoters by an oncogenic transcription factor. *Science* 2002, 295, 1079–1082.

68. Khan, M.M.; Nomura, T.; Kim, H.; Kaul, S.C.; Wadhwa, R.; Shinagawa, T.; Ichikawa-Iwata, E.; Zhong, S.; Pandolfi, P.P.; Ishii, S. Role of PML and PML-RARalpha in Mad-mediated transcriptional repression. *Mol. Cell* 2001, 7, 1233–1243.

69. Fleischer, S.; Wiemann, S.; Will, H.; Hofmann, T.G. PML-associated repressor of transcription (PAROT), a novel KRAB-zinc finger repressor, is regulated through association with PML nuclear bodies. *Exp. Cell Res.* 2006, 312, 901–912.

70. Carbone, R.; Botrugno, O.A.; Ronzoni, S.; Insinga, A.; Di Croce, L.; Pelicci, P.G.; Minucci, S. Recruitment of the histone methyltransferase SUV39H1 and its role in the oncogenic properties

of the leukemia-associated PML-retinoic acid receptor fusion protein. Mol. Cell Biol. 2006, 26, 1288–1296.

71. Villa, R.; Pasini, D.; Gutierrez, A.; Morey, L.; Occhionorelli, M.; Vire, E.; Nomdedeu, J.F.; Jenuwein, T.; Pelicci, P.G.; Minucci, S. *et al.* Role of the polycomb repressive complex 2 in acute promyelocytic leukemia. Cancer Cell 2007, 11, 513–525.

72. Boisvert, F.M.; Kruhlak, M.J.; Box, A.K.; Hendzel, M.J.; Bazett-Jones, D.P. The transcription coactivator CBP is a dynamic component of the promyelocytic leukemia nuclear body. J. Cell Biol. 2001, 152, 1099–1106.

73. LaMorte, V.J.; Dyck, J.A.; Ochs, R.L.; Evans, R.M. Localization of nascent RNA and CREB binding protein with the PML-containing nuclear body. Proc. Natl. Acad. Sci. USA 1998, 95, 4991–4996.

74. Everett, R.D. DNA viruses and viral proteins that interact with PML nuclear bodies. Oncogene 2001, 20, 7266–7273.

75. Moller, A.; Schmitz, M.L. Viruses as hijackers of PML nuclear bodies. Arch. Immunol. Ther. Exp. (Warsz.) 2003, 51, 295–300.

76. Maul, G.G. Nuclear domain 10, the site of DNA virus transcription and replication. Bioessays 1998, 20, 660–667.

77. Regad, T.; Chelbi-Alix, M.K. Role and fate of PML nuclear bodies in response to interferon and viral infections. Oncogene 2001, 20, 7274–7286.

78. Everett, R.D.; Chelbi-Alix, M.K. PML and PML nuclear bodies: implications in antiviral defence. Biochimie 2007, 89, 819–830.

79. Chelbi-Alix, M.K.; Pelicano, L.; Quignon, F.; Koken, M.H.; Venturini, L.; Stadler, M.; Pavlovic, J.; Degos, L.; de Thé, H. Induction of the PML protein by interferons in normal and APL cells. Leukemia 1995, 9, 2027–2033.

80. Guldner, H.H.; Szostecki, C.; Grotzinger, T.; Will, H. IFN enhance expression of Sp100, an autoantigen in primary biliary cirrhosis. J. Immunol. 1992, 149, 4067–4073.

81. Stadler, M.; Chelbi-Alix, M.K.; Koken, M.H.; Venturini, L.; Lee, C.; Saib, A.; Quignon, F.; Pelicano, L.; Guillemin, M.C.; Schindler, C. Transcriptional induction of the PML growth suppressor gene by interferons is mediated through an ISRE and a GAS element. Oncogene 1995, 11, 2565–2573.

82. Grotzinger, T.; Jensen, K.; Will, H. The interferon (IFN)-stimulated gene Sp100 promoter contains an IFN-gamma activation site and an imperfect IFN-stimulated response element which mediate type I IFN inducibility. J. Biol. Chem. 1996, 271, 25253–25260.

83. Kim, T.K.; Lee, J.S.; Oh, S.Y.; Jin, X.; Choi, Y.J.; Lee, T.H.; Lee, E.; Choi, Y.K.; You, S.; Chung, Y.G. Direct transcriptional activation of promyelocytic leukemia protein by IFN regulatory factor 3 induces the p53-dependent growth inhibition of cancer cells. Cancer Res. 2007, 67, 11133–11140.

84. Grotzinger, T.; Sternsdorf, T.; Jensen, K.; Will, H. Interferon-modulated expression of genes encoding the nuclear-dot-associated proteins Sp100 and promyelocytic leukemia protein (PML). Eur. J. Biochem. 1996, 238, 554–560.

85. Maul, G.G.; Guldner, H.H.; Spivack, J.G. Modification of discrete nuclear domains induced by herpes simplex virus type 1 immediate early gene 1 product (ICP0). *J. Gen. Virol.* 1993, **74**, 2679–2690.

86. Ishov, A.M.; Maul, G.G. The periphery of nuclear domain 10 (ND10) as site of DNA virus deposition. *J. Cell Biol.* 1996, **134**, 815–826.

87. Maul, G.G.; Ishov, A.M.; Everett, R.D. Nuclear domain 10 as preexisting potential replication start sites of herpes simplex virus type-1. *Virology* 1996, **217**, 67–75.

88. Everett, R.D.; Maul, G.G. HSV-1 IE protein Vmw110 causes redistribution of PML. *EMBO J.* 1994, **13**, 5062–5069.

89. Maul, G.G.; Everett, R.D. The nuclear location of PML, a cellular member of the C3HC4 zinc-binding domain protein family, is rearranged during herpes simplex virus infection by the C3HC4 viral protein ICP0. *J. Gen. Virol.* 1994, **75**, 1223–1233.

90. Everett, R.D. ICP0, a regulator of herpes simplex virus during lytic and latent infection. *Bioessays* 2000, **22**, 761–770.

91. Chelbi-Alix, M.K.; de Thé, H. Herpes virus induced proteasome-dependent degradation of the nuclear bodies-associated PML and Sp100 proteins. *Oncogene* 1999, **18**, 935–941.

92. Everett, R.D.; Freemont, P.; Saitoh, H.; Dasso, M.; Orr, A.; Kathoria, M.; Parkinson, J. The disruption of ND10 during herpes simplex virus infection correlates with the Vmw110- and proteasome-dependent loss of several PML isoforms. *J. Virol.* 1998, **72**, 6581–6591.

93. Muller, S.; Dejean, A. Viral immediate-early proteins abrogate the modification by SUMO-1 of PML and Sp100 proteins, correlating with nuclear body disruption. *J. Virol.* 1999, **73**, 5137–5143.

94. Parkinson, J.; Everett, R.D. Alphaherpesvirus proteins related to herpes simplex virus type 1 ICP0 affect cellular structures and proteins. *J. Virol.* 2000, **74**, 10006–10017.

95. Boutell, C.; Sadis, S.; Everett, R.D. Herpes simplex virus type 1 immediate-early protein ICP0 and is isolated RING finger domain act as ubiquitin E3 ligases in vitro. *J. Virol.* 2002, **76**, 841–850.

96. Gu, H.; Roizman, B. The degradation of promyelocytic leukemia and Sp100 proteins by herpes simplex virus 1 is mediated by the ubiquitin-conjugating enzyme UbcH5a. *Proc. Natl. Acad. Sci. USA* 2003, **100**, 8963–8968.

97. Harle, P.; Sainz, B., Jr.; Carr, D.J.; Halford, W.P. The immediate-early protein, ICP0, is essential for the resistance of herpes simplex virus to interferon-alpha/beta. *Virology* 2002, **293**, 295–304.

98. Mossman, K.L.; Saffran, H.A.; Smiley, J.R. Herpes simplex virus ICP0 mutants are hypersensitive to interferon. *J. Virol.* 2000, **74**, 2052–2056.

99. Halford, W.P.; Weisend, C.; Grace, J.; Soboleski, M.; Carr, D.J.; Balliet, J.W.; Imai, Y.; Margolis, T.P.; Gebhardt, B.M. ICP0 antagonizes Stat 1-dependent repression of herpes simplex virus: implications for the regulation of viral latency. *Virol. J.* 2006, **3**, 44.

100. Chee, A.V.; Lopez, P.; Pandolfi, P.P.; Roizman, B. Promyelocytic leukemia protein mediates interferon-based anti-herpes simplex virus 1 effects. *J. Virol.* 2003, **77**, 7101–7105.

101. Everett, R.D.; Zafiropoulos, A. Visualization by live-cell microscopy of disruption of ND10 during herpes simplex virus type 1 infection. *J. Virol.* 2004, **78**, 11411–11415.

102. Lopez, P.; Jacob, R.J.; Roizman, B. Overexpression of promyelocytic leukemia protein precludes the dispersal of ND10 structures and has no effect on accumulation of infectious herpes simplex virus 1 or its proteins. *J. Virol.* 2002, **76**, 9355–9367.

103. Everett, R.D.; Parada, C.; Gripon, P.; Sirma, H.; Orr, A. Replication of ICP0 null mutant herpes simplex virus type 1 is restricted by both PML and Sp100. *J. Virol.* 2007.

104. Negorev, D.G.; Vladimirova, O.V.; Maul, G.G. Differential functions of interferon-upregulated Sp100 isoforms: herpes simplex virus type 1 promoter-based immediate-early gene suppression and PML protection from ICP0-mediated degradation. *J. Virol.* 2009, **83**, 5168–5180.

105. Negorev, D.G.; Vladimirova, O.V.; Ivanov, A.; Rauscher, F., III; Maul, G.G. Differential role of Sp100 isoforms in interferon-mediated repression of herpes simplex virus type 1 immediate-early protein expression. *J. Virol.* 2006, **80**, 8019–8029.

106. Arvin, A.M. Varicella-zoster virus. *Clin. Microbiol. Rev.* 1996, **9**, 361–381.

107. Kyratsous, C.A.; Silverstein, S.J. Components of nuclear domain 10 bodies regulate varicella-zoster virus replication. *J. Virol.* 2009, **83**, 4262–4274.

108. Moriuchi, H.; Moriuchi, M.; Smith, H.A.; Straus, S.E.; Cohen, J.I. Varicella-zoster virus open reading frame 61 protein is functionally homologous to herpes simplex virus type 1 ICP0. *J. Virol.* 1992, **66**, 7303–7308.

109. Moriuchi, H.; Moriuchi, M.; Straus, S.E.; Cohen, J.I. Varicella-zoster virus (VZV) open reading frame 61 protein transactivates VZV gene promoters and enhances the infectivity of VZV DNA. *J. Virol.* 1993, **67**, 4290–4295.

110. Kyratsous, C.A.; Walters, M.S.; Panagiotidis, C.A.; Silverstein, S.J. Complementation of a Herpes Simplex Virus ICP0 null mutant by Varicella Zoster Virus ORF61p. *J. Virol.* 2009, **83**, 10637–10643.

111. Knipe, D.M.; Howley, P.; Griffin, D.E.; Lamb, R.A.; Martin, M.A.; Roizman, B.; Straus, S.E. *Fields Virology*, 5th ed.; Lippincott-Williams and Wilkins: Philadelphia, PA, USA, 2007.

112. Ahn, J.H.; Hayward, G.S. The major immediate-early proteins IE1 and IE2 of human cytomegalovirus colocalize with and disrupt PML-associated nuclear bodies at very early times in infected permissive cells. *J. Virol.* 1997, **71**, 4599–4613.

113. Ishov, A.M.; Stenberg, R.M.; Maul, G.G. Human cytomegalovirus immediate early interaction with host nuclear structures: definition of an immediate transcript environment. *J. Cell Biol.* 1997, **138**, 5–16.

114. Wilkinson, G.W.; Kelly, C.; Sinclair, J.H.; Rickards, C. Disruption of PML-associated nuclear bodies mediated by the human cytomegalovirus major immediate early gene product. *J. Gen. Virol.* 1998, **79**, 1233–1245.

115. Korioth, F.; Maul, G.G.; Plachter, B.; Stamminger, T.; Frey, J. The nuclear domain 10 (ND10) is disrupted by the human cytomegalovirus gene product IE1. *Exp. Cell Res.* 1996, **229**, 155–158.

116. Ahn, J.H.; Brignole, E.J., III; Hayward, G.S. Disruption of PML subnuclear domains by the acidic IE1 protein of human cytomegalovirus is mediated through interaction with PML and may modulate a RING finger-dependent cryptic transactivator function of PML. *Mol. Cell Biol.* 1998, **18**, 4899–4913.

117. Lee, H.R.; Kim, D.J.; Lee, J.M.; Choi, C.Y.; Ahn, B.Y.; Hayward, G.S.; Ahn, J.H. Ability of the human cytomegalovirus IE1 protein to modulate sumoylation of PML correlates with its

functional activities in transcriptional regulation and infectivity in cultured fibroblast cells. J. Virol. 2004, 78, 6527–6542.

118. Xu, Y.; Ahn, J.H.; Cheng, M.; apRhys, C.M.; Chiou, C.J.; Zong, J.; Matunis, M.J.; Hayward, G.S. Proteasome-independent disruption of PML oncogenic domains (PODs), but not covalent modification by SUMO-1, is required for human cytomegalovirus immediate-early protein IE1 to inhibit PML-mediated transcriptional repression. J. Virol. 2001, 75, 10683–10695.

119. Kang, H.; Kim, E.T.; Lee, H.R.; Park, J.J.; Go, Y.Y.; Choi, C.Y.; Ahn, J.H. Inhibition of SUMO-independent PML oligomerization by the human cytomegalovirus IE1 protein. J. Gen. Virol. 2006, 87, 2181–2190.

120. Ahn, J.H.; Hayward, G.S. Disruption of PML-associated nuclear bodies by IE1 correlates with efficient early stages of viral gene expression and DNA replication in human cytomegalovirus infection. Virology 2000, 274, 39–55.

121. Hofmann, H.; Sindre, H.; Stamminger, T. Functional interaction between the pp71 protein of human cytomegalovirus and the PML-interacting protein human Daxx. J. Virol. 2002, 76, 5769–5783.

122. Ishov, A.M.; Vladimirova, O.V.; Maul, G.G. Daxx-mediated accumulation of human cytomegalovirus tegument protein pp71 at ND10 facilitates initiation of viral infection at these nuclear domains. J. Virol. 2002, 76, 7705–7712.

123. Marshall, K.R.; Rowley, K.V.; Rinaldi, A.; Nicholson, I.P.; Ishov, A.M.; Maul, G.G.; Preston, C.M. Activity and intracellular localization of the human cytomegalovirus protein pp71. J. Gen. Virol. 2002, 83, 1601–1612.

124. Cantrell, S.R.; Bresnahan, W.A. Interaction between the human cytomegalovirus UL82 gene product (pp71) and hDaxx regulates immediate-early gene expression and viral replication. J. Virol. 2005, 79, 7792–7802.

125. Woodhall, D.L.; Groves, I.J.; Reeves, M.B.; Wilkinson, G.; Sinclair, J.H. Human Daxx-mediated repression of human cytomegalovirus gene expression correlates with a repressive chromatin structure around the major immediate early promoter. J. Biol. Chem. 2006, 281, 37652–37660.

126. Saffert, R.T.; Kalejta, R.F. Inactivating a cellular intrinsic immune defense mediated by Daxx is the mechanism through which the human cytomegalovirus pp71 protein stimulates viral immediate-early gene expression. J. Virol. 2006, 80, 3863–3871.

127. Preston, C.M.; Nicholl, M.J. Role of the cellular protein hDaxx in human cytomegalovirus immediate-early gene expression. J. Gen. Virol. 2006, 87, 1113–1121.

128. Cantrell, S.R.; Bresnahan, W.A. Human cytomegalovirus (HCMV) UL82 gene product (pp71) relieves hDaxx-mediated repression of HCMV replication. J. Virol. 2006, 80, 6188–6191.

129. Tavalai, N.; Papior, P.; Rechter, S.; Stamminger, T. Nuclear domain 10 components promyelocytic leukemia protein and hDaxx independently contribute to an intrinsic antiviral defense against human cytomegalovirus infection. J. Virol. 2008, 82, 126–137.

130. Hwang, J.; Kalejta, R.F. Proteasome-dependent, ubiquitin-independent degradation of Daxx by the viral pp71 protein in human cytomegalovirus-infected cells. Virology 2007, 367, 334–338.

131. Kaspari, M.; Tavalai, N.; Stamminger, T.; Zimmermann, A.; Schilf, R.; Bogner, E. Proteasome inhibitor MG132 blocks viral DNA replication and assembly of human cytomegalovirus. FEBS Lett. 2008, 582, 666–672.

132. Hwang, J.; Kalejta, R.F. Human cytomegalovirus protein pp71 induces Daxx SUMOylation. *J. Virol.* 2009, 83, 6591–6598.

133. Lukashchuk, V.; McFarlane, S.; Everett, R.D.; Preston, C.M. Human cytomegalovirus protein pp71 displaces the chromatin-associated factor ATRX from nuclear domain 10 at early stages of infection. *J. Virol.* 2008, 82, 12543–12554.

134. Bresnahan, W.A.; Shenk, T.E. UL82 virion protein activates expression of immediate early viral genes in human cytomegalovirus-infected cells. *Proc. Natl. Acad. Sci. USA* 2000, 97, 14506–14511.

135. Saffert, R.T.; Kalejta, R.F. Human cytomegalovirus gene expression is silenced by Daxx-mediated intrinsic immune defense in model latent infections established in vitro. *J. Virol.* 2007, 81, 9109–9120.

136. Groves, I.J.; Sinclair, J.H. Knockdown of hDaxx in normally non-permissive undifferentiated cells does not permit human cytomegalovirus immediate-early gene expression. *J. Gen. Virol.* 2007, 88, 2935–2940.

137. Sinclair, J. Chromatin structure regulates human cytomegalovirus gene expression during latency, reactivation and lytic infection. *Biochim. Biophys. Acta* 2009, doi:10.1016/j.bbagrm.2009.08.001.

138. Torok, D.; Ching, R.W.; Bazett-Jones, D.P. PML nuclear bodies as sites of epigenetic regulation. *Front Biosci.* 2009, 14, 1325–1336.

139. Groves, I.J.; Reeves, M.B.; Sinclair, J.H. Lytic infection of permissive cells with human cytomegalovirus is regulated by an intrinsic 'pre-immediate-early' repression of viral gene expression mediated by histone post-translational modification. *J. Gen. Virol.* 2009, 90, 2364–2374.

140. Cuevas-Bennett, C.; Shenk, T. Dynamic histone H3 acetylation and methylation at human cytomegalovirus promoters during replication in fibroblasts. *J. Virol.* 2008, 82, 9525–9536.

141. Raab-Traub, N. Epstein-Barr virus in the pathogenesis of NPC. *Semin. Cancer Biol.* 2002, 12, 431–441.

142. Bell, P.; Lieberman, P.M.; Maul, G.G. Lytic but not latent replication of epstein-barr virus is associated with PML and induces sequential release of nuclear domain 10 proteins. *J. Virol.* 2000, 74, 11800–11810.

143. Amon, W.; White, R.E.; Farrell, P.J. Epstein-Barr virus origin of lytic replication mediates association of replicating episomes with promyelocytic leukemia nuclear bodies and replication compartments. *J. Gen. Virol.* 2006, 87, 1133–1137.

144. Adamson, A.L.; Kenney, S. Epstein-barr virus immediate-early protein BZLF1 is SUMO-1 modified and disrupts promyelocytic leukemia bodies. *J. Virol.* 2001, 75, 2388–2399.

145. Deng, Z.; Chen, C.J.; Zerby, D.; Delecluse, H.J.; Lieberman, P.M. Identification of acidic and aromatic residues in the Zta activation domain essential for Epstein-Barr virus reactivation. *J. Virol.* 2001, 75, 10334–10347.

146. Peng, R.; Moses, S.C.; Tan, J.; Kremmer, E.; Ling, P.D. The Epstein-Barr virus EBNA-LP protein preferentially coactivates EBNA2-mediated stimulation of latent membrane proteins expressed from the viral divergent promoter. *J. Virol.* 2005, 79, 4492–4505.

147. Szekely, L.; Pokrovskaja, K.; Jiang, W.Q.; de Thé, H.; Ringertz, N.; Klein, G. The Epstein-Barr virus-encoded nuclear antigen EBNA-5 accumulates in PML-containing bodies. *J. Virol.* 1996, **70**, 2562–2568.

148. Ling, P.D.; Peng, R.S.; Nakajima, A.; Yu, J.H.; Tan, J.; Moses, S.M.; Yang, W.H.; Zhao, B.; Kieff, E.; Bloch, K.D. Mediation of Epstein-Barr virus EBNA-LP transcriptional coactivation by Sp100. *EMBO J.* 2005, **24**, 3565–3575.

149. Lehming, N.; Le, S.A.; Schuller, J.; Ptashne, M. Chromatin components as part of a putative transcriptional repressing complex. *Proc. Natl. Acad. Sci. USA* 1998, **95**, 7322–7326.

150. Echendu, C.W.; Ling, P.D. Regulation of Sp100A subnuclear localization and transcriptional function by EBNA-LP and interferon. *J. Interferon Cytokine Res.* 2008, **28**, 667–678.

151. Sivachandran, N.; Sarkari, F.; Frappier, L. Epstein-Barr nuclear antigen 1 contributes to nasopharyngeal carcinoma through disruption of PML nuclear bodies. *PLoS Pathog.* 2008, **4**, e1000170.

152. Wu, F.Y.; Ahn, J.H.; Alcendor, D.J.; Jang, W.J.; Xiao, J.; Hayward, S.D.; Hayward, G.S. Origin-independent assembly of Kaposi's sarcoma-associated herpesvirus DNA replication compartments in transient cotransfection assays and association with the ORF-K8 protein and cellular PML. *J. Virol.* 2001, **75**, 1487–1506.

153. Katano, H.; Ogawa-Goto, K.; Hasegawa, H.; Kurata, T.; Sata, T. Human-herpesvirus-8-encoded K8 protein colocalizes with the promyelocytic leukemia protein (PML) bodies and recruits p53 to the PML bodies. *Virology* 2001, **286**, 446–455.

154. Marcos-Villar, L.; Lopitz-Otsoa, F.; Gallego, P.; Munoz-Fontela, C.; Gonzalez-Santamaria, J.; Campagna, M.; Shou-Jiang, G.; Rodriguez, M.S.; Rivas, C. Kaposi's sarcoma-associated herpesvirus protein LANA2 disrupts PML oncogenic domains and inhibits PML-mediated transcriptional repression of the survivin gene. *J. Virol.* 2009, **83**, 8849–8858.

155. Ling, P.D.; Tan, J.; Sewatanon, J.; Peng, R. The murine gammaherpesvirus 68 open reading frame 75c tegument protein induces the degradation of PML and is essential for production of infectious virus. *J. Virol.* 2008.

156. Gaspar, M.; Gill, M.B.; Losing, J.B.; May, J.S.; Stevenson, P.G. Multiple functions for ORF75c in murid herpesvirus-4 infection. *PLoS ONE.* 2008, **3**, e2781.

© 2009 by the authors; licensee Molecular Diversity Preservation International, Basel, Switzerland. This article is an open-access article distributed under the terms and conditions of the Creative Commons Attribution license (http://creativecommons.org/licenses/by/3.0/).

Copyright of Viruses (1999-4915) is the property of Molecular Diversity Preservation International (MDPI) and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.
