
# Principles of microRNA involvement in human cancers

Hui Ling¹, Wei Zhang² and George A. Calin¹

## Abstract

Naturally occurring microRNAs (miRNAs), small non-coding RNAs of 19 to 24 nucleotides (nt), are encoded in the genomes of invertebrates, vertebrates, and plants. miRNAs act as regulators of gene expression during development and differentiation at the transcriptional, posttranscriptional, and/or translational levels, although most target genes are still elusive. Many miRNAs are conserved in sequence between distantly related organisms, suggesting that these molecules participate in essential processes. In this review, we present principles related to the basic and translational research that has emerged in the last decade, a period that can be truly considered the “miRNA revolution” in molecular oncology. These principles include the regulation mechanism of miRNA expression, functions of miRNAs in cancers, diagnostic values and therapeutic potentials of miRNAs. Furthermore, we present a compendium of information about the main miRNAs that have been identified in the last several years as playing important roles in cancers. Also, we orient the reader to several additional reviews that may provide a deeper understanding of this new and exciting field of research.

## Key words

MicroRNA, non-coding RNA, cancer, mutation, biomarker

Structurally, microRNAs (miRNAs) are 19- to 24-nucleotide (nt) RNAs processed from much longer primary transcripts (hundreds to thousands of nucleotides) that arise from hairpin loop structures (60- to 110-nt) after successive enzymatic maturation steps by the ribonucleases Drosha in the nucleus and Dicer in the cytoplasm [1].

Functionally, miRNAs regulate gene expression in a sequence-specific fashion. Initially transcribed by RNA polymerase II as long, capped, polyadenylated primary miRNAs (pri-miRNAs), miRNAs undergo a complex processing mechanism. First, the double-stranded RNA-specific ribonuclease Drosha, in conjunction with its binding partner DiGeorge syndrome critical region gene 8 (DGCR8, or Pasha), processes pri-miRNAs into hairpin RNAs of 60- to 110-nt known as pre-miRNAs. Translocated from the cell nucleus to the cytoplasm by Exportin 5, pre-miRNA is processed by a ribonuclease III (Dicer) and transactivating response RNA-binding protein (TRBP, which binds human immunodeficiency virus 1) into an 18- to 24-nt duplex. Finally, the duplex interacts with a large protein, RNA-induced silencing complex (RISC), which includes argonaute proteins (AGO1-4 in humans). One strand of the miRNA duplex remains stably associated with RISC and becomes a mature miRNA, which guides the RISC complex mainly (but not exclusively) to the 3'-untranslated region (3'-UTR) of target mRNAs. Consequently, the translation and/or stability of mRNAs are impaired, causing a reduction in protein expression levels [2].

Evidences are emerging that miRNAs’ effects on gene expression may be more varied than initially proposed. For example, miRNAs can activate rather than suppress mRNA expression in particular cell cycle conditions. Upon cell cycle exit, *miR-363-3* and *let-7*, via recruitment and modification of specific micro-ribonucleoproteins (RNPs) such as AGO2 and

---

**Authors’ Affiliations:**  
¹Department of Experimental Therapeutics, ²Department of Pathology, The University of Texas MD Anderson Cancer Center, Houston, TX 77030, USA.

**Corresponding Authors:**  
George A. Calin, Department of Experimental Therapeutics, Unit 1950, The University of Texas MD Anderson Cancer Center, 1881 East Road, Houston, TX 77054, USA. Tel: +1-713-792-5461; Fax: +1-713-745-4528; Email: gcalin@mdanderson.org;  
Wei Zhang, Department of Pathology, Unit 0085, The University of Texas MD Anderson Cancer Center, 1515 Holcombe Boulevard, Houston, TX 77030, USA. Tel: +1-713-745-1103; Fax: +1-713-792-5549; Email: wzhang@mdanderson.org.

doi: 10.5732/cjc.011.10243

www.cjcsysu.com

fragile X mental retardation autosomal homolog 1 (FXR1) to AU-rich elements inside mRNA 3'-UTRs, can turn on the translation of proteins that they normally repress during cell proliferation [3]. Furthermore, miRNA-induced mRNA repression was also found to occur via binding sites located inside mRNA-coding sequences, as shown for miRNAs regulating fundamental processes such as embryonic stem cell differentiation [4]. Additionally, specific miRNAs that carry a distinct hexanucleotide terminal motif, such as miR-29b, were found to be enriched in the cell nucleus, suggesting extra miRNA functions in different subcellular compartments [5]. It has been shown that miRNAs in the nucleus may act at the promoter level, affecting transcription; for example, miR-373 binds to the cadherin 1 (CDH1) promoter and stimulates transcription [6].

Finally, miRNAs can directly interact with proteins; for example, the interaction of miR-368 with heterogeneous nuclear ribonucleoprotein (hnRNP) E2 is independent of the miRNA seed sequence, and this interaction leads to the release of CCAAT/enhancer binding protein alpha (CEBPA) mRNA from hnRNP E2-mediated translational inhibition [7]. Because each miRNA has hundreds or thousands of targets and the full coding genome is probably under the control of miRNAs, miRNAs may be involved in any type of physiologic process and pathway, such as B-cell lineage fate (miR-181), B-cell survival (miR-15a and miR-16-1), cell proliferation control (miR-125b and let-7), brain patterning (miR-430), pancreatic cell insulin secretion (miR-375), and adipocyte development (miR-145) [8].

In this review, we summarize the main principles of miRNA involvement in human cancers. These principles are important not only for scientists in general but also for oncologists, as the field of non-coding RNAs (ncRNAs) has already been shown to touch every aspect of human oncology. We also briefly introduce other categories of ncRNAs that are important in human cancers.

### Principles of miRNA Involvement in Human Cancers

#### Regulation mechanisms of miRNA expression

**miRNA alterations are ubiquitous** miRNAs are involved in the pathophysiology of all types of human tumors, both benign and malignant. miRNAs differentially expressed between tumors and normal tissues have been identified in lymphomas, breast cancers, lung cancers, papillary thyroid carcinomas, glioblastomas, hepatocellular carcinomas, pancreatic tumors, pituitary adenomas, cervical cancers, brain tumors, prostate cancers, kidney and bladder cancers, and colorectal cancers [9]. Furthermore, miRNA alterations have been identified in other human diseases, including cardiac and autoimmune disorders and psychiatric conditions such as schizophrenia [10,11]. The development of high-throughput profiling methods to detect miRNA expression in human tissues has provided invaluable tools to investigate the roles of miRNAs in both physiologic and pathologic conditions (Table 1) [12]. Data exponentially accumulated in the last 9 years clearly show that miRNA alterations play a critical role in cancer initiation and progression (Table 2) [13]. Recently, investigators using high-throughput profiling techniques observed a link in cytogenetically well-defined chronic lymphocytic leukemia (CLL) samples between the level of expression of miR-34a, a member of a miRNA family positively regulated by tumor protein 53 (TP53) gene, and the level of response to DNA damage, the TP53 status, and significantly, the degree of response to fludarabine-based treatment. Low miR-34a expression level was significantly associated with impaired DNA damage response, p53 mutations, and resistance to fludarabine, either with or without p53 deletion. Up-regulation of miR-34a expression after irradiation was associated with induction of B-cell lymphoma-associated X protein (Bax) and P21 expression but not P53-up-regulated modulator of apoptosis (Puma) expression [14]. These findings are straightforward and provide a new piece to the recently identified puzzle of miRNA involvement in drug resistance and sensitivity in patients with CLL.

**Multiple mechanisms causing abnormal miRNA expression** The main mechanism of the microRNoma (a term we coined for the full complement of miRNAs present in a genome) alteration in cancer cells is represented by aberrant gene expression, characterized by abnormal expression of mature and/or precursor miRNA sequences in comparison with corresponding normal tissues. This abnormal expression is caused by various mechanisms that can act independently or in combination, such as localization of miRNAs at cancer-associated genomic regions (CAGR) [15], epigenetic regulation of miRNA expression [16], or development of abnormalities in miRNA-processing genes and proteins, including mutations in Dicer, TRBP, and Exportin 5 [17]. In various types of tumors, sometimes multiple mechanisms account for the deregulation of a specific miRNA. For example, the tumor suppressor miR-34a is positively controlled by TP53 [18], is kept in check by Myc [19], is silenced by aberrant CpG methylation [20], and is located at 1p36 [21], a chromosomal region frequently lost in neuroblastomas. Accordingly, numerous genetic studies have identified miRNA abnormalities in human cancers by dissecting their transcriptional regulators [19,21,22]. Cancer-associated miRNAs have been located downstream of major oncogenes and tumor suppressor genes (TSGs)

Table 1. Main microRNA-profiling technologies used at the time of this writing [12]

| Type                     | Principle                                                                                   | Advantages                                                                                          | Disadvantages                                                                                       |
|--------------------------|--------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------|
| miRNA microarray         | miRNA chips are fabricated on polymer-coated surface of glass slides with mature miRNA gene-specific oligonucleotide probes. The chemical covalent-immobilized probes hybridize with biotin-labeled cDNA targets. The signal of the probe-target complex is amplified by staining of Streptavidin Alexa 647 conjugates that affinity bind to biotins of the probe-target complex and are detected by laser scanning. | High-throughput miRNA expression genome-wide profiling is concomitantly achieved on large sample collections processed in parallel using standardized procedures and conditions for data comparison. | Probes on solid substrates affect target hybridization kinetics and discrimination in detecting differences between the 5' and 3' ends of highly similar miRNAs in tissues. The dynamic range of microarray data is relatively compressed, about 2.5 orders of magnitude compared with other techniques. |
| Bead-based technology    | Polystyrene beads coated with antisense oligonucleotide probes hybridize with biotin-labeled PCR amplicon dsDNA as targets. Staining with Streptavidin phycoerythrin is followed by bead flow cytometry for signal detection. | Solution-phase probe/target hybridization kinetics, which allows for high discrimination among closely related miRNA sequences | Low-throughput profiling for subsets of miRNA to be analyzed per experiment. More bias could be introduced in sample preparation by enrichment, adaptor ligation, and PCR steps. Competitive hybridization between probes and targets of double-stranded PCR amplicons occurs. |
| Stem-loop qRT-PCR for mature miRNA | Stem-loop primer cDNA reverse transcription followed by quantitative TaqMan-based real-time PCR | High sensitivity and specificity; reliable quantitative results, useful in confirming microarray results; low cost | Low-throughput profiling for only a subset of miRNAs |
| qRT-PCR for precursor miRNA | Precursor gene-specific primers cDNA synthesis followed by quantitative real-time PCR | High sensitivity; high specificity; quantitative data | Low-throughput profiling for only a specific subset of miRNAs |
| Next generation sequencer | Genome-wide sequencing of size-selected small RNA | Ultra-throughput for miRNA and small ncRNA profiling and discovery | Data analysis is challenging. |

miRNA, microRNA; cDNA, complementary DNA; PCR, polymerase chain reaction; dsDNA, double-stranded DNA; qRT-PCR, quantitative real-time PCR.

with transcription factor activities; for example, TP53 inhibits tumorigenesis via the transcription of all members of the miR-34 family [23], and Myc promotes tumorigenesis by both positively and negatively regulating the transcription of different miRNAs (e.g., miR-17~92 cluster and let-7 family, respectively) [19]. Similarly, transcription factors that govern the programming of metastatic gene expression have been found to regulate miRNAs. For example, the pleiotropic transcription factor TWIST 1 transactivates the prometastatic miR-10b [24], and SMAD 4, which is downstream of transforming growth factor beta (TGF-β) signaling, activates miR-155 [25].

Functions of miRNAs in human cancers

miRNAs as both oncogenes and tumor suppressor genes Various mechanisms allow miRNAs to act as

either oncogenes (such as miR-21 and miR-155, which cause acute B-cell leukemia in transgenic mice models) or as TSGs (such as the miR-15a/16-1 cluster, whose deletion causes CLL in knockout mice). In particular, while some miRNAs act mainly as TSGs, other miRNAs are frequently overexpressed in human cancers and target TSGs, thereby exerting a tumorigenic function. The miR-15a/16-1 cluster, for instance, controls the expression of about 14% of all genes in the human genome [26], and these miRNAs act as TSGs in CLL by targeting the antiapoptotic gene Bcl-2 [27]. miRNAs with well-established roles as oncogenes include the miR-17~92 cluster, which is transactivated by the c-Myc oncogene and dramatically accelerates lymphomagenesis in murine models [28,29]; miR-155, which induces leukemia in transgenic murine models [30] and has an important function as a regulator of inflammation and immune response [31-33]; and miR-21, which targets important TSGs

Table 2. MicroRNA deregulation in human cancers [13]

| Human microRNA | Putative function/involved pathways | Deregulation in tumors | Molecular mechanisms and targets | Diagnostic and prognostic markers |
|-----------------|------------------------------------|------------------------|-----------------------------------|-------------------------------------|
| let-7 family    | Antitumorigenic:                   | - Down-regulation in lung, breast, gastric, ovarian, prostate, and colon cancers, CLL, and leiomyomas | Molecular mechanism:                | Poor prognosis:                     |
|                 | - Self-sufficiency in growth signals | - Insensitivity to antigrowth signals | - Represses cell proliferation/growth | - let-7a-2 low expression in        |
|                 |                                     | - Angiogenesis          | - let-7f promotes angiogenesis     | lung and ovarian cancers            |
|                 |                                     |                        | - Down-regulation of miR-98         | - let-7b discriminates high-risk uveal melanomas |
|                 |                                     |                        | Targets: CCND1, CDC25a, CDK6, CRD-BP, HOXA9, IMP-1, MYC, RAS, TLR4 | Drug resistance:                    |
|                 |                                     |                        | - Point mutation in the let-7e precursor sequence affects maturation | - let-7i affects chemotherapy potency |
|                 | Oncogenic:                         | - Hypomethylation of let-7a-3 in lung adenoma carcinomas | Molecular mechanism:                | Therapy:                            |
|                 | - Self-sufficiency in growth signals | - Overexpression in AML | - let-7a represses NF2 and decreases chemotherapy-induced apoptosis in vitro | - Intranasal delivery of let-7a adenovirus reduces growth of Ras-induced lung tumors in mice |
| miR-16-1/15a cluster (13q14.3, intron4ncRNA DLEU2) | Antitumorigenic:       | - Down-regulation in CLL, DLBCLs, multiple myeloma, pituitary adenoma, and prostate and pancreatic cancers | Molecular mechanism:                | Poor prognosis:                     |
|                 | - Self-sufficiency in growth signals | - Evasion from apoptosis | - Induce apoptosis in leukemia and prostate cancer cells | - miR-15a and miR-16 high expression in de novo aggressive CLL |
|                 |                                     |                        | - miR-16 regulates cell cycle by downregulation of G0/G1 proteins | Drug resistance:                    |
|                 |                                     |                        | - Germline mutations in B-CLL patients and NZB mouse strain | - miR-16 affects chemotherapy potency, and modulates sensitivity to vincristine in gastric cancer cell lines |
|                 |                                     |                        | Targets: ACVR2A(X. tropicalis), BCL2, CARD10, CCND1, CDK6, CDC27, CGI-38, DMTF1, MCL1, NGN2, VEGF, WNT3A |                                      |
| miR-17/18a/19a/20a/92 cluster (13q31.3, intron3C13orf25) | Oncogenic:             | - Overexpression in lung cancers and lymphomas | Molecular mechanism:                |                                      |
|                 | - Insensitivity to antigrowth signals | - Angiogenesis          | - miR-17, -18a, -19a, -20a, and -19b-1 accelerate tumor growth and increase tumor vascularization |                                      |
|                 |                                     |                        | - miR-20a has an antiapoptotic role-lymphoproliferative disease and autoimmunity in transgenic miR-17/92 cluster mice with increased expression in lymphocytes |                                      |
|                 |                                     |                        | Targets: AIB1, AML1, BIM1, CTGF, CDKN1A, E2F1, E2F2, E2F3, PTEN, TGFB R2, TSP1, Rb2/P130 |                                      |
|                 | Antitumorigenic:                   | - LOH of miR-17/92 locus in ovarian (16.5%), breast carcinoma (21.9%), and melanoma (20%) | Molecular mechanism:                |                                      |
|                 | - Self-sufficiency in growth signals |                        | - miR-17 reduces proliferation in breast cancer cells | Targets: AIB1                       |

Table 2. MicroRNA deregulation in human cancers [13] (continued)

| Human microRNA | Putative function/involved pathways | Deregulation in tumors | Molecular mechanisms and targets | Diagnostic and prognostic markers |
|-----------------|------------------------------------|------------------------|-----------------------------------|-------------------------------------|
| miR-21          | Oncogenic:                        | Overexpression in      | Molecular mechanism:             | Poor prognosis:                    |
|                 | - Self-sufficiency in growth      | glioblastomas and breast, | -miR-21 knockdown induces apoptosis in glioblastoma, hepatocarcinoma, lung and breast cancer cells | -miR-21 high expression in colon and breast cancers |
|                 | signals                           | lung, prostate, colon, gastric, esophageal, and cervical |                                   | Good prognosis:                    |
|                 | - Evasion from apoptosis          | carcinomas, uterine     | -miR-21 modulates K-Ras-dependent lung tumorigenesis; miR-21 induces invasion and metastasis in colorectal cancers | Drug resistance:                   |
|                 | - Invasion and metastasis         | leiomyosarcoma, and DLBCL |                                   | -miR-21 affects chemotherapy potency in NCI60 cells |
|                 |                                  |                        | Targets: BCL2, MASPIN, PDCD4, PTEN, TPM1, RECK, SERPINB5 |                                 |
| miR-29 family   | Antitumorigenic:                  | Down-regulation in CLL and | Molecular mechanism:             | Poor prognosis:                    |
| (various)       | - Self-sufficiency in growth      | colon, breast, and lung | -miR-29 family reverts aberrant methylation in lung cancer | -miR-29 low expression correlates with short interval from diagnosis to therapy in CLL |
|                 | signals                           | cancers and in         |                                   |                                   |
|                 | - Invasion and metastasis         | cholangiocarcinoma tumor | -miR-29 activates p53 and induces apoptosis |                                   |
|                 |                                  | models (KMCH)           | Targets: DNMT3A and B, DNMT1, MCL1, TCL1, CDK6, p85α, CDC42 |                                 |
| miR-34 family   | Antitumorigenic:                  | Down-regulation in     | Molecular mechanism:             |                                   |
| (1p36.23, and   | - Self-sufficiency in growth      | pancreatic cancer cell lines | -miR-34a induces up-regulation of TP53 pathway and down-regulation of E2F pathway in colon cancer cell lines |                                   |
| 11q23.1,        | signals                           |                        |                                   |                                   |
| intergenic)     | - Insensitivity to antigrowth     | c in colon cancer      |                                   |                                   |
|                 | signals                           |                        | Targets: BCL2, CCND1, CCNE2, CDK4/6, DLL1, E2F3, Notch1, MYCN, MET, HMGA2, SIRT1 |                                 |
|                 | - Evasion from apoptosis          |                        |                                   |                                   |
|                 | - Limitless replicative potential |                        |                                   |                                   |
| miR-143/145 cluster | Antitumorigenic:                | Down-regulation in colon adenomas and carcinomas, breast and lung cancers, cervical cancer, and B-cell malignancies | Molecular mechanism:             |                                   |
| (intergenic, 5q32); signals | - Self-sufficiency in growth signals |                                      | -miR-143 and miR-145 precursors are abnormally processed in colon cancer | Targets: ERK5, HOXA9, PARP8 |                                 |
| miR-155         | Oncogenic:                       | Overexpression in pediatric BL, Hodgkin's disease, primary mediastinal lymphomas, and DLBCL and in breast, lung, colon, and pancreatic cancers | Molecular mechanism:             | Poor prognosis:                    |
| (21q21.3,       | - Evasion from apoptosis          |                        | - Pre-B cell proliferation and lymphoma grade in miR-155 aggressive CLL | -miR-155 high expression in lung cancer, DLBCL, and |
| exon 3          |                                  |                        |                                   |                                   |
| ncRNA BIC)      |                                  |                        | Targets: AGTR1, AID, TP53INP1, RHOA |                                   |
| miR-181 family  | Oncogenic/Antitumorigenic:        | Overexpression in breast, pancreatic, and prostate cancers | Molecular mechanism:             | Poor prognosis:                    |
| (various)       | - Self-sufficiency in growth      |                        | - MYCN regulates the transcription of miR-181 cluster | -miR-181 low expression in aggressive CLL with 11q deletions |
|                 | signals                           |                        |                                   | -miR-181a high expression correlates with short interval from diagnosis to therapy in CLL |
| miR-221/222 cluster | Oncogenic:                      | Overexpression in CLL, thyroid papillary carcinoma, and glioblastoma | Molecular Mechanism:             |                                   |
| (Xp11.3,        | - Insensitivity to antigrowth     |                        | - Promotes cancer cell proliferation dependent response |                                   |
| intergenic)     | signals, angiogenesis            |                        | -miR-221/222 impair TRAIL-Overexpression contributes to liver tumorigenesis |                                   |
|                 | - Down-regulation in AML          |                        | Targets: c-KIT, P27/CDKN1, PTEN |                                   |

CLL, chronic lymphocytic leukemia; AML, acute myelogenous leukemia; NF2, neurofibromatosis type 2; ncRNA, non-coding RNA; DLBCL, diffuse large B-cell lymphoma; B-CLL, B-cell CLL; NZB, New Zealand black; LOH, loss of heterozygosity; 3'-UTR, 3'-untranslated region; E2F, E2 transcription factor; BL, bone lymphocyte; TRAIL, tumor necrosis factor-related apoptosis-inducing ligand; TGF-β, transforming growth factor beta.

such as PTEN$^{[34]}$ and programmed cell death 4 (PDCD4) in several neoplasms$^{[35-37]}$. In some instances, the same miRNA acts as an oncogene in one type of cells and as a TSG in others because the targets and mechanisms of action differ. For example, miR-221 is hyperexpressed in liver cancers because the TSG PTEN is targeted$^{[38]}$, but down-regulated in erythroid leukemias because the oncogene c-Kit is targeted$^{[39]}$ (for other examples, see Table 2). Consequently, it has become evident that miRNAs can affect all the hallmarks of malignant cells: 1) self-sufficiency in growth signals (let-7 family), 2) insensitivity to antigrowth signals (miR-17~92 cluster), 3) evasion of apoptosis (miR-34a), 4) limitless replicative potential (miR-372/373 cluster), 5) angiogenesis (miR-210), and 6) invasion and metastasis (miR-10b).

**Mutations affecting the miRNA-target complex**

Germline and somatic mutations in active pre- or pri-miRNAs may contribute to cancer predisposition and initiation (such as the miR-15a/16 cluster mutations that occur in rare families with high incidences of both CLL and breast cancer). In the initial report of sequence variations in miRNAs, we reported on two patients diagnosed with CLL, one of whom had a family history of CLL and breast cancer, a C-T homozygous substitution in the pri-miR-16-1, and 7 nt in the 3' direction after the end of the pre-miRNA$^{[40]}$. We found that this substitution was associated with low levels of mature miR-16 production, revealing a functional impact on the processing of this miRNA. Mutation in a nearly identical location in the 3'-flanking region of miR-16-1 was described in the New Zealand black (NZB) mouse, a model for human CLL that spontaneously develops the disease when it ages$^{[41]}$. In another example, scientists screened for genetic variants in 17 selected miRNAs, which were predicted to regulate key breast cancer genes, in 42 patients with familial breast cancer$^{[42]}$. They identified 7 new variants, 2 in pre-miRNAs (pre-miR-30c-1 and pre-miR-21) and 5 in pri-miRNAs (pri-miR-17, pri-miR-24-1, pri-miR-125a, pri-miR-191, and pri-miR-125b-1). Interestingly, the variants in pre-miR-30c-1 and pri-miR-17 were rare and were only observed in non-carriers of BRCA1/2 mutations. Since miR-17 can target BRCA1 and the described variant affects the processing of miR-17$^{[42]}$, miRNA genomic variations can potentially alter the regulation of key breast cancer genes.

Furthermore, polymorphisms in mRNAs targeted by miRNAs influence cancer risk. For example, the let-7 complementary single nucleotide polymorphism (SNP) sites in the K-Ras 3'-UTR that was found to be significantly associated with an increased risk of non-small cell lung carcinoma in moderate smokers$^{[43,44]}$. Genome-wide bioinformatics analysis has predicted that approximately 64% of transcribed SNPs are target SNPs

that can modify (increase or decrease) the binding energy of putative miRNA-mRNA duplexes by more than 90%. To assess whether target SNPs are implicated in breast cancer susceptibility, we conducted a case-control population study and observed that germline occurrence of rs799917-BRCA1 and rs334348-transforming growth factor receptor 1 (TGFR1) significantly varied among populations with different risks of developing breast cancer$^{[45]}$. Luciferase activity of target SNPs, allelic variants, and protein levels in cancer cell lines with different genotypes showed differential regulation of target genes following overexpression of the two interacting miRNAs (miR-638 and miR-628-5p)$^{[45]}$. We also reported that a miR-502-binding site SNP in the 3'-UTR of the SET8 gene is associated with early age of breast cancer onset$^{[46]}$. Recently, we found that genetic variants at the miR-124- binding site on the cytoskeleton-organizing IQGAP1 gene confer differential predisposition to breast cancer$^{[47]}$.

### miRNA detection as a diagnostic tool

#### Profiling miRNAs—a clinical tool

The various methods of miRNA profiling have allowed the identification of miRNA signatures associated with diagnosis, staging, prognosis, and treatment response of human cancers. For example, a specific miRNA expression signature consisting of 13 miRNAs in human CLL was reported to link to disease progression from the time of diagnosis to the time of therapy$^{[40]}$. Similarly, a signature of aberrant expression of 11 miRNAs was found to correlate well with the survival rate of patients with acute myelogenous leukemia (AML)$^{[48]}$. In another study, researchers identified miRNA expression patterns associated with the incidence, prognosis, and therapeutic outcomes of colon adenocarcinoma by using cancer-specific death as the end point; they reported that miRNAs were differentially expressed in adenomas and adenocarcinomas of the colon and that miRNA expression patterns were associated with survival$^{[49]}$. They concluded that miRNA expression patterns are systematically altered in colon adenocarcinomas$^{[49]}$. Metastatic cancer from an unknown primary (CUP) is one of the 10 most frequent cancer diagnoses worldwide and constitutes 3% to 5% of all human malignancies. Patients with CUP present with metastases (i.e., late-stage disease) without an established primary tumor (i.e., a site wherein a therapeutically curative or palliative intervention can be performed). By analyzing 17 poorly differentiated metastatic CUPs with non-diagnostic histological appearance, researchers showed that the miRNA pattern was much better at establishing the correct diagnosis than the mRNA criteria$^{[50]}$. This result is exciting as it shows that profiling a few hundreds miRNAs has a much greater predictive power for CUP diagnosis than does

profiling tens of thousands of mRNAs for primary coding genes.

**Measurement of plasma miRNAs—a new tool for clinicians** Measurement of miRNAs in body fluids such as plasma and serum may represent a gold mine of noninvasive analysis of biomarkers in cancers. It has been shown consistently that serum miRNAs remain stable after being subjected to severe conditions that would normally degrade most RNAs, such as boiling, very low or high pH levels, extended storage, and 10 freeze-thaw cycles [51]. Researchers have shown a correlation between circulating miRNA expression levels and response to a given anticancer treatment, as in the case of serum miR-21 levels that were higher in hormone-refractory prostate cancer patients whose disease was resistant to docetaxel-based chemotherapy than in patients whose disease was chemosensitive [52]. Recently, a high-throughput study generated miRNA signatures from plasma samples collected 12 to 28 months prior to detection of lung cancer and at the time of lung cancer diagnosis [53]. In this study, 21 miRNAs were identified as risk, diagnosis, and prognosis predictors and as being potentially useful in monitoring high-risk disease-free smokers. That study was one of the first to demonstrate that specific pre-disease signatures of miRNA expression in plasma can predict the development of lung cancer prior to diagnosis by conventional techniques and via a noninvasive technique. Furthermore, an independent study proposed that plasma miR-141 may represent a novel biomarker that complements carcinoembryonic antigen in detecting colon cancer with distant metastasis and that high levels of miR-141 in plasma were associated with poor prognosis [54].

### Therapeutic potential of miRNAs as drugs or drug targets

Inhibiting RNA by using miRNAs could soon represent a valid option for the treatment of specific patients [55,56]. These patients should have concordant expression between a specific miRNA and the experimentally proven targets. There would be two advantages to using miRNAs: 1) miRNAs are a “natural” product produced in human cells (unlike chemotherapeutic agents or antisense oligonucleotides), and 2) miRNAs target multiple genes from the same pathway and therefore the action occurs at multiple levels in the same pathway (for example, miR-16 targets both antiapoptotic genes Bcl-2 and Mcl-1 [27]). Two strategies for inhibiting RNA expression could be implemented to treat CLL. First, the “sandwich RNA-inhibition strategy” uses multiple agents to focus on a major molecular alteration that is clearly linked to CLL pathogenesis. Given recently published studies showing the relative efficacy of oblimersen sodium in treating relapsed or refractory CLL [57], treatment regimens that combine anti-Bcl-2 oligonucleotides and miRNAs targeting Bcl-2, such as miR-15 and miR-16, would be feasible for treating indolent CLL. Second, the “multiplex RNA-inhibition strategy” targets various molecular defects in the same pathway, such as apoptosis. With this strategy, multiple synthetic miRNAs target the overexpressed apoptosis regulators Bcl-2 (miR-15 and miR-16) and Mcl-1 (the miR-29 family) and may have a better chance of consistently and robustly reducing expression of these proteins than does a single-agent therapy. The potential use of miRNAs and/or their antisense inhibitors in cancer treatment has only recently been envisioned, and clinical trials of their use in this manner certainly will be scheduled soon.

## Other ncRNAs Important in Cancers

Despite the leading role of miRNAs as cancer-related ncRNAs in published researches, new categories of untranslated RNAs have recently emerged. ncRNAs are conventionally divided into categories of long and short RNAs. Long ncRNAs include those greater than 200 nt in length; they can reach up to 100 kb [58,59]. Elucidating the functions and characteristics of long ncRNAs is still under way; however, it is already clear that this heterogeneous class displays important regulatory functions, as shown in developmental processes wherein ncRNAs can regulate expression of homeotic genes, oncogenes, and metabolic genes [60]. Despite their smaller size, short ncRNAs are equally important in development, cell biology, and diseases. The discovery of these ncRNAs triggered a general interest in ncRNAs in the scientific community. Other ncRNAs such as large intervening ncRNAs (lincRNAs) and ultraconserved genes (UCGs) were found to be abnormally expressed in cancers and to be involved in tumorigenic mechanisms [61,62]. As the spectrum of ncRNAs is much larger than that of miRNAs (with estimates as high as 1000 000 ncRNA transcripts versus only 10 000 potential miRNAs), the impact on any aspect of basic and translational cancer research will be huge. It was recently discovered that lincRNAs, specifically one called HOTAIR, are involved in cancer metastasis [61]. These lincRNAs have many more nucleotides in their sequences than do short ncRNAs. In fact, long ncRNAs are usually composed of several hundreds nucleotides, as opposed to the 20 or so nucleotides that compose short ncRNAs. The involvement of long non-coding UCGs in cancers is suggested by their frequent location in CAGRs and their aberrant expression in several human cancers. Deregulated UCGs are cancer-specific and have prognostic implications. Similar to miRNAs, UCGs can act as oncogenes or TSGs, and UCG

expression is under the control of miRNAs. Another group of ncRNAs, the short germline-specific P-element induced wimpy testis in Drosophila (PIWI)-associated RNAs, which are involved in the regulation of transposable elements and mRNAs, might have implications for human carcinogenesis, although this connection remains elusive.

### Conclusions

There is no longer any doubt that miRNAs are involved in the regulation of pathways involved in cancer initiation, development, progression, and dissemination. The question of whether miRNAs represent the “dark side” of cancer predisposition is only beginning to be answered by studies in large populations of cancer patients. Despite a few intuitive theories in the 1960s that proposed a regulatory role for RNA in controlling gene expression through base-pair complementarity [63,64], the subsequent discovery of transcription factors quenched further research in this field. As a consequence, for many years RNA has been regarded as the exclusive intermediary molecule between DNA and protein, with the primary role in carrying the genetic information necessary for protein synthesis. Intriguingly, only about 2% of human DNA accounts for protein-coding genes, and the total number of proteins does not vary significantly between different species. On the other hand, the extent of non-protein-coding DNA, regarded for a long time as junk DNA, increases proportionally with developmental complexity [65], and over 90% [66] of the genome is actually transcribed in a developmentally regulated manner to produce ncRNAs that can be intergenic, intronic, or overlapping with

protein-coding transcripts [67,68]. miRNAs have been identified as significant new diagnostic and prognostic tools for cancer patients, and miRNA-based cancer therapy is a future option. In addition, ncRNAs display precise tissue-expression patterns [69] and are differentially expressed in pathologic conditions such as cancer and immune and heart diseases [60,62]. A progressive understanding of the implications of ncRNAs for the malignant phenotype represents the essential background needed to achieve the goal of better treatment options for cancer patients.

### Acknowledgments

Dr. Calin is supported as a fellow by The University of Texas MD Anderson Research Trust, as a research scholar by The University of Texas System Regents, and by the Chronic Lymphocytic Leukemia Global Research Foundation. Work in Dr. Calin's laboratory is supported in part by the National Institutes of Health, a Department of Defense Breast Cancer Idea Award, Developmental Research Awards from the Breast Cancer, Ovarian Cancer, Brain Cancer, Multiple Myeloma, and Leukemia Specialized Programs of Research Excellence (SPORE) grants from the National Institutes of Health, a CTT/3I-TD grant, a 2009 Seena Magowitz-Pancreatic Cancer Action Network AACR Pilot Grant, and the Arnold Foundation. We thank Sue Moreau from the Department of Scientific Publications at The University of Texas MD Anderson Cancer Center for her help with the editing of this manuscript.

Received: 2011-06-08; accepted: 2011-07-01.

### References

[1] Ambros V. The evolution of our thinking about microRNAs [J]. Nat Med, 2008, 14(10): 1036–1040.

[2] Filipowicz W, Bhattacharyya SN, Sonenberg N. Mechanisms of post-transcriptional regulation by microRNAs: are the answers in sight? [J]. Nat Rev Genet, 2008, 9(2): 102–114.

[3] Vasudevan S, Tong Y, Steitz JA. Switching from repression to activation: microRNAs can up-regulate translation [J]. Science, 2007, 318(5858): 1931–1934.

[4] Tay Y, Zhang J, Thomson AM, et al. MicroRNAs to Nanog, Oct4 and Sox2 coding regions modulate embryonic stem cell differentiation [J]. Nature, 2008, 455(7216): 1124–1128.

[5] Hwang HW, Wentzel EA, Mendell JT. A hexanucleotide element directs microRNA nuclear import [J]. Science, 2007, 315(5808): 97–100.

[6] Place RF, Li LC, Pookot D, et al. MicroRNA-373 induces expression of genes with complementary promoter sequences [J]. Proc Natl Acad Sci USA, 2008, 105(5): 1608–1613.

[7] Eiring AM, Harb JG, Neviani P, et al. miR-328 functions as an

[8] Calin GA, Croce CM. MicroRNA-cancer connection: the beginning of a new tale [J]. Cancer Res, 2006, 66(15): 7390–7394.

[9] Calin GA, Croce CM. MicroRNA signatures in human cancers [J]. Nat Rev Cancer, 2006, 6(11): 857–866.

[10] Small EM, Frost RJ, Olson EN. MicroRNAs add a new dimension to cardiovascular disease [J]. Circulation, 2010, 121(8): 1022–1032.

[11] Edwards JK, Pasqualini R, Arap W, et al. MicroRNAs and ultraconserved genes as diagnostic markers and therapeutic targets in cancer and cardiovascular diseases [J]. J Cardiovasc Transl Res, 2010, 3(3): 271–279.

[12] Liu CG, Calin GA, Volinia S, et al. MicroRNA expression profiling using microarrays [J]. Nat Protoc, 2008, 3(4): 563–578.

[13] Spizzo R, Nicoloso MS, Croce CM, et al. Snapshot: microRNAs in cancer [J]. Cell, 2009, 137(3): 586–586.

[14] Zenz T, Mohr J, Elderling E, et al. miR-34a as part of the resistance network in chronic lymphocytic leukemia [J]. Blood, 2009, 113(16): 3801–3808.

[15] Calin GA, Sevignani C, Dumitru CD, et al. Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers. [J]. Proc Natl Acad Sci USA, 2004, 101(9): 2999–3004.

[16] Esteller M. Epigenetics in cancer [J]. N Engl J Med, 2008, 358 (11): 1148–1159.

[17] Melo SA, Esteller M. A precursor microRNA in a cancer cell nucleus: get me out of here! [J]. Cell Cycle, 2011, 10(6): 922–925.

[18] Chang TC, Wentzel EA, Kent OA, et al. Transactivation of mir-34a by p53 broadly influences gene expression and promotes apoptosis [J]. Mol Cell, 2007, 26(5): 745–752.

[19] Chang TC, Yu D, Lee YS, et al. Widespread microRNA repression by myc contributes to tumorigenesis [J]. Nat Genet, 2008, 40(1): 43–50.

[20] Lodygin D, Tarasov V, Epanchintsev A, et al. Inactivation of miR-34a by aberrant CPG methylation in multiple types of cancer [J]. Cell Cycle, 2008, 7(16): 2591–2600.

[21] Wei JS, Song YK, Durinck S, et al. The MYCN oncogene is a direct target of miR-34a [J]. Oncogene, 2008, 27(39): 5204–5213.

[22] He L, He X, Lowe SW, et al. MicroRNAs join the p53 network—another piece in the tumour-suppression puzzle [J]. Nat Rev Cancer, 2007, 7(11): 819–822.

[23] He L, He X, Lim LP, et al. A microRNA component of the p53 tumour suppressor network [J]. Nature, 2007, 447(7148): 1130–1134.

[24] Ma L, Teruya-Feldstein J, Weinberg RA. Tumour invasion and metastasis initiated by microRNA-10b in breast cancer [J]. Nature, 2007, 449(7163): 682–688.

[25] Nicoloso MS, Spizzo R, Shimizu M, et al. MicroRNAs—the micro steering wheel of tumour metastases [J]. Nat Rev Cancer, 2009, 9(4): 293–302.

[26] Calin GA, Cimmino A, Fabbri M, et al. miR-15a and miR-16-1 cluster functions in human leukemia [J]. Proc Natl Acad Sci USA, 2008, 105(13): 5166–5171.

[27] Cimmino A, Calin GA, Fabbri M, et al. miR-15 and miR-16 induce apoptosis by targeting BCL2 [J]. Proc Natl Acad Sci USA, 2005, 102(39): 13944–13949.

[28] He L, Thomson JM, Hemann MT, et al. A microRNA polycistron as a potential human oncogene [J]. Nature, 2005, 435(7043): 828–833.

[29] Mendell JT. miRiad roles for the miR-17-92 cluster in development and disease [J]. Cell, 2008, 133(2): 217–222.

[30] Costinean S, Zanesi N, Pekarsky Y, et al. Pre-B cell proliferation and lymphoblastic leukemia/high-grade lymphoma in E(mu)-miR155 transgenic mice [J]. Proc Natl Acad Sci USA, 2006, 103(18): 7024–7029.

[31] Thai TH, Calado DP, Casola S, et al. Regulation of the germinal center response by microRNA-155 [J]. Science, 2007, 316(5824): 604–608.

[32] Rodriguez A, Vigorito E, Clare S, et al. Requirement of bic/ microRNA-155 for normal immune function [J]. Science, 2007, 316(5824): 608–611.

[33] Tili E, Michaille JJ, Cimino A, et al. Modulation of miR-155 and miR-125b levels following lipopolysaccharide/TNF-alpha stimulation and their possible roles in regulating the response to endotoxin shock [J]. J Immunol, 2007, 179(8): 5082–5089.

[34] Cully M, You H, Levine AJ, et al. Beyond PTEN mutations: The PI3K pathway as an integrator of multiple inputs during tumorigenesis [J]. Nat Rev Cancer, 2006, 6(3): 184–192.

[35] Meng F, Henson R, Wehbe-Janek H, et al. MicroRNA-21 regulates expression of the PTEN tumor suppressor gene in human hepatocellular cancer [J]. Gastroenterology, 2007, 133

[36] Asangani IA, Rasheed SA, Nikolova DA, et al. MicroRNA-21 (miR-21) post-transcriptionally downregulates tumor suppressor Pdcd4 and stimulates invasion, intravasation and metastasis in colorectal cancer [J]. Oncogene, 2008, 27(15): 2128–2136.

[37] Frankel LB, Christoffersen NR, Jacobsen A, et al. Programmed cell death 4 (PDCD4) is an important functional target of the microRNA miR-21 in breast cancer cells [J]. J Biol Chem, 2008, 283(2): 1026–1033.

[38] Pineau P, Volinia S, McJunkin K, et al. miR-221 overexpression contributes to liver tumorigenesis [J]. Proc Natl Acad Sci USA, 2010, 107(1): 264–269.

[39] Felli N, Fontana L, Pelosi E, et al. MicroRNAs 221 and 222 inhibit normal erythropoiesis and erythroleukemic cell growth via kit receptor down-modulation [J]. Proc Natl Acad Sci USA, 2005, 102(50): 18081–18086.

[40] Calin GA, Ferracin M, Cimmino A, et al. A microRNA signature associated with prognosis and progression in chronic lymphocytic leukemia [J]. N Engl J Med, 2005, 353(17): 1793–1801.

[41] Raveche ES, Salerno E, Scaglione BJ, et al. Abnormal microRNA-16 locus with synteny to human 13q14 linked to CLL in NZB mice [J]. Blood, 2007, 109(12): 5079–5086.

[42] Shen J, Ambrosone CB, Zhao H. Novel genetic variants in microRNA genes and familial breast cancer [J]. Int J Cancer, 2009, 124(5): 1178–1182.

[43] Pelletier C, Weidhaas JB. MicroRNA binding site polymorphisms as biomarkers of cancer risk [J]. Expert Rev Mol Diagn, 10(6): 817–829.

[44] Song FJ, Chen KX. Single-nucleotide polymorphisms among microRNA: big effects on cancer [J]. Chin J Cancer, 30(6): 381–391.

[45] Fabbri M, Valeri N, Calin GA. MicroRNAs and genomic variations: from proteus tricks to prometheus gift [J]. Carcinogenesis, 2009, 30(6): 912–917.

[46] Song F, Zheng H, Liu B, et al. An miR-502-binding site single-nucleotide polymorphism in the 3′-untranslated region of the SET8 gene is associated with early age of breast cancer onset [J]. Clin Cancer Res, 2009, 15(19): 6292–6300.

[47] Zheng H, Song F, Zhang L, et al. Genetic variants at the miR-124 binding site on the cytoskeleton-organizing IQGAP1 gene confer differential predisposition to breast cancer [J]. Int J Oncol, 2011, 38(4): 1153–1161.

[48] Garzon R, Garofalo M, Martelli MP, et al. Distinctive microRNA signature of acute myeloid leukemia bearing cytoplasmic mutated nucleophosmin [J]. Proc Natl Acad Sci USA, 2008, 105(10): 3945–3950.

[49] Schetter AJ, Leung SY, Sohn JJ, et al. MicroRNA expression profiles associated with prognosis and therapeutic outcome in colon adenocarcinoma [J]. JAMA, 2008, 299(4): 425–436.

[50] Lu J, Getz G, Miska EA, et al. MicroRNA expression profiles classify human cancers [J]. Nature, 2005, 435(7043): 834–838.

[51] Mitchell PS, Parkin RK, Kroh EM, et al. Circulating microRNAs as stable blood-based markers for cancer detection [J]. Proc Natl Acad Sci USA, 2008, 105(30): 10513–10518.

[52] Zhang HL, Yang LF, Zhu Y, et al. Serum miRNA-21: elevated levels in patients with metastatic hormone-refractory prostate cancer and potential predictive factor for the efficacy of docetaxel-based chemotherapy [J]. Prostate, 2011, 71(3): 326–331.

[53] Boeri M, Verri C, Conte D, et al. MicroRNA signatures in tissues and plasma predict development and prognosis of computed tomography detected lung cancer [J]. Proc Natl Acad Sci USA, 2011, 108(9): 3713–3718.

[54] Cheng H, Zhang L, Cogdell DE, et al. Circulating plasma miR-141 is a novel biomarker for metastatic colon cancer and predicts poor prognosis [J]. PLoS ONE, 2011, 6(3): e17745.

[55] Spizzo R, Rushworth D, Guerrero M, et al. RNA inhibition, microRNAs, and new therapeutic agents for cancer treatment [J]. Clin Lymphoma Myeloma, 2009, 9: S313–S318.

[56] Rupaimoole R, Han HD, Lopez-Berestein G, et al. MicroRNA therapeutics: principles, expectations, and challenges [J]. Chin J Cancer, 2011, 30(6): 368–370.

[57] Cheson BD. Oligimeresen for the treatment of patients with chronic lymphocytic leukemia [J]. Ther Clin Risk Manag, 2007, 3(5): 855–870.

[58] Ravasi T, Suzuki H, Pang KC, et al. Experimental validation of the regulated expression of large numbers of non-coding RNAs from the mouse genome [J]. Genome Res, 2006, 16(1): 11–19.

[59] Furuno M, Pang KC, Ninomiya N, et al. Clusters of internally primed transcripts reveal novel long noncoding RNAs [J]. PLoS Genet, 2006, 2(4): e37.

[60] Mattick JS. The genetic signatures of noncoding RNAs [J]. PLoS Genet, 2009, 5(4): e1000459.

[61] Gupta RA, Shah N, Wang KC, et al. Long non-coding RNA HOTAIR reprograms chromatin state to promote cancer metastasis [J]. Nature, 2010, 464(7291): 1071–1076.

[62] Calin GA, Liu CG, Ferracin M, et al. Ultraconserved regions encoding ncRNAs are altered in human leukemias and

[63] Jacob F, Monod J. Genetic regulatory mechanisms in the synthesis of proteins [J]. J Mol Biol, 1961, 3: 318–356.

[64] Britten RJ, Davidson EH. Gene regulation for higher cells: a theory [J]. Science, 1969, 165(891): 349–357.

[65] Taft RJ, Pheasant M, Mattick JS. The relationship between non-protein-coding DNA and eukaryotic complexity [J]. Bioessays, 2007, 29(3): 288–299.

[66] Birney E, Stamatoyannopoulos JA, Dutta A, et al. Identification and analysis of functional elements in 1% of the human genome by the encode pilot project [J]. Nature, 2007, 447 (7146): 799–816.

[67] Kapranov P, Willingham AT, Gingeras TR. Genome-wide transcription and the implications for genomic organization [J]. Nat Rev Genet, 2007, 8(6): 413–423.

[68] Carninci P, Kasukawa T, Katayama S, et al. The transcriptional landscape of the mammalian genome [J]. Science, 2005, 309 (5740): 1559–1563.

[69] Mercer TR, Dinger ME, Sunkin SM, et al. Specific expression of long noncoding RNAs in the mouse brain [J]. Proc Natl Acad Sci USA, 2008, 105(2): 716–721.


---

### Submit your next manuscript to Chinese Journal of Cancer and take full advantage of:

- Open access
- No charge to authors
- Quickly published
- Thorough peer review
- Professionally edited
- No space constraints
- Indexed by PubMed, CA, and Google Scholar

Submit your manuscript at  
www.cjcsysu.com
