
# Peptidoglycan recognition proteins: modulators of the microbiome and inflammation

**Julien Royet**, **Dipika Gupta** and **Roman Dziarski**

Abstract | All animals, including humans, live in symbiotic association with microorganisms. The immune system accommodates host colonization by the microbiota, maintains microbiota–host homeostasis and defends against pathogens. This Review analyses how one family of antibacterial pattern recognition molecules — the peptidoglycan recognition proteins — has evolved a fascinating variety of mechanisms to control host interactions with mutualistic, commensal and parasitic microorganisms to benefit both invertebrate and vertebrate hosts.

---

### Eukaryotes
Organisms (such as animals, plants and fungi) whose cells contain genetic material contained within a membrane-enclosed nucleus.

### Symbiotic
Symbiosis is a physical association between two organisms and includes mutualism, commensalism and parasitism. In common (non-scientific) usage it is often assumed to be beneficial to both organisms.

All multicellular eukaryotes live in symbiotic (mutualistic or commensal) association with microorganisms, which often form complex communities on and/or in the body of the host. These microorganisms make up the host microbiome. All eukaryotes evolved in the presence of prokaryotes, and thus eukaryotes and prokaryotes have often co-evolved to use each other to their advantage. Eukaryotes acquired prokaryotes to form two of their organelles: mitochondria and chloroplasts. Multicellular eukaryotes have developed long-lasting and usually mutually beneficial relationships with specific microorganisms, which carry out essential functions needed by the multicellular eukaryotes in exchange for a stable environment with a constant supply of nutrients^{1-3}. The most complex microbial communities have developed in the guts of mammals, in particular omnivores and herbivores^1. For example, the human gut microbiome has ten times more microorganisms than the number of cells in the human body, 100 times more genes than the human genome, and a metabolic capacity equivalent to the human liver^4. The gut microbiota provides the host with essential nutrients (such as vitamins), breaks down indigestible compounds (such as complex carbohydrates), defends against colonization by pathogens, and is required for the proper development of not only the immune system^{1,2,4}, but also the brain^5 and gut^4.

Genetic and environmental factors can result in changes in mammalian microbiomes that might predispose the host to various diseases. For example, obesity, inflammatory bowel disease and colon cancer are associated with (and probably, in part, caused by) changes in the gut microbiota^{9,6,7}. Changes in the gut microbiota affect sensitivity not only to intestinal diseases, but also to systemic diseases such as arthritis, asthma and diabetes^{8,9}; these effects are mediated by microbial products such as short-chain fatty acids^9 or ATP^{10}. Thus, any component of the host that influences the composition of the microbiota can have a profound effect on the development and well-being of the host.

It has recently been recognized that the immune system developed not only to defend the host against pathogens and to react to tissue damage, but also to accommodate host colonization by symbiotic microorganisms and to maintain microbiota–host homeostasis^{11}. As the health of the host relies on the proper formation and maintenance of the microbiota, there are multiple mechanisms that control the complex interactions between the microbiota and the host.

In this Review, we focus on one family of pattern recognition molecules of the innate immune system, peptidoglycan recognition proteins (PGRPs), which are conserved from insects to mammals and have diverse functions in antimicrobial defence^{12}. We discuss how both invertebrate and vertebrate members of the PGRP family have developed an amazing variety of mechanisms to coordinate the host response to mutualistic, commensal and parasitic microorganisms.

## Peptidoglycan recognition proteins

### Genes and proteins
PGRPs were first discovered in the haemolymph of a silkworm (*Bombyx mori*) as proteins that bind to bacterial peptidoglycan and activate the prophenoloxidase cascade, an antimicrobial host defence mechanism in insects^{13}. Cloning of PGRP genes from the fruitfly (*Drosophila melanogaster*)^{14} led to the discovery

of families of PGRP genes in insects and other invertebrates, such as molluscs and echinoderms, as well as in vertebrates, including fish, amphibians, birds and mammals^{12,15}. However, PGRPs have not been found so far in plants or in lower metazoa, such as nematodes (for example, *Caenorhabditis elegans*).

Insects usually have many PGRP genes. For example, *D. melanogaster* has 13 PGRP genes that are transcribed into 19 proteins^{14,16}, and a mosquito (*Anopheles gambiae*) has seven PGRP genes that are transcribed into nine proteins^{17}. Based on the size of the mRNA transcript, these genes are usually classified into short and long forms, known as S and L, respectively (FIG. 1). Mammals have four PGRP genes, *PGLYRP1*, *PGLYRP2*, *PGLYRP3* and *PGLYRP4*, which were initially named *PGRP-S*, *PGRP-L*, *PGRP-Iα* and *PGRP-Iβ* (on the basis of their short (S), long (L) and intermediate (I) transcript lengths) by analogy to insect PGRPs^{18}.

**Structure and specificity.** All PGRPs have at least one PGRP domain, which is ~165 amino acids long and is structurally similar to bacteriophage and bacterial type 2 amidases (which catalyse the hydrolysis of amide bonds)^{12,14–18}; the PGRP domain constitutes most of the sequence of short PGRPs (FIG. 1). Some PGRPs (for example, *D. melanogaster* PGRP-LF and mammalian PGLYRP3 and PGLYRP4) have two PGRP domains, which are homologous but not identical. Long PGRPs usually have one PGRP domain and an additional unique amino-terminal sequence (FIG. 1). Many PGRPs are secreted, and some (such as mammalian PGLYRP1, PGLYRP3 and PGLYRP4) form disulphide-linked homodimers or heterodimers (for example, PGLYRP3–PGLYRP4). Some PGRPs (such as *D. melanogaster* PGRP-LC and PGRP-LF) are transmembrane receptors, and some others are intracellular proteins (such as *D. melanogaster* full-length PGRP-LE).

The general structural design of the PGRP domain is similar to that of bacteriophage type 2 amidases in that it has three peripheral α-helices and several central β-strands^{12,15,19,20}. The front face of the molecule has a cleft that forms a peptidoglycan-binding groove, and the back of the molecule has a PGRP-specific segment that is not present in bacteriophage amidases (FIG. 1a); this region is often hydrophobic and is diverse among various PGRPs, and it might bind additional ligands or other molecules^{19}. In some PGRPs (such as *D. melanogaster* PGRP-LCa and PGRP-LF), the peptidoglycan-binding groove is disrupted, which prevents direct recognition of peptidoglycan fragments by these PGRP family members^{21,22}.

The peptidoglycan-binding groove of PGRPs typically binds the muramyl pentapeptide or tetrapeptide fragment of peptidoglycan with the highest affinity (FIG. 1b), and this binding usually induces a structural change in the PGRP domain or an interaction with another PGRP molecule. Both of these changes lock the muramyl peptide in the binding site. Many PGRPs can also discriminate between different amino acids in the peptide or between *N*-acetylmuramic acid and its anhydro form within peptidoglycan^{12,20}, and this is discussed below in more detail.

Although most PGRPs bind to peptidoglycan^{12,15,20}, some PGRPs also bind with high affinity to other microbial molecules, such as lipopolysaccharide (LPS) and lipoteichoic acid; these molecules usually bind outside the peptidoglycan-binding groove^{23–25}. In addition, certain fungus-derived molecules can be bound by several PGRPs (including bovine PGLYRP1 and human PGLYRP3 and PGLYRP4)^{24,26}.

**Control of the microbiota by insect PGRPs**

To detect and control microorganisms, insects use a relatively small number of pattern recognition molecules. Work carried out mainly using *D. melanogaster* has highlighted the central role of the members of the PGRP family in bacterial recognition and control in insects; indeed, PGRPs are the largest and most versatile family of pattern recognition molecules for microbial products in these organisms. By contrast, vertebrates have fewer PGRPs with a much narrower range of functions because, in addition to expressing PGRPs, they have evolved a vast array of other pattern recognition receptors (PRRs). These include Toll-like receptors (TLRs), NOD-like receptors (NLRs), C-type lectin receptors (CLRs), RIG-I-like receptors (RLRs), scavenger receptors and the mannose receptor. Moreover, they possess many secreted recognition molecules, such as complement, collectins, pentraxins and C-reactive protein.

**PGRPs are upstream PRRs of *D. melanogaster* immune cascades.** The functions of *D. melanogaster* PGRPs as PRRs were initially discovered by investigating the role of these proteins as sentinel receptors upstream of the pathways that regulate the systemic immune response^{27–30}. In flies, the Toll pathway is mainly activated by Gram-positive bacteria and induces the production of antimicrobial peptides (AMPs) by cells of the fat body; AMPs are then released into the circulating haemolymph, where they kill infectious bacteria^{2} (FIG. 2). Two PGRP family members (PGRP-SA and PGRP-SD) and the misnamed Gram-negative binding protein 1 (GNBP1) form a circulating PRR complex upstream of the Toll pathway^{27,31–33}. By binding to peptidoglycan from the cell walls of Gram-positive bacteria, PGRP-SA triggers a protease cascade that ultimately transforms the inactive form of Spätzle into an active ligand, which then induces Toll dimerization and downstream signalling to nuclear factor-κB (NF-κB). *D. melanogaster* mutants that have genetic defects in Toll pathway components or in the upstream PRRs are, as a result, highly susceptible to infection with Gram-positive bacteria^{27,34}.

The *D. melanogaster* immune system can discriminate between various types of bacteria based on their peptidoglycan composition^{12,35,36}. Whereas bacteria that have a lysine residue in the third position of the peptidoglycan stem peptide (Lys-type peptidoglycan, which is present in most Gram-positive bacteria) elicit the Toll pathway, bacteria with *meso*-diaminopimelic acid in this position (DAP-type peptidoglycan, which is present in most Gram-negative bacteria and in bacilli) activate another signalling cascade upstream of NF-κB known as the IMD pathway.

a

Drosophila melanogaster  
PGRP-LB  

Bacteriophage T7 lysozyme  
(type 2 amidase)  

b

Drosophila melanogaster  
PGRP-LCx–TCT complex  
(PGRP domain only)  

MurNAc(anhydro)  

GlcNAc  
L-Ala  

D-Glu  
meso-DAP  

D-Ala  
TCT (PGN anhydromonomer)  

c

| Drosophila melanogaster PGRPs |  |  |
| --- | --- | --- |
| N | C | PGRP-SB1, PGRP-SB2, PGRP-SC1, PGRP-SC2 | Secreted amidases |
| N | C | PGRP-SA | Secreted PRR for PGN; activates Toll |
| N | C | PGRP-SD | Secreted PRR for PGN; activates Toll |
| N | C | PGRP-LA | Different isoforms exist; unknown function |
| N | C | PGRP-LB | Secreted amidase; different isoforms exist |
| N | C | PGRP-LCx | Transmembrane PRR for PGN; activates IMD |
| N | C | PGRP-LCa | Transmembrane protein; PGRP-LCx co-receptor |
| N | C | PGRP-LCy | Transmembrane protein; unknown function |
| N | C | PGRP-LD | Putative mitochondrial protein; unknown function |
| N | C | PGRP-LEfl | Intracellular PRR for PGN |
| N | C | PGRP-LEpg | Extracellular PGN co-receptor |
| N | C | PGRP-LF | Non-signalling transmembrane receptor |

| Euprymna scolopes (Hawaiian bobtail squid) PGRPs |  |  |
| --- | --- | --- |
| N | C | PGRP1 | Intracellular protein; predicted amidase |
| N | C | PGRP2 | Secreted amidase |
| N | C | PGRP3 | Predicted GPI-linked membrane receptor and amidase |
| N | C | PGRP4 | Predicted transmembrane receptor |

| Danio rerio (zebrafish) PGRPs |  |  |
| --- | --- | --- |
| N | C | Pglyrp2 | Bactericidal amidase |
| N | C | Pglyrp5 | Bactericidal amidase |
| N | C | Pglyrp6 | Bactericidal amidase |

| Mammalian PGRPs |  |  |
| --- | --- | --- |
| N | C | PGLYRP1 | Secreted bactericidal protein |
| N | C | PGLYRP2 | Secreted bactericidal amidase |
| N | C | PGLYRP3 | Secreted bactericidal protein |
| N | C | PGLYRP4 | Secreted bactericidal protein |

|  |  |  |  |
| --- | --- | --- | --- |
| PGRP domain with amidase activity | PGRP domain whose PGN binding is unknown | Hydrophobic domain | Specific for DAP-type PGN |
| PGRP domain without amidase activity | RHIM domain | IMD interacting domain | Specific for Lys-type PGN |
| PGRP domain without PGN binding activity | Mitochondrial targeting domain |  |  |

Figure 1 | The structure of PGRPs. **a** | The ribbon diagrams show the conservation of structure between a *Drosophila melanogaster* amidase (peptidoglycan recognition protein-LB (PGRP-LB)) and an ancestral type 2 amidase (bacteriophage T7 lysozyme). A Zn²⁺ ion (green sphere), which is required for amidase activity, is shown in the peptidoglycan (PGN)-binding groove. The PGRP-unique segment is shown in yellow. Images are reproduced, with permission, from REF. 19 © (2003) Macmillan Publishers Ltd. All rights reserved. **b** | The PGRP domain of *D*. *melanogaster* PGRP-LCx is shown in complex with a PGN fragment, tracheal cytotoxin (TCT), the structure of which is shown on the right. Images are reproduced, with permission, from REF. 40 © (2006) The American Association for the Advancement of Science. **c** | The figure shows diagrams of the domain structure and the main functions of *D*. *melanogaster*, *Euprymna scolopes*, *Danio rerio* and mammalian PGRPs. GPI, glycosylphosphatidylinositol; PRR, pattern recognition receptor; RHIM, RIP homotypic interaction motif.

REVIEW S

Toll pathway  
A signalling cascade in insects that regulates the expression of antimicrobial peptides after infection by fungi or Gram-positive bacteria. Detection of fungi by GNBP3 and of bacteria by circulating peptidoglycan recognition proteins (PGRPs) triggers the cleavage of the cytokine Spätzle, which then activates the Toll receptor, and this initiates the activation of the NF-κB-like transcription factors Dif and Dorsal and their translocation into the nucleus. This pathway also controls dorsoventral patterning during embryogenesis.

Antimicrobial peptides (AMPs). Polypeptides that are typically less than 100 amino acids and that have an important role in innate immune responses by providing broad-spectrum antimicrobial activity against pathogenic microorganisms. There are different types of AMP that promote microbial killing by various mechanisms, but many function by disrupting the membranes of microorganisms.

IMD pathway  
A signalling cascade in insects (named after 'immunodeficiency' mutant flies) that corresponds closely to the mammalian tumour necrosis factor receptor signalling pathway. This cascade is mainly activated by Gram-negative bacteria, which are recognized by the upstream receptors PGRP-LC and PGRP-LE. Intracellular signalling involves the death domain-containing protein IMD, IkB kinase, FADD, Dredd, TAK1 and the NF-κB-like protein Relish. In contrast to the Toll pathway, the IMD pathway has no known role in development.

The receptor proteins of the IMD pathway are unrelated to Toll and all belong to the PGRP family. The main transmembrane receptor is PGRP-LC, which functions both as a microbial binding protein (through its extracellular PGRP domain) and as a signalling receptor (through its intracellular tail that interacts directly with IMD, an orthologue of mammalian RIP1)²⁸⁻³⁰,³⁷ (FIGS 2,3). In this way, PGRP-LC is similar to mammalian TLRs, but unlike the *D. melanogaster* Toll protein. By alternative splicing, the gene encoding PGRP-LC generates three protein isoforms (x, y and a) that have identical intracellular domains but different extracellular PGRP domains with various specificities. Whereas PGRP-LCa and PGRP-LCx pair to form a receptor for monomeric peptidoglycan fragments, such as tracheal cytotoxin (TCT; also known as anhydromonomer)²¹,²²,³⁸⁻⁴⁰ (FIG. 1b), PGRP-LCx homodimerizes to recognize polymeric peptidoglycan. By forming inactive heterodimers with PGRP-LCx, another member of the family — PGRP-LF (a transmembrane receptor that lacks the intracellular signalling domain) — prevents the formation of PGRP-LCa–PGRP-LCx heterodimers and, hence, functions as a negative regulator of the IMD pathway by decreasing downstream signalling⁴¹,⁴² (FIG. 2). DAP-type peptidoglycan is also

![Diagram](attachment:diagram.png)

Figure 2 | Insect PGRPs recognize peptidoglycan and regulate multiple defence pathways. Depending on their peptidoglycan (PGN) composition, bacteria can activate either the Toll pathway or the IMD pathway in *Drosophila melanogaster* cells. Extracellular bacteria that contain Lys-type PGN are recognized by peptidoglycan recognition protein-SA (PGRP-SA), PGRP-SD and Gram-negative binding protein 1 (GNBP1). This induces a protease cascade that ultimately cleaves pro-Spätzle into an active ligand for Toll, leading to nuclear factor-κB (NF-κB)-dependent signalling and antimicrobial peptide (AMP) production. Bacteria that contain DAP-type PGN elicit the IMD pathway in flies. Polymeric PGN is recognized by both PGRP-LCx and an extracellular version of PGRP-LE that comprises only the PGRP domain (PGRP-LEpg). Monomeric PGN (such as tracheal cytotoxin (TCT)) is recognized extracellularly by a PGRP-LCa–PGRP-LCx heterodimer and intracellularly by a full-length cytoplasmic form of PGRP-LE (PGRP-LEfl). Both complexes activate AMP production through NF-κB signalling. It is unclear how TCT enters the cells. Extracellular IMD pathway activation is inhibited by PGRPs with amidase activity (namely, PGRP-LB and PGRP-SC), which cleave PGN into inactive amino sugars and peptides, and by PGRP-LF, which competes with PGRP-LCa for binding to PGRP-LCx and forms non-signalling PGRP-LF–PGRP-LCx heterodimers. Some extracellular PGRPs also act as scavengers through enzymatic degradation of PGN (in the case of PGRP-SB) or as opsonins for phagocytosis (in the case of PGRP-SC). Finally, PGRP-LEfl induces the elimination of intracellular bacteria by promoting autophagy, a process that is independent of IMD.

Autophagy  
Autophagocytosis that involves the degradation of the cell’s own components through the lysosomal machinery. This process has a role in normal cell growth and development, and also has a protective role against infection with intracellular pathogens.

recognized in flies by PGRP-LE, which exists in two forms. A full-length isoform (PGRP-LEfl) detects intracellular peptidoglycan, and a cleaved form (PGRP-LEpg) enhances PGRP-LCx-mediated cell-surface recognition of polymeric peptidoglycan⁴³,⁴⁴ (FIG. 2).

In contrast to systemic immune responses, which are controlled by both the Toll and IMD pathways, AMP production locally in the respiratory and intestinal epithelia depends on the IMD pathway only. This indicates that Gram-negative bacteria are probably the main inducers of these epithelial immune responses in *D. melanogaster*.

---

**Figure 3 | Drosophila melanogaster PGRPs protect the gut from infections and maintain gut microbiome homeostasis.** DAP-type peptidoglycan (PGN) from intestinal bacteria is sensed by peptidoglycan recognition protein-LC (PGRP-LC), which triggers the IMD and p38 mitogen-activated protein kinase pathways. The activation of p38 enhances the transactivating function of ATF2, which in turn activates the transcription of dual oxidase (*Duox*). This leads to the production of reactive oxygen species (ROS) in the intestinal lumen, where they control endogenous and infectious bacteria. PGN recognition by PGRP-LC also triggers (through the IMD pathway) the cleavage and nuclear translocation of the nuclear factor-κB family member Relish, which then induces increased transcription of antimicrobial peptide (AMP) genes. However, in the gut epithelium, the IMD pathway is under tight negative control. Two of the genes that are most responsive to IMD activation encode PGRP-LB, which cleaves PGN and therefore blocks activation of the IMD pathway, and Pirk, which interferes with the plasma membrane localization of PGRP-LC. Finally, in some gut cells, the endogenous microbiota causes nuclear translocation of an inhibitor (Caudal), which prevents constant AMP production and the destruction of the normal microbiota. IKK, IκB kinase.

---

Insect PGRPs also participate in other antibacterial mechanisms. In *D. melanogaster*, full-length PGRP-LE is required to eliminate intracellular bacteria (such as *Listeria monocytogenes*) by promoting autophagy⁴³ (FIG. 2), which has a protective role against intracellular pathogens (in addition to its catabolic role in the degradation of the cell’s own components). Some extracellular PGRPs also function as bactericidal proteins (for example, PGRP-SB) or as opsonins (for example, PGRP-SC)¹²,¹⁵,⁴⁵. In various insects, several extracellular PGRPs trigger the prophenoloxidase cascade, which results in the formation of melanin and of other antimicrobial compounds that have been less well defined¹²,¹³,¹⁵,⁴⁶.

PGRPs regulate the gut microbiota in *D. melanogaster*. AMPs are constantly produced at basal levels in the uninfected gut of *D. melanogaster*, but their transcription is highly upregulated following feeding on food that is contaminated with microorganisms (such as rotten fruits). Consistently, various laboratories have shown genetically that IMD-dependent AMP production in the gut is essential for flies to resist infection by numerous ingested pathogens, such as *Serratia marcescens* or *Pseudomonas entomophila*⁴⁷,⁴⁸. Although it has not yet been unambiguously demonstrated, it is assumed that PGRP-LC mediates this IMD-dependent gut response (FIG. 3).

In addition to regulating the induction of AMP production following infection, the IMD pathway has an important role in shaping the gut response to the endogenous microbiota. The commensal bacterial community in the intestinal tract of *D. melanogaster* is much simpler than in humans and consists of ~3 × 10⁵ microbial cells of 10 to 20 different species (mainly of the *Acetobacter* and *Lactobacillus* genera)⁴⁹⁻⁵². Although the *D. melanogaster* microbiota is a new area of research, it is already clear that, as shown for vertebrates, these microorganisms influence the physiology of the host fly. The gut microbiota promotes larval growth by modulating hormonal signals⁵³ and even triggers sexual selection preference in *D. melanogaster* populations by altering the production of sex pheromones⁵⁴. This implies that a tight qualitative and quantitative control of the gut microbiota is required for the optimal adaptation of *D. melanogaster* to its environment.

The constant contact between bacteria and the gut mucosal epithelium raises the question of how the epithelium manages this continuous input of immune-activating signals derived from commensal microorganisms. Recent evidence points to the important role of PGRP family members in the tolerance of gut epithelia towards the endogenous microbiota. Of the 19 *D. melanogaster* PGRP proteins, seven have amidase activity, which allows them to cleave pro-inflammatory peptidoglycan into non-stimulatory muropeptides⁵⁵,⁵⁶. These seven proteins are PGRP-LB-A, PGRP-LB-B, PGRP-LB-C, PGRP-SB1, PGRP-SB2, PGRP-SC1 and PGRP-SC2. Through their amidase activity, PGRP-SC and PGRP-LB synergize to sustain a low basal level of peptidoglycan in the gut, a mechanism that prevents spontaneous overactivation of the IMD pathway by the

---

**Midgut epithelial cell**

**NATURE REVIEWS | IMMUNOLOGY**  
VOLUME 11 | DECEMBER 2011 | 841  

© 2011 Macmillan Publishers Limited. All rights reserved.

gut microbiota (B. Lemaitre, personal communication)  
(FIG. 3). These enzymes also prevent excessive transfer  
across the epithelial barrier (and into the haemolymph)  
of immunostimulatory peptidoglycan fragments that  
are released from gut bacteria. This function is revealed  
by the overactivation of the systemic inflammatory  
response in the fat body cells of flies that have inactivated  
catalytic PGRPs in the gut⁵⁷.

Another mechanism that flies use to render their  
endogenous microbiota invisible to their immune sys-  
tem is the modulation of PGRP-LC localization in the  
epithelial cell membrane. This function is mediated  
by the intracellular protein Pirk (poor IMD response  
upon knock-in; also known as Rudra or Pims), which,  
by sequestering PGRP-LC in the cytoplasm, prevents  
it from accessing extracellular peptidoglycan and thus  
decreases IMD signalling⁵⁸⁻⁶⁰ (FIG. 3). Accordingly, loss-  
of-function mutations in the genes encoding PGRP-SC,  
PGRP-LB and Pirk are associated with an increased  
constitutive production of AMPs in the gut, and this  
disappears when flies are grown in axenic conditions  
(free of all microorganisms). Because these genes are  
also transcriptional targets of the IMD pathway, their  
protein products are constitutively produced at basal  
levels in the gut in response to the microbiota. This  
establishes a negative feedback loop that prevents hyper-  
activation of the IMD target genes by peptidoglycan  
derived from commensal bacteria (FIG. 3). It is expected,  
although not yet demonstrated, that the dysregulated  
immune activation observed in flies with mutant forms  
of PGRP-SC, PGRP-LB and Pirk will induce a shift in  
the composition of the gut microbiota. Similar results  
have been obtained by analysing the effects of another  
IMD pathway regulator, Caudal, on the composition of  
the microbiota. This transcription factor directly sup-  
presses IMD-dependent NF-κB-mediated expression  
of AMPs in the gut by blocking AMP gene promoters⁵².  
The absence of such a control, as observed by inhibition  
of Caudal expression, results in changes in the com-  
mensal microbiota⁵². The baseline repression of the  
IMD pathway mediated by these various proteins is also  
essential for maintaining the fitness of the flies. The  
absence of PGRP-LB, PGRP-SC, Pirk or Caudal dis-  
rupts gut homeostasis, provokes epithelial cell renewal  
and decreases the lifespan of *D. melanogaster*⁵²,⁶¹  
(B. Lemaitre, personal communication).

ROS are produced at basal levels by dual oxidase  
(DUOX), which is associated with the gut membrane.  
Following infection, increases in both *Duox* gene activa-  
tion and DUOX protein expression are required to coun-  
ter the increased bacterial burden in the gut⁶³. DUOX  
enzymatic activation requires an increase in intracellular  
calcium levels and probably depends on signalling  
through an as yet unidentified upstream G protein-  
coupled receptor (and is therefore independent of  
PGRPs). By contrast, *Duox* transcriptional upregula-  
tion is triggered by both peptidoglycan-dependent and  
-independent pathways. In the peptidoglycan-dependent  
pathway, peptidoglycan recognition by PGRP-LC acti-  
vates the transcription of *Duox* through IMD, MEKK1,  
p38 and ATF2. This results in increased levels of DUOX  
in the epithelial cell membrane and thus enhances the  
release of ROS into the gut lumen⁶³ (FIG. 3).

The above findings show that *D. melanogaster* gut  
innate immune responses, which are Toll independent,  
rely on a finely tuned balance between the IMD and the  
DUOX pathways, which are both activated by upstream  
PGRP molecules. Activation or repression of these path-  
ways allows the host to adapt its intestinal antimicrobial  
response to the gut microbial load. Phenotypic analy-  
ses of flies with mutations in genes that are involved in  
these pathways demonstrate that controlling the quantity  
and the quality of the intestinal microbiota is crucial for  
the flies, and that PGRP family members are important  
players in these defence and homeostatic mechanisms.

**Control of symbiotic bacteria and parasites in other  
insects.** Haematophagous insects frequently encoun-  
ter blood-borne infectious agents, including para-  
sites and viruses. For example, female *Anopheles* spp.  
mosquitoes become infected with malaria parasites  
(*Plasmodium* spp.) during feeding on infected humans  
and can then transmit the parasites to new hosts dur-  
ing subsequent bites. Blood-meal ingestion is associated  
with an increase in the size of the microbial commu-  
nity in the gut of the female mosquito, and this triggers  
PGRP-LC-mediated NF-κB signalling, which induces  
AMP production to control the microbial burden⁶⁴  
(FIG. 4a). In mosquitoes in which PGRP-LC levels have  
been downregulated by RNA interference, gut bacterial  
loads are strongly increased compared with those in con-  
trol mosquitoes. Interestingly, this PGRP-LC-mediated  
response not only affects gut bacteria but also seems  
to decrease the infection efficiency of protozoal para-  
sites. Silencing of the gene encoding PGRP-LC results  
in an increased number of *Plasmodium* spp. oocysts in  
infected mosquitoes (FIG. 4a). Because this effect is no  
longer observed in mosquitoes in which the gut micro-  
biota has been eliminated by antibiotic treatment, these  
results indicate that PGRP-LC-mediated antibacte-  
rial defence also has an indirect antiparasitic effect in  
*Anopheles gambiae*⁶⁵. It is probable that, in addition to  
controlling the gut microbiota, some of the bacterium-  
induced defence molecules (such as AMPs and ROS)  
might interfere with the normal development of malaria  
parasites during the midgut stages of their life cycle,  
although this has not yet been clearly shown⁶⁵.

**Reactive oxygen species (ROS).** Aerobic organisms  
derive their energy from the  
reduction of oxygen. The  
metabolism of oxygen, and  
in particular its reduction  
through the mitochondrial  
electron-transfer chain,  
generates by-products such  
as superoxide (*O₂⁻*), hydrogen  
peroxide (*H₂O₂*) and hydroxyl  
radicals (*•OH*). These three  
species and the unstable  
intermediates that are formed  
by lipid peroxidation are  
known as ROS. ROS can  
damage important intracellular  
targets such as DNA,  
carbohydrates and proteins.

**Control of the microbiota through regulation of ROS  
production in *D. melanogaster*.** AMP production by the  
gut epithelium downstream of the IMD pathway is not  
the only effector function of the intestinal antimicrobial  
immune response. Consistently, flies lacking gut AMP  
expression are resistant to infection by many bacterial  
species and, as discussed earlier, IMD pathway activa-  
tion is under strong negative regulation in the gut to  
preserve the commensal microbiota and host health.  
Recent studies have shown that reactive oxygen species  
(ROS) are important microbicidal effectors in the fly  
gut⁶². Although ROS and AMPs are directed towards  
the same microbial targets, their production in the  
*D. melanogaster* gut is regulated by different mechanisms.

Anopheles gambiae

Vertebrate host
Blood feeding

Midgut epithelium
Oocyst
Salivary glands
Sporozoite
Increased bacterial burden
AMP production
Zygote
PGN
PGRPLC
Bacterium
Plasmodium spp.
gametocyte
Gametocyte
Blood meal
Plasmodium spp.
transmission

Midgut
Blood meal
Tsetse fly
PGN
Decreases damage to the host induced by the immune response
PGRP-LB (amidase activity for PGN)
Antiprotozoal activity
Trypanosome
Bacteriocyte
Wigglesworthia glossinidia
PGRP-LB
Nucleus

Figure 4 | In blood-sucking insects, PGRPs maintain the balance between symbiotic bacteria and protozoal parasites.

a | Peptidoglycan recognition protein-LC (PGRP-LC) controls intestinal symbiotic and infectious bacteria in Anopheles gambiae by inducing the production of antimicrobial peptides (AMPs). This inhibits the proliferation of gut bacteria after a mosquito blood meal. PGRP-LC activation also strongly decreases the number of Plasmodium spp. oocysts in the A. gambiae gut after a blood meal. The tripartite interaction between symbiotic bacteria, invading parasites and the mosquito immune system is important for the balance of mosquito susceptibility and resistance to malaria and perpetuates disease transmission.

b | The tsetse fly (Glossina spp.) is the sole African vector for trypanosomes and has co-evolved with the mutualistic endosymbiont Wigglesworthia glossinidia. W. glossinidia induces the expression of PGRP-LB by infected midgut cells (bacteriocytes), and PGRP-LB cleaves peptidoglycan (PGN) and decreases the damage to the host associated with the permanent presence of the endosymbiont. PGRP-LB also maintains low levels of trypanosomes by an unknown mechanism. Elimination of W. glossinidia renders tsetse flies sterile and increases their susceptibility to trypanosome infection.

PGRP family members also regulate interactions between some insects and their microbial endosymbionts. Adult tsetse flies (Glossina spp.) feed exclusively on vertebrate blood, a very unbalanced meal lacking vital components, such as many vitamins. To compensate for this nutritionally restricted diet, tsetse flies carry the obligate endosymbiont Wigglesworthia glossinidia, the presence of which is required for adequate digestion and nutrition in the flies and for their successful reproduction. The Gram-negative W. glossinidia bacteria

reside in differentiated midgut cells, where they form an organ known as a ‘bacteriome’. W. glossinidia elimination results in sterility of tsetse flies and increases their susceptibility to trypanosome infection<sup>66</sup>, demonstrating the essential role of this endosymbiont in the life cycle of tsetse flies (FIG. 4b).

The symbiosis between the tsetse fly and W. glossinidia raises two questions: how does the tsetse fly immune system tolerate the constant presence of its gut symbiont, and what is the mechanism by which W. glossinidia mediates its antitrypanosome effect? Part of the answer was obtained by studying the function of the DAP-type peptidoglycan-cleaving enzyme PGRP-LB in tsetse flies. It has been shown that W. glossinidia triggers the expression of tsetse fly PGRP-LB in its bacteriome<sup>67</sup> (FIG. 4b). It is thought that by scavenging W. glossinidia peptidoglycan, PGRP-LB prevents constitutive IMD pathway activation and, hence, establishes tolerance towards this symbiont. Consistently, decreasing the *in vivo* PGRP-LB level induces an increase in the level of AMPs and decreases the number of W. glossinidia in the tsetse fly gut. Unexpectedly, depletion of PGRP-LB also increases the susceptibility of tsetse flies to trypanosome parasites (FIG. 4b), and this indicates that PGRP-LB might have an antiprotozoal activity that confers a resistance to these parasites<sup>67</sup>. Bacteriomes containing symbiotic bacteria are common in other insects, and the mutualistic interactions between the insects and their endosymbionts are probably also controlled by PGRPs; for example, this has been suggested by the high level of expression of PGRP-LB in the bacteriome of the weevil Sitophilus zeamais<sup>68</sup>.

Thus, in insects that do not belong to the Drosophilidae family, PGRPs and the host innate immune response interact to fine-tune the homeostatic balance between tolerance to indispensable mutualistic microorganisms and resistance to pathogenic microorganisms. It is possible that this tripartite interaction between the gut microbiome of insects, PGRP-mediated antibacterial defences and parasites such as Plasmodium spp. or trypanosomes could be exploited in future interventions aiming to control disease transmission.

### Control of luminescent bacteria by squid PGRPs

PGRPs are also present in other invertebrates. In molluscs — such as oysters<sup>69</sup>, scallops<sup>70,71</sup>, squid<sup>72–76</sup> and snails<sup>77</sup> — PGRPs are expressed in the mantle, gills, haemocytes, digestive diverticula, kidneys and gonads<sup>69–72,77</sup>. Some of the mollusc PGRPs are predicted or proven to be peptidoglycan hydrolases, whereas others have defensin-like domains, indicating that they might have a direct bactericidal function<sup>69–72,77</sup>. These mollusc PGRPs probably have a role in defence against infections, because their expression is increased in response to bacterial pathogens, such as Echinostoma paraensei<sup>70</sup>.

In the Hawaiian bobtail squid, *Euprymna scolopes*, PGRPs are involved in highly specialized host-microorganism interactions that result in the establishment and maintenance of the symbiotic luminescent bacterium *Vibrio fischeri* within the squid light organ<sup>72</sup>.

These squid are nocturnal animals, and light produced by the symbiotic bacteria provides counter-illumination to moonlight and makes the squid less visible to predators.

*E. scolopes* are not associated with any bacteria at birth; however, within a few hours after hatching, the light organ becomes populated with *V. fischeri* in an association that is highly specific for this bacterial species<sup>75,76</sup>. The nascent light organ has a pair of protruding ciliated appendages on the opposing sides of three pores (FIG. 5). In response to bacteria in the surrounding water, epithelial cells of the appendages secrete mucus that covers the pores and appendages<sup>74–76</sup>. The beating of the cilia concentrates bacteria from the surrounding water into the mucus, which starts the ‘winnowing’ of *V. fischeri* from hundreds of other bacterial species present in the water, so that only *V. fischeri* enters through the pores into the ducts of the light organ. These bacteria then swim into the deep crypts, where they lose their flagella and permanently colonize the host<sup>74–76</sup>. Entry of *V. fischeri* into the ducts induces apoptosis and morphogenesis in the light organ; this involves the loss of ciliated epithelium and the shortening and eventual loss of the appendages, which prevents the entry of other bacteria into the light organ<sup>73–76</sup>.

PGRPs have been proposed to participate in the winnowing and maintenance of *V. fischeri* in the light organ<sup>72–76</sup>, but the basis for the selectivity of this winnowing is still poorly understood. *E. scolopes* has four PGRPs, and they are all expressed in the light organ<sup>72</sup>. Two PGRPs have been proposed to function as receptors on the surface of light organ cells; these are PGRP3, which has a predicted glycosylphosphatidylinositol (GPI) membrane anchor and amidase activity, and PGRP4, which is predicted to have two transmembrane domains, an extracellular PGRP domain and no amidase activity<sup>72</sup> (FIG. 5D). Light organ cells also express a TLR and an LPS-binding protein (LBP)<sup>72</sup>. It is proposed that bacterial products, such as peptidoglycan and LPS, activate the transcription factors NF-κB, ETS and Krüppel through PGRP3, PGRP4 and TLR. These transcription factors then induce the synthesis of antibacterial factors — such as mucin, nitric oxide synthase (NOS) and LBP — and also of PGRP1 and PGRP2 by the ciliated epithelial cells<sup>72</sup> (FIG. 5D).

PGRP2 is secreted with the mucus and is an amidase that hydrolyses pro-inflammatory peptidoglycan fragments<sup>74</sup>. PGRP2 is also secreted into the lumen of the crypts<sup>74</sup>, where it helps to maintain the homeostatic balance between *E. scolopes* and the permanently present *V. fischeri*. PGRP2 accomplishes this by hydrolysing peptidoglycan fragments that are constantly released from *V. fischeri* and thus preventing the inflammatory effects of these peptidoglycan fragments, which allows beneficial coexistence of the symbiont with the host<sup>74</sup>. PGRP2 might also have a low level of antibacterial activity that helps in the winnowing function of the appendage-derived mucus and in maintaining the proper numbers of *V. fischeri* in the crypts, although this antibacterial activity has not yet been experimentally proven.

Figure 5 | *Euprymna scolopes* PGRPs control colonization and maintenance of the luminescent bacterium *Vibrio fischeri* within the light organ of the squid.

**A** | The location of the light organ in the body cavity of *E. scolopes* is shown.

**B** | The figure shows the colonization of the crypts in the light organ by *V. fischeri*. Seawater contains many different bacteria, which stimulate epithelial cells of the light organ to secrete mucus containing peptidoglycan recognition protein 2 (PGRP2) and probably other antibacterial factors that select *V. fischeri* (**a**). As a result, only Gram-negative bacteria aggregate between the appendages of the light organ (**b**), and *V. fischeri* becomes dominant (**c**). *V. fischeri* then enter the ducts (**d**) and permanently colonize the crypts (**e**).

**C** | Successful establishment of *V. fischeri* in the crypts results in the loss of ciliated appendages.

**D** | Light organ cells express Toll-like receptor (TLR) and four PGRPs with different cellular locations. PGRP1 is intracellular, PGRP2 is secreted, PGRP3 has a predicted glycosylphosphatidylinositol membrane anchor and PGRP4 has two predicted transmembrane domains and an extracellular PGRP domain. It is proposed that PGRP3 and PGRP4 function as cell surface receptors for bacterial products and, together with TLR, activate the transcription factors nuclear factor-κB (NF-κB), ETS and Krüppel, which induce the synthesis of mucin, nitric oxide synthase (NOS), lipopolysaccharide binding protein (LBP), PGRP1 and PGRP2 in ciliated epithelial cells, and PGRP2 in crypt epithelial cells. Loss of PGRP1 expression in ciliated epithelial cells after colonization of the light organ by *V. fischeri* leads to apoptosis and the loss of appendages. PGRP2 is secreted by epithelial cells, first in the appendages and then in the crypts, where it degrades peptidoglycan fragments that are released by *V. fischeri*. This maintains homeostasis between the host and the symbiont.

Figure parts A–C are modified, with permission, from REF. 76 © (2004) Macmillan Publishers Ltd. All rights reserved.

PGRP1 is a predicted amidase, but it does not have a signal peptide and is located in the nuclei and the cytoplasm of ciliated epithelial cells in the appendages. The loss of PGRP1 expression by these cells (presumably when the activating signals derived from bacteria decrease) correlates with the initiation of their apoptosis<sup>73</sup>, but the exact mechanism of this process is unknown. Apoptosis of epithelial cells is followed by the loss of cilia and eventually appendages<sup>73</sup>, and this marks the completed colonization of the light organ with *V. fischeri* (FIG. 5). PGRP1 is also expressed in the crypts, but its role there is unknown.

PGRPs are also expressed by another group of marine invertebrates, the echinoderms. In the European starfish, *Asterias rubens*, PGRP-S1a is expressed by the epithelial cells lining the coelom, and PGRP-S2a is expressed by phagocytes. PGRP-S2a hydrolyses peptidoglycan and increases the phagocytosis of *Micrococcus luteus*, indicating that it has a role in the defence against bacteria<sup>78</sup>.

### Defence against infections by fish PGRPs

PGRPs have been identified in several species of fish, including zebrafish<sup>79</sup>, rockfish<sup>80</sup> and large yellow croaker<sup>81</sup>. Zebrafish have four PGRP genes, three of which have been cloned and characterized (namely, *pglyrp2*, *pglyrp5* and *pglyrp6*)<sup>79</sup>. These PGRPs are expressed widely in different tissues of adult fish, including in the skin, gills, liver, intestine and pancreas (FIG. 6a), indicating that they function both in the defence against bacteria that are present in the environment and during systemic infections. The high-level expression of PGRPs in the intestine also indicates a probable role of these proteins in the modulation of the fish gut microbiome. Zebrafish Pglyrp2, Pglyrp5 and Pglyrp6 are amidases that cleave the bond between N-acetylmuramic acid and alanine in peptidoglycan<sup>79</sup>. They are also bactericidal against both Gram-positive and Gram-negative bacteria<sup>79</sup> (FIG. 6a). Rockfish PGRPs also have both amidase and bactericidal activities<sup>80</sup>.

Zebrafish Pglyrp2 is highly expressed in eggs and in developing oocytes, and Pglyrp5 is induced during early embryogenesis and is strongly expressed at 72 hours post fertilization<sup>79</sup>. These proteins protect the developing embryo from infection by bacteria that are present in the surrounding water (FIG. 6a). Knockdown of *pglyrp5* expression during early embryogenesis (using a specific morpholino) results in increased sensitivity to infections with *Salmonella enterica* and *Bacillus subtilis* and a high level of embryonic mortality<sup>79</sup>. Similarly, inhibition of Pglyrp6 expression increases the sensitivity and mortality of zebrafish embryos to *Flavobacterium columnare* infection<sup>82</sup>. Therefore, PGRPs have an essential role in the defence against bacteria during the early stages of fish development, when adaptive immunity has not yet developed and thus, like in invertebrates, innate immunity is the only defence mechanism.

Zebrafish Pglyrp5 and Pglyrp6 might also affect multiple intracellular pathways. Inhibition of Pglyrp5 expression in the developing embryo using small interfering RNA modified the expression of genes involved in several pathways, including immune and inflammatory responses, signalling pathways, transcription and metabolism<sup>82,83</sup>. Inhibition of Pglyrp5 increases the expression of TLR2, TLR3, MAPK-interacting serine/threonine kinase 2, the interleukin-17 receptor and NF-κB<sup>83</sup> (FIG. 6a). Therefore, PGRPs can directly or indirectly downregulate the immune response to bacteria in the surrounding water, thus preventing a constant state of inflammation. However, additional studies confirming these roles of PGRPs are needed.

### Protection against colitis by mammalian PGRPs

Mammals have four PGRPs, three of which (PGLYRP1, PGLYRP3 and PGLYRP4) are bactericidal<sup>24,26,84,85</sup> (BOX 1) but have no amidase activity<sup>26,85,86</sup>, which distinguishes them from fish PGRPs, which have both activities<sup>79</sup>. Mammalian PGLYRP2, like fish PGRPs, has both amidase<sup>86,87</sup> and antibacterial activities (R.D., S.-Y. Wang and D.G., unpublished observations). PGLYRP1 is mainly expressed in the granules of neutrophils and eosinophils and contributes to the antibacterial activity of these cells<sup>88–90</sup>. PGLYRP1 is also expressed to a lesser extent in the salivary glands, tongue, throat, oesophagus, jejunum, ileum and caecum. PGLYRP2 has the highest constitutive level of expression in the liver, from where it is secreted into the blood<sup>91,92</sup>, and PGLYRP2 expression is induced by bacteria and cytokines (such as interleukin-1β and tumour necrosis factor) in the skin and in other epithelial cells, including those in all segments of the gastrointestinal tract<sup>26,93–95</sup>. PGLYRP3 and PGLYRP4 have the highest level of expression in the skin and on the mucous membranes, in particular in the mouth, throat, oesophagus and salivary glands and, to a lesser extent, in the stomach and the small and large intestines<sup>26,96,97</sup>. Thus, mammalian PGRPs are strategically positioned in all areas of the body that come into contact with environmental microorganisms and that are inhabited by commensal microorganisms, and it is thought that through their bactericidal effects mammalian PGRPs can influence the microbiota at these sites. PGRPs (in particular, PGLYRP1 and PGLYRP4) are also expressed in the mammary glands and are present in milk<sup>97,98</sup>, and thus these PGRPs could influence the mouth and gut microbiota as soon as newborns begin to drink milk.

All four mammalian PGRPs control the acquisition and maintenance of the normal gut microbiota, which protects the host from enhanced inflammation, tissue damage and colitis<sup>97</sup> (FIG. 6b). *Pglyrp1<sup>-/-</sup>*, *Pglyrp2<sup>-/-</sup>*, *Pglyrp3<sup>-/-</sup>* and *Pglyrp4<sup>-/-</sup>* mice are all more sensitive than wild-type mice to experimental ulcerative colitis induced by dextran sulphate sodium (DSS), a compound that damages the intestinal epithelium. The probable sequence of events that makes PGRP-deficient mice more sensitive to colitis is as follows (FIG. 6b). First, deficiencies in individual PGRPs induce dysbiosis in the colonic microbiota; this involves shifts in the levels of various groups of bacteria in the colon, and these alterations are unique for mice deficient in different PGRPs. For example, there is a decrease in the numbers of lactobacilli (which often have beneficial probiotic effects) in *Pglyrp2<sup>-/-</sup>*, *Pglyrp3<sup>-/-</sup>* and *Pglyrp4<sup>-/-</sup>* mice.

a Danio rerio

Skin  
(Pglyrp2  
and Pglyrp5)

Intestine  
(Pglyrp6)

Gills  
(Pglyrp5)

Gonads  
(Pglyrp2)

Liver  
(Pglyrp2  
and Pglyrp5)

Bactericidal

Pglyrp2

Amidase  
activity

Peptidoglycan

Pglyrp5

Downregulate  
expression

TLR2  
TLR3  
IL-17R  
MKNK2  
NF-κB

b Mammals

PGRP-deficient

Harmful  
microorganisms  
DSS

Colonic  
epithelium

Lamina  
propria

NK cell

IFNγ

CXCL9  
CXCL10  
CXCL11

Injury

Colitis

Bleeding

Loss of colonic  
epithelium

Ulcer

Hyperplasia of  
lamina propria

PGRP-sufficient

Beneficial  
microorganisms  
DSS

Colonic  
epithelium

Lamina  
propria

No bleeding

Healing of colonic  
epithelium

No hyperplasia  
of lamina propria

Protection from colitis

Figure 6 | Vertebrate PGRPs are bactericidal, defend against infections, regulate intestinal microorganisms and protect mammals against colitis. a | Danio rerio (zebrafish) peptidoglycan recognition proteins (PGRPs) are expressed in the adults, eggs and embryos and protect against bacterial infections during early development. Zebrafish PGRPs have amidase, bactericidal and immunomodulatory activities and are differentially expressed in various organs, including the skin, gills, intestine, liver and gonads in the adult fish. Zebrafish embryos develop in bacteria-contaminated water and rely on innate immunity for host defence. Zebrafish eggs express Pglyrp2 and the embryos express Pglyrp2, Pglyrp5 and Pglyrp6, and this protects the eggs and embryos from bacterial infections. b | Mammalian PGRPs promote beneficial intestinal microorganisms, which facilitate healing following intestinal injury and thus protect against colitis. A lack of PGRPs results in the overgrowth of harmful intestinal microorganisms

and, following intestinal injury with dextran sulphate sodium (DSS), this results in increased production of natural killer (NK) cell-attracting and -activating chemokines (namely, CXCL9, CXCL10 and CXCL11). This leads to the recruitment of NK cells to the colonic epithelium, the production of interferon-γ (IFNγ) by NK cells, the activation of IFN-inducible genes (which include NK cell-attracting chemokine genes) and eventually colitis, which is manifested by hyperplasia of the lamina propria, loss of colonic epithelium, ulceration, bleeding and death. PGRPs promote beneficial gut microorganisms that do not induce these chemokines and do not cause NK cell recruitment and IFNγ production, and thus they protect against colitis. This paradigm has been experimentally verified in a mouse model of colitis, but it probably applies to all mammals, including humans. IL-17R, interleukin-17 receptor; MKNK2, MAPK-interacting serine/threonine kinase 2; NF-κB, nuclear factor-κB; TLR, Toll-like receptor.

REVIEWS

Box 1 | PGRPs kill bacteria by activating two-component systems

Peptidoglycan recognition proteins (PGRPs) kill bacteria by exploiting bacterial two-component stress response systems, such as CssR–CssS in *Bacillus subtilis* and CpxA–CpxR in *Escherichia coli*<sup>85</sup> (see the figure). These two-component systems consist of a transmembrane sensor and a cytoplasmic regulator; they detect extracytoplasmic misfolded and aggregated bacterial proteins, which are generated under stress, exported from the cell and degraded by proteases<sup>104,105</sup>.

In the case of Gram-positive bacteria, PGRPs bind to peptidoglycan in bacterial cell walls at the sites of hydrolysis by LytE and LytF enzymes, which in *B.* *subtilis* separate daughter cells after cell division. In the case of Gram-negative bacteria, PGRPs bind uniformly to the entire outer membrane<sup>85</sup>, which is composed of lipopolysaccharide (LPS) and covers a thin peptidoglycan layer. In addition to binding peptidoglycan<sup>12,15,20</sup>, PGRPs bind to LPS and other molecules<sup>23–25</sup> using binding sites outside the peptidoglycan-binding groove. The binding of PGRPs to peptidoglycan or LPS probably induces PGRP oligomerization into ribbon-like structures<sup>106</sup>, which are then detected by the *B.* *subtilis* CssR–CssS or *E.* *coli* CpxA–CpxR two-component systems<sup>85</sup>.

Activation of the protein-sensing two-component systems not only triggers the removal of the misfolded proteins, but also signals into the cell to activate a repair and defence response, and induces membrane depolarization and the production of toxic hydroxyl radicals (•OH). This is accompanied by the cessation of all major intracellular biosynthetic reactions, probably because of the lack of energy generation through mechanisms that are dependent on the membrane potential<sup>85,105</sup>. If stress is sustained or its level is too high, the bacteria die. This suicide mechanism might have evolved as being advantageous to the entire bacterial population, because bacteria that are damaged beyond their ability to repair themselves would die quickly and not use up scarce resources. The mammalian PGRPs (and probably other PGRPs) trigger this bacterial defence/suicide mechanism to kill bacteria<sup>85</sup>. PGRPs use this mechanism both to defend against pathogens<sup>79</sup> and to control the levels of symbiotic microorganisms<sup>97</sup>; it is possible that symbiotic microorganisms have a higher tolerance to PGRPs than other bacteria<sup>26</sup>.

**Bacillus subtilis**

![Diagram](image1.png)

The stool microbiota from PGRP-deficient mice is more pro-inflammatory and more damaging to the colon than the stool microbiota from wild-type mice. So, *in vivo*, after intestinal injury (for example, by DSS), the more pro-inflammatory and damaging microbiota in the colon of PGRP-deficient mice induces the production of higher levels of chemokines that attract innate immune cells, in particular natural killer (NK) cells, to the site of injury. NK cells produce excessive amounts of interferon-γ, which results in the activation of interferon-inducible genes (including more NK cell-attracting chemokine genes) that promote intestinal damage and prevent healing. This insult leads to hyperplasia of the colonic lamina propria (the layer of cells underlying the colonic epithelium), loss of colonic epithelium, ulceration, bleeding and eventually death<sup>97</sup>.

The role of dysregulation of the colonic microbiota as the primary factor responsible for this sequence of events in PGRP-deficient mice has been shown by the ability of stools from PGRP-deficient mice, but not stools from wild-type mice, to transfer this increased sensitivity to colitis to wild-type germ-free mice<sup>97</sup>. This scenario is an experimental example of a genetic deficiency that results in dysbiosis, with the clinical consequence of increased sensitivity to colitis following damage to the colonic epithelium. Nucleotide-binding oligomerization domain 2 (*Nod2*) is another gene that, when mutated, induces intestinal dysbiosis and increased sensitivity to inflammatory bowel disease (IBD)<sup>99</sup>.

Thus, in wild-type animals, PGRPs promote the acquisition and maintenance of beneficial gut microorganisms that, following intestinal injury, promote intestinal healing and protect the animal from developing severe and usually lethal colitis (FIG. 6b). Although not yet proven, PGRPs probably have a similar protective role in all mammals, including humans, through the maintenance of beneficial microbial communities within the gut.

Conclusions and future directions

PGRPs control the levels of symbiotic microorganisms and defend against other bacteria in all animals that have so far been studied. In some animals, they have very specialized functions to control highly specialized symbionts, whereas in other animals they control and defend against a wide range of microorganisms, and even control parasites. PGRPs accomplish these tasks both through direct antibacterial activity and through indirect effects, including the induction of antimicrobial peptides and the modulation of inflammation and immune responses.

There are many questions regarding how PGRPs accomplish these functions. PGRPs kill bacteria by activating their two-component systems (BOX 1). Future work should define whether this activation occurs directly or indirectly and how this activation induces bactericidal mechanisms, such as membrane depolarization and the inhibition of biosynthetic reactions.

Although we know that PGRPs induce changes in the microbiota, future work should define these changes more precisely and should determine how alterations in the microbiota induce diseases or immunological imbalance. It should also be determined whether the effects of PGRPs on the gut microbial communities have systemic consequences in other inflammatory diseases and, if so, which microbial products mediate these effects. Although it is possible that all the *in vivo* effects of PGRPs are based on changes in the microbiota, it is likely that some effects of PGRPs (for example, PGLYRP2-dependent sensitivity to experimental arthritis<sup>100</sup>) are based on other intrinsic characteristics of PGRPs, and the mechanisms responsible for these outcomes need to be defined.

Increased sensitivity to colitis in PGRP-deficient mice raises the possibility that genetic defects in PGRPs might predispose humans to IBD. Many genes are associated with IBD<sup>101</sup>. The human *PGLYRP2* gene is located at position 19p13.12, within the IBD susceptibility locus 19p13, for which the susceptibility gene has not yet been identified<sup>101–103</sup>. Therefore, *PGLYRP2* could be one of the human IBD susceptibility genes. Mutations in other PGRP genes could also predispose patients to IBD, as many previously unknown IBD susceptibility genes are continuing to be discovered. These possibilities are currently being tested by genotyping patients with IBD. If human PGRPs are associated with IBD, it should be determined whether this effect is mediated through changes in the intestinal microbiota and whether this predisposition could be treated by oral administration of PGRPs.

Many other applications based on the interactions of PGRPs with the microbiota can be envisioned. New antimicrobial therapies could exploit the activation of bacterial and fungal two-component systems by PGRPs, PGRP fragments or PGRP mimetics. The role of PGRPs in maintaining beneficial microorganisms in the mouth and gut could be applied to develop toothpastes and mouthwashes that promote this healthy microbiota. Transgenic cows that express human PGRPs in milk could be produced, and this milk could be used to reduce the incidence of food-borne infections and to promote the maintenance of beneficial intestinal microorganisms. Furthermore, PGRPs or PGRP mimetics could be used to control insect-transmitted diseases, such as malaria and trypanosomiasis. Parallel dissection of PGRP functions in various invertebrates and vertebrates, and a more intense focus on the functions of PGRPs in mammals will shed more light on PGRP evolution and lead to new approaches to control the microbiome and to prevent and treat infections and other diseases that are affected by the microbiome.


1. Ley, R. E. *et al.* Evolution of mammals and their gut microbes. *Science* **320**, 1647–1651 (2008).
2. Turnbaugh, P. J. *et al.* The human microbiome project. *Nature* **449**, 804–810 (2007).
3. Lee, Y. K. & Mazmanian, S. K. Has the microbiota played a critical role in the evolution of the adaptive immune system? *Science* **330**, 1768–1773 (2010).
4. Round, J. L. & Mazmanian, S. K. The gut microbiota shapes intestinal immune responses during health and disease. *Nature Rev. Immunol.* **9**, 313–323 (2009).
5. Heijtz, R. D. *et al.* Normal gut microbiota modulates brain development and behavior. *Proc. Natl Acad. Sci. USA* **108**, 3047–3052 (2011).
6. Ley, R. E. Obesity and the human microbiome. *Curr. Opin. Gastroenterol.* **26**, 5–11 (2010).
7. Chung, H. & Kasper, D. L. Microbiota-stimulated immune mechanisms to maintain gut homeostasis. *Curr. Opin. Immunol.* **22**, 455–460 (2010).
8. Wen, L. *et al.* Innate immunity and intestinal microbiota in the development of type 1 diabetes. *Nature* **455**, 1109–1113 (2008).
9. Maslowski, K. M. *et al.* Regulation of inflammatory responses by gut microbiota and chemoattractant receptor GPR43. *Nature* **461**, 1282–1286 (2009).
10. Atarashi, K. *et al.* ATP drives lamina propria T<sub>H</sub>17 cell differentiation. *Nature* **455**, 808–812 (2008).
11. Cerf-Bensussan, N. & Gaboriau-Routhiau, V. The immune system and the gut microbiota: friends or foes? *Nature Rev. Immunol.* **10**, 735–744 (2010).
12. Royet, J. & Dziarski, R. Peptidoglycan recognition proteins: pleiotropic sensors and effectors of antimicrobial defences. *Nature Rev. Microbiol.* **5**, 264–277 (2007).
13. Yoshida, H., Kinoshita, K. & Ashida, M. Purification of a peptidoglycan recognition protein from hemolymph of the silkworm, *Bombyx mori*. *J. Biol. Chem.* **271**, 13854–13860 (1996).
14. Kang, D., Liu, G., Lundstrom, A., Gelius, E. & Steiner, H. A peptidoglycan recognition protein in innate immunity conserved from insects to humans. *Proc. Natl Acad. Sci. USA* **95**, 10078–10082 (1998). This was the first description of cloning of insect and mammalian PGRP genes, showing that PGRPs are conserved in evolution.
15. Dziarski, R. & Gupta, D. The peptidoglycan recognition proteins (PGRPs). *Genome Biol.* **7**, 232 (2006).
16. Werner, T. *et al.* A family of peptidoglycan recognition proteins in the fruit fly *Drosophila melanogaster*. *Proc. Natl Acad. Sci. USA* **97**, 13772–13777 (2000).
17. Christophides, G. K. *et al.* Immunity-related genes and gene families in *Anopheles gambiae*. *Science* **298**, 159–165 (2002).
18. Liu, C., Xu, Z., Gupta, D. & Dziarski, R. Peptidoglycan recognition proteins: a novel family of four human innate immunity pattern recognition molecules. *J. Biol. Chem.* **276**, 34686–34694 (2001). This study identified a family of PGRPs in mammals.
19. Kim, M.-S., Byun, M. & Oh, B.-H. Crystal structure of peptidoglycan recognition protein LB from *Drosophila melanogaster*. *Nature Immunol.* **4**, 787–793 (2003). This was the first reported crystal structure of a PGRP protein.
20. Guan, R. *et al.* Structural basis for peptidoglycan binding by peptidoglycan recognition proteins. *Proc. Natl Acad. Sci. USA* **101**, 17168–17173 (2004).
21. Mellroth, P. *et al.* Ligand-induced dimerization of *Drosophila* peptidoglycan recognition proteins
22. Chang, C. I. *et al.* Structure of the ectodomain of *Drosophila* peptidoglycan-recognition protein LCa suggests a molecular mechanism for pattern recognition. *Proc. Natl Acad. Sci. USA* **102**, 10279–10284 (2005).
23. Liu, C., Gelius, E., Liu, G., Steiner, H. & Dziarski, R. Mammalian peptidoglycan recognition protein binds peptidoglycan with high affinity, is expressed in neutrophils and inhibits bacterial growth. *J. Biol. Chem.* **275**, 24490–24499 (2000).
24. Tydell, C. C., Yuan, J., Tran, P. & Selsted, M. E. Bovine peptidoglycan recognition protein-S: antimicrobial activity, localization, secretion and binding properties. *J. Immunol.* **176**, 1154–1162 (2006).
25. Sharma, P. *et al.* Structural basis of recognition of pathogen-associated molecular patterns and inhibition of proinflammatory cytokines by camel peptidoglycan recognition protein. *J. Biol. Chem.* **286**, 16208–16217 (2011).
26. Lu, X. *et al.* Peptidoglycan recognition proteins are a new class of human bactericidal proteins. *J. Biol. Chem.* **281**, 5895–5907 (2006).
27. Michel, T., Reichhart, J. M., Hoffmann, J. A. & Royet, J. *Drosophila* Toll is activated by Gram-positive bacteria through a circulating peptidoglycan recognition protein. *Nature* **414**, 756–759 (2001). This study presented the first *in vivo* evidence that PGRP molecules can function as PRRs in insects.
28. Choe, K. M., Werner, T., Stoven, S., Hultmark, D. & Anderson, K. V. Requirement for a peptidoglycan recognition protein (PGRP) in Relish activation and antibacterial immune responses in *Drosophila*. *Science* **296**, 359–362 (2002).

NATURE REVIEWS | IMMUNOLOGY
VOLUME 11 | DECEMBER 2011 | 849
© 2011 Macmillan Publishers Limited. All rights reserved

REVIEW S

29. Gottar, M. *et al.* The *Drosophila* immune response against Gram-negative bacteria is mediated by a peptidoglycan recognition protein. *Nature* **416**, 640–644 (2002).

30. Ramet, M., Manfruelli, P., Pearson, A., Mathey-Prevot, B. & Ezekowitz, R. A. Functional genomic analysis of phagocytosis and identification of a *Drosophila* receptor for *E. coli*. *Nature* **416**, 644–648 (2002). References 28–30 demonstrated that a *D. melanogaster* PGRP functions both as a PRR and as a signalling transmembrane receptor upstream of the IMD pathway.

31. Gobert, V. *et al.* Dual activation of the *Drosophila* Toll pathway by two pattern recognition receptors. *Science* **302**, 2126–2130 (2003).

32. Bischoff, V. *et al.* Function of the *Drosophila* pattern-recognition receptor PGRP-SD in the detection of Gram-positive bacteria. *Nature Immunol.* **5**, 1175–1180 (2004).

33. Wang, L. *et al.* Peptidoglycan recognition protein-SD provides versatility of receptor formation in *Drosophila* immunity. *Proc. Natl Acad. Sci. USA* **105**, 11881–11886 (2008).

34. Rutschmann, S. *et al.* The Rel protein DIF mediates the antifungal but not the antibacterial host defense in *Drosophila*. *Immunity* **12**, 569–580 (2000).

35. Takehana, A. *et al.* Overexpression of a pattern-recognition receptor, peptidoglycan-recognition protein-LE, activates Imd/Relish-mediated antibacterial defense and the prophenoloxidase cascade in *Drosophila* larvae. *Proc. Natl Acad. Sci. USA* **99**, 13705–13710 (2002).

36. Leulier, F. *et al.* The *Drosophila* immune system detects bacteria through specific peptidoglycan recognition. *Nature Immunol.* **4**, 478–484 (2003). A demonstration that *D. melanogaster* can distinguish between different bacterial peptidoglycans. Lys-type peptidoglycans activate the Toll pathway, whereas DAP-type peptidoglycans activate the IMD pathway.

37. Choe, K. M., Lee, H. & Anderson, K. V. *Drosophila* peptidoglycan recognition protein LC (PGRP-LC) acts as a signal-transducing innate immune receptor. *Proc. Natl Acad. Sci. USA* **102**, 1122–1126 (2005).

38. Werner, T., Borge-Renberg, K., Mellroth, P., Steiner, H. & Hultmark, D. Functional diversity of the *Drosophila* PGRP-LC gene cluster in the response to lipopolysaccharide and peptidoglycan. *J. Biol. Chem.* **278**, 26319–26322 (2003).

39. Kaneko, T. *et al.* Monomeric and polymeric Gram-negative peptidoglycan but not purified LPS stimulate the *Drosophila* IMD pathway. *Immunity* **20**, 637–649 (2004).

40. Chang, C. I., Chelliah, Y., Borek, D., Mengin-Lecreulx, D. & Deisenhofer, J. Structure of tracheal cytotoxin in complex with a heterodimeric pattern-recognition receptor. *Science* **311**, 1761–1764 (2006).

41. Maillet, F., Bischoff, V., Vignal, C., Hoffmann, J. & Royet, J. The *Drosophila* peptidoglycan recognition protein PGRP-LF blocks PGRP-LC and IMD/JNK pathway activation. *Cell Host Microbe* **3**, 293–303 (2008).

42. Basbous, N. *et al.* The *Drosophila* peptidoglycan-recognition protein LF interacts with peptidoglycan-recognition protein LC to downregulate the Imd pathway. *EMBO Rep.* **12**, 327–333 (2011).

43. Yano, T. *et al.* Autophagic control of *Listeria* through intracellular innate immune mechanism. *Nature Immunol.* **9**, 908–916 (2008). A demonstration that PGRP-LE functions as a PRR for bacteria in the cytoplasm.

44. Kaneko, T. *et al.* PGRP-LC and PGRP-LE have essential yet distinct functions in the *Drosophila* immune response to monomeric DAP-type peptidoglycan. *Nature Immunol.* **7**, 715–723 (2006).

45. Mellroth, P. & Steiner, H. PGRP-SB1: an N-acetylmuramoyl-L-alanine amidase with antibacterial activity. *Biochem. Biophys. Res. Commun.* **350**, 994–999 (2006).

46. Park, J. W. *et al.* Clustering of peptidoglycan recognition protein-SA is required for sensing lysine-type peptidoglycan in insects. *Proc. Natl Acad. Sci. USA* **104**, 6602–6607 (2007).

47. Nehme, N. T. *et al.* A model of bacterial intestinal infections in *Drosophila melanogaster*. *PLoS Pathog.* **3**, e173 (2007).

48. Liehl, P., Blight, M., Vodovar, N., Boccard, F. & Lemaitre, B. Prevalence of local immune response against oral infection in a *Drosophila/Pseudomonas* infection model. *PLoS Pathog.* **2**, e56 (2006).

49. Corby-Harris, V. *et al.* Geographical distribution and diversity of bacteria associated with natural populations of *Drosophila melanogaster*. *Appl. Environ. Microbiol.* **73**, 3470–3479 (2007).

50. Cox, C. R. & Gilmore, M. S. Native microbial colonization of *Drosophila melanogaster* and its use as a model of *Enterococcus faecalis* pathogenesis. *Infect. Immun.* **75**, 1565–1576 (2007).

51. Ren, C., Webster, P., Finkel, S. E. & Tower, J. Increased internal and external bacterial load during *Drosophila* aging without life-span trade-off. *Cell Metab.* **6**, 144–152 (2007).

52. Ryu, J. H. *et al.* Innate immune homeostasis by the homeobox gene *Caudal* and commensal-gut mutualism in *Drosophila*. *Science* **319**, 777–782 (2008). A demonstration of immune tolerance in the *D. melanogaster* gut and the participation of *Caudal* in this process.

53. Storelli, G. *et al.* *Lactobacillus plantarum* promotes *Drosophila* systemic growth by modulating hormonal signals through TOR dependent nutrient sensing. *Cell Metab.* **14**, 403–414 (2011).

54. Sharon, G. *et al.* Commensal bacteria play a role in mating preference of *Drosophila melanogaster*. *Proc. Natl Acad. Sci. USA* **107**, 20051–20056 (2010).

55. Bischoff, V. *et al.* Downregulation of the *Drosophila* immune response by peptidoglycan-recognition proteins SC1 and SC2. *PLoS Pathog.* **2**, e14 (2006).

56. Zaidman-Remy, A. *et al.* The *Drosophila* amidase PGRP-LB modulates the immune response to bacterial infection. *Immunity* **24**, 463–473 (2006). References 55 and 56 showed that PGRPs with amidase activity dampen the *D. melanogaster* immune response.

57. Gendrin, M., Welchman, D. P., Poidevin, M., Hervé, M. & Lemaitre, B. Long-range activation of systemic immunity through peptidoglycan diffusion in *Drosophila*. *PLoS Pathog.* **5**, e1000694 (2009).

58. Kleino, A. *et al.* Pirk is a negative regulator of the *Drosophila* Imd pathway. *J. Immunol.* **180**, 5413–5422 (2008).

59. Lhocine, N. *et al.* PIMS modulates immune tolerance by negatively regulating *Drosophila* innate immune signaling. *Cell Host Microbe* **4**, 147–158 (2008).

60. Aggarwal, K. *et al.* Rudra interrupts receptor signaling complexes to negatively regulate the IMD pathway. *PLoS Pathog.* **4**, e1000120 (2008).

61. Buchon, N., Broderick, N. A., Chakrabarti, S. & Lemaitre, B. Invasive and indigenous microbiota impact intestinal stem cell activity through multiple pathways in *Drosophila*. *Genes Dev.* **23**, 2333–2344 (2009).

62. Ha, E. M., Oh, C. T., Bae, Y. S. & Lee, W. J. A direct role for dual oxidase in *Drosophila* gut immunity. *Science* **310**, 847–850 (2005). A demonstration that ROS production has an essential role in the gut immune response in *D. melanogaster*.

63. Ha, E. M. *et al.* Coordination of multiple dual oxidase-regulatory pathways in responses to commensal and infectious microbes in *Drosophila* gut. *Nature Immunol.* **10**, 949–957 (2009).

64. Meister, S. *et al.* *Anopheles gambiae* PGRPLC-mediated defense against bacteria modulates infections with malaria parasites. *PLoS Pathog.* **5**, e1000542 (2009).

65. Dong, Y., Manfredini, F. & Dimopoulos, G. Implication of the mosquito midgut microbiota in the defense against malaria parasites. *PLoS Pathog.* **5**, e1000423 (2009). A demonstration that the PGRP-LC-mediated response to the gut microbiota in mosquitoes has an indirect effect on parasite transmission.

66. Pais, R., Lohs, C., Wu, Y., Wang, J. & Aksoy, S. The obligate mutualist *Wigglesworthia glossinidia* influences reproduction, digestion, and immunity processes of its host, the tsetse fly. *Appl. Environ. Microbiol.* **74**, 5965–5974 (2008).

67. Wang, J., Wu, Y., Yang, G. & Aksoy, S. Interactions between mutualist *Wigglesworthia* and tsetse peptidoglycan recognition protein (PGRP-LB) influence trypanosome transmission. *Proc. Natl Acad. Sci. USA* **106**, 12133–12138 (2009). A demonstration of the role of PGRP-LB in tolerance towards symbiotic bacteria and in trypanosome transmission in tsetse flies.

68. Anselme, C., Vallier, A., Balmand, S., Fauvarque, M. O. & Heddi, A. Host PGRP gene expression and bacterial release in endosymbiosis of the weevil *Sitophilus zeamais*. *Appl. Environ. Microbiol.* **72**, 6766–6772 (2006).

69. Itoh, N. & Takahashi, K. G. Distribution of multiple peptidoglycan recognition proteins in the tissues of Pacific oyster, *Crassostrea gigas*. *Comp. Biochem. Physiol. B, Biochem. Mol. Biol.* **150**, 409–417 (2008).

70. Su, J., Ni, D., Song, L., Zhao, J. & Qiu, L. Molecular cloning and characterization of a short type peptidoglycan recognition protein (CfPGRP-S1) cDNA from Zhikong scallop *Chlamys farreri*. *Fish Shellfish Immunol.* **23**, 646–656 (2007).

71. Yang, J. *et al.* Peptidoglycan recognition protein of *Chlamys farreri* (CfPGRP-S1) mediates immune defenses against bacterial infection. *Dev. Comp. Immunol.* **34**, 1300–1307 (2010).

72. Goodson, M. S. *et al.* Identifying components of the NF-κB pathway in the beneficial *Euprymna scolopes*-Vibrio fischeri light organ symbiosis. *Appl. Environ. Microbiol.* **71**, 6934–6946 (2005).

73. Troll, J. V. *et al.* Peptidoglycan induces loss of a nuclear peptidoglycan recognition protein during host tissue development in a beneficial animal-bacterial symbiosis. *Cell. Microbiol.* **11**, 1114–1127 (2009).

74. Troll, J. V. *et al.* Taming the symbiont for coexistence: a host PGRP neutralizes a bacterial symbiont toxin. *Environ. Microbiol.* **12**, 2190–2203 (2010). This study identified the role of PGRP2 in *Euprymna scolopes-Vibrio fischeri* symbiosis.

75. Koropatnick, T. A. *et al.* Microbial factor-mediated development in a host-bacterial mutualism. *Science* **306**, 1186–1188 (2004).

76. Nyholm, S. V. & McFall-Ngai, M. J. The winnowing: establishing the squid-vibrio symbiosis. *Nature Rev. Microbiol.* **2**, 632–642 (2004).

77. Zhang, S. M., Zeng, Y. & Loker, E. S. Characterization of immune genes from the schistosome host snail *Biomphalaria glabrata* that encode peptidoglycan recognition proteins and Gram-negative bacteria binding protein. *Immunogenetics* **59**, 883–898 (2007).

78. Coteur, G. *et al.* Peptidoglycan recognition proteins with amidase activity in early deuterostomes (Echinodermata). *Dev. Comp. Immunol.* **31**, 790–804 (2007).

79. Li, X. *et al.* Zebrafish peptidoglycan recognition proteins are bactericidal amidases essential for defense against bacterial infections. *Immunity* **27**, 518–529 (2007). In this study, zebrafish PGRP genes were cloned and the amidase and bactericidal activities of these PGRPs and their role in the protection of embryos from bacterial infections were identified.

80. Kim, M. Y., Jang, J. H., Lee, J. W. & Cho, J. H. Molecular cloning and characterization of peptidoglycan recognition proteins from the rockfish, *Sebastes schlegeli*. *Fish Shellfish Immunol.* **28**, 632–639 (2010).

81. Mao, Y., Wang, J., Zhang, Z., Ding, S. & Su, Y. Cloning, mRNA expression, and recombinant expression of peptidoglycan recognition protein II gene from large yellow croaker (*Pseudosciaena crocea*). *Mol. Biol. Rep.* **37**, 3897–3908 (2010).

82. Chang, M. X. & Nie, P. RNAi suppression of zebrafish peptidoglycan recognition protein 6 (zfPGRP6) mediated differentially expressed genes involved in Toll-like receptor signaling pathway and caused increased susceptibility to *Flavobacterium columnare*. *Vet. Immunol. Immunopathol.* **124**, 295–301 (2008).

83. Chang, M. X., Wang, Y. P. & Nie, P. Zebrafish peptidoglycan recognition protein SC (zfPGRP-SC) mediates multiple intracellular signaling pathways. *Fish Shellfish Immunol.* **26**, 264–274 (2009).

84. Wang, M. *et al.* Human peptidoglycan recognition proteins require zinc to kill both Gram-positive and Gram-negative bacteria and are synergistic with antibacterial peptides. *J. Immunol.* **178**, 3116–3125 (2007).

85. Kashyap, D. R. *et al.* Peptidoglycan recognition proteins kill bacteria by activating protein-sensing two-component systems. *Nature Med.* **17**, 676–683 (2011). A demonstration that PGRPs kill bacteria through a novel mechanism based on the overactivation of bacterial two-component systems.

86. Wang, Z.-M. *et al.* Human peptidoglycan recognition protein-L is an N-acetylmuramoyl-L-alanine amidase. *J. Biol. Chem.* **278**, 49044–49052 (2003).

87. Gélius, E., Persson, C., Karlsson, J. & Steiner, H. A mammalian peptidoglycan recognition protein with N-acetylmuramoyl-L-alanine amidase activity. *Biochem. Biophys. Res. Commun.* **306**, 988–994 (2003).

88. Tydell, C. C., Yount, N., Tran, D., Yuan, J. & Selsted, M. Isolation, characterization, and antimicrobial properties of bovine oligosaccharide-binding protein. *J. Biol. Chem.* **277**, 19658–19664 (2002).

89. Dziarski, R., Platt, K. A., Gelius, E., Steiner, H. & Gupta, D. Defect in neutrophil killing and increased susceptibility to infection with non-pathogenic Gram-positive bacteria in peptidoglycan recognition protein-S (PGRP-S)-deficient mice. *Blood* **102**, 689–697 (2003).

90. Cho, J. H. *et al.* Human peptidoglycan recognition protein S is an effector of neutrophil-mediated innate immunity. *Blood* **106**, 2552–2558 (2005).

91. Xu, M., Wang, Z. & Locksley, R. M. Innate immune responses in peptidoglycan recognition protein L-deficient mice. *Mol. Cell. Biol.* **24**, 7949–7957 (2004).

92. Zhang, Y. *et al.* Identification of serum N-acetylmuramoyl-ʟ-alanine amidase as liver peptidoglycan recognition protein 2. *Biochim. Biophys. Acta* **1752**, 34–46 (2005).

93. Wang, H., Gupta, D., Li, X. & Dziarski, R. Peptidoglycan recognition protein 2 (N-acetylmuramoyl-ʟ-Ala amidase) is induced in keratinocytes by bacteria through the p38 kinase pathway. *Infect. Immun.* **73**, 7216–7225 (2005).

94. Uehara, A. *et al.* Chemically synthesized pathogen-associated molecular patterns increase the expression of peptidoglycan recognition proteins via Toll-like receptors, NOD1 and NOD2 in human oral epithelial cells. *Cell. Microbiol.* **7**, 675–686 (2005).

95. Li, X., Wang, S., Wang, H. & Gupta, D. Differential expression of peptidoglycan recognition protein 2 in the skin and liver requires different transcription factors. *J. Biol. Chem.* **281**, 20738–20748 (2006).

96. Mathur, P. *et al.* Murine peptidoglycan recognition proteins Pglyrplα and Pglyrplβ are encoded in the epidermal differentiation complex and are expressed in epidermal and hematopoietic tissues. *Genomics* **83**, 1151–1163 (2004).

97. Saha, S. *et al.* Peptidoglycan recognition proteins protect mice from experimental colitis by promoting normal gut flora and preventing induction of interferon-γ. *Cell Host Microbe* **8**, 147–162 (2010). A demonstration that mammalian PGRPs protect the host from colitis by maintaining the beneficial, normal gut microbiota.

98. Kappeler, S. R., Heuberger, C., Farah, Z. & Puhan, Z. Expression of the peptidoglycan recognition protein, PGRP, in the lactating mammary gland. *J. Dairy Sci.* **87**, 2660–2668 (2004).

99. Rehman, A. *et al.* Nod2 is essential for temporal development of intestinal microbial communities. *Gut* **60**, 1354–1362 (2011).

100. Saha, S. *et al.* PGLYRP-2 and Nod2 are both required for peptidoglycan-induced arthritis and local inflammation. *Cell Host Microbe* **5**, 137–150 (2009).

101. Budarf, M. L., Labbé, C., David, G. & Rioux, J. D. GWA studies: rewriting the story of IBD. *Trends Genet.* **25**, 137–146 (2009).

102. Rioux, J. D. *et al.* Genomewide search in Canadian families with inflammatory bowel disease reveals two novel susceptibility loci. *Am. J. Hum. Genet.* **66**, 1863–1870 (2000).

103. Lakatos, P. L., Fischer, S., Lakatos, L., Gal, I. & Papp, J. Current concept on the pathogenesis of inflammatory bowel disease-crosstalk between genetic and microbial factors: pathogenic bacteria and altered bacterial sensing or changes in mucosal integrity take “toll”? *World J. Gastroenterol.* **12**, 1829–1841 (2006).

104. Hyyryläinen, H. L. *et al.* A novel two-component regulatory system in *Bacillus subtilis* for the survival of severe secretion stress. *Mol. Microbiol.* **41**, 1159–1172 (2001).

105. Kohanski, M. A., Dwyer, D. J., Wierzbowski, J., Cottarel, G. & Collins, J. J. Mistranslation of membrane proteins and two-component system activation trigger antibiotic-mediated cell death. *Cell* **135**, 679–690 (2008).

106. Lim, J. H. *et al.* Structural basis for preferential recognition of diamino-pimelic acid-type peptidoglycan by a subset of peptidoglycan recognition proteins. *J. Biol. Chem.* **281**, 8286–8295 (2006).

**Acknowledgements**

Research in the authors’ laboratories is supported by the National Institutes of Health (D.G. and R.D.) and the Action Concertée Initiatve, Fondation pour la Recherche Médicale, Institut Universitaire de France, and Agence Nationale de la Recherche (J.R.).

**Competing interests statement**

The authors declare no competing financial interests.

**FURTHER INFORMATION**

Julien Royet’s homepage: [http://www.ibdml.univ-mrs.fr/equipes/equipe_gb.php?id=63](http://www.ibdml.univ-mrs.fr/equipes/equipe_gb.php?id=63)

Dipika Gupta’s homepage: [http://iusm-nw.medicine.iu.edu/faculty/dipika-gupta-ph-d/](http://iusm-nw.medicine.iu.edu/faculty/dipika-gupta-ph-d/)

Roman Dziarski’s homepage: [http://iusm-nw.medicine.iu.edu/faculty/roman-dziarski-ph-d/](http://iusm-nw.medicine.iu.edu/faculty/roman-dziarski-ph-d/)

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
