
Recent advances in clinical practice

# Genetic determinants of alcoholic liver disease

**Felix Stickel,¹ Jochen Hampe²**

### ABSTRACT

Alcoholic liver disease (ALD) accounts for the majority of chronic liver disease in Western countries. The spectrum of ALD includes steatosis with or without fibrosis in virtually all individuals with an alcohol consumption of >80 g/day, alcoholic steatohepatitis of variable severity in 10–35% and liver cirrhosis in approximately 15% of patients. Once cirrhosis is established, there is an annual risk for hepatocellular carcinoma of 1–2%.

Environmental factors such as drinking patterns, coexisting liver disease, obesity, diet composition and comedication may modify the natural course of ALD. Twin studies have revealed a substantial contribution of genetic factors to the evolution of ALD, as demonstrated by a threefold higher disease concordance between monozygotic twins and dizygotic twins. With genotyping becoming widely available, a large number of genetic case-control studies evaluating candidate gene variants coding for proteins involved in the degradation of alcohol, mediating antioxidant defence, the evolution and counteraction of necroinflammation and formation and degradation of extracellular matrix have been published with largely unconfirmed, impeached or even disproved associations. Recently, whole genome analyses of large numbers of genetic variants in several chronic liver diseases including gallstone disease, primary sclerosing cholangitis and non-alcoholic fatty liver disease (NAFLD) have identified novel yet unconsidered candidate genes. Regarding the latter, a sequence variation within the gene coding for patatin-like phospholipase encoding 3 (PNPLA3, rs738409) was found to modulate steatosis, necroinflammation and fibrosis in NAFLD. Subsequently, the same variant was repeatedly confirmed as the first robust genetic risk factor for progressive ALD.

---

### Natural course and pathogenesis of ALD

Unlike many other chronic liver diseases, ALD is a potentially avoidable disease since excess alcohol consumption is needed for its development. In other words, ALD only becomes a liver disease if interventions against harmful drinking do not take place or fail. However, scientific and common cognition show that alcohol consumption per se is not necessarily sufficient to elicit significant ALD since only a minority of heavy drinkers progress to severe ALD. Studies in humans estimate that liver steatosis evolves in all excessive drinkers, but only about one-third develop significant necroinflammation and fibrosis and only about 10% progress to cirrhosis. Among the latter, 1–2% per year develop hepatocellular carcinoma (HCC) as a severe complication. This variable natural course is believed to result from a complex interplay between environmental and host risk factors.

Various mechanisms are implicated in alcohol-associated acute and chronic liver injury. Among these, experimental and human data suggest a primary role for cytochrome P4502E1 (CYP2E1), a microsomal enzyme which can be induced 10–20-fold by chronic alcohol consumption. CYP2E1 metabolises ethanol to acetaldehyde, a highly toxic and mutagenic molecule, and increases oxidative stress by the production of reactive oxygen species (ROS) and lipid peroxides such as 4-hydroxy-2,3-nonenal, 4-hydroxy-2,3-alkenals and malondialdehyde. CYP2E1 induction is associated with fat accumulation, inflammation and fibrosis. Furthermore, excessive alcohol consumption can lead to an increased portosystemic uptake of endotoxins from gut bacteria which contribute to necroinflammation and fibrosis progression via various molecular mechanisms including tumour necrosis factor α (TNFα) and the CD14/toll-like receptor-4 complex to produce ROS via NADPH oxidase.

Patients with ALD often present with hyperhomocysteinaemia resulting from impaired methyl group transfer due to deficiencies of folate, vitamins B₁₂ and B₆ as well as acetaldehyde-mediated interference with enzymes involved in one-carbon metabolism. On the other hand, elevated hepatic homocysteine results in endoplasmic reticulum stress by causing misfolding of endoplasmic reticulum proteins and promotes necroinflammation.

Progressive fibrosis is crucial in chronic ALD as it may eventually result in cirrhosis, liver failure and associated clinical complications. In response to acetaldehyde, ROS, lipid peroxides and endotoxins, Kupffer cells and other inflammatory cells

---

### ALCOHOLIC LIVER DISEASE AS A COMPLEX TRAIT

Alcoholic liver disease (ALD) is a complex disease whose development depends on long-term excessive drinking and other environmental, individually acquired and inherent modifying factors (figure 1). These factors interact over time to allow for or prevent the development and progression of ALD. While many environmental factors have been identified following careful epidemiology studies in the second half of the 20th century, the more recent advances in our understanding of human genetic variation and the research thereof have prompted us to search for host genetic factors that influence the course of many complex diseases including ALD.

Downloaded from http://gut.bmj.com/ on December 15, 2014 - Published by group.bmj.com

Recent advances in clinical practice

### Key messages

- Alcoholic liver disease (ALD) is a complex trait in which environmental and host factors modulate drinking behaviors and evolution and progression of liver damage.
- Confirmed environmental risk factors for progressive ALD include quantities of alcohol, obesity, and chronic co-infection with hepatitis viruses.
- There is strong evidence supporting a genetic background as an important modulator of susceptibility to ALD.
- There is a lack of evidence for a role of HFE mutations.
- Heterozygous and homozygous carriage of PNPLA3 rs738409 (G) allele is the first confirmed genetic risk factor for progressive ALD.

become activated to secrete numerous growth factors and cytokines including TNFα, platelet-derived growth factor and transforming growth factor β1 (TGFβ1), both pivotal stimuli of hepatic stellate cell (HSC) and portal myofibroblast (MFB) proliferation and secretory function. Activated HSCs/MFBs markedly increase the production of fibrous matrix, particularly collagens, and downregulate their degradation. The latter is mediated by collagenases termed matrix metalloproteinases (MMPs) which are under the control of the corresponding tissue inhibitors of matrix metalloproteinases. Based on this multifactorial concept of ALD, polymorphic variants of genes coding for proteins involved in these processes have been selected for genetic case-control studies, as described below.

#### Host versus environmental factors modulating ALD progression

Risk factors for the progression of alcoholic fibrosis are commonly stratified into host or environmental modifiers, or genetic and non-genetic factors, respectively. By far the most crucial environmental factor is alcohol, and there is a clear relationship between the amount of alcohol consumed and the likelihood of developing ALD (figure 2). Alcoholic steatosis occurs in approximately 60% of subjects who drink >60 g/day, and the risk of developing cirrhosis is highest among those with a consumption of >120 g/day. Other undisputed risk factors for progression of fibrosis in ALD are being overweight and co-infection with hepatitis C virus.

![Figure 1](https://i.imgur.com/yourimageurl.png)
**Figure 1** Schematic account of factors of progression in alcoholic liver disease showing the multifactorial pathogenesis of the disease.

![Figure 2](https://i.imgur.com/yourimageurl.png)
**Figure 2** Relationship between the amount of alcohol consumed and the likelihood of developing cirrhotic alcoholic liver disease (ALD).

Three lines of evidence are indicative of at least a partial genetic background of the ALD phenotype: (1) women are more susceptible to ALD following the consumption of similar amounts of alcohol; (2) Hispanic subjects are more prone to developing ALD than black and white subjects; and (3) twin studies have shown that monozygotic twins have a higher prevalence of alcoholic cirrhosis than dizygotic twins. Gender-specific susceptibility to alcohol has long been recognized. Studies in humans have shown that women are more susceptible to the hepatotoxic effects of alcohol and develop ALD more quickly than men with the same daily alcohol consumption. The pathophysiology behind this increased sensitivity to alcohol is not yet fully understood but—among other factors—could be related to hormonal differences such as estrogens and their synergistic impact on oxidative stress and inflammation. Gender-related metabolic differences also exist. Women who drink the same amount of alcohol as men have higher blood ethanol levels due to higher gastric alcohol dehydrogenase levels resulting in a faster first-pass metabolism of alcohol in men or a lower volume of distribution of alcohol in women.

Differences in the prevalence of ALD and associated mortality have been reported in different ethnic groups. Stinson et al analyzed the mortality of subjects with alcoholic cirrhosis across races and found the highest mortality rates for men in white Hispanics, followed by black non-Hispanics, white non-Hispanics, and black Hispanics. For women, the order was black non-Hispanics, white Hispanics, white non-Hispanics, and black Hispanics. However, such seemingly ethnic differences in the rates of alcoholic cirrhosis and ALD could also be related to the amount and type of alcohol consumed, dietary preferences, differences in socioeconomic status, access to medical care, and/or differences in attitudes towards a healthy lifestyle.

Alcoholics often show biochemical and histological signs of iron overload reflected by raised serum ferritin levels and accentuated iron staining on liver

biopsies, respectively.²³ The identification of the genetic background of iron overload by detection of mutations within the haemochromatosis (HFE) gene gave rise to the hypothesis that their presence may also affect iron storage in alcohol abusers. However, genetic case-control studies with adequate matching of cases and controls for age, sex and alcohol consumption found no increased prevalence of HFE mutations in alcoholics with liver disease, including cirrhosis.²⁴²⁵

THE SEARCH FOR GENETIC RISK FACTORS OF ALD

Candidate gene case-control studies

Until recently, the most extensively applied methodology to identify genetic risk factors of ALD (and many other complex diseases) was through genetic case-control studies. With simple and reliable genotyping methods becoming widely available, the attempt to identify possible genetic modifiers of the risk of ALD prompted a large number of hypothesis-driven candidate gene case-control studies comparing allelic and/or genotypic frequencies of certain genetic variants (ie, single nucleotide polymorphisms, SNPs) between individuals with alcoholic cirrhosis or alcoholic hepatitis, alcoholics without apparent liver disease and healthy controls.²⁶ SNPs are the most common type of allelic variation (~90%) and can be found throughout the human genome at a frequency of 1 every 1000–2000 base pairs (http://www.ncbi.nlm.nih.gov/SNP). They refer to a relatively frequent nucleotide sequence variation (arbitrary cut-off for minor allele frequency >1%) in which a single base is replaced by another leading to an altered base triplet that potentially could code for a different amino acid when located in the coding sequence (non-synonymous SNP). If located within the coding region of a gene, such coding SNP could potentially alter the function of the generated protein (eg, enzyme activity, substrate specificity). If located in the promoter sequence, SNPs may result in altered quantitative expression of the respective gene. The large majority of SNPs (>99%) are situated in non-coding regions of genes and therefore their functional implication is often unclear, but may still affect gene splicing or transcription factor binding.²⁷ This must be considered when choosing candidate polymorphisms for association studies. In the majority of studies, genetic variants were chosen based on the known pathophysiology of ALD, assuming that genetic variation of genes involved in key events of the evolution of chronic alcoholic liver damage such as oxidative degradation of alcohol, tissue remodelling or inflammation would render their carriers either resistant or susceptible to the hepatotoxic effects of alcohol (table 1). This strategy was obviously restricted to genes selected from a panel of candidates considered important for ALD, but failed to detect as yet disregarded genetic variants that may affect the natural course of ALD even more significantly. The potential of this approach to create novel hypotheses is therefore low. Overall, despite

a large number of case-control studies since the late 1980s, no confirmed genetic factor for ALD had been established until recently.²⁸²⁹ Failure to replicate findings from index studies was often due to inherent methodological limitations, mostly with respect to sample size and characterisation/selection of cases and controls (box 1). Thus, future candidate gene case-control studies should seek to avoid these pitfalls by assuring statistical power via large-scale multicentre cooperations, careful phenotyping of cases and controls, and the selection of promising candidate variants derived from hypothesis-generating genome-wide analyses.

Genetic case-control studies testing various genetic variants for a possible association with ALD are described below and judged for their strengths and weaknesses.

Genetic variants modulating addiction to alcohol

Genetic risk factors which increase the likelihood of alcohol dependence or a strong preference for alcohol consumption are likely to influence the risk of developing ALD as they increase alcohol consumption as a prerequisite for liver injury. Combined analyses of twin studies show that the overall heritability for alcoholism is approximately 50%,³⁰ and members of families with a history of alcoholism carry a significantly increased risk of developing alcohol dependence than individuals without a history of alcoholism.³¹ Numerous candidate gene case-control studies and genome-wide analyses have been performed, and the genes associated with alcoholism in these studies belong either to neurotransmission or alcohol-metabolising genes. Of particular interest for modulating the preference for and withdrawal of alcohol are the inhibitory neurotransmitter γ-aminobutyric acid (GABA) and its corresponding receptors located on chromosomes 4, 5, 15 and X. In this regard, several studies—including a Collaborative Study on the Genetics of Alcoholism (COGA) launched by the National Institute on Alcohol Abuse and Alcoholism (NIAAA) and two genome-wide association studies—have identified and independently confirmed the association of alcohol dependence with a chromosome 4q13-q11 GABAA cluster³²³³ which was further refined to the GABRA2 gene.³⁴ Polymorphic variation of GABRA2 was also linked to alcohol withdrawal symptoms, daily alcohol consumption and the extent of positive reinforcement by alcohol consumption among non-dependent alcohol consumers.³⁵³⁶ Conceivably, genetic risk factors modulating preference and aversion towards alcohol may well affect ALD evolution and other alcohol-related organ dysfunctions, although their functional role is unrelated to liver pathophysiology. Variants of genes expressing alcohol-degrading enzymes were found to be associated with alcoholism and also with alcohol-related organ damage, and will therefore be addressed in more detail below.

Genetic variation of alcohol-metabolising enzymes

Cytosolic alcohol dehydrogenase (ADH) and mitochondrial aldehyde dehydrogenase (ALDH) are

Recent advances in clinical practice

Table 1 Genes tested for association with alcoholic liver disease (ALD)

| Gene | Existing evidence |
| --- | --- |
| **Genes involved in alcohol metabolism** |  |
| ADH1B | Conflicting results for all variants; data from meta-analysis suggest no association with ALD |
| ADH1C |  |
| CYP2E1 |  |
| CYP1A1 |  |
| ALDH2 |  |
| **Genes related to oxidative stress** |  |
| GSTM1 | Mostly negative studies; positive associations challenged by failure to replicate in subsequent studies; MnSOD variation may be associated with hepatocellular carcinoma (to be confirmed independently) |
| GSTP1 |  |
| GSTT1 |  |
| MnSOD |  |
| NAT |  |
| HFE |  |
| **Genes implicated in immune reactions** |  |
| TNFα | All conflicting or yet unconfirmed except for TNFα-238A allele which was found to be associated with ALD in several independent studies and by meta-analysis; and CD14 variation which was linked to ALD in three studies |
| Interleukin (IL)-10 |  |
| IL-1R antagonist |  |
| IL-1b |  |
| CD14 receptor |  |
| CTLA-4 |  |
| TLR4 |  |
| **Genes coding for fibrosis-associated factors** |  |
| TGFB1 | No association in sufficiently powered genetic case-control studies |
| MMP3 |  |
| **Genes involved in the modulation of steatosis** |  |
| PPARγ | As yet unconfirmed single studies of PPARγ and MTP variants with low sample sizes showing significant association with ALD; ApoE with conflicting data from two very small studies |
| MTP |  |
| ApoE |  |
| **Miscellaneous** |  |
| NFKB1 | Single study showing association of NFKB1 variant-94ins/del with fibrosis; DRD2 and SLC6A4 negative |
| DRD2 |  |
| SLC6A4 |  |

responsible for metabolising the magnitude of ingested ethanol, and their expression is highest in the liver but also detectable in the surface epithelia of luminal organs such as the gastrointestinal tract. Following absorption, alcohol is degraded by ADH in the cytosol and CYP2E1 in microsomes to acet-

Box 1 Methodological limitations in genetic case-control studies

- Low sample size (type I and II errors)
- Lack of adequate statistical power (should be at least 80% for minor allele frequency)
- Single-centre studies (selection bias)
- Insufficient characterisation of cases (assessment of alcohol consumption, comorbidities)
- Inappropriate selection of controls (eg, blood donors, healthy individuals)
- Lack of control for confounding factors (eg, overweight, coffee consumption, smoking, cannabis consumption)
- No replication of association in an independent cohort
- Population admixture (mixed ethnicity)
- Deviation from Hardy-Weinberg equilibrium, population stratification
- Lack of functional characterisation of the tested SNP (eg, intronic SNPs, no expression in liver tissue)
- Publication and time-lag bias (reluctance of researchers and editors to publish negative but equally meaningful results)

Germany assessed the role of ADH1C polymorphism in patients with alcohol-associated cancers including HCC and found a significant association of genotype ADH1C*1/1 with HCC (OR 3.53, 95% CI 1.31 to 9.47; p=0.012) compared with subjects with alcoholic cirrhosis. Thus, polymorphisms of genes coding enzymes involved in alcohol metabolism play a role in modulating alcohol dependence and tolerance (and possibly cancer), but there is no evidence of their impact on the risk of genuine ALD.

Variants of genes involved in the generation and counteraction of oxidative stress

Several enzyme systems including CYP2E1, the mitochondrial respiratory chain and the cytosolic enzymes xanthine oxidase and aldehyde oxidases are important sources of O₂⁻ and H₂O₂ in hepatocytes during ethanol oxidation. Antagonistic to these are mechanisms which counteract oxidative stress generated in ALD such as glutathione-S-transferases (GSTs) and superoxide dismutases (SODs) for which numerous SNPs have been described. While enzymes increasing oxidative pressure were not found to be associated with ALD, some controversy prevails regarding the effect of GST and SOD genetic variants. GSTs are enzymes comprising four subfamilies (GSTA, GSTM, GSTT, and GSTP) which conjugate ROS and many toxic and carcinogenic xenobiotics with glutathione. Interestingly, the isoenzyme GSTP1 is expressed in biliary epithelial cells and HSCs, and polymorphic variation at base pair 313 (A→G) within the coding region leads to an amino acid substitution (Ile→Val) at codon 105 and threefold decreased capability of GSTP1 to detoxify 4-hydroxynonenal, a highly reactive lipid peroxide promoting fibrogenesis. Genotype GSTP1 Val/Val is associated with cirrhosis development in hereditary haemochromatosis, a disease in which oxidative stress is the crucial pathogenic trigger. However, the findings of >10 case-control studies investigating GST SNPs in patients with ALD from different ethnicities have remained inconsistent and all positive findings in 'index studies' were later challenged by negative results in independent cohorts.

Similarly, results on the role of mitochondrial manganese SOD (MnSOD) SNPs in modifying the risk of ALD are controversial. Mitochondria-derived ROS are detoxified to hydrogen peroxide and water by the successive action of MnSOD and glutathione peroxidase. MnSOD is expressed with a cleavable N-terminal target sequence that facilitates its import into mitochondria. A variation (C→T) within codon 16 of the precursor protein leads to either alanine (Ala) or valine (Val) at amino acid position -9 of the target sequence which results in its enhanced translocation into mitochondria and a 40% higher concentration of active MnSOD in case of the Ala sequence. A French study of 71 patients with ALD and 79 healthy blood donors found that homozygosity for Ala in the mitochondrial targeting sequence of MnSOD

increased the risk of microvesicular steatosis threefold and that of alcoholic hepatitis and cirrhosis sixfold and 10-fold, respectively. However, in a larger study from the UK, genotyping for MnSOD in 281 patients with advanced ALD (cirrhosis/fibrosis) and 218 drinkers without liver disease found no differences in heterozygote (55% vs 50%) or homozygote (19% vs 23%) Ala allele frequency. The negative findings of the latter study illustrate the problems of the low sample size and inappropriate controls included in the earlier study. As yet, confirmed associations between GST and SOD variants with ALD have not been established.

Variations of genes controlling hepatic lipid storage

The initial and uniform alcohol-associated liver abnormality is steatosis in almost all chronic drinkers. Variants of genes that govern hepatic fat storage and mobilisation are therefore plausible modifiers of this early alcohol-related liver lesion. However, surprisingly few case-control studies have been carried out testing genetic variants of modifiers of hepatic lipid turnover in the context of ALD.

Particularly appealing as candidate genes appear to be peroxisome proliferator-activated receptors (PPARs), a group of nuclear receptors which act as transcription factors in the regulation of cell differentiation, development, tumour growth and fat metabolism. The family of PPAR comprises PPArα, PPArδ and PPArγ nuclear receptors which mediate transcriptional responses to insulin resulting in cellular glucose uptake, increased fatty acid oxidation, lipogenesis and lipid storage. A polymorphic splice variant with an amino acid substitution (Pro12Ala) following a nucleotide substitution (C→G) in the PPArγ2 gene leads to 5–6-fold increased transcriptional activity of PPArγ. Rey et al genotyped 259 healthy blood donors and 263 patients with histologically-proven non-alcoholic fatty liver disease (NAFLD) and 100 patients with alcoholic steatosis for this locus. Neither NAFLD nor ALD was associated with PPArγ genotypes, but carriage of the mutated PPArγ Ala12 allele was associated with histological necroinflammation in patients with alcoholic steatosis (OR 2.50, 95% CI 1.05 to 5.90; p=0.028).

A possible association of polymorphisms in the genes coding for apolipoprotein E (ApoE) and microsomal triglyceride transfer protein (MTP) with ALD has been tested in insufficiently powered case-control studies with inconsistent results.

Genetic polymorphisms modulating endotoxin-mediated inflammation

ALD, and in particular alcoholic hepatitis, can be complicated by marked necroinflammation characterised by cellular ballooning, cytoskeleton aggregates (Mallory-Denk bodies), apoptosis, decay of liver cells and neutrophilic infiltrates. These severe histological changes are associated with significant clinical manifestations such as subacute liver failure, massive jaundice, portosystemic

encephalopathy, infections and death.54 The triggers of this worrisome sequelae are endotoxins—that is, lipopolysaccharides (LPS) derived from intestinal Gram-negative bacteria shifted from the gut to the liver following alcohol-mediated disruption of the mucosal barrier function and changes in the gastrointestinal microbiota.55 LPS stimulate Kupffer cells (liver macrophages) to secrete proinflammatory cytokines including TNFα, a proinflammatory cytokine which can stimulate hepatocyte regeneration but also hepatocyte death by eliciting apoptosis and necrosis. Functionally, endotoxin receptors are attractive candidates for genetic case–control studies and, recently, a C→T (-159) polymorphism in the promoter region of the CD14 endotoxin receptor gene was detected and found to result in increased CD14 expression. In total, four studies have tested a possible association of CD14 genetic variants and ALD, of which the largest found an OR of 4.17 (95% CI 1.56 to 11.16, p=0.005) for developing alcoholic cirrhosis in CD14-159TT homozygous heavy drinkers compared with CC homozygotes.56 This association was confirmed in a second smaller study which also showed that CD14-159TT homozygotes have higher levels of the LPS-binding acute phase proteins LBP and soluble CD14 than carriers of the C allele.57 A third study also found a higher frequency of the CD14-159TT in subjects with alcoholic cirrhosis compared with those without, but the sample size was relatively low with only 62 cases (118 controls).58 Finally, a small genetic study from Portugal could not find an association between carriage of the CD14-159T allele and ALD.59

Several studies have selected a promoter polymorphism (C→A at position -627) in the gene coding for interleukin 10 (IL-10) which results in decreased IL-10 production to test its association with ALD.60-62 IL-10 acts as an anti-inflammatory cytokine by downregulating proinflammatory cytokines including IL-1, TNFα, IL-6, IL-8 and IL-12 and upregulation of the IL-1R antagonist to inhibit collagen I gene transcription and increase collagenase expression. Grove and coworkers genotyped 287 heavy drinkers with biopsy-proved advanced ALD, 107 alcoholics without histological evidence of ALD and 227 local healthy volunteers for the IL-10 promoter SNP and found a significantly higher fraction of the IL10-627A allele among those with advanced ALD (50%) compared with drinkers without histological evidence of ALD (34%) and healthy controls (33%). However, this association was not confirmed in two similar studies which might have missed an association due to small sample sizes.6162

Among the few associations between genetic variants and ALD that have been replicated independently are SNPs in the promoter of the TNFα gene, for which a number of SNPs have been described. The most common variants are upstream of the gene at positions -1031, -863, -857, -851, -376, -308 and -238, of which the latter two are associated with higher TNFα production. Eleven

studies investigating the roles of -308 and -238 TNFα variants were recently included in a meta-analysis which found a significant overall association (OR 1.47; 95% CI 1.05 to 2.07) of variant -238A with alcoholic cirrhosis.63 Other studies tested a possible association of ALD with carriage of variants in genes coding for the IL-1 receptor antagonist IL-1β and the cytotoxic T lymphocyte antigen-4 gene (CTLA-4), but their positive associations are as yet unconfirmed.64-67

Polymorphic variants of fibrosis-associated genes Progressive fibrosis is the hallmark of most chronic liver diseases including ALD and resembles the process of wound healing. Considering the importance of matrix accumulation in ALD progression, surprisingly few case–control studies have selected variants of fibrosis-associated genes. In this regard, the only two candidate genes selected are those coding for TGFβ1, which activates HSCs to produce collagens, and MMP3, a matrix-degrading (fibrolytic) enzyme. Two relatively large studies included subjects with alcoholic cirrhosis as cases and alcoholics without apparent liver disease as controls matched for age, gender and cumulative alcohol consumption, but found no association of MMP3 and TGFβ1 genotypes with ALD.6869

Role of PNPLA3 polymorphism in ALD The identification and repeated confirmation of a genetic polymorphism in the gene coding for patatin-like phospholipase domain-containing 3 (PNPLA3; adiponutrin; rs738409C/G, M148I) as a genetic risk factor for increased fat storage in patients with NAFLD has stimulated similar genetic analyses in patients with ALD.70 Mindful of the multiple similarities shared by NAFLD and ALD, Tian and coworkers investigated a possible relation with ALD using 17 variants in PNPLA3 and a panel of 306 ancestry-informative SNPs in a Mestizo (mixed European and Native American ancestry) population with a history of alcohol abuse.71 A significant association of PNPLA3 rs738409GG with cirrhosis was detected (OR 2.25, p=1.7×10−10). Owing to marked differences in allele frequencies of PNPLA3 rs738409G across populations, an ancestry-adjusted analysis was performed which confirmed the initial observation (adjusted OR 1.79, p=1.9×10−5). This exciting revelation has since been replicated independently in three different genetic case–control studies in the UK, Belgium and Germany (table 2). First confirmation of a role for PNPLA3 rs738409 came from a report by Seth and coworkers who studied 266 subjects with alcoholic cirrhosis and 182 heavy drinkers with normal liver enzymes and no clinical evidence of liver damage.72 Carriage of PNPLA3 rs738409G allele and GG genotype were significantly associated with alcoholic cirrhosis. Detailed analyses such as a possible association with liver enzyme levels or Child-Pugh scores were not provided in this research letter. More details were reported in a comparably sized case–control study conducted in Belgium in which patients with ALD

Recent advances in clinical practice

Table 2 Genetic case-control studies investigating PNPLA3 rs738409 in alcoholic liver disease (ALD)

| Study       | Design                                                                 | Patients                                                                                          | Results                                                                                                                                                                                                 |
|-------------|------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| Tian et al<sup>71</sup> | Genetic case-control (Mestizo subjects with strong ethnic heterogeneity) | 482 cases (alcoholic cirrhosis) 434 non-cirrhotic ALD 305 alcoholics without liver enzyme elevations | rs738409 GG associated with cirrhosis compared with controls (OR 2.25, 95% CI 1.74 to 2.9; p=1.7×10<sup>-10</sup>) and with non-cirrhotic ALD (OR 1.43, 95% CI 1.15 to 1.78, p=1.0×10<sup>-3</sup>) Association robust after ancestry correction (OR 1.81, 95% CI 1.36 to 2.41; p=4.7×10<sup>-5</sup>) |
| Seth et al<sup>72</sup> | Genetic case-control (British Caucasians)                         | 266 cases (alcoholic cirrhosis) 182 controls (heavy drinkers without clinical ALD)                | rs738409 G homozygosity associated with alcoholic cirrhosis (OR 7.34, 95% CI 2.19 to 24.52, p=0.0012) Carriage of rs738409 G allele associated with alcoholic cirrhosis (OR 1.95, 95% CI 1.34 to 2.84, p=0.00002) |
| Trepo et al<sup>73</sup> | Genetic case-control (Belgium Caucasians)                        | 330 cases (97% biopsy-proven ALD; 263 cirrhotics) 328 controls (healthy individuals without ALD)   | rs738409 G associated with ALD (OR 1.54, 95% CI 1.12 to 2.11, p=0.008) and alcoholic cirrhosis (OR 2.08, 95% CI 1.15 to 3.77, p=0.02) PNPLA mRNA expression inversely correlated with cirrhosis and portal pressure |
| Stickel et al<sup>74</sup> | Genetic case-control (German Caucasians)                       | Multicentre sample with 1043 alcoholics (210 cirrhosis, 394 non-cirrhotic ALD, 439 alcoholic controls) Population-based sample with 376 alcoholics (269 non-cirrhotic ALD, 107 alcoholic controls) Non-alcoholic healthy subjects (n=162) | Genotype rs738409 GG associated with alcoholic cirrhosis (OR 2.79, 95% CI 1.55 to 5.04, p=1.18×10<sup>-5</sup>, cirrhosis vs controls) Genotype rs738409 GG associated with alanine aminotransferase elevation (OR 2.33, 95% CI 1.27 to 4.26, p=0.0085) Confirmation of association in separate replication sample (OR 4.75, 95% CI 1.08 to 20.9, p=0.04, ALD vs alcoholic controls) Population-attributable risk of rs738409 G to cirrhosis 26.6% |

were compared with healthy controls who did not drink significant amounts of alcohol.<sup>73</sup> Although the control group poses a limitation to this study, again PNPLA3 rs738409 G was significantly associated with alcoholic liver injury and cirrhosis. The selection of healthy controls may explain why the ORs were slightly lower than in other case-control studies.

The largest study to date on PNPLA3 variation in alcoholic Caucasians was conducted in Germany and included 1043 alcoholic cases and controls from cohorts recruited in eight different German tertiary hospitals and an independent replication cohort (n=376) from a population-based sample.<sup>74</sup> rs738409 GG was significantly more frequent in patients with alcoholic liver cirrhosis and in non-cirrhotic alcoholics with elevated liver enzymes than in alcoholics without liver damage (n=439). The latter association was confirmed in an independent population-based cohort of drinkers at risk of ALD with a median alcohol intake of 300 g/week and elevated aspartate aminotransferase levels. Of note, the population-attributable risk from PNPLA3 rs738409 G—that is the reduction in incidence of ALD that would be observed if all drinkers were homozygous carriers of the protective rs738409 C allele—was calculated at 26.6%.

Despite the undoubted evidence of its important role in conferring genetic risk to develop progressive ALD, the functional implications of PNPLA3 variation remain unclear as yet, and belief among researchers prevails that its molecular role goes beyond that of mere modulation of fat storage. PNPLA3 is located on chromosome 22 and encodes a 481-amino acid protein. It shares similarities with PNPLA2, the major triglyceride hydrolase in peripheral fat tissue.<sup>75</sup> Expression of PNPLA3 among species is distinct, with high hepatic expression in humans whereas expression is low in mice and rat livers from normal animals in which adipose tissues account for the magnitude of PNPLA3.<sup>76 77</sup> However, fat challenge upregulates hepatic PNPLA3 expression in mice whereas starving leads to its downregulation.<sup>78 79</sup> In cells, PNPLA3 is localised on membranes rather than adjacent to lipid droplets as one would expect, suggesting that its function may also involve receptor-like interactions with extracellular signals.<sup>79</sup> Interestingly, the lengths and types of fatty acids to which cells are exposed lead to different PNPLA3 responses with saturated (palmitate), monounsaturated (oleate) and polyunsaturated fatty acids (linoleic acid) causing increased PNPLA3 expression, whereas long chain fatty acids such as arachidonic or eicosapentanoic acid had no such effect.<sup>76</sup> In vitro studies with human PNPLA3 showed that PNPLA3 hydrolyses triglycerides and partitions between cell membranes and lipid droplets. Stable transfection of mutant PNPLA3 rs738409 G into HuH-7 hepatoma cells led to a marked reduction of its hydro-lase activity in mutant cells compared with wild type PNPLA3, consistent with the concept of a 'loss of function' polymorphism allowing for 'lipid trapping' in the liver in case of the polymorphic variant.<sup>80</sup> However, modelling the effect of PNPLA3 on disease phenotypes proved difficult as mice deleted for Pnpla3 by gene targeting showed no propensity to increased body weight, insulin resistance, elevated liver enzyme levels or steatosis on histology.<sup>81 82</sup> Explanations for this obvious lack of in vivo relevance in mice could be species-related differences of the function of PNPLA3 in the evolution of murine and human steatosis; or it

Downloaded from http://gut.bmj.com/ on December 15, 2014 - Published by group.bmj.com

Recent advances in clinical practice

The data on PNPLA3 allow for the conclusion that PNPLA3 rs738409 GG carriers represent a genetically-defined subpopulation of high-risk subjects susceptible to progression of clinically unapparent disease to overt ALD. It remains to be established whether the PNPLA3 genotype represents a marker that will assist decision-making in clinical practice and whether it could serve as a therapeutic target. However, data on ALD from genome-wide analyses similar to those generated for other chronic liver diseases are still lacking. Conducting this kind of global scan would help to confirm PNPLA3 rs738409 as an important susceptibility gene of ALD and also potentially identify novel as yet unknown genetic variants for better screening of patients, and subsequent clinical and experimental research. Despite these limitations, functional pathway information and the genetic risk factors of ALD start to converge into a common aetiological concept, allowing for a better understanding of this serious disorder in the future.

**Funding** Research into the genetics of ALD carried out by FS was in part funded by the Swiss National Fund (grant no. 310030_138747/1).

**Competing interests** None.

**Patient consent** Obtained.

**Contributors** Both authors shared the work on conception and design, analysis and interpretation of data, drafting the article and revising it critically for important intellectual content and final approval of the version to be published.

**Provenance and peer review** Commissioned; externally peer reviewed.

---

**Figure 3** Schematic overview of the pathogenesis of alcoholic liver disease (ALD) and gene variants implicated in its aetiology. Confirmed or likely variants supported by consistent meta-analysis results and multiple candidate gene studies are shown in dark blue boxes (*PNPLA3*, *TNFα*). As yet unconfirmed or controversial risk genes with conflicting evidence are shown in light blue boxes with dashed lines. Mediators of alcohol-related hepatotoxicity are shown in red boxes. Genes are printed in italics. ADH, alcohol dehydrogenase; ALDH, aldehyde dehydrogenase; ApoE, apolipoprotein E; CYP2E1, cytochrome P450 2E1; FFA, free fatty acids; LSEC, liver sinusoidal endothelial cells; PPARγ, peroxisome proliferator-activated receptor γ; ROS, reactive oxygen species; TG, triglycerides; TNFα, tumour necrosis factor α.

could be a hint towards a specific effect on inflammatory and/or fibrogenesis-associated downstream signalling pathways of PNPLA3 distinct from mere fat storage. To clarify these unresolved questions, models with *Pnpla*-/- mice suited to study aspects of fibrosis and necroinflammation would be helpful, particularly since it remains so far unexplained why there are clear associations of PNPLA3 rs738409 G carriage with progressive fibrosis in patients with ALD and also in patients with chronic hepatitis C who drink alcohol. This phenomenon appears to be a good example of how host-environment interactions modulate the phenotype of a disease.

---

**CONCLUSION AND OUTLOOK**

In the search for genetic risk factors for ALD, much work has been carried out but most investigated genetic variants have not been confirmed in independent case-control studies. If commonly accepted quality criteria for candidate gene case-control studies are applied, only a few studies bear a certain degree of scrutiny as many of them are hampered by one or several limitations (see box 1). While genetic variation of the ADH, ALDH and GABA system have consistently been found to be associated with alcohol dependence, this risk is not further conveyed to developing ALD. In fact, only two candidate gene polymorphisms have sufficient data to consider them highly suspicious as genetic risk factors for ALD. These are *TNFα –238A* and *PNPLA3 rs738409 G*, of which only the latter is supported by robust and unequivocal data (figure 3).

---

**REFERENCES**

1. Bellentani S, Saccoccio G, Costa G, et al. Drinking habits as cofactors of risk for alcohol induced liver damage. *Gut* 1997;41:845–50.
2. Becker U, Deis A, Sørensen TI, et al. Prediction of risk of liver disease by alcohol intake, sex, and age: a prospective population study. *Hepatology* 1996;23:1025–9.
3. Fattovich G, Stroffolini T, Zagni I, et al. Hepatocellular carcinoma in cirrhosis: incidence and risk factors. *Gastroenterology* 2004;127 (5 Suppl 1):S35–50.
4. Seitz HK, Stickel F. Molecular mechanisms of alcohol-mediated carcinogenesis. *Nat Rev Cancer* 2007;7:599–612.
5. Wang Y, Millonig G, Nair J, et al. Ethanol-induced cytochrome P4502E1 causes carcinogenic etheno-DNA lesions in alcoholic liver disease. *Hepatology* 2009;50:453–61.
6. Wu D, Cederbaum AI. Oxidative stress and alcoholic liver disease. *Semin Liver Dis* 2009;29:141–54.
7. Lu Y, Zhug J, Wang X, et al. Cytochrome P450 2E1 contributes to ethanol-induced fatty liver in mice. *Hepatology* 2008;47:1483–94.
8. Lieber CS. Alcoholic fatty liver: its pathogenesis and mechanism of progression to inflammation and fibrosis. *Alcohol* 2004;34:9–19.
9. Altamirano J, Bataller R. Alcoholic liver disease: pathogenesis and new targets for therapy. *Nat Rev Gastroenterol Hepatol*. Published Online First: 2011. doi:10.1038/nrgastro.2011.134.
10. Stickel F, Herold C, Seitz HK, et al. Alcohol and methyl transfer: implication for alcohol related hepatocarcinogenesis. In: Ali S, Friedman SL, Mann DA, eds. *Liver Disease: Biochemical Mechanisms and New Therapeutic Insights*. Plymouth, Jersey, Enfield: Science Publishers (USA), 2006:45–58.
11. Ji C, Kaplowitz N. Betaine decreases hyperhomocysteinemia, endoplasmic reticulum stress, and liver injury in alcohol-fed mice. *Gastroenterology* 2003;124:1488–99.
12. Garcia-Trevijano ER, Iraburu MJ, Fontana L, et al. Transforming growth factor beta 1 induces the expression of alpha 1(I) procollagen mRNA by a hydrogen peroxide-C/EBPbeta dependent

mechanism in rat hepatic stellate cells. *Hepatology* 1999;29:960–70.

13. Svegliati-Baroni G, Inagaki Y, Rincon-Sanchez AR, *et al.* Early response of alpha2(I) collagen to acetaldehyde in human hepatic stellate cells is TGF-beta independent. *Hepatology* 2005;42:343–52.

14. Arthur MJ, Iredale JP, Mann DA. Tissue inhibitors of metalloproteinases: role in liver fibrosis and alcoholic liver disease. *Alcohol Clin Exp Res* 1999;23:940–3.

15. Naveau S, Cassard-Doulcier AM, Njike-Nakseu M, *et al.* Harmful effect of adipose tissue on liver lesions in patients with alcoholic liver disease. *J Hepatol* 2010;52:895–902.

16. Monto A, Patel K, Bostrom A, *et al.* Risks of a range of alcohol intake on hepatitis C-related fibrosis. *Hepatology* 2004;39:826–34.

17. Parés A, Caballería J, Bruguera M, *et al.* Histological course of alcoholic hepatitis. Influence of abstinence, sex and extent of hepatic damage. *J Hepatol* 1986;2:33–42.

18. Sato N, Lindros KO, Baraona E, *et al.* Sex difference in alcohol-related organ injury. *Alcohol Clin Exp Res* 2001;25:40S–5S.

19. Eagon PK. Alcoholic liver injury: influence of gender and hormones. *World J Gastroenterol* 2010;16:1377–84.

20. Frezza M, di Padova C, Pozzato G, *et al.* High blood alcohol levels in women. The role of decreased gastric alcohol dehydrogenase activity and first-pass metabolism. *N Engl J Med* 1990;322:95–9.

21. Wickramasinghe SN, Corridan B, Izaguirre J, *et al.* Ethnic differences in the biological consequences of alcohol abuse: a comparison between south Asian and European males. *Alcohol Alcohol* 1995;30:675–80.

22. Stinson FS, Grant BF, Dufour MC. The critical dimension of ethnicity in liver cirrhosis mortality statistics. *Alcohol Clin Exp Res* 2001;25:1181–7.

23. Whitfield JB, Zhu G, Heath AC, *et al.* Effects of alcohol consumption on indices of iron stores and of iron stores on alcohol intake markers. *Alcohol Clin Exp Res* 2001;25:1037–45.

24. Grove J, Daly AK, Burt AD, *et al.* Heterozygotes for HFE mutations have no increased risk of advanced alcoholic liver disease. *Gut* 1998;43:262–6.

25. Gleeson D, Evans S, Bradley M, *et al.* HFE genotypes in decompensated alcoholic liver disease: phenotypic expression and comparison with heavy drinking and with normal controls. *Am J Gastroenterol* 2006;101:304–10.

26. Stickel F, Österreicher CH. Genetic factors in alcoholic liver disease. *Alcohol Alcohol* 2006;41:209–24.

27. Bataller R, North KE, Brenner DA. Genetic polymorphisms and the progression of liver fibrosis: a critical appraisal. *Hepatology* 2003;37:493–503.

28. Lazaridis KN, Juran BD. American Gastroenterological Association future trends committee report: the application of genomic and proteomic technologies to digestive disease diagnosis and treatment and their likely impact on gastroenterology clinical practice. *Gastroenterology* 2005;129:1720–52.

29. Anstee QM, Daly AK, Day CP. Genetics of alcoholic and nonalcoholic liver disease. *Semin Liv Dis* 2011;31:128–46.

30. Goldman D, Oroszi G, Ducci F. The genetics of addictions: uncovering the genes. *Nat Rev Genet* 2005;6:521–32.

31. Warner LA, White HR, Johnson V. Alcohol initiation experiences and family history of alcoholism as predictors of problem-drinking trajectories. *J Stud Alcohol Drugs* 2007;68:56–65.

32. Williams JT, Begleiter H, Porjesz B, *et al.* Joint multipoint linkage analysis of multivariate qualitative and quantitative traits. II. Alcoholism and event-related potentials. *Am J Hum Genet* 1999;65:1148–60.

33. Long JC, Knowler WC, Hanson RL, *et al.* Evidence for genetic linkage to alcohol dependence on chromosomes 4 and 11 from an autosome-wide scan in an American Indian population. *Am J Med Genet* 1998;81:216–21.

34. Reich T, Edenberg HJ, Goate A, *et al.* Genome-wide search for genes affecting the risk for alcohol dependence. *Am J Med Genet* 1998;81:207–15.

35. Edenberg HJ, Dick DM, Xuei X, *et al.* Variations in GABRA2, encoding the alpha 2 subunit of the GABA(A) receptor, are associated with alcohol dependence and with brain oscillations. *Am J Hum Genet* 2004;74:705–14.

36. Fehr C, Sander T, Tadic A, *et al.* Confirmation of association of the GABRA2 gene with alcohol dependence by subtype-specific analysis. *Psychiatr Genet* 2006;16:9–17.

37. Crabb DW, Matsumoto M, Chang D, *et al.* Overview of the role of alcohol dehydrogenase and aldehyde dehydrogenase and their

variants in the genesis of alcohol-related pathology. *Proc Nutr Soc* 2004;63:49–63.

38. Zintzaras E, Stefanidis I, Santos M, *et al.* Do alcohol-metabolizing gene polymorphisms increase the risk of alcoholism and alcoholic liver disease? *Hepatology* 2006;43:352–61.

39. Homann N, Stickel F, König IR, *et al.* The alcohol dehydrogenase 1C*1 allele is a genetic marker for alcohol-associated cancer in heavy drinkers. *Int J Cancer* 2006;118:1998–2002.

40. Arteel GE. Oxidants and antioxidants in alcohol-induced liver disease. *Gastroenterology* 2003;124:778–90.

41. Hayes JD, Strange RC. Glutathione S-transferase polymorphisms and their biological consequences. *Pharmacology* 2000;61:154–66.

42. Berhane K, Widersten M, Fangström A, *et al.* Detoxification of propenals and other α,β-unsaturated aldehyde products of radical reactions and lipid peroxidation by human glutathione transferases. *Proc Natl Acad Sci USA* 1994;91:1480–4.

43. Stickel F, Österreicher CH, Ferenci P, *et al.* A functional polymorphism in the glutathione-S-transferase P1 gene determined the progression to cirrhosis in hereditary hemochromatosis but not in chronic hepatitis C. *Arch Intern Med* 2005;165:1835–40.

44. Khan AJ, Husain Q, Choudhuri G, *et al.* Association of polymorphism in alcohol dehydrogenase and interaction with other genetic risk factors with alcoholic liver cirrhosis. *Drug Alcohol Depend* 2010;109:190–7.

45. Wallace DC. Mitochondrial disease in man and mouse. *Science* 1999;283:1482–8.

46. Sutton A, Khoury H, Príp-Buus C, *et al.* The Ala16Val genetic dimorphism modulates the import of human manganese superoxide dismutase into rat liver mitochondria. *Pharmacogenetics* 2003;13:145–57.

47. Degoul F, Sutton A, Mansouri A, *et al.* Homozygosity for alanine in the mitochondrial targeting sequence of superoxide dismutase and risk for severe alcoholic liver disease. *Gastroenterology* 2001;120:1468–74.

48. Stewart SF, Leathart JB, Chen Y, *et al.* Valine-alanine manganese superoxide dismutase polymorphism is not associated with alcohol-induced oxidative stress or liver fibrosis. *Hepatology* 2002;36:1355–60.

49. Evans RM, Barish GD, Wang YX. PPARs and the complex journey to obesity. *Nat Med* 2004;10:355–61.

50. Rey JW, Noetel A, Hardt A, *et al.* Pro12Ala polymorphism of the peroxisome proliferator-activated receptor γ2 in patients with fatty liver diseases. *World J Gastroenterol* 2010;16:5830–7.

51. Hernández-Nazará ZH, Ruiz-Madrigal B, Martínez-López E, *et al.* Association of the epsilon 2 allele of APOE gene to hypertriglyceridemia and to early-onset alcoholic cirrhosis. *Alcohol Clin Exp Res* 2008;32:559–66.

52. Frenzer A, Butler WJ, Norton ID, *et al.* Polymorphism in alcohol-metabolizing enzymes, glutathione S-transferases and apolipoprotein E and susceptibility to alcohol-induced cirrhosis and chronic pancreatitis. *J Gastroenterol Hepatol* 2002;17:177–82.

53. Jun DW, Han JH, Jang EC, *et al.* Polymorphisms of microsomal triglyceride transfer protein gene and phosphatidylethanolamine N-methyltransferase gene in alcoholic and nonalcoholic fatty liver disease in Koreans. *Eur J Gastroenterol Hepatol* 2009;21:667–72.

54. Lucey MR, Mathurin P, Morgan TR. Alcoholic hepatitis. *N Engl J Med* 2009;360:2758–69.

55. Mandrekar P, Szabo G. Signalling pathways in alcohol-induced liver inflammation. *J Hepatol* 2009;50:1258–66.

56. Järveläinen HA, Orpana A, Perola M, *et al.* Promoter polymorphism of the CD14 endotoxin receptor gene as a risk factor for alcoholic liver disease. *Hepatology* 2001;33:1148–53.

57. Campos J, Gonzalez-Quintela A, Quinteiro C, *et al.* The -159C/T polymorphism in the promoter region of the CD14 gene is associated with advanced liver disease and higher serum levels of acute-phase proteins in heavy drinkers. *Alcohol Clin Exp Res* 2005;29:1206–13.

58. Meiler C, Mühlbauer M, Johann M, *et al.* Different effects of a CD14 gene polymorphism on disease outcome in patients with alcoholic liver disease and chronic hepatitis C infection. *World J Gastroenterol* 2005;11:6031–7.

59. Martins A, Cortez-Pinto H, Machado M, *et al.* Are genetic polymorphisms of tumour necrosis factor alpha, interleukin-10, CD14 endotoxin receptor or manganese superoxide dismutase associated with alcoholic liver disease? *Eur J Gastroenterol Hepatol* 2005;17:1099–104.

60. Grove J, Daly AK, Bassendine MF, *et al.* Interleukin 10 promoter region polymorphism and susceptibility to advanced alcoholic liver disease. *Gut* 2000;46:540–5.

61. Ladero JM, Fernandez-Arquero M, Tudela JL, *et al.* Single nucleotide polymorphisms and microsatellite alleles of tumor

Downloaded from http://gut.bmj.com/ on December 15, 2014 - Published by group.bmj.com

**Recent advances in clinical practice**

62. necrosis factor alpha and interleukin-10 genes and the risk of  
    advanced alcoholic liver disease. *Liver* 2002; **22**:245–51.  
    Bathgate AJ, Pravica V, Perrey C, *et al.* Polymorphisms in tumor  
    necrosis factor alpha, interleukin-10 and transforming growth  
    factor β1 genes and end-stage liver disease. *Eur J Gastroenterol*  
    *Hepatol* 2000; **12**:1329–33.

63. Marcos M, Gómez-Munuera M, Pastor I, *et al.* Tumor necrosis  
    factor polymorphisms and alcoholic liver disease: a HuGE review  
    and meta-analysis. *Am J Epidemiol* 2009; **170**:948–56.

64. Takamatsu M, Yamauchi M, Maezawa Y, *et al.* Correlation of  
    a polymorphism in the interleukin-1 receptor antagonist gene with  
    hepatic fibrosis in Japanese alcoholics. *Alcohol Clin Exp Res*  
    1998; **22**:141S–4S.

65. Pastor I, Laso FJ, Avila JJ, *et al.* Polymorphism in the interleukin-1  
    receptor antagonist gene is associated with alcoholism in Spanish  
    men. *Alcohol Clin Exp Res* 2000; **24**:1479–82.

66. Takamatsu M, Yamauchi M, Maezawa Y, *et al.* Genetic  
    polymorphisms of interleukin-1beta in association with the  
    development of alcoholic liver disease in Japanese patients. *Am J*  
    *Gastroenterol* 2000; **95**:1305–11.

67. Valenti L, De Feo T, Fracanzani AL, *et al.* Cytotoxic T-lymphocyte  
    antigen-4 A49G polymorphism is associated with susceptibility to  
    and severity of alcoholic liver disease in Italian patients. *Alcohol*  
    *Alcohol* 2004; **39**:276–80.

68. Stickel F, Österreicher CH, Halangk J, *et al.* No role of  
    matrixmetalloproteinase-3 genetic promoter polymorphism as  
    a risk factor for cirrhosis in alcoholic liver disease. *Alcohol Clin Exp*  
    *Res* 2008; **32**:959–65.

69. Österreicher CH, Halangk J, Berg T, *et al.* Evaluation of the  
    transforming growth factor beta-1 codon 25 (Arg→Pro)  
    polymorphism in alcoholic liver disease. *Cytokine* 2008; **42**:18–23.

70. Sookoian S, Pirola CJ. Meta-analysis of the influence of I148M  
    variant of patatin-like phospholipase domain containing 3 gene  
    (PNPLA3) on the susceptibility and histological severity of  
    nonalcoholic fatty liver disease. *Hepatology* 2011; **53**:1883–94.

71. Tian C, Stokowski RP, Kershenobich D, *et al.* Variant in PNPLA3 is  
    associated with alcoholic liver disease. *Nat Genet* 2010; **42**:21–3.

72. Seth D, Daly AK, Haber PS, *et al.* Patatin-like phospholipase  
    domain containing 3: a case in point linking genetic susceptibility  
    for alcoholic and nonalcoholic liver disease. *Hepatology*  
    2010; **51**:1463–5.

73. Trépo E, Gustot T, Degré D, *et al.* Common polymorphism in the  
    PNPLA3/adiponutrin gene confers higher risk of cirrhosis and liver  
    damage in alcoholic liver disease. *J Hepatol* 2011; **55**:906–12.

74. Stickel F, Buch S, Lau K, *et al.* Genetic variation in the PNPLA3  
    gene is associated with alcoholic liver injury in caucasians.  
    *Hepatology* 2011; **53**:86–95.

75. Zimmermann R, Strauss JG, Haemmerle G, *et al.* Fat mobilization  
    in adipose tissue is promoted by adipose triglyceride lipase.  
    *Science* 2004; **306**:1383–6.

76. Huang Y, He S, Li JZ, *et al.* A feed-forward loop amplifies  
    nutritional regulations of PNPLA3. *Proc Natl Acad Sci USA*  
    2010; **107**:7892–7.

77. Lake AC, Sun Y, Li JL, *et al.* Expression, regulation, and  
    triglyceride hydrolase activity of Adiponutrin family members. *J*  
    *Lipid Res* 2005; **46**:2477–87.

78. Hoekstra M, Li Z, Kruijt JK, *et al.* The expression level of non-  
    alcoholic fatty liver disease-related gene PNPLA3 in hepatocytes is  
    highly influenced by hepatic lipid status. *J Hepatol*  
    2010; **52**:244–51.

79. Baulande S, Lasnier F, Lucas M, *et al.* Adiponutrin,  
    a transmembrane protein corresponding to a novel dietary- and  
    obesity-linked mRNA specifically expressed in the adipose lineage.  
    *J Biol Chem* 2001; **276**:33336–44.

80. He S, McPhaul C, Li JZ, *et al.* A sequence variation (I148M) in  
    PNPIA3 associated with nonalcoholic fatty liver disease disrupts  
    triglyceride hydrolysis. *J Biol Chem* 2009; **285**:6706–15.

81. Basantani MK, Sitnick MT, Cai L, *et al.* Pnpla3/adiponutrin  
    deficiency in mice does not contribute to fatty liver disease or  
    metabolic syndrome. *J Lipid Res* 2011; **52**:318–29.

82. Chen W, Chang B, Li L, *et al.* Patatin-like phospholipase domain-  
    containing 3/adiponutrin deficiency in mice is not associated with  
    fatty liver disease. *Hepatology* 2010; **52**:1134–42.

83. Trépo E, Pradat P, Potthoff A, *et al.* Impact of patatin-like  
    phospholipase-3 (rs738409 C>G) polymorphism on fibrosis  
    progression and steatosis in chronic hepatitis C. *Hepatology*  
    2011; **54**:60–9.

84. Müller T, Buch S, Berg T, *et al.* Distinct, alcohol-modulated  
    effects of PNPLA3 genotype on progression of chronic hepatitis C.  
    *J Hepatol* 2011; **55**:1471–2.

85. Zimmer V, Lammert F. Genetics in liver disease: new concepts.  
    *Curr Opin Gastroenterol* 2011; **27**:231–9.

Downloaded from http://gut.bmj.com/ on December 15, 2014 - Published by group.bmj.com

**Genetic determinants of alcoholic liver disease**

Felix Stickel and Jochen Hampe

*Gut* 2012 61: 150-159 originally published online November 22, 2011  
doi: 10.1136/gutjnl-2011-301239

---

Updated information and services can be found at:  
http://gut.bmj.com/content/61/1/150

These include:

- **References**: This article cites 83 articles, 14 of which you can access for free at:  
  http://gut.bmj.com/content/61/1/150#BIBL

- **Email alerting service**: Receive free email alerts when new articles cite this article. Sign up in the box at the top right corner of the online article.

---

**Topic Collections**

Articles on similar topics can be found in the following collections

- Gut Education (56)
- GUT Recent advances in clinical practice (72)
- Hepatic cancer (436)
- Alcoholic liver disease (122)
- Cirrhosis (313)
- Nonalcoholic steatosis (114)
- Pancreas and biliary tract (1900)

---

**Notes**

To request permissions go to:  
http://group.bmj.com/group/rights-licensing/permissions

To order reprints go to:  
http://journals.bmj.com/cgi/reprintform

To subscribe to BMJ go to:  
http://group.bmj.com/subscribe/
