
Antimicrobial Agents and Chemotherapy

**GraXSR Proteins Interact with the VraFG ABC Transporter To Form a Five-Component System Required for Cationic Antimicrobial Peptide Sensing and Resistance in Staphylococcus aureus**

Mélanie Falord, Gouzel Karimova, Aurélia Hiron and Tarek Msadek  
*Antimicrob. Agents Chemother.* 2012, 56(2):1047. DOI: 10.1128/AAC.05054-11.  
Published Ahead of Print 28 November 2011.

---

Updated information and services can be found at:  
<http://aac.asm.org/content/56/2/1047>

These include:

SUPPLEMENTAL MATERIAL  
[Supplemental material](http://aac.asm.org/content/suppl/2011/11/28/AAC.05054-11.DC1.html)

REFERENCES  
This article cites 45 articles, 17 of which can be accessed free at: <http://aac.asm.org/content/56/2/1047#ref-list-1>

CONTENT ALERTS  
Receive: RSS Feeds, eTOCs, free email alerts (when new articles cite this article), [more »](http://aac.asm.org/content/56/2/1047)

---

Information about commercial reprint orders: <http://journals.asm.org/site/misc/reprints.xhtml>  
To subscribe to to another ASM Journal go to: <http://journals.asm.org/site/subscriptions/>

Journals.ASM.org

GraXSR Proteins Interact with the VraFG ABC Transporter To Form a Five-Component System Required for Cationic Antimicrobial Peptide Sensing and Resistance in Staphylococcus aureus

Mélanie Falord${}^{a,b,c}$ Gouzel Karimova${}^{d,e}$ Aurélia Hiron${}^{a,b}$ and Tarek Msadek${}^{a,b}$

Institut Pasteur, Biology of Gram-Positive Pathogens, Department of Microbiology, Paris, France${}^{a}$; CNRS, URA 2172, Paris, France${}^{b}$; Université Paris Diderot, Sorbonne Paris Cité, Biology of Gram-Positive Pathogens, Paris, France${}^{c}$; Institut Pasteur, Biochemistry of Macromolecular Interactions, Department of Structural Biology, Paris, France${}^{d}$; and CNRS, URA 2185, Paris, France${}^{e}$

The GraSR two-component system (TCS) controls cationic antimicrobial peptide (CAMP) resistance in *Staphylococcus aureus* through the synthesis of enzymes that increase bacterial cell surface positive charges, by D-alanylation of teichoic acids and lysylation of phosphatidylglycerol, leading to electrostatic repulsion of CAMPs. The GraS histidine kinase belongs to the “intramembrane-sensing kinases” subfamily, with a structure featuring a short amino-terminal sensing domain, and two transmembrane helices separated only by a short loop, thought to be buried in the cytoplasmic membrane. The GraSR TCS is in fact a multicomponent system, requiring at least one accessory protein, GraX, in order to function, which, as we show here, acts by signaling through the GraS kinase. The graXRS genes are located immediately upstream from genes encoding an ABC transporter, vraFG, whose expression is controlled by GraSR. We demonstrated that the VraFG transporter does not act as a detoxification module, as it cannot confer resistance when produced on its own, but instead plays an essential role by sensing the presence of CAMPs and signaling through GraS to activate GraR-dependent transcription. A bacterial two-hybrid approach, designed to identify interactions between the GraXSR and VraFG proteins, was carried out in order to understand how they act in detecting and signaling the presence of CAMPs. We identified many interactions between these protein pairs, notably between the GraS kinase and both GraX and the VraG permease, indicating the existence of an original five-component system involved in CAMP sensing and signal transduction to promote S. aureus resistance.

The major Gram-positive human pathogen *Staphylococcus aureus* causes a wide variety of infections and displays a unique adaptive potential which has made it one of the major causes of nosocomial infections today, emphasized by the emergence of antibiotic-resistant strains (35, 36). Its ubiquitous nature stems mostly from the ability to survive a large variety of environmental conditions, due in part to sensitive signal transduction pathways that allow the bacterium to monitor and rapidly respond to environmental signals. These mechanisms often involve two large families of highly conserved proteins: ABC transporters, composed of a membrane spanning domain protein (MSD) and a nucleotide binding domain protein (NBD) (18), and two-component systems (TCSs), coupled pairs of sensor histidine kinases (HK), generally membrane bound, and their cognate response regulator proteins (RR), usually acting as transcription activators (21).

Regulatory relationships between these two groups of proteins were described in Gram-positive bacteria, by demonstrating two-component system-dependent expression of colocalized ABC transporter genes (13, 19, 24), and genome analysis demonstrates a predominance of these TCS/ABC transporter modules among the Bacilli and Clostridia classes of the Firmicutes phylum (8, 11, 24). In these modules, the ABC transporter MSD protein has 10 transmembrane domains (TMDs) with an unusually long extracytoplasmic loop (180 to 300 amino acids) and the TCS belongs to the so-called “intramembrane-sensing kinase” (IMSK) subfamily of histidine kinases, conserved in low G+C Gram-positive bacteria, with a short amino-terminal sensing domain, composed of two transmembrane helices separated by a short loop of only a few amino acids (23, 37). Since they lack a typical HK extracellular sensing domain, it has been suggested that signal detection must involve a process acting at or from within the membrane interface (37), in response to antimicrobial compounds, leading to increased synthesis of the ABC transporter encoded by the neighboring genes. The ABC transporter is then thought to facilitate removal of the harmful drug and the initial stimulus of the system, which then shuts down (23, 37).

Most S. aureus strains are endowed with 16 sets of genes encoding two-component systems, with an additional one present in the methicillin resistance staphylococcal cassette chromosome mec of methicillin-resistant S. aureus (MRSA) strains (32). Only two of the S. aureus TCSs belonging to the IMSK subfamily form TCS/ABC transporter modules: BraSR/BraDE, which we have recently shown to be involved in bacitracin and nisin resistance (bacitracin resistance associated) (19), and GraSR/VraFG.

The GraSR (glycopeptide resistance associated) TCS controls cationic antimicrobial peptide (CAMP) resistance in S. aureus through D-alanylation of teichoic acids, mediated by the DltABCD enzymes, as well as MprF-dependent lysylation of phosphatidylglycerol, increasing the bacterial surface positive charge and preventing electrostatic attraction of positively charged CAMPs (23, 37). Since they lack a typical HK extracellular sensing domain, it has been suggested that signal detection must involve a process acting at or from within the membrane interface (37), in response to antimicrobial compounds, leading to increased synthesis of the ABC transporter encoded by the neighboring genes. The ABC transporter is then thought to facilitate removal of the harmful drug and the initial stimulus of the system, which then shuts down (23, 37).

---

Received 16 August 2011 Returned for modification 9 September 2011  
Accepted 21 November 2011  
Published ahead of print 28 November 2011  
Address correspondence to Tarek Msadek, tmsadek@pasteur.fr.  
Supplemental material for this article may be found at [http://aac.asm.org](http://aac.asm.org).  
Copyright © 2012, American Society for Microbiology. All Rights Reserved.  
doi:[10.1128/AAC.05054-11](https://doi.org/10.1128/AAC.05054-11)

venting CAMP binding by electrostatic repulsion (16, 33). CAMPs such as defensins are a major component of host innate immune defense systems, produced by all living organisms and killing bacteria through membrane permeation (15, 22) or by acting internally on DNA, RNA, or protein synthesis (14, 15). Expression of the *S. aureus dlt* operon and *mprF* genes is induced by the presence of CAMPs and specifically controlled by the GraSR TCS (9, 16, 33, 34, 39). GraSR control expression of the *vraFG* operon as well, located directly downstream of the *graXRS* genes and encoding an ABC transporter also playing a role in CAMP resistance (13, 16, 33, 34, 39). Additionally, the first gene of the *graRS* operon encodes GraX, a protein of unknown function that also plays a role in CAMP resistance (16, 33, 39), which we have previously shown to be required for GraSR-dependent gene expression, suggesting that it acts as an essential cofactor for the system (13).

We have recently investigated the GraSR regulon of *S. aureus*, defining the GraR binding site and uncovering new links to signal transduction pathways controlling stress response, virulence gene expression, and cell wall metabolism (13). An important question remains of CAMP signal detection by the GraS kinase. Indeed, as an IMSK, it is much shorter than most TCS histidine kinases, lacking any classical signal sensing input domains. A previous report suggested that the short loop separating the two transmembrane domains of GraS could be essential for signal sensing by directly binding CAMPs (33). However, we have shown for the closely related BraSR system that the BraE ABC transporter permease is the true sensor of the system, conferring to the BraSIMSK the ability to sense nisin or bacitracin (19). BraSR activate transcription of operons encoding two ABC transporters that play distinct and original roles in antibiotic resistance: the BraDE transporter is only involved in bacitracin sensing and signaling through BraSR, whereas the VraDE transporter acts specifically as a detoxification module and is sufficient to confer bacitracin and nisin resistance when produced on its own (19).

In this study, we sought to determine the roles of the additional partners GraX and the VraFG ABC transporter in conferring CAMP resistance and their relationship to the GraSR TCS. We demonstrated that GraX acts through the GraS HK and that the VraFG ABC transporter is involved in CAMP resistance by sensing and signaling through GraSR. Using the bacterial two-hybrid approach, we were able to show that GraS interacts with both GraX and the VraG permease. These results highlight the existence of an original five-component signaling complex involved in CAMP sensing and signal transduction to promote *S. aureus* resistance to these antimicrobial compounds.

### MATERIALS AND METHODS

**Bacterial strains and growth conditions.** *Escherichia coli* K-12 strain DH5α (Invitrogen Life Technologies) was used for cloning experiments. Plasmid constructs were first passed through the restriction modification-deficient *S. aureus* strain RN4220 (31) before introduction into *S. aureus* strain HG001, an *rsbU⁺* variant of strain NCTC 8325 (17) and its derivatives. HG001 is a genetically tractable, clinically relevant nonmutagenized *S. aureus* strain and was used for all genetic studies. Bacterial strains and plasmids are listed in Table 1. *E. coli* strains were grown in LB medium with ampicillin (100 μg/ml⁻¹) when required. *S. aureus* was grown in Trypticase soy broth (TSB; Difco) with shaking (180 rpm) at 37°C. For plasmid selection, chloramphenicol (10 μg/ml⁻¹) or erythromycin (2 μg/ml⁻¹) were added as required. *E. coli* and *S. aureus* strains were transformed by electroporation using standard protocols (44) and transformants were selected on LB or Trypticase soy agar (TSA; Difco)

plates, respectively, with the appropriate antibiotics. Colistin sulfate was used as a CAMP (Sigma-Aldrich) to induce the GraSR system in *S. aureus* strains when required.

**MIC determinations.** MIC determinations were performed in a 96-well microtiter plate (100-μl culture volume). Bacterial cultures were grown in TSB, diluted to an optical density at 600 nm (OD₆₀₀) of 0.05 and used to inoculate wells containing TSB with standard 2-fold increments of colistin concentration (vol/vol). Plates were incubated for 12 h with vigorous shaking at 37°C in a Synergy 2 thermoregulated spectrophotometer plate reader using the Gen5 microplate software (BioTek Instruments Inc., Winooski, VT). All experiments were performed at least in triplicate.

**DNA manipulations.** Oligonucleotides used in this study were synthesized by Sigma-Proligo, and their sequences are listed in Table 2. *S. aureus* HG001 chromosomal DNA was isolated using the MasterPure Gram-positive DNA purification kit (Epicentre Biotechnologies). Plasmids were isolated using a QIAprep spin miniprep kit (Qiagen), and PCR fragments were purified using the QIAquick PCR purification kit (Qiagen). T4 DNA ligase and restriction enzymes (New England Biolabs), PCR reagents, and high-fidelity Pwo thermostable DNA polymerase (Roche) were used according to the manufacturers' recommendations. Nucleotide sequencing of plasmid constructs was carried out by Genome Express-Cogenics or GATC Biotech.

**Plasmids and mutant construction.** Oligonucleotides used for PCR amplifications are listed in Table 2. The thermosensitive shuttle vector pMAD was used for introducing markerless gene deletions (3). Mutant strains of *S. aureus* HG001 used in this study were obtained by gene deletions, removing the entire coding sequence without the introduction of an antibiotic resistance gene. In a first step, two DNA fragments, of approximately 600 bp, corresponding to the chromosomal DNA regions located directly upstream and downstream from the gene(s) of interest, were generated by PCR, digested with XhoI or BsaI, and ligated using T4 DNA ligase. The ligation product was reamplified using the external primers and purified before cloning into the temperature-sensitive shuttle vector pMAD between the EcoRI/Ncol or SalI/BglII restriction sites. Nucleotide sequences of the constructs were confirmed by DNA sequencing, and the resulting plasmids were introduced into *S. aureus* HG001. Integration and excision of the pMAD derivatives leading to deletion of the chromosomal region of interest was carried out as previously described (3). Gene deletions in mutant strains were verified by PCR.

Plasmid pMK4Pprot, a derivative of vector pMK4 carrying a constitutively expressed Gram-positive promoter sequence (2), was used for gene complementation experiments using PCR-generated DNA fragments corresponding to the gene coding sequences. Oligonucleotide pairs are listed in Table 2.

Plasmid pSA14 (13) was used to measure the expression of *S. aureus* genes by constructing transcriptional fusions between gene promoter regions and the *E. coli lacZ* reporter gene. Promoter regions of the *vraFG* operon or *mprF* gene were amplified by PCR using oligonucleotides introducing BamHI/PstI restriction sites (see Table 2). The corresponding DNA fragments were then cloned between the corresponding restriction sites of the pSA14 vector, yielding plasmids listed in Table 1.

To construct the chimeric VraG MSD protein with the extracellular loop of VraE (VraG*^VraE^*), the predicted membrane topologies of VraG and VraE were determined using the TopPredII program and von Heijne's algorithm (7). The sequence of the *vraE* extracytoplasmic loop was amplified by PCR using oligonucleotides carrying BsaI restriction sites (19). Two PCR fragments corresponding to the DNA regions flanking the *vraG* extracellular loop, including the *vraF* gene, were generated and ligated to the *vraE* extracellular loop sequence, seamlessly fusing the three fragments together without adding any additional nucleotides. The resulting fragment was reamplified by PCR using the external oligonucleotides and cloned in the pMK4-Pprot plasmid, yielding plasmid pMK4-Pprot *vraFG*^*vraE*^ (19).

Site-directed mutagenesis by strand overlap extension PCR (SOE-PCR) was used to construct the *graRD51E* mutant (ST1172 strain). Two

TABLE 1 Bacterial strains and plasmids used in this study

| Strain or plasmid | Relevant genotype or description | Source, reference, or construction$^a$ |
|-------------------|---------------------------------|----------------------------------------|
| **Strains**       |                                 |                                        |
| **E. coli**       |                                 |                                        |
| DH5$\alpha$       | F$^{-}$ $\phi$80dlacZ$\Delta$M15 $\Delta$(lacZYA-argF) U169 recA1 endA1 hsdR17(rk$^{-}$, mk$^{+}$) phoA supE44 $\lambda^{-}$ thi-1 gyrA96 relA1 | Invitrogen Life Technologies           |
| EC1123            | lacZ$_{1-12}$::phoA$_{61-1413}$::lacZ$_{13-180}$, Km$^{r}$ | pKTOP$\rightarrow$DH5$\alpha$         |
| EC1124            | graX::phoA$_{61-1413}$::lacZ$_{13-180}$, Km$^{r}$        | pKTop-GraX$\rightarrow$DH5$\alpha$     |
| EC1130            | graX$_{1-368}$::phoA$_{61-1413}$::lacZ$_{13-180}$, Km$^{r}$ | pKTop-GraX$_{1-122}$$\rightarrow$DH5$\alpha$ |
| EC1131            | graX$_{1-669}$::phoA$_{61-1413}$::lacZ$_{13-180}$, Km$^{r}$ | pKTop-GraX$_{1-223}$$\rightarrow$DH5$\alpha$ |
| EC1132            | graS$_{1-117}$::phoA$_{61-1413}$::lacZ$_{13-180}$, Km$^{r}$ | pKTop-GraS$_{1-39}$$\rightarrow$DH5$\alpha$ |
| EC1133            | graS::phoA$_{61-1413}$::lacZ$_{13-180}$, Km$^{r}$        | pKTop-GraS$\rightarrow$DH5$\alpha$     |
| XL1-Blue          | F$^{'}$::Tn10 proA$^{+}$B$^{+}$ lacI$^{q}$ $\Delta$(lacZ) M15 glnV44(AS) endA1 gyrA96 recA1 thi-1 hsdR17 lac | Stratagene                            |
| DHT1              | F$^{-}$ glnV44(AS) recA1 endA1 gyrA96 (Nal$^{r}$) thi-1 hsdR17 spoT1 rfbD1 cya-854 ilv-691::Tn10 | 10                                     |
| **S. aureus**     |                                 |                                        |
| RN4220            | Restriction deficient transformation recipient | 31                                    |
| HG001             | NCTC 8325 rsbU$^{+}$           | 17                                    |
| ST1036            | $\Delta$graRS                  | 13                                    |
| ST1039            | vraF$^{'}$-lacZ, Cm$^{r}$      | 13                                    |
| ST1041            | $\Delta$graRS vraF$^{'}$-lacZ, Cm$^{r}$ | 13                                    |
| ST1052            | $\Delta$graX vraF$^{'}$-lacZ, Cm$^{r}$ | 13                                    |
| ST1068            | $\Delta$vraFG::spc, Sp$^{r}$   | 19                                    |
| ST1070            | $\Delta$graX                   | 13                                    |
| ST1074            | mprF$^{'}$-lacZ, Cm$^{r}$      | 13                                    |
| ST1075            | $\Delta$graRS mprF$^{'}$-lacZ, Cm$^{r}$ | 13                                    |
| ST1100            | $\Delta$graS                   | pMADgraS$\rightarrow$HG001           |
| ST1101            | $\Delta$graXS                  | pMADgraS$\rightarrow$ST1070         |
| ST1102            | $\Delta$vraFG vraF$^{'}$-lacZ, Cm$^{r}$ | pSA14vraF$\rightarrow$ST1068        |
| ST1103            | $\Delta$vraFG mprF$^{'}$-lacZ, Cm$^{r}$ | pSA14mprF$\rightarrow$ST1068        |
| ST1116            | $\Delta$graRS pMK4-PprotgraR, Cm$^{r}$ | 13                                    |
| ST1117            | $\Delta$graRS pMK4-Pprot, Cm$^{r}$ | 13                                    |
| ST1118            | $\Delta$graS vraF$^{'}$-lacZ, Cm$^{r}$ | pSA14vraF$\rightarrow$ST1100        |
| ST1119            | $\Delta$graXS vraF$^{'}$-lacZ, Cm$^{r}$ | pSA14vraF$\rightarrow$ST1101        |
| ST1120            | pMK4-Pprot, Cm$^{r}$           | 13                                    |
| ST1150            | $\Delta$vraFG pMK4-Pprot, Cm$^{r}$ | pMK4-Pprot$\rightarrow$ST1068       |
| ST1151            | $\Delta$vraFG pMK4-PprotvraFG, Cm$^{r}$ | pMK4-PprotvraFG$\rightarrow$ST1068  |
| ST1152            | $\Delta$graRS pMK4-PprotvraFG, Cm$^{r}$ | pMK4-PprotvraFG$\rightarrow$ST1036  |
| ST1153            | $\Delta$vraFG pMK4-PprotgraR, Cm$^{r}$, Spc$^{r}$ | pMK4-PprotgraR$\rightarrow$ST1068    |
| ST1154            | $\Delta$graX pMK4-Pprot, Cm$^{r}$ | pMK4-Pprot$\rightarrow$ST1070       |
| ST1170            | $\Delta$graS pMK4-Pprot, Cm$^{r}$ | pMK4-Pprot$\rightarrow$ST1100       |
| ST1171            | $\Delta$graS pMK4-PprotgraR, Cm$^{r}$ | pMK4-PprotgraR$\rightarrow$ST1100   |
| ST1172            | graRD51E                       | pMADgraRD51E$\rightarrow$HG001       |
| ST1173            | graRD51E pMK4-Pprot, Cm$^{r}$  | pMK4-Pprot$\rightarrow$ST1172       |
| ST1174            | graRD51E pMK4-PprotgraR, Cm$^{r}$ | pMK4-PprotgraR$\rightarrow$ST1172   |
| ST1175            | graRD51E vraF$^{'}$-lacZ, Cm$^{r}$ | pSA14vraF$\rightarrow$ST1172        |
| ST1177            | $\Delta$vraFG pMK4-PprotvraFG$^{vraE}$, Cm$^{r}$, Spc$^{r}$ | pMK4-Pprot vraFG$^{vraE}$$\rightarrow$ST1068 |
| ST1207            | $\Delta$graX pMK4-PprotgraR, Cm$^{r}$ | pMK4-PprotgraR$\rightarrow$ST1070   |
| **Plasmids**      |                                 |                                        |
| pMAD              | pE194 derivative with a thermosensitive origin of replication for deletion/replacement of genes in Gram-positive bacteria | 3                                     |
| pMADgraS          | pMAD derivative allowing deletion of the graS gene | This study                           |
| pMADgraRD51E      | pMAD derivative allowing the introduction of the D51E mutation in the graR gene | This study                           |
| pSA14             | pMK4 derivative carrying promoterless E. coli lacZ for constructing transcriptional fusions | 13                                    |
| pSA14vraF         | pSA14 derivative carrying the intergenic region between graXRS and vraFG | 13                                    |
| pSA14mprF         | pSA14 derivative carrying the mprF promoter region | 13                                    |
| pMK4Pprot         | pMK4 derivative carrying a constitutive Gram-positive promoter for gene complementation | 2                                     |
| pMK4-PprotgraR    | pMK4-Pprot derivative carrying graR | 13                                    |
| pMK4-PprotvraFG   | pMK4-Pprot derivative carrying vraFG | 19                                    |

Continued on following page

Falord et al.

TABLE 1 (continued)

| Strain or plasmid | Relevant genotype or description | Source, reference, or construction$^a$ |
|-------------------|---------------------------------|---------------------------------------|
| pMK4-PprotvraFG$^{*vraE}$ | pMK4-Pprot derivative carrying the vraFG$^{*vraE}$ genes | 19 |
| pKT25 | BACTH vector designed to express a given polypeptide fused in frame at its N-terminal end with T25 fragment; p15ori | 28 |
| pKNT25 | BACTH vector designed to express a given polypeptide fused in frame at its C-terminal end with T25 fragment; p15ori | 25, 28 |
| pUT18C | BACTH vector designed to express a given polypeptide fused in frame at its N-terminal end with T18 fragment; ColEl ori | 28 |
| pUT18 | BACTH vector designed to express a given polypeptide fused in frame at its N-terminal end with T18 fragment; ColEl ori | 28 |
| pKT25-graX | Full-length graX ORF cloned into pKT25 | This study |
| pKT25-graR | Full-length graR ORF cloned into pKT25 | This study |
| pKT25-graS | Full-length graS ORF cloned into pKT25 | This study |
| pKT25-vraF | Full-length vraF ORF cloned into pKT25 | This study |
| pKT25-vraG | Full-length vraG ORF cloned into pKT25 | This study |
| pKNT25-graX | Full-length graX ORF cloned into pKNT25 | This study |
| pKNT25-graS | Full-length graS ORF cloned into pKNT25 | This study |
| pKNT25-vraF | Full-length vraF ORF cloned into pKNT25 | This study |
| pKNT25-vraG | Full-length vraG ORF cloned into pKNT25 | This study |
| pUT18C-graX | Full-length graX ORF cloned into pUT18C | This study |
| pUT18C-graR | Full-length graR ORF cloned into pUT18C | This study |
| pUT18C-graS | Full-length graS ORF cloned into pUT18C | This study |
| pUT18C-vraF | Full-length vraF ORF cloned into pUT18C | This study |
| pUT18C-vraG | Full-length vraG ORF cloned into pUT18C | This study |
| pUT18-graX | Full-length graX ORF cloned into pUT18 | This study |
| pUT18-graS | Full-length graS ORF cloned into pUT18 | This study |
| pUT18-vraF | Full-length vraF ORF cloned into pUT18 | This study |
| pUT18-vraG | Full-length vraG ORF cloned into pUT18 | This study |
| pKTOP | Vector expressing dual reporter, PhoA$_{21-471}$/LacZ$_{5-60}$, p15 ori | 27 |
| pKTop-GraX | pKTop derivative expressing tripartite GraX/PhoA$_{21-471}$/LacZ$_{5-60}$ | This study |
| pKTop-GraX$_{1-122}$ | pKTop derivative expressing tripartite GraX$_{1-122}$/PhoA$_{21-471}$/LacZ$_{5-60}$ | This study |
| pKTop-GraX$_{1-223}$ | pKTop derivative expressing tripartite GraX$_{1-223}$/PhoA$_{21-471}$/LacZ$_{5-60}$ | This study |
| pKTop-GraS | pKTop derivative expressing tripartite GraS$_{1-346}$/PhoA$_{21-471}$/LacZ$_{5-60}$ | This study |
| pKTop-GraS$_{1-39}$ | pKTop derivative expressing tripartite GraS$_{1-39}$/PhoA$_{21-471}$/LacZ$_{5-60}$ | This study |

$^a$ Arrows indicate plasmid introduction by electroporation.

DNA fragments with overlapping 3′ and 5′ ends were generated by PCR with oligonucleotide pairs MF163/MF164 and MF165/MF166. Oligonucleotides MF164 and MF165 are complementary and carry a single nucleotide mismatch within the native graR coding sequence, changing codon 51 from an aspartate codon (GAT) to a glutamate codon (GAA) through a T → A transversion. The purified DNA fragments were mixed in equal amounts and used as a DNA matrix for SOE-PCRs with oligonucleotides MF163 and MF166. The full-length DNA fragment was then cloned between the SalI and BglII sites of the pMAD vector resulting in pMADGraRD51E, which was transformed into S. aureus strain HG001, generating strain ST1172, following integration and excision. As a result of the point mutation changing the aspartate 51 codon to glutamate, a FokI restriction site was destroyed. The absence of this restriction site within the mutated graR coding sequence was used to identify positive clones by PCR amplification of an internal DNA fragment encompassing the mutated codon using oligonucleotide pair MF167/MF168 followed by restriction with BtsCI, a neoschizomer of FokI, and the DNA fragment corresponding to the entire mutagenized chromosomal graR coding region was sequenced in order to verify the presence of the intended single base pair change.

To construct the recombinant plasmids used in the bacterial two-hybrid (BACTH) complementation assays, genes coding for the S. aureus proteins under study were PCR amplified using appropriate primers (Table 2) and genomic DNA from HG001 as a template. Briefly, the corresponding DNA fragments were amplified by PCR with the appropriate oligonucleotide pairs (see Table 2), digested with BamHI and EcoRI (for graS, graR, vraG, and vraF) or BamHI and BsaI (for graX), and cloned between the BamHI and EcoRI sites of the pKT25 and pUT18C vectors (29). The resulting recombinant plasmids expressed hybrid proteins, in which the polypeptides of interest were fused to the C terminus of the T25 or T18 fragment of adenylate cyclase (AC), respectively. For the reciprocal combinations, PCR-generated fragments (without the stop codon, see Table 2 for oligonucleotide pairs) were digested with HindIII and BamHI (for graX, graR, and graS) or BsaI and BamHI (for vraG and vraF) and cloned into the HindIII and BamHI sites of the pKNT25 (27) and pUT18 vectors (29). This second set of recombinant plasmids encoded hybrid proteins in which the polypeptides of interest were fused to the N terminus of the T25 or T18 fragment of AC, respectively.

β-Galactosidase assays. Cells were grown until an OD$_{600}$ of 1, colistin was added to the medium at 50 μg/ml$^{-1}$ or 200 μg/ml$^{-1}$ when required, and growth was pursued for 30 min. S. aureus strains carrying the different lacZ fusions were then harvested by centrifuging 2 ml culture samples (2 min, 20,800 × g). Cells were resuspended in 500 μl of Z buffer (40) with 0.5 mg/ml$^{-1}$ DNase, 5 mM dithiothreitol (DTT), and 0.1 mg/ml$^{-1}$ lyso-staphin added extemporaneously and lysed by incubation at 37°C for 30 min. Cell debris was eliminated by centrifugation (2 min, 20,800 × g), and the supernatant was either used directly for assays or stored at −20°C. Assays were performed as previously described and β-galactosidase-specific activities were expressed as Miller units/mg$^{-1}$ protein (40). Protein concentrations were determined using the Bio-Rad protein assay (Bio-Rad, Hercules, CA) (5). All experiments were carried out in triplicate.

TABLE 2 Oligonucleotides used in this study$^a$

| Name   | Sequence                                                                                   | Description                                                                 |
|--------|-------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|
| MF11   | 5′-CTCCTCGAGTTGAACGCATGTCGGAAGT-3′                                                        | graS downstream region, deletion mutant                                      |
| MF12   | 5′-AGAAGATCTGCA CCTGTGGTTCGTCAGC-3′                                                       | graS upstream region, deletion mutant                                        |
| MF116  | 5′-GTCGTCGACAATGGGATT TTAATGTTGCTGGT-3′                                                   | graR coding sequence, complementation                                        |
| MF117  | 5′-CTCCTCGAGTTATTCATGAGCCATATATCCT-3′                                                     | graX coding sequence, T18-GraX and T25-GraX fusions                          |
| MF118  | 5′-GGAGGATCCGGAGGTGATATGGATGCAAATAC-3′                                                    | graR coding sequence, T18-GraR and T25-GraR fusions                          |
| MF119  | 5′-CTGCTGCAGTTATTCATGAGCCATATATCCT-3′                                                     | graX coding sequence, T18-GraX and T25-GraX fusions                          |
| MF140  | 5′-GGAGGATCCCAAACCTAAAGTTTAT TAGCAGG-3′                                                   | graR coding sequence, T18-GraR and T25-GraR fusions                          |
| MF141  | 5′-GGTGGTCTCGAATTCTCATTTAGTATATTCATATTTTC-3′                                              | graS coding sequence, T18-GraS and T25-GraS fusions                          |
| MF142  | 5′-GGAGGATCCCCAAATACTACTAGTAGAAGATGAC-3′                                                 | vraF coding sequence, T18-VraF and T25-VraF fusions                          |
| MF143  | 5′-GAAGAATTCTTATTCATGAGCCATATATCCT-3′                                                     | vraG coding sequence, T18-VraG and T25-VraG fusions                          |
| MF144  | 5′-GGAGGATCCCAATAATTGAAATGGGTAGCTTA-3′                                                    | graX coding sequence, GraX-T18 and GraX-T25 fusions                          |
| MF145  | 5′-GAAGAATTCTTAAAATGACAAATTGTCACTTCCGA-3′                                                 | graS coding sequence, GraS-T18 and GraS-T25 fusions                          |
| MF146  | 5′-GGAGGATCCC GCAATTTTAGAAGTAAAAACA-3′                                                    | vraF coding sequence, T18-VraF and T25-VraF fusions                          |
| MF147  | 5′-GAAGAATTCTTAAAGGTCATAATTAACGCCAC-3′                                                    | vraG coding sequence, T18-VraG and T25-VraG fusions                          |
| MF148  | 5′-GGAGGATCCCACCTTAAACGAGATAATATTTAAAAATTC-3′                                             | graX coding sequence, GraX-T18 and GraX-T25 fusions                          |
| MF149  | 5′-GAAGAATTCTTATATGGAAATGTCTAATTG-3′                                                      | graS coding sequence, GraS-T18 and GraS-T25 fusions                          |
| MF152  | 5′-AAGAAGCTTTAAAGGAGTATTTTAGATGAAACC-3′                                                   | graX coding sequence, GraX-T18 and GraX-T25 fusions; and GraX fusion with     |
|        |                                                                                           | PhoA<sub>22-472</sub>/LacZ<sub>4-60</sub>                                     |
| MF153  | 5′-GGAGGATCCTCTTTAGTATATTTCTCATATTTTCTCCTGAT-3′                                           | graS coding sequence, GraS-T18 and GraS-T25 fusions; and GraS fusion with     |
|        |                                                                                           | PhoA<sub>22-472</sub>/LacZ<sub>4-60</sub>                                     |
| MF156  | 5′-AAGAAGCTTG TAGGAAAAGGATATATGGCTCATGAA-3′                                               | vraF coding sequence, VraF-T18 and VraF-T25 fusions                          |
| MF157  | 5′-GGAGGATCCTCAAATGACAAAATTGTCAC TTCCGACATGCGTT-3′                                        | vraG coding sequence, VraG-T18 and VraG-T25 fusions                          |
| MF158  | 5′-GGTGGTCTCAAGCTTATAAATTATAGGAGT GTTAAAGTG-3′                                            | graR upstream and coding region, graRD51E mutation                            |
| MF159  | 5′-GGAGGATCCTCAAGGTCATAATTAACGCCACCTAAAACAC-3′                                            | graR downstream and coding region, graRD51E mutation                          |
| MF163  | 5′-GTCGTCGACGTTCTTTAGTTAATGTAGAATTACAAG-3′                                                | graR coding sequence internal fragment                                       |
| MF164  | 5′-CCATCATATTTAGGTAATTGAACTTCCAATATAACAATTCAGGAT TAAAA-3′                                 | vraG coding sequence, VraG-T18 and VraG-T25 fusions                          |
| MF165  | 5′-TAATCCTGAAATTGTTATATTGGAAGTTC AATTACCTAAATATGATGGGTT-3′                                | graX<sub>1-368</sub> coding region, phoA<sub>61-::lacZ</sub><sub>13-1413</sub> fusion |
| MF166  | 5′-AGAAGATCTGTCTAT TGGAAAATCATAAATCG-3′                                                  | graS<sub>1-117</sub> coding region, phoA<sub>61-::lacZ</sub><sub>13-1413</sub> fusion |
| MF167  | 5′-GCAAATACTACTAGTAGAAGATGAC-3′                                                           | graX<sub>1-669</sub> coding region, phoA<sub>61-::lacZ</sub><sub>13-1413</sub> fusion |
| MF168  | 5′-ACTCATAGACACGACGATAAATCGC-3′                                                           |                                                                 |
| MF173  | 5′-GGTGGTCTCAAGCTTATAATACGTGTACAAAGTGT TTTAGGTGGCGT-3′                                    |                                                                 |
| MF174  | 5′-GGAGGATCCTATGGAATGTCTAATTGTT CGCTTGGAAAT-3′                                            |                                                                 |
| MF218  | 5′-GGAGGATCCTCAATTGTTTCATTATCATAACGACTCCCA-3′                                             |                                                                 |
| MF223  | 5′-GGAGGATCCTCTGGAAAATCAT AATCGATTAGACTAATG-3′                                            |                                                                 |
| MF224  | 5′-GGAGGATCCTCAGTTATTAT TCCATCAGCTGTTTC-3′                                               |                                                                 |

$^a$ Added restriction sites are indicated in italics.

Protein localization analysis. To examine GraX and GraS localization, we used plasmid pKTop that encodes a dual *pho-lac* reporter and allows determination of protein localization *in vivo* (1). For GraX, three pKTop derivatives were constructed in which the *pho-lac* reporter was fused in frame after the following *graX* codons: G129 (pKTop-GraX<sub>1-129</sub>), I223 (pKTop-GraX<sub>1-223</sub>), and K307 (pKTop-GraX<sub>1-307</sub>), the full-length protein. The corresponding *graX* fragments were PCR amplified from HG001 genomic DNA by using appropriate oligonucleotide pairs (MF152/MF218, MF152/MF2224, and MF152/MF153, respectively) (Table 2) and cloned between the HindIII and BamHI sites of pKTop. To determine the membrane topology of GraS, two PCR-generated *graS* DNA fragments encoding a full-length protein, GraS<sub>1-346</sub> (MF156/ MF157), and a truncated variant, GraS<sub>1-39</sub> (MF156/MF223), were digested with HindIII and BamHI and cloned into pKTop (Tables 1 and 2). For membrane protein topology assays *in vivo*, the resulting plasmids were transformed into *E. coli* DH5α and plated on dual-indicator LB agar plates containing 6-chloro-3-indoly-β-D-galactopyranoside (Red-Gal; Sigma) at 100 μg/ml<sup>-1</sup>, 5-bromo-4-chloro-3-indoly phosphate disodium salt (X-Phos; Sigma) at 80 μg/ml<sup>-1</sup>, isopropyl-β-D-thiogalactopyranoside (IPTG; 1 mM), 50 mM phosphate buffer (pH 7.0), and 50 μg/ml<sup>-1</sup> kanamycin.

BACTH complementation assay. For BACTH complementation assays, recombinant vectors (pKT25, pKNT25, pUT18C, and pUT18) carrying the *S. aureus* genes under study were cotransformed in all combinations into *E. coli* DHT1 cells (10). Transformants were plated on LB/X-Gal

(5-bromo-4-chloro-3-indoly-β-D-galactopyranoside) plus IPTG and incubated at 30°C for 24 to 36 h. Interaction efficiencies between different hybrid proteins were then quantified by measuring β-galactosidase activity in liquid cultures in a 96-well microtiter plate assay. Bacteria were grown in 300 μl LB broth in the presence of 0.5 mM IPTG and appropriate antibiotics in 2.2-ml 96-well storage plates (Thermo Scientific). Each well of the array was inoculated with an isolated colony; for each set of transformations, eight independent colonies were examined. Cells were grown at 30°C for 14 to 16 h. Before the assays, cultures were diluted 1:5 into M63 medium (44), 175 μl from each well was transferred to a flat bottom microtiter plate and an OD<sub>595</sub> was recorded using a GENios reader (Tecan). To permeabilize cells, 7 μl of SDS (0.05%) and 10 μl of chloroform were added to 200 μl of bacterial suspensions, which were transferred to a 1.2-ml polypropylene 96-well storage block (Thermo Scientific). Solutions were vigorously intermixed using an 8-channel multichannel pipette and incubated at room temperature for 30 to 40 min to allow chloroform evaporation. For the enzymatic reaction, aliquots (20 μl) of the permeabilized cells were added to a new microtiter plate, each well containing 105 μl of the following reaction mixture: PM2 buffer (70 mM Na<sub>2</sub>HPO<sub>4</sub> 12H<sub>2</sub>O, 30 mM NaHPO<sub>4</sub>H<sub>2</sub>O, 1 mM MgSO<sub>4</sub>, and 0.2 mM MnSO<sub>4</sub> [pH 7.0] plus 100 mM β-mercaptoethanol) and 0.1% o-nitrophenol-β-galactoside (ONPG). The plate was incubated at room temperature for 20 to 30 min. A total of 50 μl of 1M Na<sub>2</sub>CO<sub>3</sub> per well was added to stop the reaction. The OD<sub>405</sub> was then recorded using a GENios reader (Tecan). Enzymatic activities, *A* (in relative units), were calculated

Falord et al.

TABLE 3 CAMP resistance of different S. aureus mutants$^a$

| Strain       | Relevant genotype                          | Colistin MIC ($\mu$g/ml$^{-1}$) |
|--------------|-------------------------------------------|----------------------------------|
| ST1120       | HG001/pMK4-Pprot                          | 700                              |
| ST1117       | $\Delta$graRS/pMK4-Pprot                  | 100                              |
| ST1150       | $\Delta$vraFG/pMK4-Pprot                  | 100                              |
| ST1151       | $\Delta$vraFG/pMK4-Pprot-vraFG            | 700                              |
| ST1116       | $\Delta$graRS/pMK4-Pprot-graR             | 700                              |
| ST1152       | $\Delta$graRS/pMK4-Pprot-vraFG            | 100                              |
| ST1177       | $\Delta$vraFG/pMK4-Pprot-vraFG$^{vraE}$   | 200                              |
| HG001        |                                           | 700                              |
| ST1036       | $\Delta$graRS                             | 100                              |
| ST1070       | $\Delta$graX                             | 300                              |
| ST1100       | $\Delta$graS                             | 100                              |
| ST1101       | $\Delta$graXS                            | 100                              |
| ST1170       | $\Delta$graS/pMK4-Pprot                   | 100                              |
| ST1171       | $\Delta$graS/pMK4-Pprot-graR              | 700                              |
| ST1154       | $\Delta$graX/pMK4-Pprot                   | 300                              |
| ST1207       | $\Delta$graX/pMK4-Pprot-graR              | 700                              |
| ST1153       | $\Delta$vraFG/pMK4-Pprot-graR             | 700                              |
| ST1173       | graRD51E/pMK4-Pprot                       | 100                              |
| ST1174       | graRD51E/pMK4-Pprot-graR                  | 700                              |

$^a$ Strains were grown at 37°C in TSB with decreasing colistin concentrations. Growth was followed by measuring absorbance at 600 nm using a microtiter plate reader, and MICs were determined as the colistin concentration that inhibits growth after 12 h. Each experiment was repeated at least three times.

toxification module and that it requires the GraSR TCS to confer CAMP resistance (Table 3, strain ST1152).

To study the role of VraFG in CAMP signaling through the GraSR TCS, we constructed transcriptional lacZ fusions with the vraFG operon and mprF gene promoters, using the pSA14 vector (vraF'-lacZ and mprF'-lacZ, respectively). The fusions were introduced into strains HG001, ST1036 ($\Delta$graRS), and ST1068 ($\Delta$vraFG), and $\beta$-galactosidase activities were measured during mid-exponential growth at 37°C in TSB, with or without a 30-min treatment with 50 $\mu$g/ml$^{-1}$ colistin. As shown in Fig. 1, the VraFG ABC transporter is essential for CAMP-inducible transcription from the vraFG promoter, as well as for inducible expression of the mprF'-lacZ fusion (see Fig. S1 in the supplemental material). Expression of both transcriptional fusions in the $\Delta$vraFG mutant was lowered to the same basal levels as in the $\Delta$graRS mutant, showing that the VraFG ABC transporter is as essential as the GraSR TCS in controlling CAMP-inducible expression of these genes. As shown previously (13) and in Fig. S1, the mprF gene has a high basal level of expression in the absence of GraSR or VraFG, such that regulation of its expression is not as strong as for vraFG.

Taken together, our data suggest that the VraFG ABC transporter acts to signal CAMPs through the GraSR TCS, in agreement with our hypothesis that the GraS IMSK requires accessory partners for signal detection and transduction.

The extracellular loop of the VraG permease is required for CAMP resistance. Another ABC transporter, VraDE, closely related to VraFG, plays a central role in S. aureus bacitracin resistance (19, 42, 45). The predicted membrane topologies and primary structures of these two ABC transporters are very similar (10 TMDs for the MSD proteins; 62% identity between VraD and VraF and 39% between VraE and VraG). The most dissimilar sequence between the proteins corresponds to the large extracytoplasmic loop of 193 (VraE) and 198 (VraG) amino acids in length, located between transmembrane regions 7 and 8. Amino acid sequence identity between these two loops is only 20%, whereas the rest of the two proteins share 53% identity (19). VraDE plays no role in CAMP resistance, since a $\Delta$vraDE mutant was as resistant to colistin as the parental HG001 strain (19). To

as follows: $A = 1,000 \times (\text{OD}_{405} - \text{OD}_{405} \text{ in control well}) / (\text{OD}_{595} - \text{OD}_{595} \text{ in control well}) / t$ (min) of incubation. $\beta$-Galactosidase activity levels at least 3- to 4-fold higher than those measured in DHT1 (pKT25, pKNT25, pUT18C, pUT18) control cells were considered to indicate a positive interaction. All assays were performed eight times.

**RESULTS**

GraSR/VraFG constitute a TCS/ABC transporter module involved in CAMP resistance in Staphylococcus aureus. The VraFG ABC transporter, encoded by the genes immediately downstream from the graXRS operon, has been shown to be required for CAMP resistance in S. aureus although its specific role is not understood (16, 33, 34, 39). In order to determine its contribution to resistance, MIC values for colistin, a bacterial CAMP which we have shown to act as a good inducer of the GraSR system, were determined for strains HG001, ST1036 ($\Delta$graRS), and ST1150 ($\Delta$vraFG), by following growth in TSB at 37°C over a 12-h period, using a BioTek Synergy microplate reader and serial dilutions of the CAMP. As shown in Table 3, the $\Delta$vraFG mutant was as sensitive as the $\Delta$graRS mutant. Full complementation of both mutants could be achieved by constitutively expressing either the vraFG genes in the $\Delta$vraFG mutant or the graR gene in the $\Delta$graRS mutant, using the pMK4-Pprot vector, where gene expression is driven by the constitutive Pprot promoter (2) (Table 3) (see Materials and Methods).

Complementation of the $\Delta$graRS mutant could be achieved by expressing the graR gene alone, as overexpression of response regulator genes is well-known to bypass the requirement for the cognate histidine kinase, through phosphorylation by other phosphate donors such as acetyl phosphate or aspecific kinase activity within the cell (30). However, when the vraFG genes were overexpressed in the $\Delta$graRS mutant, the strain remained as sensitive to colistin as either the $\Delta$vraFG or $\Delta$graRS mutants, indicating that the VraFG ABC transporter cannot function on its own as a de-

FIG 1 VraFG is required for its own synthesis and colistin induction. The vraF'-lacZ fusion was introduced into S. aureus strains HG001, ST1036 ($\Delta$graRS), and ST1068 ($\Delta$vraFG). vraF'-lacZ expression was measured during mid-exponential growth at 37°C in TSB (gray bars) or after treatment with 50 $\mu$g/ml$^{-1}$ colistin (black bars). $\beta$-Galactosidase assays were performed as described in Materials and Methods. Means and standard deviation values are presented from three independent experiments.

investigate the role of the VraG extracellular loop in S. aureus, we therefore took advantage of the close similarities between VraE and VraG in order to construct a chimeric ABC transporter by domain swapping, exchanging the extracellular loop of VraG for that of VraE, and expressing the chimeric gene with the cognate vraFNBD gene using the pMK4-Pprot plasmid (19) (see Materials and Methods). The resulting plasmid, pMK4-Pprot vraFG*VraE, was introduced into the ST1068 (∆vraFG) strain.

Sensitivity to colistin was examined for strains ST1120 (HG001/pMK4-Pprot), ST1150 (∆vraFG/pMK4-Pprot), and ST1177 (∆vraFG/pMK4-PprotvraFG*VraE) by determining MICs. As shown in Table 3, whereas the ST1120 control strain displayed a high level of resistance to colistin, sensitivity of the ∆vraFG mutant could not be complemented by producing the chimeric VraG*VraE MSD protein and the VraF NBD protein. We have shown in a separate study that overproduction of the VraFG*VraE proteins can fully complement bacitracin sensitivity of a ∆vraDE mutant (19), indicating that the chimeric VraG*VraE protein is properly folded and localized to form a functional ABC transporter with VraF. Taken together, these results strongly suggest that the extracellular loop of the VraG permease plays an essential role in CAMP sensing.

GraX acts as a cofactor of GraSR by signaling through GraS. GraX, encoded by the first gene of the graXRS operon, bears weak similarities to sugar epimerases, but although it is important for CAMP resistance, acting as part of the GraXSR three-component system to control CAMP resistance associated genes (13), its precise role remains unknown. In order to investigate the role of GraX in the GraSR signal transduction pathway, we constructed mutants lacking either the graX or graS genes alone, or both the graX and graS genes, and compared colistin sensitivity of strains HG001, ST1036 (∆graRS), ST1070 (∆graX), ST1100 (∆graS), and ST1101 (∆graXS). As shown by the MIC values presented in Table 3, sensitivity of the ∆graX mutant is intermediate, in between that of the parental HG001 strain and the ∆graRS mutant, whereas deletion of the graS gene alone confers the same level of CAMP sensitivity as the ∆graRS mutation. However, sensitivity of the ∆graXS mutant is greater than that of the ∆graX mutant, showing that the ∆graS mutation is epistatic to the ∆graX mutation (Table 3, strains ST1070 and ST1101). As the effects of the two mutations are not additive, this suggests that GraX acts through the GraS histidine kinase to transduce the CAMP signal to the GraR response regulator, leading to increased CAMP resistance gene expression.

In order to confirm these results, we examined vraF'-lacZ expression in strain HG001 and the ∆graRS, ∆graX, ∆graS, and ∆graXS mutants during mid-exponential growth in TSB at 37°C after treatment with 50 µg/ml⁻¹ colistin. As shown in Fig. 2, expression of vraF'-lacZ was approximately twice as high in the ∆graX mutant than in the ∆graRS and ∆graS strains, and this residual expression was abolished in the ∆graXS mutant, in agreement with the proposed epistatic relationship between graS and graX.

Overproduction of GraR bypasses the requirement of GraS, GraX, and VraFG for colistin resistance. As shown above, GraX and the VraFG ABC transporter act in the GraSR signal transduction pathway to control CAMP resistance. In order to further determine where they act in this process, we introduced the pMK4-Pprot-graR plasmid overproducing the GraR response regulator into the HG001, ∆graRS, ∆graS, ∆graX, and ∆vraFG strains and

FIG 2 GraX acts through GraS to induce vraFG expression. The vraF'-lacZ fusion was introduced into S. aureus strains HG001, ST1036 (∆graRS), ST1070 (∆graX), ST1100 (∆graS), and ST1101 (∆graXS). vraF'-lacZ expression was measured during mid-exponential growth in TSB at 37°C after treatment with 50 µg/ml⁻¹ colistin. β-Galactosidase assays were performed as described in Materials and Methods. Means and standard deviation values are presented from three independent experiments.

compared their sensitivity to colistin. As shown by the MIC values presented in Table 3, overproduction of the GraR response regulator fully compensates the absence of GraS, GraX, and VraFG, restoring the resistance level of the parental HG001 strain. These results indicate that both GraX and the VraFG ABC transporter likely act through the GraS HK to achieve GraR phosphorylation and activation in response to the presence of CAMPs and that their main function is in signaling through the GraSR TCS.

Aspartate residue 51 of GraR is essential for its activity. Response regulators are phosphorylated on a highly conserved aspartate residue within the receiver domain (46, 48). Sequence alignments indicate that in GraR, this residue is aspartate 51 (D51). In order to investigate the role of this residue in GraR activity, a single markerless point mutation was introduced into the S. aureus chromosomal graR gene, a T→A transversion at position 153 of the graR coding sequence, changing the GAT aspartate 51 codon to a GAA glutamate codon. This was carried out by site-directed mutagenesis through SOE-PCR (20), using the pMAD plasmid (see Materials and Methods). The mutation leading to the D51E replacement in the receiver domain of GraR led to complete inactivation of the response regulator. Indeed, as shown in Table 3, the graR(D51E) mutant was as sensitive as the ∆graRS mutant and could be fully complemented by introduction of the pMK4-Pprot-graR plasmid overproducing the native GraR response regulator (Table 3, strain ST1174).

In order to confirm these results, we examined vraF'-lacZ expression in strain HG001 and the graR(D51E) mutant. As shown in Fig. 3, expression and colistin-dependent induction of vraF'-lacZ were abolished in the graR(D51E) mutant, indicating that this mutation does not act as a phosphomimetic in GraR, presumably due to local amino acid sequence variations in the active site, and that this residue is essential for GraR activation through phosphorylation, in agreement with the results shown above. Taken together, these results indicate that the D51 residue of GraR is essential for its activity.

GraX is a cytosolic protein. Our results indicate that GraX might interact with the GraS HK. As GraS is an intramembrane-sensing kinase, we wished to determine in which cellular compartment-

Falord et al.

![Figure 3](#fig3)

FIG 3 The GraR aspartate 51 residue is essential for its activity. Expression of *vraF'–lacZ* fusion was measured in the S. aureus HG001 and *graRD51E* strains (ST1039 and ST1172, respectively) during mid-exponential growth in TSB at 37°C (gray bars) or after treatment with 50 μg/ml⁻¹ colistin (black bars). β-Galactosidase assays were performed as described in Materials and Methods. Means and standard deviation values are presented from three independent experiments.

![Figure 4](#fig4)

FIG 4 GraX and GraS protein topology analysis. DH5α cells expressing different Pho-Lac fusions were plated on indicator medium with two chromogenic substrates, Red-Gal (for β-galactosidase activity) and X-Pho (for phosphatase activity). Blue coloration of the colonies (high phosphatase activity) indicates a membrane or extracellular localization of the fusion point. Red coloration of the colonies (high β-galactosidase activity) indicates cytosolic location of the fusion point. DH5α pKTop and Ctrl-PhoA+ (K32) are positive controls for β-galactosidase and phosphatase activity, respectively. Fusion endpoints are indicated for each hybrid protein.

ment this interaction might take place. We first examined the predicted membrane topology of the GraX and GraS proteins, using the TopPredII program and Engelman, Steitz, and Goldman's algorithm (7, 12). For GraX, a weakly hydrophobic putative transmembrane segment was predicted (residues 216 to 236), with the C terminus suggested to be extracellular. The predicted membrane topology of the GraS histidine kinase was as expected, with two transmembrane domains (residues 15 to 35 and 43 to 63) separated by a short seven amino acid loop (YDFPIDS) and both termini located in the cytosol.

Topological localization of both proteins was examined using a dual *pho-lac* reporter system, which consists of the *E. coli* alkaline phosphatase fragment *PhoA*₂₁₋₄₇₁, fused in-frame with the α-peptide of *E. coli* β-galactosidase, *LacZ*₅₋₆₀ (1). In this approach, an extracellular or membrane localization of the reporter leads to high alkaline phosphatase activity and low β-galactosidase activity, whereas a cytosolic location of the reporter results in high β-galactosidase activity and low alkaline phosphatase activity. In-frame insertions of the *pho-lac* reporter after three selected *graX* codons (I122, I223, K307) and two *graS* codons (P39, F346) were created using plasmid pKTop to study their localization (see Materials and Methods). Plasmids expressing the different *graX'-pholac* or *graS'-pholac* fusions were then transformed into *E. coli* strain DH5α, and the transformants were analyzed on dual-indicator LB medium plates containing both a blue chromogenic substrate for phosphatase activity (X-Phos) and a red chromogenic substrate for β-galactosidase activity (Red-Gal) (see Materials and Methods) with the appropriate control fusions for cytoplasmic and extracellular localization (Fig. 4, pKTOP and pKTOPK32, respectively).

As shown in Fig. 4, examination of the *graX*-recombinant constructs revealed that the *pho-lac* reporter fused after any of the selected residues present upstream, in or downstream from the predicted potential transmembrane domain (I122, I223, or K307, the full-length protein) resulted in hybrid proteins exhibiting high β-galactosidase activity (red phenotype), indicating a cytosolic localization of the corresponding GraX residues and confirming the *in silico* analysis predicting GraX to be a cytosolic protein. Cells expressing the full-length *graS* fusion (*graS*₁₋₃₄₆-*pholac*) exhibited a red phenotype (i.e., Lac⁺), indicating a cytosolic localization of the Pho-Lac reporter and, consequently, of the last residue, F346 of GraS (Fig. 4). Fusion of the Pho-Lac reporter after residue P39 of GraS, which lies within the short predicted extracellular loop, yielded deep-blue colonies on the dual-indicator medium, consistent with an intramembrane or extracytoplasmic localization of residue P39 of GraS (Fig. 4).

BACTH analysis of GraXSR/VraFG interactions. In order to understand the potential protein interaction network involved in the multicomponent GraXSR/VraFG signal transduction pathway, we applied the BACTH approach. In this system, based on the interaction-mediated reconstruction of a cyclic AMP-signaling (cAMP) cascade in *E. coli*, proteins of interest are produced as translational fusions with two fragments (T25 and T18) of the catalytic domain of *Bordetella pertussis* AC in an *E. coli cya* AC-deficient strain. Upon interaction of the hybrid proteins, the proximity of the T25 and T18 fragments restores AC activity, leading to cAMP synthesis, and in turn to transcriptional activation of catabolic operons (such as the lactose operon or the maltose regulon). As the BACTH system interactions result in the generation of a small diffusible molecule, cAMP, the protein interaction under study does not need to occur near the transcription machinery as in the case of yeast or other bacterial two-hybrid systems (28), making it particularly appropriate for studying *in vivo* interactions between membrane-localized proteins.

To characterize physical associations, the five proteins (GraX, GraR, GraS, VraG, and VraF) were systematically tested for pairwise interactions using the BACTH complementation assay. For this purpose, DNA fragments encoding the selected proteins were

1054 aac.asm.org
Antimicrobial Agents and Chemotherapy

TABLE 4 BACTH analysis of GraX, GraR, GraS, VraF, and VraG interactions$^a$

| Hybrid protein | T18-GraS | GraS-T18 | T18-GraX | GraX-T18 | T18-GraR | T18-VraG | VraG-T18 | T18-VraF | VraF-T18 |
|----------------|----------|----------|----------|----------|----------|----------|----------|----------|----------|
| T25-GraS       | 57       | 187      | 9        | 59       | 27       | 15       | 10       | 11       | 12       |
| GraS-T25       | 34       | 100      | 12       | 111      | 25       | 11       | 10       | 11       |          |
| T25-GraX       | 11       | 7        | 68       | 90       | 10       | 10       | 9        | 5        |          |
| GraX-T25       | 21       | 174      | 82       | 35       | 112      | 11       | 126      | 123      |          |
| T25-GraR       | 14       | 88       | 9        | 50       | 14       | 9        | 8        | 7        | 51       |
| T25-VraG       | 11       | 65       | 10       | 18       | 9        | 10       | 130      | 65       | 475      |
| VraG-T25       | 8        | 6        | 7        | 7        | 6        | 7        | 6        | 7        | 6        |
| T25-VraF       | 8        | 6        | 9        | 52       | 9        | 10       | 17       | 9        | 208      |
| VraF-T25       | 4        | 6        | 84       | 77       | 63       | 74       | 6        | 27       | 23       |

$^a$ Functional complementation between the indicated hybrid proteins was quantified by measuring $\beta$-galactosidase activities in *E. coli* DHT1 cells harboring the corresponding plasmids, as described in Materials and Methods. Hybrid proteins in which the S. aureus polypeptides are fused to T25 (T18) via their N terminus are marked as T25-X (T18-X). Hybrid proteins in which the S. aureus polypeptides are fused to T25 (T18) via their C terminus are marked as X-T25 (X-T18). Each result represents the mean value of eight independent measurements. In all cases, standard deviations were within 20% of the mean. Zip-Zip associations were used as positive controls (26). Under the same assay conditions, functional complementation between T25-Zip and T18-Zip yielded about 400 units of $\beta$-galactosidase activity. T25 and T18 associations were used as negative controls. Cells expressing these nonfused fragments gave about 10 units of $\beta$-galactosidase activity.

Falord et al.

FIG 5 The five-component signal transduction network controlling CAMP sensing and resistance in S. aureus. CAMPs are sensed by VraFG and the signal is transduced to GraS through a mechanism that likely involves interaction between VraG and GraS, probably through the extracellular loop of the VraG permease, which we have shown to be required for its specificity and activity in CAMP sensing. Activation of the GraSR system leads to increased transcription of the *dlt* operon and the *mprF* gene, leading to CAMP resistance. Double-headed arrows indicate protein-protein interactions, green arrows indicate activation of transcription, and purple arrows indicate protein synthesis and phenotypes.

small group of TCSs associated with an ABC transporter (VraFG) encoded by genes genetically linked to those of the TCS (8, 24, 43).

In *B. subtilis*, the BceAB ABC transporter was shown to be essential for bacitracin sensing by the BceSR TCS, highlighting a new mechanism for TCS-mediated signal transduction, where the ABC transporter MSD protein is thought to act as the bona fide bacitracin sensor for the BceS intramembrane-sensing kinase (4, 8, 43). We have recently shown that the BraSR/BraDE TCS/ABC transporter module in *S. aureus* functions in much the same manner, with the BraDE ABC transporter involved in sensing bacitracin and acting through the BraSR TCS to control synthesis of a second ABC transporter, VraDE, specifically involved in conferring bacitracin resistance and likely acting as a detoxification module (19).

As we show here, the GraSR TCS requires the accessory regulatory protein GraX as well as the VraFG ABC transporter in order to function, effectively forming a multicomponent system. Several systems with auxiliary regulators of histidine kinase activity have recently come to light (6), as well as many examples of TCSs where the membrane-bound histidine kinase interacts with transport proteins such as ABC transporters (47). Our results show that the GraSR system combines both of these original aspects. Indeed, in this report we examined the roles of the GraX and VraFG additional partners in conferring CAMP resistance through this complex pathway and their functional relationship with the GraSR TCS. We provide evidence indicating that GraX acts through the GraS HK and that the VraFG transporter plays an essential role in CAMP resistance by sensing and signaling their presence through GraSR. The VraFG transporter thus positively controls its own synthesis through GraSR, conceivably leading to a positive feedback loop in the presence of CAMPs. While the manuscript was under review, Yang et al. independently reported the observation

that *mprF* expression is lowered in a *vraG* mutant, in agreement with our results (49).

Complementation experiments with constitutively expressed *vraFG* genes allowed us to show that VraFG cannot confer CAMP resistance when produced on its own, indicating it is not involved in the detoxification process, since a Δ*graRS* mutant constitutively expressing *vraFG* remained highly sensitive to CAMPs. Interestingly, it is now apparent that the two *S. aureus* TCS/ABC transporter modules differ significantly in their mechanisms from those of *B. subtilis* in that sensing and resistance are systematically uncoupled in both cases: for the Bra system, sensing involves the BraDE ABC transporter, but resistance only requires the VraDE ABC transporter (19); for the Gra system, as shown here, sensing involves the VraFG transporter, but resistance involves the *dlt* and *mprF* genes. In each case, the associated ABC transporter is only involved in sensing and not resistance *per se*, unlike the Bce system of *B. subtilis* where the ABC transporter has both functions (4, 8, 43).

We used a dual Pho-Lac translational fusion approach to show that GraX is a cytosolic protein and that the GraS HK has two transmembrane segments and a short intramembrane loop with both termini exposed to the cytosol. A subsequent bacterial two-hybrid approach revealed in particular that GraS interacts with both GraX and the VraG permease and that GraX also interacts with VraF and GraR. These results suggest that GraX plays a central role in the regulatory process, indicating the existence of a multicomponent membrane-linked signaling complex involved in CAMP sensing and signal transduction. Our results allowed us to propose a model for CAMP signaling and resistance through the GraSR pathway in *S. aureus* (Fig. 5). CAMPs would first be sensed by the VraFG ABC transporter, as we have shown that a *vraFG* mutant strain no longer responds to their presence. This is

fairly unusual since typically in TCSs the signal is sensed by the external loop of the histidine kinase (21). However, this is in agreement with the fact that GraS belongs to the intramembrane- sensing histidine kinase family, thought to detect stimuli from within or at the membrane interface (37, 38). We suggest that the stimulus is sensed either by the VraFG ABC transporter and then transferred to GraS, which in turn activates GraR, or through CAMP interaction with both VraFG and the extracellular loop of GraS (Fig. 5). This would then lead to increased expression of the *dlt* operon and *mprF*, with subsequent D-alanylation of WTAs and LTAs and lysinylation of membrane phosphatidylglycerol, increasing cell surface positive charges and conferring resistance through electrostatic repulsion of CAMPs (Fig. 5).

Several important questions remain to be answered, such as how the VraFG ABC transporter senses CAMPs. We have demonstrated that the extracellular loop of VraFG is important for CAMP recognition by constructing a chimeric ABC transporter by domain swapping, exchanging the extracellular loop of VraG for that of the closely related ABC transporter permease VraE, involved in bacitracin but not in colistin detoxification (19). Our results suggest that in VraG, this extracellular loop is essential for signal recognition and could be involved in direct interaction with CAMPs and/or GraS, whose short extracellular loop could also play a role in this process. Indeed, the GraS loop has been suggested to be involved in conferring ligand recognition specificity by directly binding to distinct CAMPs (33) and may also be involved in signaling to the GraS histidine kinase.

It will be important to determine whether the VraG and GraS loops are both involved and act together in CAMP detection and how signaling to the GraS HK effectively occurs, for example, using site-directed mutagenesis and different chimeric extracellular loops. One possibility is that CAMP binding to VraG and/or GraS causes a conformational change promoting GraS-dependent activation of GraR through phosphorylation. Further work will be required to unravel the mechanism of detection and activation of the original five-component GraXSR/VraFG module required for *S. aureus* resistance to CAMPs to confirm these hypotheses, as well as the specific role of the GraX accessory protein in this signal transduction pathway.

### ACKNOWLEDGMENTS

This work was supported by research funds from the European Commission (StaphDynamics [LHSM-CT-2006-019064] and BaSysBio [LSHG- CT-2006-037469] grants), the Centre National de la Recherche Scientifique (CNRS URA 2172 and URA 2185), Agence Nationale de la Recherche (ANR NaBab and GrabIron), and the Institut Pasteur (PTR no. 256 and PTR no. 336).

We are grateful to Daniel Ladant in whose laboratory part of this work was carried out. We thank Cécile Wandersman for critical reading of the manuscript.

Mélanie Falord received a Young Scientist Fellowship from the Conseil Pasteur-Weizmann.

### REFERENCES

1. Alexeyev MF, Winkler HH. 1999. Membrane topology of the *Rickettsia prowazekii* ATP/ADP translocase revealed by novel dual *pho-lac* reporters. J. Mol. Biol. 285:1503–1513.
2. Archambaud C, Gouin E, Pizarro-Cerda J, Cossart P, Dussurget O. 2005. Translation elongation factor EF-Tu is a target for Stp, a serine-threonine phosphatase involved in virulence of *Listeria monocytogenes*. Mol. Microbiol. 56:383–396.
3. Arnaud M, Chastanet A, Debarbouille M. 2004. New vector for efficient allelic replacement in naturally nontransformable, low-GC-content, Gram-positive bacteria. Appl. Environ. Microbiol. 70:6887–6891.
4. Bernard R, Guiseppi A, Chippaux M, Foglino M, Denizot F. 2007. Resistance to bacitracin in *Bacillus subtilis*: unexpected requirement of the BceAB ABC transporter in the control of expression of its own structural genes. J. Bacteriol. 189:8636–8642.
5. Bradford M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. 72:248–254.
6. Buelow DR, Raivio TL. 2010. Three (and more) component regulatory systems—auxiliary regulators of bacterial histidine kinases. Mol. Microbiol. 75:547–566.
7. Claros MG, von Heijne G. 1994. TopPred II: an improved software for membrane protein structure predictions. Comput. Appl. Biosci. 10: 685–686.
8. Coumes-Florens S, Brochier-Armanet C, Guiseppi A, Denizot F, Foglino M. 2011. A new highly conserved antibiotic sensing/resistance pathway in firmicutes involves an ABC transporter interplaying with a signal transduction system. PLoS One 6:e15951.
9. Cui L, Lian JQ, Neoh HM, Reyes E, Hiramatsu K. 2005. DNA microarray-based identification of genes associated with glycopeptide resistance in *Staphylococcus aureus*. Antimicrob. Agents Chemother. 49: 3404–3413.
10. Dautin N, Karimova G, Ullmann A, Ladant D. 2000. Sensitive genetic screen for protease activity based on a cyclic AMP signaling cascade in *Escherichia coli*. J. Bacteriol. 182:7060–7066.
11. Dintner S, et al. 2011. Coevolution of ABC transporters and two-component regulatory systems as resistance modules against antimicrobial peptides in Firmicutes bacteria. J. Bacteriol. 193:3851–3862.
12. Engelman DM, Steitz TA, Goldman A. 1986. Identifying nonpolar transbilayer helices in amino acid sequences of membrane proteins. Annu. Rev. Biophys. Chem. 15:321–353.
13. Falord M, Mader U, Hiron A, Debarbouille M, Msadek T. 2011. Investigation of the *Staphylococcus aureus* GraSR regulon reveals novel links to virulence, stress response and cell wall signal transduction pathways. PLoS One 6:e21323.
14. Hale JD, Hancock RE. 2007. Alternative mechanisms of action of cationic antimicrobial peptides on bacteria. Expert Rev. Anti Infect. Ther. 5:951–959.
15. Hancock RE, Sahl HG. 2006. Antimicrobial and host-defense peptides as new anti-infective therapeutic strategies. Nat. Biotechnol. 24:1551–1557.
16. Herbert S, et al. 2007. Molecular basis of resistance to muramidase and cationic antimicrobial peptide activity of lysozyme in staphylococci. PLoS Pathog. 3:e102.
17. Herbert S, et al. 2010. Repair of global regulators in *Staphylococcus aureus* 8325 and comparative analysis with other clinical isolates. Infect. Immun. 78:2877–2889.
18. Higgins CF. 2001. ABC transporters: physiology, structure and mechanism—an overview. Res. Microbiol. 152:205–210.
19. Hiron A, Falord M, Valle J, Debarbouille M, Msadek T. 2011. Bacitracin and nisin resistance in *Staphylococcus aureus*: a novel pathway involving the BraS/BraR two-component system (SA2417/SA2418) and both the BraD/BraE and VraD/VraE ABC transporters. Mol. Microbiol. 81: 602–622.
20. Ho SN, Hunt HD, Horton RM, Pullen JK, Pease LR. 1989. Site-directed mutagenesis by overlap extension using the polymerase chain reaction. Gene 77:51–59.
21. Hoch JA, Silhavy TJ (ed). 1995. Two-component signal transduction. ASM Press, Washington, DC.
22. Jenssen H, Hamill P, Hancock RE. 2006. Peptide antimicrobial agents. Clin. Microbiol. Rev. 19:491–511.
23. Jordan S, Hutchings MI, Mascher T. 2008. Cell envelope stress response in Gram-positive bacteria. FEMS Microbiol. Rev. 32:107–146.
24. Joseph P, Fichant G, Quentin Y, Denizot F. 2002. Regulatory relationship of two-component and ABC transport systems and clustering of their genes in the *Bacillus/Clostridium* group, suggest a functional link between them. J. Mol. Microbiol. Biotechnol. 4:503–513.
25. Karimova G, Dautin N, Ladant D. 2005. Interaction network among *Escherichia coli* membrane proteins involved in cell division as revealed by bacterial two-hybrid analysis. J. Bacteriol. 187:2233–2243.
26. Karimova G, Pidoux J, Ullmann A, Ladant D. 1998. A bacterial two-hybrid system based on a reconstituted signal transduction pathway. Proc. Natl. Acad. Sci. U. S. A. 95:5752–5756.

27. Karimova G, Robichon C, Ladant D. 2009. Characterization of YmgF, a 72-residue inner membrane protein that associates with the *Escherichia coli* cell division machinery. J. Bacteriol. 191:333–346.
28. Karimova G, Ullmann A, Ladant D. 2000. A bacterial two-hybrid system that exploits a cAMP signaling cascade in *Escherichia coli*. Methods Enzymol. 328:59–73.
29. Karimova G, Ullmann A, Ladant D. 2001. Protein-protein interaction between *Bacillus stearothermophilus* tyrosyl-tRNA synthetase subdomains revealed by a bacterial two-hybrid system. J. Mol. Microbiol. Biotechnol. 3:73–82.
30. Kobayashi K, et al. 2001. Comprehensive DNA microarray analysis of *Bacillus subtilis* two-component regulatory systems. J. Bacteriol. 183: 7365–7370.
31. Kreiswirth BN, et al. 1983. The toxic shock syndrome exotoxin structural gene is not detectably transmitted by a prophage. Nature 305:709–712.
32. Kuroda M, et al. 2001. Whole genome sequencing of meticillin-resistant *Staphylococcus aureus*. Lancet 357:1225–1240.
33. Li M, et al. 2007. The antimicrobial peptide-sensing system *aps* of *Staphylococcus aureus*. Mol. Microbiol. 66:1136–1147.
34. Li M, et al. 2007. Gram-positive three-component antimicrobial peptide-sensing system. Proc. Natl. Acad. Sci. U. S. A. 104:9469–9474.
35. Lowy FD. 2003. Antimicrobial resistance: the example of *Staphylococcus aureus*. J. Clin. Invest. 111:1265–1273.
36. Lowy FD. 1998. *Staphylococcus aureus* infections. N. Engl. J. Med. 339: 520–532.
37. Mascher T. 2006. Intramembrane-sensing histidine kinases: a new family of cell envelope stress sensors in Firmicutes bacteria. FEMS Microbiol. Lett. 264:133–144.
38. Mascher T, Helmann JD, Unden G. 2006. Stimulus perception in bacterial signal-transducing histidine kinases. Microbiol. Mol. Biol. Rev. 70: 910–938.
39. Meehl M, Herbert S, Gotz F, Cheung A. 2007. Interaction of the GraRS two-component system with the VraFG ABC transporter to support vancomycin-intermediate resistance in *Staphylococcus aureus*. Antimicrob. Agents Chemother. 51:2679–2689.
40. Miller JH. 1972. Experiments in molecular genetics. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.
41. Nizet V. 2006. Antimicrobial peptide resistance mechanisms of human bacterial pathogens. Curr. Issues Mol. Biol. 8:11–26.
42. Pietiäinen M, et al. 2009. Transcriptome analysis of the responses of *Staphylococcus aureus* to antimicrobial peptides and characterization of the roles of *vraDE* and *vraSR* in antimicrobial resistance. BMC Genomics 10:429.
43. Rietkötter E, Hoyer D, Mascher T. 2008. Bacitracin sensing in *Bacillus subtilis*. Mol. Microbiol. 68:768–785.
44. Sambrook J, Fritsch EF, Maniatis T. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY.
45. Sass P, et al. 2008. The lantibiotic mersacidin is a strong inducer of the cell wall stress response of *Staphylococcus aureus*. BMC Microbiol. 8:186.
46. Stock JB, Ninfa AJ, Stock AM. 1989. Protein phosphorylation and regulation of adaptive responses in bacteria. Microbiol. Rev. 53:450–490.
47. Tetsch L, Jung K. 2009. The regulatory interplay between membrane-integrated sensors and transport proteins in bacteria. Mol. Microbiol. 73: 982–991.
48. Volz K. 1995. Structural and functional conservation in response regulators, p 53–64. In Hoch JA and Silhavy TJ (ed), Two-component signal transduction. ASM Press, Washington, DC.
49. Yang SJ, et al. 2012. The *Staphylococcus aureus* two-component regulatory system, GraRS, senses and confers resistance to selected cationic antimicrobial peptides. Infect. Immun. 80:74–81.
