
CCA1 and ELF3 Interact in the Control of Hypocotyl Length and Flowering Time in Arabidopsis${}^{1[W][OA]}$

Sheen X. Lu, Candace J. Webb, Stephen M. Knowles, Sally H.J. Kim, Zhiyong Wang, and Elaine M. Tobin*

University of California, Department of Molecular, Cell, and Developmental Biology, Los Angeles, California 90095 (S.X.L., C.J.W., S.M.K., S.H.J.K., E.M.T.); and Carnegie Institution for Science, Department of Plant Biology, Stanford, California 94305 (Z.W.)

The circadian clock is an endogenous oscillator with a period of approximately 24 h that allows organisms to anticipate, and respond to, changes in the environment. In Arabidopsis (*Arabidopsis thaliana*), the circadian clock regulates a wide variety of physiological processes, including hypocotyl elongation and flowering time. CIRCADIAN CLOCK ASSOCIATED1 (CCA1) is a central clock component, and CCA1 overexpression causes circadian dysfunction, elongated hypocotyls, and late flowering. EARLY FLOWERING3 (ELF3) modulates light input to the clock and is also postulated to be part of the clock mechanism. *elf3* mutations cause light-dependent arrhythmicity, elongated hypocotyls, and early flowering. Although both genes affect similar processes, their relationship is not clear. Here, we show that CCA1 represses ELF3 by associating with its promoter, completing a CCA1-ELF3 negative feedback loop that places ELF3 within the oscillator. We also show that ELF3 acts downstream of CCA1, mediating the repression of PHYTOCHROME-INTERACTING FACTOR4 (PIF4) and PIF5 in the control of hypocotyl elongation. In the regulation of flowering, our findings show that ELF3 and CCA1 either cooperate or act in parallel through the CONSTANS/ FLOWERING LOCUS T pathway. In addition, we show that CCA1 represses GIGANTEA and SUPPRESSOR OF CONSTANS1 by direct interaction with their promoters, revealing additional connections between the circadian clock and the flowering pathways.

The circadian clock is an endogenous timekeeper that generates rhythms of approximately 24 h in biological processes. Environmental signals set the pace of the clock, which then drive self-sustaining oscillations. Conceptually, the Arabidopsis (*Arabidopsis thaliana*) clock can be divided into inputs such as light and temperature, a core oscillator, and outputs including the physiological processes of leaf movement, hypocotyl elongation, and photoperiodic flowering (Johnson et al., 1998; Dowson-Day and Millar, 1999).

The core oscillator consists of three interlocking feedback loops, designated as morning, central, and evening loops (Locke et al., 2005, 2006; Zeilinger et al., 2006; Lu and Tobin, 2011). The central loop contains CIRCADIAN CLOCK ASSOCIATED1 (CCA1) and LATE ELONGATED HYPOCOTYL (LHY), two homologous Myb-domain transcription factors whose functions partially overlap (Schaffer et al., 1998; Wang and Tobin, 1998; Lu et al., 2009). CCA1 and LHY accumulate at dawn to repress the evening-expressed TIMING OF CAB EXPRESSION1 by binding to its promoter (Strayer et al., 2000; Alabadí et al., 2001). In the morning loop,

---

${}^{1}$ This work was supported by the National Institutes of Health (grant no. GM23167 to E.M.T.).

* Corresponding author; e-mail etobin@ucla.edu.

The author responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors (www.plantphysiol.org) is: Elaine M. Tobin (etobin@ucla.edu).

[W] The online version of this article contains Web-only data.

[OA] Open Access articles can be viewed online without a subscription.

www.plantphysiol.org/cgi/doi/10.1104/pp.111.189670

CCA1 and LHY promote the expression of PSEUDORESPONSE REGULATOR9 (PRR9) and PRR7, which repress the expression of CCA1 and LHY (Farré et al., 2005; Nakamichi et al., 2010). EARLY FLOWERING3 (ELF3) accumulates in the evening and has dual roles in the clock; it modulates light input and operates as a core clock component to repress PRR9 (Hicks et al., 1996; McWatters et al., 2000; Covington et al., 2001; Thines and Harmon, 2010; Dixon et al., 2011; Kolmos et al., 2011). Other factors that act close to or within the central oscillator include PRR3, PRR5, GIGANTEA (GI), CCA1 HIKING EXPEDITION, LUX ARRHYTHMO (LUX)/PHYTOCLOCK1, and ELF4 (Doyle et al., 2002; Hazen et al., 2005; Onai and Ishiura, 2005; Kim et al., 2007; Para et al., 2007; Nakamichi et al., 2010; Thines and Harmon, 2010; Dixon et al., 2011; Helfer et al., 2011).

Hypocotyl elongation is a physiological response that is controlled by both the clock and light signaling. The circadian clock regulates the expression of two growth-promoting transcription factors, PHYTOCHROME-INTERACTING FACTOR4 (PIF4) and PIF5, through a complex of the evening-expressed proteins ELF3, ELF4, and LUX (Nusinow et al., 2011). This evening complex represses PIF4 and PIF5 during the first half of the night by binding to their promoters. Later in the night, PIF4 and PIF5 mRNA levels increase and hypocotyl elongation occurs. At dawn, PIF4 and PIF5 are targeted for degradation through their interaction with the light-activated form of phytochrome B (Huq and Quail, 2002; Khanna et al., 2004). In *elf3-1*, the levels of PIF4 and PIF5 mRNA are elevated compared with the wild type, particularly during the early evening (Nusinow et al., 2011). In CCA1 overexpression plants (CCA1-OX), PIF4 and

Plant Physiology®, February 2012, Vol. 158, pp. 1079–1088, www.plantphysiol.org © 2011 American Society of Plant Biologists. All Rights Reserved. 1079

Downloaded from www.plantphysiol.org on November 14, 2015 - Published by www.plant.org

Copyright © 2012 American Society of Plant Biologists. All rights reserved.

Lu et al.

PIF5 cycle with reduced amplitudes and levels of transcript are high throughout the night, giving rise to long hypocotyls (Nozue et al., 2007; Niwa et al., 2009).

The timing of flowering in Arabidopsis is controlled by the autonomous, vernalization, GA, and photoperiodic pathways (Corbesier and Coupland, 2006; Fornara et al., 2010; Srikanth and Schmid, 2011). There are multiple points of cross talk between these pathways. For example, FLOWERING LOCUS C (FLC) operates in the autonomous and vernalization pathways (Michaels and Amasino, 1999, 2001) and represses FLOWERING LOCUS T (FT) and SUPPRESSOR OF CONSTANS1 (SOC1), whose corresponding proteins are floral integrators (Corbesier and Coupland, 2006). The circadian clock controls photoperiodic flowering chiefly through the GI-CONSTANS (CO)-FT pathway (Corbesier and Coupland, 2006; Fornara et al., 2010; Imaizumi, 2010). GI interacts with FLAVIN-BINDING, KELCH REPEAT, F-BOX1 during the day to promote CO expression through degradation of the CO repressor CYCLING DOF FAC-TOR1 (Imaizumi et al., 2003, 2005; Sawa et al., 2007). CO is stabilized in long days, FT expression is induced, and flowering occurs (Samach et al., 2000; Suárez-López et al., 2001).

CCA1-OX plants exhibit long hypocotyls, late flowering, and widespread arrhythmicity (Wang and Tobin, 1998). Light-dependent arrhythmicity, elongated hypocotyls, and early flowering result from *elf3* mutations (Zagotta et al., 1996). Although CCA1 and ELF3 overexpressors and mutants are well characterized, how CCA1 and ELF3 interact genetically in the circadian oscillator and in the control of the physiological processes of hypocotyl elongation and flowering has not been elucidated. In this study, we show that CCA1 represses ELF3 expression through an association with its promoter to close a loop that includes PRR9. We have determined that ELF3 acts downstream of CCA1 in hypocotyl elongation and that PIF4 and PIF5 are not the only factors involved in this process. In the control of flowering time, we have shown that ELF3 and CCA1 either cooperate or act in parallel through the CO-FT pathway. Finally, our results show that CCA1 binds directly to the GI and SOC1 promoters to repress their expression, providing proof of a new connection between the central and evening loops of the Arabidopsis oscillator and introducing additional links between CCA1 and the flowering pathways.

ELF3 represses PRR9 and that PRR9 represses CCA1 expression (Nakamichi et al., 2010; Dixon et al., 2011). Therefore, ELF3 positively regulates the expression of CCA1. To determine whether CCA1 forms a feedback loop with ELF3, we examined ELF3 expression in CCA1-OX and *cca1-1* plants (Fig. 1B). In CCA1-OX plants, ELF3 was repressed and its circadian rhythms abolished. This result is consistent with previous reports showing that overexpression of CCA1 results in arrhythmicity (Wang and Tobin, 1998). In *cca1-1*, ELF3 expression was slightly elevated but still exhibited circadian rhythms with evening peaks of mRNA; the maintenance of circadian rhythms in *cca1-1* plants has been reported previously (Green and Tobin, 1999). Taken together, these results show that CCA1 represses the expression of ELF3. Therefore, ELF3 and CCA1 form a negative feedback loop through PRR9.

CCA1 Represses ELF3 by Interacting with Its Promoter

The ELF3 gene contains a CCA1-binding site (CBS) between –248 and –241 relative to its transcription start site (Fig. 2A; Wang et al., 1997). To examine whether the

![Figure 1](#fig1)

**Figure 1.** CCA1 and ELF3 form a negative feedback loop. A, CCA1 expression in wild-type (WT), *elf3-1*, and ELF3-OX plants under diurnal and continuous light (LL) conditions. B, ELF3 expression in wild-type, *cca1-1*, and CCA1-OX plants under diurnal and LL conditions. Ten-day-old seedlings sampled at 4-h intervals were analyzed by qRT-PCR. Day, night, and subjective night are denoted by white, black, and hatched bars, respectively. The data are presented as means of two biological replicates ± SD. All experiments were done at least twice with similar results.

RESULTS

ELF3 and CCA1 Form a Negative Feedback Loop

To determine how ELF3 affects CCA1 expression, CCA1 mRNA levels were examined over a time course in ELF3-OX and *elf3-1* plants by quantitative reverse transcription (qRT)-PCR (Fig. 1A). Constitutive overexpression of ELF3 resulted in slightly elevated peak levels of CCA1 mRNA, whereas the peak levels of CCA1 mRNA were reduced in *elf3-1* mutant plants. This result is consistent with previous reports that

1080

Downloaded from www.plantphysiol.org on November 14, 2015 - Published by www.plant.org  
Copyright © 2012 American Society of Plant Biologists. All rights reserved.

Interaction of CCA1 and ELF3

ELF3 that contains the CBS but not another one located in the downstream coding region, suggesting that CCA1 binds directly to the promoter of *ELF3*. If this association results in repression, a pulse of CCA1 using an ethanol-inducible system (Knowles et al., 2008) should result in a reduction of *ELF3* mRNA shortly after the induction of CCA1 protein. A pulse of CCA1 causes a reduction in *ELF3* expression within 1 h (Fig. 2, C and D), suggesting that CCA1 binds directly to the *ELF3* promoter to repress its expression. In wild-type Arabidopsis, CCA1 association with the *ELF3* promoter was only observed at 2 h after dawn (zeitgeber time [ZT] 2), when CCA1 protein is most abundant, but not at 2 h after dusk (ZT14), when CCA1 protein is least abundant (Fig. 2B). These results are consistent with CCA1 repression of *ELF3* during the day and rising *ELF3* expression in the evening when CCA1 is absent.

### Interactions between CCA1 and ELF3 in the Control of Hypocotyl Length

To determine whether ELF3 and CCA1 interact genetically in the regulation of hypocotyl length, we compared hypocotyl lengths of *elf3-1*, *CCA1-OX*, and *elf3-1 CCA1-OX* plants. Under both long-day (LD) and short-day (SD) conditions, *elf3-1* and *CCA1-OX* have elongated hypocotyls (Fig. 3), as reported previously (Zagotta et al., 1996; Kim et al., 2005). The hypocotyl length of *elf3-1 CCA1-OX* plants was more similar to the hypocotyl length of *CCA1-OX* than the *elf3-1* mutant, which has shorter hypocotyls than *CCA1-OX* seedlings (Fig. 3), suggesting that additional factors

---

**Figure 2.** CCA1 binds directly to the promoter of *ELF3* to repress its expression. A, Schematic drawing of genomic *ELF3* and the regions examined by ChIP. Coding regions are represented by black boxes, untranslated regions are represented by gray boxes, and introns are represented by lines. The CBS (C) is represented by a white bar. B, Binding of CCA1 to the *ELF3* promoter in vivo. ChIP assays were performed using CCA1 antibody (Ab) with wild-type (WT), *cca1-1*, and *CCA1-OX* (OX) seedlings collected at ZT2 and ZT14. qRT-PCR was performed on the precipitates; *ELF3-P* amplifies the region that contains a CBS, and *ELF3-N* amplifies a region in the downstream coding region. Results were normalized to the input DNA. C and D, A pulse of CCA1 expression represses the accumulation of *ELF3* RNA. Control (C) and *Alc::CCA1* (D) seedlings were given an ethanol (EtOH) pulse at 32 h in continuous light (circadian time [CT]-8). qRT-PCR analysis of *ELF3* expression at the time of induction (0 h) and at 1, 2, and 4 h after ethanol treatment is presented. Diamonds, No ethanol treatment; squares, 1% ethanol. Means of two biological replicates ± SD are shown. All of these experiments were done at least twice with similar results.

---

**Figure 3.** Constitutive overexpression of CCA1 protein affects hypocotyl growth in LD and SD. Wild-type (circles), *elf3-1* (squares), *CCA1-OX* (diamonds), and *elf3-1 CCA1-OX* (triangles) plants were grown for 5 to 7 d under LD (16 h of light/8 h of dark) conditions (A) or under SD (8 h of light/16 h of dark) conditions (B). Average hypocotyl lengths ± SD (*n* = 20) are presented. All experiments were done at least twice with similar results.

---

effect of CCA1 on *ELF3* expression was by direct association with its promoter, chromatin immunoprecipitation (ChIP) was performed using anti-CCA1 antibody. Figure 2B demonstrates that an anti-CCA1 antibody efficiently immunoprecipitated a fragment of

Plant Physiol. Vol. 158, 2012
Downloaded from www.plantphysiol.org on November 14, 2015 - Published by www.plant.org
Copyright © 2012 American Society of Plant Biologists. All rights reserved.

Lu et al.

A 0.09  
B 0.03  

PIF4/Actin  
PIF5/Actin  

0.06  
0.02  

0.03  
0.01  

0.00  
0.00  

0   8   16   24  
ZT (h)  

0   8   16   24  
ZT (h)  

C 0.12  
D 0.03  

PIF4/Actin  
PIF5/Actin  

0.08  
0.02  

0.04  
0.01  

0.00  
0.00  

0   8   16   24  
ZT (h)  

0   8   16   24  
ZT (h)  

Figure 4. Transcript abundance of PIF4 and PIF5 in LD and SD. Expression levels of PIF4 (A and C) and PIF5 (B and D) transcripts in wild-type (diamonds), elf3-1 (squares), CCA1-OX (triangles), and elf3-1 CCA1-OX (circles) plants are shown. Eight-day-old seedlings grown under LD (16 h of light/8 h of dark) conditions (A and B) or 12-d-old seedlings grown under SD (8 h of light/16 h of dark) conditions (C and D) sampled at 4-h intervals were analyzed by qRT-PCR. Day and night are denoted by white and black bars, respectively. The data are presented as means of two biological replicates ± SD. All experiments were done at least twice with similar results.

downstream of CCA1 parallel to ELF3 could be involved in hypocotyl elongation. It is known that the circadian clock can affect hypocotyl elongation through its regulation of PIF4 and PIF5 (Nozue et al., 2007; Nusinow et al., 2011). To determine the mechanism that gives rise to the observed long hypocotyls in elf3-1, CCA1-OX, and elf3-1 CCA1-OX plants, we examined the expression of PIF4 and PIF5. In both LD and SD conditions, PIF4 and PIF5 expression was higher in elf3-1, CCA1-OX, and elf3-1 CCA1-OX plants than in the wild type in the dark (Fig. 4). However, there were no appreciable differences in PIF4 and PIF5 expression when elf3-1, CCA1-OX, and elf3-1 CCA1-OX plants were compared, suggesting that other factors contribute to their differing hypocotyl lengths.

Interactions between CCA1 and ELF3 in the Control of Flowering Time

We determined whether ELF3 and CCA1 interact genetically to control photoperiodic flowering by measuring flowering time and examining the expression of the flowering genes FT, SOC1, FLC, GI, and CO. Under LD growth conditions, elf3-1 CCA1-OX plants flowered before CCA1-OX plants but later than elf3-1 and wild-type plants (Fig. 5, A–C), suggesting that ELF3 acts mainly upstream of CCA1 but can also act coordinately or in parallel with CCA1. There were no significant differences in FLC expression levels among wild-type, elf3-1, CCA1-OX, and elf3-1 CCA1-OX plants (Fig. 6C), suggesting that neither CCA1 nor ELF3 is involved in the autonomous and vernalization pathways. In the elf3-1 plants, FT, GI, and CO were derepressed compared with the wild type (Fig. 6, A, D, and E), as reported previously (Kim et al., 2005). CCA1-OX plants exhibited repressed FT, SOC1, GI, and CO expression (Fig. 6, A, B, D, and E), which is in accordance with its late-flowering phenotype (Fig. 5). The expression levels of FT, SOC1, GI, and CO were similar in CCA1-OX and elf3-1 CCA1-OX plants and low compared with wild-type and elf3-1 plants (Fig. 6, A, B, D, and E). These results suggest that in LD conditions, ELF3 acts mainly

Figure 5. Constitutive overexpression of CCA1 protein delays flowering in LD and SD. A, Photograph of plants of various genotypes grown for 25 d under LD (16 h of light/8 h of dark) conditions. B and C, Flowering time of seedlings of various genotypes under LD (16 h of light/8 h of dark) conditions. D, Photograph of plants of various genotypes grown for 50 d under SD (8 h of light/16 h of dark) conditions. E and F, Flowering time of seedlings of various genotypes under SD (8 h of light/16 h of dark) conditions. Flowering time is expressed as either days to bolting or rosette leaf number. Data are means ± SD (n = 18–25). WT, Wild type.

upstream of CCA1 in the control of the expression of *FT*, *SOC1*, *GI*, and *CO*.

Arabidopsis is a facultative LD plant, and it flowers much later in SD than in LD. Constitutive overexpression of CCA1 delays flowering in SD, although the effect is not as dramatic as in LD (Fig. 5), and the *elf3-1* mutation causes significantly earlier flowering in SD (Fig. 5; Hicks et al., 1996, 2001). Similar to in LD, *elf3-1 CCA1-OX* plants flowered earlier than *CCA1-OX* plants, but later than the *elf3-1* mutant, in SD. When compared with the wild type, *elf3-1 CCA1-OX* plants flowered later, but with fewer rosette leaves. In the analysis of flowering gene expression, we obtained similar results to LD for *SOC1*, *FLC*, *GI*, and *CO* (Fig. 7, B–E), suggesting that, as in LD, ELF3 operates upstream of CCA1 in the control of gene expression in SD. However, the pattern of *FT* expression in SD was different. In *elf3-1 CCA1-OX* plants, *FT* expression was intermediate between that of *elf3-1* and *CCA1-OX* plants (Fig. 7A). Therefore, ELF3 and CCA1 act either in parallel or coordinately to control *FT* expression in SD.

### CCA1 Represses *GI* and *SOC1* by Associating with Their Promoters

The expression of *GI* and *SOC1* was significantly repressed in *CCA1-OX* (Figs. 6, B and E, and 7, B and E), and the promoters of *GI* and *SOC1* both contain CBSs (Figs. 8A and 9A; Wang et al., 1997). To examine whether the effect of CCA1 on *GI* and *SOC1* expression was by direct association with their promoters in planta, we performed ChIP in wild-type, *CCA1-OX*, and *cca1-1* lines. Figures 8B and 9B show that at ZT2, the fragments of *GI* and *SOC1* containing CBSs were significantly enriched in wild-type and *CCA1-OX* plants but not in *cca1-1* plants, suggesting that CCA1 binds to the *GI* and *SOC1* promoters in Arabidopsis. In addition, an ethanol-induced pulse of CCA1 was able to reduce *GI* and *SOC1* expression within 1 h (Figs. 8D and 9D), whereas ethanol treatment of a control line produced no changes in their expression (Figs. 8C and 9C). These results establish that the repression of *GI* and *SOC1* is due to the direct association of CCA1 with their promoters.

---

### DISCUSSION

It has been shown that ELF3 associates with the promoter of *PRR9* to repress its expression and that PRR9 represses CCA1 by interaction with its promoter (Nakamichi et al., 2010; Dixon et al., 2011). Our data that CCA1 represses *ELF3* expression by an association with its promoter (Figs. 1 and 2) close the negative feedback loop composed of ELF3, PRR9, and CCA1 (Fig. 10). These results support the idea that ELF3 is a component of the oscillator itself (Thines and Harmon, 2010; Dixon et al., 2011) in addition to its established role as a modulator of light input to the clock (Hicks et al., 1996; McWatters et al., 2000; Covington et al., 2001; Kolmos et al., 2011). We also show that ELF3 acts downstream of

CCA1 to mediate the repression of *PIF4* and *PIF5* in the control of hypocotyl elongation (Figs. 3 and 4). In the regulation of flowering, ELF3 and CCA1 act in a coordinated manner or in parallel pathways (Figs. 5–7). Finally, we have shown that CCA1 represses *GI* and *SOC1* by association with their promoters (Figs. 8 and 9), revealing additional connections between CCA1 and the flowering pathways.

Hypocotyl elongation is controlled by many factors, including the convergence of light and clock signaling (Nozue et al., 2007). A recent report has shown that ELF3 forms a complex with ELF4 and LUX to directly repress *PIF4* and *PIF5* expression and that an *elf3 pif4 pif5* triple mutant exhibits hypocotyls of similar length to wild-type plants (Nusinow et al., 2011). In this study, we found that *ELF3* expression is repressed directly by CCA1; thus, ELF3 acts downstream of CCA1 to repress *PIF4* and *PIF5*. However, *PIF4* and *PIF5* RNA levels were elevated to similar levels in *elf3-1*, *elf3-1 CCA1-OX*, and *CCA1-OX* plants, while *elf3-1* showed a weaker hypocotyl phenotype than *CCA1-OX* and *elf3-1 CCA1-OX* plants. This is consistent with CCA1 affecting additional factors besides *ELF3*, *PIF4*, and *PIF5* in controlling hypocotyl elongation (Hazen et al., 2005; Li

![Figure 6](https://i.imgur.com/placeholder.png)

**Figure 6.** Transcript abundance of flowering time genes under LD (16 h of light/8 h of dark) conditions. Expression levels of *FT* (A), *SOC1* (B), *FLC* (C), *CO* (D), and *GI* (E) transcripts in wild-type (WT; diamonds), *elf3-1* (squares), *CCA1-OX* (triangles), and *elf3-1 CCA1-OX* (circles) plants are shown. Eight-day-old seedlings grown under LD (16 h of light/8 h of dark) conditions sampled at 4-h intervals were analyzed by qRT-PCR. Day and night are denoted by white and black bars, respectively. The data are presented as means of two biological replicates ± SD. All experiments were done at least twice with similar results.

Lu et al.

**Figure 7.** Transcript abundance of flowering time genes under SD (8 h of light/16 h of dark) conditions. Expression levels of $FT$ (A), $SOC1$ (B), $FLC$ (C), $CO$ (D), and $GI$ (E) transcripts in wild-type (WT; diamonds), $elf3-1$ (squares), $CCA1-OX$ (triangles), and $elf3-1 CCA1-OX$ (circles) plants are shown. Twelve-day-old seedlings grown under SD (8 h of light/16 h of dark) conditions sampled at 4-h intervals were analyzed by qRT-PCR. Day and night are denoted by white and black bars, respectively. The data are presented as means of two biological replicates ± sd. All experiments were done at least twice with similar results.

et al., 2011). The circadian clock regulates flowering through the photoperiodic pathway that includes CO and FT. Both ELF3 and CCA1 act as negative regulators upstream of CO and FT, but their relationship in controlling the photoperiodic pathway is unclear. Under both LD and SD conditions, $elf3-1 CCA1-OX$ flowers before $CCA1-OX$ plants but after $elf3-1$ plants (Fig. 5), suggesting that ELF3 and CCA1 do not act in a linear pathway. In LD, the expression of CO and FT was high in the $elf3-1$ mutant but low in $elf3-1 CCA1-OX$ and $CCA1-OX$ plants (Fig. 6, A and D), suggesting that ELF3 could act upstream of CCA1 in the control of CO and FT. In SD, CO and FT levels are high in $elf3-1$ but low in $CCA1-OX$ plants (Fig. 7, A and D). In $elf3-1 CCA1-OX$ plants, high levels of FT message were observed, but these levels were not in accordance with the low level of CO expression (Fig. 7, A and D). This result is in agreement with a previous report showing that ELF3 can act independently of CO to control flowering time (Kim et al., 2005). It has been shown that ELF3 represses $GI$ transcription independently of CCA1, possibly through its repression of $PRR9$ (Dixon et al., 2011), and ELF3 protein is involved in facilitating the interaction between CONSTITUTIVE PHOTOMORPHOGENIC1 (COP1) and GI. The COP1-GI interaction destabilizes the GI pro-

tein to ensure proper GI function in the control of photoperiodic flowering (Yu et al., 2008). In this study, we showed that CCA1 could bind directly to the $GI$ promoter to repress its expression (Fig. 8), adding another link within the clock and between the clock and the photoperiodic flowering pathway. In accord-

**Figure 8.** CCA1 binds directly to the promoter of $GI$ to repress its expression. A, Schematic drawing of genomic $GI$ and the regions examined by ChIP. Coding regions are represented by black boxes, untranslated regions are represented by gray boxes, and introns are represented by lines. The CBS (C) is represented by a white bar. B, Binding of CCA1 to the $GI$ promoter in vivo. ChIP assays were performed with wild-type (WT), $cca1-1$, and $CCA1-OX$ (OX) seedlings collected at ZT2 and ZT14. qRT-PCR was performed on the precipitates; $GI$-P amplifies the region that contains a CBS, and $GI$-N amplifies a region in the downstream coding region. Results were normalized to the input DNA. C and D, A pulse of CCA1 expression represses the accumulation of $GI$ RNA. Control (C) and $Alc::CCA1$ (D) seedlings were given an ethanol (EtOH) pulse at 32 h in continuous light (CT-8). qRT-PCR analysis of $GI$ expression at the time of induction (0 h) and at 1, 2, and 4 h after ethanol treatment is presented. Diamonds, No ethanol treatment; squares, 1% ethanol. Means of two biological replicates ± sd are shown. All of these experiments were done at least twice with similar results.

Interaction of CCA1 and ELF3

flowering time through GI. It is possible that the function of ELF3 and CCA1 in the photoperiodic flowering pathway converge on their regulation of GI. Recent reports have shown that GI can activate flowering through a CO-independent pathway through three possible mechanisms. First, GI regulates miR172, which represses genes such as TARGET OF EAT1 and SCHLAF-MUTZE posttranscriptionally, relieving their repression of FT (Jung et al., 2007). Second, GI can interact with FT repressors such as SHORT VEGETATIVE PHASE, TEM-PRANILLO (TEM1), and TEM2 to regulate the stability of FT repressors or their accessibility to the FT promoter (Sawa and Kay, 2011). Third, GI interacts physically with the FT promoter either alone or in a complex with another protein (Sawa and Kay, 2011). The strong CO-independent FT induction in elf3-1 CCA1-OX plants under SD conditions is likely due to the fact that the flowering pathways form a highly integrated network, and CO-dependent FT activation is probably more prominent in LD. We have presented a model in which ELF3 and CCA1 independently regulate the expression of GI and GI regulates the expression of FT through both CO-dependent and -independent pathways (Fig. 10).

The MADS box transcription factor SOC1 is a key floral activator integrating multiple flowering pathways. The expression of SOC1 is positively regulated by FT (Yoo et al., 2005) and negatively regulated by FLC, which forms a floral repressor complex with other proteins and directly binds to the promoter of SOC1 (Helliwell et al., 2006). In this study, we found that elf3-1 CCA1-OX plants exhibit low levels of SOC1 message, similar to CCA1-OX plants (Figs. 6B and 7B), indicating that CCA1 acts downstream of ELF3 to negatively regulate the expression of SOC1. This result is in accordance with a previous report showing that SOC1 expression is higher in cca1-1 lhy-11 double mutants (Fujiwara et al., 2005). Furthermore, we demonstrated that CCA1 significantly represses SOC1 expression through direct interaction with its promoter (Fig. 9), illustrating a novel way that the circadian clock controls flowering: through direct regulation of a floral integrator (Fig. 10).

The examination of flowering gene expression revealed that neither ELF3 nor CCA1 participates in the

Figure 9. CCA1 binds directly to the promoter of SOC1 to repress its expression. A, Schematic drawing of genomic SOC1 and the regions examined by ChIP. Coding regions are represented by black boxes, untranslated regions are represented by gray boxes, and introns are represented by lines. The CBS (C) is represented by a white bar. B, Binding of CCA1 to the SOC1 promoter in vivo. ChIP assays were performed with wild-type (WT), cca1-1, and CCA1-OX (OX) seedlings collected at ZT2 and ZT14. qRT-PCR was performed on the precipitates; SOC1-P amplifies the region that contains a CBS, and SOC1-N amplifies a region in the downstream coding region. Results were normalized to the input DNA. C and D, A pulse of CCA1 expression represses the accumulation of SOC1 RNA. Control (C) and Alc::CCA1 (D) seedlings were given an ethanol (EtOH) pulse at 32 h in continuous light (CT-8). qRT-PCR analysis of SOC1 expression at the time of induction (0 h) and at 1, 2, and 4 h after ethanol treatment is presented. Diamonds, No ethanol treatment; squares, 1% ethanol. Means of two biological replicates ± SD are shown. All of these experiments were done at least twice with similar results.

dance with these data, gi-3 cca1-1 lhy-11 triple mutants exhibit wild-type flowering time, even though cca1-1 lhy-11 double mutants flower very early (Mizoguchi et al., 2005); this result suggests that CCA1 acts on

Plant Physiol. Vol. 158, 2012

Downloaded from www.plantphysiol.org on November 14, 2015 - Published by www.plant.org

Copyright © 2012 American Society of Plant Biologists. All rights reserved.

Figure 10. A proposed model for how the clock-associated factors CCA1, ELF3, and PRR9 affect the expression of members of the photoperiodic flowering pathway. Solid lines indicate direct regulation. Connections that were confirmed in this study are represented by thick lines. Dotted lines indicate indirect regulation (i.e. additional steps and/or components are involved but not shown here).

Lu et al.

regulation of FLC (Figs. 6C and 7C). However, the *elf3-8* mutant exhibits reduced FLC expression in a photo-period-independent manner, suggesting that ELF3 can affect FLC expression (Yu et al., 2008). In *elf3-8*, there is a base pair change at the exon 4 splice acceptor site, resulting in a truncated protein that includes 28 amino acids from a different frame before a premature stop codon (Hicks et al., 2001). In this study, we utilized the *elf3-1* mutant, which produces a truncated protein resulting from a single base pair change that gives rise to an early stop codon (Hicks et al., 1996). A detailed comparison between these two mutants has not been carried out, but they do appear to flower at the same time under LD and SD (Liu et al., 2001; Kim et al., 2005; Yu et al., 2008; Nefissi et al., 2011). *elf3-7* is a weak allele that contains a single base change at the exon 1 splice donor site (Hicks et al., 2001). Through the use of cryptic splice sites, the *elf3-7* mutant can give rise to a combination of truncated ELF3 proteins or a version that is missing eight amino acids (Hicks et al., 2001). A recent report showed that *elf3-7* has less severe hypocotyl and flowering phenotypes than *elf3-1*, but it has the same effect on CCA1 and LHY expression (Kolmos et al., 2011), showing that it is difficult to predict how a given mutant allele of ELF3 will behave. Although *elf3-1* and *elf3-8* mutants have similar flowering times, how they act on the various flowering genes, including FLC, may be different, because a complete understanding of the functions of ELF3 protein domains is lacking.

In summary, our study of the interplay between the central clock component CCA1 and ELF3 has completed the CCA1-ELF3 negative feedback loop that places ELF3 within the oscillator. We have shown that CCA1 represses *GI* and *SOC1* expression by direct association with their promoters, not only highlighting the multiple roles of CCA1 in the regulation of flowering time but also revealing additional connections between the circadian clock and the flowering pathway.


RNA Extraction and qRT-PCR

One- to 2-week-old seedlings were grown on Murashige and Skoog medium (Murashige and Skoog, 1962) with 1.5% agar. For the circadian experiments, samples were collected every 4 h either during the light/dark cycle or in continuous white light. Total RNA extraction and qRT-PCR were carried out as described previously (Lu et al., 2011b). ACTIN2 was used as a noncycling reference, and the expression levels were normalized to the control. The primers used for amplification are listed in Supplemental Table S1.

ChIP

ChIP was performed as described previously (Lu et al., 2011a) using affinity-purified anti-CCA1 antibody (Wang and Tobin, 1998) for immunoprecipitation. The primers used for amplification are listed in Supplemental Table S1.

Ethanol Pulse

Ethanol treatment was performed as described previously (Knowles et al., 2008). Control plants contained the regulator construct 35S::AlcR:$T_{NOS}$. Ten-day-old seedlings were treated with 10 min of 1% (v/v) ethanol vapor. Samples were collected at the time of induction (0 h) and at 1, 2, and 4 h after ethanol treatment.

Sequence data from this article can be found in the Arabidopsis Genome Initiative data library using the following accession numbers: CCA1 (At2g46830), ELF3 (At2g25930), *PIF4* (At2g43010), *PIF5* (At3g59060), *FT* (At1g65480), *CO* (At5g15840), *FLC* (At5g10140), *SOC1* (At2g45660), *GI* (At1g22770), and *ACTIN2* (At3g18780).

Supplemental Data

The following materials are available in the online version of this article.

Supplemental Table S1. List of PCR primer sequences.


ACKNOWLEDGMENTS

We thank David Somers for the *ELF3-OX* seeds and Chuah Cha and Matt Chiang for excellent technical assistance.

Received October 20, 2011; accepted December 16, 2011; published December 21, 2011.


LITERATURE CITED

Alabadí D, Oyama T, Yanovsky MJ, Harmon FG, Más P, Kay SA (2001) Reciprocal regulation between TOC1 and LHY/CCA1 within the Arabidopsis circadian clock. Science 293: 880–883

Corbesier L, Coupland G (2006) The quest for florigen: a review of recent progress. J Exp Bot 57: 3395–3403

Covington MF, Panda S, Liu XL, Strayer CA, Wagner DR, Kay SA (2001) ELF3 modulates resetting of the circadian clock in *Arabidopsis*. Plant Cell 13: 1305–1315

Dixon LE, Knox K, Kozma-Bognar L, Southern MM, Pokhilko A, Millar AJ (2011) Temporal repression of core circadian genes is mediated through EARLY FLOWERING 3 in Arabidopsis. Curr Biol 21: 120–125

Dowson-Day MJ, Millar AJ (1999) Circadian dysfunction causes aberrant hypocotyl elongation patterns in Arabidopsis. Plant J 17: 63–71

Doyle MR, Davis SJ, Bastow RM, McWatters HG, Kozma-Bognár L, Nagy F, Millar AJ, Amasino RM (2002) The ELF4 gene controls circadian rhythms and flowering time in Arabidopsis thaliana. Nature 419: 74–77

Farré EM, Harmer SL, Harmon FG, Yanovsky MJ, Kay SA (2005) Overlapping and distinct roles of PRR7 and PRR9 in the Arabidopsis circadian clock. Curr Biol 15: 47–54

Fornara F, de Montaigu A, Coupland G (2010) SnapShot: control of flowering in Arabidopsis. Cell 141: 550, 550.e1–550.e2

Fujiwara S, Oda A, Kamada H, Coupland G, Mizoguchi T (2005) Circadian clock components in Arabidopsis. II. LHY/CCA1 regulate the


Plant Physiol. Vol. 158, 2012

floral integrator gene SOC1 in both GI-dependent and -independent pathways. Plant Biotechnol 22: 319–325

Green RM, Tobin EM (1999) Loss of the circadian clock-associated protein 1 in Arabidopsis results in altered clock-regulated gene expression. Proc Natl Acad Sci USA 96: 4176–4179

Hazén SP, Schultz TF, Pruneda-Paz JL, Borevitz JO, Ecker JR, Kay SA (2005) LUX ARRHYTHMO encodes a Myb domain protein essential for circadian rhythms. Proc Natl Acad Sci USA 102: 10387–10392

Helfer A, Nusinow DA, Chow BY, Gehrke AR, Bulyk ML, Kay SA (2011) LUX ARRHYTHMO encodes a nighttime repressor of circadian gene expression in the Arabidopsis core clock. Curr Biol 21: 126–133

Hellíwell CA, Wood CC, Robertson M, James Peacock W, Dennis ES (2006) The Arabidopsis FLC protein interacts directly in vivo with SOC1 and FT chromatin and is part of a high-molecular-weight protein complex. Plant J 46: 183–192

Hicks KA, Albertson TM, Wagner DR (2001) EARLY FLOWERING3 encodes a novel protein that regulates circadian clock function and flowering in *Arabidopsis*. Plant Cell 13: 1281–1292

Hicks KA, Millar AJ, Carré IA, Somers DE, Straume M, Meeks-Wagner DR, Kay SA (1996) Conditional circadian dysfunction of the Arabidopsis early-flowering 3 mutant. Science 274: 790–792

Huq E, Quail PH (2002) PIF4, a phytochrome-interacting bHLH factor, functions as a negative regulator of phytochrome B signaling in Arabidopsis. EMBO J 21: 2441–2450

Imaizumi T (2010) Arabidopsis circadian clock and photoperiodism: time to think about location. Curr Opin Plant Biol 13: 83–89

Imaizumi T, Schultz TF, Harmon FG, Ho LA, Kay SA (2005) FKF1 F-box protein mediates cyclic degradation of a repressor of CONSTANS in Arabidopsis. Science 309: 293–297

Imaizumi T, Tran HG, Swartz TE, Briggs WR, Kay SA (2003) FKF1 is essential for photoperiodic-specific light signaling in Arabidopsis. Nature 426: 302–306

Johnson CH, Knight MR, Trewavas A, Kondo T (1998) A clockwork green: circadian programs in photosynthetic organisms. In PJ Lumsden, AJ Millar, eds, Biological Rhythms and Photoperiodism in Plants. BIOS Scientific Publishers, Oxford, pp 1–34

Jung JH, Seo YH, Seo PJ, Reyes JL, Yun J, Chua NH, Park CM (2007) The GIGANTEA-regulated microRNA172 mediates photoperiodic flowering independent of CONSTANS in *Arabidopsis*. Plant Cell 19: 2736–2748

Khanna R, Huq E, Kikis EA, Al-Sady B, Lanzatella C, Quail PH (2004) A novel molecular recognition motif necessary for targeting photoactivated phytochrome signaling to specific basic helix-loop-helix transcription factors. Plant Cell 16: 3033–3044

Kim WY, Fujiwara S, Suh SS, Kim J, Kim Y, Han L, David K, Putterill J, Nam HG, Somers DE (2007) ZEITLUPE is a circadian photoreceptor stabilized by GIGANTEA in blue light. Nature 449: 356–360

Kim WY, Hicks KA, Somers DE (2005) Independent roles for EARLY FLOWERING 3 and ZEITLUPE in the control of circadian timing, hypocotyl length, and flowering time. Plant Physiol 139: 1557–1569

Knowles SM, Lu SX, Tobin EM (2008) Testing time: can ethanol-induced pulses of proposed oscillator components phase shift rhythms in *Arabidopsis*? J Biol Rhythms 23: 463–471

Kolmos E, Herrero E, Bujdoso N, Millar AJ, Toth R, Gyula P, Nagy F, Davis SJ (2011) A reduced-function allele reveals that EARLY FLOWERING3 repressive action on the circadian clock is modulated by phytochrome signals in *Arabidopsis*. Plant Cell 23: 3230–3246

Li G, Siddiqui H, Teng Y, Lin R, Wan XY, Li J, Lau OS, Ouyang X, Dai M, Wan J, et al (2011) Coordinated transcriptional regulation underlying the circadian clock in Arabidopsis. Nat Cell Biol 13: 616–622

Liu XL, Covington MF, Fankhauser C, Chory J, Wagner DR (2001) ELF3 encodes a circadian clock-regulated nuclear protein that functions in an *Arabidopsis* PHYB signal transduction pathway. Plant Cell 13: 1293–1304

Locke JC, Kozma-Bognár L, Gould PD, Fehér B, Kevei E, Nagy F, Turner MS, Hall A, Millar AJ (2006) Experimental validation of a predicted feedback loop in the multi-oscillator clock of *Arabidopsis thaliana*. Mol Syst Biol 2: 59

Locke JC, Southern MM, Kozma-Bognar L, Hibberd V, Brown PE, Turner MS, Millar AJ (2005) Extension of a genetic network model by iterative experimentation and mathematical analysis. Mol Syst Biol 1: 13

Lu SX, Knowles SM, Andronis C, Ong MS, Tobin EM (2009) CIRCADIAN CLOCK ASSOCIATED1 and LATE ELONGATED HYPOCOTYL function synergistically in the circadian clock of Arabidopsis. Plant Physiol 150: 834–843

---

Lu SX, Knowles SM, Webb CJ, Celaya RB, Cha C, Siu JP, Tobin EM (2011a) The Jumonji C domain-containing protein JMJ30 regulates period length in the Arabidopsis circadian clock. Plant Physiol 155: 906–915

Lu SX, Liu H, Knowles SM, Li J, Ma L, Tobin EM, Lin C (2011b) A role for protein kinase casein kinase2 α-subunits in the Arabidopsis circadian clock. Plant Physiol 157: 1537–1545

Lu SX, Tobin EM (2011) Chromatin remodeling and the circadian clock: Jumonji C-domain containing proteins. Plant Signal Behav 6: 810–814

McWatters HG, Bastow RM, Hall A, Millar AJ (2000) The ELF3 zeitnehmer regulates light signalling to the circadian clock. Nature 408: 716–720

Michaels SD, Amasino RM (1999) FLOWERING LOCUS C encodes a novel MADS domain protein that acts as a repressor of flowering. Plant Cell 11: 949–956

Michaels SD, Amasino RM (2001) Loss of FLOWERING LOCUS C activity eliminates the late-flowering phenotype of FRIGIDA and autonomous pathway mutations but not responsiveness to vernalization. Plant Cell 13: 935–941

Mizoguchi T, Wright L, Fujiwara S, Cremer F, Lee K, Onouchi H, Mouradov A, Fowler S, Kamada H, Putterill J, et al (2005) Distinct roles of GIGANTEA in promoting flowering and regulating circadian rhythms in Arabidopsis. Plant Cell 17: 2255–2270

Murashige T, Skoog F (1962) A revised medium for rapid growth and bioassays with tobacco tissue cultures. Physiol Plant 15: 473–497

Nakamichi N, Kiba T, Henriques R, Mizuno T, Chua NH, Sakakibara H (2010) PSEUDO-RESPONSE REGULATORS 9, 7, and 5 are transcriptional regulators in the *Arabidopsis* circadian clock. Plant Cell 22: 594–605

Nefissi R, Natsui Y, Miyata K, Oda A, Hase Y, Nakagawa M, Ghorbel A, Mizoguchi T (2011) Double loss-of-function mutation in EARLY FLOWERING 3 and CRYPTOCHROME 2 genes delays flowering under continuous light but accelerates it under long days and short days: an important role for Arabidopsis CRY2 to accelerate flowering time in continuous light. J Exp Bot 62: 2731–2744

Niwa Y, Yamashino T, Mizuno T (2009) The circadian clock regulates the photoperiodic response of hypocotyl elongation through a coincidence mechanism in *Arabidopsis thaliana*. Plant Cell Physiol 50: 838–852

Nozue K, Covington MF, Duek PD, Lorrain S, Fankhauser C, Harmer SL, Maloof JN (2007) Rhythmic growth explained by coincidence between internal and external cues. Nature 448: 358–361

Nusinow DA, Helfer A, Hamilton EE, King JJ, Imaizumi T, Schultz TF, Farré EM, Kay SA (2011) The ELF4-ELF3-LUX complex links the circadian clock to diurnal control of hypocotyl growth. Nature 475: 398–402

Onai K, Ishiura M (2005) PHYTOCLOCK 1 encoding a novel GARP protein essential for the Arabidopsis circadian clock. Genes Cells 10: 963–972

Para A, Farré EM, Imaizumi T, Pruneda-Paz JL, Harmon FG, Kay SA (2007) PRR3 is a vascular regulator of TOC1 stability in the Arabidopsis circadian clock. Plant Cell 19: 3462–3473

Samach A, Onouchi H, Gold SE, Ditta GS, Schwarz-Sommer Z, Yanofsky MF, Coupland G (2000) Distinct roles of CONSTANS target genes in reproductive development of Arabidopsis. Science 288: 1613–1616

Sawa M, Kay SA (2011) GIGANTEA directly activates Flowering Locus T in *Arabidopsis thaliana*. Proc Natl Acad Sci USA 108: 11698–11703

Sawa M, Nusinow DA, Kay SA, Imaizumi T (2007) FKF1 and GIGANTEA complex formation is required for day-length measurement in Arabidopsis. Science 318: 261–265

Schaffer R, Ramsay N, Samach A, Corden S, Putterill J, Carré IA, Coupland G (1998) The late elongated hypocotyl mutation of Arabidopsis disrupts circadian rhythms and the photoperiodic control of flowering. Cell 93: 1219–1229

Srikanth A, Schmid M (2011) Regulation of flowering time: all roads lead to Rome. Cell Mol Life Sci 68: 2013–2037

Strayer C, Oyama T, Schultz TF, Raman R, Somers DE, Más P, Panda S, Kreps JA, Kay SA (2000) Cloning of the Arabidopsis clock gene TOC1, an autoregulatory response regulator homolog. Science 289: 768–771

Suárez-López P, Wheatley K, Robson F, Onouchi H, Valverde F, Coupland G (2001) CONSTANS mediates between the circadian clock and the control of flowering in Arabidopsis. Nature 410: 1116–1120

Thines B, Harmon FG (2010) Ambient temperature response establishes ELF3 as a required component of the core Arabidopsis circadian clock. Proc Natl Acad Sci USA 107: 3257–3262

Wang ZY, Kenigsbuch D, Sun L, Harel E, Ong MS, Tobin EM (1997) A Myb-related transcription factor is involved in the phytochrome regulation of an *Arabidopsis* Lhcb gene. Plant Cell 9: 491–507

Wang ZY, Tobin EM (1998) Constitutive expression of the CIRCADIAN

Lu et al.

CLOCK ASSOCIATED 1 (CCA1) gene disrupts circadian rhythms and suppresses its own expression. Cell 93: 1207–1217

Yoo SK, Chung KS, Kim J, Lee JH, Hong SM, Yoo SJ, Yoo SY, Lee JS, Ahn JH (2005) CONSTANS activates SUPPRESSOR OF OVEREXPRESSION OF CONSTANS 1 through FLOWERING LOCUS T to promote flowering in Arabidopsis. Plant Physiol 139: 770–778

Yu JW, Rubio V, Lee NY, Bai S, Lee SY, Kim SS, Liu L, Zhang Y, Irigoyen ML, Sullivan JA, et al (2008) COP1 and ELF3 control circadian function

and photoperiodic flowering by regulating GI stability. Mol Cell 32: 617–630

Zagotta MT, Hicks KA, Jacobs CI, Young JC, Hangarter RP, Meeks-Wagner DR (1996) The Arabidopsis ELF3 gene regulates vegetative photomorphogenesis and the photoperiodic induction of flowering. Plant J 10: 691–702

Zeilinger MN, Farré EM, Taylor SR, Kay SA, Doyle FJ III (2006) A novel computational model of the circadian clock in Arabidopsis that incorporates PRR7 and PRR9. Mol Syst Biol 2: 58

Table S1. List of PCR primer sequences

| Gene/allele | Purpose | Primer Name | Sequence |
|-------------|---------|-------------|----------|
| ACT 2       | qRT-PCR | qACT-2F     | 5'-GTGGATTCCAGCAGCTTCCAT-3' |
| (At3g18780) |         | qACT-2R     | 5'-GCTGAGAGATTCAGATGCCCA-3' |
| CCA 1       | qRT-PCR | qCCA1-1F    | 5'-CCTTTTACAAAACCCGGCTCTT-3' |
| (At2g46830) |         | qCCA1-1R    | 5'-AATCGGGAGGCCAAAATGA-3' |
| CO          | qRT-PCR | CO-qRT-F    | 5'-TCCATTAACCATAACGCATACAT TTC-3' |
| (At5g15840) |         | CO-qRT-R    | 5'-CGGCACAACACCCAGTTTCC-3' |
| ELF3        | qRT-PCR | ELF3q-1F    | 5'-GGGGTTTGAGAAAGTTTGCAAGTG-3' |
| (At2g25930) |         | ELF3q-1R    | 5'-ATCATTCACATGAAGCCGAGGA-3' |
|             | ChIP    | ELF3-P-F    | 5'-TCCCCCTATAATTTCATACAAC CCTA-3' |
|             |         | ELF3-P-R    | 5'-GAGGTTCTTTGATCAGTGCTTTG-3' |
|             |         | ELF3-N-F    | 5'-ACAAGCCTCACCCAGGTATG-3' |
|             |         | ELF3-N-R    | 5'-GGTTCATTATGGTGGGCATC-3' |
| FLC         | qRT-PCR | qFLC-F      | 5'-TCGCAAACGGTCTCATCGA-3' |
| (At5g10140) |         | qFLC-R      | 5'-GGCGGAGACGAGAAG-3' |
| FT          | qRT-PCR | qFT-F       | 5'-CAACCCTCACCTCCGAGAATAT-3' |
| (At1g65480) |         | qFT-R       | 5'-TTGCCAAAAGGTTGTTCCAGTT-3' |
| GI          | qRT-PCR | qGI-1F      | 5'-ACTAGCAGTGGTCGACGGTTTATC-3' |
| (At1g22770) |         | qGI-1R      | 5'-GCTGGTAGACGACACTTCAATAGATT-3' |
|             | ChIP    | GI-P-F      | 5'-ACCTTTTTCGTGGTTTTTGG-3' |
|             |         | GI-P-R      | 5'-TTCAGAAGAACAAAATCTCAATCA-3' |
|             |         | GI-N-F      | 5'-CTGGAAAGCCGATATCCAAA-3' |
|             |         | GI-N-R      | 5'-AAGTGGGTGCTCGTTATTGG-3' |
| PIF4        | qRT-PCR | PIF4-3F     | 5'-GTTGTTGACTTTGCTGTCCCGC-3' |
| (At2g43010) |         | PIF4-3R     | 5'-CGACTCAGCCGATGGAGATGTT-3' |
| PIF5        | qRT-PCR | PIF5-3F     | 5'-CGCCGGAGATCCAAATCCCAACAT-3' |
| (At3g59060) |         | PIF5-3R     | 5'-GCGGGAAATCAGACC GTGCAACAA-3' |
| SOC1        | qRT-PCR | qSOC1-F     | 5'-CTCTTGGGAGAAGGCATAGGAA-3' |
| (At2g45660) |         | qSOC1-R     | 5'-TCAAGCTGTTGCTCAATCTGTTG-3' |
|             | ChIP    | SOC1-P-F    | 5'-GTGGGGG CATATAGGTACGA-3' |
|             |         | SOC1-P-R    | 5'-AGTTGGATG GAAATGCC TGT-3' |
|             |         | SOC1-N-F    | 5'-AGCTGCAGAAAA CGAGAAGC-3' |
|             |         | SOC1-N-R    | 5'-CCCAAAAAGA ACCCCC AACTT-3' |
