
YEASTBOOK

CELL SIGNALING & DEVELOPMENT

# The Regulation of Filamentous Growth in Yeast

Paul J. Cullen* and George F. Sprague Jr.†,1

*Department of Biological Sciences, State University of New York, Buffalo, New York 14260-1300 and †Institute of Molecular Biology and Department of Biology, University of Oregon, Eugene, Oregon 97403-1229

---

## ABSTRACT

Filamentous growth is a nutrient-regulated growth response that occurs in many fungal species. In pathogens, filamentous growth is critical for host–cell attachment, invasion into tissues, and virulence. The budding yeast *Saccharomyces cerevisiae* undergoes filamentous growth, which provides a genetically tractable system to study the molecular basis of the response. Filamentous growth is regulated by evolutionarily conserved signaling pathways. One of these pathways is a mitogen activated protein kinase (MAPK) pathway. A remarkable feature of the filamentous growth MAPK pathway is that it is composed of factors that also function in other pathways. An intriguing challenge therefore has been to understand how pathways that share components establish and maintain their identity. Other canonical signaling pathways—rat sarcoma/protein kinase A (RAS/PKA), sucrose nonfermentable (SNF), and target of rapamycin (TOR)—also regulate filamentous growth, which raises the question of how signals from multiple pathways become integrated into a coordinated response. Together, these pathways regulate cell differentiation to the filamentous type, which is characterized by changes in cell adhesion, cell polarity, and cell shape. How these changes are accomplished is also discussed. High-throughput genomics approaches have recently uncovered new connections to filamentous growth regulation. These connections suggest that filamentous growth is a more complex and globally regulated behavior than is currently appreciated, which may help to pave the way for future investigations into this eukaryotic cell differentiation behavior.

---

## TABLE OF CONTENTS

- Abstract                                                                                                   23
- Introduction                                                                                               24
- The Filamentous Growth Response                                                                            24
- Signaling Pathways That Regulate Filamentous Growth                                                        25
  - Nutrient-sensing pathways                                                                                25
    - Ras2/cAMP-PKA pathway and affiliated G-protein-coupled receptor Gpr1:                                  26
    - Snf1 pathway:                                                                                         28
    - TOR pathway:                                                                                          28
    - Other sensory pathways:                                                                               29
  - The filamentation MAPK pathway: expeditions into signaling specificity                                    29
    - Kss1 is the MAP kinase for the filamentation pathway:                                                  31
    - Scaffolding proteins insulate signaling by proteins in the core module:                                32
    - Proteins that function at the head of the MAPK pathway:                                                33
    - Mechanisms of signal integration during filamentous growth                                              35

Continued

---

Copyright © 2012 by the Genetics Society of America  
doi: 10.1534/genetics.111.127456  
Manuscript received February 3, 2011; accepted for publication June 3, 2011  
¹ Corresponding author: Institute of Molecular Biology and Department of Biology, University of Oregon, Eugene, OR 97403-1229. E-mail: gsprague@molbio.uoregon.edu

Genetics, Vol. 190, 23–49 January 2012 23

CONTENTS, continued

How Do Signaling Pathways Accomplish Nutritional Cell Differentiation? 37  
Cell adhesion regulation by the flocculin Flo11 37  
Cell polarity reorganization by bud-site–selection proteins 38  
Cell elongation due to changes in the cell cycle and polarized growth 40  
Transcriptional targets of filamentation signaling pathways 40  
Perspectives 41  

FILAMENTOUS growth is a fungal differentiation behavior that occurs in response to extracellular stimuli. One stimulus that triggers filamentous growth is nutrient limitation, and filamentous growth is thought to represent a fungal scavenging response. Many different species undergo filamentous growth, including plant and animal pathogens and yeasts like the baker’s (or budding) yeast *Saccharomyces cerevisiae*. Because budding yeast is readily amenable to a variety of genetic and genome-wide approaches (Botstein and Fink 2011), relatively recent studies using this organism have shed light on how filamentous growth is regulated, what cues cause it, and what genetic pathways mediate the morphological changes. In this review article, we focus on those advances. Other review articles discuss filamentous growth regulation in filamentous fungi (Steinberg 2007), and in fungal pathogens and the immune response (Netea and Marodi 2010; Hajishengallis and Lambris 2011; Kronstad et al. 2011; Moran et al. 2011) and summarize findings not described here.

Signal transduction pathways have taken center stage in the effort to understand filamentous growth regulation in yeast. Given that many signaling pathways regulate filamentous growth, and that some of these pathways are composed of proteins that function in multiple pathways, we will stress issues that relate to signal integration and signal insulation between pathways. We will also address the important question of how signaling pathways accomplish the change in cell type from the yeast mode to the filamentous mode. Other review articles have been published recently on filamentous growth regulation (Nobile and Mitchell 2006; Verstrepen and Klis 2006; Whiteway and Bachewich 2007; Zhao et al. 2007; Bruckner and Mosch 2011), nutrient-regulated signaling pathways (Hedbacker and Carlson 2008; Zaman et al. 2008; Sengupta et al. 2010), and mitogen activated protein kinase (MAPK) regulation (Bardwell 2006; Dohlman and Slessareva 2006; Chen and Thorner 2007; Saito 2010), which may offer different perspectives than those described here.

The Filamentous Growth Response

Filamentous growth is a fungal-specific growth mode in which cells adopt a unique morphological pattern that allows expansion into new environments. The filamentation response is highly variable among species, ranging from mycelial mat or hyphal formation in true filamentous fungi to subtle changes in cell shape in yeasts. The biology that attends this response is fascinating and mysterious and ranges from contact-responsive hyphal growth (Kumamoto 2005) to behavior modification of insect species, such as the erratic behavior exhibited by “zombie ants” infected with *Ophiocordyceps* (Pontoppidan et al. 2009), to the formation of lasso-type structures in Nematode-trapping parasites (Wang et al. 2009a). Some species, like the extensively studied fission yeast *Schizosaccharomyces pombe*, have only recently been shown to undergo filamentous growth as part of their life cycles (Amoah-Buahin et al. 2005).

The hyphal growth of filamentous fungi is morphologically striking. In *Neurospora crassa*, hyphal cells are multinucleate (Ramos-Garcia et al. 2009) and grow in bifurcating branches (Ziv et al. 2009) that can undergo cell-to-cell fusion (Steinberg 2007; Aldabbous et al. 2010). Fusion is a dynamic process that occurs by hyphal-cell recognition through a MAPK-dependent sensing mechanism (Fleissner et al. 2009). Hyphal cells grow rapidly, and cell polarity can be reorganized in response to many different cues. Polarized growth is regulated by a curious structure, the Spitzenkörper (Crampin et al. 2005).

Historically, much interest in understanding filamentous growth regulation has come from studies in fungal pathogens. Pathogens like *Candida albicans* and *Aspergillus fumigatus* pose a worldwide threat to human health (Netea et al. 2008; Gastebois et al. 2009). These pathogens are particularly harmful to individuals whose immune system has been compromised by AIDS or by suppression resulting from chemotherapies and other drug treatments (Ben-Ami et al. 2008). Fungal pathogens can also be devastating to plant communities, and harvest loss as a result of damage from fungal species is a serious problem (Rispail et al. 2009). In *C. albicans*, transition to the filamentous cell type is critical for virulence (Lo et al. 1997) and depends on a multitude of extracellular factors including temperature and nutrient availability (Berman 2006). Pathogenicity of *C. albicans* involves many interrelated processes that include cell-surface variation (Nather and Munro 2008), host–cell adhesion (Latge 2010), biofilm formation (d'Enfert 2009), and chromosome reorganization (Selmecki et al. 2010). In other fungal pathogens, like *Cryptococcus neoformans*, filamentous

growth is not tightly related to pathogenicity, as cells primarily exist in the yeast cell type (Lin 2009).

Progress in defining the genetic pathways that regulate filamentous growth has benefited from studies in the versatile fungal eukaryote S. cerevisiae. Lessons learned about filamentous growth regulation in budding yeast have turned out to be true for many fungal species. Identifying and characterizing the genetic pathways that regulate filamentous growth in yeast has contributed to understanding the genetic basis of virulence in fungal pathogens and has provided a model for how eukaryotic cells differentiate into morphologically distinct patterns in response to extrinsic cues.

Budding yeast does not undergo true hyphal growth, but rather a pseudohyphal growth pattern in which cells fully separate by cytokinesis—they are not multinucleate—and remain attached to each other by proteins in the cell wall. As for many fungal species, yeast cells can transition between yeast-form growth and filamentous-form growth as part of their life cycle (Figure 1). One of the triggers for filamentous growth in yeast and many other fungal species is nutrient limitation. Both haploid and diploid yeast cells undergo a version of the response, but the stimuli that trigger it, the underlying genetic machinery, and the resulting morphological changes differ slightly between the two cell types. The term invasive growth has been applied to the filamentation phenomenon shown by haploid cells because of their ability to invade agar substrates. The term pseudohyphal growth is sometimes used to describe the response in diploid cells. In this review article, we will use the phrase filamentous growth as a general term that applies to both haploid invasive growth and diploid pseudohyphal development. We will not distinguish between these two highly related responses unless it is crucial to do so in some experimental context.

Filamentous growth in yeast can be separated into three major changes: an increase in cell length, a reorganization of polarity, and enhanced cell–cell adhesion. Assays to study filamentous growth in yeast exist on the macroscopic and microscopic levels. The enhanced cell–cell adhesion of filamentous cells is visible by inspecting yeast colonies (Figure 2A). Cells on the underside of the colonies attach to and invade the agar substratum (Figure 2B), and this invasive growth has been used as a tool to determine whether filamentous growth occurs (Roberts and Fink 1994) and to screen for mutants that are defective at filamentous growth (or are better at it than wild-type cells, e.g., Palecek et al. 2000). Changes in cell shape are visible by microscopic examination of cells, and specific assays are used to examine the response in haploid (Figure 2C) and diploid cells (Figure 2D). Using these and other assays, many of the genetic pathways that regulate filamentous growth have been uncovered. Below, we focus on the signaling pathways that regulate the response. We describe what the stimuli are, how they might be sensed, and how the activated pathways induce filamentous growth.

![Diagram](https://i.imgur.com/yourimageurl.png)

**Figure 1** The life cycle of the budding yeast *Saccharomyces cerevisiae*. The diagram shows yeast-form cells, which can be induced to undergo different growth responses depending on ploidy and growth condition. Haploid and diploid cells interconvert between the two types by mating and sporulation, respectively. Both haploid and diploid cells can undergo filamentous growth, form biofilms, or enter stationary phase (quiescence) in response to nutrient (glucose or nitrogen) limitation. Diploid cells also sporulate in response to the limitation of carbon and nitrogen sources. Secreted alcohols act as autoinducers to stimulate filamentous growth.

## Signaling Pathways That Regulate Filamentous Growth

### Nutrient-sensing pathways

In 1992, the Fink lab rejuvenated a little-known finding that the budding yeast *S. cerevisiae* undergoes filamentous growth as part of its life cycle (Gimeno et al. 1992). Their study drew attention to anecdotal observations about yeast's growth pattern (Gutlliermond 1920; Lodder 1970; Brown and Hough 1965; Eubanks and Beuchat 1982) and made use of genetic and molecular approaches to gain insights about the underlying mechanism of this unexplored behavior. The initial observation was that isolates of *S. cerevisiae* from a “wild” strain background (Σ1278b) form colonies composed of elongated cells that grow in connected chains on low-nutrient medium. This growth pattern resembles the morphology that is exhibited by some species of filamentous fungi. Filamentous growth is widely considered to represent a nutritional scavenging response, and the Fink lab connected this morphological behavior to nutrition in two important ways: first, strains defective for ammonium utilization were hyperfilamentous (Gimeno et al. 1992), suggesting a connection between nitrogen levels and filamentous growth. Subsequent studies have shown that the lack of fermentable carbon source can also be a trigger for filamentous growth (Cullen and Sprague 2000). Second, and more interesting, the global nutrient regulatory GTPase Ras2 was found to be required for filamentous growth regulation. The key experiment used an activated version of Ras2, in which the protein was “locked” in its activated (GTP-bound) state, dramatically stimulated the filamentous

properties of this organism (Gimeno *et al.* 1992). Altogether, four signaling pathways that regulate filamentous growth have been well characterized—rat sarcoma/protein kinase A (RAS/PKA), sucrose nonfermentable (SNF), target of rapamycin (TOR), and MAPK. We discuss each in turn below but concentrate on the MAPK pathway because it raises intriguing questions regarding signaling specificity.

**Ras2/cAMP-PKA pathway and affiliated G-protein-coupled receptor Gpr1:** The discovery that RAS is involved in filamentous growth provided a genetic context for elucidating components of the molecular pathway that plays a role in that growth habit (Figure 3). Yeast encode two RAS genes, *RAS1* and *RAS2*. The *RAS2* gene is expressed at higher levels than *RAS1* and is responsible for the majority of Ras function (Kataoka *et al.* 1984). Ras2 associates with and activates adenylate cyclase, a membrane-associated enzyme that produces the second messenger cyclic adenosine monophosphate (cAMP) (Toda *et al.* 1985). The Fink lab proposed that the levels of cAMP are critical for the decision of whether or not cells undergo filamentous growth (Mosch *et al.* 1996). Indeed, overexpression of the gene encoding the phosphodiesterase Pde2 dampened filamentous growth and suppressed the hyperfilamentation induced by activated RAS (Ward *et al.* 1995). As for many eukaryotes, cAMP regulates the activity of a family of protein kinases, referred to as protein kinase A (PKA). Binding of cAMP to a regulatory subunit (in yeast Bcy1) releases PKA, activating its kinase activity. Budding yeast has three different PKAs, Tpk1, Tpk2, and Tpk3, which are ~75% homologous in their catalytic domains but differ in their N-terminal regions (Toda *et al.* 1987). Subsequent studies have shown that all three Tpks associate with the regulatory subunit Bcy1 (Pan and Heitman 1999) and that Ras2/cAMP activation of PKA is required for filamentous growth (Pan and Heitman 2002).

What roles do the three Tpk proteins play in filamentous growth regulation? A breakthrough came when it was discovered that deletion of each *TPK* gene caused different phenotypes with respect to filamentous growth. Deletion of *TPK2* abolished filamentous growth, whereas deletion of *TPK1* had no effect. Deletion of *TPK3* caused hyperfilamentous growth, suggesting that Tpk3 may function in an inhibitory capacity (Robertson and Fink 1998; Pan and Heitman 1999). The three Tpks induce different target genes that regulate diverse metabolic outputs ranging from trehalose metabolism to iron uptake (Robertson *et al.* 2000). Among the substrates of Tpk2 is the transcription factor Flo8. Phosphorylation of Flo8 by Tpk2 results in activation

a stream of water, and photographed again (bottom) to reveal invaded cells. Bar, 1 cm. (C) The single cell invasive growth assay (Cullen and Sprague 2000). Cells as in B were spread onto SC medium lacking glucose as a carbon source for 1 day. Bar, 10 μM. (D) Diploid pseudohyphal growth assay (Gimeno *et al.* 1992). Homozygous diploid versions of the strains described in B were examined on SLAHD (low nitrogen) medium. Bar, 50 μM.

Figure 2 The filamentous growth response. Several biological assays permit the evaluation of the filamentous growth response in yeast, using the Σ1278b strain background. (A) Haploid wild-type (left) and *flo11* mutant (right) colonies grown on YEPD + 4% agar medium for 7 days show the Flo11-dependent colony ruffling. Bar, 0.5 cm. (B) The plate-washing assay (Roberts and Fink 1994). Haploid wild-type (left) and MAPK pathway mutant (right) cells were spotted onto YEPD medium (2% agar). After 3 days the plate was photographed (top), washed in

transient rise in cAMP levels. High-copy *GPA2* enhanced this rise in cAMP levels (Nakafuku et al. 1988; Papasavvas et al. 1992). Second, the glucose-induced increase in cAMP levels was inhibited by the mating pathway, which was mediated in some way through *Gpa2* and *Ras2* (Arkinstall et al. 1991; Papasavvas et al. 1992). On the basis of these observations, the Heitman lab reasoned that *Gpa2* might regulate RAS-dependent filamentous growth, and they showed that *gpa2/* *gpa2* homozygous mutant diploid cells are indeed defective for filamentous growth. Using both gain- and loss-of-function alleles of *GPA2*, in combination with various *RAS* alleles, a model has emerged in which *Gpa2* and *Ras2* converge on regulating adenylate cyclase (Figure 3) (Kubler et al. 1997; Lorenz and Heitman 1997). Adding to this finding is the observation that the glucose-dependent rise in cAMP levels is mediated specifically through *Gpa2* (Colombo et al. 1998), suggesting provocatively that *Gpa2* might function in some manner through a type of sugar receptor.

The involvement of a Gα subunit suggests obvious questions: what are the interacting partners (Gβ and Gγ), and what receptor associates with the G protein? These questions are relevant because the precise triggers of filamentous growth have been (until relatively recently) ill defined. Several labs independently identified a seven-transmembrane receptor of the β-adrenergic type that associates with *Gpa2* called *Gpr1* (Yun et al. 1997; Xue et al. 1998; Kraakman et al. 1999). Together with *Gpa2*, *Gpr1* regulates filamentous growth (Lorenz et al. 2000b). *Gpr1* is thought to be a sugar-sensing receptor (specifically sucrose) not a sugar transporter (Thevelein and Voordeckers 2009). The strongest evidence in support of this claim comes from Thevelein and coworkers, who used cysteine scanning mutagenesis of *Gpr1* to identify potential sites of sucrose binding (Lemaire et al. 2004). Hence, *Gpr1* functionally resembles the glucose sensor *Rgt2/Snf3* (Ozcan et al. 1996, 1998). A rigorous test to prove that *Gpr1* binds to sucrose would be a ligand-binding assay using radiolabeled sucrose. This type of experiment has not been reported for either *Gpr1* or *Rgt2*. In this and subsequent investigations it was also shown that sucrose may stimulate the filamentation response (Lemaire et al. 2004; Van de Velde and Thevelein 2008). There is some controversy surrounding this issue. In contrast to the report that the glucose-dependent rise in cAMP levels is thought to be mediated directly through *Gpa2* (Colombo et al. 1998), it has subsequently been shown that glucose induces GTP binding to *Ras2* independently of *Gpr1* and *Gpa2* (Colombo et al. 2004).

Two Gβ subunits were subsequently identified (*Gpb1* and *Gpb2*) that inhibit *Gpr1* (Harashima and Heitman 2002; Batlle et al. 2003; Peeters et al. 2006). *Gpb1/2* do not have the seven WD-40 repeats typically found in Gβ subunits but instead contain seven kelch repeats, a related protein-protein interaction motif, that results in the formation of a seven-bladed β-propeller structure typical of Gβ subunits (Harashima and Heitman 2002). *Gpa2* (Gα) interacts with *Gpb1/2* (Gβ) and with *Gpg1* (Gγ) (Harashima and Heitman

Figure 3 The RAS/PKA pathway. The G-protein coupled receptor (GPCR) *Gpr1* and its associated heterotrimeric G protein regulate the *Ras2* GTPase activating proteins (GAPs), *Ira1* and *Ira2*. *Ras2* regulates adenylate cyclase, which produces cAMP. cAMP binds to *Bcy1*, inactivating the protein, and releasing *Tpk1*, *Tpk2*, and *Tpk3* to activate *Flo8* and other targets that contribute to nutrient-regulated filamentous growth. Filled hexagons represent sucrose and other sugars.

of *Flo8* and expression of filamentation target genes (Pan and Heitman 1999).

The PKA pathway regulates filamentous growth not only by regulating the transcription factor *Flo8*, via *Tpk2*, but also by regulating the dual-specificity tyrosine-regulated kinase (DYRK) *Yak1*, via *Tpk1* (Figure 3). *Yak1* has a positive role in regulating filamentous growth (Zhang et al. 2001). Specifically, *Yak1* is phosphorylated by *Tpk1*, which inactivates the protein (Deminoff et al. 2006). In its nonphosphorylated (active) form, the protein regulates *FLO11* expression through the transcription factors *Sok2* and *Phd1* (Figure 3) (Malcher et al. 2011).

What are the upstream regulators of the RAS/PKA pathway? The search for upstream regulators of the RAS pathway has led to the identification of a G-protein-coupled receptor (GPCR) *Gpr1* (see below) and glucose limitation as key triggers of filamentous growth. The yeast genome encodes two Gα subunits, *GPA1*, which functions in the mating pathway, and *GPA2*, which was identified by homology to mammalian Gα subunits (Nakafuku et al. 1988). Several observations connect the Gα *Gpa2* to *Ras2*. First, the addition of glucose to glucose-starved cells causes a rapid but
2002). The idea that Gpb1/2 are the Gβ subunits for Gpa2 is not universally accepted, as recently reviewed by Peeters *et al.* (2007). For one thing, Gpb1/2 do not associate with Gpa2 at the switch interface regions, which is where the classical Gβ subunits bind to Ga subunits (Niranjan *et al.* 2007). Furthermore, another candidate Gβ subunit has been identified, Ascl, which contains the seven WD-40 repeats typically found in Gβ subunits (Zeller *et al.* 2007).

Mutants lacking Gpr1 or Gpa2 are defective for cAMP production and filamentous growth; thus, they are positive factors in controlling filamentous growth. Mutants lacking Gpb1/2 on the other hand are hyperfilamentous, which indicates that the Gβ subunits play an inhibitory role in pathway activation (Harashima and Heitman 2002; Batlle *et al.* 2003). This inhibition can be explained by the formation of an inactive Ga–Gβ complex, based on genetic observations that gpa2 gpb1 gpb2 triple mutants are less hyperinvasive (Harashima and Heitman 2002). Interestingly however, the triple mutant is somewhat hyperinvasive, indicating that the inhibitory role of Gβ is also mediated by interaction with an as yet unidentified factor (Harashima and Heitman 2002). The Heitman lab reasoned that Gβ mediates its inhibitory effect through the RAS pathway, on the basis of the fact that the hyperfilamentation of Gβ mutants was fully suppressed by deletion of Tpk2 (Harashima and Heitman 2002).

What is the connection between the GPCR and the RAS pathway? In an elegant study, genetic epistasis analysis showed that Gβ functions at the same level in the pathway as Ras2 (Harashima *et al.* 2006). In further support of the Ras2/GPCR connection, the two Ras2 GTPase activating proteins (GAPs), Ira1 and Ira2, were identified as Gpb1/2 interacting proteins by mass spectrometry (Harashima *et al.* 2006). Gpb1/2 associates with Ira1/2, resulting in inhibition of the Ras2 GTPase (Harashima *et al.* 2006). What is the effect of the association between Gpb1/2 with Ira1/2? The answer to this question is under some contention. In one report, Gpb1/2 are thought to associate with and stabilize Ira1/2 (Harashima *et al.* 2006), whereas, more recently, it has been proposed that Gpb1/2 target Ira1/2 for degradation (Phan *et al.* 2010). The resolution of these two opposing models will have important implications for understanding how the pathway regulates Tpk activity (Figure 3). A related discrepancy is the connection between nutrition and Ras2/Tpk2 signaling. If Gpr1 is a sucrose sensor, then does binding to sugars activate or repress Tpk2 activity? A tool that might be useful in addressing these questions is transcriptional reporters for Tpk target genes.

Snf1 pathway: Depletion of fermentable carbon sources, like glucose, can also trigger the filamentous growth response. The finding that glucose depletion is a trigger for filamentous growth came from observations from our lab, in experiments to define the stimuli that regulate the response. By removing and adding back various nutrients and examining the effects on cell and colony morphology, we showed that depletion of fermentable carbon sources, like glucose,

triggers filamentous growth (Cullen and Sprague 2000). To determine how glucose levels feed into filamentous growth regulation, several established nutrient-sensing pathways were examined, which uncovered a role for the protein kinase Snf1 in regulating filamentous growth (Cullen and Sprague 2000). Snf1 operates in a separate pathway from Gpr1, by regulating the repressors Nrg1 and Nrg2 at the FLO11 promoter (Kuchin *et al.* 2002; Vyas *et al.* 2003), a gene required for filamentous growth (see below for further discussion of FLO11). Nrg1 and Nrg2 function by recruitment of the Cyc8-Tup1 complex to promoters. Thus, two different glucose-sensing pathways, Gpr1/Gpa2/Ras2/PKA and Snf1, regulate filamentous growth in yeast.

TOR pathway: Initial observations of filamentous growth showed that limiting fixed nitrogen (specifically ammonia) is a trigger of filamentous growth (Gimeno *et al.* 1992). Specifically, mutants defective for ammonium transport were hyperfilamentous, which suggests that ammonium starvation might be a trigger for filamentous growth (Gimeno *et al.* 1992). In addition, Lorenz and Heitman (1998b) showed that the high-affinity ammonium transporter Mep2 is required for filamentous growth. The filamentation defect of the mep2 mutant arises apparently not from a defect in ammonium transport, as one might expect, but rather from a specific role for that transporter in communicating a signal through a small region in its cytosolic domain. The signal may be conveyed through a mechanism that is not well understood via the MAPK pathway (Rutherford *et al.* 2008). Nitrogen signals have subsequently been shown to be interpreted by the TOR pathway (Crespo *et al.* 2002), an evolutionarily conserved nutrient-regulatory pathway (Heitman *et al.* 1991). The serine/threonine protein kinase TOR regulates cellular homeostasis by coordinating metabolic processes with cellular nutrient levels (Sengupta *et al.* 2010). The TOR pathway regulates the transcription factor Gcn4, which is a regulator of FLO11 expression (Braus *et al.* 2003; Boeckstaens *et al.* 2008). The TOR pathway regulates filamentous growth in a manner that is apparently independent of the RAS/PKA and MAPK pathways. Evidence for this conclusion comes from the fact that rapamycin inhibits filamentous growth under nitrogen-limited conditions, an inhibition that is mediated by the TOR pathway phosphatases Tap42 and Sit4 (Cutler *et al.* 2001).

To summarize, limiting for nitrogen or glucose can induce filamentous growth. The fact that the glucose response was first observed in haploid cells (invasive growth, Figure 2, B and C) and the nitrogen limitation response first characterized in diploid cells (pseudohyphal growth, Figure 2D) may have led to the impression that the different cell types respond to different stimuli. In fact, glucose depletion induces filamentous growth in both haploid and diploid cells (Cullen and Sprague 2002; Kuchin *et al.* 2002), and nitrogen limitation also induces filamentous growth in both cell types (P. J. Cullen and G. F. Sprague, unpublished observations). How different are

P. J. Cullen and G. F. Spraguehaploid and diploid cells with respect to filamentous growth? The answer to this question is complicated: relatively few studies directly compare filamentous growth in the two cell types, and different assays are used for haploid (Figure 2, C and D) and diploid cells (Figure 2E). A further complication comes from incongruous results. It was initially reported that haploid cells undergo invasive growth better than diploid cells (Roberts and Fink 1994), although we found the opposite to be true (Cullen and Sprague 2002). Nevertheless, the expression of filamentation target genes is regulated by different stimuli in haploid compared to diploid cells (Lo and Dranginis 1998), and regulatory pathways Ras2/PKA and MAPK (discussed below) have different roles in regulating the response in haploid and diploid cells (Chen and Thorner 2010).

At this point, an important paradox should be discussed. One the one hand, glucose limitation induces filamentous growth in both haploid and diploid cells. Indeed, cells grown in nutrient-rich (high glucose) conditions do not produce pseudohyphae. But on the other hand, as stated above, glucose/sucrose is required for filamentous growth in a Gpr1-dependent manner. What is the basis for this discrepancy? Although this point has not been explicitly addressed in the literature, there are several possibilities. One is that glucose/sucrose is required for pseudohyphal growth in diploid cells enduring a low-nitrogen stress, the conditions used by the Thevelein group, to establish the requirement. A less interesting alternative is that different strains are sensitized to different nutritional requirements.

Diploid cells starved for both nitrogen and glucose undergo sporulation, which raises an important point: how do cells decide whether to undergo filamentous growth, enter stationary phase, or sporulate in response to limiting nutrients (Figure 1)? Sporulation has been extensively studied in yeast (Neiman 2011), and many of the signals that trigger meiosis and spore formation are well characterized (Engelbrecht 2003). One protein that controls the sporulation/filamentation decision is the repressor of meiosis Rme1 (van Dyk et al. 2003). Rme1p is a zinc-finger type transcriptional factor that promotes the mitotic/meiotic decision (Mitchell and Herskowitz 1986). Rme1 binds directly to the FLO11 promoter to induce cell–cell adhesion and invasive growth (van Dyk et al. 2003). Given that Rme1 is not regulated by Ras2 or the MAPK pathway (van Dyk et al. 2003), presumably other pathways regulate Rme1-induced filamentous growth. Regulators of the sporulation pathway, Ime1 and Ime2, are also required for filamentous growth (Strudwick et al. 2010), although this is true only in the SK1 genetic background. Even in that background, the requirement for Ime2 is extremely weak. Neither protein is required for agar invasion by haploids, but rather only for colony morphology changes shown by diploids (Strudwick et al. 2010).

Other sensory pathways: Several other metabolites have been identified that influence filamentous growth. One is alcohol byproducts like 1-butanol (Dickinson 1996; Lorenz et al. 2000a). Response to alcohols has now been identified as a quorum-sensing behavior. Budding yeast undergo filamentous growth in response to cell density using secreted alcohols as a gauge of its population levels (Chen and Fink 2006). Quorum sensing also occurs in *C. albicans* via sensing different secreted alcohol derivatives (Chen et al. 2004). An intact respiratory pathway, as mediated by a signaling pathway referred to as the retrograde pathway (Butow and Avadhani 2004), also regulates filamentous growth (Jin et al. 2008b). Several other metabolites that induce filamentous growth have also been identified, including tetrahydrofolate (vitamin B9). B9 levels feed into *FLO11* expression through signaling mechanisms that have not been well characterized (Guldener et al. 2004). External pH may also be sensed in some manner through a signaling pathway that regulates the transcription factor Rim101 (Lamb and Mitchell 2003).

### The filamentation MAPK pathway: expeditions into signaling specificity

Early studies from the Fink lab uncovered two signaling pathways that regulate filamentous growth. As discussed above, one major pathway is the Ras2 pathway. The other major pathway is a MAPK pathway composed of kinases that also function in the mating or pheromone response pathway (Figure 4A). The logic underlying testing for a role for the MAPK pathway in filamentous growth was that elements of the pheromone response pathway are expressed in diploid cells, even though diploid cells do not mate. What might the pathway’s function in diploids be? Liu et al. (1993) reported that four proteins required for mating in haploid cells, the p21-activated (PAK) kinase Ste20, the MAPKKK Ste11, the MAPKK Ste7, and the transcription factor Ste12 (Figure 4A), are also required for filamentous growth in diploids. In contrast, the genes encoding the pheromone receptors Ste2/Ste3, the associated heterotrimeric G protein (Gpa1, Ste4, and Ste18), and the MAPK Fus3 are not required for filamentous growth in diploids (or haploids). Thus, it seemed that haploid cells utilize the “core module” of Ste20 → Ste11 → Ste7 → Ste12 for mating, whereas diploid cells utilize that same core module for filamentous growth regulation (Figure 4A).

Although the separation of function by cell type seems a tidy way to establish specificity, the tidiness is superficial and specificity questions loom large. First, the transcription factor Ste12 functions in both pathways. How are different gene sets activated in mating and filamentous growth? Second, it soon became apparent that haploid cells execute a similar filamentous growth program that requires the same core module (Roberts and Fink 1994). An even more fundamental question therefore is how does the same module direct two distinct physiologic programs in the same cell type?

An example of this quandary comes from studies of the global regulatory Rho-family GTPase Cdc42 (Park and Bi
A

Stimulus:
↓
Receptor:
↓
G-protein/Adaptor:
↓
GEF:
↓
Rho GTPase:
↓
PAK:
↓
Adaptor:
MAPKKK:
↓
MAPKK:
↓
MAPK:
↓
Txn Factor:
↓
Response:

Pheromone
Nutrient
High
Limitation?
Osmolarity

Ste2/Ste3
Msb2
Msb2 + Hkr1

Gαβγ
Sho1+Opy2

Cdc24
Cdc42
Ste20
Ste50
Ste11

Fus3
Ste7
Kss1
Pbs2
Hog1

Ste12
Ste12+Tec1
Hot1

Mating
Filamentous
Osmotic
Growth
Tolerance


B

Mating
Filamentous Growth
HOG


Figure 4 Three MAPK pathways in yeast share common components and also contain pathway-specific factors. (A) Three MAPK pathways are shown. Colored proteins represent pathway-specific factors; protein shown in black function in multiple pathways. Scaffold-mediated interactions are shown by colored, dashed lines. Not all protein interactions are shown. Hot1 is one of a number of transcription factors for the HOG pathway. The red question mark indicates that how nutritional signals feed into filamentous growth pathway regulation is not well understood. (B) Examples of MAPK morphogenesis in yeast. The pheromone response (Mating) pathway induces distinctive polarized structures called shmoos to promote cell fusion and diploid formation. The Filamentous Growth pathway induces filamentous growth, branched chains of elongated and connected cells. Activation of the HOG pathway does not induce polarized growth. Bar, 5 μm.

et al. 1995). Although it was known that temperature-sensitive mutations in CDC42 were defective for mating (Reid and Hartwell 1977), the assumption was that this resulted from a defect in the overall cell polarity. However, the studies of Simon et al. (1995) suggested a more direct involvement of Cdc42 in the mating pathway. The salient finding was that temperature-sensitive versions of Cdc42 and its guanine nucleotide exchange factor (GEF) Cdc24 were defective in MAPK signaling, as assessed by a pheromone-inducible transcriptional reporter. Cdc42 associates with the PAK Ste20, based on two-hybrid analysis and in vitro pull downs using recombinant proteins (Simon et al. 1995; Zhao et al. 1995; Peter et al. 1996). More recently, it was shown that Cdc42 and Ste20 function in both the mating pathway and the filamentation pathway (Figure 4A) (Peter et al. 1996; Leberer et al. 1997), again raising the question of how specificity among MAPK pathways is achieved.

The depth of this puzzle has been magnified by the fact that some of the common or shared components function in yet another MAPK pathway. The Saito lab showed that elements of that same core module—Cdc42, Ste20, and Ste11 (Figure 4A)—are required to activate one of the branches of the high osmolarity glycerol response (HOG) pathway. The HOG pathway responds to changes in external osmolarity caused by exposure to media containing salt, sugar, and other small molecules (Posas and Saito 1997; O'Rourke and Herskowitz 1998; Raitt et al. 2000; Tatebayashi et al. 2006; Hohmann et al. 2007). As another example, the Ste11-interacting protein Ste50 also functions in all three MAPK pathways (Figure 4A) (Posas et al. 1998; Ramezani-Rad 2003; Tatebayashi et al. 2006; Truckses et al. 2006; Wu et al. 2006).

Hence, a common or core module regulates the expression of nonoverlapping target genes (Roberts et al. 2000) and evokes distinct morphogenetic responses depending on the stimulus. Visual inspection of cells illustrates this point. Nutrient limitation induces filamentous growth (Figure 4B). Mating pheromone induces shmoos formation (Figure 4B). Activation of the HOG pathway does not induce a morphological change (Figure 4B). Indeed, external osmolarity causes rapid depolymerization of the actin cytoskeleton (Yuzyuk et al. 2002; Yuzyuk and Amberg 2003), which might be expected to prevent cell polarization during mating and filamentous growth. The question of whether cells can simultaneously activate multiple pathways in response to multiple stimuli has been examined. Cells challenged simultaneously with pheromone and salt activate either one or the other pathway but not both (McClean et al. 2007), although this finding may represent an oversimplification of the true decision-making response (Patterson et al. 2010).

These discoveries raise important questions: (1) What is the MAPK for the filamentation pathway? (2) How is specificity maintained between kinases that function in multiple pathways? (3) What is the receptor for the filamentation MAPK pathway? Answering these questions

2007). Cdc42 is an essential protein that is required to establish cell polarity (Bender and Pringle 1989; Adams et al. 1990; Shimada et al. 2004; Gao et al. 2007; Tong et al. 2007 and references therein). It has subsequently been shown that Cdc42 functions in the mating pathway and is required for transduction of the signal initiated by the GPCR (Simon
is relevant to filamentous growth regulation and to understanding how signaling pathways maintain specificity. Given that signaling pathways in diverse organisms share common components, insights in this area may shed light on the general mystery of signaling pathway insulation. As will be seen in the discussion below, the quest to understand signaling specificity has repeatedly identified mechanisms or new pathway components that were thought to confer specificity. However, in most cases, the apparent solution was short lived. Further studies often showed that the specificity problem remained, and it is fair to say that fundamental questions regarding specificity still exist. The discussion below summarizes the history of the quest and highlights the extant questions.

**Kss1 is the MAP kinase for the filamentation pathway:** Two MAP kinases (Fus3 and Kss1) were discovered around the same time in genetic screens for regulators of the mating pathway. The protein kinase Fus3 was established early on as a regulator of the mating pathway, because it was required for pheromone-induced growth arrest, and because its overexpression resulted in heightened sensitivity to pheromone (Elion et al. 1990). However, *fus3* mutants showed only a partial mating defect, suggesting other proteins could carry out Fus3 function. The protein kinase Kss1 was identified as a high-copy suppressor of the cell-cycle arrest phenotype induced by pheromone (Courchesne et al. 1989), which suggested that it might function in opposition to the mating pathway. Indeed, *kss1* mutants showed elevated growth arrest in response to pheromone (Courchesne et al. 1989) and normal or slightly elevated *FUS1* expression (Elion et al. 1991b). Nevertheless, *fus3 kss1* double mutants were completely deficient for mating, implying that the two kinases function redundantly in the mating pathway.

Despite the above results, it was also suspected that Kss1 might be the filamentation MAPK. The *kss1* mutant had a strong invasive growth defect (Roberts and Fink 1994) and showed reduced activity of a filamentation response element (FRE) (Mosch et al. 1996). The breakthrough in assigning functions to Fus3 and Kss1 came from observations that were at first paradoxical. Deleting *FUS3* restored invasive growth to the *kss1* mutant (Roberts and Fink 1994). Moreover, Thorner and colleagues found that *ste7 fus3 kss1* triple mutants invaded the agar as well as wild-type cells (Cook et al. 1997). This new finding flew in the face of the established result that the MAPKK Ste7 was required for invasive growth. Thorner and colleagues reasoned that Fus3 and Kss1 (mainly Kss1 from genetic evidence) had an inhibitory function in filamentous growth, and that Ste7 was required to relieve that inhibition. A dual role for Kss1 in MAPK regulation could be explained by changes in its phosphorylated (active) state. Unphosphorylated Kss1 functions as an inhibitor, whereas phosphorylated Kss1, catalyzed by Ste7, functions as an activator (Cook et al. 1997).

In a parallel study, Madhani, Fink and colleagues corroborated these findings by showing that the inhibitory effect of Kss1 was mediated through the transcription factor Ste12 (Madhani et al. 1997). Using kinase-inactive versions of the Kss1 and Fus3 proteins, which maintained protein-protein interactions with their respective factors and thereby prevented cross-talk, they showed that Kss1 functions in the mating pathway only when Fus3 is absent (Madhani et al. 1997). One conclusion from these two studies is that Fus3 is the MAPK for the mating pathway, whereas Kss1 is the MAPK for the filamentous growth pathway.

It should be recognized that Kss1 also plays a role in the mating pathway. Cells lacking the mating pathway MAPK Fus3 can mate (Elion et al. 1991a), and pheromone induces the phosphorylation/activation of Fus3 and Kss1 to a similar degree (Gartner et al. 1992). Specificity between the pathways may involve transient vs. sustained pathway signaling (Sabbagh et al. 2001; Bruckner et al. 2004) rather than the more simplistic view that each MAPK pathway has its own MAPK. In this way, Kss1 can also be viewed as a shared component between the mating and filamentous growth pathways (Figure 4A).

How does Kss1 mediate its inhibitory function? To begin to answer this question, two-hybrid analysis (Fields and Song 1989) was performed using Kss1 as bait. In addition to identifying the transcription factor Ste12, two novel proteins were identified, Dig1 and Dig2 (Cook et al. 1996). Biochemical tests confirmed that Kss1 and Ste12 associate with Dig1 and Dig2, and invasive growth assays showed that the *dig1* and *dig2* mutants were strongly hyperinvasive, demonstrating that the proteins were potent negative regulators of filamentous growth. Kss1 phosphorylates Dig1 and Dig2, which suggests a mechanism for relieving the inhibitory effects of these transcriptional repressors (Cook et al. 1996). Dig1 and Dig2 also associate with Fus3 and function as negative regulators of the mating pathway (Tedford et al. 1997; Roberts et al. 2000). Subsequent experiments showed that Dig1/2 function in pathway discrimination by conferring differences in the binding to mating (Ste12) and filamentation (Ste12 and Tec1, see below) promoters (Bardwell et al. 1998).

The transcription factor Ste12 functions in both the mating and filamentation pathways (Figure 4A). How does a transcription factor induce one set of target genes in one setting (pheromone) and a different set of targets in another (nutrient limitation)? One possibility is that Ste12 associates with a protein that specifies it to filamentation regulated genes. The transcription factor Tec1 was identified as a member of the TEF-1, Tec1p, and AbaAp (TEA) or AbaAp, TEF-1, Tec1p, and Scalloped (ATTS) family that coregulates the expression of transposable elements along with Ste12 (Laloux et al. 1994). Tec1 was also required for filamentous growth (Gavrias et al. 1996). Madhani and Fink (1997) interpreted these findings to indicate that Tec1 may be the coregulator of Ste12 function. Support for their hypothesis came from the finding that purified versions of Ste12 and

Filamentous Growth Regulation 31Tec1 bind cooperatively to FREs. The distribution of Ste12 and Tec1 at target promoters *in vivo* largely bears out the hypothesis that Tec1 and Ste12 exhibit combinatorial control over filamentation pathway targets (Kohler et al. 2002; Zeitlinger et al. 2003; Chou et al. 2006).

Given that Tec1 specifies Ste12 to filamentation-specific targets, one might expect that TEC1 is not expressed during the mating response. Unexpectedly, the gene encoding the Tec1 protein is induced by pheromone (Oehlen and Cross 1998). But paradoxically, immunoblot analysis showed that the Tec1 protein is not present in cells exposed to mating pheromone. Therefore, a mechanism for regulating the levels of the Tec1 protein must exist in mating cells. Such a mechanism was identified and has come to represent a fundamental way of maintaining specificity between pathways. In response to pheromone, the activated MAPK Fus3 phosphorylates Tec1. Phosphorylated Tec1 is recognized by a ubiquitin ligase that targets Tec1 for degradation by the proteasome (Bao et al. 2004; Bruckner et al. 2004; Chou et al. 2004). Failure of Tec1 to be degraded results in cross-talk between the filamentation and mating pathways. Tec1 is subject to complex regulation, being phosphorylated at multiple residues (Bao et al. 2010) as well as being sumoylated (Wang et al. 2009b). Ste12 itself and other components of the mating/filamentation pathways are also ubiquitinated and degraded to attenuate signaling generated by these pathways (Esch et al. 2006). Persistence of Ste12, for example, can lead to a shift in filamentation over the mating response (Esch et al. 2006). Among the proteins that may regulate the turnover of Ste12 is the CDK Srb10/Cdk8 (Nelson et al. 2003). Therefore, the regulated degradation of pathway-specific proteins can result in signal discrimination.

### Scaffolding proteins insulate signaling by proteins in the core module:

Genetic screens identified a number of STE genes as encoding potential mating pathway components. Despite rigorous genetic epistasis analysis to order components into the pathway, the function of several Ste proteins had remained elusive. One of these was Ste5, a large protein whose amino acid composition suggested little of its function. To determine how Ste5 regulates the mating pathway, a directed two-hybrid approach was employed. Using this approach, three labs independently made an important discovery—Ste5 associates with multiple components in the MAP kinase cascade (Choi et al. 1994; Kranz et al. 1994; Marcus et al. 1994; Printen and Sprague 1994). Elion and colleagues expanded on this exciting finding by showing that Ste5 physically associates with Ste11, Ste7, and Fus3. They also showed that Ste5 is required for Ste11 function in the mating pathway (Choi et al. 1994). Together these studies establish Ste5 as a scaffold for the mating pathway.

In addition to its interaction with MAPK pathway kinases, Ste5 associates with the heterotrimeric Gβ (Ste4) subunit for the mating pathway (Whiteway et al. 1995). Gβ also associates with the PAK Ste20 (Leeuw et al. 1998), which thereby connects upstream signals to both the PAK and the MAPK scaffold. Ste5 functions in the cell as a dimer (Yablonski et al. 1996; Inouye et al. 1997; Feng et al. 1998) and is recruited to the plasma membrane upon binding of pheromone to receptor (Pryciak and Huntress 1998). Hence, one has a picture of a multiprotein complex localized to the site of ligand-bound receptors controlling the entire mating pathway.

The identification of a scaffold has implications about how signaling through the pathway might be regulated. How does Ste5 in fact contribute to pathway specificity? First, Ste5 promotes the interaction among kinases to increase the efficiency of signal transmission (and signal attenuation) (Choi et al. 1994). Thus, Ste5 might tether general components (Ste11 and Ste7) to a pathway-specific factor (Fus3) to prevent erroneous signaling of the upstream kinases.

Second, Ste5 selectively recruits proteins to the plasma membrane (Pryciak and Huntress 1998; van Drogen et al. 2001; Maeder et al. 2007). Plasma-membrane recruitment is a general way of increasing the local concentration of protein complexes. Ste5 is recruited to the plasma membrane in response to pheromone (Pryciak and Huntress 1998; van Drogen et al. 2001; Maeder et al. 2007; Yu et al. 2008) through its PM domain (Winters et al. 2005), its PH domain (Garrenton et al. 2006), and by Gβ recruitment (Pryciak and Huntress 1998; Mahanty et al. 1999; Winters et al. 2005). Directly targeting Ste5 to membranes substantially activates MAPK signaling (Pryciak and Huntress 1998). The PH and PM domains of Ste5 associate with phosphorylated inositol lipids in the plasma membrane (Winters et al. 2005; Garrenton et al. 2006). Specifically, PI(4,5)P2 is highly enriched in shmoo tips, which results in the polarized localization of Ste5 and associated proteins to that site (Jin et al. 2008a; Garrenton et al. 2010). In cells not exposed to pheromone, Ste5 is localized to the nucleus (Pryciak and Huntress 1998; Mahanty et al. 1999), where it is degraded by ubiquitin-mediated proteolysis by the proteasome (Garrenton et al. 2009).

Third, Ste5 causes a conformational change in Fus3 that makes it competent to be phosphorylated by Ste7 (Flatauer et al. 2005; Good et al. 2009). The way in which Fus3 and Kss1 are activated by Ste7 is fundamentally different. On the one hand, Ste7 recognizes a specific docking site in the CD/sevenmaker region of Kss1 and Fus3 that is common to MAPKs of many different species and that promotes interactions with key regulatory proteins (Kusari et al. 2004). On the other hand, whereas Ste7 readily phosphorylates Kss1, it cannot phosphorylate Fus3 without Ste5. Evidence comes in part from the fact that hyperactive versions of Ste7 induce invasive growth and Kss1 phosphorylation but not mating and Fus3 phosphorylation (Maleri et al. 2004). Ste5 contributes to specific activation of the mating pathway in response to pheromone (Flatauer et al. 2005). Induced-fit recognition between Ste7, Fus3, and Kss1 may allow docking peptides to achieve discrimination through differences in kinaseflexibility (Remenyi et al. 2005). Pathway-specific activation of different MAPK pathways also involves differences in pathway kinetics (Sabbagh et al. 2001). Kss1 induces a transient response, whereas Fus3 induces a sustained response. Studies stemming from the crystal structure determination of Fus3 have shown that Ste5 functions to unlock the Fus3 kinase for phosphorylation by the MAPKK Ste7 (Good et al. 2009). In addition, activated Fus3 (by phosphorylation) exists in a gradient, concentrating at the shmoo tip (Maeder et al. 2007).

Pbs2, the MAPKK for the HOG pathway (Brewster et al. 1993), is also thought to provide an example of a scaffold (Figure 4A). In a landmark study, Saito and colleagues identified the MAPKKK Ste11 as being required to transmit a signal in the HOG pathway (Posas and Saito 1997). Ste11 activates the HOG pathway by associating with and activating Pbs2. Pbs2 also associates with the cell-surface protein Sho1 (Maeda et al. 1995). Pbs2 therefore functions as both the scaffold and MAPKK for the HOG pathway.

To summarize, pathway-specific complexes for MAPK pathways can be constructed from general factors by pathway-dedicated scaffolds. Ste5 promotes Ste11 function in the mating pathway, whereas Pbs2 promotes Ste11 function in the HOG pathway. This overall picture is satisfying but may represent only part of the actual connections that underlie pathway specificity. For one thing, a scaffold for the filamentation pathway, although theorized, has yet to be identified (Saito 2010). Therefore, other specificity factors may also contribute to signal insulation. For another thing, the protein interactions depicted here probably represent an oversimplified view. A more accurate picture, but harder to visualize, is that proteins in these pathways exist in multi-protein complexes. For example, Sho1 associates directly with Ste11 (Zarrinpar et al. 2004; Tatebayashi et al. 2006) and with Ste50 (Tatebayashi et al. 2006), suggesting that like Pbs2, it may also serve a scaffolding role. Whether such complexes are contiguous or whether there are multiple different protein subcomplexes in the cell remains to be determined.

### Proteins that function at the head of the MAPK pathway:

The discovery that haploid cells require a core module for mating and filamentous growth implies that different receptors activate the two pathways. The receptor for the mating pathway is a seven-transmembrane heterotrimeric GPCR. But the receptor for the filamentation MAPK pathway—any protein that functions above Ste11—had not been identified.

A cell-surface regulator for the HOG pathway had been identified, called Sho1 (Maeda et al. 1995). Sho1 contains four transmembrane helices and a cytosolic SH3 domain. In the HOG pathway, Sho1 associates with the MAPKK Pbs2 by a SH3 domain–polyproline domain interaction. In a pioneering study, O'Rourke and Herskowitz (1998) reasoned that Sho1 might also function in the filamentation pathway. Their reasoning was motivated by the discovery that cross-talk occurs from the HOG pathway to the pheromone pathway.

---

**Figure 5** Model of the filamentous growth MAPK pathway. Upon nutrient limitation, expression of the gene encoding the aspartyl protease Yps1 is induced. Yps1 processes the signaling mucin Msb2 in its extracellular domain, which is required for MAPK activation. Processed Msb2 (Msb2*) associates with and functions through Sho1 to activate cytosolic signaling modules. Msb2 associates with the Rho GTPase Cdc42, and Sho1 functions in a complex with the GEF Cdc24. A straightforward possibility is that the association of activated Msb2 with Sho1 brings the GEF into close proximity with its GTPase. Activated Cdc42 binds effector proteins including the PAK Ste20, which when activated, phosphorylates the MAPKKK Ste11, thereby activating the MAPK cascade.

way in cells lacking downstream components of the HOG pathway, Pbs2 or Hog1. For example, the addition of salt to hog1 or pbs2 mutants caused activation of the mating pathway. Activation required Sho1, Ste20, and the MAPK cascade. Cross-talk was not dependent on the mating receptor, the heterotrimeric G protein (Ste4), or Ste5. Thus, Sho1 could be functionally connected to the Ste12-dependent MAPK pathway in some contexts. Further support for the identification of a Sho1→Ste12 pathway came from analysis of an “amalgamated” pathway that functioned in protein glycosylation mutants (Cullen et al. 2000). Therefore, at least one protein that might function at the head of the filamentous growth pathway is Sho1 (Figure 5).

Although the studies summarized above have the pleasing virtue of identifying a membrane protein that operates in the filamentation pathway, they compound the specificity problem because Sho1 works in the HOG pathway as well (Figure 4A). In a study to identify new regulators of filamentous growth (Cullen et al. 2004), two high-throughput approaches were employed that began to shed light on filamentous growth pathway signaling and specificity. In one approach, DNA microarray analysis identified a small number of highly induced transcriptional targets of the filamentation pathway. In a second approach, phenotypic analysis of a collection of ordered deletion mutants identified factors required for invasive growth. Comparing the two datasets

Filamentous Growth Regulation 33revealed a single common protein, **Msb2**. The **MSB2** gene was previously identified as a high-copy suppressor of temperature-sensitive *cdc24* and *cdc42* alleles (Bender and Pringle 1992), and the protein had been implicated in HOG pathway regulation (O'Rourke and Herskowitz 2002). Moreover, the amino acid sequence of the **Msb2** protein suggested it was a large cell-surface glycoprotein, with an N-terminal signal sequence and large S/T-rich extracellular domain. Together these findings suggested that **Msb2** might be a candidate cell-surface regulator of **Cdc42** in the filamentation pathway.

Does **Msb2** regulate the filamentous growth pathway? Deletion of **MSB2** resulted in a defect in pathway activity equivalent to deletion of the **SHO1** gene. **Msb2** also formed a protein complex with **Sho1** and with **Cdc42**, preferentially the active (GTP-bound) form of Cdc42 (Cullen et al. 2004). Therefore, two proteins (**Msb2** and **Sho1**) function at the head of the filamentation MAPK pathway and interface with cytosolic regulators (like **Cdc42**) to transmit a signal to downstream components (Figure 5).

Most intriguingly, **Msb2** was not required to regulate the mating pathway and had a relatively minor role in the HOG pathway, making it the first cell-surface protein to be identified with a specific role in regulating the filamentation pathway (Cullen et al. 2004). A convincing piece of evidence that **Msb2** plays a specific role in the filamentation pathway was that the **MSB2** gene is a transcriptional target of the pathway. Many examples exist of pathway-specific components encoded by genes that are induced by activation of their cognate pathways, creating a positive-feedback loop. Positive feedback loops can lead to bistable activation states, a type of cellular “memory” (Ingolia and Murray 2007) that has been reported throughout eukaryotes (Xiong and Ferrell 2003).

**Msb2** is a member of the signaling mucin family of proteins, which are general regulators of MAPK pathways that have been identified in a variety of species. The most highly characterized signaling mucin is MUC1, which regulates the RAS–rapidly accelerated fibrosarcoma (RAF)-mitogen activated protein kinase kinase (MEK)-extracellular signal regulated kinase (ERK) pathway (Singh and Hollingsworth 2006). Signaling mucins are single pass transmembrane proteins that are glycosylated in their extracellular domains (Kufe 2009; Bafna et al. 2010). A defining feature is the presence of heavily glycosylated tandem repeats that are rich in proline, threonine, and serine residues (PTS domain). To determine whether the PTS domain of **Msb2** was required for its function in the filamentation pathway, that domain was disrupted and replaced with an epitope tag. Unexpectedly, **Msb2** lacking the PTS domain was hyperactive for MAPK activity, which suggested an inhibitory role for the PTS domain in signaling mucin regulation (Cullen et al. 2004).

Signaling mucins differ in their overall structure and regulation from the highly characterized and more well-understood GPCR-type receptors. Unlike GPCRs, signaling mucins have not been studied in model systems where genetic approaches are readily available. In mammalian cells, signaling mucins are shed by post-translational processing (Litvinov and Hilkens 1993; Parry et al. 2001; Brayman et al. 2004). In many cases, the proteases that process signaling mucins have not been identified (Parry et al. 2001), and the relationship between processing and mucin regulation remains unclear (Singh and Hollingsworth 2006). We found that most of the glycosylated extracellular domain of **Msb2** is shed from cells (Vadaie et al. 2008). Given that **Msb2** is encoded by a single polypeptide, we hypothesized that **Msb2** might be subject to proteolytic processing. Examining **Msb2** shedding in a panel of protease mutants uncovered the aspartyl protease **Yps1** as being required for processing and release of the protein (Vadaie et al. 2008).

Given that the PTS domain (698–818 residues) of **Msb2** is inhibitory, we further explored the extent of that inhibition. Deletion analysis showed that most of the extracellular domain (100–950 residues) had an inhibitory function. A version of **Msb2** lacking this large domain was strongly hyperactive for MAPK signaling (Vadaie et al. 2008). The finding that **Msb2** is processed and its extracellular inhibitory domain released from cells suggests an activation mechanism (Figure 5). Cleavage-dependent activation may be a general regulatory feature of signaling mucins.

An appealing aspect to defining **Msb2** as an upstream regulator of the filamentation pathway is that it provides an explanation for how specificity is achieved at the head of the pathway: **Msb2** is a protein that functions in filamentation but not mating or shmoo response. This model is not completely satisfying, however because **Msb2** might function in at least some capacity in the HOG pathway (O'Rourke and Herskowitz 2002). This tidy notion of Msb2 as a filamentation specific component was further challenged by the recent discovery by the Saito lab that a second signaling mucin, **Hkr1**, functions together with **Msb2** in the HOG pathway (Tatebayashi et al. 2007). That is, **Msb2** and **Hkr1** are redundant for function in the HOG pathway. The **HKR1** gene was identified in a genetic screen for mutants that were osmosensitive in an *msb2Δ* (and *ssk2Δ/ssk22Δ*) background. Saito and colleagues showed that both **Msb2** and **Hkr1** associate with **Sho1** to transmit a signal to downstream components (Tatebayashi et al. 2007). Therefore, **Msb2** cannot be thought of as solely functioning in the filamentation pathway (Figure 4).

Does **Hkr1** function in the filamentation pathway? To address this question, the role of **Hkr1** in regulating the filamentous growth pathway was tested (Pitoniak et al. 2009). Unlike for **Msb2**, cells lacking **Hkr1** were not defective for filamentous growth pathway signaling. Moreover, the genes encoding the two mucins exhibited different expression patterns, and their overproduction induced non-overlapping sets of target genes. Therefore a model can be drawn where **Msb2** functions preferentially in the filamentation pathway, whereas **Hkr1** functions preferentially in the HOG pathway (Pitoniak et al. 2009). This model is

P. J. Cullen and G. F. Spraguereinforced by the recent finding that underglycosylation of **Msb2** activates the filamentous growth pathway but not the HOG pathway (Yang *et al.* 2009). Examining the role of the two mucins in promoting the phosphorylation of downstream kinases would lend further support to this possibility. Hence, differential MAPK activation by signaling mucins represents a new point of discrimination between MAPK pathways.

An important unresolved question is what do signaling mucins “sense” to induce a downstream signal? This question is unanswered for any such mucin. Specifically, in yeast it remains unclear how nutritional information is sensed or conveyed through **Msb2/Sho1** or how a change in external osmolarity is sensed by **Msb2/Sho1/Hkr1**. **Msb2** may be a mechanoreceptor that monitors mechanical stress between the plasma membrane and cell wall during osmotic stress (O'Rourke and Herskowitz 2002). Intriguingly, the mammalian signaling mucin MUC1 has been proposed to detect mechanical shear (Macao *et al.* 2006).

The legitimacy of studies focused on mucin-like glycoproteins in yeast is further validated by the fact that most (three of the four) MAPK pathways in yeast for which cell-surface proteins have been identified are regulated by large mucin-like glycoproteins. **Msb2** functions in the filamentous growth pathway (Cullen *et al.* 2004), **Msb2/Hkr1** in the HOG pathway (Tatebayashi *et al.* 2007), and **Wsc1,Wsc2,Wsc3, Mid2**, and **Mtl1** (Rodicio and Heinisch 2010) in the cell wall integrity or protein kinase C pathway (Levin 2005). These glycoproteins are structurally and mechanistically dissimilar from GPCRs. The yeast pheromone receptors bind to well-defined peptide ligands, oligomerize (Gehret *et al.* 2006), exist in inactive and activated (ligand-bound) states (Boone *et al.* 1993; Stefan and Blumer 1994), are differentially internalized depending on whether or not they are bound to ligand and cleared from the cell surface by ubiquitination (Roth and Davis 1996; Tan *et al.* 1996; Jenness *et al.* 1997; Roth and Davis 2000; Chen and Davis 2002). It will be interesting to learn how mucin-like glycoproteins are regulated to modulate MAPK activation.

Yet another cell-surface component of the HOG pathway has recently been identified. The **opy2** mutant was uncovered in a synthetic genetic array (SGA) screen (Tong *et al.* 2001) by the Whiteway lab in a search for salt-sensitive mutants in an **ssk1** background (Wu *et al.* 2006). **Opy2** was initially postulated to function exclusively in the HOG pathway (Wu *et al.* 2006), although it has subsequently been suggested to operate in the filamentation MAPK pathway as well (Yamamoto *et al.* 2010). Two-hybrid analysis, in vitro pull down, and co-immunoprecipitation showed that **Opy2** associates with the adapter protein **Ste50** (Wu *et al.* 2006; Ekiel *et al.* 2009). **Ste50** associates with **Cdc42** (Truckses *et al.* 2006) and **Ste11** (Posas *et al.* 1998; Jansen *et al.* 2001; Tatebayashi *et al.* 2006; Truckses *et al.* 2006; Garcia *et al.* 2009) and is thought to function in the membrane recruitment of **Ste11** to activated complexes at the cell surface. Evidence supporting this conclusion comes from the

fact that membrane tethering of **Ste11** can bypass the requirement for **Ste50** (Wu *et al.* 2006).

Why do different MAPK pathways require the same core module? The answer to this question is not known but hints may come from an antagonistic relationship between two of the MAPK pathways. Early findings showed that cells exposed to osmotic stress fail to undergo filamentous growth (Davenport *et al.* 1999). Inhibition of the filamentation pathway requires the HOG pathway MAPK **Hog1** (O'Rourke and Herskowitz 1998). Together with other studies, it has been proposed that **Hog1** phosphorylates a component of the filamentation pathway to inactivate it (Westfall and Thorner 2006; Shock *et al.* 2009). Therefore, the sharing of components between pathways may contribute to an either/or response.

Recently, the Dohlman and Saito labs have moved our understanding of this attenuation mechanism forward (Hao *et al.* 2008; Yamamoto *et al.* 2010). Preliminary studies uncovered a surprising result: **Kss1** is phosphorylated in response to osmotic stress (Hao *et al.* 2008). The phosphorylation of **Kss1** is transient and (as expected during cross-talk) is dramatically stimulated in cells lacking **Hog1**. In fact, the phosphorylation of **Hog1** itself is stimulated in cells containing a kinase-inactive version of **Hog1** (Hao *et al.* 2008). How then are multiple MAPK pathways inhibited by **Hog1**? Dohlman and colleagues reasoned that **Hog1** phosphorylates a shared component between the two pathways to attenuate signaling. By testing several shared components, they found that **Ste50** is phosphorylated by **Hog1**(Hao *et al.* 2008). The phosphorylation of **Ste50** was at first thought to limit the duration of **Kss1** (and **Hog1**) activation (Hao *et al.* 2008) but this claim has been more recently questioned (Shock *et al.* 2009; Patterson *et al.* 2010). Saito and colleagues confirmed that **Ste50** is a target for **Hog1** and went on to show that two protein phosphatases, **Msg5** and **Ptp3**, synergistically contribute to MAPK downregulation by **Ste50** (Yamamoto *et al.* 2010). Nevertheless, the precise target of **Hog1** in dampening the filamentous growth pathway remains unclear, and it has been suggested that an as yet unidentified target of **Hog1** contributes to pathway specificity (Saito 2010). Future studies in this area will undoubtedly move forward our understanding of signal discrimination between related MAPK pathways.

### Mechanisms of signal integration during filamentous growth

A fundamental question in pathway regulation is how information from different pathways is integrated into a coordinated response. As discussed above, the differentiation from yeast-form to filamentous-form cells requires multiple pathways: TOR, SNF, RAS, and MAPK. How do signals sent through these pathways become integrated into a coherent response? Recently, several examples of signal integration have been elucidated (Figure 6).

One way in which filamentation signals become integrated is by convergence at common target genes.
A

Rim101
Snf1
Nrg
Tor
Ras2
Flo8
Ste12 + Tec1
MAPK
FLO11

B

Opi1
Ras2
Snf1
MAPK
Rtg
Rim101
Tor

Figure 6 Mechanisms of signal integration among regulatory proteins and pathways that control filamentous growth. (A) Multiple signaling pathways converge on the FLO11 promoter to modulate gene expression (Rupp et al. 1999). Both Snf1 and Rim101 are thought to function through the transcriptional repressors Nrg1 and Nrg2 (Kuchin et al. 2002; Lamb and Mitchell 2003). (B) Multiple signaling pathways regulate the activity of the filamentation MAPK pathway, adapted from (Chavel et al. 2010). Rtg refers to the retrograde mitochondrial signaling pathway (Liu and Butow 2006). Other pathways also converge on FLO11 that are not shown here (Bruckner and Mosch 2011).

A second mechanism of signal integration involves coregulation of signaling pathways involved in filamentous growth. Two major pathways that regulate filamentous growth, RAS and MAPK, are functionally connected to each other. Specifically, Ras2 regulates the activity of the MAPK pathway at or above Cdc42 (Mosch et al. 1996). The critical experiments demonstrating this result came from gain- and loss-of-function alleles of RAS2 and CDC42. A dominant active version of Ras2, Ras2V19, activated the filamentation-specific FG(Ty)-lacZ reporter and filamentous growth. This stimulation was not observed in cells containing loss-of-function alleles of CDC42, implying that Ras2 functions at or above Cdc42 in the filamentous growth pathway (Mosch et al. 1996).

There are several ways in which Ras2 might regulate the MAPK pathway above Cdc42. Ras2 may associate with and modulate upstream components of the pathway (like Msb2, Sho1, and Cdc42). Alternatively, Ras2 may indirectly modulate MAPK activity, for example by regulating the expression of a MAPK regulatory gene. We found evidence to support the latter possibility. Specifically, Ras2 was found to regulate expression of the MSB2 gene in a Ste12-independent manner (Chavel et al. 2010). The key finding was that activated versions of Msb2 failed to bypass the ras2 mutant, whereas overexpression of the MSB2 gene did bypass ras2. This result supports the idea that Ras2 regulates the MAPK pathway indirectly, by modulating MSB2 expression. In further support of this possibility, Tpk2 was also required for MSB2 expression (Chavel et al. 2010).

Although one cannot formally exclude the possibility that Ras2 is a component of the filamentous growth pathway, experiments to date do not support that possibility. Msb2's cytosolic domain associates with Cdc42 by two-hybrid analysis, but not with Ras2 (Cullen et al. 2004). Ste50 associates with Cdc42, but not Ras2, by its RA domain (Truckses et al. 2006). Ras2 regulates MEK–ERK signaling in mammalian cells through the protein kinase RAF, which is not present in yeast (Zebisch et al. 2007). How Ras2–Tpk2 regulates MSB2 expression remains to be determined. As stated above, the Ras2–Tpk2 pathway and the MAPK pathway also converge on the FLO11 promoter (Rupp et al. 1999) and exhibit nonoverlapping effects on filamentous growth (Chen and Thorner 2010).

Several pieces of evidence suggest that coordination among signaling pathways is greater even than the foregoing discussion suggests. One study by Snyder and colleagues showed that several key transcription factors regulate each others' expression, implicating these factors as target hubs for filamentous growth. These hubs serve as master regulators that integrate different aspects of the response into a coordinated behavior (Borneman et al. 2006). Likewise, it has been suggested that an integrated molecular network may be involved in the overall regulation of filamentous growth (Prinz et al. 2004). This transcriptional network is likely to be extensive, given the number of different transcription factors that have been identified that

P. J. Cullen and G. F. Sprague
contribute to filamentous growth, such as the forkhead proteins (Zhu et al. 2000), Mss11 (Van Dyk et al. 2005), Sok2 (Pan and Heitman 2000), and Hms1 (Lorenz and Heitman 1998a), whose binding site has recently been identified (Chua et al. 2006).

A second study by Kumar and colleagues used a large-scale approach to determine whether the localization of any of the predicted protein kinases showed an altered distribution during filamentous growth (Bharucha et al. 2008). The reasoning was that altered kinase localization might reflect a role for the kinase in the filamentation response. In the study, five kinases (Fus3, Kss1, Tpk2, Sks1, and Ksp1) and the regulatory subunit Bcy1 shuttled from the cytosol to the nucleus under conditions permissive for filamentous growth (Bharucha et al. 2008). Kss1, Tpk2, and Bcy1 might be expected to have this pattern, given that they play a role in filamentous growth. Fus3 is not known to enter the nucleus in response to nutrient limitation, and this result was unexpected. Sks1 is involved in the cellular response to glucose limitation (Vagnoli and Bisson 1998) and Ksp1 is poorly characterized. The surprising finding came when it was discovered that the colocalization of these kinases was interdependent (Bharucha et al. 2008). Therefore, interlocking kinase localization (through a mechanism that remains to be determined) may coordinate the activity of different pathways that regulate filamentous growth.

A third example comes from a novel screen designed to identify regulators of the signaling mucin Msb2, which functions at the head of the MAPK pathway (Cullen et al. 2004). A high-throughput approach called secretion profiling was used to measure release of the extracellular domain of Msb2 (Chavel et al. 2010). Secretion profiling of complementary genomic collections showed that some of the proteins that regulate filamentous growth—including Op1 (Reynolds 2006), retrograde (Rtg) (Liu and Butow 2006), and Rim101 (Lamb and Mitchell 2003)—were also required to activate the MAPK pathway (Figure 6B). Some of these regulators functioned by regulating MSB2 expression. Therefore, the MSB2 gene, like other hub genes, may be a place where multiple signals converge. Accessibility to the MSB2 promoter was regulated by the histone deacetylase (HDAC) Rpd3p(L), which positively regulates filamentous growth (Chavel et al. 2010).

Together, these examples provide a glimpse of the regulatory hierarchy that controls filamentous growth. Integration of signaling circuitry at a systems level may be important for regulating complex behaviors like nutritional cell differentiation in eukaryotic cells and will likely shape future studies where systems biology approaches become more heavily utilized.

thought of as having differentiated into a distinct cell type. This idea is supported by the fact that filamentation regulatory pathways induce the expression of hundreds of genes to reconstruct the cell’s shape and biochemical properties. Three major changes are associated with filamentous growth, which can account for the morphological changes and cell-surface properties of filamentous cells. These changes include the expression of the cell adhesion molecule Flo11, a switch in polarity, and an extension of different phases of the cell cycle. Although the changes occur synchronously, each aspect is thought to be regulated by different mechanisms. For example, each response can be genetically separated from the others (Mosch and Fink 1997; Palecek et al. 2000; Cullen and Sprague 2002). As a result of these changes, cells grow away from colony interiors, become elongated, and adhere to each other and to surfaces. Robust filamentous growth is a property of certain strain backgrounds (typically Σ1278b). Most laboratory strains of S. cerevisiae have acquired mutations, presumably as a result of genetic manipulation in the laboratory, that compromise the filamentous response (Liu et al. 1996).

### Cell adhesion regulation by the flocculin Flo11

One change that is associated with filamentous growth is cell–cell adhesion. Unlike yeast-form cells that fully separate from each other after each cell cycle, filamentous cells remain connected in chains or filaments. The situation in budding yeast differs from that of filamentous fungi, which fail to undergo cytokinesis and grow as multinucleate hyphae. Filamentously growing yeast cells undergo cytokinesis but remain attached to each other through protein and polysaccharide attachments. That is, they form pseudohyphae by virtue of adhesive contacts in the cell wall.

The major cell adhesion molecule that controls filamentous growth is the adhesion/flocculin Flo11 (or Mucl) (Lambrechts et al. 1996; Lo and Dranginis 1998; Guo et al. 2000). Flo11 is one of the most intensively studied fungal adhesion molecules, and its overall structure and properties are typical of adhesion molecules in other fungal species (Verstrepen and Klis 2006). Flo11 contains a putative N-terminal signal sequence and transmembrane domain, an external Ser/Thr/Pro-rich repeat region that is heavily glycosylated, and a C-terminal glycosylphosphatidylinositol (GPI) anchor. FLO11 is a member of the FLO gene family (FLO1, FLO5, and FLO9–FLO11) and is the major expressed flocculin. Other FLO genes, which are located at subtelomeric loci, are transcriptionally silent (Guo et al. 2000; Verstrepen et al. 2004). If by some means they become expressed, the encoded Flo proteins have different and potentially unique adherence properties (Guo et al. 2000). For example, FLO10 is expressed in mutants lacking transcriptional repressors Sfl1 or Rst1/Dig1, and in these settings can substitute for FLO11 (Breitkreutz et al. 2003). In contrast, Flo1 can promote biofilm formation (Smukalla et al. 2008).

## How Do Signaling Pathways Accomplish Nutritional Cell Differentiation?

Cells undergoing filamentous growth have different properties than yeast-form cells. Filamentous cells are commonly

Filamentous Growth Regulation 37As discussed above, the $FLO11$ gene is regulated by an unusually large promoter where multiple signaling pathways converge. Changes in the $FLO11$ gene/promoter can have dramatic effects on cell adhesion. For example, altering the levels and adherence properties of Flo11 can induce novel responses, which range from the flocculation of cells in a dense pellet that falls out of solution, to the formation of buoyant aggregates of cells on broth surfaces (Fidalgo et al. 2006). Changes in $FLO11$ expression occur rapidly and are subject to epigenetic regulation, which can result in a heterogeneous population of cells with different adherence properties (Halme et al. 2004; Verstrepen et al. 2005). In pathogens, variation of proteins at the cell surface is an important feature of virulence, and is thought for example to allow fungal cells to evade detection by the immune system (Heinsbroek et al. 2005; Nather and Munro 2008).

Flo11 is required for invasive growth of cells into agar-based substrates. Indeed, natural isolates of yeast exhibit high levels of agar invasion (Casalone et al. 2005). What is the physiological basis for this response? Yeast cells undergo invasive growth in at least one “natural” setting, the fruiting bodies of grapes. The filamentous growth pathway and $FLO11$ are required for full colonization of this environment (Pitoniak et al. 2009). Presumably, therefore, invasive growth mediated by Flo11-dependent contacts allows the penetration of cells into a variety of different environments.

Flo11 is also required to mediate colonial surface expansion in a connected mat of cells or biofilm (Reynolds and Fink 2001). Biofilms are a common growth pattern in many microbial species (Parsek and Greenberg 2005). Mat formation in yeast is regulated by some of the same signaling pathways that regulate filamentous growth, as well as by nonoverlapping pathways (Martineau et al. 2007). The role that Flo11 plays in biofilm/mat expansion is not entirely clear, which highlights the fact that certain aspects of Flo11 regulation have yet to be elucidated. For example, it is not clear what Flo11 binds to to mediate its adherence properties. As a second example, cells in a biofilm are thought to glide past each other during colonial expansion in a Flo11-dependent manner (Reynolds and Fink 2001). How does a potent cell-adhesion molecule promote cellular sliding? More generally, are adherent cells capable of separating in response to changes in environmental conditions?

One hint toward answering some of these questions has come from the unexpected finding that Flo11 can be shed from cells (Karunanithi et al. 2010). Flo11 shedding was tested because two other yeast mucin-like glycoproteins, Msb2 and Hkr1, are also shed. Flo11 shedding provides a mechanism to interrupt adherence. Indeed, the overall balance in adherence properties—not maximal adherence—optimizes filamentous growth and mat formation (Karunanithi et al. 2010). Shed Flo11 surrounds yeast mats in a fluid layer that may be functionally equivalent to the mucus secretions of higher eukaryotes. Secreted mucin-like proteins may play unexpected roles in the adherence properties and virulence of microbial pathogens (Karunanithi et al. 2010).

Understanding Flo11 properties and regulation provides a direct connection to pathogenesis. The ALS gene family in *C. albicans* (Hoyer 2001) and EPA family in *C. glabrata* (De Las Penas et al. 2003) are Ser/Thr-rich glycoproteins that are connected to the cell wall by a C-terminal GPI anchor (Verstrepen and Klis 2006). In *C. albicans*, adherence molecules regulate the attachment of cells to medical devices. They promote the compaction of cells into specialized biofilms that are resistant to high levels of antibiotics and contribute to the formation of filaments of interconnected hyphal cells (Chandra et al. 2005; Blankenship and Mitchell 2006; Noble and Mitchell 2006). The regulated expression of adherence molecules also appears to control the specific adherence properties of cells in a surface-dependent manner (Verstrepen and Klis 2006). Therefore, cell adhesion regulation is an important feature of filamentous growth, biofilm formation, and pathogenesis.

A potentially related response is the formation of ruffled colonies in a Flo11-dependent manner on high-agar surfaces (Figure 2A) (Granek and Magwene 2010). Despite the visually striking nature of these colonies, the functional significance of this patterning is not clear. The degree of colony ruffling corresponds to the level of the Flo11 protein, and it has been speculated that Flo11 (St'ovicek et al. 2010) and other flocculins like Flo1 (Beauvais et al. 2009), contribute to the formation of an extracellular matrix in this setting. Given that Flo11 exists in a form that is shed from cells (Karunanithi et al. 2010), this possibility, although relatively unexplored, is intriguing.

### Cell polarity reorganization by bud-site–selection proteins

The reorganization of cell architecture that occurs during filamentous growth involves many different processes and proteins that control cell polarity, the cell cycle, and the switch from apical to isotropic growth. Cell polarity in yeast is primarily dictated by proteins that mark the ends of the cell, which connect to signaling GTPases that regulate the actin cytoskeleton. Budding yeast exhibit different growth patterns depending on cell type and nutrient levels (Chant and Pringle 1991; Sanders and Field 1995; Ni and Snyder 2001; Nelson 2003). During yeast-form growth, haploid cells bud in an axial pattern, whereas diploid cells bud in a bipolar pattern (Figure 7). The different patterns result from differential utilization of cortical landmarks, or bud-site–selection proteins, that mark the poles of the cell and that are chosen by a cascade of Ras and Rho-like GTPase modules (Park and Bi 2007). Early observations of cells undergoing filamentous growth showed a change in growth pattern (Gimeno et al. 1992). Specifically, diploid cells, which typically bud at either pole (bipolar), switched to budding only at the distal pole (distal-unipolar). As a result, cells grew away from their mothers into virgin territories in chains of connected cells.
polarity during filamentous growth requires predominate selection of Bud8 over other positional cues. In fact, filamentous growth provides part of the overall rationale for why yeast cells adopt different budding patterns in the first place: haploid and diploid cells bud distally to escape the colony and enter new territories, whereas haploid cells bud back toward each other to rapidly identify and select a mating partner (Gimeno and Fink 1992).

How is Bud8 preferentially established as the major polar landmark for filamentous cells? Although this question is not yet resolved, several pieces of information are available. Bud8 is required in haploid cells exclusively under nutrient-limiting conditions (Cullen and Sprague 2002); therefore a change in Bud8 or its regulators would be expected to occur preferentially in nutrient-limiting environments. Such a change is not likely to result from loss of the proximal or axial marks, given that they are present in filamentous cells (Cullen and Sprague 2002).

One might expect a signal transduction pathway to interface with the bud-site–selection machinery and orchestrate the change in budding pattern. Addressing this possibility is complicated for several reasons. One is that in diploid cells, in which many studies on filamentous growth have been performed, the switch from bipolar to distal-unipolar budding is subtle, given that the first bud typically emerges from the distal pole in both high- and low-nutrient conditions (Chant and Pringle 1995). Examining the change in budding in haploid cells, which switch from an axial to a distal-unipolar pattern under glucose-limiting conditions, may be easier and more fruitful. We developed the single cell invasive growth assay (Figure 2C), which in principle surmounted these obstacles by allowing quantitation of the changes in budding pattern and cell elongation of individual cells. Using this assay, we showed that components of the filamentous growth MAPK pathway play a role in the change in budding pattern. Specifically, a ste12 mutant showed a partial defect, and a ste20 mutant a more severe defect (Cullen and Sprague 2000). A role for the MAPK pathway in regulating distal-unipolar budding conflicts with a previous report (Roberts and Fink 1994). In that study, however, cells were examined on the agar surface and did not show as robust a change in budding pattern as invading cells (Roberts and Fink 1994). A greater role for Ste20 than Ste12 in promoting distal-pole budding is supported by the fact that Ste20 has a specific role in bipolar budding in diploid cells (Sheu et al. 2000). More recently, we reported a bud site defect for the msb2, shol, and ste12 mutants, although the msb2 and shol mutants had a less severe cell elongation defect than the ste12 mutant (Cullen et al. 2004). Recently, it has been reported that neither Ras2 nor the MAPK is required for distal-pole budding (Chen and Thorner 2010). Although we contend that the MAPK pathway does play a role in the switch to distal-unipolar budding, it seems clear that other pathways are also required. Identifying these pathways is an important future goal for research in this area.

Cell elongation due to changes in the cell cycle and polarized growth

During filamentous growth, yeast cells become elongated. How does a cell change its shape? At least two different mechanisms underlie the change in behavior. One is that cells undergo a change in polarity (Pruyne and Bretscher 2000a,b). Specifically, cells undergoing filamentous growth exhibit an increase in apical growth, the highly polarized growth that occurs at the very tip of cells, which differs from isotropic growth that occurs uniformly around the cell cortex. There is clear evidence to support this mechanism. Cells undergoing filamentous growth show a highly polarized actin cytoskeleton, and the polarisome machinery (Gladfelter et al. 2005), which is composed of the formin Bni1 and other proteins (Evangelista et al. 1997) and is regulated by the polarity control GTPase Cdc42, is required for cell polarization during filamentous growth. How do signaling pathways regulate polarisome function? One possibility is that the PAK kinase Ste20 may phosphorylate Bni1 under conditions of MAPK activation (Goehring et al. 2003). Nevertheless, there are many other possible ways in which Cdc42/Bni1 and other proteins might promote enhanced apical growth. Perhaps they interface with proteins whose molecular functions are to regulate tip growth or to regulate the switch between apical and isotropic growth. Potential targets include actin cables, Cln1/2-Cdc28, and GAPs and GEFs for Rho1/Cdc42.

A second mechanism that underlies the change in cell length involves an extension of the cell cycle. Differential extension of one phase of the cell cycle can tip the balance toward more apical growth over isotropic growth (Kron et al. 1994). The MAPK pathway regulates the expression of the cyclin CLN1, which encodes a G₁ cyclin (Madhani et al. 1999). The different G₁ cyclins have different effects on the filamentation response (Loeb et al. 1999; Colomina et al. 2009). Elements of the morphogenetic checkpoint (Lew and Reed 1995) including the protein kinase Swe1 (Sia et al. 1998; La Valle and Wittenberg 2001) are also important for filamentous growth regulation.

Transcriptional targets of filamentation signaling pathways

Filamentous growth can be explained in large part by changes in polarity, the cell cycle, and FLO11 expression. Over the past decade, hundreds of genes have been identified by various screening approaches that implicate many different cellular processes in the regulation of filamentous growth. Genetic screens (Mosch and Fink 1997; Lorenz and Heitman 1998a; Lorenz et al. 2000a), large scale genomic screens (Jin et al. 2008b), proteomic approaches (Xu et al. 2010), and genome-wide expression profiling (Madhani et al. 1999; Roberts et al. 2000; Breitkreutz et al. 2003) have unveiled a new picture of filamentous growth regulation that impinges on many different cellular processes. Comparative genome sequencing between a standard laboratory strain and the Σ1278b background coupled with genome-wide deletion analysis of all nonessential genes in the Σ1278b background show multiple functional differences between the two genetic backgrounds, and support the notion of a globally regulated cellular response (Dowell et al. 2010). Together these genetic and high-throughput approaches reinforce the idea that the dimorphic transition to filamentous growth is a cellular differentiation response that involves the reorganization of many aspects of cellular machinery to produce a specific cell type.

How complicated is filamentous growth? One way to explore the complexity of the response is to examine the outputs of the signaling pathways that regulate the behavior. A diverse collection of genes is induced by the filamentation MAPK pathway. One target encodes the polygalacturonidase Pgu1, an enzyme that metabolizes a component found in plant cell walls (Madhani et al. 1999). Filamentous growth occurs in the grape-producing plant *Vitis vinifera* (Pitoniak et al. 2009), one environment in which *Pgu1* may be required. Another prominent group of transcriptional targets are Ty1 transposons. The fact that transposition is induced by the filamentation pathway in response to environmental perturbation may provide a mechanism for adaptive evolution in response to stress (Morillon et al. 2000).

In addition to *PGU1* and Ty, there are many (hundreds of) targets of the signaling pathways that control filamentous growth. What are all of these genes doing? Many of the main targets and highly induced genes do not at present have a clear cellular function. For example, several genes that are considered canonical reporters for the filamentation pathway, like *YLR042c* and *SVS1*, have no clear phenotype when deleted and no established cellular function (P. J. Cullen and G. F. Sprague, unpublished data). An existing challenge is to understand at a functional and phenotypic level the roles that the target genes play in filamentous growth. One reason for the lack of phenotype could be genetic redundancy. A second reason may be that yeast undergoes critical behaviors for filamentous growth that are not obvious under standard laboratory conditions. For example, *Pgu1* may be critical for yeast cells to colonize plant tissue but *pgu1* mutants would not be expected to show a clear phenotype in laboratory settings.

Filamentous growth is also tied into core cellular processes. These include transcription by RNA polymerase II (Singh et al. 2007), protein translation (Strittmatter et al. 2006; Gilbert et al. 2007), tRNA modification (Murray et al. 1998; Abdullah and Cullen 2009), protein glycosylation (Cullen et al. 2000), the unfolded protein response (UPR) (Schroder et al. 2000, 2004), autophagy (Ma et al. 2007), and the proteasome (Prinz et al. 2004). Together these findings resonate with the current picture of the yeast genetic interaction network, where many cellular processes are connected in some manner to each other (Costanzo et al. 2010). It will be interesting to overlay onto this network the changes in basic cellular machinery that occur during filamentous growth.
One connection may exist between protein glycosylation, the UPR, and the MAPK pathway. The rationale goes as follows: defects in protein glycosylation or protein folding reduce the glycosylation/stability of the extracellular domain of *Msb2* (Yang et al. 2009). Underglycosylated *Msb2* mimics the activated form of the protein, because the extracellular domain is inhibitory and activates the MAPK pathway. In protein glycosylation mutants, *Msb2* is underglycosylated (Cullen et al. 2004; Yang et al. 2009), and the MAPK pathway is active (Cullen et al. 2000). Indeed, most perturbations to *Msb2*'s mucin homology domain (which is heavily glycosylated in mammalian mucins) (Silverman et al. 2003) results in a hyperactive protein (Cullen et al. 2004). Most intriguingly, protein glycosylation provides a readout of nutrition, because mannosyl substrates are derived from glucose-6-phosphate. Therefore, underglycosylation of *Msb2* may represent a signal to the MAPK pathway for entry into low-nutrient environments. More generally, the rates of core cellular processes may reflect overall nutritional status that becomes sensed and incorporated into the filamentation response.

Interestingly, the UPR has an inhibitory role in sporulation (Schroder et al. 2000) and mediates its inhibitory effect by recruitment of the HDAC *Rpd3* to early meiotic genes through the transcription factor *Ume6* (Schroder et al. 2004). In contrast, *Rpd3* plays a positive role in filamentous growth and is required for the expression of the *MSB2* and *STE12* genes (Chavel et al. 2010). The reciprocal roles of *Rpd3* in promoting filamentous growth and dampening meiosis suggest that *Rpd3* may be involved in the decision of whether cells should undergo filamentous growth or sporulate (Figure 1). The mechanism by which a HDAC, traditionally considered to function as a repressor of gene transcription, promotes the expression of filamentation regulatory genes is not clear. Nevertheless, microarray analysis reveals that many genes are downregulated in HDAC mutants, and *Rpd3* also positively regulates the HOG pathway (de Nadal et al. 2004).

## Perspectives

Studies of filamentous growth regulation in budding yeast have had at least two major biological impacts. The first is that yeast provides a roadmap to identify and characterize elements of the response that also occurs in other fungal species, particularly fungal pathogens. Many of the genetic pathways that regulate filamentous growth in *C. albicans* and other pathogens have been uncovered through studies in *S. cerevisiae*. As one of many possible examples, *Msb2* homologs have recently been identified in *C. albicans* and in three plant fungal pathogens. In all cases, *Msb2* presides over a MAPK pathway that is important for filamentous growth and virulence (Roman et al. 2009; Lanver et al. 2010; Liu et al. 2011; Perez-Nadales and Di Pietro 2011). The second is that filamentous growth regulation is a model for understanding eukaryotic cell differentiation. Cell differentiation in mammals involves processes that are at present complex and poorly defined. Specifically, the concept of a globally connected network of signaling pathways working in concert, although accepted, is not well understood at the molecular level. Budding yeast provides a working template to understand how signals that initiate from different pathways become routed through common modules to induce a specific behavior.

What lies ahead for studies on filamentous growth regulation in yeast? Filamentous growth represents a point of convergence between many cellular pathways—the cell cycle, cell polarity, and nutrition—and therefore is an attractive system to understand the connection between different biological processes. Of course, one of the main places where future progress is needed is in further defining the signaling pathways that regulate the response. The *Gpr1/Gpa2/Ras2* pathway is filled with controversies that make drawing a coherent picture of that pathway difficult. Paramount in this regard is resolving the paradox of whether and how carbon sources are sensed and interpreted into the decision of whether or not to undergo filamentous growth. Another area in which much progress is needed is in understanding how the MAPK pathway that regulates filamentous growth maintains its identity. It could be reasonably argued that all of the components of that pathway (*Msb2*, *Sho1*, *Cdc42*, *Ste20*, *Ste50*, *Ste11*, *Ste7*, *Kss1*, and *Ste12*), with the exception of *Tec1*, are general components that function in multiple pathways. Solving this identity crisis represents a daunting challenge in the field of cellular signaling. Probably the most important and mysterious aspect of filamentous growth regulation involves the integration of signals from multiple pathways into a coherent response. Do the MAPK, TOR, and *Ras2* pathways talk to each other, and if so, to what extent? This area in particular is ripe for future investigations.

In addition to filamentous growth, bakers' yeast undergoes other nutrient-limitation–dependent responses (Figure 1). These include entry into a quiescent state, sporulation (in diploids), microbial mat expansion, and quorum sensing. As mentioned above, an important question is how does a cell choose among these different lifestyles? Similarly, is there a relationship between these various responses? For example, *C. albicans* forms microbial mats or biofilms that are composed of multiple cell types including filamentous cells and that interface with other communities of microorganisms (Parsek and Greenberg 2005). Ultimately, the ecology of filamentous growth regulation—especially of cell populations in native settings—will be the most fun and challenging to explore.

## Acknowledgments

We thank laboratory members for helpful discussions. We apologize to investigators whose work was not cited. G.F.S. is supported by a grant from the U.S. Public Health Service (GM30027) and P.J.C. is supported by grants from the U.S.
Public Health Service (GM098629 and DE18425) and the American Cancer Society (TBE-114083).

### Literature Cited

Abdullah, U., and P. J. Cullen, 2009 The tRNA modification complex elongator regulates the Cdc42-dependent mitogen-activated protein kinase pathway that controls filamentous growth in yeast. Eukaryot. Cell 8: 1362–1372.

Adams, A. E., D. I. Johnson, R. M. Longnecker, B. F. Sloat, and J. R. Pringle, 1990 CDC42 and CDC43, two additional genes involved in budding and the establishment of cell polarity in the yeast Saccharomyces cerevisiae. J. Cell Biol. 111: 131–142.

Aldabbous, M. S., M. G. Roca, A. Stout, I. C. Huang, N. D. Read *et al.*, 2010 The ham-5, rcm-1 and rco-1 genes regulate hyphal fusion in Neurospora crassa. Microbiology 156: 2621–2629.

Amoah-Buahin, E., N. Bone, and J. Armstrong, 2005 Hyphal growth in the fission yeast Schizosaccharomyces pombe. Eukaryot. Cell 4: 1287–1297.

Arkinstall, S. J., S. G. Papasavvas, and M. A. Payton, 1991 Yeast alpha-mating factor receptor-linked G-protein signal transduction suppresses Ras-dependent activity. FEBS Lett. 284: 123–128.

Bafna, S., S. Kaur, and S. K. Batra, 2010 Membrane-bound mucins: the mechanistic basis for alterations in the growth and survival of cancer cells. Oncogene 29: 2893–2904.

Bao, M. Z., M. A. Schwartz, G. T. Cantin, J. R. Yates 3rd. and H. D. Madhani, 2004 Pheromone-dependent destruction of the Tec1 transcription factor is required for MAP kinase signaling specificity in yeast. Cell 119: 991–1000.

Bao, M. Z., T. R. Shock, and H. D. Madhani, 2010 Multisite phosphorylation of the Saccharomyces cerevisiae filamentous growth regulator Tec1 is required for its recognition by the E3 ubiquitin ligase adaptor Cdc4 and its subsequent destruction in vivo. Eukaryot. Cell 9: 31–36.

Bardwell, L., 2006 Mechanisms of MAPK signalling specificity. Biochem. Soc. Trans. 34: 837–841.

Bardwell, L., J. G. Cook, J. X. Zhu-Shimoni, D. Voora, and J. Thorner, 1998 Differential regulation of transcription: repression by unactivated mitogen-activated protein kinase Kss1 requires the Dig1 and Dig2 proteins. Proc. Natl. Acad. Sci. USA 95: 15400–15405.

Batlle, M., A. Lu, D. A. Green, Y. Xue, and J. P. Hirsch, 2003 Krh1p and Krh2p act downstream of the Gpa2p G(alpha) subunit to negatively regulate haploid invasive growth. J. Cell Sci. 116: 701–710.

Beauvais, A., C. Loussert, M. C. Prevost, K. Verstrepen, and J. P. Latge, 2009 Characterization of a biofilm-like extracellular matrix in FLO1-expressing Saccharomyces cerevisiae cells. FEM. Yeast Res. 9: 411–419.

Ben-Ami, R., R. E. Lewis, and D. P. Kontoyiannis, 2008 Immunocompromised hosts: immunopharmacology of modern antifungals. Clin. Infect. Dis. 47: 226–235.

Bender, A., and J. R. Pringle, 1989 Multicopy suppression of the cdc24 budding defect in yeast by CDC42 and three newly identified genes including the ras-related gene RSR1. Proc. Natl. Acad. Sci. USA 86: 9976–9980.

Bender, A., and J. R. Pringle, 1992 A Ser/Thr-rich multicopy suppressor of a cdc24 bud emergence defect. Yeast 8: 315–323.

Berman, J., 2006 Morphogenesis and cell cycle progression in Candida albicans. Curr. Opin. Microbiol. 9: 595–601.

Bharucha, N., J. Ma, C. J. Dobry, S. K. Lawson, Z. Yang *et al.*, 2008 Analysis of the yeast kinome reveals a network of regulated protein localization during filamentous growth. Mol. Biol. Cell 19: 2708–2717.

Blankenship, J. R., and A. P. Mitchell, 2006 How to build a biofilm: a fungal perspective. Curr. Opin. Microbiol. 9: 588–594.

Boeckstaens, M., B. Andre, and A. M. Marini, 2008 Distinct transport mechanisms in yeast ammonium transport/sensor proteins of the Mep/Amt/Rh family and impact on filamentation. J. Biol. Chem. 283: 21362–21370.

Boone, C., N. G. Davis, and G. F. Sprague Jr., 1993 Mutations that alter the third cytoplasmic loop of the α-factor receptor lead to a constitutive and hypersensitive phenotype. Proc. Natl. Acad. Sci. USA 90: 9921–9925.

Borneman, A. R., J. A. Leigh-Bell, H. Yu, P. Bertone, M. Gerstein *et al.*, 2006 Target hub proteins serve as master regulators of development in yeast. Genes Dev. 20: 435–448.

Botstein, D., and G. R. Fink, 2011 Yeast: an experimental organism for 21st century biology. Genetics 189: 695–704.

Braus, G. H., O. Grundmann, S. Bruckner, and H. U. Mosch, 2003 Amino acid starvation and Gcn4p regulate adhesive growth and FLO11 gene expression in Saccharomyces cerevisiae. Mol. Biol. Cell 14: 4272–4284.

Brayman, M., A. Thathiah, and D. D. Carson, 2004 MUC1: a multifunctional cell surface component of reproductive tissue epithelia. Reprod. Biol. Endocrinol. 2: 4.

Breitkreutz, A., L. Boucher, B. J. Breitkreutz, M. Sultan, I. Jurisica *et al.*, 2003 Phenotypic and transcriptional plasticity directed by a yeast mitogen-activated protein kinase network. Genetics 165: 997–1015.

Brewster, J. L., T. de Valoir, N. D. Dwyer, E. Winter, and M. C. Gustin, 1993 An osmosensing signal transduction pathway in yeast. Science 259: 1760–1763.

Brown, C. M., and J. S. Hough, 1965 Elongation of yeast cells in continuous culture. Nature 206: 676–678.

Brückner, S., and H. U. Mösch, 2011 Choosing the right lifestyle: adhesion and development in Saccharomyces cerevisiae. FEMS Microbiol. Rev. 36: 25–58.

Bruckner, S., T. Kohler, G. H. Braus, B. Heise, M. Bolte *et al.*, 2004 Differential regulation of Tec1 by Fus3 and Kss1 confers signaling specificity in yeast development. Curr. Genet. 46: 331–342.

Bumgarner, S. L., R. D. Dowell, P. Grisafi, D. K. Gifford, and G. R. Fink, 2009 Toggle involving cis-interfering noncoding RNAs controls variegated gene expression in yeast. Proc. Natl. Acad. Sci. USA 106: 18321–18326.

Butow, R. A., and N. G. Avadhani, 2004 Mitochondrial signaling: the retrograde response. Mol. Cell 14: 1–15.

Casalone, E., C. Barberio, L. Cappellini, and M. Polsinelli, 2005 Characterization of Saccharomyces cerevisiae natural populations for pseudohyphal growth and colony morphology. Res. Microbiol. 156: 191–200.

Chandra, J., G. Zhou, and M. A. Ghannoum, 2005 Fungal biofilms and antimycotics. Curr. Drug Targets 6: 887–894.

Chant, J., and I. Herskowitz, 1991 Genetic control of bud site selection in yeast by a set of gene products that constitute a morphogenic pathway. Cell 65: 1203–1212.

Chant, J., and J. R. Pringle, 1991 Budding and cell polarity in Saccharomyces cerevisiae. Curr. Opin. Genet. Dev. 1: 342–350.

Chant, J., and J. R. Pringle, 1995 Patterns of bud-site selection in the yeast Saccharomyces cerevisiae. J. Cell Biol. 129: 751–765.

Chavel, C. A., H. M. Dionne, B. Birkaya, J. Joshi, and P. J. Cullen, 2010 Multiple signals converge on a differentiation MAPK pathway. PLoS Genet. 6: e1000883.

Chen, H., and G. R. Fink, 2006 Feedback control of morphogenesis in fungi by aromatic alcohols. Genes Dev. 20: 1150–1161.

Chen, H., M. Fujita, Q. Feng, J. Clardy, and G. R. Fink, 2004 Tyrosol is a quorum-sensing molecule in Candida albicans. Proc. Natl. Acad. Sci. USA 101: 5048–5052.

Chen, L., and N. G. Davis, 2002 Ubiquitin-independent entry into the yeast recycling pathway. Traffic 3: 110–123.

Chen, R. E., and J. Thorner, 2007 Function and regulation in MAPK signaling pathways: lessons learned from the yeast Saccharomyces cerevisiae. Biochim. Biophys. Acta 1773: 1311–1340.

Chen, R. E., and J. Thorner, 2010 Systematic epistasis analysis of the contributions of protein kinase A- and mitogen-activated protein kinase-dependent signaling to nutrient limitation-evoked responses in the yeast Saccharomyces cerevisiae. Genetics 185: 855–870.

Choi, K. Y., B. Satterberg, D. M. Lyons, and E. A. Elion, 1994 Ste5 tethers multiple protein kinases in the MAP kinase cascade required for mating in S. cerevisiae. Cell 78: 499–512.

Chou, S., L. Huang, and H. Liu, 2004 Fus3-regulated Tec1 degradation through SCFCdc4 determines MAPK signaling specificity during mating in yeast. Cell 119: 981–990.

Chou, S., S. Lane, and H. Liu, 2006 Regulation of mating and filamentation genes by two distinct Ste12 complexes in Saccharomyces cerevisiae. Mol. Cell. Biol. 26: 4794–4805.

Chua, G., Q. D. Morris, R. Sopko, M. D. Robinson, O. Ryan *et al.*, 2006 Identifying transcription factor functions and targets by phenotypic activation. Proc. Natl. Acad. Sci. USA 103: 12045–12050.

Colombo, S., P. Ma, L. Cauwenberg, J. Winderickx, M. Crauwels *et al.*, 1998 Involvement of distinct G-proteins, Gpa2 and Ras, in glucose- and intracellular acidification-induced cAMP signaling in the yeast Saccharomyces cerevisiae. EMBO J. 17: 3326–3341.

Colombo, S., D. Ronchetti, J. M. Thevelein, J. Winderickx, and E. Martegani, 2004 Activation state of the Ras2 protein and glucose-induced signaling in Saccharomyces cerevisiae. J. Biol. Chem. 279: 46715–46722.

Colomina, N., F. Ferrezuelo, E. Verges, M. Aldea, and E. Gari, 2009 Whi3 regulates morphogenesis in budding yeast by enhancing Cdk functions in apical growth. Cell Cycle 8: 1912–1920.

Cook, J. G., L. Bardwell, S. J. Kron, and J. Thorner, 1996 Two novel targets of the MAP kinase Kss1 are negative regulators of invasive growth in the yeast Saccharomyces cerevisiae. Genes Dev. 10: 2831–2848.

Cook, J. G., L. Bardwell, and J. Thorner, 1997 Inhibitory and activating functions for MAPK Kss1 in the S. cerevisiae filamentous-growth signalling pathway. Nature 390: 85–88.

Costanzo, M., A. Baryshnikova, J. Bellay, Y. Kim, E. D. Spear *et al.*, 2010 The genetic landscape of a cell. Science 327: 425–431.

Courchesne, W. E., R. Kunisawa, and J. Thorner, 1989 A putative protein kinase overcomes pheromone-induced arrest of cell cycling in S. cerevisiae. Cell 58: 1107–1119.

Crampin, H., K. Finley, M. Gerami-Nejad, H. Court, C. Gale *et al.*, 2005 Candida albicans hyphae have a Spitzenkörper that is distinct from the polarisome found in yeast and pseudohyphae. J. Cell Sci. 118: 2935–2947.

Crespo, J. L., T. Powers, B. Fowler, and M. N. Hall, 2002 The TOR-controlled transcription activators GLN3, RTG1, and RTG3 are regulated in response to intracellular levels of glutamine. Proc. Natl. Acad. Sci. USA 99: 6784–6789.

Cullen, P. J., and G. F. Sprague Jr., 2000 Glucose depletion causes haploid invasive growth in yeast. Proc. Natl. Acad. Sci. USA 97: 13619–13624.

Cullen, P. J., and G. F. Sprague Jr., 2002 The roles of bud-site-selection proteins during haploid invasive growth in yeast. Mol. Biol. Cell 13: 2990–3004.

Cullen, P. J., J. Schultz, J. Horecka, B. J. Stevenson, Y. Jigami *et al.*, 2000 Defects in protein glycosylation cause SHO1-dependent activation of a STE12 signaling pathway in yeast. Genetics 155: 1005–1018.

Cullen, P. J., W. Sabbagh Jr. E. Graham, M. M. Irick, E. K. van Olden *et al.*, 2004 A signaling mucin at the head of

the Cdc42- and MAPK-dependent filamentous growth pathway in yeast. Genes Dev. 18: 1695–1708.

Cutler, N. S., X. Pan, J. Heitman, and M. E. Cardenas, 2001 The TOR signal transduction cascade controls cellular differentiation in response to nutrients. Mol. Biol. Cell 12: 4103–4113.

d'Enfert, C., 2009 Hidden killers: persistence of opportunistic fungal pathogens in the human host. Curr. Opin. Microbiol. 12: 358–364.

Davenport, K. D., K. E. Williams, B. D. Ullmann, and M. C. Gustin, 1999 Activation of the Saccharomyces cerevisiae filamentation/invasion pathway by osmotic stress in high-osmolarity glycogen pathway mutants. Genetics 153: 1091–1103.

De Las Penas, A., S. J. Pan, I. Castano, J. Alder, R. Cregg *et al.*, 2003 Virulence-related surface glycoproteins in the yeast pathogen Candida glabrata are encoded in subtelomeric clusters and subject to RAP1- and SIR-dependent transcriptional silencing. Genes Dev. 17: 2245–2258.

de Nadal, E., M. Zapater, P. M. Alepuz, L. Sumoy, G. Mas *et al.*, 2004 The MAPK Hog1 recruits Rpd3 histone deacetylase to activate osmoreponsive genes. Nature 427: 370–374.

Deminoff, S. J., S. C. Howard, A. Hester, S. Warner, and P. K. Herman, 2006 Using substrate-binding variants of the cAMP-dependent protein kinase to identify novel targets and a kinase domain important for substrate interactions in Saccharomyces cerevisiae. Genetics 173: 1909–1917.

Dickinson, J. R., 1996 ‘Fusel’ alcohols induce hyphal-like extensions and pseudohyphal formation in yeasts. Microbiology 142 (Pt 6): 1391–1397.

Dohlman, H. G., and J. E. Slessareva, 2006 Pheromone signaling pathways in yeast. Sci. STKE 2006(364): cm6.

Dowell, R. D., O. Ryan, A. Jansen, D. Cheung, S. Agarwala *et al.*, 2010 Genotype to phenotype: a complex problem. Science 328: 469.

Ekiel, I., T. Sulea, G. Jansen, M. Kowalik, O. Minailiuc *et al.*, 2009 Binding the atypical RA domain of Ste50p to the unfolded Opy2p cytoplasmic tail is essential for the high-osmolarity glycerol pathway. Mol. Biol. Cell 20: 5117–5126.

Elion, E. A., P. L. Grisafi, and G. R. Fink, 1990 FUS3 encodes a cdc2+/CDC28-related kinase required for the transition from mitosis into conjugation. Cell 60: 649–664.

Elion, E. A., J. A. Brill, and G. R. Fink, 1991a Functional redundancy in the yeast cell cycle: FUS3 and KSS1 have both overlapping and unique functions. Cold Spring Harb. Symp. Quant. Biol. 56: 41–49.

Elion, E. A., J. A. Brill, and G. R. Fink, 1991b FUS3 represses CLN1 and CLN2 and in concert with KSS1 promotes signal transduction. Proc. Natl. Acad. Sci. USA 88: 9392–9396.

Engebrecht, J., 2003 Cell signaling in yeast sporulation. Biochem. Biophys. Res. Commun. 306: 325–328.

Esch, R. K., Y. Wang, and B. Errede, 2006 Pheromone-induced degradation of Ste12 contributes to signal attenuation and the specificity of developmental fate. Eukaryot. Cell 5: 2147–2160.

Eubanks, V. L., and L. R. Beuchat, 1982 Increased sensitivity of heat-stressed Saccharomyces cerevisiae cells to food-grade antioxidants. Appl. Environ. Microbiol. 44: 604–610.

Evangelista, M., K. Blundell, M. S. Longtine, C. J. Chow, N. Adames *et al.*, 1997 Bni1p, a yeast formin linking cdc42p and the actin cytoskeleton during polarized morphogenesis. Science 276: 118–122.

Feng, Y., L. Y. Song, E. Kincaid, S. K. Mahanty, and E. A. Elion, 1998 Functional binding between Gbeta and the LIM domain of Ste5 is required to activate the MEKK Ste11. Curr. Biol. 8: 267–278.

Fidalgo, M., R. R. Barrales, J. I. Ibeas, and J. Jimenez, 2006 Adaptive evolution by mutations in the FLO11 gene. Proc. Natl. Acad. Sci. USA 103: 11228–11233.

Fields, S., and O. Song, 1989 A novel genetic system to detect protein-protein interactions. Nature 340: 245–246.

Flatauer, L. J., S. F. Zadeh, and L. Bardwell, 2005 Mitogen-activated protein kinases with distinct requirements for Ste5 scaffolding influence signaling specificity in Saccharomyces cerevisiae. Mol. Cell. Biol. 25: 1793–1803.

Fleissner, A., A. C. Leeder, M. G. Roca, N. D. Read, and N. L. Glass, 2009 Oscillatory recruitment of signaling proteins to cell tips promotes coordinated behavior during cell fusion. Proc. Natl. Acad. Sci. USA 106: 19387–19392.

Gao, X. D., L. M. Sperber, S. A. Kane, Z. Tong, A. H. Tong *et al.*, 2007 Sequential and distinct roles of the cadherin domain-containing protein Axl2p in cell polarization in yeast cell cycle. Mol. Biol. Cell 18: 2542–2560.

Garcia, R., J. M. Rodriguez-Pena, C. Bermejo, C. Nombela, and J. Arroyo, 2009 The high osmotic response and cell wall integrity pathways cooperate to regulate transcriptional responses to zymolyase-induced cell wall stress in Saccharomyces cerevisiae. J. Biol. Chem. 284: 10901–10911.

Garrenton, L. S., S. L. Young, and J. Thorner, 2006 Function of the MAPK scaffold protein, Ste5, requires a cryptic PH domain. Genes Dev. 20: 1946–1958.

Garrenton, L. S., A. Braunwarth, S. Irniger, E. Hurt, M. Kunzler *et al.*, 2009 Nucleus-specific and cell cycle-regulated degradation of mitogen-activated protein kinase scaffold protein Ste5 contributes to the control of signaling competence. Mol. Cell. Biol. 29: 582–601.

Garrenton, L. S., C. J. Stefan, M. A. McMurray, S. D. Emr, and J. Thorner, 2010 Pheromone-induced anisotropy in yeast plasma membrane phosphatidylinositol-4,5-bisphosphate distribution is required for MAPK signaling. Proc. Natl. Acad. Sci. USA 107: 11805–11810.

Gartner, A., K. Nasmyth, and G. Ammerer, 1992 Signal transduction in Saccharomyces cerevisiae requires tyrosine and threonine phosphorylation of FUS3 and KSS1. Genes Dev. 6: 1280–1292.

Gastebois, A., C. Clavaud, V. Aimanianda, and J. P. Latge, 2009 Aspergillus fumigatus: cell wall polysaccharides, their biosynthesis and organization. Future Microbiol. 4: 583–595.

Gavrias, V., A. Andrianopoulos, C. J. Gimeno, and W. E. Timberlake, 1996 Saccharomyces cerevisiae TEC1 is required for pseudohyphal growth. Mol. Microbiol. 19: 1255–1263.

Gehret, A. U., A. Bajaj, F. Naidner, and M. E. Dumont, 2006 Oligomerization of the yeast alpha-factor receptor: implications for dominant negative effects of mutant receptors. J. Biol. Chem. 281: 20698–20714.

Gilbert, W. V., K. Zhou, T. K. Butler, and J. A. Doudna, 2007 Cap-independent translation is required for starvation-induced differentiation in yeast. Science 317: 1224–1227.

Gimeno, C. J., and G. R. Fink, 1992 The logic of cell division in the life cycle of yeast. Science 257: 626.

Gimeno, C. J., P. O. Ljungdahl, C. A. Styles, and G. R. Fink, 1992 Unipolar cell divisions in the yeast S. cerevisiae lead to filamentous growth: regulation by starvation and RAS. Cell 68: 1077–1090.

Gladfelter, A. S., L. Kozubowski, T. R. Zyla, and D. J. Lew, 2005 Interplay between septin organization, cell cycle and cell shape in yeast. J. Cell Sci. 118: 1617–1628.

Goehring, A. S., D. A. Mitchell, A. H. Tong, M. E. Keniry, C. Boone *et al.*, 2003 Synthetic lethal analysis implicates Ste20p, a p21-activated potein kinase, in polarisome activation. Mol. Biol. Cell 14: 1501–1516.

Good, M., G. Tang, J. Singleton, A. Remenyi, and W. A. Lim, 2009 The Ste5 scaffold directs mating signaling by catalytically unlocking the Fus3 MAP kinase for activation. Cell 136: 1085–1097.

Granek, J. A., and P. M. Magwene, 2010 Environmental and genetic determinants of colony morphology in yeast. PLoS Genet. 6: e1000823.


Guldener, U., G. J. Koehler, C. Haussmann, A. Bacher, J. Kricke *et al.*, 2004 Characterization of the Saccharomyces cerevisiae Fol1 protein: starvation for Cl carrier induces pseudohyphal growth. Mol. Biol. Cell 15: 3811–3828.

Guo, B., C. A. Styles, Q. Feng, and G. R. Fink, 2000 A Saccharomyces gene family involved in invasive growth, cell-cell adhesion, and mating. Proc. Natl. Acad. Sci. USA 97: 12158–12163.

Gutlliermond, A., 1920 *The Yeasts*, John Wiley and Sons, New York.

Hajishengallis, G., and J. D. Lambris, 2011 Microbial manipulation of receptor crosstalk in innate immunity. Nat. Rev. Immunol. 11: 187–200.

Halme, A., S. Bumgarner, C. Styles, and G. R. Fink, 2004 Genetic and epigenetic regulation of the FLO gene family generates cell-surface variation in yeast. Cell 116: 405–415.

Hao, N., Y. Zeng, T. C. Elston, and H. G. Dohlman, 2008 Control of MAPK specificity by feedback phosphorylation of shared adaptor protein Ste50. J. Biol. Chem. 283: 33798–33802.

Harashima, T., and J. Heitman, 2002 The Galpha protein Gpa2 controls yeast differentiation by interacting with kelch repeat proteins that mimic Gbeta subunits. Mol. Cell 10: 163–173.

Harashima, T., S. Anderson, J. R. Yates 3rd. and J. Heitman, 2006 The kelch proteins Gpb1 and Gpb2 inhibit Ras activity via association with the yeast RasGAP neurofibromin homologs Ira1 and Ira2. Mol. Cell 22: 819–830.

Harkins, H. A., N. Page, L. R. Schenkman, C. De Virgilio, S. Shaw *et al.*, 2001 Bud8p and Bud9p, proteins that may mark the sites for bipolar budding in yeast. Mol. Biol. Cell 12: 2497–2518.

Hedbacker, K., and M. Carlson, 2008 SNF1/AMPK pathways in yeast. Front. Biosci. 13: 2408–2420.

Heinsbroek, S. E., G. D. Brown, and S. Gordon, 2005 Dectin-1 escape by fungal dimorphism. Trends Immunol. 26: 352–354.

Heitman, J., N. R. Movva, and M. N. Hall, 1991 Targets for cell cycle arrest by the immunosuppressant rapamycin in yeast. Science 253: 905–909.

Hohmann, S., M. Krantz, and B. Nordlander, 2007 Yeast osmoregulation. Methods Enzymol. 428: 29–45.

Hongay, C. F., P. L. Grisafi, T. Galitski, and G. R. Fink, 2006 Antisense transcription controls cell fate in Saccharomyces cerevisiae. Cell 127: 735–745.

Hoyer, L. L., 2001 The ALS gene family of Candida albicans. Trends Microbiol. 9: 176–180.

Ingolia, N. T., and A. W. Murray, 2007 Positive-feedback loops as a flexible biological module. Curr. Biol. 17: 668–677.

Inouye, C., N. Dhillon, T. Durfee, P. C. Zambryski, and J. Thorner, 1997 Mutational analysis of STE5 in the yeast Saccharomyces cerevisiae: application of a differential interaction trap assay for examining protein-protein interactions. Genetics 147: 479–492.

Jansen, G., F. Buhring, C. P. Hollenberg, and M. Ramezani Rad, 2001 Mutations in the SAM domain of STE50 differentially influence the MAPK-mediated pathways for mating, filamentous growth and osmotolerance in Saccharomyces cerevisiae. Mol. Genet. Genomics 265: 102–117.

Jenness, D. D., Y. Li, C. Tipper, and P. Spatarick, 1997 Elimination of defective alpha-factor pheromone receptors. Mol. Cell. Biol. 17: 6236–6245.

Jin, H., J. M. McCaffery, and E. Grote, 2008a Ergosterol promotes pheromone signaling and plasma membrane fusion in mating yeast. J. Cell Biol. 180: 813–826.

Jin, R., C. J. Dobry, P. J. McCown, and A. Kumar, 2008b Large-scale analysis of yeast filamentous growth by systematic gene disruption and overexpression. Mol. Biol. Cell 19: 284–296.

Karunanithi, S., N. Vadaie, C. A. Chavel, B. Birkaya, J. Joshi *et al.*, 2010 Shedding of the mucin-like Flocculin Flo11p reveals a new aspect of fungal adhesion regulation. Curr. Biol. 20: 1389–1395.

Kataoka, T., S. Powers, C. McGill, O. Fasano, J. Strathern *et al.*, 1984 Genetic analysis of yeast RAS1 and RAS2 genes. Cell 37: 437–445.

Kohler, T., S. Wesche, N. Taheri, G. H. Braus, and H. U. Mosch, 2002 Dual role of the Saccharomyces cerevisiae TEA/ ATTS family transcription factor Tec1p in regulation of gene expression and cellular development. Eukaryot. Cell 1: 673–686.

Kraakman, L., K. Lemaire, P. Ma, A. W. Teunissen, M. C. Donaton *et al.*, 1999 A Saccharomyces cerevisiae G-protein coupled receptor, Gpr1, is specifically required for glucose activation of the cAMP pathway during the transition to growth on glucose. Mol. Microbiol. 32: 1002–1012.

Kranz, J. E., B. Satterberg, and E. A. Elion, 1994 The MAP kinase Fus3 associates with and phosphorylates the upstream signaling component Ste5. Genes Dev. 8: 313–327.

Kron, S. J., C. A. Styles, and G. R. Fink, 1994 Symmetric cell division in pseudohyphae of the yeast Saccharomyces cerevisiae. Mol. Biol. Cell 5: 1003–1022.

Kronstad, J. W., R. Attarian, B. Cadieux, J. Choi, C. A. D’Souza *et al.*, 2011 Expanding fungal pathogenesis: Cryptococcus breaks out of the opportunistic box. Nat. Rev. Microbiol. 9: 193–203.

Kubler, E., H. U. Mosch, S. Rupp, and M. P. Lisanti, 1997 Gpa2p, a G-protein alpha-subunit, regulates growth and pseudohyphal development in Saccharomyces cerevisiae via a cAMP-dependent mechanism. J. Biol. Chem. 272: 20321–20323.

Kuchin, S., V. K. Vyas, and M. Carlson, 2002 Snf1 protein kinase and the repressors Nrg1 and Nrg2 regulate FLO11, haploid invasive growth, and diploid pseudohyphal differentiation. Mol. Cell. Biol. 22: 3994–4000.

Kufe, D. W., 2009 Mucins in cancer: function, prognosis and therapy. Nat. Rev. Cancer 9: 874–885.

Kumamoto, C. A., 2005 A contact-activated kinase signals *Candida albicans* invasive growth and biofilm development. Proc. Natl. Acad. Sci. USA 102: 5576–5581.

Kusari, A. B., D. M. Molina, W. Sabbagh Jr. C. S. Lau, and L. Bardwell, 2004 A conserved protein interaction network involving the yeast MAP kinases Fus3 and Kss1. J. Cell Biol. 164: 267–277.

La Valle, R., and C. Wittenberg, 2001 A role for the Swe1 checkpoint kinase during filamentous growth of Saccharomyces cerevisiae. Genetics 158: 549–562.

Laloux, I., E. Jacobs, and E. Dubois, 1994 Involvement of SRE element of Ty1 transposon in TEC1-dependent transcriptional activation. Nucleic Acids Res. 22: 999–1005.

Lamb, T. M., and A. P. Mitchell, 2003 The transcription factor Rim101p governs ion tolerance and cell differentiation by direct repression of the regulatory genes NRG1 and SMP1 in Saccharomyces cerevisiae. Mol. Cell. Biol. 23: 677–686.

Lambrechts, M. G., F. F. Bauer, J. Marmur, and I. S. Pretorius, 1996 Muc1, a mucin-like protein that is regulated by Mss10, is critical for pseudohyphal differentiation in yeast. Proc. Natl. Acad. Sci. USA 93: 8419–8424.

Lanver, D., A. Mendoza-Mendoza, A. Brachmann, and R. Kahmann, 2010 Sho1 and Msb2-related proteins regulate appressorium development in the smut fungus Ustilago maydis. Plant Cell 22: 2085–2101.

Latge, J. P., 2010 Tasting the fungal cell wall. Cell. Microbiol. 12: 863–872.

Leberer, E., C. Wu, T. Leeuw, A. Fourest-Lieuvin, J. E. Segall *et al.*, 1997 Functional characterization of the Cdc42p binding domain of yeast Ste20p protein kinase. EMBO J. 16: 83–97.

Leeuw, T., C. Wu, J. D. Schrag, M. Whiteway, D. Y. Thomas *et al.*, 1998 Interaction of a G-protein beta-subunit with a conserved sequence in Ste20/PAK family protein kinases. Nature 391: 191–195.

Lemaire, K., S. Van de Velde, P. Van Dijck, and J. M. Thevelein, 2004 Glucose and sucrose act as agonist and mannose as an antagonist Ligands of the G protein-coupled receptor Gpr1 in the yeast Saccharomyces cerevisiae. Mol. Cell 16: 293–299.

Levin, D. E., 2005 Cell wall integrity signaling in Saccharomyces cerevisiae. Microbiol. Mol. Biol. Rev. 69: 262–291.

Lew, D. J., and S. I. Reed, 1995 A cell cycle checkpoint monitors cell morphogenesis in budding yeast. J. Cell Biol. 129: 739–749.

Lin, X., 2009 Cryptococcus neoformans: morphogenesis, infection, and evolution. Infect. Genet. Evol. 9: 401–416.

Litvinov, S. V., and J. Hilkens, 1993 The epithelial sialomucin, episialin, is sialylated during recycling. J. Biol. Chem. 268: 21364–21371.

Liu, H., C. A. Styles, and G. R. Fink, 1993 Elements of the yeast pheromone response pathway required for filamentous growth of diploids. Science 262: 1741–1744.

Liu, H., C. A. Styles, and G. R. Fink, 1996 Saccharomyces cerevisiae S288C has a mutation in FLO8, a gene required for filamentous growth. Genetics 144: 967–978.

Liu, W., X. Zhou, G. Li, L. Li, L. Kong *et al.*, 2011 Multiple plant surface signals are sensed by different mechanisms in the rice blast fungus for appressorium formation. PLoS Pathog. 7: e1001261.

Liu, Z., and R. A. Butow, 2006 Mitochondrial retrograde signaling. Annu. Rev. Genet. 40: 159–185.

Lo, H. J., J. R. Kohler, B. DiDomenico, D. Loebenberg, A. Caccia-puoti *et al.*, 1997 Nonfilamentous *C. albicans* mutants are avirulent. Cell 90: 939–949.

Lo, W. S., and A. M. Dranginis, 1998 The cell surface flocculin Flo11 is required for pseudohyphae formation and invasion by Saccharomyces cerevisiae. Mol. Biol. Cell 9: 161–171.

Lodder, J., ed, 1970 *The Yeasts: a Taxonomic Study*, North-Holland Publishing, Amsterdam.

Loeb, J. D., T. A. Kerentseva, T. Pan, M. Sepulveda-Becerra, and H. Liu, 1999 Saccharomyces cerevisiae G1 cyclins are differentially involved in invasive and pseudohyphal growth independent of the filamentation mitogen-activated protein kinase pathway. Genetics 153: 1535–1546.

Lorenz, M. C., and J. Heitman, 1997 Yeast pseudohyphal growth is regulated by GPA2, a G protein alpha homolog. EMBO J. 16: 7008–7018.

Lorenz, M. C., and J. Heitman, 1998a Regulators of pseudohyphal differentiation in Saccharomyces cerevisiae identified through multicopy suppressor analysis in ammonium permease mutant strains. Genetics 150: 1443–1457.

Lorenz, M. C., and J. Heitman, 1998b The MEP2 ammonium permease regulates pseudohyphal differentiation in Saccharomyces cerevisiae. EMBO J. 17: 1236–1247.

Lorenz, M. C., N. S. Cutler, and J. Heitman, 2000a Characterization of alcohol-induced filamentous growth in Saccharomyces cerevisiae. Mol. Biol. Cell 11: 183–199.

Lorenz, M. C., X. Pan, T. Harashima, M. E. Cardenas, Y. Xue *et al.*, 2000b The G protein-coupled receptor gpr1 is a nutrient sensor that regulates pseudohyphal differentiation in Saccharomyces cerevisiae. Genetics 154: 609–622.

Ma, J., R. Jin, X. Jia, C. J. Dobry, L. Wang *et al.*, 2007 An interrelationship between autophagy and filamentous growth in budding yeast. Genetics 177: 205–214.

Macao, B., D. G. Johansson, G. C. Hansson, and T. Hard, 2006 Autoproteolysis coupled to protein folding in the SEA domain of the membrane-bound MUC1 mucin. Nat. Struct. Mol. Biol. 13: 71–76.

Madhani, H. D., and G. R. Fink, 1997 Combinatorial control required for the specificity of yeast MAPK signaling. Science 275: 1314–1317.

Madhani, H. D., C. A. Styles, and G. R. Fink, 1997    MAP kinases with distinct inhibitory functions impart signaling specificity during yeast differentiation. Cell 91: 673–684.

Madhani, H. D., T. Galitski, E. S. Lander, and G. R. Fink, 1999    Effectors of a developmental mitogen-activated protein kinase cascade revealed by expression signatures of signaling mutants. Proc. Natl. Acad. Sci. USA 96: 12530–12535.

Maeda, T., M. Takekawa, and H. Saito, 1995    Activation of yeast PBS2 MAPKK by MAPKKKs or by binding of an SH3-containing osmosensor. Science 269: 554–558.

Maeder, C. I., M. A. Hink, A. Kinkhabwala, R. Mayr, P. I. Bastiaens et al., 2007    Spatial regulation of Fus3 MAP kinase activity through a reaction-diffusion mechanism in yeast pheromone signalling. Nat. Cell Biol. 9: 1319–1326.

Mahanty, S. K., Y. Wang, F. W. Farley, and E. A. Elion, 1999    Nuclear shuttling of yeast scaffold Ste5 is required for its recruitment to the plasma membrane and activation of the mating MAPK cascade. Cell 98: 501–512.

Malcher, M., S. Schladebeck, and H. U. Mosch, 2011    The Yak1 protein kinase lies at the center of a regulatory cascade affecting adhesive growth and stress resistance in Saccharomyces cerevisiae. Genetics 187: 717–730.

Maleri, S., Q. Ge, E. A. Hackett, Y. Wang, H. G. Dohlman et al., 2004    Persistent activation by constitutive Ste7 promotes Kss1-mediated invasive growth but fails to support Fus3-dependent mating in yeast. Mol. Cell. Biol. 24: 9221–9238.

Marcus, S., A. Polverino, M. Barr, and M. Wigler, 1994    Complexes between STE5 and components of the pheromone-responsive mitogen-activated protein kinase module. Proc. Natl. Acad. Sci. USA 91: 7762–7766.

Martineau, C. N., J. M. Beckerich, and M. Kabani, 2007    Flo11p-independent control of “mat” formation by hsp70 molecular chaperones and nucleotide exchange factors in yeast. Genetics 177: 1679–1689.

McClean, M. N., A. Mody, J. R. Broach, and S. Ramanathan, 2007    Cross-talk and decision making in MAP kinase pathways. Nat. Genet. 39: 409–414.

Mitchell, A. P., and I. Herskowitz, 1986    Activation of meiosis and sporulation by repression of the RME1 product in yeast. Nature 319: 738–742.

Moran, G. P., D. C. Coleman, and D. J. Sullivan, 2011    Comparative genomics and the evolution of pathogenicity in human pathogenic fungi. Eukaryot. Cell 10: 34–42.

Morillon, A., M. Springer, and P. Lesage, 2000    Activation of the Kss1 invasive-filamentous growth pathway induces Ty1 transcription and retrotransposition in Saccharomyces cerevisiae. Mol. Cell. Biol. 20: 5766–5776.

Mosch, H. U., and G. R. Fink, 1997    Dissection of filamentous growth by transposon mutagenesis in Saccharomyces cerevisiae. Genetics 145: 671–684.

Mosch, H. U., R. L. Roberts, and G. R. Fink, 1996    Ras2 signals via the Cdc42/Ste20/mitogen-activated protein kinase module to induce filamentous growth in Saccharomyces cerevisiae. Proc. Natl. Acad. Sci. USA 93: 5352–5356.

Murray, L. E., N. Rowley, I. W. Dawes, G. C. Johnston, and R. A. Singer, 1998    A yeast glutamine tRNA signals nitrogen status for regulation of dimorphic growth and sporulation. Proc. Natl. Acad. Sci. USA 95: 8619–8624.

Nakafuku, M., T. Obara, K. Kaibuchi, I. Miyajima, A. Miyajima et al., 1988    Isolation of a second yeast Saccharomyces cerevisiae gene (GPA2) coding for guanine nucleotide-binding regulatory protein: studies on its structure and possible functions. Proc. Natl. Acad. Sci. USA 85: 1374–1378.

Nather, K., and C. A. Munro, 2008    Generating cell surface diversity in Candida albicans and other fungal pathogens. FEMS Microbiol. Lett. 285: 137–145.


Neiman, A. M., 2011    Sporulation in the budding yeast Saccharomyces cerevisiae. Genetics 189: 737–765.

Nelson, C., S. Goto, K. Lund, W. Hung, and I. Sadowski, 2003    Srb10/Cdk8 regulates yeast filamentous growth by phosphorylating the transcription factor Ste12. Nature 421: 187–190.

Nelson, W. J., 2003    Mum, this bud's for you: where do you want it? Roles for Cdc42 in controlling bud site selection in Saccharomyces cerevisiae. Bioessays 25: 833–836.

Netea, M. G., and L. Marodi, 2010    Innate immune mechanisms for recognition and uptake of Candida species. Trends Immunol. 31: 346–353.

Netea, M. G., G. D. Brown, B. J. Kullberg, and N. A. Gow, 2008    An integrated model of the recognition of Candida albicans by the innate immune system. Nat. Rev. Microbiol. 6: 67–78.

Ni, L., and M. Snyder, 2001    A genomic study of the bipolar bud site selection pattern in Saccharomyces cerevisiae. Mol. Biol. Cell 12: 2147–2170.

Niranjan, T., X. Guo, J. Victor, A. Lu, and J. P. Hirsch, 2007    Kelch repeat protein interacts with the yeast Galpha subunit Gpa2p at a site that couples receptor binding to guanine nucleotide exchange. J. Biol. Chem. 282: 24231–24238.

Nobile, C. J., and A. P. Mitchell, 2006    Genetics and genomics of Candida albicans biofilm formation. Cell. Microbiol. 8: 1382–1391.

O'Rourke, S. M., and I. Herskowitz, 1998    The Hog1 MAPK prevents cross talk between the HOG and pheromone response MAPK pathways in Saccharomyces cerevisiae. Genes Dev. 12: 2874–2886.

O'Rourke, S. M., and I. Herskowitz, 2002    A third osmosensing branch in Saccharomyces cerevisiae requires the Msb2 protein and functions in parallel with the Sho1 branch. Mol. Cell. Biol. 22: 4739–4749.

Octavio, L. M., K. Gedeon, and N. Maheshri, 2009    Epigenetic and conventional regulation is distributed among activators of FLO11 allowing tuning of population-level heterogeneity in its expression. PLoS Genet. 5: e1000673.

Oehlen, L., and F. R. Cross, 1998    The mating factor response pathway regulates transcription of TEC1, a gene involved in pseudohyphal differentiation of Saccharomyces cerevisiae. FEBS Lett. 429: 83–88.

Ozcan, S., J. Dover, A. G. Rosenwald, S. Wolf, and M. Johnston, 1996    Two glucose transporters in Saccharomyces cerevisiae are glucose sensors that generate a signal for induction of gene expression. Proc. Natl. Acad. Sci. USA 93: 12428–12432.

Ozcan, S., J. Dover, and M. Johnston, 1998    Glucose sensing and signaling by two glucose receptors in the yeast Saccharomyces cerevisiae. EMBO J. 17: 2566–2573.

Palecek, S. P., A. S. Parikh, and S. J. Kron, 2000    Genetic analysis reveals that FLO11 upregulation and cell polarization independently regulate invasive growth in Saccharomyces cerevisiae. Genetics 156: 1005–1023.

Pan, X., and J. Heitman, 1999    Cyclic AMP-dependent protein kinase regulates pseudohyphal differentiation in Saccharomyces cerevisiae. Mol. Cell. Biol. 19: 4874–4887.

Pan, X., and J. Heitman, 2000    Sok2 regulates yeast pseudohyphal differentiation via a transcription factor cascade that regulates cell-cell adhesion. Mol. Cell. Biol. 20: 8364–8372.

Pan, X., and J. Heitman, 2002    Protein kinase A operates a molecular switch that governs yeast pseudohyphal differentiation. Mol. Cell. Biol. 22: 3981–3993.

Papasavvas, S., S. Arkinstall, J. Reid, and M. Payton, 1992    Yeast alpha-mating factor receptor and G-protein-linked adenyl cyclase inhibition requires RAS2 and GPA2 activities. Biochem. Biophys. Res. Commun. 184: 1378–1385.

Park, H. O., and E. Bi, 2007 Central roles of small GTPases in the development of cell polarity in yeast and beyond. Microbiol. Mol. Biol. Rev. 71: 48–96.

Parry, S., H. S. Silverman, K. McDermott, A. Willis, M. A. Hollingsworth *et al.*, 2001 Identification of MUC1 proteolytic cleavage sites in vivo. Biochem. Biophys. Res. Commun. 283: 715–720.

Parsek, M. R., and E. P. Greenberg, 2005 Sociomicrobiology: the connections between quorum sensing and biofilms. Trends Microbiol. 13: 27–33.

Patterson, J. C., E. S. Klimenko, and J. Thorner, 2010 Single-cell analysis reveals that insulation maintains signaling specificity between two yeast MAPK pathways with common components. Sci. Signal. 3: ra75.

Peeters, T., W. Louwet, R. Gelade, D. Nauwelaers, J. M. Thevelein *et al.*, 2006 Kelch-repeat proteins interacting with the Galpha protein Gpa2 bypass adenylate cyclase for direct regulation of protein kinase A in yeast. Proc. Natl. Acad. Sci. USA 103: 13034–13039.

Peeters, T., M. Versele, and J. M. Thevelein, 2007 Directly from Galpha to protein kinase A: the kelch repeat protein bypass of adenylate cyclase. Trends Biochem. Sci. 32: 547–554.

Perez-Nadales, E., and A. Di Pietro, 2011 The membrane mucin Msb2 regulates invasive growth and plant infection in Fusarium oxysporum. Plant Cell 23: 1171–1185.

Peter, M., A. M. Neiman, H. O. Park, M. van Lohuizen, and I. Herskowitz, 1996 Functional analysis of the interaction between the small GTP binding protein Cdc42 and the Ste20 protein kinase in yeast. EMBO J. 15: 7046–7059.

Phan, V. T., V. W. Ding, F. Li, R. J. Chalkley, A. Burlingame *et al.*, 2010 The RasGAP proteins Ira2 and neurofibromin are negatively regulated by Gpb1 in yeast and ETEA in humans. Mol. Cell. Biol. 30: 2264–2279.

Pitoniak, A., B. Birkaya, H. S. Dionne, N. Vadiae, and P. J. Cullen, 2009 The signaling mucins Msb2 and Hkr1 differentially regulate the filamentation MAPK pathway and contribute to a multimodal response. Mol. Biol. Cell 20: 3101–3114.

Pontoppidan, M. B., W. Himaman, N. L. Hywel-Jones, J. J. Boomsma, and D. P. Hughes, 2009 Graveyards on the move: the spatio-temporal distribution of dead ophiocordyceps-infected ants. PLoS ONE 4: e4835.

Posas, F., and H. Saito, 1997 Osmotic activation of the HOG MAPK pathway via Ste11p MAPKKK: scaffold role of Pbs2p MAPKK. Science 276: 1702–1705.

Posas, F., E. A. Witten, and H. Saito, 1998 Requirement of STE50 for osmostress-induced activation of the STE11 mitogen-activated protein kinase kinase in the high-osmolarity glycerol response pathway. Mol. Cell. Biol. 18: 5788–5796.

Printen, J. A., and G. F. Sprague Jr.. 1994 Protein-protein interactions in the yeast pheromone response pathway: Ste5p interacts with all members of the MAP kinase cascade. Genetics 138: 609–619.

Prinz, S., I. Avila-Campillo, C. Aldridge, A. Srinivasan, K. Dimitrov *et al.*, 2004 Control of yeast filamentous-form growth by modules in an integrated molecular network. Genome Res. 14: 380–390.

Pruyne, D., and A. Bretscher, 2000a Polarization of cell growth in yeast. J. Cell Sci. 113(Pt 4): 571–585.

Pruyne, D., and A. Bretscher, 2000b Polarization of cell growth in yeast. I. Establishment and maintenance of polarity states. J. Cell Sci. 113(Pt 3): 365–375.

Pryciak, P. M., and F. A. Huntress, 1998 Membrane recruitment of the kinase cascade scaffold protein Ste5 by the Gbetagamma complex underlies activation of the yeast pheromone response pathway. Genes Dev. 12: 2684–2697.

Raitt, D. C., F. Posas, and H. Saito, 2000 Yeast Cdc42 GTPase and Ste20 PAK-like kinase regulate Sho1-dependent activation of the Hog1 MAPK pathway. EMBO J. 19: 4623–4631.

Ramezani-Rad, M., 2003 The role of adaptor protein Ste50-dependent regulation of the MAPKKK Ste11 in multiple signalling pathways of yeast. Curr. Genet. 43: 161–170.

Ramos-Garcia, S. L., R. W. Roberson, M. Freitag, S. Bartnicki-Garcia, and R. R. Mourino-Perez, 2009 Cytoplasmic bulk flow propels nuclei in mature hyphae of Neurospora crassa. Eukaryot. Cell 8: 1880–1890.

Reid, B. J., and L. H. Hartwell, 1977 Regulation of mating in the cell cycle of Saccharomyces cerevisiae. J. Cell Biol. 75: 355–365.

Remenyi, A., M. C. Good, R. P. Bhattacharyya, and W. A. Lim, 2005 The role of docking interactions in mediating signaling input, output, and discrimination in the yeast MAPK network. Mol. Cell 20: 951–962.

Reynolds, T. B., 2006 The Opilp transcription factor affects expression of FLO11, mat formation, and invasive growth in Saccharomyces cerevisiae. Eukaryot. Cell 5: 1266–1275.

Reynolds, T. B., and G. R. Fink, 2001 Bakers’ yeast, a model for fungal biofilm formation. Science 291: 878–881.

Rispail, N., D. M. Soanes, C. Ant, R. Czajkowski, A. Grunler *et al.*, 2009 Comparative genomics of MAP kinase and calcium-calcineurin signalling components in plant and human pathogenic fungi. Fungal Genet. Biol. 46: 287–298.

Roberts, C. J., B. Nelson, M. J. Marton, R. Stoughton, M. R. Meyer *et al.*, 2000 Signaling and circuitry of multiple MAPK pathways revealed by a matrix of global gene expression profiles. Science 287: 873–880.

Roberts, R. L., and G. R. Fink, 1994 Elements of a single MAP kinase cascade in Saccharomyces cerevisiae mediate two developmental programs in the same cell type: mating and invasive growth. Genes Dev. 8: 2974–2985.

Robertson, L. S., and G. R. Fink, 1998 The three yeast A kinases have specific signaling functions in pseudohyphal growth. Proc. Natl. Acad. Sci. USA 95: 13783–13787.

Robertson, L. S., H. C. Causton, R. A. Young, and G. R. Fink, 2000 The yeast A kinases differentially regulate iron uptake and respiratory function. Proc. Natl. Acad. Sci. USA 97: 5984–5988.

Rodicio, R., and J. J. Heinisch, 2010 Together we are strong: cell wall integrity sensors in yeasts. Yeast 27: 531–540.

Roman, E., F. Cottier, J. F. Ernst, and J. Pla, 2009 Msb2 signaling mucin controls activation of Cek1 mitogen-activated protein kinase in Candida albicans. Eukaryot. Cell 8: 1235–1249.

Roth, A. F., and N. G. Davis, 1996 Ubiquitination of the yeast a-factor receptor. J. Cell Biol. 134: 661–674.

Roth, A. F., and N. G. Davis, 2000 Ubiquitination of the PEST-like endocytosis signal of the yeast a-factor receptor. J. Biol. Chem. 275: 8143–8153.

Rupp, S., E. Summers, H. J. Lo, H. Madhani, and G. Fink, 1999 MAP kinase and cAMP filamentation signaling pathways converge on the unusually large promoter of the yeast FLO11 gene. EMBO J. 18: 1257–1269.

Rutherford, J. C., G. Chua, T. Hughes, M. E. Cardenas, and J. Heitman, 2008 A Mep2-dependent transcriptional profile links permease function to gene expression during pseudohyphal growth in Saccharomyces cerevisiae. Mol. Biol. Cell 19: 3028–3039.

Sabbagh, W. Jr. L. J. Flatauer, A. J. Bardwell, and L. Bardwell, 2001 Specificity of MAP kinase signaling in yeast differentiation involves transient vs. sustained MAPK activation. Mol. Cell 8: 683–691.

Saito, H., 2010 Regulation of cross-talk in yeast MAPK signaling pathways. Curr. Opin. Microbiol. 13: 677–683.

Sanders, S. L., and C. M. Field, 1995 Cell division. Bud-site selection is only skin deep. Curr. Biol. 5: 1213–1215.

Schroder, M., J. S. Chang, and R. J. Kaufman, 2000 The unfolded protein response represses nitrogen-starvation induced

developmental differentiation in yeast. Genes Dev. 14: 2962–2975.

Schroder, M., R. Clark, C. Y. Liu, and R. J. Kaufman, 2004 The unfolded protein response represses differentiation through the RPD3-SIN3 histone deacetylase. EMBO J. 23: 2281–2292.

Selmecki, A., A. Forche, and J. Berman, 2010 Genomic plasticity of the human fungal pathogen Candida albicans. Eukaryot. Cell 9: 991–1008.

Sengupta, S., T. R. Peterson, and D. M. Sabatini, 2010 Regulation of the mTOR complex 1 pathway by nutrients, growth factors, and stress. Mol. Cell 40: 310–322.

Sheu, Y. J., Y. Barral, and M. Snyder, 2000 Polarized growth controls cell shape and bipolar bud site selection in Saccharomyces cerevisiae. Mol. Cell. Biol. 20: 5235–5247.

Shimada, Y., P. Wiget, M. P. Gulli, E. Bi, and M. Peter, 2004 The nucleotide exchange factor Cdc24p may be regulated by autoinhibition. EMBO J. 23: 1051–1062.

Shock, T. R., J. Thompson, J. R. Yates 3rd. and H. D. Madhani, 2009 Hog1 mitogen-activated protein kinase (MAPK) interrupts signal transduction between the Kss1 MAPK and the Tec1 transcription factor to maintain pathway specificity. Eukaryot. Cell 8: 606–616.

Sia, R. A., E. S. Bardes, and D. J. Lew, 1998 Control of Swel p degradation by the morphogenesis checkpoint. EMBO J. 17: 6678–6688.

Silverman, H. S., M. Sutton-Smith, K. McDermott, P. Heal, S. H. Leir et al., 2003 The contribution of tandem repeat number to the O-glycosylation of mucins. Glycobiology 13: 265–277.

Simon, M. N., C. De Virgilio, B. Souza, J. R. Pringle, A. Abo et al., 1995 Role for the Rho-family GTPase Cdc42 in yeast mating-pheromone signal pathway. Nature 376: 702–705.

Singh, P. K., and M. A. Hollingsworth, 2006 Cell surface-associated mucins in signal transduction. Trends Cell Biol. 16: 467–476.

Singh, S. R., B. Pillai, B. Balakrishnan, A. Naorem, and P. P. Sadhale, 2007 Relative levels of RNA polII subunits differentially affect starvation response in budding yeast. Biochem. Biophys. Res. Commun. 356: 266–272.

Smukalla, S., M. Caldara, N. Pochet, A. Beauvais, S. Guadagnini et al., 2008 FLO1 is a variable green beard gene that drives biofilm-like cooperation in budding yeast. Cell 135: 726–737.

St'ovicek, V., L. Vachova, M. Kuthan, and Z. Palkova, 2010 General factors important for the formation of structured biofilm-like yeast colonies. Fungal Genet. Biol. 47: 1012–1022.

Stefan, C. J., and K. J. Blumer, 1994 The third cytoplasmic loop of a yeast G-protein-coupled receptor controls pathway activation, ligand discrimination, and receptor internalization. Mol. Cell. Biol. 14: 3339–3349.

Steinberg, G., 2007 Hyphal growth: a tale of motors, lipids, and the Spitzenkorper. Eukaryot. Cell 6: 351–360.

Strittmatter, A. W., C. Fischer, M. Kleinschmidt, and G. H. Braus, 2006 FLO11 mediated filamentous growth of the yeast Saccharomyces cerevisiae depends on the expression of the ribosomal RPS26 genes. Mol. Genet. Genomics 276: 113–125.

Strudwick, N., M. Brown, V. M. Parmar, and M. Schroder, 2010 Ime1 and Ime2 are required for pseudohyphal growth of Saccharomyces cerevisiae on nonfermentable carbon sources. Mol. Cell. Biol. 30: 5514–5530.

Taheri, N., T. Kohler, G. H. Braus, and H. U. Mosch, 2000 Asymmetrically localized Bud8p and Bud9p proteins control yeast cell polarity and development. EMBO J. 19: 6686–6696.

Tan, P. K., J. P. Howard, and G. S. Payne, 1996 The sequence NPFXD defines a new class of endocytosis signal in Saccharomyces cerevisiae. J. Cell Biol. 135: 1789–1800.

Tatebayashi, K., K. Yamamoto, K. Tanaka, T. Tomida, T. Maruoka et al., 2006 Adaptor functions of Cdc42, Ste50, and Sho1 in the yeast osmoregulatory HOG MAPK pathway. EMBO J. 25: 3033–3044.

Tatebayashi, K., K. Tanaka, H. Y. Yang, K. Yamamoto, Y. Matsushita et al., 2007 Transmembrane mucins Hkr1 and Msb2 are putative osmosensors in the SHO1 branch of yeast HOG pathway. EMBO J. 26: 3521–3533.

Tedford, K., S. Kim, D. Sa, K. Stevens, and M. Tyers, 1997 Regulation of the mating pheromone and invasive growth responses in yeast by two MAP kinase substrates. Curr. Biol. 7: 228–238.

Thevelein, J. M., and K. Voordeckers, 2009 Functioning and evolutionary significance of nutrient transceptors. Mol. Biol. Evol. 26: 2407–2414.

Toda, T., I. Uno, T. Ishikawa, S. Powers, T. Kataoka et al., 1985 In yeast, RAS proteins are controlling elements of adenylate cyclase. Cell 40: 27–36.

Toda, T., S. Cameron, P. Sass, M. Zoller, and M. Wigler, 1987 Three different genes in S. cerevisiae encode the catalytic subunits of the cAMP-dependent protein kinase. Cell 50: 277–287.

Tong, A. H., M. Evangelista, A. B. Parsons, H. Xu, G. D. Bader et al., 2001 Systematic genetic analysis with ordered arrays of yeast deletion mutants. Science 294: 2364–2368.

Tong, Z., X. D. Gao, A. S. Howell, I. Bose, D. J. Lew et al., 2007 Adjacent positioning of cellular structures enabled by a Cdc42 GTPase-activating protein-mediated zone of inhibition. J. Cell Biol. 179: 1375–1384.

Truckses, D. M., J. E. Bloomekatz, and J. Thorner, 2006 The RA domain of Ste50 adaptor protein is required for delivery of Ste11 to the plasma membrane in the filamentous growth signaling pathway of the yeast Saccharomyces cerevisiae. Mol. Cell. Biol. 26: 912–928.

Vadaie, N., H. Dionne, D. S. Akajagbor, S. R. Nickerson, D. J. Krysan et al., 2008 Cleavage of the signaling mucin Msb2 by the aspartyl protease Yps1 is required for MAPK activation in yeast. J. Cell Biol. 181: 1073–1081.

Vagnoli, P., and L. F. Bisson, 1998 The SKS1 gene of Saccharomyces cerevisiae is required for long-term adaptation of snf3 null strains to low glucose. Yeast 14: 359–369.

Van de Velde, S., and J. M. Thevelein, 2008 Cyclic AMP-protein kinase A and Snf1 signaling mechanisms underlie the superior potency of sucrose for induction of filamentation in Saccharomyces cerevisiae. Eukaryot. Cell 7: 286–293.

van Drogen, F., V. M. Stucke, G. Jorritsma, and M. Peter, 2001 MAP kinase dynamics in response to pheromones in budding yeast. Nat. Cell Biol. 3: 1051–1059.

van Dyk, D., G. Hansson, I. S. Pretorius, and F. F. Bauer, 2003 Cellular differentiation in response to nutrient availability: the repressor of meiosis, Rmelp, positively regulates invasive growth in Saccharomyces cerevisiae. Genetics 165: 1045–1058.

van Dyk, D., I. S. Pretorius, and F. F. Bauer, 2005 Mss11lp is a central element of the regulatory network that controls FLO11 expression and invasive growth in Saccharomyces cerevisiae. Genetics 169: 91–106.

Verstrepen, K. J., and F. M. Klis, 2006 Flocculation, adhesion and biofilm formation in yeasts. Mol. Microbiol. 60: 5–15.

Verstrepen, K. J., T. B. Reynolds, and G. R. Fink, 2004 Origins of variation in the fungal cell surface. Nat. Rev. Microbiol. 2: 533–540.

Verstrepen, K. J., A. Jansen, F. Lewitter, and G. R. Fink, 2005 Intragenic tandem repeats generate functional variability. Nat. Genet. 37: 986–990.

Vyas, V. K., S. Kuchin, C. D. Berkey, and M. Carlson, 2003 Snfl kinases with different beta-subunit isoforms play distinct roles in regulating haploid invasive growth. Mol. Cell. Biol. 23: 1341–1348.

Wang, B., X. Liu, W. Wu, X. Liu, and S. Li, 2009a Purification, characterization, and gene cloning of an alkaline serine protease from a highly virulent strain of the nematode-endoparasitic fungus Hirsutella rhossiliensis. Microbiol. Res. 164: 665–673.

Wang, Y., A. Abu Irqeba, M. Ayalew, and K. Suntay, 2009b Sumoylation of transcription factor Tec1 regulates signaling of mitogen-activated protein kinase pathways in yeast. PLoS ONE 4: e7456.

Ward, M. P., C. J. Gimeno, G. R. Fink, and S. Garrett, 1995 SOK2 may regulate cyclic AMP-dependent protein kinase-stimulated growth and pseudohyphal development by repressing transcription. Mol. Cell. Biol. 15: 6854–6863.

Westfall, P. J., and J. Thorner, 2006 Analysis of mitogen-activated protein kinase signaling specificity in response to hyperosmotic stress: use of an analog-sensitive HOG1 allele. Eukaryot. Cell 5: 1215–1228.

Whiteway, M., and C. Bachewich, 2007 Morphogenesis in Candida albicans. Annu. Rev. Microbiol. 61: 529–553.

Whiteway, M. S., C. Wu, T. Leeuw, K. Clark, A. Fourest-Lieuvin et al., 1995 Association of the yeast pheromone response G protein beta gamma subunits with the MAP kinase scaffold Ste5p. Science 269: 1572–1575.

Winters, M. J., R. E. Lamson, H. Nakanishi, A. M. Neiman, and P. M. Pryciak, 2005 A membrane binding domain in the ste5 scaffold synergizes with gbetagamma binding to control localization and signaling in pheromone response. Mol. Cell 20: 21–32.

Wu, C., G. Jansen, J. Zhang, D. Y. Thomas, and M. Whiteway, 2006 Adaptor protein Ste50p links the Ste11p MEKK to the HOG pathway through plasma membrane association. Genes Dev. 20: 734–746.

Xiong, W., and J. E. Ferrell Jr., 2003 A positive-feedback-based bistable 'memory module' that governs a cell fate decision. Nature 426: 460–465.

Xu, T., C. A. Shively, R. Jin, M. J. Eckwahl, C. J. Dobry et al., 2010 A profile of differentially abundant proteins at the yeast cell periphery during pseudohyphal growth. J. Biol. Chem. 285: 15476–15488.

Xue, Y., M. Batlle, and J. P. Hirsch, 1998 GPR1 encodes a putative G protein-coupled receptor that associates with the Gpa2p Galpha subunit and functions in a Ras-independent pathway. EMBO J. 17: 1996–2007.

Yablonski, D., I. Marbach, and A. Levitzki, 1996 Dimerization of Ste5, a mitogen-activated protein kinase cascade scaffold protein, is required for signal transduction. Proc. Natl. Acad. Sci. USA 93: 13864–13869.

Yamamoto, K., K. Tatebayashi, K. Tanaka, and H. Saito, 2010 Dynamic control of yeast MAP kinase network by induced association and dissociation between the Ste50 scaffold and the Opy2 membrane anchor. Mol. Cell 40: 87–98.

Yang, H. Y., K. Tatebayashi, K. Yamamoto, and H. Saito, 2009 Glycosylation defects activate filamentous growth Kss1 MAPK and inhibit osmoregulatory Hog1 MAPK. EMBO J. 28: 1380–1391.

Yu, L., M. Qi, M. A. Sheff, and E. A. Elion, 2008 Counteractive control of polarized morphogenesis during mating by mitogen-

activated protein kinase Fus3 and G1 cyclin-dependent kinase. Mol. Biol. Cell 19: 1739–1752.

Yun, C. W., H. Tamaki, R. Nakayama, K. Yamamoto, and H. Kumagai, 1997 G-protein coupled receptor from yeast Saccharomyces cerevisiae. Biochem. Biophys. Res. Commun. 240: 287–292.

Yuzyuk, T., and D. C. Amberg, 2003 Actin recovery and bud emergence in osmotically stressed cells requires the conserved actin-interacting mitogen-activated protein kinase kinase Ssk2p/MTK1 and the scaffold protein Spa2p. Mol. Biol. Cell 14: 3013–3026.

Yuzyuk, T., M. Foehr, and D. C. Amberg, 2002 The MEK kinase Ssk2p promotes actin cytoskeleton recovery after osmotic stress. Mol. Biol. Cell 13: 2869–2880.

Zaman, S., S. I. Lippman, X. Zhao, and J. R. Broach, 2008 How Saccharomyces responds to nutrients. Annu. Rev. Genet. 42: 27–81.

Zarrinpar, A., R. P. Bhattacharyya, M. P. Nittler, and W. A. Lim, 2004 Sho1 and Pbs2 Act as coscaffolds linking components in the yeast high osmolarity MAP kinase pathway. Mol. Cell 14: 825–832.

Zebisch, A., A. P. Czernilofsky, G. Keri, J. Smigelskaite, H. Sill et al., 2007 Signaling through RAS-RAF-MEK-ERK: from basics to bedside. Curr. Med. Chem. 14: 601–623.

Zeitlinger, J., I. Simon, C. T. Harbison, N. M. Hannett, T. L. Volkert et al., 2003 Program-specific distribution of a transcription factor dependent on partner transcription factor and MAPK signaling. Cell 113: 395–404.

Zeller, G. E., S. C. Parnell, and H. G. Dohlman, 2007 The RACK1 ortholog Ascl1 functions as a G-protein beta subunit coupled to glucose responsiveness in yeast. J. Biol. Chem. 282: 25168–25176.

Zhang, Z., M. M. Smith, and J. S. Mymryk, 2001 Interaction of the E1A oncoprotein with Yak1p, a novel regulator of yeast pseudohyphal differentiation, and related mammalian kinases. Mol. Biol. Cell 12: 699–710.

Zhao, X., R. Mehrabi, and J. R. Xu, 2007 Mitogen-activated protein kinase pathways and fungal pathogenesis. Eukaryot. Cell 6: 1701–1714.

Zhao, Z. S., T. Leung, E. Manser, and L. Lim, 1995 Pheromone signalling in Saccharomyces cerevisiae requires the small GTP-binding protein Cdc42p and its activator CDC24. Mol. Cell. Biol. 15: 5246–5257.

Zhu, G., P. T. Spellman, T. Volpe, P. O. Brown, D. Botstein et al., 2000 Two yeast forkhead genes regulate the cell cycle and pseudohyphal growth. Nature 406: 90–94.

Ziv, C., G. Kra-Oz, R. Gorovits, S. Marz, S. Seiler et al., 2009 Cell elongation and branching are regulated by differential phosphorylation states of the nuclear Dbf2-related kinase COT1 in Neurospora crassa. Mol. Microbiol. 74: 974–989.
