
REVIEW

# Structural insights into the coupling of virion assembly and rotavirus replication

**Shane D. Trask**¹, **Sarah M. McDonald**² and **John T. Patton**¹

**Abstract** | Viral replication is rapid and robust, but it is far from a chaotic process. Instead, successful production of infectious progeny requires that events occur in the correct place and at the correct time. Rotaviruses (segmented double-stranded RNA viruses of the Reoviridae family) seem to govern their replication through ordered disassembly and assembly of a triple-layered icosahedral capsid. In recent years, high-resolution structural data have provided unprecedented insight into these events. In this Review, we explore the current understanding of rotavirus replication and how it compares to replication of other Reoviridae family members.

---

### Segmented
Pertaining to a virus: with a genome comprising multiple distinct nucleic acid molecules; each segment is analogous to a eukaryotic chromosome, but usually encodes only 1–3 proteins.

### Icosahedral
With 30 two-fold, 20 three-fold and 12 five-fold axes of rotation (like an icosahedron or a dodecahedron). A type of symmetry common to viral capsids.

Viruses perform numerous tasks during their replication cycles — from entry into the host cell, to viral protein production and genome replication, to the assembly and egress of nascent particles. Yet, for many viruses, it is not entirely clear how each stage of the replication cycle is controlled so that processes occur at appropriate times. Segmented, double-stranded RNA (dsRNA) viruses of the Reoviridae family seem to achieve such regulation through ordered, stepwise particle disassembly and assembly. This strategy is particularly true for rotaviruses, which are well-studied Reoviridae family members that are important human and veterinary gastrointestinal pathogens. The infectious rotavirus virion is an icosahedral particle composed of three concentric protein layers surrounding 11 dsRNA genome segments¹. Loss of the outermost virion layer during entry is intimately tied to membrane penetration² and subsequent transcription of viral positive-sense RNAs ((+)RNAs) by polymerase complexes in the particle interior³,⁴. Likewise, during the early stages of particle assembly, interaction of the polymerase complexes with the innermost protein layer triggers the initiation of viral genome replication⁵,⁶. The subsequent addition of the intermediate and outer layers during particle morphogenesis effectively halts RNA synthesis and directs the rotavirus towards host cell egress. The lack of high-resolution structures for rotavirus assembly intermediates had previously hampered our understanding of assembly state-mediated regulation of replication. In recent years, however, several studies using X-ray crystallography and cryo-electron microscopy have defined structures of the rotavirus virion, subviral particles and individual proteins¹,³,⁷⁻¹². In this Review, we explore the current model of rotavirus replication, which is based primarily on this new structural information but is in accordance with numerous biochemical results. Importantly, although we focus on rotaviruses, the most medically important members of the family Reoviridae, we also incorporate relevant information from other well-studied viruses in this family, such as mammalian orthoreovirus (referred to hereafter as reovirus) and bluetongue virus.

## Architecture of the rotavirus virion
The structures of infectious and subviral rotavirus particles have been solved to near-atomic resolution using X-ray crystallography and single-particle reconstructions of cryo-electron microscopy images¹,³,¹¹,¹² (FIG. 1). At approximately 100 nm in diameter, the infectious triple-layered particle (TLP) is large compared with many other non-enveloped, icosahedral viruses. The innermost layer of the TLP, referred to as the core shell, immediately surrounds the viral dsRNA genome. The core shell is composed of 120 copies of VP2 (102 kDa), with each asymmetrical unit formed of a dimer (rather than a monomer), and hence the VP2 layer is most accurately described as having a triangulation number (T number) of 1 (REF. 11). To achieve this organization, VP2 monomers in each dimer unit adopt slightly different conformations. One conformation, VP2-A, converges tightly around the five-fold vertices, whereas the other conformation, VP2-B, sits further back and intercalates

---

**Rotavirus Molecular Biology Section, National Institute of Allergy and Infectious Diseases, National Institutes of Health, Bethesda, Maryland 20892, USA.**

**Virginia Tech Carilion Research Institute and Department of Biomedical Sciences and Pathobiology, College of Veterinary Medicine, Virginia Tech, 2 Riverside Circle, Roanoke, Virginia 24016, USA.**

**Correspondence to J.T.P.**  
**e-mail:** `jpatton@niaid.nih.gov`  
**doi:** 10.1038/nrmicro2673  
**Published online 23 January 2012**

REVIEW S

Rotavirus
![](VP2-A)
![](VP2-B)

Reovirus

Transcriptionally active subviral particle Infectious virion

Figure 1 | The rotavirus and mammalian orthoreovirus virions. A comparison of the structures of rotavirus (non-turreted) and mammalian orthoreovirus (reovirus; turreted) particles, coloured by functional similarity but not necessarily by structural homology. The rotavirus double-layered particle (DLP) (Protein Data Bank (PDB) ID 3KZ4), shown in the middle two structures, is the functional equivalent of the reovirus core (PDB ID 1EJ6)—the transcriptionally active subviral particle<sup>11,13</sup>. The first view is of a decamer formed by the inner capsid protein, VP2, which has a triangulation number (T) of 1; alternately coloured VP2-A and VP2-B subunits of the five-fold-symmetrical decamer illustrate the arrangement of the capsid. In outline are a single T = 1 asymmetrical unit formed by one VP2-A conformer and one VP2-B conformer. In the second view (a cut-away of the subviral particle), a single decamer (light blue) in the T = 1 core is highlighted. Structural evidence suggests that VP1 (not shown; see FIG. 5) binds specific positions on the core near the five-fold vertices<sup>15,16</sup>. The locations of the rotavirus RNA-capping enzyme (VP3) in the virion are not known, whereas the large turrets formed by the reovirus RNA-capping enzyme (λ2) are clearly visible in the third view. A continuous shell of rotavirus VP6 and a discontinuous layer of reovirus σ2 stabilize the inner capsid and interface with the outer capsid proteins<sup>3,11,13</sup>. The final view shows that the infectious rotavirus virion (PDB IDs 3IYU and 3N09) is coated with the trimeric protein VP7 and decorated with the VP5*–VP8* spike complex, which mediates attachment and entry<sup>1</sup>. By contrast, the reovirus core is covered with a layer of the trimeric membrane penetration protein, μ1, which is studded with the chaperone protein, σ3, to form the infectious virion (PDB ID 2CSE)<sup>19</sup>. The flexible trimeric reovirus attachment fibre, σ1, which extends from the five-fold-symmetrical turrets, has not been fully resolved in any particle reconstruction<sup>106</sup>, but the structure of the head domain has been solved independently<sup>107</sup>. The diameter of the mature virion in both cases is approximately 80 nm (excluding spikes)<sup>3,19</sup>.

Positive-sense RNAs ((+)RNAs). The RNA strands that directly encode the viral proteins and are used as templates to synthesize the negative-sense strands during replication of the genome.

Triangulation number (T number). The number of symmetrically distinct subunits that make up each of the asymmetric units of the capsid. Generally, larger virions require higher T numbers (that is, more subunits) to form their capsids.

Five-fold vertices The points on an icosahedron; for viruses of the family Reoviridae, these are the 12 axes of five-fold symmetry and the sites of positive-sense RNA extrusion.

RNA-capping enzyme An enzyme that is responsible for modifying the 5′ end of an RNA to generate a cap structure that is similar to that of eukaryotic mRNAs.

between adjacent VP2-A molecules<sup>11</sup> (FIG. 1). Both forms of VP2 fold into thin, comma-shaped plates; flexible hinge regions between the three subdomains (apical, central and dimerization) of VP2 allow subtle structural differences between the A and B conformations<sup>11</sup>. The extreme amino-terminal residues of VP2 (~1–100 of VP2-A and ~1–80 of VP2-B) are not resolved in any known rotavirus structure<sup>1,3,11,12</sup>. This flexible region of the protein resides in the particle interior, projects towards the five-fold vertex and is thought to engage the viral polymerase complex (composed of a single copy each of the RNA-dependent RNA polymerase (VP1; 125 kDa) and the viral RNA-capping enzyme (VP3; 88 kDa))<sup>11</sup>. Thus, it is currently assumed that most (if not all) five-fold vertices have a single polymerase complex, which is anchored in place via simultaneous interactions with the subdomains of multiple VP2-A and VP2-B conformers and their N-terminal tethers<sup>6</sup>. Although the overall T = 1 architecture is conserved for the inner layers of viral particles from the family Reoviridae, rotaviruses are the only members currently known to have such an internal organization<sup>11,13,14</sup>. By contrast, for reovirus and other turreted viruses of the Reoviridae family, the core shell has an externally protruding complex at each five-fold vertex, made up of a pentameric viral RNA-capping enzyme<sup>13</sup> (FIG. 1). For the Reoviridae viruses, the polymerase is bound as a monomer directly to the inside of the core shell, at a defined position slightly off-centre from the five-fold vertex (that is, there are five identical binding sites at each vertex, only one of which is occupied by the polymerase; this reflects the inherent symmetry mismatch between the monomeric polymerase and five-fold-symmetrical core shell)<sup>15,16</sup>.

Surrounding the rotavirus VP2 shell are two additional protein layers, both of which have T = 13 icosahedral symmetry (FIG. 1). The intermediate layer is thick compared with the other two layers and is made up of 260 trimers of VP6 (45 kDa monomer). VP6 is composed of two domains, a distal jelly-roll β-barrel and a proximal α-helical pedestal, both of which participate in extensive

Box 1 | The grip arm mode of VP7 assembly

Despite occupying a position directly on top of the VP6 trimers, the bottom face of VP7 and the top of VP6 have minimal contacts (according to the structure of a rotavirus double-layered particle that was ‘recoated’ with recombinant VP7; Protein Data Bank IDs 3GZT and 3GZU)³. Instead, the flexible amino terminus of VP7 latches onto a small protrusion formed by a loop in the VP6 β-jellyroll domain³ (see the figure). The amino terminus of mature VP7 begins at residue Gln51 (residues 1–50 are the cleaved signal peptide). Residues 58–62 interact with VP6 to effectively extend the β-sheet of VP6 by an additional strand (see the figure, right panel), whereas residues 63–78 curve under the VP6 protrusion and lead up into the main VP7 structure³. A glycosylation site (Asn69) in VP7 that is conserved among many group A rotaviruses is actually in the grip arm and below the virion surface (see the figure, middle panel (asterisk) and right panel)³. In certain VP7 conformers, additional amino-terminal residues (51–57) can be modelled and seem to crossover between VP7 trimers and mediate many of the trimer–trimer contacts of the VP7 layer (see the figure, left panel)³. This capsid protein ‘networking’ is similar to that seen with other virions, notably that of papillomaviruses, in which amino- and carboxy-terminal arms lash the pentameric capsomers together¹⁰⁵. Thus, the flexible amino-terminal arms of VP7 are probably responsible for many of the interactions that stabilize the outer capsid on the rotavirus virion. It is likely that these interactions are weak for a single VP7 monomer, consistent with the requirement for calcium-dependent trimerization of VP7 to assemble⁸⁴. Calcium binding and the context-sensitive interaction with VP6 (and between VP7 trimers) seem to have evolved as exquisite mechanisms to govern the highly cooperative assembly and uncoating properties of the rotavirus outer capsid²,⁸⁴.

![Figure](https://i.imgur.com/yourimageurl.png)

contacts with neighbouring subunits as they twist around each other to form a trimer¹¹,¹⁷. The symmetry mismatch with the underlying VP2 core shell results in five distinct positions for VP6 trimers. Binding of VP6 to VP2 leads to a dramatic stabilization of the very fragile core. Thus, VP6 is functionally analogous to the discontinuous σ2 protein clamps that bridge the subunits of the core shell in the reovirus particle¹³. VP6 also serves as an adapter for the rotavirus outer capsid proteins, which are crucial for attachment and entry into a host cell. Specifically, 260 trimers of the glycoprotein VP7 (37 kDa monomer) sit directly on top of the VP6 trimers and form a continuous, perforated shell. VP7 trimers are dependent on bound calcium ions for stability; two calcium ions are held at each subunit interface, requiring six total bound ions per trimer⁷. Arm-like extensions formed by the VP7 N termini account for nearly all contacts with VP6 (REFS 1,3) (BOX 1). In addition to allowing VP7 to grip the intermediate layer, the N-terminal arms of VP7 create lattice contacts with other VP7 trimers, thereby reinforcing the outer-shell integrity and allowing cooperative interaction of adjacent VP7 trimers³ (BOX 1). Protruding through the VP7 layer on the rotavirus virion are 60 trimeric spikes, 120 Å in length, that emanate from the peripentonal channels of the VP6 layer (FIG. 1). These spikes are formed by the viral attachment protein, VP4 (88 kDa), and they undergo substantial conformational changes during penetration of the host cell membrane,

Peripentonal channels  
For rotaviruses, the approximately six-fold-symmetrical ‘gaps’ in the VP6 layer that surround each of the five-fold vertices. These channels are the binding sites for VP4 trimers during assembly of the viral particle.

Serotype  
A classification of rotaviruses using comparative neutralization by monoclonal antibodies; for rotaviruses, there are two outer capsid proteins (VP4 and VP7), giving a binary serotype.

REVIEWS

Box 2 | Structurally defined rotavirus epitopes

The outer capsid proteins VP7 and VP4 (including the VP4 cleavage products, VP5* and VP8*) are the primary targets of rotavirus-neutralizing antibodies. The glycoprotein VP7 (the G antigen) and the protease-sensitive spike protein, VP4 (the P antigen), are used to classify the strain serotype of rotaviruses (for example, G1P[8]) based on sequence comparison and reactivity with neutralizing antibodies. Mapping of antibody escape mutations has led to the identification of several discrete epitopes in each outer capsid protein. The recent structures have allowed coherent, structure-based epitopes to be defined. For example, six epitopes (A–F) were mapped to VP7 by neutralization escape. By plotting these targets onto the surface of the VP7 trimer, it is immediately apparent that there are, in fact, only two unique areas that seem to be targeted by neutralizing antibodies (see the figure, part a). Epitope 7-1 lies at the three corners of each trimer and comprises residues from two adjacent VP7 subunits (and can be subdivided into 7-1a and 7-1b by this fact). Antibodies that target epitope 7-1 mostly neutralize the virus by stabilizing the capsid and preventing uncoating; indeed, epitope 7-1 includes several amino acids that are proximal to one of the calcium-binding sites which maintain the trimeric conformation of VP7. Epitope 7-2 is in the flexible region in the centre of each VP7 subunit; there is less certainty about the mechanism (or mechanisms) of neutralization by antibodies that target this epitope.

In the mature virion, VP4 is cleaved by trypsin into VP5* and VP8* (see the figure, part b). VP8* has four structurally defined epitopes. Most neutralizing antibodies directed against VP8* block virus attachment. VP5*, the membrane penetration protein, has five structurally defined epitopes. The specific mechanism of neutralization of most VP5*-specific antibodies has not been determined; however, those antibodies that are directed against epitope 5-1, at the apical hydrophobic loops (see the main text, FIG. 3), seem to block the association of VP5* with membranes (and, presumably, therefore block membrane penetration).

![Figure](https://via.placeholder.com/600x400?text=Image+Description)

Changes in the outer capsid drive entry

Rotaviruses infect enterocytes of small-intestinal villi and replicate exclusively in the cell cytoplasm (FIG. 2). Newly assembled rotavirus virions are not fully infectious; for membrane penetration, the VP4 spike protein must be proteolytically cleaved (primed) into two fragments, VP8* (28 kDa) and VP5* (60 kDa), by trypsin-like proteases of the host gastrointestinal tract. These cleavage products remain non-covalently associated with each other on the mature virion surface (FIG. 3). Structural analysis of VP4 from TLPs before and after trypsin cleavage revealed a rigidification of the spike upon proteolysis. The primed spike contains an unusual mix of trimeric, dimeric and asymmetrical elements (FIG. 3). Specifically, the proximal VP5* portion of the spike is trimeric at its base, where it is sandwiched between the VP6 and VP7 layers. As VP5* extends away from the virion, however, two of the three subunits form an extended pair, while the third VP5* subunit lays nearly flat to the virion surface. Two VP8* molecules cap the distal ends of the two upright VP5* subunits and project their extended N termini down into the base of the spike, whereas the third VP8* subunit is presumed to dissociate from the virion. Consistent with its position at the tip of the cleaved spike, VP8* seems to mediate host cell attachment for many rotavirus strains. Consequently, most neutralizing antibodies that target VP8* block viral attachment (BOX 2). The structure of the VP8* fragment alone has been solved for several different rotavirus strains. This protein has a galectin-like fold and, for many strains, can bind sialic acid moieties in vitro. However, not all rotavirus strains require terminal sialic acid for attachment in cell culture experiments. This observation suggests that alternative cell surface molecules can serve as functional receptors for rotaviruses, possibly including glycans with internal sialic acid groups. An ongoing focus of many laboratories is the identification and characterization of these undefined receptors.

In structures of the intact, primed virion, the VP5* spike conformation seems static, but several studies have shown that this protein has the capacity to undergo a substantial molecular movement. Membrane penetration requires the exposure of three hydrophobic loops in VP5*, which are tucked below the VP8* lobes in the case of the two upright conformers and nestled into the base of the upright VP5* subunits in the case of the third subunit (FIG. 3). When soluble VP4 is proteolytically cleaved in vitro, it bypasses the spike conformation altogether and forms an inverted, highly stable trimer that corresponds to the central region of VP5* (REFS 8, 35); it has been proposed that this is the 'post-penetration' conformation of VP5* (REF. 8). Rather than the elongated, quasi-asymmetrical spike, this umbrella-shaped structure folds backwards onto itself and has at its core a triple-helical coiled-coil (FIG. 3). This apparent 'fold back' mechanism, which is required to expose the hydrophobic loops in VP5*, closely parallels the maturation and refolding pathway that occurs in many viral membrane fusion proteins during entry (FIG. 3). In this pathway, a precursor protein (for example, the mature haemagglutinin (HA) trimer of influenza viruses, or the rotavirus VP4 spike) is proteolytically primed to form a metastable intermediate (the spike), which is then triggered to interact with a target membrane and refold (into the postfusion HA structure or the VP5* umbrella) to destabilize the membrane. Support for this model is growing, as it has recently been reported that

Neutralizing antibodies

Antibodies that block infectivity (for example, of a virus), usually by binding to the foreign particle (the virion) and incapacitating it in some way.

Attachment and internalization

Maturation (VP4 proteolysis)

Exocytosis

Lysis

IFN antagonist

Host translation shut-off

NSP1

NSP3

Early endosome

Translation

Cap (+)RNA

VP1, VP3

Transcription

VP5*–VP8*

DLP

Uncoating and membrane penetration

VP2

Assembly

Replication

VP4

NSP4

Rough ER

Intermediate-
membrane displacement
by VP7

VP7

Viroplasm
NSP2–NSP5

VP6

Figure 2 | The rotavirus replication cycle. The rotavirus virion first attaches to the target cell; many strains bind cell surface sialic acids through VP8* (produced by cleavage of VP4 into VP5* and VP8*) at the tips of the virion spikes. Non-clathrin-, non-caveolin-mediated endocytosis delivers the virion to the early endosome. There, reduced calcium concentrations are thought to trigger uncoating (loss of VP7) of the triple-layered particle (TLP) and membrane penetration by VP5*. Loss of the outer capsid and release of the double-layered particle (DLP) into the cytosol activates the internal polymerase complex (VP1 and VP3) to transcribe capped positive-sense RNA ((+)RNAs) from each of the 11 double-stranded RNA (dsRNA) genome segments. (+)RNAs serve either as mRNAs for synthesis of viral proteins by cellular ribosomes or as templates for synthesis of negative-sense RNA ((−)RNA) during genome replication. Non-structural protein 2 (NSP2) and NSP5 interact to form large inclusions (viroplasms) that sequester components required for genome replication and the assembly of subviral particles. Genome packaging is initiated when VP1 (and, presumably, VP3) bind the 3′ end of viral (+)RNAs. It is currently thought that interactions among the 11 (+)RNAs drive formation of the ‘assortment complex’. Condensation of the inner capsid protein, VP2, around the assortment complex triggers dsRNA synthesis by VP1. The intermediate capsid protein, VP6, then assembles onto the nascent core to form the DLP. Assembly of the outer capsid is not well understood; the current model proposes that interaction with the rotavirus transmembrane protein, NSP4, recruits DLPs and the outer capsid protein VP4 to the cytosolic face of the endoplasmic reticulum (ER). Through an undefined mechanism, the DLP–VP4–NSP4 complex buds into the ER. Subsequent removal of the ER membrane and NSP4 permits assembly of the ER-resident outer capsid protein, VP7, and formation of the TLP (see FIG. 6). Release from the infected cell exposes the virion to trypsin-like proteases of the gastrointestinal tract, resulting in the specific cleavage of VP4 into VP5* and VP8* to produce the fully infectious virion. IFN, interferon.

Galectin  
A family of sugar-binding proteins that have a similar, distinct fold.

Sialic acid moieties  
Carbohydrate functional groups added to proteins or lipids. These groups are used by several viruses (including rotaviruses and influenza viruses) as attachment factors to facilitate attachment to host cells.

Myristoylated  
Of a protein: with an amino terminus that has a covalently linked myristic acid fatty acid. This can impart a hydrophobic character to a protein or target it to a membrane.

formation of the inverted VP5* trimer accompanies infectious rotavirus entry³⁷. Furthermore, there is a specific correlation between the rearrangement of VP5* and liposome binding *in vitro*³⁸. Given that rotaviruses have no envelope of their own to fuse to the host cell, it is not fully understood how these structural changes in VP5* result in membrane penetration. The myristoylated reovirus penetration protein, µ1, forms ~7 nm pores in model membranes³⁹,⁴⁰. It may be the case that VP5* is capable of perforating the cell membrane via its hydrophobic loops in much the same way as reovirus µ1, albeit through a different physical mechanism.

VP7 seems to be the key regulator of the conformation and membrane penetration activity of VP5*. Following assembly of the VP7 layer onto the particle, this layer limits the access of trypsin to VP4, thereby allowing the

REVIEW S

Pre                     Extended                      Post
Intermediate

Rotaviruses
VP5*-VP8*

Influenza viruses
HA₂-HA₁

Figure 3 | Conformational rearrangements of the rotavirus spike during entry. A comparison of the refolding of the rotavirus spike (VP5*–VP8*) during membrane penetration with the refolding of influenza virus haemagglutinin (HA) during membrane fusion. The rotavirus spike (Protein Data Bank IDs 3IYU and 1SLQ) is formed by three subunits of VP5* and capped by two subunits of VP8*. The hydrophobic loops thought to interact with the membrane (green) and the sequence in VP5* that will refold into a coiled-coil (blue) are shown. The influenza virus HA (Protein Data Bank IDs 3HMG and 1HTM) is coloured similarly: the HA₂ trimer is held below the three receptor-binding subunits, HA₁, with the postfusion coiled-coil (blue) and fusion peptide (green) indicated. During penetration or fusion, the meta-stable ‘pre’ state is perturbed, resulting in dissociation of the receptor-binding subunits, extension of the penetration protein and insertion of the hydrophobic peptide into the host cell membrane (yellow). The stability of the ‘post’ state drives a fold-back reorganization of the penetration protein³⁶. In the case of influenza virus HA₂, this brings the host and viral (brown) membranes into apposition and drives fusion³⁶. How refolding of VP5* accomplishes membrane penetration is currently not known. Dissociation of the VP7 layer seems to trigger VP5* refolding³⁸,⁴⁴, but it is also not known whether VP5* performs membrane penetration when it is still particle associated or as a discrete complex.

protease to cleave only the region that bridges VP8* and VP5* (REFS 1,41). As described above, proteolytic processing of VP4 in the absence of VP7 and the viral particle is more extensive and leads directly to the formation of the post-penetration state of VP5* (REFS 8,35). The VP7 layer probably functions to arrest and stabilize VP5* in the upright spike conformation to allow for viral attachment¹. During rotavirus entry, it has been observed that the TLP is internalized via endocytosis and traffics to the early endosome³⁷, wherein the low calcium concentration is predicted to trigger VP7 disassembly⁴²,⁴³. The dissociation of VP7 then serves as a cue for VP5* rearrangement and allows the virion to penetrate the endosomal membrane³⁸,⁴⁴. Consistent with this model, bafilomycin A1 and extracellular Ca-EGTA, both of which increase endosomal calcium concentrations, effectively block rotavirus entry³⁷,⁴². Furthermore, inhibition of VP7 uncoating by a neutralizing antibody also blocks membrane penetration², suggesting that VP7 disassembly is an obligate step before membrane penetration by VP5*.

disassembles the virion and activates the membrane penetration activity of µl (REFS 45,46). Thus, both rotaviruses and reovirus have evolved endosome-sensing proteins (VP7 and σ3, respectively) that detect different features of the same intracellular compartment to trigger uncoating and engage the membrane penetration apparatus. The net result of attachment, uncoating and penetration of the endosomal membrane is the release of a large subviral particle into the host cell cytosol.

Uncoating induces transcription by DLPs

For rotaviruses, the double-layered particle (DLP) is delivered to the target cell following entry (FIG. 2). Viral polymerase complexes in the DLP are transcriptionally active and immediately commence synthesis of 11 species of capped, non-polyadenylated (+)RNAs using the negative-sense RNAs ((−)RNAs) of the dsRNA genome segments as templates⁴⁷ (FIG. 4). The catalytic subunit of the polymerase complex for RNA synthesis is VP1, a hollow, globular enzyme composed of a central, conserved ‘right-handed’ polymerase domain surrounded by amino- and carboxy-terminal domains unique to the Reoviridae viruses¹⁰,⁴⁸. Four tunnels lead into the catalytic centre of VP1; these serve as conduits for the entry and exit of nucleotides, template RNAs and RNA

Cathepsin
A diverse family of intracellular proteases, many members of which function in the low pH environment of the lysosome.

Negative-sense RNAs
The reverse complement of the positive-sense RNAs; for the *Reoviridae* family viruses, the negative-sense strand is used as a template during transcription to make more positive-sense RNA.

Figure 4 | Conformational changes in the subviral particle that trigger transcription.

a | Model for the transcriptionally inactive rotavirus virion. Prior to entry, the VP7 layer seems to suppress transcription by the viral polymerase complex, which consists of VP1 and VP3 (REFS 3,4). Current data suggest that a single polymerase complex resides near almost each of the five-fold-vertex channels in a rotavirus particle¹⁶. In this inactive state, the RNA-dependent RNA polymerase, VP1 (Protein Data Bank (PDB) ID 2R7U), is probably bound to the 3′ end of the negative-sense RNA ((−)RNA) strand of the double-stranded RNA (dsRNA) genome, poised for transcription of the positive-sense RNA ((+)RNA)¹⁰.

b | The VP7 layer (PDB ID 3IYU) dissociates during entry, resulting in the upwards and outwards movement of the VP6 and VP2 layers (PDB ID 3N09) of the particle³,⁵⁴. This movement results in expansion of the five-fold-vertex channel and permits extrusion of (+)RNA transcripts⁴.

c | Model for the passage of the nascent (+)RNA transcript through the polymerase complex and out of the particle. The four tunnels of VP1 permit the input (−)RNA and NTPs to support synthesis of the new (+)RNA, and allow the re-formation of the parental dsRNA pair¹⁰. The 5′ end of the (+)RNA must rapidly be recruited and capped by VP3 before exit through the five-fold-vertex channel. Biochemical data, and comparison with the bluetongue virus capping enzyme (VP4; used here as a surrogate for rotavirus VP3; PDB ID 2JH8)⁵⁰, suggest that VP3 is a multidomain protein that successively modifies the 5′ end through guanylyltransferase (GTase; step 1), (guanine-N7)-methyltransferase (N7MTase; step 2) and (nucleoside-2′-O)-methyltransferase (2′OMTase; step 3) activities to generate the mature rotavirus 5′ cap structure.

products (FIG. 5). One RNA exit tunnel directs a nascent (+)RNA towards VP3 (discussed below) and release from the viral particle, and a second exit tunnel directs the genomic (−)RNA back into the core, where it reassociates with the genomic (+)RNA (FIGS 4,5). Thus, transcription by VP1 is fully conservative. The presence of an RNA cap-binding site near the RNA template entry tunnel of VP1 suggests a mechanism whereby VP1 binds and orients the dsRNA genome to allow template recycling during transcription¹⁰. Immediately following synthesis of the (+)RNAs, but before their extrusion from the DLP via channels at the five-fold vertices, the (+)RNA molecules acquire a 5′ cap through the activities of VP3 (REF. 49). Although we do not yet have a structure for rotavirus VP3, studies of the bluetongue virus capping enzyme (VP4; 71 kDa) have shed light on the putative functional domains of this protein⁵⁰. BTV VP4 is an hourglass-shaped protein that contains four domains⁵⁰. Like rotavirus VP3, it is responsible for the entire chemistry of viral RNA capping: guanylylation, N-7-methylation, and 2′-O-methylation. However, the RNA triphosphatase activity that precedes capping has yet to be clarified by the BTV VP4 structure⁵⁰. For reovirus, RNA triphosphatase activity is encoded in a separate protein, μ2 (83 kDa), which is a minor component of the core particle and interacts with the polymerase λ3 (REF. 51); rotaviruses do not possess a minor capsid protein, suggesting that triphosphatase activity resides

in one of the polymerase complex components (VP1 or VP3). Overall, the BTV VP4 structure has suggested an ‘assembly line’ model for capping, in which the 5′ end of the RNA is transferred from one domain to the next to perform the successive reactions that generate the final 7-methyl-GpppG (REF. 50) (FIG. 4). Recent structural evidence suggests that rotavirus VP1 is located at a discrete position near the channels at the five-fold vertices in the DLP¹⁶. Therefore, VP3 probably performs its capping activities in a small space before transcript ejection (FIG. 4). By contrast, the newly made (+)RNAs of turreted Reoviridae viruses are capped as they navigate a pentameric gauntlet of capping enzymes to exit the transcribing particle¹⁵,⁵²,⁵³.

It is still not clear how changes that occur in the viral particle on loss of the outer capsid induce transcription. Structural evidence suggests that removal of VP7 causes a dilation of the vertex channels in the particle, thereby providing conduits for the influx of ions and nucleotides and the efflux of transcripts³,⁵⁴. Specifically, on recoating of a DLP with recombinant VP7, the VP6 trimers immediately surrounding the five-fold vertices move down and inwards³. This motion also drives VP2 inwards, thereby constricting the channel diameter³. These structural changes detected in the presence of VP7 are similar to those observed in the presence of VP6-specific antibodies that inhibit transcription by DLPs *in vitro*⁵⁵,⁵⁶. Thus, it is presumed that removal of the outer capsid layer during

REVIEW S

a
![Image](VP1-VP3)
(+)-RNA
RNA-RNA?
VP2
dsRNA
synthesis
VP1–VP3 binds the 11 (+)RNAs
VP2 self-assembles and binds polymerase complexes
Packaged virion core

b
![Image](RNA entry)
(−)RNA exit
NTP entry
(+)RNA exit

d
![Image](VP2-A, VP2-B, VP1)

c
![Image](Retracted PL)
![Image](Extended and Retracted PL)

Figure 5 | Assembly of the rotavirus core is coupled with genome replication.
a | Model for positive-sense RNA ((+)-RNA) packaging, core assembly and double-stranded RNA (dsRNA) genome replication. It is assumed that VP1 and VP3 associate in the viroplasm to form polymerase complexes. VP1 also binds a specific sequence at the 3′ end of each of the 11 rotavirus (+)-RNAs but remains catalytically inactive. Comparisons with mammalian orthoreovirus and influenza viruses have suggested a mechanism in which RNA–RNA interactions among the (+)-RNAs nucleate complexes containing all of the segments and associated polymerase complexes. VP2 then self-assembles and engages VP1; this triggers dsRNA synthesis by VP1.
b | Structure of VP1 with bound (+)-RNA, but held in an inactive state (Protein Data Bank (PDB) ID 2R7R). The functions of the four tunnels extending to the large catalytic centre of VP1 are identified. Prior to interaction with VP2, VP1 can bind to the 3′ end of (+)-RNAs (black) with high affinity, but remains autoinhibited<sup>10,58</sup>. Also shown is the priming loop (PL), a structural feature that is thought to be involved in regulating polymerization by VP1.
c | Model for conformational changes in the VP1 PL that initiate dsRNA synthesis. In the structure of VP1, the PL that is used to stabilize a priming nucleotide in the catalytic centre is retracted and out of place. It is thought that interaction of VP1 with VP2 promotes a conformational change in this loop that allows it to stabilize the priming nucleotide. For the sake of reference, the locations of nucleotides (turquoise sticks) and cations (green spheres) have been modelled into the catalytic centre of VP1. Here, we also use the structure of mammalian orthoreovirus λ3 with bound template and nucleotides (PDB ID 1N1H) in the catalytic centre as a surrogate for activated VP1 (REF. 48); note that the PL supports a priming nucleotide only in its extended conformation and not in its retracted state. Polymerase motifs A, C and F are indicated.
d | Model for the interaction of rotavirus VP1 with the inner surface of the core shell. Recent structural evidence and information obtained with mammalian orthoreovirus suggest that VP1 binds a specific position off-centre of a VP2 decamer<sup>15,16</sup>. The five-fold symmetry of the decamer suggests that there are potentially five VP1-binding sites per decamer, although it is likely that only a single polymerase is associated with each decamer. The RNA-capping enzyme, VP3 (not shown), is probably held in close proximity to VP1 but has not been observed to date. Interaction with VP2 may transmit information to VP1 (for example, through the PL) that activates the polymerase to synthesize negative-sense RNA (−)-RNA) using the (+)-RNA as a template, resulting in the formation of the dsRNA genome<sup>59</sup>.

entry of the host cell results in an outwards movement of VP6 and VP2 at the five-fold vertices and an opening of the channels<sup>3</sup> (FIG. 4). It is not clear, though, whether merely increasing the diameter of the channels is sufficient to induce (+)-RNA synthesis or whether a specific signal is relayed to the internally tethered polymerase complexes. An alternative hypothesis for transcriptional activation of the DLP proposes that VP1 is sensitive to conformational changes in VP2 which occur during uncoating. In support of this idea, the activity of VP1 during genome replication (that is, dsRNA synthesis) is tightly controlled by interactions with VP2 (REFS 6,57–60) (discussed below). It is therefore interesting to speculate that loss of VP7 transmits structural changes through VP6, which in turn influences the interaction of VP1 with VP2 and enables the polymerase to act.

### Genome replication and core assembly

Nascent rotavirus (+)-RNAs made in DLPs serve dual roles during the rotavirus replication cycle, acting as mRNAs for protein synthesis and as templates for genome replication (FIG. 2). The function of any given (+)-RNA during infection is thought to be determined by its intracellular localization<sup>61</sup>. Products of transcription from incoming DLPs accumulate in the cytosol and are available for translation into viral proteins by host ribosomes. Two viral non-structural proteins (NSP2 and NSP5) are thought to colocalize around transcribing DLPs, nucleating the formation of inclusion bodies termed viroplasms. The structure of NSP2 (35 kDa) has been solved: the protein forms doughnut-shaped octomers that bind both RNA and NSP5 (REFS 62,63). NSP5 (22 kDa), a largely unstructured phosphoprotein, has been shown to self-associate and interacts with RNA and NSP2 (REFS 64,65). The numerous self- and partner-specific interactions of NSP2 and NSP5 suggest that viroplasms form as large, semiregular networks designed to sequester viral RNAs and capsid proteins for assembly into nascent virions. Consistent with this model, these RNA-dense bodies are the sites of early virion assembly and further (+)-RNA transcription by nascent DLPs. Viroplasm-associated rotavirus (+)-RNAs are selectively packaged into assembling VP2 cores and replicated by VP1 into the dsRNA genome<sup>61</sup>.

Although the order of the protein and RNA interactions that occur during packaging and genome replication is not completely understood, cumulative biochemical and structural data suggest a working model (FIG. 5a). First, individual copies of VP1 (perhaps in complex with VP3) bind to the 3′ ends of the viral (+)-RNAs, creating 11 different enzyme–RNA complexes<sup>10,58</sup>. The structure of recombinant VP1 in complex with an oligonucleotide representing the 3′ end of a (+)-RNA revealed that VP1 specifically recognizes a conserved 4-nucleotide sequence (UGUG)<sup>10</sup>. Mutation of VP1 to disrupt multiple base-specific interactions diminishes dsRNA synthesis by the polymerase *in vitro*, probably by abolishing the initial RNA recognition and binding events<sup>66</sup>. The 3′ end of the (+)-RNA is held near the central active site through interactions with the phosphate backbone in a sequence-independent manner, but is positioned out-of-register
with the catalytic site<sup>10</sup> (FIG. 5b,c). Thus, the polymerase–(+)RNA complexes that first form in the viroplasm are thought to be catalytically inactive; the VP1 component would have to undergo one or more conformational changes to initiate synthesis of dsRNA.

The autoinhibited RNA-binding mechanism of VP1 may produce an accumulating population of catalytically inactive polymerase–(+)RNA complexes that are available for assortment into assembling cores (FIG. 5a). The current model for rotavirus (+)RNA assortment borrows heavily from the more established mechanisms of segmented (−)RNA viruses. Influenza A virus, in particular, has multiple genome segments, each of which contains a central ORF flanked by short, conserved 5′ and 3′ untranslated regions (UTRs). It has been proposed that the genomic (−)RNAs of influenza viruses are arranged in a specific pattern according to base-pairing interactions between the UTRs of different segments<sup>67</sup>. A recent report, in which the UTRs were swapped between influenza virus genome segments to influence how the segments are packaged, supports this model<sup>68</sup>. In addition, reverse genetics systems for reovirus have been used successfully to package and replicate two reporter genes by replacing the native ORF of a given segment with that of a transgene, suggesting that genome packaging in Reoviridae viruses is dependent entirely on the UTR sequences<sup>69,70</sup>. Similar to the influenza virus genome, rotavirus (+)RNAs seem to adopt a looped ‘pan handle’ conformation with base-paired 5′ and 3′ ends<sup>71,72</sup>. Although it has not been formally demonstrated that the UTRs of rotavirus (+)RNAs drive gene-specific interactions during packaging, they are obvious targets for future experimentation.

Following assortment, an assembling VP2 core shell engages the polymerase component of polymerase–(+)RNA complexes, thereby activating the enzymes to initiate (−)RNA strand synthesis to produce the dsRNA genome (FIG. 5a,d). *In vitro* biochemical studies have shown that the inwardly protruding VP2 N termini have a role in polymerase activation<sup>59,60</sup>, but that residues of the core shell proximal to the five-fold vertex are important for specific interaction of a given VP2 with its cognate VP1 (REF. 6). The relative sizes of VP1 and VP2, and comparison with reovirus proteins<sup>15</sup>, suggest that VP1 contacts multiple VP2 molecules (both the A and B forms) at a given five-fold vertex (FIG. 5d). The further observation that maximal *in vitro* dsRNA synthesis occurs with a 10:1 molar ratio of VP2:VP1 leads to the hypothesis that a decameric assembly of VP2 activates each VP1 monomer<sup>59</sup>. During the process of dsRNA synthesis, however, VP2 must ultimately self-assemble to form a complete, closed T = 1 shell. In this model, the rotavirus core is composed of 12 VP2 decamers (that is, 12 vertices) with 11 internally tethered polymerase–(+)RNA complexes and a single empty vertex<sup>11</sup>.

The mechanism by which VP2 activates VP1 to initiate dsRNA synthesis is currently not well understood. Some insight into this process has been provided by comparing the structure of an autoinhibited VP1 with that of a catalytically active form of the reovirus polymerase, λ3 (REFS 10,48). An extended loop is seen between the fingers and palm subdomains of the central polymerase domain in both enzymes<sup>10,48</sup> (FIG. 5c). In λ3, the loop seems to operate as a platform for stabilizing a priming nucleotide<sup>48</sup>. In the VP1 crystal structure, however, this so-called ‘priming loop’ bends away from the active site by about 90°, leaving it in a retracted state that is incapable of binding nucleotides<sup>10</sup>. On the basis of this observation, it is thought that interaction with VP2 causes structural changes in VP1 that include lifting the priming loop and repositioning the 3′ terminus of the (+)RNA to be in-register with the catalytic site<sup>10</sup>. Regardless of the mechanism, regulation of VP1 activity by VP2 is crucial for rotavirus replication, as it probably prevents premature synthesis of dsRNA and allows the virus to precisely coordinate the stages of RNA packaging, genome replication and core shell assembly.

In addition to regulating the timing of dsRNA synthesis by VP1, rotaviruses must control the assembly of their capsid proteins. When expressed in the absence of other rotavirus proteins, VP2 and VP6 will each spontaneously self-assemble into capsid-like structures<sup>17,73,74</sup>. It seems likely that rotaviruses employ a specific mechanism to prevent uncontrolled self-assembly of their structural proteins during infection. The viroplasm-forming proteins, NSP2 and NSP5, have each been shown to interact with a subset of structural proteins, (+)RNA and each other. Thus, it is likely that these two non-structural proteins help govern particle assembly. NSP5 has been shown to interact with VP2 in co-immunoprecipitation assays and may modulate the assembly of VP6 onto the core shell<sup>75</sup>. The addition of NSP5 to DLP-like structures actively displaced VP6 from the core shell *in vitro*, suggesting that this phosphoprotein competes for binding sites on VP2 (REF. 75). Thus, NSP5 might act to negatively regulate VP6 assembly, and the competing interactions of other viral proteins with NSP5 may further regulate this activity. To date, similar regulation of VP2 assembly has not been observed. However, NSP2 has been shown to interfere with VP2-dependent, VP1-mediated dsRNA synthesis *in vitro*, indicating that it might control interactions among VP1, VP2 and (+)RNA<sup>76,77</sup>. Further characterization of the network of protein and RNA interactions in the viroplasm is vital to our understanding of how rotaviruses carefully moderate particle assembly to ensure that (+)RNAs are properly encapsidated before their replication.

### Regulation of outer-capsid assembly

One of the least understood aspects of rotavirus replication is the process by which the DLP acquires the outer capsid. To fully assemble, the DLP must exit the viroplasm, associate with the spike protein (VP4) and then breach the endoplasmic reticulum (ER) membrane to gain access to the glycoprotein VP7. Rotaviruses seem to be the only Reoviridae family members to encode a structural glycoprotein. In fact, this property is extremely rare for non-enveloped viruses in general, as most of them assemble in the cytoplasm or

Reverse genetics systems  
Methods of specifically modifying a viral genome using recombinant technology (versus a forward genetics screen).

NATURE REVIEWS | MICROBIOLOGY  
VOLUME 10 | MARCH 2012 | 173  
© 2012 Macmillan Publishers Limited. All rights reserved
nucleus without involving budding steps in morphogenesis. Much in the same way that uncontrolled assembly of VP2 or VP6 would impair genome replication, premature addition of VP7 to the DLP would stifle replication by silencing (+)RNA transcription. By compartmentalizing VP7 to the ER, rotaviruses ensure that DLPs are only converted to TLPs after they leave the confines of the viroplasm.

The viral protein NSP4 (20 kDa) is an integral membrane protein that accumulates in the ER near the cytosolic viroplasms and is a key regulator of outer-capsid assembly. The N terminus of the protein extends into the ER lumen, and this terminus is glycosylated and forms intramolecular disulphide bonds<sup>78</sup>. A single-pass transmembrane sequence leads into a cytosolic coiled-coil motif that causes NSP4 to form tetramers<sup>78,79</sup>. The largely unstructured C terminus of NSP4 has been shown to bind both the DLP (through VP6) and VP4 (REFS 80–83), suggesting that it helps to chaperone the otherwise weak DLP–VP4 and VP4 trimerization interactions<sup>84</sup>. Although a specific mechanism for the extraction of DLPs from the viroplasm is not known, we postulate that interaction with NSP4 recruits DLPs into the outer-capsid assembly pathway. It is suspected that unassembled VP7 interacts with the ER-luminal or transmembrane region of NSP4 (REFS 85,86). Through an unknown mechanism, VP7 is also retained in the ER via interaction with its cleaved signal peptide<sup>87</sup>.

Although the data are incomplete, the current model for outer-capsid assembly is illustrated in FIG. 6. NSP4 recruits both DLPs (from nearby viroplasms) and VP4 to the cytosolic face of the ER membrane. Successive interaction of the DLP with surrounding NSP4 tetramers results in ER membrane deformation and budding of the DLP–VP4–NSP4 complex into the ER. Thereafter, the membrane is removed and VP7 assembles onto the particle, thereby locking VP4 into place. Biochemical and structural data support this order of assembly. Immunoelectron microscopy and fluorescence microscopy of infected cells indicate that VP4 accumulates in the narrow region between the viroplasms and the ER, possibly because it is recruited by NSP4 (REFS 88,89). In vitro assembly of the outer capsid onto DLPs requires the addition of VP4 before VP7 to generate infectious particles; the reverse order excludes VP4 from assembly<sup>84</sup>. Finally, the TLP structure indicates that the bulbous VP4 foot is probably too large to be inserted through the narrow opening formed by VP7 at the peripentonal channels<sup>1</sup>.

The mechanism by which the transient envelope is removed from the DLP is not known. Treatment of infected cells with tunicamycin (which blocks N-glycosylation) or thapsigargin (which disrupts calcium storage and chaperone functions in the ER) arrests viral morphogenesis, causing enveloped DLPs to accumulate in the ER lumen<sup>90,91</sup>. These data implicate one of the ER-resident rotavirus glycoproteins (NSP4 or VP7) in membrane penetration. An interesting but somewhat confounding finding is that both NSP4 and VP7 disrupt membranes in vitro<sup>92,93</sup>. Small interfering RNA-mediated knockdown of NSP4 expression results in a pleiotropic phenotype

Figure 6 | Rotaviruses penetrate a second membrane during virion assembly. Model for the acquisition of the rotavirus outer capsid via budding and penetration through the endoplasmic reticulum (ER) membrane. After assembly in the viroplasm, double-layered particles (DLPs) are recruited to the cytosolic face of the ER membrane through interaction with the carboxy-terminal tails of the NSP4 tetramer (Protein Data Bank ID 1G1J)<sup>79–81</sup>. The carboxy-terminal tails of NSP4 probably bind and recruit the VP4 spike protein, which is synthesized in the cytosol<sup>83</sup>. Simultaneously, in the ER lumen, outer capsid protein VP7 associates with its cleaved signal peptide (embedded in the ER membrane) and with NSP4 (REFS 85–87). The successive interaction of NSP4 with the DLP may induce the ER membrane to wrap around the DLP, causing the membrane to bud into the ER lumen. Enveloped intermediate particles have been isolated, and these particles have been found to contain DLPs and VP4 in the lumen, and transmembrane NSP4, and to have VP7 associated with their exterior face<sup>90</sup>. Through an undefined mechanism — presumably involving VP7 (REF. 95) — the ER membrane, along with NSP4, is removed from the particle, allowing VP7 to assemble onto the particle. The structure of the immature rotavirus virion has not been fully resolved; here, we use the structure of the trypsin-primed, fully infectious particle<sup>1</sup>. Prior to trypsin cleavage, VP4 does not form well-ordered spikes such as those depicted here<sup>22</sup>.
that cannot be directly linked to membrane penetration during assembly ${ }^{94,95}$. By contrast, similar knockdown of VP7 expression results in the same ‘enveloped DLP’ phenotype that is observed with tunicamycin and thapsigargin treatment and strongly suggests a direct role for VP7 in membrane disruption during assembly ${ }^{95}$.

### Virion release from infected cells

*In vitro* studies indicate that assembled rotavirus virions are capable of egressing from infected cells through more than one mechanism. The virus is released from non-polarized kidney epithelial cells by direct lysis ${ }^{96}$. However, as the morphology and function of these cells are different from those of intestinal epithelial cells, lysis may not be the only mechanism of viral release from all cell types in the infected host. Consistent with this, another report indicates that rotaviruses are released from a polarized intestinal epithelial cell line by trafficking and secretion from the apical cell surface ${ }^{97}$. Release from polarized cells seems to use a novel secretion pathway that bypasses the Golgi apparatus and lysosomes ${ }^{97}$. It is not known whether the apical targeting of virions is due to rotaviruses ‘hijacking’ the infected cell (that is, misappropriating cellular components to aid in egress, much like the actin tail-based ejection of vaccinia virus ${ }^{98}$) or whether the virus is merely accessing a pre-existing secretory pathway. With growing evidence that rotaviruses replicate in cells of the infected host other than just intestinal epithelial cells, the possibility remains that a combination of release mechanisms is involved in viral spread *in vivo*.

### Concluding remarks

Throughout the process of replication, rotaviruses seem to have evolved mechanisms to ascertain various states of particle assembly and then function accordingly to drive replication forward. During entry, these viruses use VP7 to sense that they have been endocytosed into target cells. Subsequent uncoating of VP7 from the DLP cues membrane penetration by VP5 ${ }^{+}$ and transcription by the encapsidated polymerase complexes. Later, during assembly, an intricate network of interactions among viral RNAs and proteins in the viroplasm coordinates the order of events that culminate with the assembly-associated synthesis of the dsRNA genome. Rotaviruses prevent aberrant genome replication by first forming an autoinhibited complex between the (+)RNA transcripts and the viral polymerase, VP1; genome replication is triggered through subsequent interaction of this complex with the assembling VP2 core shell. However, we do not yet have the complete picture of how the non-structural proteins NSP2 and NSP5 orchestrate — and probably modulate — assembly of viral particles. Newly formed DLPs are recognized by the viral ‘ER receptor’, NSP4, and directed towards outer-capsid assembly. Arguably, one of the least understood aspects of rotavirus biology is the process by which the assembling particle penetrates the ER membrane to acquire its outer capsid. The advancement of reverse genetics technology ${ }^{69,99}$ and, possibly, novel complementation systems ${ }^{100}$ for rotaviruses will possibly lead to a better understanding of this transient and multifactorial event.

In an interesting parallel with rotaviruses, the activity of the reverse transcriptase from *Hepadnaviridae* family viruses is intimately tied to the assembly of nascent virions. During packaging, the hepatitis B virus reverse transcriptase must first recognize a stem loop structure near the 5′ end of the pre-genomic RNA in order to be sequestered into a core particle ${ }^{101,102}$. Only then is the pre-genomic RNA converted into partial dsDNA. Reverse transcription is also required for capsid maturation, leading to subsequent membrane envelopment and egress ${ }^{103}$. Although more broad comparisons have been made across retrovirus, (+)RNA virus and dsRNA virus replication ${ }^{104}$, this specific example illustrates the functional convergence of viruses in using the particle assembly state as a precise metric, and trigger, to ensure correct coordination of replication. Further insights into the minute details of viral-particle structures and assembly intermediates may provide a greater appreciation of the commonalities that span divergent groups of viruses.

---

1. Settembre, E. C., Chen, J. Z., Dormitzer, P. R., Grigorieff, N. & Harrison, S. C. Atomic model of an infectious rotavirus particle. *EMBO J.* **30**, 408–416 (2011).  
   This work determines a high-resolution structure for the entire rotavirus virion, including the unexpected conformation of the primed spike.
   
2. Ludert, J. E., Ruiz, M. C., Hidalgo, C. & Liprandi, F. Antibodies to rotavirus outer capsid glycoprotein VP7 neutralize infectivity by inhibiting virion decapsulation. *J. Virol.* **76**, 6643–6651 (2002).
   
3. Chen, J. Z. *et al.* Molecular interactions in rotavirus assembly and uncoating seen by high-resolution cryo-EM. *Proc. Natl Acad. Sci. USA* **106**, 10644–10648 (2009).  
   This report describes the unusual ‘grip arm’ mode of VP7 assembly onto the rotavirus subviral particle.
   
4. Lawton, J. A., Estes, M. K. & Prasad, B. V. Three-dimensional visualization of mRNA release from actively transcribing rotavirus particles. *Nature Struct. Biol.* **4**, 118–121 (1997).
   
5. Chen, D. & Patton, J. T. *De novo* synthesis of minus strand RNA by the rotavirus RNA polymerase in a cell-free system involves a novel mechanism of initiation. *RNA* **6**, 1455–1467 (2000).
   
6. McDonald, S. M. & Patton, J. T. Rotavirus VP2 core shell regions critical for viral polymerase activation. *J. Virol.* **85**, 3095–3105 (2011).  
   This investigation identifies the sites of interaction between the viral RNA polymerase and the VP2 decamer that are necessary for triggering RNA synthesis.
   
7. Aoki, S. T. *et al.* Structure of rotavirus outer-layer protein VP7 bound with a neutralizing Fab. *Science* **324**, 1444–1447 (2009).
   
8. Dormitzer, P. R., Nason, E. B., Prasad, B. V. & Harrison, S. C. Structural rearrangements in the membrane penetration protein of a non-enveloped virus. *Nature* **430**, 1053–1058 (2004).  
   This article proposes the current model for membrane penetration by rotaviruses.
   
9. Dormitzer, P. R., Sun, Z. Y., Wagner, G. & Harrison, S. C. The rhesus rotavirus VP4 sialic acid binding domain has a galectin fold with a novel carbohydrate binding site. *EMBO J.* **21**, 885–897 (2002).
   
10. Lu, X. *et al.* Mechanism for coordinated RNA packaging and genome replication by rotavirus polymerase VP1. *Structure* **16**, 1678–1688 (2008).  
    This report provides a mechanistic explanation for VP1 autoinhibition and suggests several conformational changes that may be required for polymerase activity.
    
11. McClain, B., Settembre, E., Temple, B. R., Bellamy, A. R. & Harrison, S. C. X-ray crystal structure of the rotavirus inner capsid particle at 3.8 Å resolution. *J. Mol. Biol.* **397**, 587–599 (2010).
    
12. Zhang, X. *et al.* Near-atomic resolution using electron cryomicroscopy and single-particle reconstruction. *Proc. Natl Acad. Sci. USA* **105**, 1867–1872 (2008).
    
13. Reinisch, K. M., Nibert, M. L. & Harrison, S. C. Structure of the reovirus core at 3.6 Å resolution. *Nature* **404**, 960–967 (2000).
    
14. Grimes, J. M. *et al.* The atomic structure of the bluetongue virus core. *Nature* **395**, 470–478 (1998).
    
15. Zhang, X., Walker, S. B., Chipman, P. R., Nibert, M. L. & Baker, T. S. Reovirus polymerase λ3 localized by cryo-electron microscopy of virions at a resolution of 7.6 Å. *Nature Struct. Biol.* **10**, 1011–1018 (2003).
    
16. Estrozi, L. F. & Navaza, J. Ab initio high-resolution single-particle 3D reconstructions: the symmetry adapted functions way. *J. Struct. Biol.* **172**, 253–260 (2010).
    
17. Mathieu, M. *et al.* Atomic structure of the major capsid protein of rotavirus: implications for the architecture of the virion. *EMBO J.* **20**, 1485–1497 (2001).

NATURE REVIEWS | MICROBIOLOGY

VOLUME 10 | MARCH 2012 | 175

© 2012 Macmillan Publishers Limited. All rights reserved
REVIEW S

18. Matthijnssens, J. *et al.* Recommendations for the classification of group A rotaviruses using all 11 genomic RNA segments. *Arch. Virol.* **153**, 1621–1629 (2008).

19. Zhang, X. *et al.* Features of reovirus outer capsid protein μ1 revealed by electron cryomicroscopy and image reconstruction of the virion at 7.0 Å resolution. *Structure* **13**, 1545–1557 (2005).

20. Liemann, S., Chandran, K., Baker, T. S., Nibert, M. L. & Harrison, S. C. Structure of the reovirus membrane-penetration protein, μ1, in a complex with is protector protein, σ3. *Cell* **108**, 283–295 (2002).

21. Estes, M. K., Graham, D. Y. & Mason, B. B. Proteolytic enhancement of rotavirus infectivity: molecular mechanisms. *J. Virol.* **39**, 879–888 (1981).

22. Crawford, S. E. *et al.* Trypsin cleavage stabilizes the rotavirus VP4 spike. *J. Virol.* **75**, 6052–6061 (2001).

23. Yeager, M., Berriman, J. A., Baker, T. S. & Bellamy, A. R. Three-dimensional structure of the rotavirus haemagglutinin VP4 by cryo-electron microscopy and difference map analysis. *EMBO J.* **13**, 1011–1018 (1994).

24. Li, Z., Baker, M. L., Jiang, W., Estes, M. K. & Prasad, B. V. Rotavirus architecture at subnanometer resolution. *J. Virol.* **83**, 1754–1766 (2009).

25. Shaw, A. L. *et al.* Three-dimensional visualization of the rotavirus hemagglutinin structure. *Cell* **74**, 693–701 (1993).

26. Fiore, L., Greenberg, H. B. & Mackow, E. R. The VP8 fragment of VP4 is the rhesus rotavirus hemagglutinin. *Virology* **181**, 553–563 (1991).

27. Ruggeri, F. M. & Greenberg, H. B. Antibodies to the trypsin cleavage peptide VP8 neutralize rotavirus by inhibiting binding of virions to target cells in culture. *J. Virol.* **65**, 2211–2219 (1991).

28. Monnier, N. *et al.* High-resolution molecular and antigen structure of the VP8* core of a sialic acid-independent human rotavirus strain. *J. Virol.* **80**, 1513–1523 (2006).

29. Dormitzer, P. R. *et al.* Specificity and affinity of sialic acid binding by the rhesus rotavirus VP8* core. *J. Virol.* **76**, 10512–10517 (2002).

30. Blanchard, H., Yu, X., Coulson, B. S. & von Itzstein, M. Insight into host cell carbohydrate-recognition by human and porcine rotavirus from crystal structures of the virion spike associated carbohydrate-binding domain (VP8*). *J. Mol. Biol.* **367**, 1215–1226 (2007).

31. Mendez, E., Arias, C. F. & Lopez, S. Binding to sialic acids is not an essential step for the entry of animal rotaviruses to epithelial cells in culture. *J. Virol.* **67**, 5253–5259 (1993).

32. Ciarlet, M. & Estes, M. K. Human and most animal rotavirus strains do not require the presence of sialic acid on the cell surface for efficient infectivity. *J. Gen. Virol.* **80**, 943–948 (1999).

33. Kim, I. S., Trask, S. D., Babyonyshev, M., Dormitzer, P. R. & Harrison, S. C. Effect of mutations in VP5 hydrophobic loops on rotavirus cell entry. *J. Virol.* **84**, 6200–6207 (2010).

34. Tihova, M., Dryden, K. A., Bellamy, A. R., Greenberg, H. B. & Yeager, M. Localization of membrane permeabilization and receptor binding sites on the VP4 hemagglutinin of rotavirus: implications for cell entry. *J. Mol. Biol.* **314**, 985–992 (2001).

35. Dormitzer, P. R., Greenberg, H. B. & Harrison, S. C. Proteolysis of monomeric recombinant rotavirus VP4 yields an oligomeric VP5* core. *J. Virol.* **75**, 7339–7350 (2001).

36. Harrison, S. C. Viral membrane fusion. *Nature Struct. Mol. Biol.* **15**, 690–698 (2008).

37. Wolf, M., Vo, P. T. & Greenberg, H. B. Rhesus rotavirus entry into a polarized epithelium is endocytosis dependent and involves sequential VP4 conformational changes. *J. Virol.* **85**, 2492–2503 (2011). This evaluation of the rotavirus entry process identifies several cellular requirements and provides support for predicted conformational changes in the virion.

38. Trask, S. D., Kim, I. S., Harrison, S. C. & Dormitzer, P. R. A rotavirus spike protein conformational intermediate binds lipid bilayers. *J. Virol.* **84**, 1764–1770 (2010).

39. Agosto, M. A., Ivanovic, T. & Nibert, M. L. Mammalian reovirus, a nonfusogenic nonenveloped virus, forms size-selective pores in a model membrane. *Proc. Natl Acad. Sci. USA* **103**, 16496–16501 (2006).

40. Zhang, L. *et al.* Requirements for the formation of membrane pores by the reovirus myristoylated micro1N peptide. *J. Virol.* **83**, 7004–7014 (2009).

41. Arias, C. F., Romero, P., Alvarez, V. & Lopez, S. Trypsin activation pathway of rotavirus infectivity. *J. Virol.* **70**, 5832–5839 (1996).

42. Chemello, M. E., Aristimuno, O. C., Michelangeli, F. & Ruiz, M. C. Requirement for vacuolar H*-ATPase activity and Ca²⁺ gradient during entry of rotavirus into MA104 cells. *J. Virol.* **76**, 13083–13087 (2002).

43. Ludert, J. E., Michelangeli, F., Gil, F., Liprandi, F. & Esparza, J. Penetration and uncoating of rotaviruses in cultured cells. *Intervirology* **27**, 95–101 (1987).

44. Yoder, J. D. *et al.* VP5* rearranges when rotavirus uncoats. *J. Virol.* **83**, 11372–11377 (2009).

45. Ebert, D. H., Deussing, J., Peters, C. & Dermody, T. S. Cathepsin L and cathepsin B mediate reovirus disassembly in murine fibroblast cells. *J. Biol. Chem.* **277**, 24609–24617 (2002).

46. Golden, J. W., Bahe, J. A., Lucas, W. T., Nibert, M. L. & Schiff, L. A. Cathepsin S supports acid-independent infection by some reoviruses. *J. Biol. Chem.* **279**, 8547–8557 (2004).

47. Lawton, J. A., Estes, M. K. & Prasad, B. V. Mechanism of genome transcription in segmented dsRNA viruses. *Adv. Virus Res.* **55**, 185–229 (2000).

48. Tao, Y., Farsetta, D. L., Nibert, M. L. & Harrison, S. C. RNA synthesis in a cage—structural studies of reovirus polymerase λ3. *Cell* **111**, 733–745 (2002). This is the first study to report an atomic structure of an RNA polymerase for a member of the Reoviridae, revealing the four-tunnel architecture of the enzyme.

49. Chen, D., Luongo, C. L., Nibert, M. L. & Patton, J. T. Rotavirus open cores catalyze 5′-capping and methylation of exogenous RNA: evidence that VP3 is a methyltransferase. *Virology* **265**, 120–130 (1999).

50. Sutton, G., Grimes, J. M., Stuart, D. I. & Roy, P. Bluetongue virus VP4 is an RNA-capping assembly line. *Nature Struct. Mol. Biol.* **14**, 449–451 (2007). This paper reports the atomic structure of the bluetongue virus RNA-capping enzyme, the only capping enzyme of a non-turreted Reoviridae virus that has been described to date.

51. Kim, J., Parker, J. S., Murray, K. E. & Nibert, M. L. Nucleoside and RNA triphosphatase activities of orthoreovirus transcriptase cofactor μ2. *J. Biol. Chem.* **279**, 4394–4403 (2004).

52. Cheng, L. *et al.* Atomic model of a cypovirus built from cryo-EM structure provides insight into the mechanism of mRNA capping. *Proc. Natl Acad. Sci. USA* **108**, 1373–1378 (2011).

53. Mendez, I. I., Weiner, S. G., She, Y. M., Yeager, M. & Coombs, K. M. Conformational changes accompany activation of reovirus RNA-dependent RNA transcription. *J. Struct. Biol.* **162**, 277–289 (2008).

54. Libersou, S. *et al.* Geometric mismatches within the concentric layers of rotavirus particles: a potential regulatory switch of viral particle transcription activity. *J. Virol.* **82**, 2844–2852 (2008).

55. Thouvenin, E. *et al.* Antibody inhibition of the transcriptase activity of the rotavirus DLP: a structural view. *J. Mol. Biol.* **307**, 161–172 (2001).

56. Feng, N. *et al.* Inhibition of rotavirus replication by a non-neutralizing, rotavirus VP6-specific IgA mAb. *J. Clin. Invest.* **109**, 1203–1213 (2002).

57. Mansell, E. A. & Patton, J. T. Rotavirus RNA replication: VP2, but not VP6, is necessary for viral replicase activity. *J. Virol.* **64**, 4988–4996 (1990).

58. Patton, J. T. Rotavirus VP1 alone specifically binds to the 3′ end of viral mRNA, but the interaction is not sufficient to initiate minus-strand synthesis. *J. Virol.* **70**, 7940–7947 (1996).

59. Patton, J. T., Jones, M. T., Kalbach, A. N., He, Y. W. & Xiaobo, J. Rotavirus RNA polymerase requires the core shell protein to synthesize the double-stranded RNA genome. *J. Virol.* **71**, 9618–9626 (1997).

60. McDonald, S. M. & Patton, J. T. Molecular characterization of a subgroup specificity associated with the rotavirus inner capsid protein VP2. *J. Virol.* **82**, 2752–2764 (2008).

61. Silvestri, L. S., Taraporewala, Z. F. & Patton, J. T. Rotavirus replication: plus-sense templates for double-stranded RNA synthesis are made in viroplasms. *J. Virol.* **78**, 7763–7774 (2004).

62. Jayaram, H., Taraporewala, Z., Patton, J. T. & Prasad, B. V. Rotavirus protein involved in genome replication and packaging exhibits a HIT-like fold. *Nature* **417**, 311–315 (2002).

63. Jiang, X. *et al.* Cryoelectron microscopy structures of rotavirus NSP2-NSP5 and NSP2-RNA complexes: implications for genome replication. *J. Virol.* **80**, 10829–10835 (2006).

64. Martin, D., Ouldali, M., Menetrey, J. & Poncet, D. Structural organisation of the rotavirus nonstructural protein NSP5. *J. Mol. Biol.* **413**, 209–221 (2011).

65. Chnaiderman, J., Barro, M. & Spencer, E. NSP5 phosphorylation regulates the fate of viral mRNA in rotavirus infected cells. *Arch. Virol.* **147**, 1899–1911 (2002).

66. Ogden, K. M., Ramanathan, H. N. & Patton, J. T. Residues of the rotavirus RNA-dependent RNA polymerase template entry tunnel that mediate RNA recognition and genome replication. *J. Virol.* **85**, 1958–1969 (2011).

67. Noda, T. *et al.* Architecture of ribonucleoprotein complexes in influenza A virus particles. *Nature* **439**, 490–492 (2006).

68. Gao, Q. & Palese, P. Rewiring the RNAs of influenza virus to prevent reassortment. *Proc. Natl Acad. Sci. USA* **106**, 15891–15896 (2009).

69. Kobayashi, T. *et al.* A plasmid-based reverse genetics system for animal double-stranded RNA viruses. *Cell Host Microbe* **1**, 147–157 (2007).

70. Roner, M. R. & Joklik, W. K. Reovirus reverse genetics: incorporation of the CAT gene into the reovirus genome. *Proc. Natl Acad. Sci. USA* **98**, 8036–8041 (2001).

71. Li, W. *et al.* Genomic analysis of codon, sequence and structural conservation with selective biochemical-structure mapping reveals highly conserved and dynamic structures in rotavirus RNAs with potential cis-acting functions. *Nucleic Acids Res.* **38**, 7718–7735 (2010).

72. Patton, J. T., Wentz, M., Xiaobo, J. & Ramig, R. F. cis-acting signals that promote genome replication in rotavirus mRNA. *J. Virol.* **70**, 3961–3971 (1996).

73. Lepault, J. *et al.* Structural polymorphism of the major capsid protein of rotavirus. *EMBO J.* **20**, 1498–1507 (2001).

74. Zeng, C. Q. *et al.* Characterization of rotavirus VP2 particles. *Virology* **201**, 55–65 (1994).

75. Berois, M., Sapin, C., Erk, I., Poncet, D. & Cohen, J. Rotavirus nonstructural protein NSP5 interacts with major core protein VP2. *J. Virol.* **77**, 1757–1763 (2003).

76. Kattoura, M. D., Chen, X. & Patton, J. T. The rotavirus RNA-binding protein NS35 (NSP2) forms 10S multimers and interacts with the viral RNA polymerase. *Virology* **202**, 803–813 (1994).

77. Vende, P., Tortorici, M. A., Taraporewala, Z. F. & Patton, J. T. Rotavirus NSP2 interferes with the core lattice protein VP2 in initiation of minus-strand synthesis. *Virology* **313**, 261–273 (2003).

78. Bergmann, C. C., Maass, D., Poruchynsky, M. S., Atkinson, P. H. & Bellamy, A. R. Topology of the non-structural rotavirus receptor glycoprotein NS28 in the rough endoplasmic reticulum. *EMBO J.* **8**, 1695–1703 (1989).

79. Bowman, G. D. *et al.* Crystal structure of the oligomerization domain of NSP4 from rotavirus reveals a core metal-binding site. *J. Mol. Biol.* **304**, 861–871 (2000).

80. O’Brien, J. A., Taylor, J. A. & Bellamy, A. R. Probing the structure of rotavirus NSP4: a short sequence at the extreme C terminus mediates binding to the inner capsid particle. *J. Virol.* **74**, 5388–5394 (2000).

81. Taylor, J. A., O’Brien, J. A. & Yeager, M. The cytoplasmic tail of NSP4, the endoplasmic reticulum-localized non-structural glycoprotein of rotavirus, contains distinct virus binding and coiled coil domains. *EMBO J.* **15**, 4469–4476 (1996).

82. Taylor, J. A., O’Brien, J. A., Lord, V. J., Meyer, J. C. & Bellamy, A. R. The RER-localized rotavirus intracellular receptor: a truncated purified soluble form is multivalent and binds virus particles. *Virology* **194**, 807–814 (1993).

83. Au, K. S., Mattion, N. M. & Estes, M. K. A subviral particle binding domain on the rotavirus nonstructural glycoprotein NS28. *Virology* **194**, 665–673 (1993).

84. Trask, S. D. & Dormitzer, P. R. Assembly of highly infectious rotavirus particles recoated with recombinant outer capsid proteins. *J. Virol.* **80**, 11293–11304 (2006).

85. Poruchynsky, M. S. & Atkinson, P. H. Rotavirus protein rearrangements in purified membrane-enveloped intermediate particles. *J. Virol.* **65**, 4720–4727 (1991).

86. Maass, D. R. & Atkinson, P. H. Rotavirus proteins VP7, NS28, and VP4 form oligomeric structures. *J. Virol.* **64**, 2632–2641 (1990).

87. Stirzaker, S. C. & Both, G. W. The signal peptide of the rotavirus glycoprotein VP7 is essential for its retention in the ER as an integral membrane protein. *Cell* **56**, 741–747 (1989).

88. Petrie, B. L., Greenberg, H. B., Graham, D. Y. & Estes, M. K. Ultrastructural localization of rotavirus antigens using colloidal gold. *Virus Res.* **1**, 133–152 (1984).

89. Gonzalez, R. A., Espinosa, R., Romero, P., Lopez, S. & Arias, C. F. Relative localization of viroplasmic and endoplasmic reticulum-resident rotavirus proteins in infected cells. *Arch. Virol.* **145**, 1963–1973 (2000).

90. Poruchynsky, M. S., Maass, D. R. & Atkinson, P. H. Calcium depletion blocks the maturation of rotavirus by altering the oligomerization of virus-encoded proteins in the ER. *J. Cell Biol.* **114**, 651–656 (1991).

91. Michelangeli, F., Liprandi, F., Chemello, M. E., Ciarlet, M. & Ruiz, M. C. Selective depletion of stored calcium by thapsigargin blocks rotavirus maturation but not the cytopathic effect. *J. Virol.* **69**, 3838–3847 (1995).

92. Browne, E. P., Bellamy, A. R. & Taylor, J. A. Membrane-destabilizing activity of rotavirus NSP4 is mediated by a membrane-proximal amphipathic domain. *J. Gen. Virol.* **81**, 1955–1959 (2000).

93. Charpilienne, A. *et al.* Solubilized and cleaved VP7, the outer glycoprotein of rotavirus, induces permeabilization of cell membrane vesicles. *J. Gen. Virol.* **78**, 1367–1371 (1997).

94. Silvestri, L. S., Tortorici, M. A., Vasquez-Del Carpio, R. & Patton, J. T. Rotavirus glycoprotein NSP4 is a modulator of viral transcription in the infected cell. *J. Virol.* **79**, 15165–15174 (2005).

95. Lopez, T. *et al.* Silencing the morphogenesis of rotavirus. *J. Virol.* **79**, 184–192 (2005).

96. Musalem, C. & Espejo, R. T. Release of progeny virus from cells infected with simian rotavirus SA11. *J. Gen. Virol.* **66**, 2715–2724 (1985).

97. Jourdan, N. *et al.* Rotavirus is released from the apical surface of cultured human intestinal cells through nonconventional vesicular transport that bypasses the Golgi apparatus. *J. Virol.* **71**, 8268–8278 (1997).

98. Cudmore, S., Cossart, P., Griffiths, G. & Way, M. Actin-based motility of vaccinia virus. *Nature* **378**, 636–638 (1995).

99. Boyce, M., Celma, C. C. & Roy, P. Development of reverse genetics systems for bluetongue virus: recovery of infectious virus from synthetic RNA transcripts. *J. Virol.* **82**, 8339–8348 (2008).

100. Taraporewala, Z. F. *et al.* Structure-function analysis of rotavirus NSP2 octamer by using a novel complementation system. *J. Virol.* **80**, 7984–7994 (2006).

101. Knaus, T. & Nassal, M. The encapsidation signal on the hepatitis B virus RNA pregenome forms a stem-loop structure that is critical for its function. *Nucleic Acids Res.* **21**, 3967–3975 (1993).

102. Porterfield, J. Z. *et al.* Full-length hepatitis B virus core protein packages viral and heterologous RNA with similarly high levels of cooperativity. *J. Virol.* **84**, 7174–7184 (2010).

103. Perlman, D. H., Berg, E. A., O’Connor P, B., Costello, C. E. & Hu, J. Reverse transcription-associated dephosphorylation of hepadnavirus nucleocapsids. *Proc. Natl Acad. Sci. USA* **102**, 9020–9025 (2005).

104. Ahlquist, P. Parallels among positive-strand RNA viruses, reverse-transcribing viruses and double-stranded RNA viruses. *Nature Rev. Microbiol.* **4**, 371–382 (2006).

105. Wolf, M., Garcea, R. L., Grigorieff, N. & Harrison, S. C. Subunit interactions in bovine papillomavirus. *Proc. Natl Acad. Sci. USA* **107**, 6298–6303 (2010).

106. Dryden, K. A. *et al.* Early steps in reovirus infection are associated with dramatic changes in supramolecular structure and protein conformation: analysis of virions and subviral particles by cryoelectron microscopy and

107. Chappell, J. D., Prota, A. E., Dermody, T. S. & Stehle, T. Crystal structure of reovirus attachment protein σ1 reveals evolutionary relationship to adenovirus fiber. *EMBO J.* **21**, 1–11 (2002).

**Acknowledgements**

The authors thank N. Leach for careful reading of the manuscript, and S. Harrison, E. Settembre and K. Ogden for helpful discussions. S.D.T. and J.T.P. were supported by the Intramural Research Program of the US National Institutes of Allergy and Infectious Diseases (NIAID), National Institutes of Health (grant Z01 AI000788). S.M.M. was supported by the NIAID Intramural Research Program and the Virginia Tech Carilion Research Institute (Roanoke, USA).

**Competing interests statement**

The authors declare no competing financial interests.

**DATABASES**

Protein Data Bank: [http://www.rcsb.org/pdb/home/home.do](http://www.rcsb.org/pdb/home/home.do) | [1HTM](http://www.rcsb.org/pdb/home/home.do) | [1EJ6](http://www.rcsb.org/pdb/home/home.do) | [1G1](http://www.rcsb.org/pdb/home/home.do) | [1N1H](http://www.rcsb.org/pdb/home/home.do) | [1SLQ](http://www.rcsb.org/pdb/home/home.do) | [2CSE](http://www.rcsb.org/pdb/home/home.do) | [2JH8](http://www.rcsb.org/pdb/home/home.do) | [2R7R](http://www.rcsb.org/pdb/home/home.do) | [2R7U](http://www.rcsb.org/pdb/home/home.do) | [3GZT](http://www.rcsb.org/pdb/home/home.do) | [3HMG](http://www.rcsb.org/pdb/home/home.do) | [3IYU](http://www.rcsb.org/pdb/home/home.do) | [3KZ4](http://www.rcsb.org/pdb/home/home.do) | [3N09](http://www.rcsb.org/pdb/home/home.do)

**FURTHER INFORMATION**

John T. Patton’s homepage: [http://www.niaid.nih.gov/labsandresources/labs/aboutlabs/](http://www.niaid.nih.gov/labsandresources/labs/aboutlabs/)lid/rotavirus/pages/patton.aspx

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
