
RESEARCH LETTER

# The *mll6786* gene encodes a repressor protein controlling the degradation pathway for vitamin B₆ in *Mesorhizobium loti*

Takayuki Nagase¹, Andrew N. Mugo¹, Huy Nhat Chu¹, Yu Yoshikane¹, Kouhei Ohnishi² & Toshiharu Yagi¹

¹Faculty of Agriculture and Agricultural Science Program, Graduate School of Integral Arts and Science, Kochi University, Nankoku, Kochi, Japan;  
²Research Institute of Molecular Genetics, Kochi University, Nankoku, Kochi, Japan

---

**Correspondence:** Toshiharu Yagi, Faculty of Agriculture and Agricultural Science Program, Graduate School of Integral Arts and Science, Kochi University, Nankoku, Kochi 783-8502, Japan. Tel.: +81 88 8645191; fax: +81 88 8645191; e-mail: yagito@kochi-u.ac.jp

Received 16 June 2011; revised 17 January 2012; accepted 19 January 2012.

DOI: 10.1111/j.1574-6968.2012.02510.x

Editor: Wilfrid Mitchell

Keywords: pyridoxine; GntR family; pyridoxamine; *Mesorhizobium loti*; *mll6786*.

---

## Abstract

Pyridoxine is converted to succinic semialdehyde, acetate, ammonia and CO₂ through the actions of eight enzymes. The genes encoding the enzymes occur as a cluster on the chromosomal DNA of *Mesorhizobium loti*, a symbiotic nitrogen-fixing bacterium. Here, it was found that disruption of the *mll6786* gene, which is located between the genes encoding the first and eighth enzymes of the pathway, caused constitutive expression of the eight enzymes. The protein encoded by the *mll6786* gene is a member of the GntR family and is designated as PyrR. PyrR comprises 223 amino acid residues and is a dimeric protein with a subunit molecular mass of 25 kDa. The purified PyrR with a C-terminal His₆-tag could bind to an intergenic 67-bp DNA region, which contains a palindrome sequence and a deduced promoter sequence, between the *mll6786* and *mlr6787* genes, encoding PyrR and AAMS amidohydrolase, respectively.

---

## Introduction

Three kinds of microorganisms harbor a degradation pathway for pyridoxine, a free form of vitamin B₆. *Pseudomonas* MA-1 (Nelson & Snell, 1986) and *Mesorhizobium loti* (Yuan *et al.*, 2004) have pathway I, in which pyridoxine is degraded through eight enzyme-catalyzed steps (Fig. 1, top). *Arthrobacter* Cr-7 (Nelson & Snell, 1986) has pathway II, in which pyridoxine is degraded in five steps. 4-Hydroxymethyl and 5-hydroxymethyl groups attached to the pyridine ring of pyridoxine are at first oxidized in pathways I and II, respectively.

Recently, the genes encoding the eight enzymes of the pathway were identified in the *M. loti* chromosome (Fig. 1, bottom). The numbering of the genes is fixed in the RhizoBase (genome database for Rhizobia, http://bacteria.kazusa.or.jp/rhizobase/). The first enzyme, pyridoxine 4-oxidase, is encoded by the *mll6785* gene (Yuan *et al.*, 2004); the second, pyridoxal 4-dehydrogenase, by *mlr6807* (Yokochi *et al.*, 2006); the third, 4-pyridoxolactonase, by *mlr6805* (Funami *et al.*, 2005); the fourth, 4-pyridoxic acid dehydrogenase, by *mlr6792* (Ge *et al.*, 2008); the fifth, 5-formyl-3-hydroxy-2-methylpyridine-4-carboxylic acid (FHMPA) dehydrogenase, by *mlr6793* (Yokochi *et al.*, 2009); the sixth, 3-hydroxy-2-methylpyridine-4,5-dicarboxylic acid (HMPC) decarboxylase, by *mlr6791* (Mukherjee *et al.*, 2007); the seventh, 3-hydroxy-2-methylpyridine-5-carboxylic acid (HMPC) oxygenase, by *mlr6788* (Yuan *et al.*, 2006; McCulloch *et al.*, 2009); and the eighth, AAMS amidohydrolase, by *mlr6787* (Mukherjee *et al.*, 2008; Yuan *et al.*, 2008). Pyridoxamine is converted into pyridoxal by pyridoxamine-pyruvate aminotransferase encoded by *mlr6806* (Yoshikane *et al.*, 2006). Thus, the genes form a cluster, from *mll6785* to *mlr6807*, including several genes of unknown function.

The expression of genes involved in bacterial catabolic pathways is often regulated by one or several transcriptional regulators (Tropel & Van der Meer, 2004). The GntR family proteins are well known transcription factors and comprise more than 8500 members in the Pfam database (Hoskisson & Rigali, 2009). They are distributed throughout the bacterial world. The GntR regulators are subdivided into the AraR, DevA, FadR, HutC, MocR, PlmA, and YtrA subfamilies based on the sequence

(a)

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

(b)

![Gene Cluster Diagram](https://i.imgur.com/gene_cluster_diagram.png)

Fig. 1. Degradation pathway I for vitamin B₆. (a) The pathway consists of nine enzyme-catalyzed steps. (b) The cluster of genes involved in the pathway. The gene *mll6786* encoding PyrR is located between the genes, *mll6785* and *mlr6787*, encoding the enzymes of step 1a and step 8 of the pathway, respectively. Three gene symbols, *mll6794*, *mll6795* and *mll6797*, are missing because they do not exist in the Rhizobase.

similarity of their C-terminal effector-binding oligomerization domains. The FadR subfamily is the most representative GntR subfamily and can be divided into FadR and VanR subgroups based on the number of α-helices (10 and 9, respectively) in their secondary structures (Rigali et al., 2002). In the cluster of genes involved in the degradation of pyridoxine (Fig. 1b) there is one gene (*mll6786*) that encodes a probable transcriptional regulator protein. The primary structure and deduced secondary structure suggested that *mll6786* encodes a regulator protein that belongs to the VanR subgroup.

As far as we know, no study has been done on the regulation mechanism for the degradation pathway for pyridoxine. Here, we identified the protein PyrR encoded by *mll6786* as a transcriptional repressor protein. The recombinant repressor protein was over-expressed and characterized as the first step of elucidation of the regulatory mechanism for the pyridoxine-degradation pathway in *M. loti* cells.

## Materials and methods

### Bacterial strains, cultivation, plasmids, and chemicals

*Escherichia coli* strains BL21(DE3) and JM109 were purchased from Novagen (San Diego, CA) and Takara (Tokyo, Japan), respectively. *Escherichia coli* S17-1 was obtained from the National Bioresource Project (Mishima, Japan). *Mesorhizobium loti* MAFF303099 was obtained from the MAFF GenBank (Tsukuba, Japan). *Escherichia coli* cells were cultivated at 37 °C in Luria–Bertani (LB) medium containing appropriate concentrations of antibiotics. *Mesorhizobium loti* cells were cultivated at 30 °C in tryptose–yeast (TY) medium and pyridoxine (PN) synthetic medium, as described previously (Yuan et al., 2004).

Plasmids pTA2 (Toyobo, Osaka, Japan) and pET-21a (Novagen) were used for cloning and expression. pK18-mobsacB and pKRP12 (National Bioresource Project) were used for disruption of the *mll6786* gene.

The primers shown in Table 1 were purchased from Invitrogen Japan (Tokyo, Japan). 4-Pyridoxolactone (Tamura et al., 2008), FHMPDC (Yokochi et al., 2009), HMPDC (Mukherjee et al., 2007), HMPC (Yuan et al., 2006), and AAMS (Yuan et al., 2008) were prepared as described previously. A biotin-labeled marker DNA (biomarker low, biotin conjugate) was purchased from BioVentures, Inc. (Murfreesboro, TN), and marker DNA fragments (λ-HindIII) from New England Biolabs Japan, Inc. (Tokyo, Japan).

### Preparation of an *mll6786*-disruptant strain of *M. loti*

A disruption plasmid was constructed as follows. A 630-bp fragment harboring 400-bp of the 5′ end of *mll6786* plus

Repressor for vitamin B₆ degradation pathway

Table 1. Primers used in this study

| Primer Name | Sequence |
|-------------|----------|
| 6786-F      | 5′-CATATGCCCCCAGATTTCGAATTTGCGA-3′ (underline, NdeI site) |
| 6786-R      | 5′-AAGCTTCCCTCAAATCCCGTTGTCCATGGAT-3′ (underline, HindIII site) |
| 6786-mut-1F | 5′-TCTAGAGCGTCGCGAGATGAAGTGTT-3′ (underline, XbaI site) |
| 6786-mut-1R | 5′-CTGCAGCAGGCTGTCATTGCTGGAGG-3′ (underline, PstI site) |
| 6786-mut-2F | 5′-CTGCAGGTCATGACCGCGGACTTCTATT-3′ (underline, PstI site) |
| 6786-mut-2R | 5′-AAGCTTAGTCCCAATCGTAGCTGCGGCCCT-3′ (underline, HindIII site) |
| 6786-His-F  | 5′-CACCACCACCACCACCACCTGAGAT-3′ (double underline, His₆-coding site) |
| 6786-His-R  | 5′-AATCCCAGTTGTCCATGGATATCTTCGCTG-3′ |
| GSA-Biotin-R| 5′-A*TGTCTGCCGCCATGTCCAT-3′ (*biotin-labeled) |
| GSA-R       | 5′-ATGTCTGCCGCCATGTCCAT-3′ |
| GSA-68-F    | 5′-TGATCCTTTTGCAGGAGACG-3′ |
| GSA-135-F   | 5′-GCAGGATAGCCAGATTGTCA-3′ |
| GSA-321-F   | 5′-TTTCGATGAGGCGGCTACCC-3′ |

its 230-bp upstream region was amplified by PCR with primers 6786-mut-1F and 6786-mut-1R. A 640-bp fragment harboring 280-bp of the 3′ end of *mll*6786 plus its 360-bp downstream region was amplified by PCR with primers 6786-mut-2F and 6786-mut-2R. The fragments were cloned into the pTA2 vector, separately, to construct pTA2-630 and pTA2-640. Then, the 630-bp fragment cut out from pTA2-630 with XbaI and PstI was cloned into plasmid pK18mobsacB to construct pK18-630, to which the 640-bp fragment cut out from pTA2-640 with PstI and HindIII was ligated to construct pK18-1270. The 2000-bp tetracycline resistance gene obtained from pKRP12 by digestion with PstI was inserted into pK18-1270, and the resulting plasmid pK18-1270::Tc was used as the disruption plasmid. The plasmid was transferred into *M. loti* MAFF303099 via conjugation with *E. coli* S17-1/pK18-1270::Tc (Simon *et al.*, 1983) and transconjugants were selected as described previously (Yokochi *et al.*, 2006).

### Cloning of *mll*6786, preparation of pET6786-His₆, and expression and purification of the Mll6786-His₆ protein

*mll*6786 was amplified by PCR from the chromosomal DNA of *M. loti* with primers 6786-F and 6786-R. The amplified 680-bp fragment was cloned into pTA2 to construct pTA2-680. pTA2-680 was digested with NdeI and HindIII, and then the digested DNA fragment was inserted into the NdeI/HindIII sites of pET21a⁺ to construct expression plasmid pET6786. pET6786-His₆ was prepared from pET6786 with primers 6786-His-F and 6786-His-R according to the instructions for a KOD-Plus-Mutagenesis Kit (Toyobo).

*Escherichia coli* BL21(DE3) was transformed with pET6786-His₆. The transformed cells were grown aerobically in 5 mL of LB medium containing ampicillin (100 μg mL⁻¹) at 37 °C until *A*₆₀₀ₙₘ reached 0.8. The culture was then added to 200 mL of the same medium, and the inoculated cells were grown at 37 °C for 12 h. The cells were harvested by centrifugation at 8400 g for 10 min at 4 °C, and then washed with 0.9% NaCl and stored at –20 °C.

The transformed cells (1 g, w/w) were suspended in 10 mL of buffer A (50 mM potassium phosphate buffer, pH 8.0, containing 0.3 M NaCl and 0.1% 2-mercaptoethanol), and then sonicated on ice five times for 1 min at 2-min intervals, using a model W-220 sonicator (Heat Systems Ultrasonics, Farmingdale, NY). The supernatant was obtained by centrifugation at 10 000 g for 30 min at 4 °C. The precipitated cells were resuspended with buffer A and sonicated again. The supernatants were combined and used as the crude extract (18 mL). The crude extract was applied to a column containing 2 mL of Ni-NTA-affinity resin equilibrated with buffer A. The column was consecutively washed with 3 mL (each) of buffer A containing 20, 50, 100, and 250 mM imidazole. The Mll6786-His₆ protein was eluted with the buffer containing 100 mM imidazole.

#### Enzyme and protein assays

The activities of the enzymes were determined at 30 °C as described previously in the references given above. One unit of an enzyme is defined as the amount of the enzyme that catalyzed the formation of 1 nmol of the product min⁻¹. Protein concentrations were measured by the protein-dye method with bovine serum albumin as a standard (Bradford, 1976).

#### Gel retardation experiments

In the cluster of genes, a promoter region deduced with Neural Network Promoter Prediction (http://www.fruitfly.org/seq_tools/promoter.html) was found in the DNA sequence between *mll*6786 and *mlr*6787. Three biotin-labeled DNA probes incorporating parts of this region and some of the 3′ end of the *mll*6786 gene (Fig. 3a) were prepared by PCR using the chromosome of *M. loti* as a template, and primer GSA-Biotin-R with primers GSA-321-F, GSA-135-F, and GSA-68-F, respectively, for the gel shift assaying. A biotin-free 135-bp DNA probe was prepared with GSA-135-F and GSA-R as primers. The PCR conditions were essentially the same as those given previously (Yokochi *et al.*, 2006).

Typical 10-μL (total volume) reaction mixtures contained the binding buffer (32.5 mM Tris-HCl, pH 7.5, containing 25 mM NaCl, 50 mM KCl, 0.25 mM EDTA, 0.25 mM dithiothreitol, 0.2% Tween 20, and 10% glycerol), 7.4 nM labeled DNA, 20 ng of poly(dA-dT), and the purified PyrR at the concentrations indicated. After the mixture had been incubated at room temperature for 30 min, a sample was loaded onto a 3.75% polyacrylamide gel in 0.5× TBE (45 mM Tris-HCl, pH 8.3, 45 mM sodium borate, 1 mM EDTA). Samples were run at a constant 100 V for 0.5 h, and then the gel was subjected to blotting on a Hybond-N nylon membrane. The DNA fragments were linked to the membrane in a Bio-Rad Laboratories GS Gene Linker chamber. After the membrane was blocked for 20 min in the blocking buffer (1% casein, 0.1 M maleic acid, 0.1 M NaCl, pH 7.5), the membrane was incubated with 0.1% streptavidin-horseradish peroxidase conjugate (HRP; Sigma) in the blocking buffer for 20 min with gentle shaking. The membrane was washed four times with the washing buffer (0.3% Tween 20, 0.1 M maleic acid, 0.1 M NaCl, pH 7.5) for 5 min, followed by equilibration with the maleic acid buffer (0.1 M maleic acid, 0.1 M NaCl) for 5 min with gentle shaking. The membrane was put on a clean sheet of plastic wrap and the light emitted by the DNA fragments produced on incubation in Chemi-Lumi One (Nacalai tesque, Kyoto, Japan) was recorded with LAS-4000 EPUVmini (Fuji Film, Tokyo, Japan).

### Other methods

The molecular mass of the recombinant PyrR was determined by HPLC with a size-exclusion chromatograph (Shodex Protein KW-803). A calibration curve was obtained based on the elution pattern of standard proteins as described previously (Yokochi *et al.*, 2009). The subunit molecular mass was determined by SDS-PAGE as described previously (Yokochi *et al.*, 2009).

### Results and discussion

#### Identification of a potential repressor gene

The primary sequence of the *mll6786* gene product was homologous to several repressor proteins. The DMS12804 protein in *Bordetella petrii* showed the highest identity, 39%; the IP32953 protein in *Yersinia pseudotuberculosis*, 37%; and the Ymp protein in *Pseudomonas mendocina*, 37%. On the basis of this, *mll6786* might encode a repressor protein and the gene product was designated as PyrR. The secondary structure of the PyrR protein was predicted with the JPREP 3 server (http://www.compbio.dundee.ac.uk/www-jpred/). The PyrR protein had an HTH motif: the amino acid residues from V14 to S28 formed the first α-helix; those from E39 to L46, the second α-helix; and those from P51 to A62, the third α-helix. The α-helices were followed by two β-sheets (I66-V69 and G73-P77). The arrangement of the secondary structures in the PyrR protein was quite similar to that in a DNA-binding protein (YP_298823.1, PDB entry 3IHU) from *Ralstonia eutropha* JMP 134.

#### Construction of the *mll6786* gene-disruptant strain of *M. loti*

A strain of *M. loti* in which the *mll6786* gene was inactivated by insertion of a tetracycline resistance gene, was constructed and isolated as described in Materials and methods. PCR of the chromosome of the disruptant strain did not give a DNA band corresponding to the size of *mll6786*. Instead, it produced a DNA band corresponding to the size of the *mll6786::Tc* gene (Fig. 2a). Thus, an *mll6786*-disruptant strain was successfully prepared. The *mll6786*-disruptant strain grew as well as the wild-type strain in TY medium, but other phenotypic characteristics were not examined.

#### The activities of enzymes involved in degradation of vitamin B₆ in the *mll6786*-disruptant strain

If PyrR is a transcriptional repressor like the VanR subgroup proteins, the regulated enzyme activities in the

(a)
![Figure 2a](https://via.placeholder.com/300x400)

(b)
![Figure 2b](https://via.placeholder.com/300x400)

**Fig. 2.** Lengths of amplified DNA fragments and SDS-PAGE patterns of a crude extract and the purified PyrR-His₆. (a) Agarose gel electrophoretic patterns. Lane M: marker DNA fragments (λ-HindIII). Lane 1: the *mll6786* gene amplified by PCR with the wild-type *Mesorhizobium loti* chromosome as the template. Lane 2: the *mll6786::Tc* gene amplified by PCR with pK18-1270::Tc as the template. Lane 3: the *mll6786::Tc* gene amplified by PCR with the chromosome of the *mll6786* gene-disruptant strain of *M. loti* as the template. The primers used were 6786-F and 6786-R. (b) SDS-PAGE patterns. Lane 1: the crude extract (10 μg of proteins) of *Escherichia coli* BL21(DE3) transformed with pET6786-His₆. Lane 2: the purified preparation (10 μg) of PyrR.

Repressor for vitamin B₆ degradation pathway

mll6786-disruptant cells would be expected to increase following disruption of the *pyrR* gene. The enzyme activities in crude extracts of the wild-type and ml16786-disruptant *M. loti* cells grown in TY medium and PN synthetic medium are shown in Table 2. All of the enzyme activities, with the exception of pyridoxine 4-oxidase (step 1a), of the disruptant strain grown in TY medium were significantly increased compared with the wild-type strain; pyridoxal 4-dehydrogenase showed the highest specific activity of 300 nmol min⁻¹ mg⁻¹ protein (about 31-fold higher than that in the wild-type cells). The results showed that the PyrR-disruptant constitutively expressed the eight enzymes of the pyridoxine degradation pathway and that PyrR was a repressor. The wild-type cells grown in PN synthetic medium showed significantly higher enzyme activities than those grown in TY medium because the degradation pathway is induced in this medium (Guirard & Snell, 1971). When the disruptant cells were grown in PN medium, the activities of the enzymes catalyzing steps 1b, 3, and 6 of the degradation pathway were found to be significantly higher than those in cells grown in TY medium. In contrast, the activities of the enzymes catalyzing steps 4 and 5 were significantly lower than those in cells grown in TY medium. The activities of the other enzymes in the disruptant cells were almost the same regardless of the medium. As described below, because the genes *mlr6792* and *mlr6793* encoding the enzymes of step 4 and step 5, respectively, two genes may constitute an operon, the same pattern of changes in the enzyme activities may be rational. However, the results do suggest that some factor(s) in addition to the PyrR protein contribute to control of synthesis of the enzymes.

to the molecular mass of PyrR (25 000 Da) on an SDS-PAGE gel (Fig. 2b). Thus, PyrR appeared to have been cloned and expressed in the *E. coli* cells. PyrR-His₆ was purified by Ni-NTA-affinity chromatography (Fig. 2b). The molecular mass of the intact PyrR-His₆ was determined to be 50 000 Da by size-exclusion chromatography (data not shown), showing that it is a dimeric protein like other VanR family proteins.

## Interaction of PyrR with the DNA region of the cluster

Three DNA fragments were prepared and labeled with biotin (Fig. 3a) and the interaction of the DNA fragments with PyrR was then examined. It was found that the 321-bp and 135-bp fragments bound to PyrR, and their movements on the polyacrylamide gel were affected (Fig. 3b and c). In contrast, a 68-bp fragment did not bind to PyrR and its movement was not affected (Fig. 3d). The results showed that the PyrR protein bound to an intergenic 67-bp DNA region (Fig. 3a). The 67-bp DNA contains a palindrome sequence (GATTGTCAGACAATC). It has been reported that *E. coli* FadR binds to a palindrome sequence (TGGTCCGACCA or TGGTACGACCA; Xu et al., 2001). Thus, PyrR may bind to the palindrome sequence in the 67-bp DNA and inhibit the expression of the *mlr6787* operon. The palindrome sequence overlapped the predicted –10 sequence of a putative promoter for the *mlr6787* gene.

Twelve transcriptional units were detected in the gene cluster for vitamin B₆ degradation pathway I (Fig. 1b) with MicrobesOnline Operon Predictions (http://www.microbesonline.org/operons/). Five operons, *mlr6787-mlr6788*, *mlr6792-mlr6793*, *mlr6796-mlr6798-mlr6799*, *mlr6801-mlr6802-mlr6803-mlr6804*, and *mlr6806-mlr6807*, are polycistronic. Other operons are monocistronic. Two operons, beginning with *mlr6796* and *mlr6801*, encode

## Cloning, expression and purification of PyrR

A crude extract of *E. coli* cells transformed with pET6786-His₆ gave a dense protein band corresponding

### Table 2. Activities of enzymes involved in degradation of vitamin B₆ in crude extracts of the *Mesorhizobium loti* wild-type and *ml16786* disruptant strains grown in TY and PN media

| Enzyme                          | Reaction step | Gene name   | Wild type       | ml16786 disruptant     |
|---------------------------------|---------------|-------------|------------------|------------------------|
|                                 |               |             | TY medium        | PN medium              |
| Pyridoxine 4-oxidase            | 1a            | *ml16785*   | 1.5 ± 0.4        | 4.7 ± 1.0              |
| Pyridoxamine-pyruvate aminotransferase | 1b | *mlr6806*   | 0.6 ± 0.1        | 17 ± 2.6               |
| Pyridoxal 4-dehydrogenase       | 2             | *mlr6807*   | 9.6 ± 0.7        | 300 ± 22               |
| 4-Pyridoxolactonase             | 3             | *mlr6805*   | 2.0 ± 0.0        | 77 ± 14                |
| 4-Pyridoxic acid dehydrogenase  | 4             | *mlr6792*   | ND               | 240 ± 28               |
| FHMPDC dehydrogenase            | 5             | *mlr6793*   | ND               | 57 ± 5.2               |
| HMPDC decarboxylase             | 6             | *mlr6791*   | ND               | 20 ± 4.0               |
| HMPC dioxygenase                | 7             | *mlr6788*   | ND               | 61 ± 9.4               |
| AAMS amidohydrolase             | 8             | *mlr6787*   | 2.1 ± 1.1        | 12 ± 4.0               |

The activities are in mmol min⁻¹ mg⁻¹. Averages ± SD of three experiments are shown.  
ND, not detected.

Fig. 3. Gel shift assaying of DNA fragments. (a) Locations of the DNA fragments used for gel the shift assay. The 67-bp fragment contained a palindrome region shown in italics. The predicted –35 and –10 sequences of the mlr6787 promoter are underlined. (b) The biotin-labeled 321-bp DNA probe was used. Lane M: biotin-labeled marker DNA fragments. Lane 1: 7.4 nM of the probe. Lane 2: 7.4 nM of the probe incubated with 300 nM PyrR. (c) The biotin-labeled 135-bp DNA probe was used. Lane M: biotin-labeled marker DNA fragments. Lane 1: 7.4 nM of the probe. Lane 2: 7.4 nM of the probe incubated with 300 nM PyrR. Lane 3: 7.4 nM of the probe incubated with 300 nM of PyrR and 147 nM of unlabeled 135-bp fragment. (d) The biotin-labeled 68-bp fragment was used. M: biotin-labeled marker DNA fragment. Lane 1: 7.4 nM of the probe. Lane 2: 7.4 nM of the probe incubated with 300 nM PyrR.

proteins that are related to ABC transporters. The functions of three monocistronic genes, mlr6789, mlr6790 and mll6800, are unknown. Besides the mlr6787 operon, five transcriptional units are related to the vitamin B₆ degradation pathway I. Additional potential operator sites associated with these genes were searched for. Although several palindrome sequences were found upstream of genes, none of them was similar to the previously identified palindrome sequence (GATTGTCAGACAATC) and we could not find any predicted PyrR binding sites. As a next step, gel retardation assays against the potential regulatory regions should be performed.

The effect of the PyrR concentration on the gel-shift of the 135-bp fragment was examined (Fig. 4). The gel-shift

Fig. 4. Concentration dependence of the gel shift. The biotin-labeled 135-bp DNA fragment (7.4 nM) was incubated without PyrR (lanes 1 and 11), and with PyrR at concentrations of 20 nM (lane 2), 50 nM (lane 3), 100 nM (lane 4), 150 nM (lane 5), 200 nM (lane 6), 300 nM (lane 7), 400 nM (lane 8), 500 nM (lane 9), and 1000 nM (lane 10).

was concentration-dependent. With a high concentration of PyrR (Fig. 4, lanes 8–10), the fragment moved upward as an additional band, suggesting that tetrameric assembly of the PyrR protein had occurred. There are examples of such assembly following DNA binding by Gnt superfamily proteins (Hoskisson & Rigali, 2009). Although the dissociation constant of PyrR was estimated to be 167.9 ± 57.5 μM, based on the density of the main band in lanes 4–7, the value was high and may be an artifact of the

Fig. 5. Effect of pyridoxine-related compounds on PyrR binding to DNA. The biotin-labeled 135-bp DNA fragment (7.4 nM) was incubated without PyrR (lane 1) and with PyrR at a concentration of 300 nM (lanes 2–5). Pyridoxine (lane 3), pyridoxamine (lane 4), and pyridoxal (lane 5) were added at concentration of 2 mM. Biotin-labeled free and PyrR-bound probes are shown as ‘unbound’ and ‘bound’, respectively. ‘Ratio’ was calculated from band densities of bound DNA divided by unbound DNA.

Repressor for vitamin B₆ degradation pathway

blotting technique. Thus, accurate determination of the dissociation constant by another method is required.

Pyridoxine induced activities of degrading enzymes (Table 2), indicating that it could work as an effector molecule. Therefore, pyridoxine and related compounds, pyridoxamine and pyridoxal, were added to the gel retardation assays. When pyridoxal and pyridoxamine were added, the relative amount of the free form of the nucleotide probe increased (Fig. 5), demonstrating that these compounds had an inhibitory effect on binding of PyrR to the DNA. On the other hand, pyridoxine did not appear to prevent DNA binding. Further study will reveal the mode of interaction between PyrR and its effector molecules.

## References

Bradford MM (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal Chem* 72: 248–254.

Funami J, Yoshikane Y, Kobayashi H, Yokochi N, Yuan B, Iwasaki K, Ohnishi K & Yagi T (2005) 4-Pyridoxolactonase from a symbiotic nitrogen-fixing bacterium, *Mesorhizobium loti*: cloning, expression, and characterization. *Biochim Biophys Acta* 1753: 234–239.

Ge F, Yokochi N, Yoshikane Y, Ohnishi K & Yagi T (2008) Gene identification and characterization of the pyridoxine degradative enzyme 4-pyridoxic acid dehydrogenase from the nitrogen-fixing symbiotic bacterium *Mesorhizobium loti* MAFF303099. *J Biochem* 143: 603–609.

Guirard BM & Snell EE (1971) Growth characteristics of a pseudomonad which utilizes pyridoxine or pyridoxamine as a carbon source. *J Bacteriol* 108: 1318–1321.

Hoskisson PA & Rigali S (2009) Variation in form and function: the helix-turn-helix regulators of the Gnt superfamily. *Adv Appl Microbiol* 69: 1–22.

McCulloch KM, Mukherjee T, Begley TP & Ealick SE (2009) Structure of the PLP degradative enzyme 2-methyl-3-hydroxypyridine-5-carboxylic acid oxygenase from *Mesorhizobium loti* MAFF303099 and its mechanistic implications. *Biochemistry* 48: 4139–4149.

Mukherjee T, McCulloch KM, Ealick SE & Begley TP (2007) Gene identification and structural characterization of the pyridoxal 5′-phosphate degradative protein 3-hydroxy-2-methylpyridine-4,5-dicarboxylate decarboxylase from *Mesorhizobium loti* MAFF303099. *Biochemistry* 46: 13606–13615.

Mukherjee T, Hilmey DG & Begley TP (2008) PLP catabolism: identification of the 2-(acetamidomethylene)succinate hydrolase gene in *Mesorhizobium loti* MAFF303099. *Biochemistry* 47: 6233–6241.

Nelson MJ & Snell EE (1986) Enzymes of vitamin B₆ degradation. Purification and properties of 5-pyridoxic acid oxygenase from *Arthrobacter* sp. *J Biol Chem* 261: 15115–15120.

Rigali S, Derouaux A, Giannotta F & Dusart J (2002) Subdivision of the helix-turn-helix GntR family of bacterial regulators in the FadR, HutC, MocR, and YtrA subfamilies. *J Biol Chem* 277: 12507–12515.

Simon R, Priefer U & Pühler A (1983) A broad host range mobilization system for in vivo genetic engineering: transposon mutagenesis in gram negative bacteria. *Bio/Technology* 1: 784–791.

Tamura A, Yoshikane Y, Yokochi N & Yagi T (2008) Synthesis of 4-pyridoxolactone from pyridoxine using a combination of transformed *Escherichia coli* cells. *J Biosci Bioeng* 106: 460–465.

Tropel D & Van der Meer JR (2004) Bacterial transcriptional regulators for degradation pathways of aromatic compounds. *Microbiol Mol Biol Rev* 68: 474–500.

Xu Y, Heath RJ, Li Z, Rock CO & White SW (2001) The FadR-DNA complex. Transcriptional control of fatty acid metabolism in *Escherichia coli*. *J Biol Chem* 276: 17373–17379.

Yokochi N, Nishimura S, Yoshikane Y, Ohnishi K & Yagi T (2006) Identification of a new tetrameric pyridoxal 4-dehydrogenase as the second enzyme in the degradation pathway for pyridoxine in a nitrogen-fixing symbiotic bacterium, *Mesorhizobium loti*. *Arch Biochem Biophys* 452: 1–8.

Yokochi N, Yoshikane Y, Matsumoto S, Fujisawa M, Ohnishi K & Yagi T (2009) Gene identification and characterization of 5-formyl-3-hydroxy-2-methylpyridine 4-carboxylic acid 5-dehydrogenase, an NAD⁺-dependent dismutase. *J Biochem* 145: 493–503.

Yoshikane Y, Yokochi N, Ohnishi K, Hayashi H & Yagi T (2006) Molecular cloning, expression and characterization of pyridoxamine-pyruvate aminotransferase. *Biochem J* 396: 499–507.

Yuan B, Yoshikane Y, Yokochi N, Ohnishi K & Yagi T (2004) The nitrogen-fixing symbiotic bacterium *Mesorhizobium loti* has and expresses the gene encoding pyridoxine 4-oxidase involved in the degradation of vitamin B₆. *FEMS Microbiol Lett* 234: 225–230.

Yuan B, Yokochi N, Yoshikane Y, Ohnishi K & Yagi T (2006) Molecular cloning, identification and characterization of the 2-methyl-3-hydroxypyridine-5-carboxylic-acid-dioxygenase-coding gene from the nitrogen-fixing symbiotic bacterium *Mesorhizobium loti*. *J Biosci Bioeng* 102: 504–510.

Yuan B, Yokochi N, Yoshikane Y, Ohnishi K, Ge F & Yagi T (2008) Gene identification and characterization of the pyridoxine degradative enzyme α-(N-acetylamino)methylene succinic acid amidohydrolase from *Mesorhizobium loti* MAFF303099. *J Nutr Sci Vitaminol* 54: 185–190.
