
Curr. Issues Mol. Biol. 14: 47-70.

Global Regulatory Pathways and Cross-talk Control  
Pseudomonas aeruginosa Environmental Lifestyle and  
Virulence Phenotype

Kimberly A. Coggan¹ and Matthew C. Wolfgang¹,²*

¹Department of Microbiology and Immunology, University of  
North Carolina at Chapel Hill, Chapel Hill, North Carolina  
27599, USA  

²Cystic Fibrosis/Pulmonary Research and Treatment  
Center, University of North Carolina at Chapel Hill, Chapel  
Hill, North Carolina 27599, USA  

Abstract  
Pseudomonas aeruginosa is a metabolically versatile environmental bacterium and an opportunistic human pathogen that relies on numerous signaling pathways to sense, respond, and adapt to fluctuating environmental cues. Although the environmental signals sensed by these pathways are poorly understood, they are largely responsible for determining whether *P. aeruginosa* adopts a planktonic or sessile lifestyle. These environmental lifestyle extremes parallel the acute and chronic infection phenotypes observed in human disease. In this review, we focus on four major pathways (cAMP/Vfr and c-di-GMP signaling, quorum sensing, and the Gac/Rsm pathway) responsible for sensing and integrating external stimuli into coherent regulatory control at the transcriptional, translational, and post-translational level. A common theme among these pathways is the inverse control of factors involved in promoting motility and acute infection and those associated with biofilm formation and chronic infection. In many instances these regulatory pathways influence one another, forming a complex network allowing *P. aeruginosa* to assimilate numerous external signals into an integrated regulatory circuit that controls a lifestyle continuum.

Introduction  
Pseudomonas aeruginosa is a Gram-negative, metabolically versatile bacterium that exists in a wide range of environmental habitats and is a primary agent of opportunistic infection in humans, causing both acute and chronic infections (Driscoll, 2007). *P. aeruginosa* thrives in many habitats including: aquatic sediments, water exposed surfaces, soil, plant roots and leaves, and human and animal sewage (Ringen, 1952; Green, 1974; Pellet, 1983). Numerous epidemiological studies have compared the virulence properties of environmental and clinical isolates of *P. aeruginosa* (Nicas, 1986; Römling, 1994; Ferguson, 2001; Wolfgang, 2003a; Pirnay, 2009). With few exceptions it has been reported that clinical isolates from acute infections, or isolates obtained at the onset of chronic infection, are indistinguishable from environmental isolates. These studies support the following conclusions: 1) environmental strains are the primary source of *P. aeruginosa* infection, and 2) the virulence characteristics of *P. aeruginosa* are pre-existing in the environment. Furthermore, these studies indicate that the environmental lifestyle of *P. aeruginosa* involves a repertoire of phenotypes that otherwise support and contribute to opportunistic human infections.

Many *P. aeruginosa* traits that contribute to human virulence can be linked to a particular function relevant to its environmental lifestyle. It has been hypothesized that the virulence factors used by *P. aeruginosa* (and other opportunistic pathogens) to infect humans evolved as defense mechanisms against eukaryotic predators (Hilbi, 2007). Indeed, it is now well established that *P. aeruginosa* can infect and kill numerous protozoan and non-mammalian organisms that likely inhabit the same environmental space (Rahme, 1995; Hilbi, 2007). An important defense mechanism utilized by *P. aeruginosa* is the formation of surface-attached biofilms, multicellular communities encased in a self-produced polysaccharide matrix. Biofilms are intrinsically resistant to harsh conditions and provide protection from both toxic and antibacterial compounds in the environment, as well as predation (Matz, 2005).

In humans, *P. aeruginosa* is associated with both acute and chronic infections in immunocompromised individuals, and is capable of infecting a multitude of sites and tissues, including: lungs, burns, wounds, eyes, ears, and indwelling medical devices (i.e. catheters and ventilators) (Driscoll, 2007). Acute *P. aeruginosa* infections, such as nosocomial pneumonia, are invasive and cytotoxic and frequently result in substantial tissue damage, systemic spread, sepsis, and mortality. The pathogenesis of acute infections relies upon the expression of many surface-exposed and secreted virulence factors, including: toxins, proteases (delivered by a type II secretion system; T2SS), type IV pili (Tfp), flagella and a type III secretion system (T3SS) that can inject a set of eukaryote specific effectors across the plasma membrane of target cells (Sadikot, 2005). The virulence factors important for establishing acute infections are distinct from those critical for chronic infections. Chronic infections are minimally invasive and noncytotoxic. These infections involve the formation of biofilms, which in the context of human infection, protect against assault by the host immune system and provide resistance to antibiotics (Donlan, 2002; Ryder, 2007). Thus, chronic infections rarely result in systemic spread, but instead lead to unrelenting non-productive host inflammation that contributes to the resulting morbidity and mortality (Deretic, 1995).

Chronic pulmonary *P. aeruginosa* infection in individuals with the genetic disease cystic fibrosis (CF) serves as a model for environmental acquisition followed by genetic adaptation to the CF lung and has revealed an inverse relationship between expression of acute and

Corresponding author: matthew_wolfgang@med.unc.edu

Horizon Scientific Press. http://www.horizonpress.com

48 Coggan and Wolfgang

![Diagram](attachment:diagram.png)

Figure 1. Model of the regulatory pathways and cross-talk that modulate *P. aeruginosa* lifestyles. Lines depict direct regulatory mechanisms within given signaling pathways as well as cross-talk between regulatory systems. Arrows represent positive regulation and T-bars indicate negative regulation. Blue lines represent transcriptional regulation while purple lines represent enzymatic reactions. Orange lines depict post-translational modification events. Black lines illustrate post-translational regulation events. Dashed lines indicate unknown mechanisms of regulation. For more details see text references to this figure.

A. cAMP/Vfr signaling. The Chp chemosensory system modulates the enzymatic activity of CyaB, an adenylate cyclase that synthesizes the majority of intracellular cAMP. The phosphodiesterase CpdA degrades intracellular cAMP. When present in sufficient quantities, cAMP binds to and activates the transcription factor Vfr. cAMP/Vfr regulates numerous virulence factors, primarily those associated with an acute virulence phenotype, in addition to *lasR*, the master regulator of the quorum sensing hierarchy. Both *vfr* and *cpdA* are transcriptionally regulated by cAMP/Vfr creating a feedback loop to maintain cAMP homeostasis.

B. Quorum Sensing. Schematic representation of the AHL- and AQ-dependent QS systems in *P. aeruginosa*. LasR activates *lasI* expression to produce C12-HSL. The LasR/C12-HSL complex positively influences expression of the second AHL-dependent QS system (RhlRI/C4-HSL) and many other target genes. The AHL-dependent systems are transcriptionally and post-transcriptionally regulated by numerous regulators including two LuxR homologues (VqsR and QscR), RsmA, and the alternative sigma factor, RpoS. The RhlR/C4-HSL complex negatively regulates the AQ-dependent QS system.

C. Gac/Rsm pathway. The GacS/A TCS positively regulates expression of two sRNA molecules, RsmY and

RsmZ, which bind and sequester the RNA-binding protein RsmA. Activation of the HptB signaling cascade promotes chronic virulence factor expression by specifically activating *rsmY* expression. GacS activity is antagonized by the hybrid sensor kinase RetS and promoted by the orphan sensor kinase LadS. Free RsmA facilitates the expression of acute virulence factors, such as the T3SS, and represses expression of chronic virulence factors including the T6SS and EPS (Pel and Psl) involved in biofilm formation.

D. c-di-GMP signaling. c-di-GMP levels inversely control functions involved in motility (acute) and biofilm formation (chronic). Synthesis and degradation of c-di-GMP is facilitated by diguanylate cyclases and phosphodiesterases. Multiple DGCs and PDEs encoded within the *P. aeruginosa* genome appear to be spatially localized to alter local c-di-GMP concentrations and influence protein function and gene expression by largely unknown mechanisms.

E. MucA signaling. The anti-sigma factor MucA sequesters the sigma factor AlgU. During chronic CF infection, *mucA* mutation results in constitutive AlgU activation. AlgU increases expression of *algR* and together AlgU and AlgR activate the alginate biosynthetic operon. AlgR negatively regulates QS by inhibiting *rhlR* expression. AlgR inhibits *vfr* expression via an unknown mechanism resulting in decreased expression of many acute virulence factors.

F. HptB signaling. Upon activation by an unknown signal the sensor kinase PA2284 transfers a phosphoryl group specifically to HptB. HptB then relays the signal to the response regulator PA3346. Phosphorylated PA3346 functions as a Ser/Thr phosphatase and dephosphorylates PA3347. Phosphorylation is hypothesized to modulate the binding activity of PA3347. In the unphosphorylated state PA3347 is thought to bind an anti-sigma factor, resulting in the release of an unidentified sigma factor, which specifically regulates *rsmY* expression leading to expression of genes associated with swarming motility and biofilm formation.

---

The ability of *P. aeruginosa* to adapt to a diverse set of environmental conditions and to cause distinct infections relies on its ability to control gene expression in response to environmental stimuli. *P. aeruginosa* devotes a substantial proportion of its genomic real estate to regulatory systems that sense environmental conditions and integrate information to alter gene expression or behavior accordingly (Stover, 2000). The first *P. aeruginosa* genome was sequenced in 2001, revealing a remarkable number of transcriptional regulators, and our knowledge of *P. aeruginosa* regulatory pathways has grown exponentially in the subsequent years. Recent work in the field has led to the development of the hypothesis that the regulatory pathways that control the transition from free living (planktonic) to surface-adherent community (sessile) lifestyles in the environment are likely to play a similar role in dictating infection type (acute vs. chronic) in human disease. This review will focus on the regulatory pathways that have emerged as key determinants of *P. aeruginosa* lifestyle: 1) cAMP/Vfr signaling, 2) c-di-GMP signaling, 3) quorum sensing (QS), and 4) the Gac/Rsm

pathway. Two of these pathways rely on second messenger signaling molecules cAMP and c-di-GMP (adenosine 3',5'-cyclic monophosphate and bis-(3'-5')-cyclic dimeric guanosine monophosphate). Cyclic AMP is now recognized as a major mediator of acute virulence gene expression and planktonic lifestyle phenotypes in *P. aeruginosa* (Wolfgang, 2003b). In contrast, c-di-GMP appears to play a critical role in controlling chronic infection-related phenotypes and the biofilm lifestyle. The coordination of population behaviors is also essential in determining the commitment to a particular lifestyle. In *P. aeruginosa* it is evident that QS is an important regulator of acute virulence phenotypes (Whiteley, 1999; Wagner, 2004; Schuster, 2007); however, it also plays a significant role in the transition from the planktonic to sessile lifestyle and in the establishment of chronic infection (Singh, 2000; Christensen, 2007). Finally, the Gac/Rsm pathway appears to act as a global regulator of lifestyle by inversely regulating acute virulence factors and planktonic behaviors with chronic infection phenotypes and biofilm formation. Here we will concentrate on recent advances in cyclic nucleotide signaling (cAMP, c-di-GMP), QS, and post-transcriptional control by the Gac/Rsm pathway. We will first summarize each individual pathway and then highlight particular regulatory interactions between the pathways. Finally, we will discuss the implications of this complex regulatory network in controlling *P. aeruginosa* lifestyle and infection outcome.

### cAMP/Vfr signaling

The second messenger signaling molecule, cAMP, regulates multiple virulence factors involved in *P. aeruginosa* infection (Figure 1A) (Wolfgang, 2003b). Cyclic AMP is involved in regulating gene expression in many bacterial species in response to environmental cues (De Lorenzo, 1988; Botsford, 1992; Macfadyen, 1998) and is produced by the enzymatic activity of adenylate cyclase (AC) enzymes. The *P. aeruginosa* PAO1 genome encodes three ACs, the intracellular ACs, CyaA and CyaB, and a secreted effector ExoY (Stover, 2000; Wolfgang, 2003b). ACs are divided into six classes based upon their evolutionary relationships (Téllez-Sosa, 2002). ExoY belongs to the class II ACs, a group of secreted ACs that function as bacterial toxins (Yahr, 1998). ExoY is secreted via the T3SS directly into the cytoplasm of target host cells and its activity is dependent on a host-encoded cofactor (Yahr, 1998). The role of ExoY in virulence is not well understood, although the AC activity of ExoY does result in actin disruption and rounding of cultured epithelial cells (Cowell, 2005). Mutants lacking either CyaA or CyaB have reduced intracellular cAMP; however, CyaB appears to synthesize the majority of cAMP in *P. aeruginosa* under laboratory growth conditions and during model infection (Wolfgang, 2003b; Smith, 2004). In a murine model of acute pneumonia a *cyaB* mutant is attenuated, indicating a role for cAMP in *P. aeruginosa* virulence (Smith, 2004). CyaA is a cytosolic class I AC, a class found primarily in enterobacteria, while CyaB is a member of the class III AC family ubiquitous among both eukaryotes and prokaryotes (Baker, 2004). CyaB consists of a N-terminal MASE2 (membrane associated sensor 2) domain and a C-terminal AC catalytic domain (Nikolskaya, 2003). The MASE2 domain is thought to function as a membrane-anchored sensor that regulates the activity of the C-term AC catalytic domain (Nikolskaya, 2003).

#### cAMP/Vfr regulated virulence associated genes

In enteric bacteria, the effect of cAMP on gene expression is mediated through the cAMP-receptor protein (CRP), a cAMP-dependent transcriptional regulator. The well-characterized CRP protein from *Escherichia coli*, is primarily involved in regulating genes associated with carbon metabolism in response to glucose availability (Gosset, 2004). CRP functions as a homodimer and is allosterically controlled by binding of cAMP. Following formation of a cAMP-CRP complex, CRP undergoes a conformational change enabling binding of activated CRP to specific DNA sequences within target promoters, facilitating either transcriptional activation or inhibition (Botsford, 1992). In *P. aeruginosa*, cAMP influences gene regulation through its activity as an allosteric activator of the transcription factor, Vfr (virulence factor regulator). Vfr is a member of the CRP family of transcriptional regulators, and shares a high degree of homology to *E. coli* CRP (67% sequence identity) (West, 1994; Gosset, 2004). Vfr coordinately controls the expression of over 200 genes (Suh, 2002; Wolfgang, 2003b) and positively regulates the production of virulence factors important for acute *P. aeruginosa* infections, including: Tfp (West, 1994; Beatson, 2002; Wolfgang, 2003b), the T3SS (Wolfgang, 2003b), the T2SS, secreted toxins, degradative enzymes (West, 1994; Beatson, 2002; Wolfgang, 2003b; Ferrel, 2008), and the *las* and *rhl* QS systems, which control hundreds of additional genes, including many virulence factors (Albus, 1997; Croda-García, 2011). In contrast, Vfr negatively regulates the expression of flagellar genes by repressing expression of *fleQ*, which encodes the master regulator of flagellar biogenesis (Dasgupta, 2002).

##### Allosteric regulation of Vfr: cAMP-dependent/independent mechanisms

A consensus Vfr binding sequence has been proposed (Kanack, 2006), and direct binding of Vfr to DNA sequences within target promoters of several genes including those encoding ToxA (*toxA*), LasR (*las* quorum-sensing regulator), FleQ, RegA and PtxR (*toxA* expression regulators) and CpdA (a cAMP phosphodiesterase) has been demonstrated (Albus, 1997; Dasgupta, 2002; Kanack, 2006; Ferrel, 2008; Fuchs, 2010b). Fuchs et al. further identified both cAMP-dependent and -independent Vfr DNA binding mechanisms (Fuchs, 2010a). Vfr binding to the majority of target promoters specifically requires cAMP. However, cAMP is not required for Vfr to bind the *lasR* promoter *in vitro* or for *in vivo lasR* promoter activity (Fuchs, 2010a). Fuchs and colleagues (2010a) identified a single novel Vfr binding site within the *vfr* promoter, and *in vitro* transcription studies demonstrated that both cAMP and Vfr are required for *vfr* positive autoregulation (Fuchs, 2010a). A similar mechanism of autoregulation has been demonstrated for *E. coli crp* (Cossart, 1982; Hanamura, 1992).

Subtle structural differences exist between the cAMP-binding pocket of Vfr and *E. coli* CRP, including the presence of three additional amino acids in the Vfr cAMP-binding pocket (West, 1994; Beatson, 2002). Vfr contains a threonine residue (T133) at a position relative to serine 128 (S128) of *E. coli* CRP, and threonine substitution of CRP S128 results in activation of CRP by both cAMP and cGMP (Lee, 1994). Cyclic AMP-dependent gene expression in an *E. coli crp* mutant can be restored by *vfr* expression. However, CRP is not capable of complementing a *P.

aeruginosa *vfr* mutant for exotoxin A (ETA) or protease production, which suggests that Vfr and CRP cannot be functionally interchanged (West, 1994; Suh, 2002). This lack of functional complementation may be due to differences in cAMP-binding affinity, as Vfr has a higher affinity than CRP for cAMP (Serate, 2011). Alternatively, the helix-turn-helix (H-T-H) DNA binding motifs of Vfr and CRP are not identical and may reflect subtle differences in DNA binding site recognition (Kanack, 2006).

In *P. aeruginosa*, several studies have reported conflicting results regarding the ability of Vfr to bind cGMP (Beatson, 2002; Fuchs, 2010a; Serate, 2011). Aspontaneous Vfr mutant (*Vfr<sub>ΔEQERS</sub>*) that lacks 5 amino acids (EQERS) in the cyclic nucleotide-binding domain, including two residues critical for cAMP binding, was incapable of activating Tfp-dependent twitching motility, a form of surface-movement (Mattick, 2002), but retained the ability to regulate a subset of virulence phenotypes (elastase and pyocyanin production) (Beatson, 2002). Homology modeling of *Vfr<sub>ΔEQERS</sub>* on a CRP-cAMP backbone suggests that *Vfr<sub>ΔEQERS</sub>* may have a weaker affinity for cNMP molecules than *E. coli* CRP or wild type Vfr. Beatson *et al.*, suggest that the *Vfr<sub>ΔEQERS</sub>* mutant is incapable of regulating twitching motility but retains the ability to regulate other virulence factors due to altered cyclic nucleotide recognition (Beatson, 2002). Specifically, the authors suggest that another cyclic nucleotide (cGMP) may be responsible for regulating twitching motility, and the *Vfr<sub>ΔEQERS</sub>* mutant may no longer effectively bind this cyclic nucleotide, while regulation of elastase and pyocyanin production is maintained though cAMP autoregulation of Vfr. However, elastase production is also regulated by the *las* QS system (Gambello, 1991). Given that Vfr controls *lasR* expression in a cAMP-independent manner (Fuchs, 2010a), it is likely that *Vfr<sub>ΔEQERS</sub>* no longer responds to cyclic nucleotides, but can still activate cAMP-independent promoters. Fuchs *et al.* (2010a) demonstrated that apo-Vfr was unable to bind promoter DNA in the presence of cGMP and high cGMP concentrations actually inhibited Vfr-DNA complex formation (Fuchs, 2010a). In contrast, Serate and colleagues (2011) reported that cGMP was capable of facilitating Vfr-DNA complexes *in vitro*; although, a significantly higher concentration of cGMP was required compared to cAMP for DNA binding (Serate, 2011). The conflicting results from the Fuchs and Serate studies were obtained utilizing very different experimental assays, which may account for their discrepancies (Fuchs, 2010a; Serate, 2011). Although the effect of cGMP on Vfr activity has not been definitively determined, it is important to note that both of these studies utilized cGMP concentrations that are not likely to be achieved *in vivo*, and thus cGMP is unlikely to be a biologically relevant Vfr ligand.

CbpA: A cAMP-binding protein

Genome analysis studies have identified additional cyclic nucleotide-binding proteins in many bacterial species, suggesting that cAMP-binding proteins are involved in regulating a variety of processes in prokaryotes (Botsford, 1992). Bioinformatic analysis identified a novel cAMP-binding protein in *P. aeruginosa* PAO1, termed CbpA (cAMP-binding protein A). Using structural modeling and functional studies, Endoh and Engel determined that CbpA is comprised of a C-terminal CAP domain, which binds cAMP, and a N-terminal degenerate CAP domain (Endoh,

*P. aeruginosa* Environmental Lifestyle and Virulence 51

2009). A putative Vfr-binding site (VBS) was identified upstream of the *cbpA* translational start site. Mutation of the putative *cbpA*-associated VBS resulted in decreased *cbpA* transcription compared to wild type, indicating that Vfr likely regulates *cbpA* expression. Subsequent EMSA studies confirmed binding to a DNA fragment containing the VBS identified in the *cbpA* promoter (Endoh, 2009). As Vfr regulates *cbpA* transcription, the effect of *cbpA* mutation on Vfr-regulated phenotypes (protease secretion, T3S, twitching motility, and biofilm formation) was investigated. In all cases, *cbpA* mutation had no effect on Vfr-dependent phenotypes, and a *cbpA* mutant was not attenuated in a mouse model of pneumonia. 2D-PAGE determined that CbpA does not affect expression or degradation of proteins, as no significant difference in protein expression profiles between wild type and a *cbpA* mutant was observed. Taken together these results suggest that CbpA does not influence the function of any known Vfr-regulated gene products and has an unknown function in *P. aeruginosa* (Endoh, 2009). Interestingly, CbpA-GFP fusion proteins were used to determine that CbpA levels are growth-phase regulated and CbpA localizes to the flagellated old cell pole in a cAMP-dependent manner (Endoh, 2009), which may provide insight into the function of this protein.

cAMP homeostasis

As cAMP is a membrane-impermeable molecule, in order to maintain homeostasis and reset the cAMP-signaling cascade subsequent to activation, the intracellular cAMP concentration must be tightly controlled. Homeostasis can be achieved through a number of regulatory mechanisms, including control at the level of cAMP synthesis, degradation, and excretion. Signal attenuation can be achieved via degradation of cAMP to 5'-AMP by the enzymatic activity of cyclic 3'-5'-AMP phosphodiesterases. In *P. aeruginosa* the cAMP phosphodiesterase, CpdA, has been characterized. CpdA was shown to have cAMP phosphodiesterase activity *in vitro* and plays a pivotal role in cAMP homeostasis *in vivo* (Fuchs, 2010b). Furthermore, the cAMP-Vfr complex positively regulates *cpdA* expression. CpdA protein reduces intracellular cAMP levels creating a feedback loop (Fuchs, 2010b). Cyclic AMP homeostasis allows specific cAMP concentrations to be achieved, which may result in a hierarchy of gene expression within the cAMP-Vfr regulon. Upregulation of *cpdA* by cAMP-Vfr at high cAMP levels would promote cAMP degradation and likely limits the expression of cAMP-Vfr-dependent genes to a narrow temporal window.

In *P. aeruginosa* cAMP homeostasis is also regulated at the level of AC activity. In a screen to identify regulators of CyaB function, numerous components of the Chp chemotaxis-like chemosensory system were identified (Fulcher, 2010). The Chp system components are homologous to the *E. coli* chemotaxis system responsible for controlling flagellar motility (Silversmith, 1999). The Chp system was previously shown to control Tfp production and twitching motility (Darzins, 1997; Whitchurch, 2004). Additionally, the Chp system appears critical for *P. aeruginosa* virulence, as mutants are attenuated in a *Drosophila* model of *P. aeruginosa* infection. The reduced virulence displayed by these mutants was more significant than could be attributed to their role in Tfp regulation alone (D'Argenio, 2001). A subset of mutants (*pilG* (*cheY*), *pilL* (*cheW*), *pilJ*

(MCP), chpA (cheA), fimL, fimV) displayed reduced cAMP levels. Increased cAMP levels were measured in the pilK (cheR) and pilH (cheY) mutants compared to the parental strain (Fulcher, 2010). These results indicate that Chp gene cluster (pilGHIJKchpABC) along with fimL and fimV affect cAMP intracellular concentrations and exert either a positive (pilG, pilI, chpA, chpC, fimL, fimV) or negative (pilH, pilK, and chpB) regulatory effect. fimL mutants resemble cyaB mutants for a variety of phenotypes including twitching motility defects, decreased T3SS transcription, as well as decreased cAMP levels (Inclan, 2011). Additionally, it was shown that FimL, CyaB and PilJ (Delange, 2007; Inclan, 2011) are polarly localized; which suggests that FimL

functions to link the Chp system and CyaB to regulate the spatial and temporal production of cAMP to specifically influence polarly localized structures (Tfp and T3SS) (Delange, 2007; Inclan, 2011). Western blot analysis determined that CyaB expression was equivalent among all the Chp mutants compared with the parental strain (Fulcher, 2010) and thus Chp does not appear to modulate CyaB expression. To address the possibility that Chp mutants control cAMP levels by controlling cAMP degradation, the effect of PilG and PilH (both CheY homologs) on intracellular cAMP levels was assessed. Using a genetic approach it was demonstrated that PilG and PilH do not regulate cAMP levels via degradation, but specifically affect CyaB activity (Fulcher, 2010). These results support a model whereby the

---

**Planktonic Environment Sessile**

cAMP, Quorum Sensing c-di-GMP, Gac/Rsm

Acute Chronic

Invasive Host Persistent Infection

---

**Quorum Sensing T2SS**

Flagella Type IV pili

---

**Figure 2. Pseudomonas aeruginosa exhibits parallel lifestyle extremes in the environment and human host. The interplay of four global regulatory pathways (cAMP, c-di-GMP, Quorum sensing and Gac/Rsm) appears to create a phenotypic continuum that controls transition between a planktonic and sessile lifestyle within the environment; and plays an analogous role in human infection by inversely controlling acute and chronic infection phenotypes.**

1. In the environment *P. aeruginosa* can exist as planktonic cells or small groups of free living motile organisms, providing the means to colonize new environmental niches.
2. As the population increases, cells associate as a quorum, producing QS signal molecules and exhibit social behaviors (degradative enzyme and toxin secretion),
which promote nutrient acquisition and group survival among environmental predators.
3. Upon interaction with a solid surface, *P. aeruginosa* can attach via Tfp or flagella.
4. Following loose-attachment, *P. aeruginosa* may exhibit surface motility utilizing Tfp or flagella to move toward nutrients. Upon generation of an intimate surface attachment, *P. aeruginosa* may initiate microcolony formation.
5. Eventually *P. aeruginosa* becomes sessile and produces exopolysaccharides that encase the bacteria in a complex matrix, which protects the bacteria from environmental fluctuations and provides a physical barrier against predators.
6. Unknown environmental signal(s), or stochastic processes, cause members of the sessile community to become motile, leave the biofilm, and return to a planktonic lifestyle.

Chp system controls intracellular cAMP pools by affecting CyaB activity, although it remains to be determined whether this effect is direct or occurs through additional proteins.

**Environmental signals affecting intracellular cAMP**

The signal(s) resulting in activation of the Chp system are not known. However, *in vitro* culture conditions of low calcium (Wolfgang, 2003b) or high osmolarity (Rietsch, 2006) influence intracellular cAMP levels, but whether these represent relevant signals *in vivo* remains to be determined. As the Chp system also regulates Tfp production and twitching motility, the chemoattractants (phosphatidylethanolamine and/or phosphatidylcholine) known to affect twitching motility in *P. aeruginosa* may serve as signals that influence cAMP levels (Kearns, 2001; Barker, 2004).

**Quorum Sensing**

Quorum sensing (QS) is a form of bacterial cell-cell communication utilized by many species to sense population density and coordinate gene expression. QS is achieved by self-production of small diffusible signaling molecules termed autoinducers, such that increases in the bacterial population density results in accumulation of signaling molecules. Once a threshold concentration is achieved, the autoinducers bind their cognate receptors, which directly or indirectly activate gene expression. *P. aeruginosa* encodes three QS systems, two N-acyl-homoserine lactone (AHL) based signaling systems (*las* and *rhl*) and a 2-alkyl-4-quinolone (AQ) based signaling system (Figure 1B). These three QS systems are involved in the regulation of virulence factor production, biofilm maturation, and motility phenotypes. The *P. aeruginosa* QS systems are arranged hierarchically with the *las* system positively regulating both the *rhl* (Latifi, 1996; Pesci, 1997) and AQ (Wade, 2005) systems. Additionally, the *rhl* system negatively regulates the AQ system (McGarth, 2004), and each of these systems are further modulated by a plethora of regulators that function at both the transcriptional and post-transcriptional level.

**AHL-dependent QS systems**

Each of the *P. aeruginosa* QS systems utilizes a structurally distinct diffusible autoinducer and cognate regulator protein pair to control downstream gene expression. The *las* and *rhl* systems use dedicated autoinducer synthases (LasI and RhlI) to produce the acyl-homoserine lactone (acyl-HSL) molecules N-(3-oxododecanoyl)-HSL (C12-HSL) and N-butyryl-HSL (C4-HSL), respectively. When sufficient autoinducer concentrations are achieved, the molecules bind and activate their cognate receptors, LasR and RhlR (Juhas, 2005; Schuster, 2006). Together the *las* and *rhl* systems directly or indirectly regulate roughly 10% of the *P. aeruginosa* genome (Schuster, 2006). Transcriptome analysis has revealed significant overlap between the regulons of these two systems (Schuster, 2003). The most abundant class of QS-regulated genes identified by microarray studies were those that encode secreted factors (toxins and extracellular enzymes).

Activation of target genes has been shown to require C12-HSL-dependent multimerization of LasR (Kiratisin, 2002). In contrast, RhlR requires C4-HSL for activation, but is not dependent on C4-HSL binding for dimerization

(P. aeruginosa Environmental Lifestyle and Virulence 53)

(Ventre, 2003). Both the *las* and *rhl* systems contain a positive feedback loop where their regulators, LasR and RhlR, induce transcription of their cognate synthase genes allowing for rapid signal amplification (Seed, 1995; Latifi, 1996). Further, binding of the LasR-C12-HSL complex activates transcription of *rhlR* and *rhlI*, accounting for the hierarchical relationship between the two systems (Latifi, 1996; Pesci, 1997).

Biochemical analyses have revealed distinct differences in the DNA-binding properties of LasR and RhlR. Schuster et al. (2004) demonstrated that LasR recognition sequences do not require dyad symmetry, and that the presence or absence of dyad symmetry corresponds with cooperative or non-cooperative binding of LasR to promoter sequences (Schuster, 2004b). In contrast, *rhl*-responsive promoters appear to rely on dyad symmetry, highlighting probable determinants that facilitate promoter specificity. Indeed the recognition sequences for LasR and RhlR do not appear to be constrained to a single consensus sequence, rather the quorum-controlled promoters have a high degree of heterogeneity, thus generating a signaling continuum; where some promoters respond to C12-HSL signal alone, both C12- and C4-HSL signals, or C4-HSL signal alone (Schuster, 2007).

**AQ-dependent QS system**

The third *P. aeruginosa* QS system utilizes an AQ signaling molecule, designated the *Pseudomonas* Quinolone Signal (PQS; 2-heptyl-3-hydroxy-4-quinolone). PQS is synthesized by gene products encoded by the *pqsABCD* operon and *pqsH*, which are also involved in the production of other AQS including the immediate PQS precursor, 2-heptyl-4-quinolone (HHQ) (Deziel, 2005; Wade, 2005). PQS binds with high affinity to its cognate LysR-type receptor MvfR (PqsR); together PQS and MvfR control approximately 140 genes, most of which are co-regulated by the *rhl* system (Deziel, 2005). Interestingly, PQS activates *rhlI* expression (McKnight, 2000) while the *rhl* system negatively regulates the AQ-dependent QS system (McGarth, 2004). Moreover, MvfR is positively controlled by LasR-C12-HSL, indicating that a delicate balance in the ratio of C12-HSL to C4-HSL is necessary for production of PQS (McGarth, 2004).

AQ-dependent signaling is a critical component of the *P. aeruginosa* QS circuit. The exogenous addition of PQS to bacterial cultures promotes biofilm formation and positively influences the production of QS-regulated virulence factors, particularly pyocyanin and proteases (McKnight, 2000; Diggle, 2003). PQS appears to act as an allosteric regulator of MvfR as binding of MvfR to the *pqsABCD* promoter is greatly enhanced by the PQS (Wade, 2005; Xiao, 2006). The *pqsABCD* gene products direct synthesis of the precursor molecule HHQ, which is also capable of potentiating MvfR binding to the *pqsABCD* promoter (Xiao, 2006). Furthermore, a *pqsH* mutant incapable of converting HHQ to PQS, does not display a defect in MvfR-mediated gene expression or virulence, indicating that HHQ is also a functional QS molecule (Xiao, 2006).

**Regulators of QS homeostasis**

In *P. aeruginosa*, expression of QS controlled genes is largely delayed until the stationary phase of growth. While this observation suggests that the threshold concentration of autoinducers is not achieved until stationary phase,
exogenous addition of AHLs or PQS to *P. aeruginosa* strains during logarithmic growth does not result in the immediate activation of QS-dependent gene expression (Whiteley, 1999; Diggle, 2002; Schuster, 2003). In contrast, overexpression of the QS receptor proteins LasR and RhlR during logarithmic growth, in the presence of their cognate autoinducers, results in the advanced expression of many, but not all, QS-controlled genes (Schuster, 2007). Taken together, these observations suggest that activation of QS genes requires additional factors that likely control receptor expression or function, and/or act as coregulators of some QS-controlled target genes.

The transcriptional regulator RsaL, encoded by the *rsaL* gene, represses *lasI* expression through direct binding to the *lasI* promoter (de Kievit, 1999; Rampioni, 2006). Expression of *rsaL* is positively regulated by LasR-C12-HSL (de Kievit, 1999), creating a negative feedback loop within the *las* system. Negative regulation of *lasI* by RsaL serves to counterbalance the positive feedback loop in which LasR-C12-HSL binds the *lasI* promoter, enhancing *lasI* expression and C12-HSL production. The *rsaL* and *lasI* genes are divergently transcribed and coregulated. LasR binding to the *lasI-rsaL* intergenic region triggers transcription of both genes (Whiteley, 2001; Schuster, 2004b), while RsaL binding results in repression of *lasI-rsaL* transcription (Rampioni, 2007), ensuring tight control of this negative regulator. Rampioni et al. (2007) demonstrated that while RsaL and LasR are capable of binding simultaneously to the intergenic region; repression by RsaL supersedes LasR-mediated activation (Rampioni, 2007). RsaL appears to be a global regulator, controlling ~130 genes independent of its effect on *lasI* expression (Rampioni, 2007), including genes whose products are involved in antibiotic resistance and biofilm formation in *P. aeruginosa* (Rampioni, 2009). Evidence for *rsaL* upregulation was found in *P. aeruginosa* RNA harvested from sputum samples obtained from chronically infected CF patients, suggesting that RsaL plays a role in C12-HSL homeostatic control during chronic infection (Son, 2007).

Given that most QS-controlled genes are expressed during stationary phase, it is not surprising that the stationary phase sigma factor, RpoS (*σS*), has been shown to modulate the expression of approximately 40% of QS-controlled genes (Whiteley, 2000; Schuster, 2004a). Expression of *rpoS* is modestly activated by the *rhl* QS system (Latifi, 1996; Wagner, 2003); however, the presence of RpoS promoter sequences in several QS-controlled genes suggests that RpoS regulates QS gene expression directly (Schuster, 2004a). Furthermore, RpoS appears to have an indirect effect on QS gene expression as it plays a subtle role in activating *lasR* and *rhlR* expression during stationary phase (Schuster, 2004a). While the details of the interplay between QS and RpoS are poorly understood, it is clear that RpoS is involved in restricting QS control of a subset of genes until the onset of stationary phase.

In addition to RpoS, other global regulators elicit significant control over QS. Mutation of the *mvaT* gene results in premature and elevated expression of quorum-regulated genes, demonstrating that MvaT is involved in controlling the magnitude and timing of QS-dependent gene expression (Diggle, 2002). MvaT is a member of the histone-like nucleoid structuring (H-NS) protein family, involved in transcriptional silencing (Vallet, 2004). MvaT and its homolog MvaU bind specific sequences in AT-rich DNA and oligomerize to form extended protein-DNA complexes that prevent transcription (Castang, 2008). MvaT/U control the expression of approximately 150 genes. Transcriptional profiling and genome-wide location analysis (chromatin immunoprecipitation combined with tiled whole-genome microarray mapping) revealed that MvaT/U are likely to be involved in silencing as many as 350 genes. Among these, MvaT/U appear to play a direct role in regulating *lasR*, *lasI*, *rsaL*, *mvfR*, and *rpoS*, providing a link to all three *P. aeruginosa* QS systems (Castang, 2008).

While QS homeostasis is subject to control by the above global regulators, other regulators have been identified that appear to specifically modify QS. VqsR (virulence and quorum-sensing regulator), a member of the LuxR-type family of transcriptional regulators, appears to provide another feedback mechanism to finely tune the QS system. Specifically, VqsR has been shown to control production of AHL signaling molecules and virulence factors. Transcriptome analysis of a *vqsR* mutant demonstrated that VqsR regulates the expression of a subset of QS-controlled genes, although VqsR has not been shown to bind AHLs (Juhas, 2004). Further, LasR binds the *vqsR* promoter *in vitro* in the presence of C12-HSL (Li, 2007), indicating that *vqsR* expression is under the control of the *las* QS system.

The AraC-type transcriptional regulator VqsM positively regulates the QS systems. Global transcriptome analysis revealed approximately 300 genes that were directly or indirectly influenced by *vqsM* mutation. Of these, 200 genes were upregulated by VqsM. More than half of the genes positively controlled by VqsM are also activated by QS. This is likely due to VqsM control of several important QS regulators including (*mvfR*, *rhlR*, *rsaL*, *rpoS*, and *vqsR*). Overexpression of *vqsR* in the *vqsM* mutant restored the expression of AHL signaling molecules and most virulence factors, indicating that VqsM regulates QS signaling primarily through modulation of *vqsR* expression (Dong, 2005).

### Control of the QS threshold

C12-HSL reaches steady-state levels long before *P. aeruginosa* reaches stationary phase and activation of the QS system occurs (Chugani, 2001), indicating that additional homeostatic mechanisms must control the timing of QS activation. Recently, several factors involved in regulating the activation threshold of quorum-regulated genes have been identified. QscR, another LuxR homolog, modulates the timing of QS activation by repressing expression of *lasI* (Chugani, 2001). QscR exists as an oligomer or aggregate in cells, but crosslinking experiments demonstrated that QscR is also capable of forming heterodimers with LasR or RhlR. Interaction of QscR with AHL signal molecules resulted in dissociation of the QscR oligomer (Ledgham, 2003). Transcriptome studies demonstrated that QscR controls a discrete regulon independent of RhlR and LasR and can function as either an activator or repressor (Lequette, 2006). Expression of *qscR* is not affected by *lasR* mutation, despite the presence of a well-conserved “lux box” upstream of *qscR* (Ledgham, 2003).

Siehnel and colleagues identified a unique regulator (QteE) involved in inhibiting pre-quorum threshold transcription. QteE blocks the expression of QS-regulated genes without affecting transcription or translation of
lasR, indicating a post-translational control mechanism. Expression of qteE prevented LasR accumulation presumably by affecting protein stability. QteE also reduced RhlR accumulation and blocked Rhl-mediated signaling. Overexpression of lasR overcame the effects of qteE expression, suggesting that the stoichiometry between LasR and QteE is important for QteE control (Siehnel, 2010). Given these results, a model emerges whereby QteE activity exceeds LasR activity at low cell densities, but LasR activity is favored as the cell density increases. However, it has yet to be determined whether QteE affects LasR stability in the presence or absence of bound signal.

QslA was identified in a screen for proteins that interact with LasR (Seet, 2011). QslA homologs are found within several other *Pseudomonas* species, and share approximately 30% sequence identity at the amino acid level; however, none of these homologs have been characterized. The predicted secondary structure of QslA shares a conserved alpha helical structure with TraM and TraM2, anti-activators that inhibit QS activation via protein-protein interactions in *Agrobacterium tumefaciens* (Chen, 2004a). Co-immunoprecipitation assays demonstrated that QslA modulates LasR activity via protein-protein interaction. Further, QslA disrupted binding of LasR to the *lasI-rsaL* intergenic region in EMSA studies, even when a LasR-DNA complex was formed prior to the addition of purified QslA (Chen, 2004a; Seet, 2011). Chromatography analysis determined that QslA (11.8 kDa) exists in solution as a 66 kDa hexamer. Based on previous reports that LasR exists as a dimer (Schuster, 2004b), Seet *et al.* postulate that three pairs of LasR dimers interacts with a QslA hexamer. As both QscR and QslA affect LasR dimerization and DNA-binding respectively, each of these proteins is critical for establishing the QS threshold by modulating free LasR levels. Importantly, at high concentration C12-HSL disrupts QscR inhibition of LasR (and RhlR), while QslA inhibition is not affected by increasing C12-HSL concentrations, suggesting that QslA may control the overall QS threshold. Transcriptome and proteome profiles of *P. aeruginosa* propagated under a variety of experimental conditions demonstrate that QS often controls specific sets of genes in response to particular environmental cues. These results suggest that the QS circuitry may be critical to regulating genes in response to both population density and environmental stimuli.

### Gac/Rsm Pathway

The Gac/Rsm pathway appears to play a fundamental role in controlling the course of *P. aeruginosa* infection by inversely regulating the expression of virulence factors associated with acute (T3S, Tfp) and chronic (exopolysaccharides, type VI secretion (T6S)) disease (Figure 1C) (Goodman, 2004; Heurlier, 2004; Burrowes, 2006; Ventre, 2006; Brencic, 2009a; Moscoso, 2011). In contrast to cAMP/Vfr signaling and the QS systems, the Gac/Rsm pathway controls virulence factor expression post-transcriptionally, via alteration of mRNA stability or translation (Smith, 2004). Central to this phenotypic switching mechanism is the GacS/A two component system (TCS). When activated via phosphorylation, GacA promotes the expression of two small untranslated RNAs (sRNAs), RsmY and RsmZ, which sequester the mRNA-binding protein RsmA (Heurlier, 2004; Kay, 2006; Brencic, 2009a). RsmY and RsmZ are comprised

of multiple stem-loop structures containing GGA motifs in unpaired regions that facilitate high affinity binding of RsmA (Valverde, 2003). As such, RsmZ and RsmY expression levels are inversely proportionate to cellular levels of free RsmA. Deletion of *rsmY* and *rsmZ* results in phenotypes similar to those of a *gacA* mutant, suggesting that the GacS/A system primarily functions to modulate expression of these sRNA genes (Kay, 2006; Brencic, 2009a). Activated GacA binds a conserved sequence element termed the GacA box (TGTAAGN<sub>6</sub> CTTACA, N= any nucleotide) present in the *rsmY* and *rsmZ* promoters (Brencic, 2009b). RsmY and RsmZ are also part of a feedback mechanism and inhibit their own transcription by interfering with the GacS/A system, though it is unclear whether this is accomplished by inhibiting GacA phosphorylation, by preventing GacA binding to the GacA box, or by inhibiting GacS/A translation. While the mechanism of this feedback regulation remains to be determined, it is apparent that RsmY and RsmZ levels are subject to tight homeostatic control.

#### RsmA: a post-transcriptional regulator

RsmA is a member of the CsrA family of small translational regulatory proteins that have been identified in numerous bacteria and are known to regulate multiple aspects of virulence gene expression (Lapouge, 2008). Transcriptome analyses have defined the RsmA regulon, demonstrating negative control of T6S and exopolysaccharide (EPS) production involved in biofilm formation as well as positive control of T3S, Tfp synthesis, and flagellar motility (Burrowes, 2006; Brencic, 2009a). The mechanism of RsmA mediated translational repression has been extensively characterized. RsmA and RsmA-like proteins of the CsrA family bind GGA motif(s) present within the untranslated 5' sequence of target mRNAs, thus occluding the ribosome binding site and preventing ribosome recruitment (Lapouge, 2008). In contrast, the mechanism by which RsmA positively regulates expression of T3S, Tfp, and flagellar genes remains largely unknown. The *E. coli* RsmA homolog, CsrA, positively affects translation by promoting mRNA stability (Wei, 2001), which may represent a standard mode of positive regulation for this protein family. Alternatively, it has been proposed that positive regulation by RsmA may be indirectly attributed to negative regulation of one or more transcriptional repressors by RsmA (Brencic, 2009a).

#### Additional regulators of the GacS/A system

Several additional accessory regulators modulate the GacS/A system. Two hybrid sensors lacking cognate sensor kinases, RetS and LadS, have antagonistic roles relative to one another, and exert negative and positive effects respectively on *rsmZ* expression (Goodman, 2004; Laskowski, 2006; Ventre, 2006). *retS* (regulator of exopolysaccharides and T3S) and *rsmA* mutants have similar phenotypes including hyperbiofilm formation, reduced T3S expression and cytotoxicity, and diminished twitching motility (decreased Tfp expression) (Pessi, 2001; Goodman, 2004; Zolfaghar, 2005; Goodman, 2009). LadS (lost adherence sensor) was identified in a transposon library screen for mutants reduced in the ability to form biofilms (Vallet, 2001).

Both RetS and LadS are modular proteins containing N-terminal 7TMR-DISMED2 (7-transmembrane-receptor with diverse intracellular signaling modules extracellular
domain 2) and a 7TMR-DISM_7TM (seven transmembrane segments found adjacent to 7TMR-DISM domains) and C-terminal response regulator-like receiver domains, although RetS contains two receiver domains (Goodman, 2004; Ventre, 2006). The N-terminal domains of these proteins have been predicted to bind carbohydrates and their modest (35%) sequence identity suggests they may bind similar, but distinct ligands. RetS and LadS functionally interact with the GacS/A system. Interestingly, RetS activity is independent of conserved phosphorelay residues in RetS and the observation that RetS lacks autophosphorylation activity suggests that RetS exerts its regulatory function directly (Hsu, 2008; Goodman, 2009). RetS has recently been shown to heterodimerize with GacS resulting in suppression of GacS autophosphorylation and thus inhibits phosphotransfer to GacA (Goodman, 2009). This mechanism of sensor kinase inhibition via heterodimerization is novel among signaling proteins. The mechanism for GacA activation, via LadS, has not been fully characterized but recent work in *P. fluorescens* suggests that it also likely makes physical contact with GacS (Workentine, 2009). While transcriptome analysis of the RetS regulon revealed 397 target genes, LadS only affects the expression of 79, with slightly over half of these overlapping with those inversely affected by RetS (Ventre, 2006). These results indicate that RetS and LadS function to reciprocally regulate numerous gene targets via functionally altering GacS/A output.

The HptB signaling module

The *P. aeruginosa* genome is predicted to encode 12 hybrid-type kinases and 3 putative single domain Hpt proteins in addition to numerous TCS proteins (Stover, 2000; Chen, 2004b). One of these these hybrid sensors, PA2284, was identified in a transposon mutant screen for its association with the hyperbiofilm phenotype (Goodman, 2004). This hybrid sensor, along with two additional hybrid sensors, was demonstrated to transfer a phosphoryl group to the histidine phosphotransfer protein (HptB) (Hsu, 2008). HptB then relays the signal (phosphoryl group) to the orphan response regulator PA3346 as demonstrated via *in vitro* and *in vivo* experiments (Figure 1F) (Lin, 2006; Hsu, 2008).

Phosphorelay profiling assays determined that HptB does not interact directly with GacA; instead HptB interacts directly with the response regulator PA3346. PA3346 is a phosphatase consisting of a N-terminal phosphoryl signal receiver domain and a C-terminal eukaryotic type Ser/Thr phosphatase domain (Hsu, 2008). Bioinformatic analysis of the *P. aeruginosa* PAO1 genome revealed that *hptB* (PA3345), PA3346, and PA3347 are clustered closely together and reverse transcription PCR confirmed the operonic structure (Hsu, 2008). Hsu *et al.* further demonstrated via two-hybrid experiments that the output domain of PA3346 interacts directly with PA3347, a protein with similarity to an anti-sigma (*σ*) factor. These observations suggest that PA3346 regulates PA3347 via dephosphorylation, and is supported by the finding that the conserved serine-56 residue of PA3347 is phosphorylated *in vivo* and the degree of PA3347 phosphorylation inversely correlated with the amount of PA3346 (Hsu, 2008). The activity of PA3346/PA3347 was also shown to specifically impact *rsmY* expression without affecting *rsmZ*. These results led to the hypothesis that an unidentified *σ* factor is released upon activation of the HptB pathway, and the freed *σ* factor binds the *rsmY* promoter but not the *rsmZ* promoter (Figure 1F).

The HptB signaling pathway is involved in the regulation of swarming motility, a form of surface-associated motility, and biofilm formation. A *hptB* mutant is defective for swarming motility while mutants lacking PA3346 and PA3347 display enhanced swarming motility (Hsu, 2008). Biofilm synthesis and disintegration was faster in a *hptB* mutant compared to wild type, while a mutant lacking PA3347 was indistinguishable from the parent PAO1 strain (Hsu, 2008; Bordi, 2010). Introduction of the PA3346 or PA3347 mutations into a *hptB* mutant background ablated the hyperbiofilm phenotype (Bordi, 2010). The inverse phenotypes displayed by the *hptB* mutant and PA3346/PA3347-lacking strains strongly indicate that HptB antagonizes the activity of PA3346/PA3347.

Microarray studies revealed significant overlap between the *hptB* and *retS* mutant transcriptomes (Bordi, 2010). Both *retS* and *hptB* mutants displayed downregulation of T3S genes and upregulation of the *pel* genes; however only the *retS* mutant displayed increased T6S gene expression (Bordi, 2010). HptB overproduction failed to restore expression of T3S and *pel* genes in the *retS* mutant, indicating that HptB does not partner with RetS. As previously mentioned, the HptB pathway specifically alters *rsmY* expression (Bordi, 2010), although the reason for differential sRNA control is unknown. The upstream regions of *rsmY* and *rsmZ* are disparate and could be recognized by distinct regulatory factors (Brencic, 2009b, a). Secondary mutations in *gacS* or *gacA* suppress the *hptB* phenotype and suggest that HptB functions upstream or at the same level as the GacS/A system (Bordi, 2010). Taken together recent work elucidating the molecular mechanisms by which the Gac/Rsm pathway reciprocally controls acute and chronic virulence factors have uncovered novel means utilized by *P. aeruginosa* to modulate gene expression and phenotypes in response to, as yet, unknown environmental cues.

c-di-GMP signaling

c-di-GMP is an important and ubiquitous secondary signaling molecule in many bacterial species. The intracellular concentration of c-di-GMP within a cell is mediated by the opposing actions of two types of enzymes: diguanylate cyclases (DGC) containing GGDEF-domains that synthesize c-di-GMP; while phosphodiesterases (PDE) containing EAL or HD-GYP domains that degrade c-di-GMP. Several proteins have been identified that contain both GGDEF and EAL-domains, and thus may be capable of both synthesizing and degrading c-di-GMP. Additionally, proteins in various bacterial species have been identified that contain degenerate GGDEF domains, suggesting that these domains are not catalytically active but rather function as allosteric sites (Christen, 2005) and/or c-di-GMP receptors (Newell, 2009). In the *P. aeruginosa* genome, 39 genes contain either a GGDEF, an EAL, or both domains (Figure 1D) (Kulasakara, 2006). Levels of c-di-GMP influence a wide range of phenotypes in diverse bacterial species including cellular responses that effect pathogenesis, including synthesis of adhesins and EPS mediating biofilm formation, motility, secretion, cytotoxicity, synthesis of secondary metabolites, and environmental
stress adaptation. Modulation of c-di-GMP is associated with control of biofilm formation and other group behaviors (Liberati, 2006; Merritt, 2007).

In many bacterial species sessile and motile behaviors are coordinately, but inversely, regulated by c-di-GMP (Tamayo, 2007; Jonas, 2009). Biofilm formation is a surface-associated sessile behavior that requires production of an EPS matrix. Biofilm development requires a number of steps from initial surface attachment, EPS matrix formation, and elaboration of a mature biofilm structure (Ryder, 2007). Cyclic-di-GMP promotes biofilm formation, in part, by positively regulating production of the extracellular matrix at both the transcriptional and allosteric level (Simm, 2004; Hickman, 2005). Bacterial surface appendages are also important for a number of the developmental stages in biofilm formation, including motility toward a surface, initial attachment, and microcolony formation. One class of appendages involved in *P. aeruginosa* biofilm formation are the Cup (chaperone usher pathway) fimbriae. Cup fimbriae are composed of a multimer of the major fimbrial subunit forming the rod structure, which may or may not contain an attached tip adhesin to facilitate specific binding (Sauer, 2004). Multiple *cup* gene clusters are present in various strains of *Pseudomonas*, and in the PAO1 genome three (*cupA*, *cupB*, and *cupC*) have been identified (Vallet, 2001). A *cupA* gene cluster mutant is defective in attachment to solid surfaces (Vallet, 2004).

### Wsp Chemosensory System

WspR was among the first *P. aeruginosa* proteins identified that influence c-di-GMP formation. WspR consists of a CheY-like phospho-receiver domain and a GGDEF domain that has DGC activity *in vivo* and *in vitro* (D'Argenio, 2002; Hickman, 2005; Güvener, 2007). WspR is a response regulator encoded by the *wsp* operon (*wspABCDEF*R), which comprises a multi-component chemosensory system with homology to the well-characterized chemotaxis (Che) pathway of *E. coli* (Ridgway, 1977; D'Argenio, 2002; Güvener, 2007). The Wsp system consists of a putative methyl-accepting chemotaxis protein (WspA), a hybrid CheA-like histidine kinase (WspE), two CheW-like adaptor proteins (WspB and WspD) predicted to link WspA and WspE, a CheR-like methyltransferase (WspC) and a CheB-like methylesterase (WspF), predicted to modulate the methylation status and activity of WspA. Upon detection of the appropriate signal, WspA is predicted to activate the associated WspE histidine kinase, resulting in phosphorylation of the WspR receiver domain and DGC activation. Consistent with this model, purified WspR shows increased DGC activity following *in vitro* phosphorylation by the phosphate donor acetyl phosphate (Hickman, 2005). While the specific signals detected by the Wsp system are currently unknown (see below), inactivation of *wspF* (a putative methylesterase and negative regulator of Wsp signaling) results in increased cell aggregation, a wrinkly colony phenotype, enhanced biofilm formation, and increased intracellular levels of c-di-GMP. The phenotype of a *wspF* mutant is suppressed in a *wspF*, *wspR* double mutant, indicating that activation of WspR DGC activity accounts for the profound phenotypic changes. Transcriptome comparison between the wild type PAO1 strain and an isogenic *wspF* mutant identified 560 genes affected by *wspF*, including the *psl* and *pel* operons, which displayed increased transcript levels in the mutant.

These findings indicate that the *wsp* chemosensory system modulates intracellular c-di-GMP levels and regulates biofilm formation (Hickman, 2005). Complexes of WspR protein appear transiently in the cytoplasm of bacteria with an active Wsp system, suggesting that Wsp clusters form and dissolve, creating localized concentrations of c-di-GMP (Güvener, 2007). Binding of c-di-GMP to large protein complexes is known to affect their activity, and the protein complexes involved in the synthesis of Pel and Psl EPS are likely targets of WspR-generated c-di-GMP. Güvener and colleagues (2007) determined that the Wsp signal transduction pathway is stimulated in response to solid surface contact, which likely involves signal detection by a surface appendage (Güvener, 2007). Taken together these results demonstrate that WspR localizes in a phosphorylation-dependent manner, indicating that spatial and temporal activation of WspR impacts both its activity and localization.

#### Cup Fimbriae

Independent screens in strains PAO1 and PA14 identified a three-component system involved in the regulation of *cup* fimbriae expression, termed *rocA1* (regulator of *cup*), *rocR*, and *rocS1* (sadARS in PA14) (Kuchma, 2005; Kulasakara, 2005). RocA1, RocS1, and RocR have significant homology to the *Bordetella pertussis* BvgA/S/R system involved in regulation of virulence gene expression (Kulasakara, 2005). RocS1 is a sensor kinase containing a PAS domain that is predicted to localize to the cytoplasm adjacent to the two transmembrane segments. PAS domains were originally identified for their function in sensing intracellular oxygen and/or redox potential (Taylor, 1999). RocA1 is a classical response regulator containing the HTH motif typical of DNA binding proteins. RocR contains an N-terminal CheY-like phosphoryl receiver domain, suggesting that the aspartate residue present in this domain is phosphorylated by a sensor kinase (Kuchma, 2005). RocR also possesses a degenerate EAL domain suggesting that it may be involved in the hydrolysis of c-di-GMP. Further characterization of the *rocARS* transposon mutants confirmed that RocS1 and RocA1 activate transcription of the *cupB* and *cupC* gene clusters, while deletion of *rocR* resulted in a minimal increase in *cupC-lacZ* transcriptional reporter activity (Kulasakara, 2005). RocA1 is required for RocS1-mediated activation of the *cup* gene clusters. In addition, the observation that RocS1 overexpression in the absence of *rocA1* had no effect on *cup* gene expression supports a mechanism whereby RocA1 activates *cup* gene expression via RocS1 phosphorylation. RocR antagonizes the activity of RocA1, and two-hybrid analysis suggests this regulation is achieved through direct interaction of RocR with RocS1 preventing RocA1 phosphorylation (Kulasakara, 2005). Because RocR contains a degenerate EAL domain, it was initially hypothesized that RocR does not possess catalytic activity; but instead the degenerate domain would serve as a binding site for c-di-GMP and binding would then influence RocR activation. However, recent work has demonstrated the degenerate EAL domain within RocR does possess catalytic activity and facilitates the hydrolysis of c-di-GMP to the dinucleotide 5′ pGpG (Rao, 2008). Transcriptome analysis identified additional genes influenced by mutations in the RocA/R/S system, during either planktonic or biofilm growth. Wild type bacteria and

58 Coggan and Wolfgang

rocR/S mutants differentially regulated several genes during planktonic growth, but an even more significant difference in the number of differentially regulated genes was observed after 5 days of biofilm growth. Among the genes displaying the most significantly altered expression in these studies were genes encoding the T3SS, where several T3SS genes (apparatus components and effectors) were elevated in the rocR/S mutant biofilm in comparison to the wild type biofilm (Kuchma, 2005). The mechanism by which the RocA/R/S system influences the expression of additional genes is not clear.

The RocA/R/S system regulates later steps in biofilm formation; rocARS mutants display initial defects in attachment but by 24 hours display surface attachment comparable to that of wild type (Kuchma, 2005). However, during biofilm growth in a flow-cell, rocARS mutants fail to form macrocolonies with defined borders and channel networks, an architecture displayed by the wild type strain by day 5. All three components of the RocA/R/S system appear to be required for mature biofilm formation, as each individual mutant confers the same defect in mature biofilm formation by day 5 during flow cell growth (Kuchma, 2005). As many genes required for formation of the T3SS were downregulated in rocARS mutants, the RocA/R/S system may control the switch from cytotoxicity, an acute virulence phenotype, to biofilm formation, a chronic virulence phenotype or an alternative lifestyle during growth in a hostile environment.

P. aeruginosa strain PA14 is a more virulent clinical isolate than PAO1, although both share a similar host range (Rahme, 1995). The PA14 genome contains a pathogenicity island (PAPI-1) that is absent from the PAO1 genome, which carries many genes of unknown function and a fourth cup gene cluster (cupD)(He, 2004). Directly adjacent to the cupD gene cluster, are two divergently oriented sets of genes encoding TCS regulatory systems. The cupD gene cluster and these regulatory components are flanked by inverted repeats, suggesting they have been simultaneously acquired. One of these regulatory components, pvrR was previously shown to encode a “phenotype variant regulator” involved in regulating of the frequency of antibiotic resistant variants (Drenkard, 2002). PvrR contains an EAL domain, and therefore likely possesses PDE activity. The second TCS adjacent to the cupD gene cluster has been annotated rcsB (response regulator) and rcsC (sensor kinase) due to sequence homology with TCS systems present in Salmonella enterica and E. coli (Mikkelsen, 2009). Previous studies have demonstrated the contribution of these genes to PA14 virulence in both animals and plants by analyzing transposon (Rahme, 1997) or in-frame deletion mutants (He, 2004). cupD gene expression, although not induced under laboratory conditions, is activated by RcsB and is repressed by the putative cognate sensor, RcsC (Mikkelsen, 2009; Nicastro, 2009). PvrR shows high similarity (45%) to the response regulator RocR, which down-regulates cupB and cupC expression (Drenkard, 2002; Kulasakara, 2005). Indeed, Mikkelsen (2009) and colleagues demonstrated that pvrR expression antagonizes cupD gene cluster activation via RcsB. Both RocR and PvrR contain EAL domains within their output domains but lack the classical H-T-H motif, suggesting that they regulate Cup fimbriae expression by modulating c-di-GMP levels rather than by direct DNA binding (Kulasakara, 2005; Mikkelsen, 2009).

These findings support the theory that high levels of c-di-GMP promote a biofilm-like lifestyle, while low levels of c-di-GMP restrict bacteria to a planktonic mode of growth.

**Biofilm Formation and Adherence**

FimX is another protein involved in regulating motility in *P. aeruginosa* and is required for twitching motility (Huang, 2003). In addition, Tfp are required for adherence and biofilm formation in *P. aeruginosa*. FimX contains both GGDEF and EAL domains involved in modulating c-di-GMP levels, although FimX does not appear to possess DGC activity (Kazmierczak, 2006). Through analysis of a series of deletion mutants Kazmierczak et al. (2006) demonstrated that both intact GGDEF and EAL domains are required for FimX PDE activity and restrict localization to a single bacterial pole. These results support the hypothesis that the GGDEF domain of FimX functions as an allosteric site for binding of GTP, which activates FimX PDE activity. Consistent with this hypothesis, mutation of the GGDEF domain in FimX abolishes GTP binding and stimulation of PDE activity (Kazmierczak, 2006). However, recent structural studies indicate that the degenerate GGDEF domain of FimX is incapable of nucleotide binding, while the EAL domain binds c-di-GMP with high affinity inducing a conformational change that may impede FimX binding to its putative partner located at the bacterial pole to regulate Tfp production and twitching motility (Navarro, 2010; Qi, 2011).

Starkey et al., identified genes transcriptionally upregulated in response to elevated c-di-GMP levels, revealing a small (35) subset of induced genes (Starkey, 2009). Although most of the identified genes belonged to either the pel or psl gene clusters, one target belongs to a two-partner secretion (TPS) system encoding a secreted adhesin and its transporter. Borlee et al. (2010) renamed PA4625 and PA4624, cdrA (cyclic diguanylate-regulated TPS partner A) and cdrB respectively, based on the observation that transcription of this TPS system is c-di-GMP-dependent. CdrA is processed at both the C- and N-termini, although following secretion full-length CdrA appears to be the major cell-associated species. Rugose small colony variants (RSCV) are characterized by formation of large cell aggregates and increased biomass, this phenotype can be caused by wspF mutation, which leads to WspR activation (D'Argenio, 2002; Hickman, 2005). A dramatic phenotype was observed when a cdrA mutation was introduced into the wspF background, the resulting biofilm cells were less tightly packed than the cell clusters observed in a wspF mutant, suggesting that CdrA is involved in promoting auto-aggregation. As the cdrAB and psl/pel gene clusters are co-regulated, Borlee and colleagues hypothesized that CdrA may bind to either Psl or Pel EPS, this hypothesis is supported by the presence of an N-terminal carbohydrate-binding domain. Indeed, as predicted psl mutant strains did not auto-aggregate when cdrAB was overexpressed. Importantly, the production of CdrA was not affected by pslBCD mutation, nor were Psl levels significantly affected by cdrA mutation. Furthermore, CdrA was demonstrated to bind directly to Psl (Borlee, 2010). The findings from this study highlight that biofilms are not simply random distributions of secreted polymers with imbedded bacterial cells, but are complex and more ordered than originally appreciated.

Small-colony Variants

In chronic isolates from CF patients, small colony variants (SCV) are often observed and this phenotype has been correlated with poor lung function and enhanced antibiotic resistance (Häussler, 2004). SCVs are characterized by auto-aggregative properties, size, and enhanced biofilm formation. SCV phenotypes may also arise following exposure to antibiotics *in vitro* and from biofilm cultures (Drenkard, 2002).

To investigate the factors involved in the SCV phenotype, Meissner and colleagues (2007) screened 10,000 transposon mutants of the clinical SCV 20265 isolate for reversion to the wild type phenotype. Two transposon insertions mapped to the *cupA* gene cluster involved in biofilm formation (Vallet, 2001), and two others mapped to PA1119 and PA1120, encoding hypothetical proteins (Meissner, 2007). To determine the involvement of CupA expression in the SCV phenotype, Meissner et al. compared CupA expression in SCV 20265 to the wild type strain and the *cupA* gene cluster transposon mutants via immunoblot. CupA expression was elevated in SCV20265 relative to its clonal wild type while the *cupA* gene cluster transposon mutants displayed no CupA expression. CupA expression was also diminished in the other transposon mutants that displayed reversion to the wild type phenotype (Meissner, 2007).

Reversion from the SCV 20265 auto-aggregative phenotype to a wild type phenotype also resulted from insertions in PA1120 and *morA*, both of which possess a transmembrane domain and a C-terminal GGDEF domain. Previously, PA1120 was demonstrated to possess DGC activity and to modulate biofilm formation (Kulasakara, 2005). In addition to a GGDEF domain, the motility regulator MorA also contains an EAL domain. Expression of plasmid-encoded PA1120 restored CupA expression in both the SCV 20265 PA1120 and SCV 20265 *morA* mutants, suggesting that c-di-GMP levels are important for CupA fimbriae expression. Further analyses demonstrated that c-di-GMP levels were elevated (2.7-fold) in the SCV 20265 compared with wild type. Taken together these results demonstrate that alteration of global c-di-GMP levels is sufficient to induce CupA expression. Additionally, the GGDEF domain of MorA, but not the EAL domain, is important for its enzymatic activity as PA1120 overexpression could complement the SCV 20265 *morA* mutant for CupA expression (Meissner, 2007).

Ueda and Woods identified TpbA in a transposon mutant library screen for mutants with enhanced biofilm formation (Ueda, 2009b). A *tpbA* mutant displayed increased attachment to polystyrene and a defect in swarming motility, but no alteration in rhamnolipid production (Ueda, 2009a). Similar to a *wspF* mutant, the *tpbA* mutant displayed a wrinkly colony phenotype on Congo-red plates, implying increased production of EPS. Transcriptome analysis of a *tpbA* mutant revealed enhanced expression of the *pel/ACDF* genes and the biofilm matrix adhesin CdrA. The expression of *tpbA* was also upregulated, suggesting auto-regulation (Ueda, 2009a).

In independent transposon mutagenesis screens, Ueda and Wood (2009) as well as Malone and colleagues (2010), identified components of the *yfiBNR* operon as important for biofilm formation and SCV morphology. Ueda and Woods initially identified TpbB (YfiN) due to its ability

to suppress the hyperbiofilm phenotype of the *tpbA* mutant. Mutations in *yfiR* also suppressed the autoaggregative phenotype of the *tpbA* mutant (Ueda, 2009a). TpbA is a periplasmic protein with dual phosphatase activity, capable of dephosphorylating both tyrosine and serine/threonine residues (Pu, 2010). TpbB is an integral membrane protein containing a GGDEF domain, previously shown to possess DGC activity (Kulasakara, 2006). In follow-up studies, Pu and Wood demonstrated that TpbB is phosphorylated in a *tpbA* mutant, at both tyrosine and serine/threonine residues (Pu, 2010). Upon incubation of purified TpbA with phosphorylated TpbB, phosphorylation of TpbB is diminished, demonstrating that TpbB is a substrate of TpbA. Further, treatment of biofilm growth cultures with the phosphatase inhibitor trisodium orthovanadate increased biofilm formation, indicating that cellular phosphorylation enhances biofilm formation in *P. aeruginosa* (Ueda, 2009a).

Three independent insertions were mapped to *yfiR*, the first gene of a predicted three-gene operon *yfiBNR* (Malone, 2010). YfiR is a periplasmic protein, previously identified as a suppressor of the *tpbA* mutant phenotype (Ueda, 2009a). YfiB is a predicted outer-membrane lipoprotein containing a conserved OmpA peptidoglycan-binding domain. As a strong link between c-di-GMP and the SCV phenotype exists, Malone and colleagues hypothesized that the SCV phenotype of the *yfiR* mutant was the result of upregulation of TpbB activity due to release of YfiR-mediated repression. The cellular concentration of c-di-GMP measured in the *yfiR* mutant displayed a significant increase in c-di-GMP when compare to wild type; this observation coupled with the results demonstrating DGC activity of purified TpbB (lacking the transmembrane domain) support the hypothesis that YfiR represses the DGC activity of TpbB (Malone, 2010). Epistasis experiments were performed to determine the regulatory interplay of YfiB, YfiR, and TpbB. Deletion of *yfiB* did not affect attachment or colony morphology while the *yfiRB* double mutant displayed the phenotypes of a *yfiR* single mutant (SCV phenotypes), suggesting that YfiB functions upstream of YfiR. Additional experiments demonstrated that the stoichiometry of YfiR and TpbB are important for tight control of the system, as increased levels of TpbB relative to YfiR resulted in the SCV morphology (Malone, 2010). *In vitro* crosslinking experiments and bacterial two-hybrid analysis attempted to determine the link between YfiB and YfiR function. The crosslinking experiments suggested that oligomerization of YfiR depends upon the presence of YfiB, but not TpbB. The observed YfiR multimer was present at a size consistent with formation of a YfiR homodimer, yet the role of this complex was not defined. Additionally, bacterial two-hybrid analysis failed to demonstrate direct interaction between YfiR and TpbB, or between YfiB and either YfiR or TpbB.

Taken together the results of the studies performed by Wood and colleagues and Malone et al., demonstrate that the activity of the DGC TpbB is tightly controlled. Strong evidence demonstrates that TpbB is phosphorylated on both tyrosine and serine/threonine residues, and this phosphorylation is important for TpbB activity. However, it remains unclear which kinase is responsible for TpbB phosphorylation. Furthermore while it is evident that YfiR and TpbA both control TpbB function through distinct mechanisms, it remains to be determined if these involve discrete signaling pathways. Further biochemical studies

are necessary in order to deduce the mechanism by which YfiB intergrates with YfiR and TpbB.

### SadB/C/BifA pathway

Studies performed by George O'Toole's group have identified several factors that are involved in inversely regulating surface-associated behaviors (Caiazza, 2004, 2007; Kuchma and O'Toole, 2007; Merritt, 2007). Previously, a genetic screen identified surface attachment-defective (sad) mutants of *P. aeruginosa* PA14, defined by their inability to form biofilms in a microtiter dish biofilm assay (O'Toole, 1998, 2000). Subsequent characterization of mutants identified by this study revealed a genetic pathway involved in coordinately regulating surface-associated behaviors in *P. aeruginosa* PA14. The *sadB* mutant was originally identified as biofilm-defective in a genetic screen (O'Toole, 1998, 2000). Further investigation demonstrated that the *sadB* mutant did not display a defect in flagellum-mediated swimming motility or Tfp-dependent twitching motility (Caiazza, 2004). However, the *sadB* mutant displayed a significant increase in swarming motility when compared to wild type. Caiazza et al. further demonstrated that the *sadB* mutant could initiate surface attachment but was unable to transition from reversible to irreversible attachment, an early step in biofilm formation (Caiazza, 2004). Levels of SadB were positively correlated with biofilm formation and irreversible attachment and fractionation studies localized SadB to the cytoplasm.

In subsequent studies, an additional mutant *sadC*, identified in the original genetic screen (O'Toole, 1998), displayed surface-associated phenotypes mimicking the *sadB* mutant, including hyperswarming and a defect in biofilm formation. SadC is a DGC localized to the inner membrane that positively affects global c-di-GMP levels (Merritt, 2007). The *sadC* mutant was deficient in production of EPS, however *pelA/pelG* transcripts were unaffected by *sadC* mutation. Additional work identified yet another component of the pathway involved in regulating *P. aeruginosa* surface-associated behaviors, the phosphodiesterase BifA. A *bifA* mutant displays opposing phenotypes to the *sadB/C* mutants, a *bifA* mutant is deficient in swarming motility and displays increased biofilm formation (Kuchma and O'Toole, 2007). Although, BifA contains both a GGDEF and an EAL domain, BifA does not appear to possess DGC activity. Instead, the GGDEF residues likely bind the allosteric activator GTP, as PDE activity was diminished in GGDEF residue mutants (Kuchma and O'Toole, 2007). Epistasis studies have aided in elucidating the hierarchy of the aforementioned proteins and indicate that BifA and SadC function in the same genetic pathway (Merritt, 2007). Mutating *sadC* in the *bifA* background resulted in significant reduction of the hyperbiofilm phenotype and partial restoration of swarming motility. However, neither of these phenotypes were completely restored to wild type levels, suggesting that other DGCs likely contribute to overall c-di-GMP levels (Merritt, 2007). In a *sadB* mutant background, expression of *sadC* on a multicopy plasmid results in diminished biofilm formation in comparison to wild type, indicating that *sadB* is likely downstream of *sadC* in the pathway. These results support a model where environmental cues are sensed/received by SadC, resulting in altered c-di-GMP production. BifA can further regulate the signal through the activity of its EAL domain.

Lastly, SadB is predicted to be involved in the transmission of the signal to Pel machinery and components of the chemotaxis machinery in order to regulate biofilm formation and swarming motility (Caiazza, 2004, 2007; Merritt, 2007). These findings suggest that the nearly 40 *P. aeruginosa* DGC/PDE enzymes generate specific phenotypic outputs by altering subcellular c-di-GMP pools. Indeed, recent work by Merritt et al. (2010) demonstrated that changes in total c-di-GMP levels did not account for the surface-associated behaviors displayed by DGC mutants (Merritt, 2010).

### Cross-talk between signaling pathways

Our knowledge of the signaling pathways controlling *P. aeruginosa* is often based on dramatic phenotypic changes resulting from loss-of-function mutations that either fully activate or eliminate signaling under well-controlled laboratory conditions. However, in the environment or host, we hypothesize that a more gradual transition occurs between lifestyles. This concept is supported by the fact that the regulatory pathways discussed above are not insulated from one another; rather there exists substantial cross-talk between the systems indicating a more sophisticated signaling network and the potential for a continuum of phenotypes (Figure 1). To emphasize this point, we will highlight several mechanisms of cross-talk that are relevant to this review.

One of the best-described links between the lifestyle-determining regulatory pathways is the role of the cAMP/Vfr signaling system in controlling the QS response. Specifically, Vfr has been shown to control the QS regulatory cascade by directly activating transcription of *lasR* and *rhlR* (Albus, 1997; Croda-García, 2011). Furthermore, it has been shown that under planktonic growth conditions, Vfr protein levels increase substantially during the transition to stationary phase (Heurlier, 2003), a mechanism that may serve to fine-tune the timing of QS gene expression. While these findings demonstrate a regulatory link between the cAMP/Vfr and QS signaling pathways, recent studies indicate that this regulatory connection is more complicated than originally appreciated. In contrast to other Vfr-regulated targets, the control of *lasR* expression is cAMP-independent (Fuchs, 2010a). As stated previously, *vfr* expression is auto-regulated in a cAMP-dependent manner (Fuchs, 2010a). The existence of cAMP-independent regulation by Vfr implies that either basal Vfr levels are sufficient to drive *lasR* expression in the absence of cAMP, or that there are cAMP-independent mechanisms that can control *vfr* expression. Consistent with the later, it has been shown that activation of the alginate regulatory pathway results in reduced *vfr* expression without altering intracellular cAMP levels (Jones, 2010).

As mentioned above, the cAMP/Vfr pathway is linked to the regulatory pathway for conversion to mucoid in chronic *P. aeruginosa* CF infection. During the chronic phase of CF lung infection, *P. aeruginosa* frequently converts to the mucoid colony phenotype due to overproduction of the capsular polysaccharide alginate (Govan, 1996). The overproduction of alginate appears to be representative of the chronic infection phenotype, as mucoid conversion promotes persistence in the lungs and enhances resistance to immune defenses (Deretic, 1995; Pier, 2001). Mucoid conversion results, most frequently, from loss-of-function mutations in *mucA*, which encodes an anti-sigma factor (Boucher, 1997).

Inactivation of MucA releases the alternative sigma factor AlgU (AlgT) allowing AlgU to activate the transcription of numerous genes, including the gene encoding the AlgR response regulator (Figure 1E) (Wozniak, 1994; Govan, 1996). Both AlgU and AlgR are necessary for activation of the genes encoding the biosynthetic enzymes required for alginate production, resulting in the mucoid phenotype (Mohr, 1992; Martin, 1994). Loss-of-function mutations in *mucA* are also associated with reduced expression of many acute virulence factors, including: the T3SS, ETA, LasA protease, and Tfp (Figure 1E) (Mohr, 1990; Wu, 2004; Jones, 2010). Recent work by our group demonstrated that *mucA* mutation blocks the production of invasive virulence factors by inhibiting the cAMP/Vfr signaling pathway at the level of *vfr* expression (Jones, 2010). This inhibition is mediated by AlgU and AlgR, but is independent of alginate production and cAMP synthesis. Inhibition of *vfr* expression via *mucA* mutation represents a mechanism for inverse regulation of acute and chronic virulence factors during infection, supporting the hypothesis that acute virulence factors are actively repressed during chronic *P. aeruginosa* infection in the CF lung. The mechanism of *vfr* repression by AlgR has yet to be elucidated (Jones, 2010). In addition to indirectly influencing QS through reduced *vfr* expression, AlgR has been shown to negatively regulate expression of *rhlI* and *rhlR* directly (Morici, 2007). EMSA studies have demonstrated that AlgR binds specifically to the *rhlR-rhlI* promoter, further strengthening the link between chronic human infection and the repression of QS.

The c-di-GMP signaling pathway, via regulation of Alg44, also influences alginate production. It is now documented that increasing c-di-GMP levels by overexpression of DGCs enhances alginate production, and conversely PDE overexpression reduces alginate formation (Merighi, 2007). Alg44, a protein required for alginate biosynthesis, contains a PilZ domain, previously shown to interact with c-di-GMP (Amikam, 2006). Merighi and colleagues (2007) characterized the PilZ domain of Alg44 and demonstrated that mutation of residues within the PilZ domain diminished binding of c-di-GMP and alginate production. Localization studies determined that Alg44 associates with the membrane and protein localization is not affected by mutations in the PilZ domain. These results are consistent with Alg44 functioning as a regulator of the alginate polymerase (Alg8), a transmembrane protein (Merighi, 2007). The precise role of c-di-GMP binding on Alg44 function has yet to be elucidated.

Although early reports correlated elevated c-di-GMP levels with increased expression of polysaccharide synthesis genes, the mechanism facilitating this effect was unknown. FleQ was previously shown to repress expression of the *pel* operon (Dasgupta, 2002). Despite the absence of characterized c-di-GMP binding domains, FleQ bound radiolabeled c-di-GMP *in vitro*, and c-di-GMP binding inhibited FleQ-binding to the *pel* promoter transcript (Hickman, 2008). Because FleQ also activates genes involved in flagellar (acute virulence factor) biosynthesis, this regulation represents another mechanism for inverse control of acute (flagella) and chronic (Pel) virulence factors. Initial observations of Vfr-mediated repression of *fleQ* expression were perplexing, as Vfr is known to positively regulate virulence factors involved in acute infection. However, FleQ repression of chronic phenotype-associated

factors suggests that Vfr is also involved in downregulating acute virulence factors (flagella) and unexpectedly indirectly mediates expression of chronic virulence factors (Pel polysaccharide) through repression of *fleQ*.

Both mutations in *retS*, or high intracellular c-di-GMP levels, can induce biofilm formation suggesting that the Gac/Rsm pathway and c-di-GMP may intersect to control the expression of chronic virulence factors. Recently, work by Moscoso and colleagues has further established the link between the Gac/Rsm pathway and c-di-GMP by demonstrating that *retS* mutants themselves display high levels of c-di-GMP (Moscoso, 2011). RetS inversely controls the expression of the genes encoding the T3SS and the T6SS, which are associated with chronic infections (Goodman, 2004; Mougous, 2006; Brencic, 2009a). WspR DGC activity is finely controlled, however, mutation of a single residue within the I-site (WspR<sup>R242A</sup>) results in a constitutively active enzyme. Overexpression of WspR<sup>R242A</sup> enhanced biofilm formation and elevated EPS production. Additionally, the influence of c-di-GMP levels on the T3SS/T6SS switch was investigated by expressing WspR<sup>R242A</sup> in a *pelA* background. The roughly twofold increase in c-di-GMP levels observed in this strain coincided with a reduction in PcrV (T3SS) production and an increase in Hcp-1 (T6SS) production. Overexpression of another DGC, TpbB (YfiN) similarly resulted in a switch between the expression of the T3SS and the T6SS. Therefore, artificial increases in c-di-GMP, via DGC overexpression, inversely controls expression of the T3SS and the T6SS. Consistent with this result, overexpression of the PDE PA2133 in a *retS* mutant reverted the T3SS and T6SS expression profiles. In the absence of RsmY and RsmZ, DGC overexpression is no longer able to induce a switch from T3SS to T6SS expression indicating that RsmY and RsmZ are required for the c-di-GMP-dependent switch between T3SS and T6SS (Moscoso, 2011). These findings establish a direct link between the Gac/Rsm and c-di-GMP dependent pathways.

One final cross-talk mechanism is Gac/Rsm pathway control of cAMP/Vfr signaling through modulation of *vfr* expression. Microarray analysis revealed a twofold decrease in *vfr* expression in a *rsmA* mutant compared to PAO1 wild type. Additionally, analysis of *vfr* transcriptional fusions demonstrated decreased *vfr* expression in the *rsmA* mutant compared to PAO1 (Burrowes, 2006). Consistent with these findings, many of the acute virulence factors activated by RetS depend upon Vfr for expression (Goodman, 2004). In general, a consistent theme has emerged, whereby the Gac/Rsm pathway appears to promote expression of acute virulence factors though a Vfr-dependent mechanism, but represses chronic phenotypes through a Vfr-independent mechanism.

Cross-talk amongst the regulatory signaling pathways described here most often occurs at the transcriptional level (i.e. Vfr control of QS and AlgR control of QS), where different pathways regulate overlapping sets of genes. However, cross-talk can also occur via post-transcriptional regulation (i.e. c-di-GMP binding: Alg44 and FleQ). Collectively, the studies reviewed here suggest that *P. aeruginosa* signaling pathways are arranged in complex circuits with extensive cross-talk at multiple levels, thereby providing cells the ability to generate a finely tuned and progressive response to a range of stimuli.

The aforementioned pathways play a critical role in

determining *P. aeruginosa* lifestyle by detecting various environmental conditions and controlling global phenotypic changes. In many cases, the environmental cues that influence these signaling pathways remain unknown. However, in some instances potential signals have been elucidated. For example, the cAMP/Vfr signaling pathway is responsive to low calcium levels and osmotic stress (Wolfgang, 2003 and Rietsch, 2006). It has been hypothesized that the Gac/Rsm pathway may respond to carbohydrate signals, as the hybrid sensors RetS and LadS contain carbohydrate-binding domains (Goodman, 2004 and Ventre, 2006). Recent work by O'Callaghan and colleagues (2011) demonstrated that oxygen availability, detected through the anaerobic response regulator Anr, modulates *rsmY/Z* expression and thus Gac/Rsm-dependent phenotypes. QS is controlled by both the cAMP/ Vfr and Gac/Rsm pathways and thus signals that trigger these pathways, ultimately influence QS gene expression. In addition, membrane perturbations and the stringent response have previously been shown to influence QS (van Delden, 2001; Baysse, 2005), indicating that nutrient starvation/availability plays a substantial role in determining *P. aeruginosa* social behavior. The signals influencing c-di-GMP levels remain largely unknown, although recent work demonstrates that the availability of specific amino acids influence c-di-GMP levels and surface-associated behaviors (Bernier, 2011). Furthermore, surface contact or attachment itself appears to represent an important cue for c-di-GMP production (Güvener, 2007). Given the plethora of enzymes involved in c-di-GMP synthesis and degradation, it is plausible that many additional environmental cues contribute to regulation of c-di-GMP levels. The physiological relevance of these signals, in the context of a complex environmental or host niche, has yet to be determined and represents an important and daunting future direction for the field. Current progress in this area is severely hampered by the lack of tractable model systems and the fact that many of the signals mentioned are likely to have pleiotropic effects.

defense mechanism against phagocytosis by predators. Additionally, in this planktonic phase *P. aeruginosa* expresses appendages which mediate motility, providing yet another mechanism for avoiding predation and facilitating dispersal. The cAMP/Vfr signaling cascade controls the QS pathway. Once a quorum is reached, the bacteria begin to exhibit population behaviors, such as the secretion of degradative enzymes and toxins, which in significant concentrations can liberate nutrients from environmental sources and provide defense against predators and competing bacterial species. Similarly, in acute human infection the virulence phenotype of *P. aeruginosa* is dominated by the expression of virulence factors that are cytotoxic and invasive (T3SS, T2SS, flagella, Tfp) which cause tissue damage, sepsis and contribute to bacterial dissemination. The cAMP/Vfr and QS pathways largely control expression of these virulence factors.

In times of limited nutrients or substantial predation, *P. aeruginosa* can adapt a sessile biofilm lifestyle as a defense mechanism for evading predators and restricting metabolic demands. Upon attachment to a surface, the pathways leading to a sessile lifestyle (c-di-GMP, Gac/ Rsm) become activated. Attachment is initially dependent upon the expression of cAMP/Vfr-regulated genes, such as those necessary for Tfp expression and QS. As the bacteria population transitions to the sessile lifestyle these pathways are ultimately repressed and those involved in EPS production and biofilm formation are activated. During persistent or chronic human infection, *P. aeruginosa* frequently transitions to a noncytotoxic and minimally invasive virulence phenotype, adopting a biofilm lifestyle, which protects against host immune system insults and provides resistance to antibacterial therapy.

The last aspect of this model is that at any given time individuals residing in the environmental sessile community can transition from this lifestyle to once again become motile to seek new nutrient sources and more favorable environmental conditions. Similarly, we envision that individual members of a biofilm community, within the context of infection, can leave the biofilm and reemerge as individual motile bacteria. This transition may facilitate spread to a new colonization site within the CF lung and the renewed expression of acute virulence factors may contribute to the frequent bouts of exacerbation experienced by CF patients with chronic *P. aeruginosa* infection. Additionally, in the case of chronic biofilm infection of catheters, this reemergence of motile and cytotoxic bacteria may explain the periodic development of septicemia. Analogous to the environmental lifestyle transition, we hypothesize that the conversion to a chronic virulence phenotype occurs via a continuum, where the bacterial population largely transitions from cAMP/Vfr and QS-dominated phenotypes to c-di-GMP and Gac/Rsm- dominated phenotypes.

While this review primarily focuses in the regulatory pathways used by *P. aeruginosa* to generate a phenotypic continuum in response to extracellular signals, it is important to note that genetic variability also contributes to dynamic phenotypes within *P. aeruginosa* populations. This is best exemplified in chronic *P. aeruginosa* infections. Current evidence supports the idea that *P. aeruginosa* infection in CF largely involves a clonal population; however, there is abundant evidence that this population undergoes diversification through the accumulation of mutations (Bragonzi, 2009). Early evidence suggested that

Concluding Remarks
While it is convenient to think of *P. aeruginosa* infection as a switch between acute and chronic phenotypes it has become clear that infection, like environmental lifestyle, is a finely-balanced continuum and involves substantial overlap and cross-talk between environmental detection systems. We envision a model (Figure 2) whereby the transition from acute to chronic virulence phenotype during infection closely parallels the gradual changes observed during the shift from a planktonic to sessile lifestyle within an environmental reservoir. Further, a similar transition from a sessile to planktonic lifestyle can occur within the environment enabling dissemination to new environmental niches. We hypothesize that this sessile to planktonic switch also occurs during chronic human disease, although the mechanism and implications are less clear.

In the planktonic environmental lifestyle of *P. aeruginosa*, individual or small groups of free-living bacteria have the capacity to evade or defend against bacterivorous protozoan and metazoan predators (Hilbi, 2007). To do so, *P. aeruginosa* relies on cAMP/Vfr signaling to facilitate expression of many traditional “virulence factors” like the T3SS, which in an environmental reservoir is an important

these mutations are in fact selected for (Nguyen, 2006; Smith, 2006). The majority of documented loss-of-function mutations accumulate in the alginate regulatory pathway (*mucA*), QS system (*lasR*) and cAMP signaling system (*vfr* and *cyaB*) as well as the downstream targets controlled by these systems (Smith, 2006; D'Argenio, 2007; Wilder, 2009; Yang, 2011). In general, these mutations reduce the capacity of individual isolates to cause acute and invasive infection (Bragonzi, 2009), supporting the notion that acute and persistent infections involve distinct and potentially mutually exclusive phenotypic strategies. However, the maintenance of wild type alleles within the overall *P. aeruginosa* population suggests the need to maintain a full phenotypic repertoire. In particular, the spread of chronic infection (e.g. during exacerbation) to otherwise naive sites within the CF lung may require the capacity to initiate an acute virulence phenotype.

In opportunistic pathogens, regulatory systems must promote changes in gene expression to facilitate transition from an environmental reservoir to the host. Temporal control and spatial localization may allow ubiquitous signals to elicit specific responses. The ability of *P. aeruginosa* to occupy diverse environmental habitats and cause a wide variety of infections depends on a complex series of global regulatory networks that integrate diverse extracellular signals to coordinate phenotypic change. Intense study over the past decade has generated a wealth of information about the complex lifestyle of *P. aeruginosa*; however, many questions remain to be answered.

### Acknowledgements

This review was supported by a grant from the National Institutes of Health (AI069116) to M.C.W. and also in part by a grant from the Howard Hughes Medical Institute's Med into Grad Initiative to K.A.C. We thank members of the Wolfgang laboratory for their constructive suggestions and critical review of this manuscript.

### References

Albus, A.M., Pesci, E.C., Runyen-Janecky, L.J., West, S.E., and Iglewski, B.H. (1997). Vfr controls quorum sensing in *Pseudomonas aeruginosa*. J. Bacteriol. 179, 3928-3935.

Amikam, D., and Galperin, M.Y. (2006). PilZ domain is part of the bacterial c-di-GMP binding protein. Bioinformatics 22, 3-6.

Baker, D.A., and Kelly, J.M. (2004). Structure, function, and evolution of microbial adenyl and guanyl cyclases. Mol. Microbiol. 52, 1229-1242.

Barker, S.A., Vasil, A.I., Filloux, A., Ball, G., Wilderman, P.J., and Vasil, M.L. (2004). A novel extracellular phospholipase C of *Pseudomonas aeruginosa* is required for phospholipid chemotaxis. Mol. Microbiol. 53, 1089-1098.

Baysse, C., Cullinane, M., Dénervaud, V., Burrowes, E., Dow, J.M., Morrissey, J.P., Tam, L., Trevors, J.T., and O'Gara, F. (2005). Modulation of quorum sensing in *Pseudomonas aeruginosa* through alteration of membrane properties. Microbiology 151, 2529-2542.

Beatson, S.A., Whitchurch, C.B., Sargent, J.L., Levesque, R.C., and Mattick, J.S. (2002). Differential regulation of twitching motility and elastase production by Vfr in *Pseudomonas aeruginosa*. J. Bacteriol. 184, 3605-3613.

Bernier, S.P., Ha, D.G., Khan, W., Merritt, J.H., and O'Toole, G.A. (2011). Modulation of *Pseudomonas aeruginosa* surface-associated group behaviors by individual amino acids through c-di-GMP signaling. Res. Microbiol. 162, 680-688.

Bordi, C., Lamy, M.C., Ventre, I., Termine, E., Hachani, A., Fillet, S., Roche, B., Bleves, S., Méjean, V., Lazdunski, A., and Filloux, A. (2010). Regulatory RNAs and the HptB/RetS signalling pathways fine-tune *Pseudomonas aeruginosa* pathogenesis. Mol. Microbiol. 76, 1427-1443.

Borlee, B.R., Goldman, A.D., Murakmi, K., Samudrala, R., Wozniak, D.J., and Parsek, M.R. (2010). *Pseudomonas aeruginosa* uses a cyclic-di-GMP-regulated adhesin to reinforce the biofilm extracellular matrix. Mol. Microbiol. 75, 827-842.

Botsford, J.L., and Harman, J.G. (1992). Cyclic AMP in prokaryotes. Microbiol. Rev. 56, 100-122.

Boucher, J.C., Yu, H., Mudd, M.H., and Deretic, V. (1997). Mucoid *Pseudomonas aeruginosa* in cystic fibrosis: characterization of *muc* mutations in clinical isolates and analysis of clearance in a mouse model of respiratory infection. Infect. Immun. 65, 3838-3846.

Bragonzi, A., Paroni, M., Nonis, A., Cramer, N., Montanari, S., Rejman, J., Di Serio, C., Döring, G., and Tümmler, B. (2009). *Pseudomonas aeruginosa* microevolution during cystic fibrosis lung infection establishes clones with adapted virulence. Am. J. Respir. Crit. Care. Med. 180, 138-145.

Brencic, A., and Lory, S. (2009a). Determination of the regulon and identification of novel mRNA targets of *Pseudomonas aeruginosa* RsmA. Mol. Microbiol. 72, 612-632.

Brencic, A., K.A. McFarland, H. McManus, R., S. Castang, I. Mogno, S. L. Dove, and S. Lory (2009b). The GacS/GacA signal transduction system of *Pseudomonas aeruginosa* acts exclusively through its control over the transcription of the *RsmY* and *RsmZ* regulatory small RNAs. Mol. Microbiol. 73, 434-445.

Burrowes, E., Baysse, C., Adams, C., and O'Gara, F. (2006). Influence of the regulatory protein RsmA on cellular functions in *Pseudomonas aeruginosa* PA01, as revealed by transcriptome analysis. Microbiology 152 (Pt 2), 405-418.

Caiazza, N.C., and O'Toole, G.A. (2004). SadB is required for the transition from reversible to irreversible attachment during biofilm formation by *Pseudomonas aeruginosa* PA14. J. Bacteriol. 186, 4476-4486.

Caiazza, N.C., Merritt, J.H., Brothers, K.M., and O'Toole, G.A. (2007). Inverse regulation of biofilm formation and swarming motility in *Pseudomonas aeruginosa* PA14. J. Bacteriol. 189, 3603-3612.

Castang, S., McManus, H.R., Turner, K.H., and Dove, S.L. (2008). H-NS family members function coordinately in an opportunistic pathogen. Proc. Natl. Acad. Sci. USA 105, 18947-18952.

Chen, G., Wang, C., Fuqua, C., Zhang, L.H., and Chen, L. (2004a). Quorum-sensing antiactivator TraM forms a dimer that dissociates to inhibit TraR. Mol. Microbiol. 52, 1641-1651.

Chen, Y.T., Chang, H.Y., Lu, C.L., and Peng, H.L. (2004b). Evolutionary Analysis of the two-component systems in *Pseudomonas aeruginosa* PAO1. J. Mol. Evol. 59, 725-737.

Coggan and Wolfgang

Christen, M., Christen, B., Folcher, M., Schauerte, A., and Jenal, U. (2005). Identification and characterization of a cyclic di-GMP-specific phosphodiesterase and its allosteric control by GTP. J. Biol. Chem. 280, 30829-30837.

Christensen, L.D., Moser, C., Jensen, P.O., Rasmussen, T.B., Christophersen, L., Kjelleberg, S., Kumar, N., Hoiby, N., Givskov, M., and Bjarnsholt, T. (2007). Impact of *Pseudomonas aeruginosa* quorum sensing on biofilm persistence in an *in vivo* intraperitoneal foreign-body infection model. Microbiology 153, 2312-2320.

Chugani, S.A., Whiteley, M., Lee, K.M., D'Argenio, D., Manoil, C., and Greenberg, E.P. (2001). QscR, a modulator of quorum-sensing signal synthesis and virulence in *Pseudomonas aeruginosa*. Proc. Natl. Acad. Sci. USA 98, 2752-2757.

Cossart, P., and Gicquel-Sanzey, B. (1982). Regulation of expression of the *crp* gene of *Escherichia coli* K12: *in vivo* study. J. Bacteriol. 161, 454-457.

Cowell, B.A., Evans, D.J., and Fleiszig, S.M. (2005). Actin cytoskeleton disruption by ExoY and its effects on *Pseudomonas aeruginosa* invasion. FEMS Microbiol. Lett. 250, 71-76.

Croda-García, G., Grosso-Becerra, V., Gonzalez-Valdez, A., Servín-González, L., and Soberón-Chávez, G. (2011). Transcriptional regulation of *Pseudomonas aeruginosa rhlR*: role of the CRP orthologue Vfr (virulence factor regulator) and quorum sensing regulators LasR and RhlR. Microbiology 157, 2545-2555.

D'Argenio, D.A., Calfee, M.W., Rainey, P.B., and Pesci, E.C. (2002). Autolysis and autoaggregation in *Pseudomonas aeruginosa* colony morphology mutants. J. Bacteriol. 184, 6481-6489.

D'Argenio, D.A., Gallagher, L.A., Berg, C.A., and Manoil, C. (2001). *Drosophila* as a model host for *Pseudomonas aeruginosa* infection. J. Bacteriol. 183, 1466-1471.

D'Argenio, D.A., Wu, M., Hoffman, L.R., Kulasekara, H.D., Déziel, E., Smith, E.E., Nguyen, H., Ernst, R.K., Larson Freeman, T.J., Brittnacher, M., Hayden, H.S., Selgrade, S., Klausen, M., Goodlett, D.R., Burns, J.L., Ramsey, B.W., and Miller, S.I. (2007). Growth phenotypes of *Pseudomonas aeruginosa lasR* mutants adapted to the airways of cystic fibrosis patients. Mol. Micobiol. 64, 512-533.

Darzins, A., and Russel, M.A. (1997). Molecular genetic analysis of type-4 pilus biogenesis and twitching motility using *Pseudomonas aeruginosa* as a model system - a review. Gene 192, 109-115.

Dasgupta, N., Ferrell, E.P., Kanack, K.J., West, S.E., and Ramphal, R. (2002). *fleQ*, the gene encoding the major flagellar regulator of *Pseudomonas aeruginosa*, is σ⁷⁰ dependent and is downregulated by Vfr, a homolog of *Escherichia coli* cyclic AMP receptor protein. J. Bacteriol. 184, 5240-5250.

de Kieviet, T., Seed, P.C., Nezezon, J., Passador, L., and Iglewski, B.H. (1999). RsaL, a novel repressor of virulence gene expression in *Pseudomonas aeruginosa*. J. Bacteriol. 181, 2175-2184.

De Lorenzo, V., Herrero, M., Giovanni, F., and Neilands, J.B. (1988). Fur (ferric uptake regulation) protein and CAP (catabolite-activator protein) modulate transcription of fur gene in *Escherichia coli*. Eur. J. Biochem. 173, 537-546.

Delange, P.A., Collins, T.L., Pierce, G.E., and Robinson,

J.B. (2007). PilJ localizes to cell poles and is required for Type IV pilus extension in *Pseudomonas aeruginosa*. Curr. Microbiol. 55, 389-395.

Deretic, V., Schurr, M.J., and Yu, H. (1995). *Pseudomonas aeruginosa*, mucoidy and the chronic infection phenotype in cystic fibrosis. Trends Microbiol. 3, 351-356.

Deziel, E., Gopalan, S., Tampakaki, A.P., Lepine, F., Padfield, K.E., and Saucier, M. (2005). The contribution of MvfR to *Pseudomonas aeruginosa* pathogenesis and quorum sensing circuitry regulation: multiple quorum sensing-regulated genes are modulated without affecting *lasRI*, *rhlRI* or the production of N-acyl-L-homoserine lactones. Mol. Microbiol. 55, 998-1014.

Diggle, S.P., Winzer, K., Lazdunksi, A., Williams, P., and Camara, M. (2002). Advancing the quorum in *Pseudomonas aeruginosa*: MvaT and the regulation of N-acylhomoserine lactone production and virulence gene expression. J. Bacteriol. 184, 2576-2586.

Diggle, S.P., Winzer, K., Lazdunksi, A., Williams, P., and Camara, M. (2003). The *Pseudomonas aeruginosa* quinolone signal molecule overcomes the cell density-dependency of the quorum sensing hierarchy, regulates *rhl*-dependent genes at the onset of stationary phase and can be produced in the absence of LasR. Mol. Microbiol. 50, 29-43.

Dong, Y.H., Zhang, X.F., Xu, J.L., Tan, A.T., and Zhang, L.H. (2005). VqsM, a novel AraC-type global regulator of quorum-sensing signalling and virulence in *Pseudomonas aeruginosa*. Mol. Microbiol. 58, 552-564.

Donlan, R.M., and Costerton, J.W. (2002). Biofilms: survival mechanisms of clinically relevant microorganisms. Clin. Microbiol. Rev. 15, 167-193.

Drenkard, E., and Ausubel, F.M. (2002). *Pseudomonas* biofilm formation and antibiotic resistance are linked to phenotypic variation. Nature 416, 740-743.

Driscoll, J.A., Brody, S.L., and Kollef, M.H. (2007). The epidemiology, pathogenesis, and treatment of *Pseudomonas aeruginosa* infections. Drugs 67, 351-368.

Endoh, T., and Engel, J.N. (2009). CbpA: a polarly localized novel cyclic AMP-binding protein in *Pseudomonas aeruginosa*. J. Bacteriol. 191, 7193-7205.

Ferguson, M.W., Maxwell, J.A., Vincent, T.S., da Silva, J., and Olson, J.C. (2001). Comparison of the *exoS* gene and protein expression in soil and clinical isolates of *Pseudomonas aeruginosa*. Infect. Immun. 69, 2198-2210.

Ferrel, E., Carty, N.L., Colmer-Hamood, J.A., Hamood, A.N., and West, S.E. (2008). Regulation of *Pseudomonas aeruginosa ptxR* by Vfr. Microbiology 154, 431-439.

Fuchs, E.L., Brutinel, E.D., Jones, A.K., Fulcher, N.B., Urbanowski, M.L., Yahr, T.L., and Wolfgang, M.C. (2010a). The *Pseudomonas aeruginosa* Vfr regulator controls global virulence factor expression through cyclic AMP-dependent and -independent mechanisms. J. Bacteriol. 192, 3553-3564.

Fuchs, E.L., Brutinel, E.D., Klem, E.R., Fehr, A.R., Yahr, T.L., and Wolfgang, M.C. (2010b). *In vitro* and *in vivo* characterization of the *Pseudomonas aeruginosa* cyclic AMP (cAMP) phosphodiesterase CpdA, required for cAMP homeostasis and virulence factor regulation. J. Bacteriol. 192, 2779-2790.

Fulcher, N.B., Holliday, P.M., Klem, E.R., Cann, M.J., and Wolfgang, M.C. (2010). The *Pseudomonas aeruginosa* Chp chemosensory system regulates intracellular cAMP

levels by modulating adenylate cyclase activity. Mol. Microbiol. 76, 889-904.

Gambello, M.J., and Iglewski, B.H. (1991). Cloning and characterization of the *Pseudomonas aeruginosa* lasR gene, a transcriptional activator of elastase expression. J. Bacteriol. 173, 3000-3009.

Goodman, A.L., Kulasekara, B., Rietsch, A., Boyd, D., Smith, R.S., and Lory, S. (2004). A signaling network reciprocally regulates genes associated with acute infection and chronic persistence in *Pseudomonas aeruginosa*. Dev. Cell 7, 1-20.

Goodman, A.L., Merighi, M., Hyodo, M., Ventre, I., Fillox, A., and Lory, S. (2009). Direct interaction between sensor kinase proteins mediates acute and chronic disease phenotypes in a bacterial pathogen. Genes Dev. 23, 249-259.

Gosset, G., Zhang, Z., Nayyar, S., Cuevas, W.A., and Saier, Jr. M.H. (2004). Transcriptome analysis of Crp-dependent catabolite control of gene expression in *Escherichia coli*. J. Bacteriol. 186, 3515-3524.

Govan, J.R., and Deretic, V. (1996). Microbial pathogenesis in cystic fibrosis: mucoid *Pseudomonas aeruginosa* and *Burkholderia cepacia*. Microbiol. Rev. 60, 539-574.

Green, S.K., Schroth, M.N., Cho, J.J., Kominos, S.D., and Vitanza-Jack, V.B. (1974). Agricultural plants and soil as a reservoir for *Pseudomonas aeruginosa*. Appl. Environ. Microbiol. 28, 987-991.

Güvener, Z.T., and Harwood, C.S. (2007). Subcellular location characteristics of the *Pseudomonas aeruginosa* GGDEF protein, WspR, indicate that it produces cyclic-di-GMP in response to growth on surfaces. Mol. Microbiol. 66, 1459-1473.

Hanamura, A., and Aiba, H. (1992). A new aspect of transcriptional control of the *Escherichia coli crp* gene: positive autoregulation. Mol. Microbiol. 6, 2489-2497.

Häussler, S. (2004). Biofilm formation by the small colony variant of *Pseudomonas aeruginosa*. Environ. Microbiol. 6, 546-551.

He, J., Baldini, R.L., Déziel, E., Saucier, M., Zhang, Q., Liberati, N.T., Lee, D., Urbach, J., Goodman, H.M., Rahme, L.G. (2004). The broad host range pathogen *Pseudomonas aeruginosa* strain PA14 carries two pathogenicity islands harboring plant and animal virulence genes. Proc. Natl. Acad. Sci. USA 101, 2530-2535.

Heurlier, K., Denervaud, V., Pessi, G., Reimmann, C., and Haas, D. (2003). Negative control of quorum sensing by RpoN (sigma54) in *Pseudomonas aeruginosa* PAO1. J. Bacteriol. 185, 2227-2235.

Heurlier, K., Williams, F., Heeb, S., Dormond, C., Pessi, G., Singer, D., Cámara, M., Williams, P., and Haas, D. (2004). Positive control of swarming, rhamnolipid synthesis, and lipase production by the posttranscriptional RsmA/RsmZ system in *Pseudomonas aeruginosa*. J. Bacteriol. 186, 2936-2945.

Hickman, J.W., and Harwood, C.S. (2008). Identification of FleQ from *Pseudomonas aeruginosa* as a c-di-GMP-responsive transcription factor. Mol. Microbiol. 69, 376-389.

Hickman, J.W., Tifrea, D.F., and Harwood, C.S. (2005). A chemosensory system that regulates biofilm formation through modulation of cyclic diguanylate levels. Proc. Natl. Acad. Sci. USA 102, 14422-14427.

Hilbi, H., Weber, S.S., Ragaz, C., Nyfeler, Y., and Urwyler, S.


P. aeruginosa Environmental Lifestyle and Virulence 65

(2007). Environmental predators as models for bacterial pathogenesis. Environ. Microbiol. 9, 563-575.

Hsu, J.L., Chen, H.C., Peng, H.L., and Chang, H.Y. (2008). Characterization of the histidine-containing phosphotransfer protein B-mediated multistep phosphorelay system in *Pseudomonas aeruginosa* PAO1. J. Biol. Chem. 283, 9933-9944.

Huang, B., Whitchurch, C.B., and Mattick, J.S. (2003). FimX, a multidomain protein connecting environmental signals to twitching motility in *Pseudomonas aeruginosa*. J. Bacteriol. 185, 7068-7076.

Inclán, Y.F., Huseby, M.J., and Engel, J.N. (2011). FimL regulates cAMP synthesis in *Pseudomonas aeruginosa*. PloS One 6, e15867.

Jonas, K., Melefors, O., and Romling, U. (2009). Regulation of c-di-GMP metabolism in biofilms. Future Microbiol. 4, 341-358.

Jones, A.K., Fulcher, N.B., Balzer, G.J., Urbanowski, M.L., Pritchett, C.L., Schurr, M.J., Yahr, T.L., and Wolfgang, M.C. (2010). Activation of the *Pseudomonas aeruginosa* AlgU regulon through *mucA* mutation inhibits cyclic AMP/Vfr signaling. J. Bacteriol. 192, 5709-5717.

Juhas, M., Eberl, L., and Tummler, B. (2005). Quorum sensing: the power of cooperation in the world of *Pseudomonas*. Environ. Microbiol. 7, 459-471.

Juhas, M., Wiehlmann, L., Huber, B., Jordan, D., Lauber, J., Salunkhe, P., Limpert, A.S., von Götz, F., Steinmetz, I., Eberl, L., and Tummler, B. (2004). Global regulation of quorum sensing and virulence by VqsR in *Pseudomonas aeruginosa*. Microbiology 150, 831-841.

Kanack, K.J., Runyen-Janecky, L.J., Ferrell, S.J., and West, S.E. (2006). Characterization of a DNA-binding specificity and analysis of binding sites of the *Pseudomonas aeruginosa* global regulator, Vfr, a homologue of the *Escherichia coli* cAMP-receptor protein. Microbiology 152, 3485-3496.

Kay, E., Humair, B., Dénervaud, V., Riedel, K., Spahr, S., Eberl, L., Valverde, C., and Haas, D. (2006). Two GacA-dependent small RNAs modulate the quorum-sensing response in *Pseudomonas aeruginosa*. J. Bacteriol. 188, 6026-6033.

Kazmierczak, B.I., Lebron, M.B., and Murray, T.S. (2006). Analysis of FimX, a phosphodiesterase that governs twitching motility in *Pseudomonas aeruginosa*. Mol. Microbiol. 60, 1026-1043.

Kearns, D.B., Robinson, J., and Shimkets, L.J. (2001). *Pseudomonas aeruginosa* exhibits directed twitching motility up phosphatidylethanolamine gradients. J. Bacteriol. 183, 1736-1751.

Kiratisin, P., Tucker, K.D., and Passador, L. (2002). LasR, a transcriptional activator of *Pseudomonas aeruginosa* virulence genes, functions as a multimer. J. Bacteriol. 184, 4912-4919.

Kuchma, S.L., Brothers, K.M., Merritt, J.H., Liberati, N.T., Ausubel, F.M., , and O'Toole, G.A. (2007). BifA, a c-di-GMP phosphodiesterase, inversely regulates biofilm formation and swarming motility by *Pseudomonas aeruginosa* PA14. J. Bacteriol. 189, 8165-8178.

Kuchma, S.L., Connolly, J.P., and O'Toole, G.A. (2005). A three-component regulatory system regulates biofilm maturation and type III secretion in *Pseudomonas aeruginosa*. J. Bacteriol. 187, 1441-1454.

Kulasakara, H., Lee, V., Brencic, A., Liberati, N., Urbach, J.,

Coggan and Wolfgang

Miyata, S., Lee, D.G., Neely, N., Hyodo, M., Hayakawa, Y., Ausubel, F.M., and Lory, S. (2006). Analysis of *Pseudomonas aeruginosa* diguanylate cyclases and phosphodiesterases reveals a role for bis-(3'-5')-cyclic-GMP in virulence. Proc. Natl. Acad. Sci. USA 103, 2839-2844.

Kulasakara, H., Ventre, I., Kulasakara, B.R., Lazdunski, A., Fillous, A., and Lory, S. (2005). A novel two-component system controls the expression of *Pseudomonas aeruginosa* fimbrial *cup* genes. Mol. Microbiol. 55, 368-380.

Lam, J., Chan, R., Lam, K., and Costerton, J.W. (1980). Production of mucoid microcolonies by *Pseudomonas aeruginosa* within infected lungs in cystic fibrosis. Infect. Immun. 28, 546-556.

Lapouge, K., Schubert, M., Allain, F.H., and Haas, D. (2008). Gac/Rsm signal transduction pathway of -proteobacteria: from RNA recognition to regulation of social behavior. Mol. Micobiol. 67, 241-253.

Laskowski, M., and Kazmierczak, B.I. (2006). Mutational Analysis of RetS, an unusual sensor kinase-response regulator hybrid required for *Pseudomonas aeruginosa* virulence. Infect. Immun. 74, 4462-4478.

Latifi, A., Foglino, M., Tanaka, K., Williams, P., and Lazdunski, A. (1996). A hierarchical quorum-sensing cascade in *Pseudomonas aeruginosa* links the transcriptional activators LasR and RhlR (VsmR) to expression of the stationary-phase sigma factor RpoS. Mol. Micobiol. 21, 1137-1146.

Ledgham, F., Ventre, I., Soscia, C., Foglino, M., Sturgis, J.N., and Lazdunski, A. (2003). Interactions of the quorum sensing regulator QscR: interaction with itself and the other regulators of *Pseudomonas aeruginosa* LasR and RhlR. Mol. Micobiol. 48, 199-210.

Lee, E.J., Glasgow, J., Leu, S., Belduz, A.O., and Harman, J.G. (1994). Mutagenesis of the cyclic AMP receptor protein of *Escherichia coli*: targeting positions 83, 127, and 128 of the cyclic nucleotide binding pocket. Nucleic Acids Res. 22, 2894-2901.

Lequette, Y., Lee, J.H., Ledgham, F., Ladunski, A., and Greenberg, E.P. (2006). A distinct QscR regulon in the *Pseudomonas aeruginosa* quorum-sensing circuit. J. Bacteriol. 188, 3365-3370.

Li, L.L., Malone, J.E., and Iglewski, B.H. (2007). Regulation of the *Pseudomonas aeruginosa* quorum-sensing regulator VqsR. J. Bacteriol. 189, 4367-4374.

Liberati, N.T., Urbach, J.M., Miyata, S., Lee, D.G., Drenkard, E., Wu, G., Villanueva, J., Wei, T., and Ausubel, F.M. (2006). An ordered nonredundant library of *Pseudomonas aeruginosa* strain PA14 transposon insertion mutants. Proc. Natl. Acad. Sci. USA 103, 2833-2838.

Lin, C.T., Huang, Y.J., Chu, P.H., Hsu, J.L., Huang, C.H., and Peng, H.L. (2006). Identification of an HptB-mediated multi-step phosphorelay in *Pseudomonas aeruginosa* PAO1. Res. Microbiol. 157, 169-175.

Macfadyen, L.P., Ma, C., and Redfield, J. (1998). A 3',5' cyclic AMP (cAMP) phosphodiesterase modulates cAMP levels and optimizes competence in *Haemophilus influenzae*. J. Bacteriol. 180, 4401-4405.

Malone, J.G., Jaeger, T., Spangler, C., Ritz, D., Spang, A., Arrieumerlou, C., Kaever, V., Landmann, R., and Jenal, U. (2010). YfiBNR mediates cyclic di-GMP dependent small colony variant formation and persistence in *Pseudomonas*

aeruginosa*. PLoS Pathog. 6, e1000804.

Martin, D.W., Schurr, M.J., Yu, Y., and Deretic, V. (1994). Analysis of promoters controlled by the putative sigma factor AlgU regulating conversion to mucoidy in *Pseudomonas aeruginosa*: relationship between sigma E and stress response. J. Bacteriol. 176, 6688-6696.

Mattick, J.S. (2002). Type IV pili and twitching motility. Annu. Rev. Microbiol. 56, 289-314.

Matz, C., and Kjelleberg, S. (2005). Off the hook - how bacteria survive protozoan grazing. Trends Microbiol. 13, 302-307.

McGarth, S., Wade, D.S., and Pesci, E.C. (2004). Dueling quorum sensing systems in *Pseudomonas aeruginosa* control the production of the *Pseudomonas* quinolone signal (PQS). FEMS Microbiol Lett 230, 27-34.

McKnight, S.L., Iglewski, B.H., and Pesci, E.C. (2000). The *Pseudomonas* quinolone signal regulates *rhl* quorum sensing in *Pseudomonas aeruginosa*. J. Bacteriol. 182, 2702-2708.

Meissner, A., Wild, V., Simm, R., Rohde, M., Erck, C., Bredenbruch, F., Morr, M., Römling, U., and Häussler, S. (2007). *Pseudomonas aeruginosa cupA*-encoded fimbriae expression is regulated by a GGDEF and EAL domain-dependent modulation of the intracellular level of cyclic diguanylate. Environ. Microbiol. 9, 2475-2485.

Merighi, M., Lee, V.T., Hyodo, M., Hayakawa, Y., and Lory, S. (2007). The second messenger bis-(3'-5')-cyclic-GMP and its PilZ domain-containing receptor Alg44 are required for alginate biosynthesis in *Pseudomonas aeruginosa*. Mol. Micobiol. 54, 876-895.

Merritt, J.H., Brothers, K.M., Kuchma, S.L., and O'Toole, G.A. (2007). SadC reciprocally influences biofilm formation and swarming motility via modulation of exopolysaccharide production and flagellar function. J. Bacteriol. 189, 8154-8164.

Merritt, J.H., Ha, D.G., Cowles, K.N., Lu, W., Morales, D.K., Rabinowitz, J., Gitai, Z., and O'Toole, G.A. (2010). Specific control of *Pseudomonas aeruginosa* surface-associated behaviors by two c-di-GMP diguanylate cyclases. mBio 1, 1-10.

Mikkelsen, H., Ball, G., Giraud, C., and Filloux, A. (2009). Expression of *Pseudomonas aeruginosa cupD* fimbrial genes is antagonistically controlled by RcsB and the EAL-containing PvrR response regulators. PloS One 4, e6018.

Mohr, C.D., Hibler, N.S., and Deretic, V. (1992). AlgR-binding sites within the *algD* promoter make up a set of inverted repeats separated by a large intervening segment of DNA. J. Bacteriol. 174, 6624-6633.

Mohr, C.D., Rust, L., Albus, A.M., Iglewski, B.H., and Deretic, V. (1990). Expression patterns of genes encoding elastase and controlling mucoidy: co-ordinate regulation of two virulence factors in *Pseudomonas aeruginosa* isolates from cystic fibrosis. Mol. Micobiol. 4, 2103-2110.

Morici, L.A., Carterson, A.J., Wagner, V.E., Frisk, A., Schurr, J.R., Höner zu Bentrup, K., Hassett, D.J., Iglewski, B.H., Sauer, K., and Schurr, M.J. (2007). *Pseudomonas aeruginosa* AlgR represses the *Rhl* quorum-sensing system in a biofilm specific manner. J. Bacteriol. 189, 7752-7764.

Moscoso, J.A., Mikkelsen, H., Heeb, S., Williams, P., and Filloux, A. (2011). The *Pseudomonas aeruginosa* sensor RetS switches Type III and Type VI secretion via c-di-GMP signalling. Environ. Microbiol. 13, 3128-3138.

Mougous, J.D., Cuff, M.E., Raunser, S., Shen, A., Zhou, M., Gifford, C.A., and *et al.* (2006). A virulence locus of *Pseudomonas aeruginosa* encodes a protein secretion apparatus. Science 312, 1526-1530.

Navarro, M.V., De, N., Bae, N., Wang, Q., and Sondermann, H. (2010). Structural analysis of the GGDEF-EAL domain-containing c-di-GMP receptor FimX. Structure 17, 1104-1116.

Newell, P.D., Monds, R.D., and O'Toole, G.A. (2009). LapD is a bis(3'5')-cyclic dimeric GMP-binding protein that regulates surface attachment by *Pseudomonas fluorescens* Pf0-1. Proc. Natl. Acad. Sci. USA 106, 3461-3466.

Nguyen, D., and Singh, P.K. (2006). Evolving stealth: genetic adaptation of *Pseudomonas aeruginosa* during cystic fibrosis infections. Proc. Natl. Acad. Sci. USA 103, 8305-8306.

Nicas, T.I., and Iglewski, B.H. (1986). Production of elastase and other exoproducts by environmental isolates of *Pseudomonas aeruginosa*. J. Clin. Microbiol. 23, 967-969.

Nicastro, G.G., Boechat, A.L., Abe, C.M., Kaihami, G.H., and Baldini, R.L. (2009). *Pseudomonas aeruginosa* PA14 *cupD* transcription is activated by the RcsB response regulator, but repressed by its putative cognate sensor RcsC. FEMS Microbiol. Lett. 301, 115-123.

Nikolskaya, A.N., Mulkidjanian, A.Y., Beech, I.B., and Galperin, M.Y. (2003). MASE1 and MASE2: two novel integral membrane sensory domains. J. Mol. Microbiol. Biotechnol. 5, 11-16.

O'Toole, G.A., and Kolter, R. (1998). Flagellar and twitching motility are necessary for *Pseudomonas aeruginosa* biofilm development. Mol. Microbiol. 30, 295-304.

O'Toole, G.A., Gibbs, K.A., Hager, P.W., Phibbs, P.V., and Kolter, R. (2000). The global carbon metabolism regulator Crc is a component of a signal transduction pathway required for biofilm development by *Pseudomonas aeruginosa*. J. Bacteriol. 182, 425-431.

Pellet, S., Bigley, D.V., and Grimes, D.J. (1983). Distribution of *Pseudomonas aeruginosa* in a riverine ecosystem. Appl. Environ. Microbiol. 45, 328-332.

Pesci, E.C., Pearson, J.P., Seed, P.C., and Iglewski, B.H. (1997). Regulation of the *las* and *rhl* quorum-sensing systems in control of elastase and rhamnolipid biosynthesis genes. J. Bacteriol. 179, 3127-3132.

Pessi, G., Williams, G., Hindle, Z., Heurlier, K., Holden, M.T., Cámara, M., Hass, D., and Williams, P. (2001). The global posttranscriptional regulator RsmA modulates production of virulence determinants and N-Acylhomoserine lactones in *Pseudomonas aeruginosa*. J. Bacteriol. 183, 6676-6683.

Pier, G.B., Coleman, F., Grout, M., Franklin, M., and Ohman, D.E. (2001). Role of alginate O acetylation in resistance of mucoid *Pseudomonas aeruginosa* to opsonic phagocytosis. Infect. Immun. 69, 1895-1901.

Pirnay, J.P., Bilocq, F., Pot, B., Cornelis, P., Zizi, M., Van Eldere, J., Deschaght, P., Vaneechoutte, M., Jennes, S., Pitt, T., and De Vos, D. (2009). *Pseudomonas aeruginosa* population structure revisited. PLoS One 13, e7740.

Pu, M., and Wood, T.K. (2010). Tyrosine phosphatase TpbA controls rugose colony formation in *Pseudomonas aeruginosa* by dephosphorylating diguanylate cyclase TpbB. Biochem. Biophys. Res. Commun. 402, 351-355.

Qi, Y., Chuah, M.L., Dong, X., Xie, K., Luo, Z., Tang, K., and Liang, Z.X. (2011). Binding of cyclic diguanylate in the non-catalytic EAL domain of FimX induces a long-range conformational change. J. Biol. Chem. 286, 2910-2917.

Rahme, L., Stevens, E.J., Wolfort, S.F., Shao, J., and Tompkins, R.G. (1995). Common virulence factors for bacterial pathogenicity in plants and animals. Science 268, 1899-1902.

Rahme, L., Tan, M.W., Le, L., Wong, S.M., and Tompkins, R.G. (1997). Use of model plant hosts to identify *Pseudomonas aeruginosa* virulence factors. Proc. Natl. Acad. Sci. USA 94, 13245-13250.

Rampioni, G., Bertani, I., Zennaro, E., Polticelli, F., Venturi, V., and Leoni, L. (2006). The quorum-sensing negative regulator RsaL of *Pseudomonas aeruginosa* binds to the *lasI* promoter. J. Bacteriol. 188, 815-819.

Rampioni, G., Schuster, M., Greenburg, E.P., Bertani, I., Grasso, M., Venturi, V., Zennaro, E., and Leoni, L. (2007). RsaL provides quorum sensing homeostasis and functions as a global regulator of gene expression in *Pseudomonas aeruginosa*. Mol. Microbiol. 66, 1557-1565.

Rampioni, G., Schuster, M., Greenburg, E.P., Zennaro, E., and Leoni, L. (2009). Contribution of the RsaL global regulator to *Pseudomonas aeruginosa* virulence and biofilm formation. FEMS Microbiol. Lett. 301, 210-217.

Rao, F., Qi, Y., and Liang, Z.X. (2008). Catalytic mechanism of cyclic-di-GMP-specific phosphodiesterase: a study of the EAL domain-containing RocR from *Pseudomonas aeruginosa*. J. Bacteriol. 190, 3622-2631.

Ridgway, H.G., Silverman, M., and Simon, M.I. (1977). Localization of proteins controlling motility and chemotaxis in *Escherichia coli*. J. Bacteriol. 132, 657-665.

Rietsch, A., and Mekalanos, J.J. (2006). Metabolic regulation of type III secretion gene expression in *Pseudomonas aeruginosa*. Mol. Microbiol. 59, 807-820.

Ringen, L.M., and Drake, C.H. (1952). A study of the incidence of *Pseudomonas aeruginosa* from various natural sources. J. Bacteriol. 64, 841-845.

Römling, U., Wingender, J., Müller, and Tümmler, B. (1994). A major *Pseudomonas aeruginosa* clone common to patients and aquatic habitats. Appl. Environ. Microbiol. 60, 1734-1738.

Ryder, C., Byrd, M., and Wozniak, D.J. (2007). Role of polysaccharides in *Pseudomonas aeruginosa* biofilm development. Curr. Opin. Microbiol. 10, 6444-6448.

Sadikot, R.T., Blackwell, T.S., Christman, J.W., and Prince, A.S. (2005). Pathogen-host interactions in *Pseudomonas aeruginosa* pneumonia. Am. J. Respir. Crit. Care. Med. 171, 1209-1223.

Sauer, F.G., Remaut, H., Hultgren, S.J., and Waksman, G. (2004). Fiber assembly by the chaperone-usher pathway. Biochimica. Biophysica. Acta. 1694, 259-267.

Schuster, M., and Greenberg, E.P. (2006). A network of networks: quorum-sensing gene regulation in *Pseudomonas aeruginosa*. Int. J. Med. Microbiol. 296, 73-81.

Schuster, M., and Greenberg, E.P. (2007). Early activation of quorum sensing in *Pseudomonas aeruginosa* reveals the architecture of a complex regulon. BMC Genomics 8.

Schuster, M., Hawkins, A.C., Harwood, C.S., and Greenberg, E.P. (2004a). The *Pseudomonas aeruginosa* RpoS regulon and its relationship to quorum sensing. Mol. Microbiol. 51, 973-985.

Coggan and Wolfgang

Schuster, M., Lohstroh, C.P., Ogi, T. and Greenberg, E.P. (2003). Identification, timing and signal specificity of *Pseudomonas aeruginosa* quorum-controlled genes: a transcriptome analysis. J. Bacteriol. 185, 2066-2079.

Schuster, M., Urbanowski, M.L., and Greenberg, E.P. (2004b). Promoter specificity in *Pseudomonas aeruginosa* quorum sensing revealed by DNA binding of purified LasR. Proc. Natl. Acad. Sci. USA 101, 15833-15839.

Seed, P.C., Passador, L., and Iglewski, B.H. (1995). Activation of the *Pseudomonas aeruginosa lasI* gene by LasR and the *Pseudomonas* autoinducer PAI: an autoinduction regulatory hierarchy. J. Bacteriol. 177, 654-659.

Seet, Q., and Zhang, L.H. (2011). Anti-activator QslA defines the quorum sensing threshold and response in *Pseudomonas aeruginosa*. Mol. Microbiol. 80, 951-965.

Serate, J., Roberts, G.P., Berg, O., and Youn, H. (2011). Ligand response of Vfr, the virulence factor regulator in *Pseudomonas aeruginosa*. J. Bacteriol. 193, 4859-4868.

Siehnel, R., Traxler, B., An, D.D., Parsek, M.R., Schaefer, A.L., and Singh, P.K. (2010). A unique regulator controls the activation threshold of quorum-regulated genes in *Pseudomonas aeruginosa*. Proc. Natl. Acad. Sci. USA 107, 7916-7921.

Silversmith, R.E., and Bourret, R.B. (1999). Throwing the switch in bacterial chemotaxis. Trends Microbiol. 7, 16-22.

Simm, R., Morr, M., Kader, A., Nimtz, M., and Römling, U. (2004). GGDEF and EAL domains inversely regulate cyclic di-GMP levels and transition from sessility to motility. Mol. Microbiol. 53, 1123-1134.

Singh, P.K., Schaefer, A.L., Parsek, M.R., Moninger, T.O., Welsh, M.J., and Greenburg, E.P. (2000). Quorum-sensing signals indicate that cystic fibrosis lungs are infected with bacterial biofilms. Nature 407, 762-764.

Smith, E.E., Buckley, D.G., Wu, Z., Saenphimmachak, C., Hoffman, L.R., D'Argenio, Miller, S.I., Ramsey, B.W., Speert, D.P., Moskowitz, S.M., Burns, J.L., Kaul, R., and Olson M.V. (2006). Genetic adaptation by *Pseudomonas aeruginosa* to the airways of cystic fibrosis patients. Proc. Natl. Acad. Sci. USA 103, 8487-8492.

Smith, R.S., Wolfgang, M.C., and Lory, S. (2004). An adenylate cyclase-controlled signaling network regulates *Pseudomonas aeruginosa* virulence in a mouse model of acute pneumonia. Infect. Immun. 72, 1677-1684.

Son, M.S., Matthews, M.J. Jr., Kang, Y., Nguyen, D.T., and Hoang, T.T. (2007). *In vivo* evidence of *Pseudomonas aeruginosa* nutrient acquisition and pathogenesis in the lungs of cystic fibrosis patients. Infect. Immun. 75, 5313-5324.

Starkey, M., Hickman, J.H., Ma, L., Zhang, N., De Long, S., Hinz, A., Palacios, S., Manoil, C., Kirisits, J., Starner, T.D., Wozniak, D.J., Harwood, C.S., and Parsek, M.R. (2009). *Pseudomonas aeruginosa* rugose small-colony variants have adaptations that likely promote persistence in the cystic fibrosis lung. J. Bacteriol. 191, 3492-3503.

Stover, C.K., Pham, X.Q., Erwin, A.L., Mizoguchi, S.D., Warrener, P., Hickey, M.J., and et al. (2000). Complete genome sequence of *Pseudomonas aeruginosa* PA01, an opportunistic pathogen. Nature 406, 959-964.

Suh, S.J., Runyen-Janecky, L.J., Maleniak, T.C., Hager, P., MacGregor, C.H., Zielinski-Mozny, N.A., Phibbs Jr., P.V., and West, S.E. (2002). Effect of *vfr* mutation on global gene expression and catabolite repression control of *Pseudomonas aeruginosa*. Microbiology 148, 1561-1569.

Taccetti, G., Campana, S., Festini, F., Mascherini, M., and Doring, G. (2005). Early eradication therapy against *Pseudomonas aeruginosa* in cystic fibrosis patients. Eur. Resp. J. 26, 458-461.

Tamayo, R., Pratt, J.T., and Camilli, A. (2007). Roles of cyclic diguanylate in the regulation of bacterial pathogenesis. Annu. Rev. Microbiol. 61, 131-148.

Taylor, B.L., and Zhulin, I.B. (1999). PAS domains: internal sensors of oxygen, redox potential, and light. Microbiol. Mol. Bio. Rev. 73, 479-506.

Téllez-Sosa, J., Soberón, N., Vega-Segura, A., Torres-Márquez, M.E., and Cevallos, M.A. (2002). The Rhizobium etli *cyaC* product: Characterization of a Novel Adenylate Cyclase Class. J. Bacteriol. 184, 3560-3568.

Ueda, A., and Wood, T.K. (2009a). Connecting quorum sensing, c-di-GMP, Pel polysaccharide, and biofilm formation in *Pseudomonas aeruginosa* through tyrosine phosphatase TbpA (PA3885). PLoS Pathog. 5, e1000483.

Ueda, A., Attila, C., Whiteley, M., and Wood, T.K. (2009b). Uracil influences quorum sensing and biofilm formation in *Pseudomonas aeruginosa* and fluorouracil is an antagonist. Microb. Biotechnol. 2, 62-74.

Vallet, I., Diggle, S.P., Stacey, R.E., Camara, M., Ventre, I., and Lory, S. (2004). Biofilm formation in *Pseudomonas aeruginosa*: fimbrial *cup* gene clusters are controlled by the transcriptional regulator MvaT. J. Bacteriol. 186, 2880-2890.

Vallet, I., Olson, J.W., Lory, S., Lazdunski, A., and Filloux, A. (2001). The chaperone/usher pathway of *Pseudomonas aeruginosa*: identification of fimbrial genes clusters (*cup*) and their involvement in biofilm formation. Proc. Natl. Acad. Sci. USA 98, 6911-6916.

Valverde, C.H., Keel, S., and Haas, D. (2003). RsmY, a small regulatory RNA, is required with RsmZ for GacA-dependent expression of biocontrol traits in *Pseudomonas fluorescens* CHAO. Mol. Microbiol. 50, 1361-1379.

van Delden, C., Comte, R., and Bally, A.M. (2001). Stringent response activates quorum sensing and modulates cell density-dependent gene expression in *Pseudomonas aeruginosa*. J. Bacteriol. 183, 5376-5384.

Ventre, I., Goodman, A.L., Vallet-Gely, I., Vasseur, P., Soscia, C., Molin, S., Bleves, S., Lazdunski, A., Lory, S., and Filloux, A. (2006). Multiple sensors control reciprocal expression of *Pseudomonas aeruginosa* regulatory RNA and virulence genes. Proc. Natl. Acad. Sci. USA 103, 171-176.

Ventre, I., Ledgham, F., Prima, V., Lazdunski, A., Foglino, M., and Sturgis, J.N. (2003). Dimerization of the quorum sensing regulator RhlR: development of a method using EGFP fluorescence anisotropy. Mol. Microbiol. 48.

Wade, D.S., Calfee, M.W., Rocha, E.R., Ling, E.A., Engstrom, E., and Coleman, J.P. (2005). Regulation of *Pseudomonas* quinolone signal synthesis in *Pseudomonas aeruginosa*. J. Bacteriol. 187, 4372-4380.

Wagner, V., Gillis, R.J., and Iglewski, B.H. (2004). Transcriptome analysis of quorum-sensing regulation and virulence factor expression in *Pseudomonas aeruginosa*. Vaccine 6, S15-20.

Wagner, V.E., Bushnell, D., Passador, L., Brooks, A.I., and Iglewski, B.H. (2003). Microarray analysis of *Pseudomonas aeruginosa* quorum-sensing regulons:

effects of growth phase and environment. J. Bacteriol. 185, 2080-2095.

Wei, B.L., Brun-Zinkernagel, A.M., Simecka, J.W., Pruss, B.M., Babitzke, P., and Romeo, T. (2001). Positive regulation of the motility and *flhDC* expression by the RNA-binding protein CsrA of *Escherichia coli*. Mol. Microbiol. 40, 245-256.

West, S.E., Sample, A.K., and Runyen-Janecky, L.J. (1994). The *vfr* gene product, required for exotoxin A and protease production, belongs to the cyclic AMP receptor protein family. J. Bacteriol. 176, 7532-7542.

Whitchurch, C.B., Leech, A.J., Young, M.D., Kenneydy, D., Sargent, J.L., Bertrand, J.J., Semmler, A.B.T., Mellick, A.S., Martin, P.R., Alm, R.A., Hobbs, M., Beatson, S.A., Huang, B., Nguyen, L., Commoli, J.C., Engel, J.N., Darzins, A., and Mattick, J.S. (2004). Characterization od a complex chemosensory signal transduction system which controls twitching motility in *Pseudomonas aeruginosa*. Mol. Micobiol. 52, 873-893.

Whiteley, M., and Greenberg, E.P. (2001). Promoter specificity elements in *Pseudomonas aeruginosa* quorum-sensing-controlled genes. J. Bacteriol. 183, 169-204.

Whiteley, M., Lee, K.M., and Greenberg, E.P. (1999). Identification of genes controlled by quorum sensing in *Pseudomonas aeruginosa*. Proc. Natl. Acad. Sci. USA 96, 13904-13909.

Whiteley, M., Parsek, M.R., and Greenberg, E.P. (2000). Regulation of quorum sensing by RpoS in *Pseudomonas aeruginosa*. J. Bacteriol. 182, 4356-4360.

Wilder, C.N., and Allada, G., and Schuster, M. (2009). Instantaneous within-patient diversity of *Pseudomonas aeruginosa* quorum-sensing populations from cystic fibrosis lung infections. Infect. Immun. 77, 5631-5639.

Wolfgang, M.C., Kulasekara, B.R., Liang, X., Boyd, D., Wu, K., Yang, Q., Miyada, C.G., and Lory, S. (2003a). Conservation of genome content and virulence determinants among clinical and environmental isolates

of *Pseudomonas aeruginosa*. Proc. Natl. Acad. Sci. USA 100, 8484-8489.

Wolfgang, M.C., Lee, V.T., Gilmore, M.E., and Lory, S. (2003b). Coordinate regulation of bacterial virulence genes by a novel adenylate cyclase-dependent signaling pathway. Dev. Cell 4, 253-263.

Workentine, M.L., Chang, L., Ceri, H., and Turner, R.J. (2009). The GacS-GacA two-component regulatory system of *Pseudomonas fluorescens*: a bacterial two-hybrid analysis. FEMS Microbiol. Lett. 292, 50-56.

Wozniak, D.J., and Ohman, D.E. (1994). Transcriptional analysis of the *Pseudomonas aeruginosa* genes *algR*, *algB*, and *algD* reveals a hierarchy of alginate gene expression which is modulated by *algT*. J. Bacteriol. 176, 6007-6014.

Wu, W., Badrane, H., Arora, S., Baker, H.V., and Jin, S. (2004). MucA-mediated coordination of type III secretion and alginate synthesis in *Pseudomonas aeruginosa*. J. Bacteriol. 186, 7575-7585.

Xiao, G., Déziel, E., He, J., Lépine, F., Lesic, B., Castonguay, M.H., Milot, S., Tampakaki, A.P., Stachel, S.E., and Rahme, L.G. (2006). MvfR, a key *Pseudomonas aeruginosa* pathogenicity LTTR-class regulatory protein, has dual ligands. Mol. Micobiol. 62, 1689-1699.

Yahr, T.L.V., A.J., Hancock, M.K., Barbieri, J.T., and Frank, D.W. (1998). ExoY, an adenylate cyclase secreted by the *Pseudomonas aeruginosa* type III secretion system. Proc. Natl. Acad. Sci. USA 35, 13899-13904.

Yang, L., Jelsbak, L., and Molin, S. (2011). Microbial ecology and adaptation in cystic fibrosis airways. Environ. Microbiol. 13, 1682-1689.

Zolfaghari, I., Angus, A.A., Kang, P.J., To, A., Evans, D.J., and Fleiszig, S.M. (2005). Mutation of *retS*, encoding a putative hybrid two-component regulatory protein in *Pseudomonas aeruginosa*, attenuates multiple virulence mechanisms. Microbes Infect. 7, 1305-1316.

Caister Academic Press is a leading academic publisher of advanced texts in microbiology, molecular biology and medical research. Full details of all our publications at caister.com

- **MALDI-TOF Mass Spectrometry in Microbiology**
  - Edited by: M Kostrzewa, S Schubert (2016)
  - www.caister.com/malditof

- **Aspergillus and Penicillium in the Post-genomic Era**
  - Edited by: RP Vries, IB Gelber, MR Andersen (2016)
  - www.caister.com/aspergillus2

- **The Bacteriocins: Current Knowledge and Future Prospects**
  - Edited by: RL Dorit, SM Roy, MA Riley (2016)
  - www.caister.com/bacteriocins

- **Omics in Plant Disease Resistance**
  - Edited by: V Bhadauria (2016)
  - www.caister.com/opdr

- **Acidophiles: Life in Extremely Acidic Environments**
  - Edited by: R Quatrini, DB Johnson (2016)
  - www.caister.com/acidophiles

- **Climate Change and Microbial Ecology: Current Research and Future Trends**
  - Edited by: J Marxsen (2016)
  - www.caister.com/climate

- **Biofilms in Bioremediation: Current Research and Emerging Technologies**
  - Edited by: G Lear (2016)
  - www.caister.com/biorem

- **Microalgae: Current Research and Applications**
  - Edited by: MN Tsaloglou (2016)
  - www.caister.com/microalgae

- **Gas Plasma Sterilization in Microbiology: Theory, Applications, Pitfalls and New Perspectives**
  - Edited by: H Shintani, A Sakudo (2016)
  - www.caister.com/gasplasma

- **Virus Evolution: Current Research and Future Directions**
  - Edited by: SC Weaver, M Denison, M Roossinck, et al. (2016)
  - www.caister.com/virusevol

- **Arboviruses: Molecular Biology, Evolution and Control**
  - Edited by: N Vasilakis, DJ Gubler (2016)
  - www.caister.com/arbo

- **Shigella: Molecular and Cellular Biology**
  - Edited by: WD Picking, WL Picking (2016)
  - www.caister.com/shigella

- **Aquatic Biofilms: Ecology, Water Quality and Wastewater Treatment**
  - Edited by: AM Romaní, H Guasch, MD Balaguer (2016)
  - www.caister.com/aquaticbiofilms

- **Alphaviruses: Current Biology**
  - Edited by: S Mahalingam, L Herrero, B Herring (2016)
  - www.caister.com/alpha

- **Thermophilic Microorganisms**
  - Edited by: F Li (2015)
  - www.caister.com/thermophile

- **Flow Cytometry in Microbiology: Technology and Applications**
  - Edited by: MG Wilkinson (2015)
  - www.caister.com/flow

- **Probiotics and Prebiotics: Current Research and Future Trends**
  - Edited by: K Venema, AP Carmo (2015)
  - www.caister.com/probiotics

- **Epigenetics: Current Research and Emerging Trends**
  - Edited by: BP Chadwick (2015)
  - www.caister.com/epigenetics2015

- **Corynebacterium glutamicum: From Systems Biology to Biotechnological Applications**
  - Edited by: A Burkovski (2015)
  - www.caister.com/cory2

- **Advanced Vaccine Research Methods for the Decade of Vaccines**
  - Edited by: F Bagnoli, R Rappuoli (2015)
  - www.caister.com/vaccines

- **Antifungals: From Genomics to Resistance and the Development of Novel Agents**
  - Edited by: AT Coste, P Vandeputte (2015)
  - www.caister.com/antifungals

- **Bacteria-Plant Interactions: Advanced Research and Future Trends**
  - Edited by: J Murillo, BA Vinatzer, RW Jackson, et al. (2015)
  - www.caister.com/bacteria-plant

- **Aeromonas**
  - Edited by: J Graf (2015)
  - www.caister.com/aeromonas

- **Antibiotics: Current Innovations and Future Trends**
  - Edited by: S Sánchez, AL Demain (2015)
  - www.caister.com/antibiotics

- **Leishmania: Current Biology and Control**
  - Edited by: S Adak, R Datta (2015)
  - www.caister.com/leish2

- **Acanthamoeba: Biology and Pathogenesis (2nd edition)**
  - Author: NA Khan (2015)
  - www.caister.com/acanthamoeba2

- **Microarrays: Current Technology, Innovations and Applications**
  - Edited by: Z He (2014)
  - www.caister.com/microarrays2

- **Metagenomics of the Microbial Nitrogen Cycle: Theory, Methods and Applications**
  - Edited by: D Marco (2014)
  - www.caister.com/n2

Order from caister.com/order
