
# Structural dissection of Ebola virus and its assembly determinants using cryo-electron tomography

Tanmay A. M. Bharat${}^{a}$, Takeshi Noda${}^{b}$, James D. Riches${}^{a}$, Verena Kraehling${}^{c}$, Larissa Kolesnikova${}^{c}$, Stephan Becker${}^{c}$, Yoshihiro Kawaoka${}^{b,d,e,f}$, and John A. G. Briggs${}^{a,1}$

${}^{a}$Structural and Computational Biology Unit, European Molecular Biology Laboratory, 69117 Heidelberg, Germany; ${}^{b}$International Research Center for Infectious Diseases and ${}^{d}$Division of Virology, Department of Microbiology and Immunology, Institute of Medical Science, University of Tokyo, Tokyo 108-8639, Japan; ${}^{c}$Institut für Virologie, Philipps-Universität Marburg, 35043 Marburg, Germany; ${}^{e}$Exploratory Research for Advanced Technology Infection-Induced Host Responses Project, Japan Science and Technology Agency, Saitama 332-0012, Japan; and ${}^{f}$Department of Pathobiological Sciences, School of Veterinary Medicine, University of Wisconsin, Madison, WI 53711

Edited by Peter Palese, Mount Sinai School of Medicine, New York, NY, and approved February 6, 2012 (received for review December 12, 2011)

Ebola virus is a highly pathogenic filovirus causing severe hemorrhagic fever with high mortality rates. It assembles heterogeneous, filamentous, enveloped virus particles containing a negative-sense, single-stranded RNA genome packaged within a helical nucleocapsid (NC). We have used cryo-electron microscopy and tomography to visualize Ebola virus particles, as well as Ebola virus-like particles, in three dimensions in a near-native state. The NC within the virion forms a left-handed helix with an inner nucleoprotein layer decorated with protruding arms composed of VP24 and VP35. A comparison with the closely related Marburg virus shows that the N-terminal region of nucleoprotein defines the inner diameter of the Ebola virus NC, whereas the RNA genome defines its length. Binding of the nucleoprotein to RNA can assemble a loosely coiled NC-like structure; the loose coil can be condensed by binding of the viral matrix protein VP40 to the C terminus of the nucleoprotein, and rigidified by binding of VP24 and VP35 to alternate copies of the nucleoprotein. Four proteins (NP, VP24, VP35, and VP40) are necessary and sufficient to mediate assembly of an NC with structure, symmetry, variability, and flexibility indistinguishable from that in Ebola virus particles released from infected cells. Together these data provide a structural and architectural description of Ebola virus and define the roles of viral proteins in its structure and assembly.

Mononegavirales | single-stranded RNA virus | virus structure | subtomogram averaging

Ebolavirus (EBOV) and Marburg virus (MARV) constitute the family *Filoviridae* within the order *Mononegavirales*. Filoviruses are highly pathogenic, causing severe hemorrhagic fever in monkeys and humans, with high mortality rates (1). Because of the lack of approved vaccines and antiviral drugs, both EBOV and MARV are categorized as biosafety level-4 (BSL-4) pathogens.

The order *Mononegavirales* also contains several other pathogens of clinical importance, such as rabies virus (RABV), mumps virus, measles virus (MeV), and respiratory syncytial virus (RSV) (2). All members of the order possess a nonsegmented, negative-sense RNA genome, which is encapsidated by the viral nucleoprotein (NP). The NP–RNA complex acts as the template for genome replication and assembles into a helical nucleocapsid (NC) along with accessory proteins (3). This characteristic links genome replication mechanisms of mononegaviruses to their NC structure. The NC is recruited to the plasma membrane by the viral matrix protein, where it buds through the membrane to form an enveloped virion. All mononegaviruses share these fundamental characteristics.

EBOV virions contain an RNA genome and seven viral proteins: NP, VP35, VP40, GP (glycoprotein), VP30, VP24, and an RNA-dependent RNA polymerase (L). NP, VP30, VP35, and L are known to associate with the transcription and replication-competent NC (4–6). VP24 is additionally required for NC assembly (7, 8). VP40, the viral matrix protein, binds directly to the viral envelope. Expression of VP40 alone in mammalian cells can lead to formation and release of enveloped, filamentous virus-like particles (VLPs) (9–12). Expression of NP alone leads to the formation of narrow, tubular structures in the cytoplasm of the cell (13). These narrow structures can be recruited into VLPs by coexpression of VP40 (14). If NP is expressed together with VP24 and VP35, cytoplasmic clusters of NC-like structures are formed that are similar to those seen in infected cells (13). These structures are also recruited into VLPs when VP40 is coexpressed (7, 13–15). Together these studies suggest that a direct interaction between VP40 and NP can recruit NP into released VLPs and that formation of an NC with diameter similar to that in native virions requires co-expression of NP, VP24, and VP35.

Recent cryo-electron microscopy (cryoEM) investigations of MARV described the 3D structure of the MARV NC (16). The MARV NC is a left-handed helix, with the viral NP forming the innermost layer of the structure. Each NP binds to six bases of RNA. Arm-like structures protrude from alternate interfaces between NPs, and immuno-electron microscopy analysis locates VP24 and VP35 to these protrusions. The NC is incorporated into virions by envelopment at the plasma membrane initiated at one end of the NC (16, 17).

In the present study, EBOV virions were imaged using cryoEM and cryo-electron tomography (cryoET) to describe their structure in a near-native state. Image-processing techniques were applied to define the 3D structure of the NC within the virion. The EBOV NP shares ≈40% sequence homology with MARV NP (18, 19). Comparison of the morphological parameters and NC structures of EBOV with MARV allowed us to dissect the roles played by the RNA genome and filovirus NPs in determining NC structure.

In addition, Ebola VLPs were produced with different combinations of viral proteins and studied using biochemical, cryoEM, and cryoET techniques. These studies define roles for viral proteins in determining the structure of EBOV virions and their NCs, which range from mediating initial coiling of the NC helix, to helical condensation, to rigid helix formation, to NC envelopment into virions.

## Results

CryoEM and CryoET of EBOV. Zaire EBOV virions were harvested from infected Vero cells 1 d after infection in a BSL-4 laboratory. The inactivated virus pellet was released from the BSL-4 laboratory and then imaged using cryoEM. Long, filamentous mem-

---

Author contributions: T.A.M.B., T.N., S.B., Y.K., and J.A.G.B. designed research; T.A.M.B., T.N., J.D.R., V.K., and L.K. performed research; T.A.M.B., T.N., J.D.R., L.K., S.B., Y.K., and J.A.G.B. analyzed data; and T.A.M.B. and J.A.G.B. wrote the paper.

The authors declare no conflict of interest.

This article is a PNAS Direct Submission.

Data deposition: CryoEM data reported in this paper have been deposited with the Electron Microscopy Data Bank (accession no. **EMD-2043**).

${}^{1}$To whom correspondence should be addressed. E-mail: briggs@embl.de.

This article contains supporting information online at **www.pnas.org/lookup/suppl/doi:10.1073/pnas.1120453109/-/DCSupplemental**.
brane-bound particles could be observed along with spherical particles and other irregularly shaped vesicles (Fig. 1A). Several virions possessed previously described “moth-eaten” membranes (1). Some filamentous particles lacked NC structures, resulting in a smaller diameter (Fig. 1 A and B, black arrow). Many virions displayed an intact membrane and a clearly visible NC (Fig. 1B, white arrow). This variable morphology of EBOV is consistent with previous negative staining EM analysis (1).

CryoEM allows excellent preservation of the specimen in a near-native environment: this allowed us to accurately measure morphological parameters of virions. The distribution of virus lengths (Fig. 1C, Fig. S1A, and SI Materials and Methods) showed a major population of virus particles with a length of 1,028 ± 69 nm (*n* = 37), consistent with previously reported values of 970–1,200 nm for the average EBOV virion (20, 21). We also found a second population with a mean length of 1,978 ± 112 nm (*n* = 8) (Fig. 1C), as well as some longer particles (Fig. S1A). The diameter of filamentous EBOV particles that had a continuous membrane and an internalized NC was 90 ± 3 nm (*n* = 50) (Fig. 1C, Right), which is slightly smaller than that of MARV particles (92 ± 4 nm) (Fig. 1D, Right).

To understand the 3D arrangement of the virion, we performed cryoET. A slice through a representative tomogram is shown in Fig. 2A. The viral NC appears as a cylinder-like density within the particle center (white arrow in Fig. 2A, Movie S1), similar in appearance to the MARV NC (16). Regular repeats at a pitch of ≈7 nm could be observed along the length of the NC.

To resolve the structure of the EBOV NC in more detail, we applied subtomogram averaging methods on the tomography data (SI Materials and Methods), as described previously (16, 22). All of the reconstructed helices were left-handed with an inner layer decorated with arm-like protrusions in the outer layer. Of NCs whose symmetry could be unambiguously assigned, all were found to contain either 11.8 or 12.8 repeating units per turn. Combining all of the NC helices with the same symmetry into one single subtomogram averaging reconstruction enabled refinement up to 4.1 nm resolution. We also performed iterative real-space helical reconstruction using helical segments extracted from 2D cryoEM images of EBOV (Fig. S2, Left) and again obtained a reconstruction with a resolution of 4.1 nm. In contrast, subtomogram averaging and iterative real-space helical reconstruction resolved the MARV NC to better resolutions of 3.4 nm and 2.5 nm, respectively (16). This suggests that the EBOV NC has a higher amount of conformational variability or flexibility than the MARV NC.

We therefore identified the subset of NC helices that aligned successfully (SI Materials and Methods) and had a symmetry of 11.8 subunits per turn and combined them into one final reconstruction (Fig. 2 *B* and *C*, Left) with an improved resolution of 3.6 nm. The final subtomogram averaging reconstruction shows the EBOV NC helix to be left-handed with a pitch of ≈7.4 nm (Fig. 2 *B* and *C*, Left) close to that of MARV (7.5 nm) (16). An inner layer is observed with a diameter of ≈28 nm. Boomerang-shaped densities protrude outward from this inner layer, and the diameter of the entire structure is ≈40 nm. The protrusions have two lobes. A left-handed helix with a pitch of

![Figure 1](https://www.pnas.org/content/pnas/110/10/4276/F1.large.jpg)

**Fig. 1.** CryoEM of EBOV. (*A*) Low-magnification cryoEM images of purified EBOV. Protein density is black. Filamentous particles of varying lengths, spherical particles, and other irregularly shaped particles are observed. (*B*) CryoEM image of a filamentous EBOV virion. White arrow, EBOV virion with an NC. Black arrow, a thin particle without an internalized NC. (*C*) Histograms of virion length (*Left*) and diameter (*Right*) for filamentous EBOV virions containing an NC. (*D*) Corresponding histograms for MARV. More details in Fig. S1 and SI Materials and Methods.

![Figure 2](https://www.pnas.org/content/pnas/110/10/4276/F2.large.jpg)

**Fig. 2.** CryoET and 3D reconstruction of the EBOV NC from subtomogram averaging. (*A*) A slice through a reconstructed, filtered tomogram of EBOV. Protein density is black. White arrowhead indicates the rod-like NC within the virion. (*B*) Reconstruction of the EBOV NC from cryoET and subtomogram averaging (*Left*) compared with the MARV NC reconstruction (*Right*) (16). Isosurfaces have been contoured at 1.5 σ away from the mean, and the helical axis is vertical in the plane of the paper. (*C*) The same reconstructions as *B*, viewed along the helical axis.$\approx 7.5 \mathrm{~nm}$, with an inner layer from which boomerang-shaped densities protrude outward, has been observed previously for the MARV NC (Fig. 2 $B$ and $C$, Right) (16), indicating close structural similarity between the two filovirus NCs. By analogy with MARV, the inner layer likely represents NP, and the protrusions likely contain VP24 and VP35 (16).

### Relationship Between Genome Length, NC Symmetry, and NC Length.
A comparison of MARV and EBOV genome lengths, NC symmetries, and NC lengths is informative. In MARV, there are 13.8, 14.8, or 15.8 boomerang-shaped protrusions per turn of the NC helix (16), but in EBOV there are only 11.8 or 12.8 protrusions per turn. Because there are two NP monomers for each boomerang-shaped protrusion (16), on an average this translates into 29.6 MARV NPs per turn but only 24.6 EBOV NPs per turn. Both filovirus NPs have a similar molecular mass (83.2 vs. 77.8 kDa). The smaller number of EBOV NPs per turn is reflected in the smaller diameter of the EBOV NC helix (compare Fig. 2C, Left vs. Right).

The genome lengths of Zaire EBOV and Lake Victoria MARV are very similar (18,961 vs. 19,111 bases). Because there are fewer NP molecules per turn of the EBOV NC than in the MARV NC, the EBOV NC would have to be longer to package the entire genome at the same density. The mean lengths of MARV (876 nm) and EBOV (1,028 nm) virions confirm this expectation (Fig. 1 C and D, **Fig. S1**, and **SI Materials and Methods**). On the basis of the average length of EBOV, and on the number of subunits per turn of the NC helix, we calculate that a virion of 1,028 nm in length contains $\approx 3,200$ EBOV NP molecules per virion (**SI Materials and Methods**). This means that for each EBOV NP molecule, there are $5.9 \pm 0.4$ RNA bases. Like MARV (16), EBOV therefore likely packages six RNA bases per copy of NP. The longer virions, with a length of 1,978 nm, would contain $\approx 6,450$ copies of the NP and therefore probably package two copies of the genome (**SI Materials and Methods**).

### Formation of the Inner NC Helix.
After describing the structure of the EBOV NC, we wanted to understand the roles of different viral proteins in assembling the NC. To determine the minimum assembly component of EBOV NC, we purified full-length EBOV NP from mammalian cells. This sample has been previously shown to assemble together with cellular RNA, and appears by negative staining EM as coil-like structures (18). Using cryoEM, we confirmed that the sample formed loose coil-like structures (Fig. 3A). The diameter of the coils was roughly 30–40 nm but varied slightly between individual coils.

The C-terminal parts of NPs from other members of Mononegavirales like MARV and MeV are known to contain large disordered regions (16, 23, 24). Deletion of the C-terminal disordered region of the MARV NP allowed it to assemble condensed helical rods with a diameter of $\approx 33$ nm (16). To test whether this was also the case in EBOV, we expressed and purified a C-terminal deletion mutant of the EBOV NP containing only the first 451 amino acid residues [NP(1-451)]. This construct is known to be sufficient to bind RNA and assemble an NC coil (18). In contrast to the full-length NP, we found that NP(1-451) mostly formed condensed helical rods with a defined diameter and pitch (Fig. 3B). We extracted short helical segments from cryoEM images of the NP(1-451) mutant and carried out 2D alignment and averaging. The average image (Fig. 3B, Inset) shows that the diameter of the helix is $\approx 28$ nm and that the pitch of the helix is $\approx 7.4$ nm. A reconstruction of the NP(1-451) helix using real-space helical reconstruction techniques was obtained (**Fig. S2**, Center) and compared with the EBOV NC reconstruction. The N-terminal 451 residues of NP assemble into a helical structure that is similar to the innermost layer of the complete EBOV NC, suggesting that these residues form the core of the helical NC.

These data show that NP–NP oligomerization on cellular RNA forms a loose coil. In contrast, the first 451 residues of EBOV NP can oligomerize on RNA to form condensed helical rods in which

---

**Fig. 3.** Minimum assembly component of the EBOV NC. (A) CryoEM image of purified full-length EBOV NP. Protein density is black. (B) Image of purified NP(1-451). Inset: 2D average of extracted helical segments. Width of box 720 nm, protein density white. (C) Corresponding images of the NC helix purified from NP+VP40 VLPs. (D) Comparison of proportion of condensed helices (green) and loose coils (yellow) observed in the three samples. Data values are in **Table S1**.

both the diameter and helical pitch are the same as the inner layer of EBOV NC in virions. The N-terminal region of NP is thus sufficient to form the interactions around and along the helix, which define the pitch and inner diameter of the EBOV NC.

Because VP40 has been shown to bind to the C terminus of NP (25), we wanted to test whether co-expression of NP with VP40 could also lead to the formation of condensed helices. We therefore expressed both full-length NP and VP40 in mammalian cells (Materials and Methods), which leads to the formation and release of VLPs containing NC-like structures (13, 15, 25). VLPs were collected, their membranes were disrupted, and the NCs were then isolated by ultracentrifugation for imaging by cryoEM. Whereas full-length NP purified from cells in the absence of VP40 formed only loose coils, we observed that the NC helix purified from NP+VP40 VLPs formed short stretches of condensed helices punctuated by short coil-like regions (Fig. 3C). 2D averaging of the condensed helical segments showed a helix with a diameter of $\approx 28$ nm and pitch of $\sim 7.5$ nm (Fig. 3C, Inset). The NC helix purified from NP+VP40 VLPs is therefore very similar to the NC helix purified from cells expressing NP(1-451) in the absence of VP40. This similarity is further highlighted by quantification of the number of condensed helices and coils found in the three samples (Fig. 3D, **Table S1**, and **SI Materials and Methods**). These data support a model whereby the C-terminal part of NP disrupts helix condensation, and interaction of VP40 with the C-terminal part of NP relieves this disruptive effect to allow NP–NP contacts to form between turns of the helix, leading to NC condensation.Order of Protein Assembly and Formation of a Rigid NC Helix. We next prepared a series of VLPs by expression of different combinations of viral proteins along with VP40 in mammalian cells: NP+VP40, NP+VP24+VP40, NP+VP35+VP40, and NP+VP24+VP35+VP40. Previous thin-section EM analyses have indicated the presence of an NC-like structure in these VLPs (13–15). We analyzed recruitment of viral proteins into released VLPs using Western blot analysis with anti-NP, anti-VP24, anti-VP35, and anti-VP40 sera. We found that NP was recruited into VLPs by coexpression with VP40 alone (Fig. 4A). When NP, VP35, and VP40 were coexpressed, all three proteins could be detected in VLPs. These observations are consistent with previous observations that VP40 can recruit VP35 and NP independently into VLPs (13, 26). We found that only a low amount of VP24 was recruited into VLPs when it was coexpressed with NP and VP40. However, VP24 was detected in large amounts when VP35 was additionally expressed (Fig. 4A). These results indicate that NP can be directly recruited into VLPs by VP40, that VP24 and VP35 can be recruited by NP and/or VP40, and that VP35 significantly enhances the recruitment of VP24 into VLPs.

All filamentous VLPs were subjected to cryoEM and cryoET. CryoEM was used to quantify the number of VLPs with and without an internalized NC. CryoET was used to divide VLPs that contained NCs into two structural classes by a visual inspection of the filtered tomograms. The first class had an NC with short stretches of condensed helix broken at multiple points (Fig. 4C). The second class contained a rigid, largely continuous NC structure with outer protrusions (Fig. 4D). We compared the frequencies of the different classes of NCs found in the VLP samples from tomograms (Table S2) with those in EBOV virions. Together the cryoEM and cryoET data showed that in NP+VP40 VLPs, 64% of the VLPs were empty (Fig. 4E), and 36% contained broken,

discontinuous NCs. A rigid, continuous NC could not be observed in any of the NP+VP40 particles. A very similar pattern was found in NP+VP24+VP40 VLPs (73% empty, 27% broken) and in NP+VP35+VP40 VLPs (66% empty, 34% broken) (Fig. 4E).

Although the percentage of empty particles (68%) in NP+VP24+VP35+VP40 VLPs was similar to the other analyzed VLP samples, the NC, when present, was predominantly rigid: 30% of VLPs contained a rigid NC structure, whereas only 2% contained a broken or discontinuous NC. These numbers are comparable to our observations of authentic virions, in which we found that 63% of the particles were empty and 34% contained continuous rigid NCs (Fig. 4E). Unlike authentic virions, the length of the NP+VP24+VP35+VP40 VLPs was not well defined (Fig. S1C). To summarize, in the absence of NP, VP24, or VP35, a rigid NC-like structure was never observed. When NP, VP24, VP35, and VP40 were coexpressed, VLPs were obtained with rigid NCs that were morphologically similar to the full EBOV NC. These data indicate that NP, VP24, and VP35 are all required to form a rigid, continuous NC structure.

Structural Characterization of the VLPs. To detect differences in the NCs between various VLP samples, we next performed 2D classification and averaging of helical segments extracted from cryoEM images of the VLPs (SI Materials and Methods). The NP+VP40 VLPs contained an NC helix with a diameter of ≈28 nm, lacking the arm-like protrusions observed in authentic virions (Fig. 5A). Because of discontinuities in the NC helix, the NP layer in the average image appears blurred. Average NC images from NP+VP24+VP40 VLPs and NP+VP35+VP40 VLPs had the same appearance (Fig. S3).

In contrast the NP+VP24+VP35+VP40 VLPs show a rigid NC helix (Fig. 5B) with protrusions emanating from the inner NP layer (Fig. 5B, arrows), appearing similar to NC from authentic virions (Fig. 5C). We performed 3D reconstruction of NCs from the NP+VP24+VP35+VP40 sample using subtomogram averaging and real-space helical reconstruction techniques, exactly as described above for the NC within EBOV virions. The NC helix in the VLPs adopted the same symmetries (11.8 and 12.8 protrusions per turn) and structure as the NC helix in virions (Fig. 5D and E), with an inner layer decorated with boomerang-shaped outer protrusions. The resolution of the real-space helical reconstruction was 4.1 nm (Fig. S2, Right), and a selected subset of the NCs combined with subtomogram averaging reached a resolution of 3.9 nm (Fig. 5D and E). In both cases the reconstructions are the same as the NC reconstruction from virions with the same resolution (compare Figs. 2B and C with 5D and E and Fig. S2A, Left vs. Right). Thus,

Fig. 4. Protein recruitment and formation of a rigid NC. (A) Detection of viral proteins in respective VLPs. Purified VLPs were collected, and Western blot analysis using rabbit anti-NP, -40, -35, and -24 antibodies was performed. (B) A tomographic slice through an empty VLP. Protein density is black. (C) Slice through a VLP with a broken NC. Points of breakages in the NC helix have been highlighted with white arrows. (D) A VLP with a rigid NC. (E) Proportion of particles observed with a rigid NC (dark green), with an overall broken NC (orange), and without an NC (gray) in different samples. Data values are in Table S2.

Fig. 5. Location of viral proteins in the EBOV NC. (A) 2D class averages of the NC from NP+VP40 VLPs. (B) 2D class averages of the NC from NP+VP24+VP35+VP40 VLPs. (C) 2D class averages of the NC from EBOV virions. Black arrows indicate protrusions. (D) Subtomogram averaging reconstruction of the NC helix from NP+VP24+VP35+VP40 VLPs. Isosurfaces have been contoured at 1.5σ away from the mean, and the helical axis is vertical in the plane of the paper. (E) The same reconstruction viewed along the helical axis.the NC helices from NP+VP24+VP35+VP40 VLPs and from EBOV virions are indistinguishable in structure, symmetry, and flexibility.

# Discussion

## Architecture of EBOV Virions and the EBOV NC

We found that EBOV particles were largely filamentous, but other morphologies, including spherical particles and particles without an internalized NC, were also observed. Such variable morphology is consistent with earlier observations by negative staining EM (1). Within cryoEM images we could see that straight sections of virions contain a cylindrical NC along the center of the virus particle. Most filamentous EBOV virions had a length of ≈1,028 nm, although longer viruses were also observed with lengths that were approximate multiples of this length, suggesting they contain multiple NCs.

A comparison of the EBOV with the recently presented cryoET structure of MARV NC (16) sheds light on factors affecting virus assembly. Many features are shared between the two NCs. The pitch of the EBOV NC helix (7.4 nm) is almost identical to that described by cryoEM for the MARV NC (7.5 nm). The 3D structure of the EBOV NC reveals a left-handed helical structure, just like the MARVNC (16). As in MARV it shows an inner layer made up of the viral NP, which is decorated by boomerang-shaped protrusions. By analogy with MARV, one protrusion emanates from every two NPs in the inner layer. Binding of one copy of the viral phosphoprotein to two copies of the NP has also been observed by x-ray crystallography of a purified rhabdoviral complex (27).

There are also differences between the EBOV and MARV NCs. In all our analyses the EBOV NC was consistently more flexible than the MARV NC. This suggests higher intrinsic conformational flexibility in the repeating asymmetric unit of the EBOV NC. The symmetry of the two filovirus NCs differs: the EBOV NC has fewer NP subunits per turn of the helix but has more turns of the helix per virion, so that EBOV virions are longer than MARV virions. This means that the total number of NPs is approximately the same in EBOV and MARV, and the number of RNA bases per copy of NP is also the same, with each NP binding six RNA bases.

Genome replication in Mononegavirales is tightly linked to NC structure (28). A density of six bases per NP in EBOV is consistent with previous observations that only multiples of six bases can be added or removed from the replication promoter region while maintaining function (29). Binding to a multiple of six RNA bases per NP monomer is also observed in paramyxoviruses like Sendai virus and MeV (30, 31), and like EBOV (29) these viruses also have bipartite replication promoters. These facts together suggest that genome replication mechanisms of filoviruses are likely similar to those of Sendai virus and MeV and differ from RSV and rhabdoviruses like VSV and RABV, which package different numbers of RNA bases per NP (32–34).

## Structural Roles of EBOV Components in Determining NC Structure

The expression of VP40 along with NP leads to recruitment of NP into VLPs. This is likely due to binding of VP40 to the C terminus of NP (25). Expression of NP and VP40 together allows recruitment of VP35 into VLPs. For efficient VP24 recruitment into VLPs, NP, VP35, and VP40 must be expressed. This is consistent with previous morphological studies that asserted that NP, VP24, and VP35 are all necessary for NC assembly (7, 13).

The EBOV NP alone, upon binding to RNA, forms a loosely coiled helix. Removal of the C-terminal 288 residues of NP, which are predicted to contain large disordered regions, leads to formation of condensed helices instead of loose coils. The C-terminal region of NP therefore prevents condensation of the N-terminal region of NP into helices. CryoEM observations on MARV and MeV NPs have also shown that purified NP samples could form loose helices, and that C-terminally deleted NPs could assemble condensed helices (16, 23), suggesting that this is a general property shared with other mononegaviruses. The disordered C-terminal domain of EBOV NP contains binding sites for VP40. NCs purified from VLPs produced by coexpression of NP and VP40 are condensed helices indistinguishable from those formed by C-terminally deleted NP. We therefore propose that binding of VP40 to the C-terminal region of NP during virus assembly relieves its inhibitory effect, allowing the N terminus of NP to assemble a condensed helix. The condensed EBOV NP(1-451) helices have a diameter of ≈28 nm, which is the same as the 28-nm diameter of the inner NC helix in the authentic EBOV particle. MARV NP(1-390) assembles into a condensed helix with a diameter of 33 nm, which is the same as the 33-nm diameter of the inner NC helix in the authentic MARV particle (16). This comparison suggests that the N-terminal domain of NP in filoviruses is alone sufficient to define the diameter of the NC helix.

The condensed NC helix retains some flexibility and is punctuated by breaks when packaged into VLPs, or by regions of loose coil when purified. This contrasts with the viral NC, which we found to form rigid helices. Coexpression of both VP24 and VP35 with NP and VP40 was required to release VLPs containing rigid NCs, suggesting that binding of VP24 and VP35 leads to rigidification of the helix. These two proteins form boomerang-shaped protrusions emanating from the inner NP layer. NP, VP24, VP35, and VP40 together are sufficient to assemble an NC that has the same symmetry, structure, variability, and flexibility as the NC within the virion.

The NCs in NP+VP24+VP35+VP40 VLPs do not have a defined length, contrasting with the NC in authentic EBOV and MARV virions that consistently has exactly the length required to package one viral genome at a density of six RNA bases per NP. In some cases more than one NC can be incorporated into a single virion, giving a virus particle with double

![Figure 6](attachment://image.png)

Fig. 6. Steps involved in EBOV NC assembly. A schematic illustration of the samples described in this study and their assembly properties. Assembly of a virus particle is indicated by the thick arrow. Initial condensation of the NP-RNA complex can be achieved in vitro by removal of the disordered C-terminal, or in cells by coexpression with VP40 (thin arrows). The condensed helix can be converted into a rigid NC-like helix inside VLPs only if all NP, VP24, VP35, and VP40 are expressed. The resulting NC helix is indistinguishable from that in EBOV virions.

Bharat et al.  
PNAS | March 13, 2012 | vol. 109 | no. 11 | 4279or triple the expected length. These observations imply that NC length is determined by genome length.

By analysis of purified proteins, VLPs, and virions, we can propose distinct structural roles for these components in EBOV assembly. We suggest that formation of a virus particle requires packaging of the RNA genome by N-termini of NP to form a loose coil with a length defined by total genome length. We suggest that it requires condensation of the loose coils into a helix with diameter defined by the N-terminal region of NP and that this can be mediated by binding of VP40 to the C terminus of NP. We suggest that it requires rigidification of the condensed coils into a tight helix with arm-like protrusions by binding of VP24 and VP35 to alternate NPs and that these components are sufficient to define the mature EBOV NC structure (Fig. 6).

### Materials and Methods

Purification of Recombinant EBOV NP. NP or its (1–451) truncation mutant were purified from transfected HEK 293 cells using CsCl gradient centrifugation ([SI Materials and Methods](https://www.pnas.org/content/suppl/2012/07/18/1204531109.DCSupplemental)). All samples were prepared in duplicate to control for differences between sample preparations.

Preparation of VLPs and Virus. EBOV proteins were coexpressed with VP40 in HEK 293 cells. Two days after transfection, VLPs were fixed with 1% paraformaldehyde (PFA) and pelleted by ultracentrifugation through a 20% (wt/wt) sucrose cushion. The pellet was resuspended in PBS and stored at 4 °C until further investigation.

All work with infectious EBOV was performed under highest safety precautions in the BSL-4 facility at the Institut für Virologie, Philipps-Universität Marburg. Particles of EBOV that were released from infected Vero cells were collected 1 d after infection, purified by centrifugation through a 20% sucrose cushion, and fixed with 4% PFA to inactivate the virus completely ([SI Materials](https://www.pnas.org/content/suppl/2012/07/18/1204531109.DCSupplemental)).

and Methods). All samples were prepared in duplicate and initially analyzed separately to control for differences between preparations.

#### Western Blot Analysis. Purified VLPs were lysed in SDS sample buffer and separated on a PAGE Tris/glycine gel. Blots were incubated with rabbit anti-NP, anti-VP24, anti-VP35, or anti-VP40 serum as primary antibodies, and with HRP-conjugated anti-rabbit IgG antibody as a secondary antibody. Bands were detected with ECL Plus Western Blotting Detection Reagents (GE Healthcare) and visualized using VersaDoc Imaging System (Bio-Rad).

#### CryoEM and Image Analysis. For cryoEM studies, vitrified samples were imaged under standard low-dose conditions in a FEI CM120 Biotwin microscope (120 kV). For tomography an FEITF30 Polara TEM (300 kV) with energy filter was used. Tomographic tilt ranges were typically from +60° to –60°, with a total dose of 6,000–10,000 e−/nm². For each VLP sample, 8–20 tomograms were collected, and for virions more than 20 tomograms were collected.

2D data were analyzed using Bsoft (35) and Spider (36). Helical reconstruction was carried out using the real space reconstruction technique (37) implemented in the Spider package ([SI Materials and Methods](https://www.pnas.org/content/suppl/2012/07/18/1204531109.DCSupplemental)). Tomograms were reconstructed using the IMOD software suite (38). Subtomograms were extracted along the length of NCs and iteratively aligned in six dimensions, taking into account the missing wedge as described previously (16, 39). Visualization of image data was carried out in Amira (Visage Imaging) and Chimera (40).

#### ACKNOWLEDGMENTS. This work was funded by Deutsche Forschungsgemeinschaft Grants SPP 1175 (to J.A.G.B. and S.B.) and SFB 593 (to S.B.). This work was supported by National Institute of Allergy and Infectious Diseases Public Health Service research grants (to Y.K.). This study was technically supported by the use of the European Molecular Biology Laboratory IT Service unit. T.N. was supported by a Grant-in-Aid for Young Scientists from the Japan Society for the Promotion of Science.

---

1. Sanchez A, Geisbert T, Feldmann H (2007) Filoviridae: Marburg and Ebola viruses. Fields Virology, eds Knipe D, Howley P (Lippincott Williams and Wilkins, Philadelphia), 5th Ed, Vol 1, p 1409.
2. Lamb R (2007) Mononegavirales. Fields Virology, eds Knipe D, Howley P (Lippincott Williams and Wilkins, Philadelphia), 5th Ed, Vol 1, p 1357.
3. Ruigrok RW, Crépin T, Kolakofsky D (2011) Nucleoproteins and nucleocapsids of negative-strand RNA viruses. *Curr Opin Microbiol* 14:504–510.
4. Becker S, Rinne C, Hofsäss U, Klenk H-D, Mühlberger E (1998) Interactions of Marburg virus nucleocapsid proteins. *Virology* 249:406–417.
5. Mühlberger E, Lötferring B, Klenk H-D, Becker S (1998) Three of the four nucleocapsid proteins of Marburg virus, NP, VP35, and L, are sufficient to mediate replication and transcription of Marburg virus-specific monocistronic minigenomes. *J Virol* 72:8756–8764.
6. Mühlberger E, Weik M, Volchkov VE, Klenk H-D, Becker S (1999) Comparison of the transcription and replication strategies of marburg virus and Ebola virus by using artificial replication systems. *J Virol* 73:2333–2342.
7. Huang Y, Xu L, Sun Y, Nabel GJ (2002) The assembly of Ebola virus nucleocapsid requires virion-associated proteins 35 and 24 and posttranslational modification of nucleoprotein. *Mol Cell* 10:307–316.
8. Mateo M, et al. (2011) Knockdown of Ebola virus VP24 impairs viral nucleocapsid assembly and prevents virus replication. *J Infect Dis* 204(Suppl 3):S892–S896.
9. Noda T, et al. (2002) Ebola virus VP40 drives the formation of virus-like filamentous particles along with GP. *J Virol* 76:4855–4865.
10. Jasenosky LD, Neumann G, Lukashevich I, Kawaoka Y (2001) Ebola virus VP40-induced particle formation and association with the lipid bilayer. *J Virol* 75:5205–5214.
11. Timmins J, et al. (2003) Oligomerization and polymerization of the filovirus matrix protein VP40. *Virology* 312:359–368.
12. Harty RN, Brown ME, Wang G, Huibregtse J, Hayes FP (2000) A PPxY motif within the VP40 protein of Ebola virus interacts physically and functionally with a ubiquitin ligase: Implications for filovirus budding. *Proc Natl Acad Sci USA* 97:13871–13876.
13. Noda T, et al. (2006) Assembly and budding of Ebolavirus. *PLoS Pathog* 2:e99.
14. Johnson RF, Bell P, Harty RN (2006) Effect of Ebola virus proteins GP, NP and VP35 on VP40 VLP morphology. *Virol J* 3:31.
15. Licata JM, Johnson RF, Han Z, Harty RN (2004) Contribution of ebola virus glycoprotein, nucleoprotein, and VP24 to budding of VP40 virus-like particles. *J Virol* 78:7344–7351.
16. Bharat TAM, et al. (2011) Cryo-electron tomography of Marburg virus particles and their morphogenesis within infected cells. *PLoS Biol* 9:e1001196.
17. Welsch S, et al. (2010) Electron tomography reveals the steps in filovirus budding. *PLoS Pathog* 6:e1000875.
18. Noda T, Hagiwara K, Sagara H, Kawaoka Y (2010) Characterization of the Ebola virus nucleoprotein-RNA complex. *J Gen Virol* 91:1478–1483.
19. Watanabe S, Noda T, Kawaoka Y (2006) Functional mapping of the nucleoprotein of Ebola virus. *J Virol* 80:3743–3751.
20. Geisbert TW, Jahrling PB (1995) Differentiation of filoviruses by electron microscopy. *Virus Res* 39:129–150.
21. Kiley MP, et al. (1982) Filoviridae: A taxonomic home for Marburg and Ebola viruses? *Intervirology* 18:24–32.
22. Briggs JA, et al. (2009) Structure and assembly of immature HIV. *Proc Natl Acad Sci USA* 106:11090–11095.
23. Schoehn G, et al. (2004) The 12 Å structure of trypsin-treated measles virus N-RNA. *J Mol Biol* 339:301–312.
24. Longhi S, et al. (2003) The C-terminal domain of the measles virus nucleoprotein is intrinsically disordered and folds upon binding to the C-terminal moiety of the phosphoprotein. *J Biol Chem* 278:18638–18648.
25. Noda T, Watanabe S, Sagara H, Kawaoka Y (2007) Mapping of the VP40-binding regions of the nucleoprotein of Ebola virus. *J Virol* 81:3554–3562.
26. Johnson RF, McCarthy SE, Godlewski PJ, Harty RN (2006) Ebola virus VP35-VP40 interaction is sufficient for packaging 3E-5E minigenome RNA into virus-like particles. *J Virol* 80:5135–5144.
27. Green TJ, Luo M (2009) Structure of the vesicular stomatitis virus nucleocapsid in complex with the nucleocapsid-binding domain of the small polymerase cofactor, P. *Proc Natl Acad Sci USA* 106:11713–11718.
28. Kolakofsky D, Roux L, Garcin D, Ruigrok RW (2005) Paramyxovirus mRNA editing, the “rule of six” and error catastrophe: a hypothesis. *J Gen Virol* 86:1869–1877.
29. Weik M, Enterlein S, Schlenz K, Mühlberger E (2005) The Ebola virus genomic replication promoter is bipartite and follows the rule of six. *J Virol* 79:10660–10671.
30. Egelman EH, Wu SS, Amrein M, Portner A, Murti G (1989) The Sendai virus nucleocapsid exists in at least four different helical states. *J Virol* 63:2233–2243.
31. Walpita P (2004) An internal element of the measles virus antigenome promoter modulates replication efficiency. *Virus Res* 100:199–211.
32. Tawar RG, et al. (2009) Crystal structure of a nucleocapsid-like nucleoprotein-RNA complex of respiratory syncytial virus. *Science* 326:1279–1283.
33. Albertini AA, et al. (2006) Crystal structure of the rabies virus nucleoprotein-RNA complex. *Science* 313:360–363.
34. Green TJ, Zhang X, Wertz GW, Luo M (2006) Structure of the vesicular stomatitis virus nucleoprotein-RNA complex. *Science* 313:357–360.
35. Heymann JB, Belnap DM (2007) Bsoft: Image processing and molecular modeling for electron microscopy. *J Struct Biol* 157:3–18.
36. Frank J, et al. (1996) SPIDER and WEB: Processing and visualization of images in 3D electron microscopy and related fields. *J Struct Biol* 116:190–199.
37. Egelman EH (2000) A robust algorithm for the reconstruction of helical filaments using single-particle methods. *Ultramicroscopy* 85:225–234.
38. Kremer JR, Mastronarde DN, McIntosh JR (1996) Computer visualization of three-dimensional image data using IMOD. *J Struct Biol* 116:71–76.
39. Förster F, Medalia O, Zauberman N, Baumeister W, Fass D (2005) Retrovirus envelope protein complex structure in situ studied by cryo-electron tomography. *Proc Natl Acad Sci USA* 102:4729–4734.
40. Pettersen EF, et al. (2004) UCSF Chimera—a visualization system for exploratory research and analysis. *J Comput Chem* 25:1605–1612.