
Expression of the Resistance-Nodulation-Cell Division Pump AdeIJK in Acinetobacter baumannii Is Regulated by AdeN, a TetR-Type Regulator

Nicolas Rosenfeld,$^{a}$ Christiane Bouchier,$^{b}$ Patrice Courvalin,$^{a}$ and Bruno Périchon$^{a}$

Institut Pasteur, Unité des Agents Antibactériens,$^{a}$ and Institut Pasteur, Genopole Plate-Forme Génomique,$^{b}$ Paris, France

Resistance-nodulation-division efflux system AdeIJK contributes to intrinsic resistance to various drug classes in *Acinetobacter baumannii*. By whole-genome sequencing, we have identified in clinical isolate BM4587 the *adeN* gene, located 813 kbp upstream from *adeIJK*, which encodes a TetR transcriptional regulator. In one-step mutant BM4666 overexpressing *adeIJK*, the deletion of cytosine 582 (C<sub>582</sub>) in the 3′ portion of this gene was responsible for a frameshift mutation resulting in the deletion of the seven C-terminal residues. trans-Complementation of this BM4587 derivative with a plasmid expressing *adeN* restored antibiotic susceptibility to the host associated with the loss of *adeJ* overexpression. The inactivation of *adeN* in BM4587 led to a diminished susceptibility to antibiotics that are substrates for AdeIJK and to a 5-fold increase in *adeJ* expression. Taken together, these results indicate that AdeN represses AdeIJK expression. Quantitative reverse transcription-PCR (qRT-PCR) demonstrated that AdeN is constitutively expressed in BM4587 and does not regulate its own expression. Deletion of cytosine 582 and a 394-bp deletion of the 3′ part of *adeN* were found in independent one-step *adeIJK*-overexpressing mutants selected from clinical isolates BM4667 and BM4651, respectively. The corresponding alterations were located in the α9 helix, which is known to be involved in dimerization, a process essential for the activity of TetR regulators. The *adeN* gene was detected in all of the 30 *A. baumannii* strains tested and in *Acinetobacter calcoaceticus*, *Acinetobacter nosocomialis*, and *Acinetobacter pittii*.

*Acinetobacter baumannii*, a nonfermentative Gram-negative coccobacillus, is a major nosocomial pathogen responsible for epidemic infections, such as pneumonia, meningitis, and septicemia, as well as urinary tract, skin and soft tissue, and bloodstream infections with a high morbidity and mortality. These infections are often difficult to treat due to intrinsic resistance of this species, mainly due to a chromosomally encoded cephalosporinase and low membrane permeability, as well as a high propensity to acquire resistance to numerous drugs. Common in bacteria, active efflux systems play, when overexpressed, a substantial role in acquired multidrug resistance (MDR). Among the efflux systems, resistance-nodulation-cell division (RND) pumps are the most prevalent systems in Gram-negative bacteria (19).

Three RND systems, AdeABC, AdeIJK, and AdeFGH, have been characterized and reported to be responsible for MDR in *A. baumannii* (5, 6, 16). AdeABC, found in ca. 80% of clinical isolates and probably the system most often implicated in MDR (3), is responsible for high-level resistance to aminoglycosides, β-lactams, fluoroquinolones, tetracyclines, tigecycline, macrolides, chloramphenicol, and trimethoprim (16). Its expression is tightly regulated by the two-component system AdeRS (18). Point mutations in this system or the presence of an insertion sequence upstream from the operon can lead to overexpression of the *adeABC* operon (18). However, since *adeABC*-overexpressing mutants with no mutations in *adeRS* have been reported (12, 21), other mechanisms are probably involved in the regulation of *adeABC* expression.

AdeFGH is found in ca. 90% of clinical isolates, and its overexpression is responsible for high-level resistance to chloramphenicol, clindamycin, fluoroquinolones, and trimethoprim and for decreased susceptibility to tetracycline-tigecycline and sulfonamides (5). Increased expression of the pump is due to mutations in the *adeL* gene, located upstream from *adeFGH*, that encodes a putative LysR-type transcriptional regulator (5). Analysis of the expression of the efflux system established that overexpression of AdeFGH is prevalent in clinical isolates of *A. baumannii* in Canada (2).

As opposed to AdeABC and AdeFGH, AdeIJK is present in all strains and contributes to intrinsic resistance to various drugs, including β-lactams, such as ticarcillin, cephalosporins and aztreonam, fluoroquinolones, tetracyclines, tigecycline, lincosamides, rifampin, chloramphenicol, co-trimoxazole, novobiocin, and fusidic acid (6). High-level expression of AdeIJK was experimentally shown to be toxic for *A. baumannii* (6), but one-step mutants overexpressing the pump have been obtained *in vitro* (4). Transcriptomic microarray experiments and quantitative real-time reverse transcription-PCR (qRT-PCR) have shown that the levels of *adeIJK* overexpression were low compared to those of *adeABC*, suggesting a threshold for toxicity for the host (4) and that AdeIJK is tightly regulated.

No regulatory genes were found in the vicinity of *adeIJK*, and analysis of the putative promoter region of *in vitro* overexpressing mutants did not reveal any mutations (6). We report identification, by a whole-genome sequencing approach, of AdeN, a transcription regulator belonging to the TetR family which represses expression of the *adeIJK* operon.

Received 30 December 2011 Returned for modification 29 January 2012  
Accepted 18 February 2012  
Published ahead of print 27 February 2012  
Address correspondence to Patrice Courvalin, patrice.courvalin@pasteur.fr.  
Copyright © 2012, American Society for Microbiology. All Rights Reserved.  
doi:10.1128/AAC.06422-11

TABLE 1 Bacterial strains and plasmids used in this study

| Strain or plasmid | Relevant characteristic(s) $^{a}$ | Reference or source |
|-------------------|----------------------------------|---------------------|
| **Strains**       |                                  |                     |
| A. baumannii      |                                  |                     |
| BM4587            | Wild-type clinical isolate      | 4                   |
| BM4666            | BM4587adeNΔC$_{582}$ spontaneous mutant obtained on cefotaxime | 4                   |
| BM4667            | Wild-type clinical isolate      | 4                   |
| BM4668            | BM4667adeNΔC$_{582}$ spontaneous mutant obtained on tetracycline | 4                   |
| BM4651            | BM4454ΔadeABC                    | 6                   |
| BM4669            | BM4651adeNΔ$_{258-651}$ spontaneous mutant obtained on tetracycline | This study         |
| BM4685            | BM4587ΔadeN                      | This study         |
| E. coli           |                                  |                     |
| NEB5α             | fhuA2Δ(argF-lacZ) U169 phoA glnV44 Δ80(lacZ) ΔM15 gyrA96 recA1 relA1 endA1 thi-1 hsdR17 | New England Biolabs |
| CC118             | A(ara-leu) araD ΔlacX74 galE galK phoA20 thi-1 rpsE rpoB argE(Am) recAl | 17                  |
| CC118λpir         | CC118 lysogenized with λpir      | 11                  |
| HB101             | supE44 hsdS20(rB$^{-}$ mB$^{-}$) recA13 ara-14 leuB6 proA2 lacY1 galK2 rpsL20 xyl-5 mtl-1 F$^{-}$ thi-1 | 14                  |
| **Plasmids**      |                                  |                     |
| pCR2.1            | lacZα ColEl oriR Ap$^{r}$ Km$^{r}$ | Invitrogen          |
| pCRBlunt          | Km$^{r}$ Zeocin$^{r}$ ColEl oriR lacZα ccdB | Invitrogen          |
| pUC18             | Tra$^{-}$ Mob$^{-}$ Ap$^{r}$ lacZα | 26                  |
| pAT801            | Ap$^{r}$ Tc$^{r}$               | 10                  |
| pAT522            | 702-bp adeN fragment of BM4587 cloned in pAT801 | This study         |
| pAT523            | 701-bp adeNΔC$_{582}$ fragment of BM4666 cloned in pAT801 | This study         |
| pKNG101           | Suicide vector, sacB Sm$^{r}$   | 13                  |
| pRK2013           | Helper plasmid; ColEl oriR Tra Mob Km$^{r}$ | 8                   |
| pAT524            | 1,154-bp truncated adeN cloned in pKNG101 | This study         |

$^{a}$ Ap, ampicillin; Km, kanamycin; Mob, mobilizable; r, resistant; Sm, streptomycin; Tc, tetracycline; Tra, self-transferable.

---

### MATERIALS AND METHODS

#### Bacterial strains, plasmids, growth conditions, and antibiotic susceptibility testing.
The bacterial strains and plasmids used in this study are listed in Table 1. Cells were grown at 37°C in brain heart infusion (BHI) broth and agar (Difco Laboratories, Detroit, MI). Antibiotic susceptibility was tested by disk diffusion on Mueller-Hinton agar (Bio-Rad, Marnes-la-Coquette, France), and MICs were determined by the Etest procedure (bioMérieux, Marcy l’Etoile, France).

#### Production of mutants overexpressing adeIJK.
Spontaneous mutants overexpressing adeIJK were obtained from wild-type susceptible strains BM4587 and BM4667 (4) and from laboratory strain BM4651 (6) on BHI gradient plates (25) containing 0 to 16 μg/ml of either cefotaxime or tetracycline (Table 1). Briefly, colonies growing at concentrations higher than the MIC were tested for antibiotic resistance by agar diffusion, and those resistant to several drug classes, such as tetracyclines, phenicols, fluoroquinolones, sulfonamides, and macrolides, were selected for quantitative real-time reverse-transcription PCR (qRT-PCR) analysis to quantify the expression levels of AdeIJK.

#### DNA manipulations.
A. baumannii genomic DNA was extracted as described previously (24) or by using a Wizard genomic DNA purification kit (Promega, Madison, WI). DNA amplification was performed in a GeneAmp PCR system 9700 (Perkin-Elmer Cetus, Norwalk, CT) with Taq (MPbio, Illkirch, France) or Phusion (Finnzymes, Espoo, Finland) DNA polymerase. PCR elongation times and temperatures were adjusted according to the expected sizes of the PCR products and the nucleotide sequences of the primers. PCR products were purified with a QIAquick PCR purification kit (Qiagen, Inc., Chatsworth, CA). DNA fragments were extracted from agarose gels with a QIAquick gel extraction kit (Qiagen). Nucleotide sequencing was carried out with a CEQ 8000 DNA analysis system automatic sequencer (Beckman Instruments, Inc., Palo Alto, CA).

#### Recombinant DNA techniques.
DNA isolation, digestion with restriction endonucleases (New England BioLabs, Ipswich, MA), ligation with T4 DNA ligase (New England BioLabs), and transformation of

NEB5α competent *Escherichia coli* (New England BioLabs) with recombinant plasmid DNA were performed by standard methods (24).

#### TAIL-PCR amplification.
The sequence downstream from the first 200 nucleotides of adeN in BM4669 was determined by using thermal asymmetric interlaced PCRs (TAIL-PCRs) (15) using the primers listed in Table 2. The primary PCR mixture was prepared as described previously (7). For the secondary reaction, the primary reaction mixture was diluted 50-fold, and for the tertiary reaction, a 1/10-diluted secondary reaction was used.

#### Whole-genome sequencing.
Whole-genome sequencing of strains was performed with Solexa single-reads sequencing technology. Illumina library preparation and sequencing followed standard protocols developed by the supplier. Briefly, genomic DNA was sheared by nebulization, and the fragments were end repaired and phosphorylated. Blunt-end fragments were A-tailed and ligated to sequencing adapters. Fragments with an insert size of approximately 400 bp were gel extracted and enriched with 14 cycles of PCR before library quantification and validation. Hybridization of the library to the flow cell and bridge amplification were performed to generate clusters, and single reads of 100 cycles were collected on a HiSeq 2000 (Illumina, San Diego, CA). After sequencing was completed, image analysis, base calling, and error estimation were performed using Illumina Analysis Pipeline version 1.7.

#### Insertion-inactivation of adeN.
Deletion of adeN in BM4587 was carried out following the sacB-based strategy (13). Briefly, the 5′ and 3′ flanking regions of the gene were amplified by PCR with specific primers adeN1/adeN2 and adeN3/adeN4 (Table 2; Fig. 1) designed such that each fragment had, at its end closest to adeN, a short sequence of 7 nucleotides complementary to the corresponding sequence in the other fragment. The PCR products were then linked by overlapping PCR using primers adeN1/adeN4 to form an insert that was cloned in pCR2.1 (Invitrogen, San Diego, CA). The recombinant plasmid was digested by BamHI and ApaI, and the resulting fragment was ligated to BamHI/ApaI-linearized pKNG101 and introduced into competent *E. coli* CC118λpir (Table 1). The recombinant plasmid obtained, pAT524 (ΔadeN), was then trans-

Rosenfeld et al.

TABLE 2 Oligonucleotides

| Primer       | Sequence ($5' \rightarrow 3'$)${}^{a}$                                                                 | Position${}^{b}$ |
|--------------|-----------------------------------------------------------------------------------------------------------------|------------------|
| adeN1        | ggatccGCTGTTAGGGTGGGGGTGTA                                                                                   | $-387$ to $-368$ |
| adeN2${}^{c}$ | ggagatgTTTTGAACCACCAGCATGA                                                                                 | $172$–$153$      |
| adeN3${}^{d}$ | tcaaaaaCATCTCCCCGATCATTCCTCT                                                                                | $497$–$516$      |
| adeN4        | gggcccCGTGACCAAAAGTACGAATCA                                                                                 | $1091$–$1071$    |
| adeN5v       | TCGTGAGGTTTATCCGGAAG                                                                                        | $-645$ to $-636$ |
| adeN6v       | TGCATTCAACCAGAGCAA                                                                                          | $264$–$283$      |
| adeN7v       | GAAACCCCATCATGTGTGCAG                                                                                       | $1294$–$1275$    |
| adeN9        | AACGACGGCTAACACTGCTT                                                                                       | $-11$ to $-30$   |
| adeNM3       | TGCATGATCCAGTCCCTTGAG                                                                                      | $2$–$21$         |
| adeNM4       | ACCACCAGCATGATTAACGA                                                                                       | $165$–$146$      |
| adeNAYE3     | CATTCCAATGCAAAAGAATT                                                                                        | $674$–$653$      |
| adeNAYE4     | GCAGTGTAGCCGTCGTTAT                                                                                        | $-28$ to $-9$    |
| adeNAYE5     | AAAACATTTCACGGCGATAA                                                                                       | $247$–$228$      |
| AdeNAYE7     | TTATCGCCGTGAAATGTTTT                                                                                       | $228$–$247$      |
| adeNAYE8     | ATTTTGGACATCCAGAGCAC                                                                                       | $444$–$425$      |
| TailAdeN1    | TCATGCTGGTGGTTCAAAAA                                                                                       | $-122$ to $-102$ |
| TailAdeN2    | ACACGCCGCGGTATAGAAC                                                                                        | $51$–$70$        |
| TailAdeN3    | TTTTCCCCAAAATATGCTCT                                                                                       | $152$–$171$      |
| adeJ For     | GGTCATTAATATCTTTGGC                                                                                        | $1143$–$1161$    |
| adeJ Rev     | GGTACGAATACCGCTGTCA                                                                                        | $1364$–$1346$    |
| rpoB For     | TCCGCACGTAAAGTAGGAAC                                                                                       | $2114$–$2095$    |
| rpoB Rev     | ATGCCGCCTGAAAAAGTAAC                                                                                       | $1959$–$1979$    |

${}^{a}$ The restriction sites introduced in the oligonucleotides are underlined.  
${}^{b}$ Coordinates refer to the first base of the gene.  
${}^{c}$ The first 7 nucleotides are complementary to those of adeN3 indicated in italics.  
${}^{d}$ The first 7 nucleotides are complementary to those of adeN2 indicated in italics.

Determination of the TSP. To determine the transcription start point (TSP) of *adeN*, $5'$ rapid amplification of cDNA ends-PCR ($5'$ RACE-PCR) was carried out with the $5'$-RACE system for rapid amplification of cDNA ends, version 2.0 (Invitrogen), according to the manufacturer's instructions. Briefly, total RNA ($3 \mu\mathrm{g}$) was used to generate double-stranded cDNA with adeNAYE8, an *adeN*-specific antisense oligonucleotide (Table 2). A homopolymeric tail was then added to the $3'$ end of the cDNA (corresponding to the $5'$ end of the specific mRNA) using TdT (terminal deoxynucleotidyltransferase) and dCTP. Finally, tail-cDNA was used as a template for two amplification steps, using a nested gene-specific primer (adeNAYE5 then adeN2) (Table 2) and a combination of a complementary homopolymer-containing anchor primer and the corresponding adapter primer which allows amplification from the homopolymeric tail (abridged anchor primer [AAP], then abridged universal amplification primer [AUAP]). The resulting amplified fragments were purified by low-melting-point agarose gel electrophoresis, cut with appropriate restriction enzymes, cloned into pCRblunt (Invitrogen), and sequenced.

## RESULTS AND DISCUSSION

Identification of AdeN, a TetR-like regulator involved in AdeIJK regulation. Sequencing of $300 \mathrm{bp}$ upstream from *adeI*, including a putative promoter for *adeIJK*, did not reveal any mutations between susceptible BM4587 and one-step mutant BM4666 overexpressing the pump (6). Furthermore, no open reading frames (ORFs) coding for regulatory proteins were found in the vicinity of the *adeIJK* operon. To detect a putative transcriptional regulator of *adeIJK*, whole-genome sequencing of the two strains was carried out.

ferred into *A. baumannii* BM4587 by tripartite conjugation using *E. coli* CC118$\lambda$pir(pAT524) as the donor and *E. coli* HB101 containing the “helper” plasmid pRK2013 (Table 1). Deletion mutants were selected on M9 minimal medium plates containing $5 \%$ sucrose, $200 \mu\mathrm{g} / \mathrm{ml}$ streptomycin, and $60 \mu\mathrm{g} / \mathrm{ml}$ chloramphenicol. Deletion of $324 \mathrm{bp}$ in *adeN* following two recombination events was confirmed in the resulting BM4685 mutant by PCR with external primers adeN5v and adeN7v (Table 2; Fig. 2) and sequencing.

### trans-Complementation experiments. Low-copy-number plasmid pAT801 (10), conferring resistance to ampicillin and in which gene expression is under the control of the IPTG (isopropyl-$\beta$-thiogalactopyranoside)-inducible promoter Plac, was used as an *A. baumannii-E. coli* shuttle vector. A fragment containing the putative ribosome-binding site (RBS) region and either the entire *adeN* gene of BM4587 or the mutated *adeN$\Delta C_{582}$* gene of BM4666 was amplified with primers adeNAYE3 and adeNAYE4 (Table 2) and cloned in pCR2.1 (Table 1). After BamHI/XbaI restriction, the insert was ligated in BamHI/XbaI-linearized pAT801, giving rise to pAT522 (*adeN*) and to pAT523 (*adeN$\Delta C_{582}$*) (Table 1), which were electrotransformed into *A. baumannii* BM4666 (BM4587 *adeN$\Delta C_{582}$*) and BM4587, respectively. Cultures were then incubated at $37^{\circ} \mathrm{C}$ in BHI containing $100 \mu\mathrm{g} / \mathrm{ml}$ of ampicillin until the optical density at $600 \mathrm{~nm}$ reached $0.5$ to $0.6$ and expression of *adeN* or *adeN$\Delta C_{582}$* was induced for $4 \mathrm{~h}$ at $37^{\circ} \mathrm{C}$ by addition of $0.2$ or $1 \mathrm{mM}$ IPTG.

RNA isolation and qRT-PCR. *A. baumannii* total RNA was isolated from exponentially growing cells (optical density at $600 \mathrm{~nm}$ of $0.8$ to $0.9$) with a FastRNA Blue kit (MPbio). cDNA synthesis and qRT-PCR were performed with a LightCycler RNA amplification kit SYBR green I (Roche Diagnostic GmbH, Mannheim, Germany) and $0.5 \mu \mathrm{M}$ gene-specific primers (adeN6v/adeNAYE8 for *adeN*, adeNM3/adeNM4 for truncated *adeN*, adeJfor/adeJrev for *adeJ*, and rpoBfor/rpoBrev for *rpoB*) (Table 2), according to the manufacturer’s instructions. Amplification and detection of specific products were performed using the LightCycler sequence detection system (Roche) with the following cycle profile: $1$ cycle at $95^{\circ} \mathrm{C}$ for $30 \mathrm{~s}$, followed by $45$ cycles at $95^{\circ} \mathrm{C}$ for $5 \mathrm{~s}, 54^{\circ} \mathrm{C}$ for $10 \mathrm{~s}$, and $72^{\circ} \mathrm{C}$ for $20 \mathrm{~s}$. Each experiment was performed in duplicate.

![Diagram](#)

FIG 1 Insertion-inactivation of *adeN*. (A) Plasmid pAT524 ($\Delta adeN$), possessing a truncated portion of *adeN*, underwent homologous recombination in the *glnG-adeN-argF* fragment, leading to (B) integration in the *A. baumannii* chromosome. (C) A second crossover event led to loss of the entire *adeN* and of the plasmid, resulting in a truncated *adeN* gene (D).

Regulation of AdeIJK Efflux Pump in A. baumannii

$\alpha 1$ $\alpha 2$ $\alpha 3$ $\alpha 4$ $\alpha 5$ $\alpha 6$

BM4587 | MHDPYLESHHLVCEKPQTRRGIERLLALLLSATELFLEKGYDAVSLDDIVNHAGGSKTSIYKYFGNKDGLFTAICDYRREMFFKDICAFOPEQTSLKDYLIOQLIRFYKHIIQPEHIAFLRLVI
BM4666 | MHDPYLESHHLVCEKPQTRRGIERLLALLLSATELFLEKGYDAVSLDDIVNHAGGSKTSIYKYFGNKDGLFTAICDYRREMFFKDICAFOPEQTSLKDYLIOQLIRFYKHIIQPEHIAFLRLVI

$\alpha 7$ $\alpha 8$ $\alpha 9$

BM4587 | EQTQCNATLSQYLVEKCALDVQNTIAQALLISHQSGEITCTSPDHSSLMYFGILRDIEWRMIMGMPLPPNETEVIDYINYCVDIFLKGGHKV*
BM4666 | EQTQCNATLSQYLVEKCALDVQNTIAQALLISHQSGEITCTSPDHSSLMYFGILRDIEWRMIMGMPLPP + +++ + + F

FIG 2 Alignment of wild-type and mutated AdeN. AdeN from BM4587 and that from BM4666 (BM4587 adeNΔC<sub>582</sub>) are aligned. Cytosine deletion at position 582 of adeN is responsible for a frameshift leading to the replacement of 15 amino acids in the C-terminal part of AdeN (corresponding to positions 195 to 210) and to the presence of a premature stop codon at position 211. α-Helices predicted by using the [http://www.predictprotein.org](http://www.predictprotein.org) website are indicated.

Comparative analysis of the sequence of the contigs with the seven *A. baumannii* sequences deposited in GenBank indicated that the genome of BM4587 was most closely related to that of strain AB307-0294 (1). The latter possesses 199 genes annotated as “putative transcriptional regulators,” most of them belonging to the LysR (31%) or TetR (23%) family. These putative regulators were all found in both strains, and the sequence of every gene was compared with those in BM4587 and BM4666. A mutation was found in the 651-bp gene corresponding to ABBFA_001467 of strain AB307-0294, annotated as “bacterial regulatory protein, TetR family protein.” A cytosine deletion at position 582 in BM4666 introduced an alteration in the amino acid sequence corresponding to positions 195 to 210 and a premature stop codon at position 211, resulting in deletion of the 7 C-terminal residues (Fig. 2). Resequencing of the PCR fragments obtained with oligonucleotides adeN1/adeN2 (Table 2) from BM4587 and BM4666 confirmed cytosine deletion at position 582. The gene, located 813 kbp from adeIJK in AB307-0294, was named adeN.

Inactivation of adeN in susceptible BM4587. To test the role of adeN in the regulation of adeIJK expression, the gene was partially deleted by insertion-inactivation in BM4587 using pKNG101 (13) (Fig. 1). Deletion in the resulting strain, BM4685 (BM4587 ΔadeN), which was more resistant to AdeIJK substrates erythromycin, chloramphenicol, tetracycline, β-lactams, sulfonamides, and quinolones (Table 3), was confirmed by PCR and sequencing. Determination of adeJ expression levels by qRT-PCR indicated an increase of approximately 5-fold (ranging from 3.8- to 6.5-fold) in BM4685 compared with that of BM4587. This is similar to the adeJ expression increase observed in BM4666 (BM4587 adeNΔC<sub>582</sub>) relative to BM4587 (4). These data confirm the contribution of AdeN to the regulation of AdeIJK.

trans-Complementation experiments. To study if AdeN is sufficient to restore antibiotic susceptibility to BM4666 (BM4587 adeNΔC<sub>582</sub>), plasmid pAT522 (adeN) was introduced into that strain by electroporation. In the presence of IPTG (0.2 or 1 mM), the susceptibility of transformant BM4666/pAT522(adeN) to erythromycin, chloramphenicol, tetracycline, β-lactams, sulfonamides, and quinolones was restored to close to the wild-type level (Table 3). In addition, there was no difference in expression of adeJ between BM4666/pAT522(adeN) and BM4587, as determined by qRT-PCR (ratio = 1.16 ± 0.36). These results demonstrate that AdeN is sufficient to prevent overexpression of AdeIJK. trans-Complementation of BM4685 (BM4587 ΔadeN) with pAT522(adeN) increased the susceptibility to the host to close to the level of BM4587 (Table 3). Furthermore, comparison by qRT-PCR of the expression levels of adeJ in BM4685/pAT522(adeN) and in BM4587 showed that they were similar (ratio = 1.26 ± 0.16), confirming the loss of adeJ overexpression in BM4685/pAT522. Electrotransformation of pAT523 (adeNΔC<sub>582</sub>) in mutant strain BM4685 (BM4587 ΔadeN) and in wild-type BM4587 did not alter the phenotype, despite the fact that the plasmid-borne adeNΔC<sub>582</sub> was highly expressed (more than 10<sup>4</sup>-fold) (Table 3). These data confirm that AdeN is required for the repression of AdeIJK expression.

Taken together, these inactivation and trans-complementation experiments indicate that AdeN acts as a repressor of efflux pump AdeIJK expression in BM4587.

Overexpression of AdeIJK in other mutants. Two other adeIJK-overexpressing one-step mutants, BM4668 and BM4669, selected from susceptible strains BM4667 and BM4651 (ΔadeABC), respectively, were studied (Table 2). Sequence analysis of adeN from BM4668 revealed the same mutation as that in BM4666 (BM4587 adeNΔC<sub>582</sub>). The deletion occurred in a highly C-rich (75%) 12-bp sequence (from position 571 to 582) which could be a potential mutational hot spot due to a slippage of the DNA polymerase (23). Amplification of the entire adeN gene could not be obtained from BM4669 total DNA. A PCR product containing 645 bp upstream from adeN and the first 247 bp of the gene was obtained, whereas attempts with various primers located in the 3′ part of the gene were unsuccessful. The sequence of this portion of adeN was determined by TAIL-PCR (15) and showed a deletion encompassing the 394 last nucleotides of adeN, the intergenic region, and the 470 first nucleotides of argF for an ornithine carbamoyltransferase located downstream from adeN. Determination of adeJ expression levels by qRT-PCR showed that adeJ was overexpressed in the mutant compared to the parent (from 2.5- to 5-fold for BM4668/BM4667 [4] and from 3- to 4.8-fold for BM4669/BM4651). The presence of mutation/deletion events in the three one-step in vitro mutants overexpressing adeIJK constitutes additional proof for the role of AdeN in the regulation of AdeIJK expression.

Promoter analysis of adeN. The transcription start point

TABLE 3 Antibiotic susceptibility of A. baumannii strains

| Antimicrobial agent | BM4587<sup>b</sup> | BM4666 (BM4587) | BM4666/pAT 522 | BM4685 (BM4587) | BM 4685/pAT 522 |
|---------------------|-------------------|------------------|---------------|------------------|-----------------|
|                     |                   | adeNΔC<sub>582</sub><sup>c</sup> | (adeN)<sup>d</sup> | ΔadeN)<sup>e</sup> | (adeN)<sup>d</sup> |
| Erythromycin       | 6                 | 32               | 6             | 24               | 4               |
| Chloramphenicol    | 64                | >256             | 32            | 128              | 24              |
| Tetracycline       | 3                 | 8                | 1.5           | 4                | 0.75            |
| Minocycline        | 0.094             | 0.38             | 0.094         | 0.38             | 0.094           |
| Tigecycline        | 0.064             | 0.25             | 0.094         | 0.25             | 0.064           |
| Rifampin           | 1.5               | 3                | 3             | 8                | 4               |
| Ciprofloxacin      | 0.125             | 0.38             | 0.25          | 0.5              | 0.125           |
| Norfloxacin        | 1.5               | 6                | 3             | 6                | 3               |
| Trimethoprim       | 24                | >32              | 16            | >32              | 24–32           |
| Sulfamethoxazole   | 4                 | 8                | 3             | 16               | 6               |
| Co-trimoxazole     | 0.19              | 0.38             | 0.125         | 0.75             | 0.25            |
| Ticarcillin        | 8                 | 16               | NA<sup>f</sup> | 64               | NA              |
| Aztreonam          | 12                | 48               | 16            | 32               | 12              |
| Cefepime           | 1                 | 3                | 2             | 3                | 1.5             |
| Ceftazidime        | 2                 | 6                | 2             | 3                | 1.5             |
| Imipenem           | 0.25              | 0.25             | 0.25          | 0.25             | 0.25            |
| Ertapenem          | 3                 | 12               | 3             | 6                | 1.5             |
| Meropenem          | 0.25              | 1                | 0.38          | 0.75             | 0.19            |

<sup>a</sup> MICs determined by Etest.

<sup>b</sup> Wild-type clinical isolate.

<sup>c</sup> Mutant of BM4587 overexpressing adeIJK.

<sup>d</sup> Strain trans-complemented with plasmid pAT522 (adeN).

<sup>e</sup> Derivative of BM4587 deleted for adeN and overexpressing adeIJK.

<sup>f</sup> NA, not assessed because of the presence of an ampicillin resistance cassette.

TABLE 4 Amino acid identity (%) between Ade proteins of *A. baumannii* and other Acinetobacter species

| % Amino acid identity with corresponding protein of: | A. calcoaceticus | A. nosocomialis | A. pittii | A. junii | A. haemolyticus | A. parvus | A. radioresistens | A. ursingii | A. baylyi | A. johnsonii | A. lwoffii |
|----------------------------------------------------|------------------|------------------|-----------|----------|-----------------|------------|-------------------|--------------|-------------|---------------|-------------|
| *A. baumannii* protein                              |                  |                  |           |          |                 |            |                   |              |             |               |             |
| AdeN                                               | 91               | 79               | 63        | 61       | 61              | 60         | 59                | 57           | 53          | 53            |             |
| AdeI                                               | 96               | 99               | 78        | 87       | 74              | 76         | 82                | 81           | 78          | 72            |             |
| AdeJ                                               | 97               | 99               | 86        | 91       | 84              | 86         | 77                | 89           | 86          | 84            |             |
| AdeK                                               | 96               | 97               | 85        | 89       | 74              | 82         | 87                | 88           | 81          | 77            |             |

for improving reverse genetics in gram-negative bacteria: inactivation of the *blaA* gene of *Yersinia enterocolitica*. Gene 109:137–141.

14. Lacks S, Greenberg B. 1977. Complementary specificity of restriction endonucleases of *Diplococcus pneumoniae* with respect to DNA methylation. J. Mol. Biol. 114:153–168.

15. Liu YG, Whittier RF. 1995. Thermal asymmetric interlaced PCR: automatable amplification and sequencing of insert end fragments from P1 and YAC clones for chromosome walking. Genomics 25:674–681.

16. Magnet S, Courvalin P, Lambert T. 2001. Resistance-nodulation-cell division-type efflux pump involved in aminoglycoside resistance in *Acinetobacter baumannii* strain BM4454. Antimicrob. Agents Chemother. 45: 3375–3380.

17. Manoil C, Beckwith J. 1985. *TnphoA*: a transposon probe for protein export signals. Proc. Natl. Acad. Sci. U. S. A. 82:8129–8133.

18. Marchand I, Damier-Piolle L, Courvalin P, Lambert T. 2004. Expression of the RND-type efflux pump AdeABC in *Acinetobacter baumannii* is regulated by the AdeRS two-component system. Antimicrob. Agents Chemother. 48:3298–3304.

19. Nikaido H. 2011. Structure and mechanism of RND-type multidrug efflux pumps. Adv. Enzymol. Relat. Areas Mol. Biol. 77:1–60.

20. Pareja E, et al. 2006. ExtraTrain: a database of extragenic regions and

transcriptional information in prokaryotic organisms. BMC Microbiol. 6:29.

21. Peleg AY, Adams J, Paterson DL. 2007. Tigecycline efflux as a mechanism for nonsusceptibility in *Acinetobacter baumannii*. Antimicrob. Agents Chemother. 51:2065–2069.

22. Ramos JL, et al. 2005. The TetR family of transcriptional repressors. Microbiol. Mol. Biol. Rev. 69:326–356.

23. Rocha EP, Pradillon O, Bui H, Sayada C, Denamur E. 2002. A new family of highly variable proteins in the *Chlamydophila pneumoniae* genome. Nucleic Acids Res. 30:4351–4360.

24. Sambrook J, Russell W. 2001. Molecular cloning: a laboratory manual, 3rd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

25. Szybalski W, Bryson V. 1952. Genetic studies on microbial cross resistance to toxic agents. I. Cross resistance of *Escherichia coli* to fifteen antibiotics. J. Bacteriol. 64:489–499.

26. Yanisch-Perron C, Vieira J, Messing J. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.

27. Yu Z, Reichheld SE, Savchenko A, Parkinson J, Davidson AR. 2010. A comprehensive analysis of structural and sequence conservation in the TetR family transcriptional regulators. J. Mol. Biol. 400:847–864.
