
The structural biology of HIV-1: mechanistic and therapeutic insights

**Alan Engelman**¹ **and Peter Cherepanov**²

Abstract | Three-dimensional molecular structures can provide detailed information on biological mechanisms and, for cases in which the molecular function affects human health, can significantly aid in the development of therapeutic interventions. For almost 25 years, key components of the lentivirus HIV-1, including the envelope glycoproteins, the capsid and the replication enzymes reverse transcriptase, integrase and protease, have been scrutinized to near atomic-scale resolution. Moreover, structural analyses of the interactions between viral and host cell components have yielded key insights into the mechanisms of viral entry, chromosomal integration, transcription and egress from cells. Here, we review recent advances in HIV-1 structural biology, focusing on the molecular mechanisms of viral replication and on the development of new therapeutics.

---

**Zoonotic**  
Pertaining to a disease: an infection that can transfer between animals and humans.

**CD4⁺ T cells**  
A subpopulation of T cells that express the CD4 receptor. These cells aid in immune responses and are therefore also referred to as T helper cells.

HIV-1 arose through several independent zoonotic transmissions of simian immunodeficiency viruses during the past century¹⁻³. Today, HIV-1, along with its less widespread cousin HIV-2, infects more than 30 million people worldwide. Both viruses belong to the *Retroviridae*, a viral family that has left numerous scars of ancient infections in mammalian genomes; indeed, derelict retroviral sequences constitute as much as 8% of our ‘own’ DNA⁴. The evolutionary success of this family is in contrast to its deceptive simplicity: HIV-1 can persistently infect humans by subverting the innate and adaptive immune systems, despite encoding only 15 mature proteins. Viral replication at the cellular level proceeds through a series of steps that starts when a virus productively engages cell surface receptors and ends when nascent particles mature into infectious virions (FIG. 1). During this process, HIV-1 exploits a myriad of cellular factors to replicate, whereas host restriction factors fight to suppress this replication⁵,⁶. The mainstream highly active antiretroviral therapy (HAART) drug cocktails that are primarily used to target the reverse transcriptase (RT) and protease (PR) enzymes potently suppress viral loads and transmission rates, but complications can arise from compound toxicity and the emergence of resistant strains (BOX 1). Advances in structural biology can aid the development of next-generation compounds that are active against previously exploited targets, and can also help define new drug targets and boost the effectiveness of vaccination strategies. This Review proceeds step-wise through the HIV-1 replication cycle, highlighting the impact that major structural biology advances have had on our understanding of viral growth and on the development of new antiretroviral therapies.

---

**Viral entry**

The HIV-1 envelope spikes comprise trimers of non-covalently linked heterodimers consisting of the surface glycoprotein gp120 and the transmembrane glycoprotein gp41 (REFS 7–9). When triggered, these spikes initiate a cascade of conformational changes that culminates in fusion between the viral and host cell membranes and release of the viral core into the cytoplasm. HIV-1 primarily infects CD4⁺ T cells and macrophages. An initial interaction between gp120 and the surface receptor CD4 induces the formation of a bridging sheet between the inner and outer domains of the gp120 monomer, exposing the binding site for a second cell surface molecule, typically CC-chemokine receptor 5 (CCR5)¹⁰⁻¹² (FIG. 1, step 1). Engagement of this co-receptor leads to insertion of the fusion peptide, located at the amino terminus of gp41, into the cell membrane. This event triggers significant rearrangements of the trimerized amino- and carboxy-terminal heptad repeat sequences within gp41, the formation of a six-helix hairpin structure and the apposition and fusion of the viral and host cell membranes¹³⁻¹⁵ (FIG. 1, step 2).

Initial cryo-electron tomography studies provided crucial glimpses of the HIV-1 envelope and its associated conformational flexibility⁷,⁸, although the low-resolution models that were generated left many key aspects of the native structure unresolved⁹,¹⁶,¹⁷. Higher-resolution crystallographic studies using engineered HIV-1

---

**¹ Department of Cancer Immunology and AIDS,**  
Dana-Farber Cancer Institute, Boston, Massachusetts 02215, USA.  

**² Cancer Research UK London Research Institute,** Clare Hall Laboratories, South Mimms, Hertfordshire EN6 3LD, UK.  

e-mails: alan_engelman@dfci.harvard.edu; peter.cherepanov@cancer.org.uk  

doi:10.1038/nrmicro2747

REVIEW S

![Diagram](attachment:diagram.png)

Figure 1 | Schematic overview of the HIV-1 replication cycle. Those host proteins that have a role in the replication cycle and are discussed in the text are indicated. The infection begins when the envelope (Env) glycoprotein spikes engage the receptor CD4 and the membrane-spanning co-receptor CC-chemokine receptor 5 (CCR5) (step 1), leading to fusion of the viral and cellular membranes and entry of the viral particle into the cell (step 2). Partial core shell uncoating (step 3) facilitates reverse transcription (step 4), which in turn yields the pre-integration complex (PIC). Following import into the cell nucleus (step 5), PIC-associated integrase orchestrates the formation of the integrated provirus, aided by the host chromatin-binding protein lens epithelium-derived growth factor (LEDGF) (step 6). Proviral transcription (step 7), mediated by host RNA polymerase II (RNA Pol II) and positive transcription elongation factor b (P-TEFb), yields viral mRNAs of different sizes, the larger of which require energy-dependent export to leave the nucleus via host protein CRM1 (step 8). mRNAs serve as templates for protein production (step 9), and genome-length RNA is incorporated into viral particles with protein components (step 10). Viral-particle budding (step 11) and release (step 12) from the cell is mediated by ESCRT (endosomal sorting complex required for transport) complexes and ALIX and is accompanied or soon followed by protease-mediated maturation (step 13) to create an infectious viral particle. Each step in the HIV-1 life cycle is a potential target for antiviral intervention; the sites of action of clinical inhibitors (white boxes) and cellular restriction factors (blue boxes) are indicated. INSTI, integrase strand transfer inhibitor; LTR, long terminal repeat; NNRTI, non-nucleoside reverse transcriptase inhibitor; NRTI, nucleoside reverse transcriptase inhibitor.

glycoprotein constructs have been instrumental in developing entry inhibitors and elucidating the mechanistic basis of virus neutralization by antibodies. Recent studies have highlighted the striking flexibility of the core gp120 structure, which allows extreme conformational changes following CD4 engagement without destabilizing the interaction with gp41 (REFS 12,18). CD4 binds gp120 at a depression formed between the inner and outer domains, where the CD4 residue Phe43 partially fills a hydrophobic cavity (FIG. 2a). Small molecules designed to bind and extend further into this pocket display antiviral activity; thus, increasing the affinity of such molecules for gp120 might lead to the development of clinically useful inhibitors.

Most antibodies directed against gp120 are strain specific and, moreover, fail to neutralize the virus.

However, several groups recently described patient-derived gp120-reactive antibodies with broad HIV-1 neutralization activity. One group in particular took a structure-based approach to stabilize the CD4-bound conformation of gp120 using disulphide bonds, and redesigned its surface to mask positions that are exterior to the CD4-binding site. Using one such construct as bait and peripheral mononuclear cells from patients with AIDS, they isolated B cell clones that produced antibodies with remarkably broad neutralizing activity. Structural characterization of these antibodies revealed that, when bound to gp120, the heavy chains of the immunoglobulins mimic CD4 (FIG. 2a,b), with their epitopes almost precisely overlapping the primary CD4-binding site on gp120 (REFS 22,25). These results define the structural basis

Cryo-electron tomography  
A technique in which a specimen, embedded in vitreous ice, is imaged from multiple angles using electron microscopy. The resulting images are then combined to reconstruct the three-dimensional structure of the specimen.

Box 1 | Highly active antiretroviral therapy

Approximately 30 different drugs targeting four different steps in the HIV-1 replication cycle are currently approved for administration to individuals who are HIV positive in the United States (see http://www.aidsmeds.com/list.shtml). Nucleoside reverse transcriptase inhibitors (NRTIs) and non-nucleoside reverse transcriptase inhibitors (NNRTIs) target the reverse transcription step that converts the viral genomic RNA into linear double-stranded DNA, whereas protease inhibitors inhibit the protease activity that is critical for the maturation of viral particles which bud from infected cells. Two different inhibitors can block entry of the virus into new target cells by thwarting either the interaction between the viral envelope glycoprotein gp120 and the co-receptor CC-chemokine receptor 5 (CCR5) (the inhibitor maraviroc) or the formation of the six-helix bundle of transmembrane glycoprotein gp41, thus blocking fusion between the viral and cellular membranes (the inhibitor enfuvirtide). The sole integrase strand transfer inhibitor (INSTI), raltegravir, blocks the strand transfer activity of integrase, which is required for insertion of viral DNA into a host cell chromosome. Highly active antiretroviral therapy (HAART) routinely prescribes an NRTI, an NNRTI and a protease inhibitor as a single pill or in various pill combinations. This combinatorial approach to drug treatment significantly suppresses the probability of selection for, and resulting outgrowth of, resistant HIV-1 strains that quickly arise during monotherapy.

for HIV-1 neutralization by antibodies that engage the CD4-binding site. Interestingly, immunoglobulins isolated from the sera of different donors using the resurfaced gp120 construct were derived from the same precursor heavy-chain gene (IGHV1-2*02), which had subsequently undergone extensive affinity maturation ${ }^{21,22,25}$. The requirement for extensive somatic mutation to achieve viral neutralization ${ }^{21,22}$ might pose a challenge for the experimental elicitation of such antibodies. However, the recent discovery of highly potent gp120-binding antibodies with alternative modes of action suggests that there are multiple genetic pathways to achieve cross-clade HIV-1 neutralization ${ }^{20,23,24}$. These results should encourage attempts to design immunogens that elicit humoral immunity for vaccination purposes.

Peptides derived from gp41 N-terminal ${ }^{26}$ or C-terminal ${ }^{27}$ sequences, which disrupt formation of the six-helix bundle and hence membrane fusion, possess potent antiviral activity. Enfuvirtide, a peptide based on the C-terminal sequence, was licensed as Fuzeon (Roche) in 2003, although the requirement for twice-daily injections combined with the frequent appearance of resistance mutations in gp41 have limited its utility. D-peptides that target a pocket at the base of the gp41 N-terminal helical structure are also potent antivirals and may overcome some of the limitations associated with Fuzeon use ${ }^{28}$.

### Post-entry events: uncoating to integration

The HIV core, which houses the replication enzymes RT and integrase (IN) as well as the viral genomic RNA, is encased by a cone-shaped shell ${ }^{29}$ composed of the viral capsid (CA) protein. Recent work has revealed the interactions that occur among individual CA molecules and underlie the structural integrity and functionality of the protective shell ${ }^{30-32}$.

#### Uncoating

Partial CA shell dissolution, which is required for reverse transcription ${ }^{33,34}$, is a recently verified therapeutic target ${ }^{35}$ (FIG. 1, step 3). Moreover, the underlying features of the assembled shell seem to

---

Fullerene  
A closed-shell molecule consisting of hexagonal and pentameric rings of carbon.

Figure 2 | Binding of CD4 and a CD4-mimicking antibody to the gp120 core. **a** | The structure of the HIV-1 glycoprotein gp120 in complex with cellular CD4 (Protein Data Bank (PDB) accession 3JWD). Only immunoglobulin-like domain 1 (D1) of CD4 is shown; the Phe43 side chain is depicted as sticks. **b** | The VRC01 antibody–gp120 co-crystal structure (PDB accession 3NGB), oriented as in part **a**. Only the variable domains of the heavy ($V_{\mathrm{H}}$) and light ($V_{\mathrm{L}}$) chains of the antibody are shown.

determine its propensity to uncoat ${ }^{32}$. CA, which comprises independently folded N-terminal and C-terminal domains (NTD and CTD, respectively) connected by a flexible linker ${ }^{36,37}$, can assemble into ring structures containing five or six protomers ${ }^{31,32}$ (FIG. 3a,b). The rings further congregate to form a fullerene-like cone that is composed predominantly of hexamers, but also contains seven pentamers at the wide end and five at the narrow end. This arrangement produces shape declinations ${ }^{32,38}$ (FIG. 3c), and the flexibility of intramolecular NTD–CTD and intermolecular CTD–CTD interactions

REVIEW S

a
![Image](image1.png)

b
![Image](image2.png)

c
![Image](image3.png)

d
![Image](image4.png)

Figure 3 | HIV-1 capsid structures. **a** | The crystal structure of the hexameric full-length HIV-1 capsid (CA) protein assembly (Protein Data Bank (PDB) accession 3H47). Individual subunits are coloured by chain, with the amino- and carboxy-terminal domains (NTD and CTD, respectively) of each subunit indicated. **b** | The crystal structure of the pentameric full-length HIV-1 CA assembly (PDB accession 3P05). **c** | Stereo view of the model for the complete HIV-1 capsid, based on the crystal structures³². NTDs of the hexameric and pentameric CA units are shown in blue and yellow, respectively; CTDs are green. **d** | The HIV-1 CANTD in complex with PF-3450074 (PDB accession 2XDE). The orientation is a 100° rotation compared with the blue NTD in part **a**. Residues that are crucial for PF-3450074 binding, as revealed by resistance mutations⁴⁴, are indicated.

further contributes to the curvature of the shell lattice³⁰,³² (FIG. 3a,b). The high concentration of pentameric declinations that is expected at the narrow end of the cone may also serve to initiate uncoating³².

TRIM5α, a potent HIV-1 restriction factor isolated from rhesus macaques³⁹, recognizes the assembled CA structure to accelerate uncoating⁴⁰ and activate innate immune signalling pathways⁴¹. A replacement of the N-terminal RING domain of rhesus TRIM5α with that from the related human protein TRIM21 yielded a chimaera that was amenable to recombinant techniques⁴². The hybrid construct formed two-dimensional hexameric crystalline arrays in the presence of a higher-order six-fold lattice of HIV-1 CA⁴³. Such CA-templated multimerization may underlie functional HIV-1 restriction by rhesus TRIM5α through a pattern recognition mechanism, a common feature of other components of the innate immune system⁴¹. Stimulation of premature uncoating could also be a useful therapeutic approach; for example, PF-3450074, a small-molecule inhibitor of HIV-1 replication that binds to a pocket within the NTD of CA (FIG. 3d), may work by triggering premature uncoating through destabilization of CA–CA interactions³⁵,⁴⁴.

Viral DNA synthesis. Reverse transcription and integration of the resultant linear viral DNA molecule into a host cell chromosome occur within the context of nucleoprotein complex structures that are derived from the viral core (FIG. 1, steps 4–6). High-resolution HIV-1 RT structures have been available for a number of years, with initial drug- and nucleic acid template-bound crystal structures reported nearly two decades ago⁴⁵,⁶. HIV-1 RT is a heterodimer composed of p66 and p51 subunits, with p66 harbouring two functional active sites: an N-terminal RNA- and DNA-dependent DNA polymerase and a C-terminal RNase H that digests the RNA component of RNA–DNA hybrids. The polymerase domain resembles a right hand with four subdomains: fingers, thumb, palm and connection⁴⁵⁻⁴⁸ (FIG. 4a). During DNA polymerization, Mg²⁺ cations coordinated by the catalytic residues Asp110, Asp185 and Asp186 from the palm subdomain activate the DNA primer 3′-hydroxyl group and stabilize the hypothetical pentavalent α-phosphorus intermediate state within the substrate 2′-deoxyribonucleoside 5′-triphosphate (dNTP), incorporating the nucleotide into the growing DNA chain and liberating free pyrophosphate⁴⁸ (FIG. 4b).

Two classes of antiviral drug, nucleoside and non-nucleoside RT inhibitors (NRTIs and NNRTIs, respectively), inhibit DNA polymerization and are core components of HAART (BOX 1). Following phosphorylation in infected cells, NRTIs mimic natural dNTPs and are incorporated into the viral DNA by RT. Because they lack the 3′-hydroxyl group that is needed for incorporation of the subsequent nucleotide, NRTIs act as chain terminators, and viral resistance to some of these small molecules occurs through drug exclusion mechanisms. For instance, mutations of Met184 (to Val or Ile) selectively preclude the binding of oxathiolane-containing inhibitors such as 3TC (2′,3′-dideoxy-3′-thiacytidine) over dNTPs with normal deoxyribose rings⁴⁸,⁴⁹ (FIG. 4b). However, resistance to 3′-azido-3′-deoxythymidine (AZT) and other thymidine analogues puzzled researchers for some time: inexplicably, the mutant RT from AZT-resistant virus strains efficiently incorporates AZT monophosphate into the viral DNA⁵⁰. Instead of preventing incorporation, the mutant enzyme developed the ability to excise the incorporated drug from the primer strand. Remarkably, RT accomplishes this by utilizing ATP as a pyrophosphate donor to excise the incorporated drug in the form of an AZT–adenosine tetraphosphate adduct, regenerating an active 3′-hydroxyl primer terminus in a reaction that is mechanistically equivalent to the reversal of the polymerization step⁵¹,⁵². Recent structural analyses revealed that the AZT resistance mutations Lys70Arg, Thr215Tyr and Lys219Gln create an optimal ATP-binding site between the fingers and palm subdomains of RT to promote the excision reaction⁵³ (FIG. 4c).

NNRTIs are allosteric inhibitors that induce the formation of a flexible binding pocket through large conformational changes involving Tyr181, Tyr188 and the primer grip (residues 227–235 within the palm subdomain)⁵⁵,⁵⁴,⁵⁵ (FIG. 4d). The mechanistic basis of inhibition may be due to displacement of the primer grip⁵⁶ or the three-stranded β-sheet that contains the catalytic triad⁵⁵,⁵⁷. Stacking interactions between the aromatic

Figure 4 | Structural analyses of HIV-1 reverse transcriptase function and its inhibition by small molecules.

a | Overview of the HIV-1 reverse transcriptase (RT)–template–primer complex (Protein Data Bank (PDB) accession 1RTD). The subdomains of the active RT subunit are indicated (the fingers, thumb, palm and connection domains of the amino-terminal polymerase, and the RNase H domain at the carboxyl terminus); p51 is the inactive RT subunit. The structure contains a bound molecule of dTTP (shown as sticks) in the active site. Grey spheres are Mg²⁺ ions.

b | Close-up of the RT active site (PDB accession as in part a) and DNA polymerization. The 3′-hydroxyl group, absent in the original structure⁴⁸, is added for illustration purposes. The direction of nucleophilic attack is indicated by a dashed arrow. The primer, dTTP, Met184 (mutation of which results in resistance to oxathiolane-containing inhibitors), the catalytic residues and the leaving pyrophosphate group (PPᵢ) are shown as sticks. RT chains are coloured as in part a.

c | Stereo view of the ATP-binding pocket in 3′-azido-3′-deoxythymidine (AZT)-resistant HIV-1 RT (PDB accession 3KLE). The excision product (AZT–adenosine tetraphosphate (AZTppppA′)) is shown as sticks, with carbon atoms in grey. Protein chains are semitransparent surfaces (colouring as in part a); residues implicated in AZT resistance are indicated.

d | Stereo view of TMC-278 (rilpivirine; shown as sticks with carbon atoms in grey) bound to HIV-1 RT (PDB accession 2ZD1). RT residues forming the binding pocket for the non-nucleoside RT inhibitor are indicated.

side chains of Tyr181 and Tyr188 and first-generation NNRTIs such as nevirapine contribute considerably to drug binding⁴⁵, whereas the associated mutations confer resistance as a result of loss of aromatic chemistry⁵⁸. Lys103Asn is also widely associated with NNRTI resistance, and the Asn103–Tyr188 interaction in the mutant RT seems to restrict the movement of Tyr188 that is required for drug binding⁵⁹,⁶⁰. The more recently developed diarylpyrimidine NNRTIs, TMC-125 (also known as etravirine) and TMC-278 (also known as rilpivirine), retain potency in the face of first-generation NNRTI resistance mutations because their inherent flexibility contributes substantially to high-affinity binding to the mutant RT⁶¹ (FIG. 4d).

Reverse transcription is inhibited by the cellular restriction factor APOBEC3G, a virion-incorporated cytidine deaminase that impedes elongation⁶²,⁶³ and converts nascent cytidines in viral cDNA to uracils⁶⁴⁻⁶⁶. In response, HIV-1 deploys a countermeasure, the protein Vif, which antagonizes the incorporation of APOBEC3G by binding to it and inducing its degradation in virus producer cells⁶⁷,⁶⁸. Such observations highlight the importance of the Vif–APOBEC3G nexus for antiviral drug development, and small molecules that limit Vif-mediated degradation of APOBEC3G, and inhibit HIV-1 infection, have been described⁶⁹,⁷⁰.

APOBEC3G harbours two related domains, each containing cytidine deaminase motifs; the NTD mediates virion incorporation, whereas the CTD is a functional deaminase⁷¹⁻⁷³. Several structures of the CTD, derived from NMR⁷⁴⁻⁷⁶ and X-ray crystallography⁷⁷,⁷⁸, revealed a five-stranded β-sheet intermixed with

REVIEW S

a

![Image](image1.png)

b

![Image](image2.png)

c

![Image](image3.png)

Figure 5 | Retroviral intasome structures and mechanism of integrase catalysis.

a | Overview of the prototype foamy virus (PFV) intasome structure (Protein Data Bank (PDB) accession 3OY9). Viral integrase (IN) forms a dimer-of-dimers structure in which the two inner subunits are the active subunits, and the two outer subunits are catalytically inactive. The transferred viral DNA strand is the strand that harbours the terminal d(C-A) dinucleotide and becomes joined to chromosomal DNA by the action of the IN strand transfer activity. Active-site carboxylates are shown as sticks, and divalent metal ions as grey spheres. b | The PFV intasome in complex with a host DNA mimic (PDB accession 3OS2). IN subunits are shown in space-fill mode. c | DNA strand transfer. The model is based on structures of the Mn²⁺-bound intasome and target capture complex (note that IN binds Mg in vivo; see REF. 84 for details). The Asp and Glu active-site residues (HIV-1 numbering 64, 116 and 152) of IN are shown as yellow sticks. DNA is shown as magenta and blue sticks, and the invariant viral dA and dC nucleotides are indicated. The direction of the nucleophilic attack is indicated by a red dashed arrow.

six helices, with conserved elements of the catalytic zinc coordination motif — (H/C)XEX₂₃⁻²₈PCXC₂ — contributed by a pair of α-helices. These results afford important glimpses into the mechanism of HIV deamination, although additional structures that incorporate the NTD and especially the single-stranded DNA substrate will reveal a more complete picture of catalysis. Structures that include Vif should further aid the development of novel antiviral compounds.

Integration. The viral enzyme IN possesses two catalytic activities: 3′ processing and DNA strand transfer. Each end of the HIV-1 DNA long terminal repeat (LTR) is cleaved adjacent to the invariant dinucleotide sequence d(C-A), unveiling recessed 3′ termini. IN then uses the 3′-hydroxyls to cut chromosomal DNA strands across a major groove while joining the viral DNA ends to the target DNA 5′-phosphates. Host enzymes complete the integration process by repairing the single-strand gaps abutting the unjoined viral DNA 5′ ends, resulting in establishment of a stable provirus (FIG. 1, step 6). IN-mediated reversal of integration is impossible, although rare instances of cell-mediated homologous recombination across the LTRs can excise proviral DNA, leaving a single copy of the LTR behind⁷⁹. Site-specific recombinases can be similarly engineered to excise the HIV-1 provirus ex vivo⁸⁰, although such approaches appear to be far from clinical application.

Although crystal and NMR structures of various fragments of HIV-1 IN were reported over several years⁸¹, detailed views of the functional IN–viral DNA nucleoprotein complex, called the intasome, were lacking until recently. Given that clinically useful HIV-1 IN inhibitors selectively interact with the intasome rather than with free IN⁸², this dearth of structural information limited drug development. Recent successes are due to the application of X-ray crystallography to the tractable intasome of the prototype foamy virus (PFV), a member of the retroviral genus Spumavirus⁸³,⁸⁴. An overview of these advances is given here; for in-depth reviews, see REFS 85,86.

The intasome contains a dimer-of-dimers of IN, with only one subunit of each dimer binding a viral DNA end⁸³ (FIG. 5a,b). Thus, akin to RT, functional IN active sites are delegated to a subset of protein molecules within the multimeric complex. The intasome accommodates the target DNA within a cleft between the functional active sites, in a severely bent conformation (FIG. 5b,c). This contortion in the target DNA allows the intasome active sites (which are separated from one another by as much as 26.5 Å) to access their target scissile phosphodiester bonds⁸⁴. The Asp and Glu residues of the catalytic motif D,DX₁₅E coordinate two divalent metal ions, which activate the 3′-hydroxyI nucleophile and destabilize the target phosphodiester bond during strand transfer⁸³,⁸⁴ (FIG. 5c). Reversal of the reaction appears to be restricted by a conformational change that causes a 2.3 Å displacement of the newly formed viral DNA–target DNA phosphodiester bond from the IN active site following transesterification⁸⁴.

The clinically approved HIV-1 IN inhibitor,
raltegravir, and similar small molecules that are in development preferentially inhibit DNA strand transfer activity; fortuitously, IN strand transfer inhibitors (INSTIs) harbour broad antiretroviral activity^{87–89}. Results based on PFV intasome–INSTI co-crystal structures have been accordingly illuminating. INSTIs harbour two common moieties: co-planar heteroatoms (typically three oxygen atoms) that chelate the active-site metal ions^{90}, and halogenated benzyl groups, the function of which was largely speculative until recently. INSTIs engage the bound metal ions, only slightly influencing their positions within the IN active site. The halogenated benzyl groups of the INSTIs assume the position of the terminal adenine ring, primarily through interactions with the penultimate viral DNA G·C base pair and a 3_{10} helix in IN (Pro145–Gln146 in HIV-1 IN), ejecting the viral 3′-dA (with its associated 3′-hydroxyl nucleophile) from the active site^{83,88}. This displacement of the DNA strand transfer nucleophile forms the mechanistic basis of INSTI action. In addition, INSTIs sterically preclude target DNA binding, explaining the competition between target DNA and these inhibitors^{82,84}. Furthermore, the PFV model has provided important clues about the mechanism of drug resistance associated with HIV-1 IN mutations that are selected in the presence of raltegravir^{88}.

As is the case for RT, there is evidence that a second region of HIV-1 IN, in this case distal from the active site, affords an opportune location for binding of allosteric inhibitors. Lentiviruses such as HIV-1 favour integration within active genes owing to an interaction between IN and the chromatin-binding protein lens epithelium-derived growth factor (LEDGF; also known as transcriptional co-activator p75) (reviewed in REF. 91). The IN-binding domain of LEDGF is a pseudo HEAT analogous topology (PHAT) domain that consists of two units of a helix–hairpin–helix repeat^{92}. The LEDGF hotspot residues Ile365 and Asp366 at the tip of the N-terminal hairpin nestle into a cleft at the dimer interface of the HIV-1 IN catalytic core domain^{93}. A novel class of HIV-1 IN inhibitors that are capable of suppressing viral replication was recently discovered through a remarkable example of structure-based drug design. These small molecules, termed LEDGINs, mimic the LEDGF–IN interaction and inhibit protein–protein binding^{94}. Given the highly conserved nature of INSTI binding at the active site^{88,95} and the likelihood of considerable cross-resistance among INSTIs^{96}, the development of such allosteric HIV-1 IN inhibitors is highly desirable.

### Viral mRNA biogenesis and transport

Integration marks the transition from the early to late phase of HIV-1 replication, in which the focus shifts to viral gene expression followed by the assembly and egress of nascent viral particles. Transcription, which initiates from the U3 promoter within the upstream LTR (FIG. 1, step 7), requires the viral transactivator protein, Tat, for efficient elongation. Viral mRNAs are produced as a variety of alternatively spliced species. The smaller messages are exported readily from the nucleus, whereas the unspliced and singly spliced mRNAs require the action of Rev. This small viral protein acts as an adaptor, binding to the Rev response element (RRE), located within the env mRNA coding region, and to the host nuclear export factor CRM1 (also known as XPO1) (FIG. 1, step 8). Recent structural biology advances have yielded insight into the mechanisms of Tat transactivation^{97} and Rev-dependent mRNA export^{98,99}.

#### Transcription elongation

Tat recruits the cellular protein positive transcription elongation factor b (P-TEFb; comprising cyclin-dependent kinase 9 (CDK9) and cyclin T1) to the viral transactivation response (TAR) element present in viral transcripts^{100,101}. Subsequent CDK9-mediated phosphorylation of the heptad repeat residues Ser2 and Ser5 in the CTD of the large subunit of RNA polymerase II stimulates transcription elongation. Tat is largely unstructured in the absence of binding ligands^{102}. TAR binding occurs primarily via an α-helical Arg-rich motif (ARM), which inserts into the RNA major groove within the stem–loop structure^{103}. The N-terminal activation domain of Tat, which contains acidic, Pro-rich, zinc-binding motifs and core subdomains, assumes an ordered structure on P-TEFb binding^{97}. Within the complex, Tat primarily interacts with the cyclin T1 subunit, also contacting the T loop region of CDK9 (FIG. 6a). Tat binding stimulates CDK9-mediated phosphorylation of Ser2 and Ser5 of RNA polymerase II^{104}. Accordingly, reciprocal conformational changes in the kinase alter the substrate-binding surface of P-TEFb. Crucially, the fact that Tat induces conformational changes in P-TEFb suggests that it should be possible to develop anti-HIV agents which are directed against P-TEFb but have limited side effects on its normal cellular functions^{97}.

#### mRNA export

Rev binds to the RRE in a highly cooperative manner, forming an RNA-dependent dimer en route to a higher-order Rev–RNA multimer^{105,106}. The structural basis for Rev multimerization was recently elucidated by two complementary crystallographic studies^{98,99}. Rev adopts an amphipathic helical hairpin, which multimerizes via face-to-face and back-to-back symmetrical interfaces that are stabilized by conserved hydrophobic interactions (FIG. 6b). Collectively, the crystal structures^{98,99} describe both types of interface and allow modelling of a Rev multimer, which projects pairs of ARMs on one side and C-terminal nuclear export signals for latching onto the cellular nuclear export factor CRM1 on the other (FIG. 6c). The relative orientations of the ARMs in the context of the oligomer are thought to dictate the selectivity of the viral protein for the RRE structure and sequence. The model also accounts for the cooperativity of RNA binding by Rev, although a more complete structure including the RRE will be required to explain the details of protein–RNA recognition.

### Viral egress and maturation

The retroviral structural proteins CA, matrix (MA) and nucleocapsid (NC) are synthesized as parts of the precursor polypeptide Gag, and HIV-1 Gag is sufficient

---

**NMR**

A spectroscopy technique that takes advantage of magnetic properties of nuclei. Applied to structural biology, NMR affords the determination of macromolecular structures by measuring interproton distances.

**NATURE REVIEWS | MICROBIOLOGY**  
**VOLUME 10 | APRIL 2012 | 285**  
© 2012 Macmillan Publishers Limited. All rights reserved
(VPS) proteins, the actions of which are required to form the nascent particle and sever it from the plasma membrane. The intended function of VPS proteins is in the formation of multivesicular bodies, a process that is topologically identical to viral budding, as in each case a membrane-coated vesicle leaves the cytoplasm; VPS proteins also function in abscission during cell division ${ }^{116,117}$. Most class E VPS proteins are subunits of ESCRT (endosomal sorting complex required for transport) complexes, which come in four varieties (ESCRT-0, ESCRT-I, ESCRT-II and ESCRT-III). ESCRT-I and ESCRT-II function during membrane budding, whereas ESCRT-III is important for membrane scission. Recent advances have yielded structures of several class E proteins, as well as the class E protein–L domain interactions that are crucial for viral budding from infected cells (see REFS 118,119 for in-depth reviews).

The C-terminal HIV-1 Gag cleavage product p6 harbours two L domains, P(T/S)AP and LYPX ${}_{1-3}$L (REFS 120,121). The TSG101 component of ESCRT-I engages P(T/S)AP, whereas ALIX (also known as AIP1 and PDCD6IP), itself not formally an ESCRT protein, binds LYPX ${}_{1-3}$L (REFS 121,122). ALIX contains three domains, an N-terminal Bro1 domain, an interior V domain and a C-terminal Pro-rich domain (PRD). Arm 2 of the α-helical V domain interacts with LYPX ${}_{1-3}$L of p6, whereas the boomerang-shaped Bro1 domain and PRD interact with different isoforms of the ESCRT-III protein CHMP4 and with TSG101, respectively ${}^{123-127}$, accounting for the direct link that ALIX provides between ESCRT-I and ESCRT-III ${}^{121,128}$. Highlighting one potential target for the development of inhibitors of HIV-1 budding, the P(T/S)AP domain of p6 inserts into a cleft on the N-terminal UEV domain of TSG101 (REFS 129,130) (FIG. 7).

**Restriction of viral egress.** The type II transmembrane protein tetherin (also known as CD317 and BST2) inhibits the release of budding particles by retaining them on the plasma membrane of the virus producer cell ${}^{131,132}$ (FIG. 1, step 12). Tetherin consists of a short N-terminal cytoplasmic tail followed by a transmembrane region and an ~110-residue ectodomain ending on an amphipathic sequence that reconnects the protein to the plasma membrane ${}^{133}$. The hydrophobic C-terminal peptide of tetherin, initially thought to be a signal for glycosyl phosphatidylinositol modification, may in fact function as a second transmembrane domain ${}^{134}$. The unusual dual membrane-bound topology of tetherin led to several models to explain virus tethering, involving extended or laterally arranged parallel or antiparallel protein dimers at the cell surface ${}^{131}$, and several recent X-ray crystal structures revealed that the ectodomain indeed forms a parallel dimeric α-helical coiled coil ${}^{135-137}$. In addition, the tetherin dimers can further assemble head to head into tetramers via the formation of a four-helix bundle ${}^{136,137}$. However, mutations designed to ablate tetramer formation do not eliminate tetherin function, indicating that tetramerization is not essential for HIV-1 restriction ${}^{137}$. These data highlight the extended ectodomain coiled-coil dimer as the likely virus-tethering unit.

Figure 6 | Higher-order Tat and Rev structures. **a** | Crystal structure of HIV-1 Tat in complex with ATP-bound host positive translation elongation factor b (P-TEFb) (Protein Data Bank (PDB) accession 3MIA). The protein chains are shown as cartoons (left) or in space-fill mode (right). The N lobe, C lobe and T loop of cyclin-dependent kinase 9 (CDK9) are shown. ATP bound to the active site of CDK9 is shown in stick form. Grey spheres are Zn ${}^{2+}$ ions. **b** | Dimeric assemblies of the HIV-1 Rev core observed in crystals (PDB accessions 2X7L and 3LPH). Rev monomers are coloured by chain, with Arg-rich motifs (ARMs) in blue. The crystal structures illustrate two types of Rev–Rev hydrophobic interfaces, one involving Leu12 and Leu60 and the other involving Leu18 and Ile55. **c** | Model of the Rev hexamer based on the dimeric structures, shown in space-fill mode. The oligomer projects RNA-binding ARM domains (blue) on one side, with CRM1-binding nuclear export signals (not resolved in the current structures) emanating from the other side.

for assembly of virus-like particles at the plasma membrane and for budding of these particles from cells ${}^{107}$ (FIG. 1, steps 10 and 11). Through an N-terminal myristic acid ${}^{108,109}$ and conserved basic amino acid residues ${}^{110-112}$, MA contributes to the membrane association of Gag. The differential exposure of the myristic acid, through a process known as the myristyl switch ${}^{113}$, allows Gag to associate preferentially with the plasma membrane rather than with intracellular membranes. The switch can be activated by phosphatidylinositol-4,5-bisphosphate ${}^{114}$, a phospholipid that is concentrated in the inner leaflet of the plasma membrane and interacts directly with MA ${}^{115}$. Several steps along the pathway of HIV-1 assembly and particle release from cells have been targeted for antiviral drug development.

**Viral late domains and the cellular ESCRT machinery.** Retroviral budding is orchestrated by interactions between Pro-rich motifs in Gag, known as late (L) domains, and cellular class E vacuolar protein sorting

cleavage of the CA–SP1 (spacer peptide 1) junction, thus preventing maturation of the viral core<sup>147</sup>. Exposure to bevirimat leads to stabilization of the immature CA lattice in HIV-1 virions<sup>148</sup>. CAP1 is another small molecule that has been reported to elicit abnormal HIV-1 core morphologies<sup>149</sup>. Binding of CAP1 to the CA NTD involves formation of a deep hydrophobic pocket, which serves as a ligand-binding site<sup>150</sup>. The binding mode of CAP1 is therefore very different from that of PF-3450074, which engages a pre-existing pocket on the CA NTD surface<sup>35</sup> (FIG. 3d). It seems likely that the distortion in CA structure that is associated with CAP1 binding interferes with CA hexamer assembly.

Unlike the structures of the viral enzymes discussed above, which were not determined until after initial discoveries of the respective inhibitors, the structure of full-length PR<sup>151–153</sup> was determined several years before the approval of the first clinical inhibitor targeting the enzyme<sup>154</sup>. Accordingly, the development of PR inhibitors has benefited more from structure-based design efforts than the development of other antiretroviral drugs, and readers are directed to REFS 155,156 for historical accounts of the interplay between PR structure and the development of PR inhibitors and resistance mechanisms.

The nine different peptide sequences within Gag and Gag–Pol that are cleaved by PR display limited primary sequence homology. Co-crystallization of six peptide substrates with PR defined a common volume occupied by the substrates (also called the substrate envelope) and indicated that substrate shape rather than primary sequence is a key predictor of functionality<sup>157</sup>. The approved PR inhibitors are competitive inhibitors that bind to the active site of the enzyme and occupy a volume known as the inhibitor envelope. Overlays of PR–inhibitor co-crystal structures identified regions of the inhibitor envelope which protrude beyond the substrate envelope to contact amino acid residues of PR that do not contact substrate residues and that, when changed, confer drug resistance<sup>158</sup>. On the basis of these findings, it was postulated that if PR inhibitors were designed to bind precisely within the substrate envelope (and contact only those residues that are essential for PR function), then resistance mutations would be unfavourable, as they would destroy the functional activity (substrate-binding capacity) of PR. In support of this hypothesis, some novel amprenavir-based compounds do indeed display marginally improved binding profiles to drug-resistant PR compared with their binding profiles to wild-type enzyme *in vitro*<sup>159</sup>. Because compounds with enhanced binding affinities for wild-type PR bind drug-resistant enzymes less well than amprenavir, additional work is required to determine whether substrate envelope-based PR inhibitors will display beneficial profiles against drug-resistant strains in the clinic.

**Conclusions and perspectives**

HIV-1 has been analysed by structural biology techniques more than any other virus, with partial or complete structures known for all 15 of its protein components and additional structures determined for substrate- and host

Figure 7 | Virus–cell interactions and HIV-1 budding. The structure of the UEV domain of TSG101 bound to the P(T/S)AP domain of HIV-1 p6 protein (Protein Data Bank accession 3OBU), in cartoon and space-fill modes. p6 (residues 5–13; PEPTAPPEE) is shown as sticks; the carbon atoms of the core L domain, PTAP, and the flanking regions are orange and yellow, respectively. Some of the key TSG101 residues involved in the interaction are indicated on the right.

Ectodomain residues Ala88 and Gly109, which disfavour coiled-coil packing, probably impart some flexibility to the structure, perhaps facilitating insertion of the terminal anchor into the viral membrane<sup>136</sup>.

HIV-1 Vpu, also a transmembrane protein, counteracts the restriction by tetherin<sup>131,132</sup> through a mechanism that depends on a direct interaction between the viral and host proteins<sup>138,139</sup>. Previously elucidated structures of Vpu fragments yielded limited insight into the mechanism of the Vpu–tetherin interaction, although a recent NMR analysis of lipid membrane-embedded transmembrane peptides indicates a likely antiparallel helix–helix binding interface<sup>140</sup>.

**Protease and virus maturation.** The final step of the viral life cycle, which is mediated by PR and occurs concomitant with or soon after budding, converts immature particles to infectious virions via the proteolysis of the precursor peptides Gag and Gag–Pol to yield the structural components MA, CA and NC, and the enzymes PR, RT and IN<sup>141</sup> (FIG. 1, step 13). Cryo-electron tomography revealed Gag structural rearrangements that occur within immature particles during proteolysis and maturation<sup>142,143</sup>, and characterized cellular sites of HIV-1 budding<sup>144</sup>. Following cleavage of the MA–CA bond, a novel β-hairpin is formed by a salt bridge between the liberated N-terminal Pro1 and Asp51 in CA, triggering core shell assembly<sup>145</sup>. Recent evidence indicates that the morphological transitions occurring during HIV-1 particle assembly and maturation represent druggable targets. A 12-residue peptide, selected in a phage display screen for binding to the HIV-1 CA CTD, was found to potently restrict CA assembly *in vitro*<sup>146</sup>. Bevirimat, a betulinic acid derivative of herbal origin, inhibits HIV-1 replication by specifically blocking PR-mediated

factor-bound complexes. Structural biology will continue  
to have a significant impact on HIV/AIDS research by  
providing high-resolution glimpses of target protein–  
drug complexes and virus–host interactions, such as  
CA–TRIM5α, Vif–APOBEC3G or Vpu–tetherin, and  
this will reveal novel druggable sites. Despite decades of  
research, the interactions between HIV-1 and host pro-  
teins that underlie some steps in the viral life cycle — for  
example, import of the pre-integration complex into the  
nucleus (FIG. 1, step 5) — are only now being illuminated.  
The simian immunodeficiency virus Vpx protein was  
recently shown to counteract SAMHD1, the restriction  
factor that inhibits HIV-1 reverse transcription and infec-  
tion of monocytic cells^{160,161}, indicating that these protein  
complexes could also define new paradigms for antiviral  
drug development.

Further to the ongoing work with PR inhibitors,  
it will be interesting to see whether structure-based  
substrate–inhibitor envelope hypotheses will apply to  
the development of other HIV-1 inhibitors. Because  
NNRTIs form induced-fit binding pockets, they would  
appear to be poor candidates for this technique. The  
tight overlay of multiple bound drugs at the IN active  
site and the similarities in drug positions with the ejected  
terminal adenosine base^{88} hint that INSTIs could be  
another drug class to benefit from such approaches.  
Three-dimensional structures of new drug targets as  
well as inhibitor- or antibody-bound targets will predict-  
ably increase the pace of antiviral development and help  
guide vaccine development efforts^{162,163}. The advent of  
new technologies and improvements in existing meth-  
ods will also significantly influence structural virol-  
ogy. Single-particle electron cryo-microscopy has recently  
yielded near atomic-resolution structures of a number  

of so-called naked viruses, which, unlike HIV-1, lack an  
exterior envelope lipid bilayer^{164}. Although the icosa-  
hedral symmetry underlying these structures greatly  
facilitated their determination, ongoing improvements  
in instrumentation and computational science may  
well yield similar resolution structures for particles that  
possess less inherent symmetry.

The development of HAART has dramatically  
changed the face of the HIV/AIDS epidemic since the  
disease was first recognized 30 years ago. Considered  
virtually a death sentence before the advent of anti-  
retroviral drugs, HIV-1 infection is now a manageable  
chronic disease. Nonetheless, despite these remarkable  
advances, there remains significant room for improve-  
ment. Some of the drugs, in particular the PR inhibi-  
tors, exert toxic side effects. More tolerable antiviral  
regimens could improve patient compliance and con-  
sequently reduce the emergence of resistant strains.  
Although the recently approved INSTI raltegravir is  
relatively non-toxic, the ease by which it selects for  
drug-resistant strains highlights the need for second-  
generation INSTIs with more favourable genetic barri-  
ers to resistance. The development of compounds that  
inhibit functions of less explored drug targets, in par-  
ticular of the accessory HIV-1 proteins and host factors,  
would clearly also be of benefit. The availability and effi-  
cacy of the current arsenal of antiretroviral drugs should  
not be taken for granted. It is important to bear in mind  
that the majority of people infected with HIV do not  
have access to advanced treatment options. Short of an  
effective vaccination strategy, the ongoing race against  
drug resistance can best be won by a sustained effort to  
develop novel, ever more potent and affordable antiviral  
treatments.


### Single-particle electron  
cryo-microscopy  
A technique that averages  
multiple images obtained from  
transmission electron  
microscopy of homogeneous  
particles at cryogenic  
temperatures.


1. Gao, F. *et al.* Origin of HIV-1 in the chimpanzee Pan  
   troglodytes troglodytes. *Nature* **397**, 436–441 (1999).  
2. Korber, B. *et al.* Timing the ancestor of the HIV-1  
   pandemic strains. *Science* **288**, 1789–1796 (2000).  
3. Lemey, P. *et al.* Tracing the origin and history of the  
   HIV-2 epidemic. *Proc. Natl Acad. Sci. USA* **100**,  
   6588–6592 (2003).  
4. Lander, E. S. *et al.* Initial sequencing and analysis of  
   the human genome. *Nature* **409**, 860–921 (2001).  
5. Evans, D. T., Serra-Moreno, R., Singh, R. K. & Guatelli,  
   J. C. BST-2/tetherin: a new component of the innate  
   immune response to enveloped viruses. *Trends*  
   *Microbiol.* **18**, 388–396 (2010).  
6. Huthoff, H. & Towers, G. J. Restriction of retroviral  
   replication by APOBEC3G/F and TRIM5α. *Trends*  
   *Microbiol.* **16**, 612–619 (2008).  
7. Zhu, P. *et al.* Distribution and three-dimensional  
   structure of AIDS virus envelope spikes. *Nature* **441**,  
   847–852 (2006).  
8. Zanetti, G., Briggs, J. A., Grunewald, K., Sattentau,  
   Q. J. & Fuller, S. D. Cryo-electron tomographic  
   structure of an immunodeficiency virus envelope  
   complex *in situ*. *PLoS Pathog.* **2**, e83 (2006).  
9. Liu, J., Bartesaghi, A., Borgnia, M. J., Sapiro, G. &  
   Subramaniam, S. Molecular architecture of native  
   HIV-1 gp120 trimers. *Nature* **455**, 109–113 (2008).  
10. Kwong, P. D. *et al.* Structure of an HIV gp120  
   envelope glycoprotein in complex with the CD4  
   receptor and a neutralizing human antibody. *Nature*  
   **393**, 648–659 (1998).  
11. Rizzuto, C. D. *et al.* A conserved HIV gp120  
   glycoprotein structure involved in chemokine receptor  
   binding. *Science* **280**, 1949–1953 (1998).  

12. Chen, B. *et al.* Structure of an unliganded simian  
    immunodeficiency virus gp120 core. *Nature* **433**,  
    934–941 (2005).  
13. Chan, D. C., Fass, D., Berger, J. M. & Kim, P. S. Core  
    structure of gp41 from the HIV envelope glycoprotein.  
    *Cell* **89**, 263–273 (1997).  
14. Weissenhorn, W., Dessen, A., Harrison, S. C., Skehel,  
    J. J. & Wiley, D. C. Atomic structure of the ectodomain  
    from HIV-1 gp41. *Nature* **387**, 426–430 (1997).  
15. Buzon, V. *et al.* Crystal structure of HIV-1 gp41  
    including both fusion peptide and membrane proximal  
    external regions. *PLoS Pathog.* **6**, e1000880 (2010).  
16. Subramaniam, S. The SIV surface spike imaged by  
    electron tomography: one leg or three? *PLoS Pathog.*  
    **2**, e91 (2006).  
17. Zhu, P., Winkler, H., Chertova, E., Taylor, K. A. & Roux,  
    K. H. Cryoelectron tomography of HIV-1 envelope  
    spikes: further evidence for tripod-like legs. *PLoS*  
    *Pathog.* **4**, e1000203 (2008).  
18. Pancera, M. *et al.* Structure of HIV-1 gp120 with  
    gp41-interactive region reveals layered envelope  
    architecture and basis of conformational mobility.  
    *Proc. Natl Acad. Sci. USA* **107**, 1166–1171 (2010).  
19. Madani, N. *et al.* Small-molecule CD4 mimics interact  
    with a highly conserved pocket on HIV-1 gp120.  
20. Walker, L. M. *et al.* Broad and potent neutralizing  
    antibodies from an African donor reveal a new HIV-1  
    vaccine target. *Science* **326**, 285–289 (2009).  
21. Wu, X. *et al.* Rational design of envelope identifies  
    broadly neutralizing human monoclonal antibodies to  

22. Wu, X. *et al.* Focused evolution of HIV-1 neutralizing  
    antibodies revealed by structures and deep  
    sequencing. *Science* **333**, 1593–1602 (2011).  
23. Walker, L. M. *et al.* Broad neutralization coverage of  
    HIV by multiple highly potent antibodies. *Nature* **477**,  
    466–470 (2011).  
24. Corti, D. *et al.* Analysis of memory B cell responses  
    and isolation of novel monoclonal antibodies with  
    neutralizing breadth from HIV-1-infected individuals.  
    *PLoS ONE* **5**, e8805 (2010).  
25. Zhou, T. *et al.* Structural basis for broad and potent  
    neutralization of HIV-1 by antibody VRC01. *Science*  
    **329**, 811–817 (2010).  
26. Eckert, D. M. & Kim, P. S. Design of potent inhibitors  
    of HIV-1 entry from the gp41 N-peptide region. *Proc.*  
    *Natl Acad. Sci. USA* **98**, 11187–11192 (2001).  
27. Wild, C. T., Shugars, D. C., Greenwell, T. K., McDanal,  
    C. B. & Matthews, T. J. Peptides corresponding to a  
    predictive α-helical domain of human  
    immunodeficiency virus type 1 gp41 are potent  
    inhibitors of virus infection. *Proc. Natl Acad. Sci. USA*  
    **91**, 9770–9774 (1994).  
28. Welch, B. D. *et al.* Design of a potent D-peptide HIV-1  
    entry inhibitor with a strong barrier to resistance.  
    *J. Virol.* **84**, 11235–11244 (2010).  
29. Briggs, J. A. *et al.* The mechanism of HIV-1 core  
    assembly: insights from three-dimensional  
    reconstructions of authentic virions. *Structure* **14**,  
    15–20 (2006).

30. Byeon, I. J. *et al.* Structural convergence between Cryo-EM and NMR reveals intersubunit interactions critical for HIV-1 capsid function. *Cell* **139**, 780–790 (2009).

31. Pornillos, O. *et al.* X-ray structures of the hexameric building block of the HIV capsid. *Cell* **137**, 1282–1292 (2009). X-ray crystal structures reveal how individual monomers of HIV-1 CA interact to form a hexameric ring, the basic building block of the conical shell encasing the components of the viral core.

32. Pornillos, O., Ganser-Pornillos, B. K. & Yeager, M. Atomic-level modelling of the HIV capsid. *Nature* **469**, 424–427 (2011). The crystal structure of CA reveals the basic building block of the HIV-1 capsid pentamer that affords crucial shape declinations to the conical capsid shell. This structure accordingly leads to an atomic-scale model of overall shell structure.

33. Fitzon, T. *et al.* Proline residues in the HIV-1 NH2-terminal capsid domain: structure determinants for proper core assembly and subsequent steps of early replication. *Virology* **268**, 294–307 (2000).

34. Forshey, B. M., von Schwedler, U., Sundquist, W. I. & Aiken, C. Formation of a human immunodeficiency virus type 1 core of optimal stability is crucial for viral replication. *J. Virol.* **76**, 5667–5677 (2002).

35. Blair, W. S. *et al.* HIV capsid is a tractable target for small molecule therapeutic intervention. *PLoS Pathog.* **6**, e1001220 (2010).

36. Gamble, T. R. *et al.* Crystal structure of human cyclophilin A bound to the amino-terminal domain of HIV-1 capsid. *Cell* **87**, 1285–1294 (1996).

37. Gamble, T. R. *et al.* Structure of the carboxyl-terminal dimerization domain of the HIV-1 capsid protein. *Science* **278**, 849–853 (1997).

38. Ganser, B. K., Li, S., Klishko, V. Y., Finch, J. T. & Sundquist, W. I. Assembly and analysis of conical models for the HIV-1 core. *Science* **283**, 80–83 (1999). Electron microscopy of recombinant HIV-1 CA–NC protein multimers leads to the prediction of a fullerene cone structure for the viral capsid shell.

39. Stremlau, M. *et al.* The cytoplasmic body component TRIM5α restricts HIV-1 infection in Old World monkeys. *Nature* **427**, 848–853 (2004).

40. Stremlau, M. *et al.* Specific recognition and accelerated uncoating of retroviral capsids by the TRIM5α restriction factor. *Proc. Natl Acad. Sci. USA* **103**, 5514–5519 (2006).

41. Pertel, T. *et al.* TRIM5 is an innate immune sensor for the retrovirus capsid lattice. *Nature* **472**, 361–365 (2011).

42. Kar, A. K., Diaz-Griffero, F., Li, Y., Li, X. & Sodroski, J. Biochemical and biophysical characterization of a chimeric TRIM21-TRIM5α protein. *J. Virol.* **82**, 11669–11681 (2008).

43. Ganser-Pornillos, B. K. *et al.* Hexagonal assembly of a restricting TRIM5α protein. *Proc. Natl Acad. Sci. USA* **108**, 534–539 (2011).

44. Shi, J., Zhou, J., Shah, V. B., Aiken, C. & Whitby, K. Small-molecule inhibition of human immunodeficiency virus type 1 infection by virus capsid destabilization. *J. Virol.* **85**, 542–549 (2011).

45. Kohlstädt, L. A., Wang, J., Friedman, J. M., Rice, P. A. & Steitz, T. A. Crystal structure at 3.5 Å resolution of HIV-1 reverse transcriptase complexed with an inhibitor. *Science* **256**, 1783–1790 (1992). The X-ray crystal structure of the HIV-1 RT heterodimer reveals the asymmetrical nature of the protein complex and the binding site for the NNRTIs, such as nevirapine.

46. Jacobo-Molina, A. *et al.* Crystal structure of human immunodeficiency virus type 1 reverse transcriptase complexed with double-stranded DNA at 3.0 Å resolution shows bent DNA. *Proc. Natl Acad. Sci. USA* **90**, 6320–6324 (1993). The crystal structure of HIV-1 RT reveals the positioning of template nucleic acid.

47. Rodgers, D. W. *et al.* The structure of unliganded reverse transcriptase from the human immunodeficiency virus type 1. *Proc. Natl Acad. Sci. USA* **92**, 1222–1226 (1995).

48. Huang, H., Chopra, R., Verdine, G. L. & Harrison, S. C. Structure of a covalently trapped catalytic complex of HIV-1 reverse transcriptase: implications for drug resistance. *Science* **282**, 1669–1675 (1998). The X-ray crystal structure of the RT heterodimer with covalently trapped nucleic acid template, primer and dNTP reveals the mechanism of DNA polymerization. Protein–nucleic acid covalent linkage has since been adopted as a standard technique in the field.

49. Sarafianos, S. G. *et al.* Lamivudine (3TC) resistance in HIV-1 reverse transcriptase involves steric hindrance with β-branched amino acids. *Proc. Natl Acad. Sci. USA* **96**, 10027–10032 (1999).

50. Lacey, S. F. *et al.* Biochemical studies on the reverse transcriptase and RNase H activities from human immunodeficiency virus strains resistant to 3′-azido-3′-deoxythymidine. *J. Biol. Chem.* **267**, 15789–15794 (1992).

51. Arion, D., Kaushik, N., McCormick, S., Borkow, G. & Parniak, M. A. Phenotypic mechanism of HIV-1 resistance to 3′-azido-3′-deoxythymidine (AZT): increased polymerization processivity and enhanced sensitivity to pyrophosphate of the mutant viral reverse transcriptase. *Biochemistry* **37**, 15908–15917 (1998).

52. Meyer, P. R., Matsuura, S. E., Mian, A. M., So, A. G. & Scott, W. A. A mechanism of AZT resistance: an increase in nucleotide-dependent primer unblocking by mutant HIV-1 reverse transcriptase. *Mol. Cell* **4**, 35–43 (1999).

53. Tu, X. *et al.* Structural basis of HIV-1 resistance to AZT by excision. *Nature Struct. Mol. Biol.* **17**, 1202–1209 (2010).

54. Tantillo, C. *et al.* Locations of anti-AIDS drug binding sites and resistance mutations in the three-dimensional structure of HIV-1 reverse transcriptase. Implications for mechanisms of drug inhibition and resistance. *J. Mol. Biol.* **243**, 369–387 (1994).

55. Ren, J. *et al.* High resolution structures of HIV-1 RT from four RT–inhibitor complexes. *Nature Struct. Biol.* **2**, 293–302 (1995).

56. Das, K. *et al.* Crystal structures of 8-Cl and 9-Cl TIBO complexed with wild-type HIV-1 RT and 8-Cl TIBO complexed with the Tyr181Cys HIV-1 RT drug-resistant mutant. *J. Mol. Biol.* **264**, 1085–1100 (1996).

57. Esnouf, R. *et al.* Mechanism of inhibition of HIV-1 reverse transcriptase by non-nucleoside inhibitors. *Nature Struct. Biol.* **2**, 303–308 (1995).

58. Ren, J. *et al.* Structural mechanisms of drug resistance for mutations at codons 181 and 188 in HIV-1 reverse transcriptase and the improved resilience of second generation non-nucleoside inhibitors. *J. Mol. Biol.* **312**, 795–805 (2001).

59. Hsiou, Y. *et al.* The Lys103Asn mutation of HIV-1 RT: a novel mechanism of drug resistance. *J. Mol. Biol.* **309**, 437–445 (2001).

60. Das, K. *et al.* Crystal structures of clinically relevant Lys103Asn/Tyr181Cys double mutant HIV-1 reverse transcriptase in complexes with ATP and non-nucleoside inhibitor HBY 097. *J. Mol. Biol.* **365**, 77–89 (2007).

61. Das, K. *et al.* High-resolution structures of HIV-1 reverse transcriptase/TMC278 complexes: Strategic flexibility explains potency against resistance mutations. *Proc. Natl Acad. Sci. USA* **105**, 1466–1471 (2008).

62. Guo, F., Cen, S., Niu, M., Saadatmand, J. & Kleiman, L. Inhibition of formula-primed reverse transcription by human APOBEC3G during human immunodeficiency virus type 1 replication. *J. Virol.* **80**, 11710–11722 (2006).

63. Bishop, K. N., Verma, M., Kim, E. Y., Wolinsky, S. M. & Malim, M. H. APOBEC3G inhibits elongation of HIV-1 reverse transcripts. *PLoS Pathog.* **4**, e1000231 (2008).

64. Harris, R. S. *et al.* DNA deamination mediates innate immunity to retroviral infection. *Cell* **113**, 803–809 (2003).

65. Mangeat, B. *et al.* Broad antiretroviral defence by human APOBEC3G through lethal editing of nascent reverse transcripts. *Nature* **424**, 99–103 (2003).

66. Zhang, H. *et al.* The cytidine deaminase CEM15 induces hypermutation in newly synthesized HIV-1 DNA. *Nature* **424**, 94–98 (2003).

67. Sheehy, A. M., Gaddis, N. C. & Malim, M. H. The antiretroviral enzyme APOBEC3G is degraded by the proteasome in response to HIV-1 Vif. *Nature Med.* **9**, 1404–1407 (2003).

68. Yu, X. *et al.* Induction of APOBEC3G ubiquitination and degradation by an HIV-1 Vif-Cul5-SCF complex. *Science* **302**, 1056–1060 (2003).

69. Nathans, R. *et al.* Small-molecule inhibition of HIV-1 Vif. *Nature Biotech.* **26**, 1187–1192 (2008).

70. Cen, S. *et al.* Small molecular compounds inhibit HIV-1 replication through specifically stabilizing APOBEC3G. *J. Biol. Chem.* **285**, 16546–16552 (2010).

71. Navarro, F. *et al.* Complementary function of the two catalytic domains of APOBEC3G. *Virology* **333**, 374–386 (2005).

72. Newman, E. N. *et al.* Antiviral function of APOBEC3G can be dissociated from cytidine deaminase activity. *Curr. Biol.* **15**, 166–170 (2005).

73. Hache, G., Liddament, M. T. & Harris, R. S. The retroviral hypermutation specificity of APOBEC3F and APOBEC3G is governed by the C-terminal DNA cytosine deaminase domain. *J. Biol. Chem.* **280**, 10920–10924 (2005).

74. Chen, K. M. *et al.* Structure of the DNA deaminase domain of the HIV-1 restriction factor APOBEC3G. *Nature* **452**, 116–119 (2008).

75. Furukawa, A. *et al.* Structure, interaction and real-time monitoring of the enzymatic reaction of wild-type APOBEC3G. *EMBO J.* **28**, 440–451 (2009).

76. Harjes, E. *et al.* An extended structure of the APOBEC3G catalytic domain suggests a unique holoenzyme model. *J. Mol. Biol.* **389**, 819–832 (2009).

77. Holden, L. G. *et al.* Crystal structure of the anti-viral APOBEC3G catalytic domain and functional implications. *Nature* **456**, 121–124 (2008).

78. Shandilya, S. M. *et al.* Crystal structure of the APOBEC3G catalytic domain reveals potential oligomerization interfaces. *Structure* **18**, 28–38 (2010).

79. Hughes, J. F. & Coffin, J. M. Human endogenous retrovirus K solo-LTR formation and insertional polymorphisms: Implications for human and viral evolution. *Proc. Natl Acad. Sci. USA* **101**, 1668–1672 (2004).

80. Sarkar, I., Hauber, I., Hauber, J. & Buchholz, F. HIV-1 proviral DNA excision using an evolved recombinase. *Science* **316**, 1912–1915 (2007).

81. Jaskolski, M., Alexandratos, J. N., Bujacz, G. & Wlodawer, A. Piecing together the structure of retroviral integrase, an important target in AIDS therapy. *FEBS J.* **276**, 2926–2946 (2009).

82. Espeseth, A. S. *et al.* HIV-1 integrase inhibitors that compete with the target DNA substrate define a unique strand transfer conformation for integrase. *Proc. Natl Acad. Sci. USA* **97**, 11244–11249 (2000).

83. Hare, S., Gupta, S. S., Valkov, E., Engelman, A. & Cherepanov, P. Retroviral intasome assembly and inhibition of DNA strand transfer. *Nature* **464**, 232–236 (2010). X-ray crystal structures of PFV intasomes reveal the architectural foundation of retroviral-DNA integration and the mechanism by which INSTIs block DNA strand transfer activity.

84. Maertens, G. N., Hare, S. & Cherepanov, P. The mechanism of retroviral integration from X-ray structures of its key intermediates. *Nature* **468**, 326–329 (2010). X-ray crystal structures of the intasome in complex with target DNA prior to and following strand transfer; these structures define the mechanism of retroviral-DNA integration.

85. Li, X., Krishnan, L., Cherepanov, P. & Engelman, A. Structural biology of retroviral DNA integration. *Virology* **411**, 194–205 (2011).

86. Cherepanov, P., Maertens, G. N. & Hare, S. Structural insights into the retroviral DNA integration apparatus. *Curr. Opin. Struct. Biol.* **21**, 249–256 (2011).

87. Valkov, E. *et al.* Functional and structural characterization of the integrase from the prototype foamy virus. *Nucleic Acids Res.* **37**, 243–255 (2009).

88. Hare, S. *et al.* Molecular mechanisms of retroviral integrase inhibition and the evolution of viral resistance. *Proc. Natl Acad. Sci. USA* **107**, 20057–20062 (2010).

89. Koh, Y., Matreyek, K. A. & Engelman, A. Differential sensitivities of retroviruses to integrase strand transfer inhibitors. *J. Virol.* **85**, 3677–3682 (2011).

90. Grobler, J. A. *et al.* Diketo acid inhibitor mechanism and HIV-1 integrase: Implications for metal binding in the active site of phosphotransferase enzymes. *Proc. Natl Acad. Sci. USA* **99**, 6661–6666 (2002).

91. Engelman, A. & Cherepanov, P. The lentiviral integrase binding protein LEDGF/p75 and HIV-1 replication. *PLoS Pathog.* **4**, e1000046 (2008).

92. Cherepanov, P. *et al.* Solution structure of the HIV-1 integrase-binding domain in LEDGF/p75. *Nature Struct. Mol. Biol.* **12**, 526–532 (2005).

93. Cherepanov, P., Ambrosio, A. L. B., Rahman, S., Ellenberger, T. & Engelman, A. Structural basis for the recognition between HIV-1 integrase and transcriptional coactivator p75. *Proc. Natl Acad. Sci. USA* **102**, 17308–17313 (2005).

94. Christ, F. *et al.* Rational design of small-molecule inhibitors of the LEDGF/p75-integrase interaction and HIV replication. *Nature Chem. Biol.* **6**, 442–448 (2010). The structure-based design of LEDGIN compounds with antiviral activity.

95. Hare, S. *et al.* Structural and functional analyses of the second-generation integrase strand transfer inhibitor dolutegravir (S/GSK1349572). *Mol. Pharmacol.* **80**, 565–572 (2011).

REVIEWS

96. Métifiot, M., Marchand, C., Maddali, K. & Pommier, Y. Resistance to integrase inhibitors. *Viruses* **2**, 1347–1366 (2010).

97. Tahirov, T. H. *et al.* Crystal structure of HIV-1 Tat complexed with human P-TEFb. *Nature* **465**, 747–751 (2010). The crystal structure of the Tat–P-TEFb complex suggests that it should be explored as a target for antiviral development.

98. Daugherty, M. D., Liu, B. & Frankel, A. D. Structural basis for cooperative RNA binding and export complex assembly by HIV Rev. *Nature Struct. Mol. Biol.* **17**, 1337–1442 (2010).

99. DiMattia, M. A. *et al.* Implications of the HIV-1 Rev dimer structure at 3.2 Å resolution for multimeric binding to the Rev response element. *Proc. Natl Acad. Sci. USA* **107**, 5810–5814 (2010).

100. Bieniasz, P. D., Grdina, T. A., Bogerd, H. P. & Cullen, B. R. Recruitment of a protein complex containing Tat and cyclin T1 to TAR governs the species specificity of HIV-1 Tat. *EMBO J.* **17**, 7056–7065 (1998).

101. Fujinaga, K. *et al.* The ability of positive transcription elongation factor b to transactivate human immunodeficiency virus transcription depends on a functional kinase domain, cyclin T1, and Tat. *J. Virol.* **72**, 7154–7159 (1998).

102. Bayer, P. *et al.* Structural studies of HIV-1 Tat protein. *J. Mol. Biol.* **247**, 529–535 (1995).

103. Anand, K., Schulte, A., Vogel-Bachmayr, K., Scheffzek, K. & Geyer, M. Structural insights into the cyclin T1–Tat–TAR RNA transcription activation complex from EIAV. *Nature Struct. Mol. Biol.* **15**, 1287–1292 (2008).

104. Zhou, M. *et al.* Tat modifies the activity of CDK9 to phosphorylate serine 5 of the RNA polymerase II carboxyl-terminal domain during human immunodeficiency virus type 1 transcription. *Mol. Cell. Biol.* **20**, 5077–5086 (2000).

105. Jain, C. & Belasco, J. G. Structural model for the cooperative assembly of HIV-1 Rev multimers on the RRE as deduced from analysis of assembly-defective mutants. *Mol. Cell* **7**, 603–614 (2001).

106. Daugherty, M. D., Booth, D. S., Jayaraman, B., Cheng, Y. & Frankel, A. D. HIV Rev response element (RRE) directs assembly of the Rev homooligomer into discrete asymmetric complexes. *Proc. Natl Acad. Sci. USA* **107**, 12481–12486 (2010).

107. Gheysen, D. *et al.* Assembly and release of HIV-1 precursor Pr55gag virus-like particles from recombinant baculovirus-infected insect cells. *Cell* **59**, 103–112 (1989).

108. Göttlinger, H. G., Sodroski, J. G. & Haseltine, W. A. Role of capsid precursor processing and myristoylation in morphogenesis and infectivity of human immunodeficiency virus type 1. *Proc. Natl Acad. Sci. USA* **86**, 5781–5785 (1989).

109. Bryant, M. & Ratner, L. Myristoylation-dependent replication and assembly of human immunodeficiency virus 1. *Proc. Natl Acad. Sci. USA* **87**, 523–527 (1990).

110. Zhou, W., Parent, L. J., Wills, J. W. & Resh, M. D. Identification of a membrane-binding domain within the amino-terminal region of human immunodeficiency virus type 1 Gag protein which interacts with acidic phospholipids. *J. Virol.* **68**, 2556–2569 (1994).

111. Hill, C. P., Worthylake, D., Bancroft, D. P., Christensen, A. M. & Sundquist, W. I. Crystal structures of the trimeric human immunodeficiency virus type 1 matrix protein: implications for membrane association and assembly. *Proc. Natl Acad. Sci. USA* **93**, 3099–3104 (1996).

112. Rao, Z. *et al.* Crystal structure of SIV matrix antigen and implications for virus assembly. *Nature* **378**, 743–747 (1995).

113. Zhou, W. & Resh, M. D. Differential membrane binding of the human immunodeficiency virus type 1 matrix protein. *J. Virol.* **70**, 8540–8548 (1996).

114. Saad, J. S. *et al.* Structural basis for targeting HIV-1 Gag proteins to the plasma membrane for virus assembly. *Proc. Natl Acad. Sci. USA* **103**, 11364–11369 (2006).

115. Ono, A., Ablan, S. D., Lockett, S. J., Nagashima, K. & Freed, E. O. Phosphatidylinositol (4,5) bisphosphate regulates HIV-1 Gag targeting to the plasma membrane. *Proc. Natl Acad. Sci. USA* **101**, 14889–14894 (2004).

116. Carlton, J. G. & Martin-Serrano, J. Parallels between cytokinesis and retroviral budding: a role for the ESCRT machinery. *Science* **316**, 1908–1912 (2007).

117. Morita, E. *et al.* Human ESCRT and ALIX proteins interact with proteins of the midbody and function in cytokinesis. *EMBO J.* **26**, 4215–4227 (2007).

118. Hurley, J. H. & Hanson, P. I. Membrane budding and scission by the ESCRT machinery: it's all in the neck. *Nature Rev. Mol. Cell Biol.* **11**, 556–566 (2010).

119. Henne, W. M., Buchkovich, N. J. & Emr, S. D. The ESCRT pathway. *Dev. Cell* **21**, 77–91 (2011).

120. Göttlinger, H. G., Dorfman, T., Sodroski, J. G. & Haseltine, W. A. Effect of mutations affecting the p6 gag protein on human immunodeficiency virus particle release. *Proc. Natl Acad. Sci. USA* **88**, 3195–3199 (1991).

121. Strack, B., Calistri, A., Craig, S., Popova, E. & Göttlinger, H. G. AIP1/ALIX is a binding partner for HIV-1 p6 and EIAV p9 functioning in virus budding. *Cell* **114**, 689–699 (2003).

122. Garrus, J. E. *et al.* Tsg101 and the vacuolar protein sorting pathway are essential for HIV-1 budding. *Cell* **107**, 55–65 (2001).

123. Fisher, R. D. *et al.* Structural and biochemical studies of ALIX/AIP1 and its role in retrovirus budding. *Cell* **128**, 841–852 (2007).

124. Lee, S., Joshi, A., Nagashima, K., Freed, E. O. & Hurley, J. H. Structural basis for viral late-domain binding to Alix. *Nature Struct. Mol. Biol.* **14**, 194–199 (2007).

125. McCullough, J., Fisher, R. D., Whitby, F. G., Sundquist, W. I. & Hill, C. P. ALIX-CHMP4 interactions in the human ESCRT pathway. *Proc. Natl Acad. Sci. USA* **105**, 7687–7691 (2008).

126. Zhai, Q. *et al.* Structural and functional studies of ALIX interactions with YPXnL late domains of HIV-1 and EIAV. *Nature Struct. Mol. Biol.* **15**, 43–49 (2008).

127. Katoh, K. *et al.* The penta-EF-hand protein ALG-2 interacts directly with the ESCRT-I component TSG101, and Ca2+-dependently co-localizes to aberrant endosomes with dominant-negative AAA ATPase SKD1/Vps4B. *Biochem. J.* **391**, 677–685 (2005).

128. von Schwedler, U. K. *et al.* The protein network of HIV budding. *Cell* **114**, 701–713 (2003).

129. Pornillos, O., Alam, S. L., Davis, D. R. & Sundquist, W. I. Structure of the Tsg101 UEV domain in complex with the PTAP motif of the HIV-1 p6 protein. *Nature Struct. Biol.* **9**, 812–817 (2002).

130. Im, Y. J. *et al.* Crystallographic and functional analysis of the ESCRT-I /HIV-1 Gag PTAP interaction. *Structure* **18**, 1536–1547 (2010).

131. Neil, S. J., Zang, T. & Bieniasz, P. D. Tetherin inhibits retrovirus release and is antagonized by HIV-1 Vpu. *Nature* **451**, 425–430 (2008).

132. Van Damme, N. *et al.* The interferon-induced protein BST-2 restricts HIV-1 release and is downregulated from the cell surface by the viral Vpu protein. *Cell Host Microbe* **3**, 245–252 (2008).

133. Kupzig, S. *et al.* Bst-2/HM1.24 is a raft-associated apical membrane protein with an unusual topology. *Traffic* **4**, 694–709 (2003).

134. Andrew, A. J., Kao, S. & Strebel, K. C-terminal hydrophobic region in human bone marrow stromal cell antigen 2 (BST-2)/tetherin protein functions as second transmembrane motif. *J. Biol. Chem.* **286**, 39967–39981 (2011).

135. Hinz, A. *et al.* Structural basis of HIV-1 tethering to membranes by the BST-2/tetherin ectodomain. *Cell Host Microbe* **7**, 314–323 (2010).

136. Yang, H. *et al.* Structural insight into the mechanisms of enveloped virus tethering by tetherin. *Proc. Natl Acad. Sci. USA* **107**, 18428–18432 (2010).

137. Schubert, H. L. *et al.* Structural and functional studies on the extracellular domain of BST2/tetherin in reduced and oxidized conformations. *Proc. Natl Acad. Sci. USA* **107**, 17951–17956 (2010).

138. Mangeat, B. *et al.* HIV-1 Vpu neutralizes the antiviral factor Tetherin/BST-2 by binding it and directing its beta-TrCP2-dependent degradation. *PLoS Pathog.* **5**, e1000574 (2009).

139. Dube, M. *et al.* Antagonism of tetherin restriction of HIV-1 release by Vpu involves binding and sequestration of the restriction factor in a perinuclear compartment. *PLoS Pathog.* **6**, e1000856 (2010).

140. Skasko, M. *et al.* HIV-1 Vpu antagonizes the innate restriction factor BST-2 via lipid-embedded helix-helix interactions. *J. Biol. Chem.* **287**, 58–67 (2012).

141. Pettit, S. C. *et al.* The p2 domain of human immunodeficiency virus type 1 Gag regulates sequential proteolytic processing and is required to produce fully infectious virions. *J. Virol.* **68**, 8017–8027 (1994).

142. Briggs, J. A. *et al.* Structure and assembly of immature HIV. *Proc. Natl Acad. Sci. USA* **106**, 11090–11095 (2009).

143. de Marco, A. *et al.* Structural analysis of HIV-1 maturation using cryo-electron tomography. *PLoS Pathog.* **6**, e1001215 (2010).

144. Carlson, L. A. *et al.* Cryo electron tomography of native HIV-1 budding sites. *PLoS Pathog.* **6**, e1001173 (2010).

145. von Schwedler, U. K. *et al.* Proteolytic refolding of the HIV-1 capsid protein amino-terminus facilitates viral core assembly. *EMBO J.* **17**, 1555–1568 (1998).

146. Sticht, J. *et al.* A peptide inhibitor of HIV-1 assembly in vitro. *Nature Struct. Mol. Biol.* **12**, 671–677 (2005).

147. Li, F. *et al.* PA-457: A potent HIV inhibitor that disrupts core condensation by targeting a late step in Gag processing. *Proc. Natl Acad. Sci. USA* **100**, 13555–13560 (2003).

148. Keller, P. W., Adamson, C. S., Heymann, J. B., Freed, E. O. & Steven, A. C. HIV-1 maturation inhibitor bevirimat stabilizes the immature Gag lattice. *J. Virol.* **85**, 1420–1428 (2011).

149. Tang, C. *et al.* Antiviral inhibition of the HIV-1 capsid protein. *J. Mol. Biol.* **327**, 1013–1020 (2003).

150. Kelly, B. N. *et al.* Structure of the antiviral assembly inhibitor CAP-1 complex with the HIV-1 CA protein. *J. Mol. Biol.* **373**, 355–366 (2007).

151. Lapatto, R. *et al.* X-ray analysis of HIV-1 protease at 2.7 Å resolution confirms structural homology among retroviral enzymes. *Nature* **342**, 299–302 (1989).

152. Navia, M. A. *et al.* Three-dimensional structure of aspartyl protease from human immunodeficiency virus HIV-1. *Nature* **337**, 615–620 (1989).

153. Wlodawer, A. *et al.* Conserved folding in retroviral proteases: crystal structure of a synthetic HIV-1 protease. *Science* **245**, 616–621 (1989).

154. Kitchen, V. S. *et al.* Safety and activity of saquinavir in HIV infection. *Lancet* **345**, 952–955 (1995).

155. Wlodawer, A. & Erickson, J. W. Structure-based inhibitors of HIV-1 protease. *Annu. Rev. Biochem.* **62**, 543–585 (1993).

156. Wensing, A. M., van Maarseveen, N. M. & Nijhuis, M. Fifteen years of HIV protease inhibitors: raising the barrier to resistance. *Antiviral Res.* **85**, 59–74 (2010).

157. Prabu-Jeyabalan, M., Nalivaika, E. & Schiffer, C. A. Substrate shape determines specificity of recognition for HIV-1 protease: analysis of crystal structures of six substrate complexes. *Structure* **10**, 369–381 (2002).

158. King, N. M., Prabu-Jeyabalan, M., Nalivaika, E. A. & Schiffer, C. A. Combating susceptibility to drug resistance: lessons from HIV-1 protease. *Chem. Biol.* **11**, 1333–1338 (2004).

159. Nalam, M. N. L. *et al.* Evaluating the substrate-envelope hypothesis: structural analysis of novel HIV-1 protease inhibitors designed to be robust against drug resistance. *J. Virol.* **84**, 5368–5378 (2010).

160. Laguette, N. *et al.* SAMHD1 is the dendritic- and myeloid-cell-specific HIV-1 restriction factor counteracted by Vpx. *Nature* **474**, 654–657 (2011).

161. Hrecka, K. *et al.* Vpx relieves inhibition of HIV-1 infection of macrophages mediated by the SAMHD1 protein. *Nature* **474**, 658–661 (2011).

162. Wlodawer, A. Structure-based design of AIDS drugs and the development of resistance. *Vox Sang.* **83**, 23–26 (2002).

163. Burton, D. R. & Weiss, R. A. AIDS/HIV. A boost for HIV vaccine design. *Science* **329**, 770–773 (2010).

164. Grigorieff, N. & Harrison, S. C. Near-atomic resolution reconstructions of icosahedral viruses from electron cryo-microscopy. *Curr. Opin. Struct. Biol.* **21**, 265–273 (2011).

165. Flexner, C. HIV drug development: the next 25 years. *Nature Rev. Drug Discov.* **6**, 959–966 (2007).


Acknowledgements
The authors thank M. Yeager for sharing coordinates of the HIV-1 capsid model. This work was supported by grants AI070042 from the US National Institutes of Health (A.E.) and G1000917 from the UK Medical Research Council (P.C.). The opinions voiced herein in no way reflect those of these funding agencies. The authors apologize to colleagues whose work could not be cited or discussed owing to space limitations.


Competing interests statement
The authors declare no competing financial interests.


DATABASES
Protein Data Bank: [http://www.rcsb.org/pdb/home/home.do](http://www.rcsb.org/pdb/home/home.do)
1RTD | 2XDE | 2X7L | 2ZD1 | 3H47 | 3JWD | 3KLE | 3LPH | 3MIA | 3NGB | 3OBU | 3OS2 | 3OY9 | 3P05


FURTHER INFORMATION
Alan Engelman’s homepage: [http://www.dfhcc.harvard.edu/membership/profile/member/392/0/](http://www.dfhcc.harvard.edu/membership/profile/member/392/0/)
Peter Cherepanov’s homepage: [http://www.london-research-institute.org.uk/research/221](http://www.london-research-institute.org.uk/research/221)
Approved HIV drugs: [http://www.aidsmeds.com/list.shtml](http://www.aidsmeds.com/list.shtml)

ALL LINKS ARE ACTIVE IN THE ONLINE PDF
