
Immunol Res  
DOI 10.1007/s12026-012-8304-8  

---

IMMUNOLOGY AT MOUNT SINAI  

**Diverse immune evasion strategies by human cytomegalovirus**

Vanessa Noriega · Veronika Redmann · Thomas Gardner · Domenico Tortorella  

© Springer Science+Business Media, LLC 2012  

Abstract Members of the *Herpesviridae* family have the capacity to undergo both lytic and latent infection to establish a lifelong relationship with their host. Following primary infection, human cytomegalovirus (HCMV) can persist as a subclinical, recurrent infection for the lifetime of an individual. This quiescent portion of its life cycle is termed latency and is associated with periodic bouts of reactivation during times of immunosuppression, inflammation, or stress. In order to exist indefinitely and establish infection, HCMV encodes a multitude of immune modulatory mechanisms devoted to escaping the host antiviral response. HCMV has become a paradigm for studies of viral immune evasion of antigen presentation by both major histocompatibility complex (MHC) class I and II molecules. By restricting the presentation of viral antigens during both productive and latent infection, HCMV limits elimination by the human immune system. This review will focus on understanding how the virus manipulates the pathways of antigen presentation in order to modulate the host response to infection.

Keywords Human cytomegalovirus · MHC antigen presentation · Immune evasion · Latency · Lytic infection · Unique short · Proteasome · Degradation

Introduction  
Herpesviruses have coevolved with their hosts for millions of years. This has led to the development of multifaceted strategies of immune evasion that have allowed the virus to survive in the face of a hostile host immune system. In this regard, studying how different members of the herpes virus family coexist with the host provides a unique approach to better understanding host-virus interactions, fundamental elements of the immune response and general cellular processes. Human cytomegalovirus (HCMV), a member of the β-herpesvirus subfamily, is characterized by its large genome capacity and strict species specificity [1, 2]. Like HCMV, many human herpes viruses establish unique cellular sites of viral dormancy or latency [3]. As with Epstein-Barr virus (EBV), herpes simplex virus type 1 (HSV1), and human herpes virus-6 and 7, HCMV has the ability to periodically reactivate to form new transmissible infectious virions, thereby contributing to its success as a human pathogen. This review will focus on the strategies utilized by HCMV to modulate antigen presentation that contributes to viral persistence within the human host.

The HCMV life cycle  
The HCMV virion structure is comprised of a proteinaceous capsid containing a ~250-kb double-stranded DNA genome, a tegument layer surrounding the capsid composed of viral phosphoproteins, and an outer envelope containing numerous glycoprotein complexes [4] (Fig. 1). HCMV infection ensues by the initial attachment to extracellular

Vanessa Noriega, Veronika Redmann, Thomas Gardner contributed equally to this work.  

V. Noriega · V. Redmann · T. Gardner · D. Tortorella (✉)  
Department of Microbiology, Mount Sinai School of Medicine, One Gustave L. Levy Place, Box 1124, New York, NY 10029, USA  
e-mail: domenico.tortorella@mssm.edu  

Published online: 28 March 2012  
Springer

Immunology Institute at the Mount Sinai School of Medicine

![Diagram](https://i.imgur.com/yourimageurl.png)

Fig. 1 A depiction of an HCMV virion. The HCMV virion consists of an outer membranous envelope studded with glycoprotein complexes critical for viral entry and membrane fusion, a tegument layer of tegument phosphoproteins, and a capsid encapsulating a ~250-kb double-stranded DNA genome.

virus particles [21]. The viral packaging process incorporates a single, full-length viral genome within the capsid that undergoes a primary envelopment at the inner nuclear membrane, followed by de-envelopment at the outer nuclear membrane [22, 23]. The naked nucleocapsid is then released into the cytoplasm where tegumentation occurs and assembly proceeds in discrete cytoplasmic compartments [24–26]. The cytoplasmic viral capsid then undergoes secondary envelopment where it is believed to acquire an envelope from a distinct subcellular compartment whose lipid composition most closely resembles synaptic vesicles [27]. At the cell surface, the mature enveloped virion is released, completing one cycle of lytic HCMV infection approximately 72–96 h post-infection [28].

## Viral latency and sites of HCMV carriage in vivo

In certain cell types, IE genes are silenced upon HCMV entry resulting in a latent infection [29]. Viral latency can be defined as the maintenance of viral genomes in the absence of infectious virion production with the ability of the viral genome to reinitiate a full replicative cycle under specific stimuli. During times of latency, the viral genome is maintained as an extrachromosomal plasmid at a low copy number and only a subset of viral genes remain transcriptionally active [30–33]. Most notably absent are the transactivating viral IE genes, thus leading to a lack of any subsequent lytic gene expression in cells carrying the viral genome. Latent virus periodically reactivates as a productive infection with the capacity to disseminate virus. The greatest clinical threat from HCMV arises in the case of immunosuppression and reactivation of latent virus [34, 35]. In stem cell and solid organ transplant recipients and HIV-infected individuals, a primary infection and reactivated HCMV can replicate uncontrollably leading to widespread dissemination and serious disease, such as severe gastrointestinal tract infection, hepatitis, solid organ rejection, and chronic graft-versus-host disease [36].

Early attempts to define cell types that carried latent HCMV focused on analyzing transfusion-transmitted viral disease among healthy seropositive carriers [37–39]. Infectious virus could not be isolated directly from the blood of normal seropositive individuals [40], and the transmission by blood and transfusion-mediated disease could be reduced by leukocyte depletion [41]. With the development of highly sensitive polymerase chain reaction (PCR) techniques, more detailed analysis of blood from healthy carriers identified HCMV DNA in peripheral blood monocytes, but not T or B lymphocytes [42, 43]. This discovery led to the question whether latently infected monocytes acquired HCMV at an earlier stage of differentiation along the myeloid lineage. PCR-based analysis of

bone marrow-derived CD34+ progenitors demonstrated that these cells also carried HCMV genomes in vivo [44]. The actual number of mononuclear or hematopoietic progenitor cells carrying viral genomes in natural latency was deduced to be extremely low, in the range of 1 in \(10^4\)–\(10^5\) cells [45]. Furthermore, the physical conformation of HCMV genomes in peripheral blood monocytes has been assessed by PCR detection and electrophoretic separation and migrates as a circular plasmid, or episome [46]. Taken together, natural latency of HCMV is present in CD34+ progenitors and maintained in these cells as episomal molecules as these cells differentiate to CD14+ monocytes and macrophages. Notably, endothelial cells have also been proposed to be latently infected with HCMV, a cell type derived from CD34+ progenitor cells [47]. Why these cells selectively maintain HCMV genomes and why other subsets of cells arising from common progenitor cells do not carry viral DNA remains to be understood.

### Major histocompatibility antigen presentation

The human body presents a unique environment for colonization by microorganisms, and the immune system is crucial for recognizing the difference between “self” and “non-self” in all tissues. The concerted actions of the innate and adaptive branches of immunity tailor a multilayered and specific response to infection by pathogens. Innate immunity, the more evolutionarily conserved branch, includes anatomical barriers such as skin and mucosal layers and has the ability to detect pathogen-associated molecular patterns (PAMPs). Innate immunity also utilizes a variety of immune cells that respond to inflammation and disruptions in cellular equilibrium. The innate immune system reacts immediately and broadly to infection (see reviews [48, 49]). The adaptive immune response is composed of highly specialized cells whose task is the generation of responses that are maximally tailored to eliminate specific pathogens [50, 51]. A key component of the adaptive branch of immunity is the development of immunological memory [52]. This provides the host immune system with the ability to recognize and quickly eliminate pathogens during subsequent infections.

A bridge between the innate and adaptive immune systems is antigen presentation. This pathway involves the presentation of cellular and extracellular peptides by cell-surface immune receptors termed major histocompatibility complex (MHC) class I and II molecules [53]. Professional antigen-presenting cells (APCs) sample the extracellular environment and internalize antigen either by phagocytosis or by receptor-mediated endocytosis. Antigens are then displayed on MHC class II molecules to CD4+

T-lymphocytes in order to initiate an immune response [54, 55]. MHC class II molecules are loaded within endocytic compartments with peptides 18–24 amino acids in length derived from endosomal degradation [56]. CD8+ T-lymphocytes continually patrol host tissues, surveying intracellular peptides presented by MHC class I molecules. Class I molecules are typically loaded within the endoplasmic reticulum (ER) with peptides 8–10 amino acids in length derived from cytoplasmic degradation by the proteasome [57]. In addition, cross-presentation of exogenous proteins within endosomes in a class I-restricted manner provides an effective means of recognizing pathogen-infected cells [58]. Recognition of class I/II and peptide complexes by CD4+ or CD8+ T-lymphocytes aids in establishing and maximizing the capabilities of the immune system, culminating in clearance of infected cells. The adaptive immune system possesses the ability to generate memory through activation and differentiation of immune cells, somatic hypermutation of immunoglobulin antigen receptors, increased antigen presentation, and the secretion of cytokines and chemokines that provide direction to responding immune cells [59]. Together, the adaptive immune system responds effectively to both primary infections and reinfections by preventing colonization, establishing memory of the pathogen, and limiting cellular damage during an infection. This review will focus on the modulation of antigen presentation by HCMV and the methods by which the virus confronts the challenge of remaining dormant in cells. Other aspects of HCMV immune evasion and those of other viruses can be found in more comprehensive reviews [28, 60].

#### Modulation of antigen presentation by HCMV

MHC class I antigen presentation serves a crucial role in alerting the immune system to infections. The efficient establishment of latency by HCMV in the majority of the human population highlights the virus’ successful ability to evade the immune system. The HCMV genome encodes multiple immune evasion proteins during the course of infection that target different arms of antigen presentation, including MHC class I and II molecules and inhibitory and activating receptors of NK cells. Early work characterizing essential and non-essential genes of HCMV observed that infection led to the down-regulation of MHC class I molecules from the surface of infected cells [61–65]. The genomic region implicated in this phenomenon was US2 through US11 [66]. The viral genes US2, US3, US6, US10, and US11 have all subsequently been found to play different roles in modulating antigen presentation by limiting the surface expression of MHC class I and II molecules (Table 1; Fig. 2).

Immunology Institute at the Mount Sinai School of Medicine

Retention of MHC class I complexes in the ER: US3

HCMV US3 is expressed 1–4 h post-infection during the IE period of viral replication and alters MHC class I antigen presentation [67]. The US3 open reading frame (ORF) is capable of creating three transcripts through alternative splicing [27]. The largest transcript, producing a 22 kDa protein, is required for retention of fully assembled MHC class I complexes in the ER [67, 68]. This interaction with class I complexes is dependent on the transmembrane and ER-luminal domains of US3 [69, 70]. US3 can directly bind to the ER-resident chaperone tapasin to inhibit optimal peptide loading of class I [71]. Therefore, class I allelic susceptibility to US3 retention correlates with tapasin dependence for peptide loading and surface expression [71]. Interestingly, US3 can also retain the trophoblast-associated non-classical MHC class I alleles HLA-G and HLA-C [72]. US3 appears to differentially regulate the

Table 1 HCMV viral antagonists of MHC class I and II antigen presentation

| Protein/gene | Infection phase       | Function                                                                                   | References          |
|--------------|-----------------------|-------------------------------------------------------------------------------------------|---------------------|
| US3          | Immediate early       | Retains MHC class I molecules, reduces MHC class II presentation                          | [67, 68, 73, 74]    |
| US11         | Early                 | Induces degradation of MHC class I molecules                                               | [66, 76, 144]       |
| US2          | Early                 | Induces degradation of MHC class I molecules                                               | [75, 83, 85]        |
| US10         | Early                 | Retains MHC class I heavy chains, induces degradation of HLA-G                             | [66, 102, 103]      |
| US6          | Late                  | Inhibits the translocation of peptides by the TAP complex                                 | [104, 105]          |
| UL82(pp71)   | Early/late (tegument) | Delays egress of MHC class I complexes from the ER to the Golgi                            | [114]               |
| UL83(pp65)   | Early/late (tegument) | Prevents generation of viral antigenic peptides                                             | [64, 113]           |
| miRNA        |                       |                                                                                           |                     |
| US4-1        | Immediate early/early | Targets the mRNA of the aminopeptidase, ERAP1                                              | [111, 112]          |

Fig. 2 Mechanisms of HCMV-mediated downregulation of MHC class I molecules. Nascent MHC class I heavy chain enters the ER lumen and binds to calnexin before associating with β₂ microglobulin (β₂m). Protein-derived peptide fragments generated by the proteasome in the cytosol are delivered to the ER lumen by the transporter associated with antigen processing (TAP) complex. These peptides are eventually trimmed by the ER-aminopeptidase 1 (ERAP1). Tapasin bridges the MHC class I molecule and TAP to allow peptide loading of an 8–10 aa peptide creating a mature MHC class I complex. The HCMV-encoded proteins US2, US10, and US11 induce proteasomal degradation of diverse alleles of MHC class I. US3 and US10 bind and attenuate class I egress from the ER. US6 interacts with the ER-luminal domain of the TAP complex and deactivates TAP-mediated peptide translocation. The product of UL83, pp65, may be involved in the phosphorylation of HCMV proteins to prevent their proteasome-mediated degradation, thereby blocking the generation of HCMV antigenic peptides. The product of UL82, pp71, was found to delay transport of MHC class I complexes from the ER or cis-Golgi. Finally, the HCMV-encoded miRNA miR-US4-1 suppresses translation of ERAP1 to limit loading of antigenic peptides

activity of two additional HCMV-encoded immunoevasins—US3 enhances degradation of MHC class I molecules by the US2 gene product [73], while it appears to diminish destruction of class I induced by the US11 gene product (unpublished data). Additionally, US3 reduces MHC class II antigen presentation by binding to newly synthesized class II α/β heterodimers and preventing their ability to associate with the class II invariant chain [74]. The dual function of US3 in evasion of both MHC class I and II antigen processing highlights the importance of these pathways in controlling viral replication.

Induction of MHC class I heavy chain dislocation and degradation: US2, US11, and US10

HCMV US2 and US11 gene products are expressed during the early phase of infection and down-regulate class I molecules during a virus infection [66, 75]. Expression of these individual genes in U373 astrocytoma cells resulted in the robust induction of proteasome-mediated MHC class I heavy chain degradation [76, 77]. Strikingly, these viral gene products co-opt a cellular process referred to as ER quality control that is utilized to degrade misfolded ER proteins [78]. Despite the similar outcome of US2- and US11-induced class I destruction, the proteins likely exert their action independently from each other and using diverse strategies. For example, a single glutamine residue within the transmembrane domain of US11 is required to induce ubiquitination and degradation of MHC class I [79], while the US2 transmembrane domain and select residues in the cytoplasmic tail are critical for class I instability [80, 81]. Also, only US2 inhibits MHC class II antigen presentation by inducing degradation of HLA-DR-alpha and DM-alpha molecules in a cell-dependent manner [82, 83]. These differences suggest that US2 and US11 function independently to ensure a robust class I downregulation during the early phase of infection.

One striking difference between US2 and US11 function is the allelic specificity for class I destruction. US2 was observed to alter levels of HLA-A2, HLA-B27, and HLA-G gene products, but not other HLA-B, -C or -E alleles or soluble HLA-G1 [84]. The observation of an interaction between US2 and the susceptible class I proteins but not the resistant molecules further supports the specificity of US2 mediated class I destruction [85]. In a similar set of experiments, US11 induced the degradation of HLA-A2 and not HLA-G [86]. These differences are highlighted by the molecular determinants of class I that dictate US2- and US11-induced destruction. The length of the C-terminus in class I heavy chains is a basis for its susceptibility or resistance to US11-induced degradation [87], while the α2/α3 ER-luminal region of class I can dictate US2-induced degradation [85, 86, 88, 89]. Interestingly, HLA-G and -C were resistant to US2- and US11-mediated destruction in both human trophoblast and porcine endothelial cell lines, suggesting different cell types may influence the function of US2 and US11 [90]. Despite a pronounced downregulation of HLA-A2 and HLA-B27 surface expression during infection of fibroblasts with recombinant viruses expressing US2 or US11 only, the infected fibroblasts were susceptible to pp65 and IE1-specific CTLs suggesting that multiple evasin molecules may be required to significantly limit CTL activity [91, 92]. Thus, US2 and US11 together may have evolved to target many class I alleles in order to maximize evasion of antigen presentation in different cell types.

The discovery that US2 and US11 utilize cellular complexes to induce class I destruction was a major driving force to understanding the molecular mechanism of how ER proteins are transported across the ER membrane and degraded by the proteasome, a process referred to as ER-associated degradation (ERAD) [93]. The study of class I heavy chain extraction from the ER to the cytosol, known as dislocation, led to the identification of cellular factors that play key roles in ERAD. For example, the chaperone Bip, which has been associated with ERAD, binds US2 and US11, and this interaction is required for class I degradation [94]. Cellular factors specific for US11-mediated class I degradation include the p97 AAA ATPase complexed to Npl4 and Ufd1, the Derlin family of proteins as well as SeL1L, OS9, UBC6e, AUP1, and UBXD8 [95–99]. The US2 and US11 degradation pathways appear to be similar, but not identical because many cellular factors are unique to US2-mediated destruction. US2 utilizes ER-resident proteins such as signal peptide peptidase (SPP), calnexin, and calreticulin [87, 100]. The p97 AAA ATPase plays a role in dislocation of heavy chains in US2-expressing cells, but dislocation did not require Ufd1-Npl4 unlike US11-mediated class I dislocation [101]. The cofactor preferences for US2 and US11 highlight a possible intrinsic aspect of the ERAD pathway: the recognition, dislocation, and degradation of different ER substrates may require diverse cellular factors, and HCMV has evolved to take advantage of this diversity to induce class I degradation.

HCMV US10 encodes an ER membrane glycoprotein that also interacts with constituents of MHC class I antigen presentation [102]. US10 binds free class I heavy chains and delays their transport from the ER. While US10 shares some biological features with US3, such as class I binding and ER retention capabilities, US10 does not influence US2 or US11 function [102]. In addition, while US10 delays class I egress, it does not block class I maturation or surface expression [66]. US10 was most recently characterized to specifically down-regulate HLA-G molecules during overexpression and infection studies [103]. Interestingly, this downregulation occurred in a manner similar to class I targeting by US2 and US11. US10 caused the dislocation and proteasome-mediated degradation of HLA-G, but not classical class I

molecules. However, US10 utilizes a distinct pathway to induce degradation of HLA-G, as it fails to associate with dislocation components such as Derlin-1 and SeL1L [103]. These findings highlight the extent to which HCMV targets a common set of cellular substrates.

TAP inhibition: US6

Unlike US2, US3, US10, and US11, the HCMV L protein US6 affects antigen presentation by an entirely different strategy. As opposed to interacting with free class I heavy chains or fully assembled class I complexes, US6 inhibits the translocation of cytosolic peptides by the TAP complex (TAP1/2) [104]. US6 binds to the ER luminal side of TAP1 and causes a conformational change that prevents the binding of ATP [105]. Residues 89–108 in the ER-luminal domain of US6 are sufficient and necessary for this inhibition [106–108]. This inhibition of TAP activity affects not only expression of classical MHC class I alleles but also the non-classical alleles HLA-C and HLA-G in fetal cytotrophoblasts [72]. The activities of US6, in addition to US2, US3, and US11, does not affect the expression of UL18, a virally encoded MHC class I homolog supporting its specificity for HLA molecules [109, 110]. The evolution of a viral gene targeting a step of antigen presentation that affects all class I molecules suggests the importance of ceasing antigen presentation during an infection.

Downregulation of ERAP1 translation: miRNA US4-1

In addition to the immune evasion proteins expressed within the US2-11 region, an HCMV-encoded miRNA, HCMV-US4-1 (US4-1) plays a role in inhibition of a CTL response [111]. This miRNA specifically targets a messenger RNA that encodes for the ER-aminopeptidase ERAP1, which is involved in trimming of TAP-translocated peptides in the ER [112]. By down-regulating translation of the ERAP1 transcript, US4-1 inhibits the trimming of precursors of class I-restricted peptides. In an in vitro CTL assay, US4-1-mediated downregulation of ERAP1 resulted in reduced CTL-mediated killing of HCMV-infected dermal fibroblasts. miR-US4 likely acts simultaneously with the immune evasion US proteins, but unlike these immunogenic glycoproteins, viral miRNAs cannot be presented on MHC I molecules and may thus offer an additional strategy of immune evasion, particularly during latent infection.

MHC class I downregulation by tegument proteins: UL82 (pp71) and UL83 (pp65)

Finally, two HCMV-encoded phosphoproteins UL82 (pp71) and UL83 (pp65) are implicated in MHC class I downregulation [113]. pp71, the product of the UL82 gene,

is a 71-kDa phosphoprotein contained within the tegument of the HCMV virion that exerts many roles throughout the HCMV lifecycle, including MHC class I downregulation during late stages of infection [21]. Ectopic UL82 expression in human glioblastoma cells causes a dose-dependent decrease in the accumulation of cell surface MHC class I complexes [114]. pp71 was found to delay transport of MHC class I complexes from the ER or cis-Golgi. Furthermore, in glioblastoma cells infected with an immune evasion-attenuated HCMV strain, RV7186, RNAi-mediated disruption of UL82 led to increased accumulation of MHC class I complexes on the cell surface. The phosphoprotein pp65, encoded by the gene UL83, is a major component of the HCMV tegument and has been implicated in subverting both innate and adaptive immune responses. Fibroblasts infected with RVAd65, a pp65-deletion mutant of the HCMV strain AD169, are recognized by IE-specific CTLs, while cells infected with wild-type AD169 escaped lysis [113]. Serine phosphorylation of the IE protein by pp65, through a putative kinase domain may prevent its presentation on MHC I molecules. Fibroblasts transfected with IE and a mutant pp65 incapable of phosphorylation were unable to prevent IE presentation [64, 113]. A subsequent study, however, has suggested that an intimate interaction of pp65 with the polo-like kinase 1 (Plk1) may be responsible for the observed kinase activity of pp65 [115]. pp65 likely has roles in evasion of non-MHC class I-dependent immune events as well, including accumulation and destruction of the MHC class II allele HLA-DR and antagonism of signaling pathways that affect NF-κB and IRF1 [116, 117].

Functional evaluation of HCMV-mediated modulation of antigen presentation

Although the biochemical functions of HCMV-encoded immune evasion molecules have been studied extensively, their greater role in contributing to viral infection is less well understood. Almost all herpes viruses encode one or more gene products that manipulate antigen presentation implying that altering the surface expression of MHC molecules contributes to virus survival and persistence in the host. HCMV is quite extreme in this regard by encoding for at least eight factors that can alter presentation of antigenic peptides (Table 1). However, the functional significance of the HCMV viral evasins is not clear considering the numerous HCMV-specific T cells found in a seropositive individual. A CD8+ T-cell response should only pose a threat to the virus once a humoral immune response has been initiated. Therefore, CTL evasion may only be beneficial in enabling HCMV to reach sites of persistence after initial infection, or following a recurrent

infection with a diverse HCMV strain in a host with pre-
existing immunity to HCMV. Interestingly, HCMV-spe-
cific CD8+ T cells from HCMV-seropositive donors
revealed a decrease in epitope-restricted cell lysis in a
“immune evasin”-dependent manner [118]. Studies in
Rhesus monkeys suggest the function of immune evasin
molecules during a virus “super infection” [119]. Viral
MHC class I interference was found to be dispensable for
primary infection of monkeys with RhCMV, while suc-
cessful reinfection of monkeys was dependent on down-
regulation of MHC class I molecules mediated by the
RhUS2-11 region. Furthermore, animal studies using wild-
type mouse CMV and variant viruses lacking the immune
evasin genes demonstrated that the evasin molecules con-
tribute to the level of viral load in tissues [120]. Collec-
tively, these studies support the paradigm that viral
immune evasins likely narrow the “window of immune
detection”; thus providing CMV the time to generate viral
loads required for effective dissemination and transmission
of the virus.

The lack of surface class I molecules during a virus
infection makes the infected cell prone to natural killer
(NK) cell lysis. Not surprisingly, HCMV employs various
mechanisms to evade NK cell lysis, an important arm of
the innate immune system for viral clearance [68]. HCMV
has also evolved the capability to concomitantly express
MHC class I homologs in order to escape detection by NK
cells [121]. HCMV glycoproteins can directly bind and
sequester NK-activating ligands that are induced by
infection [122–124]. Additionally, selective, allele-specific
degradation of MHC class I imparts protection against both
CTL-mediated and NK-mediated cell lysis [125–127]. For
more information on viral evasion of NK cell lysis, consult
the following reviews: [60, 128–130]. Considering the
large repertoire of mechanisms that HCMV employs to
suppress the innate immune response and its well-charac-
terized ability to block antigen presentation, it is possible to
posit a temporally coordinated model of systemic HCMV
infection that demonstrates the evolutionary advantage for
innate immune suppression versus humoral immune
evasion.

**Immune evasion during latency, an open frontier**

HCMV commits a large portion of its genome to modu-
lating recognition by the immune system, most notably
through a blockade of MHC class I and II antigen pre-
sentation and NK cell evasion. This control of immune
sensing provides a transient advantage to the virus to allow
for replication of the viral genome, dissemination, and
persistence within the host. However, during a latent

infection, HCMV must utilize a distinct repertoire of
immune evasion strategies because there is very little viral
protein production [3] contributing to the antigenic pool of
peptides and the lack of expression of proteins that mod-
ulate antigen presentation. Studies have shed light on how
HCMV is able to exist in a quiescent state despite the
presence of a strong humoral immune response [28]. Of
particular interest is the UL111.5A region of the genome
that encodes for a viral homolog of the potent immuno-
suppressor interleukin-10 (cmvIL-10). The cmvIL-10 pro-
tein was identified in mononuclear cells from healthy bone
marrow and mobilized peripheral blood allograft donors,
suggesting its possible contribution to immune evasion
[131–135]. Also, surface MHC class II levels were reduced
on latently infected cells in a US2-US11 independent
manner utilizing a granulocyte-macrophage progenitor
model system, likely due to cmvIL-10 [132, 136]. Reduced
levels of MHC class II and an inhibition of allogeneic and
autologous CD4+ T-cell responses were also observed in
HCMV-infected CD34+ progenitors cells [137].

Yet another strategy of HCMV immune evasion may
be to restrict the differentiation of CD34+ progenitor cells
to dendritic cells (DCs), which are potent APCs. In vitro
infection of monocytes inhibited the differentiation of
these cells into DCs [138], thereby limiting their ability to
trigger an immune response. Interestingly, direct infection
of monocyte-derived DCs exhibited reduced abilities for
endocytosis, phagocytosis, and migration in response to
RANTES, MIP-1α, and MIP-3β [139]. These virus-
infected cells also had a reduced ability to induce pro-
liferative responses in alloreactive T cells. More recent
studies have confirmed that DCs derived from HCMV-
infected monocytes have altered phenotypes and func-
tional defects and that these effects are the result of a
blockade in GM-CSF signaling [140]. DCs generated
from PBMCs of patients with HCMV disease were found
to be impaired in their ability to stimulate allogeneic
lymphocytes [141]. Surprisingly, in another report,
HCMV was found to activate CD11c+ DCs and plas-
macytoid DCs, and infected CD11c+ DCs retained full
capacity to stimulate T cells [142]. A recent study dem-
onstrated that extracellular signal-regulated kinase-mito-
gen-activated protein kinase (ERK-MAPK) signaling
induced by IL-6 in interstitial-like DCs can drive reacti-
vation of HCMV infection in an ex vivo latency model
[143]. For a full review of CMV-induced modulation of
DCs, see [139]. Taken together, the current body of work
available regarding HCMV infection of DCs may indicate
that HCMV further modulates immune recognition in
these potent antigen presenting cells and that infection of
monocytes establishes an ideal reservoir for reactivation
from latency.

Concluding remarks

The mechanisms of immune evasion by HCMV bring about fundamental questions regarding viral evasion in the face of host immunity. For example, how are immune evasion mechanisms coordinated? Could the temporal expression of these mechanisms reveal information about the life cycle of the virus? Can the employment of some mechanisms over others, or an interruption in the cascade of viral gene expression, determine the nature of an HCMV infection? As mentioned, HCMV is capable of existing in a latent form for the lifetime of the human host, so how are immune evasion strategies modified to adjust for different states of infection and how are they involved in reactivation of an HCMV infection? Analogously, how is HCMV able to coordinate immune evasion functions in order to reactivate in the face of a fully primed immune system? Links between the differentiation state of infected cells and viral reactivation have already been identified, so there is clearly a sophisticated mechanism by which HCMV senses its cellular environment to control its own propagation. The analysis of HCMV latency is likely to be an area of increased focus in the near future. Knowledge of the molecular mechanisms of HCMV propagation and immune evasion will be the key to studies of pathogenesis and the virus life cycle within the human host.

Acknowledgments DT is supported by an American Heart Association Grant, DTRA contract #W81XWH-10-2-0048, and the Irma T. Hirschl Trust. VN is a post-doctoral trainee supported by the USPHS Institutional Research Training Award T32-AI078892. VR is a pre-doctoral trainee supported in part by an USPHS Institutional Research Training Award T32-AI07647 and the Kadlec Medical Center Foundation. TG is a pre-doctoral trainee supported in part by an USPHS Institutional Research Training Award T32-AI07647 and a Helmsley Fellowship.

References

1. Mocarski ES, Shenk T, Pass RF. Cytomegaloviruses. In: Howley DMKPM, editor. Fields virology. 5th ed. Philadelphia: Wolters Kluwer Health/Lippincott Williams & Wilkins; 2007.
2. Slobedman B, Cao JZ, Avdic S, Webster B, McAllery S, Cheung AK, et al. Human cytomegalovirus latent infection and associated viral gene expression. Futur Microbiol. 2010;5(6):883–900. doi: [10.2217/fmb.10.58](https://doi.org/10.2217/fmb.10.58).
3. Sinclair J. Human cytomegalovirus: latency and reactivation in the myeloid lineage. J Clin Virol. 2008;41(3):180–5. doi: [10.1016/j.jcv.2007.11.014](https://doi.org/10.1016/j.jcv.2007.11.014).
4. Gibson W. Structure and formation of the cytomegalovirus virion. Curr Top Microbiol Immunol. 2008;325:187–204.
5. Compton T, Nowlin DM, Cooper NR. Initiation of human cytomegalovirus infection requires initial interaction with cell surface heparan sulfate. Virology. 1993;193(2):834–41. doi: [10.1006/viro.1993.1192](https://doi.org/10.1006/viro.1993.1192).
6. Isaacson MK, Juckem LK, Compton T. Virus entry and innate immune activation. Curr Top Microbiol Immunol. 2008;325:85–100.
7. Adler B, Scrivano L, Ruzcics Z, Rupp B, Sinzger C, Koszinski U. Role of human cytomegalovirus UL131A in cell type-specific virus entry and release. J Gen Virol. 2006;87(Pt 9):2451–60. doi: [10.1099/vir.0.81921-0](https://doi.org/10.1099/vir.0.81921-0).
8. Hahn G, Revello MG, Patrone M, Percivalle E, Campanini G, Sarasini A, et al. Human cytomegalovirus UL131-128 genes are indispensable for virus growth in endothelial cells and virus transfer to leukocytes. J Virol. 2004;78(18):10023–33. doi: [10.1128/JVI.78.18.10023-10033.2004](https://doi.org/10.1128/JVI.78.18.10023-10033.2004).
9. Gerna G, Percivalle E, Lilleri D, Lozza L, Fornara C, Hahn G, et al. Dendritic-cell infection by human cytomegalovirus is restricted to strains carrying functional UL131-128 genes and mediates efficient viral antigen presentation to CD8+T cells. J Gen Virol. 2005;86(Pt 2):275–84. doi: [10.1099/vir.0.80474-0](https://doi.org/10.1099/vir.0.80474-0).
10. Ryckman BJ, Jarvis MA, Drummond DD, Nelson JA, Johnson DC. Human cytomegalovirus entry into epithelial and endothelial cells depends on genes UL128 to UL150 and occurs by endocytosis and low-pH fusion. J Virol. 2006;80(2):710–22. doi: [10.1128/JVI.80.2.710-722.2006](https://doi.org/10.1128/JVI.80.2.710-722.2006).
11. Ryckman BJ, Rainish BL, Chase MC, Borton JA, Nelson JA, Jarvis MA, et al. Characterization of the human cytomegalovirus gH/gL/UL128-131 complex that mediates entry into epithelial and endothelial cells. J Virol. 2008;82(1):60–70. doi: [10.1128/JVI.01910-07](https://doi.org/10.1128/JVI.01910-07).
12. Straschewski S, Patrone M, Walther P, Gallina A, Mertens T, Frascaroli G. Protein pUL128 of human cytomegalovirus is necessary for monocyte infection and blocking of migration. J Virol. 2011;85(10):5150–8. doi: [10.1128/JVI.02100-10](https://doi.org/10.1128/JVI.02100-10).
13. Wang D, Shenk T. Human cytomegalovirus virion protein complex required for epithelial and endothelial cell tropism. Proc Natl Acad Sci USA. 2005;102(50):18153–8. doi: [10.1073/pnas.0509201102](https://doi.org/10.1073/pnas.0509201102).
14. Cha TA, Tom E, Kemble GW, Duke GM, Mocarski ES, Spaete RR. Human cytomegalovirus clinical isolates carry at least 19 genes not found in laboratory strains. J Virol. 1996;70(1):78–83.
15. Dolan A, Cunningham C, Hector RD, Hassan-Walker AF, Lee L, Addison C, et al. Genetic content of wild-type human cytomegalovirus. J Gen Virol. 2004;85(Pt 5):1301–12.
16. Vanarsdall AL, Chase MC, Johnson DC. Human cytomegalovirus glycoprotein gO complexes with gH/gL, promoting interference with viral entry into human fibroblasts but not entry into epithelial cells. J Virol. 2011;85(22):11638–45. doi: [10.1128/JVI.05659-11](https://doi.org/10.1128/JVI.05659-11).
17. Lopper M, Compton T. Coiled-coil domains in glycoproteins B and H are involved in human cytomegalovirus membrane fusion. J Virol. 2004;78(15):8333–41. doi: [10.1128/JVI.78.15.8333-8341.2004](https://doi.org/10.1128/JVI.78.15.8333-8341.2004).
18. Ogawa-Goto K, Tanaka K, Gibson W, Moriishi E, Miura Y, Kurata T, et al. Microtubule network facilitates nuclear targeting of human cytomegalovirus capsid. J Virol. 2003;77(15):8541–7.
19. Cherrington JM, Mocarski ES. Human cytomegalovirus iel transactivates the alpha promoter-enhancer via an 18-base-pair repeat element. J Virol. 1989;63(3):1435–40.
20. Meier JL, Stinski MF. Effect of a modulator deletion on transcription of the human cytomegalovirus major immediate-early genes in infected undifferentiated and differentiated cells. J Virol. 1997;71(2):1246–55.
21. Kalejta RF. Tegument proteins of human cytomegalovirus. Microbiol Mol Biol Rev. 2008;72(2):249–65. doi: [10.1128/MMBR.00040-07](https://doi.org/10.1128/MMBR.00040-07). table of contents.
22. Gant TM, Wilson KL. Nuclear assembly. Annu Rev Cell Dev Biol. 1997;13:669–95. doi: [10.1146/annurev.cellbio.13.1.669](https://doi.org/10.1146/annurev.cellbio.13.1.669).
23. Radsak KD, Brucher KH, Georgatos SD. Focal nuclear envelope lesions and specific nuclear lamin A/C dephosphorylation during infection with human cytomegalovirus. Eur J Cell Biol. 1991;54(2):299–304.

Immunology Institute at the Mount Sinai School of Medicine

24. Eickmann M, Gicklhorn D, Radsak K. Glycoprotein trafficking in virion morphogenesis. In: Reddehase M, Lemmermann N, editors. Cytomegaloviruses: molecular biology and immunology. Wymondham: Caister Academic Press; 2006. p. 245–64.

25. Sanchez V, Angeletti PC, Engler JA, Britt WJ. Localization of human cytomegalovirus structural proteins to the nuclear matrix of infected human fibroblasts. J Virol. 1998;72(4):3321–9.

26. Trus BL, Gibson W, Cheng N, Steven AC. Capsid structure of simian cytomegalovirus from cryoelectron microscopy: evidence for tegument attachment sites. J Virol. 1999;73(3):2181–92.

27. Liu W, Zhao Y, Biegelke B. Analysis of human cytomegalovirus US3 gene products. Virology. 2002;301(1):32–42.

28. Loewendorf A, Benedict CA. Modulation of host innate and adaptive immune defenses by cytomegalovirus: timing is everything. J Intern Med. 2010;267(5):483–501. doi:10.1111/j.1365-2796.2010.02220.x.

29. Sinclair J, Sissons P. Latency and reactivation of human cytomegalovirus. J Gen Virol. 2006;87(Pt 7):1763–79. doi:10.1099/vir.0.81891-0.

30. Goodrum FD, Jordan CT, High K, Shenk T. Human cytomegalovirus gene expression during infection of primary hematopoietic progenitor cells: a model for latency. Proc Natl Acad Sci USA. 2002;99(25):16255–60. doi:10.1073/pnas.252630899.

31. Hahn G, Jores R, Mocarski ES. Cytomegalovirus remains latent in a common precursor of dendritic and myeloid cells. Proc Natl Acad Sci USA. 1998;95(7):3937–42.

32. Minton EJ, Tysoe C, Sinclair JH, Sissons JG. Human cytomegalovirus infection of the monocyte/macrophage lineage in bone marrow. J Virol. 1994;68(6):4017–21.

33. Reeves MB, MacAry PA, Lehner PJ, Sissons JG, Sinclair JH. Latency, chromatin remodeling, and reactivation of human cytomegalovirus in the dendritic cells of healthy carriers. Proc Natl Acad Sci USA. 2005;102(11):4140–5. doi:10.1073/pnas.0408994102.

34. Drew WL. Diagnosis of cytomegalovirus infection. Rev Infect Dis. 1988;10(Suppl 3):S468–76.

35. Rubin RH. Impact of cytomegalovirus infection on organ transplant recipients. Rev Infect Dis. 1990;12(Suppl 7):S754–66.

36. Soderberg-Naucler C. Does cytomegalovirus play a causative role in the development of various inflammatory diseases and cancer? J Intern Med. 2006;259(3):219–46. doi:10.1111/j.1365-2796.2006.01618.x.

37. Adler SP. Transfusion-associated cytomegalovirus infections. Rev Infect Dis. 1983;5(6):977–93.

38. Tolpin MD, Stewart JA, Warren D, Mojica BA, Collins MA, Doveikis SA, et al. Transfusion transmission of cytomegalovirus confirmed by restriction endonuclease analysis. J Pediatr. 1985;107(6):953–6.

39. Yeager AS, Grumet FC, Hafleigh EB, Arvin AM, Bradley JS, Prober CG. Prevention of transfusion-acquired cytomegalovirus infections in newborn infants. J Pediatr. 1981;98(2):281–7.

40. Jordan MC. Latent infection and the elusive cytomegalovirus. Rev Infect Dis. 1983;5(2):205–15.

41. de Graan-Hentzen YC, Gratama JW, Mudde GC, Verdonck LF, Houbiers JG, Brand A, et al. Prevention of primary cytomegalovirus infection in patients with hematologic malignancies by intensive white cell depletion of blood products. Transfusion. 1989;29(9):757–60.

42. Stanier P, Taylor DL, Kitchen AD, Wales N, Tryhorn Y, Tyms AS. Persistence of cytomegalovirus in mononuclear cells in peripheral blood from blood donors. BMJ. 1989;299(6704):897–8.

43. Taylor-Wiedeman J, Sissons JG, Borysiewicz LK, Sinclair JH. Monocytes are a major site of persistence of human cytomegalovirus

in peripheral blood mononuclear cells. J Gen Virol. 1991;72(Pt 9):2059–64.

44. Mendelson M, Monard S, Sissons P, Sinclair J. Detection of endogenous human cytomegalovirus in CD34+ bone marrow progenitors. J Gen Virol. 1996;77(Pt 12):3099–102.

45. Slobedman B, Mocarski ES. Quantitative analysis of latent human cytomegalovirus. J Virol. 1999;73(6):4806–12.

46. Bolovan-Fritts CA, Mocarski ES, Wiedeman JA. Peripheral blood CD14(+) cells from healthy subjects carry a circular conformation of latent cytomegalovirus genome. Blood. 1999;93(1):394–8.

47. Jarvis MA, Fish KN, Soderberg-Naucler C, Streblow DN, Meyers HL, Thomas G, et al. Retrieval of human cytomegalovirus glycoprotein B from cell surface is not required for virus envelopment in astrocytoma cells. J Virol. 2002;76(10):5147–55.

48. Iwasaki A, Medzhitov R. Regulation of adaptive immunity by the innate immune system. Science. 2010;327(5963):291–5. doi:327/5963/291.

49. Kumagai Y, Akira S. Identification and functions of pattern-recognition receptors. J Allergy Clin Immunol. 2010;125(5):985–92. doi:10.1016/j.jaci.2010.01.058.

50. Hirano M, Das S, Guo P, Cooper MD. The evolution of adaptive immunity in vertebrates. Adv Immunol. 2011;109:125–57. doi:10.1016/B978-0-12-387664-5.00004-2.

51. Zhang N, Bevan MJ. CD8(+) T cells: foot soldiers of the immune system. Immunity. 2011;35(2):161–8. doi:10.1016/j.immuni.2011.07.010.

52. Sheridan BS, Lefrancois L. Regional and mucosal memory T cells. Nat Immunol. 2011;12(6):485–91.

53. Neefjes J, Jongsma ML, Paul P, Bakke O. Towards a systems understanding of MHC class I and MHC class II antigen presentation. Nat Rev Immunol. 2011;11(12):823–36. doi:10.1038/nri3084.

54. Kushawah R, Hu J. Complexity of dendritic cell subsets and their function in the host immune system. Immunology. 2011;133(4):409–19. doi:10.1111/j.1365-2567.2011.03457.x.

55. Nace G, Evankovich J, Eid R, Tsung A. Dendritic cells and damage-associated molecular patterns: endogenous danger signals linking innate and adaptive immunity. J Innate Immun. 2012;4(1):6–15. doi:10.1159/000334245.

56. Rocha N, Neefjes J. MHC class II molecules on the move for successful antigen presentation. EMBO J. 2008;27(1):1–5. doi:10.1038/sj.emboj.7601945.

57. Chapman DC, Williams DB. ER quality control in the biosynthesis of MHC class I molecules. Semin Cell Dev Biol. 2010;21(5):512–9. doi:10.1016/j.semcdb.2009.12.013.

58. Kurts C, Robinson BW, Knolle PA. Cross-priming in health and disease. Nat Rev Immunol. 2010;10(6):403–14. doi:10.1038/nri2780.

59. Kracker S, Durandy A. Insights into the B cell specific process of immunoglobulin class switch recombination. Immunol Lett. 2011;138(2):97–103. doi:10.1016/j.imlet.2011.02.004.

60. Tortorella D, Gewurz BE, Furman MH, Schust DJ, Ploegh HL. Viral subversion of the immune system. Annu Rev Immunol. 2000;18:861–926. doi:10.1146/annurev.immunol.18.1.861.

61. Barnes PD, Grundy JE. Down-regulation of the class I HLA heterodimer and beta 2-microglobulin on the surface of cells infected with cytomegalovirus. J Gen Virol. 1992;73(Pt 9):2395–403.

62. Beersma MF, Bijlmakers MJ, Ploegh HL. Human cytomegalovirus down-regulates HLA class I expression by reducing the stability of class I H chains. J Immunol. 1993;151(9):4455–64.

63. del Val M, Hengel H, Hacker H, Hartlaub U, Ruppert T, Lucin P, et al. Cytomegalovirus prevents antigen presentation by blocking the transport of peptide-loaded major histocompati-

bility complex class I molecules into the medial-Golgi compartment. J Exp Med. 1992; 176(3): 729–38.

64. Gilbert MJ, Riddell SR, Li CR, Greenberg PD. Selective interference with class I major histocompatibility complex presentation of the major immediate-early protein following infection with human cytomegalovirus. J Virol. 1993; 67(6): 3461–9.

65. Yamashita Y, Shimokata K, Mizuno S, Yamaguchi H, Nishiyama Y. Down-regulation of the surface expression of class I MHC antigens by human cytomegalovirus. Virology. 1993; 193(2): 727–36. doi: [10.1006/viro.1993.1181](https://doi.org/10.1006/viro.1993.1181).

66. Jones TR, Hanson LK, Sun L, Slater JS, Stenberg RM, Campbell AE. Multiple independent loci within the human cytomegalovirus unique short region down-regulate expression of major histocompatibility complex class I heavy chains. J Virol. 1995; 69(8): 4830–41.

67. Ahn K, Angulo A, Ghazal P, Peterson PA, Yang Y, Fruh K. Human cytomegalovirus inhibits antigen presentation by a sequential multistep process. Proc Natl Acad Sci USA. 1996; 93(20): 10990–5.

68. Jones TR, Wiertz EJ, Sun L, Fish KN, Nelson JA, Ploegh HL. Human cytomegalovirus US3 impairs transport and maturation of major histocompatibility complex class I heavy chains. Proc Natl Acad Sci USA. 1996; 93(21): 11327–33.

69. Lee S, Park B, Ahn K. Determinant for endoplasmic reticulum retention in the luminal domain of the human cytomegalovirus US3 glycoprotein. J Virol. 2003; 77(3): 2147–56.

70. Zhao Y, Biegalke BJ. Functional analysis of the human cytomegalovirus immune evasion protein, pUS3 (22 kDa). Virology. 2003; 315(2): 353–61.

71. Park B, Kim Y, Shin J, Lee S, Cho K, Fruh K, et al. Human cytomegalovirus inhibits tapasin-dependent peptide loading and optimization of the MHC class I peptide cargo for immune evasion. Immunity. 2004; 20(1): 71–85.

72. Jun Y, Kim E, Jin M, Sung HC, Han H, Geraghty DE, et al. Human cytomegalovirus gene products US3 and US6 downregulate trophoblast class I MHC molecules. J Immunol. 2000; 164(2): 805–11.

73. Noriega VM, Tortorella D. Human cytomegalovirus-encoded immune modulators partner to downregulate major histocompatibility complex class I molecules. J Virol. 2009; 83(3): 1359–67. doi: [10.1128/JVI.01324-08](https://doi.org/10.1128/JVI.01324-08).

74. Hegde NR, Tomazin RA, Wisner TW, Dunn C, Boname JM, Lewinsohn DM, et al. Inhibition of HLA-DR assembly, transport, and loading by human cytomegalovirus glycoprotein US3: a novel mechanism for evading major histocompatibility complex class II antigen presentation. J Virol. 2002; 76(21): 10929–41.

75. Jones TR, Sun L. Human cytomegalovirus US2 destabilizes major histocompatibility complex class I heavy chains. J Virol. 1997; 71(4): 2970–9.

76. Wiertz EJ, Jones TR, Sun L, Bogyo M, Geuze HJ, Ploegh HL. The human cytomegalovirus US11 gene product dislocates MHC class I heavy chains from the endoplasmic reticulum to the cytosol. Cell. 1996; 84(5): 769–79.

77. Wiertz EJ, Tortorella D, Bogyo M, Yu J, Mothes W, Jones TR, et al. Sec61-mediated transfer of a membrane protein from the endoplasmic reticulum to the proteasome for destruction. Nature. 1996; 384(6608): 432–8. doi: [10.1038/384432a0](https://doi.org/10.1038/384432a0).

78. Smith MH, Ploegh HL, Weissman JS. Road to ruin: targeting proteins for degradation in the endoplasmic reticulum. Science. 2011; 334(6059): 1086–90. doi: [10.1126/science.1209235](https://doi.org/10.1126/science.1209235).

79. Lilley BN, Tortorella D, Ploegh HL. Dislocation of a type I membrane protein requires interactions between membrane-spanning segments within the lipid bilayer. Mol Biol Cell. 2003; 14(9): 3690–8. doi: [10.1091/mbc.E03-03-0192](https://doi.org/10.1091/mbc.E03-03-0192).

80. Noriega VM, Tortorella D. A bipartite trigger for dislocation directs the proteasomal degradation of an endoplasmic reticulum

membrane glycoprotein. J Biol Chem. 2008; 283(7): 4031–43. doi: [10.1074/jbc.M706283200](https://doi.org/10.1074/jbc.M706283200).

81. Oresic K, Noriega V, Andrews L, Tortorella D. A structural determinant of human cytomegalovirus US2 dictates the downregulation of class I major histocompatibility molecules. J Biol Chem. 2006; 281(28): 19395–406. doi: [10.1074/jbc.M601026200](https://doi.org/10.1074/jbc.M601026200).

82. Rehm A, Engelsberg A, Tortorella D, Korner JJ, Lehmann I, Ploegh HL, et al. Human cytomegalovirus gene products US2 and US11 differ in their ability to attack major histocompatibility class I heavy chains in dendritic cells. J Virol. 2002; 76(10): 5043–50.

83. Tomazin R, Boname J, Hegde NR, Lewinsohn DM, Altschuler Y, Jones TR, et al. Cytomegalovirus US2 destroys two components of the MHC class II pathway, preventing recognition by CD4+ T cells. Nat Med. 1999; 5(9): 1039–43. doi: [10.1038/12478](https://doi.org/10.1038/12478).

84. Barel MT, Ressing M, Pizzato N, van Leeuwen D, Le Bouteiller P, Lenfant F, et al. Human cytomegalovirus-encoded US2 differentially affects surface expression of MHC class I locus products and targets membrane-bound, but not soluble HLA-G1 for degradation. J Immunol. 2003; 171(12): 6757–65.

85. Gewurz BE, Wang EW, Tortorella D, Schust DJ, Ploegh HL. Human cytomegalovirus US2 endoplasmic reticulum-lumenal domain dictates association with major histocompatibility complex class I in a locus-specific manner. J Virol. 2001; 75(11): 5197–204. doi: [10.1128/JVI.75.11.5197-5204.2001](https://doi.org/10.1128/JVI.75.11.5197-5204.2001).

86. Barel MT, Pizzato N, van Leeuwen D, Bouteiller PL, Wiertz EJ, Lenfant F. Amino acid composition of alpha1/alpha2 domains and cytoplasmic tail of MHC class I molecules determine their susceptibility to human cytomegalovirus US11-mediated downregulation. Eur J Immunol. 2003; 33(6): 1707–16. doi: [10.1002/eji.200323912](https://doi.org/10.1002/eji.200323912).

87. Oresic K, Tortorella D. Endoplasmic reticulum chaperones participate in human cytomegalovirus US2-mediated degradation of class I major histocompatibility complex molecules. J Gen Virol. 2008; 89(Pt 5): 1122–30. doi: [10.1099/vir.0.83516-0](https://doi.org/10.1099/vir.0.83516-0).

88. Gewurz BE, Gaudet R, Tortorella D, Wang EW, Ploegh HL, Wiley DC. Antigen presentation subverted: structure of the human cytomegalovirus protein US2 bound to the class I molecule HLA-A2. Proc Natl Acad Sci USA. 2001; 98(12): 6794–9. doi: [10.1073/pnas.121172898](https://doi.org/10.1073/pnas.121172898).

89. Story CM, Furman MH, Ploegh HL. The cytosolic tail of class I MHC heavy chain is required for its dislocation by the human cytomegalovirus US2 and US11 gene products. Proc Natl Acad Sci USA. 1999; 96(15): 8516–21.

90. Schust DJ, Tortorella D, Seebach J, Phan C, Ploegh HL. Trophoblast class I major histocompatibility complex (MHC) products are resistant to rapid degradation imposed by the human cytomegalovirus (HCMV) gene products US2 and US11. J Exp Med. 1998; 188(3): 497–503.

91. Besold K, Frankenberg N, Pepperl-Klindworth S, Kuball J, Theobald M, Hahn G, et al. Processing and MHC class I presentation of human cytomegalovirus pp 65-derived peptides persist despite gpUS2-11-mediated immune evasion. J Gen Virol. 2007; 88(Pt 5): 1429–39. doi: [10.1099/vir.0.82686-0](https://doi.org/10.1099/vir.0.82686-0).

92. Besold K, Wills M, Plachter B. Immune evasion proteins gpUS2 and gpUS11 of human cytomegalovirus incompletely protect infected cells from CD8 T cell recognition. Virology. 2009; 391(1): 5–19. doi: [10.1016/j.virol.2009.06.004](https://doi.org/10.1016/j.virol.2009.06.004).

93. Vembar SS, Brodsky JL. One step at a time: endoplasmic reticulum-associated degradation. Nat Rev Mol Cell Biol. 2008; 9(12): 944–57. doi: [10.1038/nrm2546](https://doi.org/10.1038/nrm2546).

94. Hegde NR, Chevalier MS, Wisner TW, Denton MC, Shire K, Frappier L, et al. The role of BiP in endoplasmic reticulum-associated degradation of major histocompatibility complex class I heavy chain induced by cytomegalovirus proteins. J Biol Chem. 2006; 281(30): 20910–9. doi: [10.1074/jbc.M602989200](https://doi.org/10.1074/jbc.M602989200).

Immunology Institute at the Mount Sinai School of Medicine

95. Lilley BN, Ploegh HL. A membrane protein required for dislocation of misfolded proteins from the ER. Nature. 2004; 429(6994):834–40. doi:10.1038/nature02592.

96. Mueller B, Klemm EJ, Spooner E, Claessen JH, Ploegh HL. SEL1L nucleates a protein complex required for dislocation of misfolded glycoproteins. Proc Natl Acad Sci USA. 2008; 105(34):12325–30. doi:10.1073/pnas.0805371105.

97. Mueller B, Lilley BN, Ploegh HL. SEL1L, the homologue of yeast Hrd3p, is involved in protein dislocation from the mammalian ER. J Cell Biol. 2006; 175(2):261–70. doi:10.1083/jcb.200605196.

98. Ye Y, Meyer HH, Rapoport TA. The AAA ATPase Cdc48/p97 and its partners transport proteins from the ER into the cytosol. Nature. 2001; 414(6864):652–6. doi:10.1038/414652a.

99. Ye Y, Shibata Y, Yun C, Ron D, Rapoport TA. A membrane protein complex mediates retro-translocation from the ER lumen into the cytosol. Nature. 2004; 429(6994):841–7. doi:10.1038/nature02656.

100. Loureiro J, Lilley BN, Spooner E, Noriega V, Tortorella D, Ploegh HL. Signal peptide peptidase is required for dislocation from the endoplasmic reticulum. Nature. 2006; 441(7095): 894–7. doi:10.1038/nature04830.

101. Soetandyo N, Ye Y. The p97 ATPase dislocates MHC class I heavy chain in US2-expressing cells via a Ufd1-Npl4-independent mechanism. J Biol Chem. 2010; 285(42):32352–9. doi: 10.1074/jbc.M110.131649.

102. Furman MH, Dey N, Tortorella D, Ploegh HL. The human cytomegalovirus US10 gene product delays trafficking of major histocompatibility complex class I molecules. J Virol. 2002; 76(22):11753–6.

103. Park B, Spooner E, Houser BL, Strominger JL, Ploegh HL. The HCMV membrane glycoprotein US10 selectively targets HLA-G for degradation. J Exp Med. 2010; 207(9):2033–41. doi: 10.1084/jem.20091793.

104. Lehner PJ, Karttunen JT, Wilkinson GW, Cresswell P. The human cytomegalovirus US6 glycoprotein inhibits transporter associated with antigen processing-dependent peptide translocation. Proc Natl Acad Sci USA. 1997; 94(13):6904–9.

105. Ahn K, Gruhler A, Galocha B, Jones TR, Wiertz EJ, Ploegh HL, et al. The ER-luminal domain of the HCMV glycoprotein US6 inhibits peptide translocation by TAP. Immunity. 1997; 6(5): 613–21.

106. Dugan GE, Hewitt EW. Structural and functional dissection of the human cytomegalovirus immune evasion protein US6. J Virol. 2008; 82(7):3271–82. doi:10.1128/JVI.01705-07.

107. Hewitt EW, Gupta SS, Lehner PJ. The human cytomegalovirus gene product US6 inhibits ATP binding by TAP. EMBO J. 2001; 20(3):387–96. doi:10.1093/emboj/20.3.387.

108. Kyritsis C, Gorbulev S, Hutschenreiter S, Pawlitschko K, Abele R, Tampe R. Molecular mechanism and structural aspects of transporter associated with antigen processing inhibition by the cytomegalovirus protein US6. J Biol Chem. 2001; 276(51):48031–9. doi:10.1074/jbc.M108528200.

109. Kim Y, Park B, Cho S, Shin J, Cho K, Jun Y, et al. Human cytomegalovirus UL18 utilizes US6 for evading the NK and T-cell responses. PLoS Pathog. 2008; 4(8):e1000123. doi: 10.1371/journal.ppat.1000123.

110. Park B, Oh H, Lee S, Song Y, Shin J, Sung YC, et al. The MHC class I homolog of human cytomegalovirus is resistant to downregulation mediated by the unique short region protein (US) 2, US3, US6, and US11 gene products. J Immunol. 2002; 168(7): 3464–9.

111. Kim S, Lee S, Shin J, Kim Y, Evnouchidou I, Kim D, et al. Human cytomegalovirus microRNA miR-US4-1 inhibits CD8(+) T cell responses by targeting the amino peptidase ERAP1. Nat Immunol. 2011; 12(10):984–91. doi:10.1038/ni.2097.

112. Saric T, Chang SC, Hattori A, York IA, Markant S, Rock KL, et al. An IFN-gamma-induced aminopeptidase in the ER, ERAP1, trims precursors to MHC class I-presented peptides. Nat Immunol. 2002; 3(12):1169–76. doi:10.1038/ni859.

113. Gilbert MJ, Riddell SR, Plachter B, Greenberg PD. Cytomegalovirus selectively blocks antigen processing and presentation of its immediate-early gene product. Nature. 1996; 383(6602): 720–2. doi:10.1038/383720a0.

114. Trgovcich J, Cebulla C, Zimmerman P, Sedmak DD. Human cytomegalovirus protein pp71 disrupts major histocompatibility complex class I cell surface expression. J Virol. 2006; 80(2): 951–63. doi:10.1128/JVI.80.2.951-963.2006.

115. Gallina A, Simoncini L, Garbelli S, Percivalle E, Pedrali-Noy G, Lee KS, et al. Polo-like kinase 1 as a target for human cytomegalovirus pp65 lower matrix protein. J Virol. 1999; 73(2):1468–78.

116. Browne EP, Shenk T. Human cytomegalovirus UL83-coded pp65 virion protein inhibits antiviral gene expression in infected cells. Proc Natl Acad Sci USA. 2003; 100(20):11439–44. doi: 10.1073/pnas.1534570100.

117. Odeberg J, Plachter B, Branden L, Soderberg-Naucler C. Human cytomegalovirus protein pp65 mediates accumulation of HLA-DR in lysosomes and destruction of the HLA-DR alpha-chain. Blood. 2003; 101(12):4870–7. doi:10.1182/blood-2002-05-1504.

118. Khan N, Bruton R, Taylor GS, Cobbold M, Jones TR, Rickinson AB, et al. Identification of cytomegalovirus-specific cytotoxic T lymphocytes in vitro is greatly enhanced by the use of recombinant virus lacking the US2 to US11 region or modified vaccinia virus Ankara expressing individual viral genes. J Virol. 2005; 79(5):2869–79. doi:10.1128/JVI.79.5.2869-2879.2005.

119. Hansen SG, Powers CJ, Richards R, Ventura AB, Ford JC, Siess D, et al. Evasion of CD8+ T cells is critical for superinfection by cytomegalovirus. Science. 2010; 328(5974):102–6. doi:10.1126/science.1185350.

120. Lemmermann NA, Bohm V, Holtappels R, Reddehase MJ. In vivo impact of cytomegalovirus evasion of CD8 T-cell immunity: facts and thoughts based on murine models. Virus Res. 2011; 157(2): 161–74. doi:10.1016/j.virusres.2010.09.022.

121. Beck S, Barrell BG. Human cytomegalovirus encodes a glycoprotein homologous to MHC class-I antigens. Nature. 1988; 331(6153):269–72. doi:10.1038/331269a0.

122. Cosman D, Fanger N, Borges L, Kubin M, Chin W, Peterson L, et al. A novel immunoglobulin superfamily receptor for cellular and viral MHC class I molecules. Immunity. 1997; 7(2):273–82.

123. Kubin M, Cassiano L, Chalupny J, Chin W, Cosman D, Fanslow W, et al. ULBP1, 2, 3: novel MHC class I-related molecules that bind to human cytomegalovirus glycoprotein UL16, activate NK cells. Eur J Immunol. 2001; 31(5):1428–37. doi:10.1002/1521-4141(200105)31:5<1428:AID-IMMU1428>3.0.CO;2-4.

124. Welte SA, Sinzger C, Lutz SZ, Singh-Jasuja H, Sampaio KL, Eknigk U, et al. Selective intracellular retention of virally induced NKG2D ligands by the human cytomegalovirus UL16 glycoprotein. Eur J Immunol. 2003; 33(1):194–203. doi:10.1002/immu.200390022.

125. Borrego F, Ulbrecht M, Weiss EH, Coligan JE, Brooks AG. Recognition of human histocompatibility leukocyte antigen (HLA)-E complexed with HLA class I signal sequence-derived peptides by CD94/NKG2 confers protection from natural killer cell-mediated lysis. J Exp Med. 1998; 187(5):813–8.

126. Braud VM, Allan DS, O'Callaghan CA, Soderstrom K, D'Andrea A, Ogg GS, et al. HLA-E binds to natural killer cell receptors CD94/NKG2A, B and C. Nature. 1998; 391(6669):795–9. doi:10.1038/35869.

127. Lee N, Llano M, Carretero M, Ishitani A, Navarro F, Lopez-Botet M, et al. HLA-E is a major ligand for the natural killer inhibitory receptor CD94/NKG2A. Proc Natl Acad Sci USA. 1998; 95(9):5199–204.

128. Lanier LL. Up on the tightrope: natural killer cell activation and inhibition. Nat Immunol. 2008;9(5):495–502. doi:[10.1038/ni1581](https://doi.org/10.1038/ni1581).

129. Orange JS, Fassett MS, Koopman LA, Boyson JE, Strominger JL. Viral evasion of natural killer cells. Nat Immunol. 2002;3(11):1006–12. doi:[10.1038/ni1102-1006](https://doi.org/10.1038/ni1102-1006).

130. Wilkinson GW, Tomasec P, Stanton RJ, Armstrong M, Prod'homme V, Aicheler R, et al. Modulation of natural killer cells by human cytomegalovirus. J Clin Virol. 2008;41(3):206–12. doi:[10.1016/j.jcv.2007.10.027](https://doi.org/10.1016/j.jcv.2007.10.027).

131. Chang WL, Baumgarth N, Yu D, Barry PA. Human cytomegalovirus-encoded interleukin-10 homolog inhibits maturation of dendritic cells and alters their functionality. J Virol. 2004;78(16):8720–31. doi:[10.1128/JVI.78.16.8720-8731.2004](https://doi.org/10.1128/JVI.78.16.8720-8731.2004).

132. Jenkins C, Abendroth A, Slobedman B. A novel viral transcript with homology to human interleukin-10 is expressed during latent human cytomegalovirus infection. J Virol. 2004;78(3):1440–7.

133. Kotenko SV, Saccani S, Izotova LS, Mirochnichenko OV, Pestka S. Human cytomegalovirus harbors its own unique IL-10 homolog (cmvIL-10). Proc Natl Acad Sci USA. 2000;97(4):1695–700.

134. Raftery MJ, Wieland D, Gronewald S, Kraus AA, Giese T, Schonrich G. Shaping phenotype, function, and survival of dendritic cells by cytomegalovirus-encoded IL-10. J Immunol. 2004;173(5):3383–91.

135. Spencer JV, Lockridge KM, Barry PA, Lin G, Tsang M, Penfold ME, et al. Potent immunosuppressive activities of cytomegalovirus-encoded interleukin-10. J Virol. 2002;76(3):1285–92.

136. Slobedman B, Mocarski ES, Arvin AM, Mellins ED, Abendroth A. Latent cytomegalovirus down-regulates major histocompatibility complex class II expression on myeloid progenitors. Blood. 2002;100(8):2867–73. doi:[10.1182/blood.V100.8.2867](https://doi.org/10.1182/blood.V100.8.2867).

137. Cheung AK, Gottlieb DJ, Plachter B, Pepperl-Klindworth S, Avdic S, Cunningham AL, et al. The role of the human

cytomegalovirus UL111A gene in down-regulating CD4+ T-cell recognition of latently infected cells: implications for virus elimination during latency. Blood. 2009;114(19):4128–37. doi:[10.1182/blood-2008-12-197111](https://doi.org/10.1182/blood-2008-12-197111).

138. Gredmark S, Soderberg-Naucler C. Human cytomegalovirus inhibits differentiation of monocytes into dendritic cells with the consequence of depressed immunological functions. J Virol. 2003;77(20):10943–56.

139. Rolle A, Olweus J. Dendritic cells in cytomegalovirus infection: viral evasion and host countermeasures. APMIS. 2009;117(5–6):413–26. doi:[10.1111/j.1600-0463.2009.02449.x](https://doi.org/10.1111/j.1600-0463.2009.02449.x).

140. Carlier J, Martin H, Mariame B, Rauwel B, Mengelle C, Weclawiak H, et al. Paracrine inhibition of GM-CSF signaling by human cytomegalovirus in monocytes differentiating to dendritic cells. Blood. 2011;118(26):6783–92. doi:[10.1182/blood-2011-02-337956](https://doi.org/10.1182/blood-2011-02-337956).

141. Frascaroli G, Varani S, Mastroianni A, Britton S, Gibellini D, Rossini G, et al. Dendritic cell function in cytomegalovirus-infected patients with mononucleosis. J Leukoc Biol. 2006;79(5):932–40. doi:[10.1189/jlb.0905499](https://doi.org/10.1189/jlb.0905499).

142. Kvale EO, Dalgaard J, Lund-Johansen F, Rollag H, Farkas L, Midtvedt K, et al. CD11c+ dendritic cells and plasmacytoid DCs are activated by human cytomegalovirus and retain efficient T cell-stimulatory capability upon infection. Blood. 2006;107(5):2022–9. doi:[10.1182/blood-2005-05-2016](https://doi.org/10.1182/blood-2005-05-2016).

143. Reeves MB, Compton T. Inhibition of inflammatory interleukin-6 activity via extracellular signal-regulated kinase-mitogen-activated protein kinase signaling antagonizes human cytomegalovirus reactivation from dendritic cells. J Virol. 2011;85(23):12750–8. doi:[10.1128/JVI.05878-11](https://doi.org/10.1128/JVI.05878-11).

144. Shamu CE, Flierman D, Ploegh HL, Rapoport TA, Chau V. Polyubiquitination is required for US11-dependent movement of MHC class I heavy chain from endoplasmic reticulum into cytosol. Mol Biol Cell. 2001;12(8):2546–55.
