
The Capicua repressor – a general sensor of RTK signaling in development and disease

Gerardo Jiménez${}^{1,*}$, Stanislav Y. Shvartsman${}^{2}$ and Ze'ev Paroush${}^{3}$

${}^{1}$Institució Catalana de Recerca i Estudis Avançats and Institut de Biologia Molecular de Barcelona-CSIC, Parc Científic de Barcelona, Barcelona 08028, Spain  
${}^{2}$Department of Chemical and Biological Engineering and Lewis-Sigler Institute for Integrative Genomics, Princeton University, Princeton, NJ 08544, USA  
${}^{3}$Department of Developmental Biology and Cancer Research, IMRIC, Faculty of Medicine, The Hebrew University, Jerusalem 91120, Israel  

*Author for correspondence (gjcbmc@ibmb.csic.es)

Journal of Cell Science 125, 1383–1391  
© 2012. Published by The Company of Biologists Ltd  
doi: 10.1242/jcs.092965

## Summary

Receptor tyrosine kinase (RTK) signaling pathways control multiple cellular decisions in metazoans, often by regulating the expression of downstream genes. In *Drosophila melanogaster* and other systems, E-twenty-six (ETS) transcription factors are considered to be the predominant nuclear effectors of RTK pathways. Here, we highlight recent progress in identifying the HMG-box protein Capicua (CIC) as a key sensor of RTK signaling in both *Drosophila* and mammals. Several studies have shown that CIC functions as a repressor of RTK-responsive genes, keeping them silent in the absence of signaling. Following the activation of RTK signaling, CIC repression is relieved, and this allows the expression of the targeted gene in response to local or ubiquitous activators. This regulatory switch is essential for several RTK responses in *Drosophila*, from the determination of cell fate to cell proliferation. Furthermore, increasing evidence supports the notion that this mechanism is conserved in mammals, where CIC has been implicated in cancer and neurodegeneration. In addition to summarizing our current knowledge on CIC, we also discuss the implications of these findings for our understanding of RTK signaling specificity in different biological processes.

### Key words: Capicua, Cell proliferation, CIC, MAPK, RTK signaling, Transcriptional repression, Tumor suppressor

---

## Introduction

Receptor tyrosine kinase (RTK) signaling pathways regulate many biological processes in all metazoans. Their activities elicit diverse cellular responses, such as proliferation, differentiation, metabolism and migration, and abnormal RTK signaling can lead to multiple diseases, most notably cancer. RTK signaling is initiated following the binding of extracellular ligands to cell-surface RTKs, which then typically oligomerize, and either auto- or trans-phosphorylate tyrosine residues in their intracellular domains. This, in turn, stimulates an array of intracellular signaling cascades that primarily act through the small GTPase Ras, and a core of three serine/threonine kinases [Raf, mitogen-activated protein kinase kinase (MEK) and mitogen-activated protein kinase (MAPK, also known as ERK)], but also through the phosphatidylinositol-3-kinase (PI3K) and phospholipase Cγ (PLCγ) pathways (Lemmon and Schlessinger, 2010).

Because RTK signaling pathways often lead to changes in gene expression, the nuclear factors that are directly phosphorylated by components of these pathways, for example by MAPK, have a key role in the interpretation of RTK responses. In *Drosophila melanogaster*, where many RTK responses have been studied in detail, the best-characterized RTK-Ras-MAPK effectors belong to the ETS transcription factor superfamily. Thus, two ETS factors, the activator Pointed-P2 and the repressor Yan, mediate multiple RTK-regulated decisions and are direct substrates of MAPK (O'Neill et al., 1994; Brunner et al., 1994; Rebay and Rubin, 1995; Gabay et al., 1996; Tootle and Rebay, 2005). Similarly, ETS proteins in other species, such as *Caenorhabditis elegans* LIN-1 and mammalian ELK1, are important targets of Ras-MAPK regulation (Hart et al., 2000; Yordi and Muise-Helmericks, 2000), underscoring the ancient, widespread functions of ETS proteins in Ras-MAPK signaling.

Nevertheless, additional transcriptional regulators that also function as important mediators of RTK activation have come to light. In particular, genetic and molecular studies have revealed essential roles for the *Drosophila* Capicua (CIC) HMG-box repressor downstream of Torso and the epidermal growth factor receptor (EGFR) – two RTKs that propagate their signals through the Ras-Raf-MAPK cascade (Jiménez et al., 2000; Goff et al., 2001; Roch et al., 2002; Atkey et al., 2006; Astigarraga et al., 2007; Tseng et al., 2007; Löhr et al., 2009; Ajuria et al., 2011). In addition, CIC is well conserved in mammals, and recent evidence suggests that human CIC mediates RTK-dependent responses that are linked to cell proliferation and cancer (Lee et al., 2002; Kawamura-Saito et al., 2006; Lam et al., 2006; Bettegowda et al., 2011; Dissanayake et al., 2011).

Here, we provide an overview of CIC regulatory functions in both *Drosophila* and mammals, including evidence that implicates this factor in human cancer and neurodegeneration. Given that CIC operates downstream of distinct RTK pathways, we also discuss how CIC activities illustrate different mechanisms that can generate specificity in RTK signaling processes.

## Structural and functional conservation of CIC proteins

*Drosophila* CIC was identified because of its role downstream of the Torso pathway, which controls the specification of terminal (head and tail) regions of the embryo; the name Capicua stems from its Catalan meaning ‘head-and-tail’ (Jiménez et al., 2000).

CIC is conserved from cnidarians to vertebrates, with single orthologs being present in *C. elegans* (see WormBase), mice and humans [(Lee et al., 2002; Lam et al., 2006) and (G.J., unpublished)] (Fig. 1). CIC proteins share two highly conserved domains – the HMG-box that is involved in DNA binding and nuclear localization, and a C-terminal motif C1 of unknown molecular function (Jiménez et al., 2000; Lee et al., 2002; Kawamura-Saito et al., 2006; Astigarraga et al., 2007) (Fig. 1). In addition, both *Drosophila* and mammals express at least two main CIC isoforms (CIC-S and CIC-L), which differ in size and in their N-terminal regions. At present, the best-characterized activities of CIC correspond to the CIC-S isoform. The CIC-L isoform contains an extended N-terminal segment with a highly conserved domain (N1) of unknown function, and its cellular roles remain less well understood (Lam et al., 2006) (Fig. 1).

The HMG-box recognizes octameric T(G/C)AATG(A/G)A sites in target promoters and enhancers and, in all cases studied, binding of CIC to these elements leads to transcriptional repression (Kawamura-Saito et al., 2006; Lam et al., 2006; Löhr et al., 2009; Kazemian et al., 2010; Ajuria et al., 2011; Lee et al., 2011). In *Drosophila*, repression by CIC appears to be tightly coupled to RTK-dependent control of transcription – CIC represses genes that are induced by RTK signaling, and such induction occurs, at least in part, through relief of CIC repression (Jiménez et al., 2000; Goff et al., 2001; Roch et al., 2002; Atkey et al., 2006; Astigarraga et al., 2007; Tseng et al., 2007; Ajuria et al., 2011). This mode of control, whereby signal-regulated targets are maintained in a repressed state in the absence of signaling, has been termed ‘default repression’ (Barolo and Posakony, 2002; Affolter et al., 2008). In addition, increasing evidence suggests that vertebrate CIC proteins function similarly, although direct in vivo support for this idea is still limited.

### Mechanisms of CIC-mediated repression

The mechanism by which DNA-bound CIC represses targeted gene expression is not well understood. Genetic analyses indicate that the C1 motif is essential for repression in different contexts (Goff et al., 2001; Kawamura-Saito et al., 2006; Astigarraga et al., 2007), but the underlying molecular mechanism remains unknown. Instead, we discuss below the roles of two co-repressors that have been implicated in CIC repression.

During *Drosophila* embryogenesis, the repression of CIC targets, such as *huckebein* (*hkb*) and *tailless* (*tll*), requires the presence of the Groucho (GRO) co-repressor (Paroush et al., 1997; Jiménez et al., 2000; Jennings and Ish-Horowicz, 2008; Cinnamon and Paroush, 2008). Furthermore, binding of CIC to the *hkb* enhancer correlates with the association of GRO to this enhancer (Ajuria et al., 2011). Although this supports a model whereby GRO serves as a co-repressor together with CIC (Jiménez et al., 2000), the precise molecular links between these two factors remain elusive (Jennings et al., 2006; Cinnamon et al., 2008; Ajuria et al., 2011; Helman et al., 2011). In particular, there is no evidence for direct physical interactions between both proteins in vivo (Ajuria et al., 2011).

By contrast, several studies indicate that repression by mammalian CIC involves the formation of protein complexes with ataxin 1 (ATXN1) and its related factor brother of ATXN1 (BOAT1; also known as ATXN1L) (Lam et al., 2006; Bowman et al., 2007; Lim et al., 2008; Crespo-Barreto et al., 2010; Lee et al., 2011) (Boxes 1, 2; Fig. 1). This, together with evidence that ATXN1 and BOAT1 associate with co-repressors, such as nuclear receptor co-repressor 2 (NCOR2, also known as SMRT) and histone deacetylases 3 and 4 (HDAC3 and HDAC4, respectively) (Tsai et al., 2004; Mizutani et al., 2005; Bolger et al., 2007; Tong et al., 2011), strongly suggests that both ATXN1 and BOAT1 function as CIC co-repressors. Interestingly, ATXN1 interacts with the CIC motif ATXN1-BD, which has been moderately conserved throughout evolution (Lam et al., 2006) (Fig. 1). Nevertheless, there is currently no evidence to suggest that ATXN1 mediates CIC repressor activity in flies.

### Mechanisms of CIC downregulation

*Drosophila* CIC becomes phosphorylated and downregulated during several developmental processes in response to RTK activation (Jiménez et al., 2000; Goff et al., 2001; Roch et al., 2002; Astigarraga et al., 2007; Tseng et al., 2007; Ajuria et al., 2011). For example, Torso RTK signaling in the early embryo leads to the degradation of CIC, whereas EGFR signaling in the ovarian follicle induces partial relocalization of CIC to the cytoplasm.

![Figure 1](#fig1)

**Fig. 1. Structural features of *Drosophila* and human CIC proteins.** Two main isoforms, short (CIC-S) and long (CIC-L), are present in each species (Lam et al., 2006). In *Drosophila*, CIC-S fulfils most known CIC functions, whereas CIC-L appears to have specific roles in oogenesis (Rittenhouse and Berg, 1995; Jiménez et al., 2000; Goff et al., 2001; Roch et al., 2002; Dorman et al., 2004; Astigarraga et al., 2007). No differential functions have been assigned to short versus long isoforms in mammals. Functional domains that have been identified in each species are indicated, and their sequence conservation is depicted with colored boxes. Note that the nuclear localization signal (NLS) that has been identified in human CIC (Dissanayake et al., 2011) is not conserved. All CIC domains (except the HMG-box) appear to be unique to CIC proteins [(Jiménez et al., 2000; Lam et al., 2006; Astigarraga et al., 2007) and (G.J., unpublished)]. Numbers indicate amino acid positions – the size of human CIC-L is estimated from mouse CIC-L (*).

Box 1. Role of CIC in spinocerebellar ataxia type 1 neurodegeneration

Experimental work by Zoghbi and colleagues has shown that mammalian CIC forms nuclear complexes with ATXN1, a polyglutamine-repeat protein that is implicated in spinocerebellar ataxia type 1 (SCA1) neurodegeneration (Lam et al., 2006; Zoghbi and Orr, 2009). During SCA1 pathogenesis, expansion of the polyglutamine region in ATXN1 renders it neurotoxic in cerebellar Purkinje cells and other neuronal populations. How mutant ATXN1 causes SCA1 is not well understood, but increasing evidence points to dysregulated interactions with different endogenous proteins as a key mechanism of the disease (Lim et al., 2008; de Chiara et al., 2009; Zoghbi and Orr, 2009). In particular, it has been reported that, compared with normal ATXN1, glutamine-expanded ATXN1 binds less efficiently to CIC, suggesting that partial loss of the CIC-ATXN1 co-repressive activity contributes to SCA1 neurodegeneration (Lim et al., 2008) (left and right panels). Indeed, CIC repressor activity appears to be stronger in the presence of normal ATXN1 than glutamine-expanded ATXN1, both in cultured cells and in mouse cerebella (Lam et al., 2006; Crespo-Barreto et al., 2010). Thus, mice that express glutamine-expanded ATXN1 show a substantial upregulation of CIC targets such as ETV5 (right panel, A), which is attributable to partial destabilization and/or weaker repressor activity of CIC in the disease state (Lim et al., 2008; Crespo-Barreto et al., 2010; Fryer et al., 2011).

Notably, glutamine-expanded ATXN1 can also cause a stronger binding of CIC to the promoters of certain genes (right panel, B), leading to the hyper-repression of those targets (Fryer et al., 2011). Consistent with this gain-of-function effect, the genetic reduction of CIC levels substantially rescues the phenotypes of mice that express glutamine-expanded ATAXN1. A similar rescue is observed after physical exercise, which induces enhanced EGFR signaling in the brainstem and concomitant downregulation of CIC in this tissue (Fryer et al., 2011).


Normal

![Image Description](#)

SCA1

![Image Description](#)

Key

ATXN1

Glutamine-expanded ATXN1


Fig. 2. Mechanisms of CIC downregulation by RTK-Ras-MAPK signaling. (A) Drosophila CIC is directly phosphorylated by MAPK (shown in red), which binds to the C2 docking motif. In the early embryo, MAPK activation occurs in response to the graded distribution of Trunk, and induces degradation of CIC (green) in a concentration-dependent manner. By contrast, EGFR signaling in the ovarian follicle, which depends on the TGF-α-like ligand Gurken, causes partial redistribution of nuclear CIC to the cytoplasm. In this case, the area in which CIC is downregulated has a relatively sharp border (Astigarraga et al., 2007). In other contexts, such as the wing imaginal disc, EGFR signaling appears to cause degradation of CIC (Roch et al., 2002; Tseng et al., 2007; Ajuria et al., 2011). (B) In human cells, EGFR-induced phosphorylation of CIC (shown in red) depends on both MAPK and p90RSK. Two potential mechanisms for downregulation have been described, which either attenuate CIC DNA binding activity or prevent its binding to the nuclear import adaptor importin-α4 (Dissanayake et al., 2011).


The C2 motif is highly conserved in *C. elegans* and moderately conserved in vertebrates (Astigarraga et al., 2007), although no reports are yet available on C2-mediated downregulation outside of *Drosophila*. Nevertheless, increasing evidence indicates that CIC can be downregulated by EGFR signaling in mammals—proteomic analysis identified CIC as an immediate-early gene product substrate that becomes phosphorylated within 10 minutes of EGFR stimulation in HeLa cells (Olsen et al., 2006), whereas studies in mice have found a correlation between increased concentrations of EGF and reduced levels of CIC in brainstem tissues (Fryer et al., 2011).

In addition, MacKintosh and co-workers have identified two potential molecular mechanisms of CIC downregulation in cultured human cells (Dissanayake et al., 2011) (Fig. 2B). The authors found that EGFR activation leads to the phosphorylation of CIC at multiple sites by both MAPK and ribosomal protein S6 kinase II (hereafter referred to as p90RSK), which itself becomes activated downstream of MAPK. MAPK-dependent phosphorylation prevents binding of CIC to importin-α4 (also known as KPNA3), an adaptor that is required for nuclear import, but because the loss of importin-α4 binding does not result in the cytoplasmic accumulation of CIC in stimulated cells, the biological implication of this mechanism remains unclear. However, according to the same study, p90RSK phosphorylation

of CIC occurs at Ser173 near the HMG-box, and promotes binding of CIC to 14-3-3 regulatory proteins (Dissanayake et al., 2011; Morrison, 2009) (Fig. 2B). Interestingly, this interaction causes reduced binding of CIC to the TGAATGAA sequence in vitro, and correlates with transcriptional upregulation of CIC targets, such as ETV5 (see below). Because the sequence flanking Ser173 in human CIC is well conserved in insects (but not in nematodes), these results raise the possibility that interactions between CIC and 14-3-3 also occur in some invertebrates (Dissanayake et al., 2011).

In summary, CIC levels or activity are directly downregulated by RAS–MAPK signals in different biological systems, possibly through a variety of mechanisms. Nevertheless, in all cases, CIC downregulation leads to a similar outcome, namely the relief of CIC repression that facilitates expression of downstream targets.

### Specific CIC functions downstream of Torso

CIC function is best understood downstream of Torso signaling in the *Drosophila* embryo (Fig. 3). The relative simplicity of this system, its amenability to genetic manipulation and the fact that Torso signaling induces gene expression mainly by relieving CIC-mediated repression, are features that have facilitated the characterization of RTK signaling in general, and of CIC in particular. Therefore, we discuss this model to illustrate the basic mechanism of CIC function as a MAPK sensor.

The Torso receptor is present throughout the plasma membrane of the early syncytial embryo, but it is only activated at the anterior and posterior embryonic poles in response to restricted processing of its ligand Trunk. This leads to localized stimulation of the Ras–MAPK pathway, creating a gradient of MAPK activity at each embryonic pole (Gabay et al., 1997; Greenwood and Struhl, 1997; Furriols and Casanova, 2003; Li, 2005; Coppey et al., 2008) (Fig. 3C,D). In the posterior pole, MAPK activity induces expression of *hkb* and *tll*. This occurs through the alleviation of CIC repression; CIC represses *hkb* and *tll* in central regions of the embryo, whereas MAPK activity that is present at the pole downregulates CIC levels and derepresses transcription of *hkb* and *tll* (Jiménez et al., 2000; Astigarraga et al., 2007; Ajuria et al., 2011). Because graded MAPK activity creates an opposing, complementary gradient of CIC protein levels that decreases towards the pole (Fig. 3E,F) and CIC represses *hkb* more effectively than *tll*, the result is an *hkb* expression pattern that is narrower compared with that of *tll* (Jiménez et al., 2000; Cinnamon et al., 2004; de las Heras and Casanova, 2006; Astigarraga et al., 2007; Kim et al., 2010) (Fig. 3A,F). These nested patterns of *hkb* and *tll* expression are essential to regulate the morphological differentiation of posterior embryonic structures, such as the hindgut and the posterior spiracles (Greenwood and Struhl, 1997; Furriols and Casanova, 2003) (Fig. 3B).

Repression of *hkb* and *tll* by CIC is essential, because the loss of *cic* function causes derepression of both targets towards the center of the embryo, which, in turn, results in suppression of thoracic and abdominal primordia (Jiménez et al., 2000). How CIC causes differential repression of *hkb* and *tll* is not well understood, although two mechanisms are likely to be involved. First, there are differences in the number and position of CIC binding sites in the *hkb* and *tll* enhancers, which, at least in some cases, function additively [(Ajuria et al., 2011) and (L. Ajuria and G.J., unpublished)]. Second, *hkb* and *tll* appear to be controlled by different activation inputs—for example, only *hkb*, but not *tll*, requires the transcription factor Lilliputian (LILLI) for its expression (Tang et al., 2001) (Fig. 4). Presumably, different

**Fig. 3. Interpretation of Torso RTK signaling in the syncytial *Drosophila* embryo.** (A) Torso signaling downregulates CIC, thereby relieving repression of *hkb* and *tll* expression (dark- and light-blue domains, respectively). (B) Normal morphological differentiation of the embryonic tail—the most distal structures such as the anal pad (*ap*) require both *hkb* and *tll* expression, whereas sub-terminal elements, such as the eighth abdominal belt (*a8*) and posterior spiracles (*ps*), only require *tll* activity. (C) Posterior embryonic pole showing the graded nuclear accumulation of active, phosphorylated MAPK (MAPK-P). (D) Quantification of MAPK-P along the length of the embryo (0% and 100% represent the anterior and posterior ends, respectively); au, arbitrary units. (E) Complementary distribution of CIC and TLL proteins; note their overlap in 2–3 nuclei (arrowheads). (F) Quantification of CIC levels as in D. The patterns of MAPK-P and CIC are asymmetric in anterior versus posterior positions as a result of competitive interactions with BCD and MAPK phosphatases (Kim et al., 2010; Kim et al., 2011b). Panels D and F have been modified from Current Biology, 20/5, Yoosik Kim, Matthieu Coppey, Rona Grossman, Leiore Ajuria, Gerardo Jimenez, Ze’ev Paroush, Stanislav Y. Shvartsman, MAPK Substrate Competition Integrates Patterning Signals in the *Drosophila* Embryo, 446-451 (2010) with permission from Elsevier (Kim et al., 2010).

combinations of those cis- and trans-acting inputs generate different thresholds at which *hkb* and *tll* are repressed by CIC. CIC functions in a similar manner at the anterior pole, where it also regulates the spatially restricted activation of *hkb* and *tll* (Jiménez et al., 2000; Astigarraga et al., 2007). In addition, CIC regulates anterior development by establishing repressor boundaries for head patterning genes that are activated by the Bicoid (BCD) transcription factor (Löhr et al., 2009). BCD forms a protein gradient that is thought to determine differential gene expression patterns in a concentration-dependent manner—however, such patterns also depend on CIC repressor activity regulated by the Torso pathway (Löhr et al., 2009).

cell determination (Roch et al., 2002). One target that is regulated by the EGFR-CIC pathway in this context is *argos*, which encodes a secreted feedback inhibitor of EGFR signaling that is required for correct vein patterning (Freeman et al., 1992; Golembo et al., 1996; Roch et al., 2002; Shilo, 2005; Ajuria et al., 2011) (see below).

Other interactions between EGFR signaling and CIC in *Drosophila* regulate the specification of dorsoventral (DV) follicle cell fates during oogenesis (Goff et al., 2001). EGFR activation in dorsal follicle cells induces expression of the mirror (*mirr*) gene, which encodes a transcription factor that is required for the formation of dorsal appendages in the future eggshell, whereas CIC represses *mirr* in ventral cells (Goff et al., 2001; Atkey et al., 2006). Importantly, although CIC is downregulated by EGFR signaling in dorsal follicle cells (Fig. 2A), this downregulation does not appear to be essential for EGFR-dependent activation of *mirr* (see below) (Atkey et al., 2006; Astigarraga et al., 2007).

In addition, CIC is required for the expression of *pipe* in ventral follicle cells (Goff et al., 2001). The *pipe* gene, which is repressed following the activation of the EGFR signaling pathway in dorsal follicle cells, encodes a sulfotransferase that initiates the subdivision of the embryonic DV axis (Moussian and Roth, 2005). Recent studies have shown that Mirr represses *pipe* in dorsal follicle cells, and CIC-mediated repression of *mirr* supports *pipe* transcription in ventral cells (Technau et al., 2011; Fuchs et al., 2012; Andreu et al., 2012). Thus, CIC functions in a repressor circuit that is essential to translate EGFR signaling into asymmetric *pipe* expression during DV axis formation (Andreu et al., 2012).

Despite the fact that CIC performs similar roles downstream of Torso and the EGFR, the transcriptional switches that operate downstream of both receptors might not be identical. Whereas Torso-mediated induction of *hkb* and *tll* expression depends solely on CIC (and GRO) derepression, at least some EGFR signals appear to rely on both CIC-dependent and -independent mechanisms to regulate gene expression. Thus, although the activation of *argos* and *mirr* in response to EGFR signaling might be facilitated by the downregulation of CIC, there is evidence that EGFR signaling provides additional inputs that activate both genes (Roch et al., 2002; Atkey et al., 2006; Astigarraga et al., 2007; Ajuria et al., 2011). Furthermore, these EGFR inputs appear to function cooperatively with EGFR-independent transcriptional activators that ensure the robust expression of target genes (Barolo and Posakony, 2002; Atkey et al., 2006; Ajuria et al., 2011). In such cases, the main role of CIC might be to repress *argos* and *mirr* outside of their normal domains of expression, where EGFR signaling is either low or absent but the presence of cooperative factors could potentially drive weak ectopic *argos* and *mirr* expression (Roch et al., 2002; Atkey et al., 2006; Ajuria et al., 2011) (see also Fig. 4).

### Specific CIC functions downstream of EGFR signaling

EGFR signaling controls a wide range of patterning and cell fate specification events in *Drosophila* (Shilo, 2003), several of which depend on CIC. For example, localized EGFR activation in the embryonic neuroectoderm regulates neuroblast differentiation by inducing expression of the intermediate neuroblasts defective (*ind*) gene, and this occurs, at least in part, through relief of CIC repression (Weiss et al., 1998; von Ohlen et al., 2000; Ajuria et al., 2011) (Fig. 4). Similarly, the specification of prospective vein cells in the developing wing depends on EGFR signaling and downregulation of CIC activity (Roch et al., 2002; Blair, 2007; Ajuria et al., 2011). In this system, the EGFR-CIC derepression circuit has a central role, because loss of CIC function bypasses the requirement for EGFR signaling in vein

### Role of CIC in cell proliferation and tumorigenesis

In addition to its important roles in tissue patterning, CIC also acts downstream of the EGFR to regulate cell proliferation. In *Drosophila*, the EGFR-Ras-MAPK pathway promotes the proliferation of larval imaginal disc cells that give rise to adult structures, such as the wing and the eye. Accordingly, the loss of this pathway markedly reduces the growth rates of these structures (Diaz-Benjumea and Hafen, 1994; Karim and Rubin, 1998; Prober and Edgar, 2000). Similar to other EGFR-mediated

Box 2. Relationship between CIC and PEA3 pathways in mammalian development and tumorigenesis

The finding that CIC-DUX4 chimeras upregulate genes of the ETS PEA3 subfamily in Ewing-like sarcomas indicates that CIC can repress their transcription under normal conditions (see main text) (Kawamura-Saito et al., 2006; Dissanayake et al., 2011). More recently, molecular and genetic analyses have identified a crucial role for CIC-ATXN1 and CIC-BOAT1 complexes in mediating direct transcriptional repression of PEA3 genes during lung development (Lee et al., 2011). Mutant mice that lack CIC or ATXN1 and BOAT1 activities display several phenotypes, including abnormal alveolarization in developing lungs and derepression of PEA3 subfamily genes, particularly of ETV4. A molecular pathway has been defined, whereby increased ETV4 activity in these mutant mice upregulates expression of the matrix metalloproteinase 9 (Mmp9) gene, whose protein product is involved in extracellular matrix (ECM) remodeling and lung alveolarization—which explains, at least in part, the mutant phenotype (Lee et al., 2011). Interestingly, the oncogenic activities of PEA3 transcription factors in several types of tumor, such as Ewing sarcoma, melanoma or prostate cancer, lead to the upregulation of MMP family genes and other targets involved in ECM remodeling—which contributes to enhanced cell migration and metastatic behavior (de Launoit et al., 2006; Yan and Boyd, 2007; Kessenbrock et al., 2010; Hollenhorst et al., 2011). Thus, related CIC-PEA3 transcriptional circuits appear to affect ECM remodeling in both normal development and carcinogenesis. Finally, because RTK signaling can induce expression of PEA3 family genes in different vertebrate systems (Roehl and Nüsslein-Volhard, 2001; Raible and Brand, 2001; Lunn et al., 2007; Pratilas et al., 2009), it is conceivable that such RTK-dependent regulation occurs, at least in part, through a mechanism of CIC derepression.

CIC downregulation is likely to cause the derepression of downstream proliferation genes, and recent data indicate that bantam microRNA is one such CIC target in this context (Herranz et al., 2012).

Furthermore, another study in *Drosophila* suggests that a similar mechanism regulates proliferation of intestinal stem cells (ISCs) in response to EGFR signaling (Jiang et al., 2011). In this case, the inactivation of CIC causes proliferation of ectopic ISCs, which resembles the effects on ISC proliferation induced by the overexpression of EGFR ligands or constitutively active variants of the Ras-MAPK cascade. These data, again, support a role for a linear Ras-MAPK-CIC pathway in the induction of proliferation genes.

Importantly, the role of CIC in restricting cell growth appears to be conserved in humans, as mutations in CIC have been associated with different malignancies. For example, two cases of Ewing-like sarcoma have been found to result from similar chromosomal translocations that produce chimeric CIC proteins fused to the C-terminal region of double homeobox protein 4 (DUX4) (Kawamura-Saito et al., 2006). The CIC-DUX4 fusion mediates transcriptional activation instead of repression, which leads to the upregulation of genes that are normally repressed by CIC. Among these are genes that encode members of the PEA3 subfamily of ETS transcription factors, such as ETV1, ETV4 (also known as PEA3) and ETV5 (Kawamura-Saito et al., 2006). This is consistent with the fact that typical Ewing tumors express aberrant PEA3 protein chimeras with enhanced transcriptional activity (Arvand and Denny, 2001). As mentioned above, EGFR activation in cultured cells can induce expression of PEA3 family genes by relieving CIC repression, which supports the notion that CIC functions in a regulatory pathway that links RTK signals to the expression of PEA3 transcription factors (Dissanayake et al., 2011) (Box 2).

In addition, mutations in CIC have been reported in other types of tumor, such as breast cancer (Sjöblom et al., 2006). In a recent study, CIC has been identified as a main tumor suppressor in oligodendroglioma (OD) brain cancer (Bettegowda et al., 2011). Remarkably, inactivating mutations in CIC were detected in approximately half of OD tumors, many of which mapped to the DNA-binding HMG-box, presumably impeding the binding of DNA. By contrast, the sequencing of CIC-coding sequences in 298 non-OD tumor samples revealed only three missense mutations (Bettegowda et al., 2011). Although other CIC mutations might have escaped these analyses, this average frequency seems lower than the prevalence of oncogenic mutations in RTK signaling components such as EGFR, Ras or Raf, which is consistent with the notion that oncogenic RTK signals affect multiple regulators of growth (Sears et al., 1999; Murphy et al., 2002; Yoon and Seger, 2006; Dhillon et al., 2007; Meloche and Pouysségur, 2007).

responses, this involves the downregulation of CIC in the nucleus (Tseng et al., 2007). Importantly, mutations that disrupt CIC function increase the rate of cell proliferation without affecting cell size and bypass the requirement for EGFR signaling in promoting growth (Tseng et al., 2007). By contrast, cells that lack a different negative regulator of growth, TSC1 (also known as Hamartin), still depend on EGFR signaling to proliferate (Tseng et al., 2007; Hariharan and Bilder, 2006). This strongly suggests that one mechanism by which EGFR-Ras-MAPK signals induce cell proliferation in imaginal tissues is by downregulating CIC.

Insights into RTK signaling specificity

The repressor functions of CIC downstream of different RTK pathways not only provide an insight into how external signals can influence gene expression, but they are also relevant to our understanding of RTK-signaling specificity. Because RTK signals are often propagated through similar intracellular cascades, understanding the mechanisms by which these related signals elicit specific responses in each context is a long-standing question in the field (Pawson and Saxton, 1999; Tan and Kim,

1999; Simon, 2000). Here, we focus on two main sources of specificity during the regulation of transcriptional responses, emphasizing the insights that have been gained from studies of CIC function.

### Intrinsic differences between signaling networks

Although RTKs generally stimulate a similar complement of intracellular pathways, distinct features underlying the action of those pathways can lead to differential cellular responses (Kratchmarova et al., 2005; Leatherbarrow and Halfon, 2009; Lemmon and Schlessinger, 2010; Zand et al., 2011). In particular, quantitative changes in the strength or duration of RTK signaling can give rise to diverse cellular outputs (Marshall, 1995; Greenwood and Struhl, 1997; Lemmon and Schlessinger, 2010). In this context, the response of CIC to graded Torso and MAPK activities provides an excellent paradigm of transcriptional regulation by different signal intensities of RTK (Greenwood and Struhl, 1997). As described above and in Fig. 3, CIC is highly sensitive to graded MAPK activity, acting as a rheostat-like switch that translates the MAPK input into a complementary gradient of repression. Because this gradient controls the patterns of *tll* and *hkb* expression, which are, in turn, essential for the correct differentiation of embryonic structures (see above), the Torso–MAPK–CIC system illustrates the key steps that can lead from graded RTK activity to differential biological output.

Furthermore, recent studies in the Torso system have uncovered a new mechanism of specificity that is based on competition between CIC and other MAPK substrates, such as BCD (Kim et al., 2010; Kim et al., 2011a). Specifically, the presence of BCD in anterior regions of the embryo counteracts the activity of MAPK towards CIC, thereby potentiating CIC repression and contributing to asymmetric transcription of genes targeted by CIC in anterior versus posterior positions (Kim et al., 2011a). Thus, the ability of CIC to function as a MAPK sensor enables the highly precise regulation of its repressor activity, which is crucial for inducing accurate downstream responses.

### Signal integration by transcriptional enhancers

Although intrinsic differences in RTK signaling can influence the specificity of responses, this mechanism does not generally explain how RTK signals induce differential transcriptional effects in development. Instead, RTK pathways often provide a common signal, which is then interpreted by *cis*-regulatory enhancers within a target gene that integrate additional non-RTK inputs. This has been demonstrated in different *Drosophila* developmental contexts, where the EGFR-mediated activation of gene expression requires both a generic EGFR input acting on Yan and Pointed-P2, as well as other independent inputs that function in a restricted or cell-specific manner (Flores et al., 2000; Halfon et al., 2000; Xu et al., 2000; Simon, 2000).

Furthermore, this type of control also operates downstream of RTK pathways that regulate CIC. This seems logical, given that RTK–CIC pathways function mainly as derepression modules, so that transcriptional responses depend on activation inputs that can be independent of RTK signaling. For example, the anterior expression of head patterning genes, such as *cap-n-collar* (*cnc*), requires both a Torso–CIC derepression input and the BCD activator that is localized in anterior regions (Löhr et al., 2009). Because the same Torso–CIC pathway functions in anterior and posterior regions, the specific response of *cnc* in anterior regions depends on its transcriptional activation by BCD. This concept is also illustrated in Fig. 4, which compares the selective expression of *hkb* and *ind* in response to localized Torso and EGFR activation, respectively, in the early embryo. Again, the specificity of each response appears to depend on different RTK-independent activators of the *hkb* and *ind* enhancers, although positive EGFR inputs that are independent of CIC might also contribute to the specific response of *ind* (see above and Fig. 4 legend).

### Concluding remarks

Increasing evidence shows that CIC functions as a general regulator of RTK–Ras–MAPK signaling responses. The studies on CIC have not only greatly advanced our understanding of several *Drosophila* processes under the control of RTK signals, but have also revealed important CIC functions in mammals. However, several interesting questions remain to be explored. Do CIC proteins function exclusively as transcriptional repressors? Are all of their functions associated with RTK signaling, or do CIC proteins act in other regulatory pathways? Are there any fundamental differences between the activities of the CIC-S and the CIC-L isoforms? What are the mechanisms that are responsible for CIC downregulation following its phosphorylation by MAPK? How are the roles of CIC in cell proliferation coordinated with other mechanisms of cell growth control? Answers to these questions will provide fundamental insights into the mechanisms by which the interaction between RTK signaling pathways and CIC control cellular decisions, both during normal development and in disease states.

### Acknowledgements

We thank S. González-Crespo for critical comments on the manuscript, and members of our labs for many helpful discussions.

### Funding

This work is supported by ICREA, Ministerio de Ciencia e Innovación [grant number BFU2008-01875 to G.J.] and Generalitat de Catalunya [grant number 2009SGR-1075 to G.J.]; the National Institutes of Health and National Science Foundation to S.Y.S.; the National Institutes of Health (National Institute of General Medical Sciences) [grant number R01GM086537 to S.Y.S. and Z.P.]; the Israel Science Foundation (Centers of Excellence) [grant number 180/09 to Z.P.]; and the Król Charitable Foundation to Z.P. Deposited in PMC for release after 12 months.

### References

Affolter, M., Pyrowolakis, G., Weiss, A. and Basler, K. (2008). Signal-induced repression: the exception or the rule in developmental signaling? *Dev. Cell* **15**, 11-22.

Ajuria, L., Nieve, C., Winkler, C., Kuo, D., Samper, N., Andreu, M. J., Helman, A., González-Crespo, S., Paroush, Z., Courey, A. J. et al. (2011). Capicua DNA-binding sites are general response elements for RTK signaling in *Drosophila*. *Development* **138**, 915-924.

Andreu, M. J., González-Pérez, E., Ajuria, L., Samper, N., González-Crespo, S., Campuzano, S. and Jiménez, G. (2012). Mirror represses *pipe* expression in follicle cells to initiate dorsoventral axis formation in *Drosophila*. *Development*. **139**, 1110-1114.

Arvand, A. and Denny, C. T. (2001). Biology of EWS/ETS fusions in Ewing's family tumors. *Oncogene* **20**, 5747-5754.

Astigarraga, S., Grossman, R., Diaz-Delfin, J., Caelles, C., Paroush, Z. and Jiménez, G. (2007). A MAPK docking site is critical for downregulation of Capicua by Torso and EGFR RTK signaling. *EMBO J.* **26**, 668-677.

Atkey, M. R., Lachance, J. F., Walczak, M., Rebello, T. and Nilson, L. A. (2006). Capicua regulates follicle cell fate in the *Drosophila* ovary through repression of *mirror*. *Development* **133**, 2115-2123.

Barolo, S. and Posakony, J. W. (2002). Three habits of highly effective signaling pathways: principles of transcriptional control by developmental cell signaling. *Genes Dev.* **16**, 1167-1181.

Bettegowda, C., Agrawal, N., Jiao, Y., Sausen, M., Wood, L. D., Hruban, R. H., Rodriguez, F. J., Cahill, D. P., McLendon, R., Riggins, G. et al. (2011). Mutations in *CIC* and *FUBP1* contribute to human oligodendroglioma. *Science* 333, 1453-1455.

Blair, S. S. (2007). Wing vein patterning in *Drosophila* and the analysis of intercellular signaling. *Annu. Rev. Cell Dev. Biol.* 23, 293-319.

Bolger, T. A., Zhao, X., Cohen, T. J., Tsai, C. C. and Yao, T. P. (2007). The neurodegenerative disease protein ataxin-1 antagonizes the neuronal survival function of myocyte enhancer factor-2. *J. Biol. Chem.* 282, 29186-29192.

Bowman, A. B., Lam, Y. C., Jafar-Nejad, P., Chen, H. K., Richman, R., Samaco, R. C., Fryer, J. D., Kahle, J. J., Orr, H. T. and Zoghbi, H. Y. (2007). Duplication of *Atxn1l* suppresses SCA1 neuropathology by decreasing incorporation of polyglutamine-expanded ataxin-1 into native complexes. *Nat. Genet.* 39, 373-379.

Brunner, D., Ducker, K., Oellers, N., Hafen, E., Scholz, H. and Klämbt, C. (1994). The ETS domain protein pointed-P2 is a target of MAP kinase in the sevenless signal transduction pathway. *Nature* 370, 386-389.

Cinnamon, E. and Paroush, Z. (2008). Context-dependent regulation of Groucho/TLE-mediated repression. *Curr. Opin. Genet. Dev.* 18, 435-440.

Cinnamon, E., Gur-Wahnon, D., Helman, A., St Johnston, D., Jiménez, G. and Paroush, Z. (2004). Capicua integrates input from two maternal systems in *Drosophila* terminal patterning. *EMBO J.* 23, 4571-4582.

Cinnamon, E., Helman, A., Ben-Haroush Schyr, R., Orian, A., Jiménez, G. and Paroush, Z. (2008). Multiple RTK pathways downregulate Groucho-mediated repression in *Drosophila* embryogenesis. *Development* 135, 829-837.

Coppey, M., Boettiger, A. N., Berezhkovskii, A. M. and Shvartsman, S. Y. (2008). Nuclear trapping shapes the terminal gradient in the *Drosophila* embryo. *Curr. Biol.* 18, 915-919.

Crespo-Barreto, J., Fryer, J. D., Shaw, C. A., Orr, H. T. and Zoghbi, H. Y. (2010). Partial loss of Ataxin-1 function contributes to transcriptional dysregulation in spinocerebellar ataxia type 1 pathogenesis. *PLoS Genet.* 6, e1001021.

de Chiara, C., Menon, R. P., Strom, M., Gibson, T. J. and Pastore, A. (2009). Phosphorylation of S776 and 14-3-3 binding modulate ataxin-1 interaction with splicing factors. *PLoS ONE* 4, e8372.

de las Heras, J. M. and Casanova, J. (2006). Spatially distinct downregulation of Capicua repression and tailless activation by the Torso RTK pathway in the *Drosophila* embryo. *Mech. Dev.* 123, 481-486.

de Launoit, Y., Baert, J. L., Chotteau-Lelievre, A., Monte, D., Coutte, L., Mauen, S., Firlej, V., Degerny, C. and Verreman, K. (2006). The Ets transcription factors of the PEA3 group: transcriptional regulators in metastasis. *Biochim. Biophys. Acta* 1766, 79-87.

Dhillon, A. S., Hagan, S., Rath, O. and Kolch, W. (2007). MAP kinase signalling pathways in cancer. *Oncogene* 26, 3279-3290.

Diaz-Benjumea, F. J. and Hafen, E. (1994). The sevenless signalling cassette mediates *Drosophila* EGF receptor function during epidermal development. *Development* 120, 569-578.

Dissanayake, K., Toth, R., Blakey, J., Olsson, O., Campbell, D. G., Prescott, A. R. and MacKintosh, C. (2011). ERK/p90(RSK)/14-3-3 signalling has an impact on expression of PEA3 Ets transcription factors via the transcriptional repressor capicua. *Biochem. J.* 433, 515-525.

Dorman, J. B., James, K. E., Fraser, S. E., Kiehart, D. P. and Berg, C. A. (2004). *Bullwinkle* is required for epithelial morphogenesis during *Drosophila* oogenesis. *Dev. Biol.* 267, 320-341.

Flores, G. V., Duan, H., Yan, H., Nagaraj, R., Fu, W., Zou, Y., Noll, M. and Banerjee, U. (2000). Combinatorial signaling in the specification of unique cell fates. *Cell* 103, 75-85.

Freeman, M., Klämbt, C., Goodman, C. S. and Rubin, G. M. (1992). The *argos* gene encodes a diffusible factor that regulates cell fate decisions in the *Drosophila* eye. *Cell* 69, 963-975.

Fryer, J. D., Yu, P., Kang, H., Mandel-Brehm, C., Carter, A. N., Crespo-Barreto, J., Gao, Y., Flora, A., Shaw, C., Orr, H. T. et al. (2011). Exercise and genetic rescue of SCA1 via the transcriptional repressor Capicua. *Science* 334, 690-693.

Fuchs, A., Cheung, L. S., Charbonnier, E., Shvartsman, S. Y. and Pyrowolakis, G. (2012). Transcriptional interpretation of the EGF receptor signaling gradient. *Proc. Natl. Acad. Sci. USA* 109, 1572-1577.

Furriols, M. and Casanova, J. (2003). In and out of Torso RTK signalling. *EMBO J.* 22, 1947-1952.

Gabay, L., Scholz, H., Golembo, M., Klaes, A., Shilo, B. Z. and Klämbt, C. (1996). EGF receptor signaling induces *pointed P1* transcription and inactivates Yan protein in the *Drosophila* embryonic ventral ectoderm. *Development* 122, 3355-3362.

Gabay, L., Seger, R. and Shilo, B. Z. (1997). MAP kinase in situ activation atlas during *Drosophila* embryogenesis. *Development* 124, 3535-3541.

Goff, D. J., Nilson, L. A. and Morisato, D. (2001). Establishment of dorsal-ventral polarity of the *Drosophila* egg requires *capicua* action in ovarian follicle cells. *Development* 128, 4553-4562.

Golembo, M., Schweitzer, R., Freeman, M. and Shilo, B. Z. (1996). *Argos* transcription is induced by the *Drosophila* EGF receptor pathway to form an inhibitory feedback loop. *Development* 122, 223-230.

Greenwood, S. and Struhl, G. (1997). Different levels of Ras activity can specify distinct transcriptional and morphological consequences in early *Drosophila* embryos. *Development* 124, 4879-4886.

Halfon, M. S., Carmena, A., Gisselbrecht, S., Sackerson, C. M., Jimenez, F., Baylies, M. K. and Michelson, A. M. (2000). Ras pathway specificity is determined by the integration of multiple signal-activated and tissue-restricted transcription factors. *Cell* 103, 63-74.

Hariharan, I. K. and Bilder, D. (2006). Regulation of imaginal disc growth by tumor-suppressor genes in *Drosophila*. *Annu. Rev. Genet.* 40, 335-361.

Hart, A. H., Reventar, R. and Bernstein, A. (2000). Genetic analysis of *ETS* genes in *C. elegans*. *Oncogene* 19, 6400-6408.

Helman, A., Cinnamon, E., Mezuman, S., Hayouka, Z., Von Ohlen, T., Orian, A., Jiménez, G. and Paroush, Z. (2011). Phosphorylation of Groucho mediates RTK feedback inhibition and prolonged pathway target gene expression. *Curr. Biol.* 21, 1102-1110.

Herranz, H., Hong, X. and Cohen, S. M. (2012). Mutual repression by bantam miRNA and Capicua links the EGFR/MAPK and Hippo pathways in growth control. *Curr. Biol.* [Epub ahead of print] doi:10.1016/j.cub.2012.02.050.

Hollenhorst, P. C., Ferris, M. W., Hull, M. A., Chae, H., Kim, S. and Graves, B. J. (2011). Oncogenic ETS proteins mimic activated RAS/MAPK signaling in prostate cells. *Genes Dev.* 25, 2147-2157.

Jacobs, D., Beitel, G. J., Clark, S. G., Horvitz, H. R. and Kornfeld, K. (1998). Gain-of-function mutations in the *Caenorhabditis elegans lin-1* ETS gene identify a C-terminal regulatory domain phosphorylated by ERK MAP kinase. *Genetics* 149, 1809-1822.

Jennings, B. H. and Ish-Horowicz, D. (2008). The Groucho/TLE/Grg family of transcriptional co-repressors. *Genome Biol.* 9, 205.

Jennings, B. H., Pickles, L. M., Wainwright, S. M., Roe, S. M., Pearl, L. H. and Ish-Horowicz, D. (2006). Molecular recognition of transcriptional repressor motifs by the WD domain of the Groucho/TLE corepressor. *Mol. Cell* 22, 645-655.

Jiang, H., Grenley, M. O., Bravo, M. J., Blumhagen, R. Z. and Edgar, B. A. (2011). EGFR/Ras/MAPK signaling mediates adult midgut epithelial homeostasis and regeneration in *Drosophila*. *Cell Stem Cell* 8, 84-95.

Jiménez, G., Guichet, A., Ephrussi, A. and Casanova, J. (2000). Relief of gene repression by torso RTK signaling: role of *capicua* in *Drosophila* terminal and dorsoventral patterning. *Genes Dev.* 14, 224-231.

Karim, F. D., Chang, H. C., Therrien, M., Wassarman, D. A., Laverty, T. and Rubin, G. M. (1996). A screen for genes that function downstream of Ras1 during *Drosophila* eye development. *Genetics* 143, 315-329.

Karim, F. D. and Rubin, G. M. (1998). Ectopic expression of activated Ras1 induces hyperplastic growth and increased cell death in *Drosophila* imaginal tissues. *Development* 125, 1-9.

Kawamura-Saito, M., Yamazaki, Y., Kaneko, K., Kawaguchi, N., Kanda, H., Mukai, H., Gotoh, T., Motoi, T., Fukayama, M., Aburatani, H. et al. (2006). Fusion between *CIC* and *DUX4* up-regulates *PEA3* family genes in Ewing-like sarcomas with t(4;19)(q35;q13) translocation. *Hum. Mol. Genet.* 15, 2125-2137.

Kazemian, M., Blatti, C., Richards, A., McCutchan, M., Wakabayashi-Ito, N., Hammonds, A. S., Celniker, S. E., Kumar, S., Wolfe, S. A., Brodsky, M. H. et al. (2010). Quantitative analysis of the *Drosophila* segmentation regulatory network using pattern generating potentials. *PLoS Biol.* 8, e1000456.

Kessenbrock, K., Plaks, V. and Werb, Z. (2010). Matrix metalloproteinases: regulators of the tumor microenvironment. *Cell* 141, 52-67.

Kim, Y., Coppey, M., Grossman, R., Ajuria, L., Jiménez, G., Paroush, Z. and Shvartsman, S. Y. (2010). MAPK substrate competition integrates patterning signals in the *Drosophila* embryo. *Curr. Biol.* 20, 446-451.

Kim, Y., Andreu, M. J., Lim, B., Chung, K., Terayama, M., Jiménez, G., Berg, C. A., Lu, H. and Shvartsman, S. Y. (2011a). Gene regulation by MAPK substrate competition. *Dev. Cell* 20, 880-887.

Kim, Y., Paroush, Z., Nairz, K., Hafen, E., Jiménez, G. and Shvartsman, S. Y. (2011b). Substrate-dependent control of MAPK phosphorylation in vivo. *Mol. Syst. Biol.* 7, 467.

Kratchmarova, I., Blagoev, B., Haack-Sørensen, M., Kassem, M. and Mann, M. (2005). Mechanism of divergent growth factor effects in mesenchymal stem cell differentiation. *Science* 308, 1472-1477.

Lam, Y. C., Bowman, A. B., Jafar-Nejad, P., Lim, J., Richman, R., Fryer, J. D., Hyun, E. D., Duvick, L. A., Orr, H. T., Botas, J. et al. (2006). ATAXIN-1 interacts with the repressor Capicua in its native complex to cause SCA1 neuropathology. *Cell* 127, 1335-1347.

Leatherbarrow, J. R. and Halfon, M. S. (2009). Identification of receptor-tyrosine-kinase-signaling target genes reveals receptor-specific activities and pathway branchpoints during *Drosophila* development. *Genetics* 181, 1335-1345.

Lee, C. J., Chan, W. I., Cheung, M., Cheng, Y. C., Appleby, V. J., Orme, A. T. and Scotting, P. J. (2002). CIC, a member of a novel subfamily of the HMG-box superfamily, is transiently expressed in developing granule neurons. *Mol. Brain Res.* 106, 151-156.

Lee, Y., Fryer, J. D., Kang, H., Crespo-Barreto, J., Bowman, A. B., Gao, Y., Kahle, J. J., Hong, J. S., Kheradmand, F., Orr, H. T. et al. (2011). ATXN1 protein family and CIC regulate extracellular matrix remodeling and lung alveolarization. *Dev. Cell* 21, 746-757.

Lemmon, M. A. and Schlessinger, J. (2010). Cell signaling by receptor tyrosine kinases. *Cell* 141, 1117-1134.

Li, W. X. (2005). Functions and mechanisms of receptor tyrosine kinase Torso signaling: lessons from *Drosophila* embryonic terminal development. *Dev. Dyn.* 232, 656-672.

Lim, J., Crespo-Barreto, J., Jafar-Nejad, P., Bowman, A. B., Richman, R., Hill, D. E., Orr, H. T. and Zoghbi, H. Y. (2008). Opposing effects of polyglutamine expansion on native protein complexes contribute to SCA1. *Nature* 452, 713-718.

Löhr, U., Chung, H. R., Beller, M. and Jäckle, H. (2009). Antagonistic action of Bicoid and the repressor Capicua determines the spatial limits of *Drosophila* head gene expression domains. *Proc. Natl. Acad. Sci. USA* 106, 21695-21700.

Lunn, J. S., Fishwick, K. J., Halley, P. A. and Storey, K. G. (2007). A spatial and temporal map of FGF/Erk1/2 activity and response repertoires in the early chick embryo. *Dev. Biol.* 302, 536-552.

Marshall, C. J. (1995). Specificity of receptor tyrosine kinase signaling: transient versus sustained extracellular signal-regulated kinase activation. *Cell* 80, 179-185.

Meloche, S. and Pouyssegur, J. (2007). The ERK1/2 mitogen-activated protein kinase pathway as a master regulator of the G1- to S-phase transition. *Oncogene* 26, 3227-3239.

Mizutani, A., Wang, L., Rajan, H., Vig, P. J., Alaynick, W. A., Thaler, J. P. and Tsai, C. C. (2005). Boat, an AXH domain protein, suppresses the cytotoxicity of mutant ataxin-1. *EMBO J.* 24, 3339-3351.

Morrison, D. K. (2009). The 14-3-3 proteins: integrators of diverse signaling cues that impact cell fate and cancer development. *Trends Cell Biol.* 19, 16-23.

Moussian, B. and Roth, S. (2005). Dorsoventral axis formation in the *Drosophila* embryo-shaping and transducing a morphogen gradient. *Curr. Biol.* 15, R887-R899.

Murphy, L. O., Smith, S., Chen, R. H., Fingar, D. C. and Blenis, J. (2002). Molecular interpretation of ERK signal duration by immediate early gene products. *Nat. Cell Biol.* 4, 556-564.

Olsen, J. V., Blagoev, B., Gnad, F., Macek, B., Kumar, C., Mortensen, P. and Mann, M. (2006). Global, in vivo, and site-specific phosphorylation dynamics in signaling networks. *Cell* 127, 635-648.

O'Neill, E. M., Rebay, I., Tjian, R. and Rubin, G. M. (1994). The activities of two Ets-related transcription factors required for *Drosophila* eye development are modulated by the Ras/MAPK pathway. *Cell* 78, 137-147.

Paroush, Z., Wainwright, S. M. and Ish-Horowicz, D. (1997). Torso signalling regulates terminal patterning in *Drosophila* by antagonising Groucho-mediated repression. *Development* 124, 3827-3834.

Pawson, T. and Saxton, T. M. (1999). Signaling networks - do all roads lead to the same genes? *Cell* 97, 675-678.

Pratilas, C. A., Taylor, B. S., Ye, Q., Viale, A., Sander, C., Solit, D. B. and Rosen, N. (2009). (V600E)BRAF is associated with disabled feedback inhibition of RAF-MEK signaling and elevated transcriptional output of the pathway. *Proc. Natl. Acad. Sci. USA* 106, 4519-4524.

Prober, D. A. and Edgar, B. A. (2000). Ras1 promotes cellular growth in the *Drosophila* wing. *Cell* 100, 435-446.

Raible, F. and Brand, M. (2001). Tight transcriptional control of the ETS domain factors Erm and Pea3 by Fgf signaling during early zebrafish development. *Mech. Dev.* 107, 105-117.

Rebay, I. and Rubin, G. M. (1995). Yan functions as a general inhibitor of differentiation and is negatively regulated by activation of the Ras1/MAPK pathway. *Cell* 81, 857-866.

Reuter, R. and Leptin, M. (1994). Interacting functions of *snail*, *twist* and *huckebein* during the early development of germ layers in *Drosophila*. *Development* 120, 1137-1150.

Rittenhouse, K. R. and Berg, C. A. (1995). Mutations in the *Drosophila* gene bullwinkle cause the formation of abnormal eggshell structures and bicaudal embryos. *Development* 121, 3023-3033.

Roch, F., Jiménez, G. and Casanova, J. (2002). EGFR signalling inhibits Capicua-dependent repression during specification of *Drosophila* wing veins. *Development* 129, 993-1002.

Roehl, H. and Nüsslein-Volhard, C. (2001). Zebrafish *pea3* and *erm* are general targets of FGF8 signaling. *Curr. Biol.* 11, 503-507.

Sears, R., Leone, G., DeGregori, J. and Nevins, J. R. (1999). Ras enhances Myc protein stability. *Mol. Cell* 3, 169-179.

Shilo, B. Z. (2003). Signaling by the *Drosophila* epidermal growth factor receptor pathway during development. *Exp. Cell Res.* 284, 140-149.

Shilo, B. Z. (2005). Regulating the dynamics of EGF receptor signaling in space and time. *Development* 132, 4017-4027.

Simon, M. A. (2000). Receptor Tyrosine Kinases: specific outcomes from general signals. *Cell* 103, 13-15.

Sjöblom, T., Jones, S., Wood, L. D., Parsons, D. W., Lin, J., Barber, T. D., Mandelker, D., Leary, R. J., Ptak, J., Silliman, N. et al. (2006). The consensus coding sequences of human breast and colorectal cancers. *Science* 314, 268-274.

Stathopoulos, A. and Levine, M. (2005). Localized repressors delineate the neurogenic ectoderm in the early *Drosophila* embryo. *Dev. Biol.* 280, 482-493.

Tan, P. B. O. and Kim, S. K. (1999). Signaling specificity: the RTK/Ras/MAP kinase pathway in metazoans. *Trends Genet.* 15, 145-149.

Tang, A. H., Neufeld, T. P., Rubin, G. M. and Müller, H. A. (2001). Transcriptional regulation of cytoskeletal functions and segmentation by a novel maternal pair-rule gene, *lilliputian*. *Development* 128, 801-813.

Technau, M., Knispel, M. and Roth, S. (2012). Molecular mechanisms of EGF signaling-dependent regulation of *pipe*, a gene crucial for dorsoventral axis formation in *Drosophila*. *Dev. Genes Evol.* 222, 1-17.

Tong, X., Gui, H., Jin, F., Heck, B. W., Lin, P., Ma, J., Fondell, J. D. and Tsai, C.-C. (2011). Ataxin-1 and Brother of ataxin-1 are components of the Notch signalling pathway. *EMBO Rep.* 12, 428-435.

Tootle, T. L. and Rebay, I. (2005). Post-translational modifications influence transcription factor activity: a view from the ETS superfamily. *BioEssays* 27, 285-298.

Tsai, C. C., Kao, H. Y., Mitzutani, A., Banayo, E., Rajan, H., McKeown, M. and Evans, R. M. (2004). Ataxin 1, a SCA1 neurodegenerative disorder protein, is functionally linked to the silencing mediator of retinoid and thyroid hormone receptors. *Proc. Natl. Acad. Sci. USA* 101, 4047-4052.

Tseng, A.-S. K., Tapon, N., Kanda, H., Cigizoglu, S., Edelmann, L., Pellock, B., White, K. and Hariharan, I. K. (2007). Capicua regulates cell proliferation downstream of the Receptor Tyrosine Kinase/Ras signaling pathway. *Curr. Biol.* 17, 728-733.

von Ohlen, T. and Doe, C. Q. (2000). Convergence of Dorsal, Dpp, and Egfr signaling pathways subdivides the *Drosophila* neuroectoderm into three dorsal-ventral columns. *Dev. Biol.* 224, 362-372.

Weiss, J. B., Von Ohlen, T., Mellerick, D. M., Dressler, G., Doe, C. Q. and Scott, M. P. (1998). Dorso-ventral patterning in the *Drosophila* central nervous system: the intermediate neuroblasts defective homeobox gene specifies intermediate column identity. *Genes Dev.* 12, 3591-3602.

Xu, C., Kauffmann, R. C., Zhang, J., Kladny, S. and Carthew, R. W. (2000). Overlapping activators and repressors delimit transcriptional response to receptor tyrosine kinase signals in the *Drosophila* eye. *Cell* 103, 87-97.

Yan, C. and Boyd, D. D. (2007). Regulation of matrix metalloproteinase gene expression. *J. Cell Physiol.* 211, 19-26.

Yoon, S. and Seger, R. (2006). The extracellular signal-regulated kinase: multiple substrates regulate diverse cellular functions. *Growth Factors* 24, 21-44.

Yordi, J. S. and Muise-Helmericks, C. (2000). Signal transduction and the Ets family of transcription factors. *Oncogene* 19, 6503-6513.

Zand, T. P., Reiner, D. J. and Der, C. J. (2011). Ras effector switching promotes divergent cell fates in *C. elegans* vulval patterning. *Dev. Cell* 20, 84-96.

Zoghbi, H. Y. and Orr, H. T. (2009). Pathogenic mechanisms of a polyglutamine-mediated neurodegenerative disease, spinocerebellar ataxia type 1. *J. Biol. Chem.* 284, 7425-7429.
