
# LETTERS

Integrated analysis of somatic mutations and focal copy-number changes identifies key genes and pathways in hepatocellular carcinoma

Cécile Guichard${}^{1,2,14}$, Giuliana Amaddeo${}^{1,2,14}$, Sandrine Imbeaud${}^{1,2,14}$, Yannick Ladeiro${}^{1,2}$, Laura Pelletier${}^{1,2}$, Ichrafe Ben Maad${}^{1,2}$, Julien Calderaro${}^{1-3}$, Paulette Bioulac-Sage${}^{4,5}$, Mélanie Letexier${}^{6}$, Françoise Degos${}^{7}$, Bruno Clément${}^{8}$, Charles Balabaud${}^{4,5}$, Eric Chevet${}^{4}$, Alexis Laurent${}^{9,10}$, Gabrielle Couchy${}^{1,2}$, Eric Letouzé${}^{11}$, Fabien Calvo${}^{12}$ & Jessica Zucman-Rossi${}^{1,2,13}$

Hepatocellular carcinoma (HCC) is the most common primary liver malignancy. Here, we performed high-resolution copy-number analysis on 125 HCC tumors and whole-exome sequencing on 24 of these tumors. We identified 135 homozygous deletions and 994 somatic mutations of genes with predicted functional consequences. We found new recurrent alterations in four genes ($ARID1A$, $RPS6KA3$, $NFE2L2$ and $IRF2$) not previously described in HCC. Functional analyses showed tumor suppressor properties for $IRF2$, whose inactivation, exclusively found in hepatitis B virus (HBV)-related tumors, led to impaired $TP53$ function. In contrast, inactivation of chromatin remodelers was frequent and predominant in alcohol-related tumors. Moreover, association of mutations in specific genes ($RPS6KA3$-$AXIN1$ and $NFE2L2$-$CTNNB1$) suggested that Wnt/$\beta$-catenin signaling might cooperate in liver carcinogenesis with both oxidative stress metabolism and Ras/mitogen-activated protein kinase (MAPK) pathways. This study provides insight into the somatic mutational landscape in HCC and identifies interactions between mutations in oncogene and tumor suppressor gene mutations related to specific risk factors.

Hepatocellular carcinoma is the third most frequent cause of cancer-related mortality worldwide and is usually associated with specific risk factors, including hepatitis B or C infection, high alcohol intake, hemochromatosis or nonalcoholic fatty liver disease (NAFLD) caused by obesity and insulin resistance${}^{1,2}$. More than 90% of HCC cases arise in the context of chronic hepatitis and cirrhosis, and this cancer is increasingly becoming a health problem. Here, we analyzed the whole-exome sequences of 24 human HCC tumors and paired non-tumor liver tissues, with many of these tumors occurring in individuals with high alcohol intake (Online Methods and Supplementary Tables 1 and 2). In total, we analyzed 125 HCC tumors related to various risk factors to identify chromosomal alterations and validate the recurrently mutated genes found by exome sequencing (Supplementary Tables 1 and 2).

For the 24 pairs of tumors and matched normal liver, we obtained 73-fold mean sequence coverage of targeted exonic regions, with 76% of loci covered at >25-fold (Supplementary Fig. 1). To search for somatic mutations, we identified nucleotide variants that were present in all tumors and absent in corresponding non-tumor tissue, by filtering the data according to an experimental flowchart that was applied to all the samples (Supplementary Fig. 2). Then, we selected the mutations that were highly predicted to impair the function of the corresponding encoded proteins. We identified 994 mutations that were further verified by Sanger sequencing to represent somatic events altering 906 different genes (Supplementary Table 3). The number of somatic mutations with predicted functional consequences per tumor was highly variable, ranging from 5 to 121 events (Fig. 1a). As is typical in solid tumors${}^{3}$, most of the variants (74%) led to missense changes, which outnumbered small insertion and/or deletion (indel; 14%) and nonsense and splice-site (12%) modifications. It was not possible to identify chromosome rearrangements by exome sequencing; thus, the frequency of chimeric genes that was previously described${}^{4,5}$ remains to be evaluated in this data set.

The observed mutation spectrum could be indicative of specific mutagenesis mechanisms occurring in tumor cells. Of note, among the 994 verified mutations, 850 corresponded to single-nucleotide

---

1 Institut National de la Santé et de la Recherche Médicale (INSERM), Unité Mixte de Recherche (UMR)-674, Génomique Fonctionnelle des Tumeurs Solides, Institut Universitaire d’Hematologie (IUH), Paris, France.  
2 Université Paris Descartes, Labex Immuno-oncology, Sorbonne Paris Cité, Faculté de Médecine, Paris, France.  
3 Assistance Publique–Hôpitaux de Paris (AP-HP), Department of Pathology, Centre Hospitalier Universitaire (CHU) Henri Mondor, Créteil, France.  
4 INSERM, UMR-1053, Université Victor Segalen Bordeaux 2, Bordeaux, France.  
5 CHU de Bordeaux, Pellegrin Hospital, Department of Pathology, Bordeaux, France.  
6 IntegraGen, Evry, France.  
7 Department of Hepatology, AP-HP, Beaujon Hospital, Université Paris Diderot, Clichy, France.  
8 INSERM, UMR-991, Université de Rennes 1, Rennes, France.  
9 AP-HP, CHU Henri Mondor, Digestive, Hepatobiliary and Liver Transplantation, Créteil, France.  
10 INSERM, U955, Créteil, France.  
11 Programme Cartes d’Identité des Tumeurs, Ligue Nationale Contre le Cancer, Paris, France.  
12 Institut National du Cancer (INCa), Boulogne, France.  
13 AP-HP, Hopital Européen Georges Pompidou, Paris, France.  
14 These authors contributed equally to this work. Correspondence should be addressed to J.Z.-R. (zucman@cephb.fr).

Received 13 December 2011; accepted 28 March 2012; published online 6 May 2012; doi:10.1038/ng.2256

Figure 1 Profiles of verified mutations with functional consequences in HCC. (a) Distribution of the 994 somatic mutations in each HCC sample. (b) Frequency of indels and nucleotide substitutions in each HCC sample. (c,d) Types of mutations on the transcribed and non-transcribed strands for the 850 verified nucleotide substitutions identified in exome sequencing of 24 HCC tumors (mean with 95% confidence interval) (c) and for the 118 mutations identified in the set of 125 HCC tumors (absolute number of variants per HCC) (d).

variants, showing an over-representation of nucleotide transversion, particularly of G:C>T:A changes (Fig. 1b). This mutation spectrum strongly differs from that usually observed in other solid tumors, where C:G>T:A transitions are the most abundant alteration³. Moreover, in HCC tumors, G>T transversions were significantly enriched in the non-transcribed DNA strand (Fig. 1c,d), which is less efficiently repaired after genotoxic injury⁶,⁷. We then searched for association with clinical features and found that G:C>T:A changes were significantly more frequent in HCC tumors that developed on non-cirrhotic livers (*P* = 0.01; Supplementary Table 4) and in well-differentiated tumors (*P* = 0.01) than in other tumor types. These results strongly suggest that exposure to genotoxic agents could contribute to hepatocarcinogenesis independent of cirrhotic status. The best-characterized genotoxic inducer involved in HCC is exposure to aflatoxin B1 in combination with HBV infection in subtropical regions, which is associated with G>T transversion at codon 249 of TP53 (refs. 8,9). However, in the present series of individuals with HCC living in France, the causative genotoxic agent remains to be determined by epidemiological and/or toxicological studies focusing on persons with non-cirrhotic HCC.

SNP array analysis of 125 HCC tumors identified frequent chromosome gains (at 1q, 5, 6p, 7, 8q, 17q and 20) and losses (at 1p, 4q, 6q, 8p, 13q, 16, 17p and 21), as previously described in HCC¹⁰ (Supplementary Fig. 3a). Hyperploid DNA content (ploidy of ≥3) was found in 20% of tumors and was associated with poor tumor differentiation (Edmondson tumor grade III or IV, *P* = 0.03; Supplementary Table 4). We also evaluated the level of chromosomal instability in each tumor by calculating the fraction of aberrant arms (FAA; the proportion of chromosome arms altered on more than 40% of their length). Highly rearranged copy-number profiles were more frequent in HCC tumors that developed in non-cirrhotic liver (*P* = 0.04; Supplementary Table 4) or with HBV infection (*P* = 0.01) and in tumors with high serum α-fetoprotein (*P* = 0.03), large size (*P* = 0.004) or poor tumor differentiation (Edmondson tumor grade III or IV, *P* = 0.04) than in other tumor types. Focal chromosome amplifications (of <3 Mb) were identified in 32% of the tumors. We observed only one region, at 11q13.3, that was amplified in two tumors (Supplementary Table 5); this region included *CCND1* and *FGF19*, two genes previously found to be amplified in HCC¹¹. Homozygous deletions were more frequent (40%) and were usually focal (mean size = 0.2 Mb; Supplementary Table 6), and these deletions were significantly associated with tumor aggressiveness (Edmondson tumor grade III or IV, *P* = 0.04; high levels of serum α-fetoprotein, *P* = 0.008; microscopic vascular invasion, *P* = 0.009; Supplementary Table 4) and poor survival (*P* < 0.0001; Supplementary Fig. 3b). A total of 12 regions were recurrently altered by homozygous deletion; with the most frequently affected located at the *CDKN2A-CDKN2B* (6.4%), *AXIN1* (3.2%) and *IRF2* (3.2%) loci (Supplementary Fig. 4).

To obtain a more comprehensive understanding of the altered genetic landscape in HCC, we performed additional screening for mutations present in seven genes expressed in liver that were modified by homozygous deletion and/or mutation in at least three tumors

(Fig. 2 and Supplementary Tables 3 and 6). We also screened seven other genes well known to be recurrently altered in hepatocellular adenoma (*IL6ST*¹² and *HNF1A*¹³) and cases of HCC (*ARID2* (ref. 14), *TP53, APC, PIK3CA* and *KRAS*¹⁰). Mutational screening in 125 HCC samples identified 4 genes (*CTNNB1, TP53, ARID1A* and *AXIN1*) that were altered in more than 10% of the tumors, whereas the other genes were less frequently mutated. We identified a total of 118 somatic point mutations in 125 tumors (Supplementary Table 7). The spectrum of mutations showed a distribution similar to that observed in exome sequencing, with an over-representation of G>T transversions on the non-transcribed DNA strand (Fig. 1d). Hence, we confirmed the genotoxic signature in this series of HCC tumors of various etiologies.

In both the exome sequencing and tumor validation sets (*n* = 125), the Wnt/β-catenin pathway was the most frequently altered, with the occurrence of either activating mutations in *CTNNB1* (encoding β-catenin; 32.8%) or inactivating mutations in *AXIN1* (15.2%) or *APC* (1.6%) (Fig. 3 and Supplementary Table 8). *CTNNB1, AXIN1* and *APC* gene alterations were mutually exclusive (only one HCC tumor had mutations in both *CTNNB1* and *AXIN1*). The mutation spectra were classical, except for four rare *CTNNB1* mutations identified in exons 6 to 8 (Supplementary Fig. 5 and Supplementary Table 7). *CTNNB1* mutations defined a homogenous subtype of HCC not related to HBV infection (*P* = 0.001) and associated with transcriptome subclasses G5-G6 (ref. 15) (*P* < 1 × 10⁻⁹; Fig. 2 and Supplementary Table 4). In contrast, *AXIN1* and *APC* mutations occurred in HCC of various etiologies, including tumors occurring in the presence of HBV infection. With exome sequencing, unique mutations in the *FZR1, CSNK1E* and *CDC16* genes that are related to the Wnt/β-catenin pathway were also identified but were not further evaluated.

LETTERS

Number of events 3222222223233332211111111212311111111142323222211111111111223233343121223212211111211212111111000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
Figure 4 IRF2 is a new tumor suppressor gene in HCC that controls the p53 pathway. (a) Effect of *in vitro* IRF2 silencing in the HepaRG cell line. Left, increased cell proliferation was seen in cells transfected with IRF2 siRNA compared to those transfected with control siRNA in triplicate experiments with regression analysis. Right, relative mRNA and protein levels were quantified by quantitative RT-PCR (qRT-PCR) and protein blotting, respectively (*n* = 3; mean ± s.d.). (b) Effect of IRF2 overexpression in the HepaRG cell line. Left, IRF2 overexpression in the HepaRG cell line induced marked cell death compared to HepaRG cells transfected with an empty plasmid in triplicate experiments with regression analysis. Right, relative mRNA and protein levels are shown (*n* = 3; mean ± s.d.). (c) Increased apoptosis in HepaRG cells overexpressing IRF2 compared to those transfected with empty plasmid. Left, flow cytometry analysis shows a higher percentage of Annexin V (FITC)-positive and propidium iodide (PI)-positive cells with IRF2 overexpression. Right, quantification of the flow cytometry results (*n* = 3; mean ± s.d.). (d) Subcutaneous xenograft mouse model with IRF2 inactivation. Left, stable IRF2 silencing by shRNAs in the HepaRG cell line enhanced tumor growth in a subcutaneous xenograft mouse model (*n* = 5; mean tumor volume ± s.d.). Center, mRNA and protein expression analysis of IRF2 in five xenograft tumors from HepaRG cells with control or IRF2 shRNA 4 weeks after injection. Right, immunohistochemical analysis of IRF2 in a xenograft tumor from HepaRG cells with control or IRF2 shRNA confirmed IRF2 silencing in cells with IRF2 shRNA. (e) IRF2 silencing induced functional inactivation of p53. Left, protein blot analysis comparing p53, IRF2 and β-actin levels in HepaRG cells transfected with control or IRF2 siRNA after 48 h, 72 h and 96 h of transfection. Protein blot analysis shows reduced expression of p53 with IRF2 silencing. Right, strong positive correlation between p53 and IRF2 levels was observed (linear regression analysis, Pearson’s correlation). (f) p53 pathway target gene expression in HepaRG cells with silenced (blue) or enhanced (red) IRF2 expression. qRT-PCR data were normalized to mean expression with the relevant siRNA or empty vector control, as indicated by a line (*n* = 3; mean ± s.d.).

Genes encoding chromatin remodeling complex components constituted the third most frequently altered class identified in exome screening. In addition, we validated ARID1A and ARID2 mutations in 16.8% and 5.6% of the cases in the tumor validation set, respectively. The ARID1A and ARID2 proteins are part of the SWItch/Sucrose Non-Fermentable (SWI/SNF)-related chromatin remodeling complexes that controls the accessibility of promoter regions to the transcriptional machinery and has been shown to have tumor suppressor functions^{19,20}. ARID1A belongs to the Brg1-associated factor (BAF) complex and was recently shown to be inactivated in several tumor types, including gastric, ovarian and bladder carcinoma, but its involvement in HCC has never been reported^{21–24}. Here, we identified mutations predicted to inactivate ARID1A function similar to those described in other tumor types (Supplementary Fig. 5). ARID1A mutations were significantly more frequent in HCC related to alcohol intake (*P* = 0.002) than in tumors of other etiology and showed a significant association with CTNNB1 mutations (*P* = 0.05; Supplementary Table 4). In contrast, mutations in ARID2, whose protein product belongs to the polybromo- and Brg1-associated factor (PBAF) complex, were less frequent but exclusive from ARID1A mutations. Overall, ARID2 mutation frequency was similar to that previously observed in HCC^{14} but was not significantly associated with hepatitis C virus (HCV) infection or any other risk factor in our study. In exome sequencing, we identified mutations in additional genes participating in chromatin remodeling (PBRM1, SMARCA1, SMARCA2, SMARCA4, SMARCAB1 and SMARCD1) (Fig. 3). However, most of these mutations were not recurrent and were not exclusive from ARID1A and ARID2 alterations. The mutation frequencies of these six genes remains to be evaluated in larger series of HCC samples, but, overall, more than 24% of HCC tumors had a mutation in at least one gene related to chromatin remodeling, thereby indicating that this pathway might significantly contribute to hepatocyte tumorigenesis.

Among the genes less frequently mutated in HCC, we identified, for the first time in solid tumors, recurrent mutations in RPS6KA3 (9.6%), a gene located on chromosome X that encodes the ribosomal S6 protein kinase 2 (RSK2). RSK2 is a serine/threonine kinase of the Ras/MAPK signaling pathway that is directly phosphorylated and activated by extracellular-regulated kinases 1 and 2 (ERK1/2). RSK2 exerts feedback inhibition on the ERK1/2 pathway by phosphorylating and inhibiting the guanine nucleotide exchange factor SOS^{25,26}. Half of the RPS6KA3 mutations we identified led to premature stop codons or altered splicing sites, whereas four out of seven missense mutations were located close to the codons for the Ser227 and Thr557 phosphorylation sites that are required for RSK2 activation (Supplementary Fig. 5). Consequently, RPS6KA3 somatic mutations were predicted to inactivate RSK2 function. Other components of the Ras/MAPK and phosphatidylinositol 3-kinase (PI3K) pathways were rarely mutated (Fig. 3). Of note, in 11 tumors out of 12, RPS6KA3 mutations were found in HCC that developed in the absence of cirrhosis (*P* = 0.05), and alterations in

RPS6KA3 were frequently associated with AXIN1 mutations (*P* = 0.02; Supplementary Table 4), suggesting cooperation between RPS6KA3 inactivation and Wnt/β-catenin activation in tumorigenesis.

Finally, *NFE2L2* was mutated in 6.4% of HCC tumors (Fig. 3). *NFE2L2* encodes NRF2, a transcription factor that is crucial for cellular redox homeostasis. All *NFE2L2* mutations were located within the DLG and ETGE motifs, hotspots of somatic mutations previously identified in lung, esophageal, laryngeal and skin squamous cell carcinomas<sup>27–29</sup> (Supplementary Fig. 5). These mutations are known to inhibit KEAP1-mediated degradation of NRF2. Of note, six out of eight HCC samples with *NFE2L2* mutations also had *CTNNB1* mutations (*P* = 0.01), with another tumor also having an *AXIN1* mutation. Moreover, an additional tumor with a *KEAP1*-inactivating mutation was also found to have a mutation in *CTNNB1*. Hence, these results identify for the first time in HCC a role for alteration in the oxidative stress pathway, mainly in tumors with activated Wnt/β-catenin signaling.

In conclusion, we have identified new genes recurrently altered in HCC, which suggest a major role for SWI/SNF chromatin remodeling complexes and involvement of both interferon and oxidative stress pathways in hepatocellular malignant proliferation and transformation.

URLs. EGA, https://www.ebi.ac.uk/ega/; ICGC, http://www.icgc.org/; data from this study, ftp://data.dcc.icgc.org/version_8/; R spider, http://www.bioprofiling.de/R_spider.html; KEGG, http://www.genome.jp/kegg/; Reactome, http://www.reactome.org/; GEO, http://www.ncbi.nlm.nih.gov/geo/.

**METHODS**

Methods and any associated references are available in the online version of the paper.

**Accession codes.** Sequencing data have been deposited at the European Genome-phenome Archive (EGA; EGAS00001000217). Comparative genomic hybridization data have been deposited at the Gene Expression Omnibus (GEO; GSE32649). All variants and tumor features are deposited at the International Cancer Genome Consortium (ICGC) portal.

**Note:** Supplementary information is available in the online version of the paper.

**ACKNOWLEDGMENTS**

We warmly thank T. Burguiere, G. Thomas, R. Fahraeus and C. Mlynarczyk for their helpful participation in this work. We also thank J. Saric, C. Laurent, B. Le Bail, A. Rullier, A. Sa Cunha, J.T. Van Nhieu, D. Cherqui and D. Azoulay for contributing to tissue collection. This work was supported by INCa with the ICGC project, the Ligue Nationale Contre le Cancer (Cartes d’Identité des Tumeurs program), the PAIR-CHC project NoFLIC (funded by INCa and the Association pour la Recherche Contre le Cancer, ARC), the Réseau National Centre de Recherches Biocosmétiques (CRB) Foie, HEPTROMIC (Framework Programme 7, FP7) and BioIntelligence (OSEO). G.A. is supported by a fellowship from the Agence Nationale de Recherches sur le Sida et les Hepatites Virales (ANRS).

**AUTHOR CONTRIBUTIONS**

C.G., S.I., M.L. and I.B.M. designed, analyzed and verified the whole-exome sequencing data. G.A. and Y.L. designed and performed the IRF2 analyses. S.I. performed statistical analyses. C.G., G.A., Y.L., L.P., I.B.M. and G.C. performed the sequencing validation. J.C. and P.B.-S. provided samples and pathological reviewing. F.D., B.C., C.B. and A.L. provided samples and clinical information. S.I., C.G. and E.C. contributed to pathway analyses. E.L. and S.I. analyzed copy-number variant data. F.C. organized the project through the ICGC and participated in the achievement of the study. J.Z.-R. designed and coordinated the overall study. All authors contributed to writing the manuscript.

**COMPETING FINANCIAL INTERESTS**

The authors declare no competing financial interests.

Published online at http://www.nature.com/doifinder/10.1038/ng.2256.  
Reprints and permissions information is available online at http://www.nature.com/reprints/index.html.

---

1. El-Serag, H.B. & Rudolph, K.L. Hepatocellular carcinoma: epidemiology and molecular carcinogenesis. *Gastroenterology* **132**, 2557–2576 (2007).
2. Ferlay, J. *et al.* Estimates of worldwide burden of cancer in 2008: GLOBOCAN 2008. *Int. J. Cancer* **127**, 2893–2917 (2010).
3. Greenman, C. *et al.* Patterns of somatic mutation in human cancer genomes. *Nature* **446**, 153–158 (2007).
4. Tao, Y. *et al.* Rapid growth of a hepatocellular carcinoma and the driving mutations revealed by cell-population genetic analysis of whole-genome data. *Proc. Natl. Acad. Sci. USA* **108**, 12042–12047 (2011).
5. Totoki, Y. *et al.* High-resolution characterization of a hepatocellular carcinoma genome. *Nat. Genet.* **43**, 464–469 (2011).
6. Denissenko, M.F., Pao, A., Pfeifer, G.P. & Tang, M. Slow repair of bulky DNA adducts along the nontranscribed strand of the human p53 gene may explain the strand bias of transversion mutations in cancers. *Oncogene* **16**, 1241–1247 (1998).
7. Hainaut, P. & Pfeifer, G.P. Patterns of p53 G→T transversions in lung cancers reflect the primary mutagenic signature of DNA-damage by tobacco smoke. *Carcinogenesis* **22**, 367–374 (2001).
8. Bressac, B., Kew, M., Wands, J. & Ozturk, M. Selective G to T mutations of p53 gene in hepatocellular carcinoma from southern Africa. *Nature* **350**, 429–431 (1991).
9. Hsu, I.C. *et al.* Mutational hotspot in the p53 gene in human hepatocellular carcinomas. *Nature* **350**, 427–428 (1991).
10. Nault, J.C. & Zucman-Rossi, J. Genetics of hepatobiliary carcinogenesis. *Semin. Liver Dis.* **31**, 173–187 (2011).
11. Sawey, E.T. *et al.* Identification of a therapeutic strategy targeting amplified *FGF19* in liver cancer by Oncogenomic screening. *Cancer Cell* **19**, 347–358 (2011).
12. Rebouissou, S. *et al.* Frequent in-frame somatic deletions activate *gp130* in inflammatory hepatocellular tumours. *Nature* **457**, 200–204 (2009).
13. Bluteau, O. *et al.* Bi-allelic inactivation of *TCF1* in hepatic adenomas. *Nat. Genet.* **32**, 312–315 (2002).
14. Li, M. *et al.* Inactivating mutations of the chromatin remodeling gene *ARID2* in hepatocellular carcinoma. *Nat. Genet.* **43**, 828–829 (2011).
15. Boyault, S. *et al.* Transcriptome classification of HCC is related to gene alterations and to new therapeutic targets. *Hepatology* **45**, 42–52 (2007).
16. Tamura, T., Yanai, H., Savitsky, D. & Taniguchi, T. The IRF family transcription factors in immunity and oncogenesis. *Annu. Rev. Immunol.* **26**, 535–584 (2008).
17. Han, K.J., Jiang, L. & Shu, H.B. Regulation of IRF2 transcriptional activity by its sumoylation. *Biochem. Biophys. Res. Commun.* **372**, 772–778 (2008).
18. Pettersson, S., Kelleher, M., Pion, E., Wallace, M. & Ball, K.L. Role of Mdm2 acid domain interactions in recognition and ubiquitination of the transcription factor IRF-2. *Biochem. J.* **418**, 575–585 (2009).
19. Guan, B., Wang, T.L. & Shih Ie, M. ARID1A, a factor that promotes formation of SWI/SNF-mediated chromatin remodeling, is a tumor suppressor in gynecologic cancers. *Cancer Res.* **71**, 6718–6727 (2011).
20. Wilson, B.G. & Roberts, C.W. SWI/SNF nucleosome remodelers and cancer. *Nat. Rev. Cancer* **11**, 481–492 (2011).
21. Jones, S. *et al.* Frequent mutations of chromatin remodeling gene *ARID1A* in ovarian clear cell carcinoma. *Science* **330**, 228–231 (2010).
22. Wiegand, K.C. *et al.* *ARID1A* mutations in endometriosis-associated ovarian carcinomas. *N. Engl. J. Med.* **363**, 1532–1543 (2010).
23. Jones, S. *et al.* Somatic mutations in the chromatin remodeling gene *ARID1A* occur in several tumor types. *Hum. Mutat.* **33**, 100–103 (2012).
24. Wang, K. *et al.* Exome sequencing identifies frequent mutation of *ARID1A* in molecular subtypes of gastric cancer. *Nat. Genet.* **43**, 1219–1223 (2011).
25. Douville, E. & Downward, J. EGF induced SOS phosphorylation in PC12 cells involves P90 RSK-2. *Oncogene* **15**, 373–383 (1997).
26. Schneider, A., Mehmdood, T., Pannetier, S. & Hanauer, A. Altered ERK/MAPK signaling in the hippocampus of the *mrsk2_KO* mouse model of Coffin-Lowry syndrome. *J. Neurochem.* **119**, 447–459 (2011).
27. DeNicola, G.M. *et al.* Oncogene-induced Nrf2 transcription promotes ROS detoxification and tumorigenesis. *Nature* **475**, 106–109 (2011).
28. Kim, Y.R. *et al.* Oncogenic *NRF2* mutations in squamous cell carcinomas of oesophagus and skin. *J. Pathol.* **220**, 446–451 (2010).
29. Shibata, T. *et al.* Cancer related mutations in *NRF2* impair its recognition by Keap1-Cul3 E3 ligase and promote malignancy. *Proc. Natl. Acad. Sci. USA* **105**, 13568–13573 (2008).

ONLINE METHODS

Liver samples and clinical data. All 125 tumors and corresponding non-tumor liver tissues were frozen (−80 °C) after surgical resection in France. These tumors were previously included in genetic and phenotypic studies¹⁵ (Supplementary Tables 1 and 2). The study was approved by the local Ethics Committee (Paris Saint-Louis), informed consent was obtained in accordance with French legislation and the exome sequencing project was approved by the national INSERM institutional review board (IRB) committee in 2010.

Whole-exome sequencing. In all HCC samples, the ratio of tumor cells/non-tumor cells was determined to be >50%, and this was confirmed by SNP array analysis; all DNA was >10 kb. Sequence capture, enrichment and elution from 48 genomic DNA samples were performed by IntegraGen using Agilent in-solution enrichment (SureSelect Human All Exon Kit v2) with the provided biotinylated oligonucleotide probe library (Human All Exon v2–46 Mb). Briefly, 3 μg of each genomic DNA was sonicated and purified to yield fragments of 150–200 bp. Adaptor oligonucleotides were ligated on A-tailed fragments and enriched by four to six PCR cycles. Purified libraries (500 ng) were hybridized to the SureSelect library for 24 h. The eluted fraction was PCR amplified for 10–12 cycles and sequenced on an Illumina HiSeq2000 sequencer as paired-end 75-bp reads³⁰. Image analysis and base calling were performed using the Illumina Real Time Analysis (RTA) Pipeline version 1.9 with default parameters. Initial analysis of whole-exome sequencing was based on the Illumina pipeline (CASAVA1.7) and used the hg19 reference genome. The alignment algorithm used was ELANDv2. The targeted regions in each sample were sequenced to an average depth of 73.3×, with ~96.0% of the targeted regions covered by ≥1×, ~87.0% covered by ≥10× and ~75.9% covered by ≥25× (Supplementary Fig. 1).

A list of somatic variants in the coding regions and three intronic bases corresponding to consensus splicing sites was generated (Supplementary Fig. 2 and Supplementary Table 3). Common polymorphisms were removed by comparison to dbSNP131, 1000 Genomes Project data and the IntegraGen in-house database comprising 50 exomes not related to tumors or liver diseases. Quality control filtering removed variants sequenced in less than five reads, with QPHRED of <10 and with sequence size not included within the range represented by the mean size ± 3 s.d. Only nonsynonymous variants predicted to have functional consequences with Polyphen-2 were retained³¹. All putative somatic mutations were validated visually using the Integrated Genomics Viewer (IGV)³² and then verified by Sanger sequencing, as previously described¹². All mutations identified in tumors were confirmed by independent PCR and sequencing in each tumor and its normal counterpart.

SNP comparative genomic hybridization (CGH) array analysis. Samples were analyzed with Illumina HumanCNV370-Duo v1.0 chips (99 tumors and 86 non-tumor counterparts) or HumanOmniExpress BeadChips (26 tumors and non-tumor counterparts) by IntegraGen. Raw fluorescent signals were normalized in BeadStudio software³³ (Illumina) using reference clusters generated from our set of 112 matched normal samples and using threshold quantile normalization (tQN) for the HumanCNV370-Duo data³⁴. Analyses were performed using the DNACopy package (R Bioconductor) and the Genome Alteration Print (GAP) method³⁵ to determine the ploidy, the level of contamination with normal cells, the absolute copy number and the genotype corresponding to each cluster of segments. Chromosome gain, loss, high-level amplification and homozygous deletion were defined as copy number of ≥ ploidy +1, ≤ ploidy −1, > ploidy +2 and copy number ≤ 0.6, respectively. Homozygous deletions and focal amplifications were defined by at least three consecutive probes spanning less than 3 Mb (Supplementary Tables 5 and 6). All deletions and amplifications were verified manually to remove imprecise smoothing or undetected germline copy-number variation (CNV) in the paired normal sample. The FAA score was developed to evaluate the overall amount of chromosome rearrangement in a tumor. Consistent with previous findings³⁶, most chromosome aberrations were either focal or spanned an entire chromosome arm. We found that 95% of chromosome arms were altered on <30% or >70% of their length. The 40% cutoff used to compute the FAA score corresponds to the minimum between the two peaks in the distribution of alteration length.

Pathway analysis. First, we performed pathway enrichment analysis using Pathway-Express³⁷ on the list of 1,150 different genes somatically altered in HCC by mutation and/or homozygous deletion. Using hypergeometric tests, we identified KEGG signaling pathways significantly enriched in our list of altered genes (Supplementary Table 8). In a second step using the R spider tool³⁸, a statistical framework was computed from our gene list that integrated signaling and metabolic pathways from the KEGG and Reactome databases. Then, expert reviewing was performed using the literature, in order to amend the master gene network.

Quantitative RT-PCR. RNA extraction and quantitative RT-PCR were performed as previously described³⁹ using TaqMan Gene Expression (Applied Biosystems) for *IRF2* (Hs01082884_m1), *BBC3* (Hs00248075_m1), *GADD45A* (Hs00169255_m1), *CASP8* (Hs01018151_m1), *CASP9* (Hs00154261_m1), *PPM1D* (Hs01013291_m1), *CDKN1A* (5Hs00355782_m1), *CASP1* (Hs00355782_m1) and *FAS* (Hs00163653_m1). Relative levels of mRNA were determined using the $2^{-\Delta\Delta CT}$ method⁴⁰ and Eukaryotic 18S rRNA (Applied Biosystems) as an endogenous control.

Protein blot analysis. Cell lysates for protein blot analysis were prepared using standard RIPA buffer (Santa Cruz Biotechnology). Protein blot analyses were performed as previously described¹² using antibodies against IRF2 (Ab55331, Abcam; 1:500 dilution) and p53 (CM-1, a kind gift from B. Vojtesek (Recamo Project); 1:5,000 dilution).

Cell culture and transient modulation of *IRF2* expression. HepaRG cells (kindly provided by F. Zoulim (INSERM, U-1052, Centre National de la Recherche Scientifique (CNRS), UMR-5286) were grown in William’s medium (Invitrogen) supplemented with 10% FCS, 5 μg/ml insulin and 5 × 10⁻⁷ M hydrocortisone hemisuccinate (all from Invitrogen). HepG2 cells (purchased from ATCC) were cultured in DMEM with high glucose (Invitrogen) and 10% FCS. *IRF2* extinction was performed with two siRNAs (Silencer Select, Applied Biosystems, s7504 (*IRF2* siRNA 1) and s7506 (*IRF2* siRNA 2)) targeted to exons 8 and 3 of *IRF2*, respectively. Control siRNA (Block-IT Alexa Fluor Red) was purchased from Invitrogen. Cells were transfected using Lipofectamine RNAiMax (Invitrogen); knockdown efficiency was >80%. *IRF2* overexpression was performed by transfection of cell lines with the pCMV6-AC plasmid containing the *IRF2* coding sequence (Origene) with Lipofectamine LTX (Invitrogen). Results were compared to those obtained with empty pCMV6-AC. After transfection (48 h later), cells were selected for 72 h with G418 (300 μg/ml for HepG2, 400 μg/ml for HepaRG; Geneticin, Gibco). Transfection efficiency was monitored by measuring the levels of *IRF2* mRNA using qRT-PCR and *IRF2* protein using protein blot analysis. Proliferation assays were conducted by plating 2 × 10⁵ cells per well and counting cells every 24 h (for up to 120 h after plating) using a hemocytometer and trypan blue exclusion. In cell experiments, qRT-PCR and proliferation assays were performed in triplicate in at least two different cell lines and with two different siRNAs.

Flow cytometry. Apoptosis was detected by flow cytometry using the APOAF-50TST Annexin V-FITC Apoptosis Detection kit (Sigma). For each experiment, 50,000 cells were analyzed with a FACS CantoII (BD Bioscience).

Stable transfection with shRNA targeting *IRF2*. HepaRG and HepG2 cells were stably transfected with pRFP-C-RS vector with two different shRNAs targeting *IRF2* (FI348442 and FI348443, Origene) or with shRNA control (TR30015). Transfected cells were selected by treatment with puromycin (3 μg/ml for HepaRG, 5 μg/ml for HepG2; Invitrogen).

Tumor formation in a xenograft model. Female 6-week-old CD-1 mice were injected subcutaneously in the right flank with 4 × 10⁶ HepaRG or 5 × 10⁶ HepG2 cells (in 100 μl of PBS mixed with 100 μl of Matrigel (Becton Dickinson)) stably transfected with *IRF2* shRNA and in the left flank with the same number of cells stably transfected with control shRNA. Tumor size was measured twice a week by caliper. Mice were sacrificed 4 weeks after injection, and the tumors were dissected. Subcutaneous tumor volume was calculated as 0.52 × length × width².

Immunohistochemistry. Tumor samples were fixed in 4% neutral-buffered formalin for 48 h, and paraffin blocks were sectioned at a thickness of 5 μm. Endogenous peroxidases and biotins were blocked with Peroxidase-Blocking Solution (Dako Real) and the Vector Avidin/Biotin Blocking Kit (Vector Laboratories), respectively. Antigen retrieval was performed in boiling citrate buffer (pH 9.5; PowerVision). Sections were blocked with goat normal serum (Vectastain) and incubated for 1 hour with an antibody to IRF2 (1:100 dilution; rabbit polyclonal, clone H-229, Santa Cruz Biotechnology), and detection was amplified using the Vectastain ABC kit (Elite).

Statistical analysis. Contingency tables, along with tests of independence and measures of association, were performed in R using the X-squared test (group size of >5) with Yates’ continuity correction or Fisher’s Exact test (group size of ≤5) and the Kruskal-Wallis rank-sum test (continuous variables). Variables associated with overall survival were identified using Cox proportional hazards regression models (log-rank test) with the R survival library, and only individuals with HCC with curative (R0) resection were included in survival analysis (*n* = 99, exclusion of non-curative resections and liver transplants). Kaplan-Meier plots were used to describe survival rates. qRT-PCR expression data were presented as mean ± s.d., and non-parametric Mann-Whitney tests were used to compare quantitative values. All reported *P* values were two tailed, and differences were considered significant when the *P* value was less than 0.05.

30. G nirke, A. *et al.* Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing. *Nat. Biotechnol.* **27**, 182–189 (2009).
31. Adzhubei, I.A. *et al.* A method and server for predicting damaging missense mutations. *Nat. Methods* **7**, 248–249 (2010).
32. Robinson, J.T. *et al.* Integrative genomics viewer. *Nat. Biotechnol.* **29**, 24–26 (2011).
33. Peiffer, D.A. *et al.* High-resolution genomic profiling of chromosomal aberrations using Infinium whole-genome genotyping. *Genome Res.* **16**, 1136–1148 (2006).
34. Staaf, J. *et al.* Normalization of Illumina Infinium whole-genome SNP data improves copy number estimates and allelic intensity ratios. *BMC Bioinformatics* **9**, 409 (2008).
35. Popova, T. *et al.* Genome Alteration Print (GAP): a tool to visualize and mine complex cancer genomic profiles obtained by SNP arrays. *Genome Biol.* **10**, R128 (2009).
36. Beroukhim, R. *et al.* The landscape of somatic copy-number alteration across human cancers. *Nature* **463**, 899–905 (2010).
37. Draghici, S. *et al.* A systems biology approach for pathway level analysis. *Genome Res.* **17**, 1537–1545 (2007).
38. Antonov, A.V., Schmidt, E.E., Dietmann, S., Krestyaninova, M. & Hermjakob, H. R spider: a network-based analysis of gene lists by combining signaling and metabolic pathways from Reactome and KEGG databases. *Nucleic Acids Res.* **38**, W78–W83 (2010).
39. Rebouissou, S. *et al.* HNF1α inactivation promotes lipogenesis in human hepatocellular adenoma independently of SREBP-1 and carbohydrate-response element–binding protein (ChREBP) activation. *J. Biol. Chem.* **282**, 14437–14446 (2007).
40. Livak, K.J. & Schmittgen, T.D. Analysis of relative gene expression data using real-time quantitative PCR and the 2^−ΔΔCT method. *Methods* **25**, 402–408 (2001).
