
REVIEWs IN BASIC AND CLINICAL
GASTROENTEROLOGY AND HEPATOLOGY

Robert F. Schwabe and John W. Wiley, Section Editors

MicroRNAs in Pathogenesis, Diagnosis, and Treatment of
Gastroesophageal Cancers

JEE HOON SONG* and STEPHEN J. MELTZER*,†

*Division of Gastroenterology, Department of Medicine, and †Department of Oncology, Sidney Kimmel Comprehensive Cancer Center, Johns Hopkins University School of Medicine, Baltimore, Maryland

The incidence of gastroesophageal cancers is increasing each year, but despite much research, their molecular mechanisms are incompletely understood. microRNAs (miRNAs) are noncoding RNAs that have been associated with gastroesophageal carcinogenesis. We review the involvement of miRNAs in gastric and esophageal cancers and their mechanisms of regulation, effects on gene expression, and biological functions. Many miRNAs are dysregulated in gastroesophageal cancer cells via alterations in transcription, epigenetic features, or copy number of the genes that encode them. Each type of gastroesophageal tumor has a unique gene expression profile. miRNAs contribute to gastroesophageal carcinogenesis by altering expression of oncogenes and tumor suppressors to affect cell proliferation, apoptosis, and motility and invasion. A number of miRNAs, including circulating miRNAs, have been associated with tumor type or stage, or patient survival, and might be developed as diagnostic or prognostic markers. Greater understanding of the roles of miRNAs in gastroesophageal carcinogenesis could provide insights into the mechanisms of tumor development and identify therapeutic targets.

Keywords: Noncoding RNA; MicroRNA; Esophageal Carcinoma; Gastric Carcinoma.

Esophageal Cancers

Esophageal cancer ranks 8th in incidence and 6th in cancer death worldwide; an estimated 482,000 new cases of this disease and 406,000 related deaths occurred in 2008.²³ Although esophageal cancers are endemic to many parts of the world, approximately 86% of deaths from esophageal cancer occur in less developed nations.²³ Esophageal cancers are classified into 2 major histopathologic subtypes: esophageal squamous cell carcinoma (ESCC) and esophageal adenocarcinoma (EAC). ESCC is the most frequent subtype of esophageal cancer internationally, whereas EAC predominates in Western populations. ESCC and EAC represent 2 separate entities, with distinct etiologies and pathologies that entail somewhat different detection and therapeutic strategies.

ESCC is the predominant histologic subtype found in eastern Asia and eastern and southern Africa.⁴ Nicotine and alcohol abuse are the major risk factors for ESCC, and although ESCC has decreased in incidence during the past 20 years, the overall prognosis of patients with ESCC remains poor.⁴⁵ Patients are usually diagnosed with later-stage ESCC, which has limited treatment options. Therefore, there is a need for biomarkers to improve early detection.

EAC is one of the most rapidly increasing cancers in Western countries.⁶ As for ESCC, most EACs are detected at advanced stages, leading to poor overall 5-year rates of survival.⁷ Gastroesophageal reflux disease and Barrett’s esophagus (BE) are the major risk factors for EAC.⁸⁹

Abbreviations used in this paper: AUC, area under the receiver-operating characteristic curve; BE, Barrett’s esophagus; CDK, cyclin-dependent kinase; CKI, cyclin-dependent kinase inhibitor; EAC, esophageal adenocarcinoma; EGFR, epidermal growth factor receptor; EMT, epithelial-to-mesenchymal transition; ESCC, esophageal squamous cell carcinoma; miRNA, microRNA; MMP, matrix metalloproteinase; PI3K, phosphatidylinositol 3-kinase; UTR, untranslated region.

© 2012 by the AGA Institute
0016-5085/$36.00
http://dx.doi.org/10.1053/j.gastro.2012.05.003

complete understanding of EAC pathogenesis should therefore include the study of BE. Endoscopic monitoring of BE is not foolproof in predicting its progression to high-grade dysplasia or EAC. It has a low diagnostic yield and high cost, so there is a need for biomarkers to better define or predict its neoplastic progression.¹⁰

# Gastric Cancer

Gastric cancer is the 4th most common cancer and the second-highest cause of cancer death worldwide. There were approximately 989,000 new cases of gastric cancer and 738,000 related deaths in 2008. More than 70% of cases (approximately 714,000 per year) occur in developing countries, with approximately 50% of all gastric cancers occurring in eastern Asia.² Gastric carcinogenesis is a multistep process that involves many environmental and genetic factors. *Helicobacter pylori* is implicated in more than 80% of gastric cancers.¹¹ However, most of the approximately 1 billion people infected worldwide do not develop gastric cancer, so *H. pylori* infection alone is not sufficient to cause the disease. Many studies have shown that levels of the hormone gastrin,¹² as well as genetic,¹³¹⁴ dietary,¹⁵ and other chronic gastric inflammation-causing factors, appear to synergize in development of gastric cancer.

Gastric cancers are complex diseases with many classification systems. They are classified according to histologic type (intestinal or diffuse) or anatomic location (eg, cardiac, body, or antrum). This complexity makes it a challenge to study the molecular mechanisms of gastric carcinogenesis, particularly because there is inadequate information available on the anatomic site, stage, and histology of many tumors. Recent studies have focused on the involvement of miRNAs in the pathogenesis of gastric cancer.

## miRNA Biogenesis and Function

miRNAs are abundant, noncoding RNA molecules of 18–25 nucleotides that inhibit translation or promote degradation of messenger RNAs (mRNAs) of complementary sequences. Since the identification of *lin-4* in 1993 in *Caenorhabditis elegans*,¹⁶ many additional miRNAs have been identified in a wide range of species. Many miRNAs are evolutionarily conserved among remotely related organisms, indicating the involvement of miRNAs in essential cellular processes.¹⁷ More than 1500 miRNAs have been identified in humans (miRBase version 18). miRNAs are estimated to regulate expression of 30%–60%¹⁸¹⁹ of human genes; each miRNA can interact with many mRNAs, and conversely each mRNA can be regulated by multiple miRNAs. Increasing our understanding of miRNA biogenesis will provide insight into the roles of these small transcripts in normal and abnormal cellular functions and carcinogenesis.

### miRNA Biogenesis

Genes that encode most human miRNAs are located in introns, so they share promoters and other regulatory elements with exons (protein-coding regions).²⁰²¹ However, genes that encode many miRNAs are also located between protein-coding genes and have their own promoter and regulatory elements. miRNA gene loci are often in proximity to each other; the clusters of miRNA genes are transcribed from a common promoter to generate polycistronic primary transcripts (pri-miRNAs). In fact, approximately 40% of total human miRNA loci are within less than 3 kb of another miRNA locus.²²

When miRNAs are generated as monocistronic or polycistronic pri-miRNAs by RNA polymerase II²³ or III,²⁴ the pri-miRNA transcript is processed within the nucleus into a 60- to 90-nucleotide stem-loop precursor RNA (pre-miRNA) by Drosha and its cofactor, DGCR8 (the Drosha-DGCR8 complex). The resulting pre-miRNA is actively transported by exportin-5 from the nucleus to the cytoplasm, which requires RanGTP. In the cytoplasm, the pre-miRNA is further processed by the ribonuclease III endonuclease Dicer, which cuts the pre-miRNA into an RNA duplex of 20–23 bases. Either strand of the miRNA duplex can be incorporated into the RNA-induced silencing complex or degraded based on base-pairing stability at the duplex termini.²⁵ The activated guide strand loads into the RNA-induced silencing complex to silence target mRNAs, whereas the passenger miRNA strand is typically degraded.

#### miRNA Function

Posttranscriptional repression by miRNAs requires base pairing between the miRNA seed region (2–8 nucleotides) and its target mRNA sites. Many mRNA target sites reside within 3′ untranslated regions (UTRs) of mRNAs. However, there are functional miRNA target sites within the 5′ UTR and in coding regions, so studies of mRNA target sites should not be limited to 3′ UTRs.²⁶⁻²⁸

miRNAs repress gene expression by decreasing levels of mRNAs and/or reducing their translation. Although miRNAs were initially believed to function by inhibiting translation, they were later found to mediate mRNA decay by deadenylation, decapping, and exonucleolytic digestion of the mRNA²⁹⁻³¹; this is their predominant mechanism of reducing gene expression in animals.³²³³

Perfect complementarity between an miRNA and its target mRNA occurs most frequently in plant cells and results in mRNA cleavage; partial complementarity, which is more common in animals, prevents translation of the mRNA. Little is known about the precise conditions of miRNA-mRNA duplex formation required to induce mRNA decay.

### Dysregulation of miRNAs in Human Cancers

miRNAs regulate development and cell proliferation, differentiation, and apoptosis, so alterations in their expression contribute to human diseases, including cancers.
cer. Specific patterns of miRNA expression have been associated with almost every type of tumor, and a number of miRNAs are involved in tumorigenesis and tumor progression. Depending on its mRNA target, an miRNA can function as a tumor suppressor or promoter of oncogenesis.

Many studies have focused on aberrantly expressed miRNAs and their effects on tumorigenesis. However, the mechanisms that dysregulate miRNAs have not been widely studied and are poorly understood. These include aberrant miRNA biogenesis and transcription, ${ }^{34,35}$ epigenetic alteration, ${ }^{36,37}$ and amplification or loss of genomic regions that encode miRNAs. ${ }^{38,39}$

In gastric cancer cells, aberrant expression of Dicer and Drosha is associated with an altered miRNA profile compared with normal gastric cells. ${ }^{40}$ These ribonuclease III endonucleases are required for miRNA biogenesis, so disrupting their expression would be expected to alter a cell’s miRNA pattern. Beyond interference with Dicer or Drosha, the transcription of genes that encode miRNAs can be disrupted by dysfunction or dysregulation of transcription factors; regulation of miRNA genes is similar to that of protein-coding genes. In gastric cancer cells, the transcription factor E2F1 directly up-regulates the transcription of the miR-106b-25 cluster, which in turn down-regulates E2F1, establishing a negative feedback loop. ${ }^{41}$ Similarly, the transcription factor Smad3 directly up-regulates miR-200 family members. ${ }^{42}$ Moreover, miR-223 is directly activated by the transcription factor Twist in metastatic gastric tumors. ${ }^{43}$ Dysregulation of transcription factors could cause some of the miRNA abnormalities observed in cancer cells.

Levels of miRNAs in a cell can also be affected by DNA methylation. Methylation of CpG islands in promoter regions silences gene expression, and methylation of tumor suppressor genes leads to tumorigenesis. ${ }^{44}$ DNA methylation silences genes that encode miRNAs in a number of human tumor types. In gastric cancer cells, DNA methylation and histone deacetylation are involved in silencing of genes that encode miR-10b, miR-34b, miR-129, miR-139, miR-181c, miR-196b, miR-375, and miR-512-5p. ${ }^{45-51}$ There have been few studies of the effects of epigenetic modifications to miRNA genes in esophageal cancer cells. Genes encoding miR-192-2 and members of the miR-34 family are frequently methylated in ESCC tissues; this might serve as a biomarker for early-stage diagnosis. ${ }^{52}$

Genome-wide studies have shown that miRNA genes are frequently located within genomic regions at which loss of heterozygosity and amplification occur, at fragile sites, or in areas that have other types of alterations associated with transformation. ${ }^{39}$ In esophageal cancer cells, up-regulation of the miR-106b-25 polycistron is associated with variations in de novo copy number and overexpression of the gene with which it shares a promoter, MCM7. ${ }^{53}$ Conversely, expression of miR-486, which has been shown to function as a tumor suppressor, is down-regulated through loss of its locus in 25%-30% of

---

106 primary gastric cancer tissue samples. ${ }^{54}$ Dysregulation of mechanisms that control miRNA gene expression therefore appear to be important pathophysiologic events in gastroesophageal carcinogenesis.

### miRNA Expression Profiles in Gastroesophageal Tumors

Microarray assays or real-time polymerase chain reaction-based approaches can be used to determine expression patterns of miRNAs in cell lines or primary tumor samples and identify miRNAs involved in carcinogenesis. miRNA profiling provides accurate classification of tissue and tumor types. ${ }^{55,56}$ However, interpreting these data is often not straightforward; there have been conflicting reports on expression levels of several miRNAs. These inconsistent data could result from many factors, such as variations in study subject selection criteria, limitations on the sensitivity and specificity of profiling platforms, tumor heterogeneity, differences in sample collection and storage, and patient treatment regimens. Independent verification of altered expression patterns of individual miRNAs is often required to resolve these conflicts.

To minimize inconsistency and highlight the most significant miRNA profile data, we will discuss only findings reported in more than one study of miRNAs with a 1.5-fold change or more in expression between cancer and noncancer tissues.

#### Esophageal Tumors

Several independent profiling studies reported altered expression of miRNAs in ESCC tissues. ${ }^{57-61}$ miR-21 and miR-155 were overexpressed in ESCCs compared with normal tissues, whereas let-7c, miR-1, miR-99a, miR-100, miR-133a, miR-143, miR-145, miR-203, and miR-375 were underexpressed in ESCC tissues (Table 1).

Similarly, several miRNA profiling studies found altered expression of miRNAs in BE and EAC samples. ${ }^{57,60,62-64}$ In these studies, miR-21, miR-28, miR-30a-5p, miR-143, miR-145, miR-192, miR-194, and miR-215 were consistently up-regulated in EACs compared with normal esophageal tissues, whereas let-7c, miR-23a, miR-27a, miR-27b, miR-31, miR-99a, miR-100, miR-125b, miR-203, and miR-205 were down-regulated in EACs (Table 1).

Interestingly, many miRNAs found to be up-regulated in EAC compared with normal tissues were also up-regulated in BE tissues; many miRNAs that were down-regulated in EACs were also down-regulated in BE tissues. miR-21, miR-143, miR-145, miR-192, miR-194, and miR-215 were up-regulated in EAC and BE samples, whereas let-7c, miR-203, and miR-205 were down-regulated. The numbers of BE samples investigated in these studies were relatively low, which might explain their lack of coherence and inability to identify a unique miRNA signature for BE. These findings could also indicate that dysregulation of the miRNAs occurs along the entire continuum of
Table 1. Common miRNA Expression Profiles in Barrett's Esophagus and Esophageal Cancers

| BE | EAC | ESCC |
| --- | --- | --- |
| **Up-regulated miRNAs** | **Down-regulated miRNAs** | **Up-regulated miRNAs** | **Down-regulated miRNAs** |
| miR-21• | let-7c• | miR-21• | let-7c• | miR-21•• | let-7c |
| miR-143–145 cluster | miR-203• | miR-28 | miR-23a | miR-155 | miR-1 |
| miR-192 | miR-205• | miR-30a-5p | miR-27a | miR-99a | miR-100 |
| miR-194 | miR-143–145 cluster | miR-27b | miR-100 | miR-133a | miR-143–145 cluster |
| miR-215 | miR-192 | miR-31 | miR-125b | miR-203 | miR-205• |

---

**NOTE.** Bold-faced: miRNAs that are consistently dysregulated in both BE and EAC. References: BE and EAC ([57], [60], [62–64]); ESCC ([57]–[61]).

* Stepwise-up-regulated miRNAs in the progression from normal to BE to EAC.
* miRNAs also found in blood (dysregulated in same direction).

---

The miRNAs listed in Table 2 might be used to identify subtypes of gastric tumors. However, the reported levels of several of these miRNAs are not consistent among studies. This inconsistency might have resulted from inclusion of different gastric tumor subtypes (intestinal or diffuse). Several researchers focused on only one subtype, but most investigators studied both types as a combined group. Some studies reported distinct profiles for each histologic type, whereas others showed no relationship between miRNA expression profile and histopathologic subtype.

Profiling studies have therefore shown that miRNA levels are dysregulated among gastroesophageal tumor types, each with a distinct expression profile. Alterations

Table 2. Common miRNA Expression Profile in Gastric Cancer

| Gastric cancer |
| --- |
| **Up-regulated miRNAs** | **Down-regulated miRNAs** |
| miR-7 | miR-135b | miR-29c |
| miR-10a | miR-146a | miR-30a-5p |
| miR-15a-16-1 cluster | miR-192 | miR-148a |
| miR-15b-16-2 cluster | miR-194 | miR-375 |
| miR-17–92 cluster•• | miR-200b | miR-638 |
| miR-21•• | miR-214 |
| miR-23a | miR-215 |
| miR-27a•• | miR-222-221 cluster |
| miR-29b | miR-223 |
| miR-34a•• | miR-224 |
| miR-106a-363 cluster••Δ | miR-425-5p |
| miR-106b-25 cluster•• | miR-429 |
| miR-103 |
| miR-107 |
| miR-125b |

Refs: [43], [50], [54], [65]–[67].

Δ miR-363 was not up-regulated.

•• miRNAs (in the case of clusters, some members of each cluster only) were also found in blood.

in expression of miRNAs differ for each organ and histologic type and might be used to identify pathways of tumor progression and specific therapeutic targets or prognostic factors. miRNA expression profiling might be used to identify gene products involved in gastroesophageal carcinogenesis by identifying the targets of the altered miRNAs with databases such as TargetScan. Many of the miRNAs identified from profiling studies and their target genes have been evaluated in functional studies (Supplementary Tables 1 and 2). Further functional studies on these miRNAs to determine their interactions with oncogene and tumor suppressor mRNAs are needed to improve our understanding of gastroesophageal tumorigenesis.

### miRNA Involvement in Tumor Formation and Progression

Many studies have investigated altered expression levels and biological functions of miRNAs in gastric cancer ${ }^{41-43,45,47,49-51,54,66,68-104}$ and esophageal cancer ${ }^{53,59,61,105-115}$ (Supplementary Tables 1 and 2).

#### Cell Cycle

Progression through the cell cycle requires coordinated expression of cyclins, resulting in sequential activation of cyclin-dependent kinases (CDKs) and their modulators. CDKs are regulated by CDK inhibitors (CKIs), which can be organized into members of the p16 family (eg, p15, p16, p18, and p19) or the p21 family (eg, p21, p27, p28, and p57). miRNA dysregulation promotes progression of the gastroesophageal cancer cell cycle by altering levels or translation of mRNAs that encode CKIs (Figure 1). For example, the miR-106b-25 cluster is upregulated in gastric cancer; these miRNAs target the mRNAs that encode p21 and the oncogenic transcription factor E2F1. ${ }^{41}$ E2F1 is a downstream effector of the Rb pathway and a regulator of the cell cycle that promotes the G1-S transition. ${ }^{116}$ E2F1 up-regulates expression of the miR-106b-25 cluster in gastric cancer cells; the overexpressed miR-106b and miR-93 target the $E2F1$ mRNA in a negative feedback loop. These miRNAs also target the $p21$ mRNA. p21 is a downstream effector of transforming growth factor $\beta$, which is a tumor suppressor, so downregulation of p21 disrupts this pathway.

In addition to the miR-106b-25 cluster, members of the miR-222-221 cluster also target mRNAs that encode the p21 family of CKIs. ${ }^{66}$ The miR-106b-25 and miR-222-221 clusters are up-regulated in gastric cancer cells and alter the cell cycle by targeting mRNAs encoding p21 and p57 (miR-106b-25) or p27 and p57 (miR-222-221). Interestingly, the miR-106b-25 cluster also targets $p21$ mRNA in EAC cells, ${ }^{53}$ so this particular CKI is inhibited during carcinogenesis in different organs. Expression of these 2 miRNA clusters was clearly altered in profiling studies of gastric tumors (Table 2), establishing the strength of this approach and indicating the need to functionally validate additional miRNAs identified by profiling studies. Other miRNAs, such as miR-148a and miR-331-3p, also seem to target the same mRNAs as the miR-106b-25 and miR-222-221 clusters, respectively suppressing $p27$ and $E2F1$ mRNAs in gastric cancer cells. ${ }^{75,76}$

Furthermore, although miR-146a does not directly target $p21$ mRNA, it inhibits p21 expression by targeting SMAD4 mRNA. SMAD4 is a downstream factor in the transforming growth factor $\beta$ signaling pathway that induces the expression of p21. ${ }^{117}$ miR-146a is up-regulated in gastric cancer cells, and its inhibition of SMAD4 increases proliferation. ${ }^{97}$

In addition to CKIs, production of cyclins is also dysregulated during tumorigenesis. Cyclins D and E are G1 cyclins that bind to CDK4/6 and CDK2, respectively, and

---

**Figure 1. Influences of miRNAs on the cell cycle in gastroesophageal (GE) cancers. miRNAs that are up-regulated in tumors inhibit cell cycle-inhibitory genes, such as SMAD4, CDX2, p21, p57, and p27. Conversely, miRNAs that are downregulated in tumors inhibit cell cycle-stimulatory genes, such as c-myc, cyclin E, E2F1, ΔNp63, and FGFRL1. miRNAs that are dysregulated in gastric cancer (GC) only are indicated in blue boxes; those involved in esophageal cancer (EC) only are contained in pink boxes; and those implicated in both (all GE cancers) are in the purple box.**

![Diagram](#)

- **miR-9**
- **miR-146a**
- **miR-106b**
- **miR-93**
- **miR-222/221**
- **miR-148a**
- **SMAD4**
- **p21**
- **p57**
- **p27**
- **CDX2**
- **c-myc**
- **Cyclin D**
- **Cyclin E**
- **E2F1**
- **ΔNp63**
- **FGFRL1**
- **miR-429**
- **miR-29c**
- **miR-331-3p**
- **miR-203**
- **miR-210**

**Uprregulated miRNAs in GE cancers**
**Downregulated miRNAs in GE cancers**

40 SONG AND MELTZER

GASTROENTEROLOGY Vol. 143, No. 1

miR-222/221
miR-150
miR-25
miR-130b
GC only
Both

14-3-3ζ
Bad
Bak
BNIP3L
Bim
RUNX3
Bid/Bax

Apoptosis

Upregulated miRNAs
in GE cancers
Downregulated miRNAs
in GE cancers

Akt
Mcl-1
Bcl-2
PDK1
PTEN
miR-512-5p
miR-15b/16
miR-34
miR-181b
PI3K
EGFR
miR-125-5p
miR-146a
MIF
miR-451
miR-375

Figure 2. Influences of miRNAs on apoptosis in gastroesophageal (GE) cancers. miRNAs that are up-regulated in tumors inhibit pro-apoptotic genes, such as Bim, EGR2, and RUNX3. On the other hand, miRNAs that are downregulated in tumors inhibit anti-apoptotic genes, such as Mcl-1, Bcl-2, MIF, 14-3-3ζ and components of the EGFR-PI3K-Akt signaling pathway. miRNAs that are dysregulated in gastric cancer (GC) only are indicated in blue boxes; and those implicated in both (all GE cancers) are in the purple box.

limit entry into S phase. ESCC tissues have increased levels of cyclin E; induced overexpression of miR-29c in ESCC cell lines reduces levels of cyclin E, causing G₁-G₀ cell cycle arrest.¹⁰⁸ In gastric cancer cells, up-regulated miR-9 targets the mRNA that encodes CDX2, a factor that inhibits cyclin D transcription and activates p21.¹⁰⁷,¹¹⁸,¹¹⁹ Expression of the cyclin D gene is regulated by tumor suppressors and oncogene products; the oncogene c-myc, which activates proliferation, induces expression of cyclin D.¹²⁰ miR-429, which is down-regulated in gastric cancer cells, directly targets c-myc mRNA to reduce cell proliferation.³⁰

Other cell cycle-related modulators that are dysregulated in gastroesophageal tumors include p63, a homologue of p53. The isoform ΔNp63 is highly overexpressed in ESCC cells.¹¹⁴ miR-203, which is down-regulated in ESCC, targets the ΔNp63 mRNA to inhibit cell proliferation.¹¹⁴ One study reported that p63 also inhibits miR-34 family members (miR-34a and miR-34c), which are themselves p53-mediated regulators of cyclin D1 and Cdk4.¹²¹ miR-34 family members are activated by p53 to mediate cell cycle arrest and apoptosis; conversely, p63 represses transcription of miR-34 family members and induces cell proliferation by controlling cell cycle progression.¹²¹ Validating its role in the cell cycle, ΔNp63 delays mitotic exit of epithelial cells.¹²² Similarly, miR-210 is down-regulated in ESCC cells and targets fibroblast growth factor receptor-like 1 mRNA; down-regulation of this factor causes G₁/G₀ phase arrest.¹¹³

Aberrant expression of miRNAs in gastroesophageal tumors can therefore promote cell cycle progression by regulating CKIs and other cell cycle components.

### Apoptosis

Cancer cells resist apoptotic signals; this allows them to survive transformation and exposure to chemotherapeutic agents. Members of the Bcl-2 family regulate apoptosis; they are either proapoptotic (Bax, Bak, Bok, Bim, Bid, Bad, Bmf, Bik, BNIP3L, Noxa, Puma, and Hark) or antiapoptotic (Bcl-2, Bcl-xL, Bcl-w, Mcl-1, and Al/Bfl-1).¹²³ miRNA dysregulation affects apoptosis of gastroesophageal cancer cells by altering expression of Bcl-2 family members (Figure 2). Bim (also known as BCL2L11) mRNA is directly targeted and inhibited by miR-25, which is up-regulated in EAC and gastric cancer cells.⁴¹,⁵³ Bcl-2 mRNA is targeted by miR-15b/miR-16, miR-34, and miR-181b, which reduces proliferation, increases apoptosis, and increases sensitivity of multidrug-resistant gastric cancer cells to chemotherapeutics.⁷⁸,⁹⁶,¹⁰⁴ Mcl-1 mRNA is directly targeted by miR-512-5p; overexpression of this miRNA increases apoptosis of gastric cancer cells.⁵¹

Altered expression of miRNAs can affect regulators of the Bcl-2 family in gastric cancer cells. Overexpression of the transcription factor EGR2 induces p53-mediated apoptosis¹²⁴ and also activates the proapoptotic proteins BNIP3L and Bak.¹²⁵ EGR2 mRNA is targeted by miR-150, which promotes gastric cell proliferation.⁹⁵ Similarly, macrophage migration inhibitory factor (MIF) is a lymphokine that inhibits Bid- and Bax-mediated apoptosis.¹²⁶ MIF mRNA is targeted by miR-451, which is down-regulated in gastric cancer cells.⁶⁹ RUNX3 activates the proapoptotic protein Bim;¹²⁷ RUNX3 mRNA is targeted by miR-130b, which is up-regulated in gastric cancer cells.⁸¹

miRNAs also regulate cell survival signaling pathways. Epidermal growth factor receptor (EGFR) signaling via

phosphatidylinositol 3-kinase (PI3K) and Akt leads to phosphorylation and inhibition of proapoptotic Bcl-2 family members to promote cell survival. ${ }^{123}$ miR-125-5p and miR-146a directly target EGFR mRNA (miR-125-5p targets ERBB2 mRNA, a member of the EGFR family) to induce cell death; levels of both of these miRNAs are down-regulated in gastric cancer cells, leading to constitutive EGFR signaling and cell survival. ${ }^{80,85}$

Conversely, PTEN is a negative regulator of PI3K-Akt signaling. The miR-221-221 cluster promotes proliferation of gastric cancer cells by directly targeting PTEN mRNA. ${ }^{71}$ Gastric cancer cells also down-regulate miR-375, which targets the mRNA that encodes PDK1, a kinase that phosphorylates and activates Akt. ${ }^{50}$ In addition to inhibiting PI3K-Akt signaling, miR-375 also targets the mRNA that encodes 14-3-3ζ, an inhibitor of proapoptotic proteins such as Bad, and JAK2, a regulator of cell proliferation and survival. miR-375 therefore has multiple pro-apoptotic effects. ${ }^{50,73}$

miRNAs also affect activity of the transcription factor nuclear factor κB, which controls cell proliferation and apoptosis. The cytokine tumor necrosis factor α activates nuclear factor κB, which can lead to apoptosis. ${ }^{128}$ Levels of tumor necrosis factor α are reduced in ESCC cells ${ }^{129}$; miR-19a directly targets TNF-α mRNA and is a member of the miR-17-92 cluster, which is up-regulated in ESCC cells. ${ }^{111}$ In other cells, miR-9 targets NF-κB1 mRNA to inhibit proliferation; levels of miR-9 are reduced in gastric cancer cells. ${ }^{93}$

Aberrant expression of miRNAs by gastroesophageal cancer cells can reduce proapoptotic signaling by altering levels of Bcl-2 family proteins and through other pathways.

### Invasion and Metastasis

To metastasize, cancer cells must become motile and invasive to break through the extracellular matrix that surrounds the tumor and travel to other organs. Dysregulation of miRNAs can promote invasive activities of gastroesophageal cancer cells and metastasis of tumors (Figure 3). Primary tumor cells undergo an epithelial-to-mesenchymal transition (EMT) to acquire invasive and metastatic properties. EMT is characterized by a loss of cell polarity and the acquisition of a mesenchymal phenotype, characterized by down-regulation of E-cadherin. ${ }^{130}$

E-cadherin is a cell adhesion molecule in which expression is altered during carcinogenesis. In ESCC cells, the mRNA that encodes E-cadherin, CDH1, is directly targeted by miR-92a. Restoring expression of CDH1 to these cells inhibits their invasive abilities and migration. ${ }^{107}$ The ZEB family of transcription factors also regulates the EMT. ZEB1 and ZEB2 inhibit E-cadherin expression and induce EMT in various tumors. ${ }^{130}$ Members of the miR-200 family that target ZEB1 and ZEB2 mRNAs are markedly down-regulated during the EMT. ${ }^{131-133}$

In addition to cadherin and the factors that regulate it, levels of several extracellular matrix remodeling proteins and cell adhesion molecules are controlled by miRNAs. The matrix metalloproteinases (MMPs), which mediate remodeling of the extracellular matrix, mediate the EMT. The mRNA that encodes RECK, a purported tumor suppressor that inhibits EMT by modulating MMPs, ${ }^{134}$ is a direct target of miR-21 in gastric cancer cells. ${ }^{101}$ Levels of miR-21 are up-regulated in gastric cancer cells; their inhibition of RECK expression leads reduces cell invasion and migration. ${ }^{101}$

Bcl-w promotes the invasive activities of gastric cancer cells by activating MMP-2 expression and PI3K signaling to Akt and Sp1. ${ }^{135}$ miR-335, which is down-regulated in gastric cancer cells, directly targets Bcl-w and Sp1 mRNAs to prevent cell invasion and metastasis. ${ }^{98}$ Similarly, EPB41L3 encodes an actin-binding, 4.1B protein that sup-

**Figure 3. Influences of miRNAs on invasion and metastasis in gastroesophageal (GE) cancers.** miRNAs that are up-regulated in tumors inhibit invasion/metastasis-inhibitory genes, such as E-cadherin, ALCAM, EPB41L3, RECK, and KLF4. In opposition, miRNAs that are down-regulated in tumors inhibit invasion/metastasis-promoting genes, such as ZEB1/ZEB2, MYH9, and components of the Bcl-w-PI3K-Akt-SP1 signaling pathway. miRNAs that are dysregulated in gastric cancer (GC) only are indicated in blue boxes; and those involved in esophageal cancer (EC) only are contained in pink boxes.

presses metastasis by tethering the actin cytoskeleton to membrane proteins.¹³⁶ miR-223, which targets EPB41L3 mRNA, is overexpressed in metastatic gastric cancer cells, promoting their migration and invasion.⁴³ ALCAM is an adhesion molecule that inhibits cell migration and is down-regulated in gastric cells. Gastric cancer cells express increased levels of miR-192 and miR-215, which directly target ALCAM mRNA to reduce its level and promote cell migration.⁷⁹

Proteins that promote cell invasion and tumor metastasis by unclear mechanisms have also been reported as targets of miRNAs. MYH9 encodes the nonmuscle myosin heavy chain IIA, and its roles in tumor cell invasion and metastasis have been investigated.¹³⁷,¹³⁸ MYH9 mRNA is targeted by the miRNA let-7f, in which expression is down-regulated in gastric cancer cells. Overexpression of let-7f suppresses the invasive and metastatic abilities of gastric cancer cells.⁸³ Increased expression of miR-10b promotes migration and invasion of ESCC cells. This miRNA targets the mRNA that encodes the tumor suppressor KLF4.¹¹² KLF4 inhibits migration and invasion of ESCC cells in vitro and pancreatic tumor metastasis in mice.¹³⁹,¹⁴⁰

## miRNAs as Biomarkers

### In Diagnosis

Methods to detect early-stage gastroesophageal tumors could increase survival, because patients could be treated before they progress to late-stage disease. Endoscopy and microscopic examination of tissues are used to diagnose patients with gastroesophageal cancers, but less invasive methods are needed. Methods to detect tumor-derived circulating miRNAs might be used to noninvasively identify patients with gastroesophageal cancers.

miRNAs are stable in blood and exist as free miRNAs, exosomal miRNAs, or most predominantly as Argonaute2 (Ago2) protein-bound miRNAs.¹⁴¹–¹⁴³ The different forms of miRNA in blood might originate from different cell types, so patterns of miRNAs in blood samples might reflect cell type-specific miRNA expression. Studies are needed to determine whether the extracellular form of circulating miRNAs affects their use as diagnostics. If so, more stringent purification strategies might have to be developed.

Limitations of using circulating miRNAs as diagnostic markers include their low abundance in blood, difficulties in measuring quality and quantity, their lack of correlations among various profiling platforms (microarray vs quantitative polymerase chain reaction-based array vs deep sequencing), normalization issues (lack of suitable extracellular housekeeping genes), and unclear mechanisms of release and origin. The exact source of circulating miRNAs is a matter of debate, although most researchers believe they are produced from diseased tissues, which leak or actively transport them into the bloodstream. This hypothesis appears to be supported by studies of patients with gastroesophageal cancer. Most of the

miRNAs detected in serum or plasma of patients with cancer increased or decreased, compared with normal samples, as they did in tissue samples (Tables 1 and 2). However, there are contradictions between levels of miRNAs in serum and plasma, and among different tissues, so many more studies of circulating miRNA are needed.

Few systematic miRNA profiling studies have been performed using blood samples from patients with gastroesophageal cancer. Liu et al determined levels of miRNAs in pooled serum samples from patients with gastric cancer, compared with samples from healthy individuals, using Solexa sequencing.¹⁴⁴ They found that up-regulation of miR-1, miR-20a, miR-27a, miR-34a, and miR-423-5p could identify patients with gastric cancer with 80% sensitivity and 81% specificity. Zhang et al used Solexa sequencing to evaluate pooled serum samples from patients with advanced ESCC and healthy individuals. They identified a panel of 7 serum miRNAs (miR-10a, miR-22, miR-100, miR-127-3p, miR-133a, miR-148b, and miR-223) that was associated with ESCC.¹⁴⁵ A microarray analysis of serum miRNAs reported increased levels of miR-187, miR-371-5p, and miR-378 in patients with advanced-stage gastric cancer compared with healthy individuals; miR-378 alone could identify the patients with cancer with 87.5% sensitivity and 70.73% specificity.¹⁴⁶

Most studies have focused on individual miRNAs that target oncogenic or tumor suppressor mRNAs, measuring their expression levels in plasma or serum samples using real-time, reverse transcriptase polymerase chain reaction assays. miR-17-5p, miR-21, miR-106a, and miR-106b were up-regulated and let-7a was down-regulated in plasma samples from patients with advanced-stage gastric cancer.¹⁴⁷ The area under the receiver-operating characteristic curve (AUC) for diagnosis of this cancer was 0.879 for the ratio of miR-106a to let-7a.¹⁴⁷ miR-31 was up-regulated in serum and tissue samples from patients with ESCC (AUC, 0.902),¹³⁵ whereas the ratio of plasma miR-21/miR-375 had an AUC of 0.816 for identification of patients with ESCC.¹⁴⁸

However, there are not yet enough data available on circulating levels of miRNA in serum or plasma for miRNA profiles to be used as diagnostics for gastroesophageal cancers.

### As Prognostic Factors

It is a challenge to predict survival times, rate of disease progression, or outcomes of patients with gastroesophageal cancers or their response to treatment. miRNAs might be used as prognostic factors because of their stable and robust levels in tissues and the circulation. However, studies of miRNAs as prognostic factors have involved small sample sets; validation in larger, independent cohorts is required before assays for miRNAs can be used in the clinic.

Levels of miRNAs have been associated with survival times of patients with esophageal cancer, as well as tumor metastasis to lymph nodes and disease recurrence. In patients with ESCC, increased levels of miR-21,¹⁴⁹ miR-

July 2012

MICRORNAS AND TREATMENT OF GI CANCERS 43

92c, ${ }^{107}$ miR-103/107, ${ }^{58}$ miR-200c, ${ }^{149}$ and miR-296 ${ }^{61}$ were associated with shorter survival time. Conversely, reduced levels of miR-145 were also associated with shorter survival time. ${ }^{149}$ Up-regulation of miR-21 ${ }^{109,150}$ and miR-92a ${ }^{107}$ correlated with metastasis of tumors to lymph nodes, and reduced expression of miR-106a and miR-148a was associated with recurrence of ESCC after surgery. ${ }^{150}$ Up-regulation of an miRNA signature comprising miR-99a, miR-100, miR-143, miR-199a-3p, and miR-199a-5p was associated with reduced survival times of patients with EAC; up-regulation of miR-99b, miR-199a-3p, and miR-199a-5p is also associated with development of lymph node metastases. ${ }^{151}$

miRNAs have been associated with survival times and disease stage of patients with gastric cancer, as well as tumor recurrence and metastasis to the lymph nodes. High levels of miR-20b, ${ }^{65}$ miR-142-5p, ${ }^{152}$ miR-150, ${ }^{65}$ and miR-375 ${ }^{152}$ and decreased levels of miR-125a-5p, ${ }^{85}$ miR-146a, ${ }^{80}$ and miR-451 ${ }^{69}$ were associated with shorter survival times. Overexpression of miR-27a ${ }^{65}$ and miR-650 ${ }^{100}$ and reduced levels of miR-126, ${ }^{74}$ miR-146a, ${ }^{80}$ miR-148a, ${ }^{102}$ miR-218, ${ }^{91}$ miR-335, ${ }^{98}$ and miR-429 ${ }^{90}$ were associated with metastasis of gastric tumors to the lymph nodes. Advanced-stage gastric tumors had reduced levels of miR-126, ${ }^{74}$ miR-148a, ${ }^{102}$ and miR-218. ${ }^{91}$ Tumors with increased levels of miR-375 and reduced levels of miR-142-5p were more likely to recur. ${ }^{152}$

Because miRNAs regulate almost every biologic function, they are also likely to affect a patient’s response to drugs. Many tumors are resistant to chemotherapy; several miRNAs appear to predict or affect the response to chemotherapy. Expression of miR-200c correlates with response to chemotherapy, ${ }^{149}$ whereas down-regulation of miR-296 increases the sensitivity of ESCC cells to P-glycoprotein-related and nonrelated drugs. ${ }^{61}$ Up-regulation of miR-148a significantly increases sensitivity of ESCC and EAC cells to cisplatin and 5-fluorouracil. ${ }^{153}$ Similarly, overexpression of miR-15b or miR-16 increases the sensitivity of gastric cancer cells to vincristine. ${ }^{96}$

miRNAs might therefore be developed as therapeutics or reagents that increase chemosensitivity of tumor cells. They could be administered to patients, but the challenge would be to achieve organ- and tissue-specific delivery, with low levels of off-target toxicity. Delivery methods have not progressed to the point of clinical utility for gastroesophageal cancers.

## Future Directions

miRNAs are noncoding RNAs that regulate many cellular processes and are involved in the pathogenesis of many diseases, including gastroesophageal cancer. However, although we are beginning to understand the mechanisms of miRNA biogenesis and function, many gaps remain to be filled. Many studies have reported dysregulated expression of miRNAs in tumors and the effects on their mRNA targets and downstream factors. Nevertheless, the mechanisms of miRNA dysregulation are not clear. We have described some of the mechanisms discovered in gastroesophageal cancer cells, but further studies are needed. Although there have been many studies on specific miRNAs, there are many more we know nothing about (particularly of those numbered miR-1000 and higher), and more miRNAs are still being isolated. Studies are needed to identify targets and functions of miRNA as they are identified.

Each type of gastroesophageal tumor appears to have a unique miRNA expression profile. Functional analyses have shown that these miRNAs interact with mRNAs from oncogenes and tumor suppressor genes to control the cell cycle, apoptosis, and invasive and metastatic abilities of cells. Altered expression of miRNAs in gastroesophageal tumors promotes cell cycle progression via direct and indirect regulation of CKIs or other cell cycle controllers, reduces proapoptotic signaling through regulation of Bcl-2 family proteins or other apoptotic regulators, and promotes cell migration and invasion by regulating cadherin-related pathways or other adhesion molecules.

Many miRNAs could be used in diagnosis of cancer, in determining patient prognosis, or as therapeutic targets. We need to develop more stringent protocols for collecting and analyzing samples (to avoid variations in sample processing or histologic characterization) and to validate each finding in large independent cohorts. This approach will lead to selection of the best candidate miRNAs for further study, production of highly reproducible results, and reductions in inter-study discrepancies. New miRNAs are continuously being discovered and profiling technologies are rapidly changing, so setting up a standardization process for integrating data will be challenging. Development of effective in vivo delivery systems is also required if miRNAs are to be eventually used as therapeutics.

Although many additional studies need to be performed, miRNAs appear to have many useful clinical applications for patients with gastroesophageal cancers and other gastrointestinal and liver disorders. The GI community should look forward to these studies with great anticipation.

## Supplementary Material

Note: To access the supplementary material accompanying this article, visit the online version of Gastroenterology at www.gastrojournal.org, and at http://dx.doi.org/10.1053/j.gastro.2012.05.003.

## References

1. Parkin DM. Global cancer statistics in the year 2000. Lancet Oncol 2001;2:533–543.
2. Ferlay J, Shin HR, Bray F, et al. Estimates of worldwide burden of cancer in 2008: GLOBOCAN 2008. Int J Cancer 2010;127: 2893–2917.
3. Jemal A, Siegel R, Xu J, et al. Cancer statistics, 2010. CA Cancer J Clin 2010;60:277–300.
4. Siewert JR, Ott K. Are squamous and adenocarcinomas of the esophagus the same disease? Semin Radiat Oncol 2007;17: 38–44.

5. Mariette C, Finzi L, Piessen G, et al. Esophageal carcinoma: prognostic differences between squamous cell carcinoma and adenocarcinoma. World J Surg 2005; 29:39–45.
6. Enzinger PC, Mayer RJ. Esophageal cancer. N Engl J Med 2003; 349:2241–2252.
7. Corley DA, Levin TR, Habel LA, et al. Surveillance and survival in Barrett’s adenocarcinomas: a population-based study. Gastroenterology 2002; 122:633–640.
8. Cameron AJ. Management of Barrett’s esophagus. Mayo Clin Proc 1998; 73:457–461.
9. van Soest EM, Dieleman JP, Siersema PD, et al. Increasing incidence of Barrett’s oesophagus in the general population. Gut 2005; 54:1062–1066.
10. Sharma P, Falk GW, Weston AP, et al. Dysplasia and cancer in a large multicenter cohort of patients with Barrett’s esophagus. Clin Gastroenterol Hepatol 2006; 4:566–572.
11. Houghton J, Wang TC. Helicobacter pylori and gastric cancer: a new paradigm for inflammation-associated epithelial cancers. Gastroenterology 2005; 128:1567–1578.
12. Wang TC, Dangler CA, Chen D, et al. Synergistic interaction between hypergastrinemia and Helicobacter infection in a mouse model of gastric cancer. Gastroenterology 2000; 118:36–47.
13. El-Omar EM, Rabkin CS, Gammon MD, et al. Increased risk of noncardia gastric cancer associated with proinflammatory cytokine gene polymorphisms. Gastroenterology 2003; 124:1193–1201.
14. Zambon CF, Basso D, Navaglia F, et al. Increased risk of noncardia gastric cancer associated with proinflammatory cytokine gene polymorphisms. Gastroenterology 2004; 126:382–384.
15. Yamaguchi N, Kakizoe T. Synergistic interaction between Helicobacter pylori gastritis and diet in gastric cancer. Lancet Oncol 2001; 2:88–94.
16. Lee RC, Feinbaum RL, Ambros V. The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell 1993; 75:843–854.
17. Pasquinelli AE, Reinhart BJ, Slack F, et al. Conservation of the sequence and temporal expression of let-7 heterochronic regulatory RNA. Nature 2000; 408:86–89.
18. Carthew RW, Sontheimer EJ. Origins and mechanisms of miRNAs and siRNAs. Cell 2009; 136:642–655.
19. Friedman RC, Farh KK, Burge CB, et al. Most mammalian mRNAs are conserved targets of microRNAs. Genome Res 2009; 19:92–105.
20. Kim YK, Kim VN. Processing of intronic microRNAs. EMBO J 2007; 26:775–783.
21. Berezikov E. Evolution of microRNA diversity and regulation in animals. Nat Rev Genet 2011; 12:846–860.
22. Altuvia Y, Landgraf P, Lithwick G, et al. Clustering and conservation patterns of human microRNAs. Nucleic Acids Res 2005; 33: 2697–2706.
23. Lee Y, Kim M, Han J, et al. MicroRNA genes are transcribed by RNA polymerase II. EMBO J 2004; 23:4051–4060.
24. Borchert GM, Lanier W, Davidson BL. RNA polymerase III transcribes human microRNAs. Nat Struct Mol Biol 2006; 13:1097–1101.
25. Schwarz DS, Hutvagner G, Du T, et al. Asymmetry in the assembly of the RISC enzyme complex. Cell 2003; 115:199–208.
26. Lytle JR, Yario TA, Steitz JA. Target mRNAs are repressed as efficiently by microRNA-binding sites in the 5′ UTR as in the 3′ UTR. Proc Natl Acad Sci USA 2007; 104:9667–9672.
27. Tay Y, Zhang J, Thomson AM, et al. MicroRNAs to Nanog, Oct4 and Sox2 coding regions modulate embryonic stem cell differentiation. Nature 2008; 455:1124–1128.
28. Forman JJ, Legesse-Miller A, Coller HA. A search for conserved sequences in coding regions reveals that the let-7 microRNA targets Dicer within its coding sequence. Proc Natl Acad Sci USA 2008; 105:14879–14884.
29. Behm-Ansmant I, Rehwinkel J, Doerks T, et al. mRNA degradation by miRNAs and GW182 requires both CCR4:NOT deadenylase

and DCP1:DCP2 decapping complexes. Genes Dev 2006; 20:1885–1898.
30. Giraldez AJ, Mishima Y, Rihel J, et al. Zebrafish MiR-430 promotes deadenylation and clearance of maternal mRNAs. Science 2006; 312:75–79.
31. Wu L, Fan J, Belasco JG. MicroRNAs direct rapid deadenylation of mRNA. Proc Natl Acad Sci USA 2006; 103:4034–4039.
32. Baek D, Villén J, Shin C, et al. The impact of microRNAs on protein output. Nature 2008; 455:64–71.
33. Guo H, Ingolia NT, Weissman JS, et al. Mammalian microRNAs predominantly act to decrease target mRNA levels. Nature 2010; 466:835–840.
34. Thomson JM, Newman M, Parker JS, et al. Extensive post-transcriptional regulation of microRNAs and its implications for cancer. Genes Dev 2006; 20:2202–2207.
35. Merritt WM, Lin YG, Han LY, et al. Dicer, Drosha, and outcomes in patients with ovarian cancer. N Engl J Med 2008; 359:2641–2650.
36. Kozaki K, Imoto I, Mogi S, et al. Exploration of tumor-suppressive microRNAs silenced by DNA hypermethylation in oral cancer. Cancer Res 2008; 68:2094–2105.
37. Lujambio A, Ropero S, Ballestar E, et al. Genetic unmasking of an epigenetically silenced microRNA in human cancer cells. Cancer Res 2007; 67:1424–1429.
38. Varambally S, Cao Q, Mani RS, et al. Genomic loss of microRNA-101 leads to overexpression of histone methyltransferase EZH2 in cancer. Science 2008; 322:1695–1699.
39. Calin GA, Sevignani C, Dumitru CD, et al. Human microRNA genes are frequently located at fragile sites and genomic regions involved in cancers. Proc Natl Acad Sci USA 2004; 101:2999–3004.
40. Tchernitsa O, Kasajima A, Schafer R, et al. Systematic evaluation of the miRNA-ome and its downstream effects on mRNA expression identifies gastric cancer progression. J Pathol 2010; 222: 310–319.
41. Petrocca F, Visone R, Onelli MR, et al. E2F1-regulated microRNAs impair TGFbeta-dependent cell-cycle arrest and apoptosis in gastric cancer. Cancer Cell 2008; 13:272–286.
42. Ahn SM, Cha JY, Kim J, et al. Smad3 regulates E-cadherin via miRNA-200 pathway. Oncogene 2011 Oct 24 [Epub ahead of print].
43. Li X, Zhang Y, Zhang H, et al. miRNA-223 promotes gastric cancer invasion and metastasis by targeting tumor suppressor EPB41L3. Mol Cancer Res 2011; 9:824–833.
44. Jones PA, Baylin SB. The fundamental role of epigenetic events in cancer. Nat Rev Genet 2002; 3:415–428.
45. Kim K, Lee HC, Park JL, et al. Epigenetic regulation of microRNA-10b and targeting of oncogenic MAPRE1 in gastric cancer. Epigenetics 2011; 6:740–751.
46. Tsai KW, Wu CW, Hu LY, et al. Epigenetic regulation of miR-34b and miR-129 expression in gastric cancer. Int J Cancer 2011; 129:2600–2610.
47. Bao W, Fu HJ, Xie QS, et al. HER2 interacts with CD44 to up-regulate CXCR4 via epigenetic silencing of microRNA-139 in gastric cancer cells. Gastroenterology 2011; 141:2076–2087 e6.
48. Tsai KW, Hu LY, Wu CW, et al. Epigenetic regulation of miR-196b expression in gastric cancer. Genes Chromosomes Cancer 2010; 49:969–980.
49. Hashimoto Y, Akiyama Y, Otsubo T, et al. Involvement of epigenetically silenced microRNA-181c in gastric carcinogenesis. Carcinogenesis 2010; 31:777–784.
50. Tsukamoto Y, Nakada C, Noguchi T, et al. MicroRNA-375 is downregulated in gastric carcinomas and regulates cell survival by targeting PDK1 and 14-3-3-zeta. Cancer Res 2010; 70:2339–2349.
51. Saito Y, Suzuki H, Tsugawa H, et al. Chromatin remodeling at Alu repeats by epigenetic treatment activates silenced microRNA-512-5p with downregulation of Mcl-1 in human gastric cancer cells. Oncogene 2009; 28:2738–2744.

July 2012

52. Chen X, Hu H, Guan X, et al. CpG island methylation status of miRNAs in esophageal squamous cell carcinoma. Int J Cancer 2012;130:1607–1613.
53. Kan T, Sato F, Ito T, et al. The miR-106b-25 polycistron, activated by genomic amplification, functions as an oncogene by suppressing p21 and Bim. Gastroenterology 2009;136:1689–1700.
54. Oh HK, Tan AL, Das K, et al. Genomic loss of miR-486 regulates tumor progression and the OLFM4 antiapoptotic factor in gastric cancer. Clin Cancer Res 2011;17:2657–2667.
55. Lu J, Getz G, Miska EA, et al. MicroRNA expression profiles classify human cancers. Nature 2005;435:834–838.
56. Calin GA, Ferracin M, Cimmino A, et al. A MicroRNA signature associated with prognosis and progression in chronic lymphocytic leukemia. N Engl J Med 2005;353:1793–1801.
57. Feber A, Xi L, Luketich JD, et al. MicroRNA expression profiles of esophageal cancer. J Thorac Cardiovasc Surg 2008;135:255–260; discussion 260.
58. Guo Y, Chen Z, Zhang L, et al. Distinctive microRNA profiles relating to patient survival in esophageal squamous cell carcinoma. Cancer Res 2008;68:26–33.
59. Kano M, Seki N, Kikkawa N, et al. miR-145, miR-133a and miR-133b: Tumor-suppressive miRNAs target FSCN1 in esophageal squamous cell carcinoma. Int J Cancer 2010;127:2804–2814.
60. Mathe EA, Nguyen GH, Bowman ED, et al. MicroRNA expression in squamous cell carcinoma and adenocarcinoma of the esophagus: associations with survival. Clin Cancer Res 2009;15:6192–6200.
61. Hong L, Han Y, Zhang H, et al. The prognostic and chemotherapeutic value of miR-296 in esophageal squamous cell carcinoma. Ann Surg 2010;251:1056–1063.
62. Fassan M, Volinia S, Palatini J, et al. MicroRNA expression profiling in human Barrett's carcinogenesis. Int J Cancer 2011;129:1661–1670.
63. Wijnhoven BP, Hussey DJ, Watson DI, et al. MicroRNA profiling of Barrett's oesophagus and oesophageal adenocarcinoma. Br J Surg 2010;97:853–861.
64. Yang H, Gu J, Wang KK, et al. MicroRNA expression signatures in Barrett's esophagus and esophageal adenocarcinoma. Clin Cancer Res 2009;15:5744–5752.
65. Katada T, Ishiguro H, Kuwabara Y, et al. microRNA expression profile in undifferentiated gastric cancer. Int J Oncol 2009;34:537–542.
66. Kim YK, Yu J, Han TS, et al. Functional links between clustered microRNAs: suppression of cell-cycle inhibitors by microRNA clusters in gastric cancer. Nucleic Acids Res 2009;37:1672–1681.
67. Ueda T, Volinia S, Okumura H, et al. Relation between microRNA expression and progression and prognosis of gastric cancer: a microRNA expression analysis. Lancet Oncol 2010;11:136–146.
68. Ando T, Yoshida T, Enomoto S, et al. DNA methylation of microRNA genes in gastric mucosae of gastric cancer patients: its possible involvement in the formation of epigenetic field defect. Int J Cancer 2009;124:2367–2374.
69. Bandres E, Bitarte N, Arias F, et al. microRNA-451 regulates macrophage migration inhibitory factor production and proliferation of gastrointestinal cancer cells. Clin Cancer Res 2009;15:2281–2290.
70. Bou Kheir T, Futoma-Kazmierczak E, Jacobsen A, et al. miR-449 inhibits cell proliferation and is down-regulated in gastric cancer. Mol Cancer 2011;10:29.
71. Chun-Zhi Z, Lei H, An-Ling Z, et al. MicroRNA-221 and microRNA-222 regulate gastric carcinoma cell proliferation and radioresistance by targeting PTEN. BMC Cancer 2010;10:367.
72. Cui Y, Su WY, Xing J, et al. MiR-29a inhibits cell proliferation and induces cell cycle arrest through the downregulation of p42.3 in human gastric cancer. PLoS One 2011;6:e25872.

73. Ding L, Xu Y, Zhang W, et al. MiR-375 frequently downregulated in gastric cancer inhibits cell proliferation by targeting JAK2. Cell Res 2010;20:784–793.
74. Feng R, Chen X, Yu Y, et al. miR-126 functions as a tumour suppressor in human gastric cancer. Cancer Lett 2010;298:50–63.
75. Guo SL, Peng Z, Yang X, et al. miR-148a promoted cell proliferation by targeting p27 in gastric cancer cells. Int J Biol Sci 2011;7:567–574.
76. Guo X, Guo L, Ji J, et al. miRNA-331-3p directly targets E2F1 and induces growth arrest in human gastric cancer. Biochem Biophys Res Commun 2010;398:1–6.
77. Guo X, Jing C, Li L, et al. Down-regulation of VEZT gene expression in human gastric cancer involves promoter methylation and miR-43c. Biochem Biophys Res Commun 2011;404:622–627.
78. Ji Q, Hao X, Meng Y, et al. Restoration of tumor suppressor miR-34 inhibits human p53-mutant gastric cancer tumorspheres. BMC Cancer 2008;8:266.
79. Jin Z, Selaru FM, Cheng Y, et al. MicroRNA-192 and -215 are upregulated in human gastric cancer in vivo and suppress ALCAM expression in vitro. Oncogene 2011;30:1577–1585.
80. Kogo R, Mimori K, Tanaka F, et al. Clinical significance of miR-146a in gastric cancer cases. Clin Cancer Res 2011;17:4277–4284.
81. Lai KW, Koh KX, Loh M, et al. MicroRNA-130b regulates the tumour suppressor RUNX3 in gastric cancer. Eur J Cancer 2010;46:1456–1463.
82. Li X, Zhang Y, Shi Y, et al. MicroRNA-107, an oncogene microRNA that regulates tumour invasion and metastasis by targeting Dicer1 in gastric cancer. J Cell Mol Med 2011;15:1887–1895.
83. Liang S, He L, Zhao X, et al. MicroRNA let-7f inhibits tumor invasion and metastasis by targeting MYH9 in human gastric cancer. PLoS One 2011;6:e18409.
84. Liu T, Tang H, Lang Y, et al. MicroRNA-27a functions as an oncogene in gastric adenocarcinoma by targeting prohibitin. Cancer Lett 2009;273:233–242.
85. Nishida N, Mimori K, Fabbri M, et al. MicroRNA-125a-5p is an independent prognostic factor in gastric cancer and inhibits the proliferation of human gastric cancer cells in combination with trastuzumab. Clin Cancer Res 2011;17:2725–2733.
86. Otsubo T, Akiyama Y, Hashimoto Y, et al. MicroRNA-126 inhibits SOX2 expression and contributes to gastric carcinogenesis. PLoS One 2011;6:e16617.
87. Rotkrua P, Akiyama Y, Hashimoto Y, et al. MiR-9 downregulates CDX2 expression in gastric cancer cells. Int J Cancer 2011;129:2611–2620.
88. Shen R, Pan S, Qi S, et al. Epigenetic repression of microRNA-129-2 leads to overexpression of SOX4 in gastric cancer. Biochem Biophys Res Commun 2010;394:1047–1052.
89. Song YX, Yue ZY, Wang ZN, et al. MicroRNA-148b is frequently down-regulated in gastric cancer and acts as a tumor suppressor by inhibiting cell proliferation. Mol Cancer 2011;10:1.
90. Sun T, Wang C, Xing J, et al. miR-429 Modulates the expression of c-myc in human gastric carcinoma cells. Eur J Cancer 2011;47:2552–2559.
91. Tie J, Pan Y, Zhao L, et al. MiR-218 inhibits invasion and metastasis of gastric cancer by targeting the Robo1 receptor. PLoS Genet 2010;6:e1000879.
92. Wada R, Akiyama Y, Hashimoto Y, et al. miR-212 is downregulated and suppresses methyl-CpG-binding protein MeCP2 in human gastric cancer. Int J Cancer 2010;127:1106–1114.
93. Wan HY, Guo LM, Liu T, et al. Regulation of the transcription factor NF-kappaB1 by microRNA-9 in human gastric adenocarcinoma. Mol Cancer 2010;9:16.
94. Wu J, Zhang YC, Suo WH, et al. Induction of anion exchanger-1 translation and its opposite roles in the carcinogenesis of gastric cancer cells and differentiation of K562 cells. Oncogene 2010;29:1987–1996.

95. Wu Q, Jin H, Yang Z, et al. MiR-150 promotes gastric cancer proliferation by negatively regulating the pro-apoptotic gene EGR2. Biochem Biophys Res Commun 2010;392:340–345.
96. Xia L, Zhang D, Du R, et al. miR-15b and miR-16 modulate multidrug resistance by targeting BCL2 in human gastric cancer cells. Int J Cancer 2008;123:372–379.
97. Xiao B, Zhu ED, Li N, et al. Increased miR-146a in gastric cancer directly targets SMAD4 and is involved in modulating cell proliferation and apoptosis. Oncol Rep 2012;27:559–566.
98. Xu Y, Zhao F, Wang Z, et al. MicroRNA-335 acts as a metastasis suppressor in gastric cancer by targeting Bcl-w and specificity protein 1. Oncogene 2012;31:1398–1407.
99. Yang Q, Jie Z, Cao H, et al. Low-level expression of let-7a in gastric cancer and its involvement in tumorigenesis by targeting RAB40C. Carcinogenesis 2011;32:713–722.
100. Zhang X, Zhu W, Zhang J, et al. MicroRNA-650 targets ING4 to promote gastric cancer tumorigenicity. Biochem Biophys Res Commun 2010;395:275–280.
101. Zhang Z, Li Z, Gao C, et al. miR-21 plays a pivotal role in gastric cancer pathogenesis and progression. Lab Invest 2008;88:1358–1366.
102. Zheng B, Liang L, Wang C, et al. MicroRNA-148a suppresses tumor cell invasion and metastasis by downregulating ROCK1 in gastric cancer. Clin Cancer Res 2011;17:7574–7583.
103. Zhu LH, Liu T, Tang H, et al. MicroRNA-23a promotes the growth of gastric adenocarcinoma cell line MGC803 and downregulates interleukin-6 receptor. FEBS J 2010;277:3726–3734.
104. Zhu W, Shan X, Wang T, et al. miR-181b modulates multidrug resistance by targeting BCL2 in human cancer cell lines. Int J Cancer 2010;127:2520–2529.
105. Luthra R, Singh RR, Luthra MG, et al. MicroRNA-196a targets annexin A1: a microRNA-mediated mechanism of annexin A1 downregulation in cancers. Oncogene 2008;27:6667–6678.
106. Maru DM, Singh RR, Hannah C, et al. MicroRNA-196a is a potential marker of progression during Barrett’s metaplasia-dysplasia-invasive adenocarcinoma sequence in esophagus. Am J Pathol 2009;174:1940–1948.
107. Chen ZL, Zhao XH, Wang JW, et al. microRNA-92a promotes lymph node metastasis of human esophageal squamous cell carcinoma via E-cadherin. J Biol Chem 2011;286:10725–10734.
108. Ding DP, Chen ZL, Zhao XH, et al. miR-29c induces cell cycle arrest in esophageal squamous cell carcinoma by modulating cyclin E expression. Carcinogenesis 2011;32:1025–1032.
109. Hiyoshi Y, Kamohara H, Karashima R, et al. MicroRNA-21 regulates the proliferation and invasion in esophageal squamous cell carcinoma. Clin Cancer Res 2009;15:1915–1922.
110. Lee KH, Goan YG, Hsiao M, et al. MicroRNA-373 (miR-373) post-transcriptionally regulates large tumor suppressor, homolog 2 (LATS2) and stimulates proliferation in human esophageal cancer. Exp Cell Res 2009;315:2529–2538.
111. Liu M, Wang Z, Yang S, et al. TNF-alpha is a novel target of miR-19a. Int J Oncol 2011;38:1013–1022.
112. Tian Y, Luo A, Cai Y, et al. MicroRNA-10b promotes migration and invasion through KLF4 in human esophageal cancer cell lines. J Biol Chem 2010;285:7986–7994.
113. Tsuchiya S, Fujiwara T, Sato F, et al. MicroRNA-210 regulates cancer cell proliferation through targeting fibroblast growth factor receptor-like 1 (FGFRL1). J Biol Chem 2011;286:420–428.
114. Yuan Y, Zeng ZY, Liu XH, et al. MicroRNA-203 inhibits cell proliferation by repressing DeltaNp63 expression in human esophageal squamous cell carcinoma. BMC Cancer 2011;11:57.
115. Zhang T, Wang Q, Zhao D, et al. The oncogenic role of microRNA-31 as a potential biomarker in oesophageal squamous cell carcinoma. Clin Sci (Lond) 2011;121:437–447.
116. DeGregori J. The genetics of the E2F family of transcription factors: shared functions and unique roles. Biochim Biophys Acta 2002;1602:131–150.

117. Shi Y, Massague J. Mechanisms of TGF-beta signaling from cell membrane to the nucleus. Cell 2003;113:685–700.
118. Bai YQ, Miyake S, Iwai T, et al. CDX2, a homeobox transcription factor, upregulates transcription of the p21/WAF1/CIP1 gene. Oncogene 2003;22:7942–7949.
119. Takahashi K, Hirano F, Matsumoto K, et al. Homeobox gene CDX2 inhibits human pancreatic cancer cell proliferation by down-regulating cyclin D1 transcriptional activity. Pancreas 2009;38:49–57.
120. Pajic A, Spitkovsky D, Christoph B, et al. Cell cycle activation by c-myc in a burkitt lymphoma model cell line. Int J Cancer 2000;87:787–793.
121. Antonini D, Russo MT, De Rosa L, et al. Transcriptional repression of miR-34 family contributes to p63-mediated cell cycle progression in epidermal cells. J Invest Dermatol 2010;130:1249–1257.
122. Hau PM, Yip YL, Huen MS, et al. Loss of DeltaNp63alpha promotes mitotic exit in epithelial cells. FEBS Lett 2011;585:2720–2726.
123. Brumatti G, Salmanidis M, Ekert PG. Crossing paths: interactions between the cell death machinery and growth factor survival signals. Cell Mol Life Sci 2010;67:1619–1630.
124. Yokota I, Sasaki Y, Kashima L, et al. Identification and characterization of early growth response 2, a zinc-finger transcription factor, as a p53-regulated proapoptotic gene. Int J Oncol 2010;37:1407–1416.
125. Unoki M, Nakamura Y. EGR2 induces apoptosis in various cancer cell lines by direct transactivation of BNIP3L and BAK. Oncogene 2003;22:2172–2185.
126. Baumann R, Casaulta C, Simon D, et al. Macrophage migration inhibitory factor delays apoptosis in neutrophils by inhibiting the mitochondria-dependent death pathway. FASEB J 2003;17:2221–2230.
127. Yano T, Ito K, Fukamachi H, et al. The RUNX3 tumor suppressor upregulates Bim in gastric epithelial cells undergoing transforming growth factor beta-induced apoptosis. Mol Cell Biol 2006;26:4474–4488.
128. Fan Y, Dutta J, Gupta N, et al. Regulation of programmed cell death by NF-kappaB and its role in tumorigenesis and therapy. Adv Exp Med Biol 2008;615:223–250.
129. Oka M, Hirose K, Izuka N, et al. Cytokine mRNA expression patterns in human esophageal cancer cell lines. J Interferon Cytokine Res 1995;15:1005–1009.
130. Vandewalle C, Van Roy F, Berx G. The role of the ZEB family of transcription factors in development and disease. Cell Mol Life Sci 2009;66:773–787.
131. Gregory PA, Bert AG, Paterson EL, et al. The miR-200 family and miR-205 regulate epithelial to mesenchymal transition by targeting ZEB1 and SIP1. Nat Cell Biol 2008;10:593–601.
132. Korpel M, Lee ES, Hu G, et al. The miR-200 family inhibits epithelial-mesenchymal transition and cancer cell migration by direct targeting of E-cadherin transcriptional repressors ZEB1 and ZEB2. J Biol Chem 2008;283:14910–14914.
133. Park SM, Gaur AB, Lengyel E, et al. The miR-200 family determines the epithelial phenotype of cancer cells by targeting the E-cadherin repressors ZEB1 and ZEB2. Genes Dev 2008;22:894–907.
134. Oh J, Takahashi R, Kondo S, et al. The membrane-anchored MMP inhibitor RECK is a key regulator of extracellular matrix integrity and angiogenesis. Cell 2001;107:789–800.
135. Bae IH, Park MJ, Yoon SH, et al. Bcl-w promotes gastric cancer cell invasion by inducing matrix metalloproteinase-2 expression via phosphoinositide 3-kinase, Akt, and Sp1. Cancer Res 2006;66:4991–4995.
136. Cavanna T, Pokorna E, Vesely P, et al. Evidence for protein 4.1B acting as a metastasis suppressor. J Cell Sci 2007;120:606–616.
137. Medjkane S, Perez-Sanchez C, Gaggioli C, et al. Myocardin-related transcription factors and SRF are required for cytoskeletal organization and cell migration. Development 2009;136:1481–1491.

etal dynamics and experimental metastasis. Nat Cell Biol 2009;11:257–268.
138. Ismail TM, Fernig DG, Rudland PS, et al. The basic C-terminal amino acids of calcium-binding protein S100A4 promote metastasis. Carcinogenesis 2008;29:2259–2266.
139. Wei D, Kanai M, Jia Z, et al. Kruppel-like factor 4 induces p27Kip1 expression in and suppresses the growth and metastasis of human pancreatic cancer cells. Cancer Res 2008;68:4631–4639.
140. Yang Y, Goldstein BG, Chao HH, et al. KLF4 and KLF5 regulate proliferation, apoptosis and invasion in esophageal cancer cells. Cancer Biol Ther 2005;4:1216–1221.
141. Mitchell PS, Parkin RK, Kroh EM, et al. Circulating microRNAs as stable blood-based markers for cancer detection. Proc Natl Acad Sci USA 2008;105:10513–10518.
142. Turchinovich A, Weiz L, Langheinz A, et al. Characterization of extracellular circulating microRNA. Nucleic Acids Res 2011;39:7223–7233.
143. Arroyo JD, Chevillet JR, Kroh EM, et al. Argonaute2 complexes carry a population of circulating microRNAs independent of vesicles in human plasma. Proc Natl Acad Sci USA 2011;108:5003–5008.
144. Liu R, Zhang C, Hu Z, et al. A five-microRNA signature identified from genome-wide serum microRNA expression profiling serves as a fingerprint for gastric cancer diagnosis. Eur J Cancer 2011;47:784–791.
145. Zhang C, Wang C, Chen X, et al. Expression profile of microRNAs in serum: a fingerprint for esophageal squamous cell carcinoma. Clin Chem 2010;56:1871–1879.
146. Liu H, Zhu L, Liu B, et al. Genome-wide microRNA profiles identify miR-378 as a serum biomarker for early detection of gastric cancer. Cancer Lett 2012;316:196–203.
147. Tsujiura M, Ichikawa D, Komatsu S, et al. Circulating microRNAs in plasma of patients with gastric cancers. Br J Cancer 2010;102:1174–1179.

148. Komatsu S, Ichikawa D, Takeshita H, et al. Circulating microRNAs in plasma of patients with oesophageal squamous cell carcinoma. Br J Cancer 2011;105:104–111.
149. Hamano R, Miyata H, Yamasaki M, et al. Overexpression of miR-200c induces chemoresistance in esophageal cancers mediated through activation of the Akt signaling pathway. Clin Cancer Res 2011;17:3029–3038.
150. Hummel R, Hussey DJ, Michael MZ, et al. MiRNAs and their association with locoregional staging and survival following surgery for esophageal carcinoma. Ann Surg Oncol 2011;18:253–260.
151. Feber A, Xi L, Pennathur A, et al. MicroRNA prognostic signature for nodal metastases and survival in esophageal adenocarcinoma. Ann Thorac Surg 2011;91:1523–1530.
152. Zhang X, Yan Z, Zhang J, et al. Combination of hsa-miR-375 and hsa-miR-142-5p as a predictor for recurrence risk in gastric cancer patients following surgical resection. Ann Oncol 2011;22:2257–2266.
153. Hummel R, Watson DI, Smith C, et al. Mir-148a improves response to chemotherapy in sensitive and resistant oesophageal adenocarcinoma and squamous cell carcinoma cells. J Gastrointest Surg 2011;15:429–438.

Received January 1, 2012. Accepted May 8, 2012.

Reprint requests

Address requests for reprints to: Jee Hoon Song, BS, Johns Hopkins School of Medicine, 1503 E. Jefferson Street, Room 106, Baltimore, Maryland, 21287. e-mail: jsong37@jhmi.edu; fax: (410) 502-1329.

Conflicts of interest

The authors disclose no conflicts.

| microRNA           | Histologic type | Up or down in cancer | Confirmed targets                     | Phenotypes                                                                 | Ref     |
|--------------------|----------------|--------------------|--------------------------------------|-------------------------------------------------------------------------|---------|
| miR-10b            | ESCC           | UP                 | KLF4                                 | Invasion/Migration                                                      | [112]   |
| miR-17–92 cluster  | ESCC           | UP                 | TNF-α*                               | Apoptosis; Tumorigenicity                                                | [111]   |
| miR-21             | ESCC           | UP                 | PDCD4                                | Cell proliferation; Invasion                                            | [109]   |
| miR-106b-25 cluster| EAC            | UP                 | p21**, Bim***                        | Apoptosis; Cell cycle; Cell proliferation; Tumorigenicity                | [53]    |
| miR-29c            | ESCC           | DOWN               | CyclinE                              | Cell cycle; Cell proliferation; Tumorigenicity                           | [108]   |
| miR-31             | ESCC           | UP                 | EMP1, KSR2, RGS4                     | Clonogenicity; Invasion/Migration                                       | [115]   |
| miR-92a            | ESCC           | UP                 | CDH1                                 | Invasion/Migration                                                      | [107]   |
| miR-133a/b         | ESCC           | DOWN               | FSCN1                                | Cell proliferation; Invasion                                            | [59]    |
| miR-145            | ESCC           | DOWN               | FSCN1                                | Cell proliferation; Invasion                                            | [59]    |
| miR-196a           | EAC            | UP                 | ANXA1, SPRR2C, S100A9, KRT5          | Cell proliferation                                                       | [105], [106] |
| miR-203            | ESCC           | DOWN               | ΔNp63                                | Cell proliferation                                                      | [114]   |
| miR-210            | ESCC           | DOWN               | FGFR1                                | Apoptosis; Cell cycle; Cell proliferation                                | [113]   |
| miR-296            | ESCC           | UP                 | MDR1                                 | Apoptosis; Cell cycle; Cell proliferation; Drug sensitivity; Tumorigenicity | [61]    |
| miR-373            | ESCC           | UP                 | LATS2                                | Cell proliferation                                                      | [110]   |

* *Target of miR-19a from miR-17–92 cluster*
** *Target of miR-93 and miR-106b from miR-106b-25 cluster*
*** *Target of miR-25 from miR-106b-25 cluster*

| microRNA | Up or down in cancer | Confirmed targets | Phenotypes | Ref |
|----------|---------------------|------------------|------------|-----|
| let-7a   | DOWN               | RAB40C           | Cell cycle; Cell proliferation; Tumorigenicity | [99] |
| let-7f   | DOWN               | MYH 9            | Invasion/Migration; Metastasis | [83] |
| miR-9    | UP                 | CDX 2            | Cell cycle; Cell proliferation | [87] |
|          | DOWN               | NF-kappaB 1      | Cell proliferation | [93] |
| miR-10b  | DOWN               | MAPRE 1          | Cell proliferation; Clonogenicity | [45] |
| miR-15b-16 cluster | DOWN* | Bcl-2             | Apoptosis; Drug sensitivity | [96] |
| miR-21   | UP                 | RECK             | Cell proliferation; Invasion/Migration | [101] |
| miR-23a  | UP                 | IL6R             | Cell proliferation | [103] |
| miR-24   | DOWN               | AE1              | Cell proliferation | [94] |
| miR-106b-25 cluster | UP | p21●, p57●, E2F1●, Bim● | Apoptosis; Cell cycle; Cell proliferation | [41], [66] |
| miR-27a  | UP                 | PHB              | Cell proliferation | [84] |
| miR-29a  | DOWN               | p42.3            | Cell proliferation; Cell cycle | [72] |
| miR-34a/b/c | DOWN** | Bcl-2             | Apoptosis; Cell cycle; Cell proliferation; Clonogenicity | [78] |
| miR-43c  | UP                 | VEZT             | — | [77] |
| miR-107  | UP                 | DICER 1          | Invasion/Migration; Metastasis | [82] |
| miR-125a-5p | DOWN | ERBB 2            | Cell proliferation | [85] |
| miR-126  | UP                 | SOX2             | Cell proliferation | [86] |
|          | DOWN               | CRK              | Cell cycle; Cell proliferation; Invasion/Migration; Metastasis; Tumorigenicity | [74] |
| miR-129-2 | DOWN | SOX4             | Apoptosis | [88] |
| miR-130b | UP                 | RUNX 3           | Apoptosis; Cell proliferation; Cell viability | [81] |
| miR-139  | UP***              | CXCR 4           | — | [47] |
| miR-146a | UP                 | SMAD 4           | Apoptosis; Cell proliferation | [97] |
|          | DOWN               | EGFR, IRAK 1     | Invasion/Migration | [80] |
| miR-148a | DOWN               | ROCK 1, p27      | Cell cycle; Cell proliferation; Invasion/Migration; Metastasis | [75], [102] |
| miR-148b | DOWN               | CCKBR            | Cell proliferation | [89] |
| miR-150  | UP                 | EGR 2            | Cell proliferation | [95] |
| miR-181b | DOWN*              | Bcl-2            | Apoptosis; Drug sensitivity | [104] |
| miR-181c | DOWN               | NOTCH 4, KRAS    | Cell proliferation | [49] |
| miR-192/215 | UP | ALCAM            | Cell proliferation; Migration | [79] |
| miR-200  | UP****             | ZEB 1, ZEB 2     | — | [42] |
| miR-212  | DOWN               | MECP 2           | Cell proliferation | [92] |
| miR-218  | DOWN               | Robo 1           | Invasion/Migration; Metastasis | [91] |
| miR-222-221 cluster | UP | p27, p57, PTEN | Cell cycle; Cell proliferation; Invasion; Radiosensitivity | [66], [71] |
| miR-223  | UP*****            | EPB41L 3         | Metastasis | [43] |
| miR-331-3p | DOWN | E2F1             | Cell cycle; Cell proliferation; Clonogenicity | [76] |
| miR-335  | DOWN               | Bcl-w, SP 1      | Invasion/Migration; Metastasis | [98] |
| miR-375  | DOWN               | 14-3-3g, PDK 1, JAK 2 | Apoptosis; Cell proliferation; Cell viability | [73], [50] |
| miR-429  | DOWN               | c-myc            | Cell proliferation; Cell viability | [90] |
| miR-449  | DOWN               | MET, GMNN, CCNE 2, SIRT 1, HDAC 1 | Apoptosis; Cell cycle | [70] |
| miR-451  | DOWN               | MIF              | Cell proliferation; Cell viability; Radiosensitivity | [69] |
| miR-486  | DOWN               | OLFM 4           | Cell proliferation; Invasion/Migration | [54] |
| miR-512-5p | DOWN | Mcl-1             | Apoptosis | [51] |
| miR-650  | UP                 | ING 4            | Cell proliferation; Clonogenicity; Tumorigenicity | [100] |

* Multidrug resistance (MDR) gastric cancer cells.
** p53-deficient gastric cancer cells.
*** HER2KO gastric cancer cells.
**** Smad 3 overexpressing gastric cancer cells.
***** Metastatic cancers only.
• Target of miR-93 and miR-106b from miR-106b-25 cluster.
• Target of miR-25 from miR-106b-25 cluster.
