
Hemoproteins in Dissimilatory Sulfate- and Sulfur-Reducing Prokaryotes

Guy D. Fauque${}^{1,2,3}$ and Larry L. Barton${}^{4}$

${}^{1}$Laboratoire de Microbiologie IRD, UMR 180, Universités de Provence et de la Méditerranée, ESIL-GBMA, Case 925, Marseille Cedex 09, France  
${}^{2}$Laboratoire de Microbiologie, Géochimie et Ecologie Marines, CNRS UMR 6117, Campus de Luminy, Case 901, Marseille Cedex 09, France  
${}^{3}$Mediterranean Institute of Oceanography (MIO), Aix-Marseille University, UMR CNRS 7294/IRD 235, ESIL Case 925, Marseille cedex 9, France  
${}^{4}$Department of Biology, University of New Mexico, MSC03 2020, Albuquerque, New Mexico, USA  

---

### ABSTRACT

Dissimilatory sulfate and sulfur reduction evolved billions of years ago and while the bacteria and archaea that use this unique metabolism employ a variety of electron donors, H₂ is most commonly used as the energy source. These prokaryotes use multiheme *c*-type proteins to shuttle electrons from electron donors, and electron transport complexes presumed to contain *b*-type hemoproteins contribute to proton charging of the membrane. Numerous sulfate and sulfur reducers use an alternate pathway for heme synthesis and, frequently, uniquely specific axial ligands are used to secure *c*-type heme to the protein. This review presents some of the types and functional activities of hemoproteins involved in these two dissimilatory reduction pathways.

---

**Abbreviations** ..... 2  
1. Introduction ..... 3  
1.1. Dissimilatory Sulfate Reduction ..... 4  
1.2. Elemental Sulfur Reduction ..... 9  

ADVANCES IN MICROBIAL PHYSIOLOGY, VOL. 60  
ISSN: 0065-2911  
DOI: 10.1016/B978-0-12-398264-3.00001-2  
Copyright © 2012 by Elsevier Ltd.  
All rights reserved

1.3. Properties of Hemoproteins ..... 13
1.4. Metabolic Pathways for Heme Synthesis ..... 16
2. Hemoproteins in Dissimilatory SRB ..... 29
   2.1. Soluble Hemoproteins ..... 30
   2.2. Membrane-Bound Hemoproteins ..... 41
3. Cytochromes from Sulfur-Reducing Eubacteria ..... 54
   3.1. Monoheme Cytochromes ..... 54
   3.2. MultiHEME Cytochromes c ..... 56
   3.3. Sulfite Reductases ..... 62
   3.4. Complex Hemoproteins ..... 66
4. Concluding Statements ..... 69
References ..... 70

ABBREVIATIONS

A. Archaeoglobus  
APS adenylyl sulfate  
B. Bacillus  
BRC bacterial respiratory complex  
ccNir cytochrome c nitrite reductases  
CISM complex iron-sulfur molybdoenzyme  
D. Desulfovibrio  
DMSO dimethyl sulfoxide  
Dsf. Desulfuromonas  
Dsm. Desulfomicrobium  
DSR dissimilatory sulfite reductase  
Dst. Desulfotomaculum  
EPR electron paramagnetic resonance  
F₄₂₀ coenzyme  
FDH formate dehydrogenase  
FRD fumarate reductase  
G. Geobacter  
Hdr heterodisulfide reductase complex  
Hmc high-molecular mass cytochrome complex  
Mb. Methanobacterium  
Mc. Methanococcus  
MccA conserved multiHEME cytochrome c family  
MGD molybdopterin guanine dinucleotide  
Ms. Methanosarcina  
NR-SOB nitrate-reducing, sulfide-oxidizing bacteria  
P. Pyrobaculum  
 Psr polysulfide reductase  
Qmo quinone-interacting membrane oxidoreductase complex

HEMOPROTEINS IN DISSIMILATORY REDUCTION

Qrc  quinone reductase complex  
S.   Sulfurospirillum  
She. Shewanella  
SR    sulfur oxidoreductase  
SRB   sulfate-reducing bacteria  
SRO   sulfate-reducing organisms  
SRP   sulfate-reducing prokaryotes  
T.    Thermodesulfovibrio  
TMAO trimethylammonium N-oxide  
Tmc   tetraheme cytochrome complex  
TpIc₃ type I tetraheme cytochrome c₃  
W.    Wolinella  

1. INTRODUCTION

Hemoproteins are widespread in all groups of living organisms from anaerobes to aerobes, and from eukaryotes to Archaea. Hemoproteins that transfer electrons belong to the family of the cytochromes. Keilin (1925) was the first to propose the name “cytochrome” to describe a group of intracellular hemoproteins that undergo oxidation–reduction process and exhibit in the reduced-state intense absorption bands between 510 and 615 nm (Keilin, 1925). Cytochromes are pigmented electron transport proteins that absorb light at specific wavelengths via their heme cofactor. Their redox function is related to the valence change of heme iron (International Union of Biochemistry, 1992). Bacterial cytochromes mainly function either in aerobic and anaerobic respiration or in photosynthetic electron transport, by which the formation of ATP is linked to the oxidation of reduced substrates such as molecular hydrogen, reduced sulfur compounds, metals, or organic substances (Yamanaka, 1992). Cytochromes exhibit certain spectral characteristics depending on the local environments of the heme-binding sites in the protein and on the type of heme with which they are associated (Bartsch, 1968; Mathews, 1985).

This chapter will focus on the structural and functional relationships of hemoproteins (soluble and membrane-bound) involved in the dissimilatory sulfate and sulfur reduction pathways. This chapter will also present the different metabolic pathways involved in heme biosynthesis.

4 GUY D. FAUQUE AND LARRY L. BARTON

### 1.1. Dissimilatory Sulfate Reduction

Sulfur is a nonmetal element with multiple oxidation states and is one of the essential elements for life processes. Fe–S templates resulting from geochemical activity have been proposed to have a catalytic role in the origin of life (Wächtershäuser, 2010). While the last prokaryotic common ancestor remains elusive, elemental sulfur reduction, sulfate reduction, and anoxygenic photosynthesis are generally accepted to represent ancient metabolic processes (Canfield and Raiswell, 1999). In keeping with the theme of this review, we focus on relevant sulfate- and sulfur-reducing prokaryotes that are listed in Table 1. This table is not exhaustive; there are at least 60 genera and 220 species of SRB (Barton and Fauque, 2009). While there continue to be new organisms isolated with this activity, the prokaryotes thus far available are on the deep-branching lineages of the Tree of Life (Fig. 1) indicating early divergence from the prokaryotic common ancestor. It has been proposed by Canfield (2005) that the sparse distribution of sulfate reducers in the various taxonomic groups of Bacteria and Archaea Domains is attributed to the low concentration of sulfate in ocean waters until 2.4 billion years ago.

The phylogeny of the sulfate and sulfite reduction genes is an interesting point of discussion. The metabolic process of sulfate reduction is proposed to have originated 3.47 billion years ago (Shen et al., 2001; Wagner et al., 1998) and is characterized by the action of three successive enzyme steps: ATP sulfurylase, adenylyl sulfate (APS) reductase, and sulfite reductase (see Fig. 2 for reactions). The mechanism of dissimilatory sulfate reduction has been studied mostly with species of the genus *Desulfovibrio* which are the most easily and rapidly cultured sulfate reducers (Barton and Fauque, 2009; Fauque, 1995; Fauque and Ollivier, 2004; Fauque et al., 1991; Keller et al., 2011; LeGall and Fauque, 1988; Ollivier et al., 2007; Pfennig, 1989; Rabus et al., 2006; Widdel, 1988; Widdel and Hansen, 1992). Only four cytoplasmic enzymes are sufficient for the dissimilatory sulfate reduction to sulfide in an eight-electron reduction process.

#### 1.1.1. Sulfate Activation and Sulfate Reduction to Sulfite

Sulfate is an inert molecule that must be chemically activated before it can be involved in biochemical pathways. Inorganic sulfate needs first to be activated to APS by consumption of ATP (Peck, 1959). APS and PPI

HEMOPROTEINS IN DISSIMILATORY REDUCTION

Table 1 Genera of prokaryotes displaying dissimilatory sulfur reduction and dissimilatory sulfate reduction (list is not exhaustive).

| Sulfate reducers                                                                 | Elemental sulfur reducers                                                                 |
|----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|
| Archaea                                                                          | Archaea                                                                                    |
| Archaeoglobus                                                                    | Acidianus                                                                                  |
| Caldivirga                                                                       | Acidilobus                                                                                 |
| Bacteria—Group I${}^{a}$                                                         | Caldisphaera                                                                               |
| Desulfovibrio                                                                    | Caldococcus                                                                               |
| Desulfomicrobium                                                                 | Desulfurococcus                                                                            |
| Desulfoluna                                                                      | Desulfobulus                                                                               |
| Desulfocella                                                                     | Haloferax                                                                                  |
| Desulfofrigus                                                                    | Hyperthermus                                                                               |
| Desulfofustis                                                                    | Methanobacter                                                                              |
| Desulfohalobium                                                                  | Methanobacterium                                                                           |
| Desulfosporosinus                                                                | Methanococcus                                                                              |
| Desulfocurvus                                                                    | Methanosarcina                                                                             |
| Desulfonauticus                                                                  | Pyrobaculum                                                                                |
| Desulfocapsa                                                                     | Pyrococcus                                                                                 |
| Desulfonatronum                                                                  | Pyrodictium                                                                                |
| Desulforegula                                                                    | Staphylothermus                                                                            |
| Desulfotalea                                                                     | Stetteria                                                                                  |
| Desulfotomaculum${}^{b}$                                                         | Stygiolobus                                                                                |
| Desulfomonile                                                                    | Thermococcus                                                                               |
| Desulfobacula                                                                    | Thermocladium                                                                              |
| Desulfobulbus                                                                    | Thermodiscus                                                                               |
| Desulforhopalu                                                                   | Thermofilum                                                                                |
| Syntrophobacters                                                                 | Thermoplasma                                                                               |
| Thermodesulfobacterium                                                           | Thermoproteus                                                                              |
| Thermodesulfatator                                                               | Vulcanisaeta                                                                               |
| Thermodesulfovibrio                                                              | Bacteria                                                                                   |
| Bacteria—Group II                                                                | Ammonifex                                                                                  |
| Ammonifex                                                                        | Campylobacter                                                                              |
| Candidatus Desulfurudis                                                          | Desulfuromonas                                                                             |
| Desulfoarcus                                                                     | Desulfurella                                                                               |
| Desulfacium                                                                      | Desulfurivibrio                                                                            |
| Desulfobacter                                                                    | Desulfuromusa                                                                              |
| Desulfotomaculum${}^{b}$                                                         | Dethiobacter                                                                               |
| Desulfobacterium                                                                 | Hippea                                                                                     |
| Desulfatibacillum                                                                | Salmonella                                                                                 |
| Desulfoverniculus                                                               | Sulfurospirillum                                                                           |
| Desulfothermus                                                                   | Shewanella                                                                                 |
| Desulfospira                                                                     | Thermotoga                                                                                 |
| Desulfovirga                                                                     | Wolinella                                                                                  |
| Desulfovoccus                                                                    |                                                                                            |

(continued)

Table 1 (continued)

| Sulfate reducers | Elemental sulfur reducers |
|------------------|--------------------------|
| Desulfofema      |                          |
| Desulfosarcina   |                          |
| Desulforhabdus   |                          |
| Thermodesulfobium|                          |
| Thermodesulforhabdus |                      |
| Thermodesulfovibrio |                      |

*Group I bacteria have incomplete oxidation of carbon organic electron donor; Group II bacteria have complete oxidation of carbon organic electron donors.*

*Desulfotomaculum genus comprises both complete and incomplete oxidizer species.*

![Phylogeny of sulfur- and sulfate-reducing prokaryotes](#)

Figure 1 Phylogeny of sulfur- and sulfate-reducing prokaryotes. The trees of Bacteria and Archaea are based on ribosomal RNA-based phylogenies (*Madigan et al.*, 2009). Dissimilatory sulfate reduction is found in organisms encircled with a continuous line, and dissimilatory sulfur reduction is found in groups of bacteria encircled with a dashed line.

HEMOPROTEINS IN DISSIMILATORY REDUCTION

ATP
sulfurylase
SO₄²⁻ + ATP ↔ APS + PPᵢ → 2Pᵢ

Inorganic
pyrophosphatase

2H⁺ + 2e⁻
APS reductase
HSO₃⁻ → HS⁻ + 3H₂O

Bisulfite reductase

6H⁺ + 6e⁻

Figure 2 Dissimilatory sulfate reduction pathway.

(inorganic pyrophosphate) are produced from sulfate and ATP by the ATP sulfurylase (EC 2.7.7.4; ATP sulfate adenyltransferase):

SO₄²⁻ + ATP + 2H⁺ → APS + PPi ΔG°' = +46kJ/mol

ATP sulfurylase has only been purified and characterized from *Desulfovibrio (D.) gigas* and *D. desulfuricans* ATCC 27774; it is a novel metalloprotein containing cobalt and zinc (*Gavel et al.*, 1998). The formation of PPi being thermodynamically not favorable, the reaction needs to be pulled to completion by a second enzyme, an inorganic pyrophosphatase (EC 3.6.1.1; pyrophosphate phosphohydrolase) which can hydrolyze PPi according to the reaction:

PPi + H₂O → 2Pi ΔG°' = −22kJ/mol

The reduction of APS to bisulfite and AMP, catalyzed by APS reductase (EC 1.8.99.2), is the first redox reaction and is more exergonic than the pyrophosphate cleavage:

APS + H₂ → HSO₃⁻ + AMP + H⁺ ΔG°' = −69kJ/mol

APS reductase has been purified and characterized from different *Desulfovibrio* species (*Fauque et al.*, 1991; *Lampreia et al.*, 1994; *Lopez-Cortès et al.*, 2005) and also from *Archaeoglobus (A.) fulgidus* (*Lampreia et al.*, 1991). APS reductase is a major soluble cytoplasmic iron–sulfur

flavoprotein containing eight iron atoms arranged as two different [4Fe–4S] clusters and one FAD (Fauque *et al.*, 1991; Lampreia *et al.*, 1994). The specific electron donor required for the reduction of APS to bisulfite is yet unknown.

### 1.1.2. Reduction of Sulfite to Sulfide

The reduction of bisulfite to sulfide, catalyzed by sulfite reductase (EC 1.8.99.1), must compensate the energy investment of sulfate activation and yield additional ATP for growth. The standard free energy change of sulfite reduction with molecular hydrogen as electron donor is $-174 \, \text{kJ/mol}$; this could allow the regeneration of at least two ATP. The pathway of sulfite reduction to sulfide is somewhat controversial and two mechanisms have been proposed (LeGall and Fauque, 1988). The first mechanism represents the direct six-electron reduction of bisulfite to sulfide in one step, catalyzed by the dissimilatory sulfite reductase, without the formation of free intermediates. The second mechanism is a cyclic route, the so-called trithionate pathway, in which trithionate and thiosulfate are intermediates during the sulfite reduction to sulfide in three steps (Akagi, 1995; Cypionka, 1995; Rabus *et al.*, 2006). In this pathway, sulfite is first reduced to trithionate by the bisulfite reductase. Arguments against and for a trithionate pathway in species of *Desulfovibrio* have been presented, but only the isolation of mutants that will be altered with respect to one or both reductase activities would provide definitive information on the true mechanism of sulfite reduction to sulfide (Barton and Fauque, 2009; LeGall and Fauque, 1988; Oliveira *et al.*, 2008a,b; Parey *et al.*, 2010).

ATP sulfurylase, encoded on the *sat* gene, is common to both assimilatory and dissimilatory sulfate reduction pathways. APS reductase (Apr) is present in both the dissimilatory sulfate prokaryotes as well as the chemolithotrophic sulfide-oxidizing bacteria. The production of sulfide from sulfite is attributed to the siroheme dissimilatory sulfite reductase (Dsr). Prior to the evolution of the sulfate reducers, the level of sulfate in oceans was enhanced through biogenic activity of the sulfur-oxidizing anaerobic phototrophic bacteria (Habicht *et al.*, 2002) in detoxifying sulfide environments (Meyer and Kuever, 2007). As discussed by Meyer and Kuever (2007), the initial genes of dissimilatory sulfate reduction may have been the reversal of the ancestral Dsr enzymes functioning in oxidative sulfide metabolism. The enzymes of ATP sulfurylase (Sat) and APS reductase (Apr) were developed more recently.
A critical step for all organisms displaying dissimilatory sulfate reduction is the reduction of sulfite to sulfide by a siroheme dissimilatory sulfite reductase. This enzyme has at least an $\alpha_{2}\beta_{2}$ structure that is encoded by adjacent genes of *dsrA* and *dsrB*. In a study conducted by Klein *et al.* (2001), an argument is presented that the original donor of dissimilatory sulfite reductase genes, *dsrAB*, was a thermophilic organism. It would appear that *Archaeoglobus* species have acquired the *dsrAB* genes by lateral transfer from bacteria and dissimilatory sulfite reductase may have been transferred multiple times laterally across bacterial divisions (Klein *et al.*, 2001). There is a tendency to suggest that lateral gene transfer of sulfate-reducing genes is attributed to mobilizable metabolic islands; however, the genomic arrangements of the *Sat*, *AprBA*, and *DsrAB* genes along with the independent lateral gene transfers of these three gene arrangements does not support this hypothesis (Meyer and Kuever, 2007).

### 1.2. Elemental Sulfur Reduction

#### 1.2.1. Eubacteria

Elemental sulfur is probably the most widespread sulfur species in sediments and geological deposits. Many biological and chemical oxidation processes of H₂S do not directly produce sulfate but rather elemental sulfur, which rather may accumulate (Rabus *et al.*, 2006). Elemental sulfur is chemically relatively reactive and in contrast to sulfate, it requires no energy-dependent activation before a reduction can take place. The problem in the utilization of elemental sulfur mainly concerns its low solubility in water (0.16 μmol per liter at 25 °C; Boulègue, 1978). It seems probable that pure elemental sulfur (such as rhombic sulfur) cannot directly serve as the true substrate for sulfur reductase (Fig. 3). The so-called hydrophilic sulfur is probably the form available in aqueous medium; it consists of elemental sulfur associated with small portions of oxocompounds such as polythionates. Another possibility for

![Figure 3](https://i.imgur.com/1234567.png)

**Figure 3** Dissimilatory sulfur reduction pathway.
solubilization of sulfur is the cleavage of the S₈-ring of elemental sulfur by a nucleophilic attack of the HS⁻ anion resulting in the formation of polysulfide (Cammack *et al.*, 1984; Hedderich *et al.*, 1999; Steudel *et al.*, 1986).

Whereas sulfur-reducing bacteria may be mesophilic or moderately thermophilic, archaeal sulfur reducers are all extremely thermophilic. The oxidation of organic substrates in sulfur reducers may be complete, leading to CO₂ as an end product (such as in *Desulfurella*, *Desulfuromusa*, or *Desulfuromonas*), or incomplete, leading to acetate as the final product (as, for instance, in *Wolinella*, *Shewanella*, or *Sulfurospirillum*) (Liesack and Finster, 1994; Rabus *et al.*, 2006; Scott and Nealson, 1994).

The disproportionation of inorganic sulfur intermediates(also called “inorganic fermentation”) at moderate temperatures consists of a microbiologically catalyzed chemolithotrophic process in which compounds such as sulfite, thiosulfate, and elemental sulfur serve as both electron donor and acceptor and produce sulfate and hydrogen sulfide (Finster, 2008). This process is primarily carried out by SRB within the delta subclass of *Proteobacteria*. Elemental sulfur-disproportionating bacteria can be traced back in time as long as 3.5 billion years, indicating that elemental sulfur disproportionation would be one of the oldest biological processes on Earth (Finster, 2008; Philippot *et al.*, 2007). The eubacterial sulfur reducers comprise both facultative and true (or strict) respiratory microorganisms. Elemental sulfur has been reported to inhibit growth of some SRB in the presence of sulfate (e.g., *Desulfonema* species, *Desulfobacter postgatei*, *D.* sapovorans, and *Desulfomicrobium* (*Dsm.*) *acetoxidans*) (LeFaou *et al.*, 1990).

The ability to reduce elemental sulfur by prokaryotes is widespread (LeFaou *et al.*, 1990; Schauder and Kröger, 1993; Widdel and Hansen, 1992). Several genera of domain *Bacteria* and *Archaea* are able to grow by a dissimilatory reduction of elemental sulfur to sulfide in a respiratory type of metabolism (Barton and Fauque, 2009; Fauque *et al.*, 1991, 1994; Hedderich *et al.*, 1999; LeFaou *et al.*, 1990; Pfennig ad Biebl, 1976; Rabus *et al.*, 2006; Widdel and Pfennig, 1992; Zöphel *et al.*, 1988, 1991). The facultative sulfur-reducing eubacteria, such as the SRB, utilize elemental (or colloidal) sulfur as a respiratory substrate in the absence of other possible terminal electron acceptors such as sulfate, sulfite, thiosulfate, nitrate, or nitrite. The growth of many species of SRB is inhibited by elemental sulfur (Rabus *et al.*, 2006), probably because sulfur as an oxidant shifts the potential of redox couples in the medium and cells to unfavorable and positive values. Even if most of SRB are not able to grow by dissimilatory elemental sulfur reduction, some thiophilic species of SRB, belonging to the genera *Desulfomicrobium*, *Desulfovibrio*, *Desulfonauticus*, *Desulfohalobium*, *Desulfovermiculus*,

Desulfovistis, Desulfosporinus, Desulfospira, Desulfitospora, Desulfosarcina (Desulfosarcina cetonica, formerly Desulfobacterium cetonicum), and Ammonifex, use elemental sulfur as an alternative electron acceptor (Audiffren *et al.*, 2003; Beliakova *et al.*, 2006; Ben Dhia Thabet *et al.*, 2011; Biebl and Pfennig, 1977; Finster *et al.*, 1997; Friedrich *et al.*, 1996; Galushko and Rozanova, 1991; Huber *et al.*, 1996; Miroshnichenko *et al.*, 2008; Ollivier *et al.*, 1991; Robertson *et al.*, 2001; Sorokin *et al.*, (2007); Sorokin and Muyzer, 2010; Stackebrandt *et al.*, 2003).

Sulfur reducers use many different types of metabolic systems for oxidizing organic compounds. Some sulfur reducers belonging to genera Wolinella, Sulfurospirillum, and Shewanella incompletely oxidize organic substrates with acetate as an end product (Liesack and Finster, 1994; Rabus *et al.*, 2006; Scott and Nealson, 1994). Both complete and incomplete oxidation of organic electron donors is demonstrated by sulfur-reducing bacteria. Most species of the genera Desulfuromonas, Desulfuromusa, and Desulfurella, as well as Desulfospira joergensenii, are complete oxidizers. Most species of SRB able to reduce elemental sulfur are also incomplete oxidizers of organic substrates. A reduction of elemental sulfur is also performed by several members of the deep-branching bacterial hyperthermophilic genus, Thermotoga (Huber *et al.*, 1986), even if this is not an energy-gaining metabolism and that $S^{\circ}$ probably acts as a hydrogen sink during fermentative metabolism.

The presence of sulfur reductase is commonly associated with chemolithotrophic anaerobic growth; however, two sulfur-reducing bacteria are frequently associated with animals. Campylobacter rectus (formerly Wolinella (W.) rectus) is found along the gum line in the human mouth and couples H₂ oxidation with $S^{0}$ reduction (Gillespie and Barton, 1996). Salmonella enterica is commonly considered to be a heterotrophic enteric organism and has a multisubunit sulfur reductase (Hinsley and Berks, 2002). The similarity of these two sulfur-reducing systems in heterotrophic bacteria to that found in chemolithotrophic sulfur reducers may provide information on characteristics of genes for sulfur reduction.

The sulfur reductase (EC 1.97.1.3) is a constitutive enzyme in Desulfovibrio and Desulfomicrobium species, as it is also the case for the sulfur reductase from the sulfur-reducing eubacteria: W. succinogenes DSM 1740, DSM Deutsche Sammlung von Mikroorganismen, Braunschweig, Germany. aceto-xidans DSM 1675, and Sulfurospirillum (S.) deleyianum DSM 6946 (Fauque, 1994; Fauque *et al.*, 1994). The initial observations concerning the reduction of elemental sulfur to sulfide by the tetraheme cytochrome $c_{3}$ from $D$. vulgaris Miyazaki were considered to be nonphysiological (Ishimoto *et al.*, 1958). A physiological activity of tetraheme cytochrome $c_{3}$ associated with elemental sulfur reduction was only further demonstrated (Fauque *et al.*, 1979a, 1980).
The tetraheme cytochrome $c_{3}$ is the constitutive elemental sulfur reductase in several *Desulfovibrio* and *Desulfomicrobium* species from which the sulfur reductase activity can be copurified with the tetrahemoprotein (Fauque, 1994; Fauque *et al.*, 1979a). A mechanism of attack of elemental sulfur by an exposed, low-potential heme of the tetraheme cytochrome $c_{3}$ from *Dsm.* norvegicum strain Norway 4 (NCIB 8310) has been proposed. The polysulfide chains of colloidal sulfur are attacked by the reduced tetraheme cytochrome $c_{3}$, leading to a collapse of the micelles with the precipitation of S$_{8}$ molecules (Cammack *et al.*, 1984). The sulfide produced by polysulfides reduction opens up the S$_{8}$ rings by a nucleophilic attack, leading to the production of new molecules of polysulfides, which are themselves quickly reduced to sulfide by *Dsm.* norvegicum Norway 4 tetraheme cytochrome $c_{3}$ (Cammack *et al.*, 1984). Membranes isolated from *D.* gigas and *Dsm.* norvegicum Norway 4 contained hydrogenase and c-type cytochromes and catalyzed the dissimilatory sulfur reduction to sulfide. Membrane preparations of *D.* gigas were able to couple esterification of orthophosphate to electron flow from molecular hydrogen to elemental sulfur (Fauque *et al.*, 1980). Sufficient hydrogenase and tetrahemic cytochrome $c_{3}$ must be linked with the *D.* gigas cytoplasmic membrane in the correct conformation to generate proton translocation sufficient for chemiosmotic synthesis of ATP (Fauque *et al.*, 1980).

### 1.2.2. Archaea

Many genera of *Archaea* and *Bacteria* are able to grow with elemental sulfur as terminal electron acceptor in the energy metabolism (Schauder and Kröger, 1993). The dissimilatory reduction of elemental sulfur to hydrogen sulfide is linked with energy conservation as evidenced by growth on H$_{2}$ and S$^{\circ}$. The sulfur atoms of elemental sulfur, which at room temperature is S$_{8}$, are connected via S—S bonds just as the two sulfur atoms in heterodisulfide. The redox potentials of the heterodisulfide/H–S–CoB + H–S–CoM couple and that of the S$^{\circ}$/H$_{2}$S couple are also very close (around –200 mV) (Thauer *et al.*, 1977). Sulfur-reducing microorganisms and methanogens have one feature in common and it is that they both live at the expense of S—S bond reduction. Methanogenic bacteria differ, however, from sulfur reducers in that they are not dependent on an external sulfur source as electron acceptor since they are able to reoxidize the “reduced sulfur” with CO$_{2}$ or one of the other methanogenic carbon substrates. The reduction of elemental sulfur is widespread among members of the *Archaea*, including deep-branching hyperthermophilic genera.
Elemental sulfur can be used following four different types of metabolism:

1. Similar to *Thermotoga*, some members of the Archaea belonging to the genera *Pyrodictium, Pyrococcus, Thermoproteus, Desulfurococcus*, and *Thermococcus* utilize elemental sulfur reduction as a H₂ sink during fermentative metabolism (LeFaou et al., 1990).

2. The most widespread metabolism consists in the facultative or obligate chemolithoautotrophic reduction of elemental sulfur with hydrogen, accomplished by many hyperthermophiles from the Crenarchaeota, including members of the genera *Acidianus, Pyrodictium, Thermoproteus*, and *Pyrobaculum* (Kletzin et al., 2004; LeFaou et al., 1990; Widdel and Hansen, 1992).

3. *Pyrobaculum (P.) islandicum* and some members of the order *Thermoproteales* are able to respire heterotrophically with elemental sulfur in an apparently energy-gaining metabolism (LeFaou et al., 1990; Schönheit and Schäfer, 1995).

4. Some hydrogen-oxidizing methanogenic Archaea also reduce S° with methanol or hydrogen (Stetter and Gaag, 1983). It could be a secondary metabolism and may benefit the methanogens by quickly establishing reducing conditions in otherwise oxic, elemental sulfur-containing environments.

### 1.3. Properties of Hemoproteins

The natural function of hemoproteins is always linked to the basic properties of the metal center, fine-tuned by the protein environment (Pereira and Xavier, 2005; Pereira et al., 1998a,b). The heme moiety is constituted of a porphyrin, which, through its four pyrrol nitrogen, binds to one iron in an almost square planar geometry. The three most common heme groups found in hemoproteins from anaerobic microorganisms (heme b, heme c, and siroheme) are depicted in Fig. 4. Different types of porphyrins exist in hemoproteins from anaerobic bacteria. Heme type b (present in cytochromes b) and heme type c (found in mono- and multiheme cytochromes c) contain protoporphyrin IX. Uroporphyrin I, a new type and unexpected type of porphyrin, was reported in a new flavohemoprotein from *D. gigas*, the rubredoxin:oxygen oxidoreductase (Timkovich et al., 1994; Table 2). Sulfite reductases (and some nitrite reductases) contain siroheme, an iron tetrahydroporphyrin of the isobacteriochlorin type with eight carboxylic acid side-chains (Fig. 4). The hemoproteins cover an

14 GUY D. FAUQUE AND LARRY L. BARTON

![Chemical Structures](chemical_structures.png)

**Figure 4** Three heme groups commonly found in hemoproteins of dissimilatory sulfur and sulfate reducers.

extremely wide range of reduction potentials, from $-500 \, \text{mV}$ to $+400 \, \text{mV}$, explaining the presence of hemoproteins in most electron transfer chains, from aerobic to anaerobic systems. Most cytochromes from anaerobic bacteria have low reduction potentials, associated with a bis-histidinyl coordination, in agreement with the reduction potential of the environment where anaerobes live (Pereira and Xavier, 2005).

The function of hemoproteins may be divided into four groups: transport, electron transfer, storage, and chemical catalysis. Due to its redox properties, most heme function as electron transport components, transferring electrons either inside a multiredox center enzyme or between other proteins in redox chains. Hemoproteins may also display intrinsic

Table 2 Additional examples of heme-containing compounds other than cytochromes in sulfate- and sulfur-reducing prokaryotes.

| Compound                | Metal and type of heme          | Function               | Microbe                  | References                     |
|-------------------------|----------------------------------|------------------------|--------------------------|--------------------------------|
| Rubredoxin              | Iron uroporphyrin I             | Oxidase                | D. gigas                 | Timkovich et al. (1994)       |
| Bacterioferritin       | Iron coproporphyrin III         | Redox sensing          | D. desulfuricans        | Romão et al. (2000)           |
| Cobalt protein          | Cobalt sirohydrochlorin         | Unknown                | D. gigas                 | Battersby and Sheng (1982)     |
|                         |                                 |                        | D. desulfuricans        |                                |
| Catalase                | Heme                           | $H_2O_2$ catalysis      | D. gigas                 | Dos Santos et al. (2000)      |
| Cobaltochelatase        | Heme b                         | Cation insertion       | D. vulgaris              | Lobo et al. (2008b)           |
| Methyl-accepting        | c-Type heme                    | Oxygen sensing         | D. vulgaris              | Fu et al. (1994)              |
| protein                 |                                 |                        |                          |                                |
| Desulforubidin          | Siroheme                       | Sulfite reductase      | Dsm. baculatum           | Moura et al. (1988a,b)        |
|                         |                                 |                        | D. desulfuricans        | Lee et al. (1973a)            |
| Desulfoviridin         | Siroheme                       | Sulfite reductase      | D. gigas                 | Lee and Peck (1971)           |
|                         |                                 |                        | D. salexigens            | Czechowski et al. (1986)      |
|                         |                                 |                        | D. vulgaris              | Lee et al. (1973b)            |
| P-582                   | Siroheme                       | Sulfite reductase      | Dt. nigrificans          | Akagi and Adams (1973)        |
|                         |                                 |                        | Dt. nigrificans          | Trudinger (1970)              |
| Desulfofuscidins        | Siroheme                       | Sulfite reductase      | D. thermophilus          |                                |
|                         |                                 |                        | T. commune               | Hatchikian and Zeikus (1983)  |
|                         |                                 |                        | T. mobile                | Fauque et al. (1990)          |
| Sulfite reductases      | Siroamide                      | Sulfite reduction      | Desulfovibrio spp.       | Matthews et al. (1995)        |
| Oxygen reductase        | c-Type heme, copper            | Detoxification         | D. vulgaris              | Lobo et al. (2008a)           |

catalytic activities in redox reactions involving the oxidation or reduction of small molecules such as nitrite, sulfite, dioxygen, and sulfide (Pereira *et al.*, 1998a,b).

### 1.4. Metabolic Pathways for Heme Synthesis

The synthesis of heme is a multistep biosynthetic process (Fig. 5) that uses δ-aminolevulinic acid (ALA) as the precursor for biosynthesis of the tetrapyrrole moiety. While ALA can be produced from condensation of succinyl-CoA with glycine, this activity is found in nonphotosynthetic eukaryotes and several of the organisms in the alphaproteobacteria.


L-Glutamate       G-tRNA-S        Glutamyl-tRNA      GluTR         Glutamate 1-semialdehyde
                 6.1.1.17          hemA              GSAM           5.4.3.8
                                              5-Aminolevulinate
                                              PBGS            4.2.1.24
                                              Porphobilinogen
                                              PBGD            2.5.1.61
                                              Hydroxymethylbilane
                                              UROS            4.2.1.75
                                              Uroporphyrinogen III
                                              UROD            4.1.1.37
                                              Coproporphyrinogen III
                                              CPO             1.3.3.3
                                              CPDH            1.3.99.22
                                              Protoporphyrinogen IX
                                              PPO             1.3.3.4
                                              hemG            hemY
                                              Protoporphyrin IX
                                              FC              4.99.1.1
                                              hemH
                                              Heme

Sirohydrochlorin     Precorrin 2
CbiK                CysG
4.99.1.3           4.99.1.4
Co-Sirohydrochlorin Siroheme
↓                   ↓
Vitamin B12         Heme

Alternate heme
biosynthesis pathway


**Figure 5** Biosynthetic pathway for heme biosynthesis in prokaryotes. Genes and corresponding enzymes are as follows: *hemA*, glutamyl-t-RNA reductase; *hemL*, glutamate-1-semialdehyde-2,1-aminomutase; *hemB*, porphobilinogen synthase; *hemC*, porphobilinogen deaminase; *hemD*, uroporphyrinogen III synthase; *hemE*, uroporphyrinogen III decarboxylase; *hemN* and *hemF*, coproporphyrinogen III dehydrogenase; *hemG* (oxygen dependent) and *hemY* (oxygen independent) protoporphyrinogen IX oxidase; *hemH*, ferrochetalase; *cobA*, uroporphyrinogen III C-methyltransferase; *cysG*, precorrin-2 dehydrogenase; and *cbiK*, cobaltochelatase. Enzyme Commission numbers are listed for each catalytic step.

taxonomic group (Panek and O'Brian, 2002). Another process for production of ALA is found in diverse bacterial groups, including sulfate and sulfur reducers, and involves glutamyl-tRNA. Glutamate-1-semialdehyde aminotransferase, the product of *hemA* gene, requires NADPH for the conversion of glutamyl-tRNA to glutamate-1-semialdehyde (GSA). GSA aminotransferase, encoded on *hemL*, converts GSA to δ-aminolevulinic acid (ALA). The condensation of two molecules of ALA to produce porphobilinogen is achieved by porphobilinogen synthase, also known as ALA dehydratase, and this enzyme is encoded on *hemB*. In *D. vulgaris* Hildenborough, porphobilinogen synthase requires Zn²⁺ as a metal cofactor (Lobo et al., 2009) and Hg²⁺ detoxification reactions, as reviewed by Bruschi et al. (2007), would prevent inhibition of porphobilinogen synthase by Hg²⁺. The metabolic pathway for tetrapyrrole synthesis involves numerous macrocyclic molecules for the synthesis of heme, and from ALA to uroporphyrinogen III synthesis is markedly conserved in prokaryotes (Cavallaro et al., 2008). At the stage of uroporphyrinogen III in the production of heme, the biosynthetic pathway diverges into either the classic pathway or the alternate pathway.

### 1.4.1. Classic Pathway

In eukaryotes and most bacteria, four molecules of porphobilinogen are condensed by the action of porphobilinogen deaminase, product of *hemC*, to produce hydroxymethylbilane. The successive condensation of porphobilinogen to produce a linear tetrapyrrole is initiated by the action of porphobilinogen deaminase to eliminate the amino group from porphobilinogen. The linear hydroxymethylbilane molecule is converted to uroporphyrinogen III by the action of uroporphyrinogen III synthase. In the formation of the macrocyclic structure, uroporphyrinogen III synthase rearranges or inverts the fourth pyrrole to produce the asymmetric tetrapyrrole (uroporphyrinogen III; see Fig. 6). In many bacteria, all four of the acetyl groups of uroporphyrinogen III are decarboxylated by uroporphyrinogen decarboxylase to produce coproporphyrinogen III. Two propionyl side chains are oxidatively decarboxylated by coproporphyrinogen oxidase to form vinyl groups and produce protoporphyrinogen IX. As summarized in Fig. 6, two successive decarboxylation steps are required for the production of protoporphyrinogen IX from uroporphyrinogen III. The formation of protoporphyrin IX from protoporphyrinogen IX is an oxidative step with the release of 6e⁻ and 6H⁺. Oxidation of the carbon bonds between the pyrrole rings of protoporphyrinogen IX by protoporphyrinogen oxidase

Classic pathway

Uroporphyrinogen III

Coproporphyrinogen III
$\xrightarrow{2CO_2}$ Protoporphyrin IX
$\xrightarrow{4CO_2}$ Siroheme
$\xrightarrow{6e^- + 6H^+}$ Protoporphyrin IX
$\xrightarrow{Fe^{2+}}$ Protoheme

Alternate pathway

SUMT
$\xrightarrow{2 SAM}$ Precorrin-2
$\xrightarrow{2 SAH}$ Sirohydrochlorin
$\xrightarrow{Fe^{2+}, 2 H^+}$ PC2-DH
$\xrightarrow{2 CO_2}$ Nir ?
$\xrightarrow{Fe^{2+}}$ Chelatase ?

Figure 6 Late steps in heme biosynthesis utilizing classic and alternate pathways. Abbreviations: SUMT, S-adenosyl-L-methionine-dependent uroporphyrinogen III methyltransferase; SAH, S-adenosylhomocysteine; PC2-DH, precorrin-2 dehydrogenase; Nir, product of archaeal heme biosynthesis genes (*ahb-nir*).

produces protoporphyrin IX. Finally, heme is produced with $2H^+$ removed and $Fe^{2+}$ inserted into protoporphyrin IX by ferrocheletase. In addition to the production of various types of heme moieties (e.g., heme A, heme B, heme C, and heme D) for synthesis of specific cytochromes, the tetrapyrrole biosynthetic pathway contributes substrates for the synthesis of siroheme, cobaltamin, heme $d_1$, and coenzyme $F_{430}$. As employed by eukaryotes and many aerobic bacteria, the terminal sequence of the classic tetrapyrrole pathway emphasizes $O_2$ as the electron acceptor in the sequential oxidative steps associated with coproporphyrin oxidase and protoporphyrin oxidase. To accommodate the production of heme by anaerobic heterotrophic bacteria, oxygen-independent enzymes have been employed for the oxidation of coproporphyrinogen III and protoporphyrinogen IX; however, many of the sulfate- and sulfur-reducing prokaryotes lack these genes to provide enzymes for conversion of uroporphyrinogen III to heme.

### 1.4.2. Alternate Pathway

A unique pathway for heme biosynthesis appears to function for several anaerobic bacteria and archaea. With *Methanosarcina (Ms.) barkeri* strain
Fusaro DSM 804 (Buchenau *et al.*, 2006) and *D. vulgaris* strains Miyazaki F and Hildenborough (Ishida *et al.*, 1998), an alternate pathway for heme synthesis (see Fig. 1) involving precorrin-2 was proposed and has been gaining acceptance (Cavallaro *et al.*, 2008; Panek and O'Brian, 2002). Building on the initial report of Akutsu *et al.* (1993), Ishida *et al.* (1998) demonstrated that in *D. vulgaris* the methyl groups at the C-2 and C-7 positions on rings A and B (Fig. 6), respectively, of the porphyrin were from S-adenosylmethionine and not from C-2 of the original ALA molecules. In this novel heme biosynthesis proposed for *D. vulgaris*, uroporphyrinogen III is converted to precorrin-2 instead of coproporphyrinogen III (Lobo *et al.*, 2009; Raux *et al.*, 2000). While sirohydrochlorin, siroheme, and vitamin B₁₂ are produced from precorrin 2, the synthesis of heme would also be from precorrin 2. For heme production, there are two decarboxylation steps with the first attributed to precorrin-2 dehydrogenase (PC2-DH) to produce sirohydrochlorin and the enzymes for subsequent steps remain to be established.

Multifunctional enzymes have been reported to function in the conversion of uroporphyrinogen III to siroheme. As effectively discussed by Lobo *et al.* (2009), S-adenosylmethionine-dependent transmethylation of positions 2 and 7 on uroporphyrinogen III resulted in production of precorrin-2. The putative genes for this activity have different designations, with the genes encoding S-adenosyl-methionine-dependent uroporphyrinogen III methylase referred to as *cobA*, *cysG-1*, *cysG-2*, *cysG*, *uroM*, or *hemX*, and genes encoding precorrin-2 dehydrogenase named as *sirC*, *hemX*, *cysG*, or *cysGl* (Storbeck *et al.*, 2010). Here we will focus on *cobA*, *cysG*, and *sirA/B/C*. In *D. vulgaris*, *cobA* is fused with *hemD* (Lobo *et al.*, 2009) and this fused gene is found widely distributed in sulfate-reducing bacteria (SRB; see Table 4). However, the *cobA* gene is present in sulfate-reducing archaea and sulfur-reducing prokaryotes. Another gene, *cysG*, produces siroheme synthase which has been proposed to be a multifunctional protein with methyltransferase activity, NAD-coupled dehydrogenase, and ferrochetalase activity (Lobo *et al.*, 2009; Panek and O'Brian, 2002). The putative *cysG* gene designated by the JGI annotations is present in many of the sulfate-reducing prokaryotes (SRP; Table 4) even though *cobA* or the fused *cobA/hemD* genes are present. With the sulfur-reducing prokaryotes, *cysG* is found only in *Thermoproteus neutrophilus*. While *cysG* is not indicated in the JGI-published genome of *D. vulgaris* Hildenborough, Lobo *et al.* (2009) used amino acid sequence identity (26%) to *cysG* from *Escherichia coli* to reveal the presence of *cysG*². NAD-dependent precorrin-2 dehydrogenase activity was demonstrated in the putative CysG² enzyme from *D. vulgaris*, while CysG² from *E. coli* has both dehydrogenase activity(with similar NAD-binding motif) and ferrochelatase activity. The conversion of precorrin-2 to siroheme in *Bacillus (B.) megaterium* is reported to be attributed to SirA (a SAM-dependent methyltransferase), SirB (ferrochelatase), and SirC (the NAD-dependent dehydrogenase) (*Hansen et al.*, 1997; *Raux et al.*, 2000). The CysG<sup>B</sup> enzyme from *D. vulgaris* is functionally similar to SirC. However, *Lobo et al.* (2009) report a low amino acid sequence identity and similarity (~20–40%) between *D. vulgaris* SirC (CysG<sup>B</sup>) and similar proteins from *E. coli*, *B. megaterium* and *Saccharomyces cerevisiae*.

*Storbeck et al.* (2010) have proposed that *nir*-like genes are important in the archaea for the late steps of heme biosynthesis. The *nir* genes encode proteins for heme *d<sub>1</sub>* synthesis, and dissimilatory nitrite reductase contains heme *d<sub>1</sub>* as the prosthetic group. However, the only archaea known to contain cytochrome *cd<sub>1</sub>* nitrite reductase are *Pyrobaculum* sp. Therefore, *Storbeck et al.* (2010) suggest the *nir*-like genes in archaea be renamed *ahb* (archaeal heme biosynthesis)-*nir* genes. Sequence similarities between the *ahb-nir* genes of *Ms. barkeri* and *nir* genes of *Pseudomonas aeruginosa* are in the range of 30–40%. The *nir* genes (*nirD/H/J1/J2*) identified in *D. vulgaris* Hildenborough (*Storbeck et al.*, 2010) do not function to produce cytochrome *cd<sub>1</sub>* nitrite reductase but may participate in late stages of heme biosynthesis. In sulfate reducers, NirJ (encoded on *nirJ1/J2*) may be involved in the removal of propionate side chains on rings A and B of sirohydrochlorin with oxo groups added, and decarboxylation of acetate groups on Rings C and D may be attributed to NirD or NirH (see Fig. 6). The distribution of the *nir*-like genes in sulfate and sulfur reducers is discussed in Section 1.4.4.

### 1.4.3. Ferrochetalase

An important activity in the late stages of heme biosynthesis is the insertion of Fe<sup>2+</sup> into the macrocyclic porphyrin ring. With the classic pathway, Fe<sup>2+</sup> is inserted into protoporphyrin IX by ferrochelatase. In a review of 59 archaeal genomes, *Storbeck et al.* (2010) report that *hem E, hem F/N, hem G/Y,* and *hem H* were rarely observed. With archaea employing the alternate pathway for heme synthesis, protoporphyrin IX would not be produced but an intermediate from sirohydrochlorin transformation would be the substrate receiving Fe<sup>2+</sup>. Storbeck and colleagues raise the possibility that Co<sup>2+</sup> or Mg<sup>2+</sup> chelates, known to be present in archaea, could participate in this ferrochelatase activity (*Storbeck et al.*, 2010).
HEMOPROTEINS IN DISSIMILATORY REDUCTION 21

The alternate pathway for heme synthesis appears to function in many sulfate reducers and in *D. vulgaris* Hildenborough, Co²⁺ is inserted into sirohydrochlorin by CbiK, cobaltochelatase ([Lobo et al., 2008b](https://doi.org/10.1038/nature06954)). In fact, *D. vulgaris* contains two cobaltochelatases: one (CbiK<sup>C</sup>, DUV1365) is associated with cytoplasmic cobalamin synthesis and the other (CbiK<sup>P</sup>, DUV0650) is periplasmic. Both of these enzymes will insert Co²⁺ or Fe²⁺ into sirohydrochlorin with cobalt as the preferred substrate. Using recombinant studies with *E. coli* mutants deficient in sirohydrochlorin ferrochelatase, it was determined that CbiK<sup>P</sup> and CbiK<sup>C</sup> can insert ferrous ion into sirohydrochlorin. Lobo *et al.* (2008b) suggest that the role of CbiK<sup>C</sup> would be for synthesis of vitamin B₁₂ while the activity of CbiK<sup>P</sup> is less clear. Analysis of crystal structures indicates that the binding of sirohydrochlorin to CbiK is distinct from the binding of porphyrin by protoporphyrinogen ferrochelatase ([Romão et al., 2011](https://doi.org/10.1073/pnas.1017578108)). The tetrameric configuration of the CbiK<sup>P</sup>-containing molecule is stabilized by heme *b* and this protein may participate in the transport of metal-containing structures across the periplasm. While ferrochelatases are known to insert numerous divalent cations in addition to Fe²⁺ into protoporphyrin IX ([Dailey, 2002](https://doi.org/10.1016/S0006-291X(02)00900-2); [Price et al., 1989](https://doi.org/10.1016/0022-2836(89)90360-8)), the activity with other divalent chelatases requires additional study in sulfate reducers.

### 1.4.4. Bioinformatics

A review of databases available at [http://www.genome.jp](http://www.genome.jp) and [http://www.jgi.doe.gov](http://www.jgi.doe.gov) reveals that many of the genes encoding for enzymes traditionally found in the pathway for heme synthesis are not present in SRP (Tables 3 and 4). With respect to the presence of genes for heme synthesis, a distinction is seen between sulfate reducers of the archaeal and bacterial groups. *D. vulgaris* Hildenborough has a fusion of *hemeD* with *cobA*. As discussed by Panek and O'Brian (2002), the *cobA/hemD* fusion is not unique to sulfate reducers but has been reported for other bacteria. Experimental biochemistry ([Lobo et al., 2009](https://doi.org/10.1073/pnas.0900688106)) provided *in vivo* evidence that the N-terminal domain of the multifunctional enzyme functions as uroporphyrinogen III methyltransferase (CobA) and the C-terminal region has activity of uroporphyrinogen III synthase (HemD). Precorrin-2 is converted to sirohydrochlorin by an NAD-dependent precorrin-2 dehydrogenase, a product of the *CysG* gene.

In the survey by Cavallaro *et al.* (2008), *hemG/Y* for protoporphyrinogen oxidase production was present in 81% of Gram-negative bacteria and 52% of Gram-positive bacteria. The sulfate-reducing archaea neither have fused

Table 3 Putative genes for porphyrin synthesis in prokaryotes associated with reduction of sulfate.

| Prokaryote | Number of genes for porphyrin synthesis$^a$ |
|------------|------------------------------------------|
|            | L | A | B | C | D | D/CobA | E | N | Y | H | CobA | CysG | CbiK | CobN |
| Archaea    |                                          |
| Archaeoglobus fulgidus VC-16, DSM 4304               | 1 | 1 | 1 | 1 | 1 |     |   |   |   |   | 2 | 1 |   |   |
| Archaeoglobus profundus                              | 1 | 1 | 1 | 1 | 1 |     |   |   |   |   | 2 | 1 | 1 | 1 |
| Caldivirga maquilingensis TC-167                     | 2 | 1 | 1 | 1 | 1 |     |   |   |   |   | 1 | 1 |   |   |
| Bacteria   |                                          |
| Ammonifex degensii KC 4                              | 1 | 1 | 1 | 1 |   | 1   |   |   |   |   |   |   |   | 2 |
| Candidatus Desulfurudis audaxviatorMP104C             | 1 | 1 | 1 | 1 |   | 1   |   |   |   |   |   |   |   | 1 |
| Desulfarc ulus baarsii DSM 2075 AK-01                | 2 | 1 | 1 | 1 |   | 1   | 1 | 1 |   |   |   | 1 | 3 | 3 |
| Desulfatibacillum alkenivorans                       | 2 | 1 | 1 | 1 |   | 1   | 1 | 1 |   |   |   | 1 | 3 | 3 |
| Desulfobacterium autotrophicum DSM 3328              | 1 | 1 | 1 | 1 | 1 |     | 1 | 1 |   |   |   | 1 | 3 |   |
| Desulfobulbus propionicus DSM 2032                   | 1 | 1 | 1 | 1 |   | 1   | 1 | 1 | 1 | 1 |   | 3 |   | 1 |
| Desulfooccus oleovorans                              | 2 | 1 | 1 | 1 |   | 1   |   | 2 |   | 1 |   |   |   |   |
| Desulfohalobium ret haense DSM 5692                  | 1 | 1 | 1 | 1 |   | 1   |   | 1 |   |   |   | 1 | 1 |   |
| Desulfomicrobium baculatum DSM 4028                  | 1 | 1 | 1 | 1 |   | 1   | 1 | 1 |   |   |   | 1 | 2 |   |
| Desulfonatronospira thiodismutans ASO 3-1            | 1 | 1 | 1 | 1 |   | 1   |   |   |   |   |   | 1 | 2 |   |
| Desulfotalea psychrophila LSv 54                    | 1 | 1 | 1 | 1 |   | 1   | 1 | 1 | 1 | 1 |   |   |   | 2 |

Desulfotomaculum acetoxidans
DSM 771
1 1 1 1 1 1 2 1

Desulfotomaculum reducens MI-1
1 1 1 1 1 2 1

Desulfovibrio desulfuricans—G 20
1 1 1 1 1 1 1 2

Desulfovibrio desulfuricans
ATCC 27774
1 1 1 1 1 1 1 2

Desulfovibrio magneticus RS-1
1 1 1 1 1 1 2

Desulfovibrio salexigen
DSM 2638
1 1 1 1 1 1 1 2

Desulfovibrio vulgaris—
Hildenborough
1 1 1 1 1 1 2

Desulfovibrio vulgaris—
Miyazaki F
1 1 1 1 1 1 2

Desulfovibrio vulgaris—DP 4
1 1 1 1 1 1 1 2

Thermodesulfovibrio yellowstonii
DSM 11347
1 1 1 1 1 1 1 1 1


"Genes for porphyrin synthesis are as follows: letters are specific genes, that is, L, hemL; A, hemA; etc. Enzymes encoded by these genes are listed in the legend of Figure 1.

Table 4 Putative genes for porphyrin synthesis in prokaryotes associated with reduction of elemental sulfur.

| Prokaryote                                      | Number of genes for porphyrin synthesis $a$ |
|-------------------------------------------------|--------------------------------------------|
|                                                | L | A | B | C | D | D/CobA | E | N | Y | H | CobA | CysG | CbiK | CobN |
| **Archaea**                                     |                                            |
| Acidilobus saccharovorans 345-15                | 1 | 1 |    |    |    |       |   |   |   |   |     |     | 1    |      |
| Methanosarcina barkeri FusaroDSM 804            | 1 | 1 | 1 | 1 | 1 |       |   |   |   |   | 1    |     | 2    | 4    |
| Pyrobaculum islandicum DSM 4184                 | 1 | 1 | 1 | 1 | 1 |       |   |   |   |   |     |     |      | 2    |
| Thermoplasma acidophilum DSM 1728               | 1 | 1 | 1 | 1 |   |       |   |   |   |   |     |     | 1    | 1    |
| Thermoplasma volcanium GSS 1                    | 1 | 1 | 1 | 1 | 1 | 1     |   |   |   |   |     |     | 1    | 1    |
| Thermoproteus neutrophilus V24Sta               | 1 | 1 | 1 |   | 1 |       |   |   |   |   |     | 1    |      | 2    |
| **Bacteria**                                    |                                            |
| Campylobacter rectus                           | 1 | 1 | 1 | 1 | 1 |       |   |   |   |   | 3    | 2    |      |      |
| Desulfitobacterium hafniense                   | 1 | 1 | 1 | 1 |   |       |   |   |   |   | 2    | 2    |      | 1    |
| Dethiobacter alkaliphilus AHT 1                 | 1 | 1 | 1 | 1 | 1 | 1     |   |   |   |   | 1    |      | 1    |      |
| Geobacter sulfurreducens                       | 1 | 1 | 1 | 1 | 1 | 1     |   |   |   |   | 1    | 1    | 1    | 1    |
| Shewanella oneidensis                          | 1 | 1 | 2 | 1 | 1 |       |   |   |   |   | 1    | 2    | 3    | 2    | 1    | 1    |
| Sulfurospirillum deleyianum DSM 6946            | 1 | 1 | 1 | 1 | 1 |       |   |   |   |   | 1    | 2    |      | 1    |      |
| Desulfurispirillum indicum S5                   | 1 | 1 | 1 | 1 |   |       |   |   |   |   | 1    | 2    | 1    | 1    |      | 1    |
| Desulfu rivibrio alkaliphilus AHT 2             | 1 | 1 | 1 | 1 | 1 |       |   |   |   |   | 1    | 1    | 1    | 1    |      | 1    |
| Desulfurobacterium thermolithotrophum           | 1 | 1 | 1 | 1 |   |       |   |   |   |   |      |      |      |      |      |      |
| Desulfuromonas acetoxidans                      | 1 | 1 | 1 | 1 |   |       |   |   |   |   | 1    | 1    | 1    | 1    | 1    |      |
| Thermotoga neapolitana DSM 4359                 | 1 |   |   |   |   |       |   |   |   |   | 1    |      |      |      |      |      |
| Wolinella succinogenes                         | 1 | 1 | 1 | 1 | 1 |       |   |   |   |   | 1    | 2    | 1    | 1    |      |

"Genes for porphyrin synthesis are as follows: letters are specific genes, that is, L, hemL; A, hemA; etc. Enzymes encoded by these genes are listed in the legend of Figure 1."

HEMOPROTEINS IN DISSIMILATORY REDUCTION

Table 5 Distribution of putative genes for heme biosynthesis in prokaryotes.

| Genes | Archaea | Bacteria |
| --- | --- | --- |
|  | All groups | Sulfate reducers | Sulfur reducers | All groups | Sulfate reducers | Sulfur reducers |
|  | (30)${}^{a}$ (%) | (3)${}^{b}$ (%) | (6)${}^{b}$ (%) | (437)${}^{a}$ (%) | (25)${}^{b}$ (%) | (5)${}^{b}$ (%) |
| --- | --- | --- | --- | --- | --- | --- |
| hemB | 81 | 100 | 83 | 83 | 100 | 80 |
| hemC | 81 | 100 | 25 | 82 | 100 | 80 |
| hemD | 89 | 100 | 25 | 80 | 96 | 60 |
| hemE | 8 | 0 | 17 | 76 | 36 | 80 |
| hemN | 3 | 9 | 0 | 88 | 84 | 80 |
| hemG/Y | 0 | 0 | 0 | 66 | 16 | 4 |
| hemH | 8 | 0 | 33 | 80 | 20 | 60 |

The number of organisms in the various groups is given in parenthesis. All groups refer to various metabolic groups in either Archaea or Bacteria.

${}^{a}$ Calculated from Cavallaro et al. (2008) and Storbeck et al. (2010).

${}^{b}$ This report.

hemD/CobA genes (Tables 3–5) nor do they have hemE, hemN, hemY, or hemH. In the bacterial group, the early genes (hemL, hemA, hemB, and hemC) for porphyrin synthesis are present and in many of the SRB, the genes for HemD and CobA are fused and not separate. This would favor the formation of precorrin-2 from hydroxymethylbilane without the apparent accumulation of uroporphyrinogen III. However, some of the genes (hemN, hemE, and hemY) encoding enzymes for classical heme synthesis from uroporphyrinogen III are also present in sulfur and sulfate reducers. Of low frequency in distribution in the sulfate reducers is the gene for ferrochelatase (hemH). From bioinformatics studies, the genes for the latter stage of heme biosynthesis with alternate pathway by SRB remain to be identified.

Upon evaluation of the distribution of genes for heme synthesis, a high percentage of the sulfate and sulfur reducers have hem B, C, and D (Table 5). This distribution is relatively similar to distribution of heme synthesis in Bacteria and Archaea. The alternate pathway of heme synthesis appears dominant in Archaea with fewer archaea having genes for late heme synthesis (hemE, hemN, hemG/Y, and hemH). However, there are some sulfur and sulfate reducers that have genes for the classic biosynthesis pathway.

While numerous SRB use the alternate pathway for heme synthesis, the classic pathway for heme biosynthesis may be employed by several dissimilatory sulfate and metal-reducing bacteria. One of the genes of the classic pathway found in many of the prokaryotes with the alternate pathway is *hemN* and the function of this gene in the SRB is not apparent. Of the bacteria surveyed in Table 3, *hemN* is commonly distributed in these organisms except it is absent in *Desulfurobacterium thermolithotrophum*, *Thermodesulfovibrio yellowstonii*, *Caldivirga manuilingensis*, *Archaeoglobus profundus*, and *A. fulgidus*. Another gene commonly observed in the genomes of dissimilatory reducers is *hemE*; however, *hemE* is absent in *Desulfonarospira thiodismutans*, *Desulfohalibium rhaetene*, *Desulfarc ulus baarsii*, *Syntrorhobacter fumaroxidans*, and all *Desulfovibrio* in Table 4 even though these bacteria have *hemN*.

In a few organisms (*Desulfurivibrio alkaliphilus*, *Desulfobulbus propionicus*, and *Desulfotolea psychrophila*), the late genes in the classic pathway for heme synthesis are present.

For many sulfate-reducing archaea, the biosynthetic pathway for heme synthesis would appear to branch off the classic pathway from uroporphyrinogen (UROGEN) by using the enzyme S-adenosyl-L-methionine-dependent uroporphyrinogen III methyltransferase (SUMT). In *D. vulgaris* Hildenborough, the production of sirohydrochlorin is attributed to an enzyme that has multifunctions. Not only does the protein have UROGEN synthase and SUMT activities but it also has precorrin 2-dehydrogenase (PC2-DH) (Lobo et al., 2009). It is often difficult to read genome information because the genes encoding SUMT have been termed *cobA1*, *cysG-1*, *cysG-2*, *cysG*, *uroM*, or *hemX*. Additionally, the genes encoding for PC2-DH have been called *sirC*, *hemX*, *cysG*, or *cysG1*. Furthermore, the designation of the *nir* genes that encode for the enzymes producing heme *d₁* in *P. aeruginosa* and related denitrifiers have not been fully identified. Storbeck et al. (2010) make a strong argument that the radical SAM family may be candidates for involvement of heme *d₁* biosynthesis. Using amino acid sequences of the *nir* genes of *P. aeruginosa*, Storbeck et al. (2010) have identified genes in various archaea and *Desulfovibrio vulgaris* Hildenborough that are proposed to be involved in heme biosynthesis. *D. vulgaris* was proposed by Storbeck et al. (2010) to have *nirD*, *nirH*, *nirJ1*, and *nirJ2*. The gene neighborhood of *hemB* is provided in Fig. 7 and *D. vulgaris* has several *nir* genes proposed to be near *hemB*. Using these newly identified *nir* genes of *D. vulgaris*, we have found that several sulfate reducers have *nir*-like genes of considerable similarity (Table 6). When comparing the similarity of *nir* genes to those of *D. vulgaris*, similarities were also made for *hemABCD* and L.

HEMOPROTEINS IN DISSIMILATORY REDUCTION 27

Desulfotomaculum acetoxidans
![Diagram](https://i.imgur.com/PC2-DH_CysG.png)

Desulfotomaculum reducens
![Diagram](https://i.imgur.com/CobA/hemD_PC2-DH.png)

Thermodesulfovibrio yellowstonii
![Diagram](https://i.imgur.com/Cyto_C_biosynthesis_protein_CobA/hemD.png)

Desulfovibrio vulgaris Hildenborough
![Diagram](https://i.imgur.com/aprA_QmoB_nirJ1_aprB_QmoA_QmoC_nirD_nirJ2.png)

Methanosarcina barkeri fusaro
![Diagram](https://i.imgur.com/nirJ1_nirD_nirH_PC2-DH_G-tRNA-S.png)

Archaeoglobus fulgidus
![Diagram](https://i.imgur.com/L_C_CobA.png)

Figure 7 Genes in the neighborhood of putative heme biosynthesis. Abbreviations: A, B, C, D, Y, and L indicate heme genes; CobA/hemD, fused heme gene; nirJ1/J2, D, H, archaeal heme biosynthesis genes (ahb-nir); PC2-DH, precorrin-2 dehydrogenase; Apr A/B, APS reductase.

strains of Desulfovibrio, Desulfomicribium, and Desulfohalobium showed considerable similarity to *D. vulgaris* with respect to *nir* genes, a lower degree of similarity was observed between *D. vulgaris* and Desulfotomaculum, Desulfotalea, Desulfobacterium, Desulfooccus, Desulfacculus, or Desulfatibacillum.

Sulfur-reducing bacteria such as the Shewanella have multiple genes for heme synthesis because these organisms have the capability of growing both aerobically and anaerobically. As listed in Table 6, Shewanella (*She.*) amazonensis and She. baltica would appear to use the classic heme biosynthesis route since both possess *hemY* and *hemH*. The gene for

Table 6 Amino acid sequence identity of Heme and Nir proteins associated with sulfate and sulfur reducers using D. vulgaris Hildenborough as the reference protein.

| Organism                          | HemL (3168) | HemA (1461) | HemB (0856) | HemC (1890) | D/CobA (0734) | NirD   | NirH   | NirJ1  |
|-----------------------------------|-------------|-------------|-------------|-------------|---------------|--------|--------|--------|
| % Identity                        |             |             |             |             |               |        |        |        |
| Desulfovibrio vulgaris RCH 1      | 100%        | 100%        | 100%        | 100%        | 100%          | 100%   | 100%   | 100%   |
| Desulfovibrio vulgaris DP 4       | 99.76       | 99.32       | 99.70       | 99.37       | 100%          | 99.43  | 100%   | 100%   |
| Desulfovibrio vulgaris Miyazaki F | 82.46       | 81.57       | 86.93       | 78.80       | 85.49         | 68.53  | 79.44  | 89.11  |
| Desulfovibrio desulfuricans G 20  | 75.30       | 72.21       | 78.42       | 72.06       | 79.40         | 77.03  | 74.03  | 82.92  |
| Desulfovibrio desulfuricans ATCC 27774 | 69.74       | 70.62       | 68.58       | 67.11       | 69.26         | 72.30  | 67.53  | 80.49  |
| Desulfomicrobium baculatum X      | 70.41       | 61.24       | 66.06       | 66.03       | 65.33         | 65.99  | 54.49  | 80.99  |
| Desulfovibrio magneticus RS-1     | 68.74       | 60.73       | 63.89       | 63.43       | 63.54         | 65.31  | 50.97  | 71.27  |
| Desulfovibrio salexigens DSM 2638 | 65.78       | 61.78       | 70.06       | 64.82       | 63.07         | 61.90  | 49.68  | 70.72  |
| Desulfohalobium retbaense DSM 5962 | 63.48       | 58.74       | 65.84       |             |               | 61.22  | 56.33  | 71.51  |
| Desulfotomaculum reducens MI-1    | 57.79       | 40.98       |             |             |               | 49.32  | 32.03  |        |
| Desulfotalea psychrophila LSv 54  | 57.59       | 49.41       |             |             | 48.20         |        |        |        |
| Desulfurivibrio alkaliphilus AHT 2 | 58.85       | 52.13       |             | 54.87       | 52.60         |        |        |        |
| Desulfotomaculum acetoxidans DSM 771 | 57.18       | 38.68       | 58.36       |             | 48.48         | 47.30  | 37.25  |        |
| Desulfobacterium autotrophicum DSM 3382 | 59.43       | 49.88       |             |             | 56.25         | 47.97  | 40.26  |        |
| Desulfococcus oleovorans Hxd 3    | 57.52       | 48.46       |             |             | 54.75         | 47.30  | 40.26  |        |
| Desulfarculus baarsii DSM 2075    | 58.12       | 46.19       |             | 58.06       | 51.10         | 46.58  | 36.18  |        |
| Desulfatibacillum alkenivorans AK-01 | 57.14       | 47.47       |             |             | 54.12         | 49.32  | 41.18  |        |

oxygen-independent coproporphyrin III oxidase, *hemN*, is present in *She. amazonensis*, *She. baltica*, and *She. oneidensis*. While *She. oneidensis* has three *hemN* genes, Al-Sheboul and Saffarini (2011) have determined that one (*hemN1*) is most important for anaerobic heme synthesis while the other genes (*hemN2* and *hemN3*) have a relatively minor role. Although it remains to be established how many of the approximately 40 possible *c*-type cytochromes are produced by *She. oneidensis* under sulfur-reducing conditions, it is likely that this species and other sulfur-reducing *Shewanella* use the classic pathway for heme synthesis.

### 1.4.5. An Unfinished Story

Although there is a rich history of heme-related research in *D. gigas*, there is no genome analysis available for this organism. As reported earlier in this chapter, *D. gigas* has numerous cytochromes and heme-containing proteins. Earlier reports have indicated the activity of protoporphyrinogen oxidase in membrane fractions of *D. gigas* (Klemm and Barton, 1985) and purification of protoporphyrinogen oxidase has been attained (Klemm and Barton, 1987). This activity would distinguish *D. gigas* from other *Desulfovibrio*. In aerobic organisms using the classic pathway for heme synthesis, the six electrons from the oxidation of protoporphyrinogen with formation of protoporphyrin IX are diverted to O₂. In the absence of O₂, anaerobic bacteria, such as *E. coli*, have been proposed to divert the electrons from protoporphyrinogen oxidation to fumarate or nitrate as anaerobic electron acceptors (Jacobs and Jacobs, 1976). Additionally, oxidative phosphorylation was shown to be coupled to electron transport from oxidation of protoporphyrinogen and reduction of nitrite using membranes from *D. gigas* (Klemm and Barton, 1989). The mechanism whereby the protoporphyrinogen oxidase contributes to proton translocation in membrane of *D. gigas* remains to be explained; however, quinones have been implicated as hydrogen carriers in the late steps of heme synthesis (Jacobs and Jacobs, 1978).

## 2. HEMOPROTEINS IN DISSIMILATORY SRB

From a biochemical point of view, SRB contain a very complex and diversified electron carrier system. A characteristic feature of the dissimilatory sulfate reduction pathway is the involvement of multiheme *c*-type
cytochromes and iron-sulfur proteins of very low redox potentials (Barton and Fauque, 2009; Fauque *et al.*, 1991; LeGall and Fauque, 1988; Matias *et al.*, 2005; Moura *et al.*, 1988a–c; Pereira and Xavier, 2005; Pereira *et al.*, 1998a,b, 2007, 2011).

A comparative genomic analysis of energy metabolism was carried out with the 25 available genomes of SRB and archaea, including 17 Deltaproteobacteria, 4 Clostridia, 3 Archaea, and 1 Nitrospira (Pereira *et al.*, 2011). The Deltaproteobacteria contain a large number of cytochromes $c$ soluble and cytochrome $c$-associated membrane redox complexes, indicating the importance of periplasmic electron transfer pathways in these bacteria (Pereira *et al.*, 2011). In contrast, the Clostridia and Archaea groups contain practically no cytochromes $c$ or cytochrome $c$-associated membranes redox complexes (Pereira *et al.*, 2011).

Cytochromes of the $c$-type are a widespread class of hemoproteins essential for the life of almost all organisms (Moore and Pettigrew, 1990). $c$-Type cytochromes are characterized by the covalent binding of heme (Fe-protoporphyrin IX) to a polypeptide chain via one (or mainly two) thioether bonds produced as a result of the reaction of thiol groups of reduced cysteine residues with heme vinyl groups. The two cysteine residues almost always occur in the amino acid sequence CX$_{2}$CH, the so-called heme $c$-binding motif. The largest family consist of the class I $c$-type cytochromes, where the fifth and the sixth ligands of the hemes are histidine and methionine residues, respectively (Degtyarenko *et al.*, 1997; Mathews, 1985).

### 2.1. Soluble Hemoproteins

Sulfate reducers belonging to the Desulfovibrionaceae family contain a very high level of multiheme cytochromes $c$, the most abundant being TpI$c_{3}$ (Matias *et al.*, 2005). Desulfovibrio species differ from other SRO by having many $c$-type cytochromes (18 cytochromes $c$ in $D$. vulgaris Hildenborough genome, one cytochrome $c$ in $A$. fulgidus and $A$. profundus, and no cytochrome $c$ in $Dsm$. acetoxidans DSM 771, Caldivirga maquilingensis, and C. Desulfurudis audaxviator MP104C) (Pereira *et al.*, 2011).

Species of the genus Desulfovibrio contain different $c$-type cytochrome compositions. At least 18 periplasmic or membrane-bound cytochromes $c$ are present in $D$. vulgaris Hildenborough, some of which belong to the cytochrome $c_{3}$ family (Matias *et al.*, 2005; Pereira *et al.*, 2007, 2011). SRB
of the genus *Desulfovibrio* were the first nonphotosynthetic strictly anaerobic microorganisms reported to synthesize a $c$-type cytochrome (Ishimoto *et al.*, 1954; Postgate, 1954, 1956, 1984).

### 2.1.1. Monoheme $c$-Type Cytochromes (Methionine-Heme-Histidine)

Two different types of periplasmic monoheme $c$-type cytochrome are present in SRB of genera *Desulfovibrio* and *Desulfomicrobium* and their axial ligands are a methionine and a histidine residue (Fauque *et al.*, 1991). A small monoheme $c$-type cytochrome, known as cytochrome $c_{553}$, is present in several *Desulfovibrio* species. This small periplasmic monomeric hemoprotein (9 kDa) contains a single heme coordinated by a histidine and a methionine residue. It has a midpoint redox potential of ca. 0 mV and its physiological role remains unknown (Fauque *et al.*, 1991; Koller *et al.*, 1987; Moura *et al.*, 1987). The three-dimensional structures of cytochrome $c_{553}$ have been reported for *D. vulgaris* strains Hildenborough and Miyazaki F (Matias *et al.*, 2005). A periplasmic monoheme cytochrome $c_{553}$ has also been isolated and characterized from *D. desulfuricans* strain G201; it presents physicochemical and spectroscopic properties homologous to that of *D. vulgaris* Hildenborough cytochrome $c_{553}$ (Aubert *et al.*, 1998a). The monohemic cytochrome $c_{553}$ is present in five *Desulfovibrionaceae*, often in the same locus as cytochrome $c$ oxidase, suggesting it could act as its electron donor (Pereira *et al.*, 2011).

Another periplasmic monoheme $c$-type hemoprotein, cytochrome $c_{553}$ (550), has been purified and characterized only from *Dsm. baculatum* DSM 1743 and *Dsm. norvegicum* DSM 1741 (formerly known as *D. desulfuricans* Norway 4) (Fauque, 1985; Fauque *et al.*, 1979b). The *Dsm. norvegicum* cytochrome $c_{553(550)}$ has a molecular mass of 9.2 kDa, an isoelectric point of 6.6 and a midpoint redox potential of 50 mV. The characteristic feature of the absorption spectrum in the reduced state is the presence of an asymmetric alpha band with a maximum at 553 nm and a prominent shoulder at 550 nm. This is the first type of cytochrome isolated from SRB presenting a “split alpha” band at room temperature in the reduced state. The N-terminal sequence of *Dsm. norvegicum* cytochrome $c_{553(550)}$ presents no evident homology with that of *D. vulgaris* Hildenborough cytochrome $c_{553}$ and of *Dsm. norvegicum* cytochrome $c_3$ (Fauque, 1985; Fauque *et al.*, 1979b).

2.1.2. Multiheme Cytochromes c

A covalent heme attachment seems to be a key factor allowing dense packing of heme groups in multiheme $c$-type cytochromes. Bacterial $c$-type cytochromes are located either in the periplasm or in the outer membrane of Gram-negative microorganisms. Multihemic $c$-type cytochromes typically function in anaerobic bacterial electron transport chains and the growth of many anaerobically respiring microorganisms is linked to the presence of at least one $c$-type cytochrome.

Multihemic $c$-type cytochromes in prokaryotes consist of metalloproteins able to play various biochemical functions, including electron transfer and enzymatic activity (Sharma *et al.*, 2010). The multihemic $c$-type cytochromes isolated and characterized so far from sulfur- and sulfate-reducing organisms (SRO) are linked with diverse respiratory pathways involving the utilization of elemental sulfur and/or oxidized sulfur compounds as terminal electron acceptors (Fauque, 1994; Fauque *et al.*, 1991, 1994; Hedderich *et al.*, 1999; LeFaou *et al.*, 1990; LeGall and Fauque, 1988; Rabus *et al.*, 2006; Widdel and Pfennig, 1992). Multiheme cytochromes $c$ with bis-histidine axial coordination and low redox potential are classified according to Ambler (1991) as class III $c$-type cytochromes. This class includes both trihemic, tetrahemic, octahemic, and ninehemic cytochromes $c$ (Barton and Fauque, 2009; Matias *et al.*, 2005) and it shows no structural similarity with the $c$-type cytochromes of other classes.

Since the discovery in 1954 of $c$-type cytochromes in anaerobic, sulfate-reducing microorganisms, (Ishimoto *et al.*, 1954; Postgate, 1954) many eubacterial sulfate reducers were shown to possess different types of cytochromes, although in variable amount. Tetraheme cytochrome $c_3$ constitutes the predominant cytochrome in SRB (Coutinho and Xavier, 1994; Fauque *et al.*, 1991; LeGall and Fauque, 1988; Matias *et al.*, 2005). Tetraheme cytochrome $c_3$ belongs to the class III cytochromes characterized mainly by the presence of four hemes in a low-spin state with bis-histidyl coordination and quite negative redox potentials. Tetraheme cytochrome $c_3$ is the only hemoprotein reported in all *Desulfovibrio* species and it is characteristic of this genus, although it has also been reported in *Dsm. norvegicum*, *Desulfobulbus elongatus*, *Desulfocurvus vexinensis* and in two *Thermodesulfobacterium* species (Fauque *et al.*, 1991; Klouche *et al.*, 2009; Matias *et al.*, 2005; Pereira *et al.*, 1998a,b). Tetraheme cytochrome $c_3$ is a small (106–118 amino acid residues) soluble monomeric hemoprotein located in the periplasmic space. Tetraheme cytochrome $c_3$ plays a major role in the bioenergetics of sulfate reduction, mediating
the flow of electrons from periplasmic hydrogenases to respiratory trans-
membrane electron transport complexes coupled to the transfer of protons (Matias *et al.*, 2005; Pereira *et al.*, 2007). Two classes of tetraheme cyto-
chrome $c_{3}$ are present in *Desulfovibrio* species. The type I cytochrome $c_{3}$ (Type I-$c_{3}$) is the prototype of a family of multiheme cytochromes $c$ with
similar heme motifs. The cytochrome $c_{3}$ family includes the triheme cyto-
chrome $c_{7}$, the Type I-$c_{3}$, the Type II-$c_{3}$, the dimeric cytochrome $c_{3}$, the
9Hc, and the high-molecular mass cytochrome (HmcA). The Type I-$c_{3}$
has a molecular mass of around 13 kDa and contains four low redox
potential hemes (from $-120$ to $-400 \mathrm{mV}$). It may simultaneously capture
electrons and protons, which could be crucial for its biological functions.
The X-ray crystallographic structure of Type I-$c_{3}$ has been determined in
*Dsm.* norvegicum and in five *Desulfovibrio* species (Matias *et al.*, 2005).
A dimeric cytochrome $c_{3}$ [formerly called $\mathrm{cc}_{3}$ or cytochrome $c_{3}\left(M_{\mathrm{r}}\right.$
26,000)], containing two tetraheme subunits similar to TpI-$c_{3}$, has been
isolated, characterized, and crystallized from *Dsm.* norvegicum and *D.*
*gigas* (Matias *et al.*, 2005). The Type II cytochrome $c_{3}$ (Type II-$c_{3}$) consti-
tutes another group of tetraheme cytochrome $c_{3}$ with genetic, structural,
and reactivity properties different from Type I-$c_{3}$ (Valente *et al.*, 2001).
In addition to the three cytochromes discussed above (Type I-$c_{3}$, Type
II-$c_{3}$, $\mathrm{cc}_{3}$), the genome of *D.* vulgaris Hildenborough encodes for five
other tetrahemic cytochromes (Pereira *et al.*, 2007). A Split-Soret cyto-
chrome has been purified and characterized from the soluble fraction of
the sulfate/nitrate reducer *D.* desulfuricans ATCC 27774 (Costa *et al.*,
1997; Liu *et al.*, 1988); it is a 52 kDa homodimeric protein, having two
bis-histidinyl $c$-type hemes per monomer (Abreu *et al.*, 2003). A nine-
heme cytochrome $c$ (termed nonaheme cytochrome $c$) has also been puri-
fied and characterized from the soluble fraction of *D.* desulfuricans
strains ATCC 27774 and Essex 6 (Fritz *et al.*, 2001; Saraiva *et al.*, 2001).
Its three-dimensional structure was determined for both strains (Matias
*et al.*, 1999; Umhau *et al.*, 2001). The structure of *D.* desulfuricans Essex
6 nonaheme cytochrome $c$ revealed two tetraheme cytochromes $c_{3}$-like
domains with the typical arrangement of four heme centers. Both
domains flanked an extra heme buried under the protein surface (Umhau
*et al.*, 2001). A dodecaheme cytochrome $c$ (molecular mass 37.7 kDa)
has been purified, characterized, and preliminary crystallized from
*D.* desulfuricans ATCC 27774 cells grown under both sulfate- and
nitrate-respiring conditions (Coelho *et al.*, 1996; Liu *et al.*, 1988). There-
fore, it is likely to play a major function in the electron transfer system
of both respiratory chains.
2.1.3. Cytochromes for Dissimilatory Metal Reduction

Many of the sulfate-reducing and sulfur-reducing bacteria have been demonstrated to be involved in metal reduction by electron-coupled reactions involving cells with an active metabolism and not just by chemical reactions attributed to sulfide, the end product of respiration. *Geobacter sulfurreducens* contains over 100 genes coding for $c$-type cytochromes and the reduction of Fe(III) and Mn(VI) is attributed to $c$-type cytochromes (OmcE and OmcS) in the outer membrane (*Mehta et al.*, 2005). In *She. oneinedsis* MR-1, there are about 40 genes coding for $c$-type cytochromes (*Heidelberg et al.*, 2002) and metal reduction has been associated with the multiheme cytochromes of MtrC and OmcA in the outer membrane (*Shi et al.*, 2007). Electrons for the reduction of metal hydroxides on the outer surface of *G. sulfurreducens* and *She. oneinedsis* are proposed to be transferred across the periplasm from the quinone/quinol pool in the plasma membrane (*Shi et al.*, 2007). Numerous species of *Desulfovibrio* reduce metals by using Fe–S proteins or $c$-type cytochromes (*Barton and Fauque*, 2009; *Barton et al.*, 2003; *Bruschi et al.*, 2007). The reduction of metals would be attributed to hemoproteins in the periplasm or bound to the plasma membrane but unlike *Geobacter* or *Shewanella*, would not involve the outer membrane because there is no strong evidence for cytochromes in the outer membranes of *Desulfovibrio*. The presence of cytochromes in outer membrane fractions of *Desulfovibrio* may be attributed to the highly lipophilic character of periplasmic cytochromes.

Metal reductase activity is associated with monoheme or tetraheme $c$-type cytochromes isolated from sulfate- and sulfur-reducing bacteria (Table 7). As reviewed by *Barton and Fauque* (2009), the reduction of textile and food dyes may also be attributed to cytochromes produced by sulfate reducers. Metal ions would appear to interact with a specific heme and attachment of reduced metal may also be site specific. The reduction of Cr (VI), $\mathrm{CrO}_4^{2-}$ by the triheme cytochrome $c_7$, from *Dsm. acetoxidans* results in the production of Cr(III) and as indicated in Fig. 8, Cr(III) has a specific binding site in $c_7$ which is near heme IV (*Assfalg et al.*, 2002). Another site-specific metal binding was reported for the reduction of Mo(VI) by tetraheme $c_3$ isolated from *D. desulfuricans* G20 (*Pattarkine et al.*, 2006). Reduced molybdenum binds near heme (IV) of the $c$-type cytochrome (*Pattarkine et al.*, 2006). Respiratory-coupled reduction of Mo(VI) by *D. desulfuricans* with the production of $\mathrm{MoS}_2$ has been reported (*Tucker et al.*, 1997, 1998) and the reduction of molybdate by SRB is markedly enhanced by the presence of sulfide (*Biswas et al.*, 2009). As reviewed by

Table 7 Reductase activity associated with cytochromes from sulfate- and sulfur-reducing bacteria.

| Cytochrome | Organism                | $S^{0}$ | Fe (III) | Cr (VI) | Tc (VII) | Mn (IV) | V (VI) | U (VI) | Se (VI) | Mo (VI) |
|------------|-------------------------|---------|----------|---------|----------|---------|--------|--------|---------|---------|
| $c_{3}$    | $D$. vulgaris (Hildenborough) | +      | +        | +       |          | +       |        | +       |         | +       |
|            | $D$. vulgaris (Miyazaki)     |        | +        |         |          |         |        |         |         |         |
|            | $D$. gigas               |        | +        | +       |          |         |        |        |         |         |
|            | $D$. desulfuricans (Norway) |        | +        | +       |          |         |        |        |         |         |
|            | $D$. desulfuricans G-20   |        |          |         |          | +       |        |        |         | +       |
|            | $D$. fructosovorans      |        |          |         |          |         |        |        | +       |         |
| $c_{7}$    | Dsm. norvegicum          |        | +        | +       | +        |         |        |        |         |         |
|            | Dsm. acetoxidans         |        | +        | +       | +        | +       |        | +       |         |         |
| $c_{553}$  | $D$. vulgaris (Hildenborough) |        | +        |         |          |         |        |        |         |         |

Figure 8 Site on triheme cytochrome $c_{7}$ from *Dsf*. acetoxidans indicating binding of Cr(III) after Cr(VI) was reduced by the cytochrome $c_{7}$. The three heme groups in cytochrome $c_{7}$ are indicated by roman numbers and Cr(III) is indicated as a dark sphere. Used with permission: L. L. Barton & W. A. Hamilton (Eds.) (2007). *Sulphate-Reducing Bacteria*. Cambridge University Press.

Table 8 Microscopic redox potentials for SRO tetraheme cytochrome $c_3$ determined by electron paramagnetic resonance$^a$.

| Organism                | Redox potentials (mV) with associated heme |
|-------------------------|--------------------------------------------|
|                         | $\mathbf{e}_1$                             | $\mathbf{e}_2$                             | $\mathbf{e}_3$                             | $\mathbf{e}_4$                             |
|-------------------------|--------------------------------------------|--------------------------------------------|--------------------------------------------|--------------------------------------------|
| $D$. gigas              | $-315$ (HI)                                | $-306$ (HII)                               | $-235$ (HIII)                              | $-235$ (HIV)                               |
| $D$. vulgaris           | $-365$ (HIII)                              | $-350$ (HII)                               | $-320$ (HI)                                | $-300$ (HIV)                               |
| Hildenborough           |                                            |                                            |                                            |                                            |
| $Dsm$. baculatum        | $-355$ (HIII)                              | $-330$ (HI)                                | $-300$ (HIV)                               | $-150$ (HIII)                              |
| (Norway 4)              |                                            |                                            |                                            |                                            |

$^a$Data from Coutinho and Xavier (1994).

Biswas *et al.* (2009), elevated sulfide concentrations change Mo (VI) to a form that is more easily reduced to Mo(IV).

The reduction of a redox-active metal appears to be a chemical reaction between the metal ion and one or more hemes on the protein. With the monoheme (cytochrome $c_{553}$), the midpoint redox potential for the cytochrome from SRO is as follows: $D$. vulgaris Myazaki, +26 mV; $D$. vulgaris Hildenborough, +18 to +20 mV; $D$. desulfuricans NCIMB 8372; and $Dsm$. baculatum Norway 4, +40 to +50 mV (Yagi, 1994). As indicated in Table 8, distinct midpoint redox potentials are associated with each heme in the tetraheme cytochrome $c_3$ molecule. In the hexadeca-heme cytochrome $c$ isolated from $D$. vulgaris Hildenborough, the following midpoint redox potentials have been reported for the 16 hemes: +60 mV, +15 mV, $-120$ mV, $-135$ mV (7 hemes), $-190$ to $-205$ mV (5 hemes), and $-260$ mV (2 hemes) (Higuchi *et al.*, 1994). Not only must the heme have an appropriate midpoint potential to react with the metal but the heme must also be accessible and have appropriate chemistry at the heme-binding site (see the discussion on nitrite reductase in Section 2.2.1.1). As a point of reference to indicate ion specificity with heme groups, the nitrite reductase (Nrfa) from $D$. desulfuricans ATCC 27774 has five hemes (see Fig. 9) and nitrite is proposed to specifically bind to heme I (Costa *et al.*, 1996; Moura *et al.*, 2007).

HEMOPROTEINS IN DISSIMILATORY REDUCTION

![Diagram](#)

Figure 9 NrfA (nitrite reductase subunit) from *D. desulfuricans* ATCC 27774. Model indicates heme orientation as well as redox potential for each heme. Used with permission: L. L. Barton & W. A. Hamilton (Eds.) (2007). *Sulphate-Reducing Bacteria*. Cambridge University Press.

### 2.1.4. Sulfite Reductases

Sulfite reductases represent key enzymes for both biosynthetic sulfur assimilation and dissimilation of oxyanions, such as sulfate and sulfite, for energy conservation (*Fauque et al.*, 1991; *LeGall and Fauque*, 1988). Found throughout the three major kingdoms of living organisms, most of these SIR contain a siroheme that is exchange-coupled with an iron–sulfur center (*Crane et al.*, 1995).

#### 2.1.4.1. High-Spin Sulfite Reductases

Two types of sulfite reductases can be defined in SRP on the base of their physiological function. Their substrate is actually the bisulfite ion as
deduced from the acidic optimum pH of these two enzymes. The first type comprises the low-spin sulfite reductases, also called assimilatory-type sulfite reductases. They have a low molecular mass (around 27 kDa), a single polypeptide chain, and contain one \[4Fe–4S\] cluster coupled to a siroheme in a low-spin state ([Moura and Lino, 1994]). The second type is constituted by the high-spin bisulfite reductases (EC 1.8.99.1) which possess a large molecular mass (around 200 kDa) and a complex structure containing \[4Fe–4S\] clusters and siroheme. High-spin DSR contains $\alpha_{2}\beta_{2}\gamma_{m}\delta_{n}$ multimers with $\alpha$ 50 kDa, $\beta$ 45 kDa, $\gamma$ 11 kDa, and $\delta$ 8 kDa ([Fritz et al., 2001; Steuber and Kroneck, 1998]). High-spin DSR have been detected in all SRB investigated so far with the exception of *Desulfosarcina cetonica* (formerly *Desulfovibrio cetonium*), which did not contain desulfovirdin and desulforubidin ([Galushko and Rozanova, 1991]). Four different types of enzymes belonging to the high-spin DSR class (desulforubidin, desulfovirdin, desulfofuscin, and P-582) have been isolated and characterized from different genera of sulfate-reducing eubacteria ([Barton and Fauque, 2009; Fauque et al., 1991; LeGall and Fauque, 1988]). The red-brown protein, desulforubidin, belongs to the genera *Desulfomicrobium*, *Desulfocurvus*, *Desulfovibrio*, *Desulfobacterium*, *Desulfosarcina*, *Desulfuromonas*, and *Desulfobacter* ([Barton and Fauque, 2009; DerVartanian, 1994; Klouche et al., 2009; Loy et al., 2008; Ollivier et al., 1991]). The green protein, desulfovirdin, is the DSR characteristic of the genus *Desulfovibrio*, but it has also been found in some species of the genera *Desulfomonile*, *Desulfobulbus*, *Desulfococcus*, and *Desulfonema* ([DeWeerd et al., 1990; Fauque et al., 1991; Moura et al., 1988b; Rees and Patel, 2001; Steuber and Kroneck, 1998; Steuber et al., 1994, 1995; Wolfe et al., 1994]). The green protein, desulfovirdin, is the DSR characteristic of the genus *Desulfovibrio*, but it has also been found in some species of the genera *Desulfomonile*, *Desulfobulbus*, *Desulfococcus*, and *Desulfonema* ([DeWeerd et al., 1990; Fauque et al., 1991; Moura et al., 1988b; Rees and Patel, 2001]). The dark brown-colored protein, desulfofuscin, is the DSR of thermophilic eubacteria sulfate reducers such as *Thermodesulfobacterium (T.) commune*, *T. mobile*, *Thermodesulfovibrio hydrogenophilus*, and *Thermodesulfovibrio yellowstonii* ([Fauque et al., 1990; Haouari et al., 2008; Hatchikian, 1994]). P-582-type bisulfite reductase has only been reported in several species of the spore-forming, sulfate-reducing genera *Desulfotomaculum* and *Desulfosporosinus* ([Akagi and Adams, 1973; Fauque et al., 1991; Ramamoorthy et al., 2006; Stackebrandt et al., 2003; Trudinger, 1970]). These four enzymes differ mainly by the behavior of their siroheme moieties, their major optical absorption and electron
paramagnetic resonance (EPR) spectra, and their reaction with CO (Barton and Fauque, 2009; Fauque *et al.*, 1991; LeGall and Fauque, 1988). The crystal structure of desulfovirdin from *D. vulgaris* Hildenborough has been recently reported (Oliveira *et al.*, 2008a,b). A new mechanism for the dissimilatory sulfite reduction process in this species has been proposed. It involves DsrAB, DsrC, and the DsrMKJOP membrane complex, with two of the six electrons necessary for the sulfite reduction deriving from the membrane quinol pool (Oliveira *et al.*, 2008a,b).

An archaeal DSR has been purified and characterized (in three chromatographic steps; 10 mg of pure DSR obtained per 10 g of cells) from the extremely thermophilic SRB *A. fulgidus*. This enzyme is an $\alpha_2\beta_2$ tetramer of molecular mass 178 kDa (subunits of 51 and 45 kDa) and it contains two sirohemes and six [4Fe–4S] clusters per molecule (Dahl and Trüper, 2001; Dahl *et al.*, 1993, 1994; Schiffer *et al.*, 2008). The UV/Visible spectrum of *A. fulgidus* DSR exhibits bands at 281, 394, 545, and 593 nm, characteristic of siroheme in the high-spin state (Dahl and Trüper, 2001).

The X-ray structures of DSR from *A. fulgidus* have been recently determined in complex with sulfite, sulfide, nitrate, nitrite, phosphate, carbon monoxide, and cyanide (Parey *et al.*, 2010). Activity measurements indicated that *A. fulgidus* DSR is able to reduce, besides sulfite and nitrite, thiosulfate and trithionate and to produce the latter two compounds besides sulfide. On this basis, a three-step mechanism of sulfite reduction to sulfide was proposed, each step consisting of a two-electron transfer, a two-proton uptake, and a dehydration event (Parey *et al.*, 2010).

### 2.1.4.2. Low-Spin Assimilatory Sulfite Reductase from *D. vulgaris* Hildenborough

A low-molecular weight assimilatory-type sulfite reductase has been purified and characterized from *D. vulgaris* Hildenborough (Drake and Akagi, 1976; Huynh *et al.*, 1984; Lee *et al.*, 1973a). This enzyme has a molecular mass of 27.2 kDa and its optical spectrum exhibits maxima at 590, 545, and 405 nm. This hemoprotein is able to reduce sulfite in the presence of reduced methyl viologen and sulfide is the sole product of this reduction. The specific sulfite reductase activity measured at pH = 6.0 using pure periplasmic hydrogenase from *D. gigas* was 900 mU/mg of protein (Moura and Lino, 1994).

The assimilatory sulfite reductase from *D. vulgaris* Hildenborough has been studied by EPR, Mössbauer, and chemical techniques (Huynh *et al.*, 1984). This sulfite reductase was found to contain a single [4Fe–4S] cluster
and one siroheme. Since the protein contains a total of five labile sulfur atoms, it was postulated that the extra sulfur atom provides for the covalent binding between the \([4 \mathrm{Fe}-4 \mathrm{~S}]\) cluster and the heme iron.

As purified, the siroheme is low-spin ferric \((S=1 / 2)\) which exhibits characteristic EPR resonances at \(g=2.44,2.36\), and 1.77 . The \([4 \mathrm{Fe}-4 \mathrm{~S}]\) cluster is in the \(2+\) state. Similar to the hemoprotein subunit of \(E\). coli sulfite reductase, low-temperature Mössbauer spectra of sulfite reductase from \(D\). vulgaris Hildenborough also show evidence for an exchange-coupled siroheme-\([4 \mathrm{Fe}-4 \mathrm{~S}]\) unit (Huynh et al., 1984).

The presence of an assimilatory-type sulfite reductase in \(D\). vulgaris Hildenborough is surprising because this strain produces large amounts of sulfide during normal growth on sulfate and also because the enzymes responsible for dissimilatory sulfate reduction are constitutive. The physiological significance of the assimilatory-type sulfite reductase is still not understood. Peck and Lissolo have suggested that this enzyme may function in a dissimilatory mode possibly in the anaerobic disproportionation of sulfite and thiosulfate into sulfide and sulfate (Peck and Lissolo, 1988). The occurrence of the assimilatory-type sulfite reductase in \(D\). vulgaris Hildenborough does not appear to be unique as it has also been found in \(D\). desulfuricans and \(D\). gigas (Lee et al., 1973a).

### 2.1.5. Formate Dehydrogenases from Desulfovibrio Species

Formate dehydrogenases (FDH) constitute a diverse group of enzymes found in both eukaryotes and prokaryotes, which catalyze the formate oxidation to \(\mathrm{CO}_{2}\). In contrast to most FDHs from aerobic bacteria, FDHs from anaerobes contain metal cofactors. FDHs from anaerobic bacteria are in most case tungsten- or molybdenum-containing iron–sulfur proteins and additionally hemes or flavins (Moura et al., 1999). Three FDHs are encoded in the genome of \(D\). vulgaris Hildenborough, with formate, lactate or hydrogen as electron donors (Pereira et al., 2007). FDHs have been purified and characterized from four species of Desulfovibrio: \(D\). gigas NCIB 9332 (Almendra et al., 1999; Riederer-Henderson and Peck, 1986a,b), \(D\). vulgaris strains Hildenborough and Miyazaki (Sebban et al., 1995; Yagi, 1969, 1979), \(D\). desulfuricans ATCC 27774 (Costa et al., 1997), and \(D\). alaskensis NCIMB 13491 (Brondino et al., 2004).

The FDH partially purified from \(D\). vulgaris Miyazaki was able to reduce the monohemic cytochrome \(c_{553}\) but not the tetrahemic cytochrome \(c_{3}\) (Yagi, 1969, 1979). The FDH from \(D\). vulgaris Hildenborough is oxygen
sensitive and composed of three subunits containing an iron–sulfur cluster,  
a molybdenum cofactor (linked to the large subunit of 83.5 kDa),  
and heme $c$-type moieties (bound to the small subunit of 14 kDa) (Sebban *et al.*, 1995). *In vitro* measurements suggested that a monoheme  
cytochrome $c$ probably acts as a physiological partner of *D.* vulgaris  
Hildenborough FDH in the periplasm (Sebban *et al.*, 1995). The FDH from  
*D.* desulfuricans ATCC 27774 has been purified and characterized by EPR  
and Mossbauer spectroscopy (Costa *et al.*, 1997). This air-stable enzyme  
has a molecular mass of 150 kDa (three different subunits: 88, 29 and  
16 kDa) and it contains three different types of redox-active centers: a  
molybdopterin site (Mo-bound to two molybdopterin guanine dinucleotide, MGD), four $c$-type hemes, and two different [4Fe–4S centers]; selenium was also chemically detected. The presence in *D.* desulfuricans  
ATCC 27774 FDH of four $c$-type hemes in the low-spin state was confirmed by Mossbauer spectroscopy (Costa *et al.*, 1997). The *D.* gigas  
FDH is the first tungsten-containing FDH isolated and characterized so far from a SRB (Almendra *et al.*, 1999). This protein contains two  
[4Fe–4S] clusters and the tungsten is coordinated with two MGD cofactors.  
The *D.* gigas FDH (molecular mass of 120 kDa) consists of only two subunits (92 and 29 kDa) and no heme $c$-type was detected (Almendra *et al.*, 1999). The *D.* gigas FDH has been structurally characterized (gene sequence and crystal structure) (Raaijmakers *et al.*, 2001, 2002). The FDH from *D.* alaskensis NCIMB 13491 is a heterodimeric protein with a molecular weight of 126 kDa, composed of two subunits (93 and 32 kDa), which contains a [4Fe–4S] center and both molybdenum and tungsten (Brondino *et al.*, 2004).

## 2.2. Membrane-Bound Hemoproteins

### 2.2.1. Membrane-Bound Cytochromes

#### 2.2.1.1. Cytochrome $c$ Nitrite Reductase from *D.* desulfuricans  
ATCC 27774  

The dissimilatory reduction of nitrate and/or nitrite to ammonia (also called dissimilatory ammonification) can serve as the sole energy-conserving process in some species of SRB. Nitrate is reduced to ammonia (with nitrite as intermediate) by a few strains of SRB belonging mainly to *D.* desulfuricans and also to *D.* profundus, *D.* oxamicus, *D.* multispirans, *D.* simplex, *D.* furfuralis, and

D. termitidis (Lopez-Cortès et al., 2005; Moura et al., 2007; Seitz and Cypionka, 1986). Dissimilatory ammonification has also been found with Thermodesulfovibrio islandicus, Desulfobacterium catecholicum, Desulfotomaculum (Dst.) thermobenzoiicum, Desulfobulbus propionicus, Thermodesulfobium narugense, and Desulforhopalus singaporensi (Barton and Fauque, 2009; Moura et al., 2007). Depending on the strain, sulfate or nitrate may be the preferred terminal electron acceptor. Nitrate reductase is inducible by nitrate or nitrite, whereas nitrite reductase is constitutive in D. desulfuricans Essex 6 (Seitz and Cypionka, 1986). A vectorial proton translocation during nitrate or nitrite reduction has been demonstrated with whole cells of several Desulfovibrio species (Cypionka, 1995). ATP synthesis coupled to the dissimilatory reduction of nitrite to ammonia was obtained with D. gigas membranes (Barton et al., 1983).

A novel type of metabolism connecting the nitrogen and sulfur cycles has been shown in D. desulfuricans CSN, which can oxidize sulfite and thiosulfate with nitrate and nitrite as terminal electron acceptors (Krekeler and Cypionka, 1995). Nitrate-reducing, sulfide-oxidizing bacteria (NR-SOB) are able to inhibit the growth of SRB species in the presence of nitrate (Haveman et al., 2005). This inhibition could be due to an increase in redox potential or due to the production of nitrite by the NR-SOB. Nitrite (but not nitrate) is also known to inhibit the dissimilatory sulfite reduction to sulfide by DSR (Haveman et al., 2004). D. desulfuricans ATCC 27774 (DSM 6949) is the best-studied ammonifying strain of SRB from physiological and biochemical points of view. The biochemical, physiological, and spectroscopic characterization of D. desulfuricans ATCC 27774 nitrate and nitrite reductases has been recently reported by Moura et al. (2007).

One of the more widespread cytochromes in SRB is the nitrite reductase complex formed by the two cytochromes NrfA and NrfH (Rodrigues et al., 2006). NrfHA acts as a detoxifying enzyme because nitrite is a powerful inhibitor of SRB (Greene et al., 2003). The multiheme nitrite reductases [(EC 1.7.2.2 nitrite reductase) (cytochrome: ammonia-forming)] (ccNir) act on the dissimilatory ammonification process, where they catalyze the reduction of nitrite to ammonia in a unique six-electron step:

\[
\text{NO}_2^- + 8\text{H}^+ + 6\text{e}^- \rightarrow \text{NH}_4^+ + 2\text{H}_2\text{O} \quad \text{E}^\circ = +330\text{mV}
\]

Liu and Peck were the first to purify and characterize a ccNir from D. desulfuricans ATCC 27774 (Liu and Peck, 1981). Several ccNirs were isolated later from different groups of microorganisms. Over more than a decade ccNirs were erroneously considered as monomers of ~60 kDa

containing six *c*-type hemes (Moura *et al.*, 2007). The first X-ray structure, obtained for the *S.* deleyianum ccNir, clarified the situation showing the presence of five hemes per subunit (Einsle *et al.*, 1999).

The ccNir from *D* desulfuricans ATCC 27774 is obtained by membrane solubilization. The SDS-PAGE profile showed that, besides the 60 kDa subunit, another band with ~20 kDa is also present (Moura *et al.*, 2007). The associated ccNir is a high-molecular mass oligomer having 890 kDa and higher mass (Liu and Peck, 1981). The two subunits that compose this oligomer were separated with difficulty by SDS treatment. The ccNir is a complex of two different subunits, the pentahemic NrfA and the tetrahemic NrfH, implying the existence of nine different hemes. The large subunit NrfA contains the high-spin heme and four low-spin *c*-type, while the NrfH subunit encloses four heme groups in a low-spin configuration.

The UV–Visible absorption spectrum of the native *D.* desulfuricans ATCC 27774 ccNir exhibits absorption bands at 532, 409 (Soret), and 278 nm in the oxidized form and at 552, 523, and 420 nm in the reduced form, characteristic of typical heme *c*-containing proteins. In addition, a band due to high-spin heme *c* contribution is also observed at 610 nm (Liu and Peck, 1981). EPR and Mössbauer studies (Costa *et al.*, 1990b, 1996) on this enzyme revealed a complex system, involving multiple heme species with detectable magnetic interactions between them (Costa *et al.*, 1990b, 1996). Only one of the hemes is in a high-spin state and is involved in substrate binding.

Besides nitrite, ccNirs are also able to reduce other substrates; they can reduce nitrogen monoxide and hydroxylamine, two potential bound reaction intermediates, into ammonia (Costa *et al.*, 1990a; Liu *et al.*, 1987). The enzyme can also reduce sulfite to sulfide in a six-electron step (Einsle *et al.*, 2002a,b; Pereira *et al.*, 1996). The primary sequence of both subunits revealed that *D.* desulfuricans ATCC 27774 NrfA (gene-encoding cytochrome *c* nitrite reductase) binds five hemes, four of each are bis-histidine coordinated and one just by a lysine. NrfH contains 4 × bis-histidinyl coordinated hemes. The N-terminal amino acid sequence of NrfH forms a hydrophobic helix that attaches it to the membrane.

The three-dimensional structure of ccNir isolated from *D.* desulfuricans ATCC 27774 was solved at 2.3 Å (Cunha *et al.*, 2003) (Fig. 10). The protein crystallizes as a homodimer, showing the 10 hemes in a very dense packing (distance between the iron atoms: from 9 to 12.5 Å). Four hemes (hemes 2, 3, 4, and 5) are coordinated by two axial histidines with different relative orientation whereas the heme 1 is pentacoordinated (lysine in axial position). One anionic and one cationic channel of access to the active center

Figure 10 Structure of ccNir isolated from *D. desulfuricans* ATCC 27774. Used with permission: L. L. Barton & W. A. Hamilton (Eds.) (2007). Sulphate-Reducing Bacteria. Cambridge University Press.

were proposed to be the entrance of the substrate and the exit of the product, respectively. Two calcium ions are detected and one is very close to the active site.

The determination of the *D. desulfuricans* ATCC 27774 ccNir structure (*Cunha et al.*, 2003) enabled to ascertain the spatial characterization of the five hemes, namely, the axial histidine plane angles and their proximity (Fig. 10). A correlation between individual hemes obtained by spectroscopy (EPR and Mössbauer), with known reduction potentials, was undertaken (*Moura et al.*, 2007). Figure 9 represents the relationship between the heme core description and the spectroscopic and redox properties of each identified heme from the *D. desulfuricans* ATCC 27774 NrfHA complex.

Heme 1 (according to *D. desulfuricans* ATCC 27774 NrfA amino acid numbering) has the sixth axial position vacant. Thus, it is the site of substrate binding (high-spin heme, $-80 \, \text{mV}$). Heme 5 is reduced at a positive reduction potential ($+150 \, \text{mV}$) that is unusual for a heme with *bis*-His axial ligation. Due to its high reduction potential and heme solvent exposure, it was proposed that heme 5 is probably the site of electron entrance from the redox partner NrfH (Fig. 9). In the NrfH subunit, it was not possible to perform the structural assignment of the heme reduction and spectroscopic potentials, as there are no structures available for NrfH-like proteins.
A membrane-bound cytochrome \( c \) nitrite reductase has also been purified and characterized from *D. vulgaris* Hildenborough, a nonammonifying SRB. This membrane-associated complex of 760 kDa contains two cytochrome \( c \) subunits of 56 and 18 kDa and has both nitrite and sulfite reductase activities (Pereira et al., 2000). Even if this ccNir is bifunctional, the fact that it is more active in reducing nitrite than sulfite favors its physiological role as a detoxifying enzyme (Pereira et al., 2000).

### 2.2.1.2. Other Cytochromes

The hyperthermophilic sulfate-reducing archaeon, *A. fulgidus* (strain VC-6, DSM 4304), contained both \( b \)- and \( c \)-type cytochromes associated with the membrane fraction but not soluble cytochromes. The cytochrome content of the membrane extract was 3 nmol/mg protein, of which 2 nmol/mg protein were cytochrome \( b \) (Kunow et al., 1994). The reduction by reduced DMN (2,3-dimethyl-1,4-naphthoquinone) indicates that both cytochromes could have a function in the electron-transport chain mediating the reoxidation of the quinone reduced by the reduced coenzyme \( F_{420} \). A high-spin, membrane-bound dihemic cytochrome \( c \) was purified in three chromatographic steps after extraction from the membrane fraction and characterized from *Desulfomonile tiedjei* DCB-1 (Louie et al., 1997). *Desulfomonile tiedjei* is capable of dehalorespiration, obtaining energy for growth by coupling hydrogen or formate oxidation to reductive dehalogenation of 3-chlorobenzoate (DeWeerd et al., 1990). The molecular mass of this hemoprotein was about 50 kDa and the midpoint redox potential at pH 7.0 was \(-342\) mV (Louie et al., 1997). The purified *Desulfomonile tiedjei* dihemic cytochrome had a high-spin absorption spectrum, similar to those of cytochromes \( c' \) (Yamanaka, 1992). However, this cytochrome differs from cytochromes \( c' \) in heme content and molecular mass. This cytochrome was found to be coinduced with reductive dechlorination activity, suggesting it does function in the chlororespiration process (Louie and Mohn, 1999; Louie et al., 1997). Membranes of *D. vulgaris* Miyazaki F contain different types of cytochromes \( c \) (\( c_{550} \), \( c_{551} \), \( c_{556} \), tetraheme \( c_3 \)) and possibly \( d_1 \)- and \( o \)-type cytochromes (Ozawa et al., 1997).

### 2.2.2. Complex Hemoprotein: Fumarate Reductase from D. gigas

Fumarate reductase (FDR) (EC 1.3.1.6) is mostly a membrane-bound enzyme that catalyzes the reduction of fumarate to succinate. Thismultimeric protein is also able to catalyze the reverse reaction, succinate oxidation that is usually catalyzed by succinate dehydrogenase (SDH). Both enzymes share the activity, as well as cofactor (covalently bound flavin adenine dinucleotide (FAD), iron–sulfur centers, *b*-type hemes) and subunits’ (flavoprotein, iron–sulfur protein, membrane anchor subunits) composition ([Hagerhall, 1997](https://doi.org/10.1016/S0304-4165(97)00028-8)).

Fumarate reductase (FRD) is the major membrane-bound protein expressed in *D. gigas* fumarate grown cells (85% of the total FRD activity is present in the membrane fraction) ([Hatchikian and LeGall, 1972](https://doi.org/10.1016/0005-2728(72)90080-8); [Lemos et al., 2002](https://doi.org/10.1074/jbc.M109876200)). The FRD was solubilized, purified, and characterized (spectroscopic and redox studies) from *D. gigas* cells grown on a fumarate/sulfate medium at 37°C ([Lemos et al., 2002](https://doi.org/10.1074/jbc.M109876200)). The FDR contains three subunits (FdrA, FdrB, and FdrC) of apparent molecular masses of 71, 31, and 22 kDa. Studies by EPR and Visible spectroscopies allowed the identification of FAD, two *b*-type hemes, and the three canonical iron–sulfur centers present in most FRDs and SDHs ([2Fe–2S], [3Fe–4S], and [4Fe–4S]). The apparent macroscopic reduction potentials for the metal centers were determined by redox titrations at pH 7.6: +20 and −140 mV for the [3Fe–4S] and [2Fe–2S] clusters and −45 and −175 mV for the two *b*-type hemes. The redox potentials of the two heme groups are pH dependent, suggesting that fumarate reduction is probably associated with the generation of the membrane proton gradient ([Lemos et al., 2002](https://doi.org/10.1074/jbc.M109876200)). The *D. gigas* FRD is capable of both succinate oxidation and fumarate reduction, exhibiting a reduction rate approximately 30 times faster than that for oxidation and a higher specificity toward fumarate (*K*~m~ for succinate is 2 mM and for fumarate 0.02 mM). Cells of *D. gigas* are able to perform oxidative phosphorylation linked to electron transfer from molecular hydrogen to fumarate ([Barton et al., 1970](https://doi.org/10.1016/0005-2728(70)90128-6)).

*D. gigas* cells grown on fumarate express, besides the FRD, a cytochrome *bd* acting as a membrane-bound oxygen reductase ([Lemos et al., 2002](https://doi.org/10.1074/jbc.M109876200)). It seems probable that *D. gigas* cells are able to respire O₂ or, at least, to utilize these membrane-bound enzymes to scavenge oxygen, enabling to sustain the O₂-sensitive respiratory processes, such as dissimilatory sulfate reduction. An electron transfer chain obtained by coreconstitution in liposomes of *D. gigas* FRD, duroquinone (the most efficient artificial electron donor to cytochrome *bd*), and *D. gigas* cytochrome *bd* leads to succinate-driven oxygen consumption ([Lemos et al., 2002](https://doi.org/10.1074/jbc.M109876200)). *D. vulgaris* Hildenborough contained succinate dehydrogenase activity but lacked fumarate respiration. *D. desulfuricans* Essex 6 contained fumarate reductase activity and grew by fumarate respiration or by fumarate disproportionation ([Zaunmüller et al., 2006](https://doi.org/10.1016/j.femsre.2006.01.001)).
HEMOPROTEINS IN DISSIMILATORY REDUCTION 47

2.2.3. Membrane-Associated Electron Transport Complexes

It has long been apparent that the anaerobic sulfate- or sulfur-reducing bacteria lacked the respiratory complexes functioning as proton pumps as found in mitochondrial or aerobic bacterial systems. While plasma membranes of SRB contained lipophylic menaquinones as potential H⁺ and e⁻ carriers, a mechanism for pumping protons outward across the membrane was not understood until the first transmembrane redox complex was isolated. Characteristically, one of the protein subunits contains a multiheme c-type cytochrome, several [4Fe–4S] centers, and a transmembrane subunit has binding sites for two b-type hemes. Since these b-type hemes are not held in a protein by covalent linkages, it is easy for them to be lost with isolation of the complexes from the membranes. Several complexes are now reported for the SRP and several reviews (Keller and Wall, 2011; Pereira, 2008; Pereira et al., 2007, 2011) discuss their role in cell energetics.

2.2.3.1. The DsrKMJOP Complex

The dissimilatory reduction of sulfate has two reduction steps: reduction of APS to bisulfate by APS reductase and reduction of bisulfite to sulfide by dissimilatory sulfite reductase (Fig. 2). Both the APS reductase and dissimilatory sulfite reductase contain two subunits, AprAB and DsrAB, respectively. For sulfite reductase, there is an additional protein DsrC, and this cytoplasmic protein has been proposed to transfer electrons to DsrAB and DsrK (Oliveira et al., 2008a,b). DsrK is part of the Dsr complex (see Fig. 11) that transfers electrons from the periplasm or from reduced menaquinone to the sulfite reductase (Pires et al., 2006). As reviewed by Pereira et al. (2011), dsrC is highly conserved in the SRP and frequently located on SRB genomes near genes for production of the Dsr complex.

Almost all of SRP genomes thus far evaluated have the DsrMKJOP complex except C. maquilingensis and Gram-positive SRB (Junier et al., 2010). In A. fulgidus, the DsrMKJOP complex is called the Hme complex (Mander et al., 2002). The Dsr complex consists of five proteins and these are functionally similar to the Hmc complex. As reviewed by Pereira et al. (2007, 2011), the subunits of the Dsr complex are as follows: DsrM and DsrP proteins are found in the plasma membrane, DsrO is a ferredoxin-like protein located on the periplasm side of the membrane, DsrK is a Fe–S protein located on the cytoplasm side of the membrane, and DsrJ is a 3-heme cytochrome of 15 kDa. Each heme has a unique coordination

Figure 11 Electron transport complexes in the plasma membranes of sulfate reducers. Abbreviations: Dsr, DsrKMJOP complex; Qmo, QmoABC complex; Qrc, QrcABCD complex; Hmc, HmcABCDEF complex; Tmc, TmcABCD complex; Nhc, NhcABCD complex. Symbols: X, [4Fe–4S] center. The two vertical lines in DsrM, QmoC, HmcE, and TmcC indicate presence of two $b$-type hemes. Each line in a protein subunit in the periplasm represents the presence of a $c$-type cytochrome. Out, periplasm; In, cytoplasm.

(e.g., His/His, His/Met, and His/Cys) and the function of these hemes remains to be established.

### 2.2.3.2. The QmoABC Complex

The two-electron reduction step catalyzed by APS reductase would be attributed to an interface of the ApsAB with a specific membrane complex. The QmoABC (quinone-interacting, membrane-bound oxidoreductase) complex is present in most of the SRB genomes evaluated and is presumed to function in transfer of electrons from reduced menaquinone to the cytoplasmic ApsBA (Pires *et al.*, 2003). Menaquinone could be reduced by electron transfer from oxidation of $\mathrm{H}_2$ by periplasmic hydrogenase and perhaps the Qmo complex functions as a proton pump during sulfate respiration (Pereira *et al.*, 2011) (see Fig. 11). *Dst.* acetoxidans and *Candidatus Desulforudis* audaxviator lack some or all of the *qmo*

genes leading Pereira *et al.* (2011) to suggest that for some SRP the electron flow to ApsAB may not be coupled to proton pumping. QmoA (see Fig. 11) is a protein facing the cytoplasm and it contains FAD. QmoB is also a FAD-containing protein, is on the cytoplasmic side of the membrane, and has binding sites for two [4Fe–4S] centers. QmoC is an integral membrane protein that is capable of complexing with two *b*-type hemes and two [4Fe–4S] centers. There is considerable similarity between the QmoABC and HdrACE, the heterodisulfide reductase complex in methanogens. A comparison of Qmo proteins produced by sulfate and sulfur reducers is given in Table 9 and there is high level of similarity between *Desulfovibrio* sp., *Dsm.* *baculatum*, and *Desulfohalobium retbaense*. The levels of similarities of QmoABC are comparable to the similarities of AprAB in these organisms.

### 2.2.3.3. The QrcABCD Complex

The Qrc complex was first isolated from *D.* *vulgaris* Hildenborough (Venceslau *et al.*, 2010) and is similar to the Alternative Complex III (*bc*₁ complex) found in various bacteria. In terms of function, the Qrc (quinone reductase complex) operates in reverse of the Alternative Complex III with the Qrc complex transferring electrons from the periplasm to the quinone pool in the plasma membrane (Pereira *et al.*, 2011). This membrane complex is the first representative of a new family of bacterial respiratory complexes (BRC) belonging to the super family of complex iron–sulfur molybdoenzymes (CISM) proteins. The Qrc complex consists of four proteins (see Fig. 11) that are encoded on a gene cluster (*qrcA*, *qrcB*, *qrcC*, and *qrcD*). A 23 kDa cytochrome *c* with six binding sites for heme moieties is encoded on *qrcA*. A 72 kDa protein is encoded by *qrcB* and appears to be located in the periplasmic side of the membrane because it contains a twin-arginine signal peptide. The *qrcC* gene encodes for a 29 kDa redox protein that contains four iron–sulfur clusters that are positioned between quinone pool in the membrane and the catalytic center of the enzyme. A 48 kDa integral membrane protein is encoded on the *qrcD* and has 10 transmembrane helices. A detailed EPR characterization of the redox properties of the metal cofactors of Qrc complex was recently reported (Venceslau *et al.*, 2011), allowing the determination of the reduction potentials of the three [4Fe–4S] clusters and of five out of six hemes *c*. Two [4Fe–4S] clusters have a midpoint redox potential of –130 to –150 mV and the third one has a lower redox potential of –240 mV. The hemes’ midpoint redox potentials range from +100 to –130 mV. It has also been shown that Qrc forms a supercomplex with TpIc₃ and [NiFe]

Table 9 Amino acid sequence identity of Apr and Qmo proteins associated with sulfate and sulfur reducers using *D. vulgaris* Hildenborough as the reference protein.

| Organism                          | AprA   | AprB   | QmoA   | QmoB   | QmoC   |
|-----------------------------------|--------|--------|--------|--------|--------|
| % Identity                        |        |        |        |        |        |
| *Desulfovibrio vulgaris* RCH 1    | 100%   | 100%   | 100%   | 100%   | 100%   |
| *Desulfovibrio vulgaris* DP4      | 99.85  | 100    | 99.76  | 99.87  | 100    |
| *Desulfovibrio vulgaris* Miyazaki F | 93.67  | 91.62  | 87.62  | 78.63  | 83.33  |
| *Desulfovibrio desulfuricans* G20 | 91.55  | 92.11  | 84.22  | 73.82  | 82.55  |
| *Desulfovibrio desulfuricans*     | 89.31  | 89.47  | 68.52  | 65.83  | 64.38  |
| ATCC 27774                        |        |        |        |        |        |
| *Desulfomicrobium baculatum* X     | 82.20  | 83.55  | 71.53  | 59.61  | 60.46  |
| *Desulfovibrio magneticus* RS-1   | 82.53  | 88.96  | 70.07  | 62.71  | 60.15  |
| *Desulfovibrio salexigens* DSM 2638 | 82.96  | 83.44  | 67.48  | 60.86  | 58.78  |
| *Desulfobacterium autotrophicum*  |        |        |        |        |        |
| DSM 3382                          |        |        |        |        |        |
| *Desulfotalea psychrophila* LSv 54 | 64.02  | 70.50  | 53.88  | 45.92  | 40.50  |
| *Desulfurivibrio alkaliphilus* AHT 2 | 62.65  | 71.01  | 53.14  | 46.93  | 39.44  |
| *Desulfobacterium autotrophicum*  | 60.38  | 63.19  | 52.91  | 43.73  | 40.64  |
| DSM 3382                          |        |        |        |        |        |
| *Desulfococcus oleovorans* Hxd 3  | 61.42  | 62.69  | 54.83  | 42.38  | 40.10  |
| *Desulfarculus baarsii* DSM 2075  | 50.08  | 63.12  | 48.67  | 42.36  |        |
| *Desulfatibacillum alkenivorans*  | 64.23  | 55.21  | 43.97  | 40.29  |        |
| AK-01                             |        |        |        |        |        |
| *Desulfotomaculum acetoxidans*    | 49.53  | 65.41  | 52.90  | 40.66  |        |
| DSM 771                           |        |        |        |        |        |
| *Desulfotomaculum reducens* MI-1  | 49.31  | 64.38  | 50.97  | 43.23  |        |
| *Archaeoglobus fulgidus* DSM 4304 | 49.23  | 64.08  | 51.70  | 42.34  | 37.21  |
| *Archaeoglobus profundus*         | 69.40  | 48.30  | 43.25  |        | 33.58  |
| DSM 5631                          |        |        |        |        |        |
| *Thermodesulfovibrio yellowstonii* | 47.84  | 70.00  | 55.37  | 41.61  | 38.02  |
| DSM 11347                         |        |        |        |        |        |
| *Methanosarcina barkerii*         |        | 31.04  | 23.27  |        |        |

HEMOPROTEINS IN DISSIMILATORY REDUCTION

2.2.3.4. HmcABCDEF, NhcABCD, and TmcABCD Complexes

HmcABCDEF refers to the membrane complex in *D. vulgaris* Hildenborough containing a high-molecular weight cytochrome $c$ (*Rossi et al.*, 1993). As discussed by *Pereira et al.* (2007, 2011), this multiheme cytoplasmic protein accepts electrons from periplasmic hydrogenases and either directs electrons into the cytoplasm or to the menaquinone. HmcA is a 16-heme cytochrome with an unusual arrangement of hemes in the molecule (*Fig. 11*). HmcA has four $c_3$ domains with one domain having only three hemes; the remaining three domains have four hemes each and an additional heme is positioned between the third and fourth domain. HmcB spans the membrane with a domain on the periplasmic side with four [4F4–4S] centers and the C-terminal tail of the protein subunit extends into the cytoplasm. HmcC, HmcD, and HmcE are integral membrane proteins, with HmcE binding two $b$ hemes. HmcF is on the cytoplasmic side and has two 4Fe–4S centers. With syntrophic growth by *D. vulgaris*, electrons are suggested to be transferred from the cytoplasm to the periplasm by way of the 16-heme cytochrome (*Walker et al.*, 2009).

An electron transport membrane complex containing a nine-heme $c$-type cytochrome (Nhc) was initially identified by *Saraiva et al.* (2001) and was designated as 9hcA–D. The NhcA subunit contains the nine-heme cytochrome $c$; the NhcB contains four [4Fe–4S] centers; the NhcC subunit is an integral membrane protein containing two $b$-hemes; and the NhcD is a hydrophobic protein of unknown function (*Fig. 11*). The NhcABCD complex is more limited in distribution than the HmcABCDEF complex and in some organisms the nine-heme cytochrome is replaced by a cytochrome containing 13 hemes (*Pereira et al.*, 2011). A related complex, Ohc, contains an eight (octa)-heme cytochrome and is also found in some SRB. Both the Nhc and the Ohc complexes are homologous to Hmc and Tmc.

The TmcABCD complex is similar to the HmcABCDEF complex in that it transfers electrons from periplasmic hydrogenases to the cytoplasm. The Tmc complex contains two TmcA subunits on the periplasmic side of the membrane with each subunit containing a tetraheme type II cytochrome $c_3$ (*Pereira et al.*, 2006). TmcB and TmcD are cytoplasmic proteins with four [4Fe–4S] centers in TmcB. TmcC is an integral membrane protein that contains two heme $b$ moieties (*Fig. 11*).

2.2.3.5. The HmeCD, HmeABCDE, HdrDE, and HdrABC Complexes

In the archaeal sulfate reducers, several genes have been identified that are similar to heterodisulfide reduction in methanogens (*Klenk et al.*,

From gene cluster examination, it is predicted that distinct protein subunits are present for *A. profundus* and *A. fulgidus*. The complex HmeABCDE from *A. fulgidus* is presented (see Fig. 12) with four subunits (HmeACDE) isolated and HmeB predicted from gene cluster analysis (*Mander et al.*, 2004). HmeC is a $b$-type cytochrome; HemD contains a sequence similar to the Cx$_{31-38}$CCx$_{33,34}$Cx$_{2}$C motif characteristic for Hdr complex of *Ms. barkeri*. HmeA and HemC are considered to be present on the cytoplasmic side of the membrane. HmeCD isolated from *A. profundus* contains two distinct subunits (see Fig. 12) and is similar in structure to HdrDE of *Methanosarcina* sp. (*Mander et al.*, 2004). While HdrDE of *Methanosarcina* sp. interfaces with methanophenazine in the membrane, HmeCD interacts with a modified menaquinone.

---

**Figure 12** Heterodisulfide complexes from *Archaeoglobus* sp. and methanogens. HmeCD complex is from *A. profundus*. HmeA(B)CDE complex is from *A. fulgidus*. In *A. fulgidus*, subunits of HmeAE are on the periplasmic side of the membrane and HmeB is proposed from genome analysis but is not yet isolated. HdrDE complex is from *Methanosarcina* sp. and HdrABC complex is in the cytoplasm of *Methanothermobacter marburgensis*. Mvh, soluble hydrogenase and part of the Mvh:Hdr complex. Symbols: X, [4Fe–4S] center. The two vertical lines in HmeC and HdrE indicate presence of two $b$-type hemes. The three lines in periplasmic subunit of HmeABCDE represent presence of three $c$-type hemes. Out, periplasm; In, cytoplasm.

Heterodisulfide reductase-like proteins are found in methanogens. The HdrABC proteins are found in the cytoplasm of methanogens that do not contain cytochromes and HdrDE is found in methanogens with cytochromes (Hedderich *et al.*, 2005; Thauer *et al.*, 2008). The genome of Methanothermobacter thermoautotrophicus contains three different regions that contain the six subunits of the Mvh:Hdr complex (Smith *et al.*, 1997). Mander *et al.* (2004) reveal that the ORF2976-2967 of *D. vulgaris* may be the six-gene complex for a H₂: heterodisulfide oxido-reductase complex that is similar to the Mvh:Hdl complex of *A. profundus*. It has been reported by Pereira *et al.* (2011) that Hdr-like proteins of HdrA and HdrD families are found in several SRO. Although a function for a heterodisulfide complex is not understood in SRB, it may, however, be that the HdrA complex is energized by hydrogenase or pyruvate metabolism by Por (pyruvate oxido-reductase) (Pereira *et al.*, 2011) and by the oxidation of ethanol to acetate (Haverman *et al.*, 2003) and is part of the H₂-cycling system in *D. vulgaris*. On the other hand, the HdrD-containing complexes of SRO may be energized by lactate oxidation, β-oxidation, and other systems and the HdrD complexes could be electron donors for reduction sulfite by reduction of menaquinones or DrsC (Pereira *et al.*, 2011).

### 2.2.3.6. Charging of Membranes

The SRB are unique in that energy in the form of ATP is required for respiratory-based electron transfer to occur. As discussed earlier, the activation of sulfate to produce adenylyl sulfate requires two moles of ATP for each mole of sulfate activated (Ishimoto, 1959; Peck, 1959). Peck proposed that SRB could obtain ATP by oxidative phosphorylation means at sufficient level for SRB to overcome the ATP demands in sulfate activation coupled to dissimilatory sulfate reduction (Peck, 1960) and later in a hallmark publication (Peck, 1966) reported the measurement of ATP levels (P/2e⁻) with H₂ as the electron donor and sulfite or thiosulfate as the electron acceptor. The P/2e⁻ levels with the anaerobic SRB were comparable to those reported by others for aerobic bacteria. In subsequent reports, anaerobic oxidative phosphorylation measurements with *D. gigas* were recorded for H₂ oxidation and fumarate (Barton *et al.*, 1970, 1972), nitrite (Barton *et al.*, 1983), or elemental sulfur (Fauque *et al.*, 1980) as electron acceptors. With membrane fractions from *D. gigas*, Klemm and Barton (1989) demonstrated the ATP synthesis coupled to the flow of electrons from oxidation of protoporphyrinogen to reduction of nitrite. Similarly, anaerobic oxidative phosphorylation was reported in the H₂→fumarate system in the sulfur-reducing bacterium, Wolinella succinogenes (Reddy

and Peck, 1978). These papers provided the first evidence for the functioning of proton pumping and proton-driven ATP synthase activity in anaerobic bacteria and conformed to the chemiosmotic model formulated by Peter Mitchell (1979) and adopted to SRB by Wood (1978). Genes for the F-type ATP synthase genes are identified in the genomes of sulfate- and sulfur-reducing microbes and evidence has been presented to indicate the involvement of the $F_{0}F_{1}$ in sulfate respiration by *D. vulgaris* Miyazaki F (Ozawa et al., 2000).

A hallmark characteristic of a living cell is maintaining a charge of 100–200 mV on the plasma membrane (Mitchell, 1979) and this is needed to maintain the viability of SRP. A model of sulfate respiration in *Desulfovibrio* species and proteins associated with the plasma membrane supporting a membrane charge are given in Fig. 13. Information from Keller and Wall (2011), Walker et al. (2009), Pereira et al. (2007), and Pereira et al. (2011) are used to construct this model. Electron donors are principally H₂, formate, pyruvate, or lactate and electrons are shuttled through membrane complexes to the cytoplasm where sulfate is reduced. Electrons from Qmo are used for the reduction of APS to sulfite and electrons from Dsr account for the production of sulfide from sulfite. Both Qmo and Dsr are membrane complexes that could acquire electrons from reduced menaquinones. Electrons from the oxidation of H₂ or formate reduce metaquinones with the assistance of protein complexes of Qrc, Hmc, and Tmc. An alternate system of electron movement was proposed by Odom and Peck (1981) and they suggested the cycling of hydrogen. The release of H₂ from pyruvate oxidation with sulfate reduction was observed by Peck et al. (1987) and potentially this release of H₂ could be mediated by the Coo and Ech complexes. This system does not consider the role of flavodoxin, but perhaps flavodoxin would bypass the coupling of electrons from H₂ or organic acid to charging the membrane. The coupling of ATP synthesis to H₂ oxidation and sulfite reduction with membranes of *D. gigas* occurred if ferredoxin was supplied but not if flavodoxin was added (Barton et al., 1972). However, flavodoxin production occurs in *Desulfovibrio* species only under limited iron nutrition, otherwise ferredoxin is produced if iron is available (Peck and LeGall, 1982).

## CYTOCHROMES FROM SULFUR-REDUCING EUBACTERIA

### 3.1. Monoheme Cytochromes

The membrane fraction from W. succinogenes ATCC 29543 (formerly *Vibrio succinogenes*) contains two $b$- and two $c$-type cytochromes.

HEMOPROTEINS IN DISSIMILATORY REDUCTION 55

![Diagram](attachment:diagram.png)

Figure 13 Model reflecting organization of protein complexes associated with electron transport and cell energetics in SRB. See legend of Figure 11 for Qmo, Drc, Dsr, Tcm, and Hmc. Abbreviations are as follows: ATPS, proton driven ATP synthase; Coo, carbon monoxide dehydrogenase-hydrogenase membrane complex system; Ech, multisubunit membrane-bound hydrogenase; Fd, ferredoxin; Fdh, formate dehydrogenase; Hyd, periplasmic [Fe] hydrogenase; Hyn, periplasmic [NiFe] hydrogenase; Hys, periplasmic [NiFeSe] hydrogenase; Ohc, octaheme cytochrome $c$ membrane complex; Rnf, NADH:quinone oxidoreductase membrane complex; Tplc$_3$, periplasmic type I cytochrome $c_3$. Cytoplasmic enzymes are as follows: Ack, acetate kinase; Aps, adenylylsulfate reductase; Dsr, dissimilatory sulfite reductase; Ldh, lactate dehydrogenase; Pfl, pyruvate formate lyase; Por, Pyruvate:ferredoxin oxidoreductase; Pta, phosphotransacetylase; Sat, ATP-sulfurylase.

midpoint redox potentials of the cytochromes $b$ are $-20$ and $-200$ mV and those of the cytochromes $c$ are $+70$ and $-160$ mV (Kroger and Innerhofer, 1976). The two cytochromes $b$ interact in the formate-fumarate reductase system, whereas the two cytochromes $c$ do not seem to be involved in this electron transport pathway.

A high-potential $c$-type monoheme cytochrome was purified and spectroscopically characterized from W. succinogenes VPI 10659 (Moura et al., 1988a). It has a molecular mass of 8.2 kDa and a midpoint redox potential of $+100$ mV. Visible light spectroscopic studies suggested the presence of
an equilibrium between two ligand arrangements around the heme, that is, an absorption peak at 695 nm characteristic of a heme–methionine coordination (low-spin form) coexisting with a high-spin form revealed by an absorption band at 619 nm (Moura *et al.*, 1988a). A small molecular mass periplasmic $c$-type cytochrome has also been isolated from *Dsf.* *acetoxidans*; this 10 kDa hemoprotein is the first monohemic cytochrome $c$ exhibiting a bis-histidyl axial coordination and a low redox potential of –220 mV (Bruschi *et al.*, 1997).

### 3.2. Multiheme Cytochromes $c$

#### 3.2.1. Cytochromes from W. succinogenes

The genome sequence of *W.* *succinogenes* DSMZ 1740 encodes 23 endogenous cytochromes $c$. Twelve of these hemoproteins are multiheme cytochromes $c$ harboring between two and eight heme $c$-binding motifs (Baar *et al.*, 2003). Cells of *W.* *succinogenes* grown by anaerobic respiration with fumarate, polysulfide, or nitrate as terminal electron acceptor revealed different cytochrome $c$ patterns (Simon *et al.*, 2000). The pentaheme cytochrome $c$ nitrite reductase NrfA is the prominent cytochrome $c$ during dissimilatory nitrate reduction (Kern and Simon, 2011; Kern *et al.*, 2010). NrfA reduces nitrite to ammonium in the periplasmic space and is anchored to the membrane via NrfH, the tetraheme cytochrome $c$ (Einsle *et al.*, 2000; Kern and Simon, 2008).

A diheme cytochrome $c$ was purified from the *W.* *succinogenes* crude cell extract. This protein has a molecular mass of 11 kDa and the visible absorption spectrum exhibited no band at 695 nm, suggesting that methionine is not a ligand for either of the two hemes (Moura *et al.*, 1988a).

#### 3.2.2. Cytochrome $c_7$ from Desulfuromonas (*Dsf.*) acetoxidans

The final draft genome of *Dsf.* *acetoxidans* codes for a “cytochromome” of 47 putative multiheme cytochromes $c$ (Alves *et al.*, 2011). The cytoplasmic fraction of *Dsf.* *acetoxidans* strain 5071(DSM 1675) contains large amounts of low-spin $c$-type cytochromes and the membrane fraction contains minor amounts of $b$-type cytochromes (Bache *et al.*, 1983). The strain 5071 of *Dsf*.
acetoxidans contains several multiheme $c$-type cytochromes, the most abundant being the three-heme cytochrome $c_7$.

The cytochrome $c_7$ (also known as triheme cytochrome $c_3$ or cytochrome $c_{551.5}$) is the only member of the cytochrome $c_3$ family that is absent in SRB. This trihemic protein is found in members of the Geobacteraceae family like *Geobacter metallireducens*, *G. sulfurreducens*, and *Dsf. acetoxidans* which perform dissimilatory iron reduction (Lloyd, 2003). Cytochrome $c_7$ of *Dsf. acetoxidans* is the smallest representative of the multiheme cytochrome $c_3$ family with three heme groups and 68 amino acids (Bruschi, 1994). The sequence of *Dsf. acetoxidans* cytochrome $c_7$ has been aligned with those of some *Desulfovibrio* tetraheme cytochromes $c_3$ (Bruschi, 1994). The *Dsf. acetoxidans* trihemic cytochrome $c_7$ is rapidly reduced by the *D. vulgaris* strain Hildenborough [Fe] hydrogenase and it can completely reduce polysulfide with a very high specific activity of 20 μmol of hydrogen consumed per minute per milligram of protein (Pereira et al., 1997). This activity is twice as high as that reported for the purified tetrahemic cytochrome $c_3$, with the highest specific sulfur reductase activity from *Dsm. baculatum* DSM 1743 (Fauque, 1994; Fauque et al., 1979a). These results seem to indicate that cytochrome $c_7$ is probably the true terminal sulfur reductase in *Dsf. acetoxidans* but it remains to find the physiological electron donor for this trihemic protein. The *Dsf. acetoxidans* triheme cytochrome $c_7$ expressed in *D. desulfuricans* G201 retains its metal reductase activity (Aubert et al., 1998b).

The kinetic and thermodynamic characterization of *Dsf. acetoxidans* triheme cytochrome $c_7$ indicates that electrons enter the cytochrome mainly via heme I (Correia et al., 2002). The crystal structure of *Dsf. acetoxidans* fully oxidized triheme cytochrome $c_7$ has been solved at 1.9 Å resolution (Czjzek et al., 2001). A resonance Raman spectroscopic study of *Dsf. acetoxidans* triheme cytochrome $c_7$ reveals an analogous redox behavior but a less pronounced stability than that of tetrahemic cytochromes $c_3$ from *Desulfovibrio* species (Chottard et al., 2000). A comparison (by proton-NMR characterization) between the fully oxidized and reduced forms of *Dsf. acetoxidans* triheme cytochrome $c_7$ shows that the structure for the two redox forms is quite similar, confirming the special structural features of the three-heme cluster (Assfalg et al., 1998, 1999; Banci et al., 1996). The periplasmic triheme cytochrome $c_7$ plays a central role in the bioenergetic metabolism of *Dsf. acetoxidans*, given its abundant presence in all growth conditions tested (fumarate, elemental sulfur, iron-citrate as terminal electron acceptors) (Alves et al., 2011).

3.2.3. Other c-Type Multiheme Cytochromes from Des. acetoxidans

Two high-molecular mass soluble c-type cytochromes have been purified and characterized from *Dsf. acetoxidans* DSM 1675: a 50 kDa cytochrome, which is reported to contain either four (Bruschi *et al.*, 1997) or six hemes (Pereira *et al.*, 1997) and a 65 kDa cytochrome which may contain eight hemes (Pereira *et al.*, 1997). Visible and EPR spectroscopies of the oxidized cytochromes show the presence of both low-spin and high-spin hemes in the 65 kDa cytochrome and of only low-spin hemes in the 50 kDa cytochrome (Pereira *et al.*, 1997). Both cytochromes cover a wide range of redox potentials, which go from +185 to −235 mV for the 65 kDa cytochrome and from +100 to −375 mV for the 50 kDa cytochrome (Pereira *et al.*, 1997). Both cytochromes did not display hydroxylamine oxidoreductase and polysulfide reductase (Psr) activities (Bruschi *et al.*, 1997). The resonance Raman spectroscopic data confirms that the *Dsf. acetoxidans* 50 kDa cytochrome is an entirely low-spin species with two different sets of axial ligands (Chottard *et al.*, 2000).

These two c-type cytochromes purified and characterized from *Dsf. acetoxidans* belong to the members of the interesting family of low redox potential multiheme c-type cytochromes from sulfur- and SRO. Some of the hemoproteins from this family possess enzymatic functions whereas some others act only as electron carriers. Further work will be necessary to elucidate the possible physiological function of these two cytochromes c in the metabolism of *Dsf. acetoxidans*.

3.2.4. Cytochrome c Nitrite Reductase from Wolinella and Sulfurospirillum

Cytochrome c nitrite reductase (ccNiR) is a pentahemoprotein catalyzing the six-electron reduction of nitrite to ammonia as the final step in the dissimilatory pathway of nitrate ammonification (Einsle *et al.*, 1999, 2002a; Kern and Simon, 2011; Schumacher and Kroneck, 1991). Two potential reaction intermediates, such as NO or hydroxylamine, are not released during this reaction (Einsle *et al.*, 2002a). The ccNir is also able to reduce sulfite to sulfide, thus forming the only known link between the biogeochemical cycles of sulfur and nitrogen. The sulfite reductase activity of *W. succinogenes* ccNir is significantly smaller than its nitrite reductase activity but still several times higher than the one reported for dissimilatory, siroheme-containing sulfite reductases (Lukat *et al.*, 2008). The binding mode of sulfite and nitrite to the catalytic heme center of ccNiR from *W. succinogenes* was determined at a resolution of 1.7 Å. Nitrite and sulfite

both provide a pair of electrons to form the coordinative bond to the Fe (III) active site of the ccNir, and the oxygen atoms of sulfite to interact with the three active site protein residues conserved within the enzyme family (*Lukat et al.*, 2008).

Based on sequence comparison and secondary structure prediction, it has been demonstrated that cytochrome *c* nitrite reductases represent a protein family of high structural similarity. The crystal structures of NrfA from the *epsilon*-proteobacteria *S. deleyianum* DSM 6946 (*Einsle et al.*, 1999) and *W. succinogenes* (*Einsle et al.*, 2000) show that both enzymes form a stable dimer with a total of ten heme groups and two independent active sites. The crystallographic characterization of NrfAs from *W. succinogenes* (*Einsle et al.*, 2000), *E. coli* K 12 (*Bamford et al.*, 2002), *D. desulfuricans* ATCC 27774 (*Cunha et al.*, 2003), and *S. deleyianum* (*Einsle et al.*, 1999) allows structurally conserved regions to be identified. Comparison of nitrite and sulfite reductase activities from different species of bacteria shows that the relative activities vary according to the microorganism.

While NrfA is the catalytically active subunit of the ccNir system, its function *in vivo* is linked to proton translocation across the cytoplasmic membrane. In *W. succinogenes*, this function is fulfilled by a small peripheral membrane protein, NrfH (itself is a tetraheme cytochrome *c*), that functions as a quinol oxidase (*Simon et al.*, 2000). NrfH transfers electrons from the membranous quinone pool to NrfA and it is presumed that in this process protons are translocated in a Q-cycle-like mechanism. NrfH is a member of the NapC/NirT family of multi-heme cytochromes and as such a prototype of a whole family of membrane-linked quinol oxidases (*Kern and Simon*, 2011; *Kern et al.*, 2008). In *W. succinogenes*, NrfH and NrfA form a stable complex, of which diffracting crystals have been obtained (*Einsle et al.*, 2002b).

### 3.2.5. Cytochromes from Methanogenic Bacteria

Methanogenic archaea constitute a phylogenetically diverse group of strictly anaerobic Euryarchaeota with an energy metabolism that is restricted to the production of methane from CO₂ + H₂, methanol, methylamines, formate, and/or acetate (*Thauer*, 1988). Five orders of methanogens have been identified: Methanosarcinales, Methanococcales, Methanomicrobiales, Methanopyrales, and Methanobacterales. A recent review by Thauer and colleagues compared the physiological and biochemical properties of methanogenic archaea with and without cytochromes (*Thauer et al.*, 2008). Methanogenic archaea with cytochromes all belong to the order of
Methanosarcinales including the *Methanosarcina*, *Methanobus*, and *Methanoaeta* genera. All members of the Methanosarcinales order also have a broad substrate spectrum and contain methanophenazine (a functional analogue of menaquinone) (*Abken et al.*, 1998). Methanogens with cytochromes have a much higher growth yield on CO₂ + H₂ and a higher threshold concentration for H₂ than methanogens without cytochromes. Methanogenic archaea with cytochromes contain no hyperthermophilic species and have doubling times generally higher than 10 h (*Thauer et al.*, 2008).

The reduction of carbon dioxide with hydrogen to methane and energy conservation in methanogenic archaea with cytochromes is considered to involve 13 reactions in which at least four electron carriers and six membrane-associated protein complexes are implicated (*Thauer et al.*, 2008). In methanogens containing cytochromes, the first and last steps in methanogenesis from CO₂ are coupled chemiosmotically, whereas in methanogenic archaea without cytochromes, these two steps are energetically coupled by a cytoplasmic enzyme complex that mediates flavin-based electron bifurcation (a coupling mechanism that was recently discovered in Clostridia such as *Clostridium kluyveri* which does not contain ubiquinone, menaquinone, or cytochromes (*Herrmann et al.*, 2008; *Thauer et al.*, 2008; *Wang et al.*, 2010).

A number of unusual and unknown coenzymes and electron carriers (coenzymes F₄₂₀ and F₄₃₀, methanopterin, methanofuran, methanophenazine) were discovered in methanogenic bacteria (*Blaut*, 1994; *Deppenmeir et al.*, 1996; *DiMarco et al.*, 1990; *Thauer*, 1988; *Thauer et al.*, 2008; *Van Beelen et al.*, 1983). As a consequence methanogens were considered to lack classical electron carriers such as quinones and cytochromes. The first report on the presence of a cytochrome in methanogenic bacteria was the mention of a cytochrome b₅₅₉ in membranes of *Ms. barkeri* strain Fusaro DSM 804 (*Kuhn et al.*, 1979). The cells of *Ms.* “vacuolata” strain Zilina (DSM 1232), the thermophilic methanogen strain TM-1 (DSM 1825), and *Ms. barkeri* strains MS and Fusaro grown on methanol and methylamines contain at least two membrane-bound cytochromes b and one membrane-bound cytochrome c (in lower amount) (*Kuhn and Gottschalk*, 1983).

Two membrane-linked b-type cytochromes, with midpoint potentials of −325 and −183 mV, were characterized in cells of *Ms. barkeri* strain Fusaro grown on methanol or trimethylamine (*Kuhn and Gottschalk*, 1983). An additional cytochrome b formed during growth on acetate exhibited a redox potential of −250 mV (*Kuhn and Gottschalk*, 1983). A membrane-bound cytochrome b is present in *Mc. mazei* DSM 2053 and *Methanothrix soehngenii* contains equal amounts of membrane-linked b- and c-type cytochromes (*Kuhn et al.*, 1983). Methanococcoides
methylutens and *Methanobus tindarius* were reported to contain a cytochrome $c$ in concentrations of 0.306 and 0.189 $\mu$mol/g of membrane protein, respectively (Jussofie and Gottschalk, 1986). A membrane-bound cytochrome $b$ was also present in much smaller amounts (Jussofie and Gottschalk, 1986). Optical spectroscopic studies demonstrated that *Ms. barkeri* membrane preparations contained hydrogenase and a cytochrome $b$ reduced by molecular hydrogen and oxidized by methyl coenzyme M (Kemmer et al., 1987). This suggests a putative role for hydrogenase and cytochrome $b$ in methyl-group reduction and electron transfer during aceticlastic methanogenesis.

Two cytochromes $b$ ($b_{559}$ and $b_{564}$) and two cytochromes $c$ ($c_{547}$ and $c_{552}$) were reported in membranes of *Methanosarcina* strain Go1 DSM 3647 grown on methanol (Kamlage and Blaut, 1992). The midpoint potentials determined were $-135$ and $-240$ mV (cytochromes $b$) and $-140$ and $-230$ mV (cytochromes $c$). Electron transfer studies suggest the participation of one or several cytochromes in the coenzyme F$_{420}$H$_2$-dependent reduction of the heterodisulfide (Kamlage and Blaut, 1992). A cytochrome $bc$ was partially purified and characterized from *Ms. barkeri* strain MS; it is composed of three subunits (Kumazawa et al., 1994). It differs from cytochrome $bc_1$ of other microorganisms; the molecular mass of the subunit containing heme $c$ (23.4 kDa) is smaller than that of cytochrome $c_1$ (30 kDa) (Trumpower, 1990). Cytochromes $b$ and $c$ function as electron carriers during methanogenesis from acetate, methanol, and methylamines. These hemoproteins are implicated in the oxidation of these substrates during their disproportionation (Terlesky and Ferry, 1988).

Some species of methanogens, such as *Methanobacterium* (*Mb.*) thermoaotrophicum, *Ms. barkeri*, *Methanococcus* (*Mc.*) thermolithotrophicus, and *Methanobrevibacter ruminantium*, grow well with sulfite, thiosulfate, or elemental sulfur as the sole sulfur source (LeFaou et al., 1990; Stetter and Gaag, 1983). Sulfite and thiosulfate reductase activities have been detected in the crude extracts of *Mc.* thermolithotrophicus, *Mb.* thermoaotrophicum strain Marburg, and *Ms. barkeri* strains MS and 227 (LeFaou et al., 1990). An elemental sulfur reductase activity is present in the crude extracts of *Mb.* thermolithotrophicus, *Ms. barkeri* strain 227, *Methanospirillum hungatei*, and *Mb.* thermoaotrophicum Marburg and this activity appears to be constitutive (LeFaou et al., 1990). Dissimilatory sulfur reduction appears to be a primitive means of energy conservation and may have been the forerunner of energetically more efficient methanogenesis (Stetter and Gaag, 1983).

3.3. Sulfite Reductases

3.3.1. Low-Spin Assimilatory Sulfite Reductase from Ms. barkeri and Dsf. acetoxidans

Two low-molecular mass hemoproteins with sulfite reductase activity (named P₅₉₀) have been purified from Dsf. acetoxidans strain 5071 (DSM 1675) and Ms. barkeri DSM 800 (Moura and Lino, 1994; Moura et al., 1982, 1986). They were characterized by optical spectroscopic and EPR techniques. Both proteins present visible spectra similar to that of the low-spin assimilatory sulfite reductase of D. vulgaris Hildenborough; in particular there is no absorption band at 715 nm, characteristic of high-spin Fe³⁺ complexes in isobacteriochlorins. The Ms. barkeri P₅₉₀ has a molecular mass of 23 kDa and its optical visible spectrum exhibits maxima at 590, 545, and 395 nm (Moura et al., 1982). The Dsf. acetoxidans sulfite reductase has a molecular weight of 23.5 kDa and exhibits bands at 587, 545, and 405 nm (Moura et al., 1986). EPR spectra show that, as isolated, the siroheme is in a low-spin ferric state (S=1/2) with g-values at 2.44, 2.33, and 1.81 for the Dsf. acetoxidans enzyme and g-values at 2.40, 2.30, and 1.88 for the Ms. barkeri P₅₉₀ (Moura et al., 1986). Chemical analysis shows that both proteins contain one siroheme and one very negative redox potential [4Fe–4S] center by polypeptidic chain (Moura et al., 1986). The specific sulfite reductase activity measured at pH 6.0 using pure periplasmic D. gigas hydrogenase was 2790 mU/mg of protein for the Ms. barkeri P₅₉₀ and 906 mU/mg of protein for the Dsf. acetoxidans enzyme (Moura and Lino, 1994; Moura et al., 1986). The Ms. barkeri P₅₉₀ has a higher specific sulfite reductase activity than that of the dissimilatory type sulfite reductases from SRB such as desulfovirdin, desulforubidin, and desulfofuscidin (Czechowski et al., 1986). The two assimilatory-type sulfite reductases contain five iron atoms and five labile sulfur atoms; it was postulated that the extra sulfur atom could be the bridging ligand between siroheme and the [4Fe–4S] cluster (Moura and Lino, 1994). Both proteins catalyze the direct six-electron reduction of sulfite to sulfide without the formation of free intermediates. The Dsf. acetoxidans sulfite reductase is not involved in the dissimilatory sulfur reduction occurring in this organism. The Ms. barkeri P₅₉₀ could be utilized in a reverse manner to provide sulfur at the proper oxidation level for the biosynthesis of coenzyme M which contains a sulfonate group (Moura et al., 1982).

HEMOPROTEINS IN DISSIMILATORY REDUCTION

3.3.2. Octaheme Cytochrome c from She. oneidensis: A New Class of Dissimilatory Sulfite Reductase

The facultatively anaerobic mesophile gammaproteobacterium *She. oneidensis* strain MR-1 (ATCC 700550<sup>T</sup>) is a metal reducer with a very large respiratory versatility using fumarate, insoluble metal oxides, O<sub>2</sub>, trimethylamine N-oxide (TMAO), dimethyl sulfoxide (DMSO), nitrite, and nitrate as terminal electron acceptors for respiration (*Nealson and Saffarini*, 1994; *Venkateswaran et al.*, 1999). *She. oneidensis* and *She. putrefaciens* strain MR-4 are also able to utilize elemental sulfur, sulfite, thiosulfate, and tetrathionate (but not sulfate) as terminal electron acceptors (*Burns and DiChristina*, 2009; *Moser and Nealson*, 1996; *Perry et al.*, 1993). Under anaerobic conditions, formate and lactate (which is incompletely oxidized into acetate) can be utilized as sole energy and carbon sources by *She. oneidensis* (*Scott and Nealson*, 1994; *Tang et al.*, 2007). The mechanisms utilized to reduce sulfur compounds have been investigated in *She. oneidensis*. PsrABC was identified as the polysulfide and thiosulfate reductase in this strain (*Burns and DiChristina*, 2009). The mechanism of sulfite reduction was very recently investigated in *She. oneidensis* MR-1 and it was shown that the dissimilatory sulfite reductase catalyses the direct six-electron reduction of sulfite to sulfide without formation of thiosulfate and trithionate (*Shirodkar et al.*, 2011). The dissimilatory sulfite reductase is not a siroheme-containing enzyme, but it represents a new class of sulfite reductases. The terminal sulfite reductase was identified as a periplasmic octaheme c-type cytochrome with an atypical heme-binding site (*Shirodkar et al.*, 2011) and it was designated SirA (previously named as MccA; *Hartshorne et al.*, 2006). The SirA of *She. oneidensis* is the first member of the MccA family to be functionally characterized. The sulfite reductase SirA of *She. oneidensis* does not play a role in dissimilatory nitrite reduction. This enzyme does not share sequence similarities with either the NrfA nitrite reductases or the siroheme sulfite reductases (*Shirodkar et al.*, 2011).

3.3.3. Sulfite Reductase from P. islandicum

Some of the sulfur-respiring prokaryotes, most of which are hyperthermophilic archaea, are able to utilize thiosulfate and sulfite as additional electron acceptors (*Schauder and Kröger*, 1993; *Widdel and Hansen*, 1992). The hyperthermophilic crenarchaeote *P. islandicum* DSM 4184<sup>T</sup> (optimum
growth temperature of 100 °C) is a representative of this group of microorganisms (Huber *et al.*, 1987). Elemental sulfur is essential as an electron acceptor for chemolithotrophic growth of *P. islandicum* on molecular hydrogen; however, this organism can also grow on oxidized glutathione, cystine, thiosulfate, and sulfite as hydrogen acceptors. Only a few studies have been done on the enzymes catalyzing the reduction of sulfite and thiosulfate in these archaea. *P. islandicum* has been chosen to verify if a protein similar to siroheme sulfite reductase is present in sulfite- and thiosulfate-reducing prokaryotes which is not able to reduce sulfate. A soluble dissimilatory siroheme–sulfite reductase-type was purified from *P. islandicum* grown chemoorganoheterotrophically with thiosulfate as terminal electron acceptor (Molitor *et al.*, 1998). A green protein has been purified in four chromatographic steps and 3 mg electrophoretically homogeneous protein was obtained per 10 g cells. The relative molecular mass of the native protein is 170 kDa and this enzyme consists of two different subunits (molecular mass of 40 and 42 kDa), indicating that the hemoprotein is a tetramer with a quaternary structure of $\alpha_2\beta_2$ (Dahl *et al.*, 2001). The isoelectric point of pH 8.4 is unique among the DSR characterized so far (they all have an acidic isoelectric point). The oxidized protein exhibits absorption maxima at 280, 392, 578, and 710 nm (Dahl *et al.*, 2001). The protein contains two siroheme molecules per $\alpha_2\beta_2$ unit. The protein purified from *P. islandicum* presents similar physicochemical properties (molecular mass, quaternary structure, presence of siroheme and [4Fe–4S] centers) with DSR. However, this protein is unique by its isoelectric point in the basic range and could form a separate phylogenetic lineage that probably evolved prior to the divergence of bacteria and archaea (Molitor *et al.*, 1998).

### 3.3.4. Coenzyme $F_{420}$-Dependent Sulfite Reductase from Methanocaldococcus jannaschii

*Mc. jannaschii*, a deeply rooted hyperthermophilic methanogenic archaeon, grows and produces methane with sulfite as the sole sulfur source (Bult *et al.*, 1996; Jones *et al.*, 1983). Sulfite is able to induce in *Mc. jannaschii* a novel, highly active coenzyme $F_{420}$-dependent sulfite reductase (called Fsr).

Fsr was purified (three chromatographic steps) under anaerobic conditions with a low final yield of purification of 2.6%. The apparent native molecular mass of Fsr was estimated to be 350 kDa and the subunit sizeof this enzyme was 70 kDa. The UV–visible spectrum of Fsr in an as isolated form exhibited peaks at 280, 395, and 590 nm, typical of a sulfite reductase hemoprotein (Johnson and Mukhopadhyay, 2005). Purified Fsr reduces sulfite to sulfide using reduced F₄₂₀ (H₂F₄₂₀) as the electron source (Kₘ: sulfite, 12 μM; H₂F₄₂₀, 21 μM). The optimum pH and temperature for this activity were >95 °C and 7.0, respectively.

The C-terminal half of the 70 kDa Mc. jannaschii Fsr represents a dissimilatory-type siroheme sulfite reductase and the N-terminal half a H₂F₄₂₀ dehydrogenase. Fsr is able to catalyze the corresponding partial reactions. Fsr homologs were found only in two strictly hydrogenotrophic thermophilic methanogens: Methanopyrus kandleri and Methanothermobacter thermautotrophicus, and also in an obligately methylotrophic methanogen, Methanovococcoides burtonii (Johnson and Mukhopadhyay, 2008a). Previously described sulfite reductases utilize mainly cytochromes and nicotinamides as electron carriers. Therefore, this is the first report of a coenzyme F₄₂₀-dependent sulfite reductase. Fsr provides Mc. jannaschii an anabolic ability and protection from sulfite toxicity. A small sulfite reductase related to Fsr-C is present in all the methanogens investigated by Johnson and Mukhopadhyay (2008a,b). The assimilatory-type sulfite reductase (P590) from Ms. bar- keri DSM 800 most likely belongs to this group.

The newly discovered sulfite reductase (Fsr) links biological methanogenesis and sulfate reduction, two most ancient respiratory metabolisms, in a unique way. The finding of a novel type of sulfite reductase in a methanarchaeon could be directly relevant to the anaerobic oxidation of methane with sulfate, a process in which archaea closely related to methanogenic archaea are intimately involved.

A novel respiratory enzyme is produced by the methanarchaeon Mc. jannaschii; its function is to detoxify the environment of sulfite with the production of sulfide (Johnson and Mukhopadhyay, 2005). The coenzyme F₄₂₀-dependent sulfite reductase (Fsr) is comparable to the methyl-coenzyme M reductase and uses H₂ as the electron donor. The Fsr is a product of gene fusion with the N-terminal part of the polypeptide containing the H₂F₄₂₀ dehydrogenase activity and the C-terminus contains the dissimilatory siroheme sulfite reductase. The gene encoding for Fsr was transferred into Methanococcus maripaludis and converted it from sensitive to sulfite to resistant to sulfite with the production of sulfide (Johnson and Mukhopadhyay, 2008b). The Frs enzyme appears to be important for membrane-based energy transduction in these anaerobic methanogens.
3.4. Complex Hemoproteins

While *W. succinogenes* is not an archaea, it has the best-characterized sulfur reductase. In *W. succinogenes*, the reduction of sulfur by H₂ is attributed to two enzymes, a hydrogenase (Hyn) and a molybdenum-containing Psr (*Fauque et al.*, 1994; *Hedderich et al.*, 1999). While there is a strong electron couple between Hyn and Psr involving a menaquinone and a c-type cytochrome, these do not form a multienzyme complex in the membrane and are easily isolated as separate units. The Psr of *W. succinogenes* consists of three subunits and a model of this membrane association with coupled proton pumping during sulfur respiration is given in Fig. 14. Electron transfer from the hydrogenase unit to the Psr is mediated by menaquinone and not a cytochrome. The PsrA subunit is the catalytic unit, PsrB is an iron–sulfur protein, and PsrC is an integral membrane protein that serves to anchor the other subunits on the membrane. From genome examination, it appears that a multisubunit Psr is found in several prokaryotes (Table 10); however, this list is sure to expand as research in sulfur reduction continues.

### 3.4.1. Membrane-Bound Sulfur-Reducing Complex from the Hyperthermophilic *Pyrodictium abyssi*

Nine major subunits make up the hydrogenase–sulfur multienzyme complex isolated from *P. abyssi* (*Dirmeier et al.*, 1998; *Keller and Dirmeier*, 2001). This is not the largest electron transport complex isolated from microorganisms because the NADH:quinone oxidoreductase from *Paracoccus denitrificans* contains 10 subunits (*Yagi*, 1991) and the H₂:Hdr from *Ms. barkeri* has nine subunits (*Heiden et al.*, 1993). The size of the subunits of *P. abyssi* range from 82 to 24 kDa and the entire complex has a mass of about 500–550 kDa. Analysis of the complex reveals the presence of a c-type heme, one or two b-type hemes, a nickel hydrogenase, and an undetermined number of iron–sulfur clusters.

### 3.4.2. Sulfur Oxidoreductase from *S. deleyianum*

The sulfur oxidoreductase (SR) is a constitutive enzyme that is localized in the plasma membrane. As reviewed by *Fauque et al.* (1994), the SR in *S. deleyianum* is energized by a Ni–Fe–S hydrogenase and is several times

Wolinella succinogenes

![Diagram](#)

Acidianus ambivalens

![Diagram](#)

Figure 14 Model of sulfur-reducing complexes in membranes of A. ambivalens and W. succinogenes. PsrABC are sulfur-reducing subunits and HyABC are hydrogenase subunits of W. succinogenes. Symbol: MK, menaquinone. SreABC are sulfur-reducing subunits and HymLS and Isp1 are hydrogenase subunits of A. ambivalens. Symbols: SQ, sulfolobusquinone; X, [4Fe–4S] center; 2e, 2 electrons, and dashed lines indicate electron flow. The two vertical lines in HynC and Isp1 indicate presence of two b-type hemes. Out, periplasm; In, cytoplasm.

Table 10 Prokaryotes containing all three subunits (alpha, beta, and gamma) of sulfur reductase.

---

**Archaea**

- *Hyperthermus butylicus* DSM 5456
- *Sulfolobus islandicus* HVE 10/4
- *Sulfolobus solfataricus* P2

**Bacteria**

- *Aquiles aeolicus* VF5
- *Campylobacter rectus* RM 3267
- *Desulfitobacterium hafniense* DCB-2
- *Desulfomicrobium baculatum* DSM 4028
- *Hydrogenobacter thermophilus* TK-6
- *Sulfospirillum deleyianum* DSM 6946
- *Thermocrinis albus* DSM 14484
- *Wolinella succinogenes* DSM 1740

---

more active than in other bacteria. Sulfur reduction is enhanced by the presence of thiols and the SR contains at least one [4Fe–4S] center but does not contain a $b$-type cytochrome. While the genome of *S. deleyianum* (*Sikorski et al.*, 2010) indicates that the SR consists of three subunits, these components remain to be characterized.

### 3.4.3. Membrane-Bound Sulfur Reductase of Acidianus ambivalens

The acidophilic *Acidianus ambivalens* grows on elemental sulfur at $80^\circ \mathrm{C}$. A hydrogenase–sulfur reductase complex has been isolated from the membrane of *Acidianus ambivalens* and the Sr subunits are similar to those found in *W. succinogenes* (Fig. 14). SR is a molybdoenzyme and is suggested to be a member of the DMSO reductase family (*Laska et al.*, 2003). The SR gene cluster consists of five subunits. The *sreA* gene produces a 110 kDa protein that has binding motifs for a [4Fe–4S] center. The *sreB* gene encodes for a protein rich in cysteines and could coordinate a [4Fe–4S] center. SreC is a hydrophobic protein that stabilizes the Sr into the membrane and the role for SreD is unresolved at this time. The hydrogenase gene cluster consists of 12 genes and three of these encode for structural proteins. The HynL subunit contains nickel and the subunit HynS contains several Fe–S-binding motifs, making this a Ni–Fe hydrogenase. The third structural gene, *Ispl*, is an integral membrane protein and serves to bind the other subunits (HynL and HynS) into the membrane. The quinone, presumed to sulfolobusquinone, transfers electrons from hydrogenase to the sulfur reductase.

### 3.4.4. $H_2$:Sulfur Oxidoreductase Complex of Ignicoccus hospitalis

Perhaps one of the most unusual multiprotein complexes associated with sulfur reduction is found in the hyperthermophilic Archaeon *I. hospitalis* (*Küper et al.*, 2010). Using antibodies labeled with gold nanoparticles and electron microscopy for thin sections, $H_2$:Sulfur oxidoreductase from *I. hospitalis* was localized on the outer membrane of the cell. Additionally, localized on the outer membrane are the subunits for ATP synthase. This unique energizing of the outer membrane in *I. hospitalis* may explain the symbiotic relationship of *Nanoarchaeum equitans* with *I. hospitalis*.

HEMOPROTEINS IN DISSIMILATORY REDUCTION

### 3.4.5. HdrDE Complex from Methanosarcina sp.

The final steps in methane biosynthesis involve the reversible reaction between coenzyme M and coenzyme B. The reaction of CoM–S–CoB ⇌ CoM–SH + CoB–SH is catalyzed by Hdr. Two distinct types of Hdr have been isolated from methanogens with one being soluble in the cytoplasm and the other being membrane bound. In *Methanothermobacter marburgensis*, the HdrABC (Fig. 12) is soluble and has been characterized (*Hedderich et al.*, 1990, 1994). A binding motif for FAD is present in subunit Hdr as well as a binding motif for two [4Fe–4S] centers. HdrB contains 10 highly conserved cysteine residues with motifs of Cx<sub>31–38</sub>CCx<sub>33,34</sub>Cx<sub>2</sub>C characteristic of other Hdrs. There are motifs for two [4Fe–4S] centers in HdrC.

A membrane-bound HdrDE complex is found in *Ms. barkeri* and *Ms. thermophila* (*Heiden et al.*, 1994; *Simianu et al.*, 1998). This enzyme has a membrane-bound subunit (HdrE) containing binding motifs for *b*-type hemes and a cytoplasmic subunit (HdrD) with binding sites for two [4Fe–4S] centers (see Fig. 12). This HdrDE is structurally similar to HmeCD of *A. profundus* and *A. fulgidus*. The proton pumping attributed to HdrDE interfacing with the lipophylic electron carrier in the membrane, methanophenazine, in methanogens is discussed by *Ferry and Kastead* (2007). The contribution of HdrDE to energetics of methanogens is significant because these archaea have limited systems for ATP production.

## 4. CONCLUDING STATEMENTS

This chapter shows how diverse the field of hemoproteins is, even in the restricted group of anaerobic dissimilatory sulfate- and sulfur-reducing bacteria. Diverse functions as electron transfer, reduction of nitrite, elemental sulfur or sulfite are found with the same building, basic unit, the heme core.

Although there is considerable metabolic diversity in the sulfate and sulfur reducers, the differences may be most pronounced in the electron transfer systems. Electron donors are commonly H<sub>2</sub>, formate, or lactate and since the electron acceptors of sulfate or elemental sulfur, there is considerable similarity in the enzymes for dissimilarity sulfate or sulfur reduction. It is the movement of electrons between the electron donors and electron acceptors where the diversity occurs. There is an ordered use of hemoproteins with one to several *c*-type hemes used in the oxidation steps. Multihemoproteins are found in both sulfate and sulfur reducers and no doubt reflect a requirement for

survival of early microorganisms. It has been suggested by Thomas *et al.* (2008) that the large number of $c$-type cytochromes and $c$-type cytochrome membrane complexes provides for great metabolic versatility. There may be redundancy of electron transport systems with subtle regulations to promote recycling of redox compounds such as H₂, formate, or CO. These redox loops would be important for specific energy conservation processes.

The biosynthesis of the hemes by many of these archael and bacterial sulfate and sulfur reducers reflects the functioning of a primitive metabolic pathway that is distinct from that found in most other life forms. Analysis of genomes reveals that genes for heme biosynthesis are generally in the neighborhood for genes associated with sulfate respiration. The information concerning heme biosynthesis in sulfur reducers is based on a few species and future studies will be most useful to understand mechanism of heme formation and regulation of the synthesis of the products from this pathway. Additionally, future studies will enable us to determine evolution of the alternate metabolic pathway. Since some SRB have genes for the classic heme biosynthetic pathway, it may be that these organisms may have acquired genes for classic heme biosynthesis from more modern bacteria.

Electron transport complexes in the plasma membrane of the sulfate- and sulfur-reducing prokaryotes provided a mechanism for charging the plasma membrane by pumping protons out of the cytoplasm. These multiprotein complexes usually contained a catalytic subunit for oxidation of H₂ or formate and one or more subunits containing $b$-type hemes and [4Fe–4S] centers to transfer electrons to the lipophylic quinones. Multiprotein complexes are used to transfer electrons from reduced quinones to sulfate and the use of the Dsr and Qmo complexes for sulfate reduction to sulfide are highly conserved. Proton export involving quinones is concomitant with electron flow within these electron transport complexes preceding the $b/c_1$ (III) complex found in many biological systems.

---

**REFERENCES**

Abken, H.-J., Tietze, M., Brodersen, J., Baumer, S., Beiffuss, U. and Deppenmeier, U. (1998). Isolation and characterization of methanophenazine and function of phenazines in membrane-bound electron transport of Methanosarcina mazei Go1. *J. Bacteriol.* **180**, 2027–2032.

Abreu, I.A., Lourenço, A.I., Xavier, A.V., LeGall, J., Coelho, A.V., Matias, P.M., Pinto, D.M., Carrondo, M.A., Teixeira, M. and Saraiva, L.M. (2003). A novel iron centre in the split Soret cytochrome c from Desulfovibrio desulfuricans ATCC 27774. *J. Biol. Inorg. Chem.* **8**, 360–370.

HEMOPROTEINS IN DISSIMILATORY REDUCTION 71

Akagi, J.M. (1995). Respiratory sulfate reduction. In L.L. Barton (Ed.), *Biotechnology Handbooks: Sulfate-Reducing Bacteria*. Vol. 8 (pp. 89–111). New York and London: Plenum Press.

Akagi, J.M. and Adams, V. (1973). Isolation of a bisulfite reductase activity from *Desulfotomaculum nigrificans* and its identification as the carbon monoxide-binding pigment P582. *J. Bacteriol.* **116**, 372–396.

Akutsu, H., Park, J.S. and Sano, S. (1993). L-Methionine methyl is specifically incorporated into the C-2 and C-7 positions of the porphyrin of cytochrome c3 in a strictly anaerobic bacterium, *Desulfovibrio vulgaris*. *J. Am. Chem. Soc.* **115**, 12185–12186.

Almendra, M.J., Brondino, C.D., Gavel, O., Pereira, A.S., Tavares, P., Bursakov, S., Duarte, R., Caldeira, J., Moura, J.J.G. and Moura, I. (1999). Purification and characterization of a tungsten-containing formate dehydrogenase form *Desulfovibrio gigas*. *Biochemistry* **38**, 16366–16372.

Al-Sheboul, S. and Saffarini, D. (2011). Identification and analysis of the *Shewanella oneidensis* major oxygen-independent coproporphyrin III oxidase gene. *Anaerobe* **17**, 501–505.

Alves, A.S., Paquete, C.M., Fonseca, B.M. and Louro, R.O. (2011). Exploration of the "cytochromome" of *Desulfuromonas acetoxidans*, a marine bacterium capable of powering microbial fuel cells. *Metallomics* **3**, 349–353.

Ambler, R.P. (1991). Sequence variability in bacterial cytochromes c. *Biochim. Biophys. Acta* **1058**, 42–47.

Assfalg, M., Banci, L., Bruschi, M. and Turano, P. (1998). 800 MHz 1H NMR solution structure refinement of oxidized cytochrome c7 from *Desulfuromonas acetoxidans*. *Eur. J. Biochem.* **256**, 261–270.

Assfalg, M., Banci, L., Bertini, I., Bruschi, M., Giudici-Orticoni, M.T. and Turano, P. (1999). A proton-NMR investigation of the fully reduced cytochrome c7 from *Desulfuromonas acetoxidans*. Comparison between the reduced and the oxidized forms. *Eur. J. Biochem.* **266**, 634–643.

Assfalg, M., Bertini, I., Bruschi, M., Michel, C. and Turano, P. (2002). The metal reductase activity of some multi-heme cytochromes c. NMR structural characterization of the reduction of chromium (VI) to chromium (III) by cytochrome c7. *Proc. Natl. Acad. Sci. U. S. A.* **99**, 9750–9754.

Aubert, C., Leroy, G., Bianco, P., Forest, E., Bruschi, M. and Dolla, A. (1998a). Characterization of the cytochromes c from *Desulfovibrio desulfuricans* G201. *Biochem. Biophys. Res. Commun.* **242**, 213–218.

Aubert, C., Lojou, E., Bianco, P., Rousset, M., Durand, M.-C., Bruschi, M. and Dolla, A. (1998b). The *Desulfuromonas acetoxidans* triheme cytochrome c7 produced in *Desulfovibrio desulfuricans* retains its metal reductase activity. *Appl. Environ. Microbiol.* **64**, 1308–1312.

Audiffrin, C., Cayol, J.-L., Joulian, C., Casalot, L., Thomas, P., Garcia, J.-L. and Ollivier, B. (2003). *Desulfonauticus submarinus* gen. nov., sp. nov., a novel sulfate-reducing bacterium isolated from a deep-sea hydrothermal vent. *Int. J. Syst. Evol. Microbiol.* **53**, 1585–1590.

Baar, C., Eppinger, M., Raddatz, G., Simon, J., Lanz, C., Klimmek, O., Nandakumar, R., Gross, R., Rosinus, A., Keller, H., Jagtap, P., Linke, B., et al. (2003). Complete genome sequence and analysis of *Wolinella succinogenes*. *Proc. Natl. Acad. Sci. U. S. A.* **100**, 11690–11695.

Bache, R., Kroneck, P.M.H., Merkle, H. and Beinert, H. (1983). A survey of EPR-detectable components in sulfur-reducing bacteria. *Biochim. Biophys. Acta* **722**, 417–426.

Bamford, V.A., Angove, H.C., Seward, H.E., Thomson, A.J., Cole, J.A., Butt, J.N., Hemmings, A.M. and Richardson, D.J. (2002). Structure and spectroscopy of the periplasmic cytochrome c nitrite reductase from *Escherichia coli*. *Biochemistry* **41**, 2921–2931.

Banci, L., Bertini, I., Bruschi, M., Sompornpisut, P. and Turano, P. (1996). NMR characterization and solution structure determination of the oxidized cytochrome c7 from *Desulfuromonas acetoxidans*. *Proc. Natl. Acad. Sci. U. S. A.* **93**, 14396–14400.

Barton, L.L. and Fauque, G.D. (2009). Biochemistry, physiology and biotechnology of sulfate-reducing bacteria. *Adv. Appl. Microbiol.* **68**, 41–98.

Barton, L.L., LeGall, J. and Peck, H.D.Jr., (1970). Phosphorylation coupled to oxidation of hydrogen with fumarate in extracts of the sulfate-reducing bacterium, *Desulfovibrio gigas*. *Biochem. Biophys. Res. Commun.* **41**, 1036–1042.

Barton, L.L., LeGall, J. and Peck, H.D.Jr., (1972). Oxidative phosphorylation in the obligate anaerobe, *Desulfovibrio gigas*. In A. San Pietro & H. Gest (Eds.), *Horizons of Bioenergetics* (pp. 33–51). New York: Academic Press.

Barton, L.L., LeGall, J., Odom, J.M. and Peck, H.D.Jr., (1983). Energy coupling to nitrite respiration in the sulfate-reducing bacterium, *Desulfovibrio gigas*. *J. Bacteriol.* **153**, 867–871.

Barton, L.L., Plunkett, R.M. and Thomson, B.M. (2003). Reduction of metals and nonessential elements by anaerobes. In L.G. Ljungdahl, M.A. Adams, L.L. Barton, J.G. Ferry & M.K. Johnson (Eds.), *Biochemistry and Physiology of Anaerobic Bacteria* (pp. 220–234). New York: Springer.

Bartsch, R.G. (1968). Bacterial cytochromes. *Annu. Rev. Microbiol.* **22**, 181–200.

Battersby, A.R. and Sheng, Z.-C. (1982). Preparation and spectroscopic properties of Co<sup>III</sup>-isobacteriochlorins: relationship to the cobalt-containing proteins from *Desulphovibrio gigas* and *Desulphovibrio desulphuricans*. *J. Chem. Soc., Chem. Commun.* 1393–1394.

Beliakova, E.V., Rozanova, E.P., Borzenkov, I.A., Turova, T.P., Pusheva, M.A., Lysenko, A.M. and Kolganov, T.V. (2006). The new facultatively chemolithoautotrophic, moderately halophilic, sulfate-reducing bacterium *Desulfovermiculus halophilus* gen. nov., sp. nov., isolated from an oil field. *Mikrobiologiia* **75**, 201–211.

Ben Dhia Thabet, O., Wafa, T., Eltaief, K., Cayol, J.-L., Hamdi, M., Fauque, G. and Fardeau, M.-L. (2011). *Desulfovibrio legalis* sp. nov.: a moderately halophilic, sulfate-reducing bacterium isolated from a wastewater digester in Tunisia. *Curr. Microbiol.* **62**, 486–491.

Biebl, H. and Pfennig, N. (1977). Growth of sulfate-reducing bacteria with sulfur as electron acceptor. *Arch. Microbiol.* **112**, 115–117.

Biswas, K.C., Woodards, N.A., Xu, H. and Barton, L.L. (2009). Reduction of molybdate by sulfate-reducing bacteria. *Biometals* **22**, 131–140.

Blaut, M. (1994). Metabolism of methanogens. *Antonie Van Leeuwenhoek* **66**, 187–208.

Boulègue, J. (1978). Solubility of elemental sulfur in water at 298 K. *Phosphorus Sulfur Silicon Relat. Elem.* **5**, 127–128.

Brondino, C.D., Passeggi, M.C., Caldeira, J., Almendra, M.J., Deio, M.J., Moura, J.J. and Moura, I. (2004). Incorporation of either molybdenum or tungsten into formate dehydrogenase from *Desulfovibrio alaskensis* NCIMB 13491; EPR assignment of the proximal iron-sulfur cluster to the pterin cofactor in formate dehydrogenases from sulfate-reducing bacteria. *J. Biol. Inorg. Chem.* **9**, 145–151.

Bruschi, M. (1994). Cytochrome $c_3$ (Mr 26,000) isolated from sulfate-reducing bacteria and its relationships to other polyhemic cytochromes from *Desulfovibrio*. *Methods Enzymol.* **243**, 140–155.

Bruschi, M., Woudsta, M., Guigliarelli, B., Asso, M., Lojou, E., Petillot, Y. and Albergel, C. (1997). Biochemical and spectroscopic characterization of two new cytochromes isolated from *Desulfuromonas acetoxidans*. *Biochem.* **36**, 10601–10608.

Bruschi, M., Barton, L.L., Goulhen, F. and Plunkett, R.M. (2007). Enzymatic and genomic studies on the reduction of mercury and selected metallic oxyanions by sulfate-reducing bacteria. In L.L. Barton & W.A. Hamilton (Eds.), *Sulphate-Reducing Bacteria—Environmental and Engineered Systems* (pp. 435–457). Cambridge, UK: Cambridge University Press.

Buchenau, B., Kahnt, J., Heinemann, I.U., Jahn, D. and Thauer, R.K. (2006). Heme biosynthesis in *Methanosarcina barkeri* via a pathway involving two methylation reactions. *J. Bacteriol.* **188**, 8666–8668.

Bult, C.J., White, O., Olsen, G.J., Zhou, L., Fleischmann, R.D., Sutton, G.G., Blake, J.A., FitzGerald, L.M., Clayton, R.A., Gocayne, J.D., Kerlavage, A.R., Dougherty, B.A., et al. (1996). Complete genome sequence of the methanogenic archaeon, *Methanococcus jannaschii*. *Science* **273**, 1058–1073.

Burns, J.L. and DiChristina, T.J. (2009). Anaerobic respiration of elemental sulfur and thiosulfate by *Shewanella oneidensis* MR-1 requires psrA, a homolog of the phsA gene of *Salmonella enterica* serovar typhimurium LT2. *Appl. Environ. Microbiol.* **75**, 5209–5217.

Cammack, R., Fauque, G., Moura, J.J.G. and LeGall, J. (1984). ESR studies of cytochrome c from *Desulfovibrio desulfuricans* strain Norway 4: midpoint potentials of the four haems and interactions with ferredoxin and colloidal sulphur. *Biochim. Biophys. Acta* **784**, 68–74.

Canfield, D.E. (2005). The early history of atmospheric oxygen: homage to Robert A. Garrels. *Annu. Rev. Earth Planet. Sci.* **33**, 1–36.

Canfield, D.E. and Raiswell, R. (1999). The evolution of the sulfur cycle. *Am. J. Sci.* **299**, 679–723.

Cavallaro, G., Decaria, L. and Rasato, A. (2008). Genome based analysis of heme biosynthesis and uptake in prokaryotic systems. *J. Proteome Res.* **7**, 4946–4954.

Chottard, G., Kazanskaya, I. and Bruschi, M. (2000). Resonance Raman study of multihemic c-type cytochromes from *Desulfuromonas acetoxidans*. *Eur. J. Biochem.* **267**, 1050–1058.

Coelho, A.V., Matias, P.M., Sieker, L.C., Morais, J., Carrondo, M.A., Lampreia, J., Costa, C., Moura, J.J.G., Moura, I. and Le Gall, J. (1996). Preliminary crystallographic analysis and further characterization of a dodecaheme cytochrome c from *Desulfovibrio desulfuricans* ATCC 27774. *Acta Cryst* **D52**, 1202–1208.

Correia, I.J., Paquete, C.M., Louro, R.O., Catarino, T., Turner, D.J. and Xavier, A.V. (2002). Thermodynamic and kinetic characterization of trihaem cytochrome c3 from *Desulfuromonas acetoxidans*. *Eur. J. Biochem.* **269**, 5722–5730.

Costa, C., Macedo, A., Moura, I., Moura, J.J.G., LeGall, J., Berlier, Y., Liu, M.-Y. and Payne, W.J. (1990a). Regulation of the hexahaem nitrite/nitric oxide reductase of *Desulfovibrio desulfuricans*, *Wolinella succinogenes* and *Escherichia coli*. A mass spectropometry study. *FEBS Lett.* **276**, 67–70.

Costa, C., Moura, J.J.G., Moura, I., Liu, M.-Y., Peck, H.D., LeGall, J., Wang, Y. and Huynh, B.H. (1990b). Hexahaem nitrite reductase from *Desulfovibrio desulfuricans*. Mössbauer and EPR characterization of the haem groups. *J. Biol. Chem.* **254**, 14382–14387.

Costa, C., Moura, J.J.G., Moura, I., Wang, Y. and Huynh, B.H. (1996). Redox properties of cytochrome c nitrite reductase from *Desulfovibrio desulfuricans* ATCC 27774. *J. Biol. Chem.* **271**, 23191–23196.

Costa, C., Teixeira, M., LeGall, J., Moura, J.J.G. and Moura, I. (1997). Formate dehydrogenase from *Desulfovibrio desulfuricans* ATCC 27774: isolation and spectroscopic characterization of the active sites (heme, iron-sulfur centers and molybdenum). *J. Biol. Inorg. Chem.* **2**, 198–208.

Coutinho, I.B. and Xavier, A.V. (1994). Tetracytochromes. *Methods Enzymol.* **243**, 119–140.

Crane, B.R., Siegel, L.M. and Getzoff, E.D. (1995). Sulfite reductase at 1.6 Å: evolution and catalysis for reduction of inorganic anions. *Science* **270**, 59–67.

Cunha, C.A., Moura, J.J.G., Moura, I., et al. (2003). Cytochrome c nitrite reductase from *Desulfovibrio desulfuricans* ATCC 27774. The relevance of the two calcium sites in the structure of the catalytic subunit (NrfA). *J. Biol. Chem.* **278**, 17455–17465.

Cypionka, H. (1995). Solute transport and cell energetics. In: L.L. Barton (Ed.), *Biotechnology Handbooks: Sulfate-Reducing Bacteria*. Vol. 8 (pp. 151–184). New York: Plenum Press.

Czechowski, M., Fauque, G., Galliano, N., Dimon, B., Moura, I., Moura, J.J.G., Xavier, A.V., Barato, B.A.S., Lino, A.R. and LeGall, J. (1986). Purification and characterization of three proteins from a halophilic sulfate-reducing bacterium, *Desulfovibrio salexigens*. *J. Ind. Microbiol. Biotechnol.* **1**, 139–147. doi:10.1007/BF01569265.

Czjzek, M., Arnoux, P., Haser, R. and Shepard, W. (2001). Structure of cytochrome c₇ from *Desulfuromonas acetoxidans* at 1.9 Å resolution. *Acta Cryst D57*, 670–678.

Dahl, C. and Trüper, H.G. (2001). Sulfite reductase and APS reductase from *Archaeoglobus fulgidus*. *Methods Enzymol.* **331**, 427–441.

Dahl, C., Kredich, N.M., Deutzmann, R. and Trüper, H.G. (1993). Dissimilatory sulphite reductase from *Archaeoglobus fulgidus*: physio-chemical properties of the enzyme and cloning, sequencing and analysis of the reductase genes. *J. Gen. Microbiol.* **139**, 1817–1828.

Dahl, C., Speich, N. and Trüper, H.G. (1994). Enzymology and molecular biology of sulfate reduction in extremely thermophilic archaeon *Archaeoglobus fulgidus*. *Methods Enzymol.* **243**, 331–349.

Dahl, C., Molitor, M. and Trüper, H.G. (2001). Siroheme-sulfite reductase-type protein from *Pyrobaculum islandicum*. *Methods Enzymol.* **331**, 410–419.

Dailey, H.A. (2002). Terminal steps of haem biosynthesis. *Biochem. Soc. Trans.* **30**, 590–595.

Degtyarenko, K.N., North, A.C.T. and Findlay, J.B.C. (1997). PROMISE: a new database of information on prosthetic centres and metal ions in protein active sites. *Protein Eng.* **10**, 183–186.

Deppenmeir, U., Müller, V. and Gottschalk, G. (1996). Pathways of energy conservation in methanogenic archaea. *Arch. Microbiol.* **165**, 149–163.

DerVartanian, D.V. (1994). Desulforubidin: dissimilatory, high-spin sulfite reductase of *Desulfomicrobium* species. *Methods Enzymol.* **243**, 270–276.

DeWeerd, K., Mandelco, L., Tanner, R.S., Woese, C.R. and Sufilta, J.M. (1990). *Desulfomonile tiedjei* gen.nov. and sp. nov., a novel anaerobic dehalogenating, sulfate-reducing bacterium. *Arch. Microbiol.* **154**, 23–30.

DiMarco, A.A., Bobik, T.A. and Wolfe, R.S. (1990). Unusual coenzymes of methanogenesis. *Annu. Rev. Biochem.* **59**, 355–394.

Dirmeier, R., Keller, M., Frey, G., Huber, H. and Stetter, K.O. (1998). Purification and properties of an extremely thermostable membrane-bound sulfur-reducing complex from the hyperthermophilic *Pyrodictium abyssi*. *Eur. J. Biochem.* **252**, 486–491.

Dos Santos, W.G., Pacheco, I., Liu, M.-Y., Teixeira, M., Xavier, A.V. and LeGall, J. (2000). Purification and characterization of an iron superoxide dismutase and a catalase from the sulfate-reducing bacterium, *Desulfovibrio gigas*. *J. Bacteriol.* **182**, 796–804.

Drake, H.L. and Akagi, J.M. (1976). Purification of a unique bisulfite-reducing enzyme from *Desulfovibrio vulgaris*. *Biochem. Biophys. Res. Commun.* **71**, 1214–1219.

Einsle, O., Messerschmidt, A., Stach, P., Bourenkov, G.P., Bartunik, H.D., Huber, R. and Kroneck, P.M.H. (1999). Structure of cytochrome c nitrite reductase. *Nature* **400**, 476–480.

Einsle, O., Stach, P., Messerschmidt, A., Simon, J., Kröger, A., Huber, R. and Kroneck, P.M.H. (2000). Cytochrome c nitrite reductase from *Wolinella succinogenes*. Structure at 1.6 Å resolution, inhibitor binding, and haem-packing motifs. *J. Biol. Chem.* **275**, 39608–39616.

Einsle, O., Messerschmidt, A., Huber, R., Kroneck, P.M.H. and Neese, F. (2002a). Mechanism of the six-electron reduction of nitrite to ammonia by cytochrome c nitrite reductase. *J. Am. Chem. Soc.* **124**, 11737–11745.

Einsle, O., Stach, P., Messerschmidt, A., Klimmek, O., Simon, J., Kröger, A. and Kroneck, P.M.H. (2002b). Crystallization and preliminary X-ray analysis of the membrane-bound cytochrome c nitrite reductase complex (NrfHA) from *Wolinella succinogenes*. *Acta Crystallogr. D Biol. Crystallogr.* **58**, 341–342.

Fauque, G. (1985). Doctorat d’Etat Thèse, University of Technology of Compiègne, France.

Fauque, G.D. (1994). Sulfur reductase from thiophilic sulfate-reducing bacteria. *Methods Enzymol.* **243**, 353–367.

Fauque, G.D. (1995). Ecology of sulfate-reducing bacteria. In L.L. Barton (Ed.), *Sulfate-Reducing Bacteria. Biotechnology Handbooks*. Vol. 8 (pp. 217–241). New York: Plenum Press.

Fauque, G. and Ollivier, B. (2004). Anaerobes: the sulfate-reducing bacteria as an example of metabolic diversity. In A.T. Bull (Ed.), *Microbial Diversity and Bioprospecting* (pp. 169–176). Washington, DC: ASM Press.

Fauque, G., Hervé, D. and LeGall, J. (1979a). Structure-function relationship in hemoproteins: the role of cytochrome c₃ in the reduction of colloidal sulphur by sulfate-reducing bacteria. *Arch. Microbiol.* **121**, 261–264.

Fauque, G., Bruschi, M. and LeGall, J. (1979b). Purification and some properties of cytochrome c₅₅₃(550) isolated from *Desulfovibrio desulfuricans* Norway 4. *Biochem. Biophys. Res. Commun.* **86**, 1020–1029.

Fauque, G.D., Barton, L.L. and LeGall, J. (1980). Oxidative phosphorylation linked to the dissimilatory reduction of elemental sulfur by *Desulfovibrio*. *Ciba Found. Symp.* **72**, 71–86.

Fauque, G., Lino, A.R., Czechowski, M., Kang, L., DerVartanian, D.V., Moura, J.J.G., LeGall, J. and Moura, I. (1990). Purification and characterization of bisulfite reductase (desulfofuscidin) from *Desulfovibrio thermophilus* and its complexes with exogenous ligands. *Biochim. Biophys. Acta* **1040**, 112–118.

Fauque, G., LeGall, J. and Barton, L.L. (1991). Sulfate-reducing and sulfur-reducing bacteria. In J.M. Shively & L.L. Barton (Eds.), *Variations in Autotrophic Life* (pp. 271–337). London: Academic Press Limited.

Fauque, G.D., Klimmek, O. and Kroger, A. (1994). Sulfur reductase from spirilloid mesophilic sulfur-reducing eubacteria. *Methods Enzymol.* **243**, 367–383.

Ferry, J.G. and Kasteel, K.A. (2007). Methanogenesis. In R. Cavicchioli (Ed.), *Archaea: Molecular and Cellular Biology* (pp. 288–315). Washington, DC: ASM Press.

Finster, K. (2008). Microbiological disproportionation of inorganic sulfur compounds. *J. Sulfur Chem.* **3–4**, 281–292.

Finster, K., Liesack, W. and Yindall, B.J. (1997). *Desulfospira joergensenii*, gen. nov., sp. nov., a new sulfate-reducing bacterium isolated from marine surface sediment. *Syst. Appl. Microbiol.* **20**, 201–208.

Friedrich, M., Springer, N., Ludwig, W. and Schink, B. (1996). Phylogenetic positions of *Desulfofustis glycolicus* gen. nov., sp. nov., and *Syntrophobotulus glycolicus* gen. nov., sp. nov., two new strict anaerobes growing with glycolic acid. *Int. J. Syst. Bacteriol.* **46**, 1065–1069.

Fritz, G., Griesshaber, D., Seth, O. and Kroneck, P.M.H. (2001). Nonaheme cytochrome c, a new physiological electron acceptor for [Ni, Fe] hydrogenase in the sulfate-reducing bacterium *Desulfovibrio desulfuricans* Essex: primary sequence, molecular parameters, and redox properties. *Biochem* **40**, 1317–1324.

Fu, R., Wall, J.D. and Voordouw, G. (1994). DcrA, a heme-containing methyl-accepting protein from *Desulfovibrio vulgaris* Hildenborough, senses the oxygen concentration or redox potential of the environment. *J. Bacteriol.* **176**, 344–350.

Galushko, A.S. and Rozanova, E.P. (1991). *Desulfobacterium cetonicum* sp. nov. - a sulfate-reducing bacterium which oxidizes fatty acids and ketones. *Microbiology (Engl. Transl.) Mikrobiologiya (Russia)* **60**, 742–746.

Gavel, O.Y., Bursakov, S.A., Calvete, J.J., George, G.N., Moura, J.J.G. and Moura, I. (1998). ATP sulfurylases from sulfate-reducing bacteria of the genus *Desulfovibrio*. A novel metalloprotein containing cobalt and zinc. *Biochemistry* **37**, 16225–16232.

Gillespie, M.J. and Barton, L.L. (1996). Hydrogenase coupled reactions in *Campylobacter rectus*. *Anaerobe* **2**, 321–327.

Greene, E.A., Hubert, C., Nemati, M., Jennenan, G.E. and Voordouw, G. (2003). Nitrite reductase activity of sulfate-reducing bacteria prevents their inhibition by nitrate-reducing, sulphide-oxidizing bacteria. *Environ. Microbiol.* **5**, 607–617.

Habicht, K.S., Gade, M., Thamdrup, B., Berg, P. and Canfield, D.E. (2002). Calibration of sulfate levels in the Archean ocean. *Science* **298**, 2372–2374.

Hagerhall, C. (1997). Succinate: quinone oxidoreductases. Variations on a conserved theme. *Biochim. Biophys. Acta* **1320**, 107–141.

Hansen, J., Muldbjerg, M., Chérest, H. and Sardin-Kerjan, Y. (1997). Siroheme biosynthesis in Saccharomyces cerevisiae requires the product of both MET1 and MET8 genes. *FEBS Lett.* **401**, 20–24.

Haouari, O., Fardeau, M.-L., Cayol, J.-L., Fauque, G., Casiot, C., Elbaz-Poulichet, F., Hamdi, M. and Ollivier, B. (2008). *Thermodesulfovibrio hydrogeniphilus* sp. nov., a new thermophilic sulphate-reducing bacterium isolated from a Tunisian hot spring. *Syst. Appl. Microbiol.* **31**, 38–42.

Hatchikian, E.C. (1994). Desulfofuscidin: dissimilatory, high-spin sulfite reductase of thermophilic, sulfate-reducing bacteria. *Methods Enzymol.* **243**, 276–295.

Hatchikian, E.C. and LeGall, J. (1972). Evidence for the presence of a b-type cytochrome in the sulfate-reducing bacterium *Desulfovibrio gigas*, and its role in the reduction of fumarate by molecular hydrogen. *Biochim. Biophys. Acta* **267**, 479–484.

Hatchikian, E.C. and Zeikus, J.G. (1983). Characterization of a new type of dissimilatory sulfite reductase present in *Thermodesulfobacterium commune*. *J. Bacteriol.* **153**, 1211–1220.

Hartshorne, S., Richardson, D.J. and Simon, J. (2006). Multiple haem lyase genes indicate substrate specificity in cytochrome c biogenesis. *Biochem. Soc. Transactions* **34**, 146–149.

Haveman, S.A., Greene, E.A., Stilwell, C.P., Voordouw, J.K. and Voordouw, G. (2004). Physiological and gene expression analysis of inhibition of *Desulfovibrio vulgaris* Hildenborough by nitrite. *J. Bacteriol.* **186**, 7944–7950.

Haveman, S.A., Greene, E.A. and Voordouw, G. (2005). Gene expression analysis of the mechanism of inhibition of *Desulfovibrio vulgaris* Hildenborough by nitrate-reducing, sulfide-oxidizing bacteria. *Environ. Microbiol.* **7**, 1461–1465.

Haverman, S.A., Brunelle, V., Voordouw, J.K., Voordouw, G., Heidelberg, J.F. and Rabus, R. (2003). Gene expression analysis of energy metabolism mutants of *Desulfovibrio vulgaris* Hildenborough indicates an important role for alcohol dehydrogenase. *J. Bacteriol.* **185**, 4345–4353.

Hedderich, R., Berkessel, A. and Thauer, R.K. (1990). Purification and properties of heterodisulfide reductase from *Methanobacterium thermoautotrophicum* (strain Marburg). *Eur. J. Biochem.* **193**, 255–261.

Hedderich, R., Koch, J., Linder, D. and Thauer, R.K. (1994). The heterodisulfide reductase from *Methanobacterium thermoautotrophicum* contains sequence motifs characteristic of pyridine nucleotide-dependent thioredoxin reductase. *Eur. J. Biochem.* **225**, 253–261.

Hedderich, R., Klimmek, O., Kröger, A., Dirmeier, R., Keller, M. and Stetter, K.O. (1999). Anaerobic respiration with elemental sulfur and with sulfides. *FEMS Microbiol. Rev.* **22**, 353–381.

Hedderich, R., Hamann, N. and Bennati, M. (2005). Heterodisulfide reductase from methanogenic archaea: a new catalytic role for an iron-sulfur cluster. *Biol. Chem.* **386**, 961–970.

Heidelberg, J.F., Paulsen, I.T., Nelson, K.E., Gaidos, E.J., Nelson, W.C., Read, T.D., Eisen, J.A., Seshadri, R., Ward, N., Methe, B., Clayton, R.A., Meyer, T., et al. (2002). Genome sequence of the dissimilatory metal ion-reducing bacterium *Shewanella oneidensis*. *Nat. Biotechnol.* **20**, 1118–1123.

Heiden, S., Hedderich, R., Setzke, E. and Thauer, R.K. (1993). Purification of a cytochrome b containing H2:heterodisulfide oxidoreductase complex from the membranes of *Methanosarcina barkeri*. *Eur. J. Biochem.* **213**, 529–535.

Heiden, S., Hedderich, R., Setzke, E. and Thauer, R.K. (1994). Purification of a two-subunit cytochrome-b-containing heterodisulfide reductase from methanol-grown *Methanosarcina barkeri*. *Eur. J. Biochem.* **221**, 855–861.

Herrmann, G., Jayamani, E., Mai, G. and Buckel, W. (2008). Energy conservation via electron-transferring flavoprotein in anaerobic bacteria. *J. Bacteriol.* **190**, 784–791.

Higuchi, Y., Yagi, T. and Voordouw, G. (1994). Hexadecame cytochrome c. *Methods Enzymol.* **243**, 155–165.

Hinsley, A.P. and Berks, B.C. (2002). Specificity of respiratory pathways involved in the reduction of sulfur compounds *Salmonella enterica*. *Microbiology* **148**, 3631–3638.

Huber, R., Langworthy, T.A., Koning, H., Thomm, M., Woese, C.R., Sleytr, U.B. and Stetter, K.O. (1986). *Thermotoga maritima* sp. nov. represents a new genus of unique extremely thermophilic eubacteria growing up to 90°C. *Arch. Microbiol.* **144**, 324–333.

Huber, R., Kristjansson, J.K. and Stetter, K.O. (1987). *Pyrobaculum* gen. nov., a new genus of neutrophilic, rod-shaped archaebacteria from continental solfataras growing optimally at 100°C. *Arch. Microbiol.* **149**, 95–101.

Huber, H., Rossnagel, P., Woese, C.R., Rachel, R., Langworthy, T.A. and Stetter, K.O. (1996). Formation of ammonium from nitrate during chemolithoautotrophic growth of the extremely thermophilic bacterium *Ammonifex degensii* gen. nov. sp. nov. *Syst. Appl. Microbiol.* **19**, 40–49.

Huynh, B.H., Kang, L., DerVartanian, D.V., Peck, H.D.Jr., and LeGall, J. (1984). Characterization of a sulfite reductase from *Desulfovibrio vulgaris*. Evidence for the presence of a low spin siroheme and an exchange coupled siroheme-[4Fe-4S] unit. *J. Biol. Chem.* **259**, 15373–15376.

International Union of Biochemistry and Molecular Biology, (1992). London, UK: Portland Press Ltd.

Ishida, T., Yu, L., Akutu, H., Ozawa, K., Kawanishi, S., Seto, A., Inubushi, T. and Sano, S. (1998). A primitive pathway of porphyrin biosynthesis and enzymology in *Desulfovibrio vulgaris*. *Proc. Natl. Acad. Sci. U. S. A.* **95**, 4853–4858.

Ishimoto, M.J. (1959). Sulfate reduction in cell-free extracts of *Desulfovibrio*. *J. Biochem.* (Tokyo) **46**, 105–106.

Ishimoto, M., Koyama, J. and Nagai, Y. (1954). Biochemical studies on sulfate-reducing bacteria. IV. The cytochrome system of sulfate-reducing bacteria. *J. Biochem.* (Tokyo) **41**, 763–770.

Ishimoto, M., Kondo, Y., Kameyama, T., Yagi, T. and Shiraz, M. (1958). The role of cytochrome in the enzyme system of sulphate-reducing bacteria. In Science Council of Japan (Ed.), *Proceedings of the International Symposium on Enzyme Chemistry* (pp. 229–234). Tokyo and Kyoto: Maruzen.

Jacobs, N.J. and Jacobs, M.J. (1976). Nitrate, fumarate and oxygen as electron acceptors for the late steps of anaerobic heme synthesis. *Biochim. Biophys. Acta* **449**, 1–9.

Jacobs, N.J. and Jacobs, M.J. (1978). Quinones as hydrogen carriers for a late step in anaerobic heme biosynthesis in *Escherichia coli*. *Biochim. Biophys. Acta* **544**, 540–546.

Johnson, E.F. and Mukhopadhyay, B. (2005). A new type of sulfite reductase, a novel coenzyme F420-dependent enzyme, from the methanarchaeon *Methanocaldococcus jannaschii*. *J. Biol. Chem.* **280**, 38776–38786.

Johnson, E.J. and Mukhopadhyay, B. (2008a). A novel coenzyme F420 dependent sulfite reductase and a small sulfite reductase in methanogenic Archaea. In C. Dahl & C.G. Friedrich (Eds.), *Microbial Sulfur Metabolism* (pp. 202–217). Berlin: Springer-Verlag.

Johnson, E.F. and Mukhopadhyay, B. (2008b). Coenzyme F420-dependent sulfite reductase-enabled sulfite detoxification and use of sulfite as a sole sulfur source by *Methanococcus maripaludis*. *Appl. Environ. Microbiol.* **74**, 3591–3595.

Jones, W.J., Leigh, J.A., Mayer, F., Woese, C.R. and Wolfe, R.S. (1983). *Methanococcus jannaschii* sp. nov., an extremely thermophilic methanogen from a submarine hydrothermal vent. *Arch. Microbiol.* **136**, 254–261.

Junier, P., Junier, T., Podell, S., Sims, D., Detter, J.C., Lykidis, A., Han, C.S., Wigginton, N.S., Gaasterland, T. and Bernier-Latmani, R. (2010). The genome of the Gram-positive, metal- and sulfate-reducing bacterium *Desulfotomaculum reducens* strain MI-1. *Environ. Microbiol.* **12**, 2738–2754.

Jussofie, A. and Gottschalk, G. (1986). Further studies on the distribution of cytochromes in methanogenic bacteria. *FEMS Microbiol. Lett.* **37**, 15–18.

Kamlage, B. and Blaut, M. (1992). Characterization of cytochromes from *Methanosarcina* strain Go1 and their involvement in electron transport during growth on methanol. *J. Bacteriol.* **174**, 3921–3927.

Keilin, D. (1925). On cytochrome, a respiratory pigment, common to animals, yeast and higher plants. *Proc. R. Soc. Lond. B Biol. Sci.* **98**, 312–339.

Keller, M. and Dirmeier, R. (2001). Hydrogen-sulfur oxidoreductase complex from *Pyrodictium abyssi*. *Methods Enzymol.* **331**, 442–451.

Keller, L.L. and Wall, J.D. (2011). Genetics and molecular biology of the electron flow for sulfate respiration in *Desulfovibrio*. *Front. Microbiol.* **2**, 135. doi:10.3389/fmicb.2011.00135.

Keller, K.L., Wall, J.D. and Chhabra, S. (2011). Methods for engineering sulfate-reducing bacteria of the genus *Desulfovibrio*. *Methods Enzymol.* **497**, 503–517.

Kemmer, J.M., Krzycki, J.A., Prince, R.C. and Zeikus, J.G. (1987). Spectroscopic and enzymatic evidence for membrane-bound electron transport carriers and hydrogenase and their relation to cytochrome b function in *Methanosarcina barkeri*. *FEMS Microbiol. Lett.* **48**, 267–272.

Kern, M., Eisel, F., Scheithauer, J., Kranz, R.G. and Simon, J. (2010). Substrate specificity of three cytochrome c haem lyase isoenzymes from *Wolinella succinogenes*: unconventional haem c binding motifs are not sufficient for haem c attachment by NrfI and CcsA1. *Mol. Microbiol.* **75**, 122–137.

Kern, M. and Simon, J. (2008). Characterization of the NapGH quinol dehydrogenase complex involved in *Wolinella succinogenes* nitrate respiration. *Mol. Microbiol.* **69**, 1137–1152.

Kern, M. and Simon, J. (2011). Production of recombinant multi-heme cytochromes c in *Wolinella succinogenes*. *Methods Enzymol.* **486**, 429–446.

Kern, M., Einsle, O. and Simon, J. (2008). Variants of the tetrahaem cytochrome c quinol dehydrogenase NrfH characterize the menaquinol-binding site, the haem c-binding motifs and the transmembrane segment. *Biochem. J.* **414**, 73–79.

Klein, M., Fredrich, M., Roger, A.J., Hugenholtz, P., Fishbain, S., Abicht, H., Blackall, L.L., Stahl, D.A. and Wagner, M. (2001). Multiple lateral transfers of dissimilatory sulfite reductase genes between major lineages of sulfate-reducing bacteria. *J. Bacteriol.* **183**, 6028–6035.

Klemm, D.J. and Barton, L.L. (1985). Oxidation of protoporphyrinogen in the obligate anaerobe *Desulfovibrio gigas*. J. Bacteriol. **164**, 316–320.

Klemm, D.J. and Barton, L.L. (1987). Purification and properties of protoporphyrinogen oxidase from an anaerobic bacterium, *Desulfovibrio gigas*. J. Bacteriol. **169**, 5209–5215.

Klemm, D.J. and Barton, L.L. (1989). Protoporphyrinogen oxidation coupled to nitrite reduction with membranes from *Desulfovibrio gigas*. FEMS Microbiol. Lett. **61**, 61–64.

Klenk, H.P., Clayton, R.A., Tomb, J.F., White, O., Nelson, K.E., Ketchum, K.A., Dodson, R.J., Gwinn, M., Hickey, E.K., Peterson, J.D., Richardson, D.L., Kerlavage, A.R., et al. (1997). The complete genome sequence of the hyperthermophilic, sulphate-reducing archaeon *Archaeoglobus fulgidus*. Nature **390**, 364–370.

Kletzin, A., Urich, T., Muller, F., Bandeiras, T.M. and Gomez, C.M. (2004). Dissimilatory oxidation and reduction of elemental sulfur in thermophilic Archaea. J. Bioenerg. Biomembr. **36**, 77–91.

Klouche, N., Basso, O., Lascourrèges, J.-F., Cayol, J.-L., Thomas, P., Fauque, G., Fardeau, M.-L. and Magot, M. (2009). *Desulfocurvus vexinensis* gen. nov., sp. nov., a sulfate-reducing bacterium isolated from a deep subsurface aquifer. Int. J. Syst. Evol. Microbiol. **59**, 3100–3104.

Koller, K.B., Hawkridge, F.M., Fague, G. and Le Gall, J. (1987). Direct electron-transfer reactions of cytochrome-c553 from *Desulfovibrio vulgaris* Hildenborough at indium oxide electrodes. Biochem. Biophys. Res. Commun. **145**, 619–624.

Krekeler, D. and Cypionka, H. (1995). The preferred electron acceptor of *Desulfovibrio desulfuricans* CSN. FEMS Microbiol. Ecol. **17**, 271–278.

Kroger, A. and Innerhofer, A. (1976). The function of the b cytochromes in the electron transport from formate to fumarate of *Vibrio succinogenes*. Eur. J. Biochem. **69**, 497–506.

Kühn, W., Fiebig, K., Walther, R. and Gottschalk, G. (1979). Presence of a cytochrome b559 in *Methanosarcina barkeri*. FEBS Lett. **105**, 271–274.

Kuhn, W. and Gottschalk, G. (1983). Characterization of the cytochromes occurring in *Methanosarcina* species. Eur. J. Biochem. **135**, 89–94.

Kuhn, W., Fiebig, K., Hippe, H., Mah, R.A., Huser, B.A. and Gottschalk, G. (1983). Distribution of cytochromes in methanogenic bacteria. FEMS Microbiol. Lett. **20**, 407–410.

Kumazawa, Y., Fujiwara, T., Fukumori, Y., Koga, Y. and Yamanaka, T. (1994). Cytochrome bc purified from the methanogen *Methanosarcina barkeri*. Curr. Microbiol. **29**, 53–56.

Kunow, J., Linder, D., Stetter, K.O. and Thauer, R. (1994). Characterization of a membrane-bound multisubunit complex containing FAD and iron-sulfur clusters. Eur. J. Biochem. **223**, 503–511.

Küper, U., Meyer, C., Müller, V., Rachael, R. and Huber, H. (2010). Energized outer membrane and spatial separation of metabolic process in the hyperthermophilic Archaeon *Ignicoccus hospitalis*. Proc. Natl. Acad. Sci. U. S. A. **107**, 3152–3156.

Lampreia, J., Fauque, G., Speich, N., Dahl, C., Moura, I., Trüper, H.G. and Moura, J.J.G. (1991). Spectroscopic studies on APS reductase isolated from the hyperthermophilic sulfate-reducing archaebacterium *Archaeoglobus fulgidus*. Biochem. Biophys. Res. Commun. **181**, 342–347.

Lampreia, J., Pereira, A.S. and Moura, J.J.G. (1994). Adenylylsulfate reductases for sulfate-reducing bacteria. *Methods Enzymol.* **243**, 241–260.

Laska, S., Lottspeich, F. and Kletzin, A. (2003). Membrane-bound hydrogenase and sulfur reductase of the hyperthermophilic and acidophilic archaeon *Acidianus ambivalens*. *Microbiology* **149**, 2357–2371.

Lee, J.-P. and Peck, H.D.Jr., (1971). Purification of the enzyme reducing bisulfite to trithionate from *Desulfovibrio gigas* and its identification as desulfoviridin. *Biochem. Biophys. Res. Commun.* **45**, 583–589.

Lee, J.-P., LeGall, J. and Peck, H.D.Jr., (1973a). Isolation of assimilatory- and dissimilatory-type sulfite reductases from *Desulfovibrio vulgaris*. *J. Bacteriol.* **115**, 529–542.

Lee, J.-P., Yi, C.-S., LeGall, J. and Peck, H.D.Jr., (1973b). Isolation of a new pigment, desulforubidin, from *Desulfovibrio desulfuricans* (Norway strain) and its role in sulfite reduction. *J. Bacteriol.* **115**, 453–455.

LeFaou, A., Rajagopal, B.S., Daniels, L. and Fauque, G. (1990). Thiosulfate, polythionates and elemental sulfur assimilation and reduction in the bacterial world. *FEMS Microbiol. Rev.* **75**, 351–382.

LeGall, J. and Fauque, G. (1988). Dissimilatory reduction of sulfur compounds. In A.J.B. Zehnder (Ed.), *Biology of Anaerobic Microorganisms* (pp. 587–639). New York: Wiley.

Lemos, R.S., Gomes, C.M., LeGall, J., Xavier, A.V. and Teixeira, M. (2002). The quinol: fumarate oxidoreductase from the sulphate reducing bacterium *Desulfovibrio gigas*: spectroscopic and redox studies. *J. Bioenerg. Biomembr.* **34**, 21–30. doi:10.1023/A:1013814619023.

Liesack, W. and Finster, K. (1994). Phylogenetic analysis of five strains of Gram-negative, obligately anaerobic, sulfur-reducing bacteria and description of *Desulfuromusa* gen. nov., including *Desulfuromusa kysingii* sp. nov., *Desulfuromusa baki* sp. nov., and *Desulfuromusa succinoxidans* sp. nov. *Int. J. Syst. Bacteriol.* **44**, 753–758.

Liu, M.C. and Peck, H.D.Jr., (1981). The isolation of a hexaheme cytochrome from *Desulfovibrio* and identification as a new type of nitrite reductase. *J. Biol. Chem.* **265**, 13159–13164.

Liu, M.-C., Liu, M.-Y., Payne, W.J., Peck, H.D., Jr., LeGall, J. and DerVartanian, D.V. (1987). Comparative EPR studies on the nitrite reductases from *Escherichia coli* and *Wolinella succinogenes*. *FEBS Lett.* **218**, 227–230.

Liu, M.-C., Costa, C., Coutinho, I.B., Moura, J.J.G., Moura, I., Xavier, A.V. and LeGall, J. (1988). Cytochrome components of nitrate- and sulfate-respiring *Desulfovibrio desulfuricans* ATCC 27774. *J. Bacteriol.* **170**, 5545–5551.

Lloyd, J.R. (2003). Microbial reduction of metals and radionuclides. *FEMS Microbiol. Rev.* **27**, 411–425.

Lobo, S.A., Almeida, C.C., Carita, J.N., Teixeira, M. and Saraiva, L.M. (2008a). The haem-copper oxygen reductase of *Desulfovibrio vulgaris* contains a dihaem cytochrome c in subunit II. *Biochim. Biophys. Acta* **1777**, 1528–1534.

Lobo, S.A., Brindley, A.A., Romano, C.V., Leech, H.K., Warren, M.J. and Saraiva, L.M. (2008b). Two distinct roles for two functional cobaltchelatases (CbiK) in *Desulfovibrio vulgaris* Hildenborough. *Biochemistry* **47**, 5851–5857.

Lobo, S.A.L., Brindley, A., Warren, M.J. and Saraiva, L.M. (2009). Functional characterization of the early steps of tetrapyrrole biosynthesis and modification in *Desulfovibrio vulgaris* Hildenborough. *Biochem. J.* **420**, 317–325.

Lopez-Cortès, A., Bursakov, S., Figueiredo, A., Thapper, A.E., Todorovic, S., Moura, J.J.G., Ollivier, B., Moura, I. and Fauque, G. (2005). Purification and preliminary characterization of tetraheme cytochrome $c_{3}$ and adenylylsulfate reductase form the peptidolytic sulfate-reducing bacterium *Desulfovibrio aminophilus*. Bioinorg. Chem. Appl. **3**, 81–91.

Louie, T.M. and Mohn, W.W. (1999). Evidence for a chemiosmotic model of dehalorespiration in *Desulfomonile tiedjei* DCB-1. *J. Bacteriol.* **181**, 40–46.

Louie, T.M., Ni, S.H., Xun, L. and Mohn, W.W. (1997). Purification, characterization and gene sequence analysis of a novel cytochrome $c$ co-induced with reductive dechlorination in *Desulfomonile tiedjei* DCB-1. *Arch. Microbiol.* **168**, 520–527.

Loy, A., Duller, S. and Wagner, M. (2008). Evolution and ecology of microbes dissimilating sulfur compounds: insights from siroheme sulfite reductases. In C. Dahl & C. Friedrich (Eds.), *Microbial Sulfur Metabolism* (pp. 46–59). Berlin: Springer-Verlag.

Lukat, P., Rudolf, M., Stach, P., Messerschmidt, A., Kronexk, P.M.H., Simon, J. and Einsle, O. (2008). Binding and reduction of sulfite by cytochrome $c$ nitrite reductase. *Biochem* **47**, 2080–2086.

Madigan, M.T., Martinko, J.M., Dunlap, P.V. and Clark, D.P. (2009). *Brock Biology of Microorganisms* (12th edn.). New York: Pearson Benjamin Cummings.

Mander, G.J., Duin, E.C., Linder, D., Stetter, K.O. and Hedderich, R. (2002). Purification and characterization of a membrane-bound enzyme complex from the sulfate-reducing archaeon *Archaeoglobus fulgidus* related to heterodisulfide reductase from methanogenic archaea. *Eur. J. Biochem.* **269**, 1895–1904.

Mander, G.J., Pierik, A.J., Huber, H. and Hedderich, R. (2004). Two distinct heterodisulfide reductase-like enzymes in the sulfate-reducing archaeon *Archaeoglobus profundus*. *Eur. J. Biochem.* **271**, 1106–1116.

Mathews, F.S. (1985). The structure, function and evolution of cytochromes. *Prog. Biophys. Mol. Biol.* **45**, 1–56.

Matias, P.M., Coelho, R., Pereira, I.A.C., Coelho, A.V., Thompson, A.W., Sieker, L.C., LeGall, J. and Carrondo, M.A. (1999). The primary and three-dimensional structures of a nine-haem cytochrome from *Desulfovibrio desulfuricans* ATCC 27774 reveal a new member of the Hmc family. *Structure* **7**, 119–130.

Matias, P.M., Pereira, I.A., Soares, C.M. and Carrondo, M.A. (2005). Sulphate respiration from hydrogen in *Desulfovibrio* bacteria: a structural biology review. *Prog. Biophys. Mol. Biol.* **89**, 292–329.

Matthews, J.C., Timkovich, R., Liu, M.-Y. and LeGall, J. (1995). Siroamide: a prosthetic group isolated from sulfite reductases in the genus *Desulfovibrio*. *Biochemistry* **34**, 5248–5251.

Mehta, T., Coppi, M.V., Childers, S.E. and Lovley, D.R. (2005). Outer membrane c-type cytochromes required for Fe(III) and Mn(IV) oxide reduction in *Geobacter sulfurreducens*. *Appl. Environ. Microbiol.* **71**, 8634–8641.

Meyer, B. and Kuever, J. (2007). Phylogeny of the alpha and beta subunits of the dissimilatory adenosine-5′-phosphosulfate (APS) reductase form sulfate-reducing prokaryotes-origin and evolution of the dissimilatory sulfate-reduction pathway. *Microbiology* **153**, 2026–2044.

Miroshnichenko, M.L., Tourova, T.P., Kolganova, T.V., Kostrikina, N.A., Chernych, N. and Bonch-Osmolovskaya, E.A. (2008). *Ammonifex thiophilus*

sp. nov., a hyperthermophilic anaerobic bacterium from a Kamchatka hot spring. *Int. J. Syst. Evol. Microbiol.* **58**, 2935–2938.

Mitchell, P. (1979). Keilin’s respiratory chain concept and its chemiosmotic consequences. *Science* **206**, 1148–1159.

Molitor, M., Dahl, C., Molitor, I., Schäfer, U., Speich, N., Huber, R., Deutzmann, R. and Trüper, H.G. (1998). A dissimilatory siorhaem-sulfite-reductase-type protein from the hyperthermophilic archaeon *Pyrobaculum islandicum*. *Microbiol.* **144**, 529–541.

Moore, G.R. and Pettigrew, G.W. (1990). *Cytochromes c: Evolutionary, Structural and Physicochemical Aspects*. New York: Springer-Verlag.

Moser, D.P. and Nealson, K.H. (1996). Growth of the facultative anaerobe *Shewanella putrefaciens* by elemental sulfur reduction. *Appl. Environ. Microbiol.* **62**, 2100–2105.

Moura, I. and Lino, A.R. (1994). Low-spin sulfite reductases. *Methods Enzymol.* **243**, 296–303.

Moura, J.J.G., Moura, I., Santos, H., Xavier, A.V., Scandellari, M. and LeGall, J. (1982). Isolation of P<sub>590</sub> from *Methanosarcina barkeri*: evidence for the presence of sulfite reductase activity. *Biochem. Biophys. Res. Commun.* **108**, 1002–1009.

Moura, I., Lino, A.R., Moura, J.J.G., Xavier, A.V., Fauque, G., Peck, H.D.Jr., and LeGall, J. (1986). Low-spin sulfite-reductases: a new homologous group of nonheme iron-siroheme proteins in anaerobic bacteria. *Biochem. Biophys. Res. Commun.* **141**, 1032–1041.

Moura, I., Fauque, G., LeGall, J., Xavier, A.V. and Moura, J.J.G. (1987). Characterization of the cytochrome system of a nitrogen-fixing strain of a sulfate-reducing bacterium: *Desulfovibrio desulfuricans* Berre-Eau. *Eur. J. Biochem.* **162**, 547–554.

Moura, I., Liu, M.Y., Costa, C., Liu, M.C., Pai, G., Xaxier, A.V., LeGall, J., Payne, W. J. and Moura, J.J.G. (1988a). Spectroscopic characterization of a high-potential monohaem cytochrome from *Wolinella succinogenes*, a nitrate-respiring organism. Redox and spin equilibria studies. *Eur. J. Biochem.* **15**, 673–682.

Moura, I., LeGall, J., Lino, A.R., Peck, H.D., Jr., Fauque, G., Xavier, A.V., DerVartanian, D.V., Moura, J.J.G. and Huynh, B.H. (1988b). Characterization of two dissimilatory sulfite reductases (desulforubidin and desulfovibrin) from the sulfate-reducing bacteria. *J. Am. Chem. Soc.* **110**, 1075–1082.

Moura, J.J.G., Moura, I., Teixeira, M., Xavier, A.V., Fauque, G. and LeGall, J. (1988c). Nickel-containing-hydrogenases. In H. Sigel (Ed.), *Metal Ions in Biological Systems. Nickel and Its Role in Biology*. Vol. 23 (pp. 285–314). New York and Basel: Marcel Dekker Inc.

Moura, I., Pereira, A.S., Tavares, P. and Moura, J.J.G. (1999). Simple and complex iron-sulfur proteins in sulfate-reducing bacteria. In A.G. Sykes & R. Cammack (Eds.), *Advances in Inorganic Chemistry*. Vol. 47 (pp. 361–419). San Diego: Academic Press.

Moura, J.J.G., Gonzalez, P., Moura, I. and Fauque, G. (2007). Dissimilatory nitrate and nitrite ammonification by sulfate-reducing bacteria. In L.L. Barton & W.A. Hamilton (Eds.), *Sulphate-Reducing Bacteria: Environmental and Engineered Systems* (pp. 241–264). Cambridge, UK: Cambridge University Press.

Nealson, K.H. and Saffarini, D. (1994). Iron and manganese in anaerobic respiration: environmental significance, physiology, and regulation. *Annu. Rev. Microbiol.* **48**, 311–343.

Odom, J.M. and Peck, H.D.Jr., (1981). Hydrogen cycling as a general mechanism for energy coupling in the sulfate-reducing bacteria, *Desulfovibrio* sp. *FEMS Microbiol. Lett.* **12**, 47–50.

Oliveira, T.F., Vornhein, C., Matias, P.M., Venceslau, S.S., Pereira, I.A.C. and Archer, M. (2008a). Purification, crystallization and preliminary crystallographic analysis of a dissimilatory DsrAB sulfite reductase in complex with DsrC. *J. Struct. Biol.* **164**, 236–239.

Oliveira, T.F., Vornhein, C., Matias, P.M., Venceslau, S.S., Pereira, I.A. and Archer, M. (2008b). The crystal structure of *Desulfovibrio vulgaris* dissimilatory sulfite reductase bound to DsrC provides novel insights into the mechanism of sulfate respiration. *J. Biol. Chem.* **283**, 34141–34149.

Ollivier, B., Hatchikian, C.E., Prensier, G., Guezennec, J. and Garcia, J.-L. (1991). *Desulfohalobium retbaense* gen. nov., sp. nov., a halophilic sulfate-reducing bacterium from sediments of a hypersaline lake in Senegal. *Int. J. Syst. Bacteriol.* **41**, 74–81.

Ollivier, B., Cayol, J.L. and Fauque, G. (2007). Sulphate-reducing bacteria from oil field environments and deep-sea hydrothermal vents. In L.L. Barton & W.A. Hamilton (Eds.), *Sulphate-Reducing Bacteria—Environmental and Engineered Systems* (pp. 305–328). Cambridge, UK: Cambridge University Press.

Ozawa, K., Mogi, T., Suzuki, M., Kitamura, M., Nakaya, T., Anraku, Y. and Akutsu, H. (1997). Membrane-bound cytochromes in a sulfate-reducing strict anaerobe *Desulfovibrio vulgaris* Miyazaki F. *Anaerobe* **3**, 339–346.

Ozawa, K., Meikari, T., Motohashi, K., Yoshida, M. and Akutsu, H. (2000). Evidence for the presence of an F-type ATP synthase involved in sulfate respiration in *Desulfovibrio vulgaris*. *J. Bacteriol.* **182**, 2200–2206.

Panek, H. and O’Brian, M. (2002). A whole genome view of prokaryotic haem biosynthesis. *Microbiol* **148**, 2273–2282.

Parey, K., Warkentin, E., Kroneck, P.M.H. and Ermler, U. (2010). Reaction cycle of the dissimilatory sulfite reductase from *Archaeoglobus fulgidus*. *Biochemistry* **49**, 8912–8921.

Pattarkine, M.V., Tanner, J.J., Bottoms, C.A., Lee, Y.-H. and Wall, J.D. (2006). *Desulfovibrio desulfuricans* G20 tetraheme cytochrome structure at 1.5 Å and cytochrome interactions with metal complexes. *J. Mol. Biol.* **358**, 1314–1327.

Peck, H.D.Jr., (1959). The ATP-dependent reduction of sulfate with hydrogen in extracts of *Desulfovibrio desulfuricans*. *Proc. Natl. Acad. Sci. U. S. A.* **45**, 701–708.

Peck, H.D.Jr., (1960). Evidence for oxidative phosphorylation during the reduction of sulphate with hydrogen by *Desulfovibrio desulfuricans*. *J. Biol. Chem.* **235**, 2734–2738.

Peck, H.D.Jr., (1966). Phosphorylation coupled with electron transfer in extracts of the sulfate-reducing bacterium, *Desulfovibrio gigas*. *Biochem. Biophys. Res. Commun.* **22**, 112–118.

Peck, H.D.Jr., and LeGall, J. (1982). Biochemistry of dissimilatory sulfate reduction. *Philos. Trans. R. Soc. Lond. B Biol. Sci.* **298**, 443–466.

Peck, H.D.Jr., and Lissolo, T. (1988). Assimilatory and dissimilatory sulphate reduction: enzymology and bioenergetics. *Symp. Soc. Gen. Microbiol.* **42**, 99–133.

Peck, H.D., Jr., LeGall, J., Lespinat, P.A., Berlier, Y. and Fauque, G. (1987). A direct demonstration of hydrogen cycling by *Desulfovibrio vulgaris* employing membrane-inlet mass spectrometry. *FEMS Microbiol. Lett.* **40**, 295–299.

Pereira, I.A.C. (2008). Membrane complexes in *Desulfovibrio*. In C. Friedrich & C. Dahl (Eds.), *Microbial Sulfur Metabolism* (pp. 24–35). Berlin: Springer-Verlag.

Pereira, I.A.C. and Xavier, A.V. (2005). Multi-heme cytochromes and enzymes. In R.B. King (Ed.), (2nd edn.). *Encyclopedia of Inorganic Chemistry*. Vol. 5 (pp. 3360–3376). New York: Wiley.

Pereira, I.C., Abreu, I.A., Xavier, A.V.M., LeGall, J. and Teixeira, M. (1996). Nitrite reductase from *Desulfovibrio desulfuricans* (ATCC 27774)—a heterooligomer haem protein with sulfite reductase activity. *Biochem. Biophys. Res. Commun.* **224**, 611–618.

Pereira, I.A.C., Pacheco, I., Liu, M.-Y., LeGall, J., Xavier, A.V. and Teixeira, M. (1997). Multiheme cytochromes from the sulfur-reducing bacterium *Desulfuromonas acetoxidans*. *Eur. J. Biochem.* **248**, 323–328.

Pereira, I.A.C., Teixeira, M. and Xavier, A.V. (1998a). Hemeproteins in anaerobes. In R.J.P. Williams (Ed.), *Structure and Bonding*. Vol. 91 (pp. 65–89). Berlin & Heidelberg: Springer-Verlag.

Pereira, I.A.C., Romao, C.V., Xavier, A.V., LeGall, J. and Teixeira, M. (1998b). Electron transfer between hydrogenases and mono and multiheme cytochromes in *Desulfovibrio* spp. *J. Biol. Inorg. Chem.* **3**, 494–498.

Pereira, I.A.C., LeGall, J., Xavier, A.V. and Teixeira, M. (2000). Characterization of a haem c nitrite reductase from a non-ammonifiying microorganism, *Desulfovibrio vulgaris* Hildenborough. *Biochim. Biophys. Acta* **1481**, 119–130.

Pereira, P.M., Teixeira, M., Xavier, A.V., Louro, R.O. and Pereira, I.A.C. (2006). The Tmc complex from *Desulfovibrio vulgaris* Hildenborough is involved in transmembrane electron transfer from periplasmic hydrogen oxidation. *Biochemistry* **45**, 10359–10367.

Pereira, I.A.C., Haveman, S.A. and Voordouw, G. (2007). Biochemical, genetic and genomic characterization of anaerobic electron transport pathways in sulphate-reducing delta-proteobacteria. In L.L. Barton & W.A. Hamilton (Eds.), *Sulphate-Reducing Bacteria—Environmental and Engineered Systems* (pp. 215–240). Cambridge, UK: Cambridge University Press.

Pereira, I.A.C., Ramos, A.R., Grein, F., Marques, M.C., da Silva, S.M. and Venceslau, S. (2011). A comparative genomic analysis of energy metabolism in sulfate-reducing bacteria and archaea. *Front. Microbiol.* **2**, 69. doi:10.3389/fmicb.2011.00069.

Perry, K.A., Kostka, J.E., Luther, G.W.I.I. and Nealson, K.H. (1993). Mediation of sulfur speciation by a Black Sea facultative anaerobe. *Science* **259**, 801–803.

Pfennig, N. (1989). Metabolic diversity among the dissimilatory sulfate-reducing bacteria. *Antonie Van Leeuwenhoek* **56**, 127–138.

Pfennig, N. and Biebl, H. (1976). *Desulfuromonas acetoxidans* gen. nov. and sp. nov., a new anaerobic, sulfur-reducing, acetate-oxidizing bacterium. *Arch. Microbiol.* **110**, 3–12.

Philippot, P., Van Zuylen, M., Lepot, K., Fhomazzo, C., Farquhar, J. and Van Kranendonk, M.J. (2007). Early archaeal microorganisms preferred elemental sulfur, not sulfate. *Science* **317**, 1534–1537.

Pires, R.H., Lourenço, A.I., Morais, F., Teixeira, M., Xavier, A.V., Saraiva, L.M. and Pereira, I.A.C. (2003). A novel membrane-bound respiratory complex from *Desulfovibrio desulfuricans* ATCC 27774. *Biochim. Biophys. Acta* **1605**, 67–82.

Pires, R.H., Venceslau, S.S., Morais, F., Teixeira, M., Xavier, A.V. and Pereira, I.A.
C. (2006). Characterization of the *Desulfovibrio desulfuricans* ATCC 27774

MsrMKJOP complex-A membrane bound redox complex involved in the sulfate respiratory pathway. *Biochemistry* **45**, 249–262.

Postgate, J.R. (1954). Presence of cytochrome in an obligate anaerobe. *Biochem. J.* **56**, xi–xii.

Postgate, J.R. (1956). Cytochrome $c_{3}$ and desulphoviridin: pigments of the anaerobe *Desulfovibrio desulfuricans*. *J. Gen. Microbiol.* **14**, 545–572.

Postgate, J.R. (1984). *The Sulphate-Reducing Bacteria* (2nd edn.). Cambridge, UK: Cambridge University Press.

Price, M.J., Saiz, B.L. and Barton, L.L. (1989). Ferrochelatase activity in *Azospirillum brasilense* with reference to the influence of metal cations. *Biol. Met.* **2**, 31–35.

Raaijmakers, H., Teixeira, S., Dias, J.M., Almendra, M.J., Brondino, C.D., Moura, I., Moura, J.J.G. and Romão, M.J. (2001). Tungsten-containing forms dehydrogenase from *Desulfovibrio gigas*: metal identification and preliminary structural data by multi-wavelength crystallography. *J. Biol. Inorg. Chem.* **6**, 398–404.

Raaijmakers, H., Macieira, S., et al. (2002). Gene sequence and the 1.8 angstrom crystal structure of the tungsten-containing formate dehydrogenase from *Desulfovibrio gigas*. *Structure* **10**, 1261–1272.

Rabus, R., Hansen, T.A. and Widdel, F. (2006). Dissimilatory sulfate- and sulfur-reducing prokaryotes. In: M. Dworkin, S. Falkow, E. Rosenberg, K.-H. Schleifer & E. Stackebrandt (Eds.), *The Prokaryotes*. Vol. 2 (pp. 659–768). Berlin: Springer.

Ramamoorthy, S., Sass, H., Langner, H., Schumann, P.M., Kroppenstedt, R.M., Spring, S., Overmann, J. and Rosenzweig, R.F. (2006). *Desulfosporosinus lactus* sp. nov., a sulfate-reducing bacterium isolated from pristine freshwater lake sediments. *Int. J. Syst. Evol. Microbiol.* **56**, 2729–2736.

Raux, E., Schubert, H.L. and Warren, M.J. (2000). Biosynthesis of cobaltamin (vitamin B12): a bacterial conundrum. *Cell. Mol. Life Sci.* **57**, 1880–1893.

Reddy, C.A. and Peck, H.D.Jr., (1978). Electron transport phosphorylation coupled to fumarate reduction by $\mathrm{H}_{2}$ - and $\mathrm{Mg}^{2+}$ -dependent adenosine triphosphatase activity in extracts of the rumen anaerobe *Vibrio succinogenes*. *J. Bacteriol.* **134**, 982–991.

Rees, G.N. and Patel, B.K.C. (2001). *Desulforegula conservatrix* gen. nov., sp. nov., a long-chain fatty acid-oxidizing, sulfate-reducing bacterium isolated from sediments of a freshwater lake. *Int. J. Syst. Evol. Microbiol.* **51**, 1911–1916.

Riederer-Henderson, M.A. and Peck, H.D.Jr., (1986a). In vitro requirements for formate dehydrogenase from *Desulfovibrio*. *Can. J. Microbiol.* **32**, 425–429.

Riederer-Henderson, M.A. and Peck, H.D.Jr., (1986b). Properties of formate dehydrogenase from *Desulfovibrio gigas*. *Can. J. Microbiol.* **32**, 430–435.

Robertson, W.J., Bowman, J.P., Franzmann, P.D. and Mee, B.J. (2001). *Desulfosporosinus meridiei* sp. nov., a sporforming sulfate-reducing bacterium isolated from gasoline-contaminated groundwater. *Int. J. Syst. Evol. Microbiol.* **51**, 133–140.

Rodrigues, M.L., Oliveira, T.F., Pereira, I.A.C. and Archer, M. (2006). X-ray structure of the membrane-bound cytochrome c quinol dehydrogenase NrfH reveals novel haem coordination. *EMBO J.* **25**, 5951–5960.

Romão, C.V., Louro, R., Timkovich, R., Lubben, M., Liu, M.Y., LeGall, J., Xavier, A.V. and Teixeira, M. (2000). Iron-coproporphyrin III is a natural cofactor in bacterioferritin from the anaerobic bacterium, *Desulfovibrio desulfuricans*. *FEBS Lett.* **480**, 213–216.

Romão, C.V., Ladakis, D., Lobo, S.A.L., Carrondo, M.A., Brindley, A.A., Deery, S., Matias, P.M., Pickersgill, R.W., Saraiva, L.M. and Warren, M.J. (2011). Evolution in a family of chelatases facilitated by the introduction of active site asymmetry and protein oligomerization. *Proc. Natl. Acad. Sci. U. S. A.* **108**, 97–102.

Rossi, M., Pollock, W.B.R., Reij, M.W., Keon, R.G., Fu, R. and Voordouw, G. (1993). The hmc operon of *Desulfovibrio vulgaris* subsp. *vulgaris* Hildenborough encodes a potential transmembrane redox protein complex. *J. Bacteriol.* **175**, 4699–47111.

Saraiva, L.M., da Costa, P.N., Conte, C., Xavier, A.V. and LeGall, J. (2001). In the facultative sulphate/nitrate reducer *Desulfovibrio desulfuricans* ATCC 27774, the nine-haem cytochrome c is part of a membrane-bound redox complex mainly expressed in sulphate-grown cells. *Biochim. Biophys. Acta* **1520**, 63–70.

Schauder, R. and Kröger, A. (1993). Bacterial sulphur respiration. *Arch. Microbiol.* **159**, 491–497.

Schiffer, A., Parey, K., Warkentin, E., Diederichs, K., Huber, H., Stetter, K.O., Kroneck, P.M. and Ermler, U. (2008). Structure of the dissimilatory sulfite reductase from the hyperthermophilic archaeon *Archaeoglobus fulgidus*. *J. Mol. Biol.* **379**, 1063–1074.

Schönheit, P. and Schäfer, T. (1995). Metabolism hyperthermophiles. *World J. Microbiol. Biotechnol.* **11**, 26–57.

Schumacher, W. and Kroneck, P.M.H. (1991). Dissimilatory hexahaem c nitrite reductase of ‘Spirillum’ strain 5175: purification and properties. *Arch. Microbiol.* **156**, 70–74.

Scott, J.H. and Nealson, K.H. (1994). A biochemical study of the intermediary carbon metabolism of *Shewanella putrefaciens*. *J. Bacteriol.* **176**, 3408–3411.

Sebban, C., Blanchard, L., Bruschi, M. and Guerlesquin, F. (1995). Purification and characterization of the formate dehydrogenase from *Desulfovibrio vulgaris* Hildenborough. *FEMS Microbiol. Lett.* **133**, 143–149.

Seitz, H.-J. and Cypionka, H. (1986). Chemolithotrophic growth of *Desulfovibrio desulfuricans* with hydrogen coupled to ammonification of nitrate or nitrite. *Arch. Microbiol.* **146**, 63–67.

Sharma, S., Cavallaru, G. and Rosato, A. (2010). A systematic investigation of multihaeme c-type cytochromes in prokaryotes. *J. Biol. Inorg. Chem.* **15**, 559–571.

Shen, Y.A., Buick, R. and Canfield, D.E. (2001). Isotopic evidence for microbial sulphate reduction in the early Archaean era. *Nature* **410**, 77–81.

Shi, L., Squier, T.C., Zachara, J.M. and Fredrickson, J.K. (2007). Respiration of metal (hydr)oxides by *Shewanella* and *Geobacter*: a key role for multihaem c-type cytochromes. *Mol. Microbiol.* **65**, 12–20.

Shirodkar, S., Reed, S., Romine, M. and Saffarini, D. (2011). The octahaem SirA catalyses dissimilatory sulfite reduction in *Shewanella oneidensis* MR-1. *Environ. Microbiol.* **13**, 108–115.

Sikorski, J., Lapidus, A., Copeland, A., Glavina Del Rio, T., Notan, M., Lucas, S., Chen, F., Tice, H., Cheng, J.-F., Saunders, E., Bruce, D., Goodwin, L., et al. (2010). Complete genome sequence of *Sulfurospirillum deleyianum* type strain (5175T). *Stand. Genomic Sci.* **2**, 149–157.

Simianu, M., Murakami, E., Brewer, J.M. and Ragsdale, S.W. (1998). Purification and properties of the heme- and iron-sulfur-containing heterodisulfide reductase from *Methanosarcina thermiphila*. *Biochem* **37**, 10027–10039.

Simon, J., Gross, R., Klimmek, O. and Kröger, A. (2000). The genus Wolinella. In M. Dworkin (Ed.), *The Prokaryotes: An Evolving Electronic Resource for the Microbiological Community*. (3rd edn.). New York: Springer-Verlag.

Smith, D.R., Doucette-Stamm, L.A., Deloughery, C., Lee, H.-M., Dubois, J., Aldredge, T., Bashirzadeh, R., Blakely, D., Cook, R., Gilbert, K., Harrison, D., Hoang, L., et al. (1997). Complete genome sequence of Methanobacterium thermoautotrophicum deltaH: functional analysis and comparative genomics. *J. Bacteriol* **179**, 7135–7155.

Sorokin, D.Y. and Muyzer, G. (2010). Haloalkaliphilic spore-forming sulfidogens from soda lake sediments and description of *Desulfitospira alkaliphila* gen. nov., sp. nov. *Extremophiles* **14**, 313–320.

Sorokin, D.Y., Foti, M., Tindall, B.J. and Muyzer, G. (2007). *Desulfurispirillum alkaliphilum* gen. nov. sp. nov., a novel obligately anaerobic sulfur- and dissimilatory nitrate-reducing bacterium from a full-scale sulfide-removing bioreactor. *Extremophiles* **11**, 363–370.

Stackebrandt, E., Schumann, P., Schüler, E. and Hippe, H. (2003). Reclassification of *Desulfotomaculum auripigmentum* as *Desulfosporosinus auripigmenti* corrig., comb. nov. *Int. J. Syst. Evol. Microbiol.* **53**, 1439–1443.

Stetter, K.O. and Gaag, G. (1983). Reduction of molecular sulphur by methanogenic bacteria. *Nature* **305**, 309–311.

Steuber, J. and Kroneck, P.M.H. (1998). Desulfovirdin, the dissimilatory sulfite reductase from *Desulfovibrio desulfuricans* (Essex): new structural and functional aspects of the membranous enzyme. *Inorg. Chim. Acta* **276**, 52–57.

Steuber, J., Cypionka, H. and Kroneck, P.M.H. (1994). Mechanism of dissimilatory sulfite reduction by *Desulfovibrio desulfuricans*: purification of a membrane-bound sulfite reductase and coupling with cytochrome c3 and hydrogenase. *Arch. Microbiol.* **162**, 255–260.

Steuber, J., Arendsen, A.F., Hagen, W.R. and Kroneck, P.M.H. (1995). Molecular properties of the dissimilatory sulfate reductase from *Desulfovibrio desulfuricans* (Essex) and comparison with the enzyme from *Desulfovibrio vulgaris* (Hildenborough). *Eur. J. Biochem.* **233**, 873–879.

Steudel, R., Holdt, G. and Nagorka, R. (1986). On the autoxidation of aqueous sodium polysulfide. *Zeitschrift fur Naturforschung* **41b**, 1519–1522.

Storbeck, S., Rolfes, S., Raux-Deery, E., Warren, M.J., Jahn, D. and Layer, G. (2010). A novel pathway for the biosynthesis of heme in Archaea: genome-based bioinformatics predictions and experimental evidence. *Archaea* doi:10.1155/2010/17505.

Tang, Y.J., Hwang, J.S., Wemmer, D.E. and Keasling, J.D. (2007). *Shewanella oneidensis* MR-1 Fluxome under various oxygen conditions. *Appl. Environ. Microbiol.* **73**, 718–729.

Terlesky, K.C. and Ferry, J.G. (1988). Ferredoxin requirement for electron transport from the carbon monoxide dehydrogenase complex to a membrane-bound hydrogenase in acetate-grown *Methanosarcina thermophila*. *J. Biol. Chem.* **263**, 4075–4079.

Thauer, R.K. (1988). Biochemistry of methanogenesis: a tribute to Marjory Stephenson. *Microbiology* **144**, 2377–2406.

Thauer, R.K., Jungermann, K. and Decker, K. (1977). Energy conservation in chemotrophic anaerobic bacteria. *Bacteriol. Rev.* **41**, 100–180.

HEMOPROTEINS IN DISSIMILATORY REDUCTION 89

Thauer, R.K., Kaster, A.-K., Seedorf, H., Buckel, W. and Hedderich, R. (2008). Methanogenic archaea: ecologically relevant differences in energy conservation. *Nat. Rev. Microbiol.* **6**, 579–591.

Thomas, S.H., Wagner, R.D., Arakaki, A.K., Skolnick, J., Kirby, J.R., Shimkets, L.J., Sanford, R.A. and Löffler, F.E. (2008). The mosaic genome of *Anaeromyxobacter dehalogenans* strain 2PC-C suggests an aerobic common ancestor to the delta-proteobacteria. *PLoS One* **3**, e2103. doi:10.1371/journal.pone.0002103.

Timkovich, R., Burkhalter, R.S., Xavier, A.V., Chen, L. and LeGall, J. (1994). Iron uroporphyrin I and a heme c-derivative are prosthetic groups in *Desulfovibrio gigas* rubredoxin oxidase. *Bioorg. Chem.* **22**, 284–293.

Trudinger, P.A. (1970). Carbon monoxide-reacting pigment from *Desulfotomaculum nigrificans* and its possible relevance to sulfite reduction. *J. Bacteriol.* **104**, 158–170.

Trumpower, B.L. (1990). Cytochrome bc₁ of microorganisms. *Microbiol. Rev.* **54**, 101–129.

Tucker, M.D., Barton, L.L. and Thomson, B.M. (1997). Reduction and immobilization of molybdenum by *Desulfovibrio desulfuricans*. *J. Environ. Qual.* **26**, 1146–1152.

Tucker, M.D., Barton, L.L. and Thomson, B.M. (1998). Reduction of Cr, Mo, Se and U by *Desulfovibrio desulfuricans* immobilized in polyacrylamide gels. *J. Ind. Microbiol. Biotechnol.* **20**, 13–19.

Umhau, S., Fritz, G., Diederichs, K., Breed, J., Welte, W. and Kroneck, P.M.H. (2001). Three-dimensional structure of the nonaheme cytochrome c from *Desulfovibrio desulfuricans* Essex in the Fe(III) state at 1.89 Å resolution. *Biochemistry* **40**, 1308–1316.

Valente, F.M.A., Saraiva, L.M., LeGall, J., Xavier, A.V., Teixeira, M. and Pereira, I.A.C. (2001). A membrane bound cytochrome c₃: a type II c₃ from *Desulfovibrio vulgaris* Hildenborough. *ChemBiolChem* **2**, 895–905.

Van Beelen, P., Geerts, W.J., Pol, A. and Vogels, G.D. (1983). Quantification of coenzymes and related compounds from methanogenic bacteria by high-performance liquid chromatography. *Anal. Biochem.* **131**, 285–290.

Venceslau, S.S., Lino, R.R. and Pereira, I.A. (2010). The Qrc membrane complex, related to the alternative complex III, is a menaquinone reductase involved in sulfate respiration. *J. Biol. Chem.* **285**, 22774–22783.

Venceslau, S.S., Matos, D. and Pereira, I.A.C. (2011). EPR characterization of the new Qrc complex from sulphate reducing bacteria and its ability to form a supercomplex with hydrogenase and TplC₃. *FEBS Lett.* **585**, 2177–2181.

Venkateswaran, K., Moser, D.P., Dollhopf, M.E., Lies, D.P., Saffarini, D.A., MacGregor, B.J., Ringelberg, D.B., White, D.C., Nishijima, M., Sano, H., Burghardt, J., Stackebrandt, E., et al. (1999). Polyphasic taxonomy of the genus *Shewanella* and description of *Shewanella oneidensis* sp. nov. *Int. J. Syst. Evol. Microbiol.* **49**, 705–724.

Wächtershäuser, G. (2010). Chemoautotrophic origin of life: the iron-sulfur world hypothesis. In L.L. Barton, M. Mandl & A. Loy (Eds.), *Geomicrobiology: Molecular and Environmental Perspective* (pp. 1–36). Dordrecht, The Netherlands: Springer Science+Business Media B.V.

Wagner, M., Roger, A.J., Flax, J.L., Brusseau, G.A. and Stahl, D.A. (1998). Phylogeny of dissimilatory sulfate reducers supports an early origin of sulfate respiration. *J. Bacteriol.* **180**, 2975–2982.

Walker, C.B., He, Z.L., Yang, Z.K., Ringbauer, J.A., He, Q., Zhou, J.H., Voordouw, G., Wall, J.D., Arkin, A.P., Hazen, T.C., Stolyar, S. and Stahl, D.A. (2009). The electron transfer system of syntrophically grown *Desulfovibrio vulgaris*. *J. Bacteriol.* **191**, 5793–5801.

Wang, S., Huang, H., Moll, J. and Thauer, R.K. (2010). NADP+ reduction with reduced ferredoxin and NADP+ reduction with NADH are coupled via an electron-bifurcating enzyme complex in *Clostridium kluyveri*. *J. Bacteriol.* **192**, 5115–5123.

Widdel, F. (1988). Microbiology and ecology of sulfate-reducing bacteria. In A.J.B. Zehnder (Ed.), *Biology of Anaerobic Microorganisms. Ecological and Applied Microbiology*. Vol. 3 (pp. 469–585). Munich: Carl Hanser Verlag.

Widdel, F. and Hansen, T.A. (1992). The dissimilatory sulfate-and sulfur-reducing bacteria. In A. Balows, H.G. Trüper, M. Dworkin, W. Harder & K.-H. Schleifer (Eds.), (2nd edn.), *The Prokaryotes*. Vol. 1 (pp. 583–624). New York: Springer-Verlag.

Widdel, F. and Pfennig, N. (1992). The genus *Desulfuromonas* and other Gram-negative sulfur-reducing eubacteria. In A. Balows, H.G. Trüper, M. Dworkin, W. Harder & K.H. Schleifer (Eds.), *The Prokaryotes. 2nd edn.* Vol. 4 (pp. 3379–3389). New York: Springer.

Wolfe, B.M., Lui, S.M. and Cowan, J.A. (1994). Desulfoviviridin, a multimeric-dissimilatory sulfite reductase from *Desulfovibrio vulgaris* (Hildenborough). Purification, characterization, kinetics and EPR studies. *Eur. J. Biochem.* **223**, 79–89.

Wood, P.M. (1978). A chemiosmotic model for sulphate respiration. *FEBS Lett.* **95**, 12–18.

Yagi, T. (1969). Formate: cytochrome oxidoreductase of *Desulfovibrio vulgaris*. *J. Biochem. (Tokyo)* **66**, 473–478.

Yagi, T. (1979). Purification and properties of cytochrome c-553, an electron acceptor for formate dehydrogenase of *Desulfovibrio vulgaris*, Miyazaki. *Biochim. Biophys. Acta* **548**, 96–105.

Yagi, T. (1991). Bacterial NADH-quinone oxidoreductase. *J. Bioenerg. Biomembr.* **23**, 211–225.

Yagi, T. (1994). Monoheme cytochromes. *Methods Enzymol.* **243**, 104–118.

Yamanaka, T. (1992). *The Biochemistry of Bacterial Cytochromes*. Berlin: Springer-Verlag.

Zaunmüller, T., Kelly, D.J., Glöckner, F.O. and Under, G. (2006). Succinate dehydrogenase functioning by a reverse redox loop mechanism and fumarate reductase in sulfate-reducing bacteria. *Microbiol.* **152**, 2443–2453. doi:10.1099/mic.0.28849-0.

Zöphel, A., Kennedy, M.C., Beinert, H. and Kroneck, P.M.H. (1988). Investigations on microbial sulfur respiration. 1. Activation and reduction of elemental sulfur in several strains of Eubacteria. *Arch. Microbiol.* **150**, 72–77.

Zöphel, A., Kennedy, M.C., Beinert, H. and Kroneck, P.M.H. (1991). Investigations on microbial sulfur respiration: isolation, purification, and characterization of cellular components from *Spirillum 5175*. *Eur. J. Biochem.* **195**, 849–856.
