
doi:10.1016/j.jmb.2012.06.012

J. Mol. Biol. (2012) 422, 508–518

Contents lists available at www.sciencedirect.com

Journal of Molecular Biology

journal homepage: http://ees.elsevier.com.jmb

# Autoproteolytic Activation of ThnT Results in Structural Reorganization Necessary for Substrate Binding and Catalysis

Andrew R. Buller${}^{1\dagger}$, Jason W. Labonte${}^{2\dagger}$, Michael F. Freeman${}^{2}$, Nathan T. Wright${}^{3}$, Joel F. Schildbach${}^{3}$ and Craig A. Townsend${}^{1,2*}$

${}^{1}$Department of Biophysics, The Johns Hopkins University, Baltimore, MD 21218, USA  
${}^{2}$Department of Chemistry, The Johns Hopkins University, Baltimore, MD 21218, USA  
${}^{3}$Department of Biology, The Johns Hopkins University, Baltimore, MD 21218, USA  

Received 2 April 2012; received in revised form 2 June 2012; accepted 8 June 2012 Available online 15 June 2012

Edited by G. Schulz

Keywords: thienamycin biosynthesis; cis-autoproteolysis; mechanism of chloromethyl ketone inhibition; N-terminal nucleophile; X-ray crystallography

cis-Autoproteolysis is a post-translational modification necessary for the function of ThnT, an enzyme involved in the biosynthesis of the β-lactam antibiotic thienamycin. This modification generates an N-terminal threonine nucleophile that is used to hydrolyze the pantetheinyl moiety of its natural substrate. We determined the crystal structure of autoactivated ThnT to 1.8 Å through X-ray crystallography. Comparison to a mutationally inactivated precursor structure revealed several large conformational rearrangements near the active site. To probe the relevance of these transitions, we designed a pantetheine-like chloromethyl ketone inactivator and co-crystallized it with ThnT. Although this class of inhibitor has been in use for several decades, the mode of inactivation had not been determined for an enzyme that uses an N-terminal nucleophile. The co-crystal structure revealed the chloromethyl ketone bound to the N-terminal nucleophile of ThnT through an ether linkage, and analysis suggests inactivation through a direct displacement mechanism. More importantly, this inactivated complex shows that three regions of ThnT that are critical to the formation of the substrate binding pocket undergo rearrangement upon autoproteolysis. Comparison of ThnT with other autoproteolytic enzymes of disparate evolutionary lineage revealed a high degree of similarity within the proenzyme active site, reflecting shared chemical constraints. However, after autoproteolysis, many enzymes, like ThnT, are observed to rearrange in order to accommodate their specific substrate. We propose that this is a general phenomenon, whereby autoprocessing systems with shared chemistry may possess similar structural features that dissipate upon rearrangement into a mature state.

© 2012 Elsevier Ltd. All rights reserved.

*Corresponding author. Department of Chemistry, The Johns Hopkins University, Baltimore, MD 21218, USA. E-mail address: ctownsend@jhu.edu.

† A.R.B. and J.W.L. contributed equally to this work.

Present address: J. W. Labonte, Department of Chemical and Biomolecular Engineering, The Johns Hopkins University, Baltimore, MD 21218, USA; M. F. Freeman, Kekulé Institute of Organic Chemistry and Biochemistry, University of Bonn, 53223 Bonn, Germany; N. T. Wright, Department of Chemistry, James Madison University, Harrisonburg, VA 22807, USA.

Abbreviations used: CMK, chloromethyl ketone; PβS, proteasome β-subunit; GA, glycosylasparaginase; CA, cephalosporin acylase; HMK, halomethyl ketone; PDB, Protein Data Bank.

0022-2836/$ - see front matter © 2012 Elsevier Ltd. All rights reserved.

# Introduction

Autoproteolytic proteins pose unique challenges to understanding structure–function relationships because their cleavage is an intrinsic consequence of folding. This post-translational modification is involved in the regulation of protein folding, cell signaling, nucleoporin biogenesis, apoptosis, and the function of many enzymes. These studies have established that self-cleavage proceeds through an N–O(S) acyl shift mechanism: an active-site nucleophile (Thr, Ser, or Cys) attacks into its N-terminal peptide bond generating a (thia)oxazolidine intermediate (Fig. 1a). This transient species collapses to a (thio)ester, which is subsequently hydrolyzed to yield two subunits bearing new N- and C-termini. The N-terminal nucleophile (Ntn) generated by autoproteolysis is used by enzymes, such as the Thr-utilizing proteasome β-subunit (PβS), in a catalytic diad (or triad) for catalysis. It was originally believed that all Ntn-utilizing enzymes belonged to the same superfamily (dubbed the Ntn superfamily). However, subsequent research showed that a second group of enzymes exists (the D/O superfamily), which is related to the Ntn superfamily through convergent evolution. Together, these enzymes perform a diverse array of hydrolysis and acyl transfer reactions.

We recently reported use of the pantetheinyl hydrolase ThnT (Fig. 1b), a D/O enzyme involved in the biosynthesis of the potent β-lactam antibiotic thienamycin, as a model system with which to study autoproteolysis. Our analysis showed that a local conformational rearrangement is necessary for autoactivation and indicated that the majority of reported uncleaved structures from the Ntn superfamily do not, in fact, represent a cleavage-competent state. Consequently, a comparison between the functional active site before and after autoactivation is possible for only two systems, glycosylasparaginase (GA) and PβS. This lack of mechanistically relevant structural information constitutes a fundamental gap in our understanding of how a single protein sequence is optimized to facilitate both autoproteolysis and enzymatic catalysis in its mature form. Because cephalosporin acylase (CA) and penicillin acylase, both members of the Ntn superfamily, are currently the subject of directed evolution experiments to improve the industrial production of β-lactam antibiotics, further insights may provide valuable lessons in the development of enzymes with improved activity. Addressing this problem will rely on the ability to compare and contrast structural changes upon autoactivation for both Ntn and D/O superfamilies. To this end, we

---

**Fig. 1. Mechanism of autoactivation and substrate hydrolysis.**  
(a) Abstraction of a proton by a general base (B:) initiates cis-autoproteolysis with Thr (X=O, Y=CH₃), Ser (X=O, Y=H), or Cys (X=S, Y=H) nucleophile. The activated residue attacks into its N-terminal amide bond, resulting in a (thia)oxazolidine ring that collapses to a (thio)ester with concomitant protonation of the amine leaving group by a general acid (AH). The α-amine activates a water molecule for hydrolysis of the (thio)ester, creating a new N-terminal residue.  
(b) Mechanism of ThnT hydrolysis of a pantetheinylated carbapenem. The N-terminal amine acts as a general base to activate the nucleophile for attack into the carbonyl of the pantetheine side chain. The α-ammonium donates a proton back to the amine leaving group, generating an acyl-enzyme intermediate. In a manner directly comparable to autoactivation, the α-amine again activates a water molecule to hydrolyze the substrate from the enzyme, returning it to its resting state. This hydrolytic chemistry is standard among Ntn-utilizing enzymes.
sought to extend our analysis of ThnT by determining the mature structure of the enzyme and elucidating the features necessary for substrate binding and catalysis.

# Results and Discussion

## Structural transitions accompanying autoproteolysis

A 6-h room temperature incubation of ThnT purified from heterologous expression afforded autoproteolytically activated enzyme suitable for X-ray crystallography. Wild-type ThnT crystallized under similar conditions and in the same space group as the mutationally inactive precursor ThnT T282C. Diffraction to high resolution and phasing by molecular replacement yielded the 1.8-Å crystal structure of wild-type ThnT. For clarity, all residues of ThnT referenced hereafter will be numbered according to their position in the precursor protein. Autoproteolysis generates two subunits, α and β, composed of residues 1–281 and 282–399, respectively. Like most structurally characterized Ntn enzymes, the subunits of ThnT remain intimately intertwined after activation with global features comparable to their uncleaved forms (Fig. 2a). The N-6His tag and the first 23 residues of the α-subunit were not observed, as was the case for the T282C structure. A short linker sequence leading into the scissile bond of the proenzyme, residues 274–281, showed few tertiary interactions and modeled with very high average B-factors (41.4 Å²). These disordered residues comprise the C-terminus of the α-subunit, and as we anticipated, they are not observed in the structure of mature ThnT. Interestingly, comparable disorder in precursor proteins and nascent C-termini has been found in many autoproteolytic systems. This observation is readily explained as a strategy employed to promote dissociation of the C-terminus, which must occur to expose the active site of the mature enzyme.

The N-terminal residue of the β-subunit is the characteristic N-terminal threonine residue central to autoprocessing and substrate catalysis (Fig. 2b). Surprisingly, the peptide ω angle of Thr282 consistently refined to a strained 167°. Comparable strain was observed at this position in the proenzyme inactive state, and we hypothesized that this destabilization was used to help drive formation of the reactive conformation. Hence, it appears that retention of this unfavorable geometry is a consequence of the selection for an active site that can promote two different chemical reactions. The energetic penalty of straining this bond may be offset by the hydrogen bond between the carbonyl of Thr282 and the amide nitrogen of Asn217, which would be disrupted upon relaxation of the ω angle. The amide proton of Phe144 and δNH₂ of Asn217

![Figure 2](attachment:figure_2.png)

Fig. 2. Active site of autoproteolytically activated ThnT. (a) Heterodimeric ThnT has extensive quaternary interactions between the α-subunit (red) and the β-subunit (silver). The immediate environment of the catalytic, N-terminal threonine residue (shown) is composed of elements from each subunit. For clarity, only a single heterodimer of the asymmetric unit is shown. (b) Residues Phe144 and Asn217 from the α-subunit form the oxyanion hole of ThnT and the three residues D321, G322, and D323 are positioned to activate the γOH of Thr282, allowing for its role as the catalytic nucleophile. The position of a bound glycerol molecule suggests an orientation for the pantetheine moiety of the substrate. Two waters observed in the active site may be displaced upon substrate binding or repositioned to hydrolyze the acyl-enzyme intermediate generated during catalysis.
comprise the oxyanion hole of ThnT for autoactivation. ${ }^{13}$ In the major conformer of the precursor structure, a single water molecule occupied the oxyanion hole, taking the place of the reactive carbonyl. A similar placeholder is not observed for mature ThnT; instead, a glycerol molecule and additional water molecules are bound in the active site (Fig. 2b).

In addition to dissociation of the nascent C-terminus, three structural rearrangements in the vicinity of the active site were identified (Fig. 3a). Notably, the solvent-filled groove that may represent the substrate binding site is not present prior to autoactivation and is only revealed upon a conformational change in residues Arg74–Thr80, which includes cis-trans isomerization of the Gly77–Pro78 peptide bond (Fig. 3b). This dramatic structural rearrangement is promoted electrostatically by reorienting the carbonyl of Arg74 away from Asp321 and positioning it 1.4 Å closer to the side chain of Lys300. The new conformation allows the amide nitrogens of Gly76 and Gly77 to hydrogen bond with the backbone of Leu318 and Asp321, respectively. Additionally, there is a net release of four water molecules, which is entropically favorable. ${ }^{21}$ These transitions stabilize the loop

relative to the uncleaved form, which is corroborated by a decrease in the average $B$-factor of the loop from 18.5 Å² prior to cleavage to 11.6 Å² in the mature enzyme. Together, these data demonstrate that specific structural features of ThnT promote the isomerization of the Arg74–Thr80 loop.

Two other motifs undergo a structural transition upon autoproteolysis. Residues Leu318–Gly322 were observed in two conformations of one subunit in the T282C structure but collapse to a single observable state in both subunits of the wild-type enzyme (Fig. 3c). The role of this apparent conformational flexibility is unclear. However, proximity to the active site and hydrogen bonding interactions with the Arg74–Thr80 loop (described above) suggest its relevance. Residues Asp338–Phe348 were not observed in the structure of the precursor protein but were modeled in the structure of mature ThnT. A subset of these, Pro337–Asp341, refined with a high average $B$-factor of 43.9 Å², suggesting that they retain considerable flexibility after autoactivation. Conversely, Ala342–Phe348 were well resolved, with the side chain of Phe348 extending across the dimer interface and dividing the active site of the adjacent heterodimeric unit into two distinct sections. Because this conformation is not

![Figure 3](attachment:figure_3.png)

Fig. 3. Structural changes accompanying autoactivation. (a) Ribbon diagram showing the residues that undergo a major structural change upon autoproteolysis. Residues A274–N281 are disordered after autoproteolysis and were not modeled for wt-ThnT. The uncleaved structure (from PDB ID: 3S3U) is shown in red, and the mature enzyme is shown in blue, with the N-terminal Thr nucleophile boxed. (b) Isomerization of the G76–P77 peptide bond from *cis* (proenzyme state) to *trans* (mature state) is accompanied by reorganization of the entire R74–T80 loop. Waters that form hydrogen bonds with this loop in each state are displayed as spheres. For clarity, only the ribbon diagram of the mature enzyme is displayed. (c) The loop harboring S320–D321 was observed in two conformations in one subunit of the uncleaved protein. This converts to a single observable conformation after autoproteolysis. One pre-cleavage conformation would have a steric clash with the *trans* conformation of the G76–P77 loop, suggesting that the dynamics of these loops may be coupled to one another.

sterically accessible prior to dissociation of the newly formed C-terminus, formation of the observed state must occur relatively late in the maturation of the active site.

### Active-site features of ThnT

The structure of autoproteolytically activated ThnT shows that Thr282 has adopted the g− rotamer, which positions the γOH such that it forms a hydrogen bond with the α-amino group. The Ntn was originally proposed to constitute a “single-amino-acid catalytic center”; however, subsequent rigorous analysis showed that auxiliary residues are critical to catalysis. The α-amine of ThnT is adjacent to two hydrogen bond acceptors: the carboxylate of Asp321 and the carbonyl of Asn217. These residues correctly orient the nitrogen lone pair to accept a proton from the nucleophile (Fig. 2b). Interestingly, Gly322 is part of the network of residues that undergo structural rearrangement upon autoactivation, and the amide nitrogen shifts 0.8 Å to form a tight 2.6-Å hydrogen bond with the nucleophile. The oxyanion hole residues Phe144 and Asn217 maintain their orientation to stabilize the tetrahedral oxyanion intermediates generated during substrate catalysis. Hence, the hydrogen bond interactions between Asn217 and Thr282 define the proper active-site geometry for activation of both the nucleophile and the electrophile. Furthermore, this network leaves the γOH of Thr282 with an unsatisfied lone pair directed at the oxyanion hole.

ThnT and PβS have distinct evolutionary lineage but share autoactivation through *si*-face attack into the scissile bond and amidohydrolase chemistry in their mature form. Hence, it is now possible to distinguish between system-specific features and

general features of their active sites. The catalytic activity of PβS relies on a hydrogen bond network very similar to that shown in Fig. 2b. However, the active site of each protein is formed from different parts of their topologically distinct folds (Fig. 4), reinforcing their relationship through convergent evolution. Consequently, it appears that this hydrogen bond network represents an optimal balance between facile autoactivation and substrate catalysis. We recently reported that the Thr γ-methyl may accelerate the intramolecular cyclization that occurs during the N–O acyl shift in autoproteolysis. This phenomenon is known as the RRE (Reactive Rotamer Effect), and a similar effect may apply to any reaction for which the rotamer distribution may be shifted to favor the reactive state by the addition of steric bulk. This is precisely the case for ThnT and PβS, where the inactive g+ rotamer is structurally viable for both Thr and Ser but is destabilized by Thr because the γ-methyl shields a hydrogen bond with Gly322 (in ThnT) or Lys33 (in PβS). Similarly, the g+ rotamer cannot be formed for the ornithine acyltransferases (members of the D/O superfamily) because it would cause a strong 1.7-Å steric clash with the Thr111 in the oxyanion hole. Hence, comparison of ThnT with PβS and other Ntn enzymes reveals unique features of each active site that promote formation of a reactive state. Combined with the role of the RRE in autoproteolysis, we believe that these effects strongly and specifically contribute to the selection of Thr nucleophiles in autoproteolytic systems.

### Formation of a potential substrate binding pocket

The position and orientation of the bound glycerol molecule define a groove that could

![Diagram](attachment://image.png)

Fig. 4. Topology diagrams of PβS and ThnT. (a) Coloring of PβS from dark (N-terminus) to light (C-terminus) shows that the αββα architecture has unique connectivity at each half of the fold, which is composed of α-helices and a pair of antiparallel β-sheets. The nucleophile is labeled T1. (b) The global fold of ThnT arose from a gene duplication event, as evidenced by the identical connectivity of the central αββα unit (colored) characteristic of the D/O superfamily of autoproteolytic enzymes. The nucleophile, T282, is adjacent to a parallel β-strand, and the oxyanion hole element F144 is on a geometrically identical region of the protein as G47 of PβS, whereas N217 is located closer to the nucleophile in the primary sequence. Non-conserved structural motifs are colored white. Image was generated with TopDraw.

represent part of the substrate binding pocket for the pantetheine moiety of the substrate. Interestingly, residues from all three mobile loops participate in the formation of this pocket. The peptide bond of Gly77, which undergoes *cis-trans* isomerization, is 5.1 Å away from the catalytic nucleophile, further implicating the rearrangement in enzyme function. Because no other uncleaved D/O structures have been published, we cannot compare the transitions described here with homologues. The well-characterized enzymes GA and PβS from the Ntn superfamily show minor structural differences after autoproteolysis, other than dissociation of the nascent C-terminus. [14,16] However, these changes are not on the same scale as those observed for ThnT. Interestingly, prolonged incubation of ThnT resulted in further proteolysis of the α-subunit (Fig. S1). This raised the possibility that a secondary cleavage, similar to that observed for CA, [27] might also occur for ThnT. The slow timescale of the apparent hydrolysis, however, suggested that the phenomenon could be due to a contaminating protease. Although inclusion of protease inhibitors failed to prevent this cleavage, N-terminal sequencing showed that hydrolysis occurred two residues prior to the first observed residue in the crystal structure. Because this is a disordered region far from the active site, we do not believe that this secondary cleavage is relevant to the function of ThnT *in vivo*.

The scope of the structural changes revealed by our analysis of ThnT reinforces the general model that autoproteolytic maturation is more sophisticated than the obvious chemistry of bond making and bond breaking. Rather, considerable protein motion must occur as the protein shifts into a new folded state ensemble. Considering the broad implications of this hypothesis, we sought additional evidence to validate the relevance of the groups undergoing significant rearrangement by experimentally demonstrating their importance in substrate binding. To define the binding pocket and interactions with the substrate, we desired to co-crystallize the enzyme in a Michaelis complex. This has been accomplished for GA utilizing a Thr→Cys variant of the nucleophile that was capable of slow autoproteolysis and substrate hydrolysis. [15] However, a similar scheme for ThnT was not possible because the comparable Thr→Cys variant does not mature on an accessible timescale. [13] Instead, we pursued development of a covalent inhibitor harboring the pantetheine moiety that could be used with the wild-type enzyme.

### Design of a covalent chloromethyl ketone inactivator of ThnT

The halomethyl ketones (HMKs) are a potent class of inactivators that have been used to covalently

(a)

CMK Pantoamide

![image](https://i.imgur.com/1234567.png)

(b)

![image](https://i.imgur.com/8901234.png)

Fig. 5. Attachment of a covalently bound inhibitor. (a) Structure of the CMK pantoamide and the N-terminal Thr nucleophile of ThnT. (b) Active site of inactivated ThnT with the $(F_{o}-F_{c})$ difference electron density map contoured at 3.0 σ. The location of the C′ carbon and a methyl group of the inactivator is below the contour level but confirmed in subsequent refinement. Contiguous density extends from T282 onto the inactivator and is consistent with formation of an ether linkage between γO of T282 and C′ of the inhibitor. Hydrogen bonding interactions (black) define the specific mode of binding reported here.

inhibit both serine and cysteine proteases. In the case of Ser proteases, both the catalytic nucleophile and histidine of the catalytic triad can be alkylated by HMKs. [28] Alternatively, inhibition of Cys proteases results in alkylation of only the thiol, implying inhibition through a different mechanism. [29] Although HMKs have been reported to inactivate the PβS, [30] the mode and mechanism of inhibition have not been reported for a Thr protease. Hence, development of an HMK inhibitor of ThnT could simultaneously answer the long-standing question concerning the mode of attachment of HMKs to this class of enzyme while providing detailed information about the substrate binding interactions of ThnT. To this end, we used D-pantothenic acid as the starting material in a four-step synthesis of the chloromethyl ketone (CMK) derivative $N$-(4-chloro-3-oxobutyl)pantoamide (Fig. 5a) through a diazomethyl ketone intermediate (Scheme S1, see Supplemental Information for details of the synthesis). Incubation of the CMK pantoamide inhibitor with ThnT caused complete loss of pantetheine hydrolysis activity, as measured by a qualitative TLC (thin-layer chromatography) assay. We also tested the effect of the CMK pantoamide on the secondary cleavage events of ThnT and observed no changes (Fig. S1), excluding the possibility that they are

caused by ThnT performing low-efficiency intermolecular proteolysis.

### Active-site features defined by the inhibitor-bound structure

A 10-fold molar excess of the CMK pantoamide was incubated with ThnT for 1 h, which was immediately used for crystallization. Crystals grew under the same conditions as native ThnT and yielded a 2.0-Å X-ray structure. Subsequent model building and refinement clearly showed loss of electron density corresponding to the bound glycerol in the wild-type crystal (Fig. 2b). Instead, we observed continuous electron density extending onto Thr282 corresponding to a nonprotein molecule covalently linked to the γO of the active-site nucleophile (Fig. 5b). Given the high-resolution data and the structure of the CMK pantoamide, there was a single possible mode of binding that satisfied the electron density: loss of the chloride and formation of an ether linkage between Thr282 and the inhibitor. Because ThnT and PβS have similar active-site geometry, we hypothesize that this mode of inactivation is common between them.

The bridging methylene unit itself (C′) is the only component that distinguishes the bound species and the acyl-enzyme intermediate formed during substrate catalysis. Consequently, there is very little measurable distortion in the active site. All three bond angles that include the γO are compressed to ~102°. Comparison with the wild-type structure shows that the ψ angle of Thr282 has rotated 10° such that the derivatized side chain is deflected away from the oxyanion hole. Additional rotation of the ψ angle would provide space to allow the γO-bearing bond angles to relax. However, the system appears to have settled in an energetic minimum that maintains the hydrogen bond between Gly322 and Thr282 γOH. The lack of grossly unfavorable structural deviations introduced upon chemical derivatization indicates that inhibitor binding did not disrupt biologically relevant interactions.

The ketone of the inhibitor was designed to mimic the electrophilic amide of the substrate. As was predicted, the inactivated complex shows that this group has bound in the oxyanion hole of the protein, with hydrogen bonds to the backbone amide of Phe144 and the side chain of Asn217. The remainder of the inhibitor extends out toward the protein surface, with its amide nitrogen forming a hydrogen bond to Gly77. Ser104 acts as a hydrogen bond donor to the carbonyl of the amide and a hydrogen bond acceptor to the secondary hydroxyl. Simultaneous satisfaction of these hydrogen bonds restricts the inhibitor to the observed conformation. Lastly, the terminal hydroxyl forms two water-mediated hydrogen bonds with the carbonyl of Pro78 (Fig. 5b).

---

### Evidence supporting inactivation through direct displacement

Although a definitive mechanism for inactivation of ThnT is beyond the scope of the present study, the co-crystal structure establishes that ThnT is alkylated at the nucleophilic position and that few alternatives exist with respect to the mechanism of this derivatization. For the Ser proteases, the nucleophile attacks into the carbonyl forming an oxyanion that displaces the adjacent halide forming an oxirane intermediate. This three-membered ring is subsequently opened by attack of the His residue of the catalytic triad, resulting in a second covalent attachment. Were a comparable mechanism involving an oxirane to occur in the inactivation of ThnT, its formation would obey the well-known stereoelectronic requirement that attack of the oxyanion into C′ proceeds anti-periplanar to the chloride leaving group (Fig. 6b). The location of the oxyanion is well defined by the location of the oxyanion hole of ThnT. Modeling this species shows that placement of the chloride anti-periplanar to the oxyanion results in a strong steric clash with Ser320 and the carboxylate of Asp321. Because this geometry is intrinsic to oxirane formation, and the ThnT active site harbors features that would significantly impede expulsion of the chloride, we do not believe that inactivation of ThnT occurs through an oxirane intermediate. Instead, the ether linkage reported here is most similar to the thioether formed upon derivatization of the cysteine proteases papain, SARS-CoV (Severe Acute Respiratory Syndrome coronavirus) peptidase, and hepatitis A viral 3C. For the latter two enzymes, an episulfonium species was also trapped, presumably as an intermediate of the inhibition process. A comparable mechanism for ThnT would involve a highly unstable oxonium species that is unlikely to occur (Fig. 6c). Together, these constraints suggest that the mechanism of HMK inhibition in ThnT is distinct from those reported for the serine and cysteine proteases.

Because formation of the tetrahedral intermediate is reversible, a competitive and irreversible SN2 attack of T282 into C′ could account for the observed ether linkage. Notably, this reaction would proceed through a trigonal bipyramidal transition state that is intrinsically stabilized by the adjacent carbonyl π-system, and the requisite orientation of the chloride for backside attack is reasonable, as indicated by modeling the complex of ThnT and the CMK pantoamide. Consequently, we believe that the simplest, least-motion mechanism is the most likely: derivatization of ThnT occurs by direct displacement of the chloride.

---

### The role of ThnT in thienamycin biosynthesis

It has been shown that ThnT can hydrolyze a pantetheinylated carbapenam substrate, and

Autoproteolytic Activation of ThnT

---

Fig. 6. Potential mechanisms of inhibition of ThnT by the CMK inhibitor. (a) Direct displacement of the chloride follows a traditional $S_{N}2$ reaction. Evidence suggests that this is the most likely mechanism for ThnT inhibition. (b) The oxyanion of the tetrahedral intermediate expels the chloride in an intramolecular cyclization reaction. This parallels the accepted mechanism of inhibition for traditional Ser proteases, where the imidazole side chain of the catalytic triad subsequently attacks C' forming a second covalent bond to the inhibitor. Both formation of the oxirane and its collapse to an ether are unlikely to occur within the ThnT active site. (c) After formation of the oxyanion intermediate, a second nucleophilic attack occurs, as has been proposed for thioether formation in Cys proteases. The oxonium species invoked by this mechanism would be too high energy to occur.

circumstantial evidence suggests that it acts late in the biosynthetic pathway. [12,36,37] Although the data presented here do not provide a ready explanation for the inability of ThnT to cleave phosphopantetheine, the catalytic activity of ThnT is clear. Formation of the Michaelis complex places the carbonyl of the scissile amide into the oxyanion hole (Fig. 7). The majority of the enzyme-substrate interactions are in the pantothenate binding region defined by the ThnT-inhibitor structure (Fig. 5b), consistent with the capacity of ThnT to efficiently hydrolyze a variety of pantetheinylated compounds. [12] The cysteamine moiety of the substrate provides the ideal distance for the carboxylate of thienamycin to form a salt bridge with Arg148. The catalytic nucleophile is centered within the active site, and its exposure to solvent is minimized by Phe144 and Phe348 from the opposite subunit. Like other N-terminal amidohydrolases, [5,38] the α-amine deprotonates the hydroxyl of Thr282 to generate a reactive nucleophile. Attack into the substrate generates a tetrahedral intermediate whose collapse is concurrent with protonation of the amine leaving group. Studies have shown that amidohydrolases often provide a hydrogen bond acceptor to facilitate inversion of the nitrogen, [39] and we hypothesize that Ser320 may satisfy this role for ThnT (Fig. S2). Expulsion of the amino leaving group generates thienamycin and an acyl-enzyme intermediate. A water molecule in a position similar to Wat1 or Wat2 (Fig. 2b) is then activated by the α-amine, which hydrolyzes the acyl-enzyme intermediate, regenerating the ThnT active site.

### Concluding Remarks

Combined with our initial study of ThnT, the data reported here constitute the first glimpse into the complete “life cycle” of an enzyme from the D/O superfamily (Fig. S3). [13] When linked with existing information from the Ntn superfamily, these data enabled an expanded analysis of autoproteolytic activation. In all systems, dissociation of the nascent C-terminus after autoproteolysis is favored by a lack of tertiary contacts. When autoproteolysis is

Fig. 7. Model of the ThnT Michaelis complex with a pantetheinylated carbapenem. This model shows that the substrate binding cavity is well suited to accommodate a carbapenem substrate and that a positively charged residue, R148, is positioned to form a salt bridge with the carboxylate moiety of the substrate.

initiated through a *si*-face attack, the same oxyanion hole is used for both self-cleavage and substrate catalysis. Within the active site, the Thr γ-methyl facilitates this chemistry by directing the nucleophile into a reactive state. Lastly, we identified large structural rearrangements necessary for the formation of the ThnT substrate binding pocket. Although PβS is “insignificantly different” before and after autoproteolysis, most other enzymes from the Ntn superfamily (including penicillin acylase, CA, and γ-glutamyl transpeptidase) show evidence of a unique conformational rearrangement. These enzymes each contain a high degree of similarity with the ThnT proenzyme active site but are structurally divergent after autoproteolysis. We propose that this is evidence of a more general phenomenon. Autoprocessing systems with shared chemistry, regardless of lineage, will have comparable features prior to autoprocessing that may dissipate as each protein rearranges to its unique mature state. The observation that two markedly different conformations of the Arg74–Thr80 loop of ThnT (Fig. 3b) facilitate each of the hydrolytic chemistries highlights the orchestrated complexity of the structure–function relationship in autoproteolytic enzymes. Not only does sequence encode for an intrinsically reactive folded state but also, as we have shown here for ThnT, the downstream consequences of this self-cleavage result in spontaneous reorganization to the catalytically competent state of the mature enzyme.

### Materials and Methods

#### Crystallization, data collection, and structure determination

ThnT was expressed, purified, and concentrated to 7.7 μg/μL as previously described for the inactive T282C variant. The enzyme was incubated at 20 °C for 6 h prior to being flash frozen for storage at –80 °C. The ThnT-inactivator complex was prepared by adding a 10-fold molar excess of the inactivator and incubating it at 20 °C for 1 h immediately prior to use in crystallization attempts. Crystals were grown by sitting-drop vapor diffusion against a 1-mL reservoir solution of 14–18 % polyethylene glycol 3350 and 0.5–0.7 M sodium acetate. Crystals grew overnight and were cryoprotected by immersion into 20% polyethylene glycol 3350, 0.7 M sodium acetate, and 20% glycerol prior to flash freezing in N₂. Crystallization and data collection for ThnT were carried out at X6A at the National Synchrotron Light Source. Data for the ThnT–inhibitor complex were collected on BL12-2 at the Stanford Synchrotron Light Source. Both structures were solved by molecular replacement using the Phaser program, as implemented in the CCP4 package. The structure of the uncleaved T282C variant of ThnT [Protein Data Bank (PDB) ID: 3S3U], with dual occupancies removed, was used as the search model. Iterative rounds of model building were performed in Coot, using REFMAC5 for refinement. After the protein and solvent atoms were defined, TLS operators were added and further refined in REFMAC5, which yielded significant improvements in \( R_{\text{free}} \) for both models. Crystallographic and refinement statistics are listed in Table 1. Coordinates were deposited to the PDB with accession numbers 3TM1 (wt-ThnT) and 3TM2 (inactivated complex).

---

### Characterization of N-terminal hydrolysis

Purified ThnT was diluted to 2 μg/μL into buffer [50 mM Tris (pH 7.5) and 50 mM KCl, with 10% glycerol] with and without 20-fold molar excess of CMK pantoamide inhibitor and incubated at 37 °C. Samples were withdrawn every 12 h for SDS-PAGE analysis and, at 12 h and 48 h, to assay for pantethine hydrolysis activity. Samples were run on 15% SDS-PAGE for 1 h and visualized with Coomassie Blue staining (Fig. S1). Aliquots containing the α-subunit degradation products were rerun on 15% SDS-PAGE for 3 h and electroblotted onto a

---

### Table 1. Crystallographic statistics for data collection and structure refinement

| Modification | Native | O-(2-Oxo-4-pantoimidobutyl) threonine |
|--------------|--------|---------------------------------------|
| Space group  | \( P2_{1}2_{1}2 \) | \( P2_{1}2_{1}2 \) |
| Cell dimensions \( a, b, c (\AA) \) | 141.6, 67.8, 73.7 | 141.5, 67.3, 73.8 |

#### Data processing

| Parameter | Native | O-(2-Oxo-4-pantoimidobutyl) threonine |
|-----------|--------|---------------------------------------|
| Beamline  | BNL X6A | SSRL 12.2 |
| Wavelength (\AA) | 0.8001 | 0.9788 |
| Resolution (\AA) | 50–1.80 | 50–1.9 |
| Last bin | 1.85–1.80 | 2.05–2.00 |
| No. of observations | 524,211 | 320,826 |
| Completeness (%) | 96.2 (72.0) | 99.7 (98.1) |
| \( R_{\text{merge}} \) (%) | 9.7 (41.5) | 12.1 (65.3) |
| \( I / \sigma I \) | 19.9 (2.0) | 16.1 (2.1) |
| Redundancy | 8.2 (3.3) | 6.7 (5.5) |

#### Refinement

| Parameter | Native | O-(2-Oxo-4-pantoimidobutyl) threonine |
|-----------|--------|---------------------------------------|
| Total no. of reflections | 60,481 | 45,361 |
| Total no. of atoms | 5511 | 5471 |
| No. of waters | 430 | 368 |
| Figure of merit | 0.887 | 0.877 |
| \( R_{\text{work}} \) (%) | 17.0 (32.3) | 16.5 (25.3) |
| \( R_{\text{free}} \) (%) | 19.7 (35.0) | 20.2 (30.3) |
| Ramachandran plot allowed (%) | 100.0 | 99.7 |
| Ramachandran plot favored (%) | 97.9 | 97.9 |
| Ramachandran plot outliers (%) | 0.0 | 0.3 |

Values in parenthesis are for the highest-resolution shell.  
\( R_{\text{merge}} \) is \( \sum |I_o - I| / \sum I_o \), where \( I_o \) is the intensity of an individual reflection, and \( I \) is the mean intensity for multiple recorded reflections.  
\( R_{\text{work}} \) is \( ||F_o - F_c|| / F_o \), where \( F_o \) is an observed amplitude, and \( F_c \) is a calculated amplitude.  
\( R_{\text{free}} \) is the same statistic calculated over a 5% subset of the data that has not been used for refinement.

polyvinylidene fluoride membrane. The smallest α-subunit was sent to the Johns Hopkins Medical Institute Synthesis and Sequencing Facility for N-terminal sequencing. ThnT pantethine hydrolysis activity was monitored by incubation with 10 mM pantethine for 1 h followed by spotting on a TLC plate. Samples without inhibitor present turned purple upon addition of Ninhydrin reagent, consistent with hydrolysis of the pantethine moiety, whereas those with the CMK pantoamide did not stain.

### Molecular modeling of substrate docking and reactive intermediates

All steps were performed with the Discovery Studio 2.1 software suite. The carbapenem substrate and covalent intermediates were generated with the structure drawing functionality of Discovery Studio 2.1 and manually docked in the active site of ThnT. Generally, the system was typed with the CHARMM force field and minimized using an adopted basis NR algorithm. Crystallographic waters were explicitly included in all minimizations. Attempts to position the substrate or inhibitor in an orientation substantially different from the observed conformation (e.g., placing the chlorine atom inside the oxyanion hole) resulted in significant active-site distortion. All rotamers of the CMK C′–Cl bond were independently minimized before and after oxyanion formation in order to assess the capacity of ThnT to accommodate reactive intermediates. All hydrogen bonding interactions defined by the inhibitor-bound structure were constrained to ±0.2 Å of their measured values. The algorithm exited after achieving a root-mean-squared gradient of <0.1 kcal/(mol Å). All other settings were default.

---

### Acknowledgements

We gratefully acknowledge Vivian Stojanoff and Jean Jackoncic at the National Synchrotron Light Source for assistance in data collection. N-terminal sequencing was performed by Jodie Franklin at the Johns Hopkins Medical School Sequencing Facility. This work was supported by research grants National Institutes of Health RO1 AI014937 (to C.A.T.) and National Institutes of Health GM61017 (to J.F.S.) and the Dimitri V. d'Arbeloff post-doctoral fellowship (to N.T.W.).

### Supplementary Data

Supplementary data to this article can be found online at [http://dx.doi.org/10.1016/j.jmb.2012.06.012](http://dx.doi.org/10.1016/j.jmb.2012.06.012)

### References

1. Macao, B., Johansson, D. G. A., Hansson, G. C. & Hard, T. (2006). Autoproteolysis coupled to protein folding in the SEA domain of the membrane-bound MUC1 mucin. *Nat. Struct. Mol. Biol.* **13**, 71–76.
2. Lin, H. H., Chang, G. W., Davies, J. Q., Stacey, M., Harris, J. & Gordon, S. (2004). Autocatalytic cleavage of the EMR2 receptor occurs at a conserved G protein-coupled receptor proteolytic site motif. *J. Biol. Chem.* **279**, 31823–31832.
3. Rosenblum, J. S. & Blobel, G. (1999). Autoproteolysis in nucleoporin biogenesis. *Proc. Natl Acad. Sci. USA*, **96**, 11370–11375.
4. Tinel, A., Janssens, S., Lippens, S., Cuenin, S., Logette, E., Jaccard, B. *et al.* (2007). Autoproteolysis of PIDD marks the bifurcation between pro-death caspase-2 and pro-survival NF-κB pathway. *EMBO J.* **26**, 197–208.
5. Brannigan, J. A., Dodson, G., Duggleby, H. J., Moody, P. C. E., Smith, J. L., Tomchick, D. R. & Murzin, A. G. (1995). A protein catalytic framework with an N-terminal nucleophile is capable of self-activation. *Nature*, **378**, 416–419.
6. Perler, F. B., Xu, M. Q. & Paulus, H. (1997). Protein splicing and autoproteolysis mechanisms. *Curr. Opin. Chem. Biol.* **1**, 292–299.
7. Xu, Q. A., Buckley, D., Guan, C. D. & Guo, H. C. (1999). Structural insights into the mechanism of intramolecular proteolysis. *Cell*, **98**, 651–661.
8. Guan, C. D., Cui, T., Rao, V., Liao, W., Benner, J., Lin, C. L. & Comb, D. (1996). Activation of glycosylasparaginase. Formation of active N-terminal threonine by intramolecular autoproteolysis. *J. Biol. Chem.* **271**, 1732–1737.
9. Seemuller, E., Lupas, A., Stock, D., Lowe, J., Huber, R. & Baumeister, W. (1995). Proteasome from *Thermoplasma acidophilum*: a threonine protease. *Science*, **268**, 579–582.
10. Bompard-Gilles, C., Villeret, V., Davies, G. J., Fanuel, L., Joris, B., Frere, J. M. & Van Beeumen, J. (2000). A new variant of the Ntn hydrolase fold revealed by the crystal structure of L-aminopeptidase D-Ala-esterase/amidase from *Ochrobactrum anthropi*. *Structure*, **8**, 153–162.
11. Cheng, H. & Grishin, N. V. (2005). DOM-fold: a structure with crossing loops found in DmpA, ornithine acetyltransferase, and molybdenum cofactor-binding domain. *Protein Sci.* **14**, 1902–1910.
12. Freeman, M. F., McIshos, K. A., Bodner, M. J., Li, R. F. & Townsend, C. A. (2008). Four enzymes define the incorporation of coenzyme A in thiennycin biosynthesis. *Proc. Natl Acad. Sci. USA*, **105**, 11128–11133.
13. Buller, A. R., Freeman, M. F., Wright, N. T., Schildbach, J. F. & Townsend, C. A. (2012). Insights into cis-autoproteolysis reveal a reactive state formed through conformational rearrangement. *Proc. Natl Acad. Sci. USA*, **109**, 2308–2313.
14. Wang, Y. M. & Guo, H. C. (2010). Crystallographic snapshot of glycosylasparaginase precursor poised for autoprocesing. *J. Mol. Biol.* **403**, 120–130.
15. Wang, Y. M. & Guo, H. C. (2007). Crystallographic snapshot of a productive glycosylasparaginase–substrate complex. *J. Mol. Biol.* **366**, 82–92.
16. Ditzel, L., Huber, R., Mann, K., Heinemeyer, W., Wolf, D. H. & Groll, M. (1998). Conformational constraints for protein self-cleavage in the proteasome. *J. Mol. Biol.* **279**, 1187–1191.

17. Volpato, G., Rodrigues, R. C. & Fernandez-Lafuente, R. (2010). Use of enzymes in the production of semi-synthetic penicillins and cephalosporins: drawbacks and perspectives. *Curr. Med. Chem.* 17, 3855–3873.

18. Sio, C. F. & Quax, W. J. (2004). Improved β-lactam acylases and their use as industrial biocatalysts. *Curr. Opin. Biotechnol.* 15, 349–355.

19. Oinonen, C. & Rouvinen, J. (2000). Structural comparison of Ntn-hydrolases. *Protein Sci.* 9, 2329–2337.

20. Boanca, G., Sand, A., Okada, T., Suzuki, H., Kumagai, H., Fukuyama, K. & Barycki, J. J. (2007). Autoprocessing of *Helicobacter pylori* γ-glutamyltranspeptidase leads to the formation of a threonine-threonine catalytic dyad. *J. Biol. Chem.* 282, 534–541.

21. Dunitz, J. D. (1994). The entropic cost of bound water in crystals and biomolecules. *Science*, 264, 670.

22. Duggleby, H. J., Tolley, S. P., Hill, C. P., Dodson, E. J., Dodson, G. & Moody, P. C. E. (1995). Penicillin acylase has a single-amino-acid catalytic center. *Nature*, 373, 264–268.

23. Zhiryakova, D., Ivanov, I., Ilieva, S., Guncheva, M., Galunsky, B. & Stambolieva, N. (2009). Do N-terminal nucleophile hydrolases indeed have a single amino acid catalytic center? *FEBS J.* 276, 2589–2598.

24. Bond, C. S. (2003). TopDraw: a sketchpad for protein structure topology cartoons. *Bioinformatics*, 19, 311–312.

25. Jung, M. E. & Gervay, J. (1991). gem-Dialkyl effect in the intramolecular Diels–Alder reaction of 2-furfuryl methyl fumarates: the reactive rotamer effect, enthalpic basis for acceleration, and evidence for a polar transition state. *J. Am. Chem. Soc.* 113, 224–232.

26. Lqbal, A., Clifton, I. J., Bagonis, M., Kershaw, N. J., Domene, C., Claridge, T. D. W. *et al.* (2009). Anatomy of a simple acyl intermediate in enzyme catalysis: combined biophysical and modeling studies on ornithine acetyl transferase. *J. Am. Chem. Soc.* 131, 749–757.

27. Yin, J., Deng, Z., Zhao, G. & Huang, X. (2011). The N-terminal nucleophile serine of cephalosporin acylase executes the second autoproteolytic cleavage and acylpeptide hydrolysis. *J. Biol. Chem.* 286, 24476–24486.

28. Powers, J. C., Asgian, J. L., Ekici, O. D. & James, K. E. (2002). Irreversible inhibitors of serine, cysteine, and threonine proteases. *Chem. Rev.* 102, 4639–4750.

29. Yin, J., Cherney, M. M., Bergmann, E. M., Zhang, J., Huitema, C., Pettersson, H. *et al.* (2006). An episulfide (thiiranium ring) trapped in the active site of HAV 3C proteinase inactivated by peptide-based ketone inhibitors. *J. Mol. Biol.* 361, 673–686.

30. Savory, P. J., Djaballah, H., Anglier, H., Shaw, E. & Rivett, A. J. (1993). Reaction of proteasomes with peptidylchloromethanes and peptidyldiazomethanes. *Biochem. J.* 296, 601–605.

31. Prorok, M., Albeck, A., Foxman, B. M. & Abeles, R. H. (1994). Chloroketone hydrolysis by chymotrypsin and N-methylhistidyl-57-chymotrypsin: implications for the mechanism of chymotrypsin inactivation by chloroketones. *Biochemistry*, 33, 9784–9790.

32. Drenth, J., Kalk, K. H. & Swen, H. M. (1976). Binding of chloromethyl ketone substrate analogs to crystalline papain. *Biochemistry*, 15, 3731–3738.

33. Anand, K., Ziebuhr, J., Wadhwa, P., Mesters, J. R. & Hilgenfeld, R. (2003). Coronavirus main proteinase (3CL<sup>pro</sup>) structure: basis for design of anti-SARS drugs. *Science*, 300, 1763–1767.

34. Yin, J. A., Niu, C. G., Cherney, M. M., Zhang, J. M., Huitema, C., Eltis, L. D. *et al.* (2007). A mechanistic view of enzyme inhibition and peptide hydrolysis in the active site of the SARS-CoV 3C-like peptidase. *J. Mol. Biol.* 371, 1060–1074.

35. Bordwell, F. G. & Brannen, W. T. (1964). The effect of the carbonyl and related groups on the reactivity of halides in S<sub>N</sub>2 reactions. *J. Am. Chem. Soc.* 86, 4645.

36. Bodner, M. J., Phelan, R. M., Freeman, M. F., Li, R. & Townsend, C. A. (2010). Non-heme iron oxygenases generate natural structural diversity in carbapenem antibiotics. *J. Am. Chem. Soc.* 132, 12-+.

37. Bodner, M. J., Li, R. F., Phelan, R. M., Freeman, M. F., Moshos, K. A., Lloyd, E. P. & Townsend, C. A. (2011). Definition of the common and divergent steps in carbapenem β-lactam antibiotic biosynthesis. *ChemBioChem*, 12, 2159–2165.

38. Fanuel, L., Goffin, C., Cheggour, A., Devreese, B., Van Driessche, G., Joris, B. *et al.* (1999). The DmpA aminopeptidase from *Ochrobactrum anthropi* LMG7991 is the prototype of a new terminal nucleophile hydrolase family. *Biochem. J.* 341, 147–155.

39. Syrén, P. & Hult, K. (2011). Amidases have a hydrogen bond that facilitates nitrogen inversion, but esterases have not. *ChemCatChem*, 3, 853–860.

40. Bokhove, M., Yoshida, H., Hensgens, C. M. H., van der Laan, J. M., Sutherland, J. D. & Dijkstra, B. W. (2010). Structures of an isopenicillin N converting Ntn-hydrolase reveal different catalytic roles for the active site residues of precursor and mature enzyme. *Structure*, 18, 301–308.

41. Kim, Y., Kim, S., Earnest, T. N. & Hol, W. G. J. (2002). Precursor structure of cephalosporin acylase: insights into autoproteolytic activation in a new N-terminal hydrolase family. *J. Biol. Chem.* 277, 2823–2829.

42. McCoy, A. J., Grosse-Kunstleve, R. W., Adams, P. D., Winn, M. D., Storoni, L. C. & Read, R. J. (2007). Phaser crystallographic software. *J. Appl. Crystallogr.* 40, 658–674.

43. Bailey, S. (1994). The CCP4 suite: programs for protein crystallography. *Acta Crystallogr.*, Sect. D: *Biol. Crystallogr.* 50, 760–763.

44. Emsley, P. & Cowtan, K. (2004). Coot: model-building tools for molecular graphics. *Acta Crystallogr.*, Sect. D: *Biol. Crystallogr.* 60, 2126–2132.

45. Winn, M. D., Murshudov, G. N. & Papiz, M. Z. (2003). Macromolecular TLS refinement in REFMAC at moderate resolutions. *Methods Enzymol.* 374, 300–321.

46. Painter, J. & Merritt, E. A. (2006). TLSMD web server for the generation of multi-group TLS models. *J. Appl. Crystallogr.* 39, 109–111.

47. Brooks, B. R., Bruccoleri, R. E., Olafson, B. D., States, D. J., Swaminathan, S. & Karplus, M. (1983). CHARMM: a program for macromolecular energy, minimization, and dynamics calculations. *J. Comput. Chem.* 4, 187–217.
