
Critical Reviews in Microbiology, 2012; Early Online: 1–16
© 2012 Informa Healthcare USA, Inc.
ISSN 1040-841X print/ISSN 1549-7828 online
DOI: 10.3109/1040841X.2012.699025

REVIEW ARTICLE

# Bacterial resistance to cationic antimicrobial peptides

José Luis Anaya-López¹, Joel Edmundo López-Meza², and Alejandra Ochoa-Zarzosa²

¹Instituto Nacional de Investigaciones Forestales, Agrícolas y Pecuarias, Unidad de Biotecnología, Celaya, México and  
²Universidad Michoacana de San Nicolás de Hidalgo, Centro Multidisciplinario de Estudios en Biotecnología-FMVZ,  
Morelia, México

---

### Abstract

Naturally occurring cationic antimicrobial peptides (CAMPs) have been considered as promising candidates to treat infections caused by pathogenic bacteria to animals and humans. This assumption is based on their mechanism of action, which is mainly performed through electrostatic membrane interactions. Unfortunately, the rise in the reports that describe bacterial resistance to CAMPs has redefined their role as therapeutic agents. In this review, we describe the state of the art of the most common resistance mechanisms developed by bacteria to CAMPs, making special emphasis on resistance selection. Considering most of the resistance mechanisms here reviewed, the emergence of resistance is unlikely in the short term, however we also described evidences that show the evolution of resistance to CAMPs, reevaluating their use as good antibacterial agents. Finally, the knowledge related to the description of CAMP resistance mechanisms may provide useful information for improving strategies to control infections.

#### Keywords: antibacterial peptides, bacterial resistance, membrane modification

---

### Introduction

The evolution of pathogenic bacteria has allowed many microbes develop resistance mechanisms against conventional antibiotics, which has led to the search for new therapeutic alternatives; of these new therapeutics, antimicrobial peptides (AMPs) are among the most promising options. AMPs are produced by a wide variety of organisms and have a broad and nonspecific activity that makes them strong candidates for a variety of pharmaceutical applications. However, some human pathogens have developed intrinsic or inducible resistance mechanisms against AMPs, which is now being appreciated as an important virulence phenotype. Thus, understanding the molecular mechanisms of bacterial AMP resistance may provide new targets for antibiotic drug design and antimicrobial therapy of human infectious diseases.

There are many different types of AMPs with diverse structures and activities, but the cationic antimicrobial peptides (CAMPs) are the best characterized. CAMPs are essential components of defense against microbes in animals, plants and even in bacteria. Although the length and amino acid composition of CAMPs may differ, most CAMPs have common structural features and operate with similar modes of action. The general mechanism by which CAMPs damage the plasma membrane is based on electrostatic interactions and is considered universal for all described peptides. The distinct distribution of negatively charged phospholipids in the membranes of eukaryotes and prokaryotes seem to be the key to the selectivity of CAMPs. In bacteria, the interaction of CAMPs with anionic membrane phospholipids or the phosphate groups of Gram-negative lipopolysaccharide (LPS), as well as the interaction with teichoic acids in Gram-positive bacteria, occurs through electrostatic mechanisms.

Although bacterial susceptibility to CAMPs is diverse, some bacteria have evolved mechanisms of resistance, which include decreased affinity to CAMPs by substitution of anionic cell surface constituents with cationic molecules; biosynthesis and crosslinking of cell envelope components; external trapping mechanisms that bind or neutralize the CAMPs by direct secretion

---

### Address for Correspondence: Alejandra Ochoa-Zarzosa, Universidad Michoacana de San Nicolás de Hidalgo, Centro Multidisciplinario de Estudios en Biotecnología-FMVZ, La Posta Km 9.5 Carretera, Morelia-Zinapécuaro, 58893 México. E-mail: ochoaz@umich.mx

(Received 29 January 2012; revised 27 May 2012; accepted 29 May 2012)

of proteins, or the release of CAMPs binding molecules from the host cell surface; membrane efflux pumps; and production of peptidases. However, all of the CAMP resistance mechanisms described to date have limitations, and no bacteria have ever succeeded in becoming fully insensitive to a broad range of CAMPs. In this review, we discuss the state of the art of the mechanisms of bacterial resistance to CAMPs, with special emphasis on selection for resistance.

### General aspects

The excessive and inappropriate use of conventional antibiotics in the clinical treatment of human and animal infections has increased pathogen resistance against these compounds, turning them into less effective agents. As a consequence, there has been an increase in the generation of multidrug-resistant pathogens, primarily bacteria and fungi that resist the effects of most antibiotics (Heuer et al., 2006; Field et al., 2010). As a result, the pharmaceutical industry is attempting to solve this problem by looking for new compounds; however, because the pathogens have the ability to develop resistance mechanisms against these types of compounds, this strategy is not effective.

Among the efforts being made in the research and development of new biomedical alternatives, CAMPs are considered one of the most promising candidates. CAMPs are produced by a wide variety of organisms as part of their first line of defense (eukaryotes) or as a competition strategy used to acquire nutrients and space (prokaryotes). These molecules are usually short peptides (12–100 amino acid residues) and generally have a positive charge (+2 to +9), although there are also AMP neutral and negatively charged. They are amphipathic and have been isolated from bacteria, plants and animals, including humans. Because CAMPs originate in a variety of living systems, they exhibit a diversity of structures and mechanisms of action (Dawson and Liu, 2008). CAMPs fall into two broad classes, accordingly to the mechanism of synthesis. The first class includes peptides that are synthesized on large, modular enzyme complexes by microorganisms. They often incorporate unusual amino acids and may be modified by glycosylation, by ring formation, or both (Hancock and Chapple, 1999). This group includes many conventional antibiotics such as gramicidins, bacitracins, polymyxins, streptogramins and their derivatives. The second category is comprised of linear peptides, consisting almost entirely of conventional amino acid residues, which are produced by all major types of organisms. These peptides are translated using the usual ribosomal protein synthesis pathway, and this category is referred to as the Ribosomally synthesized AntiMicrobial Peptides (RAMPs) to distinguish them from the non-RAMPs of the first category (Hancock and Chapple, 1999). Some examples of RAMPs include the family of defensins in plants and animals; indolcinds and cathelicidins in mammals; bombinin, magainin

and buforin in frogs; cecropin and melittin in insects; the thionins in plants; and the bacteriocins and nisin in bacteria (López-Meza et al., 2011). The toxicity of CAMPs toward bacteria is primarily due to an initial electrostatic interaction between the peptide and the anionic phospholipid headgroups in the outer layer of the bacterial cytoplasmic membrane (Zasloff, 2002). Accordingly, several peptide-membrane interaction models have been proposed (López-Meza et al., 2011).

Several research groups that study bacteria have identified the molecular mechanisms of resistance to CAMPs. In this review, we will discuss some general aspects of these mechanisms. We will specifically emphasize the acquisition of resistance to CAMPs classified as RAMPs. However, many of the resistance mechanisms that we will describe have also been employed by non-RAMPs considered as conventional antibiotics, such as polymyxin B. Thus, we will include the description of resistance mechanisms to this kind of molecules. Accordingly, polymyxin B is a cyclic antimicrobial lipopeptide produced by the soil bacterium *Paenibacillus polymyxa*. It is classified as a RAMP and has been clinically useful as an antibiotic. Polymyxin B has a cationic net charge (Vaara et al., 2008), its permeabilizing action has been well established (Vaara, 1992), and it shares attributes with CAMPs. Moreover, numerous studies indicate that resistance to this antibiotic reflects the resistance to other mammalian peptides, suggesting that polymyxin may be used as an inexpensive marker to evaluate CAMP sensitivity (Lee et al., 2004; Tzeng et al., 2005; Cullen and Trent, 2010; Pérez-Gutiérrez et al., 2010; Keo et al., 2011). For these reasons, polymyxin B has long been used as a model compound to define the mechanisms by which CAMPs kill bacteria. In relation to the development of microorganisms that are resistant to RAMPs, most of the work has focused on the effects of human peptides, α and β-defensins or cathelicidin (LL-37), or to the synthetic or porcine protegrins.

#### Resistance to CAMPs by cell surface modification

In bacteria, the principal site of CAMP action is the cytoplasmic membrane. To reach this target, CAMPs must traverse enveloping structures. Gram-positive bacteria lack an outer membrane but possess a thick cell wall composed of heavily crosslinked polymers of teichoic or lipoteichoic acids and peptidoglycan. Lipoteichoic acid (LTA) is noncovalently inserted into the cellular membrane, and a wall teichoic acid (WTA) is covalently linked to the peptidoglycan of the cell wall (Neuhaus and Baddiley, 2003).

Gram-negative bacteria do not produce teichoic acids but have a multilayered surface structure, including a peptidoglycan matrix in the periplasmic space beneath an outer membrane. The outer layer of the outer membrane serves as the site of attachment for lipid A, an anionic dimer of glucosamine linked to fatty acid chains

and flanked by polar phosphate groups. Lipid A is covalently bonded to a core polysaccharide and a specific O-chain consisting of repeating oligosaccharide units. This complex is referred to as lipopolysaccharide (LPS), unless it lacks the O-chain, in which case it is named lipo-oligosaccharide (LOS) (Rietschel et al., 1994). LPS and LOS are important virulence factors in many bacterial infections, conferring serum resistance, antibiotic resistance and endotoxicity to the pathogen (Preston et al., 1996). In addition, the proteins anchored to the cell wall or membrane add more complexity to bacterial surfaces (Nizet, 2006). Among the adaptations of pathogenic bacteria, resistance to CAMPs highlights the modification of the components of cell walls and cytoplasmic membranes, including alteration of the net charge and structure. The net charge of the cellular membrane is based mainly on its phospholipid stoichiometry. Mammalian cell membranes are enriched in neutrally-charged lipids, such as phosphatidylcholine (PC), phosphatidylethanolamine (PE) and sphingomyelin (SM), and their cytoplasmic membranes are generally neutral in net charge. Neutral sterols, including cholesterol and ergosterol, are also found in eukaryotic membranes. In contrast, cellular membranes of many bacterial pathogens are composed of hydroxylated phospholipids, such as phosphatidylglycerol (PG), cardiolipin (CL), a dimer of PG also known as diphosphatidylglycerol, and phosphatidylserine (PS); which tend to be highly electronegative. These membrane cell composition differences are the key to the selective activity of CAMPs toward bacteria (Yeaman and Yount, 2003).

CAMPs resistance by alteration of cell surface net charge

The selective toxicity of CAMPs toward bacteria is primarily due to an initial electrostatic interaction between the CAMP and the anionic phospholipid headgroups in the outer layer of the bacterial membrane (Zasloff, 2002; Yeaman and Yount, 2003) or with components on the cell wall. It was believed that bacterial resistance induction to CAMPs was not probable because their interactions with bacterial components did not involve specific protein binding sites and that the antimicrobial activity involved electrostatic interactions with the anionic membrane components. However, Gram-positive and Gram-negative bacteria have developed mechanisms to elude CAMP activity, which essentially involves the modification of anionic cell surface constituents with cationic molecules. This reduces the anionic charge, which leads to the repulsion of the positively charged CAMPs, preventing them from reaching the cytoplasmic membrane and disrupting its integrity (Kristian et al., 2003; Tamayo et al., 2005; Tran et al., 2006; Maloney et al., 2009). In Gram-positive bacteria, the first site of contact is the cell wall, and the changes in its structure represent the first line of defense to neutralize the activity of CAMPs. Among the best-characterized mechanisms is the D-alanylation of teichoic acids, as well as the modification of phospholipids with L-lysine or L-alanine or the modification of lipid A with phosphoethanolamine (pEtN) or 4-amino-4-deoxy-L-arabinose (L-Ara4N). All of these mechanisms are described in the next sections of this review.

**Modification of teichoic acids with D-alanine**

Teichoic acids (TAs) are polymers with a relatively wide structural diversity. WTA is covalently linked to the peptidoglycan, whereas LTA is a macroamphiphile with its glycolipid anchored in the membrane and its poly (glycerophosphate) (Gro-P) chain extending into the wall (Neuhaus and Baddiley, 2003). As major constituents of the surfaces of Gram-positive bacteria, TAs have an impact on important biological processes, such as stimulation of immune response (Morath et al., 2001; Granette et al., 2005), resistance to antimicrobial agents (Poyart et al., 2003; Fabretti et al., 2006; Kovács et al., 2006), virulence, adhesion, biofilm formation and others (Neuhaus and Baddiley, 2003). Incorporation of D-alanine in LTAs reduces the negative charge of the cell envelope (Figure 1), and this process is accomplished by a two-step reaction that requires four proteins encoded by the *dlt* operon. The *dlt* operon has been characterized in 12 species and contains four genes: *dltABCD*. The 56-kDa D-alanine:D-alanyl carrier protein ligase (Dcl) is encoded by the *dltA* gene, while the 8.8-kDa D-alanyl carrier protein (Dcp)

![Figure 1](https://www.informahealthcare.com/mby)

Figure 1. Addition of D-alanine to lipoteichoic acid. The LTA type I are integrated by a poly glycerophosphate (Gro-P) anchored to the glycolipid. In this representation the structure of glycolipid was omitted. The addition of D-alanine (red) to the Gro-P portion of LTA (black) increases its positive charge and contributes to a reduction in the net negative charge. (See colour version of this figure online at www.informahealthcare.com/mby).

© 2012 Informa Healthcare USA, Inc.

Table 1. Antimicrobial peptide resistance mechanisms by cell surface alterations of human bacterial pathogens.

| CAMP resistance phenotype                                                                 | Gene(s)                  | Resistance to                                                                                          | Bacteria*                              | Reference          |
|-----------------------------------------------------------------------------------------|--------------------------|-------------------------------------------------------------------------------------------------------|----------------------------------------|--------------------|
| Addition of L-lysine or L-alanine to phosphatidylglycerol (L-PG; A-PG) in cell membrane      | mprF                     | Arenicin-1, CAP 18, gallidermin, HBD-3, HNP 1-3, LL-37 lugworm beta-sheet peptide, lysozyme, magainin II, melittin, nisin, NK-2, polymyxin B, protamine, protegrin 3 and 5, tachyplesin 1, vancomycin | Staphylococcus aureus                 | Peschel et al., 2001 |
|                                                                                        | lysC                     |                                                                                                       | Bacillus anthracis                    | Kristian et al., 2003 |
|                                                                                        | lysX                     |                                                                                                       | Listeria monocytogenes                | Nishi et al., 2004   |
|                                                                                        | PA0920                   |                                                                                                       | Mycobacterium tuberculosis            | Ernst et al., 2009   |
|                                                                                        |                          |                                                                                                       | Pseudomonas aeruginosa               | Andra et al., 2011   |
|                                                                                        |                          |                                                                                                       |                                        | Samant et al., 2009  |
|                                                                                        |                          |                                                                                                       |                                        | Thedieck et al., 2006 |
|                                                                                        |                          |                                                                                                       |                                        | Maloney et al., 2009 |
|                                                                                        |                          |                                                                                                       |                                        | Klein et al., 2009  |
| D-alanylation of lipoteichoic acid (LTA) and teichoic acid in bacterial cell wall (WTA)     | dlt operon               | Cecropin B, colistin, gallidermin, HNP 1-3, indolicidin, CRAMP, magainin II, nisin, polymyxin B, protegrin 1, 3 and 5, tachyplesin 1 and 3 | Staphylococcus aureus                 | Peschel et al., 1999 |
|                                                                                        | dltA                     |                                                                                                       | Listeria monocytogenes                | Abachin et al., 2002 |
|                                                                                        |                          |                                                                                                       | Group B Streptococcus                 | Poyart et al., 2003  |
|                                                                                        |                          |                                                                                                       | Group A Streptococcus                 | Kristian et al., 2005 |
|                                                                                        |                          |                                                                                                       | Streptococcus pneumoniae             | Kovács et al., 2006  |
|                                                                                        |                          |                                                                                                       | Enterococcus faecalis                | Fabretti et al., 2006 |
| Synthesis and extension of lipo oligosaccharide (LOS)                                      | lpxA                     | Crp 4, Fowl-1, HD-5, LL-37, polymyxin B                                                               | Neisseria meningitidis                | Jones et al., 2009   |
|                                                                                        | waaF, lgtF, galT, cstII   |                                                                                                       | Campylobacter jejuni                  | Naito et al., 2010   |
|                                                                                        |                          |                                                                                                       |                                        | Keo et al., 2011    |
| Addition of pEtN to lipid A                                                             | lpxE<sub>HP</sub>        | LL-37, protegrin 1, polymyxin B                                                                      | Helicobacter pylori                   | Tran et al., 2006   |
|                                                                                        | cj0256                   |                                                                                                       | Campylobacter jejuni                  | Cullen and Trent, 2010 |
|                                                                                        | pmrC                     |                                                                                                       | Salmonella enterica                  | Tamayo et al., 2005  |
|                                                                                        | lptA                     |                                                                                                       | Neisseria gonorrhoeae                | Lewis et al., 2009   |
|                                                                                        |                          |                                                                                                       | Neisseria meningitidis               | Tzeng et al., 2005   |
| Addition of aminoarabinose to lipid A                                                   | pmr genes                | C18G, HBD-2, polymyxin B, protegrin 1, synthetic protegrin analogs                                     | Salmonella enterica                  | Gunn et al., 2000   |
|                                                                                        |                          |                                                                                                       | Proteus mirabilis                    | McCoy et al., 2001   |
|                                                                                        |                          |                                                                                                       | Pseudomonas aeruginosa               | Moskowitz et al., 2004 |
|                                                                                        |                          |                                                                                                       | Klebsiella pneumoniae                | Cheng et al., 2010   |
| Acylation of lipid A                                                                     | pagP                     | C18G, colistin, CP 28, HBD-2, LL-37, magainin II, CRAMP, protegrin 1, PGLa, polymyxin B and E                                   | Salmonella spp.                      | Guo et al., 1998    |
|                                                                                        | rcp                      |                                                                                                       | Legionella pneumophila               | Robey et al., 2001   |
|                                                                                        | htrB                     |                                                                                                       | Haemophilus influenzae               | Starner et al., 2002 |
|                                                                                        | msbB                     |                                                                                                       | Vibrio cholerae                       | Matson et al., 2010  |
|                                                                                        | lpxM                     |                                                                                                       | Klebsiella pneumoniae                | Clements et al., 2007 |
| Phosphorylcholine in LPS                                                                | licD                     | LL-37                                                                                                 | Haemophilus influenzae               | Lysenko et al., 2000 |
| Synthesis of polysaccharide capsule                                                     | cps                      | HBD-1 and 3, HNP-1 and 2, lactoferrin, polymyxin B, protamine, CRAMP, CRAMP-18, HNP-1 and 2, LL-37, protegrin 1, polymyxin B, β-defensin-1, 2 and 3 | Klebsiella pneumoniae                | Campos et al., 2004  |
|                                                                                        | siaD                     |                                                                                                       | Neisseria meningitidis               | Spinosa et al., 2007 |
|                                                                                        | sia operon               |                                                                                                       | Staphylococcus epidermidis           | Jones et al., 2009   |
|                                                                                        | ica genes                |                                                                                                       |                                        | Vuong et al., 2004a,b |
|                                                                                        | cap                      |                                                                                                       |                                        | Kocianova et al., 2005 |

*Not all organisms are resistant to the CAMP indicated; for specific resistance, please consult the reference.

C18G, α-helical peptide derived from the carboxy terminus of platelet factor IV; CAP 18, cationic LPS-binding protein 18 from rabbit; CP 28, α-helical synthetic cationic peptide based on the cecropin-mellitin hybrid peptide CEME; CRAMP and CRAMP-18, murine cathelicidin-related peptides; Crp 4, murine homologous to human α-defensin-5; Fowl-1, heterophil-derived cathelicidin homolog fowlicidin-1; HBD-1, human β-defensin 1, HBD-2, human β-defensin 2; HBD-3, human β-defensin-3; HD-5, human α-defensin-5; HNP-1, Human neutrophil defensin 1; HNP-2, Human neutrophil defensin 2; HNP-3, Human neutrophil defensin-3; LL-37, human cathelicidin, C-terminal part of the human cationic antimicrobial protein (hCAP-18); NK-2, α-helical fragment of mammalian NK-lysin; PGLa, peptide starting with a glycine and ending with a leucine amide from magainin peptide family.

mutant of *S. aureus* and *dltA, dltB,* and *dltD* mutants of *S.*  
*xylosus* did not produce D-alanine esters in their teichoic  
acids, and these conditions increased their sensitivity to  
CAMPs (Table 1). The increased sensitivity of *dlt*-mutants  
seemed to be restricted to cationic peptides, because no  
considerable differences were observed in the inhibitory  
concentrations of the neutral peptide gramicidin D from  
*Bacillus brevis* (Peschel et al., 1999; Poyart et al., 2003).  
Furthermore, the *dltA* mutant of both *S. aureus* and *S.*  
*xylosus*, revealed normal sensitivities to gallidermin and  
nisin after complementation with the *dltABCD* operon of  
*S. xylosus*, and wild-type strains bearing additional copies  
of the *dltABCD* operon increased their tolerance against  
other cationic peptides tested (Peschel et al., 1999).

The partial or total reconstitution of CAMP resistance  
by complementation of the *dlt* mutation (Poyart et al.,  
2003; Fabretti et al., 2006; Kovács et al., 2006), and the  
relationship between the decrease in LTAs and WTA with  
the increase in susceptibility of *dlt*-mutants to CAMPs  
(Peschel et al., 1999; Kovács et al., 2006), suggested a  
direct correlation between the tolerance to CAMPs and  
the D-alanine content of the TAs.

All Gram-positive *dlt*-mutants, such as *Listeria*  
*monocytogenes* (Abachin et al., 2002), *Streptococcus*  
*agalactiae* (Poyart et al., 2003), *Streptococcus pyogenes*  
(Kristian et al., 2005), *Streptococcus pneumoniae* (Kovács  
et al., 2006), and *Enterococcus faecalis* (Fabretti et al.,  
2006) have increased their susceptibility to CAMPs  
(Table 1). In all cases, the susceptibility of *dlt*-mutants  
seems to be selective to CAMPs, because the susceptibility  
to antibiotics, such as gentamicin or penicillin, was  
not affected (Kristian et al., 2005). In *L. monocytogenes*  
the minimum inhibitory concentration (MIC) of penicillin, vancomycin, teicoplanin and bacitracin was identical  
for wild-type strains and the *dltA* mutants, whereas MICs  
of CAMPs colistin, polymyxin B and nisin were significantly  
higher in wild-type strains (Abachin et al., 2002).

However, the function of *dlt* operon seems to be  
extended far beyond CAMP resistance, due to the fact that  
the D-alanylation of the LTAs contributes to the virulence of  
pathogens, such as *L. monocytogenes* (Abachin et al., 2002)  
and Group A Streptococcus (GAS) (Kristian et al., 2005).

In a mouse model, the *L. monocytogenes dltA* mutant  
was eliminated more rapidly from the blood, liver and  

---

Resistance to cationic antimicrobial peptides 5  
spleen than with the control; and during *in vitro* assays,  
the adherence of the mutant to murine bone marrow-  
derived macrophages (BMMs) to hepatocytes (TIB73)  
and to the human epithelial cell line (Caco-2) was strongly  
restricted with respect to the wild-type strain. However,  
in the *dltA* mutant the amounts of surface proteins  
ActA, InlA and InlB, implicated in virulence, remained  
unaffected, suggesting that the decreased adherence of  
the *dltA* mutant to nonphagocytic and phagocytic cells  
might be the result of an increase in the electronegativity  
of the surface and/or the presence of adhesins possessing  
altered binding activities (Abachin et al., 2002).

The association of TA D-alanylation with GAS resistance to CAMPs and others factors acts to promote the  
organism's survival within neutrophils, a phenotype  
recently described as contributing to GAS virulence  
(Kristian et al., 2005). Therefore, the protective effect of  
D-alanylation of TAs against CAMPs and virulence is a  
consequence of partially neutralizing the negative charge  
of the anionic cell wall polymer by incorporation of  
positively charged residues. However, other cell surface  
modifications, such as lysinylation of anionic membrane  
lipids may also contribute to the resistance to CAMPs by  
alteration of cell surface net charge.

---

**Modification of phospholipids with L-lysine or L-alanine**

One of the strategies that bacteria use to evade the activity of CAMPs is to diminish the negative net charge of the  
membrane through the addition of L-lysine or L-alanine  
to PG (Figure 2).

The addition of L-lysine predominates in Gram-  
positive bacteria and is mediated by a multiple peptide  
resistance factor (MprF) encoded by the *mprF* gene  
(Peschel et al., 2001; Staubitz et al., 2004), while the alanylation of PG is recurrent in Gram-negative bacteria  
(*Pseudomonas aeruginosa*), which is mediated by the *mprF* orthologous gene (Klein et al., 2009). MprF proteins are integral membrane proteins with two separable  
functional domains: a well-conserved hydrophilic cytoplasmic domain at the C-terminus that synthesizes lysyl-  
phosphatidylglycerol (Lys-PG), an unusual derivative  
of PG modified with L-lysine; and a large hydrophobic  
domain at the N-terminus that facilitates the flipping of  
Lys-PG (Ernst et al., 2009). The lysyl group is derived from

![Figure 2](https://www.informahealthcare.com/mby)

Figure 2. Modification of the membrane net charge by the addition of L-lysine and L-alanine to phosphatidylglycerol. Dashed bonds indicate modifications of phosphatidylglycerol. The lysyl-phosphatidylglycerol synthase (L-PGS) and alanyl phosphatidylglycerol synthase (A-PGS), respectively, regulate the addition of L-lysine (R1, red) and L-alanyl (R2, blue) to phosphatidylglycerol (black). (See colour version of this figure online at www.informahealthcare.com/mby).

© 2012 Informa Healthcare USA, Inc.

aminoacyl tRNAs and these groups are usually transferred to PG (Roy, 2009). Thus, MprF exerts two specific functions that have scarcely been found in other bacterial proteins.

The incorporation of lysyl to PG confers resistance to CAMPS in S. aureus, and both domains of MprF are necessary for the resistance phenotype. The C-terminal domain and six of the fourteen proposed transmembrane segments of MprF are sufficient for the full synthesis of Lys-PG, but they do not lead to efficient CAMPS resistance, since most of the Lys-PG remains in the inner layer of the cytoplasmic membrane. However, when the N-terminal domain is coexpressed with the Lys-PG synthase domain, full CAMP resistance is achieved, and the Lys-PG is translocated to the outer layer of the membrane (Ernst et al., 2009).

Most organisms only encode a single *mprF* gene. However, the Gram-positive bacteria, *Clostridium perfringens* SM101, encode two *mprF* homologous genes (*mprF1* and *mprF2*). Whereas *mprF2* encodes a lysyl-phosphatidylglycerol synthase (L-PGS), *mprF1* encodes an alanyl phosphatidylglycerol synthase (A-PGS). The formation of alanyl phosphatidylglycerol (A-PG) and L-PG was shown to be tRNA-dependent using Ala-tRNA<sup>Ala</sup> and Lys-tRNA<sup>Lys</sup> as substrates, respectively (Staubitz et al., 2004; Roy and Ibba, 2008). The summarized mode of action of MprF is presented in Figure 3.

A similar mechanism has been elucidated in *Mycobacterium tuberculosis*. In these bacteria, the *lysX* gene is responsible for Lys-PG production and for encoding the two domains: protein lysyl-transferase and (mprF)-lysyl-tRNA synthetase (lysU). A *M. tuberculosis* mutant strain with a mutation that knocks out *lysX* is sensitive to cationic antibiotics and CAMPS. This mutant also shows an increased association with lysosome-associated membrane protein–positive vesicles and exhibits altered membrane potential compared to the wild-type strain.

The complementation of the *lysX* gene but not *mprF* restored the production of Lys-PG (Maloney et al., 2009). *mprF* was discovered in *S. aureus* by screening transposon insertion libraries in mutants sensitive to gallidermin. Disruption of *mprF* in *S. aureus* leads to the loss of synthesis of Lys-PG and enhances sensitivity to a wide range of bacterial and mammalian CAMPS (Table 1). The presence of Lys-PG in the membrane can provide up to a 30-fold increase in the MIC of certain CAMPS, although this number is variable for individual compounds. In addition, resistance towards CAMPS, mediated by lysinylation of PG, has also been found in *Bacillus anthracis* (Samant et al., 2009), *L. monocytogenes* (Thedieck et al., 2006), and *P. aeruginosa* (Klein et al., 2009). In fact, sequence comparisons with known PG synthase sequences allowed the identification of *mprF* homologues in a wide variety of microorganisms: 347 sequences of *mprF* homologues were distributed among 31 genera of Gram-positive bacteria (mostly firmicutes and actinobacteria), 59 genera of Gram-negative bacteria (mostly proteobacteria), and in three species of archaea in the genus *Methanosarcina* (Roy, 2009).

### Modification of lipid A with phosphoethanolamine and aminoarabinose

The lipid A moiety of LPS forms the outer monolayer of the outer membrane of most Gram-negative bacteria, and is the first site of interaction with CAMPS. Lipid A is synthesized on the cytoplasmic surface of the inner membrane by a conserved enzymatic pathway. Following attachment of the core oligosaccharide, lipid A is flipped to the outer surface of the inner membrane, where the O-antigen polymer is attached (Raetz et al., 2007). Modification of the phosphate groups of lipid A with amine substituents, such as pEtN or L-Ara4N, reduces the overall net negative charge of the Gram-negative LPS, thereby reducing its affinity for CAMPS. The attachment of pEtN to the disaccharide backbone of lipid A can be done directly to the C-1 position, to the phosphate group attached to C-1 and C-4′ positions, or both, depending on the bacterial species (Figure 4; Tran et al., 2006; Lewis et al., 2009; Cullen and Trent, 2010). The direct attachment of pEtN at the C-1 position is a two-step process that involves the removal of the 1-phosphate group by a lipid A 1-phosphatase (LpxE<sub>HP</sub>), followed by the addition of a pEtN residue catalyzed by phosphoethanolamine transferase (EptA<sub>HP</sub>). Insertional inactivation of *lpxE<sub>HP</sub>* in *H. pylori* results in the replacement of the pEtN residue by a phosphate group at the C-1 position of lipid A, increasing the overall negative charge of the molecule and reducing the polymyxin B MIC (>250 μg/mL, for wild-type strain, to 10 μg/mL for the *lpxE<sub>HP</sub>* mutant) (Tran et al., 2006).

The addition of pEtN to the phosphate group at the C-4′ position of lipid A in *N. gonorrhoeae* is regulated by the *lptA* gene. Insertional inactivation of the *lptA* gene significantly increased (64-fold) gonococcal susceptibility to polymyxin B, and reduced survival by 90% in natural human serum (Lewis et al., 2009). Both serum and

![Figure 3](attachment://figure_3.png)

**Figure 3.** Mode of action of *S. aureus* MprF. The synthase domain of MprF (orange) synthesizes Lys-PG from Lys-tRNA and PG, and the flippase domain of MprF (blue) translocates Lys-PG to the outer cytoplasmic membrane layer. CAMP resistance is achieved by charge repulsion only when both domains are active (Ernst et al., 2009). (See colour version of this figure online at www.informahealthcare.com/mby).

Resistance to cationic antimicrobial peptides 7

**Helicobacter pylori**

Lipid A  
LpxE<sub>HP</sub>  
Pi  

Dephosphorylated  
lipid A at C-1  

PtdEtN  
EptA<sub>HP</sub>  
DAG  

Phosphatidylethanolamine (PtdEtN)  
Campylobacter jejuni  

Cj0256  

Lipid A  

Figure 4. Modification of lipid A with phosphoethanolamine and aminoarabinose. Dashed bonds indicate modifications of lipid A. The lengths of the acyl chains are indicated. The negative net charge of lipid A (black) can be neutralized by addition of pEtN (red) to phosphate group at the C-1 and C-4′ positions of the glucosamine disaccharide in *Campylobacter jejuni* (blue numbers), this modification is catalyzed by the phosphoethanolamine transferase Cj0256 protein. The addition of pEtN in *H. pylori* is a two-step enzymatic process that involves the removal of the phosphate group at the C-1 position by the lipid A 1-phosphatase, LpxE<sub>HP</sub>, followed by the addition of a pEtN unit directly to the glucosamine disaccharide backbone by the phosphoethanolamine transferase, EptA<sub>HP</sub>. The addition of 4-amino-4-deoxy-L-arabinose (L-Ara4N; blue) to the phosphate group at C-4′ is regulated by the *pmr* operon (data adapted from McCoy et al., 2001; Moskowitz et al., 2004; Tran et al., 2006; Lewis et al., 2009; Cullen and Trent, 2010). (See colour version of this figure online at www.informahealthcare.com/mby).

polymyxin B resistance, as well as pEtN decoration of lipid A, were restored in the *lptA*-null mutant by complementation with wild-type *lptA*, while the inactivation of the same gene in *N. meningitidis* increased susceptibility to CAMPS LL-37, protegrin-1 (PG-1) and polymyxin B (Tzeng et al., 2005). Unexpectedly, the addition of pEtN to HepII of the lipooligosaccharide β-chain at positions 3 and 6 (LPS core), modulated by *lpt3* and *lpt6* genes, respectively, did not affect the susceptibility of *N. gonorrhoeae* to CAMPS (Lewis et al., 2009); suggesting that other factors different from surface charge and electrostatic interaction contribute to resistance to CAMPS.

There is some evidence suggesting that the addition of pETN to lipid A is not enough to confer resistance to CAMPS; the *cptA* and *pmrC* genes of *S. enterica* are necessary for the addition of pEtN to the LPS core and to lipid A, respectively. Their interruption only reduced the polymyxin B MIC by 2-fold, whereas elimination of L-Ara4N reduced polymyxin B MIC 94-fold. These results suggest that LPS charge is partially responsible for interactions between CAMPS and LPS and that the structure and/or location of the Ara4N modification to lipid A have an inhibitory effect on CAMP binding (Tamayo et al., 2005).

The addition of L-Ara4N to lipid A is mediated by PhoP-PhoQ through transcriptional activation of *pmrA* and *pmrB* genes. In *S. enterica*, the *pmrHFIJKLM* genes are regulated by PmrA-PmrB, and all, except *pmrM*, are necessary for the L-Ara4N addition to lipid A and polymyxin B resistance (Gunn et al., 2000). The function of *pmr* genes and their regulation by PmrA-PmrB has been characterized in the human pathogens *K. pneumoniae*, *P. aeruginosa*, and *S. enterica*. In all cases, the addition of L-Ara4N increased the resistance to CAMPS (Table 1; Moskowitz et al., 2004; Cheng et al., 2010).

### Surface structural changes

The alteration of the net charge of the bacterial cell surface is just one of the adaptations that bacteria have evolved to resist the action of CAMPS. Other mechanisms are directly related to surface structural changes, including the acylation of lipid A, the modification of LPS and LOS, and the formation of capsules. These events are described below.

#### Acylation of lipid A

The acylation of lipid A has been described in Gram-negative human pathogens, such as *Haemophilus influenzae*, *K. pneumoniae*, *Legionella pneumophila*, Salmonella spp, and *Vibrio cholerae* (Table 1) and the mechanism of resistance to CAMPs does not appear to be related to charge alterations. Therefore, it is believed that the increase of lipid A acylation promotes resistance

to CAMPS by decreasing fluidity of the outer membrane, preventing insertion of the peptide.

In *Salmonella typhimurium*, the acylation of lipid A is regulated by the acetyltransferase PagP, which can transfer a palmitate chain from the sn-1 position of a phospholipid, such as phosphatidylethanolamine (PtdEtn), to the free hydroxyl-group of the N-linked R-3-hydroxymyristate chain on the proximal glucosamine unit of lipid A or its precursors (Figure 5; Bishop et al., 2000). *Salmonella pagP* mutants increased outer membrane permeability in response to CAMPS, and the acylation of LPS increased resistance to the α-helical peptide derived from the carboxy terminus of platelet factor IV (C18G), to the peptide starting with a glycine and ending with a leucine amide from the magainin peptide family (pGLa) and PG-1 (Guo et al., 1998). *pagP* homologues have not been reported for *K. pneumoniae*, but the *K. pneumoniae ΔlpzM* mutant showed increased permeability in the outer membrane and an increased susceptibility to α-helical CAMPS (Table 1). The *lpzM* gene encodes the enzyme responsible for the secondary acylation of immature lipid A, and the increase in susceptibility to CAMPS in the Δ*lpzM* mutant is related to less-acylation of lipid A: the lipid A from wild-type LPS is primarily hexa-acylated, while the majority of the *lpzM* mutant is predominantly penta-acylated (palmitoylated pentaacyl-lipid A) (Clements et al., 2007). Consistently, the less-acylation increases the susceptibility to CAMPS; in *H. influenzae* the *htrB* mutant increased by 45-fold its sensitivity to the human defensin HBD-2 compared to wild type. It has a predominantly tetraacyl lipid A compared to the hexa-acylated parental strain (Starner et al., 2002), while in El Tor V. cholerae, which is a natural epidemic biotype relatively resistant to polymyxin B, the insertional *msb* mutation reduced its resistance to all CAMPS tested (Table 1). It shows a less-acylated lipid A compared to the wild-type strain. The FTICR mass spectra from wild type indicated the presence of tetra-, penta- and hexa-acylated lipid As, while the spectra of the *msbB* mutant had similar patterns, but they were missing the peak corresponding to the mass of the hexa-acylated lipid A, suggesting that the lipid A moiety of the *msbB* mutants is less-acylated compared to wild-type lipid A, due to the loss of the final myristate group (Matson et al., 2010).

The acylation of lipid A also plays an important role in virulence. *L. pneumophila* mutants with insertions in the *pagP*-like gene (designated *rcp* for resistance to CAMPS), decreased their resistance to different structural classes of CAMPS (Table 1). During intracellular infection using these mutants there was a reduced number of bacteria recovered. Mutations in *rcp* do not affect bacterial growth, indicating that the differences detected in bacteria recovered from host cells were not due to a generalized growth defect (Robey et al., 2001).

### Modification of lipopolysaccharides and synthesis of lipo oligosaccharides

The modification of LPS and the synthesis and extension of LOS are other mechanisms that confer resistance to CAMPS. The addition of phosphorylcholine (ChoP) to the core polysaccharide of *H. influenzae* confers resistance to cathelicidin LL-37. ChoP can be attached at the C-6 position on the β-D-galactose residue on the only

![Figure 5](https://www.informahealthcare.com/mby)

Figure 5. Acylation of lipid A mediated by PagP. In *S. typhimurium*, PagP transfers a palmitate chain (red) from the sn-1 position of a phospholipid (PtdEtn) to the free hydroxyl-group of the N-linked R-3-hydroxymyristate chain on the proximal glucosamine unit of lipid A (Bishop et al., 2000; Hwang et al., 2002). (See colour version of this figure online at www.informahealthcare.com/mby).
hexose attached to heptose III in the encapsulated type b strain (Eagan), or at the C-6 on the β-D-glucose residue on heptose I in the unencapsulated type d strain (Rd). The constitutive ChoP-expressing mutants of the Rd strain showed a survival increase >1000-fold in response to LL-37. An induction of survival was also detected in the Eagan strain. Because the Rd and Eagan strains have ChoP positioned on different hexoses at different chain extensions on the oligosaccharide, the position of ChoP and other differences in oligosaccharide structure are not determining factors in the sensitivity to LL-37 (Lysenko et al., 2000). Although ChoP is zwitterionic, the positively charged quaternary amine on choline would be predicted to orient toward the exterior of the cell, and it is plausible that the decreased killing by CAMPS is primarily due to the effect of ChoP on the surface charge.

Conversely, extension of LOS would provide protection against CAMPS. In *N. meningitidis* the *lpxA* gene encodes an enzyme responsible for the first step of the lipid A biosynthesis pathway, adding the O-linked 3-OH fatty acid to UDP-N-acetylglucosamine (Albiger et al., 2003). Thus, a *lpxA N. meningitidis* mutant strain does not express LOS and this condition makes it more susceptible to CAMPS. More than 99% of *lpxA* mutant bacteria were killed in the presence of 2.0 μM LL-37, whereas only 20% of wild-type bacteria were killed (Jones et al., 2009). However, the truncation of LOS is sufficient to reduce the resistance to CAMPS. The systemic *C. jejuni* strain 84-25 (with a full-length LOS) is ten-fold more resistant to polymyxin B compared to diarrheal strain 81–176, and is relatively resistant to human α-defensin-5 (HD-5) and the murine Crp4 homologues to HD-5. Its corresponding LOS-truncated mutant (the Δ*waaF* mutant) showed four- to six-fold increased sensitivity to polymyxin B and to CAMPS Crp4 and HD-5. Like others mutants, the complementation of *waaF* in strain 84-25 restored most polymyxin B resistance (Keo et al., 2011). In *C. jejuni*, the extension of the inner and outer cores of LOS is regulated by *cstII*, *galT*, *lgtF*, and *waaF* genes (Figure 6). Any LOS truncation, mediated by insertional inactivation of these genes, resulted in the decrease of resistance to polymyxin B (>15-fold

---

**Figure 6. Regulation of the extension of inner and outer lipooligosaccharides of *Campylobacter jejuni* strain 81-176. Inner- and outer-core regions are indicated. Transferase genes responsible for the addition of each sugar residue are shown in red. GalNAc, N-acetylgalactosamine; NeuAc, N-acetylneuraminic acid (sialic acid); Gal, galactose; Glc, glucose; Hep, heptose; PEtn, phosphoethanolamine; Kdo, 2-keto-3-deoxyoctulosonic acid (Naito et al., 2010). (See colour version of this figure online at www.informahealthcare.com/mby).**

---

Resistance to cationic antimicrobial peptides 9 decrease in MIC). Interestingly, the Δ*waaF* mutant, but not the other LOS mutant strains, also exhibited a defect in intraepithelial cell survival (Naito et al., 2010).

### Formation of cationic slime polymers and capsules

To protect themselves from the host immune response, including the host defense peptides, bacteria can encase themselves by producing elaborate extracellular matrices. The bacterial extracellular matrix is often called “slime.” *S. epidermidis* produces an extracellular, positively charged polysaccharide named polysaccharide intercellular adhesin (PIA). PIA is encoded by the *ica* gene locus that includes the *icaA*, *icaD*, *icaB*, and *icaC* genes (Heilmann et al., 1996), and the anionic extracellular polymer, poly-γ-D,L-glutamic acid (PGA), encoded by *cap* gene in which the α-amino and γ-carboxy groups of D- or L-glutamic acid are linked by isopeptide bonds (Oppermann-Sanio and Steinbüchel, 2002). *S. epidermidis ica* insertion transposon mutants (Vuong et al., 2004a,b) and Δ*cap* mutants (Kocianova et al., 2005) were more susceptible to LL-37 and human β-defensin-3 (HBD-3) than the wild type. In addition, the *ica* mutants were also susceptible to anionic antimicrobial peptide, dermcidin. It is believed that electrostatic repulsion may contribute to PIA-mediated protection against CAMPS. This assumption is supported by the finding that the efficacies of HBD-3 and LL-37 are higher under low salt conditions, which are favorable to electrostatic repulsion, in contrast to the results observed for anionic, dermcidin (Vuong et al., 2004a).

The capsular polysaccharide (CPS) or capsule is the outermost layer of bacteria and may protect them mechanically by impeding the interaction of CAMPS with the surface. *N. meningitidis* has high levels of intrinsic resistance to polymyxin B, but the unencapsulated *siaD(-C)* mutant was highly susceptible to polymyxin B, defensins, cathelicidins, and protegrins (Table 1) compared to the encapsulated bacteria (Spinosa et al., 2007). In *N. meningitidis* the CPS contributes to LL-37 resistance (Jones et al., 2009), while in *K. pneumoniae* the amount of CPS and upregulation of *cps* transcription increased when grown in the presence of polymyxin B and lactoferrin, and the interruption by insertional duplication mutagenesis of the *cps* operon increased the sensitivity to human neutrophil defensin-1, β-defensin-1, lactoferrin, protamine sulfate, and polymyxin B in comparison to wild type (Campos et al., 2004).

Because these polysaccharide matrices protect bacteria from both cationic and anionic host defense peptides, the resistance mechanism probably involves both electrostatic and mechanical (charge-independent) sequestration of host defense peptides (Koprivnjak and Peschel, 2011).

### Trapping and proteolytic degradation of CAMP

Some pathogens produce extracellular molecules that bind and trap CAMPS and, therefore, prevent their
antimicrobial action. This form of resistance can be
accomplished directly through the actions of a bacterial
surface-associated or secreted protein or indirectly by the
induced release of AMP binding molecules from the host
cell surface. Staphylokinase from S. aureus, besides binding
plasminogen that may facilitate S. aureus invasion
of tissues, also forms complexes with human neutrophil
peptide 1 and 2 (HNP-1 and 2), resulting in a greater than
80% reduced activity of these CAMPs against S. aureus
(Jin et al., 2004). Clinically isolated GAS M1 strains exhibited
significantly higher levels of resistance to LL-37 than
strains belonging to other M serotypes associated only
with superficial infections (Lauth et al., 2009). Virulent
GAS M1 strains secrete a Streptococcal inhibitor of
complement protein, called SIC, that is absent in the vast
majority of noninvasive GAS strains. The SIC-deficient
strain is more sensitive to LL-37; therefore, it has been
proposed that SIC protects GAS by direct binding and
inactivation of CAMPs (Frick et al., 2003).

The negatively charged proteoglycan molecules that
decorate the surfaces of host epithelial cells mediate
another mechanism by which bacteria trap and inactivate
CAMPs. Extracellular proteases secreted by the human
pathogens GAS, E. faecalis and P. aeruginosa degrade
decorin and other cell-surface proteoglycans, thereby
releasing dermatan sulfate. Free dermatan sulfate can
bind and inactivate human α-defensins (Schmidtchen
et al., 2001). P. aeruginosa also enhances shedding of
the heparan sulfate proteoglycan syndecan-1 from various
host cells; this phenotype has been attributed to the

actions of a 20-kDa protein virulence factor, LasA, and
can serve to neutralize CAMPs (Table 2; Park et al., 2000).
Alternatively, bacteria cell envelope-associated or
secreted proteases can contribute to disease pathogenesis
by altering host substrates or by inactivating key
components of the host immune response. Nevertheless,
certain important pathogens of humans secrete, or
express on their surface, proteases that recognize and
cleave CAMPs before they reach the membrane, thereby
abolishing their antibacterial activity. Examples of such
secreted proteases (Table 2) are aureolysin from S. aureus
that cleaves LL-37 between the Arg19-Ile20, Arg23-Ile24,
and Leu31-Val32 peptide bonds instantly annihilating its
antibacterial activity (Sieprawska-Lupa et al., 2004); ZapA
from Proteus mirabilis that cleaves HBD-1 and LL-37,
reducing their antimicrobial activity after ZapA treatment
(Belas et al., 2004); streptopain SpeB from S. pyogenes
that mediates inactivation of LL-37 (Schmidtchen
et al., 2002; Johansson et al., 2008); or elastase from P.
aeruginosa (Schmidtchen et al., 2002). Alternatively,
CAMP-degrading proteases can also be associated with
the bacterial cell envelope, either anchored in the cell
wall or in the outer membrane, such as PgtE of S. enterica
(Guina et al., 2000) and E. coli OmpT (Hui et al., 2010).

### Efflux of CAMPs

Efflux pumps are membrane-associated active transporters that promote the extrusion of toxic compounds,
including antibiotics, from the cells. This defense tool is

---

**Table 2. Antimicrobial peptide resistance mechanisms induced by binding and proteolytic degradation of CAMPs.**

| CAMP resistance phenotype | Gene(s) | Activity or resistance to | Bacteria* | Reference |
|---------------------------|---------|--------------------------|-----------|------------|
| **CAMP binding and inactivation** |  |  |  |  |
| Staphylokinase | sak | HNP-1 and 2 | Staphylococcus aureus | Jin et al., 2004 |
| M1 surface protein | emm1 | LL-37 | Group A Streptococcus | Lauth et al., 2009 |
| SIC protein | sic | LL-37 | Group A Streptococcus | Frick et al., 2003 |
| Shedding of host proteoglycans | lasA | LL-37, HNP-1 | Pseudomonas aeruginosa | Park et al., 2001 |
|  |  |  | Enterococcus faecalis | Schmidtchen et al., 2001 |
|  |  |  | Group A Streptococcus | Schmidtchen et al., 2001 |
| **CAMP proteolytic cleavage** |  |  |  |  |
| Elastase | lasB | LL-37 | Pseudomonas aeruginosa | Schmidtchen et al., 2002 |
| Gelatinase | gelE | LL-37 | Enterococcus faecalis | Schmidtchen et al., 2002 |
| Metalloproteinase | zapA | LL-37 | Proteus mirabilis | Schmidtchen et al., 2002 |
|  | aur |  | Staphylococcus aureus | Sieprawska-Lupa et al., 2004 |
|  | degP |  | Escherichia coli | Ulvatne et al., 2002 |
| Cysteine protease | speBideS | LL-37 | Group A Streptococcus | Schmidtchen et al., 2002 |
| Surface protease | pgtE | C18G | Salmonella enterica | Guina et al., 2000 |
| Aureolysin† |  | LL-37 | Staphylococcus aureus | Sieprawska-Lupa et al., 2004 |
| ZapA† |  | LL-37 | Proteus mirabilis | Schmidtchen et al., 2002 |
|  |  |  |  | Belas et al., 2004 |
| SpeB† |  | LL-37 | Streptococcus pyogenes | Schmidtchen et al., 2002 |
|  |  |  |  | Johansson et al., 2008 |

*Not all organisms are resistant to the CAMP indicated; for specific resistance, please consult the reference.

† Purified protein activity evaluation. C18G, α-helical peptide derived from the carboxy terminus of platelet factor IV; HNP-1, human neutrophil peptide 1; HNP-2, human neutrophil peptide 2.
a well-recognized mechanism of bacterial resistance to most classes of antibiotics.

Antibiotic efflux was first discovered in the 1980s, when it was described as a mechanism for tetracycline resistance in enterobacteria (Zechini and Versace, 2009). Although these pumps primarily export hydrophobic drugs, they have relatively low specificity and may also accept a subset of AMPs as substrates, contributing to the intrinsic resistance of certain pathogens to these molecules (Nizet, 2006). Certain bacterial efflux pumps may be selective for one substrate, such as tetracycline, or they may be relatively nonselective conferring a multiple drug resistance (MDR) phenotype; in the latter case, they can transport different compounds (Zechini and Versace, 2009).

Efflux pumps belong to five superfamilies that are classified into two different types accordingly to their mechanisms of action. Type 1 includes the primary transporters that couple drug extrusion from the cell with ATP hydrolysis, and type 2 includes secondary transporters energized by transmembrane electrochemical gradients of either protons or sodium ions (Levy, 2002). These mechanisms have been reported in some human pathogenic bacteria. In *N. gonorrhoeae* and *N. meningitidis*, active transport of CAMPs out of the bacterial cytoplasm and periplasmic space by the resistance-nodulation-cell division (RND)-type efflux pump MtrCDE has been reported to confer resistance to the CAMPs protegrin-1 and LL-37 (Tzeng et al., 2005). However, in a recent study, the MDR exporters (RND-type) from the *E. coli* AcrAB, *P. aeruginosa* MexAB and *S. aureus* NorA major facilitator (MF) efflux pump family were unable to confer resistance to the human CAMPs cathelicidin LL-37, HNP-1 and -3 and HD-5, HBD-2 and -3; indicating that most MDR exporters do not mediate broad CAMP resistance and that only some MDR exporters can expel some types of CAMPs (Rieg et al., 2009).

### Implications of resistance to CAMPs: the evolution of resistant organisms

In this review, we have described and discussed the mechanisms by which some Gram-positive and negative microorganisms have evaded the antimicrobial action of CAMPs, allowing their survival in the presence of toxins. However, there are mechanisms by which populations adapt to the presence of these toxins or CAMPs, being the most important the continued selection. According to the resistance mechanisms here described, the surge or presence of several different kinds of CAMPs with various modes of action at the infection site might have impeded the evolution of resistance in natural bacterial populations. Furthermore, most of the mechanisms of CAMP resistance described above involve the action of several gene products, which makes them difficult to transfer in relation to the conventional antimicrobial resistance determinants located in plasmids. However, we cannot exclude a serious concern related to the long-term risk

Resistance to cationic antimicrobial peptides 11 associated with the development and the use of CAMPs as novel therapeutic agents, mainly to treat chronic infections. Accordingly, some recent evidence suggests that the clinical use of CAMPs can induce the evolution of resistance. Furthermore, there are reports that show that this adaptation also can confer resistance to host CAMPs. In this sense, we must take advantage from the lessons learning from the development of resistance to conventional antibiotics in order to apply new efficient therapies based on CAMPs.

In a well-detailed review, Bell and Gouyon (2003) described the mechanisms established on experimental and theoretical analysis by which CAMP exposition can lead bacteria to evolve resistance. Accordingly, the evolution of resistance depends on the relationship between the frequency of exposure to the toxin or CAMP and the cost of resistance. If there is not a molecular mechanism able to resist toxins, the adaptation will not occur, but in the case of some human pathogens there is recent evidence of a high rate of resistance acquisition to CAMPs.

The first reports of evolving resistance to CAMPs showed that the *in vitro* exposure of bacteria to slowly increasing CAMP concentrations over several hundred generations, results in reversible physiological adaptation or spontaneous inheritable resistance to the CAMP tested (Peschel and Sahl, 2006). Recently, it has been described that this adaptation can also occurs in clinical isolates. Some examples of evolving resistance to CAMPs are described following.

Pexiganan is a synthetic AMP derived from frog magainin, which has been proposed as a candidate for the treatment of diabetic leg infections. With the aim to determine if pexiganan resistant mutants could be generated, in the first experiments Ge et al. (1999) were not able to obtain the mutants following the repeated passage with sub-inhibitory pexiganan concentrations. However, in further studies hereditable resistance to pexiganan was found in continued selection experiments with mutator and nonmutator strains of *E. coli* and *Pseudomonas fluorescens*, suggesting that naturally occurring mutation rates are sufficient to create selectable variation in bacterial populations (Perron et al., 2006). To achieve this, authors gradually increased peptide concentration throughout the experiment. This procedure facilitated the spread of mutations with mildly beneficial effects, which can subsequently be supplemented by further mutations of similar effect, or substituted, by mutations of large effect. This result shows that high levels of heritable resistance to a CAMP can evolve repeatedly in bacterial populations through continued selection in *in vitro* experiments. In addition, in this work authors extend their experiment to 100 serial transfers, constituting 600–700 bacterial generations, allowing time for resistant mutants to appear and spread (Perron et al., 2006). These approaches have not been done before.

Accordingly to previous evidence, acquisition of resistance can be costly for bacterial fitness (Zasloff, 2002; Boman, 2003; Hilpert et al., 2005; Jenssen et al., 2006),
but this is not the rule. *S. enterica* serovar Typhimurium can mutate at a high rate to achieve increased resistance to the cathelicidin PR-39 without negative effects on bacterial fitness (Pränting et al., 2008). Spontaneous and stable mutants decreased their susceptibility to PR-39 at a rate of \(0.4 \times 10^{-6}\) per cell per generation, and the resistance phenotype was associated with different mutations in the gene *sbmA*, coding for a transport protein. These mutants were about four times more resistant than the wild type and the resistance mechanism does not confer any measurable negative effects on bacterial fitness as showed during *in vitro* and *in vivo* experiments (Pränting et al., 2008). A similar behavior was observed by Perron et al. (2006) for *E. coli* and *P. fluorescens* fitness during the acquisition of pexiganan resistance.

In addition to the evolving resistance to CAMPs, the cross-resistance to host AMP is an issue of major concern towards the development of successful treatment strategies. It is predictable that a pathogen with resistance to a particular AMP could be resistant to another with similar action mechanisms. In this sense, there are evidences supporting this idea (Mehla and Sood, 2011). However, some pathogens have evolved cross-resistance to AMPs from diverse origin and with different action mechanisms, showing resistance to host AMP, which may compromise natural immunity. For example, *S. aureus* the predominant pathogen in diabetic leg infections (Ge et al., 2002), have evolved resistance to pexiganan and cross-resistance to HNP-1, a key component of the innate immune response in humans (Habets and Brockhurst, 2012).

In agreement, has been recently reported that *S. aureus* isolates showing reduced susceptibility to daptomycin (DAP; a calcium-dependent molecule acting as a CAMP) may also coevolve reduced *in vitro* susceptibility to the host defense CAMPs: thrombin-induced platelet microbicidal proteins (tPMPs), HNP-1 and polymyxin B; three AMP that differ in structure, action mechanism, and origin. Daptomycin is an anionic lipopeptide \(\mathrm{Ca}^{2+}\)-dependent with bactericidal activity against Gram-positive microorganisms. In the presence of \(\mathrm{Ca}^{2+}\) DAP has a similar action on membranes than CAMPs (Jung et al., 2004). Daptomycin resistant *S. aureus* strains demonstrated significant reduction in killing by the three CAMPs assessed (Mishra et al., 2011). These evidences suggest that unlike resistance to traditional antibiotics, which may select solely for cross-resistance to other antibiotics, resistance to therapeutic AMPs may also compromise natural immune defenses.

There are few reports that show the appearance of resistant strains as consequence of selection pressure, since limited AMPs have been approved or used for clinical treatment. Nisin and colistin are two examples that can describe the risk of the use of AMPs as an antimicrobial therapeutic alternative. The joint FAO/WHO approved nisin, which is produced by certain strains of *Lactococcus lactis* subsp. *lactis*, for use as an antimicrobial in food since 1969 (Delves-Broughton, 2005). However, there are reports of spontaneous nisin resistance. The frequency of

nisin resistance in *L. monocytogenes* varied from \(10^{-2}\) to \(10^{-7}\) in a strain-dependent manner without fitness costs (Gravesen et al., 2002a,b). In addition, nisin resistant *Streptococcus bovis* cultures evolve from a susceptible *S. bovis* strain and remained nisin resistant even if they were transferred 30 times (approximately 120 doublings) in the absence of nisin (Mantovani and Russell, 2001).

On the other hand, colistin is a multicomponent poly-peptide antibiotic, comprised mainly of colistin A and B that became available for clinical use in the 1960s (Li et al., 2005; Falagas and Kasiakou, 2005). Colistin belongs to the polymyxin group of AMPs, which are synthesized in nature by strains of *Paenibacillus* spp. There are two forms of colistin commercially available: colistin sulfate for oral and topical use, and colistimethate sodium for parenteral use; both can be delivered by inhalation (Li et al., 2006). The use of colistin recently received attention because of its significant effect on multidrug-resistant Gram-negative bacteria like *Acinetobacter baumannii* and *P. aeruginosa* (Li et al., 2006), but reports about acquisition of colistin resistance have risen, both in the target pathogen as in emerging pathogens like *Brevundimonas diminuta* and *Ochrobactrum anthropi* (Menuet et al., 2008). Colistin resistance has been clearly associated to selection pressure related to its use. Between 2003 and 2005, 18 colistin-resistant isolates of *K. pneumoniae* were linked with two outbreaks of bacteremia in the same hospital (Antoniadou et al., 2007). The authors of this study concluded that selective pressure due to the extensive or inadequate colistin use may lead to the emergence of colistin resistance among *K. pneumoniae* isolates, potentially increasing morbidity and mortality of critically ill patients. The number of resistance reports has increased each year in at least 14 countries, showing colistin resistance rates between 7 and 40%. Asia and Europe show the most serious situation of colistin resistance, whereas North and South America have presented few reports (Cai et al., 2012).

The mechanism of colistin resistance in *A. baumannii* is mediated by mutations in the genes *lpxA*, *lpxC* and *lpxD*, which result in the lost of the ability to produce lipid A and therefore LPS (Moffatt et al., 2010).

In conclusion, the above evidence suggests that the administration of CAMPs to patients with chronic infections may provoke the evolution of resistance that would not otherwise occur. Thus, the resistance development to CAMPs could become a threat to the efficacy of AMPs if they were used in clinical setting. However, it remains unclear how fast such adaptations can occur *in vivo* (where the presence of a cocktail of defense agents would create a selection pressure that differs greatly from that detected during *in vitro* experiments, being present only one CAMP), how specific these adaptations can be and which are the fitness cost to bacteria.

## Conclusions

In this review, we described several resistance mechanisms developed by bacteria to escape the antibiotic

properties of CAMPs. Because these kinds of molecules have remained effective against bacterial infections over many years, the emergence of resistance is unlikely in the short term, although it is not impossible since recent evidence suggest the evolution of a high rate of resistance to CAMPs in diverse experimental conditions. Considering the resistance mechanisms described, the surge or presence of several different kinds of CAMPs with diverse mechanisms of action might have delayed the evolution of resistance in natural bacterial populations. In general, further studies related to evolution of resistance to CAMPs are necessary, since much of the research in this area has been described under nonphysiological conditions in artificial environments without important immune components (e.g. immune cells, cytokine, chemokines, etc.), and using only one kind of CAMP. In this sense, CAMPs have recently been considered as host defense peptides able to modulate many immune functions in addition to their antimicrobial activity. Thus, the knowledge pertaining to the resistance mechanisms to CAMPs may provide useful information for improving strategies to control infections. For example, combinatorial treatments using CAMPs and classic antibiotics might prove to be an efficacious course of treatment for human and animal infections. In addition, there are evidences that prevent possible surge of cross-resistance toward host CAMPs, which can compromise natural immunity, and highlight the urgent attention to this issue. Thus, resistance is a realistic threat to the efficacy of CAMPs and, as happens with antibiotics, we cannot exclude the fact that resistance may evolve if microbial populations are consistently exposed to CAMPs, in particular during the treatment of chronic infections. Nevertheless, is necessary future research directed to broad our understanding of the mechanisms used by both host and microbes, which will undoubtedly lead to new therapeutic options for managing resistant microbial infections.

### Declaration of interest

This publication was supported by grants from CONACyT to A.O.Z (83895) and J.E.L.M (101451).

### References

- Abachin E, Poyart C, Pellegrini E, Milohanic E, Fiedler F, Berche P, Trieu-Cuot P. (2002). Formation of D-alanyl-lipoteichoic acid is required for adhesion and virulence of *Listeria monocytogenes*. Mol Microbiol, 43, 1–14.
- Albiger B, Johansson L, Jonsson AB. (2003). Lipoooligosaccharide-deficient *Neisseria meningitidis* shows altered pilus-associated characteristics. Infect Immun, 71, 155–162.
- Andrä J, Goldmann T, Ernst CM, Peschel A, Gutsmann T. (2011). Multiple peptide resistance factor (MprF)-mediated resistance of *Staphylococcus aureus* against antimicrobial peptides coincides with a modulated peptide interaction with artificial membranes comprising lysyl-phosphatidylglycerol. J Biol Chem, 286, 18692–18700.
- Antoniadou A, Kontopidou F, Poulakou G, Koratzanis E, Galani I, Papadomichelakis E, Kopterides P, Souli M, Armaganidis A,
- Giamarellou H. (2007). Colistin-resistant isolates of *Klebsiella pneumoniae* emerging in intensive care unit patients: first report of a multiclonal cluster. J Antimicrob Chemother, 59, 786–790.
- Baddiley J. (2000). Teichoic acids in bacterial coaggregation. Microbiology (Reading, Engl), 146 (Pt 6), 1257–1258.
- Belas R, Manos J, Suvanasuthi R. (2004). *Proteus mirabilis* ZapA metalloprotease degrades a broad spectrum of substrates, including antimicrobial peptides. Infect Immun, 72, 5159–5167.
- Bell G, Gouyon PH. (2003). Arming the enemy: the evolution of resistance to self-proteins. Microbiology (Reading, Engl), 149, 1367–1375.
- Bishop RE, Gibbons HS, Guina T, Trent MS, Miller SI, Raetz CR. (2000). Transfer of palmitate from phospholipids to lipid A in outer membranes of gram-negative bacteria. EMBO J, 19, 5071–5080.
- Boman HG. (2003). Antibacterial peptides: basic facts and emerging concepts. J Intern Med, 254, 197–215.
- Cai Y, Chai D, Wang R, Liang B, Bai N. (2012). Colistin resistance of *Acinetobacter baumannii*: clinical reports, mechanisms and antimicrobial strategies. J Antimicrob Chemother, 67, 1607–1615.
- Campos MA, Vargas MA, Regueiro V, Llompart CM, Albertí S, Bengoechea JA. (2004). Capsule polysaccharide mediates bacterial resistance to antimicrobial peptides. Infect Immun, 72, 7107–7114.
- Cheng HY, Chen YF, Peng HL. (2010). Molecular characterization of the PhoPQ-PmrD-PmrAB mediated pathway regulating polymyxin B resistance in *Klebsiella pneumoniae* CG43. J Biomed Sci, 17, 60.
- Clements A, Tull D, Jenney AW, Farn JL, Kim SH, Bishop RE, McPhee JB, Hancock RE, Hartland EL, Pearse MJ, Wijburg OL, Jackson DC, McConville MJ, Strugnell RA. (2007). Secondary acylation of *Klebsiella pneumoniae* lipopolysaccharide contributes to sensitivity to antibacterial peptides. J Biol Chem, 282, 15569–15577.
- Cullen TW, Trent MS. (2010). A link between the assembly of flagella and lipoooligosaccharide of the Gram-negative bacterium *Campylobacter jejuni*. Proc Natl Acad Sci USA, 107, 5160–5165.
- Dawson RM, Liu CQ. (2008). Properties and applications of antimicrobial peptides in biodefense against biological warfare threat agents. Crit Rev Microbiol, 34, 89–107.
- Delves-Broughton J. (2005). Nisin as a food preservative. Food Aust, 57, 525–527.
- Ernst CM, Staubitz P, Mishra NN, Yang SJ, Hornig G, Kalbacher H, Bayer AS, Kraus D, Peschel A. (2009). The bacterial defensin resistance protein MprF consists of separable domains for lipid lysinylation and antimicrobial peptide repulsion. PLoS Pathog, 5, e1000660.
- Fabretti F, Theilacker C, Baldassarri L, Kaczynski Z, Kropec A, Holst O, Huebner J. (2006). Alanine esters of enterococcal lipoteichoic acid play a role in biofilm formation and resistance to antimicrobial peptides. Infect Immun, 74, 4164–4171.
- Falagas ME, Kasiakou SK. (2005). Colistin: the revival of polymyxins for the management of multidrug-resistant gram-negative bacterial infections. Clin Infect Dis, 40, 1333–1341.
- Field D, Quigley L, O'Connor PM, Rea MC, Daly K, Cotter PD, Hill C, Ross RP. (2010). Studies with bioengineered Nisin peptides highlight the broad-spectrum potency of Nisin V. Microb Biotechnol, 3, 473–486.
- Frick IM, Akesson P, Rasmussen M, Schmidtchen A, Björck L. (2003). SIC, a secreted protein of *Streptococcus pyogenes* that inactivates antibacterial peptides. J Biol Chem, 278, 16561–16566.
- Ge Y, MacDonald D, Hait H, Lipsky B, Zasloff M, Holroyd K. (2002). Microbiological profile of infected diabetic foot ulcers. Diabet Med, 19, 1032–1034.
- Ge Y, MacDonald DL, Holroyd KJ, Thornsberry C, Wexler H, Zasloff M. (1999). *In vitro* antibacterial properties of pexiganan, an analog of magainin. Antimicrob Agents Chemother, 43, 782–788.
- Granette C, Nutten S, Palumbo E, Morath S, Hermann C, Dewulf J, Pot B, Hartung T, Hols P, Mercenier A. (2005). Enhanced anti-inflammatory capacity of a *Lactobacillus plantarum* mutant synthesizing modified teichoic acids. Proc Natl Acad Sci USA, 102, 10321–10326.
- Gravesen A, Jydegaard Axelsen AM, Mendes da Silva J, Hansen TB, Knøchel S. (2002a). Frequency of bacteriocin resistance
© 2012 Informa Healthcare USA, Inc.

development and associated fitness costs in *Listeria monocytogenes*. Appl Environ Microbiol, 68, 756–764.

Gravesen A, Ramnath M, Rechinger KB, Andersen N, Jänsch L, Héchard Y, Hastings JW, Knöchel S. (2002b). High-level resistance to class IIa bacteriocins is associated with one general mechanism in *Listeria monocytogenes*. Microbiology (Reading, Engl), 148, 2361–2369.

Guina T, Yi EC, Wang H, Hackett M, Miller SI. (2000). A PhoP-regulated outer membrane protease of *Salmonella enterica* serovar typhimurium promotes resistance to alpha-helical antimicrobial peptides. J Bacteriol, 182, 4077–4086.

Gunn JS, Ryan SS, Van Velkinburgh JC, Ernst RK, Miller SI. (2000). Genetic and functional analysis of a PmrA-PmrB-regulated locus necessary for lipopolysaccharide modification, antimicrobial peptide resistance, and oral virulence of *Salmonella enterica* serovar typhimurium. Infect Immun, 68, 6139–6146.

Guo L, Lim KB, Poduje CM, Daniel M, Gunn JS, Hackett M, Miller SI. (1998). Lipid A acylation and bacterial resistance against vertebrate antimicrobial peptides. Cell, 95, 189–198.

Habets MG, Brockhurst MA. (2012). Therapeutic antimicrobial peptides may compromise natural immunity. Biol Lett, 8, 416–418.

Hancock RE, Chapple DS. (1999). Peptide antibiotics. Antimicrob Agents Chemother, 43, 1317–1323.

Heilmann C, Schweitzer O, Gerke C, Vanittanakom N, Mack D, Götz F. (1996). Molecular basis of intercellular adhesion in the biofilm-forming *Staphylococcus epidermidis*. Mol Microbiol, 20, 1083–1091.

Heuer OE, Hammerum AM, Collignon P, Wegener HC. (2006). Human health hazard from antimicrobial-resistant enterococci in animals and food. Clin Infect Dis, 43, 911–916.

Hilpert K, Volkmer-Engert R, Walter T, Hancock RE. (2005). High-throughput generation of small antibacterial peptides with improved activity. Nat Biotechnol, 23, 1008–1012.

Hui CY, Guo Y, He QS, Peng L, Wu SC, Cao H, Huang SH. (2010). *Escherichia coli* outer membrane protease OmpT confers resistance to urinary cationic peptides. Microbiol Immunol, 54, 452–459.

Hwang PM, Choy WY, Lo EI, Chen L, Forman-Kay JD, Raetz CR, Privé GG, Bishop RE, Kay LE. (2002). Solution structure and dynamics of the outer membrane enzyme PagP by NMR. Proc Natl Acad Sci USA, 99, 13560–13565.

Jensen H, Hamill P, Hancock RE. (2006). Peptide antimicrobial agents. Clin Microbiol Rev, 19, 491–511.

Jin T, Bokarewa M, Foster T, Mitchell J, Higgins J, Tarkowski A. (2004). *Staphylococcus aureus* resists human defensins by production of staphylokinase, a novel bacterial evasion mechanism. J Immunol, 172, 1169–1176.

Johansson L, Thulin P, Sendi P, Hertzén E, Linder A, Akesson P, Low DE, Agerberth B, Norrby-Teglund A. (2008). Cathelicidin LL-37 in severe *Streptococcus pyogenes* soft tissue infections in humans. Infect Immun, 76, 3399–3404.

Jones A, Georg M, Maudsdotter L, Jonsson AB. (2009). Endotoxin, capsule, and bacterial attachment contribute to *Neisseria meningitidis* resistance to the human antimicrobial peptide LL-37. J Bacteriol, 191, 3861–3868.

Jung D, Rozek A, Okon M, Hancock RE. (2004). Structural transitions as determinants of the action of the calcium-dependent antibiotic daptomycin. Chem Biol, 11, 949–957.

Keo T, Collins J, Kunwar P, Blaser MJ, Iovine NM. (2011). Campylobacter capsule and lipoooligosaccharide confer resistance to serum and cationic antimicrobials. Virulence, 2, 30–40.

Klein S, Lorenzo C, Hoffmann S, Walther JM, Storbeck S, Piekarski T, Tindall BJ, Wray V, Nimtz M, Moser J. (2009). Adaptation of *Pseudomonas aeruginosa* to various conditions includes tRNA-dependent formation of alanyl-phosphatidylglycerol. Mol Microbiol, 71, 551–565.

Kociánova S, Vuong C, Yao Y, Voyich JM, Fischer ER, DeLeo FR, Otto M. (2005). Key role of poly-gamma-DL-glutamic acid in immune evasion and virulence of *Staphylococcus epidermidis*. J Clin Invest, 115, 688–694.

Koprivnjak T, Peschel A. (2011). Bacterial resistance mechanisms against host defense peptides. Cell Mol Life Sci, 68, 2243–2254.

Kovács M, Halfmann A, Fedtke I, Heintz M, Peschel A, Vollmer W, Hakenbeck R, Brückner R. (2006). A functional dlt operon, encoding proteins required for incorporation of d-alanine in teichoic acids in gram-positive bacteria, confers resistance to cationic antimicrobial peptides in *Streptococcus pneumoniae*. J Bacteriol, 188, 5797–5805.

Kristian SA, Datta V, Weidenmaier C, Kansal R, Fedtke I, Peschel A, Gallo RL, Nizet V. (2005). D-alanylation of teichoic acids promotes group a streptococcus antimicrobial peptide resistance, neutrophil survival, and epithelial cell invasion. J Bacteriol, 187, 6719–6725.

Kristian SA, Dürr M, Van Strijp JA, Neumeister B, Peschel A. (2003). MprF-mediated lysinylation of phospholipids in *Staphylococcus aureus* leads to protection against oxygen-independent neutrophil killing. Infect Immun, 71, 546–549.

Lauth X, von Köckritz-Blickwede M, McNamara CW, Myskowski S, Zinkernagel AS, Beall B, Ghosh P, Gallo RL, Nizet V. (2009). M1 protein allows Group A streptococcal survival in phagocyte extracellular traps through cathelicidin inhibition. J Innate Immun, 1, 202–214.

Lee H, Hsu FF, Turk J, Groisman EA. (2004). The PmrA-regulated pmrC gene mediates phosphoethanolamine modification of lipid A and polymyxin resistance in *Salmonella enterica*. J Bacteriol, 186, 4124–4133.

Levy SB. (2002). Active efflux, a common mechanism for biocide and antibiotic resistance. Symp Ser Soc Appl Microbiol, 92, 65S–71S.

Lewis LA, Choudhury B, Balthazar JT, Martin LE, Ram S, Rice PA, Stephens DS, Carlson R, Shafer WM. (2009). Phosphoethanolamine substitution of lipid A and resistance of *Neisseria gonorrhoeae* to cationic antimicrobial peptides and complement-mediated killing by normal human serum. Infect Immun, 77, 1112–1120.

Li J, Nation RL, Milne RW, Turnidge JD, Coulthard K. (2005). Evaluation of colistin as an agent against multi-resistant Gram-negative bacteria. Int J Antimicrob Agents, 25, 11–25.

Li J, Nation RL, Turnidge JD, Milne RW, Coulthard K, Rayner CR, Paterson DL. (2006). Colistin: the re-emerging antibiotic for multidrug-resistant Gram-negative bacterial infections. Lancet Infect Dis, 6, 589–601.

López-Meza JE, Ochoa-Zarzosa A, Aguilar JA, Loeza-Lara PD. (2011). Antimicrobial peptides: diversity and perspectives for their biomedical application, biomedical engineering, trends, research and technologies. In: Komorowska MA, Olsztynska-Janusi S, (eds). Biomedical Engineering, Trends, Research and Technologies. Rijeka, Croatia: Intech, 275–304.

Lysenko ES, Gould J, Bals R, Wilson JM, Weiser JN. (2000). Bacterial phosphorylcholine decreases susceptibility to the antimicrobial peptide LL-37/hCAP18 expressed in the upper respiratory tract. Infect Immun, 68, 1664–1671.

Maloney E, Stankowska D, Zhang J, Fol M, Cheng QJ, Lun S, Bishai WR, Rajagopalan M, Chatterjee D, Madiraju MV. (2009). The two-domain LysX protein of *Mycobacterium tuberculosis* is required for production of lysylated phosphatidylglycerol and resistance to cationic antimicrobial peptides. PLoS Pathog, 5, e1000534.

Mantovani HC, Russell JB. (2001). Nisin resistance of *Streptococcus bovis*. Appl Environ Microbiol, 67, 808–813.

Matson JS, Yoo HJ, Hakansson K, Dirita VJ. (2010). Polymyxin B resistance in El Tor *Vibrio cholerae* requires lipid acylation catalyzed by MsbB. J Bacteriol, 192, 2044–2052.

McCoy AJ, Liu H, Falla TJ, Gunn JS. (2001). Identification of *Proteus mirabilis* mutants with increased sensitivity to antimicrobial peptides. Antimicrob Agents Chemother, 45, 2030–2037.

Mehla J, Sood SK. (2011). Substantiation in *Enterococcus faecalis* of dose-dependent resistance and cross-resistance to pore-forming antimicrobial peptides by use of a polydiacetylene-based colorimetric assay. Appl Environ Microbiol, 77, 786–793.

Menuet M, Bittar F, Stremler N, Dubus JC, Sarles J, Raoult D, Rolain JM. (2008). First isolation of two colistin-resistant emerging pathogens, *Brevundimonas diminuta* and *Ochrobactrum anthropi*, in a woman with cystic fibrosis: a case report. J Med Case Rep, 2, 373.

Mishra NN, McKinnell J, Yeaman MR, Rubio A, Nast CC, Chen L, Kreiswirth BN, Bayer AS. (2011). *In vitro* cross-resistance to

daptomycin and host defense cationic antimicrobial peptides in clinical methicillin-resistant *Staphylococcus aureus* isolates. Antimicrob Agents Chemother, 55, 4012–4018.

• Moffatt JH, Harper M, Harrison P, Hale JD, Vinogradov E, Seemann T, Henry R, Crane B, St Michael F, Cox AD, Adler B, Nation RL, Li J, Boyce JD. (2010). Colistin resistance in *Acinetobacter baumannii* is mediated by complete loss of lipopolysaccharide production. Antimicrob Agents Chemother, 54, 4971–4977.

• Morath S, Geyer A, Hartung T. (2001). Structure-function relationship of cytokine induction by lipoteichoic acid from *Staphylococcus aureus*. J Exp Med, 193, 393–397.

• Moskowitz SM, Ernst RK, Miller SI. (2004). PmrAB, a two-component regulatory system of *Pseudomonas aeruginosa* that modulates resistance to cationic antimicrobial peptides and addition of aminoarabinose to lipid A. J Bacteriol, 186, 575–579.

• Naito M, Frirdich E, Fields JA, Pryjma M, Li J, Cameron A, Gilbert M, Thompson SA, Gaynor EC. (2010). Effects of sequential *Campylobacter jejuni* 81–176 lipo-oligosaccharide core truncations on biofilm formation, stress survival, and pathogenesis. J Bacteriol, 192, 2182–2192.

• Neuhaus FC, Baddiley J. (2003). A continuum of anionic charge: structures and functions of D-alanyl-teichoic acids in gram-positive bacteria. Microbiol Mol Biol Rev, 67, 686–723.

• Nishi H, Komatsuzawa H, Fujiwara T, McCallum N, Sugai M. (2004). Reduced content of lysyl-phosphatidylglycerol in the cytoplasmic membrane affects susceptibility to moenomycin, as well as vancomycin, gentamicin, and antimicrobial peptides, in *Staphylococcus aureus*. Antimicrob Agents Chemother, 48, 4800–4807.

• Nizet V. (2006). Antimicrobial peptide resistance mechanisms of human bacterial pathogens. Curr Issues Mol Biol, 8, 11–26.

• Oppermann-Sanio FB, Steinbüchel A. (2002). Occurrence, functions and biosynthesis of polyamines in microorganisms and biotechnological production. Naturwissenschaften, 89, 11–22.

• Park PW, Pier GB, Hinkes MT, Bernfield M. (2001). Exploitation of syndecan-1 shedding by *Pseudomonas aeruginosa* enhances virulence. Nature, 411, 98–102.

• Park PW, Pier GB, Preston MJ, Goldberger O, Fitzgerald ML, Bernfield M. (2000). Syndecan-1 shedding is enhanced by LasA, a secreted virulence factor of *Pseudomonas aeruginosa*. J Biol Chem, 275, 3057–3064.

• Pence MA, Rooijakkers SH, Cogen AL, Cole JN, Hollands A, Gallo RL, Nizet V. (2010). Streptococcal inhibitor of complement promotes innate immune resistance phenotypes of invasive M1T1 group A *Streptococcus*. J Innate Immun, 2, 587–595.

• Pérez-Gutiérrez C, Llobet E, Llompart CM, Reinés M, Bengoechea JA. (2010). Role of lipid A acylation in *Yersinia enterocolitica* virulence. Infect Immun, 78, 2768–2781.

• Perron GG, Zasloff M, Bell G. (2006). Experimental evolution of resistance to an antimicrobial peptide. Proc Biol Sci, 273, 251–256.

• Peschel A, Jack RW, Otto M, Collins LV, Staubitz P, Nicholson G, Kalbacher H, Nieuwenhuizen WF, Jung G, Tarkowski A, van Kessel KP, van Strijp JA. (2001). *Staphylococcus aureus* resistance to human defensins and evasion of neutrophil killing via the novel virulence factor MprF is based on modification of membrane lipids with l-lysine. J Exp Med, 193, 1067–1076.

• Peschel A, Otto M, Jack RW, Kalbacher H, Jung G, Götz F. (1999). Inactivation of the dlt operon in *Staphylococcus aureus* confers sensitivity to defensins, protegrins, and other antimicrobial peptides. J Biol Chem, 274, 8405–8410.

• Peschel A, Sahl HG. (2006). The co-evolution of host cationic antimicrobial peptides and microbial resistance. Nat Rev Microbiol, 4, 529–536.

• Poyart C, Pellegrini E, Marceau M, Baptista M, Jaubert F, Lamy MC, Trieu-Cuot P. (2003). Attenuated virulence of *Streptococcus agalactiae* deficient in D-alanyl-lipoteichoic acid is due to an increased susceptibility to defensins and phagocytic cells. Mol Microbiol, 49, 1615–1625.


Resistance to cationic antimicrobial peptides 15

• Pränting M, Negrea A, Rhen M, Andersson DI. (2008). Mechanism and fitness costs of PR-39 resistance in *Salmonella enterica* serovar Typhimurium LT2. Antimicrob Agents Chemother, 52, 2734–2741.

• Preston A, Mandrell RE, Gibson BW, Apicella MA. (1996). The lipo-oligosaccharides of pathogenic gram-negative bacteria. Crit Rev Microbiol, 22, 139–180.

• Raetz CR, Reynolds CM, Trent MS, Bishop RE. (2007). Lipid A modification systems in gram-negative bacteria. Annu Rev Biochem, 76, 295–329.

• Rieg S, Huth A, Kalbacher H, Kern WV. (2009). Resistance against antimicrobial peptides is independent of *Escherichia coli* AcrAB, *Pseudomonas aeruginosa* MexAB and *Staphylococcus aureus* NorA efflux pumps. Int J Antimicrob Agents, 33, 174–176.

• Rietschel ET, Kirikae T, Schade FU, Mamat U, Schmidt G, Loppnow H, Ulmer AJ, Zähringer U, Seydel U, Di Padova F. (1994). Bacterial endotoxin: molecular relationships of structure to activity and function. FASEB J, 8, 217–225.

• Robey M, O’Connell W, Gianciotto NP. (2001). Identification of *Legionella pneumophila rcp*, a pagP-like gene that confers resistance to cationic antimicrobial peptides and promotes intracellular infection. Infect Immun, 69, 4276–4286.

• Roy H, Ibba M. (2008). RNA-dependent lipid remodeling by bacterial multiple peptide resistance factors. Proc Natl Acad Sci USA, 105, 4667–4672.

• Roy H. (2009). Tuning the properties of the bacterial membrane with aminoacylated phosphatidylglycerol. IUBMB Life, 61, 940–953.

• Samant S, Hsu FF, Neyfakh AA, Lee H. (2009). The *Bacillus anthracis* protein MprF is required for synthesis of lysylphosphatidylglycerols and for resistance to cationic antimicrobial peptides. J Bacteriol, 191, 1311–1319.

• Schmidtchen A, Frick IM, Andersson E, Tapper H, Björck L. (2002). Proteases of common pathogenic bacteria degrade and inactivate the antibacterial peptide LL-37. Mol Microbiol, 46, 157–168.

• Schmidtchen A, Frick IM, Björck L. (2001). Dermatan sulphate is released by proteinases of common pathogenic bacteria and inactivates antibacterial alpha-defensin. Mol Microbiol, 39, 708–713.

• Sieprawska-Lupa M, Mydel P, Krawczyk K, Wójcik K, Puklo M, Lupa B, Suder P, Silberring J, Reed M, Pohl J, Shafer W, McAleese F, Foster T, Travis J, Potempa J. (2004). Degradation of human antimicrobial peptide LL-37 by *Staphylococcus aureus*-derived proteinases. Antimicrob Agents Chemother, 48, 4673–4679.

• Spinosa MR, Progida C, Talà A, Cogli L, Alifano P, Bucci C. (2007). The *Neisseria meningitidis* capsule is important for intracellular survival in human cells. Infect Immun, 75, 3594–3603.

• Starner TD, Swords WE, Apicella MA, McCray PB Jr. (2002). Susceptibility of nontypeable *Haemophilus influenzae* to human beta-defensins is influenced by lipo-oligosaccharide acylation. Infect Immun, 70, 5287–5289.

• Staubitz P, Neumann H, Schneider T, Wiedemann I, Peschel A. (2004). MprF-mediated biosynthesis of lysylphosphatidylglycerol, an important determinant in staphylococcal defensin resistance. FEMS Microbiol Lett, 231, 67–71.

• Tamayo R, Choudhury B, Septer A, Merighi M, Carlson R, Gunn JS. (2005). Identification of cptA, a PmrA-regulated locus required for phosphoethanolamine modification of the *Salmonella enterica* serovar typhimurium lipopolysaccharide core. J Bacteriol, 187, 3391–3399.

• Thedieck K, Hain T, Mohamed W, Tindall BJ, Nimtz M, Chakraborty T, Wehland J, Jänsch L. (2006). The MprF protein is required for lysinylation of phospholipids in listerial membranes and confers resistance to cationic antimicrobial peptides (CAMPs) on *Listeria monocytogenes*. Mol Microbiol, 62, 1325–1339.

• Tran AX, Whittimore JD, Wyrick PB, McGrath SC, Cotter RJ, Trent MS. (2006). The lipid A 1-phosphatase of *Helicobacter pylori* is required for resistance to the antimicrobial peptide polymyxin. J Bacteriol, 188, 4531–4541.

J.L. Anaya-López et al.

- Tzeng YL, Ambrose KD, Zughair S, Zhou X, Miller YK, Shafer WM, Stephens DS. (2005). Cationic antimicrobial peptide resistance in *Neisseria meningitidis*. J Bacteriol, 187, 5387–5396.
- Ulvatne H, Haukland HH, Samuelsen Ø, Krämer M, Vorland LH. (2002). Proteases in *Escherichia coli* and *Staphylococcus aureus* confer reduced susceptibility to lactoferricin B. J Antimicrob Chemother, 50, 461–467.
- Vaara M, Fox J, Loidl G, Siikanen O, Apajalahti J, Hansen F, Frimodt-Møller N, Nagai J, Takano M, Vaara T. (2008). Novel polymyxin derivatives carrying only three positive charges are effective antibacterial agents. Antimicrob Agents Chemother, 52, 3229–3236.
- Vaara M. (1992). Agents that increase the permeability of the outer membrane. Microbiol Rev, 56, 395–411.
- Vuong C, Kocianova S, Voyich JM, Yao Y, Fischer ER, DeLeo FR, Otto M. (2004a). A crucial role for exopolysaccharide modification in bacterial biofilm formation, immune evasion, and virulence. J Biol Chem, 279, 54881–54886.
- Vuong C, Voyich JM, Fischer ER, Braughton KR, Whitney AR, DeLeo FR, Otto M. (2004b). Polysaccharide intercellular adhesin (PIA) protects *Staphylococcus epidermidis* against major components of the human innate immune system. Cell Microbiol, 6, 269–275.
- Yeaman MR, Yount NY. (2003). Mechanisms of antimicrobial peptide action and resistance. Pharmacol Rev, 55, 27–55.
- Zasloff M. (2002). Antimicrobial peptides of multicellular organisms. Nature, 415, 389–395.
- Zechini B, Versace I. (2009). Inhibitors of multidrug resistant efflux systems in bacteria. Recent Pat Antiinfect Drug Discov, 4, 37–50.
