
For reprint orders, please contact: reprints@futuremedicine.com

# Acquired inducible antimicrobial resistance in Gram-positive bacteria

**Scott T Chancey**${}^{1,2}$, **Dorothea Zähner**${}^{1,2}$ & **David S Stephens**${}^{*1,2,3}$

${}^{1}$Division of Infectious Diseases, Department of Medicine, Emory University School of Medicine, Atlanta, GA 30322, USA  
${}^{2}$Department of Veterans Affairs Medical Center (Atlanta), Decatur, GA 30033, USA  
${}^{3}$Department of Microbiology & Immunology, Emory University School of Medicine, Atlanta, GA 30322, USA  

*Author for correspondence: Tel.: +1 404 728 7688 ▪ dstep01@emory.edu*

A major contributor to the emergence of antibiotic resistance in Gram-positive bacterial pathogens is the expansion of acquired, inducible genetic elements. Although acquired, inducible antibiotic resistance is not new, the interest in its molecular basis has been accelerated by the widening distribution and often 'silent' spread of the elements responsible, the diagnostic challenges of such resistance and the mounting limitations of available agents to treat Gram-positive infections. Acquired, inducible antibiotic resistance elements belong to the accessory genome of a species and are horizontally acquired by transformation/recombination or through the transfer of mobile DNA elements. The two key, but mechanistically very different, induction mechanisms are: ribosome-sensed induction, characteristic of the macrolide-lincosamide-streptogramin B antibiotics and tetracycline resistance, leading to ribosomal modifications or efflux pump activation; and resistance by cell surface-associated sensing of β-lactams (e.g., oxacillin), glycopeptides (e.g., vancomycin) and the polypeptide bacitracin, leading to drug inactivation or resistance due to cell wall alterations.

Antibiotic-resistant Gram-positive bacterial pathogens (e.g., *Streptococcus pneumoniae*, *Streptococcus pyogenes*, *Streptococcus agalactiae*, *Staphylococcus aureus*, *Enterococcus faecalis* and *Enterococcus faecium*) are responsible for community-acquired and hospital-associated infections and are an increasing public health threat. Bacterial resistance has developed to most clinically relevant antibiotics and is typically achieved through one of three mechanisms: target modification, drug inactivation or active drug efflux. These mechanisms can come with a cost to the fitness of the bacteria, but the costs can be mitigated by tight regulatory control of expression of the genes and/or proteins responsible for resistance in the absence of the antibiotic. Because constitutive expression of resistance determinants can have fitness costs and creates selective pressure against the dissemination of resistance, inducible resistance determinants disseminate more readily and persist longer in bacterial populations. In addition, the often transient nature of expression complicates clinical detection of inducible resistance by phenotypic susceptibility assays. This can lead to treatment failures because isolates may be more resistant *in vivo* than determined *in vitro* or may have a wider spectrum of resistance than expected. An understanding of the molecular basis of induction, including the physiological role and the mechanisms of sensing and responding to antibiotics, allows the evaluation of induction mechanisms as potential antimicrobial targets and aids in the design and discovery of noninducing antimicrobial agents.

While antibiotic resistance mechanisms can be intrinsic to the core genome of a bacterial species or acquired but constitutively expressed due to mutations, this review focuses on the inducible resistance genes that belong to the accessory genome of a species. These genes are often silent unless induced and are horizontally acquired by transformation/recombination or through association with mobile genetic elements (MGEs) that include: insertion sequences (IS elements), integrative and conjugative elements (ICEs, or transposons), bacteriophages and plasmids. In contrast to resistance determinants encoded on the core genome, acquired resistance genes pose a special clinical problem, since they are not present in all strains of a species, and are therefore less predictable. In addition, these resistance determinants may cross species or even genera borders and may thereby contribute to the emergence of new resistance phenomena.

Induction of resistance is defined, for the purpose of this review, as reversibly altered expression of an acquired antibiotic resistance determinant. This definition is in contrast to

---

### Keywords
- antimicrobial resistance
- Gram-positive bacteria
- inducible resistance
- mobile genetic elements
- ribosomal stalling
- transposon

---

10.2217/FMB.12.63 © 2012 Future Medicine Ltd  
*Future Microbiol.* (2012) 7(8), 959–978  
ISSN 1746-0913

the sometimes used term ‘induction’ that refers  
to the occurrence of point mutations that alter  
the genotype and become a constitutive, integral  
part of the host phenotype. Also, general stress  
responses, involving complex global regulatory  
cascades that result in nonspecific physiological  
changes that allow bacteria to better tolerate the  
stress, are not part of this definition. Both of  
these resistance mechanisms generally involve  
genes in the bacterial species core genome,  
which are often essential and therefore would be  
expected to function in most, if not all, isolates  
of a bacterial species.

---

### Gene pool for inducible antibiotic  
resistance determinants: MGEs  

Inducible antibiotic resistance genes are widely  
disseminated in Gram-positive genera as  
genetic cargo on infinitely diverse and increasingly  
complex MGEs. The predominant mechanism of horizontal  
transfer varies by genera; however, inducible antibiotic  
resistance genes have recombined into different classes of MGEs  
facilitating intra- and inter-species horizontal  
transfer. Common MGEs in Gram-positives  
include ICEs, plasmids and bacteriophages.  
Conjugation is the transfer of MGEs by direct  
cell-to-cell contact through structures encoded  
by the MGE in donor cells. ICEs encode the  
necessary proteins for excision from the host  
cell chromosome, formation of closed circular  
intermediates, and transfer by conjugation and  
integration into the chromosome of recipient  
cells (reviewed in [1]). ICEs are ubiquitous,  
having been described in virtually all classifications  
of bacteria and are promiscuous as the transfer between  
distantly related bacteria occurs readily [2].  

The most widely disseminated ICE in  
Gram-positives is the Tn916 family of conjugative  
transposons [3]. Tn916 and Tn916-like  
elements are characterized by inducible tetracycline  
resistance, but also frequently harbor  
inducible macrolide and chloramphenicol resistance  
as well as constitutively expressed resistance  
determinants (FIGURE 1A) [3]. Tn916-like  
elements have been identified in virtually all  
Gram-positive genera including *Staphylococcus*,  
*Streptococcus*, *Enterococcus* and *Clostridium* [3].  
Inducible vancomycin resistance is disseminated  
by the Tn1546-like transposons that belong to  
the Tn3 family, often associated with plasmids  
(FIGURE 1B) [4–8]. Also often associated with plasmids  
is Tn552, which carries inducible β-lactam  
resistance (FIGURE 1C). The most clinically relevant  
resistance-conferring elements in staphylococci  

are the staphylococcal cassette chromosomes  
(SCCs). SCC elements carrying methicillin  
resistance genes (SCCmec) are the determinants of methicillin-  
resistant *S. aureus* (MRSA) isolates (FIGURE 1D).  
SCCmec elements are extremely diverse and are  
classified based on the combination of the type  
of cassette chromosome recombinases (*ccr*) and  
class of *mec* gene complex [9,10].

Plasmids, which are autonomously replicating extrachromosomal elements, can also harbor inducible resistance genes and be horizontally transferred by conjugation (reviewed in [11]) or transformation. Plasmids with inducible antibiotic resistance genes are particularly important in the dissemination of resistance in *Staphylococcus* and *Enterococcus* species [10,12]. Bacteriophages can also transfer inducible antibiotic resistance genes [13]. Phages are associated with different inducible macrolide resistance genes and cassettes carried by transposons in *S. pneumoniae* and other Gram-positive species [14]. β-lactamases can be disseminated in staphylococci by bacteriophages [15]. Antibiotic stress can induce the lysogenic transfer of bacteriophages and therefore promote dissemination of resistance through induction of an SOS-like response [14]. Similarly, horizontal gene transfer can be induced by DNA-damaging compounds in *Bacillus subtilis* [16] and by antibiotic stress in *S. pneumoniae* [17].

---

### Induction of antibiotic resistance  
mediated by ribosome-associated sensing  

The ribosome is the microbial target for many  
antibiotics utilized in the treatment of Gram-  
positive bacterial infections. Antibiotics that bind  
bacterial ribosomes include the tetracyclines,  
chloramphenicol and the MLS<sub>B</sub> antibiotics:  
macrolides, lincomamides and streptogramin B.  
MLS<sub>B</sub> antibiotics are commonly used options for  
the treatment of confirmed Gram-positive infections as well as for empirical treatment of respiratory tract infections, which may be due to Gram-  
positive bacteria. Macrolides contain 14-, 15- or  
16-membered lactone rings with extensive glycosidic substitutions that are involved in binding to  
the ribosome. Structurally distinct lincomamides  
and streptogramins bind to similar sites on the  
ribosome as the macrolides. Binding of MLS<sub>B</sub>  
antibiotics blocks elongation of nascent peptides  
and inhibits protein translation, but binding of  
certain macrolides (i.e., 14- and 15-membered)  
also triggers expression of inducible MLS<sub>B</sub> and  
macrolide resistance genes.

Acquired inducible antimicrobial resistance in Gram-positive bacteria

Review

(A)
Tn917 (5.3 kb)
tnpA tnpR erm(B)

Tn3872 (23.3 kb)

Tn916 (18.0 kb)
int 5 8 7 10 6 tet(M) 12 13 14 15 16 17 19 20 21 23
xis 9 18 22 24

Tn2009 (23.5 kb)
erm(B)

Tn2010 (26.3 kb)
mef(E) mel

Tn6002 (20.8 kb)
aphA-3 erm(B)

Tn6003 (25.0 kb)
aadE

MEGA (5.5 kb)
Omega element MAS element
(2.8 kb) (4.2 kb)

(B)
Tn1546 (10.9 kb)
IR<sub>L</sub> orf1 orf2 vanR vanS vanH vanA vanX vanY vanZ IR<sub>R</sub>

(C)
Tn552 (6.5 kb)
orf271 orf480 binL blal blaR1 blaZ

(D)
SCCmec type II (52 kb)
attR mecA ccrB2 attL
IS431 mecR1 Tn554 ccrA2

Figure 1. Representative mobile genetic elements carrying inducible resistance genes.
(A) Conjugative transposon Tn916 and Tn916-like mosaic elements elements carrying inducible macrolide resistance (blue) and tetracycline resistance (yellow) genes. (B) Conjugative transposon Tn1546 encoding inducible vancomycin resistance. (C) Transposon Tn552 encoding inducible β-lactam resistance. (D) Staphylococcal cassette chromosome encoding resistance to β-lactams. Resistance determinants are indicated in blue; genes encoding regulatory (red) or accessory functions (black) for antibiotic resistance are also indicated. See text for details.

Acquired inducible resistance of Gram-positive bacteria by ribosomal-binding antibiotics occurs primarily by target modification or by active drug efflux. The most common determinant is the class of rRNA methylase genes known as *erm* (erythromycin resistance methylases) that confer resistance to the MLS<sub>B</sub> antibiotics. Erm methylases confer high-level resistance to the MLS<sub>B</sub> antibiotics by the transfer of one or two methyl groups to a specific adenine nucleotide of the 23S rRNA subunit of the 50S subunit of the ribosome. This blocks binding of MLS<sub>B</sub> antibiotics. Conversely, macrolide efflux pumps confer low-level resistance to 14- and 15-membered but less commonly 16-membered macrolides. Expression of *erm* and other MLS<sub>B</sub> resistance mechanisms likely has a cost in bacterial fitness and resistance gene expression is typically repressed in the absence of the drug.

Induction of methylase and efflux-mediated resistance involves complex molecular interactions between the antibiotics and various components of the ribosomal complex. MLS<sub>B</sub> antibiotics bind to the 50S ribosomal subunit in the nascent peptide exit tunnel (NPET), thereby disrupting the incorporation of amino acids into nascent peptides in the peptidyl transferase center (PTC). However, inducible MLS<sub>B</sub> resistance genes contain a leader peptide that evokes leader peptide-mediated stalling of the ribosome upon binding of a MLS<sub>B</sub> antibiotic. This permits expression of the resistance determinant and thus loss of susceptibility of the host to the antimicrobial compound. We review recent developments of the mechanisms by which ribosome stalling in the presence of different translation inhibiting antibiotics results in the expression of inducible resistance genes.

Review Chancey, Zähner & Stephens

Induction of ribosome methylases by MLS<sub>B</sub> antibiotics

Genes encoding Erm methylases are diverse and widespread in Gram-positive bacteria. To date, 34 erm alleles have been identified and all but two have been reported in Gram-positive pathogens [18]. The classes of erm genes, erm(A), erm(B) and erm(C) are the most widely distributed in Gram-positive pathogens (TABLE 1) [19]. Distinctions between each erm class are based on bacterial host range and the MLS<sub>B</sub> resistance phenotype that is conferred. Each erm is predominantly associated with, but not limited to, one or two genera [18, 19, 201]; erm(A) is often associated with Staphylococcus, but has been reported in four other Gram-positive genera

including Streptococcus and Enterococcus [20, 21]; erm(B) is the prevalent determinant of MLS<sub>B</sub> resistance in streptococci but has been observed in 17 additional Gram-positive genera [18, 201]. The most widely disseminated and clinically important determinant of MLS<sub>B</sub> resistance in Gram-positive pathogens is erm(C), which has been identified in at least 13 Gram-positive genera, and is the predominant determinant of MLS<sub>B</sub> resistance in staphylococci [18]. Even though Erm methylation can confer resistance to all MLS<sub>B</sub> antibiotics, expression of erm is most often inducible by common 14- and 15-membered macrolides, such as erythromycin or azithromycin. Thus, isolates expressing inducible MLS<sub>B</sub> resistance can remain susceptible to

Table 1. Acquired macrolide-lincosamide-streptogramin B resistance genes in Gram-positive human pathogens.

| Gene            | Organisms                                      | Inducers       | Confers resistance to | Leader peptide(s)<sup>*</sup>          | Ref.         |
|-----------------|------------------------------------------------|----------------|------------------------|-----------------------------------------|--------------|
| Inducible erm(A)| Staphylococcus aureus                          | M<sub>14/15</sub>| Uninduced Induced<sup>t</sup> | LP1MCTSIAVVEITLSHS                     | [20, 21, 108, 112] |
|                 | Streptococcus pyogenes                         |                | MLSK                   | LP2MGTFSVFINKVRYQPNON                  |              |
|                 | Streptococcus agalactiae                       |                |                        |                                         |              |
|                 | Enterococcus faecalis                         |                |                        |                                         |              |
|                 | Enterococcus faecium                          |                |                        |                                         |              |
| Constitutive erm(A)| S. aureus                                    | n/a           | MLSK                   | Deletions in the 5' UTR disrupt         | [30]         |
|                 | S. agalactiae                                 |                | n/a                    | attenuator structure                    |              |
|                 | Staphylococcus epidermidis                    |                |                        |                                         |              |
| Inducible erm(B)| S. pyogenes                                   | M<sub>14/15</sub>K | MLS                    | MLVFQMRYNDKTSTVLKQTKNSDYADK             | [20, 33, 112, 113] |
|                 | S. agalactiae                                 |                | MLSK                   |                                         |              |
|                 | E. faecium                                    |                |                        |                                         |              |
|                 | E. faecalis                                   |                |                        |                                         |              |
|                 | Streptococcus pneumoniae                      |                |                        |                                         |              |
|                 | Bacillus subtilis                             |                |                        |                                         |              |
|                 | Clostridium perfringens                       |                |                        |                                         |              |
| Constitutive erm(B)| S. pyogenes                                 | n/a           | MLSK<sup>t</sup>        | Deletions in the 5' UTR disrupt         | [20, 30, 112, 114] |
|                 | S. agalactiae                                 |                | n/a                    | attenuator structure                    |              |
|                 | S. pneumoniae                                 |                |                        |                                         |              |
| Inducible erm(C)| S. aureus                                     | M<sub>14/15</sub>| M<sub>14/15</sub>       | MLSK                                   | [31, 38]     |
|                 | B. subtilis                                   |                |                        | MGIFSIFVISTVHYQPNKK                    |              |
|                 | CoNS                                          |                |                        |                                         |              |
| Constitutive erm(C)| CoNS                                        | n/a           | MLSK                   | Deletions in the 5' UTR disrupt         | [31, 115]    |
|                 |                                               |                | n/a                    | attenuator structure                    |              |
| Inducible erm(K)| Bacillus licheniformis                        | M<sub>14/15</sub>| M<sub>14/15</sub>       | MLSK                                   | [34]         |
|                 |                                               |                |                        | MTHSMRLRFPTLNQ                          |              |
| Inducible mef(E)/mel | S. pyogenes                               | M<sub>14/15</sub>, M<sub>16</sub><sup>s</sup>, M<sub>14/15</sub> | AMPs           | MTASMRLR<sup>1</sup>                      | [20, 39, 40, 114] |
|                 | E. faecium                                    |                |                        |                                         |              |
|                 | S. pneumoniae                                 |                |                        |                                         |              |
|                 | C. perfringens                                |                |                        |                                         |              |
| Inducible msr(A)| S. aureus                                     | M<sub>14/15</sub>| M<sub>14/15</sub>       | LM<sub>16</sub>K                        | [31, 36, 38] |
|                 | S. epidermidis                                |                |                        | MTASMRLK<sup>1</sup>                     |              |

<sup>t</sup> Resistance conferred only after exposure to inducing antibiotics.  
<sup>s</sup> Ribosome stall sites indicated in bold.  
<sup>1</sup> Resistance 16-membered macrolides induce when possessing a monosaccharide at C-5. Disaccharide 16-membered macrolides do not induce.  
<sup>1</sup> Predicted.  
AMP: Antimicrobial peptide (i.e., LL-37); CoNS: Catalase-negative staphylococci; K: Ketolides; L: Lincomycin; M: All macrolides; M<sub>14/15</sub>: 14- and 15-membered macrolides; M<sub>16</sub>: 16-membered macrolides; n/a: Not applicable; S: Streptogramin B; UTR: Untranslated region.

16-membered MLS<sub>B</sub> antibiotics unless induced by other compounds. Expression of *erm(C)* is the best-studied example of inducible MLS<sub>B</sub> resistance and will be discussed as the model for regulation of inducible *erm* expression.

**Induction by translational attenuation: the *erm(C)* paradigm**

Resistance in Gram-positive isolates carrying *erm(C)* can be induced by common 14- and 15-membered macrolides. The expression of *erm(C)* is controlled by translational attenuation and is dependent upon the structure of the 5′ leader sequence of the *erm(C)* mRNA [22,23]. The leader sequence contains a small open reading frame that encodes a 19 amino acid leader peptide Erm(C)L (MGIFSIFVISTVHYQPNKK), which is separated from the structural gene by a 60-nucleotide spacer region (FIGURE 2A). The spacer region contains four G/C-rich inverted repeats (IR1–IR4) that are involved in folding of the leader sequence into alternate conformations during inducing and noninducing conditions (FIGURE 2A). In the absence of inducer, the energetically favorable inactive conformation is formed by annealing of IR1 with IR2 (forming stem-loop 1 + 2) and IR3 with IR4 (stem-loop 3 + 4). Stem-loop 3 + 4 sequesters the *erm(C)* ribosomal binding site (RBS<sub>2</sub>) and blocks initiation of *erm(C)* translation (FIGURE 2A). The active conformation forms when stem-loop 1 + 2 is disrupted allowing annealing of IR2 and IR3 (stem-loop 2 + 3), which frees RBS<sub>2</sub> and activates *erm(C)* translation (FIGURE 2A). The formation of the active or inactive conformation is determined by the translational status of the ribosome on the *erm(C)*L leader cistron. In the absence of inducer, unimpeded ribosomes unwind stem-loop 1 + 2, translate *erm(C)L* and dissociate from the transcript. This allows refolding of the stem-loop and the inactive conformation [24]. However, when an inducing molecule is bound in the NPET, the ribosomal complex stalls on *erm(C)*L, blocking reformation of stem-loop 1 + 2 and favoring formation of the active conformation, resulting in translation of the resistance determinant.

Ribosomal stalling on the *erm(C)*L leader peptide is dependent upon complex molecular interactions of ribosomal nucleotides and proteins with the nascent peptide as it travels the length of the ribosome through the NPET [25]. Interactions of the nascent leader peptide carboxyl terminus with nucleotides near the peptidyl transferase center dictate the position of ribosomal stalling in the presence of an inducer. In the presence of erythromycin, ribosomes stall on the amino acid sequence IFVI, the 6–9 positions of the nascent Erm(C)L peptide, and specifically on the leucine codon in the ninth position due to a failure to catalyze peptide bond formation between the leucine and the serine encoded by the tenth codon [26]. The peptidyl tRNA carrying the nine amino acid peptide MGIFSIFVI remains tightly bound to the ribosome and prevents progression of the ribosome. Sequence-independent interactions of the Erm(C)L amino terminus and ribosomal proteins approximately 8–12 amino acids from the PTC are indispensable for ribosome stalling, demonstrating the importance of the length of the leader peptide [24–26].

The ability of different MLS<sub>B</sub> drugs to induce *erm* expression is due to the structural features of the compound that influence ribosome activity, but the sequence of the *erm(C)* leader peptide is also involved in differentiating inducers and noninducers. As stated above, certain macrolides are capable of inducing MLS<sub>B</sub> resistance while others are not. Lincosamides and streptogramin B generally are not inducers. Common features of *erm*-inducing macrolides include, 14- or 15-membered macrolide rings, cladinose at the C-3 position of the lactone ring and a monosaccharide at C-5. Noninducers typically have 16-membered lactone rings, no C-3 cladinose and a disaccharide at C-5. Ketolides have ring size and C-5 monosaccharide of inducers, but no cladinose and have generally been reported as noninducers suggesting that presence of cladinose is the key structural feature required for induction. However, reports of ketolides failing to induce were based on phenotypic observations, that is their antimicrobial activity, which could reflect their excellent antimicrobial activity even in the presence of ribosomal methylation. Indeed, when measuring methylase activity or *erm(C)* expression as direct indicators for induction, several ketolides including telithromycin and cethromycin were shown to be inducers [27].

Interestingly, mutational analysis of the *erm(C)* leader peptide was able to differentiate between induction by ketolides and cladinose-containing macrolides [27] Deletion of the codons for amino acids 2–4, which shortened the peptide without disturbing the stall sequence IFVI, abolished the induction by erythromycin and its derivatives. However, this had no effect on induction by telithromycin, suggesting that ketolide induction may be by a different mechanism than cladinose-containing macrolides [27]. The differential ability of macrolides to induce

Review Chancey, Zähner & Stephens

(A) Translational attenuation of $erm(C)$

mRNA
RBS₁ $erm(C)$L
+1 IR1 IR2 IR3 IR4
Inactive conformation
— inducer
Active conformation
+ inducer
Stem-loops
Ribosome
unwinds
loop 1 + 2
RBS₁
MGIFSIFVISTVHØ
Erm(C)L
1 + 2
3 + 4
RBS₂
sequestered
Stem-loop
2 + 3
RBS₁
MNEKNIKHSSQNFØ
Erm(C)
+1
Stalled ribosome

(B) Transcriptional attenuation of $erm(K)$

DNA
RBS₁ $erm(K)$L
$P_{ermK}$
IR5 IR6 IR7 IR8
Inactive conformation
— inducer
Active conformation
+ inducer
Terminator-1 Terminator-2
Ribosome
unwinds
loop 1 + 2
RBS₁
MTTHSMRLRFPVØ
Erm(K)L
1
23
4
5
6
7
8
UUUUU
UUUUU
Transcription
terminated
RBS₁
MKKKNNHKTRGKKLR
Erm(K)
+1
Stalled ribosome
Transcription
completed

Figure 2. Translational and transcriptional attenuation of $erm$ gene expression.

(A) Translational attenuation of $erm(C)$. In the absence of inducer, ribosomes translate the leader peptide (Erm(C)L) and dissociates allowing reformation of stem-loop 1 + 2. Stem-loop 3 + 4 (red) forms in the absence of inducer and blocks ribosome binding to the $erm(C)$ RBS₂ thus preventing translation. In the presence of inducer, the ribosome stalls on $erm(C)$L disrupting stem-loop 1 + 2 and through formation of 2 + 3, prevents formation of 3 + 4 and promotes translation of the Erm(C).

(B) Transcriptional attenuation of $erm(K)$. In the absence of inducer, four stem-loop structures, including two Rho-independent transcriptional terminators (red) that prematurely terminate the $erm(K)$ transcript. In the presence of inducer, refolding of the transcript disrupts both terminators allowing completion of the $erm(K)$ transcript and subsequent translation of Erm(K).

RBS: Ribosomal binding site.

the expression of $erm(C)$ is likely related to the subtle differences in their mode of action. 14- and 15-membered macrolides and ketolides block the passage of nascent peptides through the NPET resulting in premature peptidyl-tRNA dissociation (i.e., drop off) after incorporation of approximately six to eight amino acids [28]. 14- and 15-membered and 16-membered macrolides bind in a similar location in the NPET but the C-5 disaccharide of 16-membered macrolides extends farther into the NPET and makes direct contact with the PTC, disrupting peptide bond formation and forcing peptidyl-tRNA drop off after only two to four amino acids, preventing ribosome stalling and formation of the active $erm(C)$ mRNA conformation [28,29]. Streptococcal isolates constitutively expressing $erm(C)$ genes have been reported and in all cases contain mutations that destabilize attenuator structures and lead to unregulated translation of $erm(C)$ repression of translation in the absence of inducer [30,31].

Variations of translational & transcriptional attenuation of other ribosome methylases

The other major Gram-positive methylases $erm(A)$ and $erm(B)$ are also regulated by translational attenuation. The $erm(A)$ gene is preceded by a unique attenuator structure encoding two leader peptides. In the $S.$ aureus $erm(A)$ allele, the peptides are the 15 amino acid $erm(A)$L1 (MCTSIAVVEITLSHS) and the 19 amino acid $erm(A)$L2 (MGTFSIFVINKVRYQPNON) [32]. Induction of $erm(A)$ occurs due to sequential

formation of stalled ribosome complexes on the  
$erm(A)$L1 and $erm(A)$L2 cistrons in the presence of an inducing antibiotic [32]. Stalling at  
$erm(A)$L1 induces initiation of translation of  
$erm(A)$L2. A second stalling event on $erm(A)$  
L2 then induces translation $erm(A)$ structural  
gene [32]. The sequence encoded by $erm(A)$L2  
is highly conserved with that of $erm(C)$L and  
contains the $erm(C)$ stall sequence IFVI. Like  
with $erm(C)$, stalling occurs on the leucine  
codon in the ninth position of $erm(A)$L2 [32].  
The $erm(A)$L1 open reading frame is not as conserved, and contains a different stall sequence  
(SIAVV) (TABLE 1).

The translational attenuator of $erm(B)$ allele  
is similar to $erm(C)$ with a single leader peptide coding sequence. However, the Erm(B)L  
sequence (MLVFQMRYNDKTSTVLKQTK NSDYADK) bears no resemblance to the leader  
peptides of $erm(A)$ or $erm(C)$. The Erm(B)L stall  
sequence is MRYND with stalling occurring on  
the aspartic acid codon in the tenth position [33].  
Constitutive expression of $erm(A)$ and $erm(B)$  
isolates has been observed in staphylococci,  
streptococci and enterococci and involves mutations that disrupt formation of the attenuator  
structure (TABLE 1) [29,33].

Erm(K) is a member of the Erm(D) class found in Bacillus species. The regulation of  
$erm(K)$ is unique for $erm$ genes in that it is controlled by transcriptional attenuation [34]. The  
$erm(K)$ transcript is prematurely terminated in  
the absence of inducer due to two rho-independent transcriptional attenuators in the inactive  
conformation of its mRNA (FIGURE 2B). Like the  
translational terminators, activation requires pausing of the ribosomal complex on a leader  
peptide encoded on the 5′-end of the transcript, leading to formation of the active conformation  
[34]. The $erm(K)$ leader peptide is 14 amino acids  
(MTHSMRLRTNR) and the pause sequence is  
MRLR (TABLE 1) [34].

### Induction of macrolide efflux

Active drug efflux is another major mechanism of inducible MLS$_{B}$ resistance in Gram-positive pathogens. Inducible efflux-mediated macrolide resistance was first described in Staphylococcus epidermidis and is conferred by $msr(A)$ encoding an incomplete ABC transporter [35]. $msr(A)$ has been identified in other staphylococci, including S. aureus and has recently been reported in Streptococcus, Enterococcus and Corynebacterium species (TABLE 1) [36]. $msr(A)$ is inducible by 14- and 15-membered macrolides and confers resistance to 14- and 15-membered macrolides and streptogramins, but not 16-membered macrolides or lincosamides [37]. Msr(A)-mediated resistance to telithromycin is dependent upon prior induction of $msr(A)$ by erythromycin indicating that telithromycin is not an inducer but it is a substrate for $msr(A)$-mediated efflux (TABLE 1) [38]. The mechanism of induction of $msr(A)$ has not been confirmed, but analyses of the DNA sequences immediately upstream reveal similarities to MLS$_{B}$ attenuators [CHANCEY ST, UNPUBLISHED DATA]. The putative leader sequence $msr(A)L$ is predicted to encode a small leader peptide (MTAMRLR) that is almost identical to that of $erm(K)$ [24].

In S. pneumoniae, macrolide efflux is encoded by $mef(E)$ and the $msr(A)$ homolog $mel$, which constitute an inducible operon carried on the mobile macrolide efflux genetic assembly (MEGA) (FIGURE 1A) [39,40]. $mef(E)$ is predicted to encode a major facilitator family efflux pump and $mel$ is a homolog of the incomplete ABC transporter $msr(A)$. $mef(E)/mel$ confers pneumococcal resistance to 14- and 15-membered macrolides and a modest increase in resistance to telithromycin [20]. Induction of $mefE/mel$ is controlled at the level of transcription [39]. A putative leader peptide is encoded 327-bp upstream of $mef(E)$ and is almost identical to that of $msr(A)$ in Staphylococcus including a putative leader peptide (MTAMRLR), which is identical to $msr(A)$ and similar to the transcriptionally attenuated $erm(K)$ (TABLE 1) [34]. These data suggest that both $msr(A)$ and $mef(E)/mel$ are induced by transcriptional attenuation. Homologous systems include $mef(A)/msr(D)$ primarily found in S. pyogenes and, $mef(I)/msr(D)$ in group C streptococci and S. pneumoniae (TABLE 1) [41]. The $mef(A)/msr(D)$ operon is also inducible in S. pyogenes, but induction of $mef(I)/msr(D)$ has not been reported [42]. The $msr(A)$ homolog $msrC$ confers a similar inducible resistance phenotype in Enterococcus faecium and, although the putative leader peptide (MTASMKLRFELLNNN) is longer, a transcriptional attenuator mechanism is suggested as well [43].

In addition to being induced by 14- and 15-membered macrolides, $mef(E)/mel$ is induced by telithromycin and the 16-membered macrolides tilmicosin and rosmicin [39]. Telithromycin, tilmicosin and rosmicin have a monosaccharide at C5 but no cladinose, indicating that the cladinose is not required for induction of $mef(E)/mel$. Furthermore, C5 disaccharide-containing 16-membered macrolides failed to induce $mef(E)$ expression consistent with the model proposed for induction

of *erm(C)* that states that the C5 disaccharide prevents ribosome stalling causing termination of the leader peptide synthesis prior to the stall sequence incorporation. Interestingly, *mef(E)* on the MEGA element is also induced by LL-37, a cationic antimicrobial peptide produced in human macrophages [40]. This is a significant observation because it suggests that *mef(E)* may be induced at the site of infections and therefore resistance to macrolides could be higher than predicted by *in vitro* MIC analyses [40]. LL-37 is not believed to bind ribosomes and likely induces *mef(E)* transcription through an alternate pathway, possibly through a general cell stress response [40].

**Other ribosome-mediated induction mechanisms**

The resistance of Gram-positive bacteria to other antibiotics that act upon the bacterial ribosome (e.g., chloramphenicol and tetracycline) is also inducible and controlled by transcriptional or translational attenuation. Inducible chloramphenicol resistance is similar to *erm* gene regulation and involves chloramphenicol-induced ribosomal stalling during leader peptide synthesis to alleviate translational attenuation. Inducible chloramphenicol resistance due to drug inactivation via expression of the chloramphenicol acetyl transferase has not been a subject of intensive investigation in recent years and will not be covered further here (for review see [24]).

Two distinct mechanisms of inducible tetracycline resistance have been reported in Gram-positive bacteria: translational attenuation of *tet(L)* and transcriptional attenuation of *tet(M)* [44,45]. Regulation of the plasmid-encoded tetracycline efflux gene *tet(L)* is an interesting variation of leader peptide-dependent translational attenuation. In contrast to the regulation of *erm(C)*, the formation of the active conformation of *tet(L)* mRNA is not dependent upon ribosomal stalling on the leader peptide, but on inhibition of translation of the leader peptide by tetracycline. The distinction between the two mechanisms is evident by mutations that prevent expression of the respective leader peptides. Mutants that do not express the leader peptide of *erm(C)* cannot be induced whereas similar mutation in the *tet(L)* leader sequence system results in constitutively expressed *tet(L)* [44].

The conjugative transposon-associated *tet(M)* gene confers tetracycline resistance by ribosomal protection and is regulated by a transcriptional attenuation mechanism similar to *erm(K)*. Transcription of *tet(M)* is attenuated in the absence of tetracycline by rho-independent terminators, but tetracycline-induced ribosomal stalling resolves these structures and allows completion of transcription. A significant aspect of inducible tetracycline resistance in Gram-positive bacteria is the regulatory link between *tet(M)* and the mobility of Tn916. Tetracycline induction of *tet(M)* leads to a regulatory cascade that ultimately activates expression of the genes involved in excision and transfer of the conjugative transposon. Because Tn916-like elements are carriers of genes encoding resistance to antibiotics in addition to tetracycline, tetracyclines may facilitate the dissemination of antimicrobial resistance in Gram-positive pathogens. For example, a pneumococcal clone, ST320, containing both the *mef(E)/mel* operon and the *erm(B)* methylase has been responsible for a global epidemic of invasive pneumococcal infections [46]. The *mef*-carrying element MEGA and an *erm(B)*-element have inserted into separate loci of Tn916 to generate Tn2010 (FIGURE 1A).

---

**Induction of antibiotic resistance by cell surface-mediated drug sensing**

Many important antibiotics used in the treatment of human Gram-positive pathogens interfere with bacterial cell wall synthesis. These antibiotics include but are not limited to the β-lactams (e.g., penicillin, oxacillin, cefoxitin), glycopeptides (vancomycin) and the polypeptides (bacitracin). Apart from intrinsic mechanisms that respond to stress and damage of the cell envelope, with sometimes the antibiotic itself serving as the signal (see for review [47]), Gram-positive bacteria have evolved three substantially different inducible and transferable molecular mechanisms to respond to cell wall targeting antibiotics (Figure 3).

---

**Inducible vancomycin resistance**

**Mechanism of Van-type vancomycin resistance**

Vancomycin and other glycopeptide antibiotics, such as teicoplanin, inhibit peptidoglycan cross-linking by binding to the D-alanyl-D-alanine (D-Ala-D-Ala) dipeptide terminus of lipid II, the major precursor in peptidoglycan biosynthesis. In streptomycetes, enterococci and (so far) rarely in staphylococci, resistance to glycopeptide antibiotics is achieved by incorporation of modified penta-peptide termini, such as D-alanyl-D-lactate (D-Ala-D-Lac) or D-alanyl-D-serine (D-Ala-D-Ser), instead of D-Ala-D-Ala, into lipid II. Glycopeptide antibiotics bind with less affinity to these modified termini.

Acquired inducible antimicrobial resistance in Gram-positive bacteria

**Review**

![Image Description](#)

Figure 3. Induction of antibiotic resistance via signal transduction systems. (A) Vancomycin, (B) β-lactams and (C) bacitracin. See text for details.
P: Phosphate; TCS: Two-component system.

The genetic basis for acquired vancomycin resistance is now well understood. High-level *van*-resistance in enterococci is most frequently associated with transposable elements [8], either integrated in the chromosome or located on plasmids (FIGURE 1 & TABLE 2). The prototype, the *vanA* system, is encoded by five genes: *vanR*, *vanS*, *vanH*, *vanA* and *vanX* (FIGURE 4A); *vanR* and *vanS* encode a typical two-component signal transducing system (FIGURE 3A) composed of the sensor kinase (histidine kinase), VanS, which is capable of autophosphorylation in response to an environmental signal, and the response regulator, VanR, that interacts with the phosphorylated histidine kinase [48]. VanS senses the presence of vancomycin, and Koteva *et al.* have shown in an elegant experiment based on VanS of *Streptomyces coelicolor* that vancomycin is itself a ligand for VanS [49]. Presence of the inducing compound causes phosphotransfer from VanS to the response regulator VanR, which subsequently activates transcription of the *vanHAX* genes. The *vanHAX* genes are responsible for the synthesis of the modified precursor peptides and the removal of the native D-Ala-D-Ala precursor (FIGURE 3A). The dehydrogenase VanH reduces pyruvate to D-Lac, the ligase VanA forms the ester bond between D-Ala and D-Lac, and the dipeptidase VanX cleaves the D-Ala-D-Ala bond, thereby eliminating the target of glycopeptides antibiotics. *vanY* encodes a D,D-carboxypeptidase that may contribute as an accessory protein to elevated glycopeptide resistance, and VanZ confers low-level teicoplanin resistance by a yet unknown mechanisms [6].

Several variants of *van* operons have been identified in enterococci and are referred to as different Van-types (TABLE 2; review see [50]). The most common are the types VanA and VanB. VanB differs from VanA in that it is only induced by vancomycin, not by teicoplanin. The VanS proteins of the VanA (VanS<sub>A</sub>) and VanB (VanS<sub>B</sub>) differ in their N-terminal sensor domains, which results in differential induction by vancomycin and teicoplanin, respectively [51,52]. The other identified Van-types – VanD, VanC, VanE, VanG, VanL [53], VanM [54] and VanN [4] – have been found far less frequently than VanA and VanB (TABLE 2). The VanM- and VanD-types are similar to VanA and VanB in regard that they provide D-Ala-D-Lac penta-peptide precursors. However, VanD-type resistance is constitutively expressed due to mutations in VanS or VanR [55] and in addition, mutations in the chromosome-encoded *ddl*, which encodes D-Ala-D-Ala ligase, result in the absence of the D-Ala-D-Ala precursor. Van-types VanC, VanE, VanG, VanL and VanN encode proteins that synthesize D-Ala-D-Ser precursors instead of D-Ala-D-Lac and express a serine racemase to convert L-Ser to D-Ser (FIGURE 4B). Vancomycin has a higher affinity for binding D-Ala-D-Ser than for D-Ala-D-Lac; therefore, VanC, VanE, VanG, VanL and VanN confer only low-level resistance to vancomycin, and in addition, the bacteria remain sensitive to teicoplanin. The VanE-, VanG-, VanL- and VanN-types are acquired and inducible, whereas the VanC-type is intrinsically present in the chromosomes of *Enterococcus gallinarum* and *Enterococcus casseliflavus* and is constitutively expressed. In some VanE-type strains, inducible vancomycin resistance was observed despite the fact that mutations rendered the histidine kinase VanS inactive. This finding led to the suggestion that VanR may be activated by other histidine-kinases [56]. This cross-talk phenomenon has been observed in an experimental setting between the histidine kinase PhoB and VanR in the absence of VanS (in recombinant *E. coli* mutants) [57]. However, in the presence of VanS, while not activated by a

Table 2. Characteristics of acquired vancomycin resistance systems.

| System | Resistance level | Vancomycin MIC [µg/ml] | Teicoplanin MIC [µg/ml] | Conjugation Mobility | Expression | Location | Modification Organisms¹ |
|--------|------------------|-----------------------|-------------------------|----------------------|-----------|----------|------------------------|
| VanA   | High             | 64–1000               | 16–512                  | Positive             | Inducible | Tn1546   | Plasmid or chromosome | Enterococcus faecium, E. faecalis, E. durans, E. gallinarum, E. hirae, E. casseliflavus, E. raffinosus, E. avium, E. mundtii, Streptococcus bovis, S. gallolyticus, Staphylococcus aureus (MRSA), Bacillus circulans et al. |
| VanB   | Variable         | 4–1000                | 0.5–1                   | Positive             | Inducible | Tn1547/Tn1549/Tn5382 | Plasmid or chromosome | E. faecium, E. faecalis, E. durans, E. gallinarium, S. bovis, S. gallolyticus, Clostridium bolteae |
| VanM   | High             | >256                  | >96                     | Unknown              | Inducible | —        | Plasmid                | E. faecium |
| VanD   | Moderate         | 64–128                | 4–64                    | Unknown              | Constitutive | —       | Chromosome             | E. raffinosus |
| VanE   | Low              | 8–32                  | 0.5                     | Negative             | Variable  | —        | Chromosome             | E. faecalis |
| VanG   | Low              | 16                    | 0.5                     | Unknown              | Inducible | —        | Chromosome             | E. faecalis |
| VanL   | Low              | 8                      | n.d.                    | Negative             | Variable  | —        | Chromosome             | E. faecalis |
| VanN   | Low              | 16                    | 0.5                     | Positive             | Unknown   | —        | Chromosome             | D-Ala-, D-Ser |
| VanC   | Low              | 2–32                  | 0.5–1                   | Negative             | Constitutive | —       | Chromosome             | D-Ala-, D-Ser |

¹Limited to clinically relevant species.
D-Ala: D-alanine; D-Lac: D-lactate; D-Ser: D-serine; MRSA: Methicillin-resistant Staphylococcus aureus; n.d.: Not determined.
Adapted from [50].

glycopeptide, VanS acts as a phosphatase maintaining VanR in an unphosphorylated state [58], thereby preventing cross-phosphorylation by other histidine kinases. Hence, certain mutations in VanS may affect either the kinase or phosphatase activity of VanS and thereby modulate the output of the Van system.

In clinical isolates, Van-types from different geographical sources displayed a great variety of point mutations, deletions of regulatory and accessory genes and insertions (mainly IS elements), leading to modified and fragmented operons with different expression patterns. The modifications and rearrangements have best been studied for the transposon-associated VanA- and VanB-types [59–63]. Surprisingly, a partially vancomycin-dependent vancomycin-resistant *S. aureus* (VRSA) was identified [64], caused by a mutation in the core genome-encoded D-Ala-D-Ala ligase, which rendered the strain dependent on the *van* gene cluster to provide D-Ala-D-Lac precursor for cell wall biosynthesis. A gene cluster similar to the VanA-type has been described in the Gram-positive insect pathogen *Paenibacillus popilliae*, which confers inducible vancomycin resistance and has been named VanF [65].

### Distribution of Van-type-mediated vancomycin resistance

Van-type-mediated vancomycin resistance has been encountered in many enterococcal species (vancomycin-resistant enterococci [VRE]), which are a major source of van-resistance elements. In addition, *van* gene clusters have been identified in *S. aureus* and also many coagulase-negative staphylococci, a few streptococcal species, such as *S. bovis* [66], and in many environmental nonenterococcal isolates, such as those found in the human bowel [67]. Thus far, Van-type vancomycin resistance has not been identified in clinical isolates of the major human pathogens group A and B streptococci and *S. pneumoniae*. It is commonly accepted that the *van* gene clusters in the low-GC Gram-positive bacteria, such as enterococci, originated from the high-GC Gram-positive *Streptomyces* species, which are the producers of glycopeptides, and therefore contain defensive systems for self-protection [58]. The characterization of the *van* gene pool, the host range and mechanisms affecting their distribution are an area of ongoing investigation [68].

In 1992 the first conjugal transfer of a vancomycin resistance determinant (VanA-type) from VRE to MRSA was described [69], followed by the first detection of a clinical *S. aureus* strain

with high-level (VanA-type) VRSA in 2002 [70]. Since then several VRSA strains have been identified, all of them are in MRSA strains that are hypothesized to have acquired Tn1546 in the course of a coinfection with VRE. This was confirmed in the first identified VRSA strain [71]. This strain harbored a 60-kb multiresistance-encoding conjugative plasmid that included Tn1546 (VanA) [72].

All Van-types that confer high-level vancomycin resistance are inducible (TABLE 2), which led to the suspicion that expression of the *van* gene cluster may come with a fitness cost. Foucault et al. addressed this question by introducing inducible and constitutively expressed versions of the *vanB* operon in an isogenic background [73]. The authors found that induction and constitutive expression of the *van* gene cluster had adverse effects on the ability of the strain to colonize and disseminate in mice. They concluded that the tight regulation of resistance expression drastically reduces the biological cost associated with *van*-mediated vancomycin resistance and represents an efficient evolutionary pathway for resistance determinants to become selectively neutral.

Recent research on vancomycin resistance has been driven by genomics, providing access to an extensive collection of genome and plasmid sequences. Genomics has provided insight into the evolution of the different *van* gene clusters in VRE originating from animals and human sources that links to data on glycopeptide usage in animals (as growth promoters and antibiotics) and in humans. These studies are complemented by studies that address the mechanisms, restrictions and dynamics of transmission and maintenance of resistance-conferring elements [74]. The *van* operon-conferring plasmids and transposons frequently encode additional genetic elements, such as other antibiotic resistance determinants, for example, *erm(B)* or *tet(M)*, and plasmid maintenance systems (toxin–antitoxin systems [75]), which may help preserve the elements in a defined population in the absence of vancomycin. The toxin–antitoxin systems on the other hand may limit host range [76].

Two areas of considerable research interest are the search for (lipidated-)glycopeptides with altered induction profiles, such as telavancin, which, like teicoplanin, induces only the *vanA* gene cluster, but to lower resistance levels than by vancomycin and teicoplanin [77]; and the search for compounds such as vancomycin derivatives that recognize D-Ala-D-Ala and also D-Ala-D-Lac as substrates [78].


### Figure Description:
#### Figure 4: Organization of transferable *van* gene clusters.
Produced precursors are (A) D-Ala-D-Lac and (B) D-Ala-D-Ser. The Streptomyces coelicolor *van* gene cluster (C) is shown for comparison.

#### Panel A:
- **vanA**: vanR → vanS → vanH → vanA → vanX → vanY → vanZ
  - vanR: Regulator
  - vanS: Sensor
  - vanH: Dehydrogenase
  - vanA: Ligase
  - vanX: D,D-dipeptidase
  - vanY: D,D-carboxypeptidase
  - vanZ: Unknown

- **vanB**: vanR → vanS → vanY → vanW → vanH → vanB → vanX
  - vanR: Regulator
  - vanS: Sensor
  - vanY: Unknown
  - vanW: Unknown
  - vanH: Unknown
  - vanB: Unknown
  - vanX: Unknown

- **vanD**: vanR → vanS → vanY → vanH → vanD → vanX
  - vanR: Regulator
  - vanS: Sensor
  - vanY: Unknown
  - vanH: Unknown
  - vanD: Unknown
  - vanX: Unknown

- **vanM**: vanR → vanS → vanY → vanH → vanM → vanX
  - vanR: Regulator
  - vanS: Sensor
  - vanY: Unknown
  - vanH: Unknown
  - vanM: Unknown
  - vanX: Unknown

#### Panel B:
- **vanE**: vanE → vanXY → vanT → vanR → vanS
  - vanE: Unknown
  - vanXY: Unknown
  - vanT: Serine-racemase
  - vanR: Unknown
  - vanS: Unknown

- **vanG**: vanU → vanR → vanS → vanY → vanW → vanXY → vanG → vanT
  - vanU: Unknown
  - vanR: Unknown
  - vanS: Unknown
  - vanY: Unknown
  - vanW: Unknown
  - vanXY: Unknown
  - vanG: Unknown
  - vanT: Unknown

- **vanL**: vanL → vanXY → vanT → vanTR → vanR → vanS
  - vanL: Unknown
  - vanXY: Unknown
  - vanT: Racemase
  - vanTR: Membrane-binding
  - vanR: Unknown
  - vanS: Unknown

- **vanN**: vanN → vanXY → vanT → vanR → vanS
  - vanN: Unknown
  - vanXY: Unknown
  - vanT: Unknown
  - vanR: Unknown
  - vanS: Unknown

#### Panel C:
- **Actinomycetes**: vanS ← vanR → vanJ → vanK → vanH → vanA → vanX
  - vanS: Regulatory proteins
  - vanR: Regulatory proteins
  - vanJ: Unknown
  - vanK: Accessory proteins
  - vanH: Peptidoglycan cross-linking
  - vanA: Resistance-conferring proteins
  - vanX: Resistance-conferring proteins

#### Legend:
- Red box: Regulatory proteins
- Blue box: Accessory proteins
- Black box: Resistance-conferring proteins
- Asterisk (*): Nonfunctional proteins

### Inducible β-lactam resistance
**Induction through BlaR1/I & MecR1/I**

Resistance to β-lactam antibiotics is a major problem in the treatment of Gram-positive pathogens. The two acquired, inducible β-lactam resistance mechanisms are the β-lactamases (BlaZ and its homologs) and, in S. aureus, the inducible expression of a low-affinity penicillin-binding protein (PBP), PBP2a (also named PBP2′) encoded by *mecA*. β-lactam antibiotics disrupt peptidoglycan cross-linking by binding PBPs in the cell wall; however, PBP2a allows S. aureus to maintain cell wall biosynthesis in the presence of β-lactams. PBP2a confers resistance

to the β-lactam methicillin, and resistant strains  
are therefore referred to as MRSA. The induction mechanisms of β-lactamase and PBP2a  
share common features (FIGURE 5). In the absence  
of a β-lactam, the resistance-encoding genes  
mecA and blaZ are controlled by the repressors  
MecI and BlaI, respectively. When present,  
β-lactams bind to the transmembrane signal  
transducers MecR1 and BlaR1, which results in  
autocatalytic cleavage of the C-terminal domain  
of MecR1 and BlaR1, respectively, and release  
of a metalloprotease-containing cytoplasmic  
domain of the signal-transducing molecule.  
The metalloproteases initiate cleavage of the  
repressors MecI and BlaI, a process which may  
involve additional factors [79]. The cleavage  
of MecI and BlaI leads to transcription of the  
divergently transcribed regulatory genes mecI  
and blaI, respectively, as well as the structural  
genes mecA and blaZ and hence the production  
of PBP2a or the β-lactamase. For more details  
on the BlaR1/I and MecR1/I signal-transducing  
mechanisms see [80].

The MecR1 peptide is specific for MecI, as is  
the BlaR1 peptide for BlaI; however, the repressors MecI and BlaI act interchangeably (FIGURE 3A)  
[81], such that BlaI bound to the mec regulatory  
region can be inactivated by BlaR1 in the presence of a β-lactam. The induction kinetics of  
MecR1/I and BlaR1/I vary greatly, while BlaR1  
induces BlaZ as well as PBP2a synthesis within  
minutes, MecR1 takes hours to induce production of PBP2a [82,83], and this induction has been  
observed to sometimes not be rapid enough to prevent death [84]. Hence, presence of both resistance  
determinants may provide a fitness advantage.

MRSA strains with slow PBP2a induction  
often appear sensitive to oxacillin in routine  
MIC testing on agar plates. To account for this  
phenotype, cefoxitin, a potent inducer of mecA,  
which allows MRSA to grow readily, is widely  
used as a surrogate marker for detection of  
PBP2a-mediated β-lactam resistance. However,  
there is research to more readily and reliably  
identify mecA-containing S. aureus isolates in  
routine diagnostic (for review, see [85]).

### A bla operon

![Diagram](#)

### B mec region of SCCmec cassette

| Class | Type | Inducible | Other resistance genes on SCCmec cassette | Species |
|-------|------|-----------|--------------------------------------------|---------|
| A     | II, III, VIII | + | ermA (II + VII), aad9 (II + VIII), cadB/C (II + VIII), ble (II), ant (II), tet (III) | Staphylococcus aureus, Staphylococcus epidermidis |
| B     | I, IV, VI | c⁺ | aacA-aphD (IV) | S. aureus |
| C     | VII, X | c⁺ | – | S. aureus |
| D     | V, VII, XI, X | c⁺ | – | S. aureus |
| E     | XI | + | ars | Macrococcus caseolyticus, S. aureus |

---

**Figure 5. Organization of the bla operon and SCCmec cassettes.**

⁺ Constitutive mecA expression; however, in the presence of the bla regulatory elements Blal and BlaR1 mecA expression may be under regulatory control of these elements.

ermA ars: Arsenic resistance.

The classification is based on the information provided by the SCCmec website curated by the International Working Group on the Classification of Staphylococcal Cassette Chromosome Elements [202]. See text for further details.

# Acquired inducible antimicrobial resistance in Gram-positive bacteria

## Staphylococcal cassette chromosome mec

The *mecA* gene and the *mec* regulatory genes are encoded on the (SCC*mec*, an accessory genetic region of 21–67 kb (FIGURE 1D; for review see [86]). Five classes (A–E) of SCC*mec* elements have been defined based on differences in the *mec* region (FIGURE 5B), with some classes further divided into types based on differences in the *mec* element flanking regions (FIGURE 5B). SCC*mec* elements A–C are common in *S.* aureus (FIGURE 5B) [9]. Class A is the prototype complex, which contains *mecA*, and complete *mecR* and *mecI* regulatory genes, whereas in SCC*mec* classes B and C, *mecR1* is truncated by IS1272 or IS431. Insertion of the IS elements replaces *mecI*, which abolishes induction and renders *mecA* constitutively active. In class D, *mecR1* is deleted. Class E encodes the β-lactamase *BlaZ* upstream of the *mec* element (*mecA*, *mecR1* and *mecI*). A *mec* element containing *blaZ* was first identified in *Macrococcus caseolyticus* [87], and most recently in *S.* aureus [88], designated SCC*mec* XI (FIGURE 5B). This finding fostered the speculation that *M.* caseolyticus may represent the ancestral form of the *mec* complex in staphylococci and supported earlier speculation that the *mec* element evolved by integration of *mecA* into a *blaZ* locus [89].

Many MRSA strains contain SCC*mec* and a *blaZ* locus concomitantly, and while many isolates contain nonfunctional BlaR1/I or MecR1/I regulatory components (FIGURE 5), presumably one of the two regulatory systems remains functional [90]. In particular, *mecA* transcription often seems to be under the control of the *blaR1/I* genes only. In line with these observations, the presence of the *blaZ* locus has been shown to promote *mecA* acquisition and stabilization: plasmid-based *mecA* introduced in *S.* aureus strains with different *bla/mec* regulatory backgrounds was only maintained in the presence of either a *bla* or a *mecA* regulatory region [91]. Thus the requirement for regulation of regulatory-deficient SCC*mec* elements by BlaR1/I constitutes a mechanism of host restriction. Furthermore, Rosato *et al.* observed that constitutive expression of PBP2a or β-lactamase affected bacterial growth [90]. Two later studies verified this finding and described an inverse proportional relationship between oxacillin resistance level and the rate of growth, indicating substantial fitness costs associated with expression of this resistance determinant and underlining the role of tight regulation [92,93]. A more recent study by Milheirico *et al.* found evidence that clinical isolates with a *blaZ* locus and SCC*mec* are under selective pressure not only to maintain functional *bla* regulatory components, but also an intact β-lactamase [94]. Many clinical isolates of MRSA show, despite the presence of *mecA*, decreased oxacillin MICs presumably due to secondary mutations and additional regulatory events [83], which may indicate the need to balance the advantages of resistance with a cost in fitness [84].

The curiosity of vancomycin-dependent enterococcal strains has a counterpart in the oxacillin-dependent *S.* aureus strain 2884, which was selected in vitro on oxacillin and depends on oxacillin for growth [95]. This strain has a low expression level of native PBP2, due to a loss of function in the PBP2 regulatory two-component system VraSR and an SCC*mec* element (type I, FIGURE 5). Thus, the strain depends on oxacillin to allow expression of the now essential PBP2a functions.

### Distribution of *mecA* & *blaZ*

SCC*mec* elements have also been identified in *S.* epidermidis and other coagulase-negative staphylococci, and they are considered a potential reservoir of SCC*mec* elements that may facilitate the emergence of new MRSA clones [96,97]. SCC*mec* IV (FIGURE 5B) is the most common SCC*mec* element in *S.* epidermidis, detected in approximately 40% of methicillin-resistant *S.* epidermidis from humans. These strains belong to a wide variety of genetic backgrounds [98], and SCC*mec* IV is also common among coagulase-negative staphylococci from animals [99]. However, in SCC*mec* IV, the regulatory genes are disrupted and future work will need to determine if this affects transmission and/or a fitness cost [92,93], which may influence the evolutionary success of this element [15]. Recently, a variety of ‘atypical’ and ‘nontypeable’ SCC*mec* elements have been described in coagulase-negative staphylococci [100], indicating that identification, spread and evolution of SCC*mec* elements is still ongoing, and prompts the question of where these novel elements originated from and how they may contribute to the further spread of β-lactam resistance.

In contrast to the chromosome-located SCC*mec* complex, the *blaZ* locus has been found on Tn552 and Tn552-like transposons (Tn4002 and a Tn4001-like element; FIGURE 1C) located on plasmids or are bacteriophage-associated. Even SCC*mec* type V, which is flanked by two IS431 elements, rendering it potentially transposable, has never been found on a plasmid in clinical isolates [84]. Beyond staphylococci, *blaZ* homologs

have been identified in many bacilli [101]. In addition, genes with homology to *blaZ* and the regulatory components BlaR1/BlaI have been identified in many bacterial genomes, such as in the *Clostridium difficile* genome of strain A630 [102]; however, future studies will have to provide experimental evidence whether these genetic elements indeed confer the observed β-lactam resistance to the hosting bacterium.

**Antagonism between *mecA* & *vanA* in MRSA/VRSA strains**

A methicillin- and vancomycin-resistant strain of *S. aureus* was constructed by introducing the *vanA*-containing plasmid HIP11714 into the highly oxacillin-resistant strain COL, producing strain COLVA [103]. Although this strain was clearly able to express high-level resistance to oxacillin and vancomycin, the two antibiotics had strong and mutual antagonistic effects on the expression of the drug-resistance phenotypes. As little as 40 μg/ml oxacillin reduced the vancomycin MIC value of this strain from 512 to 12 μg/ml. Similarly, as little as 50 μg/ml vancomycin reduced the oxacillin MIC from 800 to 10 μg/ml. The authors concluded that the decrease in vancomycin resistance by oxacillin suggested that the low-affinity PBP2a, which is the only transpeptidase that remains active in this strain in the presence of oxacillin, is not able to utilize the depsipeptide cell wall precursors [103]. Subsequently the synergistic effect of oxacillin and vancomycin was successfully used to treat experimental endocarditis caused by VRSA [104], and was therefore suggested for treatment of infections with VRSA. However, treatment of COLVA with oxacillin or vancomycin resulted in both cases in a change from a homogenous to a heterogeneous resistance phenotype for the noninduced antibiotic [103]. This phenotype was also observed with clinical VRSA isolates and suggested that some bacteria may tolerate the combination of both antibiotics [105]. However, the *mecA/vanA* antagonism demonstrates the importance of a tightly regulated — that is, inducible — mechanism to alleviate the cost of resistance.

**Inducible bacitracin resistance in *E. faecalis***

Bacitracin is a mixture of high-molecular-weight polypeptides produced by *Bacillus licheniformis*. Bacitracin sequesters undecaprenol pyrophosphate (UPP), the lipid carrier in peptidoglycan synthesis. By preventing dephosphorylation of UPP, bacitracin depletes

the lipid carrier, causing disruption of cell wall biosynthesis. Bacitracin is widely used in topical applications in humans and extensively in animal foods, in particular in chicken production. A one-component signal transducing system (Figure 3C) was recently described in *E. faecalis* [106], conferring inducible bacitracin resistance. This system senses Zn²⁺-bacitracin via a membrane-bound transcriptional regulator, BcrR, which subsequently induces a bacitracin-specific ABC transporter, BcrAB, and a cotranscribed UPP, BcrD (Figure 6) [106]. BcrD alone has no significant effect on bacitracin resistance [106]. BcrR has an N-terminal helix-turn-helix DNA-binding motif and four C-terminal transmembrane domains, and has been shown to induce transcription of BcrABD in the absence of auxiliary proteins [107]. Thus, BcrR is a one-component signal-transducing system. Expression of *bcrABD* depends on induction by BcrR and is specific for bacitracin — that is, it is not induced by other cell wall active compounds such as vancomycin or indirectly by the accumulation of cell wall precursors [106].

The BcrRABD system is plasmid-encoded and has been shown to be transferable by conjugation between *E. faecalis* isolates [106]. Acquisition of the *bcrRABD* system results in an increase in bacitracin MIC from 32 μg/ml to ≥256 μg/ml in the recipient strain. While *bcrRABD* was first identified on a partial sequence of plasmid pJM01 [106], recent sequence released from enterococcal plasmid pTW9 (85 kb) shows the *bcrRABD* genes adjacent to the *van* genes of Tn1546 and *erm(B)* (Annotation according to GenBank, Acc. no. NC 014726) (Figure 6). Thus, on pTW9, the *bcrRABD* region is part of a multidrug resistance-encoding conjugative plasmid. A study analyzing the distribution of antimicrobial resistance determinants in enterococci isolated from broiler chickens identified *bcrR* in all (seven out of seven) *E. faecalis* isolates and in 92% (46 out of 50) *E. faecium* isolates [108]. Further studies are needed to assess the prevalence of this mechanism in enterococci isolated from human sources. However, the past experience with other resistance determinants, such as *vanA/B* and *erm(B)/(C)* has taught us that staphylococci and streptococci should be monitored for the emergence of this newly identified resistance mechanism.

---

**Conclusion**

Acquired, inducible antimicrobial resistance in Gram-positive bacteria represents a widening silent spring that has major diagnostic and

Acquired inducible antimicrobial resistance in Gram-positive bacteria

Enterococcus faecalis AR01/DGVS  
plasmid pJM01 (partial, 4.7 kb)

E. faecalis  
plasmid pTW9 (85 kb)

Figure 6. Schematic organization of the bcrABD region on Enterococcus faecalis AR01/DGVS plasmid pJM01 (GenBank AY496968) and on the bcrABD and additional resistance determinant-containing part of E. faecalis plasmid TW9 (GenBank NC_014726). Bacitracin resistance-associated genes are indicated in black, genes for other resistance determinants and their associated regulatory genes are indicated in dark gray, all other genes are indicated in light gray.

clinical implications including accentuating the limitations on available agents to treat Gram-positive infections. Acquired, inducible antibiotic resistance determinants belong to the accessory genome of a species and are horizontally acquired by transformation/recombination or through MGEs, including IS elements, ICEs or transposons, bacteriophages and plasmids. The two major, mechanistically very different, induction mechanisms are: ribosome-sensed induction, characteristic of MLSB and tetracycline resistance; and resistance by cell surface-associated sensing of the β-lactams (e.g., oxacillin), the glycopeptides (e.g., vancomycin) and now recently the polypeptide bacitracin. Although there is general conservation of inducibility because of fitness costs, variations and evolution of inducible agents and their regulatory systems is quite striking. An understanding of the molecular basis of induction, including the physiological role and the mechanisms of sensing and responding to antibiotics, allows the evaluation of induction mechanisms for potential new antimicrobial targets and aids in the design and discovery of noninducing antimicrobial agents.

Future perspective

Clinically, a major challenge is to reliably and cost-effectively identify infections caused by bacteria with acquired inducible antibiotic resistance [109,110]. Identification is important not only for the implementation of appropriate antibiotic regimens to treat these infections, but also for instituting infection control measures to prevent dissemination of these resistant isolates. While biologic tests are used by clinical microbiology laboratories to detect some inducible resistance, they can be difficult to interpret and are not always reliable. New diagnostic tools such as microarrays or genome sequencing to test for the presence of inducible resistance mechanisms are currently being tested. The development and widespread usage of multilocus sequence typing methods in the last 15 years has greatly improved the ability to assess the distribution and dissemination of inducible resistant and multidrug-resistant isolates. The list of Gram-positive species for which an MLST strategy has been developed continues to grow. However, whole-genome sequencing has revealed surprising genetic variability even between isolates of identical MLST sequence types and clonal complexes. The cost of sequencing the entire genome of a clinical isolate has, or will, become less than the cost of individually sequencing the alleles required to determine the MLST. As more clinical isolate genomes are sequenced, a higher-resolution picture of the evolutionary history of Gram-positive bacteria will become apparent, providing insights into mechanisms of horizontal transfer of inducible resistance mechanisms which could lead to new methodologies for preventing dissemination of the resistance determinants. For example, metagenomics, or sequencing of DNA isolates directly from clinical specimens will allow to address the ecology of microbe–microbe interactions *in vivo* and will improve our understanding

Review Chancey, Zähner & Stephens

of the mechanisms of inter- and intra-species that have recently been developed for clinical
horizontal gene flow. Identification of avenues identification of bacterial species are now being
and limitations to gene flow will allow imple- developed to test for resistance determinants
mentation of strategies to prevent dissemination. [111]. Whole transcriptome analyses will help
As resistance mechanisms become more preva- elucidate complex regulatory mechanisms and
lent, matrix-assisted laser desorption/ionization identify targets to prevent induction and targets
time of flight mass spectrometry technologies for novel antimicrobials.

---

### Executive summary

#### Antibiotic resistance mechanisms in Gram-positive pathogens
- Antibiotic-resistant Gram-positive pathogens are responsible for community-acquired and hospital-associated infections and are an increasing public health threat.
- Resistance of Gram-positive pathogens to antibiotics is conferred by three primary mechanisms: target modification, drug inactivation and antimicrobial efflux.
- Expression of resistance mechanisms can come with a cost to the fitness of the bacteria, but the costs can be mitigated by tight regulatory control, in the absence of antibiotics, of expression of the genes and/or proteins responsible for resistance in the absence of the antibiotic.
- Inducible antibiotic resistance is defined, for the purpose of this review, as reversibly altered expression of acquired antibiotic resistance determinants.

#### Clinical significance of inducible antibiotic resistance in Gram-positive pathogens
- The transient nature of inducible expression complicates clinical detection of inducible resistance by phenotypic susceptibility assays and can lead to treatment failures due to failures to recognize resistance.
- Because of the reduced fitness costs, inducible resistance determinants disseminate more readily and may persist longer in bacterial populations than constitutively expressed resistance determinants.

#### Dissemination of inducible resistance determinants
- Inducible resistance determinants are widely disseminated in Gram-positive pathogens through transformation/recombination and in association with mobile genetic elements such as transposons, plasmids and bacteriophages.

#### Inducible resistance to ribosome-targeting antibiotics
- Translation-inhibiting antibiotics such as tetracycline, chloramphenicol and the macrolides–lincomamides–streptogramin B antibiotics bind ribosomes and alter ribosomal activity.
- Acquired inducible resistance of Gram-positive bacteria by ribosomal binding antibiotics occurs primarily by target modification through the activity of Erm methylases or by active drug efflux.
- Expression of *erm* or *efflux* genes is attenuated in the absence of inducing antibiotics by transcript secondary structures that block translation or force premature termination of transcription. Binding of inducers promotes ribosomal stalling and refolding of the transcript into an active conformation leading to resistance expression.
- Inducible efflux-mediated resistance to macrolides is conferred by *mef* and *msr(A)* genes, which are likely controlled by transcriptional attenuation.

#### Inducible resistance to cell wall-targeting antibiotics
- Many important antibiotics used in the treatment of Gram-positive pathogens interfere with bacterial cell wall synthesis, including the β-lactams and glycopeptides (vancomycin).

#### Inducible β-lactam resistance
- The two acquired, inducible β-lactam-resistance mechanisms are the β-lactamases (BlaZ and its homologs) and the inducible expression of a low-affinity penicillin-binding protein PBP2a (also named PBP2′) encoded by *mecA* in *Staphylococcus aureus*.
- The resistance-encoding genes *mecA* and *blaZ* are repressed in the absence of a β-lactam, controlled by the repressors MecI and BlaI. Cleavage of the repressors is mediated by the signal transducing molecules MecR1 and BlaR1 upon binding to a β-lactam.

#### Inducible Van-type vancomycin resistance
- Vancomycin and other glycopeptide antibiotics inhibit peptidoglycan cross-linking by binding to precursors in peptidoglycan biosynthesis. Resistance results from modification of these precursors resulting in less affinity to the antibiotics.
- Induction of resistance is mediated by a typical two component signal transducing system encoded by *vanR* and *vanS*, which sense the presence of inducing glycopeptides and activate transcription of the *van* gene cluster involved in the synthesis of modified peptidoglycan precursor.

#### Inducible bacitracin resistance in *Enterococcus faecalis*
- Bacitracin sequesters undecaprenol pyrophosphate, the lipid carrier in peptidoglycan synthesis. Bacitracin resistance is mediated by the ABC transporter BcrAB and the cotranscribed undecaprenol pyrophosphate BcrD. Expression of BcrABD is activated in the presence of bacitracin by a one-component signal transducing system BcrR.

Financial & competing interests disclosure

The authors have no relevant affiliations or financial involvement with any organization or entity with a financial interest in or financial conflict with the subject matter or materials discussed in the manuscript. This includes employment, consultancies, honoraria, stock ownership or options, expert testimony, grants or patents received or pending, or royalties.

No writing assistance was utilized in the production of this manuscript.

References

Papers of special note have been highlighted as:

- of interest
- of considerable interest

1. Wozniak RAF, Waldor MK. Integrative and conjugative elements: mosaic mobile genetic elements enabling dynamic lateral gene flow. Nat. Rev. Microbiol. 8(8), 552–563 (2010).
   - This review of integrative and conjugative elements discusses the core functions of these elements that allow for dynamic lateral gene flow, including regulation of conjugative transfer.
2. Guglielmini J, Quintais L, Garcillán-Barcia MP, De La Cruz F, Rocha EPC. The repertoire of ICE in prokaryotes underscores the unity, diversity, and ubiquity of conjugation. PLoS Genet. 7(8), e1002222 (2011).
3. Roberts AP, Mullany P. Tn916-like genetic elements: a diverse group of modular mobile elements conferring antibiotic resistance. FEMS Microbiol. Rev. 35(5), 856–871 (2011).
4. Lebreton F, Depardieu F, Bourdon N *et al.* D-Ala-D-Ser VanN-type transferable vancomycin resistance in Enterococcus faecium. Antimicrob. Agents Chemother. 55(10), 4606–4612 (2011).
5. Rana SW, Kumar A, Walia SK, Berven K, Cumper K. Isolation of Tn1546-like elements in vancomycin-resistant Enterococcus faecium isolated from wood frogs: an emerging risk for zoonotic bacterial infections to humans. J. Appl. Microbiol. 110(1), 35–43 (2011).
6. Courvalin P. Vancomycin resistance in Gram-positive cocci. Clin. Infect. Dis. 42(Suppl. 1), S25–S34 (2006).
7. Sletvold H, Johnsen PJ, Wikmark OG, Simonsen GS, Sundsfjord A, Nielsen KM. Tn1546 is part of a larger plasmid-encoded genetic unit horizontally disseminated among clonal Enterococcus faecium lineages. J. Antimicrob. Chemother. 65(9), 1894–1906 (2010).
8. Hegstad K, Mikalsen T, Coque TM, Werner G, Sundsfjord A. Mobile genetic elements and their contribution to the emergence of antimicrobial resistant Enterococcus faecalis and Enterococcus faecium. Clin. Microbiol. Infect. 16(6), 541–554 (2010).
9. International Working Group on the Classification of Staphylococcal Cassette Chromosome Elements Classification of Staphylococcal Cassette Chromosome mec (SCCmec): Guidelines for Reporting Novel SCCmec Elements. Antimicrob. Agents Chemother. 53(12), 4961–4967 (2009).
10. Lindsay J, Holden M. Understanding the rise of the superbug: investigation of the evolution and genomic variation of Staphylococcus aureus. Funct. Integr. Genomics 6, 186–201 (2006).
11. Smillie C, Garcillan-Barcia MP, Francia MV, Rocha EP, De La Cruz F. Mobility of plasmids. Mol. Biol. Rev. 74(3), 434–452 (2010).
12. Norman A, Hansen LH, Sørensen SJ. Conjugative plasmids: vessels of the communal gene pool. Philos. Trans. R. Soc. Lond. B Biol. Sci. 364(1527), 2275–2289 (2009).
13. Colomer-Lluch M, Jofre J, Muniesa M. Antibiotic resistance genes in the bacteriophage DNA fraction of environmental samples. PLoS One 6(3), e17549 (2011).
14. Di Luca MC, D’ercole S, Petrelli D, Prenna M, Ripa S, Vitali LA. Lysogenic transfer of mef(A) and tet(O) genes carried by φm46.1 among group A streptococci. Antimicrob. Agents Chemother. 54(10), 4464–4466 (2010).
15. Monecke S, Coombs G, Shore AC *et al.* A field guide to pandemic, epidemic and sporadic clones of methicillin-resistant Staphylococcus aureus. PLoS One 6(4), e17936 (2011).
16. Auchtung JM, Lee CA, Monson RE, Lehman AP, Grossman AD. Regulation of a Bacillus subtilis mobile genetic element by intercellular signaling and the global DNA damage response. Proc. Natl Acad. Sci. USA 102(35), 12554–12559 (2005).
17. Prudhomme M, Attaiech L, Sanchez G, Martin B, Claverys J-P. Antibiotic stress induces genetic transformability in the human pathogen Streptococcus pneumoniae. Science 313(5783), 89–92 (2006).
18. Roberts MC. Update on macrolide-lincosamide-streptogramin, ketolide, and oxazolidinone resistance genes. FEMS Microbiol. Lett. 282(2), 147–159 (2008).
19. Canu A, Leclercq R. Macrolides and lincosamides. In: Antimicrobial Drug Resistance: Mechanisms of Drug Resistance. Mayers DL (Ed.). Humana Press, NJ, USA, 211–221 (2009).
20. Mazzariol A, Koncan R, Vitali LA, Cornaglia G. Activities of 16-membered ring macrolides and telithromycin against different genotypes of erythromycin-susceptible and erythromycin-resistant Streptococcus pyogenes and Streptococcus pneumoniae. J. Antimicrob. Chemother. 59(6), 1171–1176 (2007).
21. Schwaiger K, Bauer J. Detection of the erythromycin rRNA methylase gene erm(A) in Enterococcus faecalis. Antimicrob. Agents Chemother. 52(8), 2994–2995 (2008).
22. Mayford M, Weisblum B. ermC leader peptide: amino acid sequence critical for induction by translational attenuation. J. Mol. Biol. 206(1), 69–79 (1989).
23. Horinouchi S, Weisblum B. Posttranscriptional modification of mRNA conformation: mechanism that regulates erythromycin-induced resistance. Proc. Natl Acad. Sci. USA 77(12), 7079–7083 (1980).
24. Ramu H, Mankin A, Vazquez-Laslop N. Programmed drug-dependent ribosome stalling. Mol. Microbiol. 71(4), 811–824 (2009).
25. Kramer G, Boehringer D, Ban N, Bukau B. The ribosome as a platform for co-translational processing, folding and targeting of newly synthesized proteins. Nat. Struct. Mol. Biol. 16(6), 589–597 (2009).
26. Vazquez-Laslop N, Thum C, Mankin AS. Molecular mechanism of drug-dependent ribosome stalling. Mol. Cell 30(2), 190–202 (2008).
   - Defines the importance of molecular interactions between specific ribosomal nucleotides, antibiotic ligands and nascent peptides in programmed ribosome stalling.
27. Bailey M, Chettiaih T, Mankin AS. Induction of erm(C) expression by noninducing antibiotics. Antimicrob. Agents Chemother. 52(3), 866–874 (2008).
28. Tenson T, Lovmar M, Ehrenberg M. The mechanism of action of macrolides, lincomamides and streptogramin B reveals the nascent peptide exit path in the ribosome. J. Mol. Biol. 330(5), 1005–1014 (2003).
29. Dunkle JA, Xiong L, Mankin AS, Cate JHD. Structures of the Escherichia coli ribosome with antibiotics bound near the peptidyl transferase center explain spectra of drug action. Proc. Natl Acad. Sci. USA doi:10.1073/pnas.1007988107 (2010) (Epub ahead of print).
30. Millán L, Goñi P, Cerdá P, Rubio M, Gómez-Lus R. Novel 10-bp deletion in the translational attenuator of a constitutively expressed erm(A) gene from Staphylococcus epidermidis. Int. Microbiol. 10(2), 147–150 (2007).

Review Chancey, Zähner & Stephens

1. Gatermann SG, Koschinski T, Friedrich S. Distribution and expression of macrolide resistance genes in coagulase-negative staphylococci. Clin. Microbiol. Infect. 13(8), 777–781 (2007).
2. Ramu H, Vazquez-Laslop N, Klepacki D et al. Nascent peptide in the ribosome exit tunnel affects functional properties of the A-site of the peptidyl transferase center. Mol. Cell 41(3), 321–330 (2011).

■ Reports that ribosomal stalling during translation of *erm(A)* transcripts is due to a series of orchestrated molecular interactions between inducing antibiotics and specific nascent peptides that prevent peptide bond formation in the A-site of the peptidyl transferase center.

3. Min Y-H, Kwon A-R, Yoon E-J, Shim M-J, Choi E-C. Translational attenuation and mRNA stabilization as mechanisms of *erm(B)* induction by erythromycin. Antimicrob. Agents Chemother. 52(5), 1782–1789 (2008).
4. Kwon AR, Min YH, Yoon EJ, Kim JA, Shim MJ, Choi EC. ErmK leader peptide: amino acid sequence critical for induction by erythromycin. Arch. Pharma. Res. 29(12), 1154–1157 (2006).
5. Lampson BC, Von David W, Parisi JT. Novel mechanism for plasmid-mediated erythromycin resistance by pNE24 from Staphylococcus epidermidis. Antimicrob. Agents Chemother. 30(5), 653–658 (1986).
6. Ojo KK, Striplin MJ, Ulep CC et al. Staphylococcus efflux *msr(A)* gene characterized in Streptococcus, Enterococcus, and Pseudomonas isolates. Antimicrob. Agents Chemother. 50(3), 1089–1091 (2006).
7. Le Bouter A, Leclercq R, Cattoir V. Molecular basis of resistance to macrolides, lincomamides and streptogramins in Staphylococcus saprophyticus clinical isolates. Int. J. Antimicrob. Agents 37(2), 118–123 (2011).
8. Davis KA, Crawford SA, Fiebelkorn KR, Jorgensen JH. Induction of telithromycin resistance by erythromycin in isolates of macrolide-resistant Staphylococcus spp. Antimicrob. Agents Chemother. 49(7), 3059–3061 (2005).
9. Chancey ST, Zhou X, Zähner D, Stephens DS. Induction of efflux-mediated macrolide resistance in Streptococcus pneumoniae. Antimicrob. Agents Chemother. 55(7), 3413–3422 (2011).
10. Zähner D, Zhou X, Chancey ST, Pohl J, Shafer WM, Stephens DS. Human antimicrobial peptide LL-37 induces MefE/Mel-mediated macrolide resistance in Streptococcus pneumoniae. Antimicrob. Agents Chemother. 54(8), 3516–3519 (2010).
11. Cai Y, Kong F, Gilbert GL. Three new macrolide efflux (*mef*) gene variants in Streptococcus agalactiae. J. Clin. Microbiol. 45(8), 2754–2755 (2007).
12. Vitali L, Di Luca M, Iebba V et al. Variability and Inducibility of the *mef(A)–msr(D)* Region in Erythromycin-Resistant Streptococcus pyogenes. European Society of Clinical Microbiology and Infectious Diseases. Milan, Italy (2011).
13. Reynolds E, Cove JH. Enhanced resistance to erythromycin is conferred by the enterococcal *msrC* determinant in Staphylococcus aureus. J. Antimicrob. Chemother. 55(2), 260–264 (2005).
14. Lodato PB, Rogers EJ, Lovett PS. A variation of the translation attenuation model can explain the inducible regulation of the pBC16 tetracycline resistance gene in Bacillus subtilis. J. Bacteriol. 188(13), 4749–4758 (2006).
15. Ammor MS, Gueimonde M, Danielsen M et al. Two different tetracycline resistance mechanisms, plasmid-carried *tet(L)* and chromosomally located transposon-associated *tet(M)*, coexist in Lactobacillus sakei Rits 9. Appl. Environ. Microbiol. 74(5), 1394–1401 (2008).
16. McIntosh ED, Reinert RR. Global prevailing and emerging pediatric pneumococcal serotypes. Expert Rev. Vaccines 10(1), 109–129 (2011).
17. Jordan S, Hutchings MI, Mascher T. Cell envelope stress response in Gram-positive bacteria. FEMS Microbiol. Rev. 32(1), 107–146 (2008).
18. Gao R, Stock AM. Biological insights from structures of two-component proteins. Ann. Rev. Microbiol. 63, 133–154 (2009).
19. Koteva K, Hong H-J, Wang XD et al. A vancomycin photoprobe identifies the histidine kinase VanSsc as a vancomycin receptor. Nat. Chem. Biol. 6(5), 327–329 (2010).
20. Werner G, Strommenger B, Witte W. Acquired vancomycin resistance in clinically relevant pathogens. Future Microbiol. 3(5), 547–562 (2008).
21. Evers S, Courvalin P. Regulation of VanB-type vancomycin resistance gene expression by the VanS(B)-VanR (B) two-component regulatory system in Enterococcus faecalis V583. J. Bacteriol. 178(5), 1302–1309 (1996).
22. Baptista M, Depardieu F, Courvalin P, Arthur M. Specificity of induction of glycopeptide resistance genes in Enterococcus faecalis. Antimicrob. Agents Chemother. 40(10), 2291–2295 (1996).
23. Boyd DA, Willey BM, Fawcett D, Gillani N, Mulvey MR. Molecular characterization of

Enterococcus faecalis N06–0364 with low-level vancomycin resistance harboring a novel d-Ala-d-Ser gene cluster, *vanL*. Antimicrob. Agents Chemother. 52(7), 2667–2672 (2008).
24. Xu X, Lin D, Yan G et al. *vanM*, a new glycopeptide resistance gene cluster found in Enterococcus faecium. Antimicrob. Agents Chemother. 54(11), 4643–4647 (2010).
25. Depardieu F, Kolbert M, Pruil H, Bell J, Courvalin P. VanD-type vancomycin-resistant Enterococcus faecium and Enterococcus faecalis. Antimicrob. Agents Chemother. 48(10), 3892–3904 (2004).
26. Abadia Patino L, Courvalin P, Perichon B. *vanE* gene cluster of vancomycin-resistant Enterococcus faecalis BM4405. J. Bacteriol. 184(23), 6457–6464 (2002).
27. Fisher SL, Jiang W, Wanner BL, Walsh CT. Cross-talk between the histidine protein kinase VanS and the response regulator PhoB. Characterization and identification of a VanS domain that inhibits activation of PhoB. J. Biol. Chem. 270(39), 23143–23149 (1995).
28. Hutchings MI, Hong H-J, Buttner MJ. The vancomycin resistance VanRS two-component signal transduction system of Streptomyces coelicolor. Mol. Microbiol. 59(3), 923–935 (2006).
29. Lee WG, Huh JY, Cho SR, Lim YA. Reduction in glycopeptide resistance in vancomycin-resistant enterococci as a result of vanA cluster rearrangements. Antimicrob. Agents Chemother. 48(4), 1379–1381 (2004).
30. Sung K, Khan SA, Nawaz MS. Genetic diversity of Tn1546-like elements in clinical isolates of vancomycin-resistant enterococci. Int. J. Antimicrob. Agents 31(6), 549–554 (2008).
31. Gagnon S, Lévesque S, Lefebvre B, Bourgault A-M, Labbé A-C, Roger M. vanA-containing Enterococcus faecium susceptible to vancomycin and teicoplanin because of major nucleotide deletions in Tn1546. J. Antimicrob. Chemother. doi:10.1093/jac/dkr379 (2011) (Epub ahead of print).
32. San Millan A, Depardieu F, Godreuil S, Courvalin P. VanB-Type Enterococcus faecium clinical isolate successively inducibly resistant to, dependent on, and constitutively resistant to vancomycin. Antimicrob. Agents Chemother. 53(5), 1974–1982 (2009).
33. Ribeiro T, Santos S, Marques MIM, Gilmore M, De Fátima Silva Lopes M. Identification of a new gene, *vanV*, in *vanB* operons of Enterococcus faecalis. Int. J. Antimicrob. Agents 37(6), 554–557 (2011).
34. Moubareck C, Meziane-Cherif D, Courvalin P, Perichon B. VanA-type Staphylococcus aureus strain VRSA-7 is partially dependent

on vancomycin for growth. Antimicrob.
Agents Chemother. 53(9), 3657–3663 (2009).

65. Fraimow H, Knob C, Herrero IA, Patel R.
    Putative VanRS-like two-component
    regulatory system associated with the
    inducible glycopeptide resistance cluster of
    Paenibacillus popilliae. Antimicrob.
    Agents Chemother. 49(7), 2625–2633 (2005).

66. Poyart C, Pierre C, Quesne G, Pron B, Berche
    P, Trieu-Cuot P. Emergence of vancomycin
    resistance in the genus *Streptococcus*:
    characterization of a *vanB* transferable
    determinant in *Streptococcus bovis*. Antimicrob.
    Agents Chemother. 41(1), 24–29 (1997).

67. Domingo MC, Huletsky A, Giroux R *et al.*
    High prevalence of glycopeptide resistance
    genes *vanB*, *vanD*, and *vanG* not associated
    with enterococci in human fecal flora.
    Antimicrob. Agents Chemother. 49(11),
    4784–4786 (2005).

68. Werner G, Freitas AR, Coque TM *et al.* Host
    range of enterococcal *vanA* plasmids among
    Gram-positive intestinal bacteria.
    J. Antimicrob. Chemother. 66(2), 273–282
    (2011).

69. Noble WC, Virani Z, Cree RG. Co-transfer
    of vancomycin and other resistance genes
    from *Enterococcus faecalis* NCTC 12201 to
    *Staphylococcus aureus*. FEMS Microbiol. Lett.
    72(2), 195–198 (1992).

70. Chang S, Sievert DM, Hageman JC *et al.*
    Infection with vancomycin-resistant
    *Staphylococcus aureus* containing the *vanA*
    resistance gene. *N. Engl. J. Med.* 348(14),
    1342–1347 (2003).

71. CDC. *Staphylococcus aureus* resistant to
    vancomycin – United States, 2002. *Morbid.*
    Mortal. Wkly Rep. 51, 565–567 (2002).

72. Weigel LM, Clewell DB, Gill SR *et al.*
    Genetic analysis of a high-level vancomycin-
    resistant isolate of *Staphylococcus aureus*.
    Science 302(5650), 1569–1571 (2003).

73. Foucault M-L, Depardieu F, Courvalin P,
    Grillot-Courvalin C. Inducible expression
    eliminates the fitness cost of vancomycin
    resistance in enterococci. Proc. Natl Acad. Sci.
    USA 107(39), 16964–16969 (2010).

- Directly addresses the role of inducibility
  in alleviating the fitness cost associated
  with Van-type vancomycin resistance.

74. Bisicchia P, Bui NK, Aldridge C, Vollmer W,
    Devine KM. Acquisition of VanB-type
    vancomycin resistance by *Bacillus subtilis*: the
    impact on gene expression, cell wall
    composition and morphology. Mol. Microbiol.
    81(1), 157–178 (2011).

75. Moritz EM, Hergenrother PJ. Toxin-antitoxin
    systems are ubiquitous and plasmid-encoded
    in vancomycin-resistant enterococci. Proc.
    Natl Acad. Sci. USA 104(1), 311–316 (2007).

76. Johnsen PJ, Simonsen GS, Olsvik Ø, Midtvedt
    T, Sundsfjord A. Stability, persistence, and
    evolution of plasmid-encoded VanA
    glycopeptide resistance in *Enterococci* in the
    absence of antibiotic selection *in vitro* and in
    gnotobiotic mice. Microbial Drug Resis. 8(3),
    161–170 (2002).

77. Hill CM, Krause KM, Lewis SR *et al.*
    Specificity of induction of the *vanA* and *vanB*
    operons in vancomycin-resistant enterococci
    by telavancin. Antimicrob. Agents Chemother.
    54(7), 2814–2818 (2010).

78. Xie J, Pierce JG, James RC, Okano A, Boger
    DL. A redesigned vancomycin engineered for
    Dual D-Ala-D-Ala and D-Ala-D-Lac binding
    exhibits potent antimicrobial activity against
    vancomycin-resistant bacteria. *J. Am. Chem.*
    Soc. 133(35), 13946–13949 (2011).

79. Marrero A, Mallorquí-Fernández G, Guevara
    T, García-Castellanos R, Gomis-Ruth FX.
    Unbound and acylated structures of the
    *mecR1* extracellular antibiotic-sensor domain
    provide insights into the signal-transduction
    system that triggers methicillin resistance.
    *J. Mol. Biol.* 361(3), 506–521 (2006).

80. Thumanu K, Cha J, Fisher JF, Perrins R,
    Mobashery S, Wharton C. Discrete steps in
    sensing of β-lactam antibiotics by the BlaR1
    protein of the methicillin-resistant
    *Staphylococcus aureus* bacterium. Proc. Natl
    Acad. Sci. USA 103(28), 10630–10635
    (2006).

81. Fuda CC, Fisher JF, Mobashery S. β-lactam
    resistance in *Staphylococcus aureus*: the
    adaptive resistance of a plastic genome. Cell.
    Mol. Life Sci. 62(22), 2617–2633 (2005).

82. Mckinney TK, Sharma VK, Craig WA, Archer
    GL. Transcription of the gene mediating
    methicillin resistance in *Staphylococcus aureus*
    (*mecA*) is corepressed but not coinduced by
    cognate *mecA* and β-lactamase regulators.
    *J. Bacteriol.* 183(23), 6862–6868 (2001).

83. Oliveira DC, De Lencastre H. Methicillin-
    resistance in *Staphylococcus aureus* is not
    affected by the overexpression in trans of the
    *mecA* gene repressor: a surprising observation.
    PLoS One 6(8), e23287 (2011).

84. Berger-Bächi B, Senn MM, Ender M *et al.*
    Resistance to β-lactam antibiotics. In:
    *Staphylococci in Human Disease.* Crossley KB,
    Archer G, Jeffersin K, Fowler V (Eds).
    Wiley-Blackwell, Oxford, UK, 170–192
    (2010).

85. Malhotra-Kumar S, Haccuria K, Michiels M
    *et al.* Current trends in rapid diagnostics for
    methicillin-resistant *Staphylococcus aureus* and
    glycopeptide-resistant *Enterococcus* species.
    *J. Clin. Microbiol.* 46(5), 1577–1587 (2008).

86. Malachowa N, Deleo FR. Mobile genetic
    elements of *Staphylococcus aureus.* Cell. Mol.
    Life Sci. 67(18), 3057–3071 (2010).

87. Tsubakishita S, Kuwahara-Arai K, Baba T,
    Hiramatsu K. Staphylococcal cassette
    chromosome *mec*-like element in *Macrococcus*
    *caseolyticus.* Antimicrob. Agents Chemother.
    54(4), 1469–1475 (2010).

88. Shore AC, Deasy EC, Slickers P *et al.* Detection
    of staphylococcal cassette chromosome mec
    type XI encoding highly divergent *mecA*, *mecI*,
    *mecRI*, *blaZ* and *ccr* genes in human clinical
    clonal complex 130 methicillin-resistant
    *Staphylococcus aureus.* Antimicrob. Agents
    Chemother. 55(8), 3765–3773 (2011).

89. Song MD, Wachi M, Doi M, Ishino F,
    Matsuhashi M. Evolution of an inducible
    penicillin-target protein in methicillin-resistant
    *Staphylococcus aureus* by gene fusion. FEBS Lett.
    221(1), 167–171 (1987).

90. Rosato AE, Kreiswirth BN, Craig WA, Eisner
    W, Climo MW, Archer GL. *mecA-blaZ*
    corepressors in clinical *Staphylococcus aureus*
    isolates. Antimicrob. Agents Chemother. 47(4),
    1460–1463 (2003).

91. Katayama Y, Zhang HZ, Hong D, Chambers
    HF. Jumping the barrier to β-lactam resistance
    in *Staphylococcus aureus.* J. Bacteriol. 185(18),
    5465–5472 (2003).

92. Ender M, Mcallum N, Adhikari R,
    Berger-Bachi B. Fitness cost of SCC*mec* and
    methicillin resistance levels in *Staphylococcus*
    *aureus.* Antimicrob. Agents Chemother. 48(6),
    2295–2297 (2004).

93. Lee SM, Ender M, Adhikari R, Smith JM,
    Berger-Bachi B, Cook GM. Fitness cost of
    staphylococcal cassette chromosome *mec* in
    methicillin-resistant *Staphylococcus aureus* by
    way of continuous culture. Antimicrob. Agents
    Chemother. 51(4), 1497–1499 (2007).

94. Milheirico C, Portelinha A, Krippahl L, De
    Lencastre H, Oliveira DC. Evidence for a
    purifying selection acting on the β-lactamase
    locus in epidemic clones of methicillin-resistant
    *Staphylococcus aureus.* BMC Microbiol. 11, 76
    (2011).

95. Goldstein F, Perutka J, Cuirolo A *et al.*
    Identification and phenotypic characterization
    of a β-lactam-dependent, methicillin-resistant
    *Staphylococcus aureus* strain. Antimicrob. Agents
    Chemother. 51(7), 2514–2522 (2007).

96. Garza-González E, López D, Pezina C *et al.*
    Diversity of staphylococcal cassette
    chromosome *mec* structures in coagulase-
    negative staphylococci and relationship to drug
    resistance. *J. Med. Microbiol.* 59(Pt 3),
    323–329 (2010).

97. Garza-González E, Morfín-Otero R,
    Llacá-Díaz JM, Rodriguez-Noriega E.
    Staphylococcal cassette chromosome *mec*
    (SCC*mec*) in methicillin-resistant coagulase-
    negative staphylococci. A review and the
    experience in a tertiary-care setting. Epidemol.
    Infect. 138(5), 645–654 (2010).

Review Chancey, Zähner & Stephens

98. Miragaia M, Thomas JC, Couto I, Enright MC, De Lencastre H. Inferring a population structure for *Staphylococcus epidermidis* from multilocus sequence typing data. *J. Bacteriol.* 189(6), 2540–2552 (2007).

99. Fessler AT, Kadlec K, Schwarz S. Novel apramycin resistance gene *apmA* in bovine and porcine methicillin-resistant *Staphylococcus aureus* ST398 isolates. *Antimicrob. Agents Chemother.* 55(1), 373–375 (2011).

100. Zong Z, Lü X. Characterization of a new SCCmec element in *Staphylococcus cohnii*. *PLoS One* 5(11), e14016 (2010).

101. Dubnau DA, Pollock MR. The genetics of *Bacillus licheniformis* penicillinase: a preliminary analysis from studies on mutation and inter-strain and intra-strain transformations. *J. Gen. Microbiol.* 41(1), 7–22 (1965).

102. Sebaihia M, Wren BW, Mullany P *et al.* The multidrug-resistant human pathogen *Clostridium difficile* has a highly mobile, mosaic genome. *Nat. Genet.* 38(7), 779–786 (2006).

103. Severin A, Tabei K, Tenover F, Chung M, Clarke N, Tomasz A. High level oxacillin and vancomycin resistance and altered cell wall composition in *Staphylococcus aureus* carrying the staphylococcal *mecA* and the enterococcal *vanA* gene complex. *J. Biol. Chem.* 279(5), 3398–3407 (2004).

104. Fox PM, Lampen RJ, Stumpf KS, Archer GL, Climo MW. Successful therapy of experimental endocarditis caused by vancomycin-resistant *Staphylococcus aureus* with a combination of vancomycin and

105. Hiramatsu K. Resistance to glycopeptides. In: *Staphylococci in Human Disease*. Crossley KB, Archer G, Jeffersin K, Fowler V. (Eds). Wiley, Oxford, UK, 193–209 (2010).

106. Manson JM, Keis S, Smith JM, Cook GM. Acquired bacitracin resistance in *Enterococcus faecalis* is mediated by an ABC transporter and a novel regulatory protein, BcrR. *Antimicrob. Agents Chemother.* 48(10), 3743–3748 (2004).

107. Gebhard S, Gaballa A, Helmann JD, Cook GM. Direct stimulus perception and transcription activation by a membrane-bound DNA binding protein. *Mol. Microbiol.* 73(3), 482–491 (2009).

108. Diarra MS, Rempel H, Champagne J, Masson L, Pritchard J, Topp E. Distribution of antimicrobial resistance and virulence genes in *Enterococcus* spp. and characterization of isolates from broiler chickens. *Antimicrob. Agents Chemother.* 76(24), 8033–8043 (2010).

109. Jorgensen JH, McElmeel ML, Fulcher LC, McGee L, Glennen A. Evaluation of disk approximation and single well broth tests for detection of inducible clindamycin resistance in *Streptococcus pneumoniae*. *J. Clin. Microbiol.* 49(9), 3332–3333 (2011).

110. Jorgensen JH, McElmeel ML, Fulcher LC *et al.* Collaborative evaluation of an erythromycin-clindamycin combination well for detection of inducible clindamycin resistance in β-hemolytic streptococci by use of the CLSI broth microdilution method. *J. Clin. Microbiol.* 49(8), 2884–2886 (2011).

111. Wieser A, Schneider L, Jung J, Schubert S. MALDI-TOF MS in microbiological diagnostics—identification of microorganisms and beyond. *Appl. Microbiol. Biotechnol.* 93(3), 965–974 (2011).

112. Domelier A-S, Van Der Mee-Marquet N, Arnault L *et al.* Molecular characterization of erythromycin-resistant *Streptococcus agalactiae* strains. *J. Antimicrob. Chemother.* 62(6), 1227–1233 (2008).

113. López F, Culebras E, Betriú C, Rodríguez-Avil I, Gómez M, Picazo JJ. Antimicrobial susceptibility and macrolide resistance genes in *Enterococcus faecium* with reduced susceptibility to quinupristin-dalfopristin: level of quinupristin-dalfopristin resistance is not dependent on *erm(B)* attenuator region sequence. *Diagn. Microbiol. Infect. Dis.* 66(1), 73–77 (2010).

114. Soge OO, Tivoli LD, Meschke JS, Roberts MC. A conjugative macrolide resistance gene, *mef(A)*, in environmental *Clostridium perfringens* carrying multiple macrolide and/or tetracycline resistance genes. *J. Appl. Microbiol.* 106(1), 34–40 (2009).

115. Yao S, Blaustein JB, Bechhofer DH. Erythromycin-induced ribosome stalling and RNase J1-mediated mRNA processing in *Bacillus subtilis*. *Mol. Microbiol.* 69(6), 1439–1449 (2008).

Websites

201. Marilyn C. Roberts, Ph.D. http://faculty.washington.edu/marilynr

202. Currently identified SCCmec types in *S. aureus* strains. www.sccmec.org/Pages/SCC_TypesEN.html
