
G Model
BIOTEC-6161; No. of Pages 12

ARTICLE IN PRESS

Journal of Biotechnology xxx (2012) xxx–xxx

Contents lists available at SciVerse ScienceDirect

Journal of Biotechnology

journal homepage: www.elsevier.com/locate/jbiotec

The cytosolic and extracellular proteomes of Actinoplanes sp. SE50/110 led to the identification of gene products involved in acarbose metabolism

Sergej Wendler${}^{a,b,*}$, Daniel Hürtgen${}^{a}$, Jörn Kalinowski${}^{c}$, Andreas Klein${}^{d}$, Karsten Niehaus${}^{e}$, Fabian Schulte${}^{b,e}$, Patrick Schwientek${}^{a,b,f}$, Hermann Wehlmann${}^{g}$, Udo F. Wehmeier${}^{h}$, Alfred Pühler${}^{a}$

${}^{a}$ Senior Research Group in Genome Research of Industrial Microorganisms, Center for Biotechnology, Bielefeld University, Universitätsstraße 27, 33615 Bielefeld, Germany  
${}^{b}$ CLIB-Graduate Cluster Industrial Biotechnology, CLIB2021, Völklinger Strasse 4, 40219 Düsseldorf, Germany  
${}^{c}$ Microbial Genomics and Biotechnology, Center for Biotechnology, Bielefeld University, Universitätsstraße 27, 33615 Bielefeld, Germany  
${}^{d}$ Product Supply, Bayer Pharma AG, Friedrich Ebert Str. 217-475, 42117 Wuppertal, Germany  
${}^{e}$ Department of Proteome and Metabolome Research, Faculty of Biology, Bielefeld University, Universitätsstraße 25, 33615 Bielefeld, Germany  
${}^{f}$ Genome Informatics Research Group, Faculty of Technology, Bielefeld University, Universitätsstraße 25, 33615 Bielefeld, Germany  
${}^{g}$ Biotech Development, Bayer HealthCare, Friedrich-Ebert-Str. 217-475, 42117 Wuppertal, Germany  
${}^{h}$ Lehrstuhl für Sportmedizin, Bergische Universität Wuppertal, Pauluskirchstr. 7, 42285 Wuppertal, Germany  

---

**A R T I C L E I N F O**

Article history:  
Received 24 April 2012  
Received in revised form 1 August 2012  
Accepted 10 August 2012  
Available online xxx  

Keywords:  
Proteomics  
Actinoplanes  
Cytosolic and secreted proteome  
Carbophore  
Acarbose metabolism  

---

**A B S T R A C T**

The pseudotetrasaccharide acarbose is a medically relevant secondary metabolite produced by strains of the genera *Actinoplanes* and *Streptomyces*. In this study gene products involved in acarbose metabolism were identified by analyzing the cytosolic and extracellular proteome of *Actinoplanes* sp. SE50/110 cultures grown in a high-maltose minimal medium. The analysis by 2D protein gel electrophoresis of cytosolic proteins of *Actinoplanes* sp. SE50/110 resulted in 318 protein spots and 162 identified proteins. Nine of those were acarbose cluster proteins (Acb-proteins), namely AcbB, AcbD, AcbE, AcbK, AcbL, AcbN, AcbR, AcbV and AcbZ. The analysis of proteins in the extracellular space of *Actinoplanes* sp. SE50/110 cultures resulted in about 100 protein spots and 22 identified proteins. The identifications included the three acarbose gene cluster proteins AcbD, AcbE and AcbZ. After their identification, proteins were classified into functional groups. The dominant functional groups were the carbohydrate binding, carbohydrate cleavage and carbohydrate transport proteins. The other functional groups included protein cleavage, amino acid degradation, nucleic acid cleavage and a number of functionally uncharacterized proteins. In addition, signal peptide structures of extracellularly found proteins were analyzed. Of the 22 detected proteins 19 contained signal peptides, while 2 had N-terminal transmembrane helices explaining their localization. The only protein having neither of them was enolase. Under the conditions applied, the secretome of *Actinoplanes* sp. SE50/110 was dominated by seven proteins involved in carbohydrate metabolism (PulA, AcbE, AcbD, MalE, AglE, CbpA and Cgt). Of special interest were the identified extracellular pullulanase PulA and the two solute-binding proteins MalE and AglE. The identifications suggest that *Actinoplanes* sp. SE50/110 has two maltose/maltodextrin import systems. We postulate the identified MalEFG transport system of *Actinoplanes* sp. SE50/100 as the missing acarbose-metabolite importer and present a model of acarbose metabolism that is extended by the newly identified gene products.

© 2012 Elsevier B.V. All rights reserved.

---

1. Introduction

The α-glucosidase inhibitor acarviosyl-1,4-maltose (acarbose) is produced by strains of *Actinoplanes* sp. Acarbose is used since 1990 in the treatment of Diabetes mellitus type II due to its inhibition of α-glucosidases of the human intestine and the resulting decreased release of glucose from starch- and sucrose-containing diets (*Bischoff et al.*, 1994; *Truscheit et al.*, 1981). The discovery of acarbose dates back to a screening for mammalian α-amylase, sucrase and maltase inhibitors of bacterial origin at Bayer AG in 1970 (*Wehmeier and Piepersberg*, 2004). Here, *Actinoplanes* sp. SE50/110, a Gram-positive, slow growing high-GC (>70%) actinomycete that has some unusual traits as motile spores and light-dependent carotenoid production, was identified as the natural producer (*Szanzslo and Gooder*, 1967; *Parenti and Coronelli*, 1979). Today’s production relies on optimized strains of *Actinoplanes* sp. SE50/110 and a sophisticated large-scale multi-step fermentation process (*Wehmeier and Piepersberg*, 2004).

---

* Corresponding author at: Center for Biotechnology, Bielefeld University, Universitätsstraße 27, 33615 Bielefeld, Germany. Tel.: +49 521 106 8709; fax: +49 521 106 89046.  
E-mail address: Wendler@CeBiTec.Uni-Bielefeld.de (S. Wendler).

0168-1656/$ – see front matter © 2012 Elsevier B.V. All rights reserved.  
http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of *Actinoplanes* sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

G Model
BIOTEC-6161; No. of Pages 12

ARTICLE IN PRESS

S. Wendler et al. / Journal of Biotechnology xxx (2012) xxx–xxx

Acarviosc is the inhibitory core structure of all acarbose-like metabolites and consists of a C7-cyclitol and an imino bridge bound 4-amino-4,6-dideoxyglucose. A mostly complete biosynthetic pathway for acarbose was first proposed by Zhang et al. (2002) and recently updated by Rockser and Wehmeier (2009). Some interesting notions are that the cyclitol moiety is derived from the pentose phosphate pathway (Degwert et al., 1987) and that maltose is incorporated not by successive glucose addition but as a whole (Lee et al., 1997). The gene cluster coding for the enzymes responsible for acarbose biosynthesis and associated metabolism in Actinoplanes sp. SE50/110 is known since 2001 (Apeler et al., 2001; Wehmeier, 2003). At that time the cluster was proposed to consist of 25 genes which were either dedicated to biosynthesis, extracellular starch degradation or transport. Recently our group reported the whole genome sequence of Actinoplanes sp. SE50/110 (Schwientek et al., 2011, 2012).

Acarbose is supposed to have two main functions in the environment. First, inhibiting growth of competitors and second, acting as a carbophore. The mechanism of the carbophore model is based on cycling of acarbose-derived metabolites between intra- and extracellular pools. Therefore, acarbose is synthesized intracellularly, exported to the extracellular space where it is loaded with mono and oligosaccharides that are provided by acarbose-insensitive glucosidases. Loaded acarbose is reimported and phosphorylated to avoid inhibition of host enzymes (Drepper and Pape, 1996; Rockser and Wehmeier, 2009). Finally saccharides bound to acarbose are cleaved off thereby recycling acarbose for another round and providing a net gain of carbohydrates (for review see also Wehmeier and Piepersberg, 2004; Rockser and Wehmeier, 2009). Extracellularly the transfer of acarbose or acarviosine to different saccharides creates an exclusive carbohydrate pool that inhibits a broad spectrum of α-glucosidases of competitors and provides a reasonable growth advantage for slow growing Actinoplanes sp. (Truschel et al., 1981; Schmidt et al., 1977; Brunkhorst et al., 1999; Wehmeier and Piepersberg, 2004).

Today it is known that the acarbose gene cluster consists of 22 genes. Binding assays and crystallization studies by Licht et al. (2011) convincingly showed that the supposed acarbose importer AcbHFG is involved in galactose uptake and does not belong to the acarbose cluster. Consequently, Actinoplanes sp. lacks a specific acarbose importer encoded by the acb-gene cluster. This implies that another transporter, encoded at a different genetic locus is responsible for the reimport.

Recently, the second acarbose gene cluster (gac-cluster) was identified and sequenced in Streptomyces glaucescens GLA.O (Rockser and Wehmeier, 2009). The gac-cluster consists of 29 open reading frames and contains a gene coding for an acarbose importer whose function was experimentally proven (Vahedi-Faridi et al., 2010).

Since the genome sequence of Actinoplanes sp. SE50/110 was determined recently (Schwientek et al., 2012), a large scale identification of proteins is now possible. The aim of this study was to prove the expression and localization of proteins encoded in the acarbose gene cluster of Actinoplanes sp. SE50/110. In addition it was of particular interest to find out whether additional proteins can be identified that are relevant for the acarbose metabolism. For this reason the cytosolic and extracellular proteome of Actinoplanes sp. SE50/110 were studied.

polycarbonate Erlenmeyer flasks (Corning) with Silicosen C-55 culture plugs (Hirschmann Laborgeräte) in 100ml NBS medium (Bayer HealthCare, Wuppertal) for 4d at 140rpm and 28°C in a GFL shaking incubator 3032 (GFL). The NBS medium is a glucose containing complex medium (Glc-CM) that consisted of glucose·H₂O 11g/l, peptone (Carl Roth) 4g/l, yeast extract (Oxoid) 4g/l, MgSO₄·7H₂O 1g/l, KH₂PO₄ 2g/l, K₂HPO₄·3H₂O 5.2g/l. For the inoculation of main cultures pre-cultures were centrifuged at 4.000rcf for 2min and washed twice with 45ml of sterile 150mM NaCl solution. After another centrifugation step (4.000rcf, 2min) the supernatant was decanted. The resulting pellet was resuspended in 20ml NaCl solution of which 1ml was used for inoculation (equivalent to 5% of pre-culture). For proteomics experiments cultures were cultivated in 250ml baffled polycarbonate Erlenmeyer flasks (Corning) with Silicosen C-40 culture plugs (Hirschmann Laborgeräte) in 50ml high-maltose minimal medium (Mal-MM) for 3dat 140rpm and 28°C in a GFL shaking incubator 3032 (GFL). The high-maltose minimal medium (Mal-MM) was priorly developed on the basis of the Cerestar medium (Rockser and Wehmeier, 2009) and modified regarding several aspects. It consisted of four instead of three solutions, lacked yeast extract as a complex component, included new compounds and adjusted concentrations. Solution 1 consisted of 70.7g maltose·1H₂O and 5g (NH₄)₂SO₄ ad aqua dest. 400ml, solution 2 of 6.55gK₂HPO₄·3H₂O and 5g KH₂PO₄ in 370ml aqua dest., solution 3 of 5.7g trisodium citrate·2H₂O in 200ml aqua dest. and solution 4 of 1g MgCl₂·6H₂O, 2g CaCl₂·2H₂O and 0.2ml 1:5000 trace element solution in 30ml aqua dest. The trace element solution consisted of 15.75mM FeCl₂, 25.00mM MnCl₂, 33.75mM CaCl₂, 3.75mM ZnCl₂, 0.50mM CuCl₂, 0.05mM NiCl₂, 0.02mM [Co(NH₃)₆]Cl₃ dissolved in 1M HCl. Solution 1 to 3 were combined and filter sterilized, while solution 4 was filter sterilized separately and added afterwards.

2. Material and methods

2.1. Cultivation of Actinoplanes sp. SE50/110 cultures for the analysis of cytosolic proteins

Actinoplanes sp. SE50/110 pre-cultures were inoculated with 3.5ml glycerin cryo cultures and cultivated in 500ml baffled

2.2. Cultivation of Actinoplanes sp. SE50/110 cultures for the analysis of the extracellular protein fraction

Pre-cultures were prepared in the same way as already described for cytosolic proteins in Section 2.1 with the difference that cultures were cultivated for 3d. For the inoculation of main cultures pre-cultures were centrifuged at 2.250rcf for 3min and washed with 45ml of sterile 150mM NaCl solution twice. After another centrifugation step (2.250rcf × 3min) the supernatant was decanted. The resulting pellet was resuspended in 10ml NaCl solution of which 2ml were used for inoculation (equivalent to 20% of pre-culture). Main cultures were cultivated in triplicate in 500ml baffled polycarbonate Erlenmeyer flasks (Corning) with Silicosen C-55 culture plugs (Hirschmann Laborgeräte) in 100ml fully-synthetic acarbose production medium (Mal-MM) for 3dat 140rpm and 28°C in a GFL shaking incubator 3032 (GFL). Since no information, respectively experience, regarding yields of extracellular proteins in Actinoplanes sp. SE50/110 cultures was available, higher culture volumes and an adjusted inoculation strategy applying more exponential cells were chosen to harvest sufficient amounts of secreted protein for subsequent analysis. The Mal-MM for the analysis of extracellular proteins was basically the same as for the cytosolic fraction, but had some minor differences regarding the concentrations of some components. Solution 1 consisted of 73.7g maltose·1H₂O and 5g (NH₄)₂SO₄ ad aqua dest. 400ml, solution 2 of 6.55g K₂HPO₄·3H₂O and 5g KH₂PO₄ in 300ml aqua dest., solution 3 of 184mg FeCl2·4H₂O and 5.7g trisodium citrate·2H₂O in 300ml aqua dest. and solution 4 of 1g MgCl₂·6H₂O, 2g CaCl₂·2H₂O and 0.4ml 1:5000 trace element solution in 20ml aqua dest. The trace element solution consisted of 15.75mM FeCl₂, 25.00mM MnCl₂, 33.75mM CaCl₂, 3.75mM ZnCl₂, 0.50mM NiCl₂, 0.02mM [Co(NH₃)₆]Cl₃ dissolved in 1M HCl.

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of Actinoplanes sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

G Model
BIOTEC-6161; No. of Pages 12

ARTICLE IN PRESS

S. Wendler et al. / Journal of Biotechnology xxx (2012) xxx–xxx

Solution 1 to 3 were combined and filter sterilized, while solution 4 was filter sterilized.

### 2.3. Harvesting of Actinoplanes sp. SE50/110 cultures for the analysis of cytosolic and extracellular proteins and determination of cell dry weight

For samples of cytosolic proteins main cultures were harvested by centrifugation (2 min, 4000 rcf) of whole 50 ml cultures in 50 ml centrifuge tubes (Greiner Bio-One) and washed twice with 4°C washing buffer (50 mM Tris/HCl pH 7.2). The resulting pellets were freeze-dried and weighed.

For samples of extracellular proteins three 100 ml main cultures were combined after 3 d of cultivation. 50 ml were then used for determination of cell dry weight by washing, freeze-drying and weighing as described for the cytosolic samples. The remaining 250 ml were used for extraction of proteins from the supernatant. Biomass and medium were separated by filtration with two common coffee filters. The resulting filtrate was frozen and lyophilized thereafter.

### 2.4. Quantification of acarbose in the supernatant of Actinoplanes sp. SE50/110 cultures by HPLC and UV detection

Acarbose concentration in 10 μl of filter sterilized supernatant was quantified by HPLC (KNAUER, Smartline Manager 5000, Smartline Pump 1000, UV Smartline Detector 2500 and Spark Holland BV, Triathlon autosampler). An isocratic flow of 1.7 ml/min of 64% acetonitrile, 10% methanol and 26% phosphate buffer consisting of 0.62 g/l KH₂PO₄ and 0.38 g/l K₂HPO₄·2H₂O was applied over an Hypersil APS-2 precolumn cartridge (MZ Analysentechnik, No. VK 5.4, 0.6085) and a Hypersil APS-2 analytical column (Thermo Scientific, No. 30703-124030). The temperature was adjusted to 33°C. Acarbose was detected at 210 nm against an acarbose standard kindly provided by Bayer HealthCare.

### 2.5. Extraction of cytosolic proteins from Actinoplanes sp. SE50/110 cultures

Cells were mechanically disrupted using a FastPrep FP120 (Thermo Savant) cell disrupter. 150 mg of freeze-dried cell pellet, 500 mg 0.1 mm glass beads (BioSpec Products) and 1 ml disruption buffer were filled in conical 2 ml tubes with screw caps. The disruption buffer was adjusted to pH 7.4 and contained 50 mM Tris and a cOmplete mini EDTA-free protease inhibitor cocktail tablet (Roche) per 10 ml buffer. In the first step the sample was mixed for 2 s at 6.5 m/s and cooled on ice. For cell disruption cells were homogenized thrice for 20 s at 6.5 m/s and cooled on ice between steps.

Cytosolic proteins were extracted with a modified protocol of Watt et al. (2005) in which phenol instead of acetone was used. Taken together, the main steps included a removal of cell debris by centrifugation, phenol extraction, methanol precipitation and several washing steps with 70% (v/v) ethanol and acetone.

### 2.6. Extraction of proteins from the supernatant of Actinoplanes sp. SE50/110 cultures

For extraction of extracellular proteins the method developed by Watt et al. (2005) for X. campestris was used. Here, freeze-dried supernatant described in Section 2.3 was used for protein extraction. The main steps included a separation of cells by filtration, phenol extraction, methanol precipitation and several washing steps with 70% EtOH.

### 2.7. Separation of cytosolic and extracellular proteins of Actinoplanes sp. SE50/110 cultures by 2D IEF-SDS-PAGE

The quantification and separation of cytosolic and extracellular proteins has been conducted with the same protocols. Pellets of purified proteins were rehydrated overnight in 500 μl rehydration buffer consisting of 8 M urea and 0.2% CHAPS and quantified using the Bradford assay (Bradford, 1976). For IEF in both cases 450 μg of rehydrated protein were employed and filled up with rehydration buffer to a total volume of 340 μl. Then, 2.5 μl of 28% dithiothreitol (DTT) were added and the mixture was incubated for 30 min. Thereafter, 2.5 μl of IPG Buffer pH 3–10 NL (GE Healthcare) were added and the mixture was centrifuged for 5 min at 14,000 rcf. To resolve the proteome 18 cm IPG strips with an effective pH range of 4.3–8.3 were used that so far showed the best overlap with the pI distribution in Actinoplanes. To obtain the mentioned range 24 cm Immobiline DryStrips (GE Healthcare) pH 3–10 NL were cut by 2 cm at the acidic and 4 cm at the basic end. IEF was performed using an IPGPhor (Amersham Bioscience) at 20°C and 20 μA per strip for 50,000 Vhr. The applied program included the following steps: 1 h of rehydration, 12 h at 30 V (constant), 1 h at 500 V (constant), 8 h at 1000 (gradient), 3 h at 8000 V (gradient) and ca. 3 h at 8000 V (constant).

After IEF, strips were prepared for SDS-PAGE. Therefore, strips were incubated twice for 15 min in equilibration buffer (6 M urea, 50 mM Tris, 20 g/l SDS, pH 8.8) that was supplemented by 2% DTT in the first and 2.5% iodacetamide in the second step. Separation in the second dimension was performed using a Protean II IXL Cell (Bio-Rad Laboratories) and 10.5% polyacrylamide gels. Tris-tricine was the chosen system with anode buffer consisting of 0.1 M Tris, pH 8.9 and cathode buffer consisting of 0.1 M Tris, 0.1 M tricine and 1% SDS. During the run an electric current of 30 mA per gel was applied until the bromophenol blue band reached the end of the gel (ca. 21 h). Colloidal Coomassie as described by Neuhoff et al. (1988) was used for protein staining. Each gel was shaken gently overnight on a KS 501 digital rotary shaker (IKA) with 350 ml of staining solution per gel. Gels were destained several times by gentle shaking in 500 ml of water until the background was clear. The resulting gels were scanned with a Power Look III scanner (Umax) and stored in 1% acetic acid.

### 2.8. In-gel protein digestion and identification by MALDI-TOF-MS and PMF

For protein identification, the basic protocols of Hansmeier et al. (2006) were applied. They relied on trypsin digestion and MALDI-TOF PMF identification. For this purpose the ultrafleXtreme (Bruker Daltonics) mass spectrometer and the Mascot software (Perkins et al., 1999) were used with following parameters: enzyme, trypsin; missed cleavages, 1; modifications, carbamidomethyl (C) and oxidation (M); peptide tolerance, ±100 ppm; mass values, MH⁺ and monoisotopic. For automated protein identification, Mascot was provided with all protein coding sequences of the Actinoplanes sp. SE50/110 genome, which have been determined earlier.

### 2.9. Bioinformatics tools used for analysis of cytosolic and extracellular proteins of Actinoplanes sp. SE50/110 cultures

For the analysis of mass spectrometry-based proteomics data BioTools (Bruker Daltonics) software and Qupe (Albaum et al., 2009) were used. For the analysis of secretion signal peptides SignalP (Petersen et al., 2011) was applied for prediction of secretion signals and cleavage sites, LipoP (Juncker et al., 2003) for cleaving enzymes, the TMHMM algorithm (Krogh et al., 2001) for transmembrane domains, TatP (Bendtsen et al., 2005) for the presence of a typical twin arginine translocation motif (RRxFLk) and the

---

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of Actinoplanes sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

G Model
BIOTEC-6161; No. of Pages 12

ARTICLE IN PRESS

S. Wendler et al. / Journal of Biotechnology xxx (2012) xxx–xxx

ExPASy ProtScale Tool according to Kyte and Doolittle (1982) for H-domains.

## 3. Results and discussion

### 3.1. Identification of acarbose gene cluster proteins in the cytosolic proteome of Actinoplanes sp. SE50/110

For the analysis of the cytosolic proteome Actinoplanes sp. SE50/110 cultures were grown in high-maltose minimal medium for 114 h. At the time point of sampling a cell dry weight of 16.37 g/l and an acarbose concentration of 1.14 g/l was determined in the Actinoplanes sp. SE50/110 cultures. Regarding that a wild type strain and a fully synthetic medium were used the values indicate that Actinoplanes sp. SE50/110 is growing well in this medium and that it produces a reasonable amount of acarbose.

In first experiments it was determined that more Acb-proteins were detected in the stationary than in the exponential growth phase (data not shown). Therefore cells from the stationary phase were chosen to show a proof of principle that acarbose metabolism can by studied using a proteomics approach. For the analysis of the proteome, stationary cells were harvested by centrifugation, washed, freeze-dried and mechanically disrupted. Cytosolic proteins were extracted with phenol, precipitated with methanol and separated by IEF/SDS-PAGE. For IEF 18 cm IPG strips with an effective pH range of 4.3–8.3 NL were custom-made by cutting 24 cm Immobiline DryStrips pH 3–10NL to 18 cm. This step was necessary because in prior experiments with commercially available 18 cm IPG strips no suitable pH range was found that separated the Actinoplanes proteome and especially the proteins of the acarbose gene cluster well (data not shown). After staining with colloidal Coomassie all cytosolic protein spots were picked for identification with MALDI-TOF PMF.

In Fig. 1 the cytosolic 2D gel with identified acarbose cluster proteins is shown. In total 318 protein spots were picked, of which 213 were classified as identified and 17 as uncertain by the BioTools Software using the Mascot search engine. Of the 22 acarbose cluster proteins, 9 were identified. The corresponding genes within the acarbose cluster are shown in Fig. 3. The Mascot scores of the identifications as well as a comparison of theoretical and observed pI and MW are summarized in Table 1. For the 9 identified proteins Mascot scores indicated a reliable identification. The observed pI and MW were also in good agreement with theoretical values, even though minor deviations were observed regarding pI for AcbN, AcbR & AcbV and regarding MW for AcbD & AcbV. The MW deviation of AcbD is probably due to incomplete denaturation that affects the migration in the polyacrylamide gel.

In the cytosolic gel, spot intensities and therefore the protein amounts of most Acb-proteins were relatively low compared to other cytosolic proteins (Fig. 1). Especially AcbR and AcbV were weak spots that were on the edge of the detection limit. AcbE and AcbZ on the other hand were arranged as “pearl chains” in several spots of same MW but different pI. More generally, pearl chains were observed for most proteins bigger than 90 kDa. These findings would favor a methodological artifact that is linked to protein size rather than a physiological cause, e.g. posttranslational modifications, as explanation.

The cytosolically most abundant acarbose cluster proteins were AcbE, AcbK and AcbZ. AcbK was identified by pI and MW and a distinct localization in relation to neighboring protein spots observed in prior experiments (data not shown). AcbE and AcbZ are according to their proposed function extracellular acarbose insensitive α-amylases respectively pullulanases. Moreover, both definitely have secretion signal peptides (Table 3) and were thus expected sooner in the supernatant than in the cytosol.

---

Fig. 1. Two-dimensional SDS-PAGE showing the cytosolic proteome of Actinoplanes sp. SE50/110. Actinoplanes sp. SE50/110 was cultivated for 4d in pre-culture in glucose containing complex medium. Main cultures were grown for 114h (stationary phase) in maltose containing minimal medium. Cytosolic proteins were isolated and separated using IEF with custom-made 18cm pH 4.3–8.3 IPG strips and 10% SDS-PAGE while staining of protein spots was performed using colloidal Coomassie. Proteins of picked spots were digested with trypsin, and identified by MALDI-TOF-MS and PMF. Identified acarbose gene cluster proteins are marked in the gel picture. Multiple identifications are indicated by numbers in brackets.

Taking the results of the cytosolic acarbose cluster proteins together, a proof of principle regarding the feasibility of 2D gel-based proteomic analysis of the acarbose biosynthesis can be stated. From a total of 22 acarbose cluster proteins, 9 were detected. Here the question arose why the remaining 13 were missed since we know from RNA-Seq experiments in high-maltose minimal medium that all acb cluster genes are transcribed (P. Schwientek, personal communication). For this observation there are at least two explanations. First, by prediction acarbose proteins were expected to be found in three different locations, the cytosol (16), membrane (3) and extracellular space (3). AcbWXY for example were expected to be located within the membrane and thus were not detectable by the methods applied in this study. The second explanation is based on the observation that the identified acarbose proteins have a relatively small abundance in comparison to the rest of the cytosolic proteins. It is likely that the unidentified acarbose proteins are only present in minute amounts that fall under the limit of detection.

### 3.2. Identification of acarbose gene cluster proteins in the extracellular proteome of Actinoplanes sp. SE50/110

For the analysis of the extracellular proteome Actinoplanes sp. SE50/110 cultures were grown in high-maltose minimal medium for 3d. At the time point of sampling a cell dry weight of 6.80 g/l and an acarbose concentration of 0.12 g/l were determined in the Actinoplanes sp. SE50/110 cultures. These values are significantly smaller than those observed for the cytosolic fraction which is due to higher culture volumes and shorter cultivation.

For extraction of extracellular proteins from Actinoplanes sp. SE50/110, the supernatants of three 100 ml stationary phase cultures was filtered and freeze-dried. Then extracellular proteins were extracted using phenol, precipitated with methanol and

---

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of Actinoplanes sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

G Model
BIOTEC-6161; No. of Pages 12

ARTICLE IN PRESS

S. Wendler et al./Journal of Biotechnology xxx (2012) xxx–xxx

Table 1  
Acarbose cluster proteins identified in the cytosolic and extracellular fraction of an *Actinoplanes* sp. SE50/110 culture.

| Protein | Theoretical<sup>a</sup> | Observed<sup>a</sup> | Identified<sup>b</sup> | Mascot Score<sup>c</sup> | Postulated function according to Wehmeier and Piepersberg (2004) | Postulated function according to Schwientek et al. (2012) |
| --- | --- | --- | --- | --- | --- | --- |
|  | pI/- | MW/kDa | pI/- | MW/kDa | spot picks | PMF |
| Cytosolic fraction |  |  |  |  |  |  |
| AcbB | 5.9 | 34.7 | 6.0 | 35 | 1 | 159 | dTDP-glucose | dTDP-D-glucose |
|  |  |  |  |  |  |  | 4,6-dehydratase | 4,6-dehydratase |
| AcbD | 5.2 | 78.0 | 5.1 | 90 | 1 | 248 | Acarviosyltransferase | Glycosidases |
| AcbE | 5.7 | 111.0 | 5.5 | 115 | 3 | 230,103,86 | α-Amylase | Glycosidases |
| AcbK<sup>d</sup> | 5.5 | 31.5 | 5.5 | 30 | – | – | Acarbose-7-kinase | Sugar kinases, ribokinase family |
| AcbL | 6.6 | 37.7 | 6.9 | 38 | 1 | 129 | (Polyol) dehydrogenase | Threonine dehydrogenase and related Zn-dependent dehydrogenases |
| AcbN | 6.5 | 26.5 | 6.8 | 28 | 1 | 154 | Cyclitol 7-kinase | Dehydrogenases with different specificities |
| AcbR | 5.8 | 39.6 | 6.1 | 42 | 1 | 271 | GlgC-related NDP-polyol synthase | ADP-glucose pyrophosphorylase |
| AcbV | 6.7 | 48.1 | 6.3 | 43 | 1 | 72 | GabT-like aminotransferase | 4-Aminobutyrate aminotransferase and related aminotransferases |
| AcbZ | 6.3 | 115.4 | 6.2 | 114 | 7 | 230,202,142,117,116, 104,42 | α-Amylase | Glycosidases |
| Extracellular fraction |  |  |  |  |  |  |
| AcbD | 5.2 | 78.0 | 4.9–5.8 | 78 | 19 | 6×140–120, 5×100–80, 8×80–60 | Acarviosyltransferase | Glycosidases |
| AcbE | 5.7 | 111.0 | 5.5–6.3 | 110 | 7 | 143,140,128,126,95,56, 38 | α-Amylase | Glycosidases |
| AcbZ | 6.3 | 115.4 | 7.1–7.4 | 110 | 3 | 69,60,59 | α-Amylase | Glycosidases |

a Theoretical and observed isoelectric point (pI) and molecular weight of analyzed protein spots.  
b Number of protein spot pickings in which a protein was identified. Because some protein spots were picked and identified by PMF several times, this number is not equivalent to the number of protein spots of a respective protein on the gel.  
c Mascot score of the peptide mass fingerprint (PMF) identification.  
d AcbK was identified by pI and MW and a distinct localization in relation to neighboring protein spots observed in prior experiments.

separated by IEF/SDS-PAGE using custom-made 18cm pH 4.3–8.3 NL IPG strips as described in Section 3.1. The resulting gel was stained with colloidal Coomassie and the proteins were identified by MALDI-TOF PMF. The analysis of the 100 protein spots on the gel resulted in 58 total and 22 non-redundant protein identifications. The respective 2D gel is shown in Fig. 2. Abundant spots like AcbD or Cgt (cyclomaltodextrin glucanotransferase) were picked several times to increase the likelihood of detecting proteins that may be overlaid by dominant proteins. Spots arranged in pearl chains were picked to elucidate whether those belong to one or several different proteins. Altogether 22 proteins were identified. All three as extracellular annotated Acb-proteins, AcbD, AcbE and AcbZ, were detected in the secretome (Fig. 3). AcbD and AcbE were two dominant proteins and present in higher abundances than AcbZ. Since all extracellularly expected acb-proteins were detected, the findings prove that the chosen experimental approach is suitable to study the extracellular part of acarbose-metabolism. Table 1 provides an overview of observed and theoretical pI and MW, respective Mascot scores and number of pickings for identified acarbose cluster proteins. Here, AcbZ showed the lowest Mascot scores and appeared at a slightly different pI in the extracellular fraction. However, the observed pI and MW of acarbose cluster proteins from the extracellular fraction are in good agreement with the expected values. The Mascot scores also indicate reliable identifications.

In addition to AcbD, AcbE and AcbZ, the remaining acarbose cluster genes were analyzed regarding presence of signal peptides. SignalP and SecretomeP predicted secretion signal peptides for the two acarbose exporter components AcbY and AcbX, which according to TMHMM also had 5 respectively 6 transmembrane helices. Since only AcbD, AcbE and AcbZ were present in the supernatant, prediction and experimental data concurred well.

The spatial arrangement of AcbD, AcbE and AcbZ was consistent with the findings of the cytosolic proteome (compare Figs. 1 and 2). AcbE and AcbZ were arranged as pearl chains, while AcbD was mainly concentrated at one pI in one spot. Beyond that, an interesting observation was made regarding the relative abundances that were estimated from the particular spot intensities of AcbD, AcbE and AcbZ in the cytosolic and extracellular fraction. AcbD and AcbE were present in higher abundance than AcbZ in the supernatant, while the opposite was observed in the cytosol. Since AcbD, AcbE and AcbZ are central to the carbophore model; the findings can be used to test its consistency. AcbD was proposed to have different functions. In the cytosol it was suggested as one possible enzyme responsible for the final step of the biosynthesis of acarbose (Zhang et al., 2002) or even the last step of the export (Hemker et al., 2001). Extracellularly, AcbD was predicted to catalyze the synthesis of acarbose homologs and it was experimentally proven that AcbD links mono- and oligosaccharides to acarbose (Hemker et al., 2001; Leemhuis et al., 2004). Since AcbD is very dominant in the supernatant, our findings support the extracellular role as an acarviosyltransferase. AcbE shows a similarly high abundance to AcbD. An interesting observation is the difference in relative abundance of AcbE, AcbD and AcbZ in the intracellular and extracellular fraction. While AcbZ is clearly the most abundant cytosolic acarbose protein, AcbD and AcbE are present in much higher abundance extracellularly.

3.3. Characterization of non-acarbose gene cluster proteins in the extracellular proteome of *Actinoplanes* sp. SE50/110

3.3.1. Assigning of protein spots to the corresponding genes of the *Actinoplanes* sp. SE50/110 genome

In the secretome of *Actinoplanes* sp. SE50/110 22 proteins were identified. Out of the 22 about 7 proteins were clearly dominant

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of *Actinoplanes* sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

Table 2  
Non-acarbose cluster proteins identified in the extracellular fraction of an Actinoplanes sp. SE 50/110 culture.

| Protein class                          | Protein   | Theoretical<sup>a</sup> | Observed<sup>a</sup> | Identified spot picks<sup>b</sup> | Mascot Score<sup>c</sup> | Postulated function according to Schwientek et al. (2012)                                                                 |
|----------------------------------------|-----------|------------------------|---------------------|-------------------------------|-------------------------|-----------------------------------------------------------------------------------------------------------------------|
|                                        |           | pI/-                   | pI/-                | MW/kDa                        | PMF                     |                                                                                                                       |
| Carbohydrate binding proteins          | Gsl       | 6.2                    | 6.0                 | 58                            | 1                       | 63                                                        | β-Galactoside-specific lectin                                                                                           |
|                                        | Cgt       | 8.1                    | 7.9-8.3, 5.9-6.3    | 15                            | 5                       | 128, 128, 127, 118, 116                                    | 3/Ricin-type beta-trefoil-carbohydrate-binding domain                                                                  |
|                                        | CbpA<sup>d</sup> | 5.4                    | 4.9-6.0             | 29                            | 6                       | 6×85-65                                                   | Glycosidases-family 20 carbohydrate-binding module (CBM 20)/starch-binding domain Insignificant homology to a carbohydrate binding domain# |
| Carbohydrate cleavage proteins         | ChiA      | 6.3                    | 6.2                 | 46                            | 2                       | 75, 46                                                    | Chitinase                                                                                                               |
|                                        | PulA      | 6.0                    | 6.2-7.4, 5.8-6.1, 5.8-6.1 | 190, 110, 60                  | 17                      | 124, 100, 4×90-70, 8×70-50, 58, 39, 29                     | α-1, 6-Glucosidase, pullulanase PulA and related glycosidases                                                          |
| Carbohydrate transport proteins        | AglE      | 5.7                    | 5.1-5.8             | 46                            | 5                       | 147, 80, 75, 67, 63                                       | ABC-type sugar transport system, α-glucosides-binding extracellular protein aglE                                      |
|                                        | MalE      | 8.1                    | 6.3, 7.0, 7.8       | 44                            | 3                       | 190, 171, 139                                             | Maltose-binding extracellular protein                                                                                  |
| Protein cleavage proteins              | Apy       | 5.7                    | 4.9-5.2             | 52                            | 3                       | 115, 73, 66                                               | Predicted aminopeptidase                                                                                               |
|                                        | Zmp       | 5.0                    | 4.9                 | 79                            | 1                       | 192                                                       | Zinc metalloprotease (elastase)                                                                                       |
|                                        | Ssp       | 5.0                    | 4.7, 4.9, 5.6       | 105, 84                       | 2                       | 53, 39                                                    | Subtilisin-like serine proteases                                                                                      |
|                                        | PepX      | 5.9                    | 5.1                 | 42                            | 1                       | 79                                                        | Predicted acyl esterase, X-Pro dipeptidyl-peptidase                                                                   |
| Amino acid degradation proteins        | TynA      | 5.6                    | 6.0-6.3             | 51                            | 3                       | 96, 66, 38                                                | Cu<sub>2</sub><sup>+</sup>-containing amine oxidase                                                                      |
| Nucleic acid cleavage proteins         | NucE      | 4.9                    | 4.6                 | 150, 88, 70                   | 3                       | 74, 56, 28                                                | Predicted extracellular nuclease                                                                                      |
| Other & similar to proteins of unknown function | Eno       | 4.6                    | 4.8                 | 46                            | 1                       | 100                                                       | Enolase                                                                                                                |
|                                        | Stpk      | 6.8                    | 6.7-6.9             | 82                            | 1                       | 69                                                        | WD-40 repeat-containing serine/threonine protein kinase/signal transduction                                              |
|                                        | CirA<sup>e</sup> | 4.9                    | 5.3                 | 18                            | 1                       | 64                                                        | Iron uptake                                                                                                            |
|                                        | CsdE      | 5.9                    | 6.1                 | 39                            | 1                       | 57                                                        | Cell shape determining protein                                                                                         |
|                                        | ACPL_2791 | 4.9                    | 4.6                 | 13                            | 1                       | 154                                                       | Homologue to extracellular protein precursor                                                                           |
|                                        | ACPL_1442 | 6.8                    | 6.9                 | 44                            | 1                       | 151                                                       | Protein of unknown function                                                                                            |

a Theoretical and observed isoelectric point (pI) and molecular weight of analyzed protein spots.  
b Number of protein spot pickings in which a protein was identified. Because some protein spots were picked and identified by PMF several times, this number is not equivalent to the number of protein spots of a respective protein on the gel.  
c Mascot score of the peptide mass fingerprint (PMF) identification.  
d CbpA (ACPL_3368) showed no significant match to a known protein, but an insignificant Pfam-A match with a carbohydrate binding domain (family 25) with an E-value of 0.049.  
e The closest homologue is the CirA protein of *E. coli* involved in iron uptake. In addition, an insignificant Pfam-A match with the carbohydrate binding domain (family 25) with an E-value of 0.63 was detected.

G Model
BIOTEC-6161; No. of Pages 12

ARTICLE IN PRESS

S. Wendler et al. / Journal of Biotechnology xxx (2012) xxx–xxx

Table 3  
Analysis of leader peptide sequences of extracellularly detected proteins of *Actinoplanes* sp. SE50/110 grown in high-maltose minimal medium.

| Protein class                     | Protein | SPase I/II$^{1}$ | Secretion signal peptide$^{2}$                                                                 |
|-----------------------------------|---------|------------------|--------------------------------------------------------------------------------------------|
| Acarbose cluster proteins          | AcbD    | SPase I          | MQ$\underline{\text{R}}$HARHAI$\underline{\text{A}}$AAVASLLPPSLPAHAA($\varnothing_{27}$)GA           |
|                                   | AcbE    | SPase I          | MD$\underline{\text{ER}}$LKSHIVRALLPVLVGAAGGLPAPRAAQ$\underline{\text{A}}$( $\varnothing_{32}$)ES       |
|                                   | AcbZ    | SPase I          | MVLSRH$\underline{\text{R}}$DV$\underline{\text{M}}$RR$\underline{\text{L}}$SAVLALGLVVPLLLAPPWRAS$\underline{\text{A}}$( $\varnothing_{35}$)AP |
| Carbohydrate binding proteins      | Gsl     | SPase I          | MKSVLGARIRLTRRCLAAA$\underline{\text{A}}$AVSVLLVPARA$\underline{\text{A}}$AQ$\underline{\text{A}}$( $\varnothing_{34}$)TE |
|                                   | Cgt     | SPase I          | MNRTTVRAGVLATALISGV$\underline{\text{L}}$GVAGPAL$\underline{\text{A}}$( $\varnothing_{28}$)AP            |
|                                   | CbpA    | SPase I          | MS$\underline{\text{R}}$ILRRVLLLA$\underline{\text{A}}$APVAALGFLAAP$\underline{\text{A}}$$\underline{\text{A}}$$\underline{\text{A}}$( $\varnothing_{28}$)SP |
| Carbohydrate cleavage proteins     | ChiA    | SPase I          | MIPSCARKAAPLFREGLRTRACRPSPVVLAVAALQVAAATLALPIATPAA$\underline{\text{A}}$AQ$\underline{\text{A}}$( $\varnothing_{53}$)AA |
|                                   | PulA    | SPase I          | MAASKTRWGTAVLSLLLPLLTVPA$\underline{\text{A}}$EASA($\varnothing_{29}$)AS                              |
| Carbohydrate transport proteins    | AglE    | SPase II & I     | MFGHSGSSRSVALAGALSAGLIFS$\underline{\text{L}}$AA($\varnothing_{27}$)CG                                |
|                                   | MalE    | SPase II & I     | MRIRTAGVIAASMLCLVG$\underline{\text{A}}$$\underline{\text{A}}$$\underline{\text{A}}$( $\varnothing_{21}$)CG |
| Protein cleavage Proteins          | Apy     | SPase I          | MRKRTLAVPFAAALT$\underline{\text{L}}$ALLPQP$\underline{\text{A}}$RA($\varnothing_{25}$)VD             |
|                                   | Zmp     | SPase I$^{\prime}$| VTATGARAAAVKDVLGTSTAK$\underline{\text{A}}$( $\varnothing_{22}$)TK                                 |
|                                   | Ssp     | SPase I          | MRQPSRWLT$\underline{\text{S}}$VLAIGLAAGVTTVNGSLPAT$\underline{\text{A}}$APA($\varnothing_{34}$)AP   |
|                                   | PepX    | SPase I          | MSI$\underline{\text{R}}$RATAAAAGALITVAALAVGGPATA($\varnothing_{28}$)AA                               |
| Amino acid degradation proteins    | TynA    | SPase I          | MTSAALGLTVPAQA$\underline{\text{A}}$APA($\varnothing_{17}$)AS                                      |
| Nucleic acid cleavage proteins     | NucE    | SPase I          | MRPYLRSVSAAAPQAIRALGAAALATALGFGFSGPAFA($\varnothing_{39}$)AE                                        |
| Other & similar to proteins of unknown function | Eno    | -              | MATIEAIVAREILDSRGNPTEVEVGLDDGTVGRAAVPSGASTGA                                                |
|                                   | Stpk    | -                | MRYDA(...)$\underline{\text{R}}$MRR$_{191}$GALA$\underline{\text{V}}$VCALLLLVAGTAVA$\underline{\text{W}}$R$_{2120}$NAR |
|                                   | CirA    | SPase I          | MPRTIAVIILASAATLAALAGPAQA($\varnothing_{25}$)ST                                                 |
|                                   | CsdE    | -                | MHDT(...)$\underline{\text{G}}$RRR$_{40}$VAAGAGAAVCLAIALATMFRVV/GFG$_{640}$STA                        |
|                                   | ACPL_2791 | SPase I         | MASRTVSAAVALVGAAGLSLAAAP$\underline{\text{A}}$QA($\varnothing_{28}$)AT                             |
|                                   | ACPL_1442 | SPase I         | MPKVSR$\underline{\text{R}}$RRLAVVAAMLVGAVTMGAGPGRAEA($\varnothing_{33}$)AA                          |

1) The cleaving enzyme was predicted with LipoP. As expected, cleavage by SPase II had a higher probability than cleavage by SPase I at the same position for the lipoproteins AglE and MalE. For Zmp, the cleavage enzyme was predicted with SignalP, because LipoP showed no positive prediction regarding the cleaving enzyme.

2) The cleavage site was determined with SignalP. Residues at positions $-3$ to $-1$ relative to the predicted cleavage sites are bold and underlined. The cleavage sites within the amino acid sequence are accentuated by ($\varnothing_{\#}$) with subscript numbers indicating the last amino acid of the signal peptide. The hydrophobic H domains were determined with the ExPASY ProtScale Tool according to Kyte and Doolittle (1982) and are shaded gray. Twin arginine motifs (RRxFLk) were predicted with TatP and are printed bold. As positively charged amino acids arginine (R) and lysine (K) are printed in italics and underlined. Transmembrane helices were determined by the TMHMM algorithm and are printed in italics.

(Fig. 2). Moreover, many proteins were found in several spots and different spatial arrangements. The three extracellular acarbose cluster proteins AcbD, AcbE, and AcbZ were discussed above (Section 3.2). The remaining 19 proteins with respective functions, Mascot scores as well as theoretical and observed pI are listed in Table 2. Altogether 8 of 22 proteins identified in the extracellular fraction, namely AcbD, AcbE, AcbZ, enolase, CbpA (ACPL_3368, carbohydrate binding protein A), and three proteases Apy, Zmp, and Ssp, were also present in the cytosol. However, a correlation of protein abundance and extracellular localization was not the case (Fig. 2) excluding cell leakage or disrupted cells as an explanation for the extracellular identifications. The observed MWs and pI were in most cases in good agreement with theoretical values. The two exceptions regarding both were PepX and CirA. Alternative start codons or mRNA starts that would explain the smaller size were excluded by manual revision of data generated in RNA-Seq experiments (P. Schwientek, personal communication). Degradation of those proteins by the extracellular proteases on the other hand could be an explanation (Table 2). Beyond that, NucE and PulA were found at several different molecular weights that included the expected value. The same was the case for AcbD with the difference that AcbD was mainly concentrated in one spot. Here, degradation by extracellular proteases could be an explanation. Since PulA is a rather big enzyme (191 kDa) and has 6 different functional domains distributed throughout the whole sequence, it is possible that the observed fragments (110 kDa and 60 kDa) retain operative domains and do not lose functionality.

Interestingly, many secreted proteins were arranged as pearl chains of the same MW but different pI, by which the theoretical value was usually covered. This was especially the case for PulA, but also for AcbE and MalE. The latter also showed a bigger pI difference between the spots. Similar pearl chains were observed in the secretome of *B. subtilis* and thought to be artifacts of IEF (Antelmann et al., 2001). However, we do not exclude a physiological cause as for example posttranslational modifications, e.g. glycosylation.

3.3.2. Characterization of proteins identified in the supernatant of *Actinoplanes* sp. SE50/110

18 of the 19 non-acarbose cluster proteins detected in the supernatant showed significant sequence similarities to known proteins, even though one was similar to a protein of unknown function and another similar to a class of functionally undescribed protein precursors. The only protein without a significant match showed an insignificant Pfam-A match to a carbohydrate binding domain (family 25) with an E-value of 0.049. Since this was the best functional prediction available, it was annotated as CbpA (ACPL_3368, carbohydrate binding protein A). In the next step, the identified proteins were categorized into groups of similar function (Table 2). Correlating the function and abundance of proteins it became clear that all proteins that have an assigned function and high abundance are involved in carbohydrate metabolism (Fig. 2). Since this seems to be of special importance, respective findings are discussed separately in Section 3.4. The next class of enzymes is dedicated to protein metabolism and shows a much lower abundance. Here, 4 proteases were identified with Zmp as the most dominant protein. The presence of proteases could be one of the reasons for the observed protein degradation. The observation that PulA appeared at three distinctive MWs was probably associated with a

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of *Actinoplanes* sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

G Model
BIOTEC-6161; No. of Pages 12

ARTICLE IN PRESS

S. Wendler et al. / Journal of Biotechnology xxx (2012) xxx–xxx

of a typical twin arginine translocation motif (RRxFLk) and ProtScale for H-domains. The results indicate that 19 of 22 proteins had a secretion signal and were cleaved either by SPase I or II. According to LipoP MalE and AglE were the only lipoproteins having a SPase I and II cleavage sites with a higher probability for cleavage by SPase II. Additionally, all proteins with a secretion signal had a hydrophobic domain (H-domain) while three showed a twin arginine motif. Two of the remaining three proteins showed an N-terminal transmembrane domain typically seen in membrane anchored proteins and should therefore be located mainly outside of the cell. Their presence in the supernatant is likely due to mechanical shearing. The only protein lacking a signal peptide and an N-terminal transmembrane domain was enolase, which, even though lacking both, is regularly found extracellularly in Gram-positive bacteria (Pancholi and Fischetti, 1998; Antelmann et al., 2001; Hansmeier et al., 2006; for review see Pancholi, 2001). In pathogenic bacteria enolase function is considered to be linked to bacterial adhesion during host-pathogen interactions by binding plasminogen (Pancholi and Fischetti, 1998; Pancholi, 2001). The function in non-pathogenic Gram-positive soil bacteria as Actinoplanes sp. SE50/110, C. glutamicum or B. subtilis is not known, but propositions were made that it could facilitate survival in natural soil habitats as it does for pathogenic bacteria in hosts (Antelmann et al., 2001).

3.4. Extracellular proteins of Actinoplanes sp. SE50/110 and their role in carbohydrate metabolism

The carbohydrate metabolism group of extracellular proteins was the most abundant in terms of number of identifications implying physiological relevance under the studied conditions. In total 10 of 22 extracellular proteins were attributed to this group that included the acarbose cluster proteins, but excluded enolase due to its unclear role in the supernatant. In Table 2 the respective proteins were further subdivided into a binding, cleavage and transport subgroup. An analysis of the abundance of proteins revealed that the 7 actually dominant and identified proteins all belonged to the carbohydrate metabolism group (PulA, AcbE, AcbD, MalE, AglE, CbpA and Cgt). The three remaining abundant protein spots outside of the carbohydrate metabolism category (MW 46 kDa/pI 6.0, MW 22 kDa/pI 4.9 and MW 22 kDa/pI 4.7) could not be identified by MALDI-TOF-MS and PMF, which is likely due to annotation that missed the respective genes.

3.4.1. Extracellular carbohydrate binding proteins

Within the carbohydrate binding group the two relatively small proteins Cgt (15.2 kDa) and CbpA (29.5 kDa) were produced in high amounts. The abbreviation Cgt stands for cyclomaltodextrin

Fig. 2. Two-dimensional SDS-PAGE showing the extracellular proteome of Actinoplanes sp. SE50/110. Actinoplanes sp. SE50/110 was cultivated for 3 d in pre-culture in glucose containing complex medium. Main cultures were grown for 72 h (stationary phase) in maltose containing minimal medium. Extracellular proteins were isolated and separated using IEF with custom-made 18 cm pH 4.3–8.3 IPG strips and 10% SDS-PAGE while staining of protein spots was performed using colloidal Coomassie. Proteins of picked spots were digested with trypsin, and identified by MALDI-TOF-MS and PMF. Identified proteins are marked in the gel picture. Multiple identifications are indicated by numbers in brackets.

specific cleavage by an extracellular protease that may be relevant for its activity. Aside from that, several low abundance proteins were identified that covered different functionalities like amino or nucleic acid degradation, cell shape determination, signal transduction or iron uptake (Table 2).

3.3.3. Analysis of signal peptides and transmembrane helices of proteins identified in the supernatant of Actinoplanes sp. SE50/110

The 22 proteins detected in the supernatant were analyzed regarding the presence of secretion signals or N-terminal transmembrane domains that could explain their localization. The results of this structural analysis are presented in Table 3. Different software and algorithms were used for predictions, SignalP for secretion signals and cleavage sites, LipoP for cleaving enzyme, the TMHMM algorithm transmembrane domains, TatP for the presence

cytosol
extracellular space

![Diagram](attachment:diagram.png)

Fig. 3. The cytosolic and extracellular localization of proteins encoded by genes located in the acarbose gene cluster. In addition to the genes of the acarbose gene cluster of Actinoplanes sp. SE50/110, the location of the gene products is indicated by a star symbols. Gene products of *acbE*, *acbD* and *acbZ* were identified in the cytosol and supernatant. The figure presented is a modified version of a figure published by Schwientek et al. (2012).

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of Actinoplanes sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

G Model
BIOTEC-6161; No.of Pages 12

ARTICLE IN PRESS

S. Wendler et al. / Journal of Biotechnology xxx (2012) xxx–xxx

![Diagram](attachment:diagram.png)

Fig. 4. Extended model of acarbose metabolism in *Actinoplanes* sp. SE50/110. The presented model is based on the proposed biosynthesis model according to Zhang et al. (2002) and the proposed carbophore role of acarbose according to Wehmeier and Piepersberg (2004). Beyond that, the extended model represents an update according to recent studies (Brunkhorst and Schneider, 2005; Rockser and Wehmeier, 2009; Licht et al., 2011; Vahedi-Faridi et al., 2010) and results of this work. New elements are the pullulanase PulA, and the two ABC transport systems MalEFG and AglEFG. The MalEFG transport system is suggested to be the acarbose importer in this model. Acarbose cluster proteins identified in the cytosol and extracellular space identified in this study are highlighted.

glucanotransferase and was attributed to the Cgt protein by annotation but may be misleading, since cyclomaltodextrin glucanotransferases are generally enzymes with several domains and high molecular weights. A closer analysis on Cgt from *Actinoplanes* sp. SE50/110 with Pfam-A revealed that almost the whole amino acid sequence codes for a carbohydrate binding domain. A closer look on the neighboring genes of *cgt* and *cbpA* did not exhibit the presence of clusters/operons or further specified the role of these proteins. Data generated in RNA-Seq experiments showed that the mRNAs of both genes were also found in high abundance in experiments from cells grown in high-maltose minimal medium (CbpA: Mal-MM 747 reads per kilobase of coding sequence per million mapped reads (RPKM); Mal-MM-TE 425 RPKM; Cgt: Mal-MM 11,098 RPKM; Mal-MM-TE 5861 RPKM), but were poorly expressed in glucose-grown cells (CbpA: Glc-CM 5 RPKM; Cgt: Glc-CM 91 RPKM) (P. Schwientek, personal communication). This observation hints at a putative function of Cgt and CbpA within the maltose/maltodextrin respectively starch metabolism.

3.4.2. Extracellular carbohydrate degradation enzymes

The subgroup of the carbohydrate cleavage enzymes included the two acarbose-insensitive alpha amylases AcbE and AcbZ as well as PulA (pullulanase) and the chitinase ChiA (Table 2). Of those, only PulA and AcbE were actually abundant. A Pfam-A analysis of PulA, AcbE, and AcbZ showed that all had α-amylase and pullulanase-associated domains. While the number of α-amylase domains varied, all three had two adjacent pullulanase-associated domains. Pullulanases are acarbose-insensitive enzymes which hydrolyze pullulan to maltotriose and multiples thereof (Bender et al., 1959; Wallenfels et al., 1969; Catley, 1971; Truscheit et al., 1981). Since all three enzymes, AcbE, AcbZ, and PulA also possess α-amylase domains, it is likely that they all have similar enzymatic activities. For the enzymes AcbE and AcbZ, it was already shown that they are able to hydrolyze starch and degrade pullulan, but, in contrast to other pullulanases, to maltose or glucose (Merettig, 2009). Thus one can expect a similar degradation pattern from PulA, too.

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of *Actinoplanes* sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

GModel
BIOTEC-6161; No. of Pages 12

ARTICLE IN PRESS

S. Wendler et al. / Journal of Biotechnology xxx (2012) xxx–xxx

3.4.3. The α-glucoside import system aglEFG

A protein annotated as the solute-binding protein AglE (α-glucosides-binding protein) was detected in the supernatant of Actinoplanes sp. SE50/110 cultures grown on high-maltose minimal medium. Solute-binding proteins (SBP) recognize substrates as carbohydrates, vitamins and inorganic ions with high affinities and are anchored in the membrane in Gram-positives. After recognition SBPs deliver the respective substrates to the cognate transport complex (Wilkinson and Verschueren, 2003). In general, SBPs show a highly conserved structural fold, but low sequence similarities (Berntsson et al., 2010; Sutcliffe and Russell, 1995).

A closer look on the genetic locus in Actinoplanes sp. SE50/110 revealed that the aglE gene is organized in a cluster together with two genes annotated as aglG and aglF which code for the permease components AglF and AglG of an ABC (ATP binding cassette) transport system. An AglEFG system was first detected in Sinorhizobium meliloti by Jensen et al. (2002) and constitutes a multiple α-glucoside ABC transporter for maltose, trehalose, sucrose and trisaccharides. In an in silico analysis of the carbohydrate uptake systems of Streptomyces coelicolor an aglEFG cluster was detected that, as it is also the case in Actinoplanes sp. SE50/110, lacks adjacent regulatory or catabolic α-glucosidase genes. In Ralstonia eutroplus the in silico predicted function of the AglEFG transporter was proven by heterologous expression and growth on the respective α-glucosides (Bertram et al., 2004). The importance of these findings is accentuated by the fact that maltose has a bifunctional role as an inducer of the acb-genes and also as a precursor for the biosynthesis of acarbose. We suppose that the protein MsiK (multiple sugar import protein) acts as the ATPase subunit. The ATP-binding component named MsiK was first detected in Streptomyces lividans (Hurtubise et al., 1995) and represents a universal ATPase subunit involved in many transport systems. An msiK-gene was identified in the Actinoplanes sp. SE50/110 genome while the corresponding mRNA was detected in RNA-Seq data generated in experiments of maltose and glucose grown cells (msiK: Mal-MM 343 RPKM; Mal-MM-TE 344 reads; Glc-CM 349 RPKM) (P. Schwientek, personal communication).

Brunkhorst and Schneider (2005) analyzed the maltose/maltodextrin transport in Actinoplanes sp. under congruent experimental conditions with this study, namely a minimal medium that is supplemented with maltose respectively maltodextrin as the carbon source. Based on their transport studies using different competing sugars including maltose, maltodextrin and acarbose Brunkhorst and Schneider (2005) proposed that Actinoplanes sp. has two dominant maltose transport systems. This observation is consistent with our findings. In addition, Brunkhorst and Schneider (2005) proposed that one system is an acarbose-sensitive maltose/maltodextrin transporter and the other an acarbose-insensitive maltose/sucrose/trehalose transporter. Due to the congruent experimental setups we propose that the acarbose-insensitive transporter functionally characterized by Brunkhorst and Schneider (2005) is the α-glucoside transport system AglEFG. The maltose/sucrose/trehalose transporter identified in the study of Brunkhorst and Schneider (2005) was expressed at growth on different carbon sources except glucose, suggesting an unspecific induction and a glucose repression. Beyond that Brunkhorst and Schneider (2005) stated the assumption that the maltose/sucrose/trehalose importer is probably derived from or is similar to the AglEFG of Sinorhizobium meliloti (Jensen et al., 2002) respectively S. coelicolor (Brunkhorst and Schneider, 2005). The system annotated as AglEFG which was identified in this study and the characteristics of the maltose/sucrose/trehalose transporter described by Brunkhorst and Schneider (2005) are in good agreement. Moreover, we were not able to identify genes for further maltose import systems in the Actinoplanes sp. SE50/110 genome besides aglEFG and malEFG. In their study Brunkhorst and Schneider (2005) convincingly excluded the acarbose-insensitive maltose/sucrose/trehalose transporter as a potential acarbose importer by competitive transport studies using radioactively labeled sugars. Since we propose AglEFG of Actinoplanes sp. SE50/110 as the respective maltose/sucrose/trehalose transporter, we consequently exclude AglEFG as a potential acarbose importer.

3.4.4. The relation of the maltose/maltodextrin import system malEFG and acarbose-metabolite import

3.4.4.1. The genetic locus of the maltose/maltodextrin import system MalEFG. A dominant protein spot found in the supernatant of Actinoplanes sp. SE50/110 cultures was the putative maltose binding protein MalE. An analysis of the genomic locus around the malE-gene revealed that it is located in a cluster with pulA, malFG and malR, with malR being located on the opposite strand. According to the annotation MalE is a solute-binding protein of an ABC transporter system with high affinity to maltose and maltodextrins, MalF and MalG are the permease components and MalR a LacI family transcriptional regulator. Maltose/maltodextrin metabolism and uptake through the MalEFG import system was discovered and thoroughly studied in E. coli (for review see Bordignon et al., 2010). In Gram-positive bacteria on the other hand studies were not as numerous as for enterobacteria. However, the described maltose transport system including malEFG and divergently transcribed malR has been already observed in the related actinomycetes S. coelicolor (van Wezel et al., 1997a, 1997b), S. lividans (Schlösser et al., 2001) and other Gram-positives as A. acidocaldarius (Hülsmann et al., 2000). Beyond that, it is of special interest that the genes encoding the two extracellularly dominant proteins PulA and MalE are located in close genetic proximity.

3.4.4.2. A comparative analysis of the maltose/maltodextrin-binding proteins MalE of Actinoplanes sp. SE50/110, MalE of E. coli K12 and GacH of S. glaucescens GLA.O. It was shown that acarbose transport is accomplished by the MalEFG maltose/maltodextrin transport system in E. coli K12 (Brunkhorst et al., 1999). Beyond that, the GacH protein from S. glaucescens GLA.O represents also a maltose/maltodextrin-binding protein that prefers maltose, maltotetraose and higher homologs of acarbose (component 5C) over acarbose (Vahedi-Faridi et al., 2010). These data corresponded well with the carbophore model (Wehmeier and Piepersberg, 2004) since loaded acarbose-metabolites should be the preferred substrate for import. Studies using GacH crystals showed that the protein recognizes the maltose moiety of acarbose which explains why GacHFG can also import maltose, maltotetraose and higher homologs of acarbose (Vahedi-Faridi et al., 2010). The MalE protein of Actinoplanes sp. SE50/110 has a low identity of 25% to GacH as well as MalE of S. coelicolor A3(2). Proteins with significantly higher identities to GacH than MalE are not present in the genome. However, this is not a disprove that the Actinoplanes sp. MalE protein does not recognize and import maltose, maltodextrins or acarbose since solute-binding proteins are known for exhibiting low sequence similarities, while having highly conserved fold structures (Berntsson et al., 2010). Beyond that, similarities between E. coli K12 MalE, which has also a high affinity to acarbose (Vahedi-Faridi et al., 2010), and Actinoplanes MalE as well as S. glaucescens GLA.O GacH are reasonable low (25–32%) underlining the mentioned property of utilizing identical substrates due to highly conserved fold structures, while having low sequence similarities.

3.4.4.3. The ABC transporter MalEFG of Actinoplanes sp. SE50/110 and acarbose-metabolite import. An acarbose-sensitive maltose/maltodextrin transporter was the second transport system

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of Actinoplanes sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

G Model
BIOTEC-6161; No.of Pages 12

ARTICLE IN PRESS

S. Wendler et al. / Journal of Biotechnology xxx (2012) xxx–xxx

detected by Brunkhorst and Schneider (2005). This transporter system was characterized by applying competitive transport studies of radioactively labeled sugars. Brunkhorst and Schneider (2005) supposed that the acarbose-sensitive maltose/maltodextrin transporter system in Actinoplanes sp. was derived from or is similar to the MalE-dependent maltose/maltodextrin import system of E. coli, S. coelicolor A3(2) or S. lividans and suggested it as a possible acarbose-metabolite importer due its sensitivity to respectively competitive inhibition by acarbose. In general, it is reasonable to assume that Actinoplanes sp. SE50/110 has an acarbose importer. With recent knowledge, it is also reasonable to assume that the mechanism of acarbose import in Actinoplanes sp. SE50/110 is similar to the known GacH-dependent import in S. glaucescens GLA.O (Rockser and Wehmeier, 2009; Vahedi-Faridi et al., 2010). Due to the similar experimental setup of our study and the study of Brunkhorst and Schneider (2005) we propose that the acarbose-sensitive maltose/maltodextrin transporter is represented by the MalEFG ABC transport system. Consequently, the proposition also means that the MalEFG ABC transport system is involved in maltose/maltodextrin and also acarbose-metabolite import in Actinoplanes sp. SE50/110.

### 3.5. Extended model of the acarbose metabolism of Actinoplanes sp. SE50/110

The results of this work together with recent research on the acarbose metabolism are summarized in an updated and extended model shown in Fig. 4. The scheme depicts the intra- and extracellularly detected proteins of the acarbose biosynthesis and portrays the carbophore function of acarbose with its cycling of acarbose-derived metabolites between intra- and extracellular pools. Beyond that, the acarbose metabolism model is extended by new components identified in this study. The newly added components are PulA as well as the transport systems MalEFG and AglEFG. PulA was included in the extended model due to its abundance and domain structure similarity to AcbE and AcbZ (Section 3.3) and its adjacent location to the MalEFG transport system. A further functional characterization of the newly identified elements would certainly add value, but is still very challenging for the genetically inaccessible Actinoplanes sp. SE50/110.

The extended model suggests a plausible mechanism of the interaction of extracellular carbohydrates, secreted carbohydrate degradation enzymes, membrane transport of maltose/maltodextrin and acarbose-metabolites and the expression of the acarbose gene cluster in Actinoplanes sp. SE50/110.

The sequence of induction could proceed as follows: The initial maltose and maltotriose molecules are imported via AglEFG and can induce the expression of the mal and acb-cluster genes (Wehmeier, 2003). The respective extracellular glucosidases PulA, AcbE and AcbZ will then be synthesized, secreted and can hydrolyze starch and pullulan and provide additional maltose, maltotriose and other mono- and oligosaccharides. These degradation products could then be either substrates for import by AglEFG and MalEFG or serve as substrate for AcbD. AcbD can transfer the respective sugars to acarbose (Hemker et al., 2001; Leemhuis et al., 2004) and create a pool of higher acarbose-homologs that can be imported by the proposed acarbose importer MalEFG. MalEFG has as a bifunctional role as an acarbose-metabolite and maltose/maltodextrin importer. The biological benefit provided by acarbose seems to be a growth advantage for slow growing Actinoplanes sp. that is achieved by an acarbose-tagged pool of carbohydrates that inhibits various glycolytic enzymes and transporters of competitors and is exclusive for utilization to Actinoplanes sp.

### 4. Conclusions

The conducted research presented not only the first proteomic study in Actinoplanes sp. SE50/110 but also the genus Actinoplanes and the family Micromonosporaceae. Beyond that, the study established a proof of principle that proteomics techniques are appropriate tools to study acarbose metabolism by identifying a series of enzymes relevant for the biosynthesis and function of acarbose in Actinoplanes sp. In addition, based on the identifications in the extracellular proteome and RNA-Seq studies of P. Schwientek and colleagues (personal communication) propositions were made regarding the acarbose-metabolite importer, which is an essential element of the carbophore model.

However, even though it was shown that extracellular acarbose cluster proteins are dominant proteins of the secretome, it is still difficult to identify all cytosolic acarbose cluster proteins. So far, about half of the mentioned could not be detected. The three most obvious reasons for the unidentified acarbose proteins are the resolution capacity of the chosen pH range for IEF, the generally low abundance of acarbose cluster proteins and the localization (see also Section 3.1). To detect the remaining acb-cluster proteins we plan to use zoom-in IPG strips in the physiologically interesting pH spectrum (pH 4.7–5.9), to increase the total amount of protein applied and to include a membrane fraction in the analysis. In the long term, the relative amount of acarbose cluster to all proteins could be increased by optimizing medium and culture conditions for higher acarbose production respectively cluster expression. Another necessity is a systematic proteome study of the three sub-proteomes in different culture phases. Finally, even though some new elements were identified in this study, additional experiments will have to be conducted to provide functional proof within as well as a deeper insight into acarbose metabolism.

### Acknowledgments

S. Wendler acknowledges financial support from German National Academic Foundation. As associated members P. Schwientek, F. Schulte and S. Wendler also acknowledge support from the Graduate Cluster Industrial Biotechnology (CLIB²⁰²¹) at Bielefeld University, Germany. The Graduate Cluster is supported by a grant from Bielefeld University and from the Ministry of Innovation, Science and Research (MIWF) of the federal state North Rhine-Westphalia, Germany. A. Pühler, K. Niehaus and J. Kalinowski also acknowledge a grant from the MIWF entitled “Technologieplattform PolyOmics”. We also acknowledge the financial support from Bayer AG (Leverkusen, Germany).

### References

Albaum, S., Neuweger, H., Fränzel, B., Lange, S., Mertens, D., Trötschel, C., Wolters, D., Kalinowski, J., Nattkemper, T., 2009. Qupe-a Rich Internet Application to take a step forward in the analysis of mass spectrometry-based quantitative proteomics experiments. Bioinformatics (Oxford, England) 25 (23), 3128–3134.

Antelmann, H., Tjalsma, H., Voigt, B., Ohlmeier, S., Bron, S., van Dijl, J., Hecker, M., 2001. A proteomic view on genome-based signal peptide predictions. Genome Research 11 (9), 1484–1502.

Apeler, H., Wehlmann, H., Piepersberg, W., Diaz-Guardamino, P.-M., Jarling, M., Thomas, H. & Wehmeier, U., 2001. Neue Enzyme in der Acarbose-Synthese und deren Verwendung. German patent DEOS 10021667.

Bender, H., Lehmann, J., Wallenfels, K., 1959. Pullulan, ein extracelluläres Glucan von Pullularia pullulans. Biochimica et Biophysica Acta 36 (2), 309–316.

Bendtsen, J.D., Nielsen, H., Widdick, D., Palmer, T., Brunak, S., 2005. Prediction of twin-arginine signal peptides, 6. BMC Bioinformatics, p. 167.

Berntsson, R.P., Smits, S., Schmitt, L., Slotboom, D., Poolman, B., 2010. A structural classification of substrate-binding proteins. FEBS Letters 584 (12), 2606–2617.

Bertram, R., Schlicht, M., Mahr, K., Nothaft, H., Saier, M.H., Titgemeyer, F., 2004. In silico and transcriptional analysis of carbohydrate uptake systems of Streptomyces coelicolor A3(2). Journal of Bacteriology 186 (5), 1362–1373.

Bischoff, H., Ahr, H.J., Schmidt, D., Stoltefuß, J., 1994. Acarbose - ein neues Wirkprinzip in der Diabetestherapie. Nachrichten aus Chemie, Technik und Laboratorium 42 (11), 1119–1128.

---

Please cite this article in press as: Wendler, S., et al., The cytosolic and extracellular proteomes of Actinoplanes sp. SE50/110 led to the identification of gene products involved in acarbose metabolism. J. Biotechnol. (2012), http://dx.doi.org/10.1016/j.jbiotec.2012.08.011

Bordignon, E., Grote, M., Schneider, E., 2010. The maltose ATP-binding cassette transporter in the 21st century—towards a structural dynamic perspective on its mode of action. Molecular Microbiology 77 (6), 1354–1366.

Bradford, M.M., 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Analytical Biochemistry 72, 248–254.

Brunkhorst, C., Schneider, E., 2005. Characterization of maltose and maltotriose transport in the acarbose-producing bacterium Actinoplanes sp. Research in Microbiology 156 (8), 851–857.

Brunkhorst, C., Andersen, C., Schneider, E., 1999. Acarbose, a pseudooligosaccharide, is transported but not metabolized by the maltose-maltodextrin system of Escherichia coli. Journal of Bacteriology 181 (8), 2612–2619.

Catley, B.J., 1971. Utilization of carbon sources by Pullularia pullulans for the elaboration of extracellular polysaccharides. Applied Microbiology 22 (4), 641–649.

Degwert, U., van Hülst, R., Pape, H., Herrold, R.E., Beale, J.M., Keller, P.J., Lee, J.P., 1987. Studies on the biosynthesis of the alpha-glucosidase inhibitor acarbose: valienamine, a m-C7N unit not derived from the shikimate pathway. The Journal of Antibiotics 40 (6), 855–861.

Drepper, A., Pape, H., 1996. Acarbose 7-phosphotransferase from Actinoplanes sp.: purification, properties, and possible physiological function. The Journal of Antibiotics 49 (7), 664–668.

Hansmeier, N., Chao, T.-C., Pühler, A., Tauch, A., Kalinowski, J., 2006. The cytosolic, cell surface and extracellular proteomes of the biotechnologically important soil bacterium Corynebacterium efficiens YS-314 in comparison to those of Corynebacterium glutamicum ATCC 13032. Proteomics 6 (1), 233–250.

Hemker, M., Stratmann, A., Goeke Kl Schröder, W., Lenz, J., Piepersberg, W., Pape, H., 2001. Identification, cloning, expression, and characterization of the extracellular acarbose-modifying glycosyltransferase, AcbD, from Actinoplanes sp. strain SE50. Journal of Bacteriology 183 (15), 4484–4492.

Hurtubise, Y., Shareck, F., Kluepfel, D., Morosoli, R., 1995. A cellulase/xylanase-negative mutant of Streptomyces lividans 1326 defective in cellobiose and xylobiose uptake is mutated in a gene encoding a protein homologous to ATP-binding proteins. Molecular Microbiology 17 (2), 367–377.

Hülsmann, A., Lurz, R., Scheffel, F., Schneider, E., 2000. Maltose and maltodextrin transport in the thermoacidophilic gram-positive bacterium Alicyclobacillus acidocaldarius is mediated by a high-affinity transport system that includes a maltose binding protein tolerant to low pH. Journal of Bacteriology 182 (22), 6292–6301.

Jensen, J.B., Peters, N.K., Bhuvaneswari, T.V., 2002. Redundancy in periplasmic binding protein-dependent transport systems for trehalose, sucrose, and maltose in Sinorhizobium meliloti. Journal of Bacteriology 184 (11), 2978–2986.

Juncker, A.S., Willenbrock, H., Von Heijne, G., Brunak, S., Nielsen, H., Krogh, A., 2003. Prediction of lipoprotein signal peptides in gram-negative bacteria. Protein Science: A Publication of the Protein Society 12 (8), 1652–1662.

Krogh, A., Larsson, B., von Heijne, G., Sonnhammer, E.L., 2001. Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes. Journal of Molecular Biology 305 (3), 567–580.

Kyte, J., Doolittle, R.F., 1982. A simple method for displaying the hydropathic character of a protein. Journal of Molecular Biology 157 (1), 105–132.

Lee, S., Sauerbrei, B., Niggemann, J., Egelkrout, E., 1997. Biosynthetic studies on the alpha-glucosidase inhibitor acarbose in Actinoplanes sp.: source of the maltose unit. The Journal of Antibiotics 50 (11), 954–960.

Leemhuis, H., Wehmeier, U.F., Dijkhuizen, L., 2004. Single amino acid mutations interchange the reaction specificities of cyclodextrin glycosyltransferase and the acarbose-modifying enzyme acarviosyl transferase. Biochemistry 43 (41), 13204–13213.

Licht, A., Bulut, H., Scheffel, F., Daumke, O., Wehmeier, U.F., Saenger, W., Schneider, E., Vahedi-Faridi, A., 2011. Crystal structures of the bacterial solute receptor AcbH displaying an exclusive substrate preference for β-D-galactopyranose. Journal of Molecular Biology 406 (1), 92–105.

Merettig, N., 2009. Acarbose als Carbophor: Biochemische Charakterisierung Acarbose-resistenter alpha-Glucosidasen und ihre Bedeutung für die Produzentenstämme. Bergische Universität Wuppertal, Wuppertal.

Neuhoff, V., Arol, N., Taube, D., Ehrhardt, W., 1988. Improved staining of proteins in polyacrylamide gels including isoelectric focusing gels with clear background at nanogram sensitivity using Coomassie Brilliant Blue G-250 and R-250. Electrophoresis 9 (6), 255–262.

Pancholi, V., 2001. Multifunctional alpha-enolase: its role in diseases. Cellular and Molecular Life Sciences: CMLS 58 (7), 902–920.

Pancholi, V., Fischetti, V., 1998. Alpha-enolase, a novel strong plasmin(Ogen) binding protein on the surface of pathogenic streptococci. The Journal of Biological Chemistry 273 (23), 14503–14515.

Parenti, F., Coronelli, C., 1979. Members of the genus Actinoplanes and their antibiotics. Annual Review of Microbiology 33, 389–411.

Perkins, D.N., Pappin, D.J.C., Creasy, D.M., Cottrell, J.S., 1999. Probability-based protein identification by searching sequence databases using mass spectrometry data. Electrophoresis 20 (18), 3551–3567.

Petersen, T.N., Brunak, S., von Heijne, G., Nielsen, H., 2011. SignalP 4.0: discriminating signal peptides from transmembrane regions. Nature Methods 8 (10), 785–786.

Rockser, Y., Wehmeier, U., 2009. The gac-gene cluster for the production of acarbose from Streptomyces glaucescens GLA.O: identification isolation and characterization. Journal of Biotechnology 140 (1–2), 114–123.

Schlösser, a., Weber, A., Schrempf, H., 2001. Synthesis of the Streptomyces lividans maltodextrin ABC transporter depends on the presence of the regulator MalR. FEMS Microbiology Letters 196 (1), 77–83.

Schmidt, D.D., Frommer, W., Junge, B., Müller, L., Wingender, W., Truscheit, E., Schäfer, D., 1977. alpha-Glucosidase inhibitors. New complex oligosaccharides of microbial origin. Die Naturwissenschaften 64 (10), 535–536.

Schwientek, P., Szczepanowski, R., Rückert, C., Kalinowski, J., Klein, A., Selber, K., Wehmeier, U.F., Stoye, J. & Pühler, A., 2012. The complete genome sequence of the acarbose producer Actinoplanes sp. SE50/110. BMC Genomics (accepted).

Schwientek, P., Szczepanowski, R., Rückert, C., Stoye, J., Pühler, A., 2011. Sequencing of high G+C microbial genomes using the ultrafast pyrosequencing technology. Journal of Biotechnology 155 (1), 68–77.

Sutcliffe, I.C., Russell, R.R., 1995. Lipoproteins of gram-positive bacteria. Journal of Bacteriology 177 (5), 1123–1128.

Szaniszlo, P.J., Gooder, H., 1967. Cell wall composition in relation to the taxonomy of some Actinoplanaceae. Journal of Bacteriology 94 (6), 2037–2047.

Truscheit, E., Frommer, W., Junge, B., Müller, L., Schmidt, D.D., Wingender, W., 1981. Chemistry and biochemistry of microbial α-glucosidase inhibitors. Angewandte Chemie International Edition in English 20 (9), 744–761.

Vahedi-Faridi, A., Licht, A., Bulut, H., Scheffel, F., Keller, S., Wehmeier, U.F., Saenger, W., Schneider, E., 2010. Crystal structures of the solute receptor GacH of Streptomyces glaucescens in complex with acarbose and an acarbose homolog: comparison with the acarbose-loaded maltose-binding protein of Salmonella typhimurium. Journal of Molecular Biology 397 (3), 709–723.

Wallenfels, K., Rached, I.R., Hucho, F., 1969. Klassifizierung der Pullulanase als Exoenzym mit Hilfe der Gaschromatographie. European Journal of Biochemistry 7 (2), 231–233.

Watt, S.A., Wilke, A., Patschkowski, T., Niehaus, K., 2005. Comprehensive analysis of the extracellular proteins from Xanthomonas campestris pv campestris B100. Proteomics 5 (1), 153–167.

Wehmeier, U.F., Piepersberg, W., 2004. Biotechnology and molecular biology of the alpha-glucosidase inhibitor acarbose. Applied Microbiology and Biotechnology 63 (6), 613–625.

Wehmeier, U.F., 2003. The biosynthesis and metabolism of Acarbose in Actinoplanes sp. SE 50/110: a progress report. Biocatalysis and Biotransformation 21 (4–5), 279–284.

van Wezel, G.P., White, J., Bibb, M.J., Postma, P.W., 1997a. The malEFG gene cluster of Streptomyces coelicolor A3(2): characterization, disruption and transcriptional analysis. Molecular and General Genetics: MGG 254 (5), 604–608.

van Wezel, G.P., White, J., Young, P., Postma, P.W., Bibb, M.J., 1997b. Substrate induction and glucose repression of maltose utilization by Streptomyces coelicolor A3(2) is controlled by malR, a member of the lacl-galR family of regulatory genes. Molecular Microbiology 23 (3), 537–549.

Wilkinson, A.J., Verschueren, K.H.G., 2003. Crystal structures of periplasmic solute-binding proteins in ABC transport complexes illuminate their function. In: Holland, B., Cole, S.P.C., Kuchler, K., Higgins, C.F. (Eds.), ABC Proteins: From Bacteria to Man. Elsevier, Amsterdam.

Zhang, C.-S., Stratmann, A., Block, O., Brückner, R., Podeschwa, M., Altenbach, H.-J., Wehmeier, U.F., Piepersberg, W., 2002. Biosynthesis of the C(7)-cyclitol moiety of acarbose in Actinoplanes species SE50/110. 7-O-phosphorylation of the initial cyclitol precursor leads to proposal of a new biosynthetic pathway. The Journal of Biological Chemistry 277 (25), 22853–22862.
