
THE JOURNAL OF BIOLOGICAL CHEMISTRY VOL. 287, NO. 44, pp. 37010–37020, October 26, 2012  
© 2012 by The American Society for Biochemistry and Molecular Biology, Inc. Published in the U.S.A.

# The BioC O-Methyltransferase Catalyzes Methyl Esterification of Malonyl-Acyl Carrier Protein, an Essential Step in Biotin Synthesis*

Received for publication, August 13, 2012, and in revised form, September 3, 2012 Published, JBC Papers in Press, September 10, 2012, DOI 10.1074/jbc.M112.410290

**Steven Lin**† **and John E. Cronan**‡§1  
From the Departments of †Microbiology and §Biochemistry, University of Illinois, Urbana, Illinois 61801

---

### Background: The pimelate moiety of biotin is made by a modified fatty acid synthesis pathway.
### Results: The first reaction is O-methylation of the free carboxyl of malonyl-acyl carrier protein.
### Conclusion: The methyl acceptor is malonyl-acyl carrier protein and not malonyl-CoA.
### Significance: Demonstration of this enzymatic activity completes the *E. coli* biotin synthetic pathway.

---

Recent work implicated the *Escherichia coli* BioC protein as the initiator of the synthetic pathway that forms the pimelyl moiety of biotin (Lin, S., Hanson, R. E., and Cronan, J. E. (2010) Nat. Chem. Biol. 6, 682–688). BioC was believed to be an O-methyltransferase that methylated the free carboxyl of either malonyl-CoA or malonyl-acyl carrier protein based on the ability of O-methylated (but not unmethylated) precursors to bypass the BioC requirement for biotin synthesis both *in vivo* and *in vitro*. However, only indirect proof of the hypothesized enzymatic activity was obtained because the activities of the available BioC preparations were too low for direct enzymatic assay. Because *E. coli* BioC protein was extremely recalcitrant to purification in an active form, BioC homologues of other bacteria were tested. We report that the native form of *Bacillus cereus* ATCC10987 BioC functionally replaced *E. coli* BioC *in vivo*, and the protein could be expressed in soluble form and purified to homogeneity. In disagreement with prior scenarios that favored malonyl-CoA as the methyl acceptor, malonyl-acyl carrier protein was a far better acceptor of methyl groups from S-adenosyl-L-methionine than was malonyl-CoA. BioC was specific for the malonyl moiety and was inhibited by S-adenosyl-L-homocysteine and sinefungin. High level expression of *B. cereus* BioC in *E. coli* blocked cell growth and fatty acid synthesis.

---

Biotin (vitamin H) is an essential enzyme cofactor required by all three domains of life (1, 2). It functions as a covalently bound prosthetic group that mediates the transfer of CO₂ in many vital metabolic carboxylation, decarboxylation, and transcarboxylation reactions (1, 2). Although biotin is essential, our knowledge of its biosynthesis remains fragmentary. ¹³C-Labeling studies in *Escherichia coli* indicated that most of the carbon atoms of biotin are derived from pimelic acid, a seven-carbon α,ω-dicarboxylic acid (3, 4). However, only recently has the mechanism whereby *E. coli* assembles the pimelyl intermediate been determined (5). Genetic studies in *E. coli* identified bioC and bioH as the only genes essential for biotin synthesis with unassigned functions (6). Strains having inactive bioC or bioH genes require biotin for growth, but biotin can be replaced by any of the later pathway intermediates including 7-keto-8-aminopelargonic acid (formal name 8-amino-7-oxo-nonanoic acid) (6). Because 7-keto-8-aminopelargonic acid is readily synthesized *in vitro* from a thioester-linked pimelyl moiety and L-alanine (6), BioC and BioH were assigned roles in pimelyl synthesis. Various workers have proposed contradictory roles for BioC in biotin synthesis (7, 8). However, these proposals not only lacked supporting data but also failed to address the fundamental problem of how to assemble an α,ω-dicarboxylic acyl chain.

Previously we reported the pathway of pimelyl moiety synthesis in *E. coli* (5) (Fig. 1). In the pathway BioC and BioH do not directly catalyze the synthesis of pimelate but instead provide the means to allow fatty acid synthesis to assemble the pimelate moiety (Fig. 1). BioC catalyzes transfer of the methyl group of S-adenosyl-L-methionine (SAM)² to the ω-carboxyl group of malonyl-thioester of either CoA or acyl carrier protein (ACP) to form an O-methyl ester. The product then becomes the primer for the synthesis of pimelyl-ACP by the fatty acid synthetic pathway. Methylation of the free carboxyl group of the malonyl-thioester was essential because of the extremely hydrophobic nature of the active sites of the fatty acid synthesis proteins (9). The methyl ester moiety neutralizes the carboxylate negative charge and mimics the methyl end of the monocarboxylates normally found in fatty acid synthesis. Two rounds of the standard fatty acid reductase-dehydratase-reductase reaction sequence results in the ACP thioester of ω-methyl pimelic acid (Fig. 1). At this stage the methyl group must be removed by BioH to block further elongation to azelayl-ACP methyl ester, a physiologically useless product (5). Moreover, the freed carboxyl group will later be required for the essential covalent attachment of biotin to its cognate enzymes (10). Although our prior work produced a pathway firmly based on both *in vivo* and *in vitro* data (5), a shortcoming was that our only source of BioC

---

* This work was supported, in whole or in part, by National Institutes of Health Grant AI15650.
¹ To whom correspondence should be addressed: Dept. of Microbiology, University of Illinois, B103 Chemical and Life Sciences Laboratory, 601 S. Goodwin Ave., Urbana, IL 61801. Tel.: 217-333-7919; Fax: 217-244-6697; E-mail: j-cronan@life.uiuc.edu.
² The abbreviations used are: SAM, S-adenosyl-L-methionine; ACP, acyl carrier protein; TCEP, tris(2-carboxyethyl)phosphine.

BioC Methyltransferase

A.
![Diagram A](image1.png)
Malonyl-ACP

B.
![Diagram B](image2.png)
Malonyl-ACP methyl ester

C.
![Diagram C](image3.png)
Pimeloyl-ACP methyl ester

D.
![Diagram D](image4.png)
Pimeloyl-ACP

FIGURE 1. Schematic of the synthetic pathway of the biotin pimeloyl moiety in *E. coli*. The pathway proceeds from A to D. The methyl groups are colored gray. A, the biotin synthetic pathway is initiated by BioC-catalyzed methyl transfer from SAM to malonyl-ACP. B, the malonyl ACP methyl ester enters the fatty acid synthetic cycle as the primer. Two rounds of the fatty acid chain elongation cycle gives pimeloyl-ACP methyl ester. C, pimeloyl-ACP methyl ester is hydrolyzed by BioH to form pimeloyl-ACP, which is a substrate for BioF. D, BioF begins assembly of the biotin rings by catalyzing the pyridoxal phosphate-dependent decarboxylative condensation of pimeloyl-ACP and L-alanine. KAPA, 7-keto-8-amino-pelargonic acid (also called 8-amino-7-oxononanois acid). The pathway is given in greater detail in Refs. 5 and 6.

was the *E. coli* protein solubilized from inclusion bodies. These preparations had very low activities that precluded direct enzymatic assay, although activity could be detected by bioassays. As previously reported by others (11), when overexpressed, *E. coli* BioC was invariably insoluble, either the native protein or as various affinity-tagged versions. In vitro protein synthesis also failed to yield detectable amounts of *E. coli* BioC. Given this situation we surveyed BioC proteins from a diverse set of bacteria for the ability to functionally replace *E. coli* BioC in vivo and for solubility upon high level expression. We now report the enzymatic properties of *Bacillus cereus* BioC.

EXPERIMENTAL PROCEDURES

Materials—The defined medium was M9 (12) supplemented (final concentrations) with 0.2% glucose, 0.1% vitamin assay casamino acids and 1 μg/ml thiamine except when 0.2% arabinose replaced glucose. RB medium contained 10 g of tryptone, 1 g of yeast extract, and 5 g of NaCl per liter of H₂O. 2XYT medium contained 16 g of tryptone, 10 g of yeast extract, and 5 g of NaCl per liter of H₂O. M9 was added as a buffer to LB, RB, and 2XYT media to give LB-M9, RB-M9, and 2XYT-M9 media, respectively. Agar was added to 1.5%. PMSF (Sigma-Aldrich) was dissolved in ethanol to 100 mM. *E. coli* ACP (apo form), Mtn (*E. coli* 5'-methylthioadenosine/S-adenosylhomocysteine nucleosidase), and Sfp (a *Bacillus subtilis* phosphopantetheinyl transferase) were prepared from strains DK574/pJT94, ER103, and NRD4–33, respectively, strictly according to the protocols published previously (13–15).

Plasmid Constructions—Plasmids and oligonucleotide primers are listed in Table 1. The *E. coli* metK gene was PCR-amplified from MG1655 genomic DNA using oligonucleotides B24 and B25. The DNA product was digested with NcoI and XhoI and ligated to vector pET28b+ digested with the same enzymes to give plasmid pSTL27. The *B. cereus* ATCC10987 *bioC* gene

TABLE 1  
**Bacterial strains, plasmids, and primers**

| Strains          | Relevant characteristics                          | References |
|------------------|--------------------------------------------------|------------|
| DK574/pJT94      | Apo-ACP expression strain                        | Ref. 32    |
| ER103            | Mtn expression strain                            | Ref. 14    |
| NRD4–33          | Sfp expression strain                            | Ref. 33    |
| NRD204           | MG1655ΔaraBAD::cat                               | Ref. 34    |
| STL23            | MG1655ΔbioC::FRT                                 | Ref. 5     |
| STL204           | Tuner (DE3)/pSTL29BioC expression strain          | This study |
| STL215           | Tuner (DE3)/pSTL27MetK expression strain          | This study |

| Plasmids         | Relevant characteristics                          | References |
|------------------|--------------------------------------------------|------------|
| 37454            | pJexpress401 plasmid carrying a synthetic *B. cereus* ATCC10987 *bioC* gene | DNA 2.0 Inc. |
| pSTL27           | pET28b+ plasmid encoding *E. coli* metK gene with a hexahistidine tag fusion at C terminus | This study |
| pSTL28           | pET28b+ plasmid encoding *B. cereus* BioC with a C-terminal hexahistidine tag | This study |
| pSTL29           | pET28b+ plasmid encoding native *B. cereus* BioC | This study |
| pSTL30           | pBAD24 plasmid encoding native *B. cereus* BioC | This study |

| Primers | Sequence 5' to 3' |
|---------|--------------------|
| A77     | CTC ACA ATT CCA CAA CGG TTT CCC TCT AG |
| A78     | CTT GGG GAC TCG AGA CGC TTA CC |
| B13     | GCG TTA GTA GCA CCA CCA CCA C |
| B14     | GGT GGT GCT ACT AAC GCT TAC CCT C |
| B15     | CGG TGA GCT TCG ATC AAT ACG C |
| B16     | GAT CGA AGC TCA CCG CAG C |
| B17     | CCT ACA ACG TTA TCA TTA GCA ACG C |
| B18     | GAT AAC GTT GTA GGT TTC TTC CAA ACG C |
| B19     | CCA GCA ACT GCA TGC GAG C |
| B20     | GCA GTT GCT GGA AAG TTT CCT G |
| B21     | GCT ACC TTT CAC GCG TTG TTT GTC C |
| B22     | GCG TGA AAG GTA GCC ATG ATG C |
| B23     | CCT TGG GGA AGC TTA ACG CTT ACC C |
| B24     | TTA GGT GAT ATT AAC CAT GGC AAA ACA CC |
| B25     | CGC AGG TGA AGA ACT CGA GCT TCA GAC CGG C |

BioC Methyltransferase

was synthesized in codon-optimized form by DNA2.0 Inc. and provided as plasmid 37454. For high level expression, the gene was PCR-amplified from plasmid 37454 using oligonucleotides A77 and A78. The DNA fragment was digested with BspHI and XhoI and ligated to pET28b+ digested with NcoI and XhoI to give plasmid pSTL28, which encoded BioC having a C-terminal hexahistidine tag. Subsequently two stop codons were inserted at the end of the *bioC* coding sequence by site-directed PCR mutagenesis using oligonucleotides B13 and B14 to give pSTL29 encoding the native untagged protein. For genetic complementation assays, the *bioC* gene was PCR-amplified from plasmid 37454 using oligonucleotides A77 and B23. The DNA fragment was digested with BspHI and HindIII and ligated to pBAD24 (digested with NcoI and HindIII) to give pSTL30. Mutations of the *bioC* gene were constructed by QuikChange site-directed PCR mutagenesis (Stratagene) using the following oligonucleotide sets: Y18F (B15 and B16), D110N (B17 and B18), E153A (B19 and B20), and Y256F (B21 and B22). Plasmid DNAs were extracted using QIAprep Minipreps (Qiagen). The constructs were verified by DNA sequencing by the Carver Biotechnology Center Core Sequencing Facility of the University of Illinois at Urbana-Champaign.

**Preparation of SAM—A C-terminal hexahistidine-tagged MetK protein** was expressed in strain STL215. The strain was inoculated into 5 ml of LB medium supplemented with 50 μg/ml kanamycin at 30°C and grown overnight. The cells were diluted 1:100 into 300 ml of LB-M9 medium supplemented with 50 μg/ml kanamycin. The culture was incubated at 37°C for 4.5 h. Isopropyl thiogalactopyranoside was added to 10 μM to induce protein expression for 2 h at 37°C. The cells were harvested and resuspended in 9 ml of BugBuster Master Mix (EMD) and incubated at room temp for 40 min for lysis. Twenty ml of loading buffer (25 mM HEPES, pH 7.5, 0.5 M NaCl, 20% glycerol, 5 mM TCEP, and 20 mM imidazole) was added, and the mixture was centrifuged at 20,000 × g to remove cell debris. The supernatant was incubated with 1.5 ml of nickel-nitrilotriacetic acid (Qiagen) at 4°C for 30 min. The resin was transferred to a 10-ml column. The column was washed once with 20 column volumes of loading buffer and then 40 column volumes of loading buffer containing 60 mM imidazole. The protein was eluted in loading buffer containing 500 mM imidazole and dialyzed against 25 mM HEPES (pH 7), 0.2 mM KCl, 10% glycerol, and 1 mM TCEP at 4°C overnight. Protein concentration was determined by Bradford assay (Bio-Rad).

The enzymatic synthesis of SAM was adapted from Park *et al.* (16) and Farrar *et al.* (17). The reaction consisted of 100 mM Tris-HCl (pH 8), 200 mM KCl, 100 mM MgCl₂, 20% acetonitrile, 50 mM ATP, 20 mM Met, and 0.12 mM MetK and incubated at 37°C for 2 h. One ml of 4 M sodium acetate (pH 4) was added, and the mixture was incubated on ice for 10 min to precipitate MetK. Ten ml of 0.2 M sodium acetate (pH 4) was added to the supernatant, and the mixture was transferred to a 1.5-ml Amberlite CG50 column that was equilibrated with 15 column volumes of 0.2 M sodium acetate. The column was washed with 15 column volumes of 0.2 M sodium acetate. SAM was eluted in 0.1 M H₂SO₄ containing 10% ethanol (pH ~1) and stored at −80°C. The concentration of SAM was determined by A₂₆₀ using an extinction coefficient of 14,700 M⁻¹ cm⁻¹. The purity of SAM was analyzed by HPLC. The samples were injected onto a Waters µBondapack C18 column and eluted with a linear gradient from 0.2 M ammonium acetate to methanol for 6 min at a flow rate of 0.7 ml/min at room temperature. ATP, SAM, and methylthioadenosine were monitored at 260 nm. A working solution of [methyl-³H]SAM was prepared by mixing the synthesized SAM with PerkinElmer Life Sciences [methyl-³H]SAM (80.7 Ci/mmol) at a molar ratio of ~720:1. The final concentration was adjusted to 1 mM with 10 mM H₂SO₄ (pH ~1) and 10% ethanol. The specific activity was 91.3 cpm/pmol of [methyl-³H]SAM.

**Purification of B. cereus BioC—** The native form of *B. cereus* BioC was expressed in strain STL204 as follow. The strain was inoculated into 5 ml of LB medium supplemented with 50 μg/ml kanamycin and grown overnight at 30°C. The culture was transferred to 500 ml of LB-M9 medium supplemented with 50 μg/ml kanamycin and shaken at 37°C for 6 h. The cells were harvested by centrifugation and resuspended in 10 ml of LB medium supplemented with 50 μg/ml kanamycin. The cell suspension (2.5 ml) was added to 1 liter of 2XYT-M9 medium containing 50 μg/ml kanamycin and 0.5 μM isopropyl thiogalactopyranoside. The culture was shaken at room temperature for 17 h. The cells were harvested by centrifugation, and the cell pellet was stored at −80°C.

The cells were suspended in purification buffer which was 25 mM MES (pH 6), 0.1 M LiCl, 10% glycerol, and 5 mM tris(2-carboxyethyl)phosphine (TCEP) containing 10 mM PMSF. DNase I (Sigma) 5 μg/ml was added, and the cells were lysed by two passages through a French pressure cell at 17,500 p.s.i. The soluble cell extract was obtained by centrifugation at 20,000 × g and filtration through a 0.45-μm filter (Millipore). All subsequent protein purification steps were performed at 4°C on an ACTA Purifier FPLC using columns from GE Healthcare Life Sciences. The extract was loaded at 2 ml/min into a HiTrap SP FF column that had been equilibrated with purification buffer. The column was washed with purification buffer containing 0.2 M LiCl. BioC was eluted with purification buffer containing 0.3 M LiCl, and the volume of the BioC eluate was concentrated to 20 ml with an Amicon centrifugal filter (molecular mass cutoff of 10 kDa) (Millipore). Twenty ml of purification buffer containing 2 M (NH₄)₂SO₄ was added slowly to the BioC eluent to give 1 M (NH₄)₂SO₄ final concentration. The mixture was filtered through a 0.45-μm filter before loading into a HiTrap phenyl-Sepharose column at 2 ml/min. The column was washed with purification buffer containing 1 M (NH₄)₂SO₄. BioC was eluted by a linear gradient to decrease the (NH₄)₂SO₄ concentration from 1 M to 0 over 10 column volumes. The BioC

The LC-MS/MS analysis was performed by the Carver Metabolomics Center at University of Illinois at Urbana-Champaign with an Agilent mass spectrometer (MSD Trap XCT Plus) equipped with an 1100 Agilent LC Chemstation
(version B.01.03) was used for data acquisition and processing. The HPLC flow rate was set at 0.3 ml/min. HPLC mobile phases consisted of A (15 mM ammonium formate in H₂O) and B (15 mM ammonium formate in methanol). The autosampler was kept at 5 °C. The injection volume was 5 μl. A Phenomenex Synergy 4-μm Fusion column (4 m, 100 × 4.6 mm) was used for the separation with the following gradient: 0–2 min, 98% A; 10–15 min, 0% A. The mass spectrometer was operated under positive electrospray ionization. The electrospray voltage was set to 3500 V with a scan range of *m/z* 100–600 and a drying temperature of 350 °C with the nebulizer at 35 p.s.i. and dry gas at 8 liters/min.

Preparation of Malonyl-ACP—Malonyl-ACP was synthesized from ACP (apo form) and malonyl-CoA (Sigma-Aldrich) using Sfp, a *B. subtilis* phosphopantetheinyl transferase, as described previously (15). The reaction consisted of 100 mM MES (pH 6), 10 mM MgCl₂, 10 μM Sfp, 0.5 mM ACP, and 0.75 mM malonyl-CoA. Acetyl-ACP, succinyl-ACP, and glutaryl-ACP were prepared by the same procedure, from acetyl-CoA, succinyl-CoA, and glutaryl-CoA, respectively. Malonyl-ACP methyl ester was synthesized by BioC using malonyl-ACP and SAM as substrates. The reaction contained 100 mM Tris-HCl (pH 7.5), 10 mM MgCl₂, 10% glycerol, 200 mM NaCl, 0.3 μM BioC, 3 μM Mtn, 0.5 mM malonyl-ACP, and 1 mM SAM. The reactions were incubated at 37 °C for 1 h. ACP products were purified using a Vivapure D column and analyzed by gel electrophoresis (13). To verify BioC methylation, malonyl-ACP and malonyl-ACP methyl ester were dialyzed against 2 mM ammonium acetate in a 3-kDa molecular mass cutoff dialysis cassette (Pierce) at 4 °C overnight. The ACP species were dried under vacuum, and the mass was analyzed by MALDI-TOF/electrospray ionization mass spectrometry at the University of Illinois School of Chemical Sciences Mass Spectrometry Laboratory.

BioC Methyltransferase Assay—The 50-μl reaction consisted of 100 mM sodium phosphate buffer (pH 7), 20 mM MgCl₂, 100 mM NaCl, 20% glycerol, 20 nM Mtn nucleosidase, 100 nM BioC, malonyl-ACP and \[^{methyl-3}H\]SAM. To determine the kinetic parameters for malonyl-ACP and SAM, the concentrations of \[^{methyl-3}H\]SAM and malonyl-ACP were fixed to 100 and 75 μM, respectively. A premix lacking BioC equivalent to 6.5 reactions was prepared and incubated at 37 °C for 1 min. The reactions were then initiated by adding BioC. A 50-μl sample was taken from the premix vial at each time point and transferred into a 1.5-ml tube in dry ice to quench the reaction. The ACP product was purified from \[^{methyl-3}H\]SAM using Macro-Prep High Q resin (Bio-Rad) as follows. One ml of washing buffer (50 mM sodium acetate, pH 4.5, and 25 mM NaCl) was added to the 50-μl reaction sample followed by 100 μl of Q resin (50% slurry in 20% ethanol), and the mixture was incubated for 5 min for ACP binding. The resin was pelleted by centrifugation, and the pellet was resuspended in 1 ml of washing buffer and again centrifuged. This process was repeated four more times, and then the ACP species were eluted by adding 1 ml of washing buffer containing 0.5 M NaCl. The eluent was mixed thoroughly with 4 ml of Bio-Safe II scintillation fluid (Research Products International Corp.), and \[^3H\] radioactivity was determined in a Beckman LS6500 scintillation counter. Other acyl-ACPs and malonyl-CoA were assayed by the same procedure except that

in purification of CoA species, 25 mM MES (pH 6) was used in place of sodium acetate. All reactions were repeated three times, and Michaelis-Menten kinetic parameters were determined by OriginPro 8.6.

ACP is larger and contains more negative charges than the single acidic group of CoA and hence binds more strongly to ion exchange resins than CoA. We tested the relative binding efficiencies of ACP and CoA to the Q resin, by use of \[^{14}C\]malonyl-CoA. The binding capacity of the Q columns was sufficient to capture 400 nmol of malonyl-CoA (equivalent to 8 mM in a 50-μl reaction). After extensive washing, there was ~20% loss of bound malonyl-CoA, suggesting that BioC activity on malonyl-CoA was slightly underestimated. However, the leaching of CoA from the Q resin was insignificant relative to the differences in BioC activity on malonyl-CoA versus malonyl-ACP.

Phospholipid Labeling Assay—Derivatives of *E. coli* strain NRD204 carrying either a plasmid-encoded *bioC* wild-type gene or a mutant gene were inoculated into 5 ml of RB-M9 medium supplemented with 25 μg/ml of tricarcillin:clavulanate (15:1) and grown at 30 °C overnight. One hundred μl of the cells were subcultured in 100 ml of RB-M9 medium supplemented with 25 μg/ml of tricarcillin:clavulanate (15:1) and incubated at 37 °C. Cell growth was monitored by absorbance at 260 nm every 20 min. Arabinose was added to 0.2% (w/v) to induce BioC expression when the *A*₂₆₀ₙₘ reached 0.3. To determine the rates of phospholipid labeling, 1-ml samples of cultures were transferred to 15-ml tubes containing 5 μCi of sodium \[^{1-14}C\]acetate (55.5 mCi/mmol from American Radiochemicals) and incubated at 37 °C for 20 min. Three ml of a chloroform:methanol mixture (1:2) was added and mixed thoroughly by vortex mixing to extract the lipids, and the sample was stored at 4 °C overnight. Lipid extraction began by mixing in 1 ml of chloroform and then 1 ml of H₂O. The mixture was centrifuged at 4,000 × *g* to accelerate partitioning into two phases. The bottom (chloroform) phase was transferred to a new tube, and the solvent was evaporated under nitrogen. The lipid pellet was dissolved in 50 μl of chloroform:methanol (2:1) and spotted on a silica G thin layer chromatography plate. The phospholipids were resolved by TLC using a solvent system which consisted of chloroform:methanol:acetic acid (65:25:8). Incorporation of \[^{14}C\]acetate into lipids was detected by autoradiography.

**RESULTS**

Soluble Expression and Purification of *B. cereus* BioC—A large assortment of expression constructs, growth conditions, and host cells failed to give soluble *E. coli* BioC and thus we resorted to screening putative BioCs from other bacteria. The BioCs from bacteria closely related to *E. coli* had the same recalcitrant behavior as *E. coli* BioC, and thus we went further afield. BioC homologues from *Kurthia* species, *Pseudomonas putida*, and *Chlorobium tepidum* were highly toxic to *E. coli* and could not be expressed to high levels. The only amenable BioC homologue found was that of *B. cereus* ATCC10987. The *B. cereus bio* operon is unidirectional and encodes homologues of all of the *E. coli bio* enzymes (18). The *bioC* gene is located between *bioH* and *bioB*, and its coding sequence partially overlaps that of *bioH*. This gene arrangement differs markedly from that found
BioC Methyltransferase

A.

B. cereus
E. coli
Kurthia
Cl thermo
Ch tepidum
Serratia sp.
E. herbicola

M I N K T L L Q K R F N V A A V S Y D Q Y A N V Q K K M A H S L L S T L D R R Y S A N S 44
M A - - - - T V N K Q A I A A A F G R A A A H Y E Q H A D L Q R Q S A D A L L A M L P Q R K Y T - - 44
M I D K Q L L S K R F S E H A K T Y D A Y A N V Q K N M A K Q L V D L L P Q K N S K - Q 43
M I N K T L L Q K R F N V A A V S Y D Q Y A N V Q K K M A H S L L S T L D R R Y S A N S 44
M N G - - - V J D K Q L V R R B F R R A L P T Y A G H A E V Q R R M A V R L V A L I E N A G A S T H 47
M T S A N D T V N K Q A V A S A F S R A A G S Y D A A A A L Q R D V G E R L L G M G S S H P G E - - 48
M S - - - - L V N K R A V A A A F G R A A Q S Y D S H A Q L Q R Q S A D L L L A K C G E R R P A - - 44

S I R I L E L G C G T G Y V T E Q L S N L F P K A H I T A I O F A E S - - M I A V A K T R Q N V K 91
- - H V L D A G C G P G W M S R H W R E R H - - A Q V T A L D L S P P - - - M L V Q A R Q K D A A D 87
R I N I L E I G C G T G Y L T R L L V N T F P N A S I T A V D L A P G - - - M V E V A K G I T M E D 90
S I R I L E L G C G T G Y V T E Q L S N L F P K A H I T A I O F A E S - - M I A V A K T R Q N V K 91
L G R V F E F G S G S A M L T S I L F E R Y S A N E F F A N D L V A E S R A F V E K A V T G R N V E 97
- - Q L L D A G C G T G Y F S R M W R E R G - - K R V T A L D L A P G - - - M L D V A R Q R Q A A H 91
- - S V L D A G C G P G S M S R Y W R D A G - - A E V T A L D L S L P - - - M L R Q A Q S Q Q A A Q 87

N V M F Y C E D I E R L Q L E E T - Y D V I I S N A T F Q W L N D L K Q V I R N L F H H L S I D G I 140
- - H Y L A G D I E S L P L A T A T F D L A W S N L A V Q W C G N L S T A L R E L Y R V V R P K G V 135
R V T F L C A D I E E M T L N E N - Y D L I I S N A T F Q W L N N L P G T I E Q L F T R L T P E G N 139
N V M F Y C E D I E R L Q L E E T - Y D V I I S N A T F Q W L N D L K Q V I R N L F H H L S I D G I 140
R L T F L P G D V E R L D P L P G N L D L A V S N A T V Q W L H D P A R F F D R L A T S V K P G G I 147
- - H Y U L G D I E Q V P L P D A A M D I C F S S L V V Q W C S D L P A A L A E L Y R V T R P G G V 139
- - H Y V A A D I E A L P L A D A R F D L A W S N L A V Q W C N D L G Q A L K S L H R V V R P G G A 135

L L F S T F G Q O E T F Q E L H T S F Q R A K E E K N I Q N E T S I G Q R F Y S K N Q L R H I C E V E 190
V A F T T L V Q G S L P E L H Q A W Q A V D E R P H A N - - - - - - R F L P P D E I E Q S L N G V 178
L I F S T F G I K T F Q E L H M S Y E H A K E K L Q L S I D S S P G Q L F Y A L E E E L S Q I C E E A 189
L L F S T F G Q E T F Q E L H T S F Q R A K E E E K N I Q N E T S I G Q R F Y S K N Q L R H I C E V E 190
V A F S T F G A E N M H E I A A L G E A A L P Y R S L D - - - - - - - - K I A A L S G E L F E L 187
I L F S T L A A G S L Q E L G D A W Q Q V D G E R H V N - - - - - - A F L P L T Q I R T A C A A Y 182
V A F T T L A S G S L P E L H Q A W Q A V D S R L H A N - - - - - - R F L A E E T L A E T V S A W 178

T G - - - - - - D V H V S E T C Y I E R F T E V R E F L H S I R K V G A T N S N E E S Y C Q S P S 233
H Y - - - - - - - Q H H I Q P I T L W F D D A L S A M R S L K G I G A T H L H E G - R D P R I 217
I P F S S A F P L E I T K I E K L E L E Y F Q T V R E F F T S I K K I G A R N S N K E N Y C Q R F S 239
T G - - - - - - D V H V S E T C Y I E R F T E V R E F L H S I R K V G A T N S N E E S Y C Q S P S 233
V A - - - - - - - - I E D D I V R Q E F D T P E A V L R H I R K T G V N G V A R R - A W T R S 225
R H - - - - - - - E L V T E L R T L N Y P D V M T L M R S L K G I G A T H L H Q G - R E G G L 221
R G - - - - - - - Q W G I E P V T L A F D D A L A A M R S L K G I G A T H L H A G - R H N T P 217

L F R A M L R I Y E R D F T G N E G I M A T Y H A L F M H I T K E G K R
L T R S Q L Q R L Q L A W P Q Q Q G R Y P L T Y H L F L G V I A R E
F F R E L I N I Y E T K V Q D E S G V K A T Y H C L F F K I I K H A P L P
L F R A M L R I Y E R D F T G N E G I M A T Y H A L F M H I T K E G K R
Q Y L D F L Q R Y R S A Y P S G E G V T L T W H P V Y C C F R K K K S
M S R G R L A A L Q A A Y P C R Q G Q F F P L S Y H L A Y G V I Y R E
L T R G Q L Q R L Q L A W P Q Q Q G R C L L T Y S L F W G V I E R D

B. + Biotin
- Biotin
+ Biotin
- Biotin

Wild-type
Wild-type
Vector control
Vector control

Y18F
Y18F
D110N
D110N

E153Q
E153Q
Y256F
Y256F

FIGURE 2. Alignment of BioC proteins and complementation by wild-type and mutant *B. cereus* BioC proteins. A, Clustal W alignment of *B. cereus* ATCC10987 (top line) and *E. coli* MG1655 BioC (second line) together with the BioCs of several diverse bacteria. The diverse bacteria (followed by their GenBank™ accession numbers in parentheses) are Kurthia, *Kurthia sp.* 538-KA26 (BAB39463); *Cl. thermo*, *Clostridium thermocellum* (ABN51266); *Ch tepidum*, *C. tepidum* (NP_660955); *Serratia sp.*, *Serratia marcescens* (P36571); and *E. herbicola*, *Erwinia herbicola* (O06898). The accession numbers of the *B. cereus* and *E. coli* proteins are NP_980478 and NP_415298, respectively. The residues chosen for mutagenesis are marked by asterisks above the alignment. B, complementation of an *E. coli ΔbioC* strain by plasmid-encoded *B. cereus bioC* wild-type or mutant genes was tested on biotin-free minimal medium. Note that the araBAD promoter driving BioC expression was repressed (presence of glucose and the absence of arabinose) to avoid BioC toxicity.

SAM is a highly unstable molecule (17, 19). The biologically active $(S,S)$ configuration of SAM is prone to spontaneous racemization at the sulfonium center to yield the inactive $(R,S)$-diastereomer (19). SAM can also spontaneously degrade by intramolecular $S_{N}2$ attack of the methionine carboxylate on the $\gamma$-carbon into methylthioadenosine and homocysteine (19). Typically commercial SAM is extracted from yeast and purified by HPLC to give preparations containing only $\sim 43\%$ of $(S,S)$-SAM, the biologically active species, with the remainder composed of the racemized species, degraded products such as S-adenosylhomocysteine (SAH) and as much as 10% of unidentified substances (17). These contaminants are known to interfere with SAM-dependent enzymes and are difficult to remove (17). To obtain reliable kinetics, we enzymatically synthesized $(S,S)$-SAM from ATP and L-methionine using the E. coli MetK SAM synthetase (Fig. 4, A and B). SAM was purified to homogeneity by weak ion exchange chromatography (Fig. 4C). The product was stable in sulfuric acid at $-80^{\circ} \mathrm{C}$ and gave the expected mass and fragmentation products in LC-MS and LC-MS/MS analyses (Fig. 4, D and E). The synthetic SAM was mixed with commercially enzymatically prepared [methyl-$^{3} \mathrm{H}] \mathrm{SAM}$ to a specific activity of $91.3 \mathrm{cpm} / \mu \mathrm{mol}$ of [methyl-$^{3} \mathrm{H}] \mathrm{SAM}$. Note that the Mtn (also called Pfs) nucleosidase that cleaves the glycosidic bonds of SAH (and methylthioadenosine) was included in the assay to prevent inhibition of BioC by the SAH generated in the methyl transfer reaction.

We tested malonyl-ACP and malonyl-CoA as BioC methyl acceptor substrates and found that malonyl-ACP was a much better substrate than malonyl-CoA (Fig. 5). Holo-ACP, acetyl-ACP, succinyl-ACP, and glutaryl-ACP were also assayed and no methyl transfer was detected, indicating that BioC specifically targeted malonyl-ACP. Malonyl-CoA methylation occurred, but only at much higher substrate concentrations. Under the same reaction conditions, BioC required $8 \mathrm{mM}$ malonyl-CoA to achieve $50\%$ of the activity obtained from $0.05 \mathrm{mM}$ malonyl-ACP (Fig. 5). The product of methyl transfer to malonyl-ACP was demonstrated to be malonyl-ACP methyl ester by mass spectral analysis (Fig. 6). We also tested glutaryl-CoA, succinyl-CoA, and methylmalonyl-CoA as substrates at both $4$ and $8 \mathrm{mM}$ in parallel with malonyl-CoA. Only malonyl-CoA acted as a methyl acceptor (data not shown), indicating that BioC is specific for the malonyl moiety irrespective of the thioester-linked moiety.

In further analyses, the pH profile of B. cereus BioC was determined, and the highest methyl transfer activity was at pH 7 in a sodium phosphate buffer. Within the pH range 5–7.5, the activity remained above $50\%$ but decreased drastically outside this range (Fig. 7A). B. cereus. The addition of metal ions or EDTA to the reaction neither enhanced nor reduced BioC activity significantly (Fig. 7B). The kinetic parameters for malonyl-ACP (Fig. 7C) and SAM (Fig. 7D) were determined in sodium phosphate buffer at pH 7.

Potent Inhibition of BioC Methyl Transfer by SAH and Sinefungin—In general SAM-dependent methyltransferases are inhibited in a competitive manner by SAH, the product of methyl transfer, and as expected SAH inhibited BioC in a concentration-dependent manner (see Fig. 8). Another potent methyltransferase inhibitor is sinefungin, a natural antibiotic

FIGURE 3. Purification of the native form of B. cereus BioC. A, SDS-PAGE in which the numbers indicate the protein peaks obtained during the three chromatographic steps. Lane 1, flow-through of the anion exchange column; lane 2, eluted BioC fractions from anion exchange chromatography using a HiTrap SP FF column; lane 3, eluted BioC fractions from hydrophobic interaction chromatography using a HiTrap phenyl-Sepharose column; lane 4, eluted BioC fractions from preparative size exclusion chromatography using a HiLoad Superdex 200 column. $M$, molecular weight standards; Cru, crude extract. B, elution of purified BioC from Superdex 200 analytical size exclusion column calibrated with protein standards from Bio-Rad. BioC eluted between the chicken ovalbumin (44 kDa) and equine myoglobin (17 kDa) protein standards, indicating a monomeric protein.

in E. coli where the bio operon is transcribed bidirectionally, and the bioH gene is located elsewhere in the genome. Although the sequence of B. cereus BioC shares only $26.5\%$ identity with E. coli BioC (Fig. 2A), expression of the protein supported growth of an E. coli $\Delta b i o C$ strain in the absence of biotin (Fig. 2B), and mutagenesis of residues conserved between the two proteins and BioCs encoded by a diverse set of bacteria (Fig. 2A) resulted in loss of complementation activity in three of the four mutants constructed. To obtain maximum expression in E. coli, a synthetic B. cereus bioC gene in which the codons had been altered to those favored by E. coli was used. The native protein was expressed in E. coli in soluble form by overnight expression at room temperature. B. cereus BioC was purified to apparent homogeneity in three chromatographic steps using anion exchange chromatography followed by hydrophobic interaction chromatography and size exclusion chromatography as shown by denaturing gel electrophoresis and analytical size exclusion chromatography (Fig. 3). BioC eluted from the size exclusion column as a protein of 31.2 kDa, indicating that it is monomeric in solution (Fig. 3B). The protein was stable throughout purification, assay, and storage.

Assay of BioC Methyl Transfer—The methyl transfer assay took advantage of the acidic nature of both ACP (pI of 4.1) and CoA (because of its $3'$-phosphate group). Hence substrates in thioester linkage to either ACP or CoA bound the anionic Q ion exchange resin, whereas the positively charged methyl donor, [methyl-$^{3} \mathrm{H}] \mathrm{SAM}$, passed through the column.

BioC Methyltransferase

A.
Triphosphate
ATP + Methionine —— MetK —— SAM

B. Before purification
C. After purification

D. +MS
E. +MS/MS(399.4)

FIGURE 4. Synthesis and mass spectral characterization of SAM. A, synthesis of SAM from ATP and L-methionine. B, the synthesis was monitored on HPLC at 260 nm. SAM is highly unstable and readily degrades into methylthioadenosine (which absorbs at 260 nm) and homoserine. C, SAM was purified by ion exchange chromatography to remove reactants and degraded products. D and E, purified SAM was subjected to LC-MS (D) and LC-MS/MS (E) analyses. The detected mass corresponded to the theoretical mass of 398.44 g/mol. In LC-MS/MS, fragmentation at the sulfonium ion in MS/MS gave rise to adenosine and methylthioadenosine.

To ascertain the effectiveness of the Mtn nucleosidase included in the BioC assay, we tested whether Mtn addition reversed inhibition of BioC by exogenously added SAH. Mtn addition completely neutralized the effects of 0.1 and 1 μM SAH and partially rescued inhibition by 10 μM SAH (Fig. 8). As expected because of its lack of a sulfur moiety, sinefungin was not cleaved by Mtn and remained fully inhibitory its presence. A series of SAM-related compounds were also tested. 5′-Deoxyadenosine, a byproduct of reaction catalyzed by radical SAM enzymes such as BioB, was not inhibitory, and nor was methylthioadenosine, a SAM degradation product. Adenosine and homocysteine also had no effect on BioC methylation activity (Fig. 8).

In Vivo Fatty Acid Synthesis Is Inhibited by High Level Expression of BioC—It has been reported that overexpression of BioC impaired cell growth; however, the exact cause of BioC toxicity was unclear (22). We suspected that high levels of BioC expression would lead to an elevated level of BioC methylation, resulting in depletion of the malonyl-ACP pool. Malonyl-ACP is the building block of the canonical fatty acid synthesis pathway, whereas malonyl-ACP methyl ester has no known function other than in biotin synthesis. In fatty acyl chain elongation, the

FIGURE 5. Specificity of the B. cereus BioC methyltransferase. A, the activity was determined using different acyl-ACPs as methyl acceptor. Holo-ACP is ACP with a free thiol. B, methyl transfer to malonyl-CoA.

isolated from Streptomyces griseolus (20, 21). Sinefungin, a steric and electrostatic mimic of SAM, is generally a more potent methyltransferase inhibitor than SAH (20) and completely abolished BioC activity at 10 μM and reduced BioC activity by ~60% at 0.1 μM (Fig. 8).

To monitor the rates of phospholipid synthesis, the cultures were sampled at different time points in the growth curve and incubated with \[^{14}C\]acetate for 10 min to label the phospholipid fatty acid moieties. The phospholipids were then extracted from cultures, resolved by thin layer chromatography TLC, and visualized by autoradiography (Fig. 9B). In the strain expressing wild-type BioC only a faint phosphatidylethanolamine spot was detected at the first time point and nothing at the later time points. As generally seen in *E. coli*, strains lacking an enzyme that acts early in fatty acid synthesis, the turbidity of the culture continued to increase after fatty acid synthesis ceased. This observation was previously reported with *E. coli fadD* strains (24) and other strains blocked early in the fatty acid synthesis pathway. In contrast the vector control and uninduced strains continued fatty acid and phospholipid syntheses normally, whereas as expected from the growth curve, induction of BioC Y18F expression resulted in reduced phospholipid labeling, although the reduction was not as severe as that seen upon expression of the wild-type BioC (Fig. 9B).

## DISCUSSION

In prior work we were unable to directly assay BioC methylation activity because overexpressed *E. coli* BioC was inactive and insoluble (5). Solubilization and refolding of *E. coli* BioC yielded a weakly active enzyme, the activity of which could be detected by bioassay of extracts of an *E. coli ΔbioC* strain to which the refolded enzyme had been added (5). No activity could be detected by direct assay using \[methyl-^3H\]SAM and malonyl-CoA (data not shown). Although the weak BioC activity was probably due to improper or incomplete refolding of the protein, other possibilities were inhibitory contaminants in commercial SAM (17) and that malonyl-CoA was not the correct substrate. It was also possible that BioC might be an intrinsically poor catalyst.

We previously favored malonyl-CoA over malonyl-ACP as the BioC methyl acceptor (5) because malonyl-CoA methyl ester seemed likely to take the place of acetyl-CoA, the substrate of 3-ketoacyl-ACP synthase III (FabH), which primes the synthesis of new acyl chains (23). To our surprise malonyl-CoA was found to be a poor substrate for BioC. Methylation of malonyl-CoA was detected, but only at high nonphysiological substrate concentrations (Fig. 7). BioC was saturated by 22 μM malonyl-ACP (Fig. 8), whereas the enzyme showed no signs of becoming substrate-saturated at 8 mM malonyl-CoA (Fig. 6). Given that the intracellular concentration of malonyl-CoA in *E. coli* is 35 μM (26), BioC methylation of malonyl-CoA seems most unlikely to occur *in vivo*. The ability of BioC to methylate malonyl moieties linked to either ACP or CoA is not unprecedented. Acyl-CoAs are often reasonable *in vitro* analogues of acyl-ACPs because they share the 4′-phosphopantetheine group and both are acidic molecules. When acyl-CoAs are *in vitro* substrates for enzymes known to use acyl-ACPs *in vivo*, the usual pattern is that the \(K_m\) values of the acyl-CoAs are considerably higher than those of the cognate acyl-ACPs, although the \(V_{max}\) values can be similar (27). The present case fits this pattern, although no malonyl-CoA kinetic constants could be obtained because we were unable to saturate the enzyme with the substrate. Note that the ACP moiety of malonyl-ACP is decarboxylated by 3-ketoacyl-ACP synthase. Loss of the terminal carboxylate group from the malonate moiety results in the formation of a carbanion that attacks the 3-ketoacyl-ACP synthase acyl enzyme intermediate (23). Methylation of the malonate ω-carboxyl group would block this decarboxylative condensation and thus fatty acid synthesis. Indeed, high level production of BioC should mimic the effects of inactivating FabD (24), the enzyme that converts malonyl-CoA to malonyl-ACP.

To test this scenario, strain *E. coli* NRD204 carrying a plasmid encoding the *B. cereus bioC* gene under an arabinose-inducible promoter was constructed. As a control we similarly constructed a strain that expressed the BioC Y18F protein, which failed to complement *E. coli* BioC *in vivo* (Fig. 2B). Induction of wild-type *B. cereus bioC* impaired cell growth within an hour and led to a complete blockage of growth by 3 h (Fig. 9A). No growth inhibition was detected in the vector control or in the absence of the arabinose inducer, indicating that wild-type BioC caused the growth inhibition. In contrast, induction of *bioC* Y18F expression showed normal cell growth in the first 3 h. However, this strain reached the stationary phase sooner and at a lower density than the vector control and the uninduced strain (Fig. 9A), suggesting that BioC Y18F retains partial activity and therefore required more time to deplete the malonyl-ACP pool. The differing activities of BioC Y18F in the complementation assay and the present inhibition assay is explained by the roughly 100-fold difference in expression levels of the araBAD promoter when repressed (glucose) versus induced (arabinose) (25).

BioC Methyltransferase

A.
![Graph A](graph_a.png)

B.
![Graph B](graph_b.png)

C.
![Graph C](graph_c.png)

D.
![Graph D](graph_d.png)

FIGURE 7. Properties of *B. cereus* BioC. A, the pH profile of the BioC reaction was determined using malonyl-ACP as substrate in buffers of sodium acetate (pH 4, 4.5, and 5), sodium MES (pH 5.5 and 6), sodium phosphate (pH 6.5, 7, and 7.5), and Tris-HCl (pH 8, 8.5, and 9) at 37°C for 10 min. B, metal ion dependence was tested in either 20 mM EDTA or 10 mM metal ion in sodium phosphate at pH 7. C and D, the $K_m$ and $k_{cat}$ for malonyl-ACP (C) and SAM (D) were determined in sodium phosphate at pH 7. The data were fit to Michaelis-Menten kinetics using OriginPro 8.6. All of the reactions were repeated three times. The standard errors are presented as error bars.

![Graph E](graph_e.png)

FIGURE 8. Inhibition of BioC activity by compounds structurally related to SAM. Each compound was incubated with 100 nM BioC and 50 μM malonyl-ACP on ice for 5 min. SAM was then added to 10 μM to initiate the reaction, which proceeded at 37°C for 5 min. Ade, adenine; DOA, 5′-deoxyadenosine; MTA, methylthioadenosine; HCY, homocysteine; Sin, sinefungin. The standard errors were calculated from three replicate reactions and are presented as error bars. The data to the left of the dotted line were obtained in the absence of the Mtn nucleosidase, whereas the data to the right of the dotted line were obtained in the presence of 33 nM Mtn.

nyl-ACP was that of *E. coli*, whereas the BioC was from *B. cereus*, and thus it remains possible that the malonyl-thioester of *B. cereus* ACP might be a better methyl acceptor than the substrate we tested (the residue identity of the *E. coli* and *B. cereus* ACPs is 62%). This caveat aside, it is clear that *B. cereus* BioC functions with *E. coli* ACP both *in vivo* and *in vitro*. The $K_m$ for SAM (1.9 ± 0.1 μM) falls well within the physiological concentration of SAM of 200–300 μM (26, 28),

and thus the enzyme seems well able to compete with the many SAM-dependent methyltransferases and SAM radical enzymes.

We previously postulated that BioC might be a poor catalyst (5) because the cellular demand for biotin is extremely low, suggesting that BioC might not have undergone evolutionary selection for catalytic efficiency. Indeed, the BioC $k_{cat}$ values of ~200 s⁻¹ are modest, although not nearly as low as those of the

BioC Methyltransferase

excess BioC led to deleterious redirection of flux distribution in primary metabolism or accumulation of a toxic BioC product. We have shown their first postulate to be correct; fatty acid synthesis is blocked.

The finding that malonyl-ACP is the BioC methyl acceptor raises the question of which of the three *E. coli* 3-ketoacyl-ACP synthases elongates it to the C5 species. The ACP requirement rules out synthase III (FabH) leaving synthases I (FabB) and II (FabF), both of which are known to accept ACP linked primers (23). Null mutations in either *fabB* or in *fabF* fail to engender a biotin requirement,³ and thus elongation of malonyl-ACP methyl ester is not the exclusive activity of either synthase I or II. It therefore seems that either enzyme will suffice. Unfortunately, because simultaneous inactivation of both synthases I and II is lethal (because of a lack of fatty acid elongation), a *fabB fabF* null mutant strain cannot be tested. We are currently testing the abilities of FabB and FabF to catalyze elongation of malonyl-ACP methyl ester *in vitro*.

A possible scenario is that FabD, the reversible malonyl-CoA:ACP transacylase that converts malonyl-CoA to malonyl-ACP (30), might be able to convert malonyl-ACP methyl ester to malonyl-CoA methyl ester and thereby provide a substrate for FabH. However, reversal of the physiological FabD reaction with malonyl-ACP methyl ester seems very unlikely to occur because the extant crystal structures show the guanidinium moiety of a strictly conserved arginine residue (Arg-117 in *E. coli* FabD) forms a bidentate salt bridge with the carboxylate of malonyl-CoA that orients the substrate in the active site (31). The methyl group of malonyl-ACP methyl ester would preclude salt bridge formation and undergo steric clashes with the Arg-117 side chain and nearby side chains.

The BioC-BioH pathway is the dominant, but not the only pathway for synthesis of the biotin pimeloyl moiety (6). Annotated BioC homologues are found in 569 of the 868 complete bacterial genomes currently available. However, some bacteria do not encode a recognizable BioC homologue. The documented exceptions are the α-proteobacteria where FabZ, a protein closely related to FabH, plays a key role and a subset of the bacilli that contain a pimeloyl-CoA synthetase (BioW) and BioI, a cytochrome P450 thought to generate pimeloyl moieties by chain scission of long chain fatty acyl moieties. Curiously the BioW-BioI bacilli include only *B. subtilis* and its close relatives, whereas the rest of the bacilli (e.g., *B. cereus*) use the *E. coli* pathway. We have recently reviewed bacterial biotin synthesis (6) and refer the reader to that article for more detail.

Acknowledgments—We thank Dr. Peter Yau of the Carver Biotechnology Center and Dr. Alexander Ulanov of the Carver Metabolomics Center at the University of Illinois at Urbana-Champaign for help in protein and product characterization.

REFERENCES

1. Attwood, P. V., and Wallace, J. C. (2002) Chemical and catalytic mechanisms of carboxyl transfer reactions in biotin-dependent enzymes. *Acc. Chem. Res.* 35, 113–120
2. Knowles, J. R. (1989) The mechanism of biotin-dependent enzymes.
³ J.E. Cronan, unpublished data.


FIGURE 9. Toxicity of high level BioC expression. A, the effects of high level expression of *B. cereus* BioC on growth of *E. coli* strain NRD204. The medium was RB medium either with the inducer of expression (0.2% arabinose) or without supplementation. Low level expression of Y18F BioC failed to complement a Δ*bioC E. coli* strain (Fig. 2). B, samples (1 ml) of the cultures were taken at six time points during the growth curve and incubated with [1-¹⁴C]acetate for 10 min. Phospholipids were extracted from the pulse-labeled cultures, resolved by thin layer chromatography, and visualized by autoradiography. CL, cardiolipin; PG, phosphatidylglycerol; PE, phosphatidylethanolamine. The host strain was Δ*araBAD* such that arabinose catabolism would not dilute the acetate pool.

enzymes late in the pathway. For example BioD and BioB, the last two enzymes of the pathway, are notably poor catalysts, having reported *k*~cat~ values of 0.06 and 0.002 s⁻¹, respectively. Hence, when compared with BioD (29) and BioB (17), BioC is an effective catalyst. A rationale for the disparity between the first and concluding enzymes of the biotin synthetic pathway is that BioC must have reasonable activity to effectively compete with the 3-ketoacyl-ACP synthases for malonyl-ACP. However, if BioC is overly active, it will convert too much malonyl-ACP to the methylated species and thereby block fatty acid synthesis. Hence, both BioC activity and expression must be tightly controlled. Because in *E. coli* (and in all probability *B. cereus*), the *bioC* gene is cotranscribed with the other genes of the pathway, its level of expression is coordinated with that of the later genes, and thus the activity of BioC seems set to ensure steady, low level production of malonyl-ACP methyl ester when biotin is limiting. Lévy-Schil *et al.* (22) reported that high levels of BioC expression impaired growth of *E. coli*. They postulated that an early precursor of the biotin synthetic pathway might be required by another essential metabolic pathway and that

BioC Methyltransferase

Annu. Rev. Biochem. 58, 195–221

3. Ifuku, O., Miyaoka, H., Koga, N., Kishimoto, J., Haze, S., Wachi, Y., and Kajiwara, M. (1994) Origin of the carbon atoms of biotin. Eur. J. Biochem. 220, 585–591

4. Sanyal, I., Lee, S., and Flint, D. H. (1994) Biosynthesis of pimeloyl-CoA, a biotin precursor in Escherichia coli, follows a modified fatty acid synthesis pathway. 13C-Labeling studies. J. Am. Chem. Soc. 116, 2637–2638

5. Lin, S., Hanson, R. E., and Cronan, J. E. (2010) Biotin synthesis begins by hijacking the fatty acid synthetic pathway. Nat. Chem. Biol. 6, 682–688

6. Lin, S., and Cronan, J. E. (2011) Closing in on complete pathways of biotin biosynthesis. Mol. Biosyst. 7, 1811–1821

7. Lemoine, Y., Wach, A., and Jeltsch, J. M. (1996) To be free or not. The fate of pimelate in Bacillus sphaericus and in Escherichia coli. Mol. Microbiol. 19, 645–647

8. Sanishvili, R., Yakunin, A. F., Laskowski, R. A., Skarina, T., Evdokimova, E., Doherty-Kirby, A., Lajoie, G. A., Thornton, J. M., Arrowsmith, C. H., Savchenko, A., Joachimiak, A., and Edwards, A. M. (2003) Integrating structure, bioinformatics, and enzymology to discover function. BioH, a new carboxylesterase from Escherichia coli. J. Biol. Chem. 278, 26039–26045

9. White, S. W., Zheng, J., Zhang, Y. M., and Rock. (2005) The structural biology of type II fatty acid biosynthesis. Annu. Rev. Biochem. 74, 791–831

10. Chapman-Smith, A., and Cronan, J. E., Jr. (1999) The enzymatic biotinylation of proteins. A post-translational modification of exceptional specificity. Trends Biochem. Sci. 24, 359–363

11. Tomczyk, N. H., Nettleship, J. E., Baxter, R. L., Crichton, H. J., Webster, S. P., and Campopiano, D.J. (2002) Purification and characterisation of the BioH protein from the biotin biosynthetic pathway. FEBS Lett. 513, 299–304

12. Miller, J. (1992) *A Short Course in Bacterial Genetics: A Laboratory Manual and Handbook for Escherichia coli and Related Bacteria*, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY

13. Cronan, J. E., and Thomas, J. (2009) Bacterial fatty acid synthesis and its relationships with polyketide synthetic pathways. Methods Enzymol. 459, 395–433

14. Choi-Rhee, E., and Cronan, J. E. (2005) A nucleosidase required for in vivo function of the S-adenosyl-L-methionine radical enzyme, biotin synthase. Chem. Biol. 12, 589–593

15. Quadri, L. E., Weinreb, P. H., Lei, M., Nakano, M. M., Zuber, P., and Walsh, C. T. (1998) Characterization of Sfp, a Bacillus subtilis phosphopantetheinyl transferase for peptidyl carrier protein domains in peptide synthetases. Biochemistry 37, 1585–1595

16. Park, J., Tai, J., Roessner, C. A., and Scott, A. I. (1996) Enzymatic synthesis of S-adenosyl-L-methionine on the preparative scale. Bioorg Med. Chem. 4, 2179–2185

17. Farrar, C. E., Siu, K. K., Howell, P. L., and Jarrett, J. T. (2010) Biotin synthase exhibits burst kinetics and multiple turnovers in the absence of inhibition by products and product-related biomolecules. Biochemistry 49, 9985–9996

18. Rasko, D. A., Ravel, J., Økstad, O. A., Helgason, E., Cer, R. Z., Jiang, L., Shores, K. A., Fouts, D. E., Tourasse, N. J., Angiuoli, S. V., Kolonay, J., Nelson, W. C., Kolstø, A. B., Fraser, C. M., and Read, T. D. (2004) The genome sequence of Bacillus cereus ATCC 10987 reveals metabolic adaptations and a large plasmid related to Bacillus anthracis pXO1. Nucleic Acids Res. 32, 977–988

19. Hoffman, J. L. (1986) Chromatographic analysis of the chiral and covalent instability of S-adenosyl-L-methionine. Biochemistry 25, 4444–4449

20. Borchardt, R. T., Eiden, L. E., Wu, B., and Rutledge, C. O. (1979) Sinefungin, a potent inhibitor or S-adenosylmethionine. Protein O-methyltransferase. Biochem. Biophys. Res. Commun. 89, 919–924

21. Reich, N. O., and Mashhoon, N. (1990) Inhibition of EcoRI DNA methylase with cofactor analogs. J. Biol. Chem. 265, 8966–8970

22. Lévy-Schil, S., Debussche, Rigault, S., Soubrier, F., Bacchetta, F., Lagneaux, D., Schleuniger, J., Blanch, F., Crouzet, J., and Mayaux, J.-F. (1993) Biotin biosynthetic pathway in recombinant strains of Escherichia coli overexpressing bio genes. Evidence for a limiting step upstream from KAPA. Appl. Microbiol. Biotechnol. 38, 755–762

23. Heath, R. J., and Rock, C. O. (2002) The Claisen condensation in biology. Nat. Prod. Rep. 19, 581–596

24. Harder, M. E., Ladenson, R. C., Schimmel, S. D., and Silbert, D. F. (1974) Mutants of Escherichia coli with temperature-sensitive malonyl coenzyme A-acyl carrier protein transacylase. J. Biol. Chem. 249, 7468–7475

25. Guzman, L. M., Belin, D., Carson, M. J., and Beckwith, J. (1995) Tight regulation, modulation, and high-level expression by vectors containing the arabinose PBAD promoter. J. Bacteriol. 177, 4121–4130

26. Bennett, B. D., Kimball, E. H., Gao, M., Osterhout, R., Van Dien, S. J., and Rabinowitz, J. D. (2009) Absolute metabolite concentrations and implied enzyme active site occupancy in Escherichia coli. Nat. Chem. Biol. 5, 593–599

27. Borgaro, J. G., Chang, A., Machutta, C. A., Zhang, X., and Tonge, P. J. (2011) Substrate recognition by β-ketoacyl-ACP synthases. Biochemistry 50, 10678–10686

28. Halliday, N. M., Hardie, K. R., Williams, P., Winzer, K., and Barrett, D. A. (2010) Quantitative liquid chromatography-tandem mass spectrometry profiling of activated methyl cycle metabolites involved in LuxS-dependent quorum sensing in Escherichia coli. Anal. Biochem. 403, 20–29

29. Yang, G., Sandalova, T., Lohman, K., Lindqvist, Y., and Rendina, A. R. (1997) Active site mutants of Escherichia coli dethiobiotin synthetase. Effects of mutations on enzyme catalytic and structural properties. Biochemistry 36, 4751–4760

30. Ruch, F. E., and Vagelos, P. R. (1973) The isolation and general properties of Escherichia coli malonyl coenzyme A-acyl carrier protein transacylase. J. Biol. Chem. 248, 8086–8094

31. Oefner, C., Schulz, H., D’Arcy, A., and Dale, G. E. (2006) Mapping the active site of Escherichia coli malonyl-CoA-acyl carrier protein transacylase (FabD) by protein crystallography. Acta Crystallogr. D Biol. Crystallogr. 62, 613–618

32. Cronan, J. E. (2006) A family of arabinose-inducible Escherichia coli expression vectors having pBR322 copy control. Plasmid 55, 152–157

33. De Lay, N. R., and Cronan, J. E. (2008) Genetic interaction between the Escherichia coli AcpT phosphopantetheinyl transferase and the YejM inner membrane protein. Genetics 178, 1327–1337

34. De Lay, N. R., and Cronan, J. E. (2007) In vivo functional analyses of the type II acyl carrier proteins of fatty acid biosynthesis. J. Biol. Chem. 282, 20319–20328

Enzymology:
The BioC O-Methyltransferase Catalyzes Methyl Esterification of Malonyl-Acyl Carrier Protein, an Essential Step in Biotin Synthesis

Steven Lin and John E. Cronan
J. Biol. Chem. 2012, 287:37010-37020.
doi: 10.1074/jbc.M112.410290 originally published online September 10, 2012

---

Access the most updated version of this article at doi: [10.1074/jbc.M112.410290](http://dx.doi.org/10.1074/jbc.M112.410290)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

[Click here](#) to choose from all of JBC's e-mail alerts

This article cites 33 references, 7 of which can be accessed free at [http://www.jbc.org/content/287/44/37010.full.html#ref-list-1](http://www.jbc.org/content/287/44/37010.full.html#ref-list-1)
