
Biochimica et Biophysica Acta 1827 (2013) 145–160

Contents lists available at SciVerse ScienceDirect

Biochimica et Biophysica Acta

journal homepage: www.elsevier.com/locate/bbabio

Review

Unifying concepts in anaerobic respiration: Insights from dissimilatory sulfur metabolism ✩

Fabian Grein ¹, Ana Raquel Ramos, Sofia S. Venceslau, Inês A.C. Pereira ✩*

Instituto de Tecnologia Química e Biológica, Universidade Nova de Lisboa, Oeiras, Portugal

---

**ARTICLE INFO**

Article history:

Received 16 July 2012  
Received in revised form 3 September 2012  
Accepted 4 September 2012  
Available online 11 September 2012  

Keywords:  
Anaerobic respiration  
Dissimilatory sulfur metabolism  
Sulfate reducing bacteria  
Sulfur oxidizing bacteria  
Redox module  
Respiratory membrane complex  

---

**ABSTRACT**

Behind the versatile nature of prokaryotic energy metabolism is a set of redox proteins having a highly modular character. It has become increasingly recognized that a limited number of redox modules or building blocks appear grouped in different arrangements, giving rise to different proteins and functionalities. This modularity most likely reveals a common and ancient origin for these redox modules, and is obviously reflected in similar energy conservation mechanisms. The dissimilation of sulfur compounds was probably one of the earliest biological strategies used by primitive organisms to obtain energy. Here, we review some of the redox proteins involved in dissimilatory sulfur metabolism, focusing on sulfate reducing organisms, and highlight links between these proteins and others involved in different processes of anaerobic respiration. Noteworthy are links to the complex iron–sulfur molybdoenzyme family, and heterodisulfide reductases of methanogenic archaea. We discuss how chemiosmotic and electron bifurcation/confurcation may be involved in energy conservation during sulfate reduction, and how introduction of an additional module, multiheme cytochromes c, opens an alternative bioenergetic strategy that seems to increase metabolic versatility. Finally, we highlight new families of heterodisulfide reductase-related proteins from non-methanogenic organisms, which indicate a widespread distribution for these protein modules and may indicate a more general involvement of thiol/disulfide conversions in energy metabolism. This article is part of a Special Issue entitled: The evolutionary aspects of bioenergetic systems.

© 2012 Elsevier B.V. All rights reserved.

---

### 1. Introduction

The dissimilatory metabolism of sulfur compounds is likely to have been among the earliest energy-yielding processes to sustain life [1,2]. In the early anoxic Earth H₂S and SO₂ were emitted by volcanic and hydrothermal sources, and photolysis of these compounds would also generate elemental sulfur and sulfate [3,4]. Both H₂S and S⁰ could sustain anoxygenic photosynthesis that would produce sulfate, or other oxidized sulfur species, and organic matter. Sulfate and S⁰ could serve as electron acceptors for H₂ oxidation, and disproportionation of S⁰ and sulfur compounds of intermediate oxidation state (thiosulfate, sulfite) was another possible biological strategy. There is evidence that photosynthetic processes were established at least 3.5 billion years ago [5,6], and dissimilatory sulfur metabolism was also already present at this time, either as sulfate reduction or sulfur disproportionation, as indicated by sulfur isotope fractionation studies [7–9] and microfossil records [10]. However, this biological activity had little impact on the biogeochemical cycling of sulfur until ~2.45 billion years ago [11], when a rise in atmospheric oxygen levels (Great Oxidation Event) promoted the increase of the oceanic sulfate concentration from weathering of sulfide minerals on land [12–15]. The increased oxygenation of the atmosphere was likely due to the activity of oxygen-producing cyanobacteria, which seem to have emerged at approximately the same time when O₂ started to increase, and much later than once believed [16–18]. The rising O₂ promoted weathering of continental pyrite and an increase in oceanic sulfate concentration to low mM levels [12,13,15]. However, for most of the Proterozoic the deep ocean waters remained anoxic and sulfidic or ferruginous, overlaid by an oxygenated surface layer [12,19–21], a state that may have been perpetuated until as recently as ~600 million years ago by anoxygenic photosynthesis with sulfide as electron donor [22]. After a second major oxidation event in the Neoproterozoic, the deep ocean waters became oxygenated and the sulfate levels rose to present day levels (28 mM), marking the start of the modern sulfur cycle, where biological sulfate reduction plays a major role, particularly in marine sediments

---

Abbreviations: SRO, Sulfate reducing organisms; SOB, Sulfur oxidizing bacteria; LUCA, Last universal common ancestor; CISM, Complex iron–sulfur molybdoenzymes; Tplc₃, Type I cytochrome c₃; Tpllc₃, Type II cytochrome c₃

✩ This article is part of a Special Issue entitled: The evolutionary aspects of bioenergetic systems.

* Corresponding author at: ITQB/UNL, Av. da Republica-EAN, 2780-157 Oeiras, Portugal. Tel.: +351 214468327; fax: +351 4469314.

E-mail address: ipereira@itqb.unl.pt (I.A.C. Pereira).

¹ Presently at Institut für Medizinische Mikrobiologie, Immunologie & Parasitologie, Abteilung Pharmazeutische Mikrobiologie, Rheinische Friedrich-Wilhelms-Universität Bonn, Bonn, Germany.

0005-2728/$ – see front matter © 2012 Elsevier B.V. All rights reserved.

http://dx.doi.org/10.1016/j.bbabio.2012.09.001

where it is responsible for about 50% of carbon remineralization [23]. Overall, it is clear that there was an intimate connection between the history of Earth's atmosphere and the biogeochemical cycle of sulfur (reviewed in [24,25]).

The start of widespread biological sulfate reduction between 2.45 and 2.35 billion years ago is derived from the large increase in mass-dependent sulfur isotope fractionations observed during this period (reviewed in [24,26]). A limited incidence of biological sulfate reduction in the very early Earth is also reflected in the fact that this metabolic trait is not dispersed among prokaryotic organisms, and might have initially been restricted to some early branching thermophilic sulfate reducers. The emergence of mesophilic sulfate reducing organisms (SRO) apparently coincided, or shortly followed the increase in oceanic sulfate levels [27,28]. This radiation of mesophilic SRO seems to have taken place after the rapid diversification of bacterial lineages observed during the Archaeal eon, where a significant expansion of energy metabolism genes apparently occurred [29].

A striking feature of energy metabolism/respiratory proteins is their modular character, which has been described as being based on a “redox protein construction kit” [30], from which different combinations of a limited number of protein modules originate different protein complexes with diverse physiological functions. This modular character, which is observed in many protein families, denotes a conservative approach from Nature in using a limited number of original parts to derive new metabolic features. However, it probably also reflects the

high level of gene exchange that was present in the pool of LUCA organisms [31], as well as the high incidence of lateral gene transfer in later prokaryotes [32]. In sulfur-metabolizing organisms we find interesting and unique variations of respiratory proteins that reflect their ancient origin and their close environmental association with other anaerobic organisms, in particular with methanogens. Here, we present a short review of respiratory proteins involved in dissimilatory sulfur metabolism, focusing on SRO, and discuss new “parts” of the “redox protein construction kit” that are strongly associated with sulfur metabolism but show also links to other respiratory proteins (Fig. 1) [33]. We will not discuss several respiratory membrane proteins that are present in SRO, but also in many other classes of prokaryotes, and thus are not specifically related to sulfur metabolism. A discussion of these can be found in [33].

## 2. The AprBA and DsrAB terminal reductases and their evolution

There are two biological pathways of sulfate reduction. In the assimilatory pathway, which is widespread in the three domains of life, sulfate is reduced to sulfide in small amounts and this is transformed into cysteine, from which other biological sulfur-containing molecules are derived [34]. In the dissimilatory pathway, which is restricted to five bacterial and two archaeal lineages, sulfate is the terminal electron acceptor of the respiratory pathway producing large quantities of sulfide [35–37]. The two pathways (Fig. 2) start with activation of sulfate by reaction with ATP to form adenosine-5′-phosphosulfate (APS), a step catalyzed

![Diagram](attachment:diagram.png)

**Fig. 1.** The modular nature of sulfate respiration and related proteins. A) Redox modules i.e. building blocks from the “redox construction kit” [30] that pertain to SRO. B) Heterodisulfide reductases of methanogens. C) Trimeric respiratory enzymes including the CISM family and others (Hyn hydrogenase). D) Conserved respiratory proteins of SRO (for exceptions see text; only the “minimum” unit DsrMK is present in a few organisms). E) Periplasmic and membrane complexes of cytochrome-rich SRO (mainly *Deltaproteobacteria*). The proteins and respective cofactors are represented schematically (see text for descriptions).

F. Grein et al. / Biochimica et Biophysica Acta 1827 (2013) 145–160

Sulfate  
ATP  
2Pi ← PPi  
Adenosine 5′-  
phosphosulfate (APS)  

ATP  
ADP  
3′-Phosphoadenosine 5′-  
phosphosulfate (PAPS)  

Thioredoxin<sub>Red</sub>  
Thioredoxin<sub>Ox</sub>  
Sulfite  

NADPH or Fd<sub>red</sub>  
NADP⁺ or Fd<sub>ox</sub>  
O-acetylserine sulfhydrylase  

Cysteine  

Assimilatory pathway  

Sulfate  
ATP  
2Pi ← PPi  
Adenosine 5′-  
phosphosulfate (APS)  

2 e⁻  
AMP  
APS  
reductase  
(AprAB)  

Sulfite  

6 e⁻  
Dissimilatory sulfite  
reductase  
DsrAB/ DsrC  

Sulfide  

Disimilatory pathway  

Fig. 2. The prokaryotic assimilatory and dissimilatory pathways of sulfate reduction.

by the trimeric sulfate adenylyl transferase (Sat), also known as ATP  
sulfurylase [38,39]. The formation of APS is endergonic and is driven by  
hydrolysis of the pyrophosphate formed by a pyrophosphatase (soluble  
or membrane-bound). So, the activation of sulfate to APS is considered  
to consume two ATP equivalents. In the prokaryotic assimilatory path-  
way APS is converted to 3′-phosphoadenosine-5′-phosphosulfate  
(PAPS) by the adenylyl sulfate kinase (CysC), PAPS is reduced to sulfite  
by a thioredoxin-dependent PAPS reductase (CysH), and finally sulfite  
is reduced to sulfide by an assimilatory sulfite reductase that is either  
multimeric and NADPH-dependent (CysJ) or a monomeric ferredoxin-  
dependent enzyme [40]. In the dissimilatory pathway APS is reduced  
to sulfite by the APS reductase (AprBA), a heterodimeric iron-sulfur  
flavoenzyme [41–44]. Sulfite is reduced by the dissimilatory sulfite  
reductase DsrAB, a siroheme containing protein [45,46], with the in-  
volvement of the small protein DsrC (see below) [47–51]. Another  
small protein DsrD, which is often encoded downstream of dsrAB,  
might also be involved in sulfite reduction, possibly in a regulatory  
role, but its exact function is still unknown [52]. Interestingly, the dsrD  
gene is strongly downregulated in the presence of high sulfide concen-  
trations [53]. In many anoxygenic phototrophic and chemolithotrophic  
sulfur oxidizing bacteria (SOB), the Sat, AprBA, DsrAB and DsrC proteins  
are also present, and thought to be involved in reverse oxidative reac-  
tions (reviewed in [54]). DsrAB and DsrC (and the associated DsrMKJOP  
complex, see Section 4.2) are also present in organisms that reduce  
sulfite, thiosulfate or organosulfonate compounds.

The evolution of the dissimilatory sulfate reduction pathway has been  
investigated by phylogenetic analysis of the sat [32,55], aprBA  
[32,56–59] and mostly of the dsrAB genes [32,57,60–65]. These stud-  
ies indicate a mostly vertical inheritance for these genes, but also sev-  
eral episodes of lateral gene transfer (LGT).

The APS reductase is an αβ heterodimer containing a FAD group in  
the AprA subunit and two [4Fe−4S]<sup>2+/1+</sup> clusters in the AprB subunit.  
AprBA is an example of a modular redox protein, as the AprA subunit  
shows strong structural similarity (although low sequence identity)  
to the module/family of flavoproteins containing fumarate reductase  
and aspartate oxidase, and AprB includes a domain similar to the bac-  
terial ferredoxin module [42]. The aprA and aprB genes share a similar  
evolutionary profile resulting from vertical inheritance and concurrent  
LGT. Several aprBA genes of SRO were acquired by LGT, namely  
among members of the Syntrophobacterales, Thermodesulfobacterium,  
Thermodesulfovibrio, Archaeoglobus and some deltaproteobacterial line-  
ages [56,59]. The aprBA of SOB diverge into two phylogenetic lineages  
in which one, represented by AprBA from Allochromatium vinosum, is  
the authentic SOB group (lineage I, congruent with the monophyletic  
DsrAB phylogeny), and the other, represented by AprBA from  
Chlorobium tepidum (lineage II, discordant with DsrAB phylogeny)  
was acquired by LGT from SRO [58,59]. These two lineages correlate  
with different gene organizations (Fig. 3) and different physiological  
partners of AprBA, which are the integral membrane protein AprM in  
the case of SOB lineage I, and the QmoABC membrane complex [66]  
in the case of SRO and SOB lineage II [58,59]. This is further supported  
by homology modeling of AprBA from the two groups, which suggests  
different interacting partners for AprB [67].

The DsrAB sulfite reductase forms an α<sub>2</sub>β<sub>2</sub> unit, containing two  
siroheme cofactors, per αβ unit, coupled to a [4Fe−4S] iron-sulfur  
cluster through the cysteine heme axial ligand. However, only one of  
the cofactors is catalytically active [50,68]. This protein is part of a  
large family, all sharing the same coupled cofactor, that includes also  
the assimilatory sulfite and nitrite reductases, and other proteins  
[40,69]. This family constitutes another module of the “redox construc-  
tion kit”, and probably diverged from a very ancient and primitive  
organism. The DsrA/DsrB proteins have also a modular character  
since they include a ferredoxin domain, which was probably the elec-  
tron donor to a precursor enzyme that was later incorporated into  
the reductase gene sequence [45,69]. The dsrA and dsrB genes are  
paralogous, and seem to have derived from a gene duplication event  
preceding the divergence of the Archaea and Bacteria domains  
[45,57,64,65], in agreement with a very early onset of biological sulfite  
reduction. Furthermore, the assimilatory sulfite/nitrite reductases also  
display an internal two-fold symmetry of a module that is similar to  
DsrA/DsrB, suggesting they also resulted from a gene duplication event  
[40,68–70]. In fact, the core domains of DsrAB form a unit that is  
superimposable with the structures of the assimilatory enzymes  
(Fig. 4) [50], further stressing the common origin of the assimilatory

F. Grein et al. / Biochimica et Biophysica Acta 1827 (2013) 145–160

# Sulfate Reducing Organisms

## Desulfovibrio vulgaris Hildenborough
![Diagram](https://i.imgur.com/1234567.png)

## Thermodesulfovibrio yellowstonii
![Diagram](https://i.imgur.com/2345678.png)

## Desulfotomaculum reducens
![Diagram](https://i.imgur.com/3456789.png)

# Sulfur Oxidizing Bacteria

## Allochromatium vinosum
(SOB lineage I)
![Diagram](https://i.imgur.com/4567890.png)

## Thiobacillus denitrificans
![Diagram](https://i.imgur.com/5678901.png)

## Chlorobium tepidum
(SOB lineage II)
![Diagram](https://i.imgur.com/6789012.png)

Fig. 3. Genomic organization of the sat, apr and qmo genes in selected SRO and SOB. sat, ATP sulfurylase; aprBA, APS reductase; aprM, transmembrane protein; qmoABC, subunits of the Qmo complex; hdrBC, subunits of heterodisulfide reductase. Adapted from [58,59].

---

and dissimilatory enzymes from an ancestral gene that was present in one of the earliest life forms on Earth [64,69,70]. In the assimilatory enzymes the second cofactor was lost during evolution, indicating that in both families the process of gene duplication was associated with loss of function from one of the catalytic sites. A key difference between the assimilatory and dissimilatory sulfite reductases is that the former reduce sulfite directly to sulfide, whereas the latter form, *in vitro*, a mixture of products including also trithionate and thiosulfate, in relative proportions that depend on reaction conditions [71]. The physiological significance of these products is doubtful, as they may result from the absence of an essential component in the system, DsrC [50] (see Section 4.2), and be produced by further reaction of sulfite with semi-reduced intermediates present at the active site.

The phylogeny of DsrAB has been thoroughly investigated [32,57,60–65], and indicates a main pathway of vertical transmission, with a few episodes of LGT involving members of Thermodesulfovibacterium and some low-GC Gram-positive bacteria of the phylum Firmicutes (Desulfotomaculum subclusters Ib, Ic, Id and Ie, Moorella thermoacetica and Ammonifex degensii) that acquired dsrAB from a deltaproteobacterial donor. The archaeal Archaeoglobi also have dsrAB genes of bacterial origin, indicating a cross-domain LGT. The DsrAB from SOB form a group clearly separated from SRO, while the DsrAB from the crenarchaeotal genus Pyrobaculum form a third group that represents the deepest branch in the dsrAB tree [62,63]. In the purple SOB A. vinosum [48,72], and the green SOB C. tepidum [73] it has been shown that the dsrAB and other dsr genes are essential for oxidation of sulfur globules stored in the periplasm, which are intermediates in the oxidation of sulfide and thiosulfate. In fact, most DsrAB-containing SOB are sulfur-storing members of the Chlorobi and Proteobacteria phyla [62]. If DsrAB from Pyrobaculum is of true archael origin, then the duplication of the dsr genes preceded the divergence of Bacteria and Archaea, and the ancestral DsrAB functioned in the reductive direction [63].

---

## Fig. 4. Superposition of the core structures of assimilatory and dissimilatory sulfite reductases.
The A1A2 and B1B2 domains of DsrA (blue) and DsrB (pink) subunits of the D. vulgaris Hildenborough dissimilatory sulfite reductase (PDB ID: 2v4j) [50] are superimposed on the structure of the E. coli assimilatory sulfite reductase (PDB ID: 1aop, green) [70]. The ferredoxin domains and the N- and C-terminal regions of the D. vulgaris DsrAB are omitted for clarity.
The spread of the sulfate reduction genes through a mobilizable metabolic island has been suggested [60], and gained support from the identification of genomic fragments from unidentified marine organisms containing a complete set of sulfate reduction genes [74]. However, this is unlikely to be a general mechanism, given the patchy distribution of *apr*, *dsr* and related genes in SRO, and the fact that the DsrAB tree topology is not congruent with that of AprBA, indicating independent LGT events. Nevertheless, the *Thermodesulfobacteriaceae* and *Archaeoglobi* have similar branching positions in both the AprBA and DsrAB trees pointing to a concomitant acquisition of these genes conferring the capacity to reduce sulfate to sulfide [59]. In contrast, the ancestors of *Thermodesulfovibrio* may have been sulfite reducers (congruent DsrAB and 16S rRNA phylogenies, but not AprBA) that acquired the ability to respire sulfate later.

### 3. Modularity of simple respiratory membrane complexes

The modular nature of redox proteins is particularly evident in membrane-associated respiratory complexes. The simplest family of such complexes is the complex iron–sulfur molybdoenzyme family (CISM) that operates on a variety of reducing or oxidizing substrates, including formate, nitrate and several sulfur compounds (thiosulfate, DMSO, polysulfide and tetrathionate) [75,76]. This family is widespread in bacteria and greatly contributes to the flexibility of their respiratory chains [76,77]. Phylogenetic analysis indicates that most members of this family are very ancient and were likely present in LUCA [78]. CISM proteins include three subunits, or redox modules: i) a catalytic subunit that binds a pterin-guanine dinucleotide cofactor (that includes either Mo or W), and an \([Fe-S]\) cluster; ii) a four-cluster subunit that binds four \([4Fe-4S]^{2+/1+}\) centers and is responsible for electron transfer between the membrane and catalytic subunits; and iii) a membrane subunit that has a quinone-binding site and is responsible for anchoring the other subunits to the membrane and for quinol oxidation/quinone reduction [76] (Fig. 1B). The quinone-interacting membrane subunit is the one showing more variation, and it can be broadly divided in two families: the first one comprises smaller proteins with 4 or 5 transmembrane helices (TMH), as in the case of FdnI of formate dehydrogenase and NarI of nitrate reductase, respectively. This family, which is usually referred to as the NarI-family, binds two hemes \(b\) on opposite sides of the membrane [79–81]. The hemes are coordinated by histidines present in two TMH in the case of NarI and three in the case of FdnI. The second family, which is usually referred to as the NrfD/PsrC family, includes between 8 and 10 TMH [82,83]. Sequence alignments indicate no conserved histidines to serve as heme ligands, and the structurally characterized member of this family (PsrC) does not contain hemes [82]. However, heterologous production of another protein from this family (DsrP from *A. vinosum*; see Section 4.2) unexpectedly resulted in a heme \(b\)-containing protein [84]. Therefore, it cannot be excluded that some members of this family may bind hemes.

The archetypal trimeric organization including a membrane anchor protein, an electron transfer subunit and a catalytic subunit is also found in a variety of other respiratory enzymes such as membrane-bound uptake hydrogenases, succinate dehydrogenases/fumarate reductases and others [30,85,86]. In many cases these membrane complexes are involved in energy conservation through charge separation and redox loops [87,88]. However, many succinate dehydrogenases/fumarate reductases are tetrameric, containing two membrane-bound subunits, which suggests that single membrane anchor subunits may have resulted from gene fusions (or vice-versa). In SRO several respiratory membrane complexes are variations of this archetypal organization and have a specific role in dissimilatory sulfur metabolism. The first two examples, QmoABC and DsrMKJOP (see Section 4), are strictly conserved in SRO and are physiological partners of the two terminal reductases AprBA and DsrAB. These complexes are also present in other organisms that assimilate sulfur compounds such as SOB and, in the case of the Dsr complex, in sulfite/thiosulfate/organosulfonate reducers, so they seem to have a dedicated role in sulfur metabolism. An interesting feature of QmoABC and DsrMKJOP is that they both contain subunits that are closely related to subunits of heterodisulfide reductases (Hdr). These enzymes, present in methanogenic archaea, are responsible for reducing the heterodisulfide of coenzymes B and M (CoB-S-S-CoM), which is formed in the last step of methanogenesis and is the terminal electron acceptor of the respiratory chain [89–91]. The second group of SRO membrane complexes, which includes Qrc and the Hmc/Tmc/Nhc family (see Section 6), is specific for those SRO that are rich in multiheme cytochromes \(c\) (mainly of the *Deltaproteobacteria* class).

### 4. The strictly conserved Qmo and Dsr membrane complexes

Energy conservation in SRO remains to be fully elucidated. One of the main questions that persisted for many years was the nature of the physiological electron donors to the APS and sulfite reductases. The demonstration that SRO could grow with \(\mathrm{H}_{2}\) as sole energy source [92] was a landmark achievement, since it demonstrated that sulfate reduction was associated with energy conservation through oxidative phosphorylation, and thus a membrane-associated electron transfer chain had to be present to generate a proton-motive force [93]. The role of quinones in sulfate respiration was disregarded for a long time (despite menaquinones being widespread in SRO [94]), because the redox potential of menaquinol \((E^{\circ}=-75 \, \mathrm{mV})\) was not thought to be low enough to allow reduction of APS to sulfite \((E^{\circ}=-60 \, \mathrm{mV})\) or sulfite to sulfide \((E^{\circ}=-116 \, \mathrm{mV})\). The recent study of membrane complexes in SRO, together with genetic and transcriptomic studies, and the explosion in genomic information, have contributed to a better understanding of how membrane complexes may be involved in sulfate reduction and contribute to energy conservation [33,95–97]. The nature and mechanisms of these complexes are not straightforward, as discussed below, which has hampered our understanding of how they contribute to energy conservation. We have put forward some proposals, but these still need experimental validation. Elucidating the complete pathway and mechanism of sulfate respiration is important to understand this major biogeochemical process, and a key requirement for models used to track sulfur-isotope fractionations in ancient geological samples [98].

#### 4.1. QmoABC

The Qmo complex was first described through its isolation and characterization from *Desulfovibrio desulfuricans* ATCC 27774 [66]. It is composed of three subunits, one membrane-bound (QmoC) and two cytoplasmic (QmoA and QmoB), which are all related to subunits of Hdrs. The modular character of QmoABC is quite unique and interesting, relative to the trimeric arrangement of membrane/electron transfer/catalytic proteins discussed above (Figs. 1 and 5). In fact, the QmoC subunit is unprecedented among respiratory complexes as it is constituted by the fusion of two modules: a cytochrome \(b\) transmembrane domain (of the NarI family) homologous to HdrE, and a hydrophilic cytoplasmic domain containing two \([4Fe-4S]\) cluster binding sites, homologous to the electron transfer subunit HdrC. The QmoA and QmoB subunits are both flavoproteins homologous to HdrA, the soluble Hdr subunit that has been proposed to perform flavin-based electron bifurcation in methanogens [99]. Curiously, HdrE is part of the membrane-associated HdrDE enzyme present in methylotrophic methanogens, whereas HdrC is part of the soluble HdrABC present in hydrogenotrophic methanogens [91] (see Section 7). QmoA is smaller than HdrA and includes only the flavin-binding site, whereas QmoB contains additionally two \([4Fe-4S]\) cluster binding sites and a further domain, not present in HdrA, that is homologous to MvhD, an electron transferring subunit of the \(F_{420}\)-non-reducing hydrogenase that forms a complex with the soluble HdrABC [99,100]. Cofactor analysis confirmed
F. Grein et al. / Biochimica et Biophysica Acta 1827 (2013) 145–160

![Diagram](attachment:diagram.png)

Fig. 5. Schematic representation of Hdr proteins and related complexes from SRO. Similar colors denote sequence identity. Cubes—[4Fe−4S] clusters, CCG—CXnCCGXmCX2C sequence motif, transmembrane helices are in dark blue, signal peptide in grey, H—conserved histidines, and /—hemes c.

that the Qmo complex binds two hemes b, two FAD groups and several [Fe–S] clusters [66]. The redox potentials of the two QmoC hemes are +75 and −20 mV. Since the two hemes are reduced by quinols and the qmo genes were found next to aprBA, it was proposed that QmoABC was involved in electron transfer from the quinone pool to AprBA [66]. Recently, a deletion mutant of the qmoABC genes in Desulfovibrio vulgaris Hildenborough confirmed that the Qmo complex is essential for sulfate, but not sulfite, reduction [101].

The qmo genes are conserved in all SRO genomes sequenced to date (except Caldvirga maquilingensis), usually as part of a sat-aprBA-qmoABC gene cluster [33,59] (Fig. 3), and they have a phylogenetic profile congruent with aprBA from SRO and SOB lineage II [59,67]. However, in Gram-positive SRO the qmoC gene is absent, and is possibly replaced by the soluble hdrBC genes [33,59,102]. The QmoABC complex is also present in SOB of lineage II (see Section 2) [54,58], and in C. tepidum it was shown to be essential in oxidation of sulfite as an intermediary in the sulfur oxidation pathway [103,104]. In some SOB, as in Gram-positive SRO, the qmoC gene is replaced by two hdrBC genes (Fig. 3) [58,59].

Recently, we demonstrated that there is a direct interaction between the Qmo complex and AprBA, involving QmoA [105]. However, electron transfer between quinol analogues to AprBA through QmoABC could not be observed. We suggested that the reduction of APS by menaquinol has to be energy-driven due to the small difference in redox potential between menaquinol (E′ = −75 mV) and APS (E′ APS/SO32− = −60 mV), and to the fact that the membrane potential (~150 mV) has to be overcome when transferring electrons from the quinone binding site in QmoC (likely situated towards the periplasmic side of the membrane) to AprBA in the cytoplasm. This reaction cannot be driven by the membrane potential due to the topology of the Qmo subunits, as the electron flow goes against this potential. Instead, we proposed that a third partner is required to couple the reduction of APS by menaquinol to a second more favorable reaction. Based on the similarity of QmoB to HdrA, which is responsible for electron bifurcation [99,106], we proposed that a process of reverse electron bifurcation, i.e. electron confurcation, operates during APS reduction [105]. In such a process a low-redox potential electron donor is required to allow oxidation of menaquinol by APS. Menaquinol and a cytoplasmic reductant of low redox potential (from ferredoxin, H2, formate or NADH oxidation) could both serve as electron donors to the Qmo complex, which would confurcate electrons to the APS reductase (Fig. 6A). The favorable reduction of APS by this low potential electron donor would drive the unfavorable reduction of APS by menaquinol. The FAD group of QmoA or QmoB could serve as the confurcating center, where a high redox potential “hot” flavosemiquinone [107] would be generated by the first electron coming from the low potential donor, and would then be a favorable electron acceptor for a second electron coming from menaquinol, and in practice “pulling” this electron from the quinone. Overall, the advantage of this process is that it allows for the coupling of APS reduction with chemiosmotic energy conservation. This idea of electron confurcation during APS reduction expands the growing concept that electron bifurcation/confurcation may be an ancestral form of energy coupling involving two-electron centers such as flavins, quinones, or the Mo and W metals [91,99,106–112].

### 4.2. DsrMKJOP

The dsrMKJOP genes were first identified in the purple SOB A. vinosum as part of a large gene cluster involved in the oxidation of intracellular sulfur, containing also the dsrAB and dsrC genes (Fig. 7) [72]. These genes encode a multimeric transmembrane complex that was first isolated and characterized from Archaeoglobus fulgidus [113], where it was named Hme (for Hdr-like menaquinol-oxidizing enzyme complex), and later from D. desulfuricans ATCC 27774 [51], where the dsr nomenclature was adopted since it was already clear that in several genomes of SRO, SOB and sulfite reducers, the dsrMKJOP genes were associated with dsrAB and dsrC. In the SOB A. vinosum each subunit of this complex was shown to be essential for sulfur oxidation [114], and a membrane fraction enriched in DsrKJO contained also DsrAB and DsrC, suggesting an interaction between these proteins [48]. The whole complex was also recently purified using an affinity tagged DsrJ [84].

The DsrMKJOP complex is another interesting variation in the family of respiratory complexes (Figs. 1 and 5). It includes two periplasmic subunits: DsrJ, a periplasmic tri-heme cytochrome c that shows no sequence similarity to any proteins in the databases; and DsrO that belongs to the family of four cluster proteins present in CISM (although in some SRO one of the [4Fe−4S] binding sites is missing). Two integral

F. Grein et al. / Biochimica et Biophysica Acta 1827 (2013) 145–160

**Fig. 6.** Proposed mechanisms of APS and sulfite reduction. A) Electron confurcation hypothesis (grey dashed line): menaquinol (MQH₂) and a cytoplasmic low-redox potential partner both donate electrons to the Qmo complex, which transfers them to AprBA for APS reduction [105]. B) The four electron reduction of sulfite by DsrAB generates a persulfide in DsrC, which by displacement of sulfide forms an intramolecular disulfide bridge, DsrCox. This oxidized form of DsrC is reduced by the DsrK protein of the DsrMKJOP complex [50].

membrane subunits are present: DsrM that is homologous to HdrE, binds two hemes b, has five to six TMH, and belongs to the Narl family; and DsrP that has 10 TMH and belongs to the NrfD/PsrC family. Finally, DsrK is a cytoplasmic iron-sulfur protein homologous to the catalytic subunit HdrD, which is responsible for heterodisulfide reduction by the membrane-bound HdrDE complex [115,116]. DsrK has two typical binding sites for [4Fe-4S]²⁺/¹⁺ clusters and one CCG domain (see Section 7) containing a five-cysteine motif that in Hdrs binds a catalytic [4Fe-4S]³⁺ cluster responsible for heterodisulfide reduction [89]. The characteristic EPR signal of this [4Fe-4S]³⁺ cluster is also detected in the A. fulgidus and D. desulfuricans complexes [51,113], suggesting the involvement of DsrK in thiol/disulfide chemistry. The Dsr complex

**Archaeoglobus profundus**
dsrC dsrO dsrP dsrM dsrK dsrJ

**Caldivirga maquilingensis**
dsrC dsrB dsrA dsrM dsrK

**Syntrophobacter fumaroxidans**
dsrA dsrB cbiA dsrC fd

**Desulfotomaculum reducens, C. Desulforudis audaxviator and Ammonifex degensii**
fd dsrM dsrK dsrC cbiA

**Desulfotomaculum acetoxidans**
fd dsrM dsrK dsrC cbiA dsrA dsrB dsrD

**Thermodesulfovibrio yellowstonii**
dsrC dsrM dsrK dsrJ dsrO dsrP

**Allochromatium vinosum**
dsrA dsrB dsrC dsrM dsrK dsrJ dsrO dsrP

**Fig. 7.** Genomic organization of the dsr genes in selected SRO and SOB. dsrAB, dissimilatory sulfite reductase; dsrMKJOP, subunits of the Dsr complex; dsrC, DsrC protein; cbiA, cobyrinic acid a,c-diamide synthase; fd, ferredoxin. Adapted from [33].

isolated from *D. desulfuricans* contained the full complement of cofactors, including two hemes *b* and three hemes *c* per molecule [51].

The DsrMKJOP complex appears to be a combination of two sub-complexes: the DsrMK proteins are closely related to the HdrDE proteins of methanogens, and the DsrOP subunits correspond to two of the CISM units, interacting with a new module, DsrJ. The presence of two distinct quinone-interacting proteins is striking. The DsrJOP module is probably involved in electron exchange between the periplasm and the quinone pool, and DsrMK between the membrane pool and the cytoplasm, possibly involving some form of quinone cycling. However, the complex seems to function as a complete unit as all the subunits are detected upon isolation from *A. fulgidus* and *D. desulfuricans*, so direct transmembrane electron transfer cannot be discarded. An exception is found in *Archaeoglobus profundus* where only a DsrMK complex was isolated [117], even though a single complete *dsrOPMKJ* locus is present in the genome. In the archaeon *C. maquilingensis* and the Gram-positive SRO (with the exception of the recently sequenced species of *Desulfosporinus*) only the *dsrMK* genes are present, suggesting this is a minimum functional unit of the complex [33]. Several SRO contain the *dsrMKJOP* genes, and one or more copies of *dsrMK*. Curiously, in the clostridial organisms (and in three *Deltaproteobacteria*) a ferredoxin gene is also present, which might be related to the absence of the *dsrJOP* genes [33]. The function of the DsrJ cytochrome remains enigmatic. This is a quite unique cytochrome as the three hemes have distinct ligation: His/His, His/Met and the unusual His/Cys coordination [51,118]. There are few proteins carrying this type of heme *c* coordination, and one of them, SoxXA is involved in thiosulfate oxidation [119–121]. In *A. vinosum* the Cys coordinating this heme was shown to be essential for oxidation of sulfur globules, suggesting a catalytic role for DsrJ [118]. However, in SRO no sulfur chemistry is thought to occur in the periplasm. The DsrJ cytochrome is poorly reduced by the periplasmic Type I cytochrome $c_3$ (see Section 5) [51,122].

The substrate of the DsrK subunit is proposed to be the small thiol protein DsrC, which has two conserved cysteines in a C-terminal swinging arm, and is strictly conserved in all organisms that contain a DsrAB [49–51,72,113,123]. DsrC is one of the most highly abundant energy metabolism proteins in SRO [124,125], and *dsrC* is also one of the most abundant genes in metatranscriptomic analysis of communities containing SRO and SOB [126,127], reflecting its key role in dissimilatory sulfur metabolism. DsrC belongs to a larger family (*e.g.* *Escherichia coli* TusE) where only the last Cys is conserved, that is involved in sulfur trafficking [128]. In *A. vinosum* it was recently shown that DsrC can accept sulfur from the DsrEFH proteins (present only in SOB) [129]. The two Cys of DsrC are redox active and can form a disulfide bond [49,123]. The crystal structure of the *D. vulgaris* DsrAB in complex with DsrC revealed that the C-terminal arm of this protein penetrates inside the DsrAB structure, such that its penultimate cysteine comes into close contact with the siroheme catalytic site [50]. This feature was confirmed in more recent structures from different organisms [130,131], and in the case of *Desulfovibrio gigas* a different conformation of the DsrC arm was also detected in which the arm is retracted and the two cysteines are in close contact [130]. We have proposed a new mechanism for sulfite reduction involving DsrC (Fig. 6B) [50], in which $\text{SO}_3^{2-}$ is reduced by four electrons to an $S^0$ state ($\text{Fe}^{\text{III}} - S^0 - \text{OH}$ intermediate), which is attacked by the DsrC penultimate Cys forming a persulfide. This is displaced by the other DsrC Cys forming an intramolecular disulfide bridge (DsrC$_{\text{ox}}$), which is reduced by DsrK, forming sulfide and regenerating DsrC for another catalytic cycle. An analogous model was proposed for the sulfur oxidation pathway [84]. This mechanism is supported by the observations that DsrC interacts with DsrK in both *A. vinosum* [84] and *D. vulgaris* (S.S. Venceslau and I.A.C. Pereira, unpublished results). The reductant for DsrC$_{\text{ox}}$ is presumably menaquinol, and the involvement of DsrMK in the process is analogous to the reduction of the CoM-S-S-CoB disulfide by methanophenazine catalyzed by HdrDE, which is coupled to energy conservation [132].

However, based on the presence of the DsrJOP module, it seems likely that the reduction of DsrC$_{\text{ox}}$ may involve a third partner and a more complex mechanism, which may also require electron bifurcation/confurcation, as discussed for the Qmo/Apr couple. The redox potential of DsrC$_{\text{ox}}$ has not been determined yet, but usually disulfides have E°' in the order of $-150$ to $-200$ mV, so menaquinol might also not work as sole reductant.

## 5. Multiheme cytochromes c as periplasmic redox modules in SRO

The most well studied genus of SRO, *Desulfovibrio*, belongs to the deltaproteobacterial sulfate reducers, which are characterized by an abundant pool of multiheme cytochromes $c$ ([133,134] and references therein). The prototype of this family is the tetraheme cytochrome $c_3$ (more precisely called Type I cytochrome $c_3$ or Tpl$c_3$), which was the first cytochrome $c$ to have been described in an anaerobe [135,136], and is one of the most highly expressed proteins in *Desulfovibrio* spp. Although it was believed for a long time to be an essential protein for sulfate reduction [137], the Tpl$c_3$ is in fact absent from many sulfate reducers, and some SRO such as *C. maquilingensis*, *Desulfotomaculum acetoxidans* and *Candidatus Desulforudis audaxviator* contain no cytochromes $c$ at all [33]. The SRO can be divided in two physiologically distinct groups: the first group has a high content of cytochromes $c$ and includes *Thermodesulfovibrio yellowstonii* and the deltaproteobacterial SRO (with the exception of the psychrophilic *Desulfotalea psychrophila* that has a small number [138]); the second group has few or no cytochromes $c$ and includes the archaeal and clostridial SRO [33]. The two groups differ in the relevance of periplasmic electron transfer pathways to their energy metabolism, as discussed below. It should be pointed out that all genomes of deltaproteobacterial SRO sequenced so far belong to organisms that have the potential to grow by either or both, formate and hydrogen, which is likely to be a common trait among this group of organisms.

The Tpl$c_3$ is the periplasmic electron acceptor of hydrogenases and formate dehydrogenases. The presence of the *cycA* gene coding for the Tpl$c_3$ in the genomes of SRO (often in multiple copies) correlates with the presence of periplasmic hydrogenases and formate dehydrogenases that lack a membrane subunit for direct quinone reduction [33], in contrast to most bacteria [86,139]. In several cases these enzymes have a dedicated cytochrome $c_3$ subunit [133,140,141]. These soluble uptake hydrogenases and formate dehydrogenases are usually present in several copies in the deltaproteobacterial SRO [33,141]. The enzymes have distinct expression patterns that depend on factors such as substrate concentration or metal availability. For example in *D. vulgaris* Hildenborough, the expression of hydrogenases depends on hydrogen concentration [142] and whether Ni and Se are available [143]. Likewise, different formate dehydrogenases are expressed in this organism in the presence of either Mo or W [144].

The Tpl$c_3$ is another redox module of the “construction kit”, having a compact tetraheme arrangement that performs a proton-coupled two-electron transfer [145,146]. Other cytochromes of the same family are subunits of membrane-associated complexes [133,147], described below. These include the sixteen-heme high molecular mass cytochrome (HmcA) [148] and the nine-heme cytochrome $c$ (NhcA) [149], both of which contain several Tpl$c_3$ domains, but also the Type II cytochrome $c_3$ (TplI$c_3$ or TmcA), which has small structural differences relative to Tpl$c_3$, but lacks its characteristic positive surface region [150–152]. Further members of the Tpl$c_3$ family are present in other *Deltaproteobacteria*, including the triheme cytochrome $c_7$ from *Desulfuromonas acetoxidans* [153] and the PpcA cytochromes from *Geobacter* spp. [154]. Besides the Tpl$c_3$ family, the deltaproteobacterial SRO contain other types of multiheme cytochromes $c$, as described in detail in [33,134].

As a recipient of electrons from H₂ or formate oxidation, the Tpl$c_3$ then functions as a hub for periplasmic redox networks, as it can deliver this reducing power to several membrane complexes (Qrc,

Tmc, Hmc, Nhc, and possibly others, but not the Dsr complex) or other cytochromes c [122,133,134]. We have argued that using soluble dehydrogenases and Tplc₃, rather than direct quinone reduction, confers to the deltaproteobacterial SRO a higher metabolic flexibility, as electrons can be shuttled through several alternative pathways [155]. Thus, the Deltaproteobacteria SRO may derive additional electrons from intracellular cycling of redox intermediates such as hydrogen and formate [95,137,156], relative to the other groups of SRO. A high content of multiheme cytochromes c seems to be characteristic of soil and sediment Proteobacteria, such as Geobacter, Shewanella, Anaeromyxobacter and Desulfovibrio, which are subjected to variable redox conditions. Thomas et al. have argued that having a high number of multiheme cytochromes c is a hallmark of metabolically versatile anaerobes that have to adapt to environments with fluctuating redox conditions [157]. It has also been suggested that the large pool of cytochromes c in Geobacter act as capacitors, sustaining viability and motility for short periods of time as cells move between heterogeneously dispersed metal oxides [158]. The versatile nature of SRO is reflected in the fact that they can even grow in the absence of sulfate, in syntrophy with other organisms that consume H₂ and/or formate, such as methanogens [159–161]. In fact, SRO were found to be still abundant in methanogenic zones of marine sediments [162].

### 6. Cytochrome c-associated membrane complexes of deltaproteobacterial SRO

The presence of Tplc₃ in SRO correlates also with the presence of several membrane redox complexes having a periplasmic cytochrome c subunit. These complexes have also a highly modular character, as discussed above, and they are either involved in quinone reduction (Qrc and Nhc) or transmembrane electron transfer (Tmc and Hmc). They accept electrons from the Tplc₃ and/or seem to be involved in syntrophic metabolism.

#### 6.1. The QrcABCD complex

The membrane-associated Quinone Reductase Complex (Qrc) was first described as a molybdopterin oxidoreductase involved in H₂ oxidation, by screening a library of Desulfovibrio alaskensis G20 transposon mutants for strains deficient in syntrophic growth with a methanogen [163]. Three mutants were identified with mutations in the cycA gene (Tplc₃), hydB ([FeFe] hydrogenase) and mopB (coding for a putative molybdopterin oxidoreductase). The cycA and mopB mutants were also impaired in their ability to grow with H₂ or formate (but not lactate) as electron donors for sulfate reduction, pointing to their involvement in the electron transfer chain from H₂ or formate to sulfate [163]. The Qrc complex was isolated from D. vulgaris Hildenborough, where it was shown to act as a Tplc₃:menaquinone oxidoreductase, but not to be a molybdopterin oxidoreductase, as it lacks a molybdenum or tungsten cofactor [155].

The Qrc complex is composed of four subunits, three periplasmic (QrcABC) and one integral membrane subunit (QrcD) (Figs. 1 and 8). QrcA is a membrane-anchored hexa- or pentaheme cytochrome c, QrcB is a membrane-anchored protein of the molybdopterin oxidoreductase family, but which does not contain a molybdopterin cofactor. QrcC is a four cluster protein and QrcD is an integral membrane protein of the NrfD/PsrC family. The three QrcBCD subunits are analogous to the three subunits of CISM complexes discussed above. Thus, Qrc is an interesting variation of the CISM family that includes additionally a cytochrome c subunit [155]. In addition, its subunits are also closely related to some subunits of the Alternative Complex III (Act), which performs the reverse reaction of oxidizing the quinone pool and reducing a periplasmic redox partner [164–166]. Like Qrc, the Act has a subunit related to molybdopterin oxidoreductases, which lacks a molybdopterin cofactor, as also observed for the Nqo3/NuoG subunit of Complex I [167]. The function of this protein in Qrc is presently unknown, as it is also the case for its homologues in Act and Nuo complexes, and it may have only a structural role. The D. vulgaris QrcABCD complex contains six hemes c, one [3Fe−4S]¹⁺/⁰ cluster and three [4Fe−4S]²⁺/¹⁺ [155], whose redox potentials were determined by EPR [168].

The Qrc complex is efficiently reduced by periplasmic hydrogenases and formate dehydrogenases only in the presence of Tplc₃, and can reduce menaquinone analogues, having activity as Tplc₃:menaquinone oxidoreductase [155]. Thus, Qrc constitutes the missing link between Tplc₃ and the quinone pool. The qrcABCD genes are present in Deltaproteobacteria SRO that have Tplc₃ and hydrogenases or formate dehydrogenases lacking a membrane subunit for direct quinone reduction [33,155]. The fact that it is essential for growth of D. alaskensis G20 in H₂ or formate and sulfate [163], indicates that Qrc is the physiological electron acceptor of the Tplc₃ in this organism, and cannot be replaced by other complexes such as Tmc and Hmc, which are also present in this organism. Furthermore, Qrc also seems to be implicated in syntrophic growth of this organism [163,169] and also D. vulgaris [170]. In D. vulgaris, Qrc forms a supramolecular complex with the Tplc₃ and a periplasmic hydrogenase [168]. The quinone binding site in QrcD is located close to the [3Fe−4S]¹⁺/⁰ cluster of QrcC [155]. Energy conservation by QrcABCD will depend on whether proton uptake for quinone reduction occurs on the periplasmic side of the membrane (electroneutral process), or from the cytoplasm (electrogenic process) as it has been proposed for PsrC [82]. We have suggested that the Qrc and Qmo complexes may be involved in a redox loop mechanism that sustains electron transport across the membrane to the cytoplasmic reduction of sulfate, coupled to proton motive force generation during sulfidogenic growth on H₂ or formate [155].

The evolutionary relationship between Qrc, CISM complexes and Act is an interesting issue that deserves further study. Qrc may have evolved from a CISM complex by association of a cytochrome c and loss of the molybdopterin cofactor. Yanyushin et al. have also proposed that the

![Figure 8](#fig-8)

Fig. 8. Schematic representation of proteins from Qrc, Act and CISM complexes (NarGHI and FdnGHI). Similar colors denote sequence identity. Cubes—[4Fe−4S] clusters, pyramid—[3Fe−4S] cluster, transmembrane helices are in dark blue, signal peptide in grey, H—conserved histidines, /-hemes c, MoCo-molybdopterin cofactor.

Act complex arose from Qrc by acquisition of additional subunits [166], which would place Qrc as a stepping stone in evolution of bacterial complexes. Whatever the case, Qrc is an excellent example of how a different function can be achieved with a minimal modification of subunits, a strategy that forms the basis for the diversity and flexibility of bacterial energy metabolism.

### 6.2. The Hmc, Tmc and related complexes

The Hmc and Tmc are both transmembrane complexes with subunits in the periplasm, membrane and cytoplasm (Figs. 1 and 5). From a global perspective they share some features: a cytochrome c subunit of the Tplc₃ family, one membrane cytochrome b and a cytoplasmic protein of theHdrD/DsrK family [96]. This suggests a similar function for both complexes in electron exchange between the periplasm and cytoplasm (or vice-versa), possibly involving thiol/disulfide exchanges. Curiously, most organisms that contain Hmc have also a Tmc complex [33].

The Hmc complex of D. vulgaris was the first transmembrane complex to be discovered in SRO [171]. This complex includes six subunits, HmcABCDEF, and has a composition that is strikingly similar to the Dsr complex (Fig. 1): one cytochrome c subunit (HmcA), one four-cluster protein (HmcB), two integral membrane proteins of the NrfD (HmcC) and NarI (HmcE) families and a cytoplasmic subunit homologous to HdrD (HmcF). However, the sequence identity between the Dsr and Hmc proteins is quite low, which suggests they may be paralogues that have diverged considerably. The cytochrome c subunit is actually completely different in the two complexes and points to a different function and/or physiological partner in the periplasm. HmcA is a large cytochrome with sixteen hemes organized in four Tplc₃-like domains (reviewed in [133]), and it can be reduced by this cytochrome [172]. Initial studies implicated Hmc in hydrogen uptake metabolism [173,174], but the hmc genes are downregulated during growth with H₂ [142,175]. These genes also have a low expression level in D. vulgaris Hildenborough grown in lactate/sulfate, relative to other membrane complexes [95]. A clear phenotype was observed for an hmc deletion mutant that was severely impaired in syntrophic growth on lactate with a methanogen [170]. Furthermore, comparative transcriptional analysis between syntrophic and sulphidogenic growth of D. vulgaris also indicated an upregulation of the Hmc complex in the former conditions [170]. A model was proposed in which reduced ferredoxin served as electron donor to Hmc, which then transferred electrons to Tplc₃ and this to periplasmic hydrogenases. However, there is no evidence to suggest that ferredoxin may interact with Hmc, and it seems more likely to be an electron donor to the Coo hydrogenase that is also upregulated, since energy-conserving hydrogenases are known to interact with ferredoxin [176]. This model also does not agree with previous observations that the hmc deletion mutant produced more H₂ than wild-type D. vulgaris from lactate, pyruvate or formate with limiting sulfate [156].

The hmc genes are present in all Desulfovibrio spp. sequenced to date, with the exceptions of D. desulfuricans ATCC 27774 and D. piger [33]. In these two species, instead of Hmc there is an Nhc complex, which is characterized by having a nine-heme cytochrome subunit (NhcA) that is very similar to the C-terminal domain of HmcA [149]. The Nhc complex is simpler than Hmc, lacking the cytochrome b and the cytoplasmic HdrD-like subunits. Thus, it should transfer electrons from the periplasm to the quinone pool [133].

The Tmc complex has four structural proteins, TmcABCD (in a α₂βγδ arrangement), and was isolated from D. vulgaris Hildenborough [177]. TmcA is a tetraheme cytochrome very similar to Tplc₃, also known as Type II cytochrome c₃ (Tplc₃, previously also called acidic cytochrome c₃) [150–152]. TmcB is a cytoplasmic protein of the HdrD and DsrK family with a CCG domain, and which is very similar to HmcF. TmcC is an integral membrane cytochrome b, homologous to HmcE and of the NarI/HdrE/DsrM family. Finally, TmcD is a tryptophan-rich subunit with no

homology to any protein in the database [177]. TmcA is effectively reduced by the hydrogenase/Tplc₃ couple [150,151,178] and all the redox centers of Tmc are reduced with H₂ [177]. The tmcA gene is also upregulated during growth of D. vulgaris with H₂ [175], suggesting that Tmc is involved in transmembrane electron transfer from periplasmic H₂ oxidation. In D. vulgaris grown in lactate/sulfate the tmc genes are expressed at about the same level as the dsrMKJOP genes, indicating a functional role also in this growth condition [95]. However, a mutant deleted in the tmc genes had no apparent phenotype [205], which is not surprising as other proteins can most likely substitute for its function (Hmc, Qrc coupled with Qmo/Dsr, Ohc or others). In fact, a certain degree of interchangeability is presumably the reason for the presence of multiple cytochrome c-associated complexes in the Deltaproteobacteria SRO [33]. Another example of these is the Ohc complex (for Octaheme cytochrome complex) [141], whose function is unknown, and is expressed at low levels in D. vulgaris [95].

The Hmc and Tmc complexes have in common with the Dsr complex the presence of an HdrD-related subunit. The presence of the typical catalytic cofactor for thiol-disulfide catalysis in the Tmc complex was confirmed through the characteristic [4Fe-4S]³⁺ cluster EPR signal [177]. Thus, Hmc and Tmc may also act as disulfide reductases, possibly on the DsrCox protein, and thus link the periplasm with sulfite reduction. Given the considerable number of Hdr-related proteins in SRO and other organisms, we dedicate the following section to an analysis of these proteins, which have not been much investigated up to date outside of methanogens.

### 7. Hdr-related proteins as widespread redox modules in anaerobic respiration

In a recent genomic analysis of energy metabolism genes in 25 species of SRO we described the very high number of genes related to heterodisulfide reductases [33], which has also been pointed out by other authors in the context of individual genomes [102,179–182], or in other classes of organisms such as the acetogenic Moorella thermoacetica [183]. The abundance of Hdr-like proteins in SRO [33,51,66,102,113,117,141,180] may indicate they were present in ancestral organisms, and/or that there was substantial exchange of genetic material between methanogens and SRO (and other classes of organisms), which could be due to their sharing common habitats. Hdrs are representative enzymes of a group of quite widespread proteins responsible for reduction of disulfides or oxidation of thiols [90], but they belong to a larger family that includes proteins that seem to have other functions (see below). In methanogenic archaea, the heterodisulfide is not an external substrate, but is produced in the final step of methanogenesis. By analogy, it is possible that thiol/disulfides may be generated in other anaerobes and be involved in the respiratory chain, which would suggest that a sulfur-based energy metabolism, of obvious ancient origin, could be more widespread than presently considered [90,109].

There are two types of Hdr enzymes [91]: in methanogens without cytochromes a soluble HdrABC is present [89], which forms a complex with the MvhADG hydrogenase. This complex couples the endergonic reduction of ferredoxin by H₂ with the exergonic reduction of the heterodisulfide by H₂, through an electron bifurcation process [99]. In methanogens with cytochromes, a membrane-bound enzyme is present, HdrDE, which uses the quinone-like cofactor methanophenazine as electron donor in a process coupled to energy conservation [116,132,184]. The key subunits in Hdrs are the catalytic subunits (HdrB in the soluble enzyme and HdrD in the membrane-bound enzyme; actually HdrD resembles a fusion of the HdrBC proteins), and the HdrA subunit that contains a FAD group presumed to be responsible for bifurcation of electrons coming from the Mvh hydrogenase. There are several proteins related to both HdrA and HdrD in the genomes of SRO [33]. Several of these are multidomain proteins, and Strittmatter et al. proposed two new types of Hdr subunits, HdrF and HdrL based on proteins encoded in the

Desulfobacterium autotrophicum HRM2 genome (see below) [180]. Overall, both HdrA and HdrD (or more precisely the CCG domain) can be considered as additional modules of the “redox construction kit” that we discuss further below.

### 7.1. HdrA-related proteins

A complete set of *hdrABC* genes is found in many SRO, either next to a set of *mvhDGA* genes for an Mvh [NiFe] Hase, or next to a set of *floxABCD* genes (for flavin oxidoreductase) [33]. In many cases the *hdrBC* genes are absent (*hdrA-mvhDGA* or *hdrA-floxABCD* sets). We proposed that these proteins are part of electron-transfer pathways from oxidation of H₂ or ethanol involving energy coupling through electron bifurcation. A group of multidomain HdrA-like proteins was defined by Strittmatter et al. as HdrL. These are large proteins containing an NADH binding site and, in some cases, a fumarate reductase domain [180] (Fig. 9). With a few exceptions, they are restricted to the sulfate/sulfite reducing *Deltaproteobacteria* and *Firmicutes*. It is noteworthy that some of the HdrL (and HdrA) proteins contain selenocysteine, and that there is a conserved CxxCxxCxxCxxxC motif of unknown function present in all available HdrL sequences. The *hdrL* genes are usually found in loci together with *hdrA* and genes coding for a formate dehydrogenase or a pyruvate:ferredoxin oxidoreductase [33], indicating that pyruvate and formate may serve as electron donors for reduction of HdrA/L. Often an *mvhD* gene is found next to *hdrA* or fused to it (as also observed in QmoA).

Hdr proteins are also noteworthy in the genome of the acetogenic bacterium *M. thermoacetica* [183], where they are present in four gene loci including three HdrL proteins. One is an *hdrABC* locus, the other includes the *hdrLBC* genes next to the acetyl-CoA synthase (*acs*) genes, the third is an *hdrLBC–floxABCD* cluster and finally there is an *hdrDL* locus, where *hdrD* shows high similarity to *hmcF*.

HdrA and HdrD modules are also present in the benzoyl-coenzyme A reductase complex BamBCDEFGHI, present in several anaerobes capable of degrading aromatic compounds [185–187]. This large complex includes the active site subunit BamB, which contains a tungstopterin cofactor, and the iron–sulfur protein BamC that shows similarity to the electron transfer subunit of hydrogenases. BamD and BamE are

![Diagram](attachment:diagram.png)

Fig. 9. Schematic representation of Hdr-related proteins. Similar colors denote sequence identity. Cubes—[4Fe–4S] clusters, CCG—CXₙCCGXₘCX₂C sequence motif, transmembrane helices are in dark blue, H—conserved histidines, and CXXCH—possible heme c binding sequence.

HdrD- and HdrA-like proteins, while BamF shows similarity to MvhD and contains selenocysteine. BamGHI are similar to the soluble components of Complex I [187]. The BamBCDEFGHI complex is another striking example of the highly modular character of redox proteins, in this case with a quite intricate arrangement that suggests a complex mechanism.

### 7.2. The CCG protein family

The catalytic subunits of Hdrs (HdrB and HdrD) are characterized by the presence of the so-called CCG domain (Pfam database accession number PF02754). In this domain up to five cysteines with the sequence CXₙCCGXₘCX₂C are usually found, where two tandem cysteines are followed by a glycine, which led to the designation of the CCG domain. To date more than 5000 protein sequences are present in the databases that contain either one or two CCG domains.

The family of CCG proteins can be divided into three main groups: I—proteins lacking TMH containing one or two CCG domains, and optionally additional [Fe–S] clusters; II—proteins predicted to be membrane bound; and III—proteins that contain a FAD binding site. The proteins in groups II and III are large proteins with a highly modular character as they include several distinct domains.

Group I includes the HdrB and HdrD proteins, which contain two CCG domains (Fig. 9). The C-terminal domain binds the catalytic [4Fe–4S] cluster, while the N-terminal domain provides ligands to a zinc site [188]. Many proteins in this group are membrane associated, although they lack TMH, suggesting a monotopic membrane anchoring [84,189–191]. Several of the proteins belonging to group I are part of membrane complexes in organisms capable of dissimilatory sulfur metabolism, as described above: DsrK, TmcB and HmcF. The [4Fe–4S]³⁺ characteristic EPR signal has been detected in the case of DsrK and TmcB [51,177]. The substrate of these proteins has been proposed to be DsrC.

Many group I proteins are subunits of oxidoreductases, including succinate:quinone oxidoreductase, thiol:fumarate reductase, glycolate oxidase, anaerobic glycerol-3-phosphate dehydrogenase and lactate dehydrogenase. The SdhE subunit of *Sulfolobus solfataricus* P2 succinate dehydrogenase has been studied in detail [190]. SdhE has two CCG domains that bind a [4Fe–4S]³⁺/²⁺ cluster and a zinc, and serves as the monotopic membrane anchor of the enzyme. The function of the [4Fe–4S]³⁺/²⁺ cluster is not known, but it has been speculated that it either mediates electron transfer to the quinone pool or that it has a structural role [190]. The thiol:fumarate reductase (Tfr) from *Methanobacterium autotrophicum* uses the thiols CoM and CoB as electron donors for the reduction of fumarate, producing succinate and CoB-S-S-CoM. TfrA is a flavoprotein carrying the catalytic site for the fumarate reduction, while TfrB is an iron–sulfur protein containing two CCG domains, that probably oxidizes the thiol substrates in analogy to Hdr [192]. CCG proteins present in *E. coli* are subunits of the glycolate oxidase (GlcF) and anaerobic glycerol-3-phosphate dehydrogenase subunit (GlpC). While Glc catalyzes the oxidation of glycolate to glyoxylate, Glp converts glycerol-3-phosphate into dihydroxyacetone phosphate. However, neither of these subunits has been biochemically characterized with respect to the CCG domain. A CCG protein is also a subunit of a recently described family of lactate dehydrogenases named LldEFG or LutABC [193–195] that is also widespread in SRO [33]. The BamB protein, which is part of the large benzoyl-CoA reductase complex BamBCDEFGHI discussed above [187], is also a member of this group.

Another member of the group I proteins is a subunit of the Isp [NiFe] uptake hydrogenases present in both archaea and bacteria [139,196,197]. Isp hydrogenases include two subunits (Isp1 and Isp2) similar to the HdrDE or DsrMK modules, besides the typical large and small hydrogenase subunits. The presence of an HdrD-like subunit suggests a link to sulfur metabolism, and in fact the best characterized

members of this family are from the SOB *Thiocapsa roseopersina* [197] and *A. vinosum* [196].

The CCG proteins belonging to group II are characterized by the presence of predicted TMH. This group includes the new family of HdrF proteins recently proposed by Strittmater et al. [180], based on proteins from *Db. autotrophicum* (Fig. 9). These multidomain proteins typically contain 4 to 6 predicted TMH at the N-terminus. In HdrF1, this transmembrane domain is followed by a GlpC-like and an HdrC-like domain binding four \[4Fe–4S\] clusters. The *hdrF1* gene is followed by a gene similar to *hdrB* that was named *hdrF1′*. This arrangement is only found in 10 gene loci, and is restricted to deltaproteobacterial SRO. *Db. autotrophicum* contains also a fusion protein of HdrF1 and HdrF1′ designated as HdrF2.

In a large number of HdrF proteins, the transmembrane region is annotated as a NarI domain. In NarI, the hemes are ligated by conserved histidines located in TMH 2 and 5, respectively. These residues are conserved in the HdrF1 proteins and in each case they are predicted to be located in transmembrane helices. The distribution of the TMH along NarI is rather even. In contrast, a rather large soluble loop is predicted to be present in HdrF1 proteins. Interestingly, a conserved CXXCH motif can be found in all available sequences. Since this loop is predicted to be located in the periplasm, this would allow covalently binding of heme *c* to this motif. If this is true, HdrF1 and HdrF2 are unique proteins containing hemes *b*, *c* and \[Fe–S\] clusters. However, only ten sequences are available to date limiting statistical validation of the alignments, and further experimental evidence will be needed to corroborate this.

Far more distributed is the HdrF3 protein, which is widespread amongst the Deltaproteobacteria, but also in other bacteria and in two archaea. It lacks the GlpC-like superdomain and contains either one or two CCG domains. Only 3 of the 4 histidines that ligate the hemes *b* in NarI are conserved. Thus, HdrF3 may bind only one heme or the second heme has an alternative coordination. However, there is another conserved histidine in helix 5, which may be another candidate ligand. The CXXCH motif present in HdrF1 is missing. In some SRO and other organisms, the gene for HdrF3 is located next to *etfAB* genes encoding an electron-transfer flavoprotein indicating that these proteins transport electrons to or from HdrF3.

The FAD-binding CCG proteins of group III are a new class of Hdr-like proteins that have not been described before and we propose to name HdrG. They are widespread among the Bacteria (in particular within the Proteobacteria and the Firmicutes), and are also present in the Euryarchaeota. These proteins include an N-terminal GlcD multidomain region fused to another GlpC multidomain region that includes either one or two CCG sites. The GlpC multidomain consists of a FAD binding region (PF01565) (not related to HdrA) that is followed by one or two FAD oxidase domains (PF02913). The glycolate oxidase D (GlcD) domain is related to several D-lactate dehydrogenases. In the Betaproteobacteria and in the Chlorobi there are HdrG proteins containing an additional N-terminal domain of unknown function (DUF3683). It appears that HdrG are most likely FAD linked oxidoreductases. In some cases the *hdrG* genes are located next to lactate transporters and in some cases also next to the three-subunit L-lactate dehydrogenase mentioned above. However, in the majority of cases the gene localization does not allow speculations about the protein function. As for the CCG proteins of group II there is yet no biochemically characterized HdrG protein.

In summary, it appears that the CCG domain represents a quite widespread module in redox proteins, where it may have developed different functions. These include the ligation of an \[Fe–S\] cluster with a catalytic or structural role, the ligation of a Zn site with a so far unknown function, and possibly others. Further research on this family is certainly needed, since only a few proteins are biochemically characterized, despite the large number of CCG proteins in the database, and so far no structural information is available. A phylogenetic analysis of this family is also warranted.


8. Conclusions

The modular nature of respiratory proteins is well apparent in proteins from SRO. In particular, several membrane-associated redox complexes from these organisms present new and interesting variations of the typical trimeric arrangement of simple respiratory proteins (catalytic, electron transfer and membrane anchor/quinone binding subunits). These variations may reflect the fact that the SRO complexes do not act directly on organic/inorganic substrates, as observed in the CISM family, but rather interact with other redox proteins, which considerably complicates *in vitro* studies and elucidation of their bioenergetic mechanisms. The SRO membrane complexes have a dedicated role in sulfur metabolism as they are also found in many SOB and organisms dissimilating other sulfur compounds, such as sulfite, thiosulfate and organosulfonates. Several of the proteins involved are related to subunits of heterodisulfide reductases of methanogenic archaea, which probably reflects a common ancient origin of sulfur-metabolizing organisms and methanogens, and their close environmental association. The mechanisms of energy conservation of these membrane complexes of SRO have not been clearly established, but may involve both chemiosmotic and electron bifurcation/confurcation processes that seem to be ancestral forms of energy coupling.

A subset of SRO, mainly of the Deltaproteobacteria, relies on multiheme cytochromes *c* as additional redox modules to diversify their respiratory metabolism. The prototype protein is the Tplc₃ that functions as a hub in periplasmic electron transfer pathways, with links to several membrane complexes having also a cytochrome *c* subunit. One of these complexes, QrcABCD, is closely related to the CISM family and seems to be a cross-point in the evolution of bacterial complexes. It is an excellent example of how a different function can be achieved with a minimal modification of subunits.

Finally, the Hdr proteins, namely HdrD and HdrA, seem to be model proteins for a larger family with a wide distribution. In particular, a large group of proteins include the CXₙCCGXₘCX₂C sequence motif (CCG domain), characteristic of HdrD/HdrB. Novel proteins of the Hdr family have been proposed, namely HdrL (related to HdrA), and HdrF and HdrG (related to HdrD), which are constituted by multiple domains. The function of many of these proteins is still unknown, but their similarity to Hdrs may suggest that sulfur-based metabolic pathways may be more widespread than presently considered.


Acknowledgements

This work was supported by grants PTDC/QUI-BIQ/100591/2008 and PTDC/BIA-MIC/104030/2008 funded by Fundação para a Ciência e Tecnologia (FCT, Portugal) and FEDER program. ARR and SSV are recipients of fellowships SFRH/BD/60500/2009 and SFRH/BPD/79823/2011, respectively. We thank David T. Johnston for critical reading and helpful discussions.


References

[1] D.E. Canfield, R. Raiswell, The evolution of the sulfur cycle, Am. J. Sci. 299 (1999) 697–723.

[2] D.E. Canfield, M.T. Rosing, C. Bjerrum, Early anaerobic metabolisms, Philos. Trans. R. Soc. Lond. B Biol. Sci. 361 (2006) 1819–1834.

[3] J. Farquhar, J. Savarino, S. Airieau, M.H. Thiemens, Observation of wavelength-sensitive mass-independent sulfur isotope effects during SO₂ photolysis: implications for the early atmosphere, J. Geophys. Res. 106 (2001) 32829–32839.

[4] A.A. Pavlov, J.F. Kasting, Mass-independent fractionation of sulfur isotopes in Archean sediments: strong evidence for an anoxic Archean atmosphere, Astrobiology 2 (2002) 27–41.

[5] M.F. Hohmann-Marriott, R.E. Blankenship, Evolution of photosynthesis, Annu. Rev. Plant Biol. 62 (2011) 515–548.

[6] M.M. Tice, D.R. Lowe, Photosynthetic microbial mats in the 3,416-Myr-old ocean, Nature 431 (2004) 549–552.

[7] P. Philippot, M. Van Zuilen, K. Lepot, C. Thomazo, J. Farquhar, M.J. Van Kranendonk, Early Archaeal microorganisms preferred elemental sulfur, not sulfate, Science 317 (2007) 1534–1537.

[8] Y. Shen, R. Buick, D.E. Canfield, Isotopic evidence for microbial sulphate reduction in the early Archaeon era, Nature 410 (2001) 77–81.

[9] D. Wacey, N. McLoughlin, M.J. Whitehouse, M.R. Kilburn, Two coexisting sulfur metabolisms in a ca. 3400 Ma sandstone, Geology 38 (2010) 1115–1118.

[10] D. Wacey, M.R. Kilburn, M. Saunders, J. Cliff, M.D. Brasier, Microfossils of sulphur-metabolizing cells in 3.4-billion-year-old rocks of Western Australia, Nat. Geosci. 4 (2011) 698–702.

[11] J. Farquhar, H.M. Bao, M. Thiemens, Atmospheric influence of Earth's earliest sulfur cycle, Science 289 (2000) 756–758.

[12] D.E. Canfield, A new model for Proterozoic ocean chemistry, Nature 396 (1998) 450–453.

[13] D.E. Canfield, K.S. Habicht, B. Thamdrup, The Archean sulfur cycle and the early history of atmospheric oxygen, Science 288 (2000) 658–661.

[14] J. Farquhar, M. Peters, D.T. Johnston, H. Strauss, A. Masterson, U. Wiechert, A.J. Kaufman, Isotopic evidence for Mesoarchaean anoxia and changing atmospheric sulphur chemistry, Nature 449 (2007) 706–709.

[15] K.S. Habicht, M. Gade, B. Thamdrup, P. Berg, D.E. Canfield, Calibration of sulfate levels in the Archean Ocean, Science 298 (2002) 2372–2374.

[16] F.U. Battistuzzi, A. Feijao, S.B. Hedges, A genomic timescale of prokaryote evolution: insights into the origin of methanogenesis, phototrophy, and the colonization of land, BMC Evol. Biol. 4 (2004).

[17] D.F. Feng, G. Cho, R.F. Doolittle, Determining divergence times with a protein clock: update and reevaluation, Proc. Natl. Acad. Sci. 94 (1997) 13028–13033.

[18] B. Rasmussen, I.R. Fletcher, J.J. Brocks, M.R. Kilburn, Reassessing the first appearance of eukaryotes and cyanobacteria, Nature 455 (2008) 1101–U1109.

[19] A.D. Anbar, A.H. Knoll, Proterozoic ocean chemistry and evolution: a bioinorganic bridge? Science 297 (2002) 1137–1142.

[20] D.E. Canfield, S.W. Poulton, G.M. Narbonne, Late-Neoproterozoic deep-ocean oxygenation and the rise of animal life, Science 315 (2007) 92–95.

[21] C. Scott, T.W. Lyons, A. Bekker, Y. Shen, S.W. Poulton, X. Chu, A.D. Anbar, Tracing the stepwise oxygenation of the Proterozoic ocean, Nature 452 (2008) 456–459.

[22] D.T. Johnston, F. Wolfe-Simon, A. Pearson, A.H. Knoll, Anoxygenic photosynthesis modulated Proterozoic oxygen and sustained Earth's middle age, Proc. Natl. Acad. Sci. 106 (2009) 16925–16929.

[23] B.B. Jørgensen, Mineralization of organic matter in the sea-bed — the role of sulphate reduction, Nature 390 (1982) 364–370.

[24] J. Farquhar, N.P. Wu, D.E. Canfield, H. Oduro, Connections between sulfur cycle evolution, sulfur isotopes, sediments, and base metal sulfide deposits, Econ. Geol. 105 (2010) 509–533.

[25] T.W. Lyons, B.C. Gill, Ancient sulfur cycling and oxygenation of the early biosphere, Elements 6 (2010) 93–99.

[26] D.T. Johnston, Multiple sulfur isotopes and the evolution of Earth's surface sulfur cycle, Earth Sci. Rev. 106 (2011) 161–183.

[27] C.E. Blank, Evolutionary timing of the origins of mesophilic sulphate reduction and oxygenic photosynthesis: a phylogenomic dating approach, Geobiology 2 (2004) 1–20.

[28] C.E. Blank, Phylogenomic dating—the relative antiquity of archaeal metabolic and physiological traits, Astrobiology 9 (2009) 193–219.

[29] L.A. David, E.J. Alm, Rapid evolutionary innovation during an Archaeal genetic expansion, Nature 469 (2011) 93–96.

[30] F. Baymann, E. Lebrun, M. Brugna, B. Schoepp-Cothenet, M.T. Giudici-Orticoni, W. Nitschke, The redox protein construction kit: pre-last universal common ancestor evolution of energy-conserving enzymes, Philos. Trans. R. Soc. Lond. B Biol. Sci. 358 (2003) 267–274.

[31] C.R. Woese, Interpreting the universal phylogenetic tree, Proc. Natl. Acad. Sci. 97 (2000) 8392–8396.

[32] Y. Boucher, C.J. Douady, R.T. Papke, D.A. Walsh, M.E.R. Boudreau, C.L. Nesbo, R.J. Case, W.F. Doolittle, Lateral gene transfer and the origins of prokaryotic groups, Annu. Rev. Genet. 37 (2003) 283–328.

[33] I.A.C. Pereira, A.R. Ramos, F. Grein, M.C. Marques, S.M. da Silva, S.S. Venceslau, A comparative genomic analysis of energy metabolism in sulfate reducing bacteria and archaea, Front. Microbiol. 2 (2011) 69.

[34] T. Leustek, M.N. Martin, J.A. Bick, J.P. Davies, Pathways and regulation of sulfur metabolism revealed through molecular and genetic studies, Annu. Rev. Plant Physiol. Plant Mol. Biol. 51 (2000) 141–165.

[35] L.L. Barton, G.D. Fauque, Biochemistry, physiology and biotechnology of sulfate-reducing bacteria, Adv. Appl. Microbiol. 68 (2009) 41–98.

[36] G. Muyzer, A.J. Stams, The ecology and biotechnology of sulphate-reducing bacteria, Nat. Rev. Microbiol. 6 (2008) 441–454.

[37] R. Rabus, T. Hansen, F. Widdel, Dissimilatory sulfate- and sulfur-reducing prokaryotes, in: M.e.a. Dworkin (Ed.), The Prokaryotes, vol. 2, Springer-Verlag, New York, 2007, pp. 659–768, [http://link.springer-ny.com/link/service/books/10125/](http://link.springer-ny.com/link/service/books/10125/).

[38] Y. Taguchi, M. Sugishima, K. Fukuyama, Crystal structure of a novel zinc-binding ATP sulfurylase from *Thermus thermophilus* HB8, Biochemistry 43 (2004) 4111–4118.

[39] T.C. Ullrich, M. Blaesse, R. Huber, Crystal structure of ATP sulfurylase from *Saccharomyces cerevisiae*, a key enzyme in sulfate activation, EMBO J. 20 (2001) 316–329.

[40] B.R. Crane, E.D. Getzoff, The relationship between structure and function for the sulfite reductases, Curr. Opin. Struct. Biol. 6 (1996) 744–756.

[41] G. Fritz, T. Büchert, P.M.H. Kroneck, The function of the [4Fe−4S] clusters and FAD in bacterial and archaeal adenylylsulfate reductases. Evidence for flavin-catalyzed reduction of adenosine 5′-phosphosulfate, J. Biol. Chem. 277 (2002) 26066–26073.

[42] G. Fritz, A. Roth, A. Schiffer, T. Buchert, G. Bourenkov, H.D. Bartunik, H. Huber, K.O. Stetter, P.M. Kroneck, U. Ermler, Structure of adenylylsulfate reductase from the hyperthermophilic *Archaeoglobus fulgidus* at 1.6-Å resolution, Proc. Natl. Acad. Sci. 99 (2002) 1836–1841.

[43] J. Lampreia, A.S. Pereira, J.J.G. Moura, Adenylylsulfate reductases from sulfate-reducing bacteria, Methods Enzymol. 243 (1994) 241–260.

[44] N. Speich, C. Dahl, P. Heisig, A. Klein, F. Lottspeich, K.O. Stetter, H.G. Trüper, Adenylylsulphate reductase from the sulfate-reducing archaeon *Archaeoglobus fulgidus* — cloning and characterization of the genes and comparison of the enzyme with other iron-sulfur flavoproteins, Microbiology 140 (1994) 1273–1284.

[45] C. Dahl, N.M. Kredich, R. Deutzmann, H.G. Trüper, Dissimilatory sulphite reductase from *Archaeoglobus fulgidus*: physico-chemical properties of the enzyme and cloning, sequencing and analysis of the reductase genes, J. Gen. Microbiol. 139 (1993) 1817–1828.

[46] I. Moura, J. LeGall, A.R. Lino, H.D. Peck, G. Fauque, A.V. Xavier, D.V. Dervartanian, J.J.G. Moura, B.H. Huynh, Characterization of two dissimilatory sulfite reductases (desulforubidin and desulfovirdin) from the sulfate-reducing bacteria — mossbauer and EPR studies, J. Am. Chem. Soc. 110 (1988) 1075–1082.

[47] J.R. Cort, S.V. Mariappan, C.Y. Kim, M.S. Park, T.S. Peat, G.S. Waldo, T.C. Terwilliger, M.A. Kennedy, Solution structure of *Pyrobaculum aerophilum* DsrC, an archaeal homologue of the gamma subunit of dissimilatory sulfite reductase, Eur. J. Biochem. 268 (2001) 5842–5850.

[48] C. Dahl, S. Engels, A.S. Pott-Sperling, A. Schulte, J. Sander, Y. Lübbe, O. Deuster, D.C. Brune, Novel genes of the dsr gene cluster and evidence for close interaction of Dsr proteins during sulfur oxidation in the phototrophic sulfur bacterium *Allochromatium vinosum*, J. Bacteriol. 187 (2005) 1392–1404.

[49] G.J. Mander, M.S. Weiss, R. Hedderich, J. Kahnt, U. Ermler, E. Warkentin, X-ray structure of the gamma-subunit of a dissimilatory sulfite reductase: fixed and flexible C-terminal arms, FEBS Lett. 579 (2005) 4600–4604.

[50] T.F. Oliveira, C. Vonrhein, P.M. Matias, S.S. Venceslau, I.A.C. Pereira, M. Archer, The crystal structure of *Desulfovibrio vulgaris* dissimilatory sulfite reductase bound to DsrC provides novel insights into the mechanism of sulfate respiration, J. Biol. Chem. 283 (2008) 34141–34149.

[51] R.H. Pires, S.S. Venceslau, F. Morais, M. Teixeira, A.V. Xavier, I.A. Pereira, Characterization of the *Desulfovibrio desulfuricans* ATCC 27774 DsrMKJOP complex—A membrane-bound redox complex involved in the sulfate respiratory pathway, Biochemistry 45 (2006) 249–262.

[52] N. Mizuno, G. Voordouw, K. Miki, A. Sarai, Y. Higuchi, Crystal structure of dissimilatory sulfite reductase D (DsrD) protein — possible interaction with B- and Z-DNA by its winged-helix motif, Structure 11 (2003) 1133–1140.

[53] S.M. Caffrey, G. Voordouw, Effect of sulfide on growth physiology and gene expression of *Desulfovibrio vulgaris* Hildenborough, Antonie Van Leeuwenhoek 97 (2010) 11–20.

[54] N.U. Frigaard, C. Dahl, Sulfur metabolism in phototrophic sulfur bacteria, Adv. Microb. Physiol. 54 (2009) 103–200.

[55] D. Sperling, U. Kappler, A. Wynen, C. Dahl, H.G. Trüper, Dissimilatory ATP sulfurylase from the hyperthermophilic sulfate reducer *Archaeoglobus fulgidus* belongs to the group of homo-oligomeric ATP sulfurylases, FEMS Microbiol. Lett. 162 (1998) 257–264.

[56] M.W. Friedrich, Phylogenetic analysis reveals multiple lateral transfers of adenosine-5′-phosphosulfate reductase genes among sulfate-reducing microorganisms, J. Bacteriol. 184 (2002) 278–289.

[57] W.M. Hipp, A.S. Pott, N. Thum-Schmitz, I. Faath, C. Dahl, H.G. Truper, Towards the phylogeny of APS reductases and sirohaem sulfite reductases in sulfate-reducing and sulfur-oxidizing prokaryotes, Microbiology 143 (Pt. 9) (1997) 2891–2902.

[58] B. Meyer, J. Kuever, Molecular analysis of the distribution and phylogeny of dissimilatory adenosine-5′-phosphosulfate reductase-encoding genes (aprBA) among sulfur-oxidizing prokaryotes, Microbiology 153 (2007) 3478–3498.

[59] B. Meyer, J. Kuever, Phylogeny of the alpha and beta subunits of the dissimilatory adenosine-5′-phosphosulfate (APS) reductase from sulfate-reducing prokaryotes—origin and evolution of the dissimilatory sulfate-reduction pathway, Microbiology 153 (2007) 2026–2044.

[60] M. Klein, M. Friedrich, A.J. Roger, P. Hugenholtz, S. Fishbain, H. Abicht, L.L. Blackall, D.A. Stahl, M. Wagner, Multiple lateral transfers of dissimilatory sulfite reductase genes between major lineages of sulfate-reducing prokaryotes, J. Bacteriol. 183 (2001) 6028–6035.

[61] O. Larsen, T. Lien, N.K. Birkeland, Dissimilatory sulfite reductase from *Archaeoglobus profundus* and *Desulfotomaculum thermocisternum*: phylogenetic and structural implications from gene sequences, Extremophiles 3 (1999) 63–70.

[62] A. Loy, S. Duller, C. Baranyi, M. Mussmann, J. Ott, I. Sharon, O. Beja, D. Le Paslier, C. Dahl, M. Wagner, Reverse dissimilatory sulfite reductase as phylogenetic marker for a subgroup of sulfur-oxidizing prokaryotes, Environ. Microbiol. 11 (2009) 289–299.

[63] A. Loy, S. Duller, M. Wagner, Evolution and ecology of microbes dissimilating sulfur compounds: insights from siroheme sulfite reductases, in: C. Dahl, C. Friedrich (Eds.), Microbial Sulfur Metabolism, Springer, Heidelberg, 2007, pp. 46–59.

[64] M. Molitor, C. Dahl, I. Molitor, U. Schafer, R. Huber, R. Deutzmann, H.G. Truper, A dissimilatory sirohaem-sulfite-reductase-type protein from the hyperthermophilic archaeon *Pyrobaculum islandicum*, Microbiology 144 (Pt. 2) (1998) 529–541.

[65] M. Wagner, A.J. Roger, J.L. Flax, G.A. Brusseau, D.A. Stahl, Phylogeny of dissimilatory sulfite reductases supports an early origin of sulfate respiration, J. Bacteriol. 180 (1998) 2975–2982.

[66] R.H. Pires, A.I.C. Lourenço, F. Morais, M. Teixeira, A.V. Xavier, L.M. Saraiva, I.A. Pereira, A novel membrane-bound respiratory complex from *Desulfovibrio desulfuricans* ATCC 27774, Biochim. Biophys. Acta 1605 (2003) 67–82.

[67] B. Meyer, J. Kuever, Homology modeling of dissimilatory APS reductases (AprBA) of sulfur-oxidizing and sulfate-reducing prokaryotes, PLoS One 3 (2008) e1514.

[68] A. Schiffer, K. Parey, E. Warkentin, K. Diederichs, H. Huber, K.O. Stetter, P.M. Kroneck, U. Ermler, Structure of the dissimilatory sulfite reductase from the hyperthermophilic archaeon *Archaeoglobus fulgidus*, J. Mol. Biol. 379 (2008) 1063–1074.

[69] A. Dhillon, S. Goswami, M. Riley, A. Teske, M. Sogin, Domain evolution and functional diversification of sulfite reductases, Astrobiology 5 (2005) 18–29.

[70] B.R. Crane, L.M. Siegel, E.D. Getzoff, Sulfite reductase structure at 1.6 Ångstrom — evolution and catalysis for reduction of inorganic anions, Science 270 (1995) 59–67.

[71] J.M. Akagi, Respiratory sulfate reduction, in: L.L. Barton (Ed.), Sulfate-reducing Bacteria, Plenum Press, New York, 1995, pp. 89–111.

[72] A.S. Pott, C. Dahl, Sirohaem sulfite reductase and other proteins encoded by genes at the *dsr* locus of *Chromatium vinosum* are involved in the oxidation of intracellular sulfur, Microbiology 144 (1998) 1881–1894.

[73] C. Holkenbrink, S.O. Barbas, A. Mellerup, H. Otaki, N.U. Frigaard, Sulfur globule oxidation in green sulfur bacteria is dependent on the dissimilatory sulfite reductase system, Microbiology 157 (2011) 1229–1239.

[74] M. Mussmann, M. Richter, T. Lombardot, A. Meyerdierks, J. Kuever, M. Kube, F.O. Glockner, R. Amann, Clustered genes related to sulfate respiration in uncultured prokaryotes support the theory of their concomitant horizontal transfer, J. Bacteriol. 187 (2005) 7126–7137.

[75] A. Magalon, J.G. Fedor, A. Walburger, J.H. Weiner, Molybdenum enzymes in bacteria and their maturation, Coord. Chem. Rev. 255 (2011) 1159–1178.

[76] R.A. Rothery, G.J. Workun, J.H. Weiner, The prokaryotic complex iron-sulfur molybdoenzyme family, Biochim. Biophys. Acta 1778 (2008) 1897–1929.

[77] D.J. Richardson, Bacterial respiration: a flexible process for a changing environment, Microbiology 146 (2000) 551–571.

[78] B. Schoepf-Cothenet, R. van Lis, P. Philippot, A. Magalon, M.J. Russell, W. Nitschke, The ineluctable requirement for the trans-iron elements molybdenum and/or tungsten in the origin of life, Sci. Rep. 2 (2012).

[79] B.C. Berks, M.D. Page, D.J. Richardson, A. Reilly, A. Cavill, F. Outen, S.J. Ferguson, Sequence analysis of subunits of the membrane-bound nitrate reductase from a denitrifying bacterium: the integral membrane subunit provides a prototype for the dihaem electron-carrying arm of a redox loop, Mol. Microbiol. 15 (1995) 319–331.

[80] M.G. Bertero, R.A. Rothery, M. Palak, C. Hou, D. Lim, F. Blasco, J.H. Weiner, N.C. Strynadka, Insights into the respiratory electron transfer pathway from the structure of nitrate reductase A, Nat. Struct. Biol. 10 (2003) 681–687.

[81] M. Jormakka, S. Tornroth, B. Byrne, S. Iwata, Molecular basis of proton motive force generation: structure of formate dehydrogenase-N, Science 295 (2002) 1863–1868.

[82] M. Jormakka, K. Yokoyama, T. Yano, M. Tamakoshi, S. Akimoto, T. Shimamura, P. Curmi, S. Iwata, Molecular mechanism of energy conservation in polysulfide respiration, Nat. Struct. Mol. Biol. 15 (2008) 730–737.

[83] J. Simon, M. Kern, Quinone-reactive proteins devoid of haem b form widespread membrane-bound electron transport modules in bacterial respiration, Biochem. Soc. Trans. 36 (2008) 1011–1016.

[84] F. Grein, I.A.C. Pereira, C. Dahl, Biochemical characterization of individual components of the *Allochromatium vinosum* DsrMKJOP transmembrane complex aids understanding of complex function in vivo, J. Bacteriol. 192 (2010) 6369–6377.

[85] G. Cecchini, I. Schroder, R.P. Gunsalus, E. Maklashina, Succinate dehydrogenase and fumarate reductase from *Escherichia coli*, Biochim. Biophys. Acta 1553 (2002) 140–157.

[86] P.M. Vignais, B. Billoud, Occurrence, classification, and biological function of hydrogenases: an overview, Chem. Rev. 107 (2007) 4206–4272.

[87] J. Simon, R.J. van Spanning, D.J. Richardson, The organisation of proton motive and non-proton motive redox loops in prokaryotic respiratory systems, Biochim. Biophys. Acta 1777 (2008) 1480–1490.

[88] M. Jormakka, B. Byrne, S. Iwata, Proton motive force generation by a redox loop mechanism, FEBS Lett. 545 (2003) 25–30.

[89] R. Hedderich, N. Hamann, M. Bennati, Heterodisulfide reductase from methanogenic archaea: a new catalytic role for an iron-sulfur cluster, Biol. Chem. 386 (2005) 961–970.

[90] R. Hedderich, O. Klimmek, A. Kroger, R. Dirmeier, M. Keller, K.O. Stetter, Anaerobic respiration with elemental sulfur and with disulfides, FEMS Microbiol. Rev. 22 (1999) 353–381.

[91] R.K. Thauer, A.K. Kaster, H. Seedorf, W. Buckel, R. Hedderich, Methanogenic archaea: ecologically relevant differences in energy conservation, Nat. Rev. Microbiol. 6 (2008) 579–591.

[92] W. Badziong, R.K. Thauer, J.G. Zeikus, Isolation and characterization of *Desulfovibrio* growing on hydrogen plus sulfate as the sole energy source, Arch. Microbiol. 116 (1978) 41–49.

[93] R.K. Thauer, E. Stackebrandt, W.A. Hamilton, Energy metabolism and phylogenetic diversity of sulfate-reducing bacteria, in: L.L. Barton, W.A. Hamilton (Eds.), Sulphate Reducing Bacteria — Environmental and Engineered Systems, Cambridge University Press, Cambridge, 2007, pp. 1–37.

[94] M.D. Collins, F. Widdel, Respiratory quinones of sulfate-reducing and sulfur-reducing bacteria — a systematic investigation, Syst. Appl. Microbiol. 8 (1986) 8–18.

[95] K.L. Keller, J.D. Wall, Genetics and molecular biology of the electron flow for sulfate respiration in *Desulfovibrio*, Front. Microbiol. 2 (2011) 135.

[96] I.A.C. Pereira, Membrane complexes in *Desulfovibrio*, in: C. Friedrich, C. Dahl (Eds.), Microbial Sulfur Metabolism, Springer-Verlag, 2008, pp. 24–35.

[97] J.Z. Zhou, Q. He, C.L. Hemme, A. Mukhopadhyay, K. Hillesland, A.F. Zhou, Z.L. He, J.D. Van Nostrand, T.C. Hazen, D.A. Stahl, J.D. Wall, A.P. Arkin, How sulfate-reducing microorganisms cope with stress: lessons from systems biology, Nat. Rev. Microbiol. 9 (2011) 452–466.

[98] A.S. Bradley, W.D. Leavitt, D.T. Johnston, Revisiting the dissimilatory sulfate reduction pathway, Geobiology 9 (2011) 446–457.

[99] A.K. Kaster, J. Moll, K. Parey, R.K. Thauer, Coupling of ferredoxin and heterodisulfide reduction via electron bifurcation in hydrogenotrophic methanogenic archaea, Proc. Natl. Acad. Sci. 108 (2011) 2981–2986.

[100] A. Stojanovic, G.J. Mander, E.C. Duin, R. Hedderich, Physiological role of the F420-non-reducing hydrogenase (*Mvh*) from *Methanothermobacter marburgensis*, Arch. Microbiol. 180 (2003) 194–203.

[101] G.M. Zane, H.C. Yen, J.D. Wall, Effect of the deletion of *qmoABC* and the promoter-distal gene encoding a hypothetical protein on sulfate reduction in *Desulfovibrio vulgaris* Hildenborough, Appl. Environ. Microbiol. 76 (2010) 5500–5509.

[102] P. Junier, T. Junier, S. Podell, D.R. Sims, J.C. Detter, A. Lykidis, C.S. Han, N.S. Wigginton, T. Gaasterland, R. Bernier-Latmani, The genome of the Gram-positive metal- and sulfate-reducing bacterium *Desulfotomaculum reducens* strain MI-1, Environ. Microbiol. 12 (2010) 2738–2754.

[103] L.K. Chan, T.S. Weber, R.M. Morgan-Kiss, T.E. Hanson, A genomic region required for phototrophic thiosulfate oxidation in the green sulfur bacterium *Chlorobium tepidum* (syn. *Chlorobaculum tepidum*), Microbiology 154 (2008) 818–829.

[104] J. Rodriguez, J. Hiras, T.E. Hanson, Sulfite oxidation in *Chlorobaculum tepidum*, Front. Microbiol. 2 (2011) 112.

[105] A.R. Ramos, K.L. Keller, J.D. Wall, I.A.C. Pereira, The membrane QmoABC complex interacts directly with the dissimilatory adenosine 5′-phosphosulfate reductase in sulfate reducing bacteria, Front. Microbiol. 3 (2012) 137.

[106] G. Herrmann, E. Jayamani, G. Mai, W. Buckel, Energy conservation via electron-transferring flavoprotein in anaerobic bacteria, J. Bacteriol. 190 (2008) 784–791.

[107] W. Nitschke, M.J. Russell, Redox bifurcations: mechanisms and importance to life now, and at its origin: a widespread means of energy conversion in biology unfolds, Bioessays 34 (2011) 106–109.

[108] F. Li, J. Hinderberger, H. Seedorf, J. Zhang, W. Buckel, R.K. Thauer, Coupled ferredoxin and crotonyl coenzyme A (CoA) reduction with NADH catalyzed by the butyryl-CoA dehydrogenase/Etf complex from *Clostridium kluyveri*, J. Bacteriol. 190 (2008) 843–850.

[109] W.F. Martin, Hydrogen, metals, bifurcating electrons, and proton gradients: the early evolution of biological energy conservation, FEBS Lett. (2012) 485–493.

[110] W. Nitschke, M.J. Russell, Hydrothermal focusing of chemical and chemiosmotic energy, supported by delivery of catalytic Fe, Ni, Mo/W, Co, S and Se, forced life to emerge, J. Mol. Evol. 69 (2009) 481–496.

[111] G.J. Schut, M.W. Adams, The iron-hydrogenase of *Thermotoga maritima* utilizes ferredoxin and NADH synergistically: a new perspective on anaerobic hydrogen production, J. Bacteriol. 191 (2009) 4451–4457.

[112] S. Wang, H. Huang, J. Moll, R.K. Thauer, NADP⁺ reduction with reduced ferredoxin and NADP⁺ reduction with NADH are coupled via an electron-bifurcating enzyme complex in *Clostridium kluyveri*, J. Bacteriol. 192 (2010) 5115–5123.

[113] G.J. Mander, E.C. Duin, D. Linder, K.O. Stetter, R. Hedderich, Purification and characterization of a membrane-bound enzyme complex from the sulfate-reducing archaeon *Archaeoglobus fulgidus* related to heterodisulfide reductase from methanogenic archaea, Eur. J. Biochem. 269 (2002) 1895–1904.

[114] J. Sander, S. Engels-Schwarzlose, C. Dahl, Importance of the DsrMKJOP complex for sulfur oxidation in *Allochromatium vinosum* and phylogenetic analysis of related complexes in other prokaryotes, Arch. Microbiol. 186 (2006) 357–366.

[115] S. Heiden, R. Hedderich, E. Setzke, R.K. Thauer, Purification of a two-subunit cytochrome *b*-containing heterodisulfide reductase from methanol-grown *Methanosarcina barkeri*, Eur. J. Biochem. 221 (1994) 855–861.

[116] A. Kunkel, M. Vaupel, S. Heim, R.K. Thauer, R. Hedderich, Heterodisulfide reductase from methanol-grown cells of *Methanosarcina barkeri* is not a flavoenzyme, Eur. J. Biochem. 244 (1997) 226–234.

[117] G.J. Mander, A.J. Pierik, H. Huber, R. Hedderich, Two distinct heterodisulfide reductase-like enzymes in the sulfate-reducing archaeon *Archaeoglobus profundus*, Eur. J. Biochem. 271 (2004) 1106–1116.

[118] F. Grein, S.S. Venceslau, L. Schneider, P. Hildebrandt, S. Todorovic, I.A.C. Pereira, C. Dahl, DsrJ, an essential part of the DsrMKJOP transmembrane complex in the purple sulfur bacterium *Allochromatium vinosum*, is an unusual triheme cytochrome *c*, Biochemistry 49 (2010) 8290–8299.

[119] V.A. Bamford, S. Bruno, T. Rasmussen, C. Appia-Ayme, M.R. Cheesman, B.C. Berks, A.M. Hemmings, Structural basis for the oxidation of thiosulfate by a sulfur cycle enzyme, EMBO J. 21 (2002) 5599–5610.

[120] M.R. Cheesman, P.J. Little, B.C. Berks, Novel heme ligation in a c-type cytochrome involved in thiosulfate oxidation: EPR and MCD of SoxAX from *Rhodovulum sulfidophilum*, Biochemistry 40 (2001) 10562–10569.

[121] J.R. Kilmartin, M.J. Maher, K. Krusong, C.J. Noble, G.R. Hanson, P.V. Bernhardt, M.J. Riley, U. Kappler, Insights into structure and function of the active site of SoxAX cytochromes, J. Biol. Inorg. Chem. 286 (2011) 24872–24881.

[122] S.M. da Silva, I. Pacheco, I.A.C. Pereira, Electron transfer between periplasmic formate dehydrogenase and cytochromes *c* in *Desulfovibrio desulfuricans* ATCC 27774, J. Biol. Inorg. Chem. 17 (2012) 831–838.

[123] J.R. Cort, U. Selan, A. Schulte, F. Grimm, M.A. Kennedy, C. Dahl, *Allochromatium vinosum* DsrC: solution-state NMR structure, redox properties, and interaction with DsrEFH, a protein essential for purple sulfur bacterial sulfur oxidation, J. Mol. Biol. 382 (2008) 692–707.

[124] S.A. Haveman, V. Brunelle, J.K. Voordouw, G. Voordouw, J.F. Heidelberg, R. Rabus, Gene expression analysis of energy metabolism mutants of *Desulfovibrio vulgaris* Hildenborough indicates an important role for alcohol dehydrogenase, J. Bacteriol. 185 (2003) 4345–4353.

[125] J.D. Wall, A.P. Arkin, N.C. Balci, B. Rapp-Giles, Genetics and genomics of sulfate respiration in *Desulfovibrio*, in: C. Dahl, C.G. Friedrich (Eds.), Microbial Sulfur Metabolism, Springer-Verlag, Heidelberg, 2008, pp. 1–12.

[126] D.E. Canfield, F.J. Stewart, B. Thamdrup, L. De Brabandere, T. Dalsgaard, E.F. Delong, N.P. Revsbech, O. Ulloa, A cryptic sulfur cycle in oxygen-minimum-zone waters off the Chilean coast, Science 330 (2010) 1375–1378.

[127] F.J. Stewart, A.K. Sharma, J.A. Bryant, J.M. Eppley, E.F. DeLong, Community transcriptomics reveals universal patterns of protein sequence conservation in natural microbial communities, Genome Biol. 12 (2011).

[128] Y. Ikeuchi, N. Shigi, J. Kato, A. Nishimura, T. Suzuki, Mechanistic insights into multiple sulfur mediators sulfur relay by involved in thiouridine biosynthesis at tRNA wobble positions, Mol. Cell 21 (2006) 97–108.

[129] Y. Stockdreher, S.S. Venceslau, M. Josten, H.G. Sahl, I.A.C. Pereira, C. Dahl, Cytoplasmic sulfurtransferases in the purple sulfur bacterium *Allochromatium vinosum*: evidence for sulfur transfer from DsrEFH to DsrC, PLoS One 7 (2012) e40785.

[130] Y.C. Hsieh, M.Y. Liu, V.C.C. Wang, Y.L. Chiang, E.H. Liu, W.G. Wu, S.I. Chan, C.J. Chen, Structural insights into the enzyme catalysis from comparison of three forms of dissimilatory sulphite reductase from *Desulfovibrio gigas*, Mol. Microbiol. 78 (2010) 1101–1116.

[131] T.F. Oliveira, E. Franklin, J.P. Afonso, A.R. Khan, N.J. Oldham, I.A.C. Pereira, M. Archer, Structural insights into dissimilatory sulfite reductases: structure of desulfuroidin from *Desulfomicrobium norvegicum*, Front. Microbiol. 2 (2011) 71.

[132] U. Deppenmeier, The membrane-bound electron transport system of *Methanosarcina* species, J. Bioenerg. Biomembr. 36 (2004) 55–64.

[133] P.M. Matias, I.A.C. Pereira, C.M. Soares, M.A. Carrondo, Sulphate respiration from hydrogen in *Desulfovibrio* bacteria: a structural biology overview, Prog. Biophys. Mol. Biol. 89 (2005) 292–329.

[134] C.V. Romão, M. Archer, S.A. Lobo, R.O. Louro, I.A.C. Pereira, L.M. Saraiva, M. Teixeira, M.P.M., Diversity of heme proteins in sulfate reducing bacteria, in: K.M. Kadish, K.M. Smith, R. Guilard (Eds.), Handbook of Porphyrin Science, vol. 19 (89), World Scientific Publishing Co., Singapore, 2012, pp. 139–230.

[135] J.R. Postgate, Presence of cytochrome in an obligate anaerobe, Biochem. J. 56 (1954) xi.

[136] M. Ishimoto, J. Koyama, Y. Nagai, Role of a cytochrome in thiosulfate reduction by a sulfate reducing bacterium, Seikagaku Zasshi 26 (1954) 303.

[137] J.M. Odom, H.D. Peck Jr., Hydrogen cycling as a general mechanism for energy coupling in the sulfate-reducing bacteria, *Desulfovibrio* sp. FEMS Microbiol. Lett. 12 (1981) 47–50.

[138] R. Rabus, A. Ruepp, T. Frickey, T. Rattei, B. Fartmann, M. Stark, M. Bauer, A. Zibat, T. Lombardot, I. Becker, J. Amann, K. Gellner, H. Teeling, W.D. Leuschner, F.O. Glockner, A.N. Lupas, R. Amann, H.P. Klenk, The genome of *Desulfotalea psychrophila*, a sulfate-reducing bacterium from permanently cold Arctic sediments, Environ. Microbiol. 6 (2004) 887–902.

[139] M.E. Pandelia, W. Lubitz, W. Nitschke, Evolution and diversification of Group 1 [NiFe] hydrogenases. Is there a phylogenetic marker for O₂-tolerance? Biochim. Biophys. Acta 1817 (2012) 1565–1575.

[140] J.F. Heidelberg, R. Seshadri, S.A. Haveman, C.L. Hemme, I.T. Paulsen, J.F. Kolonay, J.A. Eisen, N. Ward, B. Methe, L.M. Brinkac, S.C. Daugherty, R.T. Deboy, R.J. Dodson, A.S. Durkin, R. Madupu, W.C. Nelson, S.A. Sullivan, D. Fouts, D.H. Haft, J. Selengut, J.D. Peterson, T.M. Davidsen, N. Zafar, L.W. Zhou, D. Radune, G. Dimitrov, M. Hance, K. Tran, H. Khouri, J. Gill, T.R. Utterback, T.V. Feldblyum, J.D. Wall, G. Voordouw, C.M. Fraser, The genome sequence of the anaerobic, sulfate-reducing bacterium *Desulfovibrio vulgaris* Hildenborough, Nat. Biotechnol. 22 (2004) 554–559.

[141] I.A.C. Pereira, S.A. Haveman, G. Voordouw, Biochemical, genetic and genomic characterization of anaerobic electron transport pathways in sulfate-reducing delta-proteobacteria, in: L.L. Barton, W.A. Allan Hamilton (Eds.), Sulphate-reducing Bacteria: Environmental and Engineered Systems, Cambridge University Press, 2007, pp. 215–240.

[142] S.A. Caffrey, H.S. Park, J.K. Voordouw, Z. He, J. Zhou, G. Voordouw, Function of periplasmic hydrogenases in the sulfate-reducing bacterium *Desulfovibrio vulgaris* Hildenborough, J. Bacteriol. 189 (2007) 6159–6167.

[143] F.M.A. Valente, C.C. Almeida, I. Pacheco, J. Carita, L.M. Saraiva, I.A.C. Pereira, Selenium is involved in regulation of periplasmic hydrogenase gene expression in *Desulfovibrio vulgaris* Hildenborough, J. Bacteriol. 188 (2006) 3228–3235.

[144] S.M. da Silva, C. Pimentel, F.M.A. Valente, C. Rodrigues-Pousada, I.A.C. Pereira, Tungsten and molybdenum regulation of formate dehydrogenase expression in *Desulfovibrio vulgaris* Hildenborough, J. Bacteriol. 193 (2011) 2909–2916.

[145] R.O. Louro, Proton thrusters: overview of the structural and functional features of soluble tetraheme cytochromes $c_3$, J. Biol. Inorg. Chem. 12 (2007) 1–10.

[146] A.V. Xavier, Thermodynamic and choreographic constraints for energy transduction by cytochrome $c$ oxidase, Biochim. Biophys. Acta 1658 (2004) 23–30.

[147] I.A.C. Pereira, M. Teixeira, A.V. Xavier, Hemeproteins in anaerobes, Struct. Bond. 91 (1998) 65–89.

[148] P.M. Matias, A.V. Coelho, F.M.A. Valente, P.D., J. LeGall, A.V. Xavier, I.A.C. Pereira, M.A. Carrondo, Sulfate respiration in *Desulfovibrio vulgaris* Hildenborough: structure of the 16-heme cytochrome $c$ HmcA at 2.5 Å resolution and a view

of its role in transmembrane electron transfer, J. Biol. Chem. 277 (2002) 47907–47916.

[149] P.M. Matias, R. Coelho, I.A.C. Pereira, A.V. Coelho, A.W. Thompson, L.C. Sieker, J.L. Gall, M.A. Carrondo, The primary and three-dimensional structures of a nine-haem cytochrome $c$ from *Desulfovibrio desulfuricans* ATCC 27774 reveal a new member of the Hmc family, Structure 7 (1999) 119–130.

[150] L. Pieulle, X. Morelli, P. Gallice, E. Lojou, P. Barbier, M. Czjzek, P. Bianco, F. Guerlesquin, E.C. Hatchikian, The type I/type II cytochrome $c_3$ complex: an electron transfer link in the hydrogen-sulfate reduction pathway, J. Mol. Biol. 354 (2005) 73–90.

[151] F.M.A. Valente, L.M. Saraiva, J. LeGall, A.V. Xavier, M. Teixeira, I.A.C. Pereira, A membrane-bound cytochrome $c_3$: a type II cytochrome $c_3$ from *Desulfovibrio vulgaris* Hildenborough, Chembiochem 2 (2001) 895–905.

[152] S. Norager, P. Legrand, L. Pieulle, C. Hatchikian, M. Roth, Crystal structure of the oxidised and reduced acidic cytochrome $c_3$ from *Desulfovibrio africanus*, J. Mol. Biol. 290 (1999) 881–902.

[153] I.J. Correia, C.M. Paquete, R.O. Louro, T. Catarino, D.L. Turner, A.V. Xavier, Thermodynamic and kinetic characterization of trihaem cytochrome $c_3$ from *Desulfuromonas acetoxidans*, Eur. J. Biochem. 269 (2002) 5722–5730.

[154] L. Morgado, M. Bruix, M. Pessanha, Y.Y. Londer, C.A. Salgueiro, Thermodynamic characterization of a triheme cytochrome family from *Geobacter sulfurreducens* reveals mechanistic and functional diversity, Biophys. J. 99 (2010) 293–301.

[155] S.S. Venceslau, R.R. Lino, I.A.C. Pereira, The Qrc membrane complex, related to the alternative complex III, is a menaquinone reductase involved in sulfate respiration, J. Biol. Chem. 285 (2010) 22774–22783.

[156] G. Voordouw, Carbon monoxide cycling by *Desulfovibrio vulgaris* Hildenborough, J. Bacteriol. 184 (2002) 5903–5911.

[157] S.H. Thomas, R.D. Wagner, A.K. Arakaki, J. Skolnick, J.R. Kirby, L.J. Shimkets, R.A. Sanford, F.E. Löffler, The mosaic genome of *Anaeromyxobacter dehalogenans* strain 2CP-C suggests an aerobic common ancestor to the delta-proteobacteria, PLoS One 3 (2008) e2103.

[158] A. Esteve-Núñez, J. Sosnik, P. Visconti, D.R. Lovley, Fluorescent properties of c-type cytochromes reveal their potential role as an extracytoplasmic electron sink in *Geobacter sulfurreducens*, Environ. Microbiol. 10 (2008) 497–505.

[159] M.P. Bryant, L.L. Campbell, C.A. Reddy, M.R. Crabill, Growth of *Desulfovibrio* in lactate or ethanol media low in sulfate in association with H₂-utilizing methanogenic bacteria, Appl. Environ. Microbiol. 33 (1977) 1162–1169.

[160] K.L. Hillesland, D.A. Stahl, Rapid evolution of stability and productivity at the origin of a microbial mutualism, Proc. Natl. Acad. Sci. 107 (2010) 2124–2129.

[161] C.M. Plugge, W. Zhang, J.C. Scholten, A.J. Stams, Metabolic flexibility of sulfate-reducing bacteria, Front. Microbiol. 2 (2011) 81.

[162] J. Leloup, H. Fossing, K. Kohls, L. Holmkvist, C. Borowski, B.B. Jørgensen, Sulfate-reducing bacteria in marine sediment (Aarhus Bay, Denmark): abundance and diversity related to geochemical zonation, Environ. Microbiol. 11 (2009) 1278–1291.

[163] X. Li, Q. Luo, N.Q. Wofford, K.L. Keller, M.J. McInerney, J.D. Wall, L.R. Krumholz, A molybdopterin oxidoreductase is involved in H₂ oxidation in *Desulfovibrio desulfuricans* G20, J. Bacteriol. 191 (2009) 2675–2682.

[164] X. Gao, Y. Xin, R.E. Blankenship, Enzymatic activity of the alternative complex III as a menaquinol:auracyanin oxidoreductase in the electron transfer chain of *Chloroflexus aurantiacus*, FEBS Lett. 583 (2009) 3275–3279.

[165] M.M. Pereira, P.N. Refojo, G.O. Hreggvidsson, S. Hjorleifsdottir, M. Teixeira, The alternative complex III from *Rhodothermus marinus* – a prototype of a new family of quinol:electron acceptor oxidoreductases, FEBS Lett. 581 (2007) 4831–4835.

[166] M.F. Yanyushin, M.C. del Rosario, D.C. Brune, R.E. Blankenship, New class of bacterial membrane oxidoreductases, Biochemistry 44 (2005) 10037–10045.

[167] L.A. Sazanov, P. Hinchliffe, Structure of the hydrophilic domain of respiratory complex I from *Thermus thermophilus*, Science 311 (2006) 1430–1436.

[168] S.S. Venceslau, D. Matos, I.A.C. Pereira, EPR characterization of the new Qrc complex from sulfate reducing bacteria and its ability to form a supercomplex with hydrogenase and TplC₃, FEBS Lett. 585 (2011) 2177–2181.

[169] X.Z. Li, M.J. McInerney, D.A. Stahl, L.R. Krumholz, Metabolism of H₂ by *Desulfovibrio alaskensis* G20 during syntrophic growth on lactate, Microbiology 157 (2011) 2912–2921.

[170] C.B. Walker, Z.L. He, Z.K. Yang, J.A. Ringbauer, Q. He, J.H. Zhou, G. Voordouw, J.D. Wall, A.P. Arkin, T.C. Hazen, S. Stolyar, D.A. Stahl, The electron transfer system of syntrophically grown *Desulfovibrio vulgaris*, J. Bacteriol. 191 (2009) 5793–5801.

[171] M. Rossi, W.B.R. Pollock, M.W. Reij, R.G. Keon, R. Fu, G. Voordouw, The *hmc* operon of *Desulfovibrio vulgaris* subsp. *vulgaris* Hildenborough encodes a potential transmembrane redox protein complex, J. Bacteriol. 175 (1993) 4699–4711.

[172] I.A.C. Pereira, C.V. Romão, A.V. Xavier, J. LeGall, M. Teixeira, Electron transfer between hydrogenases and mono and multihaeme cytochromes in *Desulfovibrio* spp. J. Biol. Inorg. Chem. 3 (1998) 494–498.

[173] A. Dolla, B.K.J. Pohorelic, J.K. Voordouw, G. Voordouw, Deletion of the *hmc* operon of *Desulfovibrio vulgaris* subsp. *vulgaris* Hildenborough hampers hydrogen metabolism and low-redox-potential niche establishment, Arch. Microbiol. 174 (2000) 143–151.

[174] R.G. Keon, R. Fu, G. Voordouw, Deletion of two downstream genes alters expression of the *hmc* operon of *Desulfovibrio vulgaris* subsp. *vulgaris* Hildenborough, Arch. Microbiol. 167 (1997) 376–383.

[175] P.M. Pereira, Q. He, F.M.A. Valente, A.V. Xavier, J.Z. Zhou, I.A.C. Pereira, R.O. Louro, Energy metabolism in *Desulfovibrio vulgaris* Hildenborough: insights from transcriptome analysis, Antonie Van Leeuwenhoek 93 (2008) 347–362.

[176] R. Hedderich, L. Forzi, Energy-converting [NiFe] hydrogenases: more than just H₂ activation, J. Mol. Microbiol. Biotechnol. 10 (2005) 92–104.

[177] P.M. Pereira, M. Teixeira, A.V. Xavier, R.O. Louro, I.A.C. Pereira, The Tmc complex from *Desulfovibrio vulgaris* Hildenborough is involved in transmembrane electron transfer from periplasmic hydrogen oxidation, Biochemistry 45 (2006) 10359–10367.

[178] F.M.A. Valente, A.S.F. Oliveira, N. Gnadt, I. Pacheco, A.V. Coelho, A.V. Xavier, M. Teixeira, C.M. Soares, I.A.C. Pereira, Hydrogenases in *Desulfovibrio vulgaris* Hildenborough: structural and physiologic characterisation of the membrane-bound [NiFeSe] hydrogenase, J. Biol. Inorg. Chem. 10 (2005) 667–682.

[179] A.V. Callaghan, B.E.L. Morris, I.A.C. Pereira, M.J. McInerney, R.N. Austin, J.T. Groves, J.J. Kukor, J.M. Suflita, L.Y. Young, G.J. Zylstra, B. Wawrik, The genome sequence of *Desulfatibacillum alkenivorans* AK-01: a blueprint for anaerobic alkane oxidation, Environ. Microbiol. 14 (2012) 101–113.

[180] A.W. Strittmatter, H. Liesegang, R. Rabus, I. Decker, J. Amann, S. Andres, A. Henne, W.F. Fricke, R. Martinez-Arias, D. Bartels, A. Goesmann, L. Krause, A. Puhler, H.P. Klenk, M. Richter, M. Schuler, F.O. Glockner, A. Meyerdierks, G. Gottschalk, R. Amann, Genome sequence of *Desulfobacterium autotrophicum* HRM2, a marine sulfate reducer oxidizing organic carbon completely to carbon dioxide, Environ. Microbiol. 11 (2009) 1038–1055.

[181] H.P. Klenk, R.A. Clayton, J.F. Tomb, O. White, K.E. Nelson, K.A. Ketchum, R.J. Dodson, M. Gwinn, E.K. Hickey, J.D. Peterson, D.L. Richardson, A.R. Kerlavage, D.E. Graham, N.C. Kyrpides, R.D. Fleischmann, J. Quackenbush, N.H. Lee, G.G. Sutton, S. Gill, E.F. Kirkness, B.A. Dougherty, K. McKenney, M.D. Adams, B. Loftus, J.C. Venter, et al., The complete genome sequence of the hyperthermophilic, sulphate-reducing archaeon *Archaeoglobus fulgidus*, Nature 390 (1997) 364–370.

[182] S. Spring, A. Lapidus, M. Schroder, D. Gleim, D. Sims, L. Meincke, T.G. Del Rio, H. Tice, A. Copeland, J.F. Cheng, S. Lucas, F. Chen, M. Nolan, D. Bruce, L. Goodwin, S. Pitluck, N. Ivanova, K. Mavromatis, N. Mikhailova, A. Pati, A. Chen, K. Palaniappan, M. Land, L. Hauser, Y.J. Chang, C.D. Jeffries, P. Chain, E. Saunders, T. Brettin, J.C. Detter, M. Goker, J. Bristow, J.A. Eisen, V. Markowitz, P. Hugenholtz, N.C. Kyrpides, H.P. Klenk, C. Han, Complete genome sequence of *Desulfotomaculum acetoxidans* type strain (5575(T)), Stand. Genomic Sci. 1 (2009) 242–252.

[183] E. Pierce, G. Xie, R.D. Barabote, E. Saunders, C.S. Han, J.C. Detter, P. Richardson, T.S. Brettin, A. Das, L.G. Ljungdahl, S.W. Ragsdale, The complete genome sequence of *Moorella thermoacetica* (*f*. *Clostridium thermoaceticum*), Environ. Microbiol. 10 (2008) 2550–2573.

[184] T. Ide, S. Baumer, U. Deppenmeier, Energy conservation by the H2: heterodisulfide oxidoreductase from *Methanosarcina mazei* Go1: identification of two proton-translocating segments, J. Bacteriol. 181 (1999) 4076–4080.

[185] J.W. Kung, C. Loffler, K. Dorner, D. Heintz, S. Gallien, A. Van Dorsselaer, T. Friedrich, M. Boll, Identification and characterization of the tungsten-containing class of benzoyl-coenzyme A reductases, Proc. Natl. Acad. Sci. 106 (2009) 17687–17692.

[186] C. Loffler, K. Kuntze, J.R. Vazquez, A. Rugor, J.W. Kung, A. Bottcher, M. Boll, Occurrence, genes and expression of the W/Se-containing class II benzoyl-coenzyme A reductases in anaerobic bacteria, Environ. Microbiol. 13 (2011) 696–709.

[187] S. Wischgoll, D. Heintz, F. Peters, A. Erxleben, E. Sarnighausen, R. Reski, A. Van Dorsselaer, M. Boll, Gene clusters involved in anaerobic benzoate degradation of *Geobacter metallireducens*, Mol. Microbiol. 58 (2005) 1238–1252.

[188] N. Hamann, G.J. Mander, J.E. Shokes, R.A. Scott, M. Bennati, R. Hedderich, Cysteine-rich CCG domain contains a novel [4Fe−4S] cluster binding motif as deduced from studies with subunit B of heterodisulfide reductase from *Methanothermobacter marburgensis*, Biochemistry 46 (2007) 12875–12885.

[189] S.T. Cole, K. Eiglmeier, S. Ahmed, N. Honore, L. Elmes, W.F. Anderson, J.H. Weiner, Nucleotide-sequence and gene-polypeptide relationships of the *glpABC* operon encoding the anaerobic Sn-glycerol-3-phosphate dehydrogenase of *Escherichia coli* K12, J. Bacteriol. 170 (1988) 2448–2456.

[190] N. Hamann, E. Bill, J.E. Shokes, R.A. Scott, M. Bennati, R. Hedderich, The CCG-domain-containing subunit SdhE of succinate:quinone oxidoreductase from *Sulfolobus solfataricus* P2 binds a [4Fe−4S] cluster, J. Biol. Inorg. Chem. 14 (2009) 457–470.

[191] R.S. Lemos, A.S. Fernandes, M.M. Pereira, C.M. Gomes, M. Teixeira, Quinol:fumarate oxidoreductases and succinate:quinone oxidoreductases: phylogenetic relationships, metal centres and membrane attachment, Biochim. Biophys. Acta 1553 (2002) 158–170.

[192] S. Heim, A. Künkel, R.K. Thauer, R. Hedderich, Thiol:fumarate reductase (Tfr) from *Methanobacterium thermoautotrophicum* — identification of the catalytic sites for fumarate reduction and thiol oxidation, Eur. J. Biochem. 253 (1998) 292–299.

[193] Y.R. Chai, R. Kolter, R. Losick, A widely conserved gene cluster required for lactate utilization in *Bacillus subtilis* and its involvement in biofilm formation, J. Bacteriol. 191 (2009) 2423–2430.

[194] G.E. Pinchuk, D.A. Rodionov, C. Yang, X.Q. Li, A.L. Osterman, E. Dervyn, O.V. Geydebrekht, S.B. Reed, M.F. Romine, F.R. Collart, J.H. Scott, J.K. Fredrickson, A.S. Beliaev, Genomic reconstruction of *Shewanella oneidensis* MR-1 metabolism reveals a previously uncharacterized machinery for lactate utilization, Proc. Natl. Acad. Sci. 106 (2009) 2874–2879.

[195] M.T. Thomas, M. Shepherd, R.K. Poole, A.H. van Vliet, D.J. Kelly, B.M. Pearson, Two respiratory enzyme systems in *Campylobacter jejuni* NCTC 11168 contribute to growth on L-lactate, Environ. Microbiol. 13 (2011) 48–61.

[196] H. Ogata, P. Kellers, W. Lubitz, The crystal structure of the [NiFe] hydrogenase from the photosynthetic bacterium *Allochromatium vinosum*: characterization of the oxidized enzyme (Ni-A state), J. Mol. Biol. 402 (2010) 428–444.

[197] L.S. Palagyí-Meszaros, J. Maroti, D. Latinovics, T. Balogh, E. Klement, K.F. Medzihradszky, G. Rakhely, K.L. Kovacs, Electron-transfer subunits of the NiFe hydrogenases in *Thiocapsa roseopersicina* BBS, FEBS J. 276 (2009) 164–174.
