
www.impactaging.com                                                                 AGING, August 2012, Vol. 4, No 8

**Caloric restriction: is mammalian life extension linked to p53?**

Paola Tucci${}^{1,2}$

1 Medical Research Council, Toxicology Unit, Leicester University, Leicester LE19HN, UK  
2 Department of Pharmaco-Biology, University of Calabria, 87036 Rende (CS), Italy  

Key words: p53, senescence, aging, caloric restriction, apoptosis, cell death, cell cycle  
Abbreviations: CR, caloric restriction; ROS, reactive oxygen species; mTOR, mammalian target of rapamycin; IGF-1, insulin-like growth factor-1; IGF-1R, IGF-1 receptor  
Received: 8/6/12; Accepted: 8/21/12; Published: 8/22/12  
Correspondence to: Paola Tucci, PhD; E-mail: paola.tucci@unical.it  

Copyright: © Tucci. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.

**Abstract:** Caloric restriction, that is limiting food intake, is recognized in mammals as the best characterized and most reproducible strategy for extending lifespan, retarding physiological aging and delaying the onset of age-associated diseases. The aim of this mini review is to argue that p53 is the connection in the abilities of both the Sirt-1 pathway and the TOR pathway to impact on longevity of cells and organisms. This novel, lifespan regulating function of p53 may be evolutionarily more ancient than its relatively recent role in apoptosis and tumour suppression, and is likely to provide many new insights into lifespan modulation.

---

**INTRODUCTION**

Caloric restriction (CR), that is limiting food intake, is recognized in mammals as the best characterized and most reproducible strategy for extending lifespan, retarding physiological aging and delaying the onset of age-associated diseases [1]. Restricted calorie intake modifies the rate of aging and cellular pathology, reduces the age-associated accumulation of oxidatively damaged proteins, lipids and DNA and also prevents many of the changes in gene expression and transcriptional activity that normally occur with ageing [1]. Although several theories have been advanced over the years to explain the anti-ageing effects of CR, one favoured hypothesis is that CR acts by decreasing oxidative stress [2]. Biologically, different animal species are characterised by markedly different lifespan. For example, mice have relatively short (around 2-year) mean longevity, whereas humans live to a mean of 70-80 years. Animals with higher metabolic rates often have shorter life spans. The higher the metabolic rate of an organism, the greater the production of reactive oxygen species (ROS) and hence the shorter the lifespan; however, in some species the strict correlation between metabolic rate and lifespan is not maintained. Birds and primates, for example, tend to live longer than would be predicted by their metabolic rates. This is because, at a given metabolic rate, mitochondria from these species tend to produce fewer ROS. Thus the accumulation of molecular damage and the resulting increase of oxidative stress caused by ROS was thought to contribute to aging rather than metabolic rate itself and provided the strongest correlation with overall longevity [3]. More recently, however, became evident that the ROS model cannot be the only driving-force of aging. No animal lives long enough to experience the consequences of damage by ROS, since other factors terminate its life first [4]. Further, the overexpression of major antioxidant enzymes, which decrease free radicals, does not extend the lifespan of mice [5], and superoxide dismutases, the major reactive oxygen species regulating autophagy [6], protect against oxidative stress but have little or no effect on lifespan in C. Elegans [7].

Thus despite impressive progress in identifying the key components of the CR pathway, many proximal effectors of CR induced longevity remain unknown to date. This can be related to the fact that CR causes a myriad of transcriptional and physiological changes that are related to its positive effects on lifespan [8-12]. Are these benefits due to passive effects of lowered caloric

www.impactaging.com                                                                 525                                                                 AGING, August 2012, Vol. 4 No. 8
intake or is it the result of a highly evolved and regulated process?

The aim of this short review is to argue that CR is indeed a regulated process and that its main regulator pathways, mTOR and Sirt-1, are connected to the p53 pathway.

### The mTOR/Sirt-1 pathway

The CR-driven reduction of metabolic rate involves downregulation of the nutrient sensor mammalian target of rapamycin (mTOR), the evolutionarily conserved serine/threonine protein kinase that is strongly involved in most cellular functions and implicated in stimulating cell growth [13-21]. The IGF-1-AKT-TOR network is an evolutionarily conserved pathway that transmits survival signals in cells in response to growth factor stimulation. mTOR is able to regulate both apoptosis and autophagy, and therefore affects the fate of cells. Indeed, apoptosis is clearly implicated in cancer [22-55] as well as in neurodegeneration [56-63]. The binding of a growth factor (IGF-1) to its tyrosine kinase receptor (IGF-1R) results in the recruitment and activation of PI3 kinase to the plasma membrane receptor, which in turn phosphorylates the inositides, increasing the local concentration of PIP3 and PIP2 at the plasma membrane. This increase in lipid second messengers recruits and activates the PDK and AKT protein kinases at the plasma membrane where AKT is then fully activated by phosphorylation of ser-473 and thr-308 [64-71]. AKT has several substrates that are antiapoptotic such as FOXO, BAD [72-73] and MDM2 [74]. In addition the activated AKT protein moves to the cell nucleus where it phosphorylates the forkhead transcription factors. These events result in a program leading to antiapoptotic signalling, preparation for entry into the cell cycle and cell growth, and communication with the TOR kinase pathway, which senses nutrient levels (glucose and amino acids) in the environment. This is accomplished by AKT-1 phosphorylation and inactivation of TSC2 [75-78], which forms a TSC1-TSC2 protein complex that is a GAP for the RHEB G-protein. RHEB, in turn, activates the TOR kinase [79-81]. Thus, an active AKT-1 activates the TOR kinase, both of which are positive signals for cell growth (an increase in cell mass) and division. Furthermore, CR, which reduces the levels of insulin and IGF-1 in serum, has been shown to extend lifespan and delay the onset of age-associated pathologies through inhibition of TOR [82-85].

The absence of glucose in the cell also increases the levels of AMP, a coactivator of AMPK. Active AMPK positively regulates the activity of the TSC1-TSC2

complex by phosphorylating the TSC2 protein (resulting in the opposite activity to the AKT-1 phosphorylation of TSC2), which then turns off the RHEB G-protein and reduces TOR activity [86].

The TOR kinase regulates two processes that can account for the observed effect of dietary restriction on longevity: translation of selected mRNAs in the cell and autophagy. The first is the rate of protein synthesis, which is modulated by the effect of TOR on the ribosomal protein S6 kinase (S6K) and on the translation initiation factor 4E-binding protein (4E-BP) [87]. It is therefore possible that inhibition of TOR just leads to reduction in the rate of protein synthesis and this is the mechanism of its lifespan prolonging activity. Inhibition of translation may shift cell metabolism to a physiological state that favours maintenance and repair and this may lead to extension of lifespan. Regulation of autophagy is another process by which TOR may affect cell longevity [88]. Accumulating evidence demonstrates that longevity pathways interact with the autophagic process to regulate diverse cellular functions including growth, differentiation, response to nutrient deprivation and oxidative stress, cell death, as well as macromolecule and organelle turnover. This entails the formation of double-membrane vesicles in the cell cytoplasm that engulf cytoplasmic components, including defective mitochondria, and move them to the lysosomal compartment where they are degraded. Autophagy can be induced by stress and also by caloric restriction. The mechanisms by which enhanced autophagy can improve organismal health and longevity are largely elusive. As one possibility, increased autophagy might improve cellular resistance to stress by augmenting the metabolic buffering capacity of cells. Alternatively, autophagy might enhance organellar turnover and mediate a “cleaning effect”, thereby preventing the accumulation of damaged/old organelles [89-105]. But only physiological levels of autophagy can promote survival under stressful conditions. Reduced autophagy may be an oncogenic event and contribute to tumour progression, while enhanced autophagy is activated in tumour cells in which the availability of oxygen and nutrients is poor and represents an adaptive survival mechanism to overcome drug-induced cellular stress and cytotoxicity [106-116]. Thus autophagy is negatively regulated by mTOR, whose activity can be inhibited by rapamycin and caloric restriction [117-124]. Thus, it has been shown that CR slows down aging through inhibition of TOR. On the other hand, it has also been demonstrated that CR activates NAD(+)-dependent deacetylases, called sirtuins, known to be involved in aging and age-related diseases, thus extending lifespan [125-127] (Figure 1). The Sirt-2 gene was shown to regulate lifespan in

www.impactaging.com 526 AGING, August 2012, Vol. 4 No. 8yeast; increased dosage extended life span, and loss of function shortened it [128]. A similar relationship exists between the Sirt-2 gene in C. elegans and life span [129]. Because of the enormous evolutionary divergence between yeast and C. elegans, it is likely that Sirt-2-related genes determine the life span in a broad spectrum of organisms, including mammals. Caloric restriction/nutrient deprivation stimulates autophagy through the activation of the mammalian Sirt-1 gene [130-135]. Indeed, Sirt-1 functions as a metabolic sensor that detects the increase in NAD+ concentrations resulting from enhanced NADH oxidation. Once activated, Sirt-1, which is an NAD-dependent HDAC, can deacetylate essential autophagic modulators and may affect cellular pathways involved in glucose homeostasis.

[142-168]. In response to various stress signals, p53 selectively regulates a set of its target genes and initiates various stress responses, including cell cycle arrest, apoptosis, and/or senescence, to exert its function in DNA damage and tumor suppression. In addition, p53 may play a dual role in autophagy regulation. On the one hand, nuclear p53 can induce autophagy promoting the transcription of proapoptotic and cell cycle-arresting genes. In contrast, cytoplasmic p53 degradation exerts an autophagy-inhibitory function [169]. Loss of p53 thus provides two levels of growth advantage to tumor cells; it removes two mechanisms of eliminating the cell in response to genotoxic stress, and at the same time, enables cell survival under limiting nutrient conditions. Paradoxically, each of these exploits the same process, autophagy, utilizing its opposing functions.

The existence of p53 in short-lived organisms that do not develop adult cancers, such as flies and worms, suggests that tumor suppression is not the only or, indeed, the original function of p53. Indeed, recent studies have shown that p53 and its family members, including p63 and p73, all play important roles in reproduction [170, 171]. Emerging evidence has suggested that p53 is also an important but complex player in the regulation of aging and longevity in worms, flies, mice, and humans. The impact of p53 on aging and longevity in humans has been recently indicated by several epidemiological studies [172, 173]. Moreover, longevity is always coupled with the age of attaining reproductive maturity in animals. The later in life that reproductive maturity occurs, the greater the longevity of that animal. Animals will most often not reproduce in times of stress and starvation of nutrients and will shut down their reproductive processes. Indeed, p53 in adult worms and flies is predominantly localized in the germline where it is employed in the prevention of reproduction in response to stress signals such as DNA damage and starvation. Thus p53 has its origins, in an evolutionary sense, as a germline surveillance molecule under conditions of starvation or DNA damage [171]. It is only in vertebrates, where the body plan requires self-renewal of tissues (flies and worms are largely post-mitotic as adults, except for the germline), where the p53 protein is found in somatic tissues and takes on the function of a tumor suppressor.

p53 interacts with IGF-1, TOR and Sirt-1 pathways, the critical pathways that regulate aging and longevity [174, 175] (Figure 2). There are two major connections between the proteins of these three pathways that form a rapid and a slower response to stress signals after activation of p53. First, the rapid signal transduction

**CALORIC RESTRICTION**

- Reduced levels of Insulin and IGF-1
  - ↓ AKT
    - mTOR
- Absence of Glucose
  - ↓ AMPK
    - mTOR
- Enhanced NAD⁺ concentration
  - ↓ Sirt-1
    - Autophagy

**LONGEVITY**

Figure 1. The regulation of longevity by caloric restriction.

But which pathway is indispensable for prolonged lifespan by CR, inhibiting TOR or activating sirtuins? Evidence has emerged that sirtuins and mTOR are involved in the same longevity pathway [136]. Importantly, resveratrol, an activator of sirtuins, antagonizes the mTOR/S6K pathway [137]. Therefore, the two notions that CR prolongs lifespan either by activating sirtuins or by deactivating TOR are, in fact, complementary: CR deactivates the mTOR pathway in part by activating Sirt-1.

## The connection with p53

The p53 protein and its encoding gene were first identified in 1979 because of its association with cancer [138-141] and its function as a tumor suppressor gene

www.impactaging.com 527 AGING, August 2012, Vol. 4 No. 8
pathway responds to DNA damage by the activation of p53 and AMPK, which in turn activates TSC2 via phosphorylation [176, 177]. This inactivates RHEB and then mTOR and shuts down translation while turning on autophagy. These events are p53 dependent in a cell, as well as TSC1–TSC2 dependent after DNA damage, as demonstrated by using cells that had no p53, TSC1, or TSC2 genes (from knockout mice). A different stress signal such as glucose starvation rapidly activates this pathway involving p53 and AMPK. AMPK can induce p53 by promoting phosphorylation on serine-15, a site known to be important for the activation of p53 [176]. There is a second wave of communication between p53 and the IGF-1–AKT and TOR pathways. Thus, Mak et al. [178] first demonstrated that PTEN was induced by

the activation of p53. Cell lines or mouse tissues that transcriptionally activated the PTEN gene after DNA damage also induced TSC2 gene transcription. The p53-mediated induction of PTEN and TSC2 acts in the same way as the faster p53–AMPK pathway. Increasing PTEN levels shuts down AKT activity and relieves its inhibition on TSC2, resulting in the inactivation of TOR, loss of phosphorylation of S6 kinase, and activation of autophagy [176, 179]. Similarly, the activation of AKT-1 and mTOR by the presence of nutrients and growth factors leads to the AKT-1-dependent activation of MDM2 by phosphorylation, which enhances its activity as a ubiquitin ligase and moves it into the nucleus so that it more effectively degrades and inactivates p53 [180].

![Diagram](attachment:diagram.png)

Figure 2. Linking caloric restriction (CR), sirtuins, and mTOR to the p53 pathway. CR and resveratrol activate autophagy through sirtuins, thus extending lifespan. The intracellular mTOR pathway via inputs of PI-3K, AMPK and other sensors integrates nutrient availability and drives cell growth and aging. Rapamycin and resveratrol inhibit the Sirtuin/mTOR network. CR and p53 may also inhibit mTOR activity through upregulation of known negative regulators PTEN, TSC2 and AMPK. The products of two p53 target genes, Sestrin 1 and 2 activate AMPK, which phosphorylates TSC2 and stimulates its GAP activity enabling mTOR inhibition. Glucose starvation inhibits mTOR by promoting TSC1/2 activation.

Conclusion

Here, we have reviewed the role of caloric restriction in longevity and argued that p53 is the connection in the abilities of both the Sirt-1 pathway and the TOR pathway to impact on longevity of cells and organisms. Furthermore the integration of the p53 pathway with the IGF-1 and TOR pathways brings together a number of overlapping concepts that play a central role in life processes. Through the transcriptional regulation of different target genes (PTEN, AMPK, Sestrin 1/2, TSC2) p53 negatively regulates the insulin/IGF-1 and TOR signaling, creating an interpathway network that permits cells to inhibit cell growth and division to avoid the introduction of errors during these processes under stress conditions (Figure 3). In this way, p53 increases the fidelity of these processes over the lifetime of an organism. Since decreased TOR/insulin/IGF-1 signaling extends lifespan, p53 may regulate aging and longevity through its down-regulation of the signaling of these two critical pathways.

![Diagram](https://i.imgur.com/yourimage.png)

Figure 3. The new and complex role of p53 in regulating aging and longevity through the transcriptional regulation of different target genes.

This novel, lifespan regulating function of p53 may be evolutionarily more ancient than its relatively recent role in apoptosis and tumour suppression, and is likely to provide many new insights into lifespan modulation.

ACKNOWLEDGEMENTS

This work has been supported by the Medical Research Council, UK.

Conflict of Interest Statement

The author of this manuscript has no conflict of interests to declare.

REFERENCES

1. Masoro EJ. Caloric restriction and aging: an update. Exp. Gerontol. 2009; 35:299–305.
2. Sohal RS, Weindruch R. Oxidative stress, caloric restriction, and aging. Science. 1996; 273:59–63.
3. Gems D, Doonan R. Antioxidant defense and aging in C. elegans. Cell Cycle. 2009; 8:1681-1687.
4. Blagosklonny MV. Aging and immortality: quasi-programmed senescence and its pharmacologic inhibition. Cell Cycle 2006; 5:2087-2102.
5. Pérez VI, Van Remmen H, Bokov A, Epstein CJ, Vijg J, Richardson A. The overexpression of major antioxidant enzymes does not extend the lifespan of mice. Aging Cell. 2009; 8:73-75.
6. Chen Y, Azad MB, Gibson SB. Superoxide is the major reactive oxygen species regulating autophagy. Cell Death Differ. 2009; 16:1040–1052.
7. Doonan R, McElwee JJ, Matthijssens F, Walker GA, Houthoofd K, Back P, et al. Against the oxidative damage theory of aging: superoxide dismutases protect against oxidative stress but have little or no effect on lifespan in Caenorhabditis elegans. Genes Dev. 2008; 22:3236-3241.
8. Mair W, Dillin A. Aging and survival: the genetics of lifespan extension by dietary restriction. Annu Rev Biochem. 2008; 77:727-754.
9. Bauer J et al. Comparative transcriptional profiling identifies takeout as a gene that regulates lifespan. Aging. 2010; 2:298-310.
10. McKeller MR, Herrera-Rodriguez S, Ma W, Ortiz-Quintero B, Rangel R, Candé C, Sims-Mourtada JC, Melnikova V, Kashi C, Phan LM, Chen Z, Huang P, Dunner K Jr, Kroemer G, Singh KK, Martinez-Valdez H. Vital function of PRELI and essential requirement of its LEA motif. Cell Death Dis. 2010; 1:e21.
11. Zhao C, Tan YC, Wong WC, Sem X, Zhang H, Han H, Ong SM, Wong KL, Yeap WH, Sze SK, Kourilsky P, Wong SC. The CD14(+/low)CD16(+) monocyte subset is more susceptible to spontaneous and oxidant-induced apoptosis than the CD14(+)CD16(-) subset. Cell Death Dis. 2010; 1:e95.
12. Bose R, Moors M, Tofighi R, Cascante A, Hermanson O, Ceccatelli S. Glucocorticoids induce long-lasting effects in neural stem cells resulting in senescence-related alterations. Cell Death Dis. 2010; 1:e92.
13. Yan L, Mieulet V, Lamb RF. Nutrient regulation of mTORC1 and cell growth. Cell Cycle. 2010; 9:13, 2473-2474.
14. Ma B, Sen T, Asnaghi L, Valapala M, Yang F, Hose S, McLeod DS, Lu Y, Eberhart C, Zigler JS Jr, Sinha D. βA3/A1-Crystallin controls anoikis-mediated cell death in astrocytes by modulating PI3K/AKT/mTOR and ERK survival pathways through the PKD/Bit1-signaling axis. Cell Death Dis 2011; 2:e217.
15. Soliman GA. The integral role of mTOR in lipid metabolism. Cell Cycle 2011; 10:861-862
16. Zha X, Sun Q, Zhang H. mTOR upregulation of glycolytic enzymes promotes tumor development. Cell Cycle 2011; 10:1015-1016.
17. Kim S, Snyder SH. Nutrient amino acids signal to mTOR via inositol polyphosphate multikinase. Cell Cycle 2011; 10:1708-10.
18. Conn CS, Qian SB. mTOR signaling in protein homeostasis: Less is more? Cell Cycle 2011; 10:1940-1947.

19. Won Jun Oh, Jacinto E. mTOR complex 2 signaling and functions. Cell Cycle 2011; 10:2305-2316.
20. Pani G. From growing to secreting: New roles for mTOR in aging cells. Cell Cycle 2011; 10:2450-2453.
21. Anisimov VN, Zabezhinski MA, Popovich IG, Piskunova TS, Semenchenko AV, Tyndyk M, Yurova M, Rosenfeld S, Blagosklonny MV. Rapamycin increases lifespan and inhibits spontaneous tumorigenesis in inbred female mice. Cell Cycle 2011; 10:4230-4236.
22. Schneider-Jakob S, Corazza N, Badmann A, Sidler D, Stuber-Roos R, Keogh A, Frese S, Tschan M, Brunner T. Synergistic induction of cell death in liver tumor cells by TRAIL and chemotherapeutic drugs via the BH3-only proteins Bim and Bid. Cell Death Dis. 2010; 1:e86.
23. Reis CR, van der Sloot AM, Natoni A, Szegezdi E, Setroikromo R, Meijer M, Sjollema K, Stricher F, Cool RH, Samali A, Serrano L, Quax WJ. Rapid and efficient cancer cell killing mediated by high-affinity death receptor homotrimerizing TRAIL variants. Cell Death Dis. 2010; 1:e83.
24. Heidari N, Hicks MA, Harada H. GX15-070 (obatoclax) overcomes glucocorticoid resistance in acute lymphoblastic leukemia through induction of apoptosis and autophagy. Cell Death Dis. 2010; 1:e76.
25. Flourakis M, Lehen'kyi V, Beck B, Raphaël M, Vandenberghe M, Abeele FV, Roudbaraki M, Lepage G, Mauroy B, Romanin C, Shuba Y, Skryma R, Prevarskaya N. Orai1 contributes to the establishment of an apoptosis-resistant phenotype in prostate cancer cells. Cell Death Dis. 2010; 1:e75.
26. Jiang CC, Lai F, Tay KH, Croft A, Rizos H, Becker TM, Yang F, Liu H, Thorne RF, Hersey P, Zhang XD. Apoptosis of human melanoma cells induced by inhibition of B-RAFV600E involves preferential splicing of bimS. Cell Death Dis. 2010; 1:e69.
27. Gonzalez-Mejia ME, Voss OH, Murnan EJ, Doseff AI. Apigenin-induced apoptosis of leukemia cells is mediated by a bimodal and differentially regulated residue-specific phosphorylation of heat-shock protein-27. Cell Death Dis. 2010; 1:e64.
28. Fricker M, O'Prey J, Tolkovsky AM, Ryan KM. Phosphorylation of Puma modulates its apoptotic function by regulating protein stability. Cell Death Dis. 2010; 1:e59.
29. Placzek WJ, Wei J, Kitada S, Zhai D, Reed JC, Pellecchia M. A survey of the anti-apoptotic Bcl-2 subfamily expression in cancer types provides a platform to predict the efficacy of Bcl-2 antagonists in cancer therapy. Cell Death Dis. 2010; 1:e40.
30. Olsson M, Zhivotovsky B. Caspases and cancer. Cell Death Differ 2011; 18:1441-1449.
31. Graber TE, Holcik M. Distinct roles for the cellular inhibitors of apoptosis proteins 1 and 2. Cell Death Dis 2011; 2:e135.
32. Mühlethaler-Mottet A, Flahaut M, Balmas Bourloud K, Nardou K, Coulon A, Liberman J, Thome M, Gross N. Individual caspase-10 isoforms play distinct and opposing roles in the initiation of death receptor-mediated tumour cell apoptosis. Cell Death Dis 201; 2:e125.
33. Cheung HH, St Jean M, Beug ST, Lejmi-Mrad R, LaCasse E, Baird SD, Stojdl D, Screaton RA, Korneluk RG. MG1 and NIK regulate apoptosis induced by Smac mimetic compounds. Cell Death Dis 2011; 2:e146.
34. Yivgi-Ohana N, Eifer M, Addadi Y, Neeman M, Gross A. Utilizing mitochondrial events as biomarkers for imaging apoptosis. Cell Death Dis 2011; 2:e166.
35. Rauert H, Stühmer T, Bargou R, Wajant H, Siegmund D. TNFR1 and TNFR2 regulate the extrinsic apoptotic pathway in myeloma cells by multiple mechanisms. Cell Death Dis 2011; 2: e194.
36. Teng X, Cheng W, Qi B, Yu T, Ramachandran K, Boersma MD, Hattier T, Lehmann P, Pineda FJ, Hardwick JM. Gene-dependent cell death in yeast. Cell Death Dis 2011; 2: e188.
37. Knight RA, Melino G. Cell death in disease: from 2010 onwards. Cell Death Dis 2011; 2: e202.
38. Seervi M, Joseph J, Sobhan PK, Bhavya B, Santhoshkumar TR. Essential requirement of cytochrome c release for caspase activation by procaspase-activating compound defined by cellular models. Cell Death Dis 2011; 2: e207.
39. Gonzalez-Halphen D, Ghelli A, Iommarini L, Carelli V, Espositi MD. Mitochondrial complex I and cell death: a semi-automatic shotgun model. Cell Death Dis 2011; 2: e222.
40. Kober AM, Legewie S, Pforr C, Fricker N, Eils R, Krammer PH, Lavrik IN. Caspase-8 activity has an essential role in CD95/Fas-mediated MAPK activation. Cell Death Dis 2011; 2: e212.
41. Vanlangenakker N, Bertrand MJM, Bogaert P, Vandenberghe P, Vanden Berghe T. TNF-induced necroptosis in L929 cells is tightly regulated by multiple TNFR1 complex I and II members. Cell Death Dis 2011; 2: e230.
42. Lee SH, Jung Y, Chung J, Oh A, Lee S, Choi D, Jang S, Jang K, Paik S, Ha N, Park B. Novel tumor suppressive function of Smad4 in serum starvation-induced cell death through PAK1-PUMA pathway. Cell Death Dis 2011; 2: e235.
43. Wu Y, Tan H, Huang Q, Sun X, Zhu X, Shen H. zVAD-induced necroptosis in L929 cells depends on autocrine production of TNFα mediated by the PKC-MAPKs-AP-1 pathway. Cell Death Differ 2010; 18:26-37.
44. Wang X, Olberding K, White C, Li C. Bcl-2 proteins regulate ER membrane permeability to luminal proteins during ER stress-induced apoptosis. Cell Death Differ 2010; 18:38-47.
45. Lian J, Wu X, He F, Karnak D, Tang W, Meng Y, Xiang D, Ji M, Lawrence T, Xu L. A natural BH3 mimetic induces autophagy in apoptosis-resistant prostate cancer via modulating Bcl-2-Becn1 interaction at endoplasmic reticulum. Cell Death Differ 2010; 18:60-71.
46. Choi H, Chang J, Jung Y. Peroxiredoxin 6 interferes with TRAIL-induced death-inducing signaling complex formation by binding to death effector domain caspase. Cell Death Differ 2010; 18:405-414.
47. Niture S, Jaiswal A. INrf2 (Keap1) targets Bcl-2 degradation and controls cellular apoptosis. Cell Death Differ 2010; 18:439-451.
48. De Zio D, Bordi M, Tino E, Lanzuolo C, Ferraro E, Mora E, Ciccosanti F, Fimia GM, Orlando V, Cecconi F. The DNA repair complex Ku70/86 modulates Apaf1 expression upon DNA damage. Cell Death Differ 2010; 18:516-527.
49. Lalier L, Cartron P, Olivier C, Logé C, Bougras G, Robert JM, Oliver Land, Vallette FM. Prostaglandins antagonistically control Bax activation during apoptosis. Cell Death Differ 2010; 18:528-537.
50. Schug ZT, Gonzalvez F, Houtkooper RH, Vaz FM, Gottlieb E. BID is cleaved by caspase-8 within a native complex on the mitochondrial membrane. Cell Death Differ 2010; 18:538-548.
51. Sirois I, Raymond M, Brassard N, Cailhier J, Fedjaev M, Hamelin K, London I, Bendayan M, Pshezhetsky A, Hébert M. Caspase-3-dependent export of TCTP: a novel pathway for

antiapoptotic intercellular communication. Cell Death Differ 2010; 18:549-562.

52. Zhang L, Lopez H, George N, Liu X, Pang X, Luo X. Selective involvement of BH3-only proteins and differential targets of Noxa in diverse apoptotic pathways. Cell Death Differ 2010; 18: 864-873.

53. Ge R, Wang Z, Zeng Q, Xu X, Olumi AF. F-box protein 10, an NF-kB-dependent anti-apoptotic protein, regulates TRAIL-induced apoptosis through modulating c-Fos/c-FLIP pathway. Cell Death Differ 2011; 18: 1184-1195.

54. Hui K, Kanungo A, Elia A, Henderson JT. Caspase-3 deficiency reveals a physiologic role for Smac/DIABLO in regulating programmed cell death. Cell Death Differ 2011; 18: 1780-1790.

55. Badiola N, Penas C, Miñano-Molina A, Barneda-Zahonero B, Fadó R, Sánchez-Opazo G, Comella J, Sabriá J, Zhu C, Blomgren K, Casas C, Rodríguez-Alvarez J. Induction of ER stress in response to oxygen-glucose deprivation of cortical cultures involves the activation of the PERK and IRE-1 pathways and of caspase-12. Cell Death Dis 2011; 2: e149.

56. Cordeiro MF, Guo L, Coxon KM, Duggan J, Nizari S, Normando EM, Sensi SL, Sillito AM, Fitzke FW, Salt TE, Moss SE. Imaging multiple phases of neurodegeneration: a novel approach to assessing cell death in vivo. Cell Death Dis. 2010; 1:e3.

57. Engel T, Tanaka K, Jimenez-Mateos EM, Caballero-Caballero A, Prehn JH, Henshall DC. Loss of p53 results in protracted electrographic seizures and development of an aggravated epileptic phenotype following status epilepticus. Cell Death Dis. 2010; 1:e79.

58. Di Guardo G, Marruchella G. Prions and neuronal death. Cell Death Dis. 2010; 1:e6.

59. Bunk EC, König HG, Bernas T, Engel T, Henshall DC, Kirby BP, Prehn JH. BH3-only proteins BIM and PUMA in the regulation of survival and neuronal differentiation of newly generated cells in the adult mouse hippocampus. Cell Death Dis. 2010; 1:e15.

60. Corasaniti MT, Melino G, Navarra M, et al. Death of cultured human neuroblastoma-cells induced by hiv-1 gp120 is prevented by nmda receptor antagonists and inhibitors of nitric-oxide and cyclooxygenase. Neurodegeneration. 1995; 3:315-321.

61. Bagetta G, Corasaniti MT, Melino G, et al. Lithium and tacrine increase the expression of nitric-oxide synthase messenger-RNA in the hippocampus of rat. Biochem. and Biophys. Res. Comm. 1997; 3:1132-1139.

62. Agostini M, Tucci P, Chen H. et al. p73 regulates maintenance of neural stem cell. Biochem. and Biophys. Res. Comm. 2010; 403:13-17.

63. Lim MP, Devi LA, Rozenfeld R. Cannabidiol causes activated hepatic stellate cell death through a mechanism of endoplasmic reticulum stress-induced apoptosis. Cell Death Dis 2011; 2: e170.

64. Blume-Jensen P, Hunter T. Oncogenic kinase signalling. Nature. 2001; 411:355–365.

65. Romano S, D'Angelillo A, Staibano S, Ilardi G, Romano MF. FK506-binding protein 51 is a possible novel tumoral marker. Cell Death Dis. 2010; 1:e55.

66. Knauer SK, Heinrich UR, Bier C, Habtemichael N, Docter D, Helling K, Mann WJ, Stauber RH. An otoprotective role for the apoptosis inhibitor protein survivin. Cell Death Dis. 2010; 1:e51.

67. Giampietri C, Petrungaro S, Coluccia P, Antonangeli F, Giannakakis K, Faraggiana T, Filippini A, Cossu G, Ziparo E. c-Flip

overexpression affects satellite cell proliferation and promotes skeletal muscle aging. Cell Death Dis. 2010; 1:e38.

68. Pasupuleti N, Matsuyama S, Voss O, Doseff AI, Song K, Danielpour D, Nagaraj RH. The anti-apoptotic function of human αA-crystallin is directly related to its chaperone activity. Cell Death Dis. 2010; 1:e31.

69. Chu KM, Minogue S, Hsuan JJ, Waugh MG. Differential effects of the phosphatidylinositol 4-kinases, PI4KIIα and PI4KIIIβ, on Akt activation and apoptosis. Cell Death Dis. 2010; 1:e106.

70. Mitchell G, Fillinger J, Sittadjody S, Avila J, Burd R, Limesand K. IGF1 activates cell cycle arrest following irradiation by reducing binding of ΔNp63 to the p21 promoter. Cell Death Dis. 2010; 1:e50.

71. Sears D, Luong P, Yuan M, Nteliopoulos G, Man YK, Melo JV, Basu S. Functional phosphoproteomic analysis reveals cold-shock domain protein A to be a Bcr-Abl effector-regulating proliferation and transformation in chronic myeloid leukemia. Cell Death Dis. 2010; 1:e93.

72. Datta SR, Dudek H, Tao X, Masters S, Fu H, Gotoh Y, Greenberg ME. Akt phosphorylation of BAD couples survival signals to the cell-intrinsic death machinery. Cell. 1997; 91:231–241.

73. Del Peso L, Gonzalez-Garcia M, Page C, Herrera R, Nunez G. Interleukin-3-induced phosphorylation of BAD through the protein kinase Akt. Science. 1997; 278:687–689.

74. Zhou BP, Liao Y, Xia W, Zou Y, Spohn B, Hung MC. HER-2/neu induces p53 ubiquitination via Akt-mediated MDM2 phosphorylation. Nat. Cell Biol. 2001; 3:973–982.

75. Inoki K, Li Y, Zhu T, Wu J, Guan KL. TSC2 is phosphorylated and inhibited by Akt and suppresses mTOR signalling. Nat. Cell Biol. 2002; 4:648–657.

76. Potter CJ, Pedraza LG, Xu T. Akt regulates growth by directly phosphorylating Tsc2. Nat. Cell Biol. 2002; 4:658–665.

77. Liu H, Zhang Y, Yang N, Zhang Y, Liu X, Li C, Zhao Y, Wang Y, Zhang G, Yang P, Guo F, Sun Y, Jiang C. A functionalized single-walled carbon nanotube-induced autophagic cell death in human lung cells through Akt-TSC2-mTOR signaling. Cell Death Dis 2011; 2: e159.

78. Kang Y, Lu M, Guan K. The TSC1 and TSC2 tumor suppressors are required for proper ER stress response and protect cells from ER stress-induced apoptosis. Cell Death Differ 2010; 18: 133-144.

79. Gao X, Pan D. TSC1 and TSC2 tumor suppressors antagonize insulin signaling in cell growth. Genes & Dev. 2001; 15:1383–1392.

80. Inoki K, Li Y, Xu T, Guan KL. Rheb GTPase is a direct target of TSC2 GAP activity and regulates mTOR signaling. Genes & Dev. 2003; 17:1829–1834.

81. Zhang Y, Gao X, Saucedo LJ, Ru B, Edgar BA, Pan D. Rheb is a direct target of the tuberous sclerosis tumour suppressor proteins. Nat. Cell Biol. 2003; 5:578–581.

82. Perumalsamy LR, Nagala M, Banerjee P, Sarin AA. A hierarchical cascade activated by non-canonical Notch signaling and the mTOR-Rictor complex regulates neglect-induced death in mammalian cells. Cell Death Differ. 2009; 6:879-889.

83. Levine B, Kroemer G. Autophagy in aging, disease and death: the true identity of a cell death impostor. Cell Death Differ. 2009; 16:1-2.

84. Huang Q, Wu YT, Tan HL, Ong CN, Shen HM. A novel function of poly(ADP-ribose) polymerase-1 in modulation of autophagy
and necrosis under oxidative stress. Cell Death Differ. 2009; 16: 264-277.
85. Sarkar S, Ravikumar B, Floto RA, Rubinsztein DC. Rapamycin and mTOR-independent autophagy inducers ameliorate toxicity of polyglutamine-expanded huntingtin and related proteinopathies. Cell Death Differ. 2009; 16:46-56.
86. Inoki K, Zhu T, Guan KL. TSC2 mediates cellular energy response to control cell growth and survival. Cell. 2003; 115: 577–590.
87. Hay N. Sonenberg N. Upstream and downstream of mTOR. Genes & Dev. 2004; 18:1926–1945.
88. Silver N, Proctor GB, Arno M. Carpenter GH. Activation of mTOR coincides with autophagy during ligation-induced atrophy in the rat submandibular gland. Cell Death Dis. 2010; 1:e14.
89. Vellai T. Autophagy genes and ageing. Cell Death Differ. 2009; 16:94–102.
90. Levine B. Kroemer G. Autophagy in aging, disease and death: the true identity of a cell death impostor. Cell Death Differ. 2009; 16:1–2.
91. Eisenberg-Lerner A, Bialik S, Simon HU. Kimchi A. Life and death partners: apoptosis, autophagy and the cross-talk between them. Cell Death Differ. 2009; 16:966–975.
92. Scarlatti F, Granata R, Meijer AJ, Codogno AP. Does autophagy have a license to kill mammalian cells? Cell Death Differ. 2009; 16:12–20.
93. Noda T, Fujita N, Yoshimori T. The late stages of autophagy: how does the end begin? Cell Death Differ. 2009; 16:984–990.
94. Fimia GM, Piacentini M. Toward the understanding of autophagy regulation and its interplay with cell death pathways. Cell Death Differ. 2009; 16:933–934.
95. Ling L, Tan K, Lin H, Chiu G. The role of reactive oxygen species and autophagy in safingol-induced cell death. Cell Death Dis 2011; 2: e129.
96. Donadelli M, Dando I, Zaniboni T, Costanzo C, Dalla Pozza E, Scupoli MT, Scarpa A, Zappavigna S, Marra M, Abbruzzese A, Bifulco M, Caraglia M, Palmieri M. Gemcitabine/cannabinoid combination triggers autophagy in pancreatic cancer cells through a ROS-mediated mechanism. Cell Death Dis 2011; 2: e152.
97. Russo R, Berliocchi L, Adornetto A, Varano GP, Cavaliere F, Nucci F, Rotiroti D, Morrone LA, Bagetta G, Corasaniti MT. Calpain-mediated cleavage of Beclin-1 and autophagy deregulation following retinal ischemic injury in vivo. Cell Death Dis 2011; 2: e144.
98. Kuwahara Y, Oikawa T, Ochiai Y, Roudkenar MH, Fukumoto M, Shimura T, Ohtake Y, Ohkubo Y, Mori S, Uchiyama Y, Fukumoto. Enhancement of autophagy is a potential modality for tumors refractory to radiotherapy M. Cell Death Dis 2011; 2: e177.
99. Tomic T, Botton T, Cerezo M, Robert G, Luciano F, Puissant A, Gounon P, Allegra M, Bertolotto C, Bereder JM, Tartare-Deckert S, Bahadoran P, Auberger P, Ballotti R, Rocchi S. Metformin inhibits melanoma development through autophagy and apoptosis mechanisms. Cell Death Dis 2011; 2: e199.
100. Kang R, Zeh HJ, Lotze MT, Tang D. The Beclin 1 network regulates autophagy and apoptosis. Cell Death Differ 2011; 18: 571-580.
101. Barth JMI, Szabad J, Hafen E, Köhler K. Autophagy in Drosophila ovaries is induced by starvation and is required for oogenesis. Cell Death Differ 2010; 18: 915-924.
102. Vara D, Salazar M, Olea-Herrero N, Guzmán M, Velasco G, Diaz-Laviada I. Anti-tumoral action of cannabinoids on

hepatocellular carcinoma: role of AMPK-dependent activation of autophagy. Cell Death Differ 2011; 18: 1099-1111.
103. Laussmann MA, Passante E, Düssmann H, Rauen JA, Würstle ML, Delgado ME, Devocelle M, Prehn JHM, Rehm M. Proteasome inhibition can induce an autophagy-dependent apical activation of caspase-8. Cell Death Differ 2011; 18: 1584–1597.
104. Gomes LC, Di Benedetto G, Scorrano L. Essential amino acids and glutamine regulate induction of mitochondrial elongation during autophagy. Cell Cycle 2011; 10: 2635-2639.
105. Boehner S, Lainey E, Kroemer G. Coordinated epigenetic regulation of autophagy and apoptosis. Cell Cycle 2011; 10: 2832-3.
106. Liang XH, Jackson S, Seaman M, Brown K, Kempkes B, Hibshoosh H et al. Induction of autophagy and inhibition of tumorigenesis by beclin 1. Nature. 1999; 402:672–676.
107. Nishida K, Kyoi S, Yamaguchi O, Sadoshima J, Otsu K. The role of autophagy in the heart. Cell Death Differ. 2009; 16:31–38.
108. Deng L, Feng J, Broaddus RR. The novel estrogen-induced gene ELG121 regulates autophagy and promotes cell survival under stress. Cell Death Dis. 2010; e32.
109. Bennett HL, Fleming JT, O’Prey J, Ryan KM, Leung HY. Androgens modulate autophagy and cell death via regulation of the endoplasmic reticulum chaperone glucose-regulated protein 78/BiP in prostate cancer cells. Cell Death Dis. 2010; 1:e72.
110. Marino ML, Fais S, Djavaheri-Mergny M, Villa A, Meschini S, Lozupone F, Venturi G, Della Mina P, Pattingre S, Rivoltini L, Codogno P, De Milito A. Proton pump inhibition induces autophagy as a survival mechanism following oxidative stress in human melanoma cells. Cell Death Dis. 2010; 1:e87.
111. Heidari N, Hicks MA, Harada H. GX15-070 (obatoclax) overcomes glucocorticoid resistance in acute lymphoblastic leukemia through induction of apoptosis and autophagy. Cell Death Dis. 2010; 1:e76.
112. Li Q, Li H, Roughton K, Wang X, Kroemer G, Blomgren K, Zhu C. Lithium reduces apoptosis and autophagy after neonatal hypoxia-ischemia. Cell Death Dis. 2010; 1:e56.
113. McCoy F, Hurwitz J, McTavish N, Paul I, Barnes C, O'Hagan B, Odrzywol K, Murray J, Longley D, McKerr G, Fennell DA. Obatoclax induces Atg7-dependent autophagy independent of beclin-1 and BAX/BAK. Cell Death Dis. 2010; 1:e108.
114. Ruela-de-Sousa RR, Fuhler GM, Blom N, Ferreira CV, Aoyama H, Peppelenbosch MP. Cytotoxicity of apigenin on leukemia cell lines: implications for prevention and therapy. Cell Death Dis. 2010; 1:e19.
115. Wirawan E, Vande Walle L, Kersse K, Cornelis S, Claerhout S, Vanoverberghe I, Roelandt R, De Rycke R, Verspurten J, Declercq W, Agostinis P, Vanden Berghe T, Lippens S, Vandenabeele P. Caspase-mediated cleavage of Beclin-1 inactivates Beclin-1-induced autophagy and enhances apoptosis by promoting the release of proapoptotic factors from mitochondria. Cell Death Dis. 2010; 1:e18.
116. Liang C. Negative regulation of autophagy. Cell Death Differ. 2010; 12:1807-1815.
117. Foster DA, Toschi A. Targeting mTOR with rapamycin. Cell Cycle. 2009; 8:1026-1029.
118. Sarkar1 S, Ravikumar B, Floto RA, Rubinsztein DC. Rapamycin and mTOR-independent autophagy inducers ameliorate toxicity of polyglutamineexpanded huntingtin and related proteinopathies. Cell Death Differ. 2009; 16:46–56.
119. Blagosklonny MV. TOR-driven aging. Cell Cycle. 2009; 8:4055-4059.
120. Blagosklonny MV. Rapamycin and quasi-programmed aging. Cell Cycle. 2010; 9:1859-1862.
121. Blagosklonny MV. Increasing healthy lifespan by suppressing aging in our lifetime: preliminary proposal. Cell Cycle. 2010; 9:4788-4794.
122. Blagosklonny MV. Revisiting the antagonistic pleiotropy theory of aging: TOR-driven program and quasi-program. Cell Cycle. 2010; 9:3151-3156.
123. Kim J, Guan KL. Regulation of the autophagy initiating kinase ULK1 by nutrients: Roles of mTORC1 and AMPK. Cell Cycle 2011; 10:1337-1338.
124. Yu J, Henske EP. TORC1 activation and autophagy inhibition: Opposing forces in Rheb-driven tumorigenesis. Cell Cycle 2011; 10:3803-3804.
125. Cohen HY, Miller C, Bitterman KJ, Wall NR, Hekking B, Kessler Bet al. Calorie restriction promotes mammalian cell survival by inducing the SIRT1 deacetylase. Science. 2004; 305:390-392.
126. Wood JG, Rogina B, Lavu S, Howitz K, Helfand SL, Tatar Met al. Sirtuin activators mimic caloric restriction and delay ageing in metazoans. Nature. 2004; 430:686–689.
127. Mostoslavsky R, Manel Esteller and Alejandro Vaquero. At the crossroad of lifespan, calorie restriction, chromatin and disease. Cell Cycle. 2010; 9:1907-1912.
128. Kaeberlein M, McVey M, Guarente L. The SIR2/3/4 complex and SIR2 alone promote longevity in Saccharomyces cerevisiae by two different mechanisms. Genes Develop. 1999; 13:2570–2580.
129. Tissenbaum HA, Guarente L. Increased dosage of a sir-2 gene extends lifespan in Caenorhabditis elegans. Nature. 2001; 410:227–230.
130. Morselli E, Maiuri MC, Markaki M, Megalou E, Pasparaki A, Palikaras K, Criollo A, Galluzzi L, Malik SA Vitale I, Michaud M, Madeo F, Tavernarakis N, Kroemer G. Caloric restriction and resveratrol promote longevity through the Sirtuin-1-dependent induction of autophagy. Cell Death Dis. 2010; 1:e10.
131. Blagosklonny MV. Linking calorie restriction to longevity through sirtuins and autophagy: any role for TOR. Cell Death Dis. 2010; 1:e12.
132. M Campagna M, D Herranz D, MA Garcia MA, L Marcos-Villar L, J Gonzalez-Santamaria J, P Gallego P, S Gutierrez S, M Collado M, M Serrano M, M Esteban E, and C Rivas C. SIRT1 stabilizes PML promoting its sumoylation. Cell Death Differ 2010; 18:72-79.
133. Stein S, Matter CM. Protective roles of SIRT1 in atherosclerosis. Cell Cycle 2011; 10:640-47.
134. Antosh M, Whitaker R, Kroll A, Hosier S, Chang C, Bauer J, Cooper L, Neretti N, Helfand SL. Comparative transcriptional pathway bioinformatic analysis of dietary restriction, Sir2, p53 and resveratrol life span extension in Drosophila. Cell Cycle 2011; 10:904-11.
135. Milner J, Allison SJ. SIRT1, p53 and mitotic chromosomes. Cell Cycle 2011; 10:3049.
136. Medvedik O, Lamming DW, Kim KD, Sinclair DA. MSN2 and MSN4 link calorie restriction and TOR to Sirtuin-mediated lifespan extension in Saccharomyces cerevisiae. PLoS Biol. 2007; 5:e261.
137. Demidenko ZN, Blagosklonny MV. At concentrations that inhibit mTOR, resveratrol suppresses cellular senescence. Cell Cycle. 2009; 8:1901-1904.
138. Linzer DIH, Levine AJ. Characterization of a 54K dalton cellular SV40 tumor antigen present in SV40-transformed cells and uninfected embryonal carcinoma cells. Cell. 1979; 17:43–52.
139. Lane D, Crawford LV. T-antigen is bound to a host protein in SV40-transformed cells. Nature. 1979; 278:261–263.
140. Deleo AB. et al. Detection of a transformed-related antigen in chemically induced sarcomas and other transformed cells of the mouse. Proc. Natl Acad. Sci. USA. 1979; 76:2420–2424.
141. Mahmoudi S, Henriksson S, Farnebo L, Roberg K, Farnebo M. WRAP53 promotes cancer cell survival and is a potential target for cancer therapy. Cell Death Dis 2010; 2:e114.
142. Finlay CA, Hinds PW, Levine AJ. The p53 proto-oncogene can act as a suppressor of transformation. Cell. 1989; 57:1083–1093.
143. Chen PL, Chen YL, Bookstein R, Lee WH. Genetic mechanisms of tumor suppression by the human p53 gene. Science. 1990; 250:1576–1580.
144. Mitchell GC, Fillinger JL, Sittadjody S, Avila JL, Burd R, Limesand KH. IGF1 activates cell cycle arrest following irradiation by reducing binding of ΔNp63 to the p21 promoter. Cell Death Dis. 2010; 1:e50.
145. Engel T, Tanaka K, Jimenez-Mateos EM, Caballero-Caballero A, Prehn JH, Henshall DC. Loss of p53 results in protracted electrographic seizures and development of an aggravated epileptic phenotype following status epilepticus. Cell Death Dis. 2010; 1:e79.
146. Fricker M, O'Prey J, Tolkovsky AM, Ryan KM. Phosphorylation of Puma modulates its apoptotic function by regulating protein stability. Cell Death Dis. 2010; 1:e59.
147. Meley D, Spiller DG, White MR, McDowell H, Pizer B, Sée V. p53-mediated delayed NF-kB activity enhances etoposide-induced cell death in medulloblastoma. Cell Death Dis. 2010; 1:e41.
148. Rello-Varona S, Kepp O, Vitale I, Michaud M, Senovilla L, Jemaà M, Joza N, Galluzzi L, Castedo M, Kroemer G. An automated fluorescence videomicroscopy assay for the detection of mitotic catastrophe. Cell Death Dis. 2010; 1:e25.
149. Gonzalez-Cano L, Herreros-Villanueva M, Fernandez-Alonso R, Ayuso-Sacido A, Meyer G, Garcia-Verdugo JM, Silva A, Marques MM, Marin MC. p73 deficiency results in impaired self renewal and premature neuronal differentiation of mouse neural progenitors independently of p53. Cell Death Dis. 2010; 1:e109.
150. Yuan M, Luong P, Hudson C, Gudmundsdottir K, Basu S. c-Abl phosphorylation of ΔNp63α is critical for cell viability. Cell Death Dis. 2010; 1:e16.
151. Ramadan S, Terrinoni A, Catani MV, et al. p73 induces apoptosis by different mechanisms. Biochem. Biophys. Res. Comm. 2005; 331:713-717.
152. Catani MV, Rossi A, Costanzo A, et al. Induction of gene expression via activator protein-1 in the ascorbate protection against UV-induced damage. Biochemical Journal. 2001; 356:77-85.
153. Melino G. p63 is a suppressor of tumorigenesis and metastasis interacting with mutant p53. Cell Death Differ. 2011; 18:1487-1499.

154. Collavin L, Lunardi A, Del Sal G. p53-family proteins and their regulators: hubs and spokes in tumor suppression. Cell Death Differ. 2010; 17901-17911.
155. Yalcin-Ozuysal O, Fiche M, Gutierrez M, Wagner KU, Raffoul W, Brisken C. Antagonistic roles of Notch and p63 in controlling mammary epithelial cell fates. Cell Death Differ. 2010; 17:1600-1612.
156. Cheok CF, Kua N, Kaldis P, Lane DP. Combination of nutlin-3 and VX-680 selectively targets p53 mutant cells with reversible effects on cells expressing wild-type p53. Cell Death Differ. 2010; 17:1486-1500.
157. Marine JC. p53 stabilization: the importance of nuclear import. Cell Death Differ. 2010; 17:191-192.
158. Lambertz I, Nittner D, Mestdagh P, Denecker G, Vandesompele J, Dyer MA, Marine JC. Monoallelic but not biallelic loss of Dicer1 promotes tumorigenesis in vivo. Cell Death Differ. 2010; 17:633-641.
159. Christoffersen NR, Shalgi R, Frankel LB, Leucci E, Lees M, Klausen M, Pilpel Y, Nielsen FC, Oren M, Lund AH. p53-independent upregulation of miR-34a during oncogene-induced senescence represses MYC. Cell Death Differ. 2010; 17:236-245.
160. Hermeking H. The miR-34 family in cancer and apoptosis. Cell Death Differ. 2010; 17:193-199.
161. Davies L, Spiller D, White MRH, Grierson I, Paraoan L. PERP expression stabilizes active p53 via modulation of p53-MDM2 interaction in uveal melanoma cells. Cell Death Dis 2011; 2: e136.
162. Koster R, Timmer-Bosscha H, Bischoff R, Gietema JA, de Jong S. Disruption of the MDM2-p53 interaction strongly potentiates p53-dependent apoptosis in cisplatin-resistant human testicular carcinoma cells via the Fas/FasL pathway. Cell Death Dis 2011; 2: e148.
163. Sermeus A, Michiels C. Reciprocal influence of the p53 and the hypoxic pathways. Cell Death Dis 2011; 2: e164.
164. Vaseva AV, Yallowitz AR, Marchenko ND, Xu S, Moll UM. Blockade of Hsp90 by 17AAG antagonizes MDMX and synergizes with Nutlin to induce p53-mediated apoptosis in solid tumors. Cell Death Dis 2011; 2: e156.
165. Graupner V, Alexander E, Overkamp T, Rothfuss O, De Laurenzi V, Gillissen BF, Daniel PT, Schulze-Osthoff K, Essmann F. Differential regulation of the proapoptotic multidomain protein Bak by p53 and p73 at the promoter level. Cell Death Differ 2011; 18: 1130-1139.
166. Gao W, Shen Z, Shang L, Wang X. Upregulation of human autophagy-initiation kinase ULK1 by tumor suppressor p53 contributes to DNA-damage-induced cell death. Cell Death Differ 2011; 18: 1598-1607.
167. Spinnler C, Hedström E, Li H, de Lange J, Nikulenkov F, Teunisse AF, Verlaan-de Vries M, Grinkevich V, Jochemsen AG, Selivanova G. Abrogation of Wip1 expression by RITA-activated p53 potentiates apoptosis induction via activation of ATM and inhibition of HdmX. Cell Death Differ. 2011; 18:1736-1745.
168. Marcel V, Dichtel-Danjoy ML, Sagne C, Hafsi H, Ma D, Ortiz-Cuaran S, Olivier M, Hall J, Mollereau B, Hainaut P, Bourdon JC. Biological functions of p53 isoforms through evolution: lessons from animal and cellular models. Cell Death Differ. 2011; 18:1815-1824.
169. Maiuri MC, Tasdemir E, Criollo A, Morselli E, Vicencio JM, Carnuccio R, Kroemer G. Control of autophagy by oncogenes and tumor suppressor genes. Cell Death Differ. 2009; 16:87–93.

170. Hu W, Feng Z, Teresky AK, Levine AJ. p53 regulates maternal reproduction through LIF. Nature. 2007; 450:721-724.
171. Levine AJ, Tomasini R, McKeon FD, Mak TW, Melino G. The p53 family: guardians of maternal reproduction. Nat Rev Mol Cell Biol. 2011; 12:259-265.
172. Van Heemst D, Mooijaart SP, Beekman M, et al. Variation in the human TP53 gene affects old age survival and cancer mortality. Exp Gerontol. 2005; 40:11-5.
173. Orsted DD, Bojesen SE, Tybjaerg-Hansen A, Nordestgaard BG. Tumor suppressor p53 Arg-72Pro polymorphism and longevity, cancer survival, and risk of cancer in the general population. J Exp Med. 2007; 204:1295-1301.
174. Bauer JH, Helfand SL. Sir2 and longevity. The p53 connection. Cell Cycle. 2009; 8:1818-1822.
175. Jung-Hynes B, Ahmad N. Role of p53 in the anti-proliferative effects of Sirt1 inhibition in prostate cancer cells. Cell Cycle. 2009; 8:1478-1483.
176. Feng Z, Zhang H, Levine AJ, Jin S. The coordinate regulation of the p53 and mTOR pathways in cells. Proc. Natl. Acad. Sci. 2005; 102:8204-8209.
177. Saha AK, Xu X, Balon TW, Brandon A, Kraegen EW, Ruderman NB. Insulin resistance due to nutrient excess: Is it a consequence of AMPK downregulation?. Cell Cycle 2011; 10: 3447-3451.
178. Stambolic V, MacPherson D, Sas D, Lin Y, Snow B, Jang Y, Benchimol S, Mak TW. Regulation of PTEN transcription by p53. Mol. Cell. 2001; 8:317–325.
179. Matthew EM, Lori S. Hart, Aristotelis Astrinidis, Arunasalam Navaraj, Nathan G. Dolloff, David T. Dicker, Elizabeth P. Henske, and Wafik S. El-Deiry. The p53 target Plk2 interacts with TSC proteins impacting mTOR signaling, tumor growth and chemosensitivity under hypoxic conditions. Cell Cycle. 2009; 8:4168-4175.
180. Gottlieb TM, Leal JF, Seger R, Taya Y, Oren M. Cross-talk between Akt, p53 and Mdm2: Possible implications for the regulation of apoptosis. Oncogene. 2002; 21:1299–1303.
