
Structural Basis for Microtubule Binding and Release by Dynein  
William B. Redwine *et al.*  
Science 337, 1532 (2012);  
DOI:10.1126/science.1224151  

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by [clicking here](https://www.sciencemag.org).

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines [here](https://www.sciencemag.org).

The following resources related to this article are available online at www.sciencemag.org (this information is current as of January 26, 2014):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:
[http://www.sciencemag.org/content/337/6101/1532.full.html](http://www.sciencemag.org/content/337/6101/1532.full.html)

Supporting Online Material can be found at:
[http://www.sciencemag.org/content/suppl/2012/09/19/337.6101.1532.DC1.html](http://www.sciencemag.org/content/suppl/2012/09/19/337.6101.1532.DC1.html)

This article cites 53 articles, 17 of which can be accessed free:
[http://www.sciencemag.org/content/337/6101/1532.full.html#ref-list-1](http://www.sciencemag.org/content/337/6101/1532.full.html#ref-list-1)

This article has been cited by 8 articles hosted by HighWire Press; see:
[http://www.sciencemag.org/content/337/6101/1532.full.html#related-urls](http://www.sciencemag.org/content/337/6101/1532.full.html#related-urls)

This article appears in the following subject collections:
Biochemistry
[http://www.sciencemag.org/cgi/collection/biochem](http://www.sciencemag.org/cgi/collection/biochem)

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2012 by the American Association for the Advancement of Science; all rights reserved. The title *Science* is a registered trademark of AAAS.

proposed existence of subsurfaces in water nano-
crystals (6). Torchet *et al.* gave a correct estimate
for the transition between amorphous and crystal-
line behavior (somewhere between $n = 200$ and
1000) based on the electron diffraction experi-
ment (26).

The determination of the size range in which
the critical nuclei of crystal water are formed is
crucial information for resolving the crystalliza-
tion mechanism of water. The onset of crystal-
linity defines a mechanistic branching point and
affects barrier heights controlling water aggre-
gation and hence measurable nucleation rates,
visible morphologies, and macroscopic proper-
ties (40). In the future, it will be interesting to
systematically study to what extent the crystalli-
zation process is a function of temperature, which
can be influenced by the expansion conditions.
The experimental technique presented here opens
the door to systematic studies in this direction.
Furthermore, the size-selected IR spectra in the
OH-stretching region will provide benchmarks
for theory to determine the predominate cluster
structures in the size range of $n = 50$ to 500. This
information will help to refine water interaction
potentials that are used for predicting macro-
scopic properties of water (40). More broadly,
the method should be applicable to alcohols (24)
and many other protic solvents, as well as other
dopants (such as potassium and cesium).

**References and Notes**

1. M. Kulmala, Science **302**, 1000 (2003).
2. L. Delzeit, D. Blake, J. Geophys. Res. Planets **33**, 379
   (2001).
3. R. Zhang, A. Khalizov, L. Wang, M. Hu, W. Xu, Chem. Rev.
   **112**, 1957 (2012).
4. E. M. Knipping *et al.*, Science **288**, 301 (2000).
5. U. Buck, F. Huiskens, Chem. Rev. **100**, 3863 (2000).
6. V. Buch, S. Baurecker, J. P. Devlin, U. Buck,
   J. K. Kazimirski, Int. Rev. Phys. Chem. **23**, 375 (2004).
7. U. Buck, H. Meyer, Phys. Rev. Lett. **52**, 109 (1984).
8. U. Buck, J. Phys. Chem. **98**, 5190 (1994).
9. U. Buck, I. Ettischer, M. Melzer, V. Buch, J. Sadlej,
   Phys. Rev. Lett. **80**, 2578 (1998).
10. K. Liu, J. D. Cruzan, R. J. Saykally, Science **271**, 929
    (1996).
11. E. G. Diken, W. H. Robertson, M. A. Johnson, J. Phys. Chem.
    A **108**, 64 (2004).
12. C. J. Gruenloh *et al.*, Science **276**, 1678 (1997).
13. T. Watanabe, T. Ebata, S. Tanabe, N. Mikami, J. Chem.
    Phys. **105**, 408 (1996).
14. W. Roth *et al.*, Chem. Phys. **239**, 1 (1998).
15. K. Mizuse, T. Hamashima, A. Fujii, J. Phys. Chem. A **113**,
    12134 (2009).
16. T. Hamashima, K. Mizuse, A. Fujii, J. Phys. Chem. A **115**,
    620 (2011).
17. C. P. Schulz, R. Haugstätter, H.-U. Tittes, I. V. Hertel IV,
    Phys. Rev. Lett. **57**, 1703 (1986).
18. I. V. Hertel IV, C. Hüglin, C. Nitsch, C. P. Schulz,
    Phys. Rev. Lett. **67**, 1767 (1991).
19. S. Schütte, U. Buck, Int. J. Mass Spectrom. **220**, 183
    (2002).
20. C. Bobbert, S. Schütte, C. Steinbach, U. Buck, Eur. Phys. J. D
    **19**, 183 (2002).
21. C. Steinbach *et al.*, J. Phys. Chem. A **108**, 6165
    (2004).
22. C. Steinbach, U. Buck, J. Phys. Chem. A **110**, 3128
    (2006).
23. U. Buck, I. Dauster, B. Gao, Z. F. Liu, J. Phys. Chem. A
    **111**, 12355 (2007).
24. R. M. Forck *et al.*, Phys. Chem. Chem. Phys. **14**, 3004
    (2012).
25. R. M. Forck *et al.*, Phys. Chem. Chem. Phys. **14**, 9054
    (2012).
26. G. Torchet, P. Schwartz, J. Farges, M. F. de Feraudy,
    B. Raoult, J. Chem. Phys. **79**, 6196 (1983).
27. J. Huang, L. S. Bartell, J. Phys. Chem. **99**, 3924 (1995).
28. O. Rodriguez Jr., J. M. Lisy, J. Phys. Chem. Lett. **2**, 1444
    (2011).
29. R. M. Forck *et al.*, J. Chem. Phys. **132**, 221102 (2010).
30. I. Dauster, M. A. Suhm, U. Buck, T. Zeuch, Phys. Chem.
    Chem. Phys. **10**, 83 (2008).
31. S. Krischok, O. Hoff, J. Gunster, R. Souda, V. Kempter,
    Nucl. Instrum. Methods Phys. Res. Sect. B **203**, 124
    (2003).
32. J. H. Kim, Y. K. Kim, H. Kang, J. Phys. Chem. C **113**, 321
    (2009).
33. W. A. Alexander, J. P. Wiens, T. K. Minton, G. M. Nathanson,
    Science **335**, 1072 (2012).
34. Y. Ferro, A. Allouche, J. Chem. Phys. **118**, 10461
    (2003).
35. D. R. Haynes, N. J. Tro, S. M. George, J. Chem. Phys. **96**,
    8502 (1992).
36. C. Hock *et al.*, Phys. Rev. Lett. **103**, 073401 (2009).
37. P. W. Forsyński, P. Zielke, D. Luckhaus, J. Corbett,
    R. Signorell, J. Chem. Phys. **134**, 094314 (2011).
38. C. Steinbach, U. Buck, Phys. Chem. Chem. Phys. **7**, 986
    (2005).
39. J. T. O'Brien, E. R. Williams, J. Am. Chem. Soc. **134**,
    10228 (2012).
40. E. B. Moore, V. Molinero, Nature **479**, 506 (2011).

**Acknowledgments:** T.Z. acknowledges the ongoing support of
M. Suhm and funding by the Deutsche Forschungsgemeinschaft
(GRK 782, grant ZE 890-1-1). P.S. acknowledges grant
P208/11/0161 of Grant Agency of the Czech Republic. All
information regarding the generation of the presented IR
spectra from raw data are given in the text and the
supplementary materials.

**Supplementary Materials**
www.sciencemag.org/cgi/content/full/337/6101/1529/DC1
Supplementary Text
Fig. S1

31 May 2012; accepted 9 August 2012
10.1126/science.1225468

---

# Structural Basis for Microtubule Binding and Release by Dynein

William B. Redwine,$^{1,2*}$ Rogelio Hernández-López,$^{1*}$ Sirui Zou,$^{2}$ Julie Huang,$^{2}$
Samara L. Reck-Peterson,$^{2}$ Andres E. Leschziner$^{1\dagger}$

Cytoplasmic dynein is a microtubule-based motor required for intracellular transport and cell division. Its movement involves coupling cycles of track binding and release with cycles of force-generating nucleotide hydrolysis. How this is accomplished given the ~25 nanometers separating dynein's track- and nucleotide-binding sites is not understood. Here, we present a subnanometer-resolution structure of dynein’s microtubule-binding domain bound to microtubules by cryo–electron microscopy that was used to generate a pseudo-atomic model of the complex with molecular dynamics. We identified large rearrangements triggered by track binding and specific interactions, confirmed by mutagenesis and single-molecule motility assays, which tune dynein’s affinity for microtubules. Our results provide a molecular model for how dynein’s binding to microtubules is communicated to the rest of the motor.

Dyneins are adenosine 5′-triphosphate (ATP)-driven molecular motors that move toward the minus ends of microtubules (MTs) (I). The superfamily includes axonemal dyneins, which power the movements of cilia, and those that transport cargo, which include cytoplasmic dyneins 1 (“cytoplasmic”) and 2 (“intraflagellar”) (2). The transport of organelles, ribonucleoprotein complexes, and proteins by cytoplasmic dynein is required for cellular homeostasis, cell-cell communication, cell division, and cell migration (3), and defects in these processes result in neurological disease in humans

(4). Despite cytoplasmic dynein’s role in such diverse activities and recent advances in characterizing its structure and motility, many aspects of its molecular mechanism remain poorly understood.

The core of the cytoplasmic dynein holoenzyme is a homodimer of ~500-kD motor-containing subunits (Fig. 1A). The major functional elements include (i) a “tail” domain required for dimerization and cargo binding (5); (ii) the force-generating “head” or “motor domain” (6, 7), a ring containing six AAA+ adenosine triphosphatase (ATPase) domains (8–10); (iii) a “linker” connecting the head and tail, required for motility (6, 7, 11, 12); (iv) the “stalk”—a long, antiparallel coiled-coil that emerges from AAA4 (13, 14); and (v) the MT-binding domain (MTBD), a small α-helical domain at the end of the stalk responsible for binding the MT track (15–17). Unlike the other cytoskeletal molecular motors, kinesin and myosin, dynein does not have its ATPase and polymer track binding sites located within a single domain. With 25 nm separating AAA1, the main site of ATP hydrolysis, and the MTBD, an unresolved question is how dynein coordinates the cycles of nucleotide hydrolysis and MT binding and release required for its motion.

The mechanism coupling nucleotide hydrolysis to MT affinity has been suggested to be sliding in which the two helices in the stalk’s coiled-coil

1Department of Molecular and Cellular Biology, Harvard University, 52 Oxford Street, Cambridge, MA 02138, USA. 2Department of Cell Biology, Harvard Medical School, 240 Longwood Avenue, Boston, MA 02115, USA.

*These authors contributed equally to this work.

†To whom correspondence should be addressed. E-mail: aleschziner@mcb.harvard.edu

adopt different registries by using alternative sets of hydrophobic heptad repeats (17–19). Dynein’s stalk can be fixed in a specific registry by fusing it to a coiled-coil of known structure, such as that of seryl-tRNA synthetase (SRS) (18). Changing the length of the first stalk helix (CC1) relative to the second (CC2) shifts their alignment between high-affinity (“α”) and low-affinity (“β+”) registries and alters the affinity of the MTBD for MTs by up to a factor of 20 (Fig. 1B) (18). Engineered disulfide cross-linking in the monomeric dynein motor domain showed that its stalk explores multiple registries in solution and that a given registry is coupled to a specific MT affinity (19). Fixing the stalk registry also uncoupled the nucleotide state of the head from MT affinity (19).

Understanding the molecular mechanism by which stalk sliding is coupled to nucleotide hydrolysis and MT affinity would be aided by a structure of dynein’s MTBD bound to MTs. Crystal structures are available for a low-affinity dynein MTBD fused to SRS through a short segment of the stalk (Fig. 1B) (17), and an adenosine 5′-diphosphate (ADP)-bound, presumably high-affinity dynein monomer (20). Only low-resolution structures of the MT-bound form have been reported (17, 21). Here we describe a cryo-electron microscopy (cryo-EM) reconstruction of the MTBD bound to MTs at subnanometer resolution, using a high-affinity version of the SRS-MTBD construct (18).

We collected images of MTs highly decorated with the SRS-MTBD under cryogenic conditions (Fig. 1C and fig. S1A) and adapted an image-processing method (22, 23) to solve its structure bound to 14-protofilament MTs. The SRS-MTBD binds to α-tubulin and β-tubulin at the intradimer interface and is positioned to the side of the protofilament, as previously reported (17, 21) (Fig. 1D, fig. S1B, and movie S1). In the reconstruction, the portion including the MT, the MTBD, and the beginning of the stalk has a resolution of 9.7 Å (fig. S1D), where α helices become visible.

We used molecular dynamics (MD) and our cryo-EM reconstruction to obtain pseudo-atomic models of the low- and high-affinity states of the MTBD bound to MTs. First, we rigid-body docked the atomic-resolution structures of the tubulin dimer (24, 25) and the low-affinity MTBD into our map (Fig. 2, A and C; fig. S2A; and movies S1 and S2) and used MD to resolve steric clashes between a helix (H1) in the MTBD and the MT (Fig. 2, A and C; fig. S2, A and B; and movies S1 and S2). We then performed explicit-solvent molecular dynamics flexible fitting (MDFF) to shift the low-affinity MTBD structure to the high-affinity conformation in our reconstruction (fig. S2C). In addition to the MD force field, MDFF uses a potential-energy term derived from the experimental map and restraints on secondary structure to drive conformational changes that better fit the map (26, 27). The initial MDFF model agreed well within the experimental density of the MTBD (fig. S2C); however, the stalk remained in the low-affinity β+ registry present in the starting model. MDFF likely did not shift the registry of the coiled-coil due to the lower resolution at the tip of the stalk segment (fig. S1); a movement of the CC1 helix to the α registry would make it protrude from the map, incurring a penalty in the simulations. We achieved the shift by applying targeted molecular dynamics (TMD) (28) to the tip of the coiled-coil (fig. S2D), using the Ca coordinates of the half-heptad shift in our construct to guide the final position of the stalk during the simulations. This final model (Fig. 2, B and D, and movie S3), which we refer to as the high-affinity MTBD, has the highest cross-correlation with the experimental map (figs. S2 and S3 and movies S4 and S5).

We repeated the MDFF calculations, using the MTBD from the recent crystal structure of an ADP-bound dynein monomer (20) (fig. S4). The only difference in the resulting pseudo-atomic model is in the stalk, where the dynein monomer’s structure is missing density for one of the helices next to the MTBD (fig. S4, A and B). The similarity in the crystal structures of the MTBD in the low-affinity and ADP-bound states is likely due to the absence of MTs; our results suggest that the conformation we observe in our MT-bound high-affinity structure is stabilized by its interactions with β-tubulin (fig. S5A and tables S1 and S2). MDFF of the ADP-bound dynein monomer’s MTBD into our cryo-EM map results in a large change in the angle between the MTBD and the stalk in the dynein monomer structure (fig. S4, A and B). This change makes the docking of the dynein monomer into our map compatible with previously reported measurements of the MT-stalk angle (17, 21) and our two-dimensional analysis of images of monomer-decorated MTs (fig. S4, D and E).

The cryo-EM map shows three points of continuous density between the MT and dynein’s MTBD: the H1-H2 loop and helices H3 and H6 (Fig. 2B). Several parts of the structure are unchanged by its interaction with the MT, especially helices H6, H5, and CC2, with root mean square deviations (RMSDs) between those of the low- and high-affinity models of 1.4, 1.4, and 1.8 Å,

Fig. 1. Cryo-EM reconstruction of the cytoplasmic dynein high-affinity MTBD bound to a MT. (A) Schematic of dimeric cytoplasmic dynein. Major features relevant to this study are indicated. The MTBD is depicted in its low- (light blue) and high- (dark blue) affinity states during a step along the MT. (B) Schematic of the fusion constructs between the MTBD and seryl-tRNA synthetase (SRS) that fix the heptad registry of the stalk. The low-affinity construct has an additional four amino acids (yellow) inserted in CC1 (black) relative to the high-affinity construct. (C) Cryo-EM image of MTs highly decorated with the high-affinity SRS-MTBD construct (top) and a class average generated from segments of decorated 14-protofilament MTs (bottom). Scale bars: 50 nm (top) and 25 nm (bottom). (D) Three-dimensional reconstruction of the MTBD-MT complex, filtered to the calculated resolution of 9.7 Å (fig. S1). The black solid line represents a slice through the volume, which is shown on the right viewed from the minus end of the MT. The MT polarity is indicated and the dashed line shows the location of the MT seam. The SRS has been omitted due to its lower resolution (fig. S1).

RESEARCH

respectively. The largest changes are the repositioning of H1 (RMSD = 10.1 Å) and an opening of CC1 in the coiled-coil next to the MTBD (RMSD = 8.1 Å) (Fig. 2, A to D, and movies S1, S2, S3, and S6), a movement anchored at the proline kink present in CC1. The final position of H1 is stabilized by multiple interactions with an acidic patch in H12 of β-tubulin not fully occupied in the low-affinity state (fig. S5A). This patch also stabilizes the high-affinity state of kinesin (29).

We monitored hydrogen bonds and salt bridges formed between the MTBD and the MT during MD simulations (tables S1 and S2); the high-affinity MTBD formed more hydrogen bonds with the MT (fig. S6) and electrostatic interactions at H1, H3, and H6 (fig. S5 and table S1). Nearly all of these residues are highly conserved, and mutating them results in defects in MT binding (18, 30) (fig. S5). The importance of salt bridges to the MTBD-MT interaction is consistent with the sensitivity of dynein's motility to ionic strength (fig. S7).

Our structural analysis suggested that the MTBD contains residues that lower its own affinity for the MT. In the MD simulations, basic residues in H1 and H6 formed salt bridges that alternated between intramolecular and intermolecular partners. In the MT-bound high-affinity conformation, H1-K3298 switched between a glutamate on β-tubulin and a conserved glutamate in CC1 (E3289) (Fig. 3A and fig. S8A); neither contact can be formed by H1-K3298 in the low-affinity conformation (Fig. S5A). H6-R3382 switched from an intramolecular interaction with a conserved glutamate in the same helix (H6-E3378) in the low-affinity unbound state to an intermolecular interaction with a cluster of glutamates on α-tubulin upon binding (Fig. 3B and fig. S8B); the intramolecular interaction might weaken the MTBD-MT interaction in both the low- and high-affinity conformations. The importance of the two MTBD glutamates involved in the intramolecular salt bridges had previously been recognized; substitution of CC1-E3289 and H6-E3378 with alanine increased dynein's MT-binding affinity (18, 30) and reduced its ATP-stimulated release from MTs, respectively (30). We hypothesized that these phenotypes resulted from the competition between MT and MTBD residues in CC1 and H6 for salt-bridge formation with the basic residues K3298 and R3382 in the MTBD.

To test this prediction, we mutated the residues equivalent to E3289 and E3378 in CC1 and H6 of *Saccharomyces cerevisiae* dynein to either an isosteric but neutral (Q) or a basic amino acid (K) to disrupt the salt bridge (Q) or introduce an intramolecular charge repulsion (K) that may favor intermolecular interactions between H1-K3298 or H6-R3382 and acidic residues on the MT surface. Single-molecule motility assays that monitored the movement of purified mutant dyneins showed significant increases in dynein's run length and small decreases in velocity that paralleled the severity of the mutation (E → Q → K) (Fig. 4, A and B, and fig. S9). Most notably, the basic substitutions CC1-E3289K and H6-E3378K increased dynein's run length by fivefold and sixfold, respectively (Fig. 4B), and the double mutant increased the run length even further (fig. S10). These results suggest that cytoplasmic dynein has been selected for submaximal processivity. The effects observed with the mutants are not due to a strengthened interaction with the unstructured C-terminal tails of tubulin (E-hooks). Although their removal decreased the run length of all constructs tested, in agreement with previous studies (31), the trend of increasing processivity (wild type → E3289K → E3378K) remained (fig. S11).

These findings provide a molecular model for how dynein couples its affinity for MTs with the nucleotide state of the motor domain (Fig. 4, C to E, and movie S6). We describe the transition from low to high affinity, but suggest that the proposed changes are reversible. During a diffusive search for its next binding site (Fig. 4C), an unbound MTBD is in the low-affinity conformation with its stalk in the β+ registry, H1 oriented perpendicular to the MT axis and intramolecular salt bridges at key MT-binding residues. Consistent with this, a nuclear magnetic resonance study found that an unconstrained, minimal MTBD in solution exists in the β+ registry and displays low affinity for MTs (32). Upon binding (Fig. 4D), transition to a high-affinity conformation occurs.

![Figure 2](#fig-2)

**Fig. 2.** The high-affinity, MT-bound state of the dynein MTBD is characterized by repositioning of helices H1 and CC1. (A) Rigid-body docking of the low-affinity MTBD structure into our cryo-EM density. (B) Pseudo-atomic model of the high-affinity MTBD bound to MTs generated by MDFF and TMD (see text for details). (C) Enlargement of the structure shown in (A), with its orientation indicated in (A). (D) Enlargement of the structure shown in (B), with its orientation indicated in (B). The cryo-EM map is shown as a transparent gray mesh. The MTBD is colored according to the scheme shown at the bottom of the figure, and structural elements are indicated in the different views. H1 (orange/red) is the element with the largest movement in the transition to the high-affinity conformation; H3 and H6 (dark blue) are major contact points with the MT (green). α- and β-tubulin are indicated (green). MT polarity is indicated in (A) and (B). H1 protrudes from the cryo-EM map and clashes with the MT in the rigid-body docked low-affinity state (A and C).

Fig. 3. Behavior of dynamic salt bridges in the MTBD as determined by MD. (A) K3298 in H1 of the MTBD alternates between an intermolecular salt bridge with E420 on β-tubulin (MTBD-MT) and an intramolecular salt bridge with E3289 on CC1 of the MTBD (intra-MTBD). (B) R3382 in H6 of the MTBD alternates between an intermolecular salt bridge with E414 and E420 on α-tubulin (MTBD-MT) and an intramolecular salt bridge with E3378 in H6 (intra-MTBD). Single-letter amino acid code and number are indicated for Bos taurus tubulin and Mus musculus cytoplasmic dynein (E, Glu; K, Lys; R, Arg). Time stamps for frames from MD simulations are indicated. “Intermediate” refers to a position midway between MTBD-MT and intra-MTBD salt bridges.

---

Fig. 4. Dynamic salt bridges reduce dynein motility. Bar graphs of (A) mean velocities and (B) characteristic run lengths of fluorescently labeled S. cerevisiae dynein bearing the equivalent of the indicated M. musculus mutations moving on MTs. Error bars: SD (A) and SE (B). Velocity and run length differences between wild type (WT) and Q (Gln) mutants, as well as between Q and K (Lys) mutations at the same position, are statistically significant (*t* test, *P* < 0.01 for velocity, and two-tailed Kolmogorov-Smirnov test, *P* < 0.01 for run length). The data for the double mutant (E→K at both CC1 and H6) was omitted because run lengths could only be determined under more stringent motility conditions (fig. S10). (C to E) Molecular model for the coordination of nucleotide state and MT binding by dynein (see text for details). (C) Unbound dynein in the low-affinity conformation; H1 is colored orange. (D) Initial interaction with a new binding site. (E) Repositioning of H1 (now in red) leads to the formation of new ionic interactions with β-tubulin (green cylinder) that stabilize the high-affinity state of the MTBD. The repositioning of H1 is accompanied by a movement in CC1; both movements are indicated by solid black arrows. The conformational change in the MTBD biases the registry of the coiled-coil toward the high-affinity α state, a change that can propagate to the motor domain (white/gray arrow). Ionic interactions are indicated with dashed lines. The identities of the helices in the MTBD are indicated by the key.

tion involves a large displacement of H1, stabilized by new salt bridges with β-tubulin, and an opening of CC1 at the base of the stalk (Fig. 4E). The movements of H1 and CC1 likely constrain the registries that can be explored by the stalk, biasing the distribution toward the high-affinity α registry (Fig. 4E). Propagation of this signal to the head would elicit conformational changes that produce a movement of the linker domain and a displacement of dynein toward the MT minus end.

Our analysis of dynamic salt bridges reveals that cytoplasmic dynein has been selected for submaximal processivity. Whereas kinesin has diversified its functional repertoire through gene duplication and divergence (33), cytoplasmic dynein is expressed from a single locus and may have evolved suboptimal processivity to increase the dynamic range of its regulation. High processivity could also be detrimental when multiple dyneins and kinesins must balance their actions on a single cargo (34). Consistent with this idea, intraflagellar dyneins, responsible for long, unidirectional transport within cilia (35, 36), contain neutral or basic residues at the equivalent of H6-E3378 (fig. S12), which would likely increase their processivity.

**References and Notes**

1. I. R. Gibbons, *Cell Motil. Cytoskeleton* **32**, 136 (1995).
2. P. Höök, R. B. Vallee, *J. Cell Sci.* **119**, 4369 (2006).
3. R. D. Vale, *Cell* **112**, 467 (2003).
4. R. B. Vallee, G. E. Seale, J.-W. Tsai, *Trends Cell Biol.* **19**, 347 (2009).
5. R. B. Vallee, J. C. Williams, D. Varma, L. E. Barnhart, *J. Neurobiol.* **58**, 189 (2004).
6. S. A. Burgess, M. L. Walker, H. Sakakibara, P. J. Knight, K. Oiwa, *Nature* **421**, 715 (2003).
7. A. J. Roberts *et al.*, *Cell* **136**, 485 (2009).
8. I. R. Gibbons, B. H. Gibbons, G. Mocz, D. J. Asai, *Nature* **352**, 640 (1991).
9. T. Kon, M. Nishiura, R. Ohkura, Y. Y. Toyoshima, K. Sutoh, *Biochemistry* **43**, 11266 (2004).
10. S. L. Reck-Peterson, R. D. Vale, *Proc. Natl. Acad. Sci. U.S.A.* **101**, 1491 (2004).
11. S. L. Reck-Peterson *et al.*, *Cell* **126**, 335 (2006).
12. T. Shima, T. Kon, K. Imamura, R. Ohkura, K. Sutoh, *Proc. Natl. Acad. Sci. U.S.A.* **103**, 17736 (2006).
13. A. P. Carter, C. Cho, L. Jin, R. D. Vale, *Science* **331**, 1159 (2011).
14. T. Kon, K. Sutoh, G. Kurisu, *Nat. Struct. Mol. Biol.* **18**, 638 (2011).
15. M. P. Koonce, *J. Biol. Chem.* **272**, 19714 (1997).
16. M. A. Gee, J. E. Heuser, R. B. Vallee, *Nature* **390**, 636 (1997).
17. A. P. Carter *et al.*, *Science* **322**, 1691 (2008).
18. I. R. Gibbons *et al.*, *J. Biol. Chem.* **280**, 23960 (2005).
19. T. Kon *et al.*, *Nat. Struct. Mol. Biol.* **16**, 325 (2009).
20. T. Kon *et al.*, *Nature* **484**, 345 (2012).
21. N. Mizuno *et al.*, *EMBO J.* **23**, 2459 (2004).
22. C. V. Sindelar, K. H. Downing, *J. Cell Biol.* **177**, 377 (2007).
23. C. V. Sindelar, K. H. Downing, *Proc. Natl. Acad. Sci. U.S.A.* **107**, 4111 (2010).
24. D. B. Wells, A. Aksimentiev, *Biophys. J.* **99**, 629 (2010).
25. J. Löwe, H. Li, K. H. Downing, E. Nogales, *J. Mol. Biol.* **313**, 1045 (2001).
26. L. G. Trabuco, E. Villa, K. Mitra, J. Frank, K. Schulten, *Structure* **16**, 673 (2008).
27. L. G. Trabuco, E. Villa, E. Schreiner, C. B. Harrison, K. Schulten, *Methods* **49**, 174 (2009).
28. J. Schlitter, M. Engels, P. Krüger, *J. Mol. Graph.* **12**, 84 (1994).
29. S. Uchimura, Y. Oguchi, Y. Hachikubo, S. Ishiwata, E. Muto, *EMBO J.* **29**, 1167 (2010).
30. M. P. Koonce, I. Tikhonenko, *Mol. Biol. Cell* **11**, 523 (2000).
31. Z. Wang, M. P. Sheetz, *Biophys. J.* **78**, 1955 (2000).
32. L. McNaughton, I. Tikhonenko, N. K. Banavali, D. M. LeMaster, M. P. Koonce, *J. Biol. Chem.* **285**, 15994 (2010).
33. E. M. Dagenbach, S. A. Endow, *J. Cell Sci.* **117**, 3 (2004).
34. M. A. Welte, *Curr. Biol.* **14**, R525 (2004).
35. C. Iomini, V. Babaev-Khaimov, M. Sassaroli, G. Piperno, *J. Cell Biol.* **153**, 13 (2001).
36. J. A. Laib, J. A. Marin, R. A. Bloodgood, W. H. Guilford, *Proc. Natl. Acad. Sci. U.S.A.* **106**, 3190 (2009).

**Acknowledgments:** We thank A. Carter (Laboratory of Molecular Biology-Medical Research Council) for reagents and advice, C. Sindelar (Yale), V. Ramey [University of California (UC)-Berkeley], E. Egelman (University of Virginia), and R. Sinkovits (UC-San Diego) for sharing processing scripts and helpful advice; M. Sotomayor (Harvard) and R. Gaudet (Harvard) for advice concerning MD; J. Hogle (Harvard), M. Strauss (Harvard), and M. Wolf (Harvard) for help with film and the use of a film scanner; and E. Nogales (UC-Berkeley), N. Francis (Harvard), and D. Pellman (Harvard) for critically reading the manuscript, as well as all the members of the Leschziner and Reck-Peterson Labs for advice and helpful discussions. EM data were collected at the Center for Nanoscale Systems (CNS), a member of the National Nanotechnology Infrastructure Network (NNIN), which is supported by the National Science Foundation under NSF award no. ECS-0335765. CNS is part of Harvard University. MD simulations were run on the Odyssey cluster supported by the Faculty of Arts and Sciences Science Division Research Computing Group, Harvard University. S.L.R.-P. is funded by the Rita Allen Foundation, the Harvard Armenise Foundation, and an NIH New Innovator award (1 DP2 OD004268-1). A.E.L. was funded in part by a Research Fellowship from the Alfred P. Sloan Foundation. R.H.-L. was supported in part by Consejo Nacional de Ciencia y Tecnología and Fundación México en Harvard. The cryo-EM map was deposited at the EM Data Bank (EMDB-5439) and pseudo-atomic models at the Protein Data Bank (PDB-3]1T and -3]1U).

**Supplementary Materials**

www.sciencemag.org/cgi/content/full/337/6101/1532/DC1

Materials and Methods  
Figs. S1 to S12  
Tables S1 to S3  
References (37–53)  
Movies S1 to S6  

2 May 2012; accepted 20 July 2012  
10.1126/science.1224151

---

**Specifying and Sustaining Pigmentation Patterns in Domestic and Wild Cats**

Christopher B. Kaelin,$^{1,2*}$ Xiao Xu,$^{3,4*}$ Lewis Z. Hong,$^{2}$ Victor A. David,$^{3}$ Kelly A. McGowan,$^{2}$ Anne Schmidt-Küntzel,$^{3,5}$ Melody E. Roelke,$^{3,6}$ Javier Pino,$^{7}$ Joan Pontius,$^{3,6}$ Gregory M. Cooper,$^{1}$ Hermogenes Manuel,$^{2}$ William F. Swanson,$^{8}$ Laurie Marker,$^{5}$ Cindy K. Harper,$^{9}$ Ann van Dyk,$^{10}$ Bisong Yue,$^{4}$ James C. Mullikin,$^{11}$ Wesley C. Warren,$^{12}$ Eduardo Eizirik,$^{13,14}$ Lidia Kos,$^{7}$ Stephen J. O'Brien,$^{3}$$^{\dagger\dagger}$ Gregory S. Barsh,$^{1,2}$$^\dagger$ Marilyn Menotti-Raymond$^{3}$

Color markings among felid species display both a remarkable diversity and a common underlying periodicity. A similar range of patterns in domestic cats suggests a conserved mechanism whose appearance can be altered by selection. We identified the gene responsible for tabby pattern variation in domestic cats as *Transmembrane aminopeptidase Q* (*Taqpep*), which encodes a membrane-bound metalloprotease. Analyzing 31 other felid species, we identified *Taqpep* as the cause of the rare king cheetah phenotype, in which spots coalesce into blotches and stripes. Histologic, genomic expression, and transgenic mouse studies indicate that paracrine expression of *Endothelin3* (*Edn3*) coordinates localized color differences. We propose a two-stage model in which *Taqpep* helps to establish a periodic pre-pattern during skin development that is later implemented by differential expression of *Edn3*.

T he molecular basis and evolutionary variation of periodic mammalian color patterns have been difficult to investigate from genetic crosses of model organisms. Domestic cats (*Felis catus*) exhibit heritable variation of tabby markings—mackerel versus blotched—that provide an opportunity for such genetic analysis (1). Tabby markings are a composite of two features: (i) a light background component in which individual hairs have extensive light bands, and (ii) a superimposed darker component in which hairs have little or no banding. In mackerel cats, the dark component is organized into narrow vertical stripes with a constant and regular spacing, whereas in blotched cats, the dark component is expanded into a less organized structure with wide whorls (Fig. 1A). Periodic color patterns in other felids may represent the same process; for example, dark tabby markings in domestic cats may be homologous to black stripes or spots in tigers or cheetahs, respectively (2).

A logical explanation for tabby patterning involves the Agouti-melanocortin receptor system, in which Agouti protein, a paracrine signaling molecule released from dermal papillae, acts on overlying hair follicle melanocytes to inhibit the melanocortin 1 receptor (Mc1r), causing a switch from the production of black/brown eumelanin to red/yellow pheomelanin (3, 4). According to this hypothesis, dark tabby stripes are areas in which Agouti signaling is suppressed or surmounted during hair growth and regeneration. However, known components of the Agouti-melanocortin
