
CHAPTER FIFTEEN

# 4-Methylideneimidazole-5-One-Containing Aminomutases in Enediyne Biosynthesis

**Jeremy R. Lohman**, **Ben Shen**

* *Department of Chemistry, The Scripps Research Institute, Jupiter, Florida, USA*
* † *Department of Molecular Therapeutics, The Scripps Research Institute, Jupiter, Florida, USA*
* ‡ *Natural Products Library Initiative at TSRI, The Scripps Research Institute, Jupiter, Florida, USA*
* 1 *Corresponding author: e-mail address: shenb@scripps.edu*

## Contents

1. Introduction                                                                                       300
2. Methods                                                                                            310
   2.1 Overproduction and purification of SgcC4 from E. coli                                          310
   2.2 Overproduction and purification of MdpC4 from S. lividans TK-64                                311
   2.3 In vitro assay of MIO-containing aminomutases SgcC4 and MdpC4                                  312
   2.4 Determination of the MIO-prosthetic group in the recombinant SgcC4 protein                      314
   2.5 Crystallization of SgcC4 with substrates or inhibitors to determine catalytic mechanism         315
3. Conclusion                                                                                         315
Acknowledgment                                                                                       317
References                                                                                           317

## Abstract

Many natural products contain unusual aromatic β-amino acids or moieties derived therefrom. The biosynthesis of these β-amino acids was first elucidated during a biosynthetic study of the enediyne antitumor antibiotic C-1027, when an enzyme, SgcC4, was discovered to convert L-tyrosine to (S)-β-tyrosine. SgcC4 is similar in sequence and structure to 4-methylideneimidazole-5-one (MIO)-containing ammonia lyases. Whereas the ammonia lyases use the electrophilic power of the MIO group to catalyze the release of ammonia from aromatic amino acids to generate α,β-unsaturated carboxylic acids as final products, SgcC4 retains the α,β-unsaturated carboxylic acid and amine as intermediates and reappends the amino group to the β-carbon, affording a β-amino acid as the final product. The study of SgcC4 led to the subsequent discovery of other MIO-containing aminomutases with altered substrate specificity and product stereochemistry, including MdpC4 from the biosynthetic pathway of the enediyne antitumor antibiotic maduropeptin. This chapter describes protocols for the enzymatic and structural characterization of these MIO-containing aminomutases as exemplified by SgcC4 and MdpC4. These protocols are applicable to the study of other aminomutases.

---

Methods in Enzymology, Volume 516  
ISSN 0076-6879  
http://dx.doi.org/10.1016/B978-0-12-394291-3.00007-1  
© 2012 Elsevier Inc. All rights reserved.

1. INTRODUCTION

Aminomutases catalyze the reversible exchange of an amine and hydrogen between two vicinal carbons, typically of L-amino acids, to generate β-amino acids. Enzymes that carry out the aminomutase reaction are divided into two groups based on their catalytic mechanisms. The first group use either SAM and a [4Fe–4S] iron–sulfur cluster or adenosylcobalamin to generate a radical capable of abstracting a hydrogen from the β-carbon of an amino acid, and subsequent amine migration is facilitated by pyridoxal phosphate; these enzymes have been extensively reviewed (Frey, 2001; Vey & Drennan, 2011). The other group of aminomutases use an electrophilic 4-methylideneimidazole-5-one (MIO) prosthetic group to catalyze the rearrangement (Cooke, Christianson, & Bruner, 2009; Turner, 2011), and they are the focus of this chapter. The β-amino acid products of MIO-containing aminomutases are mainly found in the secondary metabolites of bacteria, fungi, and plants, which include natural products such as enediynes, nonribosomal peptides, and terpenoids (Fig. 15.1).

The MIO cofactor is spontaneously generated posttranslationally from a three-amino acid motif, Xaa-Ala-Ser-Gly-Xaa, in which the amine of the glycine attacks the carbonyl of the alanine, generating a five-membered ring, which subsequently undergoes two dehydrations to generate this potent electrophile (Fig. 15.2A) (Schwede, Rétey, & Schulz, 1999). The best understood MIO-containing enzymes are histidine ammonia lyase (HAL), phenylalanine ammonia lyase (PAL), and tyrosine ammonia lyase (TAL), which remove the amino group from L-histidine, L-phenylalanine, and L-tyrosine to generate α,β-unsaturated carboxylic acids (E)-urocanic acid, (E)-cinnamic acid, and (E)-4-hydroxycinnamic acid (HCA), respectively (Langer, Langer, & Rétey, 2001; Poppe, 2001; Poppe & Rétey, 2005). There are two proposed mechanisms for ammonia lyase activity, an “E1cB” mechanism and a Friedel–Crafts-type reaction-based mechanism (Poppe & Rétey, 2005). The “E1cB” mechanism starts with attack of the amino group onto the MIO cofactor, which enhances its leaving ability. Subsequent deprotonation of the β-carbon results in elimination of the MIO-bound amine and product formation. The MIO-bound amine is then protonated and released to solvent as free ammonia, regenerating the functional MIO-prosthetic group (Fig. 15.2B, path a). MIO-containing ammonia lyases and aminomutases likely share a catalytic mechanism for the difficult amine removal step, but rather than release the MIO-bound amine to

MIO-Containing Aminomutases

C-1027  
Kedarcidin  
Maduropeptin  

Myxovalargin  
Chondramide C  
Taxol  

Edeine A  
Andrimid  

Figure 15.1 Examples of natural products containing β-amino acids or derived moieties shaded in gray.

solvent as in ammonia lyases, aminomutases catalyze a Michael addition that appends the amine to the α,β-unsaturated carboxylic acid intermediate, affording a β-amino acid as the final product (Fig. 15.2B, path b).

The first reported MIO-containing aminomutase, the SgcC4 tyrosine aminomutase (TAM), arose from biosynthetic analysis of the antitumor antibiotic C-1027 from *Streptomyces globisporus*, which contains an (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety thought to originate from tyrosine (Fig. 15.3A) (Liu, Christenson, Standage, & Shen, 2002). Sequencing of the C-1027 gene cluster did not reveal genes with similarity to known radical-based aminomutases. Rather, there was a putative enzyme, SgcC4, with significant homology (~35–40/50–55% identity/similarity) to the HALs and PALs, possessing the signature Ala-Ser-Gly motif for MIO formation. Deletion of SgcC4 from the pathway abolished C-1027 production, confirming its biosynthetic necessity (Christenson, Liu, Toney, & Shen, 2003). The activity of SgcC4 as a TAM was confirmed *in vitro* to convert L-tyrosine to (S)-β-tyrosine, establishing its role in the biosynthesis of the β-amino acid moiety of C-1027 (Fig. 15.3A) (Christenson, Liu, et al., 2003).

A

![image](https://i.imgur.com/1234567.png)

Ala     Ser     Gly

MIO

B

![image](https://i.imgur.com/8901234.png)

MIO     L-Tyr

Path a

HCA

MIO

Path b

HCA

MIO

(S)-β-Tyr

C

E + α-L-Tyr ⇌ E − α-L-Tyr ⇌ E − HCA ⇌ E − β-Tyr ⇌ E + β-Tyr

Rate limiting

Fast

Slow

E − HCA

Figure 15.2 Proposed mechanisms of MIO-containing ammonia lyases and aminomutases: (A) posttranslational formation of the MIO-prosthetic group from the conserved Ala-Ser-Gly motif, (B) MIO-mediated “E1cB” mechanism for ammonia lyases (path a) and aminomutases (path b), and (C) a kinetic model for a MIO-containing aminomutase catalysis, based on the mechanism of SgcC4, with the thermodynamically favored HCA and free amine products accumulating owing to off-path release from the active site.

A

![image](https://i.imgur.com/5678901.png)

L-Tyr

SgcC2

SgcC

Enediyne core

Benzoazoline

Deoxy aminosugar

B

![image](https://i.imgur.com/2345678.png)

L-Tyr

MdpC2

MdpC7

MdpC

MdpC6

MdpC3

MdpC8

Enediyne core

Aminosugar

Dimethylsalicylate

Figure 15.3 Proposed biosynthetic pathways for the β-amino acid-derived moieties in the enediynes (A) C-1027 and (B) maduropeptin, featuring the MIO-containing aminomutases SgcC4 and MdpC4. SgcC and MdpC, flavin-dependent hydroxylases; SgcC1 and MdpC1, adenylation enzymes; SgcC2 and MdpC2, peptidyl-carrier-proteins; SgcC3 and MdpC3, flavin-dependent halogenases; SgcC5 and MdpC5, condensation enzymes; MdpC6, methyltransferase; MdpC7, aminotransferase; MdpC8, ketoreductase.
Various techniques were used to further investigate the detailed biochemical properties of SgcC4 (Christenson, Liu, et al., 2003; Christenson, Wu, Spies, Shen, & Toney, 2003). A MIO-deletion mutant, SgcC4 (Ser153Ala), was generated by changing the Ala-Ser-Gly motif to Ala-Ala-Gly, which was incapable of MIO formation, hence catalysis. A difference spectrum of the SgcC4 wild-type and (Ser153Ala) mutant proteins displayed a characteristic peak at 313 nm, proving direct evidence for the presence of the MIO cofactor (Christenson, Wu, et al., 2003; Röther, Merkel, & Rétey, 2000). Further, the MIO cofactor is inactivated by borohydride and cyanide, and this inactivation was protected by the presence of HCA or L-tyrosine but not L-phenylalanine. Substrate specificity and enzyme reaction mechanism were determined using kinetic analysis, and a full time course for the SgcC4-catalyzed interconversion among L-tyrosine, β-tyrosine, and HCA was measured and analyzed to provide estimates for the rate constants in a minimal mechanism (Fig. 15.2C). Table 15.1 summarizes kinetic values of SgcC4 and their comparison with other ammonia lyases and aminomutases known to date. Activity was weak or not detectable with L-phenylalanine (as suggested by KCN inactivation), L-3,4-dihydroxyphenylalanine, or L-3-chlorotyrosine, which were other possible intermediates for biosynthesis of the β-amino acid moiety (Fig. 15.3A), establishing SgcC4 as the link between primary metabolism and C-1027 biosynthesis. Intriguingly, SgcC4 can act as an aminotransferase, as coincubation with L-3-chlorotyrosine and HCA produced β-tyrosine, indicating that the amine remains bound for an extended period if the intermediate leaves the active site. SgcC4 produced (S)-β-tyrosine as the main product during short incubation times, matching the expected stereochemistry for C-1027, but upon extended incubation, HCA and (R)-β-tyrosine are produced, making SgcC4 a weak TAL and β-tyrosine racemase (Christenson, Liu, et al., 2003; Christenson, Wu, et al., 2003).

An X-ray crystal structure of SgcC4 was solved (PDB 2OHY), unambiguously demonstrating the presence of the MIO cofactor (Christianson, Montavon, Van Lanen, Shen, & Bruner, 2007). It revealed that the structures of MIO-containing aminomutases and ammonia lyases are almost identical, lending support to the idea that their mechanisms are similar (Fig. 15.2B). A key structural difference was postulated to account for aminomutase rather than ammonia lyase chemistry. Specifically, an extended surface loop on SgcC4 carries Tyr303, which hydrogen bonds to Glu71, and together they act as a gate to the active site, sequestering intermediates away from solvent, whereas for the ammonia lyases the corresponding loop allows access to
Table 15.1 Kinetic data for SgcC4 and other characterized MIO-containing aminomutases and ammonia lyases

| Enzyme$^a$ | Substrate | $K_m$ (mM) | $k_{cat}$ ($s^{-1}$) | $k_{cat}$ ($s^{-1}$) | References |
|------------|-----------|------------|----------------------|----------------------|------------|
|            |           |            | (for $\beta$-aa)     | (for $\alpha$, $\beta$-unsat. acid) |          |
| SgcC4      | Tyr       | 0.028      | 0.01                 | 0.0012               | Christenson, Wu, et al. (2003) |
|            | 3-Cl-Tyr  | 1.54       | 0.0067               |                      |            |
|            | 3-HO-Tyr  | 6.07       | 0.0048               |                      |            |
| MdpC4      | Tyr       |            |                      |                      | Van Lanen, Oh, Liu, Wendt-Pienkowski, and Shen (2007) |
| CmTAL      | Tyr       | 0.384      | 0.0009               | 0.011                | Krug and Müller (2009) |
| CmdF       | Tyr       | 0.377      | 0.0072               | 0.0005               | Krug and Müller (2009) |
| MxTAM      | Tyr       |            |                      |                      | Krug and Müller (2009) |
| MfTAM      | Tyr       |            |                      |                      | Krug and Müller (2009) |
| PaPAM      | Phe       | 0.01       | 0.061                |                      | Ratnayake, Wanninayake, Geiger, and Walker (2011) |
| SmPAL      | Phe       | 0.023      |                      | 4.79                 | Xiang and Moore (2005) |
| SeTAL      | Tyr       | 0.015      |                      | 0.015                | Berner et al. (2006) |
|            | Phe       | 2.86       |                      | 0.0038               |            |
| SsTAL      | Tyr       |            |                      |                      | Zhu, Liao, Ye, and Zhang (2012) |

RsTAL    Tyr   0.074   4.32    Louie et al. (2006)
        Phe   1.277   0.058   
SgHAL   His   0.6     Not reported   Wu, Kroening, White, and Kendrick (1992)
MxHAL   His           Krug and Müller (2009)
PpHAL   His   5.8     0.020   Hernandez and Phillips (1993)
NpPAL   Phe   0.045   1.96    Moffitt et al. (2007)
AvPAL   Phe   0.06    4.3     Moffitt et al. (2007)
RtPAL   Phe           Calabrese, Jordan, Boodhoo, Sariaslani, and Vannelli (2004)
PcPAL   Phe   0.017   22      Appert, Logemann, Hahlbrock, Schmid, and Amrhein (1994)
        Tyr   2.5     0.3     
TcPAM   Tyr   0.057   0.053   0.012   Feng, Wanninayake, Strom, Geiger, and Walker (2011)

*See Fig. 15.5 legend for accession numbers for each of the enzymes.
the active site (Fig. 15.4A and B). Further cocrystallization studies with the mechanism-based inhibitors $\alpha,\alpha$-difluoro-$\beta$-tyrosine (PDB 2QVE), $\alpha$, $\alpha$-difluoro-$\beta$-4-methoxyphenylalanine (PDB 2RJS), and $\alpha/\beta$-epoxy-HCA (PDB 2RJR) revealed substrate binding contacts and important catalytic residues, including the conserved catalytic base Tyr63 (Fig. 15.4C)

![Image Description]

Figure 15.4 Structural characterization of SgcC4: (A) Overview of SgcC4 and RsTAL active sites with the extended alpha-helix closing off the modeled tyrosine substrate in SgcC4 but not in RsTAL; (B) surface representation of the active sites with the SgcC4 active site closing off from solvent, whereas that of RsTAL exposed; and (C) active-site residues contacting the substrate. All panels are color-coded to regions in the sequence alignment (Fig. 15.5, i.e., blue, L1; cyan, L2; green, L3; yellow, L4; orange, L5; red, L6; magenta, L7), and the substrate (tyrosine for SgcC4 and HCA for RsTAL) is colored gray. PDB 3KDZ used for SgcC4 with the mutated Y63F remodeled as tyrosine. PDB 2O7B used for RsTAL with HCA bound in the active site.(Christianson, Montavon, Festin, et al., 2007; Montavon, Christianson, Festin, Shen, & Bruner, 2008). Subsequently, Tyr63 was mutated to phenylalanine, which abolished activity. The SgcC4 (Tyr63Phe) mutant was cocrystallized with tyrosine (PDB 3KDZ), which demonstrated MIO binding to the amine (Cooke & Bruner, 2010). Taken together, the structural studies revealed that SgcC4 uses an “E1cB” mechanism (Fig. 15.2B).

MdpC4, a homolog of SgcC4, was subsequently found in the biosynthetic gene cluster for maduropeptin, an enediyne antitumor antibiotic produced by *Actinomadura madurae* (Van Lanen, Oh, Liu, Wendt-Pienkowski, & Shen, 2007). Expression of *mdpC4* in *Escherichia coli* was attempted but failed. Subsequently, MdpC4 was successfully overproduced in *Streptomyces lividans* as a soluble, functional protein. *In vitro* characterization of MdpC4 confirmed its role as a TAM, revealing the cryptic intermediary of (*S*)-β-tyrosine in maduropeptin biosynthesis (Fig. 15.3B). Characterization of these two secondary metabolite-associated aminomutases led to the discovery of other members of this class. Figure 15.5 summarizes sequence comparisons between selected MIO-containing ammonia lyases and aminomutases, highlighting key regions and residues differentiating these two subclasses of enzymes. Homologs of SgcC4 were found in various myxobacteria, including CmdF from the chondramide biosynthetic pathway in *Chondromyces crocatus* (Rachid, Krug, Weissman, & Müller, 2007), and two from the myxoalargin pathways in *Myxococcus fulvus* (MfTAM) and *Myxococcus sp.* Mx-BO (MxTAM) (Krug & Müller, 2009). *In vitro* characterization of CmdF confirmed it as a TAM and revealed (*R*)-β-tyrosine as the preferred product, while *in vivo* characterization supported the functional assignments of MfTAM and MxTAM as TAMs, producing (*S*)-β-tyrosine. The homolog from *Cupriavidus metallidurans* (CmTAL) predominantly displayed TAL activity, but possessed weak TAM activity that produced (*R*)-β-tyrosine (Krug & Müller, 2009). Two other characterized aminomutases are the phenylalanine aminomutases from the plant *Taxus canadensis* (taxol producer), TcPAM (Feng et al., 2011; Walker, Klettke, Akiyama, & Croteau, 2004), and the Gram-negative bacterium *Pantoea agglomerans* (andrimid producer), PaPAM (Jin, Fischbach, & Clardy, 2006; Ratnayake et al., 2011), which produce (*R*)-β-phenylalanine and (*S*)-β-phenylalanine, respectively.

Bioinformatic analyses of the aminomutases are useful for discerning substrate specificity, as the residues binding to the phenol end of tyrosine are polar (Fig. 15.4C), whereas those binding to phenylalanine are hydrophobic (to our knowledge, there are no reports of histidine aminomutases or natural
L1
62 96
SgcC4-S IYGVT TGY GEMIYMQ VDKSKEVELQT NIVRSHSA
MdpC4-S VYGVT TGY GEMIYMLV DRANEVELQT NIVRSHSA G
CmTAL-R IYGVS TGF GELVH NWVDIEHG RALQENLLR SHCA G
CmdF-R IYGVN TGF GELVP VMIPRQHK RELQENLIR SHAAG
MxTAM-S MYGVN TGF GELIN VIIPPQFK SDLQHNLLR SSHAAG
MfTAM-S MYGVN TGF GELIN VIIPPQFK SDLQHNLLR SSHAAG
PaPAM-S IYGVT SMG GFV NYIVPIAKA SELQNLLINA VATN
SmPAL IYGVN ISMG GFVDHL VPVSQARL QENLINAVAT N
SsTAL IYGVT TGF GDSVNRQ ISP EKTARLQ ELIRYHLNG
SeTAL IYGLT QGF GPLVL FDADSELE --QGGSLIISH LGTG
RsTAL VYGLT TGF GPLAN RLISGENVR TLQANLVHH LASG
SgHAL VYGVS TGF GALAS RHIGTELRA QLRNIVRSHA AG
MxHAL AYGINT GFG TLAEE VRIDKKDL RDILQRNILLS HACG
PpHAL AYGINT GFG LLAST RIAHDLEN LQRSILVSHA AG
NpPAL IYGVT SGFG GMADV VISREQAA ELQTNIWFLK SG
AvPAL IYGVT SGFG MANVA ISREQASEL QTNLVWFLKT G
RtPAL VYGVT TGF G-- GSADTRTED AISLQKALLE HQLCG
PcPAL SYGVT TGF G-- ATSHRRTKQ GGALKELIR FLNAG
TcPAM-R IYGVT TGF G-- ACSSRRTNQ LSELQESLIR CLLAG

L2
150 161
LGASG DLAPLSH
LGASG DLAPLSH
LGASG DLAPLSH
LGASG DLSPLSH
LGASG DLAPLSH
LGASG DLAPLSH
LGASG DLGPLAA
LGTSG DLGPLAA
VGASG DLVPLCY
VSASG DLQPLAH
VGASG DLTPLAH
LGCSG DLAPLSH
VGASG DLAPLAH
VGASG DLAPLAH
IGASG DLVPLSY
IGASG DLVPLSY
ISASG DLSPLSY
ITASG DLVPLSY
VSASG DLIPLAY

L3
197 208
FKEGL ALINGTS
FKEGL ALINGTS
YKEGL ALINGTS
FKGGL TLINGTS
FKEGL ALVNGTS
FKEGL ALVNGTS
FKEGL ALINGTS
YKDGL ALINGTS
AKDGL ALINGTS
VREAL AFVNGTG
HRDAL ALVNGTS
EKEGL ALLNGTD
AKEGL ALVNGTQ
AKEGL ALLNGTQ
PKEGL AMMNGTS
PKEGL AMMNGTS
PKEGL LVNGTA
PKEGL ALVNGTA
AKEGL AVNGTS

L4
273 312
SgcC4-S GSGLTVEHADL RRELQKD KEAGKDV QRSEIYLQ KAYS LRA
MdpC4-S GSGLTVEHADL RRTVQERK REEESVQ RTDIYMQ KAYS LRA
CmTAL-R DSARLSGHGELS AEMKTRAGE AKNTG-TGV FIQKAYTL RC
CmdF-R GSQILSDHQDLM KEMVARSGV GNDVDTGV YLQDAYTL RA
MxTAM-S GTGLMRDHQDIM RAISSERTS HSNDVEETE IYLQNAYS LRC
MfTAM-S GTGLMRDHQDIM RAISSERTS HSNDVEETE IYLQNAYS LRC
PaPAM-S DSSLAVNEHEVE KLIAEEMD GLVKAS--- NHQIEDAYS IRC
SmPAL DSHLAVNELDTE QTLAGEMGT VAKAG---SLAIEDAYS IRC
SsTAL GSSLAMTYDQV VDLNVPLDG --RHYQRLTRS IQDRYS LRC
SeTAL RGMVRDGSRP--- ----------LQEPYS LRC
RsTAL GSARVVRHVIAE RRLDAGDIG TEPEAG---- QDAYS LRC
SgHAL GSGLTGHHQDD APRV------- QDAYS VRC
MxHAL DS-ELVESHVNC SKV-------- QDPYS LRC
PpHAL DSSEVSLSHKN CDKV------- QDPYS LRC
NpPAL DSSLVREELDG KHEYRGK---- DLIQDRYS LRC
AvPAL NSQLVRDELDG KHDYRDH---- ELIQDRYS LRC
RtPAL GSRFAVHHEEV KVKDDEG---- ILRQDRYBL RT
PcPAL GSAYVKA AQKLHEMDPLQ ---KPKQDRYAL RT
TcPAM-R SSPFQDLSRE YYSIDKLK--- KPKQDRYAL RS

L5
339 358
NDNPL FFEG---KEIF HGANFHG
TDNPL FFEG---REIF HGANFHG
NDNPL FFED---GELF HGGNFHG
NDNPL IFDVP-EQT FHGANFHG
NDNPV ILDTP-AETY HGANFHG
NDNPL IDQTT-EEVF HNHGFHG
NDNP IVLPEE-AEV EHNGHFHG
SDNPL FDPST-GAVY SGGNFYG
QDNPI TYEG---ELL HGGNFHA
TDNPV FPPDGSVP ALHGGNFMG
VDNPV VLVPD---GRVES NGNFHG
TDNPL VFADT-ERIV SCGNFHG
SDNPL VFAAE-GDVI SGGNFHA
TDNPL IDVEN-QVSY HGGNFLG
TDNPL IDVDN-QASY HGGNFLG
TDNPL IDVEN-KTSH HGGNFQA
NDNPL IDVSR-NKAI HGGNFQG
NDNPI IDHAN-DRAL HGANFQG

L6
406 416
SgcC4-S LHS GFAGAQYP
MdpC4-S LNS GFAGAQYP
CmTAL-R LSC GFAGAQYP
CmdF-R LLC GFEGGQYL
MxTAM-S LFC GFEGGQYL
MfTAM-S LFC GFEGGQYL
PaPAM-S LRL GLMGQFM
SmPAL LRL GLMGQFM
SsTAL LQH GFKGMQIA
SeTAL RGA GLAGVQIS
RsTAL LNS GFMAQVT
SgHAL VDS GLMIAQYT
MxHAL LNS GFMAQVT
PpHAL VNS GFMAQVT
NpPAL VNMG LKGLQIS
AvPAL VNMG LKGLQIC
RtPAL LSY HCKGLDIA
PcPAL LDY GFKGAEIA
TcPAM-R VDY GLKGLDIA

L7
437 444
SNGDN QDV
SNGDN QDI
SNGDN QDV
SNGSN QDV
SNGQN QDI
SNGQN QDI
TTGDF QDI
STADF QDI
TEAHN QDK
TNGWN QDH
TNAAN QDV
SSAMQ EDH
SSAGR EDH
TSANQ EDH
AEQFN QNI
AEQFN QNI
AEMAN QAV
AEQHN QDV
AEQHN QDI
products bearing β-histidine). Yet, these analyses break down when trying to discriminate aminomutases from ammonia lyases and when predicting product stereochemistry, which is likely due to the independent evolution of aminomutases from ammonia lyases in different contexts. This independent evolution is highlighted by prokaryotic and eukaryotic ammonia lyases, which have structural differences, and these differences are retained in the aminomutases that evolved from them (Calabrese, Jordan, Boodhoo, Sariaslani, & Vannelli, 2004; Ritter & Schulz, 2004). This observation explains why attempts using mutagenesis to change SgcC4 or CmdF from aminomutases to ammonia lyases to understand the structural basis have been largely unsuccessful (Cooke & Bruner, 2010; Krug & Müller, 2009).

The MIO-containing aminomutases can be exploited to generate new α- and β-amino acids (Christenson, Wu, et al., 2003). To this end, the donor (i.e., amino acids) and acceptor (i.e., α,β-unsaturated carboxylic acids) specificities of the TcPAM enzyme were characterized; it accepts poor aminomutase substrates, such as the extended amino acid (S)-styryl-α-alanine, as good amine donors, and a variety of minimally altered arylacrylates as acceptors in generating new (S)-α- and (R)-β-amino acids (Wanninayake, Deporre, Ondari, & Walker, 2011). Under very high concentrations of ammonia as a donor, TcPAM can generate the same amino acids, which may be of industrial value (Wu et al., 2009; Szymanski et al., 2009).

This chapter describes protocols for *in vitro* characterization of MIO-containing aminomutases as exemplified by SgcC4 and MdpC4 from the C-1027 and maduropeptin biosynthetic pathways. These protocols should be applicable to mechanistic and structural characterizations of other MIO-containing aminomutases.

---

Figure 15.5 Sequence comparison of MIO-containing aminomutases and ammonia lyases. Residue numbers corresponding to SgcC4 are shown above the alignment. Conserved active-site residues are marked with (Δ), residues directly involved in substrate specificity with (▲), and those implicated in product outcome or stereochemistry with (Λ). The residue blocks correspond to loops of the inserted structure as well as the colored loops in Fig. 15.4A: L1 (blue), L2 (teal), L3 (green), L4 (yellow), L5 (orange), L6 (red), and L7 (fuchsia). Given in parenthesis are accession numbers: SgcC4 (AAL06680), MdpC4 (ABY66005), CmTAL (YP_582386), CmdF (Q0VZ68), MxTAM (B8ZV94), MfTAM (B8ZV93), PaPAM (AAO39102), SmPAL (AAF81735), SeTAL (ABC88669), SsTAL (AEV23249), RsTAL (ABA81174), SgHAL (YP_001824123), MxHAL (YP_631662), PpHAL (P21310), NpPAL (ACC80688), AvPAL (ABA23593), RtPAL (AAA33883), PcPAL (P24481), TcPAM (AAT47186). (R)- or (S)-stereochemistry for the products is designated behind the abbreviation.
2. METHODS

2.1. Overproduction and purification of SgcC4 from E. coli

1. Design PCR primers with 5′ NdeI and 3′ HindIII restriction sites in the correct frame for amplification of the *sgcC4* gene from *S. globisporus* genomic DNA or cosmid DNA containing the C-1027 biosynthetic gene cluster.

2. Prepare PCR reactions with the kit of your choice and amplify the *sgcC4* gene. Vary the concentration of an additive, such as DMSO, betaine, or 1,2-propanediol, and annealing temperature.

3. Purify the PCR product by gel electrophoresis, digest with NdeI and HindIII, and repurify the fragment by ethanol precipitation.

4. Ligate the digested PCR product into the same sites of pET28a (Novagen, Madison, WI) to afford the expression construct.

5. Introduce the resultant construct by transformation into an appropriate cloning strain such as *E. coli* DH5α, and plate on LB medium containing 25 μg/mL kanamycin. Isolate individual colonies for plasmid preparation, and sequence isolated plasmids to confirm PCR fidelity.

6. Introduce a confirmed SgcC4 expression plasmid by transformation into *E. coli* BL21 (DE3), and plate on LB medium containing 25 μg/mL kanamycin.

7. Pick a single colony from the plate, inoculate 10 mL of LB medium supplemented with 2 mM MgCl₂ and 25 μg/mL kanamycin, and incubate at 37°C overnight.

8. Transfer a 1-mL aliquot of the overnight culture to 50 mL of LB medium supplemented with 2 mM MgCl₂ and 25 μg/mL kanamycin, and grow at 37°C and 250 rpm until the optical density at 600 nm (*OD*₆₀₀) reaches 0.5 (after approximately 2.5 h). Transfer cells to 25°C and 250 rpm, and, when cells reach thermal equilibrium, induce SgcC4 expression by adding isopropyl β-D-1-thiogalactopyranoside to 25 μg/mL; continue incubation at 25°C and 250 rpm overnight.

9. Pellet the cells by centrifugation at 5000 rpm for 15 min. On ice, resuspend the cell pellet in 35 mL of buffer (50 mM sodium phosphate, pH 8.8, 300 mM NaCl, and 10 mM imidazole), and add 1 mg of lysozyme.

10. Lyse the cells by sonication on ice (10 s sonication per minute, repeated six times), centrifuge the lysate at 15,000 rpm for 1 h, and collect the supernatant.

11. Add 2 mL of Ni-NTA agarose slurry (Qiagen, Valencia, CA) to the supernatant, and gently shake on ice for 1 h in order to allow the His<sub>6</sub>-tagged SgcC4 protein to bind to the Ni-NTA agarose.
12. Load the Ni-NTA agarose-containing solution onto a column, and allow the liquid to drain away. Wash the Ni-NTA agarose with 30 mL of the above buffer. Elute the column with the same buffer but containing 200 mM imidazole. Pool the SgcC4-containing fractions, and concentrate SgcC4 to ~15 mg/mL using a Vivaspin 50,000 MWC (Sartorius Stedium, Goettingen, Germany).
13. Dialyze purified protein against 100 mM Tris–HCl, pH 8.5, 1 mM DTT, and 16% glycerol (for enzyme assay) or against 20 mM Tris–HCl, pH 7.5, and 100 mM NaCl (for crystallization). Flash-freeze convenient aliquots of the purified SgcC4 in liquid nitrogen and store at –80 °C.
14. Determine protein concentrations with the Bradford assay. This procedure affords purified SgcC4 wild type and its Ser153Ala and Ser153Cys mutant proteins as N-terminal His<sub>6</sub>-fusion proteins with an average final yield of ~65 mg/L of culture.

### 2.2. Overproduction and purification of MdpC4 from *S. lividans* TK-64

1. Design PCR primers with 5′ and 3′ ends according to the pET-30 Xa/LIC vector using the ligation-independent cloning kit as described by Novagen, and amplify by PCR the *mdpC4* gene from *A. madurae* genomic DNA or cosmid DNA containing the maduropeptin biosynthetic gene cluster.
2. Prepare PCR reactions with the kit of your choice and amplify the *mdpC4* gene. Vary the concentration of an additive, such as DMSO, betaine, or 1,2-propanediol, and annealing temperature.
3. Purify the PCR product by gel electrophoresis, insert into pET-30 Xa/LIC using the ligation-independent cloning kit to generate the intermediate clone pBS10007, and sequence to confirm PCR fidelity.
4. Digest pBS10007 with *Nde*I and *Eco*RI, purify the 1.7-kb DNA fragment by gel electrophoresis, and ligate into the same sites of pUW201PW to afford the expression construct pBS10009 (Van Lanen et al., 2007).
5. Transform *E. coli* ET12567 with pBS10009, and isolate nonmethylated plasmid.
6. Introduce nonmethylated pBS10009 into *S. lividans* TK-64 by PEG-mediated protoplast transformation following the standard procedure (Kieser, Bibb, Buttner, Chater, & Hopwood, 2000), and incubate at 28 °C.

7. Overlay transformation plates after 16 h with 1 mL of water supplemented with 50 μg/mL thiostrepton. Pick single colonies after 3 days, and transfer to a fresh R2YE-agar plate containing 50 μg/mL thiostrepton for a second round of antibiotic selection.

8. Inoculate 5 mL of liquid R2YE medium (Kieser et al., 2000) supplemented with 5 μg/mL thiostrepton with a loopful of mycelium, and grow at 28 °C and 250 rpm for 2 days. Homogenize the mycelium, and use 1 mL to inoculate 50 mL of liquid R2YE medium in a 250-mL baffled flask containing 4 g of glass beads and 5 μg/mL thiostrepton. Incubate the resulting culture at 28 °C and 250 rpm for 3 days, and harvest the mycelium by centrifugation at 5000 rpm and 4 °C for 15 min.

9. On ice, resuspend mycelium in 100 mM Tris–HCl, pH 8.0, 300 mM KCl, and 5 mM β-mecraptoethanol, and lyse mycelium using a French Press with two passes at 15,000 psi. Purify the overproduced SgcC4 protein as described in Section 2.1.

10. Desalt the purified MdpC4 protein into 20 mM Tris–HCl, pH 8.0, with 0.5 mM DTT using PD-10 columns (GE Healthcare), and store as 50% glycerol stocks at –20 °C. This procedure affords purified MdpC4 protein as an N-terminal His<sub>6</sub>-fusion protein at ~30 mg/L of culture.

### 2.3. In vitro assay of MIO-containing aminomutases SgcC4 and MdpC4

1. Incubate purified SgcC4 or MdpC4 (0.5 mg/mL) with L-tyrosine (0.5 mM) in 100 mM Tris–HCl, pH 8.8, at 25 °C. To determine kinetic constants, the concentration of L-tyrosine is varied.

2. Terminate the reaction by adding HCl to pH < 2, remove the precipitated enzymes by centrifugation, and collect the supernatant for HPLC analysis.

3. To directly analyze the reaction, use the following conditions. Analyze supernatant from step 2 by HPLC on an analytical Apollo C-18 column (250 mm × 4.6 mm, 5 μm; Grace Davison, Deerfield, IL) with UV–vis detection at 280 nm. Elute the column at a flow rate of 1.0 mL/min with a series of linear gradients from 0.1% trichloroacetic acid in 10% acetonitrile (A) to 0.1% trichloroacetic acid in 90% acetonitrile (B) in the following manner (beginning time and ending time with linear increase to % B): 0–2 min, 5% B; 2–19 min, 50% B; 19–24 min, 100% B; 24–32 min, 100% B; and 32–35 min, 5% B. Determine the peaks corresponding to HCA, L-tyrosine, and (S)-β-tyrosine by comparison
with authentic standards (Sigma–Aldrich, St. Louis, MO and PepTech Corporation, Burlington, MA) (see Fig. 15.6A for a representative HPLC chromatogram).

4. To increase sensitivity in kinetic characterization experiments, derivatize the amino acids with *o*-phthaldialdehyde (OPA), and analyze the resultant isoindole OPA derivatives by HPLC with UV–vis or fluorescence detection. Thus, neutralize the reaction supernatants from step 2 with KOH to pH 9, prepare fresh OPA reagent by dissolving 4.0 mg of OPA in 50 μL of ethanol and diluting to 4.5 mL with 0.1 M sodium borate, pH 10.4, containing 15 μL of Brij 35 (30%, w/v) and 11 μL of β-mercaptoethanol, and mix equal parts of neutralized reaction and the freshly prepared OPA reagent for 1 min at room temperature. Analyze OPA derivatives by HPLC on an Alltech Adsorbosphere C-18 column

![Figure 15.6](#fig:15.6)

Figure 15.6 Representative data from biochemical analysis of the MIO-containing SgcC4 wild-type and mutant proteins: (A) HPLC chromatogram of SgcC4-catalyzed conversion of L-tyrosine to β-tyrosine with HCA as a side-product analyzed directly with UV detection at 280 nm, (B) HPLC chromatogram of SgcC4-catalyzed conversion of L-tyrosine to β-tyrosine with HCA as a side-product analyzed after OPA derivatization with UV detection at 280 nm, (C) chiral HPLC chromatogram following the SgcC4-catalyzed conversion of L-tyrosine to (S)-β-tyrosine with (R)-β-tyrosine accumulating over time, and (D) UV–vis difference spectrum between SgcC4 wild-type and the Ser153Ala mutant proteins.(150 mm × 4.6 mm, 5 μm, Grace Davison) with UV detection at 330 nm or fluorescence detection with excitation at 340 nm and emission at 455 nm. Elute the column at a flow rate of 1 mL/min with a series of linear gradients from 50 mM sodium acetate, pH 5.7, and 5% THF (A), to methanol (B), and acetonitrile (C). The elution gradient is formed as follows (A:B:C): from 90:10:0 to 35:65:0 over 0–15 min, hold 35:65:0 over 15–20 min, from 35:65:0 to 0:50:50 over 20–25 min, hold 0:50:50 over 25–30 min, and from 0:50:50 to 90:10:0 over 30–35 min. Determine the peaks corresponding to HCA, L-tyrosine, and (S)-β-tyrosine by comparison with authentic standards (see Fig. 15.6B for a representative HPLC chromatogram).

5. To determine the stereochemistry of the β-tyrosine product, incubate 0.3–0.7 mg/mL of enzyme with 1.0 mM L-tyrosine in 50 mM CHES, pH 9.0, and 50 mM KCl, at 25 °C. Terminate the reaction by adding acetic acid to pH 4, and remove protein by filtration. Analyze 20 μL of the reaction mixture by HPLC on an Astec Chirobiotic-T column (250 mm × 4.6 mm, 5 μm, Sigma-Aldrich) with UV detection at 280 nm. Elute the column at a flow rate of 1 mL/min isocratically with methanol, acetic acid, and triethylamine (100:0.1:0.1). Determine the peaks corresponding to L-tyrosine, (R)-β-tyrosine, and (S)-β-tyrosine by comparison to authentic standards (see Fig. 15.6C for a representative HPLC chromatogram).

### 2.4. Determination of the MIO-prosthetic group in the recombinant SgcC4 protein

The MIO group is inactivated by borohydride and cyanide, which can be used to confirm its presence. The MIO group can be directly probed by site-directed mutagenesis.

1. Treat SgcC4 with either 10 mM NaBH₄ or 2 mM KCN. Remove excess reagents by filtration.
2. Protect the MIO group by incubating SgcC4 with substrate (1 mM) during 2 mM KCN treatment. Remove substrate and reagent by filtration.
3. Assay the treated (step 1) or protected and treated (step 2) SgcC4 for aminomutase activity as described in Section 2.3. Consistent with the presence of MIO, pretreatment of SgcC4 with NaBH₄ or KCN should render the enzyme inactive. Addition of substrate should protect enzymatic activity.
4. Mutate the serine in the MIO group to alanine of SgcC4 using the quick-change mutagenesis kit (Agilent Technologies, Santa Clara, CA).
5. Obtain UV absorption spectra of the SgcC4 wild-type and mutant proteins at the same concentration. Subtract the mutant spectra from the wild type. The resulting difference spectrum should have a characteristic peak at 313 nm if the MIO is formed in the wild type (see Fig. 15.6D for a representative difference UV–vis spectrum of SgcC4).

### 2.5. Crystallization of SgcC4 with substrates or inhibitors to determine catalytic mechanism

1. Mix SgcC4 (1.5 μL of 10 mg/mL in 20 mM Tris–HCl, pH 7.5, 1 mM β-mercaptoethanol, and 100 mM NaCl, with low millimolar concentrations of inhibitor or substrate) with 1.5 μL of reservoir solution (4.2–4.8 M sodium formate, 50–200 mM trimethylamine N-oxide) on a coverslip, for hanging-drop vapor diffusion at 4 °C. Needle-shaped crystals should appear within 2 days and continue to grow for approximately a week.

2. Transfer crystals to a cryoprotectant solution consisting of the mother liquor with 20% glycerol, and briefly soak before flash freezing in liquid nitrogen.

3. Collect, merge, and reduce diffraction data, and use PDB 2OHY as a model for molecular replacement to solve the structure. Refine coordinates and rebuild using a suitable program such as Phenix (Adams et al., 2002) and COOT (Emsley, Lohkamp, Scott, & Cowtan, 2010).

---

### 3. CONCLUSION

Before the discovery of SgcC4, the only known family of aminomutases required pyridoxal phosphate and either SAM and a [4Fe–4S] iron–sulfur cluster or adenosylcobalamin for activity. An exception was noted from an aminomutase, purified from *Bacillus brevis* Vm4 and associated with the biosynthesis of edeine, which required ATP for activity; however, no gene has been identified for this enzyme (Kuryloborowska & Abramsky, 1972). The characterization of SgcC4 revealed that the electrophilic power of the MIO cofactor could be harnessed to carry out the aminomutase reaction, eliminating the need for expensive cofactors and energy input. This discovery also had a broader impact on the field of natural product discovery. Previously, the MIO-containing enzymes were only associated with ammonia lyase activity, and consequently, many MIO-containing aminomutases were misannotated as HALs or PALs, hiding their true prevalence. These aminomutases can now be used as probes to search for novel
biosynthetic gene clusters, or when β-amino acids are found in natural products, their biosynthetic gene clusters can be located through cloning the responsible aminomutase. As the numbers of this new family of enzymes continue to grow, they should greatly facilitate future efforts to fully characterize their enzyme reaction mechanism and catalytic landscape and exploit their utilities in engineered biosynthesis and biocatalysis. Manipulation of β-amino acid biosynthesis for natural product structural diversity has already been demonstrated in the engineered production of several C-1027 analogs (Kennedy et al., 2007; Van Lanen et al., 2005).

Kedarcidin, another enediyne antitumor antibiotic produced by Streptoalloteichus sp. ATCC 53650, contains an exotic (R)-β-amino-chloroazatyrosine moiety (Fig. 15.1). Cloning of the kedarcidin gene cluster revealed an MIO-containing aminomutase homolog, KedY4, suggesting that the β-amino-chloroazatyrosine may be derived from azatyrosine and incorporated into kedarcidin in a homologous manner to C-1027 and MDP (Van Lanen et al., 2005, 2007). Further mechanistic and structural characterizations of homologous machinery for biosynthesis of exotic β-amino acids, such as β-azatyrosine, and their subsequent incorporation into complex natural products promise to greatly expand the scope of possibilities for biocatalysis and pathway engineering.

Finally, the enediyne antitumor antibiotic neocarzinostatin does not contain a β-amino acid or derived moiety thereof, and it features a deoxy amino sugar. Intriguingly, within the neocarzinostatin biosynthetic gene cluster, there are no canonical aminotransferases for deoxy amino sugar biosynthesis (Liu et al., 2005). Rather, there is a SgcC4 homolog (53/67% identity/similarity), NcsC3, for which the Ala-Ser-Gly motif has been truncated to Pro-Val, and the Tyr63 catalytic base mutated to Pro; hence, it cannot not be a functional MIO-containing aminomutase. Instead, NcsC3 has been proposed to catalyze Michael addition of an exogenous amino group to an α,β-unsaturated 4-ketosugar intermediate, serving as a functional equivalent of the aminotransferase (Liu et al., 2005). The latter activity of NcsC3 is reminiscent of what is proposed for the second half of MIO-aminomutase catalysis (Fig. 15.2B). It could be that the evolutionary plasticity of this enzyme family has allowed it to gain a new activity in an unusual sugar biosynthetic pathway. Mechanistic and structural characterizations of new members of the MIO-containing aminomutases therefore hold great promise for discovering novel biochemistry, enzymology, and catalytic mechanisms and for exploiting their utilities in natural product structural diversity by engineered biosynthesis.
ACKNOWLEDGMENT

This work was supported in part by National Institute of Health (NIH) grant CA078747.

REFERENCES

Adams, P. D., Grosse-Kunstleve, R. W., Hung, L. W., Ioerger, T. R., McCoy, A. J., Moriarty, N. W., et al. (2002). PHENIX: Building new software for automated crystallographic structure determination. *Acta Crystallographica D*, 58, 1948–1954.

Appert, C., Logemann, E., Hahlbrock, K., Schmid, J., & Amrhein, N. (1994). Structural and catalytic properties of the four phenylalanine ammonia-lyase isoenzymes from parsley (*Petroselinum Crispum Nym.*). *European Journal of Biochemistry*, 225, 491–499.

Berner, M., Krug, D., Bihlmaier, C., Vente, A., Müller, R., & Bechthold, A. (2006). Genes and enzymes involved in caffeic acid biosynthesis in the actinomycete *Saccharothrix espanaensis*. *Journal of Bacteriology*, 188, 2666–2673.

Calabrese, J. C., Jordan, D. B., Boodhoo, A., Sariaslani, S., & Vannelli, T. (2004). Crystal structure of phenylalanine ammonia lyase: Multiple helix dipoles implicated in catalysis. *Biochemistry*, 43, 11403–11416.

Christenson, S. D., Liu, W., Toney, M. D., & Shen, B. (2003). A novel 4-methylideneimidazole-5-one-containing tyrosine aminomutase in enediyne antitumor antibiotic C-1027 biosynthesis. *Journal of the American Chemical Society*, 125, 6062–6063.

Christenson, S. D., Wu, W., Spies, M. A., Shen, B., & Toney, M. D. (2003). Kinetic analysis of the 4-methylideneimidazole-5-one-containing tyrosine aminomutase in enediyne antitumor antibiotic C-1027 biosynthesis. *Biochemistry*, 42, 12708–12718.

Christianson, C. V., Montavon, T. J., Festin, G. M., Cooke, H. A., Shen, B., & Bruner, S. D. (2007). The mechanism of MIO-based aminomutases in beta-amino acid biosynthesis. *Journal of the American Chemical Society*, 129, 15744–15745.

Christianson, C. V., Montavon, T. J., Van Lanen, S. G., Shen, B., & Bruner, S. D. (2007). The structure of L-tyrosine 2,3-aminomutase from the C-1027 enediyne antitumor antibiotic biosynthetic pathway. *Biochemistry*, 46, 7205–7214.

Cooke, H. A., & Bruner, S. D. (2010). Probing the active site of MIO-dependent aminomutases, key catalysts in the biosynthesis of beta-amino acids incorporated in secondary metabolites. *Biopolymers*, 93, 802–810.

Cooke, H. A., Christianson, C. V., & Bruner, S. D. (2009). Structure and chemistry of 4-methylideneimidazole-5-one containing enzymes. *Current Opinion in Chemical Biology*, 13, 460–468.

Emsley, P., Lohkamp, B., Scott, W. G., & Cowtan, K. (2010). Features and development of Coot. *Acta Crystallographica D*, 66, 486–501.

Feng, L., Wanninayake, U., Strom, S., Geiger, J., & Walker, K. D. (2011). Mechanistic, mutational, and structural evaluation of a *Taxus* phenylalanine aminomutase. *Biochemistry*, 50, 2919–2930.

Frey, P. A. (2001). Radical mechanisms of enzymatic catalysis. *Annual Review of Biochemistry*, 70, 121–148.

Hernandez, D., & Phillips, A. T. (1993). Purification and characterization of *Pseudomonas putida* histidine ammonia-lyase expressed in *Escherichia coli*. *Protein Expression and Purification*, 4, 473–478.

Jin, M., Fischbach, M. A., & Clardy, J. (2006). A biosynthetic gene cluster for the acetyl-CoA carboxylase inhibitor andrimid. *Journal of the American Chemical Society*, 128, 10660–10661.

Kennedy, D. R., Gawron, L. S., Ju, J., Liu, W., Shen, B., & Beerman, T. A. (2007). Single chemical modifications of the C-1027 enediyne core, a radiomimetic antitumor drug,

affect both drug potency and the role of ataxia-telangiectasia mutated in cellular responses to DNA double-strand breaks. *Cancer Research*, 67, 773–781.

Kieser, T., Bibb, M., Buttner, M., Chater, K., & Hopwood, D. (2000). *Practical Streptomyces genetics* (2nd ed.). Norwich, England: John Innes Foundation.

Krug, D., & Müller, R. (2009). Discovery of additional members of the tyrosine aminomutase enzyme family and the mutational analysis of CmdF. *Chembiochem*, 10, 741–750.

Kuryloborowska, Z., & Abramsky, T. (1972). Biosynthesis of β-tyrosine. *Biochimica et Biophysica Acta, General Subjects*, 264, 1–10.

Langer, B., Langer, M., & Rétey, J. (2001). Methylidene-imidazolone (MIO) from histidine and phenylalanine ammonia-lyase. *Advances in Protein Chemistry*, 58, 175–214.

Liu, W., Christenson, S. D., Standage, S., & Shen, B. (2002). Biosynthesis of the enediyne antitumor antibiotic C-1027. *Science*, 297, 1170–1173.

Liu, W., Nonaka, K., Nie, L., Zhang, J., Christenson, S. D., Bae, J., et al. (2005). The neo-carzinostatin biosynthetic gene cluster from *Streptomyces carzinostaticus* ATCC 15944 involving two iterative type I polyketide synthases. *Chemistry and Biology*, 12, 293–302.

Louie, G. V., Bowman, M. E., Moffitt, M. C., Baiga, T. J., Moore, B. S., & Noel, J. P. (2006). Structural determinants and modulation of substrate specificity in phenylalanine-tyrosine ammonia-lyases. *Chemistry and Biology*, 13, 1327–1338.

Moffitt, M. C., Louie, G. V., Bowman, M. E., Pence, J., Noel, J. P., & Moore, B. S. (2007). Discovery of two cyanobacterial phenylalanine ammonia lyases: Kinetic and structural characterization. *Biochemistry*, 46, 1004–1012.

Montavon, T. J., Christianson, C. V., Festin, G. M., Shen, B., & Bruner, S. D. (2008). Design and characterization of mechanism-based inhibitors for the tyrosine aminomutase SgTAM. *Bioorganic & Medicinal Chemistry Letters*, 18, 3099–3102.

Poppe, L. (2001). Methylidene-imidazolone: A novel electrophile for substrate activation. *Current Opinion in Chemical Biology*, 5, 512–524.

Poppe, L., & Rétey, J. (2005). Friedel-Crafts-type mechanism for the enzymatic elimination of ammonia from histidine and phenylalanine. *Angewandte Chemie*, 44, 3668–3688.

Rachid, S., Krug, D., Weissman, K. J., & Müller, R. (2007). Biosynthesis of (R)-beta-tyrosine and its incorporation into the highly cytotoxic chondramides produced by *Chondromyces crocatus*. *The Journal of Biological Chemistry*, 282, 21810–21817.

Ratnayake, N. D., Wanninayake, U., Geiger, J. H., & Walker, K. D. (2011). Stereochemistry and mechanism of a microbial phenylalanine aminomutase. *Journal of the American Chemical Society*, 133, 8531–8533.

Ritter, H., & Schulz, G. E. (2004). Structural basis for the entrance into the phenylpropanoid metabolism catalyzed by phenylalanine ammonia-lyase. *The Plant Cell*, 16, 3426–3436.

Röther, D., Merkel, D., & Rétey, J. (2000). Spectroscopic evidence for a 4-methylidene imidazol-5-one in histidine and phenylalanine ammonia-lyases. *Angewandte Chemie*, 39, 2462–2464.

Schwede, T. F., Rétey, J., & Schulz, G. E. (1999). Crystal structure of histidine ammonia-lyase revealing a novel polypeptide modification as the catalytic electrophile. *Biochemistry*, 38, 5355–5361.

Szymanski, W., Wu, B., Weiner, B., de Wildeman, S., Feringa, B. L., & Janssen, D. B. (2009). Phenylalanine aminomutase-catalyzed addition of ammonia to substituted cinnamic acids: A route to enantiopure alpha- and beta-amino acids. *The Journal of Organic Chemistry*, 74, 9152–9157.

Turner, N. J. (2011). Ammonia lyases and aminomutases as biocatalysts for the synthesis of α-amino and β-amino acids. *Current Opinion in Chemical Biology*, 15, 234–240.

Van Lanen, S. G., Dorrestein, P. C., Christenson, S. D., Liu, W., Ju, J., Kelleher, N. L., et al. (2005). Biosynthesis of the beta-amino acid moiety of the enediyne antitumor antibiotic

C-1027 featuring beta-amino acyl-S-carrier protein intermediates. *Journal of the American Chemical Society*, 127, 11594–11595.

Van Lanen, S. G., Oh, T. J., Liu, W., Wendt-Pienkowski, E., & Shen, B. (2007). Characterization of the maduropeptin biosynthetic gene cluster from *Actinomadura madurae* ATCC 39144 supporting a unifying paradigm for enediyne biosynthesis. *Journal of the American Chemical Society*, 129, 13082–13094.

Vey, J. L., & Drennan, C. L. (2011). Structural insights into radical generation by the radical SAM superfamily. *Chemical Reviews*, 111, 2487–2506.

Walker, K. D., Klettke, K., Akiyama, T., & Croteau, R. (2004). Cloning, heterologous expression, and characterization of a phenylalanine aminomutase involved in Taxol biosynthesis. *The Journal of Biological Chemistry*, 279, 53947–53954.

Wanninayake, U., Deporre, Y., Ondari, M., & Walker, K. D. (2011). (S)-Styryl-α-alanine used to probe the intermolecular mechanism of an intramolecular MIO-aminomutase. *Biochemistry*, 50, 10082–10090.

Wu, P. C., Kroening, T. A., White, P. J., & Kendrick, K. E. (1992). Purification of histidase from Streptomyces griseus and nucleotide sequence of the hutH structural gene. *Journal of Bacteriology*, 174, 1647–1655.

Wu, B., Szymanski, W., Wietzes, P., de Wildeman, S., Poelarends, G. J., Feringa, B. L., et al. (2009). Enzymatic synthesis of enantiopure alpha- and beta-amino acids by phenylalanine aminomutase-catalysed amination of cinnamic acid derivatives. *Chembiochem*, 10, 338–344.

Xiang, L., & Moore, B. S. (2005). Biochemical characterization of a prokaryotic phenylalanine ammonia lyase. *Journal of Bacteriology*, 187, 4286–4289.

Zhu, Y., Liao, S., Ye, J., & Zhang, H. (2012). Cloning and characterization of a novel tyrosine ammonia lyase-encoding gene involved in bagremycins biosynthesis in *Streptomyces sp.* *Biotechnology Letters*, 34, 269–274.
