
Clinical Microbiology
Reviews

Adaptive and Mutational Resistance: Role
of Porins and Efflux Pumps in Drug
Resistance

Lucía Fernández and Robert E. W. Hancock
Clin. Microbiol. Rev. 2012, 25(4):661. DOI:
10.1128/CMR.00043-12.

---

Updated information and services can be found at:
http://cmr.asm.org/content/25/4/661

These include:

This article cites 270 articles, 180 of which can be accessed
free at: http://cmr.asm.org/content/25/4/661#ref-list-1

Receive: RSS Feeds, eTOCs, free email alerts (when new
articles cite this article), more»

---

Information about commercial reprint orders: http://journals.asm.org/site/misc/reprints.xhtml
To subscribe to to another ASM Journal go to: http://journals.asm.org/site/subscriptions/

Journals.ASM.org

Adaptive and Mutational Resistance: Role of Porins and Efflux Pumps in Drug Resistance

Lucía Fernández and Robert E. W. Hancock

Centre for Microbial Diseases and Immunity Research, Department of Microbiology and Immunology, University of British Columbia, Vancouver, British Columbia, Canada

---

**INTRODUCTION**  
Structure and Function of Porins ..... 662  
Efflux Systems in Bacteria ..... 663  

**MUTATIONAL RESISTANCE**  
Resistance Due to Mutations in Porin-Encoding Genes ..... 666  
Mutational Resistance Related to Efflux Pumps ..... 668  
Local regulators of efflux pumps ..... 668  
Global regulatory pathways ..... 670  

**ADAPTIVE RESISTANCE**  
Porins and Adaptive Resistance ..... 671  
Adaptation Mediated by Efflux Pumps ..... 671  

**IMPACT OF PORINS AND EFFLUX PUMPS ON THERAPY**  
Outer Membrane Permeabilizers ..... 673  
Negating Efflux Pump Activity ..... 673  

**CONCLUDING REMARKS** ..... 675  

**ACKNOWLEDGMENTS** ..... 675  

**REFERENCES** ..... 675  

---

**INTRODUCTION**

Over the past decades, we have observed a rapid increase in the number of antibiotic-resistant clinical isolates, together with the low rate of development and introduction of new antimicrobials. This combination represents a serious threat to human medicine, as we could one day be left unarmed against bacterial pathogens. The result of this would be increases in morbidity and mortality rates associated with infectious diseases that could reach levels akin to those seen prior to the antibiotic era. The extraordinary ability of microbes to acquire antibiotic resistance is easier to understand when analyzed from an evolutionary perspective. Thus, while the use of antibiotics as therapeutics started less than 70 years ago, bacterial resistance mechanisms have been coevolving with natural antimicrobial compounds for billions of years, as recent studies of the evolution of β-lactamases clearly demonstrated (69, 81). Despite this, the spread of resistance was limited because resistant strains are often less virulent and, consequently, less competitive than the sensitive strains from which they originated in the absence of selective pressure. However, since the introduction of antibiotics into the clinic, this pressure has dramatically escalated, resulting in a considerable acceleration of the evolution and spread of resistance markers in bacteria. This is particularly concerning due to the fact that, once acquired, resistance to antimicrobials is lost at a fairly slow pace (101). Indeed, low levels of resistance will persist in the population for some time even after the removal of a specific antibiotic from the market and rapidly return to prior levels if the antibiotic is reintroduced.

The emergence of drug resistance in the clinical environment has been a constant threat since the beginning of the antibiotic era. For example, penicillin-resistant strains of Staphylococcus aureus were already isolated in 1944, just 2 years after the introduction of this antibiotic in the market (106). A similar trend has been observed for practically all antibiotics developed to date, with resistance observed before or shortly after first clinical use and a gradual increase in the proportion of resistant isolates over time. Initially, resistance typically affected a single antibiotic or antibiotic class. However, this changed with the phenomenon of multiresistance, which was first observed in enterobacteria during the late 1950s and has been spreading ever since (121). The situation has now become so critical that a number of microorganisms, known popularly as superbugs, are resistant to most antibacterial agents currently available, thereby representing a severe therapeutic challenge.

There are three principal types of antibiotic resistance, namely, intrinsic, acquired, and adaptive. Intrinsic resistance comprises all of the inherent properties provided by the characteristics of a particular microorganism that limit the action of antimicrobials. A good example is the possession of a semipermeable outer membrane with low permeability, as is the case for the Gram-negative pathogens *Pseudomonas aeruginosa* and *Acinetobacter baumannii* or the constitutive efflux pumps observed in many bacteria. The second type is acquired resistance, in which an originally susceptible microbe can become resistant either by incorporating new genetic material (plasmids, transposons, integrons, and naked DNA, etc.) or as a result of mutations. Although some mutational events may result in a large increase in the MIC, termed here breakthrough resistance, when it takes organisms from clinically susceptible to clinically resistant, more frequently, they confer low-level resistance. These small changes in the MIC are hard to

Address correspondence to Robert E. W. Hancock, bob@hancocklab.com.  
Copyright © 2012, American Society for Microbiology. All Rights Reserved.  
doi:10.1128/CMR.00043-12

Fernández and Hancock

detect and are often missed during the analysis of clinical isolates; therefore, we have insufficient information on their occurrence in the clinic. Despite this, a growing number of authors believe that low-level resistance might play a decisive role in the gradual increase in global resistance, in particular in the creeping up over time of the baseline MICs (effectively, the concentration of the average “susceptible” organism) ([13](#ref-13), [59](#ref-59)). Indeed, there is laboratory evidence that the accumulation of independent mutations with a low impact on antibiotic susceptibility can lead to high-level resistance in a stepwise manner. For instance, El’Garch et al. ([57](#ref-57)) showed an additive effect of mutations in the genes *galU*, *nuoG*, *mexZ*, and *rplY* on the resistance of *P. aeruginosa* to the aminoglycoside tobramycin, whereby the individual mutations caused only 2-fold resistance but the quadruple mutant was 16-fold more resistant. Finally, there is adaptive resistance, a relatively unexplored area that involves a temporary increase in the ability of a bacterium to survive an antibiotic insult due to alterations in gene and/or protein expression as a result of exposure to an environmental trigger, e.g., stress, nutrient conditions, growth state, and subinhibitory levels of the antibiotics themselves. In contrast to intrinsic and acquired resistance mechanisms, which are stable and can be transmitted vertically to subsequent generations, adaptive resistance has a transient nature and usually reverts upon the removal of the inducing condition.

The mechanisms of bacterial resistance to antimicrobials are extraordinarily diverse and can be specific, whereby the primary role in the cell is to resist the action of toxic compounds, or nonspecific, in which the resistance factor is a component of other cellular functions but also exerts a protective effect against antibiotics. A major mechanism of resistance involves the inactivation of the antimicrobial by enzymes such as β-lactamases or aminoglycoside-modifying enzymes. Also, resistance can be achieved through mutations that affect the intracellular target for a given antimicrobial drug. Even if the antibiotic reaches the intracellular milieu and there are no changes in target affinity, bacteria can enhance their resistance by actively expelling the antibiotic out of the cells in a process termed efflux. Another mechanism is the reduced entry of the antibiotic into the bacterial cell due to modifications of the cell surface that limit interactions with the drug, as is the case for lipid A modifications, or reduce the number of entry channels, like porins. Even in the absence of permeability changes, the generally restricted uptake of antibiotics through the semipermeable outer membrane acts in synergy with other resistance mechanisms such as enzymatic degradation and efflux. This review focuses on the participation of porins and efflux pumps in antimicrobial resistance, and thus, we describe these mechanisms in greater detail.

### Structure and Function of Porins

Bacterial membranes constitute a selective permeability barrier that can effectively offer protection from harmful compounds in the extracellular environment while providing sufficient nutrients to the cell ([108](#ref-108)). To enable the uptake of these nutrients through their membranes, bacteria have evolved various mechanisms. One of these mechanisms is the formation of water-filled pores that extend across the membrane and facilitate the uptake of hydrophilic compounds up to a certain size exclusion limit. The proteins that form these channels are called porins and were first characterized in 1976 for *Escherichia coli* ([173](#ref-173)). Since then, porins have been identified in both Gram-negative bacteria and mycobacteria

and have been found to participate in a wide array of functions. Porins are located in the outer membrane (or the waxy outer layer of mycobacteria) and provide a sieving function whereby the outer membrane is generally permeable to hydrophilic compounds below a specific exclusion limit that varies from organism to organism and is determined by the diameter of the channels present in each individual organism.

The structure of porins is remarkably distinct among membrane proteins. Thus, in addition to lacking a classic hydrophobic region, they consist of transmembrane antiparallel β-strands with alternating hydrophobic amino acids (facing outwards) and hydrophilic amino acids (facing inwards) assembled into distinctive β-barrels rather than hydrophobic α-helices, which are more often found in proteins located in the cytoplasmic membrane. There are several classes of porins, including the so-called general porins, which are involved in determining the permeability barrier, as well as more specific porins that enable the uptake of specific substrates (e.g., LamB, which is involved in the uptake of maltose and malto-dextrins) and the iron-regulated outer membrane proteins (OMPs), which engage a cytoplasmic membrane energization system to enable the specific uptake of relatively rare iron complexes with secreted bacterial siderophores (e.g., the ferric enterochelin channel FepA). We will largely describe the general porins, since they are most pertinent to the issue of antibacterial susceptibility and resistance.

The amphipathic β-barrels of porins are connected by short periplasmic turns and by (usually longer) extracellular loops, which are generally surface exposed. Specific loops, e.g., the third loop (L3) of the general porins, bend into the channel at approximately half the height of the pore. Together with hydrophilic amino acids from several β-strands, this creates a constriction zone with an electrostatic field, created by acidic residues in loop 3 and basic residues in the opposite β-strand, which plays an important role in the selectivity of the pore for the size and charge of permeating molecules ([108](#ref-108)). The β-strand domains are very variable in sequence (although conserved in any given porin among individual strains) but are a definitive feature of porins and are highly relevant in determining the exceptional stability of these proteins, e.g., to proteases and even heating in the potent detergent sodium dodecyl sulfate ([1](#ref-1)). In contrast, the domains located in the cell surface show a much lower level of conservation from strain to strain, a feature that has been assumed to constitute a mechanism to evade adaptive immune responses. Although some porins have a monomeric structure, others are organized into trimers ([108](#ref-108)). For instance, in *E. coli*, OmpA and OmpF are examples of a monomeric porin and a trimeric porin, respectively.

The predominant function of the general porins, e.g., OmpF and PhoE of *E. coli*, is to create a size-selective defined channel for the diffusion of hydrophilic molecules with some preference for molecules with charges opposite those of the amino acids lining the channels. The levels of porins in the bacterial cell can be fairly high, up to \(10^6\) copies per cell ([1](#ref-1)), although by regulating the expression of porins in response to environmental stimuli, cells can to some extent control the permeability of their membrane to solutes. Porins tend to have exclusion limits approaching the size of many antibiotics, and thus, they tend to limit the rate of diffusion of these molecules, contributing in this way to intrinsic resistance. In addition, we now know that the roles of porins are quite diverse and include acting as receptors for bacteriocins, bacteriophages, and elements of the immune system, including antibodies,

| Species                     | Porin       | Antibiotic(s)                                                                 | Reference(s) |
|-----------------------------|-------------|------------------------------------------------------------------------------|--------------|
| Pseudomonas aeruginosa      | OprD        | Carbapenems                                                                | 12, 140      |
| Escherichia coli            | OmpC        | β-Lactams                                                                  | 99           |
|                             | OmpF        | β-Lactams                                                                  | 99           |
| Serratia marcescens         | OmpF        | β-Lactams                                                                  | 237, 258     |
|                             | OmpC        | β-Lactams                                                                  | 258          |
| Klebsiella pneumoniae       | OmpK 35     | Cephalosporins, carbapenems, fluoroquinolones, and chloramphenicol             | 35, 53, 54   |
|                             | OmpK 36     | Carbapenems                                                                | 54, 231      |
| Enterobacter cloacae        | OmpF        | Carbapenems                                                                | 54           |
| Enterobacter aerogenes      | OmpC        | Carbapenems                                                                | 235          |
|                             | OmpF        | Carbapenems                                                                | 235          |
|                             | Omp36       | Imipenem, cefepime, and cefpirome                                           | 241          |
| Klebsiella oxytoca          | OmpK 36     | Carbapenems                                                                | 36           |
| Neisseria gonorrhoeae       | PIB         | β-Lactams, tetracycline, and fluoroquinolones                                | 73           |
| Salmonella enterica         | OmpC        | Cephalosporins                                                             | 161          |
|                             | OmpF        | Chloramphenicol and imipenem                                                | 9, 243       |
| Vibrio cholerae             | OmpU        | Cephalosporins                                                             | 193          |
| Acinetobacter baumannii     | CarO        | Carbapenems                                                                | 130          |

Family    | Species                  | Pump*                     | Regulator(s)           | Antibiotic resistance to*:         | Reference(s)
--------- | ------------------------ | ------------------------- | ---------------------- | --------------------------------- | ------------
ABC       | E.coli                   | McbEF*                    | EmrR(MprA)             | FQ                              | 70, 136
          | E.coli                   | MacAB-TolC                |                        | ML                              | 107
          | E.faecalis               | EfrAB                     |                        | FQ, TC                          | 120
          | E.faecium                | MsrC                      |                        | ML                              | 229
          | M.tuberculosis           | Rv0194                    |                        | AP, EM, NB, VM                  | 47
          | M.tuberculosis           | Rv1258C(Tap)              |                        | FQ, RF, TC                      | 227
          | M.tuberculosis           | Rv2686c-Rv2688c           |                        | FQ                              | 196
          | N.gonorrhoeae            | MacAB                     |                        | ML                              | 215
          | S.aureus                 | AbcA                      | MgrA, NorG             | BL                              | 246
          | S.epidermidis            | MsrA*                     |                        | EM, SG                          | 213
          | S.maltophilia            | Smlt 2642-Smlt 2643       |                        | ML                              | 43
          | S.maltophilia            | Smlt 1537-Smlt 1539       |                        | ML                              | 43
          | S.marcescens             | SmdAB                     |                        | NF, TC                          | 156
          | S.pneumoniae             | PatA, PatB                |                        | FQ                              | 150
          | S.pneumoniae             | SP2073-SP2075             |                        | FQ, NB                          | 211
          | S.Typhimurium            | MacAB                     |                        | EM                              | 182
          | V.cholerae               | VcaM                      |                        | FQ, TC                          | 98
SMR       | A.baumannii              | AbeS                      |                        | CM, FQ, EM, NB                  | 233
          | M.smegmatis              | Mmr                       |                        | FQ                              | 128
          | S.marcescens             | SsmE                      |                        | NF                              | 163
MFS       | A.baumannii              | TetA                      |                        | TC                              | 253
          | A.baumannii              | TetB                      |                        | TC and minocycline              | 253
          | A.baumannii              | CmlA                      |                        | CM                              | 253
          | Bordetella bronchiseptica | CmlB1                     |                        | CM                              | 103
          | Clostridium difficile     | Cme                       |                        | EM                              | 118
          | E.aerogenes              | QepA*                     |                        | FQ                              | 195
          | E.faecium                | EfmA                      |                        | FQ                              | 184
          | E.coli                   | Mef(B)*                   |                        | ML                              | 133
          | E.coli                   | QepA*, QepA2*             |                        | FQ                              | 30, 265
          | E.coli                   | EmrB                      | EmrR(MprA)             | NA                              | 26
          | E.coli                   | MdfA(CmlA/Cmr)            |                        | AG, CM, EM, FQ, RF, TC          | 181
          | Listeria monocytogenes   | Lde                       |                        | FQ                              | 75
          | M.smegmatis              | LfrA                      | LfrR                   | FQ                              | 128
          | S.Typhimurium            | EmrAB                     |                        | NA, NB                          | 182
          | S.Typhimurium            | MdfA                      |                        | CM, NF, TC                      | 182
          | S.marcescens             | SmfY                      |                        | NF                              | 225
          | S.aureus                 | LmrS                      |                        | TM, CM                          | 62
          | S.aureus                 | NorA                      |                        | FQ                              | 249
          | S.aureus                 | NorB                      | MgrA, NorG             | FQ                              | 245
          | S.aureus                 | NorC                      | MgrA                   | FQ                              | 247
          | S.aureus                 | MdeA                      |                        | FU, NB                          | 97
          | S.aureus                 | SdrM                      |                        | NF                              | 263
          | S.aureus                 | Tet 38                    | MgrA                   | TC                              | 245
          | S.maltophilia            | Smlt 0032                 |                        | ML                              | 43
          | Streptococcus pyogenes   | MefA                      |                        | ML                              | 39
          | V.cholerae               | VceCAB                    |                        | CM, EM, NA                      | 40
MATE      | A.baumannii              | AbeM                      |                        | FQ, AG                          | 236
          | E.cloacae                | EmmdR                     |                        | FQ, TM                          | 86
          | E.coli                   | NorE                      |                        | FQ                              | 266
          | E.coli                   | YdhE                      |                        | AG, NF, CI                      | 166
          | N.gonorrhoeae            | NorM                      |                        | CC                              | 216
          | N.meningitidis           | NorM                      |                        | CC                              | 216
          | S.Typhimurium            | MdtK                      |                        | NF                              | 182
          | S.aureus                 | MepA                      | MepR                   | FQ, TI                          | 159
          | V.cholerae               | NorM                      |                        | AG, FQ                          | 228
          | V.cholerae               | VcmB, VcmD, VcmH, VcmN     |                        | AG, FQ                          | 21
          | V.parahaemolyticus       | NorM                      |                        | AG, NF, CI                      | 166

* An asterisk means that the pump-encoding genes are plasmid borne; alternative names for the pumps are in parentheses.
** Not all known substrates of the efflux pumps are included, only antibiotics. AG, aminoglycosides; AP, ampicillin; BL, β-lactams; CC, cationic compounds; CI, ciprofloxacin; CM, chloramphenicol; CL, clindamycin; CP, cephalosporins; EM, erythromycin; FQ, fluoroquinolones; ML, macrolides; NA, nalidixic acid; NB, novobiocin; NF, norfloxacin; RF, rifampin; SG, streptomycin B; SM, sulfamethoxazole; TC, tetracycline; TI, tigecycline; TM, trimethoprim; VM, vancomycin.

| Species                | Pump                     | Regulator(s)                                                                 | Antibiotic resistance to$^a$:                                                                                          | Reference(s) |
|------------------------|--------------------------|-----------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|--------------|
| A. baumannii           | AdeABC                   | AdeT, AdeRS                                                                  | AG, BL, CM, EM, TC, FQ                                                                                             | 144          |
| A. baumannii           | AdeFGH                   | AdeL                                                                         | FQ, TC, TI, CM, CL, TM, SM                                                                                        | 42           |
| A. baumannii           | AdeIJK                   |                                                                             | BL, CM, EM, FQ, FU, NB, RF, TC, TM                                                                                 | 46           |
| Aeromonas hydrophila   | AheABC                   | AheR                                                                         | BL, EM, FU, TC, TM                                                                                                | 90           |
| Burkholderia cenocepacia | CeoAB-OpcM              | CeoR                                                                        | CM, FQ, TM                                                                                                        | 172          |
| Burkholderia pseudomallei | AmrAB-OprA             | AmrR                                                                        | AG, ML                                                                                                            | 165          |
| B. pseudomallei        | BpeAB-OprB               | BpeR                                                                         | AG, ML                                                                                                            | 31, 32       |
| B. pseudomallei        | BpeEF-OprC               | BpeT                                                                         | CM, TC                                                                                                            | 112          |
| C. jejuni              | CmeABC                   |                                                                             | AP, CM, CT, EM, FQ, TC                                                                                            | 3, 131       |
| E. aerogenes           | AcrAB-TolC               | AcrR                                                                         | CM, FQ, NB, TC                                                                                                    | 203          |
| E. aerogenes           | EefABC                   |                                                                             | CM, CI, EM, TC                                                                                                    | 154          |
| E. cloacae             | AcrAB-TolC               |                                                                             | AG, BL, CM, FQ, EM, TC, TI                                                                                        | 199          |
| E. coli                | AcrAB-TolC               | AcrR, MarA, RobA, SoxS, MarR, SdiA                                           | BL, NB, EM (ML), CM, TC, FQ                                                                                        | 63, 142      |
| E. coli                | AcrAD-TolC               |                                                                             | AG, FU, NB                                                                                                        | 212          |
| E. coli                | AcrEF-TolC               | AcrS                                                                         | BL, NB, EM (ML), CM, TC, FQ                                                                                       | 143          |
| E. coli                | MdtABC-TolC              | BaeSR                                                                        | NB                                                                                                               | 14, 170      |
| E. coli                | YhiUV-TolC               | EvaAS                                                                        | NB                                                                                                               | 183          |
| H. influenzae          | AcrAB-TolC               |                                                                             | RF, EM, NB                                                                                                        | 221          |
| K. pneumoniae          | AcrAB                    | AcrR                                                                         | FQ                                                                                                               | 158          |
| N. gonorrhoeae         | MtrCDE                   | MtrR, MtrA                                                                   | AZ, ML, RF, PN, CI                                                                                               | 80, 139      |
| P. aeruginosa          | MexAB-OprM               | MexR, PA 3574, ArmR                                                         | AG, BL, CM, ML, NB, TC, TM                                                                                        | 126, 201, 202 |
| P. aeruginosa          | MexCD-OprJ               | NfxB                                                                         | CM, CP, FQ, TC,                                                                                                  | 200          |
| P. aeruginosa          | MexEF-OprN               | MexT                                                                         | CM, FQ                                                                                                            | 110          |
| P. aeruginosa          | MexXY (AmrAB)            | MexZ (AmrR)                                                                  | AG, FQ, ML, TC, zwitterionic BL, TI                                                                               | 2, 164, 259  |
| P. aeruginosa          | MexJK-OprM               | MexL                                                                         | AG, CI, EM, TC                                                                                                    | 38           |
| P. aeruginosa          | MexVW-OprM               |                                                                             | CM, RM, FQ, TC                                                                                                    | 124          |
| Proteus mirabilis     | AcrAB-TolC                |                                                                             | AP, CI, CM, TC, TI, TM                                                                                            | 254          |
| S. marcescens          | SdeAB                    |                                                                             | CM, FQ                                                                                                            | 113, 114     |
| S. marcescens          | SdeCDE                   |                                                                             | NB                                                                                                               | 20           |
| S. marcescens          | SdeXY                    |                                                                             | FQ, TC                                                                                                            | 34           |
| S. maltophilia         | SmeDEF                   | SmeT                                                                         | EM, FQ, TC                                                                                                        | 7, 270       |
| S. maltophilia         | SmeABC                   | SmeRS                                                                        | AG, BL, FQ                                                                                                        | 129          |
| S. maltophilia         | SmeIJK                   |                                                                             | AG, CI, TC                                                                                                        | 43           |
| S. maltophilia         | SmeYZ                    | Smlt 2199-2130                                                               | AG                                                                                                               | 43           |
| S. Typhimurium         | AcrAB                    | AcrR                                                                         | BL, CM, FQ, NB, EM, RF, TC                                                                                        | 115, 177     |
| S. Typhimurium         | MdtABC                   |                                                                             | NB                                                                                                               | 182          |
| S. Typhimurium         | MsdABC/TolC              |                                                                             | NB                                                                                                               | 182          |
| V. cholerae            | VexAB-TolC               |                                                                             | EM, NB, PO                                                                                                        | 22           |
| V. cholerae            | VexEF-TolC               |                                                                             | EM, NF, NB, TC, TM                                                                                                | 205          |

$^a$ Not all known substrates of the efflux pumps are included, only antibiotics. AG, aminoglycosides; AP, ampicillin; AZ, azithromycin; BL, β-lactams; CC, cationic compounds; CI, ciprofloxacin; CM, chloramphenicol; CL, clindamycin; CP, cephalosporins; CT, cefotaxime; EM, erythromycin; FQ, fluoroquinolones; FU, fusidic acid; ML, macrolides; NA, nalidixic acid; NB, novobiocin; NF, norfloxacin; PN, penicillin; PO, polymyxin B; RF, rifampin; SM, sulfamethoxazole; TC, tetracycline; TI, tigecycline; TM, trimethoprim; VM, vancomycin.
of antibiotics, such as macrolides and tetracycline, and can be found only in bacteria, both Gram negative and Gram positive. Examples of the three subfamilies are Bmr of *Bacillus subtilis* (176), QacA of *S. aureus* (239), and MefA of *Streptococcus pyogenes* (39), respectively.

Proteins from the MATE family share a similar topology with proteins of the MFS. However, they constitute a different group due to the low level of homology at the amino acid sequence level. These proteins have 12 transmembrane regions and use sodium gradients to carry out the export of toxic compounds like fluoroquinolones, aminoglycosides, and cationic dyes. Members of this family include NorM of *Vibrio parahaemolyticus* and YdhE, its *E. coli* homolog (166), among others (Table 2).

One of the most relevant efflux pump families in the clinical context is the RND family, which has been characterized best in Gram-negative bacteria. These pumps consist of three elements: an inner membrane pump protein with 12 transmembrane regions and two large periplasmic loops, a so-called membrane fusion protein, and an outer membrane protein that forms a so-called channel-tunnel. The pump protein is usually trimeric and appears to work in a rotary fashion in which individual subunits become alternately protonated and then engage and subsequently disengage substrate molecules, possibly capturing them from the cytoplasmic membrane-periplasm interface. The two halves of the inner membrane component generally show a very striking degree of similarity, indicating the probable occurrence of a tandem duplication event. The substrates of RND pumps are very diverse and comprise antibiotics, biocides, toxic fatty acids, bile salts, aromatic hydrocarbons, inhibitors of fatty acid biosynthesis, detergents, homoserine lactones, and dyes. The function of the so-called membrane fusion protein is not well understood but is thought to operate as a grappling hook to bring the base of the outer membrane channel-tunnel into alignment with the inner membrane pump. The outer membrane channel comprises a trimeric arrangement of a 12-stranded β-barrel (4 strands from each monomer of the homotrimer) with very long coiled-coil alpha-helical segments on the periplasmic side which at their base contact the pump and are thought to open and close at the base through an iris diaphragm-like uncoiling.

Almost all RND systems are able to pump out multiple antibiotics and couple this drug efflux with proton antiport. The two best-characterized RND pumps are AcrAB-TolC of *E. coli* (142, 63) and MexAB-OprM of *P. aeruginosa* (201). However, there are numerous examples of RND efflux pumps with a demonstrated role in antibiotic resistance (Table 3).

The best-known role of efflux pumps is their ability to export antibiotics and other drugs out of bacterial cells. As shown in Tables 2 and 3, there are many pumps from all five families that participate in the resistance of human-pathogenic bacteria to clinically relevant antibiotics. However, their ubiquitous nature, being present in all microorganisms, including pathogens and nonpathogens, suggests that their evolution and spread occurred independently of the generalized use of antimicrobials in humans. Thus, it is highly likely that efflux pumps play an important defensive role against different toxic compounds that bacteria may encounter in their environment. For example, the natural habitat of *E. coli* is the gastrointestinal tract, where bile salts have a significant presence. Therefore, it is not surprising that the RND pump AcrAB-TolC displays a high affinity toward bile salts, indicating that this might be its natural substrate (240). In addition to this

role that facilitates the colonization of the enteric tract, AcrAB also benefits *E. coli* cells by conferring resistance to antibiotics (142). Nevertheless, recent reports described possible alternative roles for efflux pumps in addition to protection from toxic molecules. A very interesting function is the possible participation of some efflux systems in the secretion of quorum-sensing signals. For example, Lamarche and Déziel (116) showed that MexEF-OprN can export HHQ (4-hydroxy-2-heptylquinoline), a precursor of the *Pseudomonas* quinolone signal (PQS). Additionally, OprM has been related to the pathogenicity of this microorganism (93). The BpeAB-OprB efflux pump of *Burkholderia pseudomallei* was also essential for the production of quorum-sensing autoinducers as well as virulence factors like siderophores and phospholipase C, which are controlled by quorum sensing (31). However, this link seems to be strain dependent, as a recent study showed no effect of mutations in BpeAB-OprB on quorum-sensing-related factors in a different strain (162). Also, the Blt transporter of *B. subtilis* was shown to be involved in the efflux of spermidine (262). More recently, a study of *P. aeruginosa* showed that the overexpression of the pump MexCD-OprJ increases the presence of long-chain fatty acids in the exometabolome, which may indicate that these are the natural substrates of this transporter (234). All of these examples suggest that it is possible that drug transporters initially performed a role in the secretion of physiologically relevant substrates and later proved useful to expel toxic compounds, including antibiotics and, possibly, toxic secondary metabolites.

### MUTATIONAL RESISTANCE

The spontaneous occurrence of mutations due to error-prone DNA synthesis instigates the phenotypic variability that drives the evolution of bacterial populations, enabling them to adapt to pressures posed by the surrounding environment. Thus, depending on the specific nature of these pressures, different mutations will prove to be advantageous for the bacterial cells. The selection of mutations related to antibiotic resistance occurs in this manner. Although certain changes that confer resistance to antimicrobial compounds are actually detrimental to the growth and/or virulence of the microbe, in the context of antibiotic therapy, they may actually be essential for survival and thus will be selected. These mutations are then transmitted to subsequent generations and will generally be fairly stable, particularly in the presence of antibiotic pressure. Secondary mutations can also accumulate to reverse any deficits in growth or virulence engendered by the primary resistance mutation. With this in mind, it is easy to understand the fast and efficient process of artificial selection for resistant mutants brought by less than a century of antibiotic use in human medicine. Intriguingly, the removal of the selective pressure by resting an antibiotic (i.e., ceasing or decreasing usage in the clinic) can lead to much lower levels of resistance to that antibiotic. However, upon the reintroduction of the antibiotic, resistance levels tend to rebound to those levels occurring prior to the rest period. There is a growing body of evidence indicating that mutations affecting the expression or functionality of porins and efflux pumps have a major role in the increased resistance of clinical isolates.

#### Resistance Due to Mutations in Porin-Encoding Genes

The bacterial cell envelope is an effective semipermeable barrier to substances present in the environment. This is especially the case for Gram-negative organisms, which possess not only a cytoplasm-
Porins and Efflux Pumps

Wild-type

Acquisition of mutational resistance

Mutant: porin loss
Mutant: narrow channel
Mutant: decreased expression

FIG 1 Examples of different mechanisms of acquisition of mutational resistance associated with porins. The blue circles represent the antibiotic molecules, and the red cross indicates that the antibiotic cannot cross the outer membrane. Abbreviations: IM, inner membrane; OM, outer membrane; PP, periplasmic space.

mic membrane but also an outer membrane consisting of an inner layer of phospholipids and an outer layer of lipopolysaccharides. In order to enter the cell, antibiotics need to overcome the outer membrane barrier, which can occur through three different mechanisms depending on the specific type of antimicrobial (83). The major route of entry for hydrophilic antibiotics is through the water-filled channels formed by outer membrane proteins. Therefore, the number and type of porins possessed by a cell will determine the permeability and, consequently, the antibiotic susceptibility or resistance of the microorganism. For instance, *P. aeruginosa* and *A. baumannii*, which are both known for their high levels of resistance to antimicrobials, have low membrane permeability, which underpins their general (intrinsic) resistance to many antibiotics. Thus, mutations affecting the expression and/or function of porins have a direct impact on the susceptibility of bacteria to antimicrobials. These mutations can have different effects, such as (most commonly) porin loss, a modification of the size or conductance of the porin channel, or a lower expression level of a porin (Fig. 1). However, all of these changes result in a limited, substantially slower diffusion of the antimicrobial into the cell and, consequently, reduced bacterial killing. In particular, porin-related mutations can substantially influence resistance to β-lactams, fluoroquinolones, tetracycline, and chloramphenicol.

The impact of the loss of porin on antibiotic resistance has been known for decades. One of the earliest examples is the involvement of the OmpF porin from *E. coli* in resistance to β-lactams, which was determined in 1981 (84). Since then, porins involved in antibiotic resistance have been identified in many bacterial species (Table 1). In some cases, antibiotic-resistant strains display a complete loss of one or more porins. For example, a lack of the basic amino-acid-selective outer membrane porin OprD in *P. aeruginosa* carbapenem-resistant clinical isolates can be due to a point mutation leading to an early termination of translation, a deletion, or the presence of an insertion element within the porin-encoding gene (218, 261) or can be due to regulatory mutations (110, 187). Point mutations in the promoter region that result in a lower level of transcription are another cause of porin loss. For instance, an *Enterobacter cloacae* isolate resistant to ertapenem, which appeared to lack the porin OmpF according to SDS-PAGE analysis, carried a mutation in the promoter that led to a 20-fold reduction in the expression level of the *ompF* gene (54).

However, not all resistant isolates display an apparent reduction in the expression level of porins, and in this case, resistance can be due to mutations that affect the proper function of porins. For instance, it is well known that certain mutations in loop 3 of different porins result in increased resistance to antibiotics. As mentioned above, loop 3 dips into the interior of the channel to form the constriction zone and thus participates in the electrostatic field and the narrowest part of the channel within the pore lumen (108). This has a direct impact on the transport of molecules across the porin channel. For instance, a G-to-D mutation in loop 3 of the OmpF/OmpC-like protein of *Enterobacter aerogenes* leads to a loss of conductance of the pore and, as a result, to decreased susceptibility to β-lactams (48). Similarly, mutations in loop 3 of the porin PIB of *Neisseria gonorrhoeae* confer resistance to penicillin and tetracycline in strains that simultaneously overexpress an efflux pump due to an *mtrR* mutation (189, 190). In *Vibrio cholerae*, a D116A mutation in OmpU, which was predicted to be an antibiotic binding site, leads to increased resistance to cephalosporins (193). On the other hand, imipenem binding in *P. aeruginosa* OprD necessitates the presence of intact loops 2 and 3 (185), while external loops 5, 7, and 8 served to constrict the OprD channel entrance and prevent the nonspecific passage of antibiotics (96). Mutations in the constriction zone reduced the passage of

Fernández and Hancock

cefotaxime and other β-lactams through the OmpC channel of several resistant *E. coli* isolates (138).

A third type of mutation includes those affecting the regulatory proteins that control the expression of porin-encoding genes. For instance, the *ompB* operon, which contains the genes *ompR* and *envZ*, is known to regulate the expression of OmpC and OmpF in *E. coli* (204). Jaffe et al. (99) described the identification of spontaneous cefoxitin-resistant mutants of *E. coli* with mutations in *ompB*. In addition to the *ompB* locus, many other proteins, like Rob, SoxS, and MarA, and small RNAs, like *micF*, are known to participate in the regulation of the transcription of porin genes (49). Potentially, mutations in any of these loci could lead to changes in porin expression and, consequently, changes in antibiotic resistance. Another example is the lower level of expression of the porin OprD of *P. aeruginosa* following the occurrence of mutations that constitutively activate the ParRS two-component system, which is necessary for the downregulation of this porin in the presence of certain antimicrobial peptides (169), while a variety of other regulatory mutants exist, including *nfxC* mutations that upregulated the efflux pump MexEF-OprN while downregulating OprD (110, 186, 187).

Some studies have demonstrated the selection of bacteria carrying mutations in porin-encoding genes during antibiotic therapy or under laboratory conditions mimicking an antimicrobial treatment regimen. For example, *E. coli* isolates from a patient undergoing treatment with several antimicrobials showed changes in the OmpC protein that resulted in four different variants, all of which conferred increased resistance to cefotaxime (138). Likewise, Oteo et al. (192) described the recovery of three consecutive isolates of *E. coli* from a patient treated with ertapenem. The first isolate was susceptible to carbapenems, whereas the second one was resistant to imipenem due to the loss of the OmpF and OmpC porins. Interestingly, the third *E. coli* isolate recovered the susceptible phenotype. *In vitro* experiments of exposure to antibiotics showed similar results. For example, the exposure of *P. aeruginosa* strains to meropenem under laboratory conditions selected carbapenem-resistant cells via the loss of the porin OprD (88), while this mutant or the regulatory mutant influencing OprD and efflux expression are commonly found in the clinic during intensive imipenem therapy of *P. aeruginosa* infections (68, 140).

Mutational resistance related to porin changes can be linked to the stepwise increase in the resistance of pathogens and is thought to participate in the phenomenon of MIC creep (60). Thus, the loss of any particular porin generally has only a minor to moderate effect on the overall resistance of the microorganism, leading to low-level resistance. However, the accumulation of a sequence of independent mutational events affecting various resistance mechanisms can gradually confer increasing resistance to the bacterium until it acquires high-level resistance. For this reason, it is fairly common to observe bacterial strains in which the effect of the loss of porin is enhanced by additional mechanisms. For example, *Serratia marcescens* meropenem-resistant strains overproduce AmpC β-lactamase and lack the porin OmpF (237). Similarly, carbapenem-resistant *Klebsiella pneumoniae* and *E. coli* strains with carbapenemases can also show a lack of certain outer membrane proteins (79, 117). In *K. pneumoniae*, resistance to ciprofloxacin has been related to mutations in *gyrA*, *parC*, and *ompK35* in what appears to be a multistep process of resistance acquisition (35). Another clear example is a *P. aeruginosa* carbapenem-resistant

isolate that carried an integron harboring a β-lactamase gene, overexpressed two efflux pumps, and lacked OprD (146). A recent study also described *E. coli* clinical isolates that were resistant to cefpirome and cefepime, lacked OmpC and OmpF, and, at the same time, displayed increased levels of production of the TEM-1 or OXA-1 β-lactamase (18).

Overall, mutations that lead to the loss, downregulation, or alteration of porins have a direct impact on resistance by limiting the rate at which an antibiotic can enter the cell and thus enhance the influence of secondary resistance mechanisms (efflux and degradative enzymes like β-lactamases, etc.) that take advantage of the lower rate of antibiotic uptake. There is evidence that these mutations may occur throughout the course of therapy in the clinic. As a result, monitoring the occurrence of these changes in bacterial isolates will be helpful to predict evolution toward a resistant phenotype.

### Mutational Resistance Related to Efflux Pumps

The possession of active multidrug efflux pumps contributes to the intrinsic resistance of a bacterial pathogen. The occurrence of a mutational event leading to an increased expression level of a given pump will inevitably result in the acquisition of even greater resistance to all of the antibiotic substrates of that pump. In some cases, these mutations cause amino acid changes that make a pump more efficient at extruding the antibiotics out of the cell. For instance, Vettoretti et al. (252) described an F1018L substitution in MexY, an RND transporter of *P. aeruginosa*, which increased resistance to aminoglycosides, fluoroquinolones, and the β-lactam cefepime, mediated by the MexXY efflux pump. Notably, this mutation was present in clinical isolates from cystic fibrosis (CF) patients. Another interesting phenomenon was observed for *Salmonella enterica* serovar Typhimurium, in which the presence of the IS1 or IS10 insertion element in the promoter region upstream of the genes encoding the AcrEF pump enhanced the transcription of this operon, thereby conferring resistance to fluoroquinolones (191). Nonetheless, the vast majority of mutations affecting antibiotic export by efflux systems occur in genes encoding proteins with a regulatory function.

The overexpression of efflux systems is frequently associated with a loss of fitness and virulence properties. For example, mutants of *P. aeruginosa* overexpressing the RND transporters MexAB and MexCD are less able to survive in water, show a reduced level of production of proteases and phenazines, and are less virulent than the wild type in a worm model (220). Consequently, mutations that increase the expression levels of efflux pumps are largely advantageous in the presence of antibiotics. For this reason, microorganisms need to ensure that genes encoding efflux pumps are expressed only when necessary, and therefore, their expression is tightly regulated by both local and global regulators.

#### Local regulators of efflux pumps

The genes coding for efflux pump components are often linked to a regulatory gene, the product of which may exert a repressor or activator role. There are good examples of this among the *P. aeruginosa* Mex efflux pumps (Fig. 2). For instance, resistant strains with the *nalB* phenotype, which display increased resistance to β-lactams, were found to display an enhanced transcription of the RND system MexAB-OprM (208). This overexpression was due to mutations in the gene located adjacent to the *mexAB-oprM* operon in the chromosome, *mexR*, which encodes a repressor of the MarR family (126).

Porins and Efflux Pumps

![Diagram](#)

FIG 2 Schematic representation of the complex pathways that regulate the expression of the Mex efflux systems and the porin OprD in *P. aeruginosa*. Red and blue arrows represent activation and repression, respectively. Abbreviations: AG, aminoglycosides; AMPs, antimicrobial peptides; BL, beta-lactams; CA, carbapenems; CI, ciprofloxacin; CM, chloramphenicol; CP, cationic peptides; EM, erythromycin; FQ, fluoroquinolones; ML, macrolides; NB, novobiocin; TC, tetracycline; TI, ticarcillin; TM, trimethoprim; IM, inner membrane; OM, outer membrane; C4-HSL, N-butyryl-L-homoserine lactone; ROS, reactive oxygen species.

Even a single-base substitution in *mexR* can suffice to produce a resistant phenotype (202). Conversely, an insertion of the IS21 element in the *mexR* gene was identified in a clinical strain that was resistant to ticarcillin and aztreonam (24). Mutations in *nalB* have been found quite often in resistant clinical isolates, in particular in those from patients who had been treated with β-lactams (134, 272). The efflux pump MexCD-OprJ is similarly regulated by the product of the *nfxB* gene, which is adjacent to but in an opposite orientation from the operon encoding the pump components (188). *nfxB* mutants were first isolated due to their norfloxacin-resistant phenotype and also displayed hypersusceptibility to β-lactams and aminoglycosides (92). The β-lactam susceptibility phenotype was later shown to be caused by a reduced level of production of β-lactamase (155). The overexpression of MexCD-OprJ has deleterious consequences for bacterial fitness in infections, reducing characteristics such as siderophore, protease, and phenazine production; cytotoxicity; motility; and, very significantly, virulence (220, 234). In spite of these clear disadvantages, *nfxB* mutants are readily selected following exposure to fluoroquinolones, azithromycin, and even biocides like triclosan both *in vitro* and during clinical therapy (37, 100, 168). Another regulator is MexZ, which belongs to the TetR family and normally represses the transcription of the *mexXY* genes by binding to the *mexZ-mexX* intergenic region (157, 264). Mutations in MexZ have been found to participate in the acquisition of moderate aminoglycoside resistance in CF isolates of *P. aeruginosa* (256) and may be a significant cause of impermeability-type aminoglycoside resistance, which occurs in 10% or more of patients treated extensively with aminoglycosides.

In contrast to the examples described above, the transcription of the operon encoding MexEF-OprN, briefly mentioned above, is subject to activation instead of repression. This positive regulation is mediated by the LysR-type regulator MexT, encoded by the gene immediately upstream of *mexEF-oprN* (109). Strains that overproduce the MexEF-OprN efflux pump are called *nfxC*-type strains, because of their resistance to norfloxacin. In many cases, this phenotype is due to mutations that result in the activation of MexT, which is normally inactive in certain *Pseudomonas* strains, including laboratory strains, due to the insertion of 8 nucleotides within the *mexT* open reading frame (ORF) (152). In addition to resistance to fluoroquinolones, chloramphenicol, and trimethoprim, *nfxC* mutants typically exhibit decreased susceptibility to imipenem due to the downregulation of the porin OprD (67, 68). Another characteristic of these mutants is their greater susceptibility to several β-lactams and aminoglycosides, which might be related to some extent to the downregulation of the *mexAB-oprM* operon observed for these mutant strains (153). Interestingly, Sobel et al. (230) observed that the mutation of the *mexS* gene, which is under the transcriptional control of MexT and codes for an oxidoreductase, leads to an *nfxC*-like phenotype. Moreover, some studies of clinical isolates indicated that additional unidentified genes may be responsible for this phenotype (135). Mutants of the *nfxC* type have been isolated in the clinic, but they are not very common. The most likely explanation for this is their reduced virulence (102, 109, 111), together with the fact that these mutations appear to be less effective at engendering antibiotic resistance *in vivo* than they are under laboratory conditions (77).

Local regulators of efflux pumps have also been demonstrated to be important for the acquisition of mutational resistance in microorganisms other than *P. aeruginosa*. For instance, mutations in the regulatory proteins AdeR and AdeL from the opportunistic
pathogen A. baumannii result in increased antibiotic resistance due to the overexpression of the AdeABC and AdeFGH pumps, respectively (42, 91). Other regulators include AcrR from *E. coli* and *S. enterica*, a repressor of the AcrAB pump, and SmeT from *Stenotrophomonas maltophilia*, which represses the expression of SmeDEF (319). This tight regulation also exists in Gram-positive bacteria. This is the case for the novel transporter MepA of *S. aureus*, a member of the MATE family, which is regulated by the product of an adjacent gene, MepR, encoding a MarR family transcriptional regulator (159). MepR acts as a negative regulator of the *mepRAB* operon, and mutations in *mepR* lead to the overexpression of the efflux pump, thereby conferring low-level resistance to tigecycline.

Global regulatory pathways. In addition to local regulators, the expression of efflux systems is also under the control of global regulatory pathways. For instance, the regulation of the MexAB-OprM pump of *P. aeruginosa* is fairly complex and involves regulators other than MexR. Indeed, clinical strains with *mexAB-oprM* overexpression without any alteration in the sequence of the *mexR* gene have been isolated (232). To date, two additional genes have been found to enhance the transcription of *mexAB-oprM* upon mutation. So-called *nalC* mutants are mutated in a TetR-type transcriptional regulator encoded by PA3721 (28). This causes the overexpression of the PA3719-PA3720 operon, which is generally repressed by the product of PA3721 (78). Indeed, the overexpression of the gene PA3719 in a high-copy-number plasmid in a wild-type strain leads to a *nalC* phenotype. More recently, it was demonstrated that the product of the *armR* (PA3719) gene binds to and associates with MexR *in vivo*, reducing its suppression of *mexAB-oprM* expression (44). Multidrug resistance via MexAB-OprM overexpression can also occur due to mutations in *nalD*, which encodes a TetR family repressor (230), and *nalD* mutations have been observed for *P. aeruginosa* clinical strains.

In the *Enterobacteriaceae*, the best-studied example of an efflux pump under the control of a complex global regulatory network is AcrAB, which is present in several species, including *E. coli*, *S. enterica*, and *K. pneumoniae*, among others. In *E. coli*, the *acrAB* genes are located next to the regulator-encoding gene *acrR*, which exerts a negative control over its expression. Additionally, the positive regulator MarA also plays a major role in governing *acrAB* expression (8). The gene encoding this regulator is located in the three-gene *marRAB* locus, named after its role in multiple-antibiotic resistance. The product of *marR* is a repressor of the *mar* operon, and consequently, mutations in this gene promote the overexpression of the AcrAB efflux pump, leading to multidrug resistance. Mutations in the *mar* locus have been identified in fluoroquinolone-resistant clinical isolates (145). In addition to these direct regulators, mutations in the *mppA* gene, which encodes a murein peptide binding protein, also led to increased expression levels of MarA with the consequent overproduction of AcrAB and reduced expression levels of the porin OmpF (127). The exact mechanism for this effect of MppA on *marA* transcription has not yet been determined. The *mar* locus is also part of a stress response regulon that involves regulation by the positive regulators encoded by *sox* and *robA*. Thus, mutations in these genes also influence the expression of *acrAB* (8). In *K. pneumoniae* and *S. enterica*, the expression of *acrAB* is governed by an additional global regulator, RamA (217, 267). In *Salmonella*, field isolates carrying mutations in the regulatory region of *ramRA* or an insertion in *soxR*, increasing the expression levels of *ramA* and

soxS, respectively, displayed increased resistance to fluoroquinolones (104).

The expression of efflux pumps and, consequently, antibiotic resistance can also be under the control of quorum-sensing signals. This is the case for the AcrAB pump of *E. coli*, which is regulated by the quorum-sensing-dependent protein SdiA, a cell division regulator (207). Links between quorum sensing and efflux pumps have also been found for MexAB-OprM of *P. aeruginosa* (153, 223) and BpeAB-OprB of *B. pseudomallei* (31). Also, in *P. aeruginosa*, the expression of the MexEF-OprN transporter depends on the global regulator MvaT, which participates in the quorum-sensing circuits and regulates processes such as biofilm formation, swarming motility, and the production of virulence factors (52). A recent study revealed that mutations in *mvaT* promote the expression of the *mexEF-oprN* operon, which leads to resistance to several antibiotics (260). However, the clinical relevance of this type of mutation has yet to be determined.

Another important class of regulator is the two-component systems, which are crucial for the rapid fine-tuning of bacterial responses to changes in the environment surrounding the microbe. Two-component systems also participate in controlling the expression of efflux pump-encoding genes, and consequently, mutations in these systems can also mediate the resistance of microorganisms to antimicrobial compounds. In some cases, the genes encoding these systems are very specific, and they are located near the pump-encoding genes, acting as local regulators. For instance, the previously mentioned two-component system AdeRS of *A. baumannii* acts as a regulator of the adjacent *adeABC* operon (91). However, two-component regulators can also exert an influence on the transcription of efflux pumps encoded by genes located in separate regions of the chromosome as part of a more general response. Recently, the ParRS system of *P. aeruginosa* was identified as an important regulator of resistance to several antibiotics in response to the presence of certain antimicrobial peptides, including the clinically relevant polymyxins (61, 169). The transcriptional responses mediated by ParRS include the upregulation of the *arn* operon, involved in peptide resistance, as well as the enhanced expression of the MexXY pump and the downregulation of the OprD porin, which determine susceptibility to aminoglycosides and imipenem, respectively. Significantly, the dysregulation of *mexXY* and *oprD* occurs independently of MexZ, the local repressor of MexXY (169). It was further determined that gain-of-function mutations in ParRS, resulting in the constitutive activation of this system, conferred resistance not only to polymyxins but also to aminoglycosides, fluoroquinolones, and β-lactams. Moreover, the occurrence of this class of mutations was demonstrated in clinical isolates from both CF and non-CF patients. Another example of a two-component system linked to efflux is BaeSR from *E. coli*, which is involved in the envelope stress response and regulates the expression of the MdtABC efflux pump (119).

To make matters worse, it is not uncommon to observe clinical strains carrying several mutations that affect the expression of efflux pumps. For example, Tomás et al. (242) found several β-lactam-resistant strains that simultaneously carried mutations in *mexR*, *nalC*, and *nalD*. This reinforces the concept that the gradual accumulation of low-impact mutations is a major driving force of antibiotic resistance in the clinic. Furthermore, considering the broad range of mutations that affect efflux in bacteria as well as their impact on resistance to a range of different com-
pounds, this accumulation is doubtless decisive in the evolution of multidrug resistance in pathogens. Therefore, the ability to prevent the action of pumps would represent an effective way of tackling this phenomenon, as we discuss below.

**ADAPTIVE RESISTANCE**

Microbes can alter their transcriptome very rapidly in response to changes in the surrounding environment, thereby increasing their chances of survival. Some of these modifications confer to the bacterium a greater ability to withstand an antimicrobial challenge. There are many environmental cues that can lead to the temporary acquisition of resistance to a given antibiotic, including ion concentrations, temperature, and, very importantly, exposure to nonlethal doses of antimicrobials (59). It is becoming increasingly clear that the adaptations that bacterial cells undergo during the infection, together with repeated and/or prolonged exposure to antimicrobials throughout treatment, provide one explanation as to why apparently sensitive strains often cannot be efficiently eradicated in the clinic with antimicrobial therapy. Also, both adaptive and low-level mutational resistances are very good candidates for participating in the phenomenon of the upward creeping of baseline MICs over time. For all these reasons, adaptive resistance is attracting considerable attention and becoming the subject of numerous studies. As mentioned above, the expression of genes encoding porins and efflux pumps is finely regulated in order to respond to certain signals, thereby altering the resistance of a bacterium depending on the growth conditions. This is particularly relevant if these triggers include the conditions found by the microorganism inside the host or the doses of antibiotics encountered during therapy.

**Porins and Adaptive Resistance**

There are several examples whereby exposure to antibiotics regulates the expression of porins and, in consequence, the permeability of the cell envelope. For instance, the exposure of *E. coli* cells to chlorotetracycline or tetracycline leads to the downregulation of numerous porins (132, 269). In a similar fashion, subinhibitory concentrations of the benzodiazepine drug diazepam promote the development of adaptive multiresistance in *E. coli* and *K. pneumoniae* by depleting porin expression and inducing efflux systems (238). This phenotype led to decreased susceptibility to norfloxacin, chloramphenicol, tetracycline, nalidixic acid, and β-lactams. Likewise, *Moraxella catarrhalis* responds to exposure to aminopenicillins by reducing the expression level of the porin M35, consequently developing adaptive resistance to these antibiotics (101). M35 is also regulated by temperature, being downregulated during growth at 26°C or 42°C compared to growth at 37°C, as well as by osmotic stress and iron limitation. Adaptation to the biocide benzalkonium chloride in *E. coli* resulted in a reduced presence of several OMPs, such as OmpA, OmpF, and OmpT (23). The authors of that study observed that this conferred increased tolerance to not only this quaternary ammonium compound but also antibiotics such as chloramphenicol, ciprofloxacin, nalidixic acid, ampicillin, and cefotaxime. In *Serratia marcescens*, the presence of the two major porins OmpF and OmpC varies according to osmotic stress, temperature, pH, and added salicylate (19). Salicylate is known to induce a multiple-antibiotic-resistance phenotype in *E. coli* by promoting the expression of the *marRAB* operon (5).

The number and type of porins in the outer membrane of a bacterial cell determine its ability to allow nutrient uptake as well as its susceptibility to toxic compounds. With this in mind, it is not surprising that the expression of genes encoding porins is well orchestrated by regulatory pathways that, in some cases, are coordinated with stress networks. A good example of this is the regulation of the porins OmpC and OmpF of *E. coli*. OmpF has a somewhat larger pore size than OmpC; therefore, the relative proportions of these two proteins have a dramatic impact on the type and amount of solutes that can enter the cell. Thus, when the environment surrounding the bacteria has low osmolarity, there is a considerable increase in the presence of OmpF in the outer membrane, which facilitates the influx of nutrients. In contrast, in nutrient-rich environments where the osmolarity is high, e.g., *in vivo*, it would be detrimental to express a high level of OmpF, which would allow toxic compounds to enter the cell. Therefore, under high-osmolarity conditions, there is a downregulation of OmpF and an upregulation of OmpC, leading to increased resistance to β-lactams. This response is mediated by the two-component regulator OmpR-EnvZ (82). Other environmental signals, like high temperature, oxidative stress, or salicylate, modulate the expression of porins via the upregulation of the antisense RNA *micF*, which diminishes OmpF translation (15, 55). Acidic pH is another condition that increases OmpC levels while decreasing OmpF levels.

A study by Viveiros et al. (255) analyzed the responses of *E. coli* to tetracycline exposure. This work showed that the cells initially undergo stress responses mediated by the global regulators MarA, SoxS, and Rob, which regulate the expressions of OmpC and OmpF. However, this early response is followed by a long-term adaptation in which MarA works together with MicF and OmpX, but not SoxS and Rob, to downregulate OmpC and OmpF expression. OmpX is an outer membrane protein that is expressed in an inverse manner relative to the expression of other porins (55). This appears to be because OmpX production saturates the chaperones necessary for the folding of porins, causing other unfolded porins to be degraded by DegP proteases (255). The expression of OmpX is enhanced upon exposure to different environmental stresses, including antibiotics like fluoroquinolones and novobiocin, salicylate, high-ionic-strength buffer, and the chelator dipyridyl (56). Additional studies indicated that the cascades that regulate porin expression in *E. coli* may be even more complex than this, involving numerous regulatory proteins and sRNAs, depending on the specific environmental signal involved (29).

**Adaptation Mediated by Efflux Pumps**

Certain environmental conditions, including exposure to antimicrobial compounds, have an impact on the expression of efflux systems. The differential expression of genes encoding efflux pumps due to changes in the extracellular milieu reflects the importance of these systems for the adaptation of microorganisms to their surroundings. In particular, in the case of pathogens, this is an indication that efflux pumps may play a role in survival within the host. A clear example is the NorA pump of *S. aureus*, which was recently found to be regulated by iron availability (51). This regulation was mediated by the regulatory protein Fur, and induction could be repressed by the addition of FeCl₃ to the growth medium. Moreover, the authors of that study suggested that NorA, which confers low-level resistance to several antibiotics, may also participate in the secretion of siderophores, which might be its natural function. As another example, in *S. aureus*, resistance to moxifl

Fernández and Hancock

floxac**in** can be acquired by the upregulation of NorB during growth at a low pH (244). This regulation is exerted via the negative regulator MgrA, which must be in a phosphorylated form to be active in preventing *norB* transcription. When the pH is low, the levels of phosphorylated MgrA decrease considerably, hence the resistant phenotype.

In the Gram-negative pathogen *A. baumannii*, the expression of several efflux pumps varies depending on the concentration of NaCl (95). Thus, physiological NaCl concentrations increase resistance to antibiotics such as levofloxacin and amikacin through the upregulation of efflux pumps, which could be demonstrated because this response could be suppressed by using an efflux pump inhibitor (EPI). NaCl, as well as ethanol, can also induce the expression of the *E. coli* pump AcrAB in a process that does not involve the participation of the regulator AcrR (141). The presence of oxidative compounds is another factor that contributes to the dysregulation of efflux pump expression. Thus, the aminoglycoside pump MexXY of *P. aeruginosa* is induced by reactive oxygen species (ROS) via the product of the gene PA5471 (65). This is relevant to infections because the microorganism will encounter ROS, e.g., during infections of the CF lung. On the other hand, the RND efflux system AcrAB of *S. enterica* serovar Typhimurium demonstrates a higher level of expression in response to indole and paraquat (180). Indeed, the biological oxidant indole elicits the expression of four pumps from this pathogen, namely, AcrAB, AcrD, MdtABC, and RmrAB. In the case of AcrAB, indole exerts this action via the induction of RamA, which in turn is induced by RamR (179). In contrast, paraquat relies on another regulator, SoxS, and not RamA for the activation of the *acrAB* operon. Likewise, RamA was not required for the induction of *acrAB* expression by bile salts (180). The complex regulation of AcrAB, which is induced through different pathways depending on the specific environmental signal, is further complicated due to its regulation by the *mar* locus. In fact, MarA is necessary for the upregulation of *acrAB* and the acquisition of resistance to ciprofloxacin upon exposure to salicylate (85). Salicylate also leads to ciprofloxacin resistance in *Campylobacter jejuni* by preventing the binding of the repressor CmeR to the promoter of the *cmeABC* operon, which codes for an efflux system (226). Moreover, salicylate facilitated the selection of ciprofloxacin-resistant mutants in this pathogen.

As was the case with porins, antimicrobials play a very important role in the induction of adaptive resistance by the overexpression of efflux systems. Indeed, when bacteria become exposed to subinhibitory concentrations of the antibiotic during therapy, which can easily happen if antibiotics are misused, cells will not be killed and, moreover, will become more resistant to a subsequent antibiotic challenge. In that sense, it is worth highlighting the dangers associated with the overuse of biocides, which are increasingly being related to the acquisition of both adaptive and mutational resistances that can affect clinically relevant antibiotics. For instance, chlorhexidine upregulates the genes encoding the MexCD pump of *Pseudomonas* in a process dependent on the activity of the regulator AlgU (64). Another example is the induction of multiresistance in *S. maltophilia* by exposure to triclosan, which binds to the repressor protein SmeT, thereby releasing it from its operator upstream of the *smeDEF* operon (89). The use of antibiotics in the clinic also poses dangers with regard to adaptive resistance. In *Streptococcus pneumoniae*, for instance, exposure to antimicrobials promotes the expression of various efflux pumps. For instance, fluoroquinolones upregulate the ABC transporters PatA

and PatB and confer resistance to norfloxacin, ciprofloxacin, and levofloxacin (58). The same effect was observed by incubating *S. pneumoniae* cells in the presence of another DNA-damaging agent, mitomycin C. The authors of that study explained that this phenomenon was likely related to the induction of the competence pathway, as part of a global response to DNA damage. Additionally, in *S. pneumoniae*, the efflux system *mef(E)-mel*, located on the small mobile genetic element MEGA (macrolide efflux genetic assembly), is induced by macrolides and antimicrobial peptides (33, 268). In the case of macrolides, induction was dependent on the amino sugar attached to the C-5 of the macrolide lactone ring but not on the size of the ring. Thus, macrolides with a monosaccharide, but not those with a disaccharide, enhanced *mef(E)* expression. The upregulation of this system leads to resistance to 14- and 15-membered macrolides, which include clinically important macrolides such as erythromycin, azithromycin, and clarithromycin (33). The fact that LL37 can also induce macrolide resistance through this mechanism is concerning, as this peptide is part of the human host defense system (268). Erythromycin also has the ability to induce adaptive resistance in *Burkholderia pseudomallei* and *Staphylococcus* by upregulating the expression of the efflux pumps BpeAB-OprB and MsrA, respectively (31, 213). In *P. aeruginosa*, the acquisition of adaptive resistance to aminoglycosides has been known for decades. Indeed, this knowledge has been essential for the design of improved therapeutic regimens, which now involve the administration of a higher dose with a lower frequency within the range permitted by toxicity risks (16, 45). To a large extent, this adaptive phenotype is due to the aminoglycoside-induced expression of the genes encoding the MexXY efflux pump, which varies according to the aminoglycoside concentration (94). This pump has also been shown to be involved in the antagonism between divalent cations and aminoglycosides (147). The induction of *mexXY* by aminoglycosides requires the upregulation of the gene PA5471, which in turn inhibits the repressor MexZ (65).

The overexpression of efflux pumps is also involved in the complex resistance mechanisms displayed by bacteria in the biofilm state. Several examples of this have been observed for *P. aeruginosa*. A very interesting case is the ABC transporter encoded by the PA1875-PA1876-PA1877 operon. Mutants in this pump exhibited a susceptible phenotype for different antibiotics, such as gentamicin, tobramycin, and ciprofloxacin, which was specific to biofilms, whereas planktonic cells had resistance levels similar to those of the wild-type strain (271). The authors of that study demonstrated that this was due to the upregulation of the PA1875-PA1876-PA1877 genes in biofilm-forming cells. The exposure of biofilms to antibiotics also contributes to the development of adaptive resistance; in some cases, the molecular mechanisms involved differ from those of planktonic cells. For example, a challenge of *P. aeruginosa* biofilms with colistin triggers the adaptation of the upper layers of the biofilm through the overexpression of the lipopolysaccharide (LPS) modification (*arn*) operon as well as the efflux pump *mexAB-oprM* (194), both of which were proposed to be necessary for acquired resistance to colistin. The deeper layers of the biofilm, however, lacked the capacity to develop this response and remained sensitive to colistin. For this reason, Pamp et al. (194) strongly recommended the use of combined therapy for the treatment of biofilm-related infections. Also, Gillis et al. (74) described the importance of the MexAB-OprM and MexCD-OprJ efflux pumps for the growth of *P. aeruginosa* biofilms in the
presence of the macrolide azithromycin. Thus, while *mexAB-oprM* showed constitutive expression independently of antibiotic exposure, *mexCD-oprJ* overexpression was dependent on azithromycin. Interestingly, this induction was observed only in biofilms and not in planktonic cells.

It is clear that the dysregulation of porins and efflux pumps in response to external cues, including antimicrobials, is an important mechanism for the acquisition of adaptive resistance. Thus, microorganisms can transiently increase their ability to withstand the presence of antibiotics by limiting the entry of antibiotics into the cell or by expelling them more efficiently. Considering this, it is essential to determine which conditions may trigger this phenomenon during the course of infection in order to minimize the risk of therapeutic failure.

### IMPACT OF PORINS AND EFFLUX PUMPS ON THERAPY

Given the importance of efflux pumps and porins in the acquisition of resistance to antimicrobials in bacterial pathogens, the development of strategies that reverse these mechanisms would have a direct impact on therapeutic success. Indeed, efflux pumps themselves have become antimicrobial targets in their own right and are the subject of substantial interest in the drug development industry. At a time when few new antibiotics are being introduced and bacteria can display very high levels of antibiotic resistance, the possibility of finding new ways of making antibiotics more efficacious, by overcoming resistance mechanisms, provides new challenges.

#### Outer Membrane Permeabilizers

The loss or downregulation of porins results in a reduced permeability of the outer membrane. As a result, when resistance is acquired by this mechanism, the use of compounds that can increase outer membrane permeability would highly facilitate the penetration of the antibiotic into the bacterial cell and increase susceptibility. Thus, combined therapy with an antibiotic and a permeabilizing agent would be more successful in eradicating an infection than the administration of the antibiotic alone. Some of the best-characterized classes of permeabilizers include chelators, polycations such as cationic antimicrobial peptides and polymyxin B nonapeptide (PMBN), cholic acid derivatives, and squalamine derivatives (105, 222, 250). Cationic peptides are a very attractive option due to their additional activities as immune modulators, antimicrobials, and antibiofilm agents. Their impact on outer membrane permeability is due to their mechanism of interaction with the outer membrane component LPS as part of their self-promoted uptake into cells. Thus, these peptides interact with the outer membrane at the site where LPS is cross-bridged by divalent cations, since the polycationic peptides have a higher affinity for these sites than the native divalent cations (usually Mg²⁺ and Ca²⁺). This causes local outer membrane perturbation and permeabilization and enables the uptake of the peptides and other antimicrobials through the disturbed outer membrane (83). The effectiveness of such polycations has been observed for various bacteria, including *E. coli*, *S. enterica*, *K. pneumoniae*, *E. cloacae*, *P. aeruginosa*, *A. baumannii*, *Proteus vulgaris*, and *S. marcescens* (72, 105, 123, 250). For instance, the polycation polyethyleneimine (PEI) had an impact on the permeabilities of the outer membranes of several Gram-negative pathogens, including *E. coli*, *P. aeruginosa*, and *S. enterica* serovar Typhimurium, and increased their susceptibilities to hydrophobic antibiotics such as clindamycin, erythromycin, fucidin, novobiocin, and rifampin, even though PEI did not have bactericidal activity (87). However, this effect was inhibited by millimolar concentrations of MgCl₂. PMBN also increased the susceptibility of Gram-negative microorganisms to hydrophobic antibiotics (248). To date, however, these compounds have not been successfully developed for the clinic; for example, PMBN failed due to toxicity. Synthetic peptides have also been demonstrated to show synergy with antibiotics such as the fluoroquinolone ciprofloxacin and the β-lactam carbenicillin (224).

Interestingly, permeabilizers also potentiated the effects of antimicrobials in the eradication of *P. aeruginosa* biofilms (10). This could be promising given that the treatment of infections involving biofilms is extremely difficult. In that sense, a recent study indicated that the synergy between the aminoglycoside tobramycin and the permeabilizer CSA-13, a cholic acid derivative with antimicrobial properties, could facilitate treatment of *P. aeruginosa* biofilms (171). Another compound with permeabilizing properties is lactic acid, which was shown to be more effective than EDTA, a chelator, against *E. coli* O157:H7, *P. aeruginosa*, and *S. enterica* serovar Typhimurium (4). A variety of other permeabilizers has been described (83). Permeabilizers can also enable the use of new antimicrobial compounds that are not very effective because of their low ability to enter the cell. For instance, bacteriophage endolysin EL188, which was considered to be limited to Gram-positive pathogens, inhibited *P. aeruginosa* when combined with certain outer membrane permeabilizers, in particular EDTA (25).

Even though permeabilizing agents might provide improved antibiotic therapy regimens, this would most prominently affect resistance to hydrophobic and amphipathic compounds. However, changes in porins largely affect susceptibility to hydrophilic antibiotics. In that sense, more research is necessary to find strategies that can facilitate the penetration of hydrophilic antimicrobial drugs into the cells, e.g., by investigating a broader range of peptides and mimetics (224), in order to increase the activity of antibiotics and reduce the selection of resistant strains. Intriguingly, the permeabilization of bacterial cells also has the potential to overcome efflux-mediated resistance due to the synergy between low outer membrane permeability and multidrug efflux.

### Negating Efflux Pump Activity

There has been considerable effort directed toward the development of compounds that can counteract the effect of efflux pumps on antibiotic resistance in both Gram-negative and Gram-positive bacteria. Indeed, efflux systems are now considered important drug targets for the development of novel therapeutics, although again, this concept has not yet been translated to the clinic. The potential usefulness of efflux pump inhibitors (EPIs) for potentiating the activity of antibiotics has been clearly demonstrated *in vitro* for different pathogens, including *P. aeruginosa*, *E. coli*, *A. baumannii*, and *S. enterica*, among others. The inhibition of efflux pump activity enables the greater accumulation of drugs inside bacterial cells, allowing for enhanced killing. Theoretically, this can be achieved by different mechanisms, such as competition with the antibiotics for the inner membrane pump component (the most commonly utilized method), the blockage of the channel formed in the outer membrane, the inhibition of the energy source used by the pump, interference with pump assembly, or an alteration of the transcriptional regulation of the pump-encoding
Fernández and Hancock

genes. Structurally, EPIs are very diverse and have been proposed to include compounds such as diamines like phenylalanyl-arginyl-β-naphthylamide (PAβN) (MC-207,110), the energy uncoupler carbonyl cyanide *m*-chlorophenylhydrazone (CCCP), globomycin (an inhibitor of the enzyme that processes lipoprotein precursors), pyridopyrimidines, arylpiperazine derivatives, tetracycline analogs, and quinoline derivatives. *In vitro* data show that the lack of efflux activity causes not only lower intrinsic resistance but also a reduced risk of selection of resistant mutant strains ([137](https://doi.org/10.1128/CMR.16.4.582-600.2003)). Furthermore, combined therapy that includes an antibiotic and an EPI may enhance the activity spectrum of the antibiotic.

The first described EPI was PAβN, which inhibited the activity of the pumps that contribute mostly to fluoroquinolone resistance in *P. aeruginosa*, namely, MexAB-OprM, MexCD-OprJ, and MexEF-OprN ([137](https://doi.org/10.1128/CMR.16.4.582-600.2003)). Later, PAβN was also demonstrated to work by inhibiting RND pumps in other relevant Gram-negative bacteria, making this compound broad spectrum. Thus, it potentiates the activities of macrolides in *Haemophilus influenzae* and of flofenicol in *E. coli* and *S. enterica*. However, PAβN is not active against the MFS pumps responsible for fluoroquinolone resistance in Gram-positive bacteria. Interestingly, PAβN, which is a substrate of efflux pumps, does not affect the efflux of all pump substrates. This suggests that not all antibiotics bind to the same residues of the transporter and that this inhibitor exerts its activity by competing with certain substrates for binding to the pump. Lomovskaya et al. ([137](https://doi.org/10.1128/CMR.16.4.582-600.2003)) also demonstrated that the combination of PAβN and a fluoroquinolone significantly reduced the frequency of selection of resistant bacteria. This is probably due to the fact that the inhibition of the MexAB-OprM efflux pump decreased intrinsic resistance to such a degree that the effect of single mutations affecting fluoroquinolone resistance was insufficient to make cells resistant enough to survive at the tested concentrations. This same phenomenon was demonstrated with similar compounds in a neutropenic mouse model ([78](https://doi.org/10.1099/mic.0.2006/000086-0), [209](https://doi.org/10.1128/AAC.49.1.355-361.2005)). Also, *C. jejuni* displayed a 1,000-fold reduction in the emergence of resistant colonies when simultaneously exposed to erythromycin and PAβN ([151](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)). PAβN is only one of a large number of candidate compounds active against RND pumps of Gram-negative bacteria. In contrast to PAβN, other EPIs are quite specific with regard to the pump that they inhibit. For instance, pyridopyrimidines block the efflux of all substrates by MexAB-OprM without affecting transport by the other *P. aeruginosa* RND pumps ([174](https://doi.org/10.1099/mic.0.2006/000086-0)). Another group of inhibitors comprises semisynthetic tetracycline analogs, which selectively inhibit tetracycline pumps, the most potent of which is 13-cyclopropylthio-5-OH tetracycline (13-CPTC), a compound that competitively binds to the TetB pump of *E. coli* ([175](https://doi.org/10.1099/mic.0.2006/000086-0)). In Gram-positive pathogens, pumps responsible for fluoroquinolone resistance have also been investigated in order to search for possible inhibitors. One example is INF392, which reduces the ability of NorA to efflux ciprofloxacin in *S. aureus* ([149](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)).

To search for effective EPIs, Cortez-Cordova and Kumar ([41](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)) pointed out the value of utilizing a strain lacking all efflux pumps as a background strain for assessing the response of a specific pump to a given inhibitor candidate without the interference of the other efflux systems of the bacterium. In their study, those researchers investigated the activity of PAβN in inhibiting the AdeFGH pump of *A. baumannii* by overexpressing this system in a *P. aeruginosa* laboratory strain without efflux systems. The identification of EPI candidates involves not only high-throughput screening under *in vitro* conditions but also *in silico* analysis to determine the most appropriate directions for the design of new derivatives. A recent study by Rahman et al. ([206](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)) described the *in silico* screening of compounds that can inhibit efflux. More specifically, those authors looked for compounds that were designed escort molecules, which were able to establish interactions with antibiotics. The formation of a complex between the escort and the antibiotic would modulate the activity of efflux pumps.

The results obtained by studies with EPIs indicate that this is a very promising strategy to overcome multidrug resistance. However, there are still substantial barriers to their use in the clinic. Ideally, it would be advantageous to develop broad-spectrum EPIs, and more experiments are necessary to firmly demonstrate their efficacy in realistic animal models, especially with target-resistant isolates, as well as to evaluate their safety and possible side effects. An additional concern is the development of resistance to EPIs themselves. For example, PAβN is known to affect the integrity of the membrane, and therefore, it might induce resistance mechanisms that can reduce the penetration of certain antibiotics. Also, the overexpression of EPI-resistant pumps would lead to resistance. For example, mutants resistant to the EPI reserpine were selected in *S. pneumoniae* strains and were caused by the overexpression of the pump PatA ([71](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)). Another important aspect is the existence of opposite effects on the activities of different antibiotics. For example, 1-(1-naphthylmethyl)-piperazine potentiates the effect of tetracycline in *A. baumannii*, but it increases the resistance of this pathogen to the more clinically relevant antibiotic tigecycline ([17](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)). EPIs have contributed substantially to studies of the involvement of efflux pumps in the resistance of clinical isolates ([50](https://doi.org/10.1128/AAC.47.11.3511-3516.2003), [125](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)), by demonstrating their ability to reverse resistance. However, a recent study found that reserpine did not accurately predict the overexpression of efflux pumps in *Staphylococcus* ([66](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)).

In addition to efflux pump inhibitors, some recent studies have attempted to design novel strategies to counteract efflux in human-pathogenic bacteria. For instance, Al-Hamad et al. ([6](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)) tested the use of a polyclonal antibody against the ABC efflux pump SmrA of *S. maltophilia*. Thus, when the antibody was combined with antibiotics of clinical relevance, a decrease in MICs could be observed. An alternative way of inhibiting pumps is by modulating their expression. For instance, the nonsteroidal anti-inflammatory drug (NSAID) diclofenac led to a downregulation of efflux systems in *S. aureus* and decreased resistance to fluoroquinolones but not to other antibiotics ([210](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)). Conversely, an antisense phosphorothioate oligodeoxynucleotide against OprM ([257](https://doi.org/10.1128/AAC.47.11.3511-3516.2003)) led to the repression of the transcription of the *oprM* gene and increased antibiotic susceptibility.

In-depth studies of the structure and function of efflux pumps also have the potential to be very useful for developing new antimicrobials that are not good efflux pump substrates and, as a result, would be minimally affected by efflux pump overexpression. For instance, the overexpression of the NorA and PmrA pumps of *S. aureus* and *S. pneumoniae*, respectively, does not affect resistance to certain fluoroquinolones such as levofloxacin, moxifloxacin, gemifloxacin, gatifloxacin, and garenoxacin. Presumably, this is due to their increased hydrophobicity, which allows a very efficient entry of the fluoroquinolones into the cell or reduces their affinity for pump components. Another example is tigecycline, a very recent tetracycline-like antibiotic, which is not recognized by tetracycline-specific efflux pumps such as TetA-TetD, TetK, and TetM. Also, insights into the

contribution of pumps to antibiotic susceptibility can be obtained diagnostically. Thus, immunological or quantitative reverse transcription-PCR (RT-qPCR) analysis of efflux pump expression may indicate the presence of an MDR phenotype and help in the choice of the most appropriate type of therapy.

**CONCLUDING REMARKS**

The broad use and environmental presence of antimicrobial compounds pose an intense selective pressure on microbes. In such a milieu, rapid adaptation is key to survival, and bacteria have been enormously successful in this endeavor. This adaptation involves several steps, including initially the transient modulation of expression of multiple genes and proteins that increase the chances of withstanding antibiotic challenges until normal conditions return. These transient changes can be stabilized by mutation, and even small increases in MICs can accumulate, leading eventually in a stepwise fashion to breakthrough resistance. Among the mechanisms that bacteria utilize to become more resistant to antibiotics either temporarily or permanently, the modulation of the influx and efflux of antimicrobials is particularly prominent. On the one hand, the exposure of bacterial cells to subinhibitory doses of antibiotics or other environmental cues dysregulates the expression of porins and/or efflux systems, making the cells able to survive a subsequent challenge with a higher, normally inhibitory dose. Conversely, intensive and prolonged exposure to antibiotics, in the context of therapy, can select those cells harboring mutations that decrease permeability by reduced porin expression or promote the efflux of the antibiotic by pumps.

Given the importance of porins and efflux pumps in the rise of antibiotic resistance in pathogens, it has become a challenge to counter their effects. In particular, efflux systems have become the targets of an exciting group of compounds, designated efflux pump inhibitors. These substances do not need to exert antimicrobial activity but rather potentiate the effects of clinically relevant antibiotics by inhibiting efflux pumps. This makes them potentially useful for the treatment of infections caused by multiresistant organisms due to increased efflux and may also allow, in some cases, the utilization of lower antibiotic doses. Nevertheless, efflux pump inhibitors are still the subject of studies to show if they are safe for clinical use and whether they will represent a clear improvement in therapy. Since they were first tested in the clinic years ago but have not yet achieved any notable successes, there is clearly some way to go. Currently, the main use of efflux pump inhibitors is to study the influence of pumps on the resistance of clinical isolates. Conversely, there is currently no specific strategy that has been advanced to counteract the effects of porin loss or downregulation. To date, permeabilizing agents have been shown mainly to enhance the action of hydrophobic antibiotics rather than hydrophilic drugs, which are the usual substrates of porins. Therefore, the development of new molecules that increase the permeability of bacterial membranes to hydrophilic compounds would be very useful as a combination therapy strategy.

Overall, the study of the molecular mechanisms involved in antibiotic resistance through changes in porins and efflux pumps will provide a better picture of the processes that bacterial pathogens undergo inside the patient. A better understanding of the mechanisms of resistance should ultimately provide new adjunctive alternatives to improve the efficacy of antibiotic therapy programs, at a time when the development of new antimicrobial compounds based on the direct inhibition of bacteria is very limited. Specifically, these studies may help combat multidrug-resistant

strains and infections caused by biofilms, both of which are now a serious challenge.

**ACKNOWLEDGMENTS**

Our work on antibiotic resistance is funded by grants from Cystic Fibrosis Canada and the Canadian Institutes for Health Research. R.E.W.H. holds a Canada research chair in microbiology. L.F. was supported by a postdoctoral fellowship from the Fundacion Alfonso Martin Escudero (Spain).

We apologize to all the authors whose work is not mentioned in this review despite their valuable contributions to the knowledge of resistance mediated by porins and efflux pumps. The vast number of publications on this topic made it impossible to include all articles.

**REFERENCES**

1. Achouak W, Heulin T, Pagès JM. 2001. Multiple facets of bacterial porins. FEMS Microbiol. Lett. 199:1–7.
2. Aires JR, Kohler T, Nikaido H, Plesiat P. 1999. Involvement of an active efflux system in the natural resistance of *Pseudomonas aeruginosa* to aminoglycosides. Antimicrob. Agents Chemother. 43:2624–2628.
3. Akiba M, Lin J, Barton YW, Zhang Q. 2006. Interaction of CmeABC and CmeDEF in conferring antimicrobial resistance and maintaining cell viability in *Campylobacter jejuni*. J. Antimicrob. Chemother. 57:52–60.
4. Alakomi HL, et al. 2000. Lactic acid permeabilizes gram-negative bacteria by disrupting the outer membrane. Appl. Environ. Microbiol. 66: 2001–2005.
5. Alekshun MN, Levy SB. 1999. Alteration of the repressor activity of MarR, the negative regulator of the *Escherichia coli marRAB* locus, by multiple chemicals *in vitro*. J. Bacteriol. 181:4669–4672.
6. Al-Hamad A, Burnie J, Upton M. 2011. Enhancement of antibiotic susceptibility of *Stenotrophomonas maltophilia* using a polyclonal antibody developed against an ABC multidrug efflux pump. Can. J. Microbiol. 57:820–828.
7. Alonso A, Martinez JL. 2000. Cloning and characterization of SmeDEF, a novel multidrug efflux pump from *Stenotrophomonas maltophilia*. Antimicrob. Agents Chemother. 44:3079–3086.
8. Aono R, Tsukagoshi N, Yamamoto M. 1998. Involvement of outer membrane protein TolC, a possible member of the *mar-sox* regulon, in maintenance and improvement of organic solvent tolerance of *Escherichia coli* K-12. J. Bacteriol. 180:938–944.
9. Armand-Lefèvre L, et al. 2003. Imipenem resistance in *Salmonella enterica* serovar Wien related to porin loss and CMY-4 beta-lactamase production. Antimicrob. Agents Chemother. 47:1165–1168.
10. Ayres HM, Payne DN, Furr JR, Russell AD. 1998. Effect of permeabilizing agents on antibacterial activity against a simple *Pseudomonas aeruginosa* biofilm. Lett. Appl. Microbiol. 27:79–82.
11. Ball PR, Shales SW, Chopra I. 1980. Plasmid-mediated tetracycline resistance in *Escherichia coli* involves increased efflux of the antibiotic. Biochem. Biophys. Res. Commun. 93:74–81.
12. Ballestero S, et al. 1996. Carbapenem resistance in *Pseudomonas aeruginosa* from cystic fibrosis patients. J. Antimicrob. Chemother. 38:39–45.
13. Baquero F. 2001. Low-level antibacterial resistance: a gateway to clinical resistance. Drug Resist. Updat. 4:93–105.
14. Baranova N, Nikaido H. 2002. The BaeSR two-component regulatory system activates transcription of the *yegMNOB (mdtABCD)* transporter gene cluster in *Escherichia coli* and increases its resistance to novobiocin and deoxycholate. J. Bacteriol. 184:4168–4176.
15. Barbosa TM, Levy SB. 2000. Differential expression of over 60 chromosomal genes in *Escherichia coli* by constitutive expression of MarA. J. Bacteriol. 182:3467–3474.
16. Barclay ML, et al. 1996. Adaptive resistance to tobramycin in *Pseudomonas aeruginosa* lung infection in cystic fibrosis. J. Antimicrob. Chemother. 37:1155–1164.
17. Bean DC, Wareham DW. 2009. Paradoxical effect of 1-(1-naphthylmethyl)-piperazine on resistance to tetracyclines in multidrug-resistant *Acinetobacter baumannii*. J. Antimicrob. Chemother. 63:349–352.
18. Beceiro A, et al. 2011. False extended-spectrum beta-lactamase phenotype in clinical isolates of *Escherichia coli* associated with increased expression of OXA-1 or TEM-1 penicillases and loss of porins. J. Antimicrob. Chemother. 66:2006–2010.
19. Begic S, Worobec EA. 2006. Regulation of *Serratia marcescens ompF* and

Fernández and Hancock

ompC porin genes in response to osmotic stress, salicylate, temperature and pH. Microbiology 152:485–491.

20. Begic S, Worobec EA. 2008. Characterization of the *Serratia marcescens* SdeCDE multidrug efflux pump studied via gene knockout mutagenesis. Can. J. Microbiol. 54:411–416.
21. Begum A, et al. 2005. Gene cloning and characterization of four MATE family multidrug efflux pumps from *Vibrio cholerae* non-O1. Microbiol. Immunol. 49:949–957.
22. Bina JE, Provenzano D, Wang C, Bina XR, Mekalanos JJ. 2006. Characterization of the *Vibrio cholerae vexAB* and *vexCD* efflux systems. Arch. Microbiol. 186:171–181.
23. Bore E, et al. 2007. Adapted tolerance to benzalkonium chloride in *Escherichia coli* K-12 studied by transcriptome and proteome analyses. Microbiology 153:935–946.
24. Boutoille D, et al. 2004. Detection of an IS21 insertion sequence in the *mexR* gene of *Pseudomonas aeruginosa* increasing beta-lactam resistance. FEMS Microbiol. Lett. 230:143–146.
25. Briers Y, Walmagh M, Lavigne R. 2011. Use of bacteriophage endolysin EL188 and outer membrane permeabilizers against *Pseudomonas aeruginosa*. J. Appl. Microbiol. 110:778–785.
26. Brooun A, Tomashek JJ, Lewis K. 1999. Purification and ligand binding of EmrR, a regulator of a multidrug transporter. J. Bacteriol. 181:5131–5133.
27. Butaye P, Cloeckaert A, Schwarz S. 2003. Mobile genes coding for efflux-mediated antimicrobial resistance in Gram-positive and Gram-negative bacteria. Int. J. Antimicrob. Agents 22:205–210.
28. Cao L, Srikumar R, Poole K. 2004. MexAB-OprM hyperexpression in NalC-type multidrug-resistant *Pseudomonas aeruginosa*: identification and characterization of the *nalC* gene encoding a repressor of PA3720-PA3719. Mol. Microbiol. 53:1423–1436.
29. Castillo-Keller M, Vuong P, Misra R. 2006. Novel mechanism of *Escherichia coli* porin regulation. J. Bacteriol. 188:576–586.
30. Cattoir V, Poirel L, Nordmann P. 2008. Plasmid-mediated quinolone resistance pump QepA2 in an *Escherichia coli* isolate from France. Antimicrob. Agents Chemother. 52:3801–3804.
31. Chan YY, Chua KL. 2005. The *Burkholderia pseudomallei* BpeAB-OprB efflux pump: expression and impact on quorum sensing and virulence. J. Bacteriol. 187:4707–4719.
32. Chan YY, Tan TM, Ong YM, Chua KL. 2004. BpeAB-OprB, a multidrug efflux pump in *Burkholderia pseudomallei*. Antimicrob. Agents Chemother. 48:1128–1135.
33. Chancey ST, Zhou X, Zähner D, Stephens DS. 2011. Induction of efflux-mediated macrolide resistance in *Streptococcus pneumoniae*. Antimicrob. Agents Chemother. 55:3413–3422.
34. Chen J, Kuroda T, Huda MN, Mizushima T, Tsuchiya T. 2003. An RND-type multidrug efflux pump SdeXY from *Serratia marcescens*. J. Antimicrob. Chemother. 52:176–179.
35. Chen FJ, Lauderdale TL, Ho M, Lo HJ. 2003. The roles of mutations in *gyrA*, *parC*, and *ompK35* in fluoroquinolone resistance in *Klebsiella pneumoniae*. Microb. Drug Resist. 9:265–271.
36. Chen LR, Zhou HW, Cai JC, Zhang R, Chen GX. 2009. Combination of IMP-4 metallo-beta-lactamase production and porin deficiency causes carbapenem resistance in a *Klebsiella oxytoca* clinical isolate. Diagn. Microbiol. Infect. Dis. 65:163–167.
37. Chuanchuen R, et al. 2001. Cross-resistance between triclosan and antibiotics in *Pseudomonas aeruginosa* is mediated by multidrug efflux pumps: exposure of a susceptible mutant strain to triclosan selects nfxB mutants overexpressing MexCD-OprJ. Antimicrob. Agents Chemother. 45:428–432.
38. Chuanchuen R, Narasaki CT, Schweizer HP. 2002. The MexJK efflux pump of *Pseudomonas aeruginosa* requires OprM for antibiotic efflux but not for efflux of triclosan. J. Bacteriol. 184:5036–5044.
39. Clancy J, et al. 1996. Molecular cloning and functional analysis of a novel macrolide-resistance determinant, *mefA*, from *Streptococcus pyogenes*. Mol. Microbiol. 22:867–879.
40. Colmer JA, Fralick JA, Hamood AN. 1998. Isolation and characterization of a putative multidrug resistance pump from *Vibrio cholerae*. Mol. Microbiol. 27:63–72.
41. Cortez-Cordova J, Kumar A. 2011. Activity of the efflux pump inhibitor phenylalanine-arginine β-naphthylamide against the AdeFGH pump of *Acinetobacter baumannii*. Int. J. Antimicrob. Agents 37:420–424.
42. Coyne S, Rosenfeld N, Lambert T, Courvalin P, Périchon B. 2010. Overexpression of resistance-nodulation-cell division pump AdeFGH

confers multidrug resistance in *Acinetobacter baumannii*. Antimicrob. Agents Chemother. 54:4389–4393.
43. Crossman LC, et al. 2008. The complete genome, comparative and functional analysis of *Stenotrophomonas maltophilia* reveals an organism heavily shielded by drug resistance determinants. Genome Biol. 9:R74. doi:10.1186/gb-2008-9-4-r74.
44. Daigle DM, et al. 2007. Protein modulator of multidrug efflux gene expression in *Pseudomonas aeruginosa*. J. Bacteriol. 189:5441–5451.
45. Daikos GL, Lolans VT, Jackson GG. 1991. First-exposure adaptive resistance to aminoglycoside antibiotics in vivo with meaning for optimal clinical use. Antimicrob. Agents Chemother. 35:117–123.
46. Damier-Piolle L, Magnet S, Bremont S, Lambert T, Courvalin P. 2008. AdeIJK, a resistance-nodulation-cell division pump effluxing multiple antibiotics in *Acinetobacter baumannii*. Antimicrob. Agents Chemother. 52:557–562.
47. Danilchanka O, Mailaender C, Niederweis M. 2008. Identification of a novel multidrug efflux pump of *Mycobacterium tuberculosis*. Antimicrob. Agents Chemother. 52:2503–2511.
48. D’E, et al. 2001. A new mechanism of antibiotic resistance in Enterobacteriaceae induced by a structural modification of the major porin. Mol. Microbiol. 41:189–198.
49. De la Cruz MA, Calva E. 2010. The complexities of porin genetic regulation. J. Mol. Microbiol. Biotechnol. 18:24–36.
50. DeMarco CE, et al. 2007. Efflux-related resistance to norfloxacin, dyes, and biocides in bloodstream isolates of *Staphylococcus aureus*. Antimicrob. Agents Chemother. 51:3235–3239.
51. Deng X, et al. 2012. Expression of multidrug resistance efflux pump gene *norA* is iron responsive in *Staphylococcus aureus*. J. Bacteriol. 194:1753–1762.
52. Diggle SP, Winzer K, Lazdunski A, Williams P, Cámara M. 2002. Advancing the quorum in *Pseudomonas aeruginosa*: MvaT and the regulation of N-acylhomoserine lactone production and virulence gene expression. J. Bacteriol. 184:2576–2586.
53. Doménech-Sánchez A, et al. 2003. Role of *Klebsiella pneumoniae* OmpK35 porin in antimicrobial resistance. Antimicrob. Agents Chemother. 47:3332–3335.
54. Doumith M, Ellington MJ, Livermore DM, Woodford N. 2009. Molecular mechanisms disrupting porin expression in ertapenem-resistant *Klebsiella* and *Enterobacter spp.* clinical isolates from the UK. J. Antimicrob. Chemother. 63:659–667.
55. Dupont M, D’E, Chollet R, Chevalier J, Pagès JM. 2004. *Enterobacter aerogenes* OmpX, a cation-selective channel mar- and osmo-regulated. FEBS Lett. 569:27–30.
56. Dupont M, James CE, Chevalier J, Pagès JM. 2007. An early response to environmental stress involves regulation of OmpX and OmpF, two enterobacterial outer membrane pore-forming proteins. Antimicrob. Agents Chemother. 51:3190–3198.
57. El’Garch F, Jeannot K, Hocquet D, Llanes-Barakat Plesiat P. 2007. Cumulative effects of several nonenzymatic mechanisms on the resistance of *Pseudomonas aeruginosa* to aminoglycosides. Antimicrob. Agents Chemother. 51:1016–1021.
58. El Garch F, et al. 2010. Fluoroquinolones induce the expression of *patA* and *patB*, which encode ABC efflux pumps in *Streptococcus pneumoniae*. J. Antimicrob. Chemother. 65:2076–2082.
59. Fernández L, Breidenstein EBM, Hancock REW. 2011. Creeping baseline and adaptive resistance to antibiotics. Drug Resist. Updat. 14:1–21.
60. Fernandez L, Breidenstein EBM, Hancock REW. 2011. Importance of adaptive and stepwise changes in the rise and spread of antimicrobial resistance, p 43–72. In Keen PL, Montforts MHMM (ed), Antimicrobial resistance in the environment. John Wiley & Sons, Inc, Hoboken, NJ.
61. Fernández L, et al. 2010. Adaptive resistance to the “last hope” antibiotics polymyxin B and colistin in *Pseudomonas aeruginosa* is mediated by the novel two-component regulatory system ParR-ParS. Antimicrob. Agents Chemother. 54:3372–3382.
62. Floyd JL, Smith KP, Kumar SH, Floyd JT, Varela MF. 2010. LmrS is a multidrug efflux pump of the major facilitator superfamily from *Staphylococcus aureus*. Antimicrob. Agents Chemother. 54:5406–5412.
63. Fralick JA. 1996. Evidence that TolC is required for functioning of the Mar/AcrAB efflux pump of *Escherichia coli*. J. Bacteriol. 178:5803–5805.
64. Fraud S, Campigotto AJ, Chen Z, Poole K. 2008. MexCD-OprJ multidrug efflux system of *Pseudomonas aeruginosa*: involvement in chlorhexidine resistance and induction by membrane-damaging agents dependent upon the

AlgU stress response sigma factor. Antimicrob. Agents Chemother. 52: 4478–4482.

65. Fraud S, Poole K. 2011. Oxidative stress induction of the MexXY multidrug efflux genes and promotion of aminoglycoside resistance development in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 55: 1068–1074.

66. Frempong-Manso E, Raygada JL, Demarco CE, Seo SM, Kaatz GW. 2009. Inability of a reserpine-based screen to identify strains overexpressing efflux pump genes in clinical isolates of *Staphylococcus aureus*. Int. J. Antimicrob. Agents 33:360–363.

67. Fukuda H, Hosaka M, Hirai K, Iyobe S. 1990. New norfloxacin resistance gene in *Pseudomonas aeruginosa* PAO. Antimicrob. Agents Chemother. 34:1757–1761.

68. Fukuda H, et al. 1995. *nfxC*-type quinolone resistance in a clinical isolate of *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 39:790–792.

69. Garau G, Di Guilmi AM, Hall BG. 2005. Structure-based phylogeny of the metallo-β-lactamases. Antimicrob. Agents Chemother. 49:2778–2784.

70. Garrido MC, Herrero M, Kolter R, Moreno F. 1988. The export of the DNA replication inhibitor microcin B17 provides immunity for the host cell. EMBO J. 7:1853–1862.

71. Garvey MI, Piddock LJ. 2008. The efflux pump inhibitor reserpine selects multidrug-resistant *Streptococcus pneumoniae* strains that overexpress the ABC transporters PatA and PatB. Antimicrob. Agents Chemother. 52:1677–1685.

72. Giacometti A, et al. 2000. Comparative activities of polycationic peptides and clinically used antimicrobial agents against multidrug-resistant nosocomial isolates of *Acinetobacter baumannii*. J. Antimicrob. Chemother. 46:807–810.

73. Gill MJ, et al. 1998. Gonococcal resistance to beta-lactams and tetracycline involves mutation in loop 3 of the porin encoded at the *penB* locus. Antimicrob. Agents Chemother. 42:2799–2803.

74. Gillis RJ, et al. 2005. Molecular basis of azithromycin-resistant *Pseudomonas aeruginosa* biofilms. Antimicrob. Agents Chemother. 49:3858–3867.

75. Godreuil S, Galimand M, Gerbaud G, Jacquet C, Courvalin P. 2003. Efflux pump Lde is associated with fluoroquinolone resistance in *Listeria monocytogenes*. Antimicrob. Agents Chemother. 47:704–708.

76. Griffith JK, et al. 1992. Membrane transport proteins: implications of sequence comparisons. Curr. Opin. Cell Biol. 4:684–695.

77. Griffith DC, et al. 2006. Pharmacodynamics of levofloxacin against *Pseudomonas aeruginosa* with reduced susceptibility due to different efflux pumps: do elevated MICs always predict reduced *in vivo* efficacy? Antimicrob. Agents Chemother. 50:1628–1632.

78. Griffith D, Lomovskaya O, Lee V, Dudley M. 1999. Potentiation of levofloxacin by a broad-spectrum efflux pump inhibitor in mouse models of infection caused by *Pseudomonas aeruginosa*, abstr 1496. Abstr. 39th Intersci. Conf. Antimicrob. Agents Chemother., San Francisco, CA.

79. Gülmez D, et al. 2008. Carbapenem-resistant *Escherichia coli* and *Klebsiella pneumoniae* isolates from Turkey with OXA-48-like carbapenemases and outer membrane protein loss. Int. J. Antimicrob. Agents 31:523–526.

80. Hagman KE, et al. 1995. Resistance of *Neisseria gonorrhoeae* to antimicrobial hydrophobic agents is modulated by the *mtrRCDE* efflux system. Microbiology 141:611–622.

81. Hall BG, Barlow M. 2004. Evolution of the serine β-lactamases: past, present and future. Drug Resist. Updat. 7:111–123.

82. Hall MN, Silhavy TJ. 1981. The *ompB* locus and the regulation of the major outer membrane porin proteins of *Escherichia coli* K12. J. Mol. Biol. 146:23–43.

83. Hancock REW. 1984. Alterations in outer membrane permeability. Annu. Rev. Microbiol. 38:237–264.

84. Harder KJ, Nikaido H, Matsuhashi M. 1981. Mutants of *Escherichia coli* that are resistant to certain beta-lactam compounds lack the *ompF* porin. Antimicrob. Agents Chemother. 20:549–552.

85. Hartog E, Menashe O, Kler E, Yaron S. 2010. Salicylate reduces the antimicrobial activity of ciprofloxacin against extracellular *Salmonella enterica* serovar Typhimurium, but not against *Salmonella* in macrophages. J. Antimicrob. Chemother. 65:888–896.

86. He GX, et al. 2011. EmmdR, a new member of the MATE family of multidrug transporters, extrudes quinolones from *Enterobacter cloacae*. Arch. Microbiol. 193:759–765.

87. Helander IM, Alakomi HL, Latva-Kala K, Koski P. 1997. Polyethyleneimine is an effective permeabilizer of gram-negative bacteria. Microbiology 143:3193–3199.

88. Henrichfreise B, Wiegand I, Luhmer-Becker I, Wiedemann B. 2007. Development of resistance in wild-type and hypermutable *Pseudomonas aeruginosa* strains exposed to clinical pharmacokinetic profiles of meropenem and ceftazidime simulated *in vitro*. Antimicrob. Agents Chemother. 51:3642–3649.

89. Hernández A, Ruiz FM, Romero A, Martínez JL. 2011. The binding of triclosan to SmeT, the repressor of the multidrug efflux pump SmeDEF, induces antibiotic resistance in *Stenotrophomonas maltophilia*. PLoS Pathog. 7:e1002103. doi:10.1371/journal.ppat.1002103.

90. Hernould M, Gagne S, Fournier M, Quentin C, Arpin C. 2008. Role of the AheABC efflux pump in *Aeromonas hydrophila* intrinsic multidrug resistance. Antimicrob. Agents Chemother. 52:1559–1563.

91. Higgins PG, Schneiders T, Hamprecht A, Seifert H. 2010. *In vivo* selection of a missense mutation in *adeR* and conversion of the novel *blaOXA-164* gene into *blaOXA-58* in carbapenem-resistant *Acinetobacter baumannii* isolates from a hospitalized patient. Antimicrob. Agents Chemother. 54:5021–5027.

92. Hirai K, Suzue S, Irikura T, Iyobe S, Mitsuhashi S. 1987. Mutations producing resistance to norfloxacin in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 31:582–586.

93. Hirakata Y, et al. 2002. Multidrug efflux systems play an important role in the invasiveness of *Pseudomonas aeruginosa*. J. Exp. Med. 196:109–118.

94. Hocquet D, et al. 2003. MexXY-OprM efflux pump is necessary for adaptive resistance of *Pseudomonas aeruginosa* to aminoglycosides. Antimicrob. Agents Chemother. 47:1371–1375.

95. Hood MI, Jacobs AC, Sayood K, Dunman PM, Skaar EP. 2010. *Acinetobacter baumannii* increases tolerance to antibiotics in response to monovalent cations. Antimicrob. Agents Chemother. 54:1029–1041.

96. Huang H, Jeanteur D, Pattus F, Hancock REW. 1995. Membrane topology and site-specific mutagenesis of *Pseudomonas aeruginosa* porin OprD. Mol. Microbiol. 16:931–941.

97. Huang J, et al. 2004. Novel chromosomally encoded multidrug efflux transporter MdeA in *Staphylococcus aureus*. Antimicrob. Agents Chemother. 48:909–917.

98. Huda N, et al. 2003. Molecular cloning and characterization of an ABC multidrug efflux pump, VcaM, in non-O1 *Vibrio cholerae*. Antimicrob. Agents Chemother. 47:2413–2417.

99. Jaffe A, Chabbert YA, Semonin O. 1982. Role of porin proteins OmpF and OmpC in the permeation of beta-lactams. Antimicrob. Agents Chemother. 22:942–948.

100. Jeannot K, et al. 2008. Resistance and virulence of *Pseudomonas aeruginosa* clinical strains overproducing the MexCD-OprJ efflux pump. Antimicrob. Agents Chemother. 52:2455–2462.

101. Jetter M, Spaniol V, Troller R, Aebi C. 2010. Down-regulation of porin M35 in *Moraxella catarrhalis* by aminopenicillins and environmental factors and its potential contribution to the mechanism of resistance to aminopenicillins. J. Antimicrob. Chemother. 65:2089–2096.

102. Jin Y, Yang H, Qiao M, Jin S. 2011. MexT regulates the type III secretion system through MexS and PtrC in *Pseudomonas aeruginosa*. J. Bacteriol. 193:399–410.

103. Kadlec K, Kehrenberg C, Schwarz S. 2007. Efflux-mediated resistance to florfenicol and/or chloramphenicol in *Bordetella bronchiseptica*: identification of a novel chloramphenicol exporter. J. Antimicrob. Chemother. 59:191–196.

104. Kehrenberg C, Cloeckaert A, Klein G, Schwarz S. 2009. Decreased fluoroquinolone susceptibility in mutants of *Salmonella* serovars other than Typhimurium: detection of novel mutations involved in modulated expression of *ramA* and *soxS*. J. Antimicrob. Chemother. 64:1175–1180.

105. Kikuchi K, Bernard EM, Sadownik A, Regen SL, Armstrong D. 1997. Antimicrobial activities of squalamine mimics. Antimicrob. Agents Chemother. 41:1433–1438.

106. Kirby WMM. 1944. Extraction of a highly potent penicillin inactivator from penicillin-resistant staphylococci. Science 99:452–453.

107. Kobayashi N, Nishino K, Yamaguchi A. 2001. Novel macrolide-specific ABC-type efflux transporter in *Escherichia coli*. J. Bacteriol. 183:5639–5644.

108. Koebnik R, Locher KP, Van Gelder P. 2000. Structure and function of bacterial outer membrane proteins: barrels in a nutshell. Mol. Microbiol. 37:239–253.

Fernández and Hancock

109. Kohler T, Epp SF, Curty LK, Pechère JC. 1999. Characterization of MexT, the regulator of the MexE-MexF-OprN multidrug efflux system of *Pseudomonas aeruginosa*. J. Bacteriol. 181:6300–6305.

110. Kohler T, et al. 1997. Characterization of MexE-MexF-OprN, a positively regulated multidrug efflux system of *Pseudomonas aeruginosa*. Mol. Microbiol. 23:345–354.

111. Kohler T, van Delden C, Curty LK, Hamzehpour MM, Pechère JC. 2001. Overexpression of the MexEF-OprN multidrug efflux system affects cell-to-cell signaling in *Pseudomonas aeruginosa*. J. Bacteriol. 183: 5213–5222.

112. Kumar A, Chua KL, Schweizer HP. 2006. Method for regulated expression of single-copy efflux pump genes in a surrogate *Pseudomonas aeruginosa* strain: identification of the BpeEF-OprC chloramphenicol and trimethoprim efflux pump of *Burkholderia pseudomallei* 1026b. Antimicrob. Agents Chemother. 50:3460–3463.

113. Kumar A, Worobec EA. 2002. Fluoroquinolone resistance of *Serratia marcescens*: involvement of a proton gradient-dependent efflux pump. J. Antimicrob. Chemother. 50:593–596.

114. Kumar A, Worobec EA. 2005. Cloning, sequencing and characterization of the SdeAB efflux pump of *Serratia marcescens*. Antimicrob. Agents Chemother. 49:1495–1501.

115. Lacroix FJ, et al. 1996. *Salmonella typhimurium acrB*-like gene: identification and role in resistance to biliary salts and detergents and in murine infection. FEMS Microbiol. Lett. 135:161–167.

116. Lamarche MG, Déziel E. 2011. MexEF-OprN efflux pump exports the *Pseudomonas* quinolone signal (PQS) precursor HHQ (4-hydroxy-2-heptylquinoline). PLoS One 6:e24310. doi:10.1371/journal.pone.0024310.

117. Landman D, Bratu S, Quale J. 2009. Contribution of OmpK36 to carbapenem susceptibility in KPC-producing *Klebsiella pneumoniae*. J. Med. Microbiol. 58:1303–1308.

118. Lebel S, Bouttier S, Lambert T. 2004. The *cme* gene of *Clostridium difficile* confers multidrug resistance in *Enterococcus faecalis*. FEMS Microbiol. Lett. 238:93–100.

119. Leblanc SK, Oates CW, Raivio TL. 2011. Characterization of the induction and cellular role of the BaeSR two-component envelope stress response of *Escherichia coli*. J. Bacteriol. 193:3367–3375.

120. Lee EW, Huda MN, Kuroda T, Mizushima T, Tsuchiya T. 2003. EfrAB, an ABC multidrug efflux pump in *Enterococcus faecalis*. Antimicrob. Agents Chemother. 47:3733–3738.

121. Levy SB, Marshall B. 2004. Antibacterial resistance worldwide: causes, challenges and responses. Nat. Med. 10:S122–129.

122. Levy SB, McMurry L. 1978. Plasmid-determined tetracycline resistance involves new transport systems for tetracycline. Nature 276:90–92.

123. Li C, et al. 1999. Antimicrobial activities of amine- and guanidine-functionalized cholic acid derivatives. Antimicrob. Agents Chemother. 43:1347–1349.

124. Li Y, et al. 2003. A new member of the tripartite multidrug efflux pumps, MexVW-OprM, in *Pseudomonas aeruginosa*. J. Antimicrob. Chemother. 52:572–575.

125. Li X-Z, Nikaido H. 2004. Efflux-mediated drug resistance in bacteria. Drugs 64:159–204.

126. Li XZ, Nikaido H, Poole K. 1995. Role of *mexA-mexB-oprM* in antibiotic efflux in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 39:1948–1953.

127. Li H, Park JT. 1999. The periplasmic murein peptide-binding protein MppA is a negative regulator of multiple antibiotic resistance in *Escherichia coli*. J. Bacteriol. 181:4842–4847.

128. Li XZ, Zhang L, Nikaido H. 2004. Efflux pump-mediated intrinsic drug resistance in *Mycobacterium smegmatis*. Antimicrob. Agents Chemother. 48:2415–2423.

129. Li X-Z, Zhang L, Poole K. 2002. SmeC, an outer membrane multidrug efflux protein of *Stenotrophomonas maltophilia*. Antimicrob. Agents Chemother. 46:333–343.

130. Limansky AS, Mussi MA, Viale AM. 2002. Loss of a 29-kilodalton outer membrane protein in *Acinetobacter baumannii* is associated with imipenem resistance. J. Clin. Microbiol. 40:4776–4778.

131. Lin J, Michel LO, Zhang Q. 2002. CmeABC functions as a multidrug efflux system in *Campylobacter jejuni*. Antimicrob. Agents Chemother. 46:2124–2131.

132. Lin XM, Yang JN, Peng XX, Li H. 2010. A novel negative regulation mechanism of bacterial outer membrane proteins in response to antibiotic resistance. J. Proteome Res. 9:5952–5959.

133. Liu J, Keelan P, Bennett PM, Enne VI. 2009. Characterization of a novel macrolide efflux gene, *mef(B)*, found linked to *sul3* in porcine *Escherichia coli*. J. Antimicrob. Chemother. 63:423–426.

134. Llanes C, et al. 2004. Clinical strains of *Pseudomonas aeruginosa* overproducing MexAB-OprM and MexXY efflux pumps simultaneously. Antimicrob. Agents Chemother. 48:1797–1802.

135. Llanes C, et al. 2011. Role of the MexEF-OprN efflux system in low-level resistance of *Pseudomonas aeruginosa* to ciprofloxacin. Antimicrob. Agents Chemother. 55:5676–5684.

136. Lomovskaya O, Kawai F, Matin A. 1996. Differential regulation of the *mcb* and *emr* operons of *Escherichia coli*: role of *mcb* in multidrug resistance. Antimicrob. Agents Chemother. 40:1050–1052.

137. Lomovskaya O, et al. 2001. Identification and characterization of inhibitors of multidrug resistance efflux pumps in *Pseudomonas aeruginosa*: novel agents for combination therapy. Antimicrob. Agents Chemother. 45:105–116.

138. Lou H, et al. 2011. Altered antibiotic transport in OmpC mutants isolated from a series of clinical strains of multi-drug resistant *E. coli*. PLoS One 6:e25825. doi:10.1371/journal.pone.0025825.

139. Lucas CE, Balthazar JT, Hagman KE, Shafer WM. 1997. The MtrR repressor binds the DNA sequence between the *mtrR* and *mtrC* genes of *Neisseria gonorrhoeae*. J. Bacteriol. 179:4123–4128.

140. Lynch MJ, Drusano GL, Mobley HL. 1987. Emergence of resistance to imipenem in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 31:1892–1896.

141. Ma D, Alberti M, Lynch C, Nikaido H, Hearst JE. 1996. The local repressor AcrR plays a modulating role in the regulation of *acrAB* genes of *Escherichia coli* by global stress signals. Mol. Microbiol. 19:101–112.

142. Ma D, et al. 1993. Molecular cloning and characterization of *acrA* and *acrE* genes of *Escherichia coli*. J. Bacteriol. 175:6299–6313.

143. Ma D, Cook DN, Hearst JE, Nikaido H. 1994. Efflux pumps and drug resistance in Gram-negative bacteria. Trends Microbiol. 2:489–493.

144. Magnet S, Courvalin P, Lambert T. 2001. Resistance-nodulation-cell division-type efflux pump involved in aminoglycoside resistance in *Acinetobacter baumannii* strain BM4454. Antimicrob. Agents Chemother. 45: 3375–3380.

145. Maneewannakul K, Levy SB. 1996. Identification for *mar* mutants among quinolone-resistant clinical isolates of *Escherichia coli*. Antimicrob. Agents Chemother. 40:1695–1698.

146. Maniati M, et al. 2007. A highly carbapenem-resistant *Pseudomonas aeruginosa* isolate with a novel *blaVIM-4/blaP1b* integron overexpresses two efflux pumps and lacks OprD. J. Antimicrob. Chemother. 60:132–135.

147. Mao W, Warren MS, Lee A, Mistry A, Lomovskaya O. 2001. MexXY-OprM efflux pump is required for antagonism of aminoglycosides by divalent cations in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 45:2001–2007.

148. Marger MD, Saier MH, Jr. 1993. A major superfamily of transmembrane facilitators that catalyse uniport, symport and antiport. Trends Biochem. Sci. 18:13–20.

149. Markham PN, Westhaus E, Klyachko K, Johnson ME, Neyfakh AA. 1999. Multiple novel inhibitors of the NorA multidrug transporter of *Staphylococcus aureus*. Antimicrob. Agents Chemother. 43:2404–2408.

150. Marrer E, et al. 2006. Involvement of the putative ATP-dependent efflux proteins PatA and PatB in fluoroquinolone resistance of a multidrug-resistant mutant of *Streptococcus pneumoniae*. Antimicrob. Agents Chemother. 50:685–693.

151. Martinez A, Lin J. 2006. Effect of an efflux pump inhibitor on the function of the multidrug efflux pump CmeABC and antimicrobial resistance in *Campylobacter*. Foodborne Pathog. Dis. 3:393–402.

152. Maseda H, Saito K, Nakajima A, Nakae T. 2000. Variation of the *mexT* gene, a regulator of the MexEF-OprN efflux pump expression in wild-type strains of *Pseudomonas aeruginosa*. FEMS Microbiol. Lett. 192:107–112.

153. Maseda H, et al. 2004. Enhancement of the *mexAB-oprM* efflux pump expression by a quorum-sensing autoinducer and its cancellation by a regulator, MexT, of the *mexEF-oprN* efflux pump operon in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 48:1320–1328.

154. Masi M, Pages JM, Villard C, Pradel E. 2005. The *eefABC* multidrug efflux pump operon is repressed by HNS in *Enterobacter aerogenes*. J. Bacteriol. 187:3894–3897.

155. Masuda N, Sakagawa E, Ohya S, Gotoh N, Nishino T. 2001. Hyper-susceptibility of the *Pseudomonas aeruginosa nfxB* mutant to beta-

lactams due to reduced expression of the *ampC* beta-lactamase. Antimicrob. Agents Chemother. 45:1284–1286.

156. Matsuo T, et al. 2008. SmdAB, a heterodimeric ABC-type multidrug efflux pump, in *Serratia marcescens*. J. Bacteriol. 190:648–654.

157. Matsuo Y, Eda S, Gotoh N, Yoshihara E, Nakae T. 2004. MexZ-mediated regulation of *mexXY* multidrug efflux pump expression in *Pseudomonas aeruginosa* by binding on the *mexZ-mexX* intergenic DNA. FEMS Microbiol. Lett. 238:23–28.

158. Mazzariol A, Zuliani J, Cornaglia G, Rossolini GM, Fontana R. 2002. AcrAB efflux system: expression and contribution to fluoroquinolone resistance in *Klebsiella* spp. Antimicrob. Agents Chemother. 46:3984–3986.

159. McAleese F, et al. 2005. A novel MATE family efflux pump contributes to the reduced susceptibility of laboratory-derived *Staphylococcus aureus* mutants to tigecycline. Antimicrob. Agents Chemother. 49:1865–1871.

160. McMurry L, Petrucci RE, Jr, Levy SB. 1980. Active efflux of tetracycline encoded by four genetically different tetracycline resistance determinants in *Escherichia coli*. Proc. Natl. Acad. Sci. U. S. A. 77:3974–3977.

161. Medeiros AA, O’Brien TF, Rosenberg EY, Nikaido H. 1987. Loss of OmpC porin in a strain of *Salmonella typhimurium* causes increased resistance to cephalosporins during therapy. J. Infect. Dis. 156:751–757.

162. Mima T, Schweizer HP. 2010. The BpeAB-OprB efflux pump of *Burkholderia pseudomallei* 1026b does not play a role in quorum sensing, virulence factor production, or extrusion of aminoglycosides but is a broad-spectrum drug efflux system. Antimicrob. Agents Chemother. 54:3113–3120.

163. Minato Y, Shahcheraghi F, Ogawa W, Kuroda T, Tsuchiya T. 2008. Functional gene cloning and characterization of the SsmE multidrug efflux pump from *Serratia marcescens*. Biol. Pharm. Bull. 31:516–519.

164. Mine T, Morita Y, Kataoka A, Mizushima T, Tsuchiya T. 1999. Expression in *Escherichia coli* of a new multidrug efflux pump, MexXY, from *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 43:415–417.

165. Moore RA, DeShazer D, Reckseidler S, Weissman A, Woods DE. 1999. Efflux-mediated aminoglycoside and macrolide resistance in *Burkholderia pseudomallei*. Antimicrob. Agents Chemother. 43:465–470.

166. Morita Y, et al. 1998. NorM, a putative multidrug efflux protein, of *Vibrio parahaemolyticus* and its homolog in *Escherichia coli*. Antimicrob. Agents Chemother. 42:1778–1782.

167. Moussatova A, Kandt C, O’Mara ML, Tieleman DP. 2008. ATP-binding cassette transporters in *Escherichia coli*. Biochim. Biophys. Acta 1778:1757–1771.

168. Mulet X, et al. 2009. Azithromycin in *Pseudomonas aeruginosa* biofilms: bactericidal activity and selection of *nfxB* mutants. Antimicrob. Agents Chemother. 53:1552–1560.

169. Muller C, Plésiat P, Jeannot K. 2011. A two-component regulatory system interconnects resistance to polymyxins, aminoglycosides, fluoroquinolones, and β-lactams in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 55:1211–1221.

170. Nagakubo S, Nishino K, Hirata T, Yamaguchi A. 2002. The putative response regulator BaeR stimulates multidrug resistance of *Escherichia coli* via a novel multidrug exporter system, MdtABC. J. Bacteriol. 184:4161–4167.

171. Nagnant C, et al. 2010. Interaction between tobramycin and CSA-13 on clinical isolates of *Pseudomonas aeruginosa* in a model of young and mature biofilms. Appl. Microbiol. Biotechnol. 88:251–263.

172. Nair BM, Cheung KJ, Jr, Griffith A, Burns JL. 2004. Salicylate induces an antibiotic efflux pump in *Burkholderia cepacia* complex genomovar III (*B. cenocepacia*). J. Clin. Invest. 113:464–473.

173. Nakae T. 1976. Identification of the outer membrane protein of *E. coli* that produces transmembrane channels in reconstituted vesicle membranes. Biochem. Biophys. Res. Commun. 71:877–884.

174. Nakayama K, et al. 2004. MexAB-OprM specific efflux pump inhibitors in *Pseudomonas aeruginosa*. Part 3: optimization of potency in the pyridopyrimidine series through the application of a pharmacophore model. Bioorg. Med. Chem. Lett. 14:475–479.

175. Nelson ML, Levy SB. 1999. Reversal of tetracycline resistance mediated by different bacterial tetracycline resistance determinants by an inhibitor of the Tet(B) antiport protein. Antimicrob. Agents Chemother. 43:1719–1724.

176. Neyfakh AA. 1992. The multidrug efflux transporter of *Bacillus subtilis* is a structural and functional homolog of the *Staphylococcus* NorA protein. Antimicrob. Agents Chemother. 36:484–485.


177. Nikaido H, Basina M, Nguyen V, Rosenberg EY. 1998. Multidrug efflux pump AcrAB of *Salmonella typhimurium* excretes only those beta-lactam antibiotics containing lipophilic side chains. J. Bacteriol. 180:4686–4692.

178. Nikaido H, Pagès JM. 2012. Broad-specificity efflux pumps and their role in multidrug resistance of Gram-negative bacteria. FEMS Microbiol. Rev. 36:340–363.

179. Nikaido E, Shirosaka I, Yamaguchi A, Nishino K. 2011. Regulation of the AcrAB multidrug efflux pump in *Salmonella enterica* serovar Typhimurium in response to indole and paraquat. Microbiology 157:648–655.

180. Nikaido E, Yamaguchi A, Nishino K. 2008. AcrAB multidrug efflux pump regulation in *Salmonella enterica* serovar Typhimurium by RamA in response to environmental signals. J. Biol. Chem. 283:24245–24253.

181. Nilsen IW, Bakke I, Vader A, Olsvik O, El-Gewely MR. 1996. Isolation of *cmr*, a novel *Escherichia coli* chloramphenicol resistance gene encoding a putative efflux pump. J. Bacteriol. 178:3188–3193.

182. Nishino K, Latifi T, Groisman EA. 2006. Virulence and drug resistance roles of multidrug efflux systems of *Salmonella enterica* serovar Typhimurium. Mol. Microbiol. 59:126–141.

183. Nishino K, Yamaguchi A. 2002. EvgA of the two-component signal transduction system modulates production of the YhiUV multidrug transporter in *Escherichia coli*. J. Bacteriol. 184:2319–2323.

184. Nishioka T, Ogawa W, Kuroda T, Katsu T, Tsuchiya T. 2009. Gene cloning and characterization of EfmA, a multidrug efflux pump, from *Enterococcus faecium*. Biol. Pharm. Bull. 32:483–488.

185. Ochs MM, Bains M, Hancock REW. 2000. Role of putative loops 2 and 3 in imipenem passage through the specific porin OprD of *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 44:1983–1985.

186. Ochs MM, Lu CD, Hancock REW, Abdelal AT. 1999. Amino acid-mediated induction of the basic amino acid-specific outer membrane porin OprD from *Pseudomonas aeruginosa*. J. Bacteriol. 181:5426–5432.

187. Ochs MM, McCusker MP, Bains M, Hancock REW. 1999. Negative regulation of the *Pseudomonas aeruginosa* outer membrane porin OprD selective for imipenem and basic amino acids. Antimicrob. Agents Chemother. 43:1085–1090.

188. Okazaki T, Hirai K. 1992. Cloning and nucleotide sequence of the *Pseudomonas aeruginosa nfxB* gene, conferring resistance to new quinolones. FEMS Microbiol. Lett. 76:197–202.

189. Olesky M, Hobbs M, Nicholas RA. 2002. Identification and analysis of amino acid mutations in porin IB that mediate intermediate-level resistance to penicillin and tetracycline in *Neisseria gonorrhoeae*. Antimicrob. Agents Chemother. 46:2811–2820.

190. Olesky M, Zhao S, Rosenberg RL, Nicholas RA. 2006. Porin-mediated antibiotic resistance in *Neisseria gonorrhoeae*: ion, solute, and antibiotic permeation through PIB proteins with *penB* mutations. J. Bacteriol. 188:2300–2308.

191. Olliver A, VallÉ M, Chaslus-Dancla E, Cloeckaert A. 2005. Overexpression of the multidrug efflux operon *acrEF* by insertional activation with IS1 or IS10 elements in *Salmonella enterica* serovar Typhimurium DT204 *acrB* mutants selected with fluoroquinolones. Antimicrob. Agents Chemother. 49:289–301.

192. Oteo J, et al. 2008. Emergence of imipenem resistance in clinical *Escherichia coli* during therapy. Int. J. Antimicrob. Agents 32:534–537.

193. Pagel M, et al. 2007. Phenotypic characterization of pore mutants of the *Vibrio cholerae* porin OmpU. J. Bacteriol. 189:8593–8600.

194. Pamp SJ, Gjermansen M, Johansen HK, Tolker-Nielsen T. 2008. Tolerance to the antimicrobial peptide colistin in *Pseudomonas aeruginosa* biofilms is linked to metabolically active cells, and depends on the *pmr* and *mexAB-oprM* genes. Mol. Microbiol. 68:223–240.

195. Park YJ, et al. 2009. Accumulation of plasmid-mediated fluoroquinolone resistance genes, *qepA* and *qnrS1*, in *Enterobacter aerogenes* co-producing RmtB and class A β-lactamase LAP-1. Ann. Clin. Lab. Sci. 39:55–59.

196. Pasca MR, et al. 2004. Rv2686c-Rv2687c-Rv2688c, an ABC fluoroquinolone efflux pump in *Mycobacterium tuberculosis*. Antimicrob. Agents Chemother. 48:3175–3178.

197. Paulsen IT, Skurray RA. 1993. Topology, structure and evolution of two families of proteins involved in antibiotic and antiseptic resistance in eukaryotes and prokaryotes—an analysis. Gene 124:1–11.

198. Paulsen IT, Sliwinski MK, Saier MH, Jr. 1998. Microbial genome analyses: global comparisons of transport capabilities based on phylogenies, bioenergetics and substrate specificities. J. Mol. Biol. 277:573–592.

199. Perez A, et al. 2007. Cloning, nucleotide sequencing, and analysis of the

Fernández and Hancock

AcrAB-TolC efflux pump of *Enterobacter cloacae* and determination of its involvement in antibiotic resistance in a clinical isolate. Antimicrob. Agents Chemother. 51:3247–3253.

200. Poole K, et al. 1996. Overexpression of the *mexC-mexD-oprJ* efflux operon in *nfxB*-type multidrug resistant strains. Mol. Microbiol. 21:713–724.

201. Poole K, Krebes K, McNally C, Neshat S. 1993. Multiple antibiotic resistance in *Pseudomonas aeruginosa*: evidence for involvement of an efflux operon. J. Bacteriol. 175:7363–7372.

202. Poole K, et al. 1996. Expression of the multidrug resistance operon *mexA-mexB-oprM* in *Pseudomonas aeruginosa*: *mexR* encodes a regulator of operon expression. Antimicrob. Agents Chemother. 40:2021–2028.

203. Pradel E, Pagès JM. 2002. The AcrAB-TolC efflux pump contributes to multidrug resistance in the nosocomial pathogen *Enterobacter aerogenes*. Antimicrob. Agents Chemother. 46:2640–2643.

204. Pratt LA, Hsing W, Gibson KE, Silhavy TJ. 1996. From acids to *osmZ*: multiple factors influence synthesis of the OmpF and OmpC porins in *Escherichia coli*. Mol. Microbiol. 20:911–917.

205. Rahman MM, et al. 2007. Molecular cloning and characterization of all RND-type efflux transporters in *Vibrio cholerae* non-O1. Microbiol. Immunol. 51:1061–1070.

206. Rahman SS, Simovic I, Gibbons S, Zloh M. 2011. *In silico* screening for antibiotic escort molecules to overcome efflux. J. Mol. Model. 17:2863–2872.

207. Rahmati S, Yang S, Davidson AL, Zechiedrich EL. 2002. Control of the AcrAB multidrug efflux pump by quorum-sensing regulator SdiA. Mol. Microbiol. 43:677–685.

208. Rella M, Haas D. 1982. Resistance of *Pseudomonas aeruginosa* PAO to nalidixic acid and low levels of beta-lactam antibiotics: mapping of chromosomal genes. Antimicrob. Agents Chemother. 22:242–249.

209. Renau TE, et al. 1999. Inhibitors of efflux pumps in *Pseudomonas aeruginosa* potentiate the activity of the fluoroquinolone antibacterial levofloxacin. J. Med. Chem. 42:4928–4931.

210. Riordan JT, et al. 2011. Alterations in the transcriptome and antibiotic susceptibility of *Staphylococcus aureus* grown in the presence of diclofenac. Ann. Clin. Microbiol. Antimicrob. 10:30. doi:10.1186/1476-0711-10-30.

211. Robertson GT, Doyle TB, Lynch AS. 2005. Use of an efflux-deficient *Streptococcus pneumoniae* strain panel to identify ABC-class multidrug transporters involved in intrinsic resistance to antimicrobial agents. Antimicrob. Agents Chemother. 49:4781–4783.

212. Rosenberg EY, Ma D, Nikaido H. 2000. AcrD of *Escherichia coli* is an aminoglycoside efflux pump. J. Bacteriol. 182:1754–1756.

213. Ross JI, et al. 1990. Inducible erythromycin resistance in staphylococci is encoded by a member of the ATP-binding transport super-gene family. Mol. Microbiol. 4:1207–1214.

214. Rouch DA, Cram DS, DiBerardino D, Littlejohn TG, Skurray RA. 1990. Efflux-mediated antiseptic resistance gene *qacA* from *Staphylococcus aureus*: common ancestry with tetracycline- and sugar-transport proteins. Mol. Microbiol. 4:2051–2062.

215. Rouquette-Loughlin CE, Balthazar JT, Shafer WM. 2005. Characterization of the MacA-MacB efflux system in *Neisseria gonorrhoeae*. J. Antimicrob. Chemother. 56:856–860.

216. Rouquette-Loughlin C, Dunham SA, Kuhn M, Balthazar JT, Shafer WM. 2003. The NorM efflux pump of *Neisseria gonorrhoeae* and *Neisseria meningitidis* recognizes antimicrobial cationic compounds. J. Bacteriol. 185:1101–1106.

217. Ruzin A, Visalli MA, Keeney D, Bradford PA. 2005. Influence of transcriptional activator RamA on expression of multidrug efflux pump AcrAB and tigecycline susceptibility in *Klebsiella pneumoniae*. Antimicrob. Agents Chemother. 49:1017–1022.

218. Sanbongi Y, et al. 2009. Classification of OprD sequence and correlation with antimicrobial activity of carbapenem agents in *Pseudomonas aeruginosa* clinical isolates collected in Japan. Microbiol. Immunol. 53:361–367.

219. Sánchez P, Alonso A, Martinez JL. 2002. Cloning and characterization of SmeT, a repressor of the *Stenotrophomonas maltophilia* multidrug efflux pump SmeDEF. Antimicrob. Agents Chemother. 46:3386–3393.

220. Sánchez P, et al. 2002. Fitness of *in vitro* selected *Pseudomonas aeruginosa nalB* and *nfxB* multidrug resistant mutants. J. Antimicrob. Chemother. 50:657–664.

221. Sanchez L, Pan W, Vinas M, Nikaido H. 1997. The *acrAB* homolog of *Haemophilus influenzae* codes for a functional multidrug efflux pump. J. Bacteriol. 179:6855–6857.

222. Savage PB, Li C. 2000. Cholic acid derivatives: novel antimicrobials. Expert. Opin. Invest. Drugs 9:263–272.

223. Sawada I, Maseda H, Nakae T, Uchiyama H, Nomura N. 2004. A quorum-sensing autoinducer enhances the *mexAB-oprM* efflux-pump expression without the MexR-mediated regulation in *Pseudomonas aeruginosa*. Microbiol. Immunol. 48:435–439.

224. Scott MG, Yan H, Hancock REW. 1999. Biological properties of structurally related α-helical cationic antimicrobial peptides. Infect. Immun. 67:2005–2009.

225. Shahcheraghi F, et al. 2007. Molecular cloning and characterization of a multidrug efflux pump, SmfY, from *Serratia marcescens*. Biol. Pharm. Bull. 30:798–800.

226. Shen Z, Pu XY, Zhang Q. 2011. Salicylate functions as an efflux pump inducer and promotes the emergence of fluoroquinolone-resistant *Campylobacter jejuni* mutants. Appl. Environ. Microbiol. 77:7128–7133.

227. Siddiqi N, et al. 2004. *Mycobacterium tuberculosis* isolate with a distinct genomic identity overexpresses a Tap-like efflux pump. Infection 32:109–111.

228. Singh AK, Haldar R, Mandal D, Kundu M. 2006. Analysis of the topology of *Vibrio cholerae* NorM and identification of amino acid residues involved in norfloxacin resistance. Antimicrob. Agents Chemother. 50:3717–3723.

229. Singh KV, Malathum K, Murray BE. 2001. Disruption of an *Enterococcus faecium* species-specific gene, a homologue of acquired macrolide resistance genes of staphylococci, is associated with an increase in macrolide susceptibility. Antimicrob. Agents Chemother. 45:263–266.

230. Sobel ML, Hocquet D, Cao L, Plesiat P, Poole K. 2005. Mutations in PA3574 (*malD*) lead to increased MexAB-OprM expression and multidrug resistance in laboratory and clinical isolates of *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 49:1782–1786.

231. Song W, et al. 2009. *In vivo* selection of carbapenem-resistant *Klebsiella pneumoniae* by OmpK36 loss during meropenem treatment. Diagn. Microbiol. Infect. Dis. 65:447–449.

232. Srikumar R, Paul CJ, Poole K. 2000. Influence of mutations in the *mexR* repressor gene on expression of the MexA-MexB-OprM multidrug efflux system of *Pseudomonas aeruginosa*. J. Bacteriol. 182:1410–1414.

233. Srinivasan VB, Rajamohan G, Gebreyes WA. 2009. Role of AbeS, a novel efflux pump of the SMR family of transporters, in resistance to antimicrobial agents in *Acinetobacter baumannii*. Antimicrob. Agents Chemother. 53:5312–5316.

234. Stickland HG, Davenport PW, Lilley KS, Griffin JL, Welch M. 2010. Mutation of *nfxB* causes global changes in the physiology and metabolism of *Pseudomonas aeruginosa*. J. Proteome Res. 9:2957–2967.

235. Stürenburg E, Sobottka I, Mack D, Laufs R. 2002. Cloning and sequencing of *Enterobacter aerogenes* OmpC-type osmoporin linked to carbapenem resistance. Int. J. Med. Microbiol. 291:649–654.

236. Su XZ, Chen J, Mizushima T, Kuroda T, Tsuchiya T. 2005. AbeM, an H+-coupled *Acinetobacter baumannii* multidrug efflux pump belonging to the MATE family of transporters. Antimicrob. Agents Chemother. 49:4362–4364.

237. Suh B, et al. 2010. Outbreak of meropenem-resistant *Serratia marcescens* comedited by chromosomal AmpC beta-lactamase overproduction and outer membrane protein loss. Antimicrob. Agents Chemother. 54:5057–5061.

238. Tavío MM, et al. 2004. Enhanced active efflux, repression of porin synthesis and development of Mar phenotype by diazepam in two enterobacteria strains. J. Med. Microbiol. 53:1119–1122.

239. Tennent JM, et al. 1989. Physical and biochemical characterization of the *qacA* gene encoding antiseptic and disinfectant resistance in *Staphylococcus aureus*. J. Gen. Microbiol. 135:1–10.

240. Thanassi DG, Cheng LW, Nikaido H. 1997. Active efflux of bile salts by *Escherichia coli*. J. Bacteriol. 179:2512–2518.

241. Thiolas A, Bornet C, Davin-Régli A, Pagès JM, Bollet C. 2004. Resistance to imipenem, cefepime, and cefpirome associated with mutation in Omp36 osmoporin of *Enterobacter aerogenes*. Biochem. Biophys. Res. Commun. 317:851–856.

242. Tomás M, et al. 2010. Efflux pumps, OprD porin, AmpC beta-lactamase, and multiresistance in *Pseudomonas aeruginosa* isolates from cystic fibrosis patients. Antimicrob. Agents Chemother. 54:2219–2224.

243. Toro CS, Lobos SR, Calderón I, Rodríguez M, Mora GC. 1990. Clinical isolate of a porinless *Salmonella typhi* resistant to high levels of chloramphenicol. Antimicrob. Agents Chemother. 34:1715–1719.

244. Truong-Bolduc QC, et al. 2011. Implication of the NorB efflux pump in

the adaptation of *Staphylococcus aureus* to growth at acid pH and in resistance to moxifloxacin. Antimicrob. Agents Chemother. 55:3214–3219.

245. Truong-Bolduc QC, Dunman PM, Strahilevitz J, Projan SJ, Hooper DC. 2005. MgrA is a multiple regulator of two new efflux pumps in *Staphylococcus aureus*. J. Bacteriol. 187:2395–2405.

246. Truong-Bolduc QC, Hooper DC. 2007. The transcriptional regulators NorG and MgrA modulate resistance to both quinolones and β-lactams in *Staphylococcus aureus*. J. Bacteriol. 189:2996–3005.

247. Truong-Bolduc QC, Strahilevitz J, Hooper DC. 2006. NorC, a new efflux pump regulated by MgrA of *Staphylococcus aureus*. Antimicrob. Agents Chemother. 50:1104–1107.

248. Tsubery H, Ofek I, Cohen S, Fridkin M. 2000. Structure-function studies of polymyxin B nonapeptide: implications to sensitization of gram-negative bacteria. J. Med. Chem. 43:3085–3092.

249. Ubukata K, Itoh-Yamashita N, Konno M. 1989. Cloning and expression of the *norA* gene for fluoroquinolone resistance in *Staphylococcus aureus*. Antimicrob. Agents Chemother. 33:1535–1539.

250. Vaara M. 1992. Agents that increase the permeability of the outer membrane. Microbiol. Rev. 56:395–411.

251. Venkatraman J, Nagana Gowda GA, Balaram P. 2002. Structural analysis of synthetic peptide fragments from EmrE, a multidrug resistance protein, in a membrane-mimetic environment. Biochemistry 41:6631–6639.

252. Vettoretti L, et al. 2009. Efflux unbalance in *Pseudomonas aeruginosa* isolates from cystic fibrosis patients. Antimicrob. Agents Chemother. 53:1987–1997.

253. Vila J, Martí S, Sánchez-Céspedes J. 2007. Porins, efflux pumps and multidrug resistance in *Acinetobacter baumannii*. J. Antimicrob. Chemother. 59:1210–1215.

254. Visalli MA, Murphy E, Projan SJ, Bradford PA. 2003. AcrAB multidrug efflux pump is associated with reduced levels of susceptibility to tigecycline (GAR-936) in *Proteus mirabilis*. Antimicrob. Agents Chemother. 47:665–669.

255. Viveiros M, et al. 2007. Antibiotic stress, genetic response and altered permeability of *E. coli*. PLoS One 2:e365. doi:10.1371/journal.pone.0000365.

256. Vogne C, Aires JR, Bailly C, Hocquet D, Plésiat P. 2004. Role of the multidrug efflux system MexXY in the emergence of moderate resistance to aminoglycosides among *Pseudomonas aeruginosa* isolates from patients with cystic fibrosis. Antimicrob. Agents Chemother. 48:1676–1680.

257. Wang H, et al. 2010. *oprM* as a new target for reversion of multidrug resistance in *Pseudomonas aeruginosa* by antisense phosphorothioate oligodeoxynucleotides. FEMS Immunol. Med. Microbiol. 60:275–282.

258. Weindorf H, Schmidt H, Martin HH. 1998. Contribution of overproduced chromosomal beta-lactamase and defective outer membrane

porins to resistance to extended-spectrum beta-lactam antibiotics in *Serratia marcescens*. J. Antimicrob. Chemother. 41:189–195.

259. Westbrock-Wadman S, et al. 1999. Characterization of a *Pseudomonas aeruginosa* efflux pump contributing to aminoglycoside impermeability. Antimicrob. Agents Chemother. 43:2975–2983.

260. Westfall LW, et al. 2006. *mvaT* mutation modifies the expression of the *Pseudomonas aeruginosa* multidrug efflux operon *mexEF-oprN*. FEMS Microbiol. Lett. 255:247–254.

261. Wolter DJ, Hanson ND, Lister PD. 2004. Insertional inactivation of *oprD* in clinical isolates of *Pseudomonas aeruginosa* leading to carbapenem resistance. FEMS Microbiol. Lett. 236:137–143.

262. Woolridge DP, et al. 1997. Efflux of the natural polyaminespermidine facilitated by the *Bacillus subtilis* multidrug transporter Blt. J. Biol. Chem. 272:8864–8866.

263. Yamada Y, Hideka K, Shiota S, Kuroda T, Tsuchiya T. 2006. Gene cloning and characterization of SdrM, a chromosomally encoded multidrug efflux pump, from *Staphylococcus aureus*. Biol. Pharm. Bull. 29:554–556.

264. Yamamoto M, et al. 2009. Role of MexZ and PA5471 in transcriptional regulation of *mexXY* in *Pseudomonas aeruginosa*. Microbiology 155:3312–3321.

265. Yamane K, et al. 2007. New plasmid-mediated fluoroquinolone efflux pump, QepA, found in an *Escherichia coli* clinical isolate. Antimicrob. Agents Chemother. 51:3354–3360.

266. Yang S, Clayton SR, Zechiedrich EL. 2003. Relative contributions of the AcrAB, MdfA and NorE efflux pumps to quinolone resistance in *Escherichia coli*. J. Antimicrob. Chemother. 51:545–556.

267. Yassein MA, Ewis HE, Lu C-D, Abdelal AT. 2002. Molecular cloning and characterization of the *Salmonella enterica* serovar Paratyphi B *rma* gene, which confers multiple drug resistance in *Escherichia coli*. Antimicrob. Agents Chemother. 44:10–13.

268. Zähner D, et al. 2010. Human antimicrobial peptide LL-37 induces MefE/Mel-mediated macrolide resistance in *Streptococcus pneumoniae*. Antimicrob. Agents Chemother. 54:3516–3519.

269. Zhang DF, Jiang B, Xiang ZM, Wang SY. 2008. Functional characterisation of altered outer membrane proteins for tetracycline resistance in *Escherichia coli*. Int. J. Antimicrob. Agents 32:315–319.

270. Zhang L, Li X-Z, Poole K. 2001. SmeDEF multidrug efflux pump contributes to intrinsic multidrug resistance in *Stenotrophomonas maltophilia*. Antimicrob. Agents Chemother. 45:3497–3503.

271. Zhang L, Mah TF. 2008. Involvement of a novel efflux system in biofilm-specific resistance to antibiotics. J. Bacteriol. 190:4447–4452.

272. Ziha-Zarifi I, Llanes C, Köhler T, Pechere JC, Plesiat P. 1999. *In vivo* emergence of multidrug-resistant mutants of *Pseudomonas aeruginosa* overexpressing the active efflux system MexA-MexB-OprM. Antimicrob. Agents Chemother. 43:287–291.

---

Lucía Fernández completed her Ph.D. in Microbiology at the University of Oviedo, Spain, investigating the mechanisms of virulence of the fish pathogen *Yersinia ruckeri*. She is currently a postdoctoral researcher at the University of British Columbia (Canada). Her work focuses on understanding the regulation of adaptive resistance in the opportunistic pathogen *Pseudomonas aeruginosa*, with particular emphasis on the resistance induced by antimicrobial peptides.

Robert E. W. Hancock, O.C., O.B.C., F.R.S.C., is a Professor of Microbiology and Immunology at the University of British Columbia, Vancouver, Canada, and holds a Canada Research Chair in Health and Genomics. His research interests include small cationic peptides as novel antimicrobials and modulators of innate immunity, the development of novel treatments for antibiotic-resistant infections, the systems biology of innate immunity and *Pseudomonas aeruginosa*, and antibiotic uptake and resistance. He has published more than 570 papers and reviews, has 42 patents awarded, and is an ISI highly cited author in microbiology, with more than 30,000 citations. He has won several awards, including the Aventis Pharmaceuticals Award, the leading award for antimicrobial research, and Canada’s three top prizes for Health Research, and is an Officer of the Order of Canada.
