
# An enzyme-trap approach allows isolation of intermediates in cobalamin biosynthesis

Evelyne Deery¹, Susanne Schroeder¹,⁴, Andrew D Lawrence¹,⁴, Samantha L Taylor¹, Arefeh Seyedarabi², Jitka Waterman³, Keith S Wilson³, David Brown¹, Michael A Geeves¹, Mark J Howard¹, Richard W Pickersgill²* & Martin J Warren¹*

The biosynthesis of many vitamins and coenzymes has often proven difficult to elucidate owing to a combination of low abundance and kinetic lability of the pathway intermediates. Through a serial reconstruction of the cobalamin (vitamin B₁₂) pathway in *Escherichia coli* and by His tagging the terminal enzyme in the reaction sequence, we have observed that many unstable intermediates can be isolated as tightly bound enzyme-product complexes. Together, these approaches have been used to extract intermediates between precorrin-4 and hydrogenobyrinic acid in their free acid form and permitted the delineation of the overall reaction catalyzed by CobL, including the formal elucidation of precorrin-7 as a metabolite. Furthermore, a substrate-carrier protein, CobE, that can also be used to stabilize some of the transient metabolic intermediates and enhance their onward transformation, has been identified. The tight association of pathway intermediates with enzymes provides evidence for a form of metabolite channeling.

The isolation and characterization of biochemical pathways for vitamins and coenzymes has often proved technically challenging, mainly because of the low abundance and kinetic lability of the metabolic intermediates¹. Such is the case with the intermediates generated en route to the biosynthesis of adenosyl-cobalamin (vitamin B₁₂; Fig. 1), the antipernicious factor. This biosynthesis requires around thirty enzyme-mediated steps for the complete *de novo* construction of the coenzyme and represents one of the most intricate syntheses found in nature²,³. The biosynthesis of the corrin ring component is mediated via distinct aerobic and anaerobic pathways that oversee the transformation of the tetrapyrrole primogenitor, uroporphyrinogen III, into cobyric acid.

Corrin ring synthesis involves considerable decoration of the tetapyrrole framework and includes the chemically challenging process of ring contraction to expel the C20 *meso* position to allow for the tighter coordination of the centrally chelated cobalt ion²⁻⁴. Overall, the transformation of uroporphyrinogen III into hydrogenobyrinic acid (HBA), the first stable intermediate along the pathway⁵,⁶, involves eight S-adenosyl-L-methionine (SAM)-dependent methylations, extrusion of an integral macrocyclic carbon atom (ring contraction), decarboxylation of an acetate side chain and methyl migration³ (Fig. 1). There are still a number of specific questions to be addressed on the mechanisms underpinning many of these transformations as well as general questions about how pathways handle large numbers of labile intermediates and how such intricate pathways evolved.

This paper addresses some of these questions through a study of the aerobic cobalamin pathway, which is outlined in Figure 1. The intermediates of the pathway are termed precorrin-*n*, where *n* refers to the number of methyl groups that have been added to the framework⁷. The pathway is initiated by the action of a bis-methylase, CobA, which methylates uroporphyrinogen III at positions C2 and C7 to generate precorrin-2 (ref. 8). A further methylation at C20 by CobI synthesizes precorrin-3A⁹. This compound acts as the substrate for the monooxygenase CobG, which generates a tertiary alco-

hol at C20 and forms a γ-lactone with the acetate side chain on ring A¹⁰,¹¹ (precorrin-3B). Another methyltransferase, CobJ, methylates at C17 and mediates ring contraction by catalyzing a masked pinacol rearrangement¹¹,¹². Additional methylations by CobM and CobF at C11 and C1, the latter with loss of the extruded C20 position as acetic acid, results in the synthesis of precorrin-5 and precorrin-6A, respectively¹³,¹⁴. The macrocyclic ring system is reduced by the action of CobK, producing precorrin-6B¹⁵. This compound is the substrate for CobL, a multifunctional enzyme that promotes the decarboxylation of the acetic acid side chain at C12 and methylation at the northern (C5) and southern (C15) *meso* positions and in so doing makes precorrin-8 (ref. 16). Finally, CobH catalyzes a 1,5-sigmatropic rearrangement that involves the migration of the methyl group from C11 to C12, generating HBA¹⁷.

Most of the aforementioned intermediates have only been isolated after esterification and in very small quantities. To further investigate many of these highly unusual reactions, however, simple reproducible protocols need to be established for the production of these compounds. To achieve this, we attempted to build the pathway recombinantly in *Escherichia coli*, an organism that does not house the cobalamin biosynthetic pathway. We demonstrated that it was possible to generate HBA in *E. coli*⁶ by overproducing enzymes (CobA, CobI, CobG, CobJ, CobM, CobF, CobK, CobL and CobH) that manage the transformation of uroporphyrinogen III into the comparatively stable orange-colored HBA. This ring-contracted corrin intermediate can be purified from a cell lysate of this strain by ion exchange and reverse-phase chromatography⁶. However, attempts to produce the intervening intermediates between uroporphyrinogen III and HBA using the same strategy of building the pathway one enzyme at a time were not successful. It seems that the intermediates are too unstable to be purified.

In this paper, we outline a strategy for the isolation of five pathway intermediates between precorrin-4 and HBA in their free acid forms, stabilized as protein complexes. This approach resulted in the isolation and characterization of a new intermediate along the

---

¹School of Biosciences, University of Kent, Canterbury, Kent, UK.  
²School of Biological and Chemical Sciences, Queen Mary University of London, London, UK.  
³Structural Biology Laboratory, Department of Chemistry, University of York, York, UK.  
⁴These authors contributed equally to this work.  
*e-mail: m.j.warren@kent.ac.uk or r.w.pickersgill@qmul.ac.uk*

ARTICLE NATURE CHEMICAL BIOLOGY DOI:10.1038/NCHEMBIO.1086

![Chemical Structures](chemical_structures.png)

Figure 1 | Transformation of uroporphyrinogen III into HBA and its position in the pathway in relation to adenosylcobalamin biosynthesis. There are nine enzymes required for the synthesis of HBA from uroporphyrinogen III, six of which are SAM-dependent methyltransferases. The transformation of HBA into adenosylcobalamin requires a further eight steps. In the adenosylcobalamin structure, Ad stands for adenosine.

pathway, precorrin-7 (1); the use of this template for the combinatorial synthesis of a new pathway analog; and the identification of pathway carrier protein. The results have implications not only for understanding how complex pathways enhance and control the stability of labile intermediates by substrate channeling but also for how pathways can be manipulated for the synthesis of cofactor analogs.

### RESULTS

#### Enzymes as traps for pathway intermediates
In the absence of a downstream enzyme in a pathway, we wondered whether labile intermediates could be stabilized as enzyme-product complexes. To investigate this, we constructed a number of plasmids where the corrin pathway was built up in an ordered fashion but where the terminally encoded enzyme also contained a His tag. In this way, it should be possible to isolate quickly the terminal enzyme in the pathway and determine whether it contains any bound intermediate. Thus, the following constructs were made: *cobA-cobI-cobG-cobJ*<sup>*</sup>, *cobA-cobI-cobG-cobJ-cobM*<sup>*</sup>, *cobA-cobI-cobG-cobJ-cobM-cobF*<sup>*</sup>, *cobA-cobI-cobG-cobJ-cobM-cobF-cobK*<sup>*</sup>, *cobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobL*<sup>*</sup> and *cobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobL-cobH*<sup>*</sup> (where the asterisk represents a His-tagged encoded version of the gene).

After *E. coli* was transformed with these plasmids and the His-tagged protein was purified from the appropriate individual strains, we found that CobJ<sup>*</sup>, CobM<sup>*</sup>, CobF<sup>*</sup>, CobK<sup>*</sup> and CobH<sup>*</sup> all purified with an associated chromophore that varied in color from blue to yellow and red (Fig. 2). Only CobL<sup>*</sup> failed to purify with a bound pigment. An SDS gel of the isolated His-tagged proteins is shown in Supplementary Results, Supplementary Figure 1.

#### Isolation of precorrin-4 and factor IV with CobJ<sup>*</sup>
When CobJ<sup>*</sup> was isolated aerobically from the recombinant strain harboring *cobA-cobI-cobG-cobJ*<sup>*</sup>, the protein purified with a strong blue color. The spectrum obtained from this blue pigment-protein complex had maxima at 378 nm, 477 nm, 510 nm, 593 nm and 641 nm (Fig. 2a), whereas the free pigment had a mass of *m/z* 907 [M+H]<sup>+</sup>, all of which is consistent with the presence of factor IV (ref. 12), the oxidized form of precorrin-4.

When the purification was repeated under anaerobic conditions, CobJ<sup>*</sup> was found to purify with a yellow color, consistent with the presence of precorrin-4 (Fig. 2a). Exposure of the anaerobically isolated CobJ to oxygen led to a rapid transformation into the blue pigment. Evidence that the yellow pigment was precorrin-4 came from incubating this protein-bound complex with SAM and NADPH and purified CobF, CobM, CobK, CobL and CobH, which resulted in the appearance of the orange-colored HBA and its characteristic spectrum (data not shown).

#### Isolation of complexes with CobF<sup>*</sup> and CobM<sup>*</sup>
We isolated a very light yellow pigment with CobM<sup>*</sup> after extraction and purification of the protein by immobilized metal affinity chromatography from the recombinant strain harboring *cobA-cobI-cobG-cobJ-cobM*<sup>*</sup> (Fig. 2b), whereas we observed a dark yellow pigment associated with CobF<sup>*</sup> after its purification from the strain containing *cobA-cobI-cobG-cobJ-cobM-cobF*<sup>*</sup> (Fig. 2c). The spectra obtained from CobM<sup>*</sup> and CobF<sup>*</sup> were not very defined, and the attached chromophores were not characterized in detail. Moreover, incubation of these complexes with the downstream enzymes for HBA synthesis did not result in the production

NATURE CHEMICAL BIOLOGY DOI:10.1038/NCHEMBIO.1086

**Figure 2 | Spectral panoply of cobalamin intermediates isolated either via an enzyme-product complex, carrier-protein complex or by in vitro incubation.**

(a) Picture insets: purified CobJ* isolates blue (aerobic) or yellow (anaerobic) from a strain overproducing CobA-CobI-CobG-CobJ*. CobE* has no coloration when purified from a strain producing CobA-CobI-CobG-CobJ-CobE*. Spectra are of anaerobically purified CobJ* (gray), aerobically purified CobJ* (black line), the intermediate released from aerobically purified CobJ* (red line) and purified CobE* (dashed line). PC, precorrin; F, factor.

(b) Picture insets: purified CobM* isolates pale yellow from a strain overproducing CobA-CobI-CobG-CobJ-CobM*. CobE* has a stronger coloration when purified from a strain producing CobA-CobI-CobG-CobM-CobE*. Spectra are of purified CobM* (black line), precorrin-5 produced by incubating precorrin-4 with CobM (red dashed line) and purified CobE* (black dashed line).

(c) Picture insets: purified CobF* isolates yellow from a strain overproducing CobA-CobI-CobG-CobJ-CobM-CobF*. CobE* has stronger coloration when purified from a strain producing CobA-CobI-CobG-CobJ-CobM-CobF-CobE*. Spectra are of purified CobF* (black line), precorrin-6A produced by incubating precorrin-4 with CobM and CobF (red dashed line) and purified CobE* (black dashed line).

(d) Picture insets: purified CobK* isolates yellow from a strain overproducing CobA-CobI-CobG-CobJ-CobM-CobF-CobK*. CobE* has less coloration when purified from a strain producing CobA-CobI-CobG-CobJ-CobM-CobF-CobK-CobE*. Spectra are of purified CobK* (black line), intermediate released from purified CobK* (red line) and purified CobE* (black dashed line).

(e) Picture insets: purified CobL<sup>C*</sup> isolates colorless from a strain overproducing CobA-CobI-CobG-CobJ-CobM-CobF-CobK-CobL<sup>C*</sup>. CobE* has yellow coloration when purified from a strain producing CobA-CobI-CobG-CobJ-CobM-CobF-CobK-CobL<sup>C</sup>-CobE*. Spectra are of CobL<sup>C*</sup> (black line), purified CobE* (black dashed line) and the intermediate released from purified CobE* (red line).

(f) Picture insets: purified CobL* isolates colorless from strain overproducing CobA-CobI-CobG-CobJ-CobM-CobF-CobK-CobL*. CobE* has yellow coloration when purified from a strain producing CobA-CobI-CobG-CobJ-CobM-CobF-CobK-CobL-CobE*. Spectra are of purified CobL* (black line), purified CobE* (black dashed line) and the intermediate released from purified CobE* (red line).

(g) Picture insets: purified CobH* isolates orange from a strain overproducing CobA-CobI-CobG-CobJ-CobM-CobF-CobK-CobL-CobH*. CobE* is also orange when purified from a strain producing CobA-CobI-CobG-CobJ-CobM-CobF-CobK-CobL-CobH-CobE*. Spectra are of purified CobH* (black line), the intermediate released from purified CobH* (red line) and purified CobE* (black dashed line).

of HBA. Thus, the compounds bound to CobM* and CobF* are not pathway intermediates and most likely represent either oxidized products or derailed pathway analogs. Such derailed analogs have been observed previously and result from either overmethylation or methylation, a consequence of a particularly high concentration of biosynthetic enzyme.

### Isolation of precorrin-5 and precorrin-6A

Although precorrin-5 and precorrin-6A were not found to be associated with CobM* and CobF*, these intermediates could be generated by adding the enzymes sequentially to the CobJ-precorrin-4 complex. Thus, addition of SAM and CobM resulted in the appearance of precorrin-5 (Fig. 2b), a pale yellow compound that was very unstable and turned purple within minutes inside the anaerobic chamber in the absence of the next enzyme of the pathway. This purple material seems to be a dead-end compound, as addition of further enzymes did not result in any further spectral changes. Addition of SAM plus CobM and CobF to the CobJ-precorrin-4 complex generated precorrin-6A (Fig. 2c). This compound was rapidly reduced to precorrin-6B when it was incubated with NADPH and CobK, whereas the further addition of CobL led to the appearance of precorrin-8, from which HBA was produced by the addition of CobH (data not shown).

### Isolation of precorrin-6B

When CobK* was produced and purified from the recombinant strain containing *cobA-cobI-cobG-cobJ-cobM-cobF-cobK*, we found it to be associated with a yellow pigment that had an absorption maximum at 348 nm (Fig. 2d). This enzyme-product complex could be converted into HBA by incubation with the downstream enzymes CobL and CobH (data not shown). When released from CobK, the intermediate was found to have a mass of 897.4 [M+H]<sup>+</sup>, which is that expected for precorrin-6B.

### CobL* does not form a stable enzyme-product complex

The only protein within this section of the cobalamin biosynthetic pathway that does not seem to bind anything substantial is CobL. Thus, CobL* isolated from the strain harboring *cobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobL* gave a spectrum without clear distinguishing features, with only minor absorptions (Fig. 2e,f).

### CobH* binds HBA

When CobH* was purified from the strain harboring *cobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobL-cobH*, it was found to purify with a strong orange chromophore (Fig. 2g and Supplementary Fig. 2). A UV-visible scan of the protein solution showed that it contained

a slightly altered UV-visible spectrum of HBA with absorption maxima at 330 nm and 484 nm (Fig. 2g). The CobH*-HBA complex could be dissociated by boiling, which resulted in a concomitant change in the spectrum to that of free HBA with absorption maxima at 329 nm, 496 nm and 522 nm (Fig. 2g) and a mass of m/z 881.4 [M+H]+ observed by MS. The structure of this free acid form of HBA was further confirmed by NMR (Supplementary Table 1a) and by the crystallization (Supplementary Fig. 2) and subsequent X-ray structure determination of the CobH-HBA complex. Previous reports have highlighted a tight association between CobH and HBA¹⁷,²⁰.

### CobE binds a number of different intermediates

The lack of a binding protein for precorrin-8 led us to think that this role could be filled by CobE, whose function within the aerobic B₁₂ pathway has been the subject of some speculation⁴. In the current work, we have shown that CobE is not required for the in vitro transformation of precorrin-4 into HBA as this conversion is mediated by CobM, CobF, CobK, CobL and CobH alone. To investigate the possibility that CobE may act as a substrate carrier, we modified cobE to include a fusion encoding a His₆ tag (CobE*) and then cloned it in tandem into a range of plasmids with the genes responsible for the production of precorrin-4, precorrin-5, precorrin-6A, precorrin-6B, precorrin-8 or HBA. Doing so resulted in the following constructs: cobA-cobI-cobG-cobJ-cobE*, cobA-cobI-cobG-cobM-cobE*, cobA-cobI-cobG-cobJ-cobM-cobF-cobE*, cobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobE*, cobA-cobI-cobG-cobM-cobF-cobK-cobL-cobE* and cobA-cobI-cobG-cobM-cobF-cobK-cobL-cobH-cobE*. When CobE* was isolated from the resulting strains transformed with these plasmids, it was found to purify in association with a range of different pigments (Fig. 2). An SDS gel of CobE* isolated from these strain is shown in Supplementary Figure 1.

When CobE* was isolated from the strain harboring cobA-cobI-cobG-cobJ-cobE*, there was little evidence of anything binding the protein (Fig. 2a). However, when CobE* was isolated from strains with the genes required for precorrin-5 and precorrin-6A (that is, from strains containing cobA-cobI-cobG-cobJ-cobM-cobE* and cobA-cobI-cobG-cobJ-cobM-cobF-cobE*, respectively), the protein was found to purify with strong absorption spectra (Fig. 2b,c). The spectra did not correspond to free precorrin-5 or precorrin-6A, but it did have some similarity to the spectra observed with CobM* and CobF*. Unfortunately, as with CobM* and CobF*, incubation of the isolated CobE*s from the precorrin-5 and precorrin-6A strains with the downstream enzymes for HBA synthesis did not result in a change in the spectra. However, CobE* isolated from the strain with the genetic components for precorrin-6B synthesis (that is, containing cobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobE*) had a broad absorption spectrum around 350 nm, although it was not as defined as the product-complex spectrum that was observed when CobK* was purified (Fig. 2d). Pointedly, incubation of CobE* with CobL and CobH resulted in the production of HBA. Thus, CobE* has the ability to bind precorrin-6B.

CobE* was also found to bind precorrin-8 and HBA, as isolation of CobE* from strains harboring cobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobL-cobE*andcobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobL-cobH-cobE* resulted in yellow- and orange-colored protein, respectively (Fig. 2f,g). The yellow color was shown to be precorrin-8, as incubation of this protein with CobH resulted in HBA production. The structure of precorrin-8 was confirmed by NMR (Supplementary Table 1b). Notably, CobE is able to bind precorrin-8, whereas CobL does not. Overall, therefore, CobE is able to bind precorrin-6B, precorrin-8 and HBA. As a result, we propose that its role in the aerobic pathway is to act as a substrate carrier protein or substrate chaperone, binding excess labile compounds and ensuring that they are delivered to the next enzyme in the pathway.

---

**Figure 3 | Role of CobE in the stabilization of precorrin-8 and the structure determination of the protein.** (a) Comparison of the stability of free and CobE-bound precorrin-8 (PC8) as monitored by loss of spectral features over time. (b) Comparison of the amount of HBA produced from precorrin-6B (PC6B) with CobL and CobH in the presence and absence of CobE. (c) Structure of CobE. The main crevice in the protein is formed between the β-sheet, α-helices α₅ and α₆ and the loops between β1-α1 and β2-α3 of CobE, where nonprotein ligand molecules (glycerol and sulfate) are located. (d) Space-filling figure demonstrating the steric and electrostatic fit of HBA (fitted by PyMOL) into the cavity in CobE. The CobH surface is colored according to charge with positive regions shown in blue and negative regions in red. HBA is shown in stick representation.

In support of this theory, we observed that CobE was able to stabilize precorrin-8. When left free in solution, precorrin-8 bleaches with a half-life of 3–4 h (Fig. 3a and Supplementary Fig. 3). In contrast, when bound to CobE, comparatively little precorrin-8 is lost (Fig. 3a). Moreover, the yield of HBA from precorrin-6B through incubation with CobL, CobH and SAM is approximately doubled in the presence of CobE (Fig. 3b and Supplementary Fig. 3). To understand how CobE, which is a comparatively small protein of around 150 amino acids, may act to stabilize the various pathway intermediates that it binds, we solved the structure of the protein by X-ray crystallography (Fig. 3c). This revealed that CobE has a globular α/β fold with dimensions approximately 30 Å × 35 Å × 35 Å, with a single domain containing a five-stranded β-sheet with five associated α-helices and one 3₁₀-helix (Fig. 3c). The β-sheet and α-helices α₅ and α₆ create the walls of a cleft of approximate dimensions 8 Å × 15 Å × 17 Å. The cleft is most likely the binding site of the protein, and in the present structure, several nonprotein ligand molecules are bound therein. The presence of conserved basic residues lining this cleft strongly suggests their involvement in binding groups such as the negative charges on the side chain groups of the corrin ring. Modeling of HBA in the cavity suggests that CobE can easily accommodate molecules the size of HBA in a slightly bent conformation both in terms of shape and electrostatic complementarity (Fig. 3d), though further experimental evidence is needed to confirm the scope of CobE function.

Although crystallography of CobE-bound complexes has not yet been achieved, NMR spectra of CobE bound to a range of compounds were recorded. ¹⁵N-¹H HSQC spectra of CobE before and after the addition of HBA and precorrin-6B (Supplementary Fig. 4a,b) support binding of the intermediates by this protein.

NATURE CHEMICAL BIOLOGY DOI: 10.1038/NCHEMBIO.1086

**Figure 4 | Structure analysis of CobL.** (a) Cartoon demonstrating the dissection of CobL. The protein was dissected to give CobL<sup>N</sup> and CobL<sup>C</sup>, representing the N- and C-terminal regions of the protein. These regions align with the separate CbiE and CbiT proteins found in the anaerobic pathway (Supplementary Fig. 8). The figure also highlights one of the mutations that was introduced into CobL at position 257, within the second SAM-binding site, to inactivate this region of the protein. The SAM-binding regions and tetramerization domain (TetraD) are also indicated. (b) Cartoon of the tetrameric arrangement of four *R. capsulatus* CobL<sup>C</sup> subunits. Each polypeptide chain is a different color. The four SAH molecules, one in each of the four subunits, are drawn as sticks. (c) The SAM-binding site of the CobL<sup>C</sup> subunit showing SAH bound at the C-terminal end of the Rossmann-like β/α fold. SAH binds between two glycine-rich loops that follow β1 and β4, respectively. The precorrin-binding cleft is adjacent to the sulfur of SAH. Mutation of either Glu276 or Gly257 prevents SAM binding and destroys CobL<sup>C</sup> C15 methylation and decarboxylation activity.

Changes in chemical shifts further suggest that a substantial structural change in CobE must occur upon interaction. Closer inspection of the data (Supplementary Fig. 4) confirms both similarities and differences between the chemical shift changes observed for CobE with each intermediate. Apo CobE <sup>15</sup>N HSQC data contains 62% of the resonances expected for the protein, with this number rising to 80% in the presence of ligands. To probe the specificity of binding of HBA to CobE further, we performed quenching fluorescence assays that confirmed a 1:1 stoichiometric HBA-CobE interaction with a *K*<sub>d</sub> of 0.08 μM (Supplementary Fig. 5). Additionally, <sup>15</sup>N HSQC NMR titration-curve data sets were created using variable final concentrations between 0.0 mM and 1.0 mM of HBA, with a final constant 0.2-mM concentration of CobE. Tracking of HBA addition using chemical shift changes confirmed the tight interaction observed by fluorescence quenching (*K*<sub>d</sub> = 0.12 μM; Supplementary Fig. 6a). Fitting this data also yielded a second event with a lower *K*<sub>d</sub> of 309 μM (Supplementary Fig. 6b), though the basis for this is unknown.

### CobL dissection

With the basis for CobE function established, we returned to the one enzyme that had proven uninformative in the reconstruction assays, CobL. We used the precorrin-6B–CobK<sup>*</sup> complex to study the reaction catalyzed by CobL, which is responsible for the C5 and C15 methylations and the decarboxylation of the acetate side chain attached to the C12. When precorrin-6B-bound CobK was incubated with SAM and CobL, the solution turned a brighter yellow shade, which was associated with the generation of precorrin-8 (Supplementary Fig. 7). Analysis of the resulting compound revealed that it had a mass of *m/z* 881.4 [M+H]<sup>+</sup>, which agrees with such a transformation.

Comparative studies suggest that CobL has arisen from a fusion between two separate classes of SAM methyltransferases, the canonical B<sub>12</sub> biosynthetic class III type and a class I methyltransferase. Indeed, in the anaerobic pathway, the ortholog of CobL is sometimes found as two separate genes, which is the case in *Salmonella enterica*, where they are termed CbiE and CbiT. To investigate the role of the two regions within CobL, the gene was dissected to give *cobL*<sup>N</sup>, which encodes the N-terminal part of

NATURE CHEMICAL BIOLOGY | VOL 8 | NOVEMBER 2012 | www.nature.com/naturechemicalbiology

ARTICLE NATURE CHEMICAL BIOLOGY DOI:10.1038/NCHEMBIO.1086

![Chemical Structures and Diagrams](image.png)

Figure 5 | Transformation of precorrin-6B into HBA and C5-desmethyl-HBA. The acetic acid side chain attached to C12 of precorrin-6B is decarboxylated concomitantly with the methylation at C15 to generate precorrin-7 by the C-terminal region of CobL. Precorrin-7 is then methylated by the N-terminal region of CobL at C5 to give precorrin-8, which is converted into HBA by CobH. However, precorrin-7 can also act as an out-of-turn substrate for CobH. The structures of HBA and C5-desmethyl-HBA were determined by X-ray crystallography after binding them to CobH. The electron density for the C5 methyl group is clearly absent in the desmethyl compound. The map shown is the maximum likelihood (σA) weighted 2Fobs – Fcalc density (contoured 1σ) in blue.

though it does not bring the active centers of the two domains close together, could influence the accessibility of the N-terminal domain’s active center to CobE or CobH.

### Isolation and characterization of precorrin-7

To investigate the ability of CobE to bind precorrin-7, we produced CobE* in a strain harboring cobA-cobI-cobG-cobJ-cobM-cobF-cobK-cobL<sup>C</sup>-cobE*. The protein was indeed found to purify with precorrin-7 attached (Fig. 2e). As with precorrin-8, CobE also seems to stabilize precorrin-7 as the bound complex has an extended half-life in comparison to the unbound intermediate. The binding of precorrin-7 to CobE could also be monitored by NMR (Supplementary Fig. 4c). After the protein was gently denatured, we performed NMR analysis of the liberated precorrin-7. The data support the presence of major and minor tautomeric states (Supplementary Table 1c) that differ in the presence of a proton attached at the C8 position in the minor state (Supplementary Fig. 10). This idea was confirmed using <sup>13</sup>C-<sup>1</sup>H HSQC, heteronuclear multiple bond correlation spectroscopy and TOCSY data sets (Supplementary Fig. 11). Such resonances are broad, suggesting strong evidence for chemical exchange in this state. <sup>1</sup>H-<sup>13</sup>C HSQC spectra were also collected at 5 °C and 40 °C in an attempt to minimize the presence of the minor form, but further deconvolution of the data was not possible.

The major state of precorrin-7 is consistent with the data shown in Supplementary Table 1c. The NMR confirms the presence of methyl groups at positions C1, C2, C7, C11, C12, C15 and C17. This demonstrates that CobL<sup>C</sup> is responsible for the C15 methylation and decarboxylation of the acetate side chain on C12. From a mechanistic perspective, it is likely that the decarboxylation and methylation are concerted reactions (Fig. 5).

To generate a form of CobL that only has C5 methylase activity, with no C15 methylase or decarboxylase activity, we introduced two separate point mutations, G257R and E276A, in the SAM-binding site of the CobL<sup>C</sup> domain of full-length CobL (Fig. 4c and Supplementary Fig. 8). Incubation of CobL<sup>G257R</sup> or CobL<sup>E276A</sup> with precorrin-6B-bound CobK and SAM did not alter the spectrum of the substrate. However, when CobL<sup>C</sup> was added to the reaction, the spectrum of precorrin-8 was produced (Supplementary Fig. 7), which changed to HBA when CobH was added. CobL<sup>G257R</sup> or CobL<sup>E276A</sup> are therefore able to use precorrin-7 as a substrate and function solely as C5 methyltransferases (Supplementary Fig. 7). This series of experiments clearly delineate the reaction sequence for the transformation of precorrin-6B into precorrin-8.

### Synthesis of C5-desmethyl-HBA

A further unexpected result was obtained by incubating precorrin-7 with CobH. Here the compound with an absorption maximum at 397 nm was readily transformed into a compound with a spectrum that was similar to, but distinct from, HBA (Supplementary Fig. 12). MS analysis of this new compound revealed that it is 14 Da smaller than HBA, consistent with the formation of C5-desmethyl-HBA (2; m/z of 867.4 [M+H]<sup>+</sup>). It thus seems that precorrin-7 acts as an out-of-turn substrate for CobH. The structure of the compound was confirmed by crystallizing the CobH–desmethyl-HBA complex, from which the resulting 1.4-Å data clearly revealed the absence of the C5 methyl group (Fig. 5).

### DISCUSSION

This report describes an approach for trapping intermediates involved in the cobalamin biosynthetic pathway in their natural (free acid) form as enzyme-product complexes. CobJ, CobK and CobH were found to bind their products tightly and protect them from the external environment. Furthermore, CobE, a protein to which no function had been formally assigned, was found to

NATURE CHEMICAL BIOLOGY | VOL 8 | NOVEMBER 2012 | www.nature.com/naturechemicalbiology
associate with a number of intermediates including precorrin-6B, precorrin-7, precorrin-8 and HBA. Our *in vitro* data support a role for CobE in the stabilization of intermediates such as precorrin-7 and precorrin-8, most likely through binding to the large crevice found at the opposite end of the protein to the N and C termini. The inclusion of CobE in ex vivo assays of precorrin-6B to HBA also leads to an enhancement in overall yield. Thus, CobE has a role as a substrate carrier or chaperone protein, shuttling labile intermediates between different Cob enzymes.

The observation that several stable enzyme-product complexes can be isolated from the aerobic pathway is unusual for a metabolic pathway and is probably linked to the low stability of the pathway intermediates. The high affinity of the intermediates for the enzymes requires very low dissociation rate constants that are incompatible with the expected flux through the pathway. This implies that the enzymes retain the unstable intermediates until some signal or interaction allows the release of the product to be efficiently used by the next enzyme in the pathway. The established hypothesis for such behavior is called metabolite channeling; that is, the enzyme holds onto its product to pass it efficiently, if not directly, to the succeeding step in the biosynthesis. At one time this was a contentious area²¹,²², but there are now several clear-cut examples where substrate channeling does occur (reviewed in refs. 23–25). We believe that this is the first report of a series of enzymes that form stable product complexes in the absence of a stable multienzyme complex. Channeling requires a series of transient interactions between the enzymes along the pathway to allow intermediates to be passed from enzyme to enzyme. Given that the cytoplasm of *E. coli* seems to act as a sieve that permits the diffusion of small molecules while limiting the movement of larger ones²⁶,²⁷, the observed formation of stable enzyme-product complexes would suggest that the enzymes of the cobalamin pathway must loosely associate in the form of a metabolon²⁸ to allow the pathway to operate.

However, our research does not differentiate between ‘real channeling’ and ‘enhanced transfer’ between enzymes, which can occur between two enzymes if there is a higher probability of substrate rebinding to the downstream enzyme. This will require further research to measure the loss of substrate to side reactions as a function of enzyme concentration in pairs of enzymes and in the complete pathway.

The research describes a simple but highly efficient method for the isolation of pathway intermediates, thereby allowing the elucidation of metabolic pathways in a fairly rapid timescale. This method will be particularly applicable to pathways involving labile compounds. This report also describes the combinatorial synthesis of a nonphysiological corrin analog, 5-desmethyl-HBA. Such an approach could be used to make a range of cobaltoporphyrin and corrin derivatives, molecules that are of increasing interest in the bioenergy field as they have been shown to have the ability to oxidize water²⁹,³⁰. This approach can also be used to make useful probes for the study of cobalamin-dependent reactions and derivatives of the cofactor for enhanced drug delivery³¹.

### METHODS

#### Construction of plasmids.
All cobalamin biosynthetic genes were amplified by PCR using genomic DNA from either *Rhodobacter capsulatus* SB1003 (*cobA*, *cobI*, *cobJ*, *cobF*, *cobM*, *cobK*, *cobL* and *cobH*), *Brucella melitensis* 16M (*cobG*, *cobE*), *Pseudomonas denitrificans* (*cobG*) or *Pseudomonas aeruginosa* (*cobE*). The *cob* genes were amplified individually with primers containing either Asel or NdeI restriction sites at the 5′ end and a SpeI site at the 3′ end to allow them to be cloned contiguously within the plasmid as previously described⁶. Genes were cloned into modified pET3a and pET14b plasmids as highlighted in Supplementary Table 2.

#### Recombinant protein overproduction and purification.
*E. coli* strain BL21 star (DE3) pLysS was transformed with the relevant plasmids. The recombinant strains were grown in LB medium at 37 °C. The protein overproduction was induced with isopropyl β-D-galactopyranoside (0.4 mM). The cells were resuspended in buffer A (500 mM NaCl, 5 mM imidazole, 20 mM HEPES, pH 7.5) and sonicated.

The protein was purified using chelating Sepharose charged with NiSO₄. Unbound proteins were washed off with buffer A and with buffer A containing 60 mM imidazole. Proteins were eluted with buffer A containing 400 mM imidazole and passed through a PD10 column equilibrated with buffer B (20 mM HEPES, pH 7.5, and 100 mM NaCl).

Purification of proteins with trapped intermediates. The different *E. coli* strains transformed with plasmids to enable the trapping of the various pathway intermediates (Supplementary Table 2) were grown in 2YT at 28 °C for 24 h. The protein purification procedure was identical to the method above except that NaCl was lowered to 100 mM.

Site-directed mutagenesis. The CobL<sup>G257R</sup> mutant was created using the QuikChange II site-directed mutagenesis kit (Agilent). CobL<sup>E276A</sup> was constructed following the site-specific mutagenesis by overlap extension protocol³².

Assays from PC4 to HBA or C5-desmethyl-HBA. All assays were performed in buffer B. SAM was added at a final concentration of 0.58 mM, NADPH was added at 15 μM and all proteins were added at 2 μM. The reactions were incubated at 21 °C. Anaerobic conditions were obtained using an anaerobic glove box (Belle Technology).

MS. Samples were separated on an Agilent 1100 series HPLC coupled to a micrOTOF-Q II (Bruker) mass spectrophotometer using an Ace 5 AQ column (2.1 mm × 150 mm; Advanced Chromatography Technologies) maintained at 30 °C and with a flow rate of 0.2 ml min⁻¹. The mobile phase consisted of 0.1% (v/v) acetic acid in water (solvent A) and 100% methanol (solvent B). The initial conditions were set at 95% A, 5% B. The concentration of solvent B was increased with a linear gradient to 80% over 30 min and was changed back to the initial conditions at 40 min. The mass spectrometer was operated in the positive electrospray mode.

NMR of precorrin-7, precorrin-8 and HBA. Samples of precorrin-7, precorrin-8 and HBA for NMR analysis were prepared from their respective strains bound to either CobE<sup>*</sup> or CobH<sup>*</sup>. Precorrin-7 and precorrin-8 were purified trapped with CobE as described above, whereas HBA was trapped with CobH. The complexes were then heated to denature the protein and leave the intermediate free in solution. After lowering the pH to around 5, the pigments were applied to LiChroprep RP18 columns. The compounds were briefly washed with water and eluted in 50% (v/v) ethanol. After lyophilization, they were finally dissolved in deuterium oxide (99.9 atom D). All NMR experiments were carried out using a 600-MHz <sup>1</sup>H Varian UnityINOVA spectrometer with a 5-mm z-PFG HCN NMR probe or a 600-MHz <sup>1</sup>H Bruker Avance III spectrometer with a 5-mm QCI-F cryoprobe.

NMR details are given in the Supplementary Methods. <sup>1</sup>H chemical shift referencing was based on the position of the water resonance, <sup>13</sup>C referencing used <sup>1</sup>H and <sup>13</sup>CH/<sup>13</sup>C gyromagnetic ratios to define indirect carrier position³³, and all data were obtained at 25 °C unless otherwise stated. Data processing was completed using NMRPipe³⁴, and resonance assignments were completed using CCPN analysis 2 (ref. 35).

Crystallography. CobE crystals were obtained from 0.1 M 2-(N-morpholino) ethanesulfonic acid buffer (pH 6.9), 2 M ammonium sulfate and 5% (w/v) dioxane³⁶, and the structure was solved as outlined in the Supplementary Methods. The final R-factor and R-free are 0.194 and 0.268, respectively, for a model with good stereochemistry (Supplementary Table 3a). CobL<sup>C</sup> crystals were produced as described previously³⁷ and solved at 2.7 Å by molecular replacement using the *Corynebacterium diphtheriae* putative precorrin-6Y C5,15-methyltransferase targeted domain structure (Protein Data Bank (PDB) code: 3HM2) as a search model. The *C. diphtheria* methyltransferase sequence is 41% identical to that of *R. capsulatus* CobL<sup>C</sup>, and MOLREP³⁸ gave a model with a score of 0.314 that could be successfully rebuilt and refined using CNS³⁹ and BUCANEER⁴⁰ or REFMAC⁴¹ with manual rebuilding using COOT⁴² (Supplementary Fig. 9). The final R-factor and R-free are 0.185 and 0.254, respectively, for a model with good stereochemistry (Supplementary Table 3b). The CobH complexes were crystallized using 25–30% PEG 8K, 0.2 M ammonium sulfate and 0.1 M cacodylate pH 6.5. The structures were solved by molecular replacement using 1I1H²⁰ and refined using similar methods to those described for CobL<sup>C</sup> (Supplementary Table 3c)⁴³⁻⁴⁵.

Molecular modeling. Docking was used to fit HBA into the binding cavity of CobE. There was a single orientation of HBA that fitted well with the acidic side chains mapping to basic regions of CobE. No adjustments of the conformation of HBA or CobE were necessary. The CobL structure was modeled by docking the N-terminal domains of CbiE onto the C-terminal domains of CobL<sup>C</sup>. Again, this was a straightforward modeling procedure achieved using molecular docking in PyMOL.

Accession codes. PDB: coordinates and structure factor amplitudes have been deposited in the PDB with accession codes 2W6K (CobE), 3NJR (CobL<sup>C</sup>), 4FDV (CobH–HBA) and 4AU1 (CobH–desmethyl-HBA).
ARTICLE

Received 2 April 2012; accepted 5 September 2012; published online 7 October 2012

**References**

1. Webb, M.E., Marquet, A., Mendel, R.R., Rebeille, F. & Smith, A.G. Elucidating biosynthetic pathways for vitamins and cofactors. *Nat. Prod. Rep.* **24**, 988–1008 (2007).
2. Battersby, A.R. How nature builds the pigments of life: the conquest of vitamin B₁₂. *Science* **264**, 1551–1557 (1994).
3. Warren, M.J., Raux, E., Schubert, H.L. & Escalante-Semerena, J.C. The biosynthesis of adenosylcobalamin (vitamin B₁₂). *Nat. Prod. Rep.* **19**, 390–412 (2002).
4. Blanche, F. *et al.* Vitamin B₁₂: how the problem of its biosynthesis was solved. *Angew. Chem. Int. Edn. Engl.* **34**, 383–411 (1995).
5. Blanche, F. *et al.* Hydrogenobyrinic acid—isolation, biosynthesis, and function. *Angew. Chem. Int. Edn. Engl.* **29**, 884–886 (1990).
6. McGoldrick, H.M. *et al.* Identification and characterization of a novel vitamin B₁₂ (cobalamin) biosynthetic enzyme (CobZ) from *Rhodobacter capsulatus*, containing flavin, heme, and Fe-S cofactors. *J. Biol. Chem.* **280**, 1086–1094 (2005).
7. Uzar, H.C., Battersby, A.R., Carpenter, T.A. & Leeper, F.J. Biosynthesis of porphyrins and related macrocycles. Part 28. Development of a pulse labeling method to determine the C-methylation sequence for vitamin B₁₂. *J. Chem. Soc. Perkin Trans.* **1**, 1689–1696 (1987).
8. Blanche, F., Debussche, L., Thibaut, D., Crouzet, J. & Cameron, B. Purification and characterization of S-adenosyl-l-methionine: uroporphyrinogen III methyltransferase from *Pseudomonas denitrificans*. *J. Bacteriol.* **171**, 4222–4231 (1989).
9. Warren, M.J. *et al.* Enzymatic synthesis and structure of precorrin-3, a trimethylpyrrocorphin intermediate in vitamin B₁₂ biosynthesis. *Biochemistry* **31**, 603–609 (1992).
10. Debussche, L. *et al.* Biosynthesis of vitamin B₁₂: structure of precorrin-3B, the trimethylated substrate of the enzyme catalysing ring contraction. *J. Chem. Soc. Chem. Commun.* 1100–1103 (1993).
11. Scott, A.I. *et al.* Biosynthesis of vitamin B₁₂. Discovery of the enzymes for oxidative ring contraction and insertion of the fourth methyl group. *FEBS Lett.* **331**, 105–108 (1993).
12. Thibaut, D. *et al.* Biosynthesis of vitamin-B₁₂—the structure of factor-IV, the oxidized form of precorrin-4. *J. Chem. Soc. Chem. Commun.* 513–515 (1993).
13. Min, C. *et al.* Isolation, structure and genetically engineered synthesis of precorrin-5. *J. Am. Chem. Soc.* **115**, 10380–10381 (1993).
14. Thibaut, D., Blanche, F., Debussche, L., Leeper, F.J. & Battersby, A.R. Biosynthesis of vitamin B₁₂: structure of precorrin-6x octamethyl ester. *Proc. Natl. Acad. Sci. USA* **87**, 8800–8804 (1990).
15. Blanche, F. *et al.* Precorrin-6x reductase from *Pseudomonas denitrificans*: purification and characterization of the enzyme and identification of the structural gene. *J. Bacteriol.* **174**, 1036–1042 (1992).
16. Blanche, F. *et al.* Biosynthesis of vitamin B₁₂ in *Pseudomonas denitrificans*: the biosynthetic sequence from precorrin-6y to precorrin-8x is catalyzed by the *cobL* gene product. *J. Bacteriol.* **174**, 1050–1052 (1992).
17. Thibaut, D. *et al.* The final step in the biosynthesis of hydrogenobyrinic acid is catalyzed by the *cobH* gene product with precorrin-8x as the substrate. *J. Bacteriol.* **174**, 1043–1049 (1992).
18. Santander, P.J., Kajiwara, Y., Williams, H.J. & Scott, A.I. Structural characterization of novel cobalt corrinoids synthesized by enzymes of the vitamin B₁₂ anaerobic pathway. *Bioorg. Med. Chem.* **14**, 724–731 (2006).
19. Scott, A.I., Warren, M.J., Roessner, C.A., Stolowich, N.J. & Santander, P.J. Development of an overmethylation strategy for corrin synthesis. Multi-enzyme preparation of pyrrocorphins. *J. Chem. Soc. Chem. Commun.* 593–597 (1990).
20. Shipman, L.W., Li, D., Roessner, C.A., Scott, A.I. & Sacchettini, J.C. Crystal structure of precorrin-8x methyl mutase. *Structure* **9**, 587–596 (2001).
21. Pettersson, G. No convincing evidence is available for metabolite channeling between enzymes forming dynamic complexes. *J. Theor. Biol.* **152**, 65–69 (1991).
22. Wu, X.M., Gutfreund, H., Lakatos, S. & Chock, P.B. Substrate channeling in glycolysis: a phantom phenomenon. *Proc. Natl. Acad. Sci. USA* **88**, 497–501 (1991).
23. Huang, X., Holden, H.M. & Raushel, F.M. Channeling of substrates and intermediates in enzyme-catalyzed reactions. *Annu. Rev. Biochem.* **70**, 149–180 (2001).
24. Jørgensen, K. *et al.* Metabolon formation and metabolic channeling in the biosynthesis of plant natural products. *Curr. Opin. Plant Biol.* **8**, 280–291 (2005).
25. Miles, E.W., Rhee, S. & Davies, D.R. The molecular basis of substrate channeling. *J. Biol. Chem.* **274**, 12193–12196 (1999).

26. McGuffee, S.R. & Elcock, A.H. Diffusion, crowding & protein stability in a dynamic molecular model of the bacterial cytoplasm. *PLoS Comput. Biol.* **6**, e1000694 (2010).
27. Mika, J.T., van den Bogaart, G., Veenhoff, L., Krasnikov, V. & Poolman, B. Molecular sieving properties of the cytoplasm of *Escherichia coli* and consequences of osmotic stress. *Mol. Microbiol.* **77**, 200–207 (2010).
28. Robinson, J.B. Jr., Inman, L., Sumegi, B. & Srere, P.A. Further characterization of the Krebs tricarboxylic acid cycle metabolon. *J. Biol. Chem.* **262**, 1786–1790 (1987).
29. Dogutan, D.K. *et al.* Hangman corroles: efficient synthesis and oxygen reaction chemistry. *J. Am. Chem. Soc.* **133**, 131–140 (2011).
30. Lee, C.H., Dogutan, D.K. & Nocera, D.G. Hydrogen generation by hangman metalloporphyrins. *J. Am. Chem. Soc.* **133**, 8775–8777 (2011).
31. Waibel, R. *et al.* New derivatives of vitamin B₁₂ show preferential targeting of tumors. *Cancer Res.* **68**, 2904–2911 (2008).
32. Sambrook, J. & Russell, D. Site-specific mutagenesis by overlap extension in *Molecular Cloning: A Laboratory Manual* 3rd edn., Ch. 13, 13.36–13.39 (Cold Spring Harbor Laboratory Press, 2001).
33. Wishart, D.S. & Sykes, B.D. Chemical-shifts as a tool for structure determination. *Methods Enzymol.* **239**, 363–392 (1994).
34. Delaglio, F. *et al.* NMRpipe—a multidimensional spectral processing system based on UNIX pipes. *J. Biomol. NMR* **6**, 277–293 (1995).
35. Vranken, W.F. *et al.* The CCPN data model for NMR spectroscopy: development of a software pipeline. *Proteins* **59**, 687–696 (2005).
36. Vévodová, J., Graham, R.M., Raux, E., Warren, M.J. & Wilson, K.S. Crystallization and preliminary structure analysis of CobE, an essential protein of cobalamin (vitamin B₁₂) biosynthesis. *Acta Crystallogr. Sect. F Struct. Biol. Commun.* **61**, 442–444 (2005).
37. Seyedarabi, A. *et al.* Cloning, purification and preliminary crystallographic analysis of cobalamin methyltransferases from *Rhodobacter capsulatus*. *Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.* **66**, 1652–1656 (2010).
38. Vagin, A. & Teplyakov, A. Molecular replacement with MOLREP. *Acta Crystallogr. D Biol. Crystallogr.* **66**, 22–25 (2010).
39. Brünger, A.T. Version 1.2 of the Crystallography and NMR system. *Nat. Protoc.* **2**, 2728–2733 (2007).
40. Cowtan, K. The Buccaneer software for automated model building. 1. Tracing protein chains. *Acta Crystallogr. D Biol. Crystallogr.* **62**, 1002–1011 (2006).
41. Murshudov, G.N., Vagin, A.A. & Dodson, E.J. Refinement of macromolecular structures by the maximum-likelihood method. *Acta Crystallogr. D Biol. Crystallogr.* **53**, 240–255 (1997).
42. Emsley, P., Lohkamp, B., Scott, W.G. & Cowtan, K. Features and development of Coot. *Acta Crystallogr. D Biol. Crystallogr.* **66**, 486–501 (2010).
43. Battye, T.G.G., Kontogiannis, L., Johnson, O., Powell, H.R. & Leslie, A.G.W. iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM. *Acta Crystallogr. D Biol. Crystallogr.* **67**, 271–281 (2011).
44. Blanc, E. *et al.* Refinement of severely incomplete structures with maximum likelihood in BUSTER-TNT. *Acta Crystallogr. D Biol. Crystallogr.* **60**, 2210–2221 (2004).
45. Winter, G. xia2: an expert system for macromolecular crystallography data reduction. *J. Appl. Crystallogr.* **43**, 186–190 (2010).

**Acknowledgments**

This work was supported by grants from the Biotechnology and Biological Sciences Research Council (BB/E024203 and BB/I013334) and the Wellcome Trust (091163/Z/10/Z). We thank M. Rowe for additional NMR technical support. Diffraction data were collected at the European Synchrotron Radiation Facility, Grenoble, France (for CobE) and the Diamond Light Source, Oxfordshire, UK (for CobL and CobH). We thank C. Roessner (Texas A&M University) for a clone of the *P. denitrificans cobG*.

**Author contributions**

E.D. designed and performed most of the experiments and analysis with support from A.D.L. and S.S.; A.D.L. performed MS analysis. S.L.T. performed all NMR data acquisition, which was analyzed with M.J.H. S.S., A.S. and R.W.P. contributed the CobL<sup>C</sup> and CobH-HBA structures and CobL and CobE-HBA models. J.W. and K.S.W. contributed the CobE structure. D.B. and S.S. determined the CobH-5-desmethyl-HBA structure. M.A.G. provided insight into substrate channeling. M.J.W. directed all aspects of the project. E.D. and M.J.W. wrote the manuscript.

**Competing financial interests**

The authors declare no competing financial interests.

**Additional information**

Supplementary information and chemical compound information is available in the online version of the paper. Reprints and permissions information is available online at http://www.nature.com/reprints/index.html. Correspondence and requests for materials should be addressed to M.J.W. or R.W.P.
