
LETTER

Fucose sensing regulates bacterial intestinal colonization

Alline R. Pacheco${}^{1,2}$, Meredith M. Curtis${}^{1,2}$, Jennifer M. Ritchie${}^{3}$, Diana Munera${}^{3}$, Matthew K. Waldor${}^{3}$, Cristiano G. Moreira${}^{1,2}$ & Vanessa Sperandio${}^{1,2}$

The mammalian gastrointestinal tract provides a complex and competitive environment for the microbiota ${}^{1}$. Successful colonization by pathogens requires scavenging nutrients, sensing chemical signals, competing with the resident bacteria and precisely regulating the expression of virulence genes ${}^{2}$. The gastrointestinal pathogen enterohaemorrhagic *Escherichia coli* (EHEC) relies on inter-kingdom chemical sensing systems to regulate virulence gene expression ${}^{3,4}$. Here we show that these systems control the expression of a novel two-component signal transduction system, named FusKR, where FusK is the histidine sensor kinase and FusR the response regulator. FusK senses fucose and controls expression of virulence and metabolic genes. This fucose-sensing system is required for robust EHEC colonization of the mammalian intestine. Fucose is highly abundant in the intestine ${}^{5}$. *Bacteroides theta-iotaomicron* produces multiple fucosidases that cleave fucose from host glycans, resulting in high fucose availability in the gut lumen ${}^{6}$. During growth in mucin, *B. theta-iotaomicron* contributes to EHEC virulence by cleaving fucose from mucin, thereby activating the FusKR signalling cascade, modulating the virulence gene expression of EHEC. Our findings suggest that EHEC uses fucose, a host-derived signal made available by the microbiota, to modulate EHEC pathogenicity and metabolism.

The gastrointestinal tract is inhabited by trillions of commensal bacteria that play crucial roles in human physiology ${}^{1}$. This fundamental relationship between the host and microbiota relies on chemical signalling and nutrient availability ${}^{2}$, and invading pathogens compete for these resources through the precise coordination of virulence traits. EHEC colonizes the colon, leading to haemorrhagic colitis ${}^{7}$. EHEC colonization depends on the locus of enterocyte effacement (LEE) pathogenicity island ${}^{7}$. This pathogenicity island encodes a regulator for its own expression, *ler*, and a type III secretion system, a molecular syringe which injects effectors into the host cell, leading to the formation of attaching and effacing lesions on enterocytes. Attaching and effacing lesions are characterized by remodelling of the host cell cytoskeleton, leading to the formation of a pedestal-like structure beneath the bacteria ${}^{7}$. LEE expression is regulated by an inter-kingdom chemical signalling system involving the host hormones adrenaline and/or noradrenaline and the microbial flora produced signal autoinducer-3 (AI-3) ${}^{8}$. These signals are sensed by two histidine sensor kinases, QseC (ref. 3) and QseE (ref. 4), which initiate a signalling cascade that promotes virulence.

Histidine sensor kinases, together with response regulators, comprise a two-component system, which plays a major role in bacterial signal transduction. Upon sensing a signal, the histidine sensor kinases autophosphorylates and then transfers its phosphate to the response regulator. Subsequently, most response regulators bind DNA, promoting changes in gene expression ${}^{9}$. The cognate response regulator for QseC is QseB and for QseE it is QseF (Fig. 1a). QseBC and QseEF repress expression of the z0462 and z0463 genes (Fig. 1b) ${}^{10,11}$. QseB repression of z0462/z0463 expression is direct, whereas QseF-mediated repression is indirect (Fig. 1c, d), in agreement with QseF being a $\sigma^{54}$-dependent transcriptional activator ${}^{12}$. QseF activates the expression of a repressor of z0462/z0463.

The z0462/z0463 genes are within a pathogenicity island (O-island 20 (OI-20)) ${}^{13}$, which is found in EHEC O157:H7 strains and enteropathogenic *E. coli* strains exclusively from the O55:H7 serotype (which gave rise to the O157:H7 serotype), but absent in all other *E. coli* strains whose genomes are currently publicly available. This pathogenicity island is organized in three transcriptional units (Supplementary Fig. 1). The genes z0462/z0463 encode for a putative two-component system: z0462 encodes a histidine sensor kinase with eight transmembrane domains that shares similarity (around 30% similarity at the amino acid level) with a glucose-6-phosphate sensor, UhpB; z0463 encodes a response regulator with a receiver and a DNA-binding domain.

![Figure 1](#)

Figure 1 | The two-component system FusKR of EHEC. a, The QseC/QseE signalling cascade. QseC senses AI-3 and adrenaline and noradrenaline. QseE senses adrenaline and noradrenaline and SO${}_{4}$ and PO${}_{4}$. QseC phosphorylates QseB (which activates flagella), KpdE (activates the LEE) and QseF. QseE only phosphorylates QseF. QseBC and QseEF repress the expression of z0462 and z0463. AE, attaching and effacing. b, qRT–PCR of z0462 in wild type (WT), $\Delta$qseB, $\Delta$qseC, $\Delta$qseE and $\Delta$qseF in DMEM media. Gene expression is represented as fold differences normalized to wild type. Error bars indicate standard deviations of $\Delta$ddCt values ($n = 18$ biological samples per strain; $*P < 0.01$, $**P < 0.001$, Student’s $t$-test). c, Electrophoretic mobility shift assay (EMSA) of z0463 with QseB and QseF. d, EMSA of *qseE* (positive control) with QseF. e, Autophosphorylation of Z0462 in liposomes (top panel), and a Coomassie gel of Z0462 (lower panel) (loading control). f, Phosphotransfer from Z0462 (in liposomes) to Z0463 (ratio 1 HK: 4 RR) (top panel), loading control Coomassie gel of Z0462 and Z0463 (lower panel).

${}^{1}$Department of Microbiology, University of Texas Southwestern Medical Center, Dallas, Texas 75390-9048, USA. ${}^{2}$Department of Biochemistry, University of Texas Southwestern Medical Center, Dallas, Texas 75390-9048, USA. ${}^{3}$Division of Infections Diseases, Brigham and Women’s Hospital, Harvard Medical School, Boston, Massachusetts 02115, USA.

domain (Supplementary Fig. 2). Z0462 in liposomes is a functional histidine sensor kinase (Fig. 1e) and it transfers its phosphate to Z0463 (Fig. 1f). Hence, Z0462 and Z0463 constitute a cognate two-component system.

Transcriptomic studies (Supplementary Tables 4 and 5) indicated that Z0462/Z0463 act mainly as repressors of transcription. Z0462 and Z0463 repress LEE gene expression (Supplementary Fig. 3). Transcription of all LEE operons is increased in Δz0462 (deletion mutant of z0462) and Δz0463 (deletion mutant of z0463) and plasmid complementation restored the expression of *ler* to wild-type levels (Fig. 2a–d). Transcription of the LEE genes is activated by Ler<sup>14</sup>. The response regulator Z0463 directly represses *ler* transcription, and subsequently the other LEE operons, and phosphorylation of Z0463 increases its affinity to *ler* (Fig. 2e, f and Supplementary Figs 6–8).

As expected with the increased LEE transcription, both Δz0462 and Δz0463 secreted more EspB, a LEE-encoded protein (Fig. 2g), and formed more pedestals than wild type (Fig. 2h, i). Therefore, Z0462/Z0463 repress attaching and effacing lesion formation. However, expression of other genes encoding non-LEE-encoded type III secretion system effectors, not involved in attaching and effacing lesion formation, are activated by Z0462/Z0463 (Supplementary Fig. 4).

Expression of *z0463* is increased by the mucus produced by intestinal HT29 cells. EHEC-infected undifferentiated HT29 cells were used as negative controls, because they do not produce mucus (Fig. 3a and Supplementary Fig. 9). Z0462 is a predicted hexose-phosphate sensor, hence, Z0462 may sense sugars in the mucus. Fucose is a major component of mucin glycoproteins, it is abundant in the intestine<sup>5</sup>, and fucose utilization is important for EHEC intestinal colonization<sup>15,16</sup>. In

![Figure 2](attachment://image.png)

Figure 2 | *z0462/z0463* regulates LEE expression. **a**, qRT–PCR of LEE genes in wild type and Δz0462 in DMEM media (*n* = 18; error bars, s.d.; **\*\***P < 0.001). **b**, qRT–PCR of *ler* in wild type, Δz0462 and z0462+ (Δz0462 rescued with plasmid complementation of *z0462*) in DMEM media (*n* = 18; error bars, s.d.; NS, not significant; ***P** < 0.0001). **c**, qRT–PCR of LEE genes in WT and Δz0463 in DMEM media (*n* = 18; error bars, s.d.; **\*\***P < 0.001, ***P** < 0.0001). **d**, qRT–PCR of *ler* in WT, Δz0463 and z0463+ (Δz0463 rescued with plasmid complementation of *z0463*) in DMEM (*n* = 18; **\*\***P ≤ 0.001, ***P** < 0.0001; NS, not significant *P* > 0.05; Student’s *t*-test). **e**, Representation of the Ler and

Z0463 regulation of the LEE operons. **f**, EMSA of *ler* with Z0463 with *ler* and *kan* (kanamycin) cold probes. Ratios represent hot:cold probe. **g**, Western blot of EspB in supernatants of wild type, Δz0462, z0462+, Δz0463 and z0463+ strains. BSA was added as a loading control. **h**, Fluorescent actin staining assay of HeLa cells infected with wild-type EHEC, Δz0462, z0462+, Δz0463 and z0463+, stained with fluorescein isothiocyanate–phalloidin (actin, green) and propidium iodide (bacterial and HeLa DNA, red). Original magnification, ×63. **i**, Quantification of fluorescent actin staining assay (*n* = 600 cells; ***P** < 0.0001; *P* > 0.05 = NS; Student’s *t*-test; error bars, s.d.).

Figure 3 | Z0462 and Z0463 is a fucose-sensing two-component system.

a, qRT–PCR of z0463 in wild type in the presence of undifferentiated non-mucus-producing HT29 or differentiated mucus-producing HT29 cells. Error bar indicates standard deviations of ΔddCt values (n = 18; ***P < 0.0001; Student’s t-test). b, qRT–PCR of fucose-utilization genes in wild-type EHEC, Δz0462 and Δz0463 in DMEM (D<sub>600 nm</sub> = 1.0) (n = 18; †P ≤ 0.05; *P ≤ 0.01; **P ≤ 0.001; Student’s t-test). c, Growth curves of wild type, Δz0462 and Δz0463 strains in M9 minimal-media with L-fucose as a sole carbon source. (n = 6; significance between generation times calculated through analysis of variance (ANOVA) P ≤ 0.01, error bars too small to be visible on the graph). d, FusK autophosphorylation (in liposomes) in the presence of L-fucose, D-glucose or D-ribose (top panel), and Coomassie gel of FusK in liposomes (lower panel) (loading control). No signal indicates no activating sugar signal added. e, Quantification of FusK autophosphorylation. Phosphorylation represented as fold-change compared to absence of signal. Error bar, standard deviation of fold-change values. (n = 3; ***P < 0.0001; P > 0.05 = NS; Student’s t-test). f, Schematic representation of the fusRK operon to z0461. MFS, major facilitator superfamily. g, qRT–PCR of z0461 in wild type and ΔfusK (also called Δz0462) (n = 18; **P < 0.001; Student’s t-test). h, Growth curves of wild type and Δz0461 in M9 medium with fucose as a sole carbon source (n = 6; significance between generation times calculated through ANOVA P ≤ 0.01, error bars too small to be visible on the graph). i, qRT–PCR of fucA, fucP and fucR in wild type and Δz0461 (n = 18, **P < 0.001; Student’s t-test).

E. coli, L-fucose utilization requires the fuc genes and their activator FucR (ref. 17). Z0462/Z0463 repress the expression of the fuc genes (Fig. 3b), and Δz0462 and Δz0463 grow faster with L-fucose as a sole carbon source compared to wild type (generation times of 92.4 min for wild type, 64 min for Δz0462 and 74 min for Δz0463) (Fig. 3c). Therefore, Z0462/Z0463 regulates fucose utilization, and this response is specific to fucose, with the mutants and wild type growing at similar rates with other carbon sources (Supplementary Fig. 10). Z0462 senses fucose, but not glucose or D-ribose (Fig. 3d, e). The concentration of fucose used (100 μM) is physiologically relevant to the mammalian intestine<sup>18</sup>. Hence we renamed the Z0462 protein as FusK for fucose-sensing histidine kinase and its cognate response regulator, and Z0463 as FusR for fucose-sensing response regulator.

FusKR shares homology to the UhpAB two-component system. UhpAB senses glucose-6-phosphate and activates expression of the uhpT gene which encodes a hexose-phosphate major facilitator super-family transporter<sup>19–21</sup>. FusKR represses transcription of the z0461 gene downstream of fusKR, which encodes a predicted major super-family transporter (Fig. 3f, g and Supplementary Figs 3, 11). Δz0461 has decreased growth (lower concentration of cells) with fucose as a sole carbon source (generation times of 88.2 min for wild type and 96.6 min for Δz0461) (Fig. 3h), but grows with glucose similarly to wild type (Supplementary Fig. 12), indicating that z0461 is involved in optimal fucose import. Transcription of the fuc operons is linked to fucose uptake, fucose yields fuculose-1-phosphate which is the inducer of the FucR activator of the fuc operons<sup>17,21–23</sup>. Transcription of the fuc genes is decreased in Δz0461 (Fig. 3i). Fucose induces FusKR, which represses z0461, decreasing fucose import and the intracellular levels of the fuculose-1-phosphate inducer of FucR. In further support of this indirect regulation of the fuc genes, FusR does not bind to the fucPIKUR promoter region (Supplementary Fig. 13), in contrast to the direct regulation of the LEE (Fig. 2).

The ΔfusK is not responsive to fucose, given that expression of ler is repressed by fucose in wild type, but not in ΔfusK (Fig. 4a). B. theta-iotaomicron produces multiple fucosidases that cleave fucose from host glycans, resulting in high fucose availability in the lumen<sup>2</sup>. B. thetaiotaomicron supplies mucin-derived fucose to EHEC, reducing ler expression, whereas in free fucose there is no change in ler expression whether B. thetaiotaomicron is present or not (Fig. 4b). Expression of ler is decreased when EHEC is grown in mucin compared to fucose (Fig. 4b), consistent with the increased expression of fusR in mucin (Fig. 3a).

In vitro competitions between ΔfusK and wild type, and Δler (which does not express the LEE) in the presence or absence of B. thetaiotaomicron, with either fucose or mucin as a sole carbon source were performed. The competition index between ΔfusK and wild type was 1 (Supplementary Fig. 14a) both in the absence or presence of B. thetaiotaomicron during growth in fucose, indicating

The important role FusK has in the metabolism and virulence of EHEC means it is also important for intestinal colonization. Competition assays in infant rabbits demonstrated that the wild type outcompeted ΔfusK tenfold (competition index of 0.12), which is statistically different (*P* = 0.039) from a control competition assay, in which the wild type (*lacZ+*) was competed against a ΔlacZ (competition index of 0.7) (Fig. 4c). Hence, a functional FusK is necessary for robust EHEC intestinal colonization. To determine whether the decreased ability of ΔfusK to colonize the mammalian intestine was due to uncontrolled expression of the LEE and/or fucose utilization, we performed competition experiments between wild type and ΔfusKΔfucR, which does not express the *fuc* genes. The double mutant was outcompeted by the wild type with a similar competition index to the ΔfusK and wild type competition (Fig. 4c), indicating that fucose utilization does not play a major role in FusK-mediated intestinal colonization, and the burden of LEE overexpression by ΔfusK is a stronger determinant of its decreased fitness within the intestine.

FusKR repression of LEE expression in the mucus layer prevents superfluous energy expenditure. Once in close contact to the epithelial surface, the QseCE adrenergic-sensing systems are triggered to activate virulence both directly through the QseCE cascade, and indirectly by repression of *fusKR* (Supplementary Fig. 15). EHEC competes with commensal *E. coli* (*γ*-Proteobacteria), but not with *B. thetaiotaomicron*, for the same carbon sources (for example, fucose) within the mammalian intestine^{15,24–28}. Commensal *E. coli*, however, are not found in close contact with the epithelia, being in the mucus layer, where it is counterproductive for EHEC to invest resources to utilize fucose, when EHEC can efficiently use other carbon sources such as galactose, hexuronates or mannose, which are not used by commensal *E. coli* within the intestine^{15}. Additionally, in contrast to commensal *E. coli*, EHEC is found closely associated with the intestinal epithelium^{25}. Therefore, EHEC can utilize nutrients exclusively available at the surface of the epithelial cells. Consequently, the decreased expression of the *fuc* operon through fucose-sensing by FusKR (Fig. 3) may prevent EHEC from expending energy in fucose utilization in the mucus layer, where it competes with commensal *E. coli* for this resource, and focus on using other carbon sources, not used by that competitor. Thus, the colonization defect of ΔfusK results from its inability to correctly time virulence and metabolic gene expression.

**Figure 4 | FusK in pathogen-microbiota-host associations.**  
a, qRT-PCR of *ler* in WT or ΔfusK. RNAs extracted from cultures grown in M9 with either D-glucose or L-fucose as sole carbon sources. Error bar indicates standard deviations of ΔddCt values (*n* = 18; ††*P* < 0.02; \**P* < 0.01; *P* > 0.05 = NS; Student’s *t*-test).  
b, qRT-PCR of *ler* in WT in the presence or absence of *B. thetaiotaomicron*. RNA samples from cultures grown in DMEM containing L-fucose or mucin. Error bar, standard deviations of ΔddCt values. (*n* = 18; **\**P* < 0.01; ***\**P* < 0.0001; *P* > 0.05 = NS; Student’s *t*-test).  
c, Competition assays between WT and ΔfusK or ΔfusKΔfucR. Infant rabbits were orogastrically inoculated with 1:1 mixtures of ΔfusK and wild-type EHEC or ΔlacZ and wild-type EHEC (which is *lacZ+*) or ΔfusKΔfucR and wild-type EHEC. Colony-forming units in the mid-colon were determined 2 days post-inoculation. Each point represents a competition index (CI). Horizontal lines represent the geometric mean value for each group (*n* = 2 litters (6–11 animals) ΔlacZ and ΔfusKΔfucR, *n* = 3 litters (11 animals) ΔfusK; asterisk, †*P* < 0.05; Mann–Whitney test).

that, in the presence of free fucose, *B. thetaiotaomicron* does not affect the competition between ΔfusK and wild type, and that the growth advantage of ΔfusK in fucose is counteracted by decreased LEE expression in wild type. When these experiments were performed with mucin as a sole carbon source the competition index between ΔfusK and WT was 0.1 in the absence, and 1 in the presence of *B. thetaiotaomicron* (Supplementary Fig. 14b). In the absence of *B. thetaiotaomicron* there is no free fucose, therefore ΔfusK will not have a growth advantage. Furthermore, the ΔfusK overexpresses genes from the LEE, which constitutes an energy burden. Meanwhile, expression of *fusR* is activated in mucus (Fig. 3a), further repressing the LEE genes in the wild type. This scenario, however, reverts to a competition index of 1 in the presence of *B. thetaiotaomicron*, which releases fucose from mucin, conferring a growth advantage to ΔfusK, counteracting the wild type previous growth advantage (Supplementary Fig. 14b). Similar results were obtained in competitions between ΔfusK and Δ*ler*, consistent with the role of LEE gene expression being an energy burden in ΔfusK (Supplementary Fig. 14c).

**METHODS SUMMARY**

Strains and plasmids are listed in Supplementary Tables 1 and 2. Molecular biology techniques were performed as previously described^{29}. Primers used in quantitative reverse transcription PCR (qRT-PCR) and cloning are listed in Supplementary Table 3. Construction of isogenic mutants was performed using a lambda-red method as previously described^{30}. FusK liposome studies were performed as previously described^{3}.

Full Methods and any associated references are available in the online version of the paper.

Received 23 May; accepted 27 September 2012.  
Published online 18 November 2012.

1. Kau, A.L., Ahern, P.P., Griffin, N.W., Goodman, A.L. & Gordon, J.I. Human nutrition, the gut microbiome and the immune system. *Nature* **474**, 327–336 (2011).
2. Fischbach, M.A. & Sonnenburg, J.L. Eating for two: how metabolism establishes interspecies interactions in the gut. *Cell Host Microbe* **10**, 336–347 (2011).
3. Clarke, M.B., Hughes, D.T., Zhu, C., Boedeker, E.C. & Sperandio, V. The QseC sensor kinase: a bacterial adrenergic receptor. *Proc. Natl Acad. Sci. USA* **103**, 10420–10425 (2006).
4. Reading, N.C., Rasko, D.A., Torres, A.G. & Sperandio, V. The two-component system QseEF and the membrane protein QseG link adrenergic and stress sensing to bacterial pathogenesis. *Proc. Natl Acad. Sci. USA* **106**, 5889–5894 (2009).
5. Robbe, C., Capon, C., Coddeville, B. & Michalski, J.C. Structural diversity and specific distribution of O-glycans in normal human mucins along the intestinal tract. *Biochem. J.* **384**, 307–316 (2004).
6. Xu, J. et al. A genomic view of the human-*Bacteroides thetaiotaomicron* symbiosis. *Science* **299**, 2074–2076 (2003).
7. Kaper, J.B., Nataro, J.P. & Mobley, H.L. Pathogenic *Escherichia coli*. *Nature Rev. Microbiol.* **2**, 123–140 (2004).

8. Sperandio, V., Torres, A. G., Jarvis, B., Nataro, J. P. & Kaper, J. B. Bacteria–host communication: the language of hormones. *Proc. Natl Acad. Sci. USA* **100**, 8951–8956 (2003).

9. Stock, A. M., Robinson, V. L. & Goudreau, P. N. Two-component signal transduction. *Annu. Rev. Biochem.* **69**, 183–215 (2000).

10. Hughes, D. T., Clarke, M. B., Yamamoto, K., Rasko, D. A. & Sperandio, V. The QseC adrenergic signaling cascade in enterohemorrhagic *E. coli* (EHEC). *PLoS Pathog.* **5**, e1000553 (2009).

11. Reading, N. C., Rasko, D., Torres, A. G. & Sperandio, V. A transcriptome study of the QseEF two-component system and the QseG membrane protein in enterohaemorrhagic *Escherichia coli* O157:H7. *Microbiology* **156**, 1167–1175 (2010).

12. Barrios, H., Valderrama, B. & Morett, E. Compilation and analysis of $\sigma^{54}$-dependent promoter sequences. *Nucleic Acids Res.* **27**, 4305–4313 (1999).

13. Perna, N. T. *et al.* Genome sequence of enterohaemorrhagic *Escherichia coli* O157:H7. *Nature* **409**, 529–533 (2001).

14. Mellies, J. L., Barron, A. M. & Carmona, A. M. Enteropathogenic and enterohemorrhagic *Escherichia coli* virulence gene regulation. *Infect. Immun.* **75**, 4199–4210 (2007).

15. Fabich, A. J. *et al.* Comparison of carbon nutrition for pathogenic and commensal *Escherichia coli* strains in the mouse intestine. *Infect. Immun.* **76**, 1143–1152 (2008).

16. Snider, T. A., Fabich, A. J., Conway, T. & Clinkenbeard, K. D. E. *coli* O157:H7 catabolism of intestinal mucin-derived carbohydrates and colonization. *Vet. Microbiol.* **136**, 150–154 (2009).

17. Chen, Y. M., Zhu, Y. & Lin, E. C. The organization of the *fuc* regulon specifying L-fucose dissimilation in *Escherichia coli* K12 as determined by gene cloning. *Mol. Gen. Genet.* **210**, 331–337 (1987).

18. Jaswal, V. M., Babbar, H. S. & Mahmood, A. Changes in sialic acid and fucose contents of enterocytes across the crypt-villus axis in developing rat intestine. *Biochem. Med. Metab. Biol.* **39**, 105–110 (1988).

19. Island, M. D., Wei, B. Y. & Kadner, R. J. Structure and function of the *uhp* genes for the sugar phosphate transport system in *Escherichia coli* and *Salmonella typhimurium*. *J. Bacteriol.* **174**, 2754–2762 (1992).

20. Weston, L. A. & Kadner, R. J. Identification of Uhp polypeptides and evidence for their role in exogenous induction of the sugar phosphate transport system of *Escherichia coli* K-12. *J. Bacteriol.* **169**, 3546–3555 (1987).

21. Weston, L. A. & Kadner, R. J. Role of *uhp* genes in expression of the *Escherichia coli* sugar-phosphate transport system. *J. Bacteriol.* **170**, 3375–3383 (1988).

22. Zhu, Y. & Lin, E. C. An evolvant of *Escherichia coli* that employs the L-fucose pathway also for growth on L-galactose and D-arabinose. *J. Mol. Evol.* **23**, 259–266 (1986).

23. Chen, Y. M., Tobin, J. F., Zhu, Y., Schleif, R. F. & Lin, E. C. Cross-induction of the L-fucose system by L-rhamnose in *Escherichia coli*. *J. Bacteriol.* **169**, 3712–3719 (1987).

24. Kamada, N. *et al.* Regulated virulence controls the ability of a pathogen to compete with the gut microbiota. *Science* **336**, 1325–1329 (2012).

25. Miranda, R. L. *et al.* Glycolytic and gluconeogenic growth of *Escherichia coli* O157:H7 (EDL933) and *E. coli* K-12 (MG1655) in the mouse intestine. *Infect. Immun.* **72**, 1666–1676 (2004).

26. Chang, D. E. *et al.* Carbon nutrition of *Escherichia coli* in the mouse intestine. *Proc. Natl Acad. Sci. USA* **101**, 7427–7432 (2004).

27. Autieri, S. M. *et al.* L-fucose stimulates utilization of D-ribose by *Escherichia coli* MG1655 $\Delta f u c A O$ and *E. coli* Nissle 1917 $\Delta f u c A O$ mutants in the mouse intestine and in M9 minimal medium. *Infect. Immun.* **75**, 5465–5475 (2007).

28. Fox, J. T., Drouillard, J. S., Shi, X. & Nagaraja, T. G. Effects of mucin and its carbohydrate constituents on *Escherichia coli* O157 growth in batch culture fermentations with ruminal or fecal microbial inoculum. *J. Anim. Sci.* **87**, 1304–1313 (2009).

29. Sambrook, J., Fritsch, E. F. & Maniatis, T. *Molecular Cloning: a Laboratory Manual* 2nd edn (Cold Spring Harbor Laboratory Press, 1989).

30. Datsenko, K. A. & Wanner, B. L. One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. *Proc. Natl Acad. Sci. USA* **97**, 6640–6645 (2000).

**Supplementary Information** is available in the online version of the paper.

**Acknowledgements** We thank M. Kendall for comments. We thank the microarray core facility. This work was supported by the National Institutes of Health (NIH) Grants AI053067, AI77853 and AI077613, and the Burroughs Wellcome Fund (to V.S.) and NIH Grant AI42347 and HHMI (to M.K.W.). M.M.C. was supported through NIH Training Grant 5T32 AI7520-14. The contents are solely the responsibility of the authors and do not represent the official views of the NIH NIAID.

**Author Contributions** A.R.P. led the project and performed experiments, designed experiments and wrote the paper. M.M.C. J.M.R., D.M., M.K.W. and C.G.M. helped with some experiments. V.S. designed experiments and wrote the paper.

**Author Information** Microarray data are deposited in the Gene Expression Omnibus under accession number GSE34991. Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to V.S. (vanessa.sperandio@utsouthwestern.edu).

RESEARCH LETTER

# METHODS

**Bacterial strains, plasmids and growth conditions.** Strains and plasmids are listed in Supplementary Tables 1 and 2. *E. coli* strains were grown aerobically at 37 °C in DMEM (Gibco) or LB media unless otherwise stated. For studies involving fucose utilization, bacterial cultures were grown in M9 minimal media containing 0.4% L-fucose, D-glucose, L-rhamnose, D-galactose or D-xylose (Sigma) as the sole carbon source. For the co-culture experiments between EHEC and *B. thetaiotamicron*, these strains were grown anaerobically at 37 °C in DMEM buffer (lacking glucose and pyruvate) with or without mucin or free fucose, at a 1:1 ratio of EHEC to *B. thetaiotamicron*. Enumeration of EHEC was performed through serial dilution of these cultures in McConkey agar containing streptomycin (EHEC strain 86-24 is streptomycin resistant, whereas *B. thetaiotamicron* is streptomycin sensitive). Enumeration of *B. thetaiotamicron* was performed through serial plating in TYG medium supplemented with 10% horse blood in the presence of gentamycin (*B. thetaiotamicron* is gentamycin resistant, whereas EHEC is gentamycin sensitive).

**Recombinant DNA techniques.** Molecular biology techniques were performed as previously described²⁹. Primers used in qRT–PCR and cloning are listed in Supplementary Table 2.

**Isogenic mutant construction.** Construction of isogenic ΔfusK, ΔfusR, Δz0461, Δler and ΔfusKΔfucR mutants was performed using a lambda-red mediated recombination method as previously described³⁰. Primers used to construct these knockouts are depicted in Supplementary Table 3. Briefly, a PCR product used to create a mutation was generated using primers containing homologous regions to sequences flanking z0462 (for ΔfusK mutant), z0463 (for ΔfusR mutant), z0461, ler, and fucR to amplify a chloramphenicol resistance gene from pKD3. EHEC 86-24 cells harbouring pKD46 were electroporated using the PCR product and selected for chloramphenicol resistance. Nonpolar mutants were generated by resolving the chloramphenicol-resistant clones with resolvase encoded by pCP20. For complementation of the mutants, z0462 and z0463 previously cloned into ZeroBlunt TOPO (Invitrogen), were digested with BamHI and SalI then cloned into pACYC184, generating the plasmids pARP12 and pARP13, respectively. pARP12 was electroporated into ΔfusK to generate the ARP09-complemented strain; pARP13 was electroporated into ΔfusR cells to generate the ARP10-complemented strain.

**FusR purification.** FusR was cloned into ZeroBlunt TOPO, digested using XhoI and HindIII restriction enzymes and then cloned into pBADMycHisA, generating pARP11. pARP11 was subsequently transformed into TOP10 cells, generating the ARP04 strain. ARP04 strain was grown in LB to $D_{600nm} = 0.6$ at 37 °C, then protein expression was induced by addition of arabinose (final concentration of 0.2%) and overnight growth at 25 °C. FusR was subsequently purified using nickel columns (Qiagen).

**Nested deletion analyses.** Transcriptional fusions of the ler promoter with promoterless lacZ have been described previously³¹. To integrate the transcriptional fusions into the chromosome, *E. coli* MC4100 was lysogenized with phage λ45, generating strains FS14 and FS16.

**FusK purification and reconstitution into liposomes.** FusK was cloned into ZeroBlunt TOPO, digested using XhoI and HindIII restriction enzymes and then cloned into pBADMycHisA. This plasmid was subsequently transformed into TOP10 cells, generating the ARP03 strain. The ARP03 strain was grown in LB media at 37 °C until $D_{600nm} = 0.5$ then protein expression was induced by addition of arabinose (final concentration of 0.2%) and growth for 5 h at 30 °C. Cells were collected, resuspended in 50 ml of lysis buffer (50 mM phosphate buffer pH 8.0, 1% deoxycholic acid, 10 mM imidazol, 300 mM NaCl, 15% glycerol, 5 mM dithiothreitol (DTT) and 100 μl protease inhibitor cocktail), then lysed using emulsiflex. Lysates were incubated for 1 h for solubilization and then isolated by centrifugation at 18,000 r.p.m. for 30 min. Soluble fraction was collected by ultracentrifugation at 45,000 r.p.m. for 1 h to obtain membrane fraction, then membranes were resuspended in lysis buffer and incubated with nickel beads for 1 h at 4 °C with gentle agitation. The membrane suspension and clear lysates were loaded into nickel-nitriloacetic acid (Ni-NTA) columns, washed with wash buffer (50 mM phosphate buffer pH 8.0, 20 mM imidazol, 300 mM NaCl, 5 mM DTT and 0.1% deoxycholic acid) and eluted in three steps with elution buffer (250 mM imidazol, 300 mM NaCl, 1 mM DTT and 0.1% deoxycholic acid). Protein was concentrated using centricons with a molecular cutoff of 30,000 kDa, then protein concentration was determined by the Bradford protein assay method. Liposomes were loaded with FusK at a ratio of 20:1. Liposomes were reconstituted as described previously³². FusK presence in liposomes was confirmed by western blot using anti-Myc antibody (Invitrogen).

**Autophosphorylation and phosphotransfer assays.** Autophosphorylation assays were performed as described previously³. A concentration of 100 μM L-fucose or D-glucose was used. The bands were quantified using ImageQuant software version 3.2. Quantification of triplets was performed. The Student’s t-test was used to determine statistical significance. A P-value of less than 0.05 was considered significant.

**RNA extraction and qRT–PCR.** RNA from six biological replicates (experiments performed three independent times, for a total of 18 independent biological replicates) was extracted using RiboPure kit according to the manufacturer’s instructions (Invitrogen). Primer validation and real-time PCR was performed as described previously³³. Gene expression is represented as fold differences compared to the wild type EHEC strain 86-24. Error bars indicate the standard deviations of the $\Delta\Delta C_t$ values. The Student t-test was used to determine statistical significance. A P value of less than 0.05 was considered significant.

**Microarrays.** Microarray experiments and analysis were performed as previously described³⁴. The GeneChip *E. coli* Genome 2.0 array system (Affymetrix) was used to compare the gene expression in EHEC strain 86-24 to that of ΔfusK and ΔfusR strains. The output from scanning the Affymetrix GeneChip *E. coli* 2.0 was obtained using GCOS software v 1.4 according to the manufacturer’s instructions. Comparisons were performed using the analysis tools within GCOS v 1.4, by selecting the appropriate array, CHP file for comparison and baseline values. Custom analysis scripts were written in Perl to complete multiple array analyses. Expression data can be accessed using accession number (GSE34991) at the NCBI GEO database.

**Fluorescent actin staining (FAS) assay.** Fluorescein actin staining (FAS) assays were performed as previously described³⁵. Pedestal enumeration was performed in 600 infected cells. The Student t-test was used to determine statistical significance. A P-value of less than 0.05 was considered significant.

**In vitro competition assays.** Bacterial strains were grown for 18 h in LB at 37 °C, resuspended in DMEM without glucose and inoculated 1:100 in DMEM (without glucose, without pyruvate) containing fucose or mucin as the sole carbon source. *B. thetaiotamicron* was grown in TYG medium, resuspended in DMEM (without glucose) and inoculated at a 1:9 ratio. In vitro competitions were carried out anaerobically and samples were collected hourly for serial dilution and plating for the colony-forming units count. A competition index was determined by the ratio of ΔfusK to wild-type EHEC or the ratio of ΔfusK to Δler.

**Infant rabbit infection studies.** Litters of 3-day-old infant rabbits were infected as described previously³⁶. Individual rabbits were orogastrically inoculated (approximately $5 \times 10^8$ colony-forming units per 90 g of animal’s weight) with 1:1 mixtures of wild-type (ΔlacZ) EHEC and the ΔfusK of ΔfusKΔfucR. The animals were killed 2 days postinoculation and colonic tissue samples removed and homogenized before microbiological analysis. The number of wild-type and fusK mutant cells present in the tissue homogenate was determined by serial dilution and plating on media containing streptomycin and 5-bromo-4-chloro-3-indolyl-β-D-galactoside (X-gal) as previously described³⁶. Competition indexes were calculated as the ratio of ΔfusK to wild type or ΔfusKfucR to wild type in tissue homogenates divided by the ratio of ΔfusK to wild type or ΔfusKfucR to wild type in the input. The competition index was compared to the competition index value obtained when otherwise isogenic lacZ+ (wild type) and ΔlacZ strains were given to rabbits. Differences *in cis* were compared using the Mann–Whitney test, where a P-value of less than 0.05 was considered significant. All animal experiments performed were approved by the IACUC offices of University of Texas Southwestern Medical Center and Brigham and Women’s Hospital. (ΔlacZ n = 2 litters (6–11 animals) and ΔfusKΔfucR, n = 3 litters (11 animals)).

**β-galactosidase activity assays.** The bacterial strains FS14, FS15 and FS16 were transformed with pFusR or empty vector (pBADMycHisA) and grown in aerobically in DMEM containing 0.2% arabinose at 37 °C to $D_{600nm} = 0.8$. The cultures were diluted 1:100 in Z buffer (60 mM Na₂HPO₄·7H₂O and 50 mM β-mercaptoethanol) and assayed for β-galactosidase activity by using o-nitrophenyl-β-D-galactopyranoside as substrate as previously described³⁷.

**Electrophoretic mobility shift assay (EMSA).** EMSAs were performed using purified FusR–Myc–His and radiolabelled probes. Primers were end-labelled using [γ³²P]ATP and T4 polynucleotide kinase (NEB), and subsequently used on a PCR to generate radiolabelled probes. End-labelled amplicons were run on a 6% polyacrylamide gel, excised and purified using a Qiagen gel extraction kit. To test the ability of FusR to bind directly to its target promoters, increasing amounts of FusR (0 to 4.35 μM) were incubated with end-labelled probe (10 ng) for 20 min at 4 °C in binding buffer (500 μg ml⁻¹ BSA, 50 ng poly(deoxynosinic-deoxycytidylic) (dIdC) acid, 6 mM HEPES, pH 7.5, 5 mM EDTA, 3 mM DTT, 300 mM KCl and 25 mM MgCl₂). A sucrose solution was used to stop the reaction²⁹. The reactions were run on a 6% polyacrylamide gel for 6.5 h at 180 V. The gels were dried under vacuum and EMSAs were visualized by autoradiography.

**DNase I footprinting.** DNase I footprint was performed as previously described³⁸. Briefly: primer Ler-18FP-R (Supplementary Table 2) was end-labelled using [γ³²P]ATP and T4 polynucleotide kinase (NEB) and used in a PCR with cold primer Ler-299FP-F(Supplementary Table 2) to generate probe LerFP. The resulting end-labelled probe was used in a binding reaction (described in the EMSA subsection) for 20 min at room temperature. At this time, a 1:100 dilution of

©2012 Macmillan Publishers Limited. All rights reserved

DNase I (NEB) and the manufacturer-supplied buffer were added to the reaction and digestion proceeded for 7 min at room temperature. The digestion reaction was stopped by addition of 100 μl of stop buffer (200 mM NaCl, 2 mM EDTA and 1% SDS). Protein extraction was performed using phenol–chloroform and DNA was precipitated using 5 M NaCl, 100% ethanol and 1 μl glycogen. The DNase reactions were run in a 6% polyacrylamide-urea gel next to a sequencing reaction (Epicentre). Amplicon generated using primers Ler-299FP-F and Ler-18FP-R (radiolabelled) was used as a template for the sequencing reaction. The footprint was visualized by autoradiography.

31. Sharp, F. C. & Sperandio, V. QseA directly activates transcription of *LEE1* in enterohemorrhagic *Escherichia coli*. *Infect. Immun.* **75**, 2432–2440 (2007).
32. Janausch, I. G., Garcia-Moreno, I., Lehnen, D., Zeuner, Y. & Unden, G. Phosphorylation and DNA binding of the regulator DcuR of the fumarate-responsive two-component system DcuSR of *Escherichia coli*. *Microbiology* **150**, 877–883 (2004).

33. Walters, M. & Sperandio, V. Quorum sensing in *Escherichia coli* and *Salmonella*. *Int. J. Med. Microbiol.* **296**, 125–131 (2006).
34. Kendall, M. M., Rasko, D. A. & Sperandio, V. Global effects of the cell-to-cell signaling molecules autoinducer-2, autoinducer-3, and epinephrine in a *luxS* mutant of enterohemorrhagic *Escherichia coli*. *Infect. Immun.* **75**, 4875–4884 (2007).
35. Knutton, S., Baldwin, T., Williams, P. H. & McNeish, A. S. Actin accumulation at sites of bacterial adhesion to tissue culture cells: basis of a new diagnostic test for enteropathogenic and enterohemorrhagic *Escherichia coli*. *Infect. Immun.* **57**, 1290–1298 (1989).
36. Ritchie. K. M., Thorpe, C. M., Rogers, A. B. & Waldor, M. K. Critical roles for *stx₂*, *eae*, and *tir* in *Escherichia coli*-induced diarrhea and intestinal inflammation in infant rabbits. *Infect. Immun.* **71**, 7129–7139 (1989).
37. Miller, J. H. *Experiments in Molecular Genetics*. (Cold Spring Harbor Laboratory Press, 1972).
38. Sperandio, V. How the bacterial flora and the epithelial cell get along. *Trends Microbiol.* **8**, 544 (2000).

©2012 Macmillan Publishers Limited. All rights reserved
