
# Structure of measles virus hemagglutinin bound to its epithelial receptor nectin-4

Xiaoai Zhang${}^{1,6}$, Guangwen Lu${}^{1,6}$, Jianxun Qi${}^{1}$, Yan Li${}^{1}$, Yan He${}^{2}$, Xiang Xu${}^{3}$, Jia Shi${}^{2}$, Catherine W-H Zhang${}^{2}$, Jinghua Yan${}^{1}$ & George F Gao${}^{1,4,5}$

Measles virus is a major public health concern worldwide. Three measles virus cell receptors have been identified so far, and the structures of the first two in complex with measles virus hemagglutinin (MV-H) have been reported. Nectin-4 is the most recently identified receptor in epithelial cells, and its binding mode to MV-H remains elusive. In this study, we solved the structure of the membrane-distal domain of human nectin-4 in complex with MV-H. The structure shows that nectin-4 binds the MV-H β4-β5 groove exclusively via its N-terminal IgV domain; the contact interface is dominated by hydrophobic interactions. The binding site in MV-H for nectin-4 also overlaps extensively with those of the other two receptors. Finally, a hydrophobic pocket centered in the β4-β5 groove is involved in binding to all three identified measles virus receptors, representing a potential target for antiviral drugs.

Although a vaccine is available for measles virus, the virus is a major cause of childhood morbidity and mortality worldwide ${}^{1,2}$, with severe outbreaks occurring in China and Europe in recent years ${}^{3}$. According to the World Health Organization, ~20 million children are affected by measles each year, and in 2010 alone the virus killed ~139,300 individuals ([http://www.who.int/mediacentre/factsheets/fs286/en/](http://www.who.int/mediacentre/factsheets/fs286/en/)). The acute clinical manifestations of measles virus infection include fever, coughing, photophobia and a symptomatic red rash over most of the body ${}^{4}$. In some persistent cases, the virus can also infect the central nervous system, leading to encephalitis and subacute sclerosing panencephalitis ${}^{5}$. In several developing countries where vaccination coverage is low, measles has become a serious public health threat.

Measles virus belongs to the *Morbillivirus* genus in the family Paramyxoviridae ${}^{4}$. The virus contains a nonsegmented negative-sense RNA genome, which encodes genes for the nucleocapsid protein (NP), an RNA polymerase (L), a phosphoprotein (P), a matrix protein (M), a receptor binding hemagglutinin (H), a membrane fusion protein (F) and virulence factors (C and V). The envelope H protein is responsible for specific interaction with cellular receptors, initiating viral infection. Glycoprotein F then mediates viral envelope fusion with the host cell membrane, leading to virus entry ${}^{4}$.

So far, three cellular receptors for MV-H have been identified, including the signaling lymphocyte activation molecule (SLAM), the complement regulatory molecule CD46 and the epithelial protein nectin-4 (refs. 6–10). Although both wild-type and vaccine strain viruses can use SLAM and nectin-4, CD46 is recognized mainly by the vaccine strain of the Edmonston virus ${}^{7,11,12}$. In addition to their variant binding characteristics for different MV-Hs, these three receptors also exhibit differential tissue and cell expression patterns. CD46 is detected on all nucleated cells ${}^{13}$, and SLAM is expressed mainly on the surface of activated B, T and dendritic cells and monocytes ${}^{14}$. Nectin-4 is the most recently identified measles virus receptor, and it is found on epithelial cells ${}^{6,10}$. Nectin-4 is highly expressed in various tissues such as the human airway, skin, lung, prostate and stomach ${}^{15,16}$.

Structural elucidation of the MV-H–SLAM and MV-H–CD46 complexes, and of MV-H alone, has recently been reported ${}^{17-20}$. These structures (with MV-H derived from the vaccine strain of the Edmonston virus) reveal a cubic six-bladed β-propeller fold for the H protein. A concave side groove formed by the β4 and β5 blades of MV-H is the receptor-binding site, accommodating both SLAM and CD46. However, the binding mode between MV-H and nectin-4 has not yet been elucidated. This issue is notable because earlier mutational studies identify both common and variant key residues used by MV-H to interact with these three receptors ${}^{19,21-24}$, indicating possibly overlapping receptor-binding footprints for nectin-4, CD46 and SLAM ${}^{19}$.

Nectin-4 belongs to the nectin and nectin-like (Necl) family of proteins, which includes nine members (nectin-1 to nectin-4 and Necl-1 to Necl-5) ${}^{25}$. Members of this protein family contain three immunoglobulin-like domains (V-C-C sets) and have an important role in mediating cell-cell adhesion ${}^{26,27}$. The nectin and Necl proteins are also involved in other processes, including cell polarization, differentiation, movement, proliferation, survival and immune recognition ${}^{27}$. Homophilic (*cis* or *trans*) or heterophilic dimerization is believed to be important in facilitating their functions ${}^{28}$. Nectin-4 is involved in adherens junction formation via trans-homophilic

---

1 CAS Key Laboratory of Pathogenic Microbiology and Immunology, Institute of Microbiology, Chinese Academy of Sciences, Beijing, China.  
2 Beijing QuantoBio Biotechnology Co. Ltd., Beijing, China.  
3 College of Life Science, Anhui Agricultural University, Hefei, China.  
4 Research Network of Immunity and Health, Beijing Institutes of Life Science, Chinese Academy of Sciences, Beijing, China.  
5 Chinese Center for Disease Control and Prevention, Changping District, Beijing, China.  
6 These authors contributed equally to this work. Correspondence should be addressed to G.F.G. (<gaof@im.ac.cn>).

Received 15 June; accepted 1 October; published online 2 December 2012; doi:10.1038/nsmb.2432

ARTICLES

Table 1 Data collection and structure refinement statistics

| MV-H–nectin-4v |
|------------------|
| **Data collection** |
| Space group | $C222_{1}$ |
| Cell dimensions $a, b, c (\AA)$ | 82.4, 171.8, 117.4 |
| Resolution ($\AA$) | 50-3.1 (3.21–3.10) |
| $R_{\text{merge}}$ | 0.076 (0.548) |
| $I / \sigma I$ | 25.4 (3.52) |
| Completeness (%) | 99.9 (100.0) |
| Redundancy | 6.9 (7.0) |

| **Refinement** |
| Resolution ($\AA$) | 37.13–3.10 |
| No. reflections | 15,471 |
| $R_{\text{work}} / R_{\text{free}}$ | 0.265/0.281 |
| No. atoms |
| Protein | 4,855 |
| Water | 79 |
| $B$-factors |
| Protein | 93.8 |
| Water | 37.6 |
| r.m.s. deviations |
| Bond lengths ($\AA$) | 0.008 |
| Bond angles ($^\circ$) | 1.080 |

A single crystal was used to collect the data. Values in parentheses are for the highest-resolution shell.

four-stranded (S1–S4) $\beta$-sheet. Nevertheless, extra components of $\alpha$-helices (H1–H4) and $\beta$-strands (Sa1 and Sa2) were observed in blades $\beta 1$, $\beta 3$ and $\beta 4$ (Supplementary Fig. 1). Wide areas of MV-H are covered by N-glycans ${ }^{18,29}$. Compared with the Edmonston MV-H protein, an extra N-glycosylation site was present in wild-type MV-H at position 416, which was generated from substitution of the aspartic acid in Edmonston MV-H by asparagine. Accordingly, we observed clear electron density for the glycan moiety at this position, covalently linked to Asn416 of MV-H (Fig. 1a).

In the asymmetric unit of the crystallized complex, each nectin-4v molecule adopted a V-set immunoglobulin fold (IgV) ${ }^{30}$ (Fig. 1c), although this fold contained an extra A' strand that is not commonly found in canonical IgV domains ${ }^{31}$. One of the two molecules bound to the concave side groove of the H protein formed by blades $\beta 4$ and $\beta 5$ (Fig. 1a,b), whereas the other was located next to the MV-H $\beta 2$ blade. Earlier structural studies on the other two MV receptors (CD46 and SLAM) both show that the $\beta 4$-$\beta 5$ groove accommodates the receptor entities ${ }^{19,20}$. Accordingly, in our structure of the complex, nectin-4v buried a total surface area of ~1,050 Å² with MV-H at the $\beta 4$-$\beta 5$ site; this was about two-fold larger than that observed for the nectin-4v molecule bound to blade $\beta 2$ (only 558 Å²). Therefore, the $\beta 2$-nectin-4v interaction is probably the result of crystal packing during complex crystallization (confirmed by further mutagenesis experiments described below). At the receptor-binding site, the C terminus of the nectin-4v molecule pointed away from the MV-H protein, corresponding well to a *trans* interaction between the virus and susceptible cells.

### MV-H dimer

MV-H exists as disulfide-linked dimers on the viral surface that further associate to form tetramers ${ }^{4}$. Earlier studies on both single MV-H and MV-H-receptor complexes all show a dimeric arrangement of the H head ${ }^{17-20}$. Despite the presence of a single H protein in the MV-H–nectin-4v structure, simple symmetry operations could yield an MV-H dimer joined predominantly via the $\beta 1$ blade of the proteins. A series of hydrophobic and aromatic residues covered the solvent-exposed surface of blade $\beta 1$, forming an extensive dimer interface (Fig. 2a). None of the two nectin-4v molecules in the asymmetric unit interfered with dimer formation.

As we expected, this $\beta 1$-mediated dimer organization was very similar to those observed for the reported MV-H dimers ${ }^{17-20}$. Superimposition of the dimeric MV-H–nectin-4v complex onto a ligand-free MV-H dimer (PDB 2RKC) yielded an r.m.s. deviation of

interaction and/or heterophilic interaction with nectin-1 (ref. 15). Most recently, nectin-4 has been recognized as the third receptor for measles virus in epithelial cells.

To elucidate the molecular basis for measles virus recognition of nectin-4, we solved the structure of the membrane-distal domain (the first V-set immunoglobulin domain) of nectin-4 (nectin-4v) in complex with the MV-H protein. The structure shows that nectin-4 binds to the $\beta 4$-$\beta 5$ groove of MV-H using its N-terminal domain, mainly via extensive hydrophobic interactions. Detailed comparison with two previously solved MV-H-receptor structures presents, for the first time, the overlapping yet variant binding sites in MV-H for nectin-4, CD46 and SLAM. Furthermore, a hydrophobic pocket centered in the MV-H $\beta 4$-$\beta 5$ groove is involved in binding to all three receptors, representing a potential target for antiviral drugs.

---

### RESULTS

#### Overall structure of MV-H–nectin-4v complex

We solved the structure of nectin-4v bound to MV-H (derived from wild-type strain IC-B) at a resolution of 3.1 Å (Table 1). Three molecules, including one MV-H protein and two nectin-4v molecules, were present in the crystallographic asymmetric unit. Nearly the entire MV-H chain (Asp156–Cys606) was traceable in the electron density map. Overall, MV-H in the complex folded into a six-bladed $\beta$-propeller structure (Fig. 1a,b), similar to that of the vaccine strain Edmonston MV-H ${ }^{17}$. Each of the six blades ($\beta 1$–$\beta 6$) comprised a

![Figure 1](#fig1)

**Figure 1** Nectin-4 binds to the $\beta 4$-$\beta 5$ groove of MV-H via its N-terminal IgV domain. (a) Structure of nectin-4v bound to MV-H. Two nectin-4v molecules are present in the asymmetric unit of the complex crystal, but only the one in the receptor binding site of MV-H is shown. MV-H $\beta$-blades are colored according to their affiliations and labeled; nectin-4v is purple. The glycan moiety attached to MV-H Asn416 and the nectin-4v C terminus are highlighted. (b) A flipped view (about 180°) of the same structure in a. To clarify the MV-H–nectin-4v binding mode, interacting MV-H blades ($\beta 4$ and $\beta 5$) are shown in surface and engaging nectin-4v loops are red. (c) Close view of nectin-4v molecule in structure of complex. Overall, nectin-4v exhibits a typical IgV-like fold. Strands are labeled accordingly.

Figure 2 Dimeric organization of the MV-H–nectin-4v complex.

(a) Surface representation of MV-H dimer yielded by symmetry operations. H protein is in green and cyan, and nectin-4v is in purple and orange. The joining interface in the β1-blade is yellow. The MV-H stalk region, which is not included in the structural study, can also contribute to the glycoprotein dimerization via disulfide bridges. The stalk and the viral membrane are indicated. (b) An alignment of the dimeric structures between the MV-H–nectin-4v complex (green) and a previously reported ligand-free MV-H (cyan). The well-aligned H entities are shown. (c) A magnified view highlighting the loop connecting the S2 and S3 strands of MV-H blade β4, which undergoes conformational change upon nectin-4v binding. The inserted nectin-4v F-G loop is also indicated.

hydrogen bonds with MV-H Gly465 and Tyr543, respectively (Fig. 3b). Site II involved the nectin-4v B-C loop and the intervening loop connecting MV-H blades β4 and β5. Residues Gln30, Gly32 and Gln33 of nectin-4v and Thr498, Tyr499 and Asp505 of MV-H were located in this site, contributing to receptor-ligand interactions via potential polar contacts. Furthermore, the apolar carbon atoms from nectin-4v Val31 and Gly32 also stacked against Leu500 of MV-H (Fig. 3c). Finally, four consecutive residues, His52–Tyr55, in the nectin-4v C′-C″ loop contacted MV-H residues Lys387–Lys389 and Gln391 in the β3 blade and Tyr499 and Leu500 in the β4-β5 loop, constituting the main interactions in site III (Fig. 3c). Overall, site I conferred strong stabilizing forces via extensive hydrophobic interactions, with key residues in this site exhibiting unambiguous electron density (Supplementary Fig. 2).

In this hydrophobic interaction-dominated binding mode, Phe101–Gly104 in nectin-4v had pivotal roles in MV-H recognition. Therefore, we mutated these amino acids and tested the binding of the resulting mutants to surface-expressed MV-H in 293T cells via flow cytometry. As we expected, simultaneous mutation of the four residues (mutations F101S, P102S, A103S and G104Y) almost completely abolished nectin-4v binding to MV-H, and even the single F101S mutation markedly impaired binding. For the nectin-4v molecule attached to the MV-H β2 blade in the structure of the complex, the main interface residues included nectin-4v Asp4 and Glu7. However, mutation of these two amino acids (mutations D4K and E7A) did not affect receptor-ligand interaction (Fig. 3d), further indicating that this β2-attached nectin-4v is only the result of crystal packing (see also results above).

The MV-H–nectin-4v binding interface

As nectin-4v residues involved in interaction with MV-H were almost exclusively localized in the F-G, B-C and C′-C″ loops, we divided the contacting interface into three parts (sites I–III; Fig. 3a). Site I was mainly the nectin-4v F-G loop interacting with two hydrophobic patches in MV-H. Residues Ser99–Phe106 of nectin-4v made extensive contacts with MV-H residues Tyr524, Leu526, Tyr541 and Tyr543 of the β5 blade on one side and Pro458, Met459, Leu462–Gly465, Leu482 and Phe483 of the β4 blade on the other. Nectin-4v Ala103 also formed two

Figure 3 Binding interface between MV-H (green) and nectin-4v (purple).

(a) An overall view of the binding interface, which is divided into three parts, as indicated, that are further illuminated in b and c. Dashed lines, rough propeller boundaries for β3, β4 and β5. (b) The interaction details in site I of a. (c) The interaction details in sites II and III of a. Dashed lines indicate H-bonds. (d) Mutagenesis analysis by flow cytometry. The nectin-4v tetramers of wild type and indicated mutants are prepared and tested for the binding to MV-H. The shaded gray histograms represent the negative control with PE-conjugated streptavidin, and the unfilled green histograms represent the binding of the indicated proteins to 293T cells.

Figure 4 Comparison of MV-H binding to its three receptors. (a) Superimposition of the three complex structures thus far determined. MV-H protein is shown in surface; the receptors are presented as ribbons. Blades β4 and β5 are orange and yellow, respectively, nectin-4v is magenta, SLAM is cyan and CD46 is blue. (b) Comparison of individual receptor footprints in MV-H. (c) Integrated view of the footprints mapped onto the surface of a single MV-H protein. Shared residues and hydrophobic pocket are highlighted and labeled. (d) Close view of the hydrophobic pocket in c. Color selections are: MV-H-CD46 (yellow, H; light blue, CD46), MV-H-SLAM (orange, H; cyan, SLAM) and MV-H-nectin-4v (green, H; magenta, nectin-4v). (e) Comparison of interaction details within hydrophobic pocket between CD46 and nectin-4v. The pocket is in surface representation and its contacting entities are in cartoon mode. Residues are labeled. Top, nectin-4v; bottom, CD46.

(Supplementary Table 1). Accordingly, the three receptors buried similar surface areas in MV-H: ~1,050 Å² for nectin-4v, ~1,226 Å² for SLAM and ~1,117 Å² for CD46. Notably, the MV-H interfaces that engage its receptors overlapped around a small hydrophobic pocket centered in the β4-β5 groove (Fig. 4c). A series of residues located either near or inside this pocket, including Leu464, Leu482, Phe483, Leu500, Tyr524, Tyr543 and Ser548, sterically contacted all three receptors. Reciprocally, the three receptors either extended from or lay over the pocket, binding to variant steric positions of the H protein.

A hydrophobic pocket for all three receptors

Notably, the binding sites within MV-H for the three receptors shared a hydrophobic pocket in the β4-β5 groove¹⁹,²⁰. Therefore, we further explored the detailed interactions at this small site. Sterically, this pocket was located at the boundary between blades β4 and β5, with the β4-(S2-S3) loop lining one side of the pocket and the β5 strands S2 and S3 lining the other (Fig. 4d). We observed a conformational change in the β4-(S2-S3) loop upon nectin-4v binding (see results above and Fig. 2c). This change increased the cavity volume, allowing for the accommodation of the interacting receptor residues. Therefore, the pocket underwent a switch from a ‘closed’ to an ‘open’ state after binding of the H protein to nectin-4v. We also observed similar open pocket conformations in the structures of MV-H bound to CD46 and SLAM (Fig. 4d). Nevertheless, the interaction modes for the three receptors in this pocket were not the same. Nectin-4v and CD46 similarly inserted into the pocket using a surface-exposed loop (the F-G loop for nectin-4v and D′-D loop for CD46), whereas SLAM only lay over the site using its C′ strand to cover the pocket and with a valine residue facing the pocket (Fig. 4d).

Overall, both nectin-4v and CD46 used a diresidue motif (Phe-Pro in nectin-4v and Pro-Pro in CD46) to inhabit the pocket. Nectin-4v inserted deeper into the pocket than CD46, leading to more extensive interactions at the site. As for nectin-4v, Phe101 and Pro102 interacted with a series of highly hydrophobic pocket residues, including Pro458, Met459, Leu462, Ala463, Leu464, Leu526, Tyr541 and Tyr543, and the apolar carbons of Thr528, representing an extremely strong stacking force (Fig. 4e, top). The consecutive prolines (Pro38 and Pro39) in

Comparison of MV-H binding to its three receptors

Considering the two reported structures of complexes (PDB 3INB, MV-H-CD46; PDB 3ALW, MV-H-SLAM)¹⁹,²⁰ and the one solved in this study, we compared the binding modes of MV-H with its three receptors. The receptor-binding footprints on the MV-H protein were defined by residues within 4.5 Å of the interacting receptor molecules in each of the three structures of complexes.

Overall, MV-H used the same β4-β5 groove to recognize all three receptors (Fig. 4a). SLAM has an immunoglobulin-like fold in its membrane-distal ectodomain that is responsible for MV-H binding. In the structure of the complex, SLAM was predominantly located on the concave surface of the MV-H β5 blade. Of the 27 residues in the receptor-binding interface, 18 were located in the β5 blade and the flanking loops. The remaining contacting residues resided in the β4 and β6 blades (Fig. 4b, top, Supplementary Table 1). Unlike SLAM, nectin-4v inserted residues into the β4-β5 groove of MV-H (Fig. 4a). Other than four residues in β3, the majority of interface residues in MV-H that contact nectin-4v were almost equally distributed in the β4 and β5 blades (9 in blade β4, 11 in β5 and 5 in the β4-β5 loop; Fig. 4b, middle, Supplementary Table 1). For CD46, its two N-terminal short consensus repeats arranged into a rod-like structure, interacting more with MV-H blade β4 than with β5 (Fig. 4a). The binding interface for this receptor involved 26 residues, of which 15 were localized in β4 and 7 in β5 (Fig. 4b, bottom, Supplementary Table 1).

Although all the footprints in MV-H for the three receptors were localized mainly in the β4-β5 groove, they were still largely variant, as more than half of the defined interface residues were exclusive to individual receptors (Fig. 4c). Nevertheless, MV-H used an equivalent number of residues to contact the respective receptors
CD46, which were relatively toward the outside of the pocket, contacted only Lys460, Asn461, Leu464, Leu500, Tyr541 and Tyr543 of the H protein (Fig. 4e, bottom). Hence, among the three receptors, nectin-4 interacted most extensively at this site.

# DISCUSSION

In this study, we solved the structure of MV-H in complex with its newly identified epithelial receptor nectin-4. The structure shows that nectin-4 generally binds to the same concave β4-β5 groove as the other two reported MV-H receptors, SLAM and CD46 (refs. 19,20). Nevertheless, the footprints in MV-H for these three receptors show differences. The SLAM footprint in MV-H extends mostly toward the top of the β-propeller domain and the β5 blade, whereas the interface residues for CD46 extend mainly toward the bottom of the β-propeller groove and the β4 blade. Nectin-4, in contrast, engages extensively both the MV-H β4 and β5 blades (see results above, Fig. 4a–c and Supplementary Table 1). Despite these differences, the defined interfaces overlap, and several MV-H residues are used by all three receptors. Thus, it is impossible for MV-H to simultaneously bind these receptors.

Notably, both nectin-4 and CD46 use a diresidue motif to penetrate into a hydrophobic pocket at the center of the β4-β5 groove. SLAM, in contrast, only lies over the pocket. In this sense, MV-H recognizes nectin-4 in a more similar manner to CD46 than to SLAM. We also noticed that nectin-4 inserts deeper into the pocket than CD46, leading to more extensive hydrophobic interactions at the site. This suggests that MV-H favors binding nectin-4 over binding CD46 or SLAM. Despite having the least buried surface area (as discussed in Results), nectin-4 has the strongest binding affinity for MV-H among the three receptors<sup>10</sup>. Furthermore, these observations also suggest that antiviral drug design could target this pocket. Small molecules that specifically bind to the pocket would preclude MV-H attachment to nectin-4 and CD46 because both receptors inhabit the pocket during MV-H binding. For SLAM, which lies over the pocket, a drug molecule of sufficient size could also inhibit the MV-H-SLAM interactions. Despite the availability of live vaccines for measles virus, no effective antiviral drugs are clinically available for the treatment of measles virus patients with complications.

It was accepted that a third receptor for measles virus is present specifically in epithelial cells long before nectin-4 was identified as a receptor<sup>21,22</sup>. Several mutagenesis studies have successfully identified a series of key residues in MV-H that are essential for effective epithelial invasion, including Leu464, Leu482, Phe483, Tyr524, Tyr541 and Tyr543 (refs. 21,22). In the MV-H–nectin-4v structure, all these residues extensively interact with nectin-4v. These results, together with the mutational analysis of nectin-4v in this study, provide evidence validating our structure of the complex reported here. Through elucidation of those previously unidentified hydrophobic residues in site I and the majority of residues involved in sites II and III that provide potential hydrophilic interactions, our structure clearly defines the MV-H binding site for its epithelial receptor. Before this study, no residues in nectin-4 essential for MV-H interaction had been characterized. Therefore, our work represents the first detailed view, to our knowledge, of interactions between MV-H and nectin-4 and extends the understanding of measles virus infection.

The nectin and Necl proteins exhibit an overall rod-like fold with one IgV domain followed by two IgC domains. The membrane-distal IgV domains of these molecules are also targeted as cell entry receptors by viruses such as herpes simplex virus type 1 (HSV-1)<sup>32</sup>, measles virus<sup>10</sup> and poliovirus<sup>33</sup>. Notably, nectin and Necl proteins share a similar fold for the terminal IgV domain, whereas the interacting viral ligands are very different in both sequence and structure.

In an earlier study on the interaction between HSV-1 gD and nectin-1, the structure of the complex shows that the virus-engaging residues are predominantly localized on the IgV ‘rear’ (CC′C″FG) sheet<sup>34</sup>. Notably, MV-H recognizes nectin-4 by engaging the rear sheet loops. A majority of the gD-binding residues in nectin-1 are equivalent in position to those of MV-H interaction in nectin-4, and many of these virus-engaging residues are also involved in the dimerization of these two molecules (Supplementary Fig. 3). Therefore, different viruses probably convergently evolve to mimic the nectin and Necl dimerization interface in their viral surface proteins for host cell attachment and entry. This raises the possibility of a common binding mechanism for the nectin- and Necl-recognition viruses. Conversely, nectin-1 and nectin-4 have different sequences of residues contacting the viruses, indicating that the characteristic dimer-interface residues of the nectin and Necl proteins determine their specificities for different viruses. These issues should be pursued in the future.

## METHODS

Methods and any associated references are available in the [online version of the paper](https://www.nature.com/articles/nsmb.2432).

### Accession codes

Protein Data Bank: coordinates have been deposited with accession code **4GJT**.

### Note

Supplementary information is available in the [online version of the paper](https://www.nature.com/articles/nsmb.2432).

## ACKNOWLEDGMENTS

This work was supported by the 973 Project of the China Ministry of Science and Technology (MOST, grants 2011CB504703 and 2010CB911902, G.F.G.). We acknowledge assistance by the staff at the Shanghai Synchrotron Radiation Facility of China. We thank Y. Zhang, X. Yu and T. Zhao for their expert technical assistance. G.F.G. is a leading principal investigator of the Innovative Research Group of the National Natural Science Foundation of China (grant 81021003).

## AUTHOR CONTRIBUTIONS

G.F.G. conceived, designed and supervised the project. X.Z., G.L., X.X., Y.L., Y.H., J.S. and C.W.-H.Z. did the experiments. J.Q. collected the data and solved the structure. G.L., J.Y. and G.F.G. analyzed the data and wrote the paper.

## COMPETING FINANCIAL INTERESTS

The authors declare no competing financial interests.

Published online at [http://www.nature.com/doifinder/10.1038/nsmb.2432](http://www.nature.com/doifinder/10.1038/nsmb.2432).

Reprints and permissions information is available online at [http://www.nature.com/reprints/index.html](http://www.nature.com/reprints/index.html).

---

1. Mankertz, A. *et al.* Molecular genotyping and epidemiology of measles virus transmission in the World Health Organization European Region, 2007–2009. *J. Infect. Dis.* **204** (suppl. 1), S335–S342 (2011).
2. Rota, P.A., Featherstone, D.A. & Bellini, W.J. Molecular epidemiology of measles virus. *Curr. Top. Microbiol. Immunol.* **330**, 129–150 (2009).
3. Anonymous. Increased transmission and outbreaks of measles, European Region, 2011. *Wkly. Epidemiol. Rec.* **86**, 559–564 (2011).
4. Griffin, D.E. Measles virus. in *Fields Virology* (Fields, B.N. *et al.*) 1551–1585 (Lippincott, Williams & Wilkins, Philadelphia, 2007).
5. Reuter, D. & Schneider-Schaulies, J. Measles virus infection of the CNS: human disease, animal models, and approaches to therapy. *Med. Microbiol. Immunol.* (Berl.) **199**, 261–271 (2010).
6. Noce, R.S. *et al.* Tumor cell marker PVRL4 (nectin 4) is an epithelial cell receptor for measles virus. *PLoS Pathog.* **7**, e1002240 (2011).
7. Dörig, R.E., Marcil, A., Chopra, A. & Richardson, C.D. The human CD46 molecule is a receptor for measles virus (Edmonston strain). *Cell* **75**, 295–305 (1993).
8. Naniche, D. *et al.* Human membrane cofactor protein (CD46) acts as a cellular receptor for measles virus. *J. Virol.* **67**, 6025–6032 (1993).
9. Tatsuo, H., Ono, N., Tanaka, K. & Yanagi, Y. SLAM (CDw150) is a cellular receptor for measles virus. *Nature* **406**, 893–897 (2000).
10. Mühlebach, M.D. *et al.* Adherens junction protein nectin-4 is the epithelial receptor for measles virus. *Nature* **480**, 530–533 (2011).
11. Yanagi, Y., Takeda, M., Ohno, S. & Hashiguchi, T. Measles virus receptors. *Curr. Top. Microbiol. Immunol.* **329**, 13–30 (2009).
12. Dhiman, N., Jacobson, R.M. & Poland, G.A. Measles virus receptors: SLAM and CD46. *Rev. Med. Virol.* **14**, 217–229 (2004).

---

NATURE STRUCTURAL & MOLECULAR BIOLOGY VOLUME 20 NUMBER 1 JANUARY 2013 71
13. Lizzewski, M.K., Post, T.W. & Atkinson, J.P. Membrane cofactor protein (MCP or CD46): newest member of the regulators of complement activation gene cluster. *Annu. Rev. Immunol.* **9**, 431–455 (1991).

14. Schwartzberg, P.L., Mueller, K.L., Qi, H. & Cannons, J.L. SLAM receptors and SAP influence lymphocyte interactions, development and function. *Nat. Rev. Immunol.* **9**, 39–46 (2009).

15. Reymond, N. *et al.* Nectin4/PRR4, a new afadin-associated member of the nectin family that trans-interacts with nectin1/PRR1 through V domain interaction. *J. Biol. Chem.* **276**, 43205–43215 (2001).

16. Brancati, F. *et al.* Mutations in PVRL4, encoding cell adhesion molecule nectin-4, cause ectodermal dysplasia-syndactyly syndrome. *Am. J. Hum. Genet.* **87**, 265–273 (2010).

17. Colf, L.A., Juo, Z.S. & Garcia, K.C. Structure of the measles virus hemagglutinin. *Nat. Struct. Mol. Biol.* **14**, 1227–1228 (2007).

18. Hashiguchi, T. *et al.* Crystal structure of measles virus hemagglutinin provides insight into effective vaccines. *Proc. Natl. Acad. Sci. USA* **104**, 19535–19540 (2007).

19. Santiago, C., Celma, M.L., Stehle, T. & Casasnovas, J.M. Structure of the measles virus hemagglutinin bound to the CD46 receptor. *Nat. Struct. Mol. Biol.* **17**, 124–129 (2010).

20. Hashiguchi, T. *et al.* Structure of the measles virus hemagglutinin bound to its cellular receptor SLAM. *Nat. Struct. Mol. Biol.* **18**, 135–141 (2011).

21. Tahara, M. *et al.* Measles virus infects both polarized epithelial and immune cells by using distinctive receptor-binding sites on its hemagglutinin. *J. Virol.* **82**, 4630–4637 (2008).

22. Leonard, V.H. *et al.* Measles virus blind to its epithelial cell receptor remains virulent in rhesus monkeys but cannot cross the airway epithelium and is not shed. *J. Clin. Invest.* **118**, 2448–2458 (2008).

23. Massé, N. *et al.* Measles virus (MV) hemagglutinin: evidence that attachment sites for MV receptors SLAM and CD46 overlap on the globular head. *J. Virol.* **78**, 9051–9063 (2004).

24. Vongpunsawad, S., Oezgun, N., Braun, W. & Cattaneo, R. Selectively receptor-blind measles viruses: identification of residues necessary for SLAM- or CD46-induced fusion and their localization on a new hemagglutinin structural model. *J. Virol.* **78**, 302–313 (2004).

25. Takai, Y., Miyoshi, J., Ikeda, W. & Ogita, H. Nectins and nectin-like molecules: roles in contact inhibition of cell movement and proliferation. *Nat. Rev. Mol. Cell Biol.* **9**, 603–615 (2008).

26. Takai, Y. & Nakanishi, H. Nectin and afadin: novel organizers of intercellular junctions. *J. Cell Sci.* **116**, 17–27 (2003).

27. Ogita, H. & Takai, Y. Nectins and nectin-like molecules: roles in cell adhesion, polarization, movement, and proliferation. *IUBMB Life* **58**, 334–343 (2006).

28. Takai, Y., Ikeda, W., Ogita, H. & Rikitake, Y. The immunoglobulin-like cell adhesion molecule nectin and its associated protein afadin. *Annu. Rev. Cell Dev. Biol.* **24**, 309–342 (2008).

29. Hu, A., Cattaneo, R., Schwartz, S. & Norrbý, E. Role of N-linked oligosaccharide chains in the processing and antigenicity of measles virus hemagglutinin protein. *J. Gen. Virol.* **75**, 1043–1052 (1994).

30. Chothia, C. & Jones, E.Y. The molecular structure of cell adhesion molecules. *Annu. Rev. Biochem.* **66**, 823–862 (1997).

31. Bork, P., Holm, L. & Sander, C. The immunoglobulin fold: Structural classification, sequence patterns and common core. *J. Mol. Biol.* **242**, 309–320 (1994).

32. Spear, P.G., Eisenberg, R.J. & Cohen, G.H. Three classes of cell surface receptors for alphaherpesvirus entry. *Virology* **275**, 1–8 (2000).

33. Mendelsohn, C.L., Wimmer, E. & Racaniello, V.R. Cellular receptor for poliovirus: molecular cloning, nucleotide sequence, and expression of a new member of the immunoglobulin superfamily. *Cell* **56**, 855–865 (1989).

34. Zhang, N. *et al.* Binding of herpes simplex virus glycoprotein D to nectin-1 exploits host cell adhesion. *Nat. Commun.* **2**, 577 (2011).

ONLINE METHODS

Protein expression and purification. Expression and purification of the nectin-4v protein followed a published procedure$^{35}$. In brief, the coding sequence for nectin-4 residues 32–145 (the first IgV domain) was subcloned into the NdeI and XhoI sites of pET-21a and was subsequently transformed into *Escherichia coli* strain BL21 (DE3) for expression. The protein was then purified to homogeneity via nickel-chelate affinity chromatography and gel filtration.

The H head of the wild-type measles virus (IC-B strain) was expressed using the Bac-to-Bac baculovirus expression system (Invitrogen). The cDNA fragment encoding residues 156–617 was cloned into pFastBac1 vector with an N-terminal His$_{6}$ tag. The baculovirus gp67 signal sequence was used to facilitate secretion of the protein, as this works with other viral glycoproteins$^{36}$. The recombinant baculovirus was first amplified in Sf21 cells and then used to infect HighFive cells to produce soluble MV-H protein. For purification, we followed the canonical procedure for His-tagged proteins$^{37}$ by first removing the majority of impurities using Ni-NTA resins (GE Healthcare) and then polishing purification using a Superdex 200 column (GE Healthcare).

Tetramer preparation and cell staining. Tetramers used in this study were purchased from QuantoBio using their customized service for detection of MV-H surface expression. A His$_{6}$ tag followed by a four-residue linker (sequence GSGG) and then a biotin tag (sequence GLNDIFEAQKIEWHE) was engineered to the C terminus of the nectin-4v protein. Wild-type and three mutants (D4K E7A, F101S and a tetramutant in which 101-FPAG-104 is replaced with 101-SSSY-104) of nectin-4v were expressed in *E. coli* strain BL21 (DE3) and purified as described above for the nonengineered nectin-4v protein. The purified proteins were then biotinylated as described$^{38}$. The excessive free biotin was removed by gel filtration. After the streptavidin shift assays, proteins with high biotinylation efficiency were incubated with phycoerythrin-conjugated streptavidin (Sigma).

The nectin-4$^{-}$ HEK-293T cells$^{6}$ were used to express the MV-H protein. The full-length MV-H coding sequence was first cloned into the pLEGFP-C1 vector via the XhoI and BamHI restriction sites. This yielded a plasmid encoding a recombinant protein with an EGFP tag fused to the N terminus of MV-H. The plasmid was transfected into 293T cells, and 24 h after transfection the cells were visualized under fluorescent microscope for expression of the recombinant proteins on the cell surface. The majority of the green fluorescence signals were observed on the cell surface, demonstrating the proper membrane localization of MV-H. The cells were then harvested and stained with nectin-4v tetramers (0.05 mg ml$^{-1}$) and analyzed by flow cytometry. Surface binding of wild-type nectin-4v to 293T cells was compared with the individual mutant tetramers.

Crystallization and data collection. To obtain crystals of the complex, purified MV-H and nectin-4v proteins were mixed in a 1:1 molar ratio and incubated at 4 °C overnight. The resulting preparation of complex at a concentration of 10 mg ml$^{-1}$ was used to screen commercially available crystallization kits (Hampton Research) for crystal growth conditions by sitting-drop vapor diffusion.

The conditions were then optimized and high-quality crystals of the complex were finally obtained in 0.1 M magnesium formate dihydrate and 15% (w/v) polyethylene glycol 3350 at 4 °C.

For data collection, crystals were cryoprotected in mother liquor containing 16% (v/v) glycerol and flash cooled to 100 K. Diffraction data were collected at Shanghai Synchrotron Radiation Facility beamline BL17U (wavelength, 1.00000 Å) and indexed, integrated and scaled using HKL2000 (ref. 39). Data collection and processing statistics are in Table 1.

Structure determination. The structure of nectin-4v bound to MV-H was solved by molecular replacement. Using the reported free MV-H structure (PDB [2RKC](http://www.pdb.org/pdb/explore/explore.do?structureId=2RKC)) and the coordinates of nectin-1 residues 35–144 (PDB [3U83](http://www.pdb.org/pdb/explore/explore.do?structureId=3U83)) as input models, the structure of the complex was successfully solved. The initial model was obtained by PHASER$^{40}$ from the CCP4 program suite$^{41}$. Restrained rigid-body refinement and manual model building were then done with REFMAC5$^{42}$ and COOT$^{43}$, respectively. Further rounds of refinement were done with Phenix.refine$^{44}$ using rigid-body refinement, isotropic ADP refinement and TLS refinement. During model building and refinement, the stereochemistry of the structure was monitored by PROCHECK$^{45}$. The Ramachandran plot distribution for residues in the structure of the complex was 75.8, 20.1 and 4.1% for most favored, additionally and generously allowed regions, respectively. Statistics are in Table 1. All figures were generated using PyMOL ([http://www.pymol.org/](http://www.pymol.org/)).

35. Xu, X., Zhang, X., Lu, G. & Cai, Y. Purification, crystallization and preliminary X-ray analysis of the first IgV domain of human nectin-4. *Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.* **F68**, 942–945 (2012).

36. Zhang, W. *et al.* Crystal structure of the swine-origin A (H1N1)-2009 influenza A virus hemagglutinin (HA) reveals similar antigenicity to that of the 1918 pandemic virus. *Protein Cell* **1**, 459–467 (2010).

37. Lu, G. *et al.* hNUDT16: a universal decapping enzyme for small nucleolar RNA and cytoplasmic mRNA. *Protein Cell* **2**, 64–73 (2011).

38. Liu, J. *et al.* Crystal structure of cell adhesion molecule nectin-2/CD112 and its binding to immune receptor DNAM-1/CD226. *J. Immunol.* **188**, 5511–5520 (2012).

39. Otwinowski, Z. & Minor, W. Processing of X-ray diffraction data collected in oscillation mode. *Methods Enzymol.* **276**, 307–326 (1997).

40. Read, R.J. Pushing the boundaries of molecular replacement with maximum likelihood. *Acta Crystallogr. D Biol. Crystallogr.* **57**, 1373–1382 (2001).

41. Collaborative Computing Project Number 4. The CCP4 suite: programs for protein crystallography. *Acta Crystallogr. D Biol. Crystallogr.* **50**, 760–763 (1994).

42. Murshudov, G.N., Vagin, A.A. & Dodson, E.J. Refinement of macromolecular structures by the maximum-likelihood method. *Acta Crystallogr. D Biol. Crystallogr.* **53**, 240–255 (1997).

43. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. *Acta Crystallogr. D Biol. Crystallogr.* **60**, 2126–2132 (2004).

44. Adams, P.D. *et al.* PHENIX: a comprehensive Python-based system for macromolecular structure solution. *Acta Crystallogr. D Biol. Crystallogr.* **66**, 213–221 (2010).

45. Laskowski, R.A., MacArthur, M.W., Moss, D.S. & Thornton, J.M. PROCHECK: a program to check the stereochemical quality of protein structures. *J. Appl. Crystallogr.* **26**, 283–291 (1993).
