
Chemistry & Biology

Article

Terpendole E, a Kinesin Eg5 Inhibitor, Is a Key Biosynthetic Intermediate of Indole-Diterpenes in the Producing Fungus Chaunopycnis alba

Takayuki Motoyama,$^{1}$ Toshiaki Hayashi,$^{1}$ Hiroshi Hirota,$^{1}$ Masashi Ueki,$^{1}$ and Hiroyuki Osada$^{1,*}$

$^{1}$Chemical Biology Core Facility, Chemical Biology Department, RIKEN Advanced Science Institute, Wako-shi, Saitama 351-0198, Japan

*Correspondence: hisyo@riken.jp

http://dx.doi.org/10.1016/j.chembiol.2012.10.010

## SUMMARY

Terpendole E is the first natural product inhibitor of kinesin Eg5. Because terpendole E production is unstable, we isolated and analyzed the terpendole E biosynthetic gene cluster, which consists of seven genes encoding three P450 monooxygenases (TerP, TerQ, and TerK), an FAD-dependent monooxygenase (TerM), a terpene cyclase (TerB), and two prenyltransferases (TerC and TerF). Gene knockout and feeding experiments revealed that terpendole E is a key intermediate in terpendole biosynthesis and is produced by the action of the key enzyme TerQ from paspaline, a common biosynthetic intermediate of indole-diterpenes. TerP converts terpendole E to a downstream intermediate specific to terpendole biosynthesis and converts paspaline to shunt metabolites. We successfully overproduced terpendole E by disrupting the terP gene. We propose that terpendole E is a key biosynthetic intermediate of terpenes and related indole-diterpenes.

## INTRODUCTION

The kinesin Eg5 is required for the separation of centrosomes during mitosis. Thus, Eg5 inhibitors can arrest the cell cycle of cancer cells at the M phase. Eg5 inhibitors are expected to be potent anticancer drugs with fewer side effects than conventional mitotic inhibitors that directly inhibit polymerization of microtubules (Knight and Parrish, 2008; Sarli and Giannis, 2008). Terpendole E (1) (Figure 1A) is the first described kinesin Eg5 inhibitor of natural product origin (Nakazawa et al., 2003), but it is produced transiently and is difficult to isolate. Therefore, elucidation of the biosynthetic mechanism is important to be able to produce adequate quantities of terpendole E (1) and to isolate new analogs with potent biological activity.

Terpendole E (1) is an indole-diterpene secondary metabolite produced by the Ascomycota fungus Chaunopycnis alba (Albophoma yamanashiensis). Some indole-diterpenes are tremorgenic mycotoxins that are produced to control insect and mammalian predators. Indole-diterpenes are produced from several Ascomycota filamentous fungi belonging to the evolutionarily separated Eurotiomycetes and Sordariomycetes groups (Saikia et al., 2008); C. alba belongs to the Sordariomy-

## RESULTS

### Isolation of the Terpendole E (1) Biosynthetic Gene Cluster

The homolog of the paxP gene was amplified by degenerate PCR from genomic DNA of the terpendole E (1) producer RK99-F33. The amplified 0.5 kb PCR product was cloned and sequenced. The deduced amino acid sequence of the partial putative terP product, TerP, showed high similarity to the P450

Chemistry & Biology

Terpendole Biosynthetic Gene Cluster

**Figure 1. Terpendole E (1) Is an Indole-Diterpene Compound with Similarity to Paspaline (6) and Lolicine A (5)**

(A) Terpendole E (1) and related indole-diterpene compounds.

(B) The paxilline (2) biosynthetic pathway.

chemical structures). These data indicate that terP is involved in the biosynthesis of terpendoles and that terpendole E (1) is either a biosynthetic intermediate or a shunt metabolite from another biosynthetic intermediate.

**Terpendole E (1) Is a Biosynthetic Intermediate in Indole-Diterpene Biosynthesis**

To determine whether terpendole E (1) is a biosynthetic intermediate in indole-diterpene biosynthesis and is converted to other biosynthetic intermediates, we constructed a C. alba strain (∆GC+P-48) in which all ter genes except terP were deleted (Figure 3). As controls, we constructed strains (∆GC-2 and ∆GC-3) in which all seven ter genes were deleted (Figure 3) and strains (∆GC+Q-3 and ∆GC+Q-5) in which all ter genes except terQ (another P450 gene) were deleted (Figure 3A). These strains were defective in terpendole production (data not shown), confirming that this gene cluster is essential for terpendole biosynthesis. These strains were next used for feeding experiments (Figure 4). The ∆GC+P-48 strain converted terpendole E (1) to another compound (Figure 4A). In contrast, the ∆GC-2 strain and the ∆GC+Q-3 strain did not metabolize terpendole E (1). The terpendole E (1) metabolite was isolated from the ∆GC+P-48 strain, and its structure was determined. The observed mass was 437.2570, and the chemical formula C₂₇H₃₅O₄N (437.2566) was predicted. NMR analysis (Figure S1; Table S2) indicated that this compound is a new, to our knowledge, terpendole I derivative, 13-desoxyterpendole I (Figure 6, compound 11). The ∆GC+Q-3 strain converted 13-desoxyterpendole I (11) to terpendole I (8) (Figure 4B). Terpendole I (8) is proposed to be a biosynthetic intermediate of terpendole C (10) (Young et al., 2009). Our data thus indicate that terpendole C (10) is biosynthesized via terpendole E (1).

**terP Knockout Strains Accumulate Terpendole E (1)**

In *P. paxilli*, PaxP is required for the metabolism of paspaline (6). Because paspaline (6) and terpendole E (1) have very similar structures (Figures 1A and 1B), we predicted that the PaxP homolog TerP would metabolize terpendole E (1) and that terpendole E (1) could be overproduced by knocking out terP. To test this, the terP gene was disrupted by Agrobacterium tumefaciens-mediated transformation. All the terP knockout strains analyzed (P-9, P-18, and P-27) accumulated terpendole E (1) (Figure 2B, data for P-27 are shown). The P-27 strain accumulated 36 mg/l terpendole E (1) after 2 days. In contrast, the wild-type strain accumulated terpendole E (1) only transiently on day 1 (2.5 mg/l) and predominantly accumulated terpendole I (8), terpendole K (9), and terpendole C (10) (see Figure 6 for

**13-Desoxypaxilline (7) Is Not a Biosynthetic Intermediate in Terpendole Biosynthesis**

13-Desoxypaxilline (7) is a biosynthetic intermediate of paxilline (2) and aflatrems (3) (Nicholson et al., 2009; Saikia et al., 2007) and is proposed to be a biosynthetic intermediate of lolitrem B (4) and terpendole C (10) (Li, 2010; Young et al., 2009). The ∆GC+P-48 strain was able to convert paspaline (6) to 13-desoxypaxilline (7), α-PC-M6 (12), and a putative 13-desoxypaxilline analog (13) with +16 mass of 13-desoxypaxilline (7) (Figure 4C; see Figure 6 for chemical structure of 12). The ∆GC+P-48 strain

1612 Chemistry & Biology 19, 1611–1619, December 21, 2012 ©2012 Elsevier Ltd All rights reserved

Chemistry & Biology

Terpendole Biosynthetic Gene Cluster

A
16 kb terpendole biosynthetic gene cluster

mermaid
graph LR
    orf1 --> terK --> terC --> terB --> terQ --> terM --> terP --> terF --> orf2 --> orf3
    terK[P450] --> terC[PT] --> terB[TC] --> terQ[P450] --> terM[OXY] --> terP[P450] --> terF[PT]


B
Wild type

| 1 d | 2 d | 3 d |
| --- | --- | --- |
| ![Wild Type 1d](wild_type_1d.png) | ![Wild Type 2d](wild_type_2d.png) | ![Wild Type 3d](wild_type_3d.png) |


P-27 (ΔterP)

| 1 d | 2 d | 3 d |
| --- | --- | --- |
| ![P-27 1d](p27_1d.png) | ![P-27 2d](p27_2d.png) | ![P-27 3d](p27_3d.png) |


Figure 2. Isolation of the Terpendole Biosynthetic Gene Cluster

(A) Terpendole biosynthetic gene cluster. Genes with similarity to known indole-diterpene biosynthetic genes are shaded.

(B) A terP knockout strain (P-27) accumulates terpendole E (1). The wild-type and P-27 strains were cultured in FDY for 1–3 days, and cultures were analyzed by HPLC. Terpendoles were detected by absorbance at 280 nm. For further details, see Tables S1 and S3.

also converted α-PC-M6 (12) to 13-desoxypaxilline (7) (Figure 4D), suggesting that TerP converts paspaline (6) to 13-desoxypaxilline (7) via α-PC-M6 (12). The ΔGC+Q-3 strain metabolized α-PC-M6 (12) to some unidentified compounds (Figure 4D) and also metabolized 13-desoxypaxilline (7) to paxilline (2) and to an unidentified compound (14) with +14 mass of 13-desoxypaxilline (7) (Figure 4E). However, the ΔGC+Q-3 strain did not convert α-PC-M6 (12) or 13-desoxypaxilline (7) to the pathway intermediates 13-desoxyterpendole I (11) or terpendole I (8) (Figures 4D and 4E). Our data therefore indicate that 13-desoxypaxilline (7) is a shunt metabolite in terpendole biosynthesis.

### Terpendole E (1) Is Biosynthesized from Paspaline (6) via C11 Hydroxylation by TerQ

Because PaxQ catalyzes C13 hydroxylation of paspaline, we hypothesized that the paspaline C11-hydroxylating enzyme is TerQ, a PaxQ homolog. If this hypothesis is true, we reasoned that a terQ knockout strain should accumulate paspaline (6). To test this, we constructed the terQ knockout strains Q-10 and Q-102 and found that neither strain could produce terpendole E (1) (Figure 5), indicating that TerQ is a C11-hydroxylating enzyme. The terQ knockout strains accumulated paspaline (6), but this gradually decreased, and 13-desoxypaxilline (7) accumulated as the major product, suggesting that paspaline (6) has been converted to 13-desoxypaxilline (7). α-PC-M6 (12) and a putative 13-desoxypaxilline analog (13) also accumulated in the terQ knockout strains. These products were predicted to be converted by TerP because PaxP is known to convert paspaline (6) to 13-desoxypaxilline (7) (Figure 1B). To prove this, we constructed terP/terQ double-knockout strains (PQ-2 and PQ-4) by disrupting the terQ gene in the terP disruptant P-27. As predicted, PQ-2 and PQ-4 accumulated paspaline (6) without producing 13-desoxypaxilline (7) (Figure 5), suggesting that TerP has dual activity and is able to convert terpendole E (1) to 13-desoxyterpendole I (11) and paspaline (6) to 13-desoxypaxilline (7). Finally, the ΔGC+Q-3 strain was fed with paspaline (6) and was shown to convert paspaline (6) to terpendole E (1) (Figure 4C), confirming that TerQ is a C11-hydroxylating enzyme.

#### TerQ Is a Key Enzyme in the Biosynthesis of Terpendole E (1) and Other Terpendoles

To compare the substrate specificities of TerQ and PaxQ, we constructed two PaxQ-expressing *C. alba* strains (ΔGC+paxQ-3 and ΔGC+paxQ-5) in which all ter genes except terQ were deleted and the terQ ORF was replaced with the paxQ ORF (Figure 3). The ΔGC+paxQ-3 strain could metabolize 13-desoxypaxilline (7) to paxilline (2) (Figure 4E) but could not metabolize paspaline (6) to terpendole E (1) (Figure 4C), suggesting that PaxQ and TerQ have different substrate specificities and that TerQ is a key enzyme in the biosynthesis of terpendole E (1) and its downstream metabolites in *C. alba*.

### DISCUSSION

We propose a biosynthetic pathway for terpendoles (Figure 6). The terpendole biosynthetic gene cluster is composed of seven

Chemistry & Biology

Terpendole Biosynthetic Gene Cluster

Figure 3. Manipulation of the Terpendole Biosynthetic Gene Cluster

Genes with similarity to known indole-diterpene biosynthetic genes are shaded. Marker genes are shown as filled pentagons. Arrowheads are annealing sites for the PCR primers used for analysis of the terpendole biosynthetic gene cluster.

genes (Table S1; Figure 2A) and is similar to other indole-diterpene biosynthetic gene clusters (Table S3), particularly for lolitrem B (4), suggesting that the biosynthetic mechanism for terpendoles and lolitrem B (4) are very similar. The terpendole E (1) cluster lacks a gene for GGPPS (geranylgeranyl diphosphate synthase), and we could not isolate homologs of other GGPPS genes (paxG, atmG, and ltmG) by using degenerate PCR primers (data not shown). We therefore speculate that the GGPPS gene for terpendole biosynthesis has low sequence similarity to other GGPPS genes involved in indole-diterpene biosynthesis. The terpendole biosynthetic gene cluster does not contain a transcription factor gene (Table S1). Similarly, no transcription factor gene has been found in gene clusters for paxilline (2) and lolitrem B (4) (Saikia et al., 2008). In this study, we used a fermentation medium FDY for terpendole production. Terpendole production started within 1 day, and high-level production was observed at 2 and 3 days of cultures (Figure 2). Similar high-level terpendole production was also observed in two additional fermentation media (YG and PD; data not shown). These data suggest that regulation of terpendole production is not strict.

Five of the seven genes in the terpendole biosynthetic cluster are common to indole-diterpene biosynthesis. TerC, TerM, and TerB are homologs of enzymes required for biosynthesis of paspaline (6), the first recognized intermediate in indole-diterpene biosynthesis. The mechanism of precursor cyclization to paspaline (6) has not yet been elucidated. However, Pyr4, a homolog of TerB, was recently shown to be a new terpene cyclase for meroterpene biosynthesis (Itoh et al., 2010), suggesting that TerB may also be a terpene cyclase for terpendole biosynthesis.

TerP and TerQ are homologs of P450s involved in the downstream metabolism of paspaline. TerQ is a key enzyme involved in terpendole E (1) biosynthesis. Homologs of TerQ create structural diversity in the indole-diterpenes; for example, PaxQ and AtmQ produce different products from the same substrate, 13-desoxypaxilline (7) (Nicholson et al., 2009). TerQ is uniquely able to hydroxylate paspaline (6) at C11, and the presence of 11-OH enables creation of a C11-C12 epoxy structure (Figure 6) by TerP. TerP converts terpendole E(1) to 13-desoxyterpendole I (11), which is predicted to be an important intermediate in the biosynthesis of terpendoles. We confirmed that 13-desoxyterpendole I (11) is produced from terpendole E (1) by TerP in feeding experiments with a TerP-expressing strain. The C11-C12 epoxy structure is also observed in lolitrem B (4) of N. lolii,

suggesting that the lolitrem B (4) biosynthetic enzyme LtmQ also has C11-hydroxylation activity. 13-Desoxypaxilline (7) is a biosynthetic intermediate in paxilline (2) and aflatrein (3) producers. However, in C. alba, this compound seems to be a shunt metabolite.

Two of the seven terpendole biosynthetic enzymes are specific to the terpendole producer (TerF and TerK) and the lolitrem B (4) producer (LtmF and LtmK). LtmF and LtmK are candidate enzymes for the conversion of terpendole I (terpendole J), and for the conversion of terpendole J to terpendole C (10), respectively (Young et al., 2009), suggesting that TerF and TerK are also required for conversion of terpendole I (8) to terpendole C (10).

Previous studies have indicated that paspaline (6) is a common key biosynthetic intermediate of indole-diterpenes (Saikia et al., 2008). Our data indicate that terpendole E (1), biosynthesized from paspaline (6), is a key intermediate of terpendole biosynthesis. We propose that there are at least two indole-diterpene biosynthetic pathways (Figure 7). One, functional in C. alba, converts paspaline (6) to 11-OH-containing compounds such as terpendole E (1), which is used as a biosynthetic intermediate. The alternative route, found in P. paxilli and A. flavus, instead converts paspaline (6) to the C11-C12 double bond-containing 13-desoxypaxilline (7), which serves as an important biosynthetic intermediate. Terpendole E (1) is converted to 13-desoxyterpendole I (11) containing a C11-C12 epoxy structure. Demethylation of C12 in the presence or absence of 11-OH produces a C11-C12 epoxy structure and a C11-C12 double bond, respectively. It appears that N. lolii also uses terpendole E (1) or a terpendole E-like compound as an important intermediate because this fungus produces a terpendole E-like compound, lolicine A (5), and lolitrem B (4) also has a C11-C12 epoxy structure. Lolitrem B (4) shows potent tremorgenic activity (Gallagher et al., 1981), and production of this mycotoxin could potentially be controlled by intervening at the C11-C12 epoxy formation step. Our data suggest that the biosynthetic pathways for indole-diterpenes may differ between Sordariomycetes (C. alba, N. lolii, and E. festucae) and Eurotiomycetes (P. paxilli and A. flavus). Interestingly, P. paxilli has an additional biosynthetic gene predicted to encode an unknown membrane protein (PaxA) that is proposed to function at postpaspaline steps (Table S3) (Saikia et al., 2008). A. flavus also has a PaxA homolog (AtmA) (Nicholson et al., 2009). Thus, PaxA homologs may have functions specific to paxilline and aflatrein biosynthesis.

In the wild-type C. alba strain, terpendole E (1) accumulated only transiently at early culture times (Figure 2B). Disruption of the terP gene caused overproduction of terpendole E (1) (Figure 2B). terP disruption blocks two points: conversion of

Chemistry & Biology

Terpendole Biosynthetic Gene Cluster

A
B
C
D
E

$\Delta$GC
0.6
0
25
30
35
0.15
0
20
30
0.5
0
25
35
0.2
0
20
30
0.5
0
25
30
35

$\Delta$GC+P
0.6
0
25
30
35
0.15
0
20
30
0.5
0
25
35
0.2
0
20
30
0.5
0
25
30
35

$\Delta$GC+Q
0.6
0
25
30
35
0.15
0
20
30
0.5
0
25
35
0.2
0
20
30
0.5
0
25
30
35

$\Delta$GC+paxQ
0.5
0
25
30
35
0.2
0
20
30
0.5
0
25
30
35

TerP
1 → 11
TerP
11 → ND
TerP
6 → 7, 12, 13
TerP
12 → 7
TerP
7 → ND
TerQ
1 → ND
TerQ
11 → 8
TerQ
6 → 1
TerQ
12 → 11, 8
TerQ
7 → 2, 14
PaxQ
6 → ND
PaxQ
7 → 2

time (min)

Figure 4. Analysis of the Terpendole Biosynthetic Pathway by Feeding Experiments

Each strain was fed with terpendole E (1) (A), 13-desoxyterpendole E (11) (B), paspaline (6) (C), $\alpha$-PC-M6 (12) (D), and 13-desoxypaxilline (7) (E). Terpendoles were detected by absorbance at 280 nm. ND, not detected. For further details, see Figure S1 and Table S2.

terpendole E (1) to 13-desoxyterpendole I (11) and conversion of paspaline (6) to 13-desoxypaxilline (7) (Figure 6). Inhibition of the latter step causes accumulation of paspaline (6) and its conversion to terpendole E (1) by the action of TerQ, making terP disruption a very efficient way to overproduce terpendole E (1). C. alba also produces terpendoles that have ACAT (acyl-CoA: cholesterol acyltransferase) inhibitory activity (Huang et al., 1995a, 1995b; Tomoda et al., 1995) or tremorgenic activity (Munday-Finch et al., 1997). Thus, pathway engineering of the terpendole biosynthetic gene cluster will enable us to isolate new terpendoles having highly potent biological activities.

SIGNIFICANCE

Terpendole E, an indole-diterpene compound, is the first natural product inhibitor of kinesin Eg5. However, terpendole E production in vivo is unstable, and thus, further analysis of its action is difficult. Here, we isolated and analyzed the terpendole E biosynthetic gene cluster, which revealed that terpendole E is a key early intermediate in terpendole biosynthesis, explaining its transient appearance. Terpendole E was successfully overproduced by gene knockout of the bispecific enzyme TerP. Importantly, terpendole E is not produced during biosynthesis of indole-diterpenes in distantly related fungi. We propose that indole-diterpene biosynthetic pathways can be classified into two groups.

The isolated terpendole E biosynthetic gene cluster will be an important tool in isolating new terpendole E analogs with more potent biological activities.

EXPERIMENTAL PROCEDURES

Strains, Media, Culture, and Transformation

The terpendole E (1) producer Chaunopycnis alba RK99-F33 (Nakazawa et al., 2003) was used as the parental strain. C. alba was grown on YGA plates (0.5% yeast extract, 2% glucose, and 2% Bacto agar; Difco, Detroit, MI) or PDA plates (3.9% potato dextrose agar; Difco), or in YG media (0.5% yeast extract and 2% glucose), PD media (2.4% potato dextrose; Difco), or FDY media (1.5% glucose, 1.5% soluble starch, 1% corn steep liquor, 1% dried yeast, 0.3% malt extract, 0.03% MgSO₄·7H₂O, 0.1% KH₂PO₄, and 0.1% agar [pH 6.0]) at 25°C or 28°C. C. alba was transformed by the Agrobacterium tumefaciens-mediated transformation method (ATMT) (de Groot et al., 1998; Rho et al., 2001) as described previously (Motoyama et al., 2008). A. tumefaciens strain C58 was cultured with a conidial suspension of C. alba. Transformants were selected by 500 μg/ml hygromycin B (Wako Pure Chemical, Osaka, Japan) or 4 mg/ml geneticin (Wako). In subsequent cultures of C. alba transformants, the appropriate selection reagents (200 μg/ml hygromycin B or 1 mg/ml geneticin) were added as needed. Plasmids were amplified in Escherichia coli DH5α (Sambrook et al., 1989). The paxilline (2) producer Penicillium paxilli PN2013 was purchased from ATCC. P. paxilli was grown on PDA or in PD. E. coli was grown in LB, and transformation was performed by the standard method (Sambrook et al., 1989).

DNA Manipulations

Total DNA was extracted from C. alba and P. paxilli using the DNeasy plant total DNA isolation kit (QIAGEN KK, Tokyo, Japan). PCR amplifications

Q-10 (ΔterQ)                                      PQ-2 (ΔterP ΔterQ)

|  |  |
| --- | --- |
| 0.85 | 0.85 |
| 1d | -0.05 |
|  |  |
| 0.4 | 0.4 |
|  |  |
| -0.05 | -0.05 |
| 22 24 26 28 30 32 34 36 | 22 24 26 28 30 32 34 36 |

|  |  |
| --- | --- |
| 0.85 | 0.85 |
| 2d | -0.05 |
|  |  |
| 0.4 | 0.4 |
|  |  |
| -0.05 | -0.05 |
| 22 24 26 28 30 32 34 36 | 22 24 26 28 30 32 34 36 |

|  |  |
| --- | --- |
| 0.85 | 0.85 |
| 3d | -0.05 |
|  |  |
| 0.4 | 0.4 |
|  |  |
| -0.05 | -0.05 |
| 22 24 26 28 30 32 34 36 | 22 24 26 28 30 32 34 36 |

time (min)                                    time (min)

Figure 5. TerQ Is Involved in Conversion of Paspaline (6) to Terpendole E (1)

A terQ knockout strain (Q-10) accumulates paspaline (6) and 13-desoxypaxilline (7). A terP/terQ double-knockout strain accumulates paspaline (6). Recombinant strains were cultured in FDY for 1–3 days and cultures were analyzed by HPLC. Terpendoles were detected by absorbance at 280 nm.

B-resistance gene expression unit was amplified from pCSN45 by PCR by using Mlul-HYG-F and Xbal-HYG-R primers. The 1.6 kb fragment was gel purified, digested with Mlul and Xbal, and gel purified (fragment 2). The downstream sequence of terQ was amplified from the genomic DNA of C. alba by PCR with Mlul-3′terQ-F and Ssel-3′terQ-R primers. The 1.0 kb fragment was gel purified, digested with Mlul and Ssel, and gel purified (fragment 3). The vector sequence of pBI121 between the right border and left border was amplified from pBI121 by PCR with Xhol-RB and Ssel-LB primers. The 8.7 kb fragment was gel purified, digested with Xhol and Ssel, and gel purified (fragment 4). The four fragments were ligated to yield a terQ disruption vector, pBI-terQ::HPH. The A. tumefaciens strain transformed with this plasmid was used for ATMT. terQ disruptants were selected by PCR with terQ-check-F and terQ-check-R primers, which hybridize just upstream and just downstream of terQ ORF, respectively. This primer set can amplify the 2.1 kb terQ ORF from wild-type strains, the 1.6 kb hygromycin B-resistance gene expression unit from terQ disruptants, and both fragments from ectopic transformants. The Q-10 strain was selected as a terQ gene-disrupted strain. To exclude the effect of ectopic integrations, Q-102, a second terQ knockout strain, was constructed similarly by using pBI-terQ::GEN (see next paragraph).

terP/terQ double-knockout strains were constructed as follows. The upstream sequence of terQ was amplified from the genomic DNA of C. alba by PCR using Xhol-5′terQ-F and Xbal-5′terQ-R primers. The 1.0 kb fragment was gel purified, digested with Xhol and Xbal, and gel purified (fragment 1). pUC-GEN, a plasmid containing a geneticin-resistance gene expression unit, was constructed as follows. The geneticin-resistance gene ORF was amplified from pBI121 by PCR with InFu-5NPTII and InFu-3NPTII primers (fragment A). The 3.4 kb fragment containing the vector, the trpC promoter, and the trpC terminator sequences was amplified from pUC-HYG ([Motoyama et al., 2005](https://doi.org/10.1016/j.jmb.2005.03.019)) by PCR by using InFu-PtrpC and InFu-TtrpC primers (fragment B). Fragments A and B were connected by the InFusion system (Takara Bio) to yield pUC-GEN. The geneticin-resistance gene expression unit was amplified from pUC-GEN by PCR with Mlul-HYG-F and Xbal-HYG-R primers. The 1.4 kb fragment was gel purified, digested with Mlul and Xbal, and gel purified (fragment 2). The downstream sequence of terQ was amplified from the genomic DNA of C. alba by PCR with Mlul-3′terQ-F and Ssel-3′terQ(3kb)-R primers. The 3.0 kb fragment was gel purified, digested with Mlul and Ssel, and gel purified (fragment 3). The vector sequence of pBI121 between the right border and left border was amplified from pBI121 by PCR with Xhol-RB and Ssel-LB primers. The 8.7 kb fragment was gel purified, digested with Xhol and Ssel, and gel purified (fragment 4). The four fragments were ligated to yield a terQ disruption vector, pBI-terQ::GEN. The A. tumefaciens strain transformed with this plasmid was used for ATMT. terQ disruptants of the P-27 terP disruptant were selected by PCR with terQ-check-F and terQ-check-R primers, which hybridize just upstream and just downstream of the terQ ORF, respectively. This primer set can amplify the 2.1 kb terQ ORF from wild-type strains, the 1.4 kb geneticin-resistance gene expression unit from terQ disruptants, and both fragments from ectopic transformants.

Gene cluster-deleted strains were constructed as follows. The 2.0 kb downstream sequence of terK was amplified from the genomic DNA of C. alba by

Construction of Recombinant Strains

The PCR primers used for plasmid construction are listed in Table S4.

terP disruptants were constructed as follows. The upstream sequence of terP was amplified from the genomic DNA of C. alba by PCR using Xhol-5′terP-F and Xbal-5′terP-R primers. The 1.0 kb fragment was gel purified, digested with Xhol and Xbal, and gel purified (fragment 1). The hygromycin B-resistance gene expression unit was amplified from pCSN45 ([Motoyama et al., 2005](https://doi.org/10.1016/j.jmb.2005.03.019)) by PCR by using Mlul-HYG-F and Xbal-HYG-R primers. The 1.6 kb fragment was gel purified, digested with Mlul and Xbal, and gel purified (fragment 2). The downstream sequence of terP was amplified from the genomic DNA of C. alba by PCR with Mlul-3′terP-F and Ssel-3′terP-R primers. The 1.0 kb fragment was gel purified, digested with Mlul and Ssel, and gel purified (fragment 3). The vector sequence of pBI121 between the right border and left border was amplified from pBI121 by PCR with Xhol-RB and Ssel-LB primers. The 8.7 kb fragment was gel purified, digested with Xhol and Ssel, and gel purified (fragment 4). The four fragments were ligated to yield a terP disruption vector, pBI-terP::HPH. The A. tumefaciens strain transformed with this plasmid was used for ATMT. terP disruptants were selected by PCR with terP-check-F and terP-check-R primers, which hybridize just upstream and just downstream of the deleted terP ORF, respectively. This primer set can amplify 1.2 kb terP ORF from wild-type strains, the 1.6 kb hygromycin B-resistance gene expression unit from terP disruptants, and both fragments from ectopic transformants. terP disruptants express a truncated TerP peptide (1–139). To exclude the effect of ectopic integrations, a number of knockout strains were isolated.

terQ disruptants were constructed as follows. The upstream sequence of terQ was amplified from the genomic DNA of C. alba by PCR using Xhol-5′terQ-F and Xbal-5′terQ-R primers. The 1.0 kb fragment was gel purified, digested with Xhol and Xbal, and gel purified (fragment 1). The hygromycin

Chemistry & Biology

Terpendole Biosynthetic Gene Cluster

indole-3-glycerol phosphate + GGPP

$\downarrow$  
TerC?  

$\downarrow$  
TerM?  

$\downarrow$  
TerB?  

paspaline (6)  
$\downarrow$  
TerQ  

terpendole E (1)  
$\downarrow$  
TerP  

13-desoxy-terpendole I (11)  
$\downarrow$  
TerQ  

terpendole I (8)  
$\downarrow$  
TerF?  

$\downarrow$  
TerK?  

terpendole C (10)  
$\downarrow$  
terpendole K (9)

$\alpha$-PC-M6 (12)  

TerP  

13-desoxy-paxilline (7)  

paxilline (2)  

TerQ  

TerC: PT  
TerM: OXY  
TerB: TC  
TerQ: P450  
TerP: P450  
TerF: PT  
TerK: P450  

A strain in which all of the ter genes except terQ were deleted was constructed as follows. The 4.0 kb terQ gene was amplified from the genomic DNA of C. alba by PCR using 5Q-1.5 kb and 3Q-0.5 kb-2 primers (fragment 1). The 1.6 kb hygromycin B-resistance gene expression unit was amplified from pCSN45 by PCR with InFu_3Q-5HPH and 3HPH primers (fragment 2). A 10.9 kb fragment containing the 6.9 kb vector sequence of pRI101, the 2.0 kb downstream sequence of terK, and the 2.0 kb downstream sequence of terF was amplified from pRI-GC::GEN by PCR with InFu_5Q-GC_LEFT-R and InFu-GC_RIGHT-F primers (fragment 3). The three fragments were connected by the InFusion system to yield a gene cluster-replacing vector, pRI-GC::terQ-HYG. The A. tumefaciens strain transformed with this plasmid was used for ATMT. Gene cluster-replaced strains were selected by PCR with GCOutside-check-F40 and GCOutside-check-R30-2 primers, which hybridize just outside of the terK-downstream sequence and just outside of the terF-downstream sequence. This primer set can amplify the 9.6 kb fragment containing the terK-downstream sequence, the terQ gene, the hygromycin B-resistance gene expression unit, the terF-downstream sequence from the gene cluster-replaced strains, and the 19.3 kb gene cluster from wild-type strains and ectopic transformants.

A strain in which all of the ter genes except terP were deleted was constructed as follows. The 3.8 kb terP gene was amplified from the genomic DNA of C. alba by PCR using 5P-1.5 kb and 3P-0.5 kb primers (fragment 1). A 12.5 kb fragment containing the 6.9 kb vector sequence of pRI101, the 2.0 kb downstream sequence of terK, the 1.6 kb hygromycin B-resistance gene expression unit, and the 2.0 kb downstream sequence of terF was amplified from pRI-GC::terQ-HYG by PCR with InFu_5P-GC_LEFT-R and InFu_3P-5HPH primers (fragment 2). The two fragments were connected by the InFusion system to yield a gene cluster-replacing vector, pRI-GC::terP-HYG. The A. tumefaciens strain transformed with this plasmid was used for ATMT. Gene cluster-replaced strains were selected by PCR with GCOutside-check-F40 and GCOutside-check-R30-2 primers, which hybridize just outside the terK-downstream sequence and just outside the terF-downstream sequence. This primer set can amplify the 9.4 kb fragment containing the terK-downstream sequence, the terP gene, the hygromycin B-resistance gene expression unit, the terF-downstream sequence from the gene cluster-replaced strains, and the 19.3 kb gene cluster from wild-type strains and ectopic transformants.

paxQ-introduced strains were constructed as follows. The 2.0 kb paxQ ORF was amplified from the genomic DNA of Penicillium paxilli PN2013 by PCR using InFu-PterQ_paxQ-F and InFu-TterQ_paxQ-R primers (fragment 1). From pRI-GC::terQ-HYG, a 14.5 kb fragment without the terQ ORF was amplified by PCR with InFu-PterQ-R and InFu-TterQ-F primers (fragment 2). The two fragments were connected by the InFusion system to yield a paxQ-introducing vector, pRI-GC::paxQ-HYG. The A. tumefaciens strain transformed with this plasmid was used for ATMT. The ΔGC-2 strain was used as the parental strain. paxQ-introduced strains were selected by PCR with GCOutside-check-F40 and GCOutside-check-R30-2 primers, which hybridize just outside the terK-downstream sequence and just outside the terF-downstream sequence. From the transformants in which paxQ is introduced by homologous recombinations, this primer set can amplify the 9.6 kb fragment containing the terK-downstream sequence, the paxQ gene expression unit, the hygromycin B-resistance gene expression unit, and the terF-downstream sequence. From wild-type strains and ectopic transformants, this primer set can amplify the 5.4 kb fragment containing the terK-downstream sequence, the geneticin-resistance gene expression unit, and the terF-downstream sequence.

**Preparation of Terpendoles**

Terpendole E (1) was isolated from the wild-type strain or the P-27 strain. Terpendoles I (8), C (10), and K (9) were isolated from the wild-type strain. 13-Desoxypaxilline (7) and α-PC-M6 (12) were isolated from the Q-10 strain. 13-Desoxyterpendole I (11) was isolated from the wild-type strain or the terpendole E-fed ΔGC+P-48 strain. Paxilline (2) was purchased from Sigma.

**Production and Purification of Terpendoles**

C. alba strains were cultured at 28°C for 96 hr in a 30 l jar fermenter containing 15 l of the producing medium FDY. Cultures were treated with 2 volumes

Figure 6. Terpendole Biosynthetic Pathway

Seven gene products are proposed to be involved in the biosynthesis. Paspaline (6) can be converted to a pathway intermediate terpendole E (1) or to a shunt metabolite α-PC-M6 (12).

PCR using InFu-GC_LEFT-F and InFu-GC_LEFT-R primers (fragment 1). The 1.4 kb geneticin-resistance gene expression unit was amplified from pUC-GEN by PCR with 5HPH and 3HPH primers (fragment 2). The 2.0 kb downstream sequence of terF was amplified from the genomic DNA of C. alba by PCR with InFu-GC_RIGHT-F and InFu-GC_RIGHT-R primers (fragment 3). The 6.9 kb vector sequence of pRI101 (Takara Bio) between the right border and left border was amplified from pRI101 by PCR with InFu-RB and InFu-LB primers (fragment 4). The four fragments were connected by the InFusion system to yield a gene cluster deletion vector, pRI-GC::GEN. The A. tumefaciens strain transformed with this plasmid was used for ATMT. Gene cluster-deleted strains were selected by PCR with GCOutside-check-F40 and GCOutside-check-R30-2 primers, which hybridize just outside of the terK-downstream sequence and just outside of the terF-downstream sequence. This primer set can amplify the 5.4 kb fragment containing the terK-downstream sequence, the geneticin-resistance gene expression unit, the terF-downstream sequence from the gene cluster-deleted strains, and the 19.3 kb gene cluster from wild-type strains and ectopic transformants.

indole-3-glycerol phosphate + GGPP

$\downarrow \downarrow$

![Chemical Structure](chemical_structure_1.png)

TerQ

paspaline (6)

PaxP

C11-C12 double bond

terpendole E (1)

TerP

$\downarrow$

13-desoxy-terpendole I (11)

C11-C12 epoxide

$\downarrow \downarrow$

terpendole C (10)

Chaunopycnis alba

lollitrem B (4)

Neotyphodium lolii

Epichloë festucae

PaxQ

13-desoxypaxilline (7)

paxilline (2)

Penicillium paxilli

aflatrein (3)

Aspergillus flavus

**Figure 7. Two Indole-Diterpene Biosynthetic Pathways**

Paspaline (6) is a common intermediate in both pathways. Terpendole E (1) is proposed to be a key biosynthetic intermediate of terpendole C (10) and lollitrem B (4). 13-Desoxypaxilline (7) is a key biosynthetic intermediate of paxilline (2) and aflatrein (3).

20% B from 50 to 60 min. LC/MS was carried out using an Applied Biosystems Q Trap LC/MS/MS system with a reverse phase column (XTerra MS C18, 2.1 × 150 mm, Waters, Tokyo, Japan) at a flow rate of 0.2 ml/min. The gradient system was MeCN (solvent B) in 0.05% acetic acid-H₂O (solvent A): 5% to 95% B from 0 to 30 min, 95% B from 30 to 50 min, and re-equilibration with 5% B from 50 to 60 min. Positive-ion electrospray ionization (ESI) was used for the detection of terpendoles. Conditions for MS were an ion spray voltage of 5.0 kV, a curtain gas pressure at 15 psi, a nebulizer gas pressure at 40 psi, an auxiliary gas pressure at 50 psi, and an ion source temperature of 400°C.

**Identification of Terpendoles**

The high-resolution mass spectra were recorded on a JEOL EI-MS (JMS-SX102A). The ¹H-, ¹³C-, and ²D NMR spectra were produced using a Bruker DRX-500 (500 MHz) spectrometer.

**Feeding Experiments**

C. alba strains were cultured for 24 hr at 28°C in cylindrical flasks containing 70 ml of the producing medium FDY. A sample of 0.5 ml culture was then mixed with 0.5 ml 0.1% agar solution and 1.67 μl 30 mM terpendoles (final concentration of 50 μM) and the mixture was incubated for 24 hr at 28°C. The culture was treated with 2 volumes of acetone and the supernatant was concentrated under N₂ flow to remove acetone. The resulting aqueous solution was extracted twice with ethyl acetate. The organic layer was dried under N₂ flow. The dry material was dissolved in 200 μl of MeCN and analyzed by LC/MS.

of acetone, and the supernatant was concentrated in vacuo to remove acetone. The resulting aqueous solution was extracted twice with ethyl acetate. The organic layer was dried in vacuo. The dried material was subjected to medium-pressure silica gel chromatography (CombiFlash, Teledyne Isco, Lincoln, NE) using an 80 g RediSep column. The column was equilibrated with chloroform and eluted with 1% methanol in chloroform. The eluted fraction was dried in vacuo. The dried material was subjected to preparative HPLC using a reverse phase column (PEGASIL ODS, 20 × 250 mm, Senshu Scientific, Tokyo, Japan) with a flow rate of 8 ml/min. The gradient system was MeCN (solvent B) in H₂O (solvent A): 20% to 80% B from 0 to 10 min, 80% to 100% B from 10 to 20 min, and 100% B from 20 to 60 min.

Alternatively, C. alba strains were cultured at 28°C in cylindrical flasks containing 70 ml of the producing medium. Terpendoles were purified similarly without the medium-pressure gel chromatography purification step.

**Analysis of Terpendoles**

Terpendoles were analyzed by HPLC or LC/MS. HPLC was performed using a reverse phase column (ODS-3, 4.6 × 250 mm, GL Sciences, Tokyo, Japan) with a flow rate of 1 ml/min. The gradient system was MeCN (solvent B) in H₂O (solvent A): 20% to 80% B from 0 to 10 min, 80% to 100% B from 10 to 20 min, 100% B from 20 to 50 min, and re-equilibration with

**ACCESSION NUMBERS**

Sequence data for the terpendole biosynthetic gene cluster is available from DDBJ/EMBL/GenBank under Accession number AB725916.

**SUPPLEMENTAL INFORMATION**

Supplemental Information includes one figure and four tables and can be found with this article online at [http://dx.doi.org/10.1016/j.chembiol.2012.10.010](http://dx.doi.org/10.1016/j.chembiol.2012.10.010).

**ACKNOWLEDGMENTS**

This study was supported in part by the Chemical Biology Project of RIKEN, Grants-in-Aid for Scientific Research (KAKENHI) and the Program for Promotion of Basic and Applied Researches for Innovations in Bio-oriented Industry.

Chemistry & Biology

Terpendole Biosynthetic Gene Cluster

Received: June 18, 2012  
Revised: September 26, 2012  
Accepted: October 2, 2012  
Published: December 20, 2012  

**REFERENCES**

de Groot, M.J., Bundock, P., Hooykaas, P.J., and Beijersbergen, A.G. (1998). *Agrobacterium tumefaciens*-mediated transformation of filamentous fungi. Nat. Biotechnol. 16, 839–842.

Gallagher, R.T., White, E.P., and Mortimer, P.H. (1981). Ryegrass staggers: isolation of potent neurotoxins lolitrem A and lolitrem B from staggers-producing pastures. N. Z. Vet. J. 29, 189–190.

Huang, X.H., Nishida, H., Tomoda, H., Tabata, N., Shiomi, K., Yang, D.J., Takayanagi, H., and Omura, S. (1995a). Terpendoles, novel ACAT inhibitors produced by *Albophoma yamanashiensis*. II. Structure elucidation of terpenes A, B, C and D. J. Antibiot. 48, 5–11.

Huang, X.H., Tomoda, H., Nishida, H., Masuma, R., and Omura, S. (1995b). Terpendoles, novel ACAT inhibitors produced by *Albophoma yamanashiensis*. I. Production, isolation and biological properties. J. Antibiot. 48, 1–4.

Itoh, T., Tokunaga, K., Matsuda, Y., Fujii, I., Abe, I., Ebizuka, Y., and Kushiro, T. (2010). Reconstitution of a fungal meroterpenoid biosynthesis reveals the involvement of a novel family of terpene cyclases. Nat. Chem. 2, 858–864.

Knight, S.D., and Parrish, C.A. (2008). Recent progress in the identification and clinical evaluation of inhibitors of the mitotic kinesin KSP. Curr. Top. Med. Chem. 8, 888–904.

Li, S.M. (2010). Prenylated indole derivatives from fungi: structure diversity, biological activities, biosynthesis and chemoenzymatic synthesis. Nat. Prod. Rep. 27, 57–78.

Motoyama, T., Kadokura, K., Ohira, T., Ichiishi, A., Fujimura, M., Yamaguchi, I., and Kudo, T. (2005). A two-component histidine kinase of the rice blast fungus is involved in osmotic stress response and fungicide action. Fungal Genet. Biol. 42, 200–212.

Motoyama, T., Ochiai, N., Morita, M., Iida, Y., Usami, R., and Kudo, T. (2008). Involvement of putative response regulator genes of the rice blast fungus *Magnaporthe oryzae* in osmotic stress response, fungicide action, and pathogenicity. Curr. Genet. 54, 185–195.

Munday-Finch, S.C., Wilkins, A.L., Miles, C.O., Tomoda, H., and Omura, S. (1997). Isolation and structure elucidation of lolilline, a possible biosynthetic precursor of the lolitrem family of tremorgenic mycotoxins. J. Agric. Food Chem. 45, 199–204. http://dx.doi.org/10.1021/jf960396r.

Nakazawa, J., Yajima, J., Usui, T., Ueki, M., Takatsuki, A., Imoto, M., Toyoshima, Y.Y., and Osada, H. (2003). A novel action of terpendole E on the motor activity of mitotic Kinesin Eg5. Chem. Biol. 10, 131–137.

Nicholson, M.J., Koulman, A., Monahan, B.J., Pritchard, B.L., Payne, G.A., and Scott, B. (2009). Identification of two aflatrem biosynthesis gene loci in *Aspergillus flavus* and metabolic engineering of *Penicillium paxilli* to elucidate their function. Appl. Environ. Microbiol. 75, 7469–7481.

Rho, H.S., Kang, S., and Lee, Y.H. (2001). *Agrobacterium tumefaciens*-mediated transformation of the plant pathogenic fungus, *Magnaporthe grisea*. Mol. Cells 12, 407–411.

Saikia, S., Parker, E.J., Koulman, A., and Scott, B. (2007). Defining paxilline biosynthesis in *Penicillium paxilli*: functional characterization of two cytochrome P450 monooxygenases. J. Biol. Chem. 282, 16829–16837.

Saikia, S., Nicholson, M.J., Young, C., Parker, E.J., and Scott, B. (2008). The genetic basis for indole-diterpene chemical diversity in filamentous fungi. Mycol. Res. 112, 184–199.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989). Molecular Cloning: A Laboratory Manual (Cold Spring Harbor, New York: Cold Spring Harbor Laboratory).

Sarli, V., and Giannis, A. (2008). Targeting the kinesin spindle protein: basic principles and clinical implications. Clin. Cancer Res. 14, 7583–7587.

Tomoda, H., Tabata, N., Yang, D.J., Takayanagi, H., and Omura, S. (1995). Terpendoles, novel ACAT inhibitors produced by *Albophoma yamanashiensis*. III. Production, isolation and structure elucidation of new components. J. Antibiot. 48, 793–804.

Young, C., McMullan, L., Telfer, E., and Scott, B. (2001). Molecular cloning and genetic analysis of an indole-diterpene gene cluster from *Penicillium paxilli*. Mol. Microbiol. 39, 754–764.

Young, C.A., Bryant, M.K., Christensen, M.J., Tapper, B.A., Bryan, G.T., and Scott, B. (2005). Molecular cloning and genetic analysis of a symbiosis-expressed gene cluster for lolitrem biosynthesis from a mutualistic endophyte of perennial ryegrass. Mol. Genet. Genomics 274, 13–29.

Young, C.A., Tapper, B.A., May, K., Moon, C.D., Schardl, C.L., and Scott, B. (2009). Indole-diterpene biosynthetic capability of epichloë endophytes as predicted by *ltm* gene analysis. Appl. Environ. Microbiol. 75, 2200–2211.

Zhang, S., Monahan, B.J., Tkacz, J.S., and Scott, B. (2004). Indole-diterpene gene cluster from *Aspergillus flavus*. Appl. Environ. Microbiol. 70, 6875–6883.
