
REVIEW
Open

Personalized Cancer Medicine: Molecular Diagnostics, Predictive Biomarkers, and Drug Resistance

D Gonzalez de Castro¹, PA Clarke², B Al-Lazikani² and P Workman²

The progressive elucidation of the molecular pathogenesis of cancer has fueled the rational development of targeted drugs for patient populations stratified by genetic characteristics. Here we discuss general challenges relating to molecular diagnostics and describe predictive biomarkers for personalized cancer medicine. We also highlight resistance mechanisms for epidermal growth factor receptor (EGFR) kinase inhibitors in lung cancer. We envisage a future requiring the use of longitudinal genome sequencing and other omics technologies alongside combinatorial treatment to overcome cellular and molecular heterogeneity and prevent resistance caused by clonal evolution.

Our molecular understanding of cancer causation and progression has been hugely enabled by genome sequencing and other large-scale omics approaches, leading to the discovery and development of molecularly targeted drugs and companion diagnostics for personalized, precision treatment.¹ Of course, the outcome of cancer treatment is not determined only by the variation in the genetic makeup of a tumor. Interpatient differences in pharmacokinetics and changes in drug levels during treatment (aspects that are outside the scope of this article) are also likely to contribute to therapy resistance. Therefore, personalized treatment requires not only the characterization of the tumor cells but also individualized drug administration, as set out in the Pharmacologic Audit Trail.²

Here we focus on the current status and issues facing molecular cancer diagnostics and especially discuss predictive biomarkers. In addition, we emphasize mechanisms of resistance to EGFR kinase inhibitors as a paradigm for the major challenge of drug resistance we now face in targeted therapy and personalized medicine. Finally, we anticipate a future in which longitudinal genome sequencing and other omics technologies will inform adaptive combinatorial treatment to tackle genetic and phenotypic heterogeneity and overcome drug resistance. We begin by giving an overview of some of the challenges in kinase inhibitor discovery and development.

THE EMERGENCE OF KINASE INHIBITORS FOR CANCER TREATMENT

Protein kinase inhibitors now play a leading role in the treatment of cancer, exemplifying small-molecule exploitation of oncogene addiction.³⁴ A total of 24 small-molecule kinase inhibitors have been approved for use as therapeutic agents, 17 of which are for cancer. In addition, four monoclonal antibodies acting on protein kinase targets have also been licensed for cancer therapy.

A recent report from the Pharmaceutical Research and Manufacturers of America suggests a very conservative approach to drug discovery. The report indicated that a significant proportion of industry activity in oncology is directed toward a relatively small number of targets, as shown by the fact that >20% of the projects involving the clinical development of cancer drugs focus on only eight common kinase targets. In order of popularity, these are VEGF/VEGFR, the lipid kinase PI3K, human epidermal growth factor receptor 2 (HER2), mTOR, EGFR, MET, PDGF/PDGFR, and KIT (http://www.phrma.org/sites/default/files/1000/phrmamedicinesindevelopmentcancer2012.pdf; http://www.forbes.com/sites/brucebooth/2012/06/07/cancer-drug-targets-the-march-of-the-lemmings/). In fact, with respect to preclinical development, the “congestion” of activity centering on these same targets is even greater.

On the other hand, our own mining of data from the ChEMBL⁵ (http://www.ebi.ac.uk/chembl/) and canSAR⁶ (https://cansar.icr.ac.uk) databases gives us an estimate of ~395 kinase inhibitors that are in clinical development, representing a high proportion (33%) of the total of ~1,200 cancer drugs currently in clinical development overall. Moreover, these 395 kinase inhibitors are believed to act on ~110 primary declared targets, with many of them modulating more

---

¹Molecular Diagnostics Department, The Institute of Cancer Research and the Royal Marsden NHS Foundation Trust, London, UK; ²Cancer Research UK Cancer Therapeutics Unit, The Institute of Cancer Research, London, UK. Correspondence: P Workman (paul.workman@icr.ac.uk); D Gonzalez de Castro (david.gonzalez-de-castro@icr.ac.uk)

Received 20 November 2012; accepted 30 November 2012; advance online publication 30 January 2013. doi: 10.1038/clpt.2012.237

252                                                                                                     VOLUME 93 NUMBER 3 | MARCH 2013 | www.nature.com/cpt

than one kinase.⁵,⁶ In addition, there is considerable further potential in this target class in the context of cancer. Our recent analysis⁷ has identified 42 actual or potential kinase targets with cancer-causing mutations or other genomic abnormalities from the total of 479 cancer-related genes listed in the Cancer Gene Census⁸ (http://www.sanger.ac.uk/genetics/CGP/Census). Also, only a small proportion of the 518 human protein kinases have been functionally annotated with “selective” small-molecule inhibitors.⁹

The surprising imbalance of drug discovery and development activity may well be attributable to limitations in the availability of knowledge and technical resources.¹⁰ For instance, limitations in the understanding of the underlying biological processes and the lack of suitable assays, chemical tool libraries, and informative biomarkers make the exploration of new targets both more difficult and more risky than pursuing those that are already well understood, validated, and shown to be successful. Less well studied kinases and other novel targets not only require enhanced investment but they also carry greater risk of failure; these are matters of major concern for the pharmaceutical industry from a commercial point of view. Such issues must be addressed through new paradigms such as nonprofit drug discovery and development programs and public–private partnership, which have the potential to increase creativity and innovation and reduce unnecessary duplication.⁹,¹⁰

MOLECULAR DIAGNOSTICS AND PREDICTIVE BIOMARKERS

From the initial pioneering experience with the HER2 antibody trastuzumab in breast cancer, to the BCR-ABL1 inhibitor imatinib in chronic myeloid leukemia and the EGFR kinase inhibitors gefitinib and erlotinib (Figure 1) in non-small-cell lung cancer (NSCLC), through to recent experience with the BRAF inhibitor vemurafenib in melanoma and the dual ALK-MET inhibitor crizotinib in NSCLC,¹,¹¹ it has been recognized that the successful development and use of kinase inhibitors for cancer therapy is very much dependent on predictive biomarkers for patient selection. In the era of personalized cancer medicine, companion diagnostics have jumped to the front line of targeted prescribing of therapeutics. In our multidisciplinary team meetings we are now commonly faced with clinical decisions about individual patients involving the molecular profiling of their tumor tissue.¹¹

Arguably, most of the recent attention concerning molecular cancer diagnostics in the clinical setting has been focused on predictive biomarkers of response to therapy, such as KRAS mutations in metastatic colorectal cancer (mCRC),¹²,¹³ EGFR mutations in advanced NSCLC,¹⁴–¹⁶ and BRAF mutations in metastatic malignant melanoma.¹⁷,¹⁸ The presence or absence of these predictive markers is directly linked to the response rates of particular targeted therapies with small-molecule kinase inhibitors or antibodies. Consequently, testing for them has become a critical step in the pathological diagnosis of the above-mentioned tumors.

Somewhat away from the spotlight, but still very important, there are many other clinical applications of molecular diagnostics in oncology (Table 1). For example, the molecular characterization of lymphomas and leukemias is now an integral part of the diagnosis, and several molecular abnormalities have been included in the latest World Health Organization classification of hematological malignancies.¹⁹ Similarly, molecular analysis of soft-tissue sarcomas is emerging as a critical tool for differential diagnosis. Such analysis includes SS18-SSX fusions in synovial

![Chemical structures of epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors and their molecular modes of binding to the target.](image.png)

**Figure 1** Chemical structures of epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors and their molecular modes of binding to the target.  
(a) Two-dimensional (2D) structure of reversible inhibitor gefitinib and the three-dimensional (3D) structure in complex with EGFR (PDB code 3UG2).  
(b) 2D structure of reversible inhibitor erlotinib and the 3D structure of the binding site of EGFR in complex with erlotinib (PDB code 4HJO).  
(c) 2D structure of the potent irreversible inhibitor afatinib (BIBW-2992) and the 3D structure of the binding site of EGFR in complex with afatinib, showing the covalent interaction with Cys797, highlighted in orange (PDB code 4G5J). PDB, Protein Data Bank.

REVIEW

Table 1 Examples of molecular biomarker investigations used in clinical practice to guide diagnosis and therapeutic decisions

| Diagnostic |  |  |
| --- | --- | --- |
| Acute leukemias | PML-RARA |  |
|  | BCR-ABL1 |  |
|  | CBFB-MYH11 |  |
|  | ETV6-RUNX1 | WHO 2008 classification of leukemias |
|  | RUNX1-RUNX1T1 |  |
|  | MLL-rearranged |  |
|  | TCF3-PBX1 |  |
|  | RBM15-MKL1 |  |
| MPD | JAK2 | Mutations confirm diagnosis of clonal MPD |
|  | SS18-SSX1/SSX2 | Synovial sarcoma |
|  | PAX3/PAX7-FOXO1A | Alveolar rhabdomyosarcoma |
|  | EWSR1-FLI1 | Ewing's sarcoma |
|  | EWSR1-ERG |  |
|  | EWSR1-NR4A3 | Extraskeletal myxoid chondrosarcoma |
|  | TAF15-NR4A3 |  |
| Sarcomas | EWSR1-ATF1 | Clear cell sarcoma (and angiomatoid fibrous histiocytoma) |
|  | EWSR1-CREB1 |  |
|  | ASPSCR1-TFE3 | Alveolar soft-part sarcoma (and renal cell carcinoma) |
|  | FUS-DDIT3 | Myxoid liposarcoma |
|  | FUS-CREB3L2 | Low-grade fibromyxoid sarcoma |
|  | JAZF1-SUZ12 | Endometrial stromal sarcoma |
|  | ETV6-NTRK3 | Congenital fibrosarcoma (and secretory breast carcinoma) |

| Predictive |  |  |
| --- | --- | --- |
| NSCLC | EGFR | Mutations predict response to TKI |
|  | ALK | Rearrangements predict response to ALK-inhibitors |
| GIST | KIT and PDGFRA | Mutations predict response to c-KIT/PDGFRα inhibitors |
| mCRC | KRAS | Mutations predict lack of response to anti-EGFR antibodies |
| Melanoma | BRAF | Mutations predict response to specific BRAF inhibitors |
| Breast cancer | HER2 | Amplifications predict response to anti-HER2 antibodies |

| Prognostic |  |  |
| --- | --- | --- |
| CLL | TP53 | Mutations are indicative of poor outcome |
|  | IGHV | Lack of mutations is indicative of poor outcome |
| AML | FLT3-ITD | Mutations are indicative of poor outcome |
| mCRC | BRAF | Mutations are indicative of poor outcome |
| Breast cancer | OncotypeDx | Risk stratification (21-gene expression signature) |
|  | Mammaprint | Risk stratification (70-gene expression signature) |
|  | IHC4 | Risk stratification (4-protein IHC expression) |

| Disease monitoring |  |  |
| --- | --- | --- |
| CML | BCR-ABL1 | Minimal residual disease detection |
| APML | PML-RARA | Minimal residual disease detection |
| ALL | IGHV-TCR rearrangements | Minimal residual disease detection |

ALL, acute lymphoblastic leukemia; AML, acute myeloid leukaemia; APML, acute promyelocytic leukemia; CLL, chronic lymphocytic leukemia; CML, chronic myeloid leukemia; EGFR, epidermal growth factor receptor; GIST, gastro-intestinal stromal tumors; HER2, human epidermal growth factor receptor 2; IHC, immunohistochemistry; mCRC, metastatic colorectal cancer; MPD, myeloproliferative diseases; NSCLC, non-small-cell lung cancer; TKI, tyrosine kinase inhibitor; WHO, World Health Organization.

sarcomas, EWSR1 fusions in Ewing's sarcoma, and PAX3/7-FKHR fusions in alveolar rhabdomyosarcomas.²⁰

Another important aspect of molecular diagnostics is the analysis of prognostic markers in certain malignancies such as chronic lymphocytic leukemia (including TP53 mutations,²¹,²² IGHV mutation,²³ and CLLU1 expression²⁴), and breast cancer (including recurrence risk stratification using the OncotypeDx and Mammaprint gene expression signatures, or the IHC4 immunohistochemistry method that measures the expression of the estrogen receptor, the progesterone receptor, human EGFR2/HER2, and Ki-67²⁵–²⁸). No less critical is the use of molecular monitoring of residual disease in chronic myeloid leukemia by determining BCR-ABL1 expression²⁹,³⁰ and in pediatric acute lymphoblastic leukemia (ALL) by assessing immunoglobulin and T-cell receptor gene rearrangements.³¹,³²

Given the increasingly critical role of molecular investigations in the clinical management of cancer patients, there is a clear need for developing robust, high-quality diagnostic tests and for their corresponding technical and clinical validation. Thorough technical validation is a prerequisite for establishing the performance characteristics of a methodology; these include sensitivity, specificity, and limits of detection and coverage as part of a standardized framework for the validation and verification of clinical molecular genetic tests.³³ Highly sensitive methods such as amplification-refractory mutation system, allele-specific real-time PCR, mass spectrometry, and high-resolution melting, among others, are now widely used to increase the detection rate of genetic abnormalities, thereby reducing the need for accurate tumor cell purification/selection and increasing the clinical value of the analysis.³⁴

However, increased sensitivity may lead to the detection of “subclinical” mutations, that is, those that are present in a small subclone of the tumor, potentially leading to a negative impact on response to certain therapies. For example, in a significant proportion of responding patients, concomitant, low-level EGFR p.T790M mutations associated with a shorter progression-free survival—so-called “gatekeeper mutations” that lead to reduced drug binding and resistance to tyrosine kinase inhibitors (TKIs) gefitinib and erlotinib in patients with NSCLC (see below)—are seen pretreatment alongside other EGFR mutations that cause sensitivity to these agents.³⁵ The presence of multiple mutations in a single tumor biopsy has become even more relevant with the advent of increasingly sensitive methods such as deep sequencing. This is especially so, given our growing knowledge of the oligoclonal heterogeneity and clonal selection in human cancers.³⁶,³⁷

In view of this heterogeneity, one may contest whether a single biopsy of the primary tumor can provide the required information to treat the metastatic disease years later and whether multiple biopsies may be needed from each patient to ascertain the true molecular status of the tumor at the particular time point of treatment with respect to actionable mutations. On the other hand, there is evidence that the key initial driver mutations are generally present in both the primary and the metastatic biopsy specimens,³⁸ and clinical trials have demonstrated that there is benefit in testing primary resected material to treat metastatic disease.¹⁷ Nevertheless, especially with the discovery of new, secondary mutations and other changes that may have therapeutic implications, we need to reassess our approach to molecular profiling by selecting the most appropriate sample to biopsy (such as cytological specimens from fine-needle aspirates or even circulating plasma DNA) so as to minimize invasive procedures.³⁹

It is clear, therefore, that there is a need to ensure robust clinical validation of companion diagnostics, preferably within the context of randomized clinical trials in which the detection of particular molecular biomarkers by means of a particular methodology can be clearly linked to patient outcome. In this regard, emerging data on subgroup analysis in retrospective cohorts have shown different mutations in particular genes may not all be identical with respect to response and outcome data. This is exemplified by recent data in colorectal cancer showing that KRAS p.G13D mutations seem to have less impact than KRAS codon 12 mutations in determining resistance to anti-EGFR antibody therapy.⁴⁰ Similarly, in gastrointestinal stromal tumors, mutations in exon 11 of the KIT gene are generally associated with good response and increased survival at a 400 mg dose of imatinib, whereas mutations in exon 9 of the same gene require either a higher dose of imatinib (600 mg or 800 mg) or a different TKI such as sunitinib to achieve a significant response.⁴¹,⁴² This troublesome clinical issue is not made any easier by the imprecise and biologically ambiguous labeling of targeted drugs, involving the use of generic terms such as “mutations” or “activating mutations” instead of a defined list of specific mutations that have been proven beyond significant uncertainty to be associated with increased or decreased response to particular therapies.

With the availability of increasing numbers of novel targeted drugs approved in the first-line setting as well as experimental drugs in phase I, II, and III clinical trials, there is now an emerging need to move away from single-biomarker analysis. It is necessary, instead, to perform “panel testing” for a variety of potentially actionable biomarkers on which selection of targeted therapies may be based.⁴³ Although the initial assumption may be that such a strategy would increase the cost of diagnostics, a reasonable argument may be made, from the health economics viewpoint, that this may not be the case in terms of cost-effectiveness overall.

First, it is important to keep in mind the requirement that patients with advanced metastatic disease must be treated as early as possible: assuming an average turnaround time of 7–10 working days for a diagnostic test, sequential analysis of two clinically actionable biomarkers in the same specimen (such as EGFR and ALK in NSCLC) can take up to one month from the time of referral instead of the one to two weeks required for concomitant or panel testing. These waiting times associated with sequential analysis will increase considerably with the implementation of further companion diagnostics for additional new drugs; in many cases this would lead to urgent treatment decisions needing to be made without a complete molecular picture being available. It can be argued that early targeted treatment should translate into better outcomes with increased survival

benefit, resulting in more cost-effective treatment. Second, real-time multiplexed analysis of several markers can increase the potential selection of patients for many different clinical trials, thereby speeding the recruitment rate and reducing the cost of early trials. Finally, many of the predictive molecular markers may also have potential prognostic impact; in the absence of comprehensive testing it may take many more years for their full clinical value to become clear, which is unacceptable.

It is important to consider the context of the tumor when selecting targeted therapies based on molecular genotyping. Different types of tumors may originate from different cellular types and are frequently driven by different combinations of genetic alterations. For example, BRAF inhibitors have shown unprecedented success in the treatment of malignant melanomas with BRAF-V600 mutations. BRAF-V600 mutations are also observed, with varying frequency, in other cancers. In some of these cancers, for example, in nonmelanoma malignancies such as hairy-cell leukemia,^44^ BRAF inhibitors can achieve similar degrees of response. In other cancers, such as colorectal cancer, more complex underlying mechanisms preclude therapeutic success, including a feedback loop that increases the expression of EGFR. This latter finding suggests that a combinatorial therapy against both BRAF and EGFR kinases should be adopted in such cases.^45^

**RESISTANCE TO TKIs IN NSCLC: A PARADIGM**

After the initial promise of targeted therapies, drug resistance is now emerging as the major obstacle to progress in the field, and this includes kinase inhibitors. Of note, molecular mechanisms of drug resistance are currently being defined, resulting in pharmacologically actionable opportunities to restore sensitivity.^46,47^ In many ways, the experience with TKIs in NSCLC exemplifies the successes and challenges of personalized cancer medicine. NSCLC is a major cause of death worldwide, estimated at 1.4 million fatalities per annum (http://www.who.int/cancer/en/; http://globocan.iarc.fr/factsheets/populations/factsheet.asp?uno=900). The finding that EGFR TKIs were effective and had particular activity in 15–30% of patients with EGFR mutations was a major advance; however, resistance sets in after approximately one year of treatment.

Resistance to TKIs in NSCLC provides a useful paradigm, and we as well as others have published commentaries on this aspect recently.^48,49^ Resistance mechanisms have been identified through numerous methods, including mechanistic investigation of laboratory models and profiling of patients’ tumors. The investigations reveal the existence of a range of devious molecular tricks to elude EGFR TKIs, including ingenious means of maintaining addiction to the canonical signaling pathways downstream of EGFR as well as other less obvious but no less effective tactics (Figure 2). Thus, we see the second-site exon 20 EGFR gatekeeper mutation p.T790M that reduces drug binding in ~50% of all cases of EGFR mutation; the p.T790M mutation plus EGFR amplification in ~8% of cases; TK switching or receptor dimerization through MET amplification in ~5-19% of cases; overexpression of AXL and its ligand GAS6 in 20 and 25% of cases, respectively; and activating mutation of the PI3K p110α-encoding gene PIK3CA in ~5% of cases. There is also evidence of nuclear factor-κB signaling being implicated as a resistance mechanism to avoid TKI-induced apoptosis, for example through the low expression of the nuclear factor-κB inhibitory protein IkB.^50^

Although resistance mechanisms are yet to be elucidated in a significant proportion of patients (~30%), it is now clear that tumor heterogeneity and drug resistance may be detected not only at the genomic level but also at the morphologic level. It is a fascinating finding that transformation of NSCLC into the small-cell lung cancer (SCLC) phenotype emerges in 5–15% of cases receiving treatment with TKIs.^51^ This may indicate that there is a minority SCLC subpopulation originally present that is selected for by therapy. Alternatively, the finding may be attributable to tumor plasticity via unknown mechanisms. A finding from a recent study illustrates another interesting form of phenotypic heterogeneity; NSCLC with squamous morphology and immunophenotype (by immunohistochemistry), and containing EGFR mutations, was reclassified after a secondary biopsy as “mixed adeno-squamous or adenocarcinomas with squamous morphology.”^52^ Furthermore, in ~20% of patients there is also intriguing evidence of the emergence of a drug-resistant and more malignant, invasive, and metastatic phenotype with stem cell-like traits. This is known as the epithelial–mesenchymal transition and may be partly linked to AXL activation.^51^ Epithelial–mesenchymal transition is regulated by a variety of factors, including receptor TKs, transforming growth factor-β, Notch, Snail family transcription factors, and tumor-stromal interactions. Suppression of cell adhesion protein E-cadherin and increased expression of vimentin and/or fibronectin are common end points and useful epithelial–mesenchymal transition markers.^53,54^

Of note, many of these mechanisms that cause resistance to TKIs in NSCLC suggest the use of various actionable therapeutic maneuvers depending on the molecular or phenotypic evidence. These therapeutic approaches include the use (alone or in combination with EGFR TKIs) of inhibitors of MET, AXL, PI3 kinase (or downstream targets thereof), and IKKB, and the introduction of SCLC chemotherapy or drugs acting on epithelial-mesenchymal transition cells^53^, together with inhibition of transforming growth factor-β.^55^

New-generation irreversible inhibitors such as afatinib show potent activity against mutant EGFR with the second-site p.T790M gatekeeper mutation that reduces drug binding, as well as against wild-type EGFR (Figure 1). Although activity has been observed in preclinical models, clinical activity in gefitinib- or erlotinib-resistant tumors is yet to be observed.^56^ Of note, studies in a human NSCLC xenograft model showed that sensitive wild-type and drug-resistant EGFR-mutant cells exhibited differences in growth kinetics, with EGFR p.T790M mutant cells growing more slowly than wild-type ones; based on this finding, evolutionary modeling led to optimization of dosing to prolong the clinical benefit of TKIs against EGFR-mutant NSCLC by delaying the development of resistance.^57^ Furthermore, the research showed the effectiveness of using high-dose, pulsed, once-weekly afatinib with daily, low-dose erlotinib in delaying the emergence of p.T790M-mediated resistance. However, the combination of the two EGFR TKIs could lead to overlapping toxicities, comprising

Figure 2 Biochemical pathways leading to resistance to small-molecule epidermal growth factor receptor (EGFR)-targeting drugs such as gefitinib and erlotinib in non-small-cell lung cancer (NSCLC). Simplified pathway diagram showing EGFR signaling through the RAS/MEK/ERK and PI3K/PDK1/AKT pathways, illustrating the points of mutation/amplification in EGFR TKI resistance, along with other mechanisms. The resistance mechanisms include the EGFR p.T790M gatekeeper mutation, amplification of EGFR p.T790M, MET amplification, PI3KCA mutation, and an at least two-fold increase in the expression of GAS6 and its receptor AXL. Incidence rates are given where known. The FAS/nuclear factor-κB (NF-κB) signaling arm downstream of the FAS death receptor has also been shown to be important in EGFR tyrosine kinase inhibitor resistance. In addition, epithelial-to-mesenchymal (EMT) transition changes, perhaps associated with increased activity of AXL, and transformation from the NSCLC to the small-cell lung cancer (SCLC) phenotype can lead to decreased responsiveness. The identification of various resistance mechanisms suggests that a range of clinically actionable therapies could be used to overcome the resistance. For more details, see text.

Figure 3 Envisaging the future of personalized precision medicine for cancer treatment. Future approaches will be based on adaptive therapy in response to information from tumor profiling using multiple technologies, including next-generation sequencing to identify predictive and resistance biomarkers, and incorporating analyses of clonal, morphological, and anatomical heterogeneity and their variations longitudinally in real time.

rashes and diarrhea. Alternative dosing strategies were therefore recommended for erlotinib versus afatinib.

Given the multiplicity of mechanisms of resistance to TKIs, it is likely that the optimal therapeutic tactics will be highly dependent on the actual mechanism in each individual patient. Together with the reality that logical therapies that can be derived from known resistance mechanisms may not always hold up in the clinic, this highlights the need for novel therapeutic strategies based on detailed molecular profiling of individual relapsed tumors.

FUTURE HORIZON FOR PRECISION CANCER MEDICINE

Progressively, and with increasing rapidity, the power of large-scale whole-genome sequencing and other “omics” technologies being used in discovery mode in thousands of patients is defining large numbers of pathogenic driver mutations that are the focus of addiction for cancer cells. These addictions, together with genetics-related synthetic lethalties and other tumor vulnerabilities, have yielded new targets for the current and next generation of molecular therapeutic drugs to treat defined, genetically stratified subgroups of patients with cancer. The hope is that such a personalized strategy in cancer therapy will, in due course, replace the conventional one-size-fits-all cytotoxic chemotherapy approach.

However, extraordinary genetic heterogeneity, both intratumor and intertumor, is being revealed in cancers, especially through deep sequencing. Together with alterations in biochemical signaling pathways and feedback loops—as well as morphological variations, which will also have a genetic basis—this genetic heterogeneity and Darwinian clonal evolution in the face of the selective pressure of therapy provides an explanation for *de novo* and acquired resistance, which is now known to be the major limitation of molecularly targeted drugs, just as was the case with cytotoxic chemotherapy.

The solution to the inevitable challenge of polygenic cancer drug resistance is to identify not only all exploitable molecular abnormalities but also the full range of resistance mechanisms and thereafter to employ precision combinatorial targeted therapy matched specifically to the fully defined tumor profile, as depicted in **Figure 3**. The future of such truly individualized therapies will require not only complete drugging of all possible targets in the cancer genome but also the development of an enhanced comprehensive portfolio of companion predictive biomarkers.

Although whole-exome sequencing is fast becoming much more affordable and is increasingly common in clinical research (for example, to identify resistance mechanisms), it is unlikely to enter routine clinical practice in the next few years. However, with DNA sequencing costs plummeting, the pace of change may well surprise us. An approach that is more rapidly implementable in the short- to mid-term is the assessment of a defined panel of known actionable biomarkers in tumor tissue using next-generation sequencing. There is also potential for sequencing of circulating tumor cells or cell-free DNA to capture a wider picture of the multiple primary and metastatic clones and facilitate repeat analysis, which might also be further enabled by metabolomics or molecular imaging.

We therefore envisage a future requiring adaptive combinatorial treatment to counteract the cellular and molecular heterogeneity of cancer and to prevent or overcome drug resistance caused by clonal evolution (**Figure 3**). In addition to tailoring drug dosages and schedules to optimize pharmacokinetic profiles, this adaptive approach will require not only consideration of anatomic heterogeneity but also the use of longitudinal genome sequencing or other large-scale omics profiling coupled to iterative switching of therapy in order to match the evolving tumor profile and the resulting vulnerabilities.


ACKNOWLEDGMENTS

P.A.C., B.A.-L., and P.W. are grateful for major core support from Cancer Research UK (programme grant C309/A8725). P.W. is a Cancer Research UK Life Fellow. D.G.d.C. thanks Cancer Research UK for funding through the Stratified Medicine Programme (C21807/A13641). The authors acknowledge funding to The Institute of Cancer Research and the Royal Marsden Hospital as a Cancer Research UK Centre and from the National Institute of Health Research to our Biomedical Research Centre. We apologize to the authors of numerous excellent papers that could not be cited because of space constraints. We thank our many colleagues, collaborators, and laboratory staff for valuable discussions. We thank Val Cornwell for excellent administrative support.


CONFLICT OF INTEREST

P.W., B. A.-L., and P.A.C. are employees of The Institute of Cancer Research, which has a commercial interest in the discovery and development of anticancer drugs, including kinase inhibitors, and operates a Rewards to Inventors scheme. P.W. is a former employee of AstraZeneca and declares relevant commercial interactions with Yamanouchi (now Astellas), Piramed Pharma (acquired by Roche), Genentech, Vernalis, Novartis, Chroma Therapeutics, Astex Pharmaceuticals, AstraZeneca, Cyclacel, Onyx Pharmaceuticals, Merck Serono, Sareum, Janssen, Wilex, and Nextech Ventures. B. A.-L. is a former employee of Inpharmatica. D.G.d.C. is an employee of the Royal Marsden Foundation Trust and has received honoraria from AstraZeneca, Roche Products, Roche Molecular Systems, GSK, Boehringer Ingelheim, and GE Healthcare.


© 2013 American Society for Clinical Pharmacology and Therapeutics


1. Collins, I. & Workman, P. New approaches to molecular cancer therapeutics. *Nat. Chem. Biol.* **2**, 689–700 (2006).
2. Yap, T.A., Sandhu, S.K., Workman, P. & de Bono, J.S. Envisioning the future of early anticancer drug development. *Nat. Rev. Cancer* **10**, 514–523 (2010).
3. Zhang, J., Yang, P.L. & Gray, N.S. Targeting cancer with small molecule kinase inhibitors. *Nat. Rev. Cancer* **9**, 28–39 (2009).
4. Giannas, G. *et al.* Kinases as targets in the treatment of solid tumors. *Cell. Signal.* **22**, 984–1002 (2010).
5. Gaulton, A. *et al.* ChEMBL: a large-scale bioactivity database for drug discovery. *Nucleic Acids Res.* **40**, D1100–D1107 (2012).
6. Halling-Brown, M.D., Bulusu, K.C., Patel, M., Tym, J.E. & Al-Lazikani, B. canSAR: an integrated cancer public translational research and drug discovery resource. *Nucleic Acids Res.* **40**, D947–D956 (2012).
7. Patel M., Halling-Brown, M.D., Tym, J.E., Workman, P. & Al-Lazikani, B. Objective assessment of cancer genes for drug discovery. *Nat. Rev. Drug. Discov.* **12**, 35–50 (2012).
8. Futreal, P.A. *et al.* A census of human cancer genes. *Nat. Rev. Cancer* **4**, 177–183 (2004).
9. Knapp, S.B.J., *et al.* A public-private partnership to unlock the untargeted kinome. *Nat. Chem. Biol.* **9**, 3–6 (2013).
10. Edwards, A.M., Isserlin, R., Bader, G.D., Frye, S.V., Willson, T.M. & Yu, F.H. Too many roads not taken. *Nature* **470**, 163–165 (2011).
11. Hoelder, S., Clarke, P.A. & Workman, P. Discovery of small molecule cancer drugs: successes, challenges and opportunities. *Mol. Oncol.* **6**, 155–176 (2012).
12. Peeters, M. *et al.* Randomized phase III study of panitumumab with fluorouracil, leucovorin, and irinotecan (FOLFIRI) compared with FOLFIRI alone as second-line treatment in patients with metastatic colorectal cancer. *J. Clin. Oncol.* **28**, 4706–4713 (2010).
13. Van Cutsem, E. *et al.* Cetuximab plus irinotecan, fluorouracil, and leucovorin as first-line treatment for metastatic colorectal cancer: updated analysis of overall survival according to tumor KRAS and BRAF mutation status. *J. Clin. Oncol.* **29**, 2011–2019 (2011).
14. Fukuoka, M. *et al.* Biomarker analyses and final overall survival results from a phase III, randomized, open-label, first-line study of gefitinib versus carboplatin/paclitaxel in clinically selected patients with advanced non-small-cell lung cancer in Asia (IPASS). *J. Clin. Oncol.* **29**, 2866–2874 (2011).
15. Mok, T.S. *et al.* Gefitinib or carboplatin-paclitaxel in pulmonary adenocarcinoma. *N. Engl. J. Med.* **361**, 947–957 (2009).
16. Rosell, R. *et al.* Erlotinib versus standard chemotherapy as first-line treatment for European patients with advanced EGFR mutation-positive non-small-cell lung cancer (EURTAC): a multicentre, open-label, randomised phase 3 trial. *Lancet Oncol.* **13**, 239–246 (2012).


VOLUME 93 NUMBER 3 | MARCH 2013 | [www.nature.com/cpt](http://www.nature.com/cpt)

17. Chapman, P.B. *et al.*; BRIM-3 Study Group. Improved survival with vemurafenib in melanoma with BRAFV600E mutation. *N. Engl. J. Med.* **364**, 2507–2516 (2011).
18. Hauschild, A. *et al.* Dabrafenib in BRAF-mutated metastatic melanoma: a multicentre, open-label, phase 3 randomised controlled trial. *Lancet* **380**, 358–365 (2012).
19. Vardiman, J.W. *et al.* The 2008 revision of the World Health Organization (WHO) classification of myeloid neoplasms and acute leukemia: rationale and important changes. *Blood* **114**, 937–951 (2009).
20. Thway, K. *et al.* Utility of sarcoma-specific fusion gene analysis in paraffin-embedded material for routine diagnosis at a specialist centre. *J. Clin. Pathol.* **63**, 508–512 (2010).
21. Gonzalez, D. *et al.* Mutational status of the TP53 gene as a predictor of response and survival in patients with chronic lymphocytic leukemia: results from the LRF CLL4 trial. *J. Clin. Oncol.* **29**, 2223–2229 (2011).
22. Pospisilova, S. *et al.* ERIC recommendations on TP53 mutation analysis in chronic lymphocytic leukemia. *Leukemia* **26**, 1458–1461 (2012).
23. Oscier, D. *et al.* Prognostic factors identified three risk groups in the LRF CLL4 trial, independent of treatment allocation. *Haematologica* **95**, 1705–1712 (2010).
24. Gonzalez, D. *et al.* CLLU1 expression has prognostic value in CLL after first-line therapy in younger patients and those with mutated IGHV genes. *Haematologica*, (2012); e-pub ahead of print 16 August 2012.
25. Paik, S. *et al.* A multigene assay to predict recurrence of tamoxifen-treated, node-negative breast cancer. *N. Engl. J. Med.* **351**, 2817–2826 (2004).
26. Glas, A.M. *et al.* Converting a breast cancer microarray signature into a high-throughput diagnostic test. *BMC Genomics* **7**, 278 (2006).
27. Bartlett, J.M. *et al.* Mammostrat as an Immunohistochemical Multigene Assay for Prediction of Early Relapse Risk in the Tamoxifen Versus Exemestane Adjuvant Multicenter Trial Pathology Study. *J. Clin. Oncol.* (2012); e-pub ahead of print 8 October 2012.
28. Cuzick, J. *et al.* Prognostic value of a combined estrogen receptor, progesterone receptor, Ki-67, and human epidermal growth factor receptor 2 immunohistochemical score and comparison with the Genomic Health recurrence score in early breast cancer. *J. Clin. Oncol.* **29**, 4273–4278 (2011).
29. Kantarjian, H.M. *et al.* Dasatinib or imatinib in newly diagnosed chronic-phase chronic myeloid leukemia: 2-year follow-up from a randomized phase 3 trial (DASISION). *Blood* **119**, 1123–1129 (2012).
30. Kantarjian, H.M. *et al.* Nilotinib versus imatinib for the treatment of patients with newly diagnosed chronic phase, Philadelphia chromosome-positive, chronic myeloid leukaemia: 24-month minimum follow-up of the phase 3 randomised ENESTnd trial. *Lancet Oncol.* **12**, 841–851 (2011).
31. Schrappe, M. *et al.* Late MRD response determines relapse risk overall and in subsets of childhood T-cell ALL: results of the AIEOP-BFM-ALL 2000 study. *Blood* **118**, 2077–2084 (2011).
32. Conter, V. *et al.* Molecular response to treatment redefines all prognostic factors in children and adolescents with B-cell precursor acute lymphoblastic leukemia: results in 3184 patients of the AIEOP-BFM ALL 2000 study. *Blood* **115**, 3206–3214 (2010).
33. Mattocks, C.J. *et al.*; EuroGentest Validation Group. A standardized framework for the validation and verification of clinical molecular genetic tests. *Eur. J. Hum. Genet.* **18**, 1276–1288 (2010).
34. Kimura, T. *et al.* Clinical benefit of high-sensitivity KRAS mutation testing in metastatic colorectal cancer treated with anti-EGFR antibody therapy. *Oncology* **82**, 298–304 (2012).
35. Su, K.Y. *et al.* Pretreatment epidermal growth factor receptor (EGFR) T790M mutation predicts shorter EGFR tyrosine kinase inhibitor response duration in patients with non-small-cell lung cancer. *J. Clin. Oncol.* **30**, 433–440 (2012).
36. Gerlinger, M. *et al.* Intratumor heterogeneity and branched evolution revealed by multiregion sequencing. *N. Engl. J. Med.* **366**, 883–892 (2012).
37. Anderson, K. *et al.* Genetic variegation of clonal architecture and propagating cells in leukaemia. *Nature* **469**, 356–361 (2011).

38. Colombino, M. *et al.* BRAF/NRAS mutation frequencies among primary tumors and metastases in patients with melanoma. *J. Clin. Oncol.* **30**, 2522–2529 (2012).
39. Perkins, G. *et al.* Multi-purpose utility of circulating plasma DNA testing in patients with advanced cancers. *PLoS ONE* **7**, e47020 (2012).
40. Tejpar, S., Celik, I., Schlichting, M., Sartorius, U., Bokemeyer, C. & Van Cutsem, E. Association of KRAS G13D tumor mutations with outcome in patients with metastatic colorectal cancer treated with first-line chemotherapy with or without cetuximab. *J. Clin. Oncol.* **30**, 3570–3577 (2012).
41. Blanke C.D. *et al.* Long-term results from a randomized phase II trial of standard- versus higher-dose imatinib mesylate for patients with unresectable or metastatic gastrointestinal stromal tumors expressing KIT. *J. Clin. Oncol.* **26**, 620–625 (2008).
42. Verweij, J. *et al.* Progression-free survival in gastrointestinal stromal tumours with high-dose imatinib: randomized trial. *Lancet* **364**, 1127–1134 (2004).
43. Kim, E.S. *et al.* The BATTLE trial: personalizing therapy for lung cancer. *Cancer Discov.* **1**, 44–53 (2011).
44. Dietrich, S., Glimm, H., Andrulis, M., von Kalle, C., Ho, A.D. & Zenz, T. BRAF inhibition in refractory hairy-cell leukemia. *N. Engl. J. Med.* **366**, 2038–2040 (2012).
45. Prahallad, A. *et al.* Unresponsiveness of colon cancer to BRAF(V600E) inhibition through feedback activation of EGFR. *Nature* **483**, 100–103 (2012).
46. Al-Lazikani, B., Banerji, U. & Workman, P. Combinatorial drug therapy for cancer in the post-genomic era. *Nat. Biotechnol.* **30**, 679–692 (2012).
47. Garraway L.A. & Janne P.A. Circumventing cancer drug resistance in the era of personalized medicine. *Cancer Discov.* **2**, 214–226 (2012).
48. Workman, P. & Clarke, P.A. Resisting targeted therapy: fifty ways to leave your EGFR. *Cancer Cell.* **19**, 437–440 (2011).
49. Postel-Vinay, S. & Ashworth, A. AXL and acquired resistance to EGFR inhibitors. *Nat. Genet.* **44**, 835–836 (2012).
50. Bivona, T.G. *et al.* FAS and NF-kB signalling modulate dependence of lung cancers on mutant EGFR. *Nature* **471**, 523–526 (2011).
51. Sequist, L.V. *et al.* Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. *Sci. Transl. Med.* **3**, 75ra26 (2011).
52. Rekhtman, N. *et al.* Clarifying the spectrum of driver oncogene mutations in biomarker-verified squamous carcinoma of lung: lack of EGFR/KRAS and presence of PIK3CA/AKT1 mutations. *Clin. Cancer Res.* **18**, 1167–1176 (2012).
53. Voulgari, A. & Pintzas, A. Epithelial-mesenchymal transition in cancer metastasis: mechanisms, markers and strategies to overcome drug resistance in the clinic. *Biochim. Biophys. Acta* **1796**, 75–90 (2009).
54. Nurwidya, F., Takahashi, F., Murakami, A. & Takahashi, K. Epithelial mesenchymal transition in drug resistance and metastasis of lung cancer. *Cancer Res. Treat.* **44**, 151–156 (2012).
55. Huang, S. *et al.* MED12 controls the response to multiple cancer drugs through regulation of TGF-β receptor signaling. *Cell* **151**, 937–950 (2012).
56. Miller, V.A. *et al.* Afatinib versus placebo for patients with advanced, metastatic non-small-cell lung cancer after failure of erlotinib, gefitinib, or both, and one or two lines of chemotherapy (LUX-Lung 1): a phase 2b/3 randomized trial. *Lancet Oncol.* **13**, 528–538 (2012).
57. Chmielecki, J. *et al.* Optimization of dosing for EGFR-mutant non-small cell lung cancer with evolutionary cancer modeling. *Sci. Transl. Med.* **3**, 90ra59 (2011).
58. Yap, T.A. & Workman, P. Exploiting the cancer genome: strategies for the discovery and clinical development of targeted molecular therapeutics. *Annu. Rev. Pharmacol. Toxicol.* **52**, 549–573 (2012).
59. Chen, R. *et al.* Personal omics profiling reveals dynamic molecular and medical phenotypes. *Cell* **148**, 1293–1307 (2012).

This work is licensed under a Creative Commons Attribution-NonCommercial-No Derivative Works 3.0 License. To view a copy of this license, visit http://creativecommons.org/licenses/by-nc-nd/3.0/
