
Current Topics

Stem Cell Research for Regenerative Medicine/Personalized Medicine

---

### Transcription Factor Network in Embryonic Stem Cells: Heterogeneity under the Stringency

#### Yoko Nakai-Futatsugi and Hitoshi Niwa*

Laboratory for Pluripotent Stem Cell Studies, RIKEN Center for Developmental Biology;  
2–2–3 Minatojima-minamimachi, Chuo-ku, Kobe 650–0047, Japan.

Received October 30, 2012

Leukemia inhibitory factor (LIF) signaling regulates transcription factors to maintain the self-renewability and pluripotency of embryonic stem (ES) cells. Recently, we have proposed a network model that consists of transcription factors such as Klf4, Sox2, Tbx3, Nanog, and Oct3/4, which form a parallel pathway downstream from LIF signaling (*Nature*, 460, 2009, Niwa *et al.*). In this parallel pathway, the transcription factors maintain the pluripotency of ES cells through mutual balance with some degree of redundancy and compensation. While self-renewability and pluripotency are maintained well under such seemingly stringent regulation, studies of single cells revealed heterogeneity among individual ES cells. This heterogeneity may underlie the mechanism that allows ES cells to exit self-renewal and enter into differentiation to exert pluripotency. Here we focus on recent studies on the heterogeneity of ES cells and discuss their inherent metastability.

**Key words** embryonic stem cell; transcription factor network; metastability

---

### 1. INTRODUCTION

During mouse development, pluripotency is established in the inner cell mass (ICM) of the blastocyst around embryonic day 3.0 (E3.0) (Fig. 1). Pluripotency is the capacity of an individual cell to give rise to all other cell types of the body and the germ line, which is an emergent property realized in the ICM. Pluripotency persists until the ICM develops into the primitive ectoderm (PrEc) by E5.5 (Fig. 1). Embryonic stem (ES) cells are the cells captured from the ICM during this short time period (24 h *in vivo*), which continue to multiply while remaining pluripotent indefinitely under certain culture conditions.^(1,2)^ The factor that maintains the cells in this “forever-young” state was identified^(3)^ and determined to be identical to the cytokine, leukemia inhibitory factor (LIF).^(4,5)^ LIF signaling is mediated *via* the signaling molecule Stat3^(6)^ and regulates the pluripotency transcription factor network.^(7)^ This regulation maintains the core transcription factor Oct3/4 at a certain level, which apparently makes the population ‘homogeneous’ with indefinite self-renewability and pluripotency.

---

#### Early blastocyst                      Late blastocyst                       Egg cylinder stage
Inner cell mass (ICM)                     Epiblast                                Primitive ectoderm
Nanog                                    Gata6                                   Visceral endoderm
Trophectoderm                           Primitive endoderm                      Parietal endoderm
                                          Extraembryonic ectoderm                Proamniotic cavity
                                                                                   Somatic cell lineages
                                                                                   Germ cell lineage
                                                                                   Ectoderm
                                                                                   Mesoderm
                                                                                   Endoderm

---

**Fig. 1.** Pluripotent Lineages in the Mouse Embryo  
Modified from Niwa H, *Development*, 134, 635–646 (2007).

---

The authors declare no conflict of interest.

*e-mail*: niwa@cdb.riken.jp

© 2013 The Pharmaceutical Society of Japan

February 2013

how do the cells differentiate by exiting this well-maintained condition? Detailed studies on single cells showed that ES cells are heterogeneous and fluctuating even under the same culture conditions, which supposedly makes them continually poised for differentiation in response to external signals.

## 2. TRANSCRIPTION FACTOR NETWORK SAFE-GUARDING PLURIPOTENCY

In pluripotent cells of the ICM, transcription factors such as Oct3/4, Sox2, and Nanog are specifically expressed, while in its counterpart, the trophoectoderm, another transcription factor, Cdx2, is specifically expressed. The transcription factors that govern the pluripotency become downregulated during differentiation, while other transcription factors such as Gata6 (a primitive endoderm (PrE) marker) become upregulated (Fig. 1). Thus transcription factors are key elements that define the undifferentiated state of the ICM and its derivatives, ES cells. Among them, Oct3/4 and Sox2 are considered to be core pluripotency transcription factors: Oct3/4 is essential to establish pluripotent cells in the ICM and in the absence of Oct3/4 the cells of the ICM eventually differentiate into the tropheblast lineage; and Sox2 cooperatively acts with Oct3/4. How LIF signaling is connected to these core factors was depicted from one aspect by a gain-of-function study in the absence of LIF: exogenous expression of Klf4 maintained Sox2 (but not Nanog or Tbx3) in the absence of LIF, and this pathway was significantly suppressed by the inhibition of Stat3 signaling. Likewise, exogenous expression of Tbx3 maintained Nanog (but not Klf4 or Sox2) in the absence of LIF, and this pathway was suppressed by the inhibition of phosphatidylinositol-3-OH kinase (PI3K)-Akt signaling and maintained by the constitutive activation of PI3K-Akt. In addition, the inhibition of mitogen-activated protein kinase (MAPK) significantly upregulated both Tbx3 and Nanog, which indicates that the Tbx3-Nanog pathway is positively and negatively regulated by PI3K-Akt and MAPK, respectively. Regardless of the artificial alteration in each factor, however, the level of Oct3/4 was always maintained.

Taking those results together, we developed a model in which Oct3/4 is protected by two independent parallel pathways (Fig. 2A). The most upstream signals of this parallel pathway, Stat3 and PI3K-Akt, are both activated by LIF. However, the withdrawal of LIF does not cause immediate collapse of the network (Fig. 2B). Upon the withdrawal of LIF, Oct3/4 did not show significant reduction at least for 24 h. Although Tbx3 and Klf4 showed a rather rapid response, they were not completely depleted within 24 h but continued to decrease by day 4. This duration appears rather long for the dephosphorylation of signaling molecules and translocation/degradation of transcription factors after the withdrawal of LIF. One explanation for this may be the involvement of positive feedback from Oct3/4, Sox2, or Nanog to the entire network. Although the Stat3-Klf4-Sox2 pathway was illustrated at one side of the parallel pathway, constitutive activation of Stat3 did not exclusively activate Klf4 and Sox2, but instead in long-term (3-week) culture it brought all factors to normal levels in the absence of LIF. This suggests that there is also crosstalk among the factors, which may activate or suppress each other when one becomes too low or too high, respectively.

In this model, the mutual regulation of transcription factors

### Fig. 2.
(A) Parallel Pathway Network Downstream from LIF (B) Trend Curve of Expression Levels of the Core Transcription Factors after Withdrawal of LIF

(A) Stat3 signaling and PI3K-Akt signaling predominantly activate Klf4-Sox2 and Tbx3-Nanog, respectively. (B) Oct3/4 shows a delayed response. (Modified from ref. 7)

maintains the core factor Oct3/4 at a certain level. The expression level of Oct3/4 is crucial for maintaining ES cells in an undifferentiated state, as a slight increase in its expression level leads to differentiation into the PrE and mesoderm, while a slight decrease in its expression level leads to differentiation into the trophoectoderm. The existence of a parallel pathway with a feedback loop and crosstalk may explain how the expression level of Oct3/4 is maintained within such a critical range.

## 3. MONITORING THE NETWORK STATUS WITH THE INDICATOR REX1

As mentioned above, regulating the Oct3/4 level is crucial for ES cells to maintain their pluripotency. Although the level of each transcription factor likely increases and decreases easily, the alterations are compensated for by a mutual balance in the network. How can this 'balance' (rather than single components) in the network be monitored?

Rex1 (also known as Zfp42) is a transcription factor that is strongly expressed in the ICM but downregulated in the epiblast thus is commonly used as a marker of pluripotency. ${ }^{12)}$ Although it makes no contribution to the maintenance of pluripotency, ${ }^{13)}$ its promoter activity sensitively reflects the expression level of Oct3/4 *in vitro*: when $Oct3 / 4$ is either up-regulated OR downregulated, the promoter activity of Rex1 is turned off ${ }^{11)}$ (Fig. 3). Thus the insertion of $GFP$ under the Rex1-promoter (Rex1-GFP) allows monitoring of the network status at the single-cell level, and indeed the dynamics indicated by Rex1 was revealed for the first time. ${ }^{14)}$

---

### 4. HETEROGENEITY OF REX1

ES cells are derived from the ICM and were thought to have a homogeneous population in culture. However, when ES cells expressing Rex1-GFP were monitored closely, there were at least two different populations: $R e x l^{+} / O c t 3 / 4^{+}$ cells; and $R e x l^{-} / O c t 3 / 4^{+}$ cells. ${ }^{14)}$ As Rex1 is exclusively expressed in the ICM *in vivo*, these could represent an ICM-like and -unlike population, respectively. Pluripotent cells in the ICM differentiate into the epiblast and then into the PrEc (Fig. 1). Rex1-negative cells appear to have a more differentiated PrEc-like character as they express PrEc markers such as $F g f 5$, $T$, and $E o m e s$ at a higher level compared with Rex1-positive cells and express undifferentiated markers such as $K l f 4, T b x 3$, Nanog, and $E s r r b$ at a lower level. ${ }^{14)}$ Interestingly, these two conditions were found to be interconvertible: when either Rex1-positive or -negative cells were isolated, both eventually gave rise to a mixed population of Rex1-positive and -negative cells again. ${ }^{14)}$

As reasoned earlier (Fig. 3), in Rex1-positive cells the transcription factor network should be well balanced with Oct3/4 at a firmly stable level, while in Rex1-negative cells it should be slightly unbalanced. The heterogeneity and interconvertibility of Rex1-expression in ES cells can be interpreted as fluctuations in the network providing an opportunity for differentiation but also allowing a return to a balanced condition.

---

### 5. HETEROGENEITY OF OTHER LINAGE-SPECIFIC MOLECULES IN ES CELLS

Nanog is a homeodomain protein expressed in pluripotent cells, and its downregulation is an early marker of differentiation. However, a study on Nanog-null ES cells showed that it is dispensable for the self-renewal of ES cells, although still required for germline development *in vivo*. ${ }^{15)}$ The ICM at E3.5 shows a random “salt-and-pepper” pattern of Nanog and Gata6 expression, ${ }^{16)}$ including overlapping expression of both in a single cell, ${ }^{17)}$ which afterward segregate into the epiblast and the PrE (Fig. 1) by E4.5. ${ }^{16,17)}$ ES cells in normal culture conditions also consistently show heterogeneous expression of Nanog, ${ }^{15,18)}$ and ES cells with low levels of Nanog predominantly express the PrE marker $G a t a 6 .{ }^{18)}$ As in the heterogeneity of Rex1, Nanog-high and -low conditions are reversible. ${ }^{15,18)}$

Tbx3 and Klf4 also show heterogeneous expression by immunostaining. ${ }^{7)}$ Although their expression is not always synchronized, ${ }^{7)}$ both show a close correlation with Rex1-GFP expression. ${ }^{14)}$ This strongly suggests that, as in Rex1-positive/-negative cells, both Tbx3-positive/-negative and Klf4-positive/-negative conditions should be reversible.

February 2013

**Early blastocyst**                                      **Late blastocyst**

![Diagram](attachment:diagram.png)

Fig. 4. Trapped Steady-State Heterogeneity in ES Cells

Expression patterns of Nanog in embryos and in ES cells are shown. While acquiring indefinite self-renewability, ES cells may also have acquired indefinite heterogeneity.

---

### 6. MECHANISMS OF HETEROGENEITY

Which mechanisms allow ES cells to show such transcriptional heterogeneity? A recent study has shown the involvement of the nucleosome remodeling and deacetylation (NuRD) complex in ES cell heterogeneity.²⁴ Generally, the NuRD complex functions as a transcriptional repressor. In ES cells, it is required for differentiation by silencing the pluripotency-associated genes.²⁵ It binds proximally to the transcription start sites of pluripotency-associated genes such as *Klf4*, *Klf5*, *Rexl*, *Tbx3*, *Esrrb*, and *Zfp57* even in the undifferentiated state, albeit it has an inhibitory function.²⁴ In mutant ES cells lacking NuRD activity, heterogeneous expression of pluripotency-associated factors such as *Klf4* becomes homogeneous, and their overall expression is elevated.²⁴ Thus NuRD silences the genes in ES cells in an approximate balance with putative activators, which may be one of the mechanisms of silencing developmental genes in ES cells incompletely while still maintaining them poised for activation.

Another transcriptional repressor, *Hes1*, also shows heterogeneous expression in ES cells with an oscillatory period of 3–5 h.²⁶ It regulates the cell fate via repression of Notch signaling.²⁷ ES cells expressing low and high levels of *Hes1* are prone to differentiate into neural and mesodermal cells, respectively, and *Hes1*-null ES cells display less heterogeneity in both the differentiation timing and fate choice.²⁶

Not only the transcriptional regulators but also the structure of our network model itself can drive heterogeneity (Fig. 2A). If the balance of the network is maintained by feedback and crosstalk among the transcription factors, there should be a time delay before the balance is adjusted after an alteration in one of the factors. Thus this delay may appear as heterogeneity in terms of expression levels of individual transcription factors composing the network. If there is an applicable mathematical model, the time delay may allow us to predict the network structure, or vice versa.

---

### Whether ES cells are inherently metastable is still under discussion. Smith's group showed that ES cells form a homogeneous population when MAPK and glycogen synthase kinase 3 (GSK3) are blocked: in a defined medium with a MAPK inhibitor and a GSK3 inhibitor (so-called 2i-culture), ES cells are not only homogeneous but are also able to self-renew without LIF and serum.²⁸ This may suggest that ES cells are inherently homogeneous when the external signaling from MAPK that induces differentiation is removed. However, whether ES cells are indeed shielded from all external signals (to achieve the so-called "ground state"²⁸) in the presence of these two inhibitors is controversial, as the GSK3 inhibitor added here consequently becomes a Wnt-signaling activator²⁹ (because Wnt-signaling is blocked by GSK3, and Wnt-signaling is involved in many important cell biological activities), and also as it has recently been shown that the inhibition of GSK3 activates the pluripotency factor *Esrrb* via repression of the transcriptional repressor *Tcf3*.³⁰

---

### 7. CONCLUSION

ES cells are the cells captured during the short time period when the pluripotent cells of the ICM differentiate into the early epiblast. They are able to self-renew indefinitely, contrary to the fate of the ICM that is programmed to undergo differentiation within 24 h. To acquire this self-renewability, some genes change their expression profile during the transformation from ICM to ES cells: the expression of differentiation factors such as *GATA6* and *Nr5a2* is decreased and that of self-renewing factors such as *Tcf15* and *Nodal* is increased.³¹ Nevertheless, they inherit the pluripotency of their origin. The heterogeneity of ES cells may represent chaotic expression of lineage-specific molecules during a brief time window in the ICM (Fig. 4). Interestingly, all the heterogeneities described here are interconvertible. This may be interpreted as, although the chaotic expression of the factors during immediate-early

differentiation is only for a limited period of time *in vivo*, ES cells are trapped in the steady state and thus they continue fluctuating *in vitro* indefinitely.

While maintaining the self-renewability of ES cells is one area of interest, determining the mechanism of their exit from self-renewal and entry into differentiation is another. When ES cells are in a homogeneous population under culture conditions such as 2i-culture, they do not differentiate. And, as described here, the heterogeneity of lineage-specific factors accompanies biases in the competency of ES cells to differentiate. Thus fluctuations of self-renewing regulators could be responsible for making ES cells always poised to undergo differentiation.

We do not consider our proposed network model (Fig. 2A) as a master model that covers the whole transcription factor network. Rather it represents one of the ways to dissect the entire network. Still, based on its stringent but flexible parallel pathway, we can interpret both the safeguard of self-renewability and the adjustable heterogeneity that continuously provide an opportunity for exerting pluripotency in response to external signals.

**REFERENCES**

1) Evans MJ, Kaufman MH. Establishment in culture of pluripotential cells from mouse embryos. *Nature*, **292**, 154–156 (1981).

2) Martin GR. Isolation of a pluripotent cell line from early mouse embryos cultured in medium conditioned by teratocarcinoma stem cells. *Proc. Natl. Acad. Sci. U.S.A.*, **78**, 7634–7638 (1981).

3) Smith AG, Hooper ML. Buffalo rat liver cells produce a diffusible activity which inhibits the differentiation of murine embryonal carcinoma and embryonic stem cells. *Dev. Biol.*, **121**, 1–9 (1987).

4) Smith AG, Heath JK, Donaldson DD, Wong GG, Moreau J, Stahl M, Rogers D. Inhibition of pluripotential embryonic stem cell differentiation by purified polypeptides. *Nature*, **336**, 688–690 (1988).

5) Williams RL, Hilton DJ, Pease S, Willson TA, Stewart CL, Gearing DP, Wagner EF, Metcalf D, Nicola NA, Gough NM. Myeloid leukaemia inhibitory factor maintains the developmental potential of embryonic stem cells. *Nature*, **336**, 684–687 (1988).

6) Niwa H, Burdon T, Chambers I, Smith A. Self-renewal of pluripotent embryonic stem cells is mediated via activation of STAT3. *Genes Dev.*, **12**, 2048–2060 (1998).

7) Niwa H, Ogawa K, Shimosato D, Adachi K. A parallel circuit of LIF signalling pathways maintains pluripotency of mouse ES cells. *Nature*, **460**, 118–122 (2009).

8) Nichols J, Zevnik B, Anastassiadis K, Niwa H, Klewe-Nebenius D, Chambers I, Schöler H, Smith A. Formation of pluripotent stem cells in the mammalian embryo depends on the POU transcription factor Oct4. *Cell*, **95**, 379–391 (1998).

9) Yuan H, Corbi N, Basilico C, Dailey L. Developmental-specific activity of the FGF-4 enhancer requires the synergistic action of Sox2 and Oct-3. *Genes Dev.*, **9**, 2635–2645 (1995).

10) Paling NR, Wheadon H, Bone HK, Welham MJ. Regulation of embryonic stem cell self-renewal by phosphoinositide 3-kinase-dependent signaling. *J. Biol. Chem.*, **279**, 48063–48070 (2004).

11) Niwa H, Miyazaki J, Smith AG. Quantitative expression of Oct-3/4 defines differentiation, dedifferentiation or self-renewal of ES cells. *Nat. Genet.*, **24**, 372–376 (2000).

12) Rogers MB, Hosler BA, Gudas LJ. Specific expression of a retinoic acid-regulated, zinc-finger gene, Rex-1, in preimplantation embryos, trophoblast and spermatocytes. *Development*, **113**, 815–824 (1991).

13) Masui S, Ohtsuka S, Yagi R, Takahashi K, Ko MS, Niwa H. Rex1/Zfp42 is dispensable for pluripotency in mouse ES cells. *BMC Dev. Biol.*, **8**, 45 (2008).

14) Toyooka Y, Shimosato D, Murakami K, Takahashi K, Niwa H. Identification and characterization of subpopulations in undifferentiated ES cell culture. *Development*, **135**, 909–918 (2008).

15) Chambers I, Silva J, Colby D, Nichols J, Nijmeijer B, Robertson M, Vrana J, Jones K, Grotewold L, Smith A. Nanog safeguards pluripotency and mediates germline development. *Nature*, **450**, 1230–1234 (2007).

16) Chazaud C, Yamanaka Y, Pawson T, Rossant J. Early lineage segregation between epiblast and primitive endoderm in mouse blastocysts through the Grb2-MAPK pathway. *Dev. Cell*, **10**, 615–624 (2006).

17) Plasa B, Piliszek A, Frankenberg S, Artus J, Hadjantonakis AK. Distinct sequential cell behaviours direct primitive endoderm formation in the mouse blastocyst. *Development*, **135**, 3081–3091 (2008).

18) Singh AM, Hamazaki T, Hankowski KE, Terada N. A heterogeneous expression pattern for Nanog in embryonic stem cells. *Stem Cells*, **25**, 2534–2542 (2007).

19) Hayashi K, Lopes SM, Tang F, Surani MA. Dynamic equilibrium and heterogeneity of mouse pluripotent stem cells with distinct functional and epigenetic states. *Cell Stem Cell*, **3**, 391–401 (2008).

20) Furusawa T, Ohkoshi K, Honda C, Takahashi S, Tokunaga T. Embryonic stem cells expressing both platelet endothelial cell adhesion molecule-1 and stage-specific embryonic antigen-1 differentiate predominantly into epiblast cells in a chimeric embryo. *Biol. Reprod.*, **70**, 1452–1457 (2004).

21) Canham MA, Sharov AA, Ko MS, Brickman JM. Functional heterogeneity of embryonic stem cells revealed through translational amplification of an early endodermal transcript. *PLoS Biol.*, **8**, e1000379 (2010).

22) Falco G, Lee SL, Stanghellini I, Bassey UC, Hamatani T, Ko MS. Zscan4: a novel gene expressed exclusively in late 2-cell embryos and embryonic stem cells. *Dev. Biol.*, **307**, 539–550 (2007).

23) Zalzman M, Falco G, Sharova LV, Nishiyama A, Thomas M, Lee SL, Stagg CA, Hoang HG, Yang HT, Indig FE, Wersto RP, Ko MS. Zscan4 regulates telomere elongation and genomic stability in ES cells. *Nature*, **464**, 858–863 (2010).

24) Reynolds N, Latos P, Hynes-Allen A, Loos R, Leaford D, O'Shaughnessy A, Mosaku O, Signolet J, Brennecke P, Kalkan T, Costello I, Humphreys P, Mansfield W, Nakagawa K, Strouboulis J, Behrens A, Bertone P, Hendrich B. NuRD suppresses pluripotency gene expression to promote transcriptional heterogeneity and lineage commitment. *Cell Stem Cell*, **10**, 583–594 (2012).

25) Kaji K, Caballero IM, MacLeod R, Nichols J, Wilson VA, Hendrich B. The NuRD component Mbd3 is required for pluripotency of embryonic stem cells. *Nat. Cell Biol.*, **8**, 285–292 (2006).

26) Kobayashi T, Mizuno H, Imayoshi I, Furusawa C, Shirahige K, Kageyama R. The cyclic gene Hes1 contributes to diverse differentiation responses of embryonic stem cells. *Genes Dev.*, **23**, 1870–1875 (2009).

27) Kobayashi T, Kageyama R. Hes1 regulates embryonic stem cell differentiation by suppressing Notch signaling. *Genes Cells*, **15**, 689–698 (2010).

28) Ying QL, Wray J, Nichols J, Batlle-Morera L, Doble B, Woodgett J, Cohen P, Smith A. The ground state of embryonic stem cell self-renewal. *Nature*, **453**, 519–523 (2008).

29) Loh KM, Lim B. A precarious balance: pluripotency factors as lineage specifiers. *Cell Stem Cell*, **8**, 363–369 (2011).

30) Martello G, Sugimoto T, Diamanti E, Joshi A, Hannah R, Ohtsuka S, Göttgens B, Niwa H, Smith A. Esrrb is a pivotal target of the Gsk3/Tcf3 axis regulating embryonic stem cell self-renewal. *Cell Stem Cell*, **11**, 491–504 (2012).

31) Tang F, Barbacioru C, Bao S, Lee C, Nordman E, Wang X, Lao K, Surani MA. Tracing the derivation of embryonic stem cells from the inner cell mass by single-cell RNA-Seq analysis. *Cell Stem Cell*, **6**, 468–478 (2010).
