
Journal of Bioscience and Bioengineering
VOL. 116 No. 1, 91–100, 2013

www.elsevier.com/locate/jbiosc

---

**Gene cluster and regulation system for 1,1-dichloro-2,2-bis(4-chlorophenyl) ethylene (DDE) degradation in *Janibacter* sp. TYM3221**

Phuong Anh Thi Nguyen,$^{1}$ Thu Hang Thi Trinh,$^{1}$ Yuki Fukumitsu,$^{1}$ Jun Shimodaira,$^{1}$ Keisuke Miyauchi,$^{2}$ Masahiro Tokuda,$^{3}$ Daisuke Kasai,$^{1}$ Eiji Masai,$^{1}$ and Masao Fukuda$^{1,*}$

Department of Bioengineering, Nagaoka University of Technology, Kamitomioka, Nagaoka, Niigata 940-2188, Japan,$^{1}$ Department of Civil and Environmental Engineering, Tohoku Gakuin University, Chuo, Tagajo, Miyagi 985-8537, Japan,$^{2}$ and Sugiyama Chemical & Industrial Laboratory, Totsuka, Yokohama, Kanagawa 245-0064, Japan$^{3}$

Received 28 November 2012; accepted 16 January 2013  
Available online 13 February 2013

A DDE-degrading bacterium, *Janibacter* sp. TYM3221, is able to grow on biphenyl and degrades 1,1-dichloro-2,2-bis(4-chlorophenyl)ethylene (DDE) via a meta-ring cleavage pathway. The *bphAa* gene, encoding a biphenyl dioxygenase large subunit, was previously demonstrated to be involved in the degradation of DDE in TYM3221. The *bph* gene cluster, containing *orf2* and *bphDAAAbAcAdBCST* was cloned and characterized. Reverse transcription-PCR (RT-PCR) analysis indicated that these genes were transcribed as an operon. The real-time RT-PCR on *orf2*, *bphAa*, *bphC*, and *bphS* suggest the presence of the inducible *orf2* promoter (*orf2p*) and constitutive *bphAa* promoter (*bphAap*). The TYM3221 *bphST* conducted biphenyl-dependent inducible activation plus constitutive basal activation of *orf2p* and constitutive activation of *bphAap* in a *rhodococcal* host strain, *Rhodococcus erythropolis* IAM1399, suggesting that expression of the TYM3221 *bph* operon depends on the *bphST*-coded two-component regulatory system. Both of these promoters were also induced by the *bphS1T1* of a biphenyl degrader, *Rhodococcus jostii* RHA1, and contained the 24-bp consensus sequences of RHA1 *bphS1T1*-dependent promoters. The replacement of RHA1 *bphS1* with TYM3221 *bphS* in combination with RHA1 *bphT1* suggests that TYM3221 *bphS* is responsible for low inducible and high constitutive activation of *orf2p* in IAM1399 by the TYM3221 *bphST*-system. Expression of *bphAaAbAcAdBC* in IAM1399 resulted in the transformation of DDE to the meta-ring cleavage product via 2,3-hydroxylation, suggesting that these genes are involved in DDE degradation.

© 2013, The Society for Biotechnology, Japan. All rights reserved.

[Key words: *Janibacter*; Biphenyl; DDE degradation genes; Biphenyl dioxygenase; Two-component system]

---

1,1-Dichloro-2,2-bis(4-chlorophenyl)ethylene (DDE), a primary decomposition product of 1,1,1-trichloro-2,2-bis(4-chlorophenyl)ethane (DDT), has been reported to be more persistent in the environment than DDT and has caused serious environmental problems worldwide (1). Study of the bacterial degradation system of DDE is useful for the development of effective treatment technologies for DDE contamination. Aerobic DDE degradation has been reported in biphenyl-utilizing bacteria, *Pseudomonas acidovorans* M3GY (2) and *Terrabacter* sp. DDE-1 (3). Both the strains degraded DDE only when they were grown on biphenyl. Based on the intermediate degradation products DDE degradation pathway of these strains was estimated as follows. As illustrated in Fig. 1A, DDE is predicted to be initially attacked by a hydroxylating dioxygenase preferentially at 2,3-position to form 3DCDD, followed by conversion to 3DCBD by a dehydrogenase. This product is then converted to DCPA via TCOA by a meta-ring cleavage dioxygenase and a hydrolase. DCPA is thought to be subsequently degraded to 4CB and/or 4CPA. However, the genes involved in aerobic DDE degradation have not been identified.

In our previous study, a gram-positive DDE degrader, *Janibacter* sp. TYM3221, was isolated from DDE contaminated soil using biphenyl as the sole source of carbon and DDE degradation by TYM3221 was characterized as follows (4). TYM3221 cells grown on LB medium without biphenyl exhibited constitutive DDE degradation activity that is enhanced by the addition of biphenyl. When TYM3221 cells were incubated with DDE, the 3,4-dihydroxylated product, the meta-ring cleavage product of the 2,3-dihydroxylated product, DCPA, 4CPA, and 4CB were detected (4), suggesting that DDE is initially hydroxylated at both the 2,3- and 3,4-positions, but only the 2,3-dihydroxylated product is further degraded to 4CPA and 4CB through the meta-ring cleavage pathway (Fig. 1A). A 3.4-kb DNA region was cloned and contained the entire *bphAaAbAc* genes and the part of *bphAd*, which encode the large, small, ferredoxin, and ferredoxin reductase subunits of a biphenyl dioxygenase, respectively. The gene inactivation of *bphAa* resulted in the loss of the ability to degrade DDE and to utilize biphenyl, indicating that *bphAa*-coded biphenyl dioxygenase is involved in the degradation of biphenyl and DDE in the TYM3221 strain (4).

Because little is known about the genes involved in aerobic DDE degradation, the detailed study on the DDE degradation genes is expected. In this study, the enzyme and regulatory genes responsible for the initial steps of DDE degradation in TYM3221 were

* Corresponding author. Tel.: +81 258 47 9405; fax: +81 258 47 9450.  
E-mail address: masao@vos.nagaokaut.ac.jp (M. Fukuda).

1389-1723/$ — see front matter © 2013, The Society for Biotechnology, Japan. All rights reserved.  
http://dx.doi.org/10.1016/j.jbiosc.2013.01.007

THI NGUYEN ET AL.
J. BIOSCI. BIOENG.,

A

![Chemical Structure Diagram]

B

1.0 kb  
SacI  
|  
1.3-kb probe  
SacI SacI  
EcoRI  
SacI  
SacI  
SacI  
SacI  
EcoRI  
SacI  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|  
|
TABLE 1. Strains and plasmids used in this study.

| Strain or plasmid | Relevant characteristic(s) $^{a}$ | Source or reference |
|-------------------|----------------------------------|---------------------|
| Strain           |                                  |                     |
| Janibacter sp.   | DDE degrader, BPH$^{+}$          | This study          |
| TYM 3221         |                                  |                     |
| Rhodococcus      | Wild type, BPH$^{-}$             | IAM culture         |
| erythropolis     |                                  | collection$^{b}$    |
| IAM 1399         |                                  |                     |
| (=ATCC 15963)    |                                  |                     |
| E. coli DH5      | supE44 hsdR17 recA1 endA1 gyrA96 thi-1 relA1 | 5 |                     |
| Plasmid          |                                  |                     |
| Charomid 9-36    | Ap$^{r}$, cos                    | 6                   |
| pUC18            | Cloning vector, Ap$^{r}$         | 7                   |
| pBluescript      | Cloning vector, Ap$^{r}$         | 8                   |
| II KS(+)         |                                  |                     |
| pBlueKpn::Xba    | pBluescript II KS(+) with new Xbal site in place of KpnI site | 9 |                     |
| pBSL             | pBluescript II KS(+) with new Spel site in place of KpnI site | 10 |                     |
| pKLA1            | Promoter probe vector containing luciferase structural gene, luxAB from V.harveyi, Km$^{r}$(neo) | 11 |                     |
| PK4              | Rhodococcus-E.coli shuttle vector, Km$^{r}$ | 12 |                     |
| PK4XS            | pK4 with new Spel site in place of Xbal site | This study |                     |
| pFAJ 2574        | Rhodococcus-E.coli shuttle vector, Cm$^{r}$ | 13 |                     |
| PCS1             | Charomid 9-36 carrying 2769-bp SacI fragment | 4 |                     |
| PCS2             | Charomid 9-36 carrying 670-bp SacI fragment | 4 |                     |
| PCS3             | Charomid 9-36 carrying 2719-bp SacI fragment | This study |                     |
| PCS4             | Charomid 9-36 carrying 4154-bp SacI fragment | This study |                     |
| PCE1             | Charomid 9-36 carrying 7843-bp EcoRI fragment | This study |                     |
| PUS1             | pUC 18 with 2769-bp SacI fragment of pCS1 | 4 |                     |
| PBS2             | KS+ with 670-bp SacI fragment of pCS2 | 4 |                     |
| PBS3             | KS+ with 2719-bp SacI fragment of pCS3 | This study |                     |
| PBS4             | KS+ with 4154-bp SacI fragment of pCS4 | This study |                     |
| PUE1             | pUC 18 with 7843-bp EcoRI fragment of pCE1 | This study |                     |
| PBST             | pBlueKpn::Xba with 6.6-kb fragment including 5.6-kb EcoRV-EcoRI fragment of pUE1 and 1.0-kb EcoRI-HincII fragment of pBS4, carrying TYM3221bphST | This study |                     |
| PBS              | pBlueKpn::Xba with 5.9-kb BglII fragment of pBST carrying TYM3221bphS | This study |                     |
| PBAB62           | pBSL with 6.2-kb fragment carrying RHA1bphS1T1 | 10 |                     |
| PBSR             | pBlueKpn::Xba with 5.0-kb fragment including 3.5-kb SpeI-PstI and 1.5-kb PstI-BamHI fragments of pBAB62, carrying RHA1bphS1 | This study |                     |
| PKXT1            | pK4XS with 735-bp EcoRI-Spel PCR fragment carrying RHA1bphT1 | This study |                     |
| PKLF2            | pKLA1 with 627-bp HincII fragment of pBS3 carrying promoter region of orf2 | This study |                     |
| pKLFAa           | pKLA1 with 670-bp SacI fragment of pBS2 carrying promoter region of bphAa | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |
| pKLFAaST         | pKLFAaST with 6.6-kb Xbal fragment of pBST carrying TYM3221bphST | This study |                     |

$^{a}$ BPH$^{+}$, growth on biphenyl; BPH$^{-}$, no growth on biphenyl; Ap$^{r}$, ampicillin resistance; Km$^{r}$, kanamycin resistance; Cm$^{r}$, chloramphenicol resistance.
$^{b}$ IAM, Institute of Applied Microbiology.

DNA manipulation and nucleotide sequencing techniques, including total DNA isolation, construction of deletion derivatives, and nucleotide sequencing, were performed as described previously (14). Nucleotide sequences were analyzed using MacVector software (MacVector, Inc., USA). Homology searches were performed using the NCBI database along with the BLAST program (http://blast.ncbi.nlm.nih.gov). Pairwise alignments were performed using the EMBOSS alignment tool (http://www.ebi.ac.uk/Tools/emboss/align).

Reverse transcription-PCR (RT-PCR) and real-time RT-PCR TYM3221 cells were grown at 30°C in LB medium to an OD$_{600}$ of 0.5 and were further incubated for 8h at 30°C in the presence or absence of 1 mM biphenyl (Wako Pure Chemical Industries, Japan). Total RNA was extracted from 10 ml culture as described previously (15). cDNA was synthesized from 2 μg of total RNA using ReverTra Ace (Toyobo Co., Ltd, Japan) with random hexamers (Takara Bio Inc., Japan) in a 20-μl reaction mixture. The PCR reaction was performed in a 25-μl reaction mixture using 1 unit of Ex-Taq polymerase (Takara), 1 μl of synthesized cDNA as the template, 2 μl of 2.5 mM dNTP mixture, and 0.8 μM of the corresponding specific primer pair listed in Table 2. The PCR conditions were as follows: 27 cycles of 96°C for 30s, 58°C for 30s, and 72°C for 90s.

Real-time PCR was performed using the StepOne™ instrument (Applied Biosystems, Inc., USA). The primer pairs were designed using Primer Express Software, Version 3 (Applied Biosystems) and are listed in Table 2. Normalized Quantification of each gene sequence was performed according to the procedure provided by Applied Biosystems. The PCR-amplified template DNA for each primer set was subjected to 10-fold serial dilutions and the resulting diluted samples were

TABLE 2. Primers used in this study.

| Primer name | Target region or gene | Sequence (5′–3′) |
|-------------|----------------------|------------------|
| RT-PCR      |                      |                  |
| Orf1F       | 1                    | GCAAGCCGCCGAGCAAAG |
| Orf2R       | 1                    | AATGGTCGAAGCCGATGCT |
| FF2         | 2                    | TTGTAGGACGCTCGCCCCTC |
| RbphD       | 2                    | TCGCCTCGTAGGACGGTTG |
| FRTbphD     | 3                    | GGTGGTCCCGTTGGAGGTG |
| RbphAa      | 3                    | ACTTCGGCTCGCCCCAGATA C |
| FbphAa      | 4                    | GGATGATGACCTCCGCTGACTG |
| RbphAd      | 4                    | CGACCGACGACAACCTCCAC |
| FbphAd      | 5                    | AGCCCGCCGTCACTTTCAG |
| RbphC       | 5                    | ATCGCTCCCAAACCTGCTGG |
| FbphC       | 6                    | CCTACGACAGGTGTGTGGACG |
| RbphS       | 6                    | TTGGTGAGCCGAGCCCCTC |
| FbphS       | 7                    | GACAACGGCAGCGGCTTC |
| RbphT       | 7                    | AGGCACTCCTTGGTGATGTATCC |
| Real-time RT-PCR |                   |                  |
| TYM16S      | 16SrRNA              | GGAGCTTGCCTCGAGTGGAT |
| TYM16R      | 16SrRNA              | TCCAGTATTAGACGCGTTTCC |
| Orf2F       | orf2                 | TGGCACCTGAACGCTGTGT |
| Orf2R       | orf2                 | AATGGTCGAAGCCGATGCT |
| BphAaF      | bphAa                | GTCTTCCCTGAACCAGTGCCG |
| BphAaR      | bphAa                | TCGTAGGCCCAAACCGTGATA |
| BphCF       | bphC                 | TGCTGGAGACTACGTGCGATGG |
| BphCR       | bphC                 | TGTGGTCGTTGGTGTCGCCCT |
| BphSF       | bphS                 | CATCGAGAGGTTTGATCCGGT |
| BphSR       | bphS                 | ACCAACAAAGCCTCCAATGG |

subjected to real-time PCR using the corresponding primer set to construct the standard curve. The copy number of a target gene in a sample was estimated using this standard curve and was normalized by division by that of TYM322116SrRNA gene. The real-time PCR was conducted in a total reaction volume of 20 μl containing 1 μl of a 5-fold diluted solution of synthesized cDNA, 200 nM primers, and 10 μl of the SYBR Green PCR Master Mix (Applied Biosystems). The reaction conditions of real-time PCR were as follows: 95°C for 10 min, 40 cycles of 95°C for 15 s, and 60°C for 60 s.

Luciferase assay  
The reporter plasmids were constructed using the promoter probe vector pKLA1 containing the luxAB luciferase genes as a reporter. The orf2p and bphAap fragments were inserted into the Sall site of pKLA1 to construct pKLF2 and pKLFAa, respectively. The DNA fragment carrying TYM3221 bphS and bphT was inserted into the Xbal sites of pKLF2 and pKLFAa to obtain pKLF2ST and pKLFAaST, respectively. The DNA fragments containing RHA1bphS1 and TYM3221bphS were inserted in the Xbal site of pKLF2 to construct pKLF2SR and pKLF2S, respectively. The RHA1bphT1 gene fragment was inserted into the Xbal site of pFAJ2574 to construct bphT1 expression plasmid pFTCm. The resulting plasmids were introduced into R. erythropolis IAM1399 via electroporation (16) and the transformants were grown at 30°C in LB containing the appropriate antibiotics. The cells were washed, suspended in 10 ml of 1/5 LB supplemented with the appropriate antibiotics at an OD600 of 0.5, and incubated at 30°C for 5 h in the absence or presence of either 1 mM biphenyl or 200 μM DDE (Wako). Luciferase activity was measured as previously described (17) and was expressed in relative light units (RLU) per unit of OD600.

DDE transformation assay  
The plasmids pK4A, pk4AB, pK4ABC, and pK4ABCD were constructed as described in Table 1 and were introduced via electroporation into R. erythropolis IAM1399 harboring plasmid pFST1 (16). The resulting transformants were grown at 30°C in 100 ml of 1/5 LB medium containing kanamycin and chloramphenicol to an OD600 of 1.0 and were further incubated with 1 mM of biphenyl for 12 h. The cells were washed twice and suspended in 20 mM sodium phosphate buffer (pH 7.0) to an OD600 of 10.0. The cell suspensions of 5 ml each were incubated with 200 μM of DDE at 30°C. An ethyl acetate extract of each reaction mixture was trimethylsilylated or methylated as described previously (4). Gas chromatography-mass spectrometry (GC-MS) analysis was performed using a model 6890 GC system equipped with a model 5973 mass selective detector and an Ultra-2 capillary column (50 m by 0.2 mm; Agilent Technologies Co., USA). The injector and detector temperatures were 250°C and 305°C, respectively. The initial oven temperature was 100°C. The temperature was sequentially ramped to 200°C at 10°C/min and to 300°C at 20°C/min or 200°C at 20°C/min and to 300°C at 5°C/min, and was held at 300°C for 11.5 min. The UV-visible spectrum absorbance measurements of the supernatants were performed using a DU-800 spectrophotometer (Beckman Coulter). The samples were scanned over a wavelength range of 300 nm–600 nm.

Nucleotide sequence accession number  
The nucleotide sequence of 15,244 bp reported in this study has been deposited in the DDBJ, EMBL, and GenBank nucleotide sequence databases under the accession number AB733643.

RESULTS

Cloning and nucleotide sequences of bph gene cluster  
The entire 15,244-bp DNA region includes the inserts of the pCS3, pCS2, pCS1, pCE1, and pCS4 plasmids, as illustrated in Fig. 1B. The DNA sequence analysis of the 15,244-bp DNA region revealed 11 open reading frames (ORFs), and a partial ORF, ORF1 (Fig. 1). The nucleotide sequence of ORF1 showed 92% and 66% amino acid sequence identity to the putative integrase/transposase catalytic region from Nocardioides sp. JS614 and the IstA transposase from Rhodococcus erythropolis N186/21, respectively (Table 3). IstA has been reported as an integrase/transposase core domain of the transposable element IS1415 that was found near the thrRBCD responsible for the biodegradation of thiocarbamate and s-triazine herbicides in R. erythropolis N186/21 (18). The putative amino acid sequence of ORF2 shared identities with the putative Enoyl-CoA hydratase/isomerases encoded by ROP-pROB02-01700 from Rhodococcus opacus B4 as well as ro-10119 and ro-08049 from Rhodococcus jostii RHA1. The next 9 ORFs, whose gene products were similar to R. jostii RHA1 proteins involved in

TABLE 3. Identification of bph genes functions.

| Gene   | No. of amino acids | Representative ortholog(s)                                                                 | Identity (%)a | Accession no. |
|--------|--------------------|-------------------------------------------------------------------------------------------|---------------|----------------|
| orf1   | 275b               | Integrase, catalytic region from Nocardioides sp. JS614                                      | 92            | ABL82259       |
|        |                    | Transposase (IstA) from Rhodococcus erythropolis NI86/21                                     | 66            | AAC45485       |
| orf2   | 195                | Enoyl-CoA hydratase/isomerase family protein from R. opacus B4                                | 44            | BAH47177       |
|        |                    | Probable enoyl-CoA hydratase from R. jostii RHA1                                             | 43            | ABH00312       |
| bphD   | 281                | 2,6-Dioxo-6-phenylhexa-3-enoate hydrolase from Mycobacterium rhodesiae JS60                     | 64            | EHB59054       |
|        |                    | 2-Hydroxy-6-oxohepta-2,4-dienoate hydratase (EtbD1) from R. jostii RHA1                        | 52            | ABG99091       |
| bphAd  | 412                | Ferredoxin reductase (BpdA) from Rhodococcus sp. M5                                           | 64            | AAB07753       |
|        |                    | Biphenyl 2,3-dioxygenase, reductase (BphAd) from R. jostii RHA1                               | 64            | ABG99104       |
| bphB   | 280                | 2,3-Dihydroxy-4-phenylhexa-4,6-diene dehydrogenase (BpdD) from Rhodococcus sp. M5               | 80            | AAB07754       |
|        |                    | Cis-2,3-dihydrobiphenyl-2,3-diol dehydrogenase (BphB) from R. jostii RHA1                     | 46            | ABG99101       |
| bphC   | 295                | 2,3-Dihydroxybiphenyl 1,2-dioxygenase I (BpdE) from Rhodococcus sp. M5                         | 78            | AAB07755       |
|        |                    | 2,3-Dihydroxybiphenyl 1,2-dioxygenase I (BphC1) from R. jostii RHA1                           | 67            | ABG99102       |
| bphS   | 1587               | Sensor kinase, two-component system (BphS1) from R. jostii RHA1                                | 62            | ABG99099       |
|        |                    | Receptor-like histidine kinase (BpdS) from Rhodococcus sp. M5                                 | 60            | AAB52543       |
| bphT   | 209                | Response regulator (BphT) from R. erythropolis TA421                                           | 71            | BAA34262       |
|        |                    | Response regulator (BpdT) from Rhodococcus sp. M5                                            | 70            | AAB52544       |
| orf3   | 203                | Response regulator two-component system (BphT1) from R. jostii RHA1                           | 69            | ABG99098       |
|        |                    | Hypothetical protein TBFG_12051 from Mycobacterium tuberculosis F11                          | 36            | ABR06384       |

a Percent identity obtained by aligning the deduced amino acid sequences by use of the EMBOSS alignment tool.  
b Partial amino acid sequence.

biphenyl degradation, were designated as *bphD*, *bphAa*, *bphAb*, *bphAc*, *bphAd*, *bphB*, *bphC*, *bphS*, and *bphT*, which encoded a 2-hydroxy-6-oxo-6-phenylhexa-2,4-dienoate hydrolase, the large subunit, the small subunit, the ferredoxin subunit, and the ferredoxin reductase subunit of a biphenyl dioxygenase, a 2,3-dihydroxy-4-phenylhexa-4,6-diene dehydrogenase, a 2,3-dihydroxybiphenyl 1,2-dioxygenase, and the sensor kinase and response regulator of a two-component system regulator, respectively. ORF3 encoded a protein of unknown function.

**Transcriptional analysis of *bph* pathway in TYM3221**

To investigate the transcriptional organization of the *bph* genes, reverse transcription-PCR (RT-PCR) analysis was performed using the total RNA extracted from TYM3221 cells grown in LB in the presence or absence of biphenyl. The RT-PCR amplification of regions 1–7, indicated in Fig. 1B, was performed using the primer pairs listed in Table 2. No RT-PCR product was detected for region 1, representing the intergenic region between *orf1* and *orf2*. RT-PCR products of expected sizes were observed for regions 2–7, including the intergenic gene sequences of *orf2*, *bphD*, *bphAa*, *bphAb*, *bphAc*, *bphAd*, *bphB*, *bphC*, *bphS*, and *bphT*, independent of the presence of biphenyl (Fig. 1C). These results indicate that the genes from *orf2* to *bphT* are co-transcribed as an operon in TYM3221, and we designated this operon as the *bph* operon. The RT-PCR products of the cells grown in the presence of biphenyl appeared to be denser than those of the cells grown in the absence of biphenyl. To confirm biphenyl-dependent upregulated transcription of the *bph* operon, real-time RT-PCR was performed using primer pairs for the intragenic sequences of *orf2*, *bphAa*, *bphC*, and *bphS* as indicated in Fig. 1B. The transcription levels of *orf2*, *bphAa*, *bphC*, and *bphS* of cells grown in the presence of biphenyl were 13.1-, 2.1-, 1.7-, and 2.7-fold higher than those of the cells grown in the absence of biphenyl, respectively (Fig. 2). These results represent the inducible transcription of *orf2* and the basal plus inducible transcription of *bphAa*, *bphC*, and *bphS*, suggesting that the inducible and constitutive promoters are located adjacent upstream from *orf2* and *bphAa*, respectively. In addition, the transcription level in the cells grown in the absence of biphenyl of *bphC* was about twice as high as that of *bphAa*. This result may suggest the presence of an additional constitutive promoter in adjacent upstream from *bphC*.

![Figure 2](https://i.imgur.com/placeholder.png)

FIG. 2. Transcription levels of the *orf2*, *bphAa*, *bphC*, and *bphS* genes in TYM3221. Locations of the amplified regions and sequences of the primer sets used are presented in Fig. 1B and Table 2. Total RNAs were extracted from TYM3221 cells grown in LB medium in the absence (open bars) or presence (filled bars) of biphenyl. The transcription level of each gene was normalized to that of the 16S rRNA. The data are the means of three independent experiments, and the error bars represent 1 standard deviation.

Characterization of the *bph* operon regulatory system

The *bphS* and *bphT* genes coding for sensor kinase BphS and response regulator BphT, respectively, were located downstream from *bphC*. BphS of TYM3221 contains the sequence motifs, H, N, D, and G boxes conserved among sensor histidine kinases as is the case with that of RHA1 (10). The H box contains the site of autophosphorylation, the N, D, and G boxes serve the ATP binding and catalytic site (19,20). The residue His1397 of TYM3221 in the H box was predicted to be responsible for autophosphorylation of BphS. BphT of TYM3221 contains Asp9, Asp54, and Lys104, which are highly conserved among response regulators. These residues are proposed to be required for phosphorylation of the response regulator (19,21). Because the presences of inducible and constitutive promoters are suggested to be adjacent upstream from *orf2* and *bphAa*, respectively, sequence analyses of these upstream regions were performed. As illustrated in Fig. 3A, the sequence analysis revealed the presence of the 24-bp consensus sequence in both the upstream regions, which are conserved among the biphenyl/ethylbenzene-inducible BphST-dependent promoters of *R. jostii* RHA1 (22). The 24-bp consensus sequence is shared among all of the biphenyl-inducible promoters in the strain RHA1, and the replacement of the core region of benzoate dioxygenase gene promoter (*benAp*) with the 24-bp consensus sequence resulted in biphenyl/ethylbenzene-inducible BphST-dependent transcription from the replaced promoter, suggesting that the 24-bp consensus sequence is enough for biphenyl/ethylbenzene-inducible BphST-dependent transcription. Therefore, we designated the upstream regions from *orf2* and *bphAa* containing the 24-bp consensus sequence as *orf2p* and *bphAap*, respectively. Because the existence of the 24-bp consensus sequence suggests the involvement of *bphS* and *bphT* in the transcriptional induction of *orf2*, *bphAa*, and *bphC* by biphenyl, promoter assay experiments in the presence of *bphST* were performed in a host strain *R. erythropolis* IAM1399. The *orf2p* and *bphAap* fragments were inserted in a *luxAB* reporter plasmid pKLA1 to construct pKLF2 and pKLFAa, respectively. IAM1399 cells containing either pKLF2 or pKLFAa showed little luciferase activity originating from *orf2p* and *bphAap*, similar to that of IAM1399 harboring pKLA1 (Fig. 3B). When TYM3221 *bphST* was inserted in pKLF2 to construct pKLF2ST, IAM1399 containing pKLF2ST exhibited strong basal luciferase activity in the absence of biphenyl and induced elevated luciferase activity in the presence of biphenyl. When TYM3221 *bphST* was inserted in pKLFAa to construct pKLFAaST, IAM1399 containing pKLFAaST exhibited strong basal luciferase activity in the absence of biphenyl and no induced elevated luciferase activity in the presence of biphenyl. These results suggested that TYM3221 BphST is essential for the basal promoter activities of both *orf2p* and *bphAap* and that TYM3221 BphST conducts biphenyl-dependent transcriptional induction of *orf2p* but not that of *bphAap*. Because biphenyl is structurally similar to DDE, induction activity of DDE was examined (Fig. 3C). IAM1399 cells containing either pKLF2ST or pKLFAaST did not exhibit an elevation of luciferase activity in the presence of DDE, suggesting DDE is not the inducing substrate of TYM3221 BphST.

Based on the above results, TYM3221 BphST appears to be deficient in biphenyl-dependent transcriptional induction. To address the cause of this induction deficiency, RHA1 BphS1T1 was employed in place of TYM3221 BphST (Fig. 3D). The pFST1 plasmid containing RHA1 *bphS1T1* was introduced into IAM1399 harboring either pKLF2 or pKLFAa, and the resulting recombinants were subjected to luciferase assays. In contrast with the results of TYM3221 BphST, obvious biphenyl-dependent transcriptional induction was observed with both recombinants harboring pKLF2 and pKLFAa. These results indicate that both the 24-bp consensus sequences in *orf2p* and *bphAap* works well in biphenyl-dependent

THI NGUYEN ET AL.

A Consensus sequences

TYM3221-orf2 CCCATCCAC TTTCCGTAATT TTGCCGGATGTTTCAGGGAGGTCA CAGCCCCTTCTACTGAGGCAG
TYM3221-bphAa GTGTCCGACTTCCGTAGTTTTCCGGATGCCGT CCTGTCGGAGCGCCGGCACGCTTACCCCCAC
RHA1-bphAa ACCGGCGGGTTCCGTAGTTTTCCCCGGATGTT CGCCGATGAGATCTGGATC ACTATGGATGCCA
RHA1-etbAa1 GCCGCTGTTTTCAGTAGTTTCTCCGGATGTAG CAAGTG TGGAACCAACTGTTTACTAGGGGAAC
RHA1-etbAa2 ACCACTATTTTCCGTAGTTTCCCCGGATGCC GGAGAGCGTGGGCCAGTTGTTTAGTAGGAGCA

B

| pKLA1 | Rate of Induction |
| --- | --- |
|  | Biphenyl - |
|  | Biphenyl + |
| pKLF2 |  |
| pKLFAa |  |
| pKLFAaST |  |
| pKLFAaST |  |

0 0.5 1.0 1.5

Luciferase activity (RLU/A<sub>600</sub> × 10<sup>3</sup>)

C

| pKLF2ST | DDE - |
| --- | --- |
| pKLFAaST | DDE + |


0 100 200 300 400

Luciferase activity (RLU/A<sub>600</sub> × 10<sup>3</sup>)

D

| pKLF2+pFST1 | Biphenyl - |
| --- | --- |
| pKLFAa+pFST1 | Biphenyl + |


0 200 400 600 800 1000

Luciferase activity (RLU/A<sub>600</sub> × 10<sup>3</sup>)

E

| pKLF2 | Biphenyl - |
| --- | --- |
| pKLF2S | Biphenyl + |
| pKLF2SR |  |
| pKLF2+pFTCm |  |
| pKLF2S+pFTCm |  |
| pKLF2SR+pFTCm |  |


0 500 1000 1500 2000

Luciferase activity (RLU/A<sub>600</sub> × 10<sup>3</sup>)

FIG. 3. Transcriptional activation of orf2p and bphAap. (A) Multiple sequence alignment of the regions upstream from orf2 and bphAa in TYM3221 and bphAa, etbAa1, and etbAa2 of RHA1 containing the 24-bp consensus sequence. A shaded box indicates the 24-bp consensus sequence. Boxed nucleotides indicate the transcriptional start sites. (B, C) Effects of biphenyl (B) and DDE (C) on the transcriptional activation of orf2p and bphAap by TYM3221 BphST. The results of pKLA1, pKLF2, and pKLFAa are visualized at magnification of 100 in the inset of the panel B. Plasmids pKLA1, a promoter probe vector; pKLF2, pKLA1 with the insert of orf2p; pKLFAa, pKLA1 with the insert of bphAap; pKLFAaST, pKLFAa with the insert of TYM3221 bphST; pKLFAaST, pKLFAa with the insert of TYM3221 bphST. (D) Effect of biphenyl on the transcriptional activation of orf2p and bphAap by RHA1 BphS1T1. The pFST1 plasmid contained RHA1 bphS1T1. (E) Transcriptional activation of orf2p by the combination of TYM3221 BphS and RHA1 BphT1. The pFTCm plasmid contained RHA1 bphT1. The pKLF2S and pKLF2SR plasmids are pKLF2 derivatives with inserts of TYM3221 bphS and RHA1 bphS1, respectively. IAM1399 cells containing the corresponding plasmids were incubated for 5 h in the 1/5 LB medium in the absence (open bars) or presence (filled bars) of the substrate and were subjected to luciferase assays. The data are the means of three independent experiments, and the error bars represent 1 standard deviation.

transcriptional induction of orf2p and bphAap by RHA1 BphS1T1. In contrast, it is suggested that TYM3221 BphST works poorly in biphenyl-dependent transcriptional induction of orf2p and bphAap in IAM1399 through their 24-bp consensus sequences, although it has strong activity in basal transcriptional activation of orf2p and bphAap.

To address the origin of the poor induction properties of TYM3221 BphST, a heterologous two-component system composed of TYM3221 bphS and RHA1 bphT1 was constructed in IAM1399 using pKLF2S and pFTCm containing TYM3221 bphS and RHA1 bphT1, respectively. The IAM1399 harboring pKLF2S and

pFTCm exhibited strong basal luciferase activity in the absence of biphenyl and weak elevation of luciferase activity in the presence of biphenyl (Fig. 3E). In contrast, IAM1399 harboring pKLF2SR and pFTCm, which contained RHA1 bphS1 and bphT1, respectively, showed little luciferase activity in the absence of biphenyl and strong elevation of luciferase activity in the presence of biphenyl. These results suggest that TYM3221 BphS is mainly responsible for high basal transcriptional activation and low biphenyl-dependent induction. However, there remains a possibility that TYM3221 BphT itself is also involved in the high basal transcriptional activation.

Expression of BphA, BphB, BphC, and BphD in R. erythropolis IAM1399

To elucidate the involvement of TYM3221 *bph* genes in DDE degradation, the *bphA*, *bphAB*, *bphABC*, and *bphABCD* genes were independently expressed in *R*. *erythropolis* IAM1399, which does not degrade DDE. Because the transcription from TYM3221 *bphAap* was strongly induced by biphenyl under the control of RHA1 BphS1T1 as described above, IAM1399 harboring a RHA1 BphS1T1-expression plasmid, pFST1 was used as a host strain to express the TYM3221 *bph* genes. The DNA fragments separately carrying the TYM3221 *bphAap-bphAaAbAcAd*, *bphAap-bphAaAbA cAdbphB*, *bphAap-bphAaAbAcAdbphBbphC*, and *bphAap-bphAaAbA cAdbphBbphCbphD* genes were independently cloned into pK4 to construct the plasmids, pK4A, pK4AB, pK4ABC, and pK4ABCD respectively. These plasmids were individually introduced via electroporation into the strain IAM1399 harboring pFST1, and the resulting recombinant strains grown in 1/5 LB in the presence of

biphenyl were incubated with 200 μM of DDE at 30°C. Each culture was extracted with ethyl acetate and subjected to GC–MS analysis after either trimethylsilylation or methylation.

When IAM1399 harboring pK4A and pFST1 was incubated with DDE, GC–MS analyses of the trimethylsilylated extracts after 8 h of incubation revealed the presence of major and minor products at the retention times of 22.1 and 23.6 min, respectively (Fig. 4A and C). The mass spectrum of the major product at the retention time of 22.1 min was almost identical to that of the trimethylsilylated derivative of 4DBD, which was illustrated in Fig. 1A and reported previously (4). The minor product at the retention time of 23.6 min was identified as the trimethylsilylated derivative of 3DCBD based on the mass spectrum (Fig. 4E). The molecular ion at *m/z* 494 represents the isotopic signature of tetrachloride. The major fragment ions were indicative of the loss of one trimethylsilyl group and one methyl group (*m/z*, 406); one trimethylsilyl group and

FIG. 4. Transformation of DDE by IAM1399 cells containing the TYM3221 *bphAaAbAcAd* genes. (A–D) Total ion chromatograms of trimethylsilylated (A, C) and methylated (B, D) culture extracts at the start (A, B) and at 8 h (C, D) of incubation. IAM1399 cells harboring pK4A containing *bphAaAbAcAd* were incubated in 5 ml of 20 mM sodium phosphate buffer in the presence of 200 μM of DDE. The cultures were extracted with ethyl acetate, trimethylsilylated or methylated and subjected to GC–MS. The peaks of DDE and its metabolites are indicated by vertical arrows. (E–G) Mass spectra of the trimethylsilylated (E) and methylated derivative (G) of 3DCBD and 5DCBO (F). The *m/z* values of the peaks, which are the highest in each cluster of peaks, are indicated above the corresponding peaks. (H) The possible conversion of the 3,4-dihydroxylated product of DDE, 4DCDD to trimethylsilylated derivative of 4DBD and 5DCBO during trimethylsilylation and methylation, respectively.

chloride ion (m/z, 386); one trimethylsilyl group, one methyl group, and one chloride ion (m/z, 369); and one methyl group, one trimethylsilyl group and two chloride ions (m/z, 334).

GC–MS analysis of methylated extract indicated the presence of major and minor products at the retention times of 24.7 and 25.2 min, respectively (Fig. 4B and D). Although the mass spectrum of the major product at the retention time of 24.7 min was almost identical to that of 4-[2,2-dichloro-1-(4-chlorophenyl)vinyl]-1-chloro-hydroxybenzene, which had been reported previously by Hay and Focht (2), it did not agree with the methylated derivative containing a methoxy group. As this product was not methylated, we estimate that this product is 5-[2,2-dichloro-1-(4-chlorophenyl) vinyl]-2-chloro-1a,5a-dihydro-1aH,5aH-benzo[1,2-b]oxirane (5DCBO), containing an epoxy group (Fig. 4F). Although such an epoxy compound might be the reaction product of a biphenyl dioxygenase, we estimate that 5DCBO was formed from the possible 3,4-dihydroxylated product of DDE, 4DCDD during methylation (Fig. 4H). Because 5DCBO is thought not to be trimethylsilylated and the major product after trimethylsilylation was the trimethylsilylated derivative of 4DBD, which can be formed from 4DCDD by dehydrochlorination during trimethylsilylation.

The minor product at the retention time of 25.2 min was identified as the methylated derivative of 3DCBD based on the mass spectrum (Fig. 4G). The molecular ion at m/z 378 represents the isotopic signature of tetrachloride. The major fragment ion was indicative of the loss of one methyl group and one chloride ion (m/z, 328). The intermediate product 3DCBD appears to be formed from the possible 2,3-dihydroxylated product of DDE, 3DCDD during trimethylsilylation or methylation. These results suggest that TYM3221 BphA catalyzes hydroxylation of DDE at both the 2,3- and 3,4-positions to yield two dihydrodiol products, 3DCDD and 4DCDD, respectively, and that TYM3221 BphA predominantly catalyzes the 3,4-hydroxylation.

GC–MS analyses of the trimethylsilylated extracts of IAM1399 harboring pK4AB and pFST1 after 8 h of incubation with DDE indicate the increase of trimethylsilylated derivative of 3DCBD as compared with those of IAM1399 harboring pK4A and pFST1 (data not shown). These results suggest that the TYM3221 BphB is active on the conversion of 2,3-dihydroxylated intermediate to 3DCBD.

When IAM1399 harboring pK4ABC and pFST1 was incubated with DDE, accumulation of a yellow-colored product with an absorbance maximum at 404 nm was observed (Fig. 5A). GC–MS analysis of the methylated extract after 24 h of incubation revealed the presence of TCOA at a retention time of 25.5 min, which was estimated to be the meta-ring cleavage product of 3DCBD (Fig. 5B). These results suggest that the *bphC* gene in pK4ABC is involved in conversion of 3DCBD to the meta-ring cleavage product. When IAM1399 containing *bphAaAbA-cAdbphBbphCbphD* was incubated with DDE, the meta-ring cleavage product of 3DCBD accumulated and no further transformation was observed, although it transformed the meta-ring cleavage product of 2,3-dihydroxybiphenyl produced from biphenyl (data not shown).

## DISCUSSION

In this study, the *orf2bphDAaAbAcAdBCST* gene cluster from Janibacter sp. TYM3221 has been cloned and characterized. The products of TYM3221 *bphAaAbAcAdBCST* are similar to those of the *bph* genes and *bpd* genes in the biphenyl/PCB degraders, RHA1 (10,16) and Rhodococcus sp. M5 (23,24), respectively. TYM3221 *bph* genes appear to be derived from the same ancestral origin as Rhodococcus PCB degraders. Among these three strains, the RHA1 *bphAaAbAcAdC1BS1T1* gene cluster is uniquely organized; the *bphB*, 2,3-dihydrobiphenyl-2,3-diol dehydrogenase gene is located downstream from the *bphC1*, 2,3-dihydroxybiphenyl 1,2-

VOL. 116, 2013

![Diagram](#)

FIG. 6. A proposed mechanism for the transcriptional regulation of *bph* genes in TYM3221. The *bph* genes are co-transcribed from the biphenyl-inducible *orf2p* and the constitutive *bphAap*, which are both regulated by the two-component system, BphST. Stronger transcriptional induction from *orf2p* than the expected BphST-dependent induction suggest the involvement of an additional unknown regulatory system. Partial termination of transcription within the *bphS* gene was indicated by the remarkable reduction of transcription inside *bphS*. A stem-loop structure was found in the N-terminal region of *bphS*. Regulatory proteins and their activations are indicated by gray ellipses and dotted arrows, respectively. The angled arrows represent the start and extent of transcription, and their thicknesses indicate the strength of transcription. Horizontal open arrowheads and a stem-loop specify the validated or putative promoters and the possible stem-loop-forming sequence. Genes in the genetic map at the bottom are distributed corresponding to the starts of transcription and stem-loop illustrated above.

from the *bphS* initiation codon. However, it was not followed by the T-rich sequence characteristic of a ρ-independent terminator. It may therefore conduct an incomplete termination of transcription originating from the upstream promoters and allow read-through transcription of *bphST* (Fig. 6).

Two-component systems responsible for the transcriptional induction of degradation enzymes have been characterized in the biphenyl/PCB pathway of *Rhodococcus* sp. M5 (24) and *R. jostii* RHA1 (10,22,25,26), the o-xylene pathway of *Rhodococcus* sp. DK17 (27), and the tetralin pathway of *Rhodococcus* sp. TFB (28). Homologs of *bphST* with a high sequence similarity were found in the biphenyl/PCB degradation gene cluster of *Rhodococcus erythropolis* TA421 (29), the isopropylbenzene degradation gene cluster of *Rhodococcus erythropolis* BD2 (30), and the benzene degradation gene cluster of *R. opacus* B4 (26,31), suggesting the wide distribution of this type of two-component system among the aromatic compound degradation genes of *Rhodococcus* genus.

The real-time RT-PCR analysis of TYM3221 revealed biphenyl-dependent transcriptional induction of *orf2* and biphenyl-independent basal transcription of *bphAa*. The level of biphenyl-dependent induced transcription of *bphAa* was approximately equivalent to that of *orf2* that is located upstream from *bphAa*, suggesting biphenyl-dependent transcriptional induction was originated primarily from *orf2p* (Fig. 6). In the IAM1399 strain, TYM3221 BphST conducted high biphenyl-independent basal activation of both *orf2p* and *bphAap* and low biphenyl-dependent transcriptional induction of *orf2p*. No biphenyl-dependent transcriptional induction was observed with *bphAap* in IAM1399 under the control of TYM3221 BphST. These results in IAM1399 support the inducible transcription from *orf2p* and the constitutive transcription from *bphAap* indicated by the real-time RT-PCR. However, the biphenyl-dependent induction ratio of *orf2p* in TYM3221 is apparently higher than that in IAM1399 under the control of TYM3221 BphST and the basal transcription from *orf2p* in IAM1399 is high, suggesting the circumstances in transcriptional activation of *orf2p* by BphST are not the same between TYM3221 and IAM1399. These results may have originated from the presence of an additional unknown regulatory system governing *orf2p* in TYM3221 (Fig. 6). In total, the TYM3221 *bph* operon appears to be unique in that the *bph* gene cluster contains both inducible and constitutive *bphST*-dependent promoters.

IAM1399 harboring TYM3221 *bphAaAbAcAdbphBbphC* transformed DDE to a meta-ring cleavage product. Because the involvement of *bphAa* in DDE degradation in TYM3221 has been indicated by a *bphAa*-inactivated mutant previously (4), our results suggest that TYM3221 *bphAaAbAcAdbphBbphC* is involved in DDE degradation in the strain TYM3221. Although TYM3221 *bphAaAbA-cAd*-coded dioxygenase hydroxylated DDE at both the 2,3- and 3,4-positions to yield 3DCDD and 4DCDD in IAM1399, respectively, IAM1399 containing *bphAaAbAcAdbphBbphC* accumulated the meta-ring cleavage product of 3DCBD, which was produced from 2,3-hydroxylation of DDE. The TYM3221 *bphC*-coded ring-cleavage dioxygenase appears to be specific not to 4DBD but to 3DCBD. We failed to detect the transformation of the meta-ring cleavage product of 3DCBD in IAM1399 containing *bphAaAbAcAdbphBbphCbphD*. Transformation of the meta-ring cleavage product from DDE was also very slow in the TYM3221 strain. More sophisticated examination including a purified enzyme assay will be required to determine the activity of *bphD*-coded hydrolase against the meta-ring cleavage product of 3DCBD.

Although TYM3221 BphA preferentially transformed DDE to the 3,4-dihydrodiol product, 4DCDD, our results suggest the total degradation pathway through the 2,3-dihydrodiol intermediate, 3DCDD. Continuous transformation of 3DCDD and accumulation of 4DCDD appear to compensate for an inefficient production of 3DCDD by TYM3221 BphA. In summary, we demonstrated that the TYM3221 *bphAaAbAcAdBC* genes are involved in DDE degradation and are regulated by the *bphST*-coded two-component regulatory system. Although expression of the *bph* gene cluster depends on the *bphST*-system, it is incompletely inducible. Such an incomplete induction by the two-component system is interesting and may suggest that biphenyl is not a suitable inducing substrate for the TYM3221 *bphST*-system.

### ACKNOWLEDGMENTS

We thank Drs. René De Mot and Robert van der Geize for the kind gifts of plasmid pFAJ2574 and pSelAct, respectively. This work was supported in part by a grant from the Ministry of Agriculture, Forestry and Fisheries of Japan (Genomics for Agricultural Innovation, QTL-GMB0005) and by a grant-in-aid for Scientific Research (B) (22380050) and a Supported Program for the Strategic Research Foundation at Private Universities, 2008–2012 from the Ministry of Education, Culture, Sports, Science, and Technology of Japan.

### References

1. Thomas, J. E., Ou, L. T., and All-Agely, A.: DDE remediation and degradation, Rev. Environ. Contam. Toxicol., **194**, 55–69 (2008).
2. Hay, A. G. and Focht, D. D.: Cometabolism of 1,1-dichloro-2,2-bis(4-chlorophenyl)ethylene by *Pseudomonas acidovorans* M3GY grown on biphenyl, Appl. Environ. Microbiol., **64**, 2141–2146 (1998).
3. Aislabie, J., Davison, A. D., Boul, H. L., Franzmann, P. D., Jardine, D. R., and Karuso, P.: Isolation of *Terrabacter* sp. strain DDE-1, which metabolizes 1,1-dichloro-2,2-bis(4-chlorophenyl)ethylene when induced with biphenyl, Appl. Environ. Microbiol., **65**, 5607–5611 (1999).
4. Nguyen, A. T., Sato, Y., Iwasaki, T., Miyauchi, K., Tokuda, M., Kasai, D., Masai, E., and Fukuda, M.: Characterization of the 1,1-dichloro-2,2-bis(4-chlorophenyl)ethylene (DDE) degradation system in *Janibacter* sp. TYM3221, Enzyme Microb. Technol., **49**, 532–539 (2011).
5. Hanahan, D.: Studies on transformation of *Escherichia coli* with plasmids, J. Mol. Biol., **166**, 557–580 (1983).
6. Saito, I. and Stark, G. R.: Charomids: cosmid vectors for efficient cloning and mapping of large or small restriction fragments, Proc. Natl. Acad. Sci. USA, **83**, 8664–8668 (1986).
7. Yanisch-Perron, C., Vieira, J., and Messing, J.: Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors, Gene, **33**, 103–119 (1985).
8. Short, J. M., Fernandez, J. M., Sorge, J. A., and Huse, W. D.: Lambda Zap: a bacteriophage lambda expression vector with in vivo excision properties, Nucleic Acids Res., **16**, 7583–7600 (1988).
9. Araki, N., Suzuki, T., Miyauchi, K., Kasai, D., Masai, E., and Fukuda, M.: Identification and characterization of uptake systems for glucose and fructose in *Rhodococcus jostii* RHA1, J. Mol. Microbiol. Biotechnol., **20**, 125–136 (2011).

10. Takeda, H., Yamada, A., Miyauchi, K., Masai, E., and Fukuda, M.: Characterization of transcriptional regulatory genes for biphenyl degradation in *Rhodococcus* sp. strain RHA1, J. Bacteriol., **186**, 2134–2146 (2004).

11. Yamada, A., Kishi, H., Sugiyama, K., Hatta, T., Nakamura, K., Masai, E., and Fukuda, M.: Two nearly identical aromatic compound hydrolase genes in a strong polychlorinated biphenyl degrader, *Rhodococcus* sp. strain RHA1, Appl. Environ. Microbiol., **64**, 2006–2012 (1998).

12. Hashimoto, Y., Nishiyama, M., Yu, F., Watanabe, I., Horinouchi, S., and Beppu, T.: Development of a host-vector system in a *Rhodococcus* strain and its use for expression of the cloned nitrile hydratase gene cluster, J. Gen. Microbiol., **138**, 1003–1010 (1992).

13. De Mot, R., Nagy, I., De Schrijver, A., Pattanapipitpaisal, P., Schoofs, G., and Vanderleyden, J.: Structural analysis of the 6 kb cryptic plasmid pFAJ2600 from *Rhodococcus erythropolis* NI86/21 and construction of *Escherichia coli-Rhodococcus* shuttle vectors, Microbiology, **143**(Pt 10), 3137–3147 (1997).

14. Abe, T., Masai, E., Miyauchi, K., Katayama, Y., and Fukuda, M.: A tetrahydrofolate-dependent O-demethylase, LigM, is crucial for catabolism of vanillate and syringate in *Sphingomonas paucimobilis* SYK-6, J. Bacteriol., **187**, 2030–2037 (2005).

15. Gonçalves, E. R., Hara, H., Miyazawa, D., Davies, J. E., Eltis, L. D., and Mohn, W. W.: Transcriptomic assessment of isozymes in the biphenyl pathway of *Rhodococcus* sp. strain RHA1, Appl. Environ. Microbiol., **72**, 6183–6193 (2006).

16. Masai, E., Yamada, A., Healy, J. M., Hatta, T., Kimbara, K., Fukuda, M., and Yano, K.: Characterization of biphenyl catabolic genes of gram-positive poly-chlorinated biphenyl degrader *Rhodococcus* sp. strain RHA1, Appl. Environ. Microbiol., **61**, 2079–2085 (1995).

17. Sukda, P., Gouda, N., Ito, E., Miyauchi, K., Masai, E., and Fukuda, M.: Characterization of a transcriptional regulatory gene involved in dibenzofuran degradation by *Nocardioides* sp. strain DF412, Biosci. Biotechnol. Biochem., **73**, 508–516 (2009).

18. Nagy, I., Schoofs, G., Vanderleyden, J., and De Mot, R.: Transposition of the IS21-related element IS1415 in *Rhodococcus erythropolis*, J. Bacteriol., **179**, 4635–4638 (1997).

19. Stock, J. B., Ninfa, A. J., and Stock, A. M.: Protein phosphorylation and regulation of adaptive responses in bacteria, Microbiol. Rev., **53**, 450–490 (1989).

20. Stock, A. M., Robinson, V. L., and Goudreau, P. N.: Two-component signal transduction, Annu. Rev. Biochem., **69**, 183–215 (2000).

21. Parkinson, J. S. and Kofoid, E. C.: Communication modules in bacterial signaling proteins, Annu. Rev. Genet., **26**, 71–112 (1992).

22. Shimodaira, J., Furusawa, Y., Miyazawa, Y., Kasai, D., Miyauchi, K., Masai, E., and Fukuda, M.: The 24-bp consensus sequence responsible for regulation of the BphS1T1 two-component system in a hybrid promoter, J. Biosci. Bioeng., **113**, 279–285 (2012).

23. Wang, Y., Garnon, J., Labbé, D., Bergeron, H., and Lau, P. C.: Sequence and expression of the *bpdC1C2BADE* genes involved in the initial steps of biphenyl/chlorobiphenyl degradation by *Rhodococcus* sp. M5, Gene, **164**, 117–122 (1995).

24. Labbé, D., Garnon, J., and Lau, P. C.: Characterization of the genes encoding a receptor-like histidine kinase and a cognate response regulator from a biphenyl/polychlorobiphenyl-degrading bacterium, *Rhodococcus* sp. strain M5, J. Bacteriol., **179**, 2772–2776 (1997).

25. Takeda, H., Hara, N., Sakai, M., Yamada, A., Miyauchi, K., Masai, E., and Fukuda, M.: Biphenyl-inducible promoters in a polychlorinated biphenyl-degrading bacterium, *Rhodococcus* sp. RHA1, Biosci. Biotechnol. Biochem., **68**, 1249–1258 (2004).

26. Takeda, H., Shimodaira, J., Yukawa, K., Hara, N., Kasai, D., Miyauchi, K., Masai, E., and Fukuda, M.: Dual two-component regulatory systems are involved in aromatic compound degradation in a polychlorinated-biphenyl degrader, *Rhodococcus jostii* RHA1, J. Bacteriol., **192**, 4741–4751 (2010).

27. Kim, D., Chae, J. C., Zylstra, G. J., Sohn, H. Y., Kwon, G. S., and Kim, E.: Identification of two-component regulatory genes involved in O-Xylene degradation by *Rhodococcus* sp. strain DK17, J. Microbiol., **43**, 49–53 (2005).

28. Tomás-Gallardo, L., Santero, E., Camafeita, E., Calvo, E., Schlömann, M., and Floriano, B.: Molecular and biochemical characterization of the tetralin degradation pathway in *Rhodococcus* sp. strain TFB, Microb. Biotechnol., **2**, 262–273 (2009).

29. Arai, H., Koso, S., Taguchi, K., Maeda, M., Song, E., Fuji, F., Chung, S., and Kudo, T.: Two sets of biphenyl and PCB degradation genes on a linear plasmid in *Rhodococcus erythropolis* TA421, J. Ferment. Bioeng., **86**, 595–599 (1998).

30. Stecker, C., Johann, A., Herzberg, C., Averhoff, B., and Gottschalk, G.: Complete nucleotide sequence and genetic organization of the 210-kilobase linear plasmid of *Rhodococcus erythropolis* BD2, J. Bacteriol., **185**, 5269–5274 (2003).

31. Na, K. S., Kuroda, A., Takiguchi, N., Ikeda, T., Ohtake, H., and Kato, J.: Isolation and characterization of benzene-tolerant *Rhodococcus opacus* strains, J. Biosci. Bioeng., **99**, 378–382 (2005).
