
Review Article

# Cancer Cells Acquire Resistance to Anticancer Drugs: An Update

Hsing-Pang Lu, Chuck C.K. Chao

The efficacy of cancer chemotherapy is often affected by the emergence of resistant cancer cells. While biochemical and pharmacological mechanisms have been proposed to explain chemoresistance, the genes involved in this process have not been fully identified. We previously used genomic DNA microarrays and quantitative RT-PCR to identify the genes associated with resistance to chemotherapeutic drugs, particularly to the genotoxic agent cisplatin. Notably, knockdown of the cisplatin resistance (CPR) genes that we identified was shown to reduce chemoresistance and to suppress the growth of tumor xenographs in cisplatin-treated mice, indicating that the newly identified CPR genes may represent potential therapy candidates to limit chemoresistance and to improve the efficacy of anticancer drugs. In addition to genetic mutations, researchers have found that epigenetic changes and alternative splicing of specific genes may also allow cancer cells to become resistant to chemotherapeutic drugs. In this article, the authors present an overview of the latest findings in this field, including genetic changes, epigenetic changes and alternative splicing. (Biomed J 2012;35:464-72)

Key words: alternative splicing, cisplatin, chemoresistance, DNA microarray, epigenetic modifications

When cancer cells are first discovered in a patient, many drugs can help to get rid of them. But over time, cancer cells begin to withstand those first-line drugs and continue to grow and spread. Accumulating evidences have shown that the heterogeneity of the cancer cell population influences the cellular response to chemotherapeutic drugs. Several cellular alterations have been proposed to explain resistance to anticancer drugs. The conditions of cisplatin treatment also influence the response of cancer cells. For instance, chronic and long-term exposure to increasing concentrations of cisplatin appears to permanently increase the levels of the nucleophile and detoxifying molecules glutathione and metallothionein which are associated with chemoresistance. Acute exposure to cisplatin on a monthly basis leads to defects on the surface of the cell membrane and reduces drug accumulation inside the cells. On the other hand, weekly pulsed-exposure to cisplatin leads to changes in folate metabolism and oncogene expression. In addition, a single lethal concentration of cisplatin may cause DNA adducts in the treated cells, and in turn cell cycle arrest at the G2 phase and apoptosis. The finding that cisplatin-DNA adducts bind to several cellular proteins, termed cisplatin-damaged-DNA recognition proteins, has attracted considerable attention in this field.[1,2] These proteins include signals that enhance cell survival by mediating DNA repair while others induce cell death by conferring sensitivity to the drug. Over the past decade, numerous genes involved in these pathways have been described;[3] for further information, please refer to the website of the Pharmacogenetics Knowledge Base (PharmGKB; http://www.pharmgkb.org/). It is important to keep in mind that differences may exist between *in vitro* studies and the patients since the environment of a tumor is remarkably different from that of cultured cells in terms of nutrients, growth factors, hormones, pH, intercellular communication, and oxygenation state. In addition, the various oncogene and protein kinase signaling pathways are likely to be differentially regulated in these two environments.

## Multiple pathways of anti-cancer drug resistance

Cisplatin resistance (CPR) is the most studied example in anticancer resistance. Herein, its pharmacokinetic views

---

From the Department of Biochemistry and Molecular Biology, and Institute of Biomedical Sciences, College of Medicine, Chang Gung University, Taoyuan, Taiwan.

Received: Apr. 11, 2012; Accepted: Jun. 1, 2012

Correspondence to: Prof. Chuck C.K. Chao, Department of Biochemistry and Molecular Biology, and Institute of Biomedical Sciences, College of Medicine, Chang Gung University. 259 Wen-Hwa 1st Road, Kweishan, Taoyuan 333, Taiwan (R.O.C.). Tel: 886-3-2118136; Fax: 886-3-2118700; E-mail: cckchao@mail.cgu.edu.tw

DOI: 10.4103/2319-4170.104411

and genetic changes will be used for discussion of the mechanism of anticancer drug resistance. For cisplatin, the mechanisms involved in this process include decreased drug influx, increased drug efflux, activation of detoxification systems, alteration of the drug targets, increased DNA repair, impaired apoptosis, and altered oncogene expression.[4-7] Changes triggered by cisplatin selection in the resistant cells involve a secondary layer of complexity that may include alterations in growth factors and hormone responsiveness, ion transport, nutrient transport and utilization, thymidilate metabolism, oncogene and protein kinase signaling pathways, chromosome structure, and gene expression. Over the past decade, a few specific genes involved in these pathways have been described. The gene products that play crucial roles in the regulation of cells in response to cisplatin have been summarized [Figure 1].[8] For example, the influx of cisplatin into the cells is regulated by SLC31A1 (CTR1), while the efflux out of the cell is controlled by ABCC2 (MRP2), ATP7A, and ATP7B. Once cisplatin is inside the cell, the primary anticancer mechanism is the formation of Pt-DNA adducts, which leads to cell-cycle arrest and apoptosis. Genes involved in DNA repair (e.g., MSH6

and MLH1 for mismatch repair; XRCC1, ERCC1, ERCC2, and XPA for nucleotide excision repair; HMGB1 for recognition of Pt-DNA adducts) decrease the sensitivity of the cells to cisplatin. Several detoxification-related genes (e.g., myeloperoxidase (MPO), superoxide dismutase 1 (SOD1), glutathione S-transferases M1 (GSTM1), NAD(P)H:quinone oxidoreductase 1 (NQO1), glutathione S-transferases P1 (GSTP1), and metallothioneins (MT)) are also responsible for lowering the intracellular concentration of cisplatin, and therefore play an important role in cellular resistance to the drug. In addition, the DNA polymerase variants POLH and POLB, which are normally involved in translesional DNA replication, have been shown to provide tolerance to cisplatin-based drugs, and therefore represent important determinants of the cellular responses to cisplatin.[8] The model depicted in Figure 1 can also be applied to other Pt-containing drugs as well as some genotoxic and chemotherapeutic compounds.

### Genetic changes

DNA microarrays have been used to identify genome-wide gene expression patterns that can be used to predict drug responses along with metastasis, disease relapse, and prognosis of cancer patients.[9,10] Cisplatin-resistant cervix carcinoma HeLa cell lines, which were obtained by repeatedly treating parental HeLa cells with increasing concentrations of cisplatin,[11] have been used to identify the genes involved in cisplatin resistance. Of the altered genes we identified, nine were overexpressed at least twofold in the moderately resistant cells, and more than fourfold in highly resistant cells. Using gene knockdown experiments, we confirmed that the genes identified were involved in cisplatin resistance in various degrees. We recently reported that knockdown of CPR genes sensitized tumor cell lines to cisplatin, but not to the mitosis-disrupting agents such as vincristine and taxol. Thus, these observations suggest that CPR genes may be involved in cells resistance to genotoxic drugs.[12] Specifically, we showed that N-ethylmaleimide-sensitive factor attachment protein alpha (NAPA) and Cbp/p300-interacting transactivator, with Glu/Asp-rich carboxy-terminal domain, 2 (CITED2) (see below) may play a major role in acquired cisplatin resistance and that this process may rely on the tumor suppressor p53.[12-15] Overall, our results confirm the notion that genetic changes are important for acquired resistance during cancer chemotherapy.

### Gene upregulation in genotoxic drug resistance

The genes (NAPA, CITED2, CABIN1, ADM, HIST1H1A, EHD1, MARK2, PTPN21, and MVD), here referred to as cisplatin resistance (CPR) genes [Table 1], were initially reported by our group for their ability to modify the response of non-tumor, but viral transformed, human

Figure 1: Pharmacokinetic pathways of cisplatin resistance. Several cellular alterations have been proposed to explain drug resistance, including decreased drug influx, increased drug efflux, activation of detoxification systems, alteration of the targets of the drug, increased DNA repair, impaired apoptosis, and altered oncogene expression. Thirty gene products involved in separate pathways are indicated. For additional information, one may refer to the website of the Pharmacogenetics Knowledge Base. (PharmGKB; http://www.pharmgkb.org/)

Table 1: Levels of upregulated CPR Genes in cisplatin-resistant HeLa cells assessed by DNA microarray analysis

| Gene Symbol (RefSeq ID) | Function | R1/HeLa | R3/HeLa |
|--------------------------|----------|---------|---------|
| NAPA (NM_003827)         | Intracellular transport | 2.02±0.33* | 7.18±3.11* |
| CITED2 (NM_006079)       | Regulation of transcription | 2.58±0.11* | 4.38±0.53* |
| CABIN1 (NM_012295)       | Calcineurin binding protein | 3.75±0.24* | 4.66±0.27* |
| ADM (NM_001124)          | C21-steroid hormone biosynthesis | 4.81±0.02* | 4.16±0.06* |
| HIST1H1A (NM_005325)     | Histone cluster | 4.57±2.93* | 7.21±6.26* |
| EHD1 (NM_006795)         | EH-domain containing | 2.25±0.13* | 4.48±0.87† |
| MARK2 (NM_004945)        | Kinase phosphorylation | 2.27±0.52* | 4.02±1.28* |
| PTPN21 (NM_007039)       | Tyrosine phosphatase | 2.32±0.01* | 4.07±1.06* |
| MVD (NM_002461)          | Cholesterol metabolism | 2.72±0.21* | 4.20±0.84† |

Abbreviations: *: p<0.05; †: p<0.01; R1: Moderate resistance; R3: High resistance.

embryonic kidney HEK293 cells to cisplatin.[12] By using short-hairpin RNA (shRNA) to knockdown the CPR genes individually or in combination, we were able to sensitize HEK293 cells to genotoxic drugs including cisplatin. Among the treatments performed, shRNA knockdown of NAPA — which interacts with its receptor ER-SNARE (soluble N-ethylmaleimide-sensitive fusion protein-attachment protein receptor) found in the endoplasmic reticulum (ER) and implicated in protein trafficking — was the most efficient treatment able to sensitize cells to cisplatin. Furthermore, shRNA knockdown of a single CPR gene was sufficient to partially reverse acquired cisplatin resistance in HeLa cells. Sensitization to cisplatin following knockdown of CPR genes was also observed in tumorigenic cell lines, such as ovarian cancer cells Sk-ov-3, lung cancer cells H1155, and nasopharyngeal cancer cells CG-1.[12] In addition, cells with low CPR genes levels were protected against cisplatin-induced apoptosis. Based on these results, we propose that the CPR genes identified may represent potential candidates for novel target therapies aimed at preventing resistance to cisplatin during chemotherapy.

Among the CPR candidates we identified, only ADM is an extracellular protein, while other five gene products (EHD1, MARK2, MVD, NAPA and PTPN21) are cytoplasmic, and three (CABIN1, CITED2 and HIST1H1A) are nuclear proteins. The CPR proteins have not been described in the canonical pathways of the MetaCore software (Version 6.3; Build 25485; GeneGo pathway analysis software for systems biology, St. Joseph, MI). ADM has been found to represent an anti-apoptotic factor in different cell lines as well as in vivo.[16,17] Although the mechanism of ADM in preventing apoptosis remains unclear, the impairment of apoptosis may still explain acquired cisplatin resistance. An “extracellular region” of ADM is directly upregulated through canonical pathways of transcription factors (SRF and ESR1) in the nucleus. ADM may negatively regulate some transcription factors (e.g. GATA-4) in the nucleus via cytoplasmic GRK2, and may dysregulate c-Src which normally passes signal to p300 in the cytoplasm. Therefore,

p300 which regulates many transcription factors, including GATA-4, may be downregulated by ADM. Although GATA-4 is a downstream target of p300, it is probably not part of the pathway of ADM-mediated drug resistance since GATA-4 has been shown to provide a protective role against oxidative stress in the heart.[18] These results suggest that downregulation of p300 by ADM may be a therapeutic target to counteract cisplatin resistance. Our results also indicate that, NAPA and CITED2 may play a major role in acquired cisplatin resistance as seen in the HeLa cell culture and mouse models studied.[12–13] Interestingly, the regulation of cisplatin resistance by both proteins also involves p53 (see below).

**p53-dependent ER/Golgi pathways in genotoxic drug resistance**

We found that NAPA protects the cell against cisplatin.[13] Accordingly, knockdown of NAPA using lentivirus-encoding shRNA (shNAPA) induced ER stress similar to the effect of cisplatin treatment in HEK293 cells. A low dose of cisplatin also elicited a mild ER-stress response associated with the accumulation of the protective proteins BiP and NAPA. Remarkably, knockdown of NAPA induced apoptosis and enhanced cisplatin-induced apoptosis and growth inhibition, thereby sensitizing cancer cells to cisplatin. On the other hand, overexpression of NAPA increased resistance to cisplatin by reducing cisplatin-induced ER stress and apoptosis as well as growth inhibition. The modulatory effects of shNAPA required the tumor suppressor p53 since the effects of NAPA knockdown were reduced by the p53 inhibitor PFT-α (pifithrin-α) and these effects were also reduced in p53-null lung cancer H1299 cells. A partial reversal of cisplatin resistance was also observed in cisplatin-resistant HeLa cells following knockdown of NAPA. Importantly, a combined cisplatin/shNAPA treatment suppressed the growth of tumor xenographs in nude mice. Taken together, these observations suggest that NAPA represents a target of cisplatin, and that knockdown of NAPA expression may improve cisplatin-based cancer therapy. A working hypothesis of ER damage and the related signal

drugs vincristine and taxol. Accordingly, we observed that knockdown of GS28 reduced the accumulation of p53 and its proapoptotic target Bax. Conversely, ectopic expression of GS28 induced the accumulation of p53 and Bax as well as the proapoptotic phosphorylation of p53 on Ser46. Further experiments showed that these cellular responses could be abrogated by the p53 inhibitor PFT-α, indicating that GS28 may affect the stability and activity of p53. However, the modulatory effects of GS28 on cisplatin sensitivity and p53 stability were absent in p53-null H1299 cells. As expected, ectopic expression of p53 in H1299 cells restored the modulatory effects of GS28 on sensitivity to cisplatin. Notably, GS28 was found to form a complex with MDM2 (murine double minute 2), an E3 ligase of p53, in H1299 cells. Furthermore, the ubiquitination of p53 was reduced in GS28-overexpressing cells, confirming that GS28 enhances the stability of the p53 protein.[21] Taken together, these results suggest that GS28 may potentiate cells to genotoxic agents-induced apoptosis by inhibiting the ubiquitination and degradation of p53 [Figure 3].

## Epigenetic changes

Phenotypic diversity arises in tumors just as it does in developing organisms, and tumor recurrence frequently manifests due to the selective survival advantage of divergent drug-resistant cells. In addition to genetic mutations, researchers have found that epigenetic changes represent non-mutational mechanisms that are also involved in drug resistance. Unlike the genetic changes that occur in cancer cells, however, epigenetic modifications can occur quickly in response to environmental changes. For example, accumulating evidences suggest that a small population of the so-called “cancer stem cells” are intrinsically refractory to a variety of anticancer drugs, an observation possibly due to enhanced drug efflux.[22] Studies have shown that epigenetic modifications are involved in this process, suggesting that acquired drug resistance does not necessarily require stable heritable genetic alterations.[23]

## Histone deacetylases play an important role in epigenetic regulation

Histone deacetylases (HDACs) play an important role in the epigenetic regulation of gene expression by catalyzing the removal of acetyl groups which stimulates chromatin condensation and represses transcription. Given that aberrant epigenetic changes are a hallmark of cancer, HDACs represent promising targets for pharmacological treatments. Inhibition of HDAC can induce cell cycle arrest, promote cell differentiation, and stimulate cell death in tumor cells. These properties have prompted numerous preclinical and clinical investigations to evaluate the efficacy of HDAC inhibitors against various malignancies.

Hsing-Pang Lu and Chuck C.K. Chao

Acquired resistance to anticancer drugs

is the generation of reactive oxygen species (ROS) which appear to play an important role in activating the intrinsic apoptosis pathway.[24]

**Emergence of drug-resistant cancer cells requires lysine (K)-specific demethylase 5A (KDM5A)**

While modeling the acute response to various anticancer agents in drug-sensitive human tumor cell lines, we consistently detected a small population of reversibly “drug-tolerant” cells. These cells demonstrated a more than 100-fold reduction in drug sensitivity and maintained viability via engagement of insulin-like growth factor 1 receptor (IGF1R) signaling and via altered chromatin states that required the HDAC lysine-specific demethylase 5A (KDM5A/RBP2/Jarid1A). This drug-tolerant phenotype was transiently acquired at low frequency by individual cells within the population. Notably, the drug-tolerant cancer cell subpopulation could be selectively ablated by the treatment with IGF1R inhibitors or by chromatin-modifying agents. Together, these findings suggest that populations of cancer cells may employ dynamic survival strategies in which individual cells transiently assume a reversible drug-tolerant state to protect the population from eradication by potentially lethal drugs.[25,26]

**Epigenetic modulation for overcoming resistance to anticancer hormonal therapy**

It has been found that epigenetic changes are responsible for the resistance that many breast cancers acquire against the estrogen-blocking drug tamoxifen following 18 months of treatment. Surprisingly, while no mutations that correlated with resistance were detected in the treated breast cancer cells, resistant cancer cells expressed the survival gene Akt at much higher levels than susceptible cells. Apparently, cancer cells used histone tags, or chemical markers on chromatin, to expose the Akt gene and increase its transcription. The higher signal of Akt allowed cancer cells to stay alive even in the presence of tamoxifen and this by stimulating growth and proliferation and by preventing cell death.[27] Therefore, tumor cells may tag genes responsible for chemoresistance in order to pass them to the next generations, thus giving daughter cells an advantage in the presence of the drug.

**Nuclear pore complex architecture regulates resistance to anticancer drugs independently of genetic mutations**

Cancer cells may also use a trick from viruses in order to switch between resistance and susceptibility. Kohtz and colleagues explored the reasons why ovarian carcinomas sometimes become resistant to a drug and why cells become

Biomed J Vol. 35 No. 6  
November - December 2012

| Class I | HDAC 1 | Nucleus | Androgen Receptor |
| --- | --- | --- | --- |
|  |  |  | P53 |
|  |  |  | MyoD |
|  |  |  | E2F-1 |
|  |  |  | SHP |
|  |  |  | Stat3 |
|  | HDAC 2 | Nucleus | Glucocorticoid Receptor |
|  |  |  | Bcl-6 |
|  |  |  | YY-1 |
|  |  |  | Stat3 |
|  | HDAC 3 | Nucleus | RelA |
|  |  |  | YY-1 |
|  |  |  | GATA-1 |
|  |  |  | SHP |
|  |  |  | Stat3 |
|  | HDAC 8 | Nucleus |  |
| Class IIa | HDAC 4 | Nucleus/Cytoplasm | GATA-1 |
|  |  |  | GCMa |
|  |  |  | HP-1 |
|  | HDAC 5 | Nucleus/Cytoplasm | Smad7 |
|  |  |  | GCMa |
|  |  |  | HP-1 |
|  | HDAC 7 | Nucleus/Cytoplasm | Plag1/Plag2 |
|  | HDAC 9 | Nucleus/Cytoplasm |  |
| Tubacin | Class IIb | HDAC 6 | Mostly Cytoplasm | α-tubulin |
|  |  |  | Hsp90 |
|  |  |  | SMP |
|  |  |  | Smad7 |
|  | HDAC 10 | Mostly Cytoplasm |  |
| Class IV | HDAC 11 | Nucleus/Cytoplasm |  |
| Class III | EX-527 | Sirt1 | Nucleus | P53 |
|  |  |  | FOXO |
|  |  |  | NF-kB |
|  |  |  | PGC-1α |
|  | Sirt2 | Cytoplasm | Histone H4 |
|  |  |  | α-tubulin |
|  | Sirt3 | Nucleus/Mitochondria | Acetyl-CoA synthetases |
|  | Sirt4 | Mitochondria | Glutamate dehydrogenase |
|  | Sirt5 | Mitochondria |  |
|  | Sirt6 | Nucleus | DNA polymerase B |
|  | Sirt7 | Nucleus | RNA polymerase I |

susceptible again after the drug is no longer taken. [28] Such a change is unlikely to be caused by genetic mutations, because these are usually not reversed so quickly. Prior studies have shown that nuclear pores, which help transport proteins from the nucleus to the cytoplasm and vice versa, could influence gene expression by interacting with chromatin at the periphery of the nucleus. Specifically, nuclear pores can activate transcription by shielding DNA from repressors, or by hindering transcription because repressor proteins lurk in the region near the nuclear boundary, suggesting that nuclear pores might be altered in different cancer cell types. Using electron microscopy to visualize the nuclei of cisplatin-resistant cancer cells, researchers observed that nuclear pores looked abnormal: the pore complexes appeared to be hollow and disassembled and to contain a gatekeeper protein in the center, a finding similar to the pores affected by viruses that co-opt cellular transport machinery for their own purposes. Partial knockdown of nucleoporin p62 (NUP62) by small-interfering RNA (siRNA) conferred resistance to cisplatin in cultured high-grade ovarian carcinoma cells. Treatments with NUP62 siRNA and cisplatin left resistant cells in a state of dormancy and some dormant cells could be induced to proliferate by transient induction of NUP62 expression from an ectopic expression construct. [28] In vivo, the dormant cells may be located at metastatic sites or even at the site of the original tumor until a factor induces their malignancy and makes them start growing again. These results suggest that functional links exist between nuclear pore complex architecture, chromatin regulation, and cancer cell survival.

We also found that the anti-apoptotic function of CITED2 during cisplatin treatment is also p53-dependent. [14] A recent study showed that knockdown of CITED2 using siRNA in Pt complex-resistant ovarian cancer cells improved the sensitivity of these cells to Pt compounds. [29] DNA damage-induced phosphorylation of p53 enhances the association of this protein with the CBP/p300 transcriptional co-activators which results in increased acetylation and stability of p53. [30–32] Yet, these gene products are unlikely to interact in cancer cells since NAPA is found in the ER [33] whereas CITED2 transcriptional regulatory proteins are found in the nucleus. [34] Furthermore, CITED2 modifies acetyltransferase like p300 that may modulates chromatin proteins, leading to chemoresistance-associated gene expression in a epigenetic manner. In this case, CITED2, like NAPA, may represent a target of cisplatin and may regulate drug sensitivity by influencing the stability of p53. This model could represent a new molecular mechanism to explain cisplatin resistance in cancer cells.

**Alternative splicing**

Yet another way that cancer cells use to acquire drug resistance is associated with protein processing. In normal

cells, signal-activated RAS recruits BRAF (or v-Raf murine sarcoma viral oncogene homolog B1) to the cell membrane and activates its kinase domain through dimerization. Activated BRAF, in turn, triggers MEK and ERK protein kinases, through phosphorylation, and promotes cell proliferation and survival. In contrast, the mutant BRAFV600E continuously sends signals to MEK and ERK even in the absence of activation by RAS. Notably, BRAFV600E is highly sensitive to inhibition by vemurafenib. Between 40 and 80% of melanoma patients have a mutated BRAF (BRAFV600E) which turns on cellular growth and division signaling pathways. Vemurafenib exploits the fact that BRAF proteins in healthy cells pair up with other BRAF proteins to form a multiprotein complex, while the mutated BRAF protein acts as a single compound. This single structure can be hundreds of times more effective in activating cell division than the normal paired BRAF complexes. Vemurafenib targets tumor cells by only inhibiting the stand-alone mutant version, while allowing the twinned version in healthy cells to act unimpaired [Figure 4]. Unfortunately, many patients develop resistance to vemurafenib within 18 months, and their tumors progress. Some of the resistant tumor cells generate a variant form of BRAF that is shorter. However, the shorter BRAF protein is not due to a mutation in the protein-coding region of DNA. Instead, deletions of exons in the gene lead to alternative splicing that generate the shorter version, which can bind to itself, rendering the protein undetectable under

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 4: The two faces of oncogenic BRAF. A, In normal cells, signal-activated RAS recruits BRAF to the cell membrane and activates its kinase domain through dimerization. Active BRAF, in turn, triggers MEK and ERK protein kinases, through phosphorylation (denoted with the letter P), to promote cell proliferation and survival. B, Mutant BRAFV600E continuously sends signals to MEK and ERK, even in the absence of activation by RAS. BRAFV600E is highly sensitive to the anticancer drug vemurafenib. Poulikakos et al. reported that BRAFV600E essentially works as a monomer. C, The authors also show that p61BRAFV600E—the truncated variant of BRAFV600E has an increased propensity to form dimers and that this is associated with resistance to vemurafenib. [Adapted from ref. [36]].

Biomed J Vol. 35 No. 6  
November - December 2012

vemurafenib treatment, processes which appear to account for vemurafenib resistance.[35]

These findings are fascinating from a mechanistic standpoint, but we may ask whether they are clinically relevant. Solit and colleagues reported that six of 19 samples from patients with drug-resistant melanoma expressed BRAFV600E truncated variants similar to p61BRAFV600E. However, this sample number was too small to assess the prevalence of p61B-RAFV600E in the human population. Nonetheless, the finding is still impressive since it suggests that enhanced BRAFV600E dimerization may significantly contribute to vemurafenib resistance in patients with melanoma. In addition to the need to analyze larger sample sizes, these observations should be corroborated in animal models. This study and others indicate that any event promoting RAF dimerization may lead to resistance to vemurafenib and possibly to other similar RAF inhibitors.[35] How can we get around this problem? Co-administering RAF and MEK inhibitors could be a solution. Indeed, as p61B-RAFV600E-expressing cells have been found to remain sensitive to a MEK inhibitor, shutting down both RAF and MEK activity might prevent the growth of resistant cells.[35]

## Conclusions

While cancer cells use an intimidating array of tactics to evade drug therapies, researchers are developing ways to target resistant cells. Given that the mechanisms underlying chemoresistance in cancer therapy are getting clear, targeted therapy or combination therapy can be established in laboratory models and clinical settings. Although the expanding tumor cell population may be successfully targeted, drug-resistant cells may persist and sustain the tumor or enter in a state of dormancy before igniting a future relapse. Researchers are now making more progress into understanding how cancer cells acquire drug resistance, and they are finding that genetic mutations are just one of many strategies cancers use to evade death. In this mini-review, we have presented new mechanisms of anticancer resistance, revealing additional changes in cancer receiving therapy. The findings in this selected literature convincingly explain the loss of several powerful target therapies. In human trials, adding a compound that removes histone tags to tamoxifen regimen can make resistant breast cancers sensitive to the drug again. Similarly, ongoing Phase II trials are testing a combination of vemurafenib and another drug that inhibits a compound in the cellular division pathway in melanoma patients that are resistant to vemurafenib. And for those cancers that have switched between quiescence and active growth, simply treating the patients again with the same medicine later on can sometimes be effective. While there are general principles that apply to cancer resistance, current treatments still require a tailored approach that uses frequent biopsies of tumors to see the nature of the genetic and epigenetic mutations that they have acquired. Unfortunately, with more thorough genetic sequencing, it is also becoming clear that there is no single answer, even for a single patient. Instead, combination approaches usually provide the best results.

## Acknowledgments

The authors would like to thank the members of their research group for helpful discussions. This review is based on studies supported in part by the National Science Council, Taiwan (NSC96-2320-B-182-034, NSC97-2320-B-182-024-MY3), Chang Gung University and Hospitals (CMRPD150293, CMRPD180023, CMRP190052), and the Foundation for the Advancement of Outstanding Scholarship.

## REFERENCES

1. Chao CC. Damage-recognition proteins as a potential indicator of DNA-damage-mediated sensitivity or resistance of human cells to ultraviolet radiation. Biochem J 1992;282:203-7.
2. Chu G. Cellular responses to cisplatin. The roles of DNA-binding proteins and DNA repair. J Biol Chem 1994;269:787-90.
3. Chao CC. A search for the genes involved in resistance to cisplatin chemotherapy: Review of the experimental evidence. Curr Top Pharmacol 2010;14:47-57.
4. Chao CC. Molecular basis of cis-diamminedichloroplatinum (II) resistance: A review. J Formos Med Assoc 1996;95:893-900.
5. Dempke W, Voigt W, Grothey A, Hill BT, Schmoll HJ. Cisplatin resistance and oncogenes--A review. Anticancer Drugs 2000;11:225-36.
6. Siddik ZH. Cisplatin: Mode of cytotoxic action and molecular basis of resistance. Oncogene 2003;22:7265-79.
7. Stewart DJ. Mechanisms of resistance to cisplatin and carboplatin. Crit Rev Oncol Hematol 2007;63:12-31.
8. Klein TE, Chang JT, Cho MK, Easton KL, Fergerson R, Hewett M, et al. Integrating genotype and phenotype information: an overview of the PharmGKB project. Pharmacogenetics Research Network and Knowledge Base. Pharmacogenomics J 2001;1:167-70.
9. Potti A, Dressman HK, Bild A, Riedel RF, Chan G, Sayer R, et al. Genomic signatures to guide the use of chemotherapeutics. Nat Med 2006;12:1294-300.
10. Wulfkuhle J, Espina V, Liotta L, Petricoin E. Genomic and proteomic technologies for individualisation and improvement of cancer treatment. Eur J Cancer 2004;40:2623-32.
11. Chao CC, Lee YL, Cheng PW, Lin-Chao S. Enhanced host cell reactivation of damaged plasmid DNA in HeLa cells resistant to cis-diamminedichloroplatinum (II). Cancer Res 1991;51:601-5.
12. Wu ZZ, Lu HP, Chao CC. Identification and functional analysis of genes which confer resistance to cisplatin in tumor cells. Biochem Pharmacol 2010;80:262-76.

Hsing-Pang Lu and Chuck C.K. Chao  
Acquired resistance to anticancer drugs  

13. Wu ZZ, Chao CC. Knockdown of NAPA using short-hairpin RNA sensitizes cancer cells to cisplatin: Implications to overcome chemoresistance. Biochem Pharmacol 2010; 80:827-37.  
14. Wu ZZ, Sun NK, Chao CC. Knockdown of CITED2 using short-hairpin RNA sensitizes cancer cells to cisplatin through stabilization of p53 and enhancement of p53-dependent apoptosis. J Cell physiol 2011; 226:2415-28.  
15. Wu ZZ, Sun NK, Chien KY, Chao CC. Silencing of the SNARE protein NAPA sensitizes cancer cells to cisplatin by inducing ERK1/2 signaling, synoviolin ubiquitination and p53 accumulation. Biochem Pharmacol 2011; 82:1630-40.  
16. Kaafarani I, Fernandez-Sauze S, Berenguer C, Chinot O, Delfino C, Dusset C, et al. Targeting adrenomedullin receptors with systemic delivery of neutralizing antibodies inhibits tumor angiogenesis and suppresses growth of human tumor xenografts in mice. FASEB J 2009; 23:3424-35.  
17. Shichiri M, Hirata Y. Regulation of cell growth and apoptosis by adrenomedullin. Hypertens Res 2003; 26 Suppl: S9-14.  
18. Suzuki YJ, Evans T. Regulation of cardiac myocyte apoptosis by the GATA-4 transcription factor. Life Sci 2004; 74:1829-38.  
19. Yamasaki S, Yagishita N, Nishioka K, Nakajima T. The roles of synoviolin in crosstalk between endoplasmic reticulum stress-induced apoptosis and p53 pathway. Cell Cycle 2007; 6:1319-23.  
20. Yamasaki S, Yagishita N, Sasaki T, Nakazawa M, Kato Y, Yamadera T, et al. Cytoplasmic destruction of p53 by the endoplasmic reticulum-resident ubiquitin ligase ‘Synoviolin’. EMBO J 2007; 26:113-22.  
21. Sun NK, Huang SL, Chien KY, Chao CC. Golgi-SNARE GS28 potentiates cisplatin-induced apoptosis by forming GS28-MDM2-p53 complexes and by preventing the ubiquitination and degradation of p53. Biochem J 2012; 444:303-14.  
22. Trumpf A, Wiestler OD. Mechanisms of Disease: Cancer stem cells--targeting the evil twin. Nat Clin Pract Oncol 2008; 5:337-47.  
23. Glasspool RM, Teodoridis JM, Brown R. Epigenetics as a mechanism driving polygenic clinical drug resistance. Br J Cancer 2006; 94:1087-92.  
24. Carew JS, Giles FJ, Nawrocki ST. Histone deacetylase inhibitors: Mechanisms of cell death and promise in combination cancer therapy.  

25. Sharma SV, Lee DY, Li B, Quinlan MP, Takahashi F, Maheswaran S, et al. A chromatin-mediated reversible drug-tolerant state in cancer cell subpopulations. Cell 2010; 141:69-80.  
26. Workman P, Travers J. Cancer: Drug-tolerant insurgents. Nature 2010; 464:844-5.  
27. Raha P, Thomas S, Munster PN. Epigenetic modulation: A novel therapeutic target for overcoming hormonal therapy resistance. Epigenomics 2011; 3:451-70.  
28. Kinoshita Y, Kalir T, Rahaman J, Dottino P, Kohtz DS. Alterations in nuclear pore architecture allow cancer cell entry into or exit from drug-resistant dormancy. Am J Pathol 2012; 180:375-89.  
29. Yanagie H, Hisa T, Ogata A, Miyazaki A, Nonaka Y, Nishihiro T, et al. Improvement of sensitivity to platinum compound with siRNA knockdown of upregulated genes in platinum complex-resistant ovarian cancer cells *in vitro*. Biomed Pharmacother 2009; 63:553-60.  
30. Ito A, Lai CH, Zhao X, Saito S, Hamilton MH, Appella E, et al. p300/CBP-mediated p53 acetylation is commonly induced by p53-activating agents and inhibited by MDM2. EMBO J 2001; 20:1331-40.  
31. Shi Y, Venkataraman SL, Dodson GE, Mabb AM, LeBlanc S, Tibbetts RS. Direct regulation of CREB transcriptional activity by ATM in response to genotoxic stress. Proc Natl Acad Sci USA 2004; 101:5898-903.  
32. Vassilev A, Yamauchi J, Kotani T, Prives C, Avantaggiati ML, Qin J, et al. The 400 kDa subunit of the PCAF histone acetylase complex belongs to the ATM superfamily. Mol Cell 1998; 2:869-75.  
33. Clary DO, Griff IC, Rothman JE. SNAPs, a family of NSF attachment proteins involved in intracellular membrane fusion in animals and yeast. Cell 1990; 61:709-21.  
34. Barbera JP, Rodriguez TA, Greene ND, Weninger WJ, Simeone A, Copp AJ, et al. Folic acid prevents exencephaly in Cited2 deficient mice. Hum Mol Genet 2002; 11:283-93.  
35. Poulikakos PI, Persaud Y, Janakiraman M, Kong X, Ng C, Moriceau G, et al. RAF inhibitor resistance is mediated by dimerization of aberrantly spliced BRAF (V600E). Nature 2011; 480:387-90.  
36. Lavoie H, Therrien M. Cancer: A drug-resistant duo. Nature 2011; 480:329-30.
