
Int. J. Mol. Sci. 2013, 14, 4560-4579; doi:10.3390/ijms14034560

OPEN ACCESS

**International Journal of Molecular Sciences**

ISSN 1422-0067  
www.mdpi.com/journal/ijms

---

**Review**

**Small Regulatory RNAs in the Control of Motility and Biofilm Formation in E. coli and Salmonella**

Franziska Mika *and* Regine Hengge

Institut für Biologie-Mikrobiologie, Freie Universität Berlin, Berlin 14195, Germany;  
E-Mail: rhenggea@zedat.fu-berlin.de

* Author to whom correspondence should be addressed; E-Mail: franziska.mika@fu-berlin.de; Tel.: +49-30-838-546-70; Fax: +49-30-838-531-18.

Received: 18 January 2013; in revised form: 10 February 2013 / Accepted: 16 February 2013 / Published: 26 February 2013

---

**Abstract:** Biofilm formation in *Escherichia coli* and other enteric bacteria involves the inverse regulation of the synthesis of flagella and biofilm matrix components such as amyloid curli fibres, cellulose, colanic acid and poly-N-acetylglucosamine (PGA). Physiologically, these processes reflect the transition from growth to stationary phase. At the molecular level, they are tightly controlled by various sigma factors competing for RNA polymerase, a series of transcription factors acting in hierarchical regulatory cascades and several nucleotide messengers, including cyclic-di-GMP. In addition, a surprisingly large number of small regulatory RNAs (sRNAs) have been shown to directly or indirectly modulate motility and/or biofilm formation. This review aims at giving an overview of these sRNA regulators and their impact in biofilm formation in *E. coli* and *Salmonella*. Special emphasis will be put on sRNAs, that have known targets such as the mRNAs of the flagellar master regulator FlhDC, the stationary phase sigma factor σ<sup>S</sup> (RpoS) and the key biofilm regulator CsgD that have recently been shown to act as major hubs for regulation by multiple sRNAs.

**Keywords:** biofilm matrix; c-di-GMP; cellulose; curli; CsgD; flagella; flhDC; Hfq; RpoS; sRNA

Int. J. Mol. Sci. 2013, 14

1. Structural Organization and Regulation of Biofilms

Bacterial biofilms are surface-attached multicellular aggregates, in which cells are embedded in an extracellular matrix. In *E. coli* and related bacteria this matrix consists of proteinaceous components including various adhesins as well as amyloids such as curli fibers, which can be interwoven with the exopolysaccharides cellulose, poly-β-1,6-D-*N*-acetylglucosamine (PGA) and colanic acid (for a review see [1,2]). These components form elaborate three-dimensional structures, surrounding the bacteria and protecting them against desiccation, antibiotics and the immune system of the host [3,4]. The composition of the biofilm matrix varies depending on temperature, growth conditions and genetic background of the strains. In pathogenic *E. coli* living within the host or on abiotic surfaces (37 °C), type I fimbriae or the adhesin AG43 are involved in initial attachment, PGA stabilizes permanent attachment and also curli fibers, which contribute to surface attachment, can be a predominant matrix component [5–7]. Bacteria growing in the environment or on abiotic surfaces at lower temperatures (<30 °C), form differently composed biofilms, using flagella for initial attachment and curli, cellulose and colanic acid as a matrix in the mature biofilm [1,8–10]. However, *E. coli* K-12 laboratory strains do not produce cellulose, as well as some natural isolates [11,12]. In *Salmonella enterica* serovar Typhimurium the process and regulation of biofilm formation is similar as in *E. coli*, with the matrix consisting of curli and cellulose [13].

### 1.1. The Switch from Motile to Sessile Lifestyle—Inverse Regulation of Flagellar and Curli/Cellulose Control Cascades

In *E. coli* K-12 the transition from a planktonic lifestyle to biofilm life is regulated by two inversely controlled transcriptional feedforward cascades, the FlhDC + σ<sup>70</sup>/σ<sup>F</sup> “flagellar” cascade and the σ<sup>S</sup>/MlrA/CsgD cascade, which are active in post-exponentially growing and in stationary phase cells, respectively. Their final outputs—flagella production or the synthesis of the biofilm matrix components curli and cellulose—are mutually exclusive due to complementary sigma factor requirements, the abundant DNA-binding regulator FliZ and opposite regulation by c-di-GMP (Figure 1A and [14,15]). Signals that drive and modulate this lifestyle transition are nutrient limitation, low temperature and cell envelope alterations.

FlhDC, the class I activator of the flagellar regulatory cascade, is upregulated in the late or post-exponential growth phase when planktonic cells become motile and thus adopt a “foraging strategy” to improve their nutrient status [16]. FlhDC controls the σ<sup>F</sup> sigma factor, which is encoded by the class II gene *fliA* and which in turn transcribes class III genes, that together with class II genes generate flagella (Figure 1A; for a review see [17]). In parallel, the stationary phase sigma factor σ<sup>S</sup> is already induced, but the formation of σ<sup>S</sup>-containing RNAP holoenzyme remains inefficient until accessory factors such as Crl and Rsd also accumulate [18]. In addition, the expression of many σ<sup>S</sup>-dependent genes is also inhibited by factors that belong to the flagellar cascade: First, the class II regulator FliZ can bind to an element with overlapping specificity with σ<sup>S</sup>-dependent promoters and thereby can reduce or delay the transcription of a subset of σ<sup>S</sup>-dependent genes [14,15]. The second mode of control is exerted by c-di-GMP, which is an important second messenger implicated in flagella, biofilm and virulence control [19,20]. c-di-GMP activates transcription of *csgD* which

encodes the curli/cellulose regulator CsgD. However, the $\sigma^{F}$-dependent phosphodiesterase (PDE) YhjH counteracts this by degrading c-di-GMP. By maintaining low levels of c-di-GMP via YhjH, the flagellar cascade also prevents inhibition of flagella motor function by YcgR, a PilZ domain protein that is activated upon c-di-GMP binding [21,22].

Figure 1. Motility and biofilm control networks in *E. coli*. (A) Transcriptional network, that controls switching between motility and biofilm formation. (B) sRNA network, that controls motility and biofilm formation at the mRNA level. Regulatory effects within the transcriptional network (grey arrows) are shown only in part due to space constraints. For distinct regulatory motifs and their function within the entire network, see Figure 3. Blue boxes: targets with “hub” mRNAs/proteins; grey rectangles: transcription factors; pentagon: c-di-GMP-binding effector protein; red boxes: c-di-GMP signaling modules (red letters: diguanylate cyclase, blue letters: phosphodiesterase); green ovals: sRNAs; dotted lines: indirect effects; straight lines: direct effects.

(A)
![Diagram A](#)

(B)
![Diagram B](#)

When nutrient availability further declines cells enter into stationary phase, where $\sigma^{S}$ takes over RNAP core enzyme at the expense of $\sigma^{70}$, $\sigma^{F}$ and other sigma subunits [18]. As a result, the transcription of $flhDC$ comes to an end and, due to proteolytic degradation of FlhDC and $\sigma^{F}$, the entire flagellar cascade is shut down, including the expression of the $y hjH$ phosphodiesterase gene [14,23,24]. $\sigma^{S}$ also contributes to a shutdown of flagella activity by driving the expression of the two diguanylate cyclases (DGCs) YegE and YedQ, which generate c-di-GMP that activates YcgR to slow down flagella rotation [14,25]. Furthermore, $\sigma^{S}$ activates the expression of the transcription factor MlrA, which, together with $\sigma^{S}$-RNAP and at temperatures below 30 °C, activates transcription at the $csgD$ promoter [26–28]. This process is further modulated by OmpR and several other transcription factors [29–31]. CsgD (or AgfD, as it is called in *Salmonella*) then drives the synthesis of the biofilm matrix components curli and cellulose [32]. Curli production is directly activated by binding of the transcription factor CsgD to the promoter of the $csgBAC$ operon [33], which encodes the major curli subunit CsgA as well as the nucleator protein CsgB and a periplasmic accessory protein [34,35]. The regulation of cellulose synthesis is subject to additional c-di-GMP control via the CsgD-dependent DGC YaiC (AdrA) and the PDE YoaD, with the cellulose synthase BcsA directly binding c-di-GMP at its PilZ domain [30,32,36–38].

Although flagella *de novo* expression is downregulated when cells induce the formation of curli fibres and cellulose, it should be noted that flagella are important features for the initial attachment of bacterial cells to surfaces during the transition phase [9,10].

### 1.2. Regulation of Other Biofilm Matrix Components: Colanic Acid and PGA

Other components of the *E. coli* biofilm matrix like colanic acid and PGA are not affected by the curli/cellulose control cascade, but also belong to regulons of biofilm-associated transcription factors (Figure 1A). Colanic acid production depends on the two-component phosphorelay system RcsC/RcsD/RcsB, which can sense perturbations in the cell envelope and seems to play a role in biofilm maturation [39,40]. In part by cooperating with various other transcription factors, the phosphorylated response regulator RcsB not only activates the genes involved in colanic acid, but also stimulates the expression of the sRNA RprA and many other target genes [41–45]. In addition, RcsB inhibits $flhDC$ expression and thereby exerts a negative effect on motility [45]. $\sigma^{S}$ modulates the Rcs system by controlling the expression of the small protein YmgB, which stimulates activity of the Rcs system [46,47]. Overall, the Rcs system seems to inversely regulate flagella expression and late biofilm functions.

Interestingly, PGA synthesis seems inversely regulated with curli expression, which is repressed by high NaCl concentration and high temperature [48,49], which allows the bacteria to modify biofilm composition in a changing environment. At the level of translation, the $pgaABCD$ operon is inversely regulated with the $flhDC$ operon (Figure 1B). This control is due to CsrA (carbon storage regulator), a RNA binding protein, which exerts its function by binding to the 5'-UTR of mRNAs, thereby affecting their turnover [50–52]. CsrA levels are high when carbon source availability does not yet limit growth and contributes to flagella expression, at the same time repressing the biofilm matrix component PGA. CsrA also down-regulates the genes for two diguanylate cyclases, YcdT and YdeH [53], which promote PGA synthesis [54]. CsrA is antagonized by two sRNAs, CsrB/CsrC [55] (and see below).

Int. J. Mol. Sci. 2013, 14

2. Regulation of Biofilm Formation by Small Noncoding RNAs

Regulatory small RNAs (sRNAs) are involved in the regulation of almost every physiological process in the bacterial cell. While a major focus of research has been the role of multiple sRNAs in outer membrane protein homeostasis and various stress responses in *E. coli* and *Salmonella* [56–59], recently many sRNAs have been discovered to regulate flagella expression and biofilm formation [60–64]. Those summarized in this review are about 80–200 nt in size and directly interact with mRNA targets [65–67]. Notably, however, this is not the only mode of action of sRNAs, and two sRNAs that interact with the protein CsrA [68] and prevent the latter from binding to several biofilm-relevant mRNAs, will also be shortly mentioned.

Most mRNA-binding sRNAs discovered to date are encoded in *trans* and bind to their target mRNAs either in the 5'-UTR (5'-untranslated region) or in the ORF (open reading frame). Negatively acting sRNAs can interfere with ribosome binding by directly blocking the mRNA region containing the SD (Shine Dalgarno) sequence and they can promote degradation by recruiting RNaseE [65,66]; in addition, even premature transcription termination may be triggered by sRNAs binding to a nascent mRNA [60,69]. Positively acting sRNAs can open inhibitory mRNA secondary structures, that block the SD sequence or they can rearrange the structure of the 5'-UTR in a way that gives ribosomes access to ribosome loading sites further upstream [65,66]. Most of these sRNAs require the RNA-binding protein Hfq as a cofactor, which influences stability and promotes interactions between sRNAs and target mRNAs [70].

### 2.1. sRNA-Mediated Regulation of the Flagellar Cascade—flhDC mRNA as a Major Site for sRNA Action

Recent work by the groups of Storz and Gottesman has established the *flhDC* operon, which encodes the master regulator of flagella expression in *E. coli* and related bacteria, as a new direct target for several known and one newly characterized sRNA(s) (Figure 1B and [61,62]). McfS (multi-cellular adhesive sRNA) directly activates the *flhDC* mRNA by binding to two regions in the *flhD* 5'-UTR [61]. McfS is part of the CRP regulon that controls carbon metabolism [61]. The *flhD* 5'-UTR is predicted to fold into a long inhibitory stem-loop structure that sequesters the SD sequence by a mechanism similar to that established for *rpoS* mRNA (Figure 2A,B). McfS seems to open this structure in order to increase accessibility for ribosomes [61]. The second sRNA, whose overexpression increases swimming in soft agar plates is MicA (regulator of *ompA* mRNA) [62,71], which down-regulates several outer membrane porins and other targets in a σ<sup>E</sup>-dependent fashion in *E. coli* and *Salmonella* [59,72,73]. The direct target of MicA, that mediates activation of swimming has not yet been identified [62].

Figure 2. “hub” mRNAs—secondary structures of relevant 5'UTR parts and sRNA binding sites. (A) Structure prediction of *flhD* 5'UTR [61]. (B) Predicted and mapped structure of *rpoS* 5'UTR [74] (C) Predicted and mapped structure of *csgD* 5'UTR [75]. sRNA binding sites were computationally predicted and proven by compensatory basepair exchanges (all except GcvB) [41, 60–62, 64, 75–77] or mapped [64, 75]. Positions on the mRNA sequence are numbered according to the transcriptional start site. Bold nucleotides: coding region; black box: start codon; red box: Shine Dalgarno Sequence; colored bars: sRNA binding sites; blue nucleotides: unfolded stem loop structures.

(A) flhD mRNA

![Image Description](#)

(B) rpoS mRNA

![Image Description](#)

(C) csgD mRNA

![Image Description](#)

De Lay and Gottesman found six sRNAs to downregulate *flhDC* expression, thereby inhibiting flagella production, namely ArcZ, OmrA, OmrB, OxyS, SdsR and GadY (Figure 1B and [62]). The first four of these sRNAs act by direct basepairing upstream and partially overlapping the SD sequence, as demonstrated by the introduction of compensatory basepair exchanges, and it is likely, that the sRNAs interfere with ribosome binding ([62] and Figure 2A). ArcZ shares an additional binding site with McsS further upstream [61,62]. ArcZ (for *arc*-associated sRNA Z) controls app. Sixteen percent of the genes in the *Salmonella* genome and an *arcZ* deletion was shown previously to increase swimming [63,78]. ArcZ is negatively controlled by the ArcB/ArcA two component system [76] and was recently shown to indirectly stimulate biofilm formation in *Salmonella* [63]. ArcZ also stimulates the accumulation of $\sigma^S$ [76], which in turn activates the expression of curli fibres and cellulose (see above), and—via sigma factor competition for the limiting cellular amount of RNAP core enzyme—may contribute to indirect downregulation of $\sigma^{70}/\sigma^F$-mediated flagella expression. The highly similar sRNAs OmrA and OmrB (OmpR regulated sRNAs A and B), which are transcribed from tandem genes, regulate a set of outer membrane porins and also influence biofilm formation by inhibiting translation of *csgD* mRNA (see below and [75,79]). OxyS (oxidative stress-related) is induced by hydrogen peroxide stress via the OxyR regulator [80,81] and was recently also demonstrated to be involved in cross-species regulation by downregulating a mRNA target of its eukaryotic predator *C. elegans* [82]. Two additional sRNAs, SdsR and GadY, were found to inhibit *flhDC*, but their modes of action have not yet been clarified [62]. SdsR (sigma S-dependent sRNA) regulates the porin OmpD in *Salmonella* and was also reported to be involved in the control of biofilm formation by activating *csgD* independently of $\sigma^S$ and negatively affecting $\sigma^S$ expression in *Salmonella* [63,71]. GadY (*gad* gene-related sRNA) requires $\sigma^S$ for expression and is a small anti-sense RNA overlapping the 3'-end of the *gadX* gene, which encodes a major acid tolerance regulator. GadY acts as a positive regulator of acid stress response genes by basepairing to the *gadXW* mRNA, thereby recruiting RNase III to cleave between *gadX* and *gadW*, which results in stabilization of both monocistronic derivatives of the *gadXW* mRNA [83–85].

Furthermore, cells entering into stationary phase induce the sRNAs CsrB/CsrC, which sequester and thus inactivate CsrA [55,86,87]. Since the RNA-binding protein CsrA activates FlhDC expression and downregulates the expression of the biofilm-related *pga* gene products [50,52] as well as that of several c-di-GMP-producing enzymes that are involved in PGA production [51,53,54], this induction of CsrB/CsrC may be crucial for throwing the switch from the production of flagella to that of the biofilm matrix component PGA [68]. In summary, at least ten sRNAs previously known to be involved in the regulation of very diverse physiological functions in the cell have now been characterized as regulators of flagella production and motility. The large majority of these sRNAs down-regulate flagella synthesis by directly binding to *flhDC* mRNA. Since flagella are not only essential for motility but also contribute to the initial attachment to surfaces, these sRNAs can also influence biofilm formation. Moreover, several of these sRNAs have additional and more direct effects on the expression of distinct biofilm components as detailed in the following.

Int. J. Mol. Sci. 2013, 14

2.2. sRNA-Mediated Regulation of the Curli Control Cascade—rpoS, ydaM and csgD mRNAs as sRNA Targets

sRNA control in the curli control cascade (Figure 1B) occurs in complex arrangements with multiple effects by distinct sRNAs on transcriptional and other regulators. In this system rpoS and csgD serve as “hubs” for signal integration, because in addition to being under multiple transcriptional control, their mRNAs—like that of flhDC—are regulated by several sRNAs each [88]. The regulation of σ<sup>S</sup> and CsgD levels has far-reaching but different physiological consequences. While σ<sup>S</sup> is a global master regulator, which affects more than 500 stationary phase and stress-induced genes in E. coli [89], the regulator CsgD, which is itself dependent on σ<sup>S</sup> for expression, controls a more specific set of genes with important functions in biofilm formation [33,36,60].

σ<sup>S</sup> expression is positively controlled by the three sRNAs DsrA, RprA and ArcZ, which bind to overlapping sites in the 5'-UTR of rpoS mRNA, thereby opening an inhibitory structure, that buries the SD sequence (Figure 2B) [41,76,77]. All of these sRNAs require the presence of the RNA chaperone Hfq to act on rpoS mRNA, although to different degrees [90]. DsrA (downstream region of rcsA) was initially discovered as an antagonist of H-NS-dependent transcriptional repression. Its increased expression at low temperature (25 °C) seems to contribute to increased levels of σ<sup>S</sup> under these conditions [91,92]. DsrA also directly interferes with the expression of H-NS [93], which may produce indirect and inverse effects on σ<sup>S</sup> and FlhDC, since H-NS downregulates σ<sup>S</sup> [94] and indirectly activates flhDC expression by inhibiting the expression of the flhDC repressor HdfR [95]. The transcription of RprA (RpoS regulator A) is activated by the Rcs phosphorelay system, which also negatively regulates flhDC transcription [45,96] and colanic acid synthesis [39,40,42,97]. Despite its ability to directly bind to rpoS mRNA, the physiological role of RprA in σ<sup>S</sup> control has remained unclear, since RprA has to be overproduced to exert effects on σ<sup>S</sup> levels. By contrast, mutations that eliminate ArcZ significantly reduce σ<sup>S</sup> levels. That ArcZ expression is under negative control of the ArcB/ArcA system and in a feedback cycle also interferes with ArcB expression [76], indirectly contributes to the multiple negative effects of this two-component system on σ<sup>S</sup>. Thus, the response regulator ArcA directly inhibits rpoS transcription and the sensor kinase ArcB also promotes proteolysis of σ<sup>S</sup> by acting as a phosphor donor for the σ<sup>S</sup>-specific proteolytic targeting factor and response regulator, RssB [98]. Via the sensory function of ArcB [99] and signal transduction via ArcA, ArcZ and RssB, σ<sup>S</sup> expression and turnover are linked to the redox state of the respiratory chain, which in turn is determined by the balance between energy source and oxygen supplies [76,98]. A fourth sRNA regulator of σ<sup>S</sup>, OxyS, has an indirect negative effect on σ<sup>S</sup> expression, possibly by sequestering Hfq [81]. In addition, OxyS also inhibits flhDC expression (see above). OxyS responds to oxidative stress and is transcriptionally activated by OxyR [80].

The second “hub” for sRNA-mediated signal integration in the curli/cellulose control cascade is csgD mRNA, which is a direct target for the sRNAs Mcas, RprA and Omra/OmrB [60,61,64,75,88]. All these sRNAs as well as GcvB sRNA negatively affect CsgD expression, but for the latter a direct interaction with csgD mRNA has not yet been demonstrated [64]. A structural hallmark of csgD mRNA is the ability to form a relatively long and stable stem-loop (SL) structure in its 5'-UTR [75] (Figure 2C). Mcas has binding sites upstream of this SL as well as at a downstream position not yet overlapping the SD, but despite its ability to inhibit 30S binding in vitro, the latter interaction had no

relevance for in-vivo downregulation of *csgD* [64]. Since McaS seems to be degraded together with *csgD* mRNA, it seems likely that its direct role is to induce degradation rather than inhibition of ribosome binding to *csgD* mRNA [64]. RprA binds to a position overlapping with the first McaS binding site upstream of the SL as well as to a second position overlapping the SD sequence (with both binding sites actually also partially overlapping the predicted but not yet experimentally confirmed binding site for GcvB) [60, 64]. When present in excess, RprA interferes with *csgD* translation and efficiently downregulates *csgD* mRNA levels by a mechanism that may involve increased *csgD* mRNA turnover and/or premature termination of transcription [60]. In addition, RprA can indirectly inhibit transcription of *csgD* as it also binds to the beginning of the coding sequence of *ydaM* mRNA thereby downregulating the expression of the DGC YdaM, which is required for *csgD* transcription [60]—this is actually the first report on a sRNA directly controlling the expression of a c-di-GMP-generating enzyme. The recognition site for OmrA and OmrB is located within the SL region of *csgD* mRNA [75]. Expression of OmrA/OmrB is activated by the EnvZ/OmpR system in response to changes in osmolarity [79]. OmrA and OmrB are identical at their 5'-ends but differ after the first 21 nucleotides [100]. By binding to the SL region of *csgD* mRNA, they prevent ribosome binding from a distance suggesting that they block a ribosome loading site located further upstream of the actual translation start site [75]. In contrast to McaS and RprA, OmrA/OmrB do not seem to reduce *csgD* mRNA levels [75]. However, like RprA, OmrB was found to downregulate also the expression of a *ydaM::gfp* fusion when overproduced and binding site predictions indicate OmrA/OmrB may bind overlapping with the RprA binding site in *ydaM* mRNA, suggesting that not only RprA but also OmrA/OmrB indirectly inhibits *csgD* transcription via the diguanylate cyclase YdaM [101].

Taken together, McaS, RprA, and OmrA/OmrB all downregulate *csgD* expression by binding to partially overlapping regions in the 5'-region of *csgD* mRNA, but the details of the molecular mechanisms involved seem to be different. In conclusion, a total of at least seven sRNAs, *i.e.*, DsrA, ArcZ, RprA, McaS, OmrA, OmrB, and GcvB, can influence the biosynthesis of the biofilm matrix components curli and cellulose by either affecting σ<sup>S</sup> regulation or by downregulating the key biofilm regulator CsgD.

## 3. sRNAs Contribute to Inverse Regulation of Flagella and Biofilm Components in Different Regulatory Patterns

As described above, the timing of flagella and curli/cellulose expression is under complex transcriptional control. sRNAs can contribute to the decision whether flagella or matrix components are produced and provide an additional regulatory layer to the inverse regulation of these components. While some sRNAs seem to exert their regulatory roles under “housekeeping” or non-stress conditions, some sRNAs are highly induced upon exposure to stresses such as high osmolarity (OmrA/B), cell envelope perturbation (MicA, RprA) or oxidative stress (OxyS), suggesting that they mainly contribute to controlling their targets under these specific conditions. As an outcome, biofilm matrix production is integrated with multiple stress responses. For instance, in response to severe cell surface stress sensed by the Rcs system, RprA could further induce the general stress response master regulator σ<sup>S</sup> and at the same time shut down the expression of the cell envelope-located and energy-consuming machinery for the production of biofilm matrix components.

sRNAs are key players in several types of regulatory ‘motifs’, that generate distinct physiological outputs (Figure 3): Group I sRNAs stimulate biofilm functions and down-regulate flagella; Group II sRNAs have the opposite impact, *i.e.*, stimulate flagella expression, but interfere with the synthesis of biofilm matrix; Group III sRNAs downregulate both functions; and finally Group IV is defined by sRNAs with multiple and more complex roles in the control of flagella and biofilm functions.

**Figure 3.** Regulatory sRNA control modules can be grouped into 4 classes according to their regulatory output with respect to expression of flagella and biofilm components. For an explanation of the symbols, see legend of Figure 1. Grey arrows: shown in Salmonella only.

| I anti flagella/ pro biofilm |  |
| --- | --- |
| flagella — sRNA → biofilm matrix | $\downarrow$ ArcB/A HdfR $\uparrow$ HNS $\downarrow$ FlhDC $\rightarrow$ ArcZ $\rightarrow$ $\sigma^S$ FlhDC DsrA $\rightarrow$ $\sigma^S$ type I fimbriae CsgD CsgD |

| II pro flagella/ anti biofilm |  |
| --- | --- |
| flagella ← sRNA — biofilm matrix | $\downarrow$ CRP $\downarrow$ FlhDC $\leftarrow$ McaS $\leftarrow$ $\sigma^S$ $\downarrow$ CsgD pgaA |

| III anti flagella/ anti biofilm |  |
| --- | --- |
| flagella — sRNA — biofilm matrix | $\downarrow$ OmpR Rcs $\uparrow$ AG43 $\downarrow$ FlhDC $\rightarrow$ OmrA/B $\sigma^S$ $\downarrow$ FlihDC OxyR $\downarrow$ CsgD $\leftarrow$ YdaM CsgD |

| IV complex multiple effects |  |
| --- | --- |
|  | $\downarrow$ Rcs $\leftarrow$ YmgB $\downarrow$ FlhDC $\rightarrow$ RprA $\rightarrow$ $\sigma^S$ $\downarrow$ CsgD $\leftarrow$ YdaM |

A major representative of Group I is clearly ArcZ, which binds to the *flhDC* 5'-UTR and downregulates *flhDC* [62], as well as type I fimbriae in *Salmonella* [63,78]. Thereby it counteracts motility and/or initial attachment. On the other side it activates *rpoS* translation [76], which can contribute to *csgD* expression; in addition, ArcZ seems to have a $\sigma^S$-independent activating effect on CsgD in *Salmonella* [63]. ArcZ and $\sigma^S$ are under negative control by the ArcB/ArcA two-component

system and thereby linked to the redox state of the respiratory chain [76,98]. In addition, DsrA may fall into Group I sRNAs, as it directly stimulates translation of *rpoS* mRNA, whereas it interferes with the expression of H-NS, which—by repressing the expression of the *flhDC* repressor HdfR—acts as an indirect activator of FlhDC expression. In addition, GadY can be seen as a Group I representative, since it down-regulates FlhDC in an indirect manner [62], while also positively controlling the expression of GadX and thereby acid stress response genes [83,85], which are clearly relevant in a mature biofilm that contains zones of low oxygen content where acid-generating fermentation probably prevails over respiration.

Group II is exemplified by McsS, which activates flagella by direct interaction with *flhDC* mRNA and on the other side directly inhibits CsgD expression [61,64]. At the transcriptional level, the expression of FlhDC as well as of McsS is activated by cAMP-CRP, *i.e.*, when nutrient availability is suboptimal and cells are in the post-exponential phase of the growth cycle. Since the effects of McsS on biofilm formation seem most prominent at 37 °C and it also activates PGA production [61], McsS could play a role in pathogenic *E. coli*.

Group III is represented by OmrA/B, which by binding directly to *flhDC* and *csgD* mRNA [62,75], can down-regulate both motility and the expression of CsgD-controlled curli fibres and cellulose. FlhDC, OmrA/B and CsgD are all under complex control of the EnvZ/OmpR two-component system, which senses changes in osmolarity. Thus, CsgD expression is stimulated by OmpR at low osmolarity but is inhibited at high osmolarity [30,48,102], with the latter conditions inducing OmrA/B [79] as well as σ<sup>S</sup> [103]. OmrA/B may thus further support OmpR-mediated repression of flagella synthesis and at the same time contribute to CsgD not being expressed despite the strong accumulation of σ<sup>S</sup> under high osmolarity conditions. In addition, OmpR is a direct target of OmrA/B in a possibly homeostatic negative feedback loop. Also the oxidative stress-induced OxyS may be an example of a Group III sRNAs, as it turns down flagella expression by direct interaction [62] and has indirect negative effects on σ<sup>S</sup> expression [104] and therefore possibly on biofilm functions.

Finally, the Group IV paradigm is RprA. RprA directly downregulates *csgD* mRNA when overexpressed, but also has indirect and multiple regulatory influences on the two cascades. As an activator of σ<sup>S</sup>, it may stimulate the induction of σ<sup>S</sup>-dependent genes under yet unknown conditions. Moreover, RprA can inhibit the expression of the diguanylate cyclase YdaM and thereby can interfere indirectly with *csgD* transcription. By activating the expression of RprA and repressing that of FlhDC, the Rcs system is an important additional player in this circuitry. To make things even more complex, RprA itself may be targeted by *csgD* mRNA which—when present in sufficient amounts—can block RprA activity of other mRNAs [60]. Finally, RprA and the Rcs system have been implicated in biofilm maturation [40], which probably involves targets in addition to FlhDC or CsgD.

In conclusion, sRNAs of Group I and II are clearly involved in the mutually exclusive expression of flagella and biofilm matrix components. By adding a RNA-based layer of regulation to the primary transcriptional control, they probably contribute to robustness of switching between these functions, which are associated with post-exponential growth and stationary phase, respectively. The overall function of Group III sRNA seems a shut-down of expression of large structures within the cell envelope—no matter, whether these are involved in motility or biofilm functions. This pattern may be particularly beneficial under conditions of sudden cell envelope stress. In addition, it could generate a

temporal pattern of biofilm gene expression, for instance if a sRNA down-regulates flagella and early, but not late biofilm functions.

# 4. Species-Specific Variations in the Regulation by sRNAs

Overall, the transcriptional network that coordinates flagella and curli/cellulose expression is generally conserved between *E. coli* and *Salmonella*. Yet, although many of the sRNAs in this regulatory network are found in both species as well (except McaS which is absent from the *Salmonella* genome), it should be noted, that their target sequences on mRNAs are less well conserved. The *flhDC* 5'-UTR is in fact rather different in *Salmonella* and *E. coli*; also, the *csgD* 5'-UTR is not well conserved, although the prominent stem-loop region described above for *E. coli* exists also in *Salmonella*. Recently it has been reported, that *E. coli* RprA cannot regulate *csgD::gfp* from *Salmonella* and vice versa [105]. As shown in numerous compensatory basepair experiments, the introduction of even a single mismatch in a sRNA/mRNA duplex structure can completely abolish interaction. This suggests that small but consequential variations in sRNA/mRNA regulation can evolve very easily. Thus, despite an overall similarity of sRNAs and relevant mRNAs, the contributions of the sRNA network to motility/biofilm regulation may differ and will have to be elucidated in each species separately.

# 5. Conclusions and Perspectives

In the inverse regulation of flagella synthesis and biofilm functions, communicating transcription factor cascades (Figure 1A) set the stage, on which sRNAs can then fine-tune and integrate additional environmental signals in the actual rates of synthesis of key protein players (Figures 1B and 3). Moreover, sRNAs increase the dynamics of responses, because they skip the time-consuming step of protein biosynthesis and in generally are subject to rapid turnover, which allows for switching responses on and off very rapidly. For enteric bacteria like *E. coli*, which must adapt as fast as possible when moving from a host to a rather unpredictable outside environment and back, this may provide a crucial fitness advantage.

In addition, the flagella synthesis and biofilm control system demonstrates that a sRNA and its “target” mRNA does not act as an isolated pair, but rather represent the smallest functional unit in larger RNA-based networks. With overlapping binding sites for several sRNAs in relatively small regions of target mRNAs (e.g., the 5'-UTR of *csgD* mRNA consists of 148 nucleotides only), the regulatory outcome depends on rates of expression and the levels of the sRNAs and the mRNA, the relative affinities of the sRNAs for the binding sites on the mRNA and the action of the RNA chaperone Hfq on all the RNAs involved. When sRNAs are present in excess over the mRNA, a hierarchy of signals transmitted by sRNA based on different affinities can be established. In cases where the mRNA is present in excess over the sRNAs, several signals can be integrated at the same time; in addition, the excess mRNA may sequester sRNAs and prevent them from also acting on other mRNAs.

These considerations are likely to be relevant for *csgD* mRNA and the sRNAs able to bind to it, since *csgD* expression is sharply and very strongly turned on under very specific conditions, i.e., when cells grown below 30 °C enter into stationary phase. In contrast, the expression of the sRNAs is less

drastically regulated [14,60]. Thus, highly expressed csgD mRNA may actually regulate other actions of the sRNAs it can bind to. Following the same logics, a large RNA mimicking a physiological mRNA target was recently found to sequester and thereby inactivate the sRNA MicM (ChiX) [106,107]. These examples suggest that the current narrow perspective of “sRNAs regulate target mRNAs” should be replaced by a concept of non-hierarchical mutual control [60]. As a consequence, larger non-hierarchical RNA-based networks can be envisioned, in which distinct sRNAs contact specific sets of mRNAs, each of which in turn can contact several sRNAs. Depending on competition for binding sites and/or sequestration, effects of a sudden increase or decrease of any of the sRNA or mRNA could then propagate within the network and influence several outputs.

With the mRNAs of three transcriptional master regulators acting as RNA network “hubs” interconnected by many sRNAs, which in turn make connections to numerous other mRNAs, the FlhDC/σS/CsgD network seems an ideal case to study (i) the regulatory behavior of a non-hierarchical RNA-based network; (ii) its integration with the hierarchical transcription factor network that provides the regulatory frame, and finally; (iii) its interlinkage with complex control by the nucleotide second messenger c-di-GMP, which in fact is a tiny RNA in its own right.

## Acknowledgments

Biofilm-related research in the Hengge laboratory has been funded by the Deutsche Forschungsgemeinschaft (He 1556/13, He 1556/14) and the European Research Council under the European Unions Seventh Framework Programme (ERC-AdG 249780 to RH).

## Conflict of Interest

The authors declare no conflict of interest.

## References

1. Karatan, E.; Watnick, P. Signals, regulatory networks, and materials that build and break bacterial biofilms. *Microbiol. Mol. Biol. Rev.* **2009**, *73*, 310–347.
2. Flemming, H.C.; Wingender, J. The biofilm matrix. *Nat. Rev. Microbiol.* **2010**, *8*, 623–633.
3. Branda, S.S.; Vik, S.; Friedman, L.; Kolter, R. Biofilms: The matrix revisited. *Trends Microbiol.* **2005**, *13*, 20–26.
4. Anderson, G.G.; O'Toole, G.A. Innate and induced resistance mechanisms of bacterial biofilms. *Curr. Top. Microbiol. Immunol.* **2008**, *322*, 85–105.
5. Danese, P.N.; Pratt, L.A.; Dove, S.L.; Kolter, R. The outer membrane protein, antigen 43, mediates cell-to-cell interactions within Escherichia coli biofilms. *Mol. Microbiol.* **2000**, *37*, 424–432.
6. Agladze, K.; Wang, X.; Romeo, T. Spatial periodicity of Escherichia coli K-12 biofilm microstructure initiates during a reversible, polar attachment phase of development and requires the polysaccharide adhesin PGA. *J. Bacteriol.* **2005**, *187*, 8237–8246.

7. Korea, C.G.; Ghigo, J.M.; Beloin, C. The sweet connection: Solving the riddle of multiple sugar-binding fimbrial adhesins in *Escherichia coli*: Multiple *E. coli* fimbriae form a versatile arsenal of sugar-binding lectins potentially involved in surface-colonisation and tissue tropism. *Bioessays* 2011, 33, 300–311.

8. Beloin, C.; Roux, A.; Ghigo, J.M. *Escherichia coli* biofilms. *Curr. Top. Microbiol. Immunol.* 2008, 322, 249–289.

9. Pratt, L.A.; Kolter, R. Genetic analysis of *Escherichia coli* biofilm formation: Roles of flagella, motility, chemotaxis and type I pili. *Mol. Microbiol.* 1998, 30, 285–293.

10. Wood, T.K.; Gonzalez Barrios, A.F.; Herzberg, M.; Lee, J. Motility influences biofilm architecture in *Escherichia coli*. *Appl. Microbiol. Biotechnol.* 2006, 72, 361–367.

11. Da Re, S.; Ghigo, J.M. A CsgD-independent pathway for cellulose production and biofilm formation in *Escherichia coli*. *J. Bacteriol.* 2006, 188, 3073–3087.

12. Bokranz, W.; Wang, X.; Tschape, H.; Romling, U. Expression of cellulose and curli fimbriae by *Escherichia coli* isolated from the gastrointestinal tract. *J. Med. Microbiol.* 2005, 54, 1171–1182.

13. Zogaj, X.; Nimtz, M.; Rohde, M.; Bokranz, W.; Romling, U. The multicellular morphotypes of *Salmonella typhimurium* and *Escherichia coli* produce cellulose as the second component of the extracellular matrix. *Mol. Microbiol.* 2001, 39, 1452–1463.

14. Pesavento, C.; Becker, G.; Sommerfeldt, N.; Possling, A.; Tschowri, N.; Mehlis, A.; Hengge, R. Inverse regulatory coordination of motility and curli-mediated adhesion in *Escherichia coli*. *Genes Dev.* 2008, 22, 2434–2446.

15. Pesavento, C.; Hengge, R. The global repressor FliZ antagonizes gene expression by sigmaS-containing RNA polymerase due to overlapping DNA binding specificity. *Nucleic Acids Res.* 2012, 40, 4783–4793.

16. Zhao, K.; Liu, M.; Burgess, R.R. Adaptation in bacterial flagellar and motility systems: From regulon members to “foraging”-like behavior in *E. coli*. *Nucleic Acids Res.* 2007, 35, 4441–4452.

17. Chevance, F.F.; Hughes, K.T. Coordinating assembly of a bacterial macromolecular machine. *Nat. Rev. Microbiol.* 2008, 6, 455–465.

18. Hengge, R. The General Stress Response in Gram-Negative Bacteria. In *Bacterial Stress Responses*; Storz, G., Hengge, R., Eds.; ASM Press: Washington, DC, USA, 2011; pp. 251–289.

19. Jenal, U.; Malone, J. Mechanisms of cyclic-di-GMP signaling in bacteria. *Annu. Rev. Genet.* 2006, 40, 385–407.

20. Hengge, R. Principles of c-di-GMP signalling in bacteria. *Nat. Rev. Microbiol.* 2009, 7, 263–273.

21. Ryjenkov, D.A.; Simm, R.; Roemling, U.; Gomelsky, M. The PilZ domain is a receptor for the second messenger c-di-GMP: The PilZ domain protein YcgR controls motility in enterobacteria. *J. Biol. Chem.* 2006, 281, 30310–30314.

22. Paul, K.; Nieto, V.; Carlquist, W.C.; Blair, D.F.; Harshey, R.M. The c-di-GMP binding protein YcgR controls flagellar motor direction and speed to affect chemotaxis by a “backstop brake” mechanism. *Mol. Cell* 2010, 38, 128–139.

23. Barembruch, C.; Hengge, R. Cellular levels and activity of the flagellar sigma factor FliA of *Escherichia coli* are controlled by FlgM-modulated proteolysis. *Mol. Microbiol.* 2007, 65, 76–89.

24. Tomoyasu, T.; Ohkishi, T.; Ukyo, Y.; Tokumitsu, A.; Takaya, A.; Suzuki, M.; Sekiya, K.; Matsui, H.; Kutsukake, K.; Yamamoto, T. The ClpXP ATP-dependent protease regulates flagellum synthesis in *Salmonella enterica* serovar typhimurium. *J. Bacteriol.* 2002, 184, 645–653.

25. Boehm, A.; Kaiser, M.; Li, H.; Spangler, C.; Kasper, C.A.; Ackermann, M.; Kaever, V.; Sourjik, V.; Roth, V.; Jenal, U. Second messenger-mediated adjustment of bacterial swimming velocity. *Cell* 2010, 141, 107–116.

26. Brown, P.K.; Dozois, C.M.; Nickerson, C.A.; Zuppardo, A.; Terlonge, J.; Curtiss, R., 3rd. MlrA, a novel regulator of curli (AgF) and extracellular matrix synthesis by *Escherichia coli* and *Salmonella enterica* serovar Typhimurium. *Mol. Microbiol.* 2001, 41, 349–363.

27. Weber, H.; Pesavento, C.; Possling, A.; Tischendorf, G.; Hengge, R. Cyclic-di-GMP-mediated signalling within the sigma network of *Escherichia coli*. *Mol. Microbiol.* 2006, 62, 1014–1034.

28. Ogasawara, H.; Yamamoto, K.; Ishihama, A. Regulatory role of MlrA in transcription activation of *csgD*, the master regulator of biofilm formation in *Escherichia coli*. *FEMS Microbiol. Lett.* 2010, 312, 160–168.

29. Ogasawara, H.; Yamada, K.; Kori, A.; Yamamoto, K.; Ishihama, A. Regulation of the *Escherichia coli csgD* promoter: Interplay between five transcription factors. *Microbiology* 2010, 156, 2470–2483.

30. Prigent-Combaret, C.; Brombacher, E.; Vidal, O.; Ambert, A.; Lejeune, P.; Landini, P.; Dorel, C. Complex regulatory network controls initial adhesion and biofilm formation in *Escherichia coli* via regulation of the *csgD* gene. *J. Bacteriol.* 2001, 183, 7213–7223.

31. Gerstel, U.; Park, C.; Romling, U. Complex regulation of *csgD* promoter activity by global regulatory proteins. *Mol. Microbiol.* 2003, 49, 639–654.

32. Roemling, U.; Rohde, M.; Olsen, A.; Normark, S.; Reinkoster, J. AgfD, the checkpoint of multicellular and aggregative behaviour in *Salmonella typhimurium* regulates at least two independent pathways. *Mol. Microbiol.* 2000, 36, 10–23.

33. Ogasawara, H.; Yamamoto, K.; Ishihama, A. Role of the biofilm master regulator CsgD in cross-regulation between biofilm formation and flagellar synthesis. *J. Bacteriol.* 2011, 193, 2587–2597.

34. Barnhart, M.M.; Chapman, M.R. Curli biogenesis and function. *Annu. Rev. Microbiol.* 2006, 60, 131–147.

35. Blanco, L.P.; Evans, M.L.; Smith, D.R.; Badtke, M.P.; Chapman, M.R. Diversity, biogenesis and function of microbial amyloids. *Trends Microbiol.* 2012, 20, 66–73.

36. Brombacher, E.; Dorel, C.; Zehnder, A.J.; Landini, P. The curli biosynthesis regulator CsgD co-ordinates the expression of both positive and negative determinants for biofilm formation in *Escherichia coli*. *Microbiology* 2003, 149, 2847–2857.

37. Roemling, U.; Sierralta, W.D.; Eriksson, K.; Normark, S. Multicellular and aggregative behaviour of *Salmonella typhimurium* strains is controlled by mutations in the *agfD* promoter. *Mol. Microbiol.* 1998, 28, 249–264.

38. Amikam, D.; Galperin, M.Y. PilZ domain is part of the bacterial c-di-GMP binding protein. *Bioinformatics* 2006, 22, 3–6.

39. Stout, V. Identification of the promoter region for the colanic acid polysaccharide biosynthetic genes in *Escherichia coli* K-12. *J. Bacteriol.* 1996, 178, 4273–4280.
40. Majdalani, N.; Gottesman, S. The Rcs phosphorelay: A complex signal transduction system. *Annu. Rev. Microbiol.* 2005, 59, 379–405.
41. Majdalani, N.; Hernandez, D.; Gottesman, S. Regulation and mode of action of the second small RNA activator of RpoS translation, RprA. *Mol. Microbiol.* 2002, 46, 813–826.
42. Ferrieres, L.; Clarke, D.J. The RcsC sensor kinase is required for normal biofilm formation in *Escherichia coli* K-12 and controls the expression of a regulon in response to growth on a solid surface. *Mol. Microbiol.* 2003, 50, 1665–1682.
43. Castanie-Cornet, M.P.; Cam, K.; Bastiat, B.; Cros, A.; Bordes, P.; Gutierrez, C. Acid stress response in *Escherichia coli*: Mechanism of regulation of *gadA* transcription by RcsB and GadE. *Nucleic Acids Res.* 2010, 38, 3546–3554.
44. Hagiwara, D.; Sugiura, M.; Oshima, T.; Mori, H.; Aiba, H.; Yamashino, T.; Mizuno, T. Genome-wide analyses revealing a signaling network of the RcsC-YojN-RcsB phosphorelay system in *Escherichia coli*. *J. Bacteriol.* 2003, 185, 5735–5746.
45. Francez-Charlot, A.; Laugel, B.; van Gemert, A.; Dubarry, N.; Wiorowski, F.; Castanie-Cornet, M.P.; Gutierrez, C.; Cam, K. RcsCDB His-Asp phosphorelay system negatively regulates the *flhDC* operon in *Escherichia coli*. *Mol. Microbiol.* 2003, 49, 823–832.
46. Tschowri, N.; Busse, S.; Hengge, R. The BLUF-EAL protein YcgF acts as a direct anti-repressor in a blue-light response of *Escherichia coli*. *Genes Dev.* 2009, 23, 522–534.
47. Tschowri, N.; Lindenberg, S.; Hengge, R. Molecular function and potential evolution of the biofilm-modulating blue light-signalling pathway of *Escherichia coli*. *Mol. Microbiol.* 2012, 85, 893–906.
48. Jubelin, G.; Vianney, A.; Beloin, C.; Ghigo, J.M.; Lazzaroni, J.C.; Lejeune, P.; Dorel, C. CpxR/OmpR interplay regulates curli gene expression in response to osmolarity in *Escherichia coli*. *J. Bacteriol.* 2005, 187, 2038–2049.
49. Dorel, C.; Lejeune, P.; Rodrigue, A. The Cpx system of *Escherichia coli*, a strategic signaling pathway for confronting adverse conditions and for settling biofilm communities? *Res. Microbiol.* 2006, 157, 306–314.
50. Wei, B.L.; Brun-Zinkernagel, A.M.; Simecka, J.W.; Pruss, B.M.; Babitzke, P.; Romeo, T. Positive regulation of motility and *flhDC* expression by the RNA-binding protein CsrA of *Escherichia coli*. *Mol. Microbiol.* 2001, 40, 245–256.
51. Jonas, K.; Edwards, A.N.; Ahmad, I.; Romeo, T.; Romling, U.; Melefors, O. Complex regulatory network encompassing the Csr, c-di-GMP and motility systems of *Salmonella Typhimurium*. *Environ. Microbiol.* 2010, 12, 524–540.
52. Wang, X.; Dubey, A.K.; Suzuki, K.; Baker, C.S.; Babitzke, P.; Romeo, T. CsrA post-transcriptionally represses *pgaABCD*, responsible for synthesis of a biofilm polysaccharide adhesin of *Escherichia coli*. *Mol. Microbiol.* 2005, 56, 1648–1663.
53. Jonas, K.; Edwards, A.N.; Simm, R.; Romeo, T.; Romling, U.; Melefors, O. The RNA binding protein CsrA controls cyclic di-GMP metabolism by directly regulating the expression of GGDEF proteins. *Mol. Microbiol.* 2008, 70, 236–257.

54. Boehm, A.; Steiner, S.; Zaehringer, F.; Casanova, A.; Hamburger, F.; Ritz, D.; Keck, W.; Ackermann, M.; Schirmer, T.; Jenal, U. Second messenger signalling governs *Escherichia coli* biofilm induction upon ribosomal stress. *Mol. Microbiol.* 2009, 72, 1500–1516.

55. Jonas, K.; Melefors, O. The *Escherichia coli* CsrB and CsrC small RNAs are strongly induced during growth in nutrient-poor medium. *FEMS Microbiol. Lett.* 2009, 297, 80–86.

56. Froehlich, K.S.; Papenfort, K.; Berger, A.A.; Vogel, J. A conserved RpoS-dependent small RNA controls the synthesis of major porin OmpD. *Nucleic Acids Res.* 2012, 40, 3623–3640.

57. Benjamin, J.A.; Desnoyers, G.; Morissette, A.; Salvail, H.; Masse, E. Dealing with oxidative stress and iron starvation in microorganisms: An overview. *Can. J. Physiol. Pharmacol.* 2010, 88, 264–272.

58. Vogel, J.; Papenfort, K. Small non-coding RNAs and the bacterial outer membrane. *Curr. Opin. Microbiol.* 2006, 9, 605–611.

59. Gogol, E.B.; Rhodius, V.A.; Papenfort, K.; Vogel, J.; Gross, C.A. Small RNAs endow a transcriptional activator with essential repressor functions for single-tier control of a global stress regulon. *Proc. Natl. Acad. Sci. USA* 2011, 108, 12875–12880.

60. Mika, F.; Busse, S.; Possling, A.; Berkholz, J.; Tschowri, N.; Sommerfeldt, N.; Pruteanu, M.; Hengge, R. Targeting of *csgD* by the small regulatory RNA RprA links stationary phase, biofilm formation and cell envelope stress in *Escherichia coli*. *Mol. Microbiol.* 2012, 84, 51–65.

61. Thomason, M.K.; Fontaine, F.; de Lay, N.; Storz, G. A small RNA that regulates motility and biofilm formation in response to changes in nutrient availability in *Escherichia coli*. *Mol. Microbiol.* 2012, 84, 17–35.

62. De Lay, N.; Gottesman, S. A complex network of small non-coding RNAs regulate motility in *Escherichia coli*. *Mol. Microbiol.* 2012, 86, 524–538.

63. Monteiro, C.; Papenfort, K.; Hentrich, K.; Ahmad, I.; le Guyon, S.; Reimann, R.; Grantcharova, N.; Romling, U. Hfq and Hfq-dependent small RNAs are major contributors to multicellular development in *Salmonella enterica* serovar Typhimurium. *RNA Biol.* 2012, 9, 489–502.

64. Jorgensen, M.G.; Nielsen, J.S.; Boysen, A.; Franch, T.; Moller-Jensen, J.; Valentin-Hansen, P. Small regulatory RNAs control the multi-cellular adhesive lifestyle of *Escherichia coli*. *Mol. Microbiol.* 2012, 84, 36–50.

65. Gottesman, S.; Storz, G. Bacterial small RNA regulators: Versatile roles and rapidly evolving variations. *Cold Spring Harb. Perspect. Biol.* 2011, 3, doi:10.1101/cshperspect.a003798.

66. Storz, G.; Vogel, J.; Wassarman, K.M. Regulation by small RNAs in bacteria: Expanding frontiers. *Mol. Cell* 2011, 43, 880–891.

67. Beisel, C.L.; Storz, G. Base pairing small RNAs and their roles in global regulatory networks. *FEMS Microbiol. Rev.* 2010, 34, 866–882.

68. Romeo, T.; Vakulskas, C.A.; Babitzke, P. Post-transcriptional regulation on a global scale: Form and function of Csr/Rsm systems. *Environ. Microbiol.* 2012, 15, 313–324.

69. Bossi, L.; Schwartz, A.; Guillemardet, B.; Boudvillain, M.; Figueroa-Bossi, N. A role for Rho-dependent polarity in gene regulation by a noncoding small RNA. *Genes Dev.* 2012, 26, 1864–1873.

70. Vogel, J.; Luisi, B.F. Hfq and its constellation of RNA. *Nat. Rev. Microbiol.* 2011, 9, 578–589.

Int. J. Mol. Sci. 2013, 14

71. Udekwu, K.I.; Darfeuille, F.; Vogel, J.; Reimegard, J.; Holmqvist, E.; Wagner, E.G. Hfq-dependent regulation of OmpA synthesis is mediated by an antisense RNA. *Genes Dev.* 2005, 19, 2355–2366.

72. Papenfort, K.; Pfeiffer, V.; Mika, F.; Lucchini, S.; Hinton, J.C.; Vogel, J. SigmaE-dependent small RNAs of Salmonella respond to membrane stress by accelerating global omp mRNA decay. *Mol. Microbiol.* 2006, 62, 1674–1688.

73. Johansen, J.; Rasmussen, A.A.; Overgaard, M.; Valentin-Hansen, P. Conserved small non-coding RNAs that belong to the sigmaE regulon: Role in down-regulation of outer membrane proteins. *J. Mol. Biol.* 2006, 364, 1–8.

74. Soper, T.J.; Woodson, S.A. The rpoS mRNA leader recruits Hfq to facilitate annealing with DsrA sRNA. *RNA* 2008, 14, 1907–1917.

75. Holmqvist, E.; Reimegard, J.; Sterk, M.; Grantcharova, N.; Romling, U.; Wagner, E.G. Two antisense RNAs target the transcriptional regulator CsgD to inhibit curli synthesis. *EMBO J.* 2010, 29, 1840–1850.

76. Mandin, P.; Gottesman, S. Integrating anaerobic/aerobic sensing and the general stress response through the ArcZ small RNA. *EMBO J.* 2010, 29, 3094–3107.

77. Majdalani, N.; Cunning, C.; Sledjeski, D.; Elliott, T.; Gottesman, S. DsrA RNA regulates translation of RpoS message by an anti-antisense mechanism, independent of its action as an antisilencer of transcription. *Proc. Natl. Acad. Sci. USA* 1998, 95, 12462–12467.

78. Papenfort, K.; Said, N.; Welsink, T.; Lucchini, S.; Hinton, J.C.; Vogel, J. Specific and pleiotropic patterns of mRNA regulation by ArcZ, a conserved, Hfq-dependent small RNA. *Mol. Microbiol.* 2009, 74, 139–158.

79. Guillier, M.; Gottesman, S. Remodelling of the Escherichia coli outer membrane by two small regulatory RNAs. *Mol. Microbiol.* 2006, 59, 231–247.

80. Altuvia, S.; Weinstein-Fischer, D.; Zhang, A.; Postow, L.; Storz, G. A small, stable RNA induced by oxidative stress: Role as a pleiotropic regulator and antimutator. *Cell* 1997, 90, 43–53.

81. Zhang, A.; Altuvia, S.; Tiwari, A.; Argaman, L.; Hengge-Aronis, R.; Storz, G. The OxyS regulatory RNA represses rpoS translation and binds the Hfq (HF-I) protein. *EMBO J.* 1998, 17, 6061–6068.

82. Liu, H.; Wang, X.; Wang, H.D.; Wu, J.; Ren, J.; Meng, L.; Wu, Q.; Dong, H.; Wu, J.; Kao, T.Y.; et al. Escherichia coli noncoding RNAs can affect gene expression and physiology of Caenorhabditis elegans. *Nat. Commun.* 2012, 3, 1073.

83. Opdyke, J.A.; Kang, J.G.; Storz, G. GadY, a small-RNA regulator of acid response genes in Escherichia coli. *J. Bacteriol.* 2004, 186, 6698–6705.

84. Tramonti, A.; de Canio, M.; de Biase, D. GadX/GadW-dependent regulation of the Escherichia coli acid fitness island: Transcriptional control at the gadY-gadW divergent promoters and identification of four novel 42 bp GadX/GadW-specific binding sites. *Mol. Microbiol.* 2008, 70, 965–982.

85. Opdyke, J.A.; Fozo, E.M.; Hemm, M.R.; Storz, G. RNase III participates in GadY-dependent cleavage of the gadX-gadW mRNA. *J. Mol. Biol.* 2011, 406, 29–43.

86. Liu, M.Y.; Gui, G.; Wei, B.; Preston, J.F., 3rd; Oakford, L.; Yuksel, U.; Giedroc, D.P.; Romeo, T. The RNA molecule CsrB binds to the global regulatory protein CsrA and antagonizes its activity in *Escherichia coli*. *J. Biol. Chem.* **1997**, *272*, 17502–17510.

87. Suzuki, K.; Wang, X.; Weilbacher, T.; Pernestig, A.K.; Melefors, O.; Georgellis, D.; Babitzke, P.; Romeo, T. Regulatory circuitry of the CsrA/CsrB and BarA/UvrY systems of *Escherichia coli*. *J. Bacteriol.* **2002**, *184*, 5130–5140.

88. Boehm, A.; Vogel, J. The *csgD* mRNA as a hub for signal integration via multiple small RNAs. *Mol. Microbiol.* **2012**, *84*, 1–5.

89. Weber, H.; Polen, T.; Heuveling, J.; Wendisch, V.F.; Hengge, R. Genome-wide analysis of the general stress response network in *Escherichia coli*: SigmaS-dependent genes, promoters, and sigma factor selectivity. *J. Bacteriol.* **2005**, *187*, 1591–1603.

90. Soper, T.; Mandin, P.; Majdalani, N.; Gottesman, S.; Woodson, S.A. Positive regulation by small RNAs and the role of Hfq. *Proc. Natl. Acad. Sci. USA* **2010**, *107*, 9602–9607.

91. Sledjeski, D.D.; Gupta, A.; Gottesman, S. The small RNA, DsrA, is essential for the low temperature expression of RpoS during exponential growth in *Escherichia coli*. *EMBO J.* **1996**, *15*, 3993–4000.

92. Sledjeski, D.; Gottesman, S. A small RNA acts as an antisilencer of the H-NS-silenced *rcsA* gene of *Escherichia coli*. *Proc. Natl. Acad. Sci. USA* **1995**, *92*, 2003–2007.

93. Lease, R.A.; Cusick, M.E.; Belfort, M. Riboregulation in *Escherichia coli*: DsrA RNA acts by RNA:RNA interactions at multiple loci. *Proc. Natl. Acad. Sci. USA* **1998**, *95*, 12456–12461.

94. Barth, M.; Marschall, C.; Muffler, A.; Fischer, D.; Hengge-Aronis, R. Role for the histone-like protein H-NS in growth phase-dependent and osmotic regulation of sigmaS and many sigmaS-dependent genes in *Escherichia coli*. *J. Bacteriol.* **1995**, *177*, 3455–3464.

95. Ko, M.; Park, C. Two novel flagellar components and H-NS are involved in the motor function of *Escherichia coli*. *J. Mol. Biol.* **2000**, *303*, 371–382.

96. Majdalani, N.; Chen, S.; Murrow, J.; St John, K.; Gottesman, S. Regulation of RpoS by a novel small RNA: the characterization of RprA. *Mol. Microbiol.* **2001**, *39*, 1382–1394.

97. Vianney, A.; Jubelin, G.; Renault, S.; Dorel, C.; Lejeune, P.; Lazzaroni, J.C. *Escherichia coli tol* and *rcs* genes participate in the complex network affecting curli synthesis. *Microbiology* **2005**, *151*, 2487–2497.

98. Mika, F.; Hengge, R. A two-component phosphotransfer network involving ArcB, ArcA, and RssB coordinates synthesis and proteolysis of sigmaS (RpoS) in *E. coli*. *Genes Dev.* **2005**, *19*, 2770–2781.

99. Malpica, R.; Franco, B.; Rodriguez, C.; Kwon, O.; Georgellis, D. Identification of a quinone-sensitive redox switch in the ArcB sensor kinase. *Proc. Natl. Acad. Sci. USA* **2004**, *101*, 13318–13323.

100. Wassarman, K.M.; Repoila, F.; Rosenow, C.; Storz, G.; Gottesman, S. Identification of novel small RNAs using comparative genomics and microarrays. *Genes Dev.* **2001**, *15*, 1637–1651.

101. Mika, F.; Hengge, R. Freie Universität Berlin, Germany, unpublished work.

102. Roemling, U.; Bian, Z.; Hammar, M.; Sierralta, W.D.; Normark, S. Curli fibers are highly conserved between *Salmonella typhimurium* and *Escherichia coli* with respect to operon structure and regulation. *J. Bacteriol.* **1998**, *180*, 722–731.

103. Muffler, A.; Traulsen, D.D.; Lange, R.; Hengge-Aronis, R. Posttranscriptional osmotic regulation of the sigmaS subunit of RNA polymerase in *Escherichia coli*. *J. Bacteriol.* **1996**, *178*, 1607–1613.

104. Altuvia, S.; Zhang, A.; Argaman, L.; Tiwari, A.; Storz, G. The *Escherichia coli* OxyS regulatory RNA represses *fhlA* translation by blocking ribosome binding. *EMBO J.* **1998**, *17*, 6069–6075.

105. Latasa, C.; Garcia, B.; Echeverz, M.; Toledo-Arana, A.; Valle, J.; Campoy, S.; Garcia-del Portillo, F.; Solano, C.; Lasa, I. *Salmonella* biofilm development depends on the phosphorylation status of RcsB. *J. Bacteriol.* **2012**, *194*, 3708–3722.

106. Overgaard, M.; Johansen, J.; Moller-Jensen, J.; Valentin-Hansen, P. Switching off small RNA regulation with trap-mRNA. *Mol. Microbiol.* **2009**, *73*, 790–800.

107. Figueroa-Bossi, N.; Valentini, M.; Malleret, L.; Fiorini, F.; Bossi, L. Caught at its own game: Regulatory small RNA inactivated by an inducible transcript mimicking its target. *Genes Dev.* **2009**, *23*, 2004–2015.

© 2013 by the authors; licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution license (http://creativecommons.org/licenses/by/3.0/).

Copyright of International Journal of Molecular Sciences is the property of MDPI Publishing and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.
