
Clinical Microbiology Reviews

Antimicrobial Resistance and Virulence: a Successful or Deleterious Association in the Bacterial World?

Alejandro Beceiro, María Tomás and Germán Bou
Clin. Microbiol. Rev. 2013, 26(2):185. DOI: 10.1128/CMR.00059-12.

---

Updated information and services can be found at:
http://cmr.asm.org/content/26/2/185

These include:

**REFERENCES**
This article cites 460 articles, 252 of which can be accessed free at: http://cmr.asm.org/content/26/2/185#ref-list-1

**CONTENT ALERTS**
Receive: RSS Feeds, eTOCs, free email alerts (when new articles cite this article), [more](http://cmr.asm.org/content/26/2/185)

---

Information about commercial reprint orders: http://journals.asm.org/site/misc/reprints.xhtml
To subscribe to to another ASM Journal go to: http://journals.asm.org/site/subscriptions/

Journals.ASM.org

Antimicrobial Resistance and Virulence: a Successful or Deleterious Association in the Bacterial World?

Alejandro Beceiro, María Tomás, Germán Bou

Servicio de Microbiología, Complejo Hospitalario Universitario A Coruña-INIBIC, A Coruña, Spain; Spanish Network for Research in Infectious Diseases (REIPI), Spain#

SUMMARY ..... 186
INTRODUCTION ..... 186
RELATIONSHIP BETWEEN MECHANISMS OF ANTIMICROBIAL RESISTANCE AND VIRULENCE ..... 182
Resistance to β-Lactams and Impact on Virulence ..... 187
PBP modifications ..... 187
β-Lactamase expression ..... 190
Permeability and porins ..... 192
Efflux pumps ..... 193
Resistance to Aminoglycosides and Effect on Virulence ..... 195
Aminoglycoside-modifying enzymes ..... 195
Efflux pumps ..... 195
Ribosomal methylases ..... 195
Ribosomal mutations ..... 195
Resistance to Fluoroquinolones and Effect on Virulence ..... 195
Target modification (topoisomerase and DNA gyrase mutations) ..... 195
Efflux pumps ..... 196
Qnr proteins ..... 196
Porins ..... 196
Resistance to Tetracyclines and Tigecycline and Effect on Virulence ..... 196
Efflux pumps ..... 197
Ribosomal protection ..... 197
Antibiotic modification ..... 197
Tigecycline ..... 197
Resistance to Macrolides and Effect on Virulence ..... 197
Efflux pumps ..... 197
Ribosomal methylation and modification ..... 198
Resistance to Glycopeptides and Effect on Virulence ..... 198
Cell wall modifications ..... 198
Modified peptidoglycan target (D-Ala-D-Lac or D-Ala-D-Ser) ..... 198
Resistance to Oxazolidinones (Linezolid) and Effect on Virulence ..... 198
rRNA mutations ..... 199
rRNA methylation ..... 199
Resistance to Colistin, Polymyxin B, and Antimicrobial Peptides and Effect on Virulence ..... 199
LPS modifications ..... 199
Increased production of nonessential antimicrobial targets ..... 200
HIGHLY VIRULENT AND MULTIREESISTANT WORLDWIDE DISSEMINATED CLONES ..... 200
COSELECTION OF MECHANISMS OF ANTIMICROBIAL RESISTANCE AND VIRULENCE ..... 201
Plasmids ..... 201
Integrative and Conjugative Elements ..... 202
Phage-Mediated Transduction ..... 203
Outer Membrane Vesicles ..... 204
COMPENSATORY MUTATIONS ..... 204
GLOBAL RESPONSES AND THEIR EFFECT ON ANTIMICROBIAL RESISTANCE AND VIRULENCE ..... 207
Two-Component Regulatory Systems ..... 207
PhoP-PhoQ ..... 207
PmrA-PmrB ..... 208
CbrA-CbrB ..... 208
WalK-WalR ..... 208
(continued)

Address correspondence to Germán Bou, German.Bou.Arevalo@sergas.es.
# www.reipi.org.

A.B. and M.T. contributed equally to this article.

Copyright © 2013, American Society for Microbiology. All Rights Reserved.

doi:10.1128/CMR.00059-12

Stress by DNA Damage: SOS Response 208  
Mutator Phenotypes 209  
Persister Cells 210  
Alarmone Guanosine Tetraphosphate 210  
Alternative Sigma Factor σ<sup>B</sup> 211  
NOVEL ANTIVIRULENCE THERAPIES: MECHANISMS OF RESISTANCE TO ANTIVIRULENCE COMPOUNDS 211  
Cell-to-Cell Signaling Inhibitors 211  
RND Efflux Pump Inhibitors 213  
Antimicrobial Compounds with Antivirulence Activity 213  
Resistance to Antivirulence Components 213  
DISCUSSION 214  
How Does Increased Antimicrobial Resistance Affect Virulence? 214  
What Are the Perspectives in the Near Future in Relation to Antimicrobial Resistance and Virulence? 215  
Are There Alternatives to Antimicrobial Therapy for Controlling Bacterial Multiresistance and Virulence? 216  
Are Antibiotic Resistance and Virulence Increasingly Linked in the Development of Infectious Processes? 217  
ACKNOWLEDGMENTS 218  
REFERENCES 218  
AUTHOR BIOS 229  

---

**SUMMARY**

Hosts and bacteria have coevolved over millions of years, during which pathogenic bacteria have modified their virulence mechanisms to adapt to host defense systems. Although the spread of pathogens has been hindered by the discovery and widespread use of antimicrobial agents, antimicrobial resistance has increased globally. The emergence of resistant bacteria has accelerated in recent years, mainly as a result of increased selective pressure. However, although antimicrobial resistance and bacterial virulence have developed on different timescales, they share some common characteristics. This review considers how bacterial virulence and fitness are affected by antibiotic resistance and also how the relationship between virulence and resistance is affected by different genetic mechanisms (e.g., coselection and compensatory mutations) and by the most prevalent global responses. The interplay between these factors and the associated biological costs depend on four main factors: the bacterial species involved, virulence and resistance mechanisms, the ecological niche, and the host. The development of new strategies involving new antimicrobials or nonantimicrobial compounds and of novel diagnostic methods that focus on high-risk clones and rapid tests to detect virulence markers may help to resolve the increasing problem of the association between virulence and resistance, which is becoming more beneficial for pathogenic bacteria.

---

**INTRODUCTION**

Bacteria are present both inside and on the surface of the human body, especially on the skin and the mucous membranes. Most of these bacteria are innocuous, many are beneficial, and some are even necessary. However, other bacteria, which are categorized as pathogens, are able to colonize, invade, and damage the host and thus cause illness. Pathogenicity is an ability of an agent to cause disease, and the pathogenic bacteria possess several factors that enable them to enhance their virulence (i.e., the degree of pathogenicity). Most pathogens make use of a combination of two properties to cause disease: (i) toxicity, the degree to which a substance causes harm, and (ii) invasiveness, the ability to penetrate into the host and spread [1]. The final balance of an infectious disease process will depend on the virulence or pathogenicity of the microbe as well as the host status in relation to risk factors such as immune status, age, diet, and stress, which determine the host susceptibility to infection. Hosts and bacteria have coevolved over millions of years, during which pathogenic bacteria have modified their virulence to adapt to the host defense systems. This contrasts with the relatively recent evolution of antimicrobial resistance (defined as the ability of an organism to resist the action of an antimicrobial agent to which it was previously susceptible). Although medical practice has limited the development and spread of pathogens, this has led to a global increase in antibiotic resistance. The evolution and spread of resistance are relatively recent and have occurred mainly in the last 50 years, i.e., since antibiotics were first used. Therefore, virulence and resistance have evolved over very different timescales.

Despite the difference in the evolution of these processes, they share some common characteristics. (i) From a biological point of view, both processes are necessary for bacteria to survive under adverse conditions. Virulence mechanisms are necessary to overcome host defense systems, and the development of antimicrobial resistance is essential to enable pathogenic bacteria to overcome antimicrobial therapies and to adapt to and survive in competitive and demanding environments (new niches). Immune defense systems and antibiotic pressure represent bottlenecks for survival of the bacterial population, as they greatly limit the capacity for growth and lead to decreased microbial diversity [2, 3]. (ii) Virulence and resistance factors are similar in that most of the determinants have been transmitted between species or genera by horizontal gene transfer (HGT); the transfer of DNA fragments (mobile genetic elements [MGEs]) is probably the main genetic mechanism of dissemination and coselection of virulence and resistance genes, although other mechanisms such as compensatory or adaptive mutations may also be involved [4, 5], as will be discussed below. (iii) Antibiotic resistance is often associated with infection and is therefore also related to virulence, as in the cases of biofilm-producing microorganisms or intracellular infections [6, 7]. (iv) Other characteristics that are common to virulence and resistance include the direct involvement of efflux pumps [8], porins [9], cell wall alterations [10], and two-component systems that activate or repress the expression of various genes, such as those involved in resistance and virulence [11].

In healthy individuals, opportunistic pathogens are not able to produce infection because they lack the necessary mechanisms of toxicity and invasiveness that enable primary pathogens to overcome the host's defenses.
come the host immune system. However, in some individuals, such as immunocompromised patients, opportunistic pathogens can produce infection, which can be prevented mainly by the use of antimicrobial therapies. Certain multiresistant opportunistic species, such as *Pseudomonas aeruginosa* and *Acinetobacter baumannii*, can colonize niches where many other species cannot survive (environments with high antibiotic pressure) and can even displace the commensal flora. This is one example of how antimicrobial resistance can increase the virulence or fitness of certain species in some environments, often helping these species to colonize new niches. Therefore, although antibiotic resistance is not in itself a virulence factor, in certain situations it is a key factor in development of infection, and it may be considered a virulence-like factor in specific ecological niches which antibiotic-resistant bacteria are able to colonize. This is especially true in the hospital environment (intensive care units, burn units, etc.), in which if a opportunistic pathogen is drug resistant, it can cause disease more readily (12). In environments where selective antibiotic pressure prevails, some opportunistic pathogens are able to colonize new ecological niches because of their plasticity and ability to adapt through the acquisition or development of mechanisms of resistance and persistence.

The use of antibiotics has changed the natural evolution of bacteria by reducing susceptible pathogenic populations and increasing resistant populations. Resistance is often associated with a fitness cost because the genetic burden required for resistance may be deleterious in antibiotic-free environments. In this case, restriction of the use of antibiotics has been proposed, with the aim of eradicating resistant bacteria (13). However, the genetic background of resistant pathogens allows them to persist in the presence of minimal concentrations of antibiotics or even in the absence of these, as discussed throughout this review (14). Hypermutation, compensatory mutations, and cross coselection are a few of the many mechanisms that favor the persistence of resistant pathogens and even, in some cases, selection of the most virulent and most resistant pathogens.

This review considers the relationship between virulence and resistance, including the role of increasing resistance in relation to fitness costs. Increased resistance is associated in most cases, either directly or indirectly, with decreased virulence and fitness. However, evidence also shows the opposite, and it is increasingly evident that the relationship is often of greater benefit to the pathogen, resulting in a growing public health problem.

This review also considers the impacts of resistance to the main antimicrobial agents used in clinical practice as well as the genetic events associated with evolution of pathogens on virulence and/or fitness costs. *In vivo* and *in vitro* laboratory examples and clinical studies of specific pathogens are presented, and the interplay between both of these important bacterial characteristics is analyzed in detail. Because of the importance of the interactions between resistance and virulence, these aspects are always considered together. Partial analyses that consider aspects of resistance and/or virulence separately were disregarded for the purposes of this review.

## RELATIONSHIP BETWEEN MECHANISMS OF ANTIMICROBIAL RESISTANCE AND VIRULENCE

Throughout this section we will refer frequently to **Table 1**, in which we have attempted to clarify and summarize the examples given in the text of relationships between virulence and resistance

determinants in the most commonly studied or clinically relevant pathogens. To simplify the numerous studies reported in the literature, we have classified the examples according to families of antibiotics for which the resistance mechanisms have been examined.

### Resistance to β-Lactams and Impact on Virulence

β-Lactam antibiotics are a large class of antibiotics that have a β-lactam ring in their molecular structure. They are the most widely used antibiotics and include penicillin derivatives, cephalosporins, monobactams, carbapenems, and β-lactamase inhibitors.

Resistance to β-lactams involves several mechanisms, which are different in Gram-positive and Gram-negative bacteria. In Gram-positive microbes, mutations and/or reduction of or expression alterations in penicillin-binding proteins (PBPs) are the most important mechanisms, followed by β-lactamase production. Conversely, in Gram-negative microorganisms, the most prevalent mechanism of β-lactam resistance is the production of β-lactamases, followed by permeability alterations, extrusion by efflux pumps, and to a lesser extent PBP alterations. Before further discussion of this topic, it is important to point out that interpretation of the data and any conclusions drawn may differ greatly depending on the specific microorganism under study, even within a family, e.g., *Enterobacteriaceae*. Therefore, caution must be exercised when discussing this topic, and conclusions should be assumed to be organism specific.

#### PBP modifications

The PBPs involved in resistance and virulence are PBP2 (encoded by *mecA*) (in *Staphylococcus aureus*), PBP2b-PBPX (in *Streptococcus pneumoniae*), and PBP7-8 (in *A. baumannii*) (**Table 1**).

Previously reported data suggest that expression of homogeneous methicillin resistance in *S. aureus* influences the biofilm phenotype and attenuates virulence (it reduced protease production and significantly reduced virulence in a mouse model of device-related infection) (15). Clinical isolates of *S. aureus* can express biofilm phenotypes promoted by the major cell wall autolysin and the fibronectin (Fn)-binding proteins or the polysaccharide intercellular adhesin (PIA) and the polymeric N-acetylglucosamine (PNAG), which are synthesized and exported by proteins encoded by the *icaADBC* gene cluster. Biofilm production in methicillin-susceptible *S. aureus* (MSSA) strains is associated with PIA/PNAG, whereas methicillin-resistant isolates express an Atl/FnBP-mediated biofilm phenotype (which produces a proteinaceous biofilm), which suggests a relationship between biofilm production and susceptibility to β-lactam antibiotics (16). Rudkin et al. reached similar conclusions after demonstrating that methicillin resistance reduces the virulence of health care-associated methicillin-resistant *S. aureus* (MRSA) by interfering with the *agr* quorum-sensing (QS) system in such a way that the ability of the bacteria to secrete cytolytic toxins is reduced (15). These authors state that methicillin resistance induces cell wall alterations that affect the Agr quorum-sensing system of the bacteria. This leads to reduced expression of the toxin and lowered virulence in a murine model of sepsis. This interesting finding may explain why some strains of hospital-acquired MRSA show a reduced ability to spread in the community. It may also explain the recent increase in the incidence of community-associated MRSA (CA-MRSA) strains, which typically express less penicillin-binding protein 2a (encoded by *mecA*) and thus main-
| Antimicrobial group | Mechanism of resistance | Implication in virulence | Pathogen(s) | Reference(s) |
| --- | --- | --- | --- | --- |
| β-Lactams | PBPs modifications<br>Penicillin-binding protein 2 (*mecA*)<br>SCC*mec*<br>PBP2b-PBPX<br>PBP7-8<br>B-Lactamases<br>CIT-M-type ESBLs<br>OXA-10-like, OXA24, and SFO-1<br>AmpC<br>AmpC/AmpD/AmpR | Regulation of Agr quorum-sensing system; biofilm formation; attenuated virulence in mouse model; infection persistence<br>Attenuated virulence in mouse model<br>Attenuated virulence in mouse model<br>Usually plasmid borne; increased virulence not clearly demonstrated<br>Fitness cost in common host (changes in peptidoglycan composition)<br>Fitness cost and virulence; AmpR (transcriptional regulator of *ampC*) also controls expression of alginate production and quorum-sensing system; type 3 fimbrial gene expression and biofilm formation | S. aureus<br>S. pneumoniae<br>A. baumannii<br>E. coli<br>E. coli<br>S. enterica<br>P. aeruginosa, K. pneumoniae | 15<br>17, 19, 25<br>31, 32<br>34<br>35<br>10, 37, 38 |
|  | B-Lactamases (ESBL)<br>IMP types<br>PER-1 | Invasion of epithelial cells; plasmid-carried genes<br>No significant impact in virulence in animal model | K. pneumoniae<br>P. aeruginosa<br>A. baumannii | 36<br>33<br>41 |
|  | Porins<br>OmpA<br>Omp33-36<br>CarO<br>OprD-like<sup>a</sup><br>OmpC<br>OmpF<br>OmpK35-K36 | Adhesion cell; induction of cell death; biofilm formation<br>Cell adhesion; induction of cell death; biofilm formation<br>Attenuated virulence in mouse model<br>Adhesion, cell invasion, and intestinal colonization (Crohn's disease)<br>Adhesion to Hep-2 cells<br>Resistance to phagocytosis; metabolic fitness cost | A. baumannii<br>A. baumannii<br>A. baumannii<br>E. coli<br>E. coli<br>K. pneumoniae | 49, 56<br>49, 56<br>55<br>55<br>58<br>60<br>9 |
|  | Efflux pumps<br>AdeABC<br>AcrAB-TolC<br>Mex system | Colonization, infection, and persistence of microorganism in host<br>Colonization, infection, and persistence of microorganism in host<br>Colonization, persistence, and expression of virulence genes; MexAB is involved in quorum-sensing/quorum-quenching system; MexCD is associated with regulation of type III secretion system; MexEF transduction system | A. baumannii<br>Enterobacteriaceae<br>P. aeruginosa | 148, 149, 437<br>68<br>66, 94, 95, 96, 102 |
|  | SecDF | Expression of virulence genes | E. coli, B. subtilis, S. aureus | 103 |
| Aminoglycosides | Efflux pumps (as for β-lactams above)<br>Ribosomal methylases/RmtC<br>Ribosomal mutations/RpsL | No fitness cost<br>Fitness cost | E. coli<br>E. coli, Salmonella spp., M. tuberculosis | 104<br>13, 111-115 |
| Fluoroquinolones | Target modifications (topoisomerases and DNA gyrases) | No fitness cost; presence of the type III secretion system genes<br>Fitness cost<br>Higher risk of invasive illness or death<br>Virulence gene expression and cell viability<br>No fitness cost<br>Fitness cost<br>Inactivation of the *sarA* regulator gene<br>Decreased colonization | P. aeruginosa<br>S. enterica<br>S. Typhimurium<br>S. flexneri<br>E. coli<br>A. baumannii<br>S. aureus<br>S. pneumoniae | 119<br>120<br>123<br>124<br>125<br>121<br>129<br>132 |
| Tetracyclines and tigecyclines | Efflux pumps<br>NorC, Tet38, and AbcA)<br>AcrAB-TolC (as for β-lactams)<br>Mex system (as for β-lactams)<br>BepDE system<br>Qnr<br>Porins (as for β-lactams) | Increased expression of virulence genes but decreased fitness<br>Contribution to virulence (unknown mechanism)<br>Ribosomal mutations and others increase antibiotic resistance but with deleterious effects on fitness | S. aureus<br>B. suis<br>E. coli | 133<br>134<br>140 |
|  | tet genes such as Tet(A) or Tet(B)<br>AdeABC (as for β-lactams)<br>Ribosomal protection<br>tet44<br>Mutations in 16S rRNA |  | C. fetus<br>H. pylori | 142<br>143 |

Antibiotic modification

| **tetX gene** | No fitness cost in *B. fragilis*; probable fitness cost in aerobic Gram-negative bacteria |
| --- | --- |

Macrolides

| **Efflux pumps** | Decreased twitching motility |
| MuxABC-Omp | Excretion of acyl homoserine lactone quorum-sensing molecules (biofilms, siderophores, and phospholipase C) |
| BpEAB-OpRB | Fitness cost |
| MtrC-MtrD-MtrE | Presence of virulence genes (*gelE*) |
| erm class genes | Fitness cost |
| 23S rRNA mutation |  |

Glycopeptides

| **Cell wall modifications** | Fitness cost |
| GSA | Attenuated virulence in nonmammalian model system *G. mellonella* |
| Modified peptidoglycan target | Fitness cost |
| VanA and VanB phenotypes |  |

Oxazolidones (linezolid)

| **rRNA mutations** | Coagulase-negative staphylococci, *S. aureus*, *S. pneumoniae* |
| rRNA methylation | *S. aureus* |
|  | Enterococcus spp. |

Colistin, polymyxin B

| **Lipopolysaccharide modifications** | Global regulation, including virulence and resistance |
| PmrA-PmrB | Increased virulence in mouse model |
| SurA, TolB, and Gnd | Global regulation, including virulence and resistance |
| PhoP-PhoQ | Fitness cost by restructuring of the bacterial surface |
| LpxA, LpxD, or LpxC<sup>b</sup> | Fitness cost in *in vitro* experiments and decreased virulence in mouse model |
| PmrC-PmrA-PmrB<sup>b</sup> | Regulation of virulence and resistance (adaptation to swarming motility) |
| PsrA | *P. aeruginosa* |
| Efflux pumps | Increased virulence in mouse model of gastric infections; involved in proliferation capacity inside macrophages and epithelial cells |
| YejABEF |  |

| **Increased production of nonessential antimicrobial target** | Increased virulence, carrying virulence factors such as toxins |
| **Overproduction of OMVs** | Increased virulence, evasion of phagocytosis, and complement resistance |
| **Overproduction of bacterial capsule** |  |

| **P. aeruginosa** | 151 |
| **B. pseudomallei** | 153 |
| **N. gonorrhoeae** | 154 |
| **E. faecalis** | 155 |
| **C. jejuni** | 156, 157 |
| **S. aureus** | 130, 161 |
| **S. aureus** | 162 |
| **Enterococcus spp.** | 165 |
| **Coagulase-negative staphylococci, S. aureus, S. pneumoniae** | 170, 171 |
| **S. aureus** | 173, 174 |
| **S. enterica** | 175 |
| **S. enterica** | 177 |
| **A. baumannii** | 197, 439 |
| **A. baumannii** | 190, 191 |
| **A. baumannii** | 188, 193, 194 |
| **P. aeruginosa** | 195 |
| **S. enterica** | 198 |
| **E. coli** | 199 |
| **K. pneumoniae, S. pneumoniae, P. aeruginosa** | 200, 201 |

<sub>*a* Implication in resistance not clearly demonstrated.</sub>

<sub>*b* Implication in virulence not clearly demonstrated.</sub>
tain full virulence for success in the community setting. This is a typical example of how the acquisition of resistance to a specific antibiotic (oxacillin) is related to a decrease in virulence.

However, Queck et al. described the presence of a *psm* gene (*psm-mec*), associated with staphylococcal methicillin resistance, which encodes a mobile genetic element (MGE), SCCmec (17). Phenol-soluble modulins (PSMs), which are staphylococcal cytolytic toxins, play a crucial role in immune evasion. Although all known PSMs are core genome encoded, Queck et al. described this gene inside the SCCmec clusters of types II and III in a series of staphylococcal strains, including strains of *S. aureus*, *S. epidermidis*, *S. saprophyticus*, *S. pseudintermedius*, and *S. sciuri*. This very interesting study revealed a previously unknown role for methicillin resistance clusters in staphylococcal pathogenesis and showed that both antibiotic resistance and virulence determinants may be combined in staphylococcal MGEs (the exception to the above-mentioned rule).

*S. pneumoniae* was previously considered to be universally susceptible to penicillin. However, reports of isolates with decreased susceptibility to penicillin have increased worldwide in recent years (18). In one study, Rieux et al. analyzed the relationship between acquisition of β-lactam resistance and loss of virulence in *S. pneumoniae* (19). The authors transformed a virulent and penicillin-susceptible strain with *pbpX* and *pbp2b* from a penicillin-resistant strain to assess the relationship between acquired resistance and virulence. After transformation, the virulence of these receptor strains was significantly reduced.

Penicillin resistance in *S. pneumoniae* caused by PBP modifications may occur via different mechanisms, such as acquisition of an additional low-affinity PBP, overexpression of an endogenous low-affinity PBP, alteration of endogenous PBPs by point mutations or homologous recombination, or a combination of these (20). Several studies have assessed whether resistance to this first-line antibiotic may affect the virulence of pneumococci. For instance, Azoulay-Dupuis et al. examined the relationship between virulence and penicillin susceptibility in an experimental murine model of peritoneal infection and capsular type, using 122 clinical strains of *S. pneumoniae* belonging to 24 different serotypes (21). All 32 virulent strains were penicillin susceptible, while all 41 strains with reduced susceptibility to penicillin were avirulent, independently of the serotype. In a later study, Fernandez et al. used a rabbit model of meningitis to test the inflammatory activity induced by three different strains of pneumococci, belonging to serotypes 3, 6B, and 23F (the prevalent serotypes in Western Europe), with different susceptibilities to β-lactams (22). The authors' conclusions supported the idea that penicillin-resistant pneumococcus isolates are avirulent in immunocompetent mice, regardless of the isolation site (23, 24). Overall, the data suggest that acquisition of penicillin resistance in pneumococci is associated with loss of virulence in different models of infection.

Data from studies of Gram-negative bacilli are very scarce, and the impact of PBP alterations on virulence is unclear. With the aim of simplifying the interpretation of the data, this review will focus only on *A. baumannii*. Russo et al. have demonstrated that PBP7-8 contributes to both the *in vitro* and *in vivo* survival of *A. baumannii* (25). These authors screened a random transposon mutant library and identified a mutant, AB307.27, which contained a transposon insertion in *pbpG*, encoding the putative low-molecular mass PBP7-8. This mutant showed lower survival in a rat soft tissue infection model and in a rat pneumonia model than the isogenic wild-type strain. Although no clear data have been obtained with regard to the putative role of PBP alterations in β-lactam resistance of *A. baumannii* (they are an important resistance mechanism in other species, such as *P. aeruginosa*), it has been suggested that such alterations (at least in PBP7-8) may lead to a decrease in the virulence of *A. baumannii* (25, 26).

**β-Lactamase expression.** As stated above, β-lactamase production is the main mechanism of β-lactam resistance in Gram-negative pathogens. However, it is not clear whether expression of β-lactamases affects the virulence or fitness of these bacteria or whether any general conclusions can be drawn. Examples of β-lactamases previously reported to be involved in virulence are listed in Table 1.

*Escherichia coli* ST131 (O25:H4), which produces the CTX-M-15 extended-spectrum β-lactamase (ESBL), has emerged internationally as a multidrug-resistant (MDR) pathogen (see Highly Virulent Multiresistant Worldwide Disseminated Clones, below). This β-lactamase and other CTX-M-type enzymes are also prevalent in other virulent *E. coli* clones, such as Shiga toxin-producing serotype O111:H8 (27) and O26:H11 (28), and even in *E. coli* strains isolated from chicken and pig farms in Spain, highlighting the potential risk of zoonotic transmission (29). Thus, virulence factors have been identified in *E. coli* isolates that produce CTX-M-type ESBL (30), and *E. coli* ST131, which produces the widespread NDM-1 enzyme, has been found to exhibit a wide array of virulence factors (31). Therefore, β-lactamase enzymes and virulence genes may coexist in specific *E. coli* clones worldwide, possibly as a result of stepwise coevolution processes.

However, it is not clear whether the presence of a specific β-lactamase gene impairs the ability of the microorganism to cause damage, in terms of host colonization, invasiveness (pathogenicity) or fitness costs, and robust conclusions cannot be drawn from the data obtained so far. At least in *E. coli*, the production of a CTX-M-1 enzyme does not appear to affect virulence. Dubois et al. reported the isolation, from a patient with neonatal meningitis, of an *E. coli* strain with three different plasmids, one of which produced CTX-M-1 β-lactamase (32). The plasmid that encoded the β-lactamase did not increase the incidence of meningitis in a newborn mouse model, thus suggesting that the CTX-M-type enzyme does not increase the virulence of *E. coli* in this animal model (32). Very similar findings were obtained in a study that evaluated the virulence of *P. aeruginosa* carrying *blaIMP* (a metallo-β-lactamase gene) in an endogenous bacteremia model, and it was concluded that the presence of this enzyme did not significantly affect the virulence of the *P. aeruginosa* PAO1 parent strain (33).

The impact of β-lactamases in relation to biological fitness costs in bacteria has been poorly studied. The interaction between resistance mechanisms and bacterial fitness will decide the fate of a specific bacterial strain once the selective pressure exerted by antibiotics disappears. In a study of this topic, our group demonstrated quantitative changes in the peptidoglycan composition in *E. coli* strains expressing OXA-24, OXA-10-like, and SFO-1 (with its upstream regulator AmpR) β-lactamases; the changes were reflected by a decrease in the level of cross-linked muropeptides and an increase in the average length of the peptidoglycan chains. These changes were associated with a statistically significant fitness cost, which was demonstrated both *in vitro* and *in vivo* in a mouse model of systemic infection (34) (Fig. 1). The biological cost associated with these changes indicates the importance of the interaction between β-lactamases and peptidoglycan metabolism.
Another interesting example is the role of *ampC* expression and its impact on fitness or virulence in *P. aeruginosa*. In 2008, Moya et al. reported that inactivation of *ampD* in *P. aeruginosa* led to a partially derepressed phenotype, characterized by a moderately high level of *ampC* expression, which could be induced even further due to the presence of two additional *ampD* genes (*ampDh2* and *ampDh3*). Sequential inactivation of these genes resulted in full derepression in the triple mutant, which exhibited 1,000-fold overexpression of the *ampC* gene relative to the wild-type isogenic strain, thus causing a loss of virulence due to alterations in peptidoglycan recycling (10). Due to this multiplicity of *ampD* genes, fitness costs and virulence are scarcely affected in *P. aeruginosa*, despite increases in β-lactam MICs, so that this bacterium is able to develop resistance to β-lactam antibiotics without a significant loss of fitness. Only *ampDh3* was found to be important for virulence in *P. aeruginosa*. In this example, resistance and virulence are chromosomally regulated, showing how the multiplicity of *ampD* genes may constitute a regulatory mechanism for the *ampC* gene (β-lactam resistance) and maintenance of the cell wall (thus affecting virulence).

In Enterobacteriaceae and *P. aeruginosa*, the transcriptional regulator AmpR, a member of the LysR family, regulates the expression of a chromosomal β-lactamase, AmpC. However, AmpR appears to have a wider physiological role in these organisms because, in addition to regulating expression of *ampC*, AmpR regulates the expression of sigma factor AlgT/U (alginate production) and production of some quorum-sensing (QS)-regulated virulence factors (37, 38). The authors compared DNA microarrays of the *P. aeruginosa* PAO1 strain and its isogenic *ampR* deletion mutant PAO Δ*ampR*, and they found that AmpR regulates AmpC expression (resistance to β-lactams) and also non-β-lactam antibiotic resistance by modulating the MexEF-OprN efflux pump (37, 38). This is a good example of how such organisms may simultaneously regulate antibiotic resistance in addition to biofilm formation and QS-regulated acute virulence factors. Similarly, AmpR has been found to upregulate capsule synthesis (and, therefore, resistance to serum killing) and to modulate biofilm formation and type 3 fimbrial gene expression in a clonal strain of *K. pneumoniae* producing the plasmid-encoded cephalosporinase DHA-1 (39), which suggests that AmpR may regulate several virulence factors in addition to *ampC* expression.

It appears to be of great interest to study this effect with the nosocomial pathogen *A. baumannii*, in which the main mechanism involved in carbapenem resistance is the production of OXA-type β-lactamases (40). Although the involvement of these enzymes in resistance to β-lactams (specifically carbapenems) is a cause of great concern, very few studies have addressed the impact of these enzymes on the virulence of *A. baumannii*. Sechi et al. investigated the presence and association of various virulence determinants in 20 *A. baumannii* isolates, 13 of which were *bla*<sub>PER-1</sub> positive (41). These authors found a relationship between PER-1 and cell adhesion in *Acinetobacter* strains, although the exact mechanisms of the association remain unknown. In *A. baumannii*, OXA-24 is one of the main enzymes involved in carbapenem resistance, at least in some countries (42, 43). One of the most striking features of this enzyme, and of all OXA-type enzymes in general, is its scarce presence in Enterobacteriaceae.

In February 2006, a patient colonized with an MDR *A. baumannii* sequence type 56 (ST56) strain (updated in the new multilocus sequence type [MLST] database as ST121) was admitted to

FIG 1 Data from *in vitro* competition experiments with *E. coli* MG1655 carrying the recombinant plasmids pBGS18-TEM1, pBGS18-SFO1, pBGS18-AmpR-SFO1, pBGS18-OXA10-like, pBGS18-FOX4, and pBGS18-CTX-M-32. The strains expressing AmpR-SFO1, OXA10-like, and OXA24 β-lactamases showed significant fitness costs. The CI values obtained with different β-lactamases are plotted and are representative of eight different experiments, with the median CI values shown by horizontal lines. (Adapted from reference 34.)

Determination of the biological cost may also provide information about the epidemiology of β-lactamases and help explain the low incidence of some of these enzymes in specific pathogens, such as *E. coli*. A similar example of the effect of β-lactamases on fitness cost in Enterobacteriaceae is provided by the expression of AmpC of *Salmonella enterica* (35), which was associated with reduced growth rate and invasiveness, although the authors did not observe any major variations in the peptidoglycan composition of this microorganism.

Extended-spectrum β-lactamase (ESBL)-producing *Klebsiella pneumoniae* strains have been suggested to possess higher pathogenic potential than nonproducers. Sahly et al. examined the ability of 58 ESBL-producing and 152 nonproducing isolates of *K. pneumoniae* to express type 1 and 3 fimbrial adhesins, which are important traits in microbial adherence and invasion of host cells (36). Although the adherence of ESBL- and non-ESBL-producing strains to epithelial cells did not differ significantly (*P* > 0.05), the proportion of strains able to invade (>5% relative invasion) ileo-cecal and bladder epithelial cells was significantly higher among ESBL producers (81% and 27.6%, respectively) than among non-ESBL producers (61% and 10%, respectively). Likewise, the proportion of ESBL producers coexpressing both fimbrial adhesins was significantly higher than that of non-ESBL producers. On acquisition of the ESBL SHV-12-encoding plasmids, two transconjugants started to produce type 3 fimbriae, although expression of type 1 fimbriae was not affected. Overall, the data demonstrated that an ESBL plasmid appeared to upregulate the expression of one or more genes, resulting in a higher invasion capacity. It remains unclear whether this effect resulted from direct SHV-type expression or expression of another plasmid-borne gene.

Beceiro et al.

A. pMMA2

![Diagram of pMMA2](#)

- **Septicolisin**
- **tonB-dependent receptor**
- **bla<sub>OXA-24</sub>**

Approx. 1 kbp

B. pEK499

![Diagram of pEK499](#)

- **pemI**
- **pemK**
- **tet(A)**
- **aac6'-lb-cr**
- **bla<sub>CTX-M-15</sub>**
- **aadA5**
- **sulII**
- **mph(A)**
- **vagC**
- **vagD**
- **ccdB**

C. pRSB107

![Diagram of pRSB107](#)

- **ccdA**
- **ccdB**
- **pemI**
- **pemK**
- **aph**
- **strB**
- **sulII**
- **strA**
- **bla<sub>TEM-1</sub>**
- **dhfR**
- **mph(A)**
- **catA**
- **tetA**
- **orf82***
- **ugpB**
- **ugpE**
- **iutA**
- **iucC**
- **iucA**
- **vagC**
- **vagD**
- **ugpC**
- **ugpA**
- **iucD**
- **iucB**
- **vagC**
- **vagD**

Approx. 10 kbp

FIG 2 Partial structures and descriptions of different plasmids carrying virulence factors and antimicrobial resistance determinants together. Light gray, virulence genes; dark gray, antimicrobial resistance genes; black, plasmid maintenance genes. (A) Plasmid pMMA2, isolated from clinical isolate AbH12O-A2, which caused a large nosocomial outbreak. (Adapted from reference 436.) (B) Plasmid pEK499, which carries up to 10 resistance genes, including the genes encoding resistance to β-lactamases (bla<sub>TEM-1</sub>, bla<sub>CTX-M-15</sub>, and bla<sub>OXA-1</sub>), aminoglycosides (aac6-Ib-cr), macrolides [mph(A)], chloramphenicol (catB4), tetracycline [tet(A)], streptomycin (aadA5), and sulfonamide (sulII). It possesses two copies of the vagD-vagC virulence-associated system and the toxin/antitoxin pemI-pemK and ccdA-ccdB systems, which are involved in plasmid maintenance by postsegregation killing processes. (Reprinted from reference 229.) (C) Plasmid pRSB107, which carries genes encoding resistance to the following antibiotics: β-lactams (bla<sub>TEM-1</sub>), aminoglycosides (aph), streptomycin (strA-strB), sulfonamide (sulII), macrolides [mph(A)], trimethoprim (dhfR), chloramphenicol (catB4), and tetracycline [tet(A)]. This plasmid carries four putative virulence-associated determinants: an aerobactin iron acquisition siderophore system (iucA, iucB, iucC, iucD, and iutA), a putative high-affinity Fe<sup>2+</sup> uptake system (orf82), an sn-glycerol-3-phosphate transport system (ugpB, ugp, ugpE, and ugpA), and two copies of the virulence-associated genes vagC-vagD. Approx., approximately. (Reprinted from reference 237.)

a hospital in Madrid, Spain. The strain spread very quickly and caused the largest nosocomial outbreak ever reported in the literature (377 patients were colonized/infect ed with A. baumannii; of these, 290 were colonized/infect ed with antibiotic type 1 associated with an MDR clone, AbH12O-A2) (44). A. baumannii clone AbH12O-A2 exhibited a broad antimicrobial drug resistance profile, was resistant to carbapenems, and showed susceptibility only to tigecycline and colistin. One of the main characteristics of this strain was its capacity to produce invasive infections (the annual incidence of A. baumannii-induced bacteremia increased from 0.03 episodes/100,000 bed days in 2002 to 1.1/100,000 bed days in 2007, which corresponded to the peak of the outbreak caused by clone AbH12O-A2). The plasmid isolated from this epidemic strain was named pMMA2. Sequencing of nucleotides revealed the presence of the carbapenemase OXA-24 (42) flanked by XerC/XerD binding sites. The genes for two putative plasmid-encoded virulence factors, septicolysin and TonB-dependent receptor, were also found surrounding the bla<sub>OXA-24</sub> gene. The first of these was a cholesterol-dependent cytolysin and has been reported to be produced by pathogenic bacteria such as Clostridium perfringens, Bacillus anthracis, and S. pneumoniae (45). The TonB-dependent receptor gene has been associated with virulence and iron uptake in A. baumannii (46). Analysis of the structure of this pMMA2 plasmid (Fig. 2A) is of interest and provides information about how this MDR clone has been selected, possibly by a putative coevolution process in which a carbapenem resistance gene plus two putative virulence factors may facilitate its persistence and virulence in the hospital setting. In this case, resistance and virulence coexist in the same bacterium to yield a highly successful microorganism that is able to cause very serious infections in hospital settings.

Permeability and porins. Porins are β-barrel membrane proteins that cross cell membranes and act as a pore through which molecules, such as nutrients, toxins, and antibiotics, can diffuse. Alterations, modification, and reduction in the expression of porins have all been associated with antimicrobial resistance to some extent (47). Porins have a clear role in virulence and resistance. We emphasize studies involving A. baumannii (OmpA, Omp33-36, CarO, and OprD-like porins) and Enterobacteriaceae (OmpC/OmpF and OmpK35/OmpK36 porins), as shown in Table 1.

It is assumed that bacterial outer membrane proteins with a porin function not only control the entry of antimicrobials into bacterial cells but also control the virulence of the microorganism. The A. baumannii OmpA protein has recently been associated with resistance to cephalothin and cephaloridine in A. baumannii (48). Indeed, it has been reported that OmpA porin induces death of epithelial and dendritic human cells through mitochondrial

targeting (49) and is also involved in biofilm formation (50, 51). Another two porins, CarO and Omp33-36, have been associated with carbapenem resistance in this species (52, 53). Other porins (such as OprD), initially associated with resistance to carbapenems, have recently been associated with other physiological functions (similarly to the OprQ-like protein of *P. aeruginosa*), and it is possible that they contribute to the adaptation to magnesium-and/or iron-depleted environments (54). Fernandez-Cuenca et al. demonstrated attenuated virulence of a slow-growing pan-drug-resistant *A. baumannii* strain associated with decreased expression of genes encoding the CarO and OprD-like porins (55). The Omp33-36 protein of *A. baumannii*, which is involved in carbapenem resistance, has also been suggested to be involved in apoptosis and modulation of autophagy processes (unpublished data). Moreover, Cabral et al. recently reported the involvement of both CarO and Omp33-36 porins in biofilm formation in *A. baumannii* (51). Both OmpA and Omp33-36 proteins and also the TonB-dependent copper receptor have recently been identified as fibronectin-binding proteins (FBPs) and as being involved in the interaction between this pathogen and fibronectin, thus furthering the understanding of *A. baumannii* adherence to host cells (56).

In *E. coli*, the loss of OmpC contributes to both antibiotic resistance and reduced antibody-dependent bactericidal activity (57). Moreover, the OmpC protein has been associated with adhesion, cell invasion, and intestinal colonization in patients with Crohn’s disease (58). Bekhit et al. demonstrated that both OmpC and OmpF (both of which are involved in antimicrobial resistance) are essential for the survival of *E. coli* under extremely acidic conditions (59). OmpF has also been associated with adhesion to Hep-2 cells (60). The OmpK35 and OmpK36 outer membrane porins of *K. pneumoniae* are involved in resistance to cefazolin, cephalothin, and cefoxitin and increased meropenem and cefepime MICs. A significant (*P* < 0.05) increase in the susceptibility to phagocytosis was detected in both ΔOmpK36 and ΔOmpK35/36 mutants. In a mouse peritonitis model, the ΔOmpK36 mutant also exhibited significantly lower virulence, whereas both mutants together presented the highest median lethal dose (*LD*~50~) of these strains. Overall, these data suggest that porin deficiency in *K. pneumoniae* may increase antimicrobial resistance while simultaneously decreasing virulence. Similar findings have been described for other members of Enterobacteriaceae and for *Neisseria meningitidis*, *P. aeruginosa*, and *Vibrio* spp. (61–64).

Overall, the data strongly suggest a role for outer membrane proteins (porins) in microorganism virulence, as well as in resistance to β-lactams and other antibiotics. Therefore, the acquisition of a phenotypic advantage such as antimicrobial resistance is linked to a deleterious effect on bacterial survival in various hosts and ecological niches.

Efflux pumps. Multidrug efflux pumps of bacterial pathogens are involved in intrinsic and acquired resistance to antimicrobial compounds, including those naturally present at mucosal surfaces; the pumps enable bacteria to grow on such surfaces and can thus be considered as being involved in colonization (65). Moreover, multidrug efflux pumps can discharge molecules involved in the quorum-sensing-regulated expression of virulence determinants (66). Efflux pumps have also been shown to be important for detoxification of intracellular metabolites and are involved in bacterial virulence (in both animal and plant hosts), as well as cell homeostasis and intercellular signal trafficking. Drug efflux can be

FIG 3 Functional role of bacterial multidrug efflux pumps in natural microbial ecosystems. (Reprinted from reference 67 with permission from John Wiley and Sons.)

driven by a proton gradient (such as resistance-nodulation-division [RND], small multidrug resistance [SMR], multidrug and toxic compound extrusion [MATE], and major facilitator superfamily [MFS] transporters) or by energy derived from ATP hydrolysis (ATP-binding cassette [ABC]) (Fig. 3) (67).

Some efflux pumps, especially those in the RND family, have been shown to play a role in the pathogenicity of bacteria, mainly affecting colonization, infection, and the persistence of microorganisms in the host (68). Among Enterobacteriaceae and other Gram-negative rods, the involvement of the AcrAB-TolC efflux pump in antimicrobial resistance (including resistance to β-lactams, tigecycline, and others) and virulence has been extensively studied in species of clinical importance, including *E. coli*, *K. pneumoniae*, *S. enterica* serovar Typhimurium, *Enterobacter aerogenes*, *Enterobacter cloacae*, *Vibrio cholerae*, *Francisella tularensis*, *Burkholderia pseudomallei*, *P. aeruginosa*, *Proteus mirabilis*, and *Brucella* spp. (69–85), as well as nonpathogenic microorganisms such as *Ralstonia solanacearum* and *Erwinia* spp. among others (86, 87). The ability of bacterial pathogens to colonize, infect, and cause disease depends on their capacity to resist antibiotics, antimicrobial compounds produced by the host (such as bile acids, fatty acids, and other detergent-like molecules), and also components of the immune system (e.g., antimicrobial peptides). Clearly, such resistance may be mediated by active efflux systems belonging to the RND family of transporters, which can extrude antimicrobial agents as well as the plethora of different compounds described above. Therefore, abrogation of efflux mechanisms undoubtedly affects both virulence and resistance to antibiotics in clinical practice, in terms of colonization and host

Beceiro et al.

![Figure](https://i.imgur.com/1234567.png)

FIG 4 Results of *in vivo* competition experiments in a mouse model of systemic infection (median CI values are shown by horizontal lines). The figure reflects the competition index values of the *acrA* and *tolC* efflux pump component mutant strains compared with the wild-type EcD64 and JC194 *E. cloacae* clinical isolates. Decreased fitness is observed when the efflux system is inactivated. (Adapted from reference 89.)

of *mexAB-oprM* (*nalB* mutants) or *mexCD-oprJ* (*nfxB* mutants) are the most frequently encountered among clinical isolates (94). Various studies linking efflux pumps to virulence in this species have produced controversial results. Overproduction of the multidrug efflux pump determinants MexAB-OprM and MexCD-OprJ has been associated with decreased survival in water, production of phenazines and proteases, and virulence (94). Linares et al. studied *P. aeruginosa* strains in which overproduction of either MexCD-OprJ or MexEF-OprN was associated with a reduction in the transcription of the type III secretion system (TTSS) regulon due to the lack of expression of the *exsA* gene, which encodes the master regulator of the type III secretion system (95, 96). Finally, a multidrug-resistant *P. aeruginosa* strain with enhanced virulence, designated the “Liverpool epidemic strain” (LES), has recently been described in cystic fibrosis (CF) patients from the United Kingdom (97). This work identified an isolate, LES431, with high levels of β-lactamase activity coupled with upregulation of QS-regulated virulence genes, which because of its history might be considered a highly virulent variant of the clone. Moreover, this strain is characterized by upregulation of the MexAB-OprM efflux pump in relation to resistance to β-lactam antibiotics, and in contrast to the above-described case, it shows greater virulence than the wild-type strain (this issue will be discussed below) (98, 99). Therefore, it appears that other factors in addition to efflux pump upregulation are involved in modulating the virulence of *P. aeruginosa*.

The GacA/RsmA/RsmB (RsmZ) signal transduction system is a conserved pathway involved in a variety of adaptive functions in *P. aeruginosa*. RsmA has numerous homologs in both Gram-negative and Gram-positive bacteria. Loss of RsmA alters the expression of genes involved in a variety of pathways and systems that are important for virulence, including iron acquisition, biosynthesis of the *Pseudomonas* quinolone signal, and the formation of multidrug efflux pumps and motility. Loss of *rsmA* results in increased expression of the genes encoding the MexEF-OprN pump. In *P. aeruginosa*, the MexEF-OprN pump has been found to extrude numerous antibiotics and *Pseudomonas* quinolone signal from the cell. Thus, levels of *mexS* mRNA, which encodes a transcriptional repressor of MexEF-OprN, were four times higher in the *rsmA* mutant than in PAO1, which reflects the complexity of Mex pump regulation (100). In addition to its role in resistance to antibiotics, the MexEF-OprN efflux pump is an important system that modulates the virulence of *P. aeruginosa* through the export of specific quorum-sensing (QS) regulatory molecules, especially 4-hydroxy-2-heptylquinoline (HHQ) (101). Different bacterial virulence factors are regulated by QS, thus highlighting the great potential of this system to attenuate microbial virulence. Moreover, hyperexpression of the MexAB-OprM multidrug efflux system results in reduced levels of several extracellular virulence factors known to be regulated by QS. Autoinducers (AIs) are a family of acylated homoserine lactones found in a number of Gram-negative bacteria, accumulation of which in the growth medium controls cell density and triggers expression of specific genes after reaching a critical concentration (66). After entering cells, the AI (with the aid of a transcriptional activator) activates target genes in response to increased bacterial cell density. Two homoserine lactone AIs (PAI-1 and PAI-2) have been characterized in *P. aeruginosa*, which, together with their QS regulators, LasR and RhlR, act to stimulate production of a number of extracellular virulence factors in *P. aeruginosa*. Importantly, reentry of PAI-1 is

Clinical Microbiology Reviews

reduced by the efflux activity of *P. aeruginosa* pumps, which leads to a reduction in the expression of PAI-1-dependent genes (including virulence factors). This MexAB-OprM mechanism constitutes a link between increased antimicrobial resistance and decreased QS-mediated virulence in *P. aeruginosa* (66, 102).

The SecDF system has been studied in *E. coli*, *Bacillus subtilis*, and *S. aureus*. In *S. aureus*, the SecDF system has been associated with impaired cell division, reduced resistance to the β-lactam oxacillin and to the glycopeptide vancomycin, and altered expression of virulence genes (*atl, coa, hla, hld*, and *spa*) (103).

### Resistance to Aminoglycosides and Effect on Virulence

Aminoglycosides are used to treat a wide range of infections caused by both Gram-negative and Gram-positive bacteria. The mechanism of action is related to binding to the A site of the 16S rRNA, which forces A1492 and A1493 to “flip out” of helix 44 (104). This ribosome interaction leads to a loss of translational fidelity and thus to the accumulation of erroneous proteins and bacterial cell death. Resistance to aminoglycosides is usually due to production of aminoglycoside-modifying enzymes (acetyltransferases [AACs], phosphoryltransferases [APHs], and adenyltransferases [ANTs]), reduced intracellular antibiotic accumulation (by outer membrane alterations, two-component systems or efflux pumps), or mutation of ribosomal proteins or RNA (105). Methylation of the aminoacyl site of 16S rRNA also confers a high level of resistance to clinically important aminoglycosides, such as amikacin, tobramycin, and gentamicin. This event is mediated by a group of 16S rRNA methyltransferases. Six acquired 16S rRNA methyltransferase genes confer resistance to aminoglycosides, namely, *armA, rmtA, rmtB, rmtC, rmtD*, and *rmtE*.

#### Aminoglycoside-modifying enzymes

Although no direct relationship between virulence and resistance has been found, aminoglycoside resistance genes (encoding AACs, APHs, or ANTs) are often located in different MGEs, such as integrons and plasmids (106–108). These genes are therefore usually coselected with other antibiotic resistance genes, such as those encoding β-lactamases.

#### Efflux pumps

Expression of efflux pumps is usually associated with increased MICs of the clinically used aminoglycosides (amikacin, gentamicin, or tobramycin). Examples include efflux pumps belonging to the RND family, such as AcrAB-TolC of *E. cloacae* (109), AdeABC of *A. baumannii* (110), or MexXY of *P. aeruginosa* (105). The impact of efflux pump expression on virulence is described above in the section on β-lactams and resistance conferred by efflux pumps. Thus, the previous discussion can also be applied to aminoglycosides.

#### Ribosomal methylases

In microorganisms, ArmA/Rmt methyltransferases have recently been found to cause a high level of resistance to 4,6-disubstituted aminoglycosides through methylation of the G1405 residue in 16S rRNA (such as ArmA and RmtA to -E). In prokaryote ribosomes, methylation of rRNA is necessary to optimize ribosomal function, so that prokaryotic microorganisms harbor housekeeping methyltransferases in order to modulate this function. In *E. coli*, the RsmF housekeeping methyltransferase methylates at C1407, a nucleotide that is very close to G1405 (the position of RmtC methylation) and that forms part of the same aminoglycoside-binding pocket of the 16S rRNA. By using a series of isogenic *E. coli* mutants, Gutierrez et al. showed that acquisition of RmtC does not have a fitness cost for the bacterium: coupling between housekeeping and acquired methylases subverts the methylation architecture of the 16S rRNA and favors selection of

Arm/Rmt methyltransferases, thus constituting a potential threat when microorganisms are treated with aminoglycosides (104).

Ribosomal mutations. The tendency of aminoglycoside-resistant bacteria to have a competitive disadvantage in an environment free of antibiotics may be related to the nature of the resistance mechanism. For instance, numerous types of resistance that are mediated by chromosomal mutations result from target alterations. If this occurs, the final fitness cost may become apparent if the target alterations associated with the resistance phenotype display suboptimal functionality, as shown for mutations in the gene *rpsL*, which confers resistance to streptomycin and other aminoglycosides in *E. coli*, *Salmonella* spp., and Gram-positive bacteria, including *Mycobacterium tuberculosis* (13, 111–115). Acquired gentamicin resistance mediated by permeability changes in *Enterococcus faecalis* has been described, although the impact on virulence was not evaluated (116).

In conclusion, although chromosomal mutations that result in ribosomal alteration and resistance affect fitness, they will probably be selected in microorganisms only under high aminoglycoside pressure. This would explain why this mechanism is not often identified in the clinical setting. Thus, microorganisms will be selected by other antimicrobial resistance mechanisms with a lower fitness cost, such as aminoglycoside-modifying enzymes, which cannot yield any fitness cost in the absence of antibiotic pressure.

### Resistance to Fluoroquinolones and Effect on Virulence

Several mechanisms of resistance to quinolones have been described: target modification (topoisomerase and DNA gyrase mutations), efflux pumps, Qnr (plasmid-mediated gene encoding quinolone resistance), porins, and quinolone-modifying enzymes (117). The primary targets of quinolones differ in Gram-negative and Gram-positive bacteria (DNA gyrase and topoisomerase IV, respectively). Modifications to both targets result in resistance to quinolones. As mentioned above, multidrug efflux pumps are involved in resistance to different antibiotics, including quinolones. Nonetheless, the widespread presence of plasmid-mediated quinolone resistance determinants, particularly *qnr* genes, has become apparent. In the worldwide clone of *E. coli* 025-ST131, plasmids harboring different quinolone-resistance genes have been reported; these include *qnrB2* and *aac(6′)-Ib-cr* (118). Another mechanism involved in resistance to quinolones is a decrease in the expression of outer membrane proteins (porins). So far, there are no reports relating quinolone-modifying enzymes to virulence.

#### Target modification (topoisomerase and DNA gyrase mutations)

The cost of quinolone resistance caused by topoisomerase mutations has been studied in *P. aeruginosa, Salmonella* spp., *Shigella flexneri, E. coli*, and *A. baumannii* (Table 1). Kugelberg et al. analyzed *P. aeruginosa* norfloxacin-resistant mutants and showed that “no-cost” and compensatory mutations are common in quinolone-resistant *P. aeruginosa* with mutations in topoisomerase. The authors commented on the significance of these mutations for the persistence of bacteria resistant to quinolones and established the importance of implementing strategies for treating and preventing the spread of resistant strains before they become stable bacterial populations (119). In contrast, the high level of resistance to ciprofloxacin in *in vitro*-derived mutants of *S. enterica* is associated with fitness costs. In the absence of evidence of compensatory evolution, such fitness costs may account for the lack of emergence and spread (to date) of highly resistant *S. enterica* clones via the farm-to-fork route (120). Moreover, quinolone-

resistant Salmonella spp. usually contain chromosomal point mutations that result in alterations of the A subunit of DNA gyrase. Similar findings have been reported for ciprofloxacin resistance in *A. baumannii* showing mutations in topoisomerase and DNA gyrase (121, 122). However, with strains of *S.* Typhimurium that are resistant to nalidixic acid, infection was found to be associated with a 3.15-fold risk of invasive illness or death within 90 days of infection compared with that observed for infection with pansusceptible strains (123), which revealed apparently contradictory results. Other Enterobacteriaceae species (e.g., *Shigella flexneri*) that overexpress topoisomerase IV genes can compensate for the loss of topoisomerase I in terms of DNA supercoiling, virulence gene expression (partially), and cell viability in general (124). Finally, *E. coli* resistant to ciprofloxacin due to *parC* and *gyrA* mutations showed no decreases in *in vitro* or *in vivo* growth rates (125). The biological cost of quinolone resistance differs between bacterial species and depends on the level of resistance and the number of resistance mutations, and highly resistant mutants with multiple mutations show a significantly lower level of fitness than the wild-type strains. However, for low-level-resistant mutants with single mutations, the cost depends on the bacterial species.

In strains of *P. aeruginosa*, mutations in *gyrA* at codon Ile83Thr and in *parC* at codon Leu87Ser have been associated with expression of the type III secretion system (TTSS) genes (*exoS*, *exoT*, *exoU*, and *exoY*), which are involved in virulence (126). In another study of infections caused by clinical strains of *P. aeruginosa*, fluoroquinolone resistance and type III secretion system virulence were independently associated with poor patient outcome. The observed positive relationship between fluoroquinolone resistance, carriage of *exoU* (encoding a cytotoxin), and enhanced cytotoxicity suggests that adverse outcomes related to fluoroquinolone-resistant *P. aeruginosa* infections may be attributable to associated enhanced TTSS-mediated virulence. The results of this study suggest coselection of fluoroquinolone resistance and enhanced TTSS-mediated virulence in *P. aeruginosa* clinical isolates harboring *exoU* (127). Finally, in a recent study, Agnello and Wong-Beringer emphasized concern about the problem in the near future in regard to the coevolution of resistance to quinolones (mutations in *gyrA*, *gyrB*, *parC*, and *parE*) and increased virulence (expression of ExoU and ExoS), which will favor the development of virulent genotypes, particularly in quinolone-rich environments (128).

Studies of Gram-positive bacteria have focused mainly on *S. aureus* and *S. pneumoniae*. Several studies have associated the resistance of *S. aureus* to ciprofloxacin (mutations in DNA gyrase) with the *sarA* gene regulator (see “Alternative Sigma Factor σ<sup>B</sup>” below) (129, 130). However, the use of subinhibitory concentrations of ciprofloxacin to treat infections caused by *S. aureus* strains not only selects for highly resistant strains but also induces the production of virulence factors such as fibronectin-binding proteins, which may promote persistent infection among drug-resistant survivors (131). In a comparison between isogenic quinolone-resistant (IQR) isolates of *S. pneumoniae* and their fluoroquinolone-susceptible parents with mutations in GyrA (Ser81Phe) and ParC (Ser79Phe), the relative growth efficiencies revealed a reduction in nasal colonization for the resistant isolates during competitive or noncompetitive lung infections. Moreover, isogenic quinolone resistance may have reduced ability to initiate infections in the absence of fluoroquinolone selection. This suggests that the

correct use of antimicrobial drugs may maintain the prevalence of IQR clones at low levels because of the relatively low fitness of these clones (132).

**Efflux pumps.** Efflux pump systems have been studied in relation to β-lactam resistance, and the results can be extrapolated to quinolone resistance. However, in this section we discuss some specific examples of efflux pumps involved in quinolone resistance and virulence, such as the already-described AcrAB-TolC (*Enterobacteriaceae*) and Mex (*P. aeruginosa*) systems, as well as MgrA (a global regulator of the NorA, NorB, NorC, Tet38, and AbcA efflux pumps in *S. aureus*), BepDE (*Brucella suis*), and SmeDEF (*Stenotrophomonas maltophilia*), which will be described here and summarized in Table 1.

Study of the relationship between efflux-mediated resistance and virulence of *S. aureus* can be addressed by analysis of global regulators such as the MgrA type, which is a homolog of MarR and SarA and is involved in regulation of the expression of α-hemolysin, protein A, lipase, protease, and coagulase, which are virulence gene products as well as autolysins, and type 8 capsular polysaccharide. Moreover, multidrug resistance efflux transporters such as NorA, NorB, NorC, Tet38, and AbcA, which are involved in resistance to quinolones, are controlled by the MgrA regulator (133).

Two functional RND efflux pumps that may contribute to virulence have been studied in *Brucella suis*. Specifically, there are two membrane fusion protein-RND translocases of *B. suis*, encoded by the *bepDE* and *bepFG* loci. The resistance profile of *B. suis* in the presence of multicopy *bepDE* indicates that BepDE is able to extrude antibiotics, including tetracycline, doxycycline, and fluoroquinolones. Only the BepFG-defective mutant showed moderate attenuation in *in vitro* assays with HeLa cells, and the activities of both *bepDE* and *bepFG* promoters were induced in the intracellular environment of HeLa cells (134).

The SmeDEF efflux pump has been studied in *Stenotrophomonas maltophilia* showing resistance to quinolones and erythromycin (135). Overexpression of the system in this bacterium was associated with decreased fitness as well as a decline in cell size when isolates were grown in a rich medium (136).

**Qnr proteins.** The plasmid-mediated *qnr* genes products are pentapeptide repeat proteins, which lead to quinolone resistance. The mechanism of action, which is based on protection of DNA gyrase and topoisomerase IV, has been studied in great detail in strains with the *qnrA1* gene and is presumably similar to the mode of action of other Qnr proteins (117, 137). Michon et al. determined the effect of *qnr* acquisition on fitness by examining *in vitro* growth curves and studying *in vitro* pairwise competition and *in vivo* single culture and pairwise competition (138). These authors concluded that plasmidic *qnrA3* enhances *Escherichia coli* fitness in the absence of antibiotic exposure, thus favoring its possible spreading.

**Porins.** The role of porins in binonium resistance/virulence has already been discussed in the section on resistance to β-lactams, and the conclusions can be extrapolated to fluoroquinolones.

## Resistance to Tetracyclines and Tigecycline and Effect on Virulence

Tetracyclines are bacteriostatic antibiotics that bind reversibly to the ribosome 30S subunit, thereby inhibiting the initiation of protein synthesis; these antibiotics enter bacterial cells by passive diffusion or by active transport. The main resistance mechanisms are efflux systems encoded by the *tet* genes, such as *tetA* and *tetB* (also
involved in virulence), although other mechanisms of resistance to tetracyclines have been described, such as ribosomal protection and antibiotic modification (*tetX*) (139). Although studies investigating links between tetracycline resistance mechanisms and virulence/fitness are scarce, here we report some examples of the main mechanisms of resistance to tetracyclines involved in virulence (summarized in Table 1).

Efflux pumps. As the Tet efflux systems are monocomponent, structurally different from the RND efflux family, and specific for tetracyclines, they are discussed separately. A study in the absence of selective pressure was performed in infants during the first year of life. Although the children were not exposed to tetracycline, resistance to this antibiotic was detected in 12% of the strains of *E. coli* collected, all of them with *tetA* or *tetB* genes encoding efflux pumps. The *tetA*- and *tetB*-positive strains carried the virulence genes for P fimbriae and aerobactin, respectively, more often than the susceptible strains. However, strains that were resistant and susceptible were simultaneously isolated from 11 of the children; the tetracycline-resistant strains were present in significantly lower numbers, which indicated reduced fitness (140). In this example, it can be seen that even in the absence of antibiotic pressure (tetracyclines), a high percentage of strains express resistance mechanisms and different virulence genes. However, this genetic burden involves a certain fitness cost. Although it is not clear why the tetracycline-resistant strains carried P fimbria and aerobactin genes more often than the susceptible strains, the authors hypothesized that these genes increase the persistence of *E. coli* in the intestinal microbiota, thus increasing the possibility of contact between the bacteria and the antibiotic, and then selection of resistant strains.

Tet efflux pumps also can be transported in MGEs, such as *tetG* in *Salmonella* genomic island 1 (SGI1), which contains 6 to 9 virulence determinants and a multidrug resistance region that confers resistance to tetracyclines (and other antimicrobials). Because of their repertory of resistance and virulence genes, these types of isolates may become clinically relevant (141).

Ribosomal protection. Few studies have investigated resistance due to ribosomal modifications and virulence. The *tet*(44) gene, which has been reported to be involved in resistance in *Campylobacter fetus*, is located in a pathogenicity island (PAI) that encodes the components necessary for the bacterial type IV secretion system (142) showing a coselection phenomenon.

In *Helicobacter pylori*, resistance to tetracycline is uncommon and is due to the accumulation of changes that affect ribosome affinity (mutations in the 16S rRNA gene) and possibly to other functions such as efflux systems and decreased expression of porins. Dailidiene et al. suggested that the rare tetracycline resistance in *H. pylori* is probably due to these multiple mutations with deleterious effects on fitness and the accumulation of many defective functions (143).

Antibiotic modification. In addition to efflux and ribosomal protection, there is a third resistance mechanism, i.e., the modification or inactivation of tetracyclines by one of the flavin-dependent monooxygenases encoded by the *tetX*, *tet*(34), and *tet*(37) genes, which have similar properties. The *tetX* gene, which was found in an isolate of the anaerobe *Bacteroides fragilis*, confers resistance when it is transformed in *E. coli*; however, so far this mechanism has not been found in any clinical isolates (144, 145). No studies of *tetX* and its fitness/virulence cost have been published to date, but the absence of other clinical pathogens with this

mechanism indicates that its expression may have a significant cost in Gram-negative aerobes.

Tigecycline. Tigecycline is one of the glycylcyclines, a new class of antimicrobials derived from tetracyclines. Mechanisms of tigecycline resistance, such as the RND efflux system, also provide resistance to tetracyclines. Although tigecycline remains more active than other antimicrobials usually tested against MDR pathogens (e.g., fluoroquinolones and β-lactams), several tigecycline-resistant strains have been described in recent years (146).

*P. aeruginosa*, *P. mirabilis*, and *Morganella morganii* are intrinsically resistant to tigecycline because of the expression of RND family transporters such as MexAB-OprM, MexCD-OprJ, and AcrAB (83, 85, 147). Upregulation of the AcrAB efflux system, which confers tigecycline resistance, has been described in *E. coli*, *K. pneumoniae*, and *E. cloacae* (81, 82, 84), and the AdeABC efflux appears to be involved in tigecycline resistance in *A. baumannii* (148, 149) (Table 1). Thus, the increased resistance to tigecycline mediated by efflux systems will be directly involved in changes in the bacterial virulence, as reviewed in “Efflux pumps” above.

Resistance to Macrolides and Effect on Virulence

In Gram-negative bacteria, resistance to macrolides is associated mainly with the overexpression of efflux pumps. However, in Gram-positive pathogens, resistance to macrolides has been associated with efflux pumps (*mef* class genes), which confer a phenotype of low-level resistance, and alteration of the target site by ribosomal methylation (*erm* class genes), which is involved in high-level resistance to all macrolides, lincomamides, and streptogramin B (122).

Efflux pumps. Three efflux pumps have been associated with macrolide resistance and with virulence mechanisms: MuxABC-Omp in *P. aeruginosa*, BpEAB-OprB in *Burkholderia pseudomallei* KHM, and MtrC-MtrD-MtrE in *Neisseria gonorrhoeae* (Table 1).

Recently, the MuxABC-Omp system has been shown to play a role in resistance to novobiocin, aztreonam, macrolides, and tetracycline in a laboratory mutant of *P. aeruginosa* (150). Inactivation of *muxA* in the *muxABC-ompB* operon showed attenuated virulence associated with decreased twitching motility and elevated resistance to ampicillin and carbenicillin (151).

The BpEAB-OprB system has been reported to act as an efflux pump for aminoglycosides and macrolides in *B. pseudomallei* strain KHM (from Singapore) and to play an important role in virulence and quorum sensing. This efflux pump was also required for optimal production of biofilms, siderophores, and phospholipase C; the excretion of acyl homoserine lactones (AHLs) (quorum-sensing molecules), also depended on BpeAB-OprB function (80, 152). However, Mima and Schweizer obtained different results with *B. pseudomallei* strain 1026b (153). These authors concluded that BpeAB-OprB does not mediate efflux of aminoglycosides but that it extrudes other antibiotics, including fluoroquinolones, clindamycin, macrolides, and tetracyclines. On the other hand, BpeAB-OprB mutants were not impaired in excretion of acyl homoserine lactones, swimming motility, or siderophore production, but AmrAB-OprA and BpeAB-OprB mutants were impaired in biofilm formation (153).

The MtrC-MtrD-MtrE efflux pump system of *N. gonorrhoeae* is also involved in virulence and resistance to antibiotics. In an experimental gonococcal genital tract infection, derepression of the *mtrCDE* operon, via deletion of *mtrR*, was found to confer
increased fitness *in vivo* and increased the antimicrobial resistance *in vitro* (154).

**Ribosomal methylation and modification.** Two mechanisms of macrolide resistance based on methylation ribosomal have been studied in relation to virulence: the *erm* gene and 23S rRNA mutations (Table 1).

Regarding methylases, *E. faecalis* isolates from swine livestock in China have shown a significant correlation between the presence of the *gelE* virulence gene (gelatinase protein) and erythromycin resistance (*erm* gene). The authors suggested that enterococci isolated from swine should be regarded cautiously because they may act as reservoirs for antimicrobial resistance and virulence genes (155).

Macrolide resistance and associated fitness costs have been reported in *Campylobacter jejuni* (156, 157), and the 23S rRNA mutation A2074C, which confers a high level of macrolide resistance, was associated with a fitness cost in this bacterium (156). Moreover, Zeitouni et al. reported *in vitro* experiments that demonstrated that macrolide resistance imposed a fitness cost; however, a spontaneous mutant that evolved *in vivo* had a colonization capacity that was similar to that of the susceptible strain (158). The effect of ribosomal mutations on antimicrobial resistance and fitness cost was described above (see “Resistance to Aminoglycosides and Effect on Virulence”).

Deletions in *S. pneumoniae* ribosomal protein L4 have also been related to resistance to macrolides, oxazolidinones, and chloramphenicol and to low fitness (159). Decreased growth rates may also be due to the fact that L4 mutations perturb the structure of 23S rRNA. Bacteria may develop compensatory mutations to adapt to a loss of fitness (160). This topic will be discussed in the Compensatory Mutations section below.

**Resistance to Glycopeptides and Effect on Virulence**

*S. aureus* can become less susceptible to vancomycin in two different ways. One mechanism, which confers a moderate increase in MICs (resistance to vancomycin or glycopeptides, known as the vancomycin-intermediate *S. aureus* [VISA] or glycopeptide-intermediate *S. aureus* [GISA] phenotype, respectively), is mediated by chromosomal mutations that alter the cell wall physiology and result in increased cell wall thickness, which limits the access of glycopeptides to the D-Ala-D-Ala target in the peptidoglycan precursors (122). In another mechanism, *S. aureus* and *Enterococcus* spp. can develop full glycopeptide resistance by acquisition of *van* genes, which are responsible for the production of a modified peptidoglycan in which the precursors no longer contain the D-Ala-D-Ala dipeptide but contain a glycopeptide-resistant depsipeptide comprising D-Ala-D-Lac or D-Ala-D-Ser. VanA and VanB are the most prevalent types in *Enterococcus* clinical isolates, and VanA is most prevalent in the case of isolates of *S. aureus*.

The effects of these mechanisms on virulence are discussed below and summarized in Table 1.

**Cell wall modifications.** In an *in vivo* model of endocarditis (involving cardiac vegetations in mice), GISA isolates showed attenuated virulence, which was probably due to faster clearance from the blood and reduced fitness. Isolates with the GISA phenotype showed lowered infectivity, which could be explained by altered expression of global virulence regulators (130, 161, 162). Moreover, Peleg et al. found similar results in relation to the reduced susceptibility to vancomycin and the decreased pathogenicity of *S. aureus* in a nonmammalian model system (*Galleria mellonella*) (162).

In *S. aureus*, resistance to teicoplanin is associated with lowered fitness resulting from slower growth, thickening of the bacterial cell wall, increased N-acetylglucosamine incorporation, and decreased hemolysis (163). Transcriptomic studies showed downregulation of some virulence-associated genes as resistance increased. Elimination of the teicoplanin-resistant mutants and selection of teicoplanin-hypersusceptible survivors in the tissue cages indicated that glycopeptide resistance imposes a fitness burden on *S. aureus* and is selected against *in vivo*, with restoration of fitness leading to loss of resistance. Renzoni et al. found that teicoplanin-resistant methicillin-resistant *S. aureus* (MRSA) strains were more prevalent in intracellular locations, which might confer a significant fitness benefit over teicoplanin-susceptible MRSA strains and further protect them from cell wall-active antibiotics in which intracellular activity is limited (164).

**Modified peptidoglycan target (D-Ala-D-Lac or D-Ala-D-Ser).** A study of the variation in virulence of glycopeptide-resistant *E. faecalis* (VanA and VanB phenotypes) in a rat polymicrobial model of peritonitis suggests variations in virulence between *E. faecalis* that have the VanA and VanB gene cluster and those that do not. Slightly decreased bacterial proliferation and increased peritoneal and systemic inflammatory responses were observed for two transconjugant *E. faecalis* strains expressing VanA and VanB. The authors concluded that the increased fitness cost could lead to decreased concentrations of bacteria in the peritoneal fluid. However, the *in vitro* growth rate did not differ significantly between the strains, suggesting that the cost of resistance is not of major importance in these *E. faecalis* strains, especially strains containing the *vanA* gene (165).

Other studies show that induction of resistance to vancomycin greatly reduces the fitness of *Enterococcus* spp. For example, Foucault et al. showed that constitutive or induced resistance led to a great reduction in growth rate, colonization, and transmission of *Enterococcus* spp., which may explain the low occurrence of constitutively resistant clinical isolates. However, these authors also showed that both *in vitro* and *in vivo*, carriage of Tn1549 (encoding VanB-type vancomycin resistance), when inducible, had no cost to the *Enterococcus* host. These findings indicate that the regulation of resistance mediated by a two-component regulatory system drastically reduces the biological cost associated with vancomycin resistance (166). Moreover, the same authors reported that induction of *S. aureus* VanA-type resistance is costly for the MRSA host, whereas in the absence of induction, its biological cost is minimal (167). Thus, the potential for the dissemination of vancomycin-resistant *S. aureus* (VRSA) clinical isolates should not be underestimated.

**Resistance to Oxazolidinones (Linezolid) and Effect on Virulence**

Resistance or reduced susceptibility to oxazolidinones remains very uncommon among staphylococci (especially *S. aureus*) and has mainly been reported to be associated with chromosomal mutations (rRNA mutations and rRNA methylation by the Cfr methyltransferase) brought about by prolonged exposure to this type of drug. Resistance to several antiribosomal agents, including oxazolidinones, phenicols, lincomamides, and streptogramin A (PhLOPSA phenotype), is associated with the ribosomal modification brought about by the Cfr protein (168). Virulence studies of rRNA
mutations, such as the rRNA methylation associated with resistance to oxazolidones (linezolid [LZD]), have been carried out (Table 1).

**rRNA mutations.** In *S. aureus*, the G2576T mutation in 23S rRNA genes, resulting in ribosomal target modification, caused resistance to linezolid (LZD) that was proportional to the number of mutated copies present in the bacterial genome (169). Mutational resistance usually occurs in coagulase-negative staphylococci (170). Other mutations associated with LZD resistance include A2059G and G2057A in enterococci and *S. pneumoniae*, respectively (122). Moreover, in *S. pneumoniae*, both the G2576T mutation in the 23S rRNA gene and mutations in *spr0188* (50S ribosomal protein L3), *spr01887* (ABC transporter ATP-binding/membrane-spanning protein), and *spr0333* (conserved hypothetical/rRNA methyltransferase) conferred resistance to LZD. These compensatory mutations in both *S. aureus* and *S. pneumoniae* have been associated with changes in the fitness cost (Table 1). Hence, a combination of whole-genome transformation and sequencing was carried out to demonstrate the dual role of these mutations in both linezolid resistance and fitness compensation (171) (see Compensatory Mutations below).

**rRNA methylation.** The *cfr* gene was originally discovered in a multiresistant *Staphylococcus* isolate of animal origin (172). This gene encodes an RNA methyltransferase that modifies a conserved adenine at position 2503 of the 23S rRNA. This gene is typically plasmid borne in *S. aureus* isolates. *cfr* acquisition in *S. aureus* has been associated with low fitness cost. This could be the explanation for the apparent spread of the *cfr* gene among pathogens. However, in some clinical isolates of *S. aureus*, *cfr* coexpressed with the *erm* gene, which encodes a methyltransferase with another targeting 23S rRNA residue, A2058. The dimethylation of A2058 by Erm increases the fitness cost associated with Cfr-mediated methylation of A2503 (173, 174).

In *S. pneumoniae*, deletions in ribosomal protein L4 have been associated with resistance to oxazolidinones, macrolides, and chloramphenicol (see “Resistance to Macrolides and Effect on Virulence” above).

### Resistance to Colistin, Polymyxin B, and Antimicrobial Peptides and Effect on Virulence

#### LPS modifications. Several mechanisms of resistance to polymyxins such as colistin and other antimicrobial peptides have been described, most of which are related to changes in lipopolysaccharides (LPS) and loss of affinity, mediated by two-component systems (TCSs), which will be discussed below and also in a separate section. Other mechanisms involved in antimicrobial peptide resistance include efflux systems and increased production of nonessential targets (Table 1).

The relationship between resistance to antimicrobial peptides (such as polymyxins) and virulence has been best characterized in *Salmonella* spp. Multiple genes are thought to be involved in LPS modification by addition of 4-aminoarabinose (Ara4N), which creates a positively charged LPS with low affinity and reduced binding to antimicrobial peptides. The ubiquitous two-component regulatory system PmrA-PmrB is one of the most important systems for increasing antimicrobial peptide resistance in response to environmental conditions. *pmrA* and *pmrF* operon mutants were unable to add Ara4N to lipid A but caused decreased virulence against the wild-type strains when orally administered, although not when they were intraperitoneally administered, in an *in vivo* mouse infection model. Thus, the addition of Ara4N is probably important to overcome the innate defenses of the host in intestinal tissues (175). Strains with reduced susceptibility to colistin showed mutations in *pmrA* and *pmrB* but also exhibited low fitness costs such as slightly slower growth *in vitro*; growth rates were unaffected in mouse infection (176). Other genes not regulated by PmrA-PmrB, such as *surA*, *tolB*, and *gnd*, which are necessary to develop polymyxin resistance, are also thought to be involved in virulence, as decreased virulence is observed in mice infected with strains with mutations of these genes (177).

This complex regulation system is able to activate genes involved in multiple bacterial functions and with different regulatory activities in each genus or species. For instance, in the phylogenetically close species *Yersinia pseudotuberculosis*, the PmrF operon is not PmrA-PmrB regulated as in *Salmonella* spp. but is regulated by another two-component system, the PhoP-PhoQ system. In *Y. pseudotuberculosis*, the PmrF operon is essential to develop polymyxin resistance but is not involved in virulence, unlike in *Salmonella* spp. (178). The PhoP-PhoQ operon is another important two-component system (179, 180) that can regulate membrane proteins, such as UgtL, YqjA, and PagP (181, 182), which are capable of modifying lipid A, thus inducing resistance to antimicrobial peptides, and which are involved in different ways in virulence in *Salmonella* spp. (180–185).

It is interesting to highlight the resistance to colistin observed in two extremely multidrug-resistant pathogens, *A. baumannii* and *P. aeruginosa*. Since the reintroduction of colistin for the treatment of infections caused by these agents over the past decade, colistin-resistant strains of *A. baumannii* have emerged both *in vitro* (186, 187) and *in vivo* during the course of clinical infections (188, 189). So far, two mechanisms of colistin resistance have been described in *A. baumannii*. The first involves total loss of LPS caused by a deficiency in lipid A biosynthesis, by inactivation of one of three genes involved, *lpxA*, *lpxD*, or *lpxC* (190, 191). In the second mechanism (studied by our research group), lipid A modification mediated by the addition of phosphoethanolamine results in loss of colistin affinity and subsequent resistance; these modifications are regulated by the PmrAB two-component system (186, 192). LPS and lipid A from *A. baumannii* are involved in diverse biological activities such as toxicity, pyrogenicity, mitogenicity, and activation of a proinflammatory immune response. It has been suggested that modification or loss of LPS production can increase resistance to colistin, while simultaneously decreasing bacterial fitness and virulence (193, 194). Although loss of infectivity has been observed in a strain with a high level of resistance to colistin, selected during treatment (194), it is not clear whether this is a universal phenomenon. Moreover, a colistin-resistant clone has been identified in Spain; this clone was selected during treatment with its infectivity intact, but it presented an altered antimicrobial resistance profile and became more susceptible to other antibiotics than the parental strain. Similar alterations in the MICs caused by LPS modification have been observed in other studies (190). Thus, there appears to be an inverse relationship between the increase in colistin resistance in *A. baumannii* mediated by changes in the LPS and the virulence of the species, or at least in the loss of resistance to other antibiotics, which can be considered a loss of fitness/virulence in an environment with high antibiotic pressure. In *P. aeruginosa*, the increased virulence is associated with increased resistance to polymyxin B and also to gentamicin and ciprofloxacin when the virulence is due to
Beceiro et al.

TABLE 2 Highly virulent and multiresistant worldwide disseminated clones

| Clone                | Species            | Antibiotic resistance                                                                 | Virulence                                                                                          | Origin (yr)           | Reference(s) |
|----------------------|--------------------|-------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|-----------------------|--------------|
| CC17                 | E. faecalis        | β-Lactams, quinolones, glycopeptides                                                 | Putative pathogenicity island; toxin Esp                                                          | USA (1982)            | 206, 208     |
| PMEN1 (Spain23FST81) | S. pneumoniae      | β-Lactams, chloramphenicol, tetracycline, fluoroquinolones, macrolides                  | blp bacteriocin locus                                                                              | Spain (1970)          | 209, 210     |
| ST131 (O25:H4)       | E. coli            | β-Lactams, trimethoprim, fluoroquinolones, tetracyclines, aminoglycosides, macrolides    | vagC-vagD                                                                                         | Calgary, Canada (2000) | 215          |
| Liverpool epidemic   | P. aeruginosa      | β-Lactams, fluoroquinolones, aminoglycosides                                          | Upregulation of MexAB-OprM; quorum-sensing system                                                   | UK (1988)             | 97, 98       |
| NAP1/027             | C. difficile       | Quinolones                                                                         | Hyperproduction of toxins TcdA and TcdB; protease Cwp84, the high-molecular-wt S-layer protein, and fibronectin-binding protein Fbp68 | North America, Europe | 217          |
| BVE                  | E. faecalis        | β-Lactams, glycopeptides                                                            | Putative pathogenicity island                                                                     | USA (mid-1980)        | 218          |
| USA300               | S. aureus         | β-Lactams (methicillin)                                                             | Toxins (PVL), adhesins, enzymes, and immunomodulators                                              | USA                   | 219          |
| ST175                | P. aeruginosa      | β-Lactams, fluoroquinolones, aminoglycosides                                          | AmpR-activating mutations                                                                         | Europe                | 203, 221     |

the development of swarming motility. This complex adaptation produces overexpression of different virulence factors, such as the type III secretion system, extracellular proteases, and factors associated with iron transport. The swarming cells of *P. aeruginosa* PAO1 exhibit greater resistance to those antibiotics, and although the molecular basis of this resistance has not been studied, it is probably influenced by the complex modifications necessary for this motility (195). Gooderham et al. have studied a *P. aeruginosa psrA* mutant, which exhibited supersusceptibility to polymyxin B and indocilin, owing to outer membrane permeabilization. This is thought to be a consequence of altered metabolism due to up- or downregulation of 178 genes in the *psrA* mutant, including dysregulated genes involved in virulence factors such as a type III secretion system, biofilm formation, and adhesion or motility (196). The TCS PhoP-PhoQ has also been shown to be involved in resistance and virulence in this species. The PhoQ sensor protein plays a major role in polymyxin B resistance, and it has also been demonstrated that mutation of *phoQ* caused reduced biofilm formation, attachment to epithelial cells, cytotoxicity, virulence in lettuce leaf, and competitiveness in rat lung infections (197).

Other antimicrobial peptide resistance mechanisms, which are possibly not PhoP-PhoQ or PmrA-PmrB regulated, have been described in *Salmonella* spp., such as the putative ABC transporter encoded by the *yejABEF* operon. Mutants with mutations of this transporter system (*ΔyejF*) showed increased susceptibility to polymyxin B and human defensins, as well as decreased proliferation capacity inside macrophages and epithelial cells. Decreased virulence was also observed in mouse gastric infection models (198).

Increased production of nonessential antimicrobial targets. Outer membrane vesicles (OMVs) are produced by most Gram-negative bacteria. They have been associated with resistance to antimicrobial peptides, such as colistin and polymyxin B; for example, an *E. coli* mutant that overproduces OMVs can survive treatments with these antibiotics due to the absorption capacity of the OMVs, probably because of the presence of surface constituents similar to that of the OM. Addition of OMVs to the culture medium increases the rate of survival against colistin or polymyxin B. Treatment with antimicrobial peptides induces vesicle production in the enterotoxigenic *E. coli* (ETEC) and K-12 *E. coli* ATCC strains, thereby altering their antibiotic resistance phenotype. Hypervesiculation may be an induced innate immune bac-

HIGHLY VIRULENT AND MULTIREESISTANT WORLDWIDE DISSEMINATED CLONES

In recent years, clones that are resistant to many antibiotics and carry virulence factors have spread globally, and they are considered highly successful or high-risk clones (202, 203). These clones are derived from both animals (204) and humans (205). In this section, we highlight some of the most important of these clones, which are also included in Table 2.

(i) Dissemination of the *Enterococcus faecium* epidemic-virulent clonal complex 17 (CC17), which is associated with the majority of hospital outbreaks and clinical infections of this species in five continents, is extremely widespread. This complex is characterized by ampicillin and quinolone resistance and by the presence of a putative pathogenicity island (206, 207). Some authors believe that *E. faecium* CC17 isolates have been circulating around the world for more than 30 years and that they have progressively acquired additional virulence and antibiotic resistance determinants, which would explain the recent success of this species in the hospital environment. An interesting study performed by Billström et al. aimed to determine the putative relationship between virulence and antibiotic resistance in *E. faecium* blood culture isolates over a 6-year period. Two hundred sixty-three isolates were screened, and the presence of Esp (enterococcal surface protein, which is involved in virulence) was significantly correlated with resistance to ampicillin, ciprofloxacin, and imipenem (*P* < 0.01)
(208). Overall, the data suggest a positive correlation between resistance and virulence in *Enterococcus* spp. (at least in *E. faecium*), in contrast with that found in *S. aureus* and *S. pneumoniae*.

(ii) *Streptococcus pneumoniae* clone PMEN1 (Spain23FST81), which is estimated to have originated around 1970, is widely distributed throughout Europe, Asia, Africa, and the Americas. PMEN1 isolates are multilocus sequence type (MLST) 81, have a common pulsed-field gel electrophoresis (PFGE) profile, possess consistent multilocus enzyme electrophoresis (MEE) patterns, and have identical PBP patterns and ribotypes. In addition to penicillin, most PMEN1 isolates are also resistant to chloramphenicol and tetracycline, and many isolates also exhibit additional resistance to fluoroquinolones and macrolides (209). This clone has several virulence factors, even at the *blp* bacteriocin locus. This locus may also play a role in systemic virulence, as demonstrated in the mouse pneumonia model, in which a deletion of the *blp* locus response regulator leads to a decrease in systemic virulence relative to that of the wild-type strain (209). The regulatory portion of the locus consists of genes encoding a small peptide pheromone (*blpC*), a histidine kinase (*blpH*), and a response regulator (*blpR*), and it also encodes an ABC transporter (*blpA-blpB*), which cleaves and exports the pheromone, the bacteriocins, and their associated immunity proteins (210).

(iii) *E. coli* ST131 (O25:H4), associated with CTX-M extended-spectrum β-lactamases, has emerged internationally as a multidrug-resistant pathogen (211–213). This clone, which was described for the first time in 2000, became distributed worldwide by 2008 and is possibly the most widely distributed resistant clone (e.g., 30 to 60% of the fluoroquinolone-resistant isolates belong to this clone) (214, 215). Coselection of resistance genes (CTX-M) and virulence genes in plasmids of the IncFII group, such as pEK499 (Fig. 2B), appears to be important in explaining the extensive distribution (216) (see “Plasmids” below).

(iv) A multidrug-resistant *P. aeruginosa* strain with enhanced virulence, designated the “Liverpool epidemic strain” (LES), has recently been described in cystic fibrosis patients from the United Kingdom (97). This strain is characterized by upregulation of the efflux pump MexAB-OprM in relation to resistance to β-lactam antibiotics and aminoglycosides and by increased virulence in relation to the quorum-sensing system (see “Resistance to Antivirulence Components” below) (98, 99).

(v) The *Clostridium difficile* ribotype NAP1/027 epidemic hypervirulent and quinolone-resistant clone is genetically specific and displays high infectivity, which due to hyperproduction of toxins TcdA and TcdB, protease Cwp84, the high-molecular-weight S-layer protein, and fibronectin-binding protein Fbp68, combined with the antibiotic resistance, may have contributed to selection and worldwide spread of the clone, aided by residual concentrations of fluoroquinolones (217).

(vi) *E. faecalis* strains which are members of the clonal cluster with Bla (β-lactamase) and Vanr and which are isolated from endocarditis infections (*E. faecalis* that is β-lactamase positive and vancomycin resistant from endocarditis infections [BVE]) were found to contain a previously described putative pathogenicity island (PAI). Subtle variations within the structure of the pathogenicity island enable strains harboring the element to modulate virulence; these variations occur at a high frequency (218).

(vii) MRSA strain USA300 has been shown to be responsible for an epidemic of community-associated infections (219), involving mostly skin and soft tissue but also more serious invasive syndromes such as pneumonia, severe sepsis, and endocarditis. MRSA strains are particularly serious and potentially lethal pathogens that not only are resistant to several antibiotics but also possess virulence mechanisms, including toxins, adhesins, enzymes, and immunomodulators. An understanding of the biological basis of MRSA virulence and future directions for research, including potential vaccines and antivirulence therapies that are currently under development, might allow clinicians to treat and prevent MRSA infections more successfully. Moreover, Lin et al. have demonstrated the possibility of reverse zoonotic transmission of the MRSA isolates (220). These authors described the first case of a USA300 genotype in a pig. The presence of multiple virulence profiles within an MRSA genotype in these animals suggests the potential emergence of new MRSA clones by gain or loss of additional virulence genes.

(viii) *P. aeruginosa* clone ST175 has been characterized as a high-risk clone due to an increasing prevalence of nosocomial infections and multidrug resistance (203). It has recently been demonstrated that the extreme drug resistance is due to a combination of several mechanisms: AmpR-activating mutations in this clone (overexpression of AmpC), OprD inactivation, mutations in GyrA and ParC, MexXY-OprM overexpression, and the expression of the class 1 integron *aadB* gene. Indeed, AmpR can act as a global regulator, and its activation can be implicated in quorum sensing, alginate production, biofilm formation, and the expression of several other virulence factors (221).

## COSELECTION OF MECHANISMS OF ANTIMICROBIAL RESISTANCE AND VIRULENCE

Horizontal gene transfer (HGT) is a very important mechanism (in addition to spontaneous mutation) responsible for the development of antibiotic resistance in bacteria. This is the process whereby DNA fragments, the so-called mobile genetic elements (MGEs), can be transferred between bacteria of the same or different species. The classification of MGEs involved in bacterial evolution and adaptation is continuously being updated. However, probably the most widely accepted classification is that of Burrus et al. (222), which considers that integrative and conjugative elements (ICEs) include transposons, integrons, integrative plasmids, genomic islands, and other, unclassified elements (i.e., those integrated in chromosomes). Plasmids and ICEs are the main genetic mechanisms for the dissemination and coselection of virulence and resistance genes, which are transferred and multiplied via three main ways of HGT: conjugation, transformation, and transduction. Indeed, biofilms appear to facilitate horizontal transmission due the high microbial density of the populations (222–224), and subinhibitory concentrations of antibiotics also appear to increase the rate of HGT (223).

### Plasmids

Plasmids are extrachromosomal and self-replicating elements that are not essential to bacteria but which often carry and disseminate genes that confer to the bacteria certain characteristics such as resistance, virulence, the ability to metabolize rare substances, and persistence under extreme conditions. In particular, conjugative plasmids play an important role in the evolution of pathogenic bacteria because they are readily transmitted by horizontal transfer both between and within species (225, 226).

Some interesting studies have analyzed the nature of plasmids before the so-called antibiotic era. Studies of a collection of enter-
Beceiro et al.

obacterial strains obtained between 1917 and 1954 have shown that most of the plasmids isolated nowadays are essentially the same as those isolated during the preantibiotic era. Analysis of 84 different plasmids, none of which carried a resistance determinant, revealed that 65 belong to current plasmid groups. Thus, the acquisition of antibiotic resistance through recruitment of new genes within the original plasmids, as a result of the use of the antibiotics, is of great clinical importance (227, 228).

The major incompatibility (Inc) group involved in transfer of resistance and virulence genes is the IncF group; transmission of IncF plasmids is a clear example of how virulence and resistance can increase via coselection, probably due to antibiotic pressure. This has led on numerous occasions to the emergence of strains or outbreaks of clones that are especially virulent and multidrug resistant. The wide dissemination of the *E. coli* sequence type ST131 clone is one example of this (see Highly Virulent and Multiresistant Worldwide Disseminated Clones above). Although there is no clear explanation for the successful dissemination, it appears that the IncFII group plasmids (e.g., pEK499) (Fig. 2B) often carry the CTX-M-15 β-lactamase (216) and may be involved in spread of the clone. These plasmids carry resistance genes in multiple families and also carry virulence genes (e.g., the pEK499 plasmid carries two copies of the *vagC-vagD* system, which is involved in cellular division and is necessary to maintain the virulence of *K. pneumoniae*). It is important to highlight the multiple maintenance systems present in these types of plasmids, for example, the Hok-Mok postsegregation killing proteins and the *pemI-pemK* and *ccdA-ccdB* toxin/antitoxin systems. These systems ensure stable transmission and maintenance in the absence of any antibiotic selective pressure (229, 230). Lavigne et al. have recently attempted to clarify the ample capacity of this clone to spread, as well as the involvement of the CTX-M β-lactamases. These authors developed two *in vivo* models to compare CTX-M-15-producing and non-ESBL-producing *E. coli* ST131 strains with non-ST131 strains. They suggest that although expression of CTX-M-15 is not correlated with enhanced virulence, it can improve the persistence of a strain during infection. This indicates that the secret of the success of the clone is in achieving a perfect balance between virulence and antibiotic resistance (231). Although there is significant diversity in the structure of the plasmids and the evolution of resistance mechanisms that have affected clone ST131, plasmids that carry resistance genes have been essential in the rapid spread of these virulent clones of *E. coli* (232, 233).

Tetracycline resistance also can be coselected with virulence factors. For example, a virulence plasmid of a porcine enterotoxigenic *E. coli* (ETEC) strain carries a Tn10 transposon that carries the tetracycline resistance genes *tetA* and *tetC* (encoding efflux systems). The toxin-specific locus caused the enterotoxigenicity of the strain, which contains two heat-stable enterotoxin genes, *sta* and *stb*. In this example, the authors observed coselection of virulence and tetracycline resistance, but they did not analyze the biological cost of carrying this virulence plasmid (234). Coselection of tetracycline was also observed in *C. perfringens* in a study that analyzed the nonreplicating transposon Tn916, which is involved in the conjugation of a replicating plasmid carrying putative virulence genes and two tetracycline resistance mechanisms, *tetA(P)* (efflux system) and *tetB(P)*, which provide ribosomal protection (235, 236).

Not only does antibiotic pressure promote the spread of plasmids with resistance determinants (and often also virulence factors), but certain environments (in which biodiversity is very high and there is a correspondingly large metagenome) encourage bacterial interchange and the distribution of these mechanisms. Plasmid pRSB107 (120 megabases), which was isolated from bacteria present in sludge from a sewage treatment plant, encoded resistance to at least 9 different antimicrobials in *E. coli* transformants. It also encoded or carried four putative virulence factors: an aerobactin iron acquisition siderophore system, a high-affinity Fe²⁺ uptake system, a glycerol-3-phosphate transport system, and the virulence-associated genes *vagC* and *vagD* (237). Under suitable conditions, plasmid chimeras can be created from different plasmids acquired by horizontal transmission from multiple environments (Fig. 2C).

Finally, another example of the recent evolution of coselection is the hybrid resistance-virulence plasmid of *Salmonella* spp. Some virulence plasmids which are very frequent in human infections, such as pSEV in *S. enterica* serovar Enteritidis or pLST and pSTV in *S. enterica* serovar Typhimurium, carry a conserved region of 7.8 kb carrying the *spv* genes (*Salmonella* plasmid virulence) and other virulence genes (238–242). It has been shown that 7 to 8% of ampicillin-resistant isolates of *S. enterica* serovar Typhimurium in the United Kingdom and Spain carry the pUO-StVR2 virulence-resistance hybrid plasmid, which is derived from the plasmid pSTV after acquisition of a 45-kb region including the class 1 integron InH, with genes encoding resistance to at least five different antimicrobial families (243, 244).

The TraT protein, which is an external outer membrane lipoprotein associated with plasmid conjugation and also with several virulence mechanisms (e.g., serum resistance, phagocytosis, and biofilm formation), also plays an important role. Thus, plasmids have a doubly important role, as they spread resistance genes and the *traT* genes are directly involved in bacterial virulence (3, 245).

Carrying plasmids that encode both resistance and virulence factors leads to selection of resistance determinants in bacteria in noninfective environments subjected to antibiotic pressure. Reciprocally, in infective environments, simultaneous selection of virulence determinants and resistance factors can occur in bacteria, even in the absence of antibiotic selective pressure (3). Thus, coselection appears to have occurred in the recent evolution that has taken place in the postantibiotic era, and it could help bacteria to adapt more easily to new environments. The distribution of these types of plasmids among common human pathogens may become a major health problem in the near future.

### Integrative and Conjugative Elements

Integrative and conjugative elements (ICEs) are self-transmissible mobile genetic elements that help in the horizontal dispersal of genes located in another higher DNA molecule with replicative capacity, such as plasmids and chromosomes. With the introduction of new methodologies, such as metagenomics and massively parallel sequencing, it has been shown that ICEs may have an even more significant impact than plasmids in HGT (246). Hypothetically, genetic elements such as integrons and transposons can capture virulence genes and disperse them jointly with resistance genes, although not much is known about the relationship between virulence and resistance factors in the same ICE, except for some specific examples such as those described below. This may suggest the difficulty of coselection of both types of factors by ICEs. However, there may be some bias, since many studies seek the cause of the dissemination of antibiotic resistance in bacterial
populations by exclusively locating and sequencing the resistance markers and the neighboring genetic locations. Furthermore, transposons and integrons that carry virulence genes are less extended than those carrying resistance genes or catabolic pathways; indeed, virulence factors are usually related to prophage-like structures (which are less likely to carry resistance genes). The main explanation for this is the difference in the evolutionary time scale of virulence and resistance for the hosts and the drugs, respectively. The current pathogens are the product of a long period of evolution along with the host and a short but intense period of evolution, of only a few decades, involving antibiotic pressure. While phages have a narrow host range, transposons can readily insert themselves into a broad host range of plasmids, thus providing functions to a large number of hosts (247); this facilitates the spread, in this case, of antibiotic resistance factors.

However, there are some clear examples in which coselection of resistance and virulence occurs via ICEs. In *Vibrio cholerae*, the enzyme bis-(3′,5′)-cyclic dimeric GMP increases biofilm formation and decreases virulence and motility, so that it is considered an important factor for persistence in the environment, and is regulated by diguanylate cyclases (DGCs). Two DGCs, DgcK and DgcL, are encoded by integrating conjugative elements of the SXT/R391 family, and thus the acquisition of these ICEs could improve the survival of this species in aquatic environments. SXT/R391 ICEs are directly involved in the transmission of multiple drug resistance in different species, including *Vibrio* species (248). Indeed, analysis of some isolates from cholera outbreaks has revealed the involvement of SXT/R391-like genes in the transmission of virulence and resistance (248, 249). The *in vitro* plasticity and capacity of the SXT-related ICE from *Vibrio cholerae* to transfer virulence genes to other species such as *E. coli* has also been demonstrated (250).

Genomic islands are also involved in the dissemination of virulent and resistant isolates; for instance, *Salmonella* genomic island 1 (SGI1) encodes resistance to different antibiotics. The *S.* enterica serovar Typhimurium phage type DT104, carrying SGI1, is disseminated worldwide, like other major clones. Because of its virulence and resistance gene repertoires, isolates with an SGI1 variant are a risk for rapid dissemination (141, 251, 252). The OmpR-EnvZ operon system, which is encoded in *Salmonella* pathogenicity island 2, is also involved in the regulation of resistance and virulence. This system responds to environmental conditions by modifying the expression of OmpC and OmpF porins, which are involved in antibiotic resistance in this species. Furthermore, the OmpR-EnvZ system also regulates another system encoded in the same pathogenicity island, the SsrA-SsrB operon, which in turn regulates the type III secretion system. OmpR mutants are unable to replicate inside the macrophage cell, thereby reducing its virulence in the absence of this operon (253, 254). In another example, the entire genome of *Streptococcus suis*, a zoonotic agent of disease in young pigs which can also cause infection in humans, has been sequenced. The major finding was a pathogenic island of 89 kb, named ICESsu<sub>SC84</sub>, which is a composite structure with several integrated MGEs. This island encodes several resistance mechanisms, such as resistance to aminoglycosides and tetracyclines and an antibiotic export/resistance system; it also encodes a surface-anchored protein, LPXTG, a virulence factor that facilitates binding of bacteria to the eukaryotic cell and is also located in *Streptococcus agalactiae* (255, 256). Sequencing of strain ATCC 17978 of *A. baumannii* revealed the existence of up to

28 putative alien islands (islands formed by genes obtained from other species), including 17% of the total open reading frames (ORFs). This indicates that this species has acquired a large amount of foreign DNA and also highlights the genetic plasticity of the species. Sixteen of these islands contain putative genes involved in virulence, such as type IV secretion systems and others. Seven different putative alien islands also contained genes that probably encode drug resistance proteins (on the basis of their sequences). Although no island that encoded virulence and resistance factors together was identified, this study highlights the genetic plasticity of *A. baumannii* (as it can capture foreign genes of both types easily), which appears to be one of the secrets of its successful dissemination and persistence in recent decades (257, 258).

Biofilms play an important role in HGT. The structure of biofilms promotes HGT, especially by conjugation, and because of the high density and close proximity of the cells, the conjugation itself can even stimulate biofilm production. Indeed, transformation appears to be necessary for biofilm formation and stabilization (259). A type IV secretion system is involved in biofilm formation and contributes to cell-to-cell contact, thus mediating DNA transfer (260). There is therefore a positive feedback between the horizontal exchange of genes and biofilm formation, which favors movement of resistance genes and virulence factors, especially in the presence of antibiotic selective pressure (261, 262).

### Phage-Mediated Transduction

In transduction, fragments of bacterial DNA are included as part of viral DNA, and when this viral particle infects other bacterial cells, DNA is integrated and replicated in the host bacterial cell. There are numerous examples of different virulence genes carried by prophages in *E. coli*: e.g., Shiga toxin (STX) (263, 264); genes of effector proteins such as bacteriophages, nleA-H, and Cif; and cytolethal distending toxin (CDT) (265). These prophages can spread virulence factors to other species that may already possess resistance mechanisms that favor the persistence or dissemination of both in the environment. Recently, a particularly virulent outbreak comprising 3,842 cases of human infections was produced in Germany by a Shiga toxin-producing *E. coli* (STEC) strain belonging to serotype O104:H4 with virulence features common to the enterohemorrhagic *E. coli* pathotype, which previously carried the plasmid-encoded TEM-1 and CTX-M-15 β-lactamases. The Shiga toxin was probably transduced from other enterohemorrhagic *E. coli* strains (266, 267). Indeed, the presence in animal environments of bacteriophages carrying different resistance genes, such as *bla*<sub>TEM</sub>, *bla*<sub>CTX-M</sub>, and *mecA*, has recently been demonstrated, suggesting their capacity to act as environmental vectors of virulence and resistance (268).

Finally, we will comment on the role of the SOS response (bacterial response to the DNA damage) in the activation of production of lysogenic prophages and, therefore, the increased genetic exchange by transduction. For example in *S. aureus*, SOS response-inducing antibiotics, such as β-lactams and ciprofloxacin, may favor the dissemination of virulence factors by promoting replication and horizontal transfer of pathogenicity islands (130, 161).
Beceiro et al.

# Outer Membrane Vesicles

Membrane vesicles, which are released from the cell envelope, are a vehicle for numerous lipids and proteins and play an important role in the interaction between host and pathogen. The vesicles are a relatively novel or poorly studied mechanism of coselection and joint dissemination of virulence and resistance factors. It has been demonstrated that OMV production increases in response to bacterial stress or environmental factors, e.g., during the colonization or infection of host tissues (269, 270).

Indeed, OMVs often contain numerous virulence factors, including adhesins, toxins, antigens such as lipopolysaccharide (LPS) (271), and other outer membrane (OM) components that influence the infection and host response. Thus, enterotoxigenic *E. coli* secretes the enterotoxin LT, which is delivered by the OMVs from the OM into mammalian cells (272). *Staphylococcus aureus* releases membrane vesicles (not OMVs), produced during *in vivo* infection in a mouse pneumonia model, harboring class A β-lactamase and virulence-related proteins, which are able to induce apoptosis of HEP-2 cells (273). In the presence of gentamicin, *P. aeruginosa* increases the release of membrane vesicles carrying hydrolytic enzymes (274). Quorum-sensing molecules or adhesins are also transported by OMVs and enable the bacteria to interact with eukaryotic cells (275).

However, it is not only virulence factors that can be directly transported on vesicles. Horizontal transfer of plasmids carrying antibiotic resistance genes has been demonstrated to occur through OMVs, which reveals a direct link between resistance and virulence. Our group has demonstrated that *A. baumannii* releases Omp33-36 and OmpA porins (involved in resistance and virulence) within OMVs (276). Also in *A. baumannii*, we have studied an MDR epidemic strain, AbH12O-A2, that is able to release AmpC and OXA-24 β-lactamases (277). This type of mechanism of spread of virulence and/or resistance determinants probably occurs in most pathogenic species.

In general, bacteria produce a relatively enriched protein profile of virulence factors within OMVs compared with the virulence factor complement of the OM (271). Therefore, more studies with pathogenic species are required to determine the real importance of vesicles as vehicles of codissemination of virulence and resistance factors.

## COMPENSATORY MUTATIONS

In the recent past, it was assumed that the development of antibiotic resistance was inexorably linked to virulence and to fitness costs and that in the absence of antimicrobials in the environment, the susceptible strain would be more competitive than the resistant strain (which may display, e.g., lower growth rates, invasiveness, and transmission capacity). However, antimicrobial treatment could hypothetically reduce the fitness of the susceptible strain, thus creating more favorable circumstances for resistant mutants with a higher fitness in this environment and leading to the development of resistant populations. Many studies in recent years have highlighted the importance of the ability of resistant mutants to adapt and recover their fitness and virulence by secondary-site mutations or compensatory mutations. Molecular studies with different bacteria (both laboratory and clinical strains) show that the recognition of additional compensatory mutations is key to understanding the evolution of microorganisms in recent decades, during which antimicrobial agents have been extensively used. However, the type and number of compensatory mutations and the level of compensation depend on multiple factors, such as the particular species, the resistance mechanism, and the environmental conditions. A recently published review by Andersson and Hughes (278) highlights some of the main examples of chromosomal compensation of fitness costs. There are three ways of restoring the fitness of resistant mutants: direct restoration of the efficiency, replacement of the function by another similar function, and reduction of the need for that function, as explained in further detail below:

(i) Some recent studies provide a better understanding of fitness compensation. For example, Billal et al. carried out an interesting whole-genome analysis with transformants of *S. pneumoniae* that were susceptible or resistant to linezolid (171). On the basis of the transformation of the genome of a resistant mutant into two susceptible strains, the group characterized all mutations associated with resistance to this antibiotic and mutations associated with adaptive compensation. The involvement of some of the genes in resistance and fitness compensation investigated in this study is shown in Fig. 5A. The cumulative effect of two different ABC transporters (encoded by *spr1021* and *spr1887*) may at least partly increase the fitness cost of chromosomal mutations involved in linezolid resistance (23S rRNA mutations). Excessive intracellular concentrations of linezolid due to modification of the target (23S rRNA) may be compensated for by the overexpression of ABC transporters, especially Spr1887, which is responsible for the extracellular excretion of free linezolid. Other mutations, such as amino acid changes in the 50S ribosomal proteins L3 and L16, are involved in restoring the loss of fitness in these linezolid-resistant strains of *S. pneumoniae*. These changes are probably involved in enhancing the stabilization of the peptidyltransferase center of the ribosome (involved in the translation of proteins), modified with the unfavorable G2576T 23S rRNA mutation (171). Thus, in this case, two different types of events have compensated for the loss of fitness: increased expression or overexpression of a new function (ABC transporter) and direct restoration of the efficiency of an affected function (modification of L3 and L16 ribosomal proteins). A similar example is shown in Fig. 5B. Björkman et al. performed several studies with a mouse infection model (278–280) and showed that in mutant strains of *S. enterica*, the increased fitness cost of high-level resistance to streptomycin could be compensated for by single mutations in the *rpsL* gene. This gene encodes 30S ribosomal protein S12, which offers streptomycin resistance at no cost (K42R) or with a fitness cost (K42N). However, if in addition to the K42N mutation, the strain suffers new compensatory mutations in the *rpsD* gene (which encodes 30S ribosomal protein S4), such as K205T, the fitness cost will be compensated for (278–280).

(ii) Another way of restoring fitness is to replace one function with another function of similar efficacy. This strategy is used by strains of *S. enterica* that are resistant to peptide deformylase inhibitors (PDFIs), such as actinonin. In most bacteria, translation starts with a formylated Met-tRNAAi, and when translation is complete, the formyl group is removed by the peptide deformylase enzyme. The PDFIs inhibit peptide deformylase activity by accumulating intracellular toxic formylated peptides. Therefore, to acquire resistance to PDFIs, *S. enterica* can gain mutations in the *fmt* and *folD* genes, thereby decreasing the addition of formyl groups to the Met-tRNAAi; the translation and growth rates become reduced because of the need to initiate translation with unformylated Met-tRNAAi. In this case, fitness is restored by amplification

Resistance and Virulence: a Successful Association?

FIG 5 (A) Relationship between resistance, fitness, and compensatory mutations. The figure reflects the fitness (measured as growth capacity) of the S. pneumoniae wild-type (WT) strain (1974), the linezolid-resistant mutant (1974M1), the T-7 mutant (the 1974 WT strain with the main mutation involved in resistance, the 23S rRNA mutation G2576), which is less fit than strain 1974M1 (indicating the existence of compensatory mutations in the latter), the T-7 mutant with overexpression of the *spr1021* gene, encoding an ABC transporter), which displays increased resistance to linezolid but with a clear biological cost, and finally the T-7<sup>*spr0333,spr1021,spr1887*</sup> mutant (T-7 mutant with overexpression of the *spr1021* and *spr1887* genes and also the methyltransferase gene *spr0333*), which appears to restore the original fitness by the action of the new ABC transporter encoded by *spr1887* (those commented on in the text are highlighted). (Reprinted from reference 171, which was published under an open-access license.) (B) High-level resistance to streptomycin in mutants of S. enterica subsp. enterica serovar Typhimurium. The fitness costs are reflected on the $y$ axis. The K42R mutation in the protein encoded by *rpsL* gene is free of any fitness cost; however, the appearance of the K42N mutation in the same protein is associated with a biological cost, which can be almost totally compensated for by the secondary mutation K205T in RpsD protein. (Modified from reference 278 by permission from Macmillan Publishers Ltd.)

of tandemly repeated *metZ* and *metW* genes that encode tRNAi. Increased expression of these genes results in increased tRNAi levels, which can compensate for the lack of addition of formyl groups to the Met-tRNAi and allow initiation of translation without formylated Met-tRNAi. In Salmonella, peptide deformylase activity is replaced by high tRNAi levels, so that resistance to PDFIs is acquired at the lowest possible cost (281).

(iii) The third way to restore the fitness by means of compensatory mutations is to reduce the need of the bacteria for the altered function. Albarracín et al. (282) have studied an example of this in β-lactam-resistant transformants of S. pneumoniae induced by mutations in three penicillin-binding proteins, PBP1a, PBP2x, and PBP2b. Basically, the simple PBP2b mutant used in this study had resistance to piperacillin but also reduced fitness relative to that of the parental strain (measured in both *in vitro* and *in vivo* models). Thereafter, transformation with two new PBP mutations, in PBP1a and PBP2x, maintained the piperacillin MIC and increased the cefotaxime MIC; remarkably, the transformation also increased the fitness of the triple transformant of S. pneumoniae. In this example, the proteins PBP1a and PBP2x not only showed a complementary function of PBP2b (cell wall maintenance) but also contributed to increasing the resistance to β-lactams antibiotics (282).

These and other recent studies (278, 283–288) highlight the importance of compensatory mutations that enable bacteria to adapt to and subsist with the biological cost associated with the gain in antibiotic resistance. Handel et al. developed a mathematical model that helps us to understand how far the adaptive phenomenon is relevant in the process of emergence of resistance (5). One of the most important aspects regarding the possible emergence of resistance as a result of compensatory mutations is the level of treatment to which a specific microbial population is exposed. A diagram summarizing the dynamics of the development of resistance, proposed by these authors, is shown in Fig. 6. This theoretical model shows that once a resistant mutant that is less fit than the parental strain appears in an environment, it could eventually restore its fitness and could compete with the original susceptible strain if it is able to survive long enough to acquire one or more advantageous compensatory mutations. This model considers four possibilities. In a hypothetical first situation, A, without treatment, resistant mutants can appear only via unlikely mutations, which also have a fitness cost and therefore will not prevail; resistance will not emerge. Oceans or seas are examples of such environments in which antibiotics are present at low levels. Similarly, in hypothetical situation B, if the treatment covers only a small percentage of the population or antimicrobials are present at low levels, resistant mutants are unlikely to appear, but they may appear, because of the low number of individuals subjected to antibiotic pressure. In this scenario, resistant mutants could acquire a level of fitness similar to that of the parental strain. This could include bacteria that colonize or infect anatomical niches with low levels of distribution of the antibiotic in a treated host (e.g., gut microbiota in treatment for otitis). In a third situation, C, in which treatment levels are higher, resistant mutants with a low degree of fitness would probably emerge; these could lead to the emergence of resistant mutants with restored fitness due to compensatory mutations, although this is unlikely due to the low number of resistant mutants. A real example of this is the nosocomial environment, in which antibiotics are present and there is a high risk of selection of resistant clones. In the last case, D, in which the population would be massively subjected to the drug, emergence of resistant mutants from the original generations or from the following generations would be likely; these mutants would have restored fitness and would therefore be able to prevail.

Beceiro et al.

no emergence no emergence emergence no emergence emergence emergence

![Image Description](image1.png)

FIG 6 Theoretical model proposed by Handel et al. of the emergence and persistence of antimicrobial resistance in different ecological niches. The bars indicate the fitness of the strains in a given situation, the solid arrows indicate acquisition of mutations that occur frequently due to the large size of the original population, and the dashed arrows indicate acquisition of mutations that occur infrequently in a small part of the original population. If the resistant mutant is less fit than the wild-type strain, it will tend to extinction by competition; however, if during competition the resistant strain develops compensatory mutations, it could eventually emerge and persist. The antibiotic levels in the environment will largely determine this selection. (A) In an environment without antibiotic treatment (e.g., seas), resistant strains are associated with low fitness and will not emerge. (B) In an environment with a certain level of antibiotics (e.g., during infection in a compartment of the treated patient with a low level of antibiotics or without antibiotic distribution), the fitness of the wild-type strain is reduced, which enables emergence of a resistant mutant. One frequent conversion and two infrequent conversions are required. (C) In an environment with antibiotics (e.g., a nosocomial environment), resistant mutants readily emerge, and only one infrequent conversion is necessary. (D) In an environment with a massive presence of antibiotics (e.g., during antibiotic treatment), the low fitness of the original population readily enables the emergence and persistence of resistant populations. (Reprinted from reference 5, which was published under an open-access license.)

over time, for instance, in treatment of infection of an animal or human host with antimicrobial therapy (5).

Another theoretical model of the emergence of resistant mutants on a time scale that includes the proportion of treated individuals is shown in Fig. 7A. This illustrates several resistant mutants for which the fitness costs are different from those associated

![Image Description](image2.png)

Fraction of treated infected individuals

FIG 7 (A) Time until emergence of resistance, based on a theoretical model obtained from simulations of the deterministic model of years until emergence of the resistant strains (see reference 5 for more details). Fitness levels of the resistant strains 1, 2, and 3 are 75%, 85%, and 95%, respectively, of those of the susceptible strain in the absence of antibiotics. Vertical lines represent the level of treatment necessary for the fitness of the resistant strains to equal that of the susceptible original strain. The conversion rate of the strains due to the compensatory mutations equals \(10^{-1}\) (for other rates, see reference 5). (B) Probability that resistance will emerge within 1 year, based on a theoretical model obtained from stochastic simulations. The parameters used are the same as for panel A. (Reprinted from reference 5, which was published under an open-access license.)

Resistance and Virulence: a Successful Association?

FIG 8 Model describing the signals controlling expression of PhoP-PhoQ-regulated determinants and the interaction between the PhoP-PhoQ and PmrA-PmrB two-component systems, as well as some of the genes and phenotypes governed by the PhoP-PhoQ system. (Reprinted from reference 180 with permission.)

ance of resistant prevalent mutants and the percentage of the population treated. As reflected in the figure, the temporal changes or the probability of emergence is much higher when low levels of the population are treated, although these changes are more unlikely to occur with high levels of treatment. This should be considered when implementing treatment strategies and when analyzing epidemiological data.

These types of models help us to understand why compensatory mutations are fundamental to the wide dissemination of resistant clones, despite the initial increase in fitness costs and virulence. The role of such advantageous mutations in the persistence or reversion of the resistant mutations is not entirely clear. Theoretical findings supported by clinical and experimental studies suggest that resistance and compensatory mutations are easier to achieve than reversion mutations, and therefore compensatory mutations are a barrier to the loss of resistance in resistant adapted mutants (289, 290). However, the genetic reversion of resistant mutants after a period of absence of drugs has been described in both experimental and clinical studies, even in individual patients (289). According to the model of Schulz zur Wiesch et al., these situations, in which decreased resistance occurs, may be explained by clonal changes in the environment, even when the resistant clones are as fit as the susceptible strains (289–291), and the reversal of resistance through genetic events is less likely (289, 291). At the same time, reversal will be possible when the compensatory mutations are not sufficient to balance the fitness of the original strain. Often as a consequence of a decrease in antibiotic pressure, the resistance levels of a bacterial population decrease to minimal levels but do not disappear, thus making the persistent population more resistant than the original population, even in the absence of direct pressure (288, 292). In addition to searching for explanations for the persistence of resistant bacterial populations in the absence of antibiotic pressure, the increasingly common phenomenon of coresistance should be taken into consideration.

GLOBAL RESPONSES AND THEIR EFFECT ON ANTIMICROBIAL RESISTANCE AND VIRULENCE

Due to the extent of the review, we do not discuss in this work the interplay between antimicrobial resistance and virulence on overall and global stress responses depending on MarA, SoxS, and RobA (among some other examples), as this subject warrants a separate review.

Two-Component Regulatory Systems

Bacterial pathogens can survive hostile environmental conditions during host infection by modulating their genetic arsenal. They often use two-component systems (TCSs) to regulate their responses and adapt to environmental changes. Such systems are frequently involved in changes in antibiotic resistance and virulence, as previously mentioned. These systems comprise a histidine kinase sensor protein, which is integrated in the inner membrane, and a cytoplasmic response regulator. The function of both proteins is mediated by phosphorylation reactions and conformational changes (293–296).

*P. aeruginosa* strain PAO1 carries up to 10% of its genes as regulatory genes, including those encoding 72 response regulators and 62 sensor kinases. Of these gene products, several TCSs are involved in both virulence and antibiotic resistance: PhoP-PhoQ (297), PmrA-PmrB (298), CbrA-CbrB (11), WalK-WalR (299), Ppr-PprA (300), GacA-GacS (301), and PvrR (302). Similar TCSs have been studied in *Salmonella* (PmrA-PmrB and PhoP-PhoQ) (180, 303, 304) and *E. coli* (BasR-BasS and PmrA-PmrB) (304, 305).

PhoP-PhoQ. The TCS PhoP-PhoQ governs virulence, motility, invasion, intracellular survival, the adaptive response to low environmental concentrations of Mg²⁺, resistance to aminoglycosides and antimicrobial peptides, and multiple cellular activities in several Gram-negative species (179, 295). Groisman proposed a model that reflects the role of PhoP-PhoQ in *Salmonella* and how it interacts with PmrA-PmrB (180) (Fig. 8). Low concentrations of

Mg²⁺ promote transcription of PhoP-activated genes, and high concentrations of Mg²⁺ promote repression of these genes. Addition of Mg²⁺ to Salmonella culture medium increases the susceptibility of the bacteria to magainin (an antimicrobial peptide) by more than 1,000 times (306). Approximately 40 genes are regulated by the PhoP-PhoQ system in Salmonella (307), including virulence genes such as *mgtCB*, which is involved in intramacrophage survival (308), and possibly the *spv* and *ssa/sse* genes, isolated from a virulence plasmid and a pathogenicity island, respectively (309, 310). In Salmonella, mutations of the PhoQ sensor alter the Mg²⁺ sensitivity threshold and render the bacteria avirulent (306). However, PhoP-PhoQ also mediates resistance to antimicrobial peptides or polymyxin B through activation of *pagL* and *pagP*, encoding a lipid A 3-O-deacylase and a lipid A palmitoyltransferase, respectively. These modifications decrease the negative charge of the LPS, thus affecting the electrostatic interactions and decreasing the affinity between the LPS and colistin or polymyxin B (311, 312).

Similarly, in *P. aeruginosa*, PhoP-PhoQ regulates resistance to polymyxin at low concentrations of Mg²⁺. Overexpression of this system is associated with a high degree of resistance to polymyxin B (313), and a *phoP* laboratory mutant was found to be highly susceptible to this antibiotic (314). A *phoQ* mutant also exhibited high susceptibility to quinolones (315). On the other hand, the PhoQ sensor is necessary in *P. aeruginosa* for biofilm formation (316), and different *in vivo* studies with *phoP* mutants have demonstrated the role of this TCS in virulence in a neutropenic mouse model (313) and in chronic rat lung infection (295), as well as reduced twitching motility, biofilm formation, and cytotoxicity to human lung epithelial cells and loss of competitiveness in chronic rat lung infections (197). PhoP orthologs associated with antimicrobial peptide resistance and virulence have been found in other species, such as *E. coli*, *S. flexneri*, *Y. pestis*, and *M. tuberculosis* (317–320).

PmrA-PmrB. The TCS PmrA-PmrB (also known as BasR-BasS), which can also be induced by PhoP, regulates other LPS modifications. However, PmrA-PmrB regulation can be PhoP-PhoQ independent and can be activated by high concentrations of Fe³⁺ or low pH (Fig. 8). One of the main roles of this system is the modification of lipid A by the addition of 4-aminoarabinose and phosphoethanolamine, which decreases the negative charge of LPS (321, 322). Addition of 4-aminoarabinose and phosphoethanolamine is mediated by the *pmrE* gene or the operons *pmrHFIJKLM* (323) and *pmrC* (324), respectively. Strains of Salmonella with mutations in these genes exhibit decreased virulence *in vivo*, which highlights the role of these mechanisms in virulence (175). Interestingly, the antimicrobial resistance mediated by 4-aminoarabinose is associated with higher survival in murine intestine, which may be related to the fact that LPS is essential for recognition of the host immune system; modifications to the LPS may alter the identification of the microbial pathogen and avoid the innate response with a decreased Toll-like receptor–LPS interaction. This would also be associated with antimicrobial peptides and polymyxin resistance (321). In *E. coli*, treatment with bile salts leads to induction of PmrA-PmrB, lipid A changes, and upregulation of the efflux system AcrAB, which is involved in resistance to multiple antibiotics (305).

The PmrA-PmrB system, which has also been described in *P. aeruginosa*, induces modification of lipid A, by the addition of 4-aminoarabinose, and subsequent antimicrobial peptide resistance (295). Modifications to the LPS which are induced by PhoP-PhoQ, PmrA-PmrB, or other systems involving palmitate and aminoarabinose are very similar to those produced by this species in cystic fibrosis, in which the host inflammatory response is increased (314).

CbrA-CbrB. The TCS CbrA-CbrB of *P. aeruginosa*, previously described as being involved in the carbon and nitrogen metabolic usage, is also involved in several mechanisms of antibiotic resistance (to polymyxin B, ciprofloxacin, and tobramycin) and virulence (swarming, biofilm formation, and cytotoxicity). A recent study with a *cbrA* deletion mutant of *P. aeruginosa* showed enhanced biofilm formation and *in vitro* cytotoxicity in human bronchial epithelial cells but also showed defective swarming motility (which is involved in a complex process of adaptation, involving different virulence genes). Indeed, this system can regulate the expression of PmrA-PmrB and PhoP-PhoQ in *P. aeruginosa* (11).

WalK-WalR. Although the WalK-WalR system in *S. aureus* is known to be involved in the control of cell wall synthesis, single amino acid mutations in both genes of vancomycin-susceptible *S. aureus* (VSSA) strains have recently been shown to cause resistance to vancomycin and daptomycin. In addition, the virulence decreased dramatically in an *in vivo* model (*G. mellonella*) of *S. aureus* infection, and the *in vitro* biofilm formation also decreased. These mutations partly reproduced the typical VISA phenotypes (299).

Other TCSs involved in virulence and resistance have been described, such as RocS2/RocS1-RocA1 of *P. aeruginosa*, which is able to control the *cupC* gene (involved in assembly of fimbriae) and the *mexAB-oprM* genes (encoding a multidrug efflux pump) (325). Other examples, also in *P. aeruginosa*, include the PprA-PprB system, which is involved in aminoglycoside resistance, outer membrane permeability, cytotoxicity mediated by the type III secretion system, biofilm formation, and quorum sensing (300, 326, 327), and the GacS-GacA system, which is associated with the production of small-colony variants that affect motility, biofilm formation, and antibiotic resistance (301). The RppA response regulator of *Proteus mirabilis* is necessary for the natural resistance of this species to polymyxin B and is also involved in biofilm formation and urothelial cell invasion (328).

These and other studies note the important role of TCSs in the development of *in vivo* resistance to different families of antibiotics, as well as the regulation of several bacterial processes, including virulence. Inactivation by mutations in these systems often involves loss of virulence.

## Stress by DNA Damage: SOS Response

Bacterial pathogens are subjected to many external assaults that can damage the DNA; the SOS response is the main mechanism for repairing damaged DNA. The SOS response involves several proteins but only two major enzymes: LexA, which regulates the repair system inhibiting gene expression through obstructing the binding of RNA polymerase to DNA (329), and RecA, which interacts with the single-stranded fragments of DNA (damaged DNA) and involves autocatalytic cleavage of LexA and other repressors, finally producing expression of the SOS response genes (330, 331). Once the damaged DNA is repaired, RecA expression decreases and LexA represses expression of all the SOS response genes. Although this repair mechanism is widespread among bacteria, it can vary greatly between species (332).
The SOS response is induced by antibiotics; e.g., some quinolones cause direct DNA damage (333), some β-lactams do not directly damage DNA but interfere with cell wall synthesis (334), and some quinolones interfere with DNA replication, even at subinhibitory levels (335). This response is also involved in the following: transmission of virulence factors such as Shiga-like toxin in *E. coli* (336), dissemination of mobile elements such as pathogenicity islands in *S. aureus* (130, 161) and *E. coli* (337), and increased expression of genes necessary to transfer the integrating conjugative element SXT of *Vibrio cholerae*, a carrier of mechanisms of resistance to multiple classes of antibiotics (338). An interesting finding by Guerin et al. (339) involved a conserved LexA-binding motif overlapping the promoter regions of chromosomal and mobile integrons (often carriers of multiple antibiotic resistance genes). Thus, in a normal environment, the SOS system represses the integrase protein, but under stressful conditions, the SOS response may activate the reordering of the integrons in the chromosome and facilitate the acquisition of exogenous resistance cassettes. Thus, exposure to antibiotics may induce the SOS response and activate integron recombination. This mechanism would not usually have any biological cost, under usual conditions, because it would not be activated. All these findings indicate that the SOS system is not only a DNA repair mechanism but also an important mechanism of induction and dissemination of antibiotic resistance mechanisms and virulence factors.

A clear example of the above is the study by Bisognano et al. of *S. aureus* strains that are resistant to quinolones (131). Exposure of these strains to low concentrations of ciprofloxacin raises the fibronectin-mediated attachment, which enhances the virulence by tissue attachment and cellular invasion. Electrophoretic mobility assays showed specific binding of the repressor LexA to *recA* and also to *fnbB*, which encodes a fibronectin-binding protein, a known virulence factor in *S. aureus*. Thus, exposure to subinhibitory concentrations of ciprofloxacin may enhance production of a virulence factor involved in colonization and invasion, which could favor the carrier state and increase the risk of acquiring a severe infection.

It has been shown in *E. coli* that β-lactams, fluoroquinolones, and trimethoprim induce the SOS response, thereby decreasing the bactericidal activity of these antibiotics (334, 340, 341). Similar induction of the SOS response has also been described in *V. cholerae* (342) and *S. aureus* (130, 161). Indeed, the SOS response by itself can also induce the expression of virulence factors (130, 161, 343). Mellies et al. observed, in enteropathogenic *E. coli* (EPEC), that the virulence factor type III secretion system, encoded by the locus of enterocyte effacement (LEE), showed LexA-dependent regulation (343). EPEC produces intestinal lesions, mediated by a type III secretion system, and protein effector molecules, which are injected into the host cell. In an *in vitro* assay, the purified LexA protein bound to LEE promoters, which would prevent transcription by inhibition of binding to the RNA polymerase.

We have recently studied the involvement of RecA in resistance and virulence in *A. baumannii*. Although LexA is not present in this species, our results suggest the involvement of RecA in DNA repair through recombinatorial repair. *A. baumannii recA* mutants were 20 times more sensitive to macrophages than the parental strains. In an *in vivo* murine model, the mutant showed attenuated virulence, probably due to low survival in the host. Indeed, the *recA* mutant exhibited higher susceptibility to some β-lactams, colistin, trimethoprim-sulfamethoxazole, and fluoro-

quinolones than the wild-type parent strain. In this case, this protein directly links both virulence and antimicrobial resistance to a process of DNA repair (344). Similar studies have highlighted the importance of RecA as a regulator of virulence in *S. Typhimurium* (345), *Pasteurella multocida* (346), and *Burkholderia* spp. (347).

### Mutator Phenotypes

Mutator (or hypermutable) strains are microorganisms that exhibit an increased mutation frequency due to a defective DNA repair system. The methyl-directed mismatch repair system (MutS, MutL, MutH, or UvrD) is the most frequently affected system in mutator populations (348). These mutator strains are known to have evolutionary advantages in new environments or stressful situations. Most of these mutations are neutral or even deleterious, but they may be favorable under directional selection (349, 350). For instance, under antibiotic pressure (351) or during infection (352), the mutator phenotype confers an advantage over the wild-type strain, thereby increasing its relevance as a clinical problem. Although the mutator strains can explore new ecosystems more quickly by increasing the rate of appearance of new mutations, this advantage disappears when the adaptation occurs. Even if the mutation rate is not reduced, the mutator strain may lose fitness over the long term (350) or in secondary environments after infection (352).

Although the prevalence of these phenotypes is variable, they are overrepresented in chronic infections caused by *P. aeruginosa* (353, 354, 355), *S. aureus* (356), and *Haemophilus influenzae* (357). In cystic fibrosis (CF), they are also frequently produced by hypermutable strains (strains more resistant than the nonmutator strains). In CF, the mutator strains of *P. aeruginosa* develop resistance to antipseudomonal antimicrobials much more frequently than nonmutators (353, 358), with efficient development of resistance through chromosomal mutation (359). Moreover, the persistence of *P. aeruginosa* in the lungs of CF patients results in changed resistance and altered virulence; acute injury is decreased, but chronic inflammation is increased, which promotes metabolic adaptation of the organism to the microaerobic conditions in the lungs of CF patients (360), usually leading to the loss of function of genes involved in virulence. It has also been observed, in a CF mouse model of chronic colonization, that the mutator strains favor long-term persistence; these strains compete with the wild-type strains, their fitness increases over time, and the numbers of adaptive mutations also increase (354). It has also been shown in this species that strains with a defective DNA repair system have enhanced microcolony-based growth, which is promoted by high biofilm production, thereby relating resistance and virulence (361). However, once again, because of the accumulation of multiple deleterious mutations in other, non-CF lung environments, adaptation to long-term persistence in CF-associated infection is not cost free. A reduction in transmission capacity was observed in that study; in a similar study with mutator isolates from CF patients, reduced virulence and fitness were observed on establishment of lung infection, suggesting a reduced potential for colonization of new environments (362).

Except in *P. aeruginosa*, the mutator isolates usually represent a low percentage of the population because of the burden of deleterious mutations. However, the mutators are sometimes linked to hypervirulent clones, as in the epidemic *N. meningitidis* serogroup A, which is the only serogroup capable of transcontinental spread and disease and which includes a high prevalence of isolates withelevated mutability. Richardson et al. showed that 57% of serogroup A isolates studied displayed elevated mutability, indicating the important role of this factor in the spread and evolution of serogroup A strains of *N. meningitidis* (363). A similar situation can be observed by contemplating the global distribution and success of the W-Beijing genotype of *Mycobacterium tuberculosis*, which is highly virulent and pathogenic. In a study of 55 W-Beijing isolates, most of the isolates carried missense mutations in *mut* genes, showing that successive alterations of the defective DNA repair systems may provide a selective advantage to the bacteria to adapt and persist, including resistance to antituberculosis drugs (364).

Hypermutation of strains of *S.* Typhimurium has also been studied in an *in vivo* mouse model, by comparing the fitnesses of wild-type and DNA repair-defective mutator strains. After 66 to 132 generations, the MICs of nalidixic acid and rifampin for the mutator strains were much higher than those for the wild-type strains, and in both types of strains the evolved populations of mice were fitter than the parental strain. In secondary environments, i.e., growing in LB medium under laboratory conditions, there was no general loss of fitness (measured as competition index [CI]) between the evolved and the parental populations. However, analysis of the capability to utilize various carbon sources showed that none of the wild-type lineages had lost any metabolic functions but that all the mutator strains had one or more defects. The authors of this study finally associated adaptation to the infection in mice by the mutators with a loss of fitness in secondary environments due to reduced metabolic capability (352). The role of the mutator phenotype in the pathogenesis of *E. coli* during chronic urinary tract infections was also shown in *in vivo* models (365).

In conclusion, hypermutation confers advantages such as the rapid acquisition of chromosomal resistance by antibiotic pressure, and it also enables modification of virulence and metabolism in order to persist in specific environments, although usually with a biological cost outside the primary environment.

### Persister Cells

Persisters are dormant variants of microbial populations that are tolerant to antimicrobials (not really resistant). They favor the recalcitrance of chronic infections to therapy and are more frequent in the biofilm state, which partly explains why it is difficult to treat infections caused by pathogens in biofilms. Persisters are particularly frequent in *P. aeruginosa* infections in patients with cystic fibrosis (366) and in *Candida albicans* infections in patients with oral thrush biofilm (367), although they have also been described in other species such as *S. aureus* (368) and *E. coli* (369).

Although the mechanism of antimicrobial tolerance in these subpopulations is not yet clear, it does not appear to operate at a mutational level. The SOS response (370, 371) and environmental stresses, such as oxidative stress (372), appear to be involved in the activation of persister formation. The tolerance involves overproduction of several toxin/antitoxin systems, such as RelE, which inhibits translation, thus preventing the lethal action of aminoglycosides to act over protein synthesis (373), as well as TisAB, a DNA damage-induced toxin that promotes the production of ciprofloxacin-tolerant cells (374), and HipAB, other toxin/antitoxin system involved in blocking translation and tolerance to different antibiotic families (371). Induction of the AcrAB-TolC efflux pump, by means of oxidative stress, also increases the production of quinolone-tolerant persister cells (372).

Overall, the data summarized above show that persistent subpopulations are a clear example of how pathogenic populations can evolve and adapt to survive and persist in the environment. In such antimicrobial-tolerant populations, which are difficult to eradicate, tolerance is closely linked to the expression of different virulence factors.

#### Alarmone Guanosine Tetraphosphate

The guanosine tetraphosphate alarmones, which are intracellular signaling molecules collectively known as (p)ppGpp, are a clear link between antibiotic resistance and virulence. Levels of (p)ppGpp are correlated with the expression of virulence traits, including survival of stress in bacteria such as *Campylobacter jejuni, Brucella abortus, Streptococcus mutans*, and *Bacillus subtilis* (375–378), biofilm formation in *E. coli, S. mutans*, and *Listeria monocytogenes* (377, 379, 380), antibiotic resistance in *E. coli* and *Brucella abortus* (376, 381–383) and infection persistence in *M. tuberculosis, S. mutans*, and *E. coli* (376, 381, 384–387). In some cases, animal models have provided unequivocal evidence of the role of (p)ppGpp in stress responses, vancomycin tolerance, and virulence in *E. faecalis* (388). It is not yet clear whether (p)ppGpp plays a direct role in the expression of genes that confer vancomycin tolerance to *E. faecalis* or if it initiates a regulatory cascade that leads to tolerance. Until recently, bifunctional RelA was considered the only enzyme responsible for controlling (p)ppGpp metabolism in Gram-positive bacteria. However, two related small enzymes, designated RelP and RelQ, were recently identified and shown to function as true alarmone synthetases in *S. mutans, S. pneumoniae*, and *B. subtilis* (389–391). Recent studies have revealed that bacteria have evolved different modes of (p)ppGpp regulation and that the effects of (p)ppGpp on cell physiology vary greatly among different organisms (390–392). Abranches et al. studied the inhibitory concentrations of vancomycin for *E. faecalis* and found that the MIC was the same for the *relA* mutant and the wild-type strain and also that the mutant strain grew faster in the presence of subinhibitory concentrations of the drug and survived better in time-kill studies. However, vancomycin MICs for the *relQ* and *relAQ* strains were lower and growth of these strains was slower or impaired in the presence of subinhibitory concentrations, and they were killed more rapidly by vancomycin; these characteristics were more pronounced in the *relAQ* strain (388). The findings confirm the involvement of these enzymes in the control of metabolism of the ppGpp molecules (Fig. 9).

However, these mechanisms are better characterized in other bacteria, such as *S. aureus* and *E. coli*. Gao et al. (393) carried out a comparative and functional genomic study of sequential *S. aureus* isolates from a patient with persistent and recurrent *S. aureus* infection after therapy with multiple antimicrobials (including linezolid) had failed. The results showed that the mutations in *relA* caused accumulation of (p)ppGpp, which was associated with reduced growth and attenuated virulence in the *G. mellonella* model; furthermore, mutations in *rlmN* (encoding a ribosomal methyltransferase that methylates 23S rRNA at position A2503) caused a reduction in linezolid susceptibility. The association between (p)ppGpp levels and resistance to antibiotics has been observed in *E. coli*. It has been demonstrated that artificially raising (p)ppGpp levels increases β-lactam tolerance in *E. coli* (382), and mutant cells lacking RelA are more
Resistance and Virulence: a Successful Association?

FIG 9 Growth curves and time-kill curves for *E. faecalis* OG1RF, *relA* mutant (*relA* encodes an enzyme responsible for controlling (p)ppGpp metabolism in Gram-positive bacteria), *relQ* mutant (*relQ* encodes an enzyme related to constitutive expression of ppGpp in nonstressed cells), and *relAQ* double mutant strains in the presence of subinhibitory concentrations of vancomycin ([386], [390]). (A) Growth curves reveal that the *relA* mutant grows faster than *E. faecalis* OG1RF (a strain harboring the *ebp* gene, which encodes endocarditis and biofilm formation-associated pilus operon). However, *relQ* and *relAQ* mutants strains display slow or impaired growth in the presence of vancomycin. (B) Time-kill curves reveal that the *relA* mutant survives better than the *relQ* and *relAQ* mutants, which were killed more rapidly by vancomycin. (Reprinted from reference [388] with permission.)

susceptible to β-lactams ([383]). In the latter case, resistance to microcin J25 was shown to involve (p)ppGpp-dependent induction of an efflux pump responsible for lowering the intracellular level of the peptide ([383]).

In a comparison of genomic and functional markers in strains of *S. aureus* in a patient with persistent and recurrent infections, mutations were detected in two genes, *relA* and *rlmN*. The mutation in the *relA* gene was associated with accumulation of the intracellular signaling molecule alarmone guanosine 3′,5′-bis-(diphosphate) (ppGpp) and permanent activation of the stringent response. The second mutation found in *rlmN*, which encodes a ribosomal methyltransferase that also methylates the 23S rRNA at position A2503, caused a reduction in linezolid susceptibility. These results show the capacity of adaptation of *S. aureus* and also how subtle molecular changes cause major alterations in bacterial behavior ([393]).

### Alternative Sigma Factor σ<sup>B</sup>

The alternative sigma factor σ<sup>B</sup> modulates the general stress response in certain Gram-positive bacteria, including *S. aureus*, *B. subtilis*, and *L. monocytogenes* ([394]). In *S. aureus*, alternative sigma factor σ<sup>B</sup> controls the expression of multiple genes, including virulence determinants such as α-hemolysin (encoded by *hla*), fibronectin-binding protein A (encoded by *fnbA*), and global regulators (*SarA* and/or *Agr* locus); it also promotes capsule production ([395]) and increases the resistance levels of methicillin-resistant *S. aureus* (MRSA) and glycopeptide-intermediate *S. aureus* (GISA) strains ([396], [397]). It has recently been shown that inactivation of the staphylococcal factor σ<sup>B</sup>-controlled *yabJ-spoVG* operon, which codes for *B. subtilis* YabJ and SpoVG sequence homologs, significantly reduces the level of transcription of the *cap* operon and impedes capsule formation in capsular polysaccharide-producing strains ([398]). Schulthess et al. ([396]), showed that deletion of the factor σ<sup>B</sup>-dependent *yabJ-spoVG* operon in MRSA and GISA strains decreased the resistance levels in a similar way as observed for the deletion of factor σ<sup>B</sup>, suggesting that factor σ<sup>B</sup> exerts its effect on methicillin and glycopeptide

resistance via the gene products of the *yabJ-spoVG* locus, in particular, via SpoVG. It remains to be determined which of the multiple chromosomal genes affecting methicillin or glycopeptide intermediate resistance levels are controlled by the factor σ<sup>B</sup>-SpoVG cascade. In *L. monocytogenes*, σ<sup>B</sup> proteins have also been associated with the vancomycin stress response. Factor σ<sup>B</sup> may contribute to monitoring and maintaining cell wall integrity by regulating certain genes and factors that are important in stress responses, cell metabolism, and virulence ([399]).

#### NOVEL ANTIVIRULENCE THERAPIES: MECHANISMS OF RESISTANCE TO ANTIVIRULENCE COMPOUNDS

Antivirulence therapies are based on inhibition of bacterial virulence and do not affect bacterial growth; those antivirulence compounds can be administered in combination with antibiotic treatment, thereby decreasing the selective pressure on bacteria and preventing the development of resistance to these antibiotics ([400]). Inhibition of the following mechanisms of virulence enables interruption of colonization and infection processes or of the development of bacterial infection: toxin production, adhesion, bacterial secretory systems, cell-to-cell signaling, iron metabolism, and virulence mechanisms involved in the host immune response and antibiotic resistance (Table 3). Of all the antivirulence compounds included in Table 3, we will comment on several compounds from two groups (cell-to-cell signaling inhibitors and RND efflux pump inhibitors) because of their involvement in resistance and virulence.

##### Cell-to-Cell Signaling Inhibitors

Bacteria were previously thought to act independently of stimuli. However, more recent studies show that bacteria can communicate with each other through complex signaling networks ([400]). Molecules that act by inhibiting this mechanism have been described. A couple of examples are as follows. (i) Compounds that inhibit the expression of virulence genes and also have a synergistic effect on the bactericidal action of certain antibiotics are beginning to appear, e.g., the compound tomatidine, which displays the

| Antivirulence strategies | Compound/name in study | Target/mode of action | Reference(s) |
| --- | --- | --- | --- |
| Toxins | Per-6-(3-aminopropylthio)-β-cyclodextrin | PA heptamer pore/blocks toxin binding | 440 |
|  | 2(R)-2-[4-Fluoro-3-methyl-1-phenyl]sulfonyl]-N-hydroxy-2-(tetrahydro-2H-pyran-4-y)acetamide | LF subunit/inhibited by protease activity | 441 |
|  | Cisplatin | LF subunit/inhibited by degradation of MAPKK | 442 |
|  | Symsorb-Pk | Gb3/blocks toxin binding | 443, 444 |
| Adhesion and colonization | Bicyclic 2-pyridones (plicides) | PapD/prevents pilus assembly | Urinary tract infection; replacement of conventional antibiotics | 445 |
|  | Virstatin | ToxT/inhibitor of regulator of pilus assembly | Replacement of conventional antibiotics | 446 |
| Bacterial secretion system | Acylated hydrazones of salicylaldehydes | Yop/inhibition of type III secretion systems | Chlamydia and Shigella sp. infections; replacement of conventional antibiotics | 447, 448 |
|  | 2-imino-5-arylidene thiazolidinone | Sip/inhibition of type III secretion systems | Salmonella, Pseudomonas, and Yersinia sp. infections; replacement of conventional antibiotics | 449 |
|  | Diaryl cyanotriole | Inhibition of sortase A | Activity in vitro against S. aureus | 450 |
| Cell-to-cell signaling | Furanones (C-30) | AHLs/inhibition of quorum sensing | Pseudomonas infections; replacement of conventional antibiotics | 451, 452, 453 |
|  | Thiophenones | IcaC, LrgB/inhibition of quorum-sensing | Staphylococcus epidermidis infections | 454 |
|  | Substrate analogs 3/oxoC12D10 | LasR/all competition, inhibition of quorum sensing | Pseudomonas infections; replacement of conventional antibiotics | 432, 454 |
|  | Substrate analogs/C4 | LuxR/TraR antagonist, inhibition of quorum sensing | Vibrio fischeri infections; replacement of conventional antibiotics | 455 |
|  | 7-Fluoroinodole | Inhibition of quorum sensing | Pseudomonas infections; replacement of conventional antibiotics | 456, 457 |
|  | Catechins (galloyl group) | Lux system/inhibition of quorum sensing | Vibrio harveyi and Eikenella corrodens infections; replacement of conventional antibiotics |  |
| Tomatidine |  | Agr system inhibitor | S. aureus infections; potentiator of the action of aminoglycoside antimicrobials | 403 |
| 26%HACC-loaded PMMA (chitosan derivative) |  | icaAD-icaR/inhibition of biofilm formation, including antibiotic-resistant strains | Implant infections and osteomyelitis caused by methicillin-resistant S. aureus | 402 |
| Salicylate |  | marA-fimB regulator/inhibition of biofilm formation | In vitro activity against E. coli | 458 |
| LED 209 |  | QscE/inhibition of quorum-sensing | Postexposure to the pathogen; replacement of conventional antibiotics | 458 |
| Iron metabolism | Acyladenylate derivatives | MbtA in M. tuberculosis, YbtE in Y. pestis, inhibition of aryl acyl adenylating enzymes | In vitro activity against M. tuberculosis and Y. pestis | 459, 460 |
|  | Piridine derivative/HTS85K | BasE/nonnucleoside inhibitor | A. baumannii infections | 294 |
| Virulence factors involved in host immune response | Sulfamoyl D-Ala | mPTPB inhibitor/resistance to stress oxidative innate immunity | Prevention of M. tuberculosis growth in host cells | 461 |
|  | I-A09 | DltA/protection against cationic effectors of the innate immunity | In vitro activity against B. subtilis strains that are more sensitive to vancomycin | 462 |
| RND efflux pump inhibitors | 1-(1-Naphthylmethyl)-piperazine, phenyl-arginine-β-naphthylamide | RND efflux pumps, cholera toxin factor, toxin-coregulated pilus/increased susceptibility and inhibition of production of virulence factors | In vitro activity against V. cholerae | 404 |
|  | Trifluoromethyl ketones (12 compounds) | RND efflux pumps and quorum-sensing inhibitor/ regulation of cell functions (virulence, biofilms, and motility) | In vitro activity against C. violaceum O26 and E. coli | 401 |

effector-enhanced bactericidal action of aminoglycosides and also inhibits the expression of virulence genes linked to the Agr system in strains of S. aureus that are susceptible and resistant to various antimicrobials (403). (ii) The new quaternized chitosan derivative hydroxypropyltrimethyl ammonium chloride chitosan with a 26% degree of quaternary ammonium substitution (26%HACC) displays strong antibacterial activity against Staphylococcus spp. and simultaneously good biocompatibility with osteogenic cells. It was found that 26%HACC-loaded polymethylmethacrylate (PMMA) markedly downregulated the expression of *icaAD*, which encodes essential enzymes for polysaccharide intercellular adhesion biosynthesis, upregulated the expression level of *icaR*, which negatively mediates *icaAD* expression, and also downregulated the expression of *mecA*, which encodes membrane-bound enzymes known to be penicillin-binding proteins. Hence, the 26%HACC-loaded PMMA prevents biofilm formation by Staphylococcus spp., including methicillin-resistant strains, on the surface of bone cement and downregulates virulence-associated gene expression of antibiotic-resistant Staphylococcus spp., thus providing a promising new strategy for combating implant infections and osteomyelitis (402).

### RND Efflux Pump Inhibitors

RND efflux pumps play a role in the pathogenicity of bacteria, and they mainly affect colonization, infection, and the persistence of microorganisms in the host (66). Moreover, these efflux pumps are involved in the QS-regulated expression of virulence determinants (68). Several RND pump inhibitors have been described. We analyze two examples: (i) The 1-(1-naphthylmethyl)-piperazine (NMP) and phenyl-arginine-β-naphthylamide (PAN), which act as inhibitors of RND efflux pumps and virulence factors in *Vibrio cholerae*, such as the cholera toxin and the toxin-coregulated pilus (404), have been suggested as a useful tool for the treatment of cholera infections. (ii) Of 12 trifluoromethyl ketone compounds tested, 6 proved to be effective inhibitors of the quorum-sensing response by *Chromobacterium violaceum* 026, as well as inhibitors of the RND efflux pumps of CV026 and *E. coli*. This finding is of clinical relevance and may be exploited for the prevention of QS responses of infecting bacteria (401). Inhibitors of efflux pump systems have therefore been suggested as a useful tool for the treatment of cholera infections.

### Antimicrobial Compounds with Antivirulence Activity

Some studies have investigated the antivirulence activity of antibiotics such as azithromycin (macrolide), linezolid (oxazolidinones), rifampin, and ciprofloxacin. Azithromycin and rifampin present a clear anti-inflammatory activity; however, in recent years there have been publications showing the antivirulence activity of these compounds. Kohler et al. associated azithromycin treatment with inhibition of QS in *P. aeruginosa* infections, which presents a clinical benefit for the patient. However, the authors commented that if the treatment is discontinuous, there is an increased risk of colonization by highly virulent bacteria, which highlights the need to study the positive and negative implications of antivirulence therapies (405). Azithromycin has been shown to improve pulmonary function without reducing the number of bacteria. It is known that in the development of *P. aeruginosa* infection, the pathogen releases virulence factors that affect the epithelial integrity, thus exposing the epithelium to additional bacterial infiltration. Administration of azithromycin in patients with cystic fibrosis attenuated the effect on epithelial integrity (406). On the other hand, in a recent study with animal models, it was shown that treatment with azithromycin inhibits intracellular killing of multidrug-resistant nontuberculous mycobacteria (NTM) within macrophages. The authors concluded that azithromycin blocks the development of autophagy by promoting the development of persistent infections by NTM (mainly *M. abscessus*) in patients with cystic fibrosis (407). Rifampin can attenuate the cellular damage induced by multidrug- and pan-drug-resistant *A. baumannii* clinical isolates without being significantly bactericidal. Indeed, the cytoprotective effect of rifampin was observed as a decrease in the number of dead cells induced by *A. baumannii*, achieved by reducing oxidative stress and proinflammatory cytokine release (408).

In relation to linezolid (LZD), one study examined the application of subinhibitory concentrations that suppress virulence factors in methicillin-resistant *S. aureus* infections, which may be associated with a reduction of endogenous pyrogens. This may at least partly explain early defervescence observed in LZD-treated individuals (409). Moreover, systemic treatment with LZD has recently been shown to have an inhibitory effect on biofilm development in methicillin-resistant *S. aureus* (MRSA) infection of the endotracheal tube (410). Finally, ciprofloxacin associated with lytic bacteriophages has been used in the treatment of the *K. pneumoniae* infections. The combination treatment not only killed the bacteria but inhibited biofilm formation and significantly restricted the formation of resistant variants in comparison with individual treatments (411).

In vitro experiments have shown that the application of probiotic isolates of *Lactobacillus rhamnosus* GG prevents invasion of the upper respiratory tract by group A macrolide-resistant streptococci carrying the *prtF1* gene, which encodes fibronectin (Fn)-binding F1 invasion (412). This effect is due to competition between *L. rhamnosus* GG and group A macrolide-resistant streptococci for Fn binding in the inhibition process, which confirms the antagonistic action of *L. rhamnosus* GG against group A streptococci.

### Resistance to Antivirulence Components

To date, the only known study of the mechanism of resistance to antivirulence components is that by Maeda et al. concerning resistance to furanone C-30 (99). Furanones interact with the transcriptional regulator descending signal LASR acyl homoserine lactone and therefore attenuate the virulence in a pulmonary infection model of *P. aeruginosa* in mice. The authors worked with mutants of the *mexR* and *nalC* genes, which are negative regulators of the MexAB-OprM efflux pump, and observed expulsion of compound C-30 by the MexAB-OprM efflux pump in the mutants (Fig. 10). Furthermore, the results were confirmed in the Liverpool epidemic strain (overexpression of MexAB-OprM, *nalC* and *mexR* mutants), which is known to increase morbidity in patients with cystic fibrosis (98, 413). On the other hand, a *mexA* mutant with reduced MexAB-OprM efflux pump activity was previously found to be less virulent in a pathogenesis model (*P. aeruginosa* and *C. elegans*) (414). Moreover, Maeda et al. reported that the *mexR* mutant (with enhanced MexAB-OprM efflux pump activity) is as virulent as the wild-type strain and is much more virulent in the presence of the QS inhibitor (C-30) than the wild-type strain (99).

Beceiro et al.

a

| PA14 | mexR | nalC |
| --- | --- | --- |
| rhIR |  |  |
| MeOH | MeOH | MeOH |
|  | C-30 | C-30 |

b

| Pyocyanin (OD<sub>520</sub>/OD<sub>600</sub>) |
| --- |
| 0.010 |
| 0.008 |
| 0.006 |
| 0.004 |
| 0.002 |
| 0.000 |
| PA14 | mexR | nalC | phzM |

FIG 10 The *mexR* and *nalC* mutations decrease C-30 inhibition of *P. aeruginosa* QS phenotypes. (a) Swarming motility. The *rhlR* mutant (*rhlR* encodes the two transcriptional regulators of the acyl homoserine lactone system of *P. aeruginosa*) was used as a negative control. MeOH was used as a negative control for C-30. (b) Production of pyocyanin (phenazine toxic metabolites). An asterisk indicates statistical significance. A PA14 *phz* mutant (a strain without phenazine production) was used as a negative control. (Reprinted from reference 99 by permission from Macmillan Publishers Ltd.)

DISCUSSION

Finally, we address the following series of questions and reflections that arise in relation to the prospects in the near future.

How Does Increased Antimicrobial Resistance Affect Virulence?

Throughout this review we have examined some examples of the most important factors that affect the complex process of bacterial evolution, specifically the evolution of antibiotic resistance and virulence. There are no general solutions to this puzzle, and the association between virulence and resistance in a specific pathogen will depend on the interactions between the multiple factors associated with bacteria and their environments.

The final effect (whether positive or negative) of the association between bacterial virulence and antimicrobial resistance depends mainly on four factors. (i) The first factor is the bacterial species. Some microorganisms readily acquire antibiotic resistance mechanisms and evolve rapidly in response to antibiotic pressure (e.g., *P. aeruginosa* and *A. baumannii*). However, others remain fully susceptible to penicillin, although this has long been the treatment of choice for infections caused by the particular pathogen (e.g., *S. pyogenes*). Therefore, in the first type of bacteria, virulence will be

Clinical Microbiology Reviews
more greatly influenced by the acquisition of resistance. (ii) The second factor is specific virulence and resistance mechanisms. These are involved in both processes at the same time (e.g., the AcrAB-TolC efflux pump of *E. coli*, which expels fatty acids and bile salts and also antibiotics) ([415](#), [416](#)) or are involved indirectly (e.g., the PhoP-regulated resistance to colistin in *P. aeruginosa*, which causes changes in the LPS and loss of affinity and is simultaneously involved in a decrease in virulence via lower biofilm production and lower cytotoxicity) ([197](#)). (iii) The third factor is the environment or ecological niche. This factor largely determines the development of the infection. Clear examples include the following: two-component systems, which regulate virulence and resistance at the same time and which depend on external stimuli; the presence/absence of specific molecules (e.g., depletion of iron during the course of an infection in the host); the antibiotic concentration, which depends on the anatomical site of an infection; and a high NaCl concentration in the environment, which in *A. baumannii* may trigger a response that increases the resistance to several antibiotics through upregulation of efflux pumps and release of outer membrane proteins involved in virulence ([417](#)). (iv) The fourth factor is the host (the immune system). Although in the strictest sense the host apparently does not affect the virulence of a pathogen, the host/pathogen interaction is key in the development of an infection and therefore in how the acquisition of resistance affects virulence.

The concentration of antibiotics in an environment will determine the importance of the antibiotic pressure on bacterial evolution in that niche, and selection will be directed toward the success of the most resistant pathogens. However, during colonization and infection, the most virulent pathogens will be the most successful and will therefore be the most likely to survive. Emergence of antibiotic resistance in a bacterial population is related to different factors, the following of which are critical: the rate of mutation toward resistance, the acquisition of horizontally transmitted genes, and the fitness cost to the bacteria. If resistance implies a fitness cost for the microorganism, the growth rate would not be sufficient to compete with the fitness of susceptible bacteria, with the killing mediated by immune system cells, or with the dynamics of the body fluids during the process of colonization or infection. The fitness cost of the mechanism and the possibility of being selected through the coselection process will have a strong impact on the success of the selection. Note that we are not considering another factor here, the epidemicity, or the capacity to survive/persist and disseminate in specific environments, which may deserve by itself further review.

Thus, in a nosocomial environment, resistant and/or virulent bacteria (coselction) may be readily selected as a result of selective antibiotic pressure, and the effect of the biological cost of the resistance mechanism is secondary to the strong antibiotic pressure. However, the situation is different in the community environment. In environments in which there is no direct selective antibiotic pressure (low concentrations of antibiotics), increased resistance will probably have a negative effect on the relationship between resistance and virulence, and if the biological cost of this mechanism is high, the mechanism will tend not to be selected.

---

## What Are the Perspectives in the Near Future in Relation to Antimicrobial Resistance and Virulence?

Our current knowledge of the world of pathogenic microbes is possibly only the tip of the iceberg. Most basic studies seek direct solutions to major clinical problems and overlook the genetic backgrounds of most microorganisms that are not directly involved in infectious processes. Thus, for example, most structural and phylogenetic studies on plasmids are performed with clinical isolates or isolates with antimicrobial resistance mechanisms, and those not associated with clinical environments are disregarded. This leads to a huge bias in our knowledge of plasmid biology and our understanding of the dissemination of antimicrobial resistance. The explosion of new molecular techniques in the last decade will gradually allow us to overcome this bias and obtain a much broader view of the microbial world, which will lead to a better understanding of the specific interactions between resistance and virulence and bacterial functioning in general.

Regarding what we can expect in the near future, we must consider how bacterial virulence evolved prior to the presence of large amounts of antibiotics in the environment. When low concentrations of antibiotics were in nature, those antibiotics probably had a small effect on bacterial resistance/virulence. However, this scenario changed dramatically in the 1940s, when antibiotics began to be used widely in clinical practice; the evolution of virulence has been greatly affected by the indiscriminate use of antibiotics and has probably moved irreversibly in a new direction. The development of virulence factors and the evolution of resistance determinants in pathogens overlap considerably, and it is difficult to consider them as independent processes. It is important to emphasize the global impact on the microbiota, including the human microbiota. The huge amounts of antibiotics used in virtually all environments ([223](#)) cause displacement (input of exogenous antibiotic-resistant bacteria) or modification (antibiotic-resistant mutants) of the microbiota. Globally, the nonpathogenic species, commensals, are now less susceptible than before ([3](#)), regardless of their ecological niche. Moreover, strains that are poorly adapted or poor colonizers of new niches, but which are resistant, are now more likely to replace those well-adapted or colonizing species that are more susceptible to the presence of antibiotics ([418](#)). The impact of these changes on human health is so far unknown.

Thus, microorganisms, such as opportunistic pathogens, are able to compete in new niches where previously only commensals or nonpathogenic microorganisms existed. One clear example of this is the appearance in the past decades of opportunistic nosocomial pathogens (e.g., *P. aeruginosa* and *A. baumannii*) that are not capable of producing infection in healthy hosts but display a high capacity to survive, persist on surfaces (invasive equipment, etc.), and adapt to the nosocomial environment, where they may cause infection in patients with weakened immune systems. The ability to acquire antibiotic resistance is one of the main characteristics of such species. Other pathogens, such as *E. coli* as an example among others, do not acquire resistance to antibiotics as readily (the relative prevalence of *A. baumannii* or *P. aeruginosa* in hospital settings is currently increasing at a greater rate than that of *E. coli*). The capacity to develop or acquire resistance and the ability to persist in complicated “hostile” environments are key factors in explaining the increment in the number of infections at least in hospitalized patients. Jawad et al. concluded that in *A. baumannii*, both desiccation tolerance and MDR phenotype may contribute to the maintenance of this species in the hospital setting and may partly explain its propensity to cause prolonged outbreaks of nosocomial infections ([419](#)). This is a typical example of a microorganism in which the clinical importance parallels the increase in antibiotic use and selection of resistance strains.
Beceiro et al.

Once the fitness cost of the resistance mechanisms is compensated if necessary with compensatory mutations, these pathogens can probably also increase their virulence by acquiring new virulence factors, as in, e.g., *A. baumannii*. Thus, we must consider the following, at least taken as example with *A. baumannii*. Are more infections already caused by *A. baumannii*? Are these strains more resistant? Are these strains more virulent? Probably, and considering the difficulties in providing a convincing and rational response to these questions, affirmative answers to the three questions can be considered. To date, there are probably more *A. baumannii* infections (we can simply search the appropriate keyword in PubMed to get an approximate idea of the increasing numbers of infections and nosocomial outbreaks), and the strains often are more virulent; these processes are linked as a result of coselection and coevolution (44). More infections are already occurring because *A. baumannii* has adapted to living in the presence of antibiotics, and furthermore, it has evolved to cause greater damage to hosts in order to survive inside the host and hospital environments (44). Apart from specific examples like this, the effects that these changes will have on human health are so far unknown. However, if this hypothesis is correct, microbiological studies in the near future will show whether or not *A. baumannii* is becoming more virulent and more resistant.

Bearing in mind that antibiotics are present in the vast majority of bacterial niches on the planet and that we are unlikely to stop using these compounds, resistance does not appear to be reversible, although it is an increasingly urgent problem (420). Given the huge number of bacterial generations that occur within during a single human generation, it seems unlikely that the limited arsenal of antibiotics available at present or that is likely to be available in the immediate future can compete in biological terms with the enormous capacity for bacterial pathogens to evolve and change (421, 422). Clearly, microorganisms will be increasingly resistant in the future. However, will they be more virulent? In this review we have discussed numerous examples of the association between virulence with resistance, which can be positive or negative, with the final balance determined by the ability of the bacterium to survive or adapt in a specific ecological niche. However, the huge genetic arsenal that bacteria can make use of to compensate for or overcome the fitness costs suggests that in the near future virulent and resistant clones will emerge simultaneously. Processes that favor bacterial success include coselection (in same MGE, plasmids, transposons, islands, etc.) of the ability to spread resistance and virulence genes, along with compensatory mutations, hypermutation (which favors the development of both processes), and the effect of SOS-inducing antimicrobial agents (such as quinolones).

Genetic mechanisms that spread resistance genes often encode resistance to different families of antibiotics, which means that the withdrawal of a family of antibiotics would not cause the reduction or disappearance of strains resistant to that family (423). Carrying virulence genes also confers on bacteria some evolutionary advantage during the colonization and infection processes, which thus favors the resistant strains. This plasticity provides pathogens access to new strategies to explore new environments in which the normal microflora would be at a disadvantage. There are clear examples of the worldwide distribution of highly epidemic multiresistant clones that are selected mainly as a result of encoded multiresistance; these are the so-called high-risk clones, such as the KPC carbapenemase-positive *K. pneumoniae* ST258 clone, the *E. coli* ST131, ST38, ST393, and ST405 clones, which usually carry CTX-M-15 β-lactamase, and the carbapenem-resistant ST175 and O12 clones of *P. aeruginosa* (202, 203, 424). These clones are extremely well adapted to new environments with a high presence of antibiotics, and they are often more virulent. The PMEN1 clone of *S. pneumoniae*, which is distributed worldwide, is resistant to chloramphenicol and tetracycline, and many isolates are also resistant to fluoroquinolones and macrolides; the genome encodes several virulence factors, including the *blp* bacteriocins and cell wall surface-anchored proteins (210). Another example is the multiresistant and virulent Liverpool epidemic strain of *P. aeruginosa* in patients with cystic fibrosis (98, 425, 426). We note that the resistance of all these MDR clones with this genetic arsenal is increasing as a result of the so-called genetic capitalism concept, in which resistant organisms tend to become even more resistant (427). In addition to the importance of coselection, we highlight the role of compensatory mutations in fitness costs. It is relatively easy for bacteria to compensate for the increased fitness cost associated with new resistance, as resistant pathogens are able to maintain (via compensatory mutations, for example) this resistance and additionally retrieve or even increase their fitness. Therefore, compensatory evolution could stabilize resistant populations, even in the absence of antibiotics, so that they will be able compete with the other commensal microflora on equal terms. These compensatory mechanisms usually follow one of the three above-described strategies, and alternative mechanisms are used for the same function, thus reducing the need to carry out the function and restoring its effectiveness. Although the role of these compensations in the stability of resistant bacterial populations outside the laboratory is not yet clear, some authors have identified such mechanisms in the clinical environment and have recognized this as a potential problem in the coming years with the current treatments used (278).

## Are There Alternatives to Antimicrobial Therapy for Controlling Bacterial Multiresistance and Virulence?

When analyzing the prospects for the future, the urgent need for new antibiotics to treat emerging bacteria that are resistant to almost all antibiotics (extremely drug resistant [XDR]) should be taken into consideration. No truly new antibiotics have been developed against Gram-negative pathogens that have emerged since the 1990s. There is clearly a need to develop and market new antibiotics or compounds capable of increasing permeability in bacteria or of evading efflux and avoiding mutational resistance, among other approaches (428).

The development of innovative antivirulence therapies as a new way to combat resistant and/or virulent pathogens is a promising alternative to conventional antimicrobial treatment. An example is the proanthocyanidins, used in clinical practice, whose mechanism of action is based on binding to the filaments or fimbriae of *E. coli* and other bacteria in the urine and preventing these bacteria from adhering to the walls of the lower tract mucosa. The proanthocyanidin intake type helps to prevent recurrence infections, as recent studies show (429).

Several reviews that address new virulence inhibitors, both *in vivo* and *in vitro*, have been published. Moreover, many of these antivirulence compounds are capable of inhibiting the virulence of different species that have a common mechanism of virulence. Theoretically, the use of antivirulence therapies directed exclusively against such virulent microorganisms would avoid acting

Clinical Microbiology Reviews

against nonvirulent bacteria. This approach would therefore lead to preservation of susceptible nonpathogenic bacteria, which would act as a barrier for colonization by virulent populations; antibiotic pressure would also be lowered, thus minimizing the risk for horizontal spread of drug resistance genes (430, 431). The combined use of antibiotic therapy and antivirulence compounds could become an effective alternative within a few years (400, 420, 432). However, as with antibiotics, resistance to antivirulence compounds has already emerged, such as to furanone through increased expression of the MexAB-OprM efflux pump in the multiresistant and virulent Liverpool epidemic strain of *P. aeruginosa* (413). Finally, we highlight the antivirulence activity of some already commercialized antimicrobials, such as rifampin against *A. baumannii* (408), subinhibitory concentrations of linezolid against *S. aureus* (409), and azithromycin, a macrolide that exhibits antibacterial activity against *P. aeruginosa* and has also displayed antivirulence activity and anti-quorum-sensing activity (decreasing virulence and cooperation) (405, 406). Further studies of antibiotic activities other than bactericidal or bacteriostatic activities are required, as these could help in the development of new antimicrobial therapies for the treatment of infectious diseases.

It is interesting to note the possible role that vaccines may play in the evolution of pathogenic bacteria in the near future. Advances in our understanding of the immune response and molecular biology suggest that in coming years, we will witness a major advancement in the development and emergence of numerous vaccines (433). The long-term impact of vaccines would theoretically decrease populations of pathogens, thus reducing the possibility of the development of new mechanisms of resistance, either by new mutations or by horizontal transmission from other resistant pathogens. However, it is possible that in the long term, the clones/serotypes used in these vaccines may be displaced by other clones, present in low numbers prior to introduction of the vaccine and prevalent after its use. This is of some concern, and the selection of virulent serotypes has already been reported. In Utah, the number of cases of necrotizing pneumococcal pneumonia increased after the introduction of the seven-valent pneumococcal conjugate vaccine (PCV7). The prevalence of the non-PCV7 serotype was 49% before the use of the vaccine and increased to 88% after the introduction of PCV7; between these prevalences was that of serotype 3, which is associated with necrotizing pneumococcal pneumonias (434). Therefore, in addition to antibiotic pressure, the use of vaccines will exercise an increasing and important selective pressure on bacterial evolution, especially among the pathogenic microorganisms; whether both pressures will jointly exercise selection is still unknown. One promising alternative is the use of vaccines that specifically target strains or clones that are particularly important from a clinical point of view, such as MDR epidemic clones (i.e., *E. coli* ST131 or *K. pneumoniae* ST258) or virulent and resistant clones (i.e., the S. *pneumoniae* PMEN1 clone and the Liverpool epidemic strain of *P. aeruginosa* [LES]). Development of vaccines with epitopes that are conserved among strains of these specific traits would enable us to combat the emergence of these high-risk clones without damaging nonpathogenic commensal microorganisms. Thus, Wieser et al. have recently developed a vaccine in mice against extraintestinal pathogenic *E. coli* (ExPEC), which may be cost-effective for use in selected patient groups and is based on epitopes of several virulence-associated ExPEC proteins (435). Other alternatives are not

discussed in the present review, as they are probably out of our aim here.

## Are Antibiotic Resistance and Virulence Increasingly Linked in the Development of Infectious Processes?

Although most microbiological studies focus separately on virulence or resistance, the relationship and mutual biological impact of one event on the other is increasingly being studied. On writing this review, we realized that the joint detailed analysis of both processes will provide a better understanding of the relationship between virulence and antimicrobial resistance. Similarities become apparent when these processes are investigated together. Fundamentally, virulence is directly associated with the development of resistance from the point of view that (theoretically) just when the host shows clear signs of infection or disease (i.e., when the virulent pathogens are present), antibiotic therapy is administered; however, in the absence of infection (the normal flora of the host, without pathogenic microorganisms), exposure to antibiotics is probably much lower, so that the possibility of developing resistance due to the lack of antimicrobial pressure is also lower.

We have discussed two alternative scenarios in this review. Thus, in some cases increased resistance is accompanied by increased virulence, as in the following examples: in plasmids carrying both factors (229, 237, 436); during activation of the SOS system, which facilitates the spread of the resistance- and virulence-encoding genes at the same time (336, 337); and with accumulation of alarmones in *E. coli*, which is associated with biofilm formation and tolerance to β-lactams, among some others (382). However, in other examples, the increased antimicrobial resistance reduces the virulence (or fitness cost) of the microorganisms: during the acquisition in *E. coli* of β-lactamases, such as OXA-10, OXA-24, and SFO-1 (in which changes in the peptidoglycans have a fitness cost and thus explain the low incidence of some β-lactamases in this species) (34), and the fitness cost associated with the acquisition of vancomycin resistance in methicillin-resistant *S. aureus* (167), among many other examples. There is also a third option, in which there is apparently no significant effect on the virulence, such as the acquisition of the β-lactamases CTX-M-1, TEM-1, and CTX-M-32 in *E. coli* (32, 34) or of the *bla*~IMP~ metallo-β-lactamase in *P. aeruginosa* (33). The regulatory two-component systems in many bacterial pathogens, such as *S. aureus*, represent a clear example of the link between resistance and virulence (395, 396). Such associations also occurred prior to the massive use of antibiotics, so that in the future (with the development of new molecular biology techniques), new signs of these relationships could be identified.

This association between resistance and virulence follows a Darwinian model, in which those traits that confer a specific advantage will be selected (sooner or later) and become fixed. Those associations with a positive effect (increased resistance plus increased virulence) will be selected very rapidly. Those in which selection is apparently negative (i.e., increased resistance correlated with diminished virulence) will undergo a longer selection process, until a specific virulence advantage is selected and becomes fixed in the population. The opposite may also occur, so that increased virulence will lead to decreased resistance. In this case, compensatory mutations may arise to equilibrate the balance, and increased resistance and virulence will finally proceed together to confer the bacteria with a selective advantage.

tunately for humankind, this is merely a matter of time, as with all evolutionary processes. To face this threat, further in-depth studies are necessary to investigate this association, and new nonantimicrobial therapies are necessary to attempt to control the emergence of high-risk clones in multiple pathogenic species in the near future. Indeed, future trends in clinical microbiology laboratories should include identification of pathogens and susceptibility analysis but should also include the development of molecular techniques and identification of high-risk clones or known virulent or epidemic clones (such as the *E. coli* ST131 clone, *S. aureus* USA300, and the Liverpool clone of *P. aeruginosa* [LES]), along with the development of tests for the rapid detection of the most important virulence markers.

### ACKNOWLEDGMENTS

We thank Ana Fernández, Clara Povoa, Astrid Pérez, and Carmen Gayoso for their help in composing the figures.

Research in our laboratory is supported by grants from the Instituto de Salud Carlos III, the Spanish Network for Research in Infectious Diseases (REIPI RD 12/0015/0014), the Fondo de Investigaciones Sanitarias (PI08/1368, PS09/00687, and PI12/00552 to G.B. and PI10/00056 to M.T.), SERGAS (PS07/90), and the Xunta de Galicia (07CSA050916PR). A.B. is supported by the Secretaria de Estado de Investigación, Desarrollo e Innovación (program Juan de la Cierva), and M.T. is supported by the Instituto de Salud Carlos III (program Miguel Servet), both from the Ministerio de Economía y Competitividad.

### REFERENCES

1. Madigan MT, Martinko JM, Dunlap PV, Clark DP. 2009. Brock biology of microorganisms, 12th ed. Pearson Education Inc., Upper Saddle River, NJ.
2. Feldman MW, Laland KN. 1996. Gene-culture coevolutionary theory. Trends Ecol. Evol. 11:453–457.
3. Martínez JL, Baquero F. 2002. Interactions among strategies associated with bacterial infection: pathogenicity, epidemicity, and antibiotic resistance. Clin. Microbiol. Rev. 15:647–679.
4. Burrus V, Waldor MK. 2004. Shaping bacterial genomes with integrative and conjugative elements. Res. Microbiol. 155:376–386.
5. Handel A, Regoes RR, Antia R. 2006. The role of compensatory mutations in the emergence of drug resistance. PLoS Comput. Biol. 2:e137. doi:[10.1371/journal.pcbi.0020137](https://doi.org/10.1371/journal.pcbi.0020137).
6. Patel R. 2005. Biofilms and antimicrobial resistance. Clin. Orthop. Relat. Res. 437:41–47.
7. Seral C, Van Bambeke F, Tulkens PM. 2003. Quantitative analysis of gentamicin, azithromycin, telithromycin, ciprofloxacin, moxifloxacin, and oritavancin LY333328 activities against intracellular *Staphylococcus aureus* in mouse J774 macrophages. Antimicrob. Agents Chemother. 47:2283–2292.
8. Barbosa TM, Levy SB. 2000. Differential expression of over 60 chromosomal genes in *Escherichia coli* by constitutive expression of MarA. J. Bacteriol. 182:3467–3474.
9. Tsai YK, Fung CP, Lin JC, Chen JH, Chang FY, Chen TL, Siu LK. 2011. *Klebsiella pneumoniae* outer membrane porins OmpK35 and OmpK36 play roles in both antimicrobial resistance and virulence. Antimicrob. Agents Chemother. 55:1485–1493.
10. Moya B, Juan C, Alberti S, Perez JL, Oliver A. 2008. Benefit of having multiple *ampD* genes for acquiring beta-lactam resistance without losing fitness and virulence in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 52:3694–3700.
11. Yeung AT, Bains M, Hancock RE. 2011. The sensor kinase CbrA is a global regulator that modulates metabolism, virulence, and antibiotic resistance in *Pseudomonas aeruginosa*. J. Bacteriol. 193:918–931.
12. Lye DC, Earnest A, Ling ML, Lee TE, Yong HC, Fisher DA, Krishnan P, Hsu LY. 2011. The impact of multidrug resistance in healthcare-associated and nosocomial Gram-negative bacteremia on mortality and length of stay: cohort study. Clin. Microbiol. Infect. 18:502–508.
13. Andersson DI, Levin BR. 1999. The biological cost of antibiotic resistance. Curr. Opin. Microbiol. 2:489–493.
14. Andersson DI, Hughes D. 2011. Persistence of antibiotic resistance in bacterial populations. FEMS Microbiol. Rev. 35:901–911.
15. Rudkin JK, Edwards AM, Bowden MG, Brown EL, Pozzi C, Waters EM, Chan WC, Williams P, O’Gara JPRC, and Massey. 2012. Methicillin resistance reduces the virulence of healthcare-associated methicillin-resistant *Staphylococcus aureus* by interfering with the agr quorum sensing system. J. Infect. Dis. 205:798–806.
16. Pozzi C, Waters EM, Rudkin JK, Schaeffer CR, Lohan AJ, Tong P, Loftus BJ, Pier GB, Fey PD, Massey RC, O’Gara JP. 2012. Methicillin resistance alters the biofilm phenotype and attenuates virulence in *Staphylococcus aureus* device-associated infections. PLoS Pathog. 8:e1002626. doi:[10.1371/journal.ppat.1002626](https://doi.org/10.1371/journal.ppat.1002626).
17. Queck SY, Khan BA, Wang R, Bach TH, Kretschmer D, Chen L, Kreiswirth BN, Peschel A, Deleo FR, Otto M. 2009. Mobile genetic element-encoded cytotoxin connects virulence to methicillin resistance in MRSA. PLoS Pathog. 5:e1000533. doi:[10.1371/journal.ppat.1000533](https://doi.org/10.1371/journal.ppat.1000533).
18. Hicks LA, Chien YW, Taylor TH, Haber M, Klugman KP. 2011. Outpatient antibiotic prescribing and nonsusceptible *Streptococcus pneumoniae* in the United States. 1996–2003. Clin. Infect. Dis. 53:631–639.
19. Rieux V, Carbon C, Azoulay-Dupuis E. 2001. Complex relationship between acquisition of beta-lactam resistance and loss of virulence in *Streptococcus pneumoniae*. J. Infect. Dis. 184:66–72.
20. Zapun A, Contreras-Martel C, Vernet T. 2008. Penicillin-binding proteins and beta-lactam resistance. FEMS Microbiol. Rev. 32:361–385.
21. Azoulay-Dupuis E, Rieux V, Muffat-Joly M, Bedos JP, Vallee E, Rivier C, Isturiz R, Carbon C, Moine P. 2000. Relationship between capsular type, penicillin susceptibility, and virulence of human *Streptococcus pneumoniae* isolates in mice. Antimicrob. Agents Chemother. 44:1575–1577.
22. Fernandez A, Cabellos C, Tubau F, Linares J, Viladrich PF, Gudiol F. 2001. Relationship between penicillin and cephalosporin resistance of *Streptococcus pneumoniae* strains and its inflammatory activity in the experimental model of meningitis. Med. Microbiol. Immunol. 190:135–138.
23. Bédos JP, Rolin O, Bouanchaud DH, Pocidallo JJ. 1991. Relationship between virulence and resistance to antibiotics in pneumococci. Contribution of experimental data obtained in an animal model. Pathol. Biol. 39:984–990.
24. Briles DE, Crain MJ, Gray BM, Forman C, Yother J. 1992. Strong association between capsular type and virulence for mice among human isolates of *Streptococcus pneumoniae*. Infect. Immun. 60:111–116.
25. Russo TA, MacDonald U, Beanan JM, Olson R, MacDonald II, Sauberan SL, Luke NR, Schultz LW, Umland TC. 2009. Penicillin-binding protein 7/8 contributes to the survival of *Acinetobacter baumannii* in vitro and in vivo. J. Infect. Dis. 199:513–521.
26. Cayo R, Rodriguez MC, Espinal P, Fernandez-Cuenca F, Ocampo-Sosa AA, Pascual A, Ayala JA, Vila J, Martinez-Martinez L. 2011. Analysis of genes encoding penicillin-binding proteins in clinical isolates of *Acinetobacter baumannii*. Antimicrob. Agents Chemother. 55:5907–5913.
27. Valat C, Haenni M, Saras E, Auvray F, Forest K, Oswald E, Madec JY. 2012. CTX-M-15 extended-spectrum beta-lactamase in a Shiga toxin-producing *Escherichia coli* isolate of serotype O111:H8. Appl. Environ. Microbiol. 78:1308–1309.
28. Ishii Y, Kimura S, Alba J, Shiroto K, Otsuka M, Hashizume N, Tamura K, Yamaguchi K. 2005. Extended-spectrum beta-lactamase-producing Shiga toxin gene (*Stx1*)-positive *Escherichia coli* O26:H11: a new concern. J. Clin. Microbiol. 43:1072–1075.
29. Cortes P, Blanc V, Mora A, Dahbi G, Blanco JE, Blanco M, Lopez C, Andreu A, Navarro F, Alonso MP, Bou G, Blanco J, Llangostera M. 2010. Isolation and characterization of potentially pathogenic antimicrobial-resistant *Escherichia coli* strains from chicken and pig farms in Spain. Appl. Environ. Microbiol. 76:2799–2805.
30. Pitout JD, Laupland KB, Church DL, Menard ML, Johnson JR. 2005. Virulence factors of *Escherichia coli* isolates that produce CTX-M-type extended-spectrum beta-lactamases. Antimicrob. Agents Chemother. 49:4667–4670.
31. Peirano G, Schreckenberger PC, Pitout JD. 2011. Characteristics of NDM-1-producing *Escherichia coli* isolates that belong to the successful and virulent clone ST131. Antimicrob. Agents Chemother. 55:2986–2988.
32. Dubois D, Prasadaro NV, Mittal R, Bret L, Roujou-Gris M, Bonnet

R. 2009. CTX-M beta-lactamase production and virulence of *Escherichia coli* K1. Emerg. Infect. Dis. 15:1988–1990.

33. Aoki S, Hirakata Y, Kondoh A, Gotoh N, Yanagihara K, Miyazaki Y, Tomono K, Yamada Y, Kohno S, Kamihira S. 2004. Virulence of metallo-beta-lactamase-producing *Pseudomonas aeruginosa* in vitro and in vivo. Antimicrob. Agents Chemother. 48:1876–1878.

34. Fernandez A, Perez A, Ayala JA, Mallo S, Rumbo-Feal S, Tomas M, Poza M, Bou G. 2012. Expression of OXA-type and SFO-1 beta-lactamases induces changes in peptidoglycan composition and affects bacterial fitness. Antimicrob. Agents Chemother. 56:1877–1884.

35. Morosini MI, Ayala JA, Baquero F, Martinez JL, Blazquez J. 2000. Biological cost of AmpC production for *Salmonella enterica* serotype Typhimurium. Antimicrob. Agents Chemother. 44:3137–3143.

36. Sahly H, Navon-Venezia S, Roesler L, Hay A, Carmeli Y, Podschun R, Hennequin C, Forestier C, Ofek I. 2008. Extended-spectrum beta-lactamase production is associated with an increase in cell invasion and expression of fimbrial adhesins in *Klebsiella pneumoniae*. Antimicrob. Agents Chemother. 52:3029–3034.

37. Balasubramanian D, Schneper L, Merighi M, Smith R, Narasimhan G, Lory S, Mathee K. 2012. The regulatory repertoire of *Pseudomonas aeruginosa* AmpC β-lactamase regulator AmpR includes virulence genes PLoS One 7:e34067. doi:[10.1371/journal.pone.0034067](https://doi.org/10.1371/journal.pone.0034067).

38. Kong KF, Jayawardena SR, Indulkar SD, Del Puerto A, Koh CL, Hoiby N, Mathee K. 2005. *Pseudomonas aeruginosa* AmpR is a global transcriptional factor that regulates expression of AmpC and PoxB beta-lactamases, proteases, quorum sensing, and other virulence factors. Antimicrob. Agents Chemother. 49:4567–4575.

39. Hennequin C, Robin F, Cabrolie N, Bonnet R, Forestier C. 2012. Characterization of a DHA-1-producing *Klebsiella pneumoniae* strain involved in an outbreak and role of the AmpR regulator in virulence. Antimicrob. Agents Chemother. 56:288–294.

40. Dijkshoorn L, Nemec A, Seifert H. 2007. An increasing threat in hospitals: multidrug-resistant *Acinetobacter baumannii*. Nat. Rev. Microbiol. 5:939–951.

41. Sechi LA, Karadenizli A, Deriu A, Zanetti S, Kolayli F, Balikci E, Vahaboglu H. 2004. PER-1 type beta-lactamase production in *Acinetobacter baumannii* is related to cell adhesion. Med. Sci. Monit. 10:180–184.

42. Bou G, Oliver A, Martínez-Beltrán J. 2000. OXA-24: a novel class D beta-lactamase with carbapenemase activity in an *Acinetobacter baumannii* clinical strain. Antimicrob. Agents Chemother. 44:1556–1561.

43. Santillana E, Beceiro A, Bou G, Romero A. 2007. Crystal structure of the carbapenemase OXA-24 reveals insights into the mechanism of carbapenem hydrolysis. Proc. Natl. Acad. Sci. U. S. A. 104:5354–5359.

44. Acosta J, Merino M, Viedma E, Poza M, Sanz F, Otero JR, Chaves F, Bou G. 2011. Multidrug-resistant *Acinetobacter baumannii* harboring OXA-24 carbapenemase, Spain. Emerg. Infect. Dis. 17:1064–1067.

45. Rosado CJ, Kondos S, Bull TE, Kuiper MJ, Law RH, Buckle AM, Voskoboinik I, Bird PI, Trapani JA, Whisstock JC, Dunstone MA. 2008. The MACPF/CDC family of pore-forming toxins. Cell. Microbiol. 10:1765–1774.

46. Dorsey CW, Tolmasky ME, Crosa JH, Actis LA. 2003. Genetic organization of an *Acinetobacter baumannii* chromosomal region harbouring genes related to siderophore biosynthesis and transport. Microbiology 149:1227–1238.

47. Nikaido H. 2003. Molecular basis of bacterial outer membrane permeability revisited. Microbiol. Mol. Biol. Rev. 67:593–656.

48. Sugawara E, Nikaido H. 2012. OmpA is the principal nonspecific slow porin of *Acinetobacter baumannii*. J. Bacteriol. 194:4089–4096.

49. Lee JS, Choi CH, Kim JW, Lee JC. 2010. *Acinetobacter baumannii* outer membrane protein A induces dendritic cell death through mitochondrial targeting. J. Microbiol. 48:387–392.

50. Gaddy JA, Tomaras AP, Actis LA. 2009. The *Acinetobacter baumannii* 19606 OmpA protein plays a role in biofilm formation on abiotic surfaces and in the interaction of this pathogen with eukaryotic cells. Infect. Immun. 77:3150–3160.

51. Cabral MP, Soares NC, Aranda J, Parreira JR, Rumbo C, Poza M, Valle J, Calamia V, Lasa I, Bou G. 2011. Proteomic and functional analyses reveal a unique lifestyle for *Acinetobacter baumannii* biofilms and a key role for histidine metabolism. J. Proteome Res. 10:3399–3417.

52. Limansky AS, Mussi MA, Viale AM. 2002. Loss of a 29-kilodalton outer membrane protein in *Acinetobacter baumannii* is associated with imipenem resistance. J. Clin. Microbiol. 40:4776–4778.

53. del Mar Tomás M, Beceiro A, Pérez A, Velasco D, Moure R, Villanueva R, Martínez-Beltrán J, Bou G. 2005. Cloning and functional analysis of the gene encoding the 33- to 36-kilodalton outer membrane protein associated with carbapenem resistance in *Acinetobacter baumannii*. Antimicrob. Agents Chemother. 49:5172–5175.

54. Catel-Ferreira M, Nehme R, Molle V, Aranda J, Bouffartigues E, Chevalier S, Bou G, Jouenne T, De E. 2012. Deciphering the function of the outer membrane protein OprD homologue of *Acinetobacter baumannii*. Antimicrob. Agents Chemother. 56:3826–3832.

55. Fernandez-Cuenca F, Smani Y, Gomez-Sanchez MC, Docobo-Perez F, Caballero-Moyano FJ, Dominguez-Herrera J, Pascual A, Pachón J. 2011. Attenuated virulence of a slow-growing pandrug-resistant *Acinetobacter baumannii* is associated with decreased expression of genes encoding the porins CarO and OprD-like. Int. J. Antimicrob. Agents 38:548–549.

56. Smani Y, McConnell MJ, Pachón J. 2012. Role of fibronectin in the adhesion of *Acinetobacter baumannii* to host cells. PLoS One 7:e33073. doi:[10.1371/journal.pone.0033073](https://doi.org/10.1371/journal.pone.0033073).

57. Liu YF, Yan JJ, Lei HY, Teng CH, Wang MC, Tseng CC, Wu JJ. 2012. Loss of outer membrane protein C in *Escherichia coli* contributes to both antibiotic resistance and escaping antibody-dependent bactericidal activity. Infect. Immun. 80:1815–1822.

58. Rolhion N, Carvalho FA, Darfeuille-Michaud A. 2007. OmpC and the sigmaE regulatory pathway are involved in adhesion and invasion of the Crohn’s disease-associated *Escherichia coli* strain LF82. Mol. Microbiol. 63:1684–1700.

59. Bekhit A, Fukamachi T, Saito H, Kobayashi H. 2011. The role of OmpC and OmpF in acidic resistance in *Escherichia coli*. Biol. Pharm. Bull. 34:330–334.

60. Chart H, Rowe B. 1989. The outer membrane protein of enteropathogenic *Escherichia coli*, described as the ‘localized adherence factor’, is OmpF and probably not involved in adhesion to HEp-2 cells. FEMS Microbiol. Lett. 52:291–295.

61. Duperthuy M, Binesse J, Le Roux F, Romestand B, Caro A, Got Givaudan A, Mazel D, Bachere E, Destoumieux-Garzon D. 2010. The major outer membrane protein OmpU of *Vibrio splendidus* contributes to host antimicrobial peptide resistance and is required for virulence in the oyster Crassostrea gigas. Environ. Microbiol. 12:951–963.

62. Fito-Boncompte L, Chapalain A, Bouffartigues E, Chaker H, Lesouhaitier O, Gicquel G, Bazire A, Madi A, Connil N, Veron W, Taupin L, Toussaint B, Cornelis P, Wei Q, Shioya K, Déziel E, Feuilloley MG, Orange N, Dufour A, Chevalier S. 2011. Full virulence of *Pseudomonas aeruginosa* requires OprF. Infect. Immun. 79:1176–1186.

63. Arhin A, Boucher C. 2010. The outer membrane protein OprQ and adherence of *Pseudomonas aeruginosa* to human fibronectin. Microbiology 156:1415–1423.

64. Tunbridge AJ, Stevanin TM, Lee M, Marriott HM, Moir JW, Read RC, Dockrell DH. 2006. Inhibition of macrophage apoptosis by *Neisseria meningitidis* requires nitric oxide detoxification mechanisms. Infect. Immun. 74:729–733.

65. Coudeyras S, Nakusi L, Charbonnel N, Forestier C. 2008. A tripartite efflux pump involved in gastrointestinal colonization by *Klebsiella pneumoniae* confers a tolerance response to inorganic acid. Infect. Immun. 76:4633–4641.

66. Evans K, Passador L, Srikumar R, Tsang E, Nezezon J, Poole K. 1998. Influence of the MexAB-OprM multidrug efflux system on quorum sensing in *Pseudomonas aeruginosa*. J. Bacteriol. 180:5443–5447.

67. Martinez JL, Sanchez MB, Martinez-Solano L, Hernandez A, Garnodia L, Fajardo A, Alvarez-Ortega C. 2009. Functional role of bacterial multidrug efflux pumps in microbial natural ecosystems. FEMS Microbiol. Rev. 33:430–449.

68. Piddock LJ. 2006. Multidrug-resistance efflux pumps—not just for resistance. Nat. Rev. Microbiol. 4:629–636.

69. Posadas DM, Martin FA, Sabio y Garcia JV, Spera JM, Delpino MV, Baldi P, Campos E, Cravero SL, Zorreguieta A. 2007. The TolC homologue of *Brucella suis* is involved in resistance to antimicrobial compounds and virulence. Infect. Immun. 75:379–389.

70. Lavigne JP, Sotto A, Nicolas-Chanoine MH, Bouziges N, Bourg G, Davin-Regli A, Pages JM. 2012. Membrane permeability, a pivotal function involved in antibiotic resistance and virulence in *Enterobacter aerogenes* clinical isolates. Clin. Microbiol. Infect. 18:539–545.

71. Gil H, Platz GJ, Forestal CA, Monfett M, Bakshi CS, Sellati TJ, Furie MB, Benach JL, Thanassi DG. 2006. Deletion of TolC orthologs in

Beceiro et al.

Francisella tularensis identifies roles in multidrug resistance and virulence. Proc. Natl. Acad. Sci. U. S. A. 103:12897–12902.

72. Hirakata Y, Srikumar R, Poole K, Gotoh N, Suematsu T, Kohno S, Kamihira S, Hancock RE, Speert DP. 2002. Multidrug efflux systems play an important role in the invasiveness of Pseudomonas aeruginosa. J. Exp. Med. 196:109–118.

73. Imuta N, Nishi J, Tokuda K, Fujiyama R, Manago K, Iwashita M, Sarantuya J, Kawano Y. 2008. The Escherichia coli efflux pump TolC promotes aggregation of enteropathogenic E. coli O42. Infect. Immun. 76:1247–1256.

74. Buckley AM, Webber MA, Cooles S, Randall LP, La Ragione RM, Woodward MJ, Piddock LJ. 2006. The AcrAB-TolC efflux system of Salmonella enterica serovar Typhimurium plays a role in pathogenesis. Cell. Microbiol. 8:847–856.

75. Bina XR, Lavine CL, Miller MA, Bina JE. 2008. The AcrAB RND efflux system from the live vaccine strain of Francisella tularensis is a multiple drug efflux system that is required for virulence in mice. FEMS Microbiol. Lett. 279:226–233.

76. Bina XR, Provenzano D, Nguyen N, Bina JE. 2008. Vibrio cholerae RND family efflux systems are required for antimicrobial resistance, optimal virulence factor production, and colonization of the infant mouse small intestine. Infect. Immun. 76:3595–3605.

77. Bailey AM, Ivens A, Kingsley R, Cottell JL, Wain J, Piddock LJ. 2010. RamA, a member of the AraC/XylS family, influences both virulence and efflux in Salmonella enterica serovar Typhimurium. J. Bacteriol. 192:1607–1616.

78. Ferhat M, Atlan D, Vianney A, Lazzaroni JC, Doublet P, Gilbert C. 2009. The TolC protein of Legionella pneumophila plays a major role in multi-drug resistance and the early steps of host invasion. PLoS One 4:e7732. doi:10.1371/journal.pone.0007732.

79. Webber MA, Bailey AM, Blair JM, Morgan E, Stevens MP, Hinton JC, Ivens A, Wain J, Piddock LJ. 2009. The global consequence of disruption of the AcrAB-TolC efflux pump in Salmonella enterica includes reduced expression of SPI-1 and other attributes required to infect the host. J. Bacteriol. 191:4276–4285.

80. Chan YY, Chua KL. 2005. The Burkholderia pseudomallei BpeAB-OprB efflux pump: expression and impact on quorum sensing and virulence. J. Bacteriol. 187:4707–4719.

81. Keeney D, Ruzin A, Bradford PA. 2007. RamA, a transcriptional regulator, and AcrAB, an RND-type efflux pump, are associated with decreased susceptibility to tigecycline in Enterobacter cloacae. Microb. Drug Resist. 13:1–6.

82. Keeney D, Ruzin A, McAleese F, Murphy E, Bradford PA. 2008. MarA-mediated overexpression of the AcrAB efflux pump results in decreased susceptibility to tigecycline in Escherichia coli. J. Antimicrob. Chemother. 61:46–53.

83. Ruzin A, Keeney D, Bradford PA. 2005. AcrAB efflux pump plays a role in decreased susceptibility to tigecycline in Morganella morganii. Antimicrob. Agents Chemother. 49:791–793.

84. Ruzin A, Visalli MA, Keeney D, Bradford PA. 2005. Influence of transcriptional activator RamA on expression of multidrug efflux pump AcrAB and tigecycline susceptibility in Klebsiella pneumoniae. Antimicrob. Agents Chemother. 49:1017–1022.

85. Visalli MA, Murphy E, Projan SJ, Bradford PA. 2003. AcrAB multidrug efflux pump is associated with reduced levels of susceptibility to tigecycline (GAR-936) in Proteus mirabilis. Antimicrob. Agents Chemother. 47:665–669.

86. Al-Karablieh N, Weingart H, Ullrich MS. 2009. The outer membrane protein TolC is required for phytoalexin resistance and virulence of the fire blight pathogen Erwinia amylovora. Microb. Biotechnol. 2:465–475.

87. Brown DG, Swanson JK, Allen C. 2007. Two host-induced Ralstonia solanacearum genes, acrA and dinF, encode multidrug efflux pumps and contribute to bacterial wilt virulence. Appl. Environ. Microbiol. 73:2777–2786.

88. Lister IM, Raftery C, Mecsas J, Levy SB. 2012. Yersinia pestis AcrAB-TolC in antibiotic resistance and virulence. Antimicrob. Agents Chemother. 56:1120–1123.

89. Perez A, Poza M, Fernandez A, Mdel Fernandez CC, Mallo S, Merino M, Rumbo-Feal S, Cabral MP, Bou G. 2012. Involvement of the AcrAB-TolC efflux pump in the resistance, fitness, and virulence of Enterobacter cloacae. Antimicrob. Agents Chemother. 56:2084–2090.

90. Padilla E, Llobet E, Domenech-Sanchez A, Martinez-Martinez L, Ben-Goechea JA, Alberti S. 2010. Klebsiella pneumoniae AcrAB efflux pump contributes to antimicrobial resistance and virulence. Antimicrob. Agents Chemother. 54:177–183.

91. Bialek S, Lavigne JP, Chevalier J, Marcon E, Leflon-Guibout V, Davin A, Moreau R, Pages JM, Nicolas-Chanoine MH. 2010. Membrane efflux and influx modulate both multidrug resistance and virulence of Klebsiella pneumoniae in a Caenorhabditis elegans model. Antimicrob. Agents Chemother. 54:4373–4378.

92. Poole K, Gotoh N, Tsujimoto H, Zhao Q, Wada A, Yamasaki T, Neshat S, Yamagishi J, Li XZ, Nishino T. 1996. Overexpression of the mexC-mexD-oprJ efflux operon in nfxB-type multidrug-resistant strains of Pseudomonas aeruginosa. Mol. Microbiol. 21:713–724.

93. Köhler T, Michéa-Hamzehpour M, Henze U, Gotoh N, Curty LK, Pechère JC. 1997. Characterization of MexE-MexF-OprN, a positively regulated multidrug efflux system of Pseudomonas aeruginosa. Mol. Microbiol. 23:345–354.

94. Sanchez P, Linares JF, Ruiz-Diez B, Campanario E, Navas A, Baquero F, Martinez JL. 2002. Fitness of in vitro selected Pseudomonas aeruginosa nalB and nfxB multidrug resistant mutants. J. Antimicrob. Chemother. 50:657–664.

95. Linares JF, Lopez JA, Camafeita E, Albar JP, Rojo F, Martinez JJ. 2005. Overexpression of the multidrug efflux pumps MexCD-OprJ and MexEF-OprN is associated with a reduction of type III secretion in Pseudomonas aeruginosa. J. Bacteriol. 187:1384–1391.

96. Köhler T, van Delden C, Curty LK, Hamzehpour MM, Pechere JC. 2001. Overexpression of the MexEF-OprN multidrug efflux system affects cell-to-cell signaling in Pseudomonas aeruginosa. J. Bacteriol. 183:5213–5222.

97. Salunkhe P, Smart CH, Morgan JA, Panagea S, Walshaw MJ, Hart CA, Geffers R, Tummler B, Winstanley C. 2005. A cystic fibrosis epidemic strain of Pseudomonas aeruginosa displays enhanced virulence and antimicrobial resistance. J. Bacteriol. 187:4908–4920.

98. Tomas M, Doumith M, Warner M, Turton JF, Beceiro A, Bou G, Livermore DM, Woodford N. 2010. Efflux pumps, OprD porin, AmpC beta-lactamase, and multiresistance in Pseudomonas aeruginosa isolates from cystic fibrosis patients. Antimicrob. Agents Chemother. 54:2219–2224.

99. Maeda T, García-Contreras R, Pu M, Sheng L, Garcia LR, Tomás M, Wood TK. 2012. Quorum quenching quandary: resistance to antivirulence compounds. ISME J. 6:493–501.

100. Burrowes E, Baysse C, Adams C, O’Gara F. 2006. Influence of the regulatory protein RsmA on cellular functions in Pseudomonas aeruginosa PAO1: as revealed by transcriptome analysis. Microbiology 152:405–418.

101. Lamarche MG, Deziel E. 2011. MexEF-OprN efflux pump exports the Pseudomonas quinolone signal (PQS) precursor HHQ (4-hydroxy-2-heptylquinoline). PLoS One 6:e24310. doi:10.1371/journal.pone.0024310.

102. Minagawa S, Inami H, Kato T, Sawada S, Yasuki T, Miyairi S, Horikawa M, Okuda J, Gotoh N. 2012. RND type efflux pump system MexAB-OprM of Pseudomonas aeruginosa selects bacterial languages, 3-oxo-acyl-homoserine lactones, for cell-to-cell communication. BMC Microbiol. 12:70.

103. Quiblier C, Zinkernagel AS, Schuepbach RA, Berger-Bachi B, Senn MM. 2011. Contribution of SecDF to Staphylococcus aureus resistance and expression of virulence factors. BMC Microbiol. 11:72.

104. Gutierrez B, Escudero JA, San Millan A, Hidalgo L, Carrilero L, Ovejero CM, Santos-Lopez A, Thomas-Lopez D, Gonzalez-Zorn B. 2012. Fitness cost and interference of Arm/Rmt aminoglycoside resistance with the RsmF housekeeping methyltransferases. Antimicrob. Agents Chemother. 56:2335–2341.

105. Poole K. 2005. Aminoglycoside resistance in Pseudomonas aeruginosa. Antimicrob. Agents Chemother. 49:479–487.

106. Fournier PE, Vallenet D, Barbe V, Audic S, Ogata H, Poirel L, Richet H, Robert C, Mangenot S, Abergel C, Nordmann P, Weissenbach J, Raoult D, Claverie JM. 2006. Comparative genomics of multidrug resistance in Acinetobacter baumannii. PLoS Genet. 2:e7. doi:10.1371/journal.pgen.0020007.

107. Juan C, Mulet X, Zamorano L, Albertí S, Pérez JL, Oliver A. 2009. Detection of the novel extended-spectrum beta-lactamase OXA-161 from a plasmid-located integron in Pseudomonas aeruginosa clinical isolates from Spain. Antimicrob. Agents Chemother. 53:5288–5290.

108. Pérez-Llarena FJ, Fernández A, Zamorano L, Kerff F, Beceiro A, Aracil B, Cercenado E, Miro E, Oliver A, Oteo J, Navarro F, Bou G. 2012.

Characterizacion of a novel IMP-28 metallo-β-lactamase from a spanish Klebsiella oxytoca clinical isolate. Antimicrob. Agents Chemother. 56: 4540–4543.

109. Perez A, Canle D, Latasa C, Poza M, Beceiro A, Tomas Mdel M, Fernandez A, Mallo S, Perez S, Molina F, Villanueva R, Lasa I, Bou G. 2007. Cloning, nucleotide sequencing, and analysis of the AcrAB-TolC efflux pump of Enterobacter cloacae and determination of its involvement in antibiotic resistance in a clinical isolate. Antimicrob. Agents Chemother. 51:3247–3253.

110. Coyne S, Courvalin P, Perichon B. 2011. Efflux-mediated antibiotic resistance in Acinetobacter spp. Antimicrob. Agents Chemother. 55:947–953.

111. Olkkola S, Juntunen P, Heiska H, Hytyiainen H, Hanninen ML. 2010. Mutations in the *rpsL* gene are involved in streptomycin resistance in Campylobacter coli. Microb. Drug Resist. 16:105–110.

112. Paulander W, Maisnier-Patin S, Andersson DI. 2009. The fitness cost of streptomycin resistance depends on *rpsL* mutation, carbon source and RpoS (sigmaS). Genetics 183:539–546.

113. Christensen EG, Gram L, Kastbjerg VG. 2011. Sublethal triclosan exposure decreases susceptibility to gentamicin and other aminoglycosides in Listeria monocytogenes. Antimicrob. Agents Chemother. 55:4064–4071.

114. Sun YJ, Luo JT, Wong SY, Lee AS. 2010. Analysis of *rpsL* and *rrs* mutations in Beijing and non-Beijing streptomycin-resistant Mycobacterium tuberculosis isolates from Singapore. Clin. Microbiol. Infect. 16: 287–289.

115. Zarazaga M, Tenorio C, Del Campo R, Ruiz-Larrea F, Torres C. 2002. Mutations in ribosomal protein L16 and in 23S rRNA in Enterococcus strains for which evernimicin MICs differ. Antimicrob. Agents Chemother. 46:3657–3659.

116. Aslangul E, Massias L, Meulemans A, Chau F, Andremont A, Courvalin P, Fantin B, Ruimy R. 2006. Acquired gentamicin resistance by permeability impairment in Enterococcus faecalis. Antimicrob. Agents Chemother. 50:3615–3621.

117. Rice LB. 2012. Mechanisms of resistance and clinical relevance of resistance to β-lactams, glycopeptides, and fluoroquinolones. Mayo Clin. Proc. 87:198–208.

118. Pomba C, da Fonseca JD, Baptista BC, Correia JD, Martínez-Martínez L. 2009. Detection of the pandemic O25-ST131 human virulent Escherichia coli CTX-M-15-producing clone harboring the *qnrB2* and *aac(6')-Ib-cr* genes in a dog. Antimicrob. Agents Chemother. 53:327–328.

119. Kugelberg E, Lofmark S, Wretlind B, Andersson DI. 2005. Reduction of the fitness burden of quinolone resistance in Pseudomonas aeruginosa. J. Antimicrob. Chemother. 55:22–30.

120. O’Regan E, Quinn T, Frye JG, Pages JM, Porwollik S, Fedorka-Cray PJ, McClelland M, Fanning S. 2010. Fitness costs and stability of a high-level ciprofloxacin resistance phenotype in Salmonella enterica serotype Enteritidis: reduced infectivity associated with decreased expression of Salmonella pathogenicity island 1 genes. Antimicrob. Agents Chemother. 54:367–374.

121. Smani Y, Lopez-Rojas R, Dominguez-Herrera J, Docobo-Perez F, Marti S, Vila J, Pachon J. 2012. In vitro and in vivo reduced fitness and virulence in ciprofloxacin-resistant Acinetobacter baumannii. Clin. Microbiol. Infect. 18:E1–E4.

122. Rossolini GM, Mantengoli E, Montagnani F, Pollini S. 2010. Epidemiology and clinical relevance of microbial resistance determinants versus anti-Gram-positive agents. Curr. Opin. Microbiol. 13:582–588.

123. Helms M, Simonsen J, Molbak K. 2004. Quinolone resistance is associated with increased risk of invasive illness or death during infection with Salmonella serotype Typhimurium. J. Infect. Dis. 190:1652–1654.

124. McNairn E, Ni Bhriain N, Dorman CJ. 1995. Overexpression of the Shigella flexneri genes coding for DNA topoisomerase IV compensates for loss of DNA topoisomerase I: effect on virulence gene expression. Mol. Microbiol. 15:507–517.

125. Kishii R, Takei M. 2009. Relationship between the expression of *ompF* and quinolone resistance in Escherichia coli. J. Infect. Chemother. 15: 361–366.

126. Tran QT, Nawaz MS, Deck J, Foley S, Nguyen K, Cerniglia CE. 2011. Detection of type III secretion system virulence and mutations in *gyrA* and *parC* genes among quinolone-resistant strains of Pseudomonas aeruginosa isolated from imported shrimp. Foodborne Pathog. Dis. 8:451–453.

127. Wong-Berenger A, Wiener-Kronish J, Lynch S, Flanagan J. 2008.


Comparison of type III secretion system virulence among fluoroquinolone-susceptible and -resistant clinical isolates of Pseudomonas aeruginosa. Clin. Microbiol. Infect. 14:330–336.

128. Agnello M, Wong-Beringer A. 2012. Differentiation in quinolone resistance by virulence genotype in Pseudomonas aeruginosa. PLoS One 7:e42973. doi:10.1371/journal.pone.0042973.

129. Lamichhane-Khadka R, Cantore SA, Riordan JT, Delgado A, Norman AE, Duenas S, Zaman S, Horan S, Wilkinson BJ, Gustafson JE. 2009. sarA inactivation reduces vancomycin-intermediate and ciprofloxacin resistance expression by Staphylococcus aureus. Int. J. Antimicrob. Agents 34:136–141.

130. Cirz RT, Jones MB, Gingles NA, Minogue TD, Jarrahi B, Peterson SN, Romesberg FE. 2007. Complete and SOS-mediated response of Staphylococcus aureus to the antibiotic ciprofloxacin. J. Bacteriol. 189:531–539.

131. Bisognano C, Kelley WL, Estoppey T, Francois P, Schrenzel J, Li D, Lew DP, Hooper DC, Cheung AL, Vaudaux P. 2004. A recA-LexA-dependent pathway mediates ciprofloxacin-induced fibronectin binding in Staphylococcus aureus. J. Biol. Chem. 279:9064–9071.

132. Johnson CN, Briles DE, Benjamin WH, Jr, Hollingshead SK, Waites KB. 2005. Relative fitness of fluoroquinolone-resistant Streptococcus pneumoniae. Emerg. Infect. Dis. 11:814–820.

133. Truong-Bolduc QC, Hooper DC. 2010. Phosphorylation of MgrA and its effect on expression of the NorA and NorB efflux pumps of Staphylococcus aureus. J. Bacteriol. 192:2525–2534.

134. Martin FA, Posadas DM, Carrica MC, Cravero SL, O’Callaghan D, Zorreguieta A. 2009. Interplay between two RND systems mediating antimicrobial resistance in Brucella suis. J. Bacteriol. 191: 2530–2540.

135. Alonso A, Martínez JL. 2000. Cloning and characterization of SmeDEF, a novel multidrug efflux pump from Stenotrophomonas maltophilia. Antimicrob. Agents Chemother. 44:3079–3086.

136. Alonso A, Morales G, Escalante R, Campanario E, Sastre L, Martinez JL. 2004. Overexpression of the multidrug efflux pump SmeDEF impairs Stenotrophomonas maltophilia physiology. J. Antimicrob. Chemother. 53:432–434.

137. Cattoir V, Nordmann P. 2009. Plasmid-mediated quinolone resistance in gram-negative bacterial species: an update. Curr. Med. Chem. 16: 1028–1046.

138. Michon A, Allou N, Chau F, Podglajen I, Fantin B, Cambau E. 2011. Plasmidic *qnrA3* enhances Escherichia coli fitness in absence of antibiotic exposure. PLoS One 6:e24552. doi:10.1371/journal.pone.0024552.

139. Speer BS, Shoemaker NB, Salyers AA. 1992. Bacterial resistance to tetracycline: mechanisms, transfer, and clinical significance. Clin. Microbiol. Rev. 5:387–399.

140. Karami N, Nowrouzian F, Adlerberth I, Wold AE. 2006. Tetracycline resistance in Escherichia coli and persistence in the infantile colonic microbiota. Antimicrob. Agents Chemother. 50:156–161.

141. Beutlich J, Jahn S, Malorny B, Hauser E, Hühn S, Schroeter A, Rodicio MR, Appel B, Threlfall J, Mevius D, Guerra B. 2011. Antimicrobial resistance and virulence determinants in European Salmonella genomic island 1-positive Salmonella enterica isolates from different origins. Appl. Environ. Microbiol. 77:5655–5664.

142. Abril C, Brodard I, Perreten V. 2010. Two novel antibiotic resistance genes, *tet44* and *ant6-Ib*, are located within a transferable pathogenicity island in Campylobacter fetus subsp. *fetus*. Antimicrob. Agents Chemother. 54:3052–3055.

143. Dailidiene D, Bertoli MT, Miciuleviciene J, Mukhopadhyay AK, Dailide G, Pascasio MA, Kupcinskas L, Berg DE. 2002. Emergence of tetracycline resistance in Helicobacter pylori: multiple mutational changes in 16S ribosomal DNA and other genetic loci. Antimicrob. Agents Chemother. 46:3940–3946.

144. Speer BS, Bedzyk L, Salyers AA. 1991. Evidence that a novel tetracycline resistance gene found on two Bacteroides transposons encodes an NADP-requiring oxidoreductase. J. Bacteriol. 173:176–183.

145. Yang W, Moore IF, Koteva KP, Bareich DC, Hughes DW, Wright GD. 2004. TetX is a flavin-dependent monooxygenase conferring resistance to tetracycline antibiotics. J. Biol. Chem. 279:52346–52352.

146. Kresken M, Leitner E, Brauers J, Geiss HK, Halle E, von Eiff C, Peters G, Seifert H. 2009. Susceptibility of common aerobic pathogens to tigecycline: results of a surveillance study in Germany. Eur. J. Clin. Microbiol. Infect. Dis. 28:83–90.

147. Dean CR, Visalli MA, Projan SJ, Sum PE, Bradford PA. 2003. Efflux-mediated resistance to tigecycline (GAR-936) in Pseudomonas aeruginosa PAO1. Antimicrob. Agents Chemother. 47:972–978.

148. Marchand I, Damier-Piolle L, Courvalin P, Lambert T. 2004. Expression of the RND-type efflux pump AdeABC in *Acinetobacter baumannii* is regulated by the AdeRS two-component system. Antimicrob. Agents Chemother. 48:3298–3304.
149. Peleg AY, Potoski BA, Rea R, Adams J, Sethi J, Capitano B, Husain S, Kwak EJ, Bhat SV, Paterson DL. 2007. *Acinetobacter baumannii* bloodstream infection while receiving tigecycline: a cautionary report. J. Antimicrob. Chemother. 59:128–131.
150. Mima T, Kohira N, Li Y, Sekiya H, Ogawa W, Kuroda T, Tsuchiya T. 2009. Gene cloning and characteristics of the RND-type multidrug efflux pump MuxABC-OpmB possessing two RND components in *Pseudomonas aeruginosa*. Microbiology 155:3509–3517.
151. Yang L, Chen L, Shen L, Surette M, Duan K. 2011. Inactivation of MuxABC-OpmB transporter system in *Pseudomonas aeruginosa* leads to increased ampicillin and carbenicillin resistance and decreased virulence. J. Microbiol. 49:107–114.
152. Chan YY, Bian HS, Tan TM, Mattmann ME, Geske GD, Igarashi J, Hatano T, Suga H, Blackwell HE, Chua KL. 2007. Control of quorum sensing by a *Burkholderia pseudomallei* multidrug efflux pump. J. Bacteriol. 189:4320–4324.
153. Mima T, Schweizer HP. 2010. The BpeAB-OprB efflux pump of *Burkholderia pseudomallei* 1026b does not play a role in quorum sensing, virulence factor production, or extrusion of aminoglycosides but is a broad-spectrum drug efflux system. Antimicrob. Agents Chemother. 54: 3113–3120.
154. Warner DM, Shafer WM, Jerse AE. 2008. Clinically relevant mutations that cause derepression of the *Neisseria gonorrhoeae* MtrC-MtrD-MtrE Efflux pump system confer different levels of antimicrobial resistance and in vivo fitness. Mol. Microbiol. 70:462–478.
155. Zou LK, Wang HN, Zeng B, Li JN, Li XT, Zhang AY, Zhou YS, Yang X, Xu CW, Xia QQ. 2011. Erythromycin resistance and virulence genes in *Enterococcus faecalis* from swine in China. New Microbiol. 34:73–80.
156. Hao H, Dai M, Wang Y, Peng D, Liu Z, Yuan Z. 2009. 23S rRNA mutation A2074C conferring high-level macrolide resistance and fitness cost in *Campylobacter jejuni*. Microb. Drug Resist. 15:239–244.
157. Han F, Pu S, Wang F, Meng J, Ge B. 2009. Fitness cost of macrolide resistance in *Campylobacter jejuni*. Int. J. Antimicrob. Agents 34:462–466.
158. Zeitouni S, Collin O, Andraud M, Ermel G, Kempf I. 2012. Fitness of macrolide resistant *Campylobacter coli* and *Campylobacter jejuni*. Microb. Drug Resist. 18:101–108.
159. Wolter N, Smith AM, Farrell DJ, Schaffner W, Moore M, Whitney CG, Jorgensen JH, Klugman KP. 2005. Novel mechanism of resistance to oxazolidinones, macrolides, and chloramphenicol in ribosomal protein L4 of the pneumococcus. Antimicrob. Agents Chemother. 49:3554–3557.
160. Gillespie SH. 2001. Antibiotic resistance in the absence of selective pressure. Int. J. Antimicrob. Agents 17:171–176.
161. Maiques E, Ubeda C, Campoy S, Salvador N, Lasa I, Novick RP, Barbe J, Penades JR. 2006. Beta-lactam antibiotics induce the SOS response and horizontal transfer of virulence factors in *Staphylococcus aureus*. J. Bacteriol. 188:2726–2729.
162. Peleg AY, Monga D, Pillai S, Mylonakis E, Moellering RC, Eliopoulos GM. 2009. Reduced susceptibility to vancomycin influences pathogenicity in *Staphylococcus aureus* infection. J. Infect. Dis. 199:532–536.
163. McCallum N, Karauzum H, Getzmann R, Bischoff M, Majcherczyk P, Berger-Bachi B, Landmann R. 2006. In vivo survival of teicoplanin-resistant *Staphylococcus aureus* and fitness cost of teicoplanin resistance. Antimicrob. Agents Chemother. 50:2352–2360.
164. Renzoni A, Huggler E, Kelley WL, Lew D, Vaudaux P. 2009. Increased uptake and improved intracellular survival of a teicoplanin-resistant mutant of methicillin-resistant *Staphylococcus aureus* in non-professional phagocytes. Chemotherapy 55:183–188.
165. Plantefeve G, Dupont H, Hubert V, Garry L, Pous C, Carbon C, Montravers P. 2003. Impact of elements containing glycopeptide resistance genes on expression of virulence in *Enterococcus faecalis* peritonitis: a pilot study with rats. Antimicrob. Agents Chemother. 47:1560–1564.
166. Foucault ML, Depardieu F, Courvalin P, Grillot-Courvalin C. 2010. Inducible expression eliminates the fitness cost of vancomycin resistance in enterococci. Proc. Natl. Acad. Sci. U. S. A. 107:16964–16969.
167. Foucault ML, Courvalin P, Grillot-Courvalin C. 2009. Fitness cost of VanA-type vancomycin resistance in methicillin-resistant *Staphylococcus aureus*. Antimicrob. Agents Chemother. 53:2354–2359.
168. Long KS, Poehlsgaard J, Kehrenberg C, Schwarz S, Vester B. 2006. The Cfr rRNA methyltransferase confers resistance to phenicols, lincosamides, oxazolidinones, pleuromutilins, and streptogramin A antibiotics. Antimicrob. Agents Chemother. 50:2500–2505.
169. Besier S, Ludwig A, Zander J, Brade V, Wichelhaus TA. 2008. Linezolid resistance in *Staphylococcus aureus*: gene dosage effect, stability, fitness costs, and cross-resistances. Antimicrob. Agents Chemother. 52:1570–1572.
170. Farrell DJ, Mendes RE, Ross JE, Jones RN. 2009. Linezolid surveillance program results for 2008 LEADER (Program for 2008). Diagn. Microbiol. Infect. Dis. 65:392–403.
171. Billal DS, Feng J, Leprohon P, Legare D, Ouellette M. 2011. Whole genome analysis of linezolid resistance in *Streptococcus pneumoniae* reveals resistance and compensatory mutations. BMC Genomics 12:512.
172. Kehrenberg C, Schwarz S. 2006. Distribution of florfenicol resistance genes *fexA* and *cfr* among chloramphenicol-resistant *Staphylococcus* isolates. Antimicrob. Agents Chemother. 50:1156–1163.
173. LaMarre JM, Locke JB, Shaw KJ, Mankin AS. 2011. Low fitness cost of the multidrug resistance gene *cfr*. Antimicrob. Agents Chemother. 55: 3714–3719.
174. Locke JB, Rahawi S, Lamarre J, Mankin AS, Shaw KJ. 2012. Genetic environment and stability of *cfr* in methicillin-resistant *Staphylococcus aureus* CM05. Antimicrob. Agents Chemother. 56:332–340.
175. Gunn JS, Ryan SS, Van Velkinburgh JC, Ernst RKSI, and Miller. 2000. Genetic and functional analysis of a PmrA-PmrB-regulated locus necessary for lipopolysaccharide modification, antimicrobial peptide resistance, and oral virulence of *Salmonella enterica* serovar Typhimurium. Infect. Immun. 68:6139–6146.
176. Sun S, Negrea A, Rhen M, Andersson DI. 2009. Genetic analysis of colistin resistance in *Salmonella enterica* serovar Typhimurium. Antimicrob. Agents Chemother. 53:2298–2305.
177. Tamayo R, Ryan SS, McCoy AJ, Gunn JS. 2002. Identification and genetic characterization of PmrA-regulated genes and genes involved in polymyxin B resistance in *Salmonella enterica* serovar Typhimurium. Infect. Immun. 70:6770–6778.
178. Marceau M, Sebbane F, Ewann F, Collyn F, Lindner B, Campos MA, Bengoechea JA, Simonet M. 2004. The *pmrF* polymyxin-resistance operon of *Yersinia pseudotuberculosis* is upregulated by the PhoP-PhoQ two-component system but not by PmrA-PmrB, and is not required for virulence. Microbiology 150:3947–3957.
179. Prost LR, Miller SI. 2008. The salmonellae PhoQ sensor: mechanisms of detection of phagosomal signals. Cell. Microbiol. 10:576–582.
180. Groisman EA. 2001. The pleiotropic two-component regulatory system PhoP-PhoQ. J. Bacteriol. 183:1835–1842.
181. Shi Y, Cromie MJ, Hsu FF, Turk J, Groisman EA. 2004. PhoP-regulated *Salmonella* resistance to the antimicrobial peptides magainin 2 and polymyxin B. Mol. Microbiol. 53:229–241.
182. Shi Y, Latifi T, Cromie MJ, Groisman EA. 2004. Transcriptional control of the antimicrobial peptide resistance *ugtL* gene by the *Salmonella* PhoP and SlyA regulatory proteins. J. Biol. Chem. 279:38618–38625.
183. Kawasaki K, Ernst RK, Miller SI. 2004. 3-O-deacylation of lipid A by PagL, a PhoP/PhoQ-regulated deacylase of *Salmonella typhimurium*, modulates signaling through Toll-like receptor 4. J. Biol. Chem. 279: 20044–20048.
184. Guo L, Lim KB, Poduje CM, Daniel M, Gunn JS, Hackett M, Miller SI. 1998. Lipid A acylation and bacterial resistance against vertebrate antimicrobial peptides. Cell 95:189–198.
185. Kox LF, MM Wösten Groisman EA. 2000. A small protein that mediates the activation of a two-component system by another two-component system. EMBO J. 19:1861–1872.
186. Beceiro A, Llobet E, Aranda J, Bengoechea JA, Doumith M, Hornsey M, Dhanji H, Chart H, Bou G, Livermore DM, Woodford N. 2011. Phosphoethanolamine modification of lipid A in colistin-resistant variants of *Acinetobacter baumannii* mediated by the *pmrAB* two-component regulatory system. Antimicrob. Agents Chemother. 55:3370–3379.
187. Fernández-Reyes M, Rodríguez-Falcón M, Chiva C, Pachón J, Andreu D, Rivas L. 2009. The cost of resistance to colistin in *Acinetobacter baumannii*: a proteomic perspective. Proteomics 9:1632–1645.
188. López-Rojas R, Domínguez-Herrera J, McConnell MJ, Docobo-Peréz F, Smani Y, Fernández-Reyes M, Rivas L, Pachón J. 2011. Impaired virulence and *in vivo* fitness of colistin-resistant *Acinetobacter baumannii*. J. Infect. Dis. 203:545–548.
189. Valencia R, Arroyo LA, Conde M, Aldana JM, Torres MJ, Fernández-

Cuenca F, Garnacho-Montero J, Cisneros JM, Ortíz C, Pachón J, Aznar J. 2009. Nosocomial outbreak of infection with pan-drug-resistant *Acinetobacter baumannii* in a tertiary care university hospital. Infect. Control Hosp. Epidemiol. 30:257–263.

190. Moffatt JH, Harper M, Harrison P, Hale JD, Vinogradov E, Seemann T, Henry R, Crane B, St Michael F, Cox AD, Adler B, Nation RL, Li J, Boyce JD. 2010. Colistin resistance in *Acinetobacter baumannii* is mediated by complete loss of lipopolysaccharide production. Antimicrob. Agents Chemother. 54:4971–4977.

191. Henry R, Vithanage N, Harrison P, Seemann T, Coutts S, Moffatt JH, Nation RL, Li J, Harper M, Adler B, Boyce JD. 2012. Colistin-resistant, lipopolysaccharide-deficient *Acinetobacter baumannii* responds to lipo-polysaccharide loss through increased expression of genes involved in the synthesis and transport of lipoproteins, phospholipids, and poly-β-1:6-N-acetylglucosamine. Antimicrob. Agents Chemother. 56:59–69.

192. Adams MD, Nickel GC, Bajaksouzian S, Lavender H, Murthy AR, Jacobs MR, Bonomo RA. 2009. Resistance to colistin in *Acinetobacter baumannii* associated with mutations in the PmrAB two-component system. Antimicrob. Agents Chemother. 53:3628–3634.

193. López-Rojas R, Jiménez-Mejías ME, Lepe JAJ, Pachón. 2011. *Acinetobacter baumannii* resistant to colistin alters its antibiotic resistance profile: a case report from Spain. J. Infect. Dis. 204:1147–1148.

194. Rolain JM, Roch A, Castanier M, Papazian LD, Raoult. 2011. *Acinetobacter baumannii* resistant to colistin with impaired virulence: a case report from France. J. Infect. Dis. 204:1146–1147.

195. Overhage J, Bains M, Brazas MD, Hancock RE. 2008. Swarming of *Pseudomonas aeruginosa* is a complex adaptation leading to increased production of virulence factors and antibiotic resistance. J. Bacteriol. 190:2671–2679.

196. Gooderham WJ, Bains M, McPhee JB, Wiegand I, Hancock RE. 2008. Induction by cationic antimicrobial peptides and involvement in intrinsic polymyxin and antimicrobial peptide resistance, biofilm formation, and swarming motility of PsrA in *Pseudomonas aeruginosa*. J. Bacteriol. 190:5624–5634.

197. Gooderham WJ, Gellatly SL, Sanschagrin F, McPhee JB, Bains M, Cosseau C, Levesque RC, Hancock RE. 2009. The sensor kinase PhoQ mediates virulence in *Pseudomonas aeruginosa*. Microbiology 155:699–711.

198. Eswarappa SM, Panguluri KK, Hensel M, Chakravortty D. 2008. The *yejABEF* operon of *Salmonella* confers resistance to antimicrobial peptides and contributes to its virulence. Microbiology 154:666–678.

199. Manning AJ, Kuehn MJ. 2011. Contribution of bacterial outer membrane vesicles to innate bacterial defense. BMC Microbiol. 11:258.

200. Llobet E, Tomás JM, Bengoechea JA. 2008. Capsule polysaccharide is a bacterial decoy for antimicrobial peptides. Microbiology 154:3877–3886.

201. Cortés G, Borrell N, de Astorza B, Gómez C, Sauleda J, Albertí S. 2002. Molecular analysis of the contribution of the capsular polysaccharide and the lipopolysaccharide O side chain to the virulence of *Klebsiella pneumoniae* in a murine model of pneumonia. Infect. Immun. 70:2583–2590.

202. Willems RJ, Hanage WP, Bessen DEEJ, and Feil. 2011. Population biology of Gram-positive pathogens: high-risk clones for dissemination of antibiotic resistance. FEMS Microbiol. Rev. 35:872–900.

203. Woodford N, Turton JF, Livermore DM. 2011. Multiresistant Gram-negative bacteria: the role of high-risk clones in the dissemination of antibiotic resistance. FEMS Microbiol. Rev. 35:736–755.

204. Kumar R, Yadav BR, Anand SK, Singh RS. 2011. Molecular surveillance of putative virulence factors and antibiotic resistance in *Staphylococcus aureus* isolates recovered from intra-mammary infections of river buffaloes. Microb. Pathog. 51:31–38.

205. Lozano C, Rezusta A, Gomez P, Gomez-Sanz E, Baez N, Martin-Saco G, Zarazaga M, Torres C. 2012. High prevalence of spa types associated with the clonal lineage CC398 among tetracycline-resistant methicillin-resistant *Staphylococcus aureus* strains in a Spanish hospital. J. Antimicrob. Chemother. 67:330–334.

206. Galloway-Peña JR, Nallapareddy SR, Arias CA, Eliopoulos GM, Murray BE. 2009. Analysis of clonality and antibiotic resistance among early clinical isolates of *Enterococcus faecium* in the United States. J. Infect. Dis. 200:1566–1573.

207. Leavis HL, Willems RJ, Top J, Bonten MJ. 2006. High-level ciprofloxacin resistance from point mutations in *gyrA* and *parC* confined to global hospital-adapted clonal lineage CC17 of *Enterococcus faecium*. J. Clin. Microbiol. 44:1059–1064.

208. Billström H, Lund B, Sullivan A, Nord CE. 2008. Virulence and anti-

microbial resistance in clinical *Enterococcus faecium*. Int. J. Antimicrob. Agents 32:374–377.

209. Throup JP, Koretke KK, Bryant AP, Ingraham KA, Chalker AF, Ge Y, Marra A, Wallis NG, Brown JR, Holmes DJ, Rosenberg M, Burnham MK. 2000. A genomic analysis of two-component signal transduction in *Streptococcus pneumoniae*. Mol. Microbiol. 35:566–576.

210. Hiller NL, Eutsey RA, Powell E, Earl JP, Janto B, Martin DP, Dawid S, Ahmed A, Longwell MJ, Dahlgren ME, Ezzo S, Tettelin H, Daugherty SC, Mitchell TJ, Hillman TA, Buchinsky FJ, Tomasz A, de Lencastre H, Sá-Leão R, Post JC, Hu FZ, Ehrlich GD. 2011. Differences in genotype and virulence among four multidrug-resistant *Streptococcus pneumoniae* isolates belonging to the PMEN1 clone. PLoS One 6:e28850. doi:[10.1371/journal.pone.0028850](https://doi.org/10.1371/journal.pone.0028850).

211. Johnson JR, Johnston B, Clabots C, Kuskowski MA, Castanheira M. 2010. *Escherichia coli* sequence type ST131 as the major cause of serious multidrug-resistant *E. coli* infections in the United States. Clin. Infect. Dis. 51:286–294.

212. Blanco J, Mora A, Mamani R, Lopez C, Blanco M, Dahbi G, Herrera A, Blanco JE, Alonso MP, Garcia-Garrote F, Chaves F, Orellana MA, Martínez-Martínez L, Calvo J, Prats G, Larrosa MN, González-López JJ, López-Cerero L, Rodríguez-Baño J, Pascual A. 2011. National survey of *Escherichia coli* causing extraintestinal infections reveals the spread of drug-resistant clonal groups O25b:H4-B2-ST131: O15:H1-D-ST393 and CGA-D-ST69 with high virulence gene content in Spain. J. Antimicrob. Chemother. 66:2011–2021.

213. Mora A, Blanco M, Lopez C, Mamani R, Blanco JE, Alonso MP, Garcia-Garrote F, Dahbi G, Herrera A, Fernandez A, Fernández B, Agulla A, Bou G, Blanco J. 2011. Emergence of clonal groups O1:HNM-D-ST59: O15:H1-D-ST393: O20:H34/HNM-D-ST354: O25b:H4-B2-ST131 and ONT:H21:42-B1-ST101 among CTX-M-14-producing *Escherichia coli* clinical isolates in Galicia, northwest Spain. Int. J. Antimicrob. Agents 37:16–21.

214. Rogers BA, Sidjabat HE, Paterson DL. 2011. *Escherichia coli* O25b-ST131: a pandemic, multiresistant, community-associated strain. J. Antimicrob. Chemother. 66:1–14.

215. Pitout JD, Gregson DBL, Campbell Laupland KB. 2009. Molecular characteristics of extended-spectrum-beta-lactamase-producing *Escherichia coli* isolates causing bacteremia in the Calgary Health Region from 2000 to 2007: emergence of clone ST131 as a cause of community-acquired infections. Antimicrob. Agents Chemother. 53:2846–2851.

216. Suzuki S, Shibata N, Yamane K, Wachino J, Ito K, Arakawa Y. 2009. Change in the prevalence of extended-spectrum-beta-lactamase-producing *Escherichia coli* in Japan by clonal spread. J. Antimicrob. Chemother. 63:72–79.

217. Deneve C, Bouttier SB, Dupuy F, Barbut A, Collignon Janoir C. 2009. Effects of subinhibitory concentrations of antibiotics on colonization factor expression by moxifloxacin-susceptible and moxifloxacin-resistant *Clostridium difficile* strains. Antimicrob. Agents Chemother. 53:5155–5162.

218. Shankar N, Baghdayan AS, Gilmore MS. 2002. Modulation of virulence within a pathogenicity island in vancomycin-resistant *Enterococcus faecalis*. Nature 417:746–750.

219. Watkins RR, David MZ, Salata RA. 2012. Current concepts on the virulence mechanisms of meticillin-resistant *Staphylococcus aureus*. J. Med. Microbiol. 61:1179–1193.

220. Lin YE, Barker Kislow J, Kaldhone P, Stemper ME, Pantrangi M, Moore FM, Hall M, Fritsche TR, Novicki T, Foley SL, Shukla SK. 2011. Evidence of multiple virulence subtypes in nosocomial and community-associated MRSA genotypes in companion animals from the upper midwestern and northeastern United States. Clin. Med. Res. 9:7–16.

221. Cabot G, Ocampo-Sosa AA, Domínguez MA, Gago JF, Juan C, Tubau F, Rodríguez C, Moyà B, Peña C, Martínez-Martínez L, Oliver A. 2012. Genetic markers of widespread extensively drug-resistant XDR *Pseudomonas aeruginosa* high-risk clones. Antimicrob. Agents Chemother. 56:6349–6357.

222. Burrus V, Pavlovic G, Decaris B, Guédon G. 2002. Conjugative transposons: the tip of the iceberg. Mol. Microbiol. 46:601–610.

223. Aminov RI. 2011. Horizontal gene exchange in environmental microbiota. Front. Microbiol. 2:158.

224. Wozniak RA, Waldor MK. 2010. Integrative and conjugative elements: mosaic mobile genetic elements enabling dynamic lateral gene flow. Nat. Rev. Microbiol. 8:552–563.

225. Actis LA, Tolmasky ME, Crosa JH. 1999. Bacterial plasmids: replication

of extrachromosomal genetic elements encoding resistance to antimicrobial compounds. Front. Biosci. 4:43–62.

226. Johnson TJ, Nolan LK. 2009. Pathogenomics of the virulence plasmids of *Escherichia coli*. Microbiol. Mol. Biol. Rev. 73:750–774.

227. Datta N, Hughes VM. 1983. Plasmids of the same Inc groups in Enterobacteria before and after the medical use of antibiotics. Nature 306: 616–617.

228. Hughes VM, Datta N. 1983. Conjugative plasmids in bacteria of the ‘pre-antibiotic’ era. Nature 302:725–726.

229. Woodford NA, Carattoli Karisik E, Underwood A, Ellington MJ, Livermore DM. 2009. Complete nucleotide sequences of plasmids pEK204, pEK499, and pEK516, encoding CTX-M enzymes in three major *Escherichia coli* lineages from the United Kingdom, all belonging to the international O25:H4-ST131 clone. Antimicrob. Agents Chemother. 53:4472–4482.

230. Karisik E, Ellington MJ, Pike R, Warren RE, Livermore DM, Woodford N. 2006. Molecular characterization of plasmids encoding CTX-M-15 beta-lactamases from *Escherichia coli* strains in the United Kingdom. J. Antimicrob. Chemother. 58:665–668.

231. Lavigne JP, Vergunst AC, Goret I, Sotto A, Combescure C, Blanco J, O'Callaghan D, Nicolas-Chanoine MH. 2012. Virulence potential and genomic mapping of the worldwide clone *Escherichia coli* ST131. PLoS One 7:e34294. doi: [10.1371/journal.pone.0034294](https://doi.org/10.1371/journal.pone.0034294).

232. Jadhav S, Hussain A, Devi S, Kumar A, Parveen S, Gandham N, Wieler LH, Ewers C, Ahmed N. 2011. Virulence characteristics and genetic affinities of multiple drug resistant uropathogenic *Escherichia coli* from a semi-urban locality in India. PLoS One 6:e18063. doi: [10.1371/journal.pone.0018063](https://doi.org/10.1371/journal.pone.0018063).

233. Karisik E, Ellington MJ, Livermore DM, Woodford N. 2008. Virulence factors in *Escherichia coli* with CTX-M-15 and other extended-spectrum beta-lactamases in the UK. J. Antimicrob. Chemother. 61:54–58.

234. Fekete PZ, Brzuszkiewicz E, Blum-Oehler G, Olasz F, Szabó M, Gottschalk G, Hacker J, Nagy B. 2012. DNA sequence analysis of the composite plasmid pTC conferring virulence and antimicrobial resistance for porcine enterotoxigenic *Escherichia coli*. Int. J. Med. Microbiol. 302:4–9.

235. Sloan J, McMurry LM, Lyras D, Levy SB, Rood JI. 1994. The *Clostridium perfringens* Tet P determinant comprises two overlapping genes: *tetAP*, which mediates active tetracycline efflux, and *tetBP*, which is related to the ribosomal protection family of tetracycline-resistance determinants. Mol. Microbiol. 11:403–415.

236. Bannam TL, Teng WL, Bulach D, Lyras D, Rood JI. 2006. Functional identification of conjugation and replication regions of the tetracycline resistance plasmid pCW3 from *Clostridium perfringens*. J. Bacteriol. 188: 4942–4951.

237. Szczepanowski R, Braun S, Riedel V, Schneiker S, Krahn I, Pühler A, Schlüter A. 2005. The 120 592 bp IncF plasmid pRSB107 isolated from a sewage-treatment plant encodes nine different antibiotic-resistance determinants, two iron-acquisition systems and other putative virulence-associated functions. Microbiology 151:1095–1111.

238. Olsen JE, Brown DJ, Thomsen LE, Platt DJ, Chadfield MS. 2004. Differences in the carriage and the ability to utilize the serotype associated virulence plasmid in strains of *Salmonella enterica* serotype Typhimurium investigated by use of a self-transferable virulence plasmid, pOG669. Microb. Pathog. 36:337–347.

239. Rychlik I, Gregorova D, Hradecka H. 2006. Distribution and function of plasmids in *Salmonella enterica*. Vet. Microbiol. 112:1–10.

240. Bouwman CW, Kohli M, Killoran A, Touchie GA, Kadner RJ, Martin NL. 2003. Characterization of SrgA, a *Salmonella enterica* serovar Typhimurium virulence plasmid-encoded paralogue of the disulfide oxidoreductase DsbA, essential for biogenesis of plasmid-encoded fimbriae. J. Bacteriol. 185:991–1000.

241. Chu C, Chiu CH. 2006. Evolution of the virulence plasmids of non-typhoid *Salmonella* and its association with antimicrobial resistance. Microbes Infect. 8:1931–1936.

242. Mendoza MC, Herrero A, Rodicio MR. 2009. Evolutionary engineering in *Salmonella*: emergence of hybrid virulence-resistance plasmids in non-typhoid serotypes. Enferm. Infecc. Microbiol. Clin. 27:37–43.

243. Guerra B, Soto S, Helmuth R, Mendoza MC. 2002. Characterization of a self-transferable plasmid from *Salmonella enterica* serotype Typhimurium clinical isolates carrying two integron-borne gene cassettes together with virulence and drug resistance genes. Antimicrob. Agents Chemother. 46:2977–2981.

244. Herrero A, Mendoza MC, Threlfall EJ, Rodicio MR. 2009. Detection of *Salmonella enterica* serovar Typhimurium with pUO-StVR2-like virulence-resistance hybrid plasmids in the United Kingdom. Eur. J. Clin. Microbiol. Infect. Dis. 28:1087–1093.

245. Sukupolvi S, O'Connor CD. 1990. TraT lipoprotein, a plasmid-specified mediator of interactions between gram-negative bacteria and their environment. Microbiol. Rev. 54:331–341.

246. Toleman MA, Walsh TR. 2011. Combinatorial events of insertion sequences and ICE in Gram-negative bacteria. FEMS Microbiol. Rev. 35: 912–935.

247. Rowe-Magnus AD, Davies J, Mazel D. 2002. Impact of integrons and transposons on the evolution of resistance and virulence. Curr. Top. Microbiol. Immunol. 264:167–188.

248. Bordeleau E, Brouillette E, Robichaud N, Burrus V. 2010. Beyond antibiotic resistance: integrating conjugative elements of the SXT/R391 family that encode novel diguanylate cyclases participate to c-di-GMP signalling in *Vibrio cholerae*. Environ. Microbiol. 12:510–523.

249. Kiiru JN, Saidi SM, Goddeeris BM, Wamae NC, Butaye P, Kariuki SM. 2009. Molecular characterisation of *Vibrio cholerae* O1 strains carrying an SXT/R391-like element from cholera outbreaks in Kenya: 1994–2007. BMC Microbiol. 9:275.

250. Osorio CR, Marrero J, Wozniak RA, Lemos ML, Burrus V, Waldor MK. 2008. Genomic and functional analysis of ICEPdaSpal: a fish-pathogen-derived SXT-related integrating conjugative element that can mobilize a virulence plasmid. J. Bacteriol. 190:3353–3361.

251. Golding GR, Olson AB, Doublet B, Cloeckaert A, Christianson S, Graham MR, Mulvey MR. 2007. The effect of the *Salmonella* genomic island 1 on in vitro global gene expression in *Salmonella enterica* serovar Typhimurium LT2. Microbes Infect. 9:21–27.

252. Mulvey MR, Boyd DA, Olson AB, Doublet B, Cloeckaert A. 2006. The genetics of *Salmonella* genomic island 1. Microbes Infect. 8:1915–1922.

253. Feng X, Oropeza R, Kenney LJ. 2003. Dual regulation by phospho-OmpR of *ssrA/B* gene expression in *Salmonella* pathogenicity island 2. Mol. Microbiol. 48:1131–1143.

254. Lee AK, Detweiler CS, Falkow S. 2000. OmpR regulates the two-component system SsrA-SsrB in *Salmonella* pathogenicity island 2. J. Bacteriol. 182:771–781.

255. Holden MT, Hauser H, Sanders M, Ngo TH, Cherevach I, Cronin A, Goodhead I, Mungall K, Quail MA, Price C, Rabbinowitsch E, Sharp S, Croucher NJ, Chieu TB, Mai NT, Diep TS, Chinh NT, Kehoe M, Leigh JA, Ward PN, Dowson CG, Whatmore AM, Chanter N, Iversen P, Gottschalk M, Slater JD, Smith HE, Spratt BG, Xu J, Ye C, Bentley S, Barrell BG, Schultsz C, Maskell DJ, Parkhill J. 2009. Rapid evolution of virulence and drug resistance in the emerging zoonotic pathogen *Streptococcus suis*. PLoS One 4:e6072. doi: [10.1371/journal.pone.0006072](https://doi.org/10.1371/journal.pone.0006072).

256. Glaser P, Rusniok C, Buchrieser C, Chevalier F, Frangeul L, Msadek T, Zouine M, Couvé E, Lalioui L, Poyart C, Trieu-Cuot P, Kunst F. 2002. Genome sequence of *Streptococcus agalactiae*, a pathogen causing invasive neonatal disease. Mol. Microbiol. 45:1499–1513.

257. Smith MG, Gianoulis TA, Pukatzki S, Mekalanos JJ, Ornston LN, Gerstein M, Snyder M. 2007. New insights into *Acinetobacter baumannii* pathogenesis revealed by high-density pyrosequencing and transposon mutagenesis. Genes Dev. 21:601–614.

258. Soares NC, Cabral MP, Parreira JR, Gayoso C, Barba MJ, Bou G. 2009. 2-DE analysis indicates that *Acinetobacter baumannii* displays a robust and versatile metabolism. Proteome Sci. 7:37.

259. Molin S, Tolker-Nielsen T. 2003. Gene transfer occurs with enhanced efficiency in biofilms and induces enhanced stabilisation of the biofilm structure. Curr. Opin. Biotechnol. 14:255–261.

260. Hayes CS, Aoki SK, Low DA. 2010. Bacterial contact-dependent delivery systems. Annu. Rev. Genet. 44:71–90.

261. Bahar O, Goffer T, Burdman S. 2009. Type IV pili are required for virulence, twitching motility, and biofilm formation of *acidovorax avenae* subsp. *Citrulli*. Mol. Plant Microbe Interact. 22:909–920.

262. Varga JJ, Therit B, Melville SB. 2008. Type IV pili and the CcpA protein are needed for maximal biofilm formation by the gram-positive anaerobic pathogen *Clostridium perfringens*. Infect. Immun. 76:4944–4951.

263. Smith HW, Green P, Parsell Z. 1983. Vero cell toxins in *Escherichia coli* and related bacteria: transfer by phage and conjugation and toxic action in laboratory animals, chickens and pigs. J. Gen. Microbiol. 129:3121–3137.

264. O'Brien AD, Newland JW, Miller SF, Holmes RK, Smith HW, Formal SB. 1984. Shiga-like toxin-converting phages from *Escherichia coli* strains

that cause hemorrhagic colitis or infantile diarrhea. Science 226:694–696.

265. Janka A, Bielaszewska M, Dobrindt U, Greune L, Schmidt MA, Karch H. 2003. Cytotoxle distending toxin gene cluster in enterohemorrhagic *Escherichia coli* O157:H− and O157:H7: characterization and evolutionary considerations. Infect. Immun. 71:3634–3638.

266. Blanco J. 2012. Stx2a-producing enteroaggregative *Escherichia coli* O104:H4-ST678. Microbiological diagnostic already, for this and other STEC/VTEC serotypes! Enferm. Infecc. Microbiol. Clin. 30:84–89.

267. Beutin L, Martin A. 2012. Outbreak of Shiga toxin-producing *Escherichia coli* STEC O104:H4 infection in Germany causes a paradigm shift with regard to human pathogenicity of STEC strains. J. Food Prot. 75: 408–418.

268. Colomer-Lluch M, Imamovic L, Jofre J, Muniesa M. 2011. Bacteriophages carrying antibiotic resistance genes in fecal waste from cattle, pigs, and poultry. Antimicrob. Agents Chemother. 55:4908–4911.

269. McBroom AJ, Johnson AP, Vemulapalli S, Kuehn MJ. 2006. Outer membrane vesicle production by *Escherichia coli* is independent of membrane instability. J. Bacteriol. 188:5385–5392.

270. Kuehn MJ, Kesty NC. 2005. Bacterial outer membrane vesicles and the host-pathogen interaction. Genes Dev. 19:2645–2655.

271. Ellis TN, Kuehn MJ. 2010. Virulence and immunomodulatory roles of bacterial outer membrane vesicles. Microbiol. Mol. Biol. Rev. 74:81–94.

272. Kesty NC, Mason KM, Reedy M, Miller SE, Kuehn MJ. 2004. Enterotoxigenic *Escherichia coli* vesicles target toxin delivery into mammalian cells. EMBO J. 23:4538–4549.

273. Gurung M, Moon DC, Choi CW, Lee JH, Bae YC, Kim J, Lee YC, Seol SY, Cho DT, Kim SI, Lee JC. 2011. *Staphylococcus aureus* produces membrane-derived vesicles that induce host cell death. PLoS One 6:e27958. doi:[10.1371/journal.pone.0027958](https://doi.org/10.1371/journal.pone.0027958).

274. Kadurugamuwa JL, Beveridge TJ. 1995. Virulence factors are released from *Pseudomonas aeruginosa* in association with membrane vesicles during normal growth and exposure to gentamicin: a novel mechanism of enzyme secretion. J. Bacteriol. 177:3998–4008.

275. Mashburn LM, Whiteley M. 2005. Membrane vesicles traffic signals and facilitate group activities in a prokaryote. Nature 437:422–425.

276. Mendez JA, Soares ND, Mateos JM, Gayoso C, Rumbo C, Aranda J, Tomas MM, Bou G. 2012. Extracellular proteome of a highly invasive multidrug-resistant clinical strain of *Acinetobacter baumannii*. J. Proteome Res. 11:5678–5694.

277. Rumbo C, Fernandez-Moreira E, Merino M, Poza M, Mendez JA, Soares NC, Mosquera A, Chaves F, Bou G. 2011. Horizontal transfer of the OXA-24 carbapenemase gene via outer membrane vesicles: a new mechanism of dissemination of carbapenem resistance genes in *Acinetobacter baumannii*. In Antimicrob. Agents Chemother. 55:3084–3090.

278. Andersson DI, Hughes D. 2010. Antibiotic resistance and its cost: is it possible to reverse resistance? Nat. Rev. Microbiol. 8:260–271.

279. Björkman JD, Hughes Andersson DI. 1998. Virulence of antibiotic-resistant *Salmonella typhimurium*. Proc. Natl. Acad. Sci. U. S. A. 95: 3949–3953.

280. Björkman J, Samuelsson P, Andersson DI, Hughes D. 1999. Novel ribosomal mutations affecting translational accuracy, antibiotic resistance and virulence of *Salmonella typhimurium*. Mol. Microbiol. 31: 53–58.

281. Nilsson AI, Zorzet A, Kanth A, Dahlström S, Berg OG, Andersson DI. 2006. Reducing the fitness cost of antibiotic resistance by amplification of initiator tRNA genes. Proc. Natl. Acad. Sci. U. S. A. 103:6976–6981.

282. Albarracín Orio AG, Piñas GE, Cortes PR, Gian MB, Echenique J. 2011. Compensatory evolution of pbp mutations restores the fitness cost imposed by β-lactam resistance in *Streptococcus pneumoniae*. PLoS Pathog. 7:e1002000. doi:[10.1371/journal.ppat.1002000](https://doi.org/10.1371/journal.ppat.1002000).

283. Marcusson LL, Frimodt-Møller N, Hughes D. 2009. Interplay in the selection of fluoroquinolone resistance and bacterial fitness. PLoS Pathog. 5:e1000541. doi:[10.1371/journal.ppat.1000541](https://doi.org/10.1371/journal.ppat.1000541).

284. Pränting M, Andersson DI. 2010. Mechanisms and physiological effects of protamine resistance in *Salmonella enterica* serovar Typhimurium LT2. J. Antimicrob. Chemother. 65:876–887.

285. Luo N, Pereira S, Sahin O, Lin J, Huang S, Michel L, Zhang Q. 2005. Enhanced *in vivo* fitness of fluoroquinolone-resistant *Campylobacter jejuni* in the absence of antibiotic selection pressure. Proc. Natl. Acad. Sci. U. S. A. 102:541–546.

286. Paulander W, Maisnier-Patin S, Andersson DI. 2007. Multiple mechanisms to ameliorate the fitness burden of mupirocin resistance in *Salmonella typhimurium*. Mol. Microbiol. 64:1038–1048.

287. Hall AR, MacLean RC. 2011. Epistasis buffers the fitness effects of rifampicin-resistance mutations in *Pseudomonas aeruginosa*. Evolution 65:2370–2379.

288. Sander P, Springer B, Pramananan T, Sturmfels A, Kappler M, Pletschette M, Bottger EC. 2002. Fitness cost of chromosomal drug resistance-conferring mutations. Antimicrob. Agents Chemother. 46: 1204–1211.

289. Schulz zur Wiesch P, Engelstädter J, Bonhoeffer S. 2010. Compensation of fitness costs and reversibility of antibiotic resistance mutations. Antimicrob. Agents Chemother. 54:2085–2095.

290. Andersson DI. 2003. Persistence of antibiotic resistant bacteria. Curr. Opin. Microbiol. 6:452–456.

291. Levin BR. 2001. Minimizing potential resistance: a population dynamics view. Clin. Infect. Dis. 33(Suppl. 3):S161–S169.

292. Chiew YF, Yeo SF, Hall LM, Livermore DM. 1998. Can susceptibility to an antimicrobial be restored by halting its use? The case of streptomycin versus Enterobacteriaceae. J. Antimicrob. Chemother. 41:247–251.

293. Stephenson K, Hoch JA. 2002. Virulence- and antibiotic resistance-associated two-component signal transduction systems of Gram-positive pathogenic bacteria as targets for antimicrobial therapy. Pharmacol. Ther. 93:293–305.

294. Parkinson JS. 1993. Signal transduction schemes of bacteria. Cell 73: 857–871.

295. Gooderham WJ, Hancock RE. 2009. Regulation of virulence and antibiotic resistance by two-component regulatory systems in *Pseudomonas aeruginosa*. FEMS Microbiol. Rev. 33:279–294.

296. Stock AM, Robinson VL, Goudreau PN. 2000. Two-component signal transduction. Annu. Rev. Biochem. 69:183–215.

297. McPhee JB, Bains M, Winsor G, Lewenza S, Kwasnicka A, Brazas MD, Brinkman FS, Hancock RE. 2006. Contribution of the PhoP-PhoQ and PmrA-PmrB two-component regulatory systems to Mg²⁺-induced gene regulation in *Pseudomonas aeruginosa*. J. Bacteriol. 188:3995–4006.

298. Moskowitz Ernst SMRK, Miller SI. 2004. PmrAB, a two-component regulatory system of *Pseudomonas aeruginosa* that modulates resistance to cationic antimicrobial peptides and addition of aminoarabinose to lipid A. J. Bacteriol. 186:575–579.

299. Howden BP, McEvoy CR, Allen DL, Chua K, Gao W, Harrison PF, Bell J, Coombs G, Bennett-Wood V, Porter JL, Robins-Browne R, Davies JK, Seemann T, Stinear TP. 2011. Evolution of multidrug resistance during *Staphylococcus aureus* infection involves mutation of the essential two component regulator WalKR. PLoS Pathog. 7:e1002359. doi:[10.1371/journal.ppat.1002359](https://doi.org/10.1371/journal.ppat.1002359).

300. Wang Y, Ha U, Zeng L, Jin S. 2003. Regulation of membrane permeability by a two-component regulatory system in *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 47:95–101.

301. Davies JA, Harrison JJ, Marques IL, Foglia GR, Stremick CA, Storey DG, Turner RJ, Olson ME, Ceri H. 2007. The GacS sensor kinase controls phenotypic reversion of small colony variants isolated from biofilms of *Pseudomonas aeruginosa* PA14. FEMS Microbiol. Ecol. 59:32–46.

302. Drenkard E, Ausubel FM. 2002. *Pseudomonas* biofilm formation and antibiotic resistance are linked to phenotypic variation. Nature 416:740–743.

303. Kaufmann GF, Park J, Janda KD. 2008. Bacterial quorum sensing: a new target for anti-infective immunotherapy. Expert Opin. Biol. Ther. 8:719–724.

304. Winfield MD, Groisman EA. 2004. Phenotypic differences between *Salmonella* and *Escherichia coli* resulting from the disparate regulation of homologous genes. Proc. Natl. Acad. Sci. U. S. A. 101:17162–17167.

305. Kus JV, Gebremedhin A, Dang V, Tran SL, Serbanescu A, Foster DB. 2011. Bile salts induce resistance to polymyxin in enterohemorrhagic *Escherichia coli* O157:H7. J. Bacteriol. 193:4509–4515.

306. García Vescovi E, Soncini FC, Groisman EA. 1996. Mg2+ as an extracellular signal: environmental regulation of *Salmonella* virulence. Cell 84:165–174.

307. Miller SI, Mekalanos JJ. 1990. Constitutive expression of the *phoP* regulon attenuates *Salmonella* virulence and survival within macrophages. J. Bacteriol. 172:2485–2490.

308. Blanc-Potard AB, Groisman EA. 1997. The *Salmonella selC* locus contains a pathogenicity island mediating intramacrophage survival. EMBO J. 16:5376–5385.

309. Libby SJ, Adams LG, Ficht TA, Allen C, Whitford HA, Buchmeier NA,

Beceiro et al.

310. Ochman H, Soncini FC, Solomon F, Groisman EA. 1996. Identification of a pathogenicity island required for Salmonella survival in host cells. Proc. Natl. Acad. Sci. U. S. A. 93:7800–7804.
311. Bishop RE, Gibbons HS, Guina T, Trent MS, Miller SI, Raetz CR. 2000. Transfer of palmitate from phospholipids to lipid A in outer membranes of gram-negative bacteria. EMBO J. 19:5071–5080.
312. Trent MS, Pabich W, Raetz CR, Miller SI. 2001. A PhoP/PhoQ-induced Lipase (PagL) that catalyzes 3-O-deacylation of lipid A precursors in membranes of Salmonella typhimurium. J. Biol. Chem. 276:9083–9092.
313. Macfarlane EL, Kwasnicka A, Ochs MM, Hancock RE. 1999. PhoP-PhoQ homologues in *Pseudomonas aeruginosa* regulate expression of the outer-membrane protein OprH and polymyxin B resistance. Mol. Microbiol. 34:305–316.
314. Ernst RK, Yi EC, Guo L, Lim KB, Burns JL, Hackett M, Miller SI. 1999. Specific lipopolysaccharide found in cystic fibrosis airway *Pseudomonas aeruginosa*. Science 286:1561–1565.
315. Kwon DH, Lu CD. 2006. Polyamines induce resistance to cationic peptide, aminoglycoside, and quinolone antibiotics in *Pseudomonas aeruginosa* PAO1. Antimicrob. Agents Chemother. 50:1615–1622.
316. Ramsey MM, Whiteley M. 2004. *Pseudomonas aeruginosa* attachment and biofilm development in dynamic environments. Mol. Microbiol. 53:1075–1087.
317. Pérez E, Samper S, Bordas Y, Guilhot C, Gicquel B, Martín C. 2001. An essential role for phoP in *Mycobacterium tuberculosis* virulence. Mol. Microbiol. 41:179–187.
318. Kato A, Tanabe H, Utsumi R. 1999. Molecular characterization of the PhoP-PhoQ two-component system in *Escherichia coli* K-12: identification of extracellular Mg²⁺-responsive promoters. J. Bacteriol. 181:5516–5520.
319. Moss JE, Fisher PE, Vick B, Groisman EA, Zychlinsky A. 2000. The regulatory protein PhoP controls susceptibility to the host inflammatory response in *Shigella flexneri*. Cell. Microbiol. 2:443–452.
320. Oyston PC, Dorrell N, Williams K, Li SR, Green M, Titball RW, Wren BW. 2000. The response regulator PhoP is important for survival under conditions of macrophage-induced stress and virulence in *Yersinia pestis*. Infect. Immun. 68:3419–3425.
321. Gunn JS. 2008. The Salmonella PmrAB regulon: lipopolysaccharide modifications, antimicrobial peptide resistance and more. Trends Microbiol. 16:284–290.
322. Zhou Z, Ribeiro AA, Lin S, Cotter RJ, Miller SI, Raetz CR. 2001. Lipid A modifications in polymyxin-resistant *Salmonella typhimurium*: PMRA-dependent 4-amino-4-deoxy-L-arabinose, and phosphoethanolamine incorporation. J. Biol. Chem. 276:43111–43121.
323. Tamayo R, Choudhury B, Septer A, Merighi M, Carlson R, Gunn JS. 2005. Identification of *cptA*, a PmrA-regulated locus required for phosphoethanolamine modification of the *Salmonella enterica* serovar Typhi-murium lipopolysaccharide core. J. Bacteriol. 187:3391–3399.
324. Lee H, Hsu FF, Turk J, Groisman EA. 2004. The PmrA-regulated *pmrC* gene mediates phosphoethanolamine modification of lipid A and polymyxin resistance in *Salmonella enterica*. J. Bacteriol. 186:4124–4133.
325. Sivaneson M, Mikkelsen H, Ventre I, Bordi C, Filloux A. 2011. Two-component regulatory systems in *Pseudomonas aeruginosa*: an intricate network mediating fimbrial and efflux pump gene expression. Mol. Microbiol. 79:1353–1366.
326. Dong YH, XFZhang Soo HM, Greenberg EP, Zhang LH. 2005. The two-component response regulator PprB modulates quorum-sensing signal production and global gene expression in *Pseudomonas aeruginosa*. Mol. Microbiol. 56:1287–1301.
327. Giraud C, Bernard CS, Calderon V, Yang L, Filloux A, Molin S, Fichant G, Bordi C, de Bentzmann S. 2011. The PprA-PprB two-component system activates CupE, the first non-archetypal *Pseudomonas aeruginosa* chaperone-usher pathway system assembling fimbriae. Environ. Microbiol. 13:666–683.
328. Jiang SS, Liu MC, Teng LJ, Wang WB, Hsueh PR, Liaw SJ. 2010. *Proteus mirabilis pmrI*, an RppA-regulated gene necessary for polymyxin B resistance, biofilm formation, and urothelial cell invasion. Antimicrob. Agents Chemother. 54:1564–1571.
329. Little JW, Mount DW. 1982. The SOS regulatory system of *Escherichia coli*. Cell 29:11–22.
330. Cox MM. 1991. The RecA protein as a recombinational repair system. Mol. Microbiol. 5:1295–1299.
331. Little JW, Edmiston SH, Pacelli LZ, Mount DW. 1980. Cleavage of the *Escherichia coli lexA* protein by the *recA* protease. Proc. Natl. Acad. Sci. U. S. A. 77:3225–3229.
332. Butala M, Zgur-Bertok D, Busby SJ. 2009. The bacterial LexA transcriptional repressor. Cell. Mol. Life Sci. 66:82–93.
333. Drlica K, Zhao X. 1997. DNA gyrase, topoisomerase IV, and the 4-quinolones. Microbiol. Mol. Biol. Rev. 61:377–392.
334. Miller C, Thomsen LE, Gaggero C, Mosseri R, Ingmer H, Cohen SN. 2004. SOS response induction by beta-lactams and bacterial defense against antibiotic lethality. Science 305:1629–1631.
335. Phillips I, Culebras E, Moreno F, Baquero F. 1987. Induction of the SOS response by new 4-quinolones. J. Antimicrob. Chemother. 20:631–638.
336. Mühldorfer I, Hacker J, Keusch GT, Acheson DW, Tschäpe H, Kane AV, Ritter A, Olschläger T, Donohue-Rolfe A. 1996. Regulation of the Shiga-like toxin II operon in *Escherichia coli*. Infect. Immun. 64:495–502.
337. Ubeda C, Maiques E, Tormo MA, Campoy S, Lasa I, Barbé J, Novick RP, Penadés JR. 2007. SaPI operon I is required for SaPI packaging and is controlled by LexA. Mol. Microbiol. 65:41–50.
338. Beaber JW, Hochhut B, Waldor MK. 2004. SOS response promotes horizontal dissemination of antibiotic resistance genes. Nature 427:72–74.
339. Guerin E, Cambray G, Sanchez-Alberola N, Campoy S, Erill I, Da Re S, Gonzalez-Zorn B, Barbé J, Ploy MC, Mazel D. 2009. The SOS response controls integron recombination. Science 324:1034.
340. Shaw KJ, Miller N, Liu X, Lerner D, Wan J, Bittner A, Morrow BJ. 2003. Comparison of the changes in global gene expression of *Escherichia coli* induced by four bactericidal agents. J. Mol. Microbiol. Biotechnol. 5:105–122.
341. Ysern P, Clerch B, Castaño M, Gilbert I, Barbé J, Llagostera M. 1990. Induction of SOS genes in *Escherichia coli* and mutagenesis in *Salmonella typhimurium* by fluoroquinolones. Mutagenesis 5:63–66.
342. Baharoglu Z, Mazel D. 2011. Vibrio cholerae triggers SOS and mutagenesis in response to a wide range of antibiotics: a route towards multiresistance. Antimicrob. Agents Chemother. 55:2438–2441.
343. Mellies JL, Haack KR, Galligan DC. 2007. SOS regulation of the type III secretion system of enteropathogenic *Escherichia coli*. J. Bacteriol. 189:2863–2872.
344. Aranda JC, Bardina Beceiro A, Rumbo S, Cabral MP, Barbé J, Bou G. 2011. *Acinetobacter baumannii* RecA protein in repair of DNA damage, antimicrobial resistance, general stress response, and virulence. J. Bacteriol. 193:3740–3747.
345. Buchmeier NA, Lipps CJ, So MY, Heffron F. 1993. Recombination-deficient mutants of *Salmonella typhimurium* are avirulent and sensitive to the oxidative burst of macrophages. Mol. Microbiol. 7:933–936.
346. Cárdenas M, Fernández de Henestrosa AR, Campoy S, Perez de Rozas AM, Barbé J, Badiola I, Llagostera M. 2001. Virulence of *Pasteurella multocida recA* mutants. Vet. Microbiol. 80:53–61.
347. Cuccui J, Easton A, Chu KK, Bancroft GJ, Oyston PC, Titball RW, Wren BW. 2007. Development of signature-tagged mutagenesis in *Burkholderia pseudomallei* to identify genes important in survival and pathogenesis. Infect. Immun. 75:1186–1195.
348. LeClerc JE, Li B, Payne WL, Cebula TA. 1996. High mutation frequencies among *Escherichia coli* and *Salmonella* pathogens. Science 274:1208–1211.
349. Sniegowski PD, Gerrish PJ, Lenski RE. 1997. Evolution of high mutation rates in experimental populations of *E. coli*. Nature 387:703–705.
350. Giraud A, Matic I, Tenaillon O, Clara A, Radman M, Fons M, Taddei F. 2001. Costs and benefits of high mutation rates: adaptive evolution of bacteria in the mouse gut. Science 291:2606–2608.
351. Maciá MD, Borrell N, Segura M, Gómez C, Pérez JL, Oliver A. 2006. Efficacy and potential for resistance selection of antipseudomonal treatments in a mouse model of lung infection by hypermutable *Pseudomonas aeruginosa*. Antimicrob. Agents Chemother. 50:975–983.
352. Nilsson AI, Kugelberg E, Berg OG, Andersson DI. 2004. Experimental adaptation of *Salmonella typhimurium* to mice. Genetics 168:1119–1130.
353. Oliver AR, Cantón Campo P, Baquero F, Blázquez J. 2000. High frequency of hypermutable *Pseudomonas aeruginosa* in cystic fibrosis lung infection. Science 288:1251–1254.
354. Mena A, Maciá MD, Borrell N, Moya B, de Francisco T, Pérez JL, Oliver A. 2007. Inactivation of the mismatch repair system in *Pseudomonas*

nas aeruginosa attenuates virulence but favors persistence of oropharyngeal colonization in cystic fibrosis mice. J. Bacteriol. 189:3665–3668.

355. Henwood CJ, Livermore DM, James D, Warner M. 2001. Antimicrobial susceptibility of *Pseudomonas aeruginosa*: results of a UK survey and evaluation of the British Society for Antimicrobial Chemotherapy disc susceptibility test. J. Antimicrob. Chemother. 47:789–799.

356. Prunier AL, Malbruny B, Laurans M, Brouard J, Duhamel JF, Leclercq R. 2003. High rate of macrolide resistance in *Staphylococcus aureus* strains from patients with cystic fibrosis reveals high proportions of hypermutable strains. J. Infect. Dis. 187:1709–1716.

357. Román F, Cantón R, Pérez-Vázquez M, Baquero F, Campos J. 2004. Dynamics of long-term colonization of respiratory tract by *Haemophilus influenzae* in cystic fibrosis patients shows a marked increase in hypermutable strains. J. Clin. Microbiol. 42:1450–1459.

358. Ferroni A, Guillemot D, Moumile K, Bernede C, Le Bourgeois M, Waernessyckle S, Descamps P, Sermet-Gaudelus I, Lenoir G, Berche P, Taddei F. 2009. Effect of mutator *P. aeruginosa* on antibiotic resistance acquisition and respiratory function in cystic fibrosis. Pediatr. Pulmonol. 44:820–825.

359. Oliver A. 2010. Mutators in cystic fibrosis chronic lung infection: Prevalence, mechanisms, and consequences for antimicrobial therapy. Int. J. Med. Microbiol. 300:563–572.

360. Oliver A, Mena A. 2010. Bacterial hypermutation in cystic fibrosis, not only for antibiotic resistance. Clin. Microbiol. Infect. 16:798–808.

361. Conibear TC, Collins SL, Webb JS. 2009. Role of mutation in *Pseudomonas aeruginosa* biofilm development. PLoS One 4:e6289. doi:10.1371/journal.pone.0006289.

362. Montanari SA, Oliver Salerno P, Mena A, Bertoni G, Tümmler B, Cariani L, Conese M, Döring G, Bragonzi A. 2007. Biological cost of hypermutation in *Pseudomonas aeruginosa* strains from patients with cystic fibrosis. Microbiology 153:1445–1454.

363. Richardson AR, Yu Z, Popovic T, Stojiljkovic I. 2002. Mutator clones of *Neisseria meningitidis* in epidemic serogroup A disease. Proc. Natl. Acad. Sci. U. S. A. 99:6103–6107.

364. Ebrahimi-Rad M, Bifani P, Martin C, Kremer K, Samper S, Rauzier J, Kreiswirth B, Blazquez J, Jouan M, van Soolingen D, Gicquel B. 2003. Mutations in putative mutator genes of *Mycobacterium tuberculosis* strains of the W-Beijing family. Emerg. Infect. Dis. 9:838–845.

365. Labat F, Pradillon O, Garry L, Peuchmaur M, Fantin B, Denamur E. 2005. Mutator phenotype confers advantage in *Escherichia coli* chronic urinary tract infection pathogenesis. FEMS Immunol. Med. Microbiol. 44:317–321.

366. Mulcahy LR, Burns JL, Lory S, Lewis K. 2010. Emergence of *Pseudomonas aeruginosa* strains producing high levels of persister cells in patients with cystic fibrosis. J. Bacteriol. 192:6191–6199.

367. Lafleur MD, Qi Q, Lewis K. 2010. Patients with long-term oral carriage harbor high-persister mutants of *Candida albicans*. Antimicrob. Agents Chemother. 54:39–44.

368. Lechner S, Lewis K, Bertram R. 2012. *Staphylococcus aureus* persisters tolerant to bactericidal antibiotics. J. Mol. Microbiol. Biotechnol. 22: 235–244.

369. Hansen S, Lewis K, Vulić M. 2008. Role of global regulators and nucleotide metabolism in antibiotic tolerance in *Escherichia coli*. Antimicrob. Agents Chemother. 52:2718–2726.

370. Dörr TK, Lewis Vulić M. 2009. SOS response induces persistence to fluoroquinolones in *Escherichia coli*. PLoS Genet. 5:e1000760. doi:10.1371/journal.pgen.1000760.

371. Lewis K. 2010. Persister cells. Annu. Rev. Microbiol. 64:357–372.

372. Wu Y, Vulic M, Keren I, Lewis K. 2012. Role of oxidative stress in persister tolerance. Antimicrob. Agents Chemother. 56:4922–4926.

373. Keren I, Shah D, Spoering A, Kaldalu N, Lewis K. 2004. Specialized persister cells and the mechanism of multidrug tolerance in *Escherichia coli*. J. Bacteriol. 186:8172–8180.

374. Dörr T, Vulić M, Lewis K. 2010. Ciprofloxacin causes persister formation by inducing the TisB toxin in *Escherichia coli*. PLoS Biol. 8:e1000317. doi:10.1371/journal.pbio.1000317.

375. Gaynor EC, Wells DH, MacKichan JK, Falkow S. 2005. The *Campylobacter jejuni* stringent response controls specific stress survival and virulence-associated phenotypes. Mol. Microbiol. 56:8–27.

376. Kim S, Watanabe K, Suzuki H, Watarai M. 2005. Roles of *Brucella abortus* SpoT in morphological differentiation and intramacrophagic replication. Microbiology 151:1607–1617.

377. Lemos JA, Brown TA, Burne RA. 2004. Effects of RelA on key virulence properties of planktonic and biofilm populations of *Streptococcus mutans*. Infect. Immun. 72:1431–1440.

378. Zhang S, Haldenwang WG. 2003. RelA is a component of the nutritional stress activation pathway of the *Bacillus subtilis* transcription factor sigma B. J. Bacteriol. 185:5714–5721.

379. Balzer GJ, McLean RJ. 2002. The stringent response genes *relA* and *spoT* are important for *Escherichia coli* biofilms under slow-growth conditions. Can. J. Microbiol. 48:675–680.

380. Taylor CM, Beresford M, Epton HA, Sigee DC, Shama G, Andrew PW, Roberts IS. 2002. *Listeria monocytogenes relA* and *hpt* mutants are impaired in surface-attached growth and virulence. J. Bacteriol. 184:621–628.

381. Greenway DL, England RR. 1999. The intrinsic resistance of *Escherichia coli* to various antimicrobial agents requires (p) pGpp and sigmas. Lett. Appl. Microbiol. 29:323–326.

382. Joseleau-Petit D, Thévenet D, D'Ari R. 1994. (p)pGpp concentration, growth without PBP2 activity, and growth-rate control in *Escherichia coli*. Mol. Microbiol. 13:911–917.

383. Pomares MF, Vincent PA, Farias RN, Solomon RA. 2008. Protective action of (p) pGpp in microcin J25-sensitive strains. J. Bacteriol. 190:4328–4334.

384. Braeken K, Moris M, Daniels R, Vanderleyden J, Michiels J. 2006. New horizons for (p)ppGpp in bacterial and plant physiology. Trends Microbiol. 14:45–54.

385. Korch SB, Henderson TA, Hill TM. 2003. Characterization of the *hipA7* allele of *Escherichia coli* and evidence that high persistence is governed by (p) ppGpp synthesis. Mol. Microbiol. 50:1199–1213.

386. Nascimento MM, Lemos JA, Abranches J, Lin VK, Burne RA. 2008. Role of RelA of *Streptococcus mutans* in global control of gene expression. J. Bacteriol. 190:28–36.

387. Primm TP, Andersen SJ, Mizrahi V, Avarbock D, Rubin H, Barry CE. 2000. The stringent response of *Mycobacterium tuberculosis* is required for long-term survival. J. Bacteriol. 182:4889–4898.

388. Abranches J, Martinez AR, Kajfasz JK, Chavez V, Garsin DA, Lemos JA. 2009. The molecular alarmone (p)ppGpp mediates stress responses, vancomycin tolerance, and virulence in *Enterococcus faecalis*. J. Bacteriol. 191:2248–2256.

389. Battesti A, Bouveret E. 2009. Bacteria possessing two RelA/SpoT-like proteins have evolved a specific stringent response involving the acyl carrier protein-SpoT interaction. J. Bacteriol. 191:616–624.

390. Lemos JA, Lin VK, Nascimento MM, Abranches J, Burne RA. 2007. Three gene products govern (p)ppGpp production by *Streptococcus mutans*. Mol. Microbiol. 65:1568–1581.

391. Nanamiya H, Kasai K, Nozawa A, Yun CS, Narisawa T, Murakami K, Natori Y, Kawamura F, Tozawa Y. 2008. Identification and functional analysis of novel pppGpp synthetase genes in *Bacillus subtilis*. Mol. Microbiol. 67:291–304.

392. Potrykus K, Cashel M. 2008. (p)ppGpp: still magical? Annu. Rev. Microbiol. 62:35–51.

393. Gao W, Chua K, Davies JK, Newton HJ, Seemann T, Harrison PF, Holmes NE, Rhee HW, Hong JI, Hartland EL, Stinear TPTP, Howden BP. 2010. Two novel point mutations in clinical *Staphylococcus aureus* reduce linezolid susceptibility and switch on the stringent response to promote persistent infection. PLoS Pathog. 6:e1000944. doi:10.1371/journal.ppat.1000944.

394. van Schaik W, Abee T. 2005. The role of sigmaB in the stress response of Gram-positive bacteria—targets for food preservation and safety. Curr. Opin. Biotechnol. 16:218–224.

395. Cheung AL, Bayer AS, Zhang G, Gresham H, Xiong YQ. 2004. Regulation of virulence determinants *in vitro* and *in vivo* in *Staphylococcus aureus*. FEMS Immunol. Med. Microbiol. 40:1–9.

396. Schulthes B, Meier S, Homerova D, Goerke C, Wolz C, Kormanec J, Berger-Bachi B, Bischoff M. 2009. Functional characterization of the sigmaB-dependent *yabJ-spoVG* operon in *Staphylococcus aureus*: role in methicillin and glycopeptide resistance. Antimicrob. Agents Chemother. 53:1832–1839.

397. Chen HY, Chen CC, Fang CS, Hsieh YT, Lin MH, Shu JC. 2011. Vancomycin activates sigma in vancomycin-resistant *Staphylococcus aureus* resulting in the enhancement of cytotoxicity. PLoS One 6:e24472. doi:10.1371/journal.pone.0024472.

398. Meier S, Goerke C, Wolz C, Seidl K, Homerova D, Schulthes B, Kormanec J, Berger-Bächi B, Bischoff M. 2007. sigmaB and the sigmaB-

dependent arlRS and yabJ-spoVG loci affect capsule formation in Staphylococcus aureus. Infect. Immun. 75:4562–4571.

399. Shin JH, Kim J, Kim SM, Kim S, Lee JC, Ahn JM, Cho JY. 2010. sigmaB-dependent protein induction in *Listeria monocytogenes* during vancomycin stress. FEMS Microbiol. Lett. 308:94–100.

400. Rasko DA, Sperandio V. 2010. Anti-virulence strategies to combat bacteria-mediated disease. Nat. Rev. Drug Discov. 9:117–128.

401. Varga ZG, Armada A, Cerca P, Amaral L, Mior Ahmad Subki MA, Savka MA, Szegedi E, Kawase M, Motohashi N, Molnar J. 2012. Inhibition of quorum sensing and efflux pump system by trifluoromethyl ketone proton pump inhibitors. In Vivo 26:277–285.

402. Tan H, Peng Z, Li Q, Xu X, Guo S, Tang T. 2012. The use of quaternised chitosan-loaded PMMA to inhibit biofilm formation and downregulate the virulence-associated gene expression of antibiotic-resistant staphylococcus. Biomaterials 33:365–377.

403. Mitchell G, Lafrance M, Boulanger S, Séguin DL, Guay I, Gattuso M, Marsault E, Bouarab K, Malouin F. 2012. Tomatidine acts in synergy with aminoglycoside antibiotics against multiresistant *Staphylococcus aureus* and prevents virulence gene expression. J. Antimicrob. Chemother. 67:559–568.

404. Bina XR, Philippart JA, Bina JE. 2009. Effect of the efflux inhibitors 1-1-naphthylmethyl-piperazine and phenyl-arginine-beta-naphthylamide on antimicrobial susceptibility and virulence factor production in *Vibrio cholerae*. J. Antimicrob. Chemother. 63:103–108.

405. Kohler T, Perron GG, Buckling A, van Delden C. 2010. Quorum sensing inhibition selects for virulence and cooperation in *Pseudomonas aeruginosa*. PLoS Pathog. 6:e1000883. doi:[10.1371/journal.ppat.1000883](https://doi.org/10.1371/journal.ppat.1000883).

406. Halldorsson S, Gudjonsson T, Gottfredsson M, Singh PK, Gudmundsson GH, Baldursson O. 2010. Azithromycin maintains airway epithelial integrity during *Pseudomonas aeruginosa* infection. Am. J. Respir. Cell Mol. Biol. 42:62–68.

407. Renna M, Schaffner C, Brown K, Shang S, Tamayo MH, Hegyi K, Grimsey NJ, Cusens D, Coulter S, Cooper J, Bowden AR, Newton SM, Kampmann B, Helm J, Jones A, Haworth CS, Basaraba RJ, DeGroote MA, Ordway DJ, Rubinsztein DC, Floto RA. 2011. Azithromycin blocks autophagy and may predispose cystic fibrosis patients to mycobacterial infection. J. Clin. Invest. 121:3554–3563.

408. Smani Y, Domínguez-Herrera J, Pachón J. 2011. Rifampin protects human lung epithelial cells against cytotoxicity induced by clinical multi and pandrug-resistant *Acinetobacter baumannii*. J. Infect. Dis. 203:1110–1119.

409. Yoshizawa S, Tateda K, Saga T, Ishii Y, Yamaguchi K. 2012. Virulence-suppressing effects of linezolid on methicillin-resistant *Staphylococcus aureus*: possible contribution to early defervescence. Antimicrob. Agents Chemother. 56:1744–1748.

410. Fernández-Barat L, Ferrer M, Sierra JM, Soy D, Guerrero L, Vila J, Li Bassi G, Cortadellas N, Martínez-Olondris P, Rigol M, Esperatti M, Luque N, Saucedo LM, Agustí C, Torres A. 2012. Linezolid limits burden of methicillin-resistant *Staphylococcus aureus* in biofilm of tracheal tubes. Crit. Care Med. 40:2385–2389.

411. Verma V, Harjai K, Chhibber S. 2009. Restricting ciprofloxacin-induced resistant variant formation in biofilm of *Klebsiella pneumoniae* B5055 by complementary bacteriophage treatment. J. Antimicrob. Chemother. 64:1212–1218.

412. Princivalli MS, Paoletti C, Magi G, Palmieri C, Ferrante L, Facinelli B. 2009. *Lactobacillus rhamnosus* GG inhibits invasion of cultured human respiratory cells by prtF1-positive macrolide-resistant group A streptococci. Lett. Appl. Microbiol. 48:368–372.

413. Jermy A. 2011. Antimicrobials: disruption of quorum sensing meets resistance. Nat. Rev. Microbiol. 9:767.

414. Mahajan-Miklos S, Tan MW, Rahme LG, Ausubel FM. 1999. Molecular mechanisms of bacterial virulence elucidated using a *Pseudomonas aeruginosa-Caenorhabditis elegans* pathogenesis model. Cell 96:47–56.

415. Zgurskaya HI, Nikaido H. 1999. AcrA is a highly asymmetric protein capable of spanning the periplasm. J. Mol. Biol. 285:409–420.

416. Ma D, Cook DN, Alberti M, Pon NG, Nikaido H, Hearst JE. 1995. Genes *acrA* and *acrB* encode a stress-induced efflux system of *Escherichia coli*. Mol. Microbiol. 16:45–55.

417. Hood MI, Jacobs AC, Sayood K, Dunman PM, Skaar EP. 2010. *Acinetobacter baumannii* increases tolerance to antibiotics in response to monovalent cations. Antimicrob. Agents Chemother. 54:1029–1041.

418. Livermore DM. 2003. Bacterial resistance: origins, epidemiology, and impact. Clin. Infect. Dis. 36:S11–23.


419. Jawad A, Seifert H, Snelling AM, Heritage J, Hawkey PM. 1998. Survival of *Acinetobacter baumannii* on dry surfaces: comparison of outbreak and sporadic isolates. J. Clin. Microbiol. 36:1938–1941.

420. Baquero F, TM Coque, de la Cruz F. 2011. Ecology and evolution as targets: the need for novel eco-evo drugs and strategies to fight antibiotic resistance. Antimicrob. Agents Chemother. 55:3649–3660.

421. Barrett JF. 2005. Can biotech deliver new antibiotics? Curr. Opin. Microbiol. 8:498–503.

422. Becker D, Selbach M, Rollenhagen C, Ballmaier M, Meyer TF, Mann M, Bumann D. 2006. Robust *Salmonella* metabolism limits possibilities for new antimicrobials. Nature 440:303–307.

423. Cantón R, Ruiz-Garbajosa P. 2011. Co-resistance: an opportunity for the bacteria and resistance genes. Curr. Opin. Pharmacol. 11:477–485.

424. Cantón R, González-Alba JM, Galán JC. 2012. CTX-M enzymes: origin and diffusion. Front. Microbiol. 3:110.

425. Winstanley C, Langille MG, Fothergill JL, Kukavica-Ibrulj I, Paradis-Bleau C, Sanschagrin F, Thomson NR, Winsor GL, Quail MA, Lennard N, Bignell A, Clarke L, Seeger K, Saunders D, Harris D, Parkhill J, Hancock RE, Brinkman FS, Levesque RC. 2009. Newly introduced genomic prophage islands are critical determinants of *in vivo* competitiveness in the Liverpool epidemic strain of *Pseudomonas aeruginosa*. Genome Res. 19:12–23.

426. Ashish A, Shaw M, Winstanley C, Ledson MJ, Walshaw MJ. 2012. Increasing resistance of the Liverpool epidemic strain LES of *Pseudomonas aeruginosa* Psa to antibiotics in cystic fibrosis CF—a cause for concern? J. Cyst. Fibros. 11:173–179.

427. Cantón R, Coque TM, Baquero F. 2003. Multi-resistant Gram-negative bacilli: from epidemics to endemics. Curr. Opin. Infect. Dis. 16:315–325.

428. Livermore D. 2011. Discovery research: the scientific challenge of finding new antibiotics. J. Antimicrob. Chemother. 66:1941–1944.

429. Lavigne JP, Vitrac X, Bernard L, Bruyère F, Sotto A. 2011. Propolis can potentialise the anti-adhesion activity of proanthocyanidins on uropathogenic *Escherichia coli* in the prevention of recurrent urinary tract infections. BMC Res. Notes 4:522.

430. Cegelski L, Marshall GR, Eldridge GR, Hultgren SJ. 2008. The biology and future prospects of antivirulence therapies. Nat. Rev. Microbiol. 6:17–27.

431. Clatworthy AE, Pierson E, Hung DT. 2007. Targeting virulence: a new paradigm for antimicrobial therapy. Nat. Chem. Biol. 3:541–548.

432. Escaich S. 2008. Antivirulence as a new antibacterial approach for chemotherapy. Curr. Opin. Chem. Biol. 12:400–408.

433. Rappuoli R. 2000. Reverse vaccinology. Curr. Opin. Microbiol. 3:445–450.

434. Bender JM, Ampofo K, Korgenski K, Daly J, Pavia AT, Mason EO, Byington CL. 2008. Pneumococcal necrotizing pneumonia in Utah: does serotype matter? Clin. Infect. Dis. 46:1346–1352.

435. Wieser AE, Romann Magistro G, Hoffmann C, Nörenberg D, Weinert K, Schubert S. 2010. A multiepitope subunit vaccine conveys protection against extraintestinal pathogenic *Escherichia coli* in mice. Infect. Immun. 78:3432–3442.

436. Merino M, Acosta J, Poza M, Sanz F, Beceiro AF, Chaves Bou G. 2010. OXA-24 carbapenemase gene flanked by XerC/XerD-like recombination sites in different plasmids from different *Acinetobacter* species isolated during a nosocomial outbreak. Antimicrob. Agents Chemother. 54:2724–2727.

437. Héritier C, Poirel L, Lambert T, Nordmann P. 2005. Contribution of acquired carbapenem-hydrolyzing oxacillases to carbapenem resistance in *Acinetobacter baumannii*. Antimicrob. Agents Chemother. 49:3198–3202.

438. Reference deleted.

439. Prost LR, Daley ME, Bader MW, Klevit RE, Miller SI. 2008. The PhoQ histidine kinases of *Salmonella* and *Pseudomonas* spp. are structurally and functionally different: evidence that pH and antimicrobial peptide sensing contribute to mammalian pathogenesis. Mol. Microbiol. 69:503–519.

440. Karginov VA, Nestorovich EM, Moayeri M, Leppla SH, Bezrukov SM. 2005. Blocking anthrax lethal toxin at the protective antigen channel by using structure-inspired drug design. Proc. Natl. Acad. Sci. U. S. A. 102:15075–15080.

441. Shoop WL, Xiong Y, Wiltsie J, Woods A, Guo J, Pivnichny JV, Felcetto T, Michael BF, Bansal A, Cummings RT, Cunningham BR, Friedlander AM, Douglas CM, Patel SB, Wisniewski D, Scapin G, Salowe SP, Zaller DM, Chapman KT, Scolnick EM, Schmatz DM, Bartizal K,

MacCoss M, Hermes JD. 2005. Anthrax lethal factor inhibition. Proc. Natl. Acad. Sci. U. S. A. 102:7958–7963.

442. Moayeri M, Wiggins JF, Lindeman RE, Leppla SH. 2006. Cisplatin inhibition of anthrax lethal toxin. Antimicrob. Agents Chemother. 50: 2658–2665.

443. Armstrong GD, Rowe PC, Goodyer P, Orrbine E, Klassen TP, Wells G, MacKenzie A, Lior H, Blanchard C, Auclair F. 1995. A phase I study of chemically synthesized verotoxin (Shiga-like toxin) Pk-trisaccharide receptors attached to chromosorb for preventing hemolytic-uremic syndrome. J. Infect. Dis. 171:1042–1045.

444. Trachtman HA, Cnaan Christen E, Gibbs K, Zhao S, Acheson DW, Weiss R, Kaskel FJ, Spitzer A, Hirschman GH. 2003. Effect of an oral Shiga toxin-binding agent on diarrhea-associated hemolytic uremic syndrome in children: a randomized controlled trial. JAMA 290:1337–1344.

445. Chorell E, Pinkner JS, Phan G, Edvinsson S, Buelens F, Remaut H, Waksman G, Hultgren SJ, Almqvist F. 2010. Design and synthesis of C-2 substituted thiazolo and dihydrothiazolo ring-fused 2-pyridones: pilicides with increased antivirulence activity. J. Med. Chem. 53:5690–5695.

446. Shakhnovich EA, Hung DT, Pierson E, Lee K, Mekalanos JJ. 2007. Virstatin inhibits dimerization of the transcriptional activator ToxT. Proc. Natl. Acad. Sci. U. S. A. 104:2372–2377.

447. Bailey L, Gylfe A, Sundin C, Muschiol S, Elofsson M, Nordström P, Henriques-Normark B, Lugert R, Waldenström A, Wolf-Watz H, Bergström S. 2007. Small molecule inhibitors of type III secretion in Yersinia block the Chlamydia pneumoniae infection cycle. FEBS Lett. 581: 587–595.

448. Veenendaal AK, Sundin C, Blocker AJ. 2009. Small-molecule type III secretion system inhibitors block assembly of the Shigella type III secretton. J. Bacteriol. 191:563–570.

449. Felise HB, Nguyen HV, Pfuetzner RA, Barry KC, Jackson SR, Blanc MP, Bronstein PA, Kline T, Miller SI. 2008. An inhibitor of gram-negative bacterial virulence protein secretion. Cell Host Microbe 4:325–336.

450. Oh KB, Kim SH, Lee J, Cho WJ, Lee T, Kim S. 2004. Discovery of diarylacrylonitriles as a novel series of small molecule sortase A inhibitors. J. Med. Chem. 47:2418–2421.

451. Hentzer M, Wu H, Andersen JB, Riedel K, Rasmussen TB, Bagge N, Kumar N, Schembri MA, Song Z, Kristoffersen P, Manefield M, Costerton JW, Molin S, Eberl L, Steinberg P, Kjelleberg S, Høiby N,

Givskov M. 2003. Attenuation of *Pseudomonas aeruginosa* virulence by quorum sensing inhibitors. EMBO J. 22:3803–3815.

452. Manefield M, Rasmussen TB, Henzter M, Andersen JB, Steinberg P, Kjelleberg S, Givskov M. 2002. Halogenated furanones inhibit quorum sensing through accelerated LuxR turnover. Microbiology 148:1119–1127.

453. Lönn-Stensrud J, Naemi AO, Benneche T, Petersen FC, Scheie AA. 2012. Thiophenones inhibit *Staphylococcus epidermidis* biofilm formation at nontoxic concentrations. FEMS Immunol. Med. Microbiol. 65: 326–334.

454. Escaich S. 2010. Novel agents to inhibit microbial virulence and pathogenicity. Expert Opin. Ther. Pat. 20:1401–1418.

455. Lee JH, Kim YG, Cho MH, Kim JA, Lee J. 2012. 7-Fluoroindole as an antivirulence compound against *Pseudomonas aeruginosa*. FEMS Microbiol. Lett. 329:36–44.

456. Matsunaga T, Nakahara A, Minnatul KM, Noiri Y, Ebisu S, Kato A, Azakami H. 2010. The inhibitory effects of catechins on biofilm formation by the periodontopathogenic bacterium, *Eikenella corrodens*. Biosci. Biotechnol. Biochem. 74:2445–2450.

457. Ni N, Choudhary G, Li M, Wang B. 2008. Pyrogallol and its analogs can antagonize bacterial quorum sensing in *Vibrio harveyi*. Bioorg. Med. Chem. Lett. 18:1567–1572.

458. Vila J, Soto S. 2012. Salicylate increases the expression of marA and reduces *in vitro* biofilm formation in uropathogenic *Escherichia coli* by decreasing type 1 fimbriae expression. Virulence 3:280–285.

459. Miethke M, Bisseret P, Beckering CL, Vignard D, Eustache J, Marahiel MA. 2006. Inhibition of aryl acid adenylation domains involved in bacterial siderophore synthesis. FEBS J. 273:409–419.

460. Neres J, Labello NP, Somu RV, Boshoff HI, Wilson DJ, Vannada J, Chen L, Barry CE, Bennett EM, Aldrich CC. 2008. Inhibition of siderophore biosynthesis in *Mycobacterium tuberculosis* with nucleoside bisubstrate analogues: structure-activity relationships of the nucleobase domain of 5′-O-[N-(salicyl)sulfamoyl]adenosine. J. Med. Chem. 51: 5349–5370.

461. Zhou B, He Y, Zhang X, Xu J, Luo Y, Wang Y, Franzblau SG, Yang Z, Chan RJ, Liu Y, Zheng J, Zhang ZYZY. 2010. Targeting mycobacterium protein tyrosine phosphatase B for antituberculosis agents. Proc. Natl. Acad. Sci. U. S. A. 107:4573–4578.

462. May JJ, Finking R, Wiegeshoff F, Weber TT, Bandur N, Koert U, Marahiel MA. 2005. Inhibition of the D-alanine:D-alanyl carrier protein ligase from *Bacillus subtilis* increases the bacterium’s susceptibility to antibiotics that target the cell wall. FEBS J. 272:2993–3003.


Alejandro Beceiro, Ph.D., is a researcher recruited by the program Juan de Cierva (Secretary of State for Research, Development and Innovation, Spain). He performs his research activity at the Institute for Biomedical Research of A Coruña-University Hospital of A Coruña, Spain, where he also received his training as a Ph.D. student (2001 to 2007). He continued his research career as a postdoctoral fellow at the University Hospital Virgen del Rocío, University Hospital Son Espases (Spain), and the Health Protection Agency (United Kingdom) (2008 to 2010). His research interests encompass studies in the basic and clinical microbiology of some of the most clinically important pathogens, especially *A. baumannii*, focusing on the study of mechanisms of resistance to antimicrobials and mechanisms of virulence.


María Tomás Carmona, M.D. and Ph.D., obtained her medical degree from the University of Granada, Andalucia, Spain (1994 to 2000), and her microbiology specialty at A Coruña Hospital, Galicia, Spain (2001 to 2005). Over the years, she has investigated different mechanisms of resistance to antimicrobials in several pathogens, such as *A. baumannii* and *P. aeruginosa* (featuring over 30 publications about this topic) in various research centers (2006 to 2007 at CIB, Madrid, Spain; 2008 to 2009 at HPA, London, United Kingdom). Currently, she is working with a Miguel Servet contract from Instituto Salud Carlos III (research contract, National Health System) initiating new research on the relationship between resistance and virulence mechanisms in these pathogens. Moreover, she is the coordinator of the Genomic Diagnosis Unit in the Clinical Microbiology Department at University Hospital of A Coruña (Galicia, Spain), whose director is Dr. Germán Bou and which examines new molecular techniques for the diagnosis of infectious diseases.

Continued next page

Beceiro et al.

Germán Bou received his Ph.D. from the Molecular Biology Center (CSIC)-Autonoma University, Madrid, Spain. He also completed a residence in clinical microbiology at Ramon y Cajal Hospital. Afterwards, he was granted a postdoctoral position with the Fulbright Scholarship program to work in the Laboratory of Medicine at Mayo Clinic, Rochester, MN. Dr. Bou then served as an investigator at the National Health System in Spain (2001 to 2005) and as a consultant in clinical microbiology (2005 to 2010), and at present he is the Head of the Microbiology Department of the University Hospital A Coruña (CHUAC). Since 2009, he has been associate professor of medical microbiology at Santiago of Compostela University. Dr. Bou’s research focuses on the understanding of the molecular basis for antimicrobial resistance in human pathogens, on the epidemiology of nosocomial infections, and on the mechanism of pathogenicity of *Acinetobacter baumannii*. So far, he has published more than 100 international peer-reviewed manuscripts on these topics.
