
# Activation of the Yeast Hippo Pathway by Phosphorylation-Dependent Assembly of Signaling Complexes

Jeremy M. Rock,$^{1,2}$ Daniel Lim,$^{1}$ Lasse Stach,$^{3}$ Roksana W. Ogrodowicz,$^{3}$ Jamie M. Keck,$^{4*}$ Michele H. Jones,$^{4}$ Catherine C. L. Wong,$^{5}$ John R. Yates III,$^{5}$ Mark Winey,$^{4}$ Stephen J. Smerdon,$^{3}$ Michael B. Yaffe,$^{1}$ Angelika Amon$^{1,2†}$

Scaffold-assisted signaling cascades guide cellular decision-making. In budding yeast, one such signal transduction pathway called the mitotic exit network (MEN) governs the transition from mitosis to the G₁ phase of the cell cycle. The MEN is conserved and in metazoans is known as the Hippo tumor-suppressor pathway. We found that signaling through the MEN kinase cascade was mediated by an unusual two-step process. The MEN kinase Cdc15 first phosphorylated the scaffold Nud1. This created a phospho-docking site on Nud1, to which the effector kinase complex Dbf2-Mob1 bound through a phosphoserine-threonine binding domain, in order to be activated by Cdc15. This mechanism of pathway activation has implications for signal transmission through other kinase cascades and might represent a general principle in scaffold-assisted signaling.

**S**ignaling complex assembly must be tightly controlled in both space and time. The use of protein scaffolds and phosphorylation-dependent protein-protein interactions is central to this spatiotemporal control. The mitotic exit network (MEN) is a conserved guanosine triphosphate (GTPase) signaling cascade that governs exit from mitosis (1, 2). During anaphase, MEN signaling complexes assemble at spindle pole bodies (SPBs) (3, 4) to trigger exit from mitosis and to couple this cell cycle transition with nuclear position (5). The Ras-like GTPase Tem1 and the Polo protein kinase Cdc5 coordinately recruit the Hippo-like kinase Cdc15 to SPBs (3). Once localized to SPBs, Cdc15 is activated to phosphorylate the kinase Dbf2 and its coactivator Mob1 (6). Phosphorylation activates Dbf2-Mob1, which then promotes the release of the MEN effector protein phosphatase Cdc14 from the nucleus, resulting in exit from mitosis (2).

Scaffold proteins serve as assembly platforms for kinase cascades and may function as signaling insulators (7). Our results show that, rather than functioning as a passive platform onto which MEN components assemble, the SPB-resident MEN scaffold Nud1 is a dynamic participant in MEN signal transmission. Nud1 is a phosphoprotein and its phosphorylation increases during mitosis (fig. S1, A to C, and table S1) (8–10). We generated a *NUD1* allele in which the 38 high-confidence mitotic phosphorylation sites and 4 lower-confidence sites were mutated to alanine (henceforth *nud1-42A*). The *nud1-42A* protein was stable and localized to SPBs, but phosphorylation during mitosis was reduced (fig. S1, B to D).

To examine the effects of the *nud1-42A* allele on MEN activity, we introduced the allele into a strain expressing the temperature-sensitive *nud1-44* allele under the control of the galactose-inducible and glucose-repressible *GAL1-10* promoter. *GAL-nud1-44 nud1-42A* cells, like MEN loss-of-function mutants, arrested in late anaphase with inactive Dbf2-Mob1 and nuclear-restricted Cdc14 under conditions in which *nud1-44* is inactive (Fig. 1, A and B, and fig. S1, E and F). Thus, *nud1-42A* cells are defective in MEN signaling.

Localization of the MEN components Tem1, Cdc15, Dbf2, and Mob1 to SPBs is essential for Dbf2-Mob1 activation and requires *NUD1* (3, 4, 11). Localization of Nud1, Bfa1 [a Tem1 GTPase-activating protein (GAP) complex component], Tem1, and Cdc15 was normal in *nud1-42A* cells (12) (fig. S2, A to D), but Mob1 and Dbf2 were absent from SPBs (fig. S2, E and F). *nud1-42A* cells also harbored mispositioned anaphase spindles and detached astral microtubules (fig. S1F).

---

### Figures

#### Fig. 1. Dbf2-Mob1 recruitment to SPBs and MEN activation requires Nud1 phosphorylation.
(A and B) Dbf2 kinase activity and cell cycle progression in *GAL-nud1-44 NUD1* (A29878) and *GAL-nud1-44 nud1-42A* (A29881) cells. Cells were arrested in G₁ with α-factor and released under conditions in which *nud1-44* is inactive (12). (C) Mob1 localization in anaphase *NUD1* (A29453) and *nud1-3A* (A31169) cells. (D) Mob1 localization in anaphase *NUD1* (A24631) and *nud1-39A* (A31477) cells. DAPI, differential interference contrast. (E) Growth of 10-fold serial dilutions of A2587, A29248, A29685, A29500, and A32295 cells on YEP plates containing either galactose (YEPRG) or glucose (YEPD) (12).

---

### References
1. 
2. 
3. 
4. 
5. 
6. 
7. 
8. 
9. 
10. 
11. 
12. 

---

### Affiliations
$^{1}$David H. Koch Institute for Integrative Cancer Research, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.  
$^{2}$Howard Hughes Medical Institute, Massachusetts Institute of Technology, Cambridge, MA 02139, USA.  
$^{3}$Division of Molecular Structure, MRC National Institute for Medical Research, The Ridgeway, London NW7 1AA, UK.  
$^{4}$Department of Molecular, Cellular, and Developmental Biology, UCB 347, University of Colorado, Boulder, CO 80309, USA.  
$^{5}$Department of Chemical Physiology, Scripps Research Institute, La Jolla, CA 92037, USA.

*Present address: Division of Hematology and Medical Oncology and OHSU Knight Cancer Institute, Oregon Health and Science University, Portland, OR 97239, USA.

†Corresponding author. E-mail: angelika@mit.edu

Thus, the *nud1-42A* allele is defective in recruitment of Dbf2-Mob1 to SPBs and astral microtubule anchorage (II).

Further analyses (12) (fig. S3, A to C) revealed that Nud1 T78 was especially critical for MEN signaling, with two additional residues, S53 and S63, contributing to this function. A *NUD1* allele carrying alanine substitutions of S53, S63, and T78 (*nud1-39A* allele; table S1) facilitated normal Mob1 localization and restored viability to cells expressing the *GAL-nud1-44* allele under restrictive conditions (Fig. 1, D and E). *nud1-39A* cells did exhibit a 2-min delay in exit from mitosis (fig. S5B), suggesting that other phosphorylation sites in Nud1 play a very minor role in MEN activation. *nud1-39A* cells also harbored mispositioned anaphase spindles and detached astral microtubules (fig. S5, A, C, and D). The spindle positioning and astral microtubule anchoring defect of *nud1-39A* cells was suppressed by forcing localization of the astral microtubule anchor Spc72 to SPBs (fig. S5E), suggesting that *nud1-39A* cells are defective in Spc72 recruitment. Indeed, Spc72 localization to SPBs was impaired in *nud1-39A* cells (fig. S5F). These results distinguish the two Nud1 functions: MEN activation and astral microtubule anchorage.

Localization of Mob1 to SPBs was disrupted in *nud1-3A* cells as it was in *nud1-42A* cells (Fig.

1C), but astral microtubule organization was not affected (fig. S5A). In contrast, a *NUD1* allele in which all mitotic phosphorylation sites were mutated to alanine with the exception of S53, S63, and T78 (*nud1-39A* allele; table S1) facilitated normal Mob1 localization and restored viability to cells expressing the *GAL-nud1-44* allele under restrictive conditions (Fig. 1, D and E). *nud1-39A* cells did exhibit a 2-min delay in exit from mitosis (fig. S5B), suggesting that other phosphorylation sites in Nud1 play a very minor role in MEN activation. *nud1-39A* cells also harbored mispositioned anaphase spindles and detached astral microtubules (fig. S5, A, C, and D). The spindle positioning and astral microtubule anchoring defect of *nud1-39A* cells was suppressed by forcing localization of the astral microtubule anchor Spc72 to SPBs (fig. S5E), suggesting that *nud1-39A* cells are defective in Spc72 recruitment. Indeed, Spc72 localization to SPBs was impaired in *nud1-39A* cells (fig. S5F). These results distinguish the two Nud1 functions: MEN activation and astral microtubule anchorage.

Next, we sought to identify the kinase(s) that phosphorylates Nud1 S53, S63, and T78. Cdc15 is a likely candidate because it, like Nud1 phosphorylation, is essential for Dbf2-Mob1 SPB localization (fig. S6) (13). Mob1 coimmunoprecipitated with Nud1 in anaphase-arrested cells in which *CDC15* is active (*cdc14-3* cells) but not in cells in which *CDC15* is inactive (*cdc14-3 cdc15-2* cells; Fig. 2A). Cdc15 kinase activity was also sufficient for localization of Dbf2-Mob1 to SPBs. Constitutive targeting of Cdc15 to SPBs by fusing *CDC15* to the SPB (*CDC15-SPB*) (3) led to recruitment of Dbf2-Mob1 to SPBs soon after expression of *CDC15-SPB* and induced premature phosphorylation of Nud1 (Fig. 2B and fig. S7). To directly test whether Cdc15 phosphorylates Nud1 in vivo, we raised phospho-specific antibodies to S53, S63, and T78 (fig. S8). The antibodies bound to wild-type Nud1 only in anaphase cells (Fig. 2C). T78 phosphorylation depended on *CDC15*, S63 phosphorylation was partially dependent, and S53 phosphorylation was not (Fig. 2D). Consistent with this, premature targeting of Cdc15 to SPBs was sufficient to cause phosphorylation of Nud1 T78 and S63, but not S53 (Fig. 2E). In vitro, Cdc15 phosphorylated a recombinant N-terminal fragment of Nud1 (amino acids 1 to 150) but not the *nud1-3A* protein (Fig. 2F). Thus, Cdc15 phosphorylates

![Figure 2](#fig-2)

**Fig. 2. Phosphorylation of Nud1 by Cdc15 in anaphase promotes Mob1 recruitment to SPBs.** (A) Nud1-3V5 was immunoprecipitated from anaphase-arrested A31602 and A31661 cells, and bound Mob1 (IPα-HA), Nud1 (IPα-V5), Nud1 phosphorylated at T78 (IPα-pT78), total Mob1 (input α-HA), and total Nud1 (input α-V5) were examined. (B) Localization of Cdc15 and Mob1 in G1/S, metaphase, and anaphase *MET3-CDC15-SPB MOB1-mCherry* (A28499) cells grown in medium lacking methionine to induce expression of *CDC15-SPB*. (C) Phosphorylation of Nud1 T78 (α-pT78), S63 (α-pS63), S53 (α-pS53), and total Nud1 during a synchronized cell cycle (A24513). Percent metaphase (% meta) and anaphase (% ana) cells are shown. (D) Phosphorylation of Nud1 as in Fig. 2C in anaphase-arrested *NUD1-3V5 cdc14-3* (A29851) and *NUD1-3V5 cdc14-3 cdc15-2* (A31661) cells. (E) Phosphorylation of Nud1 as in Fig. 2C in S-phase-arrested *NUD1-3V5* (A24513) and *NUD1-3V5 MET3-CDC15-SPB* (A31422) cells (12). *CDC15-SPB* expression was induced as in Fig. 2B. (F) Phosphorylated Nud1, immunoprecipitated Cdc15 [Cdc15 IP; wild type (WT): A24957; kinase-dead (KD) Cdc15: A30371], and GST fusion protein substrate added to the kinase reaction (Substrate) are shown.
Nud1 on residues T78 and S63 during anaphase to promote binding of Dbf2-Mob1 to Nud1.

Mob1 bound to SPBs in the absence of Dbf2 (Fig. 3A), but Dbf2 did not bind to SPBs in the absence of Mob1 (Fig. 3B). This observation, together with the finding that Mob1 and Nud1 form a complex in a CDC15-dependent manner (Fig. 2A), suggests that Mob1 binds Nud1 in a phosphorylation-dependent manner. To test this, we assayed the interaction of recombinant Mob1 with Nud1 phosphopeptides in vitro. Glutathione S-transferase (GST)-Mob1 (amino acids 79 to 314) that contains the conserved Mob1 core but not the N terminus unique to yeast (14) preferentially bound to phosphorylated T78 Nud1 peptides (Fig. 3C). Oriented peptide library screening (12) revealed that Mob1 displayed a preference for Y or F in the –2 position and preferred aliphatic, hydrophobic, or R residues in the +1 to +4 positions. The Nud1 protein sequence surrounding T78 shows features consistent with these results (fig. S9, A to C). We measured binding of GST-Mob1 to filter-bound peptides predicted to be optimal ligands on the basis of the peptide library screening. Mob1 bound to these optimal peptides in a phosphorylation-specific manner (Fig. 3D). The binding of Mob1 to the optimal ligand was stronger than that to the Nud1 pT78 peptide (dissociation constant \(K_{\mathrm{d}}\) of 174 nM and 2.4 μM, respectively; fig. S9, D and E). These values are similar to those observed for the interaction of FHA phosphopeptide binding domains with their optimal and physiologic peptides (15, 16). Thus, Mob1 directly binds Nud1 T78 in a phosphorylation-dependent manner.

We attempted to crystallize human (hMob1) and yeast Mob1 as complexes with the Nud1

pT78 and optimal phosphopeptides. Of these combinations, hMob1 bound to the peptide TVARIYHpSVVRYAPS yielded crystals of sufficient quality for structure determination and refinement at a resolution of 2.1 Å (table S4). The phosphopeptide binds across a shallow pocket on the Mob1 surface (Fig. 4A and fig. S10A). The structural basis for selection of aromatic residues in the pSer/pThr –2 position (fig. S9, A and B) is explained by the packing of the phenolic side chain of Tyr –2 against the aliphatic portion of K84. In yeast Mob1, this lysine is substituted by a proline, which would also make favorable van der Waals interactions. Discrimination for small- to medium-sized hydrophobic residues at the +1 and +2 positions is also explained by their interactions with a largely conserved hydrophobic depression on the Mob1 surface.

The structure explains the phospho-dependence of Mob1 binding through extensive interactions of the phosphoserine with three basic residues (K132, R133, and R136) that form part of a positively charged pocket located on the face opposite to that of the Dbf2-interacting surface. In yeast Mob1, three structurally equivalent arginine residues, R253, R254, and R257, form the conserved basic pocket and directly coordinate a sulfate ion derived from the crystallization mother liquor (Fig. 4B and fig. S10B) (14). Thus, the Mob1 protein family contains a distinct class of phosphoserine-threonine binding domains.

To test whether these arginine residues are important for Mob1 binding to Nud1, we assayed the interaction of GST-mob1-R253A,R254A, R257A (henceforth mob1-3RA) with phosphopeptides. The mob1-3RA protein was properly folded (fig. S11A), but failed to interact with

Nud1 pT78 peptides (Fig. 4C). Thus, the basic pocket of Mob1 mediates phosphopeptide binding. Phosphopeptide binding was also essential for Mob1 function in vivo. The mob1-R→A proteins interacted with Dbf2 (fig. S11B) but did not localize to SPBs (Fig. 4D and fig. S11C) nor support viability (Fig. 4E and fig. S11D). Furthermore, these cells lacked Dbf2 kinase activity and failed to release Cdc14 from the nucleus or to exit from mitosis (Fig. 4F). This cell cycle arrest was not suppressed by a dominant, constitutively active Dbf2 allele (\(DBF2-HyA\); fig. S11E) (17). Thus, even if the defect in Dbf2 activation is corrected in mob1-3RA cells, the failure of mob1-3RA to bind phosphopeptides and presumably target Dbf2 to its relevant substrates results in an inability to exit from mitosis. Indeed, the phosphopeptide binding property of Mob1 may also be critical for targeting Dbf2-Mob1 to its substrates. Mob1 binds experimentally verified phosphorylation sites in the Dbf2-Mob1 substrate Hof1 (fig. S12) (18).

Our results show that MEN signaling requires an unusual two-step process (Fig. 4G). Rather than directly activating its downstream kinase Dbf2-Mob1, Cdc15 first creates phospho-docking sites on the MEN scaffold Nud1. Nud1 phosphorylation recruits Dbf2-Mob1 to SPBs followed by Cdc15-dependent activation of Dbf2-Mob1. As Cdc15 alone is sufficient to activate Dbf2-Mob1 in vitro (6), why do cells use a two-step mechanism for Dbf2-Mob1 activation? A two-step scaffold-assisted mechanism of activating downstream kinases may afford many advantages. In addition to allowing for greater regulatory power, this mechanism mitigates the “biphasic effect” in which scaffold proteins can exhibit concentration-

![Figure 3](https://www.science.org/cms/asset/3aebc7e0-4f8f-4d8f-bc8f-3aebc7e04f8f/figure_3.png)

**Fig. 3. Phosphorylation-dependent binding of Mob1 to Nud1.** (A) Mob1 localization in anaphase MOB1-eGFP (A24631) and MOB1-eGFP \(dbf2 \Delta\) TAB6-1 (A32656) cells. TAB6-1 is a dominant active CDC14 allele that was necessary to keep \(dbf2 \Delta\) cells alive (20). (B) Dbf2 localization in anaphase DBF2-eGFP (A29921) and DBF2-eGFP GAL-mob1-77 (A32654) cells. eGFP, enhanced green fluorescent protein. (C) Bead-bound phosphorylated (pT78, pS53, or pS63) or nonphosphorylated (T78, S53, or S63) Nud1 peptides were incubated with GST-Mob1 and input and eluates analyzed. (D) Binding of GST-Mob1 to an array of peptide spots. Abbreviations for the amino acid residues are as follows: A, Ala; H, His; I, Ile; R, Arg; S, Ser; T, Thr; V, Val; and Y, Tyr.
dependent titration effects (19). Increasing concentrations of scaffold proteins initially favor interaction of partner proteins but, at higher concentrations, will sequester partner proteins into separate complexes. The dependency of terminal kinase binding on scaffold phosphorylation may ensure that terminal kinases only engage in complexes in which their upstream kinases are bound and active. Such a protective mechanism predicts that MEN signaling ought to be resistant to the overexpression of individual signaling components, and this is what we observe (fig. S13).

A two-step mechanism may also facilitate signal transmission and effector activation at

![Figure 4](#fig4)

Fig. 4. Mob1 binding to phospho-Nud1 through a conserved basic pocket. (A) Structure of the hMob1-phosphopeptide complex. The peptide is shown as yellow sticks and hMob1 is shown in ribbon representation overlaid with a transparent electrostatic potential surface. Surface electrostatic potential between –10 kT (red) and +10 kT (blue) is shown. (B) View of the basic pocket of Mob1. pSer is coordinated by hMob1 K132, R133, and R136, and the SO₄²⁻ ion by yeast Mob1 R253, R254, and R257. (C) Binding of phosphorylated (pT78) or nonphosphorylated (T78) Nud1 peptides to GST-Mob1 or GST-mob1-3RA as in Fig. 3C. (D) Mob1 localization in anaphase A31500 and A31504 cells. (E) Growth of A2587, A32452, A32586, and A32589 cells on YEPRG and YEPD plates as in Fig. 1E. (F) Dbf2 kinase activity and cell cycle progression in MOB1 GAL-mob1-77 (A32818) and mob1-3RA GAL-mob1-77 (A32823) cells grown as in Fig. 1A. (G) Model for Dbf2-Mob1 activation. (H) Growth of A2587, A5588, A32336, A33289, A33551, and A33285 cells on YEPD+methionine, CSM-methionine+glucose, or CSM-methionine+raffinose and galactose (CSMRG) plates as in Fig. 1E. (I) Anaphase kinetics in A1411, A33291, A32668, A33549, A33287, and A33283 cells. Cells were arrested in G₁ with α-factor in YEPRaff-methionine medium and released into CSM-methionine+raffinose medium to induce expression of the Mob1-Nud1 fusion. Galactose was added (arrow) to induce expression of GAL-TEV1.
distinct sites in the cell. Dbf2-Mob1 activated by Cdc15 at SPBs must phosphorylate Cdc14 in the nucleolus. This predicts that the Mob1-Nud1 interaction must be dynamic. In agreement, fusion of *MOB1* to *NUD1* (*MOB1-NUD1*) resulted in the constitutive localization of Dbf2 to SPBs (fig. S14, A and B) but, despite the proper localization pattern, did not restore viability to *mob1-77* cells at restrictive conditions (fig. S14C). Indeed, expression of *MOB1-NUD1* was lethal in wild-type cells because Cdc14 could not be released from the nucleolus (fig. S14, D to F). This lethality appears to result from failure of Dbf2-Mob1 to dissociate from SPBs because a Mob1-Nud1 fusion in which Mob1 could be proteolytically released from Nud1 restored proper MEN regulation (Fig. 4, H and I). Furthermore, lethality caused by expression of *MOB1-NUD1* was not suppressed by expression of a dominant, constitutively active Dbf2-HyA allele (fig. S14G), indicating that even hyperactive Dbf2 that cannot dissociate from SPBs is not functional. During MEN signaling, dynamic interaction between Mob1 and Nud1 may be achieved by Cdc15 first generating a high-affinity scaffold docking site for Dbf2-Mob1. Phosphorylation of Dbf2-Mob1 by Cdc15 could then, in addition to activating the kinase, decrease affinity to the

Nud1 scaffold, thereby promoting the dissociation of Dbf2-Mob1 from SPBs.

In this study, we have uncovered a distinct mechanism of scaffold-assisted activation of a kinase cascade. We propose that the two-step activation of terminal kinases may be a common feature of scaffold-assisted signaling that provides a potentially general means to avoid the biphasic effect and organize signaling pathways in which the site of pathway activation differs from the essential site of action.

### References and Notes

1. A. Hergovich, B. A. Hemmings, *Semin. Cell Dev. Biol.* **23**, 794 (2012).
2. F. Stegmeier, A. Amon, *Annu. Rev. Genet.* **38**, 203 (2004).
3. J. M. Rock, A. Amon, *Genes Dev.* **25**, 1943 (2011).
4. M. Valerio-Santiago, F. Monje-Casas, *J. Cell Biol.* **192**, 599 (2011).
5. A. K. Caydasi, G. Pereira, *Exp. Cell Res.* **318**, 1421 (2012).
6. A. S. Mah, J. Jang, R. J. Deshaies, *Proc. Natl. Acad. Sci. U.S.A.* **98**, 7325 (2001).
7. M. C. Good, J. G. Zalatan, W. A. Lim, *Science* **332**, 680 (2011).
8. H. Maekawa, C. Priest, J. Lechner, G. Pereira, E. Schiebel, *J. Cell Biol.* **179**, 423 (2007).
9. C. J. Park *et al.*, *Eukaryot. Cell* **7**, 444 (2008).
10. J. M. Keck *et al.*, *Science* **332**, 1557 (2011).
11. U. Gruneberg, K. Campbell, C. Simpson, J. Grindlay, E. Schiebel, *EMBO J.* **19**, 6475 (2000).
12. Materials and methods are available as supplementary materials on Science Online.
13. C. König, H. Maekawa, E. Schiebel, *J. Cell Biol.* **188**, 351 (2010).
14. S. Mrkobrada, L. Boucher, D. F. Ceccarelli, M. Tyers, F. Sicheri, *J. Mol. Biol.* **362**, 430 (2006).
15. D. Durocher *et al.*, *Mol. Cell* **6**, 1169 (2000).
16. Z. Ding *et al.*, *Biochemistry* **46**, 2684 (2007).
17. M. Geymonat, A. Spanos, G. de Bettignies, S. G. Sedgwick, *J. Cell Biol.* **187**, 497 (2009).
18. A. E. Elia, L. C. Cantley, M. B. Yaffe, *Science* **299**, 1228 (2003).
19. A. Levchenko, J. Bruck, P. W. Sternberg, *Proc. Natl. Acad. Sci. U.S.A.* **97**, 5818 (2000).
20. W. Shou *et al.*, *Mol. Cell* **8**, 45 (2001).

**Acknowledgments:** We thank I. Cheeseman, F. Solomon, and the Amon lab for comments and E. Schiebel for reagents. Coordinates and structure factors for the hMob1-phosphopeptide complex have been deposited with the Protein Data Bank (PDBID: 4JIZ). Supported by the NIH (GM056800 to A.A.; CA112967 and ES015339 to M.B.Y; P41 RR011823 to J.R.Y. III and T. N. Davis, principal investigator; and GM51312 to M.W.); MRC UK (U117584228) to S.J.S; MRC centenary fellowship to L.S.; NIH F32 GM086038 to J.M.K.; and NSF Predoctoral Fellowship to J.M.R. A.A. is an investigator of the Howard Hughes Medical Institute.

**Supplementary Materials**

www.sciencemag.org/cgi/content/full/science.1235822/DC1

Materials and Methods

Figs. S1 to S14

Tables S1 to S4

References

29 January 2013; accepted 13 March 2013

Published online 11 April 2013;

10.1126/science.1235822

---

# ATAXIN-2 Activates PERIOD Translation to Sustain Circadian Rhythms in *Drosophila*

Chunghun Lim* and Ravi Allada†

Evidence for transcriptional feedback in circadian timekeeping is abundant, yet little is known about the mechanisms underlying translational control. We found that ATAXIN-2 (ATX2), an RNA-associated protein involved in neurodegenerative disease, is a translational activator of the rate-limiting clock component PERIOD (PER) in *Drosophila*. ATX2 specifically interacted with TWENTY-FOUR (TYF), an activator of PER translation. RNA interference–mediated depletion of *Atx2* or the expression of a mutant ATX2 protein that does not associate with polyadenylate-binding protein (PABP) suppressed behavioral rhythms and decreased abundance of PER. Although ATX2 can repress translation, depletion of *Atx2* from *Drosophila* S2 cells inhibited translational activation by RNA-tethered TYF and disrupted the association between TYF and PABP. Thus, ATX2 coordinates an active translation complex important for PER expression and circadian rhythms.

Daily oscillations of gene expression underlie time-keeping in animals (I). A transcriptional feedback network of core clock genes drives circadian transcription, whereas posttranscriptional modification of the clock genes is important for precise timing of the clock (2). Emerging evidence suggests that other modes of posttranscriptional regulation play more pivotal roles in clock mechanisms than previously appreciated (3–5). Yet the molecular mechanisms of this important facet of the clock remain to be elucidated. The product of the *Drosophila* gene *twenty-four* (*tyf*) is required for the efficient translation of the core clock protein PERIOD (PER) (6). *tyf* mutants display weak behavioral rhythms with decreased abundance of PER specifically in pacemaker neurons. TYF associates with polyadenylate [poly(A)]–binding protein (PABP) as well as with *per* RNA. TYF can activate translation of reporter RNAs in vitro. TYF lacks known functional domains, including RNA-binding domains. We used a proteomic strategy to identify the fly homolog of

the neurodegenerative disease gene ATAXIN-2 (ATX2) as a translational coactivator of TYF. TYF can stimulate translation when tethered to a reporter RNA (6). We used an RNA-tethering assay in *Drosophila* S2 cells to map a domain necessary and sufficient for such activation to the C-terminal 290 amino acids of TYF (Fig. 1A, TYF-C5). To identify components that interact with this domain, we affinity-purified TYF complexes in S2 cells. Two proteins specifically associated with TYF-C5 and full-length TYF (Fig. 1, B and C, arrows). Mass spectrometric analyses identified the larger and smaller molecular-size proteins as ATX2 and PABP, respectively. We confirmed that TYF-C5 was necessary and sufficient to specifically associate with ATX2 and PABP (Fig. 1D).

Polyglutamine expansion in human ATX2 (also ATXN2) leads to neurodegenerative diseases such as spinocerebellar ataxia type 2 (SCA2) (7–9) or an increased risk of amyotrophic lateral sclerosis and Parkinsonism (10, 11). ATX2 associates with polyribosomes through its N-terminal like-Sm (Lsm) and like-Sm-associated (LsmA) domains or C-terminal poly(A)-binding protein-interacting motif 2 (PAM2) domain (12, 13). The ATX2-PABP interaction is conserved from yeast to human (12–14). TYF and PABP separately bound to different regions of ATX2 in a ribonuclease-insensitive manner (figs. S1 and S2A). Deletion of the PAM2 region abolished binding of PABP but not TYF to ATX2 (fig. S2A). A PABP-independent interaction of TYF with ATX2 was also observed in extracts of fly heads (fig. S2B). To test the function of the ATX2-TYF interaction, we generated

Department of Neurobiology, Northwestern University, Evanston, IL 60208, USA.

*Present address: School of Nano-Bioscience and Chemical Engineering, Ulsan National Institute of Science and Technology, Ulsan 689-798, Republic of Korea.

†Corresponding author. E-mail: r-allada@northwestern.edu

www.sciencemag.org SCIENCE VOL 340 17 MAY 2013 875
Activation of the Yeast Hippo Pathway by  
Phosphorylation-Dependent Assembly of Signaling Complexes  
Jeremy M. Rock *et al.*  
Science 340, 871 (2013);  
DOI:10.1126/science.1235822  

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by [clicking here](https://www.sciencemag.org).

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines [here](https://www.sciencemag.org).

The following resources related to this article are available online at www.sciencemag.org (this information is current as of April 21, 2015):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:  
[http://www.sciencemag.org/content/340/6134/871.full.html](http://www.sciencemag.org/content/340/6134/871.full.html)

Supporting Online Material can be found at:  
[http://www.sciencemag.org/content/suppl/2013/04/10/science.1235822.DC1.html](http://www.sciencemag.org/content/suppl/2013/04/10/science.1235822.DC1.html)

A list of selected additional articles on the Science Web sites related to this article can be found at:  
[http://www.sciencemag.org/content/340/6134/871.full.html#related](http://www.sciencemag.org/content/340/6134/871.full.html#related)

This article cites 19 articles, 11 of which can be accessed free:  
[http://www.sciencemag.org/content/340/6134/871.full.html#ref-list-1](http://www.sciencemag.org/content/340/6134/871.full.html#ref-list-1)

This article has been cited by 10 articles hosted by HighWire Press; see:  
[http://www.sciencemag.org/content/340/6134/871.full.html#related-urls](http://www.sciencemag.org/content/340/6134/871.full.html#related-urls)

This article appears in the following subject collections:  
Cell Biology  
[http://www.sciencemag.org/cgi/collection/cell_biol](http://www.sciencemag.org/cgi/collection/cell_biol)

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2013 by the American Association for the Advancement of Science; all rights reserved. The title *Science* is a registered trademark of AAAS.
