
JACS
JOURNAL OF THE AMERICAN CHEMICAL SOCIETY

A Previously Unrecognized Kanosamine Biosynthesis Pathway in Bacillus subtilis

Natasha D. Vetter,$^{\dagger}$ David M. Langill,$^{\dagger}$ Shazia Anjum,$^{\dagger,\ddagger}$ Julie Boisvert-Martel,$^{\dagger}$ Rajendra C. Jagdhane,$^{\dagger}$ Egiroh Omene,$^{\dagger}$ Hongyan Zheng,$^{\dagger,\|}$ Karin E. van Straaten,$^{\dagger}$ Isaac Asiamah,$^{\ddagger}$ Ed S. Krol,$^{\ddagger}$ David A. R. Sanders,$^{\dagger}$ and David R. J. Palmer$^{*,\dagger}$

$^{\dagger}$Department of Chemistry, and $^{\ddagger}$College of Pharmacy and Nutrition, University of Saskatchewan, 110 Science Place, Saskatoon, SK, Canada, S7N 5C9

S Supporting Information

---

**ABSTRACT:** The *ntd* operon in *Bacillus subtilis* is essential for biosynthesis of 3,3′-neotrehalosadiamine (NTD), an unusual nonreducing disaccharide reported to have antibiotic properties. It has been proposed that the three enzymes encoded within this operon, NtdA, NtdB, and NtdC, constitute a complete set of enzymes required for NTD synthesis, although their functions have never been demonstrated *in vitro*. We now report that these enzymes catalyze the biosynthesis of kanosamine from glucose-6-phosphate: NtdC is a glucose-6-phosphate 3-dehydrogenase, NtdA is a pyridoxal phosphate-dependent 3-oxo-glucose-6-phosphate:glutamate aminotransferase, and NtdB is a kanosamine-6-phosphate phosphatase. None of these enzymatic reactions have been reported before. This pathway represents an alternate route to the previously reported pathway from *Amycolatopsis mediterranei* which derives kanosamine from UDP-glucose.

(PLP)-dependent 3-oxo-glucose-6-phosphate:glutamate aminotransferase, and NtdB, a kanosamine-6-phosphate phosphatase. Kanosamine, or 3-amino-3-deoxy-D-glucose, is known to have antibiotic${}^{1,2}$ and antifungal${}^{3}$ properties. A kanosamine biosynthetic pathway starting from UDP-glucose was first proposed for *Bacillus pumilus*.${}^{4}$ Later, such a pathway was characterized in *Amycolatopsis mediterranei*,${}^{5,6}$ based on incubation of cell-free extracts of the bacteria with UDP-glucose, NAD, and glutamine; an incubation with 6,6′-2H₂-UDP-glucose resulted in 6,6′-2H₂-kanosamine in 5% yield.${}^{5}$ Most recently, an operon in *Bacillus cereus* UW85, named the *kab* operon, was proposed to encode enzymes that synthesize kanosamine.${}^{7}$ This hypothesis was based on the resemblance of the *kab* operon to the *ntd* operon in *B. subtilis*, although the *ntd* operon was so named because it was proposed to synthesize the disaccharide NTD, which is made up of two kanosamine residues connected by a 1,1′-α,β linkage. *B. cereus* UW85 is known to produce kanosamine and not NTD,${}^{8}$ but the function of the *kab* operon has not been experimentally investigated.

The *ntd* operon, as described by Inaoka et al., is required for the biosynthesis of NTD,${}^{9,10}$ which is reported to have antibiotic properties including inhibition of the growth of *Staphylococcus aureus*.${}^{11}$ The production of NTD has been reported for *Bacillus pumilus* and *Bacillus circulans*.${}^{11,12}$ Although the operon is normally dormant in *B. subtilis* such that no NTD is produced, a mutant strain in which the operon is activated secretes NTD. Moreover, when *Escherichia coli* is transformed with a plasmid bearing the operon, that bacterium also produces NTD. The authors thus concluded that the *ntd* operon “contains a complete set of genes required for NTD biosynthesis.”${}^{9}$ However, neotrehalose is not known to be produced by this organism, and the same authors point out that the addition of neotrehalose or trehalose to cultures did not result in increased NTD. This strongly suggests that NTD is synthesized by the linking of kanosamine monosaccharides or closely related derivatives. One would therefore predict that a complete set of genes necessary for NTD biosynthesis would include a nucleotidylyltransferase and/or glycosyltransferase in order to form the glycosidic bond. The three enzymes encoded within the *ntd* operon show no apparent sequence similarity to such enzymes. Instead, the encoded proteins NtdA, NtdB, and

---

Kanosamine is a naturally occurring antibiotic and component of oligosaccharide antibiotics such as kanamycin (Figure 1). Although this product has been known for many years, the biosynthetic pathway for kanosamine in bacteria has not been demonstrated unequivocally.

![Structures of kanosamine and NTD](https://i.imgur.com/1234567.png)

Figure 1. Structures of kanosamine and NTD.

Here we show that the enzymes encoded by the *ntd* operon from *Bacillus subtilis*, which were proposed to catalyze the complete synthesis of 3,3′-neotrehalosadiamine (NTD), in fact synthesize kanosamine. This is in agreement with a published proposal for the function of an homologous operon in *Bacillus cereus*, but by a different series of reactions than previously proposed. Here we show that glucose-6-phosphate is converted in three steps to kanosamine by the action of NtdC, a glucose-6-phosphate 3-dehydrogenase, NtdA, a pyridoxal phosphate

Received: January 29, 2013  
Published: April 11, 2013

© 2013 American Chemical Society 5970 dx.doi.org/10.1021/ja4010255 | J. Am. Chem. Soc. 2013, 135, 5970–5973

Journal of the American Chemical Society

NtdC are predicted by sequence alignment to be a PLP-dependent aminotransferase, a member of the HAD superfamily, and an NAD-dependent dehydrogenase, respectively. The unusual structure, interesting properties, and undetermined biosynthesis prompted us to investigate NTD further. We recently described the first chemical synthesis of NTD.¹³

We subcloned each of *ntdA*,¹⁴ *ntdB*, and *ntdC* into pET-28b and expressed them in *E. coli* BL21-Gold cells. The N-terminal hexahistidine-tagged proteins were purified by nickel-affinity chromatography. NtdC is homologous with inositol dehydrogenase (IDH) and several other NAD-dependent monosaccharide dehydrogenases. Although this made UDP-glucose a less likely candidate substrate, IDH can accommodate inositol substrates conjugated to aromatic and carbohydrate moieties,¹⁵ and therefore recognition of a nucleotidylyl sugar was an intriguing possibility. However, in our hands NtdC shows no activity with UDP-glucose using the standard UV assay monitoring the absorbance at 340 nm due to the appearance of NADH or NADPH. Glucose is a weak substrate for the enzyme (as is inositol). The strong resemblance of NtdB to phosphatases such as sucrose-6-phosphate phosphatase suggested to us that a 6-phospho-sugar might be the correct substrate for NtdC, and indeed this enzyme showed high specificity for the NAD-dependent oxidation of glucose 6-phosphate, and no NADP-dependent activity.

This information led us to the hypothesis that the three enzymes functioned as shown in Figure 2, generating

![Figure 2. Observed kanosamine pathway.](https://i.imgur.com/yourimage.png)

kanosamine rather than NTD. The proposed intermediate 3-oxo-D-glucose-6-phosphate is the hemiacetal form of a 1,3-dicarbonyl compound. Such compounds are known to absorb near 300 nm in basic aqueous solution, due to enolate and/or enone formation.¹⁶⁻¹⁸ Consistent with this, the NtdC-catalyzed reaction produced the UV spectra shown in Figure 3A. By recycling the NADH using lactate dehydrogenase and pyruvate, the absorbance at 340 nm can be removed to reveal the absorbance due to the product around 310 nm (Figure 3B). The addition of an excess of L-glutamate and purified NtdA effectively removes the absorbance at 310 nm, leaving only the NADH absorbance at 340 nm (Figure 3C). No other amino acid gave this result, demonstrating the specificity of the NtdA-catalyzed reaction.

Kanosamine was synthesized chemically by the method of Guo and Frost⁵ and converted to kanosamine 6-phosphate using hexokinase and ATP. A mixture of kanosamine-6-

![Figure 3. (A) The increase in absorbance over time in the UV-visible spectrum due to oxidation of glucose-6-phosphate and reduction of NAD by NtdC at pH 8.0. (B) The reaction in A in the presence of excess pyruvate and lactate dehydrogenase. The recycling of NADH eliminates the absorbance at 340 nm, revealing the increase in absorbance at 310 nm associated with 3-oxo-D-glucose-6-phosphate. (C) The reaction in A in the presence of excess L-glutamate and NtdA. Removal of the intermediate reveals the increasing absorbance due to NADH at 340 nm. (D) Reaction of kanosamine-6-phosphate and 2-oxoglutarate in the presence of NtdA, leading to the formation of 3-oxo-D-glucose-6-phosphate. In all cases, spectra were collected at 5-min intervals.](https://i.imgur.com/yourimage.png)

phosphate and 2-oxoglutarate in the presence of NtdA again gave rise to the absorbance at 310 nm (Figure 3D), which was not present when the reaction was performed in an excess of NtdC and NADH. This demonstrated the reversibility of the process and established the function of NtdA as shown in Figure 2. Confirmation of this was provided using reversed-phase ion-pairing HPLC with evaporative light scattering detection, with which we could observe the consumption of kanosamine-6-phosphate and the production of glucose-6-phosphate in the presence of NtdA, NtdC, 2-oxoglutarate, and NADH (Supporting Information, Figure S1).

NtdB phosphatase activity was investigated using several phosphosugar substrates, including NTD-6-phosphate, NTD-6′-phosphate, NTD-6,6′-bisphosphate, glucose-6-phosphate, glucose-1-phosphate, glucosamine-6-phosphate, and kanosamine-6-phosphate. The disaccharide analogs were synthesized by adaptation of our recently reported synthesis of NTD¹³ as described in the Supporting Information. NtdB is a very efficient catalyst of the phosphatase reaction of kanosamine-6-phosphate (*k*~cat~ = 32 ± 3 s⁻¹, *K*~m~ = 101 ± 8 μM), but has no effect on glucose-6-phosphate, indicating selectivity at the 3-position. There was a trace amount of activity (<1%) using glucosamine-6-phosphate, but no other compound tested, including the synthesized disaccharide phosphates, glucose-6-phosphate, or para-nitrophenyl phosphate, showed significant reactivity. Consistent with our results, Huang et al. pointed out that highly selective members of the HAD superfamily tend to have specificity constants >10⁵ s⁻¹ M⁻¹.¹⁹ This strongly suggests that NtdB, and by extension, the *ntd* operon, has

dx.doi.org/10.1021/ja4010255 | J. Am. Chem. Soc. 2013, 135, 5970–5973

Journal of the American Chemical Society

evolved for a particular activity, namely the biosynthesis of kanosamine.

Our results are in notable contrast to the previously described biosynthetic pathway for kanosamine. Guo and Frost as well as Arakawa et al. have described enzymes from Amycolatopsis mediterranei which generate kanosamine from UDP-glucose and shown that kanosamine is subsequently phosphorylated by a kanosamine kinase. A similar finding has since been reported in Streptomyces kanamyceticus.

Kanosamine biosynthesis was previously observed in B. pumilis. Umezawa et al. observed that kanosamine could be formed using cell-free extracts from UDP-glucose, ATP, NAD, Mg²⁺, and glutamine but also observed kanosamine synthesis from glucose and ATP (and the absence of UTP). The authors did not consider that this was due to the formation of glucose-6-phosphate, but ruled out the formation from glucose by showing that the addition of 3-oxo-D-glucose did not lead to kanosamine synthesis. In light of our results, Umezawa's observations appear consistent with the pathway of Figure 2, except that they observed that either glutamine or ammonia addition was necessary for kanosamine production, unlike the PLP-dependent glutamate reaction we observe. The use of cell-free extracts in those experiments rather than purified enzymes makes a direct comparison to our work difficult. Homologues of ntdA, ntdB, and ntdC are present in the B. pumilus genome.

Kevany et al. hypothesized that the kab operon from B. cereus UW85, which is very similar to the ntd operon, encodes a kanosamine biosynthetic pathway. In keeping with previously described kanosamine biosynthesis, they proposed that the pathway began with UDP-glucose. Specifically, KabC (which is 58% identical to NtdC, 75% sequence similarity) was proposed to be a UDP-glucose 3-dehydrogenase; KabA (58% identical to NtdA, 77% similar), a glutamine-dependent aminotransferase; and KabB (57% identical to NtdB, 74% similar), a UDP glycosyl hydrolase. There are no published experiments on the Kab enzymes, but given the similarity of the two operons, we now propose that these authors were correct in that the operon encodes enzymes that synthesize kanosamine, but incorrect in the specifics of the biosynthetic steps.

Each of the enzymatic functions reported here belong to well-established classes of biochemical reactions, but none have been observed before for these substrates. In fact, there are no reports of 3-oxo-D-glucose-6-phosphate. Attempts to synthesize this compound in our laboratory have met with failure. We did synthesize 3-oxo-D-glucose, and in alkaline aqueous buffer it absorbs strongly at 310 nm (Supporting Information, Figure S3). We were also able to detect the presence of a product consistent with 3-oxo-D-glucose-6-phosphate in the lactate dehydrogenase-coupled NtdC-catalyzed reaction of glucose-6-phosphate using quadrupole ion trap mass spectrometry. The starting material and product fragmented with the loss of phosphate (m/z 96.97); neutral loss experiments showed a loss of 162 mass units for glucose-6-phosphate and 160 mass units for the reaction product, which was consistent with the precursor ion investigation of the phosphate ion fragment (Supporting Information, Figures S8–S10). The apparent lability of 3-oxo-D-glucose-6-phosphate suggests that this intermediate does not accumulate and might be sequestered between the active sites of an NtdC-NtdA complex.

An important implication of these results is that the biosynthesis of NTD must require additional enzymatic activities. Such activities are likely to include a kanosamine phosphoisomerase, a nucleotidylyltransferase, and a glycosyl-

transferase. It is also likely that already identified proteins encoded within the B. subtilis genome are responsible, but the catalytic promiscuity of these enzymes is as yet unrevealed. Jakeman's observation of promiscuity among bacterial nucleotidylyltransferases Cps2L and RmlA3, which tolerate a variety of substituents at the 3-position, supports this hypothesis.

In summary, we have found the ntd operon in B. subtilis to encode a pathway for kanosamine synthesis. The operon would be better named the kab operon as proposed for the homologous operon from B. cereus UW85. The enzymes appear to be efficient and highly selective. This pathway represents a distinct strategy for kanosamine synthesis from that previously described.

---

### ASSOCIATED CONTENT

#### Supporting Information
Chemical syntheses, spectral data, molecular biology, protein purification, assay conditions, HPLC conditions and chromatogram. This material is available free of charge via the Internet at http://pubs.acs.org.

---

### AUTHOR INFORMATION

**Corresponding Author**
dave.palmer@usask.ca

**Present Addresses**
$^{§}$S.A.: Cholistan Institute of Desert Studies, The Islamia University of Bahawalpur, Pakistan  
$^{\|}$H.Z.: Department of Chemical Engineering and Applied Chemistry, University of Toronto  

**Notes**
The authors declare no competing financial interest.

---

### ACKNOWLEDGMENTS
This work was supported by a Saskatchewan Health Research Foundation Group grant to the Molecular Design Research Group, an NSERC Undergraduate Summer Research Award to E.O., and the University of Saskatchewan. Prof. Kozo Ochi provided ntdABC. We thank Mr. Ken Thoms and the other staff of the Saskatchewan Structural Sciences Centre.

---

### REFERENCES
(1) Iwai, Y.; Tanaka, H.; Oiwa, R.; Shimizu, S.; Omura, S. *Biochim. Biophys. Acta* **1977**, 498, 223.  
(2) Muhizi, T.; Grelier, S.; Coma, V. *J. Agric. Food Chem.* **2009**, 57, 8770.  
(3) Janiak, A. M.; Milewski, S. *Medical Mycology* **2001**, 39, 401.  
(4) Umezawa, S.; Shibahar., S.; Omoto, S.; Takeuchi, T.; Umezawa, H. *J. Antibiot.* **1968**, 21, 485.  
(5) Guo, J. T.; Frost, J. W. *J. Am. Chem. Soc.* **2002**, 124, 10642.  
(6) Arakawa, K.; Muller, R.; Mahmud, T.; Yu, T. W.; Floss, H. G. J. *Am. Chem. Soc.* **2002**, 124, 10644.  
(7) Kevany, B. M.; Rasko, D. A.; Thomas, M. G. *Appl. Environ. Microbiol.* **2009**, 75, 1144.  
(8) Milner, J. L.; SiloSuh, L.; Lee, J. C.; He, H. Y.; Clardy, J.; Handelsman, J. *Appl. Environ. Microbiol.* **1996**, 62, 3061.  
(9) Inaoka, T.; Takahashi, K.; Yada, H.; Yoshida, M.; Ochi, K. *J. Biol. Chem.* **2004**, 279, 3885.  
(10) Inaoka, T.; Ochi, K. *J. Bacteriol.* **2007**, 189, 65.  
(11) Tsuno, T.; Ikeda, C.; Numata, K.; Tomita, K.; Konishi, M.; Kawaguchi, H. *J. Antibiot.* **1986**, 39, 1001.  
(12) Numata, K. I.; Satoh, F.; Hatori, M.; Miyaki, T.; Kawaguchi, H. *J. Antibiot.* **1986**, 39, 1346.

(13) Anjum, S.; Vetter, N. D.; Rubin, J. E.; Palmer, D. R. J. Tetrahedron 2013, 69, 816.

(14) van Straaten, K. E.; Langill, D. M.; Palmer, D. R. J.; Sanders, D. A. R. Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun. 2009, 65, 426.

(15) Daniellou, R.; Phenix, C. P.; Tam, P. H.; Laliberte, M. C.; Palmer, D. R. J. Org. Biomol. Chem. 2005, 3, 401.

(16) Barycki, J. J.; O'Brien, L. K.; Strauss, A. W.; Banaszak, L. J. J. Biol. Chem. 2000, 275, 27186.

(17) Iglesias, E. New J. Chem. 2002, 26, 1352.

(18) Barash, I.; Pupkin, G.; Netzer, D.; Kashman, Y. Plant Physiol. 1982, 69, 23.

(19) Huang, H.; Patskovsky, Y.; Toro, R.; Farelli, J. D.; Pandya, C.; Almo, S. C.; Allen, K. N.; Dunaway-Mariano, D. Biochemistry 2011, 50, 8937.

(20) Park, J. W.; Park, S. R.; Nepal, K. K.; Han, A. R.; Ban, Y. H.; Yoo, Y. J.; Kim, E. J.; Kim, E. M.; Kim, D.; Sohng, J. K.; Yoon, Y. J. Nat. Chem. Biol. 2011, 7, 843.

(21) Huestis, M. P.; Aish, G. A.; Hui, J. P. M.; Soo, E. C.; Jakeman, D. L. Org. Biomol. Chem. 2008, 6, 477.
