
in bats flying in their natural habitat. Further, navigation requires more than the hippocampal spatial signal: It also entails decision-making, goal-directed behaviors, sensory-motor integration, and other cognitive processes (mediated by brain structures such as the striatum, cerebellum, and prefrontal and parietal cortices). Thus, to elucidate the neural basis of real-life navigation in bats (27, 29), it would be essential to record neural activity from the hippocampal formation (and additional brain structures) in bats navigating over distances of kilometers.

### References and Notes

1. P. Andersen, R. G. Morris, D. Amaral, T. Bliss, J. O'Keefe, *The Hippocampus Book* (Oxford Univ. Press, New York, 2007).
2. E. I. Moser, E. Kropff, M.-B. Moser, *Annu. Rev. Neurosci.* **31**, 69 (2008).
3. N. Ulanovsky, C. F. Moss, *Nat. Neurosci.* **10**, 224 (2007).
4. N. Ulanovsky, C. F. Moss, *Hippocampus* **21**, 150 (2011).
5. M. M. Yartsev, M. P. Witter, N. Ulanovsky, *Nature* **479**, 103 (2011).
6. T. Ono, K. Nakamura, H. Nishijo, S. Eifuku, *J. Neurophysiol.* **70**, 1516 (1993).
7. C. G. Kentros, N. T. Agnihotri, S. Streater, R. D. Hawkins, E. R. Kandel, *Neuron* **42**, 283 (2004).
8. A. D. Ekstrom et al., *Nature* **425**, 184 (2003).
9. J. J. Knierim, B. L. McNaughton, G. R. Poe, *Nat. Neurosci.* **3**, 209 (2000).
10. N. Ulanovsky, *Curr. Biol.* **21**, R886 (2011).

11. C. D. Harvey, F. Collman, D. A. Dombeck, D. W. Tank, *Nature* **461**, 941 (2009).
12. E. J. Henriksen et al., *Neuron* **68**, 127 (2010).
13. N. Ludvig, H. M. Tang, B. C. Gohil, J. M. Botero, *Brain Res.* **1014**, 97 (2004).
14. M. A. Wilson, B. L. McNaughton, *Science* **261**, 1055 (1993).
15. S. P. Jadhav, C. Kemere, P. W. German, L. M. Frank, *Science* **336**, 1454 (2012).
16. R. Hayman, M. A. Verriotis, A. Jovalekic, A. A. Fenton, K. J. Jeffery, *Nat. Neurosci.* **14**, 1182 (2011).
17. J. J. Knierim, B. L. McNaughton, *J. Neurophysiol.* **85**, 105 (2001).
18. K. J. Jeffery, R. L. Anand, M. I. Anderson, *Exp. Brain Res.* **169**, 218 (2006).
19. Materials and methods are available as supplementary materials on Science Online.
20. This telemetry system did not allow recording of low-frequency local field potentials because of inherent high-pass limitations (19). The bat's wings hitting on the telemetry transmitter caused some recording artifacts (Fig. 1G and fig. S11), but these artifacts constituted only a small fraction of the flight time (3.6 ± 0.91% of the total flight time per day, mean ± SD). This fraction decayed to 0.11% after 9 recording days (fig. S11D), enabling us to collect neural spiking data with almost no movement-related interference.
21. A. A. Fenton et al., *J. Neurosci.* **28**, 11250 (2008).
22. S. Royer, A. Sirota, J. Patel, G. Buzsáki, *J. Neurosci.* **30**, 1777 (2010).
23. R. F. Langston et al., *Science* **328**, 1576 (2010).
24. T. J. Wills, F. Cacucci, N. Burgess, J. O’Keefe, *Science* **328**, 1573 (2010).
25. J. Koenig, A. N. Linder, J. K. Leutgeb, S. Leutgeb, *Science* **332**, 592 (2011).

26. E. Hori, E. Tabuchi, N. Matsumura, T. Ono, H. Nishijo, *Front. Behav. Neurosci.* **5**, 36 (2011).
27. N. Ulanovsky, C. F. Moss, *Proc. Natl. Acad. Sci. U.S.A.* **105**, 8491 (2008).
28. J. S. Taube, M. Shinder, *Hippocampus* **23**, 14 (2013).
29. A. Tsoar et al., *Proc. Natl. Acad. Sci. U.S.A.* **108**, E718 (2011).
30. W. M. Brown, A. Bäcker, *Neural Comput.* **18**, 1511 (2006).

**Acknowledgments:** We thank A. Treves, D. Derdikman, J.-M. Fellous, L. Las, Y. Yovel, M. Ahrens, and C. Brody for comments on the manuscript; R. Raz, K. Kamenitz, and M. Geva-Sagiv for the 3D trajectory reconstruction system; C. Stengel for the neural telemetry system; B. Pasmanter and G. Ankaoua for mechanical designs; A. Rubin and A. Averkin for assistance with neural recordings and video calibrations; S. Kaufman, T. Dayan, and S. Dror for bat training; M. Weinberg and A. Tuval for veterinary oversight; C. Ra'anani and R. Eilam for histology; and M. P. Witter for advice on reconstruction of tetrode-track locations. This study was supported by research grants from the European Research Council (ERC-NEUROBAT), the Human Frontiers Science Program (HFSP RGP0062/2009-C), the Israel Science Foundation (ISF 1017/08), and the Minerva Foundation to N.U. and a Lev-Zion predoctoral excellence fellowship to M.M.Y.

**Supplementary Materials**

www.sciencemag.org/cgi/content/full/340/6130/367/DC1

Materials and Methods  
Figs. S1 to S22  

References  

18 January 2013; accepted 21 February 2013  

10.1126/science.1235338  

---

# Developmental Decline in Neuronal Regeneration by the Progressive Change of Two Intrinsic Timers

Yan Zou,$^{1\ast}$ Hui Chiu,$^{1\ast}$ Anna Zinovyeva,$^{2}$ Victor Ambros,$^{2}$ Chiou-Fen Chuang,$^{1\dagger}$ Chieh Chang$^{1,3\dagger}$

Like mammalian neurons, *Caenorhabditis elegans* neurons lose axon regeneration ability as they age, but it is not known why. Here, we report that *let-7* contributes to a developmental decline in anterior ventral microtubule (AVM) axon regeneration. In older AVM axons, *let-7* inhibits regeneration by down-regulating LIN-41, an important AVM axon regeneration-promoting factor. Whereas *let-7* inhibits *lin-41* expression in older neurons through the *lin-41* 3′ untranslated region, *lin-41* inhibits *let-7* expression in younger neurons through Argonaute ALG-1. This reciprocal inhibition ensures that axon regeneration is inhibited only in older neurons. These findings show that a *let-7–lin-41* regulatory circuit, which was previously shown to control timing of events in mitotic stem cell lineages, is reutilized in postmitotic neurons to control postdifferentiation events.

W e use *Caenorhabditis elegans* to study developmental decline in neuronal regeneration (Fig. 1A) (1). As in vertebrates, advancing development leads to decreased axon regenerative capacity in *C. elegans* (Fig. 1C) (2–4). The timing mechanism that controls developmental decline in neuronal regeneration is poorly understood (2–5). Because heterochronic genes are implicated in regulating developmental timing and aging in *C. elegans* (6), we hypothesized that they might regulate developmental decline in neuronal regeneration. The heterochronic pathway involves a number of microRNA-regulated posttranscriptional genetic circuits (7, 8), including an important interaction between the

$^1$Division of Developmental Biology, Cincinnati Children's Hospital Research Foundation, Cincinnati, OH 45229, USA.  
$^2$RNA Therapeutics Institute and Program in Molecular Medicine, University of Massachusetts Medical School, Worcester, MA 01605, USA. $^3$Department of Neurology and Neurosurgery, McGill University, Montreal, Quebec H3A 1B1, Canada.  
$\ast$These authors contributed equally to this work.  
$\dagger$Corresponding author. E-mail: chieh.chang@cchmc.org (C.C.); chiou-fen.chuang@cchmc.org (C.-F.C.)

let-7 microRNA and its direct target, *lin-41*, which encodes a tripartite motif (TRIM) protein (9, 10). We show here that *let-7* and *lin-41* function in postmitotic neurons to time their differentiation and postdifferentiation events. Our study reveals that the intrinsic timing mechanism that controls developmental decline in neuronal regeneration depends on the progressive increase of *let-7* and the progressive decrease of *lin-41* in neurons, with *let-7–lin-41* reciprocal inhibition having a role in this process.

MicroRNA expression is either spatially restricted or temporally regulated in neuronal development (11–14). To explore the role of microRNAs in neuronal regeneration, we examined anterior ventral microtubule (AVM) axon regeneration in mutants defective in microRNA biogenesis, *dcr-1* and *alg-1* (15). Although *dcr-1* hypomorphic mutant animals displayed normal AVM axon regeneration (fig. S1A), regenerating AVM neurons of adult *alg-1* mutants extended axons 2.5 times the length of those in adult wild-type animals (Fig. 1, B to D). In addition, regenerating axons of *alg-1* adults often displayed compact growth cones (fig. S1C). These regeneration phenotypes in adult *alg-1* mutants are reminiscent of those in wild-type animals at an earlier developmental stage (Fig. 1, B to D, and fig. S1), which suggests that *alg-1* mutations may retard a normal developmental decline in axon regeneration. Consistent with this conclusion, we observe that, although AVM axon regeneration in *alg-1* mutants is similar to that of wild-type animals at the larval L2

stage (Fig. 1, C and D), during advanced L3-YA (young adult) stages, AVM axon regeneration is significantly reduced in wild-type animals, but is unchanged in *alg-1* mutants until day 4 of the adult stage (Fig. 1, C and D).

We hypothesize that ALG-1–dependent microRNAs are likely to be expressed late to contribute to the observed developmental decline in axon regeneration. Stem-loop reverse transcription polymerase chain reaction (RT-PCR) was used to globally survey mature microRNA expression in *C. elegans* (fig. S2A). Among 90 microRNAs surveyed (fig. S2), we identified 10 late-onset microRNAs whose expression coincides with the developmental decline in axon regeneration (Fig. 2A). Among the 10 late-onset microRNAs identified, only *let-7*, *mir-84*, and *mir-241* are expressed in AVM neurons (Fig. 2, B to H, and fig. S3A). Out of those, only *let-7* maturation is affected in *alg-1* mutants (Fig. 2I and fig. S3B). In our assay using RNA preparations from staged animals, expression of the mature *let-7* microRNA is almost undetectable from embryonic to L2 stages and becomes apparent at L3 (Fig. 2A). To study specifically the expression of *let-7* in AVM, we developed a 2.9-kilobase *let-7* promoter reporter that stably expresses in AVM and whose expression levels in the whole animal at different stages correlate very well with the whole-animal stem-loop RT-PCR results (Fig. 2A and fig S3C). This *let-7* reporter is expressed at relatively low levels in AVM at L1 and L2 stages (Fig. 2, C and J) but is significantly elevated from L3 onward (Fig. 2, F and J). This indicates that *let-7* is expressed at the right place and right time to contribute to the developmental decline in AVM axon regeneration. Several lines of evidence indicate that the effect of *alg-1* mutations

on AVM axon regeneration can be attributed to *let-7* inactivation in AVM. First, *let-7* mutants display enhanced AVM axon regeneration to a similar extent to *alg-1* mutants (Fig. 2, K to M). Second, the effect of *alg-1* mutations on AVM axon regeneration is rescued by the *let-7* overexpression in AVM (Fig. 2M). Last, although the *alg-1* RNA interference (RNAi) causes enhanced AVM axon regeneration, it does not further enhance regeneration in *let-7* mutants (Fig. 2M).

The temperature-sensitive allele used in this study, *let-7(n2853ts)*, is a point mutation in the 5′ seed region of the *let-7* microRNA. *let-7(n2853ts)* animals have normal *let-7* activity at the permissive temperature of 15°C but have reduced *let-7* activity at the nonpermissive temperature of 20°C. In *let-7(n2853ts)* mutants at 20°C, AVM axons regenerated to twice the length of those regenerated in wild-type animals (Fig. 2M), which suggested that *let-7* mutations recapitulate the *alg-1* mutant phenotype in enhancing AVM axon regeneration (Fig. 2M). *let-7(n2853ts)* likely represents a loss-of-function mutation in AVM at 20°C, because *let-7(n2853ts)* mutants display a similar degree of AVM axon regeneration at 23°C as at 20°C (average axon regeneration is 105 μm at 23°C and 100 μm at 20°C, *P* = 0.32). Like *alg-1* mutants, *let-7(n2853ts)* animals also display a retarded developmental decline in AVM axon regeneration (fig. S4, A and B). Regenerating AVM neurons in *let-7(n2853ts)* mutants not only extended longer axons, they also frequently displayed compact growth cones (fig. S1D), a phenotype shared by *alg-1* mutants (fig. S1C). Some regenerating axons in *let-7* mutants only stop outgrowth as they are about to enter the nerve ring (Fig. 2L). The effect of *let-7* mutations on AVM axon regeneration appears to be cell

autonomous, as expression of *let-7* in AVM significantly rescued the *let-7* mutant phenotype of enhancing AVM axon regeneration (Fig. 2M).

*lin-4*, like *let-7*, is a developmental timing microRNA and is expressed strongly in AVM (14). AVM axon regeneration in *lin-4* mutants is not significantly different from that in wild-type animals (fig. S5A), which suggests that different developmental-timing microRNAs play different roles in AVM: *lin-4* is used to time AVM axon connectivity as reported previously (14); *let-7* is used to time developmental decline in AVM axon regeneration.

To determine whether the effect of *let-7* on axon regeneration represents a prior role for *let-7* during development of the AVM to establish its regenerative potential or, rather, an acute requirement for *let-7* during the process of regeneration, we performed temperature-shift experiments. We observed that *let-7(n2853ts)* animals shifted to the nonpermissive temperature after injury exhibit better axon regeneration than wild-type animals (fig. S5). These data indicate that *let-7* has an immediate effect in regenerating AVM neurons to inhibit their regenerative capacity.

*let-7* is expressed only weakly in AVM and anterior lateral microtubule (ALM) mechanosensory neurons at the L1 stage (Fig. 2J) when AVM and ALM axon development is under way, which suggests that a low level of *let-7* may be necessary for normal axon development. To test whether elevated *let-7* would inhibit AVM and ALM axon outgrowth in development, we forced early expression of *let-7* in AVM and ALM using a cell-specific *mec-4* promoter, which enables gene expression from the late embryonic stage onward. Forced early expression of *let-7* in AVM and ALM limits their axon outgrowth during


![Figure 1](attachment://image.png)

**Fig. 1.** *alg-1* mutations slow developmental decline in AVM axon regeneration. (A) AVM axon before, immediately after, and 24 hours after laser surgery in a wild-type animal. D (distal end) and P (proximal end) of severed axons. Red arrow points to the focus of laser plasma. Anterior is left, dorsal up. (B) The positions of regenerating axon termini in wild-type L2-stage animals, wild-type YAs, and *alg-1* YAs. In each scatter plot, the top line (dorsal nerve cord), the bottom line (ventral nerve cord), and the wild-type AVM morphology before surgery (green) are shown. (C and D) Average AVM axon length in wild type (C) and *alg-1* mutants (D) at different stages. Asterisks indicate cases in which latter stage animals differ from L2-stage animals at **\*\***P < 0.01; \*\*\**P* < 0.001. Error bars indicate SEM.

development (fig. S6, A to G), which suggests that premature let-7 expression closes early the window of plasticity for axon outgrowth.

hbl-1 and lin-41 are critical direct targets of let-7 in C. elegans developmental timing (10, 16). hbl-1 encodes a transcription factor and LIN-41 is a member of TRIM family of proteins. Analysis of the promoter activity of hbl-1 and lin-41 genes by using green fluorescent protein–fused promoter (promoter::GFP) reporters showed that lin-41 is expressed in AVM and ALM, but hbl-1 is not (Fig. 3A). Also, lin-41 mutants (but not hbl-1 mutants) display significantly reduced AVM axon regeneration (Fig. 3, C and E). lin-41 mutations suppress the let-7 mutant phenotype of enhancing AVM axon regeneration, which suggests that lin-41 acts downstream of let-7 in AVM (Fig. 3E). Consistent with a positive role for lin-41 in promoting axon regeneration, overexpressing lin-41 in AVM enhanced AVM axon regeneration (Fig. 3, D and E).

Consistent with a developmental down-regulation of lin-41 in AVM by rising let-7, an engineered genomic reporter containing the 5′, coding, and 3′ fragments of the lin-41 gene showed a stage-dependent decline in AVM (Fig. 3B). This down-regulation of lin-41 is likely via

direct interaction of let-7 with the lin-41 3′ untranslated region (3′UTR), as the expression of a GFP::lin-41 3′UTR sensor in adult AVM was higher in let-7 mutants than in wild-type animals (fig. S7), and ectopic expression of let-7 in younger AVM was sufficient to repress GFP::lin-41 3′UTR (fig. S8).

lin-29, a zinc finger transcription factor of the C2H2 type (17), is known to be inhibited by lin-41 (10). We show that lin-29 mutations cause enhanced AVM axon regeneration but do not further enhance regeneration in let-7 mutants (Fig. 3E). In addition, lin-29 mutations suppress the lin-41 mutant phenotype of reducing AVM axon regeneration (Fig. 3E). Further, the level of the LIN-29::GFP fusion protein in AVM at the L1 stage is significantly higher in lin-41 mutants than in wild-type animals (Fig. 3F). Together, our results indicate that let-7/lin-41 is acting via lin-29 to control AVM axon regeneration.

A conserved mitogen-activated protein kinase (MAPK) pathway, including the DLK-1 MAPK kinase and p38 kinase–PMK-3, controls axon regeneration in C. elegans (4). This pathway is negatively regulated by the conserved protein RPM-1 (4). We show that dlk-1 and pmk-3 mutations suppress the enhanced AVM axon re-

generation of let-7 mutants, which suggests that let-7 may act upstream of dlk-1 and pmk-3 (fig. S9A). Further, lin-41 mutations do not suppress the enhanced AVM axon regeneration of rpm-1 mutants, whereas pmk-3 and dlk-1 mutations do, which suggests that lin-41 may act upstream of or in parallel to rpm-1 (fig. S9A). Analysis of the RPM-1 level shows no difference between lin-41 mutants and wild-type animals (fig. S9B), which indicates that either LIN-41 is acting in parallel to RPM-1 or LIN-41 is acting through a posttranslational mechanism to indirectly regulate RPM-1 activity but not its level.

In wild-type animals, mature let-7 microRNA is expressed at low levels before the L3 stage (Figs. 2A and 4A). In lin-41 mutants, let-7 is precociously expressed at the L1 stage (Fig. 4A), which indicates that lin-41 prevents the early-onset expression of let-7. The expression intensity of a Plet-7::GFP reporter in the whole animal or in AVM is slightly lower instead of higher in lin-41 mutants than in wild-type animals at L1 to L3 stages (Fig. 4, B and C), which suggests that lin-41 inhibits early expression of let-7 posttranscriptionally.

Recent studies of mouse stem cells showed that mLin41 E3 ubiquitin ligase activity regulates

**Fig. 2. The let-7 expression in AVM and ALM and the let-7 mutant phenotype in enhancing AVM axon regeneration.**

(A) Stem-loop RT-PCR analysis of RNA isolated from populations of staged animals revealed late-onset expression of several mature microRNAs.

(B) Broad let-7 expression in neurons at L3.

(C to H) The let-7 promoter drives GFP expression in AVM and ALM touch neurons in L2-stage (C to E) and YA (F to H) animals. The Pmec-4::dsRED reporter labels AVM and ALM.

(I) Mature let-7 levels by stem-loop RT-PCR in alg-1 mutants versus wild-type animals at L3 and YA stages. The value below each lane indicates amount of let-7 normalized to actin.

(J) Average fluorescent intensity of the Plet-7::GFP reporter normalized to the Pmec-4::dsRED in AVM. Regenerating AVM axons in wild-type adult animals (K) and adult let-7(n2853) mutants (L). The dashed box in (L) delineates the growth cone, whose blown-up image is shown in the inset.

(M) Average AVM axon length. Asterisks indicate cases in which mutants differ from wild-type or a specific comparison is significantly different at ***P < 0.001.

Argonaute2 turnover (18). Our study shows that the ALG-1 Argonaute is required for let-7 maturation in *C. elegans* (Fig. 2I). To test whether LIN-41 inhibits the early-onset expression of let-7 through negative regulation of ALG-1, *alg-1* mutations were examined for their effects on let-7 expression in *lin-41* mutants. In *lin-41; alg-1* double mutants, let-7 levels were restored to late-onset expression (Fig. 4A). Because only the ectopic (early) expression but not the normal

(late) expression of let-7 was affected by *alg-1* mutations in *lin-41* mutants, this result suggests that LIN-41 may inhibit ALG-1 only at the early stage to repress early-onset let-7 expression. Indeed, LIN-41 is no longer able to down-regulate let-7 expression in the adult stage (Fig. 4D).

Consistent with this interpretation, the level of the ALG-1::GFP fusion protein in AVM at the L1 stage was significantly higher in *lin-41*

mutants than in wild-type animals (Fig. 4E). Using a *lin-41* promoter construct that drives the FLAG-tagged LIN-41 protein expression mainly in neurons (Fig. 3A), we tested whether immunoprecipitation (IP) of ALG-1 would coprecipitate the FLAG-tagged LIN-41 protein. Co-IP of FLAG::LIN-41 with ALG-1 was observed (Fig. 4F and fig. S10A), which suggests that these two proteins may form a complex in neurons. The relatively low LIN-41::FLAG signal

---

**Fig. 3. The lin-41 expression study and the lin-41 effect on AVM axon regeneration.**

(A) Broad *lin-41* expression in neurons at L2. SDQR is an interneuron.

(B) The *let-7* promoter reporter and the engineered *lin-41* genomic reporter are temporally regulated in AVM neurons. Typical regenerating AVM axons in adult *lin-41* mutants (C) and Ex[*Pmec-4::lin-41*] transgenic animals (D). Red arrowhead points to the axon break, and white arrowhead

points to the terminus of regenerating axons. Anterior is left, dorsal up. (E) Average AVM axon length. Asterisks indicate cases in which mutants or transgenic animals significantly differ from wild-type or a comparison between mutants is significantly different at ***P < 0.001***. (F) Analysis of the level of the LIN-29::GFP fusion protein in AVM in *lin-41* mutants versus wild-type animals at L1. Images shown are expression of the LIN-29::GFP fusion protein in the AVM nucleus.

---

**Fig. 4. LIN-41 inhibits let-7 expression through ALG-1 at the early stage.**

(A) Analysis of temporal expression of mature *let-7* microRNA in wild-type and *lin-41* mutants by the stem-loop RT-PCR assay. Analysis of the *let-7* promoter activity in the whole animal (B) and AVM (C) in *lin-41* mutants versus wild-type animals. (D) Stem-loop RT-PCR analysis of the mature *let-7* microRNA in wild-type and Ex[*Plin-41::lin-41*]. (E) Analysis of the expression of the ALG-1::GFP fusion protein in AVM in *lin-41* mutants versus wild-type animals at L1. (F) IP of ALG-1 coprecipitates FLAG-tagged LIN-41. (G) Model of *let-7*-signaled developmental decline in AVM axon regeneration.

intensity in the ALG-1 IP suggests that only a subpopulation of LIN-41 is in complexes with ALG-1. Similarly, IP of GFP-tagged LIN-41 pulled down a subset of the ALG-1 protein population (fig. S10B). IP of small subsets of protein populations with components of microRNA-induced silencing complex has precedents (15, 19).

Even though *let-7* levels decline significantly by day 4 into the adult stage (fig. S11, A and B), AVM axon regeneration remained significantly enhanced in aged animals that either overexpress *lin-41* or contain *let-7* mutations (fig. S11, C and D), which indicates that the *let-7/lin-41* pathway regulates axon regeneration even in aged AVM. Furthermore, *let-7* mutations significantly enhance, whereas *lin-41* mutations significantly reduce, axon regeneration in several neurons that coexpress both genes, which suggests that the role of *let-7/lin-41* in regulating axon regeneration can be extended beyond AVM neurons (fig. S12).

In this study, we show that *let-7* contributes to a developmental decline in AVM axon regeneration. We identify LIN-41 as an important AVM axon regeneration-promoting factor. *let-7* represses *lin-41* expression to inhibit AVM axon regeneration in older neurons. Our results suggest a negative regulatory loop between *let-7*

and *lin-41* (Fig. 4G). In younger neurons, *lin-41* inhibits *let-7* expression through negative regulation of ALG-1. In older neurons, up-regulation of *let-7* overcomes the *lin-41* inhibition and, in turn, represses the *lin-41* expression through the *lin-41* 3′ UTR. Like *C. elegans*, mammals also exhibit a developmental decline in axon regeneration. Our findings suggest that it may be possible to enhance axon regeneration after injury through therapeutic inhibition of the *let-7* microRNA in neurons and, thereby, to restore their youthful regenerative capacity.

### References and Notes

1. H. Chiu, A. Alqadah, C.-F. Chuang, C. Chang, *Cell Adhes. Migr.* **5**, 387 (2011).
2. Z. Wu *et al.*, *Proc. Natl. Acad. Sci. U.S.A.* **104**, 15132 (2007).
3. C. V. Gabel, F. Antoine, C.-F. Chuang, A. D. T. Samuel, C. Chang, *Development* **135**, 3623 (2008).
4. M. Hammarlund, P. Nix, L. Hauth, E. M. Jorgensen, M. Bastiani, *Science* **323**, 802 (2009).
5. J. L. Goldberg, M. P. Klassen, Y. Hua, B. A. Barres, *Science* **296**, 1860 (2002).
6. M. Boehm, F. Slack, *Science* **310**, 1954 (2005).
7. R. C. Lee, R. L. Feinbaum, V. Ambros, *Cell* **75**, 843 (1993).
8. B. Wightman, I. Ha, G. Ruvkun, *Cell* **75**, 855 (1993).
9. A. E. Pasquinelli *et al.*, *Nature* **408**, 86 (2000).
10. F. J. Slack *et al.*, *Mol. Cell* **5**, 659 (2000).
11. R. J. Johnston Jr., O. Hobert, *Nature* **426**, 845 (2003).
12. S. Chang, R. J. Johnston Jr., C. Frøkjaer-Jensen, S. Lockery, O. Hobert, *Nature* **430**, 785 (2004).
13. Y.-W. Hsieh, C. Chang, C.-F. Chuang, *PLoS Genet.* **8**, e1002864 (2012).
14. Y. Zou, H. Chiu, D. Domenger, C.-F. Chuang, C. Chang, *Sci. Signal.* **5**, ra43 (2012).
15. T. F. Duchaine *et al.*, *Cell* **124**, 343 (2006).
16. S. Y. Lin *et al.*, *Dev. Cell* **4**, 639 (2003).
17. A. E. Rougvie, V. Ambros, *Development* **121**, 2491 (1995).
18. A. Rybak *et al.*, *Nat. Cell Biol.* **11**, 1411 (2009).
19. C. M. Hammell, I. Lubin, P. R. Boag, T. K. Blackwell, V. Ambros, *Cell* **136**, 926 (2009).

**Acknowledgments:** We thank the *C. elegans* Genetic Center for *C. elegans* strains; the WormBase for readily accessible information; F. Ciamacco, B. Bayne, and C. Carrelli for technical assistance; A. Fire for *C. elegans* vectors; and V. Cleghon for critical reading of the manuscript. This work was funded by grants from the Whitehall Foundation (C.C. and C.-F.C.), the March of Dimes Foundation (C.C.), the Canada Foundation for Innovation (C.C.), by Alfred P. Sloan Research Fellowships to C.-F.C., by NIH grant R01 GM098026 to C.-F.C., and by NIH grant R01 GM034028-25 to V.A.

**Supplementary Materials**

www.sciencemag.org/cgi/content/full/340/6130/372/DC1

Materials and Methods  
Figs. S1 to S12  
Table S1  
References (20–23)  

10 October 2012; accepted 8 March 2013  
10.1126/science.1231321

---

## A Neural Marker of Perceptual Consciousness in Infants

Sid Kouider${}^{1,2*}$ Carsten Stahlhut${}^{2}$ Sofie V. Gelskov${}^{1,3}$ Leonardo S. Barbosa${}^{1}$ Michel Dutat${}^{1}$ Vincent de Gardelle${}^{1}$ Anne Christophe${}^{1}$ Stanislas Dehaene${}^{4,5,6,7}$ Ghislaine Dehaene-Lambertz${}^{5,6,7}$

Infants have a sophisticated behavioral and cognitive repertoire suggestive of a capacity for conscious reflection. Yet, demonstrating conscious access in infants remains challenging, mainly because they cannot report their thoughts. Here, to circumvent this problem, we studied whether an electrophysiological signature of consciousness found in adults, corresponding to a late nonlinear cortical response [~300 milliseconds (ms)] to brief pictures, already exists in infants. We recorded event-related potentials while 5-, 12-, and 15-month-old infants ($N = 80$) viewed masked faces at various levels of visibility. In all age groups, we found a late slow wave showing a nonlinear profile at the expected perceptual thresholds. However, this late component shifted from a weak and delayed response in 5-month-olds (starting around 900 ms) to a more sustained and faster response in older infants (around 750 ms). These results reveal that the brain mechanisms underlying the threshold for conscious perception are already present in infancy but undergo a slow acceleration during development.

R ecent research shows that, in the first year of life, preverbal infants already display an impressive array of cognitive competences. For instance, their eye movements betray a capacity to monitor other people’s beliefs at 7 months (1) and to draw probabilistic predictions about visual scenes at 12 months (2). Given these complex behaviors, one might consider it obvious that infants already have a conscious experience of their environment. However, this conclusion is unwarranted, because the presence of unconscious priming and “blind-

“sight” behaviors in normal and impaired adults (3, 4) shows that sophisticated processing can occur without consciousness.

How, then, might one test whether the brain mechanisms for conscious access are already present in infancy? Studying consciousness and its neural correlates in adults requires the collection of subjective reports of experience, classically through psychophysical paradigms contrasting visible and invisible stimuli (5). It is not impossible to obtain subjective reports from nonverbal organisms. For instance, monkeys can be trained

1Laboratoire de Sciences Cognitives et Psycholinguistique, EHESS/CNRS/ENS-DEC, 75005 Paris, France. 2Section for Cognitive Systems, Department of Informatics and Mathematical Modeling, Technical University of Denmark, 2800 Kongens Lyngby, Denmark. 3Danish Research Center for Magnetic Resonance, Copenhagen University Hospital, 2650 Hvidovre, Denmark. 4Collège de France, 75231 Paris, France. 5INSERM, U992, Cognitive Neuroimaging Unit, 91191 Gif-sur-Yvette, France. 6CEA, NeuroSpin Center, 91191 Gif-sur-Yvette, France. 7Université Paris XI, 91405 Orsay, France.  
*Corresponding author. E-mail: sid.kouider@ens.fr

Developmental Decline in Neuronal Regeneration by the Progressive Change of Two Intrinsic Timers

Yan Zou et al.

Science 340, 372 (2013);
DOI: 10.1126/science.1231321

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by clicking here.

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines here.

The following resources related to this article are available online at www.sciencemag.org (this information is current as of May 27, 2015):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:
http://www.sciencemag.org/content/340/6130/372.full.html

Supporting Online Material can be found at:
http://www.sciencemag.org/content/suppl/2013/04/17/340.6130.372.DC1.html

A list of selected additional articles on the Science Web sites related to this article can be found at:
http://www.sciencemag.org/content/340/6130/372.full.html#related

This article cites 23 articles, 10 of which can be accessed free:
http://www.sciencemag.org/content/340/6130/372.full.html#ref-list-1

This article has been cited by 6 articles hosted by HighWire Press; see:
http://www.sciencemag.org/content/340/6130/372.full.html#related-urls

This article appears in the following subject collections:
Neuroscience
http://www.sciencemag.org/cgi/collection/neuroscience

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2013 by the American Association for the Advancement of Science; all rights reserved. The title Science is a registered trademark of AAAS.
