
JACS
JOURNAL OF THE AMERICAN CHEMICAL SOCIETY

Subscriber access provided by UNIV OF NORTH CAROLINA

Communication

**Uncovering the Enzymes that Catalyze the Final Steps in Oxytetracycline Biosynthesis**

Peng Wang, Ghader Bashiri, Xue Gao, Michael R. Sawaya, and Yi Tang

J. Am. Chem. Soc., Just Accepted Manuscript • DOI: 10.1021/ja403516u • Publication Date (Web): 26 Apr 2013

Downloaded from http://pubs.acs.org on April 29, 2013

---

### Just Accepted

"**Just Accepted**" manuscripts have been peer-reviewed and accepted for publication. They are posted online prior to technical editing, formatting for publication and author proofing. The American Chemical Society provides "Just Accepted" as a free service to the research community to expedite the dissemination of scientific material as soon as possible after acceptance. "Just Accepted" manuscripts appear in full in PDF format accompanied by an HTML abstract. "Just Accepted" manuscripts have been fully peer reviewed, but should not be considered the official version of record. They are accessible to all readers and citable by the Digital Object Identifier (DOI®). "Just Accepted" is an optional service offered to authors. Therefore, the "Just Accepted" Web site may not include all articles that will be published in the journal. After a manuscript is technically edited and formatted, it will be removed from the "Just Accepted" Web site and published as an ASAP article. Note that technical editing may introduce minor changes to the manuscript text and/or graphics which could affect content, and all legal disclaimers and ethical guidelines that apply to the journal pertain. ACS cannot be held responsible for errors or consequences arising from the use of information contained in these "Just Accepted" manuscripts.

---

ACS Publications  
High quality. High impact.

Journal of the American Chemical Society is published by the American Chemical Society. 1155 Sixteenth Street N.W., Washington, DC 20036  
Published by American Chemical Society. Copyright © American Chemical Society. However, no copyright claim is made to original U.S. Government works, or works produced by employees of any Commonwealth realm Crown government in the course of their duties.

Journal of the American Chemical Society

Uncovering the Enzymes that Catalyze the Final Steps in Oxytetracycline Biosynthesis

Peng Wang†, Ghader Bashiri‡, Xue Gao†, Michael R. Sawaya§, and Yi Tang†,+

†Department of Chemical and Biomolecular Engineering, ‡Department of Chemistry and Biochemistry, §Howard Hughes Medical Institute, UCLA-DOE Institute for Genomics and Proteomics, University of California Los Angeles

‡Structural Biology Laboratory, Maurice Wilkins Centre for Molecular Biodiscovery and School of Biological Sciences, The University of Auckland, Auckland, New Zealand

Supporting Information Placeholder

ABSTRACT: Tetracyclines are a group of natural products sharing a linearly-fused four-ring scaffold, which is essential for their broad-spectrum antibiotic activities. Formation of the key precursor anhydrotetracycline 3 during oxytetracycline 1 biosynthesis has been previously characterized. However, the enzymatic steps that transform 3 into 1, including the additional hydroxylation at C5 and the final C5a-C11a reduction, have remained elusive. Here we report two redox enzymes, OxyS and OxyR, are sufficient to convert 3 to 1. OxyS catalyzes two sequential hydroxylations at C6 and C5 positions of 3 with opposite stereochemistry, while OxyR catalyzes the C5a-C11a reduction using F420 as a cofactor to produce 1. The crystal structure of OxyS was obtained to provide insights into the tandem C6, C5-hydroxylation steps. The substrate specificities of OxyS and OxyR were shown to influence the relative ratio of 1 and tetracycline 2.

or 7 is reduced to 2 or 1, respectively. An earlier study proposed that the enzyme TchA encoded outside of the ctc gene cluster in S. aureofaciens is involved in this final reduction step of chlorotetracycline 4 (as well as 2) biosynthesis. However, recent evidences suggest that this enzyme may not play a direct role in C5a-C11a reduction, but may be a coenzyme F420:L-glutamate ligase that is essential in the biosynthesis of the F420 cofactor itself (Figure S1). Initial bioinformatic analysis revealed no F420-binding enzyme is encoded in either the oxy or the ctc cluster. However, recent heterologous expression of the oxy cluster (spanning otrA to otrB), which resulted in the production of 1 in Myxococcus xanthus and Streptomyces lividans K4-114, strongly suggests that all the enzymes required to convert 3 to 1 are encoded within the cluster. This therefore prompted us to examine the uncharacterized oxy genes.

OxyR is a small protein (16 kDa) encoded immediately adjacent, and in opposite polarity to oxyS, and the two are cotranscribed from the bidirectional promoter (Figure S2). A similar genetic arrangement of homologs of OxyS and OxyR is also observed in the ctc gene cluster. OxyR shares 63% and 64% similarities to CtcR from ctc cluster and DacO4 from the recently sequenced dactylocycline (dac) gene cluster, respectively. A new phylogenetic profiling method was recently used to uncover previously unassigned, F420-containing enzymes in bacteria. Intriguingly, bioinformatic analysis using these newly reannotated enzymes now shows OxyR, CtcR and DacO4 are predicted to use F420 as a cofactor. The up-to-date in silico analysis and the oxyR-oxyS genetic arrangement led us to examine the role of OxyR. We inactivated oxyR gene in the oxy pathway that is heterologously transplanted in S. lividans K4-114, which resulted in elimination of biosynthesis of 1 (Figure S3). No known intermediate leading to and including 3 was accumulated from this strain. This hinted that OxyR participates in the transformation of 3 to 1, most likely following the C6 hydroxylation of 3 to 6 catalyzed by OxyS. The 1, 3, 5-triketo containing 6 is known to be highly unstable, which may account for the lack of intermediates in the ΔoxyR strain. To probe the timing of OxyR, we constructed a recombinant S. lividans strain that overexpresses both OxyS and OxyR using the ermE* promoter (Supplementary Methods). 3 was then supplemented to a liquid culture of this strain as a substrate for biotransformation. Following 3 days of culturing, the production of 1 was detected (Figure S3), indicating that only OxyS and OxyR are sufficient to convert 3 to 1 in vivo. To investigate the role of OxyR in the pathway, and whether it is indeed the F420-dependent tetracycline dehydrogenase, we then turned to in vitro experiments using purified enzymes.

The final C5a-C11a reduction step has been proposed to involve a F420-dependent tetracycline dehydrogenase, in which intermediates 6

ACS Paragon Plus Environment

Journal of the American Chemical Society

Page 2 of 5

![Chemical Structures and Reactions](chemical_structures.png)

Figure 1. The proposed mechanisms of the transformations of 3 to 1 and 2. The Fl-4a-OOH bound OxyS catalyzes the first hydroxylation at C6 of 3 to form 6, which is in equilibrium between keto and enol forms. The latter is hydroxylated at C5 by the reoxidized OxyS-Fl-4a-OOH to form the intermediate 7. Holo-OxyR with reduced F₄₂₀ catalyzes the C5a-C11a reduction to afford the final product 1. If C5 hydroxylation does not take place, OxyR reduces 6 into 2; Insets: i) The parallel oxidation and reduction steps that produce 4 and 5; ii) The regeneration of F₄₂₀-Ox to F₄₂₀-H₂ catalyzed by F₄₂₀-dependent glucose-6-phosphate dehydrogenase (Mbt-FGD1) using glucose-6-phosphate (G6P) as substrate.

The polyhistidine-tagged OxyR was expressed and purified from *S. lividans* TK64 (Figure S4). OxyR purifies with a light yellow color indicating that it is bound to a cofactor. Purified holo-OxyR was denatured and the resulting yellow supernatant was subjected to LC-MS analysis. The analysis showed released compounds with UV absorption spectrum, retention time, and mass consistent with coenzyme F₄₂₀ purified from *Mycobacterium smegmatis* cells (Figure S5). This result hints that the small OxyR is the elusive F₄₂₀-dependent C5a-C11a reductase. To generate the reduced form of F₄₂₀ cofactor (F₄₂₀-H₂) in situ (Figure 1, inset ii), the F₄₂₀-dependent glucose-6-phosphate dehydrogenase from *M. tuberculosis* (Mtb-FGD1) was expressed and purified from *M. smegmatis*. When holo-OxyS purified from *Escherichia coli* (Figure S4) and holo-OxyR were added to 3 in the presence of NADPH and the F₄₂₀ regeneration system, we detected the complete conversion of 3 to 1 and 2 (Figure 2e). This is therefore consistent with our in vivo result that OxyS and OxyR are sufficient to produce the mature tetracycline scaffold. The unexpected production of 1, instead of 2, as the major product in this assay indicates that the C5 hydroxylation step that converts 6 to 7 is also catalyzed by OxyS (Figure 1 and see discussion below). Recombinant OxyR purified from *E. coli* does not exhibit any yellow color and does not release F₄₂₀ upon denaturation, as would be expected from the lack of F₄₂₀ in *E. coli* metabolism. This apo-OxyR, when reconstituted with F₄₂₀, along with holo-OxyS, similarly catalyzed the conversion of 3 to 1 and 2 with a product ratio of 7:1 (Figure 2d).

Removal of OxyS from the reaction led to no conversion of 3 (Figure 2f), as expected since formation of the C5a-C11a double bond requires C6 hydroxylation and dearomatization of the C ring of 3 by OxyS. Exclusion of OxyR from the reaction led to complete consumption of 3, but no stable product can be isolated for characterization (Figure 2g). At very early time points (~1 min), uncharacterized products with masses (*m/z* 443 \[M+H\]⁺, *m/z* 459 \[M+H\]⁺) consistent with those of singly and doubly hydroxylated products can be detected in the OxyS-only reaction mixture (Figure S6). However, these products rapidly degraded, in line with our observation that the ΔoxyR *S. lividans* strain does not accumulate any isolatable intermediates.

The mechanisms of the OxyS and OxyR reactions are shown in Figure 1. OxyS catalyzes the stereospecific hydroxylation of 3 at C6 via the oxidized OxyS-Fl-4a-OOH form using the monooxygenase mechanism. We propose the resulting 6 can be released by OxyS, and can be captured by OxyR to reduce the C5a-C11a double bond using the low-potential hydride provided by the F₄₂₀-H₂ cofactor and yield 2. However, OxyS can recapture 6 and perform an additional hydroxylation with opposite stereochemistry at C5 to yield 7. During their total synthesis of tetracycline, Myers and coworkers reported a C6 hydroperoxide analog of 6 that can equilibrate between ketone and enol forms at C11-C11a-C5a-C5, which confirms that vinylogous C5 proton in 6 is sufficiently acidic. We propose that C5 hydroxylation of enol-6 can also be initiated through the base-catalyzed proton extraction of the C11 hydroxyl. However, the C6-substituted and dearomatized 6 must bind in a different conformation in the active site of OxyS compared to that of 3. This may account for the C5 carbanion attacking Fl-4a-OOH from the opposite face to give the flipped stereochemistry of the hydroxyl group at C5. The resulting 7 is then released and reduced by OxyR to yield 1. The 8-hydroxy-7-desmethyl-5-deazariboflavin part of the oxidized F₄₂₀ (F₄₂₀-Ox) is then reduced by Mbt-FGD1 using G6P as hydride donor (Figure 1, inset ii).

Our reconstitution of the final step of 1 biosynthesis raises questions with regard to the lack of C5 hydroxylation in the ctc or dac pathways (Figure 1, inset i). We examined the effect of substituting OxyR homologs from these pathways on product distribution from the in vitro assays. Both CtcR and DacO4 enzymes were purified in apo forms from *E. coli* (Figure S4) and reconstituted with F₄₂₀ into the holo forms. Analysis of products showed that although 1 is still the dominant product, using CtcR or DacO4 decreased the ratio of 1 to 2 (Figure 2h and 2i). Notably when CtcR is used, the ratio is decreased to below 2:1. These mix and match assays showed that 1) both CtcR and DacO4 are the F₄₂₀-dependent C5a-C11a reductase in the respective pathways; and 2) compared to OxyR, CtcR displays a

stronger affinity for 6 in competition with OxyS for the second
hydroxylation step, albeit the latter still dominates to afford 1 as the
major product.

structure of OxyS is similar to other FAD dependent monooxygenases
found in the pathways of aromatic polyketides, including aklavinone-11
hydroxylase RdmE (PDB ID: 3ihg) from the rhodomycin pathway, 18 as
well as PgaE (PDB: 2qa1) and CabE (PDB: 2qa2) from angucycline
pathways (Figure S7). 19 Attempts to obtain OxyS structures bound to
3 were not successful. To model likely interactions between 3 and the
substrate binding pocket of OxyS, we performed structural alignment
with RdmE in complex with the cocrystallized, aklavinone 8 (Figures 3
and S9). Using the superimposed structures of OxyS and RdmE (the
RMSD is 1.51 Å for 282 matching alpha carbons) and the coordinates
of 8, we can propose a putative binding pocket and binding orientation
for 3. The tetracycline substrate is anchored in a narrow hydrophobic
cleft at the interface between the FAD-biding domain and the middle
domain, as indicated for 8 in Figure 3. The middle domain provides a
hydrophobic and aromatic patch (Trp211-Leu217, OxyS numbering,
for alignment see Figure S7) against the D and C rings of 8. On the
other side of the pocket, the well conserved patch PAGG helps to
position the substrate with respect to the FAD isoalloxazine ring. As
shown in Figure 3 for 8, and can be envisioned for 3, the planar C-D
rings are perpendicular to the isoalloxazine ring of Fl ox, with the C6
carbon placed 4.7 Å away from the bridgehead C4a of Fl ox. Similarly,
the C5 carbon is also located within striking distance (4.4 Å) of C4a,
which may explain how OxyS can hydroxylate both positions.
However, in order to achieve the opposite stereochemical outcomes,
the substrate 3 and 6 must be tilted with respect to the FAD ring to
facilitate attack of C6 and C5 on Fl-C4a-OOH from the opposite faces.
Hence it is likely that slight perturbation in the geometry of the binding
pocket can alter the ability and stereoselectivity of OxyS in catalyzing
the hydroxylation steps.

To test this hypothesis, we further generated a modeled
configuration of 3 in the active site of OxyS based on the D-and C-ring
coordinates of 8. This is to account for the nonplanar nature of the A
and B rings in 3 (Supplemental Methods and Figure S9). Comparison
of the modeled positions of 8 and 3 led us to mutate His 47 and
Phe 215, which appear to be in close contact with the substrates (Figure
3 and S9). The N1 of His47 imidazole side chain is located 3.7 Å away
from the C3 oxygen atom in 8, suggesting it might hydrogen bond to
the same hydroxyl position in the A ring of 3 and 6. When the assay
was performed in the presence of a His47Ala mutant of OxyS (Figure
S4), a decreasing ratio of 1 to 2 was observed. This implies that
disrupting the possible hydrogen bonding interactions can indeed lead
to a repositioning of the substrate and decrease the efficiency of the C5
hydroxylation step. Interestingly, we also observed the emergence of a
new product from the assay that has similar UV and identical mass
(m/z 445 [M+H]+) as 2 (Figure 2j and S8). Although the structure of
the compound cannot be determined due to low amounts produced
from the in vitro assay, it is possible that this is the 6R-OH analog of 2
formed from the altered binding of 3 in the mutant active site.

On the other hand, Phe 215 is located near the entrance of the
active site and the phenyl side chain is positioned close to the C ring.
The proximity of the bulky side chain may also play a role in orienting
the position of 3 and 6 in the active site with respect to Fl-4a-OOH. To
investigate the role of this residue, we made the mutation of Phe215Ile.
Surprisingly, the slight decrease in the volume of the side chain at
position 215 led to dramatic changes in the product ratio of 1:2 (Figure
3i), in which the amount of 2 now exceeded that of 1. Although these
mutagenesis studies were based on a structural alignment using the
different substrate 8, and the exact roles of these residues may differ
during catalysis, it is clear that the ability of OxyS to hydroxylate both
C6 and C5 has been intricately tuned during evolution. In Cts 8 and
DacO1, variations in the active-site configuration can lead to the
exclusive production of the C6 hydroxylated products, as well as the

Figure 2. Analysis of OxyR and OxyS functions. The HPLC traces (270
nm) of the authentic compounds (a) 1, (b) 2 and (c) 3; and the
enzymatic reaction products by (d) OxyS, apo-OxyR and F420; (e)
OxyS and holo-OxyR; (f) holo-OxyR only; (g) OxyS only; (h) OxyS,
apo-CtcR and F420; (i) OxyS, apo-DacO4 and F420; (i) OxyS-H47A,
apo-OxyR and F420. (j) OxyS-F215I, apo-OxyR and F420. Final
concentrations are 20 μM OxyS, 20 μM OxyR, 100 μM 3 and 20 μM
NADPH. To regenerate the reduced form of F420, 2 μM FGD1, 20 μM
F420, and 2 mM G6P are added. The asterisk indicates an
uncharacterized compound with the same mass as 2.

We attempted to examine if OxyS homologs have differential
abilities to catalyze the C5 hydroxylation step. Unfortunately, we were
not able to obtain soluble forms of either Cts8 or DacO1 from
Streptomyces and E. coli. Therefore, to understand the structural
basis of the tandem hydroxylation reactions and to guide mutational
approaches, we determined the crystal structure of OxyS in complex
with oxidized flavin (Fl ox) to 2.6 Å resolution (see Table S1 for
statistics). The monomeric OxyS is comprised of three structural
domains, including the FAD-binding domain (residues 1-175 and 271-
389), the middle domain (residues 176-270) and the C-terminal
thioredoxin-like domain (residues 390-503) (Figure 3). The overall

opposite stereochemistry at C6 observed in dactylocyclinone (Figure 1, inset 1). Similarly, the additional substitutions in D-rings of anhydrochlorotetracycline and anhydrodactylocyclinone, such as the C7 chlorine and C8 methoxy, can further lead to differences in substrate orientation in the corresponding active sites.

![Image Description](#)

FAD binding domain

Aklavinone 8

Middle domain

C-terminal thioredoxin-like domain

FAD

H47

F215

Figure 3. Overall structure alignment of OxyS with the complex of RdmE cocrystallized with aklavinone 8 (palecyan, PDB ID: 3ihg). RdmE is shown in pale cyan while OxyS is colored according to domains. The RMSD is 1.51 Å for 282 matching alpha carbons. The magnified region of the catalytic pocket indicates possible orientation of the substrate 3 or 6 using the cocrystallized 8 (shown in pale cyan). Mutation of H47A and F215I increased the amount of 2 to 1.

In conclusion, we showed that OxyR, CtcR and DacO4 are F₄₂₀-dependent reductases catalyzing the key C5a-C11a reduction step in respective tetracycline biosynthetic pathways. Our in vitro results demonstrate that secondary metabolic pathways in actinobacteria can use F₄₂₀ in natural product biosynthesis, which expands the utility of this cofactor beyond a nicotinamide equivalent in methogenic archaea²⁰ and mycobacteria.¹² This is in line with the isolation of F₄₂₀ from the fermentation broths of numerous actinomycetes.²¹ Other OxyR-like enzymes in secondary metabolism exist in the database, including ActVA2 (64% similarity), an enzyme of unknown function from the actinorhodin biosynthetic pathway.²² Using structural-guided mutagenesis, we showed that OxyS is responsible for both C6 and C5 hydroxylation steps during the conversion of 3 to 1. While substrate specificities of OxyS and OxyR are important for the observed relative ratio of 1 to 2, it is the unexpected ability of OxyS to catalyze the C5 hydroxylation that results in formation of 1 in the oxy pathway.

ASSOCIATED CONTENT

Supporting Information Available

Collection and refinement statistics table for OxyS structure; LC-MS analysis of in vivo assays and OxyR cofactor; detailed experimental methods. Coordinates of OxyS were deposited in the Protein Data Bank with accession code 4k2x. This information is available free of charge via the internet at [http://pubs.acs.org](http://pubs.acs.org).

AUTHOR INFORMATION

Corresponding Author

yitang@ucla.edu

Notes

The authors declare no competing financial interests.

ACKNOWLEDGMENT

We acknowledge the financial supports to Y. T. from an NSF CBET 1159759 and to Y. T. from NIH 1R01GM097200. We thank Prof. Neil Garg for helpful discussions.

REFERENCES

(1) (a) Chopra, I.; Roberts, M. *Microbiol. Mol. Biol. Rev.* 2001, 65, 232. (b) Hertweck, C.; Luzhetskyy, A.; Rebets, Y.; Bechthold, A. *Nat. Prod. Rep.* 2007, 24, 162.

(2) (a) Thaker, M.; Spanogiannopoulos, P.; Wright, G. D., *Cell. Mol. Life Sci.*, 2010, 67, 419. (b) Nelson, M. L.; Levy, S. B., *Ann. N.Y. Acad. Sci.*, 2011, 1241, 17.

(3) (a) Zhang, W.; Ames, B. D.; Tsai, S. C.; Tang, Y. *Appl. Environ. Microbiol.* 2006, 72, 2573. (b) Zhang, W.; Watanabe, K.; Cai, X.; Jung, M. E.; Tang, Y.; Zhan, J. *J. Am. Chem. Soc.* 2008, 130, 6068. (c) Béhal, V.; Hošťálek, Z.; Vaněk, Z. *Biotechnol. Lett.* 1979, 1, 177. (d) Binnie, C.; Warren, M.; Butler, M. J. *J. Bacteriol.* 1989, 171, 887. (e) Butler, M. J.; Gedge, B. N. *Biotechnol. Tech.* 1989, 4, 235. (f) Hunter, I. S.; Hill, R. A. In *Biotechnology of Antibiotics*; 2 ed.; Strohl, W. R., Ed.; Marcel Dekker: New York, 1997, p 659.

(4) (a) Peric-Concha, N.; Borovicka, B.; Long, P. F.; Hranueli, D.; Waterman, P. G.; Hunter, I. S. *J. Biol. Chem.* 2005, 280, 37455. (b) Wang, P.; Kim, W.; Pickens, L. B.; Gao, X.; Tang, Y. *Angew. Chem. Int. Ed.* 2012, 51, 11136.

(5) Brodersen, D. E.; Clemons, W. M., Jr.; Carter, A. P.; Morgan-Warren, R. J.; Wimberly, B. T.; Ramakrishnan, V. *Cell* 2000, 103, 1143.

(6) (a) Mccormick, J. R. D.; Hirsch, U.; Sjolander, N. O.; Doerschuk, A. P. *J. Am. Chem. Soc.* 1960, 82, 5006. (b) Rhodes, P. M.; Winskill, N.; Friend, E. J.; Warren, M. *J. Gen. Microbiol.* 1981, 124, 329. (c) Walsh, C. *Acc. Chem. Res.* 1986, 19, 216.

(7) Nakano, T.; Miyake, K.; Endo, H.; Dairi, T.; Mizukami, T.; Katsumata, R. *Biosci. Biotechnol. Biochem.* 2004, 68, 1345.

(8) Nocek, B.; Evdokimova, E.; Proudfoot, M.; Kudritska, M.; Grochowski, L. L.; White, R. H.; Savchenko, A.; Yakunin, A. F.; Edwards, A.; Joachimiak, A. *J. Mol. Biol.* 2007, 372, 456.

(9) Stevens, D. C.; Henry, M. R.; Murphy, K. A.; Boddy, C. N. *Appl. Environ. Microbiol.* 2010, 76, 2681.

(10) McDowell, K. J.; Thamchaipenet, A.; Hunter, I. S. *J. Bacteriol.* 1999, 181, 3025.

(11) Ryan, M. J.; Patent, U. S., Ed. United States, 1999.

(12) Selengut, J. D.; Haft, D. H. *J. Bacteriol.* 2010, 192, 5788.

(13) Bashiri, G.; Rehan, A. M.; Greenwood, D. R.; Dickson, J. M.; Baker, E. N. *PLoS One* 2010, 5, e15803.

(14) (a) Purwantini, E.; Daniel, L., *J. Bacteriol.* 1996, 178, 2861. (b) Bashiri, G.; Squire, C. J.; Moreland, N. J.; Baker, E. N. *J. Biol. Chem.* 2008, 283, 17531.

(15) Eirich, L. D.; Vogels, G. D.; Wolfe, R. S. *J. Bacteriol.* 1979, 140, 20.

(16) Walsh, C. T.; Wencewicz, T. A. *Nat. Prod. Rep.* 2013, 30, 175.

(17) Charest, M. G.; Siegel, D. R.; Myers, A. G. *J. Am. Chem. Soc.* 2005, 127, 8292.

(18) Lindqvist, Y.; Koskiniemi, H.; Jansson, A.; Sandalova, T.; Schnell, R.; Liu, Z.; Mantsala, P.; Niemi, J.; Schneider, G. *J. Mol. Biol.* 2009, 393, 966.

(19) Koskiniemi, H.; Metsa-Ketela, M.; Dobritzsch, D.; Kallio, P.; Korhonen, H.; Mantsala, P.; Schneider, G.; Niemi, J. *J. Mol. Biol.* 2007, 372, 633.

(20) Graham, D. E.; White, R. H., *Nat. Prod. Rep.* 2002, 19, 133.

(21) (a) Coats, J. H.; Li, G. P.; Kuo, M. S.; Yurek, D. A. *J. Antibiot. (Tokyo)* 1989, 42, 472. (b) Kuo, M. T.; Yurek, D. A.; Coats, J. H.; Li, G. P., *J. Antibiot. (Tokyo)* 1989, 42, 475.

(22) Caballero, J. L.; Martinez, E.; Malpartida, F.; and Hopwood, D. A. *Mol. Gen. Genet.* 1991, 230, 401.

Page 5 of 5
Journal of the American Chemical Society

TOC Figure

Anhydrotetracycline $\xrightarrow{\text{OxyS}}$ 5a11a-Dehydrooxytetracycline $\xrightarrow{\text{OxyR}}$ Oxytetracycline
