
Send Orders of Reprints at reprints@benthamscience.org

**508** Current Genomics, 2012, 13, 508-518

# Genome-Wide RNAi Longevity Screens in *Caenorhabditis elegans*

Melana E. Yanos${}^{1,2,\S}$, Christopher F. Bennett${}^{1,3,\S}$ and Matt Kaeberlein${}^{1,4,*}$

${}^{1}$Department of Pathology, University of Washington, Seattle, WA, USA; ${}^{2}$Department of Psychology, University of Washington, Seattle, WA, USA; ${}^{3}$Molecular and Cellular Biology Program, University of Washington, Seattle, WA, USA; ${}^{4}$Institute of Aging Research, Guangdong Medical College, Dongguan 523808, China

**Abstract:** Progress in aging research has identified genetic and environmental factors that regulate longevity across species. The nematode worm *Caenorhabditis elegans* is a genetically tractable model system that has been widely used to investigate the molecular mechanisms of aging, and the development of RNA interference (RNAi) technology has provided a powerful tool for performing large-scale genetic screens in this organism. Genome-wide screens have identified hundreds of genes that influence lifespan, many of which fall into distinct functional classes and pathways. The purpose of this review is to summarize the results of large-scale RNAi longevity screens in *C. elegans*, and to provide an in-depth comparison and analysis of their methodology and most significant findings.

Received on: April 02, 2012 - Revised on: June 14, 2012 - Accepted on: July 25, 2012

Keywords: Dietary restriction, FOXO, Genomic, Longevity, IGF-1, Insulin, Mitochondria.

## INTRODUCTION

### A Brief History of Longevity Genetics Research in *C. elegans*

Progress in aging research has identified genetic and environmental factors that regulate longevity across species [1-3]. The nematode worm *Caenorhabditis elegans* has become an invaluable model system for investigating the molecular mechanisms of aging and longevity, offering the advantages of its relatively low cost, short lifespan, and conservation of key nutrient and stress-responsive signaling pathways in mammals.

Prior to assuming a significant role in aging research, however, *C. elegans* had already been established as a useful model organism for studies in genetics and molecular biology. The development of a modern laboratory approach utilizing *C. elegans* can largely be attributed to the pioneering research of Sydney Brenner and John Sulston, whose work on the identification of *C. elegans* genes and DNA composition was first published in *Genetics* in 1974 [4, 5].

*C. elegans* measure approximately 1 mm in length in adulthood, have a reproductive life cycle of approximately 3½ days when raised at a temperature of 20°C, and are easy to obtain in large populations [6-8]. Animals are maintained in solid or liquid medium, and are typically raised at a temperature range of 15°C to 25°C. The worms can be frozen for long-term storage and remain viable upon thaw. Although worms are self-fertilizing hermaphrodites, males occasionally appear and can be used to transfer genetic markers upon crossing.

When a bacterial food source is abundant, worms develop through four larval stages (L1-L4) into adulthood. Under conditions of starvation, *C. elegans* enter an alternate stage of development at the second larval molt called dauer and can survive in this arrested state at least four to eight times its normal lifespan [9-11]. Upon introduction to food, dauer larvae resume their developmental progression to adulthood.

All embryonic and postembryonic cell lineages have been traced for *C. elegans*. Patterns of cell division, migration, and programmed cell death are remarkably constant, and gives rise to a total of 959 somatic cells in the adult hermaphrodite [12-14]. Of these, 302 have been classified as neural cells whose structure and connectivity has also been thoroughly investigated [15]. In addition, the entire *C. elegans* genome has been sequenced, further contributing to the powerful knowledge infrastructure that has been built on this model organism [16].

Studies that laid the groundwork for aging research in *C. elegans* began nearly thirty years ago, when seminal research by Michael Klass, Tom Johnson, and others employed mutagenesis techniques to isolate long-lived mutants; this led to the formal naming and description of the first *C. elegans* longevity gene *age-1* [17, 18]. A decade later, a second wave of research, spurred by the development of new methods and technologies, pushed beyond the identification of individual longevity genes and began to describe genetic and molecular pathways that regulate aging. It was first discovered that mutation of *daf-2*, a gene involved in repressing dauer formation, could extend lifespan by more than two-fold, and that this lifespan extension was dependent on *daf-16* [19, 20]. Subsequently, mutation of *age-1* was shown to similarly require *daf-16* for lifespan extension, suggesting that *daf-2* and *age-1* function within a single genetic pathway [21]. Soon after this, the molecular functions of *daf-2*, *age-1*, and *daf-16* were uncovered; DAF-2 functions as an

---

*Address correspondence to this author at the Department of Pathology, University of Washington, Box 357470, Seattle, WA 98195-7470, USA; Tel: (206) 543-4849; Fax: (206) 543-3644; E-mail: kaeber@uw.edu

$\S$ These authors contributed equally to this work.

Genome-Wide RNAi Longevity Screens in Caenorhabditis elegans

insulin-like receptor that promotes the activity of the AGE-1 phosphatidylinositol-3-hydroxy kinase that antagonizes the activity of DAF-16, a FOXO-family transcription factor [22-25]. Together, these factors defined an insulin/IGF-1-like signaling (IIS) pathway that modulates longevity in C. elegans (Fig. 1).

Several genetic factors that modulate lifespan independently of DAF-16 have been characterized in detail: mimetics of dietary restriction (DR), mutations in mitochondrial electron transport chain (ETC) enzymes, and mutations that activate the hypoxic response. These include the loss of function alleles of the mitochondrial coenzyme Q biosynthetic enzyme CLK-1 [28-30], a series of *eat* mutations that reduce pharyngeal pumping and food consumption [31], and mutations that stabilize the hypoxic response transcription factor, HIF-1 [32-35]. Further studies have suggested that each of these classes of longevity factors act within distinct epistasis groups, leading to the current model for at least four distinct longevity pathways in *C. elegans* (Fig. 1).

Based on studies of the IIS pathway, DAF-16-dependence or independence has come to serve as the most utilized genetic test for grouping longevity factors. Mutations that increase lifespan in a DAF-16-dependent manner are generally referred to as acting within the IIS pathway, although it is now clear that multiple factors can modulate DAF-16 activity by mechanisms that are both similar and different from reduced *daf-2* activity [3]. For example, several studies have identified a signal from the germline that limits longevity in a DAF-16-dependent manner [3]. Laser ablation of the germline results in lifespan extension that is requires both *daf-16* and *daf-12*, a putative hormone receptor [26]. Researchers observed similar lifespan extension in *mes-1* and *glp-1* mutants, both lacking germ cells. Furthermore, lifespan extension in *mes-1* or *glp-1* mutants is suppressed by *daf-16* null mutation or by ablation of somatic gonad precursor cells. Specifically, the absence of undifferentiated germ-line stem cells during development or adulthood was shown to result in lifespan extension [27].

However, this four pathway model is complicated by evidence of known sites of crosstalk between these pathways. For example, the target of rapamycin (TOR) kinase interacts with IIS, DR, and HIF-1 [36-39]. As another example, some (but not all) mutations that impair mitochondrial function and extend lifespan also activate HIF-1 and require HIF-1 for lifespan extension [40]. Thus, it is clear that these genetically distinct pathways are actually components of a complex longevity network. Whether this network will ultimately be found to converge on similar downstream mechanisms for longevity control remains to be determined.

---

**Fig. (1). Simplified model of signaling pathways that influence lifespan in *C. elegans*.** Genetic factors (rectangles) and environmental factors (ovals) that are found to affect lifespan have been grouped into several major longevity pathways. (1) Within the germline signaling pathway, mutation of *mes-1* or *glp-1* prevents development of germ cells and results in lifespan extension. This lifespan extension is dependent on both DAF-16 and DAF-12, a putative hormone receptor. (2) The canonical insulin/IGF-1-like signaling (IIS) pathway involves DAF-2, an insulin-like receptor that promotes activity of the AGE-1 phosphatidylinositol-3-hydroxy kinase, which in turn antagonizes the activity of DAF-16, a FOXO-family transcription factor. Reduction of DAF-2 activity thus activates the longevity-promoting effects of DAF-16. (3) The dietary restriction (DR) pathway has been reported to act independently of IIS. Recent studies suggest that DR inhibits TOR activity, and thereby reduces protein translation and promotes autophagy in *C. elegans*. This model has been proposed as a possible mechanism for the lifespan-extending effects of DR. Interestingly, crosstalk exists between TOR signaling and components of other signaling pathways, such as interactions with DAF-16 or HIF-1. (4) Exposure to hypoxic conditions results in inhibition of VHL-1-dependent ubiquitination of HIF-1 and the subsequent stabilization of HIF-1 to promote lifespan extension. Deletion of *hif-1*, however, has also been found to have a temperature-dependent effect on lifespan, with *hif-1* mutants being long-lived at 25°C but showing similar lifespan to wild-type worms at 15°C or 20°C. Temperature-dependent lifespan extension by deletion of *hif-1* is dependent on DAF-16 under some conditions and may act via TOR-signaling under other conditions (dotted lines). (5) Mutation or RNAi knockdown of genes encoding components of the mitochondrial electron transport chain (ETC) can promote longevity by two different mechanisms: activating the mitochondrial unfolded protein response (UPR<sup>mt</sup>) and stabilization of HIF-1.

510 Current Genomics, 2012, Vol. 13, No. 7 Yanos et al.

# A Cautionary Note on Methodology

One of the unique features about *C. elegans* as a model organism is that the vast majority of studies are performed in a common genetic “wild type” background referred to as N2. Thus, it is often assumed that experimental methodology among different labs is relatively constant. However, recent controversies have demonstrated that even seemingly subtle differences in methods used for longevity studies can have substantial effects on outcome. Three important methodological components that can influence lifespan are (1) the use of live versus dead *E. coli* as a food source, (2) the temperature at which the experiments are performed, and (3) the use of 5-fluorodeoxyuridine (FUDR) to prevent hatching of progeny.

A majority of *C. elegans* labs use the *E. coli* strain OP50 as a food source. The bacteria are cultured on the surface of a nutrient-agar Nematode Growth Medium (NGM), and the animals crawl through the bacterial lawn to feed [41]. Several years ago, it was discovered that simply killing the *E. coli* food via UV-radiation or antibiotic treatment was sufficient to increase mean and maximum lifespan of *C. elegans* [42], and a similar effect was found when a non-pathogenic *Bacillus subtilis* food source was used [43]. This has led to the model that bacterial pathogenesis limits the survival of the population when live bacterial food is used. Interestingly, several long-lived mutants, including components of the IIS pathway, are resistant to known bacterial pathogens [43, 44], raising the possibility that at least a portion of the lifespan extension in these cases results not from a direct effect on aging but instead from reduced death due to infection. For this reason, some groups (including ours) choose to use killed bacterial food for a majority of lifespan experiments, with the exception of RNAi experiments (see below for details) [41]. One consequence of this is that some interventions that appear to extend lifespan on live *E. coli* may no longer extend lifespan on killed *E. coli* if their mechanism of action involves reducing death due to bacterial pathogenesis.

Temperature was one of the first major determinants of *C. elegans* longevity to be identified. Similar to the case in fruit flies [45], lifespan decreases as temperature increases, within the range of about 10°C–30°C [46]. The most common temperature used for studies of *C. elegans* aging is 20°C; however, many groups use 25°C, and a few use 15°C. It has been generally assumed that an intervention that extends or shortens lifespan at one temperature will have a proportionately similar effect on lifespan at another temperature, but there is little experimental data to support this assumption. Instead, recent studies of HIF-1 have demonstrated that, at least for this longevity factor, temperature is a key determinant of whether deletion of *hif-1* shortens or lengthens lifespan [47]. At 25°C, animals carrying the *hif-1(ia4)* null allele are long-lived, while at 15°C they have a lifespan that is not significantly different from wild type N2 animals [32, 38]. Contradictory results have been reported for experiments performed at 20°C, with some studies showing lifespan extension and others detecting no effect on lifespan [32–35, 48]. These conflicting reports initially resulted in confusion, until the temperature-dependent nature of the *hif-1(ia4)* longevity phenotype was uncovered [49]. Interestingly, stabilization of HIF-1 by deletion of the gene coding for the VHL-1 E3 ligase extends lifespan throughout the temperature range from 15–25°C [32]. Thus, temperature plays a critical role in determining the longevity effect of *hif-1* deletion but has little, if any, effect on the lifespan extension from HIF-1 stabilization (Fig. 1).

The chemical FUDR has become widely used in *C. elegans* aging studies, in order to prevent hatching of progeny during lifespan experiments [41, 50]. If FUDR is not used, experimental animals must be transferred to fresh NGM plates every 24 to 48 hours during the first 7 to 14 days of the experiment. In contrast, when FUDR is added to the NGM beginning at L4, animals will become sterile and need only be transferred to new plates when the food supply becomes limiting. Several groups have reported that at concentrations used for aging studies (50–100 μg/ml), FUDR does not alter the longevity of wild type N2 animals [51]. However, recent work suggests that some genes influence longevity differently depending on whether FUDR is present or not. For example, FUDR has been found to significantly increase lifespan in mutants deficient in *tub-1*, a gene involved in modulation of fat storage, and a similar lifespan-extending effect has been reported for the mitochondrial mutant *gas-1* [52, 53]. The effects of knock-down of genes that modulate histone H3 lysine 4 trimethylation also vary depending on whether animals are sterile or fertile [54]. Thus, *C. elegans* aging studies that utilize FUDR may yield results that are different from studies performed in the absence of FUDR, as the chemical itself may be acting on specific genes and genetic pathways under investigation.

## Methods and Mechanisms of Dietary Restriction

DR, which has been described as nutrient restriction without malnutrition, is without question the most widely studied longevity intervention. There are reports of lifespan extension from DR in many different species, including yeast, nematodes, fruit flies, mice, rats, dogs, fish, spiders, and monkeys [55–57].

In addition to *eat-2* mutation, a number of methods for DR have been developed for *C. elegans* utilizing solid and liquid mediums, and each method has been shown to be effective in increasing lifespan [38, 46, 58–62]. However, a complete understanding of the mechanism by which DR promotes longevity is complicated by results showing that different DR methods extend lifespan by both independent and overlapping genetic pathways [63, 64]. Whereas DR by bacterial deprivation on solid medium, *eat-2* mutation, or axenic liquid medium extends lifespan independent of *daf-16*, lifespan extension resulting from bacterial dilution in liquid medium is at least partially dependent on *daf-16*. Thus, individual genes that have been identified to act within the DR longevity pathway may be specific to the DR method under study.

Interestingly, mutations that cause defects in sensory perception in *C. elegans* results in *daf-16*-dependent lifespan extension, suggesting that longevity is partially regulated by environmental cues [65]. This hypothesis is further supported by research showing that laser ablation of specific sensory neurons, individually or in combination, has also been found to extend lifespan [66]. Food sensing has also been shown to play a role in lifespan extension from DR that

is independent of *daf-16* [67], suggesting that sensory perception can modulate lifespan by multiple mechanisms.

Recently, the possible involvement of the TOR pathway in DR has generated considerable interest in the aging field. The TOR pathway is a highly conserved pathway that has been implicated in regulation of metabolic processes and aging in both invertebrate and mammalian models [68, 69]. Molecular components of TOR signaling were originally identified through yeast studies involving rapamycin, a drug that is known to specifically inhibit TOR complex 1 (TORC1) signaling. [70]. TORC1 consists of TOR kinase partnered with Raptor, PRAS40 (proline-rich AKT substrate 40 kDa), and a Gβ L protein [71]. Inhibition of the TORC1 results in decreased protein translation, through reduced phosphorylation of S6 kinase (S6K) or activation of 4E-binding protein (4EBP), an inhibitor of eukaryotic translation initiation factor 4E (eIF4E) [72, 73].

Several lines of evidence suggest that TORC1 signaling plays an important role in lifespan extension from DR in different organisms; genetic or pharmacological inhibition of TORC1 is sufficient to extend lifespan in yeast, nematodes, flies, and mice, and in yeast, nematodes, and flies, inhibition of TORC1 fails to increase lifespan additively with DR [74]. In *C. elegans*, reduced mRNA translation and induction of autophagy downstream of TORC1 appear to be important for lifespan extension [75, 76].

### Genome-Wide RNAi Methods in *C. elegans*

Before RNAi technology was developed, aging research in *C. elegans* was largely limited to “forward” genetic screens in which random mutations were identified, lifespan quantified, and in some cases the gene was identified through classical methods. The discovery of gene knockdown by RNA interference (RNAi) in *C. elegans* paved the way for a revolution in the pace of aging research. Initially, injection of double-stranded RNA was found to be an effective method of genetic interference [77]. Soon afterwards, it was found that soaking worms in dsRNA solution or feeding worms bacteria engineered to produce dsRNA could also induce RNAi knockdown [78, 79], and that feeding was as equally as effective as injection methods [80]. Development of the bacterial feeding RNAi method led to construction of genome-wide RNAi libraries, which have become extremely useful tools for *C. elegans* research. The Ahringer library contains 16,757 clones, which were generated by cloning gene-specific genomic fragments between two inverted T7 promoters [81, 82]. The Vidal library was more recently developed and contains 11,511 clones, produced by Gateway cloning of full-length open reading frame (ORF) cDNAs into a double T7 vector [83].

The construction of genome-wide RNAi libraries in *C. elegans* provided a more systematic way of studying aging. For the first time, individual genes could be knocked down and the effect on lifespan directly measured for a majority of the *C. elegans* genes. Other aging-associated phenotypes, such as fecundity, developmental rate, and lipofuscin accumulation could be measured on a large scale. To date, hundreds of RNAi screens have been performed in *C. elegans* for a variety of phenotypes, and much of this data is curated on Wormbase [84]. The foremost purpose of this review is to

revisit the first large-scale genetic screens that identified a number of functional classes of genes that influence aging and to compare and contrast these studies. We will provide an in-depth analysis of the consistencies and inconsistencies between published results, and highlight differences in methodologies that may have influenced their findings. Lastly, an important aim of this review will be to relate the early publications with successive trends in aging research.

#### INITIAL RNAi SCREENS TO IDENTIFY LONGEVITY GENES AND PATHWAYS

The era of longevity genomics was ushered in with parallel genome-wide RNAi screens for increased lifespan performed by the Ruvkun and Kenyon labs. The first published reports from both labs examined genes on chromosomes I and II and came to the same primary conclusion: knockdown of mitochondrial electron transport chain (ETC) genes by RNAi is sufficient to extend lifespan in *C. elegans* [85, 86]. The Kenyon lab study authored by Dillin *et al.* was published first, and established that reduced ETC function extended lifespan of both *daf-2* and *daf-16* mutant animals, suggesting that mitochondrial respiration controlled longevity independent of the IIS pathway. The RNAi clones with the most robust effect on lifespan were *nuo-2*, *cyc-1*, *cco-1*, and *atp-3* which are subunits of complex I, III, IV, and V, respectively [85].

Consistent with these observations, the Ruvkun lab publication authored by Lee *et al.* [86] found that 15% of the RNAi clones that extended lifespan in their screen corresponded to nuclear encoded mitochondrial genes. Similar to Dillin *et al.*, this study also found that knockdown of complex I, III, and IV subunits could extend lifespan of both wild type and *daf-16(mgDf47)* animals. Interestingly, Lee *et al.* also identified a handful of mitochondrial RNAi clones, such as mitochondrial carrier gene F13G3.7 and phosphoglycerate mutase gene F57B10.3, that extended lifespan in a DAF-16-dependent manner, thus suggesting interplay between mitochondrial function and IIS with respect to longevity control. In addition, the researchers found that RNAi inactivation of only 20% of known mitochondrial genes tested had a detectable lifespan phenotype. This may be explained by the efficiency of knockdown inherent to different RNAi clones; or perhaps a more intriguing possibility is that distinct types of mitochondrial dysfunction lead to lifespan extension.

Both labs proceeded to publish follow-up papers describing the comprehensive results of their screens of the entire Ahringer RNAi library [87, 88]. Epistasis analysis was used to place the newly identified longevity genes into pathways and functional categories. The results of these epistasis tests are summarized below (Table 1). Hansen *et al.* [87] reported 23 additional novel longevity factors along with previously reported longevity genes including *cyc-1*, *cco-1*, *nuo-2*, *atp-3*, *cchl-1* and *daf-2*. To study if the newly identified longevity genes acted within the IIS and germline pathways, the effect of RNAi on lifespan was determined in *daf-16(mu86)* animals and *daf-2(e1370)* animals. For those genes showing DAF-16-dependent lifespan extension, RNAi knockdown was also performed in *glp-1(e2141)* animals that lack a germline when raised at high temperatures and *daf-12(rh61rh411)* animals.

Table 1. Epistasis Relationships for Longevity Genes. A Limited Set of Epistasis Studies have been Performed in Conjunction with Different RNAi Screens to Characterize Novel Longevity Genes. Data are shown for Experiments Combining RNAi Knock-down of the Longevity Genes in Combination with Mutation of *daf-16*, *glp-1*, *eat-2*, or *sir-2.1*. Dependence Indicates that the Factor is Required for Further Life Span Extension from the RNAi Clone (e.g. DAF-16 Dependence means the RNAi Clone Fails to Extend Life Span in a *daf-16* Mutant Background). Genes Annotated As Mitochondrial are Listed in the “Mitochondrial” Group. Gene Names are Annotated According to Wormbase. Superscript Numbers Represent the Study in which the RNAi Clone was Identified. ${}^{1}$Chen, D., *et al.*, Aging Cell, 2007, ${}^{2}$Curran, S.P. and G. Ruvkun, PLoS Genet, 2007, ${}^{3}$Dillin, A., *et al.*, Science, 2002, ${}^{4}$Hamilton, B., *et al.*, Genes Dev, 2005, ${}^{5}$Hansen, M., *et al.*, PLoS Genet, 2005, ${}^{6}$Hansen, M., *et al.*, Aging Cell, 2007, ${}^{7}$Kim, Y. and H. Sun, Aging Cell, 2007, ${}^{8}$Lee, S.S., *et al.*, Nat Genet, 2003, ${}^{9}$Pan, K.Z., *et al.*, Aging Cell, 2007

| DAF-16 |
| --- |
| Dependent |
| $aat-8^{7}, acs-5^{7}, age-1^{4}, akt-1^{4}, amt-2^{7}, asm-3^{7}, atp-2^{2}, B0546.3^{7}, C09B7.2^{4}, C32H11.1^{4}, C36H8.1^{4}, C39E9.1^{4}, C42C1.3^{7}, C46G7.2^{7}, C56G2.1^{2}, cct-6^{2}, ced-3^{2},$<br>$col-93^{4}, cpna-3^{4}, crn-5^{2}, cutl-28^{7}, cwp-4^{4}, D1054.14^{2}, daf-2^{5,7,2}, ddl-1^{5}, ddl-2^{5}, ddl-3^{5}, eIF2b/iftb-1^{6}, eIF4G/ifg-1^{6}, epc-1^{7}, erm-1^{2}, ero-1^{2}, F09C11.1^{7},$<br>$F13G3.7^{8}, F19B6.1^{2}, F21H12.1^{4}, F26A3.4^{2}, F28B3.5^{8}, F42A6.1^{7}, F49C12.9^{7}, F53F4.11^{2}, F55B11.1^{4}, F56D5.5^{7}, F57B10.3^{8}, gei-15^{2}, glp-1^{2}, gpi-1^{5}, grl-19^{7},$<br>$hrp-1^{2}, inx-14^{4}, inx-8^{7}, inx-9^{7,2}, K07H8.8^{7}, maoc-1^{5}, mcm-2^{2}, mrpl-50^{2}, nol-1^{2}, npa-1^{2}, nuo-1^{2}, pod-1^{7}, pos-1^{2}, prx-5^{2}, R05A10.5^{4}, R10H10.7^{7}, ral-1^{7}, sas-5^{2},$<br>$scrm-8^{4}, sel-5^{2}, sem-5^{2}, set-9^{4}, sinh-1^{5}, spe-26^{7}, spg-7^{2}, sre-25^{2}, str-49^{2}, T05A1.5^{7}, T05G5.10^{4}, T06G6.4^{4}, T07A9.8^{2}, tag-300^{2}, tpa-1^{2}, trim-9^{4}, ttr-1^{5}, ttr-5^{7},$<br>$unc-52^{2}, W07G9.1^{7}, wdr-23^{2}, wts-1^{2}, Y105C5B.12^{2}, Y119D3_446.d^{7}, Y39F10C.1^{4}, Y39H10A.6^{4}, Y43h11AL.2^{4}, Y45F10D.8^{7}, Y46H3C.1/.2^{4}, Y51H4A.m^{7},$<br>$Y54E10BR.4^{2}, Y56A3A.19^{2}, Y56A3A.9^{4}, Y65B4BR.5^{2}, Y71H2_390.d^{7}, Y71H2AR.2^{4}, Y75B8A.13^{4}, Y75B8A.33^{4}, ZC132.3^{2}, ZK1127.5^{2}, ZK686.2^{2}$ |
| Dependent on GLP-1; Germline Signaling |
| $ddl-1^{5}, ddl-2^{5}, ddl-3^{5}, sinh-1^{5}, ttr-1^{5}$ |
| Independent |
| $abcx-1^{1}, acdh-13^{7}, aco-2^{4}, asb-2^{5}, atp-3^{3,5,2}, atp-4^{5}, atp-5^{5}, B0491.5^{1}, B0511.6^{2}, C48E7.2^{1}, cchl-1^{5,8}, cco-1^{3,5,8}, cco-2^{5}, cct-4^{2}, ceh-18^{4}, cyc-1^{3,5}, D2030.4^{8},$<br>$D2085.3^{1}, dic-1^{2}, drr-1^{5}, drr-2^{5}, egl-45^{2}, eif-1^{2}, eif-3.B^{2}, eif-3.F^{2,1}, eIF4G/ifg-1^{9}, elt-6^{7}, exos-3^{1}, F08B4.7^{7}, F09F7.5^{4}, F13B6.1^{7}, F14B4.3^{1}, F26E4.4^{1},$<br>$F26E4.6^{8,2}, F33D4.5^{7}, F59C6.5^{1,2}, hhat-2^{7}, htp-3^{2}, ifg-1^{2}, inf-1^{2}, K01C8.7^{8}, K08E3.5^{4}, K11B4.1^{1}, K12H4.5^{7}, LLC1.3^{7}, lpd-5^{1}, mars-1^{7}, mrpl-10^{1}, mrpl-24^{1},$<br>$mrpl-47^{8}, mrps-30^{1}, mtch-1^{2}, nars-1^{1}, nol-5^{1}, nuo-2^{3,5}, nuo-3^{5}, nuo-4^{4,5}, nuo-5^{5}, pat-4^{5,7,2}, pat-6^{5}, phi-37^{2}, qars-1^{7}, R08E3.3^{4}, R53.4^{1}, rab-10^{5}, rha-2^{5}, ril-1^{5},$<br>$ril-2^{5}, rpl-19^{1,6}, rpl-4^{6}, rps-11^{2}, rps-15^{6}, rps-22^{6}, rps-23^{7}, rps-3^{2}, rps-5^{7}, rps-8^{2}, S6K/rsks-1^{6,9}, sams-1^{5}, scl-8^{7}, sco-1^{1}, spg-7^{1}, spt-4^{4}, symk-1^{2}, T02H6.11^{8},$<br>$T20H4.5^{4}, T22B11.5^{7}, T28A8.6^{7}, T28D6.4^{7}, tars-1^{1}, tes-1^{7}, tkt-1^{7}, TOR/let-363^{6}, unc-62^{2}, vha-6^{2}, W03G1.5^{7}, W09C5.8^{8}, Y39G10AR.8^{1}, Y48G1A.4^{1},$<br>$Y54E5A.7^{4}, Y66A7A1^{7}, Y69A2A_2991.c^{7}, Y71H2_388.c^{7}$ |

| EAT-2 |
| --- |
| Dependent |
| $drr-1^{5}, drr-2^{5}, pat-4^{5}, rab-10^{5}, sams-1^{5}, TOR/let-363^{6}$ |
| Independent |
| $abcx-1^{1}, B0491.5^{1}, C48E7.2^{1}, D2085.3^{1}, eIF2b/iftb-1^{6}, eif3.F^{1}, eIF4G/ifg-1^{6,9}, exos-3^{1}, F14B4.3^{1}, F26E4.4^{1}, F59C6.5^{1}, K11B4.1^{1}, lpd-5^{1}, mrpl-10^{1}, mrpl-24^{1},$<br>$mrps-30^{1}, nars-1^{1}, nol-5^{1}, pat-6^{5}, R53.4^{1}, rha-2^{5}, ril-1^{5}, ril-2^{5}, rpl-19^{1}, rps-15^{6}, rps-22^{6}, S6K/rsks-1^{6,9}, sco-1^{1}, spg-7^{1}, tars-1^{1}, Y39G10AR.8^{1}, Y48G1A.4^{1}$ |

| SIR2.1 |
| --- |
| Dependent |
| $C36H8.1^{4}, C39E9.1^{4}, ceh-18^{4}, cpna-3^{4}, F09F7.5^{4}, F21H12.1^{4}, F55B11.1^{4}, R05A10.5^{4}, scrm-8^{4}, Y71H2AR.2^{4}, Y75B8A.13^{4}$ |
| Independent |
| $aco-2^{4}, age-1^{4}, akt-1^{4}, C09B7.2^{4}, C32H11.1^{4}, col-93^{4}, cwp-4^{4}, eIF2b/iftb-1^{6}, eIF4G/ifg-1^{9}, inx-14^{4}, K08E3.5^{4}, nuo-4^{4}, R08E3.3^{4}, rps-15^{6}, S6K/rsks-1^{6,9}, set-$<br>$9^{4}, spt-4^{4}, T05G5.10^{4}, T06G6.4^{4}, T20H4.5^{4}, TOR/let-363^{6}, trim-9^{4}, Y39F10C.1^{4}, Y39H10A.6^{4}, Y43h11AL.2^{4}, Y46H3C.1/.2^{4}, Y54E5A.7^{4}, Y56A3A.9^{4},$<br>$Y75B8A.33^{4}$ |

| Mitochondrial |
| --- |
| $aco-2^{4}, ant-1.1^{7}, asb-2^{5}, asg-2^{4}, atp-2^{2}, atp-3^{3,5,2}, atp-4^{5}, atp-5^{5}, C33A12.1^{7}, C33F10.12^{4}, C47E12.2^{7}, cchl-1^{5,8}, cco-1^{3,4,5,8}, cco-2^{5}, cyc-1^{3,5}, cyc-2.1^{7}, cyp-$<br>$33E2^{7}, D2030.4^{4,8}, F13G3.7^{8}, F26E4.6^{4,8,2}, F28B3.5^{8}, F29C4.2^{7}, F43G9.1^{4}, F55B11.1^{4}, F57B10.3^{4,8}, F59C6.5^{1,2}, idh-1^{4}, K01C8.7^{8}, K12H4.5^{7}, lpd-5^{1}, mrpl-$<br>$10^{1}, mrpl-12^{7}, mrpl-24^{1}, mrpl-47^{4,8}, mrps-30^{1}, mrps-33^{7}, mrps-9^{7}, mtch-1^{2}, nduf-2.2^{7}, nuo-1^{2}, nuo-2^{3,5}, nuo-3^{5}, nuo-4^{4,5,7}, nuo-5^{5}, phi-37^{2}, qars-1^{7}, R53.4^{1},$<br>$sco-1^{1}, spg-7^{1,2}, T02H6.11^{8}, T20H4.5^{4,7}, T22B11.5^{7}, tag-174^{7}, ucr-1^{7}, W09C5.8^{4,8}, Y119D3_463.b^{7}, Y37D8A18^{7}, Y53G8A_9248.c^{7}, Y53G8A_9248.d^{7},$<br>$Y55F3B_743.b^{7}, Y56A3A.19^{7,2}, Y57G11C.12^{7}, Y71H2_378.a^{7}, Y71H2_388.d^{7}, yars-1^{7}, ZK809.3^{7}$ |

No Epistasis

$ant-1.1^{7}$, $asg-2^{4}$, C18E9.4/.10${}^{4}$, C26B2.2${}^{4}$, C27B7.7${}^{4}$, C32H11.5${}^{4}$, C33A12.1${}^{7}$, C33F10.12${}^{4}$, C35A11.3${}^{4}$, C47E12.2${}^{7}$, $cec-3^{4}$, $clec-186^{4}$, $clec-227^{4}$, $cyc-2.1^{7}$,

$cyp-33E2^{7}$, D1054.8${}^{4}$, E03H12.5${}^{4}$, $egl-3^{4}$, F29C4.2${}^{7}$, F35D2.3${}^{4}$, F40F8.5${}^{4}$, F43G9.1${}^{4}$, F49F1.12${}^{4}$, F55B11.5/.3${}^{4}$, $gcy-29^{4}$, H06H21.8${}^{4}$, $idh-1^{4}$, K07H8.1${}^{4}$,

K10B4.3${}^{4}$, $max-1^{4}$, $mecr-1^{4}$, $mrpl-12^{7}$, $mrps-33^{7}$, $mrps-9^{7}$, $nas-38^{4}$, $nduf-2.2^{7}$, $nhr-14^{4}$, $nhr-154^{4}$, $pfn-2^{4}$, $pghm-1^{4}$, $pup-2^{4}$, $rmf-5^{4}$, $rpl-30^{6}$, $rpl-6^{6}$, $rps-$

$10^{6}$, $rps-26^{6}$, $rps-6^{6}$, $set-15^{4}$, $set-18^{8}$, $sid-2^{4}$, $sru-17^{4}$, $srw-20^{4}$, T05A1.4${}^{4}$, T26H5.1${}^{4}$, $tag-174^{7}$, $tag-60^{4}$, $tba-7^{7}$, $ubh-4^{4}$, $ucr-1^{7}$, $unc-83^{4}$, Y119D3_463.b${}^{7}$,

Y37D8A.12${}^{4}$, Y37D8A18${}^{7}$, Y39A3C_82.a${}^{7}$, Y43F4B.7${}^{7}$, Y43F8B.12${}^{4}$, Y46H3C.6${}^{4}$, Y53F4B.23${}^{4}$, Y53G8A_1734.g${}^{7}$, Y53G8A_2702.a${}^{7}$, Y53G8A_9248.b${}^{7}$,

Y53G8A_9248.c${}^{7}$, Y53G8A_9248.d${}^{7}$, Y54E5A.2${}^{8}$, Y55F3B_743.b${}^{7}$, Y71H2_378.a${}^{7}$, Y71H2_385.b${}^{7}$, Y71H2_388.d${}^{7}$, Y92C3A.1${}^{4}$, $yars-1^{7}$, ZK809.3${}^{7}$

GLP-1 and DAF-12 both function in the germline longevity pathway (Fig. 1) to modulate longevity through DAF-16. The genes $ttr-1$, $sinh-1$, $ddl-1$, $ddl-2$, and $ddl-3$ were identified from this analysis as novel DAF-16-dependent longevity genes that also function, at least partially, in the germline signaling pathway.

In order to determine whether newly identified longevity genes acted by a mechanism similar to DR, Hansen *et al.* [87] combined RNAi knockdown with mutation of $eat-2$. Knockdown of $sams-1$ (C49F5.1), $rab-10$ (T23H2.5), $drr-1$ (F45H10.4), and $drr-2$ (T12D8.2) were unable to further extend the lifespan of $eat-2$ animals, suggesting that they function in the DR pathway. In contrast, RNAi knockdown of ETC components further extended the lifespan of $eat-$ 2($adl1116$) animals, supporting the idea that DR and ETC knockdown act by distinct mechanisms. Interestingly, a total of only three genes were unable to be placed into known longevity pathways including two proteins with no obvious homologs, $ril-1$ (C53A5.1) and $ril-2$ (C14C10.3), and the DEAH RNA helicase $rha-2$ (C06E1.10).

The genome-wide RNAi study conducted by the Ruvkun lab, authored by Hamilton *et al.* [88], identified a total of 89 additional aging genes with disparate functions including cell structure, cell surface proteins, cell signaling, cellular metabolism, and protein turnover. Of the 66 genes with previously known functions, 17 corresponded to various aspects of carbon metabolism, including citric acid cycle enzymes and subunits of complexes I, IV, and V of the ETC. Researchers also speculated that protein translation might play a role in lifespan regulation, based on the identification of $iff-1$ (T05G5.10), a gene that has homology to the translation initiation factor eIF5A. Other hits from this screen included two genes containing PH domains known to interact with phosphatidylinositol lipids, multiple G protein-coupled receptors, protein processing and degradation genes such as proteases and ubiquitin ligases/hydrolases, and chromatin modifying factors.

To further investigate genes that directly affect mitochondrial function, Hamilton *et al.* [88] measured the induction of the mitochondrial unfolded protein response (UPR${}^{\mathrm{mt}}$) using a GFP reporter strain for activation of HSP-6, a mitochondrial chaperone. Impaired assembly of mitochondrial multi-subunit complexes or disruptions in protein folding or processing caused by inactivation of mitochondrial chaperones and proteases has been previously shown to induce this stress response [89]. They determined that RNAi of ETC subunits and aconitase activated the $hsp-6::$ GFP reporter, in addition to genes of unknown function Y53F4B.23 and Y75B8A.33, the tripartite motif-containing protein C39F7.2, and the cadherin homolog $cdh-12$.

Tests for epistasis placed 35 of the identified genes with the most robust effects on lifespan within IIS or sirtuin longevity pathways. SIR-2.1 is orthologous to the yeast Sir2 protein, which functions as an NAD-dependent histone deacetylase [90]. Overexpression of Sir2 is sufficient to extend lifespan in yeast [91, 92], and overexpression of $sir-2.1$ was reported to also extend lifespan in *C. elegans* [93], although that result has since become controversial [94]. Of the 35 RNAi clones tested, 26 required DAF-16 for lifespan extension, 11 required SIR-2.1, and 9 required both genes. This indicates that SIR-2.1 and DAF-16 likely influence lifespan in *C. elegans* via both overlapping and distinct mechanisms.

The initial genome-wide RNAi screens revealed that knockdown of hundreds of genes can have positive effects on *C. elegans* longevity. Interestingly, the majority of the genes could be placed into canonical longevity pathways including insulin/IGF, germline, TOR signaling, and mitochondrial respiration. Thus, it may be that the majority of signaling pathways that contribute to aging in *C. elegans* have already been discovered. The recurrent identification of mitochondrial genes in each of the screens collectively suggests that attenuating mitochondrial function in *C. elegans* has positive effects on lifespan, despite the effects of lower ATP production, small body size, slow pumping, slow growth rate, and sterility [85, 86]. Initially, researchers hypothesized that decreasing mitochondrial function lowers oxidative damage and thereby promotes longevity. As described in the next section, however, there is not a perfect correlation between resistance to ROS and longevity. Furthermore, the reduction in mitochondrial function must be a partial rather than severe in order to extend lifespan, as shown by RNAi dilution experiments [95]. Recently, the Dillin lab has proposed a model in which induction of UPR${}^{\mathrm{mt}}$ can promote longevity [96]. RNAi of several ETC components and some mitochondrial genes shown to increase lifespan, also induce this response [88, 89]. Thus, an intriguing possibility is that UPR${}^{\mathrm{mt}}$ induction may better protect *C. elegans* from protein misfolding and aggregation events that occur with age.

COMPARISON OF GENOME-WIDE LONGEVITY SCREENS AND METHODOLOGIES USED

Together, the initial genome-wide longevity screens performed by the Kenyon and Ruvkun labs identified approximately 120 RNAi clones that significantly increased lifespan (Table 1). Although similar functional groups were identified in each screen, somewhat surprisingly, only 4 RNAi clones were found to extend lifespan by both labs, corresponding to

| Paper      | Initial/Secondary Strain | RNAi | Temp.(°C)    | FUDR | CH   | Epistasis                          |
|------------|-------------------------|------|--------------|------|------|-----------------------------------|
| Dillin     | N2/N2                  | Egg  | 25           | NA   | I    | daf-16(mu 86)                      |
|            |                         |      |              |      |      | daf-2(e 1370)                      |
| Lee        | N2/N2                  | L1   | 20^a         | +    | I/II | daf-16(mgDf 47)                    |
| Hansen     | fer-15; fem-1/N2       | Egg  | 20 and 25^b  | -    | ALL  | daf-16(mu 86)                      |
|            |                         |      |              |      |      | daf-2(e 1370)                      |
|            |                         |      |              |      |      | glp-1(e 2141)                      |
|            |                         |      |              |      |      | daf-12(rh61rh 411)                 |
|            |                         |      |              |      |      | eat-2(adl 116)                     |
| Hamilton   | N2/rrf-3               | L1   | 25^c         | +    | ALL  | daf-16(mgDf 47)                    |
|            |                         |      |              |      |      | sir-2.1(ok 434)                    |

adversely affect development at early embryonic and larval stages. Of the 57 genes that were screened, RNAi knockdown of 24 clones was found to extend lifespan significantly in wild-type *C. elegans*. Most of these genes have been implicated in regulation of mRNA translation and mitochondrial functions. RNAi knockdown of all the identified genes further extended lifespan of both *daf-2* mutants and *daf-16; daf-2* double mutants, leading researchers to conclude that these genes are likely to affect lifespan independently of IIS. In addition, the lifespan-extending RNAi clones were also assayed for effects on stress-resistance and fecundity. All of the RNAi clones tested, except for one, reduced fecundity significantly compared to wild-type. Knockdown of most of these genes also resulted in increased resistance to heat stress, while knockdown of ~50% of these genes resulted in increased resistance to juglone, a compound that induces oxidative stress.

A similar study by Curran and Ruvkun [99] investigated a broader set of RNAi clones, screening knockdown of 2,700 genes essential for development. Post-developmental RNAi knockdown of 64 (~2.4%) of these genes was found to extend lifespan, a 4-fold increase in frequency of lifespan extension relative to their prior genome-wide screen (89/16,000, ~0.6%) [88]. Major functional classes of genes uncovered in this study include those involved in protein synthesis, RNA binding/processing and chromatin-associated factors, signaling molecules, and mitochondrial function. Lifespan extension for many of the identified genes, most notably those that encode signaling molecules and RNA-related genes, was dependent on *daf-16*, suggesting that these genes act within the IIS pathway to regulate longevity. Other gene inactivations, particularly those targeting protein synthesis machinery and mitochondrial function, appeared to extend lifespan independent of *daf-16*. However, many of the RNAi clones that were not dependent on *daf-16* for lifespan extension still induced nuclear localization of *daf-16* and expression of its target *sod-3*, as visualized in transgenic GFP animals; this was especially true for genes involved in protein synthesis.

Comparison of these two screens of genes required for development [98, 99] reveals only four genes in common – *eIF-3f* (D2013.7), F59C6.5, *spg-7* (Y47G6A.10) and *atp-3* (F27C1.7) – and technical considerations may partially explain the lack of overlap. Chen *et al.* tested RNAi knockdown of essential genes in wild-type (N2) animals, whereas Curran and Ruvkun utilized the RNAi-sensitive *eri-1(mg366)* strain. Furthermore, Chen *et al.* tested a very small subset of 57 essential genes, whereas Curran and Ruvkun tested 2,700. The results of epistasis experiments relating to the IIS pathway also showed some inconsistencies between studies; whereas Chen *et al.* found that all lifespan extending RNAi clones from their screen could extend lifespan independent of *daf-16*, Curran and Ruvkun found many genes appear to act within the IIS pathway to regulate longevity. However, as with the different genome-wide RNAi screens, both studies yielded similar functional classes of genes. Notably, both RNAi screens revealed longevity genes that are involved in mitochondrial function, including two genes, F26E4.6 and *atp-3*, that had been identified to increase lifespan in previous genome-wide screens [88]; however,

prior research had indicated that lifespan extension resulting from inactivation of mitochondrial genes required initiation of RNAi treatment during larval development [85]. Both groups speculated that the discrepancy might be explained by the variable effectiveness of RNAi. Curran and Ruvkun utilized an RNAi-sensitive strain in which post-developmental knockdown of mitochondrial components might be effective, and Chen *et al.* identified mitochondrial RNAi clones with strong larval arrest phenotypes in wild-type worms. It may also be that the novel mitochondrial genes from these studies extend lifespan by a mechanism that is different from the mitochondrial genes that were examined in previous screens.

Among the essential genes that influence aging, one particularly important class are factors that are involved in cytoplasmic mRNA translation [100]. Curran and Ruvkun identified 10 genes that promote mRNA translation and protein synthesis; several of these RNAi clones targeted components of the eukaryotic translation initiation factor complex (eIF4F) and increased lifespan up to 50% longer than with vector control RNAi. Chen *et al.* identified 10/24 genes involved in translation initiation as well as other components of translational machinery such as ribosomal biogenesis and tRNA synthesis. The involvement of protein synthesis and translation in aging has been further validated by studies from the Kenyon, Tavernarakis, and Kapahi labs, demonstrating that reduction in mRNA translation is often associated with increased lifespan in *C. elegans* [76, 101, 102]. RNAi of multiple ribosomal protein genes and translation initiation factors can extend lifespan, as can mutant alleles of ribosomal-protein S6 kinase (*S6K/rsks-1*) and the translation initiation factors eIF4G (*igf-1*), and eIF4E (*ife-2*).

IDENTIFYING LONGEVITY GENES USING CORRELATED PHENOTYPES

Rather than use extended lifespan as the primary phenotype, some groups have screened for alternative phenotypes, such as altered stress resistance, prior to testing for effects on longevity. The list of such studies is extensive and beyond the scope of this review; however, two representative examples are described in detail here.

Kim and Sun [103] sought to explore the relationship between ROS sensitivity and lifespan in *C. elegans* by performing an extensive screen to identify RNAi clones that provided resistance to the ROS-generating compound paraquat, and then screened positive hits for effects on longevity. A total of 84 RNAi clones were identified that conferred paraquat resistance and greater than 10% lifespan extension. Epistasis experiments revealed that 29/52 non-mitochondrial hits from this screen were *daf-16*-dependent. These genes include proteins with disparate functions including the transport of nutrients, stress responses, chromatin remodeling/gene expression, protein-protein interactions, and protein turnover. Among the *daf-16*-independent longevity genes were *pat-4* (C29F9.7), which encodes the integrin-linked kinase (ILK) that was also identified in an independent screen [87]. Some RNAi clones differentially increased lifespan of *daf-16* null mutants and wild-type animals, such as ribosome components *rps-23* (F28D1.7) and F33D4.5, suggesting a partial interaction with IIS.

Interestingly, the majority of RNAi clones that enhanced paraquat resistance did not extend lifespan [103]. This supports the idea that resistance to ROS is not sufficient to increase lifespan. Furthermore, approximately 30% of the hits from this screen are annotated as mitochondrial proteins, such as respiratory chain components, mitochondrial ribosomes, and ADP/ATP carriers. These results are consistent with a model in which superoxide reaction caused by paraquat requires electrons derived from the respiratory chain; however, the majority of reported mitochondrial RNAi clones that extend lifespan also cause decreased resistance to paraquat [86]. Of the longevity genes identified in this study, only *4-pat-4* (C29F9.7), *nuo-3* (Y57G11C.12), *nuo-4* (K04G7.4), and T20H4.5, a NADH-quinone oxidoreductase, -- were previously identified by the initial genome-wide RNAi screens.

Mehta et al., [33] screened RNAi clones related to proteasome function for enhanced resistance to polyglutamine toxicity, under the assumption that enhanced resistance to proteotoxic stress is often an indicator of longevity in *C. elegans*. For this screen, a strain expressing a 35 glutamine repeat fused to GFP in body wall muscle cells was employed [104]. Animals expressing this transgene show an age-dependent increase in aggregation that results in paralysis, without a significant reduction in lifespan [105]. Not surprisingly, Mehta et al. 33] found that a majority of RNAi clones led to enhanced toxicity and shortened lifespan; however, knockdown of the E3 ligase gene *vhl-1* resulted in resistance to polyglutamine toxicity and increased lifespan. VHL-1 is the *C. elegans* homolog of the Von Hippel-Lindau tumor suppressor gene and functions as a negative regulator of HIF-1. Mehta et al. also observed the RNAi knock-down of the EGL-9 prolyl hydroxylase gene, which functions upstream of VHL-1 to promote degradation of HIF-1, also enhances resistance to polyglutamine toxicity and increases lifespan. Several other groups have since confirmed that stabilization of HIF-1 is sufficient to extend lifespan in worms by a mechanism distinct from IIS or DR, establishing this as a major longevity pathway [34, 35, 48]. Interestingly, this pathway also was not identified from the genome-wide longevity RNAi screens, further illustrating the point that many longevity genes still remain to be defined.

### CONCLUSION

Studies utilizing RNAi technology to identify genes and pathways that regulate the aging process in *C. elegans* have yielded an abundance of useful and interesting results. They have helped define the most important pathways and processes modulating longevity in *C. elegans*, and have served to promote the search for conserved longevity factors in other organisms [106]. Although there is consensus that mitochondrial function, mRNA translation, and IIS are key determinants of longevity, the degree of overlap across studies at the individual gene level is surprisingly small. Many factors may contribute to this lack of replication, including important differences in experimental methods and variable effectiveness and specificity of RNAi clones. With more than 300 different RNAi clones already reported to extend lifespan, it remains unknown how many more remain to be identified. Future studies will undoubtedly continue to uncover new longevity genes and mechanisms to provide a deeper understanding of the biology of aging.

---

### CONFLICT OF INTEREST

The author(s) confirm that this article content has no conflicts of interest.

### ACKNOWLEDGEMENTS

This work was supported by NIH Grant R01AG039390 to MK. MEY was supported by NIH Training Grant T32AG000057. MK is an Ellison Medical Foundation New Scholar in Aging.

### REFERENCES

[1] Kaeberlein, M. Longevity genomics across species. *Curr. Genomics*, 2007, 8(2), 73-8.

[2] Fontana, L.; Partridge, L.; Longo, V.D. Extending healthy lifespan--from yeast to humans. *Science*, 2010, 328(5976), 321-6.

[3] Kenyon, C.J. The genetics of ageing. *Nature*, 2010, 464(7288), 504-12.

[4] Sulston, J.E.; Brenner, S. The DNA of *Caenorhabditis elegans*. *Genetics*, 1974, 77(1), 95-104.

[5] Brenner, S. The genetics of *Caenorhabditis elegans*. *Genetics*, 1974, 77(1), 71-94.

[6] Nigon, V. C.R. *Hebd. Seances Acad. Sci.*, 1949, 228(13), 1161.

[7] Nigon, V.; Dougherty, E.C. Reproductive patterns and attempts at reciprocal crossing of *Rhabditis elegans* Maupas, 1900, and *Rhabditis briggsae* Dougherty and Nigon, 1949 (*Nematoda: Rhabditidae*). *J. Exp. Zool.*, 1949, 112(3), 485-503.

[8] Hirsh, D.; Oppenheim, D.; Klass, M. Development of the reproductive system of *Caenorhabditis elegans*. *Dev. Biol.*, 1976, 49(1), 200-19.

[9] Cassada, R.C.; Russell, R.L. The dauerlarva, a post-embryonic developmental variant of the nematode *Caenorhabditis elegans*. *Dev. Biol.*, 1975, 46(2), 326-42.

[10] Klass, M.; Hirsh, D. Non-ageing developmental variant of *Caenorhabditis elegans*. *Nature*, 1976, 260(5551), 523-5.

[11] Riddle, D.L.; Albert, P.S. Genetic and Environmental Regulation of Dauer Larva Development. In: *C. elegans II*. 2nd ed; Cold Spring Harbor Mongraph Series, D.L. Riddle, T. Blumenthal, B.J. Meyer, and J.R. Priess, Eds.; Vol. 33. Cold Spring Harbor Laboratory Press: Cold Spring Harbor, 1997.

[12] Sulston, J.E.; Horvitz, H.R. Post-embryonic cell lineages of the nematode, *Caenorhabditis elegans*. *Dev. Biol.*, 1977, 56(1), 110-56.

[13] Kimble, J.; Hirsh, D. The postembryonic cell lineages of the hermaphrodite and male gonads in *Caenorhabditis elegans*. *Dev. Biol.*, 1979, 70(2), 396-417.

[14] Sulston, J.E.; Schierenberg, E.; White, J.G.; Thomson, J.N. The embryonic cell lineage of the nematode *Caenorhabditis elegans*. *Dev. Biol.*, 1983, 100(1), 64-119.

[15] White, J.G.; Southgate, E.; Thomson, J.N.; Brenner, S. The Structure of the Nervous System of the Nematode *Caenorhabditis elegans*. *Philosophical Transactions of the Royal Society of London. B, Biological Sciences*, 1986, 314(1165), 1-340.

[16] Genome sequence of the nematode *C. elegans*: a platform for investigating biology. *Science*, 1998, 282(5396), 2012-8.

[17] Klass, M.R. A method for the isolation of longevity mutants in the nematode *Caenorhabditis elegans* and initial results. *Mech. Ageing Dev.*, 1983, 22(3-4), 279-86.

[18] Friedman, D.B.; Johnson, T.E. A mutation in the age-1 gene in *Caenorhabditis elegans* lengthens life and reduces hermaphrodite fertility. *Genetics*, 1988, 118(1), 75-86.

[19] Kenyon, C.; Chang, J.; Gensch, E.; Rudner, A.; Tabtiang, R. A *C. elegans* mutant that lives twice as long as wild type. *Nature*, 1993, 366(6454), 461-4.

[20] Larsen, P.L.; Albert, P.S.; Riddle, D.L. Genes that regulate both development and longevity in *Caenorhabditis elegans*. *Genetics*, 1995, 139(4), 1567-83.

[21] Dorman, J.B.; Albinder, B.; Shroyer, T.; Kenyon, C. The age-1 and daf-2 genes function in a common pathway to control the lifespan of *Caenorhabditis elegans*. *Genetics*, 1995, 141(4), 1399-406.

[22] Kimura, K.D.; Tissenbaum, H.A.; Liu, Y.; Ruvkun, G. *daf-2*, an insulin receptor-like gene that regulates longevity and diapause in *Caenorhabditis elegans*. *Science*, 1997, 277(5328), 942-6.

[23] Ogg, S.; Paradis, S.; Gottlieb, S.; Patterson, G.I.; Lee, L.; Tissenbaum, H.A.; Ruvkun, G. The Fork head transcription factor DAF-16 transduces insulin-like metabolic and longevity signals in *C. elegans*. Nature, 1997, 389(6654), 994-9.

[24] Lin, K.; Dorman, J.B.; Rodan, A.; Kenyon, C. daf-16: An HNF-3/forkhead family member that can function to double the life-span of *Caenorhabditis elegans*. Science, 1997, 278(5341), 1319-22.

[25] Morris, J.Z.; Tissenbaum, H.A.; Ruvkun, G. A phosphatidylinositol-3-OH kinase family member regulating longevity and diapause in *Caenorhabditis elegans*. Nature, 1996, 382(6591), 536-9.

[26] Hsin, H.; Kenyon, C. Signals from the reproductive system regulate the lifespan of *C. elegans*. Nature, 1999, 399(6734), 362-6.

[27] Arantes-Oliveira, N.; Apfeld, J.; Dillin, A.; Kenyon, C. Regulation of life-span by germ-line stem cells in *Caenorhabditis elegans*. Science, 2002, 295(5554), 502-5.

[28] Lakowski, B.; Hekimi, S. Determination of life-span in *Caenorhabditis elegans* by four clock genes. Science, 1996, 272(5264), 1010-3.

[29] Ewbank, J.J.; Barnes, T.M.; Lakowski, B.; Lussier, M.; Bussey, H.; Hekimi, S. Structural and functional conservation of the *Caenorhabditis elegans* timing gene *clk-1*. Science, 1997, 275(5302), 980-3.

[30] Jonassen, T.; Larsen, P.L.; Clarke, C.F. A dietary source of coenzyme Q is essential for growth of long-lived *Caenorhabditis elegans clk-1* mutants. Proc. Natl. Acad. Sci. U. S. A., 2001, 98(2), 421-6.

[31] Lakowski, B.; Hekimi, S. The genetics of caloric restriction in *Caenorhabditis elegans*. Proc. Natl. Acad. Sci. U. S. A., 1998, 95(22), 13091-6.

[32] Leiser, S.F.; Begun, A.; Kaeberlein, M. HIF-1 modulates longevity and healthspan in a temperature-dependent manner. Aging cell, 2011, 10(2), 318-26.

[33] Mehta, R.; Steinkraus, K.A.; Sutphin, G.L.; Ramos, F.J.; Shamieh, L.S.; Huh, A.; Davis, C.; Chandler-Brown, D.; Kaeberlein, M. Proteasomal regulation of the hypoxic response modulates aging in *C. elegans*. Science, 2009, 324(5931), 1196-8.

[34] Muller, R.U.; Fabretti, F.; Zank, S.; Burst, V.; Benzing, T.; Schermer, B. The von Hippel Lindau tumor suppressor limits longevity. J. Am. Soc. Nephrol., 2009, 20(12), 2513-7.

[35] Zhang, Y.; Shao, Z.; Zhai, Z.; Shen, C.; Powell-Coffman, J.A. The HIF-1 hypoxia-inducible factor modulates lifespan in *C. elegans*. PLoS One, 2009, 4(7), e6348.

[36] Vellai, T.; Takacs-Vellai, K.; Zhang, Y.; Kovacs, A.L.; Orosz, L.; Muller, F. Genetics: influence of TOR kinase on lifespan in *C. elegans*. Nature, 2003, 426(6967), 620.

[37] Hansen, M.; Chandra, A.; Mitic, L.L.; Onken, B.; Driscoll, M.; Kenyon, C. A Role for Autophagy in the Extension of Lifespan by Dietary Restriction in *C. elegans*. PLoS Genet., 2008, 4(2), e24.

[38] Chen, D.; Thomas, E.L.; Kapahi, P. HIF-1 modulates dietary restriction-mediated lifespan extension via IRE-1 in *Caenorhabditis elegans*. PLoS Genet., 2009, 5(5), e1000486.

[39] Jia, K.; Chen, D.; Riddle, D.L. The TOR pathway interacts with the insulin signaling pathway to regulate *C. elegans* larval development, metabolism and life span. Development, 2004, 131(16), 3897-906.

[40] Lee, S.J.; Hwang, A.B.; Kenyon, C. Inhibition of respiration extends *C. elegans* life span via reactive oxygen species that increase HIF-1 activity. Curr. Biol., 2010, 20(23), 2131-6.

[41] Sutphin, G.L.; Kaeberlein, M. Measuring *Caenorhabditis elegans* life span on solid media. J. Vis. Exp., 2009(27).

[42] Garigan, D.; Hsu, A.L.; Fraser, A.G.; Kamath, R.S.; Ahringer, J.; Kenyon, C. Genetic analysis of tissue aging in *Caenorhabditis elegans*: a role for heat-shock factor and bacterial proliferation. Genetics, 2002, 161(3), 1101-12.

[43] Garsin, D.A.; Villanueva, J.M.; Begun, J.; Kim, D.H.; Sifri, C.D.; Calderwood, S.B.; Ruvkun, G.; Ausubel, F.M. Long-lived *C. elegans daf-2* mutants are resistant to bacterial pathogens. Science, 2003, 300(5627), 1921.

[44] Evans, E.A.; Chen, W.C.; Tan, M.W. The DAF-2 insulin-like signaling pathway independently regulates aging and immunity in *C. elegans*. Aging cell, 2008, 7(6), 879-93.

[45] Loeb, J.; Northrop, J.H. On the influence of food and temperature upon the duration of life. J. Biol. Chem., 1917, 32(1), 103-121.

[46] Klass, M.R. Aging in the nematode *Caenorhabditis elegans*: major biological and environmental factors influencing life span. Mech. Ageing Dev., 1977, 6(6), 413-29.

[47] Leiser, S.F.; Kaeberlein, M. The hypoxia-inducible factor HIF-1 functions as both a positive and negative modulator of aging. Biol. Chem., 2010, 391(10), 1131-7.

[48] Lee, S.J.; Hwang, A.B.; Kenyon, C. Inhibition of respiration extends *C. elegans* life span via reactive oxygen species that increase HIF-1 activity. Curr. Biol., 2010, 20(23), 2131-6.

[49] Kaeberlein, M.; Kapahi, P. The hypoxic response and aging. Cell Cycle, 2009, 8(15), 2324.

[50] Gandhi, S.; Santelli, J.; Mitchell, D.H.; Stiles, J.W.; Sanadi, D.R. A simple method for maintaining large, aging populations of *Caenorhabditis elegans*. Mech. Ageing Dev., 1980, 12(2), 137-50.

[51] Hosono, R.; Mitsui, Y.; Sato, Y.; Aizawa, S.; Miwa, J. Life span of the wild and mutant nematode *Caenorhabditis elegans*. Effects of sex, sterilization, and temperature. Exp. gerontol., 1982, 17(2), 163-72.

[52] Aitlhadj, L.; Sturzenbaum, S.R. The use of FUdR can cause prolonged longevity in mutant nematodes. Mech. Ageing Dev., 2010, 131(5), 364-5.

[53] Van Raamsdonk, J.M.; Hekimi, S. FUdR causes a twofold increase in the lifespan of the mitochondrial mutant *gas-1*. Mech. Ageing Dev., 2011, 132(10), 519-21.

[54] Greer, E.L.; Maures, T.J.; Hauswirth, A.G.; Green, E.M.; Leeman, D.S.; Maro, G.S.; Han, S.; Banko, M.R.; Gozani, O.; Brunet, A. Members of the H3K4 trimethylation complex regulate lifespan in a germline-dependent manner in *C. elegans*. Nature, 2010, 466(7304), 383-7.

[55] Masoro, E.J. Overview of caloric restriction and ageing. Mech. Ageing Dev., 2005, 126(9), 913-22.

[56] Anderson, R.M.; Weindruch, R. Metabolic reprogramming, caloric restriction and aging. Trends Endocrinol. Metabol., 2010, 21(3), 134-41.

[57] Kennedy, B.K.; Steffen, K.K.; Kaeberlein, M. Ruminations on dietary restriction and aging. Cell. Mol. Life Sci., 2007, 64(11), 1323-8.

[58] Houthoofd, K.; Braeckman, B.P.; Johnson, T.E.; Vanfleteren, J.R. Life extension via dietary restriction is independent of the Ins/IGF-1 signalling pathway in *Caenorhabditis elegans*. Exp Gerontol, 2003, 38(9), 947-54.

[59] Panowski, S.H.; Wolff, S.; Aguilaniu, H.; Durieux, J.; Dillin, A. PHA-4/Foxa mediates diet-restriction-induced longevity of *C. elegans*. Nature, 2007, 447(7144), 550-5.

[60] Bishop, N.A.; Guarente, L. Two neurons mediate diet-restriction-induced longevity in *C. elegans*. Nature, 2007, 447(7144), 545-9.

[61] Houthoofd, K.; Braeckman, B.P.; Lenaerts, I.; Brys, K.; De Vreeze, A.; Van Eygen, S.; Vanfleteren, J.R. Axenic growth up-regulates mass-specific metabolic rate, stress resistance, and extends lifespan in *Caenorhabditis elegans*. Exp. Gerontol., 2002, 37(12), 1371-8.

[62] Greer, E.L.; Dowlatshahi, D.; Banko, M.R.; Villen, J.; Hoang, K.; Blanchard, D.; Gygi, S.P.; Brunet, A. An AMPK-FOXO pathway mediates longevity induced by a novel method of dietary restriction in *C. elegans*. Curr. Biol., 2007, 17(19), 1646-56.

[63] Mair, W.; Panowski, S.H.; Shaw, R.J.; Dillin, A. Optimizing dietary restriction for genetic epistasis analysis and gene discovery in *C. elegans*. PLoS One, 2009, 4(2), e4535.

[64] Greer, E.L.; Brunet, A. Different dietary restriction regimens extend lifespan by both independent and overlapping genetic pathways in *C. elegans*. Aging Cell, 2009, 8(2), 113-27.

[65] Apfeld, J.; Kenyon, C. Regulation of lifespan by sensory perception in *Caenorhabditis elegans*. Nature, 1999, 402(6763), 804-9.

[66] Alcedo, J.; Kenyon, C. Regulation of *C. elegans* longevity by specific gustatory and olfactory neurons. Neuron, 2004, 41(1), 45-55.

[67] Smith, E.D.; Kaeberlein, T.L.; Lydum, B.T.; Sager, J.; Welton, K.L.; Kennedy, B.K.; Kaeberlein, M. Age- and calorie-independent lifespan extension from dietary restriction by bacterial deprivation in *Caenorhabditis elegans*. BMC Dev. Biol., 2008, 8(1), 49.

[68] Stanfel, M.N.; Shamieh, L.S.; Kaeberlein, M.; Kennedy, B.K. The TOR pathway comes of age. Biochim. Biophys. Acta, 2009, 1790(10), 1067-74.

[69] Kapahi, P.; Chen, D.; Rogers, A.N.; Katwa, S.D.; Li, P.W.; Thomas, E.L.; Kockel, L. With TOR, less is more: a key role for the conserved nutrient-sensing TOR pathway in aging. Cell Metab., 2010, 11(6), 453-65.

[70] Heitman, J.; Movva, N.R.; Hall, M.N. Targets for cell cycle arrest by the immunosuppressant rapamycin in yeast. *Science*, 1991, 253(5022), 905-9.

[71] Swiech, L.; Perycz, M.; Malik, A.; Jaworski, J. Role of mTOR in physiology and pathology of the nervous system. *Biochim. Biophys. Acta*, 2008, 1784(1), 116-32.

[72] Meijer, A.J.; Codogno, P. Nutrient sensing: TOR's Ragtime. *Nat. Cell. Biol.*, 2008, 10(8), 881-3.

[73] Vellai, T. Autophagy genes and ageing. *Cell Death Differ.*, 2009, 16(1), 94-102.

[74] Kaeberlein, M.; Kapahi, P. Cell signaling. Aging is RSKy business. *Science*, 2009, 326(5949), 55-6.

[75] Hansen, M.; Chandra, A.; Mitic, L.L.; Onken, B.; Driscoll, M.; Kenyon, C. A role for autophagy genes in the extension of lifespan by dietary restriction in *C. elegans*. *PLoS Genet.*, 2008.

[76] Hansen, M.; Taubert, S.; Crawford, D.; Libina, N.; Lee, S.J.; Kenyon, C. Lifespan extension by conditions that inhibit translation in *Caenorhabditis elegans*. *Aging Cell*, 2007, 6(1), 95-110.

[77] Fire, A.; Xu, S.; Montgomery, M.K.; Kostas, S.A.; Driver, S.E.; Mello, C.C. Potent and specific genetic interference by double-stranded RNA in *Caenorhabditis elegans*. *Nature*, 1998, 391(6669), 806-11.

[78] Tabara, H.; Grishok, A.; Mello, C.C. RNAi in *C. elegans*: soaking in the genome sequence. *Science*, 1998, 282(5388), 430-1.

[79] Timmons, L.; Fire, A. Specific interference by ingested dsRNA. *Nature*, 1998, 395(6705), 854.

[80] Kamath, R.S.; Martinez-Campos, M.; Zipperlen, P.; Fraser, A.G.; Ahringer, J. Effectiveness of specific RNA-mediated interference through ingested double-stranded RNA in *Caenorhabditis elegans*. *Genome Biol.*, 2001, 2(1), research0002-research0002.10.

[81] Fraser, A.G.; Kamath, R.S.; Zipperlen, P.; Martinez-Campos, M.; Sohrmann, M.; Ahringer, J. Functional genomic analysis of *C. elegans* chromosome I by systematic RNA interference. *Nature*, 2000, 408(6810), 325-30.

[82] Kamath, R.S.; Fraser, A.G.; Dong, Y.; Poulin, G.; Durbin, R.; Gotta, M.; Kanapin, A.; Le Bot, N.; Moreno, S.; Sohrmann, M.; Welchman, D.P.; Zipperlen, P.; Ahringer, J. Systematic functional analysis of the *Caenorhabditis elegans* genome using RNAi. *Nature*, 2003, 421(6920), 231-7.

[83] Rual, J.F.; Ceron, J.; Koreth, J.; Hao, T.; Nicot, A.S.; Hirozane-Kishikawa, T.; Vandenhaute, J.; Orkin, S.H.; Hill, D.E.; van den Heuvel, S.; Vidal, M. Toward improving *Caenorhabditis elegans* phenome mapping with an ORFeome-based RNAi library. *Genome Res.*, 2004, 14(10B), 2162-8.

[84] Harris, T.W.; Antoshechkin, I.; Bieri, T.; Blasiar, D.; Chan, J.; Chen, W.J.; De La Cruz, N.; Davis, P.; Duesbury, M.; Fang, R.; Fernandes, J.; Han, M.; Kishore, R.; Lee, R.; Muller, H.M.; Nakamura, C.; Ozersky, P.; Petcherski, A.; Rangarajan, A.; Rogers, A.; Schindelman, G.; Schwarz, E.M.; Tuli, M.A.; Van Auken, K.; Wang, D.; Wang, X.; Williams, G.; Yook, K.; Durbin, R.; Stein, L.D.; Spieth, J.; Sternberg, P.W. WormBase: a comprehensive resource for nematode research. *Nucleic Acids Res.*, 2010, 38(Database issue), D463-7.

[85] Dillin, A.; Hsu, A.L.; Arantes-Oliveira, N.; Lehrer-Graiwer, J.; Hsin, H.; Fraser, A.G.; Kamath, R.S.; Ahringer, J.; Kenyon, C. Rates of behavior and aging specified by mitochondrial function during development. *Science*, 2002, 298(5602), 2398-401.

[86] Lee, S.S.; Lee, R.Y.; Fraser, A.G.; Kamath, R.S.; Ahringer, J.; Ruvkun, G. A systematic RNAi screen identifies a critical role for mitochondria in *C. elegans* longevity. *Nat. Genet.*, 2003, 33(1), 40-8.

[87] Hansen, M.; Hsu, A.L.; Dillin, A.; Kenyon, C. New genes tied to endocrine, metabolic, and dietary regulation of lifespan from a *Caenorhabditis elegans* genomic RNAi screen. *PLoS Genet.*, 2005, 1(1), 119-28.

[88] Hamilton, B.; Dong, Y.; Shindo, M.; Liu, W.; Odell, I.; Ruvkun, G.; Lee, S.S. A systematic RNAi screen for longevity genes in *C. elegans*. *Genes Dev.*, 2005, 19(13), 1544-55.

[89] Yoneda, T.; Benedetti, C.; Urano, F.; Clark, S.G.; Harding, H.P.; Ron, D. Compartment-specific perturbation of protein handling activates genes encoding mitochondrial chaperones. *J. Cell Sci.*, 2004, 117(Pt 18), 4055-66.

[90] Imai, S.; Armstrong, C.M.; Kaeberlein, M.; Guarente, L. Transcriptional silencing and longevity protein Sir2 is an NAD-dependent histone deacetylase. *Nature*, 2000, 403(6771), 795-800.

[91] Kaeberlein, M.; McVey, M.; Guarente, L. The SIR2/3/4 complex and SIR2 alone promote longevity in *Saccharomyces cerevisiae* by two different mechanisms. *Genes Dev.*, 1999, 13(19), 2570-80.

[92] Kaeberlein, M.; Kirkland, K.T.; Fields, S.; Kennedy, B.K. Sir2-independent life span extension by calorie restriction in yeast. *PLoS Biol.*, 2004, 2(9), E296.

[93] Tissenbaum, H.A.; Guarente, L. Increased dosage of a *sir-2* gene extends lifespan in *Caenorhabditis elegans*. *Nature*, 2001, 410(6825), 227-30.

[94] Burnett, C.; Valentini, S.; Cabreiro, F.; Goss, M.; Somogyvari, M.; Piper, M.D.; Hoddinott, M.; Sutphin, G.L.; Leko, V.; McElwee, J.J.; Vazquez-Manrique, R.P.; Orfila, A.M.; Ackerman, D.; Au, C.; Vinti, G.; Riesen, M.; Howard, K.; Neri, C.; Bedalov, A.; Kaeberlein, M.; Soti, C.; Partridge, L.; Gems, D. Absence of effects of Sir2 overexpression on lifespan in *C. elegans* and Drosophila. *Nature*, 2011, 477(7365), 482-5.

[95] Rea, S.L.; Ventura, N.; Johnson, T.E. Relationship between mitochondrial electron transport chain dysfunction, development, and life extension in *Caenorhabditis elegans*. *PLoS Biol.*, 2007, 5(10), e259.

[96] Durieux, J.; Wolff, S.; Dillin, A. The cell-non-autonomous nature of electron transport chain-mediated longevity. *Cell*, 2011, 144(1), 79-91.

[97] Smith, E.D.; Kennedy, B.K.; Kaeberlein, M. Genome-wide identification of conserved longevity genes in yeast and worms. *Mech. Ageing Dev.*, 2007, 128(1), 106-11.

[98] Chen, D.; Pan, K.Z.; Palter, J.E.; Kapahi, P. Longevity determined by developmental arrest genes in *Caenorhabditis elegans*. *Aging Cell*, 2007, 6(4), 525-33.

[99] Curran, S.P.; Ruvkun, G. Lifespan regulation by evolutionarily conserved genes essential for viability. *PLoS Genet.*, 2007, 3(4), e56.

[100] Mehta, R.; Chandler-Brown, D.; Ramos, F.J.; Shamieh, L.S.; Kaeberlein, M. Regulation of mRNA translation as a conserved mechanism of longevity control. *Adv. Exp. Med. Biol.*, 2010, 694, 14-29.

[101] Pan, K.Z.; Palter, J.E.; Rogers, A.N.; Olsen, A.; Chen, D.; Lithgow, G.J.; Kapahi, P. Inhibition of mRNA translation extends lifespan in *Caenorhabditis elegans*. *Aging Cell*, 2007, 6(1), 111-9.

[102] Syntichaki, P.; Troulinaki, K.; Tavernarakis, N. eIF4E function in somatic cells modulates ageing in *Caenorhabditis elegans*. *Nature*, 2007, 445(7130), 922-6.

[103] Kim, Y.; Sun, H. Functional genomic approach to identify novel genes involved in the regulation of oxidative stress resistance and animal lifespan. *Aging Cell*, 2007, 6(4), 489-503.

[104] Morley, J.F.; Brignull, H.R.; Weyers, J.J.; Morimoto, R.I. The threshold for polyglutamine-expansion protein aggregation and cellular toxicity is dynamic and influenced by aging in *Caenorhabditis elegans*. *Proc. Natl. Acad. Sci. U. S. A.*, 2002, 99(16), 10417-22.

[105] Steinkraus, K.A.; Smith, E.D.; Davis, C.; Carr, D.; Pendergrass, W.R.; Sutphin, G.L.; Kennedy, B.K.; Kaeberlein, M. Dietary restriction suppresses proteotoxicity and enhances longevity by an hsf-1-dependent mechanism in *Caenorhabditis elegans*. *Aging Cell*, 2008, 7(3), 394-404.

[106] Smith, E.D.; Tsuchiya, M.; Fox, L.A.; Dang, N.; Hu, D.; Kerr, E.O.; Johnston, E.D.; Tchao, B.N.; Pak, D.N.; Welton, K.L.; Promislow, D.E.; Thomas, J.H.; Kaeberlein, M.; Kennedy, B.K. Quantitative evidence for conserved longevity pathways between divergent eukaryotic species. *Genome Res.*, 2008, 18(4), 564-70.
