
# Structural and Functional Characterization of CalS11, a TDP-Rhamnose 3′-O-Methyltransferase Involved in Calicheamicin Biosynthesis

Shanteri Singh,$^{†}$ Aram Chang,$^{‡}$ Kate E. Helmich,$^{‡}$ Craig A. Bingman,$^{§}$ Russell L. Wrobel,$^{§}$ Emily T. Beebe,$^{‡}$ Shin-ichi Makino,$^{‡}$ David J. Aceti,$^{§}$ Kevin Dyer,$^{§}$ Greg L. Hura,$^{§}$ Manjula Sunkara,$^{\|}$ Andrew J. Morris,$^{\|}$ George N. Phillips, Jr.,$^{*,\dagger,\ddagger}$ and Jon S. Thorson$^{*,\dagger}$

$^{†}$Center for Pharmaceutical Research and Innovation, University of Kentucky College of Pharmacy, Lexington, Kentucky 40536, United States  
$^{‡}$Department of Biochemistry, University of Wisconsin-Madison, Madison, Wisconsin 53706, United States  
$^{§}$Physical Bioscience Division, Lawrence Berkeley National Laboratory, Berkeley California 94720, United States  
$\|$Division of Cardiovascular Medicine, University of Kentucky, Lexington, Kentucky 40536, United States  
$\bot$Department of Biochemistry and Cell Biology, Rice University, Houston, Texas 77005, United States  

**Supporting Information**

---

### ABSTRACT: Sugar methyltransferases (MTs) are an important class of tailoring enzymes that catalyze the transfer of a methyl group from S-adenosyl-L-methionine to sugar-based N-, C- and O-nucleophiles. While sugar N- and C-MTs involved in natural product biosynthesis have been found to act on sugar nucleotide substrates prior to a subsequent glycosyltransferase reaction, corresponding sugar O-methylation reactions studied thus far occur after the glycosyltransfer reaction. Herein we report the first *in vitro* characterization using $^{1}\mathrm{H}-{ }^{13} \mathrm{C}$-gHSQC with isotopically labeled substrates and the X-ray structure determination at 1.55 Å resolution of the TDP-3′-O-rhamnose-methyltransferase CalS11 from *Micromonospora echinospora*. This study highlights a unique NMR-based methyltransferase assay, implicates CalS11 to be a metal- and general acid/base-dependent O-methyltransferase, and as a first crystal structure for a TDP-hexose-O-methyltransferase, presents a new template for mechanistic studies and/or engineering.

---

The alkylation of sugars attached to natural products is a common late stage diversification tactic employed in nature.$^{1}$ While such sugar C- or N-methylation typically occurs at the sugar nucleotide stage prior to the culminating glycosyltransferase-catalyzed attachment of the sugar to a given natural product aglycon, comparator O-methyltransferases (O-MTs) characterized *in vitro* thus far operate post-glycosylation (*i.e.*, after the unique functionalized sugar has been transferred to a natural product core aglycon). As examples of O-methylated sugar-containing natural products, methylated variants of L-rhamnose are found among a range of secondary metabolites including calicheamicin (Figure 1, 1, containing 3-O-methoxy-L-Rha),$^{2}$ steffimycin (Figure 1, 2, containing 2-O-methoxy-L-Rha),$^{3,4}$ and spinosad and elloramycin (Figure 1, 3 and 4 respectively, both containing 2,3,4-tri-O-methoxy-L-Rha).$^{5,6}$ Our recent work on the reversibility of the glycosyltransferases involved in calicheamicin biosynthesis$^{7,8}$ revealed the rhamnosyltransferase CalG1 to accept TDP-3-O-methoxy-rhamnose as a substrate, suggesting the putative rhamnosyl-O-methyltransferase encoded by the calicheamicin locus (CalS11)$^{2}$ to function at the sugar nucleotide level. This contrasts recent work on the three spinosad-associated

rhamnose O-MTs (SpnH, SpnI, and SpnK), which revealed these catalysts to operate in the more typical postglycosylation style.$^{9}$

To further explore the potential biosynthetic divergence of CalS11 from the sugar O-MT postglycosylation paradigm, herein we report the *in vitro* characterization and X-ray structure determination of CalS11. This work supports the role of CalS11 as a rhamnose O-MT and specifically, as a TDP-3′-O-rhamnose-MT, the first confirmed sugar O-MT to function at the sugar nucleotide level. This work also reveals CalS11 to crystallize as a decamer, and consistent with this observation, the location and nature of the CalS11 decamer interface is unique in comparison to other MTs. The crystal structure of CalS11 in complex with S-adenosyl-L-homocysteine highlights a fairly typical MT monomeric structural fold wherein the putative CalS11 catalytic residues are highly conserved among sequence-related TDP-hexose and TylF family O-MTs. As the first biochemical and structural

Received: January 27, 2013  
Accepted: May 10, 2013  
Published: May 10, 2013

dx.doi.org/10.1021/cb400068k | ACS Chem. Biol. 2013, 8, 1632–1639

ACS Chemical Biology

![Structure 1](image1.png)
1

![Structure 2](image2.png)
2

![Structure 3](image3.png)
3

![Structure 4](image4.png)
4

Figure 1. Rhamnosylated natural products with the rhamnose moiety highlighted in blue. Structures 1, 2, 3, and 4 refer to calicheamicin, steffimycin, spinosad A, and elloramycin, respectively. The CalS11-catalyzed methylation is highlighted in yellow.

![Biosynthetic Pathway](image5.png)

Figure 2. Biosynthetic pathway of TDP-methoxy-rhamnose in *M. echinospora* en route to calicheamicin production.

![Overlay of ^13C-^1H HSQC Spectrum](image6.png)

Figure 3. Overlay of ^13C-^1H HSQC spectrum before and after CalS11 reaction. Colors black and burgundy represent TDP-L-[U-^13C]rhamnose and the corresponding ^13C-methylated product, respectively. Blue circles denote peaks from ion pairing agent tetrabutylammonium bisulfate.

characterization for a TDP-sugar O-MT, this work may present a blueprint to enable the future engineering of sugar nucleotide-dependent MTs for combinatorial applications.

### RESULTS AND DISCUSSION

**In Vitro Characterization.** TDP-3-methoxy-rhamnose biosynthesis en route to calicheamicin parallels TDP-rhamnose (TDP-Rha) biosynthesis in many other bacteria^{3–6} and requires five enzymes: TDP-4,6-dehydratase, CalS3; TDP-3,4-

CalS11 reaction and provide definitive characterization of the reaction products. This study revealed the first definitive evidence that CalS11 acts at the sugar nucleotide level. Of specific importance in this analysis is the emerging methoxy $^{1}$H and $^{13}$C signature at 3.43 and 58.64 ppm, respectively (Table 1,

(~60% at 1 h and nearly quantitative after 12 h), and a parallel reaction containing 10 mM EDTA (no product at 1 or 12 h) clearly supports the participation of divalent metal in catalysis and is consistent with an observed metal-binding ligand within the determined CalS11 structure.

**Overall Structure.** The crystal structure of CalS11 was determined at 1.55 and 2.2 Å resolution, respectively, in two different crystal forms. The 1.55 Å crystal structure (PDB 3TOS) has one decamer in the asymmetric unit, while the 2.2 Å structure (PDB 4GF5) contained two. Both belonged to space group P1 and contained one molecule of S-adenosyl homocysteine (SAH) per monomer. In addition, 3TOS contained one molecule of glutamic acid and one K⁺ bound per subunit. Of the 257 possible residues in CalS11, only the first 7 residues were not modeled because of insufficient electron density. The final structures were refined to an $R_{\text{cryst}}$ and $R_{\text{free}}$ of 14.6% and 17.3% (PDB 3TOS) and 15.9% and 21.3% (PDB 4GF5), respectively (Table 2). Unless otherwise specified, the discussion herein refers to the 1.55 Å structure (PDB 3TOS).

Each CalS11 monomer folds into a single globular domain responsible for binding SAM and the substrate. The domain exhibits a common tertiary structure consisting of a core $\alpha/\beta$ Rossmann fold characteristic of all other SAM-dependent MTs²⁹ and an $\alpha$-helical N-terminal domain containing two $\alpha$-helices, $\alpha 1$ and $\alpha 2$, that forms the top of the active site cavity (colored blue in Figure 4). In the overall subunit structure, a

Figure 3). Supplementary Figure S2 highlights a subsequent comparison of three reactions (with commercially available SAM as the donor), also monitored via NMR, to address the metal dependency of CalS11. Specifically, a comparison of product formation of a standard reaction containing no additives (no detectable product at 1 h and ~30% conversion after 12 h), a comparator reaction containing 5 mM MgCl₂

---

**Table 1. ${}^{1}$H and ${}^{13}$C Chemical Shift Values of TDP-L-[U-${}^{13}$C]Rhamnose and the Corresponding ${}^{13}$C-Methylated Product**

| atom       | dTDP-rhamnose ${}^{1}$H, ${}^{13}$C (ppm) | dTDP-3-methoxy-rhamnose ${}^{1}$H, ${}^{13}$C (ppm) |
|------------|-------------------------------------------|----------------------------------------------------|
| H1', C1'   | 5.20, 98.07                               | 5.20, 98.27                                        |
| H2', C2'   | 4.07, 73.62                               | 4.35, 69.40                                        |
| H3', C3'   | 3.62, 74.46                               | 3.34, 83.91                                        |
| H3", C3"   |                                           | 3.43, 58.64                                        |
| H4', C4'   | 3.35, 74.24                               | 3.40, 73.33                                        |
| H5', C5'   | 3.43, 75.39                               | 3.44, 75.24                                        |
| H6', C6'   | 1.30, 19.36                               | 1.31, 19.47                                        |

---

**Table 2. Summary of Crystal Parameters, Data Collection, and Refinement Statistics**

|                                   | CalS11 (4GF5)                                                                 | CalS11 (3TOS)                                                                 |
|-----------------------------------|--------------------------------------------------------------------------------|--------------------------------------------------------------------------------|
| **crystal parameters**            |                                                                                |                                                                                |
| space group                       | P1                                                                           | P1                                                                           |
| unit-cell parameters (Å)          | $a = 75.11, b = 106.24, c = 184.60$<br>$\alpha = 80.88, \beta = 81.65, \gamma = 70.02$ | $a = 78.32, b = 106.08, c = 106.33$<br>$\alpha = 68.69, \beta = 69.63, \gamma = 88.56$ |
| **data collection statistics**    |                                                                                |                                                                                |
| wavelength (Å)                    | 0.97872                                                                      | 0.9794                                                                       |
| resolution range (Å)              | 50.00–2.20 (2.24–2.20)                                                       | 50.00–1.55 (1.58–1.55)                                                       |
| no. of reflections (measured/unique) | 254196/254196                                                                | 417415/417415                                                                |
| completeness (%)                  | 94.8 (62.9)                                                                  | 96.8 (94.6)                                                                  |
| $R_{\text{merge}}^b$             | 0.110 (0.484)                                                                | 0.079 (0.297)                                                                |
| redundancy                        | 3.9 (3.7)                                                                    | 3.9 (3.8)                                                                    |
| mean $I/\sigma(I)$               | 6.30 (2.36)                                                                  | 10.00 (4.26)                                                                 |
| **refinement and model statistics** |                                                                                |                                                                                |
| resolution range (Å)              | 49.32–2.20                                                                   | 43.10–1.55                                                                   |
| no. of reflections (work/test)    | 254162/1899                                                                  | 412070/1875                                                                  |
| $R_{\text{cryst}}^c$             | 0.159 (0.196)                                                                | 0.146 (0.166)                                                                |
| $R_{\text{free}}$                | 0.213 (0.254)                                                                | 0.173 (0.251)                                                                |
| RMSD bonds (Å)                   | 0.007                                                                        | 0.019                                                                        |
| RMSD angles (deg)                | 1.186                                                                        | 1.716                                                                        |
| B factor (protein/solvent) (Å²)   | 15.1/16.98                                                                   | 13.2/33.1                                                                    |
| no. of protein atoms             | 40128                                                                        | 21291                                                                        |
| no. of waters                    | 2734                                                                         | 5202                                                                         |
| no. of auxiliary molecules        | 20 SAH, 28 sulfide                                                           | 10 SAH, 10 potassium ion, 10 glutamic acid, 14 1,2-ethanediol |
| Ramachandran plot (%)             |                                                                                |                                                                                |
| favorable region                  | 97.5                                                                         | 97.5                                                                         |
| additional allowed region         | 2.5                                                                          | 2.5                                                                          |
| disallowed region                 | 0.0                                                                          | 0.0                                                                          |
| PDB ID                           | 4GF5                                                                         | 3TOS                                                                         |

*Values in parentheses are for the highest resolution shell. $R_{\text{merge}}^b = \sum_h \sum_i |I_i(h) - \langle I(h) \rangle| / \sum_h \sum_i I_i(h)$, where $I_i(h)$ is the intensity of an individual measurement of the reflection, and $\langle I(h) \rangle$ is the mean intensity of the reflection. $R_{\text{cryst}}^c = \sum_h \|F_{\text{obs}}\| - \|F_{\text{calc}}\| / \sum_h |F_{\text{obs}}|$, where $F_{\text{obs}}$ and $F_{\text{calc}}$ are the observed and calculated structure-factor amplitudes, respectively. $R_{\text{free}}$ was calculated as $R_{\text{cryst}}$ using 5.0% of randomly selected unique reflections that were omitted from the structure refinement.*

commonly serves as a gate for the cofactor entry and product release.

While most natural product MTs exists as dimers, to our knowledge, this is the first decameric MT structure (Figure 5A). In this decameric form, also confirmed in solution by small-angle X-ray scattering (SAXS) (Figure 5B), each monomer interacts with three other monomers via two distinct interfaces. The first interface, S1 (Figure 5C and D), is 2615 Å² involving ~64 residues, accounting for 20% solvent-accessible surface area of each subunit, and involves intrasubunit interaction associated with helices α1–α5 and intercalation of helices α1 and α2 between the neighboring subunits (Figure 4). Some of the residues from the N-terminal α2 of the intercalating subunit form part of the lining of the active site. The second interface, S2 (Figure 5C and D), is 1350 Å² involving ~36 residues, accounting for 10.7% of the solvent-accessible area of the subunit, and includes loop interactions among the outer end of the β-sheet of Rossmann fold²⁹ of one subunit with β6 of other. The putative S1 and S2 interfaces observed here are distinct from O-MT structures described thus far and may serve as a potential distinguishing feature of the NDP-sugar O-MTs.

**Active Site.** The SAM/SAH binding site is located in the C-terminal end of the cleft formed by the β strands, and this position is conserved in all MTs.³¹ SAH is partially exposed to solvent and bound to the enzyme through an extensive hydrogen bond network and van der Waals interactions (Figure 6A). The CalS11-SAH interaction is mainly provided by residues in the loops L1 (between β1 and α4), L2 (between β2 and α5), and L3 (between β3 and helix α6). Specifically,

Figure 4. CalS11 monomer with secondary structural elements labeled. The N-terminal helices are colored blue, and the helices from the core MT Rossmann fold are colored brick-red. The dynamic loop L2 is colored cyan. SAH, bound glutamate and ethylene glycol are represented by stick models and are colored yellow and green, respectively. K⁺ ion is represented as a purple sphere.

central parallel β-sheet (β1–β7) is located between helices (α3 and α8) (Figure 4). The β-sheet has strand topology (β3, β2, β1, β4, β5, β7, β6) containing a single antiparallel strand (β7) flanked on both sides by helices. This is consistent with a class I MT fold,³⁰ which is described as a “doubly wound-open α/β sandwich”. A loop, L2 (colored cyan in Figure 4), between the strand β2 and helix α5, forms a lid on the SAH binding site of the globular domain that, analogous to MT homologues,

Figure 5. (A) Decamer structure of CalS11. (B) CalS11 in solution by SAXS. (C, D) Illustration of subunit–subunit interactions. Six subunits are illustrated with three represented as surface representations (two green and one blue) and three represented as ribbon cartoons (two pink and one red). For the red subunit, the major interaction surface is S1 and the minor is S2.

Figure 6. (A) Residues involved in SAH binding. (B) Residues involved in the interaction of bound glutamate (colored green) and K⁺. SAH, bound glutamate and ethylene glycol are represented as stick models and are colored yellow and green, respectively. Na⁺ ion is represented as a purple sphere, and water molecules are light blue spheres.

Figure 7. Structural overlay of CalS11 with related MTs. (A) Overlay with NovP, where NovP is colored light green and CalS11 is colored light blue. (B) Overlay of active-site residues of NovP and CalS11 with CalS11-bound potassium ion represented by the violet sphere. In panel A, the variable N-terminal secondary structures and the lids in NovP and CalS11 are labeled. In panel B, CalS11 active-site residues are highlighted in blue.

residues in L1 interact with the amino acid moiety and the ribose, L2 associates with the ribose and adenine ring, and L3 affiliates with the adenine ring of SAH (Figure 4). Loop L2 (residues 109–135) (colored cyan in Figures 4 and 6A) also facilitates the cofactor entry and SAH release. Structural comparison and primary sequence analysis of other SAM-methyltransferases reveal L1, which contains a glycine-rich consensus sequence (XGly₇₇XXXGly₈₁), and an acidic residue in loop L3 (Asp166 in CalS11) to be highly conserved.

While attempts to obtain a complex with bound TDP-Rha (in the presence or absence of SAM) by co-crystallization failed, two ligand surrogates, glutamic acid and ethylene glycol, were found to occupy the putative substrate-binding site of CalS11. The bound glutamic acid surrogate forms direct H-bonding interactions with Trp223, Asp216, and Glu217 as well as water-mediated interactions with Asp189 and Asp191 (Figure 6B). In addition, a potassium ion was found coordinated by Asp216, Glu217, glutamic acid, ethylene glycol, and water (Figure 6B). Based on a putative S_N2 mechanism, the position of the glutamic acid in the substrate binding site, and the structural conservation of the active residues in the closest homologue NovP, Asp191 is most suited as the putative CalS11 catalytic base. Interestingly, the CalS11 potassium ion coordinating residues (Asp216 and Glu217) are highly conserved among other O-MTs (Supplementary Figure S1). Assuming the observed potassium ion engages some of the residues involved in CalS11 magnesium ion binding, the metal-active distances (SAH sulfur-metal = 9.6 Å; Asp191 carboxylate-metal = 6.9 Å) are such that protein reorganiza-

tion, perhaps driven via substrate binding, would be required for direct metal participation in catalysis.

Structurally Related Proteins. While this work represents the first reported NDP-6-deoxy-hexose-O-MT structure, a CalS11-based structural similarity search returned more than 200 hits with P-values less than 0.0001 (using the jFATCAT-rigid algorithm). Upon closer analysis, the structurally related proteins display little or no overall sequence identity to each other, but most share a common SAM-dependent MT Rossmann fold. Many members also contain additional domains outside the core MT structure that play a role in substrate recognition or alternative functions. The structural similarity to CalS11 is mostly restricted to the Rossmann fold, and comparators within this set share only 6–19% sequence identities with CalS11. Supplementary Table S1 summarizes a selection of structurally and functionally related CalS11 proteins involved in natural product biosynthesis. The scope of the following discussion is focused upon the closest comparator NovP, the novobiocin L-noviose-4′-O-MT.

Structural alignment of CalS11 with NovP (Figure 7A) highlights conservation of residues in the active site where the putative catalytic base, conserved in both enzymes, is located within loop L3 of CalS11 structure. This putative catalytic base is Asp198 in NovP and, based upon structural correlation, Asp191 in CalS11 (Figure 7B). In addition, loop L2, which forms a lid over the cosubstrate SAM in NovP, is also conserved in CalS11 and likely serves a similar gating function. In NovP, this loop is a flexible helix, while in CalS11 this substructure is a dynamic loop (Figure 7A). In addition, while

both CalS11 and NovP contain a conserved acid-rich motif near the active site (Asp216 and Glu217 in CalS11, Figure 7B), only the CalS11 structure revealed a bound metal.

In recent years, structures of several sugar MTs have been solved. These include MTs that act upon sugars appended to aglycons such as RebM, rebeccamycin D-glucose-4′-O-MT, MycE, and NovP, as well as MTs that act upon sugar nucleotides, including DesVI, TDP-D-desosamine 3′-N,N-di-MT, TylM1, a TDP-D-mycaminose 3′-N,N-di-MT, and TcaB9, a TDP-D-tetronitrose C-3′-MT. Yet, despite the plethora of available sequences and structures, there remains a lack of any clear structural signature that denotes nucleophile specificity (O-, N-, C-MTs), sugar regiospecificity (2-, 3-, or 4-X-MTs where X is O, N, or C), the nature of the aglycon (*i.e.*, MTs that act upon sugars appended to NDP or aglycon), or sugar configuration (*e.g.*, D- and/or L- as well as 2-, 3-, 4- and/or 6-stereochemistry). Thus, while the current CalS11 study stands as the first biochemical and structural characterization of a nucleotide sugar O-MT, more structures are needed to delineate broader general structure–activity MT correlations and/or develop predictive models relevant to this important enzyme family.

### METHODS

#### Materials.
*E. coli* B834(DE3)pLysS and BL21(DE3) competent cells were purchased from Invitrogen (Carlsbad, CA). The pET-28b *E. coli* expression vector and thrombin were purchased from Novagen (Madison, WI). Primers were purchased from Integrated DNA Technology (Coralville, IA). Pfu DNA polymerase was purchased from Stratagene (La Jolla, CA). Restriction enzymes and T4 DNA ligase were purchased from New England Biolabs (Ipswich, MA). Benzamidine sepharose was purchased from Pharmacia (Piscataway, NJ). PD-10 column [S-<sup>13</sup>C-methyl]-L-methionine was purchased from Cambridge Isotope Laboratories, (Andover, MA), and Ni-NTA superflow columns were purchased from GE Healthcare (Piscataway, NJ). All other chemicals were reagent grade or better and purchased from Sigma (St. Louis, MO). NMR experiments were performed in 5 mm NMR tubes and analyzed via a Varian 600 MHz instrument equipped with HCX cryoprobe at UW NMRFAM. NMR data were processed using NMRPipe and MestReNova. <sup>13</sup>C–<sup>1</sup>H HSQC spectra of [S-<sup>13</sup>C-methyl]-L-methionine and [S-<sup>13</sup>C-methyl]adenosyl-L-methionine (<sup>13</sup>CH<sub>3</sub>-SAM) were acquired on a Varian 400 MHz equipped with a 5 mm OneNMR probe in the University of Kentucky College of Pharmacy NMR facility.

#### Protein Expression and Purification.
The *calS11* gene, amplified from the genomic DNA of *Micromonospora echinospora*, was separately cloned into vectors for both cell-based expression in *E. coli* and expression in a wheat germ cell-free system. For *E. coli* expression, the *calS11* gene was cloned into pET28a to provide a vector for the overproduction of N-terminal His<sub>6</sub>-CalS11 in *E. coli* BL21 (DE3). The culture was grown in the presence of 35 μg mL<sup>−1</sup> of kanamycin at 37 °C to an OD<sub>600</sub> of ~0.6, induced with 0.5 mM IPTG, and grown at 25 °C overnight. The cells were collected by centrifugation (30 min, 5000 rpm) and resuspended in buffer A (20 mM NaH<sub>2</sub>PO<sub>4</sub>, 300 mM NaCl, 10 mM imidazole, pH 7.8). The cells were lysed via incubation with 1 mg mL<sup>−1</sup> lysozyme (~50,000 U mg<sup>−1</sup>; Sigma-Aldrich, St. Louis, MO) for 30 min on ice followed by sonication (VirSonic 475; Virtis, Gardiner, NY; 100 W, 4 × 30 s pulses, ~1 min between pulses) on ice. N-His<sub>6</sub>-CalS11 was purified via affinity chromatography (5 mL HiTrap Ni-NTA chelating column, GE Healthcare, Piscataway, NJ) following standard protocol (50 mM sodium phosphate, 300 mM NaCl, pH 8.0 and a linear elution gradient of imidazole of 10–500 mM) on an AKTA Purifier 10 (GE Healthcare) to provide ~60 mg of N-His<sub>6</sub>-CalS11. Buffer exchange of pooled fractions containing the purified protein was accomplished using a PD-10 column (GE Healthcare) eluted with 20 mM Tris-HCl, 100 mM NaCl, 5 mM CaCl<sub>2</sub> pH 8. The N-His<sub>6</sub>-tag was subsequently removed by thrombin cleavage following standard protocol (Novagen, Madison, WI; 50 U of thrombin, 20 h, RT, 60 mg of N-His<sub>6</sub>-CalS11, 3.5 mL total volume). This mixture was subsequently treated with 1 mL of a slurry of benzamidine sepharose (Pharmacia, NJ) to remove thrombin, and the affinity tag cleaved CalS11 was purified via a second round of HiTrap Ni-NTA affinity chromatography. The flow-through containing desired truncated CalS11 was concentrated and exchanged with storage buffer (25 mM Tris-HCl, pH 8) using a PD-10 column. The purified CalS11 was concentrated to 11 mg mL<sup>−1</sup>, flash frozen in liquid nitrogen, and stored at −80 °C. Protein concentrations were determined by Bradford assay (Bio-Rad, Hercules, CA, USA) using BSA as a standard.

Selenomethionyl CalS11 for initial crystallization trials and X-ray crystallographic structures was produced using a standard cell-free production protocol of the Center for Eukaryotic Structural Genomics (CESG) platform for cloning, protein expression, purification, and bioinformatics management. For this study, the *calS11* gene was also amplified from genomic DNA and cloned into the pEU-His-FV vector using modifications of the protocol found in Blommel *et al.* Modifications included a two-step PCR protocol to introduce a N-terminal TEV protease cleavage site and the use of TaKaRa Ex Taq polymerase (Millipore) in the presence of 1% v/v DMSO in all PCR reactions to accommodate the GC-rich template. Cell-free expression was conducted on a 6 × 4 mL scale for 18 h using WEPRO2240H extract and a Protemist 100 protein synthesizer (CellFree Sciences, Yokohama, Japan). Selenomethionine was added to 0.6 mM, and the remaining 19 amino acids were added to 0.3 mM. His-tagged protein was purified by nickel affinity chromatography. The N-terminal His-tag was cleaved with TEV protease, and tag-free protein was isolated by subtractive nickel affinity chromatography. Size-exclusion chromatography provided additional purification and permitted exchange of CalS11 into the final protein buffer (5 mM HEPES, 50 mM NaCl, 0.3 mM TCEP, pH 7.0). CalS11 was concentrated to 16 mg mL<sup>−1</sup>, cryopreserved in 30 μL beads by dropping into liquid nitrogen, and stored at −80 °C. Complete protocols are available from Target Track (http://sbkb.org/tt/) as target CESG-GO.111124. Plasmid pEU-His-FV is available from the PSI Materials Repository (http://dnasu.asu.edu/DNASU/GetCloneDetail.do?cloneid=84283).

#### Protein Crystallization.
Initial crystallization screens were performed using an in house screen composed of UW192, IndexHT, and Salt RX (Hampton Research, Aliso Viejo, CA) by the sitting drop method using a Mosquito dispenser (TTP labTech Hertfordshire, U.K.). Crystal growth was monitored at 4 and 20 °C using Bruker Nonius Crystal Farm (Bruker AXS Inc., Madison, WI). Cell-free synthesized SeMet-CalS11 crystals were grown by mixing 1 μL of protein sample solution (16 mg mL<sup>−1</sup> CalS11), 3 mM tris(2-carboxyethyl)phosphine, 50 mM sodium chloride, 5 mM HEPES pH 8, and 1 μL of reservoir solution, 17.6% poly(ethylene glycol) monomethyl ether 5,000, 160 mM potassium-glutamate, 100 mM 2-[bisamino]-2-1,3-propanediol pH 6.5, at 20 °C using the hanging drop method. Native CalS11 crystals were grown by mixing 1 μL of protein sample solution (11 mg mL<sup>−1</sup> CalS11), 25 mM Tris, pH 8.0, 5 mg mL<sup>−1</sup> calicheamicin T<sub>0</sub> and 1 μL of reservoir solution, 25% poly(ethylene glycol) 3350, 0.2 M lithium sulfate, 100 mM Tris, pH 8.5, at 20 °C using the hanging drop method. Native CalS11 crystals were cryoprotected with 30% poly(ethylene glycol) 3350, 200 mM lithium sulfate, and 100 mM Tris, pH 8.5 and SeMet-CalS11 crystals were cryoprotected with 20% ethylene glycol, 17.6% poly(ethylene glycol) monomethyl ether, 160 mM potassium glutamate, 100 mM 2-[bisamino]-2-1,3-propanediol, pH 6.5 before flash freezing in liquid nitrogen for data collection.

#### Data Collection and Structure Refinement.
X-ray diffraction data were collected at the Life Sciences Collaborative Access Team (LS-CAT) with an X-ray wavelength of 0.9794 Å, for the SeMet-CalS11 crystals, and 0.97872 Å, for the native CalS11 crystals, at the Advanced Photon Source at Argonne National Laboratory. Data sets were indexed and scaled using HKL2000. For phasing experiments on SeMet-CalS11 (3TOS), phenix.Hyss, and ShelxD were utilized for determining the selenium substructures, autoSHARP for phasing, DM for density modification, and phenix.AutoBuild for

automatic model building.¹⁶ For the native CalS11 structure, molecular replacement starting from SeMet-CalS11 (3TOS) was accomplished using phenix.AutoMR and phenix.AutoBuild. The structures were completed with alternating rounds of manual model building with COOT²⁰ and refinement with phenix.refine. Structure quality was assessed by Procheck²¹ and Molprobity.²²

**SAXS Data.** SAXS data were collected on the SIBYLS beamline using the high throughput data collection mode as described previously.²³ Three concentrations of CalS11 were measured at 2.3, 3.5, and 7.0 mg mL⁻¹. No concentration dependence was observed. Each concentration was exposed 4 times to X-rays with 0.5, 1, 2, and 5 s exposures. No radiation damage was observed through comparison of the 0.5 and 1 s exposures. For redundancy, two buffer blanks were collected, one prior to the concentration series and one after. Scattering profiles from both buffer blanks were subtracted from the SAXS data collected from samples, and the resulting subtracted files were compared for agreement. The 2 and 5 s exposures were merged with the 1 s exposure to produce a final scattering curve for further analysis using the program PRIMUS.²⁴ The scattering data was merged to obtain the highest signal-to-noise ratio and processed by GASBOR²⁵ to determine the three-dimensional envelop. On the basis of the symmetry of crystal structure and the MW of the CalS11 decamer, P52 symmetry was imposed. Out of 10 averaged GASBOR runs, eight models were selected to generate the final model based on their orientation with respect to P52 axis using program SUPCOMB.²⁶

**Preparation of [S-¹³C-Methyl]adenosyl-L-methionine (¹³CH₃-SAM).** Isotopically labeled ¹³CH₃-SAM was prepared using 1.1 mM [S-¹³C-methyl]-L-methionine, 1.2 equiv of ATP, and 50 μg of methionine adenosyltransferase (MAT) from *Sulfolobus solfataricus*²⁷ in 20 mM NaH₂PO₄ buffer, 10 mM MgCl₂, 100 mM KCl, pH 8.0 at 37 °C for 1 h. Under these conditions, quantitative conversion to the desired product was observed based upon HPLC (Supplementary Figure S3, top panel) and HSQC (Figure 3). The crude reaction mixture was lyophilized and dissolved in 100 μL D₂O and directly used in the subsequent CalS11 reaction.

**CalS11 Reaction.** In a 5 mm NMR tube, a mixture of 1 mM TDP-L-[U-¹³C]rhamnose, 1.1 mM ¹³CH₃-SAM (Supplementary Figure S4), and 50 μg of the purified CalS11 in a final volume of 500 μL of 20 mM NaH₂PO₄ buffer prepared with D₂O, pH 8, was incubated at 30 °C for 1 h. The progress of the reaction was monitored by sensitivity enhanced ¹H–¹³C-gHSQC (Figure 3). NMR chemical shift assignments were made using 2D-planes of 3D ¹H–¹³C-HCCH-TOCSY and ¹H–¹³C-HCCH-COSY from Varian BioPack. To assess metal dependency, three separate reactions were monitored by ¹H–¹³C-gHSQC (after 1 h and overnight incubation at 30 °C), each containing commercially available SAM (Sigma, St. Louis, MO) and 12 μg of purified CalS11 in a final volume of 250 μL 25 mM KH₂PO₄, pH 8 in a 3 mm NMR tube. The first of these contained no additional additives, the second included 10 mM EDTA, and the latter was conducted in the presence of 5 mM MgCl₂.

---

### ACKNOWLEDGMENTS

This work was supported by the National Institutes of Health grants U01GM098248 (G.N.P.), CA84374 (J.S.T.) and the National Center for Advancing Translational Sciences (UL1TR000117). Use of the Advanced Photon Source was supported by the U.S. Department of Energy, Office of Science, Office of Basic Energy Sciences, under Contract No. DE-AC02-06CH11357. NMR experiments were carried out at the National Magnetic Resonance Facility at Madison (NMRFAM), which is supported by National Institutes of Health (P41RR02301, P41GM66326, P41GM66326, P41RR02301, RR02781, RR08438), National Science Foundation (DMB-8415048, BIR-9214394), U.S. Department of Agriculture and UW-Madison, WI grants. HRMS analyses were supported in part by the Center for Research in Obesity and Cardiovascular Disease (National Institutes of Health P20RR021954) and SAX analysis was supported by the U.S. Department of Energy (IDAT-DE-AC02-05CH11231). The Life Sciences Collaborative Access Team (LS-CAT) has been supported by Michigan Economic Development Corporation and the Michigan Technology Tri-Corridor.

---

### REFERENCES

(1) Singh, S., Phillips, G. N., Jr., and Thorson, J. S. (2012) The structural biology of enzymes involved in natural product glycosylation. *Nat. Prod. Rep.* 29, 1201–1237.

(2) Ahlert, J., Shepard, E., Lomovskaya, N., Zazopoulos, E., Staffa, A., Bachmann, B. O., Huang, K., Fonstein, L., Czisny, A., Whitwam, R. E., Farnet, C. M., and Thorson, J. S. (2002) The calicheamicin gene cluster and its iterative type I enediyne PKS. *Science* 297, 1173–1176.

(3) Gullon, S., Olano, C., Abdelfattah, M. S., Brana, A. F., Rohr, J., Mendez, C., and Salas, J. A. (2006) Isolation, characterization, and heterologous expression of the biosynthesis gene cluster for the antitumor anthracycline steffimycin. *Appl. Environ. Microbiol.* 72, 4172–4183.

(4) Luzhetskyy, A., Mayer, A., Hoffmann, J., Pelzer, S., Holzenkamper, M., Schmitt, B., Wohlert, S. E., Vente, A., and Bechthold, A. (2007) Cloning and heterologous expression of the aranciamycin biosynthetic gene cluster revealed a new flexible glycosyltransferase. *ChemBioChem* 8, 599–602.

(5) Ramos, A., Lombó, F., Brana, A. F., Rohr, J., Mendez, C., and Salas, J. A. (2008) Biosynthesis of elloramycin in *Streptomyces olivaceus* requires glycosylation by enzymes encoded outside the aglycon cluster. *Microbiology* 154, 781–788.

(6) Waldron, C., Matsushima, P., Rosteck, P. R., Jr., Broughton, M. C., Turner, J., Madduri, K., Crawford, K. P., Merlo, D. J., and Baltz, R. H. (2001) Cloning and analysis of the spinosad biosynthetic gene cluster of *Saccharopolyspora spinosa*. *Chem. Biol.* 8, 487–499.

(7) Zhang, C. S., Griffith, B. R., Fu, Q., Albermann, C., Fu, X., Lee, I. K., Li, L. J., and Thorson, J. S. (2006) Exploiting the reversibility of natural product glycosyltransferase-catalyzed reactions. *Science* 313, 1291–1294.

(8) Zhang, C. S., Bitto, E., Goff, R. D., Singh, S., Bingman, C. A., Griffith, B. R., Albermann, C., Phillips, G. N., and Thorson, J. S. (2008) Biochemical and structural insights of the early glycosylation steps in calicheamicin biosynthesis. *Chem. Biol.* 15, 842–853.

(9) Kim, H. J., White-Phillip, J. A., Ogasawara, Y., Shin, N., Isiorho, E. A., and Liu, H. W. (2010) Biosynthesis of spinosyn in *Saccharopolyspora spinosa*: synthesis of permethylated rhamnose and characterization of the functions of SpnH, SpnI, and SpnK. *J. Am. Chem. Soc.* 132, 2901–2903.

(10) Delaglio, F., Grzesiek, S., Vuister, G. W., Zhu, G., Pfeifer, J., and Bax, A. (1995) NMRPipe: a multidimensional spectral processing system based on UNIX pipes. *Biomol. NMR* 6, 277–293.

---

### ASSOCIATED CONTENT

#### Supporting Information

CalS11 sequence alignment and table of structurally related proteins, the key CalS11 metal dependency NMR spectra, and MAT reaction HPLC, HRMS, and NMR characterization data. This material is available free of charge via the Internet at http://pubs.acs.org.

---

### AUTHOR INFORMATION

**Corresponding Author**

*E-mail:* jsthorson@uky.edu; georgep@rice.edu

**Notes**

The authors declare the following competing financial interest(s): The authors report competing interests. JST is a co-founder of Centrose (Madison, WI).

(11) Blommel, P. G., Martin, P. A., Seder, K. D., Wrobel, R. L., and Fox, B. G. (2009) High throughput protein expression and purification: Methods and protocols. *Methods Mol. Biol.* 498, 55–73.

(12) Makino, S., Goren, M. A., Fox, B. G., and Markley, J. L. (2010) Cell-free protein synthesis technology in NMR high-throughput structure determination. *Methods Mol. Biol.* 607, 127–147.

(13) Vinarov, D. A., Newman, C. L., Tyler, E. M., Markley, J. L., and Shahan, M. N. (2006) Wheat germ cell-free expression system for protein production. *Curr. Protoc. Sci.*, 1 DOI: 10.1002/0471140864.ps0518s44.

(14) Zolnai, Z., Lee, P. T., Li, J., Chapman, M. R., Newman, C. S., Phillips, G. N., Jr., Rayment, I., Ulrich, E. L., Volkman, B. F., and Markley, J. L. (2003) Project management system for structural and functional proteomics: Sesame. *J. Struct. Funct. Genomics* 4, 11–23.

(15) Otwinowski, Z., and Minor, W. (1997) Processing of X-ray diffraction data collected in oscillation mode. *Methods Enzymol.* 276, 307–326.

(16) Adams, P. D., Afonine, P. V., Bunkoczi, G., Chen, V. B., Davis, I. W., Echols, N., Headd, J. J., Hung, L. W., Kapral, G. J., Grosse-Kunstleve, R. W., McCoy, A. J., Moriarty, N. W., Oeffner, R., Read, R. J., Richardson, D. C., Richardson, J. S., Terwilliger, T. C., and Zwart, P. H. (2010) PHENIX: a comprehensive python-based system for macromolecular structure solution. *Acta Crystallogr. D* 66, 213–221.

(17) Sheldrick, G. M. (2008) A short history of SHELX. *Acta Crystallogr. A* 64, 112–122.

(18) delaFortelle, E., and Bricogne, G. (1997) Maximum-likelihood heavy-atom parameter refinement for multiple isomorphous replacement and multiwavelength anomalous diffraction methods. *Methods Enzymol.* 276, 472–494.

(19) Cowtan, K. D., and Main, P. (1996) Phase combination and cross validation in iterated density-modification calculations. *Acta Crystallogr. D* 52, 43–48.

(20) Emsley, P., and Cowtan, K. (2004) Coot: model-building tools for molecular graphics. *Acta Crystallogr. D* 60, 2126–2132.

(21) Laskowski, R. A., MacArthur, M. W., Moss, D. S., and Thornton, J. M. (1993) Procheck - a program to check the stereochemical quality of protein structures. *J. Appl. Crystallogr.* 26, 283–291.

(22) Davis, I. W., Leaver-Fay, A., Chen, V. B., Block, J. N., Kapral, G. J., Wang, X., Murray, L. W., Arendall, W. B., Snoeyink, J., Richardson, J. S., and Richardson, D. C. (2007) MolProbity: all-atom contacts and structure validation for proteins and nucleic acids. *Nucleic Acids Res.* 35, W375–W383.

(23) Hura, G. L., Menon, A. L., Hammel, M., Rambo, R. P., Poole, Ii, F. L., Tsutakawa, S. E., Jenney, F. E., Jr, Classen, S., Frankel, K. A., Hopkins, R. C., Yang, S.-j., Scott, J. W., Dillard, B. D., Adams, M. W. W., and Tainer, J. A. (2009) Robust, high-throughput solution structural analyses by small angle X-ray scattering (SAXS). *Nat. Methods* 6, 606–612.

(24) Konarev, P. V., Petoukhov, M. V., Volkov, V. V., and Svergun, D. I. (2006) ATSAS 2.1, a program package for small-angle scattering data analysis. *J. Appl. Crystallogr.* 39, 277–286.

(25) Petoukhov, M. V., and Svergun, D. I. (2003) New methods for domain structure determination of proteins from solution scattering data. *J. Appl. Crystallogr.* 36, 540–544.

(26) Kozin, M. B., and Svergun, D. I. (2001) Automated matching of high- and low-resolution structural models. *J. Appl. Crystallogr.* 34, 33–41.

(27) Porcelli, M., Cacciapuoti, G., Carteni-Farina, M., and Gambacorta, A. (1988) S-adenosylmethionine synthetase in the thermophilic archaebacterium *Sulfolobus solfataricus*: Purification and characterization of two isoforms. *FEBS J.* 177, 273–280.

(28) Patallo, E. P., Blanco, G., Fischer, C., Brana, A. F., Rohr, J., Mendez, C., and Salas, J. A. (2001) Deoxysugar methylation during biosynthesis of the antitumor polyketide elloramycin by *Streptomyces olivaceus*: Characterization of three methyltransferase genes. *J. Biol. Chem.* 276, 18765–18774.

(29) Rossmann, M. G., Moras, D., and Olsen, K. W. (1974) Chemical and biological evolution of nucleotide-binding protein. *Nature* 250, 194–199.

(30) Schubert, H. L., Blumenthal, R. M., and Cheng, X. (2003) Many paths to methyltransfer: a chronicle of convergence. *Trends Biochem. Sci.* 28, 329–335.

(31) Martin, J. L., and McMillan, F. M. (2002) SAM (dependent) I AM: the S-adenosylmethionine-dependent methyltransferase fold. *Curr. Opin. Struct. Biol.* 12, 783–793.

(32) Singh, S., Mccoy, J. G., Zhang, C., Bingman, C. A., Phillips, G. N., and Thorson, J. S. (2008) Structure and mechanism of the rebeccamycin sugar 4′-O-methyltransferase RebM. *J. Biol. Chem.* 283, 22628–22636.

(33) Akey, D. L., Li, S., Konwerski, J. R., Confer, L. A., Bernard, S. M., Anzai, Y., Kato, F., Sherman, D. H., and Smith, J. L. (2011) A new structural form in the SAM/metal-dependent O-methyltransferase family, MycE from the mycinamicin biosynthetic pathway. *J. Mol. Biol.* 413, 438–450.

(34) Gómez García, I., Stevenson, C. E. M., Usón, I., Freel Meyers, C. L., Walsh, C. T., and Lawson, D. M. (2010) The crystal structure of the novobiocin biosynthetic enzyme NovP: the first representative structure for the TylF O-methyltransferase superfamily. *J. Mol. Biol.* 395, 390–407.

(35) Burgie, E. S., and Holden, H. M. (2008) Three-dimensional structure of DesVI from *Streptomyces venezuelae*: a sugar N,N-dimethyltransferase required for dTDP-desosamine biosynthesis. *Biochemistry* 47, 3982–3988.

(36) Carney, A. E., and Holden, H. M. (2010) Molecular architecture of TylM1 from *Streptomyces fradiae*: an N,N-dimethyltransferase involved in the production of dTDP-D-mycaminose. *Biochemistry* 50, 780–787.

(37) Bruender, N. A., Thoden, J. B., Kaur, M., Avey, M. K., and Holden, H. M. (2010) Molecular architecture of a C-3′-methyltransferase involved in the biosynthesis of D-tetronitrose. *Biochemistry* 49, 5891–5898.

(38) Liscombe, D. K., Louie, G. V., and Noel, J. P. (2010) Architectures, mechanisms and molecular evolution of natural product methyltransferases. *Nat. Prod. Rep.* 29, 1238–1250.
