
FGF Receptors: Cancer Biology and Therapeutics

Masaru Katoh and Hitoshi Nakagama

Division of Integrative Omics and Bioinformatics, National Cancer Center, 5-1-1 Tsukiji, Chuo Ward, Tokyo 104-0045, Japan

Published online in Wiley Online Library (wileyonlinelibrary.com).  
DOI 10.1002/med.21288

---

**Abstract:** Fibroblast growth factors (FGFs) are involved in a variety of cellular processes, such as stemness, proliferation, anti-apoptosis, drug resistance, and angiogenesis. Here, FGF signaling network, cancer genetics/genomics of FGF receptors (FGFRs), and FGFR-targeted therapeutics will be reviewed. FGF signaling to RAS-MAPK branch and canonical WNT signaling cascade mutually regulate transcription programming. FGF signaling to PI3K-AKT branch and Hedgehog, Notch, TGFβ, and noncanonical WNT signaling cascades regulate epithelial-to-mesenchymal transition (EMT) and invasion. Gene amplification of *FGFR1* occurs in lung cancer and estrogen receptor (ER)-positive breast cancer, and that of *FGFR2* in diffuse-type gastric cancer and triple-negative breast cancer. Chromosomal translocation of *FGFR1* occurs in the 8p11 myeloproliferative syndrome and alveolar rhabdomyosarcoma, as with *FGFR3* in multiple myeloma and peripheral T-cell lymphoma. *FGFR1* and *FGFR3* genes are fused to neighboring *TACC1* and *TACC3* genes, respectively, due to interstitial deletions in glioblastoma multiforme. Missense mutations of *FGFR2* are found in endometrial uterine cancer and melanoma, and similar *FGFR3* mutations in invasive bladder tumors, and *FGFR4* mutations in rhabdomyosarcoma. Dovitinib, Ki23057, ponatinib, and AZD4547 are orally bioavailable FGFR inhibitors, which have demonstrated striking effects in preclinical model experiments. Dovitinib, ponatinib, and AZD4547 are currently in clinical trial as anticancer drugs. Because there are multiple mechanisms of actions for FGFR inhibitors to overcome drug resistance, FGFR-targeted therapy is a promising strategy for the treatment of refractory cancer. Whole exome/transcriptome sequencing will be introduced to the clinical laboratory as the companion diagnostic platform facilitating patient selection for FGFR-targeted therapeutics in the era of personalized medicine.

© 2013 Wiley Periodicals, Inc. Med. Res. Rev., 00, No. 0, 1–21, 2013

**Key words:** FGF receptor; gene amplification; translocation; point mutation; tyrosine kinase inhibitor

---

### 1. INTRODUCTION

Receptor tyrosine kinases (RTKs) are transmembrane proteins that function as growth factor receptors, each having an extracellular ligand-binding domain, and a cytoplasmic tyrosine

---

**Correspondence to:** Masaru Katoh, Division of Integrative Omics and Bioinformatics, National Cancer Center, 5-1-1 Tsukiji, Chuo Ward, Tokyo 104-0045, Japan. E-mail: mkatoh-kkr@umin.ac.jp.

Medicinal Research Reviews, 00, No. 0, 1–21, 2013  
© 2013 Wiley Periodicals, Inc.

2. KATO H AND NAKAGAMA

kinase domain. ${ }^{1,2}$ The RTK superfamily consists of multiple families, including the epidermal growth factor receptor (EGFR) family, ${ }^{3,4}$ the vascular endothelial growth factor receptor (VEGFR) family, ${ }^{5,6}$ the fibroblast growth factor receptor (FGFR) family, ${ }^{7,8}$ the RET family, ${ }^{9,10}$ and the anaplastic lymphoma kinase (ALK) family. ${ }^{11,12}$ RTKs are involved in a variety of physiological responses during embryogenesis and adult tissue homeostasis. Tumor cells become proliferative, and anti-apoptotic as a result of aberrant or oncogenic activation of RTKs during carcinogenesis, in a process called as addiction to oncogenic RTKs, or simply oncogenic addiction. ${ }^{13}$ Because cancer cells with oncogenic addiction are exceptionally susceptible to appropriate RTK inhibitors, RTKs are the most intensively pursued classes of targets for anticancer drugs. ${ }^{14}$ Small-molecule tyrosine kinase inhibitors (TKIs) or human/humanized monoclonal antibodies targeted to the EGFR and VEGFR signaling systems are currently utilized as cancer-targeting drugs in the clinic. ${ }^{3,4,6,15-17}$

Carcinogenesis is a multistep process in which epigenetic changes and genetic alterations accumulate in the genomes of cancer cells with germline variations, such as single nucleotide polymorphisms (SNPs) and copy number variations (CNVs). Epigenetic changes and genetic alterations lead to the heterogeneity of cancer cells, which results in the evolution of subclones with more malignant phenotypes. ${ }^{18-20}$ Drugs targeted to the EGFR or VEGFR signaling systems are powerful tools for personalized cancer therapy; however, the evolution of drug-resistant clones is one of the most serious issues in the field of clinical oncology. ${ }^{21}$ Tumors become resistant to EGFR-or VEGFR-targeting drugs as a result of secondary mutations of the targeted RTK or the addiction to downstream effectors and other classes of RTKs. ${ }^{14}$ To address the TKI-resistance problem as a result of subclonal gene amplification and overexpression of FGFRs, a novel class of anticancer drugs targeted to FGFRs has been developed by global pharmaceutical companies.

FGF1 (aFGF), FGF2 (bFGF), FGF3 (INT2), FGF4, FGF5, FGF6, FGF7 (KGF), FGF8, FGF9, FGF10, FGF16, FGF17, FGF18, FGF19, FGF20, FGF21, FGF22, and FGF23 are secreted-type ligands of the FGF family. FGFs bind to multiple FGFRs to transduce signals in target cells. ${ }^{7}$ Heparan sulfate proteoglycans (HSPGs) and Klotho family members are associated with FGFRs to adjust FGF binding to FGFRs. ${ }^{8,22,23}$ FGF signaling is coordinated in a tissue-specific manner based on the interaction of FGFs, FGFRs, HSPGs, and Klotho-type co-receptors. The FGFR family consists of FGFR1b, FGFR1c, FGFR2b, FGFR2c, FGFR3b, FGFR3c, and FGFR4, which share a common domain architecture consisting of extracellular immunoglobulin-like domains and cytoplasmic tyrosine kinase domain. ${ }^{7,8}$ Three-dimensional structure of FGFRs will be reviewed in detail elsewhere. The $FGFR1$ gene at the human chromosomal locus 8p12 encodes FGFR1b and FGFR1c isoforms, which display divergence in the third immunoglobulin-like domain due to alternative splicing. The $FGFR2$ gene at the human chromosomal locus 10q26 and the $FGFR3$ gene at the human chromosomal locus 4p16.3 also encode two isoforms. $FGFR$ genes are proto-oncogenes activated in human cancers as a result of gene amplification, chromosomal translocation, and point mutation. ${ }^{24-28}$

FGFR-targeted drugs exert direct as well as indirect anticancer effects, because FGFRs on cancer cells and endothelial cells are involved in tumorigenesis and vasculogenesis, respectively. ${ }^{29}$ In this manuscript, recent advances in cancer therapeutics targeted to FGFRs will be reviewed with the following structure: Overview of FGF signaling cascades and their cross-talk with other regulatory signaling cascades will be described in the early part; genetic and genomic alterations of FGFRs in human cancers will be described in the middle part; FGFR-targeted therapeutics and perspectives will be described in the last part.

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 3

![Diagram](attachment:FGF_Signaling_Pathway.png)

Figure 1. Fibroblast growth factor (FGF) signaling and the responses of target cells. FGF signals are transduced to the RAS-MAPK, and PI3K-AKT signaling branches via FGFRs and FRS2. FGF signals are also transduced to the DAG-PKC, and IP3-Ca²⁺-releasing signaling branches via FGFRs and PLCγ. FGF signals are involved in stemness, proliferation, anti-apoptosis, drug resistance, angiogenesis, epithelial-to-mesenchymal transition (EMT), and invasion in target cells.

## 2. OVERVIEW OF FGF SIGNALING

FGF triggers the autophosphorylation of FGFR at a key tyrosine residue in an activation loop of the tyrosine kinase domain. FGFR autophosphorylation results in a structural change of the tyrosine kinase domain from an inactive form to an active form.³⁰ The activated tyrosine kinase domain of FGFR then phosphorylates other tyrosine residues at substrate-binding sites along with FGFR-bound adaptor molecules in a stepwise manner (Fig. 1). The phosphorylation of tyrosine residue in the C-terminal region of FGFR enables the binding site of phospholipase Cγ (PLCγ) to recruit and activate PLCγ for the catalysis of phosphatidylinositol diphosphate (PIP₂) to diacylglycerol (DAG) and inositol triphosphate (IP₃).³¹ Activated FGFR phosphorylates FGF receptor substrate 2 (FRS2) to recruit the GRB2 adaptor molecule.⁷,⁸ FGF signals are transduced to the RAS-MAPK or PI3K-AKT signaling cascades through FRS2 and GRB2, to the PKC or PKD pathways through PLCγ and DAG, and to the Ca²⁺-releasing cascade through PLCγ and IP₃ (Fig. 1). SRC tyrosine kinase are also activated by FGF signals.³² FGF-induced RAS-MAPK activation is involved in cellular proliferation, whereas FGF-induced PI3K-AKT activation is involved in cellular survival. Rab5 small GTPase is a binding partner of activated FGFRs that is involved in sustaining the propagation of RAS-MAPK signaling but not PI3K-AKT signaling.³³ FGF signals trigger a variety of responses in target cells (Fig. 1).

## 3. FGF SIGNALING CASCADES AND THE STEM-CELL SIGNALING NETWORK

Embryonic stem cells (ESCs) and epiblast stem cells (EpiSCs) are pluripotent stem cells that generate derivatives of all three germ layers. Human ESCs and mouse EpiSCs are

Medicinal Research Reviews DOI 10.1002/med

4. KATO H AND NAKAGAMA

supported by FGF and TGFβ/nodal/activin signals, whereas mouse ESCs are supported by LIF and canonical WNT signals. FGF signaling cascades crosstalk with WNT, Notch, Hedgehog, and TGFβ/BMP signaling cascades during embryogenesis, adult-tissue homeostasis, and carcinogenesis.³⁴⁻³⁹

Ligands, receptors, effectors, and targets of FGF, WNT, Notch, Hedgehog, and TGFβ/BMP signaling cascades as well as their correlation and involvement in carcinogenesis had been summarized in a review article published in 2011.³⁴ In this section, involvement of stem-cell signaling network in carcinogenesis will be described from an FGF's point of view.

Mouse *Fgf3*, *Fgf4*, *Fgf10*, *Wnt1*, *Wnt3*, and *Wnt10b* are upregulated as a result of the proviral integration of mouse mammary tumor virus (MMTV) during tumorigenesis in the mammary gland.⁴⁰⁻⁴² Mammary tumors occur in MMTV-*Fgf3* or MMTV-*Wnt1* transgenic mice. Mammary tumorigenesis is accelerated by *Wnt10b* upregulation as a result of additional MMTV integration in the MMTV-*Fgf3* transgenic mice, and by *Fgf3* upregulation due to additional MMTV integration in the MMTV-*Wnt1* transgenic mice.⁴³ Mammary tumorigenesis is also accelerated in MMTV-*Wnt1/iFGFR1* bigenic mice compared with MMTV-*Wnt1* transgenic mice.⁴⁴ During colorectal tumorigenesis in mice carrying germline or somatic *Apc* mutations, components of the FGF and WNT signaling cascades were identified as common insertion sites using the Sleeping Beauty transposon system.⁴⁵ FGF and WNT cooperate during carcinogenesis; however, FGF and WNT have opposing effects in the adipocytic differentiation of mesenchymal stem cells.⁴⁷ For example, FGF10 promotes adipocytic differentiation, whereas WNT10B inhibits adipocytic differentiation.⁴⁸ FGF and WNT display cooperation or opposition in a context-dependent manner.

WNT signals are transduced to canonical and noncanonical signaling cascades.⁴⁹⁻⁵¹ WNT signaling via the frizzled receptor and the LRP5/6 co-receptor to the β-catenin-TCF/LEF branch is designated as belonging to the canonical WNT signaling cascade,⁵²,⁵³ whereas WNT signaling via the Frizzled receptor and/or the ROR1/ROR2 receptor to the RhoA-ROCK, RhoB-Rab4, Rac-JNK, and MAP3K7-NLK branches are designated as noncanonical WNT signaling cascades.⁵⁴⁻⁵⁸ To advance our understanding of the FGF-WNT signaling network, multilevel interactions between FGF and WNT signaling cascades are described in following paragraphs.

The FGF signaling pathway crosstalks with the canonical WNT signaling cascade to regulate the transcription of target genes. FGF family members, such as *FGF18* and *FGF20*, are directly upregulated by the canonical WNT signaling cascade as a result of transcriptional activation depending on the β-catenin-TCF/LEF complex.⁵⁹,⁶⁰ FGF signaling through FGFRs then induces a tyrosine phosphorylation of β-catenin at the C-terminus to release β-catenin from the adherens junction and reduces the serine/threonine phosphorylation of β-catenin at the N-terminus to release β-catenin from proteasome-mediated degradation, which results in the potentiation of the canonical WNT signaling cascade in a positive feedback regulation mechanism.⁶¹⁻⁶³ FGF2 signaling to MAPK also regulates the amplitude of LRP6 phosphorylation within its intracellular PPPS/TP motifs to potentiate the canonical WNT signaling cascade.⁶⁴ Alternatively, FGF-induced cellular proliferation can potentiate canonical WNT signaling through the increased phosphorylation of LRP6 by cyclin Y/PFTK in cells at the G2/M phase.⁶⁵ In addition, FGF signaling to the RAS-MAPK branch also regulates the transcription of *Tcf* via Ets, causing the induction and differentiation of pigment cells during the early gastrulation to neural tube closure stages of embryogenesis in *Ciona intestinalis*.⁶⁶ FGF and canonical WNT signals mutually regulate their transcription programs at the levels of ligands, receptors, and transcriptional regulators to coordinate cell fate and proliferation (Fig. 2).

The FGF signaling pathway crosstalks with noncanonical WNT signaling cascades to regulate cellular phenotype and migration. FGF7 and FGF10 signaling through FGFR2b

Medicinal Research Reviews DOI 10.1002/med

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 5

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 2. The network of fibroblast growth factor (FGF), WNT, Notch, Hedgehog, and TGFβ signaling cascades. FGF signals are transduced to the RAS-MAPK and PI3K-AKT-GSK3β signaling branches. WNT signals are transduced to the β-catenin-dependent (canonical) and β-catenin-independent (noncanonical) signaling branches. Notch and Hedgehog signals are transduced to the NICD-CSL and GLI signaling effectors, respectively. TGFβ signals are transduced to the Smad or NF-κB signaling effectors. FGF-RAS-MAPK and canonical WNT signaling cascades mutually regulate transcription programming at multiple levels. FGF-PI3K-AKT-GSK3β, Hedgehog, Notch, TGFβ and noncanonical WNT signaling cascades regulate epithelial-to-mesenchymal transition (EMT) and invasion through the transcriptional or posttranslational control of SNAI1, SNAI2, ZEB1, and ZEB2.

to SRC induce tyrosine phosphorylation of an F-actin binding protein, cortactin, which is involved in endocytosis, clathrin-dependent internalization, and the polarization of FGFR2b to the leading edge for the regulation of cellular migration.⁶⁷ SNAI1 (Snail), SNAI2 (Slug), SNAI3, ZEB1, ZEB2, TWIST1 (TWIST), and TWIST2 are transcription factors that are involved in the epithelial-to-mesenchymal transition (EMT), which is defined as fibroblastic morphological changes in epithelial cells associated with increased motility or invasiveness due to decreased cell-cell adhesion.⁶⁸⁻⁷¹ FGF signaling to the PI3K-AKT branch downregulates GSK3β activity, which leads to EMT through increased stability and nuclear translocation of SNAI1.⁴⁶,⁷² Mutations of FGFRs are genetically linked to TWIST1.⁷³ FGFR2 is expressed at the invasion front of colorectal cancer,⁷⁴ and the presence of tumor buds, defined as the spread of single cells or small clusters of dedifferentiated tumor cells, indicates that cancer cells at the invasive front are undergoing EMT.⁷⁵ Notably, Hedgehog signals activate TGFβ, and activated TGFβ induces the transcriptional upregulation of SNAI2 via the Smad3 signaling branch, and that of ZEB1 and ZEB2 via the NF-κB signaling branch.⁷⁶⁻⁷⁹ Hedgehog signals also upregulate the Notch ligand Jag2, which then induces the transcriptional upregulation of SNAI1 via the NICD-CSL signaling branch.⁷⁹,⁸⁰ In addition, Hedgehog-TGFβ signaling axis induces the transcriptional upregulation of WNT5A, which is a representative noncanonical WNT ligand.³⁴,⁸¹⁻⁸⁵ WNT5A induces the upregulation of SNAI1 protein via PKC and promotes the migration and invasion of cancer cells through the activation of noncanonical WNT signaling

Medicinal Research Reviews DOI 10.1002/med

6. KATOH AND NAKAGAMA

cascades.86–88 FGF, Hedgehog, Notch, TGFβ, and noncanonical WNT signals regulate EMT, cellular migration, tumor invasion, and metastasis (Fig. 2).

4. GENETICS AND GENOMICS OF FGFRs IN HUMAN CANCER

Genetic variations, especially SNPs, and genomic alterations, such as gene amplification, chromosomal translocation, and point mutation, are involved in the transcriptional upregulation of FGFR mRNAs and the functional activation of FGFR proteins during carcinogenesis.24–29 SNPs in the FGFR2 gene were initially discovered to be associated with an increased risk of breast cancer in a genome-wide association study;89 this finding was then confirmed using a candidate-gene approach in a larger-scale study90 and in studies in various ethnic groups.91 Breast cancer-associated SNPs, such as rs10736303, rs1219648, rs2912778, rs2912781, rs2981575, rs2981578, rs2981582, rs33971856, rs35054928, rs35393331, and rs7895676, are located within the intronic region of the FGFR2 gene. The risk allele of rs10736303 gives rise to an estrogen receptor (ER)-binding site.89 The risk allele of rs2981582 is associated with ER-positive breast cancer,92 whereas the risk allele of rs2981578 is associated with increased FGFR2 expression and potentiated responses to FGF2 in fibroblasts rather than epithelial cells.93 BRCA2 mutation carriers are associated with ER-positive breast cancer, and FGFR2rs2981575 SNP shows the strongest association with BRCA2-mutated breast cancer.94 In addition, the risk allele of rs1219648 is associated with lymph-node metastasis in breast cancer patients95 and secondary breast cancer after irradiation therapy for Hodgkin lymphoma.96

Gene amplification is a segmental copy number gain that causes overexpression of an oncogene or cancer-driving gene product in tumor cells. Gene amplification of FGFR1 or FGFR2 occurs in several types of human cancers.97 FGFR1 is amplified in 22% of squamous cell lung cancer,98 and 10% of ER-positive breast cancer,99 whereas FGFR2 is amplified in 9% of diffuse-type gastric cancer100 and rarely in triple-negative breast cancer.101 The human FGFR1 and ZNF703 genes at chromosomal locus 8p12 are co-amplified in some cases of breast cancer, but only ZNF703 is amplified in other cases.102 Breast cancer with 8p12 amplification can be further divided into ZNF703 subgroups with or without FGFR1 amplification. Because of rearrangement at the boundary of the amplified region (the amplicon), C-terminally truncated FGFR2 with constitutive activation is overexpressed in some cases of breast cancer and gastric cancer with FGFR2 amplification. Amplification of FGFR family genes leads to tumors with more malignant phenotypes as a result of the ligand-independent activation of FGF signaling cascades, as evidenced by the overexpression of wild-type or C-terminally truncated FGFRs (Fig. 3).

Chromosomal translocation gives rise to fusion gene products with dysregulated function. Chromosomal translocation was first discovered in hematological malignancies based on a focused genomic approach using karyotype analyses, whereas it is currently easily found in solid tumors based on a whole-genome approach using next-generation sequencing technologies.10,103 The FGFR1 gene is found fused to other genes, such as ZNF198, CEP110, BCR, FGFR1OP1, FGFR1OP2, HERVK, TRIM24, LRRFIP1, MYO18A, CUX1, and MYST3, in the 8p11 myeloproliferative syndrome manifested by myeloproliferative neoplasms and peripheral blood eosinophilia without basophilia.104 The FGFR1 gene is fused to the FOXO1 gene in alveolar rhabdomyosarcoma, and the FOXO1-FGFR1 fusion gene is amplified.105 The FGFR3 gene is fused to the MMSET gene as a result of at(4;14)(p16.3;q32) chromosomal translocation in 10–20% of multiple myeloma,106 and is fused to the ETV6 gene in peripheral T-cell lymphoma with at(4;12)(p16;p13) chromosomal translocation.107

Medicinal Research Reviews DOI 10.1002/med

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 7

Breast cancer  
Lung cancer  
Gastric cancer  

Gene amplification  

Myeloproliferative syndrome  
Multiple myeloma  

Chromosomal translocation  

Endometrial cancer  
Bladder cancer  

Missense mutation  

Orally bioavailable FGFR inhibitors  

Dovitinib  
Ponatinib  
AZD4547  

Figure 3. FGFR inhibitors targeting human cancer cells with FGFR activation. FGFRs are aberrantly activated as a result of gene amplification in breast cancer, lung cancer, and gastric cancer; chromosomal translocation in myeloproliferative syndrome and multiple myeloma; and missense mutation in endometrial cancer and bladder cancer. Dovitinib,^{130–132} Ponatinib^{138,139} and AZD4547^{142} are orally bioavailable FGFR inhibitors targeting human cancer with aberrant FGFR activation.

glioblastoma multiforme, *FGFR1* and *FGFR3* genes are fused to neighboring *TACC1* and *TACC3* genes due to interstitial deletions, respectively.^{108,109}

Fusion gene products consisting of a dimerization domain derived from the fusion partner and a tyrosine kinase domain derived from the *FGFR* family gene are constitutively active as a result of dimerization, and transduce aberrant signals as a result of altered intracellular localization (Fig. 3).

Missense mutations of the *FGFR2* gene were initially discovered in congenital skeletal disorders, such as Crouzon syndrome,^{110} Jackson-Weiss syndrome,^{111} Apert syndrome,^{112} Pfeiffer syndrome,^{113} Beare-Stevenson syndrome,^{114} and Saethre-Chotzen syndrome,^{73} which are characterized by short-limbed bone dysplasia or craniosynostosis. Missense mutations of the *FGFR3* gene also occur in congenital skeletal disorders. As a result of these findings, missense mutations of *FGFR* family genes have been sought for in various types of human cancers. Missense mutations of *FGFR2* occur in 10% of endometrial uterine cancer,^{115,116} and 10% of melanoma,^{117} mutations of *FGFR3* are found in 60% of invasive bladder tumor,^{118} and mutations of *FGFR4* are observed in 7.5% of rhabdomyosarcoma.^{119} Point mutations of *FGFR2* or *FGFR3* occurring in congenital skeletal disorders and human cancers are clustered around the third immunoglobulin-like domain, the transmembrane domain, or the tyrosine kinase domain. Amino acid substitutions around the third immunoglobulin-like domain alter the ligand-binding specificity of FGFRs, and those within the tyrosine kinase domain release FGFRs from

Medicinal Research Reviews DOI 10.1002/med

autoinhibition, giving rise to gain-of-function mutations observed in congenital skeletal disorders, uterine tumor, bladder tumor, and rhabdomyosarcoma. Conversely, in melanoma preferentially expressing the FGFR2c isoform, amino acid substitutions of FGFR 2 around the third immunoglobulin-like domain or within the tyrosine kinase domain are reported to be loss-of-function mutations, at least with respect to kinase activity. FGFRs are proposed to have dual roles in the regulation of cellular proliferation in physiology and pathology. The functional divergence of FGFR mutations may be determined in a context-dependent manner, depending on whether the mutations are driver mutations or passenger mutations, or whether FGF signaling is oncogenic or tumor suppressive.

5. CANCER THERAPEUTICS TARGETED TO FGFRs

FGFR-targeted therapeutics using small-molecule compounds is an active topic in the field of clinical oncology because FGF signaling is involved in various aspects of cancer biology, such as stemness, proliferation, anti-apoptosis, drug resistance, angiogenesis, EMT, and invasion (Fig. 1).

Small-molecule compounds fitting into the ATP-binding pockets of the tyrosine kinase domain have been developed for cancer therapeutics. PD173074, dovitinib (TKI 258), Ki23057, E7080, brivanib alaninate, intedanib (BIBF 1120), ponatinib (AP24534), MK-2461, E-3810, and AZD4547 are TKIs targeted to FGFRs (Table I). In addition, NP603and6b were recently discovered as FGFR-targeting lead compounds. PD173074 is a first-generation FGFR inhibitor that was developed as a specific FGFR inhibitor. Dovitinib, Ki23057, E7080, brivanib alaninate, intedanib, ponatinib, MK-2461, E-3810, and AZD4547 are second-generation FGFR inhibitors that target FGFRs and other RTKs. Dovitinib targets FGFR1 and FGFR3 at single-digit nanomolar levels of enzyme IC50; ponatinib targets FGFR1, FGFR2, and FGFR4 at single-digit nanomolar levels of enzyme IC50; and AZD4547 targets FGFR1, FGFR2, and FGFR3 at single-digit or sub-nanomolar levels of enzyme IC50 (Table I). Dovitinib, Ki23057, ponatinib, and AZD4547 are orally bioavailable FGFR inhibitors that repress the in vitro proliferation of cancer cells with aberrant FGFR activation at two-digit nanomolar levels of cell growth IC50 (Table II) and induce tumor regression in preclinical xenograft model experiments. Dovitinib is in phase III clinical trial for renal cell carcinoma and in phase II clinical trials for advanced breast and endometrial cancers, relapsed multiple myeloma, and urothelial cancer. Ponatinib is in phase II clinical trial as a BCR-ABL inhibitor rather than an FGFR inhibitor. AZD4547 is in phase II clinical trial for breast cancer and in phase I clinical trial for solid tumors. Details of FGFR inhibitors and their clinical trials are also reviewed elsewhere. However, it is noteworthy that endometrial cancer patients with or without FGFR2 mutations are separately enrolled in a clinical trial of dovitinib to prove its mechanisms of functions because multikinase inhibitors mentioned above may function as direct anticancer drugs targeted to FGFRs as well as indirect anticancer drugs targeted to angiogenesis. Breast cancer with gene amplification of FGFR1 or FGFR2, lung cancer with FGFR1 amplification, gastric cancer with FGFR2 amplification, endometrial cancer with FGFR2 point mutation, bladder cancer with FGFR3 point mutation, myeloproliferative syndrome with FGFR1 translocation, and multiple myeloma with FGFR3 translocation are all clinical targets of orally bioavailable FGFR inhibitors (Fig. 3).

Drug resistance is a cutting-edge theme related to FGFR-targeted therapeutics. FGFR inhibitors enhance tumor sensitivity to conventional anticancer drug such as 5-fluorouracil, irinotecan, paclitaxel, and etoposide in human cancer cells acquiring antiapoptotic potential based on aberrant FGFR activation. Additionally, FGF signaling inhibition attenuates revascularization, and reduces tumor burden in human tumors acquiring autocrine FGF

Medical Research Reviews DOI 10.1002/med

Table I. Enzyme IC₅₀ (Nanomolar) of Small-Molecule Compounds Targeting FGFRs

| Dovitinib | Intedanib | Ponatinib |
| --- | --- | --- |
| PD173074 | Brivanibᵃ | (AP24534) |
| (TKI258) | E7080 | MK-2461 |
| Ki23057 | (BIBF 1120) | E-3810 |
| AZD 4547 |

| FGR1 | 46 | 148 | 69 | 2.2 | 65 | 18 | 0.2 |
| FGR2 | 88 | 125 | 37 | 1.6 | 39 | 83 | 1.8 |
| FGR3 | 9 | 68 | 108 | 18 | 50 | 238 | 2.5 |
| FGR4 |  |  | 610 | 7.7 |  |  | 165 |
| VEGFR1 | 10 | 69 | 380 | 34 | 10 |  | 7 |
| VEGFR2 | 13 | 83 | 25 | 21 | 44 | 25 | 24 |
| VEGFR3 | 8 |  | 4.0 | 13 | 78 | 10 |  |
| PDGFRA | 210 | 100 | 5.2 | 59 | 1.1 | 175 |  |
| PDGFRB | 21 | 100 | 51 | 65 | 22 | 525 |  |
| IGFR |  |  | 39 |  |  |  | 581 |
| MET |  |  |  |  |  |  | 2.5 |
| ABL |  |  |  |  |  |  | 0.37 |
| Reference | [129] | [130, 131] | [133, 134] | [135] | [136] | [137] | [138, 139] | [140] | [141] | [142] |

a Because brivanib alaninate is hydrolyzed to the active moiety BMS-540215 in vivo, IC₅₀ of BMS-540215 is shown.

FGFR, fibroblast growth factor receptor; VEGFR, vascular endothelial growth factor receptor.

Dovitinib    Ki23057    Ponatinib    AZD 4547
Breast cancer
MDA-MB-134   FGFR1amp   186         23
SUM52PE      FGFR2amp   63          14        41
MFM 223      FGFR1/2amp 411         69
Lung cancer
H1581        FGFR1amp   216         32
DMS 114      FGFR1amp   818         108
Gastric cancer
KATO-III     FGFR2amp   64          10
SNU-16       FGFR2amp   99          25
OCUM-2MD3    FGFR2amp   <100
OCUM-8       FGFR2amp   <100
Endometrial cancer
AN3CA        FGFR2mut   500         14
MFE 280      FGFR2mut   420
MFE-296      FGFR2mut   660         61
Bladder cancer
MGH-U3       FGFR3mut   204         181
UMUC14       FGFR3mut   182         103
Colon cancer
H716         FGFR2amp   33          7
MPS
KG1a         FGFR1tra   180                  18
Multiple myeloma
KMS11        FGFR3tra/mut 90                   281
KMS18        FGFR3tra/mut 550
OPM2         FGFR3tra/mut 90
References                          [130–132]    [133, 134]    [138, 139]    [142]
amp, gene amplification; MPS, myeloproliferative syndrome; mut, missense mutation; tra, chromosomal translocation.

signaling based on FGF2 upregulation after VEGFR2-targeted therapy. In addition, FGFR inhibitors are predicted to be effective on relapsed tumors based on clonal evolution of an FGFR-activated minor subpopulation after therapy targeted to EGFRs or VEGFRs. Because there are multiple mechanisms of action for FGFR inhibitors to overcome drug resistance in human cancer, FGFR-targeted therapy is a promising strategy for the treatment of refractory cancer.

6. PERSPECTIVES

Monoclonal antibodies selectively recognizing FGF or FGFR present other options for FGFR-targeting cancer therapy. Anti-FGF19 monoclonal antibody inhibits the in vivo growth of colon cancer cells expressing FGFR4 through a blockade of the FGF19-FGFR4-FRS2-ERK signaling axis. Anti-FGFR3 monoclonal antibody inhibits in vitro cell growth and in vivo tumor burden for bladder cancer cells with FGFR3 point mutation or multiple myeloma cells with FGFR3 translocation and point mutation. Anti-FGFR2 monoclonal antibodies inhibit the in vivo growth of SNU-16 and OCUM-2M gastric cancer cells with FGFR2 gene amplification and overexpression. The development of human or humanized FGFR antibodies is crucial in advancing the clinical application of antibody-based therapy targeting FGFR (Fig. 4). Technologies related to antibody-drug conjugates and bispecific antibodies will be introduced to enhance the effectiveness of FGFR-targeting antibody drugs.

RNA-based drugs such as small interfering RNA (siRNA) and synthetic microRNA (miRNA) are emerging as cancer therapeutics. siRNA is a double-stranded, noncoding RNA that is approximately 20–25 nucleotides long, whereas miRNA is a single-stranded.

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 11

Primary tumor  
Peritoneal dissemination  
Malignant pleural effusion  
Distant metastasis  

Next-generation sequencing  
(1) Whole exome  
(2) Transcriptome  

EGFRs-  
driven  

VEGFRs-  
driven  

FGFRs-driven  

ALK-  
driven  

RET-  
driven  

Refractory  
Cured  

MoAb  

RNA-  
based  

TKI  

Cured  
Refractory  

Figure 4. Perspectives on fibroblast growth factor receptor (FGFR)-targeted therapeutics. Whole-exome sequencing combined with transcriptome sequencing will be applied in clinical laboratory test to determine driver mutations in tumor samples. Primary and refractory tumors driven by aberrantly activated FGFRs will be treated with FGFR-targeted therapeutics, such as small-molecule FGFR inhibitor (TKI), human/humanized anti-FGFR monoclonal antibody (MoAb), and RNA-based drug.

noncoding RNA of approximately 22 nucleotides in length. siRNA and miRNA are involved in RNA interference (RNAi) that controls the expression of target proteins as evidenced by mRNA degradation or translational repression. A variety of key processes during carcinogenesis, such as stemness, proliferation, EMT, invasion, and metastasis, are regulated by endogenous miRNAs. miR-433 represses FGF20 expression in Parkinson’s disease patients with the rs12720208 SNP risk allele. miR-125b represses FGFR2 expression in keratinocytes. miR-21 represses the expression of FGF signaling inhibitor Spry1 in cardiac fibroblasts. miRNAs involved in cellular proliferation, survival, angiogenesis, EMT, or invasion could be suppressed in cancer therapy, whereas miRNAs involved in the opposite processes could be overexpressed to treat cancer patients. Off-target effects and delivery method are currently obstacles for the clinical application of RNA-based drugs. The accumulation of knowledge on the miRNA network regulation of FGFR signaling will lead to the development of improved RNA-based drugs (Fig. 4).

Companion diagnostics are essential to enable the selection of patients for the FGFR-targeting therapeutics in the era of personalized medicine. Fluorescence in situ hybridization (FISH) can be utilized for the detection of FGFR gene amplification or the chromosomal translocation of FGFR family genes in the clinical laboratory, and candidate-mRNA sequencing can be utilized for the detection of point mutation of FGFR family genes. Nucleotide sequencing technology has been drastically improved in line with the Moore’s law, where bioinformatics data production rates have doubled approximately every 2 years. Whole-exome sequencing and transcriptome sequencing based on next-generation sequencing technology are currently

Medicinal Research Reviews DOI 10.1002/med

12. KATO H AND NAKAGAMA

utilized in the basic research laboratory to catalogue genetic alterations in human cancers. 164–167

Whole-exome sequencing combined with transcriptome sequencing will next be applied in the clinical laboratory test to determine driver mutations in tumor samples derived from primary tumors, peritoneal dissemination, malignant pleural effusion, or distant metastasis (Fig. 4). Primary and refractory tumors driven by aberrantly activated FGFRs will be treated with FGFR-targeting therapeutics, such as small-molecule FGFR inhibitors, human/humanized anti-FGFR monoclonal antibodies, and RNA-based drugs (Fig. 4).

CONFLICT OF INTEREST

The authors declare no conflict of interest.

ACKNOWLEDGMENTS

M.K. and H.K. contributed to the preparation of this manuscript. This study was supported in part by National Cancer Center Research and Development Fund 2011.

REFERENCES

1. Manning G, Whyte DB, Martinez R, Hunter T, Sudarsanam S. The protein kinase complement of the human genome. Science 2002; 298:1912–1934.
2. Krause DS, Van Etten RA. Tyrosine kinases as targets for cancer therapy. N Engl J Med 2005; 353:172–187.
3. Kamath S, Buolamwini JK. Targeting EGFR and HER-2 receptor tyrosine kinases for cancer drug discovery and development. Med Res Rev 2006; 26:569–594.
4. Ciardiello F, Tortora G. EGFR antagonists in cancer treatment. N Engl J Med 2008; 358:1160–1174.
5. Tammela T, Enholm B, Alitalo K, Paavonen K. The biology of vascular endothelial growth factors. Cardiovasc Res 2005; 65:550–563.
6. Ellis LM, Hicklin DJ. VEGF-targeted therapy: Mechanisms of anti-tumour activity. Nat Rev Cancer 2008; 8:579–591.
7. Eswarakumar VP, Lax I, Schlessinger J. Cellular signaling by fibroblast growth factor receptors. Cytokine Growth Factor Rev 2005; 16:139–149.
8. Beenken A, Mohammadi M. The FGF family: Biology, pathophysiology and therapy. Nat Rev Drug Discov 2009; 8:235–253.
9. Pulford K, Morris SW, Turturro F. Anaplastic lymphoma kinase proteins in growth control and cancer. J Cell Physiol 2004; 199:330–358.
10. Kwak EL, Bang YJ, Camidge DR, Shaw AT, Solomon B, Maki RG, Ou SH, Dezube BJ, Jänne PA, Costa DB, Varella-Garcia M, Kim WH, Lynch TJ, Fidias P, Stubbs H, Engelman JA, Sequist LV, Tan W, Gandhi L, Mino-Kenudson M, Wei GC, Shreeve SM, Ratain MJ, Settleman J, Christensen JG, Haber DA, Wilner K, Salgia R, Shapiro GI, Clark JW, Iafrate AJ. Anaplastic lymphoma kinase inhibition in non-small-cell lung cancer. N Engl J Med 2010; 363:1693–1703.
11. Arighi E, Borrello MG, Sariola H. RET tyrosine kinase signaling in development and cancer. Cytokine Growth Factor Rev 2005; 16:441–467.
12. Mologni L. Development of RET kinase inhibitors for targeted cancer therapy. Curr Med Chem 2011; 18:162–175.
13. Weinstein IB, Joe AK. Mechanisms of disease: Oncogene addiction—A rationale for molecular targeting in cancer therapy. Nature Clin Pract Oncol 2006; 3:448–457.
14. Zhang J, Yang PL, Gray NS. Targeting cancer with small molecule kinase inhibitors. Nat Rev Cancer 2009; 9:28–39.

Medicinal Research Reviews DOI 10.1002/med

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 13

15. Tortora G, Ciardiello F, Gasparini G. Combined targeting of EGFR-dependent and VEGF-dependent pathways: Rationale, preclinical studies and clinical applications. Nat Clin Pract Oncol 2008;5:521–530.

16. Takeuchi K, Ito F. Receptor tyrosine kinases and targeted cancer therapeutics. Biol Pharm Bull 2011;34:1774–1780.

17. Saxena R, Dwivedi A. ErbB family receptor inhibitors as therapeutic agents in breast cancer: Current status and future clinical perspective. Med Res Rev 2012;32:166–215.

18. Katoh M. Dysregulation of stem cell signaling network due to germline mutation, SNP, helicobacter pylori infection, epigenetic change and genetic alteration in gastric cancer. Cancer Biol Ther 2007;6:832–839.

19. Bennett LB, Taylor KH, Arthur GL, Rahmatpanah FB, Hooshmand SI, Caldwell CW. Epigenetic regulation of WNT signaling in chronic lymphocytic leukemia. Epigenomics 2010;2:53–70.

20. Balch C, Matei DE, Huang TH, Nephew KP. Role of epigenomics in ovarian and endometrial cancers. Epigenomics 2010;2:419–447.

21. Bertolini F, Marighetti P, Martin-Padura I, Mancuso P, Hu-Lowe DD, Shaked Y, D'Onofrio A. Anti-VEGF and beyond: Shaping a new generation of anti-angiogenic therapies for cancer. Drug Discov Today 2011;16:1052–1060.

22. Mohammadi M, Olsen SK, Goetz R. A protein canyon in the FGF-FGF receptor dimer selects from an à la carte menu of heparan sulfate motifs. Curr Opin Struct Biol 2005;15:506–516.

23. Masola V, Gambaro G, Tibaldi E, Brunati AM, Gastaldello A, D'Angelo A, Onisto M, Lupo A. Heparanase and syndecan-1 interplay orchestrates fibroblast growth factor-2-induced epithelial-mesenchymal transition in renal tubular cells. J Biol Chem 2012;287:1478–1488.

24. Grose R, Dickson C. Fibroblast growth factor signaling in tumorigenesis. Cytokine Growth Factor Rev 2005;16:179–186.

25. Katoh M. Cancer genomics and genetics of FGFR2. Int J Oncol 2008;33:233–237.

26. Ivan M, Matei D. Blockade of FGF signaling: Therapeutic promise for ovarian cancer. Cancer Biol Ther 2010;10:505–508.

27. Turner N, Grose R. Fibroblast growth factor signalling: From development to cancer. Nat Rev Cancer 2010;10:116–129.

28. Greulich H, Pollock PM. Targeting mutant fibroblast growth factor receptors in cancer. Trends Mol Med 2011;17:283–292.

29. Daniele G, Corral J, Molife LR, de Bono JS. FGF receptor inhibitors: Role in cancer therapy. Curr Oncol Rep 2012;14:111–119.

30. Bae JH, Schlessinger J. Asymmetric tyrosine kinase arrangements in activation or autophosphorylation of receptor tyrosine kinases. Mol Cells 2010;29:443–448.

31. Dailey L, Ambrosetti D, Mansukhani A, Basilico C. Mechanisms underlying differential responses to FGF signaling. Cytokine Growth Factor Rev 2005;16:233–247.

32. Cunningham DL, Sweet SM, Cooper HJ, Heath JK. Differential phosphoproteomics of fibroblast growth factor signaling: Identification of Src family kinase-mediated phosphorylation events. J Proteome Res 2010;9:2317–2328.

33. Vecchione A, Cooper HJ, Trim KJ, Akbarzadeh S, Heath JK, Wheldon LM. Protein partners in the life history of activated fibroblast growth factor receptors. Proteomics 2007;7:4565–4578.

34. Katoh M. Network of WNT and other regulatory signaling cascades in pluripotent stem cells and cancer stem cells. Curr Pharm Biotechnol 2011;12:160–170.

35. Canning CA, Lee L, Luo SX, Graham A, Jones CM. Neural tube derived Wnt signals cooperate with FGF signaling in the formation and differentiation of the trigeminal placodes. Neural Dev 2008;3:35.

36. Nyeng P, Norgaard GA, Kobberup S, Jensen J. FGF 10 maintains distal lung bud epithelium and excessive signaling leads to progenitor state arrest, distalization, and goblet cell metaplasia. BMC Dev Biol 2008;8:2.

14. KATOH AND NAKAGAMA

37. Wang Z, Li Y, Kong D, Sarkar FH. The role of Notch signaling pathway in epithelial-mesenchymal transition (EMT) during development and tumor aggressiveness. Curr Drug Targets 2010; 11: 745–751.

38. He S, Wang F, Yang L, Guo C, Wan R, Ke A, Xu L, Hu G, Xu X, Shen J, Wang X. Expression of DNMT1 and DNMT3a are regulated by GLI1 in human pancreatic cancer. PLoS One 2011; 6: e27684.

39. Janikova M, Skarda J. Differentiation pathways in carcinogenesis and in chemo-and radioresistance. Neoplasma 2012; 59: 6–17.

40. Nusse R, van Ooyen A, Cox D, Fung YK, Varmus H. Mode of proviral activation of a putative mammary oncogene (int-1) on mouse chromosome 15. Nature 1984; 307: 131–136.

41. Dickson C, Smith R, Brookes S, Peters G. Tumorigenesis by mouse mammary tumor virus: Proviral activation of a cellular gene in the common integration region int-2. Cell 1984; 37: 529–536.

42. Katoh M. WNT and FGF gene clusters. Int J Oncol 2002; 21: 1269–1273.

43. Lee FS, Lane TF, Kuo A, Shackleford GM, Leder P. Insertional mutagenesis identifies a member of the Wnt gene family as a candidate oncogene in the mammary epithelium of int-2/Fgf-3 transgenic mice. Proc Natl Acad Sci USA 1995; 92: 2268–2272.

44. Pond AC, Herschkowitz JI, Schwertfeger KL, Welm B, Zhang Y, York B, Cardiff RD, Hilsenbeck S, Perou CM, Creighton CJ, Lloyd RE, Rosen JM. Fibroblast growth factor receptor signaling dramatically accelerates tumorigenesis and enhances oncoprotein translation in the mouse mammary tumor virus-Wnt-1 mouse model of breast cancer. Cancer Res 2010; 70: 4868–4879.

45. March HN, Rust AG, Wright NA, ten Hoeve J, de Ridder J, Eldridge M, van der Weyden L, Berns A, Gadiot J, Uren A, Kemp R, Arends MJ, Wessels LF, Winton DJ, Adams DJ. Insertional mutagenesis identifies multiple networks of cooperating genes driving intestinal tumorigenesis. Nat Genet 2011; 43: 1202–1209.

46. Katoh M, Katoh M. Cross-talk of WNT and FGF signaling pathways at GSK3β to regulate β-catenin and SNAIL signaling cascades. Cancer Biol Ther 2006; 5: 1059–1064.

47. Katoh M. WNT signaling in stem cell biology and regenerative medicine. Curr Drug Targets 2008; 9: 565–570.

48. Ross SE, Hemati N, Longo KA, Bennett CN, Lucas PC, Erickson RL, MacDougald OA. Inhibition of adipogenesis by Wnt signaling. Science 2000; 289: 950–953.

49. Katoh M, Katoh M. WNT signaling pathway and stem cell signaling network. Clin Cancer Res 2007; 13: 4042–4045.

50. Chien AJ, Conrad WH, Moon RT. A Wnt survival guide: From flies to human disease. J Invest Dermatol 2009; 129: 1614–1627.

51. Filipovich A, Gehrke I, Poll-Wolbeck SJ, Kreuzer KA. Physiological inhibitors of Wnt signaling. Eur J Haematol 2011; 86: 453–465.

52. Barker N, Clevers H. Mining the Wnt pathway for cancer therapeutics. Nat Rev Drug Discov 2006; 5: 997–1014.

53. El Wakil A, Lalli E. The Wnt/β-catenin pathway in adrenocortical development and cancer. Mol Cell Endocrinol 2011; 332: 32–37.

54. Lagerström MC, Schiöth HB. Structural diversity of G protein-coupled receptors and significance for drug discovery. Nat Rev Drug Discov 2008; 7: 339–357.

55. Fukuda T, Chen L, Endo T, Tang L, Lu D, Castro JE, Widhopf GF2nd, Rassenti LZ, Cantwell MJ, Prussak CE, Carson DA, Kipps TJ. Antisera induced by infusions of autologous Ad-CD154-leukemia B cells identify ROR1 as an oncofetal antigen and receptor for Wnt5a. Proc Natl Acad Sci USA 2008; 105: 3047–3052.

56. Wang Y. Wnt/Planar cell polarity signaling: A new paradigm for cancer therapy. Mol Cancer Ther 2009; 8: 2103–2109.


Medicinal Research Reviews DOI 10.1002/med

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 15

57. Minami Y, Oishi I, Endo M, Nishita M. Ror-family receptor tyrosine kinases in noncanonical Wnt signaling: Their implications in developmental morphogenesis and human diseases. Dev Dyn 2010;239:1–15.

58. Koval A, Katanaev VL. Wnt3a stimulation elicits G-protein-coupled receptor properties of mammalian Frizzled proteins. Biochem J 2011;433:435–440.

59. Shimokawa T, Furukawa Y, Sakai M, Li M, Miwa N, Lin YM, Nakamura Y. Involvement of the FGF18 gene in colorectal carcinogenesis, as a novel downstream target of the β-catenin/T-cell factor complex. Cancer Res 2003;63:6116–6120.

60. Chamorro MN, Schwartz DR, Vonica A, Brivanlou AH, Cho KR, Varmus HE. FGF20 and DKK1 are transcriptional targets of beta-catenin and FGF-20 is implicated in cancer and development. EMBO J 2005;24:73–84.

61. El-Hariry I, Pignatelli M, Lemoine NR. FGF-1 and FGF-2 modulate the E-cadherin/catenin system in pancreatic adenocarcinoma cell lines. Br J Cancer 2001;84:1656–1663.

62. Brembeck FH, Rosário M, Birchmeier W. Balancing cell adhesion and Wnt signaling, the key role of β-catenin. Curr Opin Genet Dev 2006;16:51–59.

63. Pai R, Dunlap D, Qing J, Mohtashemi I, Hotzel K, French DM. Inhibition of fibroblast growth factor 19 reduces tumor growth by modulating β-catenin signaling. Cancer Res 2008;68:5086–5095.

64. Cervenka I, Wolf J, Mašek J, Krejci P, Wilcox WR, Kozubík A, Schulte G, Gutkind JS, Bryja V. Mitogen-activated protein kinases promote WNT/β-catenin signaling via phosphorylation of LRP6. Mol Cell Biol 2011;31:179–189.

65. Davidson G, Shen J, Huang YL, Su Y, Karaulanov E, Bartscherer K, Hassler C, Stannek P, Boutros M, Niehrs C. Cell cycle control of Wnt receptor activation. Dev Cell 2009;17:788–799.

66. Squarzoni P, Parveen F, Zanetti L, Ristoratore F, Spagnuolo A. FGF/MAPK/Ets signaling renders pigment cell precursors competent to respond to Wnt signal by directly controlling Ci-Tcf transcription. Development 2011;138:1421–1432.

67. Belleudi F, Scrofani C, Torrisi MR, Mancini P. Polarized endocytosis of the keratinocyte growth factor receptor in migrating cells: Role of SRC-signaling and cortactin. PLoS One 2011;6:e29159.

68. Thiery JP. Epithelial-mesenchymal transitions in tumour progression. Nature Rev Cancer 2002;2:442–454.

69. Katoh M. Epithelial-mesenchymal transition in gastric cancer. Int J Oncol 2005;27:1677–1683.

70. Barrallo-Gimeno A, Nieto MA. Evolutionary history of the Snail/Scratch superfamily. Trends Genet 2009;25:248–252.

71. Turley EA, Veiseh M, Radisky DC, Bissell MJ. Mechanisms of disease: Epithelial-mesenchymal transition—Does cellular plasticity fuel neoplastic progression? Nat Clin Pract Oncol 2008;5:280–290.

72. Zhou BP, Deng J, Xia W, Xu J, Li YM, Gunduz M, Hung MC. Dual regulation of Snail by GSK-3β-mediated phosphorylation in control of epithelial-mesenchymal transition. Nat Cell Biol 2004;6:931–940.

73. Paznekas WA, Cunningham ML, Howard TD, Korf BR, Lipson MH, Grix AW, Feingold M, Goldberg R, Borochowitz Z, Aleck K, Mulliken J, Yin M, Jabs EW. Genetic heterogeneity of Saethre-Chotzen syndrome, due to TWIST and FGFR mutations. Am J Hum Genet 1998;62:1370–1380.

74. Matsuda Y, Ishiwata T, Yamahatsu K, Kawahara K, Hagio M, Peng WX, Yamamoto T, Nakazawa N, Seya T, Ohaki Y, Naito Z. Overexpressed fibroblast growth factor receptor 2 in the invasive front of colorectal cancer: A potential therapeutic target in colorectal cancer. Cancer Lett 2011;309:209–219.

75. Zlobec I, Lugli A. Epithelial mesenchymal transition and tumor budding in aggressive colorectal cancer: Tumor budding as oncotarget. Oncotarget 2010;1:651–661.

76. Huber MA, Kraut N, Beug H. Molecular requirements for epithelial-mesenchymal transition during tumor progression. Curr Opin Cell Biol 2005;17:548–558.

16. KATOH AND NAKAGAMA

77. Zavadil J, Bottinger EP. TGF-β and epithelial-to-mesenchymal transitions. Oncogene 2005; 24: 5764–5774.

78. Katoh Y, Katoh M. Hedgehog signaling, epithelial-to-mesenchymal transition and miRNA. Int J Mol Med 2008; 22: 271–275.

79. Katoh Y, Katoh M. Hedgehog target genes: Mechanisms of carcinogenesis induced by aberrant hedgehog signaling activation. Curr Mol Med 2009; 9: 873–886.

80. Kasper M, Schnidar H, Neill GW, Hanneder M, Klingler S, Blaas L, Schmid C, Hauser-Kronberger C, Regl G, Philpott MP, Aberger F. Selective modulation of Hedgehog/GLI target gene expression by epidermal growth factor signaling in human keratinocytes. Mol Cell Biol 2006; 26: 6283–6298.

81. Saitoh T, Mine T, Katoh M. Frequent up-regulation of WNT5A mRNA in primary gastric cancer. Int J Mol Med 2002; 9: 515–519.

82. Roarty K, Serra R. Wnt5a is required for proper mammary gland development and TGF-β-mediated inhibition of ductal growth. Development 2007; 134: 3929–3939.

83. Katoh M, Katoh M. Transcriptional mechanisms of WNT5A based on NF-κB, Hedgehog, TGFβ, and Notch signaling cascades. Int J Mol Med 2009; 23: 763–769.

84. Pavlovich AL, Boghaert E, Nelson CM. Mammary branch initiation and extension are inhibited by separate pathways downstream of TGFβ in culture. Exp Cell Res 2011; 317: 1872–1884.

85. Krantz SB, Shields MA, Dangi-Garimella S, Munshi HG, Bentrem DJ. Contribution of epithelial-to-mesenchymal transition and cancer stem cells to pancreatic cancer progression. J Surg Res 2012; 173: 105–112.

86. Dissanayake SK, Wade M, Johnson CE, O'Connell MP, Leotlela PD, French AD, Shah KV, Hewitt KJ, Rosenthal DT, Indig FE, Jiang Y, Nickoloff BJ, Taub DD, Trent JM, Moon RT, Bittner M, Weeraratna AT. The Wnt5A/protein kinase C pathway mediates motility in melanoma cells via the inhibition of metastasis suppressors and initiation of an epithelial to mesenchymal transition. J Biol Chem 2007; 282: 17259–17271.

87. Ripka S, König A, Buchholz M, Wagner M, Sipos B, Klöppel G, Downward J, Gress T, Michl P. WNT5A—target of CUTL1 and potent modulator of tumor cell migration and invasion in pancreatic cancer. Carcinogenesis 2007; 28: 1178–1187.

88. Serra R, Easter SL, Jiang W, Baxley SE. Wnt5a as an effector of TGFβ in mammary development and cancer. J Mammary Gland Biol Neoplasia 2011; 16: 157–167.

89. Easton DF, Pooley KA, Dunning AM, Pharoah PD, Thompson D, Ballinger DG, Struwing JP, Morrison J, Field H, Luben R, Wareham N, Ahmed S, Healey CS, Bowman R. Genome-wide association study identifies novel breast cancer susceptibility loci. Nature 2007; 447: 1087–1093.

90. Huijts PE, Vreeswijk MP, Kroeze-Jansema KH, Jacobi CE, Seynaeve C, Krol-Warmerdam EM, Wijers-Koster PM, Blom JC, Pooley KA, Klijn JG, Tollenaar RA, Devilee P, van Asperen CJ. Clinical correlates of low-risk variants in FGFR2, TNRC9, MAP3K1, LSP1 and 8q24 in a Dutch cohort of incident breast cancer cases. Breast Cancer Res 2007; 9: R78.

91. Udler MS, Meyer KB, Pooley KA, Karlsins E, Struewing JP, Zhang J, Doody DR, MacArthur S, Tyrer J, Pharoah PD, Luben R, Bernstein L, Kolonel LN, Henderson BE, Le Marchand L, Ursin G, Press MF, Brennan P, Sangrajrang S, Gaborieau V, Odefrey F, Shen CY, Wu PE, Wang HC, Kang D, Yoo KY, Noh DY, Ahn SH, Ponder BA, Haiman CA, Malone KE, Dunning AM, Ostrander EA, Easton DF. FGFR2 variants and breast cancer risk: Fine-scale mapping using African American studies and analysis of chromatin conformation. Hum Mol Genet 2009; 18: 1692–1703.

92. Garcia-Closas M, Chanock S. Genetic susceptibility loci for breast cancer by estrogen receptor status. Clin Cancer Res 2008; 14: 8000–8009.

93. Huijts PE, van Dongen M, de Goeij MC, van Moolenbroek AJ, Blanken F, Vreeswijk MP, de Kruijf EM, Mesker WE, van Zwet EW, Tollenaar RA, Smit VT, van Asperen CJ, Devilee P. Allele-specific regulation of FGFR2 expression is cell type-dependent and may increase breast cancer risk through a paracrine stimulus involving FGF10. Breast Cancer Res 2011; 13: R72.

94. Gaudet MM, Kirchhoff T, Green T, Vijai J, Korn JM, Guiducci C, Segre AV, McGee K, McGuf-fog L, Kartsonaki C, Morrison J, Healey S, Sinilnikova OM, Stoppa-Lyonnet D, Mazoyer S,

Medicinal Research Reviews DOI 10.1002/med

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 17

Gauthier-Villars M, Sobol H, Longy M, Frenay M. Common genetic variants and modification of penetrance of BRCA2-associated breast cancer. PLoS Genet 2010; 6: e1001183.

95. Chen XH, Li XQ, Chen Y, Feng YM. Risk of aggressive breast cancer in women of Han nationality carrying TGFB1rs1982073C allele and FGFR2rs1219648G allele in North China. Breast Cancer Res Treat 2011; 125: 575–582.

96. Ma YP, van Leeuwen FE, Cooke R, Broeks A, Enciso-Mora V, Olver B, Lloyd A, Broderick P, Russell NS, Janus C, Ashworth A, Houlston RS, Swerdlow AJ. FGFR2 genotype and risk of radiation-associated breast cancer in Hodgkin lymphoma. Blood 2012; 119: 1029–1031.

97. Katoh M. Genetic alterations of FGF receptors: An emerging field in clinical cancer diagnostics and therapeutics. Expert Rev Anticancer Ther 2010; 10: 1375–1379.

98. Weiss J, Sos ML, Seidel D, Peifer M, Zander T, Heuckmann JM, Ullrich RT, Menon R, Maier S, Soltermann A, Moch H, Wagener P, Fischer F, Heynck S, Koker M, Schöttle J, Leenders F, Gabler F, Dabow I, Querings S, Heukamp LC, Balke-Want H, Ansén S, Rauh D, Baessmann I, Altmüller J, Wainer Z, Conron M, Wright G, Russell P, Solomon B, Brambilla E, Brambilla C, Lorimier P, Sollberg S, Brustugun OT, Engel-Riedel W, Ludwig C, Petersen I, Sänger J, Clement J, Groen H, Timens W, Sietsma H, Thunnissen E, Smit E, Heideman D, Cappuzzo F, Ligorio C, Damiani S, Hallek M, Beroukhim R, Pao W, Klebl B, Baumann M, Buettner R, Ernestus K, Stoelben E, Wolf J, Nürnberg P, Perner S, Thomas RK. Frequent and focal FGFR1 amplification associates with therapeutically tractable FGFR1 dependency in squamous cell lung cancer. Sci Transl Med 2010; 2: 62ra93.

99. Courjal F, Cuny M, Simony-Lafontaine J, Louason G, Speiser P, Zeillinger R, Rodriguez C, Theillet C. Mapping of DNA amplifications at 15 chromosomal localizations in 1875 breast tumors: Definition of phenotypic groups. Cancer Res 1997; 57: 4360–4367.

100. Deng N, Goh LK, Wang H, Das K, Tao J, Tan IB, Zhang S, Lee M, Wu J, Lim KH, Lei Z, Goh G, Lim QY, Tan AL, Sin Poh DY, Riahi S, Bell S, Shi MM, Linnaartz R, Zhu F, Yeoh KG, Toh HC, Yong WP, Cheong HC, Rha SY, Boussioutas A, Grabsch H, Rozen S, Tan P. A comprehensive survey of genomic alterations in gastric cancer reveals systematic patterns of molecular exclusivity and co-occurrence among distinct therapeutic targets. Gut 2012; 61: 673–684.

101. Turner N, Lambros MB, Horlings HM, Pearson A, Sharpe R, Natrajan R, Geyer FC, van Kouwenhove M, Kreike B, Mackay A, Ashworth A, van de Vijver MJ, Reis-Filho JS. Integrative molecular profiling of triple negative breast cancers identifies amplicon drivers and potential therapeutic targets. Oncogene 2010; 29: 2013–2023.

102. Holland DG, Burleigh A, Git A, Goldgraben MA, Perez-Mancera PA, Chin SF, Hurtado A, Bruna A, Ali HR, Greenwood W, Dunning MJ, Samarajiwa S, Menon S, Rueda OM, Lynch AG, McKinney S, Ellis IO, Eaves CJ, Carroll JS, Curtis C, Aparicio S, Caldas C. ZNF703 is a common Luminal B breast cancer oncogene that differentially regulates luminal and basal progenitors in human mammary epithelium. EMBO Mol Med 2011; 3: 167–180.

103. Katoh M, Igarashi M, Fukuda H, Nakagama H, Katoh M. Cancer genetics and genomics of human FOX family genes. Cancer Lett 2013; 328: 198–206.

104. Jackson CC, Medeiros LJ, Miranda RN. 8p11 myeloproliferative syndrome: A review. Hum Pathol 2010; 41: 461–476.

105. Liu J, Guzman MA, Pezanowski D, Patel D, Hauptman J, Keisling M, Hou SJ, Papenhausen PR, Pascasio JM, Punnett HH, Halligan GE, de Chadarevian JP. FOXO1-FGFR1 fusion and amplification in a solid variant of alveolar rhabdomyosarcoma. Mod Pathol 2011; 24: 1327–1335.

106. Grand EK, Chase AJ, Heath C, Rahemtulla A, Cross NC. Targeting FGFR3 in multiple myeloma: Inhibition of t(4;14)-positive cells by SU5402 and PD173074. Leukemia 2004; 18: 962–966.

107. Yagasaki F, Wakao D, Yokoyama Y, Uchida Y, Murohashi I, Kayano H, Taniwaki M, Matsuda A, Bessho M. Fusion of ETV6 to fibroblast growth factor receptor 3 in peripheral T-cell lymphoma with a t(4;12)(p16;p13) chromosomal translocation. Cancer Res 2001; 61: 8371–8374.

108. Katoh M, Katoh M. Recombination cluster around FGFR2-WDR11-HTPAPL locus on human chromosome 10q26. Int J Mol Med 2003; 11: 579–583.

18 • KATOH AND NAKAGAMA

109. Singh D, Chan JM, Zoppoli P, Niola F, Sullivan R, Castano A, Liu EM, Reichel J, Porrati P, Pellegatta S, Qiu K, Gao Z, Ceccarelli M, Riccardi R, Brat DJ, Guha A, Aldape K, Golfinos JG, Zagzag D, Mikkelsen T, Finocchiaro G, Lasorella A, Rabadan R, Iavarone A. Transforming fusions of FGFR and TACC genes in human glioblastoma. Science 2012;337:1231–1235.

110. Reardon W, Winter RM, Rutland P, Pulleyn LJ, Jones BM, Malcolm S. Mutations in the FGFR2 gene cause Crouzon syndrome. Nat Genet 1994;8:98–103.

111. Jabs EW, Li X, Scott AF, Meyers G, Chen W, Eccles M, Mao JI, Charnas LR, Jackson CE, Jaye M. Jackson-Weiss and Crouzon syndromes are allelic with mutations in FGFR2. Nat Genet 1994;8:275–279.

112. Wilkie AO, Slaney SF, Oldridge M, Poole MD, Ashworth GJ, Hockley AD, Hayward RD, David DJ, Pulleyn LJ, Rutland P, Malcolm S, Winter RM, Reardon W. Apert syndrome results from localized mutations of FGFR2 and is allelic with Crouzon syndrome. Nat Genet 1995;9:165–172.

113. Rutland P, Pulleyn LJ, Reardon W, Baraitser M, Hayward R, Jones B, Malcolm S, Winter RM, Oldridge M, Slaney SF, Poole MD, Wilkie AOM. Identical mutations in the FGFR2 gene cause both Pfeiffer and Crouzon syndrome phenotypes. Nat Genet 1995;9:173–176.

114. Przylepa KA, Paznekas W, Zhang M, Golabi M, Bias W, Bamshad MJ, Carey JC, Hall BD, Stevenson R, Orlow S, Cohen MM, Jr., Jabs EW. FGFR2 mutations in Beare-Stevenson cutis gyrata syndrome. Nat Genet 1996;13:492–494.

115. Dutt A, Salvesen HB, Chen TH, Ramos AH, Onofrio RC, Hatton C, Nicoletti R, Winckler W, Grewal R, Hanna M, Wyhs N, Ziaugra L, Richter DJ, Trovik J, Engelsen IB, Stefansson IM, Fennell T, Cibulskis K, Zody MC, Akslen LA, Gabriel S, Wong KK, Sellers WR, Meyerson M, Greulich H. Drug-sensitive FGFR2 mutations in endometrial carcinoma. Proc Natl Acad Sci USA 2008;105:8713–8717.

116. Byron SA, Gartside M, Powell MA, Wellens CL, Gao F, Mutch DG, Goodfellow PJ, Pollock PM. FGFR2 point mutations in 466 endometrioid endometrial tumors: Relationship with MSI, KRAS, PIK3CA, CTNNB1 mutations and clinicopathological features. PLoS One 2012;7:e30801.

117. Gartside MG, Chen H, Ibrahim OA, Byron SA, Curtis AV, Wellens CL, Bengston A, Yudt LM, Eliseenkova AV, Ma J, Curtin JA, Hyder P, Harper UL, Riedesel E, Mann GJ, Trent JM, Bastian BC, Meltzer PS, Mohammadi M, Pollock PM. Loss-of-function FGFR2 mutations in melanoma. Mol Cancer Res 2009;7:41–54.

118. Hernández S, López-Knowles E, Lloreta J, Kogevinas M, Amorós A, Tardón A, Carrato A, Serra C, Malats N, Real FX. Prospective study of FGFR3 mutations as a prognostic factor in nonmuscle invasive urothelial bladder carcinomas. J Clin Oncol 2006;24:3664–3671.

119. Taylor JG VI, Cheuk AT, Tsang PS, Chung JY, Song YK, Desai K, Yu Y, Chen QR, Shah K, Youngblood V, Fang J, Kim SY, Yeung C, Helman LJ, Mendoza A, Ngo V, Staudt LM, Wei JS, Khanna C, Catchpoole D, Qualman SJ, Hewitt SM, Merlino G, Chanock SJ, Khan J. Identification of FGFR4-activating mutations in human rhabdomyosarcomas that promote metastasis in xenotransplanted models. J Clin Invest 2009;119:3395–3407.

120. Katoh M. FGFR2 abnormalities underlie a spectrum of bone, skin, and cancer pathologies. J Invest Dermatol 2009;129:1861–1867.

121. Melville H, Wang Y, Taub PJ, Jabs EW. Genetic basis of potential therapeutic strategies for craniosynostosis. Am J Med Genet A 2010;152A:3007–3015.

122. Katoh Y, Katoh M. FGFR2-related pathogenesis and FGFR2-targeted therapeutics. Int J Mol Med 2009;23:307–311.

123. Gatius S, Velasco A, Azueta A, Santacana M, Pallares J, Valls J, Dolcet X, Prat J, Matias-Guiu X. FGFR2 alterations in endometrial carcinoma. Mod Pathol 2011;24:1500–1510.

124. Ota S, Zhou ZQ, Link JM, Hurlin PJ. The role of senescence and prosurvival signaling in controlling the oncogenic activity of FGFR2 mutants associated with cancer and birth defects. Hum Mol Genet 2009;18:2609–2621.

125. Korc M, Friesel RE. The role of fibroblast growth factors in tumor growth. Curr Cancer Drug Targets 2009;9:639–651.


Medicinal Research Reviews DOI 10.1002/med

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 19

126. Nilsson EM, Brokken LJ, Härkönen PL. Fibroblast growth factor 8 increases breast cancer cell growth by promoting cell cycle progression and by protecting against cell death. Exp Cell Res 2010;316:800–812.

127. Heinze C, Sutterlüty H, Grusch M, Grasl-Kraupp B, Berger W, Marian B. Targeting fibroblast-growth-factor-receptor-dependent signaling for cancer therapy. Expert Opin Ther Targets 2011;15:829–846.

128. Cuevas P, Carceller F, Angulo J, González-Corrochano R, Cuevas-Bourdier A, Giménez-Gallego G. Antiglioma effects of a new, low molecular mass, inhibitor of fibroblast growth factor. Neurosci Lett 2011;491:1–7.

129. Dimitroff CJ, Klohs W, Sharma A, Pera P, Driscoll D, Veith J, Steinkampf R, Schroeder M, Klutchko S, Sumlin A, Henderson B, Dougherty TJ, Bernacki RJ. Anti-angiogenic activity of selected receptor tyrosine kinase inhibitors, PD166285 and PD173074: Implications for combination treatment with photodynamic therapy. Invest New Drugs 1999;17:121–135.

130. Trudel S, Li ZH, Wei E, Wiesmann M, Chang H, Chen C, Reece D, Heise C, Stewart AK. CHIR-258, a novel, multitargeted tyrosine kinase inhibitor for the potential treatment of t(4;14) multiple myeloma. Blood 2005;105:2941–2948.

131. Chase A, Grand FH, Cross NC. Activity of TKI 258 against primary cells and cell lines with FGFR 1 fusion genes associated with the 8p11 myeloproliferative syndrome. Blood 2007;110:3729–3734.

132. Konecny GE, Kolarova T, O'Brien NA, Winterhoff B, Yang G, Qi J, Qi Z, Venkatesan N, Ayala R, Luo T, Finn RS, Kristof J, Galderisi C, Graus Porta D, Anderson L, Shi MM, Yovine A, Slamon DJ. Activity of the fibroblast growth factor receptor inhibitors dovitinib (TKI 258) and NVP-BGJ 398 in human endometrial cancer cells. Mol Cancer Ther 2013;12:632–642.

133. Shimizu T, Fujiwara Y, Osawa T, Sakai T, Kubo K, Kubo K, Nishitoba T, Kimura K, Senga T, Murooka H, Iwai A, Fukushima K, Yoshino T, Miwa A. Orally active anti-proliferation agents: Novel diphenylamine derivatives as FGF-R2 autophosphorylation inhibitors. Bioorg Med Chem Lett 2004;14:875–879.

134. Nakamura K, Yashiro M, Matsuoka T, Tendo M, Shimizu T, Miwa A, Hirakawa K. A novel molecular targeting compound as K-samII/FGF-R2 phosphorylation inhibitor, Ki 23057, for Scirrhous gastric cancer. Gastroenterology 2006;131:1530–1541.

135. Matsui J, Yamamoto Y, Funahashi Y, Tsuruoka A, Watanabe T, Wakabayashi T, Uenaka T, Asada M. E7080, a novel inhibitor that targets multiple kinases, has potent antitumor activities against stem cell factor producing human small cell lung cancer H146, based on angiogenesis inhibition. Int J Cancer 2008;122:664–671.

136. Huynh H, Ngo VC, Fargnoli J, Ayers M, Soo KC, Koong HN, Thng CH, Ong HS, Chung A, Chow P, Pollock P, Byron S, Tran E. Brivanib alaninate, a dual inhibitor of vascular endothelial growth factor receptor and fibroblast growth factor receptor tyrosine kinases, induces growth inhibition in mouse models of human hepatocellular carcinoma. Clin Cancer Res 2008;14:6146–6153.

137. Hilberg F, Roth GJ, Krssak M, Kautschitsch S, Sommergruber W, Tontsch-Grunt U, Garin-Chesa P, Bader G, Zoephel A, Quant J, Heckel A, Rettig WJ. BIBF 1120: Triple angiokinase inhibitor with sustained receptor blockade and good antitumor efficacy. Cancer Res 2008;68:4774–4782.

138. O'Hare T, Shakespeare WC, Zhu X, Eide CA, Rivera VM, Wang F, Adrian LT, Zhou T, Huang WS, Xu Q, Metcalf CA, 3rd, Tyner JW, Loriaux MM, Corbin AS, Wardwell S, Ning Y, Keats JA, Wang Y, Sundaramoorthi R, Thomas M, Zhou D, Snodgrass J, Commodore L, Sawyer TK, Dalgarno DC, Deininger MW, Druker BJ, Clackson T. AP 24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance. Cancer Cell 2009;16:401–412.

139. Gozgit JM, Wong MJ, Moran L, Wardwell S, Mohemmad QK, Narasimhan NI, Shakespeare WC, Wang F, Clackson T, Rivera VM. Ponatinib (AP 24534), a multitargeted pan-FGFR inhibitor with activity in multiple FGFR-amplified or mutated cancer models. Mol Cancer Ther 2012;11:690–699.

140. Pan BS, Chan GK, Chenard M, Chi A, Davis LJ, Deshmukh SV, Gibbs JB, Gil S, Hang G, Hatch H, Jewell JP, Kariv I, Katz JD, Kunii K, Lu W, Lutterbach BA, Paweletz CP, Qu X, Reilly JF,

KATO AND NAKAGAMA

Szewczak AA, Zeng Q, Kohl NE, Dinsmore CJ. MK-2461, a novel multitargeted kinase inhibitor, preferentially inhibits the activated c-Met receptor. Cancer Res 2010; 70:1524–1533.

141. Bello E, Colella G, Scarlato V, Oliva P, Berndt A, Valbusa G, Serra SC, D'Incalci M, Cavalletti E, Giavazzi R, Damia G, Camboni G. E-3810 is a potent dual inhibitor of VEGFR and FGFR that exerts antitumor activity in multiple preclinical models. Cancer Res 2011; 71:1396–1405.

142. Gavine PR, Mooney L, Kilgour E, Thomas AP, Al-Kadhimi K, Beck S, Rooney C, Coleman T, Baker D, Mellor MJ, Brooks AN, Klinowska T. AZD4547: An orally bioavailable, potent and selective inhibitor of the FGFR tyrosine kinase family. Cancer Res 2012; 72:2045–2056.

143. Lin N, Chen S, Pan W, Xu L, Hu K, Xu R. NP 603, a novel and potent inhibitor of FGFR 1 tyrosine kinase, inhibits hepatic stellate cell proliferation and ameliorates hepatic fibrosis in rats. Am J Physiol Cell Physiol 2011; 301:C469–C477.

144. Lallemand B, Chaix F, Bury M, Bruyère C, Ghostin J, Becker JP, Delporte C, Gelbcke M, Mathieu V, Dubois J, Prévost M, Jabin I, Kiss R. N-(2-{3-[3,5-bis(trifluoromethyl)phenyl]ureido}ethyl)-glycyrrhetinamide (6b): A novel anticancer glycyrrhetic acid derivative that targets the proteasome and displays anti-kinase activity. J Med Chem 2011; 54:6501–6513.

145. Yashiro M, Shinto O, Nakamura K, Tendo M, Matsuoka T, Matsuzaki T, Kaizaki R, Miwa A, Hirakawa K. Synergistic antitumor effects of FGFR 2 inhibitor with 5-fluorouracil on scirrhou gastric carcinoma. Int J Cancer 2010; 126:1004–1016.

146. Qiu H, Yashiro M, Zhang X, Miwa A, Hirakawa K. A FGFR 2 inhibitor, Ki23057, enhances the chemosensitivity of drug-resistant gastric cancer cells. Cancer Lett 2011; 307:47–52.

147. Lieu C, Heymach J, Overman M, Tran H, Kopetz S. Beyond VEGF: Inhibition of the fibroblast growth factor pathway and antiangiogenesis. Clin Cancer Res 2011; 17:6130–6139.

148. Desnoyers LR, Pai R, Ferrando RE, Hötzl K, Le T, Ross J, Carano R, D'Souza A, Qing J, Mohtashemi I, Ashkenazi A, French DM. Targeting FGF 19 inhibits tumor growth in colon cancer xenograft and FGF 19 transgenic hepatocellular carcinoma models. Oncogene 2008; 27:85–97.

149. Qing J, Du X, Chen Y, Chan P, Li H, Wu P, Marsters S, Stawicki S, Tien J, Totpal K, Ross S, Stinson S, Dornan D, French D, Wang QR, Stephan JP, Wu Y, Wiesmann C, Ashkenazi A. Antibody-based targeting of FGFR 3 in bladder carcinoma and t(4;14)-positive multiple myeloma in mice. J Clin Invest 2009; 119:1216–1229.

150. Zhao WM, Wang L, Park H, Chhim S, Tanphanich M, Yashiro M, Kim KJ. Monoclonal antibodies to FGFR 2 effectively inhibit growth of gastric tumor xenografts. Clin Cancer Res 2010; 16:5750–5758.

151. Liang Z, Wu H, Reddy S, Zhu A, Wang S, Blevins D, Yoon Y, Zhang Y, Shim H. Blockade of invasion and metastasis of breast cancer cells via targeting CXCR 4 with an artificial microRNA. Biochem Biophys Res Commun 2007; 363:542–546.

152. Katoh M. RNA technology targeted to the WNT signaling pathway. Cancer Biol Ther 2008; 7:275–277.

153. Marquez RT, McCaffrey AP. Advances in microRNAs: Implication for gene therapists. Hum Gene Ther 2008; 19:27–38.

154. Grosshans H, Filipowicz W. The expanding world of small RNAs. Nature 2008; 451:414–416.

155. Ajeawung NF, Li B, Kamnasaran D. Translational applications of microRNA genes in medulloblastomas. Clin Invest Med 2010; 33:E223–E233.

156. Ichimura A, Ruike Y, Terasawa K, Tsujimoto G. miRNAs and regulation of cell signaling. FEBS J 2011; 278:1610–1618.

157. Han Y, Chen J, Zhao X, Liang C, Wang Y, Sun L, Jiang Z, Zhang Z, Yang R, Chen J, Li Z, Tang A, Li X, Ye J, Guan Z, Gui Y, Cai Z. MicroRNA expression signatures of bladder cancer revealed by deep sequencing. PLoS One 2011; 6:e18286.

158. Li X, Luo F, Li Q, Xu M, Feng D, Zhang G, Wu W. Identification of new aberrantly expressed miRNAs in intestinal-type gastric cancer and its clinical significance. Oncol Rep 2011; 26:1431–1439.

159. Roldan M, Macias-Gonzalez M, Garcia R, Tinahones FJ, Martin M. Obesity short-circuits stemness gene network in human adipose multipotent stem cells. FASEB J 2011; 25:4111–4126.

CANCER BIOLOGY AND THERAPEUTICS ON FGF RECEPTORS • 21

160. Meng F, Glaser SS, Francis H, DeMorrow S, Han Y, Passarini JD, Stokes A, Cleary JP, Liu X, Venter J, Kumar P, Priester S, Hubble L, Staloch D, Sharma J, Liu CG, Alpini G. Functional analysis of microRNAs in human hepatocellular cancer stem cells. J Cell Mol Med 2012; 16:160–173.

161. Wang G, van der Walt JM, Mayhew G, Li YJ, Züchner S, Scott WK, Martin ER, Vance JM. Variation in the miRNA-433 binding site of *FGF20* confers risk for Parkinson disease by overexpression of α-synuclein. Am J Hum Genet 2008; 82:283–289.

162. Xu N, Brodin P, Wei T, Meisgen F, Eidsmo L, Nagy N, Kemeny L, Ståhle M, Sonkoly E, Pivarcsi A. MiR-125b, a microRNA downregulated in psoriasis, modulates keratinocyte proliferation by targeting FGFR2. J Invest Dermatol 2011; 131:1521–1529.

163. Thum T, Gross C, Fiedler J, Fischer T, Kissler S, Bussen M, Galuppo P, Just S, Rottbauer W, Frantz S, Castoldi M, Soutschek J, Koteliansky V, Rosenwald A, Basson MA, Licht JD, Pena JT, Rouhanifard SH, Muckenthaler MU, Tuschl T, Martin GR, Bauersachs J, Engelhardt S. MicroRNA-21 contributes to myocardial disease by stimulating MAP kinase signalling in fibroblasts. Nature 2008; 456:980–984.

164. Zhao Q, Kirkness EF, Caballero OL, Galante PA, Parmigiani RB, Edsall L, Kuan S, Ye Z, Levy S, Vasconcelos AT, Ren B, de Souza SJ, Camargo AA, Simpson AJ, Strausberg RL. Systematic detection of putative tumor suppressor genes through the combined use of exome and transcriptome sequencing. Genome Biol 2010; 11:R114.

165. Holbrook JD, Parker JS, Gallagher KT, Halsey WS, Hughes AM, Weigman VJ, Lebowitz PF, Kumar R. Deep sequencing of gastric carcinoma reveals somatic mutations relevant to personalized medicine. J Transl Med 2011; 9:119.

166. Zang ZJ, Cutcutache I, Poon SL, Zhang SL, McPherson JR, Tao J, Rajasegaran V, Heng HL, Deng N, Gan A, Lim KH, Ong CK, Huang D, Chin SY, Tan IB, Ng CC, Yu W, Wu Y, Lee M, Wu J, Poh D, Wan WK, Rha SY, So J, Salto-Tellez M, Yeoh KG, Wong WK, Zhu YJ, Futreal PA, Pang B, Ruan Y, Hillmer AM, Bertrand D, Nagarajan N, Rozen S, Teh BT, Tan P. Exome sequencing of gastric adenocarcinoma identifies recurrent somatic mutations in cell adhesion and chromatin remodeling genes. Nat Genet 2012; 44:570–574.

167. Yu M, Ting DT, Stott SL, Wittner BS, Ozsolak F, Paul S, Ciciliano JC, Smas ME, Winokur D, Gilman AJ, Ulman MJ, Xega K, Contino G, Alagesan B, Brannigan BW, Milos PM, Ryan DP, Sequist LV, Bardeesy N, Ramaswamy S, Toner M, Maheswaran S, Haber DA. RNA sequencing of pancreatic circulating tumour cells implicates WNT signalling in metastasis. Nature 2012; 487: 510–513.

---

Masaru Katoh obtained M.D. and Ph.D. degrees from the University of Tokyo in 1986 and 1993, respectively, and then joined the Laboratory of Dr. Harold Varmus. In 1998, Masaru Katoh was appointed to the Chief of Genetics and Cell Biology Section at National Cancer Center, Japan, and carried out (1) WNTome project to comprehensively clone (identify) and characterize human genes encoding WNT signaling molecules during 1998–2002, (2) post-WNTome project to identify and characterize novel human genes encoding adhesion molecules, transcription factors, epigenetic regulators, etc. during 2003–2006, and (3) stem cell signaling network project to elucidate the interaction among WNT, FGF, Notch, Hedgehog, and TGFβ/BMP signaling cascades during 2007–present. Masaru Katoh published more than 200 manuscripts as the corresponding author, which have been cited more than 3000 times by others in the Web of Science (WoS) database of Thomson Reuters. Masaru Katoh is an Academic Editor of the “PLoS ONE” journal.

Hitoshi Nakagama obtained M.D. degree from the University of Tokyo in 1982. Hitoshi Nakagama was appointed to the Chief of Biochemistry Division at National Cancer Center, Japan, in 2001, and then promoted to the Director of National Cancer Center Research Institute in 2011. Hitoshi Nakagama is an editor of the “Cancer Science” journal.
