
# Fortifying the barrier: the impact of lipid A remodelling on bacterial pathogenesis

**Brittany D. Needham**${}^{1}$ **and M. Stephen Trent**${}^{1,2}$

**Abstract |** Gram-negative bacteria decorate their outermost surface structure, lipopolysaccharide, with elaborate chemical moieties, which effectively disguises them from immune surveillance and protects them from the onslaught of host defences. Many of these changes occur on the lipid A moiety of lipopolysaccharide, a component that is crucial for host recognition of Gram-negative infection. In this Review, we describe the regulatory mechanisms controlling lipid A modification and discuss the impact of modifications on pathogenesis, bacterial physiology and bacterial interactions with the host immune system.

The bacterial cell envelope is a complex structure that protects the cell from the surrounding environment. A defining feature of Gram-negative bacteria is the presence of an outer membrane, which is an asymmetrical bilayer with glycerophospholipids confined to the inner leaflet and lipopolysaccharide (LPS) anchored to the outer leaflet${}^{1}$ (FIG. 1a,b). Similarly to most cell envelope components, LPS is made at the cytoplasmic face of the inner membrane and must be transported across the two bilayers and the periplasm to become integrated in the outer membrane${}^{1}$.

LPS is composed of three domains: the lipid A hydrophobic anchor, the core oligosaccharide and the O antigen${}^{1}$ (FIG. 1b). Some organisms (for example, *Neisseria* spp.) produce lipo-oligosaccharide (LOS), in which the repeating O antigen domain is absent and is replaced by an extended core region${}^{2}$. Lipid A, the endotoxic portion of LPS and the site for many LPS modifications, is initially synthesized as a β-1′,6-linked disaccharide of glucosamine that is both phosphorylated and fatty acylated (FIG. 1c). In some organisms, such as *Escherichia coli* K12, this structure represents the typical form of lipid A in the outer membrane. Despite initial studies reporting that lipid A could be modified with polar substituents (such as amino sugars${}^{3}$), it was nevertheless viewed as a static structure. This view changed in the late 1990s following the characterization of the lipid A biosynthesis pathway, which established a foundation for studies which discovered that lipid A is altered after synthesis${}^{4–6}$. In fact, Gram-negative organisms have evolved several LPS modification strategies that allow these organisms to adapt to their unpredictable and often hostile surroundings${}^{7}$. During and after trafficking to the cell surface, lipid A can undergo extensive remodelling, resulting in the wide variety of lipid A structures that are observed across species (FIG. 1d). This is accomplished by diverse lipid A modification enzymes that remove or add acyl chains and phosphate groups, as well as other enzymes that transfer various constituents onto the molecule, such as sugars, phosphoryl containing groups and even an amino acid, all of which can influence bacterial interactions with the host. The term lipid A modification is used throughout this Review to refer to biological modification of lipid A in the bacterial cell as well as artificial modification *in vitro*. Modifications also occur on other moieties of LPS (BOX 1), but these non-lipid A modifications are not the focus of this Review.

Lipid A alterations directly affect pathogenesis by changing outer-membrane permeability, promoting resistance to antimicrobial peptides and interfering with the ability of the host to recognize LPS as a conserved microorganism-associated molecular pattern (MAMP)${}^{7}$. The diversity of LPS modification systems is quite extraordinary, and the accumulated knowledge about these systems has provided deeper insight into bacterial mechanisms that contribute to human disease and immunity. In this Review, we describe the mechanisms that regulate lipid A remodelling, the host defence systems that recognize this molecule and the strategies that bacteria have evolved to use lipid A modifications for their own benefit. Finally, we discuss the interplay between lipid A modification systems and bacterial physiology.


**Microorganism-associated molecular pattern (MAMP).** A component of a commensal or pathogenic microorganism that is well conserved and universally recognized by the innate immune system.

**Lipopolysaccharide** is a typical MAMP, and other examples include peptidoglycan, lipoproteins and flagella. Previously referred to as PAMPs (pathogen-associated molecular patterns).


${}^{1}$The Institute of Cellular and Molecular Biology, The University of Texas at Austin.

${}^{2}$Section of Molecular Genetics and Microbiology, The University of Texas at Austin, Austin, Texas 78712, USA.

Correspondence to M.S.T.  
e-mail:  
strent@mail.utexas.edu  

doi:10.1038/nrmicro3047  
Published online 10 June 2013

REVIEW S

a

b

OM

Phospholipid

LPS

O antigen

Core

Lipid A

PG

IM

Lipoprotein

Cytoplasm

IM protein

c

Kdo

HO

HO

OH

HO

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH

OH


Box 1 | Non-lipid A modifications to lipopolysaccharide

Although most lipopolysaccharide (LPS) modifications that are known to affect pathogenesis occur on the lipid A portion of the molecule, the remaining sugars of the LPS molecule can also be modified. Extending from lipid A is the core oligosaccharide, which is subdivided into an inner and an outer core; the O antigen is attached to the outer core (FIG. 1b). Biosynthesis of these LPS domains varies more across Gram-negative organisms than biosynthesis of lipid A, but within a genus, some components are relatively well conserved. The inner core typically consists of Kdo (3-deoxy-D-manno-octulosonic acid) and heptose sugars, whereas the outer core varies in sugar composition, sugar arrangement and linkage to O antigen. O antigen, in addition to a tremendous variety in composition, can have strikingly different lengths, ranging from the complete absence of O antigen to more than 100 repeating units of sugar backbones with branching chains ${ }^{1,109}$. These different variants of LPS, particularly the truncation of O antigen, affect the ability of the bacterium to withstand harsh environments ${ }^{12,109,110}$. Various constituents, such as additional sugars, phosphate groups, phosphoethanol-amine groups and phosphorylcholine groups, can modify the LPS core regions. Similarly, known O antigen modifications include glycosylation ${ }^{111}$, acetylation ${ }^{112}$, addition of phosphoryl constituents ${ }^{1}$ and ligation of acidic repeats such as colanic and sialic ${ }^{114}$ acids.

In the context of cationic antimicrobial peptide (CAMP) resistance and Toll-like receptor 4 (TLR4) evasion, modifications to the LPS core seem to have more modest roles in pathogenesis than do variations in O antigen length and lipid A modifications, although core modifications can enhance resistance to some CAMPs and to components of the innate immune system, such as complement. A good example of how non-lipid A modifications can affect pathogenesis can be seen in Neisseria gonorrhoeae. This bacterium protects itself from the complement system in human serum by modifying lipooligosaccharide (LOS; LPS in which O antigen is absent and the core region is extended) through the addition of sialic acid and glucose, thus increasing the affinity of LOS for the complement-inhibitory proteins factor H and complement component 4b (C4b)-binding protein, respectively ${ }^{114,115}$. Furthermore, $N$. gonorrhoeae producing LOS with a terminal sugar of galactose has an increased association with dendritic cells compared to $N$. gonorrhoeae that produces LOS containing terminal N-acetylglucosamine. This association with dendritic cells leads to altered cytokine production and T cell responses that could decrease immune responses against the bacterium ${ }^{116}$.


Regulation of lipid A remodelling

Lipid A modifications are often necessary only for a portion of the bacterial life cycle, such as host colonization, and as a consequence the enzymes responsible for the modifications are subjected to both transcriptional and post-translational regulation. Many modification enzymes are embedded in the outer membrane in close proximity to lipid A, necessitating tight control for selective enzyme activity, whereas other enzymes are constitutively active regardless of their localization. Two-component systems, small RNAs (sRNAs), peptide feedback loops and substrate availability are all involved in directing the activity of these enzymes (FIG. 2).


Transcriptional control. Two-component systems are typically composed of a sensor kinase and a response regulator; the sensor kinase is autophosphorylated on stimulation and transfers its phosphate group to the response regulator, which then serves as a transcription factor. To date, various two-component systems have been implicated in the regulation of lipid A modification enzymes, including the widespread PhoPQ and PmrAB (also known as BasRS) systems, and the ParRS and CprRS systems in Pseudomonas aeruginosa ${ }^{8-13}$. Other systems, such as EvgAS, RcsBC and BvgAS, have been linked to the activity of the PhoPQ system or to


Small RNAs

Short, non-coding RNA molecules that can regulate gene expression by interacting with mRNA or can bind protein targets to modify their activity.

REVIEW S

a Transcriptional regulation
b Post-translational regulation

Acylation
Deacylation
Acylation
Deacylation

Outer membrane
PagP
PagL
L-Ara4N
Periplasm
PhoQ
PmrB
ArnT
EptA
LpxT
Inner membrane
PhoP
PhoP
PmrA
PmrA
MicA
Ribosome
phoP
MgrR
eptB
Modification
enzymes
Modification
enzymes
pEtN
P
P
P
P
P
P
MicF
lpxR
PmrR
LpxR

Figure 2 | Transcriptional and post-translational regulation of lipid A modification enzymes. a | Transcriptional control of lipid A modification enzymes includes gene regulation by two-component systems such as PhoPQ, leading to acylation and deacylation of lipid A by upregulating transcription of the genes encoding the enzymes PagP and PagL, respectively. The two-component system PmrAB leads to the addition of 4-amino-4-deoxy-L-arabinose (aminoarabinose; L-Ara4N) and phosphoethanolamine (pEtN) to lipid A by upregulating transcription of the genes encoding the inner-membrane enzymes ArnT and EptA, respectively, which modify lipid A as it is transported to the outer membrane^{10,11}. Expression of EptB (another phosphoethanolamine transferase) is repressed by the small RNA (sRNA) MgrR, which is induced by PhoPQ^{30}. Translation of the phoP mRNA is repressed by the sRNA MicA^{31}, which leads to the loss of regulation by the PhoPQ system. The sRNA MicF increases degradation of the lpxR mRNA, which encodes a lipid A deacetylase.

b | Post-translational control of lipid A modification systems includes inhibition of the kinase LpxT (which phosphorylates lipid A during transport to the outer membrane) by the small peptide PmrR, which is upregulated by the PmrAB system in response to high levels of Fe³⁺ (REF. 33). Post-translational regulation is also mediated by substrate availability. Membrane perturbation can lead to the displacement of phospholipids from the inner leaflet to the outer leaflet of the outer membrane, placing these donor substrates in close proximity to the acyltransferase PagP, and thus enhancing enzyme activity. PagP cleaves the phospholipid substrate, restoring the composition of the outer membrane and increasing the integrity of the permeability barrier by further acylating lipid A^{39}. LpxR deacylates lipid A, but this activity is inhibited by the aminoarabinose lipid A modification.

The PhoPQ system has also been shown to mediate modification of lipid A through transcriptional activation of non-coding sRNAs. Many sRNAs modulate the expression of outer-membrane proteins^{27–29}, but recently the sRNA MgrR of *E. coli* was shown to regulate lipid A modification^{30} (FIG. 2a). MgrR is a transcriptional target of PhoP and is conserved in *E. coli* and certain *Citrobacter*, *Enterobacter* and *Klebsiella* spp.^{30}. This sRNA regulates various genes, including the negative regulation of *eptB*^{30}, which encodes an enzyme that transfers phosphoethanolamine to the outer Kdo (3-deoxy-d-manno-octulosonic acid) residue of LPS^{7} (FIG. 1c). Although EptB activity results in a modest increase in resistance to the cationic antimicrobial peptide (CAMP) polymyxin B, it is unlikely that this is the main function of this enzyme because conditions that inhibit EptB (through activation of the PhoPQ system) simultaneously induce the other PhoPQ-regulated genes, which confer higher CAMP resistance^{30}. In addition to regulating an sRNA, the PhoPQ system itself is controlled by another sRNA, MicA (FIG. 2a), which inhibits translation of PhoP by competitive binding to the ribosome-binding site of the *phoP* mRNA^{31}. MicF represents another example of an sRNA that interacts with lipid A modification enzymes. This sRNA binds to *lpxR* transcripts, which encode a lipid A deacetylase (FIG. 1c; TABLE 1), and increases degradation of the mRNA by exposing regions that are susceptible to RNase E, a major contributor to RNA turnover in many bacteria^{32}.

Post-translational control. In addition to transcriptional regulation, lipid A modification enzymes are subjected to post-translational control mechanisms. For example, in *S*. Typhimurium, the PmrAB system orchestrates a delayed negative feedback loop that can be activated by Fe³⁺ and allows initial uptake of the ion but sets in

Kdo
(3-deoxy-d-manno-octulosonic acid). The sugar residue that constitutes the inner core of lipopolysaccharide. This inner core links the polysaccharide chain to lipid A.

Cationic antimicrobial peptide
A type of positively charged, amphipathic peptide that associates with the negatively charged Gram-negative membrane and is thought to disrupt the membrane, leading to cell lysis and death.
motion a shift in the cell surface charge that reduces Fe³⁺ retrieval from the extracellular environment. This feedback loop is established through PmrAB-induced expression of a short peptide, PmrR, which binds to and inhibits the lipid A-modifying enzyme LpxT³³ (FIG. 2b; TABLE 1). LpxT phosphorylates lipid A (FIG. 1c) and thereby increases the overall net negative charge of the outer membrane. However, when LpxT activity is inhibited, other modifying enzymes are capable of transferring amine-containing constituents to lipid A³⁴, and the outer membrane eventually becomes less negatively charged and so has a reduced affinity for Fe³⁺ (REF. 33). This mechanism regulates lipid A-modifying enzymes such as LpxT and those encoded by genes that are transcriptionally induced by PmrAB (such as EptA and ArnT). It also prevents the intracellular accumulation of toxic Fe³⁺ and dampens PmrA-dependent transcription in a delayed manner as a result of the lowered affinity of PmrAB-activating Fe³⁺ for the membrane. Such small peptide-mediated feedback control of the systems that regulate lipid A modification might be a common theme. In fact, other small peptides, such as SafA (in *E. coli*) and MgrB (in *E. coli*, *S.* Typhimurium and *Yersinia pestis*), are inner-membrane peptides that interact with the periplasmic domain of PhoQ and activate or repress PhoQ activity, respectively³⁵,³⁶.

Other post-translational control mechanisms rely on substrate availability. Although the acyltransferase PagP is transcriptionally upregulated by PhoPQ in organisms such as *Salmonella* spp. and *E. coli*, its activation is enhanced by environmental stress and intracellular membrane stress³⁷. For example, in *E. coli*, PagP remains dormant in the outer membrane under standard growth conditions. However, treatment with membrane-perturbing agents, such as EDTA, results in displacement of the phospholipid donor substrate of PagP from the inner to the outer leaflet of the outer membrane, placing it in close proximity to the PagP catalytic domain³⁸. PagP cleaves the phospholipid substrate, restoring the composition of the outer membrane and increasing the integrity of the permeability barrier by further acylating lipid A³⁹ (FIG. 2b).

The affinity and activity of lipid A modification enzymes are also modulated by the chemical composition of lipid A. For example, in *Yersinia enterocolitica*, LpxR deacylates lipid A at 37°C, but at 21°C LpxR activity is impeded. At 21°C, *pmrAB* and the genes necessary for aminoarabinose addition are induced, and the elevated amounts of aminoarabinose residues on lipid A seem to suppress LpxR activity⁴⁰. In *S.* Typhimurium, PagL is similarly repressed by aminoarabinose modified lipid A, although this effect seems to be temperature independent²¹.

These various forms of post-translational regulation afford quick response times for modification of the outer defence barrier, as the enzymes are already present and primed to function as soon as the appropriate signal is detected. Both transcriptional and post-translational regulation work together, sometimes as functionally redundant mechanisms, allowing Gram-negative bacteria to adapt to diverse environments and thereby ensure

their survival. The diversity of regulatory mechanisms also illustrates the importance of the lipid A structure to the membrane barrier.

### Host defences that target lipid A

Considering the intimate contact that humans have with bacteria, the frequency of colonization with pathogenic Gram-negative bacteria is astonishingly low. This is largely attributed to the formidable arsenal of host defences that eliminate invading pathogens by recognizing and responding to highly conserved components of infectious agents, known as MAMPs⁴¹. Because LPS is an essential component of the Gram-negative cell surface, it serves as an effective MAMP to trigger the innate immune system¹. The host offers a nutrient-rich but perilous environment for a bacterium. For example, intestinal colonization requires a bacterium to journey through the acidic pH of the stomach and encounter toxic compounds such as bile and antimicrobials during transit⁴², and the bloodstream is swarming with LPS-binding proteins, antibodies, complement and immune cells primed to detect LPS¹,⁴³⁻⁴⁶. Every point of entry for a bacterium is well defended, but as many of the protective mechanisms rely on lipid A detection, the modification of lipid A affords the bacterium an opportunity to evade the immune system and establish an infection.

#### Charge-dependent binding of lipid A

Charge-dependent binding of various host molecules (such as CAMPs, platelet factor 4 (PF4) and members of the complement system) to the bacterial cell surface is a major contributor to host protection. CAMPs are amphipathic molecules that are present on mammalian mucosal surfaces, in bodily secretions (such as sweat and saliva) and in phagocytic cells. CAMP production is a highly conserved defence mechanism of most organisms, but the peptides vary widely in terms of their composition and structure. The genes encoding CAMPs are among the most rapidly evolving mammalian genes⁴⁷, and it has been hypothesized that this represents an example of co-evolution, wherein the genes are under selective pressure to mutate as a consequence of the evolution of bacterial resistance to the encoded proteins⁴⁸. One of the primary mechanisms proposed for CAMP-mediated bactericidal activity is association of the positively charged peptides with negatively charged lipid A, followed by CAMP insertion into the bacterial membrane and disruption of the membrane potential, leading to cell death⁴²,⁴⁸.

PF4 is another positively charged host molecule that is conserved across vertebrates and is released following platelet activation during infection. This molecule binds the bacterial cell surface at the lipid A domain. Antibodies specific for PF4–lipid A complexes are subsequently produced, leading to increased opsonization and phagocytosis of the bacterium⁴³. Furthermore, complement proteins also bind lipid A and modulate the activation of the classical complement pathway to promote bacterial clearance⁴⁴⁻⁴⁶.

#### Toll-like receptor 4 signalling

Lipid A is recognized by the Toll-like receptor 4 (TLR4)–MD2 (also known as

---

### Complement

An innate immune defence mechanism involving many proteins that function in signalling cascades and also form cell-lysing membrane attack complexes.

### Opsonization

The tagging of pathogens by molecules such as antibodies. These molecules target the foreign entity for destruction by immune system clearance mechanisms such as phagocytosis and the complement system.

**NATURE REVIEWS | MICROBIOLOGY**

© 2013 Macmillan Publishers Limited. All rights reserved
Table 1 | Modifications to the Kdo-lipid A domain of lipopolysaccharide

| Enzyme* | Enzyme localization | Active-site topology | Pathogenic organisms† | Activity | Contributors to regulation | Effect of modification | Refs |
| --- | --- | --- | --- | --- | --- | --- | --- |
| AlmG | Inner membrane | Cytoplasmic | V.cholerae | Transfer of glycine to the hydroxyl group of the 3'-acyloxyacyl chain of lipid A | Unknown | CAMP resistance | 75 |
| ArnT (PmrK) | Inner membrane | Periplasmic | E.coli, P.mirabilis, S.Typhimurium, S.flexneri, P.aeruginosa, B.cepecia, Y.enterocolitica, Y.pestis, Y.pseudotuberculosis, K.pneumoniae, B.pertussis and B.bronchiseptica | Addition of aminoarabinose to lipid A | PmrAB, as well as ParRS and CprRS in P.aeruginosa | CAMP resistance | 7,14, 125–128 |
| EptA (LptA and PmrC) | Inner membrane | Periplasmic | E.coli, V.cholerae, H.pylori, S.Typhimurium, S.flexneri, Neisseria gonorrhoeae and Neisseria meningitidis | Addition of phospho-ethanolamine to lipid A | PmrAB | CAMP resistance | 7,125, 129,130 |
| EptB | Inner membrane | Periplasmic | E.coli, S.Typhimurium and Y.pestis | Transfer of phospho-ethanolamine to Kdo | PhoPQ and the sRNA MgrR | Modest CAMP resistance | 125,131, 132 |
| EptC | Inner membrane | Periplasmic | C.jejuni | Transfer of phospho-ethanolamine to lipid A, the flagellar rod and other substituents | Present under normal laboratory conditions | CAMP resistance and motility | 91,133 |
| FlmF1 | Inner membrane | Periplasmic | F.tularensis | Addition of glucose and mannose to lipid A | Present under normal laboratory conditions | Possible role in CAMP resistance | 134 |
| FlmF2 | Inner membrane | Periplasmic | F.tularensis | Addition of galactosamine to lipid A | Present under normal laboratory conditions | Possible role in CAMP resistance | 134,135 |
| FlmK | Inner membrane | Periplasmic | F.tularensis | Addition of glucose, mannose or galactosamine to lipid A | Present under normal laboratory conditions | TLR4 evasion and a possible role in CAMP resistance | 135 |
| KdkA | Inner membrane | Cytoplasmic | H.influenzae, V.cholerae, P.multocida, B.pertussis, A.actinomycetemcomitans and S.putrefaciens | Phosphorylation of Kdo | Present under normal laboratory conditions | Possible effect on toxin delivery | 125,136 |
| KdoH1 and KdoH2 (KdhA) | Inner membrane | Cytoplasmic | H.pylori, F.tularensis and L.pneumophila | Removal of outer Kdo region | Present under normal laboratory conditions | CAMP resistance | 137–139 |
| KdoO | Inner membrane | Cytoplasmic | B.cepecia, B.ambifaria, Y.pestis and A.haemolyticus | Hydroxylation of Kdo | Present under normal laboratory conditions | Unknown | 140 |
| LmtA | Inner membrane | Cytoplasmic | L.interrogans | Methylation of lipid A | Present under normal laboratory conditions | Unknown | 141 |
| LpxD2 | Inner membrane | Cytoplasmic | F.tularensis | Addition of shorter (C₁₆) primary acyl chains to lipid A at low temperatures | Low temperatures | Increased membrane fluidity | 72 |
| LpxE | Inner membrane | Periplasmic | F.tularensis, H.pylori and P.gingivalis | Dephosphorylation of lipid A | Present under normal laboratory conditions | CAMP resistance and TLR4 evasion | 7,125, 142 |
| LpxF | Inner membrane | Periplasmic | F.tularensis, H.pylori, P.gingivalis and L.interrogans | Dephosphorylation of lipid A | Present under normal laboratory conditions | CAMP resistance and TLR4 evasion | 7,63, 125,143, 144 |
| LpxO | Inner membrane | Cytoplasmic | S.Typhimurium, K.pneumoniae, P.aeruginosa, B.bronchiseptica and L.pneumophila | Hydroxylation of lipid A acyl chains | Present under normal laboratory conditions | Coordination of stress responses | 7,125, 145 |
| LpxP | Inner membrane | Cytoplasmic | E.coli, S.Typhimurium, Y.enterocolitica, Y.pestis, and L.pneumophila | Alternative biosynthetic acyltransferase at low temperatures | Low temperatures | Membrane integrity and fluidity | 7,146, 147 |

Table 1 (continued) | Modifications to the Kdo–lipid A domain of lipopolysaccharide

| Enzyme* | Enzyme localization | Active-site topology | Pathogenic organisms† | Activity | Contributors to regulation | Effect of modification | Refs |
| --- | --- | --- | --- | --- | --- | --- | --- |
| LpxR | Outer membrane | Extracellular | E. coli O157:H7, H. pylori, V. cholerae, S. Typhimurium, Y. enterocolitica, Y. pestis and Y. pseudotuberculosis | Deacylation of lipid A | The sRNA MicF and aminoarabinose modifications to lipid A | TLR4 evasion | 7,125, 148 |
| LpxT | Inner membrane | Periplasmic | E. coli, S. Typhimurium and Y. pestis | Phosphorylation of lipid A and recycling of undecaprenyl pyrophosphate | PmrAB and the small peptide PmrR | Unknown | 34 |
| PagL | Outer membrane | Extracellular | P. aeruginosa, B. cepacia, S. Typhimurium, B. pertussis and B. bronchiseptica | Deacylation of lipid A | PhoPQ and aminoarabinose modifications to lipid A | Decreased TLR4 activation | 7,125, 128,149 |
| PagP | Outer membrane | Extracellular | E. coli, S. Typhimurium, S. flexneri, P. aeruginosa, Y. enterocolitica, Y. pestis, Y. pseudotuberculosis, K. pneumoniae, B. pertussis, B. bronchiseptica, B. parapertussis and L. pneumophila | Acylation of lipid A | PhoPQ and membrane perturbation | Selective CAMP resistance, membrane integrity and decreased TLR4 activation | 7,125, 150–152 |

A. haemolyticus, Acinetobacter haemolyticus; A. actinomycetemcomitans, Actinobacillus actinomycetemcomitans; aminoarabinose, 4-amino-4-deoxy-L-arabinose; B. bronchiseptica, Bordetella bronchiseptica; B. parapertussis, Bordetella parapertussis; B. pertussis, Bordetella pertussis; B. ambifaria, Burkholderia ambifaria; B. cepacia, Burkholderia cepacia; CAMP, cationic antimicrobial peptide; C. jejuni, Campylobacter jejuni; E. coli, Escherichia coli; F. tularensis, Francisella tularensis; H. influenzae, Haemophilus influenzae; H. pylori, Helicobacter pylori; Kdo, 3-deoxy-D-manno-octulosonic acid; K. pneumoniae, Klebsiella pneumoniae; L. pneumophila, Legionella pneumophila; L. interrogans, Leptospira interrogans; N. gonorrhoeae, Neisseria gonorrhoeae; N. meningitidis, Neisseria meningitidis; P. multocida, Pasteurella multocida; P. gingivalis, Porphyromonas gingivalis; P. mirabilis, Proteus mirabilis; P. aeruginosa, Pseudomonas aeruginosa; S. Typhimurium, Salmonella enterica subsp. enterica serovar Typhimurium; S. putrefaciens, Shewanella putrefaciens; S. flexneri, Shigella flexneri; TLR4, Toll-like receptor 4; V. cholerae, Vibrio cholerae; Y. enterocolitica, Yersinia enterocolitica; Y. pestis, Yersinia pestis; Y. pseudotuberculosis, Yersinia pseudotuberculosis. *Alternative enzyme names in certain species are given in brackets. †Organisms containing homologues of these enzymes are excluded from the table if the respective modification has not been observed. The list of enzymes in this table is likely to expand as the Kdo–lipid A region is further characterized in a range of species.

LY96) receptor, one of many pattern recognition receptors (PRRs) of the mammalian innate immune system. This receptor is present on a wide variety of cell types, including monocytes, lymphocytes and endothelial cells¹. Binding of TLR4–MD2 to lipid A triggers a signalling cascade that leads to inflammation, cytokine production and the eventual clearance of bacteria through recruitment of effector cells, phagocytosis, cytotoxicity and activation of the complement system⁴⁹ (FIG. 3). This inflammatory response can be severe, resulting in tissue damage, organ failure and death, especially in cases of sepsis¹. Unmodified E. coli lipid A, which contains six acyl chains and two phosphate groups, is the strongest known TLR4 ligand, and lipid A modifications can weaken or abolish TLR4 signalling and change the nature of the downstream cytokine profile (see below)⁵⁰⁻⁵³.

The detection of lipid A by TLR4 begins with LPS-binding protein (LBP) and CD14 binding to LPS and the subsequent transfer of LPS to the TLR4–MD2 complex. This complex can then signal through two major pathways, which are named according to their adaptor proteins: myeloid differentiation primary response protein 88 (MYD88) and TIR domain-containing adaptor inducing IFNβ (TRIF; also known as TICAM1)⁵⁴ (FIG. 3). Severe reactions to LPS are attributed to activation of the MYD88 pathway, which induces the production of pro-inflammatory cytokines such as tumour necrosis factor (TNF), interleukin-6 (IL-6) and IL-12. The less inflammatory TRIF (or MYD88-independent) pathway occurs after endocytosis of the TLR4–MD2 receptor and is characterized by the production of interferon-β (IFNβ) and IFN-inducible proteins such as 10 kDa IFNγ-induced protein (IP10; also known as CXCL10), monocyte chemoattractant protein 1 (MCP1; also known as CCL2), RANTES (also known as CCL5) and granulocyte colony-stimulating factor (G-CSF)⁵⁵. Although it is important for mounting an optimal immune response to pathogens, the TRIF pathway does not lead to severe inflammation. Unmodified LPS from E. coli induces signalling through both pathways, but lipid A modifications can cause preferential recruitment of one adaptor protein over the other⁵⁶ (BOX 2). In vivo, bacterial lipid A modifications are also known to affect the potency of TLR4 activation, as described below⁷.

### Bacterial evasion strategies

Modification of lipid A equips Gram-negative bacteria with an ability to evade immune recognition and survive within a host. First, by changing the overall charge of the bacterial surface through the addition of chemical groups to lipid A, such as the addition of phosphoethanolamine and aminoarabinose, resistance to innate immune effectors (for example, CAMPS and complement factors) increases. Second, changes in the structure of lipid A are important for bacterial pathogenesis because they directly affect recognition by the TLR4–MD2 receptor, and the degree of lipid A acylation and phosphorylation is crucial for LPS recognition by TLR4–MD2

Pattern recognition receptors  
Receptors of the innate immune system. These receptors bind microorganism-associated molecular patterns of infecting pathogens and initiate signalling cascades which lead to inflammation, cytokine release and activation of the adaptive immune response.

Cytokine  
A signalling protein involved in the recruitment and regulation of cells that participate in the immune response.

Sepsis  
The severe and often fatal inflammatory response of the body to the overwhelming presence of infection (usually bacterial), characterized in part by organ failure.

PmrAB-induced lipid A modifications increase resistance to CAMPS, but constitutive activation of PmrAB has been shown to reduce the resistance of *E. coli* to the bile component deoxycholate⁵⁸. Furthermore, PhoPQ-mediated alterations have been shown to lower the resistance of S. Typhimurium to antibiotics in the presence of high Mg²⁺ concentrations²⁴. However, these costs of lipid A modification seem to be outweighed by a number of advantages for bacterial virulence, as exemplified by S. Typhimurium, Helicobacter pylori, Y. pestis, Francisella tularensis and Vibrio cholerae.

Salmonella enterica *subsp.* enterica serovar Typhimurium. S. Typhimurium is an intracellular pathogen that causes gastroenteritis in humans. As a long-standing model organism for studying bacterial pathogenesis, S. Typhimurium provides prime examples of complex lipid A alterations⁷ (FIG. 1d). The bacterium has a diverse lifestyle and fine-tunes the composition of lipid A in response to the surrounding environment⁷,⁵⁹. During infection, S. Typhimurium penetrates the epithelial lining of the small intestine, invades lymphoid tissue and infects host phagocytes⁶⁰. The unmodified lipid A synthesized by this organism is identical to that produced by *E. coli* K12 (FIG. 1c). However, survival is promoted in the intestinal lumen and within host cells (where the bacterium encounters CAMPs, low pH and possibly other unknown signals) by activation of the PhoPQ and PmrAB systems, leading to the addition of phosphoethanolamine and aminoarabinose by EptA and ArnT, respectively, and to acyl chain remodelling by PagP and PagL¹⁹ (FIG. 1d). Commensal and pathogenic *E. coli* strains also encode these lipid A modification enzymes, but little is known about their regulation *in vivo*. In S. Typhimurium, another enzyme, LpxO, hydroxylates lipid A (FIG. 1d) as part of a coordinated stress response⁶¹. The combination of these modifications results in a remodelled outer membrane with a reduced net negative charge and increased integrity, which enhances virulence⁹,²⁵.

Helicobacter pylori. The human stomach is the sole niche of *H. pylori*, and the organism is so well adapted to this environment that it colonizes roughly 50% of the world’s population and can persist for decades within a single host⁶². To survive chronically in the host and remain undetected, *H. pylori* uses two constitutive lipid A-mediated evasion strategies: repulsion of CAMPs (which are present at high concentrations in the gastric mucosa) and evasion of detection by TLR4 (FIG. 4a). Similarly to most Gram-negative bacteria, *H. pylori* synthesizes a hexa-acylated lipid A, but displays a tetra-acylated molecule lacking phosphate groups on the bacterial surface. This striking structural difference between the originally synthesized lipid A and the surface-exposed molecule is due to the actions of several enzymes, including dephosphorylation by LpxE and LpxF, addition of phosphoethanolamine by EptA and deacylation by LpxR (TABLE 1). These modifications confer resistance to polymyxin B as well as other biologically relevant CAMPs⁶³. The reduced acylation

(REF. 50). Bacteria can remove acyl chains and phosphate groups to evade detection by this PRR or to shift the type of cytokine response induced⁵⁷. Finally, certain modifications (such as those regulated by PhoPQ) alter the properties of the outer-membrane permeability barrier, which provides resistance to harsh pH and antibiotics, among other stresses.

There are benefits and costs associated with each modification: as bacteria adapt to protect themselves against certain assaults, this might result in the loss of protection against others. For instance, PhoPQ- and

Box 2 | Lipid A as a tool for immune system modulation

By chemically or biologically modifying the phosphate groups and acyl chains of lipid A, the therapeutic potential of this lipopolysaccharide (LPS) component can be harnessed while limiting the inflammatory effects of the molecule^{50,54,64}. In fact, a chemically detoxified mixture of monophosphorylated lipid A species (MPL) derived from Salmonella enterica subsp. enterica serovar Minnesota (the predominant species of which is 3-O-deacyl-4′-monophosphoryl lipid A; see the figure) recently became the first US Food and Drug Agency (FDA)-approved vaccine adjuvant in more than 70 years, bringing the number of FDA-approved Toll-like receptor (TLR) agonists up to three^{54,117}. We predict that TLR ligands are likely to be the future of vaccines and could benefit other areas as well, such as cancer research, gene therapy and bacterial production of pharmaceuticals.

Lipid A derivatives like MPL have been well studied for their agonistic properties in terms of immune stimulation^{54}. However, MPL is the only lipid A that has been tested in human cancer vaccine trials^{118}, and almost all studies have focused on cervical cancer, as MPL is used as the vaccine adjuvant against the oncogenic human papilloma virus (HPV)^{117}. Considering the evidence that TLR4 signalling can be biased to produce certain types of cytokine responses^{56}, further study of other modified lipid A structures, as well as chemically synthesized lipid A variants^{119}, could prove advantageous for the development of vaccine adjuvants. To facilitate such work, an Escherichia coli library was recently engineered to synthesize lipid A variants with a spectrum of endotoxicity^{120}. These lipid A variants have the potential to be used as components of whole cells, LPS or purified lipid A and might be used as vaccine adjuvants in the future.

Whole-cell vaccine strains of some organisms, such as Salmonella enterica subsp. enterica serovar Typhimurium, have been engineered to synthesize altered lipid A structures in order to reduce the toxicity of the vaccine. Heterologous expression of an antigen from Streptococcus pneumoniae in such strains provides protection against both S. Typhimurium and S. pneumoniae^{51,52}. Similarly, outer-membrane vesicles from Neisseria meningitidis producing modified lipid A are now prime candidates for vaccination against N. meningitidis^{121}, and also against other organisms for which antigens can be heterologously expressed in N. meningitidis and targeted to the vesicle lumen or surface^{122}.

---

transition between hosts coincides with a switch in the composition of lipid A^{68} (FIG. 4b). Inside the flea, Y. pestis grows at a temperature of between 21 °C and 27 °C and synthesizes a hexa-acylated lipid A similar to the highly inflammatory E. coli lipid A that strongly stimulates TLR4 (REF. 69). In humans, Y. pestis encounters a temperature of 37 °C and replaces the agonist form of lipid A with a tetra-acylated form that is a TLR4 antagonist. The tetra-acylated form of lipid A is believed to allow the pathogen to proliferate undetected in the bloodstream during the early stages of infection^{70,71}. Heterologous expression of the E. coli acyltransferase LpxL in Y. pestis restores hexa-acylated lipid A and strong TLR4-stimulatory properties^{70}. Strains that can produce only hexa-acylated lipid A are also avirulent in mice, suggesting that deacylation of lipid A is required for TLR4 evasion and is crucial for Y. pestis pathogenesis^{70}.

Similarly to Y. pestis, the lipid A of F. tularensis is modified according to temperature, affecting membrane integrity and pathogenesis in the environmental vectors of this pathogen, such as protozoa and arthropods (18–26 °C), and mammalian hosts (37 °C). This bacterium has two homologues of LpxD, an essential lipid A biosynthesis acyl transferase that incorporates longer acyl chains at high temperature (37 °C) than at lower temperatures (25 °C and 18 °C)^{72}. A mutant strain that is unable to produce lipid A with longer acyl chains is avirulent in mice and is also more susceptible to antibiotics and CAMPs owing to increased membrane permeability^{72}. These two examples of lipid A modifications emphasize the fact that appropriate adaptation in response to temperature is important for altering the fluidity of the outer membrane, and that this alteration is needed to maximize Y. pestis and F. tularensis virulence.

Vibrio cholerae. V. cholerae, the causative agent of the severe diarrhoeal disease cholera, commonly survives on marine crustaceans such as copepods and can be acquired by humans through ingestion of copepods, leading to intestinal disease^{73}. The V. cholerae O1 El Tor biotype is currently causing a seventh worldwide pandemic affecting an estimated 3–5 million people, but the previous six pandemics were caused by the classical biotype V. cholerae O1. These two biotypes can be differentiated by their sensitivity to polymyxin B^{74}, as V. cholerae O1 El Tor is 100-fold more resistant to polymyxin B than the classical V. cholerae O1 strains. This resistance is conferred by an unusual lipid A modification. A three-gene operon in V. cholerae (almEFG) encodes enzymes that transfer a glycine or diglycine residue to a 3′-linked acyl chain of lipid A^{75} (FIG. 4c). This is the first described amino acid modification of lipid A, and it has revealed a unique link between Gram-negative and Gram-positive cell wall decoration^{75,76}, as Gram-positive bacteria can transfer the amino acid alanine to wall teichoic acids and wall lipoteichoic acids and thus reduce the net negative charge of the cell wall^{76}. The amino acid modification of lipid A in V. cholerae has little effect on TLR4 activation, suggesting that it has evolved for the distinct purpose of promoting survival in the presence of CAMPs in the marine and human host environment^{75}.

and phosphorylation of lipid A also lead to decreased stimulation of TLR4 and the downstream signalling cascade^{50,64,65}. When these modification systems are inactivated through mutation, H. pylori displays hexa-acylated, bis-phosphorylated lipid A (FIG. 4a), which is a strong stimulator of TLR4 (REF. 63). The constitutive lipid A modifications to the acyl chains and phosphate groups are adaptations that allow this bacterium to persist in the harsh gastric environment amidst the several antibacterial components of the innate immune response.

Yersinia pestis and Francisella tularensis. Y. pestis is notorious for its role in human disease throughout history, causing the Black Death plague that killed approximately one-third of the European population in the fourteenth century^{66}. The bacterium has a complex life cycle, colonizing both the flea and human host^{67}. This

Biotype  
A subtype of a bacterial species that can be distinguished from other subtypes by biological characteristics such as motility, resistance to cationic antimicrobial peptides and antibiotics, and the production of virulence factors.

Pandemic  
The widespread occurrence of a human infectious disease that is spread over a large geographical region.

REVIEW S

a Chronic persistence

Wild type Helicobacter pylori Mutant

H. pylori

Gastric epithelium

CAMP resistant TLR4 evasive CAMP sensitive TLR4 agonist

b Initial evasion

37°C Yersinia pestis 21–27°C

Temperature- dependent switch

Flea infected with Y. pestis 21–27°C

37°C

TLR4 antagonist TLR4 agonist

c CAMP resistance

Classical O1 biotype Vibrio cholerae El Tor O1 biotype

Copepod host colonized with V. cholerae

Glycine Diglycine

CAMP sensitive CAMP resistant

Host modification of lipid A

Host tolerance of commensal bacteria. The host environment at mucosal surfaces is rich in LPS, as this compound is produced in abundance by commensal Gram-negative bacteria. This means that the immune response at these surfaces needs to be modulated to promote tolerance to commensals but to mount a defence against pathogens. In general, enteric bacteria produce highly stimulatory, hexa-acylated lipid A^{77}, although decorating the cell surface with such an efficient immunostimulant seems counterintuitive as a strategy for commensal bacteria. In the gastrointestinal

Figure 4 | Lipid A modification strategies that promote survival in the host. Helicobacter pylori, Yersinia pestis and Vibrio cholerae have evolved different mechanisms of lipid A modification to aid colonization of their respective niches inside the host. Differences in structure between the pairs of lipid A molecules are highlighted in red. **a** | The human stomach is the sole niche of *H. pylori*, and the bacterium has adapted to this environment by constitutively modifying lipid A to a form that resists cationic antimicrobial peptides (CAMPs) and evades the lipid A receptor, Toll-like receptor 4 (TLR4). Surface expressed *H. pylori* lipid A consists of a tetra-acylated form that lacks the 4′-phosphate group and is substituted at the C1 position with a phosphoethanolamine<sup>63</sup>. Lipid A modification mutants present a hexa-acylated, bis-phosphorylated species that is a TLR4 agonist. **b** | When residing in the flea vector, *Y. pestis* produces an endotoxic hexa-acylated lipid A that is a strong immunostimulant in humans. Following transmission to the human host, the bacterium senses a shift in temperature (from 21–27°C to 37°C) and synthesizes tetra-acylated lipid A, which escapes detection by TLR4, and immune stimulation is thereby curtailed<sup>70</sup>. **c** | *V. cholerae* normally inhabits freshwater, estuarine and oceanic environments and often colonizes marine organisms such as copepods, which are important for cholera transmission. Copepod colonization provides an environment that probably induces lipid A modifications which are crucial for marine and host survival, a prospect that is currently under investigation. The *V. cholerae* O1 El Tor biotype modifies lipid A with a glycine or diglycine residue, providing resistance to CAMPs, whereas the classical *V. cholerae* O1 biotype remains susceptible<sup>75</sup>.

lumen, tolerance of commensals is partly attributed to the limited expression of TLR4 on the apical surface of epithelial cells, which means that TLR4 expression in the subepithelial layers is important to restrict infection following damage to the outer layer of cells. On the other hand, Paneth cells in the small intestine have recently been shown to produce multiple antimicrobial proteins in order to control inflammation and infection caused by Gram-negative bacteria, and this response is triggered directly in a TLR4-dependent manner by the presence of bacteria and purified LPS<sup>78</sup>. Paneth cells seem perfectly situated to respond appropriately to commensal bacteria, producing an antimicrobial moat that keeps bacterial numbers balanced to avoid excessive inflammation but permit enough commensal colonization to impede invasion by pathogens<sup>78</sup>.

**Host detoxification of lipid A.** When the innate immune response has been triggered by Gram-negative bacteria that have successfully breached the mucosal and epithelial barriers, LPS must be inactivated or removed from the bloodstream to reset the immune response for subsequent LPS exposure. Various host response mechanisms are involved in clearance of and tolerance to LPS from commensal and pathogenic bacteria. Covalent modification of lipid A by host enzymes is one mechanism to inactivate lipid A. The mammalian lipase acyloxyacyl hydrolase (AOAH) is produced by macrophages, dendritic cells and neutrophils and cleaves all secondary linked acyl chains of lipid A (that is, the O-linked acyl chains that are attached to one of the four primary acyl chains linked directly to the sugar backbone)<sup>79</sup>. This renders the LPS molecule less inflammatory owing to impaired interactions with TLR4 (REFS 50,64). Exposure of AOAH-deficient mice to LPS results in prolonged tolerance or unresponsiveness to LPS; this effect can last for months, thus limiting the capacity of the host to react to a secondary exposure. By contrast, wild-type mice are able to recover and respond normally to LPS within 5–10 days<sup>80</sup>. AOAH activity increases during LPS challenge in rabbits and mice, suggesting that this enzyme has a primary role in LPS detoxification<sup>79,80</sup>.

Host alkaline phosphatases have also been shown to modify LPS through the removal of phosphate groups. Whether these phosphatases act on both lipid A and the sugar chains of LPS requires further study, although several studies have shown that alkaline phosphatases interact with LPS *in vivo*<sup>81–83</sup>. Similarly to deacylation, removal of the lipid A phosphate groups results in reduced TLR4 stimulation and reduced LPS potency<sup>50</sup>. Recently, a direct link between LPS and one of these phosphatases (intestinal alkaline phosphatase (Iap; also known as Alpil)) was identified in zebrafish. This particular phosphatase probably cleaves phosphate groups from the lipid A domain of LPS, and it controls the inflammatory response to gut microorganisms. Zebrafish with mutations in *iap* are sensitive to LPS, whereas in wild-type zebrafish, LPS induces TLR4-dependent signalling that upregulates the transcription of *iap*, effectively lowering LPS toxicity<sup>83</sup>.

**Lipopolysaccharide scavenging.** Many other host molecules have been identified that associate with LPS (usually at the lipid A domain) and assist in reducing endotoxin concentration in the body and trafficking LPS for disposal. These molecules have been shown to reduce LPS toxicity by direct binding to LPS or by competing with it for association with essential cofactors of TLR4 (CD14 and LBP)<sup>84</sup>. For example, pre-incubation of the antimicrobial peptide LL-37 with LPS results in reduced cytokine induction in exposed THP-1 cells compared with exposure to LPS alone, suggesting that LL-37 is an LPS scavenger<sup>85</sup>. The abundant plasma protein β₂ glycoprotein I (β₂GPI) forms complexes with LPS through an LPS-binding domain that is well conserved across vertebrates<sup>86</sup>. Binding of β₂GPI to LPS causes a conformational change in β₂GPI that leads to the association of these complexes with monocytes and subsequent clearance of LPS<sup>87</sup>. LBP is important for binding LPS and recruiting it to CD14 and TLR4 to induce immune signalling, but it has also been implicated in a mechanism of neutralizing LPS by transferring the endotoxin to high- and low-density lipoprotein and to chylomicrons, enhancing clearance of LPS following uptake by the liver<sup>88,89</sup>. The direct binding of host factors to lipid A, as well as the effects of lipid A modifications on the above mechanisms, have not yet been investigated, but further studies could reveal additional evasion strategies that are used by Gram-negative bacteria to promote pathogenesis.

**Effects on other cellular processes**

Recent studies have revealed links between lipid A modifications and diverse processes within the bacterial cell. These associations support the notion that lipid A modifications are crucial for bacterial pathogenesis and survival.

Wall teichoic acids  
Long anionic glycopolymers that are covalently linked to the peptidoglycan of Gram-positive bacteria and extend beyond the cell wall.

Wall lipoteichoic acids  
Teichoic acids that are anchored to the plasma membrane of Gram-positive bacteria and extend into the peptidoglycan layer.

Chylomicrons  
Small micelles that are composed of lipids, lipoproteins and proteins, and function to transport lipids.

REVIEWS

Multitarget lipid A modification enzymes. In *Campylobacter jejuni*, the enzyme EptC modifies both lipid A and a structural protein required for the assembly of flagella. EptC is a homologue of *E. coli* EptA, and both enzymes transfer a phosphoethanolamine residue to the 1′ and 4′ positions of lipid A (TABLE 1), masking both phosphate groups and promoting CAMP resistance. Interestingly, in *C. jejuni*, EptC also transfers an essential phosphoethanolamine to FlgG (a flagellar rod protein<sup>90</sup>), and EptC mutants lack wild-type motility and produce fewer flagella, which are required for virulence. This promiscuous *C. jejuni* enzyme also modifies carbohydrates of the LOS core and glycosylated proteins<sup>91</sup>.

Recycling of biosynthetic intermediates. During the assembly of bacterial polymers (for example, peptidoglycan, the capsule and LPS), a wide array of glycoconjugates must be trafficked across the inner membrane<sup>92</sup>. These glycan intermediates are assembled on a universal carrier lipid known as undecaprenyl phosphate (*C*<sub>55</sub>-P)<sup>93</sup>. The *C*<sub>55</sub>-P transports the glycan precursors via a pyrophosphate linkage (*C*<sub>55</sub>-PP–precursor), and when the precursor is removed, the carrier lipid is released as a pyrophosphate and requires dephosphorylation in order to be recycled<sup>93</sup>. Within the periplasm, LpxT transfers a secondary phosphate group onto lipid A at the 1 position (FIG. 1c) and uses *C*<sub>55</sub>-PP as a donor<sup>94</sup>. This finding represented a novel and unexpected link between the modification of lipid A and the recycling of *C*<sub>55</sub>-PP. Furthermore, the connection between these systems implies that *C*<sub>55</sub>-PP could serve as a high-energy phosphate donor for various periplasmic components, although these potential targets are largely unidentified.

Activation of outer-membrane proteins. As a membrane component, LPS is more than just a platform for activities at the cell surface<sup>95</sup>. It also has a role in the activation of omptins, a class of proteases that is widespread among Gram-negative enteric bacteria. The activity of one member of this class, OmpT, has been evaluated in the presence of hexa-acylated and tri-acylated lipid A forms, and the protease was found to be active only in the presence of hexa-acylated lipid A<sup>96</sup>. Interestingly, another omptin named plasminogen activator (Pla), which is central to *Y. pestis* pathogenesis, is more active in *Y. pestis* grown at 37°C and producing tetra-acylated lipid A than in cells grown at 20°C and producing hexa-acylated lipid A<sup>97</sup>.

Flagellar rod  
The central, structural component of bacterial flagella that spans the periplasm.

Flagella  
Whip- or tail-like appendages that are synthesized by many bacteria and are important for motility.

Outer-membrane vesicles  
Small, spherical outer-membrane blebs that are released from Gram-negative bacterial cells and contain membrane and periplasmic components.

Outer-membrane vesicles and toxin delivery. Lipid A modifications also affect the delivery of proteins and toxins that are associated with the outer membrane and outer-membrane vesicles (OMVs). In organisms such as *Porphyromonas gingivalis* and *P. aeruginosa*, negatively charged LPS is enriched in OMVs<sup>98,99</sup>. Furthermore, in *P. gingivalis*, the lipid A found in OMVs is more deacylated than that present in whole-cell membranes, suggesting that lipid A modification facilitates vesicle formation or the sorting of outer-membrane proteins that are packaged into vesicles<sup>98</sup>. In *P. aeruginosa*, OMV formation is stimulated by a small signalling molecule called Pseudomonas quinolone signal (PQS), which is packaged into vesicles and induces vesicle formation in *P. aeruginosa* as well as other bacteria<sup>100</sup>. The exact mechanism of OMV formation is not understood; however, PQS was found to interact with the 4′-phosphate group and acyl chains of lipid A, indicating that modification of lipid A could influence the efficiency of vesicle shedding<sup>101</sup>.

The structure of lipid A and the Kdo residues of LPS also affect the delivery of toxins. For example, two similar toxins — cholera toxin (CT), produced by *V. cholerae*, and heat-labile enterotoxin (LT), produced by enterotoxigenic *E. coli* (ETEC) — are both secreted, are structurally similar and bind the same host cell receptor. However, CT causes a more severe diarrhoeal disease than LT<sup>102</sup>. Although the topic is controversial<sup>103</sup>, the difference in toxin severity seems to be due to the partial sequestration of LT, as it binds to Kdo–lipid A on the bacterial cell surface after secretion. By contrast, the Kdo sugar attached to lipid A in *V. cholerae* is phosphorylated by Kdo kinase (KdkA) (TABLE 1), and this modification inhibits CT association with the outer membrane, thereby allowing robust toxin secretion<sup>104</sup>.

Conclusions and future prospects  
The wide variety of lipid A modifications equips Gram-negative bacteria for survival in their respective niches in the host and other environments. Other functions of lipid A are highlighted by the interplay of lipid A modifications with a diverse set of cellular processes, and both of these facets of lipid A biology emphasize the importance of a continued effort to understand the regulation, benefits and host response to lipid A in its numerous modified forms.

However, many questions remain unanswered. The continued development of animal models is paramount to more closely mimic the host infection site. Considerable progress has been made in this regard by the recent generation of a transgenic mouse expressing the human TLR4 receptor<sup>105</sup>. Furthermore, there is much to learn about how lipid A and its modifications affect the outcome of polymicrobial infections, with recent studies suggesting that LPS has an important role within such communities. For instance, purified LPS from intestinal commensal bacteria has been found to promote replication and transmission of viruses<sup>106,107</sup>. Pathogenesis of orally acquired poliovirus, which replicates in the intestine before spreading to cause a severe systemic infection, is supported by the intestinal microbiota and purified LPS<sup>107</sup>. Mouse mammary tumour virus (MMTV) manipulates the innate immune response by binding LPS and eliciting TLR4-dependent production of IL-10, a cytokine that is required for MMTV persistence<sup>106</sup>. Although these viruses bind LPS and this enhances infectivity *in vitro*, how lipid A modifications affect these phenomena is currently unknown.

Improved methods for studying lipid A modifications *in vivo* could also help elucidate any potential links between modified lipid A and diseases involving TLR4, such as inflammatory bowel disease, diabetes,
rheumatoid arthritis and atherosclerosis<sup>108</sup>. Important insights into human TLR4 polymorphisms might now be possible using the humanized mouse model<sup>105</sup>, and it should also be possible to examine how different variants of TLR4 affect susceptibility to infection by organisms with modified lipid A. This could potentially lead to more personalized medical treatment through the development of a new range of modified TLR4 agonists that could serve as customized immunomodulatory agents to induce or repress particular cytokine responses.

It remains a mystery why lipid A is essential in virtually all Gram-negative organisms. Furthermore, why lipid A biosynthesis is well conserved only to have complex (and varying) modifications occur later, sometimes constitutively, is not understood. How the modification enzymes are distributed and organized within the membrane is also unclear. For example, it is not known whether lipid rafts exist in bacteria and whether some or all of the modification enzymes are enriched at these sites. Lipid A molecules with certain modifications could also be specifically localized in the membrane. Such an organization could have a major influence on the activation of key outer-membrane proteins or on OMV formation and protein sorting.

With further study, inhibitors of modification enzymes could be developed for use as antibiotics or in combination with currently used antibiotics. Although immense progress has been made in our understanding of lipid A modification systems in Gram-negative bacteria and their importance for pathogenesis, there is much work yet to be done.

---

1. Raetz, C. R. & Whitfield, C. Lipopolysaccharide endotoxins. *Annu. Rev. Biochem.* **71**, 635–700 (2002).
2. Schneider, H. *et al.* Heterogeneity of molecular size and antigenic expression within lipo-oligosaccharides of individual strains of *Neisseria gonorrhoeae* and *Neisseria meningitidis*. *Infect. Immun.* **45**, 544–549 (1984).
3. Nummila, K., Kilpeläinen, I., Zahringer, U., Vaara, M. & Helander, I. M. Lipopolysaccharides of polymyxin B-resistant mutants of *Escherichia coli* are extensively substituted by 2-aminoethyl pyrophosphate and contain aminoarabinose in lipid A. *Mol. Microbiol.* **16**, 271–278 (1995).
4. Guo, L. *et al.* Regulation of lipid A modifications by *Salmonella typhimurium* virulence genes *phoP-phoQ*. *Science* **276**, 250–253 (1997).
5. Guo, L. *et al.* Lipid A acylation and bacterial resistance against vertebrate antimicrobial peptides. *Cell* **95**, 189–198 (1998).
6. Gunn, J. S. *et al.* PmrA–PmrB-regulated genes necessary for 4-aminoarabinose lipid A modification and polymyxin resistance. *Mol. Microbiol.* **27**, 1171–1182 (1998).
7. Raetz, C. R., Reynolds, C. M., Trent, M. S. & Bishop, R. E. Lipid A modification systems in gram-negative bacteria. *Annu. Rev. Biochem.* **76**, 295–329 (2007).
8. Roland, K. L., Martin, L. E., Esther, C. R. & Spitznagel, J. K. Spontaneous *pmrA* mutants of *Salmonella typhimurium* LT2 define a new two-component regulatory system with a possible role in virulence. *J. Bacteriol.* **175**, 4154–4164 (1993).
9. Miller, S. I., Kukral, A. M. & Mekalanos, J. J. A two-component regulatory system (*phoP phoQ*) controls *Salmonella typhimurium* virulence. *Proc. Natl Acad. Sci. USA* **86**, 5054–5058 (1989).
10. Prost, L. R. & Miller, S. I. The Salmonellae PhoQ sensor: mechanisms of detection of phagosomal signals. *Cell. Microbiol.* **10**, 576–582 (2008).
11. Gunn, J. S. The Salmonella PmrAB regulon: lipopolysaccharide modifications, antimicrobial peptide resistance and more. *Trends Microbiol.* **16**, 284–290 (2008).
12. Fernandez, L. *et al.* Adaptive resistance to the “last hope” antibiotics polymyxin B and colistin in *Pseudomonas aeruginosa* is mediated by the novel two-component regulatory system ParR-ParS. *Antimicrob. Agents Chemother.* **54**, 3372–3382 (2010).
13. Fernandez, L. *et al.* The two-component system CprRS senses cationic peptides and triggers adaptive resistance in *Pseudomonas aeruginosa* independently of ParRS. *Antimicrob. Agents Chemother.* **56**, 6212–6222 (2012).
14. Llobet, E., Campos, M. A., Gimenez, P., Moranta, D. & Bengoechea, J. A. Analysis of the networks controlling the antimicrobial-peptide-dependent induction of *Klebsiella pneumoniae* virulence factors. *Infect. Immun.* **79**, 3718–3732 (2011).
15. Hagiwara, D. *et al.* Genome-wide analyses revealing a signaling network of the RcsC-YojN-RcsB phosphorelay system in *Escherichia coli*. *J. Bacteriol.* **185**, 5735–5746 (2003).

16. Eguchi, Y. *et al.* Signal transduction cascade between EvgA/EvgS and PhoP/PhoQ two-component systems of *Escherichia coli*. *J. Bacteriol.* **186**, 3006–3014 (2004).
17. Preston, A. *et al.* *Bordetella bronchiseptica* PagP is a Bvg-regulated lipid A palmitoyl transferase that is required for persistent colonization of the mouse respiratory tract. *Mol. Microbiol.* **48**, 725–736 (2003).
18. Gooderham, W. J. *et al.* The sensor kinase PhoQ mediates virulence in *Pseudomonas aeruginosa*. *Microbiology* **155**, 699–711 (2009).
19. Richards, S. M., Strandberg, K. L., Conroy, M. & Gunn, J. S. Cationic antimicrobial peptides serve as activation signals for the *Salmonella Typhimurium* PhoPQ and PmrAB regulons *in vitro* and *in vivo*. *Front. Cell. Infect. Microbiol.* **2**, 102 (2012).
20. Garcia Vescovi, E., Soncini, F. C. & Groisman, E. A. Mg<sup>2+</sup> as an extracellular signal: environmental regulation of *Salmonella* virulence. *Cell* **84**, 165–174 (1996).
21. Kawasaki, K., Ernst, R. K. & Miller, S. I. Inhibition of *Salmonella enterica* serovar Typhimurium lipopolysaccharide deacylation by aminoarabinose membrane modification. *J. Bacteriol.* **187**, 2448–2457 (2005).
22. Winfield, M. D. & Groisman, E. A. Phenotypic differences between *Salmonella* and *Escherichia coli* resulting from the disparate regulation of homologous genes. *Proc. Natl Acad. Sci. USA* **101**, 17162–17167 (2004).
23. Moskowitz, S. M., Ernst, R. K. & Miller, S. I. PmrAB, a two-component regulatory system of *Pseudomonas aeruginosa* that modulates resistance to cationic antimicrobial peptides and addition of aminoarabinose to lipid A. *J. Bacteriol.* **186**, 575–579 (2004).
24. Murata, T., Tseng, W., Guina, T., Miller, S. I. & Nikaido, H. PhoPQ-mediated regulation produces a more robust permeability barrier in the outer membrane of *Salmonella enterica* serovar Typhimurium. *J. Bacteriol.* **189**, 7213–7222 (2007).
25. Gunn, J. S., Ryan, S. S., Van Velkinburgh, J. C., Ernst, R. K. & Miller, S. I. Genetic and functional analysis of a PmrA-PmrB-regulated locus necessary for lipopolysaccharide modification, antimicrobial peptide resistance, and oral virulence of *Salmonella enterica* serovar Typhimurium. *Infect. Immun.* **68**, 6139–6146 (2000).
26. Ernst, R. K. *et al.* Specific lipopolysaccharide found in cystic fibrosis airway *Pseudomonas aeruginosa*. *Science* **286**, 1561–1565 (1999).
27. Valentin-Hansen, P., Johansen, J. & Rasmussen, A. A. Small RNAs controlling outer membrane porins. *Curr. Opin. Microbiol.* **10**, 152–155 (2007).
28. Guillier, M., Gottesman, S. & Storz, G. Modulating the outer membrane with small RNAs. *Genes Dev.* **20**, 2338–2348 (2006).
29. Vogel, J. & Papenfort, K. Small non-coding RNAs and the bacterial outer membrane. *Curr. Opin. Microbiol.* **9**, 605–611 (2006).
30. Moon, K. & Gottesman, S. A PhoQ/P-regulated small RNA regulates sensitivity of *Escherichia coli* to antimicrobial peptides. *Mol. Microbiol.* **74**, 1314–1330 (2009).

31. Coornaert, A. *et al.* MicA sRNA links the PhoP regulon to cell envelope stress. *Mol. Microbiol.* **76**, 467–479 (2010).
32. Corcoran, C. P. *et al.* Superfolder GFP reporters validate diverse new mRNA targets of the classic porin regulator, MicF RNA. *Mol. Microbiol.* **84**, 428–445 (2012).
33. Kato, A., Chen, H. D., Latifi, T. & Groisman, E. A. Reciprocal control between a bacterium’s regulatory system and the modification status of its lipopolysaccharide. *Mol. Cell* **47**, 897–908 (2012).
34. Herrera, C. M., Hankins, J. V. & Trent, M. S. Activation of PmrA inhibits LpxT-dependent phosphorylation of lipid A promoting resistance to antimicrobial peptides. *Mol. Microbiol.* **76**, 1444–1460 (2010).
35. Eguchi, Y. *et al.* B1500, a small membrane protein, connects the two-component systems EvgS/EvgA and PhoQ/PhoP in *Escherichia coli*. *Proc. Natl Acad. Sci. USA* **104**, 18712–18717 (2007).
36. Lippa, A. M. & Goulian, M. Feedback inhibition in the PhoQ/PhoP signaling system by a membrane peptide. *PLoS Genet.* **5**, e1000788 (2009).
37. Bishop, R. E., Kim, S. H. & El Zoeiby, A. Role of lipid A palmitoylation in bacterial pathogenesis. *J. Endotoxin Res.* **11**, 174–180 (2005).
38. Jia, W. *et al.* Lipid trafficking controls endotoxin acylation in outer membranes of *Escherichia coli*. *J. Biol. Chem.* **279**, 44966–44975 (2004).
39. Bishop, R. E. Structural biology of membrane-intrinsic β-barrel enzymes: sentinels of the bacterial outer membrane. *Biochim. Biophys. Acta* **1778**, 1881–1896 (2008).
40. Reines, M. *et al.* Deciphering the acylation pattern of *Yersinia enterocolitica* lipid A. *PLoS Pathog.* **8**, e1002978 (2012).
41. Ausubel, F. M. Are innate immune signaling pathways in plants and animals conserved? *Nature Immunol.* **6**, 973–979 (2005).
42. Gallo, R. L. & Hooper, L. V. Epithelial antimicrobial defence of the skin and intestine. *Nature Rev. Immunol.* **12**, 503–516 (2012).
43. Krauel, K. *et al.* Platelet factor 4 binding to lipid A of Gram-negative bacteria exposes PF4/heparin-like epitopes. *Blood* **120**, 3345–3352 (2012).
44. Lewis, L. A., Shafer, W. M., Dutta Ray, T., Ram, S. & Rice, P. A. Phosphoethanolamine residues on the lipid A moiety of *Neisseria gonorrhoeae* lipo-oligosaccharide modulate binding of complement inhibitors and resistance to complement-killing. *Infect. Immun.* **81**, 33–42 (2012).
45. Gianrande, C. *et al.* Innate immunity probed by lipopolysaccharides affinity strategy and proteomics. *Anal. Bioanal. Chem.* **405**, 775–784 (2012).
46. Tan, L. A., Yang, A. C., Kishore, U. & Sim, R. B. Interactions of complement proteins C1q and factor H with lipid A and *Escherichia coli*: further evidence that factor H regulates the classical complement pathway. *Protein Cell* **2**, 320–332 (2011).
47. Patil, A., Hughes, A. L. & Zhang, G. Rapid evolution and diversification of mammalian α-defensins as revealed by comparative analysis of rodent and primate genes. *Physiol. Genom.* **20**, 1–11 (2004).
REVIEW S

48. Peschel, A. & Sahl, H. G. The co-evolution of host cationic antimicrobial peptides and microbial resistance. *Nature Rev. Microbiol.* **4**, 529–536 (2006).

49. Sassi, N., Paul, C., Martin, A., Bettaieb, A. & Jeannin, J. F. Lipid A-induced responses *in vivo*. *Adv. Exp. Med. Biol.* **667**, 69–80 (2009).

50. Park, B. S. *et al.* The structural basis of lipopolysaccharide recognition by the TLR4-MD-2 complex. *Nature* **458**, 1191–1195 (2009). Co-crystallization of the TLR4-MD2 complex with lipid A illuminates the importance of the interactions between the phosphates and acyl chains of lipid A, the MD2 pocket and TLR4.

51. Kong, Q. *et al.* Palmitoylation state impacts induction of innate and acquired immunity by the *Salmonella enterica* serovar Typhimurium *msbB* mutant. *Infect. Immun.* **79**, 5027–5038 (2011).

52. Kong, Q. *et al.* Phosphate groups of lipid A are essential for *Salmonella enterica* serovar Typhimurium virulence and affect innate and adaptive immunity. *Infect. Immun.* **80**, 3215–3224 (2012).

53. Hajjar, A. M., Ernst, R. K., Tsai, J. H., Wilson, C. B. & Miller, S. I. Human Toll-like receptor 4 recognizes host-specific LPS modifications. *Nature Immunol.* **3**, 354–359 (2002).

54. Casella, C. R. & Mitchell, T. C. Putting endotoxin to work for us: monophosphoryl lipid A as a safe and effective vaccine adjuvant. *Cell. Mol. Life Sci.* **65**, 3231–3240 (2008).

55. Yamamoto, M. & Akira, S. Lipid A receptor TLR4-mediated signaling pathways. *Adv. Exp. Med. Biol.* **667**, 59–68 (2009).

56. Mata-Haro, V. *et al.* The vaccine adjuvant monophosphoryl lipid A as a TRIF-biased agonist of TLR4. *Science* **316**, 1628–1632 (2007).

57. Ding, P. H., Wang, C. Y., Darveau, R. P. & Jin, L. *Porphyromonas gingivalis* LPS stimulates the expression of LPS-binding protein in human oral keratinocytes *in vitro*. *Innate Immun.* **19**, 66–75 (2012). Report demonstrating that modified lipid A, such as 4′-monophosphoryl lipid A, makes an excellent vaccine adjuvant owing to the ability to initiate safer, less inflammatory but still effective immune responses.

58. Froelich, J. M., Tran, K. & Wall, D. A *pmrA* constitutive mutant sensitizes *Escherichia coli* to deoxycholic acid. *J. Bacteriol.* **188**, 1180–1183 (2006).

59. Gopinath, S., Carden, S. & Monack, D. Shedding light on *Salmonella* carriers. *Trends Microbiol.* **20**, 320–327 (2012).

60. Ruby, T., McLaughlin, L., Gopinath, S. & Monack, D. *Salmonella’s* long-term relationship with its host. *FEMS Microbiol. Rev.* **36**, 600–615 (2012).

61. Moreira, C. G. *et al.* Virulence and stress-related periplasmic protein (VisP) in bacterial/host associations. *Proc. Natl Acad. Sci. USA* **110**, 1470–1475 (2013).

62. Parsonnet, J. *et al.* *Helicobacter pylori* infection and the risk of gastric carcinoma. *N. Engl. J. Med.* **325**, 1127–1131 (1991).

63. Cullen, T. W. *et al.* *Helicobacter pylori* versus the host: remodeling of the bacterial outer membrane is required for survival in the gastric mucosa. *PLoS Pathog.* **7**, e1002454 (2011). *H. pylori* extensively modifies all of the lipid A it produces, promoting escape from both CAMP-mediated death and TLR4 detection in the human stomach.

64. Teghanemt, A., Zhang, D., Levis, E. N., Weiss, J. P. & Gioannini, T. L. Molecular basis of reduced potency of underacylated endotoxins. *J. Immunol.* **175**, 4669–4676 (2005).

65. Qureshi, N., Takayama, K. & Ribi, E. Purification and structural determination of nontoxic lipid A obtained from the lipopolysaccharide of *Salmonella typhimurium*. *J. Biol. Chem.* **257**, 11808–11815 (1982).

66. Raoult, D. *et al.* Molecular identification by “suicide PCR” of *Yersinia pestis* as the agent of medieval black death. *Proc. Natl Acad. Sci. USA* **97**, 12800–12803 (2000).

67. Chouikha, I. & Hinnebusch, B. J. *Yersinia-flea* interactions and the evolution of the arthropod-borne transmission route of plague. *Curr. Opin. Microbiol.* **15**, 239–246 (2012).

68. Rebeil, R. *et al.* Characterization of late acyltransferase genes of *Yersinia pestis* and their role in temperature-dependent lipid A variation. *J. Bacteriol.* **188**, 1381–1388 (2006).

69. Matsuura, M., Takahashi, H., Watanabe, H., Saito, S. & Kawahara, K. Immunomodulatory effects of *Yersinia pestis* lipopolysaccharides on human macrophages. *Clin. Vaccine Immunol.* **17**, 49–55 (2010).

70. Montminy, S. W. *et al.* Virulence factors of *Yersinia pestis* are overcome by a strong lipopolysaccharide response. *Nature Immunol.* **7**, 1066–1073 (2006). *Y. pestis* modifies lipid A in a temperature-dependent manner, promoting immune evasion and survival during transition from the flea to the human host.

71. Telepnev, M. V. *et al.* Tetraacylated lipopolysaccharide of *Yersinia pestis* can inhibit multiple Toll-like receptor-mediated signaling pathways in human dendritic cells. *J. Infect. Dis.* **200**, 1694–1702 (2009).

72. Li, Y. *et al.* LPS remodeling is an evolved survival strategy for bacteria. *Proc. Natl Acad. Sci. USA* **109**, 8716–8721 (2012). *F. tularensis* has evolved to incorporate acyl chains of variable lengths into lipid A, altering the susceptibility of the membrane as well as virulence in the mouse host.

73. Sherman, I. W. *Twelve Diseases That Changed Our World*. (American Society for Microbiology Press, 2007).

74. Gangarosa, E. J., Bennett, J. V. & Boring, J. R. III. Differentiation between *Vibrio cholerae* and *Vibrio cholerae* biotype El Tor by the polymyxin B disc test: comparative results with TCBS, Monsur’s, Mueller-Hinton and nutrient agar media. *Bull. World Health Organ.* **36**, 987–990 (1967).

75. Hankins, J. V., Madsen, J. A., Giles, D. K., Brodbelt, J. S. & Trent, M. S. Amino acid addition to *Vibrio cholerae* LPS establishes a link between surface remodeling in Gram-positive and Gram-negative bacteria. *Proc. Natl Acad. Sci. USA* **109**, 8722–8727 (2012). The first description of an amino acid addition to lipid A uncovers a potential new class of lipid A modification system, as well as a striking connection between Gram-negative and Gram-positive bacteria.

76. Neuhaus, F. C. & Baddiley, J. A continuum of anionic charge: structures and functions of d-alanyl-teichoic acids in gram-positive bacteria. *Microbiol. Mol. Biol. Rev.* **67**, 686–723 (2003).

77. Munford, R. S. & Varley, A. W. Shield as signal: lipopolysaccharides and the evolution of immunity to Gram-negative bacteria. *PLoS Pathog.* **2**, e67 (2006).

78. Vaishnava, S., Behrendt, C. L., Ismail, A. S., Eckmann, L. & Hooper, L. V. Paneth cells directly sense gut commensals and maintain homeostasis at the intestinal host-microbial interface. *Proc. Natl Acad. Sci. USA* **105**, 20858–20863 (2008). LPS is important at epithelial barriers, where it stimulates a basal level of host defence to prevent adverse bacterial colonization.

79. Erwin, A. L. & Munford, R. S. Plasma lipopolysaccharide-deacylating activity (acyloxyacyl hydrolase) increases after lipopolysaccharide administration to rabbits. *Lab. Invest.* **65**, 138–144 (1991).

80. Lu, M., Varley, A. W., Ohta, S., Hardwick, J. & Munford, R. S. Host inactivation of bacterial lipopolysaccharide prevents prolonged tolerance following Gram-negative bacterial infection. *Cell Host Microbe* **4**, 293–302 (2008). The human host produces a deacylase that modifies and inactivates lipid A, effectively eliminating stimulatory LPS and allowing recovery from tolerance.

81. Tuin, A., Huizinga-Van der Vlag, A., van Loenen-Weemaes, A. M., Meijer, D. K. & Poelstra, K. On the role and fate of LPS-dephosphorylating activity in the rat liver. *Am. J. Physiol. Gastrointest. Liver Physiol.* **290**, G377–G385 (2006).

82. Koyama, I., Matsunaga, T., Harada, T., Hokari, S. & Komoda, T. Alkaline phosphatases reduce toxicity of lipopolysaccharides *in vivo* and *in vitro* through dephosphorylation. *Clin. Biochem.* **35**, 455–461 (2002).

83. Bates, J. M., Akerlund, J., Mittge, E. & Guillemin, K. Intestinal alkaline phosphatase detoxifies lipopolysaccharide and prevents inflammation in zebrafish in response to the gut microbiota. *Cell Host Microbe* **2**, 371–382 (2007). The host produces enzymes such as phosphatases to modify and inactivate lipid A, effectively controlling the inflammatory response.

84. Rosenfeld, Y., Papo, N. & Shai, Y. Endotoxin (lipopolysaccharide) neutralization by innate immunity host-defense peptides. Peptide properties and plausible modes of action. *J. Biol. Chem.* **281**, 1636–1643 (2006).

85. Scott, A. *et al.* Evaluation of the ability of LL-37 to neutralise LPS *in vitro* and *ex vivo*. *PLoS ONE* **6**, e26525 (2011).

86. Agar, C., de Groot, P. G., Marquart, J. A. & Meijers, J. C. Evolutionary conservation of the lipopolysaccharide binding site of β₂-glycoprotein I. *Thromb. Haemost.* **106**, 1069–1075 (2011).

87. Agar, C. *et al.* beta(2)-glycoprotein I: a novel component of innate immunity. *Blood* **117**, 6939–6947 (2011).

88. Alexander, C. & Rietschel, E. T. Bacterial lipopolysaccharides and innate immunity. *J. Endotoxin Res.* **7**, 167–202 (2001).

89. Vreugdenhil, A. C. *et al.* Lipopolysaccharide (LPS)-binding protein mediates LPS detoxification by chylomicrons. *J. Immunol.* **170**, 1399–1405 (2003).

90. Cullen, T. W. & Trent, M. S. A link between the assembly of flagella and lipooligosaccharide of the Gram-negative bacterium *Campylobacter jejuni*. *Proc. Natl Acad. Sci. USA* **107**, 5160–5165 (2010). Lipid A modification systems have evolved pleiotropic roles and affect other systems, such as the link between lipid A modification and the assembly of flagella in *C. jejuni*.

91. Cullen, T. W. *et al.* EptC of *Campylobacter jejuni* mediates phenotypes involved in host interactions and virulence. *Infect. Immun.* **81**, 430–440 (2012).

92. Troy, F. A., Vijay, I. K. & Tesche, N. Role of undecaprenyl phosphate in synthesis of polymers containing sialic acid in *Escherichia coli*. *J. Biol. Chem.* **250**, 156–163 (1975).

93. Bouhss, A., Trunkfield, A. E., Bugg, T. D. & Mengin-Lecreulx, D. The biosynthesis of peptidoglycan lipid-linked intermediates. *FEMS Microbiol. Rev.* **32**, 208–233 (2008).

94. Touze, T., Tran, A. X., Hankins, J. V., Mengin-Lecreulx, D. & Trent, M. S. Periplasmic phosphorylation of lipid A is linked to the synthesis of undecaprenyl phosphate. *Mol. Microbiol.* **67**, 264–277 (2008).

95. Coskun, U. & Simons, K. Cell membranes: the lipid perspective. *Structure* **19**, 1543–1548 (2011).

96. Kramer, R. A. *et al.* Lipopolysaccharide regions involved in the activation of *Escherichia coli* outer membrane protease OmpT. *Eur. J. Biochem.* **269**, 1746–1752 (2002).

97. Suomalainen, M. *et al.* Temperature-induced changes in the lipopolysaccharide of *Yersinia pestis* affect plasminogen activation by the pla surface protease. *Infect. Immun.* **78**, 2644–2652 (2010).

98. Haurat, M. F. *et al.* Selective sorting of cargo proteins into bacterial membrane vesicles. *J. Biol. Chem.* **286**, 1269–1276 (2011).

99. Kadurugamuwa, J. L. & Beveridge, T. J. Virulence factors are released from *Pseudomonas aeruginosa* in association with membrane vesicles during normal growth and exposure to gentamicin: a novel mechanism of enzyme secretion. *J. Bacteriol.* **177**, 3998–4008 (1995).

100. Mashburn, L. M. & Whiteley, M. Membrane vesicles traffic signals and facilitate group activities in a prokaryote. *Nature* **437**, 422–425 (2005).

101. Mashburn-Warren, L. *et al.* Interaction of quorum signals with outer membrane lipids: insights into prokaryotic membrane vesicle formation. *Mol. Microbiol.* **69**, 491–502 (2008).

102. Spangler, B. D. Structure and function of cholera toxin and the related *Escherichia coli* heat-labile enterotoxin. *Microbiol. Rev.* **56**, 622–647 (1992).

103. Jansson, L., Angstrom, J., Lebens, M. & Teneberg, S. No direct binding of the heat-labile enterotoxin of *Escherichia coli* to *E. coli* lipopolysaccharides. *Glycoconj. J.* **27**, 171–179 (2010).

104. Horstman, A. L., Bauman, S. J. & Kuehn, M. J. Lipopolysaccharide 3-deoxy-d-manno-octulosonic acid (Kdo) core determines bacterial association of secreted toxins. *J. Biol. Chem.* **279**, 8070–8075 (2004).

105. Hajjar, A. M. *et al.* Humanized TLR4-MD-2 mice reveal LPS recognition differentially impacts susceptibility to *Yersinia pestis* and *Salmonella enterica*. *PLoS Pathog.* **8**, e1002963 (2012). The humanized TLR4-MD2-containing mouse model confirms the different responses of humans and mice to modified lipid A, as well as providing an important tool for future studies of the role of lipid A modification systems in pathogenesis.

106. Kane, M. *et al.* Successful transmission of a retrovirus depends on the commensal microbiota. *Science* **334**, 245–249 (2011).

107. Kuss, S. K. *et al.* Intestinal microbiota promote enteric virus replication and systemic pathogenesis. *Science* **334**, 249–252 (2011).
The LPS of Gram-negative commensal bacteria affects the pathogenesis of diverse organisms, including viruses.

108. O’Neill, L. A., Bryant, C. E. & Doyle, S. L. Therapeutic targeting of Toll-like receptors for infectious and inflammatory diseases and cancer. *Pharmacol. Rev.* **61**, 177–197 (2009).

109. Murray, G. L., Attridge, S. R. & Morona, R. Regulation of *Salmonella typhimurium* lipopolysaccharide O antigen chain length is required for virulence; identification of FepE as a second Wzz. *Mol. Microbiol.* **47**, 1395–1406 (2003).

110. Zanoni, I. *et al.* Similarities and differences of innate immune responses elicited by smooth and rough LPS. *Immunol. Lett.* **142**, 41–47 (2012).

111. Bogomolnaya, L. M., Santiviago, C. A., Yang, H. J., Baumler, A. J. & Andrews-Polymenis, H. L. ‘Form variation’ of the O12 antigen is critical for persistence of *Salmonella Typhimurium* in the murine intestine. *Mol. Microbiol.* **70**, 1105–1119 (2008).

112. Kim, M. L. & Slach, J. M. Effect of acetylation (O-factor 5) on the polyclonal antibody response to *Salmonella typhimurium* O-antigen. *FEMS Immunol. Med. Microbiol.* **26**, 83–92 (1999).

113. Meredith, T. C. *et al.* Modification of lipopolysaccharide with colanic acid (M-antigen) repeats in *Escherichia coli*. *J. Biol. Chem.* **282**, 7790–7798 (2007).

114. Gulati, S. *et al.* Enhanced factor H binding to sialylated Gonococci is restricted to the sialylated lacto-N-neotetraose lipoigosaccharide species: implications for serum resistance and evidence for a bifunctional lipoigosaccharide sialyltransferase in gonococci. *Infect. Immun.* **73**, 7390–7397 (2005).

115. Ram, S. *et al.* Heptose I glycan substitutions on *Neisseria gonorrhoeae* lipoigosaccharide influence C4b-binding protein binding and serum resistance. *Infect. Immun.* **75**, 4071–4081 (2007).

116. van Vliet, S. J. *et al.* Variation of *Neisseria gonorrhoeae* lipoigosaccharide directs dendritic cell-induced T helper responses. *PLoS Pathog.* **5**, e1000625 (2009).

117. Vacchelli, E. *et al.* Trial watch: FDA-approved Toll-like receptor agonists for cancer therapy. *Oncoimmunology* **1**, 894–907 (2012).

118. Rockwell, C. E., Morrison, D. C. & Qureshi, N. Lipid A-mediated tolerance and cancer therapy. *Adv. Exp. Med. Biol.* **667**, 81–99 (2009).

119. Gaekwad, J. *et al.* Differential induction of innate immune responses by synthetic lipid a derivatives. *J. Biol. Chem.* **285**, 29375–29386 (2010).

120. Needham, B. D. *et al.* Modulating the innate immune response by combinatorial engineering of endotoxin. *Proc. Natl Acad. Sci. USA* **110**, 1464–1469 (2013).

121. Keiser, P. B. *et al.* A phase 1 study of a meningococcal native outer membrane vesicle vaccine made from a group B strain with deleted *lpxL1* and *synX*, over-expressed factor H binding protein, two PorAs and stabilized OpcA expression. *Vaccine* **29**, 1413–1420 (2011).

122. Chen, D. J. *et al.* Delivery of foreign antigens by engineered outer membrane vesicle vaccines. *Proc. Natl Acad. Sci. USA* **107**, 3099–3104 (2010).

123. Swulius, M. T. *et al.* Long helical filaments are not seen encircling cells in electron cryotomograms of rod-shaped bacteria. *Biochem. Biophys. Res. Commun.* **407**, 650–655 (2011).

124. Gibbons, H. S., Kalb, S. R., Cotter, R. J. & Raetz, C. R. Role of Mg²⁺ and pH in the modification of *Salmonella* lipid A after endocytosis by macrophage tumour cells. *Mol. Microbiol.* **55**, 425–440 (2005).

125. Trent, M. S., Stead, C. M., Tran, A. X. & Hankins, J. V. Diversity of endotoxin and its impact on pathogenesis. *J. Endotoxin Res.* **12**, 205–223 (2006).

126. Trent, M. S., Ribeiro, A. A., Lin, S., Cotter, R. J. & Raetz, C. R. An inner membrane enzyme in *Salmonella* and *Escherichia coli* that transfers 4-amino-4-deoxy-l-arabinose to lipid A: induction on polymyxin-resistant mutants and role of a novel lipid-linked donor. *J. Biol. Chem.* **276**, 43122–43131 (2001).

127. McCoy, A. J., Liu, H., Falla, T. J. & Gunn, J. S. Identification of *Proteus mirabilis* mutants with increased sensitivity to antimicrobial peptides. *Antimicrob. Agents Chemother.* **45**, 2030–2037 (2001).

128. Madala, N. E., Leone, M. R., Molinaro, A. & Dubery, I. A. Deciphering the structural and biological properties of the lipid A moiety of lipopolysaccharides from *Burkholderia cepacia* strain ASP B 2D, in *Arabidopsis thaliana*. *Glycobiology* **21**, 184–194 (2011).

129. Casabuono, A. C., van der Ploeg, C. A., Roge, A. D., Bruno, S. B. & Couto, A. S. Characterization of lipid A profiles from *Shigella flexneri* variant X lipopolysaccharide. *Rapid Commun. Mass Spectrom.* **26**, 2011–2020 (2012).

130. Lee, H., Hsu, F. F., Turk, J. & Groisman, E. A. The *PmrA*-regulated *pmrC* gene mediates phosphoethanolamine modification of lipid A and polymyxin resistance in *Salmonella enterica*. *J. Bacteriol.* **186**, 4124–4133 (2004).

131. Knirel, Y. A. *et al.* Cold temperature-induced modifications to the composition and structure of the lipopolysaccharide of *Yersinia pestis*. *Carbohydr. Res.* **340**, 1625–1630 (2005).

132. Reynolds, C. M., Kalb, S. R., Cotter, R. J. & Raetz, C. R. A phosphoethanolamine transferase specific for the outer 3-deoxy-D-manno-octulosonic acid residue of *Escherichia coli* lipopolysaccharide. Identification of the *eptB* gene and Ca²⁺ hypersensitivity of an *eptB* deletion mutant. *J. Biol. Chem.* **280**, 21202–21211 (2005).

133. Cullen, T. W., Madsen, J. A., Ivanov, P. L., Brodbelt, J. S. & Trent, M. S. Characterization of unique modification of flagellar rod protein FlgG by *Campylobacter jejuni* lipid A phosphoethanolamine transferase, linking bacterial locomotion and antimicrobial peptide resistance. *J. Biol. Chem.* **287**, 3326–3336 (2012).

134. Song, F., Guan, Z. & Raetz, C. R. Biosynthesis of undecaprenyl phosphate-galactosamine and undecaprenyl phosphate-glucose in *Francisella novicida*. *Biochemistry* **48**, 1173–1182 (2009).

135. Soni, S. *et al.* *Francisella tularensis* blue-gray phase variation involves structural modifications of lipopolysaccharide O-antigen, core and lipid A and affects intramacrophage survival and vaccine efficacy. *Front. Microbiol.* **1**, 129 (2010).

136. White, K. A., Lin, S., Cotter, R. J. & Raetz, C. R. A *Haemophilus influenzae* gene that encodes a membrane bound 3-deoxy-D-manno-octulosonic acid (Kdo) kinase. Possible involvement of kdo phosphorylation in bacterial virulence. *J. Biol. Chem.* **274**, 31391–31400 (1999).

137. Chalabaev, S., Kim, T. H., Ross, R., Derian, A. & Kasper, D. L. 3-Deoxy-D-manno-octulosonic acid (Kdo) hydrolase identified in *Francisella tularensis*, *Helicobacter pylori*, and *Legionella pneumophila*. *J. Biol. Chem.* **285**, 34330–34336 (2010).

138. Stead, C. M., Zhao, J., Raetz, C. R. & Trent, M. S. Removal of the outer Kdo from *Helicobacter pylori* lipopolysaccharide and its impact on the bacterial surface. *Mol. Microbiol.* **78**, 837–852 (2010).

139. Zhao, J. & Raetz, C. R. A two-component Kdo hydrolase in the inner membrane of *Francisella novicida*. *Mol. Microbiol.* **78**, 820–836 (2010).

140. Chung, H. S. & Raetz, C. R. Dioxygenases in *Burkholderia ambifaria* and *Yersinia pestis* that hydroxylate the outer Kdo unit of lipopolysaccharide. *Proc. Natl Acad. Sci. USA* **108**, 510–515 (2011).

141. Boon Hinckley, M. *et al.* A *Leptospira interrogans* enzyme with similarity to yeast Ste14p that methylates the 1-phosphate group of lipid A. *J. Biol. Chem.* **280**, 30214–30224 (2005).

142. Karbarz, M. J., Kalb, S. R., Cotter, R. J. & Raetz, C. R. Expression cloning and biochemical characterization of a *Rhizobium leguminosarum* lipid A 1-phosphatase. *J. Biol. Chem.* **278**, 39269–39279 (2003).

143. Wang, X., McGrath, S. C., Cotter, R. J. & Raetz, C. R. Expression cloning and periplasmic orientation of the *Francisella novicida* lipid A 4′-phosphatase LpxF. *J. Biol. Chem.* **281**, 9321–9330 (2006).

144. Coats, S. R., To, T. T., Jain, S., Braham, P. H. & Darveau, R. P. Porphyromonas gingivalis resistance to polymyxin B is determined by the lipid A 4′-phosphatase, PGN_0524. *Int. J. Oral Sci.* **1**, 126–135 (2009).

145. Gibbons, H. S., Reynolds, C. M., Guan, Z. & Raetz, C. R. An inner membrane dioxygenase that generates the 2-hydroxymyristate moiety of *Salmonella* lipid A. *Biochemistry* **47**, 2814–2825 (2008).

146. Carty, S. M., Sreekumar, K. R. & Raetz, C. R. Effect of cold shock on lipid A biosynthesis in *Escherichia coli* induction at 12 °C of an acyltransferase specific for palmitoleoyl-acyl carrier protein. *J. Biol. Chem.* **274**, 9677–9685 (1999).

147. Soderberg, M. A. & Cianciotto, N. P. Mediators of lipid A modification, RNA degradation, and central intermediary metabolism facilitate the growth of *Legionella pneumophila* at low temperatures. *Curr. Microbiol.* **60**, 59–65 (2010).

148. Reynolds, C. M. *et al.* An outer membrane enzyme encoded by *Salmonella typhimurium lpxR* that removes the 3′-acyloxyacyl moiety of lipid A. *J. Biol. Chem.* **281**, 21974–21987 (2006).

149. Trent, M. S., Pabich, W., Raetz, C. R. & Miller, S. I. A PhoP/PhoQ-induced lipase (PagL) that catalyzes 3-O-deacylation of lipid A precursors in membranes of *Salmonella typhimurium*. *J. Biol. Chem.* **276**, 9083–9092 (2001).

150. Bishop, R. E. *et al.* Transfer of palmitate from phospholipids to lipid A in outer membranes of Gram-negative bacteria. *EMBO J.* **19**, 5071–5080 (2000).

151. Fukuoka, S. *et al.* Physico-chemical and biophysical study of the interaction of hexa- and heptaacyl lipid A from *Erwinia carotovora* with magainin 2-derived antimicrobial peptides. *Biochim. Biophys. Acta* **1778**, 2051–2057 (2008).

152. Clements, A. *et al.* Secondary acylation of *Klebsiella pneumoniae* lipopolysaccharide contributes to sensitivity to antibacterial peptides. *J. Biol. Chem.* **282**, 15569–15577 (2007).

### Acknowledgements

This work was supported by the US National Institutes of Health (grants AI064184 and AI76322), by the US Army Research Office (grant 61789-MA-MUR) and by the Cystic Fibrosis Foundation (grant Trent13G0).

### Competing interests statement

The authors declare no competing financial interests.

### FURTHER INFORMATION

M. Stephen Trent’s homepage:  
<http://www.icmb.utexas.edu/directory/details.asp?id=3806>

ALL LINKS ARE ACTIVE IN THE ONLINE PDF