
Microbiology:

Structural and functional characterization of *Pseudomonas aeruginosa* AlgX: role of AlgX in alginate acetylation

Laura M. Riley, Joel T. Weadge, Perrin Baker, Howard Robinson, Jeroen D. C. Codée, Peter A. Tipton, Dennis E. Ohman and P. Lynne Howell  
*J. Biol. Chem.* published online June 18, 2013

---

Access the most updated version of this article at doi: [10.1074/jbc.M113.484931](http://dx.doi.org/10.1074/jbc.M113.484931)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

[Click here](#) to choose from all of JBC's e-mail alerts

Supplemental material:  
[http://www.jbc.org/content/suppl/2013/06/18/M113.484931.DC1.html](http://www.jbc.org/content/suppl/2013/06/18/M113.484931.DC1.html)

This article cites 0 references, 0 of which can be accessed free at  
[http://www.jbc.org/content/early/2013/06/18/jbc.M113.484931.full.html#ref-list-1](http://www.jbc.org/content/early/2013/06/18/jbc.M113.484931.full.html#ref-list-1)

JBC Papers in Press. Published on June 18, 2013 as Manuscript M113.484931  
The latest version is at http://www.jbc.org/cgi/doi/10.1074/jbc.M113.484931  

Structural and functional characterization of *Pseudomonas aeruginosa* AlgX:  
Role of AlgX in alginate acetylation  

Laura M. Riley${}^{1,\#}$, Joel T. Weadge${}^{1,\#,\$}$, Perrin Baker${}^{1}$, Howard Robinson${}^{2}$, Jeroen D.C. Codée${}^{3}$, Peter A. Tipton${}^{4}$, Dennis E. Ohman${}^{5}$, and P. Lynne Howell${}^{1,6}$  

${}^{1}$Program in Molecular Structure and Function, The Hospital for Sick Children, Toronto, ON, M5G1X8, Canada  
${}^{2}$Photon Sciences Division, Brookhaven National Laboratory, Upton, New York, 11973-5000, USA  
${}^{3}$Leiden Institute of Chemistry, Leiden University P.O. Box 9502, 2300 RA Leiden, The Netherlands  
${}^{4}$Department of Biochemistry, University of Missouri, Columbia, MO 65211, USA  
${}^{5}$Department of Microbiology and Immunology, and McGuire Veterans Affairs Medical Center, Virginia Commonwealth University Medical Center, Richmond, VA 23298-0678, USA  
${}^{6}$Department of Biochemistry, University of Toronto, Toronto, ON, M5S1A8, Canada  

Running title: Role of AlgX in alginate acetylation  

#These authors contributed equally to this work  
\$Present address: Department of Biology, Wilfrid Laurier University, Waterloo, ON, N2L 3C5, Canada  
To whom correspondence should be addressed: P. Lynne Howell, Program in Molecular Structure and Function, The Hospital for Sick Children, Toronto, ON, M5G1X8, CANADA, Tel.: (416) 813-5378; Fax: (416) 813-5379; E-mail: howell@sickkids.ca  

Keywords: alginate, polysaccharide biosynthesis, SGNH hydrolase, acetylation, X-ray crystallography, biofilms  

Background: AlgX is required for the biosynthesis and export of the exopolysaccharide alginate.  
Results: The structure of AlgX has been determined, and the functional characterization of AlgX and mutant variants performed.  
Conclusions: AlgX contains an SGNH hydrolase-like domain and carbohydrate-binding module. Mutation of the Ser-His-Asp triad *in vivo* results in non-acetylated alginate.  
Significance: This is the first structural characterization of a polysaccharide acetyl-transferase.  

SUMMARY  
The exopolysaccharide alginate, produced by mucoid *Pseudomonas aeruginosa* in the lungs of cystic fibrosis patients, undergoes two different chemical modifications as it is synthesized that alter the properties of the polymer and hence the biofilm. One modification, acetylation, causes the cells in the biofilm to adhere better to lung epithelium, form microcolonies, and resist the effects of the host immune system and/or antibiotics. Alginate biosynthesis requires twelve proteins encoded by the *algD* operon, including AlgX, and while this protein is essential for polymer production, its exact role is unknown. In this study we present the X-ray crystal structure of AlgX at 2.15 Å resolution. The structure reveals that AlgX is a two-domain protein, with an N-terminal domain with structural homology to members of the SGNH hydrolase superfamily and a C-terminal carbohydrate-binding module (CBM). A number of residues in the CBM form a substrate-recognition “pinch point” that we propose aids in alginate binding and

Copyright 2013 by The American Society for Biochemistry and Molecular Biology, Inc.
orientation. Although the topology of the N-terminal domain deviates from canonical SGNH hydrolases, the residues that constitute the Ser-His-Asp catalytic triad characteristic of this family are structurally conserved. In vivo studies reveal that site-specific mutation of these residues results in non-acetylated alginate. This catalytic triad is also required for acetyl esterase activity *in vitro*. Our data suggest that not only does AlgX protect the polymer as it passages through the periplasm but that it also plays a role in alginate acetylation. Our results provide the first structural insight for a wide group of closely-related bacterial polysaccharide acetyltransferases.

Exopolysaccharides have numerous and varied functions, and play a key role in bacterial survival as major components of biofilm matrices (1). Biofilms are of great medical importance as, once formed, they are extremely difficult to eradicate. Bacteria within the matrix are less prone to dehydration, and have increased resistance to antibiotics and the host immune system (2,3). The architecture and properties of the matrix are determined by the components present, which can include not only exopolysaccharides but also extracellular DNA and protein (2,4,5). The matrix composition is species dependent and can change in response to different external pressures (2,6). The properties or characteristics of the biofilm matrix can also be altered by the embedded bacteria using a variety of modifications, which can either alter the exopolysaccharides directly, or may alter the proteins involved in the formation, maintenance and dispersion of the biofilm (5,7-12). For example, in Escherichia coli, Staphylococcus aureus and Staphylococcus epidermidis, biofilm formation requires the partial de-N-acetylation of the exopolysaccharide poly β-1,6-N-acetyl-D-glucosamine (PNAG) (5,7,9). In the Gram negative bacterium E. coli, this modification is a pre-requisite for the export of PNAG through the outer membrane (9), whereas, in Gram positive Staphylococcus species, partial deacetylation is necessary for retention of the polymer on the cell surface (5). In contrast, while acetylation of the cellulose that forms Pseudomonas fluorescens biofilms does not affect biofilm attachment to solid surfaces or biofilm

initiation at the air-liquid interface, it does produce biofilms that are thicker and almost 10-fold stronger than its non-acetylated counterpart (12). In addition, the rate of increase in biofilm mass was observed to be higher when cellulose was acetylated, and more cells were retained within the matrix (12). Dispersion of cells from a *Pseudomonas putida* biofilm in nutrient-limiting conditions also requires modifications, but in this case to the associated proteins. A starvation-led decrease in intracellular bis-(3',5')-cyclic dimeric guanosine monophosphate (c-di-GMP) levels triggers a regulatory cascade that ultimately leads to cleavage of the protein LapA, by the cysteine protease, LapG. LapA is associated with both the outer membrane of the cell and with the exopolysaccharide biofilm components, and therefore links the cells to the biofilm matrix. LapA cleavage dissociates bacterial cells from the exopolysaccharide in the biofilm, thus enabling cellular dispersion to occur (8). Given the importance of such modifications, research into the mechanisms by which bacteria regulate and perform these modifications is required if we are to fully understand the function and purpose of specific biofilms.

*Pseudomonas aeruginosa* is an opportunistic pathogen that infects the lungs of cystic fibrosis patients, and is responsible for much of the morbidity and mortality associated with this disease (13-16). The conditions in the cystic fibrosis lung cause the bacterium to produce the exopolysaccharide alginate, a key component of *P.* aeruginosa biofilms in this environment. Current understanding of alginate biosynthesis, polymerization and export indicates the involvement of thirteen proteins, twelve of which are encoded by genes located on the *algD* operon (17,18). Two of the genes encoded in the operon, *algA* and *algD*, are involved in alginate precursor synthesis. The other ten proteins are hypothesized to form a complex through which alginate is biosynthesized and secreted (19). Alginate is initially produced as a negatively charged polymer of β-1,4-linked D-mannuronate. Post-polymerization, alginate undergoes two types of modification as it is exported through the periplasm. The C5-epimerase AlgG converts selected mannuronate residues to L-guluronate (20,21), which increases the viscosity of the
polymer and hence of the biofilm (22), while the concerted actions of AlgI, AlgJ and AlgF are responsible for the selective O-acetylation of mannuronate residues (23-25). Acetylation of the hydroxyl groups can occur at the C2 and/or C3 positions and alters biofilm architecture, augments bacterial resistance to antibiotics and opsonic phagocytosis, enhances microcolony formation, and improves biofilm adhesion (10, 26, 27). Currently, it is not known whether epimerization occurs prior to or after acetylation, but varying amounts of both modifications are known to occur on the same exopolysaccharide strand (28).

The periplasmic protein AlgX, encoded by the algD operon, has been proposed to have a number of roles in the alginate biosynthetic machinery. Previous work has demonstrated that AlgX protects alginate from degradation as the polymer traverses the periplasm (29). AlgX has also been shown to have at least two interaction partners in the periplasm. It interacts with AlgK, another essential component of the biosynthetic machinery and a protein product which is also encoded by the algD operon (30), as well as the periplasmic protease MucD (30, 31), a protein that primarily acts as a negative regulator of alginate production. Although the AlgX-MucD interaction is intriguing from a regulatory/feedback possibility, the functional basis for the interaction has not yet been determined (30-32) and interestingly, ΔmucD mutants actually have increased alginate production when compared to the wild type strain (32, 33). Despite the work that has been carried out on AlgX, its exact role has yet to be defined. Bioinformatics analyses suggest that AlgX is a two-domain protein, with an N-terminal region that is predicted to share structural homology with members of the SGNH hydrolase superfamily (34). This region of the protein also shares ~69% similarity and 30% identity with AlgJ, the putative alginate acetyltransferase (24) and has therefore been proposed to have a similar function (19). The C-terminal domain has no apparent sequence homology to any known protein (34). To determine the role of AlgX in alginate biosynthesis, we have determined its structure to 2.15 Å resolution. The structure reveals that the protein has bilobal architecture with an N-terminal SGNH hydrolase-like domain and a C-terminal carbohydrate-binding module (CBM). Although

the topology and some features of the N-terminal domain are different to those found in a canonical SGNH hydrolase domain, and no enzymatic activity has previously been ascribed to AlgX, this domain contains the Ser-His-Asp triad that is typically found in members of this enzyme superfamily. Site-specific mutation in vivo of the Ser-His-Asp triad and other key residues in the putative active site results in either non-acetylated or acetylation-reduced alginate polymer, respectively. In vitro analysis demonstrates that the protein exhibits acetyl esterase activity and that the Ser-His-Asp triad is required for this activity. Taken together our data suggest that AlgX plays two roles in alginate biosynthesis: not only does the protein protect the polymer as it passages through the periplasm (29) but it also plays a role in its acetylation, an activity that is controlled by the Ser-His-Asp catalytic triad. In addition, the structure of AlgX provides insight into the function of AlgJ and other polysaccharide acetyltransferases involved in the acetylation of biofilm polymers and other bacterial polysaccharides, such as cellulose and peptidoglycan, respectively.

### Experimental procedures

#### Bacterial Strains, Plasmids and Growth Media
The bacterial strains and plasmids used in this study are listed in Supplemental Table S1. For expression of native or selenomethionine incorporated protein, E. coli BL21-CodonPlus® (DE3)-RP or B834 Met⁻ cells transformed with pPLHJTWX were grown in rich medium (32 g tryptone, 20 g yeast extract and 5 g NaCl) at 37°C, or in M9 media according to the protocol of Lee et al. (35), respectively. DNA manipulations were performed in E. coli DH5α. E. coli DH5α and P. aeruginosa strains were either grown in LB medium or LPIA medium consisting of a 1:1 mixture of L-agar and *Pseudomonas* isolation agar at 37°C, unless otherwise stated. Where appropriate, antibiotics were added to media at the following concentrations: ampicillin at 100 μg mL⁻¹, carbenicillin at 200 μg mL⁻¹, chloramphenicol at 30 μg mL⁻¹, gentamicin at 15 μg mL⁻¹ for E. coli and 100 μg mL⁻¹ for P. aeruginosa, and kanamycin at 40 μg mL⁻¹ for E. coli.

DNA manipulations - The Gateway® recombinase-based multi-site cloning system (Invitrogen) was used as described by the manufacturer. PCR amplification was carried out using the high-fidelity DNA Polymerase, PfuTurbo® (Stratagene), and the primers were designed to contain appropriate attB sequences for recombinational cloning into the relevant Gateway vector (Supplemental Table S1). *P. aeruginosa* PAO1 genomic sequences for primer design were obtained from the Pseudomonas Genome Database (36). Restriction enzymes were from New England Biolabs. Plasmid DNA was extracted from *E. coli* using the Qiaprep Spin Miniprep kit (Qiagen). DNA fragments for cloning were purified from agarose gels using the QIAEX II Gel Extraction System (Qiagen). Sequences of cloned DNA were determined commercially (ACGT, Inc., Wheeling, IL). Plasmids carrying an origin of transfer (*oriT*) were transferred from *E. coli* to *P. aeruginosa* by triparental mating as described (37) using the conjugal helper plasmid pRK2013, and selection for *P. aeruginosa* was on LPIA with appropriate antibiotics.

The AlgX expression plasmid pPLHJTWX (34) was used to generate point mutants of Asp174, His176 and Ser269 using the QuikChange Site-Directed mutagenesis lightning kit (Agilent Technologies). In each case the native residue was substituted with alanine. Each construct was verified by DNA sequencing (ACGT DNA Technologies Corp., Toronto, ON).

Protein Expression and Purification of AlgX - The expression and purification of recombinant C-terminally His₆-tagged, codon-optimized AlgX (AlgX-His₆) for structural studies was performed as described previously (34) with minor modifications. In brief, bacterial cell pellets were thawed and resuspended in Buffer A (500 mM NaCl, 50 mM Tris-HCl pH 7.8, 2% (v/v) glycerol, 2 mM dithiothreitol) containing 1 mg mL⁻¹ of lysozyme. This suspension was incubated at 4 °C for 30 min prior to homogenization with an Avestin Eumulsiflex C3 homogenizer (3 passages at 15,000 PSI). Unlysed cells were removed from the lysate by centrifugation (5,000 × g for 15 min at 4°C) and the resultant supernatant was applied to a 5 mL Ni²⁺-NTA Superflow affinity column (Qiagen) pre-equilibrated with Buffer A. The column was washed extensively with Buffer A to remove contaminating proteins that did not bind to the affinity resin. This wash step was continued until the absorbance at 280 nm returned to zero. Bound protein was eluted from the column using a 0–150 mM imidazole (in Buffer A) gradient. The fractions containing AlgX were pooled and concentrated by Amicon ultrafiltration using a 30 kDa molecular weight cutoff (MWCO) concentrator (Millipore). The protein was further purified, and buffer exchanged into Buffer B (50 mM Tris-HCl pH 7.8, 2% (v/v) glycerol, 2 mM dithiothreitol) by size-exclusion chromatography using a HiLoad 16/60 Supradex 200 prep-grade gel filtration column (GE Healthcare).

Protein used to determine the crystal structure was expressed as a selenomethionine incorporated (SeMet) version of AlgX according to the protocol of Lee et al. (35), and purified as described above. Wild-type and the single site-directed mutants used for in vitro enzymatic analysis were expressed in rich medium at 18 °C and purified as described above with the following modification: no dithiothreitol was added to the buffers used. Prior to assessing the folding of the wild-type and mutant variants using CD spectroscopy, the protein was dialyzed into buffer C (20 mM HEPES pH 7.8). Experiments were performed with 6 μM protein in buffer C at room temperature on a Jasco J-810 spectrometer (Jasco Spectroscopic Company, Japan) and repeated in triplicate.

Crystallization and Structure Determination of AlgX - SeMet AlgX was concentrated to ~8-10 mg mL⁻¹ by Amicon ultrafiltration (30 kDa MWCO) prior to crystallization. Diffraction quality crystals of SeMet AlgX were grown by optimizing the conditions found previously for the wild-type sulphur containing protein (100 mM sodium citrate pH 5.5, 20% (w/v) PEG 3000) (34). Crystals were grown with 13-16% (w/v) PEG 3000, at temperatures of 18 – 20 °C with a pH between 5 and 5.3 with successive rounds of seeding as outlined previously (34). Crystals were soaked for 2-3 min in cryoprotectant, crystallization buffer supplemented with 25% (v/v) glycerol, prior to vitrification in liquid nitrogen (34).

In an attempt to determine an AlgX-alginate complex, the native sulphur-containing protein was co-crystallized in the presence of synthetic
tetramannurate (38). This tetramer has an acetylated ethylamine linker from the synthesis still attached at the reducing end. The tetramer was added to the protein solution (8 mg mL⁻¹) such that the tetramer had a final concentration of 1 mM. Hanging-drop vapour diffusion crystallization trials were subsequently set up using 2 μL of the protein-substrate mixture and 2 μL of mother liquor. Crystals from the co-crystallization experiments were prepared for data collection as outlined by Weadge et al. (34) with no synthetic substrate present during the cryoprotection and flash-freezing processes.

All data were collected at Station X29 of the National Synchrotron Light Source. A 0.16 mm collimator was used to collect a total of 360 images of 1° Δφ oscillations on an ADSC Quantum-315 detector with an exposure time of 0.5 s per image. SeMet and native data were integrated, reduced and scaled using d*TREK and HKL2000, respectively (Table 1). To determine the phases of the SeMet data, the positions of all ten expected selenium atoms (five seleniums per molecule) were located using HKL2MAP (39), and density modified phases were calculated using SOLVE/RESOLVE (40). The resulting electron density map was of sufficient quality to allow automatic model building using PHENIX (41) followed by manual model building with COOT (42). Refinement was carried out using PHENIX.REFINE (41) and progress was assessed by monitoring the reduction (and convergence) of the R_work and R_free. When R_work reached 25%, waters and ligands were added and TLS groups as determined by the TLSMD web server (43, 44) were applied. The native structure was solved by molecular replacement with PHASER (45) from the PHENIX package using molecule A of the SeMet-SAD structure as a search model. Refinement of the native structure was as described for the SeMet model. The refinement statistics are given in Table 1. DSSP was used to assign secondary structure elements of AlgX (46, 47). All figures relating to the structure of AlgX were created in PyMol (The PyMol Molecular Graphics System, version 1.2r3pre, Schrödinger, LLC). Surface representations showing electrostatics and residue conservation include all residue sidechains, including those not

present in the PDB deposition that could not be modelled due to lack of suitable density.

Transformation of *P. aeruginosa* - Competent cells were prepared by growing a culture of PAO1 to OD₆₀₀ 0.200. The flask was chilled on ice for 10 min. A cold centrifuge tube was filled with the culture and centrifuged at 7,649 x g for 10 min at 5 °C. The supernatant was discarded, and the pellet was resuspended in 10 mL ice-cold 150 mM MgCl₂ and centrifuged at 7,649 x g for 10 min. The pellet was again resuspended in 10 mL cold 150 mM MgCl₂ and incubated on ice for 60 min. The sample was then centrifuged at 7,649 x g for 10 min at 5 °C, and the pellet was resuspended in 1.0 mL cold 150 mM MgCl₂, on ice. To increase competency ~10-fold, the cell suspension was stored on ice overnight. Competent cells were stored for ~2 months by adding 70 μL DMSO (7% final concentration) and freezing aliquots at -80 °C. For transformations, plasmid DNA (1-5 μL) was added to 100 μL competent cells and incubated on ice for 60 min, followed by a 60 s heat shock at 42 °C. The sample was transferred to 2 mL LB broth and allowed to shake with aeration at 37 °C for 60 min for expression of selective markers. Aliquots (50-100 μL) were spread onto selective agar and incubated at 37 °C overnight.

Construction of PAO1 *algX::Gm*<sup>R</sup> - A non-polar *algX::Gm*<sup>R</sup> mutant of PAO1 was constructed using the Gateway cloning system (Invitrogen). Three entry clones were created and inserts sequenced for verification. The first was ~1 kb of the 5′ region of *algX* (amplified with primers attB4-5′X-F and attB1r-5′X-R [Supplemental Table S1]), in pDONR-P4-P1R. The second was the gentamicin resistance cassette (*Gm*<sup>R</sup>) from pUCGM (amplified with primers attB1-Gm-F and attB2-Gm-R [Supplemental Table S1]), in pDONR-221. Finally, ~1 kb of the 3′ region of *algX* (amplified with primers attB2r-3′X-F and attB3-3′X-R [Supplemental Table S1]) was cloned into pDONR-P2R-P3. The three cloned fragments were assembled in the correct order in the multisite destination and gene replacement vector, pEX183-R4R3, constructed for this study by amplification of ~2.3 kb of the multisite destination vector pDEST-R4R3 (primers NsiI-M13F and Nsi-M13R [Supplemental Table S1]), containing attR4-ccdB-
Cm<sup>R</sup>-attR3, and cloning it into the NsiI site of pEX100T. Competent ccdB-resistant E. coli cells (Invitrogen) were transformed with pEX183-R4R3 containing the cloned fragments, and cells were screened for chloramphenicol resistance. One clone (pEX205-algX::Gm<sup>R</sup>) was transferred to the chromosome of PAO1 by triparental mating with selection on LPIA agar containing carbenicillin to obtain single crossover recombinants, which were plated again on carbenicillin. Double recombinants were selected for on LPIA-sucrose (5%) agar and Gm<sup>R</sup>Cb<sup>S</sup> phenotypes were verified.

Replacement of chromosomal algX with algX* mutant alleles—Chromosomal algX was replaced with algX* site-directed mutants using the Gateway cloning system (Invitrogen). An entry vector was created with a 1.4 kb amplicon of algX, including the Shine-Delgarno sequence and stop codon (amplified with primers attB1-SDalgX-F and attB2-SDalgX-R [Supplemental Table S1]), in pDONR221. A positive clone was subjected to site-directed mutagenesis using the QuikChange Lightning kit (Agilent) and mutagenic primers (Supplemental Table S1). Residues chosen for mutagenesis were selected in order to mutate the wild-type amino acid to alanine, using the codons GCC or GCG. All clones were sequenced. The algX wild-type and mutant algX* clones were transferred to destination vector pEXApGW, which is derived from the gene replacement vector pEX100T (48). The pEXApGW-algX clones were transferred to the chromosome of PAO1 algX::Gm<sup>R</sup> by triparental mating, with selection on LPIA agar containing carbenicillin. Double recombinants were selected on LPIA-sucrose agar and Gm<sup>R</sup>Cb<sup>S</sup> phenotypes were confirmed.

Activation of alginate production in PAO1 strains—The generalized transducing phage F116L (49) was used to infect PDO351 (50), a mucA::Gm<sup>R</sup> strain, by incubating 10 μL F116L lysate (at various dilutions) with 0.2 mL of PDO351 (mid-log growth in LB broth) in the presence of 10 mM MgSO<sub>4</sub> and 5 mM CaCl<sub>2</sub>. The samples were incubated at room temperature for 10 min and then molten top agar at 50°C (3 mL LB broth with 0.7% Noble agar) was added to each tube, mixed and immediately poured onto the surface of an L-agar plate. Plates were incubated for 24 h at 37°C.

LB broth (3 mL) was added to the plate showing the best confluent plaques, and the top agar was broken up and the sample centrifuged for 10 min at 7,649 × g. The supernatant was passed through a 0.45 μm filter and 0.1 mL of the filtrate was mixed with 0.2 mL P. aeruginosa PAO1 algX* mutants (grown to mid-log in LB broth). The samples were incubated at ambient temperature for 15 min and then placed at 37°C for 1 h, shaking at 225 rpm. The samples were plated on LPIA + gentamicin and incubated at 37°C overnight.

Construction of an algX+ expression and complementation plasmid—A pBAD-araC controlled expression Gateway plasmid, pBAD-DEST49 (Invitrogen) permits arabinose-inducible expression and is compatible with cloning the algX+ fragment in pDONR199-algX via the LR reaction. To allow pBAD-DEST49 to replicate in P. aeruginosa and complement chromosomal mutations in trans, the mini-stabilization fragment (mSF) in pSS124 was amplified by PCR (using primers NdeI-pUC19MCS-F and NdeI-pUC19MCS-R [Supplemental Table S1]) and initially cloned blunt-ended with T4 DNA ligase (New England Biolabs) into the SmaI site of pLG339; this was detected by insertional inactivation of its Kan<sup>R</sup> gene and called pLG369-mSF. From there the mSF was removed by NdeI digestion and ligated into the single NdeI site of pBAD-DEST49. The ligation mix was transformed directly into competent PAO1 cells to select for mSF+ clones, and one was named pBAD506-SF. The algX+ fragment in pDONR199-algX was transferred to pBAD506-SF in a LR reaction, and one clone, pBAD510-algX+, was transferred to the mucA23 mutant PDO300 algX::Gm<sup>R</sup>.

Enzyme assay—All enzyme assays were performed at least in triplicate, in a 96-well microtiter plate, using a SpectraMax M2 (Molecular Devices, Sunnyvale, CA). Standard reactions contained 3.0 mM 3-carboxyumbelliferyl acetate, dissolved in DMSO, and 40 μg of AlgX in 100 μL of 50 mM sodium HEPES buffer (pH 8.0) at 25°C. The final DMSO concentration did not exceed 10% (v/v). Reactions were initiated by the addition of substrate and were monitored in real time for 10 min, using an excitation of 386 nm and an emission of 447 nm as previously described (51).

Background hydrolysis rates, in the absence of enzyme, were also monitored and subtracted from the enzyme-catalyzed reaction. A calibration curve for 7-hydroxycourmarin-3-carboxylic acid, the fluorescent hydrolysis product of 3-carboxyumbelliferyl acetate, was obtained under the reaction conditions and used to calculate reaction rate. The protein concentration of each variant was determined using the Pierce® BCA Protein Assay Kit (Thermo Scientific, Rockford, IL).

Sequence alignments, residue conservation and homology searches - Homologues of AlgX were identified using BLAST and sequence alignments were carried out using TCOFFEE (52, 53). Conservation of residues was determined by ConSurf (54, 55). Sequence identity and similarity scores were determined using FASTA SSEARCH (56). The SCOP Superfamily database was searched to identify similar fold families to the N-terminal domain of AlgX (57). The CAZYmes Analysis Toolkit (CAT) was used to carry out a Sequence Similarity Based Annotation of the AlgX C-terminal domain, to determine whether the AlgX CBM aligned with any of the families currently in the CAZY database (58).

Alginate purification, concentration - P. aeruginosa PAO1 *mucA::Gm<sup>R</sup> algX+* or its mutant strains were grown overnight at 37 °C in MAP medium supplemented with gentamicin. The cells were removed by centrifugation at 10,000 x g for 1 h. The concentration of alginate in the supernatants was determined using a modification of the carbazole assay of Knutson and Jeanes (59). Briefly, 30 μL of purified alginate was layered on top of 1 mL of ice-cold borate-sulfuric acid reagent (100 mM H<sub>3</sub>BO<sub>4</sub> in concentrated H<sub>2</sub>SO<sub>4</sub>). The samples were vortexed for 4 s, and placed on ice. An equal volume (30 μL) of carbazole solution (0.1% [w/v] in ethanol) was added. The mixture was heated to 55 °C for 30 min and then cooled on ice. The alginate concentration was determined spectrophotometrically at 530 nm with *Macrocystis pyrifera* alginate (Sigma) as a standard.

Acetylation assay - A chemical method described by Hestrin (60) was used to measure alginate acetylation. Briefly, 250 μL of an alginate solution was incubated with 250 μL of alkaline hydroxylamine (35 mM NH<sub>2</sub>OH, 75 mM NaOH) for 10 min at 25 °C. The reaction mixture was acidified with 250 μL of 1.0 M perchloric acid, followed by the addition of 250 μL of 70 mM ferric perchlorate (10 mL 0.5 M perchloric acid, 0.248g iron perchlorate). The concentration of acetyl groups was determined spectrophotometrically at 500 nm from a standard curve, with ethyl acetate as the substrate (10 mM acetyls = 192 μL ethyl acetate in 100 mL H<sub>2</sub>O).

NMR Analysis of alginate - Alginate for characterization by <sup>1</sup>H NMR spectroscopy was harvested from cultures grown in LB medium at 28 °C for 72 h. The cultures were diluted two-fold with 145 mM NaCl. The cells were pelleted by centrifugation and the supernatant retained. The supernatant was mixed with an equal volume of isopropanol to precipitate alginate, which was harvested by centrifugation. The alginate was dissolved in 145 mM NaCl, and precipitated and harvested as before. The procedure was repeated two more times, dissolving the alginate in 1 M NaCl, and then again in 145 mM NaCl. The final solution of alginate was incubated with 10 mg L<sup>-1</sup> Pronase for 1 h prior to precipitation. The purified alginate was dissolved in H<sub>2</sub>O and dialyzed against several changes of H<sub>2</sub>O, in 6,000 - 8,000 MWCO dialysis tubing.

The alginate for <sup>1</sup>H NMR analysis was prepared by adjusting 5 - 10 mg samples to pH 3.5 and heating to reflux for 1 h. The samples were cooled, and the pH was adjusted to 5.5. Each sample was lyophilized and then dissolved in 0.57 mL D<sub>2</sub>O containing 2 mM EDTA and 10% d<sub>6</sub>-DMSO. Spectra were acquired at 80 °C using a pulse sequence to suppress the water peak. Peak assignments for mannuronate and guluronate residues were based on the published values (61, 62).

Alginate Epimerization Assay - The relative extent of epimerization was measured using the G-lyase assay as previously described by Chitnis and Ohman (63) and Douthit et al. (64). This assay measures the relative number of guluronate (G) residues per gram of alginate by assaying the abundance of cleaved, unsaturated residues present after treatment with a guluronate specific lyase.

Briefly, alginate (65 μg) in 150 μL water was deacetylated with 65 μL of 1M NaOH for 15 min at 65 °C and then neutralized with 50 μL 1M HCl and 100 μL of lyase buffer (50 mM Tris-HCl, pH 8.0, 10 mM MgCl₂·6H₂O, 5.0 mM CaCl₂). Alginate G-lyase (10 μL) from *Klebsiella aerogenes* was added to the alginate and the samples incubated for 1 h at 25 °C. To assay for unsaturated residues, periodic acid (250 μL 0.2 M) was added and the samples were incubated at 25 °C for 40 min. After addition of sodium arsenite (100 μL of 2% in 0.5N HCl) for 1 min at 25 °C, thiobarbiturate (1 mL of 0.6% solution, pH 2.0) was added and the reactions were incubated at 65 °C for 30 min. The samples were cooled for 1 h at room temperature and centrifuged for 5 min to remove precipitate. Guluronate residue abundances were measured spectrophotometrically at 535 nm and the results compared to the value obtained with PAO1 *mucA::Gm^R algX+* alginate.

Antibody generation and Western blot analysis - AlgX was expressed and purified as described above, and used to generate antiserum from rabbits using a 70d standard protocol (Cedarlane). Prior to use, the polyclonal antisera were purified as described by Salamitou et al. (65).

PAO1 *mucA::Gm^R algX** strains were grown to OD~600~ ~1.0 in LB-no salt medium. Aliquots of 1.5 mL were removed and the cells harvested. The cells were resuspended in 2 x Laemmli buffer (~100 μL, depending upon exact OD) and heated to 100 °C for 5 min. Equal quantities of cell lysate were loaded and separated on 4% stacking, 10% resolving polyacrylamide gels prior to being transferred to polyvinylidene difluoride (PVDF). The proteins of interest were detected using rabbit polyclonal antibodies to AlgX (1:1000) or PilF (1:1000) (66). The secondary goat anti-rabbit antibody conjugated to alkaline phosphatase was used as per manufacturer's instructions and the blots developed with 5-bromo-4-chloro-3'-indolyphosphate p-toluidine and nitroblue tetrazolium chloride (BioShop).

RESULTS

AlgX is a two-domain protein - To probe the function and role of AlgX in alginate biosynthesis, we have determined its structure to 2.15 Å resolution. SeMet-incorporated protein was expressed, purified and crystallized and the structure determined using the single-wavelength anomalous dispersion (SAD) method. The SAD model, in conjunction with the molecular replacement technique, was subsequently used to determine the structure of the native (sulphur containing) protein that had been crystallized in the presence of a synthetic tetramer of polymannuronate (34,38) (Fig. 1). The presence of the alginate polymer likely explains the differences in unit cell dimensions and space group observed between the native and SeMet crystals (Table 1). Native AlgX crystallized in space group P2₁2₁2, with two molecules (molecules A and B) in the asymmetric unit. Extensive rounds of model building and refinement yielded models with good geometry, and an R_work and R_free of 17.6% and 23.1%, respectively (Table 1). Analysis using size-exclusion chromatography and dynamic light scattering (data not shown) indicates that AlgX is a monomer in solution, suggesting that the dimerization observed in the crystal is not biologically relevant. The synthetic tetramer was not visible in the final electron density maps, although a region of positive density was observed in the active site of molecule A that could be best fitted with citrate. While positive density was located in the active site of molecule B we were unable to model this density satisfactorily.

The structure of AlgX revealed a bilobal architecture, comprised of an N-terminal hydrolase domain with α/β/α topology followed by a C-terminal carbohydrate-binding module (CBM) domain with a β-sandwich jelly roll fold (Fig. 1). In molecule A, residues 247 to 258 and residue 447 were unable to be modelled due to the quality of the electron density in this region. Similarly, residues 41, 250, 251 and 447 of molecule B were unable to be modelled. The root mean square deviation (rmsd) between the two molecules in the asymmetric unit is 0.67 Å over 409 equivalent Ca positions. As molecule B is the most complete model, this model has been used in the structural analysis presented below. Two disulphide bonds are present in the structure. The first, between Cys44 and Cys229, positions two long β-strands, β4 and β5, alongside the N-terminal domain. The second, between Cys347 and Cys460, appears to fix the relative orientation of the two domains with respect to each other (Fig. 1A).

Examination of the surface potential of AlgX reveals that the surface has areas of both positive and negative electrostatic potential. However, there are two distinct regions of electropositive charge, one in each domain, that lie on the same face of the protein and overlap with regions of high sequence conservation, including critical residues identified for binding and/or catalysis in both the N-terminal and C-terminal domains (Fig. 2). As alginate is negatively charged (67), it is anticipated that the polymer may associate with AlgX via these positively-charged, highly conserved regions. Structural homology searches revealed no proteins in the FATCAT or DALI databases that resemble the entire AlgX protein. Structural homologues to each of the domains, separately, were identified, which makes the combination of the two domains found in AlgX unique amongst structures solved to date.

C-terminal domain is a carbohydrate-binding module - The C-terminal CBM domain of AlgX (residues 348-474) adopts a β-sandwich jelly roll fold, with each β-sheet consisting of four anti-parallel β-strands (Figs. 1B, 3A). This fold is the most common CBM fold found to date (68-70) and a structural homology search of the Protein Databank using FATCAT (71) revealed that this domain of AlgX is similar to other known and putative CBMs with the highest homology being to: (i) the C-terminal galactose-binding-like domain of alkaline serine protease, KP-43, from Bacillus sp. KSM-KP43 (PDB ID 1WMD; rmsd 3.06 Å over 99 equivalent Ca positions), (ii) CBM29-2 of the non-catalytic protein 1 from Piromyces equi (PDB ID 1GWL; rmsd 3.14 Å over 97 equivalent Ca positions) and (iii) the C-terminal β-sandwich domain of Thermatoga maritima β-fructosidase (PDB ID 1UYP; rmsd 3.13 Å over 92 equivalent Ca positions). Sequence alignments of P. aeruginosa AlgX with homologues from other Pseudomonas species (P. syringae, P. fluorescens, P. putida, P. brassicacearum, P. mendocina, P. alkylphenolia, P. entomophila) and the alginate-producing bacterium Azotobacter vinelandii reveal that conservation across the CBM is high, with 18% identity and 57% similarity across all nine species (Figs. 2B & C). As alginate is a linear polysaccharide, we predict that this domain will be a Type B CBM. Characteristic features of this type

of CBM include the presence of aromatic amino acids and conserved polar residues located in a groove on the surface of the protein that form a distinctive “pinch point,” through which the polysaccharide chain passes (69,70). The aromatic residues have been shown to stack with the rings of the sugar thereby stabilizing the polysaccharide-protein complex, and are important in determining specificity for particular polysaccharides. The polar residues form direct hydrogen bonds with the polymer chain to increase the stability of the complex (69,70). The pinch point of the AlgX CBM is formed by several residues, primarily a surface-exposed tryptophan, Trp400 (Figs. 3B, 3C), which is 100% conserved across the aligned Pseudomonas spp. and A. vinelandii (data not shown). Within the pinch point there is a surface-exposed polar residue, Thr398, which is highly conserved among alginate-producing species (Figs. 3B & C). There are also two positive residues that may aid in binding alginate, Arg406 and Arg364, which are located beside Trp400 and on the opposite side of the groove, respectively (Fig. 3C). Arg406 is conserved in charge, with all aligned alginate-producing species containing either an arginine or a histidine at this position (Figs. 3B & C). Arg364 is less conserved, but in all aligned species other than A. vinelandii, a basic or polar residue is found at this position (Figs. 3B & C). Two lysine residues, Lys396 and Lys410, are positioned just outside the pinch point (Figs. 3B & C). It is postulated that these positively-charged residues could be important for directing the polymer along the face of the CBM towards the N-terminal hydrolase domain. Lys410 is conserved in all aligned species except A. vinelandii and Lys396 is conserved in charge except in A. vinelandii and P. mendocina. Superposition of the CBM domain with P. equi CBM29-2 in complex with mannohexaose supports the hypothesis that this domain of AlgX may bind alginate, as the superposed mannohexaose passes through the AlgX pinch point and along the conserved path without the necessity for structural rearrangements (Figs. 3B, 3C).

N-terminal domain has an SGNH hydrolase-like core - The N-terminal hydrolase domain of AlgX (residues 1-347) has α/β/α topology. Structural homology searches using FATCAT (71) revealed
that the core of the domain shares features with proteins from the SGNH (GDSL) hydrolase superfamily (Fig. 4A), including (i) *Enterococcus faecalis* acyl hydrolase/lipase (PDB ID 1YZF; rmsd 3.11 Å over 127 equivalent Ca positions); (ii) an isoamyl acetate-hydrolyzing esterase from *Saccharomyces cerevisiae* (72) (PDB ID 3MIL, rmsd 3.05 Å over 100 equivalent Ca positions); and (iii) platelet-activating factor acetyl hydrolase Ib gamma subunit (73) (PDB ID 1ES9; rmsd 3.15 Å over 88 equivalent Ca positions). Structurally, the core of the N-terminal AlgX domain has a number of differences relative to the canonical SGNH hydrolase fold, as exemplified by the acyl hydrolase/lipase from *E. faecalis* (PDB ID 1YZF) (Fig. 4). Typically, SGNH hydrolases contain a five-stranded parallel β-sheet, surrounded by seven α-helices. Based on secondary structure assignment (46, 47), AlgX has the seven core α-helices seen in members of the SGNH fold family, helices α2, α5, α6, α8, α9, α10 and α11, but contains only the first four strands of the five-stranded parallel β-sheet. The fifth strand is replaced by a one-residue β-bridge (Trp153). Although AlgX structurally resembles SGNH hydrolases, examination of the topology reveals that the arrangement of the secondary structural elements found in the N-terminal domain is different to that of members of the SGNH hydrolase superfamily (Fig. 4B). In particular, the four core β-strands of AlgX are in a different order when compared to the arrangement (from the N- to the C-terminus) of canonical SGNH hydrolases (Fig. 4B). In addition to the variation in the topology of the α/β/α domain, AlgX also contains a number of insertions not found in a canonical SGNH hydrolase fold. These include: (1) two long, antiparallel β-strands on the side of the core hydrolase domain, strands β4 and β5 and (2) a ‘cap’ over the central part of the core domain formed by two short antiparallel β-strands, β1 and β2, and a series of short α-helices, helices α1, α3, α4 and α7. The structure also contains ten 3<sub>10</sub> helices, labelled t1 to t10.

Typically, SGNH hydrolases contain a Ser-His/Asp/Glu catalytic triad, a conserved glycine and a conserved asparagine residue located in the core central β-sheet (74). The Ser-Gly-Asn-His residues, for which this family is named, are found

in four conserved blocks, blocks I, II, III and V, respectively (Table 2). In block I the serine residue is part of the GDSL consensus sequence (letters in bold are expected to be 100% conserved) and serves not only as the nucleophile, but also as a proton donor in the oxyanion hole. The glycine and asparagine residues in blocks II (NxSxxGxT) and III (GxND), respectively, serve as proton donors in the oxyanion hole, while block V (DxxHP) contains a highly conserved histidine that acts as the catalytic base, deprotonating the hydroxyl group of the serine to make it more nucleophilic. Block V also typically contains an aspartate or glutamate three residues upstream of the histidine, which is the third member of the catalytic triad and is positioned to form a salt bridge with the imidazolium ring of the conserved histidine. Although not found in any of the predicted blocks, the SGNH-associated catalytic triad and the key Gly residue are structurally conserved in AlgX (Table 2). Superposition of Asp174, His176, Ser269 and Gly296 residues in AlgX with the catalytic triad and the Gly residue of the oxyanion hole in SGNH superfamily member *E. faecalis* acylhydrolase /lipase was performed (Fig. 5A). The superposition (RMSD of 0.60 Å across all equivalent Ca) confirms the presence and orientation of a Ser-His-Asp triad in AlgX, comprised of residues Ser269, His176 and Asp174, respectively (Figs. 5A, 5B). The spatial positioning of the catalytic centre is similar to that of serine esterases (75) and serine proteases (76). The superposition also indicates that Gly296 is positioned and oriented correctly to participate in oxyanion hole formation in AlgX. Residues Tyr328 (AlgX) and Asn78 (*E. faecalis* acylhydrolase/lipase) were not included in the superposition, but the alignment suggests that they occupy the same relative positions in the respective proteins (Fig. 5A). Thus it appears that the conserved asparagine found in block III has been replaced by Tyr328 in AlgX. AlgX is not the only protein with an SGNH or SGNH-like hydrolase fold that lacks the conserved asparagine, as the putative acetylxylan esterase from *C. acetyobutylicum* has a Ser residue in this position (PDB 1ZMB) (Fig. 4A). In addition, while the conserved aspartic acid in SGNH hydrolases is typically found three residues upstream of the catalytic histidine, in AlgX, Asp174 and His176
are only two residues apart. This is not unprecedented as the N-terminal SGNH hydrolase domain of a cellulase from *Cellvibrio japonicus* (CtCel5C-CE2, PDB ID 2WAA) has the same Asp-X-His arrangement (77). Like AlgX, CtCel5C-CE2 is also a two-domain protein with a N-terminal cellulase domain, and C-terminal CBM with non-catalytic cellulose-binding function.

Sequence alignments of the N-terminal domain of *P. aeruginosa* PAO1 AlgX with seven other *Pseudomonas* spp. (*P. syringae*, *P. fluorescens*, *P. putida*, *P. brassicacearum*, *P. mendocina*, *P. alkylphenolia*, *P. entomophila*) and *A. vinelandii* reveal that the residues in this domain share 33% identity, and 65% similarity (Fig. 2C). The catalytic triad and Gly residues, as well as the tyrosine that replaces the asparagine are 100% conserved across these species. The AlgX N-terminal domain also has 30% sequence identity and ~69% sequence similarity to AlgJ, which has previously been shown to play a role in the O-acetylation of the alginate polymer (24). The SGNH-associated residues identified in AlgX are all conserved in *P. aeruginosa* AlgJ.

The Ser-His-Asp triad is essential for acetylation of alginate - The presence of SGNH-associated residues in AlgX and its sequence similarity to the putative alginate acetyltransferase AlgJ (24, 29) suggested that AlgX may have a role in alginate acetylation. To probe the in vivo function of AlgX, a series of alanine chromosomal point mutant strains were constructed in *P. aeruginosa* PAO1, by replacing a marked chromosomal copy of *algX* with plasmid-borne *algX* mutant alleles through homologous recombination. Alginate production in these strains was then constitutively turned on by disruption of *mucA*. *mucA* encodes an anti-sigma factor and its disruption leads to activation of the *algD* operon and alginate production (33). In addition to substitutions in the Ser-His-Asp triad, mutations were also made in seven residues in close proximity to the putative active site that could play a role in alginate binding (Fig. 5C). These individual mutant AlgX strains were then tested for their ability to produce polymeric alginate. The level of AlgX expression was also assessed to ensure that any variations in alginate content were not due to reduced levels of protein production. All of the mutant strains produced

alginate at a level equivalent to that of the *P. aeruginosa* PAO1 *mucA::Gm**R***algX+** strain and had comparable levels of AlgX expression (Figs. 6A, 6D). As all mutant strains expressed alginate, the polymer was further analyzed to determine whether its acetylation state had been altered. Using a spectrophotometric assay and NMR analysis, it was found that the alginate produced by each of the S269A, H176A and D174A AlgX mutant strains was not acetylated (Figs. 6B, 6E). When a plasmid expressing *algX* was transformed into these three mutants, the acetylation of alginate was restored (data not shown). Alginate acetylation was observed to be reduced by ~50% and ~40% for the Y328A and Y275A mutant strains, respectively (Fig. 6B). Mutation of residues, R115, H175, N270, D300 and S301 had no effect on either alginate production or its acetylation (Figs. 6A, 6B). To determine if the lack of acetylation affected the second post-polymerization modification that alginate undergoes, epimerization, the alginate polymers produced were deacetylated and incubated with alginate G-lyase. This enzyme is only able to cleave the alginate polymer if mannuronate residues within the polymer have been epimerized to guluronate. Analysis of the hydrolyzed alginate revealed that the level of epimerization in the mutant strains was equivalent to that observed in the wild-type *algX+* strain, regardless of whether the alginate was acetylated in vivo or not (Fig. 6C). This suggests that the lack of acetylation in the S269A, H176A and D174A mutant strains does not affect epimerization of the polymer.

Acetyltransferase activity has not yet been demonstrated in vitro. However, acetyl esterase activity (Fig. 7A & 7B) can be assayed using a pseudo substrate, 3-carboxyumbelliferyl acetate. Using a fluorometric assay the ability of the wild-type AlgX and D174A, H176A and S269A AlgX mutant proteins to remove the acetate group from this substrate was assessed. The specific activity of wild-type AlgX was found to be low (0.0068 ± 0.00057 μmol min⁻¹ mg⁻¹) (Fig 6F), although this was not unexpected as 3-carboxyumbelliferyl acetate has been substituted for the acetate donor that would be used in vivo, the identity of which is currently unknown. Furthermore, the inability to detect acetate release when AlgX was incubated with alginate and the low specific activity suggests
that AlgX is not an acetyl esterase, as much higher  
in vitro activity would have been expected, as has  
been observed for other esterases assayed with  
similar, commercially-available analogues (78).  
The acetyl esterase activity of each of the three  
mutant proteins was found to be close to zero. The  
D174A and H176A mutants had specific activities  
of 0.00032 ± 0.00028 μmol min⁻¹ mg⁻¹ and  
0.00027 ± 0.00027 μmol min⁻¹ mg⁻¹, respectively,  
while the activity was completely abolished in the  
S269A mutant (Fig. 6F). The catalytic activity of  
AlgX followed Michaelis-Menten kinetics (data  
not shown). CD spectroscopy indicates that the  
differences in catalytic activity observed are not  
due to structural changes in the protein variants,  
and that the mutant proteins are folded (data not  
shown). These data suggest that each of these  
residues is required for in vitro removal of an  
acetate group from a donor. In addition, the results  
corroborate our in vivo data, which indicate that  
the Ser-His-Asp catalytic triad is required for  
acetylation of alginate, and add weight to the  
suggestion that AlgX is an acetyltransferase.

DISCUSSION  
In this study, we have presented the crystal  
structure of AlgX and have demonstrated that this  
protein plays a role in the O-acetylation of alginate  
in vivo, and that a Ser-His-Asp triad governs this  
activity. Our in vitro enzymatic analyses also  
support the hypothesis that AlgX is catalytically  
active and functions as an acetyltransferase. AlgX  
contains two domains, an N-terminal SGNH  
hydrolase-like domain and a C-terminal  
carbohydrate-binding module. This combination is  
not unexpected as CBMs are commonly appended  
to enzymatic domains, including SGNH  
hydrolases, where they function to bring the  
relevant polysaccharide into close proximity to the  
active site (68, 79). The CBM domain of AlgX  
does not share sequence homology with any of the  
67 CBM families currently assigned in the  
Carbohydrate-Active Enzymes database (CAZy)  
(www.cazy.org), but it does have structural  
homology with other CBMs. Superposition of the  
CBM of AlgX and a cellulose-binding domain-  
mannohexaose complex indicates that the binding  
region of the AlgX CBM comprises a pinch point  

containing a conserved tryptophan, a threonine and  
two arginine residues, and that the proposed path  
for polymer binding includes two conserved  
lysines (Figs. 3B, 3C). Crystal contacts between  
Trp 400 and Leu 214 of neighbouring molecules  
occur near the pinch point region of the CBM.  
Since CBMs are often appended to catalytic  
domains to enhance substrate binding, our efforts  
to co-crystallize the protein with the substrate may  
have been hindered by these occluding crystal  
contacts. However, the arrangement of polar,  
positively charged and aromatic residues at the  
pinch point of the CBM is consistent with the  
hypothesis that this domain binds a single chain of  
alginate. Furthermore, the surface potential of  
AlgX is such that the negatively charged alginate  
chain could conceivably bind along a readily  
apparent path that joins the conserved residues of  
the pinch point in the CBM to a positively-charged  
region on the surface of the N-terminal domain,  
which is in close proximity to the active site  
residues (Fig. 2).  

The N-terminal domain of AlgX has an  
SGNH hydrolase-like fold with some additional  
structural features. Although the importance of  
these additional features is not known, it is  
believed that they are relevant for the function of  
AlgX as part of the alginate biosynthetic complex,  
as potential binding sites for the known interaction  
partners AlgK and MucD (30, 31) or other  
members of the acetylation machinery, AlgI/J/F.  
Members of the SGNH hydrolase superfamily are  
primarily esterases and lipases, although enzymes  
with transferase activity and dual-activity enzymes  
have also been identified (74, 80-85). The  
canonical description of the SGNH hydrolase fold  
includes the presence of particular residues, Ser,  
Gly, Asn, His and Asp/Glu, in conserved blocks  
(74). However, the superfamily now encompasses  
members with atypical or missing blocks,  
suggesting that sequence divergence has occurred  
within this superfamily, while the tertiary structure  
has been closely maintained. AlgX contains the  
Ser, His, Asp and Gly residues orientated and in  
the same tertiary positions as the equivalent  
residues in canonical SGNH hydrolases. However,  
these residues in AlgX do not follow the primary  
sequence conservation of the consensus blocks.  
The increasing number of proteins with SGNH  
hydrolase-like tertiary architecture that lack some  of the canonical residues suggests that the
definition of the SGNH hydrolase superfamily
may need to be broadened, or that sub-groups
within the superfamily need to be defined based on
activity, substrate preference and structure of the
protein.
AlgX residues Asp174, His176 and Ser269
comprise the catalytic triad common to SGNH
hydrolases, and we propose that these three
residues mediate transfer of an acetyl group to
alginate (Figs. 7, 8). Ser269 is believed to have
dual functionality, with a direct role in the catalytic
mechanism and also acting, with Gly296, to form
an oxyanion hole. The donated acetyl group sits in
the oxyanion hole, and the Ser and Gly residues
act to stabilize the negative charges on the acetyl
oxyanion during the formation of the tetrahedral
intermediates in the reaction (Fig. 7). In the
majority of SGNH hydrolases, a conserved
asparagine is also important for oxyanion hole
stability, but AlgX has Tyr328 located at this
position instead. Examination of the structure
suggests that it is perhaps more likely that Tyr328
as well as Tyr275, which when mutated to alanine
reduce acetylation by 40% and 50%, respectively,
are important for the correct positioning of the
alginate polymer across the putative active site.
These residues are located on either side of the
catalytic triad, and aromatic residues frequently
bind to sugar molecules via hydrophobic
interactions (70). If Tyr328 is involved in substrate
binding and not oxyanion stabilization, this
suggests that AlgX may use a chymotrypsin/
trypsin-like serine protease mechanism in which
the oxyanion hole is formed by just the serine and
glycine residues (76,86) (Fig. 7).
AlgX is the fourth protein in the algD operon
that has now been implicated in alginate
acetylation, as previous studies have suggested that
AlgI, AlgJ and AlgF are all essential for this
process to occur (23-25,87). AlgI and AlgJ are
proposed to be a membrane bound MBOAT family
O-acetyltransferase and a periplasmic O-
acetyltransferase, respectively (25,88). The
function of AlgF is currently not known. Franklin
and Ohman (24) showed that AlgJ and AlgX share
~69% similarity and 30% identity. It is predicted
that AlgJ is also an SGNH hydrolase-like protein,
but differs from AlgX in that AlgJ does not have a
CBM domain and is anchored to the inner
membrane by a single N-terminal transmembrane
helix (25). It is not apparent why four proteins are
required for alginate acetylation, and in particular
why two periplasmic acetyltransferases would be
needed. Homologues of AlgI and AlgJ have been
identified in a number of different bacteria (88),
including the peptidoglycan acetylation proteins
PatA and PatB in Neisseria gonorrhoeae (81) and
the bi-domain proteins OatA and OatB in Gram
positive bacteria, each of which have domains
resembling both AlgI and AlgJ (80). Homologues
of AlgF and AlgX are less common, but the
acetylation of cellulose during the formation of the
Pseudomonas fluorescens pellicle also requires
four proteins that are located together in the wss
operon (12,89). WssGHI are homologous to
AlgFIJ, with amino acid sequence identities of
24%, 46%, and 33%, respectively. WssF shares no
sequence homology with any of the Alg proteins,
but is structurally predicted to belong to the SGNH
hydrolase superfamily and has the three residues
comprising the catalytic triad as well as the Gly
residue required for oxyanion hole formation.
WssF may therefore have a role similar to that of
AlgX (12). Interestingly, like AlgX, examination
of the Phyre2 model of WssF (90) reveals that the
conserved Asn residue that is found in most SGNH
hydrolases is missing and is replaced with a Thr,
suggesting that the oxyanion hole of WssF is
formed by the Ser and Gly residues only. This
provides another example of an putative SGNH
hydrolase that may have a role in acetylation and
that uses the chymotrypsin/trypsin-like oxyanion
stabilization mechanism.
Phenotypic characterization of site-specific
mutants of both AlgX and AlgJ suggest that each
protein plays a role in the acetylation of the
nascent alginate polymer (88). AlgX and AlgJ are
predicted to be structurally similar to PatB and the
SGNH hydrolase domains of OatA and OatB
(80,81). Although, PatB acetyltransferase activity
has been demonstrated in vivo (81), like us, the
authors of this study were unable to develop an in
vitro assay to demonstrate full transferase activity.
Instead, comparable to our use of 3-
carboxyumbelliferyl acetate, Moynihan and Clarke
(81) used p-nitrophenyl acetate to characterize the
first part of the reaction in vitro and comparable to
our results found that PatB has only weak esterase
activity. OatA and OatB acetyltransferase activity
13
has also been shown in vivo (80). The functional equivalency and similarity of the predicted structures of PatB, OatA and OatB reinforce our hypothesis that AlgX and AlgJ are also acetyltransferases. The resemblance of alginate acetylation to the OatA/B and PatA-PatB systems, supports the theory that the integral inner membrane protein AlgI receives an acetyl group from a cytoplasmic acetyl donor. This is then passed to AlgJ and/or AlgX. These proteins may pass the acetyl group between themselves, or acetylate the nascent polymer. The role of AlgF has not been elucidated (Fig. 8), but is known through mutational analysis to be an important part of the acetylation process (23, 25). The alginate model can also be used to illustrate a potential method of cellulose acetylation by WssFGHI, in which the acetyl group is received from an acetyl donor and transported to the periplasm by WssH.

WssI and/or WssF receive the acetyl group, and one or both of the proteins transfer it to the growing cellulose chain (Fig. 8). Like AlgF, no role has been attributed to WssG in this model.

We have described the structure of AlgX, a two-domain protein that we propose binds to alginate via its C-terminal CBM and is then able to add acetyl groups to the nascent polymer through the catalytic triad in the N-terminal SGNH hydrolase-like domain. This is the first structure of a probable polysaccharide acetyltransferase with SGNH hydrolase-like architecture, which could serve as a prototype for other proteins that have been shown to be involved in the acetylation of peptidoglycan (12, 80, 81). The hypothesis that AlgX is an acetyltransferase, and the acetylation model presented provide avenues for future studies aimed at better understanding this key polysaccharide modification.

---

**REFERENCES**

1. Cuthbertson, L., Mainprize, I. L., Naismith, J. H., and Whitfield, C. (2009) Pivotal roles of the outer membrane polysaccharide export and polysaccharide copolymerase protein families in export of extracellular polysaccharides in gram-negative bacteria. *Microbiol. Mol. Biol. Rev.* **73**, 155-177

2. Flemming, H., and Wingender, J. (2010) The biofilm matrix. *Nat. Rev. Microbiol.* **8**, 623-633

3. Høiby, N., Bjarnsholt, T., Givskov, M., Molin, S., and Ciofu, O. (2010) Antibiotic resistance of bacterial biofilms. *Int. J. Antimicrob. Agents* **35**, 322-332

4. Ghafoor, A., Hay, I. D., and Rehm, B. H. (2011) Role of exopolysaccharides in *Pseudomonas aeruginosa* biofilm formation and architecture. *Appl. Environ. Microbiol.* **77**, 5238-5246

5. Vuong, C., Kocianova, S., Voyich, J. M., Yao, Y., Fischer, E. R., DeLeo, F. R., and Otto, M. (2004) A crucial role for exopolysaccharide modification in bacterial biofilm formation, immune evasion, and virulence. *J. Biol. Chem.* **279**, 54881-54886

6. Das, T., Sharma, P. K., Busscher, H. J., van der Mei, H. C., and Krom, B. P. (2010) Role of extracellular DNA in initial bacterial adhesion and surface aggregation. *Appl. Environ. Microbiol.* **76**, 3405-3408

7. Cerca, N., Jefferson, K. K., Maira-Litran, T., Pier, D. B., Kelly-Quintos, C., Goldmann, D. A., Azeredo, J., and Pier, G. B. (2007) Molecular basis for preferential protective efficacy of antibodies directed to the poorly acetylated form of staphylococcal poly-N-acetyl-beta-(1-6)-glucosamine. *Infect. Immun.* **75**, 3406-3413

8. Gjermansen, M., Nilsson, M., Yang, L., and Tolker-Nielsen, T. (2010) Characterization of starvation-induced dispersion in *Pseudomonas putida* biofilms: genetic elements and molecular mechanisms. *Mol. Microbiol.* **75**, 815-826

9. Itoh, Y., Rice, J. D., Goller, C., Pannuri, A., Taylor, J., Meisner, J., Beveridge, T. J., Preston, J. F., 3rd, and Romeo, T. (2008) Roles of *pgaABCD* genes in synthesis, modification, and export of the *Escherichia coli* biofilm adhesin poly-beta-1,6-N-acetyl-D-glucosamine. *J. Bacteriol.* **190**, 3670-3680

10. Nivens, D. E., Ohman, D. E., Williams, J., and Franklin, M. J. (2001) Role of alginate and its O acetylation in formation of *Pseudomonas aeruginosa* microcolonies and biofilms. *J. Bacteriol.* 183, 1047-1057

11. Sauer, K., Cullen, M. C., Rickard, A. H., Zeef, L. A., Davies, D. G., and Gilbert, P. (2004) Characterization of nutrient-induced dispersion in *Pseudomonas aeruginosa* PAO1 biofilm. *J. Bacteriol.* 186, 7312-7326

12. Spiers, A. J., Bohannon, J., Gehrig, S. M., and Rainey, P. B. (2003) Biofilm formation at the air-liquid interface by the *Pseudomonas fluorescens* SBW25 wrinkly spreader requires an acetylated form of cellulose. *Mol. Microbiol.* 50, 15-27

13. Hentzer, M., Teitzel, G., Balzer, G., Heydorn, A., Molin, S., Givskov, M., and Parsek, M. (2001) Alginate overproduction affects *Pseudomonas aeruginosa* biofilm structure and function. *J. Bacteriol.* 183, 5395-5401

14. Li, Z., Kosorok, M. R., Farrell, P. M., Laxova, A., West, S. E., Green, C. G., Collins, J., Rock, M. J., and Splaingard, M. L. (2005) Longitudinal development of mucoid *Pseudomonas aeruginosa* infection and lung disease progression in children with cystic fibrosis. *JAMA* 293, 581-588

15. Linker, A., and Jones, R. S. (1966) A new polysaccharide resembling alginic acid isolated from pseudomonads. *J. Biol. Chem.* 241, 3845-3851

16. Govan, J. R., and Deretic, V. (1996) Microbial pathogenesis in cystic fibrosis: mucoid *Pseudomonas aeruginosa* and *Burkholderia cepacia*. *Microbiol. Rev.* 60, 539-574

17. Chitnis, C. E., and Ohman, D. E. (1993) Genetic analysis of the alginate biosynthetic gene cluster of *Pseudomonas aeruginosa* shows evidence of an operonic structure. *Mol. Microbiol.* 8, 583-593

18. Darzins, A., Wang, S. K., Vanags, R. I., and Chakrabarty, A. M. (1985) Clustering of mutations affecting alginate acid biosynthesis in mucoid *Pseudomonas aeruginosa*. *J. Bacteriol.* 164, 516-524

19. Franklin, M. J., Nivens, D. E., Wadge, J. T., and Howell, P. L. (2011) Biosynthesis of the *Pseudomonas aeruginosa* Extracellular Polysaccharides, Alginate, Pel, and Psl. *Front. Microbiol.* 2, 167

20. Franklin, M. J., Chitnis, C. E., Gacesa, P., Sonesson, A., White, D. C., and Ohman, D. E. (1994) *Pseudomonas aeruginosa* AlgG is a polymer level alginate C5-mannuronan epimerase. *J. Bacteriol.* 176, 1821-1830

21. Jain, S., Franklin, M. J., Ertesvåg, H., Valla, S., and Ohman, D. E. (2003) The dual roles of AlgG in C-5-epimerization and secretion of alginate polymers in *Pseudomonas aeruginosa*. *Mol. Microbiol.* 47, 1123-1133

22. Gacesa, P. (1988) Alginates. *Carbohydr. Polym.* 8, 161-182

23. Franklin, M. J., and Ohman, D. E. (1993) Identification of *algF* in the alginate biosynthetic gene cluster of *Pseudomonas aeruginosa* which is required for alginate acetylation. *J. Bacteriol.* 175, 5057-5065

24. Franklin, M. J., and Ohman, D. E. (1996) Identification of *algI* and *algJ* in the *Pseudomonas aeruginosa* alginate biosynthetic gene cluster which are required for alginate O acetylation. *J. Bacteriol.* 178, 2186-2195

25. Franklin, M. J., and Ohman, D. E. (2002) Mutant analysis and cellular localization of the AlgI, AlgJ, and AlgF proteins required for O acetylation of alginate in *Pseudomonas aeruginosa*. *J. Bacteriol.* 184, 3000-3007

26. Pier, G. B., Coleman, F., Grout, M., Franklin, M., and Ohman, D. E. (2001) Role of alginate O acetylation in resistance of mucoid *Pseudomonas aeruginosa* to opsonic phagocytosis. *Infect. Immun.* 69, 1895-1901

27. Tielen, P., Strathmann, M., Jaeger, K.-E., Flemming, H.-C., and Wingender, J. (2005) Alginate acetylation influences initial surface colonization by mucoid *Pseudomonas aeruginosa*. *Microbiol. Res.* 160, 165-176

28. Evans, L. R., and Linker, A. (1973) Production and Characterization of the Slime Polysaccharide of *Pseudomonas aeruginosa*. *J. Bacteriol.* 116, 915-924

29. Robles-Price, A., Wong, T. Y., Sletta, H., Valla, S., and Schiller, N. L. (2004) AlgX is a periplasmic protein required for alginate biosynthesis in *Pseudomonas aeruginosa*. *J. Bacteriol.* 186, 7369-7377

30. Hay, I. D., Schmidt, O., Filicheva, J., and Rehm, B. H. A. (2011) Identification of a periplasmic AlgK-AlgX-MucD multiprotein complex in *Pseudomonas aeruginosa* involved in biosynthesis and regulation of alginate. *Appl. Microbiol. Biotechnol.*

31. Gutsche, J., Remminghorst, U., and Rehm, B. H. A. (2006) Biochemical analysis of alginate biosynthesis protein AlgX from *Pseudomonas aeruginosa*: purification of an AlgX-MucD (AlgY) protein complex. *Biochimie* 88, 245-251

32. Damron, F. H., and Yu, H. D. (2011) *Pseudomonas aeruginosa* MucD regulates the alginate pathway through activation of MucA degradation via MucP proteolytic activity. *J. Bacteriol.* 193, 286-291

33. Wood, L. F., and Ohman, D. E. (2006) Independent regulation of MucD, an HtrA-like protease in *Pseudomonas aeruginosa*, and the role of its proteolytic motif in alginate gene regulation. *J. Bacteriol.* 188, 3134-3137

34. Weadge, J. T., Yip, P. P., Robinson, H., Arnett, K., Tipton, P. A., and Howell, P. L. (2010) Expression, purification, crystallization and preliminary X-ray analysis of *Pseudomonas aeruginosa* AlgX. *Acta Crystallogr.*, Sect. F: Struct. Biol. Cryst. Commun. 66, 588-591

35. Lee, J. E., Cornell, K. A., Riscoe, M. K., and Howell, P. L. (2001) Structure of *E. coli* 5'-methylthioadenosine/S-adenosylhomocysteine nucleosidase reveals similarity to the purine nucleoside phosphorylases. *Structure* 9, 941-953

36. Winsor, G. L., Van Rossum, T., Lo, R., Khaira, B., Whiteside, M. D., Hancock, R. E. W., and Brinkman, F. S. L. (2009) Pseudomonas Genome Database: facilitating user-friendly, comprehensive comparisons of microbial genomes. *Nucleic Acids Res.* 37, D483-488

37. Suh, S. J., Silo-Suh, L. A., and Ohman, D. E. (2004) Development of tools for the genetic manipulation of *Pseudomonas aeruginosa*. *J. Microbiol. Methods* 58, 203-212

38. Codee, J. D., van den Bos, L. J., de Jong, A. R., Dinkelaar, J., Lodder, G., Overkleeft, H. S., and van der Marel, G. A. (2009) The stereodirecting effect of the glycosyl C5-carboxylate ester: stereoselective synthesis of beta-mannuronic acid alginate. *J. Org. Chem.* 74, 38-47

39. Pape, T., and Schneider, T. R. (2004) HKL2MAP: a graphical user interface for phasing with SHELX programs. *J. Appl. Crystallogr.* 37, 843-844

40. Terwilliger, T. C., and Berendzen, J. (1999) Automated MAD and MIR structure solution. *Acta Crystallogr.*, Sect. D: Biol. Crystallogr. 55, 849-861

41. Adams, P. D., Afonine, P. V., Bunkoczi, G., Chen, V. B., Davis, I. W., Echols, N., Headd, J. J., Hung, L. W., Kapral, G. J., Grosse-Kunstleve, R. W., McCoy, A. J., Moriarty, N. W., Oeffner, R., Read, R. J., Richardson, D. C., Richardson, J. S., Terwilliger, T. C., and Zwart, P. H. (2010) PHENIX: a comprehensive Python-based system for macromolecular structure solution. *Acta Crystallogr.*, Sect. D: Biol. Crystallogr. 66, 213-221

42. Emsley, P., and Cowtan, K. (2004) Coot: model-building tools for molecular graphics. *Acta Crystallogr.*, Sect. D: Biol. Crystallogr. 60, 2126-2132

43. Painter, J., and Merritt, E. A. (2006) Optimal description of a protein structure in terms of multiple groups undergoing TLS motion. *Acta Crystallogr.*, Sect. D: Biol. Crystallogr. 62, 439-450

44. Painter, J., and Merritt, E. A. (2006) TLSMD web server for the generation of multi-group TLS models. *J. Appl. Crystallogr.* 39, 109-111

45. McCoy, A. J., Grosse-Kunstleve, R. W., Adams, P. D., Winn, M. D., Storoni, L. C., and Read, R. J. (2007) Phaser crystallographic software. *J. Appl. Crystallogr.* 40, 658-674

46. Joosten, R. P., te Beek, T. A., Krieger, E., Hekkelman, M. L., Hooft, R. W., Schneider, R., Sander, C., and Vriend, G. (2010) A series of PDB related databases for everyday needs. *Nucleic Acids Res.* 39, D411-419

47. Kabsch, W., and Sander, C. (1983) Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features. *Biopolymers* 22, 2577-2637

48. Choi, K. H., and Schweizer, H. P. (2005) An improved method for rapid generation of unmarked *Pseudomonas aeruginosa* deletion mutants. *BMC Microbiol.* 5, 30

49. Holloway, B. W. (1969) Genetics of *Pseudomonas*. *Bacteriol. Rev.* 33, 419-443

50. Wood, L. F., Leech, A. J., and Ohman, D. E. (2006) Cell wall-inhibitory antibiotics activate the alginate biosynthesis operon in *Pseudomonas aeruginosa*: Roles of sigma (AlgT) and the AlgW and Prc proteases. *Mol. Microbiol.* 62, 412-426

51. Chibba, A., Poloczek, J., Little, D. J., Howell, P. L., and Nitz, M. (2012) Synthesis and evaluation of inhibitors of *E. coli* PgaB, a polysaccharide de-N-acetylase involved in biofilm formation. *Org. Biomol. Chem.* 10, 7103-7107

52. Altschul, S. F., Gish, W., Miller, W., Myers, E. W., and Lipman, D. J. (1990) Basic local alignment search tool. *J. Mol. Biol.* 215, 403-410

53. Notredame, C., Higgins, D. G., and Heringa, J. (2000) T-Coffee: A novel method for fast and accurate multiple sequence alignment. *J. Mol. Biol.* 302, 205-217

54. Landau, M., Mayrose, I., Rosenberg, Y., Glaser, F., Martz, E., Pupko, T., and Ben-Tal, N. (2005) ConSurf 2005: the projection of evolutionary conservation scores of residues on protein structures. *Nucleic Acids Res.* 33, W299-302

55. Ashkenazy, H., Erez, E., Martz, E., Pupko, T., and Ben-Tal, N. (2010) ConSurf 2010: calculating evolutionary conservation in sequence and structure of proteins and nucleic acids. *Nucleic Acids Res.* 38, W529-533

56. Pearson, W. R. (1996) Effective protein sequence comparison. *Methods Enzymol.* 266, 227-258

57. Wilson, D., Pethica, R., Zhou, Y., Talbot, C., Vogel, C., Madera, M., Chothia, C., and Gough, J. (2009) SUPERFAMILY--sophisticated comparative genomics, data mining, visualization and phylogeny. *Nucleic Acids Res.* 37, D380-386

58. Park, B. H., Karpinets, T. V., Syed, M. H., Leuze, M. R., and Uberbacher, E. C. (2010) CAZymes Analysis Toolkit (CAT): web service for searching and analyzing carbohydrate-active enzymes in a newly sequenced organism using CAZy database. *Glycobiology* 20, 1574-1584

59. Knutson, C. A., and Jeanes, A. (1968) A new modification of the carbazole analysis: application to heteropolysaccharides. *Anal. Biochem.* 24, 470-481

60. Hestrin, S. (1949) The reaction of acetylcholine and other carboxylic acid derivatives with hydroxylamine, and its analytical application. *J. Biol. Chem.* 180, 249-261

61. Grasdalen, H. (1979) A P.M.R. study of the composition and sequence of uronate residues in alginate. *Carbohydr. Res.* 68, 23-31

62. Grasdalen, H. (1983) High-field 1H n.m.r. spectroscopy of alginate: sequential structure and linkage conformations. *Carbohydr. Res.* 118, 255-260

63. Chitnis, C. E., and Ohman, D. E. (1990) Cloning of *Pseudomonas aeruginosa* algG, which controls alginate structure. *J. Bacteriol.* 172, 2894-2900

64. Douthit, S. A., Dlakic, M., Ohman, D. E., and Franklin, M. J. (2005) Epimerase active domain of *Pseudomonas aeruginosa* AlgG, a protein that contains a right-handed beta-helix. *J. Bacteriol.* 187, 4573-4583

65. Salamitou, S., Lemaire, M., Fujino, T., Ohayon, H., Gounon, P., Beguin, P., and Aubert, J. (1994) Subcellular localization of *Clostridium thermocellum* ORF3p, a protein carrying a receptor for the docking sequence borne by the catalytic components of the cellulosome. *J. Bacteriol.* 176, 2828-2834

66. Koo, J., Tammam, S., Ku, S. Y., Sampaleanu, L. M., Burrows, L. L., and Howell, P. L. (2008) PilF is an outer membrane lipoprotein required for multimerization and localization of the *Pseudomonas aeruginosa* Type IV pilus secretin. *J. Bacteriol.* 190, 6961-6969

67. Høiby, N., Johansen, H., Moser, C., Song, Z., Ciofu, O., and Kharazmi, A. (2001) *Pseudomonas aeruginosa* and the in vitro and in vivo biofilm mode of growth. *Microbes Infect.* 3, 23-35

68. Hashimoto, H. (2006) Recent structural studies of carbohydrate-binding modules. *Cell. Mol. Life Sci.* 63, 2954-2967

69. Guillen, D., Sanchez, S., and Rodriguez-Sanoja, R. (2010) Carbohydrate-binding domains: multiplicity of biological roles. *Appl. Microbiol. Biotechnol.* 85, 1241-1249

70. Boraston, A. B., Bolam, D. N., Gilbert, H. J., and Davies, G. J. (2004) Carbohydrate-binding modules: fine-tuning polysaccharide recognition. *Biochem. J.* 382, 769-781

71. Ye, Y., and Godzik, A. (2003) Flexible structure alignment by chaining aligned fragment pairs allowing twists. *Bioinformatics* 19 Suppl 2, ii246-255

72. Ma, J., Lu, Q., Yuan, Y., Ge, H., Li, K., Zhao, W., Gao, Y., Niu, L., and Teng, M. (2011) Crystal structure of isoamyl acetate-hydrolyzing esterase from *Saccharomyces cerevisiae* reveals a novel active site architecture and the basis of substrate specificity. *Proteins* 79, 662-668

73. McMullen, T. W., Li, J., Sheffield, P. J., Aoki, J., Martin, T. W., Arai, H., Inoue, K., and Derewenda, Z. S. (2000) The functional implications of the dimerization of the catalytic subunits of the mammalian brain platelet-activating factor acetylhydrolase (Ib). *Protein Eng.* 13, 865-871

74. Akoh, C. C., Lee, G. C., Liaw, Y. C., Huang, T. H., and Shaw, J. F. (2004) GDSL family of serine esterases/lipases. *Prog. Lipid Res.* 43, 534-552

75. Pfeffer, J. M., Weadge, J. T., and Clarke, A. J. (2013) Mechanism of action of *Neisseria gonorrhoeae* O-acetylpeptidoglycan esterase, an SGNH serine esterase. *J. Biol. Chem.* 288, 2605-2613

76. Hedstrom, L. (2002) Serine protease mechanism and specificity. *Chem. Rev.* 102, 4501-4524

77. Montanier, C., Money, V. A., Pires, V. M., Flint, J. E., Pinheiro, B. A., Goyal, A., Prates, J. A., Izumi, A., Stalbrand, H., Morland, C., Cartmell, A., Kolenova, K., Topakas, E., Dodson, E. J., Bolam, D. N., Davies, G. J., Fontes, C. M., and Gilbert, H. J. (2009) The active site of a carbohydrate esterase displays divergent catalytic and noncatalytic binding functions. *PLoS Biol.* 7, e71

78. Weadge, J. T., and Clarke, A. J. (2006) Identification and characterization of O-acetylpeptidoglycan esterase: a novel enzyme discovered in *Neisseria gonorrhoeae*. *Biochemistry* 45, 839-851

79. Bolam, D. N., Ciruela, A., McQueen-Mason, S., Simpson, P., Williamson, M. P., Rixon, J. E., Boraston, A., Hazlewood, G. P., and Gilbert, H. J. (1998) *Pseudomonas* cellulose-binding domains mediate their effects by increasing enzyme substrate proximity. *Biochem. J.* 331 775-781

80. Bernard, E., Rolain, T., Courtin, P., Guillot, A., Langella, P., Hols, P., and Chapot-Chartier, M. P. (2011) Characterization of O-acetylation of N-acetylglucosamine: a novel structural variation of bacterial peptidoglycan. *J. Biol. Chem.* 286, 23950-23958

81. Moynihan, P. J., and Clarke, A. J. (2010) O-acetylation of peptidoglycan in gram-negative bacteria: identification and characterization of peptidoglycan O-acetyltransferase in *Neisseria gonorrhoeae*. *J. Biol. Chem.* 285, 13264-13273

82. Pickens, L. B., Sawaya, M. R., Rasool, H., Pashkov, I., Yeates, T. O., and Tang, Y. (2011) Structural and biochemical characterization of the salicyl-acyltranferase SsfX3 from a tetracycline biosynthetic pathway. *J. Biol. Chem.* 286, 41539-41551

83. Reina, J. J., Guerrero, C., and Heredia, A. (2007) Isolation, characterization, and localization of AgaSGNH cDNA: a new SGNH-motif plant hydrolase specific to *Agave americana* L. leaf epidermis. *J. Exp. Bot.* 58, 2717-2731

84. Robertson, D. L., Hilton, S., Wong, K. R., Koepke, A., and Buckley, J. T. (1994) Influence of active site and tyrosine modification on the secretion and activity of the Aeromonas hydrophila lipase/acyltransferase. *J. Biol. Chem.* 269, 2146-2150

85. Teutschbein, J., Gross, W., Nimtz, M., Milkowski, C., Hause, B., and Strack, D. (2010) Identification and localization of a lipase-like acyltransferase in phenylpropanoid metabolism of tomato (*Solanum lycopersicum*). *J. Biol. Chem.* 285, 38374-38381

86. Di Cera, E. (2009) Serine proteases. *IUBMB Life* 61, 510-515

87. Shinabarger, D., May, T. B., Boyd, A., Ghosh, M., and Chakrabarty, A. M. (1993) Nucleotide sequence and expression of the *Pseudomonas aeruginosa algF* gene controlling acetylation of alginate. *Mol. Microbiol.* 9, 1027-1035

88. Franklin, M. J., Douthit, S. A., and McClure, M. A. (2004) Evidence that the *algI/algJ* gene cassette, required for O acetylation of *Pseudomonas aeruginosa* alginate, evolved by lateral gene transfer. *J. Bacteriol.* 186, 4759-4773

89. Winsor, G. L., Lam, D. K., Fleming, L., Lo, R., Whiteside, M. D., Yu, N. Y., Hancock, R. E., and Brinkman, F. S. (2011) *Pseudomonas* Genome Database: improved comparative analysis and population genomics capability for *Pseudomonas* genomes. *Nucleic Acids Res.* 39, D596-600

90. Kelley, L. A., and Sternberg, M. J. (2009) Protein structure prediction on the Web: a case study using the Phyre server. *Nat. Protoc.* 4, 363-371

91. Baker, N. A., Sept, D., Joseph, S., Holst, M. J., and McCammon, J. A. (2001) Electrostatics of nanosystems: application to microtubules and the ribosome. *Proc. Natl. Acad. Sci. U.S.A.* 98, 10037-10041

92. Dolinsky, T. J., Czodrowski, P., Li, H., Nielsen, J. E., Jensen, J. H., Klebe, G., and Baker, N. A. (2007) PDB2PQR: expanding and upgrading automated preparation of biomolecular structures for molecular simulations. *Nucleic Acids Res.* 35, W522-525

93. Dolinsky, T. J., Nielsen, J. E., McCammon, J. A., and Baker, N. A. (2004) PDB2PQR: an automated pipeline for the setup of Poisson-Boltzmann electrostatics calculations. *Nucleic Acids Res.* 32, W665-667

94. Bera, A., Herbert, S., Jakob, A., Vollmer, W., and Gotz, F. (2005) Why are pathogenic staphylococci so lysozyme resistant? The peptidoglycan O-acetyltransferase OatA is the major determinant for lysozyme resistance of *Staphylococcus aureus*. *Mol. Microbiol.* 55, 778-787

95. Chen, V. B., Arendall, W. B., 3rd, Headd, J. J., Keedy, D. A., Immormino, R. M., Kapral, G. J., Murray, L. W., Richardson, J. S., and Richardson, D. C. (2010) MolProbity: all-atom structure validation for macromolecular crystallography. *Acta Crystallogr.*, Sect. D: *Biol. Crystallogr.* 66, 12-21

## ACKNOWLEDGMENTS

We would like to thank Yura Lobsanov, Francis Wolfram, Dustin J. Little, Jason Koo, John C.C. Whitney and G. David Smith for helpful discussions, and Patrick Yip for technical assistance. This work is supported by research grants from the Canadian Institutes of Health Research (CIHR) (#13337 to P.L.H.), the National Institutes of Health Research (GM081419 to P.A.T), National Institutes of Allergy and Infectious Disease Public Health Service (AI-19146) and Veterans Administration Medical Research (IO1BX000477) to D.E.O. and The Netherlands Organization for Scientific Research (NWO-vidi to J.D.C.C.). J.T.W was supported, in part, by postdoctoral fellowships from The Hospital for Sick Children, the CIHR Training Program in Membrane Proteins and Disease, and the Natural Sciences and Engineering Research Council of Canada. P.B. is supported by a postdoctoral fellowship from Cystic Fibrosis Canada. P.L.H. is the recipient of a Canada Research Chair. The National Synchrotron Light Source Beam line X29 is supported by the United States Department of Energy Office of Biological and Environmental Research, and the National Institutes of Health National Centre for Research Resources.

## FOOTNOTES
The atomic coordinates and structure factors (4KNC) have been deposited in the Protein Data Bank (http://www.rcsb.org/).

# FIGURE LEGENDS

Figure 1. Overall Structure of AlgX. (A) Cartoon representation of AlgX with secondary structure elements labelled as follows: $\alpha$ - $\alpha$-helix, $\beta$ - $\beta$-strand, t - $3_{10}$ helix. The N-terminal $\alpha/\beta/\alpha$ and C-terminal CBM domains are represented in blue and red, respectively. The two disulphide bonds present in the structure are coloured in yellow. The dashed lines represent residues that cannot be built or modelled due to poor quality electron density. “N” and “C” represent the N- and C-termini of the protein. (B) Topology representation of AlgX with the secondary structural elements labelled and coloured as in panel A.

Figure 2. Surface representations of AlgX. (A) Surface representation of AlgX depicting the N-terminal $\alpha/\beta/\alpha$ domain in blue and the C-terminal CBM domain in red. The white line indicates the proposed path of alginate across AlgX, between the two domains. When compared with the representations in (B) and (C), it can be seen that the path crosses conserved, mainly electropositive residues. (B) Electrostatic surface representation of AlgX. The electrostatic potential, calculated using APBS (91-93), is contoured from -7 to 7 kT/e and coloured on a gradient from red (negative) to blue (positive). (C) Representation of the surface residue conservation in eight Pseudomonads and *A. vinelandii* as calculated using ConSurf and coloured on a gradient from high (dark pink) to low (cyan) sequence conservation (54, 55).

Figure 3. C-terminal domain of AlgX is a Carbohydrate Binding Module. Cartoon representations of (A) the CBM, in red, showing the two anti-parallel $\beta$-sheets organised into a $\beta$-jelly roll fold; and (B) the superposition of the CBM (red) and *P. equi* CBM29-2 (1GWK, grey) in complex with mannohexaose. The Trp400, Thr398, Arg364 and Arg406 residues that comprise the substrate-recognition pinch point are shown in yellow. Lys396 and Lys410, located outside the pinch point and postulated to play a role in directing the alginate polymer across the protein, are coloured orange. (C) Surface representation of the CBM of AlgX showing the location of the superposed mannohexaose from *P. equi* CBM29-2. The oligosaccharide passes through the pinch point on the surface of the CBM.

Figure 4. The N-terminal domain of AlgX has an SGNH-like hydrolase fold. (A) The N-terminal SGNH hydrolase-like domain of AlgX and two representative members of the SGNH hydrolase superfamily: *E. faecalis* acylhydrolase/lipase (PDB 1YZF) and the putative acetylxylan esterase from *C. acetobutylicum* (PDB 1ZMB). In all three proteins the Ser-His-Asp triad and signature Gly residues are structurally conserved and shown in red in stick representation. The conserved SGNH Asn residue and the equivalent Tyr in AlgX are shown in brown in stick representation. The parallel $\beta$-sheet and surrounding $\alpha$-helices of the core SGNH hydrolase domains and equivalent features in AlgX are coloured in orange and blue, respectively. The secondary structural elements of AlgX and the *C. acetobutylicum* protein that are not part of the core SGNH hydrolase fold are coloured in grey. (B) Topology of the N-terminal domain of AlgX and its comparison with the typical topologies found in the SGNH hydrolases, *E. faecalis* acylhydrolase/lipase and *C. acetobutylicum* putative acetylxylan esterase. The topology diagrams are colour as in panel (A).

Figure 5. Architecture of the Active site. (A) Superposition of the Asp-His-Ser triad and conserved Gly of AlgX (shown in red) and *E. faecalis* acylhydrolase/lipase (in grey). Asn and Tyr residues of *E. faecalis* acylhydrolase/lipase and AlgX, not used in the superposition, are shown in dark grey and brown, respectively. Main chain atoms surrounding the residues of interest are coloured in pale grey, and light red/brown for the *E. faecalis* acylhydrolase/lipase and AlgX structures, respectively. (B) The bonds formed between the catalytic triad residues are shown in yellow. (C) Ten residues of AlgX wereindependently mutated on the chromosome to alanine, to probe their role in $O$-acetylation of alginate. These residues, depicted in stick representation and coloured in yellow, were selected for mutation based upon their location within the putative active site. The residues found to be important for the acetylation of alginate are boxed.

Figure 6. Role of AlgX in alginate $O$-acetylation. Analysis of the PAO1 $mucA::Gm^R$ mutant strains for: (A) alginate production, (B) $O$-acetylation of the produced alginate, and (C) epimerization. The absorbance at 548 nm is directly related to the guluronate content of alginate. (D) $algX$ was expressed at equivalent levels in all of the mutants, as indicated by Western blotting analysis. PilF, from the Type IV pilus system of $P$. aeruginosa, was used as a loading control. (E) NMR spectra of the alginate produced by the wild-type $algX+$ strain and D174A, H176A, and S269A variants. The peak at ~2.1 ppm in the $algX+$ spectrum indicates the presence of acetyl groups on the alginate. (F) In vitro activity assay showing the specific activities of the recombinant AlgX protein and the D174A, H176A, and S269A protein variants.

Figure 7. Proposed mechanism of action of AlgX. The Ser269 nucleophile attacks the carbonyl carbon of an ester-linked acetate group on the substrate acetyl donor (A), leading to the formation of a tetrahedral intermediate (B). His176 abstracts the hydroxyl proton of a mannuronate residue and the nucleophilic hydroxyl oxygen of the mannuronate residue attacks the carbonyl carbon, forming an acyl-enzyme intermediate (C). This may involve the C2 or C3 hydroxyl group on the mannuronate residue. The carbonyl double bond is reformed, releasing the acetyl group from the donor (D) and the residues revert to their resting state (E). During the reaction, the amide nitrogens of Ser269 and Gly296 stabilize the oxyanion of the tetrahedral intermediate. His176 acts as a base, increasing the nucleophilic properties of Ser269, and Asp174 neutralizes the charge that develops on His176 in the course of the reaction. R represents the, as yet unidentified, acetyl donor.

Figure 8: Model for $O$-acetylation of alginate and other bacterial polysaccharides. (A) AlgI receives an acetyl group from a cytoplasmic acetyl donor. From AlgI, the acetyl group is transferred to AlgJ and/or AlgX and then on to the alginate polymer. AlgJ and AlgX may be able to pass the acetyl group between themselves. The role of AlgF is currently unknown. (B) The four proteins of the $P$. fluorescens Wss system acetylate cellulose, and may act in a similar manner to the four Alg proteins. (C) Proposed mechanism of peptidoglycan acetylation in Gram negative bacteria as presented by Moynihan and Clarke (81). This mechanism requires two proteins, an AlgI homologue, PatA, which transports the acetate across the cytoplasmic membrane to the periplasm and an AlgJ homologue, PatB, which transfers the acetate to the sugar. (D) In Gram positive bacteria, the $N$-acetyl muramic acid residues of the peptidoglycan is $O$-acetylated by the two-domain protein, OatA (80, 94). The N-terminal domain of OatA is homologous to AlgI and the C-terminal domain is homologous to AlgJ (80). More recently a second protein, OatB, has been identified in Lactobacillus plantarum, which $O$-acetylates $N$-acetyl glucosamine peptidoglycan residues (80). OatB also has two domains homologous to AlgI and AlgJ. Ac represents acetate, dotted lines represent potential movement of the acetate after transport across the inner membrane. Proteins with similar predicted structure and function are coloured the same.
Table 1. Summary of data collection and refinement statistics

| Data Collection | Native | SeMet-SAD |
|-----------------|--------|-----------|
| Beamline        | NSLS X 29 | NSLS X 29 |
| Wavelength (Å)  | 1.075   | 0.979     |
| Space Group     | P2₁2₁2  | P2₁       |
| Cell dimensions |         |           |
| $a, b, c$ (Å)   | 123.7, 82.4, 92.7 | 46.4, 121.4, 92.1 |
| $\alpha, \beta, \gamma$ (°) | 90.0, 90.0, 90.0 | 90.0, 95.2, 90.0 |
| Total no. of reflections | 506683 | 437204 |
| No. of Unique reflections | 52314 | 68397 |
| Resolution (Å) | 50.0-2.15 (2.23-2.15)$^a$ | 43.22-2.00 (2.07-2.02) |
| Average I/σI    | 44.7 (4.0) | 23.0 (2.1) |
| Completeness (%) | 98.82 (89.26) | 90.7 (64.5) |
| Redundancy      | 9.7 (9.8) | 7.1 (6.2) |
| $R_{\text{merge}}$ (%)$^b$ | 8.1 (64.0) | 8.3 (68.1) |

| Refinement | Native | SeMet-SAD |
|------------|--------|-----------|
| $R_{\text{work}} / R_{\text{free}}$ (%)$^c$ | 17.60/23.10 | - |
| No. atoms |         |           |
| Protein   | 6504    | -         |
| Water     | 372     | -         |
| Average B-factor (Å²) |          |           |
| Protein   | 50.10   | -         |
| Water     | 50.20   | -         |
| Rms deviations |      |           |
| Bond lengths (Å) | 0.007 | -         |
| Bond angles (°) | 1.040 | -         |
| Ramachandran plot$^d$ |          |           |
| Total favoured (%) | 97.0 | -         |
| Total allowed (%) | 100 | -         |
| Coordinate error (Å)$^e$ | 0.24 | -         |
| PDB Code | 4KNC | -         |

$a$ Values in parentheses correspond to highest resolution shell  
$b$ $R_{\text{merge}} = \Sigma \Sigma |I(k) - <I>| / \Sigma I(k)$ where $I(k)$ and $<I>$ represent the intensity values of the individual measurements and the corresponding mean values. The summation is over all unique measurements  
$c$ $R_{\text{work}} = \Sigma ||F_{\text{oobs}}| - k|F_{\text{calc}}|| / |F_{\text{oobs}}|$ where $F_{\text{oobs}}$ and $F_{\text{calc}}$ are the observed and calculated structure factors, respectively. $R_{\text{free}}$ is the sum extended over a subset of reflections (5%) excluded from all stages of the refinement  
$d$ As calculated using MOLPROBITY (95)  
$e$ Maximum-Likelihood based coordinate error as determined by PHENIX (41)

Table 2. SGNH hydrolase conserved blocks.

The conserved active site SGNH hydrolase residues in *E. faecalis* acylhydrolase/lipase are found in the canonical blocks. The blocks in AlgX are not conserved and the active site residues are found in a different order in the protein.

| Classic SGNH          | Classic SGNH         | *E. faecalis* motifs | AlgX motifs   | AlgX signature |
|-----------------------|----------------------|----------------------|---------------|----------------|
| hydrolase block       | hydrolase motifs     |                      |               | residues       |
| I                     | GDSL                 | GDSI                 | GTSN          | S269           |
| II                    | NxxSxxGxT            | NAGMPGDT             | NAVSGGGF      | G296           |
| III                   | GxDND                | GAND                 | THYD          | Y328           |
| V                     | DxxHP                | DGLHF                | DHHW          | D174, H176     |

Residues in bold type are 100% conserved in SGNH hydrolases. Underlined residues are those found in the canonical block motifs or the equivalent in AlgX.

Figure 1.

A

B

C

Figure 2.

A

B

C

Most variable
Least variable

Figure 3.

A

B

K396
K410
T398
R406
W400
R364

C

K396
K410
T398
R406
W400
R364

Figure 4.

A

- AlgX N-terminal domain
- E. faecalis acylhydrolase/lipase
- C. acetobutylicum putative acetylxyran esterase

B

- AlgX N-terminal domain
- E. faecalis acylhydrolase/lipase
- C. acetobutylicum putative acetylxyran esterase

Figure 5.

A

D174/D166
H176/H169
S269/S10
Y328
N78
G296/G49

B

D174
2.6 Å
H176
2.5 Å
S269

C

R115
D174
H175
Y275
H176
Y328
N270
S269
D300
S301

Figure 6.

A

| Alginate (mg/ml) |
| --- |
| 4 |
| 3 |
| 2 |
| 1 |
| 0 |

PAO1  
algX+  
R115A  
D174A  
H175A  
H176A  
S269A  
N270A  
Y275A  
D300A  
S301A  
Y328A  

mucA::Gm<sup>R</sup>

B

| Acetlys (mM) |
| --- |
| 6 |
| 4 |
| 2 |
| 0 |

PAO1  
algX+  
R115A  
D174A  
H175A  
H176A  
S269A  
N270A  
Y275A  
D300A  
S301A  
Y328A  

mucA::Gm<sup>R</sup>

C

| Abs (535 nm) |
| --- |
| 3 |
| 2 |
| 1 |
| 0 |

PAO1  
algX+  
R115A  
D174A  
H175A  
H176A  
S269A  
N270A  
Y275A  
D300A  
S301A  
Y328A  

mucA::Gm<sup>R</sup>

D

AlgX  
PilF  

algX+  
R115A  
D174A  
H175A  
H176A  
S269A  
N270A  
Y275A  
D300A  
S301A  
Y328A  

mucA::Gm<sup>R</sup>

E

D174A  
6.0 5.0 4.0 3.0 2.0 1.0 ppm  

H176A  
6.0 5.0 4.0 3.0 2.0 1.0 ppm  

S269A  
6.0 5.0 4.0 3.0 2.0 1.0 ppm  

algX+  
6.0 5.0 4.0 3.0 2.0 1.0 ppm  

F

Specific Activity  
(µmols min<sup>-1</sup> mg<sup>-1</sup>)  
0.008  
0.006  
0.004  
0.002  
0  

algX WT D174A H176A S269A

Figure 7.

A

![Chemical Structure A](chemical_structure_a.png)

B

![Chemical Structure B](chemical_structure_b.png)

C

![Chemical Structure C](chemical_structure_c.png)

D

![Chemical Structure D](chemical_structure_d.png)

E

![Chemical Structure E](chemical_structure_e.png)

Figure 8.

![Diagram](attachment:diagram.png)

- **A**
- **B**
- **C**
- **D**

**Acetyl donor**
- AlgI
- AlgJ
- AlgX
- AlgF

**Acetyl donor**
- WssH
- WssF
- WssG

**Acetyl donor**
- PatA
- PatB

**Acetyl donor**
- OatA/OatB

OM
IM
