
Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

**Developmental Cell**

**Article**

# The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy

Kai Mao,$^{1}$ Ke Wang,$^{1}$ Xu Liu,$^{1}$ and Daniel J. Klionsky$^{1,*}$

$^{1}$Life Sciences Institute and Department of Molecular, Cellular and Developmental Biology, University of Michigan, Ann Arbor, MI 48109, USA

*Correspondence: klionsky@umich.edu

http://dx.doi.org/10.1016/j.devcel.2013.05.024

## SUMMARY

As the cellular power plant, mitochondria play a significant role in homeostasis. To maintain the proper quality and quantity of mitochondria requires both mitochondrial degradation and division. A selective type of autophagy, mitophagy, drives the degradation of excess or damaged mitochondria, whereas division is controlled by a specific fission complex; however, the relationship between these two processes, especially the role of mitochondrial fission during mitophagy, remains unclear. In this study, we report that mitochondrial fission is important for the progression of mitophagy. When mitophagy is induced, the fission complex is recruited to the degrading mitochondria through an interaction between Atg11 and Dnm1; interfering with this interaction severely blocks mitophagy. These data establish a paradigm for selective organelle degradation.

## INTRODUCTION

Mitochondria are double-membrane-bound organelles that play significant roles in a variety of cellular metabolic reactions. This organelle is central to cellular physiology, supplying energy and certain metabolites, but it also generates harmful reactive oxygen species. Thus, mitochondrial homeostasis must be maintained, which can be a costly process. As a result, cells degrade superfluous, or extensively damaged, mitochondria; however, this is a considerable structural challenge considering the extended, reticular nature of this organelle. Constitutive mitochondrial fusion and fission, as well as biogenesis and degradation, make the mitochondria highly dynamic. In the budding yeast Saccharomyces cerevisiae, a complex containing Fis1, Dnm1, Mdv1, and Caf4 controls the fission of mitochondria, whereas fusion is regulated by the action of Fzo1, Ugo1, and Mgm1 (Okamoto and Shaw, 2005). The degradation of mitochondria is mediated by mitophagy, a selective type of macroautophagy (hereafter referred to as autophagy). The study of mitophagy has attracted increasing attention in recent years; this process plays significant roles in various aspects of normal physiology, such as the removal of mitochondria during the maturation of erythroid cells, whereas its dysfunction is associated with certain pathophysiologies, such as Wolfram syndrome

2 and Parkinson disease (Kanki and Klionsky, 2009; Kanki et al., 2009b; Matsuda et al., 2010; Narendra et al., 2008, 2010; Okamoto et al., 2009; Schneers et al., 2007; Vives-Bauza et al., 2010).

In eukaryotic cells, autophagy functions as a lysosome/vacuole-dependent mechanism for the degradation of damaged or obsolete proteins and organelles and can occur in either nonselective or selective modes. Nonselective autophagy functions to sequester bulk cytoplasm into double-membrane vesicles, termed autophagosomes, which are then transported to the lysosome/vacuole where the cargo is degraded (Xie and Klionsky, 2007). In contrast, selective autophagy targets specific proteins or organelles as cargos, such as peroxisomes (pexophagy) and mitochondria. In the case of selective autophagy, a general model has been established in which a ligand on the target interacts with a specific receptor; the receptor in turn binds a scaffold protein, which links the cargo-receptor complex with the autophagy machinery (Mijaljica et al., 2012). For example, in yeast mitophagy, Atg32 is a mitochondrial protein that serves as a selective receptor (a ligand, if one exists, has not been identified), which binds the Atg11 scaffold. Atg11 is needed for subsequent engagement of the mitochondria with Atg8-PE, which lines the initial sequestering compartment, the phagophore. Due to its potentially large size, one key question with regard to mitophagy concerns the role of mitochondrial fission. Work in mammalian cells suggests that mitochondrial fission facilitates the process of mitophagy (Tanaka et al., 2010; Twig et al., 2008); however, it is not clear whether these two processes occur independently or whether they function in a coordinated manner.

Here, we show that the Atg32-Atg11 interaction marks degrading mitochondria. Furthermore, Dnm1 is recruited to these mitochondria through an interaction with Atg11. When Dnm1 loses its interaction with Atg11, the degradation of mitochondria is severely blocked. These results support the hypothesis that mitochondrial fission machinery participates in, and facilitates, mitochondrial division in an early step of mitophagy and indicate that Atg11 plays a role as a scaffold that recruits the fission components in addition to its role in connecting the target with the autophagic machinery.

## RESULTS

### The Mitochondrial Fission Complex Is Required for Mitophagy

In our recent genome-wide yeast mutant screen for mitophagy-defective strains, we found that *DNM1*, a gene encoding a

Developmental Cell 26, 1–10, July 15, 2013 ©2013 Elsevier Inc.

Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

**Developmental Cell**

Atg11 Recruits Dnm1 to Degrading Mitochondria

of mitoPho8Δ60 can be used to monitor mitophagy activity. We found that mitophagy was severely blocked in *dnm1Δ* and *fis1Δ* cells, partially blocked in *mdv1Δ* cells, and essentially normal in *caf4Δ* cells ([Figure 1](#fig1)). Previous work implied a redundant function for Mdv1 and Caf4, with Mdv1 being more important than Caf4 in mitochondrial fission ([Naylor et al., 2006](#ref1)). Accordingly, we examined the *mdv1Δ caf4Δ* double-deletion mutant, which showed a strong defect in mitophagy, similar to *dnm1Δ* and *fis1Δ* ([Figure 1](#fig1)). These results suggest that the intact mitochondrial fission machinery is required for mitophagy.

Previously, it was reported that mitochondrial fission is not required for rapamycin-induced mitophagy ([Mendl et al., 2011](#ref2)). In this study, the diminished mitophagy activity in *fis1Δ* cells is reported to be due to a secondary mutation in the *WHI2* gene. Therefore, we tested mitophagy activity in *whi2Δ* cells using the mitoPho8Δ60 assay. Mitophagy activity in *whi2Δ* cells was essentially identical to that of the isogenic wild-type cells ([Figure 1](#fig1)). The discrepancy between our results and those of Mendl et al. may be due to the different methods used to induce mitophagy. In the previous study, yeast cells were cultured in medium containing glycerol and mitophagy was induced by the addition of rapamycin; however, these conditions require 24 hr of drug treatment to induce a high level of mitophagy. In our study, mitochondrial proliferation was achieved by growth in lactic acid and mitophagy was induced by nitrogen starvation in the presence of glucose; a similar level of mitoPho8Δ60 activity was detected in the wild-type and *whi2Δ* strains within 6 hr. In general, nitrogen starvation induces a substantially stronger response than treatment with rapamycin.

**Figure 1. Mitochondrial Fission Is Required for Mitophagy**

MitoPho8Δ60 activity is reduced in strains with deletions of genes encoding mitochondrial fission proteins. Wild-type (KWY20), *atg32Δ* (KWY22), *dnm1Δ* (KDM1013), *fis1Δ* (KDM1002), *mdv1Δ* (KDM1006), *caf4Δ* (KDM1011), *mdv1Δ caf4Δ* (KDM1012), and *whi2Δ* (KDM1010) cells in the mitoPho8Δ60 background were cultured in YPL to mid-log phase and then shifted to SD-N for 6 hr. The mitoPho8Δ60 assay was performed as described in **Experimental Procedures**. Error bars correspond to the standard error and were obtained from three independent repeats. *p < 0.01; **p < 0.001; ***p < 0.0001.*

dynamins-related GTPase required for mitochondrial fission, is required for efficient mitophagy ([Kanki et al., 2009a](#ref3)). In *S. cerevisiae*, the mitochondrial fission complex consists of four components: Fis1, Dnm1, Mdv1, and Caf4. Fis1 is a conserved integral membrane protein and is required for the proper localization of Dnm1 and Mdv1 on mitochondria ([Karren et al., 2005](#ref4); [Mozdy et al., 2000](#ref5); [Tieu and Nunnari, 2000](#ref6)). Dnm1 assembles specifically at the sites where mitochondrial fission occurs ([Bleazard et al., 1999](#ref7)). Mdv1 and Caf4 redundantly bridge the interaction between Fis1 and Dnm1 ([Griffin et al., 2005](#ref8); [Tieu and Nunnari, 2000](#ref6)). Although Fis1 is distributed evenly on mitochondria, Dnm1 and Mdv1 show colocalized puncta on mitochondrial tubules, and it is thought that those puncta are the sites where mitochondrial division takes place. We first sought to determine whether all four of the corresponding gene products are involved in mitophagy.

In *S. cerevisiae*, mitochondria proliferate when cells are cultured in a nonfermentable carbon source, such as lactic acid or glycerol. When these cells are subjected to conditions of nitrogen starvation in the presence of a fermentable carbon source, such as glucose, mitophagy is induced to degrade the excess mitochondria ([Kanki and Klionsky, 2008](#ref9)). We examined the activity of mitophagy in the absence of *DNM1, FIS1, MDV1,* or *CAF4* using an enzymatic assay in which a mitochondrially targeted zymogen, mitoPho8Δ60, is activated following mitophagic delivery to the vacuole ([Kanki et al., 2009a](#ref3)). *PHO8* encodes an alkaline phosphatase that is transported to the vacuole via the ALP pathway ([Klionsky and Emr, 1989](#ref10)). Pho8Δ60, a mutant form of Pho8 in which the N-terminal 60 amino acids including the transmembrane domain have been removed, localizes to the cytoplasm and is unable to be transported to the vacuole through the ALP pathway. We fused cytochrome c oxidase subunit IV (Cox4) with Pho8Δ60 and named the fusion protein mitoPho8Δ60. This fusion protein is localized in mitochondria and its transport to the vacuole, which results in enzyme activation, is dependent on mitophagy ([Kanki et al., 2009a](#ref3)). Therefore, calculating the phosphatase activity

**The Atg32-Atg11 Interaction Marks Degrading Mitochondria during Mitophagy**

Mitochondrial fission is proposed to occur on mitochondrial termini ([Griffin et al., 2005](#ref8)), which may therefore represent a very early stage of mitophagy. Thus, the first requirement was to identify a marker that would enable us to specifically identify mitochondria destined for degradation. Previously, we and others showed that Atg32 and Atg11 directly participate in an early event of mitophagy ([Aoki et al., 2011](#ref11); [Kanki et al., 2009b](#ref12); [Kondo-Okamoto et al., 2012](#ref13); [Okamoto et al., 2009](#ref14)). When mitophagy is induced, Atg32 recruits Atg11 to the mitochondria, and their interaction is required for the delivery of mitochondria to the phagophore assembly site (PAS), the location of autophagosome formation, and eventually into the vacuole for degradation. In mammalian cells, PARK2/Parkin is recruited to, and thereby selectively marks, depolarized mitochondria ([Narendra et al., 2008](#ref15)); in yeast, however, Atg32 is evenly distributed on the mitochondrial tubules, even though only a relatively small portion of the total mitochondrial population will ultimately be degraded ([Figure S1A available online; Mao et al., 2011](#ref16)). Therefore, Atg32 alone cannot serve as the sole marker of the degrading mitochondria.

We took advantage of the bimolecular fluorescence complementation (BiFC) assay ([Sung and Huh, 2007](#ref17)), in which the Venus yellow fluorescent protein (vYFP) is split into two fragments, VN (N terminus of vYFP) and VC (C terminus of vYFP); we fused VN to Atg32 and VC to Atg11 by integrating the corresponding constructs at the chromosomal *ATG32* and *ATG11* loci. Fluorescence from these chimeras can only be observed

Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

**Developmental Cell**

Atg11 Recruits Dnm1 to Degrading Mitochondria

---

### Figure 2. BiFC 32-11 Dots Mark Degrading Mitochondria during Mitophagy

(A) A BiFC assay was performed for Atg32, Atg11, Om45, and Atg9. Cells containing BiFC pairs (Atg32-Atg11 in KDM1501, Atg32-Atg9 in KDM1519, and Atg11-Om45inKDM1520) were cultured in YPL and shifted to SD-N for 1 hr, followed by analysis by fluorescence microscopy; images are representative pictures from single Z-sections. DIC, differential interference contrast. Scale bar, 2 μm.

(B) Quantification of (A). Twelve Z-section images were projected and the percentage of cells that contained 32-11 dots was determined. Standard error was calculated from three independent experiments. *p* < 0.01.

(C) VN-ATG32 VC-ATG11 (KDM1501) cells, transformed with a plasmid encoding Mito-RFP, were cultured in SML and shifted to SD-N from 10 min to 1 hr, and the cell samples were observed by fluorescence microscopy. CellTracker Blue CMAC was used to stain the vacuolar lumen. Arrowheads indicate the 32-11 dots that localized on the mitochondrial reticulum; and arrows indicate the 32-11 dots that localized on the vacuolar periphery. All of the images are representative pictures from single Z-sections. DIC, differential interference contrast. Scale bar, 2 μm. The inset in row 5, panel 4 corresponds to the large vacuole, reducing the red intensity to demonstrate that the intravacuolar punctum corresponds to a green 32-11 dot that has not yet been degraded. See also **Figure S1**.

(Stage 0)

(SD-N) conditions. When expressing both VN-Atg32 and VC-Atg11, ~9% of the cells showed vYFP dots in growing conditions; however, when mitophagy was induced, ~42% of the cells displayed vYFP dots, indicating the colocalization of these two proteins (**Figures 2A and 2B**). In contrast, neither VN-Atg32 in combination with another autophagy-related protein fusion construct, Atg9-VC, nor a different chimera containing the Om45 mitochondrial outer membrane protein fused to VN (Om45-VN) in combination with VC-Atg11 generated fluorescent puncta in either growing or mitophagy-inducing conditions (**Figure 2A**). These results were consistent with our previous report that the interaction of Atg32 and Atg11 is enhanced (Kanki et al., 2009b). The chimeric constructs were stable under mitophagy-inducing conditions and did not show an appreciable change in protein level, indicating that the appearance of the fluorescent dots reflected colocalization and was not due to a change in protein concentration (**Figure S1B**).

Developmental Cell 26, 1–10, July 15, 2013 © 2013 Elsevier Inc. 3

Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

**Developmental Cell**

Atg11 Recruits Dnm1 to Degrading Mitochondria

We next asked whether the VN-Atg32-VC-Atg11 BiFC fluorescent puncta (hereafter referred to as 32-11 dots) represented degrading mitochondria. We first examined the activity of mitophagy in the presence of the fusion proteins VN-Atg32 and VC-Atg11. Wild-type and VN-Atg32-VC-Atg11 cells were grown in a nonfermentable carbon source and then shifted to SD-N to induce mitophagy. The mitoPho8Δ60 activity of VN-Atg32-VC-Atg11 cells was similar to that of wild-type cells, which indicated that mitophagy occurred normally when Atg32 and Atg11 were fused with VN and VC, respectively (Figure S1C). The atg32Δ strain served as a negative control, and displayed only a background level of activity. Second, we stained the mitochondria with MitoTracker Red dye and examined the localization of the 32-11 dots relative to the organelle. When mitophagy was induced following a 30 min shift to SD-N, most of the 32-11 dots were localized on the mitochondrial reticulum (Figure S1D). Subsequently, we used the FM 4-64 dye to mark the vacuole. Immediately after the shift to SD-N, no 32-11 dots were detected in proximity to the vacuole (data not shown). Within 1 hr after mitophagy induction, however, we detected 32-11 dots localized close to the vacuole limiting membrane (Figure S1E); this perivacuolar location likely corresponds to the PAS. Finally, by 6 hr after shifting to SD-N, very few 32-11 dots could be detected in wild-type cells; however, atg1Δ mutant cells that are defective in mitophagy continued to accumulate 32-11 dots, which indicated that the degradation of mitochondria marked by the VN-Atg32-VC-Atg11 BiFC interaction was dependent on an intact autophagy pathway (Figures S1F and S1G). Therefore, based on the observations that (1) VN-Atg32 and VC-Atg11 are functional fusion proteins; (2) 32-11 dots localized on mitochondria shortly after inducing mitophagy; (3) at a later time point, the 32-11 dots subsequently showed a vacuole-peripheral PAS localization; and (4) the loss of the 32-11 signal was dependent on autophagy, we concluded that the 32-11 dots were an appropriate marker to monitor degrading mitochondria.

In order to clarify the sequential steps of mitophagy, we used a plasmid containing red fluorescent protein targeted to mitochondria (Mito-RFP) and CellTracker Blue CMAC dye to mark the mitochondria and vacuole lumen, respectively, which enabled us to observe both organelles and determine the localization of the 32-11 signal. In growing conditions, when mitophagy was not induced, very few 32-11 dots were detected, which was defined as stage 0 (Figure 2C, row 1). After a short time of starvation (10–30 min), when mitophagy was initiated, 32-11 puncta were formed on the mitochondrial reticulum, which was defined as stage 1 (Figure 2C, rows 2 and 3). After longer times of starvation (approximately 40–60 min), even though most of the 32-11 puncta were still localized on the mitochondrial reticulum, some of the puncta were detected in proximity to the vacuole, which corresponded to the presence of mitophagosomes. We defined this step as stage 2 (Figure 2C, row 4). The earliest we were able to detect the vYFP signal in the vacuole was after approximately 50 min starvation, and the RFP signal was also seen in the vacuole at this time, which suggested the degradation of mitochondria in the vacuole lumen. We defined this step as stage 3 (Figure 2C, rows 5 and 6).

---

**Dnm1 Is Recruited to the Degrading Mitochondria through Its Interaction with Atg11**

Dnm1 assembles specifically at the sites where mitochondrial fission occurs. Therefore, we next asked whether Dnm1 accumulated on the degrading mitochondria, which were marked by 32-11 dots. Accordingly, we chromosomally tagged Dnm1 with mCherry and examined its localization together with the 32-11 dots. Several Dnm1-mCherry puncta that colocalized with 32-11 dots were seen when mitophagy was induced by nitrogen starvation, and this colocalization occurred on the mitochondrial reticulum, which was marked by blue fluorescent protein targeted to mitochondria (Mito-BFP) (Figure 3A). We also chromosomally tagged Mdv1 and Caf4 with mCherry. Similar to Dnm1-mCherry, both Mdv1-mCherry and Caf4-mCherry were colocalized with 32-11 dots (Figures S2A and S2B).

Based on these observations, we hypothesized that there were two groups of Dnm1, one of them assembling on the mitochondria destined for degradation in order to promote mitophagy-specific fission and the other functioning in the normal process of constitutive mitochondrial division. A key question with regard to this first population would be the mechanism through which Dnm1 can be specifically recruited to sites that correspond to mitochondria that are destined for degradation. We hypothesized that Atg32 or Atg11 might be able to interact with Dnm1, Fis1, Mdv1, and/or Caf4. To determine whether Atg32 or Atg11, which we just showed mark this population of mitochondria, played a role in Dnm1 localization, we constructed eight strains expressing different combinations of Atg11, Atg32 and the fission machinery components as BiFC chimeras: VC-Atg11 coexpressed with Dnm1-VN, VN-Atg11 with VC-Fis1, VC-Atg11 with Mdv1-VN, VC-Atg11 with Caf4-VN, VN-Atg32 with Dnm1-VC, VN-Atg32 with VC-Fis1, VN-Atg32 with Mdv1-VC, and VN-Atg32 with Caf4-VC; we used the BiFC assay to test the interactions of these chimeric pairs. All eight strains were tested in both growing (YPL) and mitophagy-inducing (SD-N) conditions. The fluorescent signal could only be observed in cells expressing VC-Atg11 and Dnm1-VN or VN-Atg11 and VC-Fis1 (Figure S2C). The chimeric constructs were stable and did not change in expression level during the course of the analysis, indicating that the fluorescent signals reflected protein localization rather than changes in protein concentration (Figure S2D).

To extend our analysis, we then examined where the interactions occurred. Therefore, we stained the mitochondria with MitoTracker Red dye and examined the localization of the Atg11-Dnm1 and Atg11-Fis1 dots relative to the organelle. Atg11-Fis1 dots were localized on the mitochondrial reticulum in both growing and starvation conditions (Figure 3B). However, in VC-Atg11 Dnm1-VN cells, although Atg11-Dnm1 dots were observed on the mitochondrial reticulum, the mitochondrial morphology was abnormal, which resembled that in dnm1Δ cells. It was reported that the BiFC assay allows the detection of weak and transient interactions, but that these interactions are stabilized by the formation of an intact YFP. Thus, we considered that the stabilized binding of VC-Atg11 and Dnm1-VN prevented the free Dnm1, which functions as an oligomer, from participating in the normal process of constitutive mitochondrial fission. Therefore, in order to provide additional free Dnm1 in the

Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

**Developmental Cell**

Atg11 Recruits Dnm1 to Degrading Mitochondria

A
32-11     Dnm1-mChe    Mito-BFP    overlap    DIC
![Image](image1)

C
VC-Atg11
Dnm1-VN Dnm1-mChe    overlap    DIC
![Image](image2)

B
Strain: VC-ATG11 DNM1-VN
Vector:
Growing    Starvation
vYFP
MitoTracker
overlap
DIC
![Image](image3)
VC-ATG11 DNM1-VN
Dnm1-3HA
Growing    Starvation
vYFP
MitoTracker
overlap
DIC
![Image](image4)
VC-ATG11
Dnm1-VN
Growing    Starvation
vYFP
MitoTracker
overlap
DIC
![Image](image5)
VN-ATG11 VC-FIS1
Growing    Starvation
vYFP
MitoTracker
overlap
DIC
![Image](image6)

**Figure 3. Atg11 Recruits Dnm1 to the Degrading Mitochondria**

(A) VN-ATG32 VC-ATG11 DNM1-mCherry cells, transformed with a plasmid encoding Mito-BFP, were cultured in SML and shifted to SD-N for 1 hr, and samples were observed by fluorescence microscopy. Arrowheads indicate the colocalized 32-11 dots with Dnm1-mCherry on the mitochondrial reticulum. All of the images are representative pictures from single Z-sections. DIC, differential interference contrast. Scale bar, 2 μm.

(B) VC-ATG11 DNM1-VN cells transformed with empty vector or pDnm1-3HA, VC-ATG11 cells transformed with pDnm1-VN, and VN-ATG11 VC-FIS1 cells transformed with empty vector were cultured in SML and shifted to SD-N for 1 hr. Samples were observed by fluorescence microscopy as in (A). Scale bar, 2 μm.

(C) VC-ATG11 DNM1-mCherry cells, transformed with pDnm1-VN, were cultured in SML and shifted to SD-N for 30 min. Samples were observed by fluorescence microscopy as in (A). Scale bar, 2 μm.

See also **Figure S2**.

cells, we transformed either a plasmid harboring Dnm1-3HA into the VC-Atg11 VN-Dnm1 cells or a plasmid harboring Dnm1-VN into the chromosomally integrated VC-Atg11 cells. In both cases, the mitochondrial morphology was normal and the Atg11-Dnm1 dots were localized on the mitochondrial reticulum (**Figure 3**B).

To further demonstrate the existence of two different populations of Dnm1, one of which interacted with Atg11, we asked whether the localization of Dnm1 was affected when Atg11 or Atg32 was absent. Accordingly, we chromosomally tagged Dnm1 with GFP in wild-type, atg11Δ, and atg32Δ cells, and tracked the mitochondria with a plasmid-driven Mito-RFP. In both growing and nitrogen-starvation conditions, the cellular pattern of Dnm1-GFP and mitochondrial morphology was normal in either atg11Δ or atg32Δ cells, which suggested that neither Atg11 nor Atg32 would affect the function of the normal mitochondrial fission machinery (**Figure S2**E). We also noticed that even though many Dnm1-GFP puncta were detected in each cell (**Figure S2**E), only a few Dnm1-Atg11 interacting dots were formed (**Figure 3**B), which also supported our hypothesis that there were two different population of Dnm1. To directly observe these two groups of Dnm1, we generated a yeast strain with VC-Atg11 and Dnm1-mCherry on the genome and transformed these cells with a plasmid harboring Dnm1-VN. The Dnm1 puncta corresponded to an oligomeric mixture of both Dnm1-mCherry and Dnm1-VN, and as a result all of the Dnm1 puncta showed a mCherry signal; however, only the small population associated with VC-Atg11 also showed a vYFP signal (**Figure 3**C).

A previous report demonstrated that mitochondrial fission happens at the ER-mitochondria contact sites ([Friedman et al., 2011](https://doi.org/10.1016/j.cell.2011.07.018)). We wondered whether mitophagy-specific fission also occurred at these sites. Accordingly, we cotransformed plasmids containing Mito-BFP and HDEL-DsRed to track the

Developmental Cell 26, 1–10, July 15, 2013 ©2013 Elsevier Inc. 5

Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

Developmental Cell

Atg11 Recruits Dnm1 to Degrading Mitochondria

**Figure 4. The ER Participates in Mitophagy-Specific Fission**

(A) VN-ATG32 VC-ATG11 (KDM1501) cells, transformed with plasmid encoding Mito-BFP and pHDEL-DsRed, were cultured in SML and shifted to SD-N for 30 min.

(B and C) VN-ATG32 VC-ATG11 MDM12-mCherry (KDM1561) and VN-ATG32 VC-ATG11 MDM34-mCherry (KDM1562) cells were cultured in YPL and shifted to SD-N for 30 min.

The cells in (A, B, and C) were analyzed by fluorescence microscopy. The images are representative pictures from single Z-sections. DIC, differential interference contrast. Scale bars, 2 μm.

examined the interaction of Dnm1 mutants with Atg11 by the BiFC assay. Almost no interacting dots were observed when the GED (~87 amino acids at the C terminus) was deleted (Figures 6A and 6B), which implied that the GED is required for binding to Atg11. We then made truncations of Dnm1 from the C terminus. The interacting dots were detected when we deleted the last 24 amino acids (Dnm1-VN24Δ); however, very few fluorescent dots were seen when the last 30 amino acids were absent (Dnm1-VN30Δ) (Figure 6A and B). To verify the observation from the BiFC assay, we carried out protein A affinity isolation with immunoglobulin G (IgG)-Sepharose. Protein-A-tagged wild-type Dnm1 or the mutant without the last 24 amino acids coprecipitated hemagglutinin (HA)-Atg11; however, the Dnm1 mutant lacking the last 30 amino acids was not able to precipitate this protein (Figure 6C).

We hypothesized that the six amino acids (EDQTLA) that exist in Dnm1 24Δ, but not Dnm1 30Δ, might be important for Dnm1 binding to Atg11. Therefore, we made two Dnm1 mutants: Dnm1 4R, in which the first four amino acids, EDQT, were substituted with arginine, and Dnm1 5A, in which the first five amino acids, EDQTL, were substituted with alanine. We then examined the interaction of these two Dnm1 mutants with Atg11 by the BiFC assay. Interacting dots were detected with wild-type Dnm1, but almost none were seen with the Dnm1 4R or 5A mutants (Figures 6D and 6E). Consistent with this result, protein-A-tagged Dnm1 4R or 5A were not able to coprecipitate HA-Atg11 by affinity isolation using IgG-Sepharose (Figure 6F). Furthermore, these mutants resulted in a block in mitophagy activity similar to that seen with the complete absence of Dnm1 (Figure S3A).

Even though Dnm1 4R or 5A lost the ability to interact with Atg11, we asked whether these two mutants still retained the normal function of Dnm1. Therefore, we used the plasmid-driven Mito-RFP to determine the mitochondrial morphology in the presence of the Dnm1 4R or 5A mutant. Mitochondrial morphology was normal in wild-type cells, whereas enlarged mitochondria were seen in *dnm1Δ* cells (Figure S3B). The

**The Dnm1-Atg11 Interaction Is Required for Mitophagy**

Dnm1 contains four domains: GTPase, middle, insert B, and GED (GTPase effector domain) (Fukushima et al., 2001; Figure 5). In order to find a Dnm1 mutant that lost its binding to Atg11, we

mitochondria and ER, respectively. We found that some of the 32-11 dots were indeed localized at the ER-mitochondria contact sites (Figure 4A). Previous work also reported an ERMES (ER-mitochondria encounter structure) complex localized at the ER-mitochondria contact sites (Kornmann et al., 2009). This complex includes four proteins: Mmm1 and Mdm12, which are on the ER side, and Mdm10 and Mdm34, which are on the mitochondria side of the complex. We fused mCherry at the C terminus of Mdm12 or Mdm34 in the yeast genome and observed the localization of these proteins together with the 32-11 dots. We observed the colocalization of 32-11 dots with some of the Mdm12-mCherry and Mdm34-mCherry puncta (Figures 4B and 4C). These observations suggested that the ER, in particular at the ER-mitochondria contact sites, might also participate in mitophagy-specific fission.

Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

**Developmental Cell**

Atg11 Recruits Dnm1 to Degrading Mitochondria

Dnm1 | GTPase | Middle | Insert B | GED
--- | --- | --- | --- | ---
GEDΔ | GTPase | Middle | Insert B |
24Δ | GTPase | Middle | Insert B |
30Δ | GTPase | Middle | Insert B |
WT |  |  |  |
5A |  |  |  |
4R |  |  |  |

Figure 5. Domain Structure of Dnm1 and GED Mutations

The domains of Dnm1 are depicted in the top diagram, and the position of the GED domain (amino acids 670–757) is indicated. The C-terminal truncations, GEDA, 24Δ, and 30Δ are depicted in the middle diagrams. The sequence of the six amino acid residues comprising E728 through A733 in the GED that are required for the interaction with Atg11 and the mutations 5A and 4R are shown at the bottom.

delivers resident hydrolases to the vacuole, the size of the cargo is relatively small; however, when the primary cargo of the Cvt pathway is overexpressed, it forms a larger complex that is no longer efficiently sequestered (Baba et al., 1997), indicating that there is a size limit for Cvt vesicle formation. Similarly, size may be an issue for organelles that are destined for degradation, especially for organelles such as mitochondria, which exist largely as an extended, reticular structure. Therefore, it is possible that large organelles need to be divided into smaller pieces in order to be effectively engulfed by phagophores.

Here, we showed that mitochondrial division is important for mitophagy, which leads to the question of how mitochondrial fission participates in this process. If fission and mitophagy occur independently, we could imagine that mitochondrial fission happens constitutively and only the resulting small mitochondrial fragments would be chosen for degradation. In this case, there would be no requirement for a direct connection between the fission complex and the mitophagy machinery. In contrast, if fission and mitophagy occur in a coordinated manner, then the mitochondria destined for degradation should be selected first and then the fission complex would be recruited to drive the separation of these mitochondria from the mitochondrial reticulum. The direct interaction between Atg11 and Dnm1 supports the second model. Therefore, we propose that when mitophagy is induced, Atg32 recruits Atg11 to the degrading mitochondria. Atg11 in turn brings Dnm1 and other mitochondrial fission proteins to these “marked” mitochondria and promotes their division. These small fragments of mitochondria are subsequently transported to the PAS where other Atg proteins accumulate, initiating the formation of mitochondria-specific autophagosomes (mitophagosomes).

As a scaffold protein and selective autophagy adaptor, Atg11 binds to a variety of cargo receptors to mediate different types of selective autophagy. For example, Atg11 binds to Atg19, Atg32, and Atg36 for cargo selection during the Cvt pathway, mitophagy, and pexophagy, respectively (Kanki et al., 2009b; Kim et al., 2001; Motley et al., 2012; Shintani et al., 2002; Yorimitsu and Klionsky, 2005). Atg11 also interacts with Atg1 and Atg17, which connects the step of cargo selection to the initiation of autophagosome formation (Yorimitsu and Klionsky, 2005). Here, we unveil a role for Atg11: recruiting mitochondrial fission machinery to facilitate mitophagy.

Previous results indicated that Atg32 is evenly distributed on the mitochondria, even in mitophagy-inducing conditions (Mao et al., 2011). In contrast, the BiFC Atg32-Atg11 interacting pair displayed a punctate pattern on the mitochondrial tubules. We suggest that these dots represent the degrading mitochondria. However, it remains unknown as to how Atg11 recognizes and binds only the Atg32 that marks the organelles, or segments of the organelles, that are to be degraded. Thus, some other factor(s) might determine this very early event in mitophagy.

The degradation of peroxisomes and chloroplasts through autophagy pathways in *Pichia pastoris*, *S. cerevisiae*, *Arabidopsis thaliana*, and other organisms has been reported (Farré et al., 2008; Hutchins et al., 1999; Motley et al., 2012; Wada et al., 2009). It was also reported that the mitochondrial fission complex (Dnm1, Fis1, Mdv1, and Caf4) controls the fission of peroxisomes in yeast and peroxisomes and chloroplasts in

**DISCUSSION**

Autophagy regulates the degradation of cytoplasmic components and organelles. In nonselective autophagy, the amount of Atg8 controls the size of the autophagosomes (Xie et al., 2008). In contrast, during selective types of autophagy, the phagophore membrane is in close apposition to the cargo, excluding bulk cytoplasm. Thus, partly different mechanisms may be involved in determining curvature of the phagophore and the ultimate size of the autophagosome. In the cytoplasm-to-vacuole targeting (Cvt) pathway, a biosynthetic route that

Developmental Cell 26, 1–10, July 15, 2013 © 2013 Elsevier Inc. 7

Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

Developmental Cell

Atg11 Recruits Dnm1 to Degrading Mitochondria

plants (Kuravi et al., 2006; Motley et al., 2008; Zhang and Hu, 2010). Thus, we propose that the mechanism we show in this study (receptor-scaffold/adaptor-fission complex) may also be relevant in selective pexophagy and chloroplast autophagy.

EXPERIMENTAL PROCEDURES

Strains, Media, and Growth Conditions
See Supplemental Experimental Procedures for a description of the yeast strains used in this paper. Yeast cells were grown in rich (YPD; 1% yeast extract, 2% peptone, and 2% glucose) or synthetic minimal (SMD; 0.67% yeast nitrogen base, 2% glucose, and auxotrophic amino acids and vitamins as needed) media. For mitochondria proliferation, cells were grown in lactate medium (YPL; 1% yeast extract, 2% peptone, and 2% lactic acid) or synthetic minimal medium with lactic acid (SML; 0.67% yeast nitrogen base, 2% lactic acid, and auxotrophic amino acids and vitamins as needed). Mitophagy was induced by shifting the cells to nitrogen-starvation medium with glucose (SD-N; 0.17% yeast nitrogen base without ammonium sulfate or amino acids and 2% glucose).

Fluorescence Microscopy
For fluorescence microscopy, yeast cells were grown to optical density 600 ~ 0.6 in YPL or SML media and shifted to SD-N for nitrogen starvation. Samples were then examined by microscopy (Delta Vision, Applied Precision) using a 100× objective and pictures were captured with a CCD camera (CoolSnap HQ; Photometrics). For each microscopy picture, 12 Z-section images were captured with a 0.3 μm distance between two neighboring sections. MitoTracker Red (Invitrogen/Molecular Probes) was used to stain the mitochondria, CellTracker Blue CMAC (Invitrogen/Molecular Probes) to stain the vacuolar lumen, and FM 4-64 (Invitrogen) to stain the vacuolar membrane.

Additional Assays
The mitoPho8Δ60 assay and immunoprecipitation were performed as described previously (Kanki et al., 2009a, 2009b).

SUPPLEMENTAL INFORMATION

Supplemental Information includes Supplemental Experimental Procedures, four figures, and one table and can be found with this article online at http://dx.doi.org/10.1016/j.devcel.2013.05.024.

in KDM1533) were cultured in YPL and shifted to SD-N for 1hr. Samples were observed by fluorescence microscopy, and all the images are representative pictures from single Z-sections. DIC, differential interference contrast.
(B) Quantification of (A). 12Z-section images were projected and the percentage of cells that contained BiFC Dnm1-Atg11 dots was determined. Standard error was calculated from three independent experiments.*p<0.01.
(C) The plasmid pCuHA-Atg11 was transformed into atg11Δ (YTS147), atg11ΔDNM1-PA (KDM1247), atg11ΔDNM1(24Δ)-PA (KDM1248), and atg11ΔDNM1(30Δ) (KDM1249) cells. Cells were cultured in SML and shifted to SD-N for 1.5hr. Cell lysates were prepared and incubated with IgG-Sepharose for affinity isolation as described in Experimental Procedures. The eluted proteins were separated by SDS-PAGE and detected with monoclonal anti-HA antibody and an antibody that binds to PA.
(D) VN-ATG11 (KDM1535) cells, transformed with pDnm1-VC, pDnm1(4R)-VC, or pDnm1(5A)-VC, were cultured in SML and shifted to SD-N for 1hr. Samples were observed by fluorescence microscopy as in (A).
(E) Quantification of (D). Twelve Z-section images were projected and the percentage of cells that contained BiFCDnm1-Atg11dots was determined. Standard error was calculated from three independent experiments.*p<0.01.
(F) The plasmid pCuHA-Atg11 together with pDnm1-VC, pDnm1(4R)-VC, or pDnm1(5A)-VC were transformed into atg11Δdnm1Δ (KDM1251) cells. The cells were cultured in SML and shifted to SD-N for 1.5hr. Cell lysates were prepared and analyzed as in (C).
See also Figure S3.

Figure 6. Mutation of the Dnm1C Terminus Blocks Mitophagy
(A) Cells containing BiFC pairs (Atg11-Dnm1 in KDM1523, Atg11-Dnm1 [GEDA] in KDM1528, Atg11-Dnm1[24Δ] in KDM1532, and Atg11-Dnm1[30Δ]

8 Developmental Cell 26, 1–10, July 15, 2013 © 2013 Elsevier Inc.

Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

**Developmental Cell**

Atg11 Recruits Dnm1 to Degrading Mitochondria

A

| Dnm1-PA: | - | WT | E728R | D729R | Q730R | T731R |
| --- | --- | --- | --- | --- | --- | --- |
| HA-Atg11: | + | + | + | + | + | + |

![Image Description](image)

IP: Dnm1-PA  
Lysate  

B

|  | SML | SD-N |
| --- | --- | --- |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
|  |  |  |
| 
Please cite this article in press as: Mao et al., The Scaffold Protein Atg11 Recruits Fission Machinery to Drive Selective Mitochondria Degradation by Autophagy, Developmental Cell (2013), http://dx.doi.org/10.1016/j.devcel.2013.05.024

Developmental Cell

Atg11 Recruits Dnm1 to Degrading Mitochondria

Motley, A.M., Ward, G.P., and Hettema, E.H. (2008). Dnm1p-dependent peroxisome fission requires Caf4p, Mdv1p and Fis1p. J. Cell Sci. 121, 1633–1640.

Motley, A.M., Nuttall, J.M., and Hettema, E.H. (2012). Pex3-anchored Atg36 tags peroxisomes for degradation in Saccharomyces cerevisiae. EMBO J. 31, 2852–2868.

Mozdy, A.D., McCaffery, J.M., and Shaw, J.M. (2000). Dnm1p GTPase-mediated mitochondrial fission is a multi-step process requiring the novel integral membrane component Fis1p. J. Cell Biol. 151, 367–380.

Narendra, D., Tanaka, A., Suen, D.F., and Youle, R.J. (2008). Parkin is recruited selectively to impaired mitochondria and promotes their autophagy. J. Cell Biol. 183, 795–803.

Narendra, D.P., Jin, S.M., Tanaka, A., Suen, D.F., Gautier, C.A., Shen, J., Cookson, M.R., and Youle, R.J. (2010). PINK1 is selectively stabilized on impaired mitochondria to activate Parkin. PLoS Biol. 8, e1000298.

Naylor, K., Igerman, E., Okreglak, V., Marino, M., Hinshaw, J.E., and Nunnari, J. (2006). Mdv1 interacts with assembled dnm1 to promote mitochondrial division. J. Biol. Chem. 281, 2177–2183.

Okamoto, K., and Shaw, J.M. (2005). Mitochondrial morphology and dynamics in yeast and multicellular eukaryotes. Annu. Rev. Genet. 39, 503–536.

Okamoto, K., Kondo-Okamoto, N., and Ohsumi, Y. (2009). Mitochondria-anchored receptor Atg32 mediates degradation of mitochondria via selective autophagy. Dev. Cell 17, 87–97.

Schweers, R.L., Zhang, J., Randall, M.S., Loyd, M.R., Li, W., Dorsey, F.C., Kundu, M., Opferman, J.T., Cleveland, J.L., Miller, J.L., and Ney, P.A. (2007). NIX is required for programmed mitochondrial clearance during reticulocyte maturation. Proc. Natl. Acad. Sci. USA 104, 19500–19505.

Shintani, T., Huang, W.-P., Stromhaug, P.E., and Klionsky, D.J. (2002). Mechanism of cargo selection in the cytoplasm to vacuole targeting pathway. Dev. Cell 3, 825–837.

Sung, M.K., and Huh, W.K. (2007). Bimolecular fluorescence complementation analysis system for in vivo detection of protein-protein interaction in Saccharomyces cerevisiae. Yeast 24, 767–775.

Tanaka, A., Cleland, M.M., Xu, S., Narendra, D.P., Suen, D.F., Karbowski, M., and Youle, R.J. (2010). Proteasome and p97 mediate mitophagy and degradation of mitofusins induced by Parkin. J. Cell Biol. 191, 1367–1380.

Tieu, Q., and Nunnari, J. (2000). Mdv1p is a WD repeat protein that interacts with the dynamin-related GTPase, Dnm1p, to trigger mitochondrial division. J. Cell Biol. 151, 353–366.

Twig, G., Elorza, A., Molina, A.J., Mohamed, H., Wikstrom, J.D., Walzer, G., Stiles, L., Haigh, S.E., Katz, S., Las, G., et al. (2008). Fission and selective fusion govern mitochondrial segregation and elimination by autophagy. EMBO J. 27, 433–446.

Vives-Bauza, C., Zhou, C., Huang, Y., Cui, M., de Vries, R.L., Kim, J., May, J., Tocilescu, M.A., Liu, W., Ko, H.S., et al. (2010). PINK1-dependent recruitment of Parkin to mitochondria in mitophagy. Proc. Natl. Acad. Sci. USA 107, 378–383.

Wada, S., Ishida, H., Izumi, M., Yoshimoto, K., Ohsumi, Y., Mae, T., and Makino, A. (2009). Autophagy plays a role in chloroplast degradation during senescence in individually darkened leaves. Plant Physiol. 149, 885–893.

Xie, Z., and Klionsky, D.J. (2007). Autophagosome formation: core machinery and adaptations. Nat. Cell Biol. 9, 1102–1109.

Xie, Z., Nair, U., and Klionsky, D.J. (2008). Atg8 controls phagophore expansion during autophagosome formation. Mol. Biol. Cell 19, 3290–3298.

Yorimitsu, T., and Klionsky, D.J. (2005). Atg11 links cargo to the vesicle-forming machinery in the cytoplasm to vacuole targeting pathway. Mol. Biol. Cell 16, 1593–1605.

Zhang, X., and Hu, J. (2010). The Arabidopsis chloroplast division protein DYNAMIN-RELATED PROTEIN5B also mediates peroxisome division. Plant Cell 22, 431–442.
