
Review

**Initiation of Infection**

# Viral evasion mechanisms of early antiviral responses involving regulation of ubiquitin pathways

Ricardo Rajsbaum${}^{1,2}$ and Adolfo García-Sastre${}^{1,2,3}$

${}^{1}$ Department of Microbiology, Icahn School of Medicine at Mount Sinai, One Gustave L. Levy Place, New York, NY 10029, USA  
${}^{2}$ Global Health and Emerging Pathogens Institute, Icahn School of Medicine at Mount Sinai, One Gustave L. Levy Place, New York, NY 10029, USA  
${}^{3}$ Department of Medicine, Icahn School of Medicine at Mount Sinai, One Gustave L. Levy Place, New York, NY 10029, USA  

Early innate and cell-intrinsic responses are essential to protect host cells against pathogens. In turn, viruses have developed sophisticated mechanisms to establish productive infections by counteracting host innate immune responses. Increasing evidence indicates that these antiviral factors may have a dual role by directly inhibiting viral replication as well as by sensing and transmitting signals to induce antiviral cytokines. Recent studies have pointed at new, unappreciated mechanisms of viral evasion of host innate protective responses including manipulating the host ubiquitin (Ub) system. Virus-mediated inhibition of antiviral factors by Ub-dependent degradation is emerging as a crucial mechanism for evading the antiviral response. In addition, recent studies have uncovered new mechanisms by which virus-encoded proteins inhibit Ub and Ub-like (Ubl) modification of host proteins involved in innate immune signaling pathways. Here we discuss recent findings and novel strategies that viruses have developed to counteract these early innate antiviral defenses.

## Host antiviral factors, signaling pathways, and the Ub role in viral responses

Early events in viral infection include attachment to the host cell membrane and entry, release of the viral RNA, DNA, and protein complexes into the cytoplasm of the cell, and early recognition by the host innate immune system. At each of these steps, host cells have developed mechanisms to limit virus infection. In turn, viruses have rapidly evolved to counteract host defenses by different mechanisms. Many recent studies have pointed at the importance of cell type- and host-specific expression of antiviral factors as crucial for limiting viral infection. Furthermore, mammalian cells detect incoming microbes to trigger signaling pathways to produce type I, type II, and type III interferons (IFNs) [1], culminating in the induction of IFN-stimulated genes (ISGs) for the establishment of an antiviral state. Some of the very early antiviral responses include intrinsic restriction factors, which are expressed in sufficient levels to inhibit the first stages of viral replication, including entry or delivery of the genetic material to the required compartment in the cell. These intrinsic restriction factors include proteins that localize to the nucleus and mediate the transcriptional repression of viruses that replicate in this subcellular compartment. Cellular restriction of viral replication relies on a broad constitutively expressed set of antiviral molecules. In addition, IFN triggers expression of inducible ISGs with antiviral activity, and even the expression of many intrinsic viral restriction factors are further increased by IFNs, for example, TRIM5α, tetherin, APOBEC3, and the IFIT family of proteins. IFN production requires recognition of virus components referred to as pathogen-associated molecular patterns (PAMPs) by pattern recognition receptors (PRRs), including Toll-like receptors (TLRs), retinoic acid-inducible gene I (RIG-I)-like receptors (RLRs), NOD-like receptors (NLRs), and C-type lectin receptors (CLRs). Engagement of PRRs triggers downstream signaling pathways resulting in phosphorylation and translocation to the nucleus of multiple transcription factors including IRF3 and IRF7 for type I IFN (IFN-I) production [2]. Binding of secreted IFN-I to its receptor (IFNaR) triggers phosphorylation of signal transducers and activators of transcription 1 (STAT1) and STAT2 by the Jak1/Tyk2 kinases. STAT1, STAT2, and IRF9 form the ISGF3 (interferon stimulated gene factor 3) complex, which is essential for induction of antiviral ISGs. Importantly, in recent years it has become evident that Ub and Ubl modification of IFN signaling components is essential for innate immune signaling function (Figure 1A) [3]. For example, upon binding of viral RNA bearing 5′-triphosphates to RIG-I, a conformational change allows TRIM25 to ubiquitinate the N-terminal 2CARD domain of RIG-I. In addition, TRIM25 may also generate free K63-linked polyubiquitin (poly-Ub) chains that bind to the 2CARD domain of RIG-I, resulting in recruitment to the mitochondrial adaptor protein MAVS. Another E3 Ub ligase, Riplet, has also been shown to ubiquitinate RIG-I. This Ub-dependent interaction between RIG-I and MAVS

Corresponding author: García-Sastre, A. (adolfo.garcia-sastre@mssm.edu).  
Keywords: antiviral; evasion of innate immunity; restriction factors; ubiquitin system.  
0966-842X/$ – see front matter  
© 2013 Elsevier Ltd. All rights reserved. http://dx.doi.org/10.1016/j.tim.2013.06.006

Review                                                                                     Trends in Microbiology August 2013, Vol. 21, No. 8

(A)                                                                                       (B)

TRIM25                                                                                   RIG-I (inactive)
Viral                                                                                     CARDS
5′ triphosphate                                                                            RD
RNA                                                                                       Helicase
Ub                                                                                         Ub
Riplet
Mitochondria                                                                              RIG-I (active)
MAVS
Ub                                                                                         Ub
TRAF3
Ub                                                                                         Ub
TBK1/IKKε                                                                                 TRAF6
Ub                                                                                         Ub
TAK1                                                                                      NEMO
IRF-3                                                                                      IKKα/β
P                                                                                          P
IRF-7                                                                                      IkB
NF-κB
IFNα/β                                                                                    Proinflammatory
cytokines

Figure 1. Regulation of the RIG-I (retinoic acid-inducible gene I) signaling pathway by ubiquitin (Ub). (A) RIG-I is present in the cytoplasm of the cell in an inactive closed conformation. Upon viral infection, 5′-triphosphate (ppp) RNA is produced and recognized by RIG-I through its C-terminal regulatory domain (RD). RIG-I then undergoes a conformational change to expose its 2CARD domain for subsequent ubiquitination by TRIM25. Another E3 ligase, Riplet, ubiquitinates the RD domain of RIG-I. Ubiquitinated RIG-I binds to the mitochondrial adaptor protein MAVS and recruits polyubiquitinated TRAF3 and TRAF6. TRAF3 activates the TBK1/IKKε kinases through Ub-dependent mechanisms for subsequent phosphorylation of IRF3/IRF7 and type I IFN production. Downstream of TRAF6, TAK1 is activated by unanchored poly-Ub chains and promotes activation of NF-κB transcription and cytokine production by Ub-dependent activation of the IKK complex and Ub-dependent degradation of IkB. (B) Ub conjugation is initiated by an E1 activating enzyme in the presence of ATP as an energy source and monoubiquitin. Ub is then transferred to an E2 conjugating enzyme. The E3 ligase confers specificity to the reaction by recognizing the E2 and the substrate for ubiquitination. Deubiquitinases (DUB) hydrolyze poly-Ub chains which are then recycled.

promotes recruitment of multiple factors, including TRAF3 and TRAF6 (TNF receptor-associated factor 3/6). TRAF3 and TRAF6 also employ Ub-dependent mechanisms to activate TBK1/IKKε for IFNα/β production or IKKα/β for subsequent nuclear factor-κB (NF-κB) signaling, respectively. In addition, the IkB kinase undergoes K48-linked polyubiquitination by the β-TrCP E3-Ub ligase complex, resulting in IkB degradation by the proteasome and activation of NF-κB [4] (summarized in Figure 1A).

Interestingly, multiple diverse viruses encode proteins that manipulate the Ub system to inhibit immune signaling and/or promote degradation of antiviral proteins. Antagonism of the IFN system by viruses has been extensively studied and almost every virus has been found to express factors to evade the IFN response. The mechanisms used by different viruses to antagonize the IFN response by inhibiting signaling components have been thoroughly reviewed recently [5,6]. However, new mechanisms of viral antagonism by manipulating the Ub system are only recently starting to emerge. Here we focus on recent studies describing novel early viral evasion mechanisms with an emphasis on those involving the Ub system.

Viral antagonism via the Ub system

Ub is a ubiquitous 76 amino acid protein important in a wide variety of cellular functions, including protein degradation by the proteasome, immune signaling, cell cycle, lysosomal degradation, autophagy, apoptosis, endocytosis, and endoplasmic reticulum-associated degradation [7]. Ub contains seven lysines (K6, K11, K27, K29, K33, K48, K63), each of which can be conjugated by another Ub to form poly-Ub chains. In addition, the C-terminal glycine residue of Ub can be conjugated to a lysine residue of another protein or Ub itself. Ub chains linked through its different lysines have specific cellular functions. Proteins covalently modified with lysine 48 (K48)-linked poly-Ub are usually targeted for degradation by the proteasome. By contrast, protein modification with K63-linked poly-Ub is involved in activation of antiviral signaling pathways [8]. In addition, unanchored K63-linked poly-Ub chains have also been proposed to activate RIG-I and kinases involved in downstream signaling pathways [9]. Ub conjugation requires an E1 activating enzyme, E2 conjugating enzyme, and an E3 ligase which confers specificity by transferring Ub to the target protein (Figure 1B). Ubiquitination is a

highly regulated process, with hundreds of E3 ligases and deubiquitinases (DUBs). Ubiquitination and deubiquitination are high-speed processes that allow turning on and off very quickly different cellular processes. Ub conjugation also plays important roles at different stages of viral infection. For example, the adenovirus internal capsid protein VI, which contains a PPxY motif, is ubiquitinated by Nedd4 E3 Ub ligases. Depletion of Nedd4 ligases attenuates accumulation of incoming adenoviral particles during entry and reduces infectivity, suggesting that ubiquitination is important for adenovirus entry [10].

**Viral antagonism of innate immune signaling pathways by deubiquitination of host factors**

Ub protein conjugation is involved at different levels of the signaling pathways to produce IFNs and other proinflammatory cytokines (Figure 1A). It is then perhaps not surprising that many viruses encode proteins that inhibit ubiquitination processes to overcome host innate responses (Figure 2). For example, nairoviruses and arteriviruses

encode for ovarian tumor (OTU) domain-containing proteases that hydrolyze Ub chains from host proteins. This activity results in the inhibition of NF-κB signaling, IFN induction, and antiviral pathways [11–13]. The papain-like protease domain 2 (PLP2), a catalytic domain of the nonstructural protein 3 (nsp3) of mouse hepatitis virus (MHV)-A59 coronavirus, can bind to IRF3 causing its deubiquitination and preventing its nuclear translocation [14]. It was also shown that nsp3 can remove K63-linked polyubiquitination of TBK1, a kinase involved in IFN induction [14]. Epstein-Barr virus (EBV) codes for the BPLF1 protein, which acts as a deubiquitinase of TRAF6 to inhibit NF-κB signaling during lytic infection, resulting in enhanced lytic replication [15]. Kaposi's sarcoma-associated herpesvirus (KSHV) encodes ORF64, which reduces ubiquitination of the viral sensor RIG-I and consequently inhibits IFN production [16]. The leader proteinase [L(pro)], a papain-like proteinase encoded by foot-and-mouth disease virus (FMDV), inhibits IFN production by removing Ub chains from RIG-I, TBK1, TRAF6, and TRAF3 [17].

![Diagram](attachment:diagram.png)

**Figure 2.** Viruses manipulate the ubiquitin (Ub) system to antagonize innate immune signaling. Upon viral infection, 5′-triphosphate (ppp) RNA is produced and recognized by RIG-I with subsequent ubiquitination by TRIM25 and Riplet. Ubiquitinated RIG-I binds to the adaptor protein MAVS. Then TBK1 is activated for phosphorylation of IRF-3 and IRF-7, and IFNα/β production. The influenza A virus NS1 protein binds to and inhibits TRIM25- and Riplet-dependent ubiquitination of RIG-I. In addition, the Kaposi’s sarcoma-associated herpesvirus (KSHV)-encoded ORF64 protein also inhibits RIG-I ubiquitination and downstream signaling. The leader proteinase [L(pro)] of foot-and-mouth disease virus (FMDV) deubiquitinates RIG-I, TBK1, TRAF6, and TRAF3 to inhibit IFN and NF-κB production. Nsp3 of mouse hepatitis virus (MHV)-A59, deubiquitinates TBK1 and IRF3, preventing activation and nuclear translocation respectively. BPLF1 of Epstein-Barr virus (EBV) deubiquitinates TRAF6 to inhibit NF-κB signaling. The OTU protease of nairoviruses and arteriviruses inhibits NF-κB signaling. TLR3, TLR7/8, and TLR9, which localize in the endosomes, recognize dsRNA, ssRNA, and unmethylated CpG-DNA, respectively. TLR3 triggers signaling pathways through the TRIF adaptor protein whereas TLR7/8 and TLR9 activate the transcription of cytokines and IFNs through MyD88-dependent pathways.

Review                                                                                     Trends in Microbiology August 2013, Vol. 21, No. 8

![Diagram](attachment:diagram.png)

Figure 3. Viral antagonism of restriction factors. The rhesus monkey TRIM5α ortholog promotes early uncoating of HIV by binding to the capsid. TRIM5α has also been linked to activation of innate signaling by catalyzing the synthesis of unanchored K63 poly-ubiquitin (poly-Ub) for NF-κB activation; however, no viral antagonist has thus far been found. APOBEC3G deaminates retroviral genomes causing mutations in the viral genome. HIV-1 Vif protein recruits an E3 Ub ligase complex composed of CUL5, ELOB, ELOC, and RBX that targets APOBEC3G for degradation. SAMHD1, which converts dNTPs to deoxynucleoside and triphosphate, reduces the dNTP supply necessary for HIV-1 replication. Vpx from SIVmac and HIV-2 recruits an E3 Ub ligase complex composed of DCAF1, the adaptor protein DDB1, and CUL4, resulting in degradation of SAMHD1. Tetherin restricts release of enveloped viruses by anchoring the host membrane to the viral membranes and preventing membrane scission. HIV-1 Vpu interacts with tetherin and promotes its Ub-dependent degradation by recruiting the E3 Ub ligase β-TrCP and associated proteins SKP1 and CUL-F. A long isoform of tetherin was found to be an activator of NF-κB signaling. Abbreviation: RT, reverse transcriptase.

TRIM25, a member of the tripartite motif (TRIM) family of E3 Ub ligases required for RIG-I activation and viral sensing, is inhibited by the NS1 protein of influenza A virus (IAV). NS1 binds to the coiled-coil domain of TRIM25, blocking its oligomerization and Ub ligase activity, preventing downstream signaling [18]. Interestingly, this evasion mechanism is species-specific because NS1 is unable to bind and inhibit TRIM25-dependent RIG-I activation in mice. Instead, the NS1 protein targets another E3 ligase, Riplet, that also activates RIG-I (Figure 2). Furthermore, NS1 protein encoded by the avian IAV subtype H5N1 preferentially interacted with TRIM25 and inhibited IFN production in chicken cells [19], despite the lack of functional RIG-I in chickens [20]. This exemplifies the versatility of zoonotic viruses to adapt to different species.

**Viral antagonism by inducing Ub-dependent degradation of antiviral factors**

Restriction factors are cellular proteins that cause direct inhibition of viral replication. The direct interaction of viral and host proteins at this interface leads to rapid evolution and positive selection at the level of both the host and the virus due to evolutionary pressure [21]. As a result, viruses targeted by these restriction factors encode proteins that inhibit their antiviral activity. Restriction factors have been more extensively studied in the context of retrovirus infections, and include members of the TRIM family, APOBEC3 (apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like editing complex) family, and the recently identified SAMHD1, among others. These restriction factors can act at different steps of viral replication but successful retroviruses have acquired viral antagonists that appear to use ubiquitination to target restriction factors to degradation (Figure 3). For example, APOBEC-3G and 3F deaminate retroviral genomes, often producing viral mutants that are incapable of replication [22]. In the absence of the HIV-1 accessory protein Vif, APOBEC3G is packed into virions, resulting in non-infectious particles. APOBEC3G causes cytosine to uracil mutations in single-stranded HIV DNA, which results in mutations in the HIV genome [23]. HIV-1 Vif protein recruits an E3 Ub ligase complex composed of CUL5, ELOB, ELOC, and RBX that targets APOBEC3G for degradation [24].

SAMHD1, a triphosphohydrolase that converts dNTPs to deoxynucleoside and triphosphate, has been reported to inhibit HIV-1 reverse transcription in dendritic cells (DCs) by reducing dNTP supply. The Vpx protein from simian immunodeficiency virus of macaques (SIVmac) and HIV-2 antagonize SAMHD1-mediated restriction. Vpx recruits the DCAF1 and CUL4 E3 Ub ligase complex resulting in degradation of SAMHD1 [25,26].

Tetherin (also known as BST-2) is a transmembrane protein with antiviral activity. Tetherin consists of a transmembrane region, an ectodomain, and a C-terminal glycosylphosphatidylinositol membrane anchor. Tetherin expression is increased in host cells by type I IFNs upon virus infection [27]. In addition, tetherin is highly expressed in plasmacytoid DCs (pDCs), which are highly specialized immune cells that secrete high amounts of IFNs upon viral recognition. Tetherin is a ligand for ILT7, a membrane receptor selectively expressed in pDCs that is involved in inhibition of TLR-mediated interferon responses in pDCs [28]. Tetherin restricts release of enveloped viruses by anchoring the host membrane to the viral membranes, preventing membrane scission and complete virus budding [27]. Because tetherin blocks release of most of the enveloped viruses, it is not surprising that many viruses including HIV-1, HIV-2, SIV, Ebola, and influenza viruses have also developed countermeasures to antagonize tetherin. The Vpu protein encoded by HIV-1 interacts with tetherin and promotes its Ub-dependent degradation by recruiting the E3 Ub ligase β-TrCP complex [29, 30]. SIV does not encode Vpu, but its Nef protein removes tetherin from the membrane [31]. HIV-2 and Ebola envelope glycoproteins also target tetherin [32, 33]. Tetherin appears to block release of IAV budding particles [34], and this effect was partially antagonized by the neuraminidase surface protein of influenza virus [35].

Recently, BclAF1 [Bcl-2 associated factor 1; also termed Btf (Bcl-2 associated transcription factor)], a nuclear protein implicated in apoptosis, transcriptional regulation, RNA processing, and the export of mRNA from the nucleus [36], has been identified as a new restriction factor for human cytomegalovirus (HCMV). Upon infection, pp71 and UL35, delivered by the virion, target BclAF1 to proteasomal degradation. Although the precise mechanism of BclAF1 degradation is still unclear, this may involve both Ub-dependent and -independent mechanisms because pp71 has been reported to induce Ub-independent proteasomal degradation of proteins [37], whereas the UL35 protein has been shown to interact with components of the CUL4 E3 Ub ligase complex (DCAF1, DDB1, and DDA1) [38]. The best-characterized substrate of pp71, the hDaxx/ATRX complex which intrinsically represses HCMV infection [39, 40], appears to be degraded in a proteasome-dependent Ub-independent manner [37]. The Ub-independent mechanism of Daxx degradation by pp71 is unusual because it requires the 19S regulatory subunit of the proteasome which normally participates in Ub-dependent protein degradation [41]. In addition to BclAF1 degradation by pp71 and UL35, later during viral replication BclAF1 is downregulated by microRNA miR-UL112-1, which is encoded by the virus [42].

**Viral antagonism of IFN signaling by targeting STAT proteins for degradation**

The NS5 protein of dengue virus (DENV) inhibits IFN-I signaling by promoting degradation of STAT2, an essential component of the ISGF3 complex required for ISG induction [43]. Recently, the UBR4 member of the N-recognin family of E3 ligases was found to be required for DENV-mediated STAT2 degradation and efficient

DENV replication [44]. Other viruses that target STAT factors required for IFN signaling to degradation by ubiquitination include hepatitis C virus (HCV), which targets STAT3 [45], and herpes simplex virus type 2 (HSV-2) and HCMV, which target STAT2 [46, 47]. In addition, the V protein encoded by the paramyxoviruses simian virus 5 (SV5) and type 2 human parainfluenza virus (HPIV2) target STAT1 and STAT2 for Ub-dependent degradation, respectively, whereas mumps virus V protein targets both STAT1 and STAT3 [48]. The respiratory syncytial virus (RSV) NS1 protein also targets STAT2 for Ub-dependent degradation using the elongin-cullin E3 ligase complex [49]. Thus it is now clear that viruses not only target host proteins involved in antiviral signaling pathways but also manipulate the highly conserved host Ub machinery to evade innate immune responses. These examples of viral antagonism also underscore the importance of Ub in host innate antiviral function.

**Dual role of antiviral factors: restriction activity versus innate immune signaling and their interplay with viruses**

Recent work is emerging to suggest that restriction factors may have a dual role in establishing efficient innate antiviral responses. These factors restrict viral replication by directly binding to and blocking the function of important viral proteins. In addition, during viral infection these same restriction factors promote innate immune signaling pathways to establish an induced antiviral state. Antiviral factors with dual roles might confer an extra layer of protection against viruses. In turn, viruses that have adapted to counteract these particular antiviral factors may have an advantage to establish productive infections. Some antiviral factors with dual roles described to date include members of the TRIM and IFIT families and tetherin.

**TRIMs**

Members of the TRIM family of E3 Ub ligases have been recently shown to have multiple effects as antiviral factors. TRIMs may act both as direct restriction factors and as antiviral factors that promote innate immune signaling pathways. The best-characterized TRIM with viral restriction activity is TRIM5α. It was found that, in contrast to human TRIM5α, the rhesus monkey TRIM5α ortholog is very efficient in inhibiting HIV-1 replication [50]. Non-human primate TRIM5α targets the HIV-1 capsid and promotes early uncoating, preventing subsequent steps of replication in monkeys. The mechanism of viral restriction by TRIM5α appears to involve capsid degradation and proteasome-dependent and -independent mechanisms [51]. In addition to its viral restriction activity, TRIM5α has also been linked to activation of innate signaling by catalyzing the synthesis of unanchored K63 poly-Ub chains that activate TAK1 kinase for subsequent NF-κB activation [52]. Although no viral protein has yet been found to antagonize TRIM5α activity directly, mutations in the capsid have been found to evade its viral restriction function [53]. However, it is unclear whether viruses directly target TRIM5α to antagonize its innate signaling function.

A few other TRIMs have been proposed to act as restriction factors of retroviruses by direct inhibition at different stages of virus replication [54]. TRIMs exemplify the constant conflict between host and viruses. The large number of TRIMs sharing similar domain organization and high sequence homology suggests that they evolved by gene duplications, potentially providing primates and other mammals with new antiviral genes targeting evolving pathogens. Consistent with this notion, several TRIMs show evidence of positive selection, suggesting constant evolutionary pressure by viruses [55]. In addition, TRIMs may act indirectly as antiviral factors because many have been recently shown to act as enhancers of innate immune signaling pathways upon PRR triggering [56, 57]. Furthermore, many TRIMs have been shown to be IFN-inducible and highly expressed in macrophages and DCs [58], underscoring their role in antimicrobial activity. Surprisingly, only a limited number of TRIMs have so far been found to be targeted by viruses. To date only TRIM25 (described above), TRIM19, and TRIM23 have been shown to be targeted by viruses. TRIM19 (also known as PML), a crucial component of nuclear bodies, has been proposed to inhibit a wide variety of viruses including the DNA viruses HSV-1, HCMV, and adenovirus, as well as the RNA viruses human foamy virus (HFV), HIV-1, vesicular stomatitis virus (VSV), rabies, lymphocytic choriomeningitis virus (LCMV), IAV, and poliovirus. Consequently, many viruses encode proteins that interact with TRIM19 resulting in its relocalization and disruption of nuclear bodies. Both LCMV and rabies viruses disrupt the nuclear bodies by the Z protein and the phosphoprotein P, respectively [59]. The ICP0 protein of HSV-1, which is important for the regulation of lytic and latent viral infection, interacts with TRIM19 and disrupts the nuclear bodies in a Ub- and proteasome-dependent manner, resulting in an impaired antiviral response [60, 61]. The viral 3C protease encoded by encephalomyocarditis virus (EMCV) also induces PML degradation by a proteasome and SUMO (small Ub-related modifier) dependent pathway [62].

Recently TRIM21, 32, and 56 were reported to regulate type I IFN production upon double-stranded DNA (dsDNA) virus infection. TRIM21 mediates K48-linked polyubiquitination and degradation of the dsDNA sensor DDX41, whereas TRIM32 and TRIM56 activate the adaptor protein STING by K63-linked polyubiquitination [63–65]. However, no viral antagonists of these TRIMs have yet been found. Finally, TRIM23 appears to be targeted by HCMV. The UL144 protein of HCMV, which is expressed at early times postinfection, interacts with TRIM23 and TRAF6. The formation of the UL144-TRIM23-TRAF6 complex promotes K63-linked autoubiquitination of TRAF6 and NF-κB activation [66]. Interestingly, TRIM23 has also been shown to positively regulate NF-κB signaling by promoting K27-linked polyubiquitination of the regulatory protein NEMO [67], raising the question of whether the HCMV UL144 protein may also regulate NF-κB signaling by TRIM23-dependent ubiquitination of NEMO.

Other restriction factors involved in innate immune signaling

Tetherin has also been recently shown to play a role in immune signaling in addition to its viral restriction activity. Whereas the short isoform of tetherin, which lacks 12 residues, was found to be more resistant to HIV-1 Vpu-mediated downregulation and more efficient in HIV-1 restriction, the long isoform was found to be an activator of NF-κB signaling. Tetherin signaling requires its extracellular domain, also involved in virion retention, and a region in the cytoplasmic tail for TRAF6 recruitment and activation of TAK1 [68, 69]. By contrast, the expression of HIV-1 Vpu inhibited the induction of NF-κB by tetherin; this inhibition required binding of Vpu to the β-TrCP E3 Ub ligase complex [70].

IFN-induced protein with tetratricopeptide repeats 3 (IFIT3, also termed ISG60) forms a complex with IFIT1 and IFIT2 resulting in inhibition of viral replication by sequestering viral RNA [71, 72]. In addition, IFIT3 also has been reported to activate IRF3 for IFN-I production by bridging TBK1 and MAVS [73]. Future studies will be needed to reveal whether viruses encode antagonists that target IFIT and other TRIMs in addition to TRIM19, TRIM23, and TRIM25.

**Viral antagonism of Ubl-dependent antiviral function**

Ubl proteins have also been implicated in antiviral functions, including ISG15 [74], SUMO [75], Nedd8 (neural precursor cell-expressed, developmentally downregulated), FAT10 (HLA-F adjacent transcript 10), MNSFB (monoclonal nonspecific suppressor factor B), and the essential autophagosomal components Atg8 and Atg12 [76]. Thus, viral antagonistic mechanisms of Ubl-modification have also been extensively described. ISG15, which is highly induced by IFNs, has been shown to have antiviral functions against IAV, HSV-1, murine gammaherpesvirus, Sindbis, Ebola, HIV-1, and vaccinia viruses. In turn, some viruses have been described to inhibit host ISGylation including Crimean Congo hemorrhagic fever virus (CCHFV), equine arteritis virus, porcine respiratory and reproductive syndrome virus, and SARS coronavirus (reviewed in [77]). These viruses target ISGylation by encoding ISG15-deconjugating enzymes such as OTU-containing enzymes (described above). Influenza B and vaccinia viruses encode proteins (the NS1 protein of influenza B virus (B/NS1) and the E3L protein of vaccinia virus) that block the transfer of ISG15 from the conjugating ligases to the target proteins [78, 79]. Interestingly, the effect of B/NS1 in antagonizing ISGylation is species-specific because it targets human but not mouse ISGylation [80, 81].

Sumoylation is another Ubl modification that has been shown to be important against viral infection, and multiple viruses have been shown to be targeted by the host SUMO machinery (see [75]). The host antiviral action of SUMO is frequently associated with PML-nuclear bodies, and many viruses have been shown to encode proteins that disrupt the nuclear bodies by targeting TRIM19/PML (discussed above).

**Viral manipulation of autophagy**

Cell autophagy, a metabolic process used to deliver cellular components to degradation and recycling pathways, has been linked to the Ub system at multiple levels. For example, the degradation and clearance of protein aggregates can be achieved by a process of selective autophagy

and involves recognition of ubiquitinated proteins. The deacetylase HDAC 6 binds polyubiquitinated misfolded proteins and targets them to aggresomes for subsequent clearance [82]. The autophagy proteins p62/sequestrome-1 and NBR1 recognize and target polyubiquitinated chains to the autophagosome [83–85]. Although autophagy has been well studied as an intracellular immunosurveillance system in the context of intracellular bacterial [86], autophagy has also been implicated in innate immune responses to viruses, and this process may act immediately following viral entry [87,88]. Neuron-specific depletion of ATG5 and ATG7 autophagy proteins reduced survival upon Sindbis virus infection [89]. TLRs which bind to viral RNA (TLR3, 7, and 8) have been shown to trigger autophagy and limit viral replication [90,91]. Two important components of the autophagy system, ATG8 and ATG12, are Ubl proteins [92], linking Ub-like conjugation to autophagy. Some viruses have also developed countermeasures to subvert autophagy. Mouse cytomegalovirus (MCMV) M45 protein inhibits the inflammatory NF-κB pathway by interacting with NEMO, the regulatory subunit of the IKK complex, and targeting it for degradation by autophagy [93]. The HSV-1 ICP34.5 virulence factor and gammaherpesvirus 68 (γHV68) vbcI2 protein block autophagy by inhibiting beclin-1, a component of the phosphatidylinositol-3 (PI3) kinase complex that recruits other autophagy proteins to the autophagosomal membrane [94,95]. Beclin-1 is also targeted for inhibition by HIV-1 Nef [96]. Moreover, autophagy can enhance adenovirus infections in airway epithelial cells [97]. Importantly, autophagy has recently emerged as a potential therapeutic target for antiviral treatment. For instance, a peptide derived from a region of the autophagy protein beclin-1 (tat-beclin 1) induces autophagy by binding to GAPR-1, a negative regulator of autophagy, and this peptide reduces HIV-1 replication *in vitro* and reduces pathogenicity of Chikungunya and West Nile virus (WNV) *in vivo* [98]. However, antiviral drugs targeting autophagy should be carefully designed because viruses can also induce autophagy for their own advantage. For example, Japanese encephalitis virus (JEV) activates autophagy and increases autophagosome-autolysosome fusion, which is essential for viral replication. In addition, knockdown of autophagy genes reduced JEV replication [99], indicating that autophagy can be beneficial for some viruses. Therefore, autophagy is an important antiviral cellular process that can be manipulated by viruses.

### Antagonism of host innate responses by virus-encoded Ub ligases

In recent years it has also become evident that viruses may encode their own Ub ligases that target host antiviral factors. Several herpesviruses have been shown to encode RING or RING-like Ub ligases that directly interact with host cell Ub machinery or induce Ub-proteasome-dependent degradation. Varicella-zoster virus (VZV) encodes ORF61, an immediate-early phosphoprotein with a RING domain, which inhibits IFN production by targeting IRF3 for Ub-dependent degradation [100]. VZV ORF61 has also been shown to interact with TRIM19 in a SUMO-dependent manner, resulting in disruption of the nuclear bodies [101]. KSHV encodes the K3 and K5 Ub E3 ligases that promote K63-linked polyubiquitination of MHC I, providing a signal for internalization via the endocytic pathway. This results in downregulation of MHC I cell surface expression and evasion of the host immune response [102]. As mentioned above, the ICP10 protein encoded by HSV-1 has been thoroughly studied and was shown to have a RING finger domain that confers E3Ub ligase activity. ICP10 has been shown to promote Ub-dependent degradation of several host proteins involved in antiviral functions [103]. Although viral E3Ub ligases have been identified in several different large DNA viruses, such as herpesviruses and poxviruses [104], E3Ub ligases encoded by RNA viruses appear to be less common. It will be interesting to see if future studies reveal novel Ub ligases encoded by these viruses.

#### Concluding remarks

Increasing evidence shows an important dual role of antiviral factors in direct interaction with viral components and inhibition of viral replication, as well as involvement in signal transduction to induce antiviral cytokines. One common aspect of these antiviral functions is ubiquitination/deubiquitination of viral or cellular proteins, as well as binding to unanchored poly-Ub chains. How viruses have evolved to block these different antiviral functions is only starting to emerge. Furthermore, the development of experimental tools to study other Ub linkages (in addition to K48 and K63-linked poly-Ub chains), which have been lacking to date, should facilitate future studies on the relationship between the Ub system and viral replication. Challenging aspects in the study of virus–Ub interactions include (i) the existence of a large number of host E3Ub ligases which are responsible for substrate specificity, (ii) expression of multiple isoforms of these E3 ligases, (iii) cell type- and host-specific expression, (iv) type of Ub linkages and (v) complex host–pathogen interactions. One potential strategy for studying virus–Ub interactions could be the development of protein microarrays containing poly-Ub chains of different linkage types that can be used to identify Ub-binding proteins encoded by viruses. A similar approach has been described recently for the identification of E3 ligase substrates in yeast [105]. The use of proteomics may facilitate future studies to understand the link between immune signaling and restriction activity of host proteins and will help design better and safer antiviral drugs.

#### Acknowledgments

Research in the laboratory of A.G.S. is supported by National Institute of Allergy and Infectious Diseases (NIAID) grants R01AI046954, U19AI083025, U54AI057158, P01AI058113, P01AI090935, and by the NIAID-funded Centers of Excellence for Influenza Research and Surveillance CEIRS) network, under a contract for a Center of Research in Influenza Pathogenesis (CRIP, HHSN266200700010C).

#### References

1. de Weerd, N.A. and Nguyen, T. (2012) The interferons and their receptors – distribution and regulation. *Immunol. Cell Biol.* 90, 483–491
2. Thompson, M.R. *et al.* (2011) Pattern recognition receptors and the innate immune response to viral infection. *Viruses* 3, 920–940
3. Oudshoorn, D. *et al.* (2012) Regulation of the innate immune system by ubiquitin and ubiquitin-like modifiers. *Cytokine Growth Factor Rev.* 23, 273–282

Review

Trends in Microbiology August 2013, Vol. 21, No. 8

4 Kanarek, N. and Ben-Neriah, Y. (2012) Regulation of NF-kappaB by ubiquitination and degradation of the IkappaBs. Immunol. Rev. 246, 77–94

5 Versteeg, G.A. and Garcia-Sastre, A. (2010) Viral tricks to grid-lock the type I interferon system. Curr. Opin. Microbiol. 13, 508–516

6 Taylor, K.E. and Mossman, K.L. (2013) Recent advances in understanding viral evasion of type I interferon. Immunology 138, 190–197

7 Randles, L. and Walters, K.J. (2012) Ubiquitin and its binding domains. Front. Biosci. 17, 2140–2157

8 Trempe, J.F. (2011) Reading the ubiquitin postal code. Curr. Opin. Struct. Biol. 21, 792–801

9 Zeng, W. et al. (2010) Reconstitution of the RIG-I pathway reveals a signaling role of unanchored polyubiquitin chains in innate immunity. Cell 141, 315–330

10 Wodrich, H. et al. (2010) A capsid-encoded PPxY-motif facilitates adenovirus entry. PLoS Pathog. 6, e1000808

11 van Kasteren, P.B. et al. (2013) Deubiquitinase function of arterivirus papain-like protease 2 suppresses the innate immune response in infected host cells. Proc. Natl. Acad. Sci. U.S.A. 110, E838–E847

12 van Kasteren, P.B. et al. (2012) Arterivirus and nairovirus ovarian tumor domain-containing Deubiquitinases target activated RIG-I to control innate immune signaling. J. Virol. 86, 773–785

13 Frias-Staheli, N. et al. (2007) Ovarian tumor domain-containing viral proteases evade ubiquitin- and ISG15-dependent innate immune responses. Cell Host Microbe 2, 404–416

14 Zheng, D. et al. (2008) PLP2, a potent deubiquitinase from murine hepatitis virus, strongly inhibits cellular type I interferon production. Cell Res. 18, 1105–1113

15 Saito, S. et al. (2013) Epstein-Barr virus deubiquitinase downregulates TRAF6-mediated NF-kappaB signaling during productive replication. J. Virol. 87, 4060–4070

16 Inn, K.S. et al. (2011) Inhibition of RIG-I-mediated signaling by Kaposi’s sarcoma-associated herpesvirus-encoded deubiquitinase ORF64. J. Virol. 85, 10899–10904

17 Wang, D. et al. (2011) The leader proteinase of foot-and-mouth disease virus negatively regulates the type I interferon pathway by acting as a viral deubiquitinase. J. Virol. 85, 3758–3766

18 Gack, M.U. et al. (2009) Influenza A virus NS1 targets the ubiquitin ligase TRIM25 to evade recognition by the host viral RNA sensor RIG-I. Cell Host Microbe 5, 439–449

19 Rajsbbaum, R. et al. (2012) Species-specific inhibition of RIG-I ubiquitination and IFN induction by the influenza A virus NS1 protein. PLoS Pathog. 8, e1003059

20 Barber, M.R. et al. (2010) Association of RIG-I with innate immunity of ducks to influenza. Proc. Natl. Acad. Sci. U.S.A. 107, 5913–5918

21 Duggal, N.K. and Emerman, M. (2012) Evolutionary conflicts between viruses and restriction factors shape immunity. Nat. Rev. Immunol. 12, 687–695

22 Bishop, K.N. et al. (2004) APOBEC-mediated editing of viral RNA. Science 305, 645

23 Harris, R.S. et al. (2012) The restriction factors of human immunodeficiency virus. J. Biol. Chem. 287, 40875–40883

24 Yu, X. et al. (2003) Induction of APOBEC3G ubiquitination and degradation by an HIV-1 Vif-Cul5-SCF complex. Science 302, 1056–1060

25 Ahn, J. et al. (2012) HIV/simian immunodeficiency virus (SIV) accessory virulence factor Vpx loads the host cell restriction factor SAMHD1 onto the E3 ubiquitin ligase complex CRL4DCAF1. J. Biol. Chem. 287, 12550–12558

26 Lahouassa, H. et al. (2012) SAMHD1 restricts the replication of human immunodeficiency virus type 1 by depleting the intracellular pool of deoxynucleoside triphosphates. Nat. Immunol. 13, 223–228

27 Swiecki, M. et al. (2013) BST-2/tetherin: structural biology, viral antagonism, and immunobiology of a potent host antiviral factor. Mol. Immunol. 54, 132–139

28 Cao, W. et al. (2009) Regulation of TLR7/9 responses in plasmacytoid dendritic cells by BST2 and ILT7 receptor interaction. J. Exp. Med. 206, 1603–1614

29 Mitchell, R.S. et al. (2009) Vpu antagonizes BST-2-mediated restriction of HIV-1 release via beta-TrCP and endo-lysosomal trafficking. PLoS Pathog. 5, e1000450

30 Douglas, J.L. et al. (2009) Vpu directs the degradation of the human immunodeficiency virus restriction factor BST-2/Tetherin via a βTrCP-dependent mechanism. J. Virol. 83, 7931–7947

31 Jia, B. et al. (2009) Species-specific activity of SIV Nef and HIV-1 Vpu in overcoming restriction by tetherin/BST2. PLoS Pathog. 5, e1000429

32 Hauser, H. et al. (2010) HIV-1 Vpu and HIV-2 Env counteract BST-2/ tetherin by sequestration in a perinuclear compartment. Retrovirology 7, 51

33 Kaletsky, R.L. et al. (2009) Tetherin-mediated restriction of filovirus budding is antagonized by the Ebola glycoprotein. Proc. Natl. Acad. Sci. U.S.A. 106, 2886–2891

34 Yondola, M.A. et al. (2011) Budding capability of the influenza virus neuraminidase can be modulated by tetherin. J. Virol. 85, 2480–2491

35 Mangeat, B. et al. (2012) Influenza virus partially counteracts restriction imposed by tetherin/BST-2. J. Biol. Chem. 287, 22015–22029

36 Sarra, H. et al. (2010) In search of a function for BCLAF1. ScientificWorldJournal 10, 1450–1461

37 Hwang, J. and Kalejta, R.F. (2007) Proteasome-dependent, ubiquitin-independent degradation of Daxx by the viral pp71 protein in human cytomegalovirus-infected cells. Virology 367, 334–338

38 Salsman, J. et al. (2012) Proteomic profiling of the human cytomegalovirus UL35 gene products reveals a role for UL35 in the DNA repair response. J. Virol. 86, 806–820

39 Saffert, R.T. and Kalejta, R.F. (2007) Human cytomegalovirus gene expression is silenced by Daxx-mediated intrinsic immune defense in model latent infections established in vitro. J. Virol. 81, 9109–9120

40 Lukashchuk, V. et al. (2008) Human cytomegalovirus protein pp71 displaces the chromatin-associated factor ATRX from nuclear domain 10 at early stages of infection. J. Virol. 82, 12543–12554

41 Winkler, L.L. et al. (2013) Ubiquitin-independent proteasomal degradation of tumor suppressors by human cytomegalovirus pp71 requires the 19S regulatory particle. J. Virol. 87, 4665–4671

42 Lee, S.H. et al. (2012) BclAF1 restriction factor is neutralized by proteasomal degradation and microRNA repression during human cytomegalovirus infection. Proc. Natl. Acad. Sci. U.S.A. 109, 9575–9580

43 Rodriguez-Madoz, J.R. et al. (2010) Inhibition of the type I interferon response in human dendritic cells by dengue virus infection requires a catalytically active NS2B3 complex. J. Virol. 84, 9760–9774

44 Morrison, J. et al. (2013) Dengue virus co-opts UBR4 to degrade STAT2 and antagonize type I interferon signaling. PLoS Pathog. 9, e1003265

45 Stevenson, N.J. et al. (2013) Hepatitis C virus targets the interferon-alpha JAK/STAT pathway by promoting proteasomal degradation in immune cells and hepatocytes. FEBS Lett. 587, 1571–1578

46 Le, V.T. et al. (2008) Human cytomegalovirus interferes with signal transducer and activator of transcription (STAT) 2 protein stability and tyrosine phosphorylation. J. Gen. Virol. 89, 2416–2426

47 Kadeppagari, R.K. et al. (2012) HSV-2 inhibits type-I interferon signaling via multiple complementary and compensatory STAT2-associated mechanisms. Virus Res. 167, 273–284

48 Ramachandran, A. and Horvath, C.M. (2009) Paramyxovirus disruption of interferon signal transduction: STATus report. J. Interferon Cytokine Res. 29, 531–537

49 Elliott, J. et al. (2007) Respiratory syncytial virus NS1 protein degrades STAT2 by using the Elongin-Cullin E3 ligase. J. Virol. 81, 3428–3436

50 Stremlau, M. et al. (2004) The cytoplasmic body component TRIM5alpha restricts HIV-1 infection in Old World monkeys. Nature 427, 848–853

51 Sastri, J. and Campbell, E.M. (2011) Recent insights into the mechanism and consequences of TRIM5alpha retroviral restriction. AIDS Res. Hum. Retroviruses 27, 231–238

52 Pertel, T. et al. (2011) TRIM5 is an innate immune sensor for the retrovirus capsid lattice. Nature 472, 361–365

53 Hatziioannou, T. et al. (2006) Generation of simian-tropic HIV-1 by restriction factor evasion. Science 314, 95

54 Uchil, P.D. et al. (2008) TRIM E3 ligases interfere with early and late stages of the retroviral life cycle. PLoS Pathog. 4, e16

55 Han, K. et al. (2011) Identification of a genomic reservoir for new TRIM genes in primate genomes. PLoS Genet. 7, e1002388

Review                                                                 Trends in Microbiology August 2013, Vol. 21, No. 8

56 Versteeg, G.A. *et al.* (2013) The E3-ligase TRIM family of proteins regulates signaling pathways triggered by innate immune pattern- recognition receptors. *Immunity* 38, 384–398

57 Uchil, P.D. *et al.* (2013) TRIM protein-mediated regulation of inflammatory and innate immune signaling and its association with antiretroviral activity. *J. Virol.* 87, 257–272

58 Rajsbaum, R. *et al.* (2008) Type I interferon-dependent and - independent expression of tripartite motif proteins in immune cells. *Eur. J. Immunol.* 38, 619–630

59 Geoffroy, M.C. and Chelbi-Alix, M.K. (2011) Role of promyelocytic leukemia protein in host antiviral defense. *J. Interferon Cytokine Res.* 31, 145–158

60 Boutell, C. *et al.* (2011) A viral ubiquitin ligase has substrate preferential SUMO targeted ubiquitin ligase activity that counteracts intrinsic antiviral defence. *PLoS Pathog.* 7, e1002245

61 Cuchet-Lourenco, D. *et al.* (2012) Herpes simplex virus 1 ubiquitin ligase ICP0 interacts with PML isoform I and induces its SUMO- independent degradation. *J. Virol.* 86, 11209–11222

62 El McHichi, B. *et al.* (2010) SUMOylation promotes PML degradation during encephalomyocarditis virus infection. *J. Virol.* 84, 11634– 11645

63 Zhang, Z. *et al.* (2013) The E3 ubiquitin ligase TRIM21 negatively regulates the innate immune response to intracellular double- stranded DNA. *Nat. Immunol.* 14, 172–178

64 Tsuchida, T. *et al.* (2010) The ubiquitin ligase TRIM56 regulates innate immune responses to intracellular double-stranded DNA. *Immunity* 33, 765–776

65 Zhang, J. *et al.* (2012) TRIM32 protein modulates type I interferon induction and cellular antiviral response by targeting MITA/STING protein for K63-linked ubiquitination. *J. Biol. Chem.* 287, 28646– 28655

66 Poole, E. *et al.* (2009) Identification of TRIM23 as a cofactor involved in the regulation of NF-kappaB by human cytomegalovirus. *J. Virol.* 83, 3581–3590

67 Arimoto, K. *et al.* (2010) Polyubiquitin conjugation to NEMO by tripartite motif protein 23 (TRIM23) is critical in antiviral defense. *Proc. Natl. Acad. Sci. U.S.A.* 107, 15856–15861

68 Cocka, L.J. and Bates, P. (2012) Identification of alternatively translated Tetherin isoforms with differing antiviral and signaling activities. *PLoS Pathog.* 8, e1002931

69 Galao, R.P. *et al.* (2012) Innate sensing of HIV-1 assembly by Tetherin induces NFkappaB-dependent proinflammatory responses. *Cell Host Microbe* 12, 633–644

70 Tokarev, A. *et al.* (2013) Stimulation of NF-kappaB activity by the HIV restriction factor BST2. *J. Virol.* 87, 2046–2057

71 Pichlmair, A. *et al.* (2011) IFIT1 is an antiviral protein that recognizes 5′-triphosphate RNA. *Nat. Immunol.* 12, 624–630

72 Diamond, M.S. and Farzan, M. (2013) The broad-spectrum antiviral functions of IFIT and IFITM proteins. *Nat. Rev. Immunol.* 13, 46–57

73 Liu, X.Y. *et al.* (2011) IFN-induced TPR protein IFIT3 potentiates antiviral signaling by bridging MAVS and TBK1. *J. Immunol.* 187, 2559–2568

74 Skaug, B. and Chen, Z.J. (2010) Emerging role of ISG15 in antiviral immunity. *Cell* 143, 187–190

75 Wimmer, P. *et al.* (2012) Human pathogens and the host cell SUMOylation system. *J. Virol.* 86, 642–654

76 van der Veen, A.G. and Ploegh, H.L. (2012) Ubiquitin-like proteins. *Annu. Rev. Biochem.* 81, 323–357

77 Zhao, C. *et al.* (2013) Interferon-induced ISG15 pathway: an ongoing virus-host battle. *Trends Microbiol.* 21, 181–186

78 Guerra, S. *et al.* (2008) Vaccinia virus E3 protein prevents the antiviral action of ISG15. *PLoS Pathog.* 4, e1000096

79 Yuan, W. and Krug, R.M. (2001) Influenza B virus NS1 protein inhibits conjugation of the interferon (IFN)-induced ubiquitin-like ISG15 protein. *EMBO J.* 20, 362–371

80 Versteeg, G.A. *et al.* (2010) Species-specific antagonism of host ISGylation by the influenza B virus NS1 protein. *J. Virol.* 84, 5423–5430

81 Sridharan, H. *et al.* (2010) Species specificity of the NS1 protein of influenza B virus: NS1 binds only human and non-human primate ubiquitin-like ISG15 proteins. *J. Biol. Chem.* 285, 7852–7856

82 Kawaguchi, Y. *et al.* (2003) The deacetylase HDAC6 regulates aggresome formation and cell viability in response to misfolded protein stress. *Cell* 115, 727–738

83 Kirkin, V. *et al.* (2009) A role for ubiquitin in selective autophagy. *Mol. Cell* 34, 259–269

84 Lamark, T. *et al.* (2009) NBR1 and p62 as cargo receptors for selective autophagy of ubiquitinated targets. *Cell Cycle* 8, 1986–1990

85 Komatsu, M. *et al.* (2007) Homeostatic levels of p62 control cytoplasmic inclusion body formation in autophagy-deficient mice. *Cell* 131, 1149–1163

86 Jo, E.K. *et al.* (2013) Roles of autophagy in elimination of intracellular bacterial pathogens. *Front. Immunol.* 4, 97

87 Jordan, T.X. and Randall, G. (2012) Manipulation or capitulation: virus interactions with autophagy. *Microbes Infect.* 14, 126–139

88 Yordy, B. and Iwasaki, A. (2011) Autophagy in the control and pathogenesis of viral infection. *Curr. Opin. Virol.* 1, 196–203

89 Orvedahl, A. *et al.* (2010) Autophagy protects against Sindbis virus infection of the central nervous system. *Cell Host Microbe* 7, 115–127

90 Xu, Y. *et al.* (2007) Toll-like receptor 4 is a sensor for autophagy associated with innate immunity. *Immunity* 27, 135–144

91 Shi, C.S. and Kehrl, J.H. (2008) MyD88 and Trif target Beclin 1 to trigger autophagy in macrophages. *J. Biol. Chem.* 283, 33175–33182

92 Geng, J. and Klionsky, D.J. (2008) The Atg8 and Atg12 ubiquitin-like conjugation systems in macroautophagy. ‘Protein modifications: beyond the usual suspects’ review series. *EMBO Rep.* 9, 859–864

93 Fliss, P.M. *et al.* (2012) Viral mediated redirection of NEMO/ IKKgamma to autophagosomes curtails the inflammatory cascade. *PLoS Pathog.* 8, e1002517

94 Ku, B. *et al.* (2008) Structural and biochemical bases for the inhibition of autophagy and apoptosis by viral BCL-2 of murine gamma- herpesvirus 68. *PLoS Pathog.* 4, e25

95 Kihara, A. *et al.* (2001) Beclin-phosphatidylinositol 3-kinase complex functions at the trans-Golgi network. *EMBO Rep.* 2, 330–335

96 Kyei, G.B. *et al.* (2009) Autophagy pathway intersects with HIV-1 biosynthesis and regulates viral yields in macrophages. *J. Cell Biol.* 186, 255–268

97 Zeng, X. and Carlin, C.R. (2013) Host cell autophagy modulates early stages of adenovirus infections in airway epithelial cells. *J. Virol.* 87, 2307–2319

98 Shoji-Kawata, S. *et al.* (2013) Identification of a candidate therapeutic autophagy-inducing peptide. *Nature* 494, 201–206

99 Jin, R. *et al.* (2013) Japanese encephalitis virus activates autophagy as a viral immune evasion strategy. *PLoS ONE* 8, e52909

100 Zhu, H. *et al.* (2011) Varicella-zoster virus immediate-early protein ORF61 abrogates the IRF3-mediated innate immune response through degradation of activated IRF3. *J. Virol.* 85, 11079–11089

101 Wang, L. *et al.* (2011) Disruption of PML nuclear bodies is mediated by ORF61 SUMO-interacting motifs and required for varicella-zoster virus pathogenesis in skin. *PLoS Pathog.* 7, e1002157

102 Boname, J.M. *et al.* (2011) Efficient internalization of MHC I requires lysine-11 and lysine-63 mixed linkage polyubiquitin chains. *Traffic* 11, 210–220

103 Boutell, C. and Everett, R.D. (2013) Regulation of alphaherpesvirus infections by the ICP0 family of proteins. *J. Gen. Virol.* 94, 465–481

104 Zhang, L. *et al.* (2009) Interplay between poxviruses and the cellular ubiquitin/ubiquitin-like pathways. *FEBS Lett.* 583, 607–614

105 Gupta, R. *et al.* (2007) Ubiquitination screen using protein microarrays for comprehensive identification of Rsp5 substrates in yeast. *Mol. Syst. Biol.* 3, 116
