
ARTICLE IN PRESS

CLS-07915; No of Pages 13

Cellular Signalling xxx (2013) xxx–xxx

Contents lists available at SciVerse ScienceDirect

Cellular Signalling

journal homepage: www.elsevier.com/locate/cellsig

Review

IL-17 family cytokines mediated signaling in the pathogenesis of inflammatory diseases

Xinyang Song, Youcun Qian*

The Key Laboratory of Stem Cell Biology, Institute of Health Sciences, Shanghai Institutes for Biological Sciences, Chinese Academy of Sciences/Shanghai Jiao Tong University School of Medicine, Shanghai 200025, China

ARTICLE INFO

Article history:
Received 5 July 2013
Accepted 26 July 2013
Available online xxxx

Keywords:
IL-17 family cytokines
Signal transduction
Inflammation

ABSTRACT

Inflammation is the immediate protective response of the body to pathogen invasions, allergen challenges, chemical exposures or physical injuries. Acute inflammation usually accompanies with transient infiltration of leukocytes, removal of danger signals and eventually tissue repair, while persistent and uncontrolled inflammation becomes a major stimulator in the progression of many chronic diseases in human, including autoimmune diseases, metabolic disorders and cancer. Interleukin (IL)-17 family is a recent classified subset of cytokines, playing critical roles in both acute and chronic inflammatory responses. In this review, we will summarize recent progresses on the signalings of this family cytokines and their impacts on the inflammatory responses or disorders.

© 2013 Published by Elsevier Inc.

Contents

1. Introduction . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
ARTICLE IN PRESS

X. Song, Y. Qian / Cellular Signalling xxx (2013) xxx–xxx

Increasing data suggest that a wide spectrum of innate immune cells, including lymphoid-tissue inducer (LTi) cells, γδT cells, invariant natural killer T (iNKT) cells, natural killer (NK) cells, neutrophils and paneth cells, are responsible for rapid production of IL-17A during early innate immune responses [3]. A substantial number of studies demonstrate that IL-17A contributes to both innate and adaptive immune responses during the physiological or pathological processes of autoimmunity, allergy, host defense and cancer [1]. IL-17A exhibits its inflammatory effects by activating NF-κB, MAPKs and C/EBP cascades to induce the production of multiple proinflammatory molecules, including cytokines, chemokines, antimicrobial peptides and matrix metalloproteinases (MMPs) [4]. IL-17A also acts in synergy with TNFα or IL-22 to amplify chemokine gene expression or antimicrobial peptide production [5,6].

The receptor proximal adaptor Act1 (an NF-kappaB activator 1; also known as CIKS, connection to IKK and SAPK/JNK) is considered as the master mediator in IL-17A signaling [7,8]. Although not fully appreciated, Act1 is likely a common signal adaptor also shared by other members mediated signaling in this family [9].

Based on the structure similarity, the cytokine family comprises six members (IL-17A, IL-17B, IL-17C, IL-17D, IL-17E and IL-17F), while its receptor family contains five members (IL-17RA, IL-17RB, IL-17RC, IL-17RD and IL-17RE) [10]. IL-17A shares 16% to 50% identity of amino acid sequence with other family members, in which IL-17F is the most homologous to IL-17A in amino acid sequence, while IL-17E has the least similarity to IL-17A [1]. The C-terminal regions of this family harbor more conserved amino acids. Four of five conserved cysteine residues in C terminus form a distinct cysteine knot structure from that of TGF-β, BMP and NGF superfamilies [11,12]. All IL-17 family receptor members contain two fibronectin III-like domains in their extracellular region and a SEF/IL-17R (SEFIR) domain in their intracellular part [10]. IL-17RA (also known as IL-17R), the typical member in the receptor family, is considered as a commonly shared receptor in this family [13]. Other receptors form different heterodimers with IL-17RA for the different signals mediated by IL-17A to IL-17F. To date, IL-17A and IL-17F signal through IL-17RA and IL-17RC heterodimers, while IL-17E triggers downstream signals through IL-17RA and IL-17RB complex. IL-17RB is also a receptor for IL-17B, although the genetic targeting evidences are still lacking for this pair. According to recent studies, the heterodimer of IL-17RA and IL-17RE serves as the receptors for IL-17C [9,10]. The receptor for IL-17D is still unknown. IL-17RD, originally called Sef, is previously considered as a negative regulator in growth factor mediated pathways [14]. IL-17RD is recently reported as a positive component in IL-17A signaling [15,16]. However, the molecular mechanism of IL-17RD in IL-17A signaling or whether there is a ligand for this orphan receptor still remains to be determined. Of note, in addition to its original family member IL-17A, growing evidences from recent years suggest that other members in this family are also critical for these inflammatory responses or disorders in their distinguishable ways (Fig. 1). In this review, we highlight the up-to-date knowledge of this family cytokines and their impacts on the pathogenesis of inflammatory diseases.

### 2. IL-17A and IL-17F

#### 2.1. Cellular sources

Deficiency of both transcription factors completely inhibited Th17 cell differentiation and IL-17A production either *in vitro* or *in vivo* [29]. T cell specific deficiency of STAT3 led to a deceased Th17 cell differentiation [30,31]. STAT3 was found to be critical for IL-6 and IL-21 signaling during Th17 cell differentiation [23,32,33]. Besides, STAT3 directly bound to IL-17A, IL-17F and IL-21 promoters for their transcription [32,34]. IRF4 was associated with the Th1 and Th2 subsets differentiation. It was recently found that IRF4 also required for the differentiation of Th17 cells [35]. Deficiency of IRF4 in T cells resulted in impaired RORγt induction upon IL-6 and TGFβ stimulation [35]. Other transcription factors, including BATF, Runx1, AHR, IKKa, IkBg and NFATc1, were also discovered to be involved in the commitment of Th17 cells recently [36–42] (Fig. 2).

In addition to Th17, several other lineages of T cells also produce both IL-17A and IL-17F. Those T cells include cytotoxic CD8⁺ T cells (Tc17), IL-17-producing γδT cells (γδ-17) or natural killer T cells (NKT-17) [43]. Unlike naïve CD4⁺ or CD8⁺ T cells, γδT cells constitutively express IL-1R, IL-23R and RORγt, so in the response to IL-23 and IL-1β, those cells directly produce IL-17A and IL-17F without IL-6 and TCR signaling. Beside to T cells, innate lymphoid cells, including monocyte, neutrophils, natural killer cells and lymphoid tissue inducer-like (LTi-like) cells, were reported to be the innate IL-17A and IL-17F producing cells [43]. More recently, during the infection of Trypanosoma cruzi, B cells were showed to be an innate source of IL-17A and IL-17F [44]. Other non-immune cells, like intestinal Paneth cells and colonic epithelial cells were found to have capabilities for producing IL-17A and IL-17F respectively [45,46] (Fig. 2). The innate IL-17A and IL-17F producing cells, either immune or non-immune cells, are responsible for rapid production of those two cytokines during certain emergent conditions such as infection or wounding. Although various cell types can produce IL-17A and IL-17F, the predominant cellular source of them depends on the inflammatory microenvironment. Th17 cells represent the broadest distributed IL-17A and IL-17F producing cells in our body and have a broad influence on the pathogenesis of inflammatory diseases of our body [47]. Innate IL-17A and IL-17F producing cells mostly function as the sentinel cells of the body, are enriched in host mucosa barriers which include lung, intestine, skin, oral cavity and reproductive tract. Both physiological or pathological functions and tissue location of those cells appear to be limited compared to Th17 cells, and remain to be further defined [43,47].

#### 2.2. Signal transduction

##### 2.2.1. Activation of IL-17A signaling pathways

Soon after its discovery in the activated murine T lymphocyte hybridoma cDNA library [48], the proinflammatory property and the downstream signaling pathways of IL-17A were uncovered. IL-17A can activate NF-κB, MAPK and C/EBP cascades to upregulate a variety of proinflammatory chemokines and cytokines expression [9]. IL-17A also synergistically works with TNFα to induce gene expression by posttranscriptional stabilization of mRNAs induced by TNFα [9]. Other studies suggest that IL-17A can activate JAK-PI3K and JAK-STAT pathways [49,50], although exact downstream molecular mechanisms are still unclear. Recently, we found that IL-17A promoted inflammatory responses by downregulation of microRNA-23b in tissue resident cells [51]. IL-17RA was initially found to be the receptor of IL-17A for its downstream signaling [52,53]. Fibroblasts which were deficient of IL-17RA failed to response to IL-17A stimulation [52,54]. Later studies showed that IL-17RC was a receptor component of IL-17A signaling by interacting with IL-17RA in an IL-17A signaling-dependent manner and it also bound to IL-17A as its receptor. IL-17RC deficient mice showed impaired downstream gene expression induction upon IL-17A stimulation [10,55]. Thus, upon IL-17A binding, IL-17RA interacts with IL-17RC to form a heterodimeric receptor complex to activate downstream signaling pathways. The receptor proximal signaling events have been illustrated recently (Fig. 3).

ARTICLE IN PRESS

X. Song, Y. Qian / Cellular Signalling xxx (2013) xxx–xxx

IL-17A          IL-17F          IL-17B          IL-17C          IL-17E          IL-17D
?               ?

IL-17RA         IL-17RC         ?               IL-17RB         IL-17RA         IL-17RE         IL-17RA         IL-17RB         IL-17RD         ?
Act1            ?               Act1            Act1            ?               ?

Autoimmunity    Autoimmunity    Th2 Response
Inflammation    Inflammation
Host Defense    Host Defense

Fig. 1. Overview of IL-17 cytokine and receptor families. IL-17 family cytokines consist of six members, IL-17A to IL17F, while the receptor family contains five members, IL-17RA to IL-17RE. IL-17A and IL-17F, the most characterized members, form either homodimer or heterodimer to bind the heterodimeric IL-17RA and IL-17RC complex and activate downstream signaling for host defense or pathogenesis of autoimmune diseases as well as other inflammatory disorders. Although IL-17B was observed to bind IL-17RB, the biological functions of this pair are still not clear. IL-17C binds to IL-17RA and IL-17RE complex to trigger downstream signaling for host defense as well as the inflammatory pathogenesis of autoimmune diseases. IL-17E associated with IL-17RA and IL-17RB complex for Th2 response related signaling. Neither the receptor for IL-17D nor the legend for IL-17RD has been identified. Act1 has been demonstrated as a common adaptor in IL-17A, IL-17F, IL-17C and IL-17E mediated signaling.

Tumor necrosis factor receptor associated factor (TRAF) proteins, which contain TRAF1 to TRAF6, have multiple roles in the immune system [56]. TRAF6 is the first reported positive adaptor in IL-17A signaling. It was discovered to be critical for IL-17A-induced NF-κB and JNK pathways activation and TRAF6 deficient MEFs showed consequently impaired IL-6 gene induction by IL-17A stimulation [57]. However, the cytoplasm region of IL-17RA does not harbor predicted TRAF6 binding motif and TRAF6 was reported not required for mRNA stabilization pathway induced by IL-17A, indicating that other players upstream of TRAF6 might exist for either TRAF6 dependent or independent pathways [58] (Fig. 3).

Through bioinformatic analysis, all IL-17R family members harbored a SEF and IL-17R (SEFIR) domain in their cytoplasmic part. Additionally, the SEFIR domain structure also existed in a cytosolic protein named Act1 (also called as CIKS) [59]. Indeed, upon IL-17 stimulation, Act1 was recruited to IL-17R by the SEFIR–SEFIR domain interaction [7,8]. TRAF6 was then recruited to Act1, and subsequently bound to IL-17R complex. Act1 not only works as a simple adaptor for recruitment of downstream molecules, but also functions as a U-box like E3 ligase, which ubiquitinates TRAF6 in a Lys63-linked non-degradative way by recruiting the Ubc13–Uev1A E2 complex for IL-17 induced activation of NF-κB pathway [60]. We recently found that upon IL-17A stimulation, both TAB2 and TAB3 were recruited to TRAF6, and silencing of expression of TAK1 or TAB2 together with TAB3 impaired IL-17A-induced activation of NF-κB but not MAPK pathways [51], indicating that the TAK1 complex is specifically required for IL-17-mediated NF-κB signaling. We also found that IL-17A downregulated the microRNA-23b (miR-23b) expression through NF-κB pathway [51]. MiR-23b subsequently targeted TAB2 and TAB3 to suppress either the proinflammatory cytokines (such as IL-17A, TNFα and IL-1β) mediated signaling and their downstream gene induction [51]. Thus, IL-17A initiates a positive feedback loop through activating NF-κB to remove the suppression of miR-23b for further NF-κB activation. Since both Act1 and IL-17R required the SEFIR domains for their binding and signal transduction, more detailed domain mapping data showed that the CC′ loop structure in SEFIR domains of the adaptor and receptor is responsible for their interaction [61]. A decoy peptide which mimicked the structure of Act1 CC′ loop suppressed IL-17A mediated inflammatory responses either in vitro or in vivo [61]. Though the SEFIR domain is critical for IL-17A mediated signaling and inflammatory responses, other biochemistry studies demonstrated that, in addition to SEFIR domain, the N-terminal domain of Act1, the “TIR-like loop”(TILL) motif and C-terminal region of IL-17RA and the cytoplasmic tail region of IL-17RC were also crucial for IL-17A signaling [62–65]. These refined domain analyses provided detailed molecular mechanism of IL-17A signaling and potential new therapeutic targets for IL-17A related diseases (Fig. 3).

TRAF6 was not essential for IL-17A mediated mRNA stabilization of TNFα induced KC while Act1 was required for this process [58], indicating that Act1 mediates both TRAF6 dependent and TRAF6 independent pathways in IL-17A signaling. Recently, the inducible kinase IKKi (also known as IKKε) was reported for the Act1 mediated

Please cite this article as: X. Song, Y. Qian, Cellular Signalling (2013), http://dx.doi.org/10.1016/j.cellsig.2013.07.021

ARTICLE IN PRESS

IL-17A and IL-17F producing cells
- Th17
- Tc17
- γδT
- NKT
- NK cell
- LTi-like cell
- Monocyte
- Neutrophil
- B cell
- Epithelial cell
- Paneth cell

IL-17E producing cells
- DC
- Macrophage
- T cell
- Eosinophil
- Mast cell
- Basophil
- Epithelial cell
- Paneth cell

IL-17C producing cells
- Epithelial cell
- Endothelial cell
- Keratinocyte
- Leukocyte

IL-17A
IL-17F
IL-17E
IL-17C

IL-17A and IL-17F targeting cells
- Epithelial cell
- Keratinocyte
- Endothelial cell
- Macrophage
- T cell
- B cell
- Fibroblast
- Muscle cell

IL-17E targeting cells
- Th2
- Th9
- NKT
- Macrophage
- Nuocyte
- ILC2 cell
- MMP-type 2 T2M cell
- NBNT cell
- Epithelial cell

IL-17C targeting cells
- Epithelial cell
- Keratinocyte
- Th17
- Monocyte

Fig. 2. The producing and targeting cells of IL-17 family cytokines. Various cell types are shown to produce IL-17A and IL-17F during different physiological or pathological conditions. T cells, which include Th17, Tc17, γδT and NKT cells, are considered as the cellular sources of IL-17A and IL-17F. Besides T cells, other immune cells such as NK cells, LTi-like cells, monocytes, neutrophils and even B cells also produce IL-17A under certain conditions. In addition to immune cells, epithelial cells and paneth cells are found to secrete IL-17A and IL-17F respectively. The cell types which are responsive to IL-17A and IL-17F are also diverse. Local tissue cells, like epithelial cells, keratinocytes, endothelial cells, fibroblasts and muscle cells are the major targeting cells of IL-17A and IL-17F. However, certain immune cells, including macrophages, T cells and B cells are reported to be responsive to IL-17A stimulation. Like IL-17A and IL-17F, various immune cells, such as DCs, macrophages, eosinophils, basophils, mast cells and T cells, as well as nonimmune epithelial cells and paneth cells, produce IL-17E upon differential challenges. The major targeting cells of IL-17E include adaptive immune cells such as T cells (Th2, Th9 and NKT cells), various innate immune cells (macrophages, nuocytes, ILC2 cells, MMP-type 2 cells and T2M cells) and epithelial cells. IL-17C is mainly produced by local tissue cells, such as epithelial cells, keratinocytes and endothelial cells, although leukocytes also produce it. So far, IL-17C is shown to target epithelial cells, keratinocytes, Th17 cells and monocytes for downstream signaling. Although sporadic observations exist (see the text), the producing and targeting cells of IL-17B and IL-17D remain to be elusive.

242 TRAF6 independent pathway. IKKi was showed to interact with Act1 in an IL-17A signal dependent manner in mouse fibroblasts. Activation of NF-κB induced by IL-17A was not affected in IKKi deficient airway epithelial cells, but KC mRNA stabilization mediated by IL-17A was severely abolished in those cells [66]. By mass spectrometry analysis, IKKi was found to directly phosphorylate Act1 at residue Ser-311 upon IL-17A stimulation. Substitution of alanine for this serine residue on Act1 impaired mRNA stabilization of KC while it did not affect IL-17A mediated NF-κB activation [66]. Another parallel study showed that TRAF2, TRAF5 as well as alternative splicing factor (ASF or SF2) also participated in for IL-17A-induced KC mRNA stabilization [67]. SF2 bound mRNAs of chemokines like KC in unstimulated cells, IL-17A ligation induced the assembling of Act1–TRAF2/TRAF5–SF2 complex which prevented SF2 from binding and cleavage of KC mRNA [67]. The Ser-311 phosphorylation of Act1 was showed to be required for its initiation of TRAF2–TRAF5 complex. Consistently, deficiency of IKKi led to abolishment of the Act1–TRAF2–TRAF5 complex formation without disrupting the association of TRAF6 with Act1, suggesting that phosphorylation at Ser-

311 of Act1 is specific for the signal transduction of mRNA stabilization cascade [66,67]. In addition, upon IL-17A stimulation, Act1 also recruited and ubiquitinated HuR, an ARE-binding protein, to stabilize the mRNA which was dissociated from SF2. The interaction between HuR and SF2 was not detected, indicating that Act1 utilizes the same adaptors TRAF2 and TRAF5 to form two distinct but functional complementary complexes for IL-17A induced mRNA stabilization. Importantly, deficiency of HuR in epithelium resulted in diminished IL-17A induced lung inflammation, agreeing with the critical role of Act1 in lung epithelium (Fig. 3).

Recently, upon IL-17A exposure, Act1 was reported to be associated with the molecular chaperone heat shock protein 90 (hsp90) by mass spectrometry analysis [68]. The association of Act1 with other signaling molecules as well as the activation of downstream signaling pathways were severely impaired by inhibitors of Hsp90 [68]. Interestingly, the psoriasis associated Act1 mutant D10N [69–71] lost its binding ability to hsp90 and failed to mediate downstream signaling of IL-17A [68]. To date, Act1 together with Hsp90 assemble a receptor proximal anchor

Please cite this article as: X. Song, Y. Qian, Cellular Signalling (2013), http://dx.doi.org/10.1016/j.cellsig.2013.07.021

ARTICLE IN PRESS

X. Song, Y. Qian / Cellular Signalling xxx (2013) xxx–xxx 5

![Diagram](attachment:IL-17A_signaling_diagram.png)

Fig. 3. The activation of IL-17A signaling. IL-17A and IL-17F can form homodimer or heterodimer to bind the IL-17RA and IL-17RC heterodimeric complex. Although less defined, IL-17F may have similar downstream signaling to IL-17A as both cytokines utilize the same heterodimeric receptor complex. Upon the ligand binding, Act1 is recruited to the receptor complex. Act1 then recruits TRAF6 and functions as an E3 ubiquitin ligase to poly-ubiquitinate TRAF6 for the activation of NF-κB and JNK pathway. The TAK1 complex (TAK1/TAB2/TAB3) downstream of TRAF6 is required for NF-κB activation. IL-17A-induced NF-κB signaling also downregulates the expression of miR-23b and therefore releases its suppression of TAB2 and TAB3 for amplifying NF-κB activation. Act1 is also essential for IL-17A induced C/EBP activation. In addition, Act1 also mediates mRNA stabilization pathway though TRAF2 and TRAF5. Normally SF2 is bound to some mRNAs to destabilize them by degradation in unstimulated cells. Upon IL-17A stimulation, Act1/TRAF5/TRAF2 complex recruits SF2 to prevent mRNA from degradation. The phosphorylation of Act1 at Ser-311 by the kinase IKKi is required for the association of Act1 with TRAF2 or TRAF5. Alternatively, Act1 also recruits and ubiquitinates HuR, an ARE-binding protein, through TARP5 and TRAF2. HuR then binds to the SF2 free mRNAs to stabilize them. HSP90 has been shown to mediate IL-17A signaling by assisting the function of Act1.

278 platform to initiate two parallel IL-17A mediated cascades: (1) TRAF6  
279 dependent pathway, and (2) IKKi-TRAF2-TRAF5 dependent pathway  
280 (Fig. 3).  
281     IL-17F, which is the most homologous to IL-17A in this family, shares  
282 the same receptor set IL-17RA-IL-17RC with IL-17A for downstream sig-  
283 naling. Surface plasmon resonance (SPR) analysis showed that human  
284 IL-17A displayed a higher binding affinity to human IL-17RA while  
285 human IL-17F preferred to bind to human IL-17RC [72]. Another study  
286 also found that the affinity of human IL-17F for human IL-17RA was  
287 considerably lower than that of IL-17RC [73]. Although both cytokines  
288 bind to the receptors with different affinities, their mediated inflamma-  
289 tory responses were completely abrogated upon abolition of either IL-17RA  
290 or IL-17RC [52,54,65,72,74], suggesting that they mediate downstream  
291 signaling through IL-17RC/IL-17RA heterodimer complex. Similar to IL-  
292 17A, IL-17F recruited Act1 and TRAF6 for downstream signaling [54]  
293 and activated NF-κB, MAPKs and C/EBP cascades in different cell types  
294 [54,75,76] (Fig. 3).  

295 2.2.2. Negative regulation of IL-17A signaling pathways  
296     Since IL-17A exhibits a broad influence to drive inflammatory  
297 responses, its signaling is supposed to be under strict control for  
298 avoiding persistent detrimental inflammation. One report found  

that IL-17A activated the kinases ERK and GSK-3β to trigger sequential phosphorylation of C/EBPβ at Thr188 and Thr179 in its regulatory 2 domain. The C/EBPβ dual-phosphorylation inhibited the IL-17A induced proinflammatory gene expression [77] (Fig. 4).

TRAF3 is a TRAF family protein which mediates TLRs or RIG-I induced type I interferon production in antiviral responses [78–80]. It is also a negative regulator of CD40 or BAFF signaling for controlling the activation of non-canonical NF-κB pathway [81–84]. We recently found that TRAF3 is a crucial negative regulator in IL-17A signaling [85]. TRAF3 overexpression in cells suppressed IL-17A mediated NF-κB and MAPK activation and subsequent proinflammatory cytokine production. In contrast, silencing of TRAF3 by siRNA exacerbated the activation of NF-κB and MAPKs as well as production of downstream cytokines. Mechanically, TRAF3 directly bound to IL-17R to disrupt the formation of IL-17R–Act1–TRAF6 signaling complex, thus inhibited the activity of IL-17 signaling. In vivo evidences also found that TRAF3 suppressed the IL-17A induced inflammatory gene expression and consequently controlled the progression of EAE in mice [85]. Thus, TRAF3 functions as a receptor proximal negative regulator in IL-17A signaling (Fig. 4).

As TRAF3, TRAF4 was lately discovered as a negative modulator of IL-17A signaling. Upon IL-17A treatment, primary cells deficient of 

Please cite this article as: X. Song, Y. Qian, Cellular Signalling (2013), http://dx.doi.org/10.1016/j.cellsig.2013.07.021

ARTICLE IN PRESS

X. Song, Y. Qian / Cellular Signalling xxx (2013) xxx–xxx

![Diagram](attachment:diagram.png)

Fig. 4. The negative regulation of IL-17A signaling. Once activated, several mechanisms are adopted for preventing persistent activation of IL-17A signaling. Upon signaling activation, TRAF3 is recruited to IL-17RA and IL-17RC (IL-17R) complex to disrupt the IL-17R–Act1–TRAF6 signaling complex formation, while TRAF4 interacts with Act1 to interfere with the Act1–TRAF6 signaling complex formation. Although IKKi is critical for IL-17A mediated mRNA stabilization by phosphorylating Act1 on Ser-311, IKKi and its related kinase TBK1 also phosphorylates Act1 on three other Ser sites to prevent its further association TRAF6 and consequently suppress downstream NF-κB activation in a TRAF6 dependent manner, thus forming a negative feed-back loop (Act1–TRAF6–the IKK related kinases–Act1). IL-17A also recruits ERK and GSK3β to sequentially phosphorylate C/EBPβ to suppress C/EBP activation. USP25, an ubiquitin-specific protease, directly removes the ubiquitination of TRAF5 and TRAF6 to suppress IL-17A-induced signaling. Another ubiquitin-specific protease A20 is induced by IL-17A signaling and specifically erases the ubiquitination of TRAF6 to suppress the signaling, forming a negative feedback loop. During long time IL-17A exposure, Act1 is ubiquitinated by SCF-βTrCP E3 ubiquitin ligase complex and degraded, therefore desensitizing cells for further IL-17A signaling.

TRAF4 showed significant exacerbation of IL-17A signaling activation and chemokine expression. Genetic abolition of TRAF4 led to an earlier onset of MOG35-55 specific Th17 cell mediated EAE pathogenesis. TRAF4 was showed to suppress IL-17A signaling by competing the binding to Act1 with TRAF6 [86]. Therefore, at a different level, TRAF4 also interferes with the assembling of positive signaling complex to block the IL-17A signaling (Fig. 4).

Protein degradation by the ubiquitin-proteasome pathway is a widely used way for desensitization of signaling by erasure of key signaling components [87]. We recently found that persistent IL-17A stimulation resulted in a significant decrease of Act1 protein level without affecting its mRNA level. Mechanically, IL-17 recruited the SCF<sup>βTrCP-1/2</sup> E3 ubiquitin ligase complex to Act1 to mediate the Lys48-linked polyubiquitination and degradation of Act1, similar to the degradation of the inhibitor of nuclear factor κBα (IκBα) by this E3 ubiquitin ligase complex, indicating that either positive or negative signaling events of IL-17R is sequentially controlled by the same degradation system [88]. The degradation of Act1 eventually led to the dismiss of downstream positive signals and subsequently desensitized cells to IL-17 stimulation (Fig. 4).

IKKi was showed to phosphorylate Act1 at Ser-311 for IL-17A induced mRNA stabilization as described [66]. Recently, we found that both IKKi and TBK1 (another IKK-related kinase) were activated and recruited to Act1 by IL-17A stimulation [89]. Both kinases can phosphorylate Act1 on three other serine sites and phosphorylation of Act1 on those sites dismissed its association with TRAF6 and consequently suppressed IL-17A induced NF-kB activation. Interestingly, TRAF6 itself was required for TBK1 activation and its binding to Act1 after IL-17A exposure [89]. Thus, besides its positive function in IL-17A signaling through phosphorylating Act1 at Ser-311, IKKi plays a redundant role with TBK1 to phosphorylate Act1 on other Ser sites to form a negative feedback loop (Act1–TRAF6–the IKK related kinases–Act1) in the signaling (Fig. 4).

Since Act1 functions as an E3 ligase for the positive signaling of IL-17A, the deubiquitinating enzyme (DUB) for reverse process may exist. The ubiquitin-specific protease USP25, one subfamily member of deubiquitinating enzyme (DUB), was found as a negative regulator of IL-17A signaling [90]. Overexpression of USP25 suppressed IL-17A triggered downstream signalings. In contrast, USP25 deficient MEFs or primary lung epithelial cells showed enhanced responses upon IL-17A exposure [90]. Genetic abolition of USP25 in mice also elevated IL-17A induced respiratory inflammation and exacerbated EAE pathology [90]. Mechanically, USP25 directly removed the ubiquitination of TRAF5 and TRAF6 induced by IL-17A and consequently erased the

Please cite this article as: X. Song, Y. Qian, Cellular Signalling (2013), http://dx.doi.org/10.1016/j.cellsig.2013.07.021

signals for either TRAF6 dependent NF-kB pathway or TRAF5 dependent mRNA stabilization pathways [90]. More recently, A20, another deubiquitinating enzyme, was reported to regulate IL-17A induced TRAF6 dependent NF-kB and MAPK activation [91]. Interestingly, A20 was induced by IL-17A stimulation, therefore mediated a negative feedback control of IL-17A signaling. Knockdown or knockout of A20 in cells resulted in enhanced IL-17A induced gene expression. Mechanically, A20 was directly recruited to the distal domain of IL-17RA and subsequently removed the ubiquitination of TRAF6 to erase the TRAF6 dependent cascades in IL-17A signaling (Fig. 4).

Psoriasis is a relapsing skin immune disorder which is characterized by hyperplasia of dermis. In psoriasis patients, IL-17A as well as IL-17F was highly expressed in psoriatic skin biopsies [110–112]. As described in IBD, GWAS analysis of psoriasis specimens also identified IL-23R and Act1 as psoriasis-associated genes [69–71,113]. In mice, injection of IL-23 into skin led to a psoriasis-like epidermal hyperplasia and the hyperplasia was ameliorated in IL-17A or IL-22 deficient mice [114]. In human keratinocytes, IL-17A synergized with TNFα, IFNγ or IL-22 to induce the expression of proinflammatory cytokine, chemokines and antibacterial peptides [6,115–118].

IL-17A is also involved in other autoimmune diseases such as SLE and T1DM. It was found to be highly expressed in either double negative (DN) T cells of SLE patients or MRL/lpr mice [119,120]. In an FcgammaR2b-deficient mouse model of SLE, the recent study showed that IL-17 and FcgammaR2b double deficient mice were protected from the development of lupus compared to FcgammaR2b deficient mice [121]. It was reported that IL-17A activated inflammatory genes as well as autoantibodies in peripheral blood mononuclear cells (PBMC) from lupus patients [122]. IL-17A also synergized with BAFF to promote B cell survival [123]. These findings suggest that IL-17 plays an important role in SLE pathogenesis. Although the role of IL-17A in human T1DM was less clear, in the animal model of T1DM, IL-17A was demonstrated to contribute to the progression of autoimmune diabetes. In non-obese diabetic (NOD) mice, blocking of IL-17A with antibodies ameliorated the pathogenesis of autoimmune diabetes [124]. In contrast, the hyperglycemia of NOD mice deficient with IL-17A was comparable to the control NOD mice [100]. Thus, the role of IL-17A in T1DM still needs to be further addressed.

### 2.3. Role of IL-17A and F in the pathogenesis of inflammatory diseases

#### 2.3.1. IL-17A and IL-17F in autoimmune diseases

Autoimmune diseases, such as rheumatoid arthritis (RA) and multiple sclerosis (MS), have long been considered as Th1, Th2 or B cell mediated diseases. After the discovery of the Th17 subset and its unique cytokines IL-17A and IL-17F, robust evidences show their boarding effects on not only MS and RA but also many other autoimmune diseases including inflammatory bowel disease (IBD), psoriasis, systemic lupus erythematosus (SLE) and type 1 diabetes mellitus (T1DM) [4].

RA, an autoimmune disease which mainly affects synovial joints, leads to a substantial loss of mobility of the patients. Compared to healthy controls or osteoarthritis patients, patients with RA shown a higher level of IL-17A in their rheumatoid synovium [92,93]. In mouse experimental models of RA, blocking of IL-17A ameliorated, while overexpression of IL-17A exacerbated the disease progression [94,95]. Deficiency of IL-17A protected mice from both collagen induced arthritis (CIA) or spontaneous autoimmune arthritis in interleukin 1 receptor antagonist (IL1RN) knockout mice [46,96]. Despite its similarity to IL-17A, contribution of IL-17F to the pathogenesis of arthritis has been shown to be extremely limited [46].

MS is a central nervous system inflammatory disease in which the protective myelin sheaths of the brain and spinal cord axons are damaged and lost. Microarray data demonstrated that IL-17A was highly upregulated in the brain lesion regions of MS patients [97]. In the active lesions of MS patient, there were more IL-17 producing T-cells than the inactive MS lesions [98]. IL-23 is a cytokine critical for Th17 cell differentiation. Deficient of IL-23p19, a specific subunit of IL-23, protected mice from EAE [99]. In contrast, Th17 cell adoptive transfer induced severe EAE in recipient mice [99]. Genetic abolition of either IL-17A or its receptor IL-17RC decreased the pathogenesis of EAE in mice [54,74,100]. IL-17F, however, was not required for the initiation of EAE but the inflammation in a later stage of the disease was attenuated in IL-17F deficient mice [54].

IBD, an autoimmune disease which is typically classified as Crohn's disease (CD) and ulcerative colitis (UC), is an inflammatory disorder of the colon and small intestine. In both CD and UC patients, IL-17A was found to be upregulated in the specimens [101,102]. The expression of IL-17F was also associated with IBD. Compared with UC patients, IL-17F expression was higher in CD patients [103]. Genome-wide association study (GWAS) studies also found IL-23R, STAT3, CCR6 and Act1 which were critical for Th17 differentiation, attraction and IL-17 signaling as IBD-associated genes [104–107]. However, in animal models of IBD, controversial results for the role of IL-17A have been observed. Neutralization of IL-17A or genetic abolition of IL-17A led to aggregative dextran sodium sulfate (DSS) induced colitis in mice, suggesting a protective role of IL-17A in the IBD model [54,108]. By contrast, compared with wild-type control, IL-17A deficient mice were resistance to DSS induced colitis in another study, indicating a pathological role of IL-17A [109]. The conflict between those studies is possibly due to the variance of intestinal microflora in different studies. IL-17F, however, was found to exacerbate the intestinal inflammation since deficiency of IL-17F in mice protected them from DSS induced colitis [54].

#### 2.3.2. IL-17A and IL-17F in allergic asthma

Asthma is a common chronic airway inflammatory disease which is classified as atopic and non-atopic. Surveys from clinical studies found that both IL-17A and IL-17F were upregulated in bronchial specimens of asthma patients [125–127]. Overexpression of both IL-17A and IL-17F led to a remarkable infiltration of neutrophil, but no eosinophil [54,128]. In OVA-induced airway inflammation, IL-17A was showed to promote the disease while IL-17F negatively regulated this allergic response [46,54,129]. However, another group also found that IL-17F was not essential for OVA induced pulmonary neutrophilia [46]. Host dust mite induced airway neutrophilia also required IL-17A [130]. Overall, although debate of the role of IL-17F in allergic response still exists, IL-17A contributes to pulmonary inflammation by promoting neutrophilia rather than eosinophilia. More recently, IL-17A, but not IL-17F, was reported to drive airway hyper-responsiveness by promoting airway smooth muscle contraction directly [131]. The detailed mechanisms of IL-17A or IL-17F in the pathogenesis of allergic asthma remain to be elucidated.

#### 2.3.3. IL-17A and IL-17F in tumorigenesis

Th17 cells had been reported to associate with tumor progression in human, although debate between studies also exists. In colorectal cancer and hepatocellular carcinoma specimens, patients characterizing with high expression of IL-17A had a poor prognosis [132,133]. However, patients suffered from ovarian cancer underwent poor prognosis when the infiltrating IL-17A producing cells and IL-17A level were low [134], suggesting that the contributions of IL-17A to tumorigenesis are dependent on the tumor type and stage.

Several studies from IL-17A deficient mice indicated that IL-17A exhibits a pro-tumorigenic role in different tumor models. Compared to wide type mice, transplantation of melanoma cell line B16 or bladder carcinoma cell line MB49 into IL-17A deficient mice resulted in delayed growth of those cells [135]. In both chemical induced and spontaneous colon cancer models, deficiency of IL-17A resulted in reduced intestinal tumor load in those mice [136,137]. More recently, one study found that both IL-23 and its receptor deficiency in CPC-APC mice protected those mice from spontaneous colon cancer, agreeing with the former studies

on IL-17A deficient mice [138]. One of the intestinal commensal bacteria, enterotoxigenic *Bacteroides fragilis* (ETBF), was reported to promote intestinal tumor development in *Apc*<sup>min/+</sup> mice by inducing Th17 responses [139]. IL-17A as well as IL-23 also promoted cutaneous chemical carcinogenesis, since IL-17A, IL-17RA and IL-23 knockout mice developed fewer skin tumors than wide type controls [140–142].

As mentioned early, Th17 cells were correlated with improved prognosis in some cases. Those IL-17A producing cells were also showed to inhibit tumor progression in mouse models. Transplantation of Th17 prevented tumor growth by activating anti-tumor CD8<sup>+</sup> cells [143]. Although several studies demonstrated that IL-17A promoted tumor metastasis by showing that lung metastasis was reduced in IL-17A deficient mice [135,144,145], other studies found that, on the contrary, increased lung metastasis was found when IL-17A was ablated [143,146]. Thus, the pro-tumor or anti-tumor property of IL-17A is likely dependent on the tumor type and stage. The role of IL-17F in tumorigenesis is not well characterized. As IL-17A, IL-17F was showed to be required for intestinal tumorigenesis in *Apc*<sup>min/+</sup> mice [147]. However, in chemical induced colon cancer model, IL-17F played a protective role in the tumor progression [148]. The exact role of IL-17F in tumor development still needs to be addressed.

### 2.3.4. IL-17A and IL-17F in host defense

Upon infection, IL-17A and IL-17F are produced either quickly by innate lymphocytes or lately by adaptive Th17 cells. The released cytokines then induce expression of host defense genes such as cytokines, chemokines and antibacterial peptide expression in either epithelial cells or keratinocytes to protect the host from harmful pathogens. IL-17A and IL-17F deficient mice were susceptible to the infection of extracellular pathogens, including *Citrobacter rodentium*, *Klebsiella pneumonia* and *Staphylococcus aureus* [46,149]. A type of commensal bacteria, segmented filamentous bacteria (SFB), preferentially induced intestinal Th17 development in steady state. Mice absent from SFB resulted in reduction of Th17 population in the intestine and increased susceptibility to *C. rodentium* infection [150]. However, another study reported that IL-17RC deficient mice were merely affected by *C. rodentium* infection while IL-22 deficient mice were susceptible to the infection [151]. Beside to extracellular pathogens, IL-17A also protected mice from intracellular bacteria or fungal infection. During *Francisella tularensis* infection, both neutralization and genetic abolition of IL-17A increased the bacterial burden in mice [152]. IL-17A, not IL-17F or IL-22, induced IL-12 production from dendritic cells and promoted Th1 response during the infection [152]. An oral opportunistic fungus, *Candida albicans*, strongly induced Th17 differentiation [153]. Deficient of IL-17A, but not IL-17F, increased the susceptibility to oral candidiasis, suggesting that IL-17A is a critical factor during the pathogenesis of this disease [153].

## 3. IL-17C

### 3.1. Cellular sources and signaling transduction

Until recently, the biological functions and pathological roles of IL-17C began to be uncovered. TLR5 agonist flagellin or *Mycoplasma pneumoniae* challenge induced IL-17C expression in the gut or lung respectively [154,155]. At cellular level, common pathogens, like *S. aureus*, *Pseudomonas aeruginosa*, *Haemophilus influenza* or *C. rodentium*, activated IL-17C transcription in keratinocytes [156], bronchial epithelial cells [157] or intestinal epithelial cells [158]. Bacterial PAMP molecules, like LPS, peptidoglycan, flagellin, poly (I:C) or CpG dinucleotide, which signal through Toll-like receptors, activated IL-17C expression in those cells accordingly [157–160]. Other factors, such as inflammatory cytokines TNFα and IL-1β, which are induced by PRR (pattern recognition receptor) signaling, also contributed to IL-17C production in those cells [158,160,161]. One report from human psoriasis patients illustrated that, besides keratinocytes, IL-17C mRNA was also located in endothelial cells and leukocytes [112]. At least in human keratinocytes, the transcriptional activation of IL-17C gene was controlled by the transcription factor NF-κB [161] (Fig. 2).

Early studies reported that IL-17C induced TNFα and IL-1β expression in a human monocylic cell line THP1 or mouse peritoneal exudate cells [162,163]. In vivo delivery of IL-17C in mouse by different strategies caused neutrophil infiltration in lung or exacerbation of collagen-induced arthritis (CIA) progression [128,163]. The downstream signal transduction of IL-17C was not revealed until recently. We and others found that IL-17C bound to IL-17RA and IL-17RE complex to trigger downstream signaling pathways [158,160,164]. Unlike IL-17RA, which has a broad expression pattern, IL-17RE was highly expressed on epithelial cells or Th17 cells [158,164,165]. As described above, IL-17RA is a common receptor shared by the IL-17 family members. Thus, IL-17RE is considered as a specific receptor for IL-17C. IL-17C induced both NF-κB and MAPK activations in mouse primary intestinal epithelial cells or human colonic tumor cell lines [158]. Abolition of IL-17RE expression either by genetic targeting or by siRNA silencing diminished IL-17C mediated activation of NF-κB and MAPKs in intestinal epithelial cells [158]. In agreeing with this, Th17 cells isolated from IL-17RE transgenic mice showed hyper-responsibility to IL-17C to produce both IL-17A and IL-17F [164]. As in IL-17A signaling, the adaptor protein Act1 was required for IL-17C function in promoting Th17 differentiation [164]. We also found that Act1 interacted with IL-17RE in an overexpression system (unpublished data). Although needed to be addressed, it is very likely that IL-17C signaling utilizes Act1 to initiate downstream cascades in other IL-17RE positive cell types such as epithelial cells (Fig. 5).

### 3.2. Role of IL-17C in the pathogenesis of inflammatory diseases

The innate sources of IL-17C indicate that this cytokine may be a critical factor in mucosal immunity. IL-17C was dramatically induced in colon tissues when mice were challenged with a mouse intestinal pathogen, *C. rodentium* [158,160,165]. During infection, the kinetics of IL-17C upregulation was similar to those of IL-22, which is a crucial factor for host defense against *C. rodentium* infection [151,158]. The early mortality phenotype of IL-17RE deficient mice after *C. rodentium* infection resembled that of IL-22 deficient mice. Furthermore, IL-17C acted with IL-22 to induce antibacterial peptides in colon tissues in a synergistic manner [158]. Studies from patients with psoriasis demonstrated that, as well as IL-17A and IL-17F, IL-17C mRNA was significantly increased in psoriatic skin [110,112]. IL-17C induced innate immunity related gene production, including inflammatory cytokines, chemokines and antibacterial peptides, from human primary epidermal keratinocytes [112,160]. IL-17C also cooperated with TNFα or IL-1β to synergistically induce such psoriasis-related gene expression in those cells [112,160]. Beside keratinocytes, human dermal microvascular endothelial cells also produced inflammatory mediators in response to IL-17C stimulation [112]. Keratinocyte-specific IL-17C transgenic mice spontaneously developed a psoriasis-like skin phenotype with increased dermal angiogenesis, immune cell infiltration and psoriasis-related gene expression [112]. Consistent with that, in an imiquimod induced mouse model of psoriasis, IL-17C was upregulated in the skin of those mice [160]. Genetic deficiency of IL-17C, or its receptor IL-17RE, significantly protected mice from imiquimod induced epidermal hyperplasia and neutrophil infiltration [160]. On contrary to the pathogenic functions of IL-17C in psoriatic models, IL-17C signaling prevented mice from dextran sulfate sodium (DSS) induced colitis [160,165]. Deficiency of IL-17C or IL-17RE in mice led to increased body weight loss and enhanced intestinal epithelium disruption of those mice [160,165]. IL-17C signaling was also shown to maintain intestinal barrier integrity by activation of tight junction protein occludin in intestinal epithelial cells [165].

In addition to its functions in cutaneous and mucosal epithelial cells, IL-17C signaling engaged T helper (Th) 17 responses in the central

ARTICLE IN PRESS

X. Song, Y. Qian / Cellular Signalling xxx (2013) xxx–xxx

![Diagram](attachment:diagram.png)

Fig. 5. The signaling of IL-17E and IL-17C. The signaling of IL-17E and IL-17C has been discovered recently. Upon the legend binding, both cytokines activate NF-κB and MAPK pathways. Similar to IL-17A and IL-17F signalings, the adaptor protein Act1 is required for both signalings. TRAF6 is also recruited into the IL-17RA–IL-17RB complex to activate downstream signaling. Classically, IL-17E utilizes this signaling complex to induce the Th2 related cytokine production. However, in certain tumor cells which highly express IL-17RB, IL-17E induces an alternative pathway to induce the tumor cell apoptosis by recruiting TRADD and FADD to IL-17RB and subsequently activating caspase induced apoptosis pathway. TRAF6 may be recruited into IL-17C signaling complex as in other IL-17 cytokine signaling. Unlike IL-17E, IL-17C induces the expression of inflammatory cytokines and chemokines and antibacterial peptides. IL-17C also promotes IL-17A and IL-17F expression in Th17 cells which highly express its receptor IL-17RE. More detailed signaling events mediated by these two cytokines still need to be illustrated.

nervous system (CNS). In the experimental autoimmune encephalomyelitis (EAE) mouse model for studying human multiple sclerosis (MS), along with IL-17A and F, IL-17C was upregulated in CNS of EAE mice [164], although the cellular sources of its induction in CNS remained unclear. Compared to control mice, IL-17C deficient mice displayed mitigatory, while IL-17RE transgenic mice exhibited aggravated EAE symptoms [164]. IL-17C pathway exacerbated Th17 cell mediated responses by promoting IL-17A and IL-17F expression from those cells [164].

### 4. IL-17E

#### 4.1. Cellular sources and signaling transduction

IL-17E (also known as IL-25) is the most divergent member to IL-17A in the IL-17 cytokine family. In an initial study, the mRNA of IL-17E was highly expressed in polarized Th2 cells [166]. Other cell sources of IL-17E production were then discovered, including epithelial cells [167,168], endothelial cells [169], eosinophils [170,171], basophils [171], alveolar macrophages [172], microglia [173] and IgE-activated mast cells [174]. Although one study showed that Erk and p38 inhibitors suppressed allergen induced IL-17E production [175], the transcriptional mechanisms of IL-17E production still remain to be characterized (Fig. 2).

IL-17RB (also known as IL-17BR and IL-17Rh1) was found to be the receptor for IL-17E [176]. IL-17RB was also found to be able to bind IL-17B, although with lower affinity than IL-17E [207]. The following studies showed that, not only IL-17RB but also IL-17RA, was required for IL-17E signaling [178]. Genetic deficiency of either IL-17RB or IL-17RA led to abolition of IL-17E mediated responses [178]. Unlike the broad targeting cell types of IL-17A, IL-17E prefers hematopoietic cells as targets for production of Th2 related cytokines. Functional IL-17RB was found to be expressed on monocytes [170,179], natural killer T (NKT) cells [180,181], non-T/non-B (NBNT) cells [166], nuocytes [182], multipotent progenitor type2 (MMPtype2) cells [183], innate type2 helper (ih2) cells [184] and type 2 cytokine-producing myeloid (T2M) cells [185]. In addition to those innate response cell populations, IL-17RB was also highly expressed on Th2 or Th9 cells which were responsive to IL-17E [167,186]. Non-immune cells, such as colonic and lung epithelial cells are also reported as responding cells to IL-17E [158,187] (Fig.2). As IL-17RA, the intracellular region of IL-17RB harbors a conserved SEFIR domain, indicating that the adaptor Act1 is likely required for its functions. Indeed, under IL-17E stimulation, Act1 was recruited to IL-17RB through the SEFIR–SEFIR interaction [188,189].

After Act1 genetic ablation, IL-17E caused Th2 cytokine related allergic pulmonary inflammation symptoms were largely ameliorated. Conditional depletion of Act1 either in T cells or epithelial cells blocked airway inflammation induced by allergens [189,190]. More recently, IL-17E dependent parasite expulsion also required epithelial-derived Act1 for Lin⁻c-Kit⁺ innate immune cell expansion in intestine [191]. As in IL-17A signaling, the adaptor molecule TRAF6 was recruited to IL-17RB and required for IL-17E mediated NF-κB activation [192] (Fig. 5).

### 4.2. Role of IL-17E in the pathogenesis of inflammatory diseases

Helminthic parasite infections are tightly linked to host type 2 immune responses. As IL-17E is a strong promoter for Th2 responses, recent studies demonstrate pivotal protective functions of IL-17E against helminthes in intestine. Exogenous delivery of IL-17E protected mice from *Trichuris muris* infection, while IL-17E deficient mice were unable to clean up the worms [193]. In another parasite infection model, along with impaired expression of Th2 cytokines, IL-17E deficient mice also displayed lower efficiency in expulsion of *Nippostrongylus brasiliensis* [194,195].

In the intestine, commensal bacteria induced IL-17E production in intestinal epithelia cells. The released cytokine limited Th17 expansion by blocking IL-23 production in macrophages [168]. Similarly, along with increased Th17 cells, IL-17E deficient mice were susceptible to EAE challenge. In CNS, the suppression of EAE pathogenesis by IL-17E was dependent on IL-13 signaling [173]. However, IL-17E also ameliorated the symptoms of DSS, trinitro-benzene-sulfonic acid (TNBS), peptideglycan (PGN) or oxazolone induced colitis independent of IL-13 [196,197]. Although harboring protective functions in host defense and autoimmune responses, IL-17E is a pathogenic factor during respiratory allergic diseases. In nasal lavages of allergic rhinitis patients, IL-17E level was higher than normal controls [198]. Epithelial specific transgenic mice of IL-17E showed enhanced mucus production and airway inflammatory responses, while in a mouse model of allergic asthma, IL-17E antibody blockade conversely reduced the airway Th2-related inflammation and goblet cell hyperplasia [167,199]. In addition, genetic abolition of IL-17E led to reduced lung allergic inflammation by attenuation of both Th2 and Th9 cytokines [186]. Likewise, usage of soluble IL-17RB or genetic abolition of IL-17RB ameliorated the lung pathology induced by allergen challenge [185,200].

The effects of IL-17E on tumor development are poorly investigated. In vivo administration of recombinant IL-17E inhibited tumor growth in multiple human tumor xenograft models [201]. IL-17E-mediated antitumor effect was believed to rely on B cell activation in those models [201]. Another study found that, to a certain extent, the anticancer agent Virulizin suppressed tumor growth by induction of IL-17E [202]. More recently, IL-17E was reported to be released by nonmalignant mammary epithelial cells (MECs) to trigger breast cancer cell apoptosis. Notably, IL-17E had no apoptotic effect on nonmalignant MECs. IL-17E treatment efficiently inhibited tumor growth without affecting the normal tissues in vivo, suggesting a potential benefit of IL-17E in cancer therapy [203] (Fig. 5).

### 5. IL-17B and IL-17D

To date, IL-17B and IL-17D are the least characterized members of this family. Although so, sporadic evidences show that these two cytokines also exhibit proinflammatory effects under certain conditions.

An early study showed that, although it was not expressed in human activated CD4⁺ T cells, the mRNA of IL-17B existed in several adult human tissues, including pancreas, small intestine, and stomach [162]. Following studies also found that IL-17B was highly expressed in chondrocytes, neurons or breast cancer cells [203–205]. Compared to IL-17E, IL-17B bound to IL-17RB with lower affinity [177], however, the downstream signal transductions of this cytokine-receptor pair remain to be addressed. Like IL-17C, IL-17B was reported to induce TNFα and IL-1β expression in THP1 cells [162]. In collagen-induced arthritis (CIA) mouse model, IL-17B was upregulated in the cartilage of CIA arthritic paws and adaptive transfer of IL-17B transduced CD4⁺ T cells dramatically exacerbates the disease progression by promoting TNFα expression in these animals [163]. In agreeing with its proinflammatory effects, neutralization of IL-17B evidently ameliorated the arthritis progression in CIA mice [163]. Unlike IL-17E, IL-17B was highly expressed in several breast cancer cell lines and overexpressed in human breast cancer specimens. Short-hairpin RNA (shRNA) mediated knockdown of IL-17B in breast cancer cell line impaired the proliferation and invasion ability of those cells [203]. However, another report demonstrated that IL-17B could harbor an antiangiogenic property by inhibiting the adhesion, migration and tubule formation ability of HECV cells in vitro [206]. As the most homologous of IL-17B, IL-17D was found to be expressed in the brain, heart, lung, pancreas, skeletal muscle and adipose tissue [207]. Although its receptor was still not identified, IL-17D stimulated IL-6, IL-8 and GM-CSF productions in endothelial cells [207] or IL-6 and IL-8 expression in chicken fibroblasts [208]. A recent study also showed that IL-17D increased cytotoxic function of human NK cells in vitro [209].

### 6. Concluding remarks

Recent progresses from both clinical and animal studies have demonstrated the critical roles of IL-17A in the pathogenesis of multiple inflammatory disorders. Anti-IL-17A therapy approaches have been tested in several clinical trials to treat inflammatory autoimmune diseases and the outcomes are quite promising. Thus, fully understanding the functional mechanisms of IL-17A signaling will be critical for clinical applications of IL-17A targeting. While IL-17F plays an important role in host defense against bacterial infections as IL-17A, its role in the pathogenesis of inflammatory disorders appears not critical and remains to be further characterized. Although recent advances uncover the biological and pathogenic functions of IL-17C and IL-17E, the in vivo functions of IL-17B and IL-17D remain to be explored. While the activation and regulation of IL-17A signaling have been well studied, the signaling of the other IL-17 family cytokines, esp. IL-17B, IL-17C and IL-17D is not clear and remains to be illustrated. Better understanding of the signaling and inflammatory responses mediated by IL-17 family cytokines will benefit for the prevention and therapy of inflammatory diseases in human.

### Conflict of interest

The authors declare no competing financial interest.

### Acknowledgments

The authors have no conflicting financial interests. Supported by grants from the National Natural Science Foundation of China (No. 81230075, No. 30930084 and No. 91029708), the 973 program (No. 2013CB944904 and No. 2010CB529705), and Science and Technology Commission of Shanghai Municipality (13JC1408900).

### References

[1] Y. Iwakura, H. Ishigame, S. Saijo, S. Nakae, Immunity 34 (2011) 149–162.

[2] T. Korn, E. Bettelli, M. Oukka, V.K. Kuchroo, Annual Review of Immunology 27 (2009) 485–517.

[3] D.J. Cua, C.M. Tato, Immunology 10 (2010) 479–489.

[4] S. Zhu, Y. Qian, Clinical Science 122 (2012) 487–511.

[5] J. Hartupee, C. Liu, M. Novotny, X. Li, T. Hamilton, Journal of Immunology 179 (2007) 4135–4141.

[6] S.C. Liang, X.Y. Tan, D.P. Luxenberg, R. Karim, K. Dunussi-Joannopoulos, M. Collins, L.A. Fouser, The Journal of Experimental Medicine 203 (2006) 2271–2279.

[7] Y. Qian, C. Liu, J. Hartupee, C.Z. Altuntas, M.F. Gulen, D. Jane-Wit, J. Xiao, Y. Lu, N. Gilttiay, J. Liu, T. Kordula, Q.W. Zhang, B. Vallance, S. Swaidani, M. Aronica, V.K. Tuohy, T. Hamilton, X. Li, Nature Immunology 8 (2007) 247–256.

ARTICLE IN PRESS

X. Song, Y. Qian / Cellular Signalling xxx (2013) xxx–xxx 11

[8] S.H. Chang, H. Park, C. Dong, The Journal of Biological Chemistry 281 (2006) 35603–35607.

[9] X. Song, Y. Qian, Cytokine 62 (2013) 175–182.

[10] S.L. Gaffen, Nature Reviews. Immunology 9 (2009) 556–567.

[11] S. Gerhardt, W.M. Abbott, D. Hargreaves, R.A. Paupit, R.A. Davies, M.R. Needham, C. Langham, W. Barker, A. Aziz, M.J. Snow, S. Dawson, F. Welsh, T. Wilkinson, T. Vaugan, G. Beste, S. Bishop, B. Popovic, G. Rees, M. Sleeman, S.J. Tuske, S.J. Coales, Y. Hamuro, C. Russell, Journal of Molecular Biology 394 (2009) 905–921.

[12] S.G. Hymowitz, E.H. Filvaroff, J.P. Yin, J. Lee, L. Cai, P. Risser, M. Maruoka, W. Mao, J. Foster, R.F. Kelley, G. Pan, A.L. Gurney, A.M. de Vos, M.A. Starovasnik, The EMBO Journal 20 (2001) 5332–5341.

[13] L.K. Ely, S. Fischer, K.C. Garcia, Nature Immunology 10 (2009) 1245–1251.

[14] D. Ron, Y. Fuchs, D.S. Chorev, The International Journal of Biochemistry & Cell Biology 40 (2008) 2040–2052.

[15] M. Mellett, P. Atzei, A. Horgan, E. Hams, T. Floss, W. Wurst, P.G. Fallon, P.N. Moynagh, Nature Communications 3 (2012) 1119.

[16] Z. Rong, A. Wang, Z. Li, Y. Ren, L. Cheng, Y. Li, Y. Wang, F. Ren, X. Zhang, J. Hu, Z. Chang, Cell Research 19 (2009) 208–215.

[17] M.J. McGeachy, D.J. Cua, Immunity 28 (2008) 445–453.

[18] M. Veldhoen, R.J. Hocking, C.J. Atkins, R.M. Locksley, B. Stockinger, Immunity 24 (2006) 179–189.

[19] P.R. Mangan, L.E. Harrington, D.B. O'Quinn, W.S. Helms, D.C. Bullard, C.O. Elson, R.D. Hatton, S.M. Wahl, T.R. Schoeb, C.T. Weaver, Nature 441 (2006) 231–234.

[20] E. Bettelli, Y. Carrier, W. Gao, T. Korn, T.B. Strom, M. Oukka, H.L. Weiner, V.K. Kuchroo, Nature 441 (2006) 235–238.

[21] H. Park, Z. Li, X.O. Yang, S.H. Chang, R. Nurieva, Y.H. Wang, Y. Wang, L. Hood, Z. Zhu, Q. Tian, C. Dong, Nature Immunology 6 (2005) 1133–1141.

[22] L.E. Harrington, R.D. Hatton, P.R. Mangan, H. Turner, T.L. Murphy, K.M. Murphy, C.T. Weaver, Nature Immunology 6 (2005) 1123–1132.

[23] T. Korn, E. Bettelli, W. Gao, A. Awasthi, A. Jager, T.B. Strom, M. Oukka, V.K. Kuchroo, Nature 448 (2007) 484–487.

[24] E.V. Acosta-Rodriguez, G. Napolitani, A. Lanzavecchia, F. Sallusto, Nature Immunology 8 (2007) 942–949.

[25] S. Aggarwal, N. Ghilardi, M.H. Xie, F.J. de Sauvage, A.L. Gurney, The Journal of Biological Chemistry 278 (2003) 1910–1914.

[26] C.A. Murphy, C.L. Langrish, Y. Chen, W. Blumenschein, T. McClanahan, R.A. Kastelein, J.D. Sedgwick, D.J. Cua, The Journal of Experimental Medicine 198 (2003) 1951–1957.

[27] Y. Chung, S.H. Chang, G.J. Martinez, X.O. Yang, R. Nurieva, H.S. Kang, L. Ma, S.S. Watowich, A.M. Jetten, Q. Tian, C. Dong, Immunity 30 (2009) 576–587.

[28] I.I. Ivanov, B.S. McKenzie, L. Zhou, C.E. Tadokoro, A. Lepelley, J.J. Lafaille, D.J. Cua, D.R. Littman, Cell 126 (2006) 1121–1133.

[29] X.O. Yang, B.P. Pappu, R. Nurieva, A. Akimzhanov, H.S. Kang, Y. Chung, L. Ma, B. Shah, A.D. Panopoulos, K.S. Schluns, S.S. Watowich, Q. Tian, A.M. Jetten, C. Dong, Immunity 28 (2008) 29–39.

[30] X.O. Yang, A.D. Panopoulos, R. Nurieva, S.H. Chang, D. Wang, S.S. Watowich, C. Dong, The Journal of Biological Chemistry 282 (2007) 9358–9363.

[31] T.J. Harris, J.F. Grosso, H.R. Yen, H. Xin, M. Kortylewski, E. Albesiano, E.L. Hipkiss, D. Getnet, M.V. Goldberg, C.H. Maris, F. Housseau, H. Yu, D.M. Pardoll, C.G. Drake, Journal of Immunology 179 (2007) 4313–4317.

[32] L. Wei, A. Laurence, K.M. Elias, J.J. O'Shea, The Journal of Biological Chemistry 282 (2007) 34605–34610.

[33] L. Zhou, I.I. Ivanov, R. Spolski, R. Min, K. Shenderov, T. Egawa, D.E. Levy, W.J. Leonard, D.R. Littman, Nature Immunology 8 (2007) 967–974.

[34] Z. Chen, A. Laurence, Y. Kanno, M. Pacher-Zavisin, B.M. Zhu, C. Tato, A. Yoshimura, L. Hennighausen, J.J. O'Shea, Proceedings of the National Academy of Sciences of the United States of America 103 (2006) 8137–8142.

[35] A. Brustle, S. Heink, M. Huber, C. Rosenplanter, C. Stadelmann, P. Yu, E. Arpaia, T.W. Mak, T. Kamradt, M. Lohoff, Nature Immunology 8 (2007) 958–966.

[36] W. Ise, M. Kohyama, B.U. Schraml, T. Zhang, B. Schwer, U. Basu, F.W. Alt, J. Tang, E.M. Oltz, T.L. Murphy, K.M. Murphy, Nature Immunology 12 (2011) 536–543.

[37] F. Zhang, G. Meng, W. Strober, Nature Immunology 9 (2008) 1297–1306.

[38] M. Veldhoen, K. Hirota, A.M. Westendorf, J. Buer, L. Dumoutier, J.C. Renauld, B. Stockinger, Nature 453 (2008) 106–109.

[39] F.J. Quintana, A.S. Basso, A.H. Iglesias, T. Korn, M.F. Farez, E. Bettelli, M. Caccamo, M. Oukka, H.L. Weiner, Nature 453 (2008) 65–71.

[40] K. Okamoto, Y. Iwai, M. Oh-Hora, M. Yamamoto, T. Morio, K. Aoki, K. Ohya, A.M. Jetten, S. Akira, T. Muta, H. Takayanagi, Nature 464 (2010) 1381–1385.

[41] L. Li, Q. Ruan, B. Hilliard, J. Devirgiliis, M. Karin, Y.H. Chen, The Journal of Experimental Medicine 208 (2011) 787–796.

[42] X.K. Liu, X. Lin, S.L. Gaffen, The Journal of Biological Chemistry 279 (2004) 52762–52771.

[43] D.J. Cua, C.M. Tato, Nature Reviews. Immunology 10 (2010) 479–489.

[44] D.A. Bermejo, S.W. Jackson, M. Gorosito-Serran, E.V. Acosta-Rodriguez, M.C. Amezquita-Vesely, B.D. Sather, A.K. Singh, S. Khim, J. Mucci, D. Liggitt, O. Campetella, M. Oukka, A. Gruppi, D.J. Rawlings, Nature Immunology 14 (2013) 514–522.

[45] N. Takahashi, I. Vanlaere, R. de Rycke, A. Cauwels, L.A. Joosten, E. Lubberts, W.B. van den Berg, C. Libert, The Journal of Experimental Medicine 205 (2008) 1755–1761.

[46] H. Ishigame, S. Kakuta, T. Nagai, M. Kadoki, A. Nambu, Y. Komiyama, N. Fujikado, Y. Tanahashi, A. Akitsu, H. Kotaki, K. Sudo, S. Nakae, C. Sasakawa, Y. Iwakura, Immunity 30 (2009) 108–119.

[47] K. Hirota, H. Ahlfors, J.H. Duarte, B. Stockinger, EMBO Reports 13 (2012) 113–120.

[48] E. Rouvier, M.F. Luciani, M.G. Mattei, F. Denizot, P. Golstein, Journal of Immunology 150 (1993) 5445–5456.

[49] F. Huang, C.Y. Kao, S. Wachi, P. Thai, J. Ryu, R. Wu, Journal of Immunology 179 (2007) 6504–6513.

[50] A. Saleh, L. Shan, A.J. Halayko, S. Kung, A.S. Gounni, Journal of Immunology 182 (2009) 3357–3365.

[51] S. Zhu, W. Pan, X. Song, Y. Liu, X. Shao, Y. Tang, D. Liang, D. He, H. Wang, W. Liu, Y. Shi, J.B. Harley, N. Shen, Y. Qian, Nature Medicine 18 (2012) 1077–1086.

[52] D. Toy, D. Kugler, M. Wolfson, T. Vanden Bos, J. Gurgel, J. Derry, J. Tocker, J. Peschon, Journal of Immunology 177 (2006) 36–39.

[53] Z. Yao, W.C. Fanslow, M.F. Seldin, A.M. Rousseau, S.L. Painter, M.R. Comeau, J.I. Cohen, M.K. Spriggs, Immunity 3 (1995) 811–821.

[54] X.O. Yang, S.H. Chang, H. Park, R. Nurieva, B. Shah, L. Acero, Y.H. Wang, K.S. Schluns, R.R. Broaddus, Z. Zhu, C. Dong, The Journal of Experimental Medicine 205 (2008) 1063–1075.

[55] A.W. Ho, S.L. Gaffen, Seminars in Immunopathology 32 (2009) 33–42.

[56] J.Y. Chung, Y.C. Park, H. Ye, H. Wu, Journal of Cell Science 115 (2002) 679–688.

[57] R. Schwandner, K. Yamaguchi, Z. Cao, The Journal of Experimental Medicine 191 (2000) 1233–1240.

[58] J. Hartupee, C. Liu, M. Novotny, D. Sun, X. Li, T.A. Hamilton, Journal of Immunology 182 (2009) 1660–1666.

[59] M. Novatchkova, A. Leibbrandt, J. Werzowa, A. Neubuser, F. Eisenhaber, Trends in Biochemical Sciences 28 (2003) 226–229.

[60] C. Liu, W. Qian, Y. Qian, N.V. Giltiay, Y. Lu, S. Swaidani, S. Misra, L. Deng, Z.J. Chen, X. Li, Science Signaling 2 (2009) ra63.

[61] C. Liu, S. Swaidani, W. Qian, Z. Kang, P. Sun, Y. Han, C. Wang, M.F. Gulen, W. Yin, C. Zhang, P.L. Fox, M. Aronica, T.A. Hamilton, S. Misra, J. Deng, X. Li, Science Signaling 4 (2011) ra72.

[62] A. Maitra, F. Shen, W. Hanel, K. Mossman, J. Tocker, D. Swart, S.L. Gaffen, Proceedings of the National Academy of Sciences of the United States of America 104 (2007) 7506–7511.

[63] R.M. Onishi, S.J. Park, W. Hanel, A.W. Ho, A. Maitra, S.L. Gaffen, The Journal of Biological Chemistry 285 (2010) 32751–32759.

[64] S.U. Sonder, S. Saret, W. Tang, D.E. Sturdevant, S.F. Porcella, U. Siebenlist, The Journal of Biological Chemistry 286 (2011) 12881–12890.

[65] A.W. Ho, F. Shen, H.R. Conti, N. Patel, E.E. Childs, A.C. Peterson, N. Hernandez-Santos, J.K. Kolls, L.P. Kane, W. Ouyang, S.L. Gaffen, Journal of Immunology 185 (2010) 1063–1070.

[66] K. Bulek, C. Liu, S. Swaidani, L. Wang, R.C. Page, M.F. Gulen, T. Herjan, A. Abbadi, W. Qian, D. Sun, M. Lauer, V. Hascall, S. Misra, M.R. Chance, M. Aronica, T. Hamilton, X. Li, Nature Immunology 12 (2011) 844–852.

[67] D. Sun, M. Novotny, K. Bulek, C. Liu, X. Li, T. Hamilton, Nature Immunology 12 (2011) 853–860.

[68] C. Wang, L. Wu, K. Bulek, B.N. Martin, J.A. Zepp, Z. Kang, C. Liu, T. Herjan, S. Misra, J.A. Carman, J. Gao, A. Dongre, S. Han, K.D. Bunting, J.S. Ko, H. Xiao, V.K. Kuchroo, W. Ouyang, X. Li, Nature Immunology (2012).

[69] A. Strange, F. Capon, C.C. Spencer, J. Knight, M.E. Weale, M.H. Allen, A. Barton, G. Band, C. Bellenguez, J.G. Bergboer, J.M. Blackwell, E. Bramon, S.J. Bumpstead, J.P. Casas, M.J. Cork, A. Corvin, P. Deloukas, A. Dilthey, A. Duncanson, S. Edkins, X. Estivill, O. Fitz Gerald, C. Freeman, E. Giardina, E. Gray, A. Hofer, U. Huffmeier, S.E. Hunt, A.D. Irvine, J. Jankowski, B. Kirby, C. Langford, J. Lascorz, J. Leman, S. Leslie, L. Mallbris, H.S. Markus, C.G. Mathew, W.H. McLean, R. McManus, R. Mossner, L. Moutsianas, A.T. Naluai, F.O. Nestle, G. Novelli, A. Onoufriadis, C.N. Palmer, C. Perricone, M. Pirinen, R. Plomin, S.C. Potter, R.M. Pujol, A. Rautanen, E. Riveira-Munoz, A.W. Ryan, W. Salmhofer, L. Samuelsson, S.J. Sawcer, J. Schalkwijk, C.H. Smith, M. Stahle, Z. Su, R. Tazi-Ahnini, H. Traupe, A.C. Viswanathan, R.B. Warren, W. Weger, K. Wolk, N. Wood, J. Worthington, H.S. Young, P.L. Zeeuwen, A. Hayday, A.D. Burden, C.E. Griffiths, J. Kere, A. Reis, G. McVean, D.M. Evans, M.A. Brown, J.N. Barker, L. Peltonen, P. Donnelly, R.C. Trembath, Nature Genetics 42 (2010) 985–990.

[70] E. Ellinghaus, D. Ellinghaus, P.E. Stuart, R.P. Nair, S. Debrus, J.V. Raelson, M. Belouchi, H. Fournier, C. Reinhard, J. Ding, Y. Li, T. Tejasvi, J. Gudjonsson, S.W. Stoll, J.J. Voorhees, S. Lambert, S. Weidinger, B. Eberlein, M. Kunz, P. Rahman, D.D. Gladman, C. Gieger, H.E. Wichmann, T.H. Karlsen, G. Mayr, M. Albrecht, D. Kabelitz, U. Mrowietz, G.R. Abecasis, J.T. Elder, S. Schreiber, M. Weichenthal, A. Franke, Nature Genetics 42 (2010) 991–995.

[71] U. Huffmeier, S. Uebe, A.B. Ekici, J. Bowes, E. Giardina, E. Korendowych, K. Juneblad, M. Apel, R. McManus, P. Ho, I.N. Bruce, A.W. Ryan, F. Behrens, J. Lascorz, B. Bohm, H. Traupe, J. Lohmann, C. Gieger, H.E. Wichmann, C. Herold, M. Steffens, L. Klareskog, T.F. Wienker, O. Fitz Gerald, G.M. Allenius, N.J. McHugh, G. Novelli, H. Burkhardt, A. Barton, A. Reis, Nature Genetics 42 (2010) 996–999.

[72] J.F. Wright, F. Bennett, B. Li, J. Brooks, D.P. Luxenberg, M.J. Whitters, K.N. Tomkinson, L.J. Fitz, N.M. Wolfman, M. Collins, K. Dunussi-Joannopoulos, M. Chatterjee-Kishore, B.M. Carreno, Journal of Immunology 181 (2008) 2799–2805.

[73] R.E. Kuestner, D.W. Taft, A. Haran, C.S. Brandt, T. Breder, K. Lum, B. Harder, S. Okada, C.D. Ostrander, J.L. Kreindler, S.J. Auja, B. Reardon, M. Moore, P. Shea, R. Schreckhise, T.R. Bukowski, S. Presnell, P. Guerra-Lewis, J. Parrish-Novak, J.L. Ellsworth, S. Jaspers, K.E. Lewis, M. Appleby, J.K. Kolls, M. Rixon, J.W. West, Z. Gao, S.D. Levin, Journal of Immunology 179 (2007) 5462–5473.

[74] Y. Hu, N. Ota, I. Peng, C.J. Refino, D.M. Danilenko, P. Caplazi, W. Ouyang, Journal of Immunology 184 (2010) 4307–4316.

[75] M. Kawaguchi, F. Kokubu, S. Matsukura, K. Ieki, M. Odaka, S. Watanabe, S. Suzuki, M. Adachi, S.K. Huang, The Journal of Pharmacology and Experimental Therapeutics 307 (2003) 1213–1220.

[76] Y. Zhou, M.L. Toh, S. Zrioual, P. Miossec, Cytokine 38 (2007) 157–164.

[77] F. Shen, N. Li, P. Gade, D.V. Kalvakolanu, T. Weibley, B. Doble, J.R. Woodgett, T.D. Wood, S.L. Gaffen, Science Signaling 2 (2009) ra8.


Please cite this article as: X. Song, Y. Qian, Cellular Signalling (2013), http://dx.doi.org/10.1016/j.cellsig.2013.07.021

ARTICLE IN PRESS

X. Song, Y. Qian / Cellular Signalling xxx (2013) xxx–xxx

[78] H. Hacker, V. Redecke, B. Blagoev, I. Kratchmarova, L.C. Hsu, G.G. Wang, M.P. Kamps, E. Raz, H. Wagner, G. Hacker, M. Mann, M. Karin, Nature 439 (2006) 204–207.

[79] G. Oganesian, S.K. Saha, B. Guo, J.Q. He, A. Shahangian, B. Zarnegar, A. Perry, G. Cheng, Nature 439 (2006) 208–211.

[80] P.H. Tseng, A. Matsuzawa, W. Zhang, T. Mino, D.A. Vignali, M. Karin, Nature Immunology 11 (2009) 70–75.

[81] G. Cheng, A.M. Cleary, Z.S. Ye, D.I. Hong, S. Lederman, D. Baltimore, Science 267 (1995) 1494–1498.

[82] S. Krajewski, J.M. Zapata, M. Krajewska, T. VanArsdale, A. Shabaik, R.D. Gascoyne, J.C. Reed, Journal of Immunology 159 (1997) 5841–5852.

[83] J.Q. He, B. Zarnegar, G. Oganesian, S.K. Saha, S. Yamazaki, S.E. Doyle, P.W. Dempsey, G. Cheng, The Journal of Experimental Medicine 203 (2006) 2413–2418.

[84] P. Xie, L.L. Stunz, K.D. Larison, B. Yang, G.A. Bishop, Immunity 27 (2007) 253–267.

[85] S. Zhu, W. Pan, P. Shi, H. Gao, F. Zhao, X. Song, Y. Liu, L. Zhao, X. Li, Y. Shi, Y. Qian, The Journal of Experimental Medicine 207 (2010) 2647–2662.

[86] J.A. Zepp, C. Liu, W. Qian, L. Wu, M.F. Gulen, Z. Kang, X. Li, Journal of Immunology 189 (2012) 33–37.

[87] K.I. Nakayama, K. Nakayama, Nature Reviews. Cancer 6 (2006) 369–381.

[88] P. Shi, S. Zhu, Y. Lin, Y. Liu, Z. Chen, Y. Shi, Y. Qian, Science Signaling 4 (2011) ra73.

[89] F. Qu, H. Gao, S. Zhu, P. Shi, Y. Zhang, Y. Liu, B. Jallal, Y. Yao, Y. Shi, Y. Qian, Molecular and Cellular Biology (2012).

[90] B. Zhong, X. Liu, X. Wang, S.H. Chang, A. Wang, J.M. Reynolds, C. Dong, Nature Immunology 13 (2012) 1110–1117.

[91] A.V. Garg, M. Ahmed, A.N. Vallejo, A. Ma, S.L. Gaffen, Science Signaling 6 (2013) ra44.

[92] S. Kotake, N. Udagawa, N. Takahashi, K. Matsuzaki, K. Itoh, S. Ishiyama, S. Saito, K. Inoue, N. Kamatani, M.T. Gillespie, T.J. Martin, T. Suda, The Journal of Clinical Investigation 103 (1999) 1345–1352.

[93] M. Ziolkowska, A. Koc, G. Luszczkyiewicz, K. Ksiezopolska-Pietrzak, E. Klimczak, H. Chwalinska-Sadowska, W. Maslinski, Journal of Immunology 164 (2000) 2832–2838.

[94] E. Lubberts, M.I. Koenders, B. Oppers-Walgreen, L. van den Bersselaar, C.J. Coenen-de Roo, L.A. Joosten, W.B. van den Berg, Arthritis and Rheumatism 50 (2004) 650–659.

[95] E. Lubberts, L.A. Joosten, F.A. van de Loo, P. Schwarzenberger, J. Kolls, W.B. van den Berg, Inflammation Research: Official Journal of the European Histamine Research Society... [et al.] 51 (2002) 102–104.

[96] S. Nakae, A. Nambu, K. Sudo, Y. Iwakura, Journal of Immunology 171 (2003) 6173–6177.

[97] C. Lock, G. Hermans, R. Pedotti, A. Brendolan, E. Schadt, H. Garren, A. Langer-Gould, S. Strober, B. Cannella, J. Allard, P. Klonowski, A. Austin, N. Lad, N. Kaminski, S.J. Galli, J.R. Oksenberg, C.S. Raine, R. Heller, L. Steinman, Nature Medicine 8 (2002) 500–508.

[98] J.S. Tzartos, M.A. Friese, M.J. Craner, J. Palace, J. Newcombe, M.M. Esiri, L. Fugger, The American Journal of Pathology 172 (2008) 146–155.

[99] C.L. Langrish, Y. Chen, W.M. Blumenschein, J. Mattson, B. Basham, J.D. Sedgwick, T. McClanahan, R.A. Kastelein, D.J. Cua, The Journal of Experimental Medicine 201 (2005) 233–240.

[100] Y. Komiyama, S. Nakae, T. Matsuki, A. Nambu, H. Ishigame, S. Kakuta, K. Sudo, Y. Iwakura, Journal of Immunology 177 (2006) 566–573.

[101] S. Fujino, A. Andoh, S. Bamba, A. Ogawa, K. Hata, Y. Araki, T. Bamba, Y. Fujiyama, Gut 52 (2003) 65–70.

[102] T. Sugihara, A. Kobori, H. Imaeda, T. Tsujikawa, K. Amagase, K. Takeuchi, Y. Fujiyama, A. Andoh, Clinical and Experimental Immunology 160 (2010) 386–393.

[103] J. Seiderer, I. Elben, J. Diegelmann, J. Glas, J. Stallhofer, C. Tillack, S. Pfennig, M. Jurgens, S. Schmechel, A. Konrad, B. Goke, T. Ochsenkuhn, B. Muller-Myhsok, P. Lohse, S. Brand, Inflammatory Bowel Diseases 14 (2008) 437–445.

[104] R.H. Duerr, K.D. Taylor, S.R. Brant, J.D. Rioux, M.J. Daly, A.H. Steinhart, C. Abraham, M. Regueiro, A. Griffiths, T. Dassopoulos, A. Bitton, H. Yang, S. Targan, L.W. Datta, E.O. Kistner, L.P. Schumm, A.T. Lee, P.K. Gregersen, M.M. Barmada, J.I. Rotter, D.L. Nicolae, J.H. Cho, Science 314 (2006) 1461–1463.

[105] J.C. Barrett, S. Hansoul, D.L. Nicolae, J.H. Cho, R.H. Duerr, J.D. Rioux, S.R. Brant, M.S. Silverberg, K.D. Taylor, M.M. Barmada, A. Bitton, T. Dassopoulos, L.W. Datta, T. Green, A.M. Griffiths, E.O. Kistner, M.T. Murtha, M.D. Regueiro, J.I. Rotter, L.P. Schumm, A.H. Steinhart, S.R. Targan, R.J. Xavier, Consortium NIG, C. Libioulle, C. Sandor, M. Lathrop, J. Belaiche, O. Dewit, I. Gut, S. Heath, D. Laukens, M. Mni, P. Rutgeerts, A. Van Gossum, D. Zelenika, D. Franchimont, J.P. Hugot, M. de Vos, S. Vermeire, E. Louis, Belgian-French IBDC, Wellcome Trust Case Control C, L.R. Cardon, C.A. Anderson, H. Drummond, E. Nimmo, T. Ahmad, N.J. Prescott, C.M. Onnie, S.A. Fisher, J. Marchini, J. Ghori, S. Bumpstead, R. Gwilliam, M. Tremelling, P. Deloukas, J. Mansfield, D. Jewell, J. Satsangi, C.G. Mathew, M. Parkes, M. Georges, M.J. Daly, Nature Genetics 40 (2008) 955–962.

[106] K. Wang, H. Zhang, S. Kugathasan, V. Annese, J.P. Bradfield, R.K. Russell, P.M. Sleiman, M. Imielinski, J. Glessner, C. Hou, D.C. Wilson, T. Walters, C. Kim, E.C. Frackelton, P. Lionetti, A. Barabino, J. Van Limbergen, S. Guthery, L. Denson, D. Piccoli, M. Li, M. Dubinsky, M. Silverberg, A. Griffiths, S.F. Grant, J. Satsangi, R. Baldassano, H. Hakonarson, The American Journal of Human Genetics 84 (2009) 399–405.

[107] C. Ciccacci, L. Biancone, D. Di Fusco, M. Ranieri, G. Condino, E. Giardina, S. Onali, T. Lepre, F. Pallone, G. Novelli, P. Borgiani, Journal of Crohn's & Colitis (2012).

[108] A. Ogawa, A. Andoh, Y. Araki, T. Bamba, Y. Fujiyama, Clinical Immunology 110 (2004) 55–62.

[109] R. Ito, M. Kita, M. Shin-Ya, T. Kishida, A. Urano, R. Takada, J. Sakagami, J. Imanishi, Y. Iwakura, T. Okanoue, T. Yoshikawa, K. Kataoka, O. Mazda, Biochemical and Biophysical Research Communications 377 (2008) 12–16.

[110] C. Johansen, P.A. Usher, R.B. Kjellerup, D. Lundsgaard, L. Iversen, K. Kragballe, The British Journal of Dermatology 160 (2009) 319–324.

[111] N.J. Wilson, K. Boniface, J.R. Chan, B.S. McKenzie, W.M. Blumenschein, J.D. Mattson, B. Basham, K. Smith, T. Chen, F. Morel, J.C. Leclon, R.A. Kastelein, D.J. Cua, T.K. McClanahan, E.P. Bowman, R. de Waal Malefyt, Nature Immunology 8 (2007) 950–957.

[112] A. Johnston, Y. Fritz, S.M. Dawes, D. Diaconu, P.M. Al-Attar, A.M. Guzman, C.S. Chen, W. Fu, J.E. Gudjonsson, T.S. McCormick, N.L. Ward, Journal of Immunology 190 (2013) 2252–2262.

[113] M. Cargill, S.J. Schrodi, M. Chang, V.E. Garcia, R. Brandon, K.P. Callis, N. Matsunami, K.G. Ardlie, D. Civello, J.J. Catanese, D.U. Leong, J.M. Panko, L.B. McAllister, C.B. Hansen, J. Papenfuss, S.M. Prescott, T.J. White, M.F. Leppert, G.G. Krueger, A.B. Begovich, The American Journal of Human Genetics 80 (2007) 273–290.

[114] H.L. Rizzo, S. Kagami, K.G. Phillips, S.E. Kurtz, S.L. Jacques, A. Blauvelt, Journal of Immunology 186 (2011) 1495–1502.

[115] M.B. Teunissen, C.W. Koomen, R. de Waal Malefyt, E.A. Wierenga, J.D. Bos, The Journal of Investigative Dermatology 111 (1998) 645–649.

[116] C. Albanesi, A. Cavani, G. Girolomoni, Journal of Immunology 162 (1999) 494–502.

[117] A. Chiricozzi, E. Guttman-Yassky, M. Suarez-Farinas, K.E. Nograles, S. Tian, I. Cardinal, S. Chimenti, J.G. Krueger, The Journal of Investigative Dermatology 131 (2011) 677–687.

[118] E.G. Harper, C. Guo, H. Rizzo, J.V. Lillis, S.E. Kurtz, I. Skorcheva, D. Purdy, E. Fitch, M. Iordanov, A. Blauvelt, The Journal of Investigative Dermatology 129 (2009) 2175–2183.

[119] Z. Zhang, V.C. Kyttaris, G.C. Tsokos, Journal of Immunology 183 (2009) 3160–3169.

[120] J.C. Crispin, M. Oukka, G. Bayliss, R.A. Cohen, C.A. Van Beek, I.E. Stillman, V.C. Kyttaris, Y.T. Juan, G.C. Tsokos, Journal of Immunology 181 (2008) 8761–8766.

[121] P. Pisitkun, H.L. Ha, H. Wang, E. Claudio, C.C. Tivy, H. Zhou, T.N. Mayadas, G.G. Illei, U. Siebenlist, Immunity 37 (2012) 1104–1115.

[122] G. Dong, R. Ye, W. Shi, S. Liu, T. Wang, X. Yang, N. Yang, X. Yu, Chinese Medical Journal 116 (2003) 543–548.

[123] A. Doreau, A. Belot, J. Bastid, B. Riche, M.C. Trescol-Biemont, B. Ranchin, N. Fabien, P. Cochat, C. Pouteil-Noble, P. Trolliet, I. Durieu, J. Tebib, B. Kassai, S. Ansieau, A. Puisieux, J.F. Eliaou, N. Bonnefoy-Berard, Nature Immunology 10 (2009) 778–785.

[124] J.A. Emamaullee, J. Davis, S. Merani, C. Toso, J.F. Elliott, A. Thiesen, A.M. Shapiro, Diabetes 58 (2009) 1302–1311.

[125] S. Molet, Q. Hamid, F. Davoine, E. Nutku, R. Taha, N. Page, R. Olivenstein, J. Elias, J. Chakir, The Journal of Allergy and Clinical Immunology 108 (2001) 430–438.

[126] C. Doe, M. Bafadhel, S. Siddiqui, D. Desai, V. Mistry, P. Rugman, M. McCormick, J. Woods, R. May, M.A. Sleeman, I.K. Anderson, C.E. Brightling, Chest 138 (2010) 1140–1147.

[127] W. Al-Ramli, D. Prefontaine, F. Chouiali, J.G. Martin, R. Olivenstein, C. Lemiere, Q. Hamid, The Journal of Allergy and Clinical Immunology 123 (2009) 1185–1187.

[128] S.D. Hurst, T. Muchamuel, D.M. Gorman, J.M. Gilbert, T. Clifford, S. Kwan, S. Menon, B. Seymour, C. Jackson, T.T. Kung, J.K. Brieland, S.M. Zurawski, R.W. Chapman, G. Zurawski, R.L. Coffman, Journal of Immunology 169 (2002) 443–453.

[129] S. Nakae, Y. Komiyama, A. Nambu, K. Sudo, M. Iwase, I. Homma, K. Sekikawa, M. Asano, Y. Iwakura, Immunity 17 (2002) 375–387.

[130] S. Lajoie, I.P. Lewkowich, Y. Suzuki, J.R. Clark, A.A. Sproles, K. Dienger, A.L. Budelsky, M. Wills-Karp, Nature Immunology 11 (2010) 928–935.

[131] M. Kudo, A.C. Melton, C. Chen, M.B. Engler, K.E. Huang, X. Ren, Y. Wang, X. Bernstein, J.T. Li, K. Atabai, X. Huang, D. Sheppard, Nature Medicine 18 (2012) 547–554.

[132] M. Tosolini, A. Kirilovsky, B. Mlecnik, T. Fredriksen, S. Mauger, G. Bindea, A. Berger, P. Bruneval, W.H. Fridman, F. Pages, J. Galon, Cancer Research 71 (2011) 1263–1271.

[133] J.P. Zhang, J. Yan, J. Xu, X.H. Pang, M.S. Chen, L. Li, C. Wu, S.P. Li, L. Zheng, Journal of Hepatology 50 (2009) 980–989.

[134] I. Kryczek, M. Banerjee, P. Cheng, L. Vatan, W. Szeliga, S. Wei, E. Huang, E. Finlayson, D. Simeone, T.H. Welling, A. Chang, G. Coukos, R. Liu, W. Zou, Blood 114 (2009) 1141–1149.

[135] L. Wang, T. Yi, M. Kortylewski, D.M. Pardoll, D. Zeng, H. Yu, The Journal of Experimental Medicine 206 (2009) 1457–1464.

[136] Y.S. Hyun, D.S. Han, A.R. Lee, C.S. Eun, J. Youn, H.Y. Kim, Carcinogenesis 33 (2012) 931–936.

[137] W.J. Chae, T.F. Gibson, D. Zelterman, L. Hao, O. Henegariu, A.L. Bothwell, Proceedings of the National Academy of Sciences of the United States of America 107 (2010) 5540–5544.

[138] S.I. Grivennikov, K. Wang, D. Mucida, C.A. Stewart, B. Schnabl, D. Jauch, K. Taniguchi, G.Y. Yu, C.H. Osterreicher, K.E. Hung, C. Datz, Y. Feng, E.R. Fearon, M. Oukka, L. Tessarollo, V. Coppola, F. Yarovinsky, H. Cheroutre, L. Eckmann, G. Trinchieri, M. Karin, Nature 491 (2012) 254–258.

[139] S. Wu, K.J. Rhee, E. Albesiano, S. Rabizadeh, X. Wu, H.R. Yen, D.L. Huso, F.L. Brancati, E. Wick, F. McAllister, F. Housseau, D.M. Pardoll, C.L. Sears, Nature Medicine 15 (2009) 1016–1022.

[140] L. Wang, T. Yi, W. Zhang, D.M. Pardoll, H. Yu, Cancer Research 70 (2010) 10112–10120.

[141] D. He, H. Li, N. Yusuf, C.A. Elmets, M. Athar, S.K. Katiyar, H. Xu, PLoS One 7 (2012) e32126.

[142] J.L. Langowski, X. Zhang, L. Wu, J.D. Mattson, T. Chen, K. Smith, B. Basham, T. McClanahan, R.A. Kastelein, M. Oft, Nature 442 (2006) 461–465.

[143] N. Martin-Orozco, P. Muranski, Y. Chung, X.O. Yang, T. Yamazaki, S. Lu, P. Hwu, N.P. Restifo, W.W. Overwijk, C. Dong, Immunity 31 (2009) 787–798.

[144] Y. Carmi, G. Rinott, S. Dotan, M. Elkabetz, P. Rider, E. Voronov, R.N. Apte, Journal of Immunology 186 (2011) 3462–3471.


Please cite this article as: X. Song, Y. Qian, Cellular Signalling (2013), http://dx.doi.org/10.1016/j.cellsig.2013.07.021

ARTICLE IN PRESS

X. Song, Y. Qian / Cellular Signalling xxx (2013) xxx–xxx 13

[145] Q. Li, Y. Han, G. Fei, Z. Guo, T. Ren, Z. Liu, Immunology Letters 148 (2012) 144–150.
[146] I. Kryczek, S. Wei, W. Szeliga, L. Vatan, W. Zou, Blood 114 (2009) 357–359.
[147] W.J. Chae, A.L. Bothwell, Biochemical and Biophysical Research Communications 414 (2011) 31–36.
[148] Z. Tong, X.O. Yang, H. Yan, W. Liu, X. Niu, Y. Shi, W. Fang, B. Xiong, Y. Wan, C. Dong, PLoS One 7 (2012) e34959.
[149] S.J. Aujla, Y.R. Chan, M. Zheng, M. Fei, D.J. Askew, D.A. Pociask, T.A. Reinhart, F. McAllister, J. Edeal, K. Gaus, S. Husain, J.L. Kreindler, P.J. Dubin, J.M. Pilewski, M.M. Myerburg, C.A. Mason, Y. Iwakura, J.K. Kolls, Nature Medicine 14 (2008) 275–281.
[150] I.I. Ivanov, K. Atarashi, N. Manel, E.L. Brodie, T. Shima, U. Karaoz, D. Wei, K.C. Goldfarb, C.A. Santee, S.V. Lynch, T. Tanoue, A. Imaoka, K. Itoh, K. Takeda, Y. Umesaki, K. Honda, D.R. Littman, Cell 139 (2009) 485–498.
[151] Y. Zheng, P.A. Valdez, D.M. Danilenko, Y. Hu, S.M. Sa, Q. Gong, A.R. Abbas, Z. Modrusan, N. Ghilardi, F.J. de Sauvage, W. Ouyang, Nature Medicine 14 (2008) 282–289.
[152] Y. Lin, S. Ritchea, A. Logar, S. Slight, M. Messmer, J. Rangel-Moreno, L. Guglani, J.F. Alcorn, H. Strawbridge, S.M. Park, R. Onishi, N. Nyugen, M.J. Walter, D. Pociask, T.D. Randall, S.L. Gaffen, Y. Iwakura, J.K. Kolls, S.A. Khader, Immunity 31 (2009) 799–810.
[153] S. Saijo, S. Ikeda, K. Yamabe, S. Kakuta, H. Ishigame, A. Akitsu, N. Fujikado, T. Kusaka, S. Kubo, S.H. Chung, R. Komatsu, N. Miura, Y. Adachi, N. Ohno, K. Shibuya, N. Yamamoto, K. Kawakami, S. Yamasaki, T. Saito, S. Akira, Y. Iwakura, Immunity 32 (2010) 681–691.
[154] L. Van Maele, C. Carnoy, D. Cayet, P. Songhet, L. Dumoutier, I. Ferrero, L. Janot, F. Erard, J. Bertout, H. Leger, F. Sebbane, A. Benecke, J.C. Renauld, W.D. Hardt, B. Ryffel, J.C. Sirard, Journal of Immunology 185 (2010) 1177–1185.
[155] Q. Wu, R.J. Martin, J.G. Rino, R. Breed, R.M. Torres, H.W. Chu, Microbes and Infection/Institut Pasteur 9 (2007) 78–86.
[156] D.B. Holland, R.A. Bojar, M.D. Farrar, K.T. Holland, FEMS Microbiology Letters 290 (2009) 149–155.
[157] P. Pfeifer, M. Voss, B. Wonnenberg, J. Hellberg, F. Seiler, P.M. Lepper, M. Bischoff, F. Langer, H.J. Schafers, M.D. Menger, R. Bals, C. Beisswenger, American Journal of Respiratory Cell and Molecular Biology 48 (2013) 415–421.
[158] X. Song, S. Zhu, P. Shi, Y. Liu, Y. Shi, S.D. Levin, Y. Qian, Nature Immunology 12 (2011) 1151–1158.
[159] E. Im, J. Jung, S.H. Rhee, Journal of Interferon & Cytokine Research: The official Journal of the International Society for Interferon and Cytokine Research 32 (2012) 583–591.
[160] V. Ramirez-Carrozzi, A. Sambandam, E. Luis, Z. Lin, S. Jeet, J. Lesch, J. Hackney, J. Kim, M. Zhou, J. Lai, Z. Modrusan, T. Sai, W. Lee, M. Xu, P. Caplazi, L. Diehl, J. de Voss, M. Balazs, L. Gonzalez Jr., H. Singh, W. Ouyang, R. Pappu, Nature Immunology 12 (2011) 1159–1166.
[161] C. Johansen, J.L. Riis, A. Gedebjerg, K. Kragballe, L. Iversen, The Journal of Biological Chemistry 286 (2011) 25487–25494.
[162] H. Li, J. Chen, A. Huang, J. Stinson, S. Heldens, J. Foster, P. Dowd, A.L. Gurney, W.J. Wood, Proceedings of the National Academy of Sciences of the United States of America 97 (2000) 773–778.
[163] Y. Yamaguchi, K. Fujio, H. Shoda, A. Okamoto, N.H. Tsuno, K. Takahashi, K. Yamamoto, Journal of Immunology 179 (2007) 7128–7136.
[164] S.H. Chang, J.M. Reynolds, B.P. Pappu, G. Chen, G.J. Martinez, C. Dong, Immunity 35 (2011) 611–621.
[165] J.M. Reynolds, G.J. Martinez, K.C. Nallaparaju, S.H. Chang, Y.H. Wang, C. Dong, Journal of Immunology 189 (2012) 4226–4230.
[166] M.M. Fort, J. Cheung, D. Yen, J. Li, S.M. Zurawski, S. Lo, S. Menon, T. Clifford, B. Hunte, R. Lesley, T. Muchamuel, S.D. Hurst, G. Zurawski, M.W. Leach, D.M. Gorman, D.M. Rennick, Immunity 15 (2001) 985–995.
[167] P. Angkasekwinai, H. Park, Y.H. Wang, Y.H. Wang, S.H. Chang, D.B. Corry, Y.J. Liu, Z. Zhu, C. Dong, The Journal of Experimental Medicine 204 (2007) 1509–1517.
[168] C. Zaph, Y. Du, S.A. Saenz, M.G. Nair, J.G. Perrigue, B.C. Taylor, A.E. Troy, D.E. Kobuley, R.A. Kastelein, D.J. Cua, Y. Yu, D. Artis, The Journal of Experimental Medicine 205 (2008) 2191–2198.
[169] Y. Sonobe, H. Takeuchi, K. Kataoka, H. Li, S. Jin, M. Mimuro, Y. Hashizume, Y. Sano, T. Kanda, T. Mizuno, A. Suzumura, The Journal of Biological Chemistry 284 (2009) 31834–31842.
[170] V. Dolgachev, B.C. Petersen, A.L. Budelsky, A.A. Berlin, N.W. Lukacs, Journal of Immunology 183 (2009) 5705–5715.
[171] Y.H. Wang, P. Angkasekwinai, N. Lu, K.S. Voo, K. Arima, S. Hanabuchi, A. Hippe, C.J. Corrigan, C. Dong, B. Homey, Z. Yao, S. Ying, D.P. Huston, Y.J. Liu, The Journal of Experimental Medicine 204 (2007) 1837–1847.
[172] C.M. Kang, A.S. Jang, M.H. Ahn, J.A. Shin, J.H. Kim, Y.S. Choi, T.Y. Rhim, C.S. Park, American Journal of Respiratory Cell and Molecular Biology 33 (2005) 290–296.
[173] M.A. Kleinschek, A.M. Owyang, B. Joyce-Shaikh, C.L. Langrish, Y. Chen, D.M. Gorman, W.M. Blumenschein, T. McClanahan, F. Brombacher, S.D. Hurst, R.A. Kastelein, D.J. Cua, The Journal of Experimental Medicine 204 (2007) 161–170.
[174] C.M. Kang, A.S. Jang, M.H. Ahn, J.A. Shin, J.H. Kim, Y.S. Choi, T.Y. Rhim, C.S. Park, American Journal of Respiratory Cell and Molecular Biology 33 (2005) 290–296.
[175] H.S. Yu, P. Angkasekwinai, S.H. Chang, Y. Chung, C. Dong, Journal of Korean Medical Science 25 (2010) 829–834.
[176] J. Lee, W.H. Ho, M. Maruoka, R.T. Corpuz, D.T. Baldwin, J.S. Foster, A.D. Goddard, D.G. Yansura, R.L. Vandlen, W.I. Wood, A.L. Gurney, The Journal of Biological Chemistry 276 (2001) 1660–1664.
[177] Y. Shi, S.J. Ullrich, J. Zhang, K. Connolly, K.J. Grzegorzewski, M.C. Barber, W. Wang, K. Wathen, V. Hodge, C.L. Fisher, H. Olsen, S.M. Ruben, I. Knyazev, Y.H. Cho, V. Kao,

K.A. Wilkinson, J.A. Carrell, R. Ebner, The Journal of Biological Chemistry 275 (2000) 19167–19176.
[178] E.A. Rickel, L.A. Siegel, B.R. Yoon, J.B. Rottman, D.G. Kugler, D.A. Swart, P.M. Anders, J.E. Tocker, M.R. Comeau, A.L. Budelsky, Journal of Immunology 181 (2008) 4299–4310.
[179] R. Caruso, C. Stolfi, M. Sarra, A. Rizzo, M.C. Fantini, F. Pallone, T.T. MacDonald, G. Monteleone, Blood 113 (2009) 3512–3519.
[180] A. Terashima, H. Watarai, S. Inoue, E. Sekine, R. Nakagawa, K. Hase, C. Iwamura, H. Nakajima, T. Nakayama, M. Taniguchi, The Journal of Experimental Medicine 205 (2008) 2727–2733.
[181] P. Stock, V. Lombardi, V. Kohlrault, O. Akbari, Journal of Immunology 182 (2009) 5116–5122.
[182] D.R. Neill, S.H. Wong, A. Bellosi, R.J. Flynn, M. Daly, T.K. Langford, C. Bucks, C.M. Kane, P.G. Fallon, R. Pannell, H.E. Jolin, A.N. McKenzie, Nature 464 (2010) 1367–1370.
[183] S.A. Saenz, M.C. Siracusa, J.G. Perrigoue, S.P. Spencer, J.F. Urban Jr., J.E. Tocker, A.L. Budelsky, M.A. Kleinschek, R.A. Kastelein, T. Kambayashi, A. Bhandoola, D. Artis, Nature 464 (2010) 1362–1366.
[184] A.E. Price, H.E. Liang, B.M. Sullivan, R.L. Reinhardt, C.J. Eisley, D.J. Erle, R.M. Locksley, Proceedings of the National Academy of Sciences of the United States of America 107 (2010) 11489–11494.
[185] B.C. Petersen, A.L. Budelsky, A.P. Baptist, M.A. Schaller, N.W. Lukacs, Nature Medicine 18 (2012) 751–758.
[186] P. Angkasekwinai, S.H. Chang, M. Thapa, H. Watarai, C. Dong, Nature Immunology 11 (2010) 250–256.
[187] M. Suzukawa, H. Morita, A. Nambu, K. Arae, E. Shimura, A. Shibui, S. Yamaguchi, K. Suzukawa, W. Nakanishi, K. Oboki, N. Kajiwara, T. Ohno, A. Ishii, H. Korner, D.J. Cua, H. Suto, T. Yoshimoto, Y. Iwakura, T. Yamasoba, K. Ohta, K. Sudo, H. Saito, K. Okumura, D.H. Broide, K. Matsumoto, S. Nakae, Journal of Immunology 189 (2012) 3641–3652.
[188] E. Claudio, S.U. Sonder, S. Saret, G. Carvalho, T.R. Ramalingam, T.A. Wynn, A. Chariot, A. Garcia-Perganeda, A. Leonardi, A. Paun, A. Chen, N.Y. Ren, H. Wang, U. Siebenlist, Journal of Immunology 182 (2009) 1617–1630.
[189] S. Swaidani, K. Bulek, Z. Kang, C. Liu, Y. Lu, W. Yin, M. Aronica, X. Li, Journal of Immunology 182 (2009) 1631–1640.
[190] S. Swaidani, K. Bulek, Z. Kang, M.F. Gulen, C. Liu, W. Yin, A. Abbadi, M. Aronica, X. Li, Journal of Immunology 187 (2011) 3155–3164.
[191] Z. Kang, S. Swaidani, W. Yin, C. Wang, J.L. Barlow, M.F. Gulen, K. Bulek, J.S. Do, M. Aronica, A.N. McKenzie, B. Min, X. Li, Immunity 36 (2012) 821–833.
[192] Y. Miazawa, H. Nakajima, K. Suzuki, T. Tamachi, K. Ikeda, J. Inoue, Y. Saito, I. Iwamoto, Journal of Immunology 176 (2006) 1013–1018.
[193] A.M. Owyang, C. Zaph, E.H. Wilson, K.J. Guild, T. McClanahan, H.R. Miller, D.J. Cua, M. Goldschmidt, C.A. Hunter, R.A. Kastelein, D. Artis, The Journal of Experimental Medicine 203 (2006) 843–849.
[194] P.G. Fallon, S.J. Ballantyne, N.E. Mangan, J.L. Barlow, A. Dasvarma, D.R. Hewett, A. McIlgorm, H.E. Jolin, A.N. McKenzie, The Journal of Experimental Medicine 203 (2006) 1105–1116.
[195] A. Zhao, J.F. Urban Jr., R. Sun, J. Stiltz, M. Morimoto, L. Notari, K.B. Madden, Z. Yang, V. Grinchuk, T.R. Ramalingam, T.A. Wynn, T. Shea-Donohue, Journal of Immunology 185 (2010) 6921–6929.
[196] R. Caruso, M. Sarra, C. Stolfi, A. Rizzo, D. Fina, M.C. Fantini, F. Pallone, T.T. MacDonald, G. Monteleone, Gastroenterology 136 (2009) 2270–2279.
[197] S.S. McHenga, D. Wang, C. Li, F. Shan, C. Lu, Cellular & Molecular Immunology 5 (2008) 425–431.
[198] G. Xu, L. Zhang, D.Y. Wang, R. Xu, Z. Liu, D.M. Han, X.D. Wang, K.J. Zuo, H.B. Li, Allergy 65 (2010) 581–589.
[199] S.J. Ballantyne, J.L. Barlow, H.E. Jolin, P. Nath, A.S. Williams, K.F. Chung, G. Sturton, S.H. Wong, A.N. McKenzie, The Journal of Allergy and Clinical Immunology 120 (2007) 1324–1331.
[200] T. Tamachi, Y. Maezawa, K. Ikeda, S. Kagami, M. Hatano, Y. Seto, A. Suto, K. Suzuki, N. Watanabe, Y. Saito, T. Tokuhisa, I. Iwamoto, H. Nakajima, The Journal of Allergy and Clinical Immunology 118 (2006) 606–614.
[201] T. Benatar, M.Y. Cao, Y. Lee, J. Lightfoot, N. Feng, X. Gu, V. Lee, H. Jin, M. Wang, J.A. Wright, A.H. Young, Cancer Immunology, Immunotherapy: CII 59 (2010) 805–817.
[202] T. Benatar, M.Y. Cao, Y. Lee, H. Li, N. Feng, X. Gu, V. Lee, H. Jin, M. Wang, S. Der, J. Lightfoot, J.A. Wright, A.H. Young, Cancer Immunology, Immunotherapy: CII 57 (2008) 1757–1769.
[203] S. Furuta, Y.M. Jeng, L. Zhou, L. Huang, I. Kuhn, M.J. Bissell, W.H. Lee, Science Translational Medicine 3 (2011) 78ra31.
[204] T. Kokubu, D.R. Haudenschild, T.A. Moseley, L. Rose, A.H. Reddi, The Journal of Histochemistry and Cytochemistry: Official Journal of the Histochemistry Society 56 (2008) 89–95.
[205] E.E. Moore, S. Presnell, U. Garrigues, A. Guilbot, E. LeGuern, D. Smith, L. Yao, T.E. Whitmore, T. Gilbert, T.D. Palmer, P.J. Horner, R.E. Kuestner, Neuromuscular Disorders: NMD 12 (2002) 141–150.
[206] A.J. Sanders, X. Guo, M.D. Mason, W.G. Jiang, Journal of Oncology 2010 (2010) 817375.
[207] T. Starnes, H.E. Broxmeyer, M.J. Robertson, R. Hromas, Journal of Immunology 169 (2002) 642–646.
[208] Y.H. Hong, H.S. Lillehoj, D.W. Park, S.H. Lee, J.Y. Han, J.H. Shin, M.S. Park, J.K. Kim, Veterinary Immunology and Immunopathology 126 (2008) 1–8.
[209] S. Al Omar, B.F. Flanagan, M. Almehmadi, S.E. Christmas, Cytokine 62 (2013) 123–130.

Please cite this article as: X. Song, Y. Qian, Cellular Signalling (2013), http://dx.doi.org/10.1016/j.cellsig.2013.07.021
