
JOURNAL OF BACTERIOLOGY, Sept. 1990, p. 4816–4826

0021-9193/90/094816-11$02.00/0

Copyright © 1990, American Society for Microbiology

# Biosynthesis of Anthraquinones by Interspecies Cloning of Actinorhodin Biosynthesis Genes in Streptomycetes: Clarification of Actinorhodin Gene Functions

PAUL L. BARTEL,$^{1\dagger}$ CHUN-BAO ZHU,$^{1\dagger}$ JAY S. LAMPEL,$^{1\ddag}$ DONALD C. DOSCH,$^1$ NEAL C. CONNORS,$^1$ WILLIAM R. STROHL,$^{1*}$ JOHN M. BEALE, JR.,$^{2||}$ AND HEINZ G. FLOSS$^2$

Department of Microbiology, The Ohio State University, 484 West 12th Avenue, Columbus, Ohio 43210,$^1$ and Department of Chemistry, University of Washington, Seattle, Washington 98195$^2$

Received 28 February 1990/Accepted 13 June 1990

Streptomyces galilaeus ATCC 31133 and ATCC 31671, producers of the anthracyclines aclacinomycin A and 2-hydroxyaklavinone, respectively, formed an anthraquinone, aloesaponarin II, when they were transformed with DNA from Streptomyces coelicolor containing four genetic loci, *actI*, *actIII*, *actIV*, and *actVII*, encoding early reactions in the actinorhodin biosynthesis pathway. Subcloning experiments indicated that a 2.8-kilobase-pair XhoI fragment containing only the *actI* and *actVII* loci was necessary for aloesaponarin II biosynthesis by *S*. galilaeus ATCC 31133. Aloesaponarin II was synthesized via the condensation of 8 acetyl coenzyme A equivalents, followed by a decarboxylation reaction as demonstrated by \[^{1,2}-{ }^{13} \mathrm{C}_{2}\]acetate feeding experiments. *S*. coelicolor B22 and B159, *actVI* blocked mutants, also formed aloesaponarin II as an apparent shunt product. Mutants of *S*. coelicolor blocked in several other steps in actinorhodin biosynthesis did not synthesize aloesaponarin II or other detectable anthraquinones. When *S*. galilaeus ATCC 31671 was transformed with the DNA carrying the *actI*, *actIII*, and *actVII* loci, the recombinant strain produced both aloesaponarin II and aklavinone, suggesting that the actinorhodin biosynthesis DNA encoded a function able to deoxygenate 2-hydroxyaklavinone to aklavinone. When *S*. galilaeus ATCC 31671 was transformed with a plasmid carrying only the intact *actIII* gene (pANT45), aklavinone was formed exclusively. These experiments indicate a function for the *actIII* gene, which is the reduction of the keto group at C-9 from the carboxy terminus of the assembled polyketide to the corresponding secondary alcohol. In the presence of the *actIII* gene, anthraquinones or anthracyclines formed as a result of dehydration and aromatization lack an oxygen function on the carbon on which the keto reductase operated. When *S*. galilaeus ATCC 31671 was transformed with the DNA carrying the *actI*, *actVII*, and *actIV* loci, the recombinant strain produced two novel anthraquinones, desoxyerythro-laccin, the 3-hydroxy analog of aloesaponarin II, and 1-*O*-methyldesoxyerythrolaccin. The results obtained in these experiments together with earlier data suggest a pathway for the biosynthesis of actinorhodin and related compounds by *S*. coelicolor.

The biosynthesis of the benzoisochromanquinone antibiotic actinorhodin (4), a polyketide antibiotic produced by Streptomyces coelicolor, has been studied intensively in recent years from both the biochemical (7, 13, 14) and genetic (15, 22–24, 30) viewpoints. Seven classes of blocked mutants have been described and placed in sequential order on the basis of cosynthetic and chemical studies (7, 13, 30). Studies on the molecular genetics of actinorhodin biosynthesis have provided fundamental knowledge on the structure and organization of antibiotic biosynthesis genes in streptomycetes, including the clustering of antibiotic biosynthesis structural genes (23, 24), production of the antibiotic by recombinant strains carrying all of the biosynthesis genes on a plasmid (23), and the potential homology shared by certain structural genes within several different pathways (i.e., the genes encoding polyketide synthases [22]). Actinorhodin biosynthesis genes also were used in the initial demonstration of the formation of hybrid antibiotics via interspecies cloning (17, 29). When the granaticin producer Streptomyces violaceoruber Tü22 was transformed with actinorhodin biosynthesis genes from *S*. coelicolor, a hybrid compound, dihydrogranatirhodin, was produced (17). When Streptomyces sp. strain AM-7161 was transformed with certain genes encoding only part of the actinorhodin pathway, the novel hybrid antibiotics mederrhodins A and B were formed (17, 29). Here we report the formation of an anthraquinone, aloesaponarin II, by actinorhodin-nonproducing streptomycetes transformed with the *actI*, *actIII*, *actVII*, and *actIV* (actinorhodin) biosynthesis genetic loci from *S*. coelicolor. Aloesaponarin II also was produced by the *S*. coelicolor *actVI* mutant strains B22 and B159.

The *actIII* gene apparently encodes a polyketide reductase (15, 22), which has been postulated to reduce one of the keto functions of the actinorhodin polyketide intermediate during its formation (15). This hypothesis, based on the DNA sequence of the *actIII* gene (15) and the inability of the *actI* and *actIII* mutants to cross-feed (30), has been difficult to prove because no corresponding intermediate has been isolated. It is probable, however, that the compound which would theoretically arise from an *actIII* mutation might be unstable, as judged from its presumed structure. In this report, we show that the presence of the *actIII* gene is

* Corresponding author.
† Present address: Department of Microbiology, State University of New York at Stony Brook, Stony Brook, NY 11794.
‡ Present address: Shanghai Institute for Pharmaceutical Industry, Shanghai, Peoples Republic of China.
§ Present address: Crop Genetics International, Inc., Hanover, MD 21076.
|| Present address: Department of Medicinal and Natural Products Chemistry, University of Texas at Austin, Austin, TX 78712.

TABLE 1. Bacterial strains and plasmids used$^a$

| Strain or plasmid | Relevant characteristics | Source or reference |
|-------------------|-------------------------|---------------------|
| **Streptomyces strain** |  |  |
| S. galilaeus 31133 | Aclacinomycin producer | ATCC |
| S. galilaeus 31671 | 2-Hydroxyaklavinone producer (actIII-equivalent negative) | ATCC |
| S. peucetius 29050 | Daunomycin producer | ATCC |
| S. azureus 14921 | Thiostrepton producer | ATCC |
| S. parvulus 2266 | Nonactin producer | D. Hopwood |
| S. coelicolor 1190 | Actinorhodin producer | D. Hopwood |
| S. coelicolor B17 | Blocked mutant (actIV) | D. Hopwood (30) |
| S. coelicolor B18 | Blocked mutant (actI) | D. Hopwood (30) |
| S. coelicolor B22 | Blocked mutant (actVI) | D. Hopwood (30) |
| S. coelicolor B40 | Blocked mutant (actVII) | D. Hopwood (30) |
| S. coelicolor B41 | Blocked mutant (actIII) | D. Hopwood (30) |
| S. coelicolor B159 | Blocked mutant (actVI) | D. Hopwood (30) |
| **Plasmid** |  |  |
| pIJ61 | Derivative of SLP1.2; LC; Thio$^r$ Neo$^r$ | D. Hopwood (34) |
| pIJ350 | Derivative of pIJ101; HC; Thio$^r$ | D. Hopwood (19) |
| pIJ702 | Derivative of pIJ350; HC; Thio$^r$ Mel$^+$ | D. Hopwood (16) |
| pIJ922 | Derivative of SCP2*; LC; Thio$^r$ | D. Hopwood (21) |
| pIJ941 | Derivative of SCP2*; LC; Hyg$^r$ Thio$^r$ | D. Hopwood (21) |
| pIJ2303 | pIJ922 with 32.5-kbp DNA insert from *S. coelicolor* containing actinorhodin biosynthesis pathway genes | D. Hopwood (23, 24) |
| pANT12 | pIJ350 with 8.8-kbp *PstI* subclone from pIJ2303; contains *actI*, *actIII*, *actIV*, and *actVII* loci | This work |
| pANT28 | pIJ941 with 8.8-kbp *PstI* subclone from pANT12; contains *actI*, *actIII*, *actIV*, and *actVII* loci | This work |
| pANT31 | pIJ61 with 8.8-kbp *PstI* subclone from pANT12; contains *actI*, *actIII*, *actIV*, and *actVII* loci | This work |
| pANT33 | pIJ941 with 4.7-kbp *PstI-SstI* subclone from pANT12; contains *actI*, *actIII*, and *actVII* loci | This work |
| pANT35 | pIJ702 with 5.3-kbp *SphI* subclone from pANT12; contains *actI*, *actVII*, and *actIV* loci | This work |
| pANT36 | pIJ702 with 5.3-kbp *SphI* subclone from pANT12; in opposite orientation of pANT35 | This work |
| pANT43 | pIJ61 with 2.8-kbp *XhoI* subclone from pANT12; contains *actI* and *actVII* loci | This work |
| pANT44 | pIJ61 with 2.2-kbp *BamHI* subclone from pANT12; contains part of *actI* locus | This work |
| pANT45 | pIJ61 with 1.1-kbp *BamHI* subclone from pANT12; contains *actIII* locus | This work |
| pANT46 | pIJ61 with 0.8-kbp *BamHI* subclone from pANT12; contains part of *actVII* locus | This work |
| pANT56 | pIJ941 with 2.8-kbp *XhoI* subclone from pANT43; contains *actI* and *actVII* loci | This work |

$^a$ Abbreviations: ATCC, American Type Culture Collection; FCRC, Frederick Cancer Research Center; LC, low-copy-number plasmid; HC, high-copy-number plasmid; Thio$^r$, thiostrepton resistance; Hyg$^r$, hygromycin resistance; Neo$^r$, neomycin resistance; Mel$^+$, production of melanin; *act*, actinorhodin genetic locus.

required for the formation of antibiotics which are lacking a hydroxyl group at the ninth carbon from the carboxy terminus of the assembled (intermediate) polyketide chain. Our data are consistent with the theory that the *actIII* gene, and similar genes of other polyketide-producing strains, encodes a polyketide ketoreductase (15). These concepts were used to predict and demonstrate the formation of the 3-hydroxy analog of aloesaponarin II, i.e., desoxyerythrolaccin, by a polyketide reductase-negative strain, *S. galilaeus* ATCC 31671, transformed with the *actI*, *actVII*, and *actIV* genetic loci. The biosynthesis of aloesaponarin II (this work) and mutactin (38) by blocked mutants of *S. coelicolor*, in conjunction with the compounds produced by the recombinant strains reported in this work, suggests a pathway for the biosynthesis of actinorhodin.

**MATERIALS AND METHODS**

Organisms and plasmids. The organisms used in this study and the sources from which they were obtained are listed in Table 1. Plasmids obtained for use or developed in this study also are listed in Table 1. Plasmid pIJ2303 (23, 24), which contains all of the actinorhodin (*act*) biosynthesis genes, was obtained from D. A. Hopwood (John Innes Institute, Norwich, United Kingdom).

Culture maintenance and growth conditions. The streptomycetes were maintained on yeast malt agar as described previously (9). Strains carrying streptomycete plasmids pIJ61 (34), pIJ350 (19), pIJ922 (21), and pIJ941 (21) or derivatives of these plasmids were grown and stored on plates containing 40 μg of thiostrepton per ml. Plates were incubated at 30°C until sporulation occurred and were stored at 4°C. Nitrate-defined medium (NDM), used in $^{14}$C- and $^{13}$C-labeling experiments, and GPS fermentation medium have been described previously (8).

Genetic manipulations. Procedures for protoplast formation, transformation, and regeneration of protoplasts for the anthracycline-producing Streptomyces strains and for preparation of chromosomal DNA have been described in detail elsewhere (20). Streptomyces plasmids were isolated by the methods described by Kieser (18). Standard procedures used for manipulating anthracycline-nonproducing streptomycetes are detailed by Hopwood et al. (16).

For the construction of pANT12, pIJ2303 was completely digested with *PstI*, treated with calf intestinal alkaline phosphatase (Boehringer Mannheim Biochemicals, Indianapolis, Ind.), ligated with *PstI*-digested pIJ350 by using T4 DNA ligase (20), and introduced by transformation into *Streptomyces galilaeus* ATCC 31133 and *Streptomyces peucetius*

ATCC 29050 by methods described previously (20). Other subclones were constructed from pANT12 by similar procedures.

DNA-DNA hybridizations. For hybridization analyses, samples of chromosomal DNA, purified from *S. galilaeus* 31133, *S. galilaeus* 31671, and *S. coelicolor* A3(2) as described previously (20), were digested with BamHI for 12 h, using 2 U of enzyme per μg of DNA. Plasmid pANT45, used as a probe for the *actIII* gene, was labeled by the <sup>32</sup>P-oligolabeling procedure described by Feinberg and Vogelstein (12), using 50 μCi of [α-<sup>32</sup>P]dCTP per μg of DNA. Southern blots and hybridizations were performed as described by Maniatis et al. (25).

Detection of polyketides. Strains to be tested for the production of new compounds were grown in 5 ml of GPS medium in test tubes (16 by 150 mm) for 7 days with rotary shaking (ca. 250 rpm; 28°C) while in a slanted position. Plasmid-bearing strains were grown in the presence of 10 μg of thiostrepton per ml. Alternatively, mycelia from a plate culture were inoculated by loop into 50 ml of either GPS medium or NDM supplemented with thiostrepton (10 μg/ml) in a 250-ml Erlenmeyer flask containing a coiled spring (9). These cultures were grown at 28°C on a rotary shaker (250 rpm) for 3 to 7 days. After incubation, the broth was extracted with an equal volume of chloroform-methanol (9:1). The extract was dried under a stream of air and dissolved in either chloroform or acetone. Concentrated extracts were spotted onto 0.25-mm silica gel thin-layer chromatography (TLC) plates (Whatman, Inc.), which were developed with either solvent system 1 (SS1), containing benzene, acetone, and methanol (100:10:1), solvent system 2 (SS2), which consisted of chloroform, heptane, and methanol (5:5:1), or solvent system 3 (cyclohexane-ethyl acetate [1:1]). The compounds on the TLC plates were visualized by their normal pigmentation and by their fluorescence under long-wave UV irradiation at 365 nm. Anthraquinones and anthracyclinones also were separated and identified by high-performance liquid chromatography (HPLC), using a solvent system of 65% methanol, 30% water, and 5% glacial acetic acid (final pH, 2.5). The compounds were detected at 254 nm with an on-line Hitachi 100-40 spectrophotometer equipped with an Altex flow cell and quantitated by using a Hewlett-Packard HP3396A integrator.

Purification of anthraquinones and anthracyclinones. For purification of large quantities of polyketide metabolites, the streptomycetes were grown in GPS complex fermentation medium (8) in a 14-liter New Brunswick Microferm fermentor. The following conditions were used: working volume, 10 liters; temperature, 28°C; agitation, 400 rpm; aeration, 8 liters of air per min; pH, initially 7.5 but not controlled. The fermentors were inoculated with 500 ml of 2-day-old cultures grown in yeast malt broth. After 5 days of growth, the entire fermentation culture broths were extracted twice with equal volumes of ethyl acetate (or, alternatively, with chloroform-methanol [9:1]). For each purification, the organic layer was collected and concentrated in vacuo. The concentrated sample was defatted with 100 ml of hexane, after which the upper (hexane-lipid) layer was discarded. The lower layer, which was dried and redissolved in a small volume of methanol, was loaded onto a Sephadex LH-20 (Pharmacia, Inc., Piscataway, N.J.) column (2.5 by 45 cm) and eluted with either methanol or methanol-methylene chloride (80:20). Fractions containing the compounds of interest (as determined by TLC) were pooled and dried in vacuo. The dried samples were dissolved in acetone, and concentrated samples were applied to a Silica Gel G column (2.5 by 30 cm), which was developed with chloroform-heptane-methanol (5:5:2) as the solvent. Fractions containing the compounds of interest (as determined by TLC) were pooled and dried in vacuo, and the dried samples were dissolved in acetone and streaked onto preparative TLC plates (silica gel, 2 mm thick; E. Merck AG, Darmstadt, Federal Republic of Germany). The plates were developed by using SS1, the appropriate bands were scraped off, and the compounds were eluted with acetone. These samples were dried in vacuo and dissolved in a small volume of acetone; if required, the samples were further purified by preparative TLC, using SS1 or SS2 as required.

[1,2-<sup>13</sup>C<sub>2</sub>]acetate feeding experiments. Cultures of *S. galilaeus*(pANT12) were grown in 50 ml of NDM in a 250-ml flask containing a stainless-steel spring to disperse the mycelia (9). After 2 days of growth, 20 ml of the culture was used to inoculate each of two 2-liter flasks containing 500 ml of NDM. The flasks (containing stainless-steel springs) were incubated with shaking (ca. 250 rpm) at 30°C. Sodium acetate (total of 0.5 g), at a ratio of [1,2-<sup>13</sup>C<sub>2</sub>]acetate (99 atoms% <sup>13</sup>C)/nonlabeled sodium acetate of 1:2, was added to the culture in two equal portions after 72 and 96 h of incubation. After 120 h of incubation, the mycelia were harvested by centrifugation at 10,000 × g for 10 min, and the culture broth supernatant was decanted and extracted with an equal volume of chloroform-methanol (9:1). The mycelial pellet was extracted once with 100 ml of hexane to remove neutral lipids. The organic layer was discarded, and the pellet was extracted twice with 100 ml of acetone. The two acetone extracts of the mycelial pellet were combined and concentrated in vacuo, followed by extraction twice with equal volumes of chloroform-methanol (9:1). These mycelial extracts were combined with the culture broth extract, and the entire extract was concentrated in vacuo. Aloesaponarin II was purified from these extracts as described above.

Analysis by NMR spectroscopy. Samples for nuclear magnetic resonance (NMR) spectroscopy were dissolved in 0.4 ml of DMSO-*d*<sub>6</sub> (99 + atoms% <sup>2</sup>H). NMR experiments were performed at a field of 7.1 T on an IBM AF-300 spectrometer employing an ASPECT-3000 data system. <sup>13</sup>C NMR spectra were acquired with continuous composite pulse proton decoupling. Nuclear Overhauser effect (NOE) studies were conducted on samples of aloesaponarin II by a one-dimensional steady-state difference method.

Chemicals. Sodium [1,2-<sup>13</sup>C<sub>2</sub>]acetate (99 atoms% <sup>13</sup>C) was obtained from Cambridge Isotope Laboratories, Woburn, Mass. Sodium [1,2-<sup>14</sup>C]acetate (55 mCi/mmol) was obtained from Dupont, NEN Research Products, Boston, Mass. Authentic standards for anthracyclines were obtained from Adria, Inc., the Frederick Cancer Research Center, and Rhône-Poulenc. Restriction endonucleases were purchased from Bethesda Research Laboratories, Inc., Gaithersburg, Md., and deuterated solvents for NMR analysis were obtained from Sigma Chemical Co., St. Louis, Mo.

---

**RESULTS**

Production of aloesaponarin II by streptomycetes transformed with actinorhodin biosynthesis genes. Subclones of pIJ2303, a plasmid containing all of the actinorhodin biosynthesis genes (23), were generated to transform anthracycline-producing strains for the purpose of searching for new hybrid polyketide structures. An 8.8-kilobase-pair (kbp) *PstI* fragment of DNA from pIJ2303, containing the *actI*, *actIII*, *actIV*, and *actVII* loci (Fig. 1), was subcloned into the high-copy-number vector pIJ350 to yield plasmid pANT12.

Vol. 172, 1990                                                                                     HYBRID ANTHRAQUINONE BIOSYNTHESIS 4819

![Diagram]

FIG. 1. Structure of part of the *act* genetic loci and restriction map of the actinorhodin biosynthesis genes. The restriction site numbers 12, 13, 14, 17, 18, and 20 correspond to the numbers given those sites by Malpartida and Hopwood (24). Sites 12a, 13a, 13b, 14a, 17a, 18a, and 18b were determined in this work. The restriction fragments which conferred aloesaponarin II production on *S.* galilaeus ATCC 31133 are identified. Abbreviations for restriction endonuclease sites: P, *PstI*; Sp, *SphI*; S, *SstI*; X, *XhoI*; B, *BamHI*.

This plasmid was introduced by transformation into *S.* galilaeus ATCC 31133, a strain that normally produces aclacinomycin A (28). When transformed with pANT12, however, this strain formed significant quantities of a new yellow compound, which was distinct from anthracyclines produced by *S.* galilaeus by its migration on TLC, elution by HPLC, and unusual orange fluorescence at 365 nm, a property not shared by the anthracyclines (Table 2). The yellow compound was purified from cultures of *S.* galilaeus 31133(pANT12) and identified by its ${}^{13}$C and ${}^{1}$H NMR spectra (Table 3) and by comparison with an authentic reference sample (35) to be 1,7-dihydroxy-9-methylanthraquinone, also known as aloesaponarin II (35; Fig. 2A). The positioning of the methyl group on the anthraquinone ring system was confirmed by proton NOE experiments.

Control experiments and subcloning of antinorhodin genes. When the 8.8-kbp *PstI* DNA fragment from pANT12 was subcloned in the low-copy-number vectors pIJ941 and pIJ61 to construct pANT28 and pANT31, respectively, and introduced by transformation into *S.* galilaeus 31133, aloesaponarin II was the major product formed by the recombinant cultures (Table 3). The amount of aloesaponarin II produced by *S.* galilaeus cultures grown in GPS medium was inversely related to the copy number of the plasmids carrying the *act* genes; *S.* galilaeus 31133 carrying pANT12 (pIJ101 replicon,

Sti${}^{-}$; reported copy number, 50 to 100 [10]), pANT31 (SLP1 replicon; reported copy number, 4 to 5 [2]), and pANT28 (SCP2* replicon; reported copy number, 1 to 2 [21]) yielded 1.1, 5.7, and 10.6 $\mu$g of aloesaponarin II per ml of whole culture broth (mycelia plus filtrate), respectively. Most of the aloesaponarin II produced by the recombinant *S.* galilaeus 31133 cultures was in the mycelial fraction; very little aloesaponarin II was found extracellularly.

*S.* galilaeus 31133 transformed with pIJ702, pIJ941, or pIJ61 did not form aloesaponarin II, indicating that its production was due to the presence of the antinorhodin biosynthesis-encoding DNA and not to the presence of the plasmids. *S.* peucetius, another anthracycline-producing streptomycete (9, 33), and the anthracycline-nonproducing streptomycetes *Streptomyces azureus* and *Streptomyces parvulus* also produced aloesaponarin II when transformed with plasmids containing the *actI*, *actIII*, *actVII*, and *actIV* loci (Table 4).

Several subclones of the *act* DNA fragment in pANT12 were prepared to determine which of the antinorhodin biosynthesis genes were needed to confer aloesaponarin II biosynthesis on *S.* galilaeus 31133. To construct subclones of the actinorhodin DNA, we determined a more detailed restriction map of the *act* DNA on pANT12. The restriction sites 12a, 13a, 13b, 14a, 17a, 18a, and 18b are sites which we

TABLE 2. Chromatographic analysis of anthraquinones and anthracyclinones produced by nonrecombinant and recombinant cultures

| Compound                      | $R_{f}$ in ${}^{b}$: | Pigment     | Fluorescence ${}^{c}$ | HPLC retention time (min) ${}^{a}$ |
|-------------------------------|---------------------|-------------|------------------------|------------------------------------|
|                               | SS1                 | SS2         | SS3                   |                                    |
| Aklavinone                    | 0.30                | 0.35        | 0.42                  | Yellow                             | 8.9                              |
| 2-Hydroxyaklavinone           | 0.09                | 0.25        | 0.11                  | Yellow                             | 14.8                             |
| Aloesaponarin II              | 0.37                | 0.33        | 0.52                  | Yellow                             | 14.9                             |
| Desoxyerythrolaccin           | 0.11                | 0.19        | 0.32                  | Red                                | 11.5                             |
| 1-O-methyldesoxyerythrolaccin | 0.37                | 0.30        | 0.52                  | Red                                | 20.1                             |

${}^{a}$ Using a solvent system of 65% methanol–30% water–5% acetic acid (pH 2.5) and a Whatman C${}_{18}$ reverse-phase column.

${}^{b}$ Solvent systems SS1 to SS3 are defined in Materials and Methods.

${}^{c}$ Color emitted when viewed under UV light at 365 nm.

TABLE 3. $^{13}$C and $^{1}$H NMR data on the structure and carbon-carbon coupling patterns for aloesaponarin II$^a$

| Carbon no.$^b$ | $^{13}$C | $^{1}$H |
| --- | --- | --- |
|  | Chemical shift (ppm)$^c$ | Multiplicity | $J_{CC}$ (Hz)$^d$ | Chemical shift (ppm) | Multiplicity | $J_{HH}$ (Hz) |
| 9' | 23.4 | q | 41.4 | 2.65 | s | – |
| 9 | 145.2 | s | 41.4 | – | – | – |
| 12 | 116.4 | s | 61.0 | – | – | – |
| 10 | 189.2 | s | 61.0 | – | – | – |
| 13 | 122.2 | s | 60.3 | – | – | – |
| 1 | 162.4 | s | 60.3 | – | – | – |
| 2 | 124.1 | d | 59.0 | 7.35 | dd | 8.0, 1.3 |
| 3 | 135.8 | d | 59.0 | 7.70 | t | 8.0 |
| 4 | 118.1 | d | 62.1 | 7.65 | dd | 8.0, 1.3 |
| 14 | 132.5 | s | 62.2 | – | – | – |
| 5 | 182.1 | s | 52.0 | – | – | – |
| 11 | 136.8 | s | 52.1 | – | – | – |
| 6 | 112.0 | d | 62.2 | 7.48 | d | 2.6 |
| 7 | 161.4 | s | 62.1 | – | – | – |
| 8 | 124.5 | d | NC | 7.06 | d | 2.6 |

$a$ Abbreviations and symbol: s, singlet; d, doublet; q, quartet; dd, doublet of doublets; $J_{CC}$, coupling constant of $^{13}$C (determined after incorporation of sodium [1,2-$^{13}$C$_2$]acetate); $J_{HH}$, coupling constant of $^{1}$H; NC, not coupled; –, no signal acquired.

$b$ See Fig. 2A.

$c$ Chemical shifts for both $^{13}$C and $^{1}$H were referenced internally to solvent resonance.

$d$ Determined by edited DEPT (distortion enhancement by polarization transfer) analysis.

---

TABLE 4. Formation of polyketides by nonrecombinant and recombinant strains

| Strain | Plasmid | Major polyketide compound(s) formed |
| --- | --- | --- |
| S. galilaeus 31133 | None | Aclacinomycin A |
| S. galilaeus 31133 | pIJ702 | Aclacinomycin A |
| S. galilaeus 31133 | pIJ61 | Aclacinomycin A |
| S. galilaeus 31133 | pIJ941 | Aclacinomycin A |
| S. galilaeus 31133 | pANT12 | Aloesaponarin II |
| S. galilaeus 31133 | pANT28 | Aloesaponarin II |
| S. galilaeus 31133 | pANT31 | Aloesaponarin II |
| S. galilaeus 31133 | pANT33 | Aloesaponarin II |
| S. galilaeus 31133 | pANT35 | Aloesaponarin II |
| S. galilaeus 31133 | pANT43 | Aloesaponarin II |
| S. galilaeus 31133 | pANT44 | Aclacinomycin A |
| S. galilaeus 31133 | pANT45 | Aclacinomycin A |
| S. galilaeus 31133 | pANT46 | Aclacinomycin A |
| S. galilaeus 31133 | pANT56 | Aloesaponarin II |
| S. galilaeus 31671 | None | 2-Hydroxyaklavinone |
| S. galilaeus 31671 | pANT12 | Aloesaponarin II |
| S. galilaeus 31671 | pANT28 | Aloesaponarin II |
| S. galilaeus 31671 | pANT31 | Aloesaponarin II |
| S. galilaeus 31671 | pANT33 | Aloesaponarin II; aklavinone |
| S. galilaeus 31671 | pANT35 | Desoxyerythrolaccin; 1-O-methyldeoxyerythrolaccin |
| S. galilaeus 31671 | pANT43 | 1-O-Methyldeoxyerythrolaccin |
| S. galilaeus 31671 | pANT44 | 2-Hydroxyaklavinone |
| S. galilaeus 31671 | pANT45 | Aklavinone |
| S. galilaeus 31671 | pANT46 | 2-Hydroxyaklavinone |
| S. peucetius 29050 | None | Daunomycin |
| S. peucetius 29050 | pANT12 | Aloesaponarin II |
| S. parvulus 2266 | None | NK$^a$ |
| S. parvulus 2266 | pANT12 | Aloesaponarin II |
| S. azureus 14921 | None | NK |
| S. azureus 14921 | pANT28 | Aloesaponarin II |
| S. azureus 14921 | pANT56 | NK |

$a$ NK, None known to be produced.

---

FIG. 2. Structures of compounds described in the text. (A) Structure of aloesaponarin II with numbered carbons, and its $^{13}$C enrichment and $^{13}$C–$^{13}$C coupling pattern after feeding with sodium [1,2-$^{13}$C$_2$]acetate. Symbols: ●, enriched carbon showing no $^{13}$C–$^{13}$C coupling; ——, pairs of enriched, $^{13}$C–$^{13}$C coupled carbons. See Table 2 for NMR data. (B) Structures of desoxyerythrolaccin (R = OH) and 1-O-methyldeoxyerythrolaccin (R = OCH$_3$) with numbered carbons. See Table 5 for NMR data. (C) Structure of actinorhodin. (D) Structures of the anthracyclones aklavinone (R = H) and 2-hydroxyaklavinone (R = OH).

**A.**
![Structure of aloesaponarin II](chemical_structure_a.png)

ALOESAPONARIN II

**B.**
![Structures of desoxyerythrolaccin and 1-O-methyldeoxyerythrolaccin](chemical_structure_b.png)

DESOXYERYTHROLACCIN  
1-O-METHYLDESOXY-  
ERYTHROLACCIN  

**C.**
![Structure of actinorhodin](chemical_structure_c.png)

ACTINORHODIN

**D.**
![Structures of aklavinone and 2-hydroxyaklavinone](chemical_structure_d.png)

AKLAVINONE  
2-HYDROXYAKLAVINONE

( *actVI* mutants) each produced a compound with the same \( R_f \) value, the same pigmentation, and the same fluorescence characteristics as aloesaponarin II (Table 2). Strain B22 also produced trace amounts of two other yellow compounds (\( R_f \) values of 0.60 and 0.70 in SS2; P. L. Bartel, unpublished data). The major pigmented product of B22 culture extracts was purified and determined by \({}^{13}\mathrm{C}\) and \({}^{1}\mathrm{H}\) NMR to be aloesaponarin II (data not shown). None of the other *S.* coelicolor blocked mutants tested, including B17, B18, B40, and B41 (described in Table 1), produced detectable levels of aloesaponarin II.

**Labeling with [1,2- \({}^{13}\mathrm{C}_2\) ]acetate.** To determine the pattern of acetate incorporation into aloesaponarin II, sodium [1,2- \({}^{13}\mathrm{C}_2\) ]acetate was administered to a culture of *S.* galilaeus 31133(pANT12), and the product was purified. The carbon-carbon spin-coupling pattern of the labeled aloesaponarin II demonstrated that all carbon atoms except C-2 were coupled to one neighboring carbon (Fig. 2A; Table 3). This finding indicates that aloesaponarin II was formed by condensation of 8 acetyl-CoA equivalents, followed by a decarboxylation which abolished coupling at C-2 by scission of the carbon-carbon bond (Fig. 2A).

**Formation of natural and hybrid polyketides by recombinant *S.* galilaeus 31671.** Mutants of *S.* galilaeus that produce 2-hydroxy derivatives of aklavinone-related anthracyclines have been isolated and characterized (26). One of these strains was deposited in the American Type Culture Collection (Rockville, Md.) as *S.* galilaeus ATCC 31671, a strain that produces 2-hydroxyaklavinone (see Fig. 2D for structure) as its major product (26). *S.* galilaeus 31671(pANT12) produced aloesaponarin II as its major product (Table 4). However, when *S.* galilaeus 31671 was transformed with pANT33, a plasmid that contains the *actI*, *actIII*, and *actVII* loci on a 4.7-kbp *PstI-SstI* DNA fragment (Fig. 1), both aloesaponarin II and aklavinone were formed (Table 4). The latter was identified by comparison with authentic aklavinone (Table 2), obtained by acid hydrolysis (0.1 N HCl, 60 min) of aclacinomycin A and purified by preparative TLC using SS1. Because *S.* galilaeus 31671, which normally produces small quantities of 2-hydroxyaklavinone, formed aklavinone as a result of being transformed with pANT33, a function encoded by pANT33 was apparently responsible for the deoxygenation at C-2 (Fig. 2D). When pANT35, a subclone of pANT12 containing a 4.3-kbp *SphI* DNA fragment with the *actI*, *actVII*, and *actIV* loci, was introduced into *S.* galilaeus 31671, at least three new compounds were produced. The major compound was identified by \({}^{13}\mathrm{C}\) and \({}^{1}\mathrm{H}\) NMR (Table 5) and comparison with authentic material as 1,3,7-trihydroxy-9-methylanthraquinone, also known as desoxyerythrolaccin (27, 35) (Fig. 2B). A second abundantly produced compound was identified by \({}^{13}\mathrm{C}\) and \({}^{1}\mathrm{H}\) NMR (data not shown) as 1-*O*-methyldeoxyerythrolaccin (Fig. 2B; Table 4). The position of the *O*-methyl group was confirmed by proton NOE experiments. After several purification steps, this compound was still slightly contaminated with a second *O*-methylated compound, either 3-*O*-methyl or 7-*O*-methyldeoxyerythrolaccin, which usually comigrated with the 1-*O*-methyl derivative. The position of the *O*-methyl group of the contaminant, however, has not been established unambiguously. When *S.* galilaeus 31671 was transformed with pANT43, which contains the *actI* and *actVII* loci (Table 1), only 1-*O*-methyldeoxyerythrolaccin (and its associated contaminant) was produced. Wild-type *S.* galilaeus 31133(pANT43), on the other hand, produced the 3-desoxy derivative of desoxyerythrolaccin, aloesaponarin II. These data demonstrate that the *actIII* gene function, not

---

**TABLE 5.** \({}^{13}\mathrm{C}\) and \({}^{1}\mathrm{H}\) NMR data on the structure of desoxyerythrolaccin isolated from *S.* galilaeus 31671(pANT35)\(^a\)

| Carbon no.\(^b\) | \({}^{13}\mathrm{C}\) Chemical shift\(^c\) (ppM) | Multiplicity\(^d\) | \({}^{1}\mathrm{H}\) Chemical shift (ppm) | Multiplicity | \(J_{\mathrm{HH}}\) (Hz) |
|------------------|----------------------------------------|-------------------|------------------------------------|--------------|---------------------|
| 1                | 163.9                                 | s                 | –                                  | –            | –                   |
| 2                | 107.0                                 | d                 | 6.52                               | d            | 2.90                |
| 3                | 164.3                                 | s                 | –                                  | –            | –                   |
| 4                | 108.1                                 | d                 | 7.19                               | d            | 2.90                |
| 5                | 182.5                                 | s                 | –                                  | –            | –                   |
| 6                | 112.3                                 | d                 | 7.45                               | d            | 2.65                |
| 7                | 161.6                                 | s                 | –                                  | –            | –                   |
| 8                | 124.7                                 | d                 | 7.15                               | d            | 2.65                |
| 9                | 144.8                                 | s                 | –                                  | –            | –                   |
| 9'               | 23.3                                  | q                 | 2.70                               | s            | –                   |
| 10               | 187.2                                 | s                 | –                                  | –            | –                   |
| 11               | 136.7                                 | s                 | –                                  | –            | –                   |
| 12               | 122.2                                 | s                 | –                                  | –            | –                   |
| 13               | 109.7                                 | s                 | –                                  | –            | –                   |
| 14               | 134.2                                 | s                 | –                                  | –            | –                   |

\({}^a\) Abbreviations and symbol: s, singlet; d, doublet; q, quartet; \(J_{\mathrm{HH}}\), \({}^{1}\mathrm{H}-{ }^{1}\mathrm{H}\) coupling constant; –, no signal acquired.

\({}^b\) See Fig. 2B.

\({}^c\) Chemical shifts for both \({}^{13}\mathrm{C}\) and \({}^{1}\mathrm{H}\) were referenced internally to solvent resonance.

\({}^d\) Determined by edited DEPT analysis.

required for aloesaponarin II formation by *S.* galilaeus 31133, is required for aloesaponarin II biosynthesis by *S.* galilaeus 31671. This finding indicates that *S.* galilaeus 31133, a strain which synthesizes aklavinone, contains an *actIII*-equivalent gene function that is lacking in the 2-hydroxyaklavinone-producing strain *S.* galilaeus 31671. To test this hypothesis, we constructed plasmid pANT45, which contains the 1,035-bp *BamHI* fragment carrying the intact *actIII* gene (15) in the low-copy-number streptomycete vector pIJ61 (34). When *S.* galilaeus 31671 was transformed with pANT45, aklavinone was the only product formed (Table 4). The structures of 2-hydroxyaklavinone produced by *S.* galilaeus 31671 and aklavinone produced by the same strain carrying pANT45 were confirmed by purifying the chloroform-methanol (9:1)-soluble products as described in Materials and Methods and analyzing them by TLC, HPLC (Table 2), and \({}^{1}\mathrm{H}\) NMR (data not shown). Thus, the *actIII* gene from *S.* coelicolor, responsible for a discrete biosynthetic step in the actinorhodin biosynthesis pathway (15), is able to complement a missing function of *S.* galilaeus 31671 to confer aklavinone formation.

In control experiments, cultures of *S.* galilaeus 31671 transformed with pIJ702, pIJ61, pIJ941, pANT44 (containing a 2.2-kbp *BamHI* fragment with part of the *actI* genetic locus), or pANT46 (containing a 0.8-kbp *BamHI* fragment with part of the *actVII* locus) produced 2-hydroxyaklavinone as their major product. In none of these cases was aklavinone, desoxyerythrolaccin, 1-*O*-methyldeoxyerythrolaccin, or aloesaponarin II produced.

Southern hybridizations to identify *actIII*-homologous sequences. Streptomyces sp. strain C5 and *S.* peucetius ATCC 29050, strains which produce daunomycin and similar anthracyclines that lack an oxygen function at C-2, contain an *actIII*-hybridizing DNA fragment (22, 33; J. L. Lampel and W. R. Strohl, unpublished data). On the other hand, *Streptomyces glaucescens* ETH 22794, a strain which produces tetracenomycin, an anthracycline that is apparently produced without such a reduction step (22, 37), does not

conferred by the activity of the gene products encoded by  
the *actI*, *actIII*, *actIV*, and *actVII* loci in the absence of an  
active *actVI* gene product, whether these are present and  
active in *S*. *coelicolor* (as in the case of the *actVI* blocked  
mutants B22 and B159) or cloned into a heterologous recipient (e.g., *S*. *galilaeus*, *S*. *peucetius*, *S*. *azureus*, or *S*.  
*parvulus*).

Minimal genetic requirements for aloesaponarin II biosynthesis. The minimal fragment of DNA encoding actinorhodin  
biosynthesis which confers aloesaponarin II production on  
*S*. *galilaeus* 31133, the 2.8-kbp *XhoI* fragment (pANT43),  
appears to contain the *actI* and *actVII* loci (24). The *XhoI*  
site is 50 bp upstream from the *BamHI* site (15), the latter of  
which appears to interrupt *actI* gene function (24). The *actI*  
locus has been reported to be transcribed on a polycistronic  
message that begins somewhere upstream of the *BamHI* site  
14 (Fig. 1) and continues through the *actVII* and *actIV*  
genetic loci (24). This suggests that the 50-bp segment  
between *XhoI* site 13b and *BamHI* site 14 probably contains  
the promoter region required for this polycistronic transcript.

The data showing that pANT43, a plasmid carrying only  
the *actI* and *actVII* loci, conferred aloesaponarin II formation on *S*. *galilaeus* 31133 indicate either that the functions  
encoded by the *actIII* and *actIV* loci are not required for  
aloesaponarin II production or that these functions are  
carried out by *S*. *galilaeus* 31133 gene products. Since  
pANT28 (containing *actI*, *actIII*, *actVII*, and *actIV* loci)  
confers aloesaponarin II production on *S*. *azureus* (a strain  
not known to produce polyketides) but pANT56 (containing  
just the *actI* and *actVII* loci) does not, it appears that the  
*actIII* and (or) *actIV* loci, or their equivalents, are required  
for aloesaponarin II biosynthesis. Since the *actIII* and *actIV*  
loci are not present in pANT43 and pANT56, plasmids  
which confer aloesaponarin II formation on *S*. *galilaeus*  
31133, these functions appear to be carried out by *S*.  
*galilaeus* gene products. When the *actI*, *actVII*, and *actIV*  
genes were cloned into an *S*. *galilaeus* strain lacking a  
functional *actIII*-like gene (strain ATCC 31671), a different  
product was formed, further indicating that the *S*. *galilaeus*  
31133 *actIII*-like gene was cross-functional.

Pathway for aloesaponarin II and actinorhodin biosynthesis. Based on the precursors required for formation of  
actinorhodin (2 × C₁₆; 2 × 8 acetyl-CoA equivalents [14]) and  
anthracyclines (C₂₁; 9 acetyl-CoA equivalents condensed  
onto a propionyl-CoA starter unit [6]) (Fig. 4), two routes are  
possible for the formation of aloesaponarin II (C₁₅): (i)  
propionyl-CoA might be used as a starter unit with the  
condensation of 6 acetyl-CoA equivalents, or (ii) an acetyl-  
CoA starter unit might be used, onto which 7 acetyl-CoA  
equivalents would be condensed, followed by a decarboxylation. The carbon-carbon coupling patterns determined by  
¹³C NMR showed that aloesaponarin II is formed by the  
condensation of 8 acetyl-CoA equivalents, followed by a  
decarboxylation at the carboxy-terminal end of the poly-  
ketide (Fig. 2A and 5). Thus, the aloesaponarin II precursor  
polyketide was not synthesized with use of a propionyl-CoA  
starter unit, consistent with formation of the aloesaponarin  
II polyketide in *S*. *galilaeus* 31133(pANT12) being catalyzed  
primarily by the *act* gene products.

The major differences between aloesaponarin II and actinorhodin monomer are the folding pattern in the third ring and the loss of one carbon by decarboxylation in the formation of aloesaponarin II (Fig. 2A, 2C, 4, and 5). Since aloesaponarin II is a shunt product in *S*. *coelicolor* B22 and also is a product of *S*. *azureus*(pANT28), it is apparent that

FIG. 3. Southern blots showing hybridization of the *actIII* gene  
(plasmid pANT45) to DNA isolated from *S*. *coelicolor* and from *S*.  
*galilaeus* 31133 and 31671. Lanes: 1, *BamHI*-digested DNA from *S*.  
*coelicolor* showing a 1,035-bp fragment (15); 2 to 4, *S*. *galilaeus*  
31133 DNA digested with *BamHI*, *PstI*, and *SstI*, respectively; 5 to  
7, *S*. *galilaeus* 31671 DNA digested with *BamHI*, *PstI*, and *SstI*,  
respectively. The hybridizations were carried out at 42°C using 50%  
formamide, with a single wash at 55°C for 1 h using 0.2× SSC buffer.

contain DNA fragments that hybridize to the *actIII* gene (22). Our data indicate that *S*. *galilaeus* 31133 possesses an  
active *actIII*-equivalent function but that *S*. *galilaeus* 31671,  
a derivative of 31133, does not. We probed *BamHI*, *PstI*,  
and *SstI* restriction endonuclease-digested DNA samples  
from cultures of each of these strains to determine whether  
there were any differences in the presence of potential  
*actIII*-hybridizing sequences. The hybridization data show  
that both *S*. *galilaeus* 31133 and 31671 contain sequences of  
the same size and restriction digestion patterns that are  
homologous to *actIII* from *S*. *coelicolor* (Fig. 3). This finding  
confirms that *S*. *galilaeus* 31133 possesses a gene homologous to the *actIII* gene, as suggested by the complementation data. Furthermore, the hybridizations suggest that the mutation of the *actIII*-homologous gene of *S*. *galilaeus* 31671, made by using *N*-nitro-*N'*-nitrosoguanidine (26), is probably a point mutation (or a series of point mutations).

DISCUSSION

Production of aloesaponarin II. In our attempts to generate  
hybrid antibiotics by transforming discrete restriction fragments of the actinorhodin biosynthesis gene cluster into  
various streptomycetes, we found that an anthraquinone,  
aloesaponarin II (35), was produced by recombinant streptomycetes that normally do not produce that compound. Aloe-saponarin II was originally isolated from aloe plants (35), and to our knowledge it has never before been detected as a bacterial product.

The formation of aloesaponarin II by streptomycetes transformed with DNA encoding actinorhodin biosynthesis was correlated with the presence of the *actI*, *actIII*, *actVII*, and *actIV* loci in certain strains. Strains B22 and B159, *actVI* mutants of *S*. *coelicolor* (30), also were found to produce aloesaponarin II as an apparent shunt product of actinorhodin biosynthesis. This concurs with the order of the *act* mutants delineated by cross-feeding and cosynthesis studies (30), which suggests that the *actVI* mutants should form products which are the result of the functions of the *actI*, *actIII*, *actIV*, and *actVII* loci, the same genetic loci present in plasmids pANT12, pANT28, and pANT31 (Table 1). Thus, our data indicate that aloesaponarin II production is

Number of
Starter Units Extender Units

A. Propionyl-CoA 9

B. Acetyl-CoA 6

C. Acetyl-CoA 6

FIG. 4. Structures and theoretical polyketide precursors of the polyketides aklavinone (A), actinorhodin (B), and aloesaponarin II (C), formed as a result of cloning *act* genes in *S*. galilaeus 31133 and 31671. The starter units and number of extending acetyl units (from malonyl-CoA) are identified for each compound. The proposed activity of the *actIII* and *actIII*-homologous genes is shown in relation to the products formed. If the *actIII* function, polyketide reductase, is present, then the keto group at the ninth carbon from the carboxyl end of the polyketide chain in each case is reduced. Upon ring closure and aromatization, a hydroxyl group is lost from that carbon and the resultant molecule is deoxygenated in that position. If the *actIII* function is not present, then the resultant molecules contain hydroxyl groups in the respective positions (marked with dots). (A) 2-Hydroxyaklavinone (A) and aklavinone (B) biosynthesis; (B) biosynthesis of the postulated hydroxyactinorhodin monomer molecule (C; which has not been found) and actinorhodin monomer (D); (C) biosynthesis of desoxyerythro-laccin (E) and its 3-deoxy analog, aloesaponarin II (F).

The unusual folding pattern is not solely caused by activities associated with anthracycline biosynthesis. We have demonstrated that the polyketide precursor of actinorhodin can undergo an entirely different type of cyclization in *S*. coelicolor to produce mutactin, representing a novel carbon skeleton (38). Mutactin is formed in traces by the wild-type strain, but its production is enhanced 100-fold in the *actVII* mutant B40 (38).

By combining the data obtained on the biosynthesis of aloesaponarin II, formed by the *actI*, *actIII*, *actVII*, and *actIV* loci (Fig. 1; Table 4), and mutactin (38), requiring only the *actI* and *actIII* loci from the actinorhodin pathway, we have developed a hypothetical route for the formation of actinorhodin, aloesaponarin II, and mutactin (Fig. 5). We propose that aloesaponarin II is formed in cultures of *S*. galilaeus 31133(pANT12) and *S*. coelicolor B22 and B159 by the following sequence of events. (i) A C₁₆ polyketide is formed by the action of the actinorhodin polyketide synthase complex (*actI* gene products [22]), which catalyzes the sequential condensation of seven acetyl-CoA equivalents (from malonyl-CoA) onto an initial acetyl-CoA starter unit (Fig. 4 and 5). (ii) The polyketide is reduced at the ninth carbon from the carboxyl end by the action of the polyketide reductase. This enzyme is encoded either by the *actIII* gene (15), if present, or by the *actIII*-equivalent gene of the *S*. galilaeus 31133 genome, as would occur when this organism is transformed with pANT35 or pANT43. The first ring would be formed by the action of gene products encoded by the *actI* locus, perhaps as a stabilization event (11), by bond formation between C-7 and C-12 (Fig. 5). These steps are shared with the biosynthesis of actinorhodin as well as mutactin (38). (iii) The closure of the second ring, by bond formation between C-5 and C-14, and aromatization of the first ring would be carried out by a function encoded by *actVII*. This step is shared with the biosynthesis of actinorhodin, but in the *actVII* mutant *S*. coelicolor B40, this step cannot occur, allowing formation of the mutactin precursor. This occurs by activation of C-6 toward aldol condensation with C-15 (Fig. 5). (iv) The postulated bicyclic intermediate is dehydrated by the gene product(s) encoded by *actIV* to yield a compound which is the precursor to either actinorhodin or aloesaponarin II (Fig. 5). In the presence of *actVI*, this precursor would be cyclized by attack of the hydroxyl oxygen on the carbonyl group,

4824 BARTEL ET AL.

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

FIG. 5. Hypothetical pathway for the biosynthesis of mutactin (38), aloesaponarin II (this work), and actinorhodin (13, 14), based on results obtained in this paper and data from other sources (7, 13–15, 38). The probable functions of the gene products of *actI* (polyketide synthase), *actIII* (polyketide reductase), *actVII* (cyclase and/or dehydratase), and *actIV* (dehydratase) are given. In the presence of the *actVI* gene product (cyclase and/or reductase), the pathway would follow reaction A to form actinorhodin; in the absence of the *actVI* gene product, the pathway would follow reaction B to form aloesaponarin II.

followed by dehydration and reduction to give the heterocyclic actinorhodin precursor, which has been isolated from an *actVa* mutant (7) (Fig. 5, reaction A). In the absence of *actVI*, however, aldol condensation of C-2 with the carbonyl group at C-15 followed by dehydrative decarboxylation would form the precursor to aloesaponarin II (Fig. 5, reaction B), which would then undergo spontaneous air oxidation to form aloesaponarin II.

Function of the *actIII* gene in polyketide formation. Malparida et al. (22) hypothesized that the *actIII* gene encodes a polyketide reductase which reduces a keto group of the actinorhodin polyketide intermediate (Fig. 4B). After this reduction a hydroxyl group would be lost from this position during ring closure and aromatization, and the resulting compound would lack an oxygen function in that position (Fig. 4). If no reduction were to occur at C-9 of the actinorhodin precursor polyketide before ring formation and aromatization, then the corresponding compound would contain a hydroxyl group in that position (Fig. 4). Hallam et al. (15) recently determined the DNA sequence for the *actIII* gene and have found it to be similar to the sequences of the ribitol dehydrogenase gene from *Klebsiella aerogenes* and alcohol dehydrogenase gene from *Drosophila melanogaster*. Considerable data also have indicated that the *actI* locus encodes a polyketide synthase function (13, 23, 24) and that *actI* and *actIII* mutants are unable to cross-feed or cosynthesize to form actinorhodin (30). Considering that these two genes operate at the beginning of the actinorhodin biosynthesis pathway, along with the sequence data indicating that the *actIII* gene product is likely a dehydrogenase, Hallam et al. (15) proposed that *actIII* encodes a β-keto reductase which reduces the β-keto group on the forming polyketide chain once it reaches a chain length of 10 carbons. Sherman et al. (31) also have recently shown that *S. violaceoruber* Tü22, a strain that produces the benzoisochromane quinone antibiotic granaticin, contains two *actIII*-homolog genes which displayed a high sequence similarity to *actIII*.

Our data indicate that the *actIII* gene of *S. coelicolor*, as well as an analogous gene of *S. galilaeus* 31133, encodes an activity which is responsible for the absence of a hydroxyl group at C-2 of anthracyclines (Fig. 2D) and C-3 of the anthraquinone (Fig. 2A and B), aloesaponarin II. *S. galilaeus* 31671, the 2-hydroxyanthracycline producer, was derived by mutation and screening from a strain that did not produce 2-hydroxyanthracyclines (26). This finding suggests that *S. galilaeus* 31671 was missing a function or functions contained in the parent strain, *S. galilaeus* ATCC 31133. Anderson et al. (1) recently showed that cell extracts prepared from the fungus *Pyrenochaeta terrestris* could reduce a trihydroxyanthraquinone, emodin, to a dihydroxyanthraquinone, chrysophanol, by an NADPH-linked enzyme reaction. Their work, which described an aromatic counterpart to deoxygenation in polyketide biosynthesis (1), raises the possibility that the *actIII* gene, as well as the analogous gene in *S. galilaeus* 31133, might encode a phenol reductase (1) rather than one which operates on the open polyketide chain (15). We tested the possibility that *S. galilaeus* ATCC 31133 or *S. galilaeus* 31671(pANT45) might show phenol reductase activity, by reducing emodin to chrysophanol and desoxyerythrolaccin to aloesaponarin II, that would be

absent in *S. galilaeus* 31671. We never observed chrysophanol or aloesaponarin II as products when emodin or desoxyerythrolaccin, respectively, were incubated with NAD(P)H and cell extracts of *S. galilaeus* ATCC 31133, *S. galilaeus* 31671(pANT45), or *S. galilaeus* ATCC 31671 (included as a negative control; data not shown). Thus, we could not demonstrate any phenol reductase activity by *S. galilaeus* 31133 or in *S. galilaeus* 31671(pANT45) which could account for the deoxygenation at C-2 of 2-hydroxyaklavinone or at C-3 of desoxyerythrolaccin. Although these are negative data, they are consistent with the hypothesis of Hopwood and colleagues (15, 22) that the *actIII* gene product encodes a reductase that acts on the polyketide chain before ring formation. Finally, we have recently shown that cell extracts of *S. galilaeus* 31671 synthesize aklavinone from aklanonic acid, the apparent earliest stable intermediate of aklavinone biosynthesis, in the presence of NADPH (and S-adenosylmethionine), which also indicates that the deoxygenation of C-2 takes place before the formation of aklanonic acid, i.e., at the stage of an open polyketide chain (N. C. Connors, P. L. Bartel, and W. R. Strohl, J. Gen. Microbiol., in press).

Although there is some evidence showing that the reduction and dehydration steps in polyketide biosynthesis may occur at defined stages of chain assembly (5, 32, 36), the results presented in this study suggest rather that the polyketide chain is completely assembled before reduction of the keto group takes place. In the formation of both aklavinone and aloesaponarin II, the carbonyl group which is reduced by the *actIII* gene product represents the ninth carbon from the carboxy terminus of the theoretical polyketide chains (Fig. 4A and C, respectively), the same relative location as that of the proposed activity in actinorhodin biosynthesis (15; Fig. 4B). According to the theories of polyketide formation (3, 11, 32), these compounds are formed by the successive additions of acetyl groups (from malonyl-CoA) to a starter unit. Thus, in actinorhodin, aloesaponarin II, and aklavinone formation, chain growth starts at the methyl terminus and proceeds to the carboxyl terminus. The fact that the activity of the *actIII* gene product occurs at the ninth carbon from the carboxyl terminus, and not at a given distance from the methyl terminus (Fig. 4), implies that this enzyme does not reduce the carbonyl group until the entire chain is assembled. Hypothetically, then, the polyketide reductase might “measure” the distance from the carboxyl end of the completely assembled chain and reduce the proper keto group. On the other hand, not all polyketides are reduced at the ninth carbon from the carboxyl end (e.g., erythromycin and tylosin), and many polyketides are reduced at multiple sites (macrolides and polyethers). For example, the macrolide milbemycin is reduced at multiple positions, including C-9. Nevertheless, the strains that produce each of these polyketides contain *actIII* homolog genes, as evidenced by DNA-DNA hybridizations (22). It is possible that a single β-keto reductase could reduce multiple positions in a manner analogous to reduction of a nascent fatty acid chain. This would require that certain keto groups on the polyketide chain, which are not reduced, must be protected. Thus, the polyketide synthase may present only the proper keto group to the associated β-keto reductase for reduction as the polyketide chain is being formed. This hypothesis is supported by studies showing that intact reduced polyketide fragments are incorporated into final macrolide molecules (5, 36). The operation of the *actIII* gene product as a β-keto reductase (15, 22) is also supported by the theory that polyketide biosynthesis is closely analogous

to, and perhaps derived from, fatty acid biosynthesis (3, 11, 31).

The data presented in this study are consistent with the hypothesis that the *actIII* gene product operates as a keto reductase (15). The fact that this reduction occurs at C-9 during the formation of actinorhodin, aloesaponarin II, and aklavinone may be a coincidence. In both cases studied here, actinorhodin (or aloesaponarin II) and anthracycline formation, the reduction occurs at a carbon which is part of the first ring formed during the biosynthesis of these various polyketide compounds. The reduction may be required for the proper or efficient folding of the polyketide chain to form the benzo- or naphthacene quinones. However, compounds such as tetracenomycin (37) and 2-hydroxyaklavinone (26) are produced by strains apparently lacking this keto reductase activity (22; data presented herein), indicating that this reduction step is not a strict requirement for anthracycline formation. Interestingly, *S. galilaeus* 31671 produces much smaller quantities of anthracyclines than does its parent strain, *S. galilaeus* 31133 (Bartel, unpublished data), possibly because of the absence of the polyketide reductase function.

Desoxyerythrolaccin and 1-*O*-methyldesoxyerythrolaccin as hybrid molecules. In the presence of only the *actI* and *actVII* loci, certain gene functions present in *S. galilaeus* strains 31133 and 31671, and a function absent from *S. galilaeus* 31671, determine which products are synthesized. Desoxyerythrolaccin and 1-*O*-methyldesoxyerythrolaccin, products of such a cloning venture, were not produced by any *S. coelicolor* strain tested. Our data also indicate that the *actIII* gene product, and its equivalent in *S. galilaeus* 31133, can react with different polyketides such as the precursors for actinorhodin or aloesaponarin II and for aklavinone. Mederhodins A and B were produced as a result of cloning DNA encoding part of the actinorhodin pathway, including the *actV* locus, in the medermycin producer *Streptomyces* sp. strain AM-7161 (17). We have used a completely different segment of the actinorhodin pathway DNA (24) to produce desoxyerythrolaccin and 1-*O*-methyldesoxyerythrolaccin. Unless these two examples are highly coincidental, these successes suggest that the formation of hybrid natural products via interspecies cloning of streptomycete antibiotic biosynthesis DNA may be a generally applicable process for the formation of new chemical structures.

### ACKNOWLEDGMENTS

We sincerely thank David A. Hopwood for the gift of bacterial strains and plasmids and for sharing unpublished information with us. We also thank Donald Ordaz for assistance with the HPLC analyses, Akira Yagi for an authentic sample of aloesaponarin II, and John Fry for his assistance with purification of compounds. We thank the Frederick Cancer Research Center, Rhône-Poulenc, and Adria for reference compounds.

This material is based on work supported by the National Science Foundation under grant DMB-8607619.

### LITERATURE CITED

1. Anderson, J. A., B.-K. Lin, H. J. Williams, and A. I. Scott. 1988. Deoxygenation of phenolic natural products. Enzymatic conversion of emodin to chrysophanol. J. Am. Chem. Soc. 110: 1623–1624.
2. Bibb, M. J., J. L. Schottel, and S. N. Cohen. 1980. A DNA cloning system for interspecies gene transfer in antibiotic-producing *Streptomyces*. Nature (London) 284:526–531.
3. Birch, A. J. 1967. Biosynthesis of polyketides and related compounds. Science 156:202–206.
4. Brockman, H., and H. Pini. 1947. Actinorhodin, ein roter

Farbstoff aus Actinomyceten. Naturwissenschaften 34:190.

5. Cane, D. E., and C.-C. Yang. 1987. Macrolide biosynthesis. 4. Intact incorporation of a chain-elongation intermediate into erythromycin. J. Am. Chem. Soc. 109:1255–1257.

6. Casey, M. L., R. C. Paulick, and H. W. Whitlock. 1978. Carbon-13 nuclear magnetic resonance study of the biosynthesis of daunomycin and islandicin. J. Org. Chem. 43:1627–1634.

7. Cole, S. P., B. A. M. Rudd, D. A. Hopwood, C.-j. Chang, and H. G. Floss. 1987. Biosynthesis of the antibiotic actinorhodin. Analysis of blocked mutants of *Streptomyces coelicolor*. J. Antibiot. 40:340–347.

8. Dekleva, M. L., and W. R. Strohl. 1987. Glucose-stimulated acidogenesis by *Streptomyces peucetius*. Can. J. Microbiol. 33:1129–1132.

9. Dekleva, M. L., J. A. Titus, and W. R. Strohl. 1985. Nutrient effects on anthracycline production by *Streptomyces peucetius* in a defined medium. Can. J. Microbiol. 31:287–294.

10. Deng, Z., T. Kieser, and D. A. Hopwood. 1988. “Strong incompatibility” between derivatives of the *Streptomyces* multi-copy plasmid pIJ101. Mol. Gen. Genet. 214:286–294.

11. Dutton, M. F. 1988. Enzymes and aflatoxin biosynthesis. Microbiol. Rev. 52:274–295.

12. Feinberg, A. P., and B. Vogelstein. 1983. A technique for radiolabeling DNA restriction endonuclease fragments to high specific activity. Anal. Biochem. 132:6–13. [Addendum: 137: 266–267.]

13. Floss, H. G., S. P. Cole, X.-G. He, B. A. M. Rudd, J. Duncan, I. Fujii, C.-j. Chang, and P. J. Keller. 1985. Biosynthesis of polyketide antibiotics, p. 283–304. In H. Kleinkauf, H. von Döhren, H. Dornauer, and G. Nesemann (ed.), Regulation of secondary metabolite formation. VCH Verlagsgesellschaft mbH, Weinheim, Federal Republic of Germany.

14. Gorst-Allman, C. P., B. A. M. Rudd, C.-j. Chang, and H. G. Floss. 1981. Biosynthesis of actinorhodin. Determination of the point of dimerization. J. Org. Chem. 46:455–457.

15. Hallam, S. E., F. Malpartida, and D. A. Hopwood. 1988. DNA sequence, transcription and deduced function of a gene involved in polyketide antibiotic biosynthesis in *Streptomyces coelicolor*. Gene 74:305–320.

16. Hopwood, D. A., M. J. Bibb, K. F. Chater, T. Kieser, C. J. Bruton, H. M. Kieser, D. J. Lydiate, C. P. Smith, J. M. Ward, and H. Schrempf. 1985. Genetic manipulation of *Streptomyces*, a laboratory manual. The John Innes Foundation, Norwich, United Kingdom.

17. Hopwood, D. A., F. Malpartida, H. M. Kieser, H. Ikeda, J. Duncan, I. Fujii, B. A. M. Rudd, H. G. Floss, and S. Omura. 1985. Production of ‘hybrid’ antibiotics by genetic engineering. Nature (London) 314:642–644.

18. Kieser, T. 1984. Factors affecting the isolation of CCC DNA from *Streptomyces lividans* and *Escherichia coli*. Plasmid 12: 19–36.

19. Kieser, T., D. A. Hopwood, H. M. Wright, and C. J. Thompson. 1982. pIJ101, a multi-copy broad host-range *Streptomyces* plasmid: functional analysis and development of DNA cloning vectors. Mol. Gen. Genet. 185:223–238.

20. Lampel, J. S., and W. R. Strohl. 1986. Transformation and transfection of the anthracycline-producing streptomycetes. Appl. Environ. Microbiol. 51:126–131.

21. Lydiate, D. J., F. Malpartida, and D. A. Hopwood. 1985. The *Streptomyces* plasmid SCP2*: its functional analysis and development into useful cloning vectors. Gene 35:223–235.

22. Malpartida, F., S. E. Hallam, H. M. Kieser, H. Motamedi, C. R. Hutchinson, M. J. Butler, D. A. Sugden, M. Warren, C. McKillop, C. R. Bailey, G. O. Humphreys, and D. A. Hopwood.

1987. Homology between *Streptomyces* genes coding for synthesis of different polyketides used to clone antibiotic biosynthesis genes. Nature (London) 325:818–821.

23. Malpartida, F., and D. A. Hopwood. 1984. Molecular cloning of the whole biosynthetic pathway of a *Streptomyces* antibiotic and its expression in a heterologous host. Nature (London) 309:462–464.

24. Malpartida, F., and D. A. Hopwood. 1986. Physical and genetic characterization of the gene cluster for the antibiotic actinorhodin in *Streptomyces coelicolor*. Mol. Gen. Genet. 205:66–73.

25. Maniatis, T., E. F. Fritsch, and J. Sambrook. 1982. Molecular cloning: a laboratory manual. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

26. Matsuzawa, Y., A. Yoshimoto, N. Shibamoto, H. Tobe, T. Oki, H. Naganawa, T. Takeuchi, and H. Umezawa. 1981. New anthracycline metabolites from mutant strains of *Streptomyces galilaeus* MA144-M1. II. Structure of 2-hydroxyaklavinone and new aklavinone glycosides. J. Antibiot. 34:959–964.

27. Mehandale, A. R., A. V. Rama Rao, I. N. Shaikh, and K. Venkataraman. 1968. Desoxyerythrolaccin and laccaic acid D. Tetrahedron Lett. 18:2231–2234.

28. Oki, T., I. Kitamura, Y. Matsuzawa, N. Shibamoto, T. Ogasawara, A. Yoshimoto, and T. Inui. 1979. Antitumor anthracycline antibiotics, aclacinomycin A and analogs. J. Antibiot. 32:801–819.

29. Ōmura, S., H. Ikeda, F. Malpartida, H. M. Kieser, and D. A. Hopwood. 1986. Production of new hybrid antibiotics, mederhodins A and B, by a genetically engineered strain. Antimicrob. Agents Chemother. 29:13–19.

30. Rudd, B. A. M., and D. A. Hopwood. 1979. Genetics of actinorhodin biosynthesis by *Streptomyces coelicolor*. J. Gen. Microbiol. 114:35–43.

31. Sherman, D. H., F. Malpartida, M. J. Bibb, H. M. Kieser, M. J. Bibb, and D. A. Hopwood. 1989. Structure and deduced function of the granaticin-producing polyketide synthase gene cluster of *Streptomyces violaceoruber* Tu22. EMBO J. 9:2717–2725.

32. Simpson, T. J. 1987. Applications of multinuclear NMR to structural and biosynthetic studies of polyketide microbial metabolites. Chem. Soc. Rev. 16:123–160.

33. Stutzman-Engwall, K. J., and C. R. Hutchinson. 1989. Multigene families for anthracycline antibiotic production in *Streptomyces peucetius*. Proc. Natl. Acad. Sci. USA 86:3135–3139.

34. Thompson, C. J., T. Kieser, J. M. Ward, and D. A. Hopwood. 1982. Physical analysis of antibiotic-resistance genes from *Streptomyces* and their use in vector construction. Gene 20:51–62.

35. Yagi, A., K. Makino, and I. Nishika. 1974. Studies on the constituents of *Aloe sapnaria* Haw. I. Structures of tetrahydroanthracene derivatives and the related anthraquinones. Chem. Pharm. Bull. 22:1159–1166.

36. Yue, S., J. S. Duncan, Y. Yamamoto, and C. R. Hutchinson. 1987. Macrolide biosynthesis. Tylactone formation involves the processive addition of three carbon units. J. Am. Chem. Soc. 109:1253–1255.

37. Yue, S., H. Motamedi, E. Wendt-Pienkowski, and C. R. Hutchinson. 1986. Anthracycline metabolites of tetracenomycin C-nonproducing *Streptomyces glaucescens*. J. Bacteriol. 167:581–586.

38. Zhang, H.-L., X.-G. He, A. Adefarati, J. Gallucci, S. P. Cole, J. M. Beale, P. J. Keller, C.-J. Chang, and H. G. Floss. 1990. Mutactin, a novel polyketide from *Streptomyces coelicolor*. Structure and biosynthetic relationship to actinorhodin. J. Org. Chem. 55:1682–1684.
