
Review

The ectodysplasin pathway: from diseases to adaptations

Alexa Sadier, Laurent Viriot, Sophie Pantalacci, and Vincent Laudet

Institut de Génomique Fonctionnelle de Lyon, Université de Lyon, Université Lyon 1, Centre National de la Recherche Scientifique (CNRS), Ecole Normale Supérieure de Lyon, 46 allée d'Italie, 69364 Lyon CEDEX 07, France

The ectodysplasin (EDA) pathway, which is active during the development of ectodermal organs, including teeth, hairs, feathers, and mammary glands, and which is crucial for fine-tuning the developmental network controlling the number, size, and density of these structures, was discovered by studying human patients affected by anhidrotic/hypohidrotic ectodermal dysplasia. It comprises three main gene products: EDA, a ligand that belongs to the tumor necrosis factor (TNF)-α family, EDAR, a receptor related to the TNFα receptors, and EDARADD, a specific adaptor. This core pathway relies on downstream NF-κB pathway activation to regulate target genes. The pathway has recently been found to be associated with specific adaptations in natural populations: the magnitude of armor plates in sticklebacks and the hair structure in Asian human populations. Thus, despite its role in human disease, the EDA pathway is a 'hopeful pathway' that could allow adaptive changes in ectodermal appendages which, as specialized interfaces with the environment, are considered hot-spots of morphological evolution.

Ectodermal dysplasias

In a section of his 1875 book *The Variation of Animals and Plants under Domestication*, Charles Darwin discusses that 'skin and the appendages of hair, feathers, hoofs, horns and teeth, are homologous over the whole body'. In this very same section entitled 'Correlated variation of homologous parts' he describes the case of a 'Hindoo family in Scinde, in which ten men, in the course of four generations, were furnished, in both jaws taken together, with only four small and weak incisor teeth and with eight posterior molars. The men thus affected have very little hair on the body, and become bald early in life. They also suffer much during hot weather from excessive dryness of the skin. It is remarkable that no instance has occurred of a daughter being thus affected' [1]. In a few sentences he described what became known as anhidrotic/hypohidrotic ectodermal dysplasia (HED) and rightly pointed out two striking features of this disease: it affects several ectodermal organs in a correlated manner, and predominantly affects men (one of the genes responsible is carried by the X chromosome). HED is in fact one of many (more than 150) relatively rare diseases, collectively termed ectodermal dysplasias, that affect the skin and other ectodermal organs.

In HED (OMIM 257980), patients lack hair (hypotrichosis), teeth (oligodontia), and nails. Teeth are often malformed in these HED patients. Sweat gland deficiency (anhidrosis/hypohydrosis) explains the inability to resist high temperatures mentioned by Darwin. Several other glands are also affected, including the lacrimal glands, the sebaceous glands, the Meibomian glands, and possibly mucous glands in the respiratory tract. Many affected individuals present a characteristic facial appearance (prominent forehead, thick lips) and in some cases an immune deficiency has been observed [2,3]. It is fascinating that this example, reported by Darwin as an interesting case of pathological variation in human, is now a seminal example of how signaling pathways can affect human health as well as be recruited in wild species for specific adaptations. Two cases have been particularly illustrative in that respect: in stickleback a variant of *eda* is associated with variation in the defensive armor [4], and in human a point mutation in the receptor *EDAR* has been shown to be under positively selection and has been linked to hair thickness and specific tooth morphology [5]. Here we review these two cases and discuss the implications of this dual link between human disease and natural variation.

From human genetics to animal mutants

More than 120 years after Darwin made his observations, the gene mutated in the X-linked forms of the disease was first mapped on chromosome X of male patients and later identified using rare female patients that harbor an X-autosome translocation [6,7]. This gene was termed *EDA* for ectodysplasin in reference to the disease. A very similar phenotype (hair defects, tooth abnormalities, and absence of sweat glands [8]), had been described in the mouse spontaneous X-linked mutant *tabby*. This facilitated the cloning of the *Eda/tabby* gene, the mouse ortholog of the human *EDA* gene [9,10], and eventually pointed to the fact the encoded protein was a member of the TNF superfamily. From then on mouse mutants which closely resemble the *tabby* mutant enabled the identification of other genes of the core ‘EDA pathway’. The mouse mutant *downless* led to the *Edar* gene, a member of TNF receptor superfamily [11,12], reviewed in [13]). Lastly, the *crinkled* mouse

Corresponding author: Laudet, V. (Vincent.Laudet@ens-lyon.fr).

Keywords: ectodysplasin; anhidrotic/hypohidrotic ectodermal dysplasia; adaptation; ectodermal appendages; signaling pathways.

0168-9525/$ – see front matter

© 2013 Elsevier Ltd. All rights reserved. http://dx.doi.org/10.1016/j.tig.2013.08.006

CrossMark

Trends in Genetics, January 2014, Vol. 30, No. 1

Review

Trends in Genetics January 2014, Vol. 30, No. 1

(A)

EDA  
EDAR  
EDARADD  

(B)

EDA  
Col  
TNF  
Furin cleavage  

EDAR  
LBD  
TM  
DD  

EDARADD  
Traf6 BS  

Key:  
- EDA X-linked mutations  
- Recessive autosomal mutations  
- Dominant autosomal mutations  

Figure 1. The canonical EDA pathway. (A) Principal model for the EDA pathway downstream signaling: the trimeric ligand EDA binds to the trimeric receptor EDAR leading to the recruitment of the adaptor EDARADD and the formation of a complex containing EDARADD, Traf6, Tab2, and Tak1. Tak1 activates the IKK complex (IKK1, IKK2, and NEMO) which ubiquitinates IkB, leading to the release of NF-kB transcription factor. NF-kB is then translocated into the nucleus to activate target genes. Figure adapted from [37]. (B) EDA, EDAR, and EDARADD proteins and their functional domains. Col, collagen domain; DD, death domain; LBD, ligand-binding domain; TM, transmembrane domain; TNF, TNF domain; Traf6 BS, Traf6 binding site. Principal mutations are indicated. For EDAR and EDARADD autosomal mutations, it is interesting to note that dominant mutations are mainly localized in the interaction domain, resulting in impaired oligomerization and a dominant negative effect.

mutant, almost identical to the tabby and downless mutants, enabled identification of the Edaradd gene, encoding a death-domain adaptor [14] (Figure 1A). Mutations in the orthologous genes in human, EDAR and EDARADD, also cause HED [3,14,15]. To date there are more than 100 different mutations, recessive or dominant, described in the X-linked EDA gene, whereas only 20 and 3 causative mutations have been found in EDAR and EDARADD, respectively [3,16,17] (Figure 1B). Hypomorphic alleles of EDA have been discovered that lead to a reduction in tooth number (affecting the incisors only) although a clinically normal phenotype has also been observed [18–20]. Of note, similar cases of ectodermal dysplasia have been attributed to mutations in EDAR and EDARADD orthologs in other mammals including dogs [21], various cattle breeds [22], and rats [23]. The similarity of the ectodermal dysplasia phenotype in mutants of these five species highlights the conserved and important role of the pathway in mammalian ectodermal appendage development.

Recent data show that the EDA pathway is not restricted to mammals and is largely conserved in most, if not all, vertebrates. The genes of the EDA pathway are expressed during feather development in chicken (with the notable difference of EDA being expressed in the mesenchyme rather than in the epithelium) [24]. In addition, inhibition

of the pathway through dominant negative receptor expression decreased the number of feather placodes (the embryonic structure that will later develop into a specific organ, here feather) [25]. More recently the EDA pathway was also implicated in ectodermal development in fish: an edar mutant in medaka is almost completely devoid of scales and exhibits a tooth phenotype [26,27]. Similarly, mutations in zebrafish eda and edar genes located in the protein at positions homologous to those mutated in human HED patients lead to defects in fins, scales, and pharyngeal teeth [28]. From these data, it is likely that the EDA pathway controls ectodermal appendages in all vertebrates and provides a common paradigm governing the development of all these structures from hairs, scales, or feathers to various glands.

From EDA to the NF-kB pathway

Given the similar clinical syndrome produced by mutation of these three genes in humans and their three counterparts in mice, it is no surprise that their products form a linear pathway: the trimeric EDA ligand binds to the trimeric EDAR receptor which, upon binding, recruits the EDARADD adaptor via death-domain/death-domain interactions (Figure 1 [29]). These interactions were tested in vitro: first, EDAR was shown to interact with one protein product of the Eda gene, the secreted protein

EDA-A1 [30–33], through its TNF-binding domain. Then, EDAR was shown to recruit the intracellular adaptor EDARADD through its death domain. Another protein product of the *Eda* gene (EDA-A2), resulting from the skipping of two amino acids by alternative splicing, was later shown to bind another TNFR receptor, EDA2R (previously termed XEDAR) [33]. A recent report based on a unique case suggests that loss of function of this gene can be associated with HED, although a mouse knockout does not induce an HED-like phenotype [34, 35].

This cascade, from EDA and EDAR to EDARADD, is reminiscent of other TNF pathways (e.g., TNF/TNFR/TRADD), therefore it was hypothesized that this early signaling would lead to activation of the NF-κB pathway [30, 36]. This was first shown *in cellulo* [14, 30, 36] and further confirmed by demonstrating the involvement of canonical partners for the NF-κB activation pathway such as TAB2, IKK, and TRAF 6 ([37], for review see [29]). Although the activation of the JNK pathway and apoptosis were also proposed, these results remain controversial because the response remains very weak as compared to other TNF [30, 36, 38]. The relevance of NF-κB activation was further suggested when mice mutant for Traf6 or IKK/NEM were shown to exhibit tabby-like phenotypes, and HED patients were identified that had mutations in these genes [39–41]. However, there is also some evidence that an alternative, non-NF-κB-dependent pathway may exist [42]. Finally, the comparison of a NF-κB-LacZ reporter activation in tabby versus wild type mice also indicated that decreased EDA signaling effectively induces a severe decrease in NF-κB activation [38, 43]. Therefore, it is clear, from the data discussed above, that the NF-κB pathway is a major transducer of the EDA pathway.

The implication of the NF-κB pathway in development came as a surprise because it is primarily known for its major role in inflammation and immunity, acting downstream of the activation of TNF immune receptors [44]. By contrast, the activation of NF-κB by the EDA pathway does not seem to be obviously linked to inflammation. This raises the recurrent question of how cells can differentiate between the types of messages transduced by the NF-κB pathway and how specific target genes are activated for developmental or inflammatory responses. Interestingly, mice mutant for core proteins of the NF-κB pathway such as TRAF6 and NEMO [40, 45] exhibit not only an HED syndrome and its developmental defects but also inflammation problems. Interestingly, recent data suggest that the EDA pathway induces the transcription of chemokines (key players of inflammation) that also have a developmental role [46]. Moreover, it has been observed that the ectodermal overexpression of the glucocorticoid receptor (GR), a well-known inhibitor of NF-κB in inflammation, induces an HED phenotype [47]. This suggests that the pathways in ectoderm development and inflammation may not be fundamentally different, and that the specificity of target gene activation may largely be determined by the general context of a given cell type.

### Tuning ectodermal appendages with the EDA pathway

Many developmental pathways other than the EDA pathway, such as Wnt, Fgf, BMP, and hedgehog, also participate in the development of ectodermal appendages. However, most of these pathways are highly pleiotropic, and disruption of even one core component of these pathways generally leads to very severe developmental defects not only in ectodermal appendages but also in other organs. By contrast, the absence of any one factor in the EDA pathway specifically affects only ectodermal appendages, and generally (with a single exception to our knowledge) does not preclude their development. Instead, these organs typically present defects in terms of shape, number, size, or position in the organism. For example, in both humans and mice, the dental formula and tooth shape is abnormal (reviewed in [13, 29]). In mice, primary (guard) hairs do not develop, but others hair types exhibit an abnormal morphology and decreased density [48–50]. In addition, mammary glands and salivary glands exhibit defects in branching and size [29, 51–54]. The mutation of *edar* in medaka has a similar effect, leading to a reduction of scale number and a drastic decrease in tooth number and organization in both the oral and pharyngeal cavities [27]. Conversely, increasing EDA pathway activity tends to increase the number, size, and density of ectodermal appendages. For example, overexpression of *Eda* in the ectoderm [under the control of the keratin 14 (*K14*) promoter (*K14-Eda* mice)] results in ectopic teeth and mammary glands, larger hair placodes, and higher dental complexity [50, 55–58]. In chicken, hyperactivation of *EDAR* during feather development results in increased feather placode density. Taken together, these findings suggest that the EDA pathway plays a conserved role in fine-tuning the size, spacing, and position (and probably thereby shape) of ectodermal organs in vertebrates.

This tuning role of the EDA pathway is reflected by its integral position in the gene regulatory network controlling ectodermal appendage development, as exemplified by studies carried out in mouse. Studies of hair and teeth development in mice have characterized the position of the EDA pathway respective to other more ‘classical’ developmental pathways (reviewed in [13, 29]). The Wnt pathway regulates both EDA [29, 59] and EDAR [60], and the activin and BMP pathways also regulate EDAR [58, 59]. Transcriptomic and developmental studies also identified downstream targets of EDAR, including Shh, FGF20, and the Wnt and/or BMP antagonists Dkk4, CCN2/CTGF, and follistatin (reviewed in [29], see also [61]). Owing to its position both downstream and upstream of key developmental pathways (Figure 2), the EDA pathway can modulate the strength and range of activation of these pathways.

In conclusion, the EDA pathway is specific for ectodermal appendage development where it serves to fine-tune growth, which can be seen in both phenotypes and molecular function. This may have allowed it to play a role in morphological changes during evolution because, in contrast to other major developmental pathways, it is not essential and therefore can be changed during evolution without producing large pleiotropic lethal effects.

### EDA pathway variation in natural populations

The ectodermal appendages, including teeth, teleost scales, bird feathers, mammalian hair, and mammary glands, are specialized interfaces with the environment,

Review

Trends in Genetics January 2014, Vol. 30, No. 1

Ectodermal placoda
![Diagram](#)

EDA pathway

Ectodermal placoda

Number
Size
Density

Low

High

Wnt
Activin
BMP

EDA
EDAR

Shh
Fgf20
Dkk
CTGF
Follistatin
Wnt10b

Morphogenesis
+/-
Feedback

Figure 2. The EDA pathway as a modulator of key developmental pathways. Proposed model for the role of the EDA pathway in the regulation of the establishment of placodes that give rise to ectodermal organs. The EDA pathway is at a center position beside other key developmental pathways. Regulators of the EDA pathway are listed on the left and targets are listed on the right. Two types of targets can be distinguished: those that are involved in morphogenesis, and those that participate in positive or negative feedback loops that are responsible for local reinforcement and diffusion range limitation of factors that control placode fate. Thus, enhanced or reduced signaling induced by the EDA pathway modulate the strength and range of activation of other key developmental pathways, ultimately fine-tuning the number, size, and position of placodes.

and these structures are known to be hot-spots of morphological evolution both at the macroevolutionary and micro-evolutionary levels. The study of their evolution is still a major topic in evolutionary biology, and the EDA pathway is of interest in this regard. Two main cases show that the genes of the EDA pathway have indeed been recruited for specific adaptations: one in a teleost fish, the stickleback, and another in our own species, *Homo sapiens*, both of which we discuss below.

In the first case a gene of the EDA pathway, namely *eda* itself, was linked to a specific adaptation in the defensive armor of the threespine stickleback (*Gasterosteus aculeatus*). In this species, parallel adaptive evolution occurred when marine fishes repeatedly colonized freshwater streams and lakes 10,000 to 20,000 years ago. Extensive studies of natural populations showed that the complete armor of 30–36 plates is favored in marine habitats, probably because it facilitates escape from predator capture and renders ingestion by the predator more difficult. By contrast, the low armor of up to nine plates typical of freshwater populations could be favored because it allows more rapid growth and thus better odds for winter survival, but other explanations such as reduced ion availability or improved swimming performance in freshwater have also been proposed ([62, 63], reviewed in [64]) (Figure 3). Whatever the cause, the reduced fitness of the full-plate phenotype in freshwater environments has been clearly demonstrated ([65] and references therein). QTL analysis and positional cloning methods [4] revealed that the gene *eda* was associated with this evolutionary change. Interestingly, despite their independent origins, the vast majority of freshwater populations present the same low-plate haplotype at the homozygous state. The low-plate allele is also found at very low frequency in marine fishes, and fishes heterozygous for this allele have mostly complete lateral plates. Freshwater low-armour populations thus evolved through the repeated selection of the same allele that seems to be retained as a cryptic variant within marine populations. The opposite scenario explains a case of reverse evolution shown in an urban freshwater lake [66] (Figure 3). In this case, the standing highly plated allele increased in frequency in the freshwater population, probably as a consequence of increased predation by trout and, within 40 years, half the population had reverted to a complete armor phenotype.

The precise mutations within the *eda* locus responsible for the low activity of the EDA pathway in freshwater morphs have not yet been identified. It is clear that this allele is a partial loss of function based on the reduced scale phenotype of medaka and zebrafish mutants in the EDA pathway genes [26–28]. Moreover, it was predicted that these mutations should affect regulatory regions and decrease EDA expression [4, 67]. Interestingly, it has also been shown that other genes of the pathway (in particular *edar*, *edaradd*, *traf6* and *nemo*) are not associated with low-plated freshwater fishes, suggesting a bias of evolvability towards the *eda* gene [67].

Review

Trends in Genetics January 2014, Vol. 30, No. 1

![Figure 3. Sticklebacks with different activity levels of the EDA pathway. (Above) The marine stickleback morph with the full armor plate phenotype with 30–36 plates. (Center) A freshwater stickleback caught in 1957 in Lake Washington, Seattle, exhibits the typical freshwater low plate phenotype with up to nine plates corresponding to reduced activity of the EDA pathway. (Below) A 'reverted' freshwater fish caught in 2006 in which a high amour plate phenotype is clearly visible. The bones have been labeled using alizarin red staining as described in [66]. Photographs courtesy of Jun Kitano.](image.png)

increased hair thickness [76] and specific tooth morphology [77,78] in East Asian populations. Interestingly, this variant was shown to enhance NF-κB activation in cells, indicating that it has a gain of function effect [75,79]. In agreement, the V370A mutation has been shown to attenuate the severity of *EDA* alleles responsible for the HED phenotype [80]. Consistently, increased *Edar* activity in transgenic mice with increased *Edar* copy number (*Edar*<sup>Tg951</sup> line) converts the hair phenotype to a typical East Asian morphology through the development of enlarged hair follicles [75,79]. However, the most convincing evidence that the V370A variant is responsible for the specific East Asian hair morphology was recently obtained by knocking-in this mutation into the endogenous mouse *Edar* gene [5]. Not only did the *Edar*<sup>V370A</sup> mice display increased hair thickness, but they also had increased mammary gland branch density as well as increased eccrine sweat gland number [5]. This latter effect is also associated with the mutation in a Han Chinese cohort, further showing that the variant has pleiotropic effects in human populations [5]. For this reason, it is still unclear which effects provided the selective value of the variant and which were hitch-hiking effects. To date, there is no specific clue either way, but it is interesting to mention one hypothesis put forward [5] suggesting that positive selection was not for a specific single trait but instead for a combination of phenotypic changes elicited by the variant, possibly through different, mutually reinforcing, selective pressures: some traits could confer environmental advantages whereas others (e.g., effects on the mammary gland) could have been subject to sexual selection.

Finally, it is important to note that reduction of the armor plate may only be the most obvious phenotypic effect linked to a reduced activity of *eda* in sticklebacks. Recently, the *eda* locus has been associated with behavioral differences in freshwater versus marine sticklebacks. Interestingly, the low-plate allele was associated with a propensity to move to new environments, suggesting that marine sticklebacks carrying the low-plate allele may be more likely to colonize freshwater environments [68]. This may be one factor among others explaining the multiple selection of this allele in freshwater fishes: other subtle (but important in terms of fitness) morphological, physiological, or behavioral changes may be associated with freshwater *eda* alleles [49]. Indeed, the pleiotropy of the pathway is well illustrated in the following example of adaptation associated with the EDA pathway.

The second case involves the receptor *EDAR* in humans, which is modified by a specific mutation in the open reading frame in Asian populations. This mutation was identified in genome-wide scans for traces of past positive selection in human populations [69–74] and is a point mutation in which a conservative amino acid change occurred: a conserved valine residue in the death domain of the protein is replaced by an alanine at position 370 (V370A) [5] (Figure 1). Although some debate exists, this mutation was found in East Asian and native American populations, and it has been suggested that it was selected in Asia or perhaps more precisely in central China ~30,000 years ago [5,75]. It correlates with

In addition to providing interesting cases for which the genomic basis of specific adaptations are identified (still the holy grail of the evo/devo field), these two examples also illustrate the dichotomy of the pleiotropy of developmental genes which can nevertheless be recruited for specific adaptations [81,82]. It has been suggested that mutations in cis-regulatory sequences are more likely to underlie phenotypic evolution than other types of genetic changes because they avoid the pleiotropic effects linked to mutations in open reading frames of encoded proteins. The case of the V370A mutation in humans illustrates that a single point mutation can be pleiotropic but that these effects, although fairly prominent, can be retained if the net adaptive value of the mutation is positive.

Studying the EDA pathway highlights the connections between diseases and evolutionary adaptations in natural populations. It can be anticipated that this will be even more prominent in coming years because recent developments at the forefront of clinical research now provide a unique tool kit for studying the involvement of the pathway in natural adaptations. Indeed, a series of reagents [a recombinant form of EDA-A1 able to cross the placental barrier (Fc-EDA-A1) and agonist anti-EDAR monoclonal antibodies], that aim to correct the HED phenotype observed in EDA pathway mutants, have been developed and were shown to rescue effectively the phenotype in both mouse and dogs [49,83,84]. This impressive proof of concept as a possible treatment of the disease also provides

tools to manipulate the level of the EDA pathway in model organisms or in natural variants, opening the way to direct functional studies.

**Concluding remarks**

In light of these examples and our knowledge of the pathway, we are convinced that the two described cases are only the tip of the iceberg—and that other adaptations will be linked to the genes of the EDA pathway. This could be the case of the V370A mutation, which could putatively evolve repeatedly in other species as seen for other genes [85, 86], and especially if the hypothesis that it drives correlated advantageous traits is correct. Of course, this could also be the case for other unknown mutations in any

of the genes in the pathway. The diverse teeth and hair morphologies, some of which are known to be adaptive in mammals, the scales and teeth morphologies observed in teleosts, and the modified feathers or scales in sauropsids, provide a fascinating reservoir to explore for changes in the EDA pathway activity (Box 1). In terms of sequence evolution at the vertebrate scale, the EDA pathway genes have been shown to combine both conserved features and evolutionary shifts [87]. These shifts are found at different taxonomic levels, suggesting that changes in the pathway may well have been linked to ectodermal organ diversification in vertebrates. Orthologs of EDA, as well as a unique non-duplicated orthologs of EDAR/Xedar/Troy, are found in invertebrate chordate genomes (Ciona and amphioxus):

**Box 1. How changing the dose of the EDA pathway affects dentition in human and mouse**

In both human and mouse, tooth number and morphology are highly sensitive to modifications in the dose of EDA signaling (Figure 1). Interestingly, in both species loss or gain of function results in a large range of phenotypic modifications.

In human, one can distinguish severe loss of function, which results in a significant reduction of primary and permanent dentition (on average 22 missing teeth, sometimes extending to complete lack of teeth) and severe shape modifications (such as typical peg-shaped teeth, reduction in tooth size and cusp number) [90, 91], from less severe cases in which no clear HED symptoms are diagnosed and in which only incisors are affected [18, 20]. In the opposite case, the gain-of-function mutation found in East Asian populations results in shovel-shaped incisors [77, 78].

In mouse, the general trends observed in tabby (Figure 1: tabby mouse molar as compared to wild type), downless, and crinkled mouse dentitions are reductions of both molar size and cusp number, as well as modifications in the segmentation of dental rows leading to 2–4 cheek teeth bearing various shape anomalies. Detailed analysis by two groups pointed out that the tabby phenotype with only two molars and simplified molar morphology is reminiscent to some insectivorous rodent species, such as the water rat Hydromys chrysogaster [56]. By contrast, overexpression of the Eda or Edar genes (see Figure 1: K14-Eda mouse molar) also leads to occasional abnormal segmentation of the molar dental row and to various shape anomalies [50, 55, 56]. The dominant dental trait of these mutants is the occurrence of longitudinal crests in both lower and upper molars. These crests resemble a rodent molar phenotype known as stephanodonty, which occurs in many murine rodents but never in mice, and is thought to be an adaptation to herbivory [94].

It is interesting to note that detailed morphological analysis revealed that the Eda/tabby and Edar/J mutations differently affect the molar phenotype [92], which is most obvious in homozygous mutants for lower molars, the Edar/downless phenotype being more severe in terms of cusp reduction. Moreover, the sensitivity of upper and lower dental rows shows variability in regards to loss of function of EDA pathway genes, the lower row tending to be more drastically modified. All this suggests that tooth morphology is sensitive to small variations in EDA pathway signaling level, and therefore that evolutionary trends in mammalian dentition—such as the reduction in tooth number or crown crestization in herbivores—are phenotypes that may have been controlled by genes of the EDA pathway.


Mouse

![Image](https://via.placeholder.com/150)

Tabby

![Image](https://via.placeholder.com/150)

WT

![Image](https://via.placeholder.com/150)

K14-Eda

EDA signaling level

Human

Severe hypodontia  
HED  
(Reviewed in Clauss et al. 2008)

Minor hypodontia  
No HED  
(Nikopensis et al. 2013)

EDAR gain of function  
Sinodontia  
(Park et al. 2012)


**Figure 1.** Tooth phenotypes resulting from alterations of the EDA pathway. Mouse upper tooth raw phenotype and human tooth phenotype are compared against EDA signaling levels ranging from loss of EDA function to increased activation. Mouse phenotypes in tabby (Eda null mutant), wild type (WT), and K14-Eda (Eda gain of function) are shown by scans of the upper tooth raw. The equivalent tooth phenotypes in human are explained in the lower part of the figure. Human severe hypodontia is reviewed in [91]; for minor hypodontia see [93]; for EDAR gain of function/sinodontia see [78]. HED, anhidrotic/hypohidrotic ectodermal dysplasia.

their developmental roles as well as their biochemical function are totally unknown to date but could bring useful information on the origin of the pathway [87, 88].

These observations can be linked to the debate around Richard Goldschmidt's suggestion that small genetic changes to developmentally important genes may have large phenotypic effects, producing a 'hopeful monster' that could be selected, thereby leading to rapid evolutionary changes [89]. The finding that a pleiotropic pathway involved in disease and adaptation is reminiscent of Goldschmidt’s hopeful monster. It is interesting to note in this respect that, from the various cases discussed in this review, we can track null mutations causing HED, hypomorphic EDA mutants causing selective tooth agenesis, and a gain of function EDAR allele in East Asian populations causing specific tooth morphology, illustrating very well the wide range of phenotypic variance associated with this pathway. In line with our prediction that the EDA pathway will be involved in other cases of adaptation, and the fact that it is one of the rare pathways for which a correction of the disease can be envisaged in humans, we would like to see the EDA pathway as a ‘hopeful pathway’.

### Acknowledgments

Work in our laboratory is supported by the Ministry of Research and Education (Agence Nationale de la Recherche, ANR) programs Quenottes (ANR-06-BLAN-0216), Bouillabaisse (ANR-09-BLAN-0127), BigTooth (ANR 2011 BSV 7 00803), and Convergdent (ANR 2011 JSV 6 00501). A.S. holds a fellowship from Fondation ARC pour la Recherche sur le Cancer. We thank Joanne Burden for critical reading of the manuscript and Jun Kitano for the stickleback pictures.

### References

1. Darwin, C. (1875) *The Variations of Animals and Plants under Domestication*, John Murray
2. Wiśniewski, S.A. *et al.* (2002) Recent advances in understanding of the molecular basis of anhidrotic ectodermal dysplasia: discovery of a ligand, ectodysplasin A and its two receptors. *J. Appl. Genet.* 43, 97–107
3. Cluzeau, C. *et al.* (2011) Only four genes (EDA1, EDAR, EDARADD, and WNT10A) account for 90% of hypohidrotic/anhidrotic ectodermal dysplasia cases. *Hum. Mutat.* 32, 70–72
4. Colosimo, P.F. *et al.* (2005) Widespread parallel evolution in sticklebacks by repeated fixation of Ectodysplasin alleles. *Science* 307, 1928–1933
5. Kamberov, Y.G. *et al.* (2013) Modeling recent human evolution in mice by expression of a selected EDAR variant. *Cell* 152, 691–702
6. Kere, J. *et al.* (1996) X-linked anhidrotic (hypohidrotic) ectodermal dysplasia is caused by mutation in a novel transmembrane protein. *Nat. Genet.* 13, 409–416
7. Srivastava, A.K. *et al.* (1996) Fine mapping of the EDA gene: a translocation breakpoint is associated with a CpG island that is transcribed. *Am. J. Hum. Genet.* 58, 126–132
8. Sofaer, J.A. (1969) Aspects of the Tabby-crinkled-downless syndrome. I. The development of Tabby teeth. *J. Embryol. Exp. Morphol.* 22, 181–205
9. Ferguson, B.M. *et al.* (1997) Cloning of Tabby, the murine homolog of the human EDA gene: evidence for a membrane-associated protein with a short collagenous domain. *Hum. Mol. Genet.* 6, 1589–1594
10. Srivastava, A.K. *et al.* (1997) The Tabby phenotype is caused by mutation in a mouse homologue of the EDA gene that reveals novel mouse and human exons and encodes a protein (ectodysplasin-A) with collagenous domains. *Proc. Natl. Acad. Sci. U.S.A.* 94, 13069–13074
11. Headon, D.J. and Overbeek, P.A. (1999) Involvement of a novel Tnf receptor homologue in hair follicle induction. *Nat. Genet.* 22, 370–374
12. Monreal, A.W. *et al.* (1999) Mutations in the human homologue of mouse dl cause autosomal recessive and dominant hypohidrotic ectodermal dysplasia. *Nat. Genet.* 22, 366–369

13 Mikkola, M.L. and Thesleff, I. (2003) Ectodysplasin signaling in development. *Cytokine Growth Factor Rev.* 14, 211–224
14 Headon, D.J. *et al.* (2001) Gene defect in ectodermal dysplasia implicates a death domain adapter in development. *Nature* 414, 913–916
15 Shimomura, Y. *et al.* (2004) A rare case of hypohidrotic ectodermal dysplasia caused by compound heterozygous mutations in the EDAR gene. *J. Invest. Dermatol.* 123, 649–655
16 Mikkola, M.L. (2009) Molecular aspects of hypohidrotic ectodermal dysplasia. *Am. J. Med. Genet. A* 149A, 2031–2036
17 Chassaing, N. *et al.* (2010) Mutations in EDARADD account for a small proportion of hypohidrotic ectodermal dysplasia cases. *Br. J. Dermatol.* 162, 1044–1048
18 Tarpey, P. *et al.* (2007) A novel Gln358Glu mutation in ectodysplasin A associated with X-linked dominant incisor hypodontia. *Am. J. Med. Genet. A* 143, 390–394
19 Han, D. *et al.* (2008) Novel EDA mutation resulting in X-linked non-syndromic hypodontia and the pattern of EDA-associated isolated tooth agenesis. *Eur. J. Med. Genet.* 51, 536–546
20 Tao, R. *et al.* (2006) A novel missense mutation of the EDA gene in a Mongolian family with congenital hypodontia. *J. Hum. Genet.* 51, 498–502
21 Casal, M.L. *et al.* (2005) Mutation identification in a canine model of X-linked ectodermal dysplasia. *Mamm. Genome* 16, 524–531
22 Drögemüller, C. *et al.* (2001) Partial deletion of the bovine ED1 gene causes anhidrotic ectodermal dysplasia in cattle. *Genome Res.* 11, 1699–1705
23 Kuramoto, T. *et al.* (2011) A rat model of hypohidrotic ectodermal dysplasia carries a missense mutation in the Edaradd gene. *BMC Genet.* 12, 91
24 Houghton, L. *et al.* (2005) The ectodysplasin pathway in feather tract development. *Development* 132, 863–872
25 Drew, C.F. *et al.* (2007) The Edar subfamily in feather placode formation. *Dev. Biol.* 305, 232–245
26 Kondo, S. *et al.* (2001) The medaka rs-3 locus required for scale development encodes ectodysplasin-A receptor. *Curr. Biol.* 11, 1202–1206
27 Atukorala, A.D. *et al.* (2010) Scale and tooth phenotypes in medaka with a mutated ectodysplasin-A receptor: implications for the evolutionary origin of oral and pharyngeal teeth. *Arch. Histol. Cytol.* 73, 139–148
28 Harris, M.P. *et al.* (2008) Zebrafish eda and edar mutants reveal conserved and ancestral roles of ectodysplasin signaling in vertebrates. *PLoS Genet.* 4, e1000206
29 Mikkola, M.L. (2008) TNF superfamily in skin appendage development. *Cytokine Growth Factor Rev.* 19, 219–230
30 Kumar, A. *et al.* (2001) The ectodermal dysplasia receptor activates the nuclear factor-kappaB, JNK, and cell death pathways and binds to ectodysplasin A. *J. Biol. Chem.* 276, 2668–2677
31 Elomaa, O. *et al.* (2001) Ectodysplasin is released by proteolytic shedding and binds to the EDAR protein. *Hum. Mol. Genet.* 10, 953–962
32 Tucker, A.S. *et al.* (2000) Edar/Eda interactions regulate enamel knot formation in tooth morphogenesis. *Development* 127, 4691–4700
33 Yan, M. *et al.* (2000) Two-amino acid molecular switch in an epithelial morphogen that regulates binding to two distinct receptors. *Science* 290, 523–527
34 Wisniewski, S.A. and Trzeciak, W.H. (2012) A new mutation resulting in the truncation of the TRAF6-interacting domain of XEDAR: a possible novel cause of hypohidrotic ectodermal dysplasia. *J. Med. Genet.* 49, 499–501
35 Newton, K. *et al.* (2004) Myodegeneration in EDA-A2 transgenic mice is prevented by XEDAR deficiency. *Mol. Cell. Biol.* 24, 1608–1613
36 Kopppinen, P. *et al.* (2001) Signaling and subcellular localization of the TNF receptor Edar. *Exp. Cell Res.* 269, 180–192
37 Morlon, A. *et al.* (2005) TAB2, TRAF6 and TAK1 are involved in NF-kappaB activation induced by the TNF-receptor, Edar and its adaptor Edaradd. *Hum. Mol. Genet.* 14, 3751–3757
38 Schmidt-Ullrich, R. *et al.* (2006) NF-kappaB transmits Eda A1/EdaR signalling to activate Shh and cyclin D1 expression, and controls post-initiation hair placode down growth. *Development* 133, 1045–1057
39 Ohazama, A. *et al.* (2004) Traf6 is essential for murine tooth cusp morphogenesis. *Dev. Dyn.* 229, 131–135

Review                                                                 Trends in Genetics January 2014, Vol. 30, No. 1

40 Naito, A. *et al.* (2002) TRAF6-deficient mice display hypohidrotic ectodermal dysplasia. *Proc. Natl. Acad. Sci. U.S.A.* 99, 8766–8771  
41 Zonana, J. *et al.* (2000) A novel X-linked disorder of immune deficiency and hypohidrotic ectodermal dysplasia is allelic to incontinentia pigmenti and due to mutations in IKK-gamma (NEMO). *Am. J. Hum. Genet.* 67, 1555–1562  
42 Pispa, J. *et al.* (2008) Edar and Troy signalling pathways act redundantly to regulate initiation of hair follicle development. *Hum. Mol. Genet.* 17, 3380–3391  
43 Dickson, K.M. *et al.* (2004) TRAF6-dependent NF-kB transcriptional activity during mouse development. *Dev. Dyn.* 231, 122–127  
44 Gilmore, T.D. and Wolenski, F.S. (2012) NF-κB: where did it come from and why? *Immunol. Rev.* 246, 14–35  
45 Shifera, A.S. (2010) The zinc finger domain of IKKγ (NEMO) protein in health and disease. *J. Cell. Mol. Med.* 14, 2404–2414  
46 Lefebvre, S. *et al.* (2012) Identification of ectodysplasin target genes reveals the involvement of chemokines in hair development. *J. Invest. Dermatol.* 132, 1094–1102  
47 Cascallana, J.L. *et al.* (2005) Ectoderm-targeted overexpression of the glucocorticoid receptor induces hypohidrotic ectodermal dysplasia. *Endocrinology* 146, 2629–2638  
48 Cui, C.Y. *et al.* (2003) Inducible mEDA-A1 transgene mediates sebaceous gland hyperplasia and differential formation of two types of mouse hair follicles. *Hum. Mol. Genet.* 12, 2931–2940  
49 Gaide, O. and Schneider, P. (2003) Permanent correction of an inherited ectodermal dysplasia with recombinant EDA. *Nat. Med.* 9, 614–618  
50 Mustonen, T. *et al.* (2004) Ectodysplasin A1 promotes placodal cell fate during early morphogenesis of ectodermal appendages. *Development* 131, 4907–4919  
51 Jaskoll, T. *et al.* (2003) Ectodysplasin receptor-mediated signaling is essential for embryonic submandibular salivary gland development. *Anat. Rec. A: Discov. Mol. Cell. Evol. Biol.* 271, 322–331  
52 Melnik, M. *et al.* (2009) Salivary gland branching morphogenesis: a quantitative systems analysis of the Eda/Edar/NFkappaB paradigm. *BMC Dev. Biol.* 9, 32  
53 Mikkola, M.L. (2011) The Edar subfamily in hair and exocrine gland development. *Adv. Exp. Med. Biol.* 691, 23–33  
54 Voutilainen, M. *et al.* (2012) Ectodysplasin regulates hormone-independent mammary ductal morphogenesis via NF-κB. *Proc. Natl. Acad. Sci. U.S.A.* 109, 5744–5749  
55 Tucker, A.S. *et al.* (2004) The activation level of the TNF family receptor, Edar, determines cusp number and tooth number during tooth development. *Dev. Biol.* 268, 185–194  
56 Kangas, A.T. *et al.* (2004) Nonindependence of mammalian dental characters. *Nature* 432, 211–214  
57 Harjunmaa, E. *et al.* (2012) On the difficulty of increasing dental complexity. *Nature* 483, 324–327  
58 Mou, C. *et al.* (2006) Generation of the primary hair follicle pattern. *Proc. Natl. Acad. Sci. U.S.A.* 103, 9075–9080  
59 Laurikkala, J. *et al.* (2002) Regulation of hair follicle development by the TNF signal ectodysplasin and its receptor Edar. *Development* 129, 2541–2553  
60 Zhang, Y. *et al.* (2009) Reciprocal requirements for EDA/EDAR/NF-kappaB and Wnt/beta-catenin signaling pathways in hair follicle induction. *Dev. Cell* 17, 49–61  
61 Härä, O. *et al.* (2012) Ectodysplasin regulates activator-inhibitor balance in murine tooth development through Fgf20 signaling. *Development* 139, 3189–3199  
62 Barrett, R.D. *et al.* (2008) Natural selection on a major armor gene in threespine stickleback. *Science* 322, 255–257  
63 Le Rouzic, A. *et al.* (2011) Strong and consistent natural selection associated with armour reduction in sticklebacks. *Mol. Ecol.* 20, 2483–2493  
64 Barrett, R.D. (2010) Adaptive evolution of lateral plates in three-spined stickleback *Gasterosteus aculeatus*: a case study in functional analysis of natural variation. *J. Fish Biol.* 77, 311–328  
65 Leinonen, T. *et al.* (2012) Multiple evolutionary pathways to decreased lateral plate coverage in freshwater threespine sticklebacks. *Evolution* 66, 3866–3875  
66 Kitano, J. *et al.* (2008) Reverse evolution of armor plates in the threespine stickleback. *Curr. Biol.* 18, 769–774  

67 Knecht, A.K. *et al.* (2007) Constraints on utilization of the EDA-signaling pathway in threespine stickleback evolution. *Evol. Dev.* 9, 141–154  
68 Barrett, R.D. *et al.* (2009) Should I stay or should I go? The Ectodysplasin locus is associated with behavioural differences in threespine stickleback. *Biol. Lett.* 5, 788–791  
69 Sabeti, P.C. *et al.* (2007) Genome-wide detection and characterization of positive selection in human populations. *Nature* 449, 913–918  
70 Carlson, C.S. *et al.* (2005) Genomic regions exhibiting positive selection identified from dense genotype data. *Genome Res.* 15, 1553–1565  
71 Williamson, S.H. *et al.* (2005) Simultaneous inference of selection and population growth from patterns of variation in the human genome. *Proc. Natl. Acad. Sci. U.S.A.* 102, 7882–7887  
72 Voight, B.F. *et al.* (2006) A map of recent positive selection in the human genome. *PLoS Biol.* 4, e72  
73 Tang, K. *et al.* (2007) A new approach for using genome scans to detect recent positive selection in the human genome. *PLoS Biol.* 5, e171  
74 Myles, S. *et al.* (2008) Identification and analysis of high Fst regions from genome-wide SNP data from three human populations. *Ann. Hum. Genet.* 72, 99–110  
75 Bryk, J. *et al.* (2008) Positive selection in East Asians for an EDAR allele that enhances NF-kappaB activation. *PLoS ONE* 3, e2209  
76 Fujimoto, A. *et al.* (2008) A replication study confirmed the EDAR gene to be a major contributor to population differentiation regarding head hair thickness in Asia. *Hum. Genet.* 124, 179–185  
77 Kimura, R. *et al.* (2009) A common variation in EDAR is a genetic determinant of shovel-shaped incisors. *Am. J. Hum. Genet.* 85, 528–535  
78 Park, J.H. *et al.* (2012) Effects of an Asian-specific nonsynonymous EDAR variant on multiple dental traits. *J. Hum. Genet.* 57, 508–514  
79 Mou, C. *et al.* (2008) Enhanced ectodysplasin-A receptor (EDAR) signaling alters multiple fiber characteristics to produce the East Asian hair form. *Hum. Mutat.* 29, 1405–1411  
80 Cluzeau, C. *et al.* (2012) The EDAR370A allele attenuates the severity of hypohidrotic ectodermal dysplasia caused by EDA gene mutation. *Br. J. Dermatol.* 166, 678–681  
81 Hoekstra, H.E. and Coyne, J.A. (2007) The locus of evolution: evo devo and the genetics of adaptation. *Evolution* 61, 995–1016  
82 Stern, D.L. and Orgogozo, V. (2008) The loci of evolution: how predictable is genetic evolution? *Evolution* 62, 2155–2177  
83 Casal, M.L. *et al.* (2007) Significant correction of disease after postnatal administration of recombinant ectodysplasin A in canine X-linked ectodermal dysplasia. *Am. J. Hum. Genet.* 81, 1050–1056  
84 Kowalczyk, C. *et al.* (2011) Molecular and therapeutic characterization of anti-ectodysplasin A receptor (EDAR) agonist monoclonal antibodies. *J. Biol. Chem.* 286, 30769–30779  
85 Christin, P-A. *et al.* (2010) Causes and evolutionary significance of genetic convergence. *Trends Genet.* 26, 400–405  
86 Martin, A. and Orgogozo, V. (2013) The loci of repeated evolution: a catalog of genetic hotspots of phenotypic variation. *Evolution* 67, 1235–1250  
87 Pantalacci, S. *et al.* (2008) Conserved features and evolutionary shifts of the EDA signaling pathway involved in vertebrate skin appendage development. *Mol. Biol. Evol.* 25, 912–928  
88 Wiens, G.D. and Glenney, G.W. (2011) Origin and evolution of TNF and TNF receptor superfamilies. *Dev. Comp. Immunol.* 35, 1324–1335  
89 Dietrich, M.R. (2003) Richard Goldschmidt: hopeful monsters and other ‘heresies’. *Nat. Rev. Genet.* 4, 68–74  
90 Lexner, M.O. *et al.* (2008) X-linked hypohidrotic ectodermal dysplasia. Genetic and dental findings in 67 Danish patients from 19 families. *Clin. Genet.* 74, 252–259  
91 Clauss, F. *et al.* (2008) Dento-craniofacial phenotypes and underlying molecular mechanisms in hypohidrotic ectodermal dysplasia (HED): a review. *J. Dent. Res.* 87, 1089–1099  
92 Charles, C. *et al.* (2009) Distinct impacts of Eda and Edar loss of function on the mouse dentition. *PLoS ONE* 4, e4985  
93 Nikopelius, T. *et al.* (2013) Non-syndromic tooth agenesis associated with a nonsense mutation in ectodysplasin-A (EDA). *J. Dent. Res.* 92, 507–511  
94 Gomes Rodrigues, H. *et al.* (2013) Roles of dental development and adaptation in rodent evolution. *Nat. Commun.* 4, 2504 http://dx.doi.org/10.1038/ncomms3504
