
Biochimica et Biophysica Acta 1841 (2014) 204–213

Contents lists available at ScienceDirect

Biochimica et Biophysica Acta

journal homepage: www.elsevier.com/locate/bbalip

---

**Identification of genes and pathways involved in the synthesis of Mead acid (20:3n – 9), an indicator of essential fatty acid deficiency**

Ikuyo Ichi ${ }^{a}$, Nozomu Kono ${ }^{b}$, Yuka Arita ${ }^{b}$, Shizuka Haga ${ }^{a}$, Kotoko Arisawa ${ }^{a}$, Misato Yamano ${ }^{a}$, Mana Nagase ${ }^{a}$, Yoko Fujiwara ${ }^{a}$, Hiroyuki Arai ${ }^{b,*}$

${ }^{a}$ Graduate School of Humanities and Sciences, Ochanomizu University, Tokyo 112-8610, Japan  
${ }^{b}$ Graduate School of Pharmaceutical Sciences, University of Tokyo, Tokyo 113-0033, Japan

---

**ARTICLE INFO**

Article history:  
Received 11 June 2013  
Received in revised form 10 October 2013  
Accepted 24 October 2013  
Available online 31 October 2013  

Keywords:  
Mead acid  
Essential fatty acid deficiency  
Elov5  
Fads  
Polyunsaturated fatty acid  

---

**ABSTRACT**

In mammals, 5,8,11-eicosatrienoic acid (Mead acid, $20: 3 \mathrm{n}-9$ ) is synthesized from oleic acid during a state of essential fatty acid deficiency (EFAD). Mead acid is thought to be produced by the same enzymes that synthesize arachidonic acid and eicosapentaenoic acid, but the genes and the pathways involved in the conversion of oleic acid to Mead acid have not been fully elucidated. The levels of polyunsaturated fatty acids in cultured cells are generally very low compared to those in mammalian tissues. In this study, we found that cultured cells, such as NIH3T3 and Hepa1–6 cells, have significant levels of Mead acid, indicating that cells in culture are in an EFAD state under normal culture conditions. We then examined the effect of siRNA-mediated knockdown of fatty acid desaturases and elongases on the level of Mead acid, and found that knockdown of Elovl5, Fads1, or Fads2 decreased the level of Mead acid. This and the measured levels of possible intermediate products for the synthesis of Mead acid such as $18: 2 \mathrm{n}-9,20: 1 \mathrm{n}-9$ and $20: 2 \mathrm{n}-9$ in the knocked down cells indicate two pathways for the synthesis of Mead acid: pathway 1) $18: 1 \mathrm{n}-9 \rightarrow(\text { Fads2 }) \rightarrow 18: 2 \mathrm{n}-9 \rightarrow 20: 2 \mathrm{n}-9 \rightarrow(\text { Fads1 }) \rightarrow 20: 3 \mathrm{n}-9$ and pathway 2) $18: 1 \mathrm{n}-9 \rightarrow(\text { Elovl5 }) \rightarrow 20: 1 \mathrm{n}-9 \rightarrow(\text { Fads2 }) \rightarrow 20: 2 \mathrm{n}-9 \rightarrow(\text { Fads1 }) \rightarrow 20: 3 \mathrm{n}-9$.

© 2013 Published by Elsevier B.V.

---

**1. Introduction**

In mammals, polyunsaturated fatty acids (PUFAs) such as linoleic acid ($\mathrm{LA} ; 18: 2 \mathrm{n}-6$) and $\alpha$ -linolenic acid ($\mathrm{ALA} ; 18: 3 \mathrm{n}-3$) cannot be synthesized de novo, and therefore are essential fatty acids (EFAs) [1]. Once LA and ALA are adequately ingested and absorbed, they can be converted to arachidonic acid ($\mathrm{AA} ; 20: 4 \mathrm{n}-6$), eicosapentaenoic acid ($\mathrm{EPA} ; 20: 5 \mathrm{n}-3$), or docosahexaenoic acid ($\mathrm{DHA} ; 22: 6 \mathrm{n}-3$). Essential fatty acid deficiency (EFAD) occurs when less than 1–2% of total calories are provided from EFAs [2]. For example, EFAD is observed in patients with intestinal malabsorption of lipids [3] and in patients on total parenteral nutrition with no fat [4]. EFAD has been related to various clinical features such as impaired growth, skin lesions, infertility and kidney abnormalities [5–7].

EFAs and their downstream long-chain PUFAs (LC-PUFAs) are localized to cell membranes as phospholipid esters and play critical roles in regulating membrane structure, dynamics and permeability. In mammals, the PUFA composition affects many cellular processes [8,9] and the activities of membrane-associated enzymes that are sensitive to the biophysical properties of lipid membrane [10]. In addition, certain classes of PUFAs are precursors of powerful eicosanoid effectors such as prostaglandins and leukotrienes [11], and thus mediate numerous physiological and pathological processes. Disruptions in proper PUFA intake and metabolism are also associated with some diseases such as coronary artery disease, hypertension, diabetes and inflammatory disorders [12].

Dietary LA and ALA are metabolized further by fatty acid desaturases and elongases (Fig. 1). Fatty acid desaturases are encoded by the Fads1, Fads2 and Fads3 genes [13]. Fads1 converts $20: 4 \mathrm{n}-3$ to $20: 5 \mathrm{n}-3$ (EPA) and converts $20: 3 \mathrm{n}-6$ to $20: 4 \mathrm{n}-6$ (AA) by catalyzing the desaturation at the $\Delta 5$ position [14,15]. Fads2 converts $18: 2 \mathrm{n}-6$ to $18: 3 \mathrm{n}-6$ and converts $18: 3 \mathrm{n}-3$ (ALA) to $18: 4 \mathrm{n}-3$ by catalyzing the desaturation at the $\Delta 6$ position in the carbon backbone [16,17]. Fads2 also catalyzes the desaturation at the $\Delta 8$ position to convert $20: 2 \mathrm{n}-6$ to $20: 3 \mathrm{n}-6$ and to convert $20: 3 \mathrm{n}-3$ to $20: 4 \mathrm{n}-3$ [18]. Thus, Fads1 and Fads2 are pivotal enzymes for the synthesis of AA and EPA in mammals. Fads3 is a putative fatty acid desaturase gene due to its high degree of sequence homology with Fads2 (62%) and Fads1 (52%), but neither $\Delta 5$ nor $\Delta 6$ desaturase activity of Fads3 has been demonstrated experimentally [18,19]. The Elovl (elongation of very long chain fatty acids) gene family is composed of seven distinct fatty acid elongase subtypes (Elovl1–7) [18,20]. Elovl1 prefers saturated fatty

* Abbreviations: AA, Arachidonic acid; ALA, $\alpha$-Linolenic acid; DHA, Docosahexaenoic acid; EFAs, Essential fatty acids; EFAD, Essential fatty acid deficiency; Elovl, Elongation of very long chain fatty acids; EPA, Eicosapentaenoic acid; Fads, Fatty acid desaturases; FBS, Fetal bovine serum; LA, Linoleic acid; PUFAs, Polyunsaturated fatty acids

* Corresponding author at: Department of Health Chemistry, Graduate School of Pharmaceutical Sciences, University of Tokyo, Hongo 7-3-1, Bunkyo-ku, Tokyo 113-0033, Japan. Tel.: +81 3 5841 4720; fax: +81 3 3818 3173.

E-mail address: harai@mol.f.u-tokyo.ac.jp (H. Arai).

1388-1981/$-$ see front matter © 2013 Published by Elsevier B.V.

http://dx.doi.org/10.1016/j.bbalip.2013.10.013

< n-6 PUFA >
HOOC $\text{MWMWMW}$
Linoleic acid (18:2n-6)

Elovl5
$\downarrow$ Fads2 ($\Delta 6$)
HOOC $\text{MWVMWM}$

20:2n-6
Fads2 ($\Delta 8$)
$\downarrow$ Elovl5
HOOC $\text{MMWMWMW}$

18:3n-6
$\downarrow$ Fads1 ($\Delta 5$)
HOOC $\text{MMMMWMW}$

Arachidonic acid (20:4n-6)

< n-3 PUFA >
HOOC $\text{MWMWMW}$
α-Linolenic acid (18:3n-3)

$\downarrow$ Fads2 ($\Delta 6$)
HOOC $\text{MWVMWM}$

18:4n-3
Elovl5
$\downarrow$ Fads2 ($\Delta 8$)
HOOC $\text{MMWMWMW}$

20:3n-3
$\downarrow$ Fads1 ($\Delta 5$)
HOOC $\text{MMMMWMW}$

Eicosapentaenoic acid (20:5n-3)

Fig. 1. Pathway of metabolism and synthesis of n−6 and n−3 PUFAs in mammals.

acid as a substrate [20]. Elovl2 and Elovl5 are involved in the elongation of polyunsaturated acyl-CoAs. Elovl3, Elovl6, and Elovl7 can elongate saturated, monounsaturated, and polyunsaturated acyl-CoAs [20]. Elovl4 is involved in the elongation of very long saturated [20] and polyunsaturated acyl-CoAs [21].

In EFAD animals, 5,8,11-eicosatrienoic acid (Mead acid, 20:3n−9) is endogenously synthesized from oleic acid, and is detected in the plasma and tissues [22,23]. Because the major biochemical changes associated with EFAD decreased AA and increased Mead acid, the triene–tetraene ratio (Mead acid/AA) is a diagnostic indicator of EFAD [24]. Mead acid is thought to be used in biological membranes as a substitute for other PUFAs. In addition, the 5-lipoxygenase product of Mead acid is reported to act as a proinflammatory mediator [25]. However, the significance of Mead acid in an EFAD state is not well understood.

Although n−3, n−6, and n−9 PUFAs are thought to be metabolized by the same desaturase and elongase enzymes (Fig. 1) [26], evidence that this occurs *in vivo* is lacking. Two desaturases and one elongase are needed to synthesize Mead acid from oleic acid. In this study, we found that mammalian cells in culture are generally in an EFAD state because of the low content of PUFA in the culture medium, and that the cells synthesize Mead acid endogenously. We exploited this property to study the genes and the pathway involved in the synthesis of Mead acid using siRNA-mediated knockdown of fatty acid desaturases and elongases.

## 2. Materials and methods

### 2.1. Materials

All chemicals were purchased from Wako Pure Chemicals (Osaka, Japan) and Sigma-Aldrich (St. Louis, MO, USA). Lipid standards were obtained from Avanti Polar Lipids (Alabaster, AL, USA). A control siRNA and siRNAs directed against Elovl2, Elovl3, Elovl4, Elovl5, Elovl6, Elovl7, Fads1, and Fads2 were obtained from Nippon EGT (Toyama, Japan). siRNAs directed against Elovl1 and Fads3 were purchased from Invitrogen (Carlsbad, CA, USA). The intermediate productions from oleic acid to Mead acid were synthesized by mutant fungus, Mortierella alpina M209-7, derived from Δ12 desaturase-defective *M. alpina* [27].

### 2.2. Animals

Male C57BL/6 mice (8 weeks) were purchased from CLEA Japan, Inc. (Tokyo, Japan) and divided into two groups (*n* = 5). Mice were housed under controlled temperature and humidity and at a 12:12-h dark-light cycle. Control mice were fed a modified AIN93G diet which contains 7% of soybean oil by weight for 4 weeks. EFAD mice were fed a modified AIN93G diet which did not contain soybean oil. This study was carried out in strict accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals of the National Institutes of Health. The protocol was approved by the Committee on the Ethics of Animal Experiments of Ochanomizu University. Mice were anesthetized with pentobarbital sodium (60 mg/kg ip) and killed by collecting the blood from the heart using a syringe containing 5 IU/ml heparin (1000 IU/ml Novo-Heparin 5000, Mochida Pharmaceutical, Tokyo, Japan) as an anticoagulant. After perfusion of physiological saline through the heart, the liver was removed. Blood was centrifuged at 3000 × g for 10 min at 4 °C to separate plasma. The liver and plasma were stored at −80 °C until analysis.

### 2.3. Cell culture

Murine-derived NIH3T3, C2C12, and 3T3E1 cells were obtained from the American Type Culture Collection (ATCC). Murine-derived Hepa1–6 cells were provided by RIKEN BRC through the National Bio-Resource Project of MEXT, Japan. These cells were maintained in Dulbecco's modified Eagle's medium supplemented with 10% fetal bovine serum (FBS) and 100 U/ml penicillin, 100 mg/ml streptomycin, and 2 mM L-glutamine at 37 °C in a humidified atmosphere of 5% CO₂. All experiments were performed in the cell lines at passages 10–15. Cells from the same passage were divided into individual wells of 6-well culture dishes (1.0 × 10⁵ cells per well). Cells from individual wells were independently subjected to measurement of mRNA and fatty acids (*n* = 3 wells).

### 2.4. siRNA transfection

Cells were plated in 6-well culture dishes at a density of 1.0 × 10⁵ cells per well. Cells at about 70% confluency were transfected with individual siRNA in serum-free medium. The cells were transfected using Lipofectamine® RNAiMAX reagent (Invitrogen, Carlsbad, CA) according to the manufacturer's recommendations. For each transfection, 2.5 μl of Lipofectamine was mixed with 250 μl of Opti-MEM (Gibco-BRL/Invitrogen, Carlsbad, CA) and incubated for 5 min at room temperature. In a separate tube, siRNA (6–62.5 pmol) was added to 250 μl of Opti-MEM. Then, the solution was added to the Lipofectamine mixture. In order to allow complex formation, the siRNA mixture was incubated for an additional 20 min at room temperature. The solution was

Table 1  
Sequence of forward and reverse primers used for quantitative RT-PCR amplification.

| Elovl1 | Forward primers (5′ to 3′) | Reverse primers (5′ to 3′) |
|--------|-----------------------------|----------------------------|
|        | ACTTCGGATGGTTCGAGTGGCCTG   | CCGAGTGTTGGAAAGACATGGAGGA |
| Elovl2 | GTCTCTCAGGGGCATCCTCACCTTG  | CCGGACATCACCTTCTCCTGCACT  |
| Elovl3 | AGTGGGCCCTCAAGCAAACCGTG    | TGAGTGGAACGCTTACGCAGGATGA |
| Elovl4 | ATGTCCACGGCATTCAAACGACACC  | ACCCAGCCACACGAAACAGGAGATA |
| Elovl5 | GGTGTGTGGGAAGGCAAATAC      | TGCAGAAGGATGAAGAAAAAG     |
| Elovl6 | TCAGCAAAGCACCCCGAACTAGGTA  | ATGAACCAACCACCCCCAGCGA    |
| Elovl7 | AGCTCATGGAGAACCGGAAG       | GTACCCCAGCCAGACATCAC      |
| Fads1  | TGGTGCCCTTCATCCTCTGT       | GTACCCCAGCCAGACATCAC      |
| Fads2  | GACATAAAGAGCCTGCATGTGTTT   | GGGCAGGTATTTCCAGCTTCTTC   |
| Fads3  | ATGAAAAGCATCGGGGACTGG      | TTGGCATCGTGGGGAAG         |

transfection solution for 6 h. Thereafter, the Opti-MEM medium containing the complexes was replaced with 2 ml of standard growth media. Cells were harvested 72 h after transfection.

### 2.4. Total RNA isolation and quantitative real-time PCR

Total RNA was extracted from cells by using Isogen (Nippongene, Toyama, Japan) and was reverse transcribed using a High Capacity cDNA Reverse Transcription Kit (Applied Biosystems, Foster City, CA). Quantitative real-time PCR was performed using SYBR® Green PCR Master Mix (Applied Biosystems, Foster City, CA) and Light-Cycler 480 (Roche Diagnostics). Target gene expression was normalized on the basis of GAPDH content ([Table 1](#table-1)).

### 2.5. Lipid analysis and GC/MS analysis

Cells (1.0 × 10⁵) were harvested by scraping into 2 ml methanol. Subsequently, chloroform and water were added to make a chloroform/methanol/water mixture of 1:2:0.8, after which lipids were extracted from the cells by the method of Bligh and Dyer [28]. Lipids were extracted from plasma (100 μl) or livers (50 mg wet weight) with chloroform/methanol/water (1:2:0.8, v/v) by the technique of Bligh and Dyer [28]. Isolated lipids were methylated with 2.5% H₂SO₄ in methanol. The resulting fatty acid methyl esters were then extracted with hexane and quantified by gas chromatography–mass

![Figure 2](#fig-2)

Fig. 2. Fatty acid compositions of murine plasma, FBS and murine-derived cultured cells. (A) Control mice were fed a diet that contained 7% soybean oil as dietary fat and EFAD mice were fed a diet that did not contain oil. Fatty acid compositions of plasma in these mice and FBS were analyzed by GC–MS. Values represent means ± SD (mice, n = 5; FBS, n = 5). Statistically significant differences compared to control murine plasma are indicated (*p < 0.05, Tukey–Kramer post hoc test). (B) Murine-derived NIH3T3, Hepa1–6, C2C12 and 3T3E1 cells were grown to confluence in medium supplemented with 10% FBS. Values represent means ± SD from three independent experiments.

I. Ichi et al. / Biochimica et Biophysica Acta 1841 (2014) 204–213

![Figure](https://i.imgur.com/yourimageurl.png)

Fig. 3. Change in the level of Mead acid in cells and medium during culture of NIH3T3 and Hepa1–6 cells. Values represent means ± SD from three independent experiments. Statistically significant differences compared to 48 h are indicated (*p < 0.05, Tukey–Kramer post hoc test).

spectrometry using a GC/MS QP2010 (Shimadzu, Kyoto, Japan) equipped with a Varian capillary column (60 m × 0.25 mm × 0.25 μm; Agilent Inc., USA). In the oven temperature program, the temperature was initiated at 120 °C for 1 min and then raised to 175 °C at 25 °C/min and to 235 °C at 5 °C/min and held for 5 min. The injection temperature was set at 250 °C, the interface at 250 °C, and the ion source adjusted to 200 °C.

### 2.6. Statistical analysis

Values were presented as means ± SD. In cell culture experiments, three wells (n = 3) were subjected to measurement of mRNA or fatty acids. In animal experiments, five mice (n = 5) were used for the measurement of mRNA or fatty acid. Statistical analysis was performed by unpaired Student's *t* test or one-factor ANOVA of variance with Tukey–Kramer post hoc test to identify significant differences (*p* < 0.05).

### 3. Results

#### 3.1. Levels of Mead acid in murine plasma and cultured mammalian cells

We measured the level of Mead acid in mice upon feeding with EFAD diets for 4 weeks. As shown previously [1,2], the levels of n-6 and n-3 PUFAs such as 18:2n-6, 20:4n-6, 18:3n-3 and 20:5n-3 in the plasma of EFAD mice were significantly lower than those of control mice (Fig. 2A). Mead acid was not detectable in the plasma of mice fed with chow diet, while it accounted for about 2% of total fatty acids in the plasma of mice fed with EFAD diets. Plasma levels of oleic acid (18:1n-9), a precursor of Mead acid, were also higher in the EFAD mice than in control mice.

In general, cultured cells acquire EFAs from FBS added to the culture medium. Because EFAs are much less abundant in FBS than in control murine plasma and tissues (Fig. 2A, Supplementary Fig. 1), the level of EFAs is low in cells cultured in FBS-containing medium. In other words, cells in culture are in an EFA-deficient state. Indeed, we detected Mead acid in cultured cells even under normal culture conditions (Fig. 2B). The levels of Mead acid in NIH3T3 and Hepa1–6 cells increased with time, especially from 72 h to 96 h (Fig. 3A and B). The level of Mead acid in the medium of NIH3T3 was decrease at 72 h, but the levels did not significantly change from 72 h to 96 h in either cell type (Fig. 3A

I. Ichi et al. / Biochimica et Biophysica Acta 1841 (2014) 204–213

![A](https://i.imgur.com/1234567.png)

Hepa 1-6 cells

![B](https://i.imgur.com/7654321.png)

NIH3T3 cells

Fig. 4. Expression of fatty acid desaturases and elongases in NIH3T3 and Hepa1–6 cells. Data were normalized to GAPDH mRNA levels. Values represent means ± SD from three independent experiments.

and B). These results indicate that the cells in culture synthesize Mead acid, although a portion of Mead acid in the cells might be derived from FBS-containing medium.

### 3.2. Expression of fatty acid desaturases and elongases in cultured cells

Among known desaturase and elongase genes, Elovl1, Elovl2, Elovl3, Elovl5, Elovl7, Fads1, Fads2 and Fads3 genes were expressed in Hepa1–6 cells (Fig. 4A). NIH3T3 cells, in which the level of Mead acid was highest among cell lines tested (Fig. 2B), expressed Elovl1, Elovl4, Elovl5, Elovl6, Fads1, Fads2 and Fads3 genes (Fig. 4B). Among these genes, expressions of Elovl family were lower in Hepa1–6 cells than in NIH3T3 cells.

### 3.3. Effect of knockdown of fatty acid desaturases and elongases on Mead acid production

siRNA treatment efficiently reduced the mRNA levels of desaturase and elongase genes in both NIH3T3 cells and Hepa1–6 cells (Supplementary Fig. 2). The levels of Mead acid were greatly decreased by knockdown of Fads1 or Fads2 in both NIH3T3 and Hepa1–6 cells (Fig. 5A and B). Knockdown of Fads3 did not decrease the Mead acid levels in either cell type. Elovl2, which is active with C20 PUFA [20], was not detectable in NIH3T3 cells, but it was detectable in Hepa1–6 cells. Elovl2 knockdown did not affect the production of Mead acid in Hepa1–6 cells (Fig. 5B). In NIH3T3 and Hepa1–6 cells, the level of Mead acid was decreased by treatment with siRNA of Elovl5 (Fig. 5A and B).

Because Elovl6 specifically elongates saturated and monounsaturated fatty acids with 16 carbons [20], knockdown of Elovl6 may suppress the production of oleic acid, a precursor of Mead acid. In fact, Elovl6 knockdown significantly decreased the levels of oleic acid and Mead acid in NIH3T3 cells (Fig. 5A), but the level of Mead acid could be recovered by addition of oleic acid (Fig. 6). These results indicate that Elovl6 is involved mainly in the synthesis of oleic acid and not in the conversion from oleic acid to Mead acid. Taken together, these findings indicate that Elovl5, Fads1 and Fads2 are involved in the synthesis of Mead acid from oleic acid.

### 3.4. Identification of the synthetic pathway from oleic acid to Mead acid in NIH3T3 cells

There are two possible pathways by which Mead acid can be synthesized from oleic acid (Fig. 7). In NIH3T3 cells, the level of 18:2n–9 was greatly decreased by Fads2 knockdown, and greatly increased by Elovl5 knockdown (Fig. 8A), while the level of 20:2n–9 was greatly increased by Fads1 knockdown (Fig. 8B). These results indicate that at least some of the Mead acid is synthesized by pathway 1. The decrease in the level of 20:1n–9 by Elovl5 knockdown (Fig. 8C) suggests that Elovl5 also catalyzes the conversion of 20:1n–9 from oleic acid. In addition, the level of 20:1n–9 was significantly increased by knockdown of Fads1 or Fads2. These results indicate that Mead acid is also synthesized through pathway 2. The increase of 18:2n–9 by Elovl5 knockdown was much higher than that of 20:1n–9 by Fads2 knockdown. In addition, the decrease of 18:2n–9 by Fads2 knockdown was much higher than that of 18:2n–9 by Elovl5 knockdown. These results indicate that Mead acid was mainly synthesized through pathway 1 in NIH3T3 cells.

To confirm that Elovl5 elongates 18:1n–9 to 20:1n–9, we measured *in vitro* elongation activity using the membrane fraction obtained from siElovl5-treated NIH3T3 cells. As shown in Fig. 9, the elongation activity towards 18:2n–6-CoA, which is a good substrate for Elovl5 [20], was greatly reduced in the membrane fraction by Elovl5 siRNA.

I. Ichi et al. / Biochimica et Biophysica Acta 1841 (2014) 204–213

**A**
![Diagram](https://i.imgur.com/1.png)

**B**
![Diagram](https://i.imgur.com/2.png)

Fig. 5. Effect of siRNA-mediated knockdown of desaturase and elongase genes on Mead acid level. The levels of Mead acid were measured in cells following treatment with control siRNA (siControl) or siRNA of the Elovl and Fads genes. Values represent means ± SD from three independent experiments. Statistically significant differences compared to siControl cells are indicated (*p < 0.05, the Tukey–Kramer post hoc test).

Elongation activity towards 18:1n−9-CoA was also reduced by Elovl5 siRNA, indicating that Elovl5 can use 18:1n−9-CoA as well as 18:2n−6-CoA as a substrate *in vivo*. Moreover, the elongation activity of Elovl5 was greater towards 18:1n−9-CoA (siControl minus siElovl5 in Fig. 9A) than towards 18:2n−6-CoA (siControl minus siElovl5 in Fig. 9B).

![Diagram](https://i.imgur.com/3.png)

Fig. 6. Effect of addition of oleic acid on the level of Mead acid in knockdown cells of Elovl6. Oleic acid (35 μM) was added to NIH3T3 cells at 48 h after Elovl6 siRNA transfection. After 24 h incubation, the levels of oleic acid and Mead acid were measured. Values represent means ± SD from three independent experiments. Statistically significant differences compared to siControl cells are indicated (*p < 0.05, Tukey–Kramer post hoc test).

### 3.5. Detection of intermediate fatty acids in the liver of EFAD mice

To examine the pathway for Mead acid synthesis in mice, we measured the level of the intermediate fatty acids in the liver of EFAD mice. In addition to Mead acid, intermediate fatty acids such as 20:1n−9 and 20:2n−9 were detected in the liver of the EFAD mice, and the levels of these fatty acids were increased by EFAD (Fig. 10A). These results, together with the finding that Elovl5, Fads1 and Fads2 were expressed in the liver (Fig. 10B), suggest that Mead acid is synthesized by pathway 2 in the liver of EFAD mice. 18:2n−9, an intermediate fatty acid of pathway 1 was also detected in the liver, although the level was very low compared to those of 20:1n−9 and 20:2n−9. It is possible that 18:2n−9 produced from 18:1n−9 by Fads2 is efficiently converted to 20:2n−9 and 20:3n−9 through pathway 1 in the murine liver. It is well known that all Fads1, Fads2 and Elovl5 genes are suppressed by dietary PUFAs [29–32]. The mRNA levels of Elovl5 and Fads2 in the liver were increased in the EFAD state as expected, while significant increase of Fads1 was not observed under the present conditions (Fig. 10B).

### 4. Discussion

In this study, we investigated the genes and pathways involved in Mead acid synthesis in cultured cells by siRNA-mediated knockdown of selected genes. Although n−9 PUFAs including Mead acid are thought to be synthesized by the same enzymes involved in the synthesis of n−3 and n−6 PUFAs [25], direct experimental evidence of this is lacking. We found that fatty acid elongase Elovl5 and fatty acid

Pathway1
HOOC / Oleic acid (18:1n-9)
↓ Fads2 (Δ6)
HOOC / 18:2n-9
↓ Elovl5
HOOC / 20:2n-9
↓ Fads1 (Δ5)
HOOC / Mead acid (20:3n-9)

Pathway2
HOOC / Oleic acid (18:1n-9)
↓ Elolv5
HOOC / 20:1n-9
↓ Fads2 (Δ8)
HOOC / 20:2n-9
↓ Fads1 (Δ5)
HOOC / Mead acid (20:3n-9)

Fig. 7. Expected pathways for synthesis of Mead acid from oleic acid.

desaturases Fads1 and Fads2 were involved in the synthesis of Mead acid from oleic acid and that two pathways are involved:
i) 18:1n−9 → (Fads2) → 18:2n−9 → (Elovl5) → 20:2n−9 → (Fads1) → 20:3n−9 and ii) 18:1n−9 → (Elovl5) → 20:1n−9 → (Fads2) → 20:2n−9 → (Fads1) → 20:3n−9. These results indicate that the enzymes for desaturation and elongation of n−3 and n−6 PUFAs also participate in the synthesis of n−9 PUFA Mead acid, and that the expressions of other enzymes do not appear to be induced in the EFAD state. Fads2 has both Δ8-desaturase and Δ6-desaturase activities [33]. Therefore, the synthesis of LC-PUFAs such as EPA and ARA from 18:3n−3 and 18:2n−6 in mammals proceeds through the ‘classical’ Δ6-pathway (Δ6 desaturation → elongation → Δ5 desaturation), or alternatively through the Δ8-pathway (elongation → Δ8 desaturation → Δ5 desaturation) (see Fig. 1). Our finding that the level of 20:1n−9 increased in cells following treatment with Fads2 siRNA indicates that the Δ8-desaturase activity of Fads2 is involved in the synthesis of Mead acid. The level of 18:1n−9 was not appreciably increased by Fads2 siRNA, possibly because of the large pool size of 18:1n−9 in the cells. However, the presence of 18:2n−9 in Elovl5 siRNA-treated cells (see Fig. 8)

A
1
18:2n-9
* 
0.8
0.6
0.4
0.2
0
siControl siElovl5 siFads1 siFads2

B
5
20:2n-9
*
4
3
2
1
0
siControl siElovl5 siFads1 siFads2

C
0.5
20:1n-9
*
0.4
0.3
0.2
0.1
0
siControl siElovl5 siFads1 siFads2

Fig. 8. Effects of knockdown of Elovl5, Fads1 and Fads2 on intermediates in the synthetic pathway for Mead acid. The intermediate products such as 18:2n−9 (A), 20:1n−9 (B), and 20:2n−9 (C) were measured in NIH3T3 cells following treatment with siRNA of Elovl5, Fads1 and Fads2. Values represent means ± SD from three independent experiments. Statistically significant differences compared to siControl cells are indicated (*p < 0.05, Tukey–Kramer post hoc test).

I. Ichi et al. / Biochimica et Biophysica Acta 1841 (2014) 204–213

Fig. 9. Elongation activity towards 18:1n−9-CoA in knockdown cells of Elovl5. Total membrane proteins (20 μg protein) in siControl and siElovl5 cells were incubated with 18:1n−9 or 18:2n−6-CoA (50 μM) and 0.075 μCi [14C]malonyl-CoA for 30 min at 37 °C. After termination of the reactions, lipids in the reaction mixture were saponified, acidified and extracted. After separating with TLC, radiolabeled fatty acids were measured with a liquid scintillation counter. Values presented are [14C]FA levels and represent means ± SD from three independent experiments. Statistically significant differences compared to siControl cells are indicated (*p < 0.05, Student's t test).

indicates an involvement of the Δ6-desaturase activity of Fads2 in the conversion of 18:1n−9 to 18:2n−9.

Mead acid was previously detected in some fish cell lines cultured in FBS-containing medium, which shows a sign of EFAD [34]. Moreover, involvement of Elovl5 and Fads1 has been suggested in the synthesis of Mead acid in fish cell lines from the following observations [34]. TF cells, which express very low levels of C18 elongase, have low amounts of Mead acid and high amounts of 18:2n−9 compared with PLHC-1 cells that synthesize high amounts of Mead acid. Another fish cell line CHSE-214, which expresses very low levels of Δ5 desaturase towards n−9 fatty acids, has low amount of Mead acid and high amount of 20:2n−9 compared with PLHC-1 cells. These results support the idea that Mead acid is synthesized by pathway 1. The presence of pathway 2 cannot be inferred from their study, because 20:1n−9 was not measured.

Ohno et al. previously showed that overexpression of Elovl5 did not increase the in vitro elongation activity of 18:1n−9-CoA, indicating that

18:1n−9-CoA was not a substrate of Elovl5. However, our observation that knockdown of Elovl5 significantly decreased the in vitro elongase activity towards 18:1n−9 (Fig. 9) and the level of C20:1n−9 in NIH3T3 cells (Fig. 8) indicates that Elovl5 can convert 18:1n−9 to C20:1n−9. At present, we cannot explain the apparent discrepancy between these studies. Even when the mRNA level of Elovl5 was decreased by more than 90%, the level of Mead acid was decreased by only 50% in NIH3T3 cells. This result raises the possibility that other elongase(s) participate in the synthesis of Mead acid from oleic acid. Among the intermediates, relatively large amounts of 20:1n−9 were detected in NIH3T3 cells as well as in the liver of mice fed the control diet. Thus, it is possible that the conversion of 18:1n−9 to 20:1n−9 occurs at certain levels even in the presence of EFA. 20:1n−9 is also an intermediate for the synthesis of very long-chain monounsaturated fatty acids such as C22:1 and C24:1, both of which are components of sphingolipids, and thus may be synthesized even under normal conditions. Among Elovl family, Elovl3 and Elovl7 can convert 18:1n−9 to 20:1n−9 in vitro

Fig. 10. The levels of the intermediate fatty acids and the synthetic enzymes for Mead acid in the murine liver. Control mice were fed a diet that contained 7% soybean oil as dietary fat and EFAD mice were fed a diet that did not contain oil. (A) The intermediate fatty acids from oleic acid to Mead acid in the liver of these control and EFAD mice were analyzed by GC–MS. Values represent means ± SD from five independent mice. Statistically significant differences compared to control murine liver are indicated (*p < 0.05, Student's t test). (B) The mRNA levels of each gene were normalized relative to GAPDH and shown relative to control murine liver. Values represent means ± SD from five independent mice. Statistically significant differences compared to control murine liver are indicated (*p < 0.05, Student's t test).
[20,35]. Thus, these Elovl may participate in the synthesis of 20:1n—9 as an intermediate for the synthesis of Mead acid, although the expression levels of these genes were very low in both NIH3T3 cells and Hepa1–6 cells. Elovl5, which is expressed ubiquitously in murine tissues [20], may participate in the synthesis of 20:1n—9 as an intermediate for the synthesis of very long-chain monounsaturated fatty acids.

Elovl1, which is ubiquitously expressed in mammalian tissues, was expressed at relatively high levels in both NIH3T3 cells and Hepa1–6 cells. Knockdown of Elovl1 did not affect the synthesis of Mead acid in either cell type, which is consistent with the consensus idea that Elovl1 participates in the synthesis of very long-chain saturated fatty acids [20]. Fads3, which was isolated as a homologous gene of Fads1 and Fads2, was also expressed in both cell types. Very recently, Rioux et al. showed that rat Fads3 displays no common Δ5-, Δ6- orΔ9-desaturase activity but is able to catalyze the unexpected Δ13-desaturation of trans-vaccenate [36]. In our study, Mead acid level was not changed by Fads3 knockdown, suggesting that Fads3 does not participate in the synthesis of n—9 PUFAs.

We identified the genes involved in Mead acid synthesis by taking advantage of the fact that PUFA levels are much lower in mammalian cell cultures than in mammalian tissues. For example, cultured Hepa1–6 cells, a murine hepatoma cell line, contained very low amounts of PUFAs such as LA, AA, and DHA compared to murine livers (Supplementary Fig. 1). When mice were fed with the EFAD diet for 4 weeks, these PUFAs were decreased to the levels similar to those of cultured Hepa1–6 cells, indicating that Hepa1–6 cells are in an EFAD state. Indeed, cultured hepatoma cells synthesized Mead acid and were similar to mice in an EFAD state. In general, cultured cells obtain EFAs from FBS added to the culture medium. The LA and ALA contents in FBS are much lower than those in control murine plasma and liver. EFAD in cultured cells may induce the cells to synthesize Mead acid from oleic acid.

The contents of Mead acid as well as other PUFAs were different between NIH3T3 and Hepa 1–6 cells, even though these cells were cultured with the same medium containing FBS (Fig. 2). The level of Mead acid in Hepa1–6 cells was much lower than that in NIH3T3 cells, although the level of oleic acid, a precursor of Mead acid, in Hepa1–6 cells was approximately half of that in NIH3T3 cells. Expression of Elovl5, which converts 18:1n—9 to 20:1n—9, was lower in Hepa 1–6 cells than in NIH3T3 cells, which may explain the lower level of Mead acid in Hepa 1–6 cells than in NIH3T3 cells. The levels of LA and AA, a downstream product of LA, in Hepa1–6 cells were also lower than those in NIH3T3 cells. Since LA has to be taken up from the culture medium, these results suggest that the activity of LA uptake in Hepa 1–6 cells was also lower than that in NIH3T3 cells. Thus, synthetic activity of PUFA including Mead acid and the cellular uptake of PUFA from the culture medium appears to depend on the type of cells.

In conclusion, our results reveal two synthesis pathways for Mead acid from oleic acid. This study also provides an approach to elucidate the biological role of Mead acid in an EFAD state at the cellular level. We are now attempting to determine the role of Mead acid in cultured cells.

## Acknowledgements

This work was supported by Grant-in-Aid for Scientific Research C (No. 23617004 to I.I.) and Grants-in-Aid for Scientific Research S (No. 20370045 to H.A.) from the Japanese Ministry of Education, Culture, Sports, Science, and Technology from the Ministry of Education, Culture, Sports, Science and Technology in Japan.

## Appendix A. Supplementary figures

Supplementary data to this article can be found online at http://dx.doi.org/10.1016/j.bbalip.2013.10.013.

# References

[1] A.J. Fulco, J.F. Mead, Metabolism of essential fatty acid, VIII. Origin of 5,8,11-eicosatrienoic acid in the fat-deficient rat, J. Biol. Chem. 234 (1959) 1411–1416.

[2] E. Aaes-Jorgensen, E.E. Leppik, H.W. Hayes, R.T. Holman, Essential fatty acid deficiency: II. In adult rats, J. Nutr. 66 (1958) 245–259.

[3] M. Press, H. Kikuchi, G.R. Thompson, Essential fatty acid deficiency secondary to intestinal malabsorption, Gut 13 (1972) 837.

[4] F.M. Martins, A. Wennberg, S. Meurling, R. Kihlberg, L. Lindmark, Serum lipids and fatty acid composition of tissues in rats on total parenteral nutrition (TPN), Lipids 19 (1984) 728–733.

[5] E. Aaes-Jorgensen, R.T. Holman, Essential fatty acid deficiency: content of polyenoic acids in testes and heart as an indicator of essential fatty acid status, J. Nutr. 65 (1958) 633–641.

[6] A.A. Spector, M.A. Yorek, Membrane lipid composition and cellular function, J. Lipid Res. 26 (1985) 1015–1035.

[7] S.M. Innis, Essential fatty acids in growth and development, Prog. Lipid Res. 130 (1991) 39–103.

[8] S. Chyb, P. Raghu, R.C. Hardie, Polyunsaturated fatty acids activate the Drosophila light-sensitive channels TRP and TRPL, Nature 397 (1999) 255–259.

[9] Y.F. Xiao, Q. Ke, S.Y. Wang, K. Auktor, Y. Yang, G.K. Wang, J.P. Morgan, A. Leaf, Single point mutations affect fatty acid block of human myocardial sodium channel alpha subunit Na+ channels, Proc. Natl. Acad. Sci. U.S.A. 98 (2001) 3606–3611.

[10] W.E. Lands, Metabolism of glycerolipides; a comparison of lecithin and triglyceride synthesis, J. Biol. Chem. 231 (1958) 883–888.

[11] C.D. Funk, Prostaglandins and leukotrienes: advances in eicosanoid biology, Science 294 (2001) 1871–1875.

[12] A.P. Simopoulos, Essential fatty acids in health and chronic disease, Am. J. Clin. Nutr. 70 (1999) 560S–569S.

[13] A. Marquardt, H. Stohr, K. White, B.H. Weber, cDNA cloning, genomic structure, and chromosomal localization of three members of the human fatty acid desaturase family, Genomics 66 (2000) 175–183.

[14] L.V. Michaelson, J.A. Napier, M. Lewis, G. Griffiths, C.M. Lazarus, A.K. Stobart, Functional identification of a fatty acid delta5 desaturase gene from Caenorhabditis elegans, FEBS Lett. 439 (1998) 215–218.

[15] H.P. Cho, M. Nakamura, S.D. Clarke, Cloning, expression, and fatty acid regulation of the human D5-desaturase, J. Biol. Chem. 274 (1999) 37335–37339.

[16] H.P. Cho, M.T. Nakamura, S.D. Clarke, Cloning, expression, and nutritional regulation of the mammalian D6-desaturase, J. Biol. Chem. 274 (1999) 471–477.

[17] J.A. Napier, S.J. Hey, D.J. Lacey, P.R. Shewry, Identification of a Caenorhabditis elegans D6-fatty-acid-desaturase by heterologous expression in Saccharomyces cerevisiae, Biochem. J. 330 (1998) 611–614.

[18] H. Guillou, D. Zadravec, P.G. Martin, A. Jacobsson, The key roles of elongases and desaturases in mammalian fatty acid metabolism: insights from transgenic mice, Prog. Lipid Res. 49 (2010) 186–199.

[19] H. Blanchard, P. Legrand, F. Pédrono, Fatty Acid Desaturase 3 (Fads3) is a singular member of the Fads cluster, Biochimie (2011) 87–90.

[20] Y. Ohno, S. Suto, M. Yamanaka, Y. Mizutani, S. Mitsutake, Y. Igarashi, T. Sassa, A. Kihara, ELOVL1 production of C24 acyl-CoAs is linked to C24 sphingolipid synthesis, Proc. Natl. Acad. Sci. U.S.A. 107 (2010) 18439–18444.

[21] P. Barabas, A. Liu, W. Xing, C.K. Chen, Z. Tong, C.B. Watt, B.W. Jones, P.S. Bernstein, D. Križaj, Role of ELOVL4 and very long-chain polyunsaturated fatty acids in mouse models of Stargardt type 3 retinal degeneration, Proc. Natl. Acad. Sci. U.S.A. 110 (2013) 5181–5186.

[22] J.F. Mead, W.H. Slaton Jr., Metabolism of essential fatty acids, III. Isolation of 5,8,11-eicosatrienoic acid from fat-deficient rats, J. Biol. Chem. 219 (1956) 705–709.

[23] J.F. Mead, The metabolism of the essential fatty acids, Am. J. Clin. Nutr. 6 (1958) 656–661.

[24] R.T. Holman, The ratio of trienoic:tetraenoic acids in tissue lipids as a measure of essential fatty acid requirement, J. Nutr. 70 (1960) 405–410.

[25] P. Patel, C. Cossette, J.R. Anumolu, S. Gravel, A. Lesimple, O.A. Mamer, J. Rokach, W.S. Powell, Structural requirements for activation of the 5-oxo-6E,8Z, 11Z,14Z-eicosatetraenoic acid (5-oxo-ETE) receptor: identification of a Mead acid metabolite with potent agonist activity, J. Pharmacol. Exp. Ther. 325 (2008) 698–707.

[26] H.D. Le, J.A. Meisel, V.E. de Meijer, K.M. Gura, M. Puder, The essentiality of arachidonic acid and docosahexaenoic acid, Prostaglandins Leukot. Essent. Fatty Acids 81 (2009) 165–170.

[27] E. Sakuradani, N. Kamada, Y. Hirano, M. Nishihara, H. Kawashima, K. Akimoto, K. Higashiyama, J. Ogawa, S. Shimizu, Production of 5,8,11-eicosatrienoic acid by a delta5 and delta6 desaturation activity-enhanced mutant derived from a delta12 desaturation activity-defective mutant of Mortierella alpina 1S-4, Appl. Microbiol. Biotechnol. 60 (2002) 281–287.

[28] E.G. Bligh, W.J. Dyer, A rapid method of total lipid extraction and purification, Can. J. Biochem. Physiol. 37 (1959) 911–917.

[29] H.P. Cho, M.T. Nakamura, S.D. Clarke, Cloning, expression, and nutritional regulation of the mammalian delta-6 desaturase, J. Biol. Chem. 274 (1999) 471–477.

[30] H.P. Cho, M. Nakamura, S.D. Clarke, Cloning, expression, and fatty acid regulation of the human Δ-5 desaturase, J. Biol. Chem. 274 (1999).

[31] R.O. Peluffo, A.M. Nervi, R.R. Brenner, Linoleic acid desaturation activity of liver microsomes of essential fatty acid deficient and sufficient rats, Biochim. Biophys. Acta 441 (1976) 25–31.

[32] Y. Wang, D. Botolin, B. Christian, J. Busik, J. Xu, D.B. Jump, Tissue-specific, nutritional, and developmental regulation of rat fatty acid elongases, J. Lipid Res. 54 (2006) 706–715.
[33] W.J. Park, K.S. Kothapalli, P. Lawrence, C. Tyburczy, J.T. Brenna, An alternate pathway to long-chain polyunsaturates: the FADS2 gene product delta8-desaturates 20: 2n—6 and 20:3n—3, J. Lipid Res. 50 (2009) 1195–1202.

[34] D.R. Tocher, J.R. Dick, J.R. Sargent, Development of an in vitro model of essential fatty acid deficiency in fish cells, Prostaglandins Leukot. Essent. Fatty Acids 53 (1995) 365–375.

[35] T. Naganuma, Y. Sato, T. Sassa, Y. Ohno, A. Kihara, Biochemical characterization of the very long-chain fatty acid elongase ELOVL7, FEBS Lett. 585 (2011) 3337–3341.

[36] V. Rioux, F. Pedrono, H. Blanchard, C. Duby, N. Boulie-Monthean, L. Bernard, E. Beauchamp, D. Catheline, P. Legrand, Trans-vaccenate is Δ13-desaturated by FADS3 in rodents, J. Lipid Res. (2013)(in press).
