
Int. J. Mol. Sci. 2013, 14, 22117-22131; doi:10.3390/ijms141122117

**OPEN ACCESS**

**International Journal of Molecular Sciences**
ISSN 1422-0067
www.mdpi.com/journal/ijms

---

**Review**

# The Ribonucleoprotein Csr Network

**Ethel Seyll and Laurence Van Melderen \***

Laboratoire de Génétique et Physiologie Bactérienne, IBMM, Faculté des Sciences, Université Libre de Bruxelles (ULB), 12 rue des Professeurs Jeener et Brachet, B-6041 Gosselies, Belgium; E-Mail: eseyll@ulb.ac.be

* *Author to whom correspondence should be addressed; E-Mail: lvmelder@ulb.ac.be; Tel.: +32-2-650-9778; Fax: +32-2-650-9770.*

Received: 18 July 2013; in revised form: 21 October 2013 / Accepted: 28 October 2013 / Published: 8 November 2013

---

**Abstract:** Ribonucleoprotein complexes are essential regulatory components in bacteria. In this review, we focus on the carbon storage regulator (Csr) network, which is well conserved in the bacterial world. This regulatory network is composed of the CsrA master regulator, its targets and regulators. CsrA binds to mRNA targets and regulates translation either negatively or positively. Binding to small non-coding RNAs controls activity of this protein. Expression of these regulators is tightly regulated at the level of transcription and stability by various global regulators (RNAses, two-component systems, alarmone). We discuss the implications of these complex regulations in bacterial adaptation.

**Keywords:** Csr network; Rsm network; global regulation; RNA-binding protein; sRNAs; post-transcriptional regulation

---

## 1. Introduction

### 1.1. A Plethora of Ribonucleoprotein Complexes in Bacteria

Ribonucleoprotein complexes are major players in gene expression regulation. The paradigm for ribonucleoprotein complexes is represented by ribosomes. Translation-competent ribosomes are formed by both stable and transient associations between various types of RNAs (rRNAs, tRNAs, mRNAs, tmRNA) and r-proteins (ribosomal proteins) (for review, see [1]). In addition, some

r-proteins are involved in feedback regulations. Under specific conditions such as starvation, these proteins negatively regulate their own translation by binding to their encoding mRNA [2].

Other ribonucleoprotein complexes involve the association of small RNAs (sRNAs) with specific proteins. The 6S RNA interacts with σ<sup>70</sup>-containing RNA polymerase and regulates transcription at specific promoters [3]. The 4.5S RNA, a component of the Signal Recognition Particle (SRP), is essential for targeting signal peptide-bearing proteins to inner membrane (for review, see [4]). Another type relies on the association of RNA chaperones, such as Hfq, with regulatory sRNAs to facilitate interaction with their mRNA targets (for review, see [5]). Other RNA chaperones play important roles under specific conditions or in specific systems, such as the cold-shock CspA protein, which destabilizes mRNA structures and thereby facilitates translation at low temperature [6]. Ribonucleases (RNases) also associate transiently with various RNA species. For example, RNaseP and RNaseE are involved in mRNAs and sRNAs maturation, function, and decay (for review, see [7]).

In this review, we focus on a third type of ribonucleoprotein complexes, which rely on the association of a regulatory protein directly with target mRNAs to modulate their translation. We will review the Csr (carbon storage regulator) network in which the CsrA regulatory protein plays a pivotal role. This network is particularly interesting as it involves sRNAs that regulate CsrA activity. We discuss in detail the various ribonucleoprotein complexes involving CsrA as well as the Csr network roles in bacterial physiology.

## 2. The Csr Network

### 2.1. CsrA Is a Global Regulator

The first component of the Csr network was described in 1993 in *E. coli* [8]. In a screen to identify genes involved in glycogen biosynthesis by transposon mutagenesis, the group of Romeo identified the *csrA* gene as encoding a negative regulator of glycogen accumulation. Subsequently, it was shown that CsrA is a 61 amino acid protein that regulates translation by binding to mRNA targets [9,10]. In addition to its implication in glycogen synthesis regulation, CsrA regulates central carbon metabolism. The observation that expression of the *pckA* gene, using a *pckA::lacZ* translational fusion, was increased in the *csrA::kan* mutant provided the first evidence of *csrA* implication in gluconeogenesis regulation [8]. *pckA* encodes phosphoenolpyruvate carboxykinase, a key regulatory enzyme in this pathway. Later on, in a study evaluating glycolysis and gluconeogenesis enzyme activities, gene expression and metabolites, it was inferred that CsrA has a global positive effect on glycolysis and a global negative effect on gluconeogenesis [11]. Accordingly, glucose consumption and extracellular levels of acetate were shown to be reduced in *csrA* depletion conditions [12]. Expression of the *acs* (acetyl-CoA synthetase) and *aceA* (isocitrate lyase) is positively regulated by CsrA [13]. In addition, ATP level is reduced while AMP and ADP levels are increased, indicating a lower energy charge [11]. In a recent paper, global changes upon *csrA* depletion (using overexpression of the sRNA CsrB, see below) were monitored using proteomics and metabolomics approaches [12]. This study confirmed that central carbon metabolism is generally affected in these conditions. Intermediates of glycolysis, phosphoenolpyruvate (PEP), acetyl-CoA and intermediates of the glyoxylate shunt are accumulating. Amino acid and fatty acid metabolisms are also impacted [12]. In species other than *E. coli*, such as

Int. J. Mol. Sci. 2013, 14

Salmonella, deletion of *csrA* was also shown to affect metabolism, i.e., maltose transport, ethanolamine utilization, and propanediol metabolism [14].

Thus, a defect in *csrA* drastically alters carbon flux distribution. Effects on gene expression have been shown for a few enzymes, raising the possibility that most of the effects might be consequences of accumulation of specific intermediates that modify enzyme activity and redirect carbon into other metabolites.

In addition to its role in central carbon metabolism, CsrA controls the switch between sessile (biofilm) and planktonic lifestyle [15]. The *csrA::kan* mutation increases adherence [8] while it drastically reduces motility. Several direct regulations mediated by CsrA are involved in this switch. CsrA negatively regulates *pgaABCD* expression which contributes to β-1,6-*N*-acetyl-D-glucosamine (PGA) synthesis [16], an exopolysaccharide involved in adhesion [17]. CsrA positively regulates *flhDC* expression, encoding the flagellar master regulator [18,19]. In addition, the level of c-di-GMP, a signaling molecule that notably controls biofilm formation and motility, is affected in the *csrA::kan* mutant (for review, see [20]). CsrA negatively regulates the expression of seven genes encoding proteins with GGDEF and/or EAL domains (responsible for diguanylate cyclase and phosphodiesterase activities, respectively) [21].

CsrA is also involved in bacterial interactions with animal and plant hosts (for reviews, see [22–24]). For example, CsrA is involved in *Salmonella typhimurium* invasion of epithelial cells and survival inside macrophages [25]. The CsrA homologues in the *Pectobacterium* genus (RsmA) play a crucial role in infection and colonization of host plants [26]. In the opportunistic human pathogen *P. aeruginosa*, RsmA is involved in the control of a wide variety of processes involved in pathogenesis, such as hydrogen cyanide synthesis [27], type IV pili expression [28], quorum sensing [29] as well as type VI and type III secretion systems [28]. In the plant-beneficial root-colonizing strain *P. fluorescens*, RsmA and its homologue RsmE, control synthesis of extracellular antifungal secondary metabolites as well as expression of the exoenzymes AprA and phospholipase C [30,31].

As described above, the *csrA::kan* mutant shows drastic phenotypes although this mutant does not display any growth defect. It is able to grow on rich medium (LB) as well as on minimal medium supplemented with gluconeogenic or glycolytic carbon sources [8]. The *kan* transposon is inserted at codon 51 in the *csrA* gene [8], leaving the possibility to produce a truncated CsrA protein that is still partially active [19,32]. This mutant is the basis for CsrA biological roles characterization and is used in most of the studies performed on the *E. coli* model. In *Salmonella*, a *csrA* deletion mutant was constructed and severe growth defect and selection of suppression mutants with time were described [33]. Similar phenotypes were observed recently in an *E. coli* uropathogenic strain [34]. *csrA* deletion mutants were also constructed in K-12 and uropathogenic *E. coli* isolates in our group and we observed similar growth defect and suppressor selection [35]. On the contrary to our earlier published data, deletion of the *glgCAP* operon in the K-12 or uropathogenic strains does not improve growth [32]. This suggests that suppressor mutants were also picked up in the K-12 Δ*csrA* Δ*glgCAP* mutant [35].

Int. J. Mol. Sci. 2013, 14

2.2. Building up the Csr Network

In the years following CsrA identification, additional components of the Csr network were identified, notably sRNAs and proteins regulating either its activity or its expression. CsrA acts as a dimer [10,36] and forms ribonucleoprotein complexes with two sRNAs, CsrB [37], and CsrC [38]. Interaction of CsrA with these regulators leads to CsrA sequestration and inhibition of activity. These sRNAs are structurally similar although they carry a variable number of CsrA binding sites [39,40]. CsrB is 366 nt long containing 22 putative CsrA binding sites. It was shown that this sRNA is able to bind up to 18 CsrA dimers (see below), as shown by molecular weight estimation (256 kDa) [37]. The CsrC sRNA (242 nt long) contains 14 putative CsrA binding sites and is able to bind nine CsrA dimers [38].

Experimental data indicate that up to one-third of CsrA may be bound to the CsrB sRNA [41]. As CsrA affinity for these sRNAs is higher than that for mRNA targets and its concentration is higher than CsrA-target dissociation constant [41,42], it is inferred that CsrB and CsrC levels determine the concentration of “active” CsrA. Regulation of expression of these sRNAs is thus crucial to control CsrA activity. A negative feedback loop is involved in this regulation. CsrA indirectly activates *csrB* and *csrC* transcription through the BarA/UvrY two-component system (TCS) [38,41,43]. The BarA-associated response regulator, UvrY, activates transcription of the regulatory sRNAs [38,43–45]. The Csr network is composed of additional feedback loops *i.e.*, CsrA directly represses its own translation, while it indirectly activates its own transcription by an unknown mechanism [46]. These regulatory loops provide a rapid mechanism to reduce *csrA* expression when concentration of free CsrA reaches critical levels. The physiological signals that trigger the Csr network through the membrane-bound BarA sensor kinase have been identified. Interestingly, signals are products of carbon metabolism such as formate, acetate, other short-chain fatty acids, and Krebs cycle intermediates, which correlates with the crucial roles played by CsrA in central carbon metabolism [47,48].

Another important player in the Csr network is the CsrD protein. CsrD is predicted to be a membrane-bound protein containing degenerate GGDEF and EAL domains [49]. It controls the decay of CsrB and CsrC in an RNase E-dependent manner and in a c-di-GMP-independent manner, although both domains appear to be necessary for CsrD activity [19,49]. The “destabilizing” activity of CsrD on CsrB and CsrC has a positive effect on CsrA activity [49]. In addition, CsrA negatively controls *csrD* expression, which indicates an additional feedback loop in the Csr network [21,49,50]. It is not excluded that other types of regulation might exist within the Csr network. For example, in *Bacillus subtilis*, CsrA activity is regulated by direct interaction with the FliW protein, rather than sRNAs, in order to regulate motility [51].

2.3. Expanding the Csr Network

In the previous section, we described the basis of the Csr network in *E. coli*. Orthologues of *csrA* and *csr* regulatory elements are detected throughout the bacterial world. Thus, Csr constitutes a conserved global regulatory network, also called Rsm (repressor of secondary metabolites) network in specific species, such as *Pectobacterium carotovorum* and in the *Pseudomonas* genus. Interestingly,

some species encode multiple *csrA* homologues. While a single copy of the *csrA* gene is present in enteric bacteria (e.g., *E. coli*, *Salmonella enterica*, and *Pectobacterium* spp.) and *P. aeruginosa*, in silico analyses identified up to five homologues (*rsmA* and *rsmE*) in specific isolates of *P. fluorescens, P. putida* KT2440, and *P. syringae* [52]. The number of Csr sRNAs homologues can also vary, e.g., *Vibrio cholerae* and *Pseudomonas fluorescens* have been shown to contain three of them [53, 54].

As described above, CsrA regulates different processes. In addition to these direct and specific regulations, the Csr partners (proteins and sRNAs) interact with other global regulatory networks. How these interactions might influence group behavior such as quorum sensing, motility and biofilm formation remain vague, although several studies have highlighted tight connections between these different pathways (Figure 1).

---

**Figure 1. Interactions between the Csr network and global regulatory circuitries. Networks of *Escherichia coli, Salmonella typhimurium, Pectobacterium carotovorum, Vibrio cholerae, Pseudomonas aeruginosa,* and *Pseudomonas fluorescens* are represented. The CsrA/RsmA/RsmE master regulator is negatively regulated by the Csr/Rsm-sRNAs, which are positively regulated by the BarA/UvrY TCS or its orthologues. Light grey, dark grey, and black lines represent transcriptional, post-transcriptional and post-translational regulations, respectively. Solid and dashed lines represent direct and indirect regulation, respectively. For details, see in the text.**

![Diagram](attachment:diagram.png)

Some of these connections take place via CsrA. In *E. coli*, CsrA modulates the ppGpp alarmone level by directly regulating *relA* expression and thereby linking the Csr network to stringent response [55]. In turn, ppGpp and the DksA transcription factor regulate *csrB* and *csrC* expression via an unknown mechanism. In *P. carotovorum*, CsrA is regulated by another regulatory cascade involving a putative N-acyl homoserine lactone receptor (ExpR), thereby linking quorum sensing and Csr [56]. The connection between these two pathways is also described in *E. coli* although at another

level and through different actors. In this case, the SdiA protein, a LuxR homologue, activates *uvrY* expression at the transcriptional level, thereby affecting *csrB* expression [43].

Interestingly, the Csr network is connected to metabolism through a CsrD orthologue (MshH) in *V. cholerae* [57]. MshH directly interacts with the EIIA<sup>Glucose</sup> enzyme from the PTS system (phosphoenolpyruvate:carbohydrate phosphotransferase system) to positively control biofilm formation in a CsrA-independent manner. Connections between the Csr network and metabolism are also described in *S. typhimurium* through the BarA/SirA TCS. The UvrY orthologue (SirA) is positively regulated by catabolite repression or directly phosphorylated by acetyl-phosphate, a product generated by acetate metabolism [58–61]. Connections between the Csr network and motility are also mediated through the UvrY orthologue in specific species, such as in *P. carotovorum*. Expression of *gacA* (homolog to *uvrY*) is positively regulated by FlhDC [62]. This regulator acts positively on the *csrB* orthologue *rsmB* via HexA, a LysR-like regulator known to control exoenzymes production, further linking the Csr network to virulence [62,63].

In *P. aeruginosa* and *P. fluorescens*, another layer of complexity is added. Cross-regulation of GacS (BarA orthologue) by the LadS and RetS hybrid sensor kinases (containing both the sensor kinase and response regulator activities) have been shown. These two regulators are involved in pathogenesis and act in an opposite manner since RetS promotes acute infections whereas LadS promotes a chronic state [64–66]. This allows the integration of multiple signals and lead to a fine-tuned response.

Interestingly, other sRNAs might be involved in the Csr network to control CsrA activity such as Mcas (multicellular adhesive) sRNA in *E. coli*, which possess a dual function. It controls biofilm formation by classical base-pairing with mRNA targets and additionally, by sequestrating CsrA [67].

In *E. coli* and *P. carotovorum*, the Csr network is connected to the general stress response via the RpoS sigma factor, highlighting further complex interactions between pivotal pathways [46,68].

## 2.4. Are the Regulatory Csr/Rsm sRNAs Redundant?

The question of Csr/Rsm sRNAs redundancy was approached in different species. In general, it appears that these sRNAs are redundant. For examples, in *P. fluorescens*, the single *rsmY* or *rsmZ* mutants are not affected for extracellular enzymes production while the double mutant is unable to produce these virulence factors [69]. In *S. typhimurium*, mutants of both *csrB* and *csrC* genes are required to obtain phenotypes similar to that of a CsrA overexpression strain [70]. Furthermore, in *E. coli*, expression of one of the Csr sRNA is increased in mutants deleted of the other one, suggesting a compensatory mechanism [38]. Although these regulatory sRNAs are functionally and structurally related and regulated by the BarA/UvrY TCS or its homologues, they present specific features. One important functional difference resides in their capacity to bind a variable number of CsrA molecules. As a result, CsrC shows lower affinity for CsrA than CsrB [38]. In addition, molecular mechanisms regulating CsrB and CsrC stability in *E. coli* are different, although both require RNaseE. CsrB degradation requires CsrD and PNPase (polynucleotide phosphorylase, an exonuclease that is also part of the degradosome (for review, see [71]) while that of CsrC is independent of these two enzymes [49]. Additional differences between CsrB and CsrC expression and/or stability are observed in other species. Although expression of CsrB and CsrC in *Y. pseudotuberculosis* is regulated by UvrY, catabolite repression contributes to differential expression of the sRNAs with CsrB being

up-regulated in a *crp* (cAMP receptor protein) mutant and CsrC down-regulated [72]. This observation provides an additional example of connection between carbon metabolism and the Csr network. In *P. aeruginosa*, global regulators belonging to the H-NS family (MvaT and MvaU) regulate expression of the *rsmZ* sRNA but not that of *rsmY* (Figure 1) [73]. In addition, RetS and the HptB protein involved in phosphotransfer cascade protein, control *rsmY* expression, whereas *rsmZ* expression is exclusively controlled by RetS [74]. Regarding sRNAs stability, data indicate that differential regulation may also occur. In *P. aeruginosa*, Hfq binds and stabilizes RsmY [75]. In *S. typhimurium*, CsrA positively regulates CsrC half-life, suggesting that CsrA binding might lead to CsrC stabilization [70].

Difference in regulation of sRNA levels may provide clues to understand the benefit of having two or more seemingly redundant sRNAs. This may allow more possibilities for integrating various signals, leading to efficient and precise regulatory responses via gene dosage effect [73].

## 3. The CsrA-Containing Ribonucleoprotein Complexes

### 3.1. Structural Information

The CsrA protein is unrelated to typical regulators in terms of amino acid sequence but contains a KH motif (single-strand RNA binding domain) [76]. NMR-based structure shows that CsrA is composed of 5 β-strands and a short α-helix followed by an unstructured carboxy-terminal region (Figure 2) [36]. The functional CsrA dimer is formed by interdigitation of two CsrA monomers, resulting in a hydrophobic core composed of 10 β-strands and two wing-like α-helices, forming a barrel-like structure [77]. Using comprehensive alanine-scanning mutagenesis, two critical regions for regulation and RNA binding were identified [78]. These two RNA-binding surfaces are located within the first and last β-strands, which lie parallel to each other on opposite sides of the dimer and form two positively charged regions [78].

**Figure 2. Structure of the RsmE-RNA complex of *P. fluorescens*. Solution structure of the 2:2 complex of RsmE with 20-nucleotide *hcnA* sequence. Protein ribbons belonging to each monomer are shown in green and grey. Heavy atoms of the two RNAs are shown in yellow (carbon and nitrogen) and red (oxygen and phosphorus). The linking phosphates are shown in orange. From [42].**

Several amino acids residues are involved in RNA-binding, with R44 being the most important, and less importantly Arg7 and Ile47 [78]. CsrA preferentially binds GGA motifs (see below) located in single-strand regions of short hairpin structures [42, 78, 79]. Sequence-specific recognition of this motif is mostly mediated by carbonyl oxygen and amide groups of the main chain, indicating that CsrA-fold itself is responsible for RNA binding specificity [42].

### 3.2. Mechanism of Action of CsrA on mRNAs Targets

CsrA regulates translation of its target mRNAs, either positively or negatively. mRNA targets contain a variable number of CsrA binding sites. In general, they are located in the untranslated leader sequence and overlap the Shine-Dalgarno (SD) sequence. The CsrA-binding consensus sequence closely matches the canonical AAGGAGGU SD sequence [42]. The number of CsrA binding sites may vary from 1 to 6, depending on the target. For example, 4 and 6 CsrA binding sites have been identified on the *glgCAP* and *pgaABCD* mRNA targets, respectively [17, 80]. In the case of negative regulation, binding of CsrA prevents translation initiation and generally leads to mRNA degradation [9]. Some variations in the molecular mechanisms have been shown. As an example, negative regulation of *sdiA* occurs by CsrA binding to two sites located within the early coding region of the *sdiA* mRNA, without binding to or occluding the SD sequence [81].

Positive regulation appears to be less common. Up to now, two positively regulated targets have been described. In the case of *flhDC*, two binding sites are located in the untranslated leader region of the transcript. CsrA binding to these sites stabilizes the transcript by inhibiting the 5'-end degradation mediated by RNaseE [18, 19]. The second example involves the regulation of the *moaA* gene, which is involved in molybdenum cofactor (MOCO) synthesis. MOCO serves as a redox center in enzymes of anaerobic metabolism. The *moa* mRNA untranslated leader sequence contains two CsrA-binding sites and constitutes a MOCO-sensing riboswitch. This is the first example of a riboswitch aptamer that interacts with two regulatory factors, a low-molecular-weight ligand and an RNA binding protein [82].

### 3.3. Deeper Understanding of the Ribonucleoprotein Complexes Formed by CsrA

Although considerable sequence variation exists among the known CsrA binding sites, a GGA motif was identified as a highly conserved and essential element [79]. This was confirmed by studies using the SELEX method (systematic evolution of ligands by exponential enrichment). The SELEX-derived consensus was determined as RUACARGGAUGU (R being a pyrimidine and the underlined GGA motif being essential) [79]. As described above, the GGA motif is often located in single-strand loops of predicted short RNA hairpins [40]. A single-strand structure appears to be mandatory for regulation since several sRNAs (*GcvB*, *RprA* and *OmrA*) that carry potential CsrA-binding sites with GGA motifs located in double-strand structures (secondary structures of these sRNAs were predicted using Mfold [53], data not shown) do not regulate nor bind CsrA [67]. Spacing distance between CsrA binding sites appears to be important for CsrA-target complex stability [83]. The optimal distance is 18 nt although CsrA dimers are able to bind to two target sites separated by 10 to 63 nt. In CsrB, the 22 potential CsrA target sites are separated by an average of 12.25 nt. This suggests that a CsrA dimer would preferentially bridge non-adjacent sites (*i.e.*, separated by at least 18 nt), giving rise to an energetically stable globular complex. Dubey *et al.* [79] proposed a model for

CsrA binding. Translation repression mediated by CsrA would involve initial binding to a high affinity site located within a hairpin loop upstream of the SD sequence. This first binding would then allow the free RNA-binding surface of CsrA to interact with downstream low-affinity sites overlapping the SD sequence. These two successive binding events would result in formation of a repression loop blocking ribosomal binding for translation initiation [83].

# 4. General Conclusions

Ribonucleoprotein complexes control multiple pathways and may act at different levels of gene expression regulation. Regulation by ribonucleoprotein complexes comes into different flavors, with for instance, structural proteins acting also as post-transcriptional regulators such as RodZ, a cytoskeletal protein, that post-transcriptionally regulates expression of type III secretion system in Shigella [84].

In this review, we have described the Csr network with a special focus on the interactions between the CsrA key regulator and various species of RNAs *i.e.*, mRNA targets and regulatory sRNAs. sRNAs regulate the key regulator availability by sequestering it, thereby limiting the cellular concentration of “active” CsrA. One might propose that, like the McaS sRNA [67], these sRNAs have additional CsrA-independent functions. The fact that Hfq binds to RsmY [75] may indicate that this sRNA possesses regulatory activity involving mRNA-base pairing, as shown for McaS [67]. As exemplified in this review, ribonucleoprotein complexes connect multiple regulatory networks to coordinate gene expression and lead to adaptation.

## Acknowledgments

We thank Thibaut Hallaert for interesting discussions and Thomas Gosset for great help in figure design. Research in LVM’s laboratory is funded by FNRS (FRSM 3.4621.12), the Interuniversity Attraction Poles Programme initiated by the Belgian Science Policy Office (MICRODEV), the Fonds Jean Brachet and the Fondation David and Alice Van Buuren. ES is funded by the FRIA.

## Conflicts of Interest

The authors declare no conflict of interest.

## References

1. Byrgazov, K.; Vesper, O.; Moll, I. Ribosome heterogeneity: Another level of complexity in bacterial translation regulation. *Curr. Opin. Microbiol.* **2013**, *16*, 133–139.
2. Babitzke, P.; Baker, C.S.; Romeo, T. Regulation of translation initiation by RNA binding proteins. *Annu. Rev. Microbiol.* **2009**, *63*, 27–44.
3. Wassarman, K.M.; Storz, G. 6S RNA regulates *E. coli* RNA polymerase activity. *Cell* **2000**, *101*, 613–623.
4. Grudnik, P.; Bange, G.; Sinning, I. Protein targeting by the signal recognition particle. *Biol. Chem.* **2009**, *390*, 775–782.
5. Vogel, J.; Luisi, B.F. Hfq and its constellation of RNA. *Nat. Rev. Microbiol.* **2011**, *9*, 578–589.

6. Jiang, W.; Hou, Y.; Inouye, M. CspA, the major cold-shock protein of Escherichia coli, is an RNA chaperone. *J. Biol. Chem.* 1997, 272, 196–202.

7. Pichon, C.; Felden, B. Proteins that interact with bacterial small RNA regulators. *FEMS Microbiol. Rev.* 2007, 31, 614–625.

8. Romeo, T.; Gong, M.; Liu, M.Y.; Brun-Zinkernagel, A.M. Identification and molecular characterization of csrA, a pleiotropic gene from Escherichia coli that affects glycogen biosynthesis, gluconeogenesis, cell size, and surface properties. *J. Bacteriol.* 1993, 175, 4744–4755.

9. Baker, C.S.; Morozov, I.; Suzuki, K.; Romeo, T.; Babitzke, P. CsrA regulates glycogen biosynthesis by preventing translation of glgC in Escherichia coli. *Mol. Microbiol.* 2002, 44, 1599–1610.

10. Dubey, A.K.; Baker, C.S.; Suzuki, K.; Jones, A.D.; Pandit, P.; Romeo, T.; Babitzke, P. CsrA regulates translation of the Escherichia coli carbon starvation gene, cstA, by blocking ribosome access to the cstA transcript. *J. Bacteriol.* 2003, 185, 4450–4460.

11. Yang, H. Pleiotropic regulation of central carbohydrate metabolism in Escherichia coli via the gene csrA. *J. Biol. Chem.* 1995, 270, 29096–29104.

12. McKee, A.E.; Rutherford, B.J.; Chivian, D.C.; Baidoo, E.K.; Juminaga, D.; Kuo, D.; Benke, P.I.; Dietrich, J.A.; Ma, S.M.; Arkin, A.P.; et al. Manipulation of the carbon storage regulator system for metabolite remodeling and biofuel production in Escherichia coli. *Microb. Cell Fact.* 2012, 11, 79.

13. Wei, B.; Shin, S.; LaPorte, D.; Wolfe, A.J.; Romeo, T. Global regulatory mutations in csrA and rpoS cause severe central carbon stress in Escherichia coli in the presence of acetate. *J. Bacteriol.* 2000, 182, 1632–1640.

14. Lawhon, S.D.; Frye, J.G.; Suyemoto, M.; Porwollik, S.; McClelland, M.; Altier, C. Global regulation by CsrA in Salmonella typhimurium. *Mol. Microbiol.* 2003, 48, 1633–1645.

15. Jackson, D.W.; Suzuki, K.; Oakford, L.; Simecka, J.W.; Hart, M.E.; Romeo, T. Biofilm formation and dispersal under the influence of the global regulator CsrA of Escherichia coli. *J. Bacteriol.* 2002, 184, 290–301.

16. Wang, X.; Preston, J.F.; Romeo, T. The pgaABCD locus of Escherichia coli promotes the synthesis of a polysaccharide adhesin required for biofilm formation. *J. Bacteriol.* 2004, 186, 2724–2734.

17. Wang, X.; Dubey, A.K.; Suzuki, K.; Baker, C.S.; Babitzke, P.; Romeo, T. CsrA post-transcriptionally represses pgaABCD, responsible for synthesis of a biofilm polysaccharide adhesin of Escherichia coli. *Mol. Microbiol.* 2005, 56, 1648–1663.

18. Wei, B.L.; Brun-Zinkernagel, A.M.; Simecka, J.W.; Pruss, B.M.; Babitzke, P.; Romeo, T. Positive regulation of motility and flhDC expression by the RNA-binding protein CsrA of Escherichia coli. *Mol. Microbiol.* 2001, 40, 245–256.

19. Yakhnin, A.V.; Baker, C.S.; Vakulskas, C.A.; Yakhnin, H.; Berezin, I.; Romeo, T.; Babitzke, P. CsrA activates flhDC expression by protecting flhDC mRNA from RNase E-mediated cleavage. *Mol. Microbiol.* 2013, 87, 851–866.

20. Boyd, C.D.; O’Toole, G.A. Second messenger regulation of biofilm formation: Breakthroughs in understanding c-di-GMP effector systems. *Annu. Rev. Cell Dev. Biol.* 2012, 28, 439–462.

21. Jonas, K.; Edwards, A.N.; Simm, R.; Romeo, T.; Romling, U.; Melefors, O. The RNA binding protein CsrA controls cyclic di-GMP metabolism by directly regulating the expression of GGDEF proteins. *Mol. Microbiol.* 2008, **70**, 236–257.

22. Cui, Y.; Chatterjee, A.; Liu, Y.; Dumenyo, C.K.; Chatterjee, A.K. Identification of a global repressor gene, *rsmA*, of *Erwinia carotovora* subsp. carotovora that controls extracellular enzymes, N-(3-oxohexanoyl)-L-homoserine lactone, and pathogenicity in soft-rotting *Erwinia* spp. *J. Bacteriol.* 1995, **177**, 5108–5115.

23. Goodman, A.L.; Kulasekara, B.; Rietsch, A.; Boyd, D.; Smith, R.S.; Lory, S. A signaling network reciprocally regulates genes associated with acute infection and chronic persistence in *Pseudomonas aeruginosa*. *Dev. Cell* 2004, **7**, 745–754.

24. Molofsky, A.B.; Swanson, M.S. *Legionella pneumophila* CsrA is a pivotal repressor of transmission traits and activator of replication. *Mol. Microbiol.* 2003, **50**, 445–461.

25. Martínez, L.C.; Yakhnin, H.; Camacho, M.I.; Georgellis, D.; Babitzke, P.; Puente, J.L.; Bustamante, V.H. Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD. *Mol. Microbiol.* 2011, **80**, 1637–1656.

26. Kõiv, V.; Andresen, L.; Broberg, M.; Frolova, J.; Somervuo, P.; Auvinen, P.; Pirhonen, M.; Tenson, T.; Mäe, A. Lack of RsmA-mediated control results in constant hypervirulence, cell elongation, and hyperflagellation in *Pectobacterium wasabiae*. *PLoS One* 2013, **8**, e54248.

27. Pessi, G.; Haas, D. Dual control of hydrogen cyanide biosynthesis by the global activator GacA in *Pseudomonas aeruginosa* PAO1. *FEMS Microbiol. Lett.* 2001, **200**, 73–78.

28. Brencic, A.; Lory, S. Determination of the regulon and identification of novel mRNA targets of *Pseudomonas aeruginosa* RsmA. *Mol. Microbiol.* 2009, **72**, 612–632.

29. Heurlier, K.; Pessi, G.; Williams, F.; Holden, M.T.G.; Mara, M.C.A.; Haas, D.; Williams, P. The global posttranscriptional regulator RsmA modulates production of virulence determinants and N-Acylhomoserine lactones in *Pseudomonas aeruginosa*. 2001, **183**, 6676–6683.

30. Heeb, S.; Blumer, C.; Haas, D. Regulatory RNA as mediator in GacA/RsmA-dependent global control of exoproduct formation in *Pseudomonas* regulatory RNA as mediator in GacA/RsmA-dependent global control of exoproduct formation in *Pseudomonas fluorescens* CHA0. *J. Bacteriol.* 2002, **184**, 1046–1056.

31. Sacherer, P.; Défago, G.; Haas, D. Extracellular protease and phospholipase C are controlled by the global regulatory gene *gacA* in the biocontrol strain *Pseudomonas fluorescens* CHA0. *FEMS Microbiol. Lett.* 1994, **116**, 155–160.

32. Timmermans, J.; van Melderen, L. Conditional essentiality of the *csrA* gene in *Escherichia coli*. *J. Bacteriol.* 2009, **191**, 1722–1724.

33. Altier, C.; Suyemoto, M.; Lawhon, S.D. Regulation of *Salmonella enterica* serovar typhimurium invasion genes by *csrA*. *Infect. Immun.* 2000, **68**, 6790–6797.

34. Mitra, A.; Palaniyandi, S.; Herren, C.D.; Zhu, X.; Mukhopadhyay, S. Pleiotropic roles of *uvrY* on biofilm formation, motility and virulence in uropathogenic *Escherichia coli* CFT073. *PLoS One* 2013, **8**, e55492.

35. Seyll, E.; Van Melden L. Université Libre de Bruxelles (ULB), Laboratoire de Génétique et Physiologie Bactérienne, Belgium; Unpublished work, 2013.

36. Gutierrez, P.; Li, Y.; Osborne, M.J.; Pomerantseva, E.; Liu, Q.; Gehring, K. Solution structure of the carbon storage regulator protein CsrA from *Escherichia coli*. *J. Bacteriol.* 2005, **187**, 3496–3501.

37. Liu, M.Y.; Gui, G.; Wei, B.; Preston, J.F., 3rd; Oakford, L.; Yuksel, U.; Giedroc, D.P.; Romeo, T. The RNA molecule CsrB binds to the global regulatory protein CsrA and antagonizes its activity in *Escherichia coli*. *J. Biol. Chem.* 1997, **272**, 17502–17510.

38. Weilbacher, T.; Suzuki, K.; Dubey, A.K.; Wang, X.; Gudapati, S.; Morozov, I.; Baker, C.S.; Georgellis, D.; Babitzke, P.; Romeo, T. A novel sRNA component of the carbon storage regulatory system of *Escherichia coli*. *Mol. Microbiol.* 2003, **48**, 657–670.

39. Romeo, T. Global regulation by the small RNA-binding protein CsrA and the non-coding RNA molecule CsrB. *Mol. Microbiol.* 1998, **29**, 1321–1330.

40. Babitzke, P.; Romeo, T. CsrB sRNA family: Sequestration of RNA-binding regulatory proteins. *Curr. Opin. Microbiol.* 2007, **10**, 156–163.

41. Gudapati, S.; Suzuki, K.; Wang, X.; Babitzke, P.; Romeo, T. Regulatory interactions of Csr components: The RNA binding protein CsrA activates csrB transcription in *Escherichia coli*. *J. Bacteriol.* 2001, **183**, 6017–6027.

42. Schubert, M.; Lapouge, K.; Duss, O.; Oberstrass, F.C.; Jelesarov, I.; Haas, D.; Allain, F.H.-T. Molecular basis of messenger RNA recognition by the specific bacterial repressing clamp RsmA/CsrA. *Nat. Struct. Mol. Biol.* 2007, **14**, 807–813.

43. Suzuki, K.; Wang, X.; Weilbacher, T.; Pernestig, A.K.; Melefors, O.; Georgellis, D.; Babitzke, P.; Romeo, T. Regulatory circuitry of the CsrA/CsrB and BarA/UvrY systems of *Escherichia coli*. *J. Bacteriol.* 2002, **184**, 5130–5140.

44. Burrowes, E.; Abbas, A.; O’Neill, A.; Adams, C.; O’Gara, F. Characterisation of the regulatory RNA RsmB from *Pseudomonas aeruginosa* PAO1. *Res. Microbiol.* 2005, **156**, 7–16.

45. Heurlier, K.; Williams, F.; Heeb, S.; Dormond, C.; Pessi, G.; Singer, D.; Camara, M.; Williams, P.; Haas, D. Positive control of swarming, rhamnolipid synthesis, and lipase production by the posttranscriptional RsmA/RsmZ system in *Pseudomonas aeruginosa* PAO1. *J. Bacteriol.* 2004, **186**, 2936–2945.

46. Yakhnin, H.; Yakhnin, A.V.; Baker, C.S.; Sineva, E.; Berezin, I.; Romeo, T.; Babitzke, P. Complex regulation of the global regulatory gene *csrA*: CsrA-mediated translational repression, transcription from five promoters by Esigma(7)(0) and Esigma(S), and indirect transcriptional activation by CsrA. *Mol. Microbiol.* 2011, **81**, 689–704.

47. Takeuchi, K.; Kiefer, P.; Reimmann, C.; Keel, C.; Dubuis, C.; Rolli, J.; Vorholt, J.A.; Haas, D. Small RNA-dependent expression of secondary metabolism is controlled by Krebs cycle function in *Pseudomonas fluorescens*. *J. Biol. Chem.* 2009, **284**, 34976–34985.

48. Chavez, R.G.; Alvarez, A.F.; Romeo, T.; Georgellis, D. The physiological stimulus for the BarA sensor kinase. *J. Bacteriol.* 2010, **192**, 2009–2012.

49. Suzuki, K.; Babitzke, P.; Kushner, S.R.; Romeo, T. Identification of a novel regulatory protein (CsrD) that targets the global regulatory RNAs CsrB and CsrC for degradation by RNase E. *Genes Dev.* 2006, **20**, 2605–2617.

50. Jonas, K.; Edwards, A.N.; Ahmad, I.; Romeo, T.; Römling, U.; Melefors, O. Complex regulatory network encompassing the Csr, c-di-GMP and motility systems of *Salmonella Typhimurium*. Environ. Microbiol. 2010, 12, 524–540.

51. Mukherjee, S.; Yakhnin, H.; Kysela, D.; Sokoloski, J.; Babitzke, P.; Kearns, D.B. CsrA-FliW interaction governs flagellin homeostasis and a checkpoint on flagellar morphogenesis in *Bacillus subtilis*. Mol. Microbiol. 2011, 82, 447–461.

52. Reimmann, C.; Valverde, C.; Kay, E.; Haas, D. Posttranscriptional repression of GacS/GacA-controlled genes by the RNA-binding protein RsmE acting together with RsmA in the biocontrol strain *Pseudomonas fluorescens* CHA0. J. Bacteriol. 2005, 187, 276–285.

53. Lenz, D.H.; Miller, M.B.; Zhu, J.; Kulkarni, R.V.; Bassler, B.L. CsrA and three redundant small RNAs regulate quorum sensing in *Vibrio cholerae*. Mol. Microbiol. 2005, 58, 1186–1202.

54. Kay, E.; Dubuis, C.; Haas, D. Three small RNAs jointly ensure secondary metabolism and biocontrol in *Pseudomonas fluorescens* CHA0. Proc. Natl. Acad. Sci. USA 2005, 102, 17136–17141.

55. Edwards, A.N.; Patterson-Fortin, L.M.; Vakulskas, C.A.; Mercante, J.W.; Potrykus, K.; Vinella, D.; Camacho, M.I.; Fields, J.A.; Thompson, S.A.; Georgellis, D.; et al. Circuitry linking the Csr and stringent response global regulatory systems. Mol. Microbiol. 2011, 80, 1561–1580.

56. Cui, Y.; Chatterjee, A.; Hasegawa, H.; Dixit, V.; Leigh, N.; Chatterjee, A.K. ExpR, a LuxR homolog of *Erwinia carotovora* subsp. *carotovora*, activates transcription of *rsmA*, which specifies a global regulatory RNA-binding protein. J. Bacteriol. 2005, 187, 4792–4803.

57. Pickering, B.S.; Smith, D.R.; Watnick, P.I. Glucose-specific enzyme IIA has unique binding partners in the *Vibrio cholerae* biofilm. MBio 2012, 3, e00228–e00212.

58. Altier, C.; Suyemoto, M.; Ruiz, A.I.; Burnham, K.D.; Maurer, R. Characterization of two novel regulatory genes affecting *Salmonella* invasion gene expression. Mol. Microbiol. 2000, 35, 635–646.

59. Lawhon, S.D.; Maurer, R.; Suyemoto, M.; Altier, C. Intestinal short-chain fatty acids alter *Salmonella typhimurium* invasion gene expression and virulence through BarA/SirA. Mol. Microbiol. 2002, 46, 1451–1464.

60. Teplitski, M.; Goodier, R.I.; Ahmer, B.M. Pathways leading from BarA/SirA to motility and virulence gene expression in *Salmonella*. J. Bacteriol. 2003, 185, 7257–7265.

61. Teplitski, M.; Goodier, R.I.; Ahmer, B.M. Catabolite repression of the SirA regulatory cascade in *Salmonella enterica*. Int. J. Med. Microbiol. 2006, 296, 449–466.

62. Cui, Y.; Chatterjee, A.; Yang, H.; Chatterjee, A.K. Regulatory network controlling extracellular proteins in *Erwinia carotovora* subsp. *carotovora*: *flhDC*, the master regulator of flagellar genes, activates *rsmB* regulatory RNA production by affecting *gacA* and *hexA (lrhA)* expression. J. Bacteriol. 2008, 190, 4610–4623.

63. Mukherjee, A.; Cui, Y.; Ma, W.; Liu, Y.; Chatterjee, A.K. *hexA* of *Erwinia carotovora* ssp. *carotovora* strain Ecc71 negatively regulates production of RpoS and *rsmB* RNA, a global regulator of extracellular proteins, plant virulence and the quorum-sensing signal, N-(3-oxohexanoyl)-L-homoserine lactone. Environ. Microbiol. 2000, 2, 203–215.

64. Ventre, I.; Goodman, A.L.; Vallet-Gely, I.; Vasseur, P.; Soscia, C.; Molin, S.; Bleves, S.; Lazdunski, A.; Lory, S.; Filloux, A. Multiple sensors control reciprocal expression of *Pseudomonas aeruginosa* regulatory RNA and virulence genes. Proc. Natl. Acad. Sci. USA 2006, 103, 171–176.

65. Workentine, M.L.; Chang, L.; Ceri, H.; Turner, R.J. The GacS-GacA two-component regulatory system of *Pseudomonas fluorescens*: A bacterial two-hybrid analysis. *FEMS Microbiol. Lett.* 2009, 292, 50–56.

66. Goodman, A.L.; Merighi, M.; Hyodo, M.; Ventre, I.; Filloux, A.; Lory, S. Direct interaction between sensor kinase proteins mediates acute and chronic disease phenotypes in a bacterial pathogen. *Genes Dev.* 2009, 23, 249–259.

67. Jorgensen, M.G.; Thomason, M.K.; Havelund, J.; Valentin-Hansen, P.; Storz, G. Dual function of the McsA small RNA in controlling biofilm formation. *Genes Dev.* 2013, 27, 1132–1145.

68. Mukherjee, A.; Cui, Y.; Ma, W.; Liu, Y.; Ishihama, A.; Eisenstark, A.; Chatterjee, A.K. RpoS (sigma-S) controls expression of rsmA, a global regulator of secondary metabolites, harpin, and extracellular proteins in *Erwinia carotovora*. *J. Bacteriol.* 1998, 180, 3629–3634.

69. Valverde, C.; Heeb, S.; Keel, C.; Haas, D. RsmY, a small regulatory RNA, is required in concert with RsmZ for GacA-dependent expression of biocontrol traits in *Pseudomonas fluorescens* CHA0. *Mol. Microbiol.* 2003, 50, 1361–1379.

70. Fortune, D.R.; Suyemoto, M.; Altier, C. Identification of CsrC and characterization of its role in epithelial cell invasion in *Salmonella enterica* serovar Typhimurium. *Infect. Immun.* 2006, 74, 331–339.

71. Mackie, G.A. RNase E: At the interface of bacterial RNA processing and decay. *Nat. Rev. Microbiol.* 2013, 11, 45–57.

72. Heroven, A.K.; Sest, M.; Pisano, F.; Scheb-Wetzel, M.; Steinmann, R.; Bohme, K.; Klein, J.; Munch, R.; Schomburg, D.; Dersch, P. Crp induces switching of the CsrB and CsrC RNAs in *Yersinia pseudotuberculosis* and links nutritional status to virulence. *Front. Cell. Infect. Microbiol.* 2012, 2, 158.

73. Brencic, A.; McFarland, K.A.; McManus, H.R.; Castang, S.; Mogno, I.; Dove, S.L.; Lory, S. The GacS/GacA signal transduction system of *Pseudomonas aeruginosa* acts exclusively through its control over the transcription of the RsmY and RsmZ regulatory small RNAs. *Mol. Microbiol.* 2009, 73, 434–445.

74. Bordi, C.; Lamy, M.C.; Ventre, I.; Termine, E.; Hachani, A.; Fillet, S.; Roche, B.; Bleves, S.; Mejean, V.; Lazdunski, A.; et al. Regulatory RNAs and the HptB/RetS signalling pathways fine-tune *Pseudomonas aeruginosa* pathogenesis. *Mol. Microbiol.* 2010, 76, 1427–1443.

75. Sonnleitner, E.; Schuster, M.; Sorger-Domenigg, T.; Greenberg, E.P.; Blasi, U. Hfq-dependent alterations of the transcriptome profile and effects on quorum sensing in *Pseudomonas aeruginosa*. *Mol. Microbiol.* 2006, 59, 1542–1558.

76. Liu, M.Y.; Yang, H.; Romeo, T. The product of the pleiotropic *Escherichia coli* gene *csrA* modulates glycogen biosynthesis via effects on mRNA stability. *J. Bacteriol.* 1995, 177, 2663–2672.

77. Heeb, S.; Kuehne, S.A.; Bycroft, M.; Crivii, S.; Allen, M.D.; Haas, D.; Câmara, M.; Williams, P. Functional analysis of the post-transcriptional regulator RsmA reveals a novel RNA-binding site. *J. Mol. Biol.* 2006, 355, 1026–1036.

78. Mercante, J.; Suzuki, K.; Cheng, X.; Babitzke, P.; Romeo, T. Comprehensive alanine-scanning mutagenesis of *Escherichia coli* CsrA defines two subdomains of critical functional importance. *J. Biol. Chem.* 2006, 281, 31832–31842.

79. Dubey, A.K.; Baker, C.S.; Romeo, T.; Babitzke, P. RNA sequence and secondary structure participate in high-affinity CsrA-RNA interaction. *RNA* **2005**, *11*, 1579–1587.

80. Baker, C.S.; Eory, L.A.; Yakhnin, H.; Mercante, J.; Romeo, T.; Babitzke, P. CsrA inhibits translation initiation of *Escherichia coli hfq* by binding to a single site overlapping the Shine-Dalgarno sequence. *J. Bacteriol.* **2007**, *189*, 5472–5481.

81. Yakhnin, H.; Baker, C.S.; Berezin, I.; Evangelista, M.A.; Rassin, A.; Romeo, T.; Babitzke, P. CsrA represses translation of *sdiA*, which encodes the N-acylhomoserine-L-lactone receptor of *Escherichia coli*, by binding exclusively within the coding region of *sdiA* mRNA. *J. Bacteriol.* **2011**, *193*, 6162–6170.

82. Patterson-Fortin, L.M.; Vakulskas, C.A.; Yakhnin, H.; Babitzke, P.; Romeo, T. Dual posttranscriptional regulation via a cofactor-responsive mRNA leader. *J. Mol. Biol.* **2013**, *425*, 3662–3677.

83. Mercante, J.; Edwards, A.N.; Dubey, A.K.; Babitzke, P.; Romeo, T. Molecular geometry of CsrA (RsmA) binding to RNA and its implications for regulated expression. *J. Mol. Biol.* **2009**, *392*, 511–528.

84. Mitobe, J.; Yanagihara, I.; Ohnishi, K.; Yamamoto, S.; Ohnishi, M.; Ishihama, A.; Watanabe, H. RodZ regulates the post-transcriptional processing of the *Shigella sonnei* type III secretion system. *EMBO Rep.* **2011**, *12*, 911–916.

© 2013 by the authors; licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution license (http://creativecommons.org/licenses/by/3.0/).

Copyright of International Journal of Molecular Sciences is the property of MDPI Publishing and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.
