
Mol Neurobiol  
DOI 10.1007/s12035-013-8596-2  

---

**Ataxin-3 Protein and RNA Toxicity in Spinocerebellar Ataxia Type 3: Current Insights and Emerging Therapeutic Strategies**

Melvin M. Evers Â· Lodewijk J. A. Toonen Â· Willeke M. C. van Roon-Mom

Received: 11 October 2013 / Accepted: 14 November 2013  
Â© The Author(s) 2013. This article is published with open access at Springerlink.com

**Abstract** Ataxin-3 is a ubiquitously expressed deubiquitinating enzyme with important functions in the proteasomal protein degradation pathway and regulation of transcription. The C-terminus of the ataxin-3 protein contains a polyglutamine (PolyQ) region that, when mutationally expanded to over 52 glutamines, causes the neurodegenerative disease spinocerebellar ataxia 3 (SCA3). In spite of extensive research, the molecular mechanisms underlying the cellular toxicity resulting from mutant ataxin-3 remain elusive and no preventive treatment is currently available. It has become clear over the last decade that the hallmark intracellular ataxin-3 aggregates are likely not the main toxic entity in SCA3. Instead, the soluble PolyQ containing fragments arising from proteolytic cleavage of ataxin-3 by caspases and calpains are now regarded to be of greater influence in pathogenesis. In addition, recent evidence suggests potential involvement of a RNA toxicity component in SCA3 and other PolyQ expansion disorders, increasing the pathogenic complexity. Herein, we review the functioning of ataxin-3 and the involvement of known protein and RNA toxicity mechanisms of mutant ataxin-3 that have been discovered, as well as future opportunities for therapeutic intervention.

**Keywords** Polyglutamine disease Â· Spinocerebellar ataxia type 3 Â· Machadoâ€“Joseph disease Â· Ataxin-3 Â· Proteolytic cleavage Â· RNA toxicity

**Abbreviations**  
ATXN3 Ataxin-3  
ER Endoplasmic reticulum  

Lodewijk J.A. Toonen and Melvin M. Evers contributed equally to this work.  
M. M. Evers Â· L. J. A. Toonen Â· W. M. C. van Roon-Mom (ðŸ“§)  
Department of Human Genetics, Leiden University Medical Center, Albinusdreef 2, 2333ZA Leiden, The Netherlands  
e-mail: w.vanroon@lumc.nl  

ERAD Endoplasmic reticulum-associated degradation  
HD Huntington's disease  
HDAC Histone deacetylase  
NES Nuclear export signal  
NLS Nuclear localization signal  
PolyQ Polyglutamine  
SCA3 Spinocerebellar ataxia type 3  
UIM Ubiquitin interacting motif  
VCP/p97 Valosin-containing protein  

**Introduction**  

Spinocerebellar ataxia type 3 (SCA3), or Machadoâ€“Joseph disease (MJD) [1], is the most common spinocerebellar ataxia [2, 3] and the second most common polyglutamine (polyQ) disease after Huntingtonâ€™s disease (HD) [4]. Similar to the other polyQ disorders, SCA3 is inherited in an autosomal dominant fashion [5], is progressively neurodegenerative and is ultimately fatal. Current therapeutic strategies are only able to provide symptomatic relief [6]. SCA3 is clinically heterogeneous, but the main feature is progressive ataxia, affecting balance, gait and speech. Other frequently described symptoms include pyramidal signs, progressive external ophthalmoplegia, dysarthria, dysphagia, rigidity, distal muscle atrophies and double vision [5, 7â€“9]. Neuropathological studies have detected widespread neuronal loss in the cerebellum, thalamus, midbrain, pons, medulla oblongata and spinal cord of SCA3 patients, as reviewed by Riess et al. [10].

SCA3 is caused by an expanded stretch of CAG triplets in the coding region of the *ATXN3* gene on chromosome 14q32.1, encoding the ataxin-3 protein [11]. Healthy individuals have up to 44 CAG repeats, whilst affected individuals have between 52 and 86 glutamine repeats. A repeat range from 45 to 51 is associated with incomplete penetrance of the
disease [11â€“13]. SCA3 patients with two mutant alleles show a more severe disease phenotype than those with a single mutant allele [14]. Also, there is a clear correlation between CAG repeat size and age of onset, though CAG repeat length only accounts for approximately 50 % of the total variability in age of onset [15]. The expanded CAG repeat leads to formation of an expanded polyQ tract in the C-terminal region of the ataxin-3 protein, leading to toxic gain of function of the protein and formation of characteristic neuronal aggregates [16]. The neurotoxic properties of these aggregates are still under debate since the number of aggregates does not mirror the level of neurodegeneration or *ATXN3* CAG repeat length [17].

Despite being a monogenetic disease, SCA3 pathogenesis has proven to be complex. Extensive studies in cell and animal models over the last decade have led to the identification of several cellular processes potentially involved in SCA3 pathology. Nonetheless, much remains to be elucidated regarding the toxicity resulting from mutant ataxin-3 RNA and protein, and a more comprehensive understanding of the many cellular processes involved would be of great benefit for the development of therapeutic strategies. In this review, current knowledge on normal and mutant polyQ expanded ataxin-3 functioning, as well as the main thoughts on toxic mechanisms of mutant ataxin-3 RNA and protein and potential therapeutic strategies will be discussed.

### Ataxin-3 Protein

The ataxin-3 protein has a molecular weight of approximately 42 kDa, depending on the size of the polyQ repeat and isoform. The CAG repeat, located in the penultimate exon, is translated into a polyQ repeat located at the C-terminus of the protein. In blood, 56 splice variants of *ATXN3* have been identified, of which 20 could potentially be translated into a functional ataxin-3 protein [18]. Of these 20 isoforms, only two isoforms that differ in their C-terminal tail have been studied extensively thus far. In this review, we will refer to the isoform of *ATXN3* most commonly expressed in brain consisting of 11 exons and translated into an ataxin-3 protein of 361 amino acids [19â€“21], based on a polyQ repeat length of 13 (Ensembl transcript ID ENST00000393287; Fig. 1).

### Cellular Localisation of Ataxin-3

Ataxin-3 is ubiquitously found throughout the cell and is able to translocate from the cytoplasm to the nucleus and back [20â€“24]. Different regions of the ataxin-3 protein influence its subcellular localisation and, although it is not yet known if ataxin-3 plays a more important role in the nucleus or the cytoplasm, enzymatically active ataxin-3 has been shown to localise to the nucleus more frequently compared to an inactive form [25]. In silico analysis predicted a nuclear localisation signal (NLS) in the proximity of the polyQ repeat at amino acid 273 to 286 (Fig. 1) [23, 26â€“28]. This NLS was shown to display a weak nuclear import activity in vitro [27, 28]. In contrast, however, mutating or deleting the proposed core sequence from amino acid 282 to 285 had no effect on subcellular distribution, thus questioning the importance of the ataxin-3 NLS in its cellular localisation [29, 30].

Another region that has been implicated in regulating ataxin-3 cellular localisation is the first 27 amino acids. Ataxin-3 lacking these first 27 amino acids could not be found in the nucleus, though the mechanism involved is still unknown [22]. Furthermore, ataxin-3 contains six potential nuclear export signals (NES) [26, 27], of which two (amino acid 77 to 99 and 141 to 158; Fig. 1), showed significant nuclear export activity [27]. The first NES overlaps with the ubiquitin binding site in the Josephin domain, but whether ubiquitin binding influences the function of this NES is unknown.

Nuclear localisation of ataxin-3 can also be influenced by phosphorylation events. Ataxin-3 exhibits six potential phosphorylation sites that are targets for the serine-threonine casein kinase 2 (CK2) [30] and one phosphorylation site that is a target for glycogen synthase kinase 3Î² (GSK 3Î²) [31]. Phosphorylation of serines 236, 340 and 352 by CK2 (Fig. 1) resulted in nuclear localisation of ataxin-3, whereas phosphorylation of the other serines are of less importance for subcellular localisation [30]. Next to nuclear localisation, phosphorylation of ataxin-3 was also found to be important for the stability of the protein [30].

Thus, the localisation of ataxin-3 is thought to be regulated by specific intrinsic localisation signals as well as posttranslational modifications and its cellular function likely depends on its localisation.

### Ataxin-3 and Regulation of Protein Degradation

The N-terminus of ataxin-3 contains a large Josephin domain (Fig. 1) that is known to have a low isopeptidase activity [32], implicating a role for ataxin-3 in the ubiquitin-proteasomal pathway [33]. The Josephin domain, together with the ubiquitin interacting motifs (UIMs), can either rescue proteins from degradation or stimulate breakdown by the removal of inhibitory poly-ubiquitin chains and by the regeneration of free and reusable ubiquitin [32â€“34]. The ubiquitin-proteasomal pathway is involved in various cellular processes such as protein degradation, endocytosis, transcriptional regulation and antigen presentation. Ubiquitination is a cascade of processes involving activating enzyme E1, transfer to ubiquitin conjugating enzymes E2 and association with ubiquitin ligases E3, resulting in addition of ubiquitins via isopeptide linkages to lysines in the targeted protein [35]. Ubiquitins can
Mol Neurobiol

Fig. 1 Schematic representation of the *ATXN3* gene, exonâ€“intron structure and protein product showing protein functional domains, posttranslational modifications and binding domains of the main interacting partners. **a** The *ATXN3* gene (Ensembl transcript ID ENST00000393287) consists of 11 exons with the start codon in exon 1 and the CAG repeat in exon 10. The shape of the boxes depict the reading frame, nt nucleotides. The height of the introns are relative to their actual length. **b** The ataxin-3 protein consists of 361 amino acids (*aa*) with a Josephin domain in the N-terminal part that contains crucial amino acids for its isopeptidase activity [cysteine 14 (C), histidine 119 (H), and asparagine 134 (N)] and two nuclear export signals (NES). The C-terminal part contains three ubiquitin interacting motifs (UIM 1 to 3), a nuclear localisation signal (NLS) and the polyglutamine (polyQ) repeat. Specific amino acids known to undergo posttranslational modifications are indicated as follows: yellow circles phosphorylation (*P*), purple eclipse ubiquitination (*Ub*), orange triangle calpain cleavage site, pink triangle caspase cleavage motif. **c** Binding domains of the main interacting partners: ubiquitin; VCP/p97 valosin-containing protein, hHR23A and hHR23B human homologues of yeast protein RAD23, and DNA.

bind individually or as a poly-ubiquitin chain. Editing and removal of poly-ubiquitin chains as well as recycling of ubiquitin is critical for cellular homeostasis. Polyubiquitin chains linked through lysines 6, 11, 27, 29, 33 and 48 target proteins for proteasomal degradation. In contrast, lysine 63 or linear polyubiquitin chains have non-proteolytic functions such as activation of kinases and autophagy, where it is proposed to be involved in the biogenesis of protein inclusions [36].

Amino acid cysteine 14, histidine 119, and asparagine 134 of the Josephin domain (Fig. 1) of ataxin-3 are essential for its isopeptidase function and are highly conserved between Josephin and other ubiquitin C-terminal hydrolases and ubiquitin-specific proteases [37, 38]. The UIMs mediate selective binding to ubiquitin chains and restrict the types of chains that can be cleaved by the Josephin domain. Ataxin-3 is known to recognise poly-ubiquitin chains of four or more ubiquitins [33, 39] and binds the poly-ubiquitin linkages lysine 48, lysine 63 and mixed linkage ubiquitin chains, with preference for lysine 63-tagged ubiquitins [32, 35]. Especially the first and second UIMs are very important for binding of poly-ubiquitin chains, since mutations of leucine 229 and 249 resulted in almost abolished binding to ubiquitins [33, 40]. Additionally, within the Josephin domain, two ubiquitin binding sites were found (Fig. 1) [41]. Whilst the significance of these ubiquitin binding sites is not completely understood, mutating the first binding site eliminated binding of both lysine 48 and 63 linked ubiquitin chains to the Josephin domain of ataxin-3 [42]. The second ubiquitin binding site was shown to be important for lysine 63 tagged ubiquitins [42].

Ataxin-3 can regulate its own ubiquitination pattern [25]. Next to self-regulation, ataxin-3 can also regulate the stability and ability of the ubiquitin ligases E3 C-terminus of heat shock protein (HSP) 70-interacting protein (CHIP) [32, 43] and Parkin [44] through deubiquitination of its targets. Upon self-ubiquitination, Parkin forms a complex with conjugating enzyme E2, resulting in the addition of an isopeptide linkage to Parkin [45]. Ataxin-3 is proposed to reduce this self-ubiquitination of Parkin by removal of the isopeptide linkage [46]. Deubiquitination of CHIP by ataxin-3 was found to be only active if the poly-ubiquitin chain on target substrates reaches a critical length, preventing extra ubiquitin incorporation [47]. Contrariwise, CHIP also mono-ubiquitates ataxin-3 at lysine 117 in the Josephin domain (Fig. 1), which directly results in enhanced isopeptidase activity of ataxin-3 [48, 49]. Interestingly, this isopeptidase activity was found to be independent of the UIMs, probably because of direct conformational changes in or near the catalytic domain [49].

Ataxin-3 has been found to bind the valosin-containing protein (VCP/p97; Fig. 1) [50, 51]. VCP/p97 has numerous functions, of which one is the regulation of misfolded endoplasmic reticulum (ER) protein degradation, a cellular process named ER-associated degradation (ERAD) [50, 52]. A potential VCP/p97 binding domain has been mapped to an arginine/lysine-rich motif just prior to the polyQ repeat [53].

ataxin-3-VCP/p97 complex is involved in assisting targeted proteins to the proteasome [54]. Ataxin-3 is also known to interact with the human homologues of yeast protein RAD23, hHR23A and hHR23B (Fig. 1) [51]. hHR23A and hHR23B are involved in DNA repair pathways as well as the delivery of ubiquitinated substrates to the proteasome for degradation [51]. The binding site of hHR23B to ataxin-3 is located in the second ubiquitin binding site of the Josephin domain, and in concordance, hHR23B was shown to compete with ubiquitin binding [38]. Cell stress resulted in altered interactions with both VCP/p97 and HR23B, which were found mainly in the cytoplasm, although no effect on protein degradation was reported [55].

To conclude, ataxin-3 is a well-established deubiquitinating enzyme, directly regulating the ubiquitin-proteasome machinery. Next to the proteasomal degradation, ataxin-3 is also implicated to be involved in regulation of misfolded ER protein degradation.

### Ataxin-3 and Transcriptional Regulation

Besides the clear role of ataxin-3 in protein degradation, ataxin-3 has been shown to be capable of regulating the transcriptional process. Ataxin-3 is, for instance, able to repress matrix metalloproteinase-2 (MMP-2) transcription, and improved nuclear localisation of ataxin-3 through phosphorylation enhances this transcriptional repression [30]. Transcriptional regulation by ataxin-3 might arise through different mechanisms, since ataxin-3 is known to interact with numerous transcriptional regulators such as TATA box-binding protein (TBP)-associated factor 4 (TAF4) [56], cAMP response element-binding protein binding protein (CBP) [57â€“59], p300 [59], p300/CBP-associated factor (PCAF) [59], nuclear receptor co-repressor (NCoR1), histone deacetylase (HDAC) 3 and 6 [60, 61], forkhead box O (FOXO) transcription factor FOXO4 [62], and RAD23 [51]. Also, direct binding of ataxin-3 to DNA can likely take place through a leucine zipper motif located at amino acid 223 to 270 (Fig. 1) [61]. This basic leucine zipper motif was previously shown to bind to the GAGGAA consensus sequence in DNA [63].

Moreover, ataxin-3 can inhibit histone acetylation and repress transcription in vivo via interaction with the polyQ repeat [59]. Ataxin-3 is also postulated to regulate transcription factors by ubiquitination of CHIP transcription complexes [47]. Upon ubiquitination of transcription factors by E2 ligases, ataxin-3 is thought to be recruited to the CHIP complex, thus stimulating the transcription of the target DNA. Deubiquitination of the monoubiquitinated transcription factors by ataxin-3 would result in repression of transcription [47, 64].

These observations suggest that next to ubiquitin-proteasomal regulation of transcription, endogenous ataxin-3 might also directly interact with important transcriptional regulators.

#### Is Ataxin-3 an Essential Protein for Normal Cellular Function?

Although the ataxin-3 protein has been well studied, it is still uncertain to what extent ataxin-3 is an essential protein for normal cellular functioning. In support of an essential role for ataxin-3, depletion of ataxin-3 using small-interference RNA (siRNA) in cultured non-neuronal human and mouse cells resulted in accumulation of ubiquitinated material in the cytoplasm, cytoskeletal disorganisation, loss of cell adhesion and increased cell death [65]. Contradictory evidence, however, suggests that ataxin-3 is not necessary for normal cellular functioning, since ataxin-3 knock-out in *Caenorhabditis elegans* did not alter the lifespan [66] and remarkably resulted in resistance to stress [67]. Likewise in another study local knock-down of endogenous ataxin-3 by injection of lentiviruses encoding short-hairpin RNAs (shRNAs) into the brain of wild-type rats did not show any toxicity [68]. Though ataxin-3 protein was only downregulated in the striatum and only for a 2-month period. In ataxin-3 knock-out mice, loss of ataxin-3 did not affect viability or fertility [69â€“71]. However, these mice did show a mild behavioural phenotype with increased anxiety, together with increased levels of ubiquitinated proteins, particularly in cells that are known to express high levels of ataxin-3 in wild-type mice [69]. Furthermore, ataxin-3 has also been proposed to serve as a neuroprotectant, since in flies expressing mutant polyQ proteins overexpression of ataxin-3 was found to alleviate neurodegeneration [72]. In contrast, double transgenic mice co-expressing mutant and wild-type ataxin-3 did not show any phenotypic improvement as compared to single transgenic SCA3 mice, suggesting that ataxin-3 may not act as neuroprotectant [73].

Whilst absence of ataxin-3 thus does not appear to be directly detrimental to cellular vitality in most studies, the subtle phenotypes observed in rodent ataxin-3 knock-out models and the fact that ataxin-3 contains several well-conserved regions amongst different species [74] may indicate that the protein is not completely dispensable.

#### Altered Properties of Mutant Ataxin-3

In SCA3, the expanded polyQ stretch in the C-terminus of ataxin-3 most likely leads to conformational changes of the protein, in turn resulting in altered binding properties, loss of protein function, altered subcellular localisation, aggregation, and perhaps altered proteolytic cleavage [75] (Fig. 2). Although in the past decade there has been extensive research

Mol Neurobiol

![Diagram](attachment:diagram.png)

Fig. 2 Schematic representation of cellular pathogenesis in SCA3. The ATXN3 gene can be transcribed into mRNA containing 11 exons. The expanded CAG repeat is located in the penultimate exon and the transcript is translated into a mutant polyQ repeat containing protein. This polyQ repeat triggers conformational changes, resulting in abnormally folded mutant ataxin-3. Mutant ataxin-3 can be proteolytically cleaved, giving rise to C-terminal fragments and possibly N-terminal fragments (dashed line) that are aggregation-prone. Full length and cleaved forms of ataxin-3 form soluble monomers, oligomers or large insoluble aggregates, both in the nucleus and in the cytoplasm that cause toxicity. Other cellular disturbances resulting from mutant ataxin-3 presence involved in SCA3 pathogenesis include transcriptional deregulation, impaired autophagy, mitochondrial dysfunction, proteasomal impairment, and compromised axonal transport. Next to mutant polyQ-induced toxicity, there is likely also an RNA toxicity component involved in the disease pathogenesis.

into the SCA3 disease mechanisms, it is still not completely understood how the ataxin-3 polyQ expansion results in the observed pathology. In brain, the ATXN3 gene expression levels were not found to be higher in the predominantly affected brain regions, suggesting that ATXN3 gene expression levels do not directly correlate with the selective neurodegeneration seen in SCA3 patients [76]. Therefore, other alterations induced by the mutant ataxin-3 protein are most likely important in SCA3 pathogenesis as well.

### Aggregation of Mutant Ataxin-3

One of the first observations made in SCA3 patient derived brain material were the intracellular aggregates in neurons of the ventral pons and less frequently in the substantia nigra, globus pallidus, dorsal medulla and dentate nucleus [16], a feature that was reproduced in cell and animal models over-expressing mutant ataxin-3 [77â€“79]. Mutant ataxin-3 is known to accumulate in the cell nucleus a property that is required for in vivo toxicity [16, 20, 80]. In line with this, transgenic SCA3 mice show a decrease of soluble mutant ataxin-3 protein in the cerebellum during disease progression, whilst aggregate formation increases and the disease phenotype progresses [81]. The nuclear environment has been suggested to promote the formation of nuclear aggregates, and additional proteins, such as TBP and CBP, were found to be recruited to the aggregates in human brain [82] and SCA3 animal models [83]. Indeed, reducing nuclear localisation of mutant ataxin-3 by pharmacological inhibition of CK2 led to a reduction in nuclear inclusions [30, 31]. The intranuclear aggregates only arise when ataxin-3 contains the expanded polyQ tract [16].

Aggregation is not confined to the nucleus, however, as overexpression of mutant ataxin-3 in cultured COS-7 cells resulted in the formation of cytosolic aggregates, mainly surrounding the nucleus, suggesting a more widespread aggregation throughout the cell [22]. Likewise, widespread axonal aggregates in fiber tracts were identified in SCA3 brain tissue, similar to the axonal aggregates seen in HD [84]. These axonal aggregates potentially interfere with the axonal transport, causing impaired cell function and cell death [85, 86] (Fig. 2). Pure expanded polyQ stretches consist of monomers that are able to undergo Î²-sheet conformational transition and can assemble into Î²-sheet-rich amyloid-like fibrils [87]. As polyQ regions modulate interactions between coiled-coil domains, it has been suggested that interactions of natural coiled-coil partners with the polyQ domain increase aggregation, whilst partners interacting at other regions of the mutant protein are able to diminish aggregate formation [88]. The length of the polyQ repeat of mutant ataxin-3 also determines its rate of aggregation and longer polyQ stretches are associated with increased levels of aggregation, both in vitro and in vivo [89, 90].

It is not yet entirely clear whether the aggregates observed in SCA3 brain consist of full-length ataxin-3, shorter ataxin-3 protein fragments, or perhaps a combination of the two. The aggregates in SCA3 brain material can be detected with several different antibodies, amongst which a polyQ-specific antibody, as well as an antibody binding approximately 60 amino acids N-terminal to the polyQ tract [16]. Aggregation of ataxin-3 can be induced by transfecting cells with a short ataxin-3 protein fragment with an expanded polyQ repeat [78, 91] and full-length mutant ataxin-3 is recruited into the insoluble complexes [16]. It can therefore be concluded that an expanded polyQ containing fragment can catalyse or act as a seed for aggregation of the full-length protein. Furthermore, ataxin-3 aggregates are ubiquitin-positive and contain certain proteasomal subunits, suggesting perturbation of ubiquitin-dependent proteasomal degradation in SCA3 [77] (Fig. 2). Indeed, mutant ataxin-3 undergoes ubiquitination to a similar extent as normal ataxin-3, but its half-life seems to be longer, perhaps indicating hampered degradation by the ubiquitin-proteasomal pathway [92], facilitating mutant ataxin-3 aggregation. Much is still unknown regarding the exact role and properties of protein aggregates in SCA3. However, increasing evidence suggests that the occurrence of shorter mutant ataxin-3 protein fragments is an important step leading to aggregate formation seen in SCA3.

### Proteolytic Cleavage and Formation of Toxic Ataxin-3 Fragments

A major line of thinking with regard to polyQ disorder pathogenesis termed the â€˜toxic fragment hypothesisâ€™ concerns the proteolytic cleavage of polyQ expanded protein. For mutant ataxin-3, this proteolytic cleavage is thought to lead to generation of cytotoxic and aggregation-prone shorter soluble fragments containing the expanded polyQ toxic entity [91, 93â€“95] (Fig. 2). In a mouse model, ataxin-3 derived cleavage fragments were shown to contain expanded polyQ-containing protein fragments C-terminal of amino acid 221 [96]. Interestingly, in the two SCA3 brains tested, the ataxin-3 C-terminal fragments were enriched in disease-relevant brain structures, such as the cerebellum and substantia nigra, compared to an unaffected brain region or control brain material [96]. In subsequent studies, several caspase and calpain proteolytic enzymes were identified that could be responsible for the generation of the potentially toxic ataxin-3 fragments. These mutant ataxin-3 fragments are highly susceptible to aggregation [97], and capable of inducing both aggregation and toxicity to a larger extent than full-length mutant ataxin-3 [78, 90].

The cysteine proteases of the caspase family are associated with apoptosis and are able to cleave proteins at specific aspartate residues [98]. Caspases are implicated in the pathogenesis of several other polyQ diseases besides SCA3, though the involvement is not straightforward and varies per disease and study [99â€“103]. Several caspase cleavage sites are predicted for ataxin-3 and the three most relevant sites were identified to be aspartates 241, 244, and/or 248 (Fig. 1), as mutations of these residues significantly impeded caspase-1-induced ataxin-3 cleavage. In the same study, ataxin-3 cleavage lead to appearance of SDS-insoluble aggregates, and the amount of aggregates could be diminished by addition of caspase inhibitors [93]. However, the involvement of caspase-mediated cleavage of ataxin-3 in the SCA3 pathology is possibly quite limited, since incubation with purified caspases 1 and 3 only showed modest cleavage of ataxin-3. Also, under the same experimental conditions, cleavage of in vitro translated ataxin-3 was found to be less efficient when compared to cleavage of huntingtin and atrophin-1, the proteins responsible for HD and dentatorubral-pallidoluysian atrophy, respectively [104]. Furthermore, caspase 1 and 3 inhibitors did not reduce aggregate formation in a SCA3 neuronal cell line [94]. To conclude, there currently is no conclusive in vitro or in vivo evidence for an involvement of caspases in the SCA3 pathogenesis, indicating that other proteolytic enzymes are likely responsible for the formation of expanded polyQ-containing ataxin-3 protein fragments.

Increasing evidence is available for the involvement of calpains in SCA3 pathogenesis. Calpains are a family of ubiquitously expressed calcium-dependent cytosolic cysteine proteolytic enzymes [105]. Based on observed fragment sizes and antibody binding, calpain-mediated cleavage of ataxin-3 has been shown to occur most convincingly at amino acid 60, 221, and 260 (Fig. 1) [96, 97, 106, 107]. Interestingly, mutant ataxin-3 was found to be more prone to calpain-2-mediated cleavage at amino acid 260 compared to wild-type ataxin-3 and resulted in the formation of C-terminal ataxin-3 fragments
highly susceptible to aggregation [97]. Inhibition of calpains resulted in reduced mutant ataxin-3 proteolysis, nuclear localisation, aggregation and alleviated toxicity in vivo [106, 107]. Conversely, enhancement of calpain activity in vivo by knocking out the calpain inhibitor calpastatin worsened the phenotype of SCA3 mice [97]. Interestingly, reduced calpastatin levels in brain tissue of SCA3 patients have been observed, indicating that depletion of calpastatin may play a role in SCA3 pathology by increasing calpain-mediated proteolysis and associated neurotoxicity [107]. Calpain-mediated ataxin-3 cleavage has also been proposed to be the determinant of neuronal specificity of pathology in SCA3. In vitro it was found that excitation-mediated calcium influx was required for ataxin-3 cleavage by calpains and subsequent formation of aggregates. This excitation-induced cleavage could only be initiated in neurons, but not in fibroblasts or glia cells [94]. Hence, in spite of some initial reports of calpain inhibitors not affecting ataxin-3 proteolysis in cell models [93, 104, 108], involvement of calpains has gained a wide base of support from both cell and animal studies [94, 97, 107].

Taken together, the mounting evidence of involvement of particularly calpains in ataxin-3 cleavage suggests an important role of these proteolytic enzymes in SCA3 pathogenesis by cleavage-induced toxicity and perhaps subsequent initiation of an aggregation cascade.

### Localisation of Mutant Ataxin-3 Protein Fragments

As outlined above, the cellular distribution of ataxin-3 is also relevant in the context of proteolytic cleavage and nuclear aggregate formation (Fig. 2). Several observations provide a strong indication that the nucleus likely is the main site of mutant ataxin-3 toxicity. An ataxin-3 construct containing only the C-terminus with the polyQ tract, but lacking the Josephin domains and UIMs, is able to aggregate in the cell nucleus or cytoplasm when coupled to a synthetic NLS or NES, respectively [29]. In the cytoplasm, however, the misfolded proteins are targets for stress-induced degradation, whereas the nuclear aggregates accumulate [29]. Similarly, in mice, the SCA3 phenotype can be exacerbated by attaching an NLS to the C-terminus of expanded ataxin-3, resulting in increased levels of nuclear aggregates and earlier death compared to the same construct without a synthetic NLS. Conversely, coupling an NES to the expanded ataxin-3 construct reduced nuclear aggregates and improved the phenotype [80]. N-terminal ataxin-3 was found to be less frequently present in SCA3 mouse brain material than C-terminal fragments, which accumulated more readily and were present in the cell nucleus [107]. The cellular localisation signals are likely crucial mediators in the process of ataxin-3 aggregate localisation and possibly toxicity, as has also been suggested for SCA1 [109].

C-terminal ataxin-3 fragments arising from calpain cleavage at amino acid 260 [106] can be subject to phosphorylation at serine 340 and serine 352 within the 3rd UIM of these fragments (Fig. 1), perhaps explaining their nuclear presence and stability [30]. Other cleavage sites, such as the proposed caspase-1 cleavage around amino acid 221 and 334 [93, 96], or cleavage at amino acid 190 [110], would result in a C-terminal fragment containing only the first 2 UIMs and the polyQ tract. Preferential nuclear localisation of this fragment is again predicted due to phosphorylation of serine residues in both the UIMs, albeit to a lesser extent than a fragment containing all 3 UIMs [30].

Some indications were found that there may also be a role for cytoplasmic inclusions and even toxic N-terminal ataxin-3 fragments in SCA3 pathogenesis. Based on N-terminal fragments observed in SCA3 mouse models, it has been suggested that simultaneous cleavage at amino acid 60 and 260 may be responsible for N-terminal ataxin-3 fragments [107]. N-terminal ataxin-3 fragments can contain the Josephin domain, which by itself has an intrinsic property to aggregate into fibrils [111]. Indeed, Josephin domain self-association may play an important role in the process of aggregate formation by mutant ataxin-3 [112]. The potential role of N-terminal fragments in SCA3 was recently investigated in a mouse model expressing an ataxin-3 fragment containing the first 259 amino acids, thus including the Josephin domain, the first UIM and part of the second UIM, but lacking the polyQ tract and third UIM. The mice in this study were phenotypically normal until approximately 9 months of age, when motor coordination deficits, such as tremors and gait ataxia, reminiscent of SCA3 became apparent and the mice died prematurely. A significant degree of neuronal cytoplasmic ataxin-3 inclusions along with neuronal death was present in the cerebellum and other SCA3-relevant brain regions of these mice [113]. The preferential cytoplasmic location of the N-terminal Josephin domain containing fragment can likely be explained by the presence of the two NES [27, 28].

Taken together, there is evidence that proteolytic cleavage of mutant ataxin-3 results in the formation of both N- and C-terminal fragments. Whilst the current consensus is that the C-terminal polyQ-containing fragments localised in the nucleus are likely the more toxic entity [78, 80, 96], there might well be a role for cytosolic N-terminal ataxin-3 fragments in SCA3 pathogenesis.

### Impaired Protein Degradation in SCA3

Though ubiquitin chain proteolytic activity does not appear to vary between wild-type and mutant ataxin-3 [32], a widespread reduction of protein deubiquitination was reported in a mutant ataxin-3 cell model [32]. This potential loss of deubiquitination function in SCA3 might in part be explainedby trapping of ataxin-3 and various other components of the proteasomal machinery in the large ubiquitin-rich aggregates [16, 114]. Mutant ataxin-3 binds the ERAD component VCP/ p97 more efficiently than wild-type ataxin-3, probably because of conformational changes [50, 55, 115]. In spite of this stronger binding, mutant ataxin-3 seems to interfere with the degradation of target substrates [55, 116]. Additionally, N-terminal ataxin-3 fragments of 259 amino acids lacking the VCP/p97-binding domain were found to result in ER stress and impaired ER-mediated unfolded protein response when expressed in a mouse model, though ERAD component levels appeared unchanged [113].

Not only ER degradation is altered in SCA3 but also autophagy, in which the degradation of cellular components through the lysosomal machinery is impaired (Fig. 2). Aggregates of mutant ataxin-3 were found to contain molecular components involved in autophagy. For instance, beclin-1, a protein crucial in the autophagy pathway was found to be trapped in protein aggregates in SCA3 brains [117]. In a rat model overexpressing mutant ataxin-3, increased beclin-1 expression resulted in clearance of the mutant protein [117]. This observation is in accordance with impairments in autophagy seen in other neurodegenerative disorders [118], and the fact that stimulation of autophagy was found to alleviate symptoms in vivo [119].

These observations suggest that the SCA3 pathology may partly be the result of loss of function of ERAD machinery as well as compromised autophagy, together resulting in impaired protein degradation, accumulation of ubiquitinated proteins, and cellular stress.

### Mitochondrial Dysfunction in SCA3

It has been suggested that in polyQ disorders increasing oxidative stress and inability to protect against free radicals with age could lead to mitochondrial dysfunction and cell damage [120â€“123]. In accordance with this, a cell model overexpressing mutant ataxin-3 with 78 CAGs showed reduced antioxidant enzyme levels, increased mitochondrial DNA damage, and reduced energy supply, which indicates impaired mitochondrial function [124]. Recently, mitochondrial DNA damage was also seen in SCA3 transgenic mice expressing full-length ataxin-3 with 98 to 104 glutamines [125]. In the disease-affected pontine nuclei of these transgenic SCA3 mice, less mitochondrial DNA copies were seen, as compared to the unaffected hippocampus [125]. Additionally, less mitochondrial DNA copy numbers were observed in the mutant cells and SCA3 patient samples, implying mitochondrial DNA damage due to oxidative stress [124]. In line with this, the antioxidant enzyme superoxide dismutase is downregulated in pontine brain tissue of SCA3 patients [62], suggesting diminished antioxidant enzyme function.

Additionally, mitochondrial dysfunction was verified by the finding that the mitochondrial respiratory chain complex II activity was somewhat compromised in SCA3 [126]. As damaged mitochondria will not be able to scavenge free radicals and prevent cell energy impairment as effectively, this process may therefore further increase oxidative stress in the cell. Oxidative stress is then able to modulate vital cellular functions, potentially resulting in activation of apoptosis or excitotoxicity, two of the main causes of neuronal death [127]. Although there is not much information regarding altered mitochondrial activity in SCA3, the above described findings indicate that, like other polyQ disorders, defects in the cellular defence mechanism against oxidative stress could play a role in the pathogenesis of SCA3.

### Transcriptional Deregulation in SCA3

Since ataxin-3 displays DNA-binding properties and interacts with transcriptional regulators, transcriptional deregulation has been suggested to play a central role in the SCA3 pathogenesis [128]. In SCA3 and other polyQ disorders, transcription factors, together with polyQ proteins, are sequestered into nuclear aggregates, resulting in deregulation of their function as transcriptional co-repressor or activator [82, 83]. Transcription of genes involved in inflammatory processes, cell signaling and cell surface-associated proteins were found to be altered in SCA3 cell and mouse models, suggesting transcriptional deregulation in SCA3 [129â€“131]. Likewise, some corresponding proteins like MMP-2, amyloid Î²-protein and interleukins were found to be significantly increased in SCA3 patient brain material [129]. However, thus far no gene expression studies have been performed on SCA3 patient material to replicate above described findings.

Ataxin-3 was shown to inhibit histone acetylase activity. However, when mutated, this transcriptional repression is impaired, and increased acetylation of total histone H3 was indeed observed in mutant ataxin-3 overexpressing cells and human SCA3 brain material, resulting in an increase of transcription in SCA3 cells [61]. This transcriptional upregulation was supported by the discovery that in cells overexpressing mutant ataxin-3, MMP-2 was found upregulated [130].

Although in SCA3 the compromised transcriptional regulation currently has been less elucidated than the impaired protein degradation, the discovery of altered transcription of various genes suggests a role of transcriptional deregulation in SCA3 pathogenesis.

### RNA Toxicity in SCA3

Until recently, it was believed that polyQ disorders are solely the result of gain of toxic protein function and, to a lesser
extent, loss of wild-type protein function. However, more and more observations suggest that next to protein toxicity, there is also RNA toxicity involved in the polyQ disease pathogenesis. Together with the observation that, of all polyQ disorder genes, ataxin-3 has the highest pathogenic CAG repeat threshold, there is increasing evidence that RNA toxicity also plays a role. This RNA toxicity could be involved in SCA3 pathology through alternative splicing, bidirectional transcription, involvement of the RNA interference pathway, as well as repeat-associated non-ATG-initiated translation.

### CAG Repeat Toxicity and Formation of Alternatively Spliced Transcripts

Next to gain of toxic polyQ protein function, the expanded CAG repeat itself was recently also suggested to play a role in the SCA3 pathogenesis. *Drosophila melanogaster* transgenes with a pure expanded CAG repeat in the 3â€² UTR of ataxin-3 displayed progressive neural dysfunction. CAG repeats interspersed by CAA codons resulted in only a mild phenotype, indicating the importance of the expanded pure CAG repeat for the toxic phenotype [132]. However, *C. elegans* expressing an untranslated CAG repeat with 83 CAGs did not appear to have any phenotype, whereas 200 CAGs resulted in premature death [133]. Furthermore, in another *D. melanogaster* model of CAG toxicity, expressing a construct with a premature termination codon before a 93 CAG repeat, no phenotype was shown, arguing in favour of polyQ protein toxicity over RNA toxicity [134]. Therefore, the size of the CAG repeat is probably critical for the contribution of RNA toxicity from the expanded CAG repeat and since in the SCA3 patient population the repeat is between 52 and 86 CAGs, this CAG repeat toxicity may not be contributing to the observed pathology.

There is some evidence that RNA toxicity may result from stable RNA hairpin structures which can sequester RNA binding proteins, resulting in misregulation of alternative splicing. Both expanded CAG and CUG-containing RNA molecules were found to co-localise with the muscleblind-like 1 (MBNL1) splicing factor in RNA foci in both muscle and neuron nuclei [133, 135â€“137]. This functional inactivation of MBNL1 due to recruitment in nuclear foci resulted in misregulation of alternative splicing, which could be partly reversed by MBNL1 overexpression [138]. The potential involvement of this alternative splicing in the SCA3 pathology was shown in a *D. melanogaster* model of SCA3 where upregulation of MBNL1 was found to increase mutant ataxin-3-induced toxicity [132]. In the same study, Li et al. [132] showed that SCA3 flies with a coding CAG repeat showed a more severe phenotype than flies with CAG repeats interrupted by CAA triplets, probably because of MBNL1 upregulation in pure CAG repeat-expressing flies. However, in the CTG expansion disorder myotonic dystrophy type 1 (DM1) the opposite occurs, where overexpression of MBNL1 rescues the DM1 phenotype in myoblasts [139].

Although some data is available arguing for the involvement of RNA toxicity in polyQ disorders, the exact mechanism of this expanded CAG repeat-induced toxicity, perhaps through deregulation of splicing, remains elusive in SCA3 pathology.

#### Bidirectional Transcription

For many genes in the genome RNA transcription occurs from both DNA strands [140, 141]. Several studies have shown that bidirectional transcription is involved in the triplet repeat disorders [142â€“146]. In SCA7, which is caused by a CAG repeat expansion in the *ATXN7* gene, an alternative promoter for an antisense noncoding RNA was found [146]. This SCA7 antisense noncoding transcript 1 (SCAANT1) was found to regulate sense *ATNX7* gene expression. More recently, two natural huntingtin antisense (HTTAS) transcripts were identified at the *HTT* locus [143]. One of the HTTAS contained the CTG repeat and this HTTAS was strongly reduced in HD brains. It was suggested that HTTAS negatively regulates huntingtin transcript expression [143].

For HD Like 2, a bacterial artificial chromosomes (BAC) mouse model with a pathogenic CTG repeat on the sense strand and expanded CAG repeat on the antisense strand at the *Junctophilin-3* locus showed RNA toxicity caused by its untranslated expanded CUG repeat as well as protein toxicity by its polyQ translated expanded CAG repeat [145].

These findings suggest that in triplet repeat disorders bidirectional RNA transcription could play a role in the disease pathology by either direct CUG toxicity or deregulation of the sense transcript. Interestingly, from all nine polyQ disorders, ataxin-3 was found to have one of the highest relative amounts of antisense transcriptional tags [140]. Because of the involvement of antisense transcripts in triplet repeat disorders, additional research is worthwhile to unravel the potential involvement of bidirectional transcription in SCA3.

#### Involvement of the RNA Interference Pathway

Recently, short CAG repeat-containing RNAs of around 21 nucleotides, originating from mutant CAG repeat-containing RNAs were found to induce cell death [147]. The toxicity was CAG repeat-dependent, since toxicity was blocked by antisense oligonucleotides against the CAG repeat sequence. These short RNAs were dicer-dependent and the mutant short CAG repeat-containing RNAs were involved in RNA interference (RNAi)-induced gene silencing of CUG repeat-encoding transcripts.

Next to small RNAs, microRNAs are also suggested to be involved in triplet repeat toxicity. MicroRNAs are small non-coding single stranded RNAs which can regulate gene expression. In DM1, the previously described splicing factor MBNL1 was found to be involved in cytoplasmic regulation of microRNAs and alteration of this microRNA processing pathway was involved in the RNA toxicity [148]. In HD, over 50 microRNAs were reported to be deregulated and could contribute to the pathogenesis in HD [149]. In a transgenic SCA3 animal, inhibition of microRNA processing strongly enhanced ataxin-3-induced neurodegeneration [150]. Furthermore, in cells overexpressing mutant ataxin-3, blocking of microRNA processing resulted in an increased toxicity [150].

Above findings advocate that both short RNAs and microRNAs contribute to the pathology seen in SCA3 and other polyQ disorders. However, the scale and importance of the involvement of RNAi pathway in the SCA3 pathogenesis remains elusive.

### Repeat-Associated non-ATG-Initiated Translation

Several reading frame variants of genes resulting in formation of alternative proteins responsible for polyQ diseases have been found, which can be categorised as (I) translational reading frame shifting and (II) repeat-associated non-ATG translation. Translational reading frame shifting was suggested using antibodies at the C-terminus in various reading frames of huntingtin and ataxin-3 [151, 152]. Next to expanded polyQ repeats, also polyalanine (polyA) stretches occur in cells derived from HD and SCA3 patients [151, 152]. These polyA stretches in the full-length ataxin-3 protein were toxic when overexpressed in *D. melanogaster* and neuronal models [153].

Repeat-associated non-ATG translation was recently proposed as a novel class of protein toxicity, in which coding RNA transcripts with mutant CAG repeats are translated in the absence of an ATG start codon [154]. This repeat-associated non-ATG translation was found in all three possible CAG repeat reading frames, resulting in the translation of proteins with polyQ, polyA, and polyserine (polyS) repeats [154]. Repeat-associated non-ATG translation together with bidirectional transcription results in seven potential reading frames translating: ATG translated polyQ proteins, non-ATG translated polyQ, polyA and polyS proteins and bidirectional non-ATG translated polyleucine (polyL), polycysteine (polyC) and polyA proteins [155]. Together with sense CAG and antisense CTG transcription, this means a total of nine possible toxic entities are potentially involved in the pathogenesis of SCA3 and other polyQ disorders.

---

### Potential Therapeutic Strategies

To date, only symptomatic treatment is available for polyQ disorders. For SCA3, pharmacological strategies have been developed to reduce the depression, Parkinsonian phenotype, restless leg syndrome and sleepiness [156â€“158]. Several clinical trials with small patient numbers have been carried out and are reviewed in Refs. [159, 160]. Clearly, there is demand for a SCA3 therapy directed at preventing or slowing the progression of neurodegeneration. For this reason, further identification of the key molecular players in the cascade leading to neuronal dysfunction and cell death in SCA3 is paramount for the development of effective therapeutic drugs. Current potential therapeutic outlooks targeting various known processes in SCA3 pathogenesis are discussed below.

#### Activation of Autophagy and the Ubiquitinâ€“Proteasome System

As silencing of mutant ataxin-3 after disease onset has been shown to rescue, to some extent, the neuropathology in a SCA3 mouse model, it can be postulated that cells are able to clear the toxic products when expression of the mutant transgene is stopped [71, 161]. For this reason, upregulation of autophagy and clearance through the ubiquitinâ€“proteasome system may be an attractive strategy to enable cells to better cope with accumulating polyQ expanded protein. Indeed, overexpression of the beclin-1 autophagic protein led to increased clearance of mutant ataxin-3 and prevented neurodegeneration in rats [117]. Also, in SCA3 mice, upregulation of autophagy by rapamycins resulted in a reduction of aggregates and the level of soluble mutant ataxin-3, whilst the level of wild-type protein appeared unaffected [162]. Similar observations have been made in HD fly and mouse models as well [119]. Targeting autophagic pathways may therefore be a suitable treatment strategy for SCA3, and may even be beneficial after disease onset. Temsirolimus, the compound extensively used to increase autophagy in mouse models, is already in use for treatment of renal cell carcinoma and as such is known to be suitable for use in patients and might therefore be a candidate for clearance modulation based treatment of SCA3 [162]. Also, lithium chloride was shown to have therapeutic potential in a *D. melanogaster* SCA3 model. The mechanism of lithium is thought to rely on upregulation of autophagy, though anti-apoptotic effects have also been implicated [163]. A phase II/III clinical trial with lithium carbonate was recently carried out for SCA3 and showed significant reduction in the progression of gait ataxia severity [164]. However, due to the limited group size, lithium carbonate did not show a significant effect on disease progression as determined by the neurological examination score for spinocerebellar ataxia. A larger clinical trial will have to be

carried out to better assess the viability of lithium as effective treatment for SCA3. Recently, H1152 was shown to be capable of ameliorating neuronal death and the neurological phenotype of SCA3 mice [165]. In this study, H1152 specifically reduced mutant ataxin-3 protein levels after intraperitoneal injection, whilst non-expanded ataxin-3 levels remained unaffected. The reduction of mutant ataxin-3 likely occurred through augmentation of the proteasome activity, thus promoting protein degradation [165]. Based on these first results, H1152 appears a promising compound for application in SCA3.

### Antioxidants

There is some evidence that defects in the cellular defence mechanism against oxidative stress could play a role in neuronal dysfunction and neurodegeneration in SCA3. Though the extent of oxidative stress involvement is not exactly known, antioxidants might be able to provide some neuroprotective effects. Free radical scavengers are able to attenuate accumulation of reactive oxygen species, and can be divided in enzymatic and non-enzymatic antioxidants. A whole range of these antioxidants have been described, providing ample opportunity for therapeutic targeting. Antioxidant-based therapies for SCA3 have been scarcely researched, but some useful lessons can be learned from the HD research field (reviewed in [166]). Notably, supplementation of the naturally occurring antioxidant creatine was shown to slow ongoing cortical atrophy in a 16-week randomised double-blind phase II clinical trial with HD patients and is therefore currently being tested in a phase III clinical trial [167]. Also, coenzyme Q10, a lipid-soluble benzoquinone possessing antioxidant potential when reduced to ubiquinol, showed some beneficial effects in HD, but did not significantly change the functional capacity score, which was the primary outcome for the trial [168]. A higher-dose clinical trial of coenzyme Q10 is currently being carried out in order to determine whether greater efficacy can be achieved. Results from these and other clinical trials with antioxidants in HD may provide interesting insights into the feasibility of antioxidant therapy for SCA3.

Though multiple clinical trials using Dantrolene for other diseases show no adverse events [171, 172], studies testing the inhibition of excessive calcium release using Dantrolene in SCA3 patients have not been performed yet. Caffeine, likely operating through adenosine $A_{2A}$ receptor antagonism [173], has shown promise in decreasing neuropathology in a SCA3 mouse model [174]. Though the mechanism of this protective effect of caffeine has not been elucidated, it can be hypothesised that the $A_{2A}$ receptor antagonism is able to normalise glutamergic transmission [175], which might in turn prevent the calcium-dependent calpain proteolysis and aggregation of mutant ataxin-3 [94]. Irrespective of the mechanism at work, caffeine consumption may be a safe and easily implementable prophylactic strategy to delay SCA3 onset [174].

#### Preventing Transcriptional Deregulation

Transcriptional deregulation resulting from mutant ataxin-3 has been observed in a SCA3 mouse model, where mRNA expression of proteins involved in signal transduction, calcium mobilisation and neuronal differentiation were found to be downregulated [131]. Given the function of the affected proteins, it might be possible that this transcriptional misregulation contributes to the pathogenic burden in SCA3. For this reason, the HDAC inhibitor sodium butyrate was tested for efficacy in SCA3 transgenic mice, and was indeed shown to reverse the mutant ataxin-3-induced histone hypoacetylation and transcriptional downregulation in the cerebellum. Also, treatment with sodium butyrate significantly improved the ataxic symptoms of these mice, thus showing good therapeutic potential [176]. In earlier mouse studies, sodium butyrate has shown good capacity in reaching the brain [177] and caused low toxicity when tested in a clinical trial for beta thalassemia [178] and a dose finding study for HD [179]. Sodium butyrate therefore seems to be a compound useful for further research with regard to efficacy for SCA3 treatment.

#### Proteolytic Cleavage Inhibition

Preventing the formation of the cleavage-induced, potentially toxic, mutant ataxin-3 fragments has been suggested as a potential therapy for SCA3. Specific caspase inhibitors have been implemented in a mouse model of HD, and were indeed found to be capable of slowing disease progression [103]. Other caspase inhibitors have been investigated in clinical trials for hepatic impairment and liver transplantation, and appear to be well tolerated [180, 181]. However, caspase functioning in the brain is complex and may be important for normal brain functioning by influencing apoptosis,

#### Modulation of Calcium Homeostasis

Altered cellular calcium homeostasis is suggested to play a role in SCA3 pathology. Mutant ataxin-3 was found to bind and activate the intracellular calcium release channel inositol 1,4,5-trisphosphate receptor type 1 [169]. Inhibiting excessive calcium release using Dantrolene in transgenic SCA3 mice improved motor performance and prevented neuronal cell loss [169]. In neuronal cells, Dantrolene was found to have an inhibitory effect on calcium release from the ER [170].

synaptic plasticity, dendritic development and formation of memory [182, 183]. Use of caspase inhibitors in the brain might interfere with these processes and would therefore be impractical as treatment for SCA3. Inhibition of calpains has shown good potential in reducing mutant ataxin-3 toxicity in various cell and animal models [94, 97, 106, 107]. Several types of calpain inhibitors targeting the active site of the enzyme are available, but lack specificity amongst calpain isoforms and other proteolytic enzymes [184, 185] and are therefore likely not suitable as potential therapy for SCA3.

### Preventing Aggregation

As described previously, the ataxin-3 protein aggregates are a hallmark of SCA3, and it is therefore likely that the process responsible for toxicity of mutant ataxin-3 also leads to formation of aggregates. Indeed, increased levels of certain molecular chaperones, such as HSP40 and HSP70, have been used to reduce both aggregation and toxicity of expanded polyQ tracts in several cell and mouse models, resulting in improved phenotypes [186, 187]. There are indications that these molecular chaperones increase the solubility of expanded polyQ [188, 189], which might in turn result in an increased degradation of the protein by the proteasome [190, 191], thereby alleviating toxicity. Trimethylamine N-oxide, glycerol and dimethyl sulfoxide are chemical chaperones that are able to influence protein folding and stabilise proteins in their native conformation. These chemical chaperones have been tested for their efficacy in cells overexpressing a truncated form of mutant ataxin-3, and both aggregate formation and cytotoxicity were reduced [192]. Whilst providing support for inhibition of aggregation as a therapeutic strategy, these chemicals are not fit for use in a clinical setting because they seem to be cytotoxic at concentrations required for potent suppression aggregation.

Some small molecules, such as Congo red, minocycline and chlorpromazine are useful inhibitors of aggregation in vitro, and have been tested in HD mouse models. Phenotypical rescue in these studies, however, was rather limited or absent [193â€“195]. Another line of research into inhibition of polyQ aggregation focusses on the use of vector encoded small antibody fragments called intrabodies. For huntingtin, intrabodies have been discovered that are capable of reducing neuropil aggregates and improve neurological symptoms in HD mouse models [196, 197]. For ataxin-3, only an aggregate-exacerbating intrabody against fibrillar polyQ proteins has been tested to date, which causes an increased cytotoxicity and cell death when combined with mutant ataxin-3 [198]. An intrabody capable of binding mutant ataxin-3 in a manner that decreases aggregation might thus be a potential candidate for relieving toxicity.

In spite of the several approaches targeting mutant ataxin-3 aggregation, no promising candidate for implementation in the clinic has been identified yet.

#### Preventing Formation Mutant Ataxin-3

From a therapeutic standpoint, an advantage of monogenetic disorders such as SCA3 is that silencing of the responsible gene should result in alleviation of mutant protein toxicity. As potential gene-silencing treatment for SCA3, non-allele-specific downregulation of all ataxin-3 transcripts have been tested in both wild-type and SCA3 rats [68]. This allele-nonspecific silencing of ataxin-3 in the striatum showed no adverse effects in both wild-type and SCA3 rats, and the SCA3 rats displayed locally reduced neuropathology.

However, because of the discussion regarding the importance of wild-type ataxin-3 functioning, an allele-specific downregulation targeting only the mutant allele is a more elegant and favourable approach for therapeutic application in SCA3, as has been suggested for monogenetic neurodegenerative diseases in general [199, 200]. Thus far, two strategies for allele-specific downregulation of mutant ataxin-3 have been postulated.

Firstly, allele-specific silencing was achieved through the RNAi pathway using shRNAs directed against a single nucleotide polymorphisms (SNP) unique to the mutant ataxin-3 transcript [201]. This targeted SNP at the 3â€² end of the *ATXN3* gene was found to be present in over 70 % of SCA3 patients [202]. The SNP-specific shRNA was able to specifically silence mutant ataxin-3 and was found to be neuroprotective in SCA3 mouse and rat models [161, 201], thus showing good promise for clinical implementation. The first trial with nanoparticle delivered siRNA for treatment of cancer has recently shown good promise for RNAi as a potential therapeutic agent regulating gene expression [203]. Suitable delivery of RNAi therapeutics in the central nervous system, however, will first need to be optimised before a similar strategy can be attempted for SCA3.

The second approach for an allele-specific reduction of mutant ataxin-3 is aimed at targeting the expanded CAG repeat directly. Small molecules, such as antisense oligonucleotides (AONs) or peptide nucleic acids (PNAs), can in vitro achieve allele-specific silencing of mutant ataxin-3 by binding to the expanded CAG repeat, probably resulting in translational blockage of mutant ataxin-3 [204, 205]. RNAi-based approaches with abasic substitutions [206] and single-stranded silencing RNAs (ssRNAs) [207] have also been shown to be capable of allele selectively inhibiting ataxin-3 expression. A main concern regarding these CAG repeat-targeting small molecules is their specificity. There are several endogenous CAG repeat-containing transcripts, like huntingtin, TBP and other ataxins, of which downregulation

might be detrimental. However, in vitro there was no evidence that endogenous CAG repeat-containing transcripts are down-regulated when targeting the expanded CAG repeat directly [204], but future in vivo studies will need to provide more insight into the possible off-target effects when using CAG-specific approaches.

Next to reducing ataxin-3, an alternative oligonucleotide therapy was proposed in which the polyQ repeat is removed from the ataxin-3 protein by exon skipping [40]. With exon skipping, AONs are used that target-specific splicing signals, masking an exon in the pre-mRNA from the splicing machinery and thus resulting in exclusion of the target exon from the mRNA [208, 209]. When the correct reading frame is maintained, subsequent translation will result in formation of an internally truncated protein. Using this exon skipping technique, the toxic polyQ expansion can be removed from ataxin-3, whilst ataxin-3 expression levels remain unaltered. Exon skipping of the polyQ encoding region of ataxin-3 has recently been tested, and was shown to indeed result in formation of a shorter ataxin-3 protein lacking the polyQ repeat. The shorter ataxin-3 protein retained the main functional domains and ubiquitin binding capacity, and was not toxic in cells [40]. This same exon skipping approach was later confirmed as secondary effect of the previously described ssRNAs directed against the CAG repeat [207].

The main advantage of AONs over siRNAs is that they are efficiently taken up by neuronal cells in vivo [209], and encouraging results using AONs have been obtained in animal models for other neurodegenerative disorders like HD and amyotrophic lateral sclerosis (ALS). In HD, after intrathecal injection, AONs were shown to diffuse throughout the non-human primate brain and could be detected in neurons affected most in SCA3, which are the cerebellum, pons, midbrain and spinal cord [210]. The safety of intrathecal injection of AONs in humans was demonstrated in a phase I trial, where no serious adverse effects were observed in AON-treated ALS patients [211]. Additionally, exon skipping is currently being tested in clinical trials for treatment of Duchenne muscular dystrophy [212].

In sum, the various oligonucleotide-based approaches in development, together with the lessons learned from other neurodegenerative disorders concerning safety, potency and distribution of oligonucleotides in the brain validates the encouraging prospective of oligonucleotide-based therapeutics as treatment for SCA3.

with a function in regulating protein degradation. More recently, ataxin-3 functioning at the level of transcriptional regulation has been shown, and the extensive list of identified protein interactors hint towards yet unknown roles of ataxin-3. In parallel with the other polyQ disorders, no clear model has been established as to how polyQ expanded ataxin-3 leads to the specific symptoms of SCA3. Though the hallmark of polyQ disorders, the intracellular aggregates, are no longer regarded as the main toxic entity, the pathways leading to their formation are still being researched extensively. The involvement of proteolytic cleavage leading to toxic, potentially aggregation-prone fragments has gained increasing evidence in cell and animal models. Also, with the advance in molecular techniques, a possible role of RNA toxicity is emerging. The many pathogenic mechanisms possibly involved in SCA3 thus clearly require further research, hopefully opening new avenues for therapeutic strategies in the process. At the same time, recent advances in oligonucleotide-based therapies may well circumvent the need of a full mechanistic understanding of disease pathways, and the polyQ disorders are ideal candidates to benefit from promising new molecular therapies.

Acknowledgments The authors are supported by ZonMw (the Netherlands), Hersenstichting (the Netherlands), The Brugling Fund (the Netherlands), AtaxiaUK (United Kingdom), patiÃ«ntenvereniging Autosomaal Dominante Cerebellaire Ataxia (ADCA) (the Netherlands), Center for Biomedical Genetics (the Netherlands), Center for Medical Systems Biology (OCM/IZ), and Integrated European Project in Omics Research of Rare Neuromuscular and Neurodegenerative Diseases (Neuromics).

Open Access This article is distributed under the terms of the Creative Commons Attribution License which permits any use, distribution, and reproduction in any medium, provided the original author(s) and the source are credited.

References

1. Haberhausen G, Damian MS, Leweke F, Muller U (1995) Spinocerebellar ataxia, type 3 (SCA3) is genetically identical to Machadoâ€“Joseph disease (MJD). J Neurol Sci 132:71â€“75
2. Ranum LP, Lundgren JK, Schut LJ, Ahrens MJ, Perlman S, Aita J, Bird TD, Gomez C, Orr HT (1995) Spinocerebellar ataxia type 1 and Machadoâ€“Joseph disease: incidence of CAG expansions among adult-onset ataxia patients from 311 families with dominant, recessive, or sporadic ataxia. Am J Hum Genet 57:603â€“608
3. Silveira I, Lopes-Cendes I, Kish S, Maciel P, Gaspar C, Coutinho P, Botez MI, Teive H, Arruda W, Steiner CE et al (1996) Frequency of spinocerebellar ataxia type 1, dentatorubropallidoluysian atrophy, and Machadoâ€“Joseph disease mutations in a large group of spinocerebellar ataxia patients. Neurology 46:214â€“218
4. Pringsheim T, Wiltshire K, Day L, Dykeman J, Steeves T, Jette N (2012) The incidence and prevalence of Huntingtonâ€™s disease: a systematic review and meta-analysis. Mov Disord 27:1083â€“1091
5. Coutinho P, Andrade C (1978) Autosomal dominant system degeneration in Portuguese families of the Azores Islands. A new genetic disorder involving cerebellar, pyramidal, extrapyramidal and spinal cord motor functions. Neurology 28:703â€“709

Final Remarks

Since the discovery of the causative gene for SCA3 in 1994, the *ATXN3* gene and resulting protein have been subject to extensive research over the last 2 decades. In this period, ataxin-3 has been established as a deubiquitinating enzyme

6. Bauer PO, Nukina N (2009) The pathogenic mechanisms of polyglutamine diseases and current therapeutic strategies. J Neurochem 110:1737â€“1765

7. Rosenberg RN (1992) Machado-Joseph disease: an autosomal dominant motor system degeneration. Mov Disord 7:193â€“203

8. Soong B, Cheng C, Liu R, Shan D (1997) Machado-Joseph disease: clinical, molecular, and metabolic characterization in Chinese kindreds. Ann Neurol 41:446â€“452

9. Teive HA, Munhoz RP, Arruda WO, Lopes-Cendes I, Raskin S, Werneck LC, Ashizawa T (2012) Spinocerebellar ataxias: genotype-phenotype correlations in 104 Brazilian families. Clinics (Sao Paulo) 67:443â€“449

10. Riess O, Rub U, Pastore A, Bauer P, Schols L (2008) SCA 3: neurological features, pathogenesis and animal models. Cerebellum 7:125â€“137

11. Kawaguchi Y, Okamoto T, Taniwaki M, Aizawa M, Inoue M, Katayama S, Kawakami H, Nakamura S, Nishimura M, Akiguchi I (1994) CAG expansions in a novel gene for Machado-Joseph disease at chromosome 14q32.1. Nat Genet 8:221â€“228

12. Durr A, Stevanin G, Cancel G, Duyckaerts C, Abbas N, Didierjean O, Chneiweiss H, Benomar A, Lyon-Caen O, Julien J et al (1996) Spinocerebellar ataxia 3 and Machado-Joseph disease: clinical, molecular, and neuropathological features. Ann Neurol 39:490â€“499

13. Padiath QS, Srivastava AK, Roy S, Jain S, Brahmachari SK (2005) Identification of a novel 45 repeat unstable allele associated with a disease phenotype at the MJD1/SCA3 locus. Am J Med Genet B Neuropsychiatr Genet 133B:124â€“126

14. Carvalho DR, La Rocque-Ferreira A, Rizzo IM, Imamura EU, Speck-Martins CE (2008) Homozygosity enhances severity in spinocerebellar ataxia type 3. Pediatr Neurol 38:296â€“299

15. Maciel P, Gaspar C, DeStefano AL, Silveira I, Coutinho P, Radvany J, Dawson DM, Sudarsky L, Guimaraes J, Loureiro JE (1995) Correlation between CAG repeat length and clinical features in Machado-Joseph disease. Am J Hum Genet 57:54â€“61

16. Paulson HL, Perez MK, Trottier Y, Trojanowski JQ, Subramony SH, Das SS, Vig P, Mandel JL, Fischbeck KH, Pittman RN (1997) Intranuclear inclusions of expanded polyglutamine protein in spinocerebellar ataxia type 3. Neuron 19:333â€“344

17. Rub U, de Vos RA, Brunt ER, Sebesteny T, Schols L, Auburger G, Bohl J, Ghebremedhin E, Gierga K, Seidel K et al (2006) Spinocerebellar ataxia type 3 (SCA3): thalamic neurodegeneration occurs independently from thalamic ataxin-3 immunopositive neuronal intranuclear inclusions. Brain Pathol 16:218â€“227

18. Bettencourt C, Santos C, Montiel R, Costa MC, Cruz-Morales P, Santos LR, Simoes N, Kay T, Vasconcelos J, Maciel P, Lima M (2010) Increased transcript diversity: novel splicing variants of Machado-Joseph disease gene (ATXN3). Neurogenetics 11: 193â€“202

19. Harris GM, Dodelzon K, Gong L, Gonzalez-Alegre P, Paulson HL (2010) Splice isoforms of the polyglutamine disease protein ataxin-3 exhibit similar enzymatic yet different aggregation properties. PLoS One 5:e13695

20. Schmidt T, Landwehrmeyer GB, Schmitt I, Trottier Y, Auburger G, Laccone F, Klockgether T, Volpel M, Epplen JT, Schols L, Riess O (1998) An isoform of ataxin-3 accumulates in the nucleus of neuronal cells in affected brain regions of SCA3 patients. Brain Pathol 8:669â€“679

21. Trottier Y, Cancel G, An-Gourfinkel I, Lutz Y, Weber C, Brice A, Hirsch E, Mandel JL (1998) Heterogeneous intracellular localization and expression of ataxin-3. Neurobiol Dis 5:335â€“347

22. Pozzi C, Valtorta M, Tedeschi G, Galbusera E, Pastori V, Bigi A, Nonnis S, Grassi E, Fusi P (2008) Study of subcellular localization and proteolysis of ataxin-3. Neurobiol Dis 30:190â€“200

23. Tait D, Riccio M, Sittler A, Scherzinger E, Santi S, Ognibene A, Maraldi NM, Lehrach H, Wanker EE (1998) Ataxin-3 is transported into the nucleus and associates with the nuclear matrix. Hum Mol Genet 7:991â€“997

24. Paulson HL, Das SS, Crino PB, Perez MK, Patel SC, Gotsdiner D, Fischbeck KH, Pittman RN (1997) Machado-Joseph disease gene product is a cytoplasmic protein widely expressed in brain. Ann Neurol 41:453â€“462

25. Todi SV, Laco MN, Winborn BJ, Travis SM, Wen HM, Paulson HL (2007) Cellular turnover of the polyglutamine disease protein ataxin-3 is regulated by its catalytic activity. J Biol Chem 282: 29348â€“29358

26. Albrecht M, Golatta M, Wullner U, Lengauer T (2004) Structural and functional analysis of ataxin-2 and ataxin-3. Eur J Biochem 271:3155â€“3170

27. Antony PM, Mantele S, Mollenkopf P, Boy J, Kehlenbach RH, Riess O, Schmidt T (2009) Identification and functional dissection of localization signals within ataxin-3. Neurobiol Dis 36:280â€“292

28. Macedo-Ribeiro S, Cortes L, Maciel P, Carvalho AL (2009) Nucleocytoplasmic shuttling activity of ataxin-3. PLoS One 4: e5834

29. Breuer P, Haacke A, Evert BO, Wullner U (2010) Nuclear aggregation of polyglutamine-expanded ataxin-3: fragments escape the cytoplasmic quality control. J Biol Chem 285:6532â€“6537

30. Mueller T, Breuer P, Schmitt I, Walter J, Evert BO, Wullner U (2009) CK2-dependent phosphorylation determines cellular localization and stability of ataxin-3. Hum Mol Genet 18:3334â€“3343

31. Fei E, Jia N, Zhang T, Ma X, Wang H, Liu C, Zhang W, Ding L, Nukina N, Wang G (2007) Phosphorylation of ataxin-3 by glycogen synthase kinase 3beta at serine 256 regulates the aggregation of ataxin-3. Biochem Biophys Res Commun 357:487â€“492

32. Winborn BJ, Travis SM, Todi SV, Scaglione KM, Xu P, Williams AJ, Cohen RE, Peng J, Paulson HL (2008) The deubiquitinating enzyme ataxin-3, a polyglutamine disease protein, edits Lys63 linkages in mixed linkage ubiquitin chains. J Biol Chem 283:26436â€“26443

33. Burnett B, Li F, Pittman RN (2003) The polyglutamine neurodegenerative protein ataxin-3 binds polyubiquitylated proteins and has ubiquitin protease activity. Hum Mol Genet 12:3195â€“3205

34. do Carmo CM, Bajanca F, Rodrigues AJ, Tome RJ, Corthals G, Macedo-Ribeiro S, Paulson HL, Logarinho E, Maciel P (2010) Ataxin-3 plays a role in mouse myogenic differentiation through regulation of integrin subunit levels. PLoS One 5:e11728

35. Fang S, Weissman AM (2004) A field guide to ubiquitylation. Cell Mol Life Sci 61:1546â€“1561

36. Lim KL, Lim GG (2011) K63-linked ubiquitination and neurodegeneration. Neurobiol Dis 43:9â€“16

37. Mao Y, Senic-Matuglia F, Di Fiore PP, Polo S, Hodsdon ME, De CP (2005) Deubiquitinating function of ataxin-3: insights from the solution structure of the Josephin domain. Proc Natl Acad Sci USA 102:12700â€“12705

38. Nicastro G, Menon RP, Masino L, Knowles PP, McDonald NQ, Pastore A (2005) The solution structure of the Josephin domain of ataxin-3: structural determinants for molecular recognition. Proc Natl Acad Sci USA 102:10493â€“10498

39. Berke SJ, Chai Y, Marrs GL, Wen H, Paulson HL (2005) Defining the role of ubiquitin-interacting motifs in the polyglutamine disease protein, ataxin-3. J Biol Chem 280:32026â€“32034

40. Evers MM, Tran HD, Zalachoras I, Pepers BA, Meijer OC, den Dunnen JT, van Ommen GJ, Aartsma-Rus A, van Roon-Mom WM (2013) Ataxin-3 protein modification as a treatment strategy for spinocerebellar ataxia type 3: Removal of the CAG containing exon. Neurobiol Dis 58C:49â€“56

41. Nicastro G, Masino L, Esposito V, Menon RP, De SA, Fraternali F, Pastore A (2009) Josephin domain of ataxin-3 contains two distinct ubiquitin-binding sites. Biopolymers 91:1203â€“1214

42. Nicastro G, Todi SV, Karaca E, Bonvin AM, Paulson HL, Pastore A (2010) Understanding the role of the Josephin domain in the PolyUb binding and cleavage properties of ataxin-3. PLoS One 5:e12430

43. Jana NR, Dikshit P, Goswami A, Kotliarova S, Murata S, Tanaka K, Nukina N (2005) Co-chaperone CHIP associates with expanded polyglutamine protein and promotes their degradation by proteasomes. J Biol Chem 280:11635â€“11640

44. Durcan TM, Kontogiannea M, Thorarinsdottir T, Fallon L, Williams AJ, Djarmati A, Fantaneanu T, Paulson HL, Fon EA (2011) The Machadoâ€“Joseph disease-associated mutant form of ataxin-3 regulates Parkin ubiquitination and stability. Hum Mol Genet 20:141â€“154

45. Durcan TM, Kontogiannea M, Bedard N, Wing SS, Fon EA (2012) Ataxin-3 deubiquitination is coupled to Parkin ubiquitination via E2 ubiquitin-conjugating enzyme. J Biol Chem 287:531â€“541

46. Durcan TM, Fon EA (2013) Ataxin-3 and its e3 partners: implications for Machadoâ€“Joseph disease. Front Neurol 4:46

47. Scaglione KM, Zavodszky E, Todi SV, Patury S, Xu P, Rodriguez-Lebron E, Fischer S, Konen J, Djarmati A, Peng J et al (2011) Ube2w and ataxin-3 coordinately regulate the ubiquitin ligase CHIP. Mol Cell 43:599â€“612

48. Todi SV, Winborn BJ, Scaglione KM, Blount JR, Travis SM, Paulson HL (2009) Ubiquitination directly enhances activity of the deubiquitinating enzyme ataxin-3. EMBO J 28:372â€“382

49. Todi SV, Scaglione KM, Blount JR, Basrur V, Conlon KP, Pastore A, Elenitoba-Johnson K, Paulson HL (2010) Activity and cellular functions of the deubiquitinating enzyme and polyglutamine disease protein ataxin-3 are regulated by ubiquitination at lysine 117. J Biol Chem 285:39303â€“39313

50. Zhong X, Pittman RN (2006) Ataxin-3 binds VCP/p97 and regulates retrotranslocation of ERAD substrates. Hum Mol Genet 15: 2409â€“2420

51. Wang G, Sawai N, Kotliarova S, Kanazawa I, Nukina N (2000) Ataxin-3, the MJD1 gene product, interacts with the two human homologs of yeast DNA repair protein RAD23, HHR23A and HHR23B. Hum Mol Genet 9:1795â€“1803

52. Liu Y, Ye Y (2012) Roles of p97-associated deubiquitinases in protein quality control at the endoplasmic reticulum. Curr Protein Pept Sci 13:436â€“446

53. Boeddri ch A, Gaumer S, Haacke A, Tzvetkov N, Albrecht M, Evert BO, Muller EC, Lurz R, Breuer P, Schugardt N et al (2006) An arginine/lysine-rich motif is crucial for VCP/p97-mediated modulation of ataxin-3 fibrillogenesis. EMBO J 25:1547â€“1558

54. Wang Q, Li L, Ye Y (2006) Regulation of retrotranslocation by p97-associated deubiquitinating enzyme ataxin-3. J Cell Biol 174:963â€“971

55. Laco MN, Cortes L, Travis SM, Paulson HL, Rego AC (2012) Valosin-containing protein (VCP/p97) is an activator of wild-type ataxin-3. PLoS One 7:e43563

56. Shimohata T, Nakajima T, Yamada M, Uchida C, Onodera O, Naruse S, Kimura T, Koide R, Nozaki K, Sano Y et al (2000) Expanded polyglutamine stretches interact with TAFII130, interfering with CREB-dependent transcription. Nat Genet 26:29â€“36

57. McCampbell A, Taylor JP, Taye AA, Robitschek J, Li M, Walcott J, Merry D, Chai Y, Paulson H, Sobue G, Fischbeck KH (2000) CREB-binding protein sequestration by expanded polyglutamine. Hum Mol Genet 9:2197â€“2202

58. Chai Y, Shao J, Miller VM, Williams A, Paulson HL (2002) Live-cell imaging reveals divergent intracellular dynamics of polyglutamine disease proteins and supports a sequestration model of pathogenesis. Proc Natl Acad Sci USA 99:9310â€“9315

59. Li F, Macfarlan T, Pittman RN, Chakravarti D (2002) Ataxin-3 is a histone-binding protein with two independent transcriptional corepressor activities. J Biol Chem 277:45004â€“45012

60. Burnett BG (2005) The polyglutamine neurodegenerative protein ataxin 3 regulates aggresome formation. Proc Natl Acad Sci USA 102:4330â€“4335

61. Evert BO, Araujo J, Vieira-Saecker AM, de Vos RA, Harendza S, Klockgether T, Wullner U (2006) Ataxin-3 represses transcription via chromatin binding, interaction with histone deacetylase 3, and histone deacetylation. J Neurosci 26:11474â€“11486

62. Araujo J, Breuer P, Dieringer S, Krauss S, Dorn S, Zimmermann K, Pfeifer A, Klockgether T, Wuellner U, Evert BO (2011) FOXO4-dependent upregulation of superoxide dismutase-2 in response to oxidative stress is impaired in spinocerebellar ataxia type 3. Hum Mol Genet 20:2928â€“2941

63. Landschulz WH, Johnson PF, McKnight SL (1988) The leucine zipper: a hypothetical structure common to a new class of DNA binding proteins. Science 240:1759â€“1764

64. Orr HT (2012) Cell biology of spinocerebellar ataxia. J Cell Biol 197:167â€“177

65. Rodrigues AJ, do Carmo CM, Silva TL, Ferreira D, Bajanca F, Logarinho E, Logarinho P (2010) Absence of ataxin-3 leads to cytoskeletal disorganization and increased cell death. Biochim Biophys Acta 1803:1154â€“1163

66. Rodrigues AJ, Coppola G, Santos C, Costa MC, Ailion M, Sequeiros J, Geschwind DH, Maciel P (2007) Functional genomics and biochemical characterization of the C. elegans orthologue of the Machadoâ€“Joseph disease protein ataxin-3. FASEB J 21:1126â€“1136

67. Rodrigues AJ, Neves-Carvalho A, Teixeira-Castro A, Rokka A, Corthals G, Logarinho E, Maciel P (2011) Absence of ataxin-3 leads to enhanced stress response in C. elegans. PLoS One 6:e18512

68. Alves S, Nascimento-Ferreira I, Dufour N, Hassig R, Auregan G, Nobrega C, Brouillet E, Hantraye P, Pedroso de Lima MC, Deglon N, de Almeida LP (2010) Silencing ataxin-3 mitigates degeneration in a rat model of Machadoâ€“Joseph disease: no role for wild-type ataxin-3? Hum Mol Genet 19:2380â€“2394

69. Schmitt I, Linden M, Khazneh H, Evert BO, Breuer P, Klockgether T, Wuellner U (2007) Inactivation of the mouse Atxn3 (ataxin-3) gene increases protein ubiquitination. Biochem Biophys Res Commun 362:734â€“739

70. Switonski PM, Fiszer A, Kazmierska K, Kurpisz M, Krzyzosiak WJ, Figiel M (2011) Mouse ataxin-3 functional knock-out model. Neuromol Med 13:54â€“65

71. Boy J, Schmidt T, Wolburg H, Mack A, Nuber S, Bottcher M, Schmitt I, Holzmann C, Zimmermann F, Servadio A, Riess O (2009) Reversibility of symptoms in a conditional mouse model of spinocerebellar ataxia type 3. Hum Mol Genet 18:4282â€“4295

72. Warrick JM, Morabito LM, Bilen J, Gordesky-Gold B, Faust LZ, Paulson HL, Bonini NM (2005) Ataxin-3 suppresses polyglutamine neurodegeneration in Drosophila by a ubiquitin-associated mechanism. Mol Cell 18:37â€“48

73. Hubener J, Riess O (2010) Polyglutamine-induced neurodegeneration in SCA3 is not mitigated by non-expanded ataxin-3: conclusions from double-transgenic mouse models. Neurobiol Dis 38:116â€“124

74. Albrecht M, Hoffmann D, Evert BO, Schmitt I, Wullner U, Lengauer T (2003) Structural modeling of ataxin-3 reveals distant homology to adaptins. Proteins 50:355â€“370

75. Jana NR, Nukina N (2004) Misfolding promotes the ubiquitination of polyglutamine-expanded ataxin-3, the defective gene product in SCA3/MJD. Neurotoxicol Res 6:523â€“533

76. Schmitt I, Brattig T, Gossen M, Riess O (1997) Characterization of the rat spinocerebellar ataxia type 3 gene. Neurogenetics 1:103â€“112

77. Schmidt T, Lindenberg KS, Krebs A, Schols L, Laccone F, Herms J, Rechsteiner M, Riess O, Landwehrmeyer GB (2002) Protein surveillance machinery in brains with spinocerebellar ataxia type 3: redistribution and differential recruitment of 26S proteasome subunits and chaperones to neuronal intranuclear inclusions. Ann Neurol 51:302â€“310

78. Ikeda H, Yamaguchi M, Sugai S, Aze Y, Narumiya S, Kakizuka A (1996) Expanded polyglutamine in the Machadoâ€“Joseph disease protein induces cell death in vitro and in vivo. Nat Genet 13: 196â€“202

79. Evert BO, Wullner U, Schulz JB, Weller M, Groscurth P, Trottier Y, Brice A, Klockgether T (1999) High level expression of

expanded full-length ataxin-3 in vitro causes cell death and
formation of intranuclear inclusions in neuronal cells. Hum
Mol Genet 8:1169â€“1176
80. Bichelmeier U, Schmidt T, Hubener J, Boy J, Ruttiger L, Habig K,
Poths S, Bonin M, Knipper M, Schmidt WJ et al (2007) Nuclear
localization of ataxin-3 is required for the manifestation of symp-
toms in SCA3: in vivo evidence. J Neurosci 27:7418â€“7428
81. Nguyen HP, Hubener J, Weber JJ, Grueninger S, Riess O, Weiss A
(2013) Cerebellar soluble mutant ataxin-3 level decreases during
disease progression in Spinocerebellar Ataxia Type 3 mice. PLoS
One 8:e62043
82. van Roon-Mom WM, Reid SJ, Faull RL, Snell RG (2005) TATA-
binding protein in neurodegenerative disease. Neuroscience 133:
863â€“872
83. Perez MK, Paulson HL, Pendse SJ, Saionz SJ, Bonini NM, Pittman
RN (1998) Recruitment and the role of nuclear localization in
polyglutamine-mediated aggregation. J Cell Biol 143:1457â€“1470
84. DiFiglia M, Sapp E, Chase KO, Davies SW, Bates GP, Vonsattel JP,
Aronin N (1997) Aggregation of huntingtin in neuronal intranuclear
inclusions and dystrophic neurites in brain. Science 277:1990â€“1993
85. Gunawardena S, Her LS, Brusch RG, Laymon RA, Niesman IR,
Gordesky-Gold B, Sintasath L, Bonini NM, Goldstein LS (2003)
Disruption of axonal transport by loss of huntingtin or expression of
pathogenic polyQ proteins in Drosophila. Neuron 40:25â€“40
86. Seidel K, den Dunnen WF, Schultz C, Paulson H, Frank S, de Vos
RA, Brunt ER, Deller T, Kampenga HH, Rub U (2010) Axonal
inclusions in spinocerebellar ataxia type 3. Acta Neuropathol 120:
449â€“460
87. Nagai Y, Inui T, Popiel HA, Fujikake N, Hasegawa K, Urade Y,
Goto Y, Naiki H, Toda T (2007) A toxic monomeric conformer of
the polyglutamine protein. Nat Struct Mol Biol 14:332â€“340
88. Petrakis S, Schaefer MH, Wanker EE, Andrade-Navarro MA (2013)
Aggregation of polyQ-extended proteins is promoted by interaction
with their natural coiled-coil partners. Bioessays 35:503â€“507
89. Streets AM, Sourigues Y, Kopito RR, Melki R, Quake SR (2013)
Simultaneous measurement of amyloid fibril formation by dynamic
light scattering and fluorescence reveals complex aggregation ki-
netics. PLoS One 8:e54541
90. Teixeira-Castro A, Ailion M, Jalles A, Brignull HR, Vilaca JL, Dias
N, Rodrigues P, Oliveira JF, Neves-Carvalho A, Morimoto RI,
Maciel P (2011) Neuron-specific proteotoxicity of mutant ataxin-3
in C. elegans: rescue by the DAF-16 and HSF-1 pathways. Hum
Mol Genet 20:2996â€“3009
91. Haacke A, Broadley SA, Boteva R, Tzvetkov N, Hartl FU, Breuer P
(2006) Proteolytic cleavage of polyglutamine-expanded ataxin-3 is
critical for aggregation and sequestration of non-expanded ataxin-3.
Hum Mol Genet 15:555â€“568
92. Matsumoto M, Yada M, Hatakeyama S, Ishimoto H, Tanimura T,
Tsuji S, Kakizuka A, Kitagawa M, Nakayama KI (2004) Molecular
clearance of ataxin-3 is regulated by a mammalian E4. EMBO J 23:
659â€“669
93. Berke SJ, Schmied FA, Brunt ER, Ellerby LM, Paulson HL (2004)
Caspase-mediated proteolysis of the polyglutamine disease protein
ataxin-3. J Neurochem 89:908â€“918
94. Koch P, Breuer P, Peitz M, Jungverdorben J, Kesavan J, Poppe D,
Doerr J, Ladewig J, Mertens J, Tuting T et al (2011) Excitation-
induced ataxin-3 aggregation in neurons from patients with
Machadoâ€“Joseph disease. Nature 480:543â€“546
95. Takahashi T, Kikuchi S, Katada S, Nagai Y, Nishizawa M, Onodera
O (2008) Soluble polyglutamine oligomers formed prior to inclu-
sion body formation are cytotoxic. Hum Mol Genet 17:345â€“356
96. Goti D, Katzen SM, Mez J, Kurtis N, Kiluk J, Ben-Haiem L,
Jenkins NA, Copeland NG, Kakizuka A, Sharp AH et al (2004) A
mutant ataxin-3 putative-cleavage fragment in brains of Machadoâ€“
Joseph disease patients and transgenic mice is cytotoxic above a
critical concentration. J Neurosci 24:10266â€“10279

97. Hubener J, Weber JJ, Richter C, Honold L, Weiss A, Murad F,
Breuer P, Wullner U, Bellstedt P, Paquet-Durand F et al (2012)
Calpain-mediated ataxin-3 cleavage in the molecular pathogenesis
of spinocerebellar ataxia type 3 (SCA3). Hum. Mol. Genet
98. Earnshaw WC, Martins LM, Kaufmann SH (1999) Mammalian
caspases: structure, activation, substrates, and functions during ap-
optosis. Annu Rev Biochem 68:383â€“424
99. Kim YJ, Yi Y, Sapp E, Wang Y, Cuiffo B, Kegel KB, Qin ZH,
Aronin N, DiFiglia M (2001) Caspase 3-cleaved N-terminal frag-
ments of wild-type and mutant huntingtin are present in normal and
Huntingtonâ€™s disease brains, associate with membranes, and under-
go calpain-dependent proteolysis. Proc Natl Acad Sci USA 98:
12784â€“12789
100. Sanchez I, Xu CJ, Juo P, Kakizaka A, Blenis J, Yuan J (1999)
Caspase-8 is required for cell death induced by expanded
polyglutamine repeats. Neuron 22:623â€“633
101. Ellerby LM, Andrusiak RL, Wellington CL, Hackam AS, Propp SS,
Wood JD, Sharp AH, Margolis RL, Ross CA, Salvesen GS et al
(1999) Cleavage of atrophin-1 at caspase site aspartic acid 109
modulates cytotoxicity. J Biol Chem 274:8730â€“8736
102. Ellerby LM, Hackam AS, Propp SS, Ellerby HM, Rabizadeh S,
Cashman NR, Trifiro MA, Pinsky L, Wellington CL, Salvesen GS
et al (1999) Kennedyâ€™s disease: caspase cleavage of the androgen
receptor is a crucial event in cytotoxicity. J Neurochem 72:185â€“
195
103. Ona VO, Li M, Vonsattel JP, Andrews LJ, Khan SQ, Chung WM,
Frey AS, Menon AS, Li XJ, Stieg PE et al (1999) Inhibition of
caspase-1 slows disease progression in a mouse model of
Huntingtonâ€™s disease. Nature 399:263â€“267
104. Wellington CL, Ellerby LM, Hackam AS, Margolis RL, Trifiro MA,
Singaraja R, McCutcheon K, Salvesen GS, Propp SS, Bromm M
et al (1998) Caspase cleavage of gene products associated with
triplet expansion disorders generates truncated fragments containing
the polyglutamine tract. J Biol Chem 273:9158â€“9167
105. Strobl S, Fernandez-Catalan C, Braun M, Huber R, Masumoto H,
Nakagawa K, Irie A, Sorimachi H, Bourenkow G, Bartunik H et al
(2000) The crystal structure of calcium-free human m-calpain sug-
gests an electrostatic switch mechanism for activation by calcium.
Proc Natl Acad Sci USA 97:588â€“592
106. Haacke A, Hartl FU, Breuer P (2007) Calpain inhibition is sufficient
to suppress aggregation of polyglutamine-expanded ataxin-3. J Biol
Chem 282:18851â€“18856
107. Simoes AT, Goncalves N, Koeppen A, Deglon N, Kugler S, Duarte
CB, de Pereira AL (2012) Calpastatin-mediated inhibition of
calpains in the mouse brain prevents mutant ataxin 3 proteolysis,
nuclear localization and aggregation, relieving Machadoâ€“Joseph
disease. Brain 135:2428â€“2439
108. Jung J, Xu K, Lessing D, Bonini NM (2009) Preventing Ataxin-3
protein cleavage mitigates degeneration in a Drosophila model of
SCA3. Hum Mol Genet 18:4843â€“4852
109. Klement IA, Skinner PJ, Kaytor MD, Yi H, Hersch SM, Clark HB,
Zoghbi HY, Orr HT (1998) Ataxin-1 nuclear localization and ag-
gregation: role in polyglutamine-induced disease in SCA1 transgen-
ic mice. Cell 95:41â€“53
110. Colomer Gould VF, Goti D, Pearce D, Gonzalez GA, Gao H, de
Bermudez LM, Jenkins NA, Copeland NG, Ross CA, Brown DR
(2007) A mutant ataxin-3 fragment results from processing at a site
N-terminal to amino acid 190 in brain of Machadoâ€“Joseph disease-
like transgenic mice. Neurobiol Dis 27:362â€“369
111. Masino L, Nicastro G, Menon RP, Dal PF, Calder L, Pastore A
(2004) Characterization of the structure and the amyloidogenic
properties of the Josephin domain of the polyglutamine-containing
protein ataxin-3. J Mol Biol 344:1021â€“1035
112. Masino L, Nicastro G, De SA, Calder L, Molloy J, Pastore A (2011)
The Josephin domain determines the morphological and mechanical
properties of ataxin-3 fibrils. Biophys J 100:2033â€“2042

13. Hubener J, Vauti F, Funke C, Wolburg H, Ye Y, Schmidt T, Wolburg-Buchholz K, Schmitt I, Gardyan A, Driessen S et al (2011) N-terminal ataxin-3 causes neurological symptoms with inclusions, endoplasmic reticulum stress and ribosomal dislocation. Brain 134:1925â€“1942

14. Chai Y, Koppenhafer SL, Shoesmith SJ, Perez MK, Paulson HL (1999) Evidence for proteasome involvement in polyglutamine disease: localization to nuclear inclusions in SCA3/MJD and suppression of polyglutamine aggregation in vitro. Hum Mol Genet 8: 673â€“682

15. Hirabayashi M, Inoue K, Tanaka K, Nakadate K, Ohsawa Y, Kamei Y, Popiel AH, Sinohara A, Iwamatsu A, Kimura Y et al (2001) VCP/p97 in abnormal protein aggregates, cytoplasmic vacuoles, and cell death, phenotypes relevant to neurodegeneration. Cell Death Differ 8:977â€“984

16. Doss-Pepe EW, Stenroos ES, Johnson WG, Madura K (2003) Ataxin-3 interactions with rad23 and valosin-containing protein and its associations with ubiquitin chains and the proteasome are consistent with a role in ubiquitin-mediated proteolysis. Mol Cell Biol 23:6469â€“6483

17. Nascimento-Ferreira I, Santos-Ferreira T, Sousa-Ferreira L, Auregan G, Onofre I, Alves S, Dufour N, Colomer Gould VF, Koeppen A, Deglon N, de Pereira AL (2011) Overexpression of the autophagic beclin-1 protein clears mutant ataxin-3 and alleviates Machadoâ€“Joseph disease. Brain 134:1400â€“1415

18. Wong E, Cuervo AM (2010) Autophagy gone awry in neurodegenerative diseases. Nat Neurosci 13:805â€“811

19. Ravikumar B, Vacher C, Berger Z, Davies JE, Luo S, Oroz LG, Scaravilli F, Easton DF, Duden R, Oâ€™Kane CJ, Rubinsztein DC (2004) Inhibition of mTOR induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of Huntington disease. Nat Genet 36:585â€“595

20. Ajayi A, Yu X, Lindberg S, Langel U, Strom AL (2012) Expanded ataxin-7 cause toxicity by inducing ROS production from NADPH oxidase complexes in a stable inducible Spinocerebellar ataxia type 7 (SCA7) model. BMC Neurosci 13:86

21. Goswami A, Dikshit P, Mishra A, Mulherkar S, Nukina N, Jana NR (2006) Oxidative stress promotes mutant huntingtin aggregation and mutant huntingtin-dependent cell death by mimicking proteasomal malfunction. Biochem Biophys Res Commun 342: 184â€“190

22. Kim SJ, Kim TS, Hong S, Rhim H, Kim IY, Kang S (2003) Oxidative stimuli affect polyglutamine aggregation and cell death in human mutant ataxin-1-expressing cells. Neurosci Lett 348:21â€“24

23. Miyata R, Hayashi M, Tanuma N, Shioda K, Fukatsu R, Mizutani S (2008) Oxidative stress in neurodegeneration in dentatorubral-pallidoluysian atrophy. J Neurol Sci 264:133â€“139

24. Yu YC, Kuo CL, Cheng WL, Liu CS, Hsieh M (2009) Decreased antioxidant enzyme activity and increased mitochondrial DNA damage in cellular models of Machadoâ€“Joseph disease. J Neurosci Res 87:1884â€“1891

25. Kazachkova N, Raposo M, Montiel R, Cymbron T, Bettencourt C, Silva-Fernandes A, Silva S, Maciel P, Lima M (2013) Patterns of mitochondrial DNA damage in blood and brain tissues of a transgenic mouse model of Machadoâ€“Joseph disease. Neurodegener Dis 11:206â€“214

26. Laco MN, Oliveira CR, Paulson HL, Rego AC (2012) Compromised mitochondrial complex II in models of Machadoâ€“Joseph disease. Biochim Biophys Acta 1822:139â€“149

27. Emerit J, Edeas M, Bricaire F (2004) Neurodegenerative diseases and oxidative stress. Biomed Pharmacother 58:39â€“46

28. Riley BE, Orr HT (2006) Polyglutamine neurodegenerative diseases and regulation of transcription: assembling the puzzle. Genes Dev 20:2183â€“2192

29. Evert BO, Vogt IR, Kindermann C, Ozimek L, de Vos RA, Brunt ER, Schmitt I, Klockgether T, Wullner U (2001) Inflammatory

genes are upregulated in expanded ataxin-3-expressing cell lines and spinocerebellar ataxia type 3 brains. J Neurosci 21:5389â€“5396

130. Evert BO, Vogt IR, Vieira-Saecker AM, Ozimek L, de Vos RA, Brunt ER, Klockgether T, Wullner U (2003) Gene expression profiling in ataxin-3 expressing cell lines reveals distinct effects of normal and mutant ataxin-3. J Neuropathol Exp Neurol 62: 1006â€“1018

131. Chou AH, Yeh TH, Ouyang P, Chen YL, Chen SY, Wang HL (2008) Polyglutamine-expanded ataxin-3 causes cerebellar dysfunction of SCA3 transgenic mice by inducing transcriptional dysregulation. Neurobiol Dis 31:89â€“101

132. Li LB, Yu Z, Teng X, Bonini NM (2008) RNA toxicity is a component of ataxin-3 degeneration in Drosophila. Nature 453: 1107â€“1111

133. Wang LC, Chen KY, Pan H, Wu CC, Chen PH, Liao YT, Li C, Huang ML, Hsiao KM (2011) Muscleblind participates in RNA toxicity of expanded CAG and CUG repeats in Caenorhabditis elegans. Cell Mol Life Sci 68:1255â€“1267

134. McLeod CJ, Oâ€™Keefe LV, Richards RI (2005) The pathogenic agent in Drosophila models of â€˜polyglutamineâ€™ diseases. Hum Mol Genet 14:1041â€“1048

135. Miller JW, Urbinati CR, Teng-Umnuay P, Stenberg MG, Byrne BJ, Thornton CA, Swanson MS (2000) Recruitment of human muscleblind proteins to (CUG)(n) expansions associated with myotonic dystrophy. EMBO J 19:4439â€“4448

136. Fardaei M, Larkin K, Brook JD, Hamshire MG (2001) In vivo colocalisation of MBNL protein with DMPK expanded-repeat transcripts. Nucleic Acids Res 29:2766â€“2771

137. Jiang H, Mankodi A, Swanson MS, Moxley RT, Thornton CA (2004) Myotonic dystrophy type 1 is associated with nuclear foci of mutant RNA, sequestration of muscleblind proteins and deregulated alternative splicing in neurons. Hum Mol Genet 13: 3079â€“3088

138. Mykowska A, Sobczak K, Wojciechowska M, Kozlowski P, Krzyzosiak WJ (2011) CAG repeats mimic CUG repeats in the misregulation of alternative splicing. Nucleic Acids Res 39: 8938â€“8951

139. Dansithong W, Paul S, Comai L, Reddy S (2005) MBNL1 is the primary determinant of focus formation and aberrant insulin receptor splicing in DM1. J Biol Chem 280:5773â€“5780

140. He Y, Vogelstein B, Velculescu VE, Papadopoulos N, Kinzler KW (2008) The antisense transcriptomes of human cells. Science 322: 1855â€“1857

141. Katayama S, Tomaru Y, Kasukawa T, Waki K, Nakanishi M, Nakamura M, Nishida H, Yap CC, Suzuki M, Kawai J et al (2005) Antisense transcription in the mammalian transcriptome. Science 309:1564â€“1566

142. Cho DH, Thienes CP, Mahoney SE, Analau E, Filipova GN, Tapscott SJ (2005) Antisense transcription and heterochromatin at the DM1 CTG repeats are constrained by CTCF. Mol Cell 20:483â€“489

143. Chung DW, Rudnicki DD, Yu L, Margolis RL (2011) A natural antisense transcript at the Huntingtonâ€™s disease repeat locus regulates HTT expression. Hum Mol Genet 20:3467â€“3477

144. Moseley ML, Zu T, Ikeda Y, Gao W, Mosemiller AK, Daughters RS, Chen G, Weatherspoon MR, Clark HB, Ebner TJ et al (2006) Bidirectional expression of CUG and CAG expansion transcripts and intranuclear polyglutamine inclusions in spinocerebellar ataxia type 8. Nat Genet 38:758â€“769

145. Wilburn B, Rudnicki DD, Zhao J, Weitz TM, Cheng Y, Gu X, Greiner E, Park CS, Wang N, Sopher BL et al (2011) An antisense CAG repeat transcript at JPH3 locus mediates expanded polyglutamine protein toxicity in Huntingtonâ€™s disease-like 2 mice. Neuron 70:427â€“440

146. Sopher BL, Ladd PD, Pineda VV, Libby RT, Sunkin SM, Hurley JB, Thienes CP, Gaasterland T, Filipova GN, La Spada AR (2011) CTCF regulates ataxin-7 expression through promotion of a

convergently transcribed, antisense noncoding RNA. Neuron 70: 1071â€“1084

147. Banez-Coronel M, Porta S, Kagerbauer B, Mateu-Huertas E, Pantano L, Ferrer I, Guzman M, Estivill X, Marti E (2012) A pathogenic mechanism in Huntingtonâ€™s disease involves small CAG-repeated RNAs with neurotoxic activity. PLoS Genet 8: e1002481

148. Rau F, Freyermuth F, Fugier C, Villemin JP, Fischer MC, Jost B, Dembele D, Gourdon G, Nicole A, Duboc D et al (2011) Misregulation of miR-1 processing is associated with heart defects in myotonic dystrophy. Nat Struct Mol Biol 18:840â€“845

149. Sinha M, Mukhopadhyay S, Bhattacharyya NP (2012) Mechanism(s) of alteration of micro RNA expressions in Huntingtonâ€™s disease and their possible contributions to the observed cellular and molecular dysfunctions in the disease. Neuromol Med 14:221â€“243

150. Bilen J, Liu N, Burnett BG, Pittman RN, Bonini NM (2006) MicroRNA pathways modulate polyglutamine-induced neurodegeneration. Mol Cell 24:157â€“163

151. Davies JE, Rubinsztein DC (2006) Polyalanine and polyserine frameshift products in Huntingtonâ€™s disease. J Med Genet 43:893â€“896

152. Gaspar C, Jannatipour M, Dion P, Laganiere J, Sequeiros J, Brais B, Rouleau GA (2000) CAG tract of MJD-1 may be prone to frameshifts causing polyalanine accumulation. Hum Mol Genet 9:1957â€“1966

153. Stochmanski SJ, Therrien M, Laganiere J, Rochefort D, Laurent S, Karemera L, Gaudet R, Vyboh K, Van Meyel DJ, Di CG et al (2012) Expanded ATXN3 frameshifting events are toxic in Drosophila and mammalian neuron models. Hum Mol Genet 21:2211â€“2218

154. Zu T, Gibbens B, Doty NS, Gomes-Pereira M, Huguet A, Stone MD, Margolis J, Peterson M, Markowski TW, Ingram MA et al (2011) Non-ATG-initiated translation directed by microsatellite expansions. Proc Natl Acad Sci USA 108:260â€“265

155. Pearson CE (2011) Repeat associated non-ATG translation initiation: one DNA, two transcripts, seven reading frames, potentially nine toxic entities! PLoS Genet 7:e1002018

156. Paulson H (2012) Machadoâ€“Joseph disease/spinocerebellar ataxia type 3. Handb Clin Neurol 103:437â€“449

157. Buhmann C, Bussopulos A, Oechsner M (2003) Dopaminergic response in Parkinsonian phenotype of Machadoâ€“Joseph disease. Mov Disord 18:219â€“221

158. Takei A, Fukazawa T, Hamada T, Sohma H, Yabe I, Sasaki H, Tashiro K (2004) Effects of tandospirone on â€œ5-HT1A receptor-associated symptomsâ€ in patients with Machadoâ€“Joseph disease: an open-label study. Clin Neuropharmacol 27:9â€“13

159. Bettencourt C, Lima M (2011) Machadoâ€“Joseph disease: from first descriptions to new perspectives. Orphanet J Rare Dis 6:35

160. Ogawa M (2004) Pharmacological treatments of cerebellar ataxia. Cerebellum 3:107â€“111

161. Nobrega C, Nascimento-Ferreira I, Onofre I, Albuquerque D, Hirai H, Deglon N, de Almeida LP (2013) Silencing mutant ataxin-3 rescues motor deficits and neuropathology in Machadoâ€“Joseph disease transgenic mice. PLoS One 8:e52396

162. Menzies FM, Huebener J, Renna M, Bonin M, Riess O, Rubinsztein DC (2010) Autophagy induction reduces mutant ataxin-3 levels and toxicity in a mouse model of spinocerebellar ataxia type 3. Brain 133:93â€“104

163. Jia DD, Zhang L, Chen Z, Wang CR, Huang FZ, Duan RH, Xia K, Tang BS, Jiang H (2013) Lithium Chloride Alleviates Neurodegeneration Partly by Inhibiting Activity of GSK3beta in a SCA3 Drosophila Model. Cerebellum..

164. Saute JA, Castilhos R, Monte T, Schuh A, Donis K, Dâ€™Ãvila R et al (2013) Safety and effectiveness of lithium carbonate for the treatment of Machadoâ€“Joseph disease: A phase 2/3 randomised, double-blind, placebo-controlled trial [abstract]. Movement Disord 28(Suppl 1):713

165. Wang HL, Hu SH, Chou AH, Wang SS, Weng YH, Yeh TH (2013) H1152 promotes the degradation of polyglutamine-expanded ataxin-3 or ataxin-7 independently of its ROCK-inhibiting effect and ameliorates mutant ataxin-3-induced neurodegeneration in the SCA3 transgenic mouse. Neuropharmacology 70:1â€“11

166. Johri A, Beal MF (2012) Antioxidants in Huntingtonâ€™s disease. Biochim Biophys Acta 1822:664â€“674

167. Hersch SM, Gevorkian S, Marder K, Moskowitz C, Feigin A, Cox M, Como P, Zimmerman C, Lin M, Zhang L et al (2006) Creatine in Huntington disease is safe, tolerable, bioavailable in brain and reduces serum 8OH2â€™dG. Neurology 66:250â€“252

168. Hyson HC, Kieburtz K, Shoulson I, McDermott M, Ravina B, de Blieck EA, Cudkowicz ME, Ferrante RJ, Como P, Frank S et al (2010) Safety and tolerability of high-dosage coenzyme Q10 in Huntingtonâ€™s disease and healthy subjects. Mov Disord 25: 1924â€“1928

169. Chen X, Tang TS, Tu H, Nelson O, Pook M, Hammer R, Nukina N, Bezprozvanny I (2008) Deranged calcium signaling and neurodegeneration in spinocerebellar ataxia type 3. J Neurosci 28:12713â€“12724

170. Makarewicz D, Zieminska E, Lazarewicz JW (2003) Dantrolene inhibits NMDA-induced 45Ca uptake in cultured cerebellar granule neurons. Neurochem Int 43:273â€“278

171. Lin CM, Neeru S, Doufas AG, Liem E, Muneer SY, Wadhwa A, Lenhardt R, Bjorksten A, Taguchi A, Kabon B et al (2004) Dantrolene reduces the threshold and gain for shivering. Anesth Analg 98:1318â€“1324, table

172. Muehlschlegel S, Rordorf G, Sims J (2011) Effects of a single dose of dantrolene in patients with cerebral vasospasm after subarachnoid hemorrhage: a prospective pilot study. Stroke 42: 1301â€“1306

173. Cunha RA, Agostinho PM (2010) Chronic caffeine consumption prevents memory disturbance in different animal models of memory decline. J Alzheimers Dis 20(Suppl 1):S95â€“S116

174. Goncalves N, Simoes AT, Cunha RA, de Almeida LP (2013) Caffeine and adenosine A(2A) receptor inactivation decrease striatal neuropathology in a lentiviral-based model of Machadoâ€“Joseph disease. Ann Neurol 73:655â€“666

175. Popoli P, Blum D, Martire A, Ledent C, Ceruti S, Abbracchio MP (2007) Functions, dysfunctions and possible therapeutic relevance of adenosine A2A receptors in Huntingtonâ€™s disease. Prog Neurobiol 81:331â€“348

176. Chou AH, Chen SY, Yeh TH, Weng YH, Wang HL (2011) HDAC inhibitor sodium butyrate reverses transcriptional downregulation and ameliorates ataxic symptoms in a transgenic mouse model of SCA3. Neurobiol Dis 41:481â€“488

177. Minamiyama M, Katsuno M, Adachi H, Waza M, Sang C, Kobayashi Y, Tanaka F, Doyu M, Inukai A, Sobue G (2004) Sodium butyrate ameliorates phenotypic expression in a transgenic mouse model of spinal and bulbar muscular atrophy. Hum Mol Genet 13:1183â€“1192

178. Collins AF, Pearson HA, Giardina P, McDonagh KT, Brusilow SW, Dover GJ (1995) Oral sodium phenylbutyrate therapy in homozygous beta thalassemia: a clinical trial. Blood 85:43â€“49

179. Hogarth P, Lovrecic L, Krainc D (2007) Sodium phenylbutyrate in Huntingtonâ€™s disease: a dose-finding study. Mov Disord 22: 1962â€“1964

180. Valentino KL, Gutierrez M, Sanchez R, Winship MJ, Shapiro DA (2003) First clinical trial of a novel caspase inhibitor: anti-apoptotic caspase inhibitor, IDN-6556, improves liver enzymes. Int J Clin Pharmacol Ther 41:441â€“449

181. Baskin-Bey ES, Washburn K, Feng S, Oltersdorf T, Shapiro D, Huyghe M, Burgart L, Garrity-Park M, van Vilsteren FG, Oliver LK et al (2007) Clinical trial of the pan-caspase inhibitor, IDN-6556, in Human Liver Preservation Injury. Am J Transplant 7: 218â€“225

182. Li Z, Sheng M (2012) Caspases in synaptic plasticity. Mol Brain 5: 15
183. Troy CM, Salvesen GS (2002) Caspases on the brain. J Neurosci Res 69:145â€“150
184. Carragher NO (2006) Calpain inhibition: a therapeutic strategy targeting multiple disease states. Curr Pharm Des 12:615â€“638
185. Donkor IO (2011) Calpain inhibitors: a survey of compounds reported in the patent and scientific literature. Progr Neurobiol 81:331â€“348
186. Adachi H, Katsuno M, Minamiyama M, Sang C, Pagoulatos G, Angelidis C, Kusakabe M, Yoshiki A, Kobayashi Y, Doyu M, Sobue G (2003) Heat shock protein 70 chaperone overexpression ameliorates phenotypes of the spinal and bulbar muscular atrophy transgenic mouse model by reducing nuclear-localized mutant androgen receptor protein. J Neurosci 23:2203â€“2211
187. Cummings CJ, Sun Y, Opal P, Antalffy B, Mestril R, Orr HT, Dillmann WH, Zoghbi HY (2001) Over-expression of inducible HSP70 chaperone suppresses neuropathology and improves motor function in SCA1 mice. Hum Mol Genet 10:1511â€“1518
188. Chan HY, Warrick JM, Gray-Board PHL, Bonini NM (2000) Mechanisms of chaperone suppression of polyglutamine disease: selectivity, synergy and modulation of protein solubility in Drosophila. Hum Mol Genet 9:2811â€“2820
189. Muchowski PJ, Schaffar G, Sittler A, Wanker EE, Hayer-Hartl MK, Hartl FU (2000) Hsp70 and hsp40 chaperones can inhibit self-assembly of polyglutamine proteins into amyloid-like fibrils. Proc Natl Acad Sci USA 97:7841â€“7846
190. Bailey CK, Andriola IF, Kampina HH, Merry DE (2002) Molecular chaperones enhance the degradation of expanded polyglutamine repeat androgen receptor in a cellular model of spinal and bulbar muscular atrophy. Hum Mol Genet 11:515â€“523
191. Verhoeft LG, Lindsten K, Masucci MG, Dantuma NP (2002) Aggregate formation inhibits proteasomal degradation of polyglutamine proteins. Hum Mol Genet 11:2689â€“2700
192. Yoshida H, Yoshizawa T, Shibasaki F, Shoji S, Kanazawa I (2002) Chemical chaperones reduce aggregate formation and cell death caused by the truncated Machado-Joseph disease gene product with an expanded polyglutamine stretch. Neurobiol Dis 10:88â€“99
193. Wood NI, Pallier PN, Wanderer J, Morton AJ (2007) Systemic administration of Congo red does not improve motor or cognitive function in R6/2 mice. Neurobiol Dis 25:342â€“353
194. Smith DL, Woodman B, Mahal A, Sathasivam K, Ghazi-Noori S, Lowden PA, Bates GP, Hockly E (2003) Minocycline and doxycycline are not beneficial in a model of Huntingtonâ€™s disease. Ann Neurol 54:186â€“196
195. Schilling G, Savonenko AV, Coonfield ML, Morton JL, Vorovich E, Gale A, Neslon C, Chan N, Eaton M, Fromholt D et al (2004) Environmental, pharmacological, and genetic modulation of the HD phenotype in transgenic mice. Exp Neurol 187:137â€“149
196. Wang CE, Zhou H, McGuire JR, Cerullo V, Lee B, Li SH, Li XJ (2008) Suppression of neuropil aggregates and neurological symptoms by an intracellular antibody implicates the cytoplasmic toxicity of mutant huntingtin. J Cell Biol 181:803â€“816
197. Snyder-Keller A, McLear JA, Hathorn T, Messer A (2010) Early or late-stage anti-N-terminal Huntingtin intrabody gene therapy reduces pathological features in B6.HDR6/l mice. J Neuropathol Exp Neurol 69:1078â€“1085
198. Kvam E, Nannenga BL, Wang MS, Jia Z, Sierks MR, Messer A (2009) Conformational targeting of fibrillar polyglutamine proteins in live cells escalates aggregation and cytotoxicity. PLoS One 4:e5727
199. Rodriguez-Lebron E, Paulson HL (2006) Allele-specific RNA interference for neurological disease. Gene Ther 13:576â€“581
200. Miller VM, Xia H, Marrs GL, Gouvion CM, Lee G, Davidson BL, Paulson HL (2003) Allele-specific silencing of dominant disease genes. Proc Natl Acad Sci USA 100:7195â€“7200
201. Alves S, Nascimento-Ferreira I, Auregan G, Hassig R, Dufour N, Brouillet E, Pedroso de Lima MC, Hantraye P, de Pereira AL, Deglon N (2008) Allele-specific RNA silencing of mutant ataxin-3 mediates neuroprotection in a rat model of Machado-Joseph disease. PLoS One 3:e3341
202. Gaspar C, Lopes-Cendes I, Hayes S, Goto J, Arvidsson K, Dias A, Silveira I, Maciel P, Coutinho P, Lima M et al (2001) Ancestral origins of the Machado-Joseph disease mutation: a worldwide haplotype study. Am J Hum Genet 68:523â€“528
203. Tabernero J, Shapiro GI, Lorusso PM, Cervantes A, Schwartz GK, Weiss GJ, Paz-Ares L, Cho DC, Infante JR, Alsina M et al (2013) First-in-humans trial of an RNA interference therapeutic targeting VEGF and KSP in cancer patients with liver involvement. Cancer Discov 3:406â€“417
204. Evers MM, Peppers BA, van Deutekom JC, Mulders SA, den Dunnen JT, Aartsma-Rus A, van Ommen GJ, van Roon-Mom WM (2011) Targeting several CAG expansion diseases by a single antisense oligonucleotide. PLoS One 6:e24308
205. Hu J, Gagnon KT, Liu J, Watts JK, Syeda-Nawaz J, Bennett CF, Swayze EE, Randolph J, Chattopadhyaya J, Corey DR (2011) Allele-selective inhibition of ataxin-3 (ATX3) expression by antisense oligomers and duplex RNAs. Biol Chem 392:315â€“325
206. Liu J, Pendergraft H, Narayanannair KJ, Lackey JG, Kuchimanchi S, Rajeev KG, Manoharan M, Hu J, Corey DR (2013) RNA duplexes with abasic substitutions are potent and allele-selective inhibitors of huntingtin and ataxin-3 expression. Nucleic Acids Res..
207. Liu J, Yu D, Aiba Y, Pendergraft H, Swayze EE, Lima WF, Hu J, Prakash TP, Corey DR (2013) ss-siRNAs allele selectively inhibit ataxin-3 expression: multiple mechanisms for an alternative gene silencing strategy. Nucleic Acids Res..
208. Spitali P, Aartsma-Rus A (2012) Splice modulating therapies for human disease. Cell 148:1085â€“1088
209. Zalachoras I, Evers MM, van Roon-Mom WM, Aartsma-Rus AM, Meijer OC (2011) Antisense-mediated RNA targeting: versatile and expedient genetic manipulation in the brain. Front Mol Neurosci 4:10
210. Kordasiewicz HB, Stanek LM, Wancewicz EV, Mazur C, McAlonis MM, Pytel KA, Artates JW, Weiss A, Cheng SH, Shihabuddin LS et al (2012) Sustained therapeutic reversal of huntingtonâ€™s disease by transient repression of huntingtin synthesis. Neuron 74:1031â€“1044
211. Miller TM, Pestronk A, David W, Rothstein J, Simpson E, Appel SH, Andres PL, Mahoney K, Allred P, Alexander K et al (2013) An antisense oligonucleotide against SOD1 delivered intrathecally for patients with SOD1 familial amyotrophic lateral sclerosis: a phase 1, randomised, first-in-man study. Lancet Neurol 12:435â€“442
212. Cirak S, Arechavala-Gomeza V, Guglieri M, Feng L, Torelli S, Anthony K, Abbs S, Garralda ME, Bourke J, Wells DJ et al (2011) Exon skipping and dystrophin restoration in patients with Duchenne muscular dystrophy after systemic phosphorodiamidate morpholino oligomer treatment: an open-label, phase 2, dose-escalation study. Lancet 378:595â€“605
