
Rv0132c of *Mycobacterium tuberculosis* Encodes a Coenzyme F₄₂₀-Dependent Hydroxymycolic Acid Dehydrogenase

**Endang Purwantini**¹², **Biswarup Mukhopadhyay**¹,²,³,⁴*

1 Department of Biochemistry, Virginia Tech, Blacksburg, Virginia, United States of America, 2 Virginia Bioinformatics Institute, Virginia Tech, Blacksburg, Virginia, United States of America, 3 Departments of Biological Sciences, Virginia Tech, Blacksburg, Virginia, United States of America, 4 Virginia Tech Carilion School of Medicine, Virginia Tech, Blacksburg, Virginia, United States of America

---

### Abstract

The ability of *Mycobacterium tuberculosis* to manipulate and evade the human immune system is in part due to its extraordinarily complex cell wall. One of the key components of this cell wall is a family of lipids called mycolic acids. Oxygenation of mycolic acids generating methoxy- and ketomycolic acids enhances the pathogenic attributes of *M. tuberculosis*. Thus, the respective enzymes are of interest in the research on mycobacteria. The generation of methoxy- and ketomycolic acids proceeds through intermediary formation of hydroxymycolic acids. While the methyltransferase that generates methoxymycolic acids from hydroxymycolic acids is known, hydroxymycolic acid dehydrogenase that oxidizes hydroxymycolic acids to ketomycolic acids has been elusive. We found that hydroxymycolic acid dehydrogenase is encoded by the *rv0132c* gene and the enzyme utilizes F₄₂₀, a deazaflavin cofactor, as electron carrier, and accordingly we called it F₄₂₀-dependent hydroxymycolic acid dehydrogenase. This is the first report on the involvement of F₄₂₀ in the synthesis of a mycobacterial cell envelope. Also, F₄₂₀-dependent hydroxymycolic acid dehydrogenase was inhibited by PA-824, and therefore, it is a previously unknown target for this new tuberculosis drug.

---

### Citation: Purwantini E, Mukhopadhyay B (2013) Rv0132c of Mycobacterium tuberculosis Encodes a Coenzyme F₄₂₀-Dependent Hydroxymycolic Acid Dehydrogenase. PLoS ONE 8(12): e81985. doi:10.1371/journal.pone.0081985

### Editor: Daniel S. Sem, Concordia University Wisconsin, United States of America

### Received June 3, 2013; Accepted October 19, 2013; Published December 11, 2013

### Copyright: © 2013 Purwantini, Mukhopadhyay. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.

### Funding: This research was supported by Virginia Bioinformatics Institute, Virginia Tech and grant 1R21AI100039 from the National Institutes of Health to BM. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

### Competing Interests: The authors have declared that no competing interests exist.

*E-mail: biswarup@vt.edu*

---

### Introduction

The cell wall of *Mycobacterium tuberculosis* (*Mtb*), the causative agent of tuberculosis [1,2], has an extraordinarily complex and very hydrophobic structure. Consequently, it offers an exceptionally low permeability and makes the *Mtb* cells poorly accessible to drugs and less vulnerable to attack by the host immune system [3]. For this reason, cell wall synthesis enzymes of *Mtb* have been targeted for TB drug development [4]. Mycolic acids (MAs) are some of the key lipid components of the mycobacterial cell wall. These “high-molecular weight beta-hydroxy fatty acids with a long alpha-alkyl side chain” [5] (Fig. S1) are constituents of mycolyl-arabinogalactan-peptidoglycan complex and trehalose mono-/di-mycolates (TMM and TDM) [6–8]. By helping to build a strong cell wall and being immunogenic [7,9,10], these complexes contribute to the development of TB [3,10–15]. *Mtb* generates three structural types of MAs which are called α-, methoxy- and keto-mycolic acids (α-, M- and K-MAs) and under *in vitro* growth conditions it does not contain epoxymycolic acids (E-MAs) that are found in *Mycobacterium smegmatis* [16]; the respective chemical structures are shown in the Supporting Material (Fig. S1). The keto- and methoxy-derivatives enhance the pathogenic nature of *Mtb* [17,18], and the bacterium uses these compounds to modulate the host immune response [9,19–21]. A recent report shows that K-MAs allow *Mtb* to form pellicle structures, which in turn make this pathogen drug-resistant [22]. Thus, the enzymes that introduce keto- and methoxy-groups in mycolic acids are of research interest [3,17,23–26]. These oxygenated lipids are generated through common immediate precursors, hydroxymycolic acids (H-MAs) (Fig. 1) [3,24,27]. Whereas it is known that in *Mtb* the conversion of H-MAs to M-MAs is catalyzed by an adenosylmethionine-dependent methyltransferase (Mma3 or CmaB) encoded by the ORF *Rv0643c* [7,24,26] (Fig. 1), the enzyme that oxidizes H-MAs to K-MAs remains unknown. We call this unknown enzyme hydroxymycolic acid dehydrogenase (HMAD). In this report we describe the gene that encodes HMAD in *Mtb* and demonstrate that the enzyme utilizes coenzyme F₄₂₀, a deazaflavin derivative, as electron carrier (Fig. 1). Thus, we named the enzyme fHMAD for F₄₂₀-dependent Hydroxy Mycolic Acid Dehydrogenase. Also, we show that fHMAD is inhibited by PA-824, a nitroimidazopyran and a new TB drug that is currently on clinical trial [28].

---

### Results and Discussion

#### Identification of *Rv0132c* as Coenzyme F₄₂₀-Dependent Hydroxymycolic Acids Dehydrogenase (fHMAD) in *M. tuberculosis*

This work began with an analysis of the available data, and the resulting hypothesis was tested via genetic analysis of an *Mtb* gene.

F420-Dependent Hydroxymycolic Acid Dehydrogenase

$\begin{aligned}
& \text{R-CH-C-OH} \\
& \quad \mid \quad \mid \quad \mid \\
& \quad \mathrm{OH} \quad (\mathrm{CH}_{2})_{m} \quad (-\mathrm{OCH}_{3}) \\
& \quad \mid \quad \mid \\
& \quad \mathrm{CH}_{3} \\
& \quad X
\end{aligned}$

Mycolic acid  
(Mycolic acid  
methyl ester)

$\mathrm{CH}_{3}-\left(\mathrm{CH}_{2}\right)_{x}-\mathrm{CH}=\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{y}-\mathrm{CH}=\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{z}-X$

Common precursor for all R groups

$\downarrow$

Methyl transferase  
(CmaA or Mma4 or Hma)  
&  
unknown hydrase

$\downarrow$

$\begin{aligned}
& \mathrm{CH}_{3}-\left(\mathrm{CH}_{2}\right)_{x}-\mathrm{C}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{y}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{z}-X \\
& \quad \mid \\
& \quad \mathrm{O}
\end{aligned}$

Epoxy-mycolic acid

$\begin{aligned}
& \mathrm{CH}_{3}-\left(\mathrm{CH}_{2}\right)_{x}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{y}-\mathrm{CH}=\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{z}-X \\
& \quad \mid \\
& \quad \mathrm{CH}_{3}
\end{aligned}$

Hydroxy-mycolic acid

$\downarrow$

Rv0132c or fHMAD

$\begin{aligned}
& \mathrm{CH}_{3}-\left(\mathrm{CH}_{2}\right)_{x}-\mathrm{CH}-\mathrm{CH}_{2}-\left(\mathrm{CH}_{2}\right)_{y-1}-\mathrm{CH}=\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{z}-X \\
& \quad \mid \quad \mid \\
& \quad \mathrm{CH}_{3} \quad \mathrm{O}
\end{aligned}$

Keto-mycolic acid

$\downarrow$

$\begin{aligned}
& \mathrm{CH}_{3}-\left(\mathrm{CH}_{2}\right)_{x}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{y}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{z}-X \\
& \quad \mid \quad \mid \\
& \quad \mathrm{H}_{3} \mathrm{C} \quad \mathrm{OCH}_{3}
\end{aligned}$

Cyclopropanated methoxy-mycolic acid

$\downarrow$

$\begin{aligned}
& \mathrm{CH}_{3}-\left(\mathrm{CH}_{2}\right)_{x}-\mathrm{CH}-\mathrm{C}-\left(\mathrm{CH}_{2}\right)_{y}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{z}-X \\
& \quad \mid \quad \mid \\
& \quad \mathrm{H}_{3} \mathrm{C} \quad \mathrm{O}
\end{aligned}$

Cyclopropanated Keto-mycolic acid

Figure 1. Proposed pathways for the synthesis of hydroxy-, keto-, methoxy- and epoxymycolic acids in mycobacteria [7,24]. A common intermediate for various R groups is used as the starting point. Where MmaA2 and CmaA2 are involved in the formation of cis cyclopropane group, CmaA2 and an yet to identified enzyme (indicated by?) catalyze trans-cyclopropanation [48,67]. The details of the individual R groups are shown in Fig. S1. * indicates that it is not known whether the cyclopropanation step follows or precedes oxygenation. All protons (except for the isolated groups) that have been target for NMR data analysis have been shown in red. The OH group shown in italics and underlined in the box at the left corner of the figure was converted to a methoxy group during saponification of mycolic acids; the process generated mycolic acids methyl esters (MAMEs).

doi:10.1371/journal.pone.0081985.g001

in *Mycobacterium smegmatis*. The rationale for the selection of $M$. *smegmatis* as the experimental host has been elaborated below.

**Selection of Mycobacterium smegmatis as a facile screening host in a search for the HMAD encoding gene of Mycobacterium tuberculosis.** As mentioned above, $M t b$ produces $\alpha-$, K- and M-MAs, and it does not contain epoxymycolic acids (E-MAs) under *in vitro* growth conditions [16]. In this regard *Mycobacterium bovis* strain BCG (BCG) is similar to $M t b$ except some of the strains of the former do not produce M-MAs as the $c m a B$ or $m m a 3$ gene of the organism is non-functional due to a point mutation [16,25,26]. $M$. *smegmatis* produces $\alpha-$, $\alpha^{\prime}-$, and E-MAs but is devoid of K- and M-MAs [16,29]. The structures of these species are shown in Fig. S1; in $M$. *smegmatis* five variations of the $\alpha$ group, $\alpha 1-$, $\alpha 2-$, $\alpha 3-$, $\alpha 4-$ and $\alpha 5-$, are found [29]. The investigation described in this report concerns only the longer aliphatic chains (the R groups) of the MAs (Fig. S1 and Fig. 1).

Fig. 1 shows the proposed pathways for synthesis of H-, K-, M- and E-MAs in wild-type and recombinant $M t b$, BCG and $M$. *smegmatis* [17,23,24,27]; a common precursor for the aliphatic chains of various MAs serves as the starting point in this scheme. The deletion of the $h m a$ gene (also called $m m a 4$ and $c m a A$) in $M t b$ abolishes the production of K- and M-MAs and causes the production of E-MAs and an intermediate that is similar to $\alpha$-MAs [17,23,27]. Heterologous expression of the $M t b$ or

BCG $h m a$ gene (*orf rv0642* or *mb0661*, respectively) in $M$. *smegmatis* allows the synthesis of H-MAs and reduces the production of $\alpha$- and E-MAs in the recombinant strain [23,24,27]. Therefore, in $M$. *tuberculosis* the $h m a$ gene encodes the enzyme that generates H-MAs as precursors for both keto and methoxy forms, and this process competes well with the E-MA formation. The accumulation of H-MAs in a $M$. *smegmatis* strain carrying heterologous $h m a$ shows that the organism lacks both Mma3 (or CmaB) and HMAD and therefore cannot transform this intermediate into M-MAs and K-MAs [23,24,27]. Accordingly, a recombinant $M$. *smegmatis* strain carrying $M t b$ $h m a$ could be used to screen candidate $M t b$ genes for HMAD activity via complementation. This is advantageous, as unlike $M t b$, $M$. *smegmatis* is not pathogenic and it grows much faster than $M t b$ or BCG [30].

**Identification of rv0132c as a candidate gene encoding HMAD.** We searched for this gene in the $M t b$ H37Rv genome [31] by using the following criteria. It must be present in both $M t b$ and BCG while absent in $M$. *smegmatis*. It should encode a dehydrogenase capable of catalyzing a two-electron transfer process, as the conversion of H-MAs to K-MAs involves the oxidation of a secondary alcohol group to a keto group. This dehydrogenase must also possess the structural elements for interaction with a hydrophobic substrate such as a mycolic acid. One of the $M t b$ ORFs that matched these characteristics was Rv0132c and it has been known as Fgd2 [31,32]. It is a structural

homolog of coenzyme F₄₂₀-dependent glucose-6-phosphate (G6P) dehydrogenase (Fgd or Fgd1) that catalyzes two-electron oxidation of G6P [glucose-6-phosphate + F₄₂₀ → 6-phosphogluconolactone + reduced F₄₂₀ (F₄₂₀H₂)] [33,34]. Coenzyme F₄₂₀ is a deazaflavin derivative that is found in all mycobacteria [35,36]. At the ground state it functions similar to nicotinamide coenzymes or NAD(P), mediating hydride transfer reactions [37]. Fgd2 does not oxidize G6P and its substrate remains unknown [32,38]. *M. smegmatis* expresses Fgd1 and it lacks Fgd2, whereas both *Mtb* and BCG carry Fgd1 and Fgd2 ([31,32]; NCBI Accession Number: NC_008596). Both Fgd1 and Fgd2 are also homologs of F₄₂₀-dependent methylenetetrahydromethanopterin reductases (Mer) that are found in methanogenic archaea [34]. To obtain some clues to the nature of the substrate that Fgd2 or Rv0132c acts on, we analyzed the primary structure of this protein based on X-ray crystallographic structures of three well characterized Mer homologs: Fgd1 of *Mtb* (PDB ID: 3B4Y) [32], F₄₂₀-dependent methylenetetrahydromethanopterin reductase from *Methanopyrus kandleri* (MkMer; PDB ID: 1EZW) [39], and an F₄₂₀-dependent secondary alcohol dehydrogenase (Adf; PDB ID: 1RHC) from *Methanoculleus thermophilicus* [40]; *M. kandleri* and *M. thermophilicus* are methanogenic archaea. In Fgd1, His⁴⁰, Ser⁷³, Val⁷⁴ and Glu¹⁰⁹ help to bind F₄₂₀ and these residues, except Ser, are functionally conserved in MkMer and Adf [39,40] as well as in Rv0132c (Fig. 2). Ser⁷³ of Fgd1 interacts with F₄₂₀ via the oxygen atom of the backbone carbonyl [32], and in Rv0132c and MkMer this residue has been substituted with Gly and in Adf the equivalent position is occupied by a Cys (Fig. 2). Ser, Cys and Gly are highly compatible in terms of their hydrophobicities and sizes [41–43]. Hydrophobe compatibility indices for Ser-Cys and Ser-Gly pairs in a scale 1–20 (1 and 20 being least and fully compatible, respectively) are 17.7 and 16.8, respectively [41]. The isoelectric points of Ser, Cys and Gly are 5.7, 6.0 and 5.1, respectively [41]. The volumes of Ser, Cys and Gly are 73, 86 and 48 cubic Angstroms, respectively, which are considered similar [44]; the amino acid volumes range from 48 cubic Angstroms for Gly to 163 cubic Angstroms for Trp. Consequently, the above-mentioned substitutions at Ser⁷³ position will not appreciably change the ability of an Fgd1 homolog to bind F₄₂₀. Therefore, in our investigation, we had considered Rv0132c as a potentially F₄₂₀-dependent enzyme. A recent report shows that Rv0132c indeed binds F₄₂₀ [38].

In our next analysis we tried to understand whether Rv0132c has the potential to transform hydrophobic substrates such as mycolic acids. Adf and MkMer interact with hydrophobic substrates whereas Fgd1 accommodates charged glucose-6-phosphate. In Adf the Val¹⁹³ and Leu²²⁷, which are hydrophobic, not only interact with the hydroxybenzyl unit of F₄₂₀ but also help to position the hydrocarbon chain of the substrate [40]. Similarly, Ala¹⁹⁷ and Tyr²²⁹ in MkMer interact with both the F₄₂₀ and the hydrophobic pterin ring of tetrahydromethanopterin [40]. Rv0132c shows partial conservation of these characteristics, as homologous residues in this protein are Ala and Glu, respectively (Fig. 2). In contrast, the equivalent positions in Fgd1 are occupied by Ser and Glu [32], which are less hydrophobic and polar, respectively. In Fgd1, Lys²³², which has a charged side chain, helps to bind a citrate ion, which is a competitive inhibitor of the enzyme [32]. In Rv0132c, Adf and MkMer, this residue has been replaced with Phe, Trp and Cys, respectively (Fig. 2). Additionally, Fgd1 utilizes a positively charged pocket formed by Lys¹⁹⁸, Lys²⁵⁹, and Arg²⁸³ to hold the phosphate group of glucose-6-phosphate [32] and these residues are not conserved in Rv0132c (Fig. 2). Hence, Rv0132c has the potential of interacting with a hydrophobic substrate.

We had observed that two tandem Arg residues in the NH₂-terminus (amino acid residues 1–27, MTGISRRTFGLAAGFGAIGAGGLGGGC; bold and underlined, characteristic residues) form a signature for translocation into the periplasmic space via a Tat-dependent protein export pathway which exists in the mycobacteria [45] and the features shown underlined (see above) represent a putative prokaryotic membrane lipid attachment site (PS00013) where Cys²⁷ could carry lipophilic substrates [46]. Indeed, as our work was complete, Rv0132c was found to be exported to the cell envelope of *Mtb* [38]. Thus, it is reasonable to assume that Rv0132c could interact with the hydrocarbon chains of the mycolic acids (R group, Fig. 1). The relevance of the demonstrated cellular location of Rv0132c to our findings has been discussed below.

**Experimental elucidation of the function of Rv0132c.** We have tested whether Rv0132c represents an F₄₂₀-dependent hydroxymycolic acids dehydrogenase (fHMAD) by introducing this gene and *hma* into *M. smegmatis*. As expected, the plasmid pEP-hma, which was constructed based on the *E. coli*-mycobacterium shuttle vector pSMT3 [47] and carried *Mtb hma* gene (*rv0642c*)

| Fgd1 | DSATVSDHFQPW44 | TSVLTPFRYNP 83 | VFLGVGTGEAL-NE113 |
|------|------------------|-------------------|--------------------|
| Rv0132c | RYVWASDHLQPW82 | TGV-TCPIYYHP121 | VFLGLGTGERL-NE151 |
| Adf | DSVWVDDHFHPW43 | TCI-TCPIRYNP 82 | VGVAVGAGEAM-NE112 |
| MkMer | EYAWICDHYNNY43 | PGI-TNPYTRH-P 73 | AIIGMGPGDKA-TF103 |

| Fgd1 | PEEKERFA130 | AAGGPAVAKY184 | CTSGKG199 | MIEIKISYD236 | QKHS261 |
|------|--------------|---------------|-----------|--------------|---------|
| Rv0132c | GNYRERHD166 | AASGPQSATL220 | AQAR--233 | RAELFAVVG269 | QPN-292 |
| Adf | PSVPVRQD127 | SGMGPKGAKL182 | MTVAAA196 | AMLIWYSVD233 | FKYK258 |
| MkMer | EDEVGPAT129 | GAQGPIMLKT185 | VNAST-N199 | AAYTCFSID235 | ERHG265 |

| Fgd1 | IEQIAKRWIV 286 |
|------|-----------------|
| Rv0132c | IEKVLANWAV 314 |
| Adf | CDTIKENYMC 283 |
| MkMer | DEDMIEAFSI 300 |

Figure 2. Comparison of the primary structures of Rv0132c (fHMAD) and three Mer homologs, Fgd1, MkMer and Adf. A ClustalW comparison was refined manually based on X-ray crystallographic structures of F₄₂₀-dependent glucose-6-phosphate dehydrogenase (Fgd1) of *Mycobacterium tuberculosis*, F₄₂₀-dependent methylenetetrahydromethanopterin reductase from *Methanopyrus kandleri* (MkMer), and F₄₂₀-dependent secondary alcohol dehydrogenase (Adf) from *Methanoculleus thermophilicus* [32,39,40,68]. Residue labels: yellow shade and underlined, determined F₄₂₀-binding residue; yellow shade, predicted F₄₂₀-interacting residue; turquoise shade, forming positively charged pocket for binding the phosphate of glucose-6-phosphate in Fgd1; green shade, residue involved in binding a citrate ion [32].

doi:10.1371/journal.pone.0081985.g002

PLOS ONE | www.plosone.org 3 December 2013 | Volume 8 | Issue 12 | e81985
under the control of its native promoter element, produced H-MAs in *M. smegmatis* mc²155 or wild-type (wt) strain (Fig. 3A, lane hma). The expression of both *hma* and *rv0132c* genes from pEP-rv0132c/hma led to the production of K-MAs (Fig. 3A, lane rv0132c/hma); *rv0132c* alone (pEP-rv0132c) did not provide either H-MAs or K-MAs (Fig. 3A, lane rv0132c). *M. smegmatis*, as such (host control; Fig. 3A, lane None) or while carrying pSMT3 (vector control; Fig. 3A, lane pSMT3), also did not produce either H-MAs or K-MAs; they contained α-, α′-, and E-MAs.

The initial identification of the individual mycolic acid bands on the TLC plates was performed via comparison with previously reported patterns [18,23,24]. Then we carried out mass spectrometric and NMR spectroscopic analysis with materials recovered from the relevant TLC bands. For the H-MAs and K-MAs bands, MALDI-TOF mass spectrometry yielded spectra that were characteristics of respective mycolic acids with 77–82 carbon atoms (Fig. 4) [23]. The mass for every characteristic H-MA ion (Fig. 4A) was 2 units higher than that for a K-MA (Fig. 4B) and this is consistent with the respective structures shown in Fig. 1. ^1H NMR data provided more detailed characterization of relevant mycolic acid species and we discuss the findings below with a focus on the H-atoms marked in red in Fig. 1. This analysis is based on previously reported NMR data on mycolic acids [23,27,29,48,49]. The resonances at 2.7 ppm observed with the E-MA preparation obtained from *M. smegmatis* mc²155 cells (Fig. 5A) were characteristics of the methine protons associated with a trans-epoxide group [23]. In the spectrum obtained with the H-MAs preparation from *M. smegmatis* (pEP-hma) strain (Fig. 5B) the resonances for the above-mentioned epoxy group were not seen and instead it exhibited a resonance at 3.5 ppm representing the methine proton on the carbon that carried the characteristic hydroxyl group of H-MA. Similarly, the ^1H resonances of the methylenic and methine groups that flank the carbonyl group in K-MAs were found at 2.31–2.39 ppm in the spectra for the K-MA preparation from *M. smegmatis* (pEP-rv0132c/hma) strain (Fig. 5C). The spectra for the E-MA, H-MA and K-MA preparations exhibited the following common resonances and this observation is consistent with previous reports [23,27,29,48,49] (Fig. 5A–C): 1.29 ppm – broad, isolated methylene proton; 0.85 ppm - triplet, terminal methyl groups; 3.71 ppm – singlet, methyl ester; 2.50 ppm - multiplet,

methine at position C-2 with respect to the terminal carboxyl group (see within the box at the left corner of Fig. 1). None of the above spectra showed the resonances of the protons that are associated with the cyclopropane groups of mycolic acids produced by *Mtb*; these resonances appear at −0.40, 0.50, and 0.58 ppm for cis-cyclopropanation and 0.01–0.16 ppm for trans-cyclopropanation [49]. Major mycolic acids produced by *M. smegmatis* lack cyclopropanation under normal growth conditions [23,50]. This modification occurs during growth at 25°C [51] and the growth temperature in our study was 37°C.

The above-described analysis showed that the heterologous expression of *Mtb hma* in *M. smegmatis* caused the suppression of the synthesis of E-MAs and the production of H-MAs, and Rv0132c protein converted the H-MAs to K-MAs. Thus, in *Mtb* Rv0132c encoded a hydroxymycolic acid dehydrogenase (HMAD).

The next step was to determine if HMAD was coenzyme F₄₂₀-dependent. The *fbiC* is a key gene for the production of F₄₂₀ chromophore in mycobacteria [52] and mycobacterial strains lacking a functional *fbiC* gene are devoid of this coenzyme [52,53]. We found that a *M. smegmatis ΔfbiC* strain [53] generated H-MAs but not K-MAs when complemented with pEP-rv0132c/hma (Fig. 3B, right most lane or the lane rv0132c/hma for Δ*fbiC*). Complementation with pEP-hma also provided H-MAs in *M. smegmatis ΔfbiC* (Fig. 3B, lane hma) and pEP-rv0132c did not provide either K-MAs or H-MAs (Fig. 3B, lanes rv0132c); the left most lane (lane rv0132c/hma for wild-type (wt) strain) served as a positive control, where production of K-MAs was observed. Hence, HMAD required F₄₂₀ for activity and we call it hereafter fHMAD.

In this context we address two sets of contradicting reports in the literature that concern the biosynthesis of H-MAs and K-MAs in *M. smegmatis* strains carrying clones for the *Mtb hma* gene. In one case the *hma* gene caused the synthesis of both H-MAs and K-MAs [23,24] and in the latter only H-MAs were found in the recombinant [18]. Our result is consistent with the latter [18], as the conversion of a hydroxyl group to a keto group would be catalyzed by an electron transfer enzyme or dehydrogenase such as fHMAD, and not by a methylase/hydrase activity such as seen in Hma.

![Figure 3](https://www.example.com/image.png)

**Figure 3. Thin layer chromatography (TLC) profiles of methyl esters of mycolic acids extracted from various *Mycobacterium smegmatis* strains grown in the absence and presence of PA-824.** Wild type (wt) and Δ*fbiC* strains of *M. smegmatis* carrying the indicated plasmids were analyzed (lane label, name of plasmid): None, no plasmid; pSMT3, pSMT3 (vector control); *hma*, pEP-hma; *rv0132c*, pEP-rv0132c; *rv0132c/hma*, pEP-rv0132c/hma. (+) and (-), cultivation of *M. smegmatis* (pEP-rv0132c/hma) with and without PA-824 (100 microgram per ml), respectively. Mycolic acid types: α, α′, epoxy (E), hydroxy (H), and keto (K) [Fig. S1 shows the respective chemical structures.]. Panel A: *rv0132c* causing the conversion of H-MAs to K-MAs in wild-type *M. smegmatis*; Panel B: Requirement of *fbiC* for the production of K-MAs in *M. smegmatis* (pEP-*rv0132c/hma*) [Note: The left most lane is for wt strain, used as control]. Panel C: Inhibition of the production of K-MAs in *M. smegmatis* (pEP-rv0132c/*hma*) by PA-824.

doi:10.1371/journal.pone.0081985.g003

PLOS ONE | www.plosone.org 12 December 2013 | Volume 8 | Issue 12 | e81985
F420-Dependent Hydroxymycolic Acid Dehydrogenase

A

| Intens. [a.u.] |  |
| --- | --- |
| 1250 |  |
| 1000 |  |
| 750 |  |
| 500 |  |
| 250 |  |
| 0 |  |

H80<sub>1220.122</sub>

| m/z |  |
| --- | --- |
| 1140 |  |
| 1160 |  |
| 1180 |  |
| 1200 |  |
| 1220 |  |
| 1240 |  |
| 1260 |  |

B

| Intens. [a.u.] |  |
| --- | --- |
| 1500 |  |
| 1250 |  |
| 1000 |  |
| 750 |  |
| 500 |  |
| 250 |  |
| 0 |  |

K80<sub>1218.087</sub>

| m/z |  |
| --- | --- |
| 1140 |  |
| 1160 |  |
| 1180 |  |
| 1200 |  |
| 1220 |  |
| 1240 |  |
| 1260 |  |

Figure 4. MALDI-TOF mass spectra of methyl esters of hydroxymycolic acids (A) and ketomycolic acids (B) recovered from engineered *Mycobacterium smegmatis* strains. Hydroxymycolic acids were obtained from the lane *hma* and ketomycolic acids were from lane *rv0132c/hma* (Fig. 3A). Only a part of each spectrum is shown and the annotations for the ion masses are based on reference [23]: labels H & K, ions from hydroxy- and ketomycolic acids preparations; numbers 77–82: total number of carbon atoms in free acids. The unlabeled peaks belong to unidentified species that were present in both preparations.

doi:10.1371/journal.pone.0081985.g004

Fig. 1 shows two mycobacterial MA oxygenation pathways, one of which leads to H-MAs, K-MAs and M-MAs, and the other is for the production of E-MAs. It has been shown that when the former operates, the latter is suppressed [17,23,24,27]. We observed a more stringent form of this regulation in our studies. The data in Fig. 3A show that when *M. smegmatis* was made capable of producing K-MAs, it did not produce E-MAs; a comparison of *rv0132c/hma* lane with any other lane in Fig. 3A leads to this conclusion. This effect was not due to the Rv0132c protein or the DNA elements cloned into pEP-rv0132c/hma, as their presence did not suppress E-MA production when the host lacked *fbiC* (Fig. 3B, lanes *hma*, *rv0132c* and *rv0132c/hma*). Hence it could be hypothesized that K-MAs either inhibit one or more E-MA synthesis enzymes and/or suppresses the expression

increased when K-MA s were produced (Fig. 3A, lane pEP-rv0132c/hma). It is possible that K-MA s enhanced the activities of one or more enzymes that generate H-MA s from α-mycolic acids (Fig. 1) and/or increased the expressions of their genes, and the prevailing fHMAD activity was not at par with the rate of H-MA production. The other explanation is that in *M. smegmatis* (pEP-rv0132c/hma) the cellular level of Hma activity was much higher than that of fHMAD. In this context we note that the overproduction of M-MA s through over-expression of Mma3 or CmaB suppresses K-MA s in *Mtb* [18, 22]. A more detailed study is needed to elucidate the mechanisms underlying these competitions between mycolic acids oxygenation pathways.

### Inhibition of fHMAD by PA-824

PA-824, a new TB drug, inhibits the formation of K-MA s and causes an accumulation of H-MA s in *Mtb* [54]. We tested whether this effect is specifically due to the inhibition of fHMAD. As shown in Fig. 3C, in the presence of PA-824, *M. smegmatis* (pEP-rv0132c/hma) accumulated a high level of H-MA s and contained a reduced level of K-MA s (Fig. 3C). Hence, PA-824 inhibited the heterologously expressed fHMAD. To establish further that fHAMD was inhibited by PA-824, we determined the relative levels of K-MA s in *M. smegmatis* (pEP-rv0132c/hma) cultivated in the presence of this drug at various concentrations. The results showed that the inhibition began at a PA-824 concentration between 10–25 microgram per ml culture and increased further as the drug concentration was raised (Fig. S2). Such a dose-dependent increase in the inhibition of K-MA synthesis by PA-824 has been reported also for *M. tuberculosis* [54]. However, the K-MA synthesis process in *M. smegmatis* (pEP-rv0132c/hma) was much less sensitive to PA-824 than that observed in wild-type *M. tuberculosis*; in *M. tuberculosis* this inhibition begins at a PA-824 concentration between 30–60 nanogram per ml culture [54]. It is possible that the higher minimum inhibitory concentration of PA-824 observed with *M. smegmatis* (pEP-rv0132c/hma) was due to the presence of a higher level of Rv0132c protein in this recombinant strain; *rv0132c* was expressed from a multi-copy plasmid [47] and was driven by both the native promoter as well as the strong and constitutive mycobacterial *hsp60* promoter [47, 55]. Another explanation is that compared to *M. tuberculosis*, *M. smegmatis* takes up PA-824 poorly and as a result for achieving an inhibitory concentration of the drug inside the cell, it had to be supplied in the culture medium at a higher concentration; wild-type *M. smegmatis* is naturally resistant to PA-824 [56]. Nevertheless, the results presented in Figs. 3C and S2 show that the phenomenon of inhibition of K-MA s synthesis by PA-824 that was observed in wild-type *M. tuberculosis* could be reproduced in a *M. smegmatis* strain carrying cloned *hma* and *rv0132c* genes from the former.

PA-824 kills *Mtb* under both aerobic and anaerobic conditions [54, 57, 58]. The anaerobic killing occurs through the reduction of PA-824 by an F${}_{420}$H${}_2$-dependent nitroreductase called Ddn, which is followed by the production of toxic NO [57]. Since F${}_{420}$H${}_2$ is produced by Fgd1, *Mtb* and BCG strains lacking Fgd1 activity are resistant to PA-824 [54]. The aerobic killing of *Mtb* by PA-824 has been thought to occur due to the elimination of K-MA s via unknown mechanisms [54, 58]. Our data has now linked this concept to a gene, *rv0132c*. Curiously, F${}_{420}$ is an integral part of both mycobacterial systems, Ddn and fHAMD, that interact with PA-824.

As mentioned above, fHMAD found to be exported to the cell envelope of *Mtb* [38]. Also, it is thought that complete mycolic acids are transported to the plasma membrane as trehalose monomycolates or TMM [59]. In combination these observations suggest that in *Mtb* at least one additional modification, formation

Figure 5. Proton NMR spectra of methyl esters of mycolic acids recovered from engineered *Mycobacterium smegmatis* strains. The sources of hydroxymycolic acids or H-MA s (A) and Ketomycolic acids or K-MA s (B) were same as that indicated in the legend of Fig. 4. Epoxymycolic acids or E-MA s (C) were from lane “None” in Fig. 3. In each case the inset shows expansion of the relevant regions.

doi:10.1371/journal.pone.0081985.g005

of respective genes. Other possibilities are the interference with the translocation of the precursor of E-MA s to the modification site such as periplasm or a flux-based competition between the two pathways. We also observed that the cellular level of H-MA s
of keto group, of otherwise complete mycolic acids occur within the cell envelop.

**Conclusion**

The hydroxymycolic acid dehydrogenase of *Mtb* was shown to be an F₄₂₀-dependent enzyme encoded by the ORF Rv0132c and it is inhibited by PA-824, a new TB drug.

Our data suggest that there is only one *bona fide* Fgd in the mycobacteria. Citing the lack of glucose-6-phosphate dehydrogenase activity in Rv0132c, it has been recently suggested that this protein should no longer be called Fgd2, and Fgd1 should be called simply Fgd [38]. Our data supports this proposal and provides a functional name for Rv0132c, F₄₂₀-dependent hydroxymycolic acid dehydrogenase (fHMAD). Coenzyme F₄₂₀ is universally present and essential in the strictly anaerobic methanogenic archaea [37]. In the bacterial domain, a similarly wide distribution of this deazaflavin derivative is seen in the Actinobacteria phylum, which includes the mycobacteria [35,36]. Every mycobacteria examined thus far contains F₄₂₀ [35,36]. As mentioned above, in the hydride transfer function F₄₂₀ mimics NAD(P). The mid-point electrode potential of the F₄₂₀/F₄₂₀H₂ couple –360 mV, which is 40 mV lower than that of the nicotinamides [37]. Perhaps in the mycobacteria F₄₂₀ participates in a set of hydride transfer reactions that cannot be accomplished at all or efficiently by the nicotinamides due to thermodynamic reasons, such as a need to operate at a lower redox potential. Such a specialized role has now been seen in the neutralization of nitrosative stress [53] (via a chemical reaction with Fgd-derived F₄₂₀H₂) and in the introduction of a key functionality to the complex mycobacterial cell envelope (the fHMAD reaction as demonstrated here). Both of these actions bring resilience to the mycobacteria against environmental stresses such as those imposed by the human immune system. It is noteworthy, that the current report presents the first example for the involvement of F₄₂₀ in the biosynthesis of mycobacterial cell wall. The nitroreductase (Ddn) that helps to activate PA-824 with F₄₂₀H₂ and the F₄₂₀-dependent enzymes that allow the mycobacteria to decolorize triphenylmethane dyes or to degrade aflatoxins [57,60–62] could also fulfill yet to be described key and normal physiologically relevant cellular functions in these organisms.

**Materials and Methods**

Oligonucleotides, Plasmid, DNA, Bacteria and Growth Conditions

Oligonucleotides, plasmids and bacteria used in this study have been described in the Supporting Material (Table S1). *M. tuberculosis* H37Rv chromosomal DNA was obtained from the National Institutes of Health’s TB Vaccine Testing and Research Materials Contract (TBVTRMC) at the Colorado State University. *E. coli* was grown in Luria-Bertani broth or solid media. *Mycobacterium smegmatis* strains were grown in Middlebrook 7H9 broth or on agar solidified medium with 0.2% glycerol as the carbon and energy source [63]. For liquid cultures Tween 80 at the concentration of 0.05% was also added. When required, *M. smegmatis* strains bearing antibiotic resistance genes were selected on or grown with kanamycin and hygromycin at the concentration of 20 and 150 microgram/ml, respectively, and for similar work with *E. coli* strains ampicillin, kanamycin, and hygromycin concentrations were 100, 20, and 150 microgram/ml, respectively. To study the effect of PA-824 on the mycolic acids content of *M. smegmatis* (pEP-rv0132c/hma), a freshly inoculated culture was grown overnight to an optical density of 0.3 at 600 nm (as measured by use of a DU800 UV/Vis Spectrophotometer,

Beckman Coulter, Brea, CA). It was then supplemented with PA-824 to a desired final concentration from a stock solution (80 mg/ml) in DMSO and grown for additional 36 h. The control culture received DMSO at a concentration of 1.25 ml/liter. PA-824 was a gift from Global Alliance for TB Drug Development (New York, NY) through the Global Health program of the RTI International (Research Triangle Park, NC).

**Molecular Biology Techniques**

*M. smegmatis* chromosomal DNA was isolated as described previously [63]. Transformation of *M. smegmatis* with plasmids was performed via electroporation [64] at 2.5 KV using an Electroporator 2510 (Eppendorf North America, Hauppauge, NY) and a cuvette with a 0.2 cm electrode-gap. For PCR amplification, Phusion polymerase with the GC buffer (Finnzymes Inc., Woburn, MA) was used. Plasmid purification and DNA recovery from agarose gels were done using Qiaprep and Qiaquick columns (Qiagen Inc., Valencia CA), respectively. Manipulations of DNA were performed using standard methods [65].

**Construction of Protein Expression Plasmids and Bacterial Strains**

The protein expression plasmids were based on pSMT3, a mycobacteria-*Escherichia coli* shuttle vector that allows selection for hygromycin resistance and gene expression under the control of the strong and constitutive *hsp60* promoter [47,55]. To generate the plasmids pEP-hma and pEP-rv0132c for the expression of *hma* (*rv0642c*) and *rv0132c* of *M. tuberculosis*, respectively, the corresponding coding sequences along with the respective upstream regions bearing the promoters and ribosome-binding sites (253 bp for *hma* and 316 bp for *rv0132c*) and a bit of the downstream sequences (4 bp for *hma* and 20 bp for *rv0132c*) were PCR-amplified from *M. tuberculosis* H37Rv chromosomal DNA and cloned into pSMT3; the cloning sites were EcoRV and ClaI for *hma* and BamHI and EcoRV for *rv0132c*. The primers used for this work have been described in the Supporting Material (Table S1). The cloned genes in pEP-hma and pEP-rv0132c were expressed in *M. smegmatis* from their native promoters and perhaps also from the plasmid resident mycobacterial *hsp60* promoter. The plasmid pEP-rv0132c/hma that allowed simultaneous expression of *hma* and *rv0132c* was constructed by cloning the *hma* coding sequence along with the respective upstream and downstream sequences as mentioned above at the EcoRV and ClaI sites (or at the 3′end of the *rv0132c* segment) of pEP-rv0132c. The construction of *M. smegmatis ΔfbiC::aph* strain has been described previously [53].

**Preparation and Analysis of Mycolic Acid Methyl Esters**

Mycolic acid methyl esters (MAMEs) were prepared as described previously [66]. Briefly, pelleted mycobacterial cells were saponified via incubation in 15% tetrabutyl ammonium hydroxide at 110°C overnight, followed by the addition of water, diazomethane, and dichloromethane and shaking at room temperature. From this mixture the MAMEs were recovered in the dichloromethane fraction and washed sequentially with equal volumes of water, 0.1N HCl and water. The dichloromethane solution of MAMEs was dried under a stream of nitrogen, dissolved in a toluene-acetonitrile mixture (2:1), and then precipitated at room temperature with an addition of acetonitrile (final toluene:acetonitrile, 2:3). The pellet of MAMEs was dissolved in dichloromethane. Analysis of MAMEs was carried out by thin layer chromatography (TLC) on an aluminum-backed silica gel plate (10×10 cm, Merck 5735-silica gel 60F254) by multiple developments using a solvent comprised of petroleumether and diethyl ether (9:1). Mycolic acid spots were revealed by charring at 110°C for 15 min after spraying with 5% ethanolic molybdophosphoric acid [66].

### Mass Spectrometric and NMR Analysis of Mycolic Acids

This work concerned the methyl esters of hydroxy- and keto-mycolic acids. After performing multi-lane TLC separation for a sample, a terminal lane was cut off and processed for color development as described above. Then using a relevant band in this lane as a guide, the desired mycolic acid spots (silica layer) were scrapped off from the rest of the lanes. From the recovered silica particles, mycolic acids were extracted with dichloromethane and analyzed via MALDI-TOF mass spectrometry at School of Chemical Sciences Mass Spectrometry Laboratory at the University of Illinois at Urbana-Champaign. The Bruker peptide calibration mixture II (Angiotensin II, Angiotensin I, Substance P, Bombesin, ACTH clip 1–17, ACTH clip 18–39, Somatostatin 28, Bradykinin Fragment 1–7, and Renin Substrate Tetradeca-peptide porcine Covered mass range: ~700Da–3200Da) was used for calibration and the matrix was 2,5-dihydroxybenzoic. A Bruker UltraflexTreme mass spectrometer (Fahrenheitstr. 4, D-28359 Bremen, Germany) equipped with a smart beam II laser was used in the positive mode to acquire MALDI-TOF mass spectra. Samples were analyzed in the Reflectron mode.

A Bruker Avance III 600 MHz available at the NMR Laboratory, Department of Chemistry, Virginia Tech, was used to obtain ¹H NMR spectra of the purified mycolic acid methyl esters preparations described above. The solvent was CDCl₃ (100%D) and the reported chemical shifts were relative to the methyl resonances of tetramethylsilane (0 ppm).

---

### Supporting Information

#### Figure S1 Structures of mycolic acids in and *Mycobacterium tuberculosis* complex and *Mycobacterium smegmatis*. The detailed

---

### References

1. Koul A, Arnoult E, Lounis N, Guillemont J, Andries K (2011) The challenge of new drug discovery for tuberculosis. Nature 469: 483–490.
2. Ginsberg AM (2010) Drugs in development for tuberculosis. Drugs 70: 2201–2214.
3. Daffe M, Draper P (1998) The envelope layers of mycobacteria with reference to their pathogenicity. Adv Microb Physiol 39: 131–203.
4. Khasnobis S, Escuyer VE, Chatterjee D (2002) Emerging therapeutic targets in tuberculosis: post-genomic era. Expert Opin Ther Targets 6: 21–40.
5. Lederer E, Adam A, Ciorbaru R, Petit JF, Wietzerbin J (1975) Cell walls of Mycobacteria and related organisms; chemistry and immunostimulant properties. Mol Cell Biochem 7: 87–104.
6. Brennan PJ (2003) Structure, function, and biogenesis of the cell wall of *Mycobacterium tuberculosis*. Tuberculosis (Edinb) 83: 91–97.
7. Takayama K, Wang C, Besra GS (2005) Pathway to synthesis and processing of mycolic acids in *Mycobacterium tuberculosis*. Clin Microbiol Rev 18: 81–101.
8. Yagi T, Mahapatra S, Mikusova K, Crick DC, Brennan PJ (2003) Polymerization of mycobacterial arabinogalactan and ligation to peptidoglycan. J Biol Chem 278: 26497–26504.
9. VanderBeken S, Al Dulayymi JR, Naessens T, Koza G, Maza-Iglesias M, et al. (2011) Molecular structure of the *Mycobacterium tuberculosis* virulence factor, mycolic acid, determines the elicited inflammatory pattern. European Journal of Immunology 41: 450–460.
10. Brennan PJ, Crick DC (2007) The cell-wall core of *Mycobacterium tuberculosis* in the context of drug discovery. Curr Top Med Chem 7: 475–488.
11. Hunter RL, Olsen M, Jagannath C, Actor JK (2006) Trehalose 6,6′-dimycolate and lipid in the pathogenesis of caseating granulomas of tuberculosis in mice. Am J Pathol 168: 1249–1261.
12. Indrigo J, Hunter RL, Jr., Actor JK (2003) Cord factor trehalose 6,6′-dimycolate (TDM) mediates trafficking events during mycobacterial infection of murine macrophages. Microbiology 149: 2049–2059.
13. Hunter RL, Olsen MR, Jagannath C, Actor JK (2006) Multiple roles of cord factor in the pathogenesis of primary, secondary, and cavitary tuberculosis, including a revised description of the pathology of secondary disease. Ann Clin Lab Sci 36: 371–386.

---

### structures of R groups in various mycolic acids are shown [4]. The reference cited here is listed in File S1.

(TIF)

#### Figure S2 Dose-dependent inhibition of K-MA production in *M. smegmatis* (pEP-rv0132c/hma) by PA-824. Wild-type *M. smegmatis* was used as control; the data in Fig. 3 show that neither the expression constructs pEP-hma and pEP-rv0132cnor the vector pSMT3 allow the production of K-MAs in *M. smegmatis*. The other details of the study have been presented in the MATERIALS AND METHODS. Mycolic acid types: α, α′, epoxy (E), hydroxy (H), and keto (K) [Fig. 1S shows the respective chemical structures.].

(TIF)

#### Table S1 Oligonucleotides, plasmids, and strains. The references cited in this table are listed in File S1.

(DOC)

#### File S1 References for Table S1 and Figure S1.

(DOC)

### Acknowledgments

We thank Furong Sun, Mass Spectrometry Lab, School of Chemical Sciences, University of Illinois at Urbana-Champaign, and Kenneth Knott, Analytical Service Laboratory, Department of Chemistry, Virginia Tech, for their help with mass spectrometry and NMR spectroscopy, respectively, the Global Alliance for TB Drug Development (New York, NY) and the RTI International (Research Triangle Park, NC) for the gift of PA-824, and the NIH TB Vaccine Testing and Research Materials Contract (TBVTRMC) at the Colorado State University for the *Mycobacterium tuberculosis* chromosomal DNA.

### Author Contributions

Conceived and designed the experiments: EP BM. Performed the experiments: EP. Analyzed the data: EP BM. Wrote the paper: EP BM.

---

### 14. Brennan PJ, Nikaido H (1995) The envelope of mycobacteria. Annu Rev Biochem 64: 29–63.
15. Fujita Y, Doi T, Sato K, Yano I (2005) Diverse humoral immune responses and changes in IgG antibody levels against mycobacterial lipid antigens in active tuberculosis. Microbiology 151: 2065–2074.
16. Minnikin DE, Minnikin SM, Parlett JH, Goodfellow M, Magnusson M (1984) Mycolic acid patterns of some species of *Mycobacterium*. Arch Microbiol 139: 225–231.
17. Dubnau E, Chan J, Raynaud C, Mohan VP, Laneelle MA, et al. (2000) Oxygenated mycolic acids are necessary for virulence of *Mycobacterium tuberculosis* in mice. Mol Microbiol 36: 630–637.
18. Yuan Y, Zhu Y, Crane DD, Barry CE, 3rd (1998) The effect of oxygenated mycolic acid composition on cell wall function and macrophage growth in *Mycobacterium tuberculosis*. Mol Microbiol 29: 1449–1458.
19. Beukes M, Lemmer Y, Deysel M, Al Dulayymi JR, Baird MS, et al. (2010) Structure-function relationships of the antigenicity of mycolic acids in tuberculosis patients. Chem Phys Lipids 163: 800–808.
20. Dao DN, Sweeney K, Hsu T, Gurcha SS, Nascimento IP, et al. (2008) Mycolic acid modification by the *mmaA4* gene of *M. tuberculosis* modulates IL-12 production. PLoS Pathog 4: e1000081.
21. Rao V, Gao F, Chen B, Jacobs WR, Jr., Glickman MS (2006) Trans-cyclopropanation of mycolic acids on trehalose dimycolate suppresses *Mycobacterium tuberculosis*-induced inflammation and virulence. J Clin Invest 116: 1660–1667.
22. Sambandan D, Dao DN, Weinrick BC, Vilcheze C, Gurcha SS, et al. (2013) Keto-Mycolic Acid-Dependent Pellicle Formation Confers Tolerance to Drug-Sensitive *Mycobacterium tuberculosis*. MBio 4.
23. Dinadayala P, Lavall F, Raynaud C, Lemassu A, Laneelle MA, et al. (2003) Tracking the putative biosynthetic precursors of oxygenated mycolates of *Mycobacterium tuberculosis*. Structural analysis of fatty acids of a mutant strain deviod of methoxy- and ketomycolates. J Biol Chem 278: 7310–7319.
24. Dubnau E, Laneelle MA, Soares S, Benichou A, Vaz T, et al. (1997) *Mycobacterium bovis* BCG genes involved in the biosynthesis of cyclopropyl keto-and hydroxy-mycolic acids. Mol Microbiol 23: 313–322.
25. Dubnau E, Marrakchi H, Smith I, Daffe M, Quemard A (1998) Mutations in the *cmaB* gene are responsible for the absence of methoxymycolic acid in *Mycobacterium bovis* BCG Pasteur. Mol Microbiol 29: 1526–1528.

26. Behr MA, Schroeder BG, Brinkman JN, Slayden RA, Barry CE, 3rd (2000) A point mutation in the *mma3* gene is responsible for impaired methoxymycolic acid production in *Mycobacterium bovis* BCG strains obtained after 1927. J Bacteriol 182: 3394–3399.

27. Yuan Y, Barry CE, 3rd (1996) A common mechanism for the biosynthesis of methoxy and cyclopropyl mycolic acids in *Mycobacterium tuberculosis*. Proc Natl Acad Sci U S A 93: 12828–12833.

28. Mukherjee T, Boshoff H (2011) Nitroimidazoles for the treatment of TB: past, present and future. Future Med Chem 3: 1427–1454.

29. Laval F, Haïtes R, Movahedzadeh F, Lemassu A, Wong CY, et al. (2008) Investigating the function of the putative mycolic acid methyltransferase UmaA: divergence between the *Mycobacterium smegmatis* and *Mycobacterium tuberculosis* proteins. J Biol Chem 283: 1419–1427.

30. O’toole R (2010) Experimental Models Used to Study Human Tuberculosis. Advances in Applied Microbiology, Vol 71 71: 75–89.

31. Cole ST, Brosch R, Parkhill J, Garnier T, Churcher C, et al. (1998) Deciphering the biology of *Mycobacterium tuberculosis* from the complete genome sequence. Nature 393: 537–544.

32. Bashiri G, Squire CJ, Moreland NJ, Baker EN (2008) Crystal structures of F<sub>420</sub>-dependent glucose-6-phosphate dehydrogenase FGD1 involved in the activation of the anti-tuberculosis drug candidate PA-824 reveal the basis of coenzyme and substrate binding. J Biol Chem 283: 17531–17541.

33. Purwantini E, Daniels L (1996) Purification of a novel coenzyme F<sub>420</sub>-dependent glucose-6-phosphate dehydrogenase from *Mycobacterium smegmatis*. J Bacteriol 178: 2861–2866.

34. Purwantini E, Daniels L (1998) Molecular analysis of the gene encoding F<sub>420</sub>-dependent glucose-6-phosphate dehydrogenase from *Mycobacterium smegmatis*. J Bacteriol 180: 2212–2219.

35. Daniels L, Bakhiet N, Harmon K (1985) Widespread Distribution of a 5-Deazaflavin Cofactor in *Actinomycetes* and Related Bacteria. Systematic and Applied Microbiology 6: 12–17.

36. Purwantini E, Gillis TP, Daniels L (1997) Presence of F<sub>420</sub>-dependent glucose-6-phosphate dehydrogenase in *Mycobacterium* and *Nocardia* species, but absence from *Streptomyces* and *Corynebacterium* species and methanogenic Archaea. FEMS Microbiol Lett 146: 129–134.

37. DiMarco AA, Bobik TA, Wolfe RS (1990) Unusual coenzymes of methanogenesis. Annu Rev Biochem 59: 355–394.

38. Bashiri G, Perkowski EF, Turner AP, Fletcher ME, Braunstein M, et al. (2012) Tat-dependent translocation of an F<sub>420</sub>-binding protein of *Mycobacterium tuberculosis*. PLoS One 7: e45003.

39. Shima S, Warkentin E, Grabarse W, Sordel M, Wicke M, et al. (2000) Structure of coenzyme F<sub>420</sub>-dependent methylenetetrahydromethanopterin reductase from two methanogenic archaea. J Mol Biol 300: 935–950.

40. Aufhammer SW, Warkentin E, Berk H, Shima S, Thauer RK, et al. (2004) Coenzyme binding in F<sub>420</sub>-dependent secondary alcohol dehydrogenase, a member of the bacterial luciferase family. Structure 12: 361–370.

41. Biro JC (2006) Amino acid size, charge, hydropathy indices and matrices for protein structure analysis. Theor Biol Med Model 3: 15.

42. Zamyatin AA (1972) Protein volume in solution. Progress in biophysics and molecular biology 24: 107–123.

43. Chothia C (1976) The nature of the accessible and buried surfaces in proteins. J Mol Biol 105: 1–12.

44. Creighton TE (1993) Proteins Structure and Molecular Properties. New York: W. H. Freeman & Co.

45. McDonough JA, Hacker KE, Flores AR, Pavelka MS, Jr., Braunstein M (2005) The twin-arginine translocation pathway of *Mycobacterium smegmatis* is functional and required for the export of mycobacterial beta-lactamases. J Bacteriol 187: 7667–7679.

46. Sutcliffe IC, Harrington DJ (202) Pattern searches for the identification of putative lipoprotein genes in Gram-positive bacterial genomes. Microbiology 148: 2065–2077.

47. Garbe TR, Barathi J, Barnini S, Zhang Y, Abou-Zeid C, et al. (1994) Transformation of mycobacterial species using hygromycin resistance as selectable marker. Microbiology 140 (Pt 1): 133–138.

48. Barkan D, Rao V, Sukenick GD, Glickman MS (2010) Redundant function of *cmaA2* and *mmaA2* in *Mycobacterium tuberculosis* cis cyclopropanation of oxygenated mycolates. J Bacteriol 192: 3661–3668.

49. Quemard A, Laneelle MA, Marrakchi H, Prome D, Dubnau E, et al. (1997) Structure of a hydroxymycolic acid potentially involved in the synthesis of oxygenated mycolic acids of the *Mycobacterium tuberculosis* complex. Eur J Biochem 250: 758–763.

50. George KM, Yuan Y, Sherman DR, Barry CE, 3rd (1995) The biosynthesis of cyclopropanated mycolic acids in *Mycobacterium tuberculosis*. Identification and functional analysis of CMAS-2. J Biol Chem 270: 27292–27298.

51. Alibaud L, Alahari A, Trivelli X, Ojha AK, Hatfull GF, et al. (2010) Temperature-dependent regulation of mycolic acid cyclopropanation in saprophytic mycobacteria: role of the *Mycobacterium smegmatis* 1351 gene (MSMEG_1351) in CIS-cyclopropanation of alpha-mycolates. J Biol Chem 285: 21698–21707.

52. Choi KP, Kendrick N, Daniels L (2002) Demonstration that *fbiC* is required by *Mycobacterium bovis* BCG for coenzyme F<sub>420</sub> and FO biosynthesis. J Bacteriol 184: 2420–2428.

53. Purwantini E, Mukhopadhyay B (2009) Conversion of NO<sub>2</sub> to NO by reduced coenzyme F<sub>420</sub> protects mycobacteria from nitrosative damage. Proc Natl Acad Sci U S A 106: 6333–6338.

54. Stover CK, Warrener P, VanDevanter DR, Sherman DR, Arain TM, et al. (2000) A small-molecule nitroimidazopyran drug candidate for the treatment of tuberculosis. Nature 405: 962–966.

55. Gaora PO (1998) Expression of genes in mycobacteria. Methods Mol Biol 101: 261–273.

56. Manjunatha UH, Boshoff H, Dowd CS, Zhang L, Albert TJ, et al. (2006) Identification of a nitroimidazo-oxazine-specific protein involved in PA-824 resistance in *Mycobacterium tuberculosis*. Proc Natl Acad Sci U S A 103: 431–436.

57. Singh R, Manjunatha U, Boshoff HI, Ha YH, Niyomrattanakit P, et al. (2008) PA-824 kills nonreplicating *Mycobacterium tuberculosis* by intracellular NO release. Science 322: 1392–1395.

58. Manjunatha U, Boshoff HI, Barry CE (2009) The mechanism of action of PA-824: Novel insights from transcriptional profiling. Commun Integr Biol 2: 215–218.

59. Varela C, Rittmann D, Singh A, Krumbach K, Bhatt K, et al. (2012) MmpL genes are associated with mycolic acid metabolism in mycobacteria and corynebacteria. Chem Biol 19: 498–506.

60. Taylor MC, Jackson CJ, Tattersall DB, French N, Peat TS, et al. (2010) Identification and characterization of two families of F<sub>420</sub>H<sub>2</sub>-dependent reductases from Mycobacteria that catalyse aflatoxin degradation. Mol Microbiol 78: 561–575.

61. Guerra-Lopez D, Daniels L, Rawat M (2007) *Mycobacterium smegmatis mc*<sup>2</sup> 155 *fbiC* and MSMEG_2392 are involved in triphenylmethane dye decolorization and coenzyme F<sub>420</sub> biosynthesis. Microbiology 153: 2724–2732.

62. Graham DE (2010) A new role for coenzyme F<sub>420</sub> in aflatoxin reduction by soil mycobacteria. Mol Microbiol 78: 533–536.

63. Mukhopadhyay B, Purwantini E (2000) Pyruvate carboxylase from *Mycobacterium smegmatis*: stabilization, rapid purification, molecular and biochemical characterization and regulation of the cellular level. Biochim Biophys Acta 1475: 191–206.

64. Snapper SB, Melton RE, Mustafa S, Kieser T, Jacobs WR, Jr. (1990) Isolation and characterization of efficient plasmid transformation mutants of *Mycobacterium smegmatis*. Mol Microbiol 4: 1911–1919.

65. Sambrook J, Fritsch E, Maniatis T (1989) Molecular Cloning, A Laboratory Manual, 2nd Ed. New York, NY: Cold Spring Harbor Laboratory Press.

66. Besra GS (1998) Preparation of Cell-Wall Fractions from Mycobacteria. In: Parish T, Stoker NG, editors. Mycobacteria Protocols. Totowa, New Jersey: Humana Press.

67. Glickman MS, Cahill SM, Jacobs WR, Jr. (2001) The *Mycobacterium tuberculosis cmaA2* gene encodes a mycolic acid trans-cyclopropane synthetase. J Biol Chem 276: 2228–2233.

68. Aufhammer SW, Warkentin E, Ermler U, Hagemeier CH, Thauer RK, et al. (2005) Crystal structure of methylenetetrahydromethanopterin reductase (Mer) in complex with coenzyme F<sub>420</sub>: Architecture of the F<sub>420</sub>/FMN binding site of enzymes within the nonprolyl cis-peptide containing bacterial luciferase family. Protein Sci 14: 1840–1849.
