
Antiviral Research 103 (2014) 39–50

**Contents lists available at ScienceDirect**

**Antiviral Research**

journal homepage: www.elsevier.com/locate/antiviral

---

**Review**

# The SARS coronavirus nucleocapsid protein – Forms and functions

Chung-ke Chang<sup>a</sup>, Ming-Hon Hou<sup>b</sup>, Chi-Fon Chang<sup>c</sup>, Chwan-Deng Hsiao<sup>d</sup>, Tai-huang Huang<sup>a,c,e,*</sup>

<sup>a</sup>Institute of Biomedical Sciences, Academia Sinica, Taipei 11529, Taiwan, ROC  
<sup>b</sup>Department of Life Science, National Chung Hsing University, Taichung 40254, Taiwan, ROC  
<sup>c</sup>The Genomics Research Center, Academia Sinica, Taipei 11529, Taiwan, ROC  
<sup>d</sup>Institute of Molecular Biology, Academia Sinica, Taipei 11529, Taiwan, ROC  
<sup>e</sup>Department of Physics, National Taiwan Normal University, Taipei 11677, Taiwan, ROC

---

## Article Info

### Article history:
- Received 28 October 2013
- Revised 8 December 2013
- Accepted 20 December 2013
- Available online 11 January 2014

### Keywords:
- SARS
- Coronavirus
- Nucleocapsid protein
- Capsid packaging
- Intrinsic disorder
- RNP

---

## Abstract

The nucleocapsid phosphoprotein of the severe acute respiratory syndrome coronavirus (SARS-CoV N protein) packages the viral genome into a helical ribonucleocapsid (RNP) and plays a fundamental role during viral self-assembly. It is a protein with multifarious activities. In this article we will review our current understanding of the N protein structure and its interaction with nucleic acid. Highlights of the progresses include uncovering the modular organization, determining the structures of the structural domains, realizing the roles of protein disorder in protein–protein and protein–nucleic acid interactions, and visualizing the ribonucleoprotein (RNP) structure inside the virions. It was also demonstrated that N-protein binds to nucleic acid at multiple sites with a coupled-allostery manner. We propose a SARS-CoV RNP model that conforms to existing data and bears resemblance to the existing RNP structures of RNA viruses. The model highlights the critical role of modular organization and intrinsic disorder of the N protein in the formation and functions of the dynamic RNP capsid in RNA viruses. This paper forms part of a symposium in Antiviral Research on “From SARS to MERS: 10 years of research on highly pathogenic human coronaviruses.”

© 2014 Elsevier B.V. All rights reserved.

---

## Contents

1. Introduction ..... 40
2. Packaging of RNP inside the virus ..... 40
3. Modular organization of the SARS-CoV N protein ..... 41
4. Structure of SARS-CoV N protein ..... 42
   - 4.1. Structure of the N-terminal domain ..... 42
   - 4.2. Structure of the C-terminal structural domain ..... 43
   - 4.3. Comparison with N proteins of other coronaviruses ..... 44
   - 4.4. The CTD dimer interface suggests possible evolutionary link between corona- and arteriviruses ..... 44
5. Biophysical aspects of SARS-CoV N protein self-association ..... 44
   - 5.1. The CTD is a transient self-association site of the SARS-CoV N protein ..... 44
   - 5.2. Electrostatic screening and phosphorylation-mimicking mutation affect SARS-CoV N protein self-association ..... 45
6. Protein-nucleic acid interaction ..... 46
   - 6.1. SARS-CoV N protein binds to nucleic acids at multiple sites ..... 46
   - 6.2. Intrinsic disorder and coupled-allosteric binding of N to nucleic acids ..... 46
7. Packaging of the SARS-CoV ribonucleocapsid ..... 47
   - 7.1. A putative model ..... 47
   - 7.2. Comparison with other viral RNP structures ..... 47
8. Future perspectives ..... 47

---

### Abbreviations:
CTD, the C-terminal domain of N protein (a.a. 248–365); DD, the di-domain comprising NTD, LKR and CTD (a.a. 45–365); HCoV, human coronavirus; IDP, intrinsically disordered protein; IDR, intrinsically disordered region; LKR, the linker region of SARS-CoV N protein (a.a. 182–247); N, nucleocapsid protein; NTD, the N-terminal domain of N protein (a.a. 45–181); MHV, mouse hepatitis virus; RMSD, root mean square deviation; RNP, ribonucleoprotein; SARS-CoV, severe acute respiratory syndrome coronavirus.

* Corresponding author at: Institute of Biomedical Sciences, Academia Sinica, Taipei 11529, Taiwan, ROC. Tel.: +886 2 2652 3036; fax +886 2 2788 7641.  
E-mail address: bmthh@ibms.sinica.edu.tw (T.-h. Huang).

0166-3542/$ - see front matter © 2014 Elsevier B.V. All rights reserved.  
http://dx.doi.org/10.1016/j.antiviral.2013.12.009

Acknowledgement . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Neuman et al. have employed single particle image analysis of 2D electron cryo-microscopy (cryo-EM) to investigate the structural organization of SARS-CoV at 4 nm resolution ([Neuman et al., 2006](https://doi.org/10.1016/j.virol.2005.11.038)) (Fig. 1A). They observed overlapping lattices arranged near the viral membrane surrounded by a disordered core. The RNP particles displayed a coiled shape when released from the viral membrane. Edge views revealed that most of the viral RNP was located within ~25 nm of the inner face of the membrane. Fifteen-nanometer-wide strands of electron-dense material can be seen emerging from a spontaneously disrupted SARS-CoV particle. The RNP is maintained in a spherically packaged form at the inner face of the membrane with no indication of icosahedral symmetry. The SARS-CoV nucleocapsid is separated from the envelope by a gap, which contains thread-like densities that connect the M protein density on the inner face of the viral membrane to a two-dimensionally ordered ribonucleoprotein layer (Fig. 1B), a feature also seen in TGEV ([Risco et al., 1996](https://doi.org/10.1016/S0042-6822(96)00075-7)). Since the carboxyl tail of M protein has been shown to interact specifically with N ([Escors et al., 2001](https://doi.org/10.1016/S0042-6822(01)00010-8); [Kuo and Masters, 2002](https://doi.org/10.1016/S0042-6822(02)00008-0); [Narayanan et al., 2000](https://doi.org/10.1016/S0042-6822(00)00122-9); [Sturman et al., 1980](https://doi.org/10.1016/0042-6822(80)90286-0)), these results suggest that the M–N interactions constrain some N molecules in close apposition to the envelope. Glycoprotein spikes were found to be aligned with the membrane-proximal layer of RNP densities, implying that protein location within the envelope is constrained by consistent S–M, M–M, and M–N interactions. The organization further implies that M is also organized in a two-dimensional lattice which was proposed to be a likely scaffold for viral assembly ([de Haan et al., 2000](https://doi.org/10.1016/S0042-6822(00)00123-0)). The stoichiometry of the unit cell at the virion surface was estimated to be approximately 1S₃:16M:4N to 1S₃:25M:4N proteins, where S₃ is a spike trimer, with the remainder of the N protein distributed throughout the virion core. Nucleoprotein molecules in the paracrystalline RNP shell appeared to be partially organized through interactions at points of contact in the RNP lattice. The distribution of density in the viral core was consistent with a membrane-proximal RNP lattice formed by local approaches of the coiled ribonucleoprotein. The cryo-EM images did not reveal any internal features within the ~25 nm-thick RNP zone proximal to the envelope. This suggests that inner core densities of mature coronaviruses are not consistently ordered with respect to the membrane. A model based on interpretation of the 2D cryo-EM data is shown on Fig. 1B.

Koster and associates, on the other hand, employed 3D cryo-electron tomography to study the structure of mouse hepatitis virus (MHV) particles ([Barcena et al., 2009](https://doi.org/10.1073/pnas.0900577106)). They showed that the viral envelope has a thickness that is almost twice that of a typical biological membrane. The extra internal layer was attributed to the C-terminal domains of the M protein. In the interior of the particles coiled structures and tubular shapes are observed, consistent with a helical nucleocapsid formed by self-association of the N protein and the genomic RNA. The RNP seems to be relatively densely packed and disorganized underneath the envelope. Consistent with previous observations, they also observed quasi-circular density profiles approximately 11 nm in diameter enclosing an empty space approximately 4 nm in diameter inside the otherwise relatively disorganized interior. The observation of only short coiled fragments in the reconstructions strongly suggests that the helical nucleocapsid is a very flexible structure that extensively twists and folds upon itself, adopting orientations that are not easily recognizable as coils in tomographic sections. The general features and global architecture observed for MHV were also observed in TGEV, suggesting a general model for the architecture of CoVs.

The pleomorphic nature of the coronavirus particle has hampered the effort to obtain high-resolution virion image at atomic resolution. Nonetheless, the cryo-EM images have provided considerable insights regarding the organization of various structural proteins, especially the virion envelope and the RNP. It also revealed a structural plasticity that may play an essential role in

the virus life cycle. The presence of partially organized and flexible N protein regions could facilitate packaging of the genomic RNA by CoVs.

## 3. Modular organization of the SARS-CoV N protein

It is perhaps surprising that prior to the outbreak of SARS the structure of coronavirus N proteins were never studied in detail. The earliest structural model of coronavirus N protein was proposed by Parker and Masters ([Masters, 1992](https://doi.org/10.1016/0042-6822(92)90108-8); [Parker and Masters, 1990](https://doi.org/10.1016/0042-6822(90)90108-8)) in the 1990s based on sequence comparison and evolutionary studies of MHV, a prototypical Group II coronavirus. In their model, the N protein comprised three domains separated by two spacers. The central domain acted as the RNA-binding domain, whereas the remaining two acidic domains presumably played a role in protein–protein interactions. Although the model provided a general overview of coronavirus N protein structure at the time, it lacked the necessary details to provide a clear picture of the structure–function relationship of the protein.

The SARS pandemic ushered a new era of structural studies on coronavirus protein structure. The SARS-CoV N protein is a 46 kDa phosphoprotein of 422 amino acids, sharing 20–30% sequence identity with the N proteins of other coronaviruses ([Marra et al., 2003](https://doi.org/10.1038/nature01515); [Rota et al., 2003](https://doi.org/10.1073/pnas.0305156100)) (Fig. 2). It forms a dimer, which constitutes the basic building block of the nucleocapsid, through its C-terminus ([Chang et al., 2005](https://doi.org/10.1073/pnas.0500662102); [Surjit et al., 2004](https://doi.org/10.1074/jbc.M402876200); [Yu et al., 2005](https://doi.org/10.1074/jbc.M412589200)). Huang et al. first solved the solution structure of the N-terminal domain (Fig. 1C), which they coined as RBD (residues 45–181) and demonstrated that this domain is capable of binding to RNA with micromolar affinity ([Huang et al., 2004b](https://doi.org/10.1074/jbc.M402876200)). The term RBD is misleading since RNA binds to N at multiple sites other than RBD. To avoid confusion we will use the acronym, NTD, from now on instead. The structure of the dimerization domain (residues 248–365) was solved by X-ray crystallography and NMR ([Chen et al., 2007](https://doi.org/10.1074/jbc.M611484200); [Takeda et al., 2008](https://doi.org/10.1074/jbc.M708856200); [Yu et al., 2006](https://doi.org/10.1074/jbc.M605886200)) (Fig. 1C). Since the dimerization domain is not just a dimerization domain and it also binds to nucleic acid we refer it as CTD instead. As shown by NMR, chromatography, and small-angle X-ray scattering (SAXS), the NTD and CTD forms two independent domains that do not interact with each other ([Chang et al., 2006](https://doi.org/10.1074/jbc.M605886200)). It was evident at this point that the original three-domain model would require extensive revision in light of these new developments.

The modular organization of SARS-CoV N was further defined in more detail by a combination of bioinformatics and biophysical methods by Chang et al. who showed that the two structural domains are interspersed by intrinsically disordered regions (IDRs) that account for ~40% of the amino-acid residues (Fig. 1C) ([Chang et al., 2006, 2009](https://doi.org/10.1074/jbc.M605886200)). A relatively new concept in structural biology, intrinsically disordered proteins (IDPs) or IDRs lack a defined tertiary structure in the native state, but play important roles in biological processes, particularly in macromolecular interactions ([Dunker et al., 2001](https://doi.org/10.1074/jbc.M100002200); [Dyson, 2011, 2012](https://doi.org/10.1016/j.jmb.2011.05.010); [Dyson and Wright, 2005](https://doi.org/10.1016/j.jmb.2005.03.026); [Xie et al., 2007](https://doi.org/10.1074/jbc.M701400200)). In the case of SARS-CoV N protein, all three IDRs (residues 1–44, 182–247, and 366–422) are able to modulate the RNA-binding activity of the NTD and CTD ([Chang et al., 2009](https://doi.org/10.1074/jbc.M808856200)). The middle IDR, which we coined LKR, and C-terminal IDR have both been implicated in the oligomerization of the N protein ([He et al., 2004a](https://doi.org/10.1074/jbc.M402876200); [Luo et al., 2006](https://doi.org/10.1074/jbc.M605886200)). The LKR includes a Ser/Arg-rich region that contains a number of putative phosphorylation sites, which may regulate N protein function ([Peng et al., 2008](https://doi.org/10.1074/jbc.M808856200); [Surjit et al., 2005](https://doi.org/10.1074/jbc.M402876200); [Wu et al., 2009](https://doi.org/10.1074/jbc.M808856200)) and N–M interaction ([He et al., 2004b](https://doi.org/10.1074/jbc.M402876200)). Based on these new findings, Chang et al. proposed a structure-based domain arrangement for SARS-CoV N protein where the NTD and CTD are sandwiched between three IDRs. Sequence alignments suggested that other coronavirus N proteins might share the

SARS-CoV -----------------MSDNGPQSQRSAPRITFGGPTDSTDNNQNGRN GARPKQRRP----------------QGLPNNTASWFTALTQHGK-EELRFPRGQGVPINTNSGPDDQIGYYRRATRR-VR 96
IBV ---------------------------------------MASGKAAGKTDA PAPVIKLGGPKPPKVGSSGNASWFQAIKKLLNTPPP KFEGSGVPDNENIKPSQQHGYWR RQAR--FK 78
MHV MSFVPGQENAGGRSSSVNRAGN GILKTTWADQTERGPNQNQR RR-NQPKQTATT------QPNSGSVVPHYSWFSGITQFKGKEFQFAEGQGVP IANGIPASEQKGYYRHNRRSFK 113
HCov OC43 MSFTPGKQSS-SRASSGNRSGN GIL---KWADQSDQVRNVQTRRR-AQPKQTATS-----QQPSGGNVVPYYSWFSGITQFKGKEFEFVEGOGPPIAPGVPA TEAKGYWYRHNRGSFK 110
MERS-CoV -----------------MASPAAPRAVSFADNNDITNT NLS-----RGRGRNPKP----------------RAAPNNTVSWYTGLTQHGK-VPLTFPPGQGVPLNANSTPAQNA GWRRQDRK-IN 86
:                                                                                   **: .:.                                                                                     *.* * :**:* * .. 
SARS-Cov GGDGKMKEISPRWYFYYLGTGPEASL PYGANKEGIVWVATEGALNTPKD HIGTRNPNNAAATVLQLPQGTTLP---KGFYAE GS----RGGSQASSRSSSRSGNSRNSTPGSSRGNSPAR 210
IBV PGKGGRKPVPD AWYFY YTGTGPAADLNWGDTQDGIVWVA AKGADTKSRSNQTRDPDKFDQYPLRFS DGGPDNFRWDFIPLNRGSRGRSTAASSA---AASRAP SREGSRGRRSD----- 191
MHV TPDGQQKQLLPRWYFY YLG TGP HAGASYGDSIEGVFWVANSQADTNTRSDIVERDPSSHEAIPTRFAPGTVLP---QGFYVEGS---GRSAPAS-R--SGSRSQSRGPN--NRARSSSNQ 222
HCov OC43 TADGNQRQLLPRWYFY YLG TGP HAKDQYGTDIDGVYWVASNQADVNTPADI VDRDPSSDEAIPTRFPPGTVLP---QGY YIEGS---GRSAPNS-R--STSTRTSSRASSAGSRSRANSGN 221
MERS-CoV TGNG-IKQLAPRWYFY YTG TGP EEAALPFRAVKDGIVWHEDGATDAPS-TFGTRNPNNDSAIVTQFAPGTKLP---KNFHIEGT---GGNSQSSSRASSVSRNSSRSSQGSRSGNST RG 198
.* : : : ***** ***** * : :*:** . * *:*.. :: * : * : ** ** .
SARS-CoV MA---SGGGETALALLLDRLNQLES KVSGK--QQOQGQT VTTKSAA-----EASKKPRQKRTATKQYNVTQAFGRRGPEQTQGNFGDQDLIRQGTDYKHW PQIAQFAPSASAFFGMSRI 321
IBV --------SG----------------DDL IARAAKI IQDQQKGS RITKAKAD----EMAHRRYCKRTIPPNYRVDQVFGRP RTKGK-EGNFGDDKMNEEGIKDGRVTAMLNLVPSSHA CLFGSRV 287
MHV RQPA-ST----------VKPDM AEEI AALVLALKGKDAGQPKVTKQSAKEVRQKILNKPRQKRTPNKQCPVQC FGKRGP NQ---NFGGSEMLKLGTSDPQFPILAELAPT VGAFFFGSKL 330
HCov OC43 RTPT-SG----------UTPDM ADQIASLV LALKGKDATKPQQT KHTTA KEVRQKILNKPRQKRSPNKQCTVQQC FGRGP NQ---NFGGEMLKLGTS DPQFPILAELAPT AGAFFFGSRL 329
MERS-CoV TSPGPSGIGAVGDDL YLDLLNRLQALESGKV--KQS QPKVITKKDAA----AAKNKM RHKRTSTKSFNMVQAFGLRGP GDLQGNFDLQLNKLGTEDPRWPQIAELAPT ASAFFGMSQF 312
* : : : : . : . :*** * . :*:: . : * ** * . *** .: .* . : :::.*: * : *:.
SARS-Cov GMEVTP------------SGTWLT TYHGAIKLDDKDPQFKDNVILLNKHIDAYKT FPPTEPKDKKKKTD-------------EAQPLP-------------QRQKKQPTVTLLPAADMDFS RQL 408
IBV TPKLQL-----------DGLHLRFEFTTVPCDDPQFDNYVKICDQCVDGVGTRPKDDEPKPKSRSSSRPA--------TRGNSPAPRQQRPKK---EKKLKKQDEADKALTSDEE---R 383
MHV ELVK--KNSGGADEPTKDVYELQYSGAVRFDSTLPGFETIMKVLENELNAYKDGGA DVVSPKPQRKGRRAQEKKDEVDNSVAKPKSSVQRNVSR-ELTPEDRSLLAQILDDGV---V 444
HCov OC43 ELAKVQNLSGNPDEPQKDVEILRYNGAIRFDSTLSGFETIMKVLENELNAYQQQDGMMNMSPKPQRQRGH--KNQQGENDNISVAVPKSRVQQNKS R-ELTAEDISLLKKMDEPY----T 442
MERS-CoV KLTHQN---------NDDHGNPVYFLRYS GAIKLDPKNPNYNK WLELLEQNIDAYKT FPKKEKKQKAPKEES-------------TDQMSEPPKEQRVQGSITQRTTRPSVQPGPMIDVNTD---- 413
* : : : . :. : : :: : ::: . : . : . . .
SARS-CoV QNSMSGASADSTQA-------------- 422
IBV NNAQLEFYDEPKVINWGDAALGENEL 409
MHV PD---GLEDDS NV-------------- 454
HCov OC43 E---------DTSEI------------- 448
MERS-CoV -------------------------- 413

Fig. 2. Multiple sequence alignments of coronavirus N proteins. Shaded positions represent conserved residues among the compared sequences. Residues in red denote aromatic residues that are postulated to be involved in base stacking interactions when binding to RNA. Secondary structure elements based on SARS-CoV N protein are shown on top of the alignment, with arrows and cylinders representing β-strands and α-helices, respectively. The alignment was calculated on the ClustalOmega server (http://www.ebi.ac.uk/Tools/msa/clustalo).

same structural organization based on intrinsic disorder predictor profiles and secondary structure predictions (Fig. 2). Determination of the NTD and CTD structures of the N proteins from infectious bronchitis virus (IBV) (Fan et al., 2005; Jayaram et al., 2006), MHV (Grossoehme et al., 2009; Ma et al., 2010) and human coronavirus OC43 (Chen et al., 2013) were in general agreement with the structure-based model. The N protein sequence of the recently discovered Middle-East respiratory syndrome coronavirus (MERS-CoV) also shares the same intrinsic disorder and secondary structure profile, which further supports the universality of the structure-based model (van Boheemen et al., 2012).

Although the original three-domain model has been partially superseded by the structure-based model, some features of the earlier model may be reconciled with the latter one. First, the second spacer and the C-terminal acidic region in the three-domain model can be mapped to the C-terminal IDR in the structure-based model. Similar to the SARS-CoV N protein, the C-terminal acidic region of MHV N protein has been shown to self-interact (Hurst et al., 2005), and it has also been reported that a C-terminal IDR in the N protein of human coronavirus strain 229E is involved in oligomerization (Lo et al., 2013). Second, the RNA-binding domain in the three-domain model could be re-defined to span both the NTD and the CTD. In fact, Hurst et al. noticed that effective binding to RNA by MHV N protein in host cells required the presence of both the NTD and CTD (Hurst et al., 2009), suggesting that the NTD and CTD formed a single bipartite RNA interaction site, a feature to be reiterated in the final SARS-CoV RNP model. In this regard, the structure-based model is an evolution of the original three-domain model that provides a more refined framework for linking the structure and function of coronavirus N proteins.

Modular structures are found in many RNA-binding proteins, including other viral nucleocapsid proteins (Draper, 1999; Lunde et al., 2007). For example, the nucleocapsid protein from bunyamwera virus is a single-stranded RNA-binding protein with two modular domains (Li et al., 2013). Constructing a protein with a modular architecture confers many advantages which would not be possible with single-domain proteins. These include: (i) Enhanced binding specificity and affinity through cooperative coupled allosteric binding of individual domains. The modular organization of a protein also allows it to present a large and flexible surface for binding to complex structural features, or multiple and extended regions of the target molecules such as RNAs. (ii) Facilitated regulation and functional expression. The relatively weak interactions of individual domains make it easier to regulate the formation and disassembly of RNP complexes when needed. Assembly and disassembly can proceed via the (un)zipping action of one module at a time with moderate free energy cost. (iii) The multiple binding sites can evolve independently, and thus enhance environmental adaptation. The modular nature of SARS-CoV N protein and N proteins from Coronaviridae in general, is clearly essential for packaging RNP and viral function.

4. Structure of SARS-CoV N protein

4.1. Structure of the N-terminal domain

The structure of the NTD of SARS-CoV N was first determined by NMR by Huang et al. (2004b). The protein adopts a unique five-stranded antiparallel β-sheet with the topology of β4–β2–β3–β1–β5 (Fig. 3A). The middle strands β2 and β3 are connected by a protruding β-hairpin (β2′–β′). The residues in the extended β-hairpin are predominantly basic with 5 of the 15 residues being arginines or lysines. The 3D folding created a positively charged pocket at the junction between the hairpin and the core structure which served as the RNA binding site, as confirmed by NMR chemical shift perturbation upon addition of a 16-mer or 32-mer RNA (Fig. 3B). NMR relaxation and heteronuclear NOE data indicate that the β-

C.-k. Chang et al./Antiviral Research 103 (2014) 39–50

(A) CoV NTD  
(B) SARS-CoV  
(C) SARS-CoV  

(D) IBV  
MHV  
HCoV-OC43  

(E) CoV CTD  
(F) SARS-CoV  
(G) IBV  

Fig. 3. Structure comparisons of coronavirus N-proteins. (A) Structure comparison of various coronavirus NTDs (Grey: SARS-CoV, 2OFX; Magenta: IBV, 2GEC; Blue, MHV, 3HD4; Cyan: HCoV OC43. 4J3 K). The surface charge distributions on (B) (SARS-CoV) and (D) (IBV, MHV and HCoV OC43) are shown in same orientations. (C) Spatial arrangement of aromatic residues in NTD speculated to be involved in base stacking interaction when binding to RNA. Residues in the loop connecting β3 and β4 strands (a.a. Gly115-Gly130) have been removed for clarity. (E) Superimposition of the CTD structures of SARS CoV (gold, 2CJR) and IBV (cyan, 2GEC). The corresponding surface charge distributions were shown on (F) and (G) for SARS-CoV and IBV, respectively. All structures and surface charge distributions were generated using PyMOL.

hairpin is highly flexible, suggesting that this region may undergo conformational adaptation upon RNA binding ([Clarkson et al., 2009](https://doi.org/10.1074/jbc.M806886200)). The structural features of the NTD are reminiscent of the β-sheet RNA recognition proteins found in many RNPs ([Draper, 1999](https://doi.org/10.1016/S0022-2836(99)80004-9)). This class of proteins has a βαββαβ fold in which the middle first and third β-strands contain characteristic aromatic residues. In the crystal structure of U1A-RNA hairpin complex three bases are stacked against conserved aromatic residues while a flexible long β-hairpin grasp the RNA against the β-sheet ([Oubridge et al., 1994](https://doi.org/10.1038/367057a0)). These aromatic residues are thought to orient bases on the protein surface, rather than select particular protein-RNA sequences. In SARS-CoV NTD there are also many conserved aromatic residues in the same structural region. Although not confirmed, it is probable that some of these aromatic residues in SARS-CoV N, in particular Tyr87, Tyr110, Tyr112, Tyr113, Tyr122, and Trp133, are on the same face of β-sheet and are conserved in coronaviruses and may play similar roles in RNP packaging (Fig. 3C). The NTD structure of SARS-CoV N was later on determined by X-ray crystallography in two crystal forms ([Saikatendu et al., 2007](https://doi.org/10.1074/jbc.M702514200)). The overall folding of the crystal structure is similar to that observed in solution by NMR, with a root mean square deviation (RMSD) of 2.6 Å over 112 superimposed Cα atoms of the monoclinic form. Significant inward shift of loops L1 and L3 and outward hinge motion of the β-hairpin were observed, resulting in the RNA-binding cleft being significant narrower and shallower in the crystal structure. It is not clear whether the difference is due to the insufficient NOE constraints in the solution structure or due to crystal packing or both. Nonetheless, the difference observed in the two structures further supports the concept that the RNA-binding cleft is deformable and is likely to adopt a different conformation upon RNA binding. Intriguingly, in the cubic form the individual monomers organized as trimeric units and the consecutive trimers stack in a right-handed twist, resulting in an overall packing of a helical tubule. At present the physiological relevance of the helical packing is unclear.

### 4.2. Structure of the C-terminal structural domain

The C-terminal structural domain (CTD) of SARS CoV N exists in dimeric form ([Chang et al., 2005, 2006](https://doi.org/10.1074/jbc.M505355200); [Yu et al., 2005](https://doi.org/10.1074/jbc.M505355200)). The crystal structure of CTD was solved in two different constructs, CTD<sub>270–370</sub> ([Yu et al., 2006](https://doi.org/10.1074/jbc.M505355200)) and CTD<sub>248–365</sub> ([Chen et al., 2007](https://doi.org/10.1074/jbc.M702514200)). Alignment of 176 corresponding Cα atoms showed a RMSD of 0.511 Å, indicating that these two structures are practically identical. However, the absence of the N-terminal 22 amino acid peptide between residues 248 and 269 in CTD<sub>270–370</sub> significantly diminished the protein-protein interaction and crystal packing, as well as its interaction with nucleic acids, as described below. Each CTD monomer is composed of eight α-helices and a β-hairpin in the following topology: α1α2α3α4α5α6β1β2α7α8 (Fig. 3E). The dimer has the shape of a rectangular slab in which the four-stranded β-sheet forms one face and the α-helices form the opposite face. The two C termini are located at the diagonal apices on the β-sheet face and the two N termini are located at the center of two opposing edges of the slab. The dimerization interface of the CTD dimer is composed of four β-strands and six α-helices with each protomer contributing one β-hairpin and helices α5, α6 and α7. The long β2 strand of one protomer pairs with the β2 strand of the other protomer to form the

four-stranded intermolecular antiparallel β-sheet that is stabilized through extensive hydrogen bonding (Chang et al., 2005). Each hairpin also interacts extensively with the hairpin from the other protomer in a domain-swapped manner. The other part of the dimerization interface is composed of helices α5 and α6, where strong hydrophobic interactions involving Trp302, Ile305, Pro310, Phe315 and Phe316 were observed. The dimer is further stabilized by hydrophobic interactions between the longest helix, α7, and the intermolecular β-sheet. The combination of hydrogen bonds and hydrophobic interactions results in a very stable dimer with a buried surface area of ~5280 Å², suggesting that the dimer is likely the native structure of coronavirus N protein. Takeda et al. have solved the solution structure of CTD₂₄₈⁻³⁶₅ and showed that the NMR structure is almost identical to the crystal structure. The backbone RMSD between the protomers of the mean NMR structure and the crystal structure of the CTD spanning residues 248–365 is 1.45 Å if residues 260–319 and 333–358 are superimposed. However, in the NMR structure the two N-termini (residues 248–265) protruding from the dimer core are disordered and lack a short helix formed by residues 259–263, whereas, in the crystal structure, they are involved in a number of intra-monomer and intra-dimer contacts and are more rigid.

Interestingly, Chang et al. also observed the formation of an octamer in the asymmetric unit of the CTD crystal. Translational stacking of the octamer forms a hollow twin helix structure with an outer diameter of ~90 Å and an inner diameter of ~45 Å, with a pitch of ~140 Å. The groove of the twin helix, which is lined with several positively charged residues, has a depth of ~22.5 Å. The N-terminal 22 amino acid residues from a.a. 248–269 play an important role in protein-protein interaction in the octamer, accounting for the absence of the octamer in the crystal structure of CTD₂₇₀⁻³₇₀. Studies of the NMR chemical shift perturbations caused by the binding of single-stranded DNA and mutational analyses have identified this mostly disordered region at the N-termini as the prime site for nucleic acid binding (Takeda et al., 2008). In addition, residues in the β-sheet region also showed significant perturbations. Mapping of the locations of these residues onto the helical model observed in the crystal structure of CTD₂₄₈⁻³⁶₅ revealed that these two regions are parts of the interior lining of the positively charged helical groove (Fig. 3F). This observation led them to propose a helical packaging model of SARS-CoV RNP, as will be elaborated in more detail in the following sections.

Due to difficulties arising from protein stability and dynamic behavior, there are no structures available for any of the full-length N proteins from coronaviruses. Fitting of the small angle X-ray scattering (SAXS) data led Chang et al. to propose a structural model for a di-domain (DD) construct spanning the NTD, LKR, and CTD of SARS-CoV N protein (a.a. 45–265) (Chang et al., 2009) (Fig. 1C). The DD dimer adopts a clamp-like open conformation in the model with LKRs serve as the two arms connecting the two NTDs to the CTD dimer. The model is consistent with the known structural features of coronavirus N proteins, namely the dimerization of the CTD and intrinsically disordered nature of the LKR, and currently remains the only structural model spanning multiple domains of coronavirus N proteins.

### 4.3. Comparison with N proteins of other coronaviruses

Comparison of SARS-CoV N protein structure with those of other viral N proteins provides valuable mechanistic and evolutionary insights. The NTD from avian infectious bronchitis virus (IBV) (Fan et al., 2005) (Jayaram et al., 2006), mouse hepatitis virus (MHV) (Grossoehme et al., 2009), and human coronavirus OC43 (HCoV-OC43) (Chen et al., 2013), as well as the CTD of IBV (Jayaram et al., 2006) have been reported. The sequence identities of SARS-CoV NTD with those of IBV and HCoV-OC43 are 34% and 47%, respectively, yet the 3D structures of these three proteins are highly homologous. The RMSD between SARS-CoV N-NTD (PDB ID: 2OFZ) and IBV-NTD (PDB ID: 2GEC) is 0.665 Å for 69 aligned Ca atoms and that between SARS-CoV NTD and HCoV-NTD is 0.838 Å for 86 aligned Ca atoms of the two proteins (Chen et al., 2013). Interestingly, the surface charge distribution of the NTD of IBV, MHV and HCoV-OC43 are significantly different from that of the SARS-CoV NTD (Fig. 3D), suggesting that they may interact with RNA differently. The structure of IBV-CTD (a.a. 219–349, PDB ID: 2GE7) is also highly homologous to that of SARS-CoV CTD (PDB ID: 2CJR). The RMSD of 182 aligned Ca atoms in a dimer between the two structures is 1.563 Å and both exist as a domain-swapped dimer. Three types of interactions (S-, L- and F-types) were observed in three forms of IBV CTD crystals. Intriguingly, type S interaction observed in crystal form 1 and 2 (Fig. 4A and B in Jayaram et al. (2006)) bears high resemblance to that observed in the helical packing of SARS-CoV CTD (Fig. 4A). Furthermore, the surface charge distribution of IBV CTD dimer also contains a positively charged strip spanning the region observed for SARS-CoV CTD dimer (Fig. 3F and G) implying similar interaction between CTD and RNA for the two coronaviruses.

#### 4.4. The CTD dimer interface suggests possible evolutionary link between corona- and arteriviruses

Sequence alignment coupled with secondary structure prediction show that many coronavirus CTDs share the ββα topology observed in SARS-CoV (Chang et al., 2005). These results raise the possibility that all coronaviruses employ the same interface mechanism for dimerization and they belong to the same structural class. The structural arrangement of CTD is also reminiscent of the dimer-interface of the nucleocapsid protein from porcine reproductive and respiratory syndrome virus (PRRSV), an arterivirus (Chang et al., 2005). Thus, there are common principles that underlie the architecture of the nucleocapsid protein in both SARS-CoV and PRRSV. The structural similarity between the N proteins of SARS-CoV and PRRSV provides valuable information for understanding the evolutionary links between corona- and arteriviruses, suggesting a possible common origin of these two proteins (Yu et al., 2006).

### 5. Biophysical aspects of SARS-CoV N protein self-association

#### 5.1. The CTD is a transient self-association site of the SARS-CoV N protein

Reports in the literature suggested that N can oligomerize through the SR-rich region or the C-terminal fragments in a concentration dependent manner (Surjit and Lal, 2008). However, these early studies were carried out using fragments that often cut through the structured region that could have adverse effect on their structures and oligomerization behavior as well. Crystal structures of coronavirus N protein led to several proposed N-polymers that could bind RNA and mimic the RNP packaging (Chen et al., 2007; Fan et al., 2005; Jayaram et al., 2006; Saikatendu et al., 2007). It is unclear whether the oligomer structure is biologically relevant, since there have been no reports of oligomer species being detected in solution. To test the possibility that the oligomer structure reflects the existence of transient interactions that have been trapped during the crystallization process, Chang et al. applied an in vitro disulfide trapping technique in an attempt to capture these transient interactions in solution (Chang et al., 2013). Specifically, using the crystal structures as guides they engineered single-site cysteine mutations at various locations and tested the ability of these mutants to spontaneously form disulfide

Fig. 4. A proposed model of the SARS-CoV ribonucleocapsid protein. The crystal packing of a 24-mer CTD domain is shown in side view (A) and top view (B). The surface charge distribution of the SARS-CoV CTD 24-mer. (C) Top view of the model shows the docking of two RNA chains (orange and yellow ribbons) onto the 24-mer CTD structure. The CTD 24-mer is shown in surface charge representation. The RNA chains were modeled with the phosphate backbone (red spheres) facing inside the groove and bases (yellow rings) pointing outward. (D) Top view of the putative CTD–RNA complex. (E) Schematic of the docking of NTD onto the CTD 24–mer-RNA complex. The NTD domains are represented by ellipsoids.

linkages through size-exclusion chromatography. SARS-CoV N contains no cysteine and none of the mutants are located close enough to form intra-dimer disulfide linkages, thus any disulfide linkage must be due to inter-dimer protein disulfide bond formation. The results suggested that fragments containing the CTD of SARS-CoV N protein are capable of transient self-association through the oligomer interface identified in the crystal structure, even though the long-lived stable helical structure of CTD was not observed in solution. Thus, the CTD dimer-dimer interaction observed in the crystal is also the preferred interaction in solution but the oligomer can form only transiently due to weak interaction as shown in the small interface area between CTD dimers (~1000 Å²). Presumably these weak interactions can be augmented by RNA binding and binding of the other N protein domains linked to the CTD through LKR in a synergistic manner and the conformation of the CTD oligomer will be further modified by N–RNA interactions. A similar strategy was applied to engineer NTD mutants. However, no significant oligomer formation was observed for the NTD fragments, suggesting that the NTD either does not form oligomers or forms oligomers through an unidentified intermolecular interface other than that identified in the NTD crystal structure.

5.2. Electrostatic screening and phosphorylation-mimicking mutation affect SARS-CoV N protein self-association

SARS-CoV N is a highly basic protein containing an excess of 25 positive charges. These charges are considered important for RNA binding, but they are also potentially deterring for the self-association of the protein through electrostatic repulsion (Huang et al., 2004b; Takeda et al., 2008). Chang et al. tested whether salt concentration affects SARS-CoV N transient self-association by disulfide trapping experiment as described above using the Q290C mutant. Gln290 is located at the interface between two dimers and the two Gln290 in the dimer are far apart, formation of disulfide bonds in the Q290C mutant would require at least two dimers to draw close together in space, resulting in the formation of tetramers or higher oligomers. The relative amount of tetramer and larger oligomers in solution increases with increasing salt concentration, suggesting that reducing charge repulsion by increasing salt concentration enhances self-association of CTD.

The N protein is heavily phosphorylated at the Ser/Arg-rich portion of the LKR region (Peng et al., 2008; Surjit et al., 2005; Zakharouchkou et al., 2005) and phosphorylation may affect nucleocytoplasmic shuttling of the N protein (Surjit et al., 2005; Wu et al., 2009). Peng et al. demonstrated that phosphorylation of the LKR by the SR protein kinase-1 (SRPK1) partially impaired the self-association of the full-length protein (Peng et al., 2008). Chang et al. examined whether changing the electrostatic properties of the protein itself could affect transient self-association (Chang et al., 2013). They chose the putative phosphorylation sites on the flexible linker as prime target, and assayed the effect of negative charges on N protein self-association by changing these sites from Ser to Glu in the Q290C mutant of di-domain constructs containing the NTD, LKR and CTD (DD<sub>Q290C</sub>, a.a. 45–365). They observed that gradual introduction of negative charges on the unstructured linker had a positive effect on the oligomerization of the DD when compared to the DD<sub>Q290C</sub> control, with maximum effect achieved when 3 negative charges were introduced per each
chain. Further increases in negative charges were less effective in enhancing DD<sub>Q290C</sub> oligomerization. Overall, the results suggest that hyperphosphorylation of the LKR, which reduces the total positive charge of the N protein, can enhance and regulate oligomerization of DD through electrostatic effects. The results suggest a biophysical mechanism where electrostatic repulsion may act as a switch to regulate N protein oligomerization.

### 6. Protein–nucleic acid interaction

#### 6.1. SARS-CoV N protein binds to nucleic acids at multiple sites

The primary function of the coronavirus N protein is to package the viral genome into a ribonucleoprotein (RNP) particle to protect the genomic RNA and for incorporation into a viable virion. Thus, N must bind to RNA tightly. During viral infection the N protein must also be readily dissociated to expose the genomic RNA for efficient expression, transcription and replication ([Lai and Cavanagh, 1997](https://doi.org/10.1016/S0042-6822(97)00035-8); [Tahara et al., 1994, 1998](https://doi.org/10.1016/0042-6822(94)90003-5)). This function demands a low energy barrier for N to dissociate from RNA. Viruses have evolved a clever tactic to achieve these two seemingly contradictory functions. The secret lies on the modular structural organization and the dynamic nature of the N protein rendered by the intrinsically disordered regions. Much information about the interaction between the N protein and RNA in coronaviruses have been gathered through studies on MHV model systems, including detection of general binding activity ([Robbins et al., 1986](https://doi.org/10.1016/0042-6822(86)90267-6)) and identification of RNA sequences that bind with high affinity to the protein ([Nelson et al., 2000](https://doi.org/10.1016/S0042-6822(00)00112-5)). However, it was the discovery of SARS-CoV that spurred research on the mechanisms behind the interaction between coronavirus N protein and nucleic acids. Studies on the nucleic acid-binding behavior of SARS-CoV N protein at the domain level have started to provide much needed insight into the binding mechanism of coronavirus N proteins. SARS-CoV N protein is a highly basic protein with excess positively charged residues mostly localized in three regions: the SR-rich region of the LKR (residues 176–204, +6 charges), the N-terminal region of the CTD (residues 248–267, +7 charges) and the C-terminal disordered region (residues 370–389, +7 charges). The nucleic acid-binding activity of the NTD was tested and confirmed early on [Huang et al. (2004a)](https://doi.org/10.1073/pnas.0404640101) due to the presence of the classic RNA-binding motif first detected in the U1-RNP ([Nagai et al., 1995](https://doi.org/10.1016/0022-2836(95)90269-8)). The effect of the other structural domain, the CTD, on nucleic acid binding was not expected since initial structures of the domain did not include the residues that interacted with nucleic acids ([Yu et al., 2006](https://doi.org/10.1074/jbc.M511822200)). Structures of longer constructs of CTD later revealed a positively charged groove on the surface of the molecule that could act as a binding site for nucleic acids ([Chen et al., 2007](https://doi.org/10.1074/jbc.M702458200)), and follow-up studies demonstrated that the CTD was capable of binding to both ssDNA and ssRNA with similar affinity as the NTD (*K*<sub>D</sub> ~ 10 μM) ([Chang et al., 2009](https://doi.org/10.1074/jbc.M808158200); [Takeda et al., 2008](https://doi.org/10.1074/jbc.M802887200)). The IDRs, on the other hand, have not been studied individually due to stability issues ([Mark et al., 2008](https://doi.org/10.1074/jbc.M802887200)). However, inclusion of the IDRs to any of the structural domains resulted in significantly increased binding affinity and binding cooperativity towards a poly (U) ssRNA under *in vitro* conditions (*K*<sub>D</sub> ~ 0.8 μM), suggesting that the IDRs are able to modulate the nucleic acid binding activity of SARS-CoV N protein ([Chang et al., 2009](https://doi.org/10.1074/jbc.M808158200)). Of particular interest is the role of the LKR in N protein–nucleic acid interaction since it contains a SR-rich motif where most of the putative phosphorylation sites are located. It has been reported that SARS-CoV N protein is hypophosphorylated within the virion ([Wu et al., 2009](https://doi.org/10.1074/jbc.M808158200)), and deletion of the SR-rich motif within the LKR resulted in formation of larger than normal RNPs that were sensitive to RNase treatment ([Peng et al., 2008](https://doi.org/10.1074/jbc.M808158200)). These observations suggest that phosphorylation of the SARS-CoV N protein at the LKR not only affects N

oligomerization, it may decrease the nucleic acid binding affinity as well.

By itself, the SARS-CoV N protein is a non-specific nucleic acid-binding protein. It has been shown to bind to single-stranded RNA (ssRNA), single-stranded DNA (ssDNA) and double-stranded DNA (dsDNA) under *in vitro* conditions ([Takeda et al., 2008](https://doi.org/10.1074/jbc.M802887200); [Tang et al., 2005](https://doi.org/10.1074/jbc.M802887200)). The non-specific nature of this binding is also expected since encapsidation of the entire viral genome would require the N protein to bind to diverse sequences with reasonable affinity. Although one could argue that the N protein may bind to a particular sequence with high affinity and package the rest of the RNA by relying on protein–protein interaction alone, such a scenario is unlikely to happen because the interaction between N protein dimers is extremely weak in the absence of nucleic acids ([Chang et al., 2006, 2013](https://doi.org/10.1074/jbc.M808158200)). Moreover, the highly charged regions are exposed to the solvent and the electrostatic forces might be the main driving force behind protein–nucleic acid binding. Indeed, nucleic acid-binding sites on the NTD and CTD identified from NMR studies were found to have strong positive surface charges ([Huang et al., 2004b; Takeda et al., 2008](https://doi.org/10.1074/jbc.M802887200)). Takeda et al. also found that mutating Lys257 and Lys258 to Gln in the CTD resulted in decreased binding affinity towards ssDNA, whereas mutating the same residues to Arg had no effect on the binding strength ([Takeda et al., 2008](https://doi.org/10.1074/jbc.M802887200)). These lines of evidence strongly indicate that SARS-CoV N protein binds to nucleic acids in a non-specific manner through electrostatic interactions.

#### 6.2. Intrinsic disorder and coupled-allosteric binding of N to nucleic acids

The discovery that multiple regions within the SARS-CoV N protein are capable of interacting with nucleic acids provides critical insights into the binding mechanism. Although the binding strength of individual binding sites towards nucleic acids is only in the micromolar range, the concerted action of these sites confers higher nucleic acid-binding affinity to the N protein as a whole. The IDRs play a special role, since their inclusion not only increases the binding affinity, but also enhances the binding allostery, enabling the N protein to bind RNA with high cooperativity ([Chang et al., 2009](https://doi.org/10.1074/jbc.M808158200)). A variety of functions were found to be associated with domains containing conserved disorder with DNA/RNA binding among the most common function ([Dunker et al., 2002](https://doi.org/10.1074/jbc.M808158200); [Dyson, 2012](https://doi.org/10.1074/jbc.M808158200)). The intrinsic disorder in protein confers several advantages in performing its biological functions, including promiscuous basal activity, enhanced specificity, higher capture radius for formation of complexes, facilitating regulation by post-translational modification ([Dyson, 2011](https://doi.org/10.1074/jbc.M808158200)). Two possible causes for the binding enhancement can be argued. First, the extended conformation of the N protein due to the presence of the IDRs increases the collision radius with nucleic acids. If the binding were further coupled to changes in protein conformation, the rate of binding would be enhanced through the “fly-casting mechanism” proposed by [Shoemaker et al. (2000)](https://doi.org/10.1074/jbc.M808158200). Second, the flexibility of the IDRs allows the optimal alignment of the multiple nucleic acid-binding sites to interact with the same nucleic acid molecule in an allosteric fashion, resulting in a “coupled allostery” effect that enhances the binding affinity of the protein towards the nucleic acid ([Hilser and Thompson, 2007](https://doi.org/10.1074/jbc.M808158200)). In addition to the IDRs, the structural domains, NTD and CTD, also could act in conjunction to enhance the binding affinity. The NTD contains a number of aromatic residues conserved among coronavirus N proteins that may interact with nucleotide bases by forming stacking interactions, whereas the strong electropositive surface formed by the CTD dimer is perfectly suited for interacting with the phosphate backbone ([Chen et al., 2007](https://doi.org/10.1074/jbc.M808158200)). Consistent with this model, mutagenesis studies conducted by Grossoehme and coworkers have found that Tyr127 onthe NTD of MHV N protein was important for binding to a transcriptional regulatory sequence RNA (Grossoehme et al., 2009).

# 7. Packaging of the SARS-CoV ribonucleocapsid

## 7.1. A putative model

Accommodation of the exceptionally large (~29 kb) SARS-CoV genome into newly formed virion spherules <100 nm in size necessitates an extremely well-packed, largely helical, supercoiling of the nucleic acid within the RNP core. The inability to observe a well-structured RNP layer inside the SARS-CoV particle and only short coiled fragments of RNP in MHV in the cryo-EM reconstructions strongly suggests that the helical nucleocapsid is a very flexible structure that extensively twists and folds upon itself (Bárcena et al., 2009). Such a dynamic structure of coronavirus RNP is not totally unexpected since RNA is known to be dynamic and exists in multiple folded and unfolded states (Dyson, 2012). Thus, RNA–protein recognition often involves an induced-fit process, in contrast to protein–B DNA interaction which most often manifests itself as molding of the protein onto the B-form DNA structure. Furthermore, the modular organization with three long IDRs of the N protein provides the N protein with considerable flexibility. No existing data supports the presence of a long-lived SARS-CoV N oligomer or intermediate in solution and the SARS-CoV genomic ssRNA by itself is unlikely to exist as a helix of the length observed in cryo-EM. Thus, packaging of SARS-CoV RNP proceeds most likely through a RNA binding-coupled packaging mechanism, as also proposed for MHV, which showed that the product RNA of mouse hepatitis virus synthesized was mostly of genome length and was found to be encapsidated by N protein (Compton et al., 1987). This suggests that coronavirus RNA synthesis is coupled to the encapsidation of nascent RNA, analogous to the replication of viruses with helical negative-strand RNA nucleocapsids. Based on available detailed 3D structural information of the SARS-CoV N protein modules and our understanding of N-RNA interaction we propose a probable model derived from the crystal structure of the CTD (Chen et al., 2007), which was shown to exist transiently in solution by disulfide trap experiment (Chang et al., 2013) (Fig. 4A and B). A putative scenario of the molecular events leading to the formation of RNP is as follows:

(1) **Initiation**: In solution initial binding of RNA at either NTD or CTD facilitates binding of other modules to RNA in a coupled-allostery manner with RNA molecule threads between the two structural domains. This initial N-RNA binary complex (RNP⁰) is highly stable and each RNA molecule may have several N protein bound at a particular time.

(2) **Growth**: The RNP⁰ could grow by either recruiting more N to the adjacent RNA sites, or it could slide or hop along the linear RNA molecule and combine with other smaller N-RNA oligomers to form a larger oligomer (RNPⁿ) of various sizes. The N proteins in RNPⁿ would pack in a structure with CTD forming the helical core, similar to that observed in the CTD crystal structure, and RNA wraps and twists around the helical groove through mostly electrostatic interaction between the positively charge residues in the groove and the phosphate backbone of the RNA molecule (Fig. 4C and D).

(3) **Packaging of NTD**: The NTD module will cap on the outside of the helical CTD–RNA complex with the charged surface in the junction between the β-sheet and the β-hairpin covering the free phosphate groups of the RNA molecule. Furthermore, RNA bases sticking out of the groove could intercalate in between the aromatic rings on the NTD core at the bottom of the β-sheet (Figs. 3C, and 4C). The presence of the long

disordered LKR permits the two structural domains considerable freedom to adapt a wide range of orientations and positions for optimal packing of the RNP complex. Likewise, the RNA molecule also possesses high freedom to adjust to local conformation by an induced-fit process. Thus, the N protein binds to RNA in a fashion resembling that of an octopus clinching onto its prey (RNA) using all its tentacles (modules) (Fig. 4E and F).

(4) **Thermodynamic basis**: Electrostatic interaction drives the formation of N-RNA complex but the multitude of weak protein–protein interactions contributes towards the self-assembly of the helical RNP. This is consistent with the concept for virus assembly that capsid proteins associate through locally weak interactions to form globally stable structures (Zlotnick, 2003).

The RNP structure proposed above would have an outer diameter of ~16 nm and an inner diameter of ~4 nm, consistent with that observed by cryo-EM. Each N dimer would bind to 7 RNA bases. The two termini would stick out of the helix and the LKR linker would be accessible to interact with the matrix protein M. The combination of a modular structure incorporating IDRs, multiple sites of moderate RNA binding affinity, and weak dimer–dimer interaction in the N protein not only allows the packaging of a stable RNP but also offers an energetically favorable condition for the expression of the viral genomic information. One can envision an unzipping mechanism for unwinding of the viral RNA molecule and dissociation of the RNA molecule from the N protein in a step-wise manner, one module at a time, without the need to overcome a high-energy barrier of dissociating a whole N protein at once. The weak interactions between N protein dimers also minimize formation of kinetic traps and allow a greater degree of regulation of RNP assembly.

## 7.2. Comparison with other viral RNP structures

At present the structures of several helical viral RNP of RNA viruses have been reported. These include rabies virus (RV) (Albertini et al., 2006), vesicular stomatitis virus (VSV) (Green et al., 2006), respiratory syncytial virus (RSV) (Tawar et al., 2009), Lassa virus (Hastie et al., 2012; Hastie et al., 2011), Rift Valley fever virus (RVFV) (Raymond et al., 2012), Bunyamwera virus (BUNV) (Ariza et al., 2013; Li et al., 2013) and Leanyer orthobunyavirus (LEAV) (Niu et al., 2013). The N proteins of these RNA viruses all possess the modular organization similar to that of the SARS-CoV N protein, namely they all consist of an N-terminal arm, two domains which are connected by a flexible hinge, and a flexible C-terminal tail. With the exception of Lassa virus, the RNA binds to the positively charged crevice between the N- and C-terminal domains that shield RNA from the environment. Thus, RNA sequestering by nucleoproteins is likely a common mechanism used by RNA viruses to protect their genomes from host defense mechanism. It also suggests that conformational change in the RNA packing is required during expression and translation. The number of RNA bases bound per N protein varies from 4 in RVFV to 11 in LEAV.

# 8. Future perspectives

Over the past 10 years considerable insights regarding the structure and function of the SARS-CoV N protein have been revealed. It is remarkable that the coronavirus N protein family shares a common modular structure organization incorporating functionally important IDRs even when they share only moderate sequence identity. New biophysical information, together with recent studies employing classical genetics and biochemical meth-
ods, have started to provide a clearer picture of how the N protein forms the RNP and what factors affect the process. However, the quest for understanding how the SARS-CoV N protein (and coronavirus N proteins in general) carries out its roles during the viral life cycle is still far from over. A critical piece of missing information lies in the atomic structure of the RNP complex, whose elucidation has been hampered by the low solubility of the complex and labile nature of the full-length N protein. It will be probably not enough to only obtain the structure of the SARS-CoV RNP, but the determination of a number of coronavirus RNPs will be necessary to ascertain whether they share a common structural code.

Another topic is the role of the N protein in the viral replication-transcription complex (RTC), which is composed of various coronavirus nonstructural proteins (Nsp’s). In MHV, the N protein has been shown to dynamically associate with the RTC ([Verheije et al., 2010](https://doi.org/10.1016/j.virol.2010.03.045)). Keane and Giedroc recently found that MHV N bound to Nsp3 with high affinity through the NTD and LKR ([Keane and Giedroc, 2013](https://doi.org/10.1074/jbc.M113.498078)). Co-localization of the N protein with the RTC has also been observed in cells infected with SARS-CoV ([Stertz et al., 2007](https://doi.org/10.1016/j.jmr.2007.01.003)), although whether there is direct physical interaction between the two remains to be seen. One problem in this field is the lack of knowledge on the functions of the individual Nsp’s, making it extremely difficult to interpret the biological relevance of N-Nsp interactions. The N protein might also associate with the RNA-dependent RNA polymerase (RdRp) in coronaviruses ([van der Meer et al., 1999](https://doi.org/10.1016/S0042-6822(99)00224-8)), but the interaction is poorly defined and more effort will be required to verify the association and clarify its role.

The SARS-CoV N protein has been reported to interact with numerous host cell proteins, such as the B23 phosphoprotein ([Zeng et al., 2008](https://doi.org/10.1016/j.virol.2008.04.020)), Smad3 ([Zhao et al., 2008](https://doi.org/10.1016/j.virol.2008.04.020)), the chemokine Cxcl16 ([Zhang et al., 2010](https://doi.org/10.1016/j.virol.2010.03.045)), translation elongation factor-1 alpha ([Zhou et al., 2008](https://doi.org/10.1016/j.virol.2008.04.020)), pyruvate kinase ([Wei et al., 2012](https://doi.org/10.1016/j.virol.2012.03.005)), and 14-3-3 ([Surjit et al., 2005](https://doi.org/10.1016/j.virol.2005.03.015)). Unfortunately, there have been few follow-up studies that independently verify these interactions, and the large variance in experimental conditions used to initially identify these interactions makes it extremely difficult to obtain a coherent picture of the SARS-CoV N protein interactome in the host cell. On the other hand, a recent IBV study employing high-throughput mass spectrometry yielded a list of cellular proteins that may potentially bind to the N protein ([Emmott et al., 2013](https://doi.org/10.1016/j.virol.2013.03.015)), and the same strategy could be applied to SARS-CoV and other coronaviruses (especially MERS-CoV) for interactome mapping. Comparisons between different coronavirus N protein interactomes should provide valuable information on host specificity and evolution of the interactions between N and host cell proteins, and may offer insight into the development of antiviral agents against coronaviruses that target interactions between host cell proteins and the N protein.

The SARS-CoV N protein has been widely used as a diagnostic target of SARS ([Surjit and Lal, 2008](https://doi.org/10.1016/j.virol.2008.04.020)). Viral N protein shows least variation in the gene sequence, therefore indicating it to be a genetically stable protein, which is a primary requirement for an efficient drug target candidate. Given its pathogenic effect inside the cell, it is not surprising that the N protein has also become a therapeutic target in antiviral therapy. Disruption of RNP formation through inhibition of either protein oligomerization or nucleic acid binding activity of nucleoproteins has been effective in the inhibition of other viruses under a laboratory setting. For example, nucleozin and its analogues were shown to inhibit influenza virus by targeting its nucleocapsid protein ([Hung et al., 2012](https://doi.org/10.1016/j.virol.2012.03.005); [Kao et al., 2010](https://doi.org/10.1016/j.virol.2010.03.045)), and compounds targeting the interaction between N protein and nucleic acids have been developed against HIV-1 ([Musah, 2004](https://doi.org/10.1016/j.virol.2004.03.005)). Recently, Lo et al. discovered an antiviral peptide that interfered with the CTD oligomerization of the HCoV-229E N protein and inhibited HCoV production ([Lo et al., 2013](https://doi.org/10.1016/j.virol.2013.03.005)). Extending these studies to SARS-CoV and other novel human coronaviruses, e.g.
MERS-CoV, could pave the way towards the discovery of new therapeutics that target the N protein.

## Acknowledgement

This work was supported by Grants NSC 101-2311-B-001-025 from The National Science Council and NHRI-EX99-9933B1 from the National Health Research Institute of the Republic of China. The NMR experiments were carried out with NMR spectrometers of the High-Field Nuclear Magnetic Resonance Center (HFNMRC) supported by Core Facility for Protein Structural Analysis, National Core facility Program for Biotechnology, The National Science Council of the Republic of China. Crystal structure was determined at the Taiwan beamline BL12B2 (in SPring-8) of the National Synchrotron Radiation Center (NSRRC).

## References

- Ababou, A., Ladbury, J.E., 2007. Survey of the year 2005: literature on applications of isothermal titration calorimetry. J. Mol. Recognit. 20, 4–14.
- Albertini, A.A.V., Wernimont, A.K., Muziol, T., Ravelli, R.B.G., Clapier, C.R., Schoehn, G., Weissenhorn, W., Ruigrok, R.W.H., 2006. Crystal structure of the rabies virus nucleoprotein-RNA complex. Science 313, 360–363.
- Ariza, A., Tanner, S.J., Walter, C.T., Dent, K.C., Shepherd, D.A., Wu, W.N., Matthews, S.V., Hiscox, J.A., Green, T.J., Luo, M., Elliott, R.M., Fooks, A.R., Ashcroft, A.E., Stonehouse, N.J., Ranson, N.A., Barr, J.N., Edwards, T.A., 2013. Nucleocapsid protein structures from orthobunyaviruses reveal insight into ribonucleoprotein architecture and RNA polymerization. In: Nucleic Acids Res. 41, 5912–5926.
- Bárcena, M., Oostergetel, G.T., Bartelink, W., Faas, F.G.A., Verkleij, A., Rottier, P.J.M., Koster, A.J., Bosch, B.J., 2009. Cryo-electron tomography of mouse hepatitis virus: insights into the structure of the coronavirion. Proc. Nat. Acad. Sci. 106, 582–587.
- Caul, E.O., Egglestone, S.I., 1979. Coronavirus-like particles present in simian faeces. Vet. Rec. 104, 168–169.
- Chang, C.-K., Sue, S.-C., Yu, T.-H., Hsieh, C.-M., Tsai, C.-K., Chiang, Y.-C., Lee, S.-J., Hsiao, H.-H., Wu, W.-J., Chang, W.-L., Lin, C.-H., Huang, T.-H., 2006. Modular organization of SARS coronavirus nucleocapsid protein. J. Biomed. Sci. 13, 59–72.
- Chang, C.K., Chen, C.M., Chiang, M.H., Hsu, Y.L., Huang, T.H., 2013. Transient oligomerization of the SARS-CoV N protein – implication for virus ribonucleoprotein packaging. PLoS ONE 8, e65045.
- Chang, C.K., Hsu, Y.L., Chang, Y.H., Chao, F.A., Wu, M.C., Huang, Y.S., Hu, C.K., Huang, T.H., 2009. Multiple nucleic acid binding sites and intrinsic disorder of severe acute respiratory syndrome coronavirus nucleocapsid protein: implications for ribonucleocapsid protein packaging. J. Virol. 83, 2255–2264.
- Chang, C.K., Sue, S.C., Yu, T.H., Hsieh, C.M., Tsai, C.K., Chiang, Y.C., Lee, S.J., Hsiao, H.H., Wu, W.J., Chang, C.F., Huang, T.H., 2005. The dimer interface of the SARS coronavirus nucleocapsid protein adapts a porcine respiratory and reproductive syndrome virus-like structure. FEBS Lett. 579, 5663–5668.
- Chen, C.Y., Chang, C.K., Chang, Y.W., Sue, S.C., Bai, H.I., Riang, L., Hsiao, C.D., Huang, T.H., 2007. Structure of the SARS coronavirus nucleocapsid protein RNA-binding dimerization domain suggests a mechanism for helical packaging of viral RNA. J. Mol. Biol. 368, 1075–1086.
- Chen, I.J., Yuann, J.M., Chang, Y.M., Lin, S.Y., Zhao, J., Perlman, S., Shen, Y.Y., Huang, T.H., Hou, M.H., 2013. Crystal structure-based exploration of the important role of Arg106 in the RNA-binding domain of human coronavirus OC43 nucleocapsid protein. Biochim. Biophys. Acta 1834, 1054–1062.
- Clarkson, M.W., Lei, M., Eisenmesser, E.Z., Labeikovsky, W., Redfield, A., Kern, D., 2009. Mesodynamics in the SARS nucleocapsid measured by NMR field cycling. J. Biomol. NMR 45, 217–225.
- Compton, S.R., Rogers, D.B., Holmes, K.V., Fertsch, D., Remenick, J., McGowan, J.J., 1987. In vitro replication of mouse hepatitis virus strain A59. J. Virol. 61, 1814–1820.
- Davies, H.A., Dourmashkin, R.R., Macnaughton, M.R., 1981. Ribonucleoprotein of avian infectious bronchitis virus. J. Gen. Virol. 53, 67–74.
- de Haan, C.A., Vennema, H., Rottier, P.J., 2000. Assembly of the coronavirus envelope: homotypic interactions between the M proteins. J. Virol. 74, 4967–4978.
- Draper, D.E., 1999. Themes in RNA-protein recognition. J. Mol. Biol. 293, 255–270.
- Dunker, A.K., Brown, C.J., Lawson, J.D., Jakoucheva, L.M., Obradovic, Z., 2002. Intrinsic disorder and protein function. Biochemistry (Moscow) 41, 6573–6582.
- Dunker, A.K., Lawson, J.D., Brown, C.J., Williams, R.M., Romero, P., Oh, J.S., Oldfield, C.J., Campen, A.M., Ratliff, C.M., Hipps, K.W., Ausio, J., Nissen, M.S., Reeves, R., Kang, C., Kissinger, C.R., Bailey, R.W., Griswold, M.D., Chiu, W., Garner, E.C., Obradovic, Z., 2001. Intrinsically disordered protein. J. Mol. Graph. Model. 19, 26–59.
- Dyson, H.J., 2011. Expanding the proteome: disordered and alternatively folded proteins. Q. Rev. Biophys. 44, 467–518.
- Dyson, H.J., 2012. Roles of intrinsic disorder in protein-nucleic acid interactions. Mol. BioSyst. 8, 97–104.

Dyson, H.J., Wright, P.E., 2005. Intrinsically unstructured proteins and their functions. Nat. Rev. Mol. Cell Biol. 6, 197–208.

Emmott, E., Munday, D., Bickerton, E., Britton, P., Rodgers, M.A., Whitehouse, A., Zhou, E.M., Hiscox, J.A., 2013. The cellular interactome of the coronavirus infectious bronchitis virus nucleocapsid protein and functional implications for virus biology. J. Virol. 87, 9486–9500.

Escors, D., Ortego, J., Laude, H., Enjuanes, L., 2001. The membrane M protein carboxy-terminus binds to transmissible gastroenteritis coronavirus core and contributes to core stability. J. Virol. 75, 1312–1324.

Fan, H., Ooi, A., Tan, Y.W., Wang, S., Fang, S., Liu, D.X., Lescar, J., 2005. The nucleocapsid protein of coronavirus infectious bronchitis virus: crystal structure of its N-terminal domain and multimerization properties. Structure 13, 1859–1868.

Green, T.J., Zhang, X., Wertz, G.W., Luo, M., 2006. Structure of the vesicular stomatitis virus nucleoprotein-RNA complex. Science 313, 357–360.

Grossoehme, N.E., Li, L., Keane, S.C., Liu, P., Dann 3rd, C.E., Leibowitz, J.L., Giedroc, D.P., 2009. Coronavirus N protein N-terminal domain (NTD) specifically binds the transcriptional regulatory sequence (TRS) and melts TRS-cTRS RNA duplexes. J. Mol. Biol. 394, 544–557.

Hastie, K.M., King, L.B., Zandonatti, M.A., Saphire, E.O., 2012. Structural basis for the dsRNA specificity of the Lassa virus NP exonuclease. PLoS ONE 7, e44211.

Hastie, K.M., Liu, T., Li, S., King, L.B., Ngo, N., Zandonatti, M.A., Woods Jr., V.L., de la Torre, J.C., Saphire, E.O., 2011. Crystal structure of the Lassa virus nucleoprotein-RNA complex reveals a gating mechanism for RNA binding. Proc. Natl. Acad. Sci. USA 108, 19365–19370.

He, R., Dobie, F., Ballantine, M., Leeson, A., Li, Y., Bastien, N., Cutts, T., Andonov, A., Cao, J., Booth, T.F., Plummer, F.A., Tyler, S., Baker, L., Li, X., 2004a. Analysis of multimerization of the SARS coronavirus nucleocapsid protein. Biochem. Biophys. Res. Commun. 316, 476–483.

He, R., Leeson, A., Ballantine, M., Andonov, A., Baker, L., Li, Y., Bastien, N., Feldmann, H., Strocher, U., Theriault, S., Cutts, T., Cao, J., Booth, T.F., Plummer, F.A., Tyler, S., Li, X., 2004b. Characterization of protein-protein interactions between the nucleocapsid protein and membrane protein of the SARS coronavirus. Virus Res. 105, 121–125.

Hilser, V.J., Thompson, E.B., 2007. Intrinsic disorder as a mechanism to optimize allosteric coupling in proteins. Proc. Natl. Acad. Sci. USA 104, 8311–8315.

Hsieh, P.K., Chang, S.C., Huang, C.C., Lee, T.T., Hsiao, C.W., Kou, Y.H., Chen, I.Y., Chang, C.K., Huang, T.H., Chang, M.F., 2005. Assembly of severe acute respiratory syndrome coronavirus RNA packaging signal into virus-like particles is nucleocapsid dependent. J. Virol. 79, 13848–13855.

Huang, Q., Yu, L., Petros, A.M., Gunasekera, A., Liu, Z., Xu, N., Hajduk, P., Mack, J., Fesik, S.W., Olejniczak, E.T., 2004a. Structure of the N-terminal RNA-binding domain of the SARS CoV nucleocapsid protein. Biochemistry (Mosc) 43, 6059–6063.

Huang, Q., Yu, L., Petros, A.M., Gunasekera, A., Liu, Z., Xu, N., Hajduk, P., Mack, J., Fesik, S.W., Olejniczak, E.T., 2004b. Structure of the N-terminal RNA-binding domain of the SARS CoV nucleocapsid protein. Biochemistry 43, 6059–6063.

Hung, H.C., Liu, C.L., Hsu, J.T., Horng, J.T., Fang, M.Y., Wu, S.Y., Ueng, S.H., Wang, M.Y., Yaw, C.W., Hou, M.H., 2012. Development of an anti-influenza drug screening assay targeting nucleoproteins with tryptophan fluorescence quenching. Anal. Chem. 84, 6391–6399.

Hurst, K., Koetzner, C., Masters, P., 2009. Identification of in vivo interacting domains of the murine coronavirus nucleocapsid protein. J. Virol. 83, 7221–7234.

Hurst, K.R., Kuo, L., Koetzner, C.A., Ye, R., Hsue, B., Masters, P.S., 2005. A major determinant for membrane protein interaction localizes to the carboxy-terminal domain of the mouse coronavirus nucleocapsid protein. J. Virol. 79, 13285–13297.

Jayaram, H., Fan, H., Bowman, B.R., Ooi, A., Jayaram, J., Collisson, E.W., Lescar, J., Prasad, B.V.V., 2006. X-Ray structures of the N- and C-terminal domains of a coronavirus nucleocapsid protein: implications for nucleocapsid formation. J. Virol. 80, 6612–6620.

Kao, R.Y., Yang, D., Lau, L.S., Tsui, W.H., Hu, L., Dai, J., Chan, M.P., Chan, C.M., Wang, P., Zheng, B.J., Sun, J., Huang, J.D., Madar, J., Chen, G., Chen, H., Guan, Y., Yuen, K.Y., 2010. Identification of influenza A nucleoprotein as an antiviral target. Nat. Biotechnol. 28, 600–605.

Keane, S.C., Giedroc, D.P., 2013. Solution structure of mouse hepatitis virus (MHV) nsp3a and determinants of the interaction with MHV nucleocapsid (N) protein. J. Virol. 87, 3502–3515.

Krijnse-Locker, J., Ericsson, M., Rottier, P.J., Griffiths, G., 1994. Characterization of the budding compartment of mouse hepatitis virus: evidence that transport from the RER to the Golgi complex requires only one vesicular transport step. J. Cell Biol. 124, 55–70.

Kuo, L., Masters, P.S., 2002. Genetic evidence for a structural interaction between the carboxy termini of the membrane and nucleocapsid proteins of mouse hepatitis virus. J. Virol. 76, 4987–4999.

Lai, M.M.C., Cavanagh, D., 1997. The molecular biology of coronaviruses. Adv. Virus Res. 48 (48), 1–100.

Laude, H., Masters, P.S., 1995. The coronavirus nucleocapsid protein. In: Siddell, S.G. (Ed.), The Coronaviridae. Plenum Press, New York.

Li, B., Wang, Q., Pan, X., Fernandez de Castro, I., Sun, Y., Guo, Y., Tao, X., Risco, C., Sui, S.F., Lou, Z., 2013. Bunyamwera virus possesses a distinct nucleocapsid protein to facilitate genome encapsidation. Proc. Natl. Acad. Sci. USA 110, 9048–9053.

Lo, Y.S., Lin, S.Y., Wang, S.M., Wang, C.T., Chiu, Y.L., Huang, T.H., Hou, M.H., 2013. Oligomerization of the carboxyl terminal domain of the human coronavirus 229E nucleocapsid protein. FEBS Lett. 587, 120–127.

Lunde, B.M., Moore, C., Varani, G., 2007. RNA-binding proteins: modular design for efficient function. Nat. Rev. Mol. Cell Biol. 8, 479–490.

Luo, H., Chen, J., Chen, K., Shen, X., Jiang, H., 2006. Carboxyl terminus of severe acute respiratory syndrome coronavirus nucleocapsid protein: self-association analysis and nucleic acid binding characterization. Biochemistry 45, 11827–11835.

Ma, Y., Tong, X., Xu, X., Li, X., Lou, Z., Rao, Z., 2010. Structures of the N- and C-terminal domains of MHV-A59 nucleocapsid protein corroborate a conserved RNA-protein binding mechanism in coronavirus. Protein Cell 1, 688–697.

Macneughton, M.R., Davies, H.A., 1978. Ribonucleoprotein-like structures from coronavirus particles. J. Gen. Virol. 39, 545–549.

Mark, J., Li, X., Cyr, T., Fournier, S., Jaentschke, B., Hefford, M.A., 2008. SARS coronavirus: unusual lability of the nucleocapsid protein. Biochem. Biophys. Res. Commun. 377, 429–433.

Marra, M.A., Jones, S.J.M., Astell, C.R., Holt, R.A., Brooks-Wilson, A., Butterfield, Y.S.N., Khattra, J., Asano, J.K., Barber, S.A., Chan, S.Y., Cloutier, A., Coughlin, S.M., Freeman, D., Girn, N., Griffith, O.L., Leach, S.R., Mayo, M., McDonald, H., Montgomery, S.B., Pandoh, P.K., Petrescu, A.S., Robertson, A.G., Schein, J.E., Siddiqui, A., Smailus, D.E., Stott, J.M., Yang, G.S., Plummer, F., Andonov, A., Artsob, H., Bastien, N., Bernard, K., Booth, T.F., Bowness, D., Czub, M., Drebot, M., Fernando, L., Flick, R., Garbutt, M., Gray, M., Grolla, A., Jones, S., Feldmann, H., Meyers, A., Kabani, A., Li, Y., Normand, S., Stroher, U., Tipples, G.A., Tyler, S., Vogrig, R., Ward, D., Watson, B., Brunham, R.C., Krajden, M., Petric, M., Skowronski, D.M., Upton, C., Roper, R.L., 2003. The genome sequence of the SARS-associated coronavirus. Science 300, 1399–1404.

Masters, P.S., 1992. Localization of an RNA-binding domain in the nucleocapsid protein of the coronavirus mouse hepatitis virus. Arch. Virol. 125, 141–160.

Masters, P.S., 2006. The molecular biology of coronaviruses. Adv. Virus Res. 66, 193–292.

Musah, R.A., 2004. The HIV-1 nucleocapsid zinc finger protein as a target of antiretroviral therapy. Curr. Top. Med. Chem. 4, 1605–1622.

Nagai, K., Oubridge, C., Ito, N., Avis, J., Evans, P., 1995. The RNP domain: a sequence-specific RNA-binding domain involved in processing and transport of RNA. Trends Biochem. Sci. 20, 235–240.

Narayanan, K., Maeda, A., Maeda, J., Makino, S., 2000. Characterization of the coronavirus M protein and nucleocapsid interaction in infected cells. J. Virol. 74, 8127–8134.

Nelson, G.W., Stohlman, S.A., Tahara, S.M., 2000. High affinity interaction between nucleocapsid protein and leader/intergenic sequence of mouse hepatitis virus RNA. J. Gen. Virol. 81, 181–188.

Neuman, B.W., Adair, B.D., Yoshioka, C., Quispe, J.D., Orca, G., Kuhn, P., Milligan, R.A., Yeager, M., Buchmeier, M.J., 2006. Supramolecular architecture of severe acute respiratory syndrome coronavirus revealed by electron cryomicroscopy. J. Virol. 80, 7918–7928.

Niu, F., Shaw, N., Wang, Y.E., Jiao, L., Ding, W., Li, X., Zhu, P., Upur, H., Ouyang, S., Cheng, G., Liu, Z.-J., 2013. Structure of the Leanyer orthobunyavirus nucleoprotein-RNA complex reveals unique architecture for RNA encapsidation. Proc. Natl. Acad. Sci. 110, 9054–9059.

Oubridge, C., Ito, N., Evans, P.R., Teo, C.H., Nagai, K., 1994. Crystal structure at 1.92 Å resolution of the RNA-binding domain of the U1A spliceosomal protein complexed with an RNA hairpin. Nature 372, 432–438.

Parker, M.M., Masters, P.S., 1990. Sequence comparison of the N genes of five strains of the coronavirus mouse hepatitis virus suggests a three domain structure for the nucleocapsid protein. Virology 179, 463–468.

Peng, T.Y., Lee, K.R., Tarn, W.Y., 2008. Phosphorylation of the arginine/serine dipeptide-rich motif of the severe acute respiratory syndrome coronavirus nucleocapsid protein modulates its multimerization, translation inhibitory activity and cellular localization. FEBS J. 275, 4152–4163.

Raymond, D.D., Piper, M.E., Gerrard, S.R., Skiniotis, G., Smith, J.L., 2012. Phleboviruses encapsidate their genomes by sequestering RNA bases. Proc. Natl. Acad. Sci. 109, 19208–19213.

Risco, C., Anton, I.M., Enjuanes, L., Carrascosa, J.L., 1996. The transmissible gastroenteritis coronavirus contains a spherical core shell consisting of M and N proteins. J. Virol. 70, 4773–4777.

Robbins, S.G., Frana, M.F., Mcgowan, J.J., Boyle, J.F., Holmes, K.V., 1986. RNA-binding proteins of coronavirus MHV – detection of monomeric and multimeric-N protein with an RNA overlay-protein blot assay. Virology 150, 402–410.

Rota, P.A., Oberste, M.S., Monroe, S.S., Nix, W.A., Campagnoli, R., Icenogle, J.P., Peñaranda, S., Bankamp, B., Maher, K., Chen, M.H., Tong, S., Tamin, A., Lowe, L., Frace, M., DeRisi, J.L., Chen, Q., Wang, D., Erdman, D.D., Peret, T.C., Burns, C., Ksiazek, T.G., Rollin, P.E., Sanchez, A., Liffick, S., Holloway, B., Limor, J., McCaustland, K., Olsen-Rasmussen, M., Fouchier, R., Günther, S., Osterhaus, A.D., Drosten, C., Pallansch, M.A., Anderson, L.J., Bellini, W.J., 2003. Characterization of a novel coronavirus associated with severe acute respiratory syndrome. Science 300, 1394–1399.

Saikatendu, K.S., Joseph, J.S., Subramanian, V., Neuman, B.W., Buchmeier, M.J., Stevens, R.C., Kuhn, P., 2007. Ribonucleocapsid formation of SARS-CoV through molecular action of the N-terminal domain of N protein. J. Virol. 81, 3913–3921.

Shoemaker, B.A., Portman, J.J., Wolynes, P.G., 2000. Speeding molecular recognition by using the folding funnel: the fly-casting mechanism. Proc. Natl. Acad. Sci. USA 97, 8868–8873.

Stertz, S., Reichelt, M., Spiegel, M., Kuri, T., Martínez-Sobrido, L., García-Sastre, A., Weber, F., Kochs, G., 2007. The intracellular sites of early replication and budding of SARS-coronavirus. Virology 361, 304–315.

Sturman, L.S., Holmes, K.V., Behnke, J., 1980. Isolation of coronavirus envelope glycoproteins and interaction with the viral nucleocapsid. J. Virol. 33, 449–462.

Surjit, M., Kumar, R., Mishra, R.N., Reddy, M.K., Chow, V.T.K., Lal, S.K., 2005. The severe acute respiratory syndrome coronavirus nucleocapsid protein is phosphorylated and localizes in the cytoplasm by 14-3-3-mediated translocation. J. Virol. 79, 11476–11486.

Surjit, M., Lal, S.K., 2008. The SARS-CoV nucleocapsid protein: a protein with multifarious activities. Infect. Genet. Evol. 8, 397–405.

Surjit, M., Liu, B., Chow, V.T.K., Lal, S.K., 2006. The nucleocapsid protein of severe acute respiratory syndrome-coronavirus inhibits the activity of cyclin-cyclin-dependent kinase complex and blocks S phase progression in mammalian cells. J. Biol. Chem. 281, 10669–10681.

Surjit, M., Liu, B., Kumar, P., Chow, V.T.K., Lal, S.K., 2004. The nucleocapsid protein of the SARS coronavirus is capable of self-association through a C-terminal 209 amino acid interaction domain. Biochem. Biophys. Res. Commun. 317, 1030–1036.

Tahara, S.M., Dietlin, T.A., Bergmann, C.C., Nelson, G.W., Kyuwa, S., Anthony, R.P., Stohlman, S.A., 1994. Coronavirus translational regulation – leader affects messenger-RNA efficiency. Virology 202, 621–630.

Tahara, S.M., Dietlin, T.A., Nelson, G.W., Stohlman, S.A., Manno, D.J., 1998. Mouse hepatitis virus nucleocapsid protein as a translational effector of viral mRNAs. Coronaviruses and Arteriviruses 440, 313–318.

Takeda, M., Chang, C.K., Ikeya, T., Guntert, P., Chang, Y.H., Hsu, Y.L., Huang, T.H., Kainosho, M., 2008. Solution structure of the C-terminal dimerization domain of SARS coronavirus nucleocapsid protein solved by the SAIL-NMR method. J. Mol. Biol. 380, 608–622.

Tang, C., Iwahara, J., Clore, G.M., 2005. Accurate determination of leucine and valine side-chain conformations using U-[15N/13C/2H]/[1H-(methine/methyl)-Leu/Val] isotope labeling, NOE pattern recognition, and methine Cgamma-Hgmma/Cbeta-Hbeta residual dipolar couplings: application to the 34-kDa enzyme IIA(chitobiose). J. Biomol. NMR 33, 105–121.

Tawar, R.G., Duquerroy, S., Vonrhein, C., Varela, P.F., Damier-Piolle, L., Castagné, N., MacLellan, K., Bedouelle, H., Bricogne, G., Bhella, D., Eléouët, J.-F., Rey, F.A., 2009. Crystal structure of a nucleocapsid-like nucleoprotein-RNA complex of respiratory syncytial virus. Science 326, 1279–1283.

van Boheemen, S., de Graaf, M., Lauber, C., Bestebroer, T.M., Raj, V.S., Zaki, A.M., Osterhaus, A.D., Haagmans, B.L., Gorbelenya, A.E., Snijder, E.J., Fouchier, R.A., 2012. Genomic characterization of a newly discovered coronavirus associated with acute respiratory distress syndrome in humans. mBio 3.

van der Meer, Y., Snijder, E.J., Dobbe, J.C., Schleich, S., Denison, M.R., Spaan, W.J., Locker, J.K., 1999. Localization of mouse hepatitis virus nonstructural proteins and RNA synthesis indicates a role for late endosomes in viral replication. J. Virol. 73, 7641–7657.

Verheije, M.H., Hagemeijer, M.C., Ulasli, M., Reggiori, F., Rottier, P.J., Masters, P.S., de Haan, C.A., 2010. The coronavirus nucleocapsid protein is dynamically associated with the replication-transcription complexes. J. Virol. 84, 11575–11579.

Wei, W.Y., Li, H.C., Chen, C.Y., Yang, C.H., Lee, S.K., Wang, C.W., Ma, H.C., Juang, Y.L., Lo, S.Y., 2012. SARS-CoV nucleocapsid protein interacts with cellular pyruvate kinase protein and inhibits its activity. Arch. Virol. 157, 635–645.

Wu, C., Yeh, S., Tsay, Y., Shieh, Y., Kao, C., Chen, Y., Wang, S., Kuo, T., Chen, D., Chen, P., 2009. Glycogen synthase kinase-3 regulates the phosphorylation of sars-coronavirus nucleocapsid protein and viral replication. J. Biol. Chem. 284, 5229–5239.

Xie, H., Vucetic, S., Iakoucheva, L.M., Oldfield, C.J., Dunker, A.K., Uversky, V.N., Obradovic, Z., 2007. Functional anthology of intrinsic disorder. 1. Biological processes and functions of proteins with long disordered regions. J. Proteome Res. 6, 1882–1898.

Yu, I.M., Gustafson, C.L.T., Diao, J., Burgner, J.W., Li, Z., Zhang, J., Chen, J., 2005. Recombinant severe acute respiratory syndrome (SARS) coronavirus nucleocapsid protein forms a dimer through its C-terminal domain. J. Biol. Chem. 280, 23280–23286.

Yu, I.M., Oldham, M.L., Zhang, J., Chen, J., 2006. Crystal structure of the severe acute respiratory syndrome (SARS) coronavirus nucleocapsid dimerization domain reveals evolutionary linkage between corona- and arteriviridae. J. Biol. Chem. 281, 17134–17139.

Zakhartchouk, A.N., Liu, Q., Petric, M., Babiuk, L.A., 2005. Augmentation of immune responses to SARS coronavirus by a combination of DNA and whole killed virus vaccines. Vaccine 23, 4385–4391.

Zeng, Y.C., Ye, L.B., Zhu, S.L., Zheng, H., Zhao, P., Cai, W.J., Su, L.Y., She, Y.L., Wu, Z.H., 2008. The nucleocapsid protein of SARS-associated coronavirus inhibits B23 phosphorylation. Biochem. Biophys. Res. Commun. 369, 287–291.

Zhang, Y.P., Zhang, R.W., Chang, W.S., Wang, Y.Y., 2010. Cxcl16 interact with SARS-CoV N protein in and out cell. Virol. Sin. 25, 369–374.

Zhao, X., Nicholls, J.M., Chen, Y.-G., 2008. Severe acute respiratory syndrome-associated coronavirus nucleocapsid protein interacts with Smad3 and modulates transforming growth factor-{beta} signaling. J. Biol. Chem. 283, 3272–3280.

Zhou, B., Liu, J., Wang, Q., Liu, X., Li, X., Li, P., Ma, Q., Cao, C., 2008. The nucleocapsid protein of severe acute respiratory syndrome coronavirus inhibits cell cytokinesis and proliferation by interacting with translation elongation factor 1{alpha}. J. Virol. 82, 6962–6971.

Zlotnick, A., 2003. Are weak protein-protein interactions the general rule in capsid assembly? Virology 315, 269–274.
