
CHEM BIO CHEM
COMMUNICATIONS

DOI: 10.1002/cbic.201300751

# Cytochrome P450 as Dimerization Catalyst in Diketopiperazine Alkaloid Biosynthesis

Takayoshi Saruwatari,${}^{[a]}$ Fumitoshi Yagishita,${}^{[b]}$ Takashi Mino,${}^{[b]}$ Hiroshi Noguchi,${}^{[a]}$ Kinya Hotta,${}^{[c]}$ and Kenji Watanabe*${}^{[a]}$

---

As dimeric natural products frequently exhibit useful biological activities, identifying and understanding their mechanisms of dimerization is of great interest. One such compound is (−)-dityptophenalone, isolated from *Aspergillus flavus*, which inhibits substance P receptor for potential analgesic and anti-inflammatory activity. Through targeted gene knockout in *A. flavus* and heterologous yeast gene expression, we determined for the first time the gene cluster and pathway for the biosynthesis of a dimeric diketopiperazine alkaloid. We also determined that a single cytochrome P450, DtpC, is responsible not only for pyrroloindole ring formation but also for concurrent dimerization of *N*-methylphenylalanyltryptophanyl diketopiperazine monomers into a homodimeric product. Furthermore, DtpC exhibits relaxed substrate specificity, allowing the formation of two new dimeric compounds from a non-native monomeric precursor, brevianamide F. A radical-mediated mechanism of dimerization is proposed.

and understanding the biosynthetic enzymes that can perform such reactions is of great interest.

To identify the genes involved in the biosynthesis of 1, double homologous recombination-mediated deletion of a set of genes from a small cluster predicted to be involved in a nonribosomal peptide product formation (Figure 1B) was performed in the *A. flavus ΔpyrG/Δku70* strain, *A. flavus* A1421 ${}^{[8]}$ (see the Supporting Information for details). Analysis of metabolites produced by those mutants confirmed that the nonribosomal peptide synthetase (NRPS) DtpA, encoded by AFLA_005440, is indispensable for the formation of 1 (Figure S6 i–iii in the Supporting Information). The chemical structure of this compound was characterized with high-resolution (HR) electrospray ionization (ESI) LC-MS, ${}^{1}$H NMR, ${}^{13}$C NMR, and polarimetry to establish that 1 is (−)-dityptophenalone (Figures S7–S11 and Table S2). This allowed us to assign the gene cluster as the dityptophenalone biosynthetic gene cluster and DtpA as the NRPS responsible for the formation of the peptide core of 1. Subsequent identification of the formation of diketopiperazine product 5 (see Figures S12–S16 and Table S3 for details of the chemical characterization) upon deletion of the second gene of the cluster AFLA_005450 (Figure S2) indicated that DtpA is responsible for the production of 5 (Figure S6 iv and v). The yield of 5 from the Δ*dtpB* strain was very low, presumably because the fungus metabolized 5 efficiently. Nevertheless, large-scale preparation of 9 permitted LC-MS, HRESIMS, ${}^{1}$H NMR, and ${}^{13}$C NMR analyses of this compound to determine the identity as cyclophenylalanyltryptophenyl. ${}^{[9]}$ The core structure of 5 indicated that DtpA accepts L-tryptophan and L-phenylalanine as its substrates and forms a phenylalanyltryptophanyl cyclic dipeptide product (Figure 1C). Sequence analysis ${}^{[10]}$ of DtpA showed the presence of two intact modules and a C-terminal condensation domain (Figure 1B), suggesting that the two modules condense L-tryptophan and L-phenylalanine together. The terminal condensation domain might be responsible for cyclization of the dipeptide to form the diketopiperazine structure. Furthermore, accumulation of 6, the N-methylated product of 5, was observed upon deletion of the third gene of the cluster, AFLA_005460 (Figures S17–S21 and Table S4). This indicated that DtpB, a predicted N-methyltransferase encoded by AFLA_005450, performs N-methylation of 5 to yield 6. This observation is in agreement with previous reports of stand-alone methyltransferases that belong to the EasF family of methyltransferases, including FgaMT, an N-methyltransferase from *Aspergillus fumigatus* involved in the biosynthesis of ergot alkaloids, ${}^{[11]}$ and EgtD, a mycobacterial N-methyltransferase involved in ergothioneine biosynthesis. ${}^{[12]}$ These results indicated

---

[a] T. Saruwatari, Prof. Dr. H. Noguchi, Prof. Dr. K. Watanabe  
Department of Pharmaceutical Sciences, University of Shizuoka  
52-1 Yada, Suruga-ku, Shizuoka 422-8526 (Japan)  
E-mail: kenji55@u-shizuoka-ken.ac.jp  

[b] F. Yagishita, Prof. Dr. T. Mino  
Department of Applied Chemistry and Biotechnology  
Graduate School of Engineering, Chiba University  
Yayoi-cho, Inage-ku, Chiba 263-8522 (Japan)  

[c] Prof. Dr. K. Hotta  
School of Biosciences, The University of Nottingham Malaysia Campus  
Jalan Broga, Semenyih, Selangor 43500 (Malaysia)  

Supporting information for this article is available on the WWW under  
http://dx.doi.org/10.1002/cbic.201300751.

---

Wiley Online Library © 2014 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim  
ChemBioChem 2014, 15, 656–659 656

CHEMBIOCHEM
COMMUNICATIONS
www.chembiochem.org

A)

![Chemical Structure 1](chemical_structure_1.png)  
(−)-ditryptophenale (1)

![Chemical Structure 2](chemical_structure_2.png)  
(+)-WIN 64821 (2)

![Chemical Structure 3](chemical_structure_3.png)  
(−)-dibrevianamide F (3)

![Chemical Structure 4](chemical_structure_4.png)  
(−)-tryprophenale (4)

B)


Ditryptophenale biosynthetic gene cluster


| Gene name | Protein name | Amino acid length | Protein homolog, GenBank accession number, Origin | Identity / Similarity (%) | Proposed protein function |
|-----------|--------------|-------------------|-------------------------------------------------|--------------------------|---------------------------|
| `dtpA`    | DtpA         | 2492             | Nonribosomal peptide synthetase module, EIT77215.1, A. oryzae | 97.7 / 98.0              | Diketopiperazine core formation |
| `dtpB`    | DtpB         | 191              | N-methyltransferase, XP_001822636.2, A. oryzae   | 71.2 / 80.9              | N-methylation            |
| `dtpC`    | DtpC         | 525              | Cytochrome P450, XP_003190172.1, A. oryzae      | 50.9 / 52.8              | Cyclization and dimerization |

C)

![Mechanism of Dimerization](mechanism_of_dimerization.png)

Figure 1. Proposed biosynthesis of (−)-ditryptophenale (1). A) Chemical structures of representative dimeric compounds. Compounds 3 and 4 were newly isolated and characterized within this study. B) A. flavus A1421 ditryptophenale biosynthetic gene cluster and predicted function of the genes in the cluster.[10] C) Proposed mechanism of dimerization in 1, 3, and 4 from 6 and 7 through a radical route catalyzed by DtpC.

that the predicted cytochrome P450 (P450) DtpC, encoded by AFLA_005460, is likely responsible for cyclization and dimerization of 6 to generate 1. To confirm the function of DtpC and understand the detailed mechanism of the reactions catalyzed by DtpC for the formation of 1, we characterized the activity of DtpC against 6 in vitro (see the Supporting Information for details).

© 2014 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim
ChemBioChem 2014, 15, 656–659 657
tails). DtpC was successfully produced in *Saccharomyces cerevisiae*, as determined by Western blotting analysis of the microsomal fraction from the *dtpC*-expressing strain (Figures S4 and S5). When 6 was mixed with the DtpC-containing yeast microsomal fraction, formation of 1 was observed as a clear UV peak (Figure 2A, 3 vs. 4). Mass and NMR spectrometric analyses, including rotation data, identified the constituent of this peak to be 1 (Figures S7–S11 and Table S2). This result conclusively identified that a P450 is capable of accepting an unfused diketopiperazine 6 as its substrate and performing concurrent pyrroloindole formation and dimerization to form 1 (Figure 1C).

Considering the structure of other dimeric diketopiperazine alkaloids \({ }^{[13]}\) and WIN 64821 analogues prepared in feeding experiments \({ }^{[5]}\) that differ in the non-tryptophan residue of the cyclic dipeptide core, we sought to examine in vitro whether DtpC would accept other diketopiperazines for dimerization. Indeed, when brevianamide F (7), a prolyltryptophanyl diketopiperazine \({ }^{[14]}\) obtained heterologously from yeast, \({ }^{[15]}\) was reacted with DtpC, the corresponding homodimeric compound (−)-dibrevianamide F (3) was obtained (Figure 1C, Figure 2B, 3 vs. 4). In addition, when DtpC was provided with a mixture of 6 and 7, a heterodimeric product that we named (−)-tryprophe naline (4) was also obtained (Figure 1C, Figure 2C, 5 vs. 6). The identity and structure of those new compounds were determined spectroscopically and by X-ray crystallography (Figures S22–S27 and Table S5 for 3 and Figures S28–S33 and Table S6 for 4).

To gain better understanding of how the dimerization may be executed by DtpC, the amino acid sequence of this P450 was analyzed with FFAS03 \({ }^{[16]}\) to predict P450s structurally similar to DtpC. This algorithm predicted that DtpC has the highest structural resemblance to the human P450 3A4 (CYP3A4). CYP3A4 is known to have a particularly relaxed substrate specificity with a flexible substrate binding pocket, and has previously been crystallized with two molecules of a relatively large inhibitor, ketoconazole, within a single active site pocket. \({ }^{[17]}\) Similar observations were made with other P450s, such as CYP158A2, \({ }^{[18]}\) responsible for dimerization of the type III polyketide natural product flaviolin, which crystallizes with two molecules of substrate near the heme molecule in the binding pocket. An homology model of the structure of DtpC, constructed with the program I-TASSER, \({ }^{[19]}\) suggests that DtpC can also have a substrate pocket large enough to accommodate two molecules of 6 comfortably in the vicinity of the heme without taking into account the flexible segment comprised of F-F′-G′-G helices that adjust to a bound ligand \({ }^{[17,18]}\) (Figures 3 and S7). The model also allows placement of the non-tryptophan residue of the diketopiperazine moiety, for example, the phenyl side chain of 6, into an open space within the binding pocket, giving a potential explanation for the substrate promiscuity observed in this P450.

Previous studies on the synthesis of diketopiperazine dimeric alkaloids \({ }^{[20]}\) indicate that C2–N10 ring closure and dimer formation occur in a concerted manner. The homology model of DtpC supports this notion, showing a possible arrangement of the monomers within the active site pocket. The model predicts that the C3–C3′ bond between the monomers can be formed through a concerted mechanism, facilitated by the proximity of the monomers within the binding pocket. The flexible segment containing the F-F′-G′-G helices is thought to play a role in adjusting the conformation of the enzyme to accommodate different substrates and facilitate the dimerization reaction.

![Figure 3](https://example.com/figure3.png)

**Figure 3.** Homology model of the structure of DtpC. Cross-section of the active site pocket near the heme (white) is shown with two molecules of cyclo-N-methylphenylalanyltryptophenyl-derived monomer bound in one possible predicted arrangement (light and dark gray). The C3–C3′ bond to be formed between the monomers is shown as a dotted line. The position of the flexible F-F′-G′-G helix-containing segment is denoted by a thick white double arrow. The phenyl groups of the monomers point toward the viewer into an open space within the active site.

![Figure 2](https://example.com/figure2.png)

**Figure 2.** In vitro analysis of the activity of DtpC against different substrates. Details are given in the Supporting Information. Peaks from separate reactions were standardized by scaling to the height of the internal standard (IS) peak. All traces were monitored at 214 nm. A) HPLC profiles of i: the authentic reference of 1 and ii: 6; iii: a reaction mixture containing heat-inactivated DtpC, indicated by (−), with 6 as a negative control; and iv: a reaction mixture containing active DtpC, indicated by (+), and 6, showing the formation of 1. B) HPLC profiles of i: the authentic reference of 3 and ii: 7; iii: a reaction mixture containing heat-inactivated DtpC with 7; iv: a reaction mixture containing DtpC and 7, showing the formation of 3. C) HPLC profiles of i: the authentic reference of 1, ii: 3, iii: 6 and iv: 7; v: a reaction mixture containing heat-inactivated DtpC with 6 and 7; vi: a reaction mixture containing DtpC, 6, and 7, showing the formation of 4.

© 2014 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim

ChemBioChem 2014, 15, 656–659   658
The Uehara Memorial Foundation (K.W.), the Mochida Memorial Foundation for Medical and Pharmaceutical Research (K.W.), The Naito Foundation Japan (K.W.), the Nagase Science and Technology Foundation Japan (K.W.), and a JSPS Fellowship for Research In Japan (K.H.).

Keywords: biosynthesis · cytochrome P450 · diketopiperazines · dimerization · filamentous fungi

[1] J. P. Springer, G. Büchi, B. Kobbe, A. L. Demain, J. Clardy, *Tetrahedron Lett.* **1977**, 18, 2403–2406.

[2] C. J. Barrow, P. Cai, J. K. Snyder, D. M. Sedlock, H. H. Sun, R. Cooper, *J. Org. Chem.* **1993**, 58, 6016–6021.

[3] G. Lian, B. Yu, *Chem. Biodiversity* **2010**, 7, 2660–2691.

[4] a) K. Katagiri, K. Sato, S. Hayakawa, T. Matsushima, H. Minato, *J. Antibiot.* **1970**, 23, 420–422; b) D. Greiner, T. Bonaldi, R. Eskeland, E. Roemer, A. Imhof, *Nat. Chem. Biol.* **2005**, 1, 143–145.

[5] J. L. Popp, L. L. Musza, C. J. Barrow, P. J. Rudewicz, D. R. Houck, *J. Antibiot.* **1994**, 47, 411–419.

[6] a) G. Y. Li, B. G. Li, T. Yang, J. F. Yan, G. Y. Liu, G. L. Zhang, *J. Nat. Prod.* **2006**, 69, 1374–1376; b) N. Boyer, K. C. Morrison, J. Kim, P. J. Hergenrother, M. Movassaghi, *Chem. Sci.* **2013**, 4, 1646–1657; c) T. S. Bugni, C. M. Ireland, *Nat. Prod. Rep.* **2004**, 21, 143–163; d) G. Ding, L. Jiang, L. Guo, X. Chen, H. Zhang, Y. Che, *J. Nat. Prod.* **2008**, 71, 1861–1865; e) C. Pérez-Balado, P. Rodríguez-Graña, Á. R. de Lera, *Chem. Eur. J.* **2009**, 15, 9928–9937; f) N. M. Gomes, T. Dethoup, N. Singburaudom, L. Gales, A. M. S. Silva, A. Kijjooa, *Phytochem. Lett.* **2012**, 5, 717–720; g) S. Cai, X. Kong, W. Wang, H. Zhou, T. Zhu, D. Li, Q. Gu, *Tetrahedron Lett.* **2012**, 53, 2615–2617; h) C. Rank, M. L. Klejnstrup, L. M. Petersen, S. Kildgaard, J. C. Frisvad, C. Held Gotfredsen, T. Ostenfeld Larsen, *Metabolites* **2012**, 2, 39–56.

[7] G. Bérubé, *Curr. Med. Chem.* **2006**, 13, 131–154.

[8] P. K. Chang, L. L. Scharfenstein, Q. Wei, D. Bhatnagar, *J. Microbiol. Methods* **2010**, 81, 240–246.

[9] M. Tullberg, K. Luthman, M. Grötlı, *J. Comb. Chem.* **2006**, 8, 915–922.

[10] S. Anand, M. V. Prasad, G. Yadav, N. Kumar, J. Shehara, M. Z. Ansari, D. Mohanty, *Nucleic Acids Res.* **2010**, 38, W487–W496.

[11] O. Rigbers, S. M. Li, *J. Biol. Chem.* **2008**, 283, 26859–26868.

[12] F. P. Seebeck, *J. Am. Chem. Soc.* **2010**, 132, 6632–6633.

[13] A. D. Borthwick, *Chem. Rev.* **2012**, 112, 3641–3716.

[14] S. Maiya, A. Grundmann, S. M. Li, G. Turner, *ChemBioChem* **2006**, 7, 1062–1069.

[15] Y. Tsunematsu, N. Ishikawa, D. Wakana, Y. Goda, H. Noguchi, H. Moriya, K. Hotta, K. Watanabe, *Nat. Chem. Biol.* **2013**, 9, 818–825.

[16] L. Jaroszewski, L. Rychlewski, Z. Li, W. Li, A. Godzik, *Nucleic Acids Res.* **2005**, 33, W284–288.

[17] M. Ekroos, T. Sjögren, *Proc. Natl. Acad. Sci. USA* **2006**, 103, 13682–13687.

[18] B. Zhao, F. P. Guengerich, A. Bellamine, D. C. Lamb, M. Izumikawa, L. Lei, L. M. Podust, M. Sundaramoorthy, J. A. Kalaitzis, L. M. Reddy, S. L. Kelly, B. S. Moore, D. Stec, M. Voehler, J. R. Falck, T. Shimada, M. R. Waterman, *J. Biol. Chem.* **2005**, 280, 11599–11607.

[19] A. Roy, A. Kucukural, Y. Zhang, *Nat. Protoc.* **2010**, 5, 725–738.

[20] a) M. Movassaghi, M. A. Schmidt, J. A. Ashenhurst, *Angew. Chem.* **2008**, 120, 1507–1509; *Angew. Chem. Int. Ed.* **2008**, 47, 1485–1487; b) S. Tadano, Y. Mukaeda, H. Ishikawa, *Angew. Chem.* **2013**, 125, 8148–8152; *Angew. Chem. Int. Ed.* **2013**, 52, 7990–7994.

Received: November 28, 2013  
Published online on February 13, 2014

Acknowledgements

We thank Prof. Yoshitaka Hamashima for enlightening discussions on the radical reaction mechanism. We acknowledge financial support from the Japan Society for the Promotion of Science (JSPS) through the “Funding Program for Next Generation World-Leading Researchers,” initiated by the Council for Science and Technology Policy (No. LS103; K.W.) and the Industrial Technology Research Grant Program in 2009 from the New Energy and Industrial Technology Development Organization (NEDO) of Japan (No. 09C46001a; K.W.). This work was also supported in part by
