
Accepted Manuscript

The Paradox of Mitochondrial Dysfunction and Extended Longevity

Erin Munkácsy, Shane L. Rea

PII: S0531-5565(14)00088-6  
DOI: doi: 10.1016/j.exger.2014.03.016  
Reference: EXG 9382  

To appear in: Experimental Gerontology  

Received date: 17 December 2013  
Revised date: 2 March 2014  
Accepted date: 7 March 2014  

Please cite this article as: Munkácsy, Erin, Rea, Shane L., The Paradox of Mitochondrial Dysfunction and Extended Longevity, Experimental Gerontology (2014), doi: 10.1016/j.exger.2014.03.016  

This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.

The Paradox of Mitochondrial Dysfunction and Extended Longevity

Erin Munkácsy${}^{1,2}$ and Shane L. Rea*${}^{1,3}$

${}^{1}$ Barshop Institute for Longevity and Aging Studies, ${}^{2}$ Department of Cell and Structural Biology, and  
${}^{3}$ Department of Physiology, University of Texas Health Science Center at San Antonio, San Antonio, TX,  
78245-3207 USA.

*Address correspondence to: Shane L. Rea, PhD, 15355 Lambda Drive, STCBM Building, Rm. 2.200.04,  
Texas Research Park, San Antonio, TX 78245-3207. Fax: 210-562-5028; E-mail: reas3@uthscsa.edu

Abstract
Mitochondria play numerous, essential roles in the life of eukaryotes. Disruption of mitochondrial function in humans is often pathological or even lethal. Surprisingly, in some organisms mitochondrial dysfunction can result in life extension. This paradox has been studied most extensively in the long-lived Mit mutants of the nematode *Caenorhabditis elegans*. In this review, we explore the major responses that are activated following mitochondrial dysfunction in these animals and how these responses potentially act to extend their life. We focus our attention on five broad areas of current research—reactive oxygen species signaling, the mitochondrial unfolded protein response, autophagy, metabolic adaptation, and the roles played by various transcription factors. Lastly, we also examine why disruption of complexes I and II differ in their ability to induce the Mit phenotype and extend lifespan.

Keywords
C. elegans, Mitochondria, Mit Mutant, Lifespan, Aging, Metabolism

1. Introduction
In four billion years, complex life appears to have arisen only once, and from an unlikely endosymbiosis between two primordial prokaryotes. By endowing their host with vastly expanded respiratory capacity, the endosymbionts enabled their host to break through the bioenergetic constraints on genome size and to evolve a vast repertoire of genes (Lane and Martin 2010). Ultimately, this new chimeric cell type gave rise to all eukaryotic life and the ancient endosymbionts became today's mitochondria. For many eukaryotes, including humans, oxidative phosphorylation by the mitochondria is the primary mode of generating ATP and complete disruption of this process is lethal. Thus, it was a surprise to find that crippling the very organelle that enables our own existence can, in some species, prolong life. Exemplifying this are the Mit mutants of the nematode *Caenorhabditis elegans*.

The first Mit mutant discovered was *clk-1(e2519)* (Wong and others 1995), so named because this mutant displayed disrupted timing of several developmental and behavioral processes, leading to the mistaken hypothesis that the *clk-1* gene specified some sort of biological clock. In fact, *clk-1* encodes the *C. elegans* ortholog of demethoxyubiquinone hydroxylase. This protein is required for the synthesis of ubiquinone, the molecule which transports electrons from both complexes I and II, as well as other metabolic enzymes (see section 4.3), to complex III of the mitochondrial electron transport chain (ETC) (Miyadera and others 2001). Since this initial discovery, it has been found that disruption of almost any subunit of the ETC, or of enzymes needed for assembly of the ETC and its co-factors, can extend lifespan in *C. elegans*. RNAi knockdown of genes encoding subunits of complexes I, III, or IV increase lifespan, on average by 50% (Zuryn and others 2010). Lifespan can also be extended by low doses of mitochondrial toxins such as antimycin A (Dillin and others 2002), ethidium bromide (Tsang and Lemire 2002),

rotenone (Dillin and others 2002; Schmeisser and others 2013a), and arsenite (Schmeisser and others 2013b). Given the apparent non-specificity of the nature of mitochondrial ETC disruption in extending life, it is surprising that lifespan is never increased in response to knockdown of any of the subunits of complex II (Ichimiya and others 2002; Kuang and Ebert 2012; Rea and others 2007). Possible reasons for this are discussed in section 4. Nuclear DNA-encoded mitochondrial proteins for which the effect of mutational or RNAi knockdown on lifespan have been reported are listed in Table 1. The positions of the ETC subunits within the respiratory complexes are shown schematically in Figure 1. Unless specified explicitly, in this review, we refer to all long-lived animals with either mutational- or RNAi-induced knockdown of mitochondrial components as Mit “mutants”.

In humans, mutations in some of the same genes that produce longevity in Mit mutants paradoxically result in devastating diseases. This observation remained puzzling until it was discovered that both humans and worms are, in fact, much alike in that they both exhibit a threshold response to mitochondrial ETC perturbation. In worms this can be demonstrated very easily using RNAi targeting individual subunits of the ETC, such as *atp-3*. Starting with dilute doses of RNAi, increasing concentrations produce no detectable phenotype until a threshold is reached. Beyond this point, progressively greater target knockdown monotonically increases lifespan up to a second threshold, beyond which lifespan decreases until animals are eventually short-lived compared to wild-type (Rea and others 2007). In humans, mitochondrial diseases also present with variable phenotypes, and in some cases may even be asymptomatic (Limongelli and others 2004). Clearly, in both organisms, compensatory mechanisms are invoked and much work has gone into studying what these processes might be. While mitochondrial disruption is obviously undesirable in humans, it is notable that even mammals can achieve a longer life by this mechanism. In mice, knockout of the cytochrome c assembly factor *Surf1* (Dell'agnello and others 2007), hemizygous knockout of *Mclk1* (the mouse ortholog of *clk-1*) (Liu and others 2005), or reduction of mitochondrial ribosomal subunit S5 (Houtkooper and others 2013), have all been reported to prolong life.

## 2. The Mit Phenotype

*C. elegans* Mit mutants display a set of co-clustering phenotypes that may be fully or only partially penetrant. In addition to longevity, these include delayed embryonic and/or larval development, small adult size, slow muscle function, and reduced fertility and fecundity (Rea and others 2007). Using an RNAi dilution series, it has been clearly established that concentrations of RNAi that extend lifespan consistently induce the entire Mit phenotype (Rea and others 2007). When RNAi doses are severe enough, however, animals can arrest and even be short-lived. It is interesting to note that, while an RNAi screen against ETC components reported neither change in developmental rate nor alteration of adult size to be predictive of lifespan, the longest-lived worms in the screen were, in fact, both the smallest in size and the slowest to develop (Zuryn and others 2010).

However, the various co-segregating traits of the Mit phenotype are linked only superficially. Uncoupling of longevity from developmental delay in the robust *isp-1(qm150)* Mit mutant, for example, was shown following the serendipitous discovery of *isp-1(qm150); ctb-1(qm189)* double mutants. Cytochrome b (CTB-1) is a mitochondrial DNA (mtDNA)-encoded subunit of complex III. The *ctb-1(qm189)* mutation almost fully mitigates the delayed development of *isp-1(qm150)* mutant worms but has very little effect on their lifespan (Feng and others 2001). In a study of the CCO-1 subunit of complex IV, Dillin and colleagues demonstrated that intestinal- or neuronal-specific RNAi knockdown were each sufficient to extend nematode lifespan. Neither treatment, however, resulted in the slow movement characteristic of Mit mutants. Conversely, *cco-1* knockdown in the body wall muscles decelerated movement but actually shortened lifespan (Durieux and others 2011). Clearly, in this system, life extension and slowness are traits that can also be uncoupled. Whether one should further conclude that different tissues control different aspects of the Mit phenotype, or that differing degrees of *cco-1*

knockdown in different tissues are required to elicit life extension, remains an open question. Nonetheless, one important point to consider for all Mit mutants is that their final phenotypic outcome almost certainly represents an integrated response from multiple tissues. Perhaps then, what is most remarkable about the Mit mutants of C. elegans is that there is such significant overlap in their final phenotype—enough to be recognized as the “Mit phenotype”. In line with earlier findings (Rea and others 2007), this indicates the existence of a “sweet spot” for life extension in the face of ETC disruption.

3. Mechanisms of Mit Mutant Life Extension

In the following sections we discuss several areas of investigation that have been pursued as possible causes of life extension in Mit mutants. This section is not meant to be exhaustive and in all likelihood the longevity of these animals reflects the outcome of multiple processes acting in concert. When studying whole organisms, it is vital to remember that we are studying an integrated system that is dynamic and which can respond to various systemic perturbations in multiple ways.

3.1. Reactive Oxygen Species (ROS) Signaling

In 1956 Denham Harman published his seminal paper proposing free radicals as the proximal cause of aging (Harman 1956). This hypothesis posits that free radicals, both from the environment and especially aerobic respiration, cause damage that accumulates over time and eventually manifests as aging and death. Accordingly, it has been proposed that Mit mutants live longer because they have reduced ETC flux, resulting in decreased generation of reactive oxygen species (ROS) (Rea 2005). The fact that the short-lived mev-1(kn21) mutant produces high levels of superoxide radical (Senoo-Matsuda and others 2001) would seem to support a causative role of ROS in aging. However, contrary to expectations, not only Mit mutants, but also the long-lived daf-2(e1370) mutant, actually show higher levels of oxidative damage than wild-type worms at young adulthood (Labuschagne and others 2013). Other measures of oxidative damage have shown no correlation with lifespan (Rea and others 2007). It is notable that the unexpected finding of high levels of oxidative damage in long-lived animals is not peculiar to C. elegans. The exceptionally long-lived naked mole rat has higher levels of oxidatively damaged proteins, lipids, and DNA than physiologically age-matched mice (Andziak and others 2006).

In recent years, it has become clear that ROS are not always damaging and in fact play important roles as signaling molecules (Ray and others 2012). As one example of this, ROS generated by dysfunctional mitochondria inhibit cytosolic translation through GCN-2 (Baker and others 2012). When this kinase is knocked out, Mit mutants show stronger induction of the mitochondrial unfolded protein response—evidence that ROS-induced GCN-2 function plays a role in mitigating mitochondrial stress (Baker and others 2012). ROS may also be required for activation of HIF-1, a transcription factor shown to be required for maximal Mit mutant longevity (Hwang and Lee 2011; Lee and others 2010) and which shall be discussed in section 3.2. Counterintuitively, it has been proposed that Mit mutants and other models of extended lifespan live longer not by lowering ROS, but by increasing it just enough to induce protective stress-response genes that bring about the animal’s subsequent longevity. This hypothesis has been termed hormesis (Masoro 1998) and specifically “mitohormesis” when ROS leak from the mitochondria (Ristow and Zarse 2010). The effects of eliminating the mitochondrial superoxide dismutase SOD-2 seem to support an hormetic role for ROS: While oxidative damage is predictably increased, sod-2 deletion not only prolongs life, but further increases longevity in the clk-1(qm30) Mit mutant (Van Raamsdonk and Hekimi 2009) and extends the lifespan of the otherwise short-lived gas-1(fc21) mutant beyond wild-type, effectively making this particular double-knockout a Mit mutant (Suthammarak and others 2013).

3.2. Transcription Factors

While numerous transcription factors have been tested for a role in the longevity of Mit mutants, others have been identified from large RNAi screens. Some of these transcription factors have been clearly linked to a specific mechanism such as autophagy and are only briefly mentioned here. Others are not so easily categorized or the nature of their role in Mit mutants has yet to be defined.

When looking for mediators of longevity, the usual suspects are DAF-16 and SKN-1, which are the C. elegans orthologs of mammalian FOXO and Nrf2, respectively. Both are required for normal lifespan and have been shown to play a role in extending life via reduced insulin signaling (Lin and others 1997; Tullet and others 2008), TOR inhibition (Robida-Stubbs and others 2012), and at least some forms of dietary restriction (Bishop and Guarente 2007; Greer and others 2007; Honjoh and others 2009). DAF-16, however, has been repeatedly shown to be unnecessary for Mit mutant longevity (Dillin and others 2002; Durieux and others 2011; Feng and others 2001; Hansen and others 2005; Kim and Sun 2007; Lee and others 2003; Ventura and others 2009). Mit mutants might be expected to require SKN-1 since this transcription factor mediates the oxidative stress response (An and Blackwell 2003) and is essential for life extension invoked by both complex I inhibitors (Schmeisser and others 2013a) and ROS inducers (Schmeisser and others 2011; Zarse and others 2012). Surprisingly, however, the short-lived skn-1(zu67) mutant, which is deficient in skn-1 isoforms a and c, achieves life extension equivalent to wild-type worms on atp-3, cco-1, or cyc-1 RNAi (Rea and others 2007; Tullet and others 2008). Even the skn-1(zu135) mutant, wherein all skn-1 isoforms (a, b and c) are knocked out, was similarly long-lived on frh-1 RNAi (Ventura and others 2009).

The first transcription factor shown to modulate longevity in Mit mutants was the C. elegans homolog of p53, CEP-1 (Ventura and others 2009), which contributes to their stress response by upregulating protective genes such as the antioxidant glutathione-S-transferase (Torgovnick and others 2010). While CEP-1 contributes to life extension in response to moderate mitochondrial disturbance, most surprising is its role in the early demise of worms with severe mitochondrial dysfunction. This is most clearly seen in worms on undiluted atp-3 RNAi: Severe knockdown of this complex V subunit resulted in larval arrest and the early death of wild-type worms and yet this same treatment caused dramatic life extension in cep-1 null worms (Ventura and others 2009). This is significant as it might be assumed that the pathology resulting from severe mitochondrial dysfunction would be a direct consequence of the impaired mitochondria themselves, but this study shows that this is not the case. Even in the face of dramatic mitochondrial impairment, pathology and life shortening seem to be the result of maladaptive cellular responses.

The next transcription factor recognized for its role in Mit mutant life extension was even more unexpected. As its name suggests, hypoxia inducible factor-1 (HIF-1) mediates survival under conditions of low oxygen. Under normal conditions, HIF-1 is targeted for degradation (Ivan and others 2001; Jiang and others 2001). Consequently, loss of hif-1 is not detrimental to otherwise wild-type worms under normal conditions (Jiang and others 2001) and can even prolong life (Zhang and others 2009). It was therefore surprising to learn that knockdown of hif-1 or aha-1—which encodes the binding partner of HIF-1 (Jiang and others 2001)—significantly abrogates life extension in several Mit mutants, in spite of normoxic conditions (Khan and others 2013; Lee and others 2010). Life extension is reduced even when hif-1 knockdown is initiated in adulthood (Lee and others 2010), an unexpected finding since the Mit phenotype has been shown to be specified during larval development (Dillin and others 2002; Rea and others 2007). This suggests that adaptation to either hypoxic conditions or dysfunctional mitochondria

share overlapping requirements (Rea 2005). Interestingly, HIF-1 also plays a role in life extension mediated by dietary restriction (Chen and others 2009).

Additional transcription factors mediating life extension in Mit mutants also overlap with those utilized by various forms of dietary restriction, despite differences between these two mechanisms (Durieux and others 2011; Wolff and Dillin 2006). An RNAi screen for transcription factors required for the longevity of the isp-1(qm150); ctb-1(qm189) double mutant identified the nuclear hormone receptor nhr-25 (Walter and others 2011). While knockdown of nhr-25 had minimal effect on either wild-type worms or the long-lived insulin-like signaling pathway mutant age-1(hx546), it significantly shortened the lifespan of Mit mutants as well as eat-2(ad1116) mutants (this latter mutation affects the worms' ability to eat and is thus a dietary restriction model) (Walter and others 2011). As the known functions of nhr-25 all pertain to development, the nature of its specific role in mediating the longevity of these animals is unknown.

In a separate RNAi screen conducted by our own group in search of transcription factors controlling Mit mutant lifespan, we found that knockdown of jun-1 completely abolished life extension in both isp-1(qm150) and tpk-1(qm162) Mit mutants but had no effect on wild-type worms (Khan and others 2013). Previously, this transcription factor was shown to mediate life extension in response to intermittent fasting (Uno and others 2013). Induced JUN-1 targets included genes related to aging and ubiquitin-dependent protein catabolism. Enrichment of these genes led the authors to propose that intermittent fasting may prolong life through increased protein turnover via SCF E3 ligase components under the control of JUN-1 (Uno and others 2013). This may also be the role of JUN-1 in Mit mutant life extension. Consistent with this, several SCF ubiquitin ligases are also upregulated in response to mitochondrial dysfunction (Nargund and others 2012).

One unexpected transcription factor identified by our screen was TAF-4, which forms part of the core transcription factor complex TFIID. A specific role in longevity was surprising, given that TAF-4 is considered a general transcription factor and its removal is embryonic lethal (Walker and others 2001). However, while wild-type worms on taf-4 RNAi did show reduced fecundity, they were otherwise phenotypically normal and had lifespan identical to controls. In contrast, both isp-1(qm150) and tpk-1(qm162) Mit mutants had their life extension completely negated by RNAi knockdown of taf-4 (Khan and others 2013). We have hypothesized previously (Khan and others 2013), that TAF-4 mediates metabolic adaptations through its interaction with the cAMP responsive element-binding protein transcription factor, CRH-1 (C. elegans CREB homolog family member 1) (Altarejos and Montminy 2011).

Two homeobox domain transcription factors, namely, CEH-18 (Khan and others 2013) and CEH-23 (Walter and others 2011), also contribute to Mit mutant longevity. The roles these transcription factors play in life extension is unknown, but given the central role of neurons in modulating the lifespan of the entire worm (Durieux and others 2011), it is interesting that CEH-23 plays a role in neuronal differentiation during development (Altun-Gultekin and others 2001). RNAi knockdown of ceh-23 specifically reduced the life extension of Mit mutants while affecting neither the long-lived age-1(hx546) nor eat-2(ad1116) mutants, nor the short-lived mev-1(kn1) mutant, and actually increased the lifespan of both wild-type and daf-16(mgDf47) worms. Significantly, transgenic overexpression of ceh-23 in either a subset of neurons or in the intestine was sufficient to extend lifespan in otherwise wild-type animals (Walter and others 2011).

Autophagy has been established as contributing to longevity in multiple worm models of life extension and will be discussed in section 3.4 with respect to Mit mutants. However, it is worth noting here that it has recently been shown that all of these models share not only a requirement for

autophagy, but also the same transcription factor to induce it, namely, the TFEB ortholog, HLH-30 (Lapierre and others 2013).

Finally, the transcription factors most associated with Mit mutants are the Activating Transcription Factor associated with Stress-1 (ATFS-1), Ubiquitin-Like Protein-5 (UBL-5), and Defective Proventriculus in Drosophila homolog-1 (DVE-1). These are key mediators of the mitochondrial unfolded protein response (UPR<sup>mt</sup>) (Benedetti and others 2006; Haynes and others 2007; Haynes and others 2010) and will be examined in section 3.3.

In summary of this section, among all the transcription factors discussed, DAF-16, SKN-1, and HLH-30 are known to mediate life extension in multiple classes of long-lived C. elegans, but only HLH-30 is active in Mit mutants. Several transcription factors required by Mit mutants are also essential in various models of dietary restriction. That said, not all Mit mutants have the exact same transcription factor requirements, and neither do all dietary restriction models, but both classes overlap in their requirement for JUN-1, HIF-1, and NHR-25. Other transcription factors seem to modulate lifespan in a manner that is specific to Mit mutants (ATFS-1, UBL-5, DVE-1, and CEH-23) or have not, to our knowledge, been tested for their effects on lifespan in other classes of long-lived C. elegans (TAF-4, CEH-18, and AHA-1).

### 3.3. Mitochondrial Unfolded Protein Response

One mechanism by which the lifespan of Mit mutants may be extended entails activation of the mitochondrial unfolded protein response (UPR<sup>mt</sup>). Much like the prototypic UPR induced by the endoplasmic reticulum (UPR<sup>ER</sup>), an accumulation of misfolded proteins in the mitochondria triggers retrograde signaling to the nucleus, whereupon mitochondrial chaperones and proteases are then upregulated to re-establish homeostasis. This process has been described in detail elsewhere (Haynes and others 2013; Pellegrino and others 2013) and here we provide only a summary (Figure 2).

Much of the work on the UPR<sup>mt</sup> of C. elegans has been undertaken by the Ron and Haynes labs. It was these investigators who first discovered two mitochondrial chaperones, namely, HSP-6 and HSP-60—orthologous to mammalian mtHSP70 and HSP60, respectively—that were specifically induced in response to disruption of mitochondrial proteostasis (Yoneda and others 2004). The identification of these two proteins allowed fluorescent reporters to be constructed that, in turn, facilitated RNAi screens to find genes necessary for their induction following mitochondrial dysfunction (Benedetti and others 2006; Haynes and others 2007; Haynes and others 2010).

The proximal mediator of UPR<sup>mt</sup> appears to be the mitochondrial protease, CLPP-1, which degrades misfolded proteins in the mitochondrial matrix (Haynes and others 2007). The resulting peptides are then exported to the cytosol by the ATP-binding cassette transporter, HAF-1, which also modulates protein import into the mitochondria (Haynes and others 2010). Because mitochondrial protein import utilizes the mitochondrial membrane potential (∆Ψ<sub>m</sub>) generated by the ETC, monitoring protein import may be an indirect way of monitoring ETC function. Diminished ∆Ψ<sub>m</sub> has been shown to activate retrograde signaling and increase replicative lifespan in yeast (Miceli and others 2011), but whether it plays a role in UPR<sup>mt</sup> induction in C. elegans is unknown.

ATFS-1 is an unusual transcription factor in that it contains both mitochondrial and nuclear targeting sequences. Under normal conditions, ATFS-1 is imported into the mitochondrial matrix and degraded by the Lon protease. During mitochondrial dysfunction, some ATFS-1 is not imported and is instead redirected to the nucleus. HAF-1 plays a role in this process but the specifics are not yet known (Haynes and others 2010; Nargund and others 2012). Also required for UPR<sup>mt</sup> activation is the homeodomain-containing transcription factor DVE-1, which undergoes nuclear redistribution downstream of CLPP-1 activation but independent of HAF-1 function (Haynes and others 2007). While
HAF-1 does not play a role in the redistribution of DVE-1, it is needed for DVE-1-dependent upregulation of the ubiquitin-like protein UBL-5 (Haynes and others 2007), also required for UPR<sup>mt</sup> induction (Benedetti and others 2006). Factors mediating signaling from CLPP-1 and HAF-1 to the nucleus independent of ATFS-1 have not yet been identified. In the nucleus, UBL-5 and DVE-1 form a transcription complex (Haynes and others 2007) that cooperates with ATFS-1 to activate a broad transcriptional response that includes induction of mitochondrial chaperones, proteases, and transporters; genes mediating mitochondrial fission; and various metabolic enzymes (Nargund and others 2012).

In an elegant series of experiments, Dillin and colleagues further delineated *C. elegans* UPR<sup>mt</sup> induction at the organismal level. Surprisingly, these investigators discovered that the UPR<sup>mt</sup> could be induced in the worm intestine by mitochondrial disruption that occurred distally in neuron or muscle cells. This non-cell-autonomous UPR<sup>mt</sup> response was not activated in the canonical manner: While intestine-specific knockdown of *ubl-5* blocked UPR<sup>mt</sup> activated by mitochondrial disruption in the intestine, it did not block UPR<sup>mt</sup> induced by the non-cell-autonomous response activated by neuronal or muscle mitochondrial disruption (Durieux and others 2011).

Based on the pioneering work of Ron and colleagues, it was suggested that induction of UPR<sup>mt</sup> may contribute to the extended lifespan of Mit mutants (Ventura and Rea 2007). In support of this hypothesis, RNAi knockdown of *ubl-5* was found to abrogate *isp-1(qm150)* life extension but to not affect wild-type worms nor other long-lived mutants (Durieux and others 2011). Additionally, while *atfs-1* RNAi did not impact the growth of wild-type worms, it impaired both *isp-1(qm150)* and *clk-1(qm30)* Mit mutant development (Nargund and others 2012). It was recently asserted that induction of UPR<sup>mt</sup> is the major mechanism for Mit mutant longevity (Houtkooper and others 2013), yet most data suggests that this cannot be the only mechanism required: Knockdown of *cco-1* in either neuronal, intestinal, or muscle cells induces UPR<sup>mt</sup> in the intestine, but only neuronal or intestinal knockdown prolongs life (Durieux and others 2011). Likewise, while *Phsp-6::GFP* is induced in most Mit mutants, it is not induced in all of them (Ventura and Rea 2007), suggesting that UPR<sup>mt</sup> is not ubiquitously required for Mit mutant longevity.

RNAi knockdown of several nuclear DNA-encoded mitochondrial genes induce UPR<sup>mt</sup> without extending life (Durieux and others 2011) (see also Figure 3A-B), as do the *mev-1(kn1)* and *gas-1(fc21)* mutations (Durieux and others 2011; Pujol and others 2013). Neither the *mev-1(kn-1)* nor *gas-1(fc21)* mutation causes pathology incompatible with longevity as both mutants can become long-lived upon additional gene knockdowns. (This short-lived mutants will be discussed further in section 4.)

If UPR<sup>mt</sup> induction is responsible for the life extension of Mit mutants, then the prediction would be that constitutive activation of UPR<sup>mt</sup> in otherwise wild-type worms should cause them to be long-lived. This has not been the case. A recent screen for worms resistant to statins found mutations in the mitochondrial-targeting sequence of *atfs-1* that caused the transcription factor to be sent into the nucleus regardless of mitochondrial status. As assayed both by transcriptional reporters and increased resistance to mitochondrial stressors such as ethidium bromide, this nuclear-directed ATFS-1 constitutively induced UPR<sup>mt</sup> in these mutants, yet they were less healthy and had reduced lifespan compared with wild-type worms (Rauthan and others 2013). This *atfs-1* gain-of-function mutation suggests that UPR<sup>mt</sup> is not causative of longevity, yet it is important to consider whether this condition truly recapitulates endogenous UPR<sup>mt</sup>. While the transcription factors DVE-1 and UBL-5 have been shown to be integral to endogenous UPR<sup>mt</sup>, they are not downstream of ATFS-1 and are presumably not activated in these mutants. Clearly, the subject of UPR<sup>mt</sup> and its role in Mit mutants and lifespan specification warrants further study.

3.4. AutophagyHaynes and colleagues have proposed three broad responses to mitochondrial dysfunction: UPR<sup>mt</sup> to restore mitochondrial homeostasis, mitophagy to remove defective mitochondria and restore cellular homeostasis, and, finally, apoptosis to eliminate the damaged cell and protect organismal homeostasis. The extent to which each pathway is induced depends on the severity and duration of the problem (Pellegrino and others 2013). We now move from discussion of the first response—UPR<sup>mt</sup>, to the second—autophagy, which encompasses degradation and turnover of numerous cellular components, including the mitochondria (where the process is called mitophagy).

Recent work by Ventura and colleagues studying *frh-1* (the worm ortholog of Frataxin, required for Fe-S cluster assembly), showed that the core autophagy regulating genes *bec-1* and *unc-51* are upregulated upon *frh-1* knockdown (Schiavi and others 2013). Both genes were found to be essential for the increased autophagy and subsequent life extension of this Mit mutant (Schiavi and others 2013). Previously, the Hekimi lab had reported that while autophagy is upregulated in response to *isp-1* and *nuo-6 RNAi*, it is not upregulated in *isp-1(qm150)* and *nuo-6(qm200)* mutants, suggesting that increased autophagy is not a universal requirement for Mit mutant life extension (Yang and Hekimi 2010). Another recent study, however, showed that RNAi knockdown of the autophagy genes *bec-1* and *vps-34* reduced lifespan in *isp-1(qm150)* and *clk-1(qm30)* mutants while having no effect on wild-type worms (Lapierre and others 2013). Additionally, several studies have shown autophagy to also be increased in other long-lived *C. elegans* mutants and required for their life extension as well (Lapierre and others 2013; Toth and others 2008). Hansen and colleagues demonstrated that these long-lived worms not only share increased autophagy, but upregulate it through the same transcription factor, namely, HLH-30, orthologous to mammalian TFEB (Lapierre and others 2013). Transgenic overexpression of *hlh-30* actually increased lifespan of wild-type worms, suggesting that autophagy is not merely differentially required in long-lived worms, but actually plays a causative role in their life extension (Lapierre and others 2013).

The role of autophagy in determining lifespan has been reviewed by Tavernarakis and colleagues (Lionaki and others 2013). Broadly, autophagy has been found to decline with age, and upregulating autophagy is believed to favor longevity through removal of damaged molecules and organelles that can otherwise accumulate and contribute to pathology. Autophagy has been shown to be protective by degrading the aggregate-prone proteins that contribute to neurodegenerative disease (Ravikumar and others 2002).

Autophagy likely also contributes to Mit mutant longevity through its role in lipolysis, another process found to be increased in long-lived worms (Lapierre and others 2012; Schiavi and others 2013). The role of lipid metabolism in determining lifespan has been most studied in worms made long-lived by elimination of the germline precursor cells and several lipases are required for this mode of longevity. One of these lipases is LIPL-4 (Wang and others 2008). Significantly, overexpression of LIPL-4 extends lifespan in otherwise wild-type worms (Wang and others 2008). This lipase is also upregulated under conditions of nutrient deprivation and, like increased autophagy, is dependent on the transcription factor HLH-30 (O'Rourke and Ruvkun 2013).

Returning to the three broad responses that can be activated by dysfunctional mitochondria, it is important to note that each response may facilitate the next if it fails to restore homeostasis. Among the myriad genes activated by ATFS-1 are several involved in mitochondrial fission (Nargund and others 2012), a process which mediates the segregation of damaged mitochondria for degradation by mitophagy (Twig and others 2008). Similarly, autophagy can contribute either to cell survival or cell death (Kourtis and Tavernarakis 2009). Thus, while increasing autophagy can be protective and even extend lifespan in multiple model organisms (Lapierre and others 2013; Pyo and others 2013; Simonsen and others 2008), excessive autophagy can result in cellular damage and death (Kang and others 2007) and contribute to premature aging (Marino and others 2008). Autophagy, like mitochondrial
dysfunction, seems to exhibit a threshold effect, being protective and extending life within a certain range, beyond which, it becomes pathological and life-shortening.

3.5. Metabolism

While the term “metabolism” encompasses many levels of metabolite processing in multicellular organisms, we use it here to refer specifically to the set of cellular reactions that cooperate to convert food sources into biomass and usable energy. The underlying premise for metabolism playing a role in aging is that a change in metabolic configuration may itself be conducive to long life. Several studies have examined changes in metabolism and its role in potentially controlling Mit mutant longevity. These studies can be grouped into two broad approaches: microarray-based analyses and metabolomics.

Of the microarray-based analyses, one of the most extensive was undertaken by Morgan and colleagues (Falk and others 2008): Utilizing whole genome affymetrix microarrays and Gene Set Enrichment Analysis, these authors analyzed eight worm lines in which an ETC subunit was disrupted either genetically (gas-1(fc21), mev-1(kn1), and isp-1(qm150)) or via RNAi (nuo-5, nduf-6, C34B2.8, D2030.4, and Y56A3A.19). Compared to wild-type worms, the six strains in which complex I was disrupted all showed upregulation of genes involved in oxidative phosphorylation (OXPHOS), pyruvate metabolism, fat metabolism, glycolysis, and the TCA cycle, as well as several genes related to stress protection—including glutathione and P450-requiring stress-response pathways. Additionally, a number of genes related to amino acid catabolism were differentially regulated in complex I disruptants. In a related study (Cristina and others 2009), the global transcriptome of the long-lived clk-1(qm30), isp-1(qm150), and cco-1 RNAi-treated worms, representing disruption of ubiquinone biosynthesis, and complexes III and IV, respectively, were examined by microarray analysis. While there was considerable variation in the expression level of individual genes among the strains, gene ontology (GO) term analysis uncovered several processes that were significantly over-represented and were essentially the same metabolic processes found in the aforementioned complex I disruptants (Falk and others 2008). All three Mit mutants upregulated components of their ETC, presumably as a primary compensation for their defects. Fatty acid β-oxidation was also increased. In addition, several alternate energy-generating pathways that bypass all or part of the ETC were upregulated, including glycolysis, the glyoxylate cycle, and glycerol fermentation (Cristina and others 2009). In summary, both studies demonstrate that disruption of the ETC in *C. elegans* is compensated for either by increasing ETC activity and feeding more substrates into the TCA cycle, or by bypassing the ETC altogether and essentially turning the worms into fermenters.

One unexpected finding made by Morgan and colleagues in their study was that, relative to wild-type animals, several amino acids were more abundant in the three genetic mutants studied. This was surprising because mRNA data indicated that the catabolic processes that degrade these amino acids were in fact also elevated. Nevertheless, alanine and the three branched-chain amino acids (leucine, isoleucine, and valine) were increased. Interestingly, glutamate was decreased. This configuration, it was suggested, likely reflects glutamate-mediated transamination of the corresponding α-ketoacids of alanine, valine, leucine, and isoleucine. Normally, degradation of these α-ketoacids requires a functional ETC and so their removal in mitochondrial mutants may sink glutamate from the cell leading to knock-on consequences for metabolism (Falk and others 2008).

In another study, Ebert and colleagues (Zuryn and others 2010) measured a variety of metabolic parameters for a panel of RNAi-induced Mit mutants. By comparing their findings alongside data collected for two different complex II RNAi disruptants, which showed no life extension, Ebert and colleagues showed that neither reduction of whole body ATP levels, decreased brood size, sensitivity to the uncoupler FCCP, reduction in adult volume, nor reduced oxygen consumption differed between the various disruptants, including those that affected complex II. Intriguingly, they also showed that while

genes required for β-oxidation (acs-2), the glyoxylate cycle (gei-7), gluconeogenesis (PEPCK), and glycolysis (gpd-3) were robustly upregulated in RNAi disruptants representing four ETC complexes, their complete abrogation by removal of the transcription factor NHR-49 had no effect on lifespan. These findings argue that despite obvious metabolic readjustments in Mit mutants, some other parameter must be working to extend life.

A clue to what this life-extending parameter might be came from our own recent metabolomics studies. We have described a suite of mass-spectrometry techniques to study the metabolic consequences of mitochondrial dysfunction in C. elegans (Butler and others 2012; Butler and others 2010; Mishur and others 2013). Using these techniques, we identified a collection of compounds, enriched in α-ketoacids and α-hydroxyacids, that were restricted to mitochondrial mutants exhibiting the classic “Mit phenotype” (Butler and others 2013). We posited that these compounds originate with a build-up of NADH in the mitochondria and consequential inhibition of the three α-ketoacid dehydrogenases (namely, α-ketoglutarate dehydrogenase, pyruvate dehydrogenase, and branched-chain ketoacid dehydrogenase). In recent work, we have obtained evidence that the α-ketoacids and α-hydroxyacids that accumulate in Mit mutants contribute to aspects of their phenotype (Mishur and others, manuscript in review). Notably, many of these compounds are structurally related to α-ketoglutarate and some are already known to act as inhibitors of a large family of enzymes called the α-ketoglutarate dependent hydroxylases (Chowdhury and others 2011; Cunliffe and others 1992; Hutchinson and others 2012). One such enzyme is EGL-9, which negatively regulates the hypoxia-inducible transcription factor HIF-1 under normoxic conditions (Epstein and others 2001). As discussed in section 3.2, HIF-1 is active and required for Mit mutant life extension even under normoxic conditions (Khan and others 2013; Lee and others 2010). In light of this, it is significant that exogenous administration of some of the α-ketoacids and α-hydroxyacids that accumulate in Mit mutants stabilize HIF-1 (Mishur and others, manuscript in review).

## 4. NADH, Ubiquinone, and the Mit Phenotype

We will conclude our review of responses to mitochondrial disruption in Mit mutants with a discussion of two of the short-lived ETC mutants as their exception may prove instructive. Life-shortening mutations have been reported in complex I (Grad and Lemire 2004; Hartman and others 2001), complex II (Ishii and others 1990), and complex III (Butler and others 2010). Since severe RNAi-mediated knockdown of some ETC subunits result in larval arrest and reduced lifespan, these life-shortening ETC mutations may likewise be situated at the far end of the response curve (Rea and others 2007). It might be assumed that these mutations produce dysfunction incompatible with longevity, however, like the role of CEP-1 upon atp-3 knockdown (Ventura and others 2009), a closer look at the two best-studied of these mutants suggest this is not the case.

### 4.1 Response to Complex II Disruption

As can be seen in Table 1, disruption of almost any ETC subunit, as well as various mitochondrial ribosomal, metabolite carrier, and chaperone proteins, among others, can all extend life. However, disruption of complex II has never been reported to increase lifespan (Ichimiya and others 2002; Kuang and Ebert 2012; Rea and others 2007). When most forms of mitochondrial disruption can extend life, it is puzzling why complex II should be different.

It has been suggested that the mitochondrial unfolded protein response (UPR<sup>mt</sup>) is responsible for Mit mutant life extension and that this response is triggered by an imbalance between nuclear- and mitochondrial-encoded subunits within respiratory complexes (Houtkooper and others 2013). Since complex II is entirely encoded by nuclear genes, knockdown of its subunits would not be expected to create such an imbalance and this has been proposed as the reason complex II inhibition does not extend life (Houtkooper and others 2013). However, as established by induction of the UPR<sup>mt</sup> reporter

Phsp-6::GFP, this response is, in fact, robustly activated by individual knockdown of three of the four complex II subunits (Figure 3A, see also (Durieux and others 2011; Pujol and others 2013)). A similar response is observed when the Phsp-6::GFP reporter is crossed into the complex II mutant, mev-1(kn1) (unpublished observation).

It is possible that UPR<sup>mt</sup> would extend the lifespan of these complex II disruptants, except that some pathology is simultaneously induced that is life shortening. There is evidence for this in that the mev-1(kn1) mutant is hypersensitive to oxygen (Ishii and others 1990) and generates excessive ROS from complex II (Senoo-Matsuda and others 2001), leading to pathological apoptosis and aging (Senoo-Matsuda and others 2003). Interestingly, evidence suggests that complex II plays a unique role in signaling apoptosis and that high ROS generation is part of this process (Grimm 2013). It could be that complex II disruption triggers this response in addition to the life extending ones. However, these same short-lived mev-1(kn1) mutants can live even longer than wild-type worms following the additional knockdown of mrps-5, which encodes a mitochondrial ribosomal subunit (Houtkooper and others 2013). This suggests that it is not overt pathology that makes mev-1(kn1) worms short-lived, but a failure to induce longevity mechanisms. Alternatively, mrps-5 knockdown may somehow counter the pathology and thereby allow the UPR<sup>mt</sup> and other protective mechanisms to extend life. Our group has previously shown Pgst-4::GFP to be a robust reporter for activation of SKN-1 (Kahn and others 2008), the latter of which, among its many roles, protects against oxidative stress (An and Blackwell 2003). While Pgst-4::GFP was only marginally induced by each of the complex II RNAi that we tested (Figure 3A), it was strongly induced by mrps-5 RNAi (Figure 3C). The SKN-1 antioxidant response induced by mrps-5 knockdown may act to counter the ROS caused by the mev-1(kn1) mutation.

Ebert and colleagues systematically analyzed the effects of RNAi knockdown of each complex I subunit on lifespan, growth rate, fecundity, oxygen consumption, ATP content, mitochondrial membrane potential (∆Ψ<sub>m</sub>), FCCP sensitivity, and gene expression. RNAi knockdown was efficacious, with mRNA levels being reduced to an average of 40% of wild-type, yet none had any effect on lifespan (Kuang and Ebert 2012) (Figure 3B). RNAi knockdown of sdhb-1, mev-1, and sdhd-1 was clearly sufficient to impact mitochondrial function, as evinced by FCCP hypersensitivity, reduced ATP, and diminished egg-laying (Kuang and Ebert 2012). These are the same complex II knockdowns that we have found to activate UPR<sup>mt</sup> (Figure 3A); and reduced egg-viability has been previously reported on both sdhb-1 and mev-1 RNAi (Rea and others 2007). Interestingly, while the reductions in ATP levels were comparable to that observed upon knockdown of subunits of complex I, III, or IV, the impact of complex II disruption on ∆Ψ<sub>m</sub> was much less (Kuang and Ebert 2012). Similarly, the upregulation of genes involved in alternate metabolic pathways was also less than observed in Mit mutants, suggesting that metabolism may be perturbed to a lesser extent, and that the worm is able to adapt without invoking the mechanisms that lead to Mit mutant longevity (Kuang and Ebert 2012).

### 4.2 Response to Complex I Disruption

Ironically, the one instance wherein knockdown of complex II subunits (specifically sdhb-1, mev-1, or sdhd-1) has been reported to extend life is in the otherwise short-lived complex I mutant, gas-1(fc21) (Pujol and others 2013). Using blue native gel electrophoresis, this study showed that the gas-1(fc21) mutation results in a reduction of assembled complex I and thereby diminishes complex I activity. Knocking down complex II apparently corrects this deficit. This loss of complex I accompanied by an upregulation of complex II is not observed in the long-lived complex I mutant, nuo-6(qm200). Taken together, these observations led the authors to hypothesize that it was the upregulation of complex II in gas-1(fc21) mutants that destabilized complex I and caused these worms to be short-lived (Pujol and others 2013). As the authors pointed out, this could explain the previously reported life extension of gas-1(fc21) worms cultured at 15°C (Hartman and others 2001) as a more fragile complex I may be stabilized at lower temperatures. However, it does not explain how an increase in complex II

should destabilize complex I. While most evidence indicates that complex II does not join supercomplex formations (Genova and Lenaz 2013), complex I depends on forming supercomplexes with III and IV to function properly (Acin-Perez and others 2004; Suthammarak and others 2009). As a consequence of this interaction, the complex III mutant isp-1(qm150) also has reduced complex I activity and yet, unlike gas-1(fc21), this mutant is long-lived (Suthammarak and others 2010). Finally, this hypothesis for the gas-1(fc21) mutant's diminished lifespan offers no clear explanation for why other manipulations such as eliminating sod-2 or daf-16 (Suthammarak and others 2013) or culturing the worms on FUDR (Van Raamsdonk and Hekimi 2011) also increase the lifespan of these worms. Indeed, all of these gene knockdowns would seem to cause further stress in the worm, arguing that, like complex II disruptants, gas-1(fc21) are not long-lived because they are able to adapt without inducing the mechanisms that produce longevity in Mit mutants.

### 4.3 Rethinking the Mitochondrial Electron Transport Chain

The NADH and succinate dehydrogenases are not unique in contributing electrons to the mitochondrial ubiquinone pool. Numerous other enzymes do so as well, including electron transfer flavoprotein (ETF) ubiquinone oxidoreductase (which transfers electrons gained from β-oxidation of fatty acids to ubiquinone), dihydroorotate dehydrogenase, glycerol-3-phosphate dehydrogenase, and proline hydroxylase, among others (Figure 4). It is, perhaps, an historical artifact that succinate dehydrogenase is counted as part of the ETC while these other enzymes are not. Indeed, succinate dehydrogenase has more in common with these other matrix-localized enzymes than with the other canonical ETC complexes: It does not pump protons, it is comprised exclusively of nuclear-encoded subunits, and it is not integral to supercomplex formation (Genova and Lenaz 2013; Schagger and Pfeiffer 2001). Succinate dehydrogenase is an enzyme of the TCA cycle and it contributes electrons collected from succinate oxidation to the ubiquinone pool. Crippling succinate dehydrogenase does not directly impair generation of the mitochondrial proton motive force, nor its generation from other electron donors. As summarized in sections 3.5 and 4.1, while metabolism is impacted in complex II disruptants, as one would expect upon impairment of the TCA cycle, it does not cause the wholesale metabolic reprogramming seen in Mit mutants (Butler and others 2013; Kuang and Ebert 2012).

As is the case for complex II, no effect on lifespan has been reported following disruption of the other metabolic pathways contributing electrons to the ubiquinone pool. An exception, however, are genes within the ETF pathway: RNAi knockdown is embryonic lethal (Kamath and others 2003; Simmer and others 2003; Sonnichsen and others 2005) and this may reflect the importance of lipid metabolism to *C. elegans* development. In light of all the above, the question seems to be less, “What is different about complex II among the ETC complexes that its disruption does not result in life extension?” and more, “What is unique about complex I among all the contributors to the ubiquinone pool that its disruption DOES result in life extension?” As has been stated, complex I, like complexes III, IV, and V, pumps protons and contains subunits encoded by mtDNA. What seems key, however, is that all the mitochondrial disruptions that result in life extension are downstream of NADH, suggesting NAD+/NADH ratio as the most proximal cause of Mit mutant longevity, as our recent metabolomics data also suggest (Butler and others 2013) (Mishur and others, manuscript in review). As explained in section 3.5, NADH is normally one of the products generated by the α-ketoacid dehydrogenases. A build-up of NADH will render these enzymes inactive and may be one explanation for the observed accumulation of α-ketoacids and α-hydroxyacids in Mit mutants (Butler and others 2013). These compounds are capable of inhibiting α-ketoglutarate dependent hydroxylases, including EGL-9 which negatively regulates HIF-1, and ultimately may also play an active part in specifying the Mit phenotype (Mishur and others, manuscript in review).

### 5. Conclusion

While the mechanism controlling life extension in Mit mutants was first viewed as a single phenomenon, after 20 years of study, evidence now increasingly indicates that different Mit mutants induce a collection of overlapping, life-promoting mechanisms. Yang and Hekimi once suggested that Mit mutants should be viewed as two separate classes: genetic and RNAi (Yang and Hekimi 2010). However, even among these two broad classes it appears that different mechanisms control life extension, depending on which gene is being knocked down or mutated. This is illustrated by two of the best studied Mit mutants—*clk-1(qm30)* and *isp-1(qm150)*—wherein additional mutations can have very different phenotypic effects: While the lifespan of *clk-1(qm30)* worms is further increased by either the *daf-2(e1370)* mutation (Lakowski and Hekimi 1996) or *sod-2* knockdown (Van Raamsdonk and Hekimi 2009; Yang and others 2007), *isp-1(qm150)* worms show either no effect or life-shortening when exposed to these same treatments (Feng and others 2001; Suthammarak and others 2013; Van Raamsdonk and Hekimi 2009). It could be possible that *clk-1(qm30)* and *isp-1(qm150)* extend life by the same mechanisms and that the opposite effects of additional knockouts/knockdowns are due to where they sit on the “dilution curve” (Rea and others 2007; Van Raamsdonk and Hekimi 2009). However, the fact that *nuo-6* RNAi further extends *isp-1(qm150)* lifespan argues that this mutant cannot already be at the threshold where further impairment is pathological (Yang and Hekimi 2010). Another peculiarity of the *clk-1(qm30)* mutant is that life extension is completely negated by whole gonad ablation (Dillin and others 2002), despite being *daf-16* independent (Lakowski and Hekimi 1996).

Without knowing the potential differences between various Mit mutants, it seems prudent to include more than the *isp-1(qm150)* and *clk-1(qm30)* mutants when testing the role of a particular mechanism in Mit mutant longevity. Especially neglected are the RNAi-induced Mit mutants. A requirement for the various transcription factors is assumed but, with the exception of CEP-1 (Ventura and others 2009), none have been verified in these worms. Even UPR<sup>mt</sup>, which by reporter induction is seen to be strongly activated in most RNAi-induced Mit mutants (Ventura and Rea 2007), has never been formally tested for its role in their life extension. Taken as a whole, current research indicates that there is not one pathway that is solely responsible for Mit mutant longevity but, rather, a network of interacting responses that cooperate to extend life.

## Acknowledgements

Financial support was provided by a fellowship from the Glenn Foundation for Medical Research and a training grant from the NIA (EM). We would also like to thank our reviewers for their thoughtful feedback and helpful recommendations.

References

Acin-Perez, R.; Bayona-Bafaluy, M.P.; Fernandez-Silva, P.; Moreno-Loshuertos, R.; Perez-Martos, A.; Bruno, C.; Moraes, C.T.; Enriquez, J.A. Respiratory complex III is required to maintain complex I in mammalian mitochondria. Molecular cell. 13:805-815; 2004

Altarejos, J.Y.; Montminy, M. CREB and the CRTC co-activators: sensors for hormonal and metabolic signals. Nature reviews Molecular cell biology. 12:141-151; 2011

Altun-Gultekin, Z.; Andachi, Y.; Tsalik, E.L.; Pilgrim, D.; Kohara, Y.; Hobert, O. A regulatory cascade of three homeobox genes, ceh-10, ttx-3 and ceh-23, controls cell fate specification of a defined interneuron class in C. elegans. Development. 128:1951-1969; 2001

An, J.H.; Blackwell, T.K. SKN-1 links C. elegans mesendodermal specification to a conserved oxidative stress response. Genes & development. 17:1882-1893; 2003

Andziak, B.; O'Connor, T.P.; Qi, W.; DeWaal, E.M.; Pierce, A.; Chaudhuri, A.R.; Van Remmen, H.; Buffenstein, R. High oxidative damage levels in the longest-living rodent, the naked mole-rat. Aging cell. 5:463-471; 2006

Baker, B.M.; Nargund, A.M.; Sun, T.; Haynes, C.M. Protective coupling of mitochondrial function and protein synthesis via the eIF2alpha kinase GCN-2. PLoS genetics. 8:e1002760; 2012

Benedetti, C.; Haynes, C.M.; Yang, Y.; Harding, H.P.; Ron, D. Ubiquitin-like protein 5 positively regulates chaperone gene expression in the mitochondrial unfolded protein response. Genetics. 174:229-239; 2006

Bishop, N.A.; Guarente, L. Two neurons mediate diet-restriction-induced longevity in C. elegans. Nature. 447:545-549; 2007

Butler, J.A.; Mishur, R.J.; Bhaskaran, S.; Rea, S.L. A metabolic signature for long life in the Caenorhabditis elegans Mit mutants. Aging cell. 12:130-138; 2013

Butler, J.A.; Mishur, R.J.; Bokov, A.F.; Hakala, K.W.; Weintraub, S.T.; Rea, S.L. Profiling the anaerobic response of C. elegans using GC-MS. PloS one. 7:e46140; 2012

Butler, J.A.; Ventura, N.; Johnson, T.E.; Rea, S.L. Long-lived mitochondrial (Mit) mutants of Caenorhabditis elegans utilize a novel metabolism. FASEB journal : official publication of the Federation of American Societies for Experimental Biology. 24:4977-4988; 2010

Chen, D.; Thomas, E.L.; Kapahi, P. HIF-1 modulates dietary restriction-mediated lifespan extension via IRE-1 in Caenorhabditis elegans. PLoS genetics. 5:e1000486; 2009

Chowdhury, R.; Yeoh, K.K.; Tian, Y.M.; Hillringhaus, L.; Bagg, E.A.; Rose, N.R.; Leung, I.K.; Li, X.S.; Woon, E.C.; Yang, M.; McDonough, M.A.; King, O.N.; Clifton, I.J.; Klose, R.J.; Claridge, T.D.; Ratcliffe, P.J.; Schofield, C.J.; Kawamura, A. The oncometabolite 2-hydroxyglutarate inhibits histone lysine demethylases. EMBO reports. 12:463-469; 2011

Cristina, D.; Cary, M.; Lunceford, A.; Clarke, C.; Kenyon, C. A regulated response to impaired respiration slows behavioral rates and increases lifespan in Caenorhabditis elegans. PLoS genetics. 5:e1000450; 2009

Cunliffe, C.J.; Franklin, T.J.; Hales, N.J.; Hill, G.B. Novel inhibitors of prolyl 4-hydroxylase. 3. Inhibition by the substrate analogue N-oxaloglycine and its derivatives. Journal of medicinal chemistry. 35:2652-2658; 1992

Dell'agnello, C.; Leo, S.; Agostino, A.; Szabadkai, G.; Tiveron, C.; Zulian, A.; Pelle, A.; Roubertoux, P.; Rizzuto, R.; Zeviani, M. Increased longevity and refractoriness to Ca(2+)-dependent neurodegeneration in Surf1 knockout mice. Human molecular genetics. 16:431-444; 2007

Dillin, A.; Hsu, A.L.; Arantes-Oliveira, N.; Lehrer-Graiwer, J.; Hsin, H.; Fraser, A.G.; Kamath, R.S.; Ahringer, J.; Kenyon, C. Rates of behavior and aging specified by mitochondrial function during development. Science. 298:2398-2401; 2002

Durieux, J.; Wolff, S.; Dillin, A. The cell-non-autonomous nature of electron transport chain-mediated longevity. Cell. 144:79-91; 2011

Epstein, A.C.; Gleadle, J.M.; McNeill, L.A.; Hewitson, K.S.; O'Rourke, J.; Mole, D.R.; Mukherji, M.; Metzen, E.; Wilson, M.I.; Dhanda, A.; Tian, Y.M.; Masson, N.; Hamilton, D.L.; Jaakkola, P.; Barstead, R.; Hodgkin, J.; Maxwell, P.H.; Pugh, C.W.; Schofield, C.J.; Ratcliffe, P.J. C. elegans EGL-9 and mammalian homologs define a family of dioxygenases that regulate HIF by prolyl hydroxylation. Cell. 107:43-54; 2001

Falk, M.J.; Zhang, Z.; Rosenjack, J.R.; Nissim, I.; Daikhin, E.; Nissim, I.; Sedensky, M.M.; Yudkoff, M.; Morgan, P.G. Metabolic pathway profiling of mitochondrial respiratory chain mutants in C. elegans. Molecular genetics and metabolism. 93:388-397; 2008

Feng, J.; Bussiere, F.; Hekimi, S. Mitochondrial electron transport is a key determinant of life span in Caenorhabditis elegans. Developmental cell. 1:633-644; 2001

Genova, M.L.; Lenaz, G. Functional role of mitochondrial respiratory supercomplexes. Biochimica et biophysica acta; 2013

Grad, L.I.; Lemire, B.D. Mitochondrial complex I mutations in Caenorhabditis elegans produce cytochrome c oxidase deficiency, oxidative stress and vitamin-responsive lactic acidosis. Human molecular genetics. 13:303-314; 2004

Greer, E.L.; Dowlatshahi, D.; Banko, M.R.; Villen, J.; Hoang, K.; Blanchard, D.; Gygi, S.P.; Brunet, A. An AMPK-FOXO pathway mediates longevity induced by a novel method of dietary restriction in C. elegans. Current biology : CB. 17:1646-1656; 2007

Grimm, S. Respiratory chain complex II as general sensor for apoptosis. Biochimica et biophysica acta. 1827:565-572; 2013

Hansen, M.; Hsu, A.L.; Dillin, A.; Kenyon, C. New genes tied to endocrine, metabolic, and dietary regulation of lifespan from a Caenorhabditis elegans genomic RNAi screen. PLoS genetics. 1:119-128; 2005

Harman, D. Aging: a theory based on free radical and radiation chemistry. Journal of gerontology. 11:298-300; 1956

Hartman, P.S.; Ishii, N.; Kayser, E.B.; Morgan, P.G.; Sedensky, M.M. Mitochondrial mutations differentially affect aging, mutability and anesthetic sensitivity in Caenorhabditis elegans. Mechanisms of ageing and development. 122:1187-1201; 2001

Haynes, C.M.; Fiorese, C.J.; Lin, Y.F. Evaluating and responding to mitochondrial dysfunction: the mitochondrial unfolded-protein response and beyond. Trends in cell biology. 23:311-318; 2013

Haynes, C.M.; Petrova, K.; Benedetti, C.; Yang, Y.; Ron, D. ClpP mediates activation of a mitochondrial unfolded protein response in C. elegans. Developmental cell. 13:467-480; 2007

Haynes, C.M.; Yang, Y.; Blais, S.P.; Neubert, T.A.; Ron, D. The matrix peptide exporter HAF-1 signals a mitochondrial UPR by activating the transcription factor ZC376.7 in C. elegans. Molecular cell. 37:529-540; 2010

Honjoh, S.; Yamamoto, T.; Uno, M.; Nishida, E. Signalling through RHEB-1 mediates intermittent fasting-induced longevity in C. elegans. Nature. 457:726-730; 2009

Houtkooper, R.H.; Mouchiroud, L.; Ryu, D.; Moullan, N.; Katsyuba, E.; Knott, G.; Williams, R.W.; Auwerx, J. Mitonuclear protein imbalance as a conserved longevity mechanism. Nature. 497:451-457; 2013

Hutchinson, S.E.; Leveridge, M.V.; Heathcote, M.L.; Francis, P.; Williams, L.; Gee, M.; Munoz-Muriedas, J.; Leavens, B.; Shillings, A.; Jones, E.; Homes, P.; Baddeley, S.; Chung, C.W.; Bridges, A.; Argyrou, A. Enabling lead discovery for histone lysine demethylases by high-throughput RapidFire mass spectrometry. Journal of biomolecular screening. 17:39-48; 2012

Hwang, A.B.; Lee, S.J. Regulation of lifespan by mitochondrial respiration: the HIF-1 and ROS connection. Aging. 3:304-310; 2011

Ichimiya, H.; Huet, R.G.; Hartman, P.; Amino, H.; Kita, K.; Ishii, N. Complex II inactivation is lethal in the nematode Caenorhabditis elegans. Mitochondrion. 2:191-198; 2002

Ishii, N.; Takahashi, K.; Tomita, S.; Keino, T.; Honda, S.; Yoshino, K.; Suzuki, K. A methyl viologen-sensitive mutant of the nematode Caenorhabditis elegans. Mutation research. 237:165-171; 1990

Ivan, M.; Kondo, K.; Yang, H.; Kim, W.; Valiando, J.; Ohh, M.; Salic, A.; Asara, J.M.; Lane, W.S.; Kaelin, W.G., Jr. HIFAlpha targeted for VHL-mediated destruction by proline hydroxylation: implications for O2 sensing. Science. 292:464-468; 2001

Jiang, H.; Guo, R.; Powell-Coffman, J.A. The Caenorhabditis elegans hif-1 gene encodes a bHLH-PAS protein that is required for adaptation to hypoxia. Proceedings of the National Academy of Sciences of the United States of America. 98:7916-7921; 2001

Kahn, N.W.; Rea, S.L.; Moyle, S.; Kell, A.; Johnson, T.E. Proteasomal dysfunction activates the transcription factor SKN-1 and produces a selective oxidative-stress response in Caenorhabditis elegans. The Biochemical journal. 409:205-213; 2008

Kamath, R.S.; Fraser, A.G.; Dong, Y.; Poulin, G.; Durbin, R.; Gotta, M.; Kanapin, A.; Le Bot, N.; Moreno, S.; Sohrmann, M.; Welchman, D.P.; Zipperlen, P.; Ahringer, J. Systematic functional analysis of the Caenorhabditis elegans genome using RNAi. Nature. 421:231-237; 2003

Kang, C.; You, Y.J.; Avery, L. Dual roles of autophagy in the survival of Caenorhabditis elegans during starvation. Genes & development. 21:2161-2171; 2007

Khan, M.H.; Ligon, M.; Hussey, L.R.; Hufnal, B.; Farber, R., 2nd; Munkacsy, E.; Rodriguez, A.; Dillow, A.; Kahlig, E.; Rea, S.L. TAF-4 is required for the life extension of isp-1, clk-1 and tpk-1 Mit mutants. Aging; 2013

Kim, Y.; Sun, H. Functional genomic approach to identify novel genes involved in the regulation of oxidative stress resistance and animal lifespan. Aging cell. 6:489-503; 2007

Kourtis, N.; Tavernarakis, N. Autophagy and cell death in model organisms. Cell death and differentiation. 16:21-30; 2009

Kuang, J.; Ebert, P.R. The failure to extend lifespan via disruption of complex II is linked to preservation of dynamic control of energy metabolism. Mitochondrion. 12:280-287; 2012

Labuschagne, C.F.; Stigter, E.C.; Hendriks, M.M.; Berger, R.; Rokach, J.; Korswagen, H.C.; Brenkman, A.B. Quantification of in vivo oxidative damage in Caenorhabditis elegans during aging by endogenous F3-isoprostane measurement. Aging cell. 12:214-223; 2013

Lakowski, B.; Hekimi, S. Determination of life-span in Caenorhabditis elegans by four clock genes. Science. 272:1010-1013; 1996

Lane, N.; Martin, W. The energetics of genome complexity. Nature. 467:929-934; 2010

Lapierre, L.R.; De Magalhaes Filho, C.D.; McQuary, P.R.; Chu, C.C.; Visvikis, O.; Chang, J.T.; Gelino, S.; Ong, B.; Davis, A.E.; Irazoqui, J.E.; Dillin, A.; Hansen, M. The TFEB orthologue HLH-30 regulates autophagy and modulates longevity in Caenorhabditis elegans. Nature communications. 4:2267; 2013

Lapierre, L.R.; Melendez, A.; Hansen, M. Autophagy links lipid metabolism to longevity in C. elegans. Autophagy. 8:144-146; 2012

Lee, S.J.; Hwang, A.B.; Kenyon, C. Inhibition of respiration extends C. elegans life span via reactive oxygen species that increase HIF-1 activity. Current biology : CB. 20:2131-2136; 2010

Lee, S.S.; Lee, R.Y.; Fraser, A.G.; Kamath, R.S.; Ahringer, J.; Ruvkun, G. A systematic RNAi screen identifies a critical role for mitochondria in C. elegans longevity. Nature genetics. 33:40-48; 2003

Limongelli, A.; Schaefer, J.; Jackson, S.; Invernizzi, F.; Kirino, Y.; Suzuki, T.; Reichmann, H.; Zeviani, M. Variable penetrance of a familial progressive necrotising encephalopathy due to a novel tRNA(Ile) homoplasmic mutation in the mitochondrial genome. Journal of medical genetics. 41:342-349; 2004

Lin, K.; Dorman, J.B.; Rodan, A.; Kenyon, C. daf-16: An HNF-3/forkhead family member that can function to double the life-span of Caenorhabditis elegans. Science. 278:1319-1322; 1997

Lionaki, E.; Markaki, M.; Tavernarakis, N. Autophagy and ageing: insights from invertebrate model organisms. Ageing research reviews. 12:413-428; 2013

Liu, X.; Jiang, N.; Hughes, B.; Bigras, E.; Shoubridge, E.; Hekimi, S. Evolutionary conservation of the clk-1-dependent mechanism of longevity: loss of mclk1 increases cellular fitness and lifespan in mice. Genes & development. 19:2424-2434; 2005

Marino, G.; Ugalde, A.P.; Salvador-Montoliu, N.; Varela, I.; Quiros, P.M.; Cadinanos, J.; van der Pluijm, I.; Freije, J.M.; Lopez-Otin, C. Premature aging in mice activates a systemic metabolic response involving autophagy induction. Human molecular genetics. 17:2196-2211; 2008

Masoro, E.J. Hormesis and the antiaging action of dietary restriction. Experimental gerontology. 33:61-66; 1998

Miceli, M.V.; Jiang, J.C.; Tiwari, A.; Rodriguez-Quinones, J.F.; Jazwinski, S.M. Loss of mitochondrial membrane potential triggers the retrograde response extending yeast replicative lifespan. Frontiers in genetics. 2:102; 2011

Mishur, R.J.; Butler, J.A.; Rea, S.L. Exometabolomic mapping of Caenorhabditis elegans: a tool to noninvasively investigate aging. Methods in molecular biology. 1048:195-213; 2013

Miyadera, H.; Amino, H.; Hiraishi, A.; Taka, H.; Murayama, K.; Miyoshi, H.; Sakamoto, K.; Ishii, N.; Hekimi, S.; Kita, K. Altered quinone biosynthesis in the long-lived clk-1 mutants of Caenorhabditis elegans. The Journal of biological chemistry. 276:7713-7716; 2001

Nargund, A.M.; Pellegrino, M.W.; Fiorese, C.J.; Baker, B.M.; Haynes, C.M. Mitochondrial import efficiency of ATFS-1 regulates mitochondrial UPR activation. Science. 337:587-590; 2012

O'Rourke, E.J.; Ruvkun, G. MXL-3 and HLH-30 transcriptionally link lipolysis and autophagy to nutrient availability. Nature cell biology. 15:668-676; 2013

Pellegrino, M.W.; Nargund, A.M.; Haynes, C.M. Signaling the mitochondrial unfolded protein response. Biochimica et biophysica acta. 1833:410-416; 2013

Pujol, C.; Bratic-Hench, I.; Sumakovic, M.; Hench, J.; Mourier, A.; Baumann, L.; Pavlenko, V.; Trifunovic, A. Succinate dehydrogenase upregulation destabilize complex I and limits the lifespan of gas-1 mutant. PloS one. 8:e59493; 2013

Pyo, J.O.; Yoo, S.M.; Ahn, H.H.; Nah, J.; Hong, S.H.; Kam, T.I.; Jung, S.; Jung, Y.K. Overexpression of Atg5 in mice activates autophagy and extends lifespan. Nature communications. 4:2300; 2013

Rauthan, M.; Ranji, P.; Aguilera Pradenas, N.; Pitot, C.; Pilon, M. The mitochondrial unfolded protein response activator ATFS-1 protects cells from inhibition of the mevalonate pathway. Proceedings of the National Academy of Sciences of the United States of America. 110:5981-5986; 2013

Ravikumar, B.; Duden, R.; Rubinsztein, D.C. Aggregate-prone proteins with polyglutamine and polyalanine expansions are degraded by autophagy. Human molecular genetics. 11:1107-1117; 2002

Ray, P.D.; Huang, B.W.; Tsuji, Y. Reactive oxygen species (ROS) homeostasis and redox regulation in cellular signaling. Cellular signalling. 24:981-990; 2012

Rea, S.L. Metabolism in the Caenorhabditis elegans Mit mutants. Experimental gerontology. 40:841-849; 2005

Rea, S.L.; Ventura, N.; Johnson, T.E. Relationship between mitochondrial electron transport chain dysfunction, development, and life extension in Caenorhabditis elegans. PLoS biology. 5:e259; 2007

Ristow, M.; Zarse, K. How increased oxidative stress promotes longevity and metabolic health: The concept of mitochondrial hormesis (mitohormesis). Experimental gerontology. 45:410-418; 2010

Robida-Stubbs, S.; Glover-Cutter, K.; Lamming, D.W.; Mizunuma, M.; Narasimhan, S.D.; Neumann-Haefelin, E.; Sabatini, D.M.; Blackwell, T.K. TOR signaling and rapamycin influence longevity by regulating SKN-1/Nrf and DAF-16/FoxO. Cell metabolism. 15:713-724; 2012

Schagger, H.; Pfeiffer, K. The ratio of oxidative phosphorylation complexes I-V in bovine heart mitochondria and the composition of respiratory chain supercomplexes. The Journal of biological chemistry. 276:37861-37867; 2001

Schiavi, A.; Torgovnick, A.; Kell, A.; Megalou, E.; Castelein, N.; Guccini, I.; Marzocchella, L.; Gelino, S.; Hansen, M.; Malisan, F.; Condo, I.; Bei, R.; Rea, S.L.; Braeckman, B.P.; Tavernarakis, N.; Testi, R.; Ventura, N. Autophagy induction extends lifespan and reduces lipid content in response to frataxin silencing in C. elegans. Experimental gerontology. 48:191-201; 2013

Schmeisser, S.; Priebe, S.; Groth, M.; Monajembashi, S.; Hemmerich, P.; Guthke, R.; Platzer, M.; Ristow, M. Neuronal ROS signaling rather than AMPK/sirtuin-mediated energy sensing links dietary restriction to lifespan extension. Molecular Metabolism. 2:92-102; 2013a

Schmeisser, S.; Schmeisser, K.; Weimer, S.; Groth, M.; Priebe, S.; Fazius, E.; Kuhlow, D.; Pick, D.; Einax, J.W.; Guthke, R.; Platzer, M.; Zarse, K.; Ristow, M. Mitochondrial hormesis links low-dose arsenite exposure to lifespan extension. Aging cell. 12:508-517; 2013b

Schmeisser, S.; Zarse, K.; Ristow, M. Lonidamine extends lifespan of adult Caenorhabditis elegans by increasing the formation of mitochondrial reactive oxygen species. Hormone and metabolic research = Hormon- und Stoffwechselforschung = Hormones et metabolisme. 43:687-692; 2011

Senoo-Matsuda, N.; Hartman, P.S.; Akatsuka, A.; Yoshimura, S.; Ishii, N. A complex II defect affects mitochondrial structure, leading to ced-3- and ced-4-dependent apoptosis and aging. The Journal of biological chemistry. 278:22031-22036; 2003

Senoo-Matsuda, N.; Yasuda, K.; Tsuda, M.; Ohkubo, T.; Yoshimura, S.; Nakazawa, H.; Hartman, P.S.; Ishii, N. A defect in the cytochrome b large subunit in complex II causes both superoxide anion overproduction and abnormal energy metabolism in Caenorhabditis elegans. The Journal of biological chemistry. 276:41553-41558; 2001

Simmer, F.; Moorman, C.; van der Linden, A.M.; Kuijk, E.; van den Berghe, P.V.; Kamath, R.S.; Fraser, A.G.; Ahringer, J.; Plasterk, R.H. Genome-wide RNAi of C. elegans using the hypersensitive rrf-3 strain reveals novel gene functions. PLoS biology. 1:E12; 2003

Simonsen, A.; Cumming, R.C.; Brech, A.; Isakson, P.; Schubert, D.R.; Finley, K.D. Promoting basal levels of autophagy in the nervous system enhances longevity and oxidant resistance in adult Drosophila. Autophagy. 4:176-184; 2008

Sonnichsen, B.; Koski, L.B.; Walsh, A.; Marschall, P.; Neumann, B.; Brehm, M.; Alleaume, A.M.; Artelt, J.; Bettencourt, P.; Cassin, E.; Hewitson, M.; Holz, C.; Khan, M.; Lazik, S.; Martin, C.; Nitzsche, B.; Ruer, M.; Stamford, J.; Winzi, M.; Heinkel, R.; Roder, M.; Finell, J.; Hantsch, H.; Jones, S.J.; Jones, M.; Piano, F.; Gunsalus, K.C.; Oegema, K.; Gonczy, P.; Coulson, A.; Hyman, A.A.; Echeverri, C.J. Full-genome RNAi profiling of early embryogenesis in Caenorhabditis elegans. Nature. 434:462-469; 2005

Suthammarak, W.; Morgan, P.G.; Sedensky, M.M. Mutations in mitochondrial complex III uniquely affect complex I in Caenorhabditis elegans. The Journal of biological chemistry. 285:40724-40731; 2010

Suthammarak, W.; Somerlot, B.H.; Opheim, E.; Sedensky, M.; Morgan, P.G. Novel Interactions between Mitochondrial Superoxide Dismutases and the Electron Transport Chain. Aging cell; 2013

Suthammarak, W.; Yang, Y.Y.; Morgan, P.G.; Sedensky, M.M. Complex I function is defective in complex IV-deficient Caenorhabditis elegans. The Journal of biological chemistry. 284:6425-6435; 2009

Torgovnick, A.; Schiavi, A.; Testi, R.; Ventura, N. A role for p53 in mitochondrial stress response control of longevity in C. elegans. Experimental gerontology. 45:550-557; 2010

Toth, M.L.; Sigmond, T.; Borsos, E.; Barna, J.; Erdelyi, P.; Takacs-Vellai, K.; Orosz, L.; Kovacs, A.L.; Csikos, G.; Sass, M.; Vellai, T. Longevity pathways converge on autophagy genes to regulate lifespan in Caenorhabditis elegans. Autophagy. 4:330-338; 2008
Tsang, W.Y.; Lemire, B.D. Mitochondrial genome content is regulated during nematode development. Biochemical and biophysical research communications. 291:8-16; 2002

Tullet, J.M.; Hertweck, M.; An, J.H.; Baker, J.; Hwang, J.Y.; Liu, S.; Oliveira, R.P.; Baumeister, R.; Blackwell, T.K. Direct inhibition of the longevity-promoting factor SKN-1 by insulin-like signaling in C. elegans. Cell. 132:1025-1038; 2008

Twig, G.; Elorza, A.; Molina, A.J.; Mohamed, H.; Wikstrom, J.D.; Walzer, G.; Stiles, L.; Haigh, S.E.; Katz, S.; Las, G.; Alroy, J.; Wu, M.; Py, B.F.; Yuan, J.; Deeney, J.T.; Corkey, B.E.; Shirihai, O.S. Fission and selective fusion govern mitochondrial segregation and elimination by autophagy. The EMBO journal. 27:433-446; 2008

Uno, M.; Honjoh, S.; Matsuda, M.; Hoshikawa, H.; Kishimoto, S.; Yamamoto, T.; Ebisuya, M.; Yamamoto, T.; Matsumoto, K.; Nishida, E. A fasting-responsive signaling pathway that extends lifespan in C. elegans. Cell reports. 3:79-91; 2013

Van Raamsdonk, J.M.; Hekimi, S. Deletion of the mitochondrial superoxide dismutase sod-2 extends lifespan in Caenorhabditis elegans. PLoS genetics. 5:e1000361; 2009

Van Raamsdonk, J.M.; Hekimi, S. FUDR causes a twofold increase in the lifespan of the mitochondrial mutant gas-1. Mechanisms of ageing and development. 132:519-521; 2011

Ventura, N.; Rea, S.L. Caenorhabditis elegans mitochondrial mutants as an investigative tool to study human neurodegenerative diseases associated with mitochondrial dysfunction. Biotechnology journal. 2:584-595; 2007

Ventura, N.; Rea, S.L.; Schiavi, A.; Torgovnick, A.; Testi, R.; Johnson, T.E. p53/CEP-1 increases or decreases lifespan, depending on level of mitochondrial bioenergetic stress. Aging cell. 8:380-393; 2009

Walker, A.K.; Rothman, J.H.; Shi, Y.; Blackwell, T.K. Distinct requirements for C.elegans TAF(II)s in early embryonic transcription. The EMBO journal. 20:5269-5279; 2001

Walter, L.; Baruah, A.; Chang, H.W.; Pace, H.M.; Lee, S.S. The homeobox protein CEH-23 mediates prolonged longevity in response to impaired mitochondrial electron transport chain in C. elegans. PLoS biology. 9:e1001084; 2011

Wang, M.C.; O'Rourke, E.J.; Ruvkun, G. Fat metabolism links germline stem cells and longevity in C. elegans. Science. 322:957-960; 2008

Wolff, S.; Dillin, A. The trifecta of aging in Caenorhabditis elegans. Experimental gerontology. 41:894-903; 2006

Wong, A.; Boutis, P.; Hekimi, S. Mutations in the clk-1 gene of Caenorhabditis elegans affect developmental and behavioral timing. Genetics. 139:1247-1259; 1995

Yang, W.; Hekimi, S. Two modes of mitochondrial dysfunction lead independently to lifespan extension in Caenorhabditis elegans. Aging cell. 9:433-447; 2010

Yang, W.; Li, J.; Hekimi, S. A Measurable increase in oxidative damage due to reduction in superoxide detoxification fails to shorten the lifespan of long-lived mitochondrial mutants of Caenorhabditis elegans. Genetics. 177:2063-2074; 2007

Yoneda, T.; Benedetti, C.; Urano, F.; Clark, S.G.; Harding, H.P.; Ron, D. Compartment-specific perturbation of protein handling activates genes encoding mitochondrial chaperones. Journal of cell science. 117:4055-4066; 2004

Zarse, K.; Schmeisser, S.; Groth, M.; Priebe, S.; Beuster, G.; Kuhlow, D.; Guthke, R.; Platzer, M.; Kahn, C.R.; Ristow, M. Impaired insulin/IGF1 signaling extends lifespan by promoting mitochondrial L-proline catabolism to induce a transient ROS signal. Cell metabolism. 15:451-465; 2012

Zhang, Y.; Shao, Z.; Zhai, Z.; Shen, C.; Powell-Coffman, J.A. The HIF-1 hypoxia-inducible factor modulates lifespan in C. elegans. PloS one. 4:e6348; 2009
Zuryn, S.; Kuang, J.; Tuck, A.; Ebert, P.R. Mitochondrial dysfunction in Caenorhabditis elegans causes metabolic restructuring, but this is not linked to longevity. Mechanisms of ageing and development. 131:554-561; 2010

Figure 1. Identity and Location of Mitochondrial Respiratory Complex Subunits. (A) Schematic of Complex I (core subunits only). (B-E) Schematics of complexes II-V, respectively. All complexes are oriented such that their matrix-facing surface is at the top. Large text - human subunit; small text - orthologous C. elegans subunit(s); $ mitochondrial DNA encoded; * no known C. elegans ortholog.

Figure 2. Activation of the Mitochondrial Unfolded Protein Response in C. elegans. In the mitochondrial matrix, misfolded proteins are degraded to peptides by the protease CLPP-1. These peptide fragments are directed out of the matrix and into the cytosol by HAF-1. This efflux prevents the import of the transcription factor ATFS-1 into the matrix where it is normally degraded and instead redirects it into the nucleus. Two additional transcription factors — DVE-1 and UBL-5 — form a complex which cooperates with ATFS-1 to induce the mitochondrial unfolded protein response. While the response of DVE-1 depends on CLPP-1 activation, and the upregulation of UBL-5 on HAF-1 function, the signaling mechanisms involved that allow each proteins to reach the nucleus, independent of ATFS-1 activation, has not been deciphered.

Figure 3. Robust Activation of UPR$^{\mathrm{mt}}$ Signaling Following Complex II Disruption is Not Sufficient to Extend C. elegans Lifespan. (A) RNAi knockdown of three out of four complex II subunits strongly activates UPR$^{\mathrm{mt}}$ (marked by a Phsp-6::GFP transcriptional reporter) but the on SKN-1 activation (marked by a Pgst-4::GFP transcriptional reporter) is minimal. Data for sdha-1 was not available. (B) RNAi knockdown of any complex II subunit in wild-type (N2) animals has no effect on lifespan (data from Figure 1b, Kuang et. al. 2012). (C) Unlike disruption of complex II subunits, life-extending RNAi knockdown of mprs-5 strongly activates both SKN-1 and UPR$^{\mathrm{mt}}$ signaling pathways (control worms same as in panel A).

Figure 4. Mit Mutant Phenotype Arises from Disruption of the Core Electron Transport Pathway Comprised of Complexes I, III, IV and V. At least eight pathways contribute electrons to the ubiquinone pool in mitochondria. Of these, only complex I pumps protons and contains subunits encoded by mitochondrial DNA.

Figure 1

![](image.png)

Figure 2

Figure 3

A
Pgst-4::GFP
(SKN-1 reporter)
Phsp-6::GFP
(UPR reporter)

vector
sdha-2
sdhb-1
mov-1
(sdhc-1)
sdhd-1

B
| Mean Lifespan ± SEM (Days) | p-Value (Log Rank) | Total (n) |
|---------------------------|--------------------|-----------|
| vector                    | 15.4 ± 0.7         | -          | 74        |
| sdha-1                    | 14.9 ± 0.7         | 0.6887     | 71        |
| sdha-2                    | 14.4 ± 0.6         | 0.3993     | 56        |
| sdhb-1                    | 15.9 ± 0.9         | 0.7640     | 81        |
| mov-1                     | 14.6 ± 0.5         | 0.1333     | 56        |
| sdhd-1                    | 16.0 ± 0.5         | 0.6547     | 84        |

C
Pgst-4::GFP
(SKN-1 reporter)
Phsp-6::GFP
(UPR reporter)

mrps-5

ACCEPTED MANUSCRIPT

NADH
NAD⁺
succinate
fumarate
proline
(S)-1-pyrroline-
5-carboxylate
acyl-CoA
2,3-enoyl-CoA

TCA Cycle (COMPLEX II)
succinate dehydrogenase

Proline Metabolism
proline dehydrogenase

Fatty Acid β-Oxidation
MCAD → ETF → ETF:QO

COMPLEX I
(NADH:ubiquinone oxidoreductase)

ubiquinone

COMPLEX III
(cytochrome c reductase)

cytochrome c

COMPLEX IV
(cytochrome c oxidase)

COMPLEX V
(ATP synthase)

Uracil Biosynthesis
dihydroorotate dehydrogenase

Sulfur Metabolism
sulfide:quinone oxidoreductase

Lipid Biosynthesis/Glycolysis
glycerol-3-phosphate dehydrogenase

Glycerophospholipid
Metabolism
Choline dehydrogenase

(S)-4,5-dihydro-orotate
orotate

sulfide
thiosulfate

dihydroxyacetone phosphate
sn-glycerol 3-phosphate

choline
betaine aldehyde

ADP + Pᵢ
ATP
H⁺

Figure 4

Table 1

| Human Ortholog | Life Extension* | Method of Knockdown | Reference** |
|-----------------|------------------|----------------------|-------------|
| COMPLEX I      |                  |                      |             |
| K09A 9.5(gas-1) | NDUFS 2          | decrease             | genetic     | [1, 2, 3†] |
| ZK 973.10(lpd-5) | NDUFS 4          | √                    | RNAi        | [4]         |
| T26A 5.3(nduf-2.2) | NDUFS 2          | √                    | RNAi        | [4]         |
| Y54E10BL.5(nduf-5) | NDUFS 5          |                      |             |             |
| F22D 6.4(nduf-6) | NDUFS 6          |                      |             |             |
| W10D 5.2(nduf-7) | NDUFS 7          |                      |             |             |
| C09H 10.3(nuo-1) | NDUFV 1          | √                    | genetic&RNAi | [5, 6†, 7†, 8] |
| T10E 9.7(nuo-2) | NDUFS 3          | √                    | RNAi        | [9, 10, 11]  |
| Y57G11C.12(nuo-3) | NDUFA 6          | √                    | RNAi        | [4, 7†, 10]  |
| K04G 7.4(nuo-4) | NDUFA 10         | √                    | RNAi        | [4, 10, 12†] |
| Y45G12B.1(nuo-5) | NDUFS 1          | √                    | RNAi        | [10]        |
| W01A 8.4(nuo-6) | NDUFB 4          | √                    | genetic&RNAi | [13]        |
| C16A 3.5       | NDUFB 9          |                      |             |             |
| C18E 9.4       | NDUFB 3          | √                    | RNAi        | [7†]        |
| C25H 3.9       | NDUFB 5          | √                    | RNAi        | [7†]        |
| C33A 12.1      | NDUFA 5          | √                    | RNAi        | [4, 7†]     |
| C34B 2.8       | NDUFA 13         |                      |             |             |
| D2030.4        | NDUFB 7          | √                    | RNAi        | [12†, 14†]  |
| F31D 4.9       | NDUFA 1          |                      |             |             |
| F37C 12.3      | NDUFAB 1         |                      |             |             |
| F42G 8.10      | NDUFB 11         |                      |             |             |
| F44G 4.2       | NDUFB 2          |                      |             |             |
| F45H 10.3      | NDUFA 7          |                      |             |             |
| F53F 4.10      | NDUFV 2          |                      |             |             |
| F59C 6.5       | NDUFB 10         | √                    | RNAi        | [5†]        |
| T20H 4.5       | NDUFS 8          | √                    | RNAi        | [4, 12†]    |
| Y51H 1A.3      | NDUFB 8          |                      |             |             |
| Y53G 8AL.2     | NDUFA 9          | √                    | RNAi        | [4]         |
| Y54F 10AM.5    | NDUFA 8          |                      |             |             |
| Y56A 3A.19     | NDUFAB 1         | √                    | RNAi        | [4, 5†]     |
| Y63D 3A.7      | NDUFA 2          |                      |             |             |
| Y71H 2AM.4     | NDUFC 2          | √                    | RNAi        | [4]         |
| Y94H 6A.8      | NDUFA 12         |                      |             |             |
| ZK 809.3       | NDUFB 6          | √                    | RNAi        | [4]         |

COMPLEX II

ACCEPTED MANUSCRIPT

| C03G 5.1(sdha-1) | SDHA | no effect | RNAi | [2, 15+] |
| --- | --- | --- | --- | --- |
| C34B 2.7(sdha-2) | SDHA | no effect | RNAi | [2, 15+] |
| F42A 8.2(sdhb-1) | SDHB | no effect | RNAi | [2, 7+, 15+] |
| T07C 4.7(mev-1) | SDHC | decrease | genetic & RNAi | [2, 11, 15+, 16] |
| F33A 8.5(sdhd-1) | SDHD | no effect | RNAi | [2, 15+] |

COMPLEX III

| C54G 4.8(cyc-1) | CYC 1 | √ | RNAi | [7+, 9, 10, 17] |
| --- | --- | --- | --- | --- |
| E04A 4.7(cyc-2.1) | CYCS (cytochrome c) | √ | RNAi | [4] |
| ZC 116.2(cyc-2.2) | CYCS (cytochrome c) |  |  |  |
| F42G 8.12(isp-1) | UQCRFS 1 | √ | genetic & RNAi | [11, 17] |
| F56D 2.1(ucr-1) | UQCRC 1 | √ | RNAi | [4, 7+] |
| VW06B3R.1(ucr-2.1) | UQCR 2 |  |  |  |
| T10B 10.2(ucr-2.2) | UQCR 2 | no effect | RNAi | [7+] |
| T24C 4.1(ucr-2.3) | UQCR 2 | decrease | genetic & RNAi | [7+, 18] |
| F45H 10.2 | UQCRQ | √ | RNAi | [4] |
| F57B 10.14 | UQCR 11 |  |  |  |
| R07E4.3 | UQCRQ | √ | RNAi | [7+] |
| T02H6.11 | UQCRB | √ | RNAi | [7+, 14+] |
| T27E9.2 | UQCRH | √ | RNAi | [7+] |

COMPLEX IV

| F26E 4.9(cco-1) | COX5B | √ | RNAi | [4, 7+, 9, 10, 11, 12+, 14+, 17] |
| --- | --- | --- | --- | --- |
| Y37D8A.14(cco-2) | COX5A | √ | RNAi | [7+, 10, 19] |
| F26E 4.6(cco-4) | COX7C | √ | RNAi | [5+, 7+, 12+, 14+] |
| F29C 4.2 | COX6C | √ | RNAi | [4, 7+] |
| F40G 9.2 | COX 17 |  |  |  |
| F54D 8.2 | COX6A 1 | √ | RNAi | [4, 7+] |
| JC8.5 | COX 11 |  |  |  |
| T06D 8.5 | COX 15 | √ | RNAi | [14] |
| W09C 5.8 | COX 4 | √ | RNAi | [4, 7+, 12, 14] |
| Y46G 5A.2 | COX 10 |  |  |  |
| Y71H2AM.5 | COX6B | √ | RNAi | [7+] |

COMPLEX V

| F35G 12.10(asb-1) | ATP5F 1(b) |  |  |  |
| --- | --- | --- | --- | --- |
| F02E 8.1(asb-2) | ATP5F 1(b) | √ | RNAi | [10] |
| K07A 12.3(asg-1) | ATP5L(g) |  |  |  |
| C53B 7.4(asg-2) | ATP5L(g) | √ | RNAi | [12+] |

C34E 10.6(atp-2)    ATP5B(β)     √   genetic&RNAi [5, 6†]
F27C 1.7(atp-3)    ATP5O(OSCP)  √   RNAi        [9, 10, 11]
T05H 4.12(atp-4)   ATP5J(F 6)   √   RNAi        [10]
C06H 2.1(atp-5)    ATP5H(d)     √   RNAi        [10]
F32D 1.2(hpo-18)   ATP5E(ε)     
F58F 12.1          ATP5D(δ)     
H28O 16.1          ATP5A 1(α)    √   RNAi        [5]
R04F 11.2          ATP5I(e)     
R05D 3.6           ATP5E(ε)     
R53.4              ATP5J 2(f)   
T26E 3.7           ATP5A 1(α)   
Y69A 2AR.18        ATP5C 1(γ)   
Y82E 9BR.3         ATP5G 3(c)   
ZC262.5            ATP5E(ε)     

OTHER±

T06D 8.6(cchl-1)   HCCS      √   RNAi        [4]
ZC 395.2(clk-1)    COQ 7     √   genetic     [20]
F59G 1.7(frh-1)    FXN       √   genetic&RNAi [11, 21, 22]
ZC 395.6(gro-1)    TRIT 1    √   genetic     [23]
C37H 5.8(hsp-6)    mtHSP 70  √   RNAi        [22]
ZK 524.3(lrs-2)    LARS 2    √   genetic     [14†]
T21C 9.1(mics-1)   OMP 25    √   genetic&RNAi [24†]
F56B 3.8(mrpl-2)   MRPL 2    √   RNAi        [25†]
W09D 10.3(mrpl-12) MRPL 12   √   RNAi        [4]
Y48E 1B.5(mrpl-37) MRPL 37   √   RNAi        [25†]
B0261.4(mrpl-47)   MRPL 47   √   RNAi        [14†]
E02A 10.1(mrps-5)  MRPS 5    √   RNAi        [25†]
F09G 8.3(mrps-9)   MRPS 9    √   RNAi        [4]
Y37D 8A.18(mrps-10)MRPS 10   √   RNAi        [4]
F21D 5.8(mrps-33)  MRPS 33   √   RNAi        [4]
F43E 2.7(mtch-1)   MTCH 1    √   RNAi        [5†]
F10D 11.1(sod-2)   SOD 2(MnSOD) √   genetic     [26†]
C08A 9.1(sod-3)    SOD 2(MnSOD) no effect genetic [26†]
ZK 637.9(tpk-1)    TPK 1     √   genetic     [4]
K08F 11.4(yars-1)  YARS 2    √   RNAi        [4]
F13G 3.7          SLC25A 44  √   RNAi        [14†]
K01C 8.7          SLC25A 32  √   RNAi        [14†]

*Blank space indicates nothing has been reported to date. *List is not exhaustive. **Numbers refer to references
listed below.‡FUdR used.

1. Hartman, P.S., et al., Mitochondrial mutations differentially affect aging, mutability and anesthetic sensitivity in Caenorhabditis elegans. Mech Ageing Dev, 2001. 122(11): p. 1187-201.
2. Pujol, C., et al., Succinate dehydrogenase upregulation destabilize complex I and limits the lifespan of gas-1 mutant. PLoS One, 2013. 8(3): p. e59493.
3. Van Raamsdonk, J.M. and S. Hekimi, FUDR causes a twofold increase in the lifespan of the mitochondrial mutant gas-1. Mech Ageing Dev, 2011. 132(10): p. 519-21.
4. Kim, Y. and H. Sun, Functional genomic approach to identify novel genes involved in the regulation of oxidative stress resistance and animal lifespan. Aging Cell, 2007. 6(4): p. 489-503.
5. Curran, S.P. and G. Ruvkun, Lifespan regulation by evolutionarily conserved genes essential for viability. PLoS Genet, 2007. 3(4): p. e56.
6. Tsang, W.Y., et al., Mitochondrial respiratory chain deficiency in Caenorhabditis elegans results in developmental arrest and increased lifespan. J Biol Chem, 2001. 276(34): p. 32240-6.
7. Zuryn, S., et al., Mitochondrial dysfunction in Caenorhabditis elegans causes metabolic restructuring, but this is not linked to longevity. Mech Ageing Dev, 2010. 131(9): p. 554-61.
8. Grad, L.I. and B.D. Lemire, Mitochondrial complex I mutations in Caenorhabditis elegans produce cytochrome c oxidase deficiency, oxidative stress and vitamin-responsive lactic acidosis. Hum Mol Genet, 2004. 13(3): p. 303-14.
9. Dillin, A., et al., Rates of behavior and aging specified by mitochondrial function during development. Science, 2002. 298(5602): p. 2398-401.
10. Hansen, M., et al., New genes tied to endocrine, metabolic, and dietary regulation of lifespan from a Caenorhabditis elegans genomic RNAi screen. PLoS Genet, 2005. 1(1): p. 119-28.
11. Rea, S.L., N. Ventura, and T.E. Johnson, Relationship between mitochondrial electron transport chain dysfunction, development, and life extension in Caenorhabditis elegans. PLoS Biol, 2007. 5(10): p. e259.
12. Hamilton, B., et al., A systematic RNAi screen for longevity genes in C. elegans. Genes Dev, 2005. 19(13): p. 1544-55.
13. Yang, W. and S. Hekimi, Two modes of mitochondrial dysfunction lead independently to lifespan extension in Caenorhabditis elegans. Aging Cell, 2010. 9(3): p. 433-47.
14. Lee, S.S., et al., A systematic RNAi screen identifies a critical role for mitochondria in C. elegans longevity. Nat Genet, 2003. 33(1): p. 40-8.
15. Kuang, J. and P.R. Ebert, The failure to extend lifespan via disruption of complex II is linked to preservation of dynamic control of energy metabolism. Mitochondrion, 2012. 12(2): p. 280-7.
16. Ishii, N., et al., A methyl viologen-sensitive mutant of the nematode Caenorhabditis elegans. Mutat Res, 1990. 237(3-4): p. 165-71.
17. Feng, J., F. Bussiere, and S. Hekimi, Mitochondrial electron transport is a key determinant of lifespan in Caenorhabditis elegans. Dev Cell, 2001. 1(5): p. 633-44.
18. Butler, J.A., et al., Long-lived mitochondrial (Mit) mutants of Caenorhabditis elegans utilize a novel metabolism. FASEB J, 2010. 24(12): p. 4977-88.
19. Suthammarak, W., et al., Complex I function is defective in complex IV-deficient Caenorhabditis elegans. J Biol Chem, 2009. 284(10): p. 6425-35.
20. Wong, A., P. Boutis, and S. Hekimi, Mutations in the clk-1 gene of Caenorhabditis elegans affect developmental and behavioral timing. Genetics, 1995. 139(3): p. 1247-59.
21. Ventura, N., et al., Reduced expression of frataxin extends the lifespan of Caenorhabditis elegans. Aging Cell, 2005. 4(2): p. 109-12.
22. Ventura, N. and S.L. Rea, Caenorhabditis elegans mitochondrial mutants as an investigative tool to study human neurodegenerative diseases associated with mitochondrial dysfunction. Biotechnol J, 2007. 2(5): p. 584-95.

23. Lakowski, B. and S. Hekimi, Determination of life-span in Caenorhabditis elegans by four clock genes. Science, 1996. 272(5264): p. 1010-3.
24. Hoffmann, M., et al., MICS-1 interacts with mitochondrial ATAD-3 and modulates lifespan in C. elegans. Exp Gerontol, 2012. 47(3): p. 270-5.
25. Houtkooper, R.H., et al., Mitonuclear protein imbalance as a conserved longevity mechanism. Nature, 2013. 497(7450): p. 451-7.
26. Van Raamsdonk, J.M. and S. Hekimi, Deletion of the mitochondrial superoxide dismutase sod-2 extends lifespan in Caenorhabditis elegans. PLoS Genet, 2009. 5(2): p. e1000361.

Highlights

We examine responses to mitochondrial dysfunction in *C. elegans* that may extend life.

We discuss ROS signaling, UPR<sup>mt</sup>, autophagy, metabolism, and transcription factors.

The consequences of complexes I versus complex II disruption are explored.
