
Journal of
Medicinal
Chemistry

Nucleotide-Binding Oligomerization Domain (NOD) Inhibitors: A
Rational Approach toward Inhibition of NOD Signaling Pathway

Ziga Jakopin*

Faculty of Pharmacy, University of Ljubljana, Aškerčeva 7, SI-1000 Ljubljana, Slovenia

---

**ABSTRACT:** Dysregulation of nucleotide-binding oligomerization domains 1 and 2 (NOD1 and NOD2) has been implicated in the pathology of various inflammatory disorders, rendering them and their downstream signaling proteins potential therapeutic targets. Selective inhibition of NOD1 and NOD2 signaling could be advantageous in treating many acute and chronic diseases; therefore, harnessing the full potential of NOD inhibitors is a key topic in medicinal chemistry. Although they are among the best studied NOD-like receptors (NLRs), the therapeutic potential of pharmacological modulation of NOD1 and NOD2 is largely unexplored. This review is focused on the scientific progress in the field of NOD inhibitors over the past decade, including the recently reported selective inhibitors of NOD1 and NOD2. In addition, the potential approaches to inhibition of NOD signaling as well as the advantages and disadvantages linked with inhibition of NOD signaling are discussed. Finally, the potential directions for drug discovery are also discussed.

---

### 1. INTRODUCTION

Humans and other multicellular organisms are constantly challenged by an enormous diversity of pathogens, including viruses, bacteria, and fungi. These are, for the most part, neutral or beneficial to the host but may cause infectious diseases. A complex network, the innate immune system, that is able to detect and respond to such threats has evolved in all multicellular organisms, while mammals have additionally developed adaptive immunity. The initial defense response is orchestrated by evolutionarily conserved sentinels, designated pattern recognition receptors (PRRs), that are highly expressed in innate immune cells like macrophages and dendritic cells but also in epithelial cells and fibroblasts. They are located on the cell surface or in the cytosol and detect conserved pathogen-associated molecular patterns (PAMPs), such as lipopolysaccharide (LPS), peptidoglycan (PGN), nucleic acids, and flagellin, that are usually present in pathogens but not in the host. On recognition of their respective ligands, PRRs trigger signaling cascades, mediated mostly by activation of the transcription factor nuclear factor κB (NF-κB) and mitogen-associated protein kinases (MAPKs), that lead to inflammatory responses. PRR subfamilies comprise Toll-like receptors (TLRs) and the C-type lectin receptors that are membrane-bound and usually located extracellularly but also present in the intracellular vesicular compartment (in the cases of TLR3, -7, -8, and -9), while nucleotide-binding oligomerization domain (NOD) like receptors (NLRs) and RIG-I-like receptors (RLRs) monitor only the intracellular milieu. In addition, the innate immune system detects endogenous products of tissue injury (DNA, ATP, uric acid), named danger-associated molecular patterns (DAMPs), that are released in the circulation following injury or cell rupture. The innate immune system in this way maintains an appropriate homeostasis between overstimulation and suppression of host immune responses.

---

NLRs are cytoplasmic proteins with a characteristic tripartite domain architecture comprising a C-terminal sensor domain, a centrally located nucleotide-binding domain (NACHT), and an N-terminal effector domain. The 23 identified mammalian NLRs have been divided into subgroups based on the structural motif of an N-terminal domain that consists of caspase-recruitment domains (CARDs), pyrin domains, or baculovirus inhibitory repeats. NLRs are further classified into two groups, depending on their function: inflammasome and nodosome NLRs.

NOD1 and NOD2, which belong to the nodosome subgroup of NLRs, play the main role in host defense, since they are the only ones that recognize bacterial peptidoglycan components, which leads to activation of NF-κB and MAPKs, resulting in an inflammatory response. This response, however, is under tight supervision of numerous regulators that keep it under control. Mutations in the NOD1 and NOD2 genes and ligand-elicited overactivation have both been shown to cause dysregulation of NLR function, which is linked to a variety of human diseases, including inflammatory disorders and cancer. This review is focused on the largely unexplored therapeutic potential of pharmacological modulation of NOD1 and NOD2 signaling and on the medicinal chemistry of NOD inhibitors.

---

### 2. NOD1 AND NOD2

#### 2.1. Structure
NOD1 and NOD2 have a tripartite domain architecture composed of (i) a C-terminal sensor domain consisting of leucine-rich repeats (LRRs) that supposedly mediate the recognition of ligands, (ii) a centrally located nucleotide-binding domain NACHT that mediates self-oligomerization and activation, and (iii) N-terminal effector domains,

---

**Received:** November 28, 2013  
**Published:** April 7, 2014  

© 2014 American Chemical Society  
6897  
dx.doi.org/10.1021/jm401841p | J. Med. Chem. 2014, 57, 6897–6918

Journal of Medicinal Chemistry

consisting of one (NOD1) or two CARD domains (NOD2) that  
mediate protein–protein interactions and are therefore essential  
for the transduction of signal (Figure 1).⁴  

![Figure 1. NOD protein domain architecture.](https://i.imgur.com/placeholder.png)

Like all NLRs, NOD1 and NOD2 belong to the AAA+  
(extended ATPase associated with various cellular activities)  
ATPase family of proteins, where they are further classified as  
signal transduction ATPases with numerous domains  
(STAND).⁶⁻⁸ The AAA+ proteins have in common a  
structurally conserved ATPase domain that assembles into  
oligomeric rings and converts the chemical energy of nucleotide  
binding/hydrolysis into conformational changes. The ATPase  
domain of most AAA+ proteins consists of several characteristic  
and conserved features, namely, the Walker A and Walker B  
boxes and the Sensor 1 and Sensor 2 motifs, which are involved in  
nucleotide binding and hydrolysis of the β−γ phosphate diester  
bond leading to specific conformational changes.⁴,⁸  

---

### Scheme 1. Major Signaling Pathways of NOD1 and NOD2

![Scheme 1. Major Signaling Pathways of NOD1 and NOD2](https://i.imgur.com/placeholder.png)

6898 dx.doi.org/10.1021/jm401841p J. Med. Chem. 2014, 57, 6897–6918

Journal of Medicinal Chemistry

**Scheme 2. Noncanonical Signaling Pathways of NOD1 and NOD2: (A) Antiviral Signaling, (B) Xenophagy, (C) Apoptosis, and (D) Inflammasome Formation**

- **bacteria**
- **MDP**
- **NOD2**
- **NLRP1/ NLRP3**
- **Caspase-1**
- **pro-IL-1β**
- **pro-IL-18**
- **IL-1β**
- **IL-18**
- **ATG16L1**
- **NOD1/2**
- **RIP2**
- **autophagosome**
- **antigens**
- **lysosome**
- **MHC II**
- **ssRNA**
- **MAVS**
- **TRAF3**
- **TBK1 IKKε**
- **IRF3**
- **IRF7**
- **APOPTOSIS**
- **IFN type I**

The Walker A box is characterized by a short GK(T/S) sequence (where the last residue is either Thr or Ser) that interacts with the phosphate moiety through bonding with the Lys residue. The Walker B box is usually characterized by a longer sequence pattern hhhhD(D/E) (h represents a hydrophobic residue). While the proximal Asp is crucial for coordinating binding of the Mg²⁺ cation that is required for nucleotide hydrolysis, the second acidic residue, usually Glu, primes a water molecule for the hydrolysis of ATP. The “extended Walker B box”, common to almost all NLRs (including NOD1 and NOD2), is a modified version of the original sequence in which the second acidic residue is missing. Instead, it consists of a conserved DE motif located three residues downstream of the first Asp and was most likely developed to functionally compensate for the missing second Asp, thus retaining the ATP hydrolysis activity. The Sensor 1 motif, which is common to all AAA+ proteins and consists of a conserved Arg, interacts with the γ-phosphate of ATP and coordinates nucleotide hydrolysis and conformational changes between subunits. On the other hand, the Sensor 2 motif, which is characterized by a conserved Arg/Lys and is involved in

nucleotide binding and hydrolysis, is generally missing in STAND class. In fact, it is replaced by a conserved His in the winged helix domain. An additional auxiliary residue GxP consists of a conserved Pro residue that interacts with the adenine moiety of the bound ATP.

**2.2. Signaling Pathways.** Bacterial PGN motifs are supposedly recognized by the LRRs of the C-terminal domain, as supported by recent biochemical evidence. However, there is no structural data to corroborate these findings. Conversely, Mo et al. have shown that MDP binds to the central NACHT domain. Both NOD1 and NOD2 have been shown to bind nucleotides (ATP, ADP), the central NACHT domain, which possesses ATPase activity, being crucial for activation and oligomerization. NOD1 and NOD2 are normally kept in an inactive state, either bound to ADP or free of nucleotide, autoinhibited through intramolecular shielding by LRRs folded over the NACHT domain. This state is also maintained by interaction with chaperone proteins including Hsp90 and SGT1.

On recognition of their cognate elicitors they undergo nucleotide exchange (ATP for ADP), causing conformational

Journal of Medicinal Chemistry

changes that lead to exposure of the NACHT domains. The binding of ATP, with its subsequent hydrolysis, drives the process of self-oligomerization that results in the activated state of NOD1/NOD2 with reduced affinity to Hsp90.¹⁵,¹⁸ Of note, the ATP hydrolysis step is also essential for deactivation of the NOD2 signaling platform, nodosome disassembly, whereas NOD1 may use a mechanism other than ATP hydrolysis for deactivation.¹⁵ In fact, the NOD1 CARD domain was recently reported to form dimers, which could be involved in a different path to formation of the signaling complex.¹⁹ These reactions are followed by migration to the plasma membrane, since it has recently been observed that the activation status of NOD1 and NOD2 in intestinal epithelial cells is also reflected by their subcellular localization. This pattern is consistent in various epithelial cell lines in which elicitor-mediated activation of NOD1 and NOD2 causes their localization to actin-rich regions at the plasma membrane (the sites of bacterial entry), which is essential for activation of NF-κB.¹⁵,²⁰,²¹ These findings have been corroborated by the recent discovery that receptor-interacting protein 2 (RIP2), the downstream component of the NOD signaling platform, nodosome, is also recruited to the plasma membrane and that its activity is dependent on its subcellular localization.²² However, although plasma membrane localization is important in certain settings and may contribute to maximize efficiency, it is not essential, since signaling can also occur from other regions.

### 2.2.1. Canonical NF-κB Pathway and Stress Kinases (Depicted in Scheme 1). At the molecular level, the signaling pathways downstream of NOD1 and NOD2 both converge on the signaling kinase RIP2. This serine-threonine/tyrosine kinase, with dual kinase specificity, is recruited and activated through homophilic CARD-CARD interactions, thus creating a signaling platform for subsequent downstream signal transduction.²⁴⁻²⁷ Tumor necrosis factor receptor associated factor 6 (TRAF6), an E3 ubiquitin ligase, is then recruited to RIP2 and subsequently autoubiquitinated. The RIP2 kinase domain has also been demonstrated to bind other E3 ubiquitin ligases such as cellular inhibitors of apoptosis (cIAPs), cIAP1 and cIAP2, which are required for K63 ubiquitination of RIP2, and X-linked IAP (XIAP). This mediates its ubiquitination, thus enabling the recruitment of linear ubiquitin assembly complex (LUBAC). LUBAC subsequently attaches ubiquitin chains to, and thus activating, the regulatory protein NF-κB essential modulator (NEMO).²⁸⁻³¹

Poly-ubiquitination of RIP2 attracts the adapter proteins TAB1/2/3 (TAK-binding proteins 1, 2, and 3) and their associated kinase, transforming growth factor (TGF) β activated kinase 1 (TAK1).²⁴,³²,³³ IkB kinase (IKK) complex is then recruited and activated, thereby assembling a multiprotein conglomerate that triggers phosphorylation of IkB, the inhibitory protein of NF-κB, causing translocation of NF-κB to the nucleus where it initiates activation of the target genes and the onset of an inflammatory response.³⁴

NOD1 and NOD2, moreover, also activate the three main families of mitogen-activated protein kinases (MAPKs) (p38, extracellular signal-regulated kinase (ERK) and c-Jun N-terminal kinase (JNK)), resulting in the induction of activator protein 1 (AP-1) activity. The AP-1 transcription factor controls several processes, including cell proliferation and differentiation and apoptosis.³³,³⁵,³⁶ Furthermore, in myeloid cells (dendritic cells, macrophages), NOD2 has been shown to interact with CARD9 adapter protein, which is required for connecting NOD2 PGN recognition function to downstream activation of MAPKs, p38,

and JNK, while it has no, or only a modest, effect on the activation of NF-κB.³⁷

#### 2.2.2. Noncanonical Signaling Pathways. In addition to the canonical NF-κB and MAPK pathways, NOD1 and NOD2 activate other noncanonical innate immunity mechanisms. They have been shown to induce activation of interferon regulatory factors (IRF) IRF3 and IRF7, resulting in induction of type I interferon (IFN), IFNβ, via formation of a multiprotein complex consisting of RIP2 and TRAF3 (shown in Scheme 1).³⁸⁻⁴¹ In addition, they are also involved in other signaling pathways of innate immunity. Viral ssRNA is yet another PAMP that has been reported to activate NOD2, causing its translocation toward the mitochondria. There it supposedly binds to, and subsequently activates, the protein MAVS (mitochondrial antiviral signaling) through its NACHT and LRR domains, resulting in IRF3 and IRF7 activation and the induction of type I IFN response (shown in Scheme 2).⁴⁰

NOD1 and NOD2 also play prominent roles in facilitating autophagosome formation via interaction with ATG16L1 (autophagy related 16-like 1), thereby enhancing xenophagy, a pivotal process leading to the clearance of pathogens, thus reducing the microbial burden (shown in Scheme 2).⁴² On the one hand, NOD1 and NOD2 have been shown to recruit ATG16L1 to the membrane to target bacteria at point of entry independently of RIP2.⁴² On the other hand, NOD2-stimulated xenophagy has been shown to be dependent on RIP2 tyrosine kinase activity which activates p38 MAPK and ERK as well as relieves the repression of xenophagy by protein phosphatase 2A (PP2A).⁴³ In addition, a xenophagy-independent role of ATG16L1 was recently uncovered; namely, it serves as a negative regulator of NOD-driven inflammatory processes.⁴⁴

Following the fusion of autophagosome with lysosome, the bacteria are degraded, yielding antigens that are then presented to T cells by MHC II molecules (except in the case of epithelial cells which typically do not express MHC II).³⁹,⁴²⁻⁴⁵

NOD1 and NOD2 have also been reported to interact with the apoptotic pathway indirectly via interaction of RIP2 with the IAP family of proteins (XIAP, cIAP1, and cIAP2).²⁹,³⁰ Moreover, NOD1 has been shown to activate caspase 8, thereby inducing caspase-8-mediated apoptosis which has been linked to its reported antitumor actions.⁴⁶

Finally, NODs have been reported to interact with certain other members of the NLR family, namely, NLRP1 (NACHT, LRR, and pyrin domain-containing protein 1) and NLRP3, thus assembling into inflammasome superstructures.⁴⁷⁻⁴⁹ Inflammasomes are signaling platforms capable of detecting a broad range of molecular signatures, including pathogenic microorganisms and sterile stressors, that activate proinflammatory cytokines IL-1β and IL-18.⁵⁰ Following recognition of their cognate ligand (DAMPs, asbestos, urea or cholesterol crystals), the inflammasome NLRs interact with adaptor protein ASC (apoptosis-associated specklike protein containing CARD) and procaspase 1, forming large multimeric protein complexes. This in turn leads to activation of caspase 1, which proteolytically activates the proinflammatory cytokines IL-1β and IL-18 (shown in Scheme 2).

### 2.3. Regulation of the NOD Signaling Pathway. The NOD1 and NOD2 signaling cascades have to be tightly controlled because of potentially harmful effects arising from overactivation or suppression of the inflammatory response. Tipping the scale in any direction would disturb this delicate balance and could lead either to serious infectious diseases or to chronic inflammatory and autoimmune diseases. Even though

dx.doi.org/10.1021/jm401841p J. Med. Chem. 2014, 57, 6897–6918

Table 1. Regulators of NOD-Signaling Network${}^{a}$

| protein                | pathway          | effect | mechanism of action                                                                                                      | ref |
|------------------------|------------------|--------|--------------------------------------------------------------------------------------------------------------------------|-----|
| Rac1GTPase and $\beta$-PIX | NOD2             | –      | Catalyzes association of NOD2 with the actin cytoskeleton, facilitating erbin-NOD2 interaction; $\beta$-PIX reinforces the Rac1GTPase effect on NOD2 | 52, 55 |
| erbin                  | NOD2             | –      | Colocalizes with NOD2 at the membrane, preventing overwhelming activation                                                   | 53, 54 |
| CD147                  | NOD2             | –      | Competes directly with RIP2 for binding to NOD2                                                                           | 56   |
| vimentin               | NOD2             | +      | Unclear mode of action; interacts with NOD2 at the plasma membrane                                                       | 57   |
| FRMPD2                 | NOD1 and NOD2    | +      | Associates with the erbin-NOD2 complex; provides a spatial control mechanism for NOD2-mediated responses                        | 58   |

Regulators of Cellular Localization

| CARD8                  | NOD2             | –      | Interacts with NOD2 NACHT domain, interfering with oligomerization; hinders membrane localization                          | 59   |
| JNKBP1                 | NOD2             | –      | Forms a complex with NOD2, blocking NOD2 oligomerization                                                                  | 60   |
| NOD2-S and NOD2-C      | NOD2             | –      | Truncated forms of NOD2; interfere with NOD2 oligomerization                                                              | 61, 62 |
| NLRC4                  | NOD1 and NOD2    | –      | Hetero-oligomerization with NOD1/2NACHT domain, preventing downstream signaling                                              | 65   |
| p62                    | NOD2             | +      | Interaction with NOD2 NACHT domain and formation of a large complex that prevents proteasomal degradation                 | 63   |
| Hsp90                  | NOD1 and NOD2    | +      | Exerts a stabilizing function, promoting maturation                                                                        | 16, 64 |
| SGT-1                  | NOD1             | +      | Exerts a stabilizing function, promoting maturation                                                                        | 17   |

Regulators of Oligomerization

| ITCH                   | NOD2 (NOD1?)     | –      | Polyubiquitinates activated RIP2 and cIAP1, targeting them for deactivation/degradation                                 | 66, 67 |
| SHIP-1                 | NOD1 and NOD2    | –      | Interferes with XIAP-RIP2 interaction                                                                                     | 68   |
| A20                    | NOD1 and NOD2    | –      | Involved in deubiquitination of RIP2                                                                                     | 69, 70 |
| TRIM27                 | NOD2             | –      | K48 ubiquitinates NOD2, rendering it prone to proteasomal degradation                                                     | 71   |
| TRAF4                  | NOD2             | –      | Interacts with an active NOD2-RIP2 complex, inhibiting subsequent ubiquitination and phosphorylation                       | 72   |
| PSMA7                  | NOD1             | –      | Promotes K48 polyubiquitination of NOD1, rendering it prone to proteasomal degradation                                   | 73   |
| TRAF2, -5, and -6      | NOD1 and NOD2    | +      | K63-ubiquitinates RIP2                                                                                                    | 24   |
| cIAP1 and -2           | NOD1 and NOD2    | +      | K63-ubiquitinates RIP2; cIAP1 preferentially binds to RIP2; cIAP1 promotes XIAP degradation                                | 29   |
| XIAP                   | NOD1 and NOD2    | +      | Polyubiquitinates RIP2 (not K63!) and recruits LUBAC                                                                      | 30, 31 |
| Pellino3               | NOD1 and NOD2    | +      | K63-ubiquitinates RIP2                                                                                                    | 74   |

Regulators of Ubiquitination

| ATG16L1                | NOD1 and NOD2    | –      | Interferes with polyubiquitination of RIP2 adapter and recruitment of RIP2 into large signaling complexes                | 44   |
| ubiquitin              | NOD1 and NOD2    | –      | Competes with RIP2 as a binding partner to NOD CARD domains                                                               | 75   |
| AAMP                   | NOD1 and NOD2    | –      | Combines with CARD domain and prevents formation of NOD2-RIP2                                                              | 76   |
| MEKK4                  | NOD2             | –      | Binds to RIP2 in unstimulated cells and sequesters it from NOD2                                                            | 77   |
| RIG-I                  | NOD2             | –      | Binds to various regions of NOD2, blunting the NOD2-induced NF-$\kappa$B activation                                        | 78   |
| GEF-H1                 | NOD1 and NOD2    | +      | Mediates Tyr phosphorylation of RIP2 by Src kinase; recruited into bacterial invasion sites and interacts with NOD1         | 79, 80 |

Regulators of RIP2 Recruitment

| PRDX4                  | NOD2             | –      | Unclear mode of action                                                                                                  | 81   |
| CENT$\beta$1           | NOD1 and NOD2    | –      | Unclear mode of action                                                                                                  | 82   |
| CAD                    | NOD2             | –      | Unclear mode of action; keeps NOD2 in an inactive state                                                                   | 83   |
| PTPN22                 | NOD2             | –      | Unclear mode of action                                                                                                  | 84   |
| Ankrd17                | NOD1 and NOD2    | +      | Unclear mode of action                                                                                                  | 85   |
| GRIM19                 | NOD2             | +      | Unclear mode of action; interacts with NOD2                                                                               | 86   |

Regulators via Unknown Mechanisms

${}^{a}$ Abbreviations: A20, ubiquitin E3 ligase A20; AAMP, angio-associated migratory protein; Ankrd17, ankyrin repeat-containing protein; ATG16L1, autophagy related 16-like 1; $\beta$-PIX, p21-activated kinase interacting exchange factor; CAD, carbamoyl phosphate synthetase/aspartate transcarbamylase/dihydroorotase; CARD8, caspase-recruitment domain 8; CENT$\beta$1, centaurin $\beta$1; cIAP, cellular inhibitor of apoptosis; FRMPD2, FERM and PDZ domain-containing 2 protein; GEF-H1, guanine nucleotide exchange factor H1; GRIM-19, eukaryotic gene associated with retinoic-interferon-induced mortality 19; Hsp90, heat shock protein 90; ITCH, itchy E3 ubiquitin protein ligase; JNKBP1, c-Jun N-terminal kinase binding protein; MEKK4, mitogen-activated protein kinase kinase 4; NLRC4, NOD-like receptor family CARD domain-containing protein 4; PSMA-7, proteasome subunit $\alpha$ type 7; PTPN22, protein tyrosine phosphatase non-receptor type 22; Rac1, Ras-related C3 botulinum toxin substrate 1; RIG-I, retinoic acid induced gene I; SGT-1, suppressor of the G2 allele of Skp1; SHIP-1, Src homology 2 domain containing inositol 5-phosphatase 1; TRAF, tumor necrosis factor associated factor; TRIM27, tripartite motif containing protein 27; XIAP, X-linked inhibitor of apoptosis.

the main events leading to NF-κB activation are established, the regulatory networks that control positive and negative regulation are still poorly understood. Several studies have shown that NOD1 and NOD2 interact with accessory proteins that, mostly negatively but also positively, regulate their signaling pathway, thus constituting an important part of the NOD-signaling network (listed in Table 1).

The regulation of signaling occurs at different stages and is also dependent on the cell type. During the early stages of NOD signaling, regulation by the cytoskeleton takes place, since disruption of cytoskeleton was shown to potentiate the response of macrophages to NOD ligands.51 In addition, given that activation of NOD depends on the concentration of ligand at the membrane, the total activation level seems to be regulated at the level of cellular localization, especially in polarized intestinal epithelial cells.52

Moreover, the oligomerization process also represents an important step in the NOD-mediated signaling cascade and is controlled by several regulatory components, which interact with the central NACHT domain. RIP2 undergoes autophosphorylation and ubiquitination in a site-specific and time-specific manner to effect the activation and subsequent deactivation of NOD signaling. In fact, the ubiquitination of RIP2 is considered to be one of the most tightly regulated steps in NOD-signaling pathway, since RIP2 kinase domain has been shown to bind various E3 ubiquitin ligases, that facilitate the posttranslational modifications involved in protein activation. However, RIP2 interacts not only with NOD1 and NOD2 through CARD-CARD interactions but also with several other proteins; therefore, numerous CARD-containing molecules can be involved in the innate immune response.

Interestingly, additional regulators of NOD signaling pathways have recently been revealed. Their mechanisms of action, however, have yet to be disclosed. Finally, a recent, comprehensive small interfering RNA (siRNA) screen has revealed several previously unknown positive and negative regulators of the NOD2 signaling pathway. Identification of the genes coding for the various classes of proteins will further increase our understanding of how this pathway is regulated.87

The negative regulation is of special importance, since NOD activation triggers many signaling pathways, including NF-κB, MAPK, and type I IFN pathways, which can result in an overwhelming inflammatory response. Mutations in the NOD2 NACHT domain, more precisely in the extended Walker B box, can cause an enhanced NF-κB activation, in contrast to the corresponding mutations in NOD1. This implies that NOD2 is more susceptible to autoactivation than NOD1 and hence needs to be more tightly controlled. The latter conclusion is corroborated by the facts that (i) many more negative regulators are reported for NOD2 than for NOD1, (ii) mutations found in NOD2 are more likely to be associated with disease, and (iii) NOD2 expression is more restricted than that of NOD1.15

### 2.4. Physiological Relevance of NOD1 and NOD2.
The cell response to bacterial peptidoglycan is mediated largely by NOD1 and NOD2, two members of the cytosol localized receptor family, which recognize PGN breakdown products following exposure to the cytosol. NOD1 recognizes primarily Gram-negative bacteria and is expressed in epithelial, immune, and various other types of cells. The pattern of expression of NOD2 appears to be more restricted, since it is expressed mainly in professional immune cells (macrophages, dendritic cells, Paneth cells) but also in epithelial cells of the intestine and lung.88 The dipeptide D-Glu-meso-DAP (iE-DAP), present in the

PGN of Gram-negative and some Gram-positive bacteria, is the minimal sequence necessary and sufficient for binding to NOD1.89–92 The presence of a preceding alanine in L-Ala-D-Glu-meso-DAP (TriDAP) enhances this response.10 NOD2, in contrast, is a general sensor, since it recognizes and binds the MDP present in the PGN of both Gram-negative and Gram-positive bacteria.11,93,94

The mechanism underlying the uptake of extracellular NOD1 and NOD2 ligands into the cytosol is still not completely clear, but several possible mechanisms have been proposed by which NOD ligands can be delivered: (i) Helicobacter pylori delivers NOD1 ligands into epithelial cells through type IV secretion systems.95 (ii) Intestinal epithelial cells and, to a minor extent, monocytes express peptide transporter PepT1, known to facilitate specifically the delivery of MDP to the cytosol,96–98 whereas PepT2 mediates the uptake of iE-DAP.99 (iii) Both MDP and M-Tri-DAP undergo clathrin-dependent endocytosis followed by ligand processing and subsequent transfer from the endosome into the cytosol. Transport takes place either out of early endosomes by SLC15A4 transporter, which enables NOD1 ligand transport, or out of mature endosomes, in which case acidification, provided by vacuolar ATPases, is required for NOD2 ligand transport.100,101 (iv) Bacterial outer membrane vesicles constitute another possible means of intracellular delivery of NOD1 ligands to epithelial cells.102,103 (v) The fact that a variety of methods of delivery (calcium shock, micro-injection) and introduction of hydrophobic groups into NOD1 and NOD2 agonists strongly enhance the potency of the compounds suggests that these synthetic derivatives are capable of crossing the membrane by passive absorption.104

NOD1 and NOD2 induce NF-κB activation following recognition of bacterial components, resulting in proinflammatory cytokine and chemokine secretion, the hallmark of an inflammatory response. In addition, NODs are involved in the production and secretion of antimicrobial peptides, such as β-and α-defensins, in the gastrointestinal tract. NOD1 and NOD2 are also capable of initiating another essential part of the antimicrobial response characterized by formation of reactive oxygen and nitrogen species via induction of nitric oxide synthase and NADPH oxidases.105,106 Their role in gastrointestinal tract innate immunity appears to be indispensable for intestinal homeostasis, since intestinal cells lacking TLRs are hyporesponsive to TLR ligands and rely mostly on NOD1 activity for orchestrating an appropriate immune response. Signaling through NOD1 also provides backup in cases where the cells are tolerant to MDP107 or NOD2 function is impaired.108 The NOD2 genotype has been shown to lack any influence on responses to specific NOD1 agonists, suggesting the independent action of the NOD1 and NOD2 pathways.108 Of note, the degradation of NODs is tightly controlled by the chaperone Hsp90, which dissociates from activated NODs. Moreover, NOD2 in MDP-stimulated macrophages was shown to undergo a proteasome-dependent degradation, leading to a tolerant state (to restimulation with MDP) to protect the host from septic shock.18

Several investigations of the function of NODs in host defense have demonstrated their essential role in controlling bacterial infections caused by Salmonella typhimurium, Citrobacter rodentium, Listeria monocytogenes, Legionella pneumophila, Clostridium difficile, Helicobacter pylori, to name only a few.38,88,105 NOD1 has been reported to recognize and initiate a response against a variety of Gram-negative bacteria, while NOD2 senses both Gram-positive and Gram-negative bacteria.

Journal of Medicinal Chemistry

Recent reports have revealed that NOD1 and NOD2 are also involved in host defense against nonbacterial pathogens, such as protozoan parasites (*Toxoplasma gondii*, *Trypanosoma cruzi*) and viral infections (respiratory syncytial virus).<sup>88</sup>

Although NOD ligands are capable of triggering an inflammatory response characterized by the secretion of cytokines and chemokines, antimicrobial peptides, and reactive oxygen and nitrogen species, the response may not be strong enough. Several in vitro studies have demonstrated that NOD1 and NOD2 ligands, in synergy with TLR ligands, act on the production of TLR-mediated inflammatory cytokines, which could potentiate the cellular response against pathogens.<sup>107,109–114</sup> This cross-talk between TLRs and NLRs is crucial for the enhancement of innate and adaptive immune responses. NOD1 and NOD2 ligands have been shown to induce Th1- and Th2-type immune responses, thus controlling the onset of an appropriate adaptive immune response.<sup>35,115</sup> Interestingly, while NOD1 and NOD2 ligands drive Th2-type immunity, combined TLR and NOD stimulation enhances Th1-type immunity.<sup>115,116</sup> Finally, a synergistic interaction between a complement protein C5a and NOD2 has also been demonstrated in the regulation of chemokine expression, thereby promoting sepsis.<sup>117</sup>

### 3. NOD PROTEINS AND DISEASES

#### 3.1. Role in Disease Etiology
Human NOD1 and NOD2 proteins are encoded by the CARD4 and CARD15 genes, respectively. Single nucleotide polymorphisms (SNPs) and mutations in the CARD4 and CARD15 genes cause dysregulation of NLR function, which has been linked to a multitude of human diseases, ranging from chronic inflammatory disorders and autoimmunity to cancer. The functions of NOD1 and NOD2 proteins can be classified either as gain of function or loss of function, depending on the activation state of NF-κB. Furthermore, the importance of the regulation of levels of ubiquitination and phosphorylation must not be overlooked. Namely, the defective NOD signaling might arise not only because of mutant NODs but also because of faulty components of the NOD signaling pathway. For instance, ITCH deficiency was shown to result in insufficient clearance of activated RIP2, leading to enhanced NF-κB activation in spite of wild type NOD2.<sup>67,118</sup>

Crohn's disease (CD) is a multifactorial inflammatory bowel disease characterized by transmural granulomatous inflammation of the gastrointestinal tract. It has been associated with three major variants of NOD2 polymorphisms in the LRR domain either by single amino acid changes, such as R702W and G908R, or by a frameshift insertion of L1007fsinsC which causes a reading frame shift and a complete loss of function.<sup>119–121</sup> This latter mutation which results in truncated isoform also prevents NOD2 from localizing to plasma membrane because of deletion of the terminal LRRs.<sup>20</sup> Several other less common variants have also been shown to be more frequent in CD patients; however, the majority of patients who develop CD actually have wild type NOD2, which indicates that in these cases the defect might lie in other components of NOD2 signaling pathway (e.g., lower expression of E3 ubiquitin ligases, mutated ATG16L1).<sup>67</sup>

The function of the extended Walker B box residues in NOD1 and NOD2 signaling is fundamentally different. On the one hand, the mutations of the first acidic residue of the extended Walker B box of both NOD1 and NOD2 lead to a complete loss of NF-κB activation.<sup>15</sup> In sharp contrast, mutation of the second acidic residue in NOD2 results in a significantly increased basal NF-κB activation, while the mutation of the corresponding amino acid in NOD1 completely abolishes NF-κB activation. These findings imply that NOD2 is more susceptible to autoactivation than NOD1 and that the mutations found in NOD2 are more likely to be associated with disease. Certain NOD2 variants in the NACHT region, more precisely in the extended Walker B box, which is essential for ATP hydrolysis, have been identified in the genetic disorders Blau syndrome (BS) (three missense mutations: R334Q, R334W, L469F)<sup>122–124</sup> and early onset sarcoidosis (EOS) (D382E, A612T).<sup>125</sup> These diseases are characterized by granulomatous inflammation of multiple organs and have been shown to lead to gain of function, resulting in increased basal NOD2-mediated NF-κB activity and autoactivation (mostly due to prominent membrane recruitment).<sup>125</sup> Similarly, NOD2 has also been associated with arthritis.<sup>126,127</sup> Furthermore, a new autoinflammatory disease, designated NOD2-associated autoinflammatory disease (NAID), has recently been reported by Yao et al. In NAID, the gene mutations IVS8 + 158, R702W, and R703C are located between the NACHT and LRR regions, and the fact that one of the variants (R703C) shows increased NF-κB activation, a gain-of-function mutation is indicated.<sup>128</sup> In addition to its strong association with inflammatory diseases, NOD2 has also been linked to atopy and asthma.<sup>129–131</sup> There is also increasing evidence for its detrimental role in certain autoimmune diseases, multiple sclerosis, and systemic lupus erythematosus.<sup>132</sup> Inappropriate activation of NOD2 may also lead to uncontrolled proliferation of certain cell types, but the role of NOD2 mutations is still unclear.<sup>133–136</sup> Increased incidence of colorectal cancer was observed in CD patients (mutations L1007fsinsC, R702W, and G908R),<sup>133–136</sup> while on the other hand a protective role of NOD2 in cancer was reported.<sup>137</sup> Genome-wide association studies have also linked SNPs in NOD2 to prostate<sup>134</sup> and endometrial<sup>135</sup> cancer. Finally, SNPs in the NOD2 gene have been linked to increased susceptibility to infectious diseases, such as leprosy<sup>138</sup> and tuberculosis.<sup>139,140</sup>

In spite of the similarity between NOD1 and NOD2, mutations of the NOD2 gene frequently lead to development of a disease, while correlations between NOD1 gene variants and diseases are not as common. A recent study of NOD proteins has revealed that these receptors have different modes of activation, given that a single mutation in the NACHT domain has contrasting effects on NOD1- and NOD2-mediated activation.<sup>15</sup> NOD1 has a somewhat controversial role in the development of cancer and cancer predisposition. While certain SNPs have been shown to disturb NOD1 function and to predispose to certain types of cancer,<sup>133</sup> a protective role of NOD1 in cancer was reported in sharp contrast.<sup>141</sup> Of note, NOD1 has also been shown to exhibit a tumor-suppressor function.<sup>46</sup> Moreover, an insertion-deletion polymorphism ND1 + 32656 within NOD1 has been linked to increased susceptibility to asthma and inflammatory bowel disease. This mutation caused the expression of isoforms lacking the LRRs, rendering these proteins constitutively active.<sup>142,143</sup> Finally, NOD1 polymorphisms have also been linked to atopic diseases.<sup>144</sup>

Besides SNPs, ligand-elicited overactivation has been shown to cause dysregulation of NOD1, thus exhibiting a detrimental effect. For instance, PGN-dependent NOD1 activation has been associated with the progression of multiple sclerosis.<sup>145</sup> Moreover, there is emerging evidence that administration of NOD1 agonists induces coronary arteritis, suggesting a role for NOD1 in cardiovascular inflammation.<sup>146–148</sup> In addition, artificially induced NOD1 activation also contributes to the development

Journal of Medicinal Chemistry

![curcumin (1)](image1)

![parthenolide (2)](image2)

![helenalin (3)](image3)

![pseudopterosin A (4)](image4)

![DHA - docosahexaenoic acid (5)](image5)

![EPA - eicosapentaenoic acid (6)](image6)

Figure 2. NOD inhibitors of natural and endogenous origin.

of ocular inflammation<sup>149</sup> and fetal inflammation.<sup>150</sup> Finally, NOD1 activation by NOD1 ligands has been shown to stimulate insulin resistance and adipose tissue inflammation in doses that elicit only minor changes in cytokine levels.<sup>151,152</sup>

### 3.2. Therapeutic Potential of NOD1 and NOD2 Inhibition

There are a number of potential targets for NOD1 and NOD2 inhibition within the innate immune system; however, they need to be exploited with care, since the system is a major coordinating node of host defense, inflammation, and apoptosis. Targeted and controlled modulation of the innate immune response have great potential for the treatment of chronic inflammatory and autoimmune diseases characterized by increased NF-κB activation. Dysregulation of NLRs is implicated in the pathology of various inflammatory disorders. Given that therapeutic intervention of NLR dysfunction is currently only possible by the administration of strong, broad-range anti-inflammatory drugs, which often display severe side effects, specific modulators of NLR signaling would be beneficial. NOD1 and NOD2 are of particular interest, since they activate the NF-κB pathway directly, and inhibition of NOD1 and NOD2 signaling could be advantageous in treating many acute and chronic diseases in which suppression of the proinflammatory response would be beneficial.<sup>153–155</sup>

NOD2 is mostly recognized for its involvement in the pathogenesis of CD, which is associated with a loss-of-function mutation resulting in loss of downstream NOD2 signaling. However, some NOD2-associated diseases are caused by mutations in the NOD2 gene that lead to gain-of-function constitutive activation (BS and EOS) or by hyperactivity of wild type NOD2. Given that hyperactivity of NOD2 is implicated in the pathophysiology of these disorders, inhibition of NOD2 and the components of its signaling pathway would appear to be a logical therapeutic strategy. Current therapy for NOD2-associated diseases consists of potent anti-inflammatory medicines (corticosteroids) and biologicals (TNF-α inhibitors and IL-1 receptor antagonists). Although the use of TNF-α inhibitors is usually effective in treating CD, there is no correlation between the response and NOD2 variants. TNF-α inhibitors are effective in alleviating BS symptoms.<sup>128</sup> Although SNPs in NOD2 gene increase susceptibility to leprosy and tuberculosis, this resulted from a loss-of-function mutation, so inhibition of NOD2 signaling does not appear to be a viable strategy. Similarly, the NOD2 mutation, which has been shown to predispose to cancer, also exhibited loss of function.<sup>136</sup>

PGN has been shown to play a role in the development of sepsis, a systemic inflammatory response. Although NOD1 and NOD2 agonists are only weak inducers of inflammatory responses, this activity can be amplified significantly by the synergistic crosstalk between NOD receptors and other PRRs.<sup>112–114,117</sup> In addition, signals of endogenous damage are also presumed to be capable of activating NOD1 and NOD2. Thus, therapeutic inhibition of NOD1 and NOD2 could be useful for treating (cardio)vascular complications arising from both infectious- and noninfectious-related multiorgan failure.<sup>146–148</sup>

Saturated fatty acids have been shown to act as agonists of NOD1 and NOD2 receptors.<sup>156</sup> In the setting of a high-fat diet, human intestinal cells can be exposed to high concentrations of fatty acids, which may cause excessive inflammation. Intestinal mast cells are activated by orally administered PGN, which in turn causes diarrhea. This effect has been inhibited by blocking the NOD1 signaling, again implying potential therapeutic possibilities.<sup>157</sup> Thus, NOD1 inhibition is thought to posit an interesting alternative in treating the excessive inflammation associated with cardiovascular, intestinal, and metabolic diseases.<sup>153</sup> Finally, certain types of cancer have been associated with NOD1 mutations. However, given their role in sensing potentially carcinogenic infectious agents (*Helicobacter pylori*, *Escherichia coli*, *Shigella flexneri*) and the antitumor effect of NOD1 activation, NOD1 inhibitors could only exacerbate the progress of the disease.

Mechanistically, suppressing NOD1 or NOD2 activity should lead to impaired host defense, which raises the main issue of concern, that of safety. However, the fact that pharmacological agents only reduce, and rarely completely inhibit, the activity of their targets reinforces the suggestion that inhibition of NOD signaling could be employed safely. Targeting the upstream pathways responsible for NF-κB signaling would be expected to leave most innate immunity defense mechanisms intact, which could be beneficial for diseases in which the specific target is involved in the disease pathogenesis.<sup>38</sup> It has been speculated that brief cycles of therapy with NOD1 or NOD2 inhibitors (administered concomitantly with antibiotics) could reduce inflammation and help to re-establish immune homeostasis.<sup>38</sup> In considering therapeutic targeting of NOD receptors with inhibitors, it is important to take into account the extensive crosstalk between NOD receptors and other PRRs, as exemplified by synergistic interaction with TLRs.<sup>153</sup> Finally, small molecule inhibitors would also provide useful pharmacological tools for elucidating the roles of these proteins in acute and chronic inflammatory diseases, as well as in normal host-defense mechanisms. Harnessing the full potential of NOD inhibitors is thus a “hot topic” in medicinal chemistry.

Journal of Medicinal Chemistry

![image](https://i.imgur.com/1234567.png)

Figure 3. SAR of indoline scaffold-based NOD inhibitors.

## INHIBITORS OF NOD1 AND NOD2 SIGNALING

### 4.1. Natural Inhibitors of NOD1/2 Signaling
Numerous natural compounds possess anti-inflammatory properties. Curcumin (1, Figure 2), a polyphenol present in the plant *Curcuma longa*, has long been known as a potent anti-inflammatory agent because of its inhibition of various signaling pathways (TNF-α, TLR4), which in turn leads to a general inhibition of NF-κB activation. It has recently been shown to suppress not only MDP-induced but also lauric acid induced NOD2 signaling, resulting in the suppression of NF-κB activation and IL-8 expression in a dose-dependent manner. Although no conclusive evidence for a specific effect of 1 on NOD2 has been provided, it has been speculated that it acts by interfering with NOD2 oligomerization. In addition, 1 has also been shown to inhibit NOD1-induced NF-κB activation. Furthermore, parthenolide (2), a representative sesquiterpene lactone, has also been shown to exert anti-inflammatory effects via down-regulation of MDP-induced and lauric acid induced NOD2 signaling, which leads to suppression of NF-κB activation and IL-8 expression. The latter effect has also been ascribed to the inhibition of ligand-dependent NOD2 oligomerization. Strikingly, 1 and 2 inhibited not only the activation of NOD2 signaling induced by a ligand of bacterial origin (MDP) but also its activation induced by an endogenous ligand, lauric acid. It is noteworthy that both compounds also inhibited NF-κB activation induced by NOD2 overexpression. The mechanism by which they inhibit NOD2 oligomerization is unknown; however, it has been proposed that their structures contain reactive centers, in the form of an α-methylene-γ-lactone group or an α,β-unsaturated carbonyl group, that enable reaction with biological nucleophiles such as the sulfhydryl groups present in the NACHT domains of NLRs and p65 subunit of NF-κB. 2 has also been shown to interfere with the ATPase activity of NLRP3, thereby hindering NLRP3 inflammasome activation. Similar inhibitory effects on NF-κB activation were also observed for helenalin (3) and several other sesquiterpene lactones; however, with the exception of 3, their mechanism of its action was not elucidated. 3 has been shown to alkylate and thus inhibit the p65 subunit of NF-κB. Importantly, the IC₅₀ values reported for inhibition by NF-κB were not influenced by the cytotoxicity of the compounds. Of note, it has also been suggested that pseudopterosins such as pseudopterosin A (4),

### 4.2. Endogenous Inhibitors of NOD1/2 Signaling
NOD proteins can also be modulated by certain endogenous molecules, namely, dietary fatty acids. While saturated fatty acids, such as lauric acid, induce the activation of NOD1/2 signaling, some unsaturated fatty acids, particularly the n-3 polyunsaturated type, exert inhibitory effects on this signaling pathway. Among the polyunsaturated fatty acids, docosahexaenoic acid (DHA) (5) and eicosapentaenoic acid (EPA) (6) exert the most prominent NOD1/2 inhibitory effects, although this suppression of NF-κB activation can, to a greater extent, be ascribed to their action on the NOD2 signaling pathway. They have been shown to prevent the self-oligomerization process of NOD2, resulting in inhibition of downstream signaling, seen as suppression of NF-κB activation and IL-8 expression. However, it has to be taken into account that fatty acids, in addition to NOD signaling, can also inhibit several other signaling pathways. Further, the inhibitory effects of 5 and 6 on NOD activation could also be ascribed to their enzymatic metabolites. However, these results underline the important role of dietary fatty acids in inflammatory processes.

### 4.3. Inhibitors of NOD1/2 Signaling from the Molecular Libraries Network Initiative
In the search for small molecule NOD1 inhibitors, an exhaustive high-throughput screening (HTS) program was conducted under the auspices of the NIH-sponsored Molecular Libraries Probe Production Center Network. Several active scaffolds with IC₅₀ values less than 10 μM were identified from a library of 290,000 compounds from the NIH Molecular Libraries Small Molecule Repository compound collection, using cell-based NF-κB driven luciferase reporter gene activity as a measure of NOD1 modulation. The main goal was to identify a compound with IC₅₀ < 1 μM and selectivity against NOD2 and TNF-α. Secondary function assays were performed in order to confirm that such compounds inhibit IL-8, a biologically relevant downstream target of NOD1 pathway. The indoline scaffold, the tetrahydroisoquinoline scaffold, and the benzimidazole scaffold met the initial screening criteria of (i) inhibition of NOD1 with IC₅₀ < 1 μM, (ii) inhibition of NOD1-induced IL-8 secretion with IC₅₀ < 1 μM, and (iii) a target selectivity over NOD2 of 5× and over TNF-α of 10×.

Journal of Medicinal Chemistry

retroamide is not
tolerated

COOEt; SMe; OCF₃ ==> increased activity
Cl; OEt; NMe₂; OMe, Br are tolerated
Me or F ==> loss in activity

![Chemical Structure 1](chemical_structure_1.png)

replacement with

loss of activity

![Chemical Structure 2](chemical_structure_2.png)

10: IC₅₀ (NOD1) = 0.15 μM
IC₅₀ (NOD2) = 0.40 μM

11: IC₅₀ (NOD1) = 0.24 μM
IC₅₀ (NOD2) = 0.26 μM

12: IC₅₀ (NOD1) = 0.08 μM
IC₅₀ (NOD2) = 0.03 μM

Figure 4. SAR of tetrahydroisoquinoline scaffold-based NOD inhibitors.

decisive for selectivity and activity
SO₂ > CO > CH₂

Cl or Me are optimal
OMe, H or NO₂ are tolerated

R₁ = R₂ = H; larger groups (e.g., Me)
==> loss of activity

replacement with Me or SH is
not tolerated

![Chemical Structure 3](chemical_structure_3.png)

Noditinib-1 (13): IC₅₀ (NOD1) = 0.56 μM
IC₅₀ (NOD2) = >20 μM
(selective; > 36×)

14: IC₅₀ (NOD1) = 0.09 μM
IC₅₀ (NOD2) = 19.9 μM
(selective; 221×)

15: IC₅₀ (NOD1) = 7.7 μM
IC₅₀ (NOD2) = 11.9 μM
(nonselective)

Figure 5. SAR of 2-aminobenzimidazole scaffold-based selective NOD1 inhibitors.

The indoline scaffold was rationally modified at three different sites R₁, R₂, and R₃ to investigate the structure–activity relationship (SAR). Numerous analogues were prepared in order to determine the effect of the size of the aliphatic group at R₂, the stereoelectronic influence of different electron-deficient and electron-rich functionalities at R₃, and the effect of the indoline substitution at R₁ (representative compounds 7–9 are shown in Figure 3). The bioactivity of these compounds is dependent on the nature of the substituent R₁; replacement of the methyl group with hydrogen resulted in complete loss of activity. Small alkyl groups such as cyclopropyl, methyl, and ethyl are tolerated at the R₂ position. Significantly, the effect of substituent R₂ depends on the electronic nature of R₃. In general, electron-withdrawing groups (NO₂, COMe) at R₃ increase the activity of compounds, while neutral or electron-donating groups result in decreased activity. Unfortunately, none of the compounds from this series exerted selectivity toward NOD1 and NOD2; therefore, this scaffold was abandoned.¹⁶⁵,¹⁶⁶

The tetrahydroisoquinoline scaffold incorporating series was modified at three sites (representative compounds 10–12 are shown in Figure 4). The tetrahydroisoquinoline moiety was found to be essential for bioactivity, since its replacement with other heterocycles (e.g., piperidine, indoline) resulted in complete loss of activity. Having the CONH group as the spacer separating the two phenyl rings was mandatory for bioactivity, since replacement with its alternative, the retroamide NHCO, led

to complete loss of activity. The stereoelectronic influence of different electron-deficient and electron-rich functionalities was determined by varying the substituent R₁ and electron-donating groups were found to augment the activity of the compounds.¹⁶⁵,¹⁶⁶ The compounds present in both the indoline and tetrahydroquinoline scaffold series exhibited nonselective inhibitory activity, inhibiting not only NOD1- but also NOD2-mediated and TNF-α-mediated NF-κB activation.

SAR analysis of the benzimidazole series (representative compounds 13–15, Figure 5) revealed that the presence of an NH₂ group at 2-position is essential, since its replacement (with SH, Me, or H) rendered the compounds inactive. Moreover, replacing the hydrogens at R₁ and R₂ with methyl groups completely abrogated the activity. In addition, variation of the type of spacer X (e.g., sulfonyl, carbonyl, methylene) demonstrated that the sulfonyl group provides the most active compounds of the series and is also essential for their selectivity. Finally, the stereoelectronic nature of the substituent at R₃ considerably affected the bioactivity. Compound 14, with the 4-chloro substituent on the aromatic ring, was 6-fold more potent than the hit compound 13, while other substituents reduced the potency.¹⁶⁵,¹⁶⁶

Compound 13, also designated noditinib 1, was found to be noncytotoxic and to inhibit selectively NOD1-dependent NF-κB activation (IC₅₀ < 1 μM). It exhibited >36-fold selectivity against NOD2-dependent and TNF-α-dependent NF-κB activation,

Journal of Medicinal Chemistry

important for selectivity

H or Me are tolerated

ML146 (16): IC₅₀ (NOD1) = 1.54 μM
IC₅₀ (NOD2) = 12.7 μM
(selective; 8×)

17: IC₅₀ (NOD1) = 0.94 μM
IC₅₀ (NOD2) > 20 μM
(selective; > 21×)

18: IC₅₀ (NOD1) = 1.22 μM
IC₅₀ (NOD2) = 18.1 μM
(selective; 15×)

19: IC₅₀ (NOD1) = 0.79 μM
IC₅₀ (NOD2) = 0.75 μM
(nonselective)

Figure 6. SAR of purine scaffold-based selective NOD1 inhibitors.

Cl or Me at positions 5- or 6-
are optimal

well tolerated groups

replacement with these groups leads to loss in activity

H > Me

GSK669 (20): IC₅₀ (NOD2) =
3.3 μM

21: IC₅₀ (NOD2) = 0.8 μM

22: IC₅₀ (NOD2) = 1.6 μM

23: IC₅₀ (NOD2) = 1.3 μM

24: IC₅₀ (NOD2) = 3.4 μM

GSK717 (25): IC₅₀ (NOD2) = 0.4 μM

26: IC₅₀ (NOD2) = 0.2 μM

Figure 7. SAR of benzimidazole diamides as selective NOD2 inhibitors.

thus meeting all the required criteria. Moreover, it showed excellent selectivity against more than 400 different kinases.¹⁶⁵,¹⁶⁶

The secondary assay confirmed the functional activity of 13 by inhibiting NOD1-dependent IL-8 secretion from cells. The 2-aminobenzimidazole scaffold based series is thus the first class of NOD1-selective inhibitors to be identified.¹⁶⁵ The mechanism of its action was investigated using ¹H NMR spectroscopy with purified NOD1 protein and 13. The latter was found to interact directly with NOD1 protein, while ATP binding was not affected. Further experiments suggest that the compound may induce a conformational change in NOD1, increase its concentration in the membrane fraction, and at the same time, reduce the concentration of RIP2 at the membrane.¹⁶⁷

During the same attempts to identify small molecule NOD1 inhibitors employing the HTS campaign, a compound of another chemical class, purine-2,6-diones (a xanthine), was identified as an active and NOD1-selective inhibitor (compound 16 (ML146)). Several compounds of this class were synthesized to gain more insight into SAR (representative compounds 16–19 shown in Figure 6).¹⁶⁸

Journal of Medicinal Chemistry

these side-chains are important
for selectivity

X = NH, O; benzylamino group is preffered
4-F substitution is tolerated
Cr(CO)₃ group is a detoxifying substructure
if removed, the compounds become cytotoxic

27: NOD2 inhibition = 73.9 %
TLR2 inhibition = 0 %
TLR4 inhibition = 0 %
TNF inhibition = 16.8 %

28: NOD2 inhibition = 41.1 %
TLR2 inhibition = 13.5 %
TLR4 inhibition = 4.4 %
TNF inhibition = 0 %

29: NOD2 inhibition = 83.2 %
TLR2 inhibition = 50.1 %
TLR4 inhibition = 80.1 %
TNF inhibition = 49.5 %

Figure 8. SAR of hydrophenalene–chromium complexes as NOD2 inhibitors.

The nature of the substituent R₁ has a profound effect on the bioactivity of compounds that was influenced not only by the number of carbons and the branching pattern but also by the degree of unsaturation of substituent R₁. The presence of a thioalkyl group (compounds 16–18) as the substituent R₂ on the xanthine ring was essential for NOD1 versus NOD2 selectivity. Replacing it by NO₂ (compound 19) enhanced the activity but resulted in complete loss of selectivity. Hydrogen, as a R₃ substituent, provided the most active compounds of the series and its replacement by larger groups (e.g., methyl) resulted in complete loss of selectivity.¹⁶⁸

The physicochemical properties of the identified probes of different class type are consistent with those of druglike molecules. The calculated log P values were 3.2 (13) and 3.5 (16). The polar surface areas ranged from 67 Å² (16) to 86 Å² (13), and both compounds contained fewer than five H-bond donors and acceptors. Their pharmacological properties were evaluated using in vitro ADMET (absorption, distribution, metabolism, excretion, and toxicity) assays. While 13 was moderately soluble at 5.9–7 μM, 16 was the least soluble (1.5–1.8 μM) across the whole pH range (5–7.4). Both probes exhibited good in vitro cell permeability using PAMPA (parallel artificial membrane permeability assay). Their metabolic stability was very low in the presence of murine microsomes (0.8% remained after 1 h of exposure) but less so in the presence of human hepatic microsomes. Finally, none of the compounds, at 50 μM concentration, were cytotoxic to immortalized human hepatocytes.¹⁶⁹

4.4. Inhibitors of NOD2 Signaling Developed at GlaxoSmithKline. The HTS screening campaign to find NOD2-selective small molecule inhibitors has also been conducted within the molecular libraries network but without success. By use of a cell-based HTS approach, namely, MDP-induced IL-8 release in HEK293 cells, with approximately 1.9 million compounds, a benzimidazole diamide designated GSK669 (20) (IC₅₀ = 3.3 μM) was identified as a selective inhibitor capable of diminishing MDP-induced IL-8 release without affecting the function of RIP2 kinase.¹⁷⁰

Even more potent and selective NOD2 inhibitors were developed using a rational SAR approach (representative compounds 20–26 are shown in Figure 7).

Varying the R₁ substituent proved to be the most tolerant of alterations in terms of retaining bioactivity. The dihydroindane moiety in compound 20 was successfully replaced by many groups (e.g., 3-isopropylphenyl (compound 21) and naphthyl (compound 22)), resulting in compounds more potent than the lead. However, replacement of R₂ proved to be a less appropriate modification, since it only allowed for minor alterations such as ortho-substitutions on the phenyl ring or its replacement by a naphthyl ring (compound 24). Introduction of a chlorine atom at position 5 or 6 (R₃) of the benzimidazole ring (e.g., compound 23) enhanced the inhibitory activity, while other changes were not tolerated. The mono- and dimethylation of amides (R₄, R₅) brought about significant changes in activity. While the methyl group introduced at position R₄ significantly enhanced the activity of the parent compound (GSK717 (25); IC₅₀ = 0.4 μM), methylation of the amide functionality at R₅ or dimethylation (R₄ = R₅ = Me) of amides resulted in loss of activity. The final set of compounds, combining the most promising fragments, showed no significant improvement, the most potent inhibitor being compound 26 (IC₅₀ = 0.2 μM).

The mode of action of the benzimidazole diamide class of NOD2-inhibitors was then subjected to in-depth investigation. These inhibitors were shown not to interfere with the cellular uptake of MDP. Moreover, they inhibited dose-dependently the release of cytokines from MDP-stimulated primary human monocytes. Their inhibitory activity was inversely related to MDP concentration, and further exploration revealed a competitive interaction between MDP and 25 by binding to the same site on NOD2 protein. Interestingly, neither ssRNA-elicted nor NOD2-overexpression-elicted NOD2 activation was blocked by 20. This inability of 20 to block the response to ssRNA could mean that it acts on other proteins of the NOD2 signaling complex. Most importantly, the inhibitors proved to be specific and selective; they failed to inhibit the activation of TLR2-, TNFR1-, and NOD1-mediated pathways (>30 μM), which share components with NOD2 downstream of TAB/

Journal of Medicinal Chemistry

Scheme 3. Depiction of NOD1/NOD2 Primary HTS Inhibition Assay

SEED CELLS
↓
TRANSFECT CELLS WITH
REPORTER SYSTEM
↓
ADD INHIBITORS
incubate 1h
↓
ADD ELICITORS
incubate 16h
↓
READOUT

HEK293T
NF-κB-luciferase
reporter
NOD1/NOD2 plasmid
↓
inhibitors
TriDAP or MDP
↓
luminometer
NF-κB luciferase activity

---

TAK1. In addition, these compounds did not inhibit the activity of more than 300 kinases, including RIP2 kinase.¹⁷⁰

4.5. Pseudopterosin-Based Inhibitors of NOD2 Signaling. Arenechromium diterpene complexes, based on the structure of pseudopterosins, have been reported as specific inhibitors of NOD2-mediated NF-κB signaling.¹⁶⁴ When screened for their influence on TLR2-, TLR4-, and TNF-mediated NF-κB activation, they appear to be selective for NOD-mediated inflammatory pathways. They exhibit certain structural requirements for the specific inhibition of NOD2 signaling (representative compounds 27–29 are shown in Figure 8). The benzyllic substituent at position C3 and the lipophilic side chain at position C7 are very important for both specificity and optimal activity on NOD2. It is noteworthy that the benzylamines are considerably more active than their ether congeners.

Moreover, bulky side chains at position C7 are not tolerated, whereas compounds with a small 2-methyl-2-propenyl, allyl, or propargyl side chain exhibit pronounced inhibition of NOD-signaling. On the other hand, the stereochemistry at the benzyllic position was found not to be essential for NOD2 specificity but appeared to affect the activity. Compounds lacking the benzyl group blocked all pathways tested. Strikingly, the presence of the Cr(CO)₃ moiety significantly affected the cytotoxicity, since the decomplexed ligand exhibited a pronounced cytotoxic effect.¹⁶⁴,¹⁷¹ Compound 27 was the most potent and specific inhibitor of the series, with IC₅₀ of 5–15 μM, depending on the cell line. Since increased concentrations of MDP diminished the

inhibitory effect of this compound, the authors speculated that it acts at the level of NOD2 in a competitive manner.

The nondruglike properties of organochromium compounds could, however, limit their use in vivo. Further, selectivity against NOD1 was not tested, so the compounds cannot be regarded as NOD2-selective inhibitors.¹⁶⁴,¹⁷¹

4.6. Overview of NOD1/NOD2 Inhibition Assays. Several assays were developed in order to evaluate the compounds for their potential inhibitory activity. HEK293T cells express low levels of endogenous NOD1 and NOD2; however, they do not express TLR2, -4, and -8 and therefore do not respond to stimulation with other classical bacterial PAMPs, in particular to contaminations with LPS. The primary HTS assay employing NOD1-overexpressing HEK293T cells transfected with NF-κB-driven luciferase reporter gene was described, which measures luciferase activity upon exposure to compounds and the subsequent stimulation with TriDAP as a measure of modulation of NOD1 activity. An analogous HTS luciferase reporter assay employing NOD2-overexpressing HEK293T cells measuring modulation of NOD2-dependent activation upon stimulation with MDP was also developed. Both assays (depicted in Scheme 3) can also be used as the initial counterscreen when searching for either NOD1- or NOD2-specific inhibitors.¹⁶⁵,¹⁶⁶,¹⁶⁸

Moreover, an adaptation of the classical luciferase reporter gene assay employing an NF-κB reporter plasmid and a constitutive β-galactosidase plasmid has recently been reported to measure NOD1- and NOD2-mediated NF-κB activation. The β-galactosidase reporter assay allows for variations in the

Journal of Medicinal Chemistry

**Scheme 4. Potential Target Sites for Inhibition of NOD Signaling**

- **MDP/IEDAP**
- **1.) ligand binding**
  - benzimidazole diamides
  - arene-chromium complexes
  - Noditinib
- **2.) ATP binding**
  - xanthine class of NOD1 inhibitors?
  - SM-122
  - Withaferin A
  - Geldanamycin
  - Saracatinib
- **NOD1/NOD2**
- **3.) self-oligomerization**
  - curcumin
  - parthenolide
  - DHA
  - EPA
- **4.) recruitment of RIP2/RIP2 kinase activity**
  - SB203580
  - gefitinib
  - erlotinib
- **5.) regulation of signal transduction**
  - POSITIVE REGULATORS
  - NEGATIVE REGULATORS
- **activation of NF-κB**
- **activation of MAPKs**
- **activation of IRF**

transfection efficiency and to monitor cell death (eventually induced by experimental settings). Both reporter assay methods use transfected HEK293T cells overexpressing NOD1 or NOD2, which yield a robust readout; thus, the methods are well suited as HTS primary assay. Of note, Grimes et al. developed a different surface plasmon resonance-based assay to screen for inhibitors/activators of the NOD2 signaling pathway using recombinant NOD2.

In terms of functional characterization of the compounds, secondary assays were developed to measure their effect on the authentic downstream effect of NF-κB activation, namely, the production of IL-8 in TriDAP- or MDP-stimulated human breast cancer epithelial cell line MCF-7 overexpressing NOD1 or NOD2, respectively. In addition, colonic epithelial HCT116 cells express both NOD1 and NOD2 and respond to their activation with increased NF-κB activity and secretion of IL-8 but do not respond to agonists of TLR2 and TLR4 receptors and therefore also represent a suitable model for the functional characterization of NOD1 as well as NOD2 ligands.

Finally, assays for specificity of the inhibitors against activation of NF-κB through other non-NOD pathways in the NF-κB driven luciferase reporter cell line were also developed (e.g., TNF-α assay).

### 5. RATIONAL TARGETING OF THE NOD SIGNALING PATHWAY

The NLR signaling pathway is one of a multitude of pathways that converge on the IKK complex and activate the transcription factor NF-κB. NOD1 and NOD2 are among the best studied NLR family proteins, but the therapeutic potential of pharmacological modulation of NOD1 and NOD2 is largely unexplored, raising several questions. Can other downstream targets also be exploited for clinical gain? Would global inhibition be preferred over inhibition of a specific downstream effector? Deeper insights into the underlying mechanisms of NOD activation and its regulation are essential for understanding NOD-related diseases and for the design of pharmacological agents targeting NOD signaling (potential target sites are shown in Scheme 4).

#### 5.1. Targeting the Ligand Binding
NOD1 and NOD2 ligands bind to the LRR domains of these proteins, but no crystal structure of ligand-bound LRR is available, making rational structure-based design impossible, although homology models of NOD1 and NOD2 have been published. The recently discovered selective inhibitors of NOD2 signaling of the benzimidazole diamide and hydrophenalene structural classes compete with MDP for binding to NOD2, suggesting that they bind to the same site, while the first selective NOD1 inhibitor has been shown to bind to NOD1 and alter its conformation and subcellular localization. Competition experiments with TriDAP have not been performed, although the inability of ligand-bound NOD1 to migrate to the plasma membrane suggests that it binds to the LRR domain, which is responsible for the membrane interaction of NOD1. Given that most NOD-related diseases are not caused by excessive ligand concentration but usually by mutations in genes coding for NOD1 and NOD2 that lead to constitutive activation, targeting of ligand binding would appear to be pointless. In fact, the most rational target has to be located downstream of ligand binding.

#### 5.2. Targeting the ATP-Binding Site
There are many successful examples in medicinal chemistry of how to target nucleotide-binding domains of proteins by small-molecule drugs, so it is reasonable to assume that the NOD proteins could

Journal of Medicinal Chemistry

![Chemical Structures](chemical_structures.png)

Figure 9. Inhibitors of NOD signaling that function via targets other than NOD receptors.

similarly be targeted. Small-molecule inhibitors targeting the ATP binding pocket of the catalytic domain of kinases have potential to become drugs devoid of major side effects; however, one of the most important challenges is the optimization of selectivity.¹⁷³ In order to become activated, both NOD1 and NOD2 have to bind ATP.¹³ Nucleotide binding and hydrolysis are both important in the activation and deactivation of signaling.¹⁵ The ATP binding site is thus an interesting target, since mutations predicted to disrupt ATP binding have also prevented activation of NOD signaling.¹³,¹⁵ However, one has to take into consideration that when mutated, ATP hydrolysis but not ATP binding is usually abolished.⁹ This suggests that small molecules targeting nucleotide binding or hydrolysis specifically could have both inhibiting and activating effects. The mechanism of action of 16, a representative of the second known structural class of NOD1 inhibitors, purine-2,6-diones, has not been described. Considering its similarity to nucleotides, it could be speculated that it binds to the nucleotide-binding domain of NOD1.

**5.3. Targeting the NOD Self-Oligomerization.** Several natural products (curcumin, parthenolide) have been reported to inhibit NOD oligomerization, supposedly by preventing the formation of the disulfide bonds joining NACHT domains.¹⁵⁸ However, they also act via unrelated mechanisms and inhibit several other inflammatory pathways. As opposed to lauric acid, which induces NOD signaling, the polyunsaturated fatty acids DHA and EPA have been strikingly demonstrated to inhibit this process by interfering with NOD oligomerization.¹⁵⁶

**5.4. Targeting the RIP2 Recruitment/RIP2 Kinase Activity.** The signaling pathways downstream of NOD1 and NOD2 oligomerization converge on the signaling kinase RIP2, which is an interesting modulator of synergistic and inhibitory interactions between the two receptors and is a viable target for pharmacological intervention with RIP2 selective inhibitors, such as SB203580 (31, Figure 9).³³,¹⁵³,¹⁷⁴ A recent study has reported the unexpected finding of RIP2 tyrosine kinase activity. The epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors, gefitinib (32) and erlotinib (33), have been shown to inhibit both RIP2 tyrosine phosphorylation and MDP-induced cytokine release at nanomolar concentrations, indicating their possible use in treating inflammatory disorders.¹⁷⁵ In fact,

they have exhibited a significant in vitro efficiency in alleviating the exacerbated cytokine responses in NOD2 hyperactive states (primary macrophages that lack ITCH and macrophage cell lines overexpressing the BS R334Q mutation).¹⁷⁵

Further downstream is another kinase, TAK1, involved in NOD signaling, which is an interesting target, since several TAK1 inhibitors (e.g., (5Z)-7-oxozeaenol (34)) have already been reported.³³ However, this inhibition would be highly non-selective, since both NLR and TLR pathways converge on TAK1, which makes it an inappropriate target for NOD related diseases. The compounds that exhibit nonselective activity and inhibit not only the NOD1 but also NOD2- and TNFα-mediated NF-κB activation may hold great potential for use as general anti-inflammatory agents, as dual or even multiple inhibitors in the low micromolar range.

**5.5. Targeting the Regulatory Network of Signal Transduction.** Activation and deactivation of NOD signaling are tightly controlled by posttranslational modifications, in particular by ubiquitination and phosphorylation of RIP2 in a site-specific and time-specific manner. Hence, NOD-associated hyperactive states could also be treated by targeting the ubiquitin-regulated signaling network.⁶⁷,¹⁷⁶ Numerous small molecule inhibitors of both cIAPs and XIAP, such as SM-122 (35), are currently in clinical trials as anticancer agents.¹⁷⁷ Since they correct excessive NOD responses in vitro, they could also be examined for down-regulation of NOD signaling.⁶⁷,¹⁷⁶ In fact, cIAP1 inhibitor has been shown as efficacious in correcting the excessive cytokine responses in NOD2 hyperactive disease models.⁶⁷ However, even if these inhibitors were repurposed for down-regulation of NOD2 signaling, they would represent a less-specific alternative to NOD2 inhibitors.¹⁷⁶

Numerous proteins are known as positive and negative regulators of NOD signaling, but none are selective pharmacologic targets for modulation of NOD function, since they play multiple roles in the cell machinery. Two strategies could be employed to inhibit NOD signaling: activation of negative regulators or suppression of positive regulators. In general, it is easier to design protein inhibitors, which makes the suppression of positive regulators a viable therapeutic strategy. However, the TRAF4 binding motif in the structure of NOD2, composed of two consecutive glutamate residues Glu279 and Glu280, could
also be targeted. TRAF4 acts as a negative regulator of NOD2; hence, a rationally designed TRAF4 agonistic peptidomimetic might provide an opportunity to selectively inhibit NOD2. Further insight into the regulation of NOD membrane trafficking may provide novel therapeutic strategies. Vimentin acts as a positive regulator of NOD2 signaling by supervising plasma membrane trafficking. Withaferin A (36), a steroidal lactone, can inhibit vimentin and thus also selectively inhibit NOD2 signaling. Vimentin thus represents a potential selective therapeutic target for treating NOD2-hyperactive states. Moreover, Hsp90 has been shown to positively regulate NOD1 and NOD2. It has been shown to maintain NOD2 stability, since its inhibition led to the proteasomal degradation of NOD2. However, it plays a major role in many other cellular processes and hence may not be suitable as a target, although numerous Hsp90 inhibitors such as geldanamycin (37) exist and have been designed as anticancer agents, which are currently in clinical trials. This target also lacks NOD1 versus NOD2 selectivity. Finally, proto-oncogene tyrosine-protein kinase (Src) has been shown to phosphorylate RIP2 in the presence of GEF-H1 which acts as an intersection between Src kinase function and NLR signaling. Several selective Src kinase inhibitors, such as saracatinib (38), are already known and could be used to treat NOD2-hyperactive states.

### 6. FUTURE DIRECTIONS AND CONCLUSIONS

A number of potential targets reside within the innate immune system, a major junction of host defense, inflammation, and apoptosis. Innate immunity is characterized by complex networks, rather than simple linear pathways, of interacting proteins that are involved in its regulation, so side effects of compounds influencing innate immunity pathways have to be anticipated. The target has to be selected to minimize the possibility of side effects, and to enhance the overall safety of this approach, the compounds were designed to specifically and selectively bind to that target. With this in mind, targeted modulation of the innate immune response holds great potential for the treatment of chronic inflammatory and autoimmune diseases, characterized by increased NF-κB activation. Dysregulation of NLRs is implicated in the pathology of various inflammatory disorders, and since therapeutic intervention of NLR dysfunction is currently only possible by the administration of strong broad-range anti-inflammatory drugs, which often display severe side effects, new, specific modulators of NLR signaling are essential. However, suppressing the NOD1 or NOD2 activity is likely to impair host defense, raising the question of safety. Targeting the upstream pathways responsible for NF-κB signaling is expected to leave most innate immunity defense mechanisms intact while still being beneficial for diseases in which the specific target is involved in the disease pathogenesis. It is also worth mentioning that several anticancer agents, including EGFR, Src, and Hsp90 inhibitors, which incidentally all belong to the class of ATP-competitive kinase inhibitors, have shown potential in correcting excessive NOD2 responses; therefore, they could be repurposed as inhibitors of NOD2 signaling.

Novel and selective NOD inhibitors, however, are urgently needed. Two distinct chemical classes of selective NOD1 inhibitors, 2-aminobenzimidazole and purine-2,6-dione, have already been identified. However, only the benzimidazole diamides have so far been established as selective NOD2 inhibitors. More insight into the structures of NOD1 and NOD2 is needed to pave ways for the development of inhibitors.

What is more, there is structural data that indicate that NOD1 CARD domains form dimers. This CARD-CARD interaction surface might also be exploited in terms of achieving NOD1 inhibition. Apart from medicinal chemistry, much remains to be done. Further in vitro as well as in vivo disease model experiments are urgently needed to investigate and establish their clinical usefulness, since little is known for certain about the actual efficacy of NOD inhibitors in NOD-hyperactive states.

In conclusion, there is a strong basis for suggesting that NOD1 and NOD2 inhibitors could have an important therapeutic role. Moreover, these inhibitors can be used as pharmacological probes in research to define their role in disease and in normal host-defense mechanisms. Harnessing the full potential of NOD inhibitors certainly constitutes a “hot topic” in medicinal chemistry.

#### AUTHOR INFORMATION

**Corresponding Author**

*Phone: +386 1 4769 574. Fax: +386 1 4258 031. E-mail: ziga.jakopin@ffa.uni-lj.si.*

**Notes**

The authors declare no competing financial interest.

**Biography**

Ziga Jakopin received his degree in Pharmacy in 2005 and his Ph.D. in Pharmaceutical Sciences in 2010 from the University of Ljubljana (Faculty of Pharmacy, Ljubljana, Slovenia). At present he is an Assistant Professor of Medicinal Chemistry at the Faculty of Pharmacy, University of Ljubljana. His main research interests concern design and synthesis of peptidomimetics and immunomodulatory compounds. His research focus at present is on the development of new ligands (agonists, inhibitors) of NOD receptors.

#### ACKNOWLEDGMENTS

This work was supported financially by the Slovenian Research Agency (Grant P1-0208). The author thanks Dr. Roger Pain for critical reading of the manuscript. The author apologizes to the authors of important contributions, which have been left out because of space constraints.

#### ABBREVIATIONS USED

A20, ubiquitin E3 ligase A20; AAA+, extended ATPase associated with various cellular activities; AAMP, angio-associated migratory protein; ADMET, absorption, distribution, metabolism, excretion, and toxicity; Ankrd17, ankyrin repeat-containing protein; AP-1, activator protein 1; ASC, apoptosis-associated specklike protein containing caspase-recruitment domain; ATG16L1, autophagy related 16-like 1; β-PIX, p21-activated kinase interacting exchange factor; BS, Blau syndrome; CAD, carbamoyl phosphate synthetase/aspartate transcarbamylase/dihydroorotase; CARD, caspase-recruitment domain; CD, Crohn's disease; CENTβ1, centaurin β1; cIAP, cellular inhibitor of apoptosis; DAMP, danger-associated molecular pattern; DHA, docosahexaenoic acid; EGFR, epidermal growth factor receptor; EOS, early onset sarcoidosis; EPA, eicosapentaenoic acid; ERK, extracellular signal-regulated kinase; FRMPD2, FERM and PDZ domain-containing 2 protein; GEF-H1, guanine nucleotide exchange factor H1; GRIM-19, eukaryotic gene associated with retinoic-interferon-induced mortality 19; Hsp90, heat shock protein 90; HTS, high-throughput screening; iE-DAP, dipeptide D-Glu-meso-DAP; IFN, interferon; IKK, IkB kinase; IL, interleukin; Ipaf, interleukin-1β-converting enzyme protease-activating factor; IRF, interferon regulatory factor;

dx.doi.org/10.1021/jm401841p J. Med. Chem. 2014, 57, 6897–6918
ITCH, itchy E3 ubiquitin protein ligase; JNK, c-Jun N-terminal kinase; JNKBP1, c-Jun N-terminal kinase binding protein; LPS, lipopolysaccharide; LRR, leucine-rich repeat; LUBAC, linear ubiquitin assembly complex; MAPK, mitogen-associated protein kinase; MAVS, mitochondrial antiviral signaling; MDP, muramyl dipeptide; MHC II, major histocompatibility complex II; MEKK4, mitogen-activated protein kinase kinase 4; MoA, mode of action; NACHT, nucleotide binding domain; NADPH, nicotinamide adenine dinucleotide phosphate; NAID, nucleotide-binding oligomerization domain 2 associated auto-inflammatory disease; NF-κB, nuclear factor κB; NEMO, nuclear factor κB essential modulator; NIH, National Institutes of Health; NLR, nucleotide binding oligomerization domain like receptor; NLRC4, nucleotide binding oligomerization domain like receptor family caspase-recruitment-domain-containing protein 4; NLRP, NACHT-, LRR-, and PYD-domain-containing protein; NOD, nucleotide-binding oligomerization domain; Pak, p21-activated kinase; PAMP, pathogen-associated molecular pattern; PAMPA, parallel artificial membrane permeability assay; PGN, peptidoglycan; PP2A, protein phosphatase 2A; PRDX4, peroxiredoxin 4; PRR, pattern recognition receptor; PSMA-7, proteasome subunit α type 7; PTPN22, protein tyrosine phosphatase non-receptor type 22; Rac1, Ras-related C3 botulinum toxin substrate 1; RIG-I, retinoic acid induced gene I; RIP2, receptor-interacting protein 2; RLR, retinoic acid induced gene I like receptor; SAR, structure–activity relationship; SGT-1, suppressor of the G2 allele of Skp1; SHIP-1, Src homology 2 domain-containing inositol 5-phosphatase 1; siRNA, small interfering ribonucleic acid; SNP, single nucleotide polymorphism; SQSTM1, sequestosome 1; Src, proto-oncogene tyrosine-protein kinase; STAND, signal transduction ATPase with numerous domains; TAB, transforming growth factor binding protein; TAK1, transforming growth factor β activated kinase 1; TLR, Toll-like receptor; TNF-α, tumor necrosis factor α; TRAF, tumor necrosis factor receptor associated factor; TriDAP, L-Ala-γ-D-Glu-meso-DAP; TRIM, tripartite motif-containing protein; XIAP, X-linked inhibitor of apoptosis

### REFERENCES

(1) Newton, K.; Dixit, V. M. Signaling in innate immunity and inflammation. *Cold Spring Harbor Perspect. Biol.* 2012, 4, a006049.

(2) Fritz, J. H.; Ferrero, R. L.; Philpott, D. J.; Girardin, S. E. Nod-like proteins in immunity, inflammation and disease. *Nat. Immunol.* 2006, 7, 1250–1257.

(3) Inohara, N.; Nuñez, G. NODs: intracellular proteins involved in inflammation and apoptosis. *Nat. Rev. Immunol.* 2003, 3, 371–382.

(4) Proell, M.; Riedl, S. J.; Fritz, J. H.; Rojas, A. M.; Schwarzenbacher, R. The Nod-like receptor (NLR) family: a tale of similarities and differences. *PLoS One* 2008, 3, e2119.

(5) Schroder, K.; Tschopp, J. The inflammasomes. *Cell* 2010, 140, 821–832.

(6) Hanson, P. I.; Whiteheart, S. W. AAA+ proteins: have engine, will work. *Nat. Rev. Mol. Cell Biol.* 2005, 6, 519–529.

(7) Ammelburg, M.; Frickey, T.; Lupas, A. N. Classification of AAA+ proteins. *J. Struct. Biol.* 2006, 156, 2–11.

(8) Leipe, D. D.; Koonin, E. V.; Aravind, L. STAND, a class of P-loop NTPases including animal and plant regulators of programmed cell death: multiple, complex domain architectures, unusual phylogenetic patterns, and evolution by horizontal gene transfer. *J. Mol. Biol.* 2004, 343, 1–28.

(9) Wendler, P.; Ciniawsky, S.; Kock, M.; Kube, S. Structure and function of the AAA+ nucleotide binding pocket. *Biochem. Biophys. Acta* 2012, 1823, 2–14.

(10) Laroui, H.; Yan, Y.; Narui, Y.; Ingersoll, S. A.; Ayyadurai, S.; Charania, M. A.; Zhou, F.; Wang, B.; Salaita, K.; Sitaraman, S. V.; Merlin,

D. L-Ala-γ-D-Glu-meso-diaminopimelic acid (DAP) interacts directly with leucine-rich region domain of nucleotide-binding oligomerization domain 1, increasing phosphorylation activity of receptor-interacting serine/threonine-protein kinase 2 and its interaction with nucleotide-binding oligomerization domain 1. *J. Biol. Chem.* 2011, 286, 31003–31013.

(11) Grimes, C. L.; De Zoysa Ariyananda, L.; Melnyk, J. E.; O’Shea, E. K. The innate immune protein Nod2 binds directly to MDP, a bacterial cell wall fragment. *J. Am. Chem. Soc.* 2012, 134, 13535–13537.

(12) Tanabe, T.; Chamaillard, M.; Ogura, Y.; Zhu, L.; Qiu, S.; Masumoto, J.; Ghosh, P.; Moran, A.; Predergast, M. M.; Tromp, G.; Williams, C. J.; Inohara, N.; Nuñez, G. Regulatory regions and critical residues of NOD2 involved in muramyl dipeptide recognition. *EMBO J.* 2004, 23, 1587–1597.

(13) Mo, J.; Boyle, J. P.; Howard, C. B.; Monie, T. P.; Davis, B. K.; Duncan, J. A. Pathogen sensing by nucleotide-binding oligomerization domain-containing protein 2 (NOD2) is mediated by direct binding to muramyl dipeptide and ATP. *J. Biol. Chem.* 2012, 287, 23057–23067.

(14) Askari, N.; Correa, R. G.; Zhai, D.; Reed, J. C. Expression, purification, and characterization of recombinant NOD1 (NLRC1): a NLR family member. *J. Biotechnol.* 2012, 157, 75–81.

(15) Zurek, B.; Proell, M.; Wagner, R. N.; Schwarzenbacher, R.; Kufer, T. A. Mutational analysis of human NOD1 and NOD2 NACHT domains reveals different modes of activation. *Innate Immun.* 2012, 18, 100–111.

(16) Mayor, A.; Martinon, F.; De Smedt, T.; Petrilli, V.; Tschopp, J. A crucial function for SGT1 and Hsp90 in inflammasome activity links mammalian and plant innate immune responses. *Nat. Immunol.* 2007, 8, 497–503.

(17) da Silva Correia, J.; Miranda, Y.; Leonard, N.; Ulevitch, R. SGT is essential for Nod1 activation. *Proc. Natl. Acad. Sci. U.S.A.* 2007, 104, 6764–6769.

(18) Lee, K. H.; Biswas, A.; Liu, Y. J.; Kobayashi, K. S. Proteasomal degradation of Nod2 protein mediates tolerance to bacterial cell wall components. *J. Biol. Chem.* 2012, 287, 39800–39811.

(19) Srimathi, T.; Robbins, S. L.; Dubas, R. L.; Hasegawa, M.; Inohara, N.; Park, Y. C. Monomer/dimer transition of the caspase-recruitment domain of human Nod1. *Biochemistry* 2008, 47, 1319–1325.

(20) Barnich, N.; Aguirre, J. E.; Reinecker, H. C.; Xavier, R.; Podolsky, D. K. Membrane recruitment of NOD2 in intestinal epithelial cells is essential for nuclear factor-{kappa}B activation in muramyl dipeptide recognition. *J. Cell Biol.* 2005, 170, 21–26.

(21) Kufer, T. A.; Kremmer, E.; Adam, A. C.; Philpott, D. J.; Sansonetti, P. J. The pattern-recognition molecule Nod1 is localized at the plasma membrane at sites of bacterial interaction. *Cell. Microbiol.* 2008, 10, 477–486.

(22) Lécine, P.; Esmiol, S.; Métais, J. Y.; Nicoletti, C.; Nourry, C.; McDonald, C.; Nuñez, G.; Hugot, J. P.; Borg, J. P.; Ollendorff, V. The NOD2-RICK complex signals from the plasma membrane. *J. Biol. Chem.* 2007, 282, 15197–15207.

(23) Philpott, D. J.; Sorbara, M. T.; Robertson, S. J.; Croitoru, K.; Girardin, S. E. NOD proteins: regulators of inflammation in health and disease. *Nat. Rev. Immunol.* 2014, 14, 9–23.

(24) Hasegawa, M.; Fujimoto, Y.; Lucas, P. C.; Nakano, H.; Fukase, K.; Nuñez, G.; Inohara, N. A critical role of RICK/RIP2 polyubiquitination in Nod-induced NF-κB activation. *EMBO J.* 2008, 27, 373–383.

(25) Inohara, N.; Koseki, T.; Lin, J.; del Peso, L.; Lucas, P. C.; Chen, F.; Ogura, Y.; Nuñez, G. An induced proximity model for NF-kappaB activation in the Nod1/RICK and RIP signaling pathways. *J. Biol. Chem.* 2000, 275, 27823–27831.

(26) Abbott, D. W.; Wilkins, A.; Asara, J. M.; Cantley, L. C. The Crohn’s disease protein, NOD2, requires RIP2 in order to induce ubiquitinylation of a novel site on NEMO. *Curr. Biol.* 2004, 14, 2217–2227.

(27) Fridh, V.; Rittinger, K. The tandem CARDs of NOD2: intramolecular interactions and recognition of RIP2. *PLoS One* 2012, 7, e34375.

(28) Abbott, D. W.; Yang, Y.; Hutti, J. E.; Madhavarapu, S.; Kelliher, M. A.; Cantley, L. C. Coordinated regulation of Toll-like receptor and

dx.doi.org/10.1021/jm401841p | *J. Med. Chem.* 2014, 57, 6897–6918

Journal of Medicinal Chemistry

NOD2 signaling by K63-linked polyubiquitin chains. Mol. Cell. Biol. 2007, 27, 6012–6025.

(29) Bertrand, M.J.M.; Doiron, K.; Labbé, K.; Komeluk, R.G.; Barker, P.A.; Saleh, M. Cellular inhibitors of apoptosis cIAP1 and cIAP2 are required for innate immunity signaling by the pattern recognition receptors NOD1 and NOD2. Immunity 2009, 30, 789–801.

(30) Krieg, A.; Correa, R.G.; Garrison, J.B.; Le Negrate, G.; Welsh, K.; Huang, Z.; Knoefel, W.T.; Reed, J.C. XIAP mediates NOD signaling via interaction with RIP2. Proc. Natl. Acad. Sci. U.S.A. 2009, 106, 14524–14529.

(31) Damgaard, R.B.; Nachbur, U.; Yabal, M.; Wong, W.W.; Fiil, B.K.; Kastirr, M.; Rieser, E.; Rickard, J.A.; Bankovacki, A.; Peschel, C.; Ruland, J.; Bekker-Jensen, S.; Mailand, N.; Kaufmann, T.; Strasser, A.; Walczak, H.; Silke, J.; Jost, P.J.; Gyrd-Hansen, M. The ubiquitin ligase XIAP recruits LUBAC for NOD2 signaling in inflammation and innate immunity. Mol. Cell 2012, 46, 746–758.

(32) Adhikari, A.; Xu, M.; Chen, Z.J. Ubiquitin-mediated activation of TAK1 and IKK. Oncogene 2007, 26, 3214–3226.

(33) Windheim, M.; Lang, C.; Peggie, M.; Plater, L.A.; Cohen, P. Molecular mechanisms involved in the regulation of cytokine production by muramyl dipeptide. Biochem. J. 2007, 404, 179–190.

(34) Strober, W.; Murray, P.J.; Kitani, A.; Watanabe, T. Signalling pathways and molecular interactions of NOD1 and NOD2. Nat. Rev. Immunol. 2006, 6, 9–20.

(35) Kobayashi, K.S.; Chamaillard, M.; Ogura, Y.; Henegariu, O.; Inohara, N.; Nuñez, G.; Flavell, R.A. Nod2-dependent regulation of innate and adaptive immunity in the intestinal tract. Science 2005, 307, 731–734.

(36) da Silva Correia, J.; Miranda, Y.; Leonard, N.; Hsu, J.; Ulevitch, R. J. Regulation of Nod1-mediated signaling pathways. Cell Death Differ. 2007, 14, 830–839.

(37) Hsu, Y.M.; Zhang, Y.; You, Y.; Wang, D.; Li, H.; Duramad, O.; Qin, X.F.; Dong, C.; Lin, X. The adaptor protein CARD9 is required for innate immune responses to intracellular pathogens. Nat. Immunol. 2007, 8, 198–205.

(38) Correa, R.G.; Milutinovic, S.; Reed, J.C. Roles of NOD1 (NLRC1) and NOD2 (NLRC2) in innate immunity and inflammatory diseases. Biosci. Rep. 2012, 608, 597–608.

(39) Shaw, M.H.; Kamada, N.; Warner, N.; Kim, Y.G.; Nuñez, G. The ever-expanding function of NOD2: autophagy, viral recognition, and T cell activation. Trends Immunol. 2011, 32, 73–79.

(40) Sabbah, A.; Chang, T.H.; Harnack, R.; Frohlich, V.; Tominaga, K.; Dube, P.H.; Xiang, Y.; Bose, S. Activation of innate immune antiviral responses by Nod2. Nat. Immunol. 2009, 10, 1073–1080.

(41) Watanabe, T.; Asano, N.; Fichtner-Feigl, S.; Gorelick, P.L.; Tsuji, Y.; Matsumoto, Y.; Chiba, T.; Fuss, I.J.; Kitani, A.; Strober, W. NOD1 contributes to mouse host defense against Helicobacter pylori via induction of type I IFN and activation of the ISGF3 signaling pathway. J. Clin. Invest. 2010, 120, 1645–1662.

(42) Travassos, L.H.; Carneiro, L.A.; Ramjeet, M.; Hussey, S.; Kim, Y.G.; Magalhães, J.G.; Yuan, L.; Soares, F.; Chea, E.; Le Bourhis, L.; Boneca, I.G.; Allaoui, A.; Jones, N.L.; Nuñez, G.; Girardin, S.E.; Philpott, D.J. Nod1 and Nod2 direct autophagy by recruiting ATG16L1 to the plasma membrane at the site of bacterial entry. Nat. Immunol. 2010, 11, 55–62.

(43) Homer, C.R.; Kabi, A.; Marina-Garcia, N.; Sreekumar, A.; Nesvizhskii, A.I.; Nickerson, K.P.; Chinnaiyan, A.M.; Nuñez, G.; McDonald, C. A dual role for receptor-interacting protein kinase 2 (RIP2) kinase activity in nucleotide-binding oligomerization domain 2 (NOD2)-dependent autophagy. J. Biol. Chem. 2013, 287, 25565–25576.

(44) Sorbara, M.T.; Ellison, L.K.; Ramjeet, M.; Travassos, L.H.; Jones, N.L.; Girardin, S.E.; Philpott, D.J. The protein ATG16L1 suppresses inflammatory cytokines induced by the intracellular sensors Nod1 and Nod2 in an autophagy-independent manner. Immunity 2013, 39, 858–873.

(45) Cooney, R.; Baker, J.; Brain, O.; Danis, B.; Pichulik, T.; Allan, P.; Ferguson, D.J.; Campbell, B.J.; Jewell, D.; Simmons, A. NOD2 stimulation induces autophagy in dendritic cells influencing bacterial handling and antigen presentation. Nat. Med. 2010, 16, 90–97.

(46) da Silva Correia, J.; Miranda, Y.; Austin-Brown, N.; Hsu, J.; Mathison, J.; Xiang, R.; Zhou, H.; Li, Q.; Han, J.; Ulevitch, R.J. Nod1-dependent control of tumor growth. Proc. Natl. Acad. Sci. U.S.A. 2006, 103, 1840–1845.

(47) Hsu, L.C.; Ali, S.R.; McGillivray, S.; Tseng, P.H.; Mariathasan, S.; Humke, E.W.; Eckmann, L.; Powell, J.J.; Dixit, V.M.; Karin, M.A. NOD2-NALP1 complex mediates caspase-1-dependent IL-1beta secretion in response to Bacillus anthracis infection and muramyl dipeptide. Proc. Natl. Acad. Sci. U.S.A. 2008, 105, 7803–7808.

(48) Pan, Q.; Mathison, J.; Fears, C.; Kravchenko, V.V.; da Silva Correia, J.; Hoffman, H.M.; Kobayashi, K.S.; Bertin, J.; Grant, E.P.; Coyle, A.J.; Sutterwala, F.S.; Ogura, Y.; Flavell, R.A.; Ulevitch, R.J. MDP-induced interleukin-1β processing requires Nod2 and CIAS/NALP3. J. Leukocyte Biol. 2007, 82, 177–183.

(49) Yoo, N.J.; Park, W.S.; Kim, S.Y.; Reed, J.C.; Son, S.G.; Lee, J.Y.; Lee, S.H. Nod1, a CARD protein, enhances pro-interleukin-1beta processing through the interaction with pro-caspase-1. Biochem. Biophys. Res. Commun. 2002, 299, 652–658.

(50) Latz, E.; Xiao, T.S.; Stutz, A. Activation and regulation of the inflammasomes. Nat. Rev. Immunol. 2013, 13, 397–411.

(51) Magalhaes, J.G.; Philpott, D.J.; Nahori, M.A.; Jehanno, M.; Fritz, J.; Le Bourhis, L.; Viala, J.; Hugot, J.P.; Giovannini, M.; Bertin, J.; Lepoivre, M.; Mengin-Lecreulx, D.; Sansonetti, P.J.; Girardin, S.E. Murine Nod1 but not its human orthologue mediates innate immune detection of tracheal cytotoxin. EMBO Rep. 2005, 12, 1201–1207.

(52) Eitel, J.; Krüll, M.; Hocke, A.C.; N'Guessan, P.D.; Zahlten, J.; Schmeck, B.; Slevogt, H.; Hippenstiel, S.; Suttrop, N.; Opitz, B. Beta-PIX and Rac1 GTPase mediate trafficking and negative regulation of NOD2. J. Immunol. 2008, 181, 2664–2671.

(53) McDonald, C.; Chen, F.F.; Ollendorff, V.; Ogura, Y.; Marchetto, S.; Lécine, P.; Borg, J.P.; Nuñez, G. A role for erbin in the regulation of Nod2-dependent NF-kappaB signaling. J. Biol. Chem. 2005, 280, 40301–40309.

(54) Kufer, T.A.; Kremmer, E.; Banks, D.J.; Philpott, D.J. Role for erbin in bacterial activation of Nod2. Infect. Immun. 2006, 74, 3115–3124.

(55) Legrand-Poels, S.; Kustermans, G.; Bex, F.; Kremmer, E.; Kufer, T.A.; Piette, J. Modulation of Nod2-dependent NF-kappaB signaling by the actin cytoskeleton. J. Cell Sci. 2007, 120, 1299–1310.

(56) Till, A.; Rosenstiel, P.; Bräutigam, K.; Sina, C.; Jacobs, G.; Oberg, H.H.; Seegert, D.; Chakraborty, T.; Schreiber, S. A role for membrane-bound CD147 in Nod2-mediated recognition of bacterial cytoinvasion. J. Cell Sci. 2008, 121, 487–495.

(57) Stevens, C.; Henderson, P.; Nimmo, E.R.; Soares, D.C.; Dogan, B.; Simpson, K.W. The intermediate filament protein, vimentin, is a regulator of NOD2 activity. Gut 2013, 62, 695–707.

(58) Lipinski, S.; Grabe, N.; Jacobs, G.; Billmann-Born, S.; Till, A.; Häsler, R.; Aden, K.; Paulsen, M.; Arlt, A.; Kraemer, L.; Hagemann, N.; Erdmann, K.S.; Schreiber, S.; Rosenstiel, P. RNAi screening identifies mediators of NOD2 signaling: implications for spatial specificity of MDP recognition. Proc. Natl. Acad. Sci. U.S.A. 2012, 109, 21426–21431.

(59) von Kampen, O.; Lipinski, S.; Till, A.; Martin, S.J.; Nietfeld, W.; Lehrach, H.; Schreiber, S.; Rosenstiel, P. Caspase recruitment domain-containing protein 8 (CARD8) negatively regulates NOD2-mediated signaling. J. Biol. Chem. 2010, 285, 19921–19926.

(60) Lecat, A.; Di Valentin, E.; Somja, J.; Jourdan, S.; Fillet, M.; Kufer, T.A.; Habraken, Y.; Sadzot, C.; Louis, E.; Delvenne, P.; Piette, J.; Legrand-Poels, S. The c-Jun N-terminal kinase (JNK)-binding protein (JNKBP1) acts as negative regulator of NOD2 protein signaling by inhibiting its oligomerization process. J. Biol. Chem. 2012, 287, 29213–29226.

(61) Rosenstiel, P.; Huse, K.; Till, A.; Hampe, J.; Hellmig, S.; Sina, C.; Billmann, S.; von Kampen, O.; Waetzig, G.H.; Platzer, M.; Seegert, D.; Schreiber, S. A short isoform of NOD2/CARD15, NOD2-S, is an endogenous inhibitor of NOD2/receptor-interacting protein kinase 2-induced signaling pathways. Proc. Natl. Acad. Sci. U.S.A. 2006, 103, 3280–3285.

(62) Kramer, M.; Boeck, J.; Reichenbach, D.; Kaether, C.; Schreiber, S.; Platzer, M.; Rosenstiel, P.; Huse, K. NOD2-C2—a novel NOD2 isoform activating NF-κB in a muramyl dipeptide-independent manner. BMC Res. Notes 2010, 3, 224–234.

(63) Park, S.; Ha, S. D.; Coleman, M.; Meshkibaf, S.; Kim, S. O. p62/SQSTM1 enhances NOD2-mediated signaling and cytokine production through stabilizing NOD2 oligomerization. PLoS One 2013, 8, e57138.

(64) Hahn, J. S. Regulation of Nod1 by Hsp90 chaperone complex. FEBS Lett. 2005, 579, 4513–4519.

(65) Damiano, J. S.; Oliveira, V.; Welsh, K.; Reed, J. C. Heterotypic interactions among NACHT domains: implications for regulation of innate immune responses. Biochem. J. 2004, 381, 213–219.

(66) Tao, M.; Scacheri, P. C.; Marinis, J. M.; Harhaj, E. W.; Matesic, L. E.; Abbott, D. W. ITCH K63-ubiquitinates the NOD2 binding protein, RIP2, to influence inflammatory signaling pathways. Curr. Biol. 2009, 19, 1255–1263.

(67) Tigno-Aranjuez, J. T.; Bai, X.; Abbott, D. W. A discrete ubiquitin-mediated network regulates the strength of NOD2 signaling. Mol. Cell. Biol. 2012, 33, 146–158.

(68) Condé, C.; Rambout, X.; Lebrun, M.; Lecat, A.; Di Valentin, E.; Dequiedt, F.; Piette, J.; Gloire, G.; Legrand, S. The inositol phosphatase SHIP-1 inhibits NOD2-induced NF-κB activation by disturbing the interaction of XIAP with RIP2. PLoS One 2012, 7, e41005.

(69) Hitotsomatsu, O.; Ahmad, R. C.; Tavares, R.; Wang, M.; Philpott, D.; Turer, E. E.; Lee, B. L.; Shiffin, N.; Advincula, R.; Malynn, B. A.; Werts, C.; Ma, A. The ubiquitin-editing enzyme A20 restricts nucleotide-binding oligomerization domain containing 2-triggered signals. Immunity 2008, 28, 381–390.

(70) Hutti, J. E.; Turk, B. E.; Asara, J. M.; Ma, A.; Cantley, L. C.; Abbott, D. W. IkappaB kinase beta phosphorylates the K63 deubiquitinating enzyme A20 to cause feedback inhibition of the NF-kappaB pathway. Mol. Cell. Biol. 2007, 27, 7451–7461.

(71) Zurek, B.; Schoultz, I.; Neeerincx, A.; Napolitano, L. M.; Birkner, K.; Bennek, E.; Sellge, G.; Lerm, M.; Meroni, G.; Söderholm, J. D.; Kufer, T. A. TRIM27 negatively regulates NOD2 by ubiquitination and proteasomal degradation. PLoS One 2012, 7, e41255.

(72) Marinis, J. M.; Homer, C. R.; McDonald, C.; Abbott, D. W. A novel motif in the Crohn’s disease susceptibility protein, NOD2, allows TRAF4 to down-regulate innate immune responses. J. Biol. Chem. 2011, 286, 1938–1950.

(73) Yang, L.; Tang, Z.; Zhang, H.; Kou, W.; Lu, Z.; Li, X.; Li, Q.; Miao, Z. PSMA7 directly interacts with NOD1 and regulates its function. Cell. Physiol. Biochem. 2013, 31, 952–959.

(74) Yang, S.; Wang, B.; Humphries, F.; Jackson, R.; Healy, M. E.; Bergin, R.; Aviello, G.; Hall, B.; McNamara, D.; Darby, T.; Quinlan, A.; Shanahan, F.; Melgar, S.; Fallon, P. G.; Moynagh, P. N. Pellino3 ubiquitinates RIP2 and mediates Nod2-induced signaling and protective effects in colitis. Nat. Immunol. 2013, 14, 927–936.

(75) Ver Heul, A. M.; Fowler, C. A.; Ramaswamy, S.; Piper, R. C. Ubiquitin regulates caspase recruitment domain-mediated signaling by nucleotide-binding oligomerization domain-containing proteins NOD1 and NOD2. J. Biol. Chem. 2013, 288, 6890–6902.

(76) Bielig, H.; Zurek, B.; Kutsch, A.; Menning, M.; Philpott, D. J.; Sansonetti, P. J.; Kufer, T. A. A function for AAMP in Nod2-mediated NF-kappaB activation. Mol. Immunol. 2009, 46, 2647–2654.

(77) Clark, N. M.; Marinis, J. M.; Cobb, B. A.; Abbott, D. W. MEKK4 sequesters RIP2 to dictate NOD2 signal specificity. Curr. Biol. 2008, 18, 1402–1408.

(78) Morosky, S. A.; Zhu, J.; Mukherjee, A.; Sarkar, S. N.; Coyne, C. B. Retinoic acid-induced gene-I (RIG-I) associates with nucleotide-binding oligomerization domain-2 (NOD2) to negatively regulate inflammatory signaling. J. Biol. Chem. 2011, 286, 28574–28583.

(79) Zhao, Y.; Alonso, C.; Ballester, I.; Song, J. H.; Chang, S. Y.; Guleng, B.; Arihiro, S.; Murray, P. J.; Xavier, R.; Kobayashi, K. S.; Reinecker, H. C. Control of NOD2 and Rip2-dependent innate immune activation by GEF-H1. Inflammatory Bowel Dis. 2012, 18, 602–612.

(80) Fukazawa, A.; Alonso, C.; Kurachi, K.; Gupta, S.; Lesser, C. F.; McCormick, B. A.; Reinecker, H. C. GEF-H1 mediated control of

NOD1 dependent NF-κB activation by Shigella effectors. PLoS Pathog. 2008, 4, e1000228.

(81) Weichert, D.; Gobom, J.; Klopfleisch, S.; Häsler, R.; Gustavsson, N.; Billmann, S.; Lehrach, H.; Seegert, D.; Schreiber, S.; Rosenstiel, P. Analysis of NOD2-mediated proteome response to muramyl dipeptide in HEK293 cells. J. Biol. Chem. 2006, 281, 2380–2389.

(82) Yamamoto-Furusho, J. K.; Barnich, N.; Xavier, R.; Hisamatsu, T.; Podolsky, D. K. Centaurin beta1 down-regulates nucleotide-binding oligomerization domains 1- and 2-dependent NF-kappaB activation. J. Biol. Chem. 2006, 281, 36060–36070.

(83) Richmond, A. L.; Kabi, A.; Homer, C. R.; Marina-Garcia, N.; Nickerson, K. P.; Nesvizhskii, A. I.; Sreekumar, A.; Chinnaiyan, A. M.; Nuñez, G.; McDonald, C. The nucleotide synthesis enzyme CAD inhibits NOD2 antibacterial function in human intestinal epithelial cells. Gastroenterology 2012, 142, 1483–1492.

(84) Spalinger, M. R.; Lang, S.; Vavricka, S. R.; Fried, M.; Rogler, G.; Scharl, M. Protein tyrosine phosphatase non-receptor type 22 modulates NOD2-induced cytokine release and autophagy. PLoS One 2013, 8, e72384.

(85) Menning, M.; Kufer, T. A. A role for the ankyrin repeat-containing protein Ankrd17 in Nod1- and Nod2-mediated inflammatory responses. FEBS Lett. 2013, 587, 2137–2142.

(86) Barnich, N.; Hisamatsu, T.; Aguirre, J. E.; Xavier, R.; Reinecker, H. C.; Podolsky, D. K. GRIM-19 interacts with nucleotide oligomerization domain 2 and serves as downstream effector of antibacterial function in intestinal epithelial cells. J. Biol. Chem. 2005, 280, 19021–19026.

(87) Warner, N.; Burberry, A.; Franchi, L.; Kim, Y. G.; McDonald, C.; Sartor, M. A.; Nuñez, G. A genome-wide siRNA screen reveals positive and negative regulators of the NOD2 and NF-κB signaling pathways. Sci. Signaling 2013, 6, rs3.

(88) Moreira, L. O.; Zamboni, D. S. NOD1 and NOD2 signaling in infection and inflammation. Front. Immunol. 2012, 3, 328.

(89) Girardin, S. E.; Boneca, I. G.; Carneiro, L. A. M.; Antignac, A.; Jéhanno, M.; Viala, J.; Tedin, K.; Taha, M. K.; Labigne, A.; Zähringer, U.; Coyle, A. J.; DiStefano, P. S.; Bertin, J.; Sansonetti, P. J.; Philpott, D. J. Nod1 detects a unique muropeptide from Gram-negative bacterial peptidoglycan. Science 2003, 300, 1584–1587.

(90) Girardin, S. E.; Travassos, L. H.; Hervé, M.; Blanot, D.; Boneca, I. G.; Philpott, D. J.; Sansonetti, P. J.; Mengin-Lecreulx, D. Peptidoglycan molecular requirements allowing detection by Nod1 and Nod2. J. Biol. Chem. 2003, 278, 41702–41708.

(91) Girardin, S. E.; Jéhanno, M.; Mengin-Lecreulx, D.; Sansonetti, P. J.; Alzari, P. M.; Philpott, D. J. Identification of the critical residues involved in peptidoglycan detection by Nod1. J. Biol. Chem. 2005, 280, 38648–38656.

(92) Chamaillard, M.; Hashimoto, M.; Horie, Y.; Masumoto, J.; Qiu, S.; Saab, L.; Ogura, Y.; Kawasaki, A.; Fukase, K.; Kusumoto, S.; Valvano, M. A.; Foster, S. J.; Mak, T. W.; Nuñez, G.; Inohara, N. An essential role for NOD1 in host recognition of bacterial peptidoglycan containing diaminopimelic acid. Nat. Immunol. 2003, 4, 702–707.

(93) Inohara, N.; Ogura, Y.; Fontalba, A.; Gutierrez, O.; Pons, F.; Crespo, J.; Fukase, K.; Inamura, S.; Kusumoto, S.; Hashimoto, M.; Foster, S. J.; Moran, A. P.; Fernandez-Luna, J. L.; Nuñez, G. Host recognition of bacterial muramyl dipeptide mediated through NOD2. J. Biol. Chem. 2003, 278, 5509–5512.

(94) Girardin, S. E.; Boneca, I. G.; Viala, J.; Chamaillard, M.; Labigne, A.; Thomas, G.; Philpott, D. J.; Sansonetti, P. J. Nod2 is a general sensor of peptidoglycan through muramyl dipeptide (MDP) detection. J. Biol. Chem. 2003, 278, 8869–8872.

(95) Viala, J.; Chaput, C.; Boneca, I. G.; Cardona, A.; Girardin, S. E.; Moran, A. P.; Athman, R.; Mémet, S.; Huerre, M. R.; Coyle, A. J.; DiStefano, P. S.; Sansonetti, P. J.; Labigne, A.; Bertin, J.; Philpott, D. J.; Ferrero, R. L. Nod1 responds to peptidoglycan delivered by the Helicobacter pylori cag pathogenicity island. Nat. Immunol. 2004, 5, 1166–1174.

(96) Ismair, M. G.; Vavricka, S. R.; Kullak-Ublick, G. A.; Fried, M.; Mengin-Lecreulx, D.; Girardin, S. E. hPepT1 selectively transports muramyl dipeptide but not Nod1-activating muramyl peptides. Can. J. Physiol. Pharmacol. 2006, 84, 1313–1319.

(97) Dalmasso, G.; Nguyen, H. T.; Charrier-Hisamuddin, L.; Yan, Y.; Laroui, H.; Demoulin, B.; Sitaraman, S. V.; Merlin, D. PepT1 mediates transport of the proinflammatory bacterial tripeptide L-Ala-{gamma}-D-Glu-meso-DAP in intestinal epithelial cells. *Am. J. Physiol.: Gastrointest. Liver Physiol.* 2010, 299, G687–G696.

(98) Ingersoll, S. A.; Ayyadurai, S.; Charania, M. A.; Laroui, H.; Yan, Y.; Merlin, D. The role and pathophysiological relevance of membrane transporter PepT1 in intestinal inflammation and inflammatory bowel disease. *Am. J. Physiol.: Gastrointest. Liver Physiol.* 2012, 302, G484–492.

(99) Swaan, P. W.; Bensman, T.; Bahadurri, P. M.; Hall, M. W.; Sarkar, A.; Bao, S.; Khantwal, C. M.; Ekins, S.; Knoell, D. L. Bacterial peptide recognition and immune activation facilitated by human peptide transporter PEPT2. *Am. J. Respir. Cell Mol. Biol.* 2008, 39, 536–542.

(100) Lee, J.; Tattoli, I.; Wojtal, K. A.; Vavricka, S. R.; Philpott, D. J.; Girardin, S. E. pH-dependent internalization of muramyl peptides from early endosomes enables Nod1 and Nod2 signaling. *J. Biol. Chem.* 2009, 284, 23818–23829.

(101) Marina-García, N.; Franchi, L.; Kim, Y. G.; Hu, Y.; Smith, D. E.; Boons, G. J.; Nuñez, G. Clathrin- and dynamin-dependent endocytic pathway regulates muramyl dipeptide internalization and NOD2 activation. *J. Immunol.* 2009, 182, 4321–4327.

(102) Kaparakis, M.; Turnbull, L.; Carneiro, L.; Firth, S.; Coleman, H. A.; Parkington, H. C.; Le Bourhis, L.; Karrar, A.; Viala, J.; Mak, J.; Hutton, M. L.; Davies, J. K.; Crack, P. J.; Hertzog, P. J.; Philpott, D. J.; Girardin, S. E.; Whitchurch, C. B.; Ferrero, R. L. Bacterial membrane vesicles deliver peptidoglycan to NOD1 in epithelial cells. *Cell. Microbiol.* 2010, 12, 372–385.

(103) Bielig, H.; Rompikuntal, P. K.; Dongre, M.; Zurek, B.; Lindmark, B.; Ramstedt, M.; Wai, S. N.; Kufer, T. A. NOD-like receptor activation by outer membrane vesicles from Vibrio cholerae non-O1 non-O139 strains is modulated by the quorum-sensing regulator HapR. *Infect. Immun.* 2011, 79, 1418–1427.

(104) Masumoto, J.; Yang, K.; Varambally, S.; Hasegawa, M.; Tomlins, S. A.; Qiu, S.; Fujimoto, Y.; Kawasaki, A.; Foster, S. J.; Horie, Y.; Mak, T. W.; Nuñez, G.; Chinnaiyan, A. M.; Fukase, K.; Inohara, N. Nod1 acts as an intracellular receptor to stimulate chemokine production and neutrophil recruitment in vivo. *J. Exp. Med.* 2006, 203, 203–213.

(105) Sorbara, M. T.; Philpott, D. J. Peptidoglycan: a critical activator of the mammalian immune system during infection and homeostasis. *Immunol. Rev.* 2011, 243, 40–60.

(106) Lipinski, S.; Till, A.; Sina, C.; Arlt, A.; Grasberger, H.; Schreiber, S.; Rosenstiel, P. DUOX2-derived reactive oxygen species are effectors of NOD2-mediated antibacterial responses. *J. Cell Sci.* 2009, 122, 3522–3530.

(107) Kim, Y. G.; Park, J. H.; Shaw, M. H.; Franchi, L.; Inohara, N.; Núñez, G. The cytosolic sensors Nod1 and Nod2 are critical for bacterial recognition and host defense after exposure to Toll-like receptor ligands. *Immunity* 2008, 28, 246–257.

(108) van Heel, D. A.; Hunt, K. A.; Ghosh, S.; Hervé, M.; Playford, R. J. Normal responses to specific NOD1-activating peptidoglycan agonists in the presence of the NOD2 frameshift and other mutations in Crohn’s disease. *Eur. J. Immunol.* 2006, 36, 1629–1635.

(109) Jakopin, Ž.; Gobec, M.; Mlinarič-Raščan, I.; Sollner Dolenc, M. Immunomodulatory properties of novel nucleotide oligomerization domain 2 (Nod2) agonistic desmuramyldipeptides. *J. Med. Chem.* 2012, 55, 6478–6488.

(110) Jakopin, Ž.; Corsini, E.; Gobec, M.; Mlinarič-Raščan, I.; Sollner Dolenc, M. Design, synthesis and biological evaluation of novel desmuramyldipeptide analogs. *Eur. J. Med. Chem.* 2011, 46, 3762–3777.

(111) Jakopin, Ž.; Gobec, M.; Kodela, J.; Hazdovac, T.; Mlinarič-Raščan, I.; Sollner Dolenc, M. Synthesis of conformationally constrained γ-D-glutamyl-meso-diaminopimelic acid derivatives as ligands of nucleotide-binding oligomerization domain protein 1 (Nod1). *Eur. J. Med. Chem.* 2013, 69, 232–243.

(112) Uehara, A.; Sugawara, Y.; Kurata, S.; Fujimoto, Y.; Fukase, K.; Kusumoto, S.; Satta, Y.; Sasano, T.; Sugawara, S.; Takada, H. Chemically synthesized pathogen-associated molecular patterns increase the expression of peptidoglycan recognition proteins via Toll-like receptors,

NOD1 and NOD2 in human oral epithelial cells. *Cell. Microbiol.* 2005, 7, 675–686.

(113) Fritz, J. H.; Girardin, S. E.; Fitting, C.; Werts, C.; Mengin-Lecreulx, D.; Caroff, M.; Cavaillon, J. M.; Philpott, D. J.; Adib-Conguy, M. Synergistic stimulation of human monocytes and dendritic cells by Toll-like receptor 4 and Nod1- and Nod2-activating agonists. *Eur. J. Immunol.* 2005, 35, 2459–2470.

(114) Tada, H.; Aiba, S.; Shibata, K.; Ohteki, T.; Takada, H. Synergistic effect of Nod1 and Nod2 agonists with Toll-like receptor agonists on human dendritic cells to generate interleukin-12 and T helper type 1 cells. *Infect. Immun.* 2005, 73, 7967–7976.

(115) Fritz, J. H.; Le Bourhis, L.; Sellge, G.; Magalhaes, J. G.; Fsihi, H.; Kufer, T. A. Nod1-mediated innate immune recognition of peptidoglycan contributes to the onset of adaptive immunity. *Immunity* 2007, 26, 445–459.

(116) Magalhaes, J. G.; Fritz, J. H.; Le Bourhis, L.; Sellge, G.; Travassos, L. H.; Selvanantham, T.; Girardin, S. E.; Gommerman, J. L.; Philpott, D. J. Nod2-dependent Th2 polarization of antigen-specific immunity. *J. Immunol.* 2008, 181, 7925–7935.

(117) Tang, H.; Amara, U.; Tang, D.; Barnes, M. A.; McDonald, C.; Nagy, L. E. Synergistic interaction between C5a and NOD2 signaling in the regulation of chemokine expression in RAW 264.7 macrophages. *Adv. Biosci. Biotechnol.* 2013, 4, 30–37.

(118) Lohr, N. J.; Molleston, J. P.; Strauss, K. A.; Torres-Martinez, W.; Sherman, E. A.; Squires, R. H.; Rider, N. L.; Chikwava, K. R.; Cummings, O. W.; Morton, D. H.; Puffenberger, E. G. Human ITCH E3 ubiquitin ligase deficiency causes syndromic multisystem autoimmune disease. *Am. J. Hum. Genet.* 2010, 86, 447–453.

(119) Lesage, S.; Zouali, H.; Cezard, J. P.; Colombel, J. F.; Belaiche, J.; Almer, S.; Tysk, C.; O’Morain, C.; Gassull, M.; Binder, V.; Finkel, Y.; Modigliani, R.; Gower-Rousseau, C.; Macry, J.; Merlin, F.; Chamaillard, M.; Jannot, A. S.; Thomas, G.; Hugot, J. P.; EPWG-IBD Group; EPIMAD Group; GETAID Group.. CARD15/NOD2 mutational analysis and genotype-phenotype correlation in 612 patients with inflammatory bowel disease. *Am. J. Hum. Genet.* 2002, 70, 845–857.

(120) Hugot, J. P.; Chamaillard, M.; Zouali, H.; Lesage, S.; Cezard, J. P.; Belaiche, J.; Almer, S.; Tysk, C.; O’Morain, C. A.; Gassull, M.; Binder, V.; Finkel, Y.; Cortot, A.; Modigliani, R.; Laurent-Puig, P.; Gower-Rousseau, C.; Macry, J.; Colombel, J. F.; Sahbatou, M.; Thomas, G. Association of NOD2 leucine-rich repeat variants with susceptibility to Crohn’s disease. *Nature* 2001, 411, 599–603.

(121) Ogura, Y.; Bonen, D. K.; Inohara, N.; Nicolae, D. L.; Chen, F. F.; Ramos, R.; Britton, H.; Moran, T.; Karaliuskas, R.; Duerr, R. H.; Achkar, J. P.; Brant, S. R.; Bayless, T. M.; Kirschner, B. S.; Hanauer, S. B.; Nuñez, G.; Cho, J. H. A frameshift mutation in NOD2 associated with susceptibility to Crohn’s disease. *Nature* 2001, 411, 603–606.

(122) Henckaerts, L.; Vermeire, S. NOD2/CARD15 disease associations other than Crohn’s disease. *Inflammatory Bowel Dis.* 2007, 13, 235–241.

(123) van Duist, M. M.; Albrecht, M.; Podswiadek, M.; Giachino, D.; Lengauer, T.; Punzi, L.; De Marchi, M. A new CARD15 mutation in Blau syndrome. *Eur. J. Hum. Genet.* 2005, 13, 742–747.

(124) Miceli-Richard, C.; Lesage, S.; Rybojad, M.; Prieur, A. M.; Manouvrier-Hanu, S.; Häfner, R.; Chamaillard, M.; Zouali, H.; Thomas, G.; Hugot, J. P. CARD15 mutations in Blau syndrome. *Nat. Genet.* 2001, 29, 19–20.

(125) Kanazawa, N.; Okafuji, I.; Kambe, N.; Nishikomori, R.; Nakata-Hizume, M.; Nagai, S.; Fuji, A.; Yuasa, T.; Manki, A.; Sakurai, Y.; Nakajima, M.; Kobayashi, H.; Fujiwara, I.; Tsutsumi, H.; Utani, A.; Nishigori, C.; Heike, T.; Nakahata, T.; Miyachi, Y. Early-onset sarcoidosis and CARD15 mutations with constitutive nuclear factor-kappaB activation: common genetic etiology with Blau syndrome. *Blood* 2005, 105, 1195–1197.

(126) Joosten, L. A.; Heinhuis, B.; Abdollahi-Roodsaz, S.; Ferwerda, G.; Lebourhis, L.; Philpott, D. J.; Nahori, M. A.; Popa, C.; Morre, S. A.; van der Meer, J. W.; Girardin, S. E.; Netea, M. G.; van der Berg, W. B. Differential function of the NACHT-LRR (NLR) members Nod1 and Nod2 in arthritis. *Proc. Natl. Acad. Sci. U.S.A.* 2008, 105, 9017–9022.

(127) Vieira, S. M.; Cunha, T. M.; Franca, R. F.; Pinto, L. G.; Talbot, J.; Turato, W. M.; Lemos, H. P.; Lima, J. B.; Verri, W. A., Jr; Almeida, S. C.; Ferreira, S. H.; Louzada-Junior, P.; Zamboni, D. S.; Cunha, F. Q. Joint NOD2/RIPK2 signaling regulates IL-17 axis and contributes to the development of experimental arthritis. *J. Immunol.* 2012, 188, 5116–5122.

(128) Yao, Q. Nucleotide-binding oligomerization domain containing 2: structure, function, and diseases. *Semin. Arthritis Rheum.* 2013, 43, 125–130.

(129) Weidinger, S.; Klopp, N.; Rümmler, L.; Wagenpfeil, S.; Baurecht, H. J.; Gauger, A.; Darsow, U.; Jakob, T.; Novak, N.; Schäfer, T.; Heinrich, J.; Behrendt, H.; Wichmann, H. E.; Ring, J.; Illig, T.; KORA Study Group. Association of CARD15 polymorphisms with atopy-related traits in a population based cohort of Caucasian adults. *Clin. Exp. Allergy* 2005, 35, 866–872.

(130) Rebane, A.; Zimmermann, M.; Aab, A.; Baurecht, H.; Koreck, A.; Karelson, M.; Abram, K.; Metsalu, T.; Pihlap, M.; Meyer, N.; Fölster-Holst, R.; Nagy, N.; Kemeny, L.; Kingo, K.; Vilo, J.; Illig, T.; Akdis, M.; Franke, A.; Novak, N.; Weidinger, S.; Akdis, C. A. Mechanisms of IFN-γ-induced apoptosis of human skin keratinocytes in patients with atopic dermatitis. *J. Allergy Clin. Immunol.* 2012, 129, 1297–1306.

(131) Macaluso, F.; Nothnagel, M.; Parwez, Q.; Petrasch-Parwez, E.; Bechara, F. G.; Epplen, J. T.; Epplen, J. T.; Hoffjan, S. Polymorphisms in NACHT-LRR (NLR) genes in atopic dermatitis. *Exp. Dermatol.* 2007, 16, 692–698.

(132) De Jager, P. L.; Graham, R.; Farwell, L.; Sawcer, S.; Richardson, A.; Behrens, T. W.; Compston, A.; Hafler, D. A.; Kere, J.; Vyse, T. J.; Rioux, J. D. The role of inflammatory bowel disease loci in multiple sclerosis and systemic lupus erythematosus. *Genes Immun.* 2006, 7, 327–334.

(133) Kutikhin, A. G. Role of NOD1/CARD4 and NOD2/CARD15 gene polymorphisms in cancer etiology. *Hum. Immunol.* 2011, 72, 955–968.

(134) Kang, M.J.; Heo, S.K.; Song, E.J.; Kim, D.J.; Han, S.Y.; Han, J.H.; Kim, B.Y.; Park, J.H. Activation of Nod1 and Nod2 induces innate immune responses of prostate epithelial cells. *Prostate* 2012, 72, 1351–1358.

(135) Ashton, K.A.; Proietto, A.; Otton, G.; Symonds, I.; McEvoy, M.; Attia, J.; Gilbert, M.; Hamann, U.; Scott, R.J. Polymorphisms in genes of the steroid hormone biosynthesis and metabolism pathways and endometrial cancer risk. *Cancer Epidemiol.* 2010, 34, 328–337.

(136) Kurzawski, G.; Suchy, J.; Kladny, J.; Grabowska, E.; Mierzejewski, M.; Jakubowska, A.; Debiak, T.; Cybulski, C.; Kowalska, E.; Szych, Z.; Domagala, W.; Scott, R.J.; Lubinski, J. The NOD2 3020insC mutation and the risk of colorectal cancer. *Cancer Res.* 2004, 64, 1604–1606.

(137) Möckelmann, N.; von Schönfels, W.; Buch, S.; von Kampen, O.; Sipos, B.; Egberts, J.H.; Rosenstiel, P.; Franke, A.; Brosch, M.; Hinz, S.; Röder, C.; Kalthoff, H.; Fölsch, U.R.; Krawczak, M.; Schreiber, S.; Bröring, C.D.; Tepel, J.; Schafmayer, C.; Hampe, J. Investigation of innate immunity genes CARD4, CARD8 and CARD15 as germline susceptibility factors for colorectal cancer. *BMC Gastroenterol.* 2009, 9, 79.

(138) Berrington, W.R.; Macdonald, M.; Khadge, S.; Sapkota, B.R.; Janer, M.; Hagge, D.A.; Kaplan, G.; Hawn, T.R. Common polymorphisms in the NOD2 gene region are associated with leprosy and its reactive states. *J. Infect. Dis.* 2010, 201, 1422–1435.

(139) Austin, C.M.; Ma, X.; Graviss, E.A. Common nonsynonymous polymorphisms in the NOD2 gene are associated with resistance or susceptibility to tuberculosis disease in African Americans. *J. Infect. Dis.* 2008, 197, 1713–1716.

(140) Azad, A.K.; Sadee, W.; Schlesinger, L.S. Innate immune gene polymorphisms in tuberculosis. *Infect. Immun.* 2012, 80, 3343–3359.

(141) Chen, G.J.; Shaw, M.H.; Redondo, G.; Nuñez, G. The innate immune receptor Nod1 protects the intestine from inflammation-induced tumorigenesis. *Cancer Res.* 2008, 68, 10060–10067.

(142) Hysi, P.; Kabesch, M.; Moffatt, M.F.; Schedel, M.; Carr, D.; Zhang, Y.; Boardman, B.; von Mutius, E.; Weiland, S.K.; Leupold, W.; Fritzsch, C.; Klopp, N.; Musk, A.W.; James, A.; Nuñez, G.; Inohara, N.


Cookson, W.O. NOD1 variation, immunoglobulin E and asthma. *Hum. Mol. Genet.* 2005, 14, 935–941.

(143) McGovern, D.P.; Hysi, P.; Ahmad, T.; van Heel, D.A.; Moffatt, M.F.; Carey, A.; Cookson, W.O.; Jewell, D.P. Association between a complex insertion/deletion polymorphism in NOD1 (CARD4) and susceptibility to inflammatory bowel disease. *Hum. Mol. Genet.* 2005, 14, 1245–1250.

(144) Weidinger, S.; Klopp, N.; Rümmler, L.; Wagenpfeil, S.; Novak, N.; Baurecht, H.J.; Groer, W.; Darsow, U.; Heinrich, J.; Gauger, A.; Schäfer, T.; Jakob, T.; Behrendt, H.; Wichmann, H.E.; Ring, J.; Illig, T. Association of NOD1 polymorphisms with atopic eczema and related phenotypes. *J. Allergy Clin. Immunol.* 2005, 116, 177–184.

(145) Shaw, P.J.; Barr, M.J.; Lukens, J.R.; McGargill, M.A.; Chi, H.; Mak, T.W.; Kanneganti, T.D. Signaling via the RIP2 adaptor protein in central nervous system-infiltrating dendritic cells promotes inflammation and autoimmunity. *Immunity* 2011, 34, 75–84.

(146) Nishio, H.; Kanno, S.; Onoyama, S.; Ikeda, K.; Tanaka, T.; Kusuhara, K.; Fujimoto, Y.; Fukase, K.; Sueishi, K.; Hara, T. NOD1 ligands induce site-specific vascular inflammation. *Arterioscler., Thromb., Vasc. Biol.* 2011, 31, 1093–1099.

(147) Gatheral, T.; Reed, D.M.; Moreno, L.; Gough, P.J.; Votta, B.J.; Sehon, C.A.; Rickard, D.J.; Bertin, J.; Lim, E.; Nicholson, A.G.; Mitchell, J.A. A key role for the endothelium in NOD1 mediated vascular inflammation: comparison to TLR4 responses. *PLoS One* 2012, 7, e42386.

(148) Fernández-Velasco, M.; Prieto, P.; Terrón, V.; Benito, G.; Flores, J.M.; Delgado, C.; Zaragoza, C.; Lavin, B.; Gómez-Parrizas, M.; López-Collazo, E.; Martín-Sanz, P.; Boscá, L. NOD1 activation induces cardiac dysfunction and modulates cardiac fibrosis and cardiomyocyte apoptosis. *PLoS One* 2012, 7, e45260.

(149) Rosenzweig, H.L.; Galster, K.T.; Planck, S.R.; Rosenbaum, J.T. NOD1 expression in the eye and functional contribution to IL-1β-dependent ocular inflammation in mice. *Invest. Ophthalmol. Visual Sci.* 2009, 50, 1746–1753.

(150) Cardenas, I.; Mulla, M.J.; Myrtolli, L.; Sfakianaki, A.K.; Norwitz, E.R.; Tadesse, S.; Guller, S.; Abrahamas, V.M. NOD1 activation by bacterial iE-DAP induces maternal-fetal inflammation and preterm labor. *J. Immunol.* 2011, 187, 980–986.

(151) Schertzer, J.D.; Tamrakar, A.K.; Magalhaes, J.G.; Pereira, S.; Bilan, P.J.; Fullerton, M.D.; Liu, Z.; Steinberg, G.R.; Giacca, A.; Philpott, D.J.; Klip, A. NOD1 activators link innate immunity to insulin resistance. *Diabetes* 2011, 60, 2206–2215.

(152) Zhao, L.; Hu, P.; Zhou, Y.; Purohit, J.; Hwang, D. NOD1 activation induces proinflammatory gene expression and insulin resistance in 3T3-L1 adipocytes. *Am. J. Physiol. Endocrinol. Metab.* 2011, 301, E587–E598.

(153) Moreno, L.; Gatheral, T. Therapeutic targeting of NOD1 receptors. *Br. J. Pharmacol.* 2013, 170, 475–485.

(154) Geddes, K.; Magalhães, J.G.; Girardin, S.E. Unleashing the therapeutic potential of NOD-like receptors. *Nat. Rev. Drug Discovery* 2009, 8, 465–479.

(155) Ulevitch, R.J. Therapeutics targeting the innate immune system. *Nat. Rev. Immunol.* 2004, 4, 512–520.

(156) Zhao, L.; Kwon, M.-J.; Huang, S.; Lee, J.Y.; Fukase, K.; Inohara, N.; Hwang, D.H. Differential modulation of Nods signaling pathways by fatty acids in human colonic epithelial HCT116 cells. *J. Biol. Chem.* 2007, 282, 11618–11628.

(157) Feng, B.S.; He, S.H.; Zheng, P.Y.; Wu, L.; Yang, P.C. Mast cells play a crucial role in Staphylococcus aureus peptidoglycan-induced diarrhea. *Am. J. Pathol.* 2007, 171, 537–547.

(158) Huang, S.; Zhao, L.; Kim, K.; Lee, D.S.; Hwang, D.H. Inhibition of Nod2 signaling and target gene expression by curcumin. *Mol. Pharmacol.* 2008, 74, 274–281.

(159) Rüngeler, P.; Castro, V.; Mora, G.; Gören, N.; Vichnewski, W.; Pahl, H.L.; Merfort, I.; Schmidt, T.J. Inhibition of transcription factor NF-kB by sesquiterpene lactones: a proposed mechanism of action. *Bioorg. Med. Chem.* 1999, 7, 2343–2352.

(160) Juliana, C.; Fernandes-Alnemri, T.; Wu, J.; Datta, P.; Solorzano, L.; Yu, J.W.; Meng, R.; Quong, A.A.; Latz, E.; Scott, C.P.; Alnemri, E.S.

Anti-inflammatory compounds parthenolide and Bay 11-7082 are direct inhibitors of the inflammasome. *J. Biol. Chem.* 2010, 285, 9792–9802.

(161) Koch, E.; Klaas, C. A.; Rüngeler, P.; Castro, V.; Mora, G.; Vichnewski, W.; Merfort, I. Inhibition of inflammatory cytokine production and lymphocyte proliferation correlates with their effect on activation of NF-κB. *Biochem. Pharmacol.* 2001, 62, 795–801.

(162) Lyβ, G.; Knorre, A.; Schmidt, T. J.; Pahl, H. L.; Merfort, I. The anti-inflammatory sesquiterpene lactone helenalin inhibits the transcription factor NF-κB by directly targeting p65. *J. Biol. Chem.* 1998, 273, 33508–33516.

(163) Mayer, A. M.; Jacobson, P. B.; Fenical, W.; Jacobs, R. S.; Glaser, K. B. Pharmacological characterization of the pseudopterosins: novel anti-inflammatory natural products isolated from the caribbean soft coral, *Pseudopterogorgia elisabethae*. *Life Sci.* 1998, 62, PL401–407.

(164) Bielig, H.; Velder, J.; Saiai, A.; Menning, M.; Meemboor, S.; Kalka-Moll, W.; Kronke, M.; Schmalz, H. G.; Kufer, T. A. Anti-inflammatory arene-chromium complexes acting as specific inhibitors of NOD2 signalling. *ChemMedChem* 2010, 5, 2065–2071.

(165) Magnuson, G.; Khan, P.; Yuan, H.; Brown, B.; Divlianska, D. B.; Stonich, D.; Peddibhotla, S.; Su, Y.; Dad, S.; Sergienko, E.; Chung, T. D. Y.; Roth, G. P.; Wimer, C.; Diaz, P.; Correa, R. G.; Reed, J. C. High throughput screening assays for NOD1 inhibitors—probe 1. *Probe Reports from the NIH Molecular Libraries Program; NIH: Bethesda, MD*, 2010.

(166) Khan, P. M.; Correa, R. G.; Divlianska, D. B.; Peddibhotla, S.; Sessions, E. H.; Magnuson, G.; Brown, B.; Suyama, E.; Yuan, H.; Mangravita-Novo, A.; Vicchiarelli, M.; Su, Y.; Vasile, S.; Smith, L. H.; Diaz, P. W.; Reed, J. C.; Roth, G. P. Identification of inhibitors of NOD1-induced nuclear factor-κB activation. *ACS Med. Chem. Lett.* 2011, 2, 780–785.

(167) Correa, R. G.; Khan, P. M.; Askari, N.; Zhai, D.; Gerlic, M.; Brown, B.; Magnuson, G.; Spreafico, R.; Albani, S.; Sergienko, E.; Diaz, P. W.; Roth, G. P.; Reed, J. C. Discovery and characterization of 2-aminobenzimidazole derivatives as selective NOD1 inhibitors. *Chem. Biol.* 2011, 18, 825–832.

(168) Magnuson, G.; Khan, P.; Yuan, H.; Brown, B.; Divlianska, D. B.; Stonich, D.; Peddibhotla, S.; Su, Y.; Dad, S.; Sergienko, E.; Chung, T. D. Y.; Roth, G. P.; Wimer, C.; Diaz, P.; Correa, R. G.; Reed, J. C. High throughput screening assays for NOD1 inhibitors—probe 2. *Probe Reports from the NIH Molecular Libraries Program; NIH: Bethesda, MD*, 2010.

(169) Hershberger, P. M.; Peddibhotla, S.; Sessions, E. H.; Divlianska, D. B.; Correa, R. G.; Pinkerton, A. B.; Reed, J. C.; Roth, G. P. Synthesis and physicochemical characterization of novel phenotypic probes targeting the nuclear factor-kappa B signaling pathway. *Beilstein J. Org. Chem.* 2013, 9, 900–907.

(170) Rickard, D. J.; Sehon, C. A.; Kasparcova, V.; Kallal, L. A.; Zeng, X.; Montoute, M. N.; Chordia, T.; Poore, D. D.; Li, H.; Wu, Z.; Eidam, P. M.; Haile, P. A.; Yu, J.; Emery, J. G.; Marquis, R. W.; Gough, P. J.; Bertin, J. Identification of benzimidazole diamides as selective inhibitors of the nucleotide-binding oligomerization domain 2 (NOD2) signaling pathway. *PLoS One* 2013, 8, e69619.

(171) Saiai, A.; Bielig, H.; Velder, J.; Neudörfl, J.-M.; Menning, M.; Kufer, T. A.; Schmalz, H.-G. Hydrophenalene-Cr(CO)₃ complexes as anti-inflammatory agents based on specific inhibition of NOD2 signalling: a SAR study. *MedChemComm* 2012, 3, 1377–1385.

(172) Zurek, B.; Bielig, H.; Kufer, T. A. Cell-based reporter assay to analyze activation of Nod1 and Nod2. *Methods Mol. Biol.* 2011, 748, 107–119.

(173) Huang, D.; Zhou, T.; Lafleur, K.; Nevado, C.; Caflisch, A. Kinase selectivity potential for inhibitors targeting the ATP binding site: a network analysis. *Bioinformatics* 2010, 26, 198–204.

(174) Jun, J. C.; Cominelli, F.; Abbott, D. W. RIP2 activity in inflammatory disease and implications for novel therapeutics. *J. Leukocyte Biol.* 2013, 94, 927–932.

(175) Tigno-Aranjuez, J. T.; Asara, J. M.; Abbott, D. W. Inhibition of RIP2's tyrosine kinase activity limits NOD2-driven cytokine responses. *Genes Dev.* 2010, 24, 2666–2677.

(176) Tigno-Aranjuez, J. T.; Abbott, D. W. Ubiquitination and phosphorylation in the regulation of NOD2 signaling and NOD2-mediated disease. *Biochim. Biophys. Acta* 2012, 1823, 2022–2028.

(177) Sun, H.; Nikolovska-Coleska, Z.; Yang, C. J.; Qian, D.; Lu, J.; Qiu, S.; Bai, L.; Peng, Y.; Cai, Q.; Wang, S. Design of small-molecule peptidic and nonpeptidic Smac mimetics. *Acc. Chem. Res.* 2008, 41, 1264–1277.

(178) Hong, D. S.; Banerji, U.; George, G. C.; Aaron, J.; Kurzrock, R. Targeting the molecular chaperone heat shock protein 90 (Hsp90): lessons learned and future directions. *Cancer Treat. Rev.* 2013, 39, 375–387.

(179) Hennequin, L. F.; Allen, J.; Breed, J.; Curwen, J.; Fennell, M.; Green, T. P.; Lambert-van der Brempt, C.; Morgentin, R.; Norman, R. A.; Olivier, A.; Otterbein, L.; Plé, P. A.; Warin, N.; Costello, G. N-(5-Chloro-1,3-benzodioxol-4-yl)-7-[2-(4-methylpiperazin-1-yl)ethoxy]-5-(tetrahydro-2H-pyran-4-yloxy)quinazolin-4-amine, a novel, highly selective, orally available, dual-specific c-Src/Abl kinase inhibitor. *J. Med. Chem.* 2006, 49, 6465–6488.
