
frontiers in
CELLULAR AND INFECTION MICROBIOLOGY

Mechanisms of post-transcriptional gene regulation in bacterial biofilms

Luary C. Martínez and Viveka Vadyvaloo*

Paul G. Allen School for Global Animal Health, Washington State University, Pullman, WA, USA

Edited by:
Wyndham W. Lathem,
Northwestern University Feinberg School of Medicine, USA

Reviewed by:
Yi-Cheng Sun, Chinese Academy of Medical Sciences and Peking Union Medical College, China
Brian Hammer, Georgia Institute of Technology, USA

*Correspondence:
Viveka Vadyvaloo, Paul G. Allen School for Global Animal Health, Washington State University, 240 SE Ott Road, Pullman, WA 99163, USA
e-mail: viveka@vetmed.wsu.edu

Biofilms are characterized by a dense multicellular community of microorganisms that can be formed by the attachment of bacteria to an inert surface and to each other. The development of biofilm involves the initial attachment of planktonic bacteria to a surface, followed by replication, cell-to-cell adhesion to form microcolonies, maturation, and detachment. Mature biofilms are embedded in a self-produced extracellular polymeric matrix composed primarily of bacterial-derived exopolysaccharides, specialized proteins, adhesins, and occasionally DNA. Because the synthesis and assembly of biofilm matrix components is an exceptionally complex process, the transition between its different phases requires the coordinate expression and simultaneous regulation of many genes by complex genetic networks involving all levels of gene regulation. The finely controlled intracellular level of the chemical second messenger molecule, cyclic-di-GMP is central to the post-transcriptional mechanisms governing the switch between the motile planktonic lifestyle and the sessile biofilm forming state in many bacteria. Several other post-transcriptional regulatory mechanisms are known to dictate biofilm development and assembly and these include RNA-binding proteins, small non-coding RNAs, toxin-antitoxin systems, riboswitches, and RNases. Post-transcriptional regulation is therefore a powerful molecular mechanism employed by bacteria to rapidly adjust to the changing environment and to fine tune gene expression to the developmental needs of the cell. In this review, we discuss post-transcriptional mechanisms that influence the biofilm developmental cycle in a variety of pathogenic bacteria.

Keywords: biofilm, post-transcriptional regulation, RNA-binding proteins, ncRNAs, riboswitch, toxin-antitoxin systems, RNases, c-di-GMP

INTRODUCTION

During their life cycles bacterial pathogens must often transit between different habitats and have to respond to continually changing environmental conditions. Rapid adaptation to these changing conditions is a key factor for survival and replication. Some bacterial pathogens exhibit multicellular behaviors as a conserved strategy for long-term bacterial survival in nature and during infections. One of these multicellular behaviors is biofilm formation (Mah and O'Toole, 2001; Matz and Kjelleberg, 2005; Anderson and O'Toole, 2008). Biofilm represents a mode of growth that enables bacteria to establish persistent relationships with their surroundings providing protection against environmental stressors, antibiotics, predation, and host immunity (Stoodley et al., 2002). This phenomenon has been observed in diverse Gram-negative and Gram-positive bacterial species. Although mixed-species biofilms predominate in most environments, single-species biofilms exist in a variety of infections. To understand the role of biofilm formation in infections there has been notable research focused on pathogenic biofilm-producer organisms in such diverse genera as *Pseudomonas*, *Vibrio*, *Escherichia*, *Salmonella*, *Listeria*, *Streptococcus*, *Staphylococcus*, *Yersinia*, and *Mycobacteria*.

Biofilm formation has a significant impact in medical and industrial settings. The formation of biofilm on many medical and technological devices may cause severe complicating problems affecting human health and industrial processes. The growth of bacterial biofilm on human tissues results in chronic infections which are challenging for antimicrobial therapies because they are extremely resistant to antibiotic treatment. This is primarily due to the increased prevalence of dormant cells, known as persisters, within the biofilm (Lewis, 2005; Hatt and Rather, 2008; Hall-Stoodley and Stoodley, 2009). This negative impact of biofilm has stimulated research aimed to identify specific components of the physical biofilm structure and regulatory aspects of the process of biofilm development toward creating anti-biofilm strategies (Sommer et al., 2013). On the other hand, despite the detrimental impact of biofilm, they are useful in engineering applications and in many natural settings where they are favored for promoting beneficial microbial associations (Currie, 2001; Singh et al., 2006; Kreth et al., 2008). Understanding the mechanisms of biofilm formation can therefore lead to its manipulation for either its enhancement or eradication. With the recent advances in molecular biology, understanding the underlying molecular basis of biofilm formation has become possible.
and this provides novel opportunities to disrupt/enhance biofilm formation.

**BIOFILM STRUCTURE: WHAT DOES IT TAKE TO FORM A BIOFILM?**

The ability to form biofilm is a universal attribute of several bacteria but the mechanism that different bacterial species employ to produce biofilm may vary according to the specific strain attributes and the diverse environments they occupy. Even though some biofilm structural components can be recognized as common features, their chemical compositions may vary. The process of biofilm formation is dynamic and complex but the stages of development seem to be conserved among a remarkable range of prokaryotes and typically involve the attachment to a surface by planktonic bacteria, replication, cell-cell adhesion to form microcolonies, maturation, and detachment (represented in Figure 1). Because these steps overlap at some point, the growth cycle of a biofilm is described in three general stages here:

**INITIAL ATTACHMENT AND DEVELOPMENT OF BIOFILM STRUCTURES**

This first stage initiates with reversible attachment of bacteria to a favorable surface and is highly dependent on the physicochemical and electrostatic interactions between the bacterial envelope itself and the substrate. Attachment occurs seconds after the bacterial cells detect required environmental signals including changes in nutrients and nutrient concentrations (glucose, indole, polyamines), inorganic molecules (iron, phosphate), pH, antimicrobials, temperature, oxygen concentration, osmolarity, and host derived signals (bile acids, hydrogen peroxide) (O’Toole and Kolter, 1998; Aparna and Yadav, 2008; Karatan and Watnick, 2009). At this point bacterial cells usually exhibit a logarithmic growth rate.

Attachment is facilitated by different adhesive organelles, e.g., flagella and type IV pili play important roles in surface aggregation in *Pseudomonas* spp. and *Vibrio cholerae*, whereas fimbriae like type 1 pili, curli, and conjugative pili are important for biofilm formation in *Escherichia coli* (Thelin and Taylor, 1996; O’Toole and Kolter, 1998; Watnick and Kolter, 1999; Jackson et al., 2002). Curli fimbriae are also produced by other enteric bacteria such as *Shigella, Salmonella, Citrobacter,* and *Enterobacter* (Smyth et al., 1996). Following initial attachment, bacterial cells multiply to form aggregated microcolonies and can inter-communicate by producing quorum sensing molecules, which is one of the key events leading to biofilm development in some bacteria (Camilli and Bassler, 2006).

**MATURATION**

During the maturation stage cell aggregates begin to grow in layers in a three-dimensional matrix (Aparna and Yadav, 2008). The maturation stage still requires adhesive organelles, however, this stage is mostly characterized by cell-to-cell interactions and formation of important surface components that contribute to the structure of the biofilm (McLean et al., 1997; Davies et al., 1998; Holden et al., 1999; Pesci et al., 1999; Whiteley et al., 1999; De Kievit et al., 2001).

A general hallmark feature that determines the mature biofilm architecture is the presence of the extracellular matrix (EM)

surrounding the resident biofilm bacteria. Besides its crucial role in maintaining biofilm structure, it enables bacteria to remain in close proximity to each other, protects embedded bacteria from desiccation, acts as a diffusion barrier, and allows bacteria to evade recognition by the host immune system. The biofilm matrix generally consists of up to 97% water, 2–5% microbial cells, 3–6% extra-polymeric substances (EPS) and ions (Sutherland, 2001). The EPS may account for 50–90% of the total organic carbon of biofilm and this is primarily composed of exopolysaccharides, but it also includes proteins (extracellular proteins and enzymes), DNA and RNA, which constitute less than 2% of the biofilm matrix (Flemming and Wingender, 2001; Sutherland, 2001; Donlan, 2002; Flemming et al., 2007). The polysaccharide composition along with other components such as proteins usually varies among different bacteria and even between strains of a single species, although there are some common polysaccharides produced by multiple species of bacteria. It has been proposed that after contact of bacteria with a surface, altered gene expression induces changes that initiate synthesis of extracellular polysaccharides since alginate, the EPS of *P. aeruginosa* biofilms, is up-regulated in recently attached cells in comparison with planktonic cells (Davies and Geesey, 1995). The systematic three dimensional development of mature *V. cholerae* biofilms following attachment, and specifically as this is related to synthesis of the EPS, *Vibrio* polysaccharide (VPS) and the three major EM proteins, RbmA, RbmC, and Bap1, has been captured in real time in elegant work done using advanced microscopy (Berk et al., 2012).

One of the most common and extensively studied matrix exopolysaccharides is the poly-N-acetylglucosamine (PGA or PNAG) that is utilized to construct the biofilm matrices (Wang et al., 2004; Izano et al., 2007, 2008; Parise et al., 2007) and is produced by diverse bacterial species, including *E. coli, Staphylococcus epidermidis, Staphylococcus aureus, Yersinia pestis, Actinobacillus* spp., *Aggregatibacter actinomycetemcomitans,* and *Bordetella* spp. Some bacteria are capable of producing multiple polysaccharides which usually confer different physiological properties to the biofilm matrix, e.g., *P. aeruginosa* makes biofilms constructed from 3 distinct exopolysaccharides (alginate, Pel, and Psl), *Bacillus subtilis* secretes 2 polymers (EPS and PGA) while *E. coli* principally synthesizes PGA, colanic acid, and cellulose. Cellulose is a commonly produced polymer among the enteric pathogens including *Salmonella, Citrobacter, Enterobacter,* and *Shigella*, where it has been strongly associated with the ability to form a rigid biofilm (Sutherland, 2001; Zogaj et al., 2001, 2003; Solano et al., 2002; Spiers et al., 2003; Da Re and Ghigo, 2006; Ude et al., 2006; Lopez et al., 2010).

The protein fraction of the EPS is generally quite large and an important component for biofilm formation that include structural proteins like adhesins, and other cell surface associated proteins like pili, flagella, curli and amyloid fibers (Prigent-Combaret et al., 2000; Toledo-Arana et al., 2001; Gohl et al., 2006; Flemming et al., 2007; Larsen et al., 2007), as well as a homologous group of large proteins, referred to as biofilm-associated proteins, found in, e.g., *Staphylococcus, Enterococcus, Vibrio,* and *Salmonella* spp. (Cucarella et al., 2001; Kristich et al., 2004; Latasa et al., 2006; Fong and Yildiz, 2007; Berk
Martínez and Vadyvaloo                                                                 Post-transcriptional biofilm regulation

![Diagram](attachment:diagram.png)

Initial attachment and irreversible binding  
**Planktonic-motile**

Development of biofilm structures  
**Sessile-Biofilm**

Maturation (EM formation)  

Detachment and dispersion  
**Planktonic-motile**

FIGURE 1 | Post-transcriptional regulatory networks directing biofilm formation. Numerous post-transcriptional regulatory factors, mainly affecting biofilm maturation at the level of synthesis or exopolysaccharides has been described currently and are represented here. The RNA-binding protein, CsrA, represses biofilm formation by directly and indirectly affecting stability of mRNA transcripts encoding important polysaccharides constituting the extracellular biofilm matrix. CsrA acts alternately by repressing GGDEF/EAL encoding proteins which determine c-di-GMP levels, or by favoring motility, rather than biofilm formation, by stabilizing transcripts encoding the master regulator of flagella, FlhDC. Several ncRNAs (OmrA/B, RprA, GcvB, Mcas) regulate biofilm by repressing csgD, encoding the CsgD master regulator involved in the production of curli fimbriae, cellulose, and c-di-GMP. Other ncRNAs (CsrB/CsrC, McaS) favor biofilm formation by blocking CsrA activity (e.g., in E. coli and Salmonella) or by positively affecting production of c-di-GMP (Qrr1-4) and expression of exopolysaccharides (e.g., V. cholerae). To exert their function, many of these sRNAs need to be bound to the RNA-binding chaperone Hfq, which has been reported to also repress biofilm by affecting the expression of the GGDEF protein, HmsT (Y. pestis), required to synthesize c-di-GMP. Changes in the levels of c-di-GMP are sensed by the GEMM riboswitch which leads to regulation of organelle biosynthesis that promotes the transformation between motile and sessile lifestyles, where increased c-di-GMP leads to increased biofilm formation and vice versa. The RNases E and G mainly cause the decay of ncRNAs that are involved in the biofilm formation. Degradation of the antitoxin mqsA transcript by the MqsR toxin, leads to inhibition of motility, and induction of csgD, favoring both curli and cellulose production. Environmental factors such as nutrient concentrations (glucose, amino acids) and other physiological stresses (osmolarity, pH, oxidative stress, antimicrobials) are important signals mediating the switch from the planktonic motile to sessile biofilm lifestyles. RNA-binding proteins are shown in orange, ncRNAs are shown in green, riboswitches are shown in pink, TA systems are shown in dark blue and RNases are shown in violet. CsgD, FlhDC, NhaR, and RpoS are major transcriptional regulators. Cellulose, PGA and EPS are exopolysaccharides. Hard arrows indicate a direct or indirect positive effect, while truncated arrows indicate a direct or indirect negative effect. Black arrows show those mechanisms that are present in E. coli and are shared in many other pathogens, while red arrows show those mechanism that are present in specific microorganisms (see text for details).

et al., 2012). The production of 3 biofilm-associated matrix proteins (RbmA, RbmC, and Bap1) and the Vibrio polysaccharide (VPS) are involved in V. cholerae biofilm formation, where the RbmA protein is specifically involved in cell-to-cell adhesion and Bapl facilitates adherence of the biofilm to surfaces (Berk et al., 2012).

Extracellular DNA (eDNA) may act as a structural component of the biofilm matrix, where it can be found in varying quantities. eDNA is a major structural component in the biofilm matrix of S. aureus (Izano et al., 2008). Its role in biofilms was confirmed in P. aeruginosa, where DNase was added to the culture medium and this resulted in dissolution of preformed biofilms

Frontiers in Cellular and Infection Microbiology www.frontiersin.org March 2014 | Volume 4 | Article 38 | 3
(Martínez and Vadyvaloo)

(Whitchurch et al., 2002; Nemoto et al., 2003; Bockelmann et al., 2006, 2007). In fact, it has been reported that eDNA facilitates the self-organization of bacterial biofilm, as it coordinates the movement of cells and is required for their assemble into the intricate network of furrows that form the biofilm (Gloag et al., 2013).

### DETACHMENT AND DISPERSION

In this stage these sessile communities (surface-attached) can give rise to planktonic (free-floating) bacteria that can rapidly multiply and disperse to colonize new surfaces. Some studies have shed light on the signals and signaling networks that lead to dispersal of biofilms. Most often the nutritional status of the environment dictates bacterial behavior, including the biofilm dispersal response that can result from both decreases and increases in environmental nutrients (Anderl et al., 2003; Walters et al., 2003; Sauer et al., 2004; Gjermansen et al., 2005; Thormann et al., 2005, 2006). Besides nutrient availability, there are other factors influencing the biofilm dispersion, such as the presence of oxygen, by-products of anaerobic metabolism, quorum sensing signaling, and levels of the chemical second messenger molecule cyclic-di-GMP (c-di-GMP). The exact mechanistic details of dispersal have not been elucidated. However, among the factors that influence biofilm detachment are synthesis of enzymes that degrade extracellular polymeric substances in the biofilm matrix, release of EPS and surface-binding proteins, induction of motility, surface production and cell lysis, hydraulic shear, sloughing, and erosion (Boyd and Chakrabarty, 1994; Lee et al., 1996; Allison et al., 1998; Jackson et al., 2002; Stoodley et al., 2002; Kaplan et al., 2003, 2004; Webb et al., 2003; Boles et al., 2005; Itoh et al., 2005; Purevdorj-Gage et al., 2005; Chambless and Stewart, 2007).

The processes described above for biofilm development are not necessarily synchronized throughout the whole biofilm, but are often localized so that at any time a small area on the surface may contain biofilm at each developmental stage. Even when cells in a biofilm are clonal populations, it has been shown that there are subpopulations of phenotypically dissimilar cell types, and that this cell heterogeneity is a consequence of alterations in gene expression in response to extracellular conditions encountered by the cells within a microenvironment (Lopez et al., 2010). Therefore, since the synthesis and assembly of biofilm matrix components is a costly and an exceptionally complex process, the transition from the planktonic state to the sessile state requires the coordinate expression and simultaneous regulation of many genes by complex genetic networks involving all levels of gene regulation.

### MECHANISMS OF POST-TRANSCRIPTIONAL REGULATION

The regulation of gene expression can potentially occur at several stages during the transfer of information from a gene to its protein product. As for many other bacterial processes, transcriptional regulation is perhaps the most well studied form of controlling biofilm production. However, post-transcriptional regulation provides a powerful way for the bacteria to rapidly adjust to the changing environment and to fine tune gene expression to the needs of the cell. Post-transcriptional regulation can be formally defined as the control of gene expression that occurs after transcription but ahead of translation. In this review we will focus only on those mechanisms that influence biofilm formation by acting at a post-transcriptional level, some of which are summarized in Figure 1.

The post-transcriptional regulatory mechanisms controlling biofilm formation include mainly the activity of RNA-binding proteins, cis- and trans-acting small non-coding RNAs, and RNases. Due to the multifactorial nature of biofilm formation, some of these mechanisms act directly or indirectly to both spatially and temporally coordinate production of the components integral to biofilm development. This is achieved by regulating the intracellular levels of c-di-GMP and by controlling genes affecting the production of major adhesive and aggregative factors, e.g., pili, flagella and fimbriae, cell surface and intercellular matrix components or production of exopolysaccharides that constitute the biofilm EM.

#### C-di-GMP: A CENTRAL MOLECULE TO SWITCH FROM PLANKTONIC TO SESSILE MODE OF LIFE

Central to the mechanism of post-transcriptional regulation of biofilm formation is a tiny cyclic RNA chemical second messenger molecule, c-di-GMP. C-di-GMP is implicated in controlling various cellular functions including virulence, motility, and adhesion, although its principal role is controlling the switch from motile planktonic lifestyle to the sessile biofilm forming state. Elevated intracellular levels of c-di-GMP promote synthesis of exopolysaccharides and enhanced auto-aggregation and surface adhesion leading to biofilm formation (Simm et al., 2004). In contrast, reduced intracellular c-di-GMP concentrations are associated with decreased biofilm formation. The mechanistic details of how post-transcriptional gene regulation is mediated by the c-di-GMP molecule will be discussed below in section Riboswitches.

C-di-GMP is synthesized by GGDEF motif containing proteins that encode diguanylate cyclase (DGC) enzyme activity required to convert two molecules of GTP to c-di-GMP. Degradation of c-di-GMP is carried out by EAL or HD-GYP motif containing proteins that encode phosphodiesterase activity leading to hydrolysis of c-di-GMP to pGpG (Ross et al., 1987; Tal et al., 1998; Romling et al., 2005). The GGDEF or EAL/HD-GYP protein encoding genes controlling c-di-GMP synthesis are present in multiple copies in bacterial genomes especially in those pathogens that infect multiple hosts, e.g., *E. coli* encodes 34, *Salmonella enterica* serovar Typhimurium encodes 27, *V. cholerae* encodes 63 (Galperin et al., 2001; Galperin, 2004) and *P. aeruginosa* encodes 39 such genes with GGDEF, EAL, or HD-GYP domains in their genomes (Kulasakara et al., 2006). Thus, it stands to reason that the c-di-GMP signal transduction is tightly synchronized and regulated to avoid interference between the functionally distinct c-di-GMP responsive systems.

Such distinct environmental niche-dependent regulation by c-di-GMP is exemplified in control of biofilm formation central to transmission of *Y. pestis* biofilm from the foregut of its flea vector. *Y. pestis*, the agent that causes bubonic plague, contains 10 genes encoding DGCs and PDEs in its genome (Sun et al., 2011). However, only two of the DGC enzymes, encoded by *hmsT* and *y3730*, have been shown to be functional in controlling synthesis of c-di-GMP to activate production of biofilm while only

Frontiers in Cellular and Infection Microbiology www.frontiersin.org March 2014 | Volume 4 | Article 38 | 4one phosphodiesterase (PDE), encoded by *hmsP*, is active and degrades c-di-GMP, reducing biofilm synthesis (Kirillina et al., 2004; Bobrov et al., 2005; Sun et al., 2011). Interestingly, despite their similar expression patterns during varied *in vitro* growth conditions and their equally robust expression in the flea gut, the production of biofilm *in vitro* appears to be mainly dependent on expression of *hmsT* while biofilm formation within the flea gut is mainly dependent on *γ3730* expression. The Y3730 DGC alone appears to promote autoaggregation indicating a potential role for this DGC in additional biofilm promoting functions (Sun et al., 2011). Additionally, Y3730 responds to a specific flea environmental cue detected by the gene product of the co-transcribed gene *hmsC* (Ren et al., 2013). The different predominant roles for each of these DGCs are driven by post-transcriptional control mechanisms still yet to be fully defined.

Another well-characterized example of how distinct phenotypes can be regulated by specific DGCs is found in *P. aeruginosa*. Mutational studies of the numerous DGCs encoded in the genomes of *P. aeruginosa* strains, PAO10 and PA14, indicate that while some DGCs impact biofilm formation, others either appear not to affect biofilm formation, or may require specific *in vivo* activating signals to initiate their regulation of biofilm formation; and yet other DGCs are essential for functions such as cytotoxicity and virulence (Kulasakara et al., 2006). Nevertheless, there is a correlation between increased amounts of c-di-GMP and biofilm formation in *P. aeruginosa*. Interestingly, unique localized pools of c-di-GMP may be found in the cell due to differential subcellular localization of DGCs and this may be intimately connected with specific signaling to cognate gene targets to produce biofilm (Merritt et al., 2010; Massie et al., 2012). This observation was made in *P. aeruginosa* when mutations in 2 DGCs, SadC and RoeA, showed insignificant changes in total intracellular c-di-GMP quantities despite their negative effects on biofilm formation. RoeA and SadC appeared to be differentially localized in the cell thus producing unique localized pools of c-di-GMP that likely specifically activate different gene targets for EPS production or increased swarming motility, to negatively impact biofilm formation (Merritt et al., 2010).

### REGULATORY RNA BINDING PROTEINS

This group contains those proteins known to bind to mRNA, and by doing so regulate translation initiation, mRNA stability, and the half-life of the message. Below we will outline the functions of the most thoroughly studied RNA binding proteins CsrA and Hfq which are able to post-transcriptionally regulate biofilm formation in various bacterial pathogens.

#### CsrA

CsrA (RsmA) proteins are a family of RNA binding proteins that are widely distributed central components of the global carbon storage regulatory system (Csr) involved in the control of many cellular functions and virulence traits, like motility, quorum sensing, carbon metabolism, interaction with hosts and biofilm production (Altier et al., 2000; Lenz et al., 2005; Heroven et al., 2008; Brencic and Lory, 2009). Homologs of CsrA proteins have been found throughout the prokaryotic world, in both Gram-negative and Gram-positive bacteria, although the majority of CsrA functional studies have been described in Gram-negative bacteria (White et al., 1996). Several bacterial genomes encode more than one CsrA homolog, such as *Legionella pneumophila*, *Pirellula*, *Pseudomonas fluorescens*, and *Coxiella burnetti* (Mercante et al., 2006). CsrA proteins control the expression of target genes at a post-transcriptional level by various methods: binding sequences overlapping the Shine–Dalgarno (SD) sequence in target mRNAs, occluding ribosome binding and translation, and enhancing mRNA degradation (Liu et al., 1995; Liu and Romeo, 1997; Baker et al., 2002, 2007; Lucchetti-Miganeh et al., 2008; Timmermans and Van Melderen, 2010). The CsrA proteins have been shown to repress biofilm formation post-transcriptionally in several different ways described below (and exemplified in Figure 1).

**Direct effects on extracellular polysaccharide production.** In *E. coli*, the major extracellular polysaccharide that is produced in biofilm formation is poly-β-1,6-N-acetyl-D-glucosamine (PGA) (Wang et al., 2004; Itoh et al., 2005). The *pgaABCD* gene locus is required for the synthesis of this biofilm polysaccharide adhesion PGA that promotes attachment, cell-to-cell adherence and stabilization of biofilm structure (Wang et al., 2004). CsrA represses *pga* gene expression and the production of PGA by binding specifically to the transcript of the *pgaA* gene. This prevents ribosome binding, affecting *pgaABCD* mRNA stability and promoting accelerated degradation of this transcript (Wang et al., 2005). Consequently, in a *csrA* mutant, the biofilm production is increased and in agreement with this, an insertional inactivation of *csrA* has no effect on biofilm in the absence of *pgaC* (Wang et al., 2004, 2005). Interestingly, it has been demonstrated that the effect of a *csrA* mutant on biofilm formation can successfully be restored by CsrA of *Campylobacter jejuni*, even though they exhibit variability in the amino acids important for RNA binding and are therefore substantially divergent (Fields and Thompson, 2012). This demonstrates both similarities and differences in both Csr systems and somewhat different mechanisms of action of CsrA among the ε- and γ-proteobacteria, as the former lacks several genes in the CsrA pathway (Kulkarni et al., 2006; Fields and Thompson, 2012).

In *P. aeruginosa*, Psl is a major extracellular polysaccharide and therefore one of the major structural components of the biofilm extracellular matrix. RsmA, the CsrA homolog, binds to *psl* mRNA and upon this binding initiates *psl* mRNA folding into a secondary stem-loop structure that blocks the SD sequence, preventing subsequent ribosome access and protein translation (Irie et al., 2010) and biofilm formation.

**Indirect effects on extracellular polysaccharide production.** CsrA can repress *E. coli* biofilm synthesis by affecting *pgaABCD* transcription via translation inhibition of the LysR-type transcriptional regulator NhaR. NhaR responds to the Na concentration and pH which are factors that activate *pgaABCD* (Goller et al., 2006). CsrA then binds to the mRNA of *nhaR* and outcompetes ribosomal binding at the translation initiation region, thus blocking *nhaR* mRNA translation and consequently repressing the transcription of *pgaABCD* and in turn biofilm production (Pannuri et al., 2012).Martínez and Vadyvaloo

---

Perhaps the primary and best-known effect of CsrA on biofilm formation is through its regulatory control of glycogen metabolism. CsrA interacts with the leader mRNA of the glgCAP operon, that includes the glycogen biosynthetic genes glgC and glgA and the gene encoding the catabolic enzyme glgP (Yang et al., 1996; Baker et al., 2002). By inhibiting glgCAP expression, CsrA represses glycogen synthesis and turnover, both processes necessary for optimum biofilm formation in the Enterobacteriaceae, since in Salmonella and E. coli, glycogen levels are observed to be positively correlated with biofilm formation (Yang et al., 1996; Bonafonte et al., 2000; Jackson et al., 2002). Glycogen is used for the generation of precursors of PGA (Wang et al., 2004) and the role of CsrA in controlling glycogen metabolism indirectly affects optimal biofilm production.

**Indirect effects by inducing motility.** There is a negative correlation between the expression of the flagella and the induction of biofilm formation. While flagella play a role in initiating initial contact with a favorable surface, they also promote motility. In E. coli, CsrA can bind and stabilize the transcript of flhDC, which encodes the master regulator of flagella synthesis (Wei et al., 2001). Thus, CsrA also indirectly represses biofilm by promoting motility through positive regulation of the flagella master regulator, FlhDC (Wei et al., 2001). In agreement with this, expression of flagella decreases upon attachment, which correlates with the decreased expression of csrA once the bacteria start to grow on surfaces. Flagella expression is later reactivated once the biofilm matures, leading to resumption of motility (Pratt and Kolter, 1998), which correlates with CsrA overexpression and leads to biofilm dispersal (Jackson et al., 2002).

**Effect on GGDEF/EAL proteins.** Until recently, CsrA was thought to affect biofilm formation only through repression of glycogen metabolism and via its positive regulatory effect on the swimming-motility master regulator flhDC (Wei et al., 2001). However, it was recently found that CsrA can also affect biofilm production by controlling the expression of proteins with GGDEF/EAL motifs by binding to them causing their destabilization and degradation. This is the first example of post-transcriptional control of c-di-GMP metabolizing proteins mediated through their mRNA stability (Jonas et al., 2008). Consistent with this, mutants of csrA have increased c-di-GMP levels. CsrA negatively controls biofilm in a c-di-GMP-dependent way, as it down-regulates the expression of several genes encoding GGDEF/EAL proteins. Although some of these proteins do not have orthologs in bacteria that also contain CsrA homologs, the link between CsrA and c-di-GMP levels could be a conserved trait, since it is present in E. coli, Salmonella and Pseudomonas (Jonas et al., 2008, 2010). In E. coli, CsrA represses the expression of at least 29 genes encoding GGDEF/EAL domain proteins, two of them, the GGDEF proteins YcdT and YdeH, encode DGCs that also inhibit motility, hence CsrA enhances motility and inhibits biofilm (Jonas et al., 2008). In the case of Salmonella, CsrA controls the expression of 8 genes encoding GGDEF/EAL motifs by both direct and indirect mechanisms (Jonas et al., 2010).

Xanthomonas campestris is the causative agent of black rot disease of cruciferous plants (Ryan et al., 2011). In these bacteria,

RsmA (CsrA) represses biofilm production by binding to the transcripts of 3 genes encoding GGDEF domain proteins. A mutation in rsmA is associated with elevated intracellular levels of c-di-GMP and increased biofilm. Mutation in rsmA is also associated with decreased expression of manA, which encodes the biofilm dispersing enzyme mannanase, and increased expression of xag, a gene encoding a glycosyl transferase, required for biofilm formation. However, these effects on xag and manA are indirect and work through the c-di-GMP-responsive regulator Clp (Lu et al., 2012).

**Activation of biofilm dispersal.** In preformed mature E. coli biofilms, the induction of CsrA expression causes release of viable planktonic cells from the biofilm, reflecting a role for CsrA in activation of biofilm dispersal (Jackson et al., 2002). Interestingly, this effect can be overridden in the presence of glucose and confirms the importance of nutrients as signals in both biofilm formation and its dispersal (Jackson et al., 2002).

**Hfq**

Hfq is a chaperone RNA-binding protein required for the virulence of many pathogenic bacteria that plays a pivotal role in the post-transcriptional regulation of large numbers of genes (Chao and Vogel, 2010). Once Hfq binds to an mRNA, it can either stabilize or promote its degradation (Vytvytska et al., 1998; Masse et al., 2003; Meibom et al., 2009). Hfq has a key role in the regulatory function of non-coding small RNA (ncRNAs) molecules as it can bind to them thereby stabilizing interactions between the ncRNA and its target mRNA as well as protecting them from RNase degradation (Moller et al., 2002; Vecerek et al., 2003; Zhang et al., 2003; Vogel and Luisi, 2011). Hfq is involved in regulation of biofilm produced by many bacteria, where it mostly positively regulates biofilm production.

In Yersinia pestis, the causative agent of plague, Hfq is required for growth of some Y. pestis strains and efficient biofilm formation when bacteria are grown in TMH, a defined medium that promotes biofilm production (Rempe et al., 2012). In these bacteria biofilm formation is necessary to block the foregut of fleas, the transmission vector for plague (Jarrett et al., 2004; Hinnebusch, 2005). Only those fleas that are blocked due to the presence of a Y. pestis biofilm in the foregut can transmit the disease to other hosts. It was observed that a mutant in hfq is affected in biofilm-mediated blockage formation during flea gut infection (Rempe et al., 2012) indicating that Hfq, is an important factor mediating transmission of the bubonic plague bacteria from fleas.

In contrast, when Y. pestis is grown in rich brain heart infusion medium (BHI), Hfq represses biofilm formation. This was demonstrated by showing that absence of hfq leads to an increase in biofilm formation in these conditions (Bellows et al., 2012). The negative effect of Hfq on biofilm formation is due to its inverse control on the abundance of HmsT and HmsP, the DGC and PDE enzymes responsible for the synthesis and degradation of c-di-GMP, respectively (Bellows et al., 2012). Hfq directly negatively regulates HmsT at a post-transcriptional level, by binding to the hmsT transcript and decreasing its stability and half-life, thus decreasing c-di-GMP levels and biofilm formation (Bellows et al., 2012). It also contributes to the positive regulation of HmsP

Frontiers in Cellular and Infection Microbiology www.frontiersin.org March 2014 | Volume 4 | Article 38 | 6at a transcriptional level perhaps by an indirect mechanism as the absence of Hfq does not affect the stability of *hmsP* mRNA (Bellows et al., 2012).

Uropathogenic *E. coli* (UPEC) are the primary cause of urinary tract infections (UTIs), and it has been proposed that these bacteria form a biofilm to adapt to the poor nutrient environment present in the urinary tract (Stanley and Lazazzera, 2004). Their persistence in the urinary tract therefore depends not only on their ability to invade host epithelial cells and multiply intracellularly, but also on their ability to form a biofilm (Bower et al., 2005; Soto et al., 2006). Hfq influences a number of virulence-related UPEC phenotypes, including biofilm formation and is critical to the ability of UPEC to effectively establish and persist within the urinary tract. Mutants in *hfq* are negatively affected in their ability to produce biofilm and consequently a persistent bacterial UTI (Kulesus et al., 2008). The different regulatory RNAs that could be interacting with Hfq to produce these effects remain to be elucidated.

Similarly to UPEC, Hfq acts to increase biofilm formation in other pathogens such as *Salmonella enterica* Typhimurium, *V. cholerae*, *P. fluorescens*, *Stenotrophomonas maltophilia*, *Erwinia amylovora*, and *Vibrio alginolyticus*. The effects of *hfq* mutation in these pathogens are not only restricted to the biofilm production; they often exhibit pleiotropic phenotypes, including defects in quorum sensing, motility, antibiotic susceptibility, growth rate, stress tolerance, and virulence (Hammer and Bassler, 2007; Kint et al., 2010; Wu et al., 2010; Liu et al., 2011; Monteiro et al., 2012; Roscetto et al., 2012; Zeng et al., 2013).

The effect of Hfq on biofilm formation seems to be dependent on the different growth conditions, which could reflect the distinct arsenal of ncRNAs that may be interacting with Hfq under different environmental conditions. Most of the Hfq-associated ncRNAs have not been identified but recent studies in *E. coli* reported that the ncRNAs McsS, RprA, GcvB, OmrA/B, ArcZ, and SdsR, are implicated in the regulation of CsgD, the major biofilm regulator (Boehm and Vogel, 2012; Jorgensen et al., 2012; Mika et al., 2012; Thomason et al., 2012).

### NON-CODING RNAs

The ncRNAs are a large group of small molecules of RNA (50–500 nts in length) that are not translated into proteins. They act as important trans-acting regulators of gene expression in bacteria, directly influencing protein synthesis at the post-transcriptional level (Gottesman and Storz, 2011) especially under stress conditions. These molecules carry out their regulatory function by base-pairing to a limited complementary sequence on the mRNAs of their cognate target gene, which leads to changes in mRNA translation or stability or both, thereby influencing the target gene expression. The ncRNAs can act by activating or repressing gene expression depending on the portion of the mRNA molecule they base-pair with (Waters and Storz, 2009). Employing ncRNAs to fine-tune the expression of genes, stabilize mRNA, and regulate biofilm formation is an energetically economical strategy (Waters and Storz, 2009). Synthesis of ncRNA is part of the on-going transcription process and likely does not require any additional enzymatic processing as far as is known, therefore defeating the requirement for additional proteins to be synthesized. This

should be especially advantageous under limiting conditions of stress that normally trigger biofilm production.

CsrB and CsrC, are two well-studied examples of ncRNAs in *E. coli* that contain many CsrA-binding sites and function by antagonizing CsrA activity by binding and sequestering this protein, counteracting its translational repression activity (Liu et al., 1997; Romeo, 1998; Weilbacher et al., 2003; Babitzke and Romeo, 2007; Heroven et al., 2008; Lucchetti-Miganeh et al., 2008; Timmermans and Van Melderen, 2010). The study of CsrB and CsrC as regulatory factors for gene expression in bacteria has greatly increased in the recent years. Orthologs of CsrB/CsrC (and CsrA), have been identified in many other gammaproteobacteria (Lapouge et al., 2008), including *Salmonella enterica* (CsrB/CsrC, and CsrA) (Martinez et al., 2011), *Pseudomonas* species (*RsmX/RsmY/RsmZ*, and *RsmA*) (Kay et al., 2005), *Vibrio* species (CsrB/CsrC/CsrD, and CsrA) (Lenz et al., 2005), *Erwinia carotovora* (*RsmB*, and *RsmA*) (Cui et al., 2008), and *Yersinia* species (CsrB/CsrC and CsrA) (Heroven et al., 2012), where they also control the expression of many genes required for a wide variety of cellular functions, including biofilm (Lucchetti-Miganeh et al., 2008; Timmermans and Van Melderen, 2010). The role of these ncRNAs in biofilm production is at a post-transcriptional level by controlling the levels of free CsrA/RsmA protein resulting in activation of biofilm formation (Figure 1). It has been demonstrated that CsrB and CsrC have a redundant function, which means that CsrB levels exhibit a compensatory increase in response to CsrC disruption and *vice versa* (Weilbacher et al., 2003). In agreement with this, it has been shown in *E. coli* and *Salmonella* that only mutants lacking both *csrB* and *csrC*, but not the single mutants in *csrB* or *csrC*, show reduction in biofilm production (Wang et al., 2005; Teplitski et al., 2006). However, other unidentified ncRNAs may act similarly to CsrB/CsrC to counteract the effects of CsrA. This has been demonstrated by the ability of CsrA from *C. jejuni* to restore the affected CsrA phenotypes in *E. coli*, even though homologs to CsrB and CsrC or other proteins involved in the Csr pathway in *E. coli* have not been identified in *C. jejuni* (Parkhill et al., 2000; Hofreuter et al., 2006; Kulkarni et al., 2006; Fields and Thompson, 2012). It has been proposed that these two ncRNAs are in turn post-transcriptionally controlled by CsrD, a GGDEF/EAL protein that binds CsrB and CsrC leading to their decay by converting them to substrates for RNase E degradation (Suzuki et al., 2006).

Although ncRNAs are widespread in nature and an increasing number of ncRNAs have been found to regulate critical pathways, the specific functions of many ncRNAs are still unknown. However, there is growing evidence that could support their role in biofilm production. These molecules often require the chaperone protein Hfq for their expression, stability, and/or function. The effect of Hfq on biofilm production varies among different bacteria and their conditions of growth, suggesting that Hfq-dependent ncRNAs specific to different environments could be the determining factors required to produce biofilm. Accordingly, studies have attempted to identify and characterize the arsenal of ncRNAs in different pathogens and under different conditions by using deep sequencing approaches and have found distinct predominating ncRNAs under different conditions. The ncRNAs that are produced in *Y. pestis*, for example, are distinct depending ongrowth environment alterations, e.g., temperature, medium and in vivo vs. in vitro (Koo et al., 2011; Beauregard et al., 2013; Yan et al., 2013).

It has been reported that iron concentrations can regulate biofilm formation via ncRNAs in the periodontal pathogen A. actinomycetemcomitans. Some ncRNAs whose expression is dependent on the ferric uptake regulator Fur, which functions to repress genes involved in iron uptake in iron-rich environments, were identified in this pathogen. It is proposed that these ncRNAs could be interacting with biofilm-associated genes, including the flp fimbrial operon and genes associated with EPS, according to in silico models (Amarasinghe et al., 2012). So far the ability of these iron-regulated ncRNAs to modulate biofilm formation has been demonstrated, but their functional target genes remain to be elucidated (Amarasinghe et al., 2012).

The ncRNAs are sometimes part of complex regulatory networks that finally induce biofilm formation by post-transcriptional control. In V. cholerae there are 4 redundant ncRNAs, called Qrr1-4 that are under control of the quorum sensing response (Lenz et al., 2004). They repress translation by binding and occluding the RBS of several mRNAs, including hapR, which encodes HapR, a transcription factor that controls the expression of genes involved in biofilm formation (Bardill et al., 2011). Each Qrr is predicted to base pair to the 5′-UTR of hapR aided by Hfq (Lenz et al., 2004). HapR indirectly represses the expression of the exopolysaccharide biosynthesis operon and alters the intracellular levels of c-di-GMP (Hammer and Bassler, 2003, 2009). Furthermore, Qrr ncRNAs may also facilitate degradation of hapR mRNA by cellular RNases as increased levels of the ncRNAs correlated with decreased levels of hapR transcript (Svenningsen et al., 2008). On the other hand, the ncRNAs Qrr1 and Qrr2 have been shown to base-pair to the 5′-UTR of the vca0939 gene to positively control its translation to a GGDEF motif containing protein and synthesis of c-di-GMP (Hammer and Bassler, 2007). It had been assumed that the four Qrr ncRNAs act in a similar way to induce the synthesis of Vca0939 as the Qrr/vca0939 pairing occurs through a region that is 100% conserved among them, which interestingly is the same region required to base-pair to the 5′-UTR of hapR (Zhao et al., 2013). All these studies demonstrate how biofilm formation is linked to the detection of cell density by multiple pathways, where at high density biofilm formation would be repressed because none of the Qrr ncRNAs would be present to repress HapR (Bardill et al., 2011).

We are beginning to understand that post-transcriptional control involving ncRNAs is highly complex and there is growing evidence for a role of these molecules as crucial regulators, alongside well-known global transcriptional regulators. Perhaps the current best characterized example of just how extremely complex and convoluted a post-transcriptional regulatory network can be is that of the regulatory network of CsgD, the global transcription factor that integrates signals to control biofilm formation in E. coli and Salmonella. CsgD induces the expression of the csgBAC operon, required for the production of curli fimbriae, and cellulose, two major adhesive factors required for biofilm, as well as the production of c-di-GMP. Furthermore, CsgD represses flagellar operons favoring the biofilm-formation phenotype (Ogasawara

et al., 2011). CsgD can be post-transcriptionally repressed by 5 discrete Hfq–dependent small RNAs (OmrA, OmrB, GcvB, Mcas, RprA) that respond to various stress conditions (Boehm and Vogel, 2012). These small RNAs directly target the 5′-UTR of the csgD mRNA hence blocking translational initiation. The ncRNAs OmrA and OmrB, are induced by the two component regulator, OmpR-EnvZ during high osmolarity conditions (Holmqvist et al., 2010). The multicellular adhesive ncRNA Mcas, present only in E. coli, Enterobacter and closely related species, is an important small RNA element that disfavors biofilm by binding and exerting inverse control on csgD and flhCD mRNAs (Thomason et al., 2012). In the absence of Mcas, csgD upregulation occurs; alternately, increased Mcas expression activates the synthesis of flhDC, encoding the flagellar master regulator (Thomason et al., 2012; Jorgensen et al., 2013). In contrast to its occlusion of the SD region in the 5′-UTR of the csgD mRNA, Mcas is able to unfold the stem-loop structure of the flhDC mRNA thus exposing the hidden SD sequence for translation initiation (Thomason et al., 2012). However, the Mcas-dependent regulation of biofilm seems to be complex, as it also activates PGA, leading to the formation of biofilm that is independent of the CsgD-pathway. Mcas-mediated activation of pga requires the presence of CsrA as Mcas can be immunoprecipitated with CsrA in an E. coli lysate and the Mcas transcript has several GGA motifs which are recognized targets for binding to CsrA (Holmqvist et al., 2010; Holmqvist and Vogel, 2013; Jorgensen et al., 2013). Mcas is therefore the first example of a ncRNA that regulates gene expression through both CsrA and Hfq RNA binding proteins, acting by two different mechanisms: base-pairing and protein titration (Jorgensen et al., 2013).

RprA is the small RNA that responds to envelope stress mediated through the phosphorelay system Rcs proteins. RprA appears to fine tune expression of csgD by targeting several branches of the CsgD network in response to environmental cues (Mika et al., 2012). RpoS, the general stress response sigma factor is activated by RprA and in turn activates csgD expression and curli biosynthesis once cells transition into stationary phase, while RprA inhibits csgD expression through the DGC YdaM. RprA appears to modulate synthesis of colonic acid in the presence of curli and also cellulose biosynthesis, possibly to balance the expression of these EPS matrix components as necessitated by a specific environmental cue (Mika et al., 2012). RprA is another example of a ncRNA that plays a dual role to repress biofilm formation. The fifth known small RNA, GcvB is expressed in response to amino acid availability to repress csgD under these specific nutritional conditions (Boehm and Vogel, 2012; Jorgensen et al., 2012).

In Salmonella typhimurium, LuxS, the synthase enzyme required in quorum sensing response, is involved in biofilm production, as it has been shown that a luxS deletion mutant is impaired in biofilm production. This defect can be restored upon complementation only with luxS along with its native promoter region. This lead to the finding that adjacent to the luxS coding sequence is a ncRNA called MicA whose balanced concentration is essential for proper biofilm formation (Kint et al., 2010). The mechanism by which MicA exerts its function is still unknown (Kint et al., 2010).
Martínez and Vadyvaloo

RIBOSWITCHES
Riboswitches are structured non-coding RNA domains that form part of the mRNA usually at the 5′-UTR where they can act in cis to control gene expression upon selectively binding ligands (Mandal and Breaker, 2004; Coppins et al., 2007; Roth and Breaker, 2009). Two main domains constitute the riboswitch: an aptamer molecule, which senses and binds a single ligand and an expression platform, usually located downstream of the aptamer that switches its secondary structure according to its ligand binding status to direct gene expression, by either transcriptional or translational mechanisms (Mandal and Breaker, 2004; Weinberg et al., 2007; Roth and Breaker, 2009). The mechanisms of riboswitches sensing and binding a metabolite are categorized into families and classes that are dependent on the required ligand, together with the secondary structure that is formed upon ligand binding.

The mechanism of post-transcriptional gene regulation by the c-di-GMP molecule is facilitated by recognition of a riboswitch. Here c-di-GMP acts as a ligand and mediates its regulation by binding to a riboswitch class in mRNA called GEMM (genes for the environment, for membranes, and for motility). This particular conserved RNA domain GEMM, is located upstream of genes encoding DGC and PDE proteins, as well as in other genes that are controlled by c-di-GMP (Sudarsan et al., 2008; Smith et al., 2010). The GEMM motif was first described in V. cholerae which carries two sequences of GEMM. One of the well-studied c-di-GMP GEMM riboswitches, Vc1, appears to regulate gbpA which encodes a chitin-binding protein required for adherence to chitin and epithelial cells and required for mammalian infection (Kirn et al., 2005; Sudarsan et al., 2008). The second Vc2 is located upstream of the tfoX-like gene vc1722 (Sudarsan et al., 2008), that has been shown to be up-regulated in V. cholerae mutants with the rugose phenotype, characterized in part by increased biofilm production (Lim et al., 2006; Beyhan et al., 2007).

TOXIN-ANTITOXIN SYSTEMS
Bacterial biofilms contain an increased prevalence of dormant cells known as persisters, which are characterized by upregulation of genes known as toxin-antitoxin (TA) modules. These TA systems typically consist of two genes organized in an operon that encodes a stable toxin that disrupts an essential cellular process and a labile antitoxin (either RNA or protein) that prevents toxicity by binding to the toxin and forming a tight complex that neutralizes the toxin (Van Melden and Saavedra De Bast, 2009). Depending on the nature of the interacting molecules, TA systems can be divided in 3 groups. In the Type I group, the antitoxin is a transcript that is antisense to the toxin mRNA and pairing of the two RNAs promotes mutual degradation. In the Type II group, both toxin and antitoxin are proteins and together they form a complex that masks the activity of the toxin. In the Type III group the toxin is a protein which is inhibited by an antitoxin RNA (Blower et al., 2011). Although the mechanism of toxicity at the molecular level is slightly different, the Type II toxins, MqsR, MazF, RelE, ChpB, YoeB, and YafQ prevent translation by cleaving RNAs (Zhang et al., 2003; Christensen et al., 2004; Gerdes et al., 2005; Brown et al., 2009; Prysyak et al., 2009).

The role of TA systems are becoming less enigmatic and they have been implicated in several functions among which include, gene regulation, control of growth, persister cell formation, programmed cell arrest, programmed cell death, and anti-phage measures (Gerdes et al., 2005; Magnuson, 2007; Van Melden and Saavedra De Bast, 2009). The role of TA systems in biofilm formation has been demonstrated as well as it is currently recognized that these systems can direct cells toward the formation of biofilm and persister cells (Kim and Wood, 2010). In mutant strains of E. coli lacking TA pairs MazF/MazE, RelE/RelB, YoeB/YefM, YafQ/DinJ and ChpB, biofilm formation has been demonstrated to be affected (Kim et al., 2009, 2010; Kolodkin-Gal et al., 2009). TA systems in cryptic prophages have been found to influence biofilm formation, e.g., deletion of toxin YpjF from the TA pair YpjF-YfjZ of the cryptic prophage CP4-57 of E. coli K-12 increased biofilm formation (Brown and Shaw, 2003; Wang et al., 2009, 2010).

The MqsR/MqsA pair of E. coli, was the first TA system found to be associated with biofilm formation when mqsR was identified among genes that were differentially regulated in biofilm cells (Ren et al., 2004). This system is conserved in 40 eubacteria and in many genera such as Y. pseudotuberculosis, Y. pestis, Bordetella bronchiseptica, and P. fluorescens (Kim et al., 2010). The toxin MqsR is an RNase belonging to the RelE family that cleaves mRNA at GCU and, to a lesser extent, GCA sequences. The antitoxin MqsA, which is located immediately downstream of mqsR in the same operon, is a DNA/binding protein that functions to neutralize MqsR toxicity (Brown et al., 2009). It has been shown that a deletion of mqsRA reduces biofilm formation in E. coli and this effect is due to MqsR favoring biofilm formation by affecting cellular motility through autoinducer-2 signaling (Gonzalez Barrios et al., 2006; Kasari et al., 2010). Furthermore, it also plays a role in E. coli biofilm and persistence by positively regulating the expression of toxin CspD and QseBC, the two component motility regulatory system (Gonzalez Barrios et al., 2006; Kim et al., 2010). On another level, MqsA is able to bind to its regulatory palindromic sequence found upstream of the CsgD promoter region, preventing CsgD transcription which in turn decreases curli biosynthesis and E. coli biofilm formation (Soo and Wood, 2013). Interestingly, this system is subject to an auto-post-transcriptional control, as free MqsR has the ability to degrade its own mRNA, and that of mqsA, thus alleviating the persister state and biofilm formation once the environmental stress is removed (Brown et al., 2013).

MqsRA can favor biofilm formation by indirectly controlling the level of c-di-GMP. This effect is the result of a mix of transcriptional and post-transcriptional mechanisms which start with the detection of stress conditions. In such conditions, the protease Lon degrades MqsA which acts as a negative regulator of rpoS by binding directly to its promoter (Wang et al., 2011). Once MqsA is degraded, the sigma factor encoded by rpoS, which controls up to 500 genes in E. coli (Hengge, 2008), is induced. This systematically leads to increases in expression of genes involved in c-di-GMP synthesis and repression of flhDC genes, leading to inhibition of motility, and induction of csgD, favoring both curli and cellulose production (Pesavento et al., 2008). Together all of these processes enhance the formation of biofilm (Wang et al., 2011).

Martínez and Vadyvaloo

Post-transcriptional biofilm regulation

Therefore, given that toxins are mRNA interferases that degrade mRNA with substrate specificity, they can be viewed as acting similarly to the global regulators like Hfq and CsrA or RNases (described below) that regulate gene expression at a post-transcriptional level by modulating mRNA decay.

RNASES
RNases are enzymes that cleave RNAs, resulting in remarkably diverse biological consequences. Their elucidated role in biofilm production is mainly that of causing decay of ncRNAs that are involved in regulating this process. The primary endonuclease involved in mRNA decay is RNase E. The RNA-binding protein CsrA activity is indirectly regulated by RNase E. The expression of the two ncRNAs, CsrB and CsrC, that regulate CsrA activity in *E. coli* are regulated by the CsrD protein, which controls their degradation. This CsrD-mediated RNA decay requires RNase E, as demonstrated by the increased half-life of both CsrB and CsrC in *rnaseE* mutants. Despite CsrD containing a GGDEF/EAL motif, its function does not involve c-di-GMP synthesis or turnover (Suzuki et al., 2006). These results indicate that RNase E plays an important role in post-transcriptionally mediating biofilm formation, by degrading ncRNAs CsrB and CsrC (Suzuki et al., 2006).

Another RNase that acts similarly to degrade ncRNAs is RNase G which is implicated in mRNA maturation and turnover processing in *E. coli*. In *P. aeruginosa*, RNase G is involved in biofilm production by controlling the levels of the ncRNA RsmZ. In these bacteria, transition to later stages of biofilm formation is regulated by three two-component regulatory systems, BfiSR, BfmSRR, and MifSR (Petrova and Sauer, 2009). BfiSR is required for transition to the irreversible attachment stage and it has been shown that it regulates biofilm development via the degradation of the ncRNA RsmZ through RNase G (CafA) (Petrova and Sauer, 2009, 2010). BfiR binds directly upstream *cafA*, encoding the RNase G, to activate its expression and once expressed, RNase G targets RsmZ under biofilm growth conditions. In *P. aeruginosa*, reduced RsmZ levels are essential to form biofilm. Thus, when *cafA* is inactivated, an increase in RsmZ levels occurs and a subsequent reduction in biofilm formation (Petrova and Sauer, 2010).

Rnase Y was found to regulate cleavage of the mRNAs of genes involved in biofilm formation in *B. subtilis*. Here, increasing quantities of the mRNA of the biofilm repressor gene *sinR* were observed with depletion of Rnase Y, resulting in less biofilm formation. Reciprocally, there appeared to be a correlation between increasing amounts of mRNAs of genes required for biofilm and concomitant increased biofilm production, with over-expression of the *rny* gene, encoding RnaseY (Lehnik-Habrink et al., 2011).

3′-UNTRANSLATED REGION (3′-UTR)
Although it is well known that 3′-UTRs of mRNA molecules in eukaryotes impact mRNA stability and translational efficiency, such experimental knowledge has been lacking for bacteria. However, Ruiz de los Mozos et al recently found that one third of the mapped mRNAs of *S. aureus* contains long 3′-UTRs (>100 nucleotides) (Ruiz de Los Mozos et al., 2013). Their specific investigation of the regulatory role of the long

3′-UTR of the *icaR* mRNA which codes for the repressor of the main exopolysaccharide compound of the *S. aureus* biofilm matrix, indicated that this 3′-UTR can base-pair with the SD sequence of the *icaR* mRNA and interfere with the translation initiation complex. This results in formation of a double stranded DNA substrate for RNaseIII and subsequent inhibition of biofilm development in *S. aureus*. This is a first report of such a mechanism involved in biofilm formation and further investigations into the presence of long 3′-UTRs in mRNAs of genes required for biofilm in other biofilm producing bacteria could reveal that this is a conserved mechanism of post-transcriptional regulation.

CONCLUSIONS
Biofilm formation and development is a fascinatingly intricate process involving finely altered gene expression, requiring complex and well-coordinated regulation to accomplish the process with high efficiency both spatially and temporally. In this review we have exemplified several of the well characterized powerful contributions that post-transcriptional regulation makes to rapidly adjust and fine tune gene expression to the developmental needs of the cell during biofilm formation. These mechanisms confirm that bacterial signal integration and gene regulation at the mRNA level might be equally sophisticated as its transcription-factor based counterpart acting at the DNA level, with 5′ UTRs of mRNAs playing an analogous role to that of complex promoters. It is however clear that with the growing body of discoveries about the complexities of post-transcriptional regulation we should expect that many new pathways and molecules play critical roles in biofilm formation. This serves as grounds for encouragement for the continued surge into biofilm regulation research which will definitely shed more light on the complex intricacies of this biological process.

AUTHOR CONTRIBUTIONS
Luary C. Martínez reviewed the literature and contributed to writing and revising this manuscript. Viveka Vadyvaloo contributed to writing and revising this manuscript.

ACKNOWLEDGMENTS
This work was supported by an NIH AI097974 and NWRCE Developmental Grant (U54 AI057141) grant to Viveka Vadyvaloo from the National Institutes of Health. Luary C. Martinez is supported by grant number NIH AI097974. We thank Angie Hinz for critical reading of this manuscript.

REFERENCES
Allison, D. G., Ruiz, B., SanJose, C., Jaspe, A., and Gilbert, P. (1998). Extracellular products as mediators of the formation and detachment of *Pseudomonas fluorescens* biofilms. *FEMS Microbiol. Lett.* 167, 179–184. doi: 10.1111/j.1574-6968.1998.tb13225.x

Altier, C., Suyemoto, M., and Lawhon, S. D. (2000). Regulation of *Salmonella* enterica serovar typhimurium invasion genes by *csrA*. *Infect. Immun.* 68, 6790–6797. doi: 10.1128/IAI.68.12.6790-6797.2000

Amarasinghe, J. J., Connell, T. D., Scannapieco, F. A., and Haase, E. M. (2012). Novel iron-regulated and Fur-regulated small regulatory RNAs in *Aggregatibacter actinomycetemcomitans*. *Mol. Oral Microbiol.* 27, 327–349. doi: 10.1111/j.2041-1014.2012.00645.x

Anderl, J. N., Zahller, J., Roe, F., and Stewart, P. S. (2003). Role of nutrient limitation and stationary-phase existence in *Klebsiella pneumoniae* biofilm resistance to

Martínez and Vadyvaloo

Post-transcriptional biofilm regulation

ampicillin and ciprofloxacin. *Antimicrob. Agents Chemother.* 47, 1251–1256. doi: 10.1128/AAC.47.4.1251-1256.2003

Anderson, G. G., and O’Toole, G. A. (2008). Innate and induced resistance mechanisms of bacterial biofilms. *Curr. Top. Microbiol. Immunol.* 322, 85–105. doi: 10.1007/978-3-540-75418-3_5

Aparna, M. S., and Yadav, S. (2008). Biofilms: microbes and disease. *Braz. J. Infect. Dis.* 12, 526–530. doi: 10.1590/S1413-86702008000600016

Babitzke, P., and Romeo, T. (2007). CsrB sRNA family: sequestration of RNA-binding regulatory proteins. *Curr. Opin. Microbiol.* 10, 156–163. doi: 10.1016/j.mib.2007.03.007

Baker, C. S., Eory, L. A., Yakhnin, H., Mercante, J., Romeo, T., and Babitzke, P. (2007). CsrA inhibits translation initiation of *Escherichia coli hfq* by binding to a single site overlapping the Shine-Dalgarno sequence. *J. Bacteriol.* 189, 5472–5481. doi: 10.1128/JB.00529-07

Baker, C. S., Morozov, I., Suzuki, K., Romeo, T., and Babitzke, P. (2002). CsrA regulates glycogen biosynthesis by preventing translation of *glgC* in *Escherichia coli*. *Mol. Microbiol.* 44, 1599–1610. doi: 10.1046/j.1365-2958.2002.02982.x

Bardill, J. P., Zhao, X., and Hammer, B. K. (2011). The *Vibrio cholerae* quorum sensing response is mediated by Hfq-dependent sRNA/mRNA base pairing interactions. *Mol. Microbiol.* 80, 1381–1394. doi: 10.1111/j.1365-2958.2011.07655.x

Beauregard, A., Smith, E. A., Petrone, B. L., Singh, N., Karch, C., McDonough, K. A., et al. (2013). Identification and characterization of small RNAs in *Yersinia pestis*. *RNA Biol.* 10, 397–405. doi: 10.4161/rna.23590

Bellows, L. E., Koestler, B. J., Karaba, S. M., Waters, C. M., and Latham, W. W. (2012). Hfq-dependent, co-ordinate control of cyclic diguanylate synthesis and catabolism in the plague pathogen *Yersinia pestis*. *Mol. Microbiol.* 86, 661–674. doi: 10.1111/mmi.12011

Berk, V., Fong, J. C., Dempsey, G. T., Develioglu, O. N., Zhuang, X., Liphardt, J., et al. (2012). Molecular architecture and assembly principles of *Vibrio cholerae* biofilms. *Science* 337, 236–239. doi: 10.1126/science.1222981

Beyhan, S., Bilecen, K., Salama, S. R., Casper-Lindley, C., and Yildiz, F. H. (2007). Regulation of rugosity and biofilm formation in *Vibrio cholerae*: comparison of *VpsT* and *VpsR* regulons and epistasis analysis of *vpsT*, *vpsR*, and *hapR*. *J. Bacteriol.* 189, 388–402. doi: 10.1128/JB.00981-06

Blower, T. R., Salmond, G. P., and Luisi, B. F. (2011). Balancing at survival’s edge: the structure and adaptive benefits of prokaryotic toxin-antitoxin partners. *Curr. Opin. Struct. Biol.* 21, 109–118. doi: 10.1016/j.sbi.2010.10.009

Bobrov, A. G., Kirillina, O., and Perry, R. D. (2005). The phosphodiesterase activity of the HmsP EAL domain is required for negative regulation of biofilm formation in *Yersinia pestis*. *FEMS Microbiol. Lett.* 247, 123–130. doi: 10.1016/j.femsle.2005.04.036

Bockelmann, U., Janke, A., Kuhn, R., Neu, T. R., Wecke, J., Lawrence, J. R., et al. (2006). Bacterial extracellular DNA forming a defined network-like structure. *FEMS Microbiol. Lett.* 262, 31–38. doi: 10.1111/j.1574-6968.2006.00361.x

Bockelmann, U., Lunsdorf, H., and Szewzyk, U. (2007). Ultrastructural and electron energy-loss spectroscopic analysis of an extracellular filamentous matrix of an environmental bacterial isolate. *Environ. Microbiol.* 9, 2137–2144. doi: 10.1111/j.1462-2920.2007.01325.x

Boehm, A., and Vogel, J. (2012). The *csgD* mRNA as a hub for signal integration via multiple small RNAs. *Mol. Microbiol.* 84, 1–5. doi: 10.1111/j.1365-2958.2012.08033.x

Boles, B. R., Thoendel, M., and Singh, P. K. (2005). Rhamnolipids mediate detachment of *Pseudomonas aeruginosa* from biofilms. *Mol. Microbiol.* 57, 1210–1223. doi: 10.1111/j.1365-2958.2005.04743.x

Bonafonte, M. A., Solano, C., Sesma, B., Alvarez, M., Montuenga, L., Garcia-Ros, D., et al. (2000). The relationship between glycogen synthesis, biofilm formation and virulence in *Salmonella enteritidis*. *FEMS Microbiol. Lett.* 191, 31–36. doi: 10.1111/j.1574-6968.2000.tb09315.x

Bower, J. M., Eto, D. S., and Mulvey, M. A. (2005). Covert operations of uropathogenic *Escherichia coli* within the urinary tract. *Traffic* 6, 18–31. doi: 10.1111/j.1600-0854.2004.00251.x

Boyd, A., and Chakrabarty, A. M. (1994). Role of alginate lyase in cell detachment of *Pseudomonas aeruginosa*. *Appl. Environ. Microbiol.* 60, 2355–2359.

Brencic, A., and Lory, S. (2009). Determination of the regulon and identification of novel mRNA targets of *Pseudomonas aeruginosa* RsmA. *Mol. Microbiol.* 72, 612–632. doi: 10.1111/j.1365-2958.2009.06670.x

Brown, B. L., Grigoriu, S., Kim, Y., Arruda, J. M., Davenport, A., Wood, T. K., et al. (2009). Three dimensional structure of the MqsR:MqsA complex: a novel TA pair comprised of a toxin homologous to RelE and an antitoxin

with unique properties. *PLoS Pathog.* 5:e1000706. doi: 10.1371/journal.ppat.1000706

Brown, B. L., Lord, D. M., Grigoriu, S., Peti, W., and Page, R. (2013). The *Escherichia coli* toxin MqsR destabilizes the transcriptional repression complex formed between the antitoxin MqsA and the mqsRA operon promoter. *J. Biol. Chem.* 288, 1286–1294. doi: 10.1074/jbc.M112.421008

Brown, J. M., and Shaw, K. J. (2003). A novel family of *Escherichia coli* toxin-antitoxin gene pairs. *J. Bacteriol.* 185, 6600–6608. doi: 10.1128/JB.185.22.6600-6608.2003

Camilli, A., and Bassler, B. L. (2006). Bacterial small-molecule signaling pathways. *Science* 311, 1113–1116. doi: 10.1126/science.1121357

Chambless, J. D., and Stewart, P. S. (2007). A three-dimensional computer model analysis of three hypothetical biofilm detachment mechanisms. *Biotechnol. Bioeng.* 97, 1573–1584. doi: 10.1002/bit.21363

Chao, Y., and Vogel, J. (2010). The role of Hfq in bacterial pathogens. *Curr. Opin. Microbiol.* 13, 24–33. doi: 10.1016/j.mib.2010.01.001

Christensen, S. K., Maenhaut-Michel, G., Mine, N., Gottesman, S., Gerdes, K., and Van Melderen, L. (2004). Overproduction of the Lon protease triggers inhibition of translation in *Escherichia coli*: involvement of the *yefM-yoeB* toxin-antitoxin system. *Mol. Microbiol.* 51, 1705–1717. doi: 10.1046/j.1365-2958.2003.03941.x

Coppins, R. L., Hall, K. B., and Groisman, E. A. (2007). The intricate world of riboswitches. *Curr. Opin. Microbiol.* 10, 176–181. doi: 10.1016/j.mib.2007.03.006

Cucarella, C., Solano, C., Valle, J., Amorena, B., Lasa, I., and Penades, J. R. (2001). Bap, a *Staphylococcus aureus* surface protein involved in biofilm formation. *J. Bacteriol.* 183, 2888–2896. doi: 10.1128/JB.183.9.2888-2896.2001

Cui, Y., Chatterjee, A., Yang, H., and Chatterjee, A. K. (2008). Regulatory network controlling extracellular proteins in *Erwinia carotovora* subsp. *carotovora*: FlhDC, the master regulator of flagellar genes, activates rsmB regulatory RNA production by affecting *gacA* and *hexA* (*lrhA*) expression. *J. Bacteriol.* 190, 4610–4623. doi: 10.1128/JB.01828-07

Currie, C. R. (2001). A community of ants, fungi, and bacteria: a multilateral approach to studying symbiosis. *Annu. Rev. Microbiol.* 55, 357–380. doi: 10.1146/annurev.micro.55.1.357

Da Re, S., and Ghigo, J. M. (2006). A CsgD-independent pathway for cellulose production and biofilm formation in *Escherichia coli*. *J. Bacteriol.* 188, 3073–3087. doi: 10.1128/JB.188.8.3073-3087.2006

Davies, D. G., and Geesey, G. G. (1995). Regulation of the alginate biosynthesis gene *algC* in *Pseudomonas aeruginosa* during biofilm development in continuous culture. *Appl. Environ. Microbiol.* 61, 860–867.

Davies, D. G., Parsek, M. R., Pearson, J. P., Iglewski, B. H., Costerton, J. W., and Greenberg, E. P. (1998). The involvement of cell-to-cell signals in the development of a bacterial biofilm. *Science* 280, 295–298. doi: 10.1126/science.280.5361.295

De Kievit, T. R., Gillis, R., Marx, S., Brown, C., and Iglewski, B. H. (2001). Quorum-sensing genes in *Pseudomonas aeruginosa* biofilms: their role and expression patterns. *Appl. Environ. Microbiol.* 67, 1865–1873. doi: 10.1128/AEM.67.4.1865-1873.2001

Donlan, R. M. (2002). Biofilms: microbial life on surfaces. *Emerg. Infect. Dis.* 8, 881–890. doi: 10.3201/eid0809.020063

Fields, J. A., and Thompson, S. A. (2012). *Campylobacter jejuni* CsrA complements an *Escherichia coli csrA* mutation for the regulation of biofilm formation, motility and cellular morphology but not glycogen accumulation. *BMC Microbiol.* 12:233. doi: 10.1186/1471-2180-12-233

Flemming, H. C., Neu, T. R., and Wozniak, D. J. (2007). The EPS matrix: the “house of biofilm cells.” *J. Bacteriol.* 189, 7945–7947. doi: 10.1128/JB.00858-07

Flemming, H. C., and Wingender, J. (2001). Relevance of microbial extracellular polymeric substances (EPSs) – Part I: Structural and ecological aspects. *Water Sci. Technol.* 43, 1–8.

Fong, J. C., and Yildiz, F. H. (2007). The *rbmBCDEF* gene cluster modulates development of rugose colony morphology and biofilm formation in *Vibrio cholerae*. *J. Bacteriol.* 189, 2319–2330. doi: 10.1128/JB.01569-06

Galperin, M. Y. (2004). Bacterial signal transduction network in a genomic perspective. *Environ. Microbiol.* 6, 552–567. doi: 10.1111/j.1462-2920.2004.00633.x

Galperin, M. Y., Nikolskaya, A. N., and Koonin, E. V. (2001). Novel domains of the prokaryotic two-component signal transduction systems. *FEMS Microbiol. Lett.* 203, 11–21. doi: 10.1111/j.1574-6968.2001.tb10814.x

Gerdes, K., Christensen, S. K., and Lobner-Olesen, A. (2005). Prokaryotic toxin-antitoxin stress response loci. *Nat. Rev. Microbiol.* 3, 371–382. doi: 10.1038/nrmicro1147

Gjermansen, M., Ragas, P., Sternberg, C., Molin, S., and Tolker-Nielsen, T. (2005). Characterization of starvation-induced dispersion in *Pseudomonas putida* biofilms. *Environ. Microbiol.* 7, 894–906. doi: 10.1111/j.1462-2920.2005.00775.x

Gloag, E. S., Turnbull, L., Huang, A., Vallotton, P., Wang, H., Nolan, L. M., et al. (2013). Self-organization of bacterial biofilms is facilitated by extracellular DNA. *Proc. Natl. Acad. Sci. U.S.A.* 110, 11541–11546. doi: 10.1073/pnas.1218898110

Gohl, O., Friedrich, A., Hoppert, M., and Averhoff, B. (2006). The thin pili of *Acinetobacter sp*. strain BD413 mediate adhesion to biotic and abiotic surfaces. *Appl. Environ. Microbiol.* 72, 1394–1401. doi: 10.1128/AEM.72.2.1394-1401.2006

Goller, C., Wang, X., Itoh, Y., and Romeo, T. (2006). The cation-responsive protein NhaR of *Escherichia coli* activates *pgaABCD* transcription, required for production of the biofilm adhesin poly-beta-1,6-N-acetyl-D-glucosamine. *J. Bacteriol.* 188, 8022–8032. doi: 10.1128/JB.01106-06

Gonzalez Barrios, A. F., Zuo, R., Hashimoto, Y., Yang, L., Bentley, W. E., and Wood, T. K. (2006). Autoinducer 2 controls biofilm formation in *Escherichia coli* through a novel motility quorum-sensing regulator (MqsR, B3022). *J. Bacteriol.* 188, 305–316. doi: 10.1128/JB.188.1.305-316.2006

Gottesman, S., and Storz, G. (2011). Bacterial small RNA regulators: versatile roles and rapidly evolving variations. *Cold Spring Harb. Perspect. Biol.* 3, 1–16. doi: 10.1101/cshperspect.a003798

Hall-Stoodley, L., and Stoodley, P. (2009). Evolving concepts in biofilm infections. *Cell. Microbiol.* 11, 1034–1043. doi: 10.1111/j.1462-5822.2009.01323.x

Hammer, B. K., and Bassler, B. L. (2003). Quorum sensing controls biofilm formation in *Vibrio cholerae*. *Mol. Microbiol.* 50, 101–104. doi: 10.1046/j.1365-2958.2003.03688.x

Hammer, B. K., and Bassler, B. L. (2007). Regulatory small RNAs circumvent the conventional quorum sensing pathway in pandemic *Vibrio cholerae*. *Proc. Natl. Acad. Sci. U.S.A.* 104, 11145–11149. doi: 10.1073/pnas.0703860104

Hammer, B. K., and Bassler, B. L. (2009). Distinct sensory pathways in *Vibrio cholerae* El Tor and classical biotypes modulate cyclic dimeric GMP levels to control biofilm formation. *J. Bacteriol.* 191, 169–177. doi: 10.1128/JB.01307-08

Hatt, J. K., and Rather, P. N. (2008). Role of bacterial biofilms in urinary tract infections. *Curr. Top. Microbiol. Immunol.* 322, 163–192. doi: 10.1007/978-3-540-75418-3_8

Hengge, R. (2008). “The two component network and the general stress sigma factor RpoS in *E.coli,*” in *Bacterial Signal Transduction Networks and Drug Targets*, ed R. Utsami (New York, NY: Springer), 40–53. doi: 10.1007/978-0-387-78885-2_4

Heroven, A. K., Bohme, K., Rohde, M., and Dersch, P. (2008). A Csr-type regulatory system, including small non-coding RNAs, regulates the global virulence regulator RovA of *Yersinia pseudotuberculosis* through RovM. *Mol. Microbiol.* 68, 1179–1195. doi: 10.1111/j.1365-2958.2008.06218.x

Heroven, A. K., Sest, M., Pisano, F., Scheb-Wetzel, M., Steinmann, R., Bohme, K., et al. (2012). Crp induces switching of the CsrB and CsrC RNAs in *Yersinia pseudotuberculosis* and links nutritional status to virulence. *Front. Cell. Infect. Microbiol.* 2:158. doi: 10.3389/fcimb.2012.00158

Hinnebusch, B. J. (2005). The evolution of flea-borne transmission in *Yersinia pestis*. *Curr. Issues Mol. Biol.* 7, 197–212.

Hofreuter, D., Tsai, J., Watson, R. O., Novik, V., Altman, B., Benitez, M., et al. (2006). Unique features of a highly pathogenic *Campylobacter jejuni* strain. *Infect. Immun.* 74, 4694–4707. doi: 10.1128/IAI.00210-06

Holden, M. T., Ram Chhabra, S., De Nys, R., Stead, P., Bainton, N. J., Hill, P. J., et al. (1999). Quorum-sensing cross talk: isolation and chemical characterization of cyclic dipeptides from *Pseudomonas aeruginosa* and other gram-negative bacteria. *Mol. Microbiol.* 33, 1254–1266. doi: 10.1046/j.1365-2958.1999.01577.x

Holmqvist, E., Reimegard, J., Sterk, M., Grantcharova, N., Romling, U., and Wagner, E. G. (2010). Two antisense RNAs target the transcriptional regulator CsgD to inhibit curli synthesis. *EMBO J.* 29, 1840–1850. doi: 10.1038/emboj.2010.73

Holmqvist, E., and Vogel, J. (2013). A small RNA serving both the Hfq and CsrA regulons. *Genes Dev.* 27, 1073–1078. doi: 10.1101/gad.220178.113

Irie, Y., Starkey, M., Edwards, A. N., Wozniak, D. J., Romeo, T., and Parsek, M. R. (2010). *Pseudomonas aeruginosa* biofilm matrix polysaccharide Psl is regulated transcriptionally by RpoS and post-transcriptionally by RsmA. *Mol. Microbiol.* 78, 158–172. doi: 10.1111/j.1365-2958.2010.07320.x

Itoh, Y., Wang, X., Hinnebusch, B. J., Preston, J. F. 3rd., and Romeo, T. (2005). Depolymerization of beta-1,6-N-acetyl-D-glucosamine disrupts the integrity of diverse bacterial biofilms. *J. Bacteriol.* 187, 382–387. doi: 10.1128/JB.187.1.382-387.2005

Izano, E. A., Amarante, M. A., Kher, W. B., and Kaplan, J. B. (2008). Differential roles of poly-N-acetylg glucosamine surface polysaccharide and extracellular DNA in *Staphylococcus aureus* and *Staphylococcus epidermidis* biofilms. *Appl. Environ. Microbiol.* 74, 470–476. doi: 10.1128/AEM.02073-07

Izano, E. A., Sadovskaya, I., Vinogradov, E., Mulks, M. H., Vellyagounder, K., Ragunath, C., et al. (2007). Poly-N-acetylg lucosamine mediates biofilm formation and antibiotic resistance in *Actinobacillus pleuropneumoniae*. *Microb. Pathog.* 43, 1–9. doi: 10.1016/j.micpath.2007.02.004

Jackson, D. W., Suzuki, K., Oakford, L., Simecka, J. W., Hart, M. E., and Romeo, T. (2002). Biofilm formation and dispersal under the influence of the global regulator CsrA of *Escherichia coli*. *J. Bacteriol.* 184, 290–301. doi: 10.1128/JB.184.1.290-301.2002

Jarrett, C. O., Deak, E., Isherwood, K. E., Oyston, P. C., Fischer, E. R., Whitney, A. R., et al. (2004). Transmission of *Yersinia pestis* from an infectious biofilm in the flea vector. *J. Infect. Dis.* 190, 783–792. doi: 10.1086/422695

Jonas, K., Edwards, A. N., Ahmad, I., Romeo, T., Romling, U., and Melefors, O. (2010). Complex regulatory network encompassing the Csr, c-di-GMP and motility systems of *Salmonella Typhimurium*. *Environ. Microbiol.* 12, 524–540. doi: 10.1111/j.1462-2920.2009.02097.x

Jonas, K., Edwards, A. N., Simm, R., Romeo, T., Romling, U., and Melefors, O. (2008). The RNA binding protein CsrA controls cyclic di-GMP metabolism by directly regulating the expression of GGDEF proteins. *Mol. Microbiol.* 70, 236–257. doi: 10.1111/j.1365-2958.2008.06411.x

Jorgensen, M. G., Nielsen, J. S., Boysen, A., Franch, T., Moller-Jensen, J., and Valentin-Hansen, P. (2012). Small regulatory RNAs control the multicellular adhesive lifestyle of *Escherichia coli*. *Mol. Microbiol.* 84, 36–50. doi: 10.1111/j.1365-2958.2012.07976.x

Jorgensen, M. G., Thomason, M. K., Havelund, J., Valentin-Hansen, P., and Storz, G. (2013). Dual function of the Mcas small RNA in controlling biofilm formation. *Genes Dev.* 27, 1132–1145. doi: 10.1101/gad.214734.113

Kaplan, J. B., Ragunath, C., Ramasubbu, N., and Fine, D. H. (2003). Detachment of *Actinobacillus actinomycetemcomitans* biofilm cells by an endogenous beta-hexosaminidase activity. *J. Bacteriol.* 185, 4693–4698. doi: 10.1128/JB.185.16.4693-4698.2003

Kaplan, J. B., Ragunath, C., Vellyagounder, K., Fine, D. H., and Ramasubbu, N. (2004). Enzymatic detachment of *Staphylococcus epidermidis* biofilms. *Antimicrob. Agents Chemother.* 48, 2633–2636. doi: 10.1128/AAC.48.7.2633-2636.2004

Karatan, E., and Watnick, P. (2009). Signals, regulatory networks, and materials that build and break bacterial biofilms. *Microbiol. Mol. Biol. Rev.* 73, 310–347. doi: 10.1128/MMBR.00041-08

Kasari, V., Kurg, K., Margus, T., Tenson, T., and Kaldalu, N. (2010). The *Escherichia coli mqsR* and *ygiT* genes encode a new toxin-antitoxin pair. *J. Bacteriol.* 192, 2908–2919. doi: 10.1128/JB.01266-09

Kay, E., Dubuis, C., and Haas, D. (2005). Three small RNAs jointly ensure secondary metabolism and biocontrol in *Pseudomonas fluorescens* CHA0. *Proc. Natl. Acad. Sci. U.S.A.* 102, 17136–17141. doi: 10.1073/pnas.0505673102

Kim, Y., Wang, X., Ma, Q., Zhang, X. S., and Wood, T. K. (2009). Toxin-antitoxin systems in *Escherichia coli* influence biofilm formation through YjgK (TabA) and fimbriae. *J. Bacteriol.* 191, 1258–1267. doi: 10.1128/JB.01465-08

Kim, Y., Wang, X., Zhang, X. S., Grigoriu, S., Page, R., Peti, W., et al. (2010). *Escherichia coli* toxin/antitoxin pair MqsR/MqsA regulate toxin CspD. *Environ. Microbiol.* 12, 1105–1121. doi: 10.1111/j.1462-2920.2009.02147.x

Kim, Y., and Wood, T. K. (2010). Toxins Hha and CspD and small RNA regulator Hfq are involved in persister cell formation through MqsR in *Escherichia coli*. *Biochem. Biophys. Res. Commun.* 391, 209–213. doi: 10.1016/j.bbrc.2009.11.033

Kint, G., De Coster, D., Marchal, K., Vanderleyden, J., and De Keersmaecker, S. C. (2010). The small regulatory RNA molecule MicA is involved in *Salmonella enterica* serovar *Typhimurium* biofilm formation. *BMC Microbiol.* 10:276. doi: 10.1186/1471-2180-10-276

Kirillina, O., Fetherston, J. D., Bobrov, A. G., Abney, J., and Perry, R. D. (2004). HmsP, a putative phosphodiesterase, and HmsT, a putative diguanylate cyclase, control Hms-dependent biofilm formation in *Yersinia pestis*. *Mol. Microbiol.* 54, 75–88. doi: 10.1111/j.1365-2958.2004.04253.x

Martínez and Vadyvaloo

Kirn, T. J., Jude, B. A., and Taylor, R. K. (2005). A colonization factor links Vibrio cholerae environmental survival and human infection. Nature 438, 863–866. doi: 10.1038/nature04249

Kolodkin-Gal, I., Verdiger, R., Shlosberg-Fedida, A., and Engelberg-Kulka, H. (2009). A differential effect of E. coli toxin-antitoxin systems on cell death in liquid media and biofilm formation. PLoS ONE 4:e6785. doi: 10.1371/journal.pone.0006785

Koo, J. T., Alleyne, T. M., Schiano, C. A., Jafari, N., and Lathem, W. W. (2011). Global discovery of small RNAs in Yersinia pseudotuberculosis identifies Yersinia-specific small, noncoding RNAs required for virulence. Proc. Natl. Acad. Sci. U.S.A. 108, E709–E717. doi: 10.1073/pnas.1101655108

Kreth, J., Zhang, Y., and Herzberg, M. C. (2008). Streptococcal antagonism in oral biofilms: Streptococcus sanguinis and Streptococcus gordonii interference with Streptococcus mutans. J. Bacteriol. 190, 4632–4640. doi: 10.1128/JB.00276-08

Kristich, C. J., Li, Y. H., Cvitkovich, D. G., and Dunny, G. M. (2004). Esp-independent biofilm formation by Enterococcus faecalis. J. Bacteriol. 186, 154–163. doi: 10.1128/JB.186.1.154-163.2004

Kulasakara, H., Lee, V., Brencic, A., Liberati, N., Urbach, J., Miyata, S., et al. (2006). Analysis of Pseudomonas aeruginosa diguanylate cyclases and phosphodiesterases reveals a role for bis-(3′-5′)-cyclic-GMP in virulence. Proc. Natl. Acad. Sci. U.S.A. 103, 2839–2844. doi: 10.1073/pnas.0511090103

Kulesus, R. R., Diaz-Perez, K., Slechta, E. S., Eto, D. S., and Mulvey, M. A. (2008). Impact of the RNA chaperone Hfq on the fitness and virulence potential of uropathogenic Escherichia coli. Infect. Immun. 76, 3019–3026. doi: 10.1128/IAI.00022-08

Kulkarni, P. R., Cui, X., Williams, J. W., Stevens, A. M., and Kulkarni, R. V. (2006). Prediction of CsrA-regulating small RNAs in bacteria and their experimental verification in Vibrio fischeri. Nucleic Acids Res. 34, 3361–3369. doi: 10.1093/nar/gkl439

Lapouge, K., Schubert, M., Allain, F. H., and Haas, D. (2008). Gac/Rsm signal transduction pathway of gamma-proteobacteria: from RNA recognition to regulation of social behaviour. Mol. Microbiol. 67, 241–253. doi: 10.1111/j.1365-2958.2007.06042.x

Larsen, P., Nielsen, J. L., Dueholm, M. S., Wetzel, R., Otzen, D., and Nielsen, P. H. (2007). Amyloid adhesins are abundant in natural biofilms. Environ. Microbiol. 9, 3077–3090. doi: 10.1111/j.1462-2920.2007.01418.x

Latasa, C., Solano, C., Penades, J. R., and Lasa, I. (2006). Biofilm-associated proteins. C. R. Biol. 329, 849–857. doi: 10.1016/j.crvi.2006.07.008

Lee, S. F., Li, Y. H., and Bowden, G. H. (1996). Detachment of Streptococcus mutans biofilm cells by an endogenous enzymatic activity. Infect. Immun. 64, 1035–1038.

Lehnik-Habrink, M., Schaffer, M., Mader, U., Diethmaier, C., Herzberg, C., and Stulke, J. (2011). RNA processing in Bacillus subtilis: identification of targets of the essential RNase Y. Mol. Microbiol. 81, 1459–1473. doi: 10.1111/j.1365-2958.2011.07777.x

Lenz, D. H., Miller, M. B., Zhu, J., Kulkarni, R. V., and Bassler, B. L. (2005). CsrA and three redundant small RNAs regulate quorum sensing in Vibrio cholerae. Mol. Microbiol. 58, 1186–1202. doi: 10.1111/j.1365-2958.2005.04902.x

Lenz, D. H., Mok, K. C., Lilley, B. N., Kulkarni, R. V., Wingreen, N. S., and Bassler, B. L. (2004). The small RNA chaperone Hfq and multiple small RNAs control quorum sensing in Vibrio harveyi and Vibrio cholerae. Cell 118, 69–82. doi: 10.1016/j.cell.2004.06.009

Lewis, K. (2005). Persister cells and the riddle of biofilm survival. Biochemistry (Mosc) 70, 267–274. doi: 10.1007/s10541-005-0111-6

Lim, B., Beyhan, S., Meir, J., and Yildiz, F. H. (2006). Cyclic-diGMP signal transduction systems in Vibrio cholerae: modulation of rugosity and biofilm formation. Mol. Microbiol. 60, 331–348. doi: 10.1111/j.1365-2958.2006.05106.x

Liu, H., Wang, Q., Liu, Q., Cao, X., Shi, C., and Zhang, Y. (2011). Roles of Hfq in the stress adaptation and virulence in fish pathogen Vibrio alginolyticus and its potential application as a target for live attenuated vaccine. Appl. Microbiol. Biotechnol. 91, 353–364. doi: 10.1007/s00253-011-3286-3

Liu, M. Y., Gui, G., Wei, B., Preston, J. F. 3rd., Oakford, L., Yuksel, U., et al. (1997). The RNA molecule CsrB binds to the global regulatory protein CsrA and antagonizes its activity in Escherichia coli. J. Biol. Chem. 272, 17502–17510. doi: 10.1074/jbc.272.28.17502

Liu, M. Y., and Romeo, T. (1997). The global regulator CsrA of Escherichia coli is a specific mRNA-binding protein. J. Bacteriol. 179, 4639–4642.

Liu, M. Y., Yang, H., and Romeo, T. (1995). The product of the pleiotropic Escherichia coli gene csrA modulates glycogen biosynthesis via effects on mRNA stability. J. Bacteriol. 177, 2663–2672.

Lopez, D., Vlamakis, H., and Kolter, R. (2010). Biofilms. Cold Spring Harb. Perspect. Biol. 2:a000398. doi: 10.1101/cshperspect.a000398

Lu, X. H., An, S. Q., Tang, D. J., McCarthy, Y., Tang, J. L., Dow, J. M., et al. (2012). RsmA regulates biofilm formation in Xanthomonas campestris through a regulatory network involving cyclic di-GMP and the Clp transcription factor. PLoS ONE 7:e52646. doi: 10.1371/journal.pone.0052646

Lucchetti-Miganeh, C., Burrowes, E., Baysse, C., and Ermel, G. (2008). The post-transcriptional regulator CsrA plays a central role in the adaptation of bacterial pathogens to different stages of infection in animal hosts. Microbiology 154, 16–29. doi: 10.1099/mic.0.2007/012286-0

Magnuson, R. D. (2007). Hypothetical functions of toxin-antitoxin systems. J. Bacteriol. 189, 6089–6092. doi: 10.1128/JB.00958-07

Mah, T. F., and O’Toole, G. A. (2001). Mechanisms of biofilm resistance to antimicrobial agents. Trends Microbiol. 9, 34–39. doi: 10.1016/S0966-842X(00)01913-2

Mandal, M., and Breaker, R. R. (2004). Gene regulation by riboswitches. Nat. Rev. Mol. Cell Biol. 5, 451–463. doi: 10.1038/nrm1403

Martinez, L. C., Yakhnin, H., Camacho, M. I., Georgellis, D., Babitzke, P., Puente, J. L., et al. (2011). Integration of a complex regulatory cascade involving the SirA/BarA and Csr global regulatory systems that controls expression of the Salmonella SPI-1 and SPI-2 virulence regulons through HilD. Mol. Microbiol. 80, 1637–1656. doi: 10.1111/j.1365-2958.2011.07674.x

Masse, E., Majdalani, N., and Gottesman, S. (2003). Regulatory roles for small RNAs in bacteria. Curr. Opin. Microbiol. 6, 120–124. doi: 10.1016/S1369-5274(03)00027-4

Massie, J. P., Reynolds, E. L., Koestler, B. J., Cong, J. P., Agostoni, M., and Waters, C. M. (2012). Quantification of high-specificity cyclic diguanylate signaling. Proc. Natl. Acad. Sci. U.S.A. 109, 12746–12751. doi: 10.1073/pnas.1115663109

Matz, C., and Kjelleberg, S. (2005). Off the hook—how bacteria survive protozoan grazing. Trends Microbiol. 13, 302–307. doi: 10.1016/j.tim.2005.05.009

McLean, R. J., Whiteley, M., Stickler, D. J., and Fuqua, W. C. (1997). Evidence of autoinducer activity in naturally occurring biofilms. FEMS Microbiol. Lett. 154, 259–263. doi: 10.1111/j.1574-6968.1997.tb12653.x

Meibom, K. L., Forslund, A. L., Kuoppa, K., Alkhuder, K., Dubail, I., Dupuis, M., et al. (2009). Hfq, a novel pleiotropic regulator of virulence-associated genes in Francisella tularensis. Infect. Immun. 77, 1866–1880. doi: 10.1128/IAI.01496-08

Mercante, J., Suzuki, K., Cheng, X., Babitzke, P., and Romeo, T. (2006). Comprehensive alanine-scanning mutagenesis of Escherichia coli CsrA defines two subdomains of critical functional importance. J. Biol. Chem. 281, 31832–31842. doi: 10.1074/jbc.M606057200

Merritt, J. H., Ha, D. G., Cowles, K. N., Lu, W., Morales, D. K., Rabinowitz, J., et al. (2010). Specific control of Pseudomonas aeruginosa surface-associated behaviors by two c-di-GMP diguanylate cyclases. MBio 1, 1–9. doi: 10.1128/mBio.00183-10

Mika, F., Busse, S., Possling, A., Berkholz, J., Tschowri, N., Sommerfeldt, N., et al. (2012). Targeting of csgD by the small regulatory RNA RprA links stationary phase, biofilm formation and cell envelope stress in Escherichia coli. Mol. Microbiol. 84, 51–65. doi: 10.1111/j.1365-2958.2012.08002.x

Moller, T., Franch, T., Hojrup, P., Keene, D. R., Bachinger, H. P., Brennan, R. G., et al. (2002). Hfq: a bacterial Sm-like protein that mediates RNA-RNA interaction. Mol. Cell 9, 23–30. doi: 10.1016/S1097-2765(01)00436-1

Monteiro, C., Papenfort, K., Hentrich, K., Ahmad, I., Le Guyon, S., Reimann, R., et al. (2012). Hfq and Hfq-dependent small RNAs are major contributors to multicellular development in Salmonella enterica serovar Typhimurium. RNA Biol. 9, 489–502. doi: 10.4161/rna.19682

Nemoto, K., Hirota, K., Murakami, K., Taniguti, K., Murata, H., Viducic, D., et al. (2003). Effect of Varidase (streptodornase) on biofilm formed by Pseudomonas aeruginosa. Chemotherapy 49, 121–125. doi: 10.1159/000070617

Ogasawara, H., Yamamoto, K., and Ishihama, A. (2011). Role of the biofilm master regulator CsgD in cross-regulation between biofilm formation and flagellar synthesis. J. Bacteriol. 193, 2587–2597. doi: 10.1128/JB.01468-10

O’Toole, G. A., and Kolter, R. (1998). Initiation of biofilm formation in Pseudomonas fluorescens WCS365 proceeds via multiple, convergent signalling pathways: a genetic analysis. Mol. Microbiol. 28, 449–461.

Pannuri, A., Yakhnin, H., Vakulskas, C. A., Edwards, A. N., Babitzke, P., and Romeo, T. (2012). Translational repression of NhaR, a novel pathway for multi-tier regulation of biofilm circuitry by CsrA. J. Bacteriol. 194, 79–89. doi: 10.1128/JB.06209-11

Frontiers in Cellular and Infection Microbiology www.frontiersin.org March 2014 | Volume 4 | Article 38 | 13

Martínez and Vadyvaloo

Parise, G., Mishra, M., Itoh, Y., Romeo, T., and Deora, R. (2007). Role of a putative polysaccharide locus in *Bordetella* biofilm development. *J. Bacteriol.* 189, 750–760. doi: 10.1128/JB.00953-06

Parkhill, J., Wren, B. W., Mungall, K., Ketley, J. M., Churcher, C., Basham, D., et al. (2000). The genome sequence of the food-borne pathogen *Campylobacter jejuni* reveals hypervariable sequences. *Nature* 403, 665–668. doi: 10.1038/35001088

Pesavento, C., Becker, G., Sommerfeldt, N., Possling, A., Tschowri, N., Mehlis, A., et al. (2008). Inverse regulatory coordination of motility and curli-mediated adhesion in *Escherichia coli*. *Genes Dev.* 22, 2434–2446. doi: 10.1101/gad.475808

Pesci, E. C., Milbank, J. B., Pearson, J. P., McKnight, S., Kende, A. S., Greenberg, E. P., et al. (1999). Quinolone signaling in the cell-to-cell communication system of *Pseudomonas aeruginosa*. *Proc. Natl. Acad. Sci. U.S.A.* 96, 11229–11234. doi: 10.1073/pnas.96.20.11229

Petrova, O. E., and Sauer, K. (2009). A novel signaling network essential for regulating *Pseudomonas aeruginosa* biofilm development. *PLoS Pathog.* 5:e1000668. doi: 10.1371/journal.ppat.1000668

Petrova, O. E., and Sauer, K. (2010). The novel two-component regulatory system BfiSR regulates biofilm development by controlling the small RNA *rsmZ* through CafA. *J. Bacteriol.* 192, 5275–5288. doi: 10.1128/JB.00387-10

Pratt, L. A., and Kolter, R. (1998). Genetic analysis of *Escherichia coli* biofilm formation: roles of flagella, motility, chemotaxis and type I pili. *Mol. Microbiol.* 30, 285–293. doi: 10.1046/j.1365-2958.1998.01061.x

Prigent-Combaret, C., Prensier, G., Le Thi, T. T., Vidal, O., Lejeune, P., and Dorel, C. (2000). Developmental pathway for biofilm formation in curli-producing *Escherichia coli* strains: role of flagella, curli and colanic acid. *Environ. Microbiol.* 2, 450–464. doi: 10.1046/j.1462-2920.2000.00128.x

Prysak, M. H., Mozdzierz, C. J., Cook, A. M., Zhu, L., Zhang, Y., Inouye, M., et al. (2009). Bacterial toxin YafQ is an endoribonuclease that associates with the ribosome and blocks translation elongation through sequence-specific and frame-dependent mRNA cleavage. *Mol. Microbiol.* 71, 1071–1087. doi: 10.1111/j.1365-2958.2008.06572.x

Purevdorj-Gage, B., Costerton, W. J., and Stoodley, P. (2005). Phenotypic differentiation and seeding dispersal in non-mucoid and mucoid *Pseudomonas aeruginosa* biofilms. *Microbiology* 151, 1569–1576. doi: 10.1099/mic.0.27536-0

Rempe, K. A., Hinz, A. K., and Vadyvaloo, V. (2012). Hfq regulates biofilm gut blockage that facilitates flea-borne transmission of *Yersinia pestis*. *J. Bacteriol.* 194, 2036–2040. doi: 10.1128/JB.06568-11

Ren, D., Bedzyk, L. A., Thomas, S. M., Ye, R. W., and Wood, T. K. (2004). Gene expression in *Escherichia coli* biofilms. *Appl. Microbiol. Biotechnol.* 64, 515–524. doi: 10.1007/s00253-003-1517-y

Ren, G. X., Yan, H. Q., Zhu, H., Guo, X. P., and Sun, Y. C. (2013). HmsC, a periplasmic protein, controls biofilm formation via repression of HmsD, a diguanylate cyclase in *Yersinia pestis*. *Environ. Microbiol.* doi: 10.1111/1462-2920.12323. [Epub ahead of print].

Romeo, T. (1998). Global regulation by the small RNA-binding protein CsrA and the non-coding RNA molecule CsrB. *Mol. Microbiol.* 29, 1321–1330. doi: 10.1046/j.1365-2958.1998.01021.x

Romling, U., Gomelsky, M., and Galperin, M. Y. (2005). C-di-GMP: the dawn-ing of a novel bacterial signalling system. *Mol. Microbiol.* 57, 629–639. doi: 10.1111/j.1365-2958.2005.04697.x

Roscetto, E., Angrisano, T., Costa, V., Casalino, M., Forstner, K. U., Sharma, C. M., et al. (2012). Functional characterization of the RNA chaperone Hfq in the opportunistic human pathogen *Stenotrophomonas maltophilia*. *J. Bacteriol.* 194, 5864–5874. doi: 10.1128/JB.00746-12

Ross, P., Weinhouse, H., Aloni, Y., Michaeli, D., Weinberger-Ohana, P., Mayer, R., et al. (1987). Regulation of cellulose synthesis in *Acetobacter xylinum* by cyclic diguanylic acid. *Nature* 325, 279–281. doi: 10.1038/325279a0

Roth, A., and Breaker, R. R. (2009). The structural and functional diversity of metabolite-binding riboswitches. *Annu. Rev. Biochem.* 78, 305–334. doi: 10.1146/annurev.biochem.78.070507.135656

Ruiz de Los Mozos, I., Vergara-Irigaray, M., Segura, V., Villanueva, M., Bitarte, N., Saramago, M., et al. (2013). Base pairing interaction between 5′- and 3′-UTRs controls *icaR* mRNA translation in *Staphylococcus aureus*. *PLoS Genet* 9:e1004001. doi: 10.1371/journal.pgen.1004001

Ryan, R. P., Vorholter, F. J., Potnis, N., Jones, J. B., Van Sluys, M. A., Bogdanove, A. J., et al. (2011). Pathogenomics of *Xanthomonas*: understanding bacterium-plant interactions. *Nat. Rev. Microbiol.* 9, 344–355. doi: 10.1038/nrmicro2558

Sauer, K., Cullen, M. C., Rickard, A. H., Zeef, L. A., Davies, D. G., and Gilbert, P. (2004). Characterization of nutrient-induced dispersion in *Pseudomonas aeruginosa* PAO1 biofilm. *J. Bacteriol.* 186, 7312–7326. doi: 10.1128/JB.186.21.7312-7326.2004

Simm, R., Morr, M., Kader, A., Nimtz, M., and Romling, U. (2004). GGDEF and EAL domains inversely regulate cyclic di-GMP levels and transition from sessility to motility. *Mol. Microbiol.* 53, 1123–1134. doi: 10.1111/j.1365-2958.2004.04206.x

Singh, R., Paul, D., and Jain, R. K. (2006). Biofilms: implications in bioremediation. *Trends Microbiol.* 14, 389–397. doi: 10.1016/j.tim.2006.07.001

Smith, K. D., Lipchock, S. V., Livingston, A. L., Shanahan, C. A., and Strobel, S. A. (2010). Structural and biochemical determinants of ligand binding by the c-di-GMP riboswitch. *Biochemistry* 49, 7351–7359. doi: 10.1021/bi100671e

Smyth, C. J., Marron, M. B., Twohig, J. M., and Smith, S. G. (1996). Fimbrial adhesins: similarities and variations in structure and biogenesis. *FEMS Immunol. Med. Microbiol.* 16, 127–139. doi: 10.1111/j.1574-695X.1996.tb00129.x

Solano, C., Garcia, B., Valle, J., Berasain, C., Ghigo, J. M., Gamazo, C., et al. (2002). Genetic analysis of *Salmonella enteritidis* biofilm formation: critical role of cellulose. *Mol. Microbiol.* 43, 793–808. doi: 10.1046/j.1365-2958.2002.02802.x

Sommer, R., Joachim, I., Wagner, S., and Titz, A. (2013). New approaches to control infections: anti-biofilm strategies against gram-negative bacteria. *Chimia (Aarau)* 67, 286–290. doi: 10.2533/chimia.2013.286

Soo, V. W., and Wood, T. K. (2013). Antitoxin MqsA represses curli formation through the master biofilm regulator CsgD. *Sci. Rep.* 3:3186. doi: 10.1038/srep03186

Soto, S. M., Smithson, A., Horcajada, J. P., Martinez, J. A., Mensa, J. P., and Vila, J. (2006). Implication of biofilm formation in the persistence of urinary tract infection caused by uropathogenic *Escherichia coli*. *Clin. Microbiol. Infect.* 12, 1034–1036. doi: 10.1111/j.1469-0691.2006.01543.x

Spiers, A. J., Bohannon, J., Gehrig, S. M., and Rainey, P. B. (2003). Biofilm formation at the air-liquid interface by the *Pseudomonas fluorescens* SBW25 wrinkly spreader requires an acetylated form of cellulose. *Mol. Microbiol.* 50, 15–27. doi: 10.1046/j.1365-2958.2003.03670.x

Stanley, N. R., and Lazazzera, B. A. (2004). Environmental signals and regulatory pathways that influence biofilm formation. *Mol. Microbiol.* 52, 917–924. doi: 10.1111/j.1365-2958.2004.04036.x

Stoodley, P., Sauer, K., Davies, D. G., and Costerton, J. W. (2002). Biofilms as complex differentiated communities. *Annu. Rev. Microbiol.* 56, 187–209. doi: 10.1146/annurev.micro.56.012302.160705

Sudarsan, N., Lee, E. R., Weinberg, Z., Moy, R. H., Kim, J. N., Link, K. H., et al. (2008). Riboswitches in eubacteria sense the second messenger cyclic di-GMP. *Science* 321, 411–413. doi: 10.1126/science.1159519

Sun, Y. C., Koumoutsi, A., Jarrett, C., Lawrence, K., Gherardini, F. C., Darby, C., et al. (2011). Differential control of *Yersinia pestis* biofilm formation in vitro and in the flea vector by two c-di-GMP diguanylate cyclases. *PLoS ONE* 6:e19267. doi: 10.1371/journal.pone.0019267

Sutherland, I. W. (2001). The biofilm matrix—an immobilized but dynamic microbial environment. *Trends Microbiol.* 9, 222–227. doi: 10.1016/S0966-842X(01)02012-1

Suzuki, K., Babitzke, P., Kushner, S. R., and Romeo, T. (2006). Identification of a novel regulatory protein (CsrD) that targets the global regulatory RNAs CsrB and CsrC for degradation by RNase E. *Genes Dev.* 20, 2605–2617. doi: 10.1101/gad.1461606

Svenningsen, S. L., Waters, C. M., and Bassler, B. L. (2008). A negative feedback loop involving small RNAs accelerates *Vibrio cholerae’s* transition out of quorum-sensing mode. *Genes Dev.* 22, 226–238. doi: 10.1101/gad.1629908

Tal, R., Wong, H. C., Calhoon, R., Gelfand, D., Fear, A. L., Volman, G., et al. (1998). Three *cdg* operons control cellular turnover of cyclic di-GMP in *Acetobacter xylinum*: genetic organization and occurrence of conserved domains in isoenzymes. *J. Bacteriol.* 180, 4416–4425.

Teplitski, M., Al-Agely, A., and Ahmer, B. M. (2006). Contribution of the SirA regulon to biofilm formation in *Salmonella enterica* serovar Typhimurium. *Microbiology* 152, 3411–3424. doi: 10.1099/mic.0.29118-0

Thelin, K. H., and Taylor, R. K. (1996). Toxin-coregulated pilus, but not mannose-sensitive hemagglutinin, is required for colonization by *Vibrio cholerae* O1 El Tor biotype and O139 strains. *Infect. Immun.* 64, 2853–2856.

Thomason, M. K., Fontaine, F., De Lay, N., and Storz, G. (2012). A small RNA that regulates motility and biofilm formation in response to changes in nutrient

Frontiers in Cellular and Infection Microbiology www.frontiersin.org March 2014 | Volume 4 | Article 38 | 14

availability in *Escherichia coli*. Mol. Microbiol. 84, 17–35. doi: 10.1111/j.1365-2958.2012.07965.x

Thormann, K. M., Duttler, S., Saville, R. M., Hyodo, M., Shukla, S., Hayakawa, Y., et al. (2006). Control of formation and cellular detachment from *Shewanella oneidensis* MR-1 biofilms by cyclic di-GMP. J. Bacteriol. 188, 2681–2691. doi: 10.1128/JB.188.7.2681-2691.2006

Thormann, K. M., Saville, R. M., Shukla, S., and Spormann, A. M. (2005). Induction of rapid detachment in *Shewanella oneidensis* MR-1 biofilms. J. Bacteriol. 187, 1014–1021. doi: 10.1128/JB.187.3.1014-1021.2005

Timmermans, J., and Van Melderen, L. (2010). Post-transcriptional global regulation by CsrA in bacteria. Cell. Mol. Life Sci. 67, 2897–2908. doi: 10.1007/s00018-010-0381-z

Toledo-Arana, A., Valle, J., Solano, C., Arrizubieta, M. J., Cucarella, C., Lamata, M., et al. (2001). The enterococcal surface protein, Esp, is involved in *Enterococcus faecalis* biofilm formation. Appl. Environ. Microbiol. 67, 4538–4545. doi: 10.1128/AEM.67.10.4538-4545.2001

Ude, S., Arnold, D. L., Moon, C. D., Timms-Wilson, T., and Spiers, A. J. (2006). Biofilm formation and cellulose expression among diverse environmental *Pseudomonas* isolates. Environ. Microbiol. 8, 1997–2011. doi: 10.1111/j.1462-2920.2006.01080.x

Van Melderen, L., and Saavedra De Bast, M. (2009). Bacterial toxin-antitoxin systems: more than selfish entities? PLoS Genet. 5:e1000437. doi: 10.1371/journal.pgen.1000437

Vecerek, B., Moll, I., Afonyushkin, T., Kaberdin, V., and Blasi, U. (2003). Interaction of the RNA chaperone Hfq with mRNAs: direct and indirect roles of Hfq in iron metabolism of *Escherichia coli*. Mol. Microbiol. 50, 897–909. doi: 10.1046/j.1365-2958.2003.03727.x

Vogel, J., and Luisi, B. F. (2011). Hfq and its constellation of RNA. Nat. Rev. Microbiol. 9, 578–589. doi: 10.1038/nrmicro2615

Vytvytska, O., Jakobsen, J. S., Balcunaite, G., Andersen, J. S., Baccarini, M., and Von Gabain, A. (1998). Host factor I, Hfq, binds to *Escherichia coli ompA* mRNA in a growth rate-dependent fashion and regulates its stability. Proc. Natl. Acad. Sci. U.S.A. 95, 14118–14123. doi: 10.1073/pnas.95.24.14118

Walters, M. C. 3rd., Roe, F., Bugnicourt, A., Franklin, M. J., and Stewart, P. S. (2003). Contributions of antibiotic penetration, oxygen limitation, and low metabolic activity to tolerance of *Pseudomonas aeruginosa* biofilms to ciprofloxacin and tobramycin. Antimicrob. Agents Chemother. 47, 317–323. doi: 10.1128/AAC.47.1.317-323.2003

Wang, X., Dubey, A. K., Suzuki, K., Baker, C. S., Babitzke, P., and Romeo, T. (2005). CsrA post-transcriptionally represses *pgaABCD*, responsible for synthesis of a biofilm polysaccharide adhesin of *Escherichia coli*. Mol. Microbiol. 56, 1648–1663. doi: 10.1111/j.1365-2958.2005.04648.x

Wang, X., Kim, Y., Hong, S. H., Ma, Q., Brown, B. L., Pu, M., et al. (2011). Antitoxin MqsA helps mediate the bacterial general stress response. Nat. Chem. Biol. 7, 359–366. doi: 10.1038/nchembio.560

Wang, X., Kim, Y., Ma, Q., Hong, S. H., Pokusaeva, K., Sturino, J. M., et al. (2010). Cryptic prophages help bacteria cope with adverse environments. Nat. Commun. 1, 147. doi: 10.1038/ncomms1146

Wang, X., Kim, Y., and Wood, T. K. (2009). Control and benefits of CP4-57 prophage excision in *Escherichia coli* biofilms. ISME J. 3, 1164–1179. doi: 10.1038/ismej.2009.59

Wang, X., Preston, J. F. 3rd., and Romeo, T. (2004). The *pgaABCD* locus of *Escherichia coli* promotes the synthesis of a polysaccharide adhesin required for biofilm formation. J. Bacteriol. 186, 2724–2734. doi: 10.1128/JB.186.9.2724-2734.2004

Waters, L. S., and Storz, G. (2009). Regulatory RNAs in bacteria. Cell 136, 615–628. doi: 10.1016/j.cell.2009.01.043

Watnick, P. I., and Kolter, R. (1999). Steps in the development of a *Vibrio cholerae* El Tor biofilm. Mol. Microbiol. 34, 586–595. doi: 10.1046/j.1365-2958.1999.01624.x

Webb, J. S., Givskov, M., and Kjelleberg, S. (2003). Bacterial biofilms: prokaryotic adventures in multicellularity. Curr. Opin. Microbiol. 6, 578–585. doi: 10.1016/j.mib.2003.10.014

Wei, B. L., Brun-Zinkernagel, A. M., Simecka, J. W., Pruss, B. M., Babitzke, P., and Romeo, T. (2001). Positive regulation of motility and *flhDC* expression by the

RNA-binding protein CsrA of *Escherichia coli*. Mol. Microbiol. 40, 245–256. doi: 10.1046/j.1365-2958.2001.02380.x

Weilbacher, T., Suzuki, K., Dubey, A. K., Wang, X., Gudapaty, S., Morozov, I., et al. (2003). A novel sRNA component of the carbon storage regulatory system of *Escherichia coli*. Mol. Microbiol. 48, 657–670. doi: 10.1046/j.1365-2958.2003.03459.x

Weinberg, Z., Barrick, J. E., Yao, Z., Roth, A., Kim, J. N., Gore, J., et al. (2007). Identification of 22 candidate structured RNAs in bacteria using the CMfinder comparative genomics pipeline. Nucleic Acids Res. 35, 4809–4819. doi: 10.1093/nar/gkm487

Whitchurch, C. B., Tolker-Nielsen, T., Ragas, P. C., and Mattick, J. S. (2002). Extracellular DNA required for bacterial biofilm formation. Science 295, 1487. doi: 10.1126/science.295.5559.1487

White, D., Hart, M. E., and Romeo, T. (1996). Phylogenetic distribution of the global regulatory gene *csrA* among eubacteria. Gene 182, 221–223. doi: 10.1016/S0378-1119(96)00547-1

Whiteley, M., Lee, K. M., and Greenberg, E. P. (1999). Identification of genes controlled by quorum sensing in *Pseudomonas aeruginosa*. Proc. Natl. Acad. Sci. U.S.A. 96, 13904–13909. doi: 10.1073/pnas.96.24.13904

Wu, X. G., Duan, H. M., Tian, T., Yao, N., Zhou, H. Y., and Zhang, L. Q. (2010). Effect of the *hfq* gene on 2,4-diacetylphloroglucinol production and the PcoI/PcoR quorum-sensing system in *Pseudomonas fluorescens* 2P24. FEMS Microbiol. Lett. 309, 16–24. doi: 10.1111/j.1574-6968.2010.02009.x

Yan, Y., Su, S., Meng, X., Ji, X., Qu, Y., Liu, Z., et al. (2013). Determination of sRNA expressions by RNA-seq in *Yersinia pestis* grown in vitro and during infection. PLoS ONE 8:e74495. doi: 10.1371/journal.pone.0074495

Yang, H., Liu, M. Y., and Romeo, T. (1996). Coordinate genetic regulation of glycogen catabolism and biosynthesis in *Escherichia coli* via the CsrA gene product. J. Bacteriol. 178, 1012–1017.

Zeng, Q., McNally, R. R., and Sundin, G. W. (2013). Global small RNA chaperone Hfq and regulatory small RNAs are important virulence regulators in *Erwinia amylovora*. J. Bacteriol. 195, 1706–1717. doi: 10.1128/JB.02056-12

Zhang, Y., Zhang, J., Hoeflich, K. P., Ikura, M., Qing, G., and Inouye, M. (2003). MazF cleaves cellular mRNAs specifically at ACA to block protein synthesis in *Escherichia coli*. Mol. Cell 12, 913–923. doi: 10.1016/S1097-2765(03)00402-7

Zhao, X., Koestler, B. J., Waters, C. M., and Hammer, B. K. (2013). Post-transcriptional activation of a diguanylate cyclase by quorum sensing small RNAs promotes biofilm formation in *Vibrio cholerae*. Mol. Microbiol. 89, 989–1002. doi: 10.1111/mmi.12325

Zogaj, X., Bokranz, W., Nimtz, M., and Romling, U. (2003). Production of cellulose and curli fimbriae by members of the family *Enterobacteriaceae* isolated from the human gastrointestinal tract. Infect. Immun. 71, 4151–4158. doi: 10.1128/IAI.71.7.4151-4158.2003

Zogaj, X., Nimtz, M., Rohde, M., Bokranz, W., and Romling, U. (2001). The multicellular morphotypes of *Salmonella typhimurium* and *Escherichia coli* produce cellulose as the second component of the extracellular matrix. Mol. Microbiol. 39, 1452–1463. doi: 10.1046/j.1365-2958.2001.02337.x

Conflict of Interest Statement: The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

Received: 08 January 2014; accepted: 08 March 2014; published online: 25 March 2014.
Citation: Martínez LC and Vadyvaloo V (2014) Mechanisms of post-transcriptional gene regulation in bacterial biofilms. Front. Cell. Infect. Microbiol. 4:38. doi: 10.3389/fcimb.2014.00038

This article was submitted to the journal Frontiers in Cellular and Infection Microbiology.

Copyright © 2014 Martínez and Vadyvaloo. This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.
