
Published April 14, 2014

# The state of the art of odorant receptor deorphanization: A report from the orphanage

Zita Peterlin,$^{1}$ Stuart Firestein,$^{2}$ and Matthew E. Rogers$^{1}$

$^{1}$Corporate Research and Development, Firmenich Incorporated, Plainsboro, NJ 08536  
$^{2}$Department of Biological Sciences, Columbia University, New York, NY 10027  

The odorant receptors (ORs) provide our main gateway to sensing the world of volatile chemicals. This involves a complex encoding process in which multiple ORs, each of which detects its own set of odorants, work as an ensemble to produce a distributed activation code that is presumably unique to each odorant. One marked challenge to decoding the olfactory code is OR deorphanization, the identification of a set of activating odorants for a particular receptor. Here, we survey various methods used to try to express defined ORs of interest. We also suggest strategies for selecting odorants for test panels to evaluate the functional expression of an OR. Integrating these tools, while retaining awareness of their idiosyncratic limitations, can provide a multi-tiered approach to OR deorphanization, spanning the initial discovery of a ligand to vetting that ligand in a physiologically relevant setting.

Enormous progress has been made in identifying ligands for G protein–coupled receptors (GPCRs) since the 5HT${}_{\mathrm{A}}$ serotonin and D${}_{2}$ dopamine receptors were identified as such 25 years ago (Bunzow et al., 1988; Fargin et al., 1988). There is, however, one notable and conspicuous exception: the odorant receptors (ORs). ORs comprise nearly 50% of the ~800 GPCRs in humans, yet ~90% of human ORs remain orphan receptors with unknown ligands. This is surprising when you consider that more than half of all non-olfactory GPCRs have been deorphaned to date. Furthermore, unlike most non-olfactory GPCRs, our olfactory perceptual experience means deorphanization strategies for ORs are not necessarily burdened by a lack of knowledge of relevant ligands (Civelli et al., 2013). The molecular mechanisms of OR activity have also been well described (e.g., Firestein, 2001). It is unlikely that fundamental differences in molecular function are the bottleneck in OR deorphanization, as ORs appear to use the same basic molecular mechanisms as other class A GPCRs in terms of ligand binding and transduction (Bockaert and Pin, 1999; Gether, 2000). Regardless, ORs are “poorly behaved” when it comes to functional assays.

Yet large-scale OR deorphanization is needed to elucidate basic principles of odor coding. Deorphanization will allow investigators to probe how OR activity is influenced by specific physicochemical properties of odorant molecules (i.e., develop odorant structure–activity relationships) and ultimately how this information is processed in the brain to elicit an “odor image” that can influence human or animal behavior by providing information on such aspects as perceived odor intensity, odor detection threshold, and valence. As Jerome Lettvin so eloquently and simply put it for the visual system with his seminal 1959 paper “What the Frog’s Eye Tells the Frog’s Brain,” the olfactory field must first know “what the nose knows” to understand what it tells the brain (Lettvin et al., 1959). This Review focuses on recent advances in OR deorphanization, from both the standpoint of receptor expression and that of ligand assay panel selection, highlighting some of the caveats that abound in this continuing but challenging endeavor.

## Methods of deorphanization

### Deorphanization using olfactory sensory neurons (OSNs)

The most significant impediment to deorphaning an OR of interest has been the lack of a suitable cell-based heterologous expression system. The first three assays discussed circumvent this block by exploiting the OSN itself as the expression system. Using OSNs decreases the difficulty of expressing functional exogenous ORs because the cellular machinery, whatever that may be, is already poised to process ORs. Moreover, the receptive field of the OR (that is, the set of ligands capable of activating the OR and the relative potency of such ligands) is presumed to be of high fidelity because the OR is coupled to its native transduction cascade. There are tradeoffs in terms of the difficulty of the methodology and the throughput rate, but the techniques of adenoviral infection, gene targeting, and imaging of the dorsal olfactory bulb (OB) have added substantially to our knowledge of select ORs.

---

Correspondence to Matthew E. Rogers: matthew.rogers@firmenich.com

Abbreviations used in this paper: CRE, cAMP response element; FLIPR, fluorescent imaging plate reader; GFP, green fluorescent protein; GPCR, G protein–coupled receptor; IRES, internal ribosome entry site; OB, olfactory bulb; OR, odorant receptor; OSN, olfactory sensory neuron; REEP, receptor expression–enhancing protein; RTP, receptor-transporting protein.

The Rockefeller University Press  
J. Gen. Physiol. Vol. 143 No. 5 527–542  
www.jgp.org/cgi/doi/10.1085/jgp.201311151

© 2014 Peterlin et al. This article is distributed under the terms of an Attribution–Noncommercial–Share Alike–No Mirror Sites license for the first six months after the publication date (see http://www.rupress.org/terms). After six months it is available under a Creative Commons License (Attribution–Noncommercial–Share Alike 3.0 Unported license, as described at http://creativecommons.org/licenses/by-nc-sa/3.0/).
The earliest approach to studying a defined OR was to use a recombinant adenovirus to deliver exogenous genes into OSNs (Zhao et al., 1998; Touhara et al., 1999). In this approach, the virus is constructed with a constitutively active promoter that drives overexpression of an OR and a green fluorescent protein (GFP) marker. An internal ribosome entry site (IRES) sequence sandwiched between the exogenous OR and GFP produces the receptor and GFP as separate proteins in infected cells (Potter et al., 2001). Introducing exogenous ORs into a native cellular environment permits proper trafficking and expression of untagged, unmodified receptors on the plasma membrane. Using the adenoviral technique, ligands for rat OR-I7—the first OR to be deorphaned—were discovered by recording OR activity ex vivo, using an electro-olfactogram (a moderate throughput screening methodology in which odorants are delivered in vapor phase to the intact olfactory epithelium) (Zhao et al., 1998). The adenoviral technique, coupled to calcium imaging of infected OSNs from dissociated olfactory epithelium (a low throughput ex vivo screening methodology in which odorants are delivered in aqueous phase to dispersed cells), was also used to confirm a ligand for mouse MOR23 (Touhara et al., 1999).

Gene-targeting strategies in mice provide another way to deorphan ORs in their native cell type. In this approach, a fluorescent marker protein sequence is inserted adjacent to the endogenous locus of a specific OR. Once again, an IRES sequence is used to mark cells expressing the particular OR. Like the adenoviral approach, the OR being gene targeted is typically left untagged and unmodified. But unlike the adenovirus approach, in which the exogenous and endogenous ORs are coexpressed in the same OSN, the gene-targeting approach permits expression of only the single OR. Two highly related mouse ORs, M71 and M72, with 96% amino acid identity, were deorphaned through gene targeting, using ex vivo calcium imaging of labeled M71 or M72 OSNs from dissociated olfactory epithelia (Bozza et al., 2002; Feinstein et al., 2004).

Fully in vivo expression systems for ORs in OSNs have now been developed by coupling gene-targeting and OB recording techniques. This allows for realistic vapor-phase odorant delivery where the presence of mucus is preserved and odorants can be delivered under physiological conditions. The olfactory system is kept intact, from cell–cell contacts in the olfactory epithelium all the way to axonal projections to the brain. Further, odorants are delivered in the vapor phase. Although it is difficult to know the precise concentration and even the final form of an odorant that reaches the olfactory neurons because of the influence of an air–water or mucus interphase (Nagashima and Touhara, 2010), vapor-phase delivery provides a more realistic physicochemical environment. In contrast, in vitro and ex vivo

(e.g., calcium imaging of dissociated olfactory epithelium) assays typically require odorants to be delivered in the water phase. Although water-phase odorant delivery may not exactly recapitulate the native physicochemical environment, it does allow for more accurate assessment of the concentrations of odorant delivered. Indeed, even higher odorant concentrations may be achieved in the water phase depending on the vapor pressure of the odorant. This may be one reason for the observed discrepancies in receptor profiles observed across the different assay systems (see below; Oka et al., 2006). Thus, although a particularly technically challenging technique, in vivo assays provide the most realistic albeit perhaps a less well defined olfactory environment in which to perform functional studies.

The in vivo recording method takes advantage of the stereotyped topography of OSN projections between the olfactory epithelium and the OB. Sensory neurons expressing a specific OR coalesce in roughly two stereotyped locations of the OB called glomeruli, one located laterally and one medially (Ressler et al., 1994; Vassar et al., 1994; Mombaerts et al., 1996). These neuropil conglomerates are composed largely of the axons of OSNs synapsing onto dendrites of mitral and tufted neurons, with these second-order neurons then projecting axons forward to cortical regions of the brain (Haberly and Price, 1977; Shepherd, 2006; Ghosh et al., 2011). Glomeruli located on the dorsal surface of the OB are accessible with minimal surgical manipulation, making them amenable to in vivo optical imaging recordings of OSN activity. Several imaging techniques have been used in the dorsal bulb to detect odorant-induced activity of OSNs of unknown OR identity. These include recording intrinsic signals (Rubin and Katz, 1999; Uchida et al., 2000), using the genetically encoded activity marker synapto-pHluorin (Bozza et al., 2004), applying calcium-indicator dyes (Oka et al., 2006), and using optogenetic reporters for calcium and membrane voltage (Storace, D.A., L.B. Cohen, and U. Sung. 2013. Association for Chemoreception Sciences Annual Meeting. Abstr. P109). Although beneficial as an in vivo technique, imaging the dorsal surface of the OB only provides a partial picture of receptor activity because it can only report on the subset of OSNs that project to the dorsal glomeruli. Although receptor activity has been recorded from other areas of the OB (Igarashi and Mori, 2005; Shirasu et al., 2014), this requires substantial surgical manipulation and still suffers from sampling only a small subpopulation of receptor types.

More recently, specific ORs have been gene targeted so that identified ORs can be screened in the dorsal OB (Belluscio et al., 2002; Bozza et al., 2002). Here, a fluorescent marker (e.g., tauGFP) is inserted in the locus just downstream of an OR that directs axons to the dorsal OB (such as M71). The resultant GFP-labeled glomerulus for that specific OR can thus be identified while

528 Odorant receptor deorphanization
functional activity can be viewed using intrinsic activity or a fluorescent activity indicator of a different wavelength. An extension of this technique expands the range of ORs that can be tested by replacing the dorsal bulb-targeting OR sequences with OR sequences that would normally direct axons elsewhere (as in the I7→M71 locus swap). The OSNs that express the replacement OR now form glomeruli located in the dorsal OB and are therefore amenable to dorsal OB imaging techniques (Soucy et al., 2009; Tsuboi et al., 2011). This gene-targeted approach in which dorsal-projecting OR loci are replaced by presumably any target OR sequence currently allows for the highest throughput of ligand screening among OSN-based systems, and it also allows odorants to be screened in the more physiological vapor phase. However, producing such a mouse line is both expensive and slow.

Contributions from OSN expression systems. Native expression systems, in which defined ORs are expressed in OSNs, have yielded important insights into fundamentals of the olfactory code. Rat OR-I7 and mouse M71/M72 provide good examples. OR-I7 has been screened with panels of diverse and panels of only subtly different chemicals, thus providing in-depth structure-activity relationship insight for this receptor (Araneda et al., 2000). The initial electro-olfactogram recordings from OR-I7 adenovirus-infected regions of the epithelium revealed that, among a diverse panel of 74 test odorants spanning multiple chemical functional groups, only saturated aliphatic *n*-aldehydes with simple backbone chains of 7–10 carbons activated OR-I7. Within this range, the eight-carbon octanal was the most potent. Octanal was then used as the lead compound in a further electro-olfactogram screen in which a medicinal chemistry approach was undertaken. 90 different compounds with subtle and systematic changes in functional group, carbon chain length, degree of saturation, and type of side-chain substitution were tested (Araneda et al., 2000). This screen revealed that OR-I7 could tolerate particular patterns of unsaturation and branching but not others. This work provided the first thorough characterization of an OR's molecular receptive field and identified citral, a partial agonist of OR-I7, as the first known modulator of any OR’s activity.

Screening OSNs from M71 or M72 IRES tauGFP gene-targeted animals (Bozza et al., 2002; Feinstein et al., 2004) provided an opportunity to investigate how closely related receptors parse odorants. These receptors differ by only 11 amino acids. In initial calcium imaging of labeled OSNs from dissociated olfactory epithelia, the two ORs both seemed narrowly tuned to acetophenone. However, later patch-clamp recordings from the dendritic knobs of labeled OSNs revealed several receptor-discriminating ligands from among

a large screen of 330 compounds (Zhang et al., 2012). How minor changes in OR amino acid composition can affect response profiles, and the implications for how confidently one can predict receptive fields based on overall sequence similarity, is an area open to active investigation.

Dose-response relationships ($\mathrm{EC}_{50}$s) of GFP-labeled M71 OSNs in the dissociated olfactory epithelium preparations display a surprising level of variability. For instance, the $\mathrm{EC}_{50}$ for acetophenone differed by more than an order of magnitude in different experiments (Bozza, et al., 2002). This variability is also evident in ex vivo patch recordings of intact epithelia, as seen in the response of the mouse MOR23 receptor to Lyral (Grosmaître et al., 2006), suggesting that it is not merely a consequence of the dissociation method. Thus, it may be an inherent biological property of OSNs. Although this phenomenon has yet to be explored in in vivo systems, such biological variability calls into question how odorant concentration is encoded when OSNs expressing the same OR can nevertheless have quite different sensitivities.

An important aspect to consider when comparing multiple expression systems is fidelity, and the OSN-based expression systems show good cross-validation. In vivo functional imaging of GFP-labeled OR-I7 glomeruli (Belluscio et al., 2002) has confirmed ex vivo electro-olfactogram and OSN calcium imaging results that indicate octanal is a potent OR-I7 ligand.

Deorphanization using heterologous expression systems. Although they most faithfully recapitulate the receptive field, OSN-based expression systems are relatively low throughput. Moreover, they have been the principal mode of extended investigation for only seven ORs: rat OR-I7 (Zhao et al., 1998), mouse OR-I7 (Bozza et al., 2002), MOR23 (Touhara et al., 1999), MOR-EG (Kajiya et al., 2001), MOR-EV (Kajiya et al., 2001), M71 (Bozza et al., 2002), and M72 (Feinstein et al., 2004). Although the main goal of high throughput compound screening is to identify more comprehensively the set of odorants that make up a given OR’s receptive field, in time this process can lead to a compendium of ORs that share a given ligand (Malnic, 2007; Touhara, 2007; Krautwurst, 2008; Reisert and Restrepo, 2009). Large scale, heterologous-based screening thus holds the promise of providing a more substantive window onto the olfactory combinatorial code.

A particularly challenging problem, however, is that many ORs are retained in the ER of heterologous cells. Failing to traffic to the cell surface, the ORs are unable to interact with the odorant (Lu et al., 2003, 2004). Models of a two-step process of intracellular OR trafficking in OSNs tend to focus primarily on the role of OSN-specific forward trafficking proteins (McClintock and Sammeta, 2003). These include assistive proteins such

as receptor-transporting protein (RTP) 1, RTP2, and receptor expression–enhancing protein (REEP) 1 (Saito et al., 2004), and certain non-olfactory GPCRs that may act as heterodimer partners (Bush and Hall, 2008). One relatively unexplored possibility, however, is the inverse scenario in which heterologous cells possess or overexpress one or more negative trafficking factor(s). Such factors may retain ORs in the ER because of protein misfolding or because they fail to recognize even a properly folded OR as appropriate export cargo. In support of this possibility, ER retention by the organelle’s quality control machinery has been documented for several non-olfactory GPCRs (Kim and Arvan, 1998; Ellgaard and Helenius, 2003; van Anken and Braakman, 2005). The OR trafficking complex in the OSN may thus involve the presence of OSN-specific forward trafficking factors or the absence of specific ER quality-control genes, both of which are largely unknown. A more thorough understanding of the apparently unique protein trafficking environment in OSNs could suggest modifications to help alleviate the bottleneck of functional OR expression in heterologous cells.

Some modified cell lines do provide an acceptable environment for functionally expressing at least certain ORs. Although systems such as *Xenopus laevis* oocytes (Katada et al., 2003; Abaffy et al., 2006, 2007), the insect Sf9 cell line (Matarazzo et al., 2005), yeast (Minic et al., 2005), even budding baculovirus (Mitsui et al., 2012) have been used, the mainstay of deorphanization through heterologous expression has been two human cell lines, HEK293 and HeLa (Fig. 1). HEK293 has been a reliable vehicle for studying mOR-EG (Kajiya et al., 2001; Katada et al., 2003, 2005; Oka et al., 2004a,b, 2006; Baud et al., 2011), making mOR-EG a staple receptor for pharmacology studies and comparison with other heterologous expression systems. mOR-EG was identified through single-cell RT-PCR (Kajiya et al., 2001) of eugenol-responsive OSNs from dissociated mouse olfactory epithelia. Rather strikingly (in retrospect), mOR-EG could be functionally expressed directly in HEK293 cells,

**Figure 1.** OR signal transduction and functional readout of the main in vitro assay systems (A–C) and ex vivo dissociated OSNs (D). Salient differences between these assays include the use of N-terminal amino acid tags (e.g., Rho) and trafficking proteins (e.g., RTP1S) to promote cell surface expression, Gαolf or Gα₁₅ G proteins that trigger calcium ion (Ca²⁺) flux or cAMP production upon ligand binding, and Ric8b or myristoylated Ric8B (mRic8b) proteins that attenuate G protein signaling. RTP2 and REEP are present in Hana3A cells but appear to only marginally improve functional OR expression. RTP1S, REEP, and Ric8b mRNAs are expressed in OSNs, but there is a paucity of evidence for their function there. ACIII, adenylyl cyclase type III; ANO2, anoctamin 2 calcium-activated chloride channel; CNGA2, cyclic nucleotide–gated ion channel A2; CRE, cAMP response element; Luc, luciferase protein; OR, odorant receptor.

without cofactors or even an N-terminal tag on the OR. Although a non-olfactory G protein, Gα15, is the OR coupling partner in HEK293 systems, mOR-EG still responded to the same ligands in the in vitro assay as those identified in the original ex vivo OSN screening (Kajiya et al., 2001). HEK293 cells have also been used to express the human ORs hOR17-4 (Spehr et al., 2003), hOR17-40 (Wetzel et al., 1999), and OR51E2 (Shepard et al., 2013), as well as the rat OR-I7 (Krautwurst et al., 1998), and the mouse ORs OR-I7 (Krautwurst et al., 1998) and Olfr78 (Shepard et al., 2013), although some of these latter cases involved the addition of a tag or leader peptide sequence to the N terminus of the OR sequence.

The most common functional readout for ORs expressed in HEK293T cells coupled to Gα15 is an intracellular increase in calcium, monitored using fluorescent calcium-binding dyes such as fura-2. This enables a fast readout, as the aqueous-delivered odorants only need to be applied transiently. But because odorants are typically applied sequentially (as opposed to concurrent application using a fluorescent imaging plate reader [FLIPR] assay, as will be discussed later), the overall time course of the experiment is prolonged and only a limited number of odorants can be tested on the same cell preparation. Such imaging thus represents only a low to mid-throughput approach.

The most widely used in vitro OR assay uses the Hana3A cell line, an extensively engineered system derived from HEK293T cells (Saito et al., 2004). The Hana3A cell line stably expresses the canonical OSN G protein α subunit Gαolf, bringing the signal coupling closer to an in vivo situation. Hana3A also stably expresses three OSN OR trafficking cofactors (RTP1, RTP2, and REEP) that have been shown to enhance cell surface expression of a subset of tested ORs (Saito et al., 2004). In addition, coexpression of an alternative splice variant of RTP1 called RTP1S (Wu et al., 2012), the guanine nucleotide exchange factor Ric8B (Von Dannecker et al., 2005, 2006; Kerr et al., 2008), and the type 3 muscarinic acetylcholine receptor (Li and Matsunami, 2011) indicates that all three appear to work synergistically to enhance OR functionality without altering ligand specificity (Zhuang and Matsunami, 2007).

In addition to this collection of cofactors, optimal cell surface expression often requires modification of the OR itself. In the Hana3A system, as in nearly all other in vitro OR assays, fusing the first 20 amino acids of rhodopsin (Rho tag) or other short noncleavable tags (e.g., Flag, c-myc, or HA) to the receptor’s N terminus is common and typically necessary for optimal functional expression. But it is possible that such permanent tags modify OR response profiles (Zhuang and Matsunami, 2007). Recently, a cleavable 17-amino acid leucine-rich N-terminal signal peptide (Lucy tag) was shown to promote cell surface OR expression in HEK293T cells (Shepard et al., 2013). The Lucy tag appears to operate synergistically with RTP1S, Ric8b, and a Flag-Rho tag. Strikingly, all 15 of the ORs tested with Lucy could be visualized on the cell surface by immunostaining. The use of cleavable tags like Lucy, where little or no tag remains on the mature OR, should help improve the in vivo relevance of screening data derived from in vitro systems.

The functional readout from Hana3A-based systems comes from pairing Gαolf to an endogenously expressed adenylate cyclase to yield odorant-induced cAMP production. The increase in cAMP is monitored via a cAMP response element (CRE)-driven luciferase reporter, yielding light as the output (Katada et al., 2003; Saito et al., 2004). A potential drawback, however, is that when used with a standard luminometer plate reader, measurable and consistent odorant-dependent luciferase production in the CRE luciferase assay can require up to four hours of odorant stimulation. The potential for OR adaptation during such prolonged odorant exposure is unknown.

An alternative to these two main HEK293T-based systems is the HeLa/Olf expression system. This has been used successfully to deorphan ORs without added trafficking cofactors, although an N-terminal Rho tag on the OR is still typically used (Shirokova et al., 2005). The HeLa/Olf system includes Gαolf and an adenylate cyclase like Hana-3A but differs in its functional readout. For the HeLa/Olf system, the odorant-induced cAMP activates an exogenously expressed variant of the CNG channel (CNGA2). cAMP binding to CNGA2 stimulates calcium influx, which can be detected using a fluorescent calcium-sensitive dye (e.g., Calcium 5). This calcium change could, in theory, be monitored for sequential applications of odorants as in the HEK assay. But in investigations using HeLa/Olf, the primary strategy has instead been to record receptor activity in real time with a FLIPR. FLIPR assays, which can be run in a multi-well format, are a widely used strategy for high throughput screening of GPCRs in the pharmaceutical industry (Emkey and Rankl, 2009). The HeLa/Olf system thus allows for reduced odorant exposure times compared with luciferase reporter systems.

**Caveats of heterologous expression systems.** Heterologous expression systems have enabled the deorphanization and extended characterization of ∼95 mouse and 41 human ORs (Mainland et al., 2014) as of the beginning of 2014. Although still this means that only ∼8 and ∼10% of each species’ OR repertoire, respectively, have been deorphaned, this contribution by in vitro studies stands in marked contrast to the mere seven ORs primarily studied using OSN-based methods and confirmed in separate studies over this same period (i.e., rat OR-I7 [Zhao et al., 1998], mouse OR-I7 [Bozza et al., 2002], MOR23 [Touhara et al., 1999], MOR-EG [Kajiya et al., 2001], MOR-EV [Kajiya et al., 2001], M71 [Bozza et al., 2002], and M72 [Feinstein et al., 2004]). Thus, heterologous
expression assays are the undisputed primary drivers of deorphanization. But several cautionary cases of assay-dependent bias should be considered when interpreting OR response profiles obtained in vitro.

One cautionary case involves the G protein identity. Some GPCRs can couple with more than one G protein α subunit. Recent evidence suggests that when a given GPCR is docked to a particular G protein α subunit, a certain subset of chemicals may serve as ligands, but when docked to an alternate G protein α subunit, the functional set of ligands may change. Thus, some ligands preferentially signal through a particular pathway (Sivertsen et al., 2013). Such stimulus signaling bias has been reported for ORs in heterologous systems and can alter the resulting odorant response profile (Shirokova et al., 2005; Oka et al., 2006; Hamana et al., 2010). Shirokova et al. (2005) showed that the noncanonical Gα₁₅, the G protein commonly used in HEK-based systems, can alter the odorant specificities and pharmacological profile of ORs. For example, when ORs6 was coupled to Gα₁₅, octanoic acid was an agonist. But when ORs6 was coupled to Gαolf, octanoic acid not only failed to activate the receptor, but it actually served as a competitive antagonist. The same study by Shirokova et al. (2005) also illustrated how the Gα₁₅ G protein displays “permissiveness”; 7 of the 14 ligands for mouse receptor olfr43 were agonists in only the Gα₁₅ but not the Gαolf background, suggesting that Gα₁₅ may enable more indiscriminate ligand binding. Although Gα₁₅ assays may thus provide a good measure of ligand-binding potential, they may not reveal the whole story with regard to receptor function. Ligand efficacy, which includes the activity of the entire receptor–G protein complex and downstream signaling cascade, is a more relevant measure of receptor activity. This suggests that assays involving the canonical G protein, Gαolf, may more accurately measure ligand efficacy and may therefore provide a better gauge of the OR’s discriminatory capacity. Although the inconsistencies depending on coupling agent make comparisons across studies difficult, they may lead us to a better understanding of how the G protein–OR complex acts as a unit in determining OR receptive fields.

Although the influence of the assay on the reported receptive fields has been documented for rat OR-I7, (Mombaerts, 2004), MOR29B (Tsuboi et al., 2011), and the trace amine-associated receptor TAAR4 (Zhang et al., 2013), this bias has been most thoroughly explored for mOR-EG (Fig. 2). Oka et al. (2006) compared the specificity and sensitivity of the mOR-EG response profile in vitro in HEK293 cells coupled to either Gα₁₅ or Gαolf, ex vivo by calcium imaging of OSNs from the dissociated olfactory epithelium of a mOR-EG-labeled transgenic mouse, and in vivo by intrinsic imaging of GFP-labeled glomeruli in this same transgenic mouse line. The response profile of mOR-EG to seven

odorants in HEK293 and dissociated OSNs was nearly identical, although some difference was observed depending on the G protein used in the HEK293 assay. In particular, the downstream coupling partner affected the ability of isoeugenol to serve as an agonist. But the most unexpected results came from the in vivo assay. Vanillin and 4H3MBA, delivered in water phase, were strong mOR-EG agonists in both HEK293 cells coexpressing Gαolf and dissociated OSNs (which also express Gαolf). Yet when OSN activity was measured in the vapor phase at the level of the glomeruli, very modest or no responses to these odorants was obtained. This was not caused by a general decrease in sensitivity in the in vivo method of recording. Indeed, an ~1,000 times increase in the sensitivity to eugenol was observed with in vivo glomerular imaging compared with the other assay systems. Intriguingly, the responses of mOR-EG glomeruli to vanillin before and after washout of the nasal cavity suggest that the olfactory mucus may account for part of the observed difference. Thus, caution extends even to ex vivo work with OSNs from dissociated olfactory epithelia; the presence of mucus and a fully intact in vivo system may better define the physiologically relevant boundaries of a receptive field. But until it is resolved how odorants may partition into the mucus, and how this affects their effective concentration, a continued benefit of ex vivo work is that the aqueous concentrations can be well controlled. This in turn provides an accurate view of which odorants can bind a given OR and enables the development of structure–activity relationships.

Contributions from heterologous expression systems. Despite these caveats, the throughput capacity of heterologously expressed ORs offers a valuable window to OR biology. For example, mOR-EG’s extensive characterization in the HEK293 system (Katada et al., 2003, 2005; Oka et al., 2004a,b) and the ability to readily express even mutated mOR-EG receptors (Katada et al., 2005; Kato et al., 2008) paved the way for detailed analysis of the relationship between agonist and antagonist ligands. From screening with blends, the odorants methyl isoeugenol, isosafrole, and dimerized isoeugenol were identified as antagonists of mOR-EG as expressed in HEK293 cells with Gα₁₅ (Oka et al., 2004a,b). These results, along with trends among identified agonists and antagonists for other ORs (Araneda et al., 2000; Sanz et al., 2005; Shirokova et al., 2005; Jacquier et al., 2006; Abaffy et al., 2007; Peterlin et al., 2008), suggest that most OR antagonists are likely to be structurally related to the agonist compounds, be odorants themselves, and exhibit competitive antagonism. But it was through a large structure–activity relationship study coupled with point mutagenesis that a mechanism for antagonism in mOR-EG was proposed and tested (Katada et al., 2005). Vanillin, a potent agonist, was used as the lead compound in a medicinal chemistry approach that varied
the substituents around three positions of vanillin's
benzene ring. 21 additional compounds were screened
and 13 were found to activate mOR-EG over a 100-fold
span in relative affinity (EC₅₀), starting in the low micro-
molar range. By combining these structure–activity rela-
tionship results with the results from point mutations

and computational homology modeling, the mOR-EG
odorant-binding site was predicted. It consisted of a
hydrophobic pocket formed by portions of transmem-
brane domains III, V, and VI. Nine amino acids within
the presumptive binding pocket were shown to be in-
volved in receptor activation. They propose that there


| Rat I7 | Mouse OR-EG |
| --- | --- |
| 1 | 2,3,4 | 5 | 6 | 7 | 7 | 8 | 9 | 9 | 9 | 10 | 7 | 7,11,12 |
| Int | Ca²⁺ | Ca²⁺ | Ca²⁺ | Ca²⁺ | Ca²⁺ | I | luc | luc | luc | Seap | cAMP | Ca²⁺ |
| Glom | OSN | COS | OSN | OB | OSN | Ocy | H3A | H3A | H3A | HEK | HEK |
| -- | -- | myc | -- | -- | -- | rho | rho | flag | HA | rho | rho |
| olf | olf | -- | olf | olf | olf | olf | olf | olf | olf | olf | olf | 15 |

Odorant
pentanal
hexanal
heptanal
octanal
nonanal
decanal
undecanal
trans-2-octenal
trans,trans-2,4-octadienal
2-octynal
citral
citronellal
5-cyclopropylpentanal
4-cyclobutylbutanal
3-cyclopentylpropanal
2-cyclohexylacetalddehyde
cycloheptanecarbaldehyde
cinnamaldehyde
eugenol
eugenol acetate
iso eugenol
eugenol methyl ether
vanillin
acetovanillin
methyl isoeugenol
isosafrole
vanillic alcohol
ethyl vanillin
creosol
3-methoxy-4-hydroxyphenylethanol
4-formyl-2-methoxyphenyl acetate
4-ethylguaiacol
guaiaacylacetone
4-ethoxy-3-methoxybenzaldehyde
veratraldehyde
4-methoxy-3-methylbenzaldehyde
4-hydroxy-3-methylbenzaldehyde
raspberry ketone
tarragol
Rose oxide
methyl vanillate
4-isopropylphenol
4-tert-butylphenol
Z95
zingiberone
4-tert-butylocyclohexanone
tricyclene
Orivone
mousse cristal
(+) Nootkatone
wolfwood

Reference
Assay
Cell Type
Tag
G-Protein

Figure 2. Response profiles of rat OR-I7 and mouse OR-EG expressed in various assay systems. The response profiles of a given OR can vary depending on the assay system. For rat OR-I7 and mouse mOR-EG, tested but inactive compounds are not included in this figure unless the compound showed agonist or antagonist activity in at least one of the assays shown. For rat OR-I7, only select agonists are reported. See Araneda et al. (2000) for a more complete listing of OR-I7 agonists. Blue boxes, heat map of reported EC₅₀ values; darkest blue, <0.1 μM; lightest blue, 660 μM; black boxes, agonist but no EC₅₀ reported; orange boxes, partial agonist; red boxes, antagonist; white boxes, tested but inactive; Int, intrinsic imaging; Ca²⁺, calcium imaging; I, current; luc, luciferase; Seap, secreted alkaline phosphatase; OB, olfactory bulb; OSN, olfactory sensory neuron; COS, African green monkey cells; Ocy, Xenopus oocytes; H3A, Hana3A cells; HEK, human embryonic kidney cells. References: 1, Belluscio et al. (2002); 2, Araneda et al. (2004); 3, Peterlin et al. (2008); 4, Kurland et al. (2010); 5, Levasseur et al. (2003); 6, Bozza et al. (2002); 7, Oka et al. (2006); 8, Repicky and Luetje (2009); 9, Zhuang and Matsunami (2007); 10, Baud et al. (2011); 11, Katada et al. (2005); 12, Oka et al. (2004a,b).
are critical electronic interactions between a phenyl ring in the binding pocket and the particular double-bond pattern in the tail of isoeugenol and related antagonists. A simple shift of the double bond by one position was sufficient to turn an isoeugenol-like antagonist into a eugenol-like agonist. Thus, this electrostatic interaction between ligand and receptor may be the switch that filters agonists from antagonists for this OR.

Heterologous expression systems can also provide deep insight about the combinatorial code, as is evident in the work of Saito et al. (2009). Using the Hana3A cell-line approach, this group produced the largest single dataset to date, in which 93 odorants, spanning a large portion of the chemical space of commercially available odorants, were tested on 62 ORs. The authors then determined the potency of each of the 63 odorants that activated at least one OR. The resulting information-dense matrix revealed intriguing and unexpected co-activation patterns by odorants both across ORs and within the receptive field of individual ORs. Because these ORs are heterologously expressed, these patterns can be probed further. The ability to devise hypotheses about chemical relationships and then revisit the same ORs, to test them with a new panel of compounds, is a great strength of heterologous expression systems.

In silico approaches to deorphanization

The challenges of expressing ORs on the cell surface and of probing them with very large numbers of ligands make computational in silico approaches an attractive alternative for deorphaning ORs. A robust in silico system could conceivably allow virtual high throughput screening of all compounds that possess physicochemical features that make them likely odorants. This would provide a substantial dataset from which to develop a detailed pharmacophore model of the chemical features and their arrangement that is required for activation of the OR in question. A glimpse of such potential was seen for the in vitro deorphaned OR1G1, which was activated by 59 of 95 diverse odorants (Sanz et al., 2005). Having such a large agonist repertoire allowed derivation of the best-defined pharmacophore for an OR to date (Sanz et al., 2008). In silico approaches that build up large datasets of predicted binders and nonbinders may thus help identify the true feature detector capabilities of each OR.

One major in silico approach that lends itself to receptor deorphanization is to construct OR models and dock potential ligands (Singer and Shepherd, 1994; Singer, 2000; Hall et al., 2004; Hummel et al., 2005; Katada et al., 2005; Lai et al., 2005, 2008, 2014; Abaffy et al., 2007; Schmiedeberg et al., 2007; Kurland et al., 2010; Anselmi et al., 2011; Baud et al., 2011; Charlier et al., 2012; Gelis et al., 2012; Launay et al., 2012). OR modeling has typically worked in close concert with a biologically expressed OR, with deorphaning of the OR in the

biological expression system preceding development and refinement of an in silico model. Models have primarily been used to help direct site mutagenesis studies, propose rationale for ligand discrimination, and address other questions of receptor biology versus serving as a stand-alone screening system. On the rare occasions where OR models have been used to try to predict novel ligands for a given OR, success rates have been mixed. Hall et al. (2004) built two models of the rat OR-I7 receptor, into which 62 putative ligands were docked and their predicted binding energy was calculated. A high binding energy should correlate with empirical binding. In their first build, no molecule had a binding energy of 30 kcal/mol or higher. When Hall et al. (2004) refined their model-building procedure, they now found odorants that bound in the 30-kcal/mol range, 69% of which were later empirically confirmed to be activating ligands. In silico screening of libraries of ligands has recently seen a resurgence. For MOR42-3, 40 candidates were selected from an initial 574 odorant panel based on docking to a receptor model; 55% of these high scorers were validated in a physiological screen (Bavan et al., 2014). Dynamic modeling has also brought improvements to in silico screening systems, correctly predicting the binding of amyl butyrate to hOR2AG1 and mOR283-2 (Gelis et al., 2012).

ODORactor, a web-based tool, takes a quite different approach to in silico–assisted deorphanization (Liu et al., 2011). OR models approach the combinatorial code from the “one OR is activated by multiple odorants” facet. ODORactor approaches the code from the “one odorant activates multiple ORs” facet, making in silico predictions of the suite of ORs that detect a given ligand. Eschewing model building, ODORactor has data-mined all published deorphaned human and mouse ORs and uses multidimensional analysis and chemoinformatics in its predictions. ODORactor may prove to be a valuable tool when the odorant is of special interest, such as for potential pheromones or other behavior-modulating chemicals (Lin et al., 2005), or odorants of practical (Corcelli et al., 2010; Li et al., 2012) or commercial relevance (Bieri et al., 2004).

Caveats to in silico approaches. Although in silico approaches show promise, at present they still benefit from validation by biological means. Some of the challenges still to be surmounted by the modeling approach are evident in work on rat OR-I7. Rat OR-I7 has been modeled independently for over a decade (Singer, 2000; Hall et al., 2004; Lai et al., 2005; Khafizov et al., 2007; Kurland et al., 2010) using various technical approaches. However, only one residue is consistently implicated in binding octanal; the remaining residues predicted to line the binding pocket are highly contested. Alignment of ORs to a template GPCR is a key step in certain types of model building. Bovine rhodopsin in its

534 Odorant receptor deorphanization
retinal-bound, inactive state was at one time the only mammalian class A GPCR with 3-D structural information (Palczewski et al., 2000). Even recent OR models still often use the rhodopsin template as part of the construction and optimization process. But now an array of other crystallized class A GPCRs in their active state are available. The GPCRAutomodel web tool enables users to select among these new templates when building an OR model (Launay et al., 2012). Whether new alignment choices lead to improved receptive field predictions remains to be seen.

Because ODORactor updates its database with each new publication, one thing to be mindful of is that its predictions are dependent on the sum quality of published data. For example, the predictions for mouse ORs made by the current version of ODORactor may be heavily weighted by the receptor and ligand profiles present in the in vitro heterologous expression work of Saito et al. (2009), given that this study contributes nearly half of all currently deorphaned mouse ORs. How ODORactor deals with conflict, given that the details of the receptive field for the same OR can vary depending on the readout system it is coupled to (Fig. 2), also needs to be considered. Finally (provided the OR can be expressed), although the positive hits predicted by ODORactor can be validated, the frequency of false negative predictions by ODORactor cannot be readily discerned. As more ORs outside the initial ODORactor build are deorphaned and compared with ODORactor predictions before and after publication, this will clarify the capabilities and constraints of this intriguing new tool.

Choosing ligands for deorphanization

Wide-scale screens with various rationale. Once a given OR has been successfully expressed, there remains the question of how to look for that first activating ligand in vast chemical space. One possibility involves a “shotgun” approach where many odorants are simply tried either alone or in blends without clearly disclosed rationale. Testing blends followed by fractionation would seem the obvious choice to increase the number of odorants screened. But there can be concerns about solvent levels in multiple-component blends. Furthermore, this technique can backfire should an antagonist be present in the same fraction as an agonist. Given that many of the currently published antagonists are structurally similar to agonists of the same receptor (Araneda et al., 2000; Oka et al., 2004a,b; Sanz et al., 2005; Shirokova et al., 2005; Jacquier et al., 2006; Abaffy et al., 2007; Peterlin et al., 2008), blends that contain related odorants may be at particular risk of generating false negatives in functional studies. However, interactions can still occur even among unexpected pairings. This was highlighted during the use of the Henkel100 blend to deorphan hOR17-4 (Spehr et al., 2003). The only agonist to emerge from the series of mixes was cyclamal.

During the follow-up medicinal chemistry exploration, however, the closely related bourgeonal was found to be active even though the mix containing it had been inactive. By fractionating that mix, it was determined that the structurally unrelated undecanal was the culprit antagonist. Thus, although precautions can be taken to divide seemingly similar components into different mixes, the encoding of odorants by ORs is still not sufficiently understood to assure that mixes will not exhibit interaction effects.

One guide to shotgun-type approaches is the data emerging from calcium imaging of OSNs from dissociated olfactory epithelia of wild-type mice or rats. The identities of the ORs are not known in this method, but because the full OR repertoire is surveyed, this approach can provide useful population-level information of the relative activity elicited by a given odorant. Although ORs are expressed at different frequencies (Khan et al., 2011), such that an odorant that activates many OSNs could conceivably be targeting just one or a few highly expressed ORs, constructing an odorant panel whose components each activate many OSNs with minimal overlap in their coverage remains a reasonable strategy for first-pass, general deorphanization of an OR. Although surveys using blends have shed light on OR repertoire coverage (Nara et al., 2011), more commonly single odorants are used. Surveys with single odorants have highlighted important trends. For example, flexible odorants appear to activate more OSNs than do highly constrained ones (Peterlin et al., 2008), flexible odorants with mid-length chains (between seven to nine carbon backbone) recruit more OSNs than do much shorter or longer ones (Sato et al., 1994; Kaluza and Breer, 2000; Araneda et al., 2004), and acids seem to be poor recruiters relative to alcohols or aldehydes of the same length (Araneda et al., 2004). Attentiveness to such trends during panel design may help maximize the potential for hits.

A second form of guidance can be found in the computational analysis of biologically relevant chemical space. This is the subset of all chemicals, such as those that can be found in larger databases such as PubChem or the Available Chemicals Directory, filtered based on general parameters that typify odorants. Chemical space is the multidimensional arrangement of these putative or known odorants based on their physicochemical descriptors and biological functional similarity. It is organized so that the displacement of two odorants relative to each other is related to the probability of their being co-detected by ORs (closer odorants being more likely to be co-detected). Determination of an appropriately weighted chemical space has helped generate highly accurate predictions of the receptive fields for the mosquito *Anopheles gambia* (Wang et al., 2010) and the fruit fly *Drosophila melanogaster* (Boyle et al., 2013). Currently, there are three models for the chemical space as
weighted using data from mammals (Haddad et al., 2008; Saito et al., 2009; Mainland et al., 2014). To help navigate and use one model of this space, Haddad et al. (2008) provided lists of common odorants that should comprise broadly dispersed initial screening panels. “Broad” here refers to the diameter of the hypersphere that encloses the odorants in multidimensional space. Models of chemical space are being used retroactively to assess the breadth of coverage of assay panels and the breadth of tuning of OR receptive fields (Saito et al., 2009; Li et al., 2012). However, panels composed by this method have yet to be deployed for initial deorphanization attempts.

Targeted screens that take the receptor into account. The above approaches to identifying the first ligand emphasize chemical diversity, focusing solely on the odorants. In contrast, the paralogue/orthologue approach uses information about the receptor to guide selection of a set of odorants to test. Paralogues within a species often share elements of their receptive fields, such as seen with M7l and M72 (96% protein identity) (Zhang et al., 2012), and MOR29A and MOR29B (95% protein homology) (Tsuboi et al., 2011). But when using the paralogue approach to testing compounds that are successful ligands of the “reference” OR against the orphan OR, there is the chance that the reference OR might be narrowly tuned. The three members of the large MOR256 family provide a telling, cautionary example. Whereas MOR256-17 responds to a variety of ligands, MOR256-8 and MOR256-22 can detect only a restricted subset of the MOR256-17 receptive field. Furthermore, the receptive fields of MOR256-8 and MOR256-22 show very sparse overlap with each other (Li et al., 2012). Thus, in this case, which OR is chosen as the reference would greatly affect success in deorphaning even related receptors.

As an alternative to comparing paralogues, looking across species to a deorphaned orthologue is often more productive. Some orthologues with very similar receptive fields include mouse and rat OR-I7 (Bozza et al., 2002), and mouse Olfr43 and human OR1Al and OR1A2 (Schmiedeberg et al., 2007). In a far-ranging survey of 17 rat ORs and their mouse orthologues and 18 chimpanzee ORs and their rhesus macaque orthologues, Adipietro et al. (2012) concluded that orthologues are nearly 2.5 times more likely to share a ligand in common than are paralogues. They also demonstrated that orthologues often share an extended ligand profile, albeit with differences in odorant EC₅₀s. These findings should provide encouragement for those seeking to extend findings from mouse ORs to human ORs.

Although robust, the orthologue approach is limited to a handful of ORs because the OR to be deorphaned must be a specific match. This is opposed to the paralogue approach that requires only family affiliation and thus has greater widespread applicability for deorphaning.

But why might comparing paralogues not be as good a predictor in terms of similarity of receptive fields as when orthologues are compared? One contributing factor may be that the current classification of ORs into families is hierarchical in nature and based on full-length sequence. A reorganization based on short motifs with predicted functional relevance may improve performance. The foundations for alternative organizations already exist. Two studies independently identified short motifs distributed throughout the OR (Liu et al., 2003; Zhang et al., 2007). Man et al. (2004) compared select mouse and human orthologues and paralogues to identify a set of 22–amino acid residues, predicted to have special impact on ligand recognition. Saito et al. (2009) identified 15 residues for which 16 optimized physicochemical property descriptors were very good at predicting the response of 62 ORs to a panel of 63 odorants. Any of these motif lists can be used to break free of hierarchical constraints and plot ORs in multidimensional space. The resultant new relationships between ORs may better incorporate known or inferred physiology.

Beyond deorphanization: Integration with the combinatorial code

When a ligand has been found for the expressed OR of interest, a common next step is to use a medicinal chemistry approach to look for additional ligands to help characterize the structure–activity relationship for that OR. For example, a battery of compounds with subtle changes may be used to examine the discriminatory capacity of the receptor. This approach has been used successfully for many receptors, including mOR-EG (Katada et al., 2005) and M72 (Zhang et al., 2012). Both receptors were tested with panels of small aromatics. Because only compounds with scaffolds similar to the initial target are tested, there may be a tendency to consider that a receptor is “narrowly tuned.” But expanding to more diversified test panels can sometimes yield a marked shift in perspective. For example, Baud et al. (2011) tested an assortment of odorants on mOR-EG, and Soucy et al. (2009) tested a collection of compounds on M72. In neither case did these ORs appear to remain so “narrowly tuned”; mOR-EG also responded to aliphatic cyclic ketones and M72 to aliphatic tiglate esters. It can be daunting to probe OR recognition of structurally diverse odorants because such a search is likely to return many inactive compounds. But this type of expanded testing provides valuable information to challenge pharmacophore models. This, in turn, can better clarify what chemical features the OR truly abstracts.

Another strategy to expand beyond an initial lead compound is to challenge the OR with odorants that the larger OR repertoire reports as “biologically similar.” This strategy can apply to subsections of the odorants as well, as in the case of substituting a cyclohexene

536 Odorant receptor deorphanization
ring in place of a benzene ring. Such odorants may appear structurally dissimilar, but as the exchanges proposed are empirically based, the approach may be particularly productive. The information on “biological similarity” can be gained from calcium imaging of OSNs from dissociated olfactory epithelia of wild-type mice or rats. Even though the identity of the ORs is not known, the large-scale patterns across the OR repertoire show how frequently two odorants are recognized by the same OSN. This information can be used to generate probability tables. Such tables may aid deorphaning efforts by suggesting what other odorants are likely activators. Knowledge of expected co-recognition rates can also flag when an OR is making a challenging discrimination, thus contributing in a special way to the combinatorial code.

Future perspective

A relatively new approach to deorphanizing human ORs involves genomics, where the presence of naturally occurring genetic variations (single-nucleotide polymorphisms) in individual OR genes are correlated with the relative activities of the ORs in vitro and the measured odor perception in humans (Keller et al., 2007; Menashe et al., 2007; Lunde et al., 2012; McRae et al., 2012; Jaeger et al., 2013; Mainland et al., 2014). This approach has been successful in correlating the activity of individual ORs with measureable perceptual characteristics like odor intensity and pleasantness ratings. These emergent genomics-based OR percept studies may represent a powerful new strategy for OR deorphanization because they can provide a direct link between OR identity and perception. However, inherent to this

---

**In vitro Assays**

Hana 3A  
HEK 293T  
HeLa/Olf  

![Diagram](https://via.placeholder.com/500x300)

---

**Ex Vivo Assays**

Electrophysiology  
Calcium Imaging  

![Diagram](https://via.placeholder.com/500x300)

---

**In Vivo Imaging**

Calcium  
Synapto-pHluorin  
Intrinsic signals  

![Diagram](https://via.placeholder.com/500x300)

---

Figure 3. Schematic overview of a comprehensive OR assay platform and the approximate single assay compound throughput. Given the inherent complexity of the combinatorial receptor code and the remaining challenges of current in vitro OR assays, a comprehensive OR assay platform would aim to balance screening throughput and organism relevance by incorporating in vitro, ex vivo, and in vivo assays. A robust in vitro platform would enable the initial screening of thousands of odorant compounds. Agonists or antagonists identified from in vitro systems could then be screened using ex vivo OSN assays (medium throughout and relevance) or in vivo imaging assays (low throughput and high relevance). A salient benefit of in vivo over ex vivo or in vitro assays is that odorants are delivered in the vapor phase to the fully intact peripheral olfactory system (olfactory epithelium and bulb). The main benefit of ex vivo over in vitro assays is the use of OSNs as an expression system for native (i.e., untagged) receptors. OB, olfactory bulb; OE, olfactory epithelium; OSNs, olfactory sensory neurons; MU, olfactory mucus.

and all other OR deorphanization strategies is the absolute requirement for a robust in vitro assay to confirm putative ligand–receptor pairings and to probe odorant structure–activity relationships.

The need for a comprehensive OR assay platform that balances throughput and response accuracy is thus critical for understanding the olfactory combinatorial code. A more robust in vitro platform (Fig. 3) could enable the identification and expression of all ORs contributing to the combinatorial activation code for an odorant or odor mixture (Malnic et al., 1999). The need for such a platform is especially pronounced given that ligands for ~90% of human ORs remain unknown. Various techniques have been adopted including using native OSNs and an array of heterologous expression systems. But each of these systems has inherent strengths and limitations, and none have all the desired criteria of a suitable assay platform. For now, the optimal screening platform should encompass an initial high throughput chemical screen using in vitro heterologous systems followed by a low throughput in vivo assay for validation. Perhaps future implementation of interdisciplinary strategies that are routinely used by the pharmaceutical industry for the discovery of GPCR agonists and modulators (Ingelse et al., 2007; Eggert, 2013) could contribute to more advanced and reliable heterologous OR systems.

Aside from the issue of proper trafficking, another reason for the relative paucity of deorphaned ORs may be that compounds used for screening have limited structural diversity and may not fully span odorant chemical space. This could be a matter of availability as commercial sources possess only limited collections of standard fragrance compounds. Moreover, commercial sources may not provide a sufficient range of subtly different odorants to serve as medicinal chemistry probes, particularly when the lead odorant has a rare or complex chemical scaffold. One relatively untapped source of both diverse and nuanced chemical libraries is the major flavor and fragrance houses. With more than 100 years of experience in chemical synthesis and purification, flavor and fragrance houses possess extraordinary collections of organoleptically defined volatile compounds that could greatly assist in both decoding the combinatorial code at the receptor level and help generate hypotheses of how that code correlates with percept. Olfactory research collaborations between academic and industry partners can be fruitful (e.g., Grabenhorst et al., 2007) and should be strengthened further to provide greater access to diverse and well-characterized odorant libraries. Ultimately, such partnerships may be an important step forward in the quest to unravel the complex and often puzzling relationships between chemical structure, receptor activity, and human perception.

The authors would like to thank Ayome Abibi, Huey-Ling Kao, and Patrick Pfister (Firmenich, Inc.) for helpful discussions.


Z. Peterlin and M.E. Rogers are employees of Firmenich, Inc. S. Firestein has ongoing experiments in the field of olfaction with Firmenich, Inc., and consults for Firmenich, Inc., as well. He has no patents with Firmenich, Inc., or vice versa, and it does not alter the authors’ adherence to all of the Journal’s policies on sharing data and materials. The authors have no additional financial interests.

Elizabeth M. Adler served as editor.

Submitted: 19 December 2013  
Accepted: 20 March 2014

# REFERENCES

Abaffy, T., H. Matsunami, and C.W. Luetje. 2006. Functional analysis of a mammalian odorant receptor subfamily. *J. Neurochem.* 97:1506–1518. http://dx.doi.org/10.1111/j.1471-4159.2006.03859.x

Abaffy, T., A. Malhotra, and C.W. Luetje. 2007. The molecular basis for ligand specificity in a mouse olfactory receptor: A network of functionally important residues. *J. Biol. Chem.* 282:1216–1224. http://dx.doi.org/10.1074/jbc.M609355200

Adipietro, K.A., J.D. Mainland, and H. Matsunami. 2012. Functional evolution of mammalian odorant receptors. *PLoS Genet.* 8:e1002821. http://dx.doi.org/10.1371/journal.pgen.1002821

Anselmi, C., A. Buonocore, M. Centini, R.M. Facino, and H. Hatt. 2011. The human olfactory receptor 17-40: Requisites for fitting into the binding pocket. *Comput. Biol. Chem.* 35:159–168. http://dx.doi.org/10.1016/j.compbiolchem.2011.04.011

Araneda, R.C., A.D. Kini, and S. Firestein. 2000. The molecular receptive range of an odorant receptor. *Nat. Neurosci.* 3:1248–1255. http://dx.doi.org/10.1038/81774

Araneda, R.C., Z. Peterlin, X. Zhang, A. Chesler, and S. Firestein. 2004. A pharmacological profile of the aldehyde receptor repertoire in rat olfactory epithelium. *J. Physiol.* 555:743–756. http://dx.doi.org/10.1113/jphysiol.2003.058040

Baud, O., S. Etter, M. Spreafico, L. Bordoli, T. Schwede, H. Vogel, and H. Pick. 2011. The mouse eugenol odorant receptor: Structural and functional plasticity of a broadly tuned odorant binding pocket. *Biochemistry.* 50:843–853. http://dx.doi.org/10.1021/bi1017396

Bavan, S., B. Sherman, C.W. Luetje, and T. Abaffy. 2014. Discovery of novel ligands for mouse olfactory receptor MOR42-3 using an in silico screening approach and in vitro validation. *PLoS One.* 9:e92064. http://dx.doi.org/10.1371/journal.pone.0092064

Belluscio, L., C. Lodovichi, P. Feinstein, P. Mombaerts, and L.C. Katz. 2002. Odorant receptors instruct functional circuitry in the mouse olfactory bulb. *Nature.* 419:296–300. http://dx.doi.org/10.1038/nature01001

Bieri, S., K. Monastyrskaja, and B. Schilling. 2004. Olfactory receptor neuron profiling using sandalwood odorants. *Chem. Senses.* 29:483–487. http://dx.doi.org/10.1093/chemse/bjh050

Bockaert, J., and J.P. Pin. 1999. Molecular tinkering of G protein-coupled receptors: an evolutionary success. *EMBO J.* 18:1723–1729. http://dx.doi.org/10.1093/emboj/18.7.1723

Boyle, S.M., S. McInally, and A. Ray. 2013. Expanding the olfactory code by in silico decoding of odor-receptor chemical space. *Elife.* 2:e01120. http://dx.doi.org/10.7554/eLife.01120

Bozza, T., P. Feinstein, C. Zheng, and P. Mombaerts. 2002. Odorant receptor expression defines functional units in the mouse olfactory system. *J. Neurosci.* 22:3033–3043.

Bozza, T., J.P. McGann, P. Mombaerts, and M. Wachowiak. 2004. In vivo imaging of neuronal activity by targeted expression of a genetically encoded probe in the mouse. *Neuron.* 42:9–21. http://dx.doi.org/10.1016/S0896-6273(04)00144-8

Bunzow, J.R., H.H. Van Tol, D.K. Grandy, P. Albert, J. Salon, M. Christie, C.A. Machida, K.A. Neve, and O. Civelli. 1988. Cloning and expression of a rat D2 dopamine receptor cDNA. *Nature*. 336:783–787. http://dx.doi.org/10.1038/336783a0

Bush, C.F., and R.A. Hall. 2008. Olfactory receptor trafficking to the plasma membrane. *Cell. Mol. Life Sci.* 65:2289–2295. http://dx.doi.org/10.1007/s00018-008-8028-z

Charlier, L., J. Topin, C. Ronin, S.K. Kim, W.A. Goddard III, R. Efremov, and J. Golebiowski. 2012. How broadly tuned olfactory receptors equally recognize their agonists. Human OR1G1 as a test case. *Cell. Mol. Life Sci.* 69:4205–4213. http://dx.doi.org/10.1007/s00018-012-1116-0

Civelli, O., R.K. Reinscheid, Y. Zhang, Z. Wang, R. Fredriksson, and H.B. Schiöth. 2013. G protein-coupled receptor deorphanizations. *Annu. Rev. Pharmacol. Toxicol.* 53:127–146. http://dx.doi.org/10.1146/annurev-pharmtox-010611-134548

Corcelli, A., S. Lobasso, P. Lopalco, M. Dibattista, R. Araneda, Z. Peterlin, and S. Firestein. 2010. Detection of explosives by olfactory sensory neurons. *J. Hazard. Mater.* 175:1096–1100. http://dx.doi.org/10.1016/j.jhazmat.2009.10.054

Egbert, U.S. 2013. The why and how of phenotypic small-molecule screens. *Nat. Chem. Biol.* 9:206–209. http://dx.doi.org/10.1038/nchembio.1206

Ellgaard, L., and A. Helenius. 2003. Quality control in the endoplasmic reticulum. *Nat. Rev. Mol. Cell Biol.* 4:181–191. http://dx.doi.org/10.1038/nrm1052

Emkey, R., and N.B. Rankl. 2009. Screening G protein-coupled receptors: Measurement of intracellular calcium using the fluorometric imaging plate reader. *Methods Mol. Biol.* 565:145–158. http://dx.doi.org/10.1007/978-1-60327-258-2_7

Fargin, A., J.R. Raymond, M.J. Lohse, B.K. Kobilka, M.G. Caron, and R.J. Lefkowitz. 1988. The genomic clone G-21 which resembles a β-adrenergic receptor sequence encodes the 5-HT<sub>IA</sub> receptor. *Nature*. 335:358–360. http://dx.doi.org/10.1038/335358a0

Feinstein, P., T. Bozza, I. Rodriguez, A. Vassalli, and P. Mombaerts. 2004. Axon guidance of mouse olfactory sensory neurons by odorant receptors and the β2 adrenergic receptor. *Cell*. 117:833–846. http://dx.doi.org/10.1016/j.cell.2004.05.013

Firestein, S. 2001. How the olfactory system makes sense of scents. *Nature*. 413:211–218. http://dx.doi.org/10.1038/35093026

Gelis, L., S. Wolf, H. Hatt, E.M. Neuhaus, and K. Gerwert. 2012. Prediction of a ligand-binding niche within a human olfactory receptor by combining site-directed mutagenesis with dynamic homology modeling. *Angew. Chem. Int. Ed.* 51:1274–1278. http://dx.doi.org/10.1002/anie.201103980

Gether, U. 2000. Uncovering molecular mechanisms involved in activation of G protein-coupled receptors. *Endocr. Rev.* 21:90–113. http://dx.doi.org/10.1210/edrv.21.1.0390

Ghosh, S., S.D. Larson, H. Hefzi, Z. Marnoy, T. Cutforth, K. Dokka, and K.K. Baldwin. 2011. Sensory maps in the olfactory cortex defined by long-range viral tracing of single neurons. *Nature*. 472:217–220. http://dx.doi.org/10.1038/nature09945

Grabenhorst, F., E.T. Rolls, C. Margot, M.A. da Silva, and M.I. Velazco. 2007. How pleasant and unpleasant stimuli combine in different brain regions: Odor mixtures. *J. Neurosci.* 27:13532–13540. http://dx.doi.org/10.1523/JNEUROSCI.3337-07.2007

Grosmaitre, X., A. Vassalli, P. Mombaerts, G.M. Shepherd, and M. Ma. 2006. Odorant responses of olfactory sensory neurons expressing the odorant receptor MOR23: A patch clamp analysis in gene-targeted mice. *Proc. Natl. Acad. Sci. USA*. 103:1970–1975. http://dx.doi.org/10.1073/pnas.0508491103

Haberly, L.B., and J.L. Price. 1977. The axonal projection patterns of the mitral and tufted cells of the olfactory bulb in the rat. *Brain Res.* 129:152–157. http://dx.doi.org/10.1016/0006-8993(77)90978-7

Haddad, R., R. Khan, Y.K. Takahashi, K. Mori, D. Harel, and N. Sobel. 2008. A metric for odorant comparison. *Nat. Methods*. 5:425–429. http://dx.doi.org/10.1038/nmeth.1197

Hall, S.E., W.B. Floriano, N. Vaidehi, and W.A. Goddard III. 2004. Predicted 3-D structures for mouse I7 and rat I7 olfactory receptors and comparison of predicted odor recognition profiles with experiment. *Chem. Senses*. 29:595–616. http://dx.doi.org/10.1093/chemse/bjh063

Hamana, H., L. Shou-xin, L. Breuils, J. Hirono, and T. Sato. 2010. Heterologous functional expression system for odorant receptors. *J. Neurosci. Methods*. 185:213–220. http://dx.doi.org/10.1016/j.jneumeth.2009.09.024

Hummel, P., N. Vaidehi, W.B. Floriano, S.E. Hall, and W.A. Goddard III. 2005. Test of the Binding Threshold Hypothesis for olfactory receptors: Explanation of the differential binding of ketones to the mouse and human orthologs of olfactory receptor 912-93. *Protein Sci.* 14:703–710. http://dx.doi.org/10.1110/ps.041119705

Igarashi, K.M., and K. Mori. 2005. Spatial representation of hydrocarbon odorants in the ventrolateral zones of the rat olfactory bulb. *J. Neurophysiol.* 93:1007–1019. http://dx.doi.org/10.1152/jn.00873.2004

Inglese, J., R.L. Johnson, A. Simeonov, M. Xia, W. Zheng, C.P. Austin, and D.S. Auld. 2007. High-throughput screening assays for the identification of chemical probes. *Nat. Chem. Biol.* 3:466–479. http://dx.doi.org/10.1038/nchembio.2007.17

Jacquier, V., H. Pick, and H. Vogel. 2006. Characterization of an extended receptive ligand repertoire of the human olfactory receptor OR17-40 comprising structurally related compounds. *J. Neurochem.* 97:537–544. http://dx.doi.org/10.1111/j.1471-4159.2006.03771.x

Jaeger, S.R., J.F. McRae, C.M. Bava, M.K. Beresford, D. Hunter, Y. Jia, S.L. Chheang, D. Jin, M. Peng, J.C. Gamble, et al. 2013. A Mendelian trait for olfactory sensitivity affects odor experience and food selection. *Curr. Biol.* 23:1601–1605. http://dx.doi.org/10.1016/j.cub.2013.07.030

Kajiya, K., K. Inaki, M. Tanaka, T. Haga, H. Kataoka, and K. Touhara. 2001. Molecular bases of odor discrimination: Reconstitution of olfactory receptors that recognize overlapping sets of odorants. *J. Neurosci.* 21:6018–6025.

Kaluza, J.F., and H. Breer. 2000. Responsiveness of olfactory neurons to distinct aliphatic aldehydes. *J. Exp. Biol.* 203:927–933.

Katada, S., T. Nakagawa, H. Kataoka, and K. Touhara. 2003. Odorant response assays for a heterologously expressed olfactory receptor. *Biochem. Biophys. Res. Commun.* 305:964–969. http://dx.doi.org/10.1016/S0006-291X(03)00863-5

Katada, S., T. Hirokawa, Y. Oka, M. Suwa, and K. Touhara. 2005. Structural basis for a broad but selective ligand spectrum of a mouse olfactory receptor: Mapping the odorant-binding site. *J. Neurosci.* 25:1806–1815. http://dx.doi.org/10.1523/JNEUROSCI.4723-04.2005

Kato, A., S. Katada, and K. Touhara. 2008. Amino acids involved in conformational dynamics and G protein coupling of an odorant receptor: targeting gain-of-function mutation. *J. Neurochem.* 107:1261–1270. http://dx.doi.org/10.1111/j.1471-4159.2008.05693.x

Keller, A., H. Zhuang, Q. Chi, L.B. Vosshall, and H. Matsunami. 2007. Genetic variation in a human odorant receptor alters odour perception. *Nature*. 449:468–472. http://dx.doi.org/10.1038/nature06162

Kerr, D.S., L.E. Von Dannecker, M. Davalos, J.S. Michaloski, and B. Malnic. 2008. Ric-8B interacts with Gaolf and Gγ13 and colocalizes with Gaolf, Gβ1 and Gγ13 in the cilia of olfactory sensory neurons. *Mol. Cell. Neurosci.* 38:341–348. http://dx.doi.org/10.1016/j.mcn.2008.03.006

Khafizov, K., C. Anselmi, A. Menini, and P. Carloni. 2007. Ligand specificity of odorant receptors. *J. Mol. Model.* 13:401–409. http://dx.doi.org/10.1007/s00894-006-0160-9

Khan, M., E. Vaes, and P. Mombaerts. 2011. Regulation of the probability of mouse odorant receptor gene choice. *Cell.* 147:907–921. http://dx.doi.org/10.1016/j.cell.2011.09.049

Kim, P.S., and P. Arvan. 1998. Endocrinopathies in the family of endoplasmic reticulum (ER) storage diseases: Disorders of protein trafficking and the role of ER molecular chaperones. *Endocr. Rev.* 19:173–202.

Krautwurst, D. 2008. Human olfactory receptor families and their odorants. *Chem. Biodivers.* 5:842–852. http://dx.doi.org/10.1002/cbdv.200890099

Krautwurst, D., K.W. Yau, and R.R. Reed. 1998. Identification of ligands for olfactory receptors by functional expression of a receptorlibrary. *Cell.* 95:917–926. http://dx.doi.org/10.1016/S0092-8674(00)81716-X

Kurland, M.D., M.B. Newcomer, Z. Peterlin, K. Ryan, S. Firestein, and V.S. Batista. 2010. Discrimination of saturated aldehydes by the rat I7 olfactory receptor. *Biochemistry.* 49:6302–6304. http://dx.doi.org/10.1021/bi100976w

Lai, P.C., M.S. Singer, and C.J. Crasto. 2005. Structural activation pathways from dynamic olfactory receptor-odorant interactions. *Chem. Senses.* 30:781–792. http://dx.doi.org/10.1093/chemse/bji070

Lai, P.C., G. Bahl, M. Gremigni, V. Matarazzo, O. Clot-Faybesse, C. Ronin, and C.J. Crasto. 2008. An olfactory receptor pseudogene whose function emerged in humans: a case study in the evolution of structure-function in GPCRs. *J. Struct. Funct. Genomics.* 9:29–40. http://dx.doi.org/10.1007/s10969-008-9043-x

Lai, P.C., B. Guida, J. Shi, and C.J. Crasto. 2014. Preferential binding of an odor within olfactory receptors: A precursor to receptor activation. *Chem. Senses.* 39:107–123. http://dx.doi.org/10.1093/chemse/bjt060

Launay, G., S. Téletchéa, F. Wade, E. Pajot-Augy, J.F. Gibrat, and G. Sanz. 2012. Automatic modeling of mammalian olfactory receptors and docking of odorants. *Protein Eng. Des. Sel.* 25:377–386. http://dx.doi.org/10.1093/protein/gzs037

Lettvin, J.Y., H.R. Maturana, W.S. McCulloch, and W.H. Pitts. 1959. What the frog’s eye tells the frog’s brain. *Proceedings of the IRE.* 47:1940–1951. http://dx.doi.org/10.1109/JRPROC.1959.287207

Levasseur, G., M.A. Persuy, D. Grebert, J.J. Remy, R. Salesse, and E. Pajot-Augy. 2003. Ligand-specific dose-response of heterologously expressed olfactory receptors. *Eur. J. Biochem.* 270:2905–2912. http://dx.doi.org/10.1046/j.1432-1033.2003.03672.x

Li, J., R. Haddad, S. Chen, V. Santos, and C.W. Luetje. 2012. A broadly tuned mouse odorant receptor that detects nitrotoluenes. *J. Neurochem.* 121:881–890. http://dx.doi.org/10.1111/j.1471-4159.2012.07740.x

Li, Y.R., and H. Matsunami. 2011. Activation state of the M3 muscarinic acetylcholine receptor modulates mammalian odorant receptor signaling. *Sci. Signal.* 4:ral1.

Lin, D.Y., S.Z. Zhang, E. Block, and L.C. Katz. 2005. Encoding social signals in the mouse main olfactory bulb. *Nature.* 434:470–477. http://dx.doi.org/10.1038/nature03414

Liu, A.H., X. Zhang, G.A. Stolovitzky, A. Califano, and S.J. Firestein. 2003. Motif-based construction of a functional map for mammalian olfactory receptors. *Genomics.* 81:443–456. http://dx.doi.org/10.1016/S0888-7543(03)00022-3

Liu, X., X. Su, F. Wang, Z. Huang, Q. Wang, Z. Li, R. Zhang, L. Wu, Y. Pan, Y. Chen, et al. 2011. ODORactor: a web server for deciphering olfactory coding. *Bioinformatics.* 27:2302–2303. http://dx.doi.org/10.1093/bioinformatics/btr385

Lu, M., F. Echeverri, and B.D. Moyer. 2003. Endoplasmic reticulum retention, degradation, and aggregation of olfactory G-protein coupled receptors. *Traffic.* 4:416–433. http://dx.doi.org/10.1034/j.1600-0854.2003.00097.x

Lu, M., L. Staszewski, F. Echeverri, H. Xu, and B.D. Moyer. 2004. Endoplasmic reticulum degradation impedes olfactory G-protein coupled receptor functional expression. *BMC Cell Biol.* 5:34. http://dx.doi.org/10.1186/1471-2121-5-34

Lunde, K., B. Egelandsdal, E. Skuterud, J.D. Mainland, T. Lea, M. Hersleth, and H. Matsunami. 2012. Genetic variation of an odorant receptor OR7D4 and sensory perception of cooked meat containing androstenone. *PLoS ONE.* 7:e35259. http://dx.doi.org/10.1371/journal.pone.0035259

Mainland, J.D., A. Keller, Y.R. Li, T. Zhou, C. Trimmer, L.L. Snyder, A.H. Moberly, K.A. Adipietro, W.L. Liu, H. Zhuang, et al. 2014. The missense of smell: functional variability in the human odorant receptor repertoire. *Nat. Neurosci.* 17:114–120. http://dx.doi.org/10.1038/nn.3598

Malnic, B. 2007. Searching for the ligands of odorant receptors. *Mol. Neurobiol.* 35:175–181. http://dx.doi.org/10.1007/s12035-007-0013-2

Malnic, B., J. Hirono, T. Sato, and L.B. Buck. 1999. Combinatorial receptor codes for odors. *Cell.* 96:713–723. http://dx.doi.org/10.1016/S0092-8674(00)80581-4

Man, O., Y. Gilad, and D. Lancet. 2004. Prediction of the odorant binding site of olfactory receptor proteins by human-mouse comparisons. *Protein Sci.* 13:240–254. http://dx.doi.org/10.1110/ps.03296404

Matarazzo, V., O. Clot-Faybesse, B. Marcet, G. Guiraudeau-Capraz, B. Atanasova, G. Devauchelle, M. Cerutti, P. Etievant, and C. Ronin. 2005. Functional characterization of two human olfactory receptors expressed in the baculovirus Sf9 insect cell system. *Chem. Senses.* 30:195–207. http://dx.doi.org/10.1093/chemse/bji015

McClintock, T.S., and N. Sammeta. 2003. Trafficking prerogatives of olfactory receptors. *Neuroreport.* 14:1547–1552. http://dx.doi.org/10.1097/00001756-200308260-00001

McRae, J.F., J.D. Mainland, S.R. Jaeger, K.A. Adipietro, H. Matsunami, and R.D. Newcomb. 2012. Genetic variation in the odorant receptor OR2J3 is associated with the ability to detect the “grassy” smelling odor, cis-3-hexen-1-ol. *Chem. Senses.* 37:585–593. http://dx.doi.org/10.1093/chemse/bjs049

Menashe, I., T. Abaffy, Y. Hasin, S. Goshen, V. Yahalom, C.W. Luetje, and D. Lancet. 2007. Genetic elucidation of human hyperosmia to isovaleric acid. *PLoS Biol.* 5:e284. http://dx.doi.org/10.1371/journal.pbio.0050284

Minic, J., M.A. Persuy, E. Godel, J. Aioun, I. Connerton, R. Salesse, and E. Pajot-Augy. 2005. Functional expression of olfactory receptors in yeast and development of a bioassay for odorant screening. *FEBS J.* 272:524–537. http://dx.doi.org/10.1111/j.1742-4658.2004.04494.x

Mitsui, K., T. Sakihama, K. Takahashi, K. Masuda, R. Fukuda, H. Hamana, T. Sato, and T. Hamakubo. 2012. Functional reconstitution of olfactory receptor complex on baculovirus. *Chem. Senses.* 37:837–847. http://dx.doi.org/10.1093/chemse/bjs067

Mombaerts, P. 2004. Genes and ligands for odorant, vomeronasal and taste receptors. *Nat. Rev. Neurosci.* 5:263–278. http://dx.doi.org/10.1038/nrn1365

Mombaerts, P., F. Wang, C. Dulac, S.K. Chao, A. Nemes, M. Mendelsohn, J. Edmondson, and R. Axel. 1996. Visualizing an olfactory sensory map. *Cell.* 87:675–686. http://dx.doi.org/10.1016/S0092-8674(00)81387-2

Nagashima, A., and K. Touhara. 2010. Enzymatic conversion of odorants in nasal mucus affects olfactory glomerular activation patterns and odor perception. *J. Neurosci.* 30:16391–16398. http://dx.doi.org/10.1523/JNEUROSCI.2527-10.2010

Nara, K., L.R. Saraiva, X. Ye, and L.B. Buck. 2011. A large-scale analysis of odor coding in the olfactory epithelium. *J. Neurosci.* 31:9179–9191. http://dx.doi.org/10.1523/JNEUROSCI.1282-11.2011

Oka, Y., A. Nakamura, H. Watanabe, and K. Touhara. 2004a. An odorant derivative as an antagonist for an olfactory receptor. *Chem. Senses*. 29:815–822. http://dx.doi.org/10.1093/chemse/bjh247

Oka, Y., M. Omura, H. Kataoka, and K. Touhara. 2004b. Olfactory receptor antagonism between odorants. *EMBO J.* 23:120–126. http://dx.doi.org/10.1038/sj.emboj.7600032

Oka, Y., S. Katada, M. Omura, M. Suwa, Y. Yoshihara, and K. Touhara. 2006. Odorant receptor map in the mouse olfactory bulb: In vivo sensitivity and specificity of receptor-defined glomeruli. *Neuron*. 52:857–869. http://dx.doi.org/10.1016/j.neuron.2006.10.019

Palczewski, K., T. Kumasaka, T. Hori, C.A. Behnke, H. Motoshima, B.A. Fox, I. Le Trong, D.C. Teller, T. Okada, R.E. Stenkamp, et al. 2000. Crystal structure of rhodopsin: A G protein-coupled receptor. *Science*. 289:739–745. http://dx.doi.org/10.1126/science.289.5480.739

Peterlin, Z., Y. Li, G. Sun, R. Shah, S. Firestein, and K. Ryan. 2008. The importance of odorant conformation to the binding and activation of a representative olfactory receptor. *Chem. Biol.* 15:1317–1327. http://dx.doi.org/10.1016/j.chembiol.2008.10.014

Potter, S.M., C. Zheng, D.S. Koos, P. Feinstein, S.E. Fraser, and P. Mombaerts. 2001. Structure and emergence of specific olfactory glomeruli in the mouse. *J. Neurosci.* 21:9713–9723.

Reisert, J., and D. Restrepo. 2009. Molecular tuning of odorant receptors and its implication for odor signal processing. *Chem. Senses*. 34:535–545. http://dx.doi.org/10.1093/chemse/bjp028

Repicky, S.E., and C.W. Luetje. 2009. Molecular receptive range variation among mouse odorant receptors for aliphatic carboxylic acids. *J. Neurochem.* 109:193–202. http://dx.doi.org/10.1111/j.1471-4159.2009.05925.x

Ressler, K.J., S.L. Sullivan, and L.B. Buck. 1994. Information coding in the olfactory system: Evidence for a stereotyped and highly organized epitope map in the olfactory bulb. *Cell*. 79:1245–1255. http://dx.doi.org/10.1016/0092-8674(94)90015-9

Rubin, B.D., and L.C. Katz. 1999. Optical imaging of odorant representations in the mammalian olfactory bulb. *Neuron*. 23:499–511. http://dx.doi.org/10.1016/S0896-6273(00)80803-X

Saito, H., M. Kubota, R.W. Roberts, Q. Chi, and H. Matsunami. 2004. RTP family members induce functional expression of mammalian odorant receptors. *Cell*. 119:679–691. http://dx.doi.org/10.1016/j.cell.2004.11.021

Saito, H., Q. Chi, H. Zhuang, H. Matsunami, and J.D. Mainland. 2009. Odor coding by a mammalian receptor repertoire. *Sci. Signal.* 2:ra9. http://dx.doi.org/10.1126/scisignal.2000016

Sanz, G., C. Schlegel, J.C. Pernollet, and L. Briand. 2005. Comparison of odorant specificity of two human olfactory receptors from different phylogenetic classes and evidence for antagonism. *Chem. Senses*. 30:69–80. http://dx.doi.org/10.1093/chemse/bji002

Sanz, G., T. Thomas-Danguin, H. Hamdani, C. Le Poupon, L. Briand, J.C. Pernollet, E. Guichard, and A. Tromelin. 2008. Relationships between molecular structure and perceived odor quality of ligands for a human olfactory receptor. *Chem. Senses*. 33:639–653. http://dx.doi.org/10.1093/chemse/bjn032

Sato, T., J. Hirono, M. Tonoike, and M. Takebayashi. 1994. Tuning specificities to aliphatic odorants in mouse olfactory receptor neurons and their local distribution. *J. Neurophysiol.* 72:2980–2989.

Schmiedeberg, K., E. Shirokova, H.P. Weber, B. Schilling, W. Meyerhof, and D. Krautwurst. 2007. Structural determinants of odorant recognition by the human olfactory receptors OR1A1 and OR1A2. *J. Struct. Biol.* 159:400–412. http://dx.doi.org/10.1016/j.jsb.2007.04.013

Shepard, B.D., N. Natarajan, R.J. Protzko, O.W. Acres, and J.L. Pluznick. 2013. A cleavable N-terminal signal peptide promotes widespread olfactory receptor surface expression in HEK293T cells. *PLoS ONE*. 8:e68758. http://dx.doi.org/10.1371/journal.pone.0068758

Shepherd, G.M. 2006. Smell images and the flavour system in the human brain. *Nature*. 444:316–321. http://dx.doi.org/10.1038/nature05405

Shirasu, M., K. Yoshikawa, Y. Takai, A. Nakashima, H. Takeuchi, H. Sakano, and K. Touhara. 2014. Olfactory receptor and neural pathway responsible for highly selective sensing of musk odors. *Neuron*. 81:165–178. http://dx.doi.org/10.1016/j.neuron.2013.10.021

Shirokova, E., K. Schmiedeberg, P. Bedner, H. Niessen, K. Willecke, J.D. Raguse, W. Meyerhof, and D. Krautwurst. 2005. Identification of specific ligands for orphan olfactory receptors. G protein-dependent agonism and antagonism of odorants. *J. Biol. Chem.* 280:11807–11815. http://dx.doi.org/10.1074/jbc.M411508200

Singer, M.S. 2000. Analysis of the molecular basis for octanal interactions in the expressed rat 17 olfactory receptor. *Chem. Senses*. 25:155–165. http://dx.doi.org/10.1093/chemse/25.2.155

Singer, M.S., and G.M. Shepherd. 1994. Molecular modeling of ligand-receptor interactions in the OR5 olfactory receptor. *Neuroreport*. 5:1297–1300. http://dx.doi.org/10.1097/00001756-199406020-00036

Sivertsen, B., N. Holliday, A.N. Madsen, and B. Holst. 2013. Functionally biased signaling properties of 7TM receptors—opportunities for drug development for the ghrelin receptor. *Br. J. Pharmacol.* 170:1349–1362. http://dx.doi.org/10.1111/bph.12361

Soucy, E.R., D.F. Albeanu, A.L. Fantana, V.N. Murthy, and M. Meister. 2009. Precision and diversity in an odor map on the olfactory bulb. *Nat. Neurosci.* 12:210–220. http://dx.doi.org/10.1038/nn.2262

Spehr, M., G. Gisselmann, A. Poplawski, J.A. Riffell, C.H. Wetzel, R.K. Zimmer, and H. Hatt. 2003. Identification of a testicular odorant receptor mediating human sperm chemotaxis. *Science*. 299:2054–2058. http://dx.doi.org/10.1126/science.1080376

Touhara, K. 2007. Deorphanizing vertebrate olfactory receptors: recent advances in odorant-response assays. *Neurochem. Int.* 51:132–139. http://dx.doi.org/10.1016/j.neuint.2007.05.020

Touhara, K., S. Sengoku, K. Inaki, A. Tsuboi, J. Hirono, T. Sato, H. Sakano, and T. Haga. 1999. Functional identification and reconstitution of an odorant receptor in single olfactory neurons. *Proc. Natl. Acad. Sci. USA*. 96:4040–4045. http://dx.doi.org/10.1073/pnas.96.7.4040

Tsuboi, A., T. Imai, H.K. Kato, H. Matsumoto, K.M. Igarashi, M. Suzuki, K. Mori, and H. Sakano. 2011. Two highly homologous mouse odorant receptors encoded by tandemly-linked MOR29A and MOR29B genes respond differently to phenyl ethers. *Eur. J. Neurosci.* 33:205–213. http://dx.doi.org/10.1111/j.1460-9568.2010.07495.x

Uchida, N., Y.K. Takahashi, M. Tanifuji, and K. Mori. 2000. Odor maps in the mammalian olfactory bulb: domain organization and odorant structural features. *Nat. Neurosci.* 3:1035–1043. http://dx.doi.org/10.1038/79857

van Anken, E., and I. Braakman. 2005. Versatility of the endoplasmic reticulum protein folding factory. *Crit. Rev. Biochem. Mol. Biol.* 40:191–228. http://dx.doi.org/10.1080/10409230591008161

Vassar, R., S.K. Chao, R. Sitcheran, J.M. Nuñez, L.B. Vosshall, and R. Axel. 1994. Topographic organization of sensory projections to the olfactory bulb. *Cell*. 79:981–991. http://dx.doi.org/10.1016/0092-8674(94)90029-9

Von Dannecker, L.E., A.F. Mercadante, and B. Malnic. 2005. Ric-8B, an olfactory putative GTP exchange factor, amplifies signal transduction through the olfactory-specific G-protein Gαolf. *J. Neurosci.* 25:3793–3800. http://dx.doi.org/10.1523/JNEUROSCI.4595-04.2005

Von Dannecker, L.E., A.F. Mercadante, and B. Malnic. 2006. Ric-8B promotes functional expression of odorant receptors. *Proc. Natl. Acad. Sci. USA.* 103:9310–9314. http://dx.doi.org/10.1073/pnas.0600697103

Wang, G., A.F. Carey, J.R. Carlson, and L.J. Zwiebel. 2010. Molecular basis of odor coding in the malaria vector mosquito *Anopheles gambiae*. *Proc. Natl. Acad. Sci. USA.* 107:4418–4423. http://dx.doi.org/10.1073/pnas.0913392107

Wetzel, C.H., M. Oles, C. Wellerdieck, M. Kuczkowiak, G. Gisselmann, and H. Hatt. 1999. Specificity and sensitivity of a human olfactory receptor functionally expressed in human embryonic kidney 293 cells and *Xenopus Laevis* oocytes. *J. Neurosci.* 19:7426–7433.

Wu, L., Y. Pan, G.Q. Chen, H. Matsunami, and H. Zhuang. 2012. Receptor-transporting protein 1 short (RTP1S) mediates translocation and activation of odorant receptors by acting through multiple steps. *J. Biol. Chem.* 287:22287–22294. http://dx.doi.org/10.1074/jbc.M112.345884

Zhang, J., G. Huang, A. Dewan, P. Feinstein, and T. Bozza. 2012. Uncoupling stimulus specificity and glomerular position in the mouse olfactory system. *Mol. Cell. Neurosci.* 51:79–88. http://dx.doi.org/10.1016/j.mcn.2012.08.006

Zhang, J., R. Pacifico, D. Cawley, P. Feinstein, and T. Bozza. 2013. Ultrasensitive detection of amines by a trace amine-associated receptor. *J. Neurosci.* 33:3228–3239. http://dx.doi.org/10.1523/JNEUROSCI.4299-12.2013

Zhang, X., X. Zhang, and S. Firestein. 2007. Comparative genomics of odorant and pheromone receptor genes in rodents. *Genomics.* 89:441–450. http://dx.doi.org/10.1016/j.ygeno.2007.01.002

Zhao, H., L. Ivic, J.M. Otaki, M. Hashimoto, K. Mikoshiba, and S. Firestein. 1998. Functional expression of a mammalian odorant receptor. *Science.* 279:237–242. http://dx.doi.org/10.1126/science.279.5348.237

Zhuang, H., and H. Matsunami. 2007. Synergism of accessory factors in functional expression of mammalian odorant receptors. *J. Biol. Chem.* 282:15284–15293. http://dx.doi.org/10.1074/jbc.M700386200
