
FEBS 36679
ARTICLE IN PRESS
No. of Pages 8, Model5G

11 June 2014

---

**Review**

**Regulation of autophagy: Modulation of the size and number of autophagosomes**

Q1 Meiyan Jin, Daniel J. Klionsky*

Q2 Life Sciences Institute, and the Department of Molecular, Cellular and Developmental Biology, University of Michigan, Ann Arbor, MI 48109, United States

---

### ARTICLE INFO

- **Article history:**
  - Received 30 April 2014
  - Revised 2 June 2014
  - Accepted 3 June 2014
  - Available online xxxx
  - Edited by Wilhelm Just

- **Keywords:**
  - Autophagy
  - Lysosome
  - Phagophore
  - Stress
  - Vacuole
  - Yeast

---

### ABSTRACT

Autophagy as a conserved degradation and recycling process in eukaryotic cells, occurs constitutively, but is induced by stress. A fine regulation of autophagy in space, time, and intensity is critical for maintaining normal energy homeostasis and metabolism, and to allow for its therapeutic modulation in various autophagy-related human diseases. Autophagy activity is regulated in both transcriptional and post-translational manners. In this review, we summarize the cytosolic regulation of autophagy via its molecular machinery, and nuclear regulation by transcription factors. Specifically, we consider Ume6-ATG8 and Pho23-ATG9 transcriptional regulation in detail, as examples of how nuclear transcription factors and cytosolic machinery cooperate to determine autophagosome size and number, which are the two main mechanistic factors through which autophagy activity is regulated.

© 2014 Published by Elsevier B.V. on behalf of the Federation of European Biochemical Societies.

---

### 1. Background

Macroautophagy, hereafter referred to as autophagy, is a conserved process among eukaryotic cells, through which cytoplasmic components are delivered to the vacuole (in yeasts or plants) or lysosomes (in mammals) for degradation [1]. The autophagy-lysosome system and the ubiquitin-proteasome system (UPS) are the two major subcellular degradation systems. However, in contrast to the UPS, which is primarily a degradative pathway, autophagy has a much greater range of functions, participating in cellular adaptation and remodeling, the recycling of macromolecular building blocks, and even biosynthetic trafficking [2].

The morphological hallmark of autophagy is a double-membrane vesicle, termed an autophagosome, which is assembled *de novo*. That is, formation of the autophagosome occurs by a process that is distinct from vesicle formation throughout the secretory pathway. In the latter, transient transport vesicles bud off from a pre-existing organelle already containing their cargo [3,4]. In contrast, the autophagosome is formed in a step-wise manner, providing a tremendous flexibility and capacity with regard to cargo sequestration. In yeast cells, autophagy occurs at a low basal, constitutive level. In response to various types of stress, or changing nutrient conditions, autophagy is upregulated. The induction of autophagy involves the recruitment of various autophagy-related (Atg) proteins to a peri-vacuolar site, termed the phagophore assembly site (PAS), where they nucleate and assemble into the initial sequestering compartment, a double-membrane structure named the phagophore. The phagophore can surround cytoplasmic materials randomly, or in the case of selective autophagy targeted cargo molecules are sequestered through direct interaction between receptors, scaffold proteins, and protein components in the phagophore membrane. Upon completion of elongation the phagophore seals to generate an intact double-membrane vesicle; at this stage, formation of the autophagosome is completed. Subsequently, the outer membrane of the autophagosome fuses with the vacuole, exposing the inner vesicle to the hydrolytic environment of the vacuole lumen. Breakdown of the vesicle membrane, and degradation of the cargo ensues (Fig. 1). The resulting macromolecules are released back into the cytosol through various permeases and recycled [1].

While the UPS is restricted to the degradation of ubiquitinated proteins that must be unfolded to gain access to the associated proteases, autophagy mediates the degradation of a wider range of targets, including soluble proteins, protein aggregates, damaged organelles, or even invasive pathogens. Non-selective autophagy is primarily a starvation response in which cytoplasm, potentially

---

*Abbreviations:* Ape1, amino peptidase I; Atg, autophagy-related; ChIP, chromatin immunoprecipitation; Cvt, cytoplasm-to-vacuole targeting; TFEB, transcription factor EB; UPS, ubiquitin-proteasome system

*Corresponding author.*

E-mail address: klionsky@umich.edu (D.J. Klionsky).

http://dx.doi.org/10.1016/j.febslet.2014.06.015

0014-5793/© 2014 Published by Elsevier B.V. on behalf of the Federation of European Biochemical Societies.

---

Please cite this article in press as: Jin, M. and Klionsky, D.J. Regulation of autophagy: Modulation of the size and number of autophagosomes. FEBS Lett. (2014), http://dx.doi.org/10.1016/j.febslet.2014.06.015

FEBS 36679
ARTICLE IN PRESS
No. of Pages 8, Model 5G

M. Jin, D.J. Klionsky / FEBS Letters xxx (2014) xxx–xxx

![Diagram](attachment:diagram.png)

Fig. 1. A model of transcriptional regulation of autophagy in yeast. When nutrients are present, the transcription of most ATG genes is repressed due to the inhibition of autophagy transcription activators, and/or activation of transcription repressors. For example, Ume6 represses the transcription of ATG8, and Pho23 represses the expression of ATG9. Repression of transcription in growing conditions leads to a relatively low amount of Atg proteins that are sufficient for basal autophagy, and consequently the autophagosomes that are formed under these conditions are predicted to be smaller in size (due to limited Atg8) and generated at a slower rate (due to limited Atg9). The Cvt pathway, which is the main type of autophagy-like process in growing conditions, generates Cvt vesicles to sequester prApe1 dodecamers and deliver them to the vacuole to allow maturation of the zymogen. After starvation, transcription activators of autophagy become functional, and the repressors are inhibited. For example, Ume6 is phosphorylated by Rim15 kinase, releasing its repression of ATG8 transcription. Upregulation of ATG gene expression allows larger amounts of Atg components, such as Atg8 and Atg9, to participate in the formation of the autophagosome. As a result, more and larger autophagosomes are generated. During starvation-induced autophagy, Atg proteins are initially recruited to a peri-vacuolar site to generate the double-membrane structure named the phagophore. The phagophore randomly sequesters cytoplasmic material, and after the expansion phase is complete it seals to generate the double-membrane autophagosome. The outer membrane of the autophagosome fuses with vacuole, releasing the inner vesicle, now termed an autophagic body. This single-membrane vesicle along with its cargo is degraded in the vacuole lumen, and the resulting macromolecules are released back into the cytosol.

including organelles that are randomly sequestered, is delivered to the vacuole to provide macromolecules for catabolism (to provide energy), or for the synthesis of essential proteins. In contrast, during selective autophagy, the phagophore membrane is in close apposition to the cargo, preventing bulk cytoplasm from being sequestered [5]. Various types of cargo are selectively degraded through autophagy including protein aggregates (aggrephagy), mitochondria (mitophagy), peroxisomes (pexophagy), lipid droplets (lipophagy), and pathogens (xenophagy) [6]. Finally, although most types of autophagy are degradative in nature, selective autophagy can also serve a biosynthetic function. For example, during the cytoplasm-to-vacuole targeting (Cvt) pathway the precursor form of amino peptidase I (prApe1) forms into a dodecamer in the cytosol. The dodecamer subsequently assembles into a larger self-interacting complex that is selectively sequestered by a phagophore. Upon completion, the double-membrane compartment is referred to as a Cvt vesicle. This vesicle targets to and fuses with the vacuole, again releasing the inner vesicle into the lumen. In this case, after lysis of the single-membrane vesicle, the contents are not degraded. Instead, a propeptide that keeps prApe1 inactive is proteolytically removed to generate the mature, active form of the hydrolase (Fig. 1) [7].

## 2. Cytoplasmic regulation of autophagy: Atg proteins

Most of our understanding of the molecular machinery of autophagy has been achieved within the past two decades. In particular, studies relying on genetic analyses in yeast contributed to the identification of the majority of the known protein components that participate in autophagy. At present, more than 30 Atg proteins have been identified in yeast, and orthologs of most of these proteins are present in higher eukaryotes, indicating an evolutionarily conserved molecular machinery. A subset of the Atg proteins that is required for autophagosome formation (or for the generation of selective double-membrane compartments including the Cvt vesicle) is referred to as the molecular core machinery of autophagy. These proteins participate in different steps of autophagy, and, based on their roles in autophagosome formation and interaction with each other, they can be divided into several functional units.

### 2.1. Atg1 kinase complex and induction

The yeast Atg1 kinase complex contains the only kinase of the core machinery, Atg1, and its regulatory subunit, Atg13. The Atg1 complex senses the signals of autophagy induction, delivered from several upstream signaling inputs, such as the target of rapamycin (TOR) [8], protein kinase A (PKA), and Sch9 pathways [9–11]. Atg1 is a serine/threonine protein kinase [12,13], and the kinase activity of Atg1 requires its interaction with Atg13 [14]. During starvation-induced autophagy, Atg1 kinase activity is upregulated, which requires association of Atg1–Atg13 with a stable ternary complex composed of Atg17–Atg31–Atg29 [14–16]. Atg17 is proposed to be the first Atg protein that is recruited to the starvation-specific PAS upon autophagy induction (Atg11 serves this role in growing conditions [17]), and its correct localization is essential for the

Please cite this article in press as: Jin, M. and Klionsky, D.J. Regulation of autophagy: Modulation of the size and number of autophagosomes. FEBS Lett. (2014), http://dx.doi.org/10.1016/j.febslet.2014.06.015

FEBS 36679
ARTICLE IN PRESS
11 June 2014

M. Jin, D.J. Klionsky / FEBS Letters xxx (2014) xxx–xxx

nucleation of other Atg proteins at this site [18]. Atg29 is a phosphoprotein that contains a C-terminal inhibitory domain; phosphorylation of the C terminus is necessary to relieve inhibition, although the mechanism and the relevant kinase are not known [19]. Similarly, the targets of Atg1 kinase that are involved in autophagy have not been well characterized. Atg1 can be autophosphorylated at Thr 266 in the activation loop, and this modification is required for Atg1 kinase activity [20]. BECN1, the mammalian homolog of yeast Vps30/Atg6, is phosphorylated by ULK1 (a mammalian Atg1 homolog) [21], and Atg9 has been identified as one of the direct targets of Atg1 in yeast [22].

in the formation of a coat-like structure by interdigitating with the Atg12-Atg5-Atg16 complex [39,40].

### 2.4. Atg9 and its cycling system

Atg9 is the only transmembrane protein in the core machinery [41]. Atg9 has a unique localization pattern different from the other core machinery proteins. Most Atg proteins are present in two populations, one being diffuse in the cytosol, and the second, that is thought to be the active pool, localized at the PAS; the latter appears as a single punctum in growing conditions that becomes much more intense (based on monitoring of fluorophore-tagged Atg proteins by microscopy) following autophagy induction. Interestingly, Atg9 localizes to multiple punctate structures. One of these punctate structures is the PAS, but the other sites are located proximal to the mitochondria; these latter populations are called Atg9 peripheral sites, Atg9 reservoirs or tubulovesicular clusters (TVCs) [41–43]. Based on studies with a temperature-sensitive Atg1 mutant, Atg9 is proposed to cycle between the peripheral sites/TVCs and the PAS. Anterograde movement to the PAS is dependent on Atg11, Atg23, Atg27 and the Arp2/3 complex, whereas retrograde trafficking depends on the Atg1-Atg13, and Atg2-Atg18 proteins [42,44,45]. The PtdIns3K complex is also required for Atg9 trafficking, which may reflect the fact that Atg18 is a PtdIns3P-binding protein. The exact function of Atg9 is not known, but it is proposed to be involved in delivering membrane to the PAS, or in directing this process. Membrane-bound Atg9 moves between the PAS and the peripheral sites/TVCs on single-membrane vesicles [46,47]. Atg9 PAS localization is dependent on Atg17; the Atg17-Atg31-Atg29 complex forms a crescent-shaped structure that may recruit the Atg9 vesicles to the early phagophore [19,48].

### 2.2. PtdIns3K complex and nucleation

Vps34 is the only phosphatidylinositol 3-kinase (PtdIns3K) in yeast, and the autophagy-specific PtdIns3K complex I, which is composed of Vps34, Vps15, Vps30, Atg14 and Atg38 is thought to act downstream of the Atg1 kinase complex (complex II, which contains Vps38 instead of Atg14, generates PtdIns3P at the endosome) [23–25]. The main function of the PtdIns3K complex is to generate PtdIns3P at the PAS, and recruit PtdIns3P binding proteins, such as Atg18, to the PAS [26,27].

### 2.3. Ubl conjugation system and expansion

Atg8 and Atg12 are two ubiquitin-like (Ubl) proteins that belong to two distinct conjugation systems that are part of the core autophagy machinery. These proteins are not homologs of ubiquitin, but have structural similarity, containing ubiquitin folds. Both of the Ubl systems are essential for the expansion of the phagophore [28,29]. A C-terminal glycine of Atg12 is stoichiometrically conjugated to an internal lysine of Atg5 in a process that is very reminiscent of ubiquitination; the first step involves activation of Atg12 by the E1-like enzyme Atg7 (a homolog of the ubiquitin-activating enzyme), which is followed by conjugation to Atg5 via an E2-like enzyme, Atg10 [29,30]. An E3 ubiquitin ligase-like enzyme for Atg12-Atg5 conjugation has not been identified. Atg12-Atg5 further forms a complex with Atg16, a small coiled-coil protein [31], and the Atg12-Atg5-Atg16 complex forms larger oligomers via homo-oligomerization mediated by the Atg16 coil-coiled domain [32].

Atg8, the second Ubl protein of autophagy, undergoes a unique type of post-translational modification. Atg8 is initially synthesized with a C-terminal extension (a single arginine residue in yeast) that is proteolytically removed by the Atg4 cysteine protease. The same E1-like enzyme that is used in the Atg12-Atg5 conjugation system, Atg7, activates the modified Atg8; however, conjugation requires a separate E2-like enzyme, Atg3, which attaches the exposed C-terminal glycine to the lipid phosphatidylethanolamine (PE) [28]. The Atg12-Atg5-Atg16 complex may act as the E3 ligase for Atg8-PE conjugation [33], although it is not essential [34]. Besides its putative E3-like activity, Atg12-Atg5-Atg16 is also required for the PAS localization of Atg8 [18]. Unlike the Atg12-Atg5-Atg16 complex that localizes exclusively on the outer membrane of the phagophore [35], Atg8-PE is initially located on both sides of the phagophore during expansion; Atg8 on the outer surface of the completed autophagosome is removed by a second Atg4-dependent cleavage (referred to as deconjugation), whereas some of the Atg8 on the inner surface remains inside the completed autophagosome [36,37]. The Atg8 that lines the concave surface of the phagophore plays an important role in cargo recognition during selective types of autophagy [38]. In contrast, the population located on the outer surface is involved in determining the size of the autophagosome and may play a role

### 3. Physiology and pathology

Autophagy occurs at a constitutive basal level, but is upregulated when it is induced by different types of stress such as starvation, hypoxia, or pathogen infection. In a nutrient-rich condition the basal level of autophagy is critical for normal homeostasis, performing housekeeping roles such as the removal of misfolded and damaged proteins, the biosynthetic delivery of vacuole-resident enzymes, or maintaining proper metabolism. Recent studies have mostly focused on stress-induced autophagy; however, basal autophagy plays a very important role as part of the quality control machinery, limiting the accumulation of damaged organelles and misfolded proteins that may contribute to many types of neurodegenerative diseases, such as Huntington, Alzheimer and Parkinson disease [49]. In the heart, basal autophagy is important for maintaining cardiomyocyte size and global cardiac structure and function [50], whereas in the liver it may play a role in preventing misfolding diseases such as α1-antitrypsin deficiency [51]. The degradation of depolarized mitochondria by basal mitophagy also prevents spurious inflammation caused by reactive oxygen species and DNA released from damaged mitochondria [52].

When a cell senses stress, autophagy is induced. Under starvation conditions, cells degrade proteins and lipids through autophagy to maintain energy homeostasis. Autophagy-defective cells display a tremendous reduction in viability during starvation, and autophagy-defective mice cannot survive after birth, when the maternal trans-placental nutrient supply is disrupted [53]. In addition autophagy triggered in oocytes by fertilization is essential for preimplantation development in mammals [54]. As mentioned above, autophagy is also induced by pathogen infection, and the selective autophagy of pathogens, xenophagy, can target and kill

Please cite this article in press as: Jin, M. and Klionsky, D.J. Regulation of autophagy: Modulation of the size and number of autophagosomes. FEBS Lett. (2014), http://dx.doi.org/10.1016/j.febslet.2014.06.015

FEBS 36679
ARTICLE IN PRESS
11 June 2014

M. Jin, D.J. Klionsky / FEBS Letters xxx (2014) xxx–xxx

---

invasive pathogens, including bacteria and viruses; therefore, this process is essential for maintaining a healthy innate immune system. Conversely, some microbes have evolved to evade or even subvert autophagy for their own purposes, establishing a replicative niche within autophagosomes, or relying on autophagy to provide nutrients [55]. Besides its functions in preventing infectious diseases, autophagy is involved in immunity by delivering antigens for major histocompatibility (MHC) presentation. The intracellular substrates of autophagy can be loaded on MHC class II, which is usually considered to present extracellular antigens, explaining how the MHC class II pathway contributes to intracellular antigen presentation [56].

- Too little autophagy is harmful, but too much autophagy can also be deleterious—uncontrolled autophagy may cause cell death. Autophagy functions as a mechanism of tumor suppression, but cancer cells can also use autophagy to help them to survive in a hypoxic and low-nutrient environment, or to carry out its cytoprotective function following anticancer treatments that may damage organelles. Furthermore, autophagy plays different roles in different tissues. Inhibition of autophagy in liver cells causes accumulation of hepatic lipids, while inhibition of autophagy in adipose tissue causes decreased white adipose mass. Therefore, a tight regulation of autophagy in terms of its magnitude and in a tissue-specific manner will be crucial for the therapy of autophagy-related diseases.

### 4. Nuclear regulation: transcriptional regulation of autophagy

Like many other cellular processes, the induction of autophagy involves not only the post-translational modification and action of the Atg proteins, but also a coordinated transcriptional activation of the ATG genes. The abnormal expression of many ATG genes has been observed in various human diseases [57–60] suggesting the physiological significance of transcriptional regulation of autophagy. Although the transcriptional induction of ATG genes was first reported more than 10 years ago, at present our knowledge of the transcription factors that are involved in autophagy regulation is very limited, even in the yeast system, where the molecular mechanism of autophagy has been most intensively studied. Some progress has been made with regard to transcriptional regulation in the mammalian system in recent years, and we briefly highlight these findings along with data from yeast below.

#### 4.1. FOXO transcription factors

The conserved forkhead box-containing protein O subfamily (FOXO) transcription factors regulate autophagy in different cell types and organisms. The FOXO proteins shuttle between the cytoplasm and the nucleus, which is regulated by phosphorylation, and they have both transcription-dependent and -independent roles in autophagy regulation. Here we consider the transcription-dependent roles of FOXO family members. There is only one FOXO protein in worms and flies, whereas there are four members, FOXO1, FOXO3, FOXO4, and FOXO6, in mammals [61]. In general, FOXO transcription factors function as activators of autophagy. Under stress stimuli such as nutrient deprivation, phosphorylation of FOXO proteins by AKT1 is blocked, and FOXO transcription factors translocate into the nucleus. In mouse muscle cells, some of the Atg genes including *Lc3b*, and *Gabarapl1* are directly targeted and activated by FOXO3 as indicated by chromatin immunoprecipitation (ChIP) assays [62,63]. In neurons, transcription of Atg genes such as *Atg3*, *Atg5*, and *Atg12* is activated by FOXO1 [64]. Transcriptional regulation of autophagy by FOXO transcription factors has also been reported in various other types of cells, including cardiomyocytes, hepatocytes, and primary renal proximal tubular cells [61]. The positive regulation of autophagy by FOXO transcription factors in muscle is conserved in *Drosophila* [65]. In budding yeast, the role of Fhl1, the homolog of FOXO transcription factors, in transcriptional regulation of autophagy has not been directly tested. However, CHIP-chip data from several independent studies revealed that this protein directly targets to the promoters of several ATG genes including *ATG13*, *ATG27* and *ATG31* [8,66], indicating that a conserved regulation may exist in the yeast system.

#### 4.2. TFEB vs ZKSCAN3

The autophagosome itself is not a degradative compartment. Accordingly, the autophagosome cooperates in the degradation of its cargos with the lysosome in order to fulfill its roles in the degradation and recycling of cytoplasmic materials. Under starvation conditions, both autophagosome and lysosome biogenesis are upregulated. For the transcriptional regulation, two master switches, transcription factor EB (TFEB) as the activator, and ZKSCAN3 as the repressor, have been reported to regulate both autophagosome and lysosome biogenesis. Overexpression of TFEB induces autophagy, while TFEB RNA interference decreases autophagy. Starvation induces TFEB translocation to the nucleus and activates the transcription of several lysosomal and autophagy genes by directly binding to their promoters [67]. This organized pattern of control is referred to as the coordinated lysosomal expression and regulation (CLEAR) network [68]. Conversely, silencing ZKSCAN3, a zinc family DNA-binding repressor protein, upregulates the mRNA level of several autophagy and lysosomal genes, and ChIP experiments suggest that ZKSCAN3 represses transcription of these target genes by direct binding at the respective promoters [69]. Interestingly, in contrast to TFEB, ZKSCAN3 accumulates in the cytosol during starvation.

#### 4.3. E2F1 vs NFKB

BNIP3 is an activator for hypoxia-induced autophagy [70], and its transcription is regulated in an opposite manner by two transcription factors, E2F1 and NFKB. *Bnip3* is a direct target of E2F1, which functions as a transcription activator. NFKB competes with E2F1 for binding at the *Bnip3* promoter, and therefore represses its expression [71]. Besides BNIP3, E2F1 also activates additional genes associated with autophagy including *LC3*, *ULK1*, and *ATG5*, and induces autophagy activity [72]. NFKB represses BNIP3, however, NFKB activates other ATG genes such as *BECN1* [73].

#### 4.4. GATA family factors

GATA family zinc finger transcription factors regulate nitrogen catabolite repression (NCR)-sensitive genes, which produce enzymes and permeases required for using an alternative (non-preferred) nitrogen source. Gln3, a yeast GATA family transcription activator, is regulated by Tor, and Gln3 is reported to positively regulate *ATG14* transcription [74]. This regulatory event is one of the first examples of transcriptional control of autophagy, and this regulation is conserved in *Drosophila* [75]. In yeast, GATA family factors include the relatively well-studied activators Gat1 and Gln3, and the repressors Gzf3 and Dal80. In addition to these factors, there are other DNA-binding proteins containing the conserved GATA-type zinc finger in yeast, such as Gat2, Gat3 and Gat4, whose functions are not clear. Whether other GATA family factors besides Gln3, have roles in the regulation of autophagy is not yet known.

In mammals, GATA1 activates the transcription of *ATG4*, *ATG12*, BNIP3, and mammalian homologs of *ATG8* (*MAP1LC3B*, *GABARAP*, *GABARAPL1*, and *GABARAPL2/GATE-16*). GATA1 directly binds at

---

Please cite this article in press as: Jin, M. and Klionsky, D.J. Regulation of autophagy: Modulation of the size and number of autophagosomes. FEBS Lett. (2014), http://dx.doi.org/10.1016/j.febslet.2014.06.015

the promoters of BNIP3, MAP1LC3B, GABARAP, and GABARAPL2 [76]. Interestingly, GATA1 activates the transcription of FOXO3, and transcriptional regulation of MAP1LC3B requires FOXO3. In contrast, knockdown of TFEB does not affect GATA1-mediated autophagy gene induction [76]. Another GATA transcription factor, GATA4, inhibits autophagy by inducing the transcription of BCL2, and repressing the transcription of ATG5, ATG7, ATG12, and BECN1 [77], which is surprising because GATA4 is generally known as a transcriptional activator. Whether GATA4 represses transcription of ATG genes through direct binding at their promoters is unclear at this time.

### 4.5. Other transcription factors

Other transcription factors such as TP53/p53 and STAT are involved in autophagy regulation, and their roles have been reviewed recently [78,79]. The function of the yeast transcriptional repressor Ume6 in autophagy regulation is discussed in detail below.

## 5. How autophagy activity is regulated: size and number of autophagosomes

Autophagy activity, which corresponds to the amount of cytoplasmic material, whether random portions of the cytoplasm or selective cargos, sequestered by autophagosomes per unit time, followed by degradation within the vacuole and permease efflux, is modulated primarily by two features of autophagosomes: size and number.

### 5.1. Regulation of the size of autophagosomes

A Cvt vesicle, which is the predominant type of sequestering compartment formed in growing conditions, is approximately 140–160 nm in diameter. Upon autophagy induction, much larger autophagosomes are formed; nitrogen-starvation induced autophagosomes range from 300 to 900 nm in diameter [80]. Therefore, the mechanism involved in determining the size of the autophagosome is an interesting question. The flexibility in the size of autophagosomes allows the sequestration of different sizes and amounts of cargo by autophagy. During non-selective autophagy, premature closure of the phagophore would result in smaller autophagosomes, which would in turn attenuate the efficiency of autophagic sequestration and degradation. During selective autophagy, the size of the cargo may determine the size of the sequestering compartment directly through protein-protein interactions between the cargo/ligand-receptor and Atg8 family proteins that are located on the concave face of the phagophore. However, there is a limit to the size of the autophagosomes that form during selective autophagy, indicating that this simple model is not the complete explanation, and there likely exists other machinery (which may be the same as that used in non-selective autophagy) to control the expansion of the phagophore. Under physiological conditions the diameter of the Cvt vesicle is essentially the size of the cargo, the Cvt complex composed primarily of prApe1 and its receptor Atg19. When the prApe1 cargo protein is overexpressed, a larger Cvt complex is formed that cannot be efficiently sequestered by a Cvt vesicle [80]. However, after starvation, when autophagy is induced and larger autophagosomes are generated, the larger Cvt complex can be sequestered and delivered to the vacuole. For both mitophagy and pexophagy, a fusion complex is needed for the efficient sequestration of the targeted organelles [81,82], suggesting that matching the size of the cargo with the phagophore may be important. Therefore, for

selective autophagy, the size of the autophagosome not only regulates efficiency, but may also be critical for the cargo selectivity. A series of studies in our lab on ATG8 and its regulation provides some clues for the regulation of autophagosome size. The amounts of both ATG8 mRNA and its protein product are elevated dramatically within a short time following nitrogen starvation or rapamycin (an inhibitor of TOR) treatment. To reveal the roles of the increased level of Atg8 during autophagy induction, several strains with various, but reduced, expression levels of ATG8 were generated through the use of heterologous promoters. Based on transmission electron microscopy, in conjunction with other autophagy assays, the correlation between Atg8 protein amount and autophagosome size/autophagy activity was determined [39]. It is still unclear how Atg8 regulates the size of the autophagosome. The observation that the Atg12-Atg5-Atg16 complex together with Atg8 can form highly oligomerized network structures on giant unilamellar vesicles in vitro, indicates the possibility that Atg8 along with the Atg12-Atg5-Atg16 complex forms a protein coat on the outer membrane of the phagophore to regulate its curvature, as discussed above [40]. As PE, a cone-shaped lipid that is the target of Atg8 conjugation, can modulate membrane curvature [83], Atg8 may also regulate the size of the autophagosome through its binding to PE, possibly due to differential distribution on the two faces of the phagophore.

Considering that the amount of Atg8 plays an important role in determining the size of the autophagosome, we became interested in understanding how the transcription of ATG8 is regulated. We identified a transcriptional repressor, Ume6, which can directly bind at the promoter of ATG8 and suppress its transcription in nutrient-rich conditions. In agreement with our previous study, ume6Δ cells, which express an elevated amount of Atg8 even in nutrient-rich conditions, generate larger autophagosomes than the wild-type cells. Phosphorylation of Ume6 by Rim15 after starvation releases its suppression, and these findings suggest a model for a regulatory signaling axis of autophagy: In nutrient-rich conditions upstream sensors such as PKA inhibit the effector kinase Rim15, resulting in active Ume6 that suppresses ATG8 transcription; the resulting low level of Atg8 is sufficient to generate the small Cvt vesicles. During starvation, inhibition of PKA allows the now active Rim15 to inhibit Ume6; the resulting increase in Atg8 allows the formation of larger autophagosomes (Fig. 1) [84]. Therefore, the Ume6-dependent control of ATG8 transcription is an example of size regulation during autophagosome formation.

### 5.2. Regulation of the number of autophagosomes

The second feature, which is as important as the size of autophagosomes, is their number, which can be viewed as the frequency of autophagosome formation. In budding yeast cells, usually only one PAS is observed, but the frequency of autophagosome formation can be increased by autophagy induction to approach a condition of continuous and accelerated autophagy. Data suggest that the half-life of prApe1 processing through the Cvt pathway is approximately 45 min [85], whereas after starvation-induced autophagy the half-life of an autophagosome is approximately 10 min [86]. There are multiple PAS sites in fission yeast, and in mammalian cells phagophore nucleation is not limited to a single location. In these systems, as a consequence of increased autophagosome formation frequency, the number of PAS is increased by autophagy induction. It had been not clear, however, which Atg proteins play a role in determining the number of autophagosomes. From our recent studies of transcriptional regulators of autophagy, we found that cells deleted for the PHO23 gene, encoding a transcription repressor, formed more (but not larger) autophagosomes relative to the wild-type cells. Pho23 regulates transcription of more than one ATG gene, but ATG9 expression is

Please cite this article in press as: Jin, M. and Klionsky, D.J. Regulation of autophagy: Modulation of the size and number of autophagosomes. FEBS Lett. (2014), http://dx.doi.org/10.1016/j.febslet.2014.06.015

changed the most among these genes in the pho23Δ mutant (Fig. 1) [87]. To determine the contribution of Atg9 to the pho23Δ phenotype we used yeast strains with different ATG9 gene expression levels. We found that the amount of Atg9 protein correlated with the numbers of autophagic bodies, supporting our hypothesis that Atg9 levels determine the number of autophagosomes [87]. Considering the role of Atg9 in autophagy as a potential membrane carrier or in directing the delivery of membrane to the PAS, we conclude that Atg9 may regulate the rate of phagophore expansion, and consequently the time required for generating each autophagosome through its cycling between the PAS and the peripheral sites/TVCs.

Mathematical modeling of the energy status of double-membrane structures suggests that a minimal change of the rim curvature on the edge of the double-membrane sheet may drive its bending and closure to form a vesicle [88]. Interestingly, Atg9 appears to localize only at the edge of the phagophore [89], suggesting another possibility that Atg9 regulates closure of the phagophore by affecting the rim curvature on the edge of this structure. If this is the case, the Atg17-Atg31-Atg29 scaffold, which forms a crescent structure with a similar curvature as that of the phagophore rim and also only localizes on the edge of the phagophore, may cooperate with Atg9 in this aspect of autophagy regulation [19,48,89]. Our studies with Atg8 and Atg9 indicate that the size and number of autophagosomes may be separately regulated by different autophagy components. This makes sense considering that modulating size may affect primarily cargo selectivity, while regulating the number of autophagosomes could be carried out to regulate mostly autophagy flux. Interestingly, Pho23 and Ume6 both play roles in the Rpd3 large complex, but Pho23 regulates ATG9 transcription in a Ume6-independent manner, suggesting a dual function of the Rpd3 complex in regulating autophagy activity.

### 6. Future directions

How the size and number of autophagosomes are regulated by different subgroups of Atg proteins is an interesting question to pursue, as it will reveal the different modulations of autophagy activity and selectivity by different autophagy proteins. Furthermore, identifying new transcriptional regulators of autophagy and the corresponding gene targets will help to dissect their functions in modulating autophagy activity. Considering that most regulatory pathways of autophagy are conserved among different organisms, a future genomic study in yeast to reveal the autophagy transcriptome would provide valuable data for these studies, and may provide information that can be used for the therapeutic treatment of autophagy-related diseases.

---

### References

[1] Xie, Z. and Klionsky, D.J. (2007) Autophagosome formation: core machinery and adaptations. Nat. Cell Biol. 9, 1102–1109.

[2] Reggiori, F. and Klionsky, D.J. (2013) Autophagic processes in yeast: mechanism, machinery and regulation. Genetics 194, 341–361.

[3] Noda, T., Suzuki, K. and Ohsumi, Y. (2002) Yeast autophagosomes: de novo formation of a membrane structure. Trends Cell Biol. 12, 231–235.

[4] Kovács, A.L., Palfia, Z., Rez, G., Vellai, T. and Kovacs, J. (2007) Sequestration revisited: integrating traditional electron microscopy, de novo assembly and new results. Autophagy 3, 655–662.

[5] Sawa-Makarska, J., Abert, C., Romanov, J., Zens, B., Ibiriću, I. and Martens, S. (2014) Cargo binding to Atg19 unmasks additional Atg8 binding sites to mediate membrane-cargo apposition during selective autophagy. Nat. Cell Biol. 16, 425–433.

[6] Mijaljica, D., Nazarko, T.Y., Brumell, J.H., Huang, W.P., Komatsu, M., Simonsen, A., Yamamoto, A., Zhang, H., Klionsky, D.J. and Devenish, R.J. (2012) Receptor protein complexes are in control of autophagy. Autophagy 8, 1701–1705.

[7] Lynch-Day, M.A. and Klionsky, D.J. (2010) The Cvt pathway as a model for selective autophagy. FEBS Lett. 584, 1359–1366.

[8] MacIsaac, K.D., Wang, T., Gordon, D.B., Gifford, D.K., Stormo, G.D. and Fraenkel, E. (2006) An improved map of conserved regulatory sites for Saccharomyces cerevisiae. BMC Bioinformatics 7, 113.

[9] Mizushima, N. (2010) The role of the Atg1/ULK1 complex in autophagy regulation. Curr. Opin. Cell Biol. 22, 132–139.

[10] Yang, Z. and Klionsky, D.J. (2010) Mammalian autophagy: core molecular machinery and signaling regulation. Curr. Opin. Cell Biol. 22, 124–131.

[11] Yorimitsu, T., Zaman, S., Broach, J.R. and Klionsky, D.J. (2007) Protein kinase A and Sch9 cooperatively regulate induction of autophagy in Saccharomyces cerevisiae. Mol. Cell 18, 4180–4189.

[12] Matsuura, A., Tsukada, M., Wada, Y. and Ohsumi, Y. (1997) Apg1p, a novel protein kinase required for the autophagic process in Saccharomyces cerevisiae. Gene 192, 245–250.

[13] Straub, M., Bredschneider, M. and Thumm, M. (1997) AUT3, a serine/threonine kinase gene, is essential for autophagocytosis in Saccharomyces cerevisiae. J. Bacteriol. 179, 3875–3883.

[14] Kamada, Y., Funakoshi, T., Shintani, T., Nagano, K., Ohsumi, M. and Ohsumi, Y. (2000) Tor-mediated induction of autophagy via an Apg1 protein kinase complex. J. Cell Biol. 150, 1507–1513.

[15] Kabeya, Y., Kamada, Y., Baba, M., Takikawa, H., Sasaki, M. and Ohsumi, Y. (2005) Atg17 functions in cooperation with Atg1 and Atg13 in yeast autophagy. Mol. Biol. Cell 16, 2544–2553.

[16] Cao, Y., Nair, U., Yasumura-Yorimitsu, K. and Klionsky, D.J. (2009) A multiple ATG gene knockout strain for yeast two-hybrid analysis. Autophagy 5, 699–705.

[17] Cheong, H., Nair, U., Geng, J. and Klionsky, D.J. (2008) The Atg1 kinase complex is involved in the regulation of protein recruitment to initiate sequestering vesicle formation for nonspecific autophagy in Saccharomyces cerevisiae. Mol. Cell Biol. 19, 668–681.

[18] Suzuki, K., Kubota, Y., Sekito, T. and Ohsumi, Y. (2007) Hierarchy of Atg proteins in pre-autophagosomal structure organization. Genes Cells 12, 209–218.

[19] Mao, K., Chew, L.H., Inoue-Aono, Y., Cheong, H., Nair, U., Popelka, H., Yip, C.K. and Klionsky, D.J. (2013) Atg29 phosphorylation regulates coordination of the Atg17-Atg31-Atg29 complex with the Atg11 scaffold during autophagy initiation. Proc. Natl. Acad. Sci. U.S.A. 110, E2875–E2884.

[20] Yeh, Y.Y., Wrasman, K. and Herman, P.K. (2010) Autophosphorylation within the Atg1 activation loop is required for both kinase activity and the induction of autophagy in Saccharomyces cerevisiae. Genetics 185, 871–882.

[21] Russell, R.C., Tian, Y., Yuan, H., Park, H.W., Chang, Y.Y., Kim, J., Kim, H., Neufeld, T.P., Dillin, A. and Guan, K.-L. (2013) ULK1 induces autophagy by phosphorylating Beclin-1 and activating VPS34 lipid kinase. Nat. Cell Biol. 15, 741–750.

[22] Papinski, D., Schuschnig, M., Reiter, W., Wilhelm, L., Barnes, C.A., Maiolica, A., Hansmann, I., Pfaffenwimmer, T., Kijanska, M., Stoffel, I., Lee, S.S., Brezovich, A., Lou, J.H., Turk, B.E., Aebersold, R., Ammerer, G., Peter, M. and Kraft, C. (2014) Early steps in autophagy depend on direct phosphorylation of Atg9 by the Atg1 kinase. Mol. Cell 53, 471–483.

[23] Araki, Y., Ku, W.C., Akioka, M., May, A.I., Hayashi, Y., Arisaka, F., Ishihama, Y. and Ohsumi, Y. (2013) Atg38 is required for autophagy-specific phosphatidylinositol 3-kinase complex integrity. J. Cell Biol. 203, 299–313.

[24] Jao, C.C., Ragusa, M.J., Stanley, R.E. and Hurley, J.H. (2013) A HORMA domain in Atg13 mediates PI 3-kinase recruitment in autophagy. Proc. Natl. Acad. Sci. U.S.A. 110, 5486–5491.

[25] Kihara, A., Noda, T., Ishihara, N. and Ohsumi, Y. (2001) Two distinct Vps34 phosphatidylinositol 3-kinase complexes function in autophagy and carboxypeptidase Y sorting in Saccharomyces cerevisiae. J. Cell Biol. 152, 519–530.

[26] Juhasz, G., Hill, J.H., Yan, Y., Sass, M., Baehrecke, E.H., Backer, J.M. and Neufeld, T.P. (2008) The class III PI(3)K Vps34 promotes autophagy and endocytosis but not TOR signaling in Drosophila. J. Cell Biol. 181, 655–666.

[27] Stromhaug, P.E., Reggiori, F., Guan, J., Wang, C.-W. and Klionsky, D.J. (2004) Atg21 is a phosphoinositide binding protein required for efficient lipidation and localization of Atg8 during uptake of amino peptidase I by selective autophagy. Mol. Biol. Cell 15, 3553–3566.

[28] Ichimura, Y., Kirisako, T., Takao, T., Satomi, Y., Shimonishi, Y., Ishihara, N., Mizushima, N., Tanida, I., Kominami, E., Ohsumi, M., Noda, T. and Ohsumi, Y. (2000) A ubiquitin-like system mediates protein lipidation. Nature 408, 488–492.

[29] Mizushima, N., Noda, T., Yoshimori, T., Tanaka, Y., Ishii, T., George, M.D., Klionsky, D.J., Ohsumi, M. and Ohsumi, Y. (1998) A protein conjugation system essential for autophagy. Nature 395, 395–398.

[30] Shintani, T., Mizushima, N., Ogawa, Y., Matsura, A., Noda, T. and Ohsumi, Y. (1999) Apg10p, a novel protein-conjugating enzyme essential for autophagy in yeast. EMBO J. 18, 5234–5241.

[31] Mizushima, N., Noda, T. and Ohsumi, Y. (1999) Apg16p is required for the function of the Apg12p-Apg5p conjugate in the yeast autophagy pathway. EMBO J. 18, 3888–3896.

[32] Kuma, A., Mizushima, N., Ishihara, N. and Ohsumi, Y. (2002) Formation of the approximately 350-kDa Apg12-Apg5-Apg16 multimeric complex, mediated by Apg16 oligomerization, is essential for autophagy in yeast. J. Biol. Chem. 277, 18619–18625.

[33] Hanada, T., Noda, N.N., Satomi, Y., Ichimura, Y., Fujioka, Y., Takao, T., Inagaki, F. and Ohsumi, Y. (2007) The Atg12-Atg5 conjugate has a novel E3-like activity for protein lipidation in autophagy. J. Biol. Chem. 282, 37298–37302.

---

Please cite this article in press as: Jin, M. and Klionsky, D.J. Regulation of autophagy: Modulation of the size and number of autophagosomes. FEBS Lett. (2014), http://dx.doi.org/10.1016/j.febslet.2014.06.015

FEBS 36679
ARTICLE IN PRESS
11 June 2014

M. Jin, D.J. Klionsky / FEBS Letters xxx (2014) xxx–xxx

[34] Cao, Y., Cheong, H., Song, H. and Klionsky, D.J. (2008) In vivo reconstitution of autophagy in Saccharomyces cerevisiae. J. Cell Biol. 182, 703–713.

[35] Mizushima, N., Yamamoto, A., Hatano, M., Kobayashi, Y., Kabeya, Y., Suzuki, K., Tokuhisa, T., Ohsumi, Y. and Yoshimori, T. (2001) Dissection of autophagosome formation using Apg5-deficient mouse embryonic stem cells. J. Cell Biol. 152, 657–668.

[36] Huang, W.-P., Scott, S.V., Kim, J. and Klionsky, D.J. (2000) The itinerary of a vesicle component, Aut7p/Cvt5p, terminates in the yeast vacuole via the autophagy/Cvt pathways. J. Biol. Chem. 275, 5845–5851.

[37] Kirisako, T., Ichimura, Y., Okada, H., Kabeya, Y., Mizushima, N., Yoshimori, T., Ohsumi, M., Takao, T., Noda, T. and Ohsumi, Y. (2000) The reversible modification regulates the membrane-binding state of Apg8/Aut7 essential for autophagy and the cytoplasm to vacuole targeting pathway. J. Cell Biol. 151, 263–276.

[38] Shintani, T., Huang, W.-P., Stromhaug, P.E. and Klionsky, D.J. (2002) Mechanism of cargo selection in the cytoplasm to vacuole targeting pathway. Dev. Cell 3, 825–837.

[39] Xie, Z., Nair, U. and Klionsky, D.J. (2008) Atg8 controls phagophore expansion during autophagosome formation. Mol. Biol. Cell 19, 3290–3298.

[40] Kaufmann, A., Beier, V., Franquelim, H.G. and Wollert, T. (2014) Molecular mechanism of autophagic membrane-scaffold assembly and disassembly. Cell 156, 469–481.

[41] Noda, T., Kim, J., Huang, W.-P., Baba, M., Tokunaga, C., Ohsumi, Y. and Klionsky, D.J. (2000) Apg9p/Cvt7p is an integral membrane protein required for transport vesicle formation in the Cvt and autophagy pathways. J. Cell Biol. 148, 465–480.

[42] Reggiori, F., Tucker, K.A., Stromhaug, P.E. and Klionsky, D.J. (2004) The Atg1-Atg13 complex regulates Atg9 and Atg23 retrieval transport from the pre-autophagosomal structure. Dev. Cell 6, 79–90.

[43] Mari, M., Griffith, J., Rieter, E., Krishnappa, L., Klionsky, D.J. and Reggiori, F. (2010) An Atg9-containing compartment that functions in the early steps of autophagosome biogenesis. J. Cell Biol. 190, 1005–1022.

[44] Sekito, T., Kawamata, T., Ichikawa, R., Suzuki, K. and Ohsumi, Y. (2009) Atg17 recruits Atg9 to organize the pre-autophagosomal structure. Genes Cells 14, 525–538.

[45] Yen, W.-L., Legakis, J.E., Nair, U. and Klionsky, D.J. (2007) Atg27 is required for autophagy-dependent cycling of Atg9. Mol. Biol. Cell 18, 581–593.

[46] Reggiori, F., Shintani, T., Nair, U. and Klionsky, D.J. (2005) Atg9 cycles between mitochondria and the pre-autophagosomal structure in yeasts. Autophagy 1, 101–109.

[47] Yamamoto, H., Kakuta, S., Watanabe, T.M., Kitamura, A., Sekito, T., Kondo-Kakuta, C., Ichikawa, R., Kinjo, M. and Ohsumi, Y. (2012) Atg9 vesicles are an important membrane source during early steps of autophagosome formation. J. Cell Biol. 198, 219–233.

[48] Ragusa, M.J., Stanley, R.E. and Hurley, J.H. (2012) Architecture of the Atg17 complex as a scaffold for autophagosome biogenesis. Cell 151, 1501–1512.

[49] Nixon, R.A. (2013) The role of autophagy in neurodegenerative disease. Nat. Med. 19, 983–997.

[50] Nakai, A., Yamaguchi, O., Takeda, T., Higuchi, Y., Hikoso, S., Taniike, M., Omiya, S., Mizote, I., Matsumura, Y., Asahi, M., Nishida, K., Hori, M., Mizushima, N. and Otsu, K. (2007) The role of autophagy in cardiomyocytes in the basal state and in response to hemodynamic stress. Nat. Med. 13, 619–624.

[51] Perlmutter, D.H. (2009) Autophagic disposal of the aggregation-prone protein that causes liver inflammation and carcinogenesis in α-1-antitrypsin deficiency. Cell Death Differ. 16, 39–45.

[52] Deretic, V. (2012) Autophagy as an innate immunity paradigm: expanding the scope and repertoire of pattern recognition receptors. Curr. Opin. Immunol. 24, 21–31.

[53] Kuma, A., Hatano, M., Matsui, M., Yamamoto, A., Nakaya, H., Yoshimori, T., Ohsumi, Y., Tokuhisa, T. and Mizushima, N. (2004) The role of autophagy during the early neonatal starvation period. Nature 432, 1032–1036.

[54] Tsukamoto, S., Kuma, A., Murakami, M., Kishi, C., Yamamoto, A. and Mizushima, N. (2008) Autophagy is essential for preimplantation development of mouse embryos. Science 321, 117–120.

[55] Ravikumar, B., Sarkar, S., Davies, J.E., Futter, M., Garcia-Arencibia, M., Green-Thompson, Z.W., Jimenez-Sanchez, M., Korolchuk, V.I., Lichtenberg, M., Luo, S., Massey, D.C., Menzies, F.M., Moreau, K., Narayanan, U., Renna, M., Siddiqi, F.H., Underwood, B.R., Winslow, A.R. and Rubinsztein, D.C. (2010) Regulation of mammalian autophagy in physiology and pathophysiology. Physiol. Rev. 90, 1383–1435.

[56] Nedjic, J., Aichinger, M., Mizushima, N. and Klein, L. (2009) Macroautophagy, endogenous MHC II loading and T cell selection: the benefits of breaking the rules. Curr. Opin. Immunol. 21, 92–97.

[57] Chen, D., Pang, S., Feng, X., Huang, W., Hawley, R.G. and Yan, B. (2013) Genetic analysis of the ATG7 gene promoter in sporadic Parkinson’s disease. Neurosci. Lett. 534, 193–198.

[58] Liu, H., He, Z., von Rutte, T., Yousefi, S., Hunger, R.E. and Simon, H.U. (2013) Down-regulation of autophagy-related protein 5 (ATG5) contributes to the pathogenesis of early-stage cutaneous melanoma. Sci. Transl. Med. 5, 202ra123.

[59] Wang, J., Pan, X.L., Ding, L.J., Liu, D.Y., Da-Peng, L. and Jin, T. (2013) Aberrant expression of Beclin-1 and LC3 correlates with poor prognosis of human hypopharyngeal squamous cell carcinoma. PLoS ONE 8, e69038.

[60] Jo, Y.K., Kim, S.C., Park, I.J., Park, S.J., Jin, D.H., Hong, S.W., Cho, D.H. and Kim, J.C. (2012) Increased expression of ATG10 in colorectal cancer is associated with lymphovascular invasion and lymph node metastasis. PLoS ONE 7, e52705.

[61] Webb, A.E. and Brunet, A. (2014) FOXO transcription factors: key regulators of cellular quality control. Trends Biochem. Sci. 39, 159–169.

[62] Mammucari, C., Milan, G., Romanello, V., Masiero, E., Rudolf, R., Del Piccolo, P., Burden, S.J., Di Lisi, R., Sandri, C., Zhao, J., Goldberg, A.L., Schiaffino, S. and Sandri, M. (2007) FoxO3 controls autophagy in skeletal muscle in vivo. Cell Metab. 6, 458–471.

[63] Zhao, J., Brault, J.J., Schild, A., Cao, P., Sandri, M., Schiaffino, S., Lecker, S.H. and Goldberg, A.L. (2007) FoxO3 coordinately activates protein degradation by the autophagic/lysosomal and proteasomal pathways in atrophying muscle cells. Cell Metab. 6, 472–483.

[64] Xu, P., Das, M., Reilly, J. and Davis, R.J. (2011) JNK regulates FoxO-dependent autophagy in neurons. Genes Dev. 25, 310–322.

[65] Demontis, F. and Perrimon, N. (2010) FOXO/4E-BP signaling in Drosophila muscles regulates organism-wide proteostasis during aging. Cell 143, 813–825.

[66] Venters, B.J., Wachi, S., Mavrich, T.N., Andersen, B.E., Jena, P., Sinnamon, A.J., Jain, P., Rolleri, N.S., Jiang, C., Hemeryck-Walsh, C. and Pugh, B.F. (2011) A comprehensive genomic binding map of gene and chromatin regulatory proteins in Saccharomyces. Mol. Cell 41, 480–492.

[67] Settembre, C., Di Malta, C., Polito, V.A., Garcia Arencibia, M., Vetrini, F., Erdin, S., Erdin, S.U., Huynh, T., Medina, D., Colella, P., Sardiello, M., Rubinsztein, D.C. and Ballabio, A. (2011) TFEB links autophagy to lysosomal biogenesis. Science 332, 1429–1433.

[68] Palmieri, M., Impey, S., Kang, H., di Ronza, A., Pelz, C., Sardiello, M. and Ballabio, A. (2011) Characterization of the CLEAR network reveals an integrated control of cellular clearance pathways. Hum. Mol. Genet. 20, 3852–3866.

[69] Chauhan, S., Goodwin, J.G., Manyam, G., Wang, J., Kamat, A.M. and Boyd, D.D. (2013) ZKSCAN3 is a master transcriptional repressor of autophagy. Mol. Cell 50, 16–28.

[70] Bellot, G., Garcia-Medina, R., Gounon, P., Chiche, J., Roux, D., Pouyssegur, J. and Mazure, N.M. (2009) Hypoxia-induced autophagy is mediated through hypoxia-inducible factor induction of BNIP3 and BNIP3L via their BH3 domains. Mol. Cell. Biol. 29, 2570–2581.

[71] Shaw, J., Yurkova, N., Zhang, T., Gang, H., Aguilar, F., Weidman, D., Scramstad, C., Weisman, H. and Kirshenbaum, L.A. (2008) Antagonism of E2F-1 regulated Bnip3 transcription by NF-κB is essential for basal cell survival. Proc. Natl. Acad. Sci. U.S.A. 105, 20734–20739.

[72] Polager, S., Ofir, M. and Ginsberg, D. (2008) E2F1 regulates autophagy and the transcription of autophagy genes. Oncogene 27, 4860–4864.

[73] Copetti, T., Bertoli, C., Dalla, E., Demarchi, F. and Schneider, C. (2009) P65/RelA modulates BECN1 transcription and autophagy. Mol. Cell. Biol. 29, 2594–2608.

[74] Chan, T.F., Bertram, P.G., Ai, W. and Zheng, X.F. (2001) Regulation of APG14 expression by the GATA-type transcription factor Gln3p. J. Biol. Chem. 276, 6463–6467.

[75] Bánsréti, A., Lukácsovich, T., Csíkos, G., Erdélyi, M. and Sass, M. (2012) PP2A regulates autophagy in two alternative ways in Drosophila. Autophagy 8, 623–636.

[76] Kang, Y.A., Sanalkumar, R., O’Geen, H., Linnemann, A.K., Chang, C.J., Bouhassira, E.E., Farnham, P.J., Keles, S. and Bresnick, E.H. (2012) Autophagy driven by a master regulator of hematopoiesis. Mol. Cell. Biol. 32, 226–239.

[77] Kobayashi, S., Volden, P., Timm, D., Mao, K., Xu, X. and Liang, Q. (2010) Transcription factor GATA4 inhibits doxorubicin-induced autophagy and cardiomyocyte death. J. Biol. Chem. 285, 793–804.

[78] Pietrocola, F., Izzo, V., Niso-Santo, M., Vacchelli, E., Galluzzi, L., Maiuri, M.C. and Kroemer, G. (2013) Regulation of autophagy by stress-responsive transcription factors. Semin. Cancer Biol. 23, 310–322.

[79] Fullgrabe, J., Klionsky, D.J. and Joseph, B. (2014) The return of the nucleus: transcriptional and epigenetic control of autophagy. Nat. Rev. Mol. Cell Biol. 15, 65–74.

[80] Baba, M., Osumi, M., Scott, S.V., Klionsky, D.J. and Ohsumi, Y. (1997) Two distinct pathways for targeting proteins from the cytoplasm to the vacuole/lysosome. J. Cell Biol. 139, 1687–1695.

[81] Mao, K., Wang, K., Liu, X. and Klionsky, D.J. (2013) The scaffold protein Atg11 recruits fission machinery to drive selective mitochondria degradation by autophagy. Dev. Cell 26, 9–18.

[82] Mao, K., Liu, X., Feng, Y. and Klionsky, D.J. (2014) The progression of peroxisomal degradation through autophagy requires peroxisomal division. Autophagy 10, 652–661.

[83] Vance, J.E. and Tasseva, G. (2013) Formation and function of phosphatidylserine and phosphatidylethanolamine in mammalian cells. Biochim. Biophys. Acta 1831, 543–554.

[84] Bartholomew, C.R., Suzuki, T., Du, Z., Backues, S.K., Jin, M., Lynch-Day, M.A., Umekawa, M., Kamath, A., Zhao, M., Xie, Z., Inoki, K. and Klionsky, D.J. (2012) Ume6 transcription factor is part of a signaling cascade that regulates autophagy. Proc. Natl. Acad. Sci. U.S.A. 109, 11206–11210.

[85] Klionsky, D.J., Cueva, R. and Yaver, D.S. (1992) Aminopeptidase I of Saccharomyces cerevisiae is localized to the vacuole independent of the secretory pathway. J. Cell Biol. 119, 287–299.

Please cite this article in press as: Jin, M. and Klionsky, D.J. Regulation of autophagy: Modulation of the size and number of autophagosomes. FEBS Lett. (2014), http://dx.doi.org/10.1016/j.febslet.2014.06.015

FEBS 36679
ARTICLE IN PRESS
11 June 2014

M. Jin, D.J. Klionsky / FEBS Letters xxx (2014) xxx–xxx

[86] Geng, J., Baba, M., Nair, U. and Klionsky, D.J. (2008) Quantitative analysis of autophagy-related protein stoichiometry by fluorescence microscopy. J. Cell Biol. 182, 129–140.

[87] M. Jin, D. He, S.K. Backues, M.A. Freeberg, X. Liu, J.K. Kim, D.J. Klionsky, Transcriptional regulation by Pho23 modulates the frequency of autophagosome formation, Curr. Biol. (2014) in press.

[88] Knorr, R.L., Dimova, R. and Lipowsky, R. (2012) Curvature of double-membrane organelles generated by changes in membrane size and composition. PLoS ONE 7, e32753.

[89] Suzuki, K., Akioka, M., Kondo-Kakuta, C., Yamamoto, H. and Ohsumi, Y. (2013) Fine mapping of autophagy-related proteins during autophagosome formation in Saccharomyces cerevisiae. J. Cell Sci. 126, 2534–2544.

Please cite this article in press as: Jin, M. and Klionsky, D.J. Regulation of autophagy: Modulation of the size and number of autophagosomes. FEBS Lett. (2014), http://dx.doi.org/10.1016/j.febslet.2014.06.015
