
OPEN

SUBJECT AREAS:
PLURIPOTENCY
TRANSCRIPTOMICS

Received 22 January 2014
Accepted 4 April 2014
Published 24 June 2014

CCL2 enhances pluripotency of human induced pluripotent stem cells by activating hypoxia related genes

Yuki Hasegawa${}^{1,2}$, Dave Tang${}^{1,2}$, Naoko Takahashi${}^{1,2}$, Yoshihide Hayashizaki${}^{1,3}$, Alistair R. R. Forrest${}^{1,2}$, the FANTOM consortium*, & Harukazu Suzuki${}^{1,2}$

${}^{1}$Omics Science Center, RIKEN Yokohama Institute, 1-7-22 Suehiro-cho Tsurumi-ku Yokohama, Kanagawa, 230-0045 Japan,  
${}^{2}$RIKEN Center for Life Science Technologies, Division of Genomic Technologies, 1-7-22 Suehiro-cho Tsurumi-ku Yokohama, Kanagawa, 230-0045 Japan, ${}^{3}$RIKEN Preventive Medicine and Diagnosis Innovation Program, 2-1 Hirosawa, Wako-shi, Saitama, 551-0198 Japan.

Correspondence and requests for materials should be addressed to Y.H. (hasegawa@gsc.riken.jp) or H.S. (harukazu@gsc.riken.jp)

* A list of participants and their affiliations appears in the Supplementary Information.

Standard culture of human induced pluripotent stem cells (hiPSCs) requires basic Fibroblast Growth Factor (bFGF) to maintain the pluripotent state, whereas hiPSC more closely resemble epiblast stem cells than true naïve state ES which requires LIF to maintain pluripotency. Here we show that chemokine (C-C motif) ligand 2 (CCL2) enhances the expression of pluripotent marker genes through the phosphorylation of the signal transducer and activator of transcription 3 (STAT3) protein. Moreover, comparison of transcriptomes between hiPSCs cultured with CCL2 versus with bFGF, we found that CCL2 activates hypoxia related genes, suggesting that CCL2 enhanced pluripotency by inducing a hypoxic-like response. Further, we show that hiPSCs cultured with CCL2 can differentiate at a higher efficiency than culturing with just bFGF and we show CCL2 can be used in feeder-free conditions in the absence of LIF. Taken together, our finding indicates the novel functions of CCL2 in enhancing its pluripotency in hiPSCs.

Induced pluripotent stem cell (iPSCs) reprogramming holds great promise in providing donor matched cells for regenerative medicine and for generating cell and animal models for studying specific genetic diseases. The technology allows us to derive pluripotent stem cells from mature and specialized cells, which can then be differentiated into various cell types${}^{1-3}$. Providing fully competent iPSCs for differentiation into any desired target cell is required for clinical usage, therefore, it is critically important to establish and maintain high quality iPSCs. Recently it was discovered that in rodents, pluripotent stem cells may be classified into two distinct states: the ES-like or “naïve” pluripotent state and the post implantation epiblast-like (EpiSC-like) or the “primed” state of pluripotency${}^{4}$. In mouse, ES-like pluripotent stem cells are distinguished from other pluripotent stem cells by self-renewal in response to LIF signaling and MEK/GSK3 inhibition (LIF/2i conditions) and by two active X chromosomes in female cells${}^{5-8}$. Epiblast stem cells (EpiSCs) depend on the FGF4 signaling pathway and are independent of LIF signaling. EpiSCs can differentiate *in vitro* into the three germ layers similar to ES cells, and are therefore considered as pluripotent, however, EpiSCs largely do not contribute to chimera formation and are thus not considered totipotent${}^{9}$.

Human iPSCs and embryonic stem cells (ESCs) are considered to exhibit the characteristics of EpiSCs like cells${}^{7,9}$. When comparing human and mouse ESCs/iPSCs, there are differences in cell morphology, lower efficiency of attachment of the cells after the passaging, X chromosome inactivation (XCI) and different requirements are necessary for cell culture maintenance (FGF/ACTIVIN in human versus LIF/STAT in mouse). Hanna et al., have recently published the first evidence for a novel “naïve” pluripotent state in humans that corresponds and shares defining features with mouse naïve ESCs${}^{10}$ although the human naïve state can be maintained only for limited passages before the cells differentiate${}^{10}$. These differences place human ESCs/iPSCs to a state closer to the mouse EpiSCs rather than to mouse ESCs${}^{10}$, which can affect the efficiency of differentiation into desired target cells${}^{11}$.

Somatic stem cells reside in niches, and environmental changes, such as temperature, extracellular matrix proteins, stromal cell contacts, and oxygen tension, have a great influence on stem cell function and differentiation. For example, Tomoda et al. have recently reported that culturing conditions can influence the X chromosome inactivation status in hiPSC${}^{12,13}$. X-chromosome reactivation (XaXa), a characteristic of naïve pluripotent state cells, rarely occurs during the reprogramming of human female somatic cells to induced
pluripotent stem cells. Furthermore, mammalian embryonic epiblasts reside in a physiologically hypoxic environment, and culturing ESCs in a hypoxic environment is known to prevent differentiation of human ESCs and enhance generation of human and mouse iPSC colonies ${ }^{14-16}$. The importance of hypoxia is further highlighted by a study by Lengner et al., where they showed that hypoxia drives human embryonic stem cells into a pre-X inactivation state by the repression of $XIST$ genes ${ }^{17}$. In addition, Mathieu et al. have shown that hypoxia can drive committed cells back to a stem cell-like state ${ }^{16}$.

The cultivation of hiPSCs commonly relies on the use of feeder cells from mouse embryonic fibroblasts (MEF), which provide a milieu of factors into the media to help maintain their undifferentiated state. This dependence on feeder cells increases the potential for xeno-contamination, as such they are not suitable for clinical use ${ }^{18}$. In order to bring hiPSCs onto the clinical stage, we sought to identify new culturing conditions that would support hiPSCs in a higher pluripotent state and allow for feeder-free culture on matrices such as laminin 511/521 ${ }^{19}$. Human iPSCs secrete laminin 511/521, one of the most important functional basement membrane components, and they can be maintained on human laminin 511 and 521 in defined culture conditions. However, large-scale production of purified or recombinant laminin 511 and 521 is difficult and expensive.

Previously, we demonstrated that a chemokine, known as Ccl2, enhanced the pluripotency of mouse iPSCs/ESCs by up-regulating $Klf4$, $Nanog$ and $Tbx3$ via the activation of the LIF-STAT3 pathway ${ }^{20}$. In this current study we extended our work on Ccl2 and mouse iPSCs to human iPSCs. Here we report the conserved role of CCL2 in enhancing the pluripotency of human iPSCs. We demonstrate the up-regulation of pluripotent markers $KLF4$, $NANOG$, $ZFP42$ and $DPPA3$ via CCL2. We performed Cap Analysis Gene Expression (CAGE), a transcriptome profiling technique, on libraries cultured with and without CCL2 and observed a consistent up-regulation of hypoxia related genes, suggesting that CCL2 induces a hypoxic-like response in hiPSCs. Lastly, we demonstrate that CCL2 can support hiPSCs cultured together with LIF in the absence of feeder cells and bFGF. These data demonstrate that CCL2 is able to influence pluripotency and improves the culturing of hiPSCs.

### Results

CCL2 enhances expression of pluripotent genes in hiPSCs possibly through JAK/STAT3 pathway. Human ESCs derived from blastocysts and human iPSCs are usually cultured in bFGF-containing medium, and LIF alone is insufficient to maintain pluripotency ${ }^{21}$. The maintenance of pluripotency in human iPSCs requires the TGFbeta/activin/nodal signaling pathway ${ }^{22}$ and not the LIF pathway, whereas mouse iPSCs/ESCs depend on the JAK/STAT3 signaling pathway (Fig. 1a). To observe the effect of CCL2 on human iPSCs (hiPSCs), we compared more than 20 passages on feeder cells hiPSCs cultured with CCL2 (CCL2_hiPSCs) with hiPSCs cultured in standard bFGF conditions (bFGF_hiPSCs).

We have previously demonstrated that culturing mouse iPSCs with Ccl2 and LIF enhanced the expression of the pluripotency marker genes $Klf4$, $Tbx3$, and $Nanog$ via a CCR2 and phosphorylated STAT3 dependent pathway ${ }^{20}$. We examined the expression level of key transcription factors of human iPSCs cultured on bFGF and CCL2. Quantitative RT-PCR showed higher expression of $NANOG$, $KLF4$, $ZFP42$ and $DPPA3$ in CCL2 treated hiPSCs compared to those cultured with bFGF (Fig. 1b). The up-regulation of these and other pluripotent marker genes was also confirmed among multiple human iPS cell lines although the level of up-regulation varied between lines (Supplementary Fig. 1S).

Previously in mouse, we failed to see an up-regulation of $Zfp42$ and $Dppa3$ with Ccl2, however, in human iPSCs we observed a significant up-regulation under CCL2 culturing conditions. $ZFP42$ is involved in the reprogramming of X-chromosome inactivation during the acquisition of pluripotency, and $DPPA3$ and $KLF4$ are known to be expressed at higher levels in mouse ES cells when compared to mouse EpiSCs ${ }^{10}$. In addition, we examined whether bFGF is necessary for the activation of down-stream transcription factors in hiPSCs cultured with CCL2. When tested in combination, hiPSCs cultured with CCL2 but without bFGF showed significantly higher expression of pluripotent marker genes compared to hiPSCs cultured with bFGF with CCL2 (Fig. 1c).

As we have previously demonstrated in mouse that CCL2 activates the JAK/STAT3 pathway, we examined the effect of CCL2 in human iPSCs. Examining the JAK/STAT3, AKT and PI3K pathways in hiPSC cultured with CCL2 on feeder cells by immunoblotting, we observed a 7-fold increase in phospho-Stat3 levels in CCL2_hiPSCs compared to bFGF_hiPSCs (Fig. 1d). Phosphorylation of STAT3 in hiPSCs cultured with bFGF was barely observed. In addition, we observed repression of $SMAD4$ expression which is a gene activated by TGF$\beta$/ACTIVIN/Nodal and bFGF signaling pathways was observed when CCL2 and/or LIF are cultured with or without bFGF (Fig. 1e). This is of interest as it has been shown that over-activation of JAK/STAT3 is sufficient to efficiently induce the conversion of EpiSCs into naive pluripotent cells in mouse ${ }^{23}$, and thus may be a key factor in pushing hiPSCs towards a higher pluripotent state.

Transcriptome analysis reveals that CCL2 enhances hypoxia related genes. To further elucidate the downstream effects of CCL2, we performed Cap Analysis Gene Expression (CAGE) on RNA extracted from hiPSCs cultured with bFGF and hiPSCs cultured with CCL2 (Supplementary Table S1). Briefly, CAGE captures the 5' end of RNA molecules as a short tag ${ }^{24}$ and the number of sequenced tags directly reflects the expression level of RNA molecules. CAGE reads are clustered into CAGE tag clusters (Supplementary Table S2), which usually lie at the 5' end of annotated gene models, thus associating tag clusters to genes (Fig. 2a). Using edgeR, a Bioconductor package for performing differential expression analysis on digital gene expression data ${ }^{25}$, we identified 219 and 119 genes (Supplementary Table S3) significantly up and down-regulated in cells cultured with CCL2, respectively (Fig. 2b). We examined enriched gene ontology (GO) terms for the up-regulated genes (Supplementary Table S4) and observed GO terms associated with transcription factor activity, development, and board organismal terms (Fig. 2c). Enriched gene ontology terms for down-regulated genes (Supplementary Table S5) included a large range of GO terms associated with lipoproteins and lipids, metabolic processes and response to stimulus (Fig. 2d), suggesting an adjustment in the metabolism of the cells.

One of the most significantly up-regulated gene (FDR = 2.106 e ${ }^{-09}$) was endothelial PAS domain protein1 (EPAS1), which is also known as hypoxia-inducible factor alpha 2. This gene is a known transcription factor involved in the induction of oxygen regulated genes. On the contrary, the most significantly down-regulated gene (FDR = 1.522625 e ${ }^{-66}$) was alpha-fetoprotein (AFP), which is a major plasma protein produced by the yolk sac and liver during fetal life and is repressed under hypoxic conditions in human hepatoma cells ${ }^{26}$. Together with the observation of the enriched GO terms, response to oxygen-containing compound and cellular response to oxygen-containing compound (Fig. 2d), we wanted to ascertain whether CCL2 was inducing a hypoxic response.

It has been reported that hypoxia enhances the generation and maintenance of induced pluripotent stem cells ${ }^{14-16}$, which perhaps was one way CCL2 enhanced pluripotency. We obtained a list of 15 GO terms related to hypoxia and extracted 225 Entrez genes associated with these GO terms (Supplementary Table S6). We ranked the expression of these genes by the false discovery rate obtained from the differential expression analysis performed on the two conditions. Examining the top 17 genes, we found that a majority of genes related to hypoxia were up-regulated when cultured with CCL2 (Fig. 2e).
www.nature.com/scientificreports

![](https://i.imgur.com/yourimage.png)

Figure 1 | CCL2 enhances the expression of pluripotent genes via the phosphorylation of STAT3. (a): Signal pathway diagrams for mouse ES/iPSCs and human ES/iPSCs. (b): Quantitative RT-PCR of four key pluripotent genes in human iPSC cultured with bFGF (white bars) and with CCL2 (black bars). Asterisks (*) denote significant p-values (Student’s t-test indicate, p-value < 0.05) and error bars denote standard deviations n = 3. (c): Quantitative RT-PCR for pluripotent marker genes on human iPSCs cultured with bFGF, CCL2 without bFGF and without bFGF and LIF. Asterisks (*) denote significant p-values (Student’s t-test indicate, p-value < 0.05) and error bars indicate standard deviations for n = 3. All primer sequences used in this study are listed in Supplementary Table S8. (d): Immuno-blots of whole cell extracts from CCL2 or bFGF cultured human iPSCs were subjected to immuno-blotting against antibodies for STAT3, phosphorylated STAT3, and GAPDH(left). Ratio of phosphorylated/total STAT3 estimated from the immune-blots. Full length gels and blots are included in the supplementary information (Supplementary Fig. S2). P-values using Student’s t-test indicate a highly significant difference (p-value < 0.01, n = 3) in STAT3 phosphorylation levels between bFGF and CCL2 conditions (right). (e): Quantitative RT-PCR for Smad4 on human iPSCs cultured on feeder cells with adding either bFGF or CCL2 together with and without LIF. Error bars indicate standard deviations for n = 3.

While the first 6 genes, were statistically significant (FDR < 0.05), the remaining 11 were not due to the variance among the replicates. As such, we performed qRT-PCR and validated most of the expression patterns between the two conditions (Supplementary Fig. S3). We observed the up-regulation of CCL2, which has been reported to be up-regulated under hypoxic conditions²⁷ and reported to induce the JAK/STAT pathway via the phosphorylation of STAT3²⁸, corroborating with observations above.

Lastly, we compared our list of 219 up-regulated genes to a list of hypoxia-induced genes (see methods) measured using a technology very similar to CAGE called oligo-capping²⁹,³⁰, and found a significant overlap of 45 genes (exact binomial test, p-value = 0.016, Supplementary Table S7). Overall our CAGE analysis suggests that CCL2 triggered a transcriptional up-regulation of genes related to hypoxia.

CCL2 drives human iPSCs into higher pluripotent state. One of the signatures of blastocyst-like (ground) or naive state iPSCs is the phosphorylation of STAT3³¹, which was observed with the addition of CCL2. Naive state hESCs, which rely on LIF signaling, also displayed high levels of pStat3¹⁰. Another signature of naive state is X chromosome reactivation and it has been reported that human iPSCs cultured under hypoxia show decreased XIST expression¹⁷. We quantified the expression of XIST and other X-linked genes by qRT-PCR to examine the X chromosome status.

We observed significantly lower XIST expression in our CCL2 cultured hiPSCs compared to the bFGF-cultured cells (Fig. 3a) and conversely, the antisense regulator of the XIST gene, TSIX showed 2.7 fold more expression in CCL2 treated conditions (Fig. 3a). In addition, we compared expression levels of 11 X-linked genes, reported to undergo X chromosome inactivation by qRT-PCR with 10 out of 11 genes still showing modestly higher expression in CCL2 treated hiPSCs (Fig. 3b).

These observations are in agreement with our observation in Fig. 1b that ZFP42 is up-regulated. ZFP42 is required for efficient elongation of TSIX and binds the DXPas34 enhancer within the TSIX promoter. ZFP42 is known to be involved in ES cell self-renewal; mouse ZFP42 is expressed in ES cells but is more heterogeneous in EpiSC⁷.

In addition, several cell adhesion genes were up-regulated in CCL2_hiPSCs including HAPLN1, MEGF10, COL12, and SDK2 in CCL2_hiPSCs confirmed by qRT-PCR (Fig. 3c). Cell adhesion is essential for embryonic development. We observed that higher number of colonies attached with hiPSCs cultured with CCL2 after passaging when compared with bFGF. To quantify this observation, we tallied colony numbers formed after 48 hours of passaging the same

SCIENTIFIC REPORTS | 4 : 5228 | DOI: 10.1038/srep05228

a

b

MAP2 NTS
HAPLN1
ACTC1
EPAS1

AHSG
APOA4 SERPINA1
AFP
APOC3

Log 2 expression of annotated CAGE tag clusters

c

CAGE tag cluster

d

e

Color Key
and Histogram

Row Z-Score

| P-value | FDR |
|---------|-----|
| 0       | 0   |
| 0       | 0   |
| 0       | 0.019 |
| 0       | 0.025 |
| 0.001   | 0.028 |
| 0.002   | 0.040 |
| 0.002   | 0.052 |
| 0.003   | 0.061 |
| 0.008   | 0.116 |
| 0.008   | 0.120 |
| 0.013   | 0.163 |
| 0.016   | 0.182 |
| 0.017   | 0.187 |
| 0.021   | 0.211 |
| 0.021   | 0.212 |
| 0.030   | 0.255 |
| 0.031   | 0.264 |

FABP1
EPAS1
VASN
RBX1
ABAT
E2F1
GATA6
CAPN2
IREB2
VHL
MMP2
LOXL2
CCL2
UCP2
AHCY
LMNA
SOD2

bFGF1 bFGF2 CCL21 CCL22 CCL23

Figure 2 | Cap Analysis Gene Expression reveals the up-regulation of hypoxia related genes and the down-regulation of genes associated with lipids and lipoproteins. (a) : A genome browser representation of CAGE reads from 5 different libraries mapping mainly to the 5'end of the EPAS1 gene. The red line at the chromosome ideogram shows the current position and the scale displays the chromosome coordinates. A CAGE tag cluster is formed by aggregating CAGE reads that lie in close proximity to each other. (b) : MA plot showing the expression differences between bFGF and CCL2 CAGE libraries. On the x-axis is the expression strength of a gene (on log 2 scale) and on the y-axis is the fold change between the two conditions. A positive fold change indicates an up-regulated gene in the CCL2 library and a negative fold change indicates down-regulation. Red dots indicate genes that are differentially expressed between the two conditions with statistical significance. Genes related to lipid metabolism are down-regulated and genes related to cell adhesion and hypoxia are up-regulated. The black lines indicate a fold change of +/-2. (c and d) : Enriched gene ontology terms are displayed as TreeMaps, where each rectangle is a single cluster representative of closely related gene ontology (GO) terms. These representatives are further grouped into “superclusters” of loosely related terms and have the same arbitrarily chosen color. (c) The list of GO terms over-represented in the list of up-regulated genes and (d) the list of GO terms over-represented in the list of down-regulated genes in the CCL2 libraries. (e) : Heatmap showing the expression strength of 17 hypoxia related genes ranked by the p-value and subsequent false discovery rate calculated by the differential gene expression analysis. Each row shows the normalized expression pattern (Z-Score) for that particular gene in the 2bFGFand3CCL2 CAGE libraries.

Figure 3 | CCL2 reactivates X chromosome and enhances differentiation ability. (a): Quantitative RT-PCR of XIST and TSIX genes in human iPSC cultured with bFGF (white bars) and with CCL2 (black bars). Asterisks (*) denote significant p-values (Student’s t-test p-value < 0.05) and error bars denote standard deviations for n = 3. (b): Quantitative RT-PCR on additional X-linked genes related to X chromosome inactivation on bFGF_hiPSCs and CCL2_hiPSCs. Asterisks (*) denote significant p-values (Student’s t-test p-value < 0.05) and error bars indicate standard deviations for n = 3. (c): Quantitative RT-PCR on human iPSC cultured with bFGF (white bars) and with CCL2 (black bars) for a panel of genes related to cell adhesion. Asterisks (*) denote significant p-values (Student’s t-test p-value < 0.05) and error bars indicate standard deviations for n = 3. (d): The number of EBs formed from same cell numbers of human iPSC cultured with bFGF (white bars) and with CCL2 (black bars). Asterisks (*) denote significant p-values (Student’s t-test p-value < 0.05) and error bars indicate standard deviations for n = 3. (d): Immunostaining and flow cytometry analysis with a cardiomyocyte marker, Desmin (i), and a neuronal marker, Nestin (ii) and on spontaneously differentiated embryoid bodies derived from iPSC cultured with bFGF and with CCL2. The red color represents both Desmin and Nestin and blue represents nuclei staining. The lower panels show FACS analysis on Desmin positive cells and Nestin positive cells of human iPSCs cultured with bFGF and with CCL2. Plotted graphs for replicate data of FACS analysis are shown in Supplementary Figure S4.

number of hiPSCs cultured with bFGF and CCL2. In comparison with cells cultured in the standard bFGF condition, hiPSCs cultured with CCL2 attached better, had a higher cell survival rate and had double the number of colonies at 48 hours (Data not shown). This result can be explained by up-regulation of a significant numbers of cell adhesion molecules.

Human iPSCs cultured with CCL2 enhance differentiation ability. We next examined the relative differentiation ability of CCL2_hiPSCs and bFGF_hiPSCs. Spontaneous differentiation was used to compare the two conditions: First, embryoid bodies (EBs) were formed by using HEMA-coated dishes with EB formation medium for 10 days, then EB were plated onto the gelatin-coated dishes and
medium was changed every other day for 14 days. CCL2 treated hiPSCs formed more EBs than the hiPSCs cultured with bFGF (Fig. 3d). The differentiated cells were then stained with the cardiomyocyte marker, Desmin and the neuronal marker, Nestin. FACS analysis showed that spontaneous differentiation of CCL2_hiPSCs resulted in significantly more Desmin positive and Nestin positive cells compared to those from the bFGF_hiPSCs (Fig. 3d). These results suggest CCL2 treated hiPSCs are more competent to differentiate into different lineages.

Human iPSCs cultured on CCL2 protein bead coated dishes can maintain their pluripotency without feeder-cells. Since hiPSCs cultured with CCL2 on feeder cells showed higher pluripotency with higher adhesion, we tested whether CCL2 allows hiPSCs to culture in feeder-free condition.

Traditionally, human iPSCs/ESCs require feeder cells for maintaining pluripotency and proliferation. Using feeder cells reduces the clinical potential of hiPSCs due to batch-to-batch variation in the feeders and time-consuming preparation processes. To address this, several feeder-free culture methods have been developed^{32,33} which typically require matrigel (or other ECM component) coated dishes. Testing various strategies for feeder free culture with CCL2 we report that human iPSCs can be cultured on gelatin-coated dishes by adding CCL2 and LIF into the medium. 24 hours after passaging the cells onto feeder-free gelatin-coated dishes, hiPSCs cultured in the presence of CCL2 and LIF attach to the plate while control cells passaged into media containing bFGF do not. After 6 days of culturing, cells cultured with CCL2 and LIF showed significant proliferation (data not shown). In order to make feeder-free condition more efficient and easier, we made CCL2 protein beads^{34}, to provide a sustained release of CCL2, similar to commercially available LIF protein beads (Fig. 4a). H1 tagged CCL2 can be immobilized on virus derived polyhedral. This can be used in conjunction with gelatin-coated dishes to release CCL2 constantly for two weeks without the need for daily medium change. We find that the combination of LIF and CCL2 protein beads could support the proliferation of human iPSCs without feeder cells (Fig. 4b).

Immunostaining with the pluripotent markers; Tra1-60 and SSEA4, confirmed that the resulting colonies were still iPSCs (Fig. 4b). In addition qRT-PCR confirmed CCL2 cultured feeder-free hiPSCs express equivalent or higher levels of pluripotency genes compared to bFGF cultured feeder dependent hiPSCs (Fig. 4c).

Finally, we examined whether LIF was necessary to maintain pluripotency in the feeder-free condition. Culturing hiPS with the immobilized CCL2 beads only showed attachment of colonies but the colonies were more differentiated (Fig. 4b), indicating a small amount of LIF is still required to keep the pluripotency.

### Discussion

Our previous work on mouse induced pluripotent stem cells (miPSCs) identified Ccl2 as an inducer of the key transcription factor genes for pluripotency, Klf4, Nanog, Sox2, and Tbx3, via the activation of the Stat3-pathway. Furthermore, it has been demonstrated that Ccl2 can increase the efficiency of miPSCs generation from mouse fibroblasts by roughly 12 fold^{35}. In the current study, we showed that CCL2 also promotes pluripotency of human iPSCs and importantly appears to push them into a stabler state by activating STAT3 pathway. We have shown that CCL2 treated human iPSCs show a higher colony attachment, up-regulate key markers of pluripotency, differentiate at higher efficiency and show indication of X chromosome reactivation.

The transcriptome profiling of hiPSCs cultured on CCL2 versus bFGF suggested that CCL2 is able to induce a hypoxic response. We found a significant overlap with our list of up-regulated genes by CCL2 with a list of genes up-regulated in a colon cancer cell line (DLD-1) cultured under hypoxic conditions^{30}. In addition to this, we examined the expression of a panel of genes associated to hypoxia and found an overall up-regulation of these genes (Fig. 2e), which we verified by qRT-PCR (Supplementary Fig. S3). In particular, the endothelial PAS domain-containing protein 1 (EPAS1), also known as hypoxia-inducible factor 2 alpha (HIF2A), was one of the most statistically significant up-regulated genes in response to CCL2. HIF2A works in tandem with HIF1A as a transcription factor and along with hypoxic conditions, is required for the normal development of the heart^{36}. Corroborating with this, was the observation of a large enrichment of gene ontology (GO) terms involved with heart development for genes up-regulated by CCL2 (Supplementary Table S4). On the contrary, we observed a large enrichment of GO terms associated with cholesterol and lipid metabolism (Supplementary Table S5) with genes down-regulated with CCL2. The down-regulation of genes related to metabolism may promote the reduction of oxygen consumption and it has been reported that hypoxia suppresses cholesterol synthesis^{37}.

It also has been reported that CCL2 is directly up-regulated by hypoxia-induced transcription^{27} and CCL2 is one of only two chemokines under the direct transcriptional control of HIF-1 regulation^{38}. Thus, the induction of HIF2A by CCL2 may activate a hypoxic response in the hiPSCs. The enhancement in pluripotency and differentiation ability by CCL2 may be explained by the observation that hypoxia enhances the generation of induced pluripotent stem cells^{14}. The role of hypoxia is well appreciated for the maintenance of undifferentiated states of embryonic, hematopoietic, mesenchymal, and neural stem cell phenotypes^{39}. It is also known that hypoxia is a hallmark of rapidly proliferating tumors and has been suggested to be a characteristic of the embryonic and adult stem cell niche^{39}. Some of the molecular pathways that have been shown to mediate these hypoxia-induced responses, such as HIF1A and Notch signaling, appear to be active in both embryonic and neoplastic pluripotent stem cells^{39}. Furthermore, hypoxia also enhances formation of embryoid bodies of human embryonic stem cells^{40}, in line with our observation that CCL2 treated hiPSCs formed more EBs than the hiPSCs cultured with bFGF (Fig. 3d). Therefore a hypoxic condition may trigger a similar signaling cascade and transcriptional program in cells to promote stemness.

We have also demonstrated the conserved role of CCL2 in activating the STAT3 pathway in hiPSCs. Recently, it was found that the activation of JAK/STAT3 is a limiting component for the induction of naive pluripotency and can promote reprogramming EpiSCs to naive pluripotency^{23,31}. Another key characteristic of naive pluripotent state cells is X-chromosome reactivation (XaXa). The XIST non-coding RNA is primarily involved in X-chromosome inactivation and we found that hiPSCs cultured with CCL2 showed a significant reduction in the XIST expression compared to hiPSCs cultured with bFGF (Fig. 3a). The TSIX antisense gene, which acts *in cis* to repress the transcription of XIST was also found to be significantly over-expressed in CCL2 hiPSCs (Fig. 3a). Together with STAT3 activation and XaXa status, CCL2 may push hiPSCs into a much more naive state.

Our initial work on miPSCs cultured on feeder cells and in feeder-free (LIF-treated) conditions identified Ccl2 as significantly over-expressed in miPSCs cultured on feeder cells. Continuing this work on hiPSCs has revealed the same activation of the STAT3-pathway and the induction of key transcription factor genes for pluripotency. The higher pluripotency state of hiPSCs cultured with CCL2 directly resulted in a higher number of colony attachment (Data not shown) and spontaneous differentiation ability (Fig. 3d, e). More importantly, we could culture human iPSCs without the use of feeder cells but by coating the culture dish with gelatin solution on protein beads that constantly secrete CCL2 (Fig. 4a). One of the risks associated with using feeder cells to culture hiPSCs is the cross contamination with viruses and other unwanted material. Thus the ability to culture hiPSCs in a feeder-free environment has important implications for therapeutic and research use.

SCIENTIFIC REPORTS | 4:5228 | DOI:10.1038/srep05228

a

BmCPV polyhedrin gene

Over-expression  
in Sf21 cells

hCCL2 gene

Over-expression  
in Sf21 cells

H1 immobilization  
signal

Crystallization of S29S  
(beads) protein and  
immobilization of  
hCCL2 in co-infected  
Sf21 cells

hCCL2 protein beads (protein  
beads with immobilized  
recombinant hCCL2)

b

Tra1-60

DAPI

Merged

SSEA4

DAPI

Merged

CCL2 only

LIF only

Gelatin only

C

Fold change

bFGF_hiPSCs on feeder cells

CCL2_hiPSCs without feeder cells

KLFA

NANOG

CDH1

DNMT3

DPPA3

OCT3/4

SOX2

Figure 4 | Feeder-free culturing of human iPSCs with CCL2 and LIF maintains pluripotency. (a): Diagram of protein beads: Schematic representation of the preparation of CCL2 polyhedra. The immobilization signal derived from the VP3 region of the Bombyx mori cytoplasmic polyhedrosis virus (BmCPV) was introduced at the C terminus of CCL2. The fusion protein is then co-expressed with BmCPV polyhedrin and incorporated into the resulting polyhedral, which is coated on a dish together with gelatin solution. (b): Fluorescence images of immunostained human iPSCs cultured on CCL2 and LIF protein beads coated dish: Cells stained with TRA1-60 (green) (i) and SSEA4 (red) (ii). The bright field image of cells cultured on CCL2 protein beads only, LIF beads only, and Gelatin solution only showed that the cells were unable to attach or failed to maintain pluripotency (iii). (c): Quantitative RT-PCR was performed on a range of pluripotency genes on bFGF_hiPSCs with feeder cells (black bars) and CCL2_iPSCs cultured without feeder cells (white bars).

### Methods

Cell culturing. The human iPSC line iPS-201B-7 was purchased from RIKEN BRC. Additional cell lines, hi25-4 (derived from human cord blood), hi76-2 (derived from T cells), hi6 (derived from fibroblast) were kindly distributed by Dr.Koseki's laboratory in RIKEN Center for Integrative Medical Sciences.

Cells were cultured on mitomycin C (Sigma Aldrich) treated SNL76/7 cells (ECACC, Porton Down, UK) in the Primate pluripotent stem cells media from ReproCell (RCHEMD 001) containing 4 ng/ml basic Fibro Growth Factor (bFGF). The cells were passaged in small clumps after treatment to remove feeder cells with CTK solution, containing 0.025% trypsin, 0.1 mM collagenase IV, 0.1 mM CaCl2 and 20% Knockout serum replacement (KSR) (Invitrogen, Carlsbad, CA) every 3–5

SCIENTIFIC REPORTS | 4 : 5228 | DOI: 10.1038/srep05228
days. Recombinant CCL2 (WAKO, Osaka, Japan) was added into culture medium with concentration of 10 ng/ml for culturing of human iPS cells. CCL2 was added into the medium everyday when the medium was changed.

### Quantitative RT-PCR.
RNAs for quantitative real time-polymerase chain reaction (qRT-PCR) were extracted from cells after removing feeder cells with CTK solution. Reverse transcription for total RNA was performed using the PrimeScript RT-PCR kit (TAKARA BIO, Otsu, Shiga, Japan) and quantitative PCR (qPCR) was done in 10 ve reaction mixtures with SYBR Premix EX Taq (TAKARA BIO) on an ABI 7500 Fast RT-PCR system (Applied BioSystems, Carlsbad, CA). The thermal cycling consisted of 40 cycles of 94°C for 5 seconds and 62.5°C for 20 seconds. The relative amount of each target gene mRNA in each sample was calculated by normalization to glyceraldehyde 3-phosphate dehydrogenase (GAPDH) mRNA using the 2-ΔΔCT method¹.

The sequences of qRT-PCR primers used in this study are shown in Supplementary Table S8.

### Immuno-blotting.
Cells were washed with phosphate-buffered saline and harvested in 100 μl mammalian protein extraction reagent lysis buffer. Samples were homogenized by passing through 26-gauge needles for 10 times and centrifuged at 13,000 rpm for 5 minutes at 4°C to prepare the supernatant. The protein concentration was determined by BCA protein assay (Thermo Scientific, Rockford, IL). Total protein (10 μg) was subjected to 4%–12% Bis-Tris Novex Gel electrophoresis and electroblotted onto nitrocellulose membranes. The membranes were probed sequentially with anti-p-Stat3 (Cell Signaling Technology (CST), Danvers, MA), anti-Stat3 (CST), anti-p-Akt (CST), anti-Akt (CST), p-Erk1/2 (Promega, Madison, WI), Erk1/2 (CST, #9102), and anti-GAPDH (Santa cruz Biotechnology Inc. Santa Cruz, CA). Blots were incubated with horseradish peroxidase-coupled anti-rabbit IgG or anti-mouse IgG and developed with ECL plus (GE Healthcare). Membranes were stripped between probing by incubation for 15 minutes at room temperature with 1X ReBlot Plus Strong Antibody Stripping Solution (Millipore, Billerica, MA). Chemiluminescence was recorded with a Fuji LAS-3000 luminescent image analyzer.

### Differentiation of hiPSCs.
For EB formation, FGF_iPSC cells were harvested by treating with 1 mg/ml collagenase IV for one hour. The clumps of cells were then transferred into 2-hydroxyethyl methacrylate (HEMA)-coated dishes for 6 days in differentiation medium (knock-out DMEM containing 20% knockout serum replacement, 2-β-mercaptoethanol, L-glutamine, penicillin-streptomycin, and non-essential amino acids). EBs were plated onto gelatin coated tissue culture dishes at very low density in DMEM medium with 20% animal serum. Medium was replaced every two days. 7 days after, the EB was plated onto the gelatin-coated dish and changed medium every other day for 14 days.

### Flow cytometry.
For detection of DESMIN and NESTIN, adherent cells were washed twice in PBS and then incubated fro 20 minutes at 37°C in 0.5% trypsin. Cells were dissociated by gentle pipetting and resuspended at approximately 0.1–1 × 10⁵ cells per milliliter in PBS + 3% normal goat serum (NGS). Cells were incubated for 40 minutes at 4°C with fluorescein-conjugated antibody to DESMIN (Millipore) and NESTIN (Millipore) or corresponding isotype control. Stained cells were analyzed for surface marker expression using FACSCalibur (BD Biosciences, San Jose, CA).

### Transcriptome analysis.
5 μg of pooled RNAs was isolated from hiPSCs cultured with bFGF and hiPSCs cultured with CCL2. Cap Analysis Gene Expression (CAGE) libraries were prepared and sequenced on the HeliScope Genetic Analysis System (Helicos Biosciences) as previously described²². The raw sequencing data is available at http://fantom.gsc.riken.jp/5/. Raw reads containing base-order addition artifacts and other low quality reads were removed using the SMS filter program (Helicos Biosciences). In addition, reads shorter than 20-nt and longer than 70-nt were removed from further analysis. Reads were mapped to the human genome (hg19) using Delve (T. Lassmann in preparation), a high-throughput short read sequence alignment tool that can recognize sequencing biases or increased error rates in homopolymer stretches. We analyzed 14,905,756 CAGE reads from 5 libraries that could be mapped with a mapping quality of 10 or higher (Supplementary table S1). All analyses were performed using R version 3.0.2, using the R Bioconductor (version 2.13) packages CAGEr (1.4.1) for CAGE tag clustering, edgeR (3.4.0) for differential expression, biomaRt (2.18.0) for obtaining Entrez gene models, GenomicFeatures (1.14.2) and GenomicRanges (1.14.3) for genome annotation and GOstats (2.28.0) for gene ontology (GO) enrichment. Enriched GO terms were summarized using the web tool REVIGO³. Tag clusters falling within +/- 200 bp of the 5'end of Entrez Gene models were annotated as the gene it overlapped.

An applied oligo-capping method²⁹, similar to CAGE, was used to observe gene expression changes in a colon cancer cell line (DLD-1) cultured in normoxic and hypoxic conditions³⁰. Briefly, the method selectively replaces the cap structure of mRNA with a synthetic oligo and oligo-cap cDNA libraries are massively sequenced. Their analysis analyzed 13,140 genes and identified 2,690 hypoxia-induced RefSeq genes (corresponding to 1,903 Entrez Genes) with at least two tag per million expression and a fold change greater than 1.5. We overlapped their list of hypoxia-induced Entrez Genes with our list and found an overlap of 45 genes. Using an exact binomial test, (number of successes = 45, number of trials = 219, and probability of success 1903/13140), the probability of the overlap was 0.01586.

We used the Benjamini-Hochberg procedure for controlling the false discovery rate and p-values less than 0.05 were deemed as statistically significant for all analyses. R analyses are publicly available at https://github.com/davetang/ccl2.

1. Takahashi, K. & Yamanaka, S. Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors. *Cell* **126**, 663–676, doi:10.1016/j.cell.2006.07.024 (2006).
2. Takahashi, K. *et al.* Induction of pluripotent stem cells from adult human fibroblasts by defined factors. *Cell* **131**, 861–872, doi:10.1016/j.cell.2007.11.019 (2007).
3. Yu, J. *et al.* Induced pluripotent stem cell lines derived from human somatic cells. *Science* **318**, 1917–1920, doi:10.1126/science.1151526 (2007).
4. Nichols, J. & Smith, A. Naive and primed pluripotent states. *Cell Stem Cell* **4**, 487–492, doi:10.1016/j.stem.2009.05.015 (2009).
5. Chin, M. H. *et al.* Induced pluripotent stem cells and embryonic stem cells are distinguished by gene expression signatures. *Cell Stem Cell* **5**, 111–123, doi:10.1016/j.stem.2009.06.008 (2009).
6. Fan, G. & Tran, J. X chromosome inactivation in human and mouse pluripotent stem cells. *Hum Genet* **130**, 217–222, doi:10.1007/s00439-011-1038-1 (2011).
7. Ying, Q. L. *et al.* The ground state of embryonic stem cell self-renewal. *Nature* **453**, 519–523, doi:10.1038/nature06968 (2008).
8. Silva, J. & Smith, A. Capturing pluripotency. *Cell* **132**, 532–536, doi:10.1016/j.cell.2008.02.006 (2008).
9. Tesar, P. J. *et al.* New cell lines from mouse epiblast share defining features with human embryonic stem cells. *Nature* **448**, 196–199, doi:10.1038/nature05972 (2007).
10. Hanna, J. *et al.* Human embryonic stem cells with biological and epigenetic characteristics similar to those of mouse ESCs. *Proc Natl Acad Sci USA* **107**, 9222–9227, doi:10.1073/pnas.1004584107 (2010).
11. Ojala, M. *et al.* Culture conditions affect cardiac differentiation potential of human pluripotent stem cells. *PLoS One* **7**, e48659, doi:10.1371/journal.pone.0048659 (2012).
12. Silva, S. S., Rowntree, R. K., Mekhoubad, S. & Lee, J. T. X-chromosome inactivation and epigenetic fluidity in human embryonic stem cells. *Proc Natl Acad Sci USA* **105**, 4820–4825, doi:10.1073/pnas.0712136105 (2008).
13. Tomoda, K. *et al.* Derivation conditions impact X-inactivation status in female human induced pluripotent stem cells. *Cell Stem Cell* **11**, 91–99, doi:10.1016/j.stem.2012.05.019 (2012).
14. Yoshida, Y., Takahashi, K., Okita, K., Ichisaka, T. & Yamanaka, S. Hypoxia enhances the generation of induced pluripotent stem cells. *Cell Stem Cell* **5**, 237–241, doi:10.1016/j.stem.2009.08.001 (2009).
15. Forristal, C. E., Wright, K. L., Hanley, N. A., Oreffo, R. O. & Houghton, F. D. Hypoxia inducible factors regulate pluripotency and proliferation in human embryonic stem cells cultured at reduced oxygen tensions. *Reproduction* **139**, 85–97, doi:10.1530/REP-09-0300 (2010).
16. Mathieu, J. *et al.* Hypoxia induces re-entry of committed cells into pluripotency. *Stem Cells* **31**, 1737–1748, doi:10.1002/stem.1446 (2013).
17. Lengner, C. J. *et al.* Derivation of pre-X inactivation human embryonic stem cells under physiological oxygen concentrations. *Cell* **141**, 872–883, doi:10.1016/j.cell.2010.04.010 (2010).
18. Martin, M. J., Muotri, A., Gage, F. & Varki, A. Human embryonic stem cells express an immunogenic nonhuman sialic acid. *Nat Med* **11**, 228–232, doi:10.1038/nm1181 (2005).
19. Vuoristo, S. *et al.* Laminin isoforms in human embryonic stem cells: synthesis, receptor usage and growth support. *J Cell Mol Med* **13**, 2622–2633, doi:10.1111/j.1582-4934.2008.00643.x (2009).
20. Hasegawa, Y. *et al.* CC chemokine ligand 2 and leukemia inhibitory factor cooperatively promote pluripotency in mouse induced pluripotent cells. *Stem Cells* **29**, 1196–1205, doi:10.1002/stem.673 (2011).
21. Thomson, J. A. *et al.* Embryonic stem cell lines derived from human blastocysts. *Science* **282**, 1145–1147 (1998).
22. James, D., Levine, A. J., Besser, D. & Hemmati-Brivanlou, A. TGFbeta/activin/nodal signaling is necessary for the maintenance of pluripotency in human embryonic stem cells. *Development* **132**, 1273–1282, doi:10.1242/dev.01706 (2005).
23. Yang, J. *et al.* Stat3 activation is limiting for reprogramming to ground state pluripotency. *Cell Stem Cell* **7**, 319–328, doi:10.1016/j.stem.2010.06.022 (2010).
24. Shiraki, T. *et al.* Cap analysis gene expression for high-throughput analysis of transcriptional starting point and identification of promoter usage. *Proc Natl Acad Sci USA* **100**, 15776–15781, doi:10.1073/pnas.2136655100 (2003).
25. Robinson, M. D., McCarthy, D. J. & Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. *Bioinformatics* **26**, 139–140, doi:10.1093/bioinformatics/btp616 (2010).
26. Mazure, N. M. *et al.* Repression of alpha-fetoprotein gene expression under hypoxic conditions in human hepatoma cells: characterization of a negative hypoxia response element that mediates opposite effects of hypoxia inducible factor-1 and c-Myc. *Cancer Res* **62**, 1158–1165 (2002).
27. Stowe, A. M. *et al.* CCL2 upregulation triggers hypoxic preconditioning-induced protection from stroke. *J Neuroinflammation* **9**, 33, doi:10.1186/1742-2094-9-33 (2012).
28. Liu, T. *et al.* Regulation of S100A4 expression via the JAK2-STAT3 pathway in rhomboid-phenotype pulmonary arterial smooth muscle cells exposure to hypoxia. *Int J Biochem Cell Biol* **44**, 1337–1345, doi:10.1016/j.biocel.2012.04.017 (2012).

29. Maruyama, K. & Sugano, S. Oligo-capping: a simple method to replace the cap structure of eukaryotic mRNAs with oligoribonucleotides. *Gene* **138**, 171–174 (1994).

30. Tsuchihara, K. *et al.* Massive transcriptional start site analysis of human genes in hypoxia cells. *Nucleic Acids Res* **37**, 2249–2263, doi:10.1093/nar/gkp066 (2009).

31. van Oosten, A. L., Costa, Y., Smith, A. & Silva, J. C. JAK/STAT3 signalling is sufficient and dominant over antagonistic cues for the establishment of naive pluripotency. *Nat Commun* **3**, 817, doi:10.1038/ncomms1822 (2012).

32. Wang, Q. *et al.* A novel xeno-free and feeder-cell-free system for human pluripotent stem cell culture. *Protein Cell* **3**, 51–59, doi:10.1007/s13238-012-2002-0 (2012).

33. Xu, R. H. *et al.* Basic FGF and suppression of BMP signaling sustain undifferentiated proliferation of human ES cells. *Nat Methods* **2**, 185–190, doi:10.1038/nmeth744 (2005).

34. Mori, H. *et al.* Immobilization of bioactive fibroblast growth factor-2 into cubic proteinous microcrystals (Bombyx mori cypovirus polyhedra) that are insoluble in a physiological cellular environment. *J Biol Chem* **282**, 17289–17296, doi:10.1074/jbc.M608106200 (2007).

35. Nagamatsu, G. *et al.* Optimal ratio of transcription factors for somatic cell reprogramming. *J Biol Chem* **287**, 36273–36282, doi:10.1074/jbc.M112.380683 (2012).

36. Dunwoodie, S. L. The role of hypoxia in development of the Mammalian embryo. *Dev Cell* **17**, 755–773, doi:10.1016/j.devcel.2009.11.008 (2009).

37. Matsumoto, K. *et al.* Effects of hypoxia on cholesterol metabolism in human monocyte-derived macrophages. *Life Sci* **67**, 2083–2091 (2000).

38. Mojsilovic-Petrovic, J. *et al.* Hypoxia-inducible factor-1 (HIF-1) is involved in the regulation of hypoxia-stimulated expression of monocyte chemoattractant protein-1 (MCP-1/CCL2) and MCP-5 (Ccl12) in astrocytes. *J Neuroinflammation* **4**, 12, doi:10.1186/1742-2094-4-12 (2007).

39. Mohyeldin, A., Garzon-Muvdi, T. & Quinones-Hinojosa, A. Oxygen in stem cell biology: a critical component of the stem cell niche. *Cell Stem Cell* **7**, 150–161, doi:10.1016/j.stem.2010.07.007 (2010).

40. Chen, H. F. *et al.* Hypoxic culture maintains self-renewal and enhances embryoid body formation of human embryonic stem cells. *Tissue Eng Part A* **16**, 2901–2913, doi:10.1089/ten.tea.2009.0722 (2010).

41. Thomsen, R., Solvsten, C. A., Linnet, T. E., Blechingberg, J. & Nielsen, A. L. Analysis of qPCR data by converting exponentially related Ct values into linearly related X0 values. *J Bioinform Comput Biol* **8**, 885–900 (2010).

42. Kanamori-Katayama, M. *et al.* Unamplified cap analysis of gene expression on a single-molecule sequencer. *Genome Res* **21**, 1150–1159, doi:10.1101/gr.115469.110 (2011).

43. Supek, F., Bosnjak, M., Skunca, N. & Smuc, T. REVIGO summarizes and visualizes long lists of gene ontology terms. *PLoS One* **6**, e21800, doi:10.1371/journal.pone.0021800 (2011).

### Acknowledgments

We are grateful to S. Yamanaka and K. Takahashi in CiRA for sharing their knowledge on how to establish and handle induced pluripotent stem cells. We also greatly appreciate to S. Koseki and D.Yamada for sharing multiple human iPS cell lines. We are thankful to H.Mori in Protein Crystal. LTD for providing CCL2 protein beads and sharing his knowledge and technology. We also greatly appreciate T.Suzuki’s technical supports. This work was supported in part by grant for FS stage A-STEP (Adaptable & Seamless Technology Transfer Program through Target-driven R&D) by JST to Y. Hasegawa.FANTOM5 was made possible by a Research Grant for RIKEN Omics Science Center from MEXT to Y. Hayashizaki a Grant of the Innovative Cell Biology by Innovative Technology (Cell Innovation Program) from the MEXT, Japan to Y. Hayashizaki and to the RIKEN Center for Life Science Technologies. We would like to thank all members of the FANTOM5 consortium for the contribution of samples and analysis of the data-set and thank GeNAS for the data production. D.T. is supported by The European Union Seventh Framework Programme under grant agreement [FP7-People-ITN-2008-238055] ('BrainTrain' project).

### Author contributions

Y. Hasegawa worked on the conception and design, collection and assembly of the data, data analysis and interpretation, manuscript writing, and administrative support. D.T. assisted with the manuscript writing, data analysis and interpretation. N.T. assisted the collection and assembly of the data, data analysis and interpretation. Y. Hayashizaki assisted with the financial and administrative support. A.F. assisted with the manuscript writing. H.S. assisted with the administrative support and manuscript writing.

### Additional information

Supplementary information accompanies this paper at http://www.nature.com/scientificreports

Accession Codes: Raw data has been deposited to DDBJ under the accession code DRA002216.

Competing financial interests: The authors declare no competing financial interests.

How to cite this article: Hasegawa, Y. *et al.* CCL2 enhances pluripotency of human induced pluripotent stem cells by activating hypoxia related genes. *Sci. Rep.* **4**, 5228; DOI:10.1038/srep05228 (2014).

This work is licensed under a Creative Commons Attribution-NonCommercial-NoDerivs 4.0 International License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder in order to reproduce the material. To view a copy of this license, visit http://creativecommons.org/licenses/by-nc-nd/4.0/
