
CHEMBIOCHEM
COMMUNICATIONS

DOI: 10.1002/cbic.201402119

# Characterization of the Calicheamicin Orsellinate C2-O-Methyltransferase CalO6

Shanteri Singh,* Nitin S. Nandurkar, and Jon S. Thorson*[a]

Although bacterial iterative type I polyketide synthases are now known to participate in the biosynthesis of a small set of diverse natural products, the subsequent downstream modification of the resulting polyketide products is poorly understood. We report the functional characterization of the putative orsellinic acid C2-O-methyltransferase, which is involved in calicheamicin biosynthesis. This study suggests that C2-O-methylation precedes C3-hydroxylation/methylation and C5-iodination and requires a coenzyme A- or acyl carrier protein-bound substrate.

Bacterial iterative type I polyketide synthases (iPKSs) are single polypeptides consisting of a series of fused domains of distinct function that operate in a progressive, iterative fashion. These multifunctional enzymes assemble structurally diverse mono- or bicyclic aromatic polyketides (Scheme 1)[1] from acyl coenzyme A (CoA) precursors. Subsequent modification/utilization of the aromatic core has been proposed or demonstrated to be dependent upon a dedicated acyltransferase (AT; chlorothricin,[2] pactamycin,[3] avilamycin,[4] tiacumicin B,[5] or calicheamicin[6]) or via adenylated intermediates (maduropeptin,[7] polyketomycin,[8] azinomycin B,[9] and neocarzinostatin[10]). A primary point of structural divergence of iPKS-derived aromatic polyketides derives from downstream “tailoring” modifications (hydroxylation, epoxidation, acylation, glycosylation, transamination, halogenation, and/or methylation) of the resulting polyketide scaffold. Published work suggests the fundamental timing and/or biochemical basis of these subsequent modifications to be also widely variant and poorly understood. For example, the naphthoic acid O-methyltransferase (O-MT) NcsB1 in neocarzinostatin biosynthesis operates on the free naphthoic acid moiety prior to adenylation,[10c] whereas MdpB1-catalyzed C-methylation during the biosynthesis of maduropeptin utilizes a CoA-tethered 6-methylsalicylic acid substrate.[7b] In contrast, TiaM-catalyzed halogenation occurs as the last step in tiacumicin B maturation, after the iPKS-derived aromatic component has been attached to nearly mature natural product.[5] Thus, additional fundamental studies are needed to provide a general understanding of the factors that dictate the order and substrate preference of post-iPKS events.

A total of six enzymes (CalO1–CalO6) are postulated to participate in constructing the iPKS-derived modified orsellinic acid (OSA, 2,4-dihydroxy-6-methylbenzoic acid) component of the prototype naturally occurring enediyne calicheamicin (Scheme 2). These include: CalO5, the iPKS OSA synthase; CalO2, the P450 hydroxylase; CalO3, the putative flavin-dependent halogenase; and CalO1/O6, putative AdoMet-dependent aromatic O-MTs. Although CalO1 and CalO2 ligand-binding and crystallography studies revealed the need for thioester (CoA or ACP)-conjugated substrates for CalO1-catalyzed methylation and CalO2-catalyzed hydroxylation reactions,[11] additional biochemical characterization of these, or the other, OSA-related enzymes in this pathway has been restricted by a lack of suitable substrates or knowledge regarding the timing of each enzymatic step. Herein, we report the biochemical characterization of the calicheamicin OSA C2-O-MT CalO6 by using S-N-acetylcysteaminyI OSA (SNAC-OSA) as a substrate. This study confirms the regiospecificity of CalO6 and, in conjunction with the prior work on CalO1/O2, offers additional insights regarding the possible enzymatic sequence for the biosynthesis of the key aromatic component of the calicheamicin aryltetrasaccharide.

For biochemical characterization, *N*-His<sub>6</sub>-CalO6 was overproduced in *E. coli* BL21(DE3) and purified by affinity chromatography. In initial attempts, the free acids 2,4-dihydroxy benzoic acid (6), salicylic acid (7), 5-iodo-OSA (8), OSA (9), 3,4-dihydroxybenzoic acid (10), or 5-iodosalicylic acid (11) failed to turnover in the presence of CalO6. Based on prior ligand-binding studies that revealed the P450 hydroxylase CalO2 to bind SNAC-linked substrate mimics,[11a] corresponding SNAC-linked substrates (1–5; Figure 1A) were synthesized and assessed as substrates for CalO6. From this putative substrate set, only SNAC-OSA (4) resulted in methylated products in the presence of CalO6. Specifically, two products (P1 and P2 with retention times of 15.3 and 20.0 min, respectively) were observed (Figure 1B). Both were confirmed as monomethylated SNAC-OSA analogues by HRMS (*m/z* calcld for C<sub>13</sub>H<sub>17</sub>NO<sub>4</sub>S: 306.07705 [M+Na]<sup>+</sup>, found: 306.07603). For a full characterization of the products, the CalO6 reaction was scaled to provide ~900 μg (30%) and ~400 μg (13%) of P1 and P2, respectively, after purification. NMR spectroscopy revealed P1 and P2 to be SNAC-2-methoxy-OSA and SNAC-4-methoxy-OSA, respectively (Figure 1C). The inability of CalO6 to utilize SNAC-2,4-dihydroxybenzoic acid (1) or SNAC-5-iodo-OSA (3) implicates CalO6 as the first step among the putative OSA tailoring reactions en route to calicheamicin. As ligand-binding studies previously revealed iodination to be critical for CalO2 binding, we put forth the puta-

[a] Prof. S. Singh, Dr. N. S. Nandurkar, Prof. J. S. Thorson  
Center for Pharmaceutical Research and Innovation  
Department of Pharmaceutical Sciences  
College of Pharmacy, University of Kentucky  
789 South Limestone Street, Lexington, KY 40536 (USA)  
E-mail: ssingh555@uky.edu  
jsthorsn@uky.edu  

Supporting information for this article is available on the WWW under  
http://dx.doi.org/10.1002/cbic.201402119.

CHEMBIOCHEM
COMMUNICATIONS
www.chembiochem.org

![calicheamicin](https://i.imgur.com/1234567.png)
calicheamicin

![azinomycin B](https://i.imgur.com/8901234.png)
azinomycin B

![avilamycin](https://i.imgur.com/5678901.png)
avilamycin

![tiacumicin B](https://i.imgur.com/2345678.png)
tiacumicin B

![neocarzinostatin](https://i.imgur.com/9012345.png)
neocarzinostatin

![chlorothricin](https://i.imgur.com/6789012.png)
chlorothricin

![maduropeptin](https://i.imgur.com/3456789.png)
maduropeptin

Scheme 1. Representative natural products that contain aromatic polyketides moieties (dark) synthesized by bacterial iterative type I PKS.

The CalO6 steady-state kinetic parameters for P1 and P2 formation were determined by monitoring product formation by HPLC (Table 1). Importantly, the CalO6 $K_m$ values are at least ten times higher than the typical kinetic values reported for aromatic O-MTs, ${}^{[10c,7b]}$ whereas the corresponding $k_{cat}$ values are approximately 1000-fold lower. This suggests the SNAC conjugates to be rather poor mimics of the putative native (CoA or ACP) substrate conjugate. However, this analysis revealed an approximately threefold kinetic bias toward P1 formation, consistent with C2-OH methylation as the preferred CalO6 regio-specificity.

Table 1. Kinetic parameters for CalO6 with various concentrations of SNAC-OSA (0.1–5 mM) and SAM (0.1–4 mM) in 25 mM Tris, 5 mM MgCl₂, pH 8.0 at 35°C.

| Product | Constant substrate | Varied substrate | $k_{cat} \times 10^{-3}$ [min⁻¹] | $K_m$ [mM] | $k_{cat}/K_m \times 10^{-3}$ [mM⁻¹ min⁻¹] |
|---------|---------------------|------------------|-----------------------------------|------------|-------------------------------------------|
| P1      | SAM                 | SNAC-OSA         | 1.03 ± 0.03                      | 1.27 ± 0.08 | 0.81                                      |
|         | SNAC-OSA            | SAM              | 0.94 ± 0.03                      | 0.30 ± 0.03 | 3.13                                      |
| P2      | SAM                 | SNAC-OSA         | 0.33 ± 0.01                      | 1.31 ± 0.11 | 0.25                                      |
|         | SNAC-OSA            | SAM              | 0.36 ± 0.01                      | 0.56 ± 0.06 | 0.64                                      |
| total   | SAM                 | SNAC-OSA         | 1.35 ± 0.03                      | 1.27 ± 0.07 | 1.06                                      |
|         | SNAC-OSA            | SAM              | 1.29 ± 0.03                      | 0.35 ± 0.04 | 3.69                                      |

© 2014 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim
ChemBioChem 2014, 15, 1418–1421 1419

CHEMBIOCHEM
COMMUNICATIONS
www.chembiochem.org

![Diagram](https://i.imgur.com/yourimage.png)

Scheme 2. The proposed pathway for OSA biosynthesis and modification en route to calicheamicin. The architecture and domain organization of the iPKS (CalO5) is illustrated at the top, and the key CalO6-catalyzed C2-OH methylation reaction is highlighted within the box. Based upon the CalO6 biochemical studies described, R is anticipated to be an ACP or CoA thioester.

A)
![Chemical Structures](https://i.imgur.com/yourimage2.png)

B)
![HPLC Chromatogram](https://i.imgur.com/yourimage3.png)

C)
![Sections of P1 and P2 NOESY Spectra](https://i.imgur.com/yourimage4.png)

Figure 1. A) Substrates assessed in this study with the CalO6 substrate SNAC-OSA (4) and CalO6 reaction products P1 and P2 highlighted within the box. B) HPLC chromatogram of a representative CalO6 reaction with 4 at 37°C for 1 h: i) SAM; ii) 4; iii)-v) CalO6 reactions containing 8, 16, and 24 μg of enzyme, respectively; S: SNAC-OSA, P1: SNAC-2-methoxy-OSA, P2: SNAC-4-methoxy OSA, *: SAH and/or SAM degradation products. C) Sections of the P1 and P2 NOESY spectra (mixing time = 300 ms). Arrows in (A) highlight key NOE connections.

A CalO6 BLAST search revealed 39% sequence identity with the mitomycin 7-O-MT MmcR,[12] 37% with the neocarzinostatin O-MT NcsB1,[10d] and 34% with the calicheamicin sugar 4′-O-MT, CalO1[11b] (Figure S1 in the Supporting Information). To further explore the basis of CalO6 regio-specificity, a CalO6 homology model based upon MmcR was constructed by using Swiss-Model (Figure 2).[13] In most MTs, general acid/base catalysis contributes to rate acceleration wherein an active-site His participates as a catalytic base, as exemplified by the rebeccamycin 4′-O-MT, RebM.[14] Sequence alignment (Figure S1) implicates His252–Glu311 as the CalO6 catalytic diad, and, consistent with this, the putative active-site base His252 is within hydrogen-bonding distance (3.1 Å) of Glu311 in the homology model. Manual docking of 4 and SAH to position the favored C2-OH at an appropriate conformation for methylation (4 C2-OH…His252 side-chain imidazole, 3.1 Å; 4 C2-OH…SAH thiol 3.3 Å) leads to an unfavorable distance for C4 methylation (4 C4…

© 2014 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim
ChemBioChem 2014, 15, 1418–1421 1420

CHEMBIOCHEM
COMMUNICATIONS
www.chembiochem.org

This work was supported by the NIH (RO1 CA84374 to J.S.T.) and the National Center for Advancing Translational Sciences (UL1TR000117).

Keywords: acyl carrier proteins · AdoMet · iterative PKSs · orsellinic acid · polyketides · S-adenosylmethionine

[1] a) O. Zhang, B. Pang, W. Ding, W. Liu, ACS Catal. 2013, 3, 1439–1447; b) S. G. Van Lanen, B. Shen, Curr. Opin. Drug Discovery Dev. 2008, 11, 186–195; c) S. G. Van Lanen, B. Shen, Curr. Top. Med. Chem. 2008, 8, 448–459; d) B. Shen, Curr. Opin. Chem. Biol. 2003, 7, 285–295.

[2] a) L. Shao, X. D. Qu, X. Y. Jia, Q. F. Zhao, Z. H. Tian, M. Wang, G. L. Tang, W. Liu, Biochem. Biophys. Res. Commun. 2006, 345, 133–139; b) X. Y. Jia, Z. H. Tian, L. Shao, X. D. Qu, Q. F. Zhao, J. Tang, G. L. Tang, W. Liu, Chem. Biol. 2006, 13, 575–585.

[3] T. Ito, N. Roongsawang, N. Shirasaka, W. L. Lu, P. M. Flatt, N. Kasanah, C. Miranda, T. Mahmud, ChemBioChem 2009, 10, 2253–2265.

[4] G. Weitnauer, A. Mühlenweg, A. Trefzer, D. Hoffmeister, R. D. Süßmuth, G. Jung, K. Welzel, A. Vente, U. Girreser, A. Bechthold, Chem. Biol. 2001, 8, 569–581.

[5] Y. Xiao, S. Li, S. Niu, L. Ma, G. Zhang, H. Zhang, G. Zhang, J. Ju, C. Zhang, J. Am. Chem. Soc. 2011, 133, 1092–1105.

[6] J. Ahlert, E. Shepard, N. Lomovskaya, E. Zazopoulos, A. Staffa, B. O. Bachmann, K. X. Huang, L. Fonstein, A. Czisny, R. E. Whitwam, C. M. Farnet, J. S. Thorson, Science 2002, 297, 1173–1176.

[7] a) S. G. Van Lanen, T. J. Oh, W. Liu, E. Wendt-Pienkowski, B. Shen, J. Am. Chem. Soc. 2007, 129, 13082–13094; b) J. Ling, G. P. Horsman, S. X. Huang, Y. Luo, S. Lin, B. Shen, J. Am. Chem. Soc. 2010, 132, 12534–12536.

[8] M. Daum, I. Peintner, A. Linnenbrink, A. Frerich, M. Weber, T. Paululat, A. Bechthold, ChemBioChem 2009, 10, 1073–1083.

[9] Q. Zhao, Q. He, W. Ding, M. Tang, Q. Kang, Y. Yu, W. Deng, Q. Zhang, J. Fang, G. Tang, W. Liu, Chem. Biol. 2008, 15, 693–705.

[10] a) W. Liu, K. Nonaka, L. Nie, J. Zhang, S. D. Christenson, J. Bae, S. G. Van Lanen, E. Zazopoulos, C. M. Farnet, C. F. Yang, B. Shen, Chem. Biol. 2005, 12, 293–302; b) H. A. Cooke, J. Zhang, M. A. Griffin, K. Nonaka, S. G. Van Lanen, B. Shen, S. D. Bruner, J. Am. Chem. Soc. 2007, 129, 7728–7729; c) Y. Luo, S. Lin, J. Zhang, H. A. Cooke, S. D. Bruner, B. Shen, J. Biol. Chem. 2008, 283, 14694–14702; d) H. A. Cooke, E. L. Guenther, Y. Luo, B. Shen, S. D. Bruner, Biochemistry 2009, 48, 9590–9598.

[11] a) J. G. McCoy, H. D. Johnson, S. Singh, C. A. Bingman, I.-K. Lei, J. S. Thorson, G. N. Phillips Jr., Proteins Struct. Funct. Bioinf. 2009, 74, 50–60; b) A. Chang, S. Singh, C. A. Bingman, J. S. Thorson, G. N. Phillips Jr., Acta Crystallogr. Sect. D Biol. Crystallogr. 2011, 67, 197–203.

[12] S. Singh, A. Chang, R. D. Goff, C. A. Bingman, S. Grüschow, D. H. Sherman, G. N. Phillips, J. S. Thorson, Proteins Struct. Funct. Bioinf. 2011, 79, 2181–2188.

[13] a) K. Arnold, L. Bordoli, J. Kopp, T. Schwede, Bioinformatics 2006, 22, 195–201; b) F. Kiefer, K. Arnold, M. Künzli, L. Bordoli, T. Schwede, Nucleic Acids Res. 2009, 37, D387–D392; c) M. C. Peitsch, Bio/Technology 1995, 13, 658–660.

[14] S. Singh, J. G. McCoy, C. A. Bingman, G. N. Phillips, J. S. Thorson, J. Biol. Chem. 2008, 283, 22628–22636.

[15] C. Zhang, B. R. Griffith, Q. Fu, C. Albermann, X. Fu, I. K. Lee, L. Li, J. S. Thorson, Science 2006, 313, 1291–1294.

Received: March 24, 2014

Acknowledgements

We thank the School of Pharmacy Analytical Instrumentation Centre (University of Wisconsin-Madison) for analytical support.
