
Enzymology:
The Three *Mycobacterium tuberculosis* Antigen 85 Isoforms Have Unique Substrates and Activities Determined by Non-active Site Regions

Keriann M. Backus, Michael A. Dolan, Conor S. Barry, Maju Joe, Peter McPhie, Helena I. M. Boshoff, Todd L. Lowary, Benjamin G. Davis and Clifton E. Barry III

J. Biol. Chem. 2014, 289:25041-25053.  
doi: 10.1074/jbc.M114.581579 originally published online July 14, 2014

---

Access the most updated version of this article at doi: [10.1074/jbc.M114.581579](10.1074/jbc.M114.581579)

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 49 references, 16 of which can be accessed free at  
[http://www.jbc.org/content/289/36/25041.full.html#ref-list-1](http://www.jbc.org/content/289/36/25041.full.html#ref-list-1)

The Three Mycobacterium tuberculosis Antigen 85 Isoforms Have Unique Substrates and Activities Determined by Non-active Site Regions*

Received for publication, June 9, 2014, and in revised form, July 11, 2014 Published, JBC Papers in Press, July 14, 2014, DOI 10.1074/jbc.M114.581579

Keriann M. Backus ${}^{\# S1}$, Michael A. Dolan ${}^{I}$, Conor S. Barry ${}^{S}$, Maju Joe ${}^{II}$, Peter McPhie ${}^{**}$, Helena I. M. Boshoff ${}^{\sharp}$, Todd L. Lowary ${}^{II}$, Benjamin G. Davis ${}^{S2}$, and Clifton E. Barry III ${}^{3}$

From the ${}^{T}$Tuberculosis Research Section, Laboratory of Clinical Infectious Diseases, and the ${}^{I}$Bioinformatics and Computational Biosciences Branch, NIAID, National Institutes of Health, Bethesda, Maryland 20892, the ${}^{S}$Department of Chemistry, University of Oxford, Chemistry Research Laboratory, Oxford OX1 3TA, United Kingdom, the ${}^{II}$Alberta Glycomics Centre and Department of Chemistry, University of Alberta, Edmonton, Alberta T6G 2G2, Canada, and the ${}^{**}$Laboratory of Biochemistry and Genetics, NIDDK, National Institutes of Health, Bethesda, Maryland 20892

---

### Background: The existence of three antigen 85 isoforms in *Mycobacterium tuberculosis* has been proposed to be due to immune evasion.

### Results: Mutating divergent amino acids from outside the enzyme active site changes enzyme activity and substrate preference.

### Conclusion: These enzymes have unique substrate preferences and are regulated by a second carbohydrate-binding site.

### Significance: Understanding the enzymatic function of these enzymes may provide novel chemotherapeutic strategies.

---

The three isoforms of antigen 85 (A, B, and C) are the most abundant secreted mycobacterial proteins and catalyze transes- terification reactions that synthesize mycolated arabinogalactan, trehalose monomycolate (TMM), and trehalose dimycolate (TDM), important constituents of the outermost layer of the cellular envelope of *Mycobacterium tuberculosis*. These three enzymes are nearly identical at the active site and have therefore been postulated to exist to evade host immunity. Distal to the active site is a second putative carbohydrate-binding site of lower homology. Mutagenesis of the three isoforms at this second site affected both substrate selectivity and overall catalytic activity *in vitro*. Using synthetic and natural substrates, we show that these three enzymes exhibit unique selectivity; antigen 85A more efficiently mycolates TMM to form TDM, whereas C (and to a lesser extent B) has a higher rate of activity using free trehalose to form TMM. This difference in substrate selectivity extends to the hexasaccharide fragment of cell wall arabinan. Mutation of secondary site residues from the most active isoform (C) into those present in A or B partially interconverts this substrate selectivity. These experiments in combination with molecular dynamics simulations reveal that differences in the N-terminal helix $\alpha$9, the adjacent Pro${}^{216}$–Phe${}^{228}$ loop, and helix $\alpha$5 are the likely cause of changes in activity and substrate selectivity. These differences explain the existence of three isoforms and will allow for future work in developing inhibitors.

---

*Mycobacterium tuberculosis* the etiological agent of tuberculosis, caused an estimated 1.3 million deaths in 2012 (1). *M. tuberculosis* is a resilient organism, persisting through long courses of antibiotics and years of dormancy within the host. Reasons for this hardiness are numerous, including a slow rate of growth and a greasy, thick, relatively impregnable cell wall (2). This cell wall has been characterized as a complex heteropolymer, whose covalently linked core consists of several macromolecules, including peptidoglycan, arabinogalactan, and mycolic acids, which subtend a layer of non-covalently associated lipids. The mycolic acid portion of the cell wall is composed of long fatty acids (C70–90) that are predominantly either covalently attached to the arabinan or found esterified to the 6- and/or 6'-hydroxyl groups of trehalose as trehalose monomycolate (TMM) ${}^{4}$ or trehalose dimycolate (TDM) (3). The biosynthesis and cell wall attachment of the mycolates are of particular interest from an anti-tubercular drug discovery perspective because many key antibiotics, including isoniazid, SQ109, delamanid, PA-824, and ethambutol (EMB), all target aspects of this pathway (4–8). The three closely related antigen 85 enzymes (Ag85A, -B, and -C) are the most abundantly secreted proteins, making up as much as 41% of the protein content of culture supernatant (9), and are responsible for the biosynthesis of TMM and TDM and covalent attachment of mycolates to arabinogalactan (Fig. 1) (10). An additional isoform (Ag85D) exists, but is inactive due to a mutation of the catalytic serine residue. Because of their prevalence and participation in mycolate metabolism, these enzymes are often proposed as attractive targets for the development of new antitubercular drugs (11, 12). To date, however, inhibitors of

---

* This work was supported, in whole or in part, by the National Institutes of Health, NIAID, Intramural Research Program. This work was also supported by a grant from the Bill and Melinda Gates Foundation (to Joanne Flynn, University of Pittsburgh).
* Supported in part by the Rhodes Trust. Present address: Dept. of Chemical Physiology, Scripps Research Institute, La Jolla, CA.
* To whom correspondence may be addressed: University of Oxford, Chemistry Research Laboratory, Mansfield Road, Oxford OX1 3TA, United Kingdom. Tel.: 44-1865-275652; E-mail: Ben.Davis@chem.ox.ac.uk.
* To whom correspondence may be addressed: Bldg. 33, Rm. 2W20D, 33 North Dr., MSC 3206, Bethesda, MD 20892-3206. Tel.: 301-435-7509; E-mail: cbarry@niaid.nih.gov.

---

${}^{4}$ The abbreviations used are: TMM, trehalose monomycolate; TDM, trehalose dimycolate; TDH, trehalose dihexanoate; Ag85, antigen 85; Ara5 and Ara6, the penta- and hexasaccharide terminus of the arabinogalactan polymer; EMB, ethambutol; RMSD, root mean square deviation; PDB, Protein Data Bank.

M. tuberculosis Ag85 Isoforms Have Unique Activities

![Trehalose](https://i.imgur.com/1234567.png)
**Trehalose**

![TMM](https://i.imgur.com/2345678.png)
**TMM**

![Ag85](https://i.imgur.com/3456789.png)
**Ag85**

![R*](https://i.imgur.com/4567890.png)
**R*=**

![TMM](https://i.imgur.com/5678901.png)
**TMM**

![TDM](https://i.imgur.com/6789012.png)
**TDM**

![Myc-Ara_n](https://i.imgur.com/7890123.png)
**Myc-Ara_n**

FIGURE 1. **Activities of antigen 85.** Antigen 85 catalyzes a transesterification reaction between an acyl donor (typically TMM) and an acceptor that can be free trehalose, another molecule of TMM, or the terminal arabinofuranose residues covalently anchored to the cell wall heteropolymer. The three common substrates are shown at the top, and the products of the enzymatic reaction are shown below each. Because they are highly insoluble, both cell wall polymer and TDM are likely to be terminal sinks for mycolic acids.

the Ag85 enzymes have shown only weak inhibition of mycobacterial growth at relatively high concentrations (6-azido-trehalose with a minimum inhibitory concentration of 200 μg/ml (13) and ebselen with a minimum inhibitory concentration of 20 μg/ml (14)).

The Ag85 enzymes have also been of interest from an immunologic perspective because they elicit a potent immune response (15) and bind fibronectin (16). Due to their immunogenicity, the Ag85A and -B isoforms are also components of some new anti-M. tuberculosis vaccine candidates (17). Because of their virtual identity in the immediate region of the active site, the three Ag85 isoforms (A, B, and C) have been proposed to play a role in evasion of the host immune response because their expression levels differ under specific environmental conditions (18, 19). However, recent evidence from whole genome sequencing has revealed no evidence that antigens (including all three Ag85 proteins) show any more genetic variation than essential proteins, leading to the conclusion that immune recognition has been positively selected (20) and casting doubt on a rationale for the three isoforms involving immune evasion. A plausible explanation for the existence of three Ag85 isoforms in M. tuberculosis and an understanding of their functions within cell wall synthesis and M. tuberculosis survival are currently lacking.

The apparent functional redundancy of the three closely related Ag85 enzymes poses an additional challenge for drug development, because the biological function of the individual enzymes and the degree of cross-talk between the isoforms remain unclear. Although individual knock-out strains for each isoform have been generated (21, 22), the triple knock-out ag85ABC in M. tuberculosis has not been reported, presumably because this is lethal. Because of the apparent ability of the Ag85 enzymes to complement each other, at least as assessed by

their ability to continue to grow in vitro (23, 24), individual mutants have thus far failed to fully dissect the functional roles of the Ag85s in cell envelope biogenesis. Work in Corynebacterium glutamicum has suggested that the Ag85 isoforms in Corynebacteria are not fully functionally redundant; mutants of individual corynebacterial Ag85 isoforms cannot be complemented through expression or overexpression of the M. tuberculosis isoforms (25). In M. tuberculosis, deletion of Ag85C resulted in a depletion of cell wall-linked mycolates, whereas ag85A and ag85B mutants show normal cell wall-associated mycolate levels. In enzyme assays measuring TMM formation from radiolabeled TDM and trehalose, Ag85A and particularly Ag85B, are markedly less active than Ag85C, although the enzymes have highly conserved sequences with nearly 100% identity of active site amino acids (13, 26) (Fig. 2). Multiple structures for Ag85A, -B, and -C have been determined using x-ray crystallography and show that these three proteins are virtually identical (backbone root mean square deviation (RMSD) of ~0.9–1.3 Å) with the single exception of the Ag85C structure bound to a covalent inhibitor (Protein Data Bank (PDB) entry 1DQY), which features a significant straightening of helix α9 and a disruption of the hydrogen bonding network of the catalytic triad. There are several regions outside of the active site that show some sequence divergence, including one region formed by two helices adjacent to the active site (α5 and α9), where additional ligands have been found in co-crystals. Regulation of lipase and esterase catalytic activity by helix and lid flexibility near an active site is well documented (27). These non-conserved residues and the movement of helix α9 adjacent to the active site suggested a plausible mechanistic explanation for the observed differences in Ag85A, -B, and -C catalytic rates and suggested that there may be subtle substrate preferences. Here, we report that these secondary site amino acids mediate

25042 JOURNAL OF BIOLOGICAL CHEMISTRY
VOLUME 289 • NUMBER 36 • SEPTEMBER 5, 2014

the flexibility of helix α9 and the configuration of the catalytic triad. The regulation conferred by these residues explains the previously observed differences in catalytic activity between Ag85A, -B, and -C and also supports a previously unreported substrate selectivity for trehalose, arabinan, and trehalose monomycolate among the Ag85 isoforms.

**MATERIALS AND METHODS**

*Protein Expression and Mutagenesis—*Plasmids were obtained from the Colorado State University tuberculosis research contract HHSN266200400091c as Rv3804/Ag85A Rv1886c/ Ag85B Rv0129c/Ag85C in pET15b (Ag85A) or pET23b (Ag85B/C) vectors containing a hexahistidine tag and were expressed in BL21 cells with overnight isopropyl 1-thio-β-D-galactopyranoside induction (500 μM) in LB medium. Cells were harvested by centrifugation and lysed by sonication, and the protein was purified from the clarified lysate by affinity purification on Talon™ resin. Proteins were concentrated and buffer-exchanged into pH 7.2 phosphate buffer (20 mM) or pH 7.2 triethanolamine buffer (20 mM). Purity was evaluated by SDS-PAGE. Mutagenesis was conducted using a Stratagene QuikChange® Lightning kit according to the manufacturer’s instructions.

*Circular Dichroism (CD)—*Circular dichroism was measured in a Jasco J-715 spectropolarimeter, scanning four times from 260 to 190 nm at 50 nm/min, time constant = 1 s, bandwidth = 1 nm, slit width = 500 μm. Protein solutions in 10 mM phosphate buffer (protein concentrations, 100 μg/ml) were held in 1-mm path length quartz cuvettes. Because of uncertainty about the protein concentrations, secondary structures were estimated as described previously (28).

*Molecular Docking—*Docking studies were performed using AutoDockTools version 1.5.4 (29) and AutoDock Vina (30). The coordinate .pdbqt file for Ag85A was prepared from PDB entry 1SFR by adding polar hydrogens and Kollman charges using AutoDockTools version 1.5.4. A grid box of 42 × 34 × 34 Å centered on the active site of Ag85A was determined. Energy-minimized .pdb coordinates for the Ara5 ligand were prepared with ChemBio3D Ultra version 12.0. Gasteiger charges were added, non-polar hydrogens were merged, and rotatable bonds were set using AutoDockTools version 1.5.4 to generate a flexible coordinate .pdbqt file. The flexible ligand coordinates were docked into the Ag85a coordinates using AutoDock Vina employing a grid box consisting of 19,575 points. The resulting docking poses were visualized and overlaid with PyMOL version 0.99rc6.

*Ag85 Trehalose Dihexanoate (TDH) MS Enzyme Assays—*Ag85 MS assays were conducted as has been described previously (26, 31).

*Purification of \[^{14}C\]TDM and \[^{14}C\]TMM—*Mid-log-phase H37Rv *M. tuberculosis* (*A*~650~ = 0.2, 1 liter in 7H9 medium) were labeled for 3 days with \[^{14}C\]acetate (2 μCi/ml; American Radio-labeled Chemicals), upon which cells were harvested by centrifugation (3000 rpm, 5 min) into 50-ml Falcon tubes and extracted into 2 ml of 2:1 chloroform/methanol for each 50-ml tube of cells for a total of 40 ml of organic extract. The organic layer was removed and dried, and the residue was resuspended in 2 ml of 2:1 chloroform/methanol. Then the entire sample was

spotted onto two silica TLC plates, which were developed for 1 h using 80:20:2 chloroform/methanol/water, exposed to phosphor storage plates, which were scanned for radioactivity. A full-sized image of the plate was printed on a clear, plastic transparency film and used to identify the bands of interest (TDM, TMM, solvent front, and baseline), which were then scraped off. The silica was rinsed several times with 1 ml of 2:1 chloroform/methanol, the solvent was removed to a fresh vial, and the sample was concentrated. Trace silica was removed by passing through a small glass wool plug, and the final sample was resuspended in 2:1 chloroform/methanol to a final concentration of 10 μCi/ml for TMM and 50 μCi/ml for TDM.

*TDM/TMM Assays—*50 μl of \[^{14}C\]TMM (100 μCi/μl, ~200 μM) stock solution in 2:1 chloroform/methanol was added to each reaction tube (Eppendorf, 1.5 ml), and all of the solvent was removed under a stream of argon. The remaining solid was resuspended by vortexing in 50 μl of protein solution (20 μM in 20 mM phosphate buffer, pH 7.2), and to this was added either 50 μl of additional buffer or 50 μl of alternate substrate. Trehalose and 6-azido-trehalose (26, 31) were tested at a final working concentration of 200 μM. Ara6 was tested at a final concentration of 200 μM to 1 mM. The reactions were mixed with vortexing and allowed to incubate overnight at 37 °C with shaking. Upon completion, the aqueous phase was extracted into 400 μl of 2:1 chloroform/methanol, transferred to a new Eppendorf tube, dried under a stream of argon, resuspended in solvent (2:1 chloroform/methanol), and spotted. TLC plates were developed for 1 h (80:20:2 chloroform/methanol/water) and exposed to a phosphor storage plate, which was scanned for radioactivity.

*Synthesis of 6-Azido-\[^{14}C\]TMM and Enzyme Reactions—*200 μl of \[^{14}C\]TMM (100 μCi/μl in 2:1 chloroform/methanol) was concentrated under a stream of argon and was resuspended in 200 μl of phosphate buffer solution of 6-azido-trehalose (500 μM) to which was added 100 μl of Ag85C (20 μM), and the reaction was vortexed and allowed to proceed overnight at 37 °C. Extraction and chromatography of the reaction were carried out as described above. A full-sized image of the TLC plate was printed on a clear transparency film, and the 6-azido-TMM product was scraped and redissolved in 800 μl of solvent (5 μCi/μl, 2:1 chloroform/methanol). Enzyme reactions for mycolation of Ara6 were conducted as with \[^{14}C\]TMM with the exception that 200 μl of 6-azido-\[^{14}C\]TMM (1000 μCi) was used in place of \[^{14}C\]TMM.

*Synthesis of Ara6—*Synthesis of Ara6 was conducted from the intermediate pentasaccharide that has been reported previously (Scheme 1, compound 2 has been described previously (32)). Reactions were carried out in oven-dried glassware. All reagents used were purchased from commercial sources and were used without further purification unless noted. Reaction solvents were purified by successive passage through columns of aluminum oxide and copper under nitrogen. Unless stated otherwise, all reactions were carried out at room temperature under a positive pressure of argon and were monitored by TLC on Silica Gel 60 F254 (0.25 mm; Merck). Spots were detected under UV light or by charring with acidified *p*-anisaldehyde solution in EtOH. Unless otherwise indicated, all column chrom-

M. tuberculosis Ag85 Isoforms Have Unique Activities

SCHEME 1. Synthesis of hexasaccharide acceptor Ara6. DBU, 1,8-diazabicyclo[5.4.0]undec-7-ene; TMSOTf, trimethylsilyl trifluoromethanesulfonate.

matography was performed on silica gel (40–60 μM). The ratio between silica gel and crude product ranged from 100 to 50:1 (w/w). ¹H NMR spectra were recorded at 500 MHz, and chemical shifts were referenced to TMS (0.0, CDCl₃). ¹H data were reported as if they were first order. ¹³C NMR (attached proton test) spectra were recorded at 125 MHz, and ¹³C chemical shifts were referenced to internal CDCl₃ (77.23, CDCl₃). Organic solutions were concentrated under vacuum at <40 °C. Methyl 5-O-{3,5-di-O-(2-O-[2,3,5-tri-O-benzoyl-β-D-arabinofuranosyl]-3,5-di-O-benzoyl-α-arabinofuranosyl)-2-O-benzoyl-α-D-arabinofuranosyl}-2,3-di-O-benzoyl-α-D-arabinofuranoside (5). To a solution of alcohol 2 (0.238 g, 0.13 mmol) and trichloroacetonitrile (0.15 ml, 1.4 mmol) in CH₂Cl₂ (5 ml) at 0 °C was added 1,8-diazabicyclo[5.4.0]undec-7-ene (1 drop). The reaction mixture was stirred at 0 °C for 30 min and then warmed to room temperature over 30 min. The solvent was then removed under vacuum, and a solution of dry hexane/toluene (2:3, 10 ml) was added. After being stirred for 5 min, this solution was quickly filtered through a short column of silica gel and Na₂SO₄ (1:1). The resulting solution was then concentrated to yield the trichloroacetimidate derivative 3 (183 mg), which was used for glycosylation without any further purification. Alternatively, the syrupy residue obtained after the initial solvent evaporation following the reaction could be filtered quickly through a silica gel column (3:2 hexanes/EtOAc with 0.15% Et₃N). The fractions containing the trichloroacetimidate derivative were concentrated, dried under high vacuum, and used immediately without any further purification. The trichloroacetimidate derivative 3 in CH₂Cl₂ (3 ml) was added to a solution of alcohol 4 (0.07 g, 0.18 mmol) in CH₂Cl₂ (4 ml) containing 4-Å molecular sieves (0.07 g; stirred already for about 30 min) at −20 °C. A solution of trimethylsilyl trifluoromethanesulfonate (1.2 μl, 0.0065 mmol) in CH₂Cl₂ (50 μl) was added dropwise over a period of 5 min. The reaction mixture was then warmed to −9 °C over a period of 25 min and quenched by the addition of Et₃N (3 drops). The solution was diluted with CH₂Cl₂ (15 ml) and filtered. The filtrate was concentrated to a syrup that was purified by column chromatography (2:1 hexanes/EtOAc) to afford 5 (163 mg, 81%) as a thick syrup: Rf 0.3 (2:1 hexanes-

Molecular Dynamics—Crystal structures of the *M*. tuberculosis Ag85A (PDB entry 1SFR), Ag85B (PDB entry 1F0N), and Ag85C

M. tuberculosis Ag85 Isoforms Have Unique Activities

A                                                                 B                                                                 C
P261-F228 Loop                                                                 Active Site                                                                 10    20    30    40    50    60
Ag85C-S124                                                                 Ag85A, B-S124                                                                 Ag85A SRPGLPVEYLQVPSPSMGARDIKVQFQSGGANSPALYLLDGLRAQQDFSGWDINTPAFEWY
Ag85C-H260                                                                 Ag85A, B-H262                                                                 Ag85B SRPGLPVEYLQVPSPSMGARDIKVQFQSGGNNSPAVYLLDGLRAQQDDYNQWDINTPAFEWY
Ag85C-E228                                                                 Ag85A, B-E230                                                                 Ag85C SRPGLPVEYLQVPSSMGRDIKVQFQGGG-PHAVYLLDGLRAQQDDYNQWDINTPAFEEY
Secondary                                                                 70    80    90   100   110   120
Site                                                                 Ag85A DQSGLSVVMPVGGQSSFYSDWQPACGKAGCQTYKWETFLTSELPGWLQANRHVKPTGSA
                                                                 Ag85B YQSGLSI VMPVGGQSSFYSDWS PACC KAGCQTYKWETFLTSELPQWL SANRA VKPTGSA
                                                                 Ag85C YQSGLSVI MPVGGQSSFYTDWYQPSQSNGQNYTYKWETFLTREMPAWLQANKGV SPTGNA
                                                                 130   140   150   160   170   180
                                                                 Ag85A VVGLSMAASSALT LAIYHPQQFVYAGAMSGLL DPSQAMGPTLI GLAMGDAGGYKASDMVG
                                                                 Ag85B AIGLSMACSSAM LAA YHPQQFYAGLS SALL DPSQGMGPSLI GLAMGDAGGYKAA DMVG
                                                                 Ag85C AVGLSMS GGSAL LAA YYPQQFPYAA SL SGFL NPESEGWWPTLI GLAMNDSGGY NANSMWG
                                                                 **                                                                 190   200   210   220   230   240
                                                                 Ag85A PKEDPAWQRNDP LLNVGKL IANNTRVVWYOGNGKPSDLGGNNLPAKFLEGFVRTSNI KFQ
                                                                 Ag85B PSSDPAWERNDPTQQIPKL VANNTRWYWCGNGTPNELGGAN PAEFLENFVRSSNI KFQ
                                                                 Ag85C PSSDPAWKRNDPMVQIPRL VANNTRIWYCNGNTPSDLGGDNIPAKFLEGLTLRTNQTFR
                                                                 **                                                                 250   260   270   280   290
                                                                 Ag85A DAYNAGGGHN GVDFDPDSGTHSWEYG AQLNAMKPD LQRAL- GATPNTGPAPQGA
                                                                 Ag85B DAYNAAGGHN AVFNFPPNGTHSWEYG AQLNAMKGDL QSSL- GAC
                                                                 Ag85C DTYAADGGR NGVFNFPPNGTHSWPYNEQLVAMKADI QHVL NGATPPAAPAAPAA


FIGURE 2. Previously reported structures and sequence alignments of the Ag85s, showing the high homology between the Ag85A, -B, and -C isoforms. PDB entries 1F0N, 1DQY, and 1DQZ are shown. A, the difference in the position of helix α9 (apo-structure 1DQZ (blue) and diethylphosphate acylated 1DQY (cyan)), adjacent Pro<sup>216</sup>-Phe<sup>228</sup> loop (blue/cyan), and key amino acids from Ag85C (Trp<sup>157</sup>, Trp<sup>158</sup>, and Thr<sup>231</sup>; yellow) and from Ag85A/B (Met<sup>159</sup>, Phe<sup>232</sup>, and Val<sup>233</sup>; green), highlighting the differences between Ag85C and Ag85A/B at a secondary site that has been found with bound trehalose (1F0N, orange carbon, red oxygen atoms) or octylthioglucose (not shown). The catalytic triad is highlighted in magenta. B, the movement of helix α9 in A results in a disruption of the hydrogen bonding of the catalytic triad (dotted lines). The hydrogen-bonded triad in Ag85C structure 1DQZ is shown in magenta, and the disrupted triad in 1DQY is shown in cyan. C, sequence alignment of Ag85A, -B, and -C indicates the highly conserved nature of the enzymes. The box indicates sequence corresponding to helix α9 and the Pro<sup>216</sup>-Phe<sup>228</sup> loop, and the catalytic triad is indicated with arrows. Stars indicate the amino acids highlighted in A. Cyan, homology with Ag85A; magenta, homology with or unique to Ag85B; white, amino acids that are unique to Ag85C. Sequences are shown without signal peptides, and sequence numbering has been made consistent with that used previously (39–41).

(PDB entry 1DQZ) were mutated or left unchanged and explicitly solvated with TIP3P water molecules and Na<sup>+</sup> and Cl<sup>-</sup> counterions using the VMD program (34). Molecular dynamics simulations were performed under isobaric-isothermal conditions with periodic boundary conditions using the NAMD (35) program (version 2.8) on the Biowulf Linux cluster at the National Institutes of Health (Bethesda, MD). Electrostatic interactions were calculated using the particle-mesh Ewald summation. The CHARMM27 (36) force field was used with CHARMM atom types and charges. Prior to the start of a simulation, an energy minimization was performed using a conjugate gradient method, followed by slow warming to 310 K in 10-K increments with each increment equilibrating over 5 ps. Once the 310 K target temperature was reached, each system was equilibrated for an additional 50 ps. Production runs were conducted at 310 K for 100 ns with data collected at 1-ns intervals. For all simulations, a 1-fs integration time step was used along with a 12 Å non-bonded cut-off. Langevin dynamics were used to maintain temperature, and a modified Nosé-Hoover Langevin piston was used to control pressure.

**Molecular Dynamics Analysis—Structures taken from molecular dynamics simulations at 1-ns intervals were aligned to each other, and an average structure was obtained using the UCSF Chimera package (37). Pairwise RMSDs of all C-α atoms were measured, and average RMSD values were added to the B-factor column of the averaged structure. The B-factor putty option was used in the PyMOL program (38) to generate tube widths and colors that correspond to the RMSD values.**

**RESULTS**

**Analysis of Structural Differences among Published Crystal Structures of the Three Ag85 Proteins—The co-crystal structure of Ag85C bound to diethyl phosphate, a serine-reactive, covalent inhibitor, is the most divergent of the highly related Ag85**

structures published to date (PDB entry 1DQY) (Fig. 2). Amino acids that differ between the Ag85 isoforms cluster to several regions, including at a secondary ligand binding site adjacent to the active site that has been serendipitously found occupied by trehalose, S-octyl-glucose, and glycerol in various structures (39–41) (Fig. 2A). Additionally, several amino acids located on helix α9 and the adjacent helix α5 are not conserved between Ag85A, -B, and -C but are highly conserved across related mycobacterial species (Fig. 2C) (data not shown). Of particular interest in Ag85C are two tryptophan residues (Trp<sup>157</sup> and Trp<sup>158</sup>; yellow in Fig. 2A) that are replaced by methionine and glycine (Met<sup>159</sup> and Gly<sup>160</sup>; green) in Ag85A/B. On the adjacent helix, leucine and threonine in Ag85C (Leu<sup>230</sup> and Thr<sup>231</sup>; yellow) are replaced by phenylalanine and valine in Ag85A/B (Phe<sup>232</sup> and Val<sup>233</sup>; green).

1DQY (Ag85C with Ser<sup>124</sup> bound to diethylphosphate) shows a significant straightening of helix α9 (39, 40, 42) (Fig. 2). This helix and the adjacent loop (blue) move substantially upon covalent modification, straightening the helix and folding the loop (cyan) into the active site, which in turn causes the catalytic glutamate (integral to helix α9) and the catalytic histidine to move away from the active site, disrupting the hydrogen bonding within the catalytic triad (Fig. 2B).

**Mutagenesis of Isoform-specific Secondary Site Residues Modulates Acyltransferase Activity Levels—We theorized that the cluster of non-conserved residues that abut the secondary binding site could be partially responsible for differences in enzyme activity. Because Ag85B has been reported as the least active isoform (10–20-fold less activity) and Ag85C has been reported as the most active in acyltransferase assays, our initial mutagenesis efforts focused on understanding the effect of converting this region of Ag85C into that found in Ag85B and vice versa. We produced wild type and chimeric proteins**

M. tuberculosis Ag85 Isoforms Have Unique Activities

A
![Image A](image_a.png)

B
![Image B](image_b.png)

C
![Image C](image_c.png)

D
![Image D](image_d.png)

E
![Image E](image_e.png)

FIGURE 3. Site-directed mutagenesis revealed that non-conserved amino acids in a secondary site of Ag85 control the relative rates of acyl transfer and may explain the previously observed differences in catalytic activity between Ag85A, -B, and -C. A–C, MS acyltransferase assays measuring the acyl transfer from TDH to trehalose (Tre) to generate trehalose monohexanoate (TMH). Selected amino acids were replaced with either the amino acids from the other Ag85 isoform (e.g. Ag85A/B residues in Ag85C) or alanine. A, Ag85C mutants; B, Ag85B mutants; C, Ag85A mutants. D and E, acyl transfer assays with the native trehalose monomycolate (\[^{14}C\]-TMM) substrate in the presence or absence of trehalose or 6-azido-trehalose as competitive substrates demonstrates a previously unknown difference in substrate specificity between Ag85A, -B, and -C, where C exhibits increased mycolation of free sugars, and A and B are more selective for TMM mycolation even in the presence of high concentrations of competing trehalose (200 μM). D shows the Ag85 TMM assay in the absence or presence of trehalose. E, TMM assays are conducted in the presence of competitive substrate 6-azido-trehalose (200 μM), which cannot form TDM. Ag85C, again, shows increased mycolation rates of the free sugar, whereas Ag85A, even in the presence of high concentrations of competitive substrate, selectively forms TDM. TLCs were developed in 80:20:2 chloroform/methanol/water.

as His-tagged fusions and purified them by immobilized metal affinity chromatography. For all mutant and chimeric proteins, no significant difference in production level in Escherichia coli was observed, and all mutants were checked for proper folding using circular dichroism (CD) (data not shown).

Site-directed mutagenesis of Ag85C was directed initially at the two tryptophan residues at positions 157 and 158. Activity was assessed using a homogeneous, soluble *in vitro* enzyme assay (26), where trehalose and TDH are mixed with Ag85 and trehalose monohexanoate formation was monitored by MS. Mutating Trp\(^{157}\) to alanine in Ag85C caused a doubling of activity, whereas mutation of Trp\(^{158}\) to alanine resulted in a 10-fold reduction in activity to levels comparable with that

observed in wild type Ag85B (Fig. 3A). Replacing both Trp\(^{157}\) and Trp\(^{158}\) with alanine also resulted in a relatively inactive enzyme. The Ag85C W157M/W158G double mutant (the wild type residues of Ag85A/B) was slightly more active than the double alanine mutant but still not very active. Further mutation of Leu\(^{230}\) to alanine or phenylalanine in the double W157A/W158A mutant also had little effect on catalytic activity; nor did adding the L230F to the W157M/W158G to restore the full Ag85A/B MGF sequence in this region alter the low level activity observed with wild type Ag85B.

In contrast, mutation of these three residues on helix α9 and helix α5 in either Ag85A or -B to the corresponding residues occurring in Ag85C resulted in a 2–10-fold enhancement in activity (Fig. 3, B and C). In both Ag85A and Ag85B, introduc-

tion of all three mutations simultaneously (M159W, G160W, and F232L (WWL)) resulted in a further enhancement of activity to levels nearly identical to Ag85C. Thus, these mutations in the secondary site of Ag85, notably positions Trp<sup>157</sup>, Trp<sup>158</sup>, and Leu<sup>230</sup> (Met<sup>159</sup>, Gly<sup>160</sup>, and Phe<sup>232</sup> in Ag85A/B), contribute directly to controlling the catalytic activity of the antigen 85 enzymes.

**Excess Trehalose Induces Loss of Activity in Ag85C—**

Although moderate throughput, homogeneous, and quantitative, the Ag85MS assay is limited by the short, synthetic nature of the lipid substrates attached to trehalose. To more fully understand the effects of these mutations on the activity of the Ag85 isoforms, we utilized an additional Ag85 enzyme assay, where \[^{14}C\]TMM (purified from *M. tuberculosis* grown in the presence of \[^{14}C\]acetic acid) was used as the mycolyl donor in *in vitro* reactions. This assay is similar to previously reported assays (13, 43) in which purified TMM was used in enzyme reactions with \[^{14}C\]trehalose, but in this case, the mycolate is carrying the label and not the disaccharide. Using lipid-labeled \[^{14}C\]TMM allowed us to monitor the transmycolation of two TMMs to form TDM and free trehalose independent of the initial mycolation of trehalose to generate \[^{14}C\]TMM. When Ag85A, -B, or -C was added to a suspension of \[^{14}C\]TMM, \[^{14}C\]TDM was formed, and this could be monitored by TLC (Fig. 3D). Reactions were allowed to proceed overnight to evaluate the formation of product at thermodynamic equilibrium. Although steady state kinetics could not be determined due to poor solubility of \[^{14}C\]TMM and the low concentrations of actual substrate used, Ag85A, -B, and -C were all able to catalyze the formation of \[^{14}C\]TDM with Ag85C, producing very slightly less TDM than the other two isoforms. However, when 500 μM trehalose was added to the reaction mixture, Ag85C converted less than half of the TMM substrate in the reaction mixture to TDM product (as quantified by phosphorimaging). In contrast, Ag85A and Ag85B maintained the same level of \[^{14}C\]TDM formation despite the presence of trehalose in the reaction mixture. This result suggests the possibility that occupancy of the second binding site of Ag85C with trehalose substantially reduces the catalytic activity of the enzyme. In contrast, Ag85A and -B were relatively insensitive to the presence of trehalose. We also incubated purified \[^{14}C\]TDM directly with Ag85C in the presence of excess unlabeled trehalose and observed negligible conversion of TDM to TMM (data not shown). Substituting in the three Ag85C residues (WWL) to Ag85B did slightly diminish the ability of this enzyme to convert \[^{14}C\]TMM to \[^{14}C\]TDM, but this effect was not augmented by the addition of trehalose. Likewise, the triple mutant of Ag85C bearing the wild type residues of Ag85B (MGF) had intermediate activity that was only somewhat affected by the addition of trehalose.

**Ag85C Shows Higher Tolerance for the Altered Substrate 6-Azido-trehalose—**

We also explored using 6-azido-trehalose, which cannot undergo a second acylation reaction, as a surrogate for trehalose to detect transmycolation reactions between TMM and trehalose. Incubation of 6-azido-trehalose and \[^{14}C\]TMM with Ag85C formed nearly equivalent amounts of TDM and 6-azido-trehalose-6′-monomycolate (6-Az-TMM) (Fig. 3E). In contrast, Ag85A continued to produce TDM efficiently and generated only a small amount of monomycolated 6-azido-trehalose. Ag85B demonstrated an intermediate activity, forming TDM preferentially but still forming an appreciable quantity of 6-azido-trehalose-6′-monomycolate. Similar to the results with trehalose, substituting the three residues that differ between Ag85C and Ag85A/B did not recapitulate this differential activity toward 6-azido-trehalose (Fig. 3, D and E).

**Molecular Dynamics Simulations Suggest Additional Determinants of Substrate Specificity—**

Because the Trp<sup>157</sup>, Trp<sup>158</sup>, and Leu<sup>230</sup> mutants failed to fully explain the observed differences in substrate specificity toward trehalose, we turned to molecular dynamics (MD) simulations to further understand the dissimilarities between the Ag85 isoforms.

MD simulations for all three of the Ag85 isoforms revealed that the N-terminal portion of helix α9, the Pro<sup>216</sup>-Phe<sup>228</sup> loop, and helix α5, all of which comprise the ligand entry site, are capable of significant conformational changes for all isoforms (Fig. 4). However, Ag85B and particularly Ag85A demonstrated greater overall helical motion for both α9 and α5. Simulations of the Ag85C Trp<sup>157</sup>, Trp<sup>158</sup>, and Leu<sup>230</sup> and the corresponding Ag85A/B Met<sup>159</sup> Gly<sup>160</sup> and Phe<sup>232</sup> mutants revealed large changes in helical flexibility when these secondary site residues are mutated, consistent with the different conformations observed from x-ray crystallography experiments (Fig. 2).

Additionally, MD simulations revealed that another amino acid in the secondary site (Thr<sup>231</sup> in Ag85C, Val<sup>233</sup> in Ag85A/B) was probably involved in determining helix flexibility and that mutagenesis at this residue might further explain the observed differences. The bend in helix α9 occurs at Thr<sup>231</sup> for Ag85C and at Val<sup>233</sup> in Ag85A/B. *In silico* mutation of position 231 in Ag85C was followed by energy minimization and a 100-ns molecular dynamics simulation at 310 K. The mutant in which Thr<sup>231</sup> was replaced with alanine revealed a dramatic reduction in flexibility of this region and very stable short distances between the catalytic serine and histidine residues of the active site (Fig. 4). Substitution at this position with the Ag85A residue, T231V, modestly increased the flexibility and motion of helix α9.

Mutation of Val<sup>233</sup> in Ag85A appears to also dramatically alter helix flexibility and triad stability. In contrast with the stabilizing affect that the alanine mutation had in Ag85C, the Ag85A V233A mutant resulted in increased helical motion and large mean distances among residues of the catalytic triad. Substitution at this position with the Ag85C residue, V233T, dramatically decreased overall motion of both helices, conferring Ag85C-like motion on Ag85A.

**Mutations at the Base of Helix α9 Affect Substrate Selectivity—**

To test our molecular dynamics predictions that single mutations at the hinge of helix α9 could partially interconvert the Ag85 isoforms, we generated a series of mutants in both Ag85A and C and tested them for changes in activity in our Ag85 assays. Mutations to Ala, Thr/Val, Leu, and Ile were conducted in both Ag85A and -C, resulting in dramatic changes in enzyme activity and substrate selectivity (Fig. 5).

For the T231A mutant of Ag85C, which had been predicted to have much less flexibility in this region and a much tighter active site, the overall rate of acylation was double that of the wild type enzyme, but even more strikingly, the rate of hydrolysis of TDM to TMM was reduced by over 10-fold.

M. tuberculosis Ag85 Isoforms Have Unique Activities

A
Ag85C 1DQZ
1DQZ/W157M
W158G L230F
1DQZ/T231V
1DQZ/T231A

B
Ag85A 1SFR
1SFR/M159W
G160W F232L
1SFR/V233T
1SFR/V233A

FIGURE 4. Molecular dynamics simulations show that mutations at Thr²³¹/Val²³³ alter the helix flexibility and the hydrogen bonding of the catalytic triad. A, degree of motion for Ag85C (PDB entry 1DQZ) and mutants (W157M, W158G, L230F, and T231V/A) measured by RMSDs of C-α atoms from molecular dynamics trajectories. Thicker tubes correspond to a higher RMSD. Side chains for residues forming part of the catalytic triad are represented as sticks. B, mutations at the analogous position in Ag85A (M159W, G160W, L233F, and V233T/A) result in changes in flexibility at the base of helix 9 and adjacent loop (residues Pro²¹⁶–Phe²²⁸) or nearby helix α5. V233T mutation confers Ag85C-like helical flexibility.

ysis was 5 times higher than that of the wild type enzyme. The Ag85A V233A mutant also shows a moderate increase in acyltransferase activity and a 2-fold increase in hydrolase activity, suggesting that possible changes in helix α9 flexibility may be linked to acyl enzyme stability or accessibility of the active site to water. As expected in Ag85C, mutation of the bulkier valine, leucine, and isoleucine all afford a 2–3-fold decrease in relative rates of acylation and hydrolysis. Likewise, incorporation of the bulky isoleucine residue into Ag85A also resulted in a decrease in the rate of acylation and hydrolysis. In the MS assay, mutation to leucine results in a large drop in activity for Ag85C and a modest increase in acyl transfer and large increase in hydrolysis for Ag85A.

In the 6-azido-trehalose [¹⁴C]TMM enzyme assays, the Ag85C T231V and T231L and T231I mutations all yielded a decrease in the production of 6-azido-TMM (Fig. 5C), supporting a model in which a change in Ag85C helix flexibility caused by the bulkier amino acid decreases the mycolation of free trehalose. The T231L and T231I mutations also induce an increase in TDM production, consistent with a shift toward Ag85A-like activity. In contrast, the Ag85C T231V mutation leads to retention of TMM but no dramatic increase in TDM production, which may reflect an intermediate phenotype. In Ag85A, the V233A and V233T mutations double the formation of TMM-6-azide and modestly decrease the formation of TDM, again consistent with a model in which inclusion of the Ag85C-like residues at this position leads to increased mycolation of free trehalose. For both Ag85A and -C, single mutations at the helix α9 bend dramatically change both the enzyme activity and substrate selectivity, which suggests that the degree of

bend of helix α9 explains the difference in Ag85A, -B, and -C enzyme activities.

Potential Differences in Ag85 Isoform-specific Mycolation of the M. tuberculosis Cell Wall—One biologically plausible reason for the observed relationship of flexibility and activity may be related to constraints imposed by mycolation of the terminal arabinose units of the insoluble heteropolymeric arabinogalactan, an important second function of the Ag85 enzymes (Fig. 1). Ara6 (Fig. 6), the hexasaccharide terminus of the arabinogalactan polymer (44), was synthesized and assayed as a mycolyl acceptor in Ag85 assays. Docking studies with the related Ara5 suggested that this molecule should bind into an extended active site on Ag85 (Fig. 6, A and B), with the free 5-OH pointing toward the helix α9 groove. The already acylated mycolate in TMM or Ara6MM should bind along this hydrophobic helix. Thus, the flexibility of the helix and steric bulk of the amino acids would be critical for regulating binding and possibly the release of the substrate. Upon reaction with Ag85 and [¹⁴C]TMM, a new mycolate-containing lipid species (asterisk) was observed (Fig. 6C). In the absence of Ara6, this species was not formed, although several minor, comigrating species were. To determine whether this species was possibly Ara6MM, 6-azido-[¹⁴C]TMM was purified and used as a donor to mycolate the Ara6. Under the 6-azido-[¹⁴C]TMM reaction conditions, exclusively this new solvent-font lipid species (asterisk) was generated, and in the absence of Ara6, no new species was observed. When 6-azido-[¹⁴C]TMM is used as a mycolate source and no TDM is generated, all three Ag85 isoforms appear to generate equal proportions of the new lipid (Fig. 6C, three right-hand lanes). We found that the new lipid species

25048 JOURNAL OF BIOLOGICAL CHEMISTRY
ASBMB
VOLUME 289 • NUMBER 36 • SEPTEMBER 5, 2014

M. tuberculosis Ag85 Isoforms Have Unique Activities

A

Relative Rates (%)

| Relative Rate Acylation | Relative Rate Hydrolysis |
|-------------------------|--------------------------|
| 500                     |                          |
| 400                     |                          |
| 300                     |                          |
| 200                     |                          |
| 100                     |                          |
| 0                       |                          |

Ag85C T231A T231V T231L T231I

B

Relative Rates (%)

| Ag85A | V233A | V233T | V233L | V233I |
|-------|-------|-------|-------|-------|
| 100   | 250   | 50    | 150   | 50    |

C

6-Az-TMM →
+6-azido-trehalose

D

6-Az-TMM →
+ 6-azido-trehalose

FIGURE 5. Differences in flexibility of the residues comprising the base of helix α9 and an adjacent loop and helix α5 correlate with changes in substrate specificity and disparity in catalytic activity. A and B, relative rates of acylation and hydrolysis were calculated as in Fig. 2. Mutations to Thr<sup>231</sup> in Ag85C (A) dramatically alter the enzyme acyltransferase and acyl hydrolase activity in the trehalose dihexanoate assay as do mutations to the corresponding position in Ag85A (Val<sup>233</sup>) (B). C, mutation of Ag85C Thr<sup>231</sup> to the bulkier Val, Leu, or Ile decreases the relative formation of 6-azido-TMM, suggesting that structural changes in helix α9 and nearby regions determine substrate specificity. D, mutation of Ag85A to the less bulky Ag85C-like residues V231T and V231A increases formation of 6-azido-TMM. The Ag85A M159W/G160W/F232L (WWL) mutant demonstrates decreased formation of TDM but does not cause increased mycolation of 6-azido-TMM. TLCs were developed in 80:20:2 chloroform/methanol/water.

could be resolved from all other species by exposing the TLC plate to two rounds of chromatography; first, separation of TDM and TMM was achieved on two-thirds of the plate (80:20:2 chloroform/methanol/water), and then the solvent front lipids were allowed to further separate on the front one-third of the plate (95:5 chloroform/methanol (Fig. 6, D and E). We postulated that the addition of high concentrations of competing trehalose (1 and 10 mM) to the reaction mixture might increase the mycolation of Ara6 by reducing the formation of the dead end TDM; however, in practice, additional trehalose did not have this effect (data not shown), although it did increase the retention of TMM in the reaction mixture. Ag85A produced more of the new lipid, both in the presence and absence of trehalose as a competitive substrate in the reaction mixture (Fig. 6) (data not shown). Due to the previously documented (45, 46) arabinan release upon treatment with the first line antimycobacterial drug EMB, we postulated that EMB treatment of

M. tuberculosis might release mycolated fragments of arabinan of similar size to the novel species observed when Ara6 was subjected to the mycolation assay. As hypothesized, EMB treatment caused the production of a novel lipid species (double asterisk), which showed approximately the same apparent Rf as the lipid generated in the from the Ag85 mycolation assay in the presence of Ara6 (asterisk) (Fig. 6, F and G).

DISCUSSION

Previously reported differences in relative Ag85 catalytic activity toward trehalose have always stood in contrast to the extreme similarity of these proteins (13, 26). Ag85C has been reported to have a higher k<sub>cat</sub> for the mycolation of free trehalose relative to Ag85B, and here we report that, in contrast with mycolation of trehalose, both Ag85A and -B show an increased capacity to mycolate TMM to TDM relative to Ag85C. Consis-

M. tuberculosis Ag85 Isoforms Have Unique Activities

A

C

B

D

E

F

G

FIGURE 6. The Ag85 substrate specificity may extend toward fragments of the mycobacterial cell wall, suggesting that the observed differences in enzyme activity may have implications in the partitioning of exported mycolates between TDM and the arabinan. A, Ara5 is docked in the active site of Ag85A (PDB entry 1SFR), revealing an extended active site that accommodates the bulk of the larger sugar. Residues are colored as in Fig. 1A. B, structures of docked Ara5 and assayed Ara6 are shown. My, mycolate attachment site. C, Ag85 TMM assays in the presence of Ara6 (500 μM) and competing trehalose (200 μM) reveal the formation of a new lipid species (*). Ag85A is most efficient at producing this new species from TMM. The new lipid (*) was the only significant product from the transesterification from TMM-6-azide to Ara6. TLC was developed in 80:20:2 chloroform/methanol/water. D, Ag85A-catalyzed formation of the new lipid species (*) in the absence (i) and presence (ii) of Ara6. E, the same TLC shown in D developed a second time in 95:5 chloroform/methanol to separate the solvent front lipid species, as indicated by the arrow. F, H37Rv M. tuberculosis treated with EMB at 10 times the minimum inhibitory concentration shows the accumulation of a new lipid species (**). TLC was developed in 100:10:0.1 chloroform/methanol/water. SQ109, which blocks the export of TMM, abolishes the formation of this species (**). G, the new lipid (**) in the EMB-treated 24-h sample from F has an Rf comparable with that of the product of the Ag85-catalyzed mycolation of Ara6 (*). TLC was developed twice first in 80:20:2, chloroform/methanol/water and then in 95:5 chloroform/methanol. Mycolation of Ara6 could not be increased further with the addition of higher concentrations of trehalose.

tent with this finding, knockouts of ag85A have been shown to have decreased TDM levels (47).

The high level of sequence conservation among these orthologs caused us to focus on the few areas of divergence starting with amino acids along helix α9, and nearby helix α5, adjacent to the active site. Mutation of the three non-conserved residues in this region in Ag85A, -B, or -C (Trp<sup>157</sup>, Trp<sup>158</sup>, and Leu<sup>230</sup> in Ag85C and Met<sup>159</sup>, Gly<sup>160</sup>, and Phe<sup>232</sup> in both A and B isoforms) to alanine or their corresponding residues from the each other altered the rates of acyl transfer and acyl hydrolysis with shorter TDH substrate but did not affect the degree of trehalose mycolation from TMM. The presence of two tryptophans in the secondary site of Ag85C (Trp<sup>157</sup> and Trp<sup>158</sup>) and absence in Ag85A/B suggests a carbohydrate binding role by

π-CH<sub>2</sub> interactions (48), which may in part explain the greater propensity for Ag85C to transesterify free trehalose when compared with Ag85A/B. However, mutation of these residues alone does not alter substrate specificity. Mutations of the helix α9 hinge do alter the substrate specificity, indicating that both carbohydrate binding at the secondary site and the related flexibility of helix α9 are required to explain the differences in rates and substrate selectivity of the Ag85 enzymes. Mutation of a single amino acid, Thr<sup>231</sup> (Ag85C) or Val<sup>233</sup> (Ag85A), partially interconverts enzyme substrate specificity. Introduction of bulkier Ag85A-like residues at Thr<sup>231</sup> (Val, Leu, or Ile) into Ag85C dramatically decreases mycolation of trehalose and 6-azido-trehalose. In contrast, introduction of the less bulky Ag85C-like residues into Ag85A at Val<sup>233</sup> (Ala or Thr) affords a

25050 JOURNAL OF BIOLOGICAL CHEMISTRY

VOLUME 289 • NUMBER 36 • SEPTEMBER 5, 2014

M. tuberculosis Ag85 Isoforms Have Unique Activities

2-fold increase in mycolation of 6-azido trehalose. Mutation of Val²³³ and Thr²³¹ also affects the relative rates of acyl transfer and acyl hydrolase activity, as measured by the mycolation of free trehalose from trehalose dihexanoate. Molecular dynamics links these key non-conserved residues to the flexibility of helices α9 and α5 and the catalytic triad position. Mutations that alter the flexibility of these helices dramatically alter the substrate selectivity, partially converting Ag85A to -C and vice versa. These results suggest that the secondary site in Ag85 is critical for the regulation of enzyme activity and substrate selectivity.

These results also allow us to propose a possible mechanism by which Ag85 activity is regulated by this secondary site. First, Ag85 binds to TMM at the active site, and transfer of the mycolyl chain from TMM to the enzyme occurs, forming the acyl enzyme intermediate. This transmycolation causes both a destabilization of the catalytic triad hydrogen bonding network and a concurrent straightening of helix α9 and folding of the Pro²¹⁶-Phe²²⁶ loop over the active site. The acyl enzyme is protected from hydrolysis by the disrupted triad and loop folded over the active site. Although the Ag85 enzymes show acyl hydrolase activity toward shorter trehalose esters, we observe negligible hydrolase activity toward TMM and TDM, suggesting that the acyl enzyme is very resistant to hydrolysis. When the acyl enzyme encounters either trehalose or TMM, these molecules bind at the secondary site, and this binding event is relayed to the active site by the reformation of the bend in α9 and restoration of the triad hydrogen bonding, mediated by Thr²³¹/Val²³³. Once bound, the trehalose or TMM can move into the active site from the secondary site and be transesterified, producing either TMM or TDM. Any mutations that alter the flexibility of helix α9 or the residues in the secondary carbohydrate binding pocket affect the substrate specificity and relative activity of Ag85. Although some polysaccharide processing enzymes have been shown to contain secondary carbohydrate binding sites, to our knowledge, this is the first mechanistic example of how carbohydrate binding at a second site can directly control enzyme activity and substrate specificity (49).

The biological relevance of the existence of three Ag85 isoforms and their differences in relative catalytic activities remains to be fully understood. During the synthesis of the M. tuberculosis cell wall, the transmycolation reaction can occur from TMM to a second unit of TMM, generating TDM, or from either TMM or TDM to free trehalose or to the cell wall arabinogalactan polymer. Both TDM formation and anchoring mycolates to the cell wall constitute structurally important endpoints that probably represent mycolate sinks; although reversible in theory, the high lipophilicity and insolubility of these molecules probably means that mycolate flow terminates with their formation. The combined evidence suggests that one plausible scenario is that the primary physiologic function of Ag85C, which has the least flexibility in helices α5 and α9, is mycolylation of TMM to form TDM. Additionally, Ag85C may play a role in the initial mycolation of trehalose 6-phosphate from putative mycolyl carrier species, such as mycolylphospholipid (50), to generate TMM within the cell. We speculate that the flexibility in these two helices may be important for inter-

action with the insoluble cell envelope and therefore that Ag85A and -B may be involved in mycolylation of the terminal arabinose units of the mature cell envelope. Alternatively, the observed difference in enzyme activity may reflect a means to partition TMM to both the cell wall and TDM, whereby Ag85C introduces the initial mycolate into each Ara6 motif, and subsequently Ag85A and -B are responsible for the addition of the remaining mycolates.

In order to more fully understand the differences in Ag85-catalyzed mycolation of the mycobacterial cell wall, we used a short arabinan substrate in TMM-mediated mycolation assays. In the presence of Ara6 and TMM, Ag85A catalyzed the formation of a single new lipid species, putatively mycolated Ara6, with Ag85B and -C producing less of the new lipid. Although we had expected to observe formation of multiple mycolation states of the Ara6, the appearance of a single band suggests either processivity in the mycolation reaction or the addition of only a single mycolic acid. Such processivity is not unprecedented, because ~75% of the reducing termini of the M. tuberculosis cell wall are fully mycolated, whereas the rest are completely unmycolated (44). Although we have been unable to fully characterize this new lipid species, despite considerable effort, lipid extracts generated from H37Rv M. tuberculosis, treated with EMB, a known inhibitor of arabinan synthesis that releases arabinan fragments, contain a product with a similar mobility on TLC (5, 45, 46, 51). Product inhibition by TDM may reduce arabinan mycolation, particularly by Ag85C, as suggested by equal formation of the new lipid in the absence of TDM by all isoforms when 6-azido-[¹⁴C]TMM is used as a mycolate source.

To date, few Ag85 inhibitors have been reported, and none have demonstrated inhibitory properties *in vivo*. The discovery that Ag85A and -B will generate large quantities of TDM even in the presence of high concentrations of competing trehalose may in part explain this paucity of inhibitors, because most groups have focused on the synthesis of trehalose analogues in attempts to generate Ag85 inhibitors. The ability of relatively subtle single amino acid changes at the secondary site of Ag85 to dramatically alter the catalytic activity of the enzymes suggests that, in addition to the active site, this portion of the enzyme may offer an alternative ligandable site that perturbs catalytic activity. Dissection of the individual roles of the Ag85 enzymes in M. tuberculosis remains challenged by the lack of the double and triple *ag85abc* mutants. Because the active sites of the Ag85 isoforms are nearly 100% conserved, this non-conserved site may also provide an additional handle for production of isoform-selective inhibitors and further characterization of individual isoform activity. These results with the Ara6 terminus suggest that Ag85A is particularly important for the synthesis of fully mycolated cell wall, but it remains to be seen whether these results have implications for cell wall turnover, TMM synthesis, or other putative functions of the Ag85 enzymes. Here we have provided the first molecular explanation of the differences in Ag85A, -B, and -C activity and propose that these differences in activity may serve important roles in the efficient synthesis and modulation of the production of the arabinogalactan-mycolate polymer and the trehalose dimycolate.

M. tuberculosis Ag85 Isoforms Have Unique Activities

Acknowledgments—Plasmids were obtained from Colorado State University tuberculosis research contract HHSN266200400091c. Molecular graphics and analyses were performed with the UCSF Chimera package. Chimera is developed by the Resource for Biocomputing, Visualization, and Informatics at the University of California, San Francisco (supported by National Institutes of Health, NIGMS, Grant 9P41GM103311). Protein structures were analyzed in PyMOL (38).

---

**REFERENCES**

1. World Health Organization (2013) Global Tuberculosis Control Report 2013, World Health Organization, Geneva
2. Barry, C. E., 3rd, and Mdluli, K. (1996) Drug sensitivity and environmental adaptation of mycobacterial cell wall components. Trends Microbiol. 4, 275–281
3. Brennan, P. J. (2003) Structure, function, and biogenesis of the cell wall of *Mycobacterium tuberculosis*. Tuberculosis 83, 91–97
4. Slayden, R. A., and Barry, C. E., 3rd (2000) The genetics and biochemistry of isoniazid resistance in *Mycobacterium tuberculosis*. Microbes Infect. 2, 659–669
5. Takayama, K., and Kilburn, J. O. (1989) Inhibition of synthesis of arabinogalactan by ethambutol in *Mycobacterium smegmatis*. Antimicrob. Agents Chemother. 33, 1493–1499
6. Tahlan, K., Wilson, R., Kastrinsky, D. B., Arora, K., Nair, V., Fischer, E., Barnes, S. W., Walker, J. R., Alland, D., Barry, C. E., 3rd, and Boshoff, H. I. (2012) SQ109 targets MmpL3, a membrane transporter of trehalose monomycolate involved in mycolic acid donation to the cell wall core of *Mycobacterium tuberculosis*. Antimicrob. Agents Chemother. 56, 1797–1809
7. Stover, C. K., Warrener, P., VanDevanter, D. R., Sherman, D. R., Arain, T. M., Langhorne, M. H., Anderson, S. W., Towell, J. A., Yuan, Y., McMurray, D. N., Kreiswirth, B. N., Barry, C. E., and Baker, W. R. (2000) A small-molecule nitroimidazopyran drug candidate for the treatment of tuberculosis. Nature 405, 962–966
8. Matsumoto, M., Hashizume, H., Tomishige, T., Kawasaki, M., Tsubouchi, H., Sasaki, H., Shimokawa, Y., and Komatsu, M. (2006) OPC-67683, a nitro-dihydro-imidazooxazole derivative with promising action against tuberculosis *in vitro* and in mice. *PLoS Med.* 3, e466
9. Fukui, Y., Hirai, T., Uchida, T., and Yoneda, M. (1965) Extracellular proteins of tubercle bacilli. IV. α and β antigens as major extracellular protein products and as cellular components of a strain (H37Rv) of *Mycobacterium tuberculosis*. Biken J. 8, 189–199
10. Jackson, M., Raynaud, C., Lanéelle, M. A., Guilhot, C., Laurent-Winter, C., Ensergueix, D., Gicquel, B., and Daffé, M. (1999) Inactivation of the antigen 85C gene profoundly affects the mycolate content and alters the permeability of the *Mycobacterium tuberculosis* cell envelope. Mol. Microbiol. 31, 1573–1587
11. Harth, G., Horwitz, M. A., Tabatadze, D., and Zamecnik, P. C. (2002) Targeting the *Mycobacterium tuberculosis* 30/32-kDa mycolyl transferase complex as a therapeutic strategy against tuberculosis: proof of principle by using antisense technology. Proc. Natl. Acad. Sci. U.S.A. 99, 15614–15619
12. Daffé, M. (2000) The mycobacterial antigens 85 complex: from structure to function and beyond. Trends Microbiol. 8, 438–440
13. Belisle, J. T., Vissa, V. D., Sievert, T., Takayama, K., Brennan, P. J., and Besra, G. S. (1997) Role of the major antigen of *Mycobacterium tuberculosis* in cell wall biogenesis. Science 276, 1420–1422
14. Favrot, L., Grzegorzewicz, A. E., Lajiness, D. H., Marvin, R. K., Boucau, J., Isailovic, D., Jackson, M., and Ronning, D. R. (2013) Mechanism of inhibition of *Mycobacterium tuberculosis* antigen 85 by ebselen. Nat. Commun. 4, 2748
15. Lim, J.-H., Park, J.-K., Jo, E.-K., Song, C.-H., Min, D., Song, Y.-J., and Kim, H.-J. (1999) Purification and immunoreactivity of three components from the 30/32-kilodalton antigen 85 complex in *Mycobacterium tuberculosis*. Infect. Immun. 67, 6187–6190
16. Kuo, C. J., Bell, H., Hsieh, C. L., Ptak, C. P., and Chang, Y. F. (2012) Novel mycobacteria antigen 85 complex binding motif on fibronectin. J. Biol. Chem. 287, 1892–1902
17. Pathan, A. A., Sander, C. R., Fletcher, H. A., Poulton, I., Alder, N. C., Beveridge, N. E., Whelan, K. T., Hill, A. V., and McShane, H. (2007) Boosting BCG with recombinant modified vaccinia ankara expressing antigen 85A: different boosting intervals and implications for efficacy trials. PLoS One 2, e1052
18. Baena, A., and Porcelli, S. A. (2009) Evasion and subversion of antigen presentation by *Mycobacterium tuberculosis*. Tissue Antigens 74, 189–204
19. Bentley-Hibbert, S. I., Quan, X., Newman, T., Huygen, K., and Godfrey, H. P. (1999) Pathophysiology of antigen 85 in patients with active tuberculosis: antigen 85 circulates as complexes with fibronectin and immunoglobulin G. Infect. Immun. 67, 581–588
20. Comas, I., Chakravartti, J., Small, P. M., Galagan, J., Niemann, S., Kremer, K., Ernst, J. D., and Gagneux, S. (2010) Human T cell epitopes of *Mycobacterium tuberculosis* are evolutionarily hyperconserved. Nat. Genet. 42, 498–503
21. Armitige, L. Y., Jagannath, C., Wanger, A. R., and Norris, S. J. (2000) Disruption of the genes encoding antigen 85A and antigen 85B of *Mycobacterium tuberculosis* H37Rv: effect on growth in culture and in macrophages. Infect. Immun. 68, 767–778
22. Puech, V., Guilhot, C., Perez, E., Tropis, M., Armitige, L., Y, Gicquel, B., and Daffé, M. (2002) Evidence for a partial redundancy of the fibronectin-binding proteins for the transfer of mycolyl residues onto the cell wall arabinogalactan termini of *Mycobacterium tuberculosis*. Mol. Microbiol. 44, 1109–1122
23. Kacem, R., De Sousa-D'Auria, C., Tropis, M., Chami, M., Gounon, P., Leblon, G., Houssin, C., and Daffé, M. (2004) Importance of mycolyltransferases on the physiology of *Corynebacterium glutamicum*. Microbiology 150, 73–84
24. Brand, S., Niehaus, K., Pühler, A., and Kalinowski, J. (2003) Identification and functional analysis of six mycolyltransferase genes of *Corynebacterium glutamicum* ATCC 13032: the genes cop1, cmt1, and cmt2 can replace each other in the synthesis of trehalose dicorynomycolate, a component of the mycolic acid layer of the cell envelope. Arch. Microbiol. 180, 33–44
25. Puech, V., Bayan, N., Salim, K., Leblon, G., and Daffé, M. (2000) Characterization of the *in vivo* acceptors of the mycolyl residues transferred by the corynebacterial PS1 and the related mycobacterial antigens 85. Mol. Microbiol. 35, 1026–1041
26. Barry, C. S., Backus, K. M., Barry, C. E., 3rd, and Davis, B. G. (2011) Precise ESI-MS assay of *M. tuberculosis* cell wall antigen-85 enzymes permits substrate profiling and design of a mechanism-based inhibitor. J. Am. Chem. Soc. 133, 13232–13235
27. Brzozowski, A. M., Derewenda, U., Derewenda, Z. S., Dodson, G. G., Lawson, D. M., Turkenburg, J. P., Bjorkling, F., Huge-Jensen, B., Patkar, S. A., and Thim, L. (1991) A model for interfacial activation in lipases from the structure of a fungal lipase-inhibitor complex. Nature 351, 491–494
28. McPhie, P. (2008) Concentration-independent estimation of protein secondary structure by circular dichroism: a comparison of methods. Anal. Biochem. 375, 379–381
29. Sanner, M. F. (1999) Python: a programming language for software integration and development. J. Mol. Graph. Model. 17, 57–61
30. Trott, O., and Olson, A. J. (2010) AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. J. Comput. Chem. 31, 455–461
31. Backus, K. M., Boshoff, H. I., Barry, C. S., Boutureira, O., Patel, M. K., D’Hooge, F., Lee, S. S., Via, L. E., Tahlan, K., Barry, C. E., 3rd, and Davis, B. G. (2011) Uptake of unnatural trehalose analogs as a reporter for *Mycobacterium tuberculosis*. Nat. Chem. Biol. 7, 228–235
32. Joe, M., Bai, Y., Nacario, R. C., and Lowary, T. L. (2007) Synthesis of the docosanasaccharide arabinan domain of mycobacterial arabinogalactan and a proposed octadecasaccharide biosynthetic precursor. J. Am. Chem. Soc. 129, 9885–9901
33. D’Souza, F. W., Ayers, J. D., McCarren, P. R., and Lowary, T. L. (2000) Arabinofuranosyl oligosaccharides from mycobacteria: synthesis and effect of glycosylation on ring conformation and hydroxymethyl group populations. J. Am. Chem. Soc. 122, 1251–1260

25052 JOURNAL OF BIOLOGICAL CHEMISTRY
VOLUME 289 • NUMBER 36 • SEPTEMBER 5, 2014

M. tuberculosis Ag85 Isoforms Have Unique Activities

34. Humphrey, W., Dalke, A., and Schulten, K. (1996) VMD: visual molecular dynamics. *J. Mol. Graphics* 14, 33–38

35. Phillips, J. C., Braun, R., Wang, W., Gumbart, J., Tajkhorshid, E., Villa, E., Chipot, C., Skeel, R. D., Kalé, L., and Schulten, K. (2005) Scalable molecular dynamics with NAMD. *J. Comput. Chem.* 26, 1781–1802

36. Brooks, B. R., Brooks, C. L., 3rd, Mackerell, A. D., Jr., Nilsson, L., Petrella, R. J., Roux, B., Won, Y., Archontis, G., Bartels, C., Boresch, S., Caflisch, A., Caves, L., Cui, Q., Dinner, A. R., Feig, M., Fischer, S., Gao, J., Hodoscek, M., Im, W., Kuczera, K., Lazaridis, T., Ma, J., Ovchinnikov, V., Paci, E., Pastor, R. W., Post, C. B., Pu, J. Z., Schaefer, M., Tidor, B., Venable, R. M., Woodcock, H. L., Wu, X., Yang, W., York, D. M., and Karplus, M. (2009) CHARMM: the biomolecular simulation program. *J. Comput. Chem.* 30, 1545–1614

37. Pettersen, E. F., Goddard, T. D., Huang, C. C., Couch, G. S., Greenblatt, D. M., Meng, E. C., and Ferrin, T. E. (2004) UCSF Chimera: a visualization system for exploratory research and analysis. *J. Comput. Chem.* 25, 1605–1612

38. DeLano, W. L. (2012) *The PyMOL Molecular Graphics System*, version 1.5.0.1, Schroedinger, LLC, New York

39. Anderson, D. H., Harth, G., Horwitz, M. A., and Eisenberg, D. (2001) An interfacial mechanism and a class of inhibitors inferred from two crystal structures of the Mycobacterium tuberculosis 30 kDa major secretory protein (Antigen 85B), a mycolyl transferase. *J. Mol. Biol.* 307, 671–681

40. Ronning, D. R., Vissa, V., Besra, G. S., Belisle, J. T., and Sacchettini, J. C. (2004) Mycobacterium tuberculosis antigen 85A and 85C structures confirm binding orientation and conserved substrate specificity. *J. Biol. Chem.* 279, 36771–36777

41. Sanki, A. K., Boucau, J., Umesiri, F. E., Ronning, D. R., and Sudeck, S. J. (2009) Design, synthesis and biological evaluation of sugar-derived esters, α-ketoesters and α-ketoamides as inhibitors for Mycobacterium tuberculosis antigen 85C. *Mol. Biosyst.* 5, 945–956

42. Ronning, D. R., Klabunde, T., Besra, G. S., Vissa, V. D., Belisle, J. T., and

Sacchettini, J. C. (2000) Crystal structure of the secreted form of antigen 85C reveals potential targets for mycobacterial drugs and vaccines. *Nat. Struct. Biol.* 7, 141–146

43. Sathyamoorthy, N., and Takayama, K. (1987) Purification and characterization of a novel mycolic acid exchange enzyme from *Mycobacterium smegmatis*. *J. Biol. Chem.* 262, 13417–13423

44. McNeil, M., Daffe, M., and Brennan, P. J. (1991) Location of the mycolyl ester substituents in the cell walls of mycobacteria. *J. Biol. Chem.* 266, 13217–13223

45. Deng, L., Mikusová, K., Robuck, K. G., Scherman, M., Brennan, P. J., and McNeil, M. R. (1995) Recognition of multiple effects of ethambutol on metabolism of mycobacterial cell envelope. *Antimicrob. Agents Chemother.* 39, 694–701

46. Xin, Y., Huang, Y., and McNeil, M. R. (1999) The presence of an endogenous endo-D-arabinase in *Mycobacterium smegmatis* and characterization of its oligoarabinoside product. *Biochim. Biophys. Acta* 1473, 267–271

47. Nguyen, L., Chinnapapagari, S., and Thompson, C. J. (2005) FbpA-dependent biosynthesis of trehalose dimycolate is required for the intrinsic multidrug resistance, cell wall structure, and colonial morphology of *Mycobacterium smegmatis*. *J. Bacteriol.* 187, 6603–6611

48. Asensio, J. L., Ardá, A., Cañada, F. J., and Jiménez-Barbero, J. (2013) Carbohydrate-Aromatic Interactions. *Acc. Chem. Res.* 46, 946–954

49. Cuyvers, S., Dornez, E., Delcour, J. A., and Courtin, C. M. (2012) Occurrence and functional significance of secondary carbohydrate binding sites in glycoside hydrolases. *Crit. Rev. Biotechnol.* 32, 93–107

50. Besra, G. S., Sievert, T., Lee, R. E., Slayden, R. A., Brennan, P. J., and Takayama, K. (1994) Identification of the apparent carrier in mycolic acid synthesis. *Proc. Natl. Acad. Sci. U.S.A.* 91, 12735–12739

51. Takayama, K., Armstrong, E. L., Kunugi, K. A., and Kilburn, J. O. (1979) Inhibition by ethambutol of mycolic acid transfer into the cell wall of *Mycobacterium smegmatis*. *Antimicrob. Agents Chemother.* 16, 240–242
