
Toxins 2014, 6, 2239-2255; doi:10.3390/toxins6082239

OPEN ACCESS

**toxins**

ISSN 2072-6651  
www.mdpi.com/journal/toxins

---

Review

**Quorum Sensing in Bacillus thuringiensis Is Required for Completion of a Full Infectious Cycle in the Insect**

Leyla Slamti*, Stéphane Perchat, Eugénie Huillet and Didier Lereclus

INRA, Unité MICALIS UMR-1319, La Minière, 78280 Guyancourt, France; AgroParisTech, UMR MICALIS, F-78352 Jouy-en-Josas, France;

E-Mails: stephane.perchat@jouy.inra.fr (S.P.); eugenie.huillet@jouy.inra.fr (E.H.); didier.lereclus@jouy.inra.fr (D.L.)

* Author to whom correspondence should be addressed; E-Mail: leyla.slamti@jouy.inra.fr; Tel.: +33-1-3465-3787.

Received: 29 May 2014; in revised form: 10 July 2014 / Accepted: 15 July 2014 / Published: 31 July 2014

---

Abstract: Bacterial cell-cell communication or quorum sensing (QS) is a biological process commonly described as allowing bacteria belonging to a same phenotype to coordinate gene expression to cell density. In Gram-positive bacteria, cell-cell communication mainly relies on cytoplasmic sensors regulated by secreted and re-imported signaling peptides. The Bacillus quorum sensors Rap, NprR, and PlcR were previously identified as the first members of a new protein family called RNPP. Except for the Rap proteins, these RNPP regulators are transcription factors that directly regulate gene expression. QS regulates important biological functions in bacteria of the Bacillus cereus group. PlcR was first characterized as the main regulator of virulence in *B. thuringiensis* and *B. cereus*. More recently, the PlcR-like regulator PlcRa was characterized for its role in cysteine metabolism and in resistance to oxidative stress. The NprR regulator controls the necrotrophic properties allowing the bacteria to survive in the infected host. The Rap proteins negatively affect sporulation via their interaction with a phosphorelay protein involved in the activation of Spo0A, the master regulator of this differentiation pathway. In this review we aim at providing a complete picture of the QS systems that are sequentially activated during the lifecycle of *B. cereus* and *B. thuringiensis* in an insect model of infection.

Keywords: RNPP; cell-cell communication; Bacillus thuringiensis; adaptation; virulence; necrotrophism; sporulation; gene expression

Toxins 2014, 6

1. Introduction

    *Bacillus thuringiensis* produces insect-specific delta-endotoxins (or Cry proteins) that form a crystal in the bacterium during growth under nutrient starvation [1]. The entomopathogenic properties of this bacterium are widely used in the world for pest control [2,3]. These beneficial properties should not hide the fact that, apart from the production of crystal toxins, *B. thuringiensis* is phenotypically and genetically indistinguishable from *B. cereus* [4], a human pathogen responsible for foodborne toxigenic infections [5]. The emetic syndrome in these infections is caused by the cereulide peptide produced by some strains [6,7]. *B. cereus* has also been described as the cause of nosocomial infections in immunocompromised patients and was involved in endocarditis, endophthalmitis or meningitis that have proved fatal [8–11]. This bacterium produces an arsenal of factors that are likely to be responsible for its opportunistic properties. These invasins allow the destruction of tissues and the multiplication of the bacteria in their host and are shared by *B. thuringiensis*. Among them, the phosphatidylinositol phospholipase C PI-PLC, the non-hemolytic enterotoxin Nhe and the hemolysins Hbl, CerAB and CytK are found. CytK, Hbl and Nhe were also linked to toxigenic infections with a diarrheic syndrome [5]. The genes encoding these proteins are under the control of the transcription regulator PlcR and thus belong to the PlcR regulon [12]. A *B. thuringiensis* or a *B. cereus* mutant devoid of PlcR will not succeed in infecting its host in an insect or a mouse model of infection [13]. These mutants are also significantly attenuated in an endophthalmitis model of infection [14].

    Besides its virulent behavior, a recent report highlighted a new lifestyle for *B. thuringiensis* in insects [15]. *B. cereus* and *B. thuringiensis* are capable of multiplying in insects, which is a privileged ecological niche for both bacteria [16]. It was shown that the *B. thuringiensis* genome contains a set of genes specifically induced after the insect death [15]. Expression of these genes, encoding degradative enzymes, such as metalloproteases, lipases, and chitinases, but also the enzymatic machinery synthesizing the lipopeptide kurstakin, depends on the transcription activator NprR. This regulator is responsible for the necrotrophic behavior of *B. thuringiensis* in its host as an *nprR* deletion mutant does not survive in the insect cadaver. Necrotrophism may allow *B. thuringiensis* to multiply and delay sporulation, which is a costly, irreversible, but necessary process to ensure proper dissemination of the bacteria in the environment.

    The spore is an ultimate form of resistance and is generally regarded as a potent infectious agent. It is the main factor responsible for dispersal and survival of *B. thuringiensis* and *B. thuringiensis* in various environments including the digestive tract, food or industrial equipment. Sporulation is under the control of the master regulator Spo0A-P [17]. In the model organism *Bacillus subtilis*, environmental cues lead to the transduction of a signal inducing the phosphorylation of Spo0A via several histidine kinases and the phosphorelay proteins Spo0F and Spo0B [18]. Spo0A-P can then either activate or repress the transcription of genes by binding to the 0A box [19]. Deletion of *spo0A* affects several pathways but most notably renders *B. thuringiensis* and *B. cereus* incapable of initiating sporulation [20].

    The expression of virulence, necrotrophism and sporulation genes depends on quorum sensing (QS) systems in *B. cereus* and *B. thuringiensis*. QS relies on small secreted molecules known as autoinducers (AI) or pheromones. The three most studied kinds of QS systems involve acyl-homoserine lactones (AI-1) used by Gram-negative bacteria; small oligopeptides (AIP) used by Gram-positive bacteria

and cyclic furanone-based compounds (AI-2) used by both Gram-negative and Gram-positive bacteria [21,22]. The peptide-based Gram-positive systems can be further classified into two categories depending if the target of the peptide is outside or inside the cell (Figure 1). Extracellular-acting AIP are recognized by a membrane-embedded histidine kinase that will phosphorylate a cognate response regulator upon peptide binding. This will activate the response regulator, enabling it to regulate the expression of target genes. A well-studied example of this type of QS is the Agr system of Staphylococcus aureus that controls the expression of virulence genes [23,24]. Intracellular acting AIP are internalized in the cell via an oligopeptide permease system. They will then bind to a sensor thereby affecting its activity as a transcriptional regulator or rendering it inactive by disrupting its interaction with another protein. The former case is exemplified by the PrgX-cCF10 system that controls conjugation in Enterococcus faecalis [25–27] and the latter by the Rap-Phr pairs involved in sporulation and competence in B. subtilis [28–30]. Common features of the extracellular- and intracellular-acting peptides are that they are genetically encoded and ribosomally translated, actively secreted and processed with some differences for these two latter steps.

Figure 1. Schematic representation of the two types of QS systems involving AIP in Gram-positive bacteria. The steps common to the two systems are on the right. The middle panel represents the extracellular-acting AIP-dependent mechanism. The left panel represents the intracellular-acting AIP-dependent mechanism. This study will focus on QS systems relying on the peptides indicated in the purple box. Black arrows represent a positive effect. Black blunt lines represent a negative effect. Purple lines indicate the circuit followed by the AIP. RR, response regulator.

![Diagram](attachment:diagram.png)

The research field of intracellular-acting communication systems has been recently marked with several important milestones, notably with the structural characterization of some of the above mentioned regulators with their interacting partners [31–37]. The structural features of the Rap, NprR, PlcR, and PrgX quorum sensors allowed their classification as members of a new protein family called RNPP [32]. These quorum sensors are characterized by the presence of six to nine tetra-tricopeptide

repeats (TPR) forming the peptide-binding domain. Except for the Rap proteins, they also contain an N-terminal helix-turn-helix (HTH) DNA-binding domain.

The role, regulation and interconnection of these RNPP QS systems in *B. thuringiensis* during infection of insect larvae will be the focus of this review.

## 2. Kill: PlcR and Virulence

PlcR was first identified in *B. thuringiensis* as the transcriptional activator of the *plcA* gene, which encodes the phosphatidylinositol-specific phospholipase C (PI-PLC) [38]. PlcR activates its own transcription and that of *plcA* at the end of the exponential growth phase. A genetic screen demonstrated that PlcR is a pleiotropic regulator that controls the transcription of several genes encoding exported proteins putatively involved in virulence [39]. As mentioned in the introduction, these factors include various degradative enzymes and cytotoxins, as well as the hemolytic and non-hemolytic enterotoxins, Hbl and Nhe. Alignment of the promoter regions activated by PlcR revealed a highly conserved palindromic sequence (TATGNANNNNANCATA) designated as the PlcR box. The –10 region of the PlcR-regulated promoters resembles that of promoters bound by an RNA polymerase associated with the Sigma A factor. However, the –35 region of these promoters significantly diverges from the –35 sequence generally recognized by Sigma A. The transcription start site of the PlcR-regulated genes can be located at various positions downstream from the palindromic sequence (from a few nucleotides up to about 200 nucleotides in the case of the *hbl* and *nhe* operons). Proteomic and transcriptomic analyses allowed determination of the complete PlcR regulon in *B. cereus* and *B. thuringiensis*. In these bacteria, PlcR positively controls the expression of 45 genes and accounts for about 80% of the secretome during early stationary phase [12,40]. Forty of the PlcR-controlled proteins are exported and the functions of these proteins are related to food supply and virulence (phospholipases, proteases, hemolysins, and toxins), cell protection and environment-sensing.

The presence of a small PlcR-regulated gene, *papR*, located 70 bp downstream from *plcR* and the beginning of *plcR* expression being tied to the onset of stationary phase hinted to a possible regulation via a QS mechanism. Deletion of *papR* abolishes the activity of PlcR and results in a significant reduction of virulence [41]. The *papR* gene product is secreted, diffusible in the extracellular medium and processed mainly as the ADLPFEF heptapeptide, corresponding to the carboxy-terminal end of the peptide [32,42]. The peptic sequence of PapR contains a putative signal for secretion by the general secretion Sec pathway, but export of the peptide via this system has not been demonstrated. Maturation of PapR depends on NprB, a protease that belongs to the PlcR regulon [43]. However, complementation experiments using the supernatant of a *plcR* mutant suggested that other proteases are involved in this process (our unpublished results). The processed form of PapR is subsequently imported into the bacterial cells via the oligopeptide permease Opp [44]. Once inside the bacteria, PapR interacts with PlcR, allowing its binding to the PlcR boxes [41]. Therefore, the PlcR-PapR complex forms a quorum sensing system in which PlcR is the quorum sensor and PapR the signaling molecule. This system allows the coordination of gene expression to cell density. Figure 2 summarizes the regulation of the expression of the *plcR* gene and the activity of its product in association with PapR. Recent structural data explained the activation mechanism of PlcR: peptide binding in the TPR domain of the protein results in more flexible HTH domains; this conformational change allows the

formation of a drastic kink in the helix linking the two domains upon DNA binding. This mechanism allows the binding of the HTH domains of the PlcR dimer in the two half sites of the palindromic PlcR box [36].

Figure 2. Schematic representation of the PlcR-PapR QS system, its regulation and its activity. The product of the *papR* gene, belonging to the PlcR regulon, is a 48 amino-acid peptide. PapR is secreted, probably via the Sec machinery, processed by the NprB protease and presumably other peptidases. PapR is found in the extracellular compartment and in the bacterial cytoplasm mainly as a seven amino-acid peptide. It is imported in the cell via the oligopeptide permease Opp. Once inside the cell PapR binds to PlcR allowing it to activate the expression of 45 genes composing the PlcR regulon. *plcR* is autoregulated and under the negative control of Spo0A-P. CodY positively controls the expression of the PlcR-dependent genes via a yet unknown mechanism.

The activation of PlcR by PapR is strain-specific, and this specificity is determined by few residues of the signaling peptide and of the quorum sensor [42,45]. Alignment of PapR sequences from various strains of the *B. cereus* group, and PlcR activation assays using various PapR have led to the identification of four specificity groups. The PlcR regulon of a strain belonging to a given specificity group can be activated by its cognate signaling peptide. However, this PlcR regulon is not, or poorly, activated by the strains producing PapR molecules from other specificity groups. This suggests a co-evolution of PlcR and PapR that may reflect niche specificities.

Recent reports have shown that the global stationary phase regulator CodY is required for expression of the PlcR regulon [46,47]. However it is not known at what stage of the activation process this pleiotropic regulator is required. CodY has previously been shown to control virulence in several other Gram-positive bacteria such as *B. anthracis*, *S. aureus*, and *Listeria monocytogenes* [48,49]. The YvfTU two-component system is also involved in *plcR* expression [50]. However, the inducing cues for signal transduction in this system are unknown and direct activation of *plcR* expression by the response regulator was not demonstrated. It was also shown that transcription of *plcR* is dependent on the growth medium and is repressed by the master regulator of sporulation, Spo0A [51]. It results that

Toxins 2014, 6

PlcR is not active in *B. thuringiensis* cells growing in a sporulation-specific medium, and that the PlcR-regulated genes are only expressed in bacteria growing in rich medium, like the LB medium, or in insect larvae [15].

## 3. Survive: NprR and Necrotrophism

In sporulation-specific medium, the neutral protease NprA is the major extracellular protein massively produced by bacteria of the *B. cereus* group [52,53]. Expression of *nprA* is controlled by NprR, a regulator whose transcriptional activity depends on the NprX signaling peptide SKPDIVG [53]. Contrary to PlcR, NprR is active in sporulation-specific growth conditions. A recent report has demonstrated that NprX binds to the TPR domain of NprR and induces a switch from a dimeric apo form to a tetrameric form active as a transcriptional regulator [37]. In association with NprX, NprR activates the transcription of at least 41 genes [15]. NprR-NprX binds to a DNA region encompassing the −35 box in the promoter region of *nprA* [53], but additional experiments are needed to precisely determine the DNA sequence recognized by the complex. The NprR regulon includes genes encoding degradative enzymes (proteases, lipases and chitinases) and a lipopeptide (kurstakin) involved in biofilm formation. Using insect larvae as an infection model, it was shown that NprR is not involved in the virulence of the bacteria. However, the NprR regulon is essential for the survival of the Bacilli in the cadaver of infected insects [15]. Therefore, the NprR-NprX system allows the bacteria to survive and eventually to sporulate in the host cadaver, thus, improving their ability to disseminate in the environment.

Figure 3 illustrates the regulation of the *nprR* and *nprX* genes as well as their activity. The *nprR* gene is located upstream from *nprA* and is followed by the *nprX* gene, which is co-transcribed with *nprR*. Contrary to *plcR*, the transcription of *nprR-nprX* is not autoregulated and is negatively controlled by CodY during the exponential growth phase. This control by CodY presumably links the expression of the NprR regulon to nutrient availability. When nutrients are available, CodY is active and represses expression of *nprR*. At the onset of the stationary phase, the expression of *nprR* is activated by PlcR, linking the virulence stage to the necrotrophic lifestyle [54]. However, two additional promoters, depending on Sigma H and Sigma E, direct transcription of *nprX*. This may allow a prolonged production of the NprX peptide to ensure expression of NprR-regulated genes during the sporulation process, while transcription of *nprR* decreases. Because it presents a typical export signal sequence, NprX is also likely to be exported via the general secretory Sec pathway. The maturation mechanism and the physiological size of the active form of the peptide have not been reported. However, peptides ranging from seven to nine amino-acids allow activation of NprR [37]. Interestingly, and in contrast to PapR₇, the active form of NprX corresponds to a central part of the C-terminal region of the pro-peptide, suggesting a different maturation process. Characterization of the factors involved in the step allowing entry of the mature form in the cell is under investigation. Preliminary results showed that several oligopeptide permease systems are responsible for the import of NprX [55], contrary to PapR that only requires Opp [44]. Similarly to the PlcR-PapR system, the NprR-NprX system is strain specific [53]. Based on the NprR and NprX peptic sequences, seven phenotypes were defined within the *B. cereus* group with a range in the stringency of the specificity across the NprR-NprX phenotypes.

Toxins 2014, 6

Figure 3. Schematic representation of the NprR-NprX QS system, its regulation and its activity. The product of the *nprX* gene is a 43 amino-acid peptide. NprX is secreted, probably through the Sec machinery and processed via an uncharacterized mechanism. The physiological form of NprX has not been determined. However, the minimal active form is seven amino-acids long. It is imported in the cell via the oligopeptide permease Opp and presumably another oligopeptide permease system. Once inside the cell, NprX binds to NprR allowing it to activate the expression of at least 41 genes composing the NprR regulon. Transcription of *nprR* is activated by PlcR-PapR and is under the negative control of CodY. *nprX* is co-transcribed with *nprR*, but also possesses a promoter under the control of SigH and SigE.

![Diagram](attachment:diagram.png)

Using the insect model of intrahemocoelic infection, analysis of the expression kinetics of *nprA* in vivo revealed that transcription of this gene significantly increases after the death of the insect to reach a maximum 48 h after inoculation [15]. Interestingly, the PlcR-dependent *mbpE* gene is expressed earlier in the same conditions and its transcription reaches a maximum 24 h after infection. This shows that *mbpE* and *nprA* that reflect the activity of the two QS systems PlcR-PapR and NprR-NprX, respectively, are sequentially activated during infection.

## 4. Resist: Rap and Sporulation

It has been recently shown that a *B. thuringiensis sigmaK* mutant was able to survive in an insect cadaver, albeit to a lower level than a wild-type strain [15]. This suggests that survival of *B. thuringiensis* cells in a host cadaver is partially independent on sporulation. However, an efficient sporulation is the warranty of a better survival outside the host. The process of sporulation has been more extensively studied in *B. subtilis* than in *B. thuringiensis* or *B. cereus* and is the result of a complex differentiation pathway. The initiation of sporulation is regulated by Spo0A, the key transcription factor that controls early stationary phase and sporulation gene expression [18]. The activity of Spo0A is governed by the multicomponent phosphorelay consisting of five histidine kinases (KinA to KinE), the Spo0F response regulator and the Spo0B phosphotransferase [56].

The phosphoryl groups are transferred from the kinases to Spo0F (Spo0F-P) then to Spo0B (Spo0B-P), which finally phosphorylates Spo0A to generate Spo0A-P [57]. The Rap phosphatases modulate the level of Spo0A-P by dephosphorylating Spo0F-P in order to prevent early sporulation [58]. The balance of the action of kinases and phosphatases allows a gradual increase in the concentration of Spo0A-P until it reaches a threshold level above which sporulation is triggered [59].

The phosphatase activity of the Rap protein is regulated by the Phr signaling peptides that directly inhibit their activity through a QS system. Rap proteins are structured in TPR domains which mediate interactions with Spo0F-P or with the Phr inhibitor peptide [35,60]. The Phr peptide restores the transfer of the phosphoryl groups by displacing Spo0F~P from a preformed complex with Rap [61]. Despite the presence of a canonical signal peptide sequence, the involvement of the typical signal peptidases in processing the pre-propeptide could not be demonstrated [62]. The pro-peptide PhrA, but not PhrE, was shown to be matured by proteases of the subtilisin family [63]. The oligopeptide permease system (Opp) then allows the import of the signaling peptide into the cytoplasm of the bacteria [64,65]. This circuit ensures the appropriate timing for the phosphatase activity of the Rap protein in order to allow the bacterial population to sporulate efficiently.

The Phr-encoding genes are generally located downstream of the rap genes and are co-transcribed from the rap gene promoter [65]. Several regulators are involved in controlling the transcription level of these genes. For example, transcription of rapA-phrA is positively regulated by the response regulator ComA [66] and negatively affected by the transcriptional regulator CodY [67]. In addition, most of the phr genes have a second promoter controlled by the alternative Sigma H factor [68].

Rap proteins have not been studied in *B. thuringiensis* thus far. The Spo0F, Spo0B, and Spo0A proteins are present in all of the *Bacillus* species and the key amino acids that govern the molecular basis for recognition specificity between these proteins are highly conserved [69–71]. Five histidine kinases involved in the sporulation of *B. anthracis* were identified by analogy with the histidine kinases found in *B. subtilis* [72]. Moreover, an *oppB* mutation significantly reduced the sporulation efficiency in *B. thuringiensis* in rich medium [44]. This suggests that the mechanisms controlling the entry in sporulation are at least partially conserved between *B. subtilis* and bacteria of the *B. cereus* group.

Among the six putative Rap-Phr systems encoded in the *B. anthracis* genome only two are involved in the regulation of sporulation initiation with a phosphatase activity against Spo0F~P [73]. The BXA0205-BXA0205Phr system, which is associated with the strongest phenotype, is located on the pXO1 virulence plasmid. The phosphatase activity of the BXA0205 Rap protein is inhibited by the carboxy-terminal pentapeptide GHTGG of the BXA0205Phr peptide. The second system, BA3790-3791, is located on the chromosome and the active form of the Phr inhibitor corresponds to a central part of the peptide sequence but has not been clearly defined.

BLAST analyses identified eight putative Rap proteins presenting between 42% and 59% identity with the amino acid sequence of the *B. anthracis* BXA0205 Rap protein (Table 1). Moreover, open reading frames (ORF) encoding putative Phr-like peptides of 44 to 96 amino acids are located at the 3' end of each identified *rap* gene. The Phr peptide corresponding to the RapK protein is the only one that has not been annotated in the database. Analysis of each Phr peptide with the SignalP 4.1 or Phobius programs [74,75] indicated a putative signal peptide cleavage site (Table 2) typical of secreted proteins, as previously described for the Phr peptides of *B. subtilis* and *B. anthracis* [28,73]. The *B. anthracis* BXA0205-BXA0205Phr system could correspond to the RapK-PhrK or the RapF-PhrF

Toxins 2014, 6

systems in the B. thuringiensis strain 407. Similarly to the bxa0205-bxa0205phr genes, the rapK-phrK and rapF-phrF genes are plasmid-borne. The BX A0205 Rap protein shares 59% identity with RapK and 52% identity with RapF. Interestingly, the carboxy-terminal pentapeptide GHTGG, defined as the active form of the BX A0205 Phr, is highly conserved in PhrK and PhrF (Table 2). The second B. anthracis Rap-Phr system, BA3790-3791, could resemble to the RapF2-PhrF2 system in the B. thuringiensis strain 407. The rapF2-phrF2 genes are located to the chromosome like the ba3790-3791 genes and the BA3790Rap protein shares 53% identity with RapF2. As the active form of the BA3791 Phr has not been clearly characterized, no conclusion could be drawn about similarities with PhrF2.

Table 1. rap genes in the B. thuringiensis strain 407 genome. Rap proteins have been identified by BLAST analysis with the amino acid sequence of B. anthracis BX A0205 Rap protein as a query.

| Locus tag<sup>a</sup> | Annotation<sup>b</sup> | % Identity with BX A0205 | Localization |
|---------------------|-----------------------|--------------------------|--------------|
| BTB_6p00030         | rapK                  | 59%                      | pBTB_6p plasmid |
| BTB_78p00160        | rapF                  | 52%                      | pBTB_78p plasmid |
| BTB_c35690          | rapF2                 | 51%                      | chromosome    |
| BTB_c10360          | rapF1                 | 47%                      | chromosome    |
| BTB_9p00050         | rapI                  | 46%                      | pBTB_9p plasmid |
| BTB_c10810          | rapI1                 | 43%                      | chromosome    |
| BTB_c17720          | rap-like<sup>c</sup>  | 45%                      | chromosome    |
| BTB_502p04160       | rapC                  | 42%                      | pBTB_502p plasmid |

<sup>a,b</sup> As annotated in the B. thuringiensis strain 407 genome sequence (accession number CP003889 to CP003898, Sheppard, 2013); <sup>c</sup> Not annotated, personal annotation.

Table 2. Amino acids sequences of the putative Phr signaling peptides of B. thuringiensis strain 407 and the BX A205Phr of B. anthracis. The BX A205Phr pentapeptide inhibitor is highlighted in orange. The putative PhrK and PhrF penta-peptide inhibitors are highlighted in purple. The putative signal peptides are underlined.

| Annotation<sup>a</sup> | Sequence                                                                                          | Length<sup>b</sup> |
|-----------------------|----------------------------------------------------------------------------------------------------|--------------------|
| BX A0205Phr           | MKKV MFSLIGLTAVFTFMFNASNVTDTQKALSEDKVVQYAH**GHTGG**                                                | 46                 |
| PhrK                  | MKKTILTL MGIITVFTLTLSNINTPKENKDPSIQKIMLMSD**GNTGG**                                               | 46                 |
| PhrF                  | MKKTLSLMGI VTILTFTIGLSNPGEVQKFIQTKVAASE**GDYGG**                                                  | 44                 |
| PhrF2                 | MIKKISSIVLGLSVLSIVSIGLNSSFTYQAGHADFP                                                              | 62                 |
| PhrF1                 | MKRISSSIGLI ITVILLNGNVVSTDQLKVNTTSVIQYTHGEPWG                                                     | 45                 |
| PhrI                  | MMKKFSLILIGVACTTGIFFSQFNNSIQTHDAKEKNDIIQQYAHGKDI                                                  | 48                 |
| PhrI1                 | MKKFRLAIVGTALVGVL SIGFNSSFTNQAVNIGDTGGA PARPDYVNIGDTGGAPARPDYVNIGDTGGAPA                             | 96                 |
| Phr-like<sup>c</sup>  | MKKISLA ILTFTCILAFGFNNFTESQQAKQLPKWDTDKQETHADL                                                    | 45                 |
| PhrC                  | MKKFKIAL LGFVSVA VLSLGLNSGTETQKASSV TESP THVIYYSHADVW                                              | 49                 |

<sup>a</sup> As annotated in the B. anthracis A2012 (accession number NC_003980, Biongiorni 2006) and in the B. thuringiensis 407 genome sequence (accession number CP003889 to CP003898, Sheppard 2013);
<sup>b</sup> Length of the entire peptidic sequence; <sup>c</sup> Not annotated, personal annotation.

This comparative analysis with the Rap-Phr systems of *B. anthracis* suggests that at least three different Rap-Phr systems (RapK-PhrK, RapF-PhrF and RapF2-PhrF2) could be involved in the control of the initiation of sporulation in *B. thuringiensis* strain 407. Further studies will be needed to examine the involvement of these three systems in the sporulation process and to understand the role of the other Rap-Phr pairs in the lifecycle of *B. thuringiensis*.

## 5. PlcRa and Other PlcR-Like QS Systems in B. thuringiensis

Analyses of the *B. thuringiensis* genome revealed the presence of three PlcR-like proteins on the basis of their primary peptic sequence homology to PlcR. PlcRa, PlcRb and PlcRc display about 29% identity and 50% similarity to PlcR and are 85% identical. The *plcR*-like genes are located in three different loci on the chromosome of *B. thuringiensis* strain 407. A small ORF, *papRa*, encoding a peptide with a putative signal sequence, is located upstream from the *plcRa* gene. In contrast, *plcRb* and *plcRc* are not associated with such ORF. A homology model for PlcRa, PlcRb and PlcRc was constructed, based on the X-ray structure of the PlcR dimer [76], and our unpublished results. Each PlcR-like monomer is composed of an HTH domain in its N-terminal region, followed by a linker helix that connects the HTH to the five TPRs and is at the interface between the two monomers. The TPR domain of the PlcR-like proteins is arranged as described for PlcR, and forms a hydrophobic pocket that can potentially accommodate a peptide.

As hypothesized, PlcRa is activated by the product of the *papRa* gene, a secreted signaling peptide [76]. As described for PapR, a typical Gram-positive N-terminal signal peptide was identified for PapRa using the SignalP program [74]. Based on a sequence alignment with the PapR peptide, the CSIPYEEY fragment, PapRa₇, was proposed as the PlcRa cognate signaling peptide. PapRa₇ corresponds to an internal region of the carboxy-terminal part of PapRa, contrary to PapR₇ whose mature form corresponds to the C-terminal end of PapR. Biochemical and genetic analyses demonstrated the role of PapRa₇ in PlcRa activity. Purified PlcRa specifically binds to a PlcRa-controlled promoter and its binding requires the presence of synthetic PapRa₇. Addition of PapRa₇ to growing cells or overexpression of *papRa* activates the expression of *abrB2*, a PlcRa-dependent gene, at the onset of stationary phase [76]. Interestingly, *papRa* is included in the NprR regulon [15]. Preliminary results also suggest that PlcR is indirectly involved in the activation of PlcRa (our unpublished data). Data about the maturation and import of PapRa have not been reported yet. Analysis of 13 *B. thuringiensis* and 13 *B. cereus* complete genomes sequences revealed that the PapRa heptapeptide is conserved in all strains suggesting that there is only one phenotype associated to this QS system, in contrast to PapR and NprR.

Transcription of *plcRa* is not autoregulated and is triggered at the beginning of stationary phase in rich medium. Analysis of the transcriptome of a *plcRa* mutant compared to that of the wild-type strain showed that 68 and 49 genes were up- and down-regulated, respectively, at the onset of stationary phase [76]. PlcRa notably positively regulates the transcription of genes involved in regulation and synthesis of cysteine and in peroxide stress resistance. Most of the PlcRa-positively controlled genes belong to regulons that are involved either directly (PerR, OhR) or indirectly (CymR) in the responses to peroxide and disulfide stresses in *B. subtilis* [77,78] and in *B. cereus* [79]. The *plcRa* mutant is more sensitive to hydrogen peroxide (H₂O₂) and diamide stresses than the isogenic wild-type strain and

cystine addition to a ΔplcRa culture improved H₂O₂ stress resistance [76]. It has previously been shown in *B. subtilis* that modifications in the intracellular concentration of cysteine lead to increased sensitivity to oxidative stresses [77,78]. Cysteine is the direct precursor of low-molecular weight thiol molecules such as bacillithiol and Coenzyme A. In *Bacillus* species, these molecules are the key actors in the maintenance of cytosolic redox balance and in adaptation to the presence of reactive oxygen species [78,80,81]. This study demonstrated for the first time the existence of regulatory connections between cysteine metabolism and the oxidative stress responses in *B. cereus*.

plcRa inactivation had no significant effect on sporulation in many common laboratory conditions. The involvement of PlcRa in the cellular response to various stresses that might be encountered during infection suggests that the PlcRa regulon might have a role during the lifecycle of *B. thuringiensis* in its host. This is currently under investigation. The role of plcRb and plcRc in the physiology of *B. thuringiensis* is yet to be determined.

## 6. Conclusions

The QS systems presented above are all intertwined via controls at the transcriptional or post-transcriptional level. We propose the following model to illustrate these interconnections (Figure 4). This figure describes the sequential activation of the four RNPP sensors described in the review and the connection of these direct QS systems with the physiological stages of the bacteria throughout the infectious process. During the early infection stage, PlcR is activated by its cognate peptide PapR and the PlcR-PapR complex switches on the virulence properties of the bacteria, resulting in the death of the insect [41]. In addition, PlcR is indirectly involved in the activation of PlcRa at the post-transcriptional level (our unpublished results). PlcRa controls cysteine metabolism and resistance to oxidative stress [76] that might be involved in resistance to stresses encountered during the infectious cycle. However, the role of this latter communication system in infection has not been demonstrated yet. PlcR-PapR also activates the expression of *nprR* and *nprX* [54]. The NprR-NprX complex activates genes involved in necrotrophism (saprophytic stage), thus allowing the bacteria to survive in the insect cadaver [15]. NprR-NprX also activates the transcription of *papRa*, a gene encoding a peptide showing similarity with PapR and mapping just upstream from the *plcRa* gene, whose product activates PlcRa. All these communication systems control the fate and the physiology of the bacteria, thus, establishing a strong coordination between regulation of gene expression, cell development and infection. In addition, we hypothesize that PapR and/or PapRa activate two other PlcR paralogs of unknown function (designated PlcRb and PlcRc) that are not associated with signaling peptides. Altogether these results show that the pathogenic and necrotrophic lifestyles of *B. thuringiensis* and *B. cereus*, and presumably their sporulation and resistance to oxidative stress, are tightly controlled by four QS systems acting sequentially during the infection process.

Toxins 2014, 6

Figure 4. Sequential activation of the QS systems throughout the infectious process of *B. thuringiensis* in the insect. Solid lines represent a transcriptional effect. Dotted lines represent a peptide- or protein-protein interaction. The grey lines symbolize a putative effect. Arrows and blunt lines represent a positive or a negative effect, respectively. The bottom panels are pictures of *Galleria mellonella* 6th instar larvae, the model insect used for the infection experiments. From left to right: live larva, dead larva partially melanized, dead larva entirely melanized. Description of the figure is given in the main text.

![Diagram](attachment:diagram.png)

*B. cereus* and *B. thuringiensis* produce a lot of extracellular proteins and peptides [81]. For example, the *B. cereus* ATCC14579 supernatant contains more than 45 μg of proteins per mL, 2 h after entry into stationary phase in LB medium [40]. New signaling peptides that might be involved in various processes, yet to be described, are likely to be found in the extracellular environment of these bacteria.

## Acknowledgments

This work was supported by the Bt-Surf ANR grant (project number ANR-12-EMMA-0005-01).

## Conflicts of Interest

The authors declare no conflict of interest.

## References

1. Deng, C.; Peng, Q.; Song, F.; Lereclus, D. Regulation of *cry* gene expression in *Bacillus thuringiensis*. Toxins 2014, 6, 2194–2209.

2. Sanahuja, G.; Banakar, R.; Twyman, R.M.; Capell, T.; Christou, P. *Bacillus thuringiensis*: A century of research, development and commercial applications. *Plant Biotechnol. J.* 2011, 9, 283–300.
3. Sanchis, V. From microbial sprays to insect-resistant transgenic plants: History of the biopesticide *Bacillus thuringiensis*. A review. *Agron. Sustain. Dev.* 2011, 31, 217–231.
4. Helgason, E.; Okstad, O.A.; Caugant, D.A.; Johansen, H.A.; Fouet, A.; Mock, M.; Hegna, I.; Kolsto, A.B. *Bacillus anthracis*, *Bacillus cereus*, and *Bacillus thuringiensis*—One species on the basis of genetic evidence. *Appl. Environ. Microbiol.* 2000, 66, 2627–2630.
5. Stenfors Arnesen, L.P.; Fagerlund, A.; Granum, P.E. From soil to gut: *Bacillus cereus* and its food poisoning toxins. *FEMS Microbiol. Rev.* 2008, 32, 579–606.
6. Agata, N.; Mori, M.; Ohta, M.; Suwan, S.; Ohtani, I.; Isobe, M. A novel dodecadepsipeptide, cereulide, isolated from *Bacillus cereus* causes vacuole formation in HEp-2 cells. *FEMS Microbiol. Lett.* 1994, 121, 31–34.
7. Ehling-Schulz, M.; Svensson, B.; Guinebretiere, M.H.; Lindback, T.; Andersson, M.; Schulz, A.; Fricker, M.; Christiansson, A.; Granum, P.E.; Martlbauer, E.; et al. Emetic toxin formation of *Bacillus cereus* is restricted to a single evolutionary lineage of closely related strains. *Microbiology* 2005, 151, 183–197.
8. Castedo, E.; Castro, A.; Martin, P.; Roda, J.; Montero, C.G. *Bacillus cereus* prosthetic valve endocarditis. *Ann. Thorac. Surg.* 1999, 68, 2351–2352.
9. Chu, W.P.; Que, T.L.; Lee, W.K.; Wong, S.N. Meningoencephalitis caused by *Bacillus cereus* in a neonate. *Hong Kong Med. J.* 2001, 7, 89–92.
10. Gaur, A.H.; Patrick, C.C.; McCullers, J.A.; Flynn, P.M.; Pearson, T.A.; Razzouk, B.I.; Thompson, S.J.; Shenep, J.L. *Bacillus cereus* bacteremia and meningitis in immunocompromised children. *Clin. Infect. Dis.* 2001, 32, 1456–1462.
11. Bottone, E.J. *Bacillus cereus*, a volatile human pathogen. *Clin. Microbiol. Rev.* 2010, 23, 382–398.
12. Gohar, M.; Faegri, K.; Perchat, S.; Ravnum, S.; Okstad, O.A.; Gominet, M.; Kolsto, A.B.; Lereclus, D. The PlcR virulence regulon of *Bacillus cereus*. *PLoS One* 2008, 3, e2793.
13. Salamitou, S.; Ramisse, F.; Brehelin, M.; Bourguet, D.; Gilois, N.; Gominet, M.; Hernandez, E.; Lereclus, D. The PlcR regulon is involved in the opportunistic properties of *Bacillus thuringiensis* and *Bacillus cereus* in mice and insects. *Microbiology* 2000, 146, 2825–2832.
14. Callegan, M.C.; Kane, S.T.; Cochran, D.C.; Gilmore, M.S.; Gominet, M.; Lereclus, D. Relationship of PlcR-regulated factors to *Bacillus endophthalmitis* virulence. *Infect. Immun.* 2003, 71, 3116–3124.
15. Dubois, T.; Faegri, K.; Perchat, S.; Lemy, C.; Buisson, C.; Nielsen-LeRoux, C.; Gohar, M.; Jacques, P.; Ramarao, N.; Kolsto, A.B.; et al. Necrotrophism is a quorum-sensing-regulated lifestyle in *Bacillus thuringiensis*. *PLoS Pathog.* 2012, 8, e1002629.
16. Raymond, B.; Johnston, P.R.; Nielsen-LeRoux, C.; Lereclus, D.; Crickmore, N. *Bacillus thuringiensis*: An impotent pathogen? *Trends Microbiol.* 2010, 18, 189–194.
17. Olmedo, G.; Ninfa, E.G.; Stock, J.; Youngman, P. Novel mutations that alter the regulation of sporulation in *Bacillus subtilis*. Evidence that phosphorylation of regulatory protein SpoOA controls the initiation of sporulation. *J. Mol. Biol.* 1990, 215, 359–372.

18. Sonenshein, A.L. Control of sporulation initiation in *Bacillus subtilis*. Curr. Opin. Microbiol. 2000, 3, 561–566.
19. Strauch, M.; Webb, V.; Spiegelman, G.; Hoch, J.A. The SpoOA protein of *Bacillus subtilis* is a repressor of the *abrB* gene. Proc. Natl. Acad. Sci. USA 1990, 87, 1801–1805.
20. Lereclus, D.; Agaisse, H.; Gominet, M.; Chauvaux, J. Overproduction of encapsulated insecticidal crystal proteins in a *Bacillus thuringiensis spo0A* mutant. Biotechnology (N. Y.) 1995, 13, 67–71.
21. Ng, W.L.; Bassler, B.L. Bacterial quorum-sensing network architectures. Annu. Rev. Genet. 2009, 43, 197–222.
22. Rutherford, S.T.; Bassler, B.L. Bacterial quorum sensing: Its role in virulence and possibilities for its control. Cold Spring Harb. Perspect. Med. 2012, 2, doi:10.1101/cshperspect.a012427.
23. Ji, G.; Beavis, R.C.; Novick, R.P. Cell density control of staphylococcal virulence mediated by an octapeptide pheromone. Proc. Natl. Acad. Sci. USA 1995, 92, 12055–12059.
24. Novick, R.P.; Geisinger, E. Quorum sensing in *Staphylococci*. Ann. Rev. Genet. 2008, 42, 541–564.
25. Suzuki, A.; Mori, M.; Sakagami, Y.; Isogai, A.; Fujino, M.; Kitada, C.; Craig, R.A.; Clewell, D.B. Isolation and structure of bacterial sex pheromone, cPD1. Science 1984, 226, 849–850.
26. Kozlowicz, B.K.; Bae, T.; Dunny, G.M. Enterococcus faecalis pheromone-responsive protein PrgX: Genetic separation of positive autoregulatory functions from those involved in negative regulation of conjugative plasmid transfer. Mol. Microbiol. 2004, 54, 520–532.
27. Cook, L.C.; Federle, M.J. Peptide pheromone signaling in *Streptococcus* and *Enterococcus*. FEMS Microbiol. Rev. 2014, 38, 473–492.
28. Perego, M.; Hoch, J.A. Cell-cell communication regulates the effects of protein aspartate phosphatases on the phosphorelay controlling development in *Bacillus subtilis*. Proc. Natl. Acad. Sci. USA 1996, 93, 1549–1553.
29. Lazazzera, B.A.; Solomon, J.M.; Grossman, A.D. An exported peptide functions intracellularly to contribute to cell density signaling in *Bacillus subtilis*. Cell 1997, 89, 917–925.
30. Perego, M. Forty years in the making: Understanding the molecular mechanism of peptide regulation in bacterial development. PLoS Biol. 2013, 11, e1001516.
31. Shi, K.; Brown, C.K.; Gu, Z.Y.; Kozlowicz, B.K.; Dunny, G.M.; Ohlendorf, D.H.; Earhart, C.A. Structure of peptide sex pheromone receptor PrgX and PrgX/pheromone complexes and regulation of conjugation in *Enterococcus faecalis*. Proc. Natl. Acad. Sci. USA 2005, 102, 18596–18601.
32. Declerck, N.; Bouillaut, L.; Chaix, D.; Rugani, N.; Slamti, L.; Hoh, F.; Lereclus, D.; Arold, S.T. Structure of PlcR: Insights into virulence regulation and evolution of quorum sensing in Gram-positive bacteria. Proc. Natl. Acad. Sci. USA 2007, 104, 18490–18495.
33. Baker, M.D.; Neiditch, M.B. Structural basis of response regulator inhibition by a bacterial anti-activator protein. PLoS Biol. 2011, 9, e1001226.
34. Gallego del Sol, F.; Marina, A. Structural basis of Rap phosphatase inhibition by Phr peptides. PLoS Biol. 2013, 11, e1001511.
35. Parashar, V.; Jeffrey, P.D.; Neiditch, M.B. Conformational change-induced repeat domain expansion regulates Rap phosphatase quorum-sensing signal receptors. PLoS Biol. 2013, 11, e1001512.
36. Grenha, R.; Slamti, L.; Nicaise, M.; Refes, Y.; Lereclus, D.; Nessler, S. Structural basis for the activation mechanism of the PlcR virulence regulator by the quorum-sensing signal peptide PapR. Proc. Natl. Acad. Sci. USA 2013, 110, 1047–1052.

Toxins 2014, 6

37. Zouhir, S.; Perchat, S.; Nicaise, M.; Perez, J.; Guimaraes, B.; Lereclus, D.; Nessler, S. Peptide-binding dependent conformational changes regulate the transcriptional activity of the quorum-sensor NprR. Nucl. Acids Res. 2013, 41, 7920–7933.

38. Lereclus, D.; Agaisse, H.; Gominet, M.; Salamitou, S.; Sanchis, V. Identification of a *Bacillus thuringiensis* gene that positively regulates transcription of the phosphatidylinositol-specific phospholipase C gene at the onset of the stationary phase. J. Bacteriol. 1996, 178, 2749–2756.

39. Agaisse, H.; Gominet, M.; Okstad, O.A.; Kolsto, A.B.; Lereclus, D. PlcR is a pleiotropic regulator of extracellular virulence factor gene expression in *Bacillus thuringiensis*. Mol. Microbiol. 1999, 32, 1043–1053.

40. Gohar, M.; Okstad, O.A.; Gilois, N.; Sanchis, V.; Kolsto, A.B.; Lereclus, D. Two-dimensional electrophoresis analysis of the extracellular proteome of *Bacillus cereus* reveals the importance of the PlcR regulon. Proteomics 2002, 2, 784–791.

41. Slamti, L.; Lereclus, D. A cell-cell signaling peptide activates the PlcR virulence regulon in bacteria of the *Bacillus cereus* group. EMBO J. 2002, 21, 4550–4559.

42. Bouillaut, L.; Perchat, S.; Arold, S.; Zorrilla, S.; Slamti, L.; Henry, C.; Gohar, M.; Declerck, N.; Lereclus, D. Molecular basis for group-specific activation of the virulence regulator PlcR by PapR heptapeptides. Nucleic Acids Res. 2008, 36, 3791–3801.

43. Pomerantsev, A.P.; Pomerantseva, O.M.; Camp, A.S.; Mukkamala, R.; Goldman, S.; Leppla, S.H. PapR peptide maturation: Role of the NprB protease in *Bacillus cereus* 569 PlcR/PapR global gene regulation. FEMS Immunol. Med. Microbiol. 2009, 55, 361–377.

44. Gominet, M.; Slamti, L.; Gilois, N.; Rose, M.; Lereclus, D. Oligopeptide permease is required for expression of the *Bacillus thuringiensis* PlcR regulon and for virulence. Mol. Microbiol. 2001, 40, 963–975.

45. Slamti, L.; Lereclus, D. Specificity and polymorphism of the PlcR-PapR quorum-sensing system in the *Bacillus cereus* group. J. Bacteriol. 2005, 187, 1182–1187.

46. Frenzel, E.; Doll, V.; Pauthner, M.; Lucking, G.; Scherer, S.; Ehling-Schulz, M. CodY orchestrates the expression of virulence determinants in emetic *Bacillus cereus* by impacting key regulatory circuits. Mol. Microbiol. 2012, 85, 67–88.

47. Lindback, T.; Mols, M.; Basset, C.; Granum, P.E.; Kuipers, O.P.; Kovacs, A.T. CodY, a pleiotropic regulator, influences multicellular behaviour and efficient production of virulence factors in *Bacillus cereus*. Environ. Microbiol. 2012, 14, 2233–2246.

48. Sonenshein, A.L. CodY, a global regulator of stationary phase and virulence in Gram-positive bacteria. Curr. Opin. Microbiol. 2005, 8, 203–207.

49. Stenz, L.; Francois, P.; Whiteson, K.; Wolz, C.; Linder, P.; Schrenzel, J. The CodY pleiotropic repressor controls virulence in gram-positive pathogens. FEMS Immunol. Med. Microbiol. 2011, 62, 123–139.

50. Brillard, J.; Susanna, K.; Michaud, C.; Dargaignaratz, C.; Gohar, M.; Nielsen-Leroux, C.; Ramarao, N.; Kolsto, A.B.; Nguyen-the, C.; Lereclus, D.; et al. The YvfTU two-component system is involved in plcR expression in *Bacillus cereus*. BMC Microbiol. 2008, 8, 183.

51. Lereclus, D.; Agaisse, H.; Grandvalet, C.; Salamitou, S.; Gominet, M. Regulation of toxin and virulence gene transcription in *Bacillus thuringiensis*. Int. J. Med. Microbiol. 2000, 290, 295–299.

52. Chitlaru, T.; Gat, O.; Gozlan, Y.; Ariel, N.; Shafferma, A. Differential proteomic analysis of the Bacillus anthracis secretome: Distinct plasmid and chromosome CO₂-dependent cross talk mechanisms modulate extracellular proteolytic activities. *J. Bacteriol.* 2006, 188, 3551–3571.

53. Perchat, S.; Dubois, T.; Zouhir, S.; Gominet, M.; Poncet, S.; Lemy, C.; Aumont-Nicaise, M.; Deutscher, J.; Gohar, M.; Nessler, S.; *et al.* A cell-cell communication system regulates protease production during sporulation in bacteria of the *Bacillus cereus* group. *Mol. Microbiol.* 2011, 82, 619–633.

54. Dubois, T.; Perchat, S.; Verplaetse, E.; Gominet, M.; Lemy, C.; Aumont-Nicaise, M.; Grenha, R.; Nessler, S.; Lereclus, D. Activity of the *Bacillus thuringiensis* NprR-NprX cell-cell communication system is co-ordinated to the physiological stage through a complex transcriptional regulation. *Mol. Microbiol.* 2013, 88, 48–63.

55. Dubois, T. Etude du système de communication cellulaire NprR-NprX au sein du groupe *Bacillus cereus*. Ph.D. Thesis, AgroParisTech, Jouy-en-Josas, France, 5 March 2012. (In French)

56. Jiang, M.; Shao, W.; Perego, M.; Hoch, J.A. Multiple histidine kinases regulate entry into stationary phase and sporulation in *Bacillus subtilis*. *Mol. Microbiol.* 2000, 38, 535–542.

57. Burbulys, D.; Trach, K.A.; Hoch, J.A. Initiation of sporulation in *Bacillus subtilis* is controlled by a multicomponent phosphorelay. *Cell* 1991, 64, 545–552.

58. Perego, M.; Glaser, P.; Hoch, J.A. Aspartyl-phosphate phosphatases deactivate the response regulator components of the sporulation signal transduction system in *Bacillus subtilis*. *Mol. Microbiol.* 1996, 19, 1151–1157.

59. Fujita, M.; Losick, R. Evidence that entry into sporulation in *Bacillus subtilis* is governed by a gradual increase in the level and activity of the master regulator Spo0A. *Genes Dev.* 2005, 19, 2236–2244.

60. Core, L.; Perego, M. TPR-mediated interaction of RapC with ComA inhibits response regulator-DNA binding for competence development in *Bacillus subtilis*. *Mol. Microbiol.* 2003, 49, 1509–1522.

61. Ishikawa, S.; Core, L.; Perego, M. Biochemical characterization of aspartyl phosphate phosphatase interaction with a phosphorylated response regulator and its inhibition by a pentapeptide. *J. Biol. Chem.* 2002, 277, 20483–20489.

62. Stephenson, S.; Mueller, C.; Jiang, M.; Perego, M. Molecular analysis of Phr peptide processing in *Bacillus subtilis*. *J. Bacteriol.* 2003, 185, 4861–4871.

63. Lanigan-Gerdes, S.; Dooley, A.N.; Faull, K.F.; Lazazzera, B.A. Identification of subtilisin, Epr and Vpr as enzymes that produce CSF, an extracellular signalling peptide of *Bacillus subtilis*. *Mol. Microbiol.* 2007, 65, 1321–1333.

64. Perego, M. A peptide export-import control circuit modulating bacterial development regulates protein phosphatases of the phosphorelay. *Proc. Natl. Acad. Sci. USA* 1997, 94, 8612–8617.

65. Perego, M.; Brannigan, J.A. Pentapeptide regulation of aspartyl-phosphate phosphatases. *Peptides* 2001, 22, 1541–1547.

66. Mueller, J.P.; Bukusoglu, G.; Sonenshein, A.L. Transcriptional regulation of *Bacillus subtilis* glucose starvation-inducible genes: Control of gsiA by the ComP-ComA signal transduction system. *J. Bacteriol.* 1992, 174, 4361–4373.

67. Molle, V.; Nakaura, Y.; Shivers, R.P.; Yamaguchi, H.; Losick, R.; Fujita, Y.; Sonenshein, A.L. Additional targets of the *Bacillus subtilis* global regulator CodY identified by chromatin immunoprecipitation and genome-wide transcript analysis. *J. Bacteriol.* 2003, 185, 1911–1922.

Toxins 2014, 6

68. McQuade, R.S.; Comella, N.; Grossman, A.D. Control of a family of phosphatase regulatory genes (phr) by the alternate sigma factor sigma-H of *Bacillus subtilis*. J. Bacteriol. 2001, 183, 4905–4909.

69. Stephenson, K.; Hoch, J.A. Evolution of signalling in the sporulation phosphorelay. Mol. Microbiol. 2002, 46, 297–304.

70. Tzeng, Y.L.; Hoch, J.A. Molecular recognition in signal transduction: The interaction surfaces of the Spo0F response regulator with its cognate phosphorelay proteins revealed by alanine scanning mutagenesis. J. Mol. Biol. 1997, 272, 200–212.

71. Zapf, J.; Sen, U.; Madhusudan; Hoch, J.A.; Varughese, K.I. A transient interaction between two phosphorelay proteins trapped in a crystal lattice reveals the mechanism of molecular recognition and phosphotransfer in signal transduction. Structure 2000, 8, 851–862.

72. Brunning, R.L.; La Clair, C.; Tang, S.; Chiang, C.; Hancock, L.E.; Perego, M.; Hoch, J.A. Characterization of sporulation histidine kinases of *Bacillus anthracis*. J. Bacteriol. 2005, 187, 6972–6981.

73. Bongiorni, C.; Stoessel, R.; Shoemaker, D.; Perego, M. Rap phosphatase of virulence plasmid pXO1 inhibits *Bacillus anthracis* sporulation. J. Bacteriol. 2006, 188, 487–498.

74. Petersen, T.N.; Brunak, S.; von Heijne, G.; Nielsen, H. SignalP 4.0: Discriminating signal peptides from transmembrane regions. Nat. Methods 2011, 8, 785–786.

75. Kall, L.; Krogh, A.; Sonnhammer, E.L. Advantages of combined transmembrane topology and signal peptide prediction—The Phobius web server. Nucl. Acids Res. 2007, 35, W429–W432.

76. Huillet, E.; Tempelaars, M.H.; Andre-Leroux, G.; Wanapaisan, P.; Bridoux, L.; Makhzami, S.; Panbangred, W.; Martin-Verstraete, I.; Abee, T.; Lereclus, D. PlcRa, a new quorum-sensing regulator from *Bacillus cereus*, plays a role in oxidative stress responses and cysteine metabolism in stationary phase. PLoS One 2012, 7, e51047.

77. Tanous, C.; Soutourina, O.; Raynal, B.; Hullo, M.F.; Mervelet, P.; Gilles, A.M.; Noirot, P.; Danchin, A.; England, P.; Martin-Verstraete, I. The CymR regulator in complex with the enzyme CysK controls cysteine metabolism in *Bacillus subtilis*. J. Biol. Chem. 2008, 283, 35551–35560.

78. Zuber, P. Management of oxidative stress in *Bacillus*. Annu. Rev. Microbiol. 2009, 63, 575–597.

79. Mols, M.; Abee, T. Primary and secondary oxidative stress in *Bacillus*. Environ. Microbiol. 2011, 13, 1387–1394.

80. Newton, G.L.; Rawat, M.; La Clair, J.J.; Jothivasan, V.K.; Budiarto, T.; Hamilton, C.J.; Claiborne, A.; Helmann, J.D.; Fahey, R.C. Bacillithiol is an antioxidant thiol produced in Bacilli. Nat. Chem. Biol. 2009, 5, 625–627.

81. Nicely, N.I.; Parsonage, D.; Paige, C.; Newton, G.L.; Fahey, R.C.; Leonardi, R.; Jackowski, S.; Mallett, T.C.; Claiborne, A. Structure of the type III pantothenate kinase from *Bacillus anthracis* at 2.0 Å resolution: Implications for coenzyme A-dependent redox biology. Biochemistry 2007, 46, 3234–3245.

© 2014 by the authors; licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution license (http://creativecommons.org/licenses/by/3.0/).

Copyright of Toxins is the property of MDPI Publishing and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.
