
Title  
Redox-dependent lipoylation of mitochondrial proteins in *Plasmodium falciparum*

Running Title  
Mitochondrial lipoylation in malaria parasites

Gustavo A. Afanador¹, Krista A. Matthews¹, David Bartee², Jolyn E. Gisselberg¹, Maroya S. Walters¹, Caren L. Freel Meyers², Sean T. Prigge¹  

¹Department of Molecular Microbiology and Immunology, Johns Hopkins School of Public Health;  
²Department of Pharmacology and Molecular Sciences, Johns Hopkins University School of Medicine, Baltimore MD

---

This article has been accepted for publication and undergone full peer review but has not been through the copyediting, typesetting, pagination and proofreading process, which may lead to differences between this version and the Version of Record. Please cite this article as doi: 10.1111/mmi.12753

This article is protected by copyright. All rights reserved.

Summary

Lipoate scavenging from the human host is essential for malaria parasite survival. Scavenged lipoate is covalently attached to three parasite proteins: the H-protein and the E2 subunits of branched chain amino acid dehydrogenase (BCDH) and α-ketoglutarate dehydrogenase (KDH). We show mitochondrial localization for the E2 subunits of BCDH and KDH, similar to previously localized H-protein, demonstrating that all three lipoylated proteins reside in the parasite mitochondrion. The lipoate ligase 1, LipL1, has been shown to reside in the mitochondrion and it catalyzes the lipoylation of the H-protein; however, we show that LipL1 alone cannot lipoylate BCDH or KDH. A second mitochondrial protein with homology to lipoate ligases, LipL2, does not show ligase activity and is not capable of lipoylating any of the mitochondrial substrates. Instead, BCDH and KDH are lipoylated through a novel mechanism requiring both LipL1 and LipL2. This mechanism is sensitive to redox conditions where BCDH and KDH are exclusively lipoylated under strong reducing conditions in contrast to the H-protein which is preferentially lipoylated under less reducing conditions. Thus, malaria parasites contain two different routes of mitochondrial lipoylation, an arrangement that has not been described for any other organism.

Introduction

*Plasmodium falciparum* (*Pf*) is the deadliest of five apicomplexan parasite species capable of causing malaria in humans (Okiro *et al.*, 2009a, Okiro *et al.*, 2009b, Mudhune *et al.*, 2011, Okiro *et al.*, 2011).

*Plasmodium* spp. first invade the liver and undergo a single round of replication. This phase is asymptomatic, in contrast to the erythrocytic stage of the parasite life cycle, which is characterized by a cyclical fever corresponding to multiple rounds of parasite replication within red blood cells. Increasing resistance to current drugs that target erythrocytic stage parasites makes it necessary to characterize pathways that are essential for parasite survival (Fidock *et al.*, 2008, Hay *et al.*, 2009, Dondorp *et al.*, 2011, Phyo *et al.*, 2012). One such pathway believed to be essential for malaria parasites is lipoate metabolism (Allary *et al.*, 2007, Spalding *et al.*, 2010b, Storm *et al.*, 2012). The parasite contains two organelles that harbor non-redundant pathways for lipoate metabolism: the apicoplast—a plastid organelle—and the mitochondrion (Gunther *et al.*, 2005, Allary *et al.*, 2007, Spalding *et al.*, 2010b). The apicoplast contains a lipoate biosynthesis pathway that is critical for liver stage parasite development, but is dispensable in the erythrocytic stage (Gunther *et al.*, 2007, Pei *et al.*, 2010, Falkard *et al.*, 2013). The mitochondrion relies on lipoate scavenging from the host, which has been proposed to be essential for parasite survival during the erythrocytic stage (Allary *et al.*, 2007, Gunther *et al.*, 2009, Storm *et al.*, 2012).

Lipoate is an essential cofactor for aerobic metabolism in oxidative decarboxylation reactions of 2-oxoacid complexes. Each complex is comprised of three subunits named E1, E2, and E3, except for glycine cleavage system which does not form a stable complex (Reed *et al.*, 1958). Lipoate is attached to a conserved lysine residue of a small lipoyl domain (LD) through an amide bond, serving as a swinging arm to shuffle reaction intermediates between active sites (Reche *et al.*, 1999, Perham, 2000). LDs are typically found at the N-terminus of the E2-subunit of the 2-oxoacid dehydrogenase complexes. The H-protein of the glycine cleavage system (GCV) functions as a LD. Lipoate shuttles the decarboxylated acid from the E1 subunit through a thioester linkage to the core of the E2-subunit where the acyl thioester is converted to the corresponding Coenzyme-A (CoA) thioester by thioester exchange.
dihydrolipoamide is oxidized to lipoamide by the dihydrolipoamide dehydrogenase (E3 subunit) to reset the catalytic cycle. The glycine cleavage system works in a similar fashion, but instead of three subunits it is composed of four loosely associated proteins: the P-protein (corresponding to the E1 subunit), the H- protein (lipoyl domain), the T-protein (core of E2-subunit) and the L-protein, which is the E3 subunit (Spalding *et al.*, 2010a).

Lipoate metabolism has been best characterized in *Escherichia coli* (*E. coli*), which has both biosynthetic and scavenging pathways (Fig. 1A and 1C). In *E. coli*, unlike in *Plasmodium* spp., these two pathways have been shown to be redundant (Morris *et al.*, 1995). Lipoate biosynthesis starts with an octanoyltransferase (*EcLipB*), which transfers the octanoyl group from octanoyl-Acyl Carrier Protein (octanoyl-ACP) to a conserved lysine of the lipoate requiring proteins (Jordan *et al.*, 2003). Then a lipoate synthase (*EcLipA*) inserts two sulfur atoms in positions C6 and C8 of the octanoyl group to produce the lipoyl moiety (Fig. 1A) (Miller *et al.*, 2000). The scavenging pathway uses a lipoate ligase (*EcLplA*) to attach lipoate to LDs in an ATP-dependent reaction (Fig. 1C) (Morris *et al.*, 1995, Fujiwara *et al.*, 2010).

Lipoate synthesis in the apicoplast of *Pf* follows the same pathway as the biosynthetic pathway described for *E. coli* (Foth *et al.*, 2005, Spalding *et al.*, 2010b). However, less is known about the mechanism of lipoate metabolism in the parasite mitochondrion, which relies on lipoate scavenged from the host (Allary *et al.*, 2007). Once lipoate is imported into the mitochondrion, there are two candidate enzymes, lipoate ligase 1 (*LipL1*) and lipoate ligase 2 (*LipL2*), which could catalyze the attachment of free lipoate to the lipoate requiring proteins. *LipL1* and *LipL2*, which have both been localized to the mitochondrion (Wrenger *et al.*, 2004, Gunther *et al.*, 2007), share about 21% sequence identity and are unlikely to be paralogs (Allary *et al.*, 2007). Three lipoate requiring proteins have been localized or predicted to be in the mitochondrion: branched chain amino acid dehydrogenase (BCDH), α-ketoglutarate dehydrogenase (KDH), and the H-protein of the GCV (Gunther *et al.*, 2005). The H-protein has been previously localized to the mitochondrion (Spalding *et al.*, 2010a) and *LipL1* can lipoylate the H-protein

This article is protected by copyright. All rights reserved.in an ATP-dependent reaction (Allary *et al.*, 2007). There is no known activity for LipL2 in the *Pf* mitochondrion, and no mechanism to lipoylate E2-BCDH or E2-KDH has yet been described.

It has been proposed that lipoate scavenged from red blood cells is essential to the parasite (Storm *et al.*, 2012). The halogenated compound 8-bromooctanoate (8-BrO) inhibits parasite growth and reduces lipoylation of lipoate-requiring proteins in the mitochondrion (Allary *et al.*, 2007). The reliance on scavenged lipoate makes this an attractive process to target with therapeutics. Yet, several questions remain unanswered: what is the compartmentalization of BCDH and KDH substrates; why does the parasite possess two non-paralogous lipoate ligases; what is their activity and substrate specificity against the three possible lipoate requiring substrates in the mitochondrion; and last, how does 8-BrO inhibit parasite growth?

Here we present mitochondrial localization data for the E2 subunits of BCDH and KDH in *Pf*. We show that LipL1 is the only active lipoate ligase in the parasite mitochondrion and is capable of forming the lipoylation reaction intermediate lipoyl-AMP. LipL1 modifies the H-protein through an ATP-dependent reaction; however LipL1 does not recognize the other substrates, BCDH and KDH. LipL2 does not exhibit ligase activity against any of the mitochondrial substrates, nor is it capable of synthesizing lipoyl-AMP. Interestingly, both LipL1 and LipL2 are required in an ATP-dependent reaction to lipoylate BCDH and KDH. These two mitochondrial lipoylation pathways show dependence on the lipoate redox state. The H-protein can be lipoylated under all conditions tested, but increased lipoylation is observed under less reducing conditions. In contrast to the H-protein, BCDH and KDH are only lipoylated under strong reducing conditions. Last, a lipoate analog that mimics the reduced form, 6,8-dichlorooctanoate (6,8-diClO), causes blood stage parasite growth inhibition similar to 8-BrO. 6,8-diClO can be scavenged by the parasite and attached to BCDH and KDH in the mitochondrion.

Modification with 6,8-diClO or 8-BrO leads to irreversible inactivation of these mitochondrial enzymes and subsequent parasite death.

Results

This article is protected by copyright. All rights reserved.The E2 subunits of BCDH and KDH localize to the parasite mitochondrion

To better understand lipoylation in the mitochondrion of *Plasmodium falciparum* (*Pf*), we first investigated the localization of the two lipoylated substrates predicted to be mitochondrial: the E2 subunits of BCDH and KDH (Gunther *et al.*, 2005). The E1β subunit of BCDH and the shared E3 subunit have been previously localized to the mitochondrion in *Pf*, showing that components of 2-oxoacid dehydrogenase reside in this compartment (Gunther *et al.*, 2005). The E2-BCDH of *Plasmodium berghei* has been localized to the mitochondrion (Falkard *et al.*, 2013). To confirm the localization of the *Pf* E2 proteins, we generated constructs in which the first 30 amino acids containing the predicted mitochondrial targeting sequence of E2-BCDH and E2-KDH were fused to green fluorescent protein (GFP). The transgene was inserted into the *Pfcg6* locus by mycobacteriophage integrase mediated recombination (Nkrumah *et al.*, 2006, Spalding *et al.*, 2010a) and GFP fluorescence from these parasites was analyzed by live microscopy. For both E2-GFP transgenic parasite lines, we observed GFP fluorescence in an organelle with branched-like morphology that co-localized with the mitochondrial marker MitoTracker (Fig. 2A). We confirmed expression of these constructs by western blot (Fig. 2B). These results demonstrate that the lipoylated E2 subunits of BCDH and KDH, in addition to the H-protein, reside in the mitochondrion with the previously localized lipoate attachment enzymes, LipL1 and LipL2 (Wrenger *et al.*, 2004, Gunther *et al.*, 2007).

Neither LipL1 nor LipL2 lipoylate $BCDH_{LD}$ or $KDH_{LD}$ in a cell-based lipoylation assay

The lipoate attachment enzymes LipL1 and LipL2 are predicted to be lipoate ligases based on sequence homology to the *E. coli* lipoate ligase (*EcLplA*) (Allary *et al.*, 2007). LipL1 can complement a lipoylation deficient *E. coli* cell line (TM136) in minimal media by adding exogenous lipoate, demonstrating its activity as a ligase (Allary *et al.*, 2007). LipL2 has been shown to have similar activity in TM136 cells but cell growth required four days of incubation, making this datum less conclusive (Allary *et al.*, 2007). An additional factor complicating these experiments involves the low level of endogenous lipoylation that remains in the TM136 line. Apparently, transposon disruption of the *lplA*

This article is protected by copyright. All rights reserved.
gene does not completely ablate lipoate ligase activity in TM136 (Fig. S1). Therefore, we used a lambda red recombination system (Datsenko *et al.*, 2000, Baba *et al.*, 2006) to generate an *E. coli* *lplA* and *lipB* deletion line (*lplA-/*lipB-*) that is completely deficient in lipoylation activity (Fig. S1). We used this new strain (JEG3) to conduct cell-based lipoylation assays. The lipoylation domains of BCDH and KDH (*BCDH*<sub>LD</sub> and *KDH*<sub>LD</sub>, respectively) were subcloned into a pGEX vector expressing GST (glutathione S-transferase) fusion proteins. To validate that *BCDH*<sub>LD</sub> and *KDH*<sub>LD</sub> are competent lipoylation substrates, we expressed each domain in wild type BL21 cells and found that they were lipoylated by *E. coli* lipoylation enzymes when visualized by western blot using an antibodies specific for lipoylated proteins (α-LA) (Fig. 2C). We next subcloned LipL1, LipL2 and *EcLplA* into a pMAL vector expressing MBP (maltose binding protein) fusion proteins. We co-expressed each LD in the JEG3 cell line with either *Pf* ligase or *EcLplA*. We found that the *EcLplA* was sufficient to lipoylate *BCDH*<sub>LD</sub> and *KDH*<sub>LD</sub> (Fig. 2D, lanes 1-2), but neither substrate was lipoylated in our *in vitro* cell-based assay by either putative *Pf* ligase when visualized by western blot (Fig. 2D, lanes 3-6). The lack of activity against *BCDH*<sub>LD</sub> and *KDH*<sub>LD</sub> was surprising since LipL1 is competent to lipoylate *EcKDH* and *EcPDH* (Fig. 2D, lanes 3-4) and LipL2 apparently lipoylates *EcPDH*, albeit poorly (Fig. 2D, lanes 5-6).

---

**LipL1 has substrate specific lipoate ligase activity**

To verify the unexpected behavior of LipL1 and LipL2, we decided to study the activity of purified recombinant enzymes. The *BCDH*<sub>LD</sub> and *KDH*<sub>LD</sub> lipoylation domains and truncated H-protein, lacking the N-terminal transit peptide, were expressed as GST fusion proteins in the lipoylation deficient JEG3 cell line and purified in the apo form. We tested recombinant LipL1 and LipL2 alone against each apo substrate in the presence or absence of ATP, and visualized lipoylation by western blot. As shown in Figure 3A, LipL1 is an active ligase only against the H-protein in an ATP-dependent reaction (top panel, lanes 1-2) in agreement with a previous report in which full length H-protein was used (Allary *et al.*, 2007). We also found that ATP is the preferred nucleotide used by LipL1 with minor activity observed with GTP (Fig. S2A). Consistent with results observed in the cell-based assay, LipL1 did not catalyze the

This article is protected by copyright. All rights reserved.
lipoylation of BCDH<sub>LD</sub> or KDH<sub>LD</sub> (Fig. 3A, top panel, lanes 3-6). Similarly, we found that LipL2 did not lipoylate any of the Pf substrates in our in vitro ligation assay (Fig. 3A, lower panel). These results, along with those from the in vitro cell-based assay, show that BCDH<sub>LD</sub> and KDH<sub>LD</sub> are lipoylated by E. coli lipoylation enzymes, but not by either candidate Pf lipoate ligase. LipL1 appears to be the only lipoate ligase with the H-protein as its only parasite substrate.

### LipL1 is the only active ligase

Lipoate ligases typically catalyze two sequential reactions. First, lipoate is activated in an adenylation reaction forming lipoyl-AMP; then the lipoyl moiety is transferred to protein substrates (Fujiwara et al., 2010). Based on the results shown in Figure 3A, LipL1 appears to catalyze both reactions when the H-protein is used as a substrate. We examined LipL1 incubated with lipoate and ATP for formation of the lipoyl-AMP intermediate. Quenched reactions were separated by HPLC and the major product peak was analyzed by mass spectrometry and found to have a mass of 534 Da, consistent with that of lipoyl-AMP (Fig. 3B). To further confirm the identity of this 534 Da product, we subjected it to collision induced dissociation (CID) MS/MS which produced a peak at 346 m/z corresponding to an AMP fragment (Fig. 3C). Similar experiments with LipL2 showed that this enzyme is not capable of synthesizing lipoyl-AMP (Fig. 3B); minor peaks in the LipL2 reaction can also be found in controls (Fig. S3). These results demonstrate that LipL2 cannot form the active lipoyl-AMP intermediate and that only LipL1 is a lipoate ligase.

We next decided to interfere with the ligase activity of LipL1 through site directed mutagenesis. Mutation of an active site lysine residue in E. coli lipoate ligase (EcLplA) abolishes both the adenylation and transfer reactions (Fujiwara et al., 1994, Fujiwara et al., 2010). Using multiple sequence alignments, we were able to identify the corresponding lysine in LipL1 (K160), but not in LipL2. We proceeded to mutate Lys160 in LipL1 (LipL1<sub>K160A</sub>) and express it in our lipoylation deficient JEG3 cell line to assess its activity. We found that LipL1<sub>K160A</sub> is not able to lipoylate E. coli proteins even though we can detect the expression of soluble mutant protein by anti-MBP western blot (Fig. 3D). We then expressed and

This article is protected by copyright. All rights reserved.
purified LipL1<sub>K160A</sub>, and tested its activity against the H-protein in our *in vitro* assay. As expected, this mutant is unable to lipoate the H-protein (Fig. 3E). We also tested the ability of LipL1<sub>K160A</sub> in forming the reaction intermediate lipoyl-AMP as done for LipL1 and LipL2. As expected, we found that LipL1<sub>K160A</sub> is unable to form lipoyl-AMP (Fig. 3B). Taken together, we found that LipL1 lipoate ligase activity depends on an active site lysine for the adenylation reaction, and possibly for the transfer reaction, similar to what has been described for *EcLplA*.

Neither LipL1 nor LipL2 acts as an amidotransferase

Lipoate metabolism has also been described in other systems such as *Bacillus subtilis* (Christensen et al., 2011; Martin et al., 2011). In *B. subtilis*, an enzyme similar to *E. coli* LipB (*BsLipM*) specifically octanoylates the H-protein, but does not modify other protein substrates (Fig. 1B). An amidotransferase is then required to transfer octanoyl (or possibly lipoyl) groups from the H-protein to the remaining four lipoate-dependent proteins in *B. subtilis* (Christensen et al., 2011; Martin et al., 2011). We tested whether LipL1 or LipL2 could act as an amidotransferase in an assay in which the only lipoate source is the holo H-protein. We generated holo H-protein by expressing the *Pf* H-protein and LipL1 in the *E. coli* lipoylation deficient JEG3 cell line. The H-protein was then purified in the holo form by GST affinity chromatography. We tested the transferase activity of either LipL1 or LipL2 and found that neither enzyme can act as an amidotransferase (Fig. 4A, lanes 1–6). Even when LipL1 and LipL2 are combined in the same reaction, they are unable to transfer lipoyl groups from the H-protein to either BCDH<sub>LD</sub> or KDH<sub>LD</sub> (Fig. 4A, lanes 7–9). Thus, neither enzyme appears to be a lipoyl amidotransferase. We did not test LipL1 or LipL2 for the octanoyl amidotransferase activity observed in *B. subtilis* because there is no known source of octanoyl groups (fatty acid biosynthesis or degradation pathways) in the parasite mitochondrion.

LipL1 and LipL2 are both required to lipoylate BCDH<sub>LD</sub> and KDH<sub>LD</sub> in a ligation reaction

This article is protected by copyright. All rights reserved.
We next tested whether both LipL1 and LipL2 are required to ligate free lipoate to BCDH<sub>LD</sub> or KDH<sub>LD</sub> in our *in vitro* ligation assay. In these reactions, equimolar amounts of LipL1 and LipL2 were combined with substrate in the presence or absence of ATP. As shown in Fig. 4B, both enzymes are required to lipoylate BCDH<sub>LD</sub> and KDH<sub>LD</sub> in an ATP-dependent ligation reaction (Fig. S2B). Thus, LipL1 and LipL2 function together as a lipoate ligase for both substrates. Taken together, our results suggest two different routes by which the mitochondrial substrates of *Pf* are lipoylated: one in which LipL1 alone is sufficient to lipoylate the H-protein, and a second route in which both LipL1 and LipL2 are necessary to lipoylate BCDH<sub>LD</sub> and KDH<sub>LD</sub>.

**LipL1 activity is essential for lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub>**

We next addressed the role of LipL1 in the lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub> catalyzed by LipL1 and LipL2. We found that substituting LipL1 with the LipL1<sub>K160A</sub> mutant abolished lipoylation of both substrates (Fig. 4C), showing that LipL1 catalytic activity is essential for the lipoylation of these substrates. We know that LipL1 is needed to catalyze the formation of the lipoyl-AMP intermediate. LipL2 could then transfer the lipoyl group to protein substrates, or it could have a non-catalytic role in recruiting these substrates for lipoylation by LipL1. In an attempt to understand LipL2 activity we decided to mutate residues that could be essential for its activity. As previously mentioned, we found no conserved lysine residue in LipL2 corresponding to LipL1 K160. A second possibility is that LipL2 could transfer lipoyl groups using a cysteine residue, as observed in *Bacillus subtilis* (Christensen et al., 2011; Martin et al., 2011). We mutated two conserved cysteine residues based on alignment against other *Plasmodium* LipL2 enzymes. We expressed and purified these two mutants (LipL2<sub>C93A</sub> and LipL2<sub>C256A</sub>) and tested their activity in our *in vitro* assay. We found that both mutants were still active in our assays, showing that LipL2 activity does not proceed through a thioester intermediate (Fig. 4D). Although LipL2 is required for the lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub>, its role in the reaction remains unclear.

**The two lipoylation routes differ in their redox dependence**

This article is protected by copyright. All rights reserved.The lipoylation assays described in the previous sections were performed using tris(2-carboxyethyl)phosphine (TCEP) as reducing agent. As observed, all substrates can be modified in the presence of TCEP, but we found that the two lipoylation routes differed in their redox dependence. In order to better understand this effect we tested three reducing agents, dithiothreitol (DTT), TCEP, and tris(hydroxypropyl)phosphine (THP) in our *in vitro* ligation assay. We first tested the lipoylation of the H-protein by LipL1 in the presence and absence of the reducing agents. As seen in Fig. 5A, the H-protein can be lipoylated by LipL1 under oxidizing or reducing conditions, albeit at different levels. There was enhanced activity of LipL1 under reducing conditions with DTT as compared to no reducing agent, perhaps because reducing agents in general help to protect lipoate from oxidative damage under aerobic conditions. However, the strong reducing agents TCEP and THP resulted in lower LipL1 activity compared to the weaker reducing agent DTT. The lower activity of LipL1 in the phosphine reducing agents may have to do with the ability of agents like TCEP to reduce the dithiolane ring of lipoate (Burns *et al.*, 1991).

We next tested the activity of both LipL1 and LipL2 in lipoylating BCDH~LD~ and KDH~LD~ under different reducing environments. In contrast to the H-protein, BCDH~LD~ and KDH~LD~ are not lipoylated in the absence of reducing agents or under reducing conditions with DTT (Fig. 5B, lanes 5–8). The LipL1/LipL2 reaction only occurs in the presence of the strong reducing agents TCEP (Fig. 5B, lane 9) or THP (Fig. S4, top panel). However, it is not clear from these results whether these strong reducing agents are acting on the lipoate redox state (Fig. 5C) or affecting the enzymes (LipL1 and/or LipL2) and/or protein substrates (BCDH~LD~ and KDH~LD~).

6,8-dichlorooctanoate (6,8-diClO) is a non-redox sensitive lipoate analog

To further probe the requirements of the *Pf* lipoylation pathways, we synthesized a lipoate analog, 6,8-dichlorooctanoate (6,8-diClO), which mimics the reduced form of lipoate but is not redox active (Fig. 6A). We generated antiserum that is specific for proteins modified with 6,8-diClO and showed that LipL1 can use 6,8-diClO as a substrate to modify the H-protein (Fig. 6B, lower panel). As shown in

This article is protected by copyright. All rights reserved.
Figure 6B, α-LA antibodies and the α-6,8-diClO antiserum are specific for their respective substrates, even when the western blots are significantly overexposed. Using these reagents we tested the different lipoylation routes and their redox dependence in our *in vitro* assay.

When 6,8-diClO was used as a substrate to modify the H-protein, DTT was no longer the most favorable reducing agent and LipL1 activity appeared to be independent of redox conditions (Fig. 6C).

The lower activity with TCEP (also observed in Fig. 5A) probably results from the instability of TCEP in phosphate buffer rather than a redox effect (Hansen *et al.*, 2009; Svagera *et al.*, 2012). The other strong reducing agent THP only decreased LipL1 activity relative to DTT when lipoate, but not 6,8-diClO was used as a substrate, indicating a redox-specific effect on lipoate itself (compare lanes 3 & 7 from Figs. 5A and 6C). A similar phenomenon was observed for the lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub>. LipL1 and LipL2 modified these substrates with 6,8-diClO independent of the redox environment (Fig. 6D and Fig. S4, bottom panel). These results demonstrate that the requirement for the strong phosphine reducing agents in the LipL1/LipL2 lipoylation reactions is actually a requirement for reduced lipoate in these reactions and that the proteins themselves do not need to be reduced in order to be functional (Fig. 5C).

The redox state of lipoate appears to be important for the lipoylation of the different mitochondrial substrates. BCDH<sub>LD</sub> and KDH<sub>LD</sub> are only modified when lipoate is in the reduced state, while the H-protein shows increased lipoylation under less reducing conditions.

### Lipoate analogs are scavenged and attached to mitochondrial substrates

We have previously shown that the lipoate analog 8-BrO inhibits the growth of blood stage *Pf* and that treatment with 8-BrO reduces the lipoylation of the protein substrates found in the mitochondrion of *Pf* (Allary *et al.*, 2007). From these results, it was not clear whether 8-BrO blocked lipoate import, activation and transfer of lipoate; or if attachment of 8-BrO to the mitochondrial substrates inhibited their activity. We know from the above experiments that 6,8-diClO can be attached to all mitochondrial substrates in our *in vitro* assay (see Fig. 6C and D). To determine if 8-BrO can be attached to the parasite proteins, we used the α-6,8-diClO antiserum, which also recognizes proteins modified with 8-BrO.
shown in Fig. 7A, 8-BrO can be attached to all three mitochondrial substrates in an ATP-dependent reaction in our *in vitro* assay, suggesting that both lipoate analogs share a similar mode of action in malaria parasites.

We also treated asexual blood stage parasites with 6,8-diClO at different concentrations as done previously for 8-BrO (Allary *et al.*, 2007). We found that 6,8-diClO significantly inhibits parasite growth at a concentration of 100 μM (Fig. 7B), similar to growth inhibition results observed for 8-BrO. To determine whether this lipoate analog is attached to mitochondrial proteins in the parasites, we treated cultured parasites with 0 or 10 μM 6,8-diClO and probed the lysates with α-6,8-diClO antiserum. Both E2-BCDH and E2-KDH were modified with 6,8-diClO, demonstrating that this compound can be imported into parasites and used by the mitochondrial lipoylation enzymes (Fig. 7C). In parallel experiments, we were able to detect lipoylated E2-BCDH, E2-KDH, and E2-PDH in parasites treated with lipoate (Fig. 7C). Lipoylation of E2-PDH is independent of lipoate scavenging, consistent with the finding that lipoate is synthesized in the apicoplast (Gunther *et al.*, 2005). We were unable to detect modification of the H-protein by either substrate, possibly due to low expression levels of this protein in the parasite.

## Discussion

The mitochondrion of *Plasmodium falciparum* (*Pf*) relies on lipoate scavenging from the host. Treatment of parasites with the lipoate analog 8-bromooctanoate (8-BrO) inhibited parasite growth and specifically blocked the lipoylation of mitochondrial proteins (Allary *et al.*, 2007). These results indicated that lipoate scavenging is essential for the survival of malaria parasites and that the lipoylation machinery in the parasite mitochondrion would likewise be essential. *Plasmodium* spp. possess two putative lipoate ligases in the genome (LipL1 and LipL2) that are believed to catalyze the attachment of free lipoate to the lipoate requiring proteins in an ATP-dependent reaction. In *Pf*, LipL1 was localized to the mitochondrion (Wrenger *et al.*, 2004) while LipL2 was reportedly found in the mitochondrion as well as the apicoplast organelle (Gunther *et al.*, 2007). Although the role of LipL2 in the apicoplast is not well

This article is protected by copyright. All rights reserved.defined, both LipL1 and LipL2 are involved in the lipoylation of three proteins in the mitochondrion of malaria parasites. We have previously shown that one of these proteins, the H-protein, is mitochondrial (Spalding *et al.*, 2010a), and in this work we confirmed the mitochondrial localization of the E2 subunits of BCDH and KDH (Fig. 2A). Thus, the mitochondrion of erythrocytic stage *Pf* contains two putative lipoate ligases responsible for the lipoylation of three protein substrates.

It was previously shown that LipL1 can catalyze the attachment of lipoate to full length H-protein in an ATP-dependent reaction (Allary *et al.*, 2007) and it was assumed that LipL1 could lipoylate all of the mitochondrial substrates. Attempts to disrupt *Pf* LipL1 and the *P. berghei* ortholog failed (Gunther *et al.*, 2009) reinforcing the idea that lipoate scavenging is essential for blood stage malaria parasites and that LipL1 plays a key role in this process. In the present study, we observed lipoylation of the H-protein by LipL1, and found that it preferentially uses ATP as a nucleotide substrate (Fig. S2A). Furthermore, we were able to demonstrate that LipL1 catalyzes formation of the reaction intermediate, lipoyl-AMP, and that LipL1 adenylate activity depends on a conserved lysine residue (Fig. 3). However, LipL1 is unable to lipoylate *Pf* BCDH and *Pf* KDH (Figs. 2D and 3A), while *Ec*LplA is able to lipoylate both parasite proteins (Fig. 2D). It appears that LipL1 can distinguish between the H-protein and the other protein substrates found in malaria parasites. This may be because H-protein sequences are divergent from other E2 lipoyl domains (Christensen *et al.*, 2011) and seem to represent a distinct clade of lipoate-dependent proteins. Indeed, phylometabolomic analysis places the H-protein (and the glycine synthesis pathway) at the base of the tree of life, suggesting that the other lipoyl domains arose subsequently (Braakman *et al.*, 2013).

The specificity of LipL1 for the H-protein is reminiscent of the lipoylation pathways described for *B.* subtilis (Fig. 1B). In *B. subtilis*, an amidotransferase (LipL) with homology to lipoate ligases shuffles the lipoyl or octanoyl moiety from the H-protein to the other protein substrates (Christensen *et al.*, 2011, Martin *et al.*, 2011). We tested LipL1 or LipL2 alone, or in complex in an amidotransferase assay using lipoyl-H-protein as substrate (parasites lack a pathway to generate octanoyl-H-protein), but did not detect any transfer of lipoyl groups to either BCDH or KDH (Fig. 4A). We also investigated whether conserved

This article is protected by copyright. All rights reserved.cysteine residues are essential for LipL2 activity, as has been shown for the *B. subtilis* LipL. Mutation of the only two cysteine residues conserved in *Plasmodium* LipL2 sequences did not affect LipL2 activity (Fig. 4D), further highlighting the differences between LipL2 and BsLipL. We also considered a recent report showing that a lipoate ligase homolog in *Saccharomyces cerevisiae* (Lip3) transfers octanoyl groups from octanoyl-CoA to protein substrates followed by sulfur insertion by a lipoate synthase (Lip5) (Schonauer *et al.*, 2009). We did not test LipL1 or LipL2 for this type of octanoyltransferase activity since the mitochondrion of malaria parasites does not contain a lipoate synthase or a biochemical pathway likely to produce octanoyl-CoA.

The lipoylation of BCDH and KDH occurs through a novel mechanism requiring both LipL1 and LipL2 (Fig. 4B). We demonstrated that LipL1 is responsible for catalyzing activation of lipoate, forming lipoyl-AMP, and that LipL2 is not capable of catalyzing this reaction (Fig. 3B and 3C). We generated a catalytically inactive mutant of LipL1 unable to form lipoyl-AMP by substituting an active site lysine (K160) with alanine, as has been done previously with *Ec*LplA (Fujiwara *et al.*, 1994, Fujiwara *et al.*, 2010). The catalytic activity of LipL1 is essential for the lipoylation of the H-protein (Fig. 3E) as well as for the combined LipL1/LipL2 lipoylation of BCDH and KDH (Fig 4C). The role of LipL2 is less clear. It is possible that LipL2 catalyzes the transfer of lipoyl groups from lipoyl-AMP to BCDH and KDH. It is also possible that LipL2 does not have a catalytic role, but instead acts as an effector helping to recruit protein substrates that would otherwise not be recognized by LipL1. In either case, the LipL1/LipL2 reaction defines a second lipoylation pathway in the parasite mitochondrion (Fig. 7D). Attempts to disrupt the gene encoding *Pf* LipL2 have so far been unsuccessful (Storm *et al.*, 2012), suggesting that this second lipoylation pathway is essential for the survival of blood stage malaria parasites.

The two mitochondrial lipoylation pathways are sensitive to redox conditions. Lipoylation of H-protein by LipL1 in the first pathway is significantly enhanced in the presence of DTT, but is less active in the presence of strong reducing agents such as TCEP and THP (Fig. 5A). The decreased lipoylation of H-protein in the presence of TCEP as compared to THP is likely due to the reactivity of TCEP in phosphate buffer (Hansen *et al.*, 2009, Svagera *et al.*, 2012). In contrast to the first pathway, the

This article is protected by copyright. All rights reserved.lipoylation of BCDH and KDH by the second pathway only occurs in the presence of TCEP or THP (Fig. 5B and S4). The redox potential for both DTT and lipoate has been measured to be -0.33 V and -0.32 V, respectively (Haramaki *et al.*, 1997, Hanson *et al.*, 2004). Currently, there is no measurement of redox potential for TCEP or THP, but experimental data show that TCEP readily reduces both DTT and lipoate (Burns *et al.*, 1991). In the absence of reducing agent, lipoate is found in the oxidized, closed-ring form, but upon reduction with TCEP or THP it presumably adopts the open-ring dihydrolipoate form (Fig. 5D).

We synthesized a structural analog (6,8-diClO) of dihydrolipoate and used this compound to probe the redox requirements of the second lipoylation pathway. Using 6,8-diClO as a substrate, all three proteins were modified regardless of the redox conditions (Fig. 6C and 6D). This demonstrates that the second pathway requires dihydrolipoate, and that the LipL1 and LipL2 enzymes are not themselves sensitive to redox conditions. These results have interesting connotations for how lipoylation may work and why two different routes are needed in the mitochondrion of a malaria parasite.

Our results suggest that dihydrolipoate is needed in the mitochondrion for the lipoylation of BCDH and KDH. Therefore, the redox potential of the parasite mitochondrion needs to be highly reducing or the parasites must employ a mechanism to reduce lipoate to dihydrolipoate. While there is no measurement of the redox potential for the *Plasmodium* mitochondrion, the redox potential of the mitochondrion of HeLa cells has been reported to be -0.36 V (Hanson *et al.*, 2004). This value is more reducing than that of lipoate, and it might be sufficient to reduce lipoate to dihydrolipoate and/or maintain lipoate in the reduced state. *Pf* has two redox systems: the glutathione and thioredoxin systems (Jortzik *et al.*, 2012). Six proteins from these systems have been localized to the mitochondrion: thioredoxin reductase (TrxR—dually localized to cytosol and mitochondrion), thioredoxin-like protein 2 (Tlp2), peroxiredoxin (Prx1m), glutaredoxin-like protein (Glp3), and peroxidase-like thioredoxin peroxidase (TPx~G1~—localized to the cytosol, mitochondria and apicoplast) (Chaudhari *et al.*, 2012, Jortzik *et al.*, 2012). These proteins are likely involved in maintaining the redox potential of the mitochondrion and/or defending against oxidative stress. It would be interesting to determine whether any of these proteins have the additional role of reducing lipoate. It has been shown that the thioredoxin reductase can reduce lipoate in an *in vitro*

This article is protected by copyright. All rights reserved.
system (Snider *et al.*, 2014). In addition to these redox systems, the dihydrolipoamide dehydrogenase (LipDH, PF3D7_1232200) likely recognizes free lipoate and reduces it. This has been observed for mammalian systems, in which lipoate is converted to dihydrolipoate in a NADH-dependent reaction in isolated mitochondrion (Constantinescu *et al.*, 1995, Arner *et al.*, 1996, Haramaki *et al.*, 1997).

In the current study we show that lipoate metabolism in the mitochondrion of *Pf* follows two lipoylation routes (Fig. 7D). In the first route, lipoylation of H-protein is catalyzed solely by the lipoate ligase LipL1. In the second route, both LipL1 and LipL2 are required to lipoylate the BCDH and KDH. LipL1 catalyzes the adenylation reaction, while LipL2 could act as a lipoyl transferase or an effector protein. The two lipoylation routes are dependent on the redox state of lipoate, a phenomenon that has not been previously observed in any other system. Furthermore, we show that all three mitochondrial substrates can be modified by lipoate analogs (8-BrO and 6,8-diClO) suggesting that the lethality of these compounds results from their attachment to essential parasite proteins by lipoylation enzymes. Our results suggest that one or more of the mitochondrial protein substrates and by extension, LipL1 and LipL2, are essential for parasite survival making mitochondrial lipoylation in *Plasmodium* an attractive target for therapeutic intervention.

## Experimental Procedures

### Materials

The reducing agents used in this study were obtained from Fluka (TCEP; tris(2-carboxyethyl)phosphine), Calbiochem (THP; tris(hydroxypropyl)phosphine), and Sigma-Aldrich (DTT; dithiothreitol and DTBA; (S)-2-Aminobutane-1,4-dithiol). Antibodies were purchased from Calbiochem (rabbit α-LA polyclonal antibodies), New England Biolabs (murine α-MBP monoclonal antibody HRP conjugated) and GE Healthcare (donkey α-Rabbit IgG HRP conjugate). The following chemicals were purchased from Sigma-Aldrich: 8-BrO, NADH, NAD⁺ and R-LA. The compound 6,8-diClO and the α-6,8-diClO antiserum were synthesized as described below.

Expression plasmids

The malaria genome resource PlasmoDB (Bahl et al., 2003) was used to identify the genes encoding the E2 subunits of BCDH (PF3D7_0303700) and KDH (PF3D7_1320800). Nucleotides encoding the first 30 amino acids of both proteins were amplified from cDNA extracted from erythrocytic stage *Plasmodium falciparum* 3D7 parasites using primers BCDH₃₀.AvrII.F and BCDH₃₀.BsiWI.R or KDH₃₀.AvrII.F and KDH₃₀.BsiWI.R (Table S1). The BCDH₃₀ and KDH₃₀ amplicons were digested with AvrII and BsiWI and ligated into parasite transfection plasmid pRL2 (Gisselberg et al., 2013) in frame with GFP, generating plasmids pRL2-BCDH₃₀GFP and pRL2-KDH₃₀GFP. Plasmid pRL2 is a variant of pLN (Nkrumah et al., 2006) in which the calmodulin promoter has been replaced with the weaker ribosomal protein L2 promoter (Balabaskaran Nina et al., 2011).

Ampicillin-resistant plasmid pGEXT (Delli-Bovi et al., 2010) was used to express lipoylation domains with an amino-terminal glutathione S-transferase (GST) tag. The nucleotides encoding the lipoylation domain of BCDH (residues V34-L139) and KDH (residues I35-N152) were amplified using primers BCDH.BamHI.F100 and BCDH.EcoRI.R417 or KDH.BamHI.F103 and KDH.EcoRI.R456 (Table S1). The BCDH_LD and KDH_LD amplicons were digested with EcoRI and BamHI and ligated into pGEXT, generating plasmids pGEXT-BCDH_LD and pGEXT-KDH_LD.

A kanamycin-resistant expression plasmid, pGEXTK, was generated by subcloning the kanamycin cassette from plasmid pRK586 (Kapust et al., 2000) into the *Bsp*HI sites of pGEXT using primers Kan.BspHI.F and Kan.BspHI.R (Table S1). The nucleotides encoding BCDH_LD and KDH_LD were subcloned from pGEXT to pGEXTK using BamHI and SalI, generating plasmids pGEXTK-BCDH_LD and pGEXTK-KDH_LD. Nucleotides encoding residues E35-K200 (lacking the predicted N-terminal mitochondrial transit peptide) of the H-protein (PF3D7_1132900) were amplified from an expression plasmid similar to pMA007 (Allary et al., 2007) with primers Hprot.BamHI.F and Hprot.SalI.R (Table S1). The resulting amplicon was digested with BamHI and SalI, and ligated into pGEXTK, generating plasmid pGEXTK-Hprot.
Expression plasmid pMALcHT (Muench *et al.*, 2003, Allary *et al.*, 2007) was used to express lipoylation enzymes as maltose binding protein (MBP) fusion proteins with a linker region composed of a tobacco etch virus (TEV) protease cut site followed by a six histidine affinity tag. Plasmid pMA006 (Allary *et al.*, 2007) expressing LipL1 (PF3D7_1314600) was renamed pMALcHT-LipL1 for this study.

The LipL1<sub>K160A</sub> mutant created by site-directed mutagenesis of pMALcHT-LipL1 using primers LipL1<sub>K160A</sub>.F and LipL1<sub>K160A</sub>.R (Table S1) generating plasmid pMALcHT-LipL1<sub>K160A</sub>. A codon harmonized gene (Fig. S5) encoding LipL2 (PF3D7_0923600) was synthesized by GeneArt and ligated to pMALcHT using the EcoRI and HindIII sites, generating pMALcHT-LipL2. Both LipL2 mutants, C93A and C276A, were created by site-directed mutagenesis of pMALcHT-LipL2 generating pMALcHT-LipL2<sub>C93A</sub> and pMALcHT-LipL2<sub>C276A</sub> (See Table S1 for primer sequences). The gene encoding *E. coli* lipopate ligase, EcLplA, was amplified from BL21 gDNA with primers LplA.BamHI.F and LplA.Sall.R (Table S1), digested with BamHI and Sall, and ligated to pMALcHT, generating pMALcHT-LplA.

### Construction of lplA-/lipB- *E. coli* (JEG3 strain)

In order to construct the JEG3 cell line a *lipB-* *E. coli* strain was obtained from the Keio deletion collection (Keio JW5089). The kanamycin resistance cassette was removed by transforming JW5089 cells with the ampicillin resistant pCP20 Flp recombinase plasmid (Cherepanov *et al.*, 1995). Colonies were selected on kanamycin and ampicillin and incubated at 30 degrees. One colony was transferred to liquid LB medium without antibiotic and incubated overnight at 42 degrees in order to induce the loss of the pCP20 plasmid. Colonies were selected on LB agar plates followed by replica plating on LB plates containing LB alone, ampicillin, or kanamycin to screen for colonies which had successfully lost both the kanamycin resistance cassette and the pCP20 plasmid. This kanamycin sensitive *lipB-* strain was called JEG1. Primers LplA.H.F and LplA.H.R (Table S1) were designed based on the homologous regions flanking the *lplA* gene originally described by Baba and coworkers (Baba *et al.*, 2006). These primers were used to amplify the kanamycin cassette flanked by FRT sites which had previously been inserted into the *lplA* locus in *lplA*-Keio strain JW4349. JEG1 cells were transformed with plasmid pKD46

This article is protected by copyright. All rights reserved.which expresses λ Red recombinase (GenBank Accession number AY048746) (Datsenko *et al.*, 2000).

Chemically competent JEG1/pKD46 cells were prepared and transformed with 400 ng of the PCR product generated above and grown in SOC medium with 1 mM L-arabinose for 2 hours at 37 degrees, after which they were selected with kanamycin on LB agar plates. Three colonies were screened for the *lplA* deletion by PCR using primers specific for the kanamycin cassette (K1.int.F and K2.int.R, Table S1) in combination with primers upstream and downstream of *lplA* (LplA.U and LplA.D, Table S1). This *lipB*-: Kan<sup>S</sup>, *lplA*-: Kan<sup>R</sup> line was called JEG2. A kanamycin sensitive line was generated as described above using the pCP20 plasmid and called JEG3. Primers LipB.U and LipB.D (Table S1) flanking the *lipB* locus were used in conjunction with primers LplA.U and LplA.D to verify the deletion of both genes in strain JEG3.

### Parasite culture for protein localization

*P. falciparum* asexual blood stage cultures were maintained at 2% hematocrit in complete medium consisting of RPMI 1640 medium with L-glutamine (Gibco) supplemented with 10% human serum (Interstate Blood Bank, Memphis, TN), 25 mM HEPES, 0.24% NaHCO<sub>3</sub>, and 12.5 μg/ml hypoxanthine (Trager *et al.*, 1997). Site-specific integration mediated by the mycobacteriophage Bxb1 integrase was used to generate cell lines with the pRL2-BCDH<sub>30</sub>GFP and pRL2-KDH<sub>30</sub>GFP plasmids described above that were integrated into the attB locus of Dd2<sup>attB</sup> parasites (Nkrumah *et al.*, 2006). Transfection of parasites was conducted as described previously (Spalding *et al.*, 2010a). Cell lines were analyzed for integration at the attB site by PCR analysis of genomic DNA using primers that flank the 5′ and 3′ integration sites. Parasites were imaged on a Nikon Eclipse E800 epifluorescence microscope. For live fluorescence, cells were incubated with 100 nM MitoTracker CMX-Ros (Invitrogen) and 1 μg/ml Hoechst 33258 to visualize parasite mitochondria and nuclei.

### Protein expression and purification

This article is protected by copyright. All rights reserved.
Plasmids pMALcHT-LipL1 and pMALcHT-LipL1<sub>K160A</sub> were transformed into BL21-Star (DE3) cells (Invitrogen) and co-transformed with the pRIL plasmid isolated from BL21-CodonPlus-RIL cells (Agilent) and plasmid pRK586 encoding the Tobacco Etch Virus (TEV) protease (Kapust et al., 2000), as previously described (Allary et al., 2007). These cells produce LipL1 or LipL1<sub>K160A</sub> fused to an amino-terminal six histidine-tag. Transformed cells were grown to mid-log phase, and the expression of recombinant proteins was induced by the addition of 0.4 mM IPTG. Cells were harvested after growth for 10 h at 20°C. LipL1 and LipL1<sub>K160A</sub> were purified by metal chelate chromatography followed by cation exchange chromatography; LipL1 was further purified by gel filtration chromatography. Plasmids pMALcHT-LipL2, pMALcHT-LipL2<sub>C93A</sub> or pMALcHT-LipL2<sub>C276A</sub> were transformed into BL21-Star (DE3) and co-transformed with the pRIL plasmid. These cells produce LipL2, LipL2<sub>C93A</sub> or LipL2<sub>C276A</sub> fused to an amino-terminal MBP tag. LipL2, LipL2<sub>C93A</sub> and LipL2<sub>C276A</sub> were affinity purified using amylose resin (NEB #E8021).

The plasmids pGEXT-BCDH<sub>LD</sub>, pGEXT-KDH<sub>LD</sub> or pGEXTK-Hprot were transformed into JEG3 cells containing the pRIL plasmid. These cells produced BCDH<sub>LD</sub>, KDH<sub>LD</sub> or the H-protein in the apo-form (non-lipoylated) with an amino-terminal GST fusion protein. The proteins were purified by GSTrap FF chromatography (GE Healthcare). All cultures were grown and harvested as described for LipL1, with the exception that JEG3 cultures were supplemented with 5 mM sodium succinate, 5 mM sodium acetate and 1% glucose. All purifications were conducted with 20 mM HEPES, 100 mM NaCl at pH 7.5 as the base buffer. For GST purifications, 1 mM DTT was used during the steps of lysis and purification. The concentration of proteins after purification was determined by Bradford assay.

### Cell-based lipoylation assay

The JEG3 lipoylation deficient *E. coli* strain was transformed with the pRIL plasmid and grown in LB medium supplemented with 1% glucose as a carbon source and 35 μg ml<sup>-1</sup> chloramphenicol. Sodium succinate (5 mM) and sodium acetate (5 mM) were added to bypass the requirement for KDH and PDH

activity respectively. JEG3/pRIL was transformed with a plasmid expressing a candidate lipoate ligase (either pMALcHT-LipL1, pMALcHT-LipL2, pMALcHT-EcLplA or pMALcHT-LipL1<sub>K160A</sub>) and/or a plasmid expressing a lipoylation substrate (pGEXTK-BCDH<sub>LD</sub> or pGEXTK-KDH<sub>LD</sub>) and selected using 100 μg ml<sup>-1</sup> ampicillin and/or 50 μg ml<sup>-1</sup> kanamycin. Transformants were grown in LB medium and supplemented as described above with the addition of 200 μM R-lipoic acid (Sigma). Plasmids pGEXTK-BCDH<sub>LD</sub> and pGEXTK-KDH<sub>LD</sub> were also transformed into BL21-Star (DE3) cells containing the pRIL plasmid and maintained in LB medium containing 35 μg ml<sup>-1</sup> chloramphenicol and 50 μg ml<sup>-1</sup> kanamycin. For the cell-based assay 20 mL cultures were grown to mid-log phase at 37 degrees and induced with 0.4 mM IPTG for 10 hours at 20 degrees. Cells were harvested by centrifugation and resuspended with 0.5 mL of buffer containing 20 mM HEPES, 100 mM NaCl at pH 7.5 and lysed by sonication. The cell lysates were clarified by centrifugation at 16,000 g and the supernatants were collected and resolved by SDS-PAGE (Invitrogen). Lipoylated proteins were visualized by western blot as described in lipoate ligase assay section. Blots were stripped by using a buffer containing 100 mM BME, 2% SDS in 62.5 mM Tris-HCl pH 7.5 at 55 degrees. After stripping, blots were blocked with 5% milk/PBS and probed with 1:5,000 murine α-MBP monoclonal antibody (HRP conjugated) in 1% milk/PBS for 1 hour at room temperature. The membranes were visualized with enhanced chemiluminescence (ECL) western substrate (Pierce) and exposed to film.

### HPLC and Mass Spectroscopy analysis of LipL1 and LipL2

HPLC analyses were performed on a Beckman Gold Nouveau System with a Grace Alltima 3 μC<sub>18</sub> Analytical Rocket column (53 mm x 7 mm). Electrospray ionization mass spectrometry (ESI-MS) was performed on a Thermo linear triple quadrupole (LTQ) XL mass spectrometer. Samples were analyzed in negative ion mode.

Reaction mixtures containing 2 mM ATP, 2 mM MgCl<sub>2</sub>, 1 mM R-LA, 5 mM TCEP in 100 mM phosphate buffer 150 mM NaCl pH 7.5 were initiated by addition of LipL1, LipL1<sub>K160A</sub> or LipL2 (3.2 mg/mL) and incubated at room temperature for 10 minutes. Reactions were quenched with ice cold

methanol (1:1 v/v with reaction mixture). Precipitated protein was removed by centrifugation at 15,000 g for 15 minutes at 4°C. The supernatant was then subjected to HPLC analysis with UV detection using the following conditions: flow rate of 3 mL/min; solvent A: Et₃NHOAc (50 mM, pH 6); solvent B: acetonitrile; method: 5% B for 3 minutes followed by a gradient of 5% to 100% B over 7 minutes. Peaks of interest were collected manually and analyzed by ESI-MS in negative ionization mode. MS/MS analysis was performed by selecting 535 ± 2 m/z for collision induced dissociation (CID) at 13% normalized collision energy.

### Synthesis of 6,8-dichlorooctanoic acid (6,8-diClO)

The preparation of 6,8-diClO was carried out using the ester hydrolysis conditions previously reported for similar analogs (Woster *et al.*, 1989). All reagents were obtained from commercial sources. A solution of ethyl 6,8-dichlorooctanoate (500 mg, 2 mmol) in EtOH (1 mL) was added to a suspension of LiOH (72 mg, 3.0 mmol) in 50:50 EtOH/water (1 mL) at room temperature. The mixture was stirred at room temperature and monitored by thin layer chromatography (CH₂Cl₂) until the ester hydrolysis reaction appeared to be complete (2 h). Water was added (20 mL), and the resulting mixture was extracted with EtOAc (3 × 10 mL). The aqueous layer containing the product was acidified to pH 2 using 1N HCl, and extracted with EtOAc (3 × 20 mL). 6,8-Dichlorooctanoic acid was obtained as an oil (386 mg, 88%, Rf = 0.76 in CH₂Cl₂). NMR characterization was performed on a 400 MHz Varian instrument. Chemical shifts are reported in parts per million from tetramethylsilane. HRMS characterization was carried out at the University of Illinois Mass Spectrometry Lab using ES ionization. ¹H NMR (CDCl₃) δ 4.10 (m, 1H); 3.70 (m, 2H); 2.37 (t, 2H); 2.11 (m, 2H); 1.76 (q, 2H); 1.65 (m, 3H), 1.50 (m, 1H). HRMS: calcd. for C₈H₁₄O₂Cl₂: m/z 235.0277 [M + Na]⁺; found: 235.0269.

### Lipoate ligase assay

Purified LipL1, LipL1_K160A, LipL2, LipL2_C93A or LipL2_C276A (1 μM) was incubated in reaction buffer (100 mM Na/K Phosphate buffer, 150 mM NaCl at pH 7.5) containing 2 mM ATP, 2 mM MgCl₂, 5 mM

This article is protected by copyright. All rights reserved.
TCEP, 200 μM R-lipoic acid and 10 μM apo-protein substrate: H-protein, BCDH<sub>LD</sub>, or KDH<sub>LD</sub>. After incubation at 37 degrees for 1 hour, the reactions were terminated with the addition of gel loading buffer and analyzed by SDS-PAGE followed by transfer to nitrocellulose membrane. The membranes were blocked with 5% milk in PBS for 30 minutes, and probed with 1:5000 rabbit polyclonal α-LA (Calbiochem) for 2 hours in 1% milk/PBS at room temperature. The membrane was washed with 1xPBS three times and then probed with 1:5000 donkey α-Rabbit IgG horseradish peroxidase (HRP) secondary antibody (GE Healthcare) in 1% milk/PBS overnight at 4 degrees. The membranes were visualized with enhanced chemiluminescence (ECL) western substrate (Pierce) and exposed to film.

All lipoylation assays were conducted similarly using the same protein concentrations and base buffer. Where indicated, ATP was omitted from the reaction and 5 mM of DTT, THP or water was used instead of TCEP. R-lipoic acid was substituted with 200 μM 6,8-dichlorooctanoate or 200 μM 8-bromoctanoate where indicated, and in these reactions 1:5000 rabbit polyclonal α-6,8-diClO was used as the primary probe for western blots.

### Amidotransferase assay

Lipoylated H-protein (Holo H-protein) was generated by co-transforming pGEXTK-Hprot and pMALcHT-LipL1 in JEG3 cells containing the pRIL plasmid. These cells express the H-protein with an amino-terminal GST tag and LipL1 with an amino-terminal MBP. One liter cultures were grown to mid-log phase and induced with 0.4 mM IPTG for 10 hours at 20 degrees. Cells were harvested as described above and Holo H-protein was purified by affinity chromatography using a GSTrap FF column (GE Healthcare), followed by anion exchange using a MonoQ column (Pharmacia). The protein concentration was determined by Bradford assay. A final concentration of 10 μM of Holo H-protein was used in all assays, in the presence of 1 μM of LipL1 and/or LipL2 and 10 μM of BCDH<sub>LD</sub> or KDH<sub>LD</sub>, 2 mM MgCl<sub>2</sub>, 5 mM TCEP in 100 mM Na/K Phosphate buffer, 150 mM NaCl at pH 7.5. The reactions were incubated and analyzed as described in the lipoate ligase assay.

This article is protected by copyright. All rights reserved.
Production of rabbit α-6,8-diClO antiserum

Compound 6,8-diClO was conjugated to keyhole limpet hemocyanin using the Inject EDC mcKLH Spin Kit (Pierce). Antibodies to the conjugate were generated in rabbits using the standard protocol of the custom antibody service Cocalico Biologicals (Reamstown, PA). The resulting antiserum was used in western blots where indicated with the label α-6,8-diClO to probe for proteins modified with either 8-bromoctanoate or 6,8-dichlorooctanoate.

Parasite culture for 6,8-diClO growth inhibition assay and labeling

All *Plasmodium falciparum* parasite lines for these experiments were grown as previously described (Allary et al., 2007) with the following exceptions: complete media was supplemented with 10% Albumax (Gibco) and parasites were maintained under a 3% O₂ and 3% CO₂ gas mixture. To assess parasite growth in the presence of 6,8-diClO, Dd2<sup>attB</sup> parasites were synchronized with 5% sorbitol (Lambros et al., 1979) and seeded into 6-well plates at 0.05% parasitemia. Parasites were treated with 0, 25, 100, 400, and 1000 μM 6,8-diClO. All dilutions of 6,8-diClO were prepared in DMSO so that the final amount of DMSO in the parasite culture was 0.05%. For all experiments culture media and drug were added fresh each day. Blood smears were made every other day to monitor parasitemia.

For western blot analysis, asynchronous 3D7 parasites were seeded at 1% parasitemia and treated with 10 μM 6,8-diClO, 2 μM R-lipoic acid in 10% EtOH or vehicle for three days. Parasites were isolated from red blood cells using 0.5% saponin in PBS as previously described (Allary et al., 2007). Parasite pellets were resuspended in RIPA Buffer (Boston BioProducts) supplemented with protease inhibitors (Complete Mini EDTA-free, Roche) and lysed by vigorous vortexing. Insoluble debris was pelleted at 10,000 g and total protein of the supernatant was measured by Bradford assay. Loading dye was added to the supernatant and samples were boiled for 5 min at 95 degrees and separated on a 4–12% gradient gel (Invitrogen) by SDS-PAGE. Western blots were conducted as described above using 1:5000 rabbit α-6,8-diClO or 1:1000 rabbit α-LA (Calbiochem) as the primary probes. Membranes were stripped and re-probed with 1:10,000 mouse α-PfHsp70 (Kumar et al., 1998) as a loading control.

Acknowledgments

This work was supported by the National Institutes of Health (R56 AI065853 to STP and R01 GM084998 to CLFM and AI099704 DB), and the Johns Hopkins Malaria Research Institute and the Bloomberg Family Foundation. This work was also made possible by UL1 TR001079 from the National Institutes of Health (NIH) National Center for Research Resources. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.

We thank Swetha Velivela and Marina Allary for help with the initial cloning of the lipoylation substrates. We also thank John Cronan for *E. coli* strain TM136 and the National BioResource Project (NIG, Japan) for strains JW5089 and JW4349. Plasmids pKD46 and pCP20 were provided by the Yale University CGSC (Coli Genetic Stock Center) which is supported by NSF program DBI-0742708. We also thank Roger McMacken for the gift of rabbit α-*EcHsp70* antiserum and Nirbhay Kumar for mouse α-*PfHsp70* antiserum.

References

Allary, M., Lu, J.Z., Zhu, L. and Prigge, S.T. (2007). Scavenging of the cofactor lipoate is essential for the survival of the malaria parasite *Plasmodium falciparum*. *Mol Microbiol* 63, 1331-1344.

Arner, E.S., Nordberg, J. and Holmgren, A. (1996). Efficient reduction of lipoamide and lipoic acid by mammalian thioredoxin reductase. *Biochem Biophys Res Commun* 225, 268-274.

Baba, T., Ara, T., Hasegawa, M., Takai, Y., Okumura, Y., Baba, M., *et al.* (2006). Construction of *Escherichia coli* K-12 in-frame, single-gene knockout mutants: the Keio collection. *Mol Syst Biol* 2, 2006 0008.

Bahle, A., Brunk, B., Crabtree, J., Fraunholz, M.J., Gajria, B., Grant, G.R., *et al.* (2003). PlasmoDB: the *Plasmodium* genome resource. A database integrating experimental and computational data. *Nucleic Acids Res* 31, 212-215.

Balabaskaran Nina, P., Morrisey, J.M., Ganesan, S.M., Ke, H., Pershing, A.M., Mather, M.W. and Vaidya, A.B. (2011). ATP synthase complex of Plasmodium falciparum: dimeric assembly in mitochondrial membranes and resistance to genetic disruption. *J Biol Chem* 286, 41312-41322.

Braakman, R. and Smith, E. (2013). The compositional and evolutionary logic of metabolism. *Phys Biol* 10, 011001.

Burns, J.A., Butler, J.C., Moran, J. and Whitesides, G.M. (1991). Selective Reduction of Disulfides by Tris(2-Carboxyethyl)Phosphine. *Journal of Organic Chemistry* 56, 2648-2650.

Chaudhari, R., Narayan, A. and Patankar, S. (2012). A novel trafficking pathway in Plasmodium falciparum for the organellar localization of glutathione peroxidase-like thioredoxin peroxidase. *FEBS J* 279, 3872-3888.

Cherapanov, P.P. and Wackernagel, W. (1995). Gene disruption in Escherichia coli: TcR and KmR cassettes with the option of Flp-catalyzed excision of the antibiotic-resistance determinant. *Gene* 158, 9-14.

Christensen, Q.H., Martin, N., Mansilla, M.C., de Mendoza, D. and Cronan, J.E. (2011). A novel amidotransferase required for lipoic acid cofactor assembly in Bacillus subtilis. *Mol Microbiol* 80, 350-363.

Cobbold, S.A., Vaughan, A.M., Lewis, I.A., Painter, H.J., Camargo, N., Perlman, D.H., *et al.* (2013). Kinetic flux profiling elucidates two independent acetyl-CoA biosynthetic pathways in Plasmodium falciparum. *J Biol Chem.*

Constantinescu, A., Pick, U., Handelman, G.J., Haramaki, N., Han, D., Podda, M., *et al.* (1995). Reduction and transport of lipoic acid by human erythrocytes. *Biochem Pharmacol* 50, 253-261.

Datsenko, K.A. and Wanner, B.L. (2000). One-step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. *Proc Natl Acad Sci USA* 97, 6640-6645.

Delli-Bovi, T.A., Spalding, M.D. and Prigge, S.T. (2010). Overexpression of biotin synthase and biotin ligase is required for efficient generation of sulfur-35 labeled biotin in E. coli. *BMC Biotechnol* 10, 73.

Dondorp, A.M., Fairhurst, R.M., Slutsker, L., Macarthur, J.R., Breman, J.G., Guerin, P.J., *et al.* (2011). The threat of artemisinin-resistant malaria. *N Engl J Med* 365, 1073-1075.

Falkard, B., Kumar, T.R., Hecht, L.S., Matthews, K.A., Henrich, P.P., Gulati, S., *et al.* (2013). A key role for lipoic acid synthesis during Plasmodium liver stage development. *Cell Microbiol.*

Fidock, D.A., Eastman, R.T., Ward, S.A. and Meshnick, S.R. (2008). Recent highlights in antimalarial drug resistance and chemotherapy research. *Trends Parasitol* 24, 537-544.

Foth, B.J., Stimmler, L.M., Handman, E., Crabb, B.S., Hodder, A.N. and McFadden, G.I. (2005). The malaria parasite Plasmodium falciparum has only one pyruvate dehydrogenase complex, which is located in the apicoplast. *Mol Microbiol* 55, 39-53.

Fujiwara, K., Maita, N., Hosaka, H., Okamura-Ikeda, K., Nakagawa, A. and Taniguchi, H. (2010). Global conformational change associated with the two-step reaction catalyzed by Escherichia coli lipoate-protein ligase A. *J Biol Chem* 285, 9971-9980.

Fujiwara, K., Okamura-Ikeda, K. and Motokawa, Y. (1994). Purification and characterization of lipoyl-AMP:N epsilon-lysine lipoyltransferase from bovine liver mitochondria. *J Biol Chem* 269, 16605-16609.

Gisselberg, J.E., Dellibovi-Ragheb, T.A., Matthews, K.A., Bosch, G. and Prigge, S.T. (2013). The Suf Iron-Sulfur Cluster Synthesis Pathway Is Required for Apicoplast Maintenance in Malaria Parasites. *PLoS Pathog* 9, e1003655.

Gunther, S., Matuschewski, K. and Muller, S. (2009). Knockout studies reveal an important role of Plasmodium lipoic acid protein ligase A1 for asexual blood stage parasite survival. *PLoS One* 4, e5510.

Gunther, S., McMillan, P.J., Wallace, L.J. and Muller, S. (2005). Plasmodium falciparum possesses organelle-specific alpha-keto acid dehydrogenase complexes and lipoylation pathways. *Biochem Soc Trans* 33, 977-980.

Gunther, S., Wallace, L., Patzewitz, E.M., McMillan, P.J., Storm, J., Wrenger, C., *et al.* (2007). Apicoplast lipoic acid protein ligase B is not essential for Plasmodium falciparum. *PLoS Pathog* 3, e189.

Hansen, R.E. and Winther, J.R. (2009). An introduction to methods for analyzing thiols and disulfides: Reactions, reagents, and practical considerations. *Anal Biochem* 394, 147-158.

Hanson, G.T., Aggeler, R., Oglesbee, D., Cannon, M., Capaldi, R.A., Tsien, R.Y. and Remington, S.J. (2004). Investigating mitochondrial redox potential with redox-sensitive green fluorescent protein indicators. *J Biol Chem* 279, 13044-13053.

Haramaki, N., Han, D., Handelman, G.J., Tritschler, H.J. and Packer, L. (1997). Cytosolic and mitochondrial systems for NADH- and NADPH-dependent reduction of alpha-lipoic acid. *Free Radic Biol Med* 22, 535-542.

Hay, S.I., Guerra, C.A., Gething, P.W., Patil, A.P., Tatem, A.J., Noor, A.M., *et al.* (2009). A world malaria map: Plasmodium falciparum endemicity in 2007. *PLoS Med* 6, e1000048.

Jordan, S.W. and Cronan, J.E. (2003). The Escherichia coli lipB gene encodes lipoyl (octanoyl)-acyl carrier protein : protein transferase. *Journal of Bacteriology* 185, 1582-1589.

Jortzik, E. and Becker, K. (2012). Thioredoxin and glutathione systems in Plasmodium falciparum. *Int J Med Microbiol* 302, 187-194.

Kapust, R.B. and Waugh, D.S. (2000). Controlled intracellular processing of fusion proteins by TEV protease. *Protein Expr Purif* 19, 312-318.

Kumar, N. and Zheng, H. (1998). Evidence for epitope-specific thymus-independent response against a repeat sequence in a protein antigen. *Immunology* 94, 28-34.

Lambros, C. and Vanderberg, J.P. (1979). Synchronization of Plasmodium falciparum erythrocytic stages in culture. *J Parasitol* 65, 418-420.

Martin, N., Christensen, Q.H., Mansilla, M.C., Cronan, J.E. and de Mendoza, D. (2011). A novel two-gene requirement for the octanoyltransfer reaction of Bacillus subtilis lipoic acid biosynthesis. *Mol Microbiol* 80, 335-349.

McMillan, P.J., Stimmler, L.M., Foth, B.J., McFadden, G.I. and Muller, S. (2005). The human malaria parasite Plasmodium falciparum possesses two distinct dihydrolipoamide dehydrogenases. *Mol Microbiol* 55, 27-38.

Miao, J., Lawrence, M., Jeffers, V., Zhao, F., Parker, D., Ge, Y., et al. (2013). Extensive lysine acetylation occurs in evolutionarily conserved metabolic pathways and parasite-specific functions during Plasmodium falciparum intraerythrocytic development. *Mol Microbiol* 89, 660-675.

Miller, J.R., Busby, R.W., Jordan, S.W., Cheek, J., Henshaw, T.F., Ashley, G.W., et al. (2000). Escherichia coli LipA is a lipoyl synthase: In vitro biosynthesis of lipoylated pyruvate dehydrogenase complex from octanoyl-acyl carrier protein. *Biochemistry* 39, 15166-15178.

Morris, T.W., Reed, K.E. and Cronan, J.E. (1995). Lipoic Acid Metabolism in Escherichia-Coli - the Lpla and Lipb Genes Define Redundant Pathways for Ligation of Lipoyl Groups to Apoprotein. *Journal of Bacteriology* 177, 1-10.

Mudhune, S.A., Okiro, E.A., Noor, A.M., Zurovac, D., Juma, E., Ochola, S.A. and Snow, R.W. (2011). The clinical burden of malaria in Nairobi: a historical review and contemporary audit. *Malar J* 10, 138.

Muench, S.P., Rafferty, J.B., McLeod, R., Rice, D.W. and Prigge, S.T. (2003). Expression, purification and crystallization of the Plasmodium falciparum enoyl reductase. *Acta Crystallogr D Biol Crystallogr* 59, 1246-1248.

Nickel, C., Rahlfs, S., Deponte, M., Koncarevic, S. and Becker, K. (2006). Thioredoxin networks in the malarial parasite Plasmodium falciparum. *Antioxid Redox Signal* 8, 1227-1239.

Nkrumah, L.J., Muhle, R.A., Moura, P.A., Ghosh, P., Hatfull, G.F., Jacobs, W.R., Jr. and Fidock, D.A. (2006). Efficient site-specific integration in Plasmodium falciparum chromosomes mediated by mycobacteriophage Bxb1 integrase. *Nat Methods* 3, 615-621.

Okirou, E.A., Al-Taiar, A., Reyburn, H., Idro, R., Berkley, J.A. and Snow, R.W. (2009a). Age patterns of severe paediatric malaria and their relationship to Plasmodium falciparum transmission intensity. *Malaria Journal* 8.

Okiro, E.A., Bitira, D., Mbabazi, G., Mpimbaza, A., Alegana, V.A., Talisuna, A.O. and Snow, R.W. (2011). Increasing malaria hospital admissions in Uganda between 1999 and 2009. *Bmc Medicine* 9.

Okiro, E.A., Mutheu, J., Gething, P.W., Juma, E. and Snow, R.W. (2009b). The Changing Patterns of Malaria Admissions since 1999 at 18 Hospitals across Kenya. *American Journal of Tropical Medicine and Hygiene* 81, 54-54.

Pei, Y., Tarun, A.S., Vaughan, A.M., Herman, R.W., Soliman, J.M., Erickson-Wayman, A. and Kappe, S.H. (2010). Plasmodium pyruvate dehydrogenase activity is only essential for the parasite's progression from liver infection to blood infection. *Mol Microbiol* 75, 957-971.

Perham, R.N. (2000). Swinging arms and swinging domains in multifunctional enzymes: catalytic machines for multistep reactions. *Annu Rev Biochem* 69, 961-1004.

Phyo, A.P., Nkhoma, S., Stepniewska, K., Ashley, E.A., Nair, S., McGready, R., *et al.* (2012). Emergence of artemisinin-resistant malaria on the western border of Thailand: a longitudinal study. *Lancet* 379, 1960-1966.

Reche, P. and Perham, R.N. (1999). Structure and selectivity in post-translational modification: attaching the biotinyl-lysine and lipoyl-lysine swinging arms in multifunctional enzymes. *Embo Journal* 18, 2673-2682.

Reed, L.J., Koike, M., Levitch, M.E. and Leach, F.R. (1958). Studies on the nature and reactions of protein-bound lipoic acid. *J Biol Chem* 232, 143-158.

Rezaie, T., Child, A., Hitchings, R., Brice, G., Miller, L., Coca-Prados, M., *et al.* (2002). Adult-onset primary open-angle glaucoma caused by mutations in optineurin. *Science* 295, 1077-1079.

Salcedo, E., Sims, P.F.G. and Hyde, J.E. (2005). A glycine-cleavage complex as part of the folate one-carbon metabolism of Plasmodium falciparum. *Trends in Parasitology* 21, 406-411.

Schonauer, M.S., Kastaniotis, A.J., Kursu, V.A., Hiltunen, J.K. and Dieckmann, C.L. (2009). Lipoic acid synthesis and attachment in yeast mitochondria. *J Biol Chem* 284, 23234-23242.
Seeber, F., Limenitakis, J. and Soldati-Favre, D. (2008). Apicomplexan mitochondrial metabolism: a story of gains, losses and retentions. *Trends in Parasitology* 24, 468-478.

Snider, G.W., Dustin, C.M., Ruggles, E.L. and Hondal, R.J. (2014). A mechanistic investigation of the C-terminal redox motif of thioredoxin reductase from Plasmodium falciparum. *Biochemistry* 53, 601-609.

Spalding, M.D., Allary, M., Gallagher, J.R. and Prigge, S.T. (2010a). Validation of a modified method for Bxb1 mycobacteriophage integrase-mediated recombination in Plasmodium falciparum by localization of the H-protein of the glycine cleavage complex to the mitochondrion. *Mol Biochem Parasitol* 172, 156-160.

Spalding, M.D. and Prigge, S.T. (2010b). Lipoic acid metabolism in microbial pathogens. *Microbiol Mol Biol Rev* 74, 200-228.

Storm, J. and Muller, S. (2012). Lipoic acid metabolism of Plasmodium--a suitable drug target. *Curr Pharm Des* 18, 3480-3489.

Svagera, Z., Hanzlikova, D., Simek, P. and Husek, P. (2012). Study of disulfide reduction and alkyl chloroformate derivatization of plasma sulfur amino acids using gas chromatography-mass spectrometry. *Anal Bioanal Chem* 402, 2953-2963.

Trager, W. and Jensen, J.B. (1997). Continuous culture of Plasmodium falciparum: its impact on malaria research. *Int J Parasitol* 27, 989-1006.

Woster, P.M., Black, A.Y., Duff, K.J., Coward, J.K. and Pegg, A.E. (1989). Synthesis and biological evaluation of S-adenosyl-1,12-diamino-3-thio-9-azadodecane, a multisubstrate adduct inhibitor of spermine synthase. *J Med Chem* 32, 1300-1307.

Wrenger, C. and Muller, S. (2004). The human malaria parasite Plasmodium falciparum has distinct organelle-specific lipoylation pathways. *Mol Microbiol* 53, 103-113.
Figure Legends

Fig. 1. Lipoate synthesis and scavenging pathways.

(A) Lipoate synthesis as observed in *E. coli*. Octanoyl groups from octanoyl-ACP (acyl carrier protein) are transferred to E2 proteins (and the H-protein) by an octanoyltransferase (LipB) followed by the insertion of sulfur atoms by a lipoate synthase (LipA).

(B) Lipoate synthesis as observed in *B. subtilis*. Octanoyl groups are transferred specifically to the H-protein by octanoyltransferase (LipM) and are subsequently distributed to the E2 proteins by an amidotransferase (LipL). As in *E. coli*, sulfur insertion is catalyzed by a lipoate synthase (LipA).

(C) Lipoate scavenging as observed in *E. coli*. Lipoate scavenged from the external environment is attached to E2 proteins (and the H-protein) by an ATP-dependent lipoate ligase (LplA).

(D) Lipoate scavenging as observed in *B. subtilis*. The lipoate ligase (LplJ) has specificity for the H-protein.

Fig. 2. BCDH and KDH localization and cell-based lipoylation assays.

(A) Epifluorescent images of live *P. falciparum* erythrocytic-stage parasites expressing GFP fused to either the BCDH leader peptide (E2-BCDH₃₀) or the KDH leader peptide (E2-KDH₃₀). The parasites were stained with MitoTracker to identify mitochondria and Hoechst to identify nuclei. Image z-stacks were deconvolved and then presented as a single combined image. In both cases, GFP fluorescence colocalizes with MitoTracker to the mitochondrion.

(B) Expression of the E2-BCDH₃₀-GFP and E2-KDH₃₀-GFP constructs. An α-GFP western blot of parasite lysates identifies both proteins with the predicted molecular mass.

(C) Lipoylation of BCDHₗₔ and KDHₗₔ in BL21 *E. coli*. The lipoylation domains (LD) of both the E2-BCDH and the E2-KDH are lipoylated in anti-lipoate (α-LA) western blots of bacterial lysate, verifying that these constructs act as lipoylation substrates.

(D) Cell-based assay of lipoate ligase activity. Lipoylation deficient JEG3 cells were used to test the activity of three candidate lipoate ligases (*Ec*LplA, LipL1 and LipL2) with two lipoylation substrates.

(BCDH<sub>LD</sub> and KDH<sub>LD</sub>). *E. coli* lipoate ligase (*EcLplA*) is sufficient to lipoylate BCDH<sub>LD</sub> and KDH<sub>LD</sub>, whereas neither LipL1 nor LipL2 lipoylate either parasite substrate.

Fig. 3. *In vitro* ligation assays.

(A) Lipoate ligase activity. Western blot analysis shows that LipL1 is an ATP-dependent lipoate ligase only when H-protein is used as substrate, while LipL2 does not display lipoate ligase activity with any substrate.

(B) HPLC traces for LipL1, LipL1<sub>K160A</sub> and LipL2 adenylation reactions. Each enzyme was reacted with lipoate and ATP for 10 minutes prior to HPLC analysis with absorbance monitored at 270 nm. The LipL1 reaction produced a major product eluting at 6.2 minutes with a mass of 534 Da as determined by ESI-MS.

(C) Identification of Lipoyl-AMP produced by LipL1. The major product generated by LipL1 was subjected to collision induced dissociation MS/MS, demonstrating that this product contains AMP.

(D) JEG3 complementation by LipL1 and LipL1<sub>K160A</sub>. LipL1 is able to lipoylate endogenous *E. coli* substrates in lipoylation deficient strain JEG3, whereas LipL1<sub>K160A</sub> is unable to catalyze this activity. Expression of both proteins was confirmed by stripping and re-probing the blot with α-MBP (lower panel).

(E) Recombinant LipL1<sub>K160A</sub> cannot lipoylate the H-protein in an *in vitro* lipoylation assay.

Fig. 4. LipL1 and LipL2 are required for BCDH<sub>LD</sub> and KDH<sub>LD</sub> lipoylation in an ATP-dependent reaction.

(A) Lipoyl amidotransferase activity. Western blot analysis shows that LipL1 and LipL2 are not able to transfer lipoyl groups from Holo-H-protein to either BCDH<sub>LD</sub> or KDH<sub>LD</sub>.

(B) Combined LipL1 + LipL2 lipoate ligase activity. Both LipL1 and LipL2 are needed for lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub> in an ATP-dependent lipoate ligase reaction. Note a minor population of *E. coli* E2-KDH found in purified LipL2.

(C) Catalytically active LipL1 is required for the lipoylation of both BCDH<sub>LD</sub> and KDH<sub>LD</sub>.
(D) Mutation of both conserved cysteine residues in LipL2 (C93A and C276A) did not prevent lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub>, showing that lipoylation does not proceed through a thioester intermediate.

Fig. 5. Redox dependence of the two lipoylation routes.

(A) Redox dependence of H-protein lipoylation. Western blot analysis shows that lipoylation of the H-protein by LipL1 is enhanced in the presence of DTT.

(B) Dependence of the LipL1 + LipL2 reaction on strong reducing conditions. Western blots show that lipoylation of BCDH<sub>LD</sub> and KDH<sub>LD</sub> occur only in the presence of the strong reducing agent TCEP. Note a minor population of *E. coli* E2-KDH found in purified LipL2.

(C) Redox states of lipoate. The oxidized ring form of lipoate is reduced to the dihydrolipoate form by treatment with TCEP or THP.

Fig. 6. 6,8-dichlorooctanoate is attached to all substrates independent of redox environment.

(A) 6,8-Dichlorooctanoate (6,8-diClO) is a structural analog of reduced lipoate (dihydrolipoate).

(B) Modification of the H-protein by lipoate and 6,8-diClO. LipL1 catalyzes the attachment of lipoate as shown in the α-LA western blot (top panels) and 6,8-diClO as shown in the α-6,8-diClO western blot (bottom panels). The α-LA antibodies and the α-6,8-diClO antiserum show very little cross-reactivity.

(C) Effect of reducing conditions on the enzymatic activity of LipL1 against the H-protein. The H-protein is modified by 6,8-diClO independent of the redox environment.

(D) Effect of reducing conditions on the enzymatic activity of LipL1 + LipL2. BCDH<sub>LD</sub> and KDH<sub>LD</sub> are modified by 6,8-diClO independent of the redox environment.

Fig. 7. Modification of mitochondrial proteins with lipoate analogs.

This article is protected by copyright. All rights reserved.(A) Attachment of 8-bromooctanoate (8-BrO) to lipoylation substrates *in vitro*. An α-6-8-diClO western blot shows that LipL1 can attach 8-BrO to the H-protein and that LipL1+LipL2 can attach 8-BrO to BCDH<sub>LD</sub> and KDH<sub>LD</sub>.

(B) Growth inhibition of *P. falciparum* treated with 6,8-diClO. The lipoate analog 6,8-diClO inhibits blood stage parasite growth in a dose-dependent manner.

(C) Uptake and attachment of LA and 6,8-diClO in parasite culture. Western blot analysis of parasite lysate shows that treatment of *P. falciparum* parasites with either 6,8-diClO or LA results in the modification of E2-BCDH and E2-KDH. Lipoylated E2-PDH is derived from a synthesis pathway in the apicoplast and not from lipoate scavenging.

(D) Mitochondrial lipoylation pathways in *Plasmodium falciparum*. The mitochondrion relies on lipoate scavenging from the host while the apicoplast contains an isolated lipoate synthesis pathway. Our data suggest that there are two routes for lipoylation of protein substrates in the mitochondrion. The H-protein is solely lipoylated by LipL1 in an ATP-dependent reaction; this activity is enhanced under less reducing conditions. Both LipL1 and LipL2 are needed for lipoylation of the E2 domains of BCDH (E2-BCDH) and KDH (E2-KDH) in an ATP-dependent reaction which only occurs under conditions capable of reducing lipoate to dihydrolipoate.
Synthesis Pathways                                      Scavenging Pathways
A. E. coli                                              B. B. subtilis                                       C. E. coli                                           D. B. subtilis

ACP-S-Octanoyl-ACP                                      ACP-S-Octanoyl-ACP                                   Lipoate                                              Lipoate
LipB                                                    LipM                                                 Hprot-N-Octanoyl-H-protein                          Hprot-N-Octanoyl-H-protein
Octanoyl-E2                                             Octanoyl-H-protein                                   LplA                                                LplJ
E2-N-Octanoyl-E2                                        LipL                                                 mmi_12753_f1                                         Hprot-N-Octanoyl-H-protein

A.
GFP          MitoTracker    Hoescht      Merge        Phase

E2-BCDH₃₀
![Image](image1)

E2-KDH₃₀
![Image](image2)

B.
50-
37-
25-
α-GFP

C.
BL21
50-
37-
α-LA
1   2
BCDHLD + -
KDHLD - +

D.
JEG3
EcLpIA       LipL1         LipL2
75-
50-
37-
1   2   3   4   5   6
BCDHLD + -   + -   + -
KDHLD  - +   - +   - +
EcPDH
EcKDH
]-BCDHLD/KDHLD
α-LA

mmi_12753_f2

This article is protected by copyright. All rights reserved.

A.
LipL1
50-
37-
H-protein
]-BCDHLD/KDHLD
LipL2
50-
37-
H-protein
]-BCDHLD/KDHLD
α-LA

|  | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| H-protein | + | - | - | - | - | - | + |  |
| BCDHLD | - | + | - | - | - | - | - |  |
| KDHLD | - | - | - | + | - | - | - |  |
| ATP | + | - | + | - | + | - | + | - |

B.
m/z = 534Da
LipL1
LipL1K160A
LipL2
Retention time (minutes)
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17

C.
346.07
NH₂
HO-O-
OH OH
AMP
calc'd m/z: 346.06
534.03
NH₂
S-S
O
OH OH
Lipoyl-AMP
calc'd m/z: 534.09
200 300 400 500 600 700 800 900
m/z

D.
IplA-/lipB-

E.
H-protein
50-
EcPDH
37-
EcKDH
α-LA
1 2
LipL1
+
-
LipL1K160A
-
+
α-MBP
1 2
LipL1
+
-
LipL1K160A
-
+

mmi_12753_f3

A.
LipL1 LipL2 LipL1+LipL2
50-
37-
Holo-H-protein
]-BCDHLD/KDHLD
α-LA

|  | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| BCDHLD | - | + | - | - | + | - | - | + | - |
| KDHLD | - | - | + | - | - | + | - | - | + |

B.
LipL1+LipL2
50-
37-
EcKDH
]-BCDHLD/KDHLD
α-LA

|  | 1 | 2 | 3 | 4 | 5 | 6 |
| --- | --- | --- | --- | --- | --- | --- |
| BCDHLD | - | - | + | - | - | - |
| KDHLD | - | - | - | - | + | - |
| ATP | + | - | + | - | + | - |

C.
BCDHLD KDHLDD
50-
37-
Holo-H-protein
]-BCDHLD/KDHLD
α-LA
EcKDH
]-BCDHLD/KDHLD
α-LA

|  | 1 | 2 | 3 | 4 |
| --- | --- | --- | --- | --- |
| LipL1 | + | - | + | - |
| LipL1K160A | - | + | - | + |

D.
50-
37-
EcKDH
BCDHLD
α-LA
50-
37-
EcKDH
KDHLDD
α-LA

|  | 1 | 2 | 3 | 4 | 5 | 6 |
| --- | --- | --- | --- | --- | --- | --- |
| LipL2 | + |  |  |  |  |  |
| LipL2C93A | - | + |  |  |  |  |
| LipL2C276A |  |  |  |  |  | + |
| ATP | + | - | + | - | + | - |

mmi_12753_f4

This article is protected by copyright. All rights reserved.

A.
LipL1

|  | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| DTT | - | + |  |  | - |  |  |  |
| TCEP | - |  | + |  | - |  |  |  |
| THP | - |  |  |  |  |  | + |  |
| ATP | + | - | + | - | + | - | + | - |

B.
LipL1 LipL2 LipL1+LipL2

|  | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| DTT | - |  |  |  |  |  | + |  | - |  |
| TCEP | + |  | - |  |  |  | - |  | + |  |
| ATP | + | - | + | - | + | - | + | - | + | - |

C.

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

mmi_12753_f5

A.
- O
  |
  C
  |
  CH₂SH
  |
  SH

≈

- O
  |
  C
  |
  CH₂Cl
  |
  Cl

B.
LipL1
50-
|   |   |
|---|---|
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|   |   |
|  
A.

|  | 1 | 2 | 3 | 4 | 5 | 6 |
| --- | --- | --- | --- | --- | --- | --- |
| LipL2 | - |  | + |  |  |  |
| DTT | + |  | - |  |  |  |
| TCEP | - |  | + |  |  |  |
| ATP | + | - | + | - | + | - |

B.
- 0 μM
- 25 μM
- 100 μM
- 400 μM
- 1000 μM

C.

|  | 1 | 2 | 3 |
| --- | --- | --- | --- |
| 6,8-diClO | - | - | + |

D.
- Lipoate Synthesis
- Apicoplast
- Lipoate Scavenging
- Lipoate → Lipoyl-H-protein (via LipL1)
- Dihydrolipoate → Lipoyl-E2-BCDH/Lipoyl-E2-KDH (via LipL1/LipL2)
- Mitochondrion
- Lipoate from host

mmi_12753_f7

This article is protected by copyright. All rights reserved.
