
TIPS-1165; No. of Pages 9

**ARTICLE IN PRESS**

---

**Review**

**Targeting autophagy in neurodegenerative diseases**

René L. Vidal${}^{1,2}$, Soledad Matus${}^{1,2}$, Leslie Bargsted${}^{2,3}$, and Claudio Hetz${}^{1,2,3,4}$

1 Neurounion Biomedical Foundation, CENPAR, Santiago, Chile  
2 Biomedical Neuroscience Institute, Faculty of Medicine, University of Chile, Santiago, Chile  
3 Program of Cellular and Molecular Biology, Institute of Biomedical Sciences, Center for Molecular Studies of the Cell, University of Chile, Santiago, Chile  
4 Department of Immunology and Infectious Diseases, Harvard School of Public Health, Boston, MA, USA  

The most prevalent neurodegenerative disorders involve protein misfolding and the aggregation of specific proteins. Autophagy is becoming an attractive target to treat neurodegenerative disorders through the selective degradation of abnormally folded proteins by the lysosomal pathway. However, accumulating evidence indicates that autophagy impairment at different regulatory steps may contribute to the neurodegenerative process. Thus, a complex scenario is emerging where autophagy may play a dual role in neurodegenerative diseases by causing the downstream effect of promoting the degradation of misfolded proteins and an upstream effect where its deregulation perturbs global proteostasis, contributing to disease progression. Challenges in the future development of therapeutic strategies to target the autophagy pathway are discussed.

### Protein misfolding

The maintenance of protein homeostasis is crucial to sustain neuronal function, especially during aging where adaptive cellular mechanisms against stress are attenuated [1]. Protein misfolding and abnormal aggregation are common hallmarks of most age-related neurodegenerative diseases [2,3], including Alzheimer’s disease (AD), amyotrophic lateral sclerosis (ALS), Parkinson’s disease (PD), and Huntington’s disease (HD), which are classified as protein-misfolding disorders (PMDs) [4,5]. This aggregation process involves the generation of highly diffusible small oligomers, fibrils, and large aggregates that are visualized as protein inclusions with amyloid properties. Thus, strategies to remove toxic oligomeric species are becoming an attractive target for future therapeutic intervention in PMDs.

Macroautophagy (here referred to as autophagy) is the main cellular catabolic route for protein aggregates and damaged organelles. Many studies in the past 10 years have shown that autophagy is an efficient mechanism for the selective degradation of aggregation-prone proteins linked to neurodegeneration and several pharmacological and genetic strategies have been developed to enhance the activity of the pathway in a disease context [6]. Complex signaling mechanisms control the activation of autophagy, which could be globally classified as mammalian target of rapamycin (mTOR)-dependent or -independent pathways. Strategies to engage both pathways have proved efficient in decreasing neurodegeneration in certain preclinical models of neurodegeneration [7,8]. The mTOR inhibitor rapamycin is the most widely used small molecule in testing the consequences of enhancing autophagy activity in many disease models [7,9]. Although mTOR-independent pathways are poorly described in terms of molecular details, various drugs have been identified to enhance autophagy through this mechanism, providing neuroprotection in diverse models of PMDs [9]. Despite these promising observations, accumulating evidence also indicates that alteration of distinct regulatory steps of autophagy may result in global dysfunction of the degradative capacity of the cell, contributing to neurodegeneration [10]. In such diseases enhancement of autophagy may have detrimental consequences, exacerbating disease progression. Here we discuss several examples illustrating a complex scenario where, depending on the strategy used and the disease context, the final outcome of manipulating autophagy is disparate. We also provide an overview of recent advances aiming to target autophagy with pharmacological and gene-therapy approaches. We also highlight the possible limitations of the currently available drugs for enhancing autophagy levels.

#### Protein misfolding disorders

Several neurodegenerative diseases share the pathological hallmark of accumulating misfolded proteins in both sporadic and genetic cases. The genes that encode these proteins are often found mutated in corresponding familial cases, which suggests that there are common pathological mechanisms that underlie neuronal dysfunction. Here we briefly introduce the molecular hallmarks of PMDs to provide relevant information about the impact of autophagy in neurodegenerative diseases.

In AD the presence of amyloid plaques and intracellular neurofibrillary tangles, formed by the amyloid β peptide and hyperphosphorylated tau respectively, are central pathological features of the disease, associated with the occurrence of neuronal impairment and memory loss [11,12]. In some rare familial forms of AD, mutations in

---

Corresponding authors: Vidal, R.L. (<rene.vidal@neurounion.com>); Hetz, C. (<chetz@med.uchile.cl>).  
Keywords: rapamycin; mTOR; autophagy; Beclin 1; aggregation; neurodegenerative disease.  
© 2014 Elsevier Ltd. All rights reserved. <http://dx.doi.org/10.1016/j.tips.2014.09.002>

Trends in Pharmacological Sciences xx (2014) 1–9

two proteins involved in amyloid β metabolism – amyloid precursor protein (APP) and presenilin 1 – have been identified [11]. PD is characterized by the selective degeneration of dopaminergic neurons of the substantia nigra pars compacta [13]. PD is associated also with the presence of intracellular inclusions known as Lewy bodies, comprising aggregated α-synuclein and polyubiquitinated proteins. Another PMD is HD, an autosomal-dominant pathology characterized by the expansion of polyglutamine repeat tracts in the N-terminal region of Huntingtin. Mutant Huntingtin produces cytoplasmic aggregates triggering neuron loss in the striatum [14,15]. Finally, ALS is a prevalent adult-onset paralytic disease involving the selective death of motor neurons in the brain and spinal cord [16]. The most common genetic causes of familial ALS are the recently defined hexanucleotide repeat expansion in the intronic region of C9orf72 and mutations in the gene encoding cytosolic superoxide dismutase 1 (SOD1) [17]. Mutations in TAR DNA-binding protein (TARDBP, also known as TDP-43) also cause familial ALS and cytoplasmic inclusions of wild type TDP-43 are found in nearly all postmortem studies of tissue derived from sporadic ALS. Thus, based on the fact that protein misfolding and aggregation are hallmarks of many brain diseases, strategies to enhance the degradation of abnormally folded proteins are predicted to have beneficial effects in alleviating neurodegeneration.

Key players in the regulation of autophagy

Autophagy regulates important biological functions such as cell survival, cell death, cell metabolism, development,

aging, infection, and immunity [18]. In other contexts, the failure to induce autophagy or over-enhancement of the pathway may underlie certain brain pathologies. Several components, known as autophagy-related genes (ATGs), regulate discrete steps in the autophagy process [18]. ATGs form diverse protein complexes in sequential steps controlling autophagosome formation and vesicle fluxes (Figure 1) [19], including autophagy induction, nucleation/autophagosome formation, vesicle expansion, cargo recognition, crosstalk between endocytosis and autophagy, and autophagosome clearance [20,21]. The initiation of the autophagy process is mediated in part by a protein kinase complex that responds to upstream signals (Atg1 and Atg13 in yeast). The serine/threonine protein kinase mTOR is a component of mTOR complex 1 (mTORC1) and acts as a regulator of autophagy by suppressing the pathway under nutrient-rich conditions. The nucleation and formation of autophagosome is regulated by enzymes involving in the generation of phosphatidylinositol 3-phosphate (PI₃P), including the class III phosphatidylinositol 3-kinase (PI₃K) VPS34, which mediates the localization of other autophagy-regulatory proteins to the pre-autophagosomal structure [21]. The nucleation complex partly comprises BCL-2-interacting protein (Beclin 1) and other fundamental elements, including VPS34. Beclin 1 is negatively regulated by the antiapoptotic proteins BCL-2 and BCL-XL at the endoplasmic reticulum (ER) membrane [22]. Beclin 1 has been also involved in the activation of ATG5/ATG7-independent autophagy [23]. Vesicle expansion is mediated by the covalent conjugation of ATG12 to ATG5, which, in association


![Diagram](attachment:diagram.png)


Figure 1. Autophagy impairment in neurodegenerative diseases. Possible defects that could alter the regulation and induction of macroautophagy in various neurodegenerative disorders. Abnormal interactions of mutant superoxide dismutase 1 (mSOD1), LRRK2, Parkin, PINK1, and mutant Huntingtin (mHTT) with Beclin 1 could alter the initiation steps of autophagy. PINK and Parkin play a key role in the elimination of damaged mitochondria and mutations in these proteins in Parkinson’s disease (PD) could interfere with mitophagy. mHTT expression leads to altered cargo recognition and autophagy failure. α-Synuclein (αsyn) can interfere with autophagy through interaction with Rab1a. Presenilin-1 (PS1) mutations cause impairment in lysosomal acidification and autophagy impairment. In PD, mutations in ATP13A2 could alter the function of lysosomes.

with ATG16, translocates to the membrane of early autophagosomes and promotes the conjugation of microtubule-associated protein light chain 3 (LC3) to phosphatidylethanolamine (PE) [19]. On conjugation, the soluble LC3-I translocates to the autophagosome membrane where it is then referred to as LC3-II. Finally, autophagosomes fuse with acidic lysosomes to acquire hydrolytic activity, forming the autophagolysosome compartment where the cargo is subsequently degraded. Monitoring LC3-II flux through the autophagy pathway is the gold standard to determine the activation of this cellular process [24].

Although starvation-induced autophagy is mainly regulated by mTOR signaling, autophagy is also controlled through mTOR-independent events (Figure 2). This mechanism is emerging as an interesting target to enhance autophagy because it avoids altering the pleiotropic responses controlled by mTOR. This alternative route was discovered through pharmacological screening aimed to define new molecular targets to enhance autophagy [25–27]. Several mTOR-independent modulators of autophagy were discovered involving fluctuations in intracellular calcium levels and the turnover of inositol phosphates.

mTOR-independent pathways are induced by agents that lower or deplete inositol 3-phosphate (IP₃) levels, such as lithium, valproate, and carbamazepine [9]. Additionally, it has been suggested that calcium signaling also modulates mTOR-independent autophagy. Calcium could also activate calpain to induce autophagy associated with the regulation of IP₃ production and cAMP [25]. Although controversial, lithium has been shown to have neuroprotective effects in various disease models and has the capacity to induce autophagy through inhibition of inositol monophosphatase (IMPase) and inositol transporters [28]. Other molecules, such as spermidine and resveratrol, have been shown to enhance autophagy through epigenetic changes, having important neuroprotective effects and prolonging lifespan in model organisms [29,30]. Finally, several stress signals can also engage autophagy [31]; where we highlight the induction of protein-folding stress at the ER. ER stress, but also nutrient starvation, has been shown to induce autophagy through the activation of the unfolded-protein response (UPR) sensor IRE1 and the downstream activation of c-Jun N-terminal kinase (JNK) [32,33], which may modulate the dissociation of the Beclin

![Diagram](attachment:diagram.png)

Figure 2. Pharmacological targeting of mammalian target of rapamycin (mTOR)-dependent and mTOR-independent autophagy. (A) Growth factors and nutrient starvation negatively modulate the mTOR pathway by inhibiting tuberous sclerosis complex (TSC) 1/2 and its inhibitory effect on Rheb, causing the activation of mTOR complex 1 (mTORC1), inhibiting autophagy. Rapamycin interacts with immunophilin FK506-binding protein 12 (FKBP12), which then stabilizes mTOR association and inhibits the kinase activity of mTOR, triggering autophagy. Stress signals emerging from the endoplasmic reticulum can also modulate autophagy, through the unfolded-protein response (UPR), transcription factors, and the c-Jun N-terminal kinase (JNK) pathway. Autophagy is also induced with drugs that decrease inositol (Ins) or inositol 1,4,5-trisphosphate (IP₃) levels in the phosphoinositol signaling pathway. Lithium and L-690,330 act by inhibiting inositol monophosphatases (IMPases) and carbamazepine and sodium valproate trigger autophagy in part by inhibiting Ins synthesis. Lithium also inhibits glycogen synthase kinase-3 beta (GSK-3β), which activates mTOR by inhibiting TSC1/2. Verapamil can also induce autophagy in a mTOR-independent manner, through inhibition of calpain. Spermidine and resveratrol enhance autophagy through epigenetic changes. Trehalose induces upregulation of a cluster of *atg* genes associated with the activation FOXO1, a transcription factor that can engage autophagy in neurons.

1 and BCL-2 complex [34]. ER stress has also been shown to enhance autophagy by the upregulation of Atgs through several UPR transcription factors and also FOXO1, a critical regulator of autophagy in neurons. The ER had been implicated as a source of membranes to generate autophagosomes and omegasome formation (autophagosome biogenesis) [35, 36]. Since ER stress is a hallmark of most PMDs affecting the nervous system, the UPR represents an interesting homeostatic pathway connecting alteration of cellular proteostasis and autophagy induction [5]. Thus, multiple regulatory mechanisms exist to integrate catabolic needs in the cell and to fine-tune autophagy levels to sustain proteostasis.

### Failure of autophagy in neurodegenerative disease

The first direct evidence linking proteostasis control and autophagy in the nervous system came from studies in which genetic inactivation of essential regulatory genes (Atg5 or Atg7) in the central nervous system was performed in mice. Inactivation of autophagy in neurons caused spontaneous neurodegeneration, promoting the spontaneous accumulation of protein aggregates, extensive neuronal loss, and premature death of the animal [37, 38]. Many reports in PMD models have indicated that abnormal interactions with specific components of the autophagy machinery may perturb cellular homeostasis, contributing to disease progression (Figure 1) [10, 39–41]. These alterations may enhance abnormal protein aggregation in PMDs, but also could contribute to the accumulation of damaged organelles, having deleterious consequences on cell physiology. For example, a polymorphism in ATG7 was correlated with the severity of HD [42]. Levels of Beclin 1 or its availability are reduced in the brain during aging, correlating with enhanced vulnerability to the development of experimental AD [43] or HD [44]. By contrast, upregulation of Beclin 1 levels has been reported in various neurodegenerative diseases, including ALS [45], PD [46], and HD [47, 48]. Although several studies have uncovered an important neuroprotective activity of Beclin 1 in PMDs using genetic manipulation (as indicated in models of PD, AD, and polyglutamine diseases [43, 44, 47, 49, 50]), reducing levels of Beclin 1 in an animal model of ALS provided protection against neurodegeneration, prolonging lifespan (see below) [51].

A few reports have shown that PMD-related proteins may directly alter the initiation of autophagy through physical interaction with the Beclin 1 complex. Mutant Huntingtin and the Huntingtin-interacting protein Rhes can sequester Beclin 1, which may result in attenuated autophagy capacity [44, 52]. Similarly, the PD-linked protein PINK1 physically interacts with Beclin 1, altering levels of autophagy [53, 54]. A recent report also indicated that Parkin, another important PD gene, interacts with Beclin 1 and modifies autophagy activity [50]. Finally, we recently reported that mutant SOD1 associates with the Beclin 1/BCL-XL complex, destabilizing this inhibitory interaction, which may perturb global autophagy levels [51].

The adapter protein p62/SQSTM1 is the most widely studied component of the cargo-recognition machinery that delivers substrates to autophagosomes. Deficiency in cargo loading into autophagosomes has been observed in cellular and animal models of HD, causing impaired protein degradation by autophagy despite increased autophagic vesicle content [55]. Several mutations in p62/SQSTM1 have been identified in ALS cases [56], consistent with the observation that p62/SQSTM1 promotes the autophagy-dependent degradation of mutant SOD1 and TDP-43 [57, 58]. Other rare genes linked to ALS are reported to alter autophagy levels, including charged multivesicular body protein-2B (CHMP2B), the lipid phosphatase Fig. 4, and UBQLN2 [59–61]. These findings suggest that deregulation of protein-degradation pathways may represent an important pathological mechanism leading to proteostasis defects in ALS [56].

In AD, global alteration in the proteolytic capacity of lysosomes has been observed, involving problems in the control of luminal pH. At the molecular level, it was shown that presenilin-1 directly regulates the acidification of lysosomes by controlling the maturation of a v-ATPase subunit [62]. In agreement with this finding, restoring lysosomal function in a mouse model of AD attenuated the progression of neuropathology in the brain [63]. Similarly, accumulating evidence indicates that lysosomal dysfunction also contributes to the occurrence of PD [64]. For example, mutations in the lysosomal ATPase ATP13A2/PARK9 in familial PD trigger drastic alterations in the degradative capacity of lysosomes, leading to an abnormal accumulation of autophagosomes [65]. Mutant forms of LRRK2/PARK2 and α-synuclein also alter lysosome-mediated autophagy pathway [66, 67] and negatively impact the trafficking of autophagy vesicles through interactions with Rab1 [68]. Other rare mutations linked to PD, such as VPS35, may also contribute to autophagy impairment due to altered vesicle trafficking [69].

Finally, the selective degradation of mitochondria by autophagy (referred to as mitophagy) is also altered in PD. The PD-related proteins PINK1 and Parkin operate as central components of the mitophagy pathway [70]. In addition, a recent study suggested that mutant LRRK2 physically interacts with Beclin 1, affecting the function of LRRK2 in mitophagy [71]. Overall, these studies highlight an emerging concept where perturbations in the function of autophagy and the lysosomal pathway may contribute to the development of neurodegenerative conditions as part of the etiology of the disease, affecting the global maintenance of proteostasis and catabolic processes involved in the clearance of damaged organelles.

### Pharmacological targeting of autophagy in neurodegenerative disease

Since autophagy operates as an efficient system to selectively degrade abnormal proteins associated with PMDs and damaged organelles, various compounds have been identified in high-throughput screening to enhance autophagy with proven efficacy in several preclinical models of disease (reviewed in [8, 72]) (Figure 3). Here we highlight selected studies indicating the therapeutic potential, but also the detrimental consequences, of enhancing autophagy levels in neurodegenerative disease (Table 1).

TIPS-1165; No. of Pages 9

ARTICLE IN PRESS

Review

Trends in Pharmacological Sciences xxx xxxx, Vol. xxx, No. x

| Resveratrol | Spermidine | Trehalose | L690,330 |
| --- | --- | --- | --- |
| ![Resveratrol](https://example.com/resveratrol.png) | ![Spermidine](https://example.com/spermidine.png) | ![Trehalose](https://example.com/trehalose.png) | ![L690,330](https://example.com/l690_330.png) |

| Rapamycin | Valproic acid | Carbamazepine | Verapamil |
| --- | --- | --- | --- |
| ![Rapamycin](https://example.com/rapamycin.png) | ![Valproic acid](https://example.com/valproic_acid.png) | ![Carbamazepine](https://example.com/carbamazepine.png) | ![Verapamil](https://example.com/verapamil.png) |

Figure 3. Molecular structure of selected molecules that can induce autophagy.

Although the brain responds poorly to global nutrient deprivation compared with other tissues, autophagy induction by rapamycin has been shown to provide protection in several experimental models of neurodegeneration. Pioneering studies by David Rubinsztein indicated that the administration of rapamycin to fly and mouse models of HD has protective consequences, enhancing the removal of mutant Huntingtin associated with clear motor recovery [73]. Rapamycin administration also reduces cognitive defects in an experimental model of AD, correlating with decreased amounts of fibrillary tangles and amyloid plaques in the brain [74–77]. Additionally, treatment with the

Table 1. Therapeutic effects of selected agents that modulate autophagy in neurodegenerative diseases${}^{a}$

| Disease | Agents | Effects | Refs |
| --- | --- | --- | --- |
| **Pharmacology** |  |  |  |
| Alzheimer's disease | Rapamycin | Attenuates cognitive deficits and reduces amyloid beta levels | [73–76] |
|  | Trehalose | Reduces Tau aggregates and improves neuronal survival | [85–87] |
|  | Methylene blue | Reduces tau aggregation levels | [92] |
| Huntington's disease | Rapamycin | Improves behavior and motor deficits and decreases Huntingtin aggregation | [72] |
|  | Trehalose | Improves motor dysfunction and decreases polyglutamine aggregation | [84] |
| Parkinson's disease | Rapamycin | Reduces dopaminergic neuron loss | [63,77] |
|  | Trehalose | Rescues motor dysfunction, reduces TDP-43 inclusion and improve the neuronal survival | [86] |
|  | Tunicamycin | Protects dopaminergic neuron on a toxicological model of the disease and flies overexpressing α-synuclein | [96] |
| ALS/FTD | Rapamycin, Carbamazepine, Spermidine and Tamoxifen | Reduces motor dysfunction, reduces TDP-43 inclusions and improve neuronal survival | [82] |
|  | Trehalose | Decreases mutant SOD1 aggregates and enhances motoneuron survival | [89,91] |
|  | Rapamycin | Has no impact or detrimental effects on mutant SOD1 transgenic mice | [79,80] |
|  | Rapamycin | Exacerbates disease progression of aVCP mice | [83] |
| **Gene Therapy** |  |  |  |
| Alzheimer's disease | LV-Beclin 1 | Reduced both intracellular and extracellular amyloid deposition | [42] |
| Parkinson's disease | LV-Beclin 1 | Reduces accumulation of α-synuclein | [48] |
| Machado-Joseph disease | LV-Beclin 1 | Rescues motor and neuropathological impairment | [100] |

${}^{a}$LV, lentivirus.

Trends in Pharmacological Sciences xxx xxxx, Vol. xxx, No. x

**Review**

rapamycin analog CCI-779 delays the pathology of HD and spinocerebellar ataxia models [73], in addition to PD [64, 78] and other neurodegenerative diseases [79]. Overall, *in vitro* studies have demonstrated that treatment with rapamycin and its derivatives promote the clearance of the most relevant aggregate-prone proteins involved in neurodegeneration, including polyglutamine- and polyalanine-containing proteins, mutant tau, α-synuclein, TDP-43, and prion protein among other PMD-related proteins [6].

In contrast with these reports, studies in ALS are more complex to interpret. Rapamycin treatment had no obvious beneficial effects [80, 81] or even detrimental consequences on ALS progression in mutant SOD1 transgenic mice [82]. Despite this, rapamycin administration to mutant TDP-43 transgenic mice delayed ALS progression [83]. In addition to ALS, TDP-43 mutations are linked to frontotemporal dementia (FTD). Impairment of mTOR signaling and altered accumulation of autophagosomes were described in valosin-containing protein (VCP) transgenic mice, a model of FTD and ALS [84]. Treatment of VCP mice with rapamycin had negative effects on the model, exacerbating histopathological alterations [84]. Thus, in certain diseases where autophagy is impaired or altered, further enhancement of the pathway may have detrimental consequences. These effects may depend on the specific pathological mechanism behind a disease gene.

One of the most interesting mTOR-independent autophagy inducers discovered so far is trehalose, a non-reducing disaccharide produced naturally by some living organisms (non-mammals) under stress conditions. Trehalose was initially described as a chemical chaperone that could stabilize protein conformations, preventing protein aggregation. Notably, trehalose has long history of use in the food industry as a preserving agent (FDA approved). Overall, trehalose treatment is able to induce the degradation of various aggregation-prone proteins through autophagy enhancement in cell culture models [26]. Remarkably, oral administration of this compound decreased mutant Huntingtin aggregation and improved motor function in a mouse model of HD, prolonging lifespan [85]. Trehalose administration also has relevant neuroprotective effects in animal models of PD, AD, oculopharyngeal muscular dystrophy, and tauopathies [86–89]. The translational potential of trehalose is evident since it provides neuroprotection through oral administration in many animal models of disease. However, it is important to highlight the fact that only a few recent studies have correlated the neuroprotective effects of trehalose with autophagy induction *in vivo* as described, for example, in models of tau-mediated pathology and AD [88]. Recently, we reported beneficial effects of trehalose administration in a mouse model of ALS associated with the induction of mTOR-independent autophagy [90]. We also showed the active engagement of autophagy fluxes in the brain using a new strategy to monitor autophagy activity in neurons *in vivo* [91]. Trehalose treatment prolonged lifespan and significantly decreased the severity of the disease, associated with enhanced degradation of mutant SOD1 [90]. Similar findings were recently found in another ALS mouse model, where trehalose administration significantly delayed disease onset and decreased neuronal loss

[92]. Trehalose treatment upregulates the transcription of a cluster of *atg* genes [90] correlating with the activation of FOXO1, a major transcription factor involved in the regulation of autophagy-related genes in neurons and during aging [93]. Collectively, the aforementioned studies inspire optimism for novel treatment options for various neurological diseases.

While rapamycin and trehalose are pharmacological agents widely tested in preclinical models of neurodegeneration, other drugs have been identified to modulate autophagy with interesting results. Small molecules such as spermidine, carbamazepine, and tamoxifen were shown to rescue motor dysfunction in mutant TDP-43 transgenic mice, correlating with enhanced autophagy levels [83]. Methylene blue (methylthioninium chloride) also attenuates tauopathy in animal models correlating with increased expression of autophagy markers [94]. In HD, it was reported that nitric oxide induction blocks autophagosome formation and this effect could be reduced by N-L-arginine methyl ester (L-NAME), protecting against HD [95]. Conversely, in models of FTD involving the loss of progranulin function, treatment with inhibitors of autophagy significantly increased the levels of progranulin, rescuing the pathological effects observed in the model [96]. However, none of these reports directly demonstrated that autophagy was mediating the effects observed on disease progression *in vivo* and the drugs used target many cellular processes beyond autophagy. Direct manipulation of essential autophagy regulators was needed in those studies to define the functional involvement of the pathway to neurodegeneration.

The proteostasis network involves dynamic interconnection between various stress pathways including autophagy and the UPR in addition to the proteasome system, the heat-shock response, and quality control mechanisms [97]. Importantly, recent discoveries in the field also suggest that neurons may even control proteostasis in peripheral tissues, expanding the possible therapeutic benefit of targeting autophagy in the brain to other affected organs [98]. Induction of mild ER stress with the pharmacological inducer tunicamycin was shown to provide neuroprotection against PD, possibly due to the upregulation of autophagy [99]. Genetic manipulation of a key transcription factor of the UPR, known as XBP1, enhances autophagy in the brain and provides protection in mouse models of ALS [45] and HD [100]. The therapeutic consequences of targeting XBP1 involved enhancement of neuronal survival, decreased load of protein aggregates, and extension of lifespan. XBP1 was proposed to enhance autophagy by the negative control of FOXO1 levels [100]. Similarly, manipulating the UPR in *Caenorhabditis elegans* can provide protection against amyloid β due to enhanced autophagy levels [101]. Thus, targeting distinct nodes of the proteostasis network may be used as a strategy to enhance autophagy in a disease context.

### Gene therapy to enhance autophagy levels

Gene-therapy approaches have been extensively used to target protein aggregation in brain diseases [102] and have been explored to manipulate autophagy in a more specific manner. This area remains poorly explored, but a few

examples are documented with positive results. Beclin 1 administration via gene therapy in animal models of PD and Lewy body disease demonstrated beneficial effects, reducing the accumulation of α-synuclein and ameliorating synaptic dysfunction [49]. Likewise, injection of lentivirus expressing Beclin 1 into the cerebellum of a mouse model of spinocerebellar ataxia type 3 (Machado–Joseph disease) significantly improved motor performance, accompanied by reduced protein aggregation [103]. Similarly, Beclin 1 gene transfer attenuated AD pathology in a transgenic mouse model of the disease [43]. Consistent with these findings, Beclin 1 haploinsufficiency exacerbated the development of experimental AD *in vivo* [43]. These results may involve the degradation of APP by the autophagic pathway [104]. By sharp contrast, we recently reported that Beclin 1 haploinsufficiency had significant protective effects in an ALS mouse model, increasing lifespan [51]. These unexpected findings were accompanied by accumulation of p62, reduced levels of LC3-II, and an altered equilibrium between monomeric and oligomeric species of mutant SOD1 in the spinal cord [51]. These observations were explained by a possible reversal of the autophagy alterations due to a physical interaction of mutant SOD1 with the Beclin 1/BCL-X<sub>L</sub> complex.

Taken together, these studies uncovered for the first time a direct role of autophagy in neurodegenerative diseases, depicting a complex scenario where the contribution of the pathway is difficult to predict and may depend on the specific disease context analyzed.

### Concluding remarks

As discussed here increasing evidence supports the idea that pharmacological manipulation of autophagy may have neuroprotective effects in certain PMDs, delaying neurodegenerative events. However, it is crucial to understand in detail the mechanisms that may underlie the impairment of autophagy in specific conditions to predict the possible detrimental effects of such therapies to enhance autophagy. To maximize potential therapeutic approaches, future studies should seek to define the nature of the autophagy defects, the cellular responses associated with these alterations, and the stages of disease progression wherein such responses occur. This information should be used to generate combinatorial strategies to bypass or revert possible autophagy defects and further enhance autophagy in a physiological window. The dynamic range of autophagy enhancement should be defined to determine the limit that could be exploited, to avoid alterations in global homeostasis due to over-degradation of cellular components.

The identification of small molecules that target mTOR-dependent and -independent autophagy pathways are opening interesting avenues for therapeutic intervention and several small molecules are available to enhance autophagy, some of which are already FDA-approved drugs. However, clinical trials remain lacking in this important field. Calibrating drug concentrations and regimens for chronic use of these compounds is a challenging issue for future clinical trials to avoid detrimental effects of overactivation of the pathway. Since most available drugs target many biological processes beyond autophagy, it is

clear that more sophisticated high-throughput screening is needed to discover novel pharmacological agents for the accurate manipulation of autophagy with higher specificity and improved pharmacokinetic and safety properties. Based on this difficulty, gene therapy may emerge as an alternative strategy to specifically enhance autophagy responses, and available data suggest important therapeutic consequences of delivering autophagy-regulatory genes locally into affected tissue in various disease models. Since autophagy is emerging as an important homeostatic pathway that fine-tunes many physiological processes such as immunity and energy metabolism [18], targeting autophagy may have important secondary effects on chronic use. It is becoming imperative to develop more sophisticated assays for drug discovery, in addition to systematically defining the consequences of these compounds globally at the level of the proteostasis network. Uncovering the possible side effects of manipulating autophagy at the systemic level remains an important subject for future validation of the pathway as a drug target and to move forward into the development of clinical trials to treat neurodegenerative diseases.

#### Acknowledgments

The authors thank Andrew Foley for critical feedback on this review. This work was funded by FONDECYT 1140549, Millennium Institute No. P09-015-F, CONICYT grant USA2013-0003, ECOS-CONICYT C13S02, The Michael J. Fox Foundation for Parkinson’s Research, the ALS Therapy Alliance, the Muscular Dystrophy Association, the Alzheimer’s Disease Association, and Foundation COPEC-UC (C.H.), Ring Initiative ACT1109 and FONDEF grant No. D11I1007 (C.H. and S.M.), FONDECYT 11121524 (S.M.), CONICYT PAI 7912010006 (R.L.V.), and a CONICYT Master’s fellowship (L.B.).

#### References

1. Taylor, R.C. *et al.* (2014) Systemic stress signalling: understanding the cell non-autonomous control of proteostasis. *Nat. Rev. Mol. Cell Biol.* 15, 211–217
2. Aguzzi, A. *et al.* (2000) Protein aggregation diseases: pathogenicity and therapeutic perspectives. *Nat. Rev. Drug Discov.* 9, 237–248
3. Soto, C. (2012) Transmissible proteins: expanding the prion heresy. *Cell* 149, 968–977
4. Soto, C. (2003) Unfolding the role of protein misfolding in neurodegenerative diseases. *Nat. Rev. Neurosci.* 4, 49–60
5. Hetz, C. *et al.* (2014) Disturbance of endoplasmic reticulum proteostasis in neurodegenerative diseases. *Nat. Rev. Neurosci.* 15, 233–249
6. Menzies, F.M. *et al.* (2011) Protein misfolding disorders and macroautophagy. *Curr. Opin. Cell Biol.* 23, 190–197
7. Sarkar, S. (2013) Regulation of autophagy by mTOR-dependent and mTOR-independent pathways: autophagy dysfunction in neurodegenerative diseases and therapeutic application of autophagy enhancers. *Biochem. Soc. Trans.* 41, 1103–1130
8. Rubinsztein, D.C. *et al.* (2012) Autophagy modulation as a potential therapeutic target for diverse diseases. *Nat. Rev. Drug Discov.* 11, 709–730
9. Sarkar, S. *et al.* (2009) Rapamycin and mTOR-independent autophagy inducers ameliorate toxicity of polyglutamine-expanded Huntingtin and related proteinopathies. *Cell Death Differ.* 16, 46–56
10. Marino, G. *et al.* (2011) Autophagy for tissue homeostasis and neuroprotection. *Curr. Opin. Cell Biol.* 23, 198–206
11. Haass, C. *et al.* (2007) Soluble protein oligomers in neurodegeneration: lessons from the Alzheimer’s amyloid beta-peptide. *Nat. Rev. Mol. Cell Biol.* 8, 101–112
12. Bossy-Wetzel, E. *et al.* (2004) Molecular pathways to neurodegeneration. *Nat. Med.* 10 (Suppl.), S2–S9
13. Dauer, W. *et al.* (2003) Parkinson’s disease: mechanisms and models. *Neuron* 39, 889–909
14. Williams, A.J. *et al.* (2008) Polyglutamine neurodegeneration: protein misfolding revisited. *Trends Neurosci.* 31, 521–528

TIPS-1165; No. of Pages 9

**Review**

15 Imarisio, S. *et al.* (2008) Huntington's disease: from pathology and genetics to potential therapies. *Biochem. J.* 412, 191–209  
16 Pasinelli, P. *et al.* (2006) Molecular biology of amyotrophic lateral sclerosis: insights from genetics. *Nat. Rev. Neurosci.* 7, 710–723  
17 Andersen, P.M. *et al.* (2011) Clinical genetics of amyotrophic lateral sclerosis: what do we really know? *Nat. Rev. Neurol.* 7, 603–615  
18 Mizushima, N. *et al.* (2008) Autophagy fights disease through cellular self-digestion. *Nature* 451, 1069–1075  
19 Maiuri, M.C. *et al.* (2007) Self-eating and self-killing: crosstalk between autophagy and apoptosis. *Nat. Rev. Mol. Cell Biol.* 8, 741–752  
20 He, C. *et al.* (2009) Regulation mechanisms and signaling pathways of autophagy. *Annu. Rev. Genet.* 43, 67–93  
21 He, C. *et al.* (2010) The Beclin 1 interactome. *Curr. Opin. Cell Biol.* 22, 140–149  
22 Pattingre, S. *et al.* (2005) Bcl-2 antiapoptotic proteins inhibit Beclin 1-dependent autophagy. *Cell* 122, 927–939  
23 Nishida, Y. *et al.* (2009) Discovery of Atg5/Atg7-independent alternative macroautophagy. *Nature* 461, 654–658  
24 Klionsky, D.J. *et al.* (2012) Guidelines for the use and interpretation of assays for monitoring autophagy. *Autophagy* 8, 445–544  
25 Williams, A. *et al.* (2008) Novel targets for Huntington's disease in an mTOR-independent autophagy pathway. *Nat. Chem. Biol.* 4, 295–305  
26 Sarkar, S. *et al.* (2007) Trehalose, a novel mTOR-independent autophagy enhancer, accelerates the clearance of mutant Huntingtin and alpha-synuclein. *J. Biol. Chem.* 282, 5641–5652  
27 Sarkar, S. (2013) Chemical screening platforms for autophagy drug discovery to identify therapeutic candidates for Huntington's disease and other neurodegenerative disorders. *Drug Discov. Today Technol.* 10, e137–e144  
28 Sarkar, S. *et al.* (2005) Lithium induces autophagy by inhibiting inositol monophosphatase. *J. Cell Biol.* 170, 1101–1111  
29 Morselli, E. *et al.* (2011) Spermidine and resveratrol induce autophagy by distinct pathways converging on the acetylproteome. *J. Cell Biol.* 192, 615–629  
30 Rubinsztein, D.C. *et al.* (2011) Autophagy and aging. *Cell* 146, 682–695  
31 Kroemer, G. *et al.* (2010) Autophagy and the integrated stress response. *Mol. Cell* 40, 280–293  
32 Ogata, M. *et al.* (2006) Autophagy is activated for cell survival after endoplasmic reticulum stress. *Mol. Cell. Biol.* 26, 9220–9231  
33 Criollo, A. *et al.* (2007) Regulation of autophagy by the inositol trisphosphate receptor. *Cell Death Differ.* 14, 1029–1039  
34 Castillo, K. *et al.* (2011) BAX inhibitor-1 regulates autophagy by controlling the IRE1alpha branch of the unfolded protein response. *EMBO J.* 30, 4465–4478  
35 Shibutani, S.T. *et al.* (2014) A current perspective of autophagosome biogenesis. *Cell Res.* 24, 58–68  
36 Axe, E.L. *et al.* (2008) Autophagosome formation from membrane compartments enriched in phosphatidylinositol 3-phosphate and dynamically connected to the endoplasmic reticulum. *J. Cell Biol.* 182, 685–701  
37 Hara, T. *et al.* (2006) Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice. *Nature* 441, 885–889  
38 Komatsu, M. *et al.* (2006) Loss of autophagy in the central nervous system causes neurodegeneration in mice. *Nature* 441, 880–884  
39 Sridhar, S. *et al.* (2012) Autophagy and disease: always two sides to a problem. *J. Pathol.* 226, 255–273  
40 Nassif, M. *et al.* (2011) Targeting autophagy in ALS: a complex mission. *Autophagy* 7, 450–453  
41 Wong, E. *et al.* (2010) Autophagy gone awry in neurodegenerative diseases. *Nat. Neurosci.* 13, 805–811  
42 Metzger, S. *et al.* (2010) Age at onset in Huntington's disease is modified by the autophagy pathway: implication of the V471A polymorphism in Atg7. *Hum. Genet.* 128, 453–459  
43 Pickford, F. *et al.* (2008) The autophagy-related protein Beclin 1 shows reduced expression in early Alzheimer disease and regulates amyloid beta accumulation in mice. *J. Clin. Invest.* 118, 2190–2199  
44 Shibata, M. *et al.* (2006) Regulation of intracellular accumulation of mutant Huntingtin by Beclin 1. *J. Biol. Chem.* 281, 14474–14485  
45 Hetz, C. *et al.* (2009) XBP-1 deficiency in the nervous system protects against amyotrophic lateral sclerosis by increasing autophagy. *Genes Dev.* 23, 2294–2306  
46 Wills, J. *et al.* (2012) Paraquat, but not maneb, induces synucleinopathy and tauopathy in striata of mice through  

**Trends in Pharmacological Sciences xxx xxxx, Vol. xxx, No. x**

inhibition of proteasomal and autophagic pathways. *PLoS ONE* 7, e30745  
47 Nascimento-Ferreira, I. *et al.* (2011) Overexpression of the autophagic Beclin-1 protein clears mutant ataxin-3 and alleviates Machado-Joseph disease. *Brain* 134, 1400–1415  
48 Wu, J.C. *et al.* (2012) The regulation of N-terminal Huntingtin (Htt552) accumulation by Beclin1. *Acta Pharmacol. Sin.* 33, 743–751  
49 Spencer, B. *et al.* (2009) Beclin 1 gene transfer activates autophagy and ameliorates the neurodegenerative pathology in alpha-synuclein models of Parkinson's and Lewy body diseases. *J. Neurosci.* 29, 13578–13588  
50 Lonskaya, I. *et al.* (2013) Tyrosine kinase inhibition increases functional Parkin-Beclin-1 interaction and enhances amyloid clearance and cognitive performance. *EMBO Mol. Med.* 5, 1247–1262  
51 Nassif, M. *et al.* (2014) Pathogenic role of BECN1/Beclin 1 in the development of amyotrophic lateral sclerosis. *Autophagy* 10, 1256–1271  
52 Mealer, R.G. *et al.* (2014) Rhes, a striatal-selective protein implicated in Huntington disease, binds Beclin-1 and activates autophagy. *J. Biol. Chem.* 289, 3547–3554  
53 Michiorri, S. *et al.* (2010) The Parkinson-associated protein PINK1 interacts with Beclin 1 and promotes autophagy. *Cell Death Differ.* 17, 962–974  
54 Arena, G. *et al.* (2013) PINK1 protects against cell death induced by mitochondrial depolarization, by phosphorylating Bcl-xL and impairing its pro-apoptotic cleavage. *Cell Death Differ.* 20, 920–930  
55 Martinez-Vicente, M. *et al.* (2010) Cargo recognition failure is responsible for inefficient autophagy in Huntington's disease. *Nat. Neurosci.* 13, 567–576  
56 Fecto, F. *et al.* (2011) SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis. *Arch. Neurol.* 68, 1440–1446  
57 Gal, J. *et al.* (2009) Sequestosome 1/p62 links familial ALS mutant SOD1 to LC3 via an ubiquitin-independent mechanism. *J. Neurochem.* 111, 1062–1073  
58 Brady, O.A. *et al.* (2011) Regulation of TDP-43 aggregation by phosphorylation and p62/SQSTM1. *J. Neurochem.* 116, 248–259  
59 Deng, H.X. *et al.* (2011) Mutations in UBQLN2 cause dominant X-linked juvenile and adult-onset ALS and ALS/dementia. *Nature* 477, 211–215  
60 Nassif, M. *et al.* (2010) Amyotrophic lateral sclerosis pathogenesis: a journey through the secretory pathway. *Antioxid. Redox Signal.* 13, 1955–1989  
61 Ferguson, C.J. *et al.* (2009) Defective autophagy in neurons and astrocytes from mice deficient in PI(3,5)P₂. *Hum. Mol. Genet.* 18, 4868–4878  
62 Lee, J.H. *et al.* (2010) Lysosomal proteolysis and autophagy require presenilin 1 and are disrupted by Alzheimer-related PS1 mutations. *Cell* 141, 1146–1158  
63 Yang, D.S. *et al.* (2011) Reversal of autophagy dysfunction in the TgCRND8 mouse model of Alzheimer's disease ameliorates amyloid pathologies and memory deficits. *Brain* 134, 258–277  
64 Dehay, B. *et al.* (2010) Pathogenic lysosomal depletion in Parkinson's disease. *J. Neurosci.* 30, 12535–12544  
65 Dehay, B. *et al.* (2012) Loss of P-type ATPase ATP13A2/PARK9 function induces general lysosomal deficiency and leads to Parkinson disease neurodegeneration. *Proc. Natl. Acad. Sci. U.S.A.* 109, 9611–9616  
66 Orenstein, S.J. *et al.* (2013) Interplay of LRRK2 with chaperone-mediated autophagy. *Nat. Neurosci.* 16, 394–406  
67 Martinez-Vicente, M. *et al.* (2008) Dopamine-modified alpha-synuclein blocks chaperone-mediated autophagy. *J. Clin. Invest.* 118, 777–788  
68 Winslow, A.R. *et al.* (2010) Alpha-synuclein impairs macroautophagy: implications for Parkinson's disease. *J. Cell Biol.* 190, 1023–1037  
69 Zavodszky, E. *et al.* (2014) Mutation in VPS35 associated with Parkinson's disease impairs WASH complex association and inhibits autophagy. *Nat. Commun.* 5, 3828  
70 Youle, R.J. *et al.* (2011) Mechanisms of mitophagy. *Nat. Rev. Mol. Cell Biol.* 12, 9–14  
71 Choubey, V. *et al.* (2014) BECN1 is involved in the initiation of mitophagy: it facilitates PARK2 translocation to mitochondria. *Autophagy* 10, 1105–1119  
72 Fleming, A. *et al.* (2011) Chemical modulators of autophagy as biological probes and potential therapeutics. *Nat. Chem. Biol.* 7, 9–17

TIPS-1165; No. of Pages 9

ARTICLE IN PRESS

Review

73 Ravikumar, B. *et al.* (2004) Inhibition of mTOR induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of Huntington disease. *Nat. Genet.* 36, 585–595

74 Majumder, S. *et al.* (2011) Inducing autophagy by rapamycin before, but not after, the formation of plaques and tangles ameliorates cognitive deficits. *PLoS ONE* 6, e25416

75 Caccamo, A. *et al.* (2010) Molecular interplay between mammalian target of rapamycin (mTOR), amyloid-beta, and tau: effects on cognitive impairments. *J. Biol. Chem.* 285, 13107–13120

76 Caccamo, A. *et al.* (2011) Naturally secreted amyloid-beta increases mammalian target of rapamycin (mTOR) activity via a PRAS 40-mediated mechanism. *J. Biol. Chem.* 286, 8924–8932

77 Spilman, P. *et al.* (2010) Inhibition of mTOR by rapamycin abolishes cognitive deficits and reduces amyloid-beta levels in a mouse model of Alzheimer’s disease. *PLoS ONE* 5, e9979

78 Malagelada, C. *et al.* (2010) Rapamycin protects against neuron death in vitro and in vivo models of Parkinson’s disease. *J. Neurosci.* 30, 1166–1175

79 Sarkar, S. *et al.* (2008) Small molecule enhancers of autophagy for neurodegenerative diseases. *Mol. Biosyst.* 4, 895–901

80 Bhattacharya, A. *et al.* (2012) Dietary restriction but not rapamycin extends disease onset and survival of the H46R/H48Q mouse model of ALS. *Neurobiol. Aging* 33, 1829–1832

81 Staats, K.A. *et al.* (2013) Rapamycin increases survival in ALS mice lacking mature lymphocytes. *Mol. Neurodegener.* 8, 31

82 Zhang, X. *et al.* (2011) Rapamycin treatment augments motor neuron degeneration in SOD1(G93A) mouse model of amyotrophic lateral sclerosis. *Autophagy* 7, 412–425

83 Wang, I.F. *et al.* (2012) Autophagy activators rescue and alleviate pathogenesis of a mouse model with proteinopathies of the TAR DNA-binding protein 43. *Proc. Natl. Acad. Sci. U.S.A.* 109, 15024–15029

84 Ching, J.K. *et al.* (2013) mTOR dysfunction contributes to vacuolar pathology and weakness in valosin-containing protein associated inclusion body myopathy. *Hum. Mol. Genet.* 22, 1167–1179

85 Tanaka, M. *et al.* (2004) Trehalose alleviates polyglutamine-mediated pathology in a mouse model of Huntington disease. *Nat. Med.* 10, 148–154

86 Peruchio, J. *et al.* (2012) Trehalose protects from aggravation of amyloid pathology induced by isoflurane anesthesia in APP(swe) mutant mice. *Curr. Alzheimer Res.* 9, 334–343

87 Rodriguez-Navarro, J.A. *et al.* (2010) Trehalose ameliorates dopaminergic and tau pathology in parkin deleted/tau overexpressing mice through autophagy activation. *Neurobiol. Dis.* 39, 423–438

88 Schaeffer, V. *et al.* (2012) Stimulation of autophagy reduces neurodegeneration in a mouse model of human tauopathy. *Brain* 135, 2169–2177

89 Davies, J.E. *et al.* (2006) Trehalose reduces aggregate formation and delays pathology in a transgenic mouse model of oculopharyngeal muscular dystrophy. *Hum. Mol. Genet.* 15, 23–31

90 Castillo, K. *et al.* (2013) Trehalose delays the progression of amyotrophic lateral sclerosis by enhancing autophagy in motoneurons. *Autophagy* 9, 1308–1320

91 Matus, S. *et al.* (2014) A new method to measure autophagy flux in the nervous system. *Autophagy* 10, 710–714

92 Zhang, X. *et al.* (2014) mTOR-independent, autophagic enhancer trehalose prolongs motor neuron survival and ameliorates the autophagic flux defect in a mouse model of amyotrophic lateral sclerosis. *Autophagy* 10, 588–602

93 Vidal, R.L. *et al.* (2014) Unspliced XBP1 controls autophagy through FoxO1. *Cell Res.* 23, 463–464

94 Congdon, E.E. *et al.* (2012) Methylthioninium chloride (methylene blue) induces autophagy and attenuates tauopathy in vitro and in vivo. *Autophagy* 8, 609–622

95 Sarkar, S. *et al.* (2011) Complex inhibitory effects of nitric oxide on autophagy. *Mol. Cell* 43, 19–32

96 Capell, A. *et al.* (2011) Rescue of progranulin deficiency associated with frontotemporal lobar degeneration by alkalinizing reagents and inhibition of vacuolar ATPase. *J. Neurosci.* 31, 1885–1894

97 Balch, W.E. *et al.* (2008) Adapting proteostasis for disease intervention. *Science* 319, 916–919

98 Mardones, P. *et al.* (2014) Control of systemic proteostasis by the nervous system. *Trends Cell Biol.* http://dx.doi.org/10.1016/j.tcb.2014.08.001

99 Fouillet, A. *et al.* (2013) ER stress inhibits neuronal death by promoting autophagy. *Autophagy* 8, 915–926

100 Vidal, R.L. *et al.* (2012) Targeting the UPR transcription factor XBP1 protects against Huntington’s disease through the regulation of FoxO1 and autophagy. *Hum. Mol. Genet.* 21, 2245–2262

101 Safra, M. *et al.* (2013) The ire-1 ER stress-response pathway is required for normal secretory-protein metabolism in *C. elegans*. *J. Cell Sci.* 126, 4136–4146

102 Hetz, C. *et al.* (2014) Targeting the unfolded protein response in disease. *Nat. Rev. Drug Discov.* 12, 703–719

103 Nascimento-Ferreira, I. *et al.* (2013) Beclin 1 mitigates motor and neuropathological deficits in genetic mouse models of Machado-Joseph disease. *Brain* 136, 2173–2188

104 Jaeger, P.A. *et al.* (2010) Regulation of amyloid precursor protein processing by the Beclin 1 complex. *PLoS ONE* 5, e11102
