
frontiers in
CELLULAR AND INFECTION MICROBIOLOGY

Quorum sensing in group A Streptococcus

Juan Cristobal Jimenez¹ and Michael J. Federle²*

1 Department of Microbiology and Immunology, College of Medicine, University of Illinois at Chicago, Chicago, IL, USA  
2 Department of Medicinal Chemistry and Pharmacognosy, Center for Pharmaceutical Biotechnology, College of Pharmacy, University of Illinois at Chicago, Chicago, IL, USA  

Edited by:  
Emmanuelle Charpentier, Umeå University, Sweden  

Reviewed by:  
Charlene Kahler, University of Western Australia, Australia  
Irene Wagner-Doebler, Helmholtz-Centre for Infection Research (HZI), Germany  

*Correspondence:  
Michael J. Federle, Department of Medicinal Chemistry and Pharmacognosy, Center for Pharmaceutical Biotechnology, College of Pharmacy, University of Illinois at Chicago, 3152 MBRB (M/C 870), 900 S. Ashland Ave., Chicago, IL 60607, USA  
e-mail: mfederle@uic.edu  

Quorum sensing (QS) is a widespread phenomenon in the microbial world that has important implications in the coordination of population-wide responses in several bacterial pathogens. In Group A Streptococcus (GAS), many questions surrounding QS systems remain to be solved pertaining to their function and their contribution to the GAS lifestyle in the host. The QS systems of GAS described to date can be categorized into four groups: regulator gene of glucosyltransferase (Rgg), Sil, lantibiotic systems, and LuxS/AI-2. The Rgg family of proteins, a conserved group of transcription factors that modify their activity in response to signaling peptides, has been shown to regulate genes involved in virulence, biofilm formation and competence. The sil locus, whose expression is regulated by the activity of signaling peptides and a putative two-component system (TCS), has been implicated on regulating genes involved with invasive disease in GAS isolates. Lantibiotic regulatory systems are involved in the production of bacteriocins and their autoregulation, and some of these genes have been shown to target both bacterial organisms as well as processes of survival inside the infected host. Finally AI-2 (dihydroxy pentanedione, DPD), synthesized by the LuxS enzyme in several bacteria including GAS, has been proposed to be a universal bacterial communication molecule. In this review we discuss the mechanisms of these four systems, the putative functions of their targets, and pose critical questions for future studies.

Keywords: quorum sensing, pheromones, cell-cell signaling, Streptococcus pyogenes, Rgg, Sil, lantibiotics, AI-2

BACTERIAL COMMUNICATION IN GRAM-POSITIVE BACTERIA

For a long time, bacteria were thought of as organisms carrying out self-sufficient and independent, unicellular lifestyles. During the last 40 years, several studies have demonstrated how, in fact, bacteria interact and establish complex social behaviors with their siblings and with other bacteria in their community to develop beneficial actions for the population, by means of conserved chemical languages. Quorum Sensing (QS) is the communication process in which bacteria produce, secrete and detect chemical signals with the purpose of triggering specific phenotypical responses. QS regulates genes involved in population-wide decisions and behaviors that are beneficial when performed as a synchronous group rather than at the individual level and which include bioluminescence, sporulation, competence, antibiotic production, biofilm formation, and secretion of virulence factors (Reviewed by Atkinson and Williams, 2009; Ng and Bassler, 2009; Rutherford and Bassler, 2012).

QS signaling in Gram-positive bacteria (Figure 1) operates through the activity of post-translationally modified oligopeptides, named autoinducing peptides or pheromones, which can range from 5 to 34 amino acids in length and can adopt either linear or cyclical conformations (Havarstein et al., 1995; Ji et al., 1995; Kuipers et al., 1995; Solomon et al., 1996; Otto et al., 1998; Mayville et al., 1999; Sturme et al., 2005). These pheromones are initially synthesized as inactive pro-peptides in the ribosome, and then exported from the cell by either the general secretion system (Sec) or by dedicated ABC transporters (Hui and Morrison, 1991; Zhang et al., 2002; Stephenson et al., 2003). During the export event, pro-peptides undergo proteolytic processing (and in some cases additional covalent modification) to generate the active pheromone, and a variety of enzymes have been involved in these maturation processes (Magnuson et al., 1994; Otto et al., 1998; An et al., 1999; Mayville et al., 1999; Zhang et al., 2002; Lanigan-Gerdes et al., 2007; Thoendel and Horswill, 2009). When the pheromones surpass threshold concentrations in the extracellular medium they are efficiently detected by transmembrane receptors of the two-component system (TCS) signal transduction family, leading to differential phosphorylation of a response regulator and consequent change in target gene expression. Alternatively, pheromones can be imported into the cytoplasm via peptide transporter complexes, most commonly the Opp/Ami oligopeptide permease, a promiscuous transporter of peptides involved in the import of nutritional peptides, peptidoglycan recycling components as well as pheromone peptides for other QS systems (Leonard et al., 1996; Lazazzera et al., 1997; Slamti and Lereclus, 2002; Fontaine et al., 2010; Mashburn-Warren et al., 2010; Chang et al., 2011). Once inside the cell, peptide pheromones bind and directly modulate the activity of transcriptional regulators inside the cell. As a result of signaling, target genes change their expression pattern and genes encoding for the pheromone pre-peptides are upregulated, increasing the production of mature pheromone

Jimenez and Federle                                                                 Cell-cell signaling in S. pyogenes

![Diagram](#)

fasciitis and toxic shock syndrome. GAS presence in the host does not necessarily correlate with a disease state, as it can be isolated from several body sites in asymptomatic individuals and is carried asymptomatically in approximately 20% of school-aged children (Kaplan and Huwe, 1980; Mead and Winn, 2000; Shaikh et al., 2010; Roberts et al., 2012). GAS is an organism that develops its complete life cycle inside the human host, passing from non-virulent asymptomatic colonization to symptomatic infection, to transmission and dissemination (Wollein Waldeoft et al., 2014). Because of this, it is of great interest to understand the ways by which GAS populations regulate their collective behaviors, and how these behaviors may be associated with adaptation to the host environment or with the switch between pathogenic and non-pathogenic states. In this review we examine the mechanism and function of four families of QS systems present in GAS: regulator gene of glucosyltransferase (Rgg), Sil, lantibiotic regulatory systems, and AI-2.

### RGG TRANSCRIPTIONAL REGULATORS

In Gram-positive bacteria, two families of conserved transcription factors have been reported to interact with imported peptide pheromones: RNPP and Rgg. The RNPP family, named for its prototypical members Rap, NprR, PlcR, and PrgX, is characterized by the presence of tetratricopeptide repeat domains (TPRs) in their C-terminal domains, defined by motifs that are involved in protein-protein interactions and are involved in peptidic pheromone binding (Blatch and Lässle, 1999; Core and Perego, 2003). Some members of the RNPP family also possess Helix-Turn-Helix (HTH) motifs for DNA binding and direct regulation of gene expression (Declerck et al., 2007; Rocha-Estrada et al., 2010). RNPP Representatives are found in *Bacillus* and *Enterococcus* species and have been shown to regulate processes of sporulation, conjugation, biofilm formation and pathogenic responses (Rocha-Estrada et al., 2010). The second conserved group of peptide-binding transcription factors is the Rgg family. Members of this family possess characteristic HTH motifs in their N-terminal domains and a C-terminal region rich in alpha-helical structures. Rgg members possess a low level of sequence similarity with members of the RNPP family and their C-terminal domains, presumed to bind peptide pheromones, have not been characterized structurally so far, and no crystallographic data of Rgg members is available yet (Fleuchot et al., 2011). The Rgg family members are widespread in the low-G+C Gram-positive bacteria and only absent in the Clostridiaceae (Fleuchot et al., 2011). Rgg proteins have been shown to behave like activators or repressors of DNA expression, while some can exhibit simultaneously both regulatory functions (Rawlinson et al., 2002; Samen et al., 2006; Anbalagan et al., 2011). The first Rgg family member was identified in the oral bacterium *Streptococcus gordonii*, in which extracellular glucosyltransferase activity required for tooth surface colonization was shown to be promoted by the activity of the Rgg protein (Sulavik et al., 1992). Since then, several other Rgg members have been characterized in Streptococcal species including *S. oralis* (Fujiwara et al., 2000), *S. thermophilus* (Fernandez et al., 2006; Ibrahim et al., 2007a; Fontaine et al., 2010), *S. salivarius* (Fontaine et al., 2010), *S. pneumoniae* (Bortoni et al., 2009), *S. mutans* (Qi et al., 1999),

and generating a positive-feedback loop (or autoinduction process), that helps strengthen the QS signaling at the individual level and increases pheromone levels in the environment to activate signaling at the population level.

Among the earliest reports describing cell-to-cell communication in bacteria arose from studies of genetic exchange in *Streptococcus pneumoniae*, where it was shown that a “hormone-like cell product” secreted by the bacteria into the culture medium activated their ability to import extracellular DNA and undergo genetic transformation in a process termed competence (Tomasz, 1965). Several years later, the signaling molecule was shown to be a processed peptide named CSP (competence-stimulating peptide), that formed part of a quorum-sensing circuit (Håvarstein et al., 1995). Since then, several pheromones and their associated QS systems have been discovered and characterized in species of the *Streptococcus* genus (Reviewed by Cook and Federle, 2014). The existence of QS systems in *Streptococcus pyogenes*, a human-restricted pathogen capable of causing a broad spectrum of diseases, was unknown until recently. Also known as Group A Streptococcus (GAS), *S. pyogenes* is a polyauxotrophic organism that resides primarily in the oropharynx and the skin of its human host and generates illnesses that range from mild, like pharyngitis and impetigo, to severe and life threatening, like necrotizing

Frontiers in Cellular and Infection Microbiology   www.frontiersin.org   September 2014 | Volume 4 | Article 127 | 2

S. agalactiae (Samen et al., 2006), and S. suis (Zheng et al., 2011). Some species may even harbor multiple rgg-like genes in their genomes, like the case of Streptococcus thermophilus, predicted to encode in its genome seven different Rgg paralogs (Ibrahim et al., 2007b). After the discovery that the deletion of a small pre-peptide gene inhibited the regulatory activity of an Rgg protein in S. thermophilus, it was recognized that activity of Rgg regulators was modulated by short peptides, constituting putative QS circuits (Ibrahim et al., 2007a). Commonly, Rgg genes are located next to a short open reading frame that encodes the propeptide of their cognate pheromone, short genes which are usually overlooked in genome annotation processes but have been predicted by *in silico* analysis (Ibrahim et al., 2007b; Fleuchot et al., 2011). Rgg pheromones have been classified in two groups to date, short hydrophobic peptides (SHPs) and peptides involved in competence pathways, termed XIPs (Table 1) (Mashburn-Warren et al., 2010; Fleuchot et al., 2011). Since some of the Rgg/pre-peptide loci show high conservation among different streptococci, it has also been shown that interspecies cross-talk can occur via SHP-pheromones (Cook et al., 2013; Fleuchot et al., 2013). In GAS, four Rgg paralogs can be identified based on sequence homology: RopB (Rgg), Rgg2, Rgg3, and ComR (Rgg4) (Chang et al., 2011; Federle, 2012).

**RopB**

The most studied Rgg member of GAS is the RopB protein. One of the important GAS “stand-alone” regulators, transcription factors whose cognate sensor partners are unknown (Kreikemeyer et al., 2003), RopB, a predicted dimeric protein, has been shown to both positively and negatively modulate the expression of several genes (Figure 2A) (Chaussee et al., 2002, 2003, 2004, 2008; Dmitriev et al., 2006, 2008; Hollands et al., 2008; Carroll et al., 2011). RopB was initially discovered in a mutant screen for regulators of the potent cysteine protease SpeB. The streptococcal pyrogenic exotoxin B, SpeB, targets a wide range of proteins from the host, including components of the extracellular matrix and modulators of the immune response, as well as GAS proteins present in its envelope or secreted into the extracellular medium (reviewed by Chiang-Ni and Wu, 2008; Nelson et al., 2011). Hence, SpeB plays an important role in modifying the host response and in reshaping the bacterial surface, a fact that is reflected in the number of factors involved in its regulation (Carroll and Musser, 2011). RopB is essential for *speB* transcription, and regulates its expression by directly binding the distal P1 promoter in the intergenic region between the *ropB* and *speB* genes. As with other genes regulated by activation factors, the *speB* P1 promoter lacks a canonical –35 promoter sequence, which may result in poor binding by RNA polymerase and low expression in absence of the transcriptional activator. Located upstream of the –35 region of P1 are a two inverted repeat sequences, common recruitment motifs for dimeric DNA-binding proteins, which RopB may interact with in order to bind and recruit polymerase and activate *speB* expression, consistent with the activity of a type II activator (Neely et al., 2003; Browning and Busby, 2004). How RopB represses the expression of target genes is less understood, but it has been shown to be able to bind two different promoters in the upstream region of the repressed, prophage encoded *spd-3* gene, which possess –35 sequences closer to the consensus, suggesting that in this case RopB binding blocks the promoter region, inhibiting polymerase recruitment (Anbalagan and Chaussee, 2013).

Even though it is a member of the Rgg family, RopB has not been widely characterized as a pheromone binding protein or as a QS component, as it lacks characteristics reported in other Rgg-pheromone systems. No obvious pheromone encoding gene has been described near the *ropB* gene. Additionally, several studies on SpeB production and mutagenesis screens performed to understand its regulators in GAS have revealed a variety of factors modulating RopB activity, but so far no pre-peptide encoding ORF. Nonetheless, several results have shown that RopB activity is affected by interaction with peptides and proteins: The regulation of *speB* expression by RopB has been shown to be dependent on cell cycle, with the highest degree of expression beginning in the stationary phase, suggesting a requirement for high cell density conditions for RopB-mediated gene activation. If *ropB* is expressed from a non-native promoter at high levels during early exponential phase, *speB* transcription is still not activated, implying that additional factors are required for RopB regulation (Neely et al., 2003). Mutations in Opp or its paralog Dpp (dipeptide permease) generate a drastic decrease in *speB* expression, suggesting that peptide import into the cell is also required for *speB* regulation (Podbielski et al., 1996; Podbielski and Leonard, 1998; Wang et al., 2005). Other results have shown the role of proteins that directly interact with RopB to affect its regulatory function: LacD.1 is an aldolase enzyme that has been proposed to link metabolic status of GAS with virulence

**Table 1 | GAS Pheromones.**

| Pre-peptide (pheromone) | Sequence | References |
|--------------------------|----------|------------|
| SHP2 (SHP2-C8)          | MKKISKFLPILILAMDIIIVGG | Chang et al., 2011 |
| SHP3 (SHP3-C8)          | MKKVNKALLFTLIMDILIIVGG | Chang et al., 2011 |
| ComS M1 (XIP)           | MLKKYKYFIFAALLSFKVVO   | Mashburn-Warren et al., 2012 |
|                          | ELSAVDWWRL             |                      |
| ComS M3 (XIP)           | MLKKVKPFLLAAVAFKVA     | Mashburn-Warren      |
|                          | RVMHEFDWWNLG           | et al., 2012         |
| SilCR                   | MNNKKTKNNFSTLSESELLK   | Hidalgo-Grass et al., 2004 |
|                          | VIGGDIFFLVIDHISMKARKK  |                      |
| Salivaracin A (SalA1)   | MSFMKNSKDILTNAIEEVSE   | Upton et al., 2001   |
|                          | KELMEVAGGKKGSGWFA      |                      |
|                          | TITDDCPNSVFVCC         |                      |
| Streptin (Streptin 1a)  | MNNTIKDFDLDLKTNKKDT    | Wescombe and Tagg, 2003 |
|                          | ATPYVGSRYLC_TPGSCWK    |                      |
|                          | LVCFTTTVK              |                      |
| Streptococcin A-FF22    | MEKNNEVINSIQEVSLEEL    | Jack et al., 1994b   |
|                          | DQIIGAGKNGVFKTISHE     |                      |
|                          | CHLN7WAFLATCCS         |                      |

*Amino-acid sequences of pheromone pre-peptides and predicted cleaved pheromones (in bold). Double-glycine and glycine-alanine cleavage motifs are italicized and underlined. Pheromone residues that undergo dehydration reactions are bold and italicized, and residues believed to form thioether bridges are bold and underlined.*

Jimenez and Federle                                                                 Cell-cell signaling in S. pyogenes

FIGURE 2 | Continued  
complete the autoinduction loop. (C) Regulation of competence genes by ComR. ComS pro-peptide is produced, secreted and processed to generate the XIP peptide. After being imported through Opp, XIP can bind ComR, which binds to the promoters of *comS* and *sigX* to activate their expression. The alternative sigma factor σX together with RNA polymerase bind to Com box sites in target genes and activate the expression of competence related genes. Promoters with open arrows are Rgg-independent. Promoters with filled arrows are activated by Rgg-proteins. Promoters with X symbol are repressed by Rgg-proteins.

gene regulation, as it is able to bind intermediates of the glycolytic pathway that are present in high concentrations during exponential growth and consequently influence gene expression (Loughman and Caparon, 2006b). LacD.1 and RopB can be co-precipitated, and in absence of LacD.1 a constitutive *ropB* allele is able to partly uncouple SpeB activity from cell cycle regulation. Other reports, based on the observation of differential gene regulation by RopB during stages of bacterial growth, have suggested that LacD.1 binds to RopB during exponential phase, changing its DNA binding preferences and affecting the repertoire of genes regulated (Anbalagan et al., 2012). Another protein involved in repressing RopB-dependent SpeB activation is Vfr (virulence factor related) (Ma et al., 2009). The *vfr* gene was identified by transposon mutagenesis, as a mutant with increased protease activity, and SpeB activity is inversely correlated with *vfr* expression; *vfr* expression is maximal in early logarithmic stages and decreased toward stationary phase. Additional work by Shelburne et al. (2011) showed that the N-terminal region of the Vfr protein, which forms part of a putative cleavable secretion signal, was able to bind RopB and was sufficient to exert the inhibiting activity. However, a factor accounting for a necessary activating signal of SpeB regulation by RopB has yet to be identified.

Studies on the scope of RopB regulation have revealed a wide range of variation in the number of controlled genes depending on the studied GAS isolate (Table 2). In the strain NZ131, which has been characterized in several studies of RopB regulated phenotypes, the deletion of *ropB* generates a pleiotropic effect, affecting the expression of 293 genes during exponential phase, and changing the expression of 567 genes during post-exponential phase. These include genes involved in metabolism, stress response, prophage excision, and virulence (Dmitriev et al., 2006). These gene changes reflect on several phenotypes shown for NZ131 Δ*ropB* mutants, including increased secretion of DNases MF1 and Spd3/MF3, decreased secretion of ClpB chaperone, lysozyme and autolisin (Watson et al., 2001); increased production of M protein and the virulence factors C5a peptidase, Streptolysin S, and Streptolysin O (Chaussee et al., 2002); increased catabolism of arginine and serine (Chaussee et al., 2003); inability to utilize alternative carbon sources (Dmitriev et al., 2006) and increased resistance to H₂O₂ (Pulliainen et al., 2008). It has been thought that this pleiotropic effect is based upon the ability of RopB to influence the expression of other regulatory factors, as it has been shown to increase the expression of *covRS* TCTS, while repressing the expression of *mga* and *sagA* stand-alone regulators (Chaussee et al., 2002). However, when ChIP-on-chip analyses were performed, revealing 125 RopB

FIGURE 2 | Rgg regulators of *Streptococcus pyogenes*. (A) RopB directly activates the expression of *speB* and its associated downstream genes, while directly repressing the prophage encoded *spd3* DNase. RopB also affects, by direct and indirect manners, the expression of a varying group of genes in different isolates. Other factors can modulate RopB activity, like LacD.1, the N-terminal peptide of the Vfr protein and unknown factors imported by the Opp and Dpp permeases. (B) The Rgg2/3 system. Left panel: in the absence of SHPs, Rgg3 is bound to the promoter region of the pheromone genes, inhibiting their expression. Addition of exogenous SHP pheromones that bind Rgg3, cause its release from DNA, while allowing Rgg2 to bind the same promoter region and promote expression of pheromone genes. Right panel: activation of the Rgg2/3 system triggers expression of *shp2*, *shp3* and their downstream genes. Translated SHP2 and SHP3 pro-peptides are secreted through an unknown exporter and processed by the activity of Eep and additional extracellular enzymes. The active SHP2-C8 and SHP3-C8 pheromones are imported via Opp to (Continued)

Jimenez and Federle                                                                 Cell-cell signaling in S. pyogenes

Table 2 | RopB regulated genes.

| Strain (M type)       | Genes that change (% genome) | Genes decreasing expression | Genes increasing expression | References         |
|-----------------------|-------------------------------|-----------------------------|-----------------------------|--------------------|
| NZ131 (M49)$^a$      | 567 (31)                     | 340                         | 227                         | Dmitriev et al., 2006 |
| CS101 (M49)$^a$      | 13                           | 3                           | 10                          | Dmitriev et al., 2008 |
| SF370 (M1)$^a$       | 45 (2.5)                     | 17                          | 28                          | Dmitriev et al., 2008 |
| MGAS5005 (M1)$^a$    | 3 (0.15)                     | 2                           | 1                           | Dmitriev et al., 2008 |
| GAS5448 (M1T1)$^b$   | 99                           | 47                          | 52                          | Hollands et al., 2008 |
| MGAS10870 (M3)$^a$   | 479 (25)                     | 27                          | 452                         | Carroll et al., 2011 |
| MGAS9937 (M3)$^a$    | 159 (8)                      | 13                          | 146                         | Carroll et al., 2011 |

Gene expression changes in ΔropB strains compared with wild type strains as shown by microarray analyses.

$^a$ Post-exponential/early stationary phase.

$^b$ In vivo.

response (LaSarre et al., 2013). Interestingly, even though other streptococci carry homologs of the *rgg2-shp2* or the *rgg3-shp3* loci (discussed below), to date GAS is the only sequenced streptococcal species that carries both *rgg2-shp2* and the *rgg3-shp3* loci, suggesting that this bacterium has evolved to use both regulators and their two cognate pheromones. As reported for other Rgg systems, an intact Opp system is required in order to import the pheromones and trigger activation of the system. Experiments also revealed that a deletion of the *rgg3* gene derepresses expression of the *shp2* and *shp3* genes, while a double mutant Δ*rgg2*Δ*rgg3* lacks nearly all expression of pheromones. A single deletion of *rgg2* also renders GAS grossly unresponsive to synthetic SHPC8 pheromones, illustrating how Rgg3 acts as a repressor of gene expression, while Rgg2 activates gene expression in the absence of the repressive effect of Rgg3 (Figure 2B). Deletion of the *eep* gene, encoding for a metalloprotease involved in pheromone processing in other Gram-positive QS systems, reduces the effectiveness of the autoinduction process, suggesting a role in the processing of SHP2/3 pre-peptides into their mature form. This effect can however be ameliorated if SHP3 is overexpressed, suggesting that additional proteins are involved in SHP2/3 processing (Chang et al., 2011).

Both Rgg2 and Rgg3 have been shown to bind directly to the promoter regions of *shp2* and *shp3* (Chang et al., 2011; LaSarre et al., 2012). Interestingly, when the precise DNA binding sites of both regulators were mapped by DNase I footprinting, Rgg2, and Rgg3 where shown to bind to the same conserved sequence in the P*shp2* and P*shp3* promoters, hence only one Rgg protein is able to bind per promoter at any given time, indicating that Rgg3 exerts its repressing activity by binding the shared DNA site and generating steric interference that inhibits Rgg2 binding (LaSarre et al., 2012). This is consistent with the fact that the HTH domains of Rgg2 and Rgg3 are distinctively analogous, sharing 71% identity and 94% similarity. A higher degree of variability is seen in the C-terminal regions of the Rgg2 and Rgg3 proteins, which are predicted to be involved in protein oligomerization and interaction with RNA polymerase, and this variability may determine the fundamental differences in the activities of these two regulators (LaSarre et al., 2012). Similar to the RopB interaction with the *speB* promoter, Rgg2 and Rgg3 bind the P*shp2* and P*shp3* promoters in their –35 regions, which have poor resemblance with consensus sequences, suggestive of a class II activator dependent promoter (LaSarre et al., 2012). Rgg2 and Rgg3 also directly bind mature SHP2 and SHP3 pheromones, exhibiting a higher affinity for the SHPC8 pheromone variants, while not binding full-length pro-peptides (Aggarwal et al., 2014). Using EMSA assays, it was revealed that either Rgg can bind the target DNA sequences in the absence of pheromones. However, when pheromones are added in increasing concentrations, Rgg3 is released from the DNA while Rgg2 remains unaffected. When the experiment is set up with both regulators to mimic the DNA-binding competition event *in vitro*, addition of synthetic pheromones affects the binding ratio, which becomes skewed in favor of Rgg2-DNA interactions over Rgg3-DNA interactions (LaSarre et al., 2012).

The function of the Rgg2/3 circuit and its contribution to the GAS lifestyle has not been completely elucidated. Apart from the

*shp2* and *shp3* genes, other major regulatory targets of the Rgg2/3 pathway are the genes located downstream of the pheromone genes (Figure 2B) (Chang et al., 2011). Downstream of *shp3*, a putative biosynthetic operon composed of nine genes is encoded, which harbors genes with diverse enzymatic functions, like shikimate dehydrogenase, sugar isomerase, glycosyltransferase, and oxidoreductase. Components from this operon have been identified in mutagenesis screens for genes affecting mucoid colony morphology, virulence in zebra fish, and have also been involved in capsule formation in GAS (Biswas and Scott, 2003; Kizy and Neely, 2009; Cole et al., 2012). A putative locus encoding homologs of *rgg3-shp3* and its downstream associated genes are also present in *Streptococcus pseudoporcinus* and *Bacillus thuringiensis* species. In the latter, an operon that encodes genes with similar enzymatic functions has been implicated in the production of thuringiensin, a wide-range insecticidal toxin predicted to inhibit RNA polymerase (Liu et al., 2010). Downstream of *shp2*, Rgg2/3 also controls a highly conserved region in GAS, which encodes a series of hypothetical genes of unknown function. This region however is required for a SHP-dependent increase in biofilm formation seen in NZ131 when the Rgg2/3 system is activated (Jimenez, unpublished results). We are currently working to elucidate the mechanism by which this system contributes to biofilm formation. Homologous loci encoding *rgg2-shp2* with its downstream-associated genes are conserved in species of the pyogenic group streptococci, including *S. canis*, *S. agalactiae*, *S. dysgalactiae* subspecies *dysgalactiae* and *equisimilis* and *S. iniae*. The *S. agalactiae* Rgg2 homolog, called RovS, has been involved in regulating expression of genes required for epithelial cell attachment and hemolysis activity (Samen et al., 2006), and we have recently shown that this transcription factor is also modulated by a SHP pheromone and that both GAS and *S. agalactiae* (also called Group B Streptococcus, GBS) can undergo interspecies QS signaling when co-cultured (Cook et al., 2013). Since GAS and GBS can be isolated from the same sites in the human host, it is intriguing to wonder if these common signaling pathways constitute a way for these two organisms to coordinate intraspecies behaviors. Alternatively, and since other pyogenic group species that carry *rgg2-shp2* homologs reside in other mammalian species, this system may serve as a conserved and efficient way to control gene expression that has evolved to control specific traits in each organism.

### ComR

ComR is another Rgg member present in GAS, involved in natural genetic transformation processes in other streptococci through the regulation of early genes in the competence cascade (Figure 2C). Competence, the temporal physiological state of DNA receptivity, relies in the coordinated and sequential expression of a series of genes, encoding proteins responsible for pheromone signaling, DNA binding, uptake, processing and recombination into the chromosome (reviewed by Johnsborg et al., 2007). In the streptococci, the master regulator of competence is SigX/ComX, an alternative sigma factor of RNA polymerase that recognizes a conserved DNA sequence termed cinbox or combobox and controls the expression of the competence “late genes” required for DNA binding, uptake and recombination (Lee
and Morrison, 1999; Peterson et al., 2000; Luo and Morrison, 2003). All the genes induced before *sigX* in the competence cascade and that are involved in the signaling processes that lead to *sigX* expression are termed “early genes.” In *S. pneumoniae*, the expression of *sigX* is under the control of a QS system, composed of the ComDE TCS which senses and responds to the competence inducing pheromone CSP. GAS and other streptococci of the pyogenic, bovis, and salivarius groups lack ComDE but have an alternative circuit, based on ComR activity, to trigger *sigX* expression and concomitant upregulation of late genes in the bacterial population. ComR was first discovered in *Streptococcus thermophilus*, a species from the salivarius group that is able to express competence genes and undergo natural transformation in an Opp-dependent manner when grown in peptide-free chemically defined medium (CDM), suggesting that import of self-produced peptides was essential for transformation (Gardan et al., 2009). To identify the regulators of *sigX* in *S. thermophilus*, the transcriptome of a mutant unable to express any competence genes (∆*opp* mutant) was compared with a strain only able to express early competence genes (∆*sigX* mutant). One of the identified components was a previously identified *rgg*-pheromone gene pair which were then shown to be required for *sigX* expression and natural transformation (Ibrahim et al., 2007a; Gardan et al., 2009). The regulator was named ComR for competence regulator and the pro-peptide gene was called *comS* for competence signal (Fontaine et al., 2010). The *sigX* promoter, bound by ComR, lacks a canonical –35 sequence and has an inverted repeat element in this region, suggestive with a Class II activator dependent promoter, similar to the *Pshp2/3* promoters activated by Rgg2 and the P1 *speB* promoter activated by RopB (Mashburn-Warren et al., 2010). *In silico* screening in streptococcal genomes revealed the presence of *comRS* orthologs in the genomes of pyogenic, bovis and mutans streptococci (Mashburn-Warren et al., 2010). The predicted ComS pro-peptides from these species had a similar features, being composed by a high degree of hydrophobic residues, possessing a net positive charge and a double tryptophan (WW) motif in their C-terminal region. Characterization of the ComRS system in *S. mutans* (that interestingly, has both ComDE and ComRS systems) replicated the results seen in *S. thermophilus*, and additionally showed that a synthetic pheromone consisting of the last 8 C-terminal amino acids of the 17 amino-acid long ComS peptide was able to trigger *sigX* expression and transformation in an Opp- and ComR-dependent manner, mimicking the activity of the putative mature pheromone. This active pheromone was named XIP for *sigX*-inducing peptide.

The effect of ComRS signaling in GAS and other pyogenic-group Streptococci, and its effect over transformation have proven to be more cryptic than in non-pyogenic streptococcal counterparts. Even though genome sequencing data has shown that horizontal transfer and genetic exchange between GAS and other streptococcal species is common, the demonstration of natural transformation of GAS in laboratory conditions has proven elusive. Genomic analysis by Mashburn et al. found *comRS* genes in all sequenced GAS genomes, and two allelic variants named M1 and M3 were found. Between these variants, ComR has a highly conserved N-terminal region, indicating similar DNA-sequence binding, but distinct C-terminal regions, suggesting a differential

interaction with pheromones (Mashburn-Warren et al., 2012). Indeed, the sequence of XIPs of M1 and M3 alleles also differ, while both contain WW motifs (Table 1). In the absence of exogenous XIP, *sigX* expression is low. Addition of synthetic XIP activates ComR and triggers expression of *sigX* in GAS strains, in an allele-specific manner. Concentrations as low as 0.5 nM XIP are sufficient to trigger *sigX* expression, and microarray analysis revealed that XIP sensing by ComR generated the upregulation of 30 different genes, 21 of these containing com boxes, and thus regulated directly by SigX (Mashburn-Warren et al., 2012). Most of the regulated genes corresponded to predicted competence late genes. Two other regulated genes of interest were *murM2* and *rocA*, involved respectively in processes of antimicrobial resistance and upregulation of the CovR TCS component, a major regulator of GAS genes. All gene functions known to be required for genetic transformation in *S. pneumoniae* and *Bacillus subtilis* were upregulated in response to XIP in GAS, but nonetheless transformation was not seen *in vitro*. Further experiments revealed that competence was blocked at the DNA uptake process, suggesting that unknown post-transcriptional regulation events were inhibiting transformation (Mashburn-Warren et al., 2012). Additional environmental signals may therefore be required to unblock the competence machinery, and the first evidence for this notion has been associated with growth in biofilms. As sessile bacterial aggregates, biofilms contain high densities of bacterial cells in close contact and studies have shown that Streptococci that undergo natural transformation do so in higher efficiencies when growing inside biofilms (Marks et al., 2012; Wei and Håvarstein, 2012). Recently, the first report of effective GAS natural transformation in the laboratory was seen in biofilm cultures (Marks et al., 2014). GAS biofilms grown on a substrate of fixed epithelial cells were shown to internalize and incorporate exogenous DNA at a low frequency, in a ComR-dependent manner. Addition of XIP generated a ten-fold increase in efficiency, but was not required for transformation, suggesting that growth on epithelia provides a signal(s) for Com system activation. The transformation phenotype could also be replicated during an *in vivo* biofilm growth assay, where intranasally colonized mice were also given donor DNA. Again, the addition of the XIP pheromone generated a ten-fold increase in the number of transformed colonies (Marks et al., 2014). These results suggest that GAS are indeed capable of undergoing genetic transformation, but an unidentified regulatory check-point requires an environmental condition that appears to be satisfied through growth on host cells. Also, a number of GAS isolates have reported mutations in the genes that form the DNA uptake complex, indicating the ability to induce the complete competence machinery is naturally lost in some of them, however the conserved maintenance of genes involved in the QS-signaling process suggests that ComR and SigX may have additional beneficial functions in non-competent strains (Woodbury et al., 2006; Mashburn-Warren et al., 2012).

### SIL SIGNALING SYSTEM

The Sil system was the first QS network characterized in GAS. The *sil* locus was discovered in a tagged-transposon mutagenesis screen performed on a highly invasive M14 strain (JS95),which was isolated from a case of necrotizing fasciitis, and is capable of generating a lethal invasive infection in a murine model (Hidalgo-Grass et al., 2002). The screen was designed to identify mutant clones that lost their ability to spread invasively from a skin infection site to other target organs in mice. One of the isolated mutants lost its ability to reach the spleen, while not affecting its ability to survive in blood, and had an insertion in a novel locus that was termed streptococcal invasion locus (*sil*). The core Sil system consists of six genes, *silABCDE* and *silCR* encoded in a putative genomic island of 15–17 Kbps in size, comprising a lower GC content than the average of the GAS genome (Hidalgo-Grass et al., 2002; Eran et al., 2007; Belotserkovsky et al., 2009) (Figure 3). The presence of *sil* genes in GAS genomes is not widespread, with only four of the 19 sequenced GAS genomes contain this system (MGAS8232, MGAS10750, Alab49, and HSC5) (Kizy and Neely, 2009; Michael-Gayego et al., 2013). Epidemiological studies have shown that the prevalence of Sil ranges from 12% in non-invasive isolates to 16–25% in invasive isolates, and that the locus is restricted to a few *emm*-types with M4 being the most common *sil*-harboring isolate (Bidet et al., 2007; Billal et al., 2008; Michael-Gayego et al., 2013). The sequenced strains that do not possess Sil have instead remnants of the genomic island, suggesting that DNA recombinations may have been responsible of loss of *sil* genes (Figure 3B) (Belotserkovsky et al., 2009).

The Sil system has sequence homology with the competence (Com) and the bacteriocin-like peptide (Blp) QS systems of *Streptococcus pneumoniae* (De Saizieu et al., 2000; Morrison and Lee, 2000). The *sil* QS locus (Figure 3A) is composed of *silAB*, a putative TCS, *silDE*, a putative ABC transporter, and *silCR*, the pheromone pro-peptide gene. An additional small ORF encoded in the complementary DNA strand and overlapping with *silCR* is *silC*, which encodes a 39 amino acid peptide disrupted by the original transposon hit during the mutant screen (Hidalgo-Grass et al., 2002). The promoter of *silC* has a *com* box, the conserved sequence found in promoter regions of genes involved in competence establishment in other streptococci (Morrison and Lee, 2000). The *silCR* gene encodes a 41 amino acid pro-peptide with

a glycine-glycine sequence motif, thought to direct cleavage in a manner consistent with Gly-Gly bacteriocin processing, which allows cleavage into a 17 amino acid active mature pheromone (Table 1). In strains carrying a functional copy of *silCR*, basal pheromone production in standard culture conditions can be quantified by immunoblotting methods and measured in supernatants using fluorescence reporter assays, with a concentration of approximately 24.5 nM being produced by the IB7 GAS isolate (Belotserkovsky et al., 2009). The multiple SilCR-regulated promoters in the *sil* locus are induced to various levels and the amount of synthetic pheromone required to trigger their expression ranges from low to high nanomolar concentrations (Eran et al., 2007; Belotserkovsky et al., 2009). The *silDE* and *silCR* genes are transcribed in a single mRNA whose expression increases in response to synthetic SilCR pheromone. Detection of SilCR is believed to occur through direct interaction of the peptide with the SilB TCS receptor which generates a corresponding phospho-transfer to the SilA and consequent change in DNA expression. The *silAB* promoter on the other hand, shows a lower degree of response to the peptide pheromone, but increases in response to H₂O₂, and is upregulated upon inoculation of the host. This suggests, that much like production of other pheromones in GAS, environmental conditions or host factors may be key to triggering their expression (Eran et al., 2007; Belotserkovsky et al., 2009; Kizy and Neely, 2009).

SilA is the predicted TCS response regulator and a member of the AlgR/AgrA/LytR family of transcriptional regulators that bind DNA as dimers via a LytTR-type domain (Nikolskaya and Galperin, 2002). SilA was shown to be necessary together with SilB, the putative sensor histidine kinase, to trigger expression of genes in response to SilCR (Belotserkovsky et al., 2009). The binding sequence for SilA was mapped in two SilCR responding promoters, revealing the presence of two direct-repeats of 9 bp that are separated by an 11 bp spacer, both of which are required for SilA activity. As demonstrated by bioinformatic analysis, most of the genes that possess putative SilA binding sites are present in the genomic island surrounding the *sil* core genes (Figure 3B). These genes increase their expression in a rapid manner after

![FIGURE 3 | Sil signaling system. (A) Model of signaling. SilCR pre-peptide is produced, secreted and processed. Mature SilCR is detected by SilB, activating the response regulator SilA which activates expression from select promoters, including the promoter for the expression of *silED-CR* genes. Expression of the *sil* locus is also in a unknown manner by a second two-component system, TrxSR. Asparagine sensing by TrxS alleviates repression of target genes by TrxR, generating an increase in expression of SilCR dependent promoters. (B) *sil* genomic island. The *sil* locus plus neighboring genes are located in a putative genomic island in the strain JS95. Chromosomal location is compared with the SF370 strain that does not possess the *sil* locus.](image.png)

Frontiers in Cellular and Infection Microbiology www.frontiersin.org September 2014 | Volume 4 | Article 127 | 8SilCR pheromone addition, and several correspond to predicted bacteriocins and/or bacteriocin maturation components, while the rest of them correspond to transposons or insertion sequences (Belotserkovsky et al., 2009). In *Streptococcus pneumoniae* the Blp bacteriocins were shown to mediate intra- and interspecies competition in cultured conditions, and also provide a competitive advantage during *in vivo* colonization experiments (Dawid et al., 2007). It is unknown whether the Blp homologs found in the *sil* island are able to produce a mature, active bacteriocin, or if they can generate immunity against Blp bacteriocins of *S. pneumoniae*, which have been shown to kill GAS *in vitro* (Lux et al., 2007). Interestingly, addition of SilCR to a culture’s growth medium also increases the expression of several genes that lack SilA binding motifs. As shown by microarray experiments, the early response to SilCR addition consists mainly of genes in the *sil* island. However, 3 h after pheromone addition other genes show a moderate but significant response. Some of these genes include *ptsABCD*, a putative mannose/fructose-specific phosphotransferase system, purine synthesis related genes *purEK* and even the *rgg2* transcriptional regulator, suggesting a possible link in the activation process of these two QS systems (Belotserkovsky et al., 2009). Strains that do not possess the core *sil* signaling genes have also been shown to exhibit differential gene expression in response to SilCR pheromone, in some cases with opposite regulatory effects than those seen in *sil* harboring strains (Salim et al., 2008). If any of the additional TCS of GAS could be also sensing SilCR has yet to be determined.

As mentioned earlier, only a fraction of GAS isolates carry the *sil* locus. Interestingly, among the strains that possess Sil, some of them also carry point mutations in components of the system. The pheromone-coding gene, *silCR*, often has a nonsense mutation that changes the ATG start codon for ATA, rendering the strain unable to produce its own peptide, yet is able to respond to exogenous peptide (i.e., strains JS95, HSC5). Other mutations reported are located in the *silD* component of the ABC transporter, where stop codon mutations truncate or divide the gene (strains MGAS8232, MGAS10750) (Michael-Gayego et al., 2013). The strains carrying these *silD* mutations should be able to respond to peptide, but won’t be able to secrete pheromones to generate an autoinduction process. Presence of non-sense mutations in signaling pathways appears to be a recurring theme among GAS QS systems, and also can be seen in the Com and the lantibiotic regulatory systems of GAS (Upton et al., 2001; Wescombe and Tagg, 2003; Mashburn-Warren et al., 2012; Wescombe et al., 2012). Selective pressures leading to such lesions remains unclear.

The clearest understanding of behavior regulated by the Sil system, and the reason for its discovery, is its effect on pathogenicity. After the first study which identified *silC* mutants with deficient invasion of the spleen, *silB* and *silC* mutants were also isolated in an independent signature-tagged mutagenesis screen looking for loss of virulence of the strain HSC5 in a zebra fish model of necrotizing fasciitis (Kizy and Neely, 2009). Interestingly, these screens have been performed in strains possessing the *sil* locus, but contain the sequence ATA in place of a functional start codon in *silCR*, impairing pheromone production. Thus, Sil seems to provide a pathogenic advantage *in vivo* in the absence of the pheromone,

and further evidence suggests SilCR pheromone actually represses pathogenicity. Normally, GAS exhibits the ability to degrade the chemokine IL-8 of human origin, and MIP2 and KC of murine origin, by the activity of the ScpC/SpyCEP serine peptidase. Chemokine degradation impairs the process of immune recruitment, resulting in the absence of infiltrating neutrophils in infected mouse tissue, thus inhibiting bacterial clearance from infected sites. Preincubation of the strain JS95 with the SilCR pheromone inhibits the chemokine-degrading activity by down-regulating the expression of ScpC in a SilA-dependent SilC-independent manner, and preincubation of bacteria with SilCR reduces skin lesion area and inhibits killing by systemic infection (Hidalgo-Grass et al., 2004, 2006; Eran et al., 2007). For this reason it has been proposed that the Sil system may play a role in controlling invasive disease, and that strains possessing Sil, but have mutations impairing SilCR expression, may exhibit increased virulence phenotypes. The mechanism of ScpC down-regulation has not been elucidated thus far, and its promoter lacks any predicted SilA binding sites. Even though its deletion affects pathogenicity, the exact function and mechanism of SilC remains cryptic. Originally, it was shown that the expression of SilC *in trans* from a plasmid was able to repress SilCR activated promoters (Hidalgo-Grass et al., 2002; Eran et al., 2007), but so far its ability to repress other genes that may affect GAS infection and lifestyle in the host has not been demonstrated. The therapeutic effect of SilCR nonetheless seems to be variable, and may depend on the presence of *sil* genes, emm type and isolate used (Salim et al., 2008), and repression of *scpC* expression by additional regulators like CovRS (Sumby et al., 2008). Recently, it was shown that signaling by Sil is activated *in vivo* during early time-points post infection in a mouse model, and that this activation phenotype and its kinetics can be replicated in culture during infection of cell-culture lines (Baruch et al., 2014). It was shown that activation of the *sil* locus was dependent on the sensing of the amino acid asparagine by the TrxRS TCS of GAS, subsequent to secretion of streptolysin toxins that induce asparagine release by mammalian cells.

Finally, isolates of *Streptococcus dysgalactiae subsp.* equisimilis, known as Group G streptococcus (GGS), also possess the *sil* locus. Prevalence of Sil in invasive isolates of GGS is higher than in GAS isolates, and all analyzed strains had a functional copy of *silCR*, although some of them also carried truncations in the *silD* gene (Belotserkovsky et al., 2009; Michael-Gayego et al., 2013). Both GAS and GGS generated a transcriptional response when incubated with SilCR pheromone produced by one another, supporting the hypothesis that interspecies signaling between these two Streptococci is possible.

## LANTIBIOTIC REGULATORY SYSTEMS

During the process of colonization and establishment of their niche, bacteria face constant competition for nutrients from other individuals. For this reason, several bacteria have devised systems to give them a competitive advantage against other bacterial organisms. One such way is the production of bacteriocins, ribosomally synthesized antimicrobial peptides (AMP) that can target bacteria in the same species or across genera, with producing bacteria expressing specific immunity proteins to protect themselvesfrom their cognate peptide. These antimicrobial molecules can act through a variety of mechanisms including membrane pore formation, cell wall synthesis inhibition, and target enzyme inhibition (Quadri, 2002; Cotter et al., 2005). Class I bacteriocins, termed lantibiotics, are post-translationally modified peptides that are produced by Gram-positive bacteria, including *Bacillus*, *Staphylococcus*, *Lactococcus* and *Streptococcus* species (Chatterjee et al., 2005). These molecules are synthesized as inactive pro-peptides and then modified through amino acid dehydration and/or thioether bridge formation to generate unusual amino acid residues, and are cleaved during the secretion process to generate a mature active lantibiotic (Kleerebezem, 2004; Chatterjee et al., 2005). The lantibiotic acts not only as an AMP to neighboring bacteria, but also has pheromone properties, as lantibiotic-producing bacteria can detect their own bacteriocin through a TCS. For these reasons, lantibiotic production in several bacteria exhibits a cell-density dependent pattern and is regulated by quorum-sensing like circuits (Kleerebezem, 2004). The machinery required for lantibiotic production is usually encoded in gene clusters of conserved architecture formed by two or more operonic units in which genes are grouped by their function. A model of a prototypical lantibiotic producing system and its mechanism is shown in Figure 4A: The inactive product of the pre-peptide gene is post-translationally modified, cleaved, and exported to give rise to the active lantibiotic, which can exert is antimicrobial activity toward sensitive individuals. The mature lantibiotic is also sensed by the TCS, which signals to generate an autoinduction process and increase the production of lantibiotic as well as activates the production of lantibiotic in its siblings. In some cases lantibiotic biosynthetic operons are shared amongst different species, meaning that a bacteriocin can act both as an inter and intraspecies signaling molecule. To remain resistant to the activity of their lantibiotic, bacteria can express membrane

bound immunity peptides that bind the bacteriocin and/or ABC transporters that prevent the accumulation of lantibiotics in the bacterial surface (Gebhard, 2012).

Certain GAS isolates have been reported to produce different lantibiotic molecules that can inhibit growth in other streptococcal species, or in sensitive GAS isolates (Tagg et al., 1973; Simpson and Tagg, 1983; Tagg and Skjold, 1984; Hynes and Tagg, 1985; Karaya et al., 2001; Upton et al., 2001). The ability to produce mature bacteriocins is not widespread amongst all GAS isolates, and in most cases conserved mutations have occurred in biosynthetic loci. This has led authors to hypothesize that some of these circuits may have lost their lantibiotic and QS ability and instead have adapted them to perform other functions (Phelps and Neely, 2007; Namprachan-Frantz et al., 2014). One lantibiotic locus present in GAS is orthologous to Salivaracin A (SalA) synthesis gene clusters. First identified and purified form cultures of *Streptococcus salivarius* strain 20P3, salivaracin A is the result of the processed product of the *salA* gene, a 51-amino acid propeptide with a Gly-Gly motif, which is modified and cleaved to generate a mature 22-amino acid lantibiotic and autoinducer pheromone (Ross et al., 1993; Upton et al., 2001). Different *sal* alleles have been described where variations in the sequence of the *salA* gene give rise to five additional variants of SalA termed SalA1 to SalA5 (Wescombe et al., 2006). The *S. salivarius salAMTXYKR* locus contains all the genes predicted to be required for the processing (*salM*, previously mistakenly annotated as *salBC*), export (*salT*), immunity (*salXY*), and sensing of the peptide (*salKR*) by the SalKR TCS (Ross et al., 1993; Wescombe et al., 2006). Salivaracin A from *S. salivarius* was shown to inhibit growth of a wide variety of GAS strains in culture. Surprisingly, the majority of analyzed GAS isolates were also shown to possess a *salA1* gene in their genome, while only a couple of M4 isolates were able to produce the active lantibiotic (Simpson et al., 1995; Upton

![Figure 4](#fig4)

**FIGURE 4 | Lantibiotic regulatory systems.** (A) Model of production and sensing. Lantibiotic pre-peptide is synthesized, exported, cleaved, and modified through thiosulfur bridge formation and aminoacid dehydration. The mature lantibiotic can exert its antibacterial effect on sensitive bacteria either by targeting the cytoplasmic membrane or inhibiting activity of target enzymes. Lantibiotic-producing bacteria express membrane immunity proteins to bind their cognate lantibiotic, or to re-export via an ABC

transporter. Two-component systems sense the lantibiotic and trigger activation of the lantibiotic gene cluster in producing and neighboring bacteria. (B) Lantibiotic gene clusters present in *Streptococcus pyogenes*. Complete or partial components of lantibiotic genes in sequenced strains of GAS compared with reference strains (in bold) for *sal* cluster (*S. salivarius* 20P3, accession AY005472), and *srt* cluster (*S. pyogenes* BL-T, accession AB030831).
et al., 2001; Wescombe et al., 2006). Analysis of genomes of these isolates revealed that most of them carried deletions in the *sal* cluster affecting the genes required for maturation and secretion of SalA (Wescombe et al., 2006), while mostly maintaining intact sequences of the downstream genes required for immunity and sensing, a phenomenon that seems to be widespread as shown also in several sequenced strains (Figure 4B). Spent cultures of a the GAS T11 strain, that is able to produce active SalA1, induces expression of *salA* mRNA in *S.* *salivarius* 20P3, as well as inducing low levels of expression of *salA1* mRNA in the M1 SF370 strain, which has an incomplete *sal* locus, meaning that this strain is still able to sense and generate a transcriptional response to the lantibiotic peptide. Nonetheless, SF370 remains sensitive to the bacteriostatic effects of SalA, although it carries the genes for its putative immunity, suggesting that the ability to generate an active lantibiotic and the process of self-signaling and autoinduction are required for efficient establishment of immunity. Reinforcing this idea, mutating the SalA maturation genes in the immune strain T11 renders it sensitive to SalA. Some GAS strains possess all the genes of the *sal* cluster (MGAS10750, MGAS315, HSC5, SSI-1), although point mutations in maturation and transport genes may still render these strains unable to produce the active lantibiotic and are sensitive to its activity (Wescombe et al., 2006; Phelps and Neely, 2007; Namprachan-Frantz et al., 2014). Evidence has suggested however that the *sal* locus may still play a role during GAS infection in the host. Analysis of GAS transposon mutants in a zebrafish model of infection detected two attenuated mutants of the HSC5 strain with insertions in *salY* and *salK* genes. Further inspection showed that the *salY* mutant exhibited decreased bacterial loads after zebrafish infection when compared with is WT counterpart, and the *salY* mutant was also attenuated for survival inside macrophages *in vitro*. The *in vivo* survival defect could be rescued if zebrafish were depleted of macrophages prior to infection (Phelps and Neely, 2007). SalY forms part of the putative ABC transporter involved in immunity to the SalA lantibiotic. When compared with the *salY* gene from *S.* *salivarius*, all GAS strains possess conserved nucleotide changes that generate non-similar changes in several amino acids of the SalY transporter. Therefore, it has been hypothesized that GAS SalY may have changed and adapted to bind and/or export other substrates. It has also been shown that *salA* expression is upregulated during murine muscle tissue infection, and recently it also was shown that human serum increases the expression of the promoter upstream of the *salRK* TCS genes, providing a possible link between host factors and activation of the *sal* locus in GAS (Loughman and Caparon, 2006a; Namprachan-Frantz et al., 2014). In addition, data have suggested that mutations in the *salRK* TCS can affect other genes outside the *sal* cluster of genes in GAS and other streptococci (Li et al., 2008; Le Breton et al., 2013). For these reasons, it was hypothesized that while the antimicrobial effect of SalA1 was lost in GAS, it may still be involved in signaling processes important for host colonization, and that this system may have evolved to perform a different function.

Another lantibiotic isolated from GAS cultures is Streptin (Karaya et al., 2001; Wescombe and Tagg, 2003). Similar to salivaricin A and other lantibiotics, all the required genes are present in one cluster, formed by the *srtIRKATCBFEG* genes predicted to be involved in streptin binding and immunity (*srtI*), detection and regulation by TCS (*srtRK*), pre-peptide (*srtA*), transport (*srtT*), modification (*srtCB*), and additional immunity through an ABC-transporter (*srtFEG*). The *srtA* gene encodes for the Streptin precursor peptide, a 46 amino acid peptide that is modified and cleaved to generate three variants of Streptin (Table 1) (Wescombe and Tagg, 2003). Levels of *srtA* mRNA are undetected at early culture time points and increases slightly at later stages of culture, but can be induced earlier if a streptin extract is added to the culture, reflecting the autoinducing ability of the lantibiotic. Similarly, potent streptin production is only obtained in two-phase cultures, where a lawn of solid media-grown bacteria is transferred to liquid culture. Interestingly, presence of SpeB has been shown to be required to produce active Streptin and to induce effective immunity in the producer strains, suggesting that this protease may be involved in the processing of the mature lantibiotic (Hynes and Tagg, 1986). Though the *srtA* gene is found in several isolates, only a few of these are able to produce the biologically active lantibiotic, as is seen with SalA. In some cases lack of production is apparently due to the lack of genes required for transport and modification, or in other cases where the *srt* locus appears intact and autoinduction does not occur due to probable lack of promoter activity, suggesting that additional elements are involved in controlling expression of the lantibiotic genes (Wescombe and Tagg, 2003). One group has shown that samples of GAS present in throat infection patients exhibit a high expression of the *srtEGI* immunity genes (Livezey et al., 2011), suggesting a requirement of host factors for *srt* gene expression and a possible role for streptin immunity-related ABC-transporters in infection of the host and establishment of GAS in its niche.

Finally, Streptococcins are another type of lantibiotic shown to be produced by GAS. There have been two types described to date, class I type streptococcins (chromosomally encoded Streptococcin A-FF22 and Streptococcin A-M49, named so for their strains of isolation) and class III type streptococcins (plasmid encoded Streptococcin A-M57) (Tagg et al., 1973; Simpson and Tagg, 1983; Tagg and Skjold, 1984; Jack and Tagg, 1992; Hynes et al., 1994; Jack et al., 1994b; Heng et al., 2004; Tagg, 2009). The most studied of these is Streptococcin A-FF22 (SA-FF22) a cationic bactericidal peptide which causes membrane potential disruption in target cells (Jack et al., 1994a). Similar to the *sal* and *srt* loci, the gene cluster involved in SA-FF22 is comprised of genes involved in production of the pre-peptides (*scnA* and *scnA'*) regulation (*scnRK*), modification and transport (*scnMT*), and immunity to the lantibiotic (*scnFEG*) (McLaughlin et al., 1999). The FF22 strain is able to produce an active Streptococcin that is able to inhibit growth in sensitive strains, while generating autoinduction in both GAS and *S.* *salivarius* and *S.* *dysgalactiae* strains that carry the complete *scn* gene cluster. The *scnA* pre-peptide gene was found only in a small percentile of analyzed strains, and all of the sequenced strains to date completely lack the locus, having only a flanking transposase A gene (Wescombe et al., 2012), and studies testing the streptococcin involvement in the GAS lifestyle have not been reported.
Jimenez and Federle

**INTERSPECIES COMMUNICATION: LuxS AND AI-2**

The autoinducer-2 molecule (AI-2) was initially discovered in the Gram-negative bacterium *Vibrio harveyi*, in which it forms part of the QS-dependent regulation of luciferase production. It was later found that the LuxS enzyme was required for AI-2 production, and since the *luxS* gene is present in the genomes of a wide variety of Gram-negative and Gram-positive bacteria, has led to the proposal of AI-2 as a universal signal for interspecies cell-to-cell communication (Miller and Bassler, 2001; Xavier and Bassler, 2003). Differing from other Gram-positive QS signals, AI-2 signaling is not peptide-based. Also, unlike the biosynthetic processes of the other QS molecules of GAS, the AI-2 synthesis pathway is tightly coupled with a metabolic cycle termed activated methyl cycle (AMC), which is involved in the utilization of S-adenosylmethionine (SAM) and the degradation of its toxic derivatives. SAM is an essential donor of methyl groups for processes of DNA, RNA, fatty acid, and protein synthesis. Its utilization generates the toxic intermediate S-adenosylhomocysteine (SAH), which is then hydrolyzed by the Pfs nucleosidase to produce S-ribosylhomocysteine (SRH) and adenine. LuxS catalyzes the cleavage of SRH to homocysteine and 4,5-dihydroxy 2,3-pentanedione (DPD). DPD then spontaneously cyclizes to form pro-AI-2 molecules capable of reacting with borate to generate the signal detected by Vibrios (Figure 5A) (Schauder et al., 2001).

Two different families of receptors have been reported to sense AI-2 molecules in different bacteria (Figure 5B). The LuxPQ family, only found in *Vibrio* spp., consisting of the LuxP periplasmic protein that binds AI-2 and in consequence modulates the activity LuxQ, a transmembrane two-component histidine kinase. Binding of AI-2 switches the activity of LuxQ from kinase to phosphatase, affecting the activity of downstream DNA-binding response regulators. The other family of receptors is the LsrB family, found in *E. coli*, *Salmonella* and members of the Rhizobiaceae and Bacillaceae families (Pereira et al., 2009). LsrB binds a non-borated derivative of the AI-2 molecule, but unlike the LuxPQ signal transduction system, LsrB delivers AI-2 to the Lsr ABC transporter, which imports the molecule into the cytoplasm. The transport of AI-2 is coupled with its phosphorylation, which both serves to its sequestration inside the cell and to enhance AI-2 binding to the LsrR transcriptional repressor protein, deactivating its repressive function (Miller et al., 2004; Xavier and Bassler, 2005).

LuxS and AI-2 have been shown to influence expression of virulence factors, motility, and biofilm formation in several bacteria, including Streptococcal species (Reviewed by Pereira et al., 2013). In GAS, LuxS is required for AI-2 synthesis, and *luxS* deletion resulted in increased expression of the haemolysin streptolysin S and Emm protein, decreased production of hyaluronic acid capsule and diminished activity of the SpeB protease due to abnormal processing (Lyon et al., 2001; Marouni and Sela, 2003). These changes resulted in increased invasiveness toward HEP-2 epithelial cells. Additionally, Siller et al. showed that the deletion of the *luxS* gene resulted in increased tolerance to acidic conditions, affected the expression of *fasX* (an sRNA involved in regulation of virulence factor expression), (Kreikemeyer et al., 2001) and upregulated expression of *sibA*, a gene encoding a secreted immunoglobulin binding protein (Siller et al., 2008).

---

FIGURE 5 | AI-2 production and sensing. (A) Reactive methyl cycle and production of DPD. SAM-dependent methyltransferases convert SAM to SAH, accumulation of which confers product-feedback inhibition on methyltransferase reactions. SAH is detoxified to SRH by Pfs. SRH is converted to homocysteine and DPD by LuxS. Homocysteine can be recycled to SAM via MetH, which generates methionine, and MetK. DPD spontaneously cyclizes and undergoes ketone hydration to form R- and S-THMF. The latter is also found in its borated-form, S-THMF-borate (reproduced and modified with permission from Federle, 2009) (B) The two families of known AI-2 receptors, LuxPQ, and LsrB, which sense extracellular AI-2 or import AI-2 to bind LsrR, respectively. LuxPQ binds the S-THMF-borate form of AI-2, while LsrB binds the R-THMF form.

One of the central debates in the topic of LuxS function is how to differentiate its QS effects from its metabolic role. Inactivation of *luxS* not only inhibits AI-2 formation, but it also could result in gene expression changes generated by defective recycling of homocysteine or accumulation of intermediates of SAM metabolism. Recently, Redanz et al. have shown that in *Streptococcus sanguinis* close to 96% of the genes that exhibit a change in expression in the Δ*luxS* mutant are due to accumulation of homocysteine in the cell (Redanz et al., 2012). Additionally, genomic analysis has revealed that several species possessing the *luxS* gene lack any homologs to the AI-2 receptors LuxPQ or Lsr, including the Streptococci (Rezzonico and Duffy, 2008). Thus, it has been

proposed that to clearly establish the role and the effects of the AI-2 system, the ΔluxS mutant phenotypes should be complemented by the addition of synthetic DPD to generate AI-2 in the culture medium (Pereira et al., 2013). No group has yet demonstrated directly the effects of exogenous AI-2 on GAS gene expression, so the nature of the LuxS-promoted phenotypes is unknown. Even though Siller et al. have suggested that luxS has a limited role in the AMC cycle in GAS (Siller et al., 2008), the overall effect of this deletion over the bacterial metabolism is unknown. Nonetheless, AI-2 has been shown to directly influence biofilm formation in other oral streptococci like *S. intermedius* and *S. gordonii* (Ahmed et al., 2008; Cuadra-Saenz et al., 2012), meaning that these bacteria may have an as yet uncharacterized AI-2 sensing machinery, and that even if GAS does not directly respond to its own AI-2, could influence the behaviors of related species in its surroundings.

### CONCLUDING REMARKS AND PERSPECTIVES

The *Streptococcus pyogenes* pan-genome harbors several varieties of QS circuits, and GAS has the ability to produce an assortment of dedicated signaling molecules that affect expression of several target genes. While there is increasing information regarding the mechanics of these QS networks, our current understanding of these systems is still in its infancy, and despite data regarding regulated genes and observed phenotypes resulting of QS signaling, we do not fully comprehend yet when and where these communication systems are triggered *in vivo*, and how GAS benefits from these processes of population-wide coordination of gene expression. Apart from the Sil system, the direct contribution of QS to GAS pathogenesis is unclear. However, its putative involvement in the control of DNA exchange (ComR) and in the sensing and production of antimicrobial substances (SalA and Srt) suggest that QS may play roles involved in the less understood process of GAS interaction with other bacteria in its milieu. Recently, Marks et al. have shown that GAS growing inside biofilms exhibit lower expression of genes involved in localized and invasive-disease, and bacteria originating from biofilm cultures had reduced virulence in a septic infection model, while showing an increased ability to asymptomatically colonize nasal associated lymphoid tissue of mice, when compared with planktonic cultured bacteria (Marks et al., 2014). There is evidence that both Rgg2/3 and Sil QS systems can affect biofilm formation (Lembke et al., 2006; Chang et al., 2011; Cook et al., 2013), suggesting a possible link between QS and the asymptomatic, commensal lifestyle of GAS inside the human host. Future efforts in these areas may be able to reveal the role of QS signaling in GAS population behavior.

With the increasing emergence of antibiotic resistance amongst infectious bacteria, there is a dire need for alternative therapeutic strategies to control pathogens. One of those strategies is the control of virulence gene expression, mainly through the manipulation of QS in a process of quorum-sensing inhibition (also referred to as “quorum quenching,” reviewed by LaSarre and Federle, 2013), with successful results of QS-inhibition during *in vivo* infection models with pathogens *Vibrio cholerae* and *Staphylococcus aureus* (Mayville et al., 1999; Wright et al., 2005; Duan and March, 2010). We think the multiple QS systems of GAS offer interesting molecular targets to block or interfere in

order to modulate the behavior of this pathogen as a way of future treatment.

### ACKNOWLEDGMENTS

Support for this work was provided by CONICYT Becas-Chile Scholarship, NIH grant AI091779, and the Burroughs Wellcome Fund Investigators of Infectious Diseases.

### REFERENCES

Aggarwal, C., Jimenez, J. C., Nanavati, D., and Federle, M. J. (2014). Multiple length peptide-pheromone variants produced by *Streptococcus pyogenes* directly bind Rgg proteins to confer transcriptional regulation. *J. Biol. Chem.* 289, 22427–22436. doi: 10.1074/jbc.M114.583989

Ahmed, N. A. A. M., Petersen, F. C., and Scheie, A. (2008). Biofilm formation and autoinducer-2 signaling in *Streptococcus intermedius*: role of thermal and pH factors. *Oral Microbiol. Immunol.* 23, 492–497. doi: 10.1111/j.1399-302X.2008.00460.x

An, F. Y., Sulavik, M. C., and Clewell, D. B. (1999). Identification and characterization of a determinant (eep) on the *Enterococcus faecalis* chromosome that is involved in production of the peptide sex pheromone. *J. Bacteriol.* 181, 5915–5921.

Anbalagan, S., and Chaussee, M. S. (2013). Transcriptional regulation of a bacteriophage encoded extracellular DNase (Spd-3) by Rgg in *Streptococcus pyogenes*. *PLoS ONE* 8:e61312. doi: 10.1371/journal.pone.0061312

Anbalagan, S., Dmitriev, A., McShan, W. M., Dunman, P. M., and Chaussee, M. S. (2012). Growth phase-dependent modulation of Rgg binding specificity in *Streptococcus pyogenes*. *J. Bacteriol.* 194, 3961–3971. doi: 10.1128/JB.06709-11

Anbalagan, S., McShan, W. M., Dunman, P. M., and Chaussee, M. S. (2011). Identification of Rgg binding sites in the *Streptococcus pyogenes* chromosome. *J. Bacteriol.* 193, 4933–4942. doi: 10.1128/JB.00429-11

Atkinson, S., and Williams, P. (2009). Quorum sensing and social networking in the microbial world. *J. R. Soc. Interface* 6, 959–978. doi: 10.1098/rsif.2009.0203

Baruch, M., Belotserkovsky, I., Hertzog, B. B., Ravins, M., Dov, E., McIver, K. S., et al. (2014). An extracellular bacterial pathogen modulates host metabolism to regulate its own sensing and proliferation. *Cell* 156, 97–108. doi: 10.1016/j.cell.2013.12.007

Belotserkovsky, I., Baruch, M., Peer, A., Dov, E., Ravins, M., Mishalian, I., et al. (2009). Functional analysis of the quorum-sensing streptococcal invasion locus (sil). *PLoS Pathog.* 5:e1000651. doi: 10.1371/journal.ppat.1000651

Beres, S. B., Carroll, R. K., Shea, P. R., Sitkiewicz, I., Martinez-Gutierrez, J. C., Low, D. E., et al. (2010). Molecular complexity of successive bacterial epidemics deconvoluted by comparative pathogenomics. *Proc. Natl. Acad. Sci. U.S.A.* 107, 4371–4376. doi: 10.1073/pnas.0911295107

Bidet, P., Courroux, C., Salgueiro, C., Carol, A., Mariani-Kurkdjian, P., Bonacorsi, S., et al. (2007). Molecular epidemiology of the sil streptococcal invasive locus in group A streptococci causing invasive infections in French children. *J. Clin. Microbiol.* 45, 2002–2004. doi: 10.1128/JCM.00104-07

Billal, D. S., Hotomi, M., Shimada, J., Fujihara, K., Ubukata, K., Sugita, R., et al. (2008). Prevalence of *Streptococcus* invasive locus (sil) and its relationship with macrolide resistance among group A Streptococcus strains. *J. Clin. Microbiol.* 46, 1563–1564. doi: 10.1128/JCM.02410-07

Biswas, I., and Scott, J. R. J. (2003). Identification of rocA, a positive regulator of covR expression in the group A streptococcus. *J. Bacteriol.* 185, 3081–3090. doi: 10.1128/JB.185.10.3081-3090.2003

Blatch, G. L., and Lässle, M. (1999). The tetratricopeptide repeat: a structural motif mediating protein-protein interactions. *Bioessays* 21, 932–939.

Bortoni, M. E., Terra, V. S., Hinds, J., Andrew, P. W., and Yesilkaya, H. (2009). The pneumococcal response to oxidative stress includes a role for Rgg. *Microbiology* 155, 4123–4134. doi: 10.1099/mic.0.028282-0

Browning, D. F., and Busby, S. J. (2004). The regulation of bacterial transcription initiation. *Nat. Rev. Microbiol.* 2, 57–65. doi: 10.1038/nrmicro787

Carroll, R. K., and Musser, J. M. (2011). From transcription to activation: how group A streptococcus, the flesh-eating pathogen, regulates SpeB cysteine protease production. *Mol. Microbiol.* 81, 588–601. doi: 10.1111/j.1365-2958.2011.07709.x

Carroll, R. K., Shelburne, S. A., Olsen, R. J., Suber, B., Sahasrabhojane, P., Kumaraswami, M., et al. (2011). Naturally occurring single amino acid replacements in a regulatory protein alter streptococcal gene expression and virulence in mice. *J. Clin. Invest.* 121, 1956–1968. doi: 10.1172/JCI 45169

Chang, J. C., LaSarre, B., Jimenez, J. C., Aggarwal, C., and Federle, M. J. (2011). Two group A streptococcal peptide pheromones act through opposing Rgg regulators to control biofilm development. *PLoS Pathog.* 7:e1002190. doi: 10.1371/journal.ppat.1002190

Chatterjee, C., Paul, M., Xie, L., and van der Donk, W. A. (2005). Biosynthesis and mode of action of lantibiotics. *Chem. Rev.* 105, 633–684. doi: 10.1021/cr03 0105v

Chaussee, M. A., Callegari, E. A., and Chaussee, M. S. (2004). Rgg regulates growth phase-dependent expression of proteins associated with secondary metabolism and stress in *Streptococcus pyogenes*. *J. Bacteriol.* 186, 7091. doi: 10.1128/JB.186.21.7091-7099.2004

Chaussee, M. A., Dmitriev, A. V., Callegari, E. A., and Chaussee, M. S. (2008). Growth phase-associated changes in the transcriptome and proteome of *Streptococcus pyogenes*. *Arch. Microbiol.* 189, 27–41. doi: 10.1007/s00203-007- 0290-1

Chaussee, M. S., Somerville, G. A., Reitzer, L., and Musser, J. M. (2003). Rgg coordinates virulence factor synthesis and metabolism in *Streptococcus pyogenes*. *J. Bacteriol.* 185, 6016–6024. doi: 10.1128/JB.185.20.6016-6024.2003

Chaussee, M. S., Sylva, G. L. G., Sturdevant, D. D. E., Smoot, L. M., Graham, M. R., Watson, R. O., et al. (2002). Rgg influences the expression of multiple regulatory loci to coregulate virulence factor expression in *Streptococcus pyogenes*. *Infect. Immun.* 70, 762–770. doi: 10.1128/IAI.70.2.762-770.2002

Chiang-Ni, C., and Wu, J.-J. (2008). Effects of streptococcal pyrogenic exotoxin B on pathogenesis of *Streptococcus pyogenes*. *J. Formos. Med. Assoc.* 107, 677–685. doi: 10.1016/S0929-6646(08)60112-6

Cole, J. N., Aziz, R. K., Kuipers, K., Timmer, A. M., Nizet, V., van Sorge, N. M., et al. (2012). A conserved UDP-glucose dehydrogenase encoded outside the hasABC operon contributes to capsule biogenesis in group A Streptococcus. *J. Bacteriol.* 194, 6154–6161. doi: 10.1128/JB.01317-12

Cook, L. C., and Federle, M. J. (2014). Peptide pheromone signaling in Streptococcus and Enterococcus. *FEMS Microbiol. Rev.* 38, 473–492. doi: 10.1111/1574-6976.12046

Cook, L. C., LaSarre, B., and Federle, M. J. (2013). Interspecies communication among commensal and pathogenic streptococci. *MBio* 4:e00382-13. doi: 10.1128/mBio.00382-13

Core, L., and Perego, M. (2003). TPR-mediated interaction of RapC with ComA inhibits response regulator-DNA binding for competence development in *Bacillus subtilis*. *Mol. Microbiol.* 49, 1509–1522. doi: 10.1046/j.1365- 2958.2003.03659.x

Cotter, P., Hill, C., and Ross, R. (2005). Bacteriocins: developing innate immunity for food. *Nat. Rev. Microbiol.* 3, 777–788. doi: 10.1038/nrmicro1273

Cuadra-Saenz, G., Rao, D. L., Underwood, A. J., Belapure, S. A., Campagna, S. R., Sun, Z., et al. (2012). Autoinducer-2 influences interactions amongst pioneer colonizing streptococci in oral biofilms. *Microbiology* 158, 1783–1795. doi: 10.1099/mic.0.057182-0

Dawid, S., Roche, A. M., and Weiser, J. N. (2007). The blp bacteriocins of *Streptococcus pneumoniae* mediate intraspecies competition both *in vitro* and *in vivo*. *Infect. Immun.* 75, 443–451. doi: 10.1128/IAI.01775-05

Declerck, N., Bouillaut, L., Chaix, D., Rugani, N., Slamti, L., Hoh, F., et al. (2007). Structure of PlcR: insights into virulence regulation and evolution of quorum sensing in Gram-positive bacteria. *Proc. Natl. Acad. Sci. U.S.A.* 104, 18490–18495. doi: 10.1073/pnas.0704501104

De Saizieu, A., Gardès, C., Flint, N., Wagner, C., Kamber, M., Mitchell, T. J., et al. (2000). Microarray-based identification of a novel *Streptococcus pneumoniae* regulon controlled by an autoinduced peptide. *J. Bacteriol.* 182, 4696–4703. doi: 10.1128/JB.182.17.4696-4703.2000

Dmitriev, A. V., McDowell, E. J., and Chaussee, M. S. (2008). Inter- and intraserotypic variation in the *Streptococcus pyogenes* Rgg regulon. *FEMS Microbiol. Lett.* 284, 43–51. doi: 10.1111/j.1574-6968.2008.01171.x

Dmitriev, A. V., McDowell, E. J., Kappler, K. V., Chaussee, M. A., Rieck, L. D., and Chaussee, M. S. (2006). The Rgg regulator of *Streptococcus pyogenes* influences utilization of nonglucose carbohydrates, prophage induction, and expression of the NAD-glycohydrolase virulence operon. *J. Bacteriol.* 188, 7230–7241. doi: 10.1128/JB.00877-06

Duan, F., and March, J. C. (2010). Engineered bacterial communication prevents *Vibrio cholerae* virulence in an infant mouse model. *Proc. Natl. Acad. Sci. U.S.A.* 107, 11260–11264. doi: 10.1073/pnas.1001294107

Eran, Y., Getter, Y., Baruch, M., Belotserkovsky, I., Padalon, G., Mishalian, I., et al. (2007). Transcriptional regulation of the sil locus by the SilCR signaling peptide and its implications on group A streptococcus virulence. *Mol. Microbiol.* 63, 1209–1222. doi: 10.1111/j.1365-2958.2007.05581.x

Federle, M. J. (2009). Autoinducer-2-based chemical communication in bacteria: complexities of interspecies signaling. *Contrib. Microbiol.* 16, 18–32. doi: 10.1159/000219371

Federle, M. J. (2012). Pathogenic streptococci speak, but what are they saying? *Virulence* 3, 92–94. doi: 10.4161/viru.3.1.18652

Fernandez, A., Borges, F., Gintz, B., Decaris, B., and Leblond-Bourget, N. (2006). The rggC locus, with a frameshift mutation, is involved in oxidative stress response by *Streptococcus thermophilus*. *Arch. Microbiol.* 186, 161–169. doi: 10.1007/s00203-006-0130-8

Fleuchot, B., Gitton, C., Guillot, A., Vidic, J., Nicolas, P., Besset, C., et al. (2011). Rgg proteins associated with internalized small hydrophobic peptides: a new quorum-sensing mechanism in streptococci. *Mol. Microbiol.* 80, 1102–1119. doi: 10.1111/j.1365-2958.2011.07633.x

Fleuchot, B., Guillot, A., Mézange, C., Besset, C., Chambellon, E., Monnet, V., et al. (2013). Rgg-associated SHP signaling peptides mediate cross-talk in Streptococci. *PLoS ONE* 8:e66042. doi: 10.1371/journal.pone. 0066042

Fontaine, L., Boutry, C., de Frahan, M. H., Delplace, B., Fremaux, C., Horvath, P., et al. (2010). A novel pheromone quorum-sensing system controls the development of natural competence in *Streptococcus thermophilus* and *Streptococcus salivarius*. *J. Bacteriol.* 192, 1444–1454. doi: 10.1128/JB.01251-09

Fujiwara, T., Hoshino, T., Oshima, T., Sobue, S., and Hamada, S. (2000). Purification, characterization, and molecular analysis of the gene encoding glucosyltransferase from *Streptococcus oralis*. *Infect. Immun.* 68, 1–10. doi: 10.1128/IAI.68.5.2475-2483.2000

Gardan, R., Besset, C., Guillot, A., Gitton, C., and Monnet, V. (2009). The oligopeptide transport system is essential for the development of natural competence in *Streptococcus thermophilus* strain LMD-9. *J. Bacteriol.* 191, 4647–4655. doi: 10.1128/JB.00257-09

Gebhard, S. (2012). ABC transporters of antimicrobial peptides in Firmicutes bacteria - phylogeny, function and regulation. *Mol. Microbiol.* 86, 1295–1317. doi: 10.1111/mmi.12078

Håvarstein, L. S., Coomaraswamy, G., and Morrison, D. A. (1995). An unmodified heptadecapeptide pheromone induces competence for genetic transformation in *Streptococcus pneumoniae*. *Proc. Natl. Acad. Sci. U.S.A.* 92, 11140–11144. doi: 10.1073/pnas.92.24.11140

Heng, N. C. K., Burtenshaw, G. A., Jack, R. W., and Tagg, J. R. (2004). Sequence analysis of pDN571, a plasmid encoding novel bacteriocin production in M-type 57 *Streptococcus pyogenes*. *Plasmid* 52, 225–229. doi: 10.1016/j.plasmid.2004.08.002

Hidalgo-Grass, C., Dan-Goor, M., Maly, A., Eran, Y., Kwinn, L. A., Nizet, V., et al. (2004). Effect of a bacterial pheromone peptide on host chemokine degradation in group A streptococcal necrotising soft-tissue infections. *Lancet* 363, 696–703. doi: 10.1016/S0140-6736(04)15643-2

Hidalgo-Grass, C., Mishalian, I., Dan-Goor, M., Belotserkovsky, I., Eran, Y., Nizet, V., et al. (2006). A streptococcal protease that degrades CXC chemokines and impairs bacterial clearance from infected tissues. *EMBO J.* 25, 4628–4637. doi: 10.1038/sj.emboj.7601327

Hidalgo-Grass, C., Ravins, M., Dan-Goor, M., Jaffe, J., Moses, A. E., and Hanski, E. (2002). A locus of group A Streptococcus involved in invasive disease and DNA transfer. *Mol. Microbiol.* 46, 87–99. doi: 10.1046/j.1365-2958.2002.03127.x

Hollands, A., Aziz, R. K., Kansal, R., Kotb, M., Nizet, V., and Walker, M. J. (2008). A naturally occurring mutation in ropB suppresses SpeB expression and reduces M1T1 group A streptococcal systemic virulence. *PLoS ONE* 3:e4102. doi: 10.1371/journal.pone.0004102

Hui, F. M., and Morrison, D. A. (1991). Genetic transformation in *Streptococcus pneumoniae*: nucleotide sequence analysis shows comA, a gene required for competence induction, to be a member of the bacterial ATP-dependent transport protein family. *J. Bacteriol.* 173, 372–381.

Hynes, W. L., Friend, V. L., and Ferretti, J. J. (1994). Duplication of the lantibiotic structural gene in M-type 49 group A streptococcus strains producing streptococcin A-M49. *Appl. Environ. Microbiol.* 60, 4207–4209.

Hynes, W. L., and Tagg, J. R. (1985). Production of broad-spectrum bacteriocin-like activity by group A streptococci of particular M-types. *Zentralblatt für Bakteriol. Mikrobiol. und Hyg.* 259, 155–164.

Hynes, W. L., and Tagg, J. R. (1986). Proteinase-related broad-spectrum inhibitory activity among group-A streptococci. *J. Med. Microbiol.* 22, 257–264. doi: 10.1099/00222615-22-3-257

Ibrahim, M., Guillot, A., Wessner, F., Algaron, F., Besset, C., Courtin, P., et al. (2007a). Control of the transcription of a short gene encoding a cyclic peptide in *Streptococcus thermophilus*: a new quorum-sensing system? *J. Bacteriol.* 189, 8844–8854. doi: 10.1128/JB.01057-07

Ibrahim, M., Nicolas, P., Bessières, P., Bolotin, A., Monnet, V., and Gardan, R. (2007b). A genome-wide survey of short coding sequences in streptococci. *Microbiology* 153, 3631–3644. doi: 10.1099/mic.0.2007/006205-0

Ikebe, T., Ato, M., Matsumura, T., Hasegawa, H., Sata, T., Kobayashi, K., et al. (2010). Highly frequent mutations in negative regulators of multiple virulence genes in group A streptococcal toxic shock syndrome isolates. *PLoS Pathog.* 6:e1000832. doi: 10.1371/journal.ppat.1000832

Jack, R., Benz, R., Tagg, J., and Sahl, H. G. (1994a). The mode of action of SA-FF22, a lantibiotic isolated from *Streptococcus pyogenes* strain FF22. *Eur. J. Biochem.* 219, 699–705. doi: 10.1111/j.1432-1033.1994.tb19986.x

Jack, R. W., Carne, A., Metzger, J., Stefanoviae, S., Sahl, H. G., Jung, G., et al. (1994b). Elucidation of the structure of SA-FF22, a lanthionine-containing antibacterial peptide produced by *Streptococcus pyogenes* strain FF22. *Eur. J. Biochem.* 220, 455–462. doi: 10.1111/j.1432-1033.1994.tb18643.x

Jack, R. W., and Tagg, J. R. (1992). Factors affecting production of the group A streptococcus bacteriocin SA-FF22. *J. Med. Microbiol.* 36, 132–138. doi: 10.1099/00222615-36-2-132

Ji, G., Beavis, R. C., and Novick, R. P. (1995). Cell density control of staphylococcal virulence mediated by an octapeptide pheromone. *Proc. Natl. Acad. Sci. U.S.A.* 92, 12055–12059. doi: 10.1073/pnas.92.26.12055

Johnsborg, O., Eldholm, V., and Håvarstein, L. S. (2007). Natural genetic transformation: prevalence, mechanisms and function. *Res. Microbiol.* 158, 767–778. doi: 10.1016/j.resmic.2007.09.004

Kagawa, T. F., O’Toole, P. W., and Cooney, J. C. (2005). SpeB-Spi: a novel protease-inhibitor pair from *Streptococcus pyogenes*. *Mol. Microbiol.* 57, 650–666. doi: 10.1111/j.1365-2958.2005.04708.x

Kaplan, E. L., and Huwe, B. B. (1980). The sensitivity and specificity of an agglutination test for antibodies to streptococcal extracellular antigens: a quantitative analysis and comparison of the streptozyme test with the anti-streptolysin O and anti-deoxyribonuclease B tests. *J. Pediatr.* 96, 367–373. doi: 10.1016/S0022-3476(80)80674-3

Kappeler, K. V., Anbalagan, S., Dmitriev, A. V., McDowell, E. J., Neely, M. N., and Chaussee, M. S. (2009). A naturally occurring Rgg variant in serotype M3 *Streptococcus pyogenes* does not activate speB expression due to altered specificity of DNA binding. *Infect. Immun.* 77, 5411–5417. doi: 10.1128/IAI.00373-09

Karaya, K., Shimizu, T., and Taketo, A. (2001). New gene cluster for lantibiotic streptin possibly involved in streptolysin S formation. *J. Biochem.* 129, 769–775. doi: 10.1093/oxfordjournals.jbchem.a002918

Kizy, A. E., and Neely, M. N. (2009). First *Streptococcus pyogenes* signature-tagged mutagenesis screen identifies novel virulence determinants. *Infect. Immun.* 77, 1854–1865. doi: 10.1128/IAI.01306-08

Kleerebezem, M. (2004). Quorum sensing control of lantibiotic production; nisin and subtilin autoregulate their own biosynthesis. *Peptides* 25, 1405–1414. doi: 10.1016/j.peptides.2003.10.021

Kreikemeyer, B., Boyle, M. D., Buttaro, B. A., Heinemann, M., and Podbielski, A. (2001). Group A streptococcal growth phase-associated virulence factor regulation by a novel operon (Fas) with homologies to two-component-type regulators requires a small RNA molecule. *Mol. Microbiol.* 39, 392–406. doi: 10.1046/j.1365-2958.2001.02226.x

Kreikemeyer, B., McIver, K. S., and Podbielski, A. (2003). Virulence factor regulation and regulatory networks in *Streptococcus pyogenes* and their impact on pathogen-host interactions. *Trends Microbiol.* 11, 224–232. doi: 10.1016/S0966-842X(03)00098-2

Kuipers, O. P., Beerthuyzen, M. M., de Ruyter, P. G., Luesink, E. J., de Vos, W. M., Luesnik, E., et al. (1995). Autoregulation of nisin biosynthesis in *Lactococcus lactis* by signal transduction. *J. Biol. Chem.* 270, 27299–27304. doi: 10.1074/jbc.270.45.27299

Lanigan-Gerdes, S., Dooley, A. N., Faull, K. F., Lazazzera, B. A., and Lanigan-Gerdes, S. (2007). Identification of subtilisin, Epr and Vpr as enzymes that produce CSF, an extracellular signalling peptide of *Bacillus subtilis*. *Mol. Microbiol.* 65, 1321–1333. doi: 10.1111/j.1365-2958.2007.05869.x

LaSarre, B., Aggarwal, C., and Federle, M. J. (2012). Antagonistic Rgg regulators mediate quorum sensing via competitive DNA binding in *Streptococcus pyogenes*. *MBio* 3:e00333-12. doi: 10.1128/mBio.00333-12

LaSarre, B., Chang, J. C., and Federle, M. J. (2013). Redundant group A streptococcus signaling peptides exhibit unique activation potentials. *J. Bacteriol.* 195, 4310–4318. doi: 10.1128/JB.00684-13

LaSarre, B., and Federle, M. J. (2013). Exploiting quorum sensing to confuse bacterial pathogens. *Mol. Biol. Rev.* 77, 73–111. doi: 10.1128/MMBR.00046-12

Lazazzera, B. A., Solomon, J. M., and Grossman, A. D. (1997). An exported peptide functions intracellularly to contribute to cell density signaling in *B. subtilis*. *Cell* 89, 917–925. doi: 10.1016/S0092-8674(00)80277-9

Le Breton, Y., Mistry, P., Valdes, K. M., Quigley, J., Kumar, N., Tettelin, H., et al. (2013). Genome-wide identification of genes required for fitness of group A Streptococcus in human blood. *Infect. Immun.* 81, 862–875. doi: 10.1128/IAI.00837-12

Lee, M., and Morrison, D. (1999). Identification of a new regulator in *Streptococcus pneumoniae* linking quorum sensing to competence for genetic transformation. *J. Bacteriol.* 181, 5004–5016.

Lembke, C., Podbielski, A., Hidalgo-grass, C., Jonas, L., Hanski, E., and Kreikemeyer, B. (2006). Characterization of biofilm formation by clinically relevant serotypes of group A streptococci. *Appl. Environ. Microbiol.* 72, 2864. doi: 10.1128/AEM.72.4.2864-2875.2006

Leonard, B. A., Podbielski, A., Hedberg, P. J., and Dunny, G. M. (1996). *Enterococcus faecalis* pheromone binding protein, PrgZ, recruits a chromosomal oligopeptide permease system to import sex pheromone cCF10 for induction of conjugation. *Proc. Natl. Acad. Sci. U.S.A.* 93, 260–264. doi: 10.1073/pnas.93.1.260

Li, M., Wang, C., Feng, Y., Pan, X., Cheng, G., Wang, J., et al. (2008). SalK/SalR, a two-component signal transduction system, is essential for full virulence of highly invasive *Streptococcus suis* serotype 2. *PLoS ONE* 3:e2080. doi: 10.1371/journal.pone.0002080

Liu, X.-Y., Ruan, L.-F., Hu, Z.-F., Peng, D.-H., Cao, S.-Y., Yu, Z.-N., et al. (2010). Genome-wide screening reveals the genetic determinants of an antibiotic insecticide in *Bacillus thuringiensis*. *J. Biol. Chem.* 285, 39191–39200. doi: 10.1074/jbc.M110.148387

Livezey, J., Perez, L., Suciu, D., Yu, X., Robinson, B., Bush, D., et al. (2011). Analysis of group A Streptococcus gene expression in humans with pharyngitis using a microarray. *J. Med. Microbiol.* 60, 1725–1733. doi: 10.1099/jmm.0.022939-0

Loughman, J. A., and Caparon, M. (2006a). Regulation of SpeB in *Streptococcus pyogenes* by pH and NaCl: a model for in vivo gene expression. *J. Bacteriol.* 188, 399–408. doi: 10.1128/JB.188.2.399-408.2006

Loughman, J. A., and Caparon, M. G. (2006b). A novel adaptation of aldolase regulates virulence in *Streptococcus pyogenes*. *EMBO J.* 25, 5414–5422. doi: 10.1038/sj.emboj.7601393

Luo, P., and Morrison, D. (2003). Transient association of an alternative sigma factor, ComX, with RNA polymerase during the period of competence for genetic transformation in *Streptococcus pneumoniae*. *J. Bacteriol.* 185, 349–358. doi: 10.1128/JB.185.1.349-358.2003

Lux, T., Nuhn, M., Hakenbeck, R., and Reichmann, P. (2007). Diversity of bacteriocins and activity spectrum in *Streptococcus pneumoniae*. *J. Bacteriol.* 189, 7741–7751. doi: 10.1128/JB.00474-07

Lyon, W. R., Madden, J. C., Levin, J. C., Stein, J. L., and Caparon, M. G. (2001). Mutation of luxS affects growth and virulence factor expression in *Streptococcus pyogenes*. *Mol. Microbiol.* 42, 145–157. doi: 10.1046/j.1365-2958.2001.02616.x

Ma, Y., Bryant, A. E., Salmi, D. B., Hayes-Schroer, S. M., McIndoo, E., Aldape, M. J., et al. (2006). Identification and characterization of bicistronic speB and prsA gene expression in the group A Streptococcus. *J. Bacteriol.* 188, 7626–7634. doi: 10.1128/JB.01059-06

Ma, Y., Bryant, A. E., Salmi, D. B., McIndoo, E., and Stevens, D. L. (2009). vfr, a novel locus affecting cysteine protease production in *Streptococcus pyogenes*. *J. Bacteriol.* 191, 3189–3194. doi: 10.1128/JB.01771-08

Magnuson, R., Solomon, J., and Grossman, A. D. (1994). Biochemical and genetic characterization of a competence pheromone from *B. subtilis*. *Cell* 77, 207–216. doi: 10.1016/0092-8674(94)90313-1

Marks, L. R., Mashburn-Warren, L., Federle, M. J., and Hakansson, A. P. (2014). *Streptococcus pyogenes* biofilm growth in vitro and in vivo and its role in colonization, virulence and genetic exchange. *J. Infect. Dis.* 210, 25–34. doi: 10.1093/infdis/jiu058

Marks, L. R., Redding, R., and Hakansson, A. (2012). High levels of genetic recombination during nasopharyngeal carriage and biofilm formation in *Streptococcus pneumoniae*. *MBio* 3:e00200-12. doi: 10.1128/mBio.00200-12

Marouni, M. J., and Sela, S. (2003). The luxS gene of *Streptococcus pyogenes* regulates expression of genes that affect internalization by epithelial cells. *Infect. Immun.* 71, 5633–5639. doi: 10.1128/IAI.71.10.5633-5639.2003

Mashburn-Warren, L., Morrison, D. A., and Federle, M. J. (2010). A novel double-tryptophan peptide pheromone controls competence in Streptococcus spp. via an Rgg regulator. *Mol. Microbiol.* 78, 589–606. doi: 10.1111/j.1365-2958.2010.07361.x

Mashburn-Warren, L., Morrison, D. A., and Federle, M. J. (2012). The cryptic competence pathway in *Streptococcus pyogenes* is controlled by a peptide pheromone. *J. Bacteriol.* 194, 4589–4600. doi: 10.1128/JB.00830-12

Mayville, P., Ji, G., Beavis, R., Yang, H., Goger, M., Novick, R. P., et al. (1999). Structure-activity analysis of synthetic autoinducing thiolactone peptides from *Staphylococcus aureus* responsible for virulence. *Proc. Natl. Acad. Sci. U.S.A.* 96, 1218–1223. doi: 10.1073/pnas.96.4.1218

McLaughlin, R. E., Ferretti, J. J., and Hynes, W. L. (1999). Nucleotide sequence of the streptococin A-FF22 lantibiotic regulon: model for production of the lantibiotic SA-FF22 by strains of *Streptococcus pyogenes*. *FEMS Microbiol. Lett.* 175, 171–177. doi: 10.1111/j.1574-6968.1999.tb13616.x

Mead, P. B., and Winn, W. C. (2000). Vaginal-rectal colonization with group A streptococci in late pregnancy. *Infect. Dis. Obstet. Gynecol.* 8, 217–219. doi: 10.1155/S1064744900000302

Michael-Gayego, A., Dan-Goor, M., Jaffe, J., Hidalgo-Grass, C., and Moses, A. E. (2013). Characterization of sil in invasive group A and G streptococci: antibodies against bacterial pheromone peptide SilCR result in severe infection. *Infect. Immun.* 81, 4121–4217. doi: 10.1128/IAI.00359-13

Miller, M. B., and Bassler, B. L. (2001). Quorum sensing in bacteria. *Annu. Rev. Microbiol.* 55, 165–199. doi: 10.1146/annurev.micro.55.1.165

Miller, S. T., Xavier, K. B., Campagna, S. R., Taga, M. E., Semmelhack, M. F., Bassler, B. L., et al. (2004). Salmonella typhimurium recognizes a chemically distinct form of the bacterial quorum-sensing signal AI-2. *Mol. Cell* 15, 677–687. doi: 10.1016/j.molcel.2004.07.020

Morrison, D. A., and Lee, M. S. (2000). Regulation of competence for genetic transformation in *Streptococcus pneumoniae*: a link between quorum sensing and DNA processing genes. *Res. Microbiol.* 151, 445–451. doi: 10.1016/S0923-2508(00)00171-6

Nakayama, J., Ruhfel, R. E., Dunny, G. M., Isogai, A., and Suzuki, A. (1994). The prgQ gene of the *Enterococcus faecalis* tetracycline resistance plasmid pCF10 encodes a peptide inhibitor, iCF10. *J. Bacteriol.* 176, 7405–7408.

Namprachan-Frantz, P., Rowe, H. M., Runft, D. L., and Neely, M. N. (2014). Transcriptional analysis of the *Streptococcus pyogenes* salivaricin locus. *J. Bacteriol.* 196, 604–613. doi: 10.1128/JB.01009-13

Neely, M. N., Lyon, W. R., Runft, D. L., and Caparon, M. (2003). Role of RopB in growth phase expression of the SpeB cysteine protease of *Streptococcus pyogenes*. *J. Bacteriol.* 185, 5166–5174. doi: 10.1128/JB.185.17.5166-5174.2003

Nelson, D., Garbe, J., and Collin, M. (2011). Cysteine proteinase SpeB from *Streptococcus pyogenes*—a potent modifier of immunologically important host and bacterial proteins. *Biol. Chem.* 392, 1077–1088. doi: 10.1515/BC.2011.208

Ng, W.-L., and Bassler, B. L. (2009). Bacterial quorum-sensing network architectures. *Annu. Rev. Genet.* 43, 197–222. doi: 10.1146/annurev-genet-102108-134304

Nikolskaya, A. N., and Galperin, M. Y. (2002). A novel type of conserved DNA-binding domain in the transcriptional regulators of the AlgR/AgrA/LytR family. *Nucleic Acids Res.* 30, 2453–2459. doi: 10.1093/nar/30.11.2453

Olsen, R. J., Laucirica, D. R., Watkins, M. E., Fease, M. L., Garcia-Bustillos, J. R., Vu, C., et al. (2012). Polymorphisms in regulator of protease B (RopB) alter disease phenotype and strain virulence of serotype M3 group A Streptococcus. *J. Infect. Dis.* 205, 1719–1729. doi: 10.1093/infdis/jir825

Otto, M., Süßmuth, R., Jung, G., and Götz, F. (1998). Structure of the pheromone peptide of the *Staphylococcus epidermidis* agr system. *FEBS Lett.* 424, 89–94. doi: 10.1016/S0014-5793(98)00145-8

Pereira, C. S., de Regt, A. K., Brito, P. H., Miller, S. T., and Xavier, K. B. (2009). Identification of functional LsrB-like autoinducer-2 receptors. *J. Bacteriol.* 191, 6975–6987. doi: 10.1128/JB.00976-09

Pereira, C. S., Thompson, J. A., and Xavier, K. B. (2013). AI-2-mediated signaling in bacteria. *FEMS Microbiol. Rev.* 37, 156–181. doi: 10.1111/j.1574-6976.2012.00345.x

Peterson, S., Cline, R. T., Tettelin, H., Sharov, V., and Morrison, D. A. (2000). Gene expression analysis of the *Streptococcus pneumoniae* competence regulons by use of DNA microarrays. *J. Bacteriol.* 182, 6192–6202. doi: 10.1128/JB.182.21.6192-6202.2000

Phelps, H. A., and Neely, M. N. (2007). SalY of the *Streptococcus pyogenes* lantibiotic locus is required for full virulence and intracellular survival in macrophages. *Infect. Immun.* 75, 4541–4551. doi: 10.1128/IAI.00518-07

Podbielski, A., and Leonard, B. A. (1998). The group A streptococcal dipeptide permease (Dpp) is involved in the uptake of essential amino acids and affects the expression of cysteine protease. *Mol. Microbiol.* 28, 1323–1334. doi: 10.1046/j.1365-2958.1998.00898.x

Podbielski, A., Pohl, B., Woischnik, M., Körner, C., Schmidt, K. H., Rozdzinski, E., et al. (1996). Molecular characterization of group A streptococcal (GAS) oligopeptide permease (opp) and its effect on cysteine protease production. *Mol. Microbiol.* 21, 1087–1099. doi: 10.1046/j.1365-2958.1996.661421.x

Pulliainen, A. T., Hytönen, J., Haataja, S., and Finne, J. (2008). Deficiency of the Rgg regulator promotes H2O2 resistance, AhpCF-mediated H2O2 decomposition, and virulence in *Streptococcus pyogenes*. *J. Bacteriol.* 190, 3225–3235. doi: 10.1128/JB.01843-07

Qi, F., Chen, P., and Caufield, P. W. (1999). Functional analyses of the promoters in the lantibiotic mutacin II biosynthetic locus in *Streptococcus mutans*. *Appl. Environ. Microbiol.* 65, 652–658.

Quadri, L. E. N. (2002). Regulation of antimicrobial peptide production by autoinducer-mediated quorum sensing in lactic acid bacteria. *Antonie Van Leeuwenhoek* 82, 133–145. doi: 10.1023/A:1020624808520

Rawlinson, E. L. A., Nes, I. F., and Skaugen, M. (2002). LasX, a transcriptional regulator of the lactocin S biosynthetic genes in *Lactobacillus sakei* L45, acts both as an activator and a repressor. *Biochimie* 84, 559–567. doi: 10.1016/S0300-9084(02)01420-7

Redanz, S., Standar, K., Podbielski, A., and Kreikemeyer, B. (2012). Heterologous expression of sahH reveals that biofilm formation is autoinducer-2-independent in *Streptococcus sanguinis* but is associated with an intact activated methionine cycle. *J. Biol. Chem.* 287, 36111–36122. doi: 10.1074/jbc.M112.379230

Rezzonico, F., and Duffy, B. (2008). Lack of genomic evidence of AI-2 receptors suggests a non-quorum sensing role for luxS in most bacteria. *BMC Microbiol.* 8:154. doi: 10.1186/1471-2180-8-154

Roberts, A. L., Connolly, K. L., Kirse, D. J., Evans, A. K., Poehling, K. A., Peters, T. R., et al. (2012). Detection of group A Streptococcus in tonsils from pediatric patients reveals high rate of asymptomatic streptococcal carriage. *BMC Pediatr.* 12:3. doi: 10.1186/1471-2431-12-3

Rocha-Estrada, J., Aceves-Diez, A. E., Guarneros, G., and de la Torre, M. (2010). The RNPP family of quorum-sensing proteins in Gram-positive bacteria. *Appl. Microbiol. Biotechnol.* 87, 913–923. doi: 10.1007/s00253-010-2651-y

Ross, K., Ronson, C., and Tagg, J. (1993). Isolation and characterization of the lantibiotic salivaricin A and its structural gene salA from *Streptococcus salivarius* 20P3. *Appl. Environ. Microbiol.* 59, 2014–2021.

Rutherford, S. T., and Bassler, B. L. (2012). Bacterial quorum sensing: its role in virulence and possibilities for its control. *Cold Spring Harb. Perspect. Med.* 2:a012427. doi: 10.1101/cshperspect.a012427

Salim, K. Y., de Azavedo, J. C., Bast, D. J., and Cvitkovitch, D. G. (2008). Regulation of sagA, siaA and scpC by SilCR, a putative signaling peptide of *Streptococcus pyogenes*. *FEMS Microbiol. Lett.* 289, 119–125. doi: 10.1111/j.1574-6968.2008.01375.x

Samen, U. M., Eikmanns, B. J., and Reinscheid, D. J. (2006). The transcriptional regulator RovS controls the attachment of *Streptococcus agalactiae* to human epithelial cells and the expression of virulence genes. *Infect. Immun.* 74, 5625–5635. doi: 10.1128/IAI.00667-06

Schauder, S., Shokat, K., Surette, M. G., and Bassler, B. L. (2001). The LuxS family of bacterial autoinducers: biosynthesis of a novel quorum-sensing signal molecule. *Mol. Microbiol.* 41, 463–476. doi: 10.1046/j.1365-2958.2001.02532.x

Shaikh, N., Leonard, E., and Martin, J. M. J. M. (2010). Prevalence of streptococcal pharyngitis and streptococcal carriage in children: a meta-analysis. *Pediatrics* 126, e557. doi: 10.1542/peds.2009-2648

Jimenez and Federle                                                                 Cell-cell signaling in S. pyogenes

Shelburne, S., a, Olsen, R. J., Makthal, N., Brown, N. G., Sahasrabhojane, P., Watkins, E. M., et al. (2011). An amino-terminal signal peptide of Vfr protein negatively influences RopB-dependent SpeB expression and attenuates virulence in *Streptococcus pyogenes*. Mol. Microbiol. 82, 1481–1495. doi: 10.1111/j.1365-2958.2011.07902.x

Siller, M., Janapatla, R. P., Pirzada, Z. A., Hassler, C., Zinkl, D., and Charpentier, E. (2008). Functional analysis of the group A streptococcal luxS/AI-2 system in metabolism, adaptation to stress and interaction with host cells. BMC Microbiol. 8:188. doi: 10.1186/1471-2180-8-188

Simpson, W. J., Ragland, N. L., Ronson, C. W., and Tagg, J. R. (1995). A lantibiotic gene family widely distributed in *Streptococcus salivarius* and *Streptococcus pyogenes*. Dev. Biol. Stand. 85, 639–643.

Simpson, W. J., and Tagg, J. R. (1983). M-type 57 group A streptococcus bacteriocin. Can. J. Microbiol. 29, 1445–1451. doi: 10.1139/m83-221

Slamti, L., and Lereclus, D. (2002). A cell-cell signaling peptide activates the PlcR virulence regulon in bacteria of the Bacillus cereus group. EMBO J. 21, 4550–4559. doi: 10.1093/emboj/cdf450

Solomon, J. M., Lazazzera, B. A., and Grossman, A. D. (1996). Purification and characterization of an extracellular peptide factor that affects two different developmental pathways in *Bacillus subtilis*. Genes Dev. 10, 2014–2024. doi: 10.1101/gad.10.16.2014

Stephenson, S., Mueller, C., Jiang, M., and Perego, M. (2003). Molecular analysis of Phr peptide processing in *Bacillus subtilis*. J. Bacteriol. 185, 4861–4871. doi: 10.1128/JB.185.16.4861-4871.2003

Sturme, M. H. J., Nakayama, J., Molenaar, D., Murakami, Y., Kunugi, R., Fujii, T., et al. (2005). An agr-like two-component regulatory system in lactobacillus plantarum is involved in production of a novel cyclic peptide and regulation of adherence. J. Bacteriol. 187, 5224–5235. doi: 10.1128/JB.187.15.5224-5235.2005

Sulavik, M. C., Tardif, G., and Clewell, D. B. (1992). Identification of a gene, rgg, which regulates expression of glucosyltransferase and influences the Spp phenotype of *Streptococcus gordonii challis*. J. Bacterial. 174, 3577–3586.

Sumby, P., Whitney, A. R., Graviss, E. A., DeLeo, F. R., and Musser, J. M. (2006). Genome-wide analysis of group a streptococci reveals a mutation that modulates global phenotype and disease specificity. PLoS Pathog. 2:e5. doi: 10.1371/journal.ppat.0020005

Sumby, P., Zhang, S., Whitney, A. R., Falugi, F., Grandi, G., Graviss, E., et al. (2008). A chemokine-degrading extracellular protease made by group A Streptococcus alters pathogenesis by enhancing evasion of the innate immune response. Infect. Immun. 76, 978–985. doi: 10.1128/IAI.01354-07

Tagg, J. R. (2009). Streptococcal bacteriocin-like inhibitory substances: some personal insights into the bacteriocin-like activities produced by streptococci good and bad. Probiotics Antimicrob. Proteins 1, 60–66. doi: 10.1007/s12602-008-9002-7

Tagg, J. R., Read, R. S., and McGiven, A. R. (1973). Bacteriocin of a group A streptococcus: partial purification and properties. Antimicrob. Agents Chemother. 4, 214–221. doi: 10.1128/AAC.4.3.214

Tagg, J. R., and Skjold, S. A. (1984). A bacteriocin produced by certain M-type 49 *Streptococcus pyogenes* strains when incubated anaerobically. J. Hyg. (Lond.). 93, 339–344. doi: 10.1017/S0022172400064883

Thoendel, M., and Horswill, A. R. (2009). Identification of *Staphylococcus aureus* AgrD residues required for autoinducing peptide biosynthesis. J. Biol. Chem. 284, 21828–21838. doi: 10.1074/jbc.M109.031757

Tomasz, A. (1965). Control of the competent state in Pneumococcus by a hormone-like cell product: an example for a new type of regulatory mechanism in bacteria. Nature 208, 155–159. doi: 10.1038/208155a0

Upton, M., Tagg, J., Wescombe, P., and Jenkinson, H. F. (2001). Intra- and inter-species signaling between *Streptococcus salivarius* and *Streptococcus pyogenes* mediated by SalA and SalA1 Lantibiotic peptides. J. Bacteriol. 183, 3931–3938. doi: 10.1128/JB.183.13.3931-3938.2001

Wang, C., Lin, C.-Y. C., Luo, Y. Y.-H., Tsai, P.-J. P., Lin, Y.-S., Lin, M. T., et al. (2005). Effects of oligopeptide permease in group A streptococcal infection. Infect. Immun. 73, 2881–2890. doi: 10.1128/IAI.73.5.2881-2890.2005

Watson, R., Smoot, J., and Musser, J. (2001). Identification of Rgg-regulated exo-proteins of *Streptococcus pyogenes*. Infect. Immun. 69, 822–831. doi: 10.1128/IAI.69.2.822-831.2001

Wei, H., and Håvarstein, L. S. (2012). Fratricide is essential for efficient gene transfer between pneumococci in biofilms. Appl. Environ. Microbiol. 78, 5897–5905. doi: 10.1128/AEM.01343-12

Wescombe, P. A., Dyet, K. H., Dierksen, K. P., Power, D. A., Jack, R. W., Burton, J. P., et al. (2012). Salivaricin G32, a homolog of the prototype *Streptococcus pyogenes* nisin-like lantibiotic SA-FF22, produced by the commensal species *Streptococcus salivarius*. Int. J. Microbiol. 2012:738503. doi: 10.1155/2012/738503

Wescombe, P. A., and Tagg, J. R. (2003). Purification and characterization of streptin, a type A1 lantibiotic produced by *Streptococcus pyogenes*. Appl. Environ. Microbiol. 69, 2737–2747. doi: 10.1128/AEM.69.5.2737-2747.2003

Wescombe, P. A., Upton, M., Dierksen, K. P., Ragland, N. L., Sivabalan, S., Wirawan, R. E., et al. (2006). Production of the lantibiotic salivaricin A and its variants by oral streptococci and use of a specific induction assay to detect their presence in human saliva. Appl. Environ. Microbiol. 72, 1459–1466. doi: 10.1128/AEM.72.2.1459-1466.2006

Wollein Waldetoft, K., Råberg, L., Waldetoft, K. W., and Ra, L. (2014). To harm or not to harm? On the evolution and expression of virulence in group A streptococci. Trends Microbiol. 22, 7–13. doi: 10.1016/j.tim.2013.10.006

Woodbury, R. L., Wang, X., and Moran, C. P. (2006). Sigma X induces competence gene expression in *Streptococcus pyogenes*. Res. Microbiol. 157, 851–856. doi: 10.1016/j.resmic.2006.07.002

Wright, J. S., Jin, R., and Novick, R. P. (2005). Transient interference with staphylococcal quorum sensing blocks abscess formation. Proc. Natl. Acad. Sci. U.S.A. 102, 1691–1696. doi: 10.1073/pnas.0407661102

Xavier, K. B., and Bassler, B. L. (2003). LuxS quorum sensing: more than just a numbers game. Curr. Opin. Microbiol. 6, 191–197. doi: 10.1016/S1369-5274(03)00028-6

Xavier, K. B., and Bassler, B. L. (2005). Interference with AI-2-mediated bacterial cell-cell communication. Nature 437, 750–753. doi: 10.1038/nature03960

Zhang, L., Gray, L., Novick, R., and Ji, G. (2002). Transmembrane topology of AgrB, the protein involved in the post-translational modification of AgrD in *Staphylococcus aureus*. J. Biol. Chem. 277, 34736–34742. doi: 10.1074/jbc.M205367200

Zheng, F., Ji, H., Cao, M., Wang, C., Feng, Y., Li, M., et al. (2011). Contribution of the Rgg transcription regulator to metabolism and virulence of *Streptococcus suis* serotype 2. Infect. Immun. 79, 1319–1328. doi: 10.1128/IAI.00193-10

Conflict of Interest Statement: The authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.

Received: 28 May 2014; paper pending published: 03 July 2014; accepted: 26 August 2014; published online: 12 September 2014.

Citation: Jimenez JC and Federle MJ (2014) Quorum sensing in group A Streptococcus. Front. Cell. Infect. Microbiol. 4:127. doi: 10.3389/fcimb.2014.00127

This article was submitted to the journal Frontiers in Cellular and Infection Microbiology.

Copyright © 2014 Jimenez and Federle. This is an open-access article distributed under the terms of the Creative Commons Attribution License (CC BY). The use, distribution or reproduction in other forums is permitted, provided the original author(s) or licensor are credited and that the original publication in this journal is cited, in accordance with accepted academic practice. No use, distribution or reproduction is permitted which does not comply with these terms.
