
# Capsid expansion mechanism of bacteriophage T7 revealed by multistate atomic models derived from cryo-EM reconstructions

Fei Guo${}^{a}$, Zheng Liu${}^{a}$, Ping-An Fang${}^{a}$, Qinfen Zhang${}^{b,c}$, Elena T. Wright${}^{d}$, Weimin Wu${}^{a}$, Ci Zhang${}^{a}$, Frank Vago${}^{a}$, Yue Ren${}^{a}$, Joanita Jakana${}^{c}$, Wah Chiu${}^{c}$, Philip Serwer${}^{d}$, and Wen Jiang${}^{a,1}$

${}^{a}$Markey Center for Structural Biology, Department of Biological Sciences, Purdue University, West Lafayette, IN 47907; ${}^{b}$State Key Laboratory of Biocontrol, School of Life Sciences, Sun Yat-sen University, Guangzhou 510275, China; ${}^{c}$National Center for Macromolecular Imaging, Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Baylor College of Medicine, Houston, TX 77030; and ${}^{d}$Department of Biochemistry, The University of Texas Health Science Center, San Antonio, TX 78229

Edited by John E. Johnson, The Scripps Research Institute, La Jolla, CA, and accepted by the Editorial Board September 23, 2014 (received for review April 21, 2014)

Many dsDNA viruses first assemble a DNA-free procapsid, using a scaffolding protein-dependent process. The procapsid, then, undergoes dramatic conformational maturation while packaging DNA. For bacteriophage T7 we report the following four single-particle cryo-EM 3D reconstructions and the derived atomic models: procapsid (4.6-Å resolution), an early-stage DNA packaging intermediate (3.5 Å), a later-stage packaging intermediate (6.6 Å), and the final infectious phage (3.6 Å). In the procapsid, the N terminus of the major capsid protein, gp10, has a six-turn helix at the inner surface of the shell, where each skewed hexamer of gp10 interacts with two scaffolding proteins. With the exit of scaffolding proteins during maturation the gp10 N-terminal helix unfolds and swings through the capsid shell to the outer surface. The refolded N-terminal region has a hairpin that forms a novel noncovalent, joint-like, intercapsomeric interaction with a pocket formed during shell expansion. These large conformational changes also result in a new noncovalent, intracapsomeric topological linking. Both interactions further stabilize the capsids by interlocking all pentameric and hexameric capsomers in both DNA packaging intermediate and phage. Although the final phage shell has nearly identical structure to the shell of the DNA-free intermediate, surprisingly we found that the icosahedral faces of the phage are slightly (~4 Å) contracted relative to the faces of the intermediate, despite the internal pressure from the densely packaged DNA genome. These structures provide a basis for understanding the capsid maturation process during DNA packaging that is essential for large numbers of dsDNA viruses.

bacteriophage T7 maturation | DNA packaging intermediates | noncovalent topological linking | procapsid | single-particle cryo-EM

**Significance**

T7 phage has been used as a model system to study dsDNA virus capsid assembly and maturation. Yet, atomic capsid models and details of capsid transformations are not elucidated. From our cryo-EM study we have derived near-atomic resolution reconstructions of the DNA-free procapsid, a DNA packaging intermediate, and the DNA-packaged, mature phage capsid. From these structures, we have derived the first near-atomic-level model of T7 capsid maturation. The structural knowledge obtained from this study can serve as a platform for analysis of other dsDNA viruses as well as a platform for the development of molecular tools such as improved phage display systems.

and P-domain, which together place the gp10 protein in the HK97 fold category (2, 13, 14). The T7 procapsid, capsid I, contains 110–140 molecules of scaffolding protein, gp9 (4, 15, 16). After scaffolding protein expulsion the spherical T7 capsid I expands to more angular intermediates, which are collectively called capsid II (2, 4, 14, 16–18).

Two DNA-free capsid IIs are purified in quantity sufficient for structural studies by cryo-EM (16). Both are produced during the normal process of wild-type T7 DNA packaging in vivo. One has an unusually low density during buoyant density centrifugation in a metrizamide density gradient (1.086 g/mL; metrizamide low density, or MLD, capsid II) and the other has a density as expected for hydrated proteins (1.28 g/mL; metrizamide high density, or MHD, capsid II) (16). The low density of MLD capsid II is caused by impermeability to metrizamide (789 Da) (16). The MLD capsid II particles are produced before MHD capsid II particles based on kinetic studies (16).

The DNA packaging of T7 phage starts at capsid I state where the DNA is packaged by the ATPases (gp18 and gp19) to pass through the portal (gp8) apparatus (19). By analyzing kinetics of in vivo-produced capsids, MLD capsid II was found to be the first postcapsid I capsid. MLD capsid II appears with the kinetics

M any dsDNA viruses, including tailed phages and herpes viruses, initially assemble a DNA-free procapsid with assistance of a network of scaffold proteins. Accompanying the exit of scaffolding proteins during subsequent ATP-driven DNA packaging, the icosahedral shell of the procapsid undergoes dramatic conformational changes and matures into a typically larger and more angular shell of the infectious phage (1–6). However, structural details, including those of capsid intermediates, are limited to the phage HK97 system (5, 7–9), for which recombinantly produced procapsid and nonphysiological conversion products were analyzed.

The packaging of the 39.937-kbp DNA genome of the short-tail *Escherichia coli* bacteriophage, T7, is a model for understanding basic principles common to dsDNA tailed phages and herpes viruses. The T7 system is also of interest because it has been used for popular biotechnologies, such as recombinant protein expression (10) and protein display on the capsid surface (11). The T7 capsid contains 415 copies of the major shell protein gp10 (12) that form a T = 7L icosahedral lattice. From low-resolution cryo-EM 3D reconstructions the tertiary topology of gp10 can be divided into four regions: N-arm, E-loop, A-domain,

Author contributions: P.S. and W.J. designed research; F.G., Z.L., P.-A.F., Q.Z., F.V., Y.R., J.J., W.C., and W.J. performed research; E.T.W. and P.S. contributed new reagents/analytic tools; F.G., W.W., C.Z., and W.J. analyzed data; and F.G., P.S., and W.J. wrote the paper.

The authors declare no conflict of interest.

This article is a PNAS Direct Submission. J.E.J. is a guest editor invited by the Editorial Board.

Data deposition: EM data reported in this paper have been deposited in the Electron Microscopy Data Bank (accession nos. EMD-6034–EMD-6037) and atomic coordinates have been deposited in the Protein Data Bank, www.pdb.org (PDB ID codes 3J7V, 3J7W, and 3J7X).

${}^{1}$To whom correspondence should be addressed. Email: jiang12@purdue.edu.

This article contains supporting information online at www.pnas.org/lookup/suppl/doi:10.1073/pnas.1407020111/-/DCSupplemental.
of an intermediate (16) but is obviously no longer in the DNA packaging pathway because it has detached from the DNA molecule that it was packaging. MLD capsid II is not produced when a nonpermissive host is infected with a T7 amber mutant defective in DNA packaging (summarized in ref. 16). Thus, MLD capsid II is an intermediate that has been altered during either cellular lysis or subsequent purification. MHD capsid II also has the appearance kinetics of an intermediate of packaging, but one that occurs later (16). Whereas MLD capsid II has the internal core stack including proteins gp8, gp14, gp15, and gp16 (16), MHD capsid II does not have the internal core stack proteins, which were presumably lost when packaged DNA exited the capsid (16).

The existence of these various capsids provides an opportunity to obtain a high-resolution (3–4 Å) analysis of structural dynamics that occur in vivo. Here we report cryo-EM structures of the shells of the following bacteriophage T7 capsids: capsid I (4.6 Å), MLD capsid II (3.5 Å), MHD capsid II (6.6 Å), and phage (3.6 Å). The two capsid II shells are the first postprocapsid, in vivo-generated shells (for any packaging system) to be subjected to high-resolution structural analysis, to our knowledge. The results reveal $(i)$ an HK97-fold shell protein with an intracapsomere, noncovalent topological linking and another intercapsomere, joint interaction, neither interaction having been found for other dsDNA tailed phages; $(ii)$ details of the interaction of gp9 scaffolding protein with the inner surface of the capsid I shell; $(iii)$ a novel refolding and externalization of the N terminus of major capsid protein, gp10; and $(iv)$ a subtle, surprising contraction of the gp10 shell in transit from MLD capsid II to phage. Based on these observations, we propose a general procapsid assembly and maturation pathway for dsDNA viruses.

### Results

#### Electron Cryomicroscopy and 3D Reconstruction
In this study, we used a T7 strain (T7A) with only the gp10A (345 aa) variant of the major capsid protein. This practice minimizes particle heterogeneity in relation to wild-type T7. Wild-type T7 capsids typically also contain ~5–10% of gp10B (398 aa); gp10B is a variant of gp10 produced by translation frame-shift near the C terminus (12). We also used polylysine-coated, ultrathin, continuous carbon transmission EM (TEM) grids to improve capsid adsorption (20). We imaged capsid I, MLD capsid II, MHD capsid II, and phage particles (Fig. 1A) using cryo-EM (Table S1). To obtain high-resolution images (Fig. 1A), extra attention was paid to imaging with parallel beam illumination and comma-free alignment.

All shell structures (Fig. 1B) were determined de novo using icosahedral reconstruction. The image processing and 3D reconstruction adopted truly independent refinements (21) and the resolutions were assessed using the “gold” standard FSC = 0.143 criterion (22–24). The MLD capsid II dataset gave the highest resolution at 3.5 Å (Fig. S1 and Table S1). At this resolution, the electron densities (Fig. 1C and Fig. S2) are well resolved for the majority of the regions, which allowed us to manually trace the backbone, build a full-atomic gp10 model, and refine the model. We used crystallographic refinements against the structural factors calculated from the original density map. The refined structures then were validated using standard X-ray crystallographic

Text

![Figure 1](#fig1)

**Fig. 1.** Single-particle 3D reconstructions of bacteriophage T7. (A) Cryo-EM images of capsid I, MLD capsid II, MHD capsid II, and phage. (B) Threefold surface view of corresponding 3D reconstructions. The surfaces are colored by radius. One T = 7L asymmetric unit is highlighted in magenta. (C) Electron densities of three representative regions (a loop, an $\alpha$-helix, and a $\beta$-strand) of MLD capsid II structure.

Guo et al. | Published online October 13, 2014 | E4607
measurements: R-factor, Ramachandran plot, and real-space correlation coefficient, which are listed in Table S2 and Fig. S3.

The phage shell was solved at 3.6 Å resolution. Owing to its close similarity in structure to that of the MLD capsid II, the atomic model of the MLD capsid II shell was used as the starting model for the phage shell, which was then further refined against the electron densities of the phage structure. The MHD capsid II shell was solved until 6.6-Å resolution, at which point the icosahedral reconstruction was indistinguishable from that of the MLD capsid II. The increased metrizamide permeability of MHD capsid II is likely due to the loss of the portal and core stack proteins to create a large opening at the portal vertex. This loss was observed by SDS/PAGE analysis of the capsid protein compositions (16).

The capsid I electron density map was solved at 4.6-Å resolution (Fig. S1 and Table S1). In this structure, several regions were less resolved, probably reflecting flexibility of gp10 in this metastable shell and possibly also heterogeneity of the capsid I particles. The atomic model of capsid I was derived from the model of MLD capsid II by using molecular dynamics flexible fitting (MDFF) (25) and crystallographic refinements as described for MLD capsid II.

Atomic Model of Capsid I Reveals a Hidden N-Terminal Helix. Capsid I is the most spherical capsid of the four. Capsid I has a rotationally averaged and maximum diameter (peak-to-peak distance of the shell in radial density plot) of 492 Å and 565 Å, respectively. The pentons are nearly closed, whereas the hexons have an 11- × 16-Å central opening. From the 4.6-Å cryo-EM electron density map an atomic model was built for most of the major capsid protein, gp10A (residues 22–343). Residues 1–21 were missing in the cryo-EM electron density map and assumed to be in nonunique conformations. A gp10A subunit of the capsid I shell (Fig. 2A) has conformation similar to the HK97 fold, which has now been well established among dsDNA viruses including tailed phages (1–3, 26–29) and human herpes virus (30).

Using previous terminology, the T7 gp10A conformation is divided into N-helix, E-loop, P-domain, and A-domain (capsid I, Fig. 2A). The E-loop, P-domain, and A-domain are analogous to those in other bacteriophages such as HK97 and P22 (1, 8). However, the six-turn N-helix (residues 24–43) (Fig. 2A) is only sometimes found in other phage procapsids. It is present in the P22 procapsid (1) but not in the HK97 procapsid (8). The N-helix is near the inner shell surface next to the P-domain, where it is covered by the scaffolding protein, gp9, as discussed in next section (Figs. 2A and C and 3B and C). Furthermore, a small insertion G-loop near the carboxyl end of the P-domain spine helix in HK97 is not observed in the T7 spine helix (i.e., α2 in Fig. 2 and Fig. S4).

Each asymmetric unit consists of seven gp10A subunits (A–G) (Fig. 1B and Fig. 2C). The Cα rmsds of the seven subunits were at 2–4 Å (Fig. S5C illustrates the differences in conformation). Subunits *a*–*f* form a skewed hexameric capsomere in capsid I, whereas five *g* subunits, from adjacent asymmetric units, form a pentameric capsomere at the fivefold vertex (Fig. 2C). Owing to the skewing, the symmetry of the hexamer is reduced to twofold (Fig. 2E). Subunits *b* and *e* are located on the two short edges of the skewed hexagon and subunits *a*, *c*, *d*, and *f* occupy the four longer edges. The E-loops of subunits *b*, *e*, and *g* had the largest bending toward the capsid center (Fig. S5C). The hexamer skewing increased from the interior N-helix to the outer A-domain (Fig. 2E). Similar patterns of changing skewness are also found in the structures of the P22 and HK97 procapsids, although this was not discussed previously (1, 7, 31). The E-loop and P-domain interact when in neighboring subunits within the same hexamer or pentamer (Fig. S6), which provides the most intracapsomeric stability in capsid I. The interactions are mainly electrostatic. They are derived from negatively charged E-loop and positively charged outer surface residues of the P-domain. In

![Figure 2](#fig2)

Fig. 2. Atomic models of major capsid protein gp10A. (A) Subunit *g* in capsid I. (B) Subunit *d* in MLD capsid II. The models are rainbow-colored from N (blue) to C (red) terminus. A salt bridge formed by Asp-103 and Arg-262 is shown as sticks in *B*. *A* and *B* are viewed from the capsid outer surface, similar to the orientation of the bottom panel in *C* and *D*. (*C* and *D*) Arrangement of subunits in a pentamer and a hexamer of the (*C*) capsid I shell and (*D*) MLD II capsid II shell in two orthogonal views (side view in top panel and top view in bottom panel). The N-helix (*C*) and N-arm (*D*) regions are colored in blue. The red asterisk indicates the P-domain/E-loop interaction of subunits *e* and *f*. (*E*) The section views of the asymmetric unit density (green) of capsid I. The locations of the sections, which have variable degrees of skewing, are indicated by red dashed arrows.

Fig. 3. Interactions of scaffolding protein gp9 with major capsid protein gp10 on the inner surface of capsid I. (A) The inner surface is radially colored. The scaffolding protein, gp9, densities (blue) are found on hexamers, but not on pentamers. Scaffolding protein in the boxed region is magnified in C. The electron densities were low-pass-filtered to 7 Å for cleaner view. (B) Zoom-in view of the N-helix region. The N-helix model (sticks) is superimposed in the electron density (wire frames). (C) Zoom-in view of the electron densities of scaffolding protein HTH region found on gp10 subunit f. (D) Model of gp10 and the gp9 HTH region. The gp10 is the same subunit, f, as shown in B.

contrast, the A-domains in hexamers have relatively little interaction with neighboring subunits owing to the skewing (Fig. 2C).

As for the threefold axes, where three neighboring capsomeres interact, the E-loops from different capsomeres directly contact each other on the outer surface and three P-domains form the inner surface (Fig. S6). In addition to the bending of the E-loop, a one-turn helix (residues 82–87) preceding β-strand βC forms only in capsid I (Fig. 2A). This short helix interacts with the P-loop of neighboring capsomere.

Previous studies of HK97 prohead I structure showed that there are significant differences among the three classes of icosahedral and quasi-threefold axes (7). However, only more subtle differences are observed among the three classes of threefold interaction in T7 capsid I (Fig. S6). In T7 capsid I, around all three classes of threefold axes, the E-loop tip of one subunit interacts with two consecutive subunits of its neighboring capsomere, one with the E-loop bending region of the threefold related subunit and another with the P-domain (P-loop between βK/βJ and α1) of the other subunit (Fig. S6). In T7 capsid I, all types of threefold axes (quasi and icosahedral) have similar center openings and adjacent twofold axes regions (Fig. S6). The similarity among the three classes of threefold axis conformations was also observed in the p22 procapsid (1).

Interactions of Scaffolding Proteins with Capsid Shell in Capsid I. We found that the scaffolding protein, gp9, resides in the procapsid interior, as expected from previous data for HK97, P22, and T7 (1, 4, 7, 14, 31). The gp9 densities were detected as additional densities at the inner surface of hexons, but not pentons, after modeling gp10A. The volumes of these hexon-specific densities were much larger than the possible volume of the unresolved 21 residues in the N terminus of gp10. Thus, we assumed that these densities were from the scaffolding protein, gp9 (blue in Fig. 3A and C). These densities presumably include the “nubbins” seen by Cerritelli et al. (4) at lower resolution. To further map gp9, we used the helix-searching function of PHENIX (32). This program located and built two antiparallel helices in double rod-shaped density segments found in gp9. These helices were located under gp10A subunits c and f, which are the subunits that had the strongest gp9 densities (Fig. 3A and D). T7 gp9 is known to be rich in helices (15). A similar helix-turn-helix (HTH) motif was observed in phi29 and P22 scaffolding protein structures at the C terminus (1, 33–35).

Stoichiometry analysis has found that gp9:gp10 molar ratio is ~1:3 (36). Thus, we propose that there are two gp9 protein subunits for each hexon. This is consistent with the quasi-twofold symmetry of the skewed hexons of capsid I. This conclusion is also supported by the observation that the six discrete gp9 densities, when viewed at lower threshold, became fused into two separate elongated densities (Fig. S7). In this model, a single gp9 subunit with an extended conformation (15) interacts with three adjacent gp10 subunits in a hexon. Only the C-terminal HTH motif is well resolved owing to its strong interactions with the gp10 c and f subunits, whereas the rest of gp9 is less resolved, probably owing to more flexible conformations or weaker interactions with the gp10. It is also worth noting that the structurally related P22 scaffolding proteins adopt an elongated dimer form in solution that is essential for the procapsid assembly (37).

Despite the nonequivalent interactions between gp9 and the different gp10 subunits in a hexon, all of the observed rod-shaped gp9 densities were unambiguously found to be approximately at a right angle to a gp10 N-helix (Fig. 3D). The strong side chain electron densities of the gp9 HTH motif primarily contact the subunit c- and f-associated gp10 N-helix, at hydrophobic residues (Leu-27, Val-31, and Leu-37) and charged residues (Lys-30 and Arg-42). Tyr-116, from the amino end of the P-domain spine helix, is also in contact with gp9. In addition, the gp9 HTH motif interacts with the A-domain of subunit d (next to c) and a (next to f). Thus, a single gp9 collectively interacts with four consecutive gp10 subunits in the hexon, which presumably helps to assemble and stabilize the skewed gp10 hexamers.

Inter- and Intracapsomeric Interlocking Interactions in Capsid II and Phage. MLD capsid II, MHD capsid II, and phage shells are all expanded, with angular shape (Fig. 1B and Movie S1). The shells of capsid II and phage are nearly identical except for a small additional inward tilt of the asymmetric unit in the phage shell. This structural difference between capsid II and phage gp10 is
further discussed in another section, below. The average diameter of MLD/MHD capsid II is 568 Å and that of the T7 phage is 564 Å. The maximum capsid diameter for all three is 662 Å, along fivefold axes. In contrast to the skewed hexamers of capsid I, the hexamers of capsid II and phage are sixfold symmetric.

Atomic models of gp10A (residues 2–344) were built for MLD capsid II and phage shells of T7 (Fig. 2 $B$ and $D$). The gp10A structure within all expanded shells had topology similar to the topology of gp10A in capsid I. However, the conversion to capsid II produced several prominent conformational changes. $(i)$ The N-helix (residues 24–43) was unfolded and the entire N-terminal segment (first 43 aa) was refolded into an extended conformation (N-arm, Fig. S8C). $(ii)$ The refolding-derived N-arm was exposed on the outer surface (Fig. 2 $B$ and $D$). Thus, both N terminus and C terminus are exposed on the outer surface in the expanded shells. $(iii)$ The apex of the A-domain was tilted outward and nearly filled the center opening of both hexons and pentons (Fig. S8B). $(iv)$ Upon capsid expansion, the E-loop was bent $\sim 30^{\circ}$ toward the outer surface (Fig. S8D). $(v)$ A novel intracapsomere, noncovalent topological linking of the salt-bridge-derived intrasubunit loop and the E-loop of neighboring subunit was formed (Fig. 4 $B$ and $D$ and Movie S2). $(vi)$ A novel intercapsomere interaction was formed by the N-terminal hairpin-like loop and the A-pocket of another capsomere (Fig. 4 $B$ and $C$). The latter two interactions are unique (among known viral structures) to the structures of T7 expanded shells. $(vii)$ The spine helix ($\alpha 2$) was kinked in capsid I state but became straighter in capsid II and phage (Fig. S4 $A$ and $C$). Similar straightening of the spine helix was previously observed for the corresponding HK97 structures (Fig. S4 $B$ and $D$).

The intrasubunit loop in the topological linking (feature $\nu$, above) occurs via a salt bridge between Asp-103 in the top-strand, $\beta D$, of the three-strand $\beta$-sheet in the P-domain and Arg-262 of the A-loop (residues 253–271) in the A-domain of the same subunit. The A-loop itself is also unique in that it is not seen in other phages, including HK97, P22, $\varepsilon 15$, and BPP-1 (1, 7,

26–28, 38). The G-loop insertion at the carboxyl end of the spine helix in HK97 does not have a counterpart in T7 gp10 (Fig. S4). Instead, the A-loop is approximately located at similar position. Unlike the covalently cross-linked chain mail in phage HK97 (28, 29), the T7 topological linking is $(i)$ intracapsomeric, instead of intercapsomeric, and $(ii)$ without covalent cross-linking, even though feature $\nu$ does involve chain-link topology. These intracapsomere topological linkings between every pair of neighboring subunits help to interlock the entire capsomere into a robust structural unit in the icosahedral lattice.

The interaction of N-hairpin (residues 11–25, feature $v i$, above) is intercapsomeric and occurs with the A-pocket of a gp10 subunit in an adjacent capsomere (Fig. 4 $B$ and $C$). The A-pocket (residues 152–156) (Fig. 4C) is located in the loop connecting the spine helix in the P-domain and the $\beta E$ strand in the A-domain (Fig. 2$B$). The complementary shape and charges between the positively charged N-hairpin tip and the negatively charged A-pocket contribute significantly to this interaction (Figs. 2$B$ and 4 $B$ and $C$). These intercapsomere joining interactions around the icosahedral twofold and quasi-twofold axes interlock all 71 capsomers (60 hexons and 11 pentons) into a single robust cage (Fig. 4$A$). Both the A-pocket and N-hairpin were originally in a disordered state in capsid I but became ordered in the expanded capsid shell, which is presumably stabilized by these newly formed interactions.

In addition to these two interactions unique to T7 structures (one intracapsomere and one intercapsomere), extensive neighboring subunit interactions ($\sim 11,300 \AA^{2}$ for one subunit with all surrounding intra/intercapsomere neighbors) are also observed at the interfaces in both A-domain and P-domain (Fig. 2$D$ and Fig. S9C). Similar interactions were observed in phage structures previously reported (e.g., interface area is about 35% of entire surface area for both T7 phage and HK97 mature head) (28, 38). For example, around each icosahedral threefold and quasithreefold axis, three subunits from different capsomers are involved in the following intercapsomere interactions. On the outer

Fig. 4. Novel intracapsomere topological linking and intercapsomere joint in capsid II and phage. (A) Outer surface view of a capsid region slightly larger than an icosahedral face. Three hexamers are highlighted in different colors (red, blue, and yellow). (B) Magnified view of two neighboring hexamers with two interactions indicated by boxes. The first interaction is the intercapsomere joint interaction (magenta box) between the N-hairpin (red) and the A-pocket (blue). The E-loop, $\beta$D-strand, and A-loop regions of the hexamers are also colored to highlight the second interaction. The second interaction is intracapsomere noncovalent topological linking (orange box). (C) Zoom-in view of the first interaction, that is, between the A-pocket (K151, Y152, N153, I156, and E157 shown as stick) and the N-hairpin (Q14, G15, K16, G17, V18, and V19 shown as stick) around twofold and pseudotwofold axes. Electron densities of this region are shown as wire frames. (D) Zoom-in view of the second interaction, which is formed by encircling of an E-loop (red) by a loop formed via a salt bridge (stick and wire frame) between R262 in the A-loop and D103 in the $\beta$D-strand of a neighboring subunit.
surface, the positively charged N terminus from three subunits (c as shown in Fig. S9A) and the negatively charged E-loop tip from another set of three subunits (e as shown in Fig. S9A) are tightly arranged in alternating pattern around the threefold axis (Fig. S9A). In contrast, for HK97, an N-terminal region has much less interaction with E-loops. This interaction likely does not contribute significant stabilization forces to the HK97 capsid (Fig. S9B). On the inner surface, the P-domain of both T7 and HK97 from three subunits (d as shown in Fig. S9 C and D) are tightly packed to generate another trio of stabilizing charge–charge interactions.

DNA Packaging-Associated Flattening of the Capsid. Analysis of capsid dimensions was the first step in finding a subtle difference between the MLD capsid II shell and the phage shell. Along the threefold axis, the MLD capsid II shell had a slightly (4 Å) larger radius than the mature phage, whereas no significant size difference was seen along the twofold and fivefold axes (Fig. S10). To test the possibility that microscope magnification variations and 3D reconstruction methods caused these subtle size differences, we have also obtained independent datasets of T7 particles from unpurified, T7-infected cell lysates that contain capsid I, capsid II, and phage particles on the same sample grid and imaged simultaneously. Similar size variations were also observed in these independent studies (39). The location dependence of size variation indicates that the icosahedral faces in the capsid shell are slightly flatter in the phage state than in the MLD capsid II state.

Further analysis of the mechanism of the size variation found no significant structural difference when the atomic models of the asymmetric units of the MLD capsid II and the phage were superimposed, ignoring absolute positions on the capsid (Fig. 5A). The rmsd for all atoms was only 0.9 Å. However, when the positions of the asymmetric units in the capsid are considered, a 1.2° inward, rigid body tilting of each asymmetric subunit was observed. The pivot point is the penton center (Fig. 5 B and C). This observation of capsid flattening is the first observed for DNA packaging intermediates produced in vivo, to our knowledge.

**Discussion**

**Scaffolding Protein-Assisted Procapsid Assembly.** Our 3D reconstruction of T7 capsid I is the first to our knowledge that reveals the twofold symmetry of two gp9 scaffolding proteins (120 copies) bound to a gp10 hexon, consistent with biochemical data on T7 scaffolding proteins (110–140 copies per procapsid) (36). The lack of penton-associated scaffolding protein density may be due to the dynamic nature of scaffolding protein–shell interactions. It is also possible that the penton-associated scaffolding proteins dissociate either as the capsid I shell is assembled or subsequently, during purification. Spontaneously expanded, capsid II-like particles are typically present in capsid I samples, which might be triggered by the partial loss of scaffolding proteins during sample purification. Alternatively, the lack of interacting scaffolding proteins might favor the formation of gp10 pentamers during capsid I assembly.

Assuming that the stoichiometry of our capsid I structure also exists for capsid I assembly intermediates, a simple assembly hypothesis is the following. The initial assembly is a complex of three gp10 subunits bound to a single gp9 subunit. This assembly (Fig. 6A) is the building block for the skewed hexamer with quasi-twofold symmetry in capsid I. Capsid I would be generated by adding hexamers and pentamers to partial capsids to complete an icosahedral lattice (capsid I) (Fig. 6B).

We note that the nonequivalent scaffolding protein distributions of T7 capsid I are different from what is observed for HK97 and P22 procapsids (1, 31). The latter have 1:1 (415 copies) and ~1:2 to ~3:4 (~200–300 copies) ratio of scaffolding to capsid protein, respectively (39, 40). This variability among phage procapsids is approximately correlated to the molecular mass of the scaffolding proteins and the available internal volume for the

![Figure 5](#fig5)

Fig. 5. Comparison of asymmetric unit positions in the MLD capsid II and mature phage shells. The top view (A) and the side views (B and C) of MLD capsid II and phage asymmetric units are superimposed. MLD capsid II and phage are colored magenta and green, respectively. The asymmetric unit of MLD capsid II undergoes ~1.2° inward tilt around the pivot point (marked by asterisk) to reach the final position in phage state.

scaffolds. In T7 capsid I, the available volume for scaffold is significantly reduced owing to the core stack (18). In the HK97 procapsid, the scaffolding protein is the N-terminal delta-domain of the shell protein gp5. Although it is smaller (approximately one-third of the mass of T7 scaffold protein), the larger copy number (415) makes it occupy similar volume when the protease proteins are considered. The P22 scaffolding protein is similar in mass to that of T7. However, the available internal volume in the P22 procapsid is larger because P22 does not have an internal core stack (only a helical barrel extension from the portal). Nonetheless, the functional role of scaffold–shell protein interaction is similar in the procapsids of P22 and HK97 (1, 8).

**Bacteriophage T7 Maturation.** The structures presented here are, to our knowledge, the first determined at a resolution sufficient to trace atomic details of the conformational changes that occur during T7 shell maturation. The previously reported quasi-atomic models of T7 capsid I and mature phage were generated by homology modeling using HK97 crystal structures as the template (2, 14). As a result, none of the unique structural features of T7 that we observed was revealed in those models. During DNA packaging, the capsid I to capsid II conversion (Fig. 6C–E and Movie S1) causes refolding and externalization of the N-terminal region of the gp10 protein. The refolding converts an α-helix (N-helix) to a coil (N-hairpin and N-arm) while executing a swinging externalization movement from the inner surface toward the outer surface (Fig. 2D, Figs. S8C and S9, and Movie S1). Given the interaction of gp9 scaffolding protein with the N-helix of gp10 in capsid I, these transitions of gp10 are probably coupled to the dissociation of the scaffolding protein gp9.

The refolding of the N-arm is accompanied by a disorder-to-order transition in the A-domain to form the A-pocket. Together, these two events generate the components for strong intercapsomeric interactions (Fig. 4C). This raises the unanswered question of whether induced fit of N-arm to A-pocket

Neither of these stabilization interactions was present in the procapsid state. They only formed in the expanded “matured” capsids. Although the maturation-generated interactions are vastly different among different phages (covalent or noncovalent, around threefold or twofold axes), they all have the same effect: stabilization of shells to retain DNA. The differences exist despite the fact that the HK97-like fold for the major capsid protein is conserved. Selection for increased capsid stability may be related to the evolutionary pressure to increase DNA packaging density to $(i)$ provide genes for adapting to propagation in conditions that vary with the hosts (48–52) and $(ii)$ survive host defenses (53, 54).

**Puzzling Flattening of the Capsid Associated with DNA Packaging.** We have observed that DNA packaging/capsid maturation is associated with a small inward tilt of the asymmetric unit. This observation was made by comparing the shells of T7 MLD capsid II and mature phage. The tilt resulted in further flattening of icosahedral faces after DNA packaging and tail attachment (Fig. 5 and Fig. S10). Currently, there are no other dsDNA tailed phages for which structures have been determined for a pair of particle states comparable to T7 MLD capsid II (a DNA packaging intermediate) and mature phage (DNA fully packaged). Small contractions were reported for HK97 and T5 phages undergoing a different process (i.e., DNA ejection) in which DNA is released from mature phage (55, 56). These contractions are counterintuitive and the opposite is expected (i.e., a swelling in phage state owing to pressure from the densely packed DNA genome) (57). We do not have a good explanation for this puzzling observation.

**Materials and Methods**

**Sample Preparation.** T7A phage, capsid I, MLD capsid II, and MHD capsid II were purified using previously published methods (16, 17). Briefly, 6 L of $E$. coli host cells were grown to $4 \times 10^8$ per milliliter and inoculated with T7A phage for the preparation of MLD/MHD capsid IIs and phage. For preparation of capsid I, a double amber mutant in genes 5/19 was used at a multiplicity of 4. After the cells spontaneously lysed, with assistance from chloroform for capsid I, the particles were twice precipitated with polyethylene glycol and resuspended. The resuspended particles were clarified by low-speed centrifugation and DNase I-digested. The particles were then purified by two CsCl gradient centrifugations, first through a step gradient and then a buoyant density gradient. The capsid II band was further separated into MLD and MHD bands using metrizamide buoyant density centrifugation. Particle bands were collected and then dialyzed into 0.2 M NaCl, 10 mM Tris·Cl (pH 7.4), and 1 mM MgCl₂.

**Cryo-EM.** A 3-$\mu$L sample was applied to a 400-mesh holey carbon grid for plunge-freezing in liquid nitrogen using FEI Vitrobot. C-flat grid CF-1.2/1.3-4C was used to prepare T7A phage. Ted Pella Ultrathin Carbon on Holey Support Film (01824) grids, coated with polylysine, were used for all other samples to increase the particle densities. An FEI Titan Krios microscope was used to image the samples on Kodak SO-163 films. Some of the T7A phage data were obtained using a JEOL 3200FSC with 20-eV slit of the in-column energy filter on a Gatan UltraScan 4000 CCD. These microscopes were operated at 300 kV at liquid nitrogen temperature in low-dose condition ($\sim$25 e/$\AA^2$). To ensure beam coherence and minimize beam tilt, parallel beam illumination was used and careful coma-free alignment was performed. T7A phage, T7A capsid I, and T7A MLD capsid II were imaged at 59,000 nominal magnification and T7A MHD capsid II was imaged at 37,000 nominal magnification.

**Image Processing and 3D Reconstructions.** Films were scanned using a Nikon Coolscan 9000 ED at a step of 6.35 $\mu$m per pixel. The final samplings of digitized images were calibrated using a MAG*I*CAL grid and found to be 1.10 Å per pixel for T7A phage, T7A capsid I, and T7A MLD capsid II and 1.81 Å per pixel for MHD capsid II images. Particles were selected from scanned micrograph images, first automatically by the ETHAN method (58) and then by manual screening with the boxer program in EMAN (59). The TEM instrument contrast transfer function parameters were determined automatically using fitctf2.py (60, 61) and were then visually validated using EMAN ctfit program.
For 3D reconstructions, whole datasets were divided into even-odd halves. The initial de novo models and subsequent iterative refinements were all independently performed for each half dataset. The images were first binned four times to obtain initial models and particle parameters assuming icosahedral symmetry. De novo initial models were built using the random model approach (21, 62–64). Briefly, a random subset (200) of particles were assigned random initial orientations and iteratively refined until convergence. Consistent icosahedral capsid structures (other than occasional difference in handedness) were obtained by repeating the random model process. Particles with inconsistent/unstable view parameters in the initial refinements were excluded in further image processing. The orientation and center parameters were then transferred to the unbinned images for high-resolution refinements that include Simplex method-based orientation/center optimization and grid-search-based refinement of defocus, astigmatism, and magnification of the images (21). The seven subunits in the asymmetric unit of the 3D reconstructions have noticeable conformational differences (Fig. S5 C and D), which resulted from nonequivalent locations and interactions. Thus, the densities of the seven nonicosahedrally related subunits were not averaged and the resolutions were evaluated based on the entire icosahedral shell. The final resolution was evaluated using the “gold” standard Fourier shell correlation (threshold = 0.143 criterion) (22–24) of the two truly independent reconstructions (Fig. S1). The datasets and refinement statistics for the four T7 particles are summarized in Table S1.

All image refinement and reconstructions were performed with in-house-developed programs (21), jspr.py (for overall workflow), jalign (for 2D alignment), and j3dr (for 3D reconstruction) that use EMAN (59) and EMAN2 (65) library functions. These programs are available from W.J.’s website ([jiang.bio.purdue.edu/jspr](http://jiang.bio.purdue.edu/jspr)) and also as a cloud-computing resource on DiaGrid ([diagrid.org/tools/cryoem](http://diagrid.org/tools/cryoem)).

### Model Building

The first gp10A atomic model was built solely based on the D subunit of the cryo-EM density map of T7A MLD capsid II. The backbone densities were contiguous and visually traceable. A Cα trace was first constructed using the baton mode and then converted to a polyalanine peptide chain in Coot (66). At 3.5 Å, large side-chain landmarks are clear enough to determine residue positions. After manually adjusting each residue position, a copy of the peptide chain of gp10A (residues 2–344) was docked to other subunits in the asymmetric unit. These copies were manually adjusted to accommodate the nonequivalent conformations of the seven subunits to better fit the densities. Treating the seven subunits as independent structures, the subsequent refinements were performed based on pseudocrystallographic refinement of the entire asymmetric unit using PHENIX (32). The asymmetric unit density map was first segmented out using Chimera (66), placed in an artificial lattice of P1 space group, and then converted to the pseudodiffraction data (mtz/hkl) using CCP4 program suite (67). The subsequent refinement was performed with iterative manual adjustment in Coot and automatic refinement by PHENIX, which is similar to the typical medium-resolution crystallographic refinement strategy except that the original cryo-EM density map was always used as the reference for each iteration of refinement.

Because the cryo-EM density of T7A phage and T7A MLD capsid II shells differed only within a few angstroms, the T7A MLD capsid II model was directly used as the initial template for T7A phage model refinement. For the T7A capsid I model, MDFF (25) of a single chain from the MLD capsid II model was first used to obtain the initial model of capsid I. Then, the model was refined using PHENIX and Coot as for MLD capsid II. The partial T7 scaffolding protein gp9 polyalanine model was built automatically using “Find helices and strands” in PHENIX against the density template cut from subunit F using Chimera (66). The qualities of the models (Table S2) were evaluated by MolProbity (68) functions integrated in PHENIX.

#### ACKNOWLEDGMENTS

This work was supported by National Institutes of Health Grants R01AI072035, S10RR023011 (to W.J.), and R01GM24365 and Welch Foundation Grant AQ-764 (to P.S.). The cryo-EM images were taken in the Purdue Biological Electron Microscopy Facility, and the Purdue Rosen Center for Advanced Computing provided the computational resource for the 3D reconstructions.

---

1. Chen DH, et al. (2011) Structural basis for scaffolding-mediated assembly and maturation of a dsDNA virus. *Proc Natl Acad Sci USA* 108(4): 1355–1360.
2. Ionel A, et al. (2011) Molecular rearrangements involved in the capsid shell maturation of bacteriophage T7. *J Biol Chem* 286(1): 234–242.
3. Jiang W, et al. (2003) Coat protein fold and maturation transition of bacteriophage P22 seen at subnanometer resolutions. *Nat Struct Biol* 10(2): 131–135.
4. Cerritelli ME, Conway JF, Cheng N, Trus BL, Steven AC (2003) Molecular mechanisms in bacteriophage T7 procapsid assembly, maturation, and DNA containment. *Adv Protein Chem* 64: 301–323.
5. Conway JF, et al. (2001) Virus maturation involving large subunit rotations and local refolding. *Science* 292(5517): 744–748.
6. Tao Y, et al. (1998) Assembly of a tailed bacterial virus and its genome release studied in three dimensions. *Cell* 95(3): 431–437.
7. Veesler D, et al. (2014) Architecture of a dsDNA viral capsid in complex with its maturation protease. *Structure* 22(2): 230–237.
8. Gertsman I, et al. (2009) An unexpected twist in viral capsid maturation. *Nature* 458(7238): 646–650.
9. Gan L, et al. (2006) Capsid conformational sampling in HK97 maturation visualized by X-ray crystallography and cryo-EM. *Structure* 14(11): 1655–1665.
10. Moffatt BA, Dunn JJ, Studier FW (1984) Nucleotide sequence of the gene for bacteriophage T7 RNA polymerase. *J Mol Biol* 173(2): 265–269.
11. Rosenberg A, et al. (1996) T7Select Phage Display System: A powerful new protein display system based on bacteriophage T7. *Innovations* 6: 1–6.
12. Dunn JJ, Studier FW (1983) Complete nucleotide sequence of bacteriophage T7 DNA and the locations of T7 genetic elements. *J Mol Biol* 166(4): 477–535.
13. Bertin A, de Frutos M, Letellier L (2011) Bacteriophage-host interactions leading to genome internalization. *Curr Opin Microbiol* 14(4): 492–496.
14. Agirrezabala X, et al. (2007) Quasi-atomic model of bacteriophage t7 procapsid shell: Insights into the structure and evolution of a basic fold. *Structure* 15(4): 461–472.
15. Cerritelli ME, Studier FW (1996) Assembly of T7 capsids from independently expressed and purified head protein and scaffolding protein. *J Mol Biol* 258(2): 286–298.
16. Serwer P (1980) A metrizamide-impermeable capsid in the DNA packaging pathway of bacteriophage T7. *J Mol Biol* 138(1): 65–91.
17. Serwer P (1975) Buoyant density sedimentation of macromolecules in sodium iodate density gradients. *J Mol Biol* 92(3): 433–448.
18. Guo F, et al. (2013) Visualization of uncorrelated, tandem symmetry mismatches in the internal genome packaging apparatus of bacteriophage T7. *Proc Natl Acad Sci USA* 110(17): 6811–6816.
19. Fujisawa H, Morita M (1997) Phage DNA packaging. *Genes Cells* 2(9): 537–545.
20. Barth OM (1985) The use of polylysine during negative staining of viral suspensions. *J Virol Methods* 11(1): 23–27.
21. Guo F, Jiang W (2014) Single particle cryo-electron microscopy and 3-D reconstruction of viruses. *Methods Mol Biol* 1117: 401–443.

---

22. Scheres SH, Chen S (2012) Prevention of overfitting in cryo-EM structure determination. *Nat Methods* 9(9): 853–854.
23. Henderson R, et al. (2012) Outcome of the first electron microscopy validation task force meeting. *Structure* 20(2): 205–214.
24. Rosenthal PB, Henderson R (2003) Optimal determination of particle orientation, absolute hand, and contrast loss in single-particle electron cryomicroscopy. *J Mol Biol* 333(4): 721–745.
25. Trabuco LG, Villa E, Mitra K, Frank J, Schulten K (2008) Flexible fitting of atomic structures into electron microscopy maps using molecular dynamics. *Structure* 16(5): 673–683.
26. Pietilä MK, et al. (2013) Structure of the archaeal head-tailed virus HSTV-1 completes the HK97 fold story. *Proc Natl Acad Sci USA* 110(26): 10604–10609.
27. Baker ML, et al. (2013) Validated near-atomic resolution structure of bacteriophage epsilon 15 derived from cryo-EM and modeling. *Proc Natl Acad Sci USA* 110(30): 12301–6.
28. Helgstrand C, et al. (2003) The refined structure of a protein catenane: The HK97 bacteriophage capsid at 3.44 Å resolution. *J Mol Biol* 334(5): 885–899.
29. Wikoff WR, et al. (2000) Topologically linked protein rings in the bacteriophage HK97 capsid. *Science* 289(5487): 2129–2133.
30. Baker ML, Jiang W, Rixon FJ, Chiu W (2005) Common ancestry of herpesviruses and tailed DNA bacteriophages. *J Virol* 79(23): 14967–14970.
31. Huang RK, et al. (2011) The Prohead-I structure of bacteriophage HK97: Implications for scaffold-mediated control of particle assembly and maturation. *J Mol Biol* 408(3): 541–554.
32. Adams PD, et al. (2010) PHENIX: A comprehensive Python-based system for macromolecular structure solution. *Acta Crystallogr D Biol Crystallogr* 66(Pt 2): 213–221.
33. Cortines JR, Weigele PR, Gilcrease EB, Casjens SR, Teschke CM (2011) Decoding bacteriophage P22 assembly: Identification of two charged residues in scaffolding protein responsible for coat protein interaction. *Virology* 421(1): 1–11.
34. Morais MC, et al. (2003) Bacteriophage phi29 scaffolding protein gp7 before and after prohead assembly. *Nat Struct Biol* 10(7): 572–576.
35. Sun Y, et al. (2000) Structure of the coat protein-binding domain of the scaffolding protein from a double-stranded DNA virus. *J Mol Biol* 297(5): 1195–1202.
36. Serwer P (1976) Internal proteins of bacteriophage T7. *J Mol Biol* 107(3): 271–291.
37. Zhang X, et al. (2013) A new topology of the HK97-like fold revealed in Bordetella bacteriophage by cryoEM at 3.5 Å resolution. *eLife* 2: e01299.
38. Yu G, et al. (2014) Single-step antibody-based affinity cryo-electron microscopy for imaging and structural analysis of macromolecular assemblies. *J Struct Biol* 187(1): 1–9.
39. Hendrix RW, Duda RL (1998) Bacteriophage HK97 head assembly: A protein ballet. *Adv Virus Res* 50: 235–288.
40. Casjens S, King J (1974) P22 morphogenesis. I: Catalytic scaffolding protein in capsid assembly. *J Supramol Struct* 2(2–4): 202–224.
41. Steven AC, Bauer AC, Bisher ME, Robey FA, Black LW (1991) The maturation-dependent conformational change of phage T4 capsid involves the translocation of specific epitopes between the inner and the outer capsid surfaces. *J Struct Biol* 106(3): 221–236.

Guo et al.   |   Published online October 13, 2014   |   E4613
42. Sonntag F, Bleker S, Leuchs B, Fischer R, Kleinschmidt JA (2006) Adeno-associated virus type 2 capsids with externalized VP1/VP2 trafficking domains are generated prior to passage through the cytoplasm and are maintained until uncoating occurs in the nucleus. *J Virol* 80(22): 11040–11054.
43. Kronenberg S, Böttcher B, von der Lieth CW, Bleker S, Kleinschmidt JA (2005) A conformational change in the adeno-associated virus type 2 capsid leads to the exposure of hidden VP1 N termini. *J Virol* 79(9): 5296–5303.
44. Jiménez-Clavero MA, Escribano-Romero E, Douglas AJ, Ley V (2001) The N-terminal region of the VP1 protein of swine vesicular disease virus contains a neutralization site that arises upon cell attachment and is involved in viral entry. *J Virol* 75(2): 1044–1047.
45. Qin L, Fokine A, O'Donnell E, Rao VB, Rossmann MG (2010) Structure of the small outer capsid protein, Soc: A clamp for stabilizing capsids of T4-like phages. *J Mol Biol* 395(4): 728–741.
46. Fokine A, et al. (2004) Molecular architecture of the prolate head of bacteriophage T4. *Proc Natl Acad Sci USA* 101(16): 6003–6008.
47. Lander GC, et al. (2008) Bacteriophage lambda stabilization by auxiliary protein gpD: Timing, location, and mechanism of attachment determined by cryo-EM. *Structure* 16(9): 1399–1406.
48. Koskella B, Brockhurst MA (2014) Bacteria-phage coevolution as a driver of ecological and evolutionary processes in microbial communities. *FEMS Microbiol Rev* 38(5): 916–931.
49. Kropinski AM, et al. (2007) The genome of epsilon15, a serotype-converting, Group E1 Salmonella enterica-specific bacteriophage. *Virology* 369(2): 234–244.
50. Miller ES, et al. (2003) Bacteriophage T4 genome. *Microbiol Mol Biol Rev* 67(1): 86–156.
51. Vander Byl C, Kropinski AM (2000) Sequence of the genome of Salmonella bacteriophage P22. *J Bacteriol* 182(22): 6472–6481.
52. Juhala RJ, et al. (2000) Genomic sequences of bacteriophages HK97 and HK022: Pervasive genetic mosaicism in the lambdoid bacteriophages. *J Mol Biol* 299(1): 27–51.
53. Duerkop BA, Palmer KL, Horsburgh MJ (2014) Enterococcal bacteriophages and genome defense. *Enterococci: From Commensals to Leading Causes of Drug Resistant Infection*, eds Gilmore MS, Clewell DB, Ike Y, Shankar N (Massachusetts Eye and Ear Infirmary, Boston).

54. Mojica FJ, Díez-Villaseñor C, Soria E, Juez G (2000) Biological significance of a family of regularly spaced repeats in the genomes of Archaea, Bacteria and mitochondria. *Mol Microbiol* 36(1): 244–246.
55. Duda RL, et al. (2009) Structure and energetics of encapsidated DNA in bacteriophage HK97 studied by scanning calorimetry and cryo-electron microscopy. *J Mol Biol* 391(2): 471–483.
56. Effantin G, Boulanger P, Neumann E, Letellier L, Conway JF (2006) Bacteriophage T5 structure reveals similarities with HK97 and T4 suggesting evolutionary relationships. *J Mol Biol* 361(5): 993–1002.
57. Smith DE, et al. (2001) The bacteriophage straight phi29 portal motor can package DNA against a large internal force. *Nature* 413(6857): 748–752.
58. Kivioja T, Ravantti J, Verkhovsky A, Ukkonen E, Bamford D (2000) Local average intensity-based method for identifying spherical particles in electron micrographs. *J Struct Biol* 131(2): 126–134.
59. Ludtke SJ, Baldwin PR, Chiu W (1999) EMAN: Semiautomated software for high-resolution single-particle reconstructions. *J Struct Biol* 128(1): 82–97.
60. Jiang W, Guo F, Liu Z (2012) A graph theory method for determination of cryo-EM image focuses. *J Struct Biol* 180(2): 343–351.
61. Yang C, et al. (2009) Estimating contrast transfer function and associated parameters by constrained non-linear optimization. *J Microsc* 233(3): 391–403.
62. Yan X, Dryden KA, Tang J, Baker TS (2007) Ab initio random model method facilitates 3D reconstruction of icosahedral particles. *J Struct Biol* 157(1): 211–225.
63. Liu X, Jiang W, Jakana J, Chiu W (2007) Averaging tens to hundreds of icosahedral particle images to resolve protein secondary structure elements using a Multi-Path Simulated Annealing optimization algorithm. *J Struct Biol* 160(1): 11–27.
64. Tang G, et al. (2007) EMAN2: An extensible image processing suite for electron microscopy. *J Struct Biol* 157(1): 38–46.
65. Emsley P, Cowtan K (2004) Coot: Model-building tools for molecular graphics. *Acta Crystallogr D Biol Crystallogr* 60(Pt 12 Pt 1): 2126–2132.
66. Pettersen EF, et al. (2004) UCSF Chimera—a visualization system for exploratory research and analysis. *J Comput Chem* 25(13): 1605–1612.
67. Collaborative Computational Project, Number 4 (1994) The CCP4 suite: Programs for protein crystallography. *Acta Crystallogr D Biol Crystallogr* 50(Pt 5): 760–763.
68. Chen VB, et al. (2010) MolProbity: All-atom structure validation for macromolecular crystallography. *Acta Crystallogr D Biol Crystallogr* 66(Pt 1): 12–21.
