
LETTER

Hepatitis A virus and the origins of picornaviruses

Xiangxi Wang${}^{1*}$, Jingshan Ren${}^{2*}$, Qiang Gao${}^{1,3*}$, Zhongyu Hu${}^{4}$, Yao Sun${}^{1}$, Xuemei Li${}^{1}$, David J. Rowlands${}^{5}$, Weidong Yin${}^{3}$, Junzhi Wang${}^{4}$, David I. Stuart${}^{2,6}$, Zihe Rao${}^{1,7,8}$ & Elizabeth E. Fry${}^{2}$

Hepatitis A virus (HAV) remains enigmatic, despite 1.4 million cases worldwide annually${}^{1}$. It differs radically from other picornaviruses, existing in an enveloped form${}^{2}$ and being unusually stable, both genetically and physically${}^{3}$, but has proved difficult to study. Here we report high-resolution X-ray structures for the mature virus and the empty particle. The structures of the two particles are indistinguishable, apart from some disorder on the inside of the empty particle. The full virus contains the small viral protein VP4, whereas the empty particle harbours only the uncleaved precursor, VP0. The smooth particle surface is devoid of depressions that might correspond to receptor-binding sites. Peptide scanning data extend the previously reported VP3 antigenic site${}^{4}$, while structure-based predictions${}^{5}$ suggest further epitopes. HAV contains no pocket factor and can withstand remarkably high temperature and low pH, and empty particles are even more robust than full particles. The virus probably uncoats via a novel mechanism, being assembled differently to other picornaviruses. It utilizes a VP2 ‘domain swap’ characteristic of insect picorna-like viruses${}^{6,7}$, and structure-based phylogenetic analysis places HAV between typical picornaviruses and the insect viruses. The enigmatic properties of HAV may reflect its position as a link between ‘modern’ picornaviruses and the more ‘primitive’ precursor insect viruses; for instance, HAV retains the ability to move from cell-to-cell by transcytosis${}^{8,9}$.

HAV is unique among picornaviruses in targeting the liver and continues to be a source of mortality despite a successful vaccine${}^{10}$. HAV isolates belong to a single serotype${}^{11}$. Unlike other picornaviruses HAV cannot shut down host protein synthesis, has a highly deoptimized codon usage and grows poorly in tissue culture. Particles are produced with a 67-residue carboxy-terminal extension of VP1, which is implicated in particle assembly (this longer form of VP1 is known as VP1-2A or PX)${}^{12}$. Particles containing the extension shroud themselves in host membrane to create enveloped viruses${}^{2}$. The extension is cleaved by host proteases to yield mature capsids${}^{12}$. While picornavirus VP4 is generally myristoylated this does not happen in HAV${}^{13}$; indeed, the putative VP4 is very small ($\sim$23 residues${}^{13}$) and it has remained unclear if it is present in virus particles${}^{14}$. The cell surface molecule T cell immunoglobulin and mucin 1 (TIM-1)${}^{15}$ acts as a receptor for HAV, and although transcytosis occurs${}^{8,9}$ it is not clear how the virus gets to the liver, its principal site of replication.

We have analysed formaldehyde-inactivated HAV genotype TZ84 (Methods). Two types of particle were separated, one containing significant amounts of viral RNA (Extended Data Fig. 1). In the RNA-containing full particles VP0 is at least partially cleaved and we detect VP4, as for other picornaviruses, whereas the empty particles harbour only VP0, and are probably similar to the empty particles frequently seen in picornavirus infections (Methods and Extended Data Fig. 1). It remains unclear whether such empty particles can encapsidate RNA and lie on the route to assembly of full particles. Even full particles appear to contain more uncleaved VP0 than is seen in other picornaviruses, in line with observations that VP0 cleavage is ‘protracted’${}^{16}$. The sedimentation coefficients are approximately 144S and 82S for the full and empty particles, respectively (Extended Data Fig. 1). 144S is a little less than the 155S expected for a full enterovirus particle, while the empty particle has a similar S value to that observed for the more expanded empty enterovirus particles${}^{17}$.

Very thin crystals ($\sim$100 $\times$ 100 $\times$ 5 $\mu$m${}^{3}$) were obtained for both particles. Diffraction data were collected at Diamond beamlines I03 and I24. Data were collected at 100K to avoid beam-induced crystal movement at room temperature (Supplementary Video 1), and were used to produce reliable atomic models at 3.0 and 3.5 Å resolution for the full and empty particles, respectively (Methods and Extended Data Table 1).

The external surface of HAV is smooth, with no canyon (Fig. 1a)${}^{18}$; shortening of the VP1 BC loop lowers the north wall while reductions in the VP2 EF and VP1 GH loops ablate the south wall of the canyon (Fig. 1b). Compared to foot-and-mouth-disease virus, the loops at the five-fold and three-fold axes in HAV are slightly raised, giving the virus the appearance of a faceted triakis icosahedron (Fig. 1a). In line with the low buoyant density in CsCl (ref. 19) there are no apertures in the capsid to permit the entry of Cs${}^{+}$ ions. The major capsid proteins, VP1–3,

![Figure 1](#fig1)

**Figure 1 | Overall structure.** a, HAV accessible surface (VP1, blue; VP2, green; VP3, red for all panels). Black lines, particle facets; white outline, biological protomer. b, Surface of the biological protomer of HAV and poliovirus. Loops forming the canyon walls in poliovirus are drawn thicker. c, HAV electrostatic surface (calculated using APBS in PyMOL). Red, negative; blue, positive; white, neutral; sulphate ions, yellow. d, HAV viewed from inside. Blue, positive $|F_{o}-F_{c}|$ electron density calculated taking the correctly positioned empty HAV from the full HAV shows that VP1 2–28 (darker density) and VP2 5–17 are better defined in the full particle.

${}^{1}$National Laboratory of Macromolecules, Institute of Biophysics, Chinese Academy of Science, Beijing 100101, China. ${}^{2}$Division of Structural Biology, University of Oxford, The Henry Wellcome Building for Genomic Medicine, Headington, Oxford OX3 7BN, UK. ${}^{3}$Sinovac Biotech Co., Ltd, Beijing 100085, China. ${}^{4}$National Institutes for Food and Drug Control, No. 2, TiantanXili, Beijing 100050, China. ${}^{5}$Institute of Molecular and Cellular Biology and Astbury Centre for Structural Molecular Biology, Faculty of Biological Sciences, University of Leeds, Leeds LS2 9JT, UK. ${}^{6}$Diamond Light Sources, Harwell Science and Innovation Campus, Didcot OX11 0DE, UK. ${}^{7}$Laboratory of Structural Biology, School of Medicine, Tsinghua University, Beijing 100084, China. ${}^{8}$State Key Laboratory of Medicinal Chemical Biology, Nankai University, Tianjin 300071, China.

*These authors contributed equally to this work.

comprise eight-stranded anti-parallel β-barrels, follow the expected pseudo $T = 3$ arrangement (where $T$ is the triangulation number) and span the thickness of the capsid (Extended Data Fig. 2a–c). The HAV full virus is mostly well ordered (however, VP4, although present, is not visible). As expected there is no evidence for structural modification attributable to formaldehyde inactivation ${ }^{17}$.

The empty and full particles are mostly very similar, with the external surface expected to be antigenically indistinguishable (r.m.s.d. for 672 Cα atoms 0.2 Å). Experiments with six antibodies confirmed this (Extended Data Fig. 3). The particle surface is remarkably negatively charged (Fig. 1c) and the fringes of the pentameric assemblies, which show some positive charge, are decorated with a string of sulphate ions derived from the crystallization media. Surface decoration may affect the hydrodynamic properties of the particles. In the empty particle the first 40 residues of VP0 and 47 residues of VP1, which adorn the particle interior near the three-fold axes, are disordered (Fig. 1d). Although neither particle contains the extended form of VP1 we can infer the point on the particle surface from which the extension would continue (Extended Data Fig. 2d).

Perhaps correlated with the lack of a canyon, HAV harbours no contiguous hydrophobic pocket in the VP1 β-barrel. The β-barrel is compressed compared to enteroviruses ${ }^{20}$ and the remaining space is largely filled with hydrophobic side chains, as in aphthoviruses and cardioviruses ${ }^{21}$ (Fig. 2a–c). In addition, lengthened βC and βH strands essentially cover what would be the entrance to the pocket. We conclude that HAV is unable to bind small molecules in the fashion characteristic of enteroviruses. Unfortunately the particle structure provides no immediate clues as to where the TIM-1 receptor ${ }^{15}$ might attach.

The organization of the protein chains broadly mirrors other picornaviruses, except for the amino terminus of VP2. At residue 53 there is a flip in the $\psi$ torsion angle of the peptide, sending the first 53 residues of VP2 across and then along the interpentamer boundary to interact strongly with the neighbouring pentamer (forming an extra strand on the VP2 β-barrel from the adjacent pentamer; Fig. 2d–g). The switch is only $\sim 5 \AA$ from the icosahedral two-fold axis, so that the net effect is to swap the N-terminal structures, in a way reminiscent of domain swaps observed in certain protein structures ${ }^{22}$. As in those cases, it produces a very similar structure overall, but alters the subunit connectivity so that in HAV adjacent protomers of one pentamer are sewn together via the adjacent pentamer (Fig. 2f, g). This arrangement is unprecedented in picornaviruses; however, it occurs in insect picorna-like viruses (for example, cricket paralysis virus (CrPV ${ }^{6}$ ); Fig. 2d).

We found that both empty and full particles were extraordinarily robust compared to other picornaviruses (remaining stable at up to $80^{\circ} \mathrm{C}$ and at pH values down to about 2 (ref. 23); Fig. 3a and Extended Data Fig. 4). It is established for many picornaviruses that the interface between the 12 pentameric assemblies that comprise the icosahedral capsid determine the particle stability ${ }^{24-26}$. In this context we questioned whether the stability of HAV might be a consequence of the VP2 domain swap at the pentamer interface, and so analysed the stability of CrPV (Extended Data Fig. 4d). Although harbouring the same domain swap, CrPV particles showed a stability profile similar to a typical picornavirus. The VP2 switch therefore does not explain the stability; however, the interaction region adjacent to the icosahedral two-fold axes, which separates during the initial stages of enterovirus uncoating ${ }^{17,27}$, shows tight packing in HAV (Fig. 3b and Extended Data Fig. 5). The complementarity is similar

Figure 2 | Structure features. a, b, HAV (a) and EV71 (b) are coloured as in Fig. 1a; light-blue mesh: pocket volume calculated with PyMOL. Magenta, EV71 pocket factor. c, Pocket close-up of strand C (left) and strand H (right) for HRV14 (yellow), FMDV (blue), EV71 (orange, pocket factor magenta) and HAV (green, note occlusion of pocket entrance). Met 224 occludes part of the empty HRV14 pocket. A and B: bulky side chains occlude the HAV and FMDV pockets (A, Leu-HAV, Tyr-FMDV; B, Phe-HAV and Tyr-FMDV). d, e, Biological protomers of CrPV (d) and FMDV (e) superimposed on HAV. The asterisk marks the VP2 N terminus that folds differently in FMDV (switch at residue 53 is indicated). HAV coloured as in a (CrPV and FMDV grey). f, HAV compared with poliovirus type 1 (1HXS) in g. The pentamer interface runs horizontally across the centre with the perpendicular two-fold axis roughly central. Surface drawn for upper pentamer. Two-fold related β-sheets are at A and B. VP1, blue and indigo; VP2, green and lime-green; VP3, red. VP4 is omitted for clarity. Upper pentamer chains are drawn thicker.

host factor (perhaps a protease) may be required for particle disassembly. A recent report suggests that HAV harbours tandem YPX₃L motifs (VP2: Y₁₄₄PHGLL₁₄₉ and Y₁₇₇PVWEL₁₈₂), which are suggested to bind the ALIX component of the ESCRT pathway, allowing the virus to engage the ESCRT complex to facilitate release of enveloped particles via exocytosis². Surprisingly, the residues implicated are buried, making their role unclear unless major conformational rearrangements render them accessible, or if they are accessible on a precursor particle.

The antigenicity of HAV is incompletely characterized, with residues S102, V171, A176 and K221 of VP1 and D70, S71, Q74 and 102–121 of VP3 implicated in neutralizing epitopes⁴. All except K221 of VP1 are proposed to form part of a single site; however, they are separated by 40–50 Å on the particle surface (Extended Data Fig. 6). To extend our knowledge, 12 monoclonal antibodies (mAbs) against HAV particles were generated (Extended Data Fig. 7), one of which, mAb 11, combined excellent neutralizing titre with VP3 western blot activity. On the basis of the particle structure a series of peptides covering all exposed virus surface regions were synthesized and tested against mAb 11. VP3 residues 67–77 and 208–219 (Extended Data Fig. 7) reacted mildly. The proximity of these regions on the surface of VP3 (Extended Data Fig. 6) suggests that both may contribute to a conformational epitope, an augmented version of that identified previously⁴. To identify further antigenic sites on HAV we applied structure-based predictive methodology⁵, which supports the above findings and suggests that VP2 residues 71 and 198 and VP3 residues 89–96 may comprise additional epitopes (Extended Data Fig. 6).

HAV possesses subtle but profound structural differences compared to previously characterized picornaviruses. A structure-based phylogeny (Fig. 4a) suggests that HAV links classical picornaviruses and the insect picorna-like viruses. Furthermore, the N-terminal domain swap renders VP2 more similar to the homologous VP1 and VP3 proteins (Fig. 4b), supporting the notion that HAV retains structural and functional features characteristic of primordial picornaviruses, which were akin to present-day insect picorna-like viruses, and that the subsequent acquisition of efficient mechanisms of cell entry allowed the explosion in diversity that characterizes present-day mammalian picornaviruses.

Online Content Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.

Received 12 May; accepted 28 August 2014.
Published online 19 October 2014.

1. WHO. Hepatitis A. Fact Sheet No. 328 http://www.who.int/mediacentre/factsheets/fs328/en/ (2013).
2. Feng, Z. *et al.* A pathogenic picornavirus acquires an envelope by hijacking cellular membranes. *Nature* **496**, 367–371 (2013).
3. Siegl, G., Weitz, M. & Kronauer, G. Stability of hepatitis A virus. *Intervirology* **22**, 218–226 (1984).
4. Ping, L.H. & Lemon, S.M. Antigenic structure of human hepatitis A virus defined by analysis of escape mutants selected against murine monoclonal antibodies. *J. Virol.* **66**, 2208–2216 (1992).
5. Borley, D.W. *et al.* Evaluation and use of in-silico structure-based epitope prediction with foot-and-mouth disease virus. *PLoS ONE* **8**, e61122 (2013).
6. Tate, J. *et al.* The crystal structure of cricket paralysis virus: the first view of a new virus family. *Nature Struct. Biol.* **6**, 765–774 (1999).
7. Squires, G. *et al.* Structure of the Triatoma virus capsid. *Acta Crystallogr. D.* **69**, 1026–1037 (2013).
8. Dotzauer, A., Brenner, M., Gebhardt, U. & Vallbracht, A. IgA-coated particles of Hepatitis A virus are translocalized antivectorially from the apical to the basolateral site of polarized epithelial cells via the polymeric immunoglobulin receptor. *J. Gen. Virol.* **86**, 2747–2751 (2005).
9. Dotzauer, A. *et al.* Hepatitis A virus-specific immunoglobulin A mediates infection of hepatocytes with hepatitis A virus via the asialoglycoprotein receptor. *J. Virol.* **74**, 10950–10957 (2000).
10. Deinhardt, F. Prevention of viral hepatitis A: past, present and future. *Vaccine* **10** (suppl. 1), 10–14 (1992).
11. Lemon, S.M. & Binn, L.N. Antigenic relatedness of two strains of hepatitis A virus determined by cross-neutralization. *Infect. Immun.* **42**, 418–420 (1983).
12. Graff, J. *et al.* Hepatitis A virus capsid protein VP1 has a heterogeneous C terminus. *J. Virol.* **73**, 6015–6023 (1999).
13. Tesar, M., Jia, X.Y., Summers, D.F. & Ehrenfeld, E. Analysis of a potential myristoylation site in hepatitis A virus capsid protein VP4. *Virology* **194**, 616–626 (1993).

14. Martin, A. & Lemon, S. M. Hepatitis A virus: from discovery to vaccines. *Hepatology* **43**, S164–S172 (2006).
15. Kaplan, G. *et al.* Identification of a surface glycoprotein on African green monkey kidney cells as a receptor for hepatitis A virus. *EMBO J.* **15**, 4282–4296 (1996).
16. Bishop, N. E. & Anderson, D. A. RNA-dependent cleavage of VP0 capsid protein in provirions of hepatitis A virus. *Virology* **197**, 616–623 (1993).
17. Wang, X. *et al.* A sensor-adaptor mechanism for enterovirus uncoating from structures of EV71. *Nature Struct. Mol. Biol.* **19**, 424–429 (2012).
18. Rossmann, M. G. *et al.* Structure of a human common cold virus and functional relationship to other picornaviruses. *Nature* **317**, 145–153 (1985).
19. Siegl, G., Frosner, G. G., Gauss-Müller, V., Tratschin, J. D. & Deinhardt, F. The physicochemical properties of infectious hepatitis A virions. *J. Gen. Virol.* **57**, 331–341 (1981).
20. De Colibus, L. *et al.* More-powerful virus inhibitors from structure-based analysis of HEV71 capsid-binding molecules. *Nature Struct. Mol. Biol.* **21**, 282–288 (2014).
21. Acharya, R. *et al.* The three-dimensional structure of foot-and-mouth disease virus at 2.9 Å resolution. *Nature* **337**, 709–716 (1989).
22. Liu, Y. & Eisenberg, D. 3D domain swapping: as domains continue to swap. *Protein Sci.* **11**, 1285–1299 (2002).
23. Walter, T. S. *et al.* A plate-based high-throughput assay for virus stability and vaccine formulation. *J. Virol. Methods* **185**, 166–170 (2012).
24. Porta, C. *et al.* Rational engineering of recombinant picornavirus capsids to produce safe, protective vaccine antigen. *PLoS Pathog.* **9**, e1003255 (2013).
25. Filman, D. J. *et al.* Structural factors that control conformational transitions and serotype specificity in type 3 poliovirus. *EMBO J.* **8**, 1567–1579 (1989).
26. Warwicker, J. Model for the differential stabilities of rhinovirus and poliovirus to mild acidic pH, based on electrostatics calculations. *J. Mol. Biol.* **223**, 247–257 (1992).
27. Garriga, D. *et al.* Insights into minor group rhinovirus uncoating: the X-ray structure of the HRV2 empty capsid. *PLoS Pathog.* **8**, e1002473 (2012).

28. Ren, J. *et al.* Picornavirus uncoating intermediate captured in atomic detail. *Nature Commun.* **4**, 1929 (2013).
29. Butan, C., Filman, D. J. & Hogle, J. M. Cryo-electron microscopy reconstruction shows poliovirus 135S particles poised for membrane interaction and RNA release. *J. Virol.* **88**, 1758–1770 (2014).
30. Riffel, N. *et al.* Atomic resolution structure of Moloney murine leukemia virus matrix protein and its relationship to other retroviral matrix proteins. *Structure* **10**, 1627–1636 (2002).

**Supplementary Information** is available in the online version of the paper.

**Acknowledgements** This work was supported by the Strategic Priority Research Program of the Chinese Academy of Sciences, grant no. XDB08020200, the Ministry of Science and Technology 973 Project (grant no. 2014CB542800), the National Major Project of Infectious Disease (2012ZX10004701) and National Science Foundation grant 81330036. D.I.S. and E.E.F. are supported by the UK Medical Research Council (grant G1000099) and J.R. by the Wellcome Trust. This work is a contribution from the Instruct Centre, Oxford. Administrative support was provided by the Wellcome Trust (075491/Z/04). We thank J. Johnson and A. Routh for supplying CrPV, and S. Lemon and K. McKnight for discussions.

**Author Contributions** Q.G., J.W., Z.H. and W.Y. prepared samples. X.W., J.R., Q.G., X.L., Y.S., E.E.F. and D.I.S. performed research. J.R., X.W., E.E.F. and D.I.S. analysed data and with D.J.R. and Z.R. wrote the manuscript, in discussion with J.W. and Z.H.

**Author Information** The coordinates and structure factors for the full and empty particles have been deposited with the RCSB Protein Data Bank under accession codes 4QPI and 4QPG, respectively. Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to D.I.S. (dave@strubi.ox.ac.uk), Z.R. (raozh@xtal.tsinghua.edu.cn) or J.W. (wangjz@nicpbp.org.cn).

model (Cαs of a pentamer, Protein Data Bank accession 1FMD) followed by non-
crystallographic symmetry (NCS) averaging and refinement using strict NCS
constraints³³. The full virus was centred at (194.1, 106.1, 85.6) and the empty
particle at (0, 221.5, 6.0). For each space group the content of the crystallographic
asymmetric unit was subjected to rigid-body refinement, followed by cyclic posi-
tional, simulated annealing and B-factor refinement using strict NCS constraints
with CNS³³. Maps were averaged used GAP (D.I.S., J. Grimes and J. Diprose,
unpublished) and models were rebuilt with COOT³⁴. Refinement of the full par-
ticle structure stuck at an R-factor of 31.8%, probably attributable to imperfect
isomorphism of the constituent crystals and to the extreme weakness of the data.
For statistics, see Extended Data Table 1. Structural comparisons used SHP³⁵.
Unless otherwise noted structural figures were prepared with PyMOL³⁶.

**Production of monoclonal antibody and neutralization assay.** Groups of 15
adult (4 weeks old) female BALB/c mice were immunized intraperitoneally with
10 μg inactivated HAV virus. All animal procedures were carried out in accordance
with the guideline for the Use of Animals in Research issued by the Institute of Bio-
physics, Chinese Academy of Sciences. Two booster doses were given at 2 weekly
intervals. Two weeks after the last immunization, blood samples were obtained from
the tail and tested by enzyme-linked immunosorbent assay (ELISA) using purified
HAV virus as antigen. Classical protocols for production of monoclonal antibody
were followed (continuous cultures of fused cells secreting antibody of predefined
specificity). For the neutralization assay, purified monoclonal antibodies at a con-
centration of 0.2 mg ml⁻¹ were initially diluted eightfold as stocks, and then seri-
ally diluted twofold with DMEM containing 2% FBS. 100 μl of twofold antibody
dilutions were mixed with 100 μl of HAV virus containing 100 TCID₅₀ for 1 h at
37 °C, and then added to monolayers of 2BS cells in cell culture flasks (T25 CM²),
meanwhile, maintaining medium was provided as well. Each dilution was replicated
three times along with one control that contained no serum dilution. After 21 days
of growth at 33 °C, the medium was removed and the cells washed three times using
PBS buffer, 1 ml of trypsin/EDTA was added and the flask left for 3 min at 37 °C.
The suspended cells were freeze-thawed five times to collect the virus. Enzyme-
linked immunosorbent assay (ELISA) was used to determine neutralizing titre. The
titres were read as the highest dilution that gave complete protection.

**Peptide-ELISA assay.** Synthetic peptides spanning the entire exposed region of
HAV VP1, VP2 and VP3 capsid proteins were synthesized by Scilight-Peptide
Company (Beijing, China). The reactivity of synthetic peptides with purified HAV
neutralizing monoclonal antibody was measured by ELISA. Briefly, 96-well plates
were coated with 100 μl well⁻¹ of individual peptide (10 μg ml⁻¹ in PBS buffer) at
4 °C overnight. An unrelated EV71 peptide was used as a negative control and inac-
tivated HAV as a positive control. The wells were then incubated sequentially with
100 μl well⁻¹ of PBST plus 5% skimmed milk powder at 37 °C for 1 h, and 100 μl
well⁻¹ of HAV neutralizing monoclonal antibodies at the indicated dilutions were
incubated at 37 °C for 1 h. Horseradish peroxidase (HRP) conjugated goat anti-
mouse IgG diluted (1:5,000) in PBST plus 1% milk powder was used as secondary
antibody at 37 °C for 1 h. Five washes with PBST were carried out between incuba-
tion steps. For colour development, 100 μl well⁻¹ of TMB mixture was added and
incubated for 10 min, followed by addition of 50 μl well⁻¹ of 1M H₃PO₄ to stop the
reaction. Absorbance was measured at 450 nm in a 96-well plate reader.

**Crystallization and data collection.** Full and empty particles were concentrated
to 1.5 and 3 mg ml⁻¹ respectively before crystallization in Greiner CrystalQuickX plates
using a version of the previously described nanolitre vapour diffusion procedure³¹
with 30% MPD and 0.5 M ammonium sulphate as precipitants and 0.1 M HEPES-
Na buffer at pH 7.5. Crystals (rectangular plates of dimensions ~100 × 100 × 5 μM)
were cryo-protected using solutions containing up to 50% MPD and 20% glycerol
depending on the crystal age. Data were collected from frozen crystals (100 K) on
beamlines I03 and I24 at Diamond Light Source UK. Diffraction images of 0.1°
rotation were recorded on a Pilatus6M detector using beam size from 0.02 × 0.02 mm²
to 0.08 × 0.06 mm², commensurate with crystal size. Using 1.0-s exposure time
and 100% beam transmission, typically 50 useful images could be collected from
one position of a crystal. For some larger crystals, data could be collected from two
positions. A total of some 200 crystals of both full and empty particles were frozen
and tested. 17 empty particle crystals diffracted and gave a data set to 3.5 Å reso-
lution with 68.0% completeness; 32 full virus crystals resulted in a data set to 3.0 Å
resolution with 44.4% completeness. Although the data suffered from the large mosaic
spread of the crystals, the minimal divergence of the X-ray beam on I03 amelio-
rated the problem.

**Structure determination.** Data were analysed using HKL2000³². The mosaicity
differed for the full and empty particle crystals ranging from 0.35° to 0.85°. The
data were inevitably very weak with average *I*/*σI* of 1.4 for the full particle data set
and 3.0 for the empty particle data sets. The structures of both particles were deter-
mined by molecular replacement. The space group for the empty particle was *P*₂₁₂₁₂₁
with unit cell dimensions *a* = 366.1 Å, *b* = 442.9 Å and *c* = 289.0 Å (30 protomers
in a crystallographic asymmetric unit). For the full virus, the unit cell dimensions
are *a* = 291.5 Å, *b* = 423.3 Å, *c* = 314.8 Å and *β* = 100.2°, space group *P*₂₁ with a full
particle in asymmetric unit. Phasing used molecular replacement with an FMDV

---

**REFERENCES**

1. Walter, T. S. *et al.* A procedure for setting up high-throughput nanolitre
   crystallization experiments. Crystallization workflow for initial screening,
   automated storage, imaging and optimization. *Acta Crystallogr. D* **61**, 651–657
   (2005).
2. Minor, W., Cymborowski, M., Otwinowski, Z. & Chruszcz, M. HKL-3000: the
   integration of data reduction and structure solution—from diffraction images to
   an initial model in minutes. *Acta Crystallogr. D* **62**, 859–866 (2006).
3. Brünger, A. T. *et al.* Crystallography & NMR system: A new software suite for
   macromolecular structure determination. *Acta Crystallogr. D* **54**, 905–921
   (1998).
4. Emsley, P., Lohkamp, B., Scott, W. G. & Cowtan, K. Features and development of
   Coot. *Acta Crystallogr. D* **66**, 486–501 (2010).
5. Stuart, D. I., Levine, M., Muirhead, H. & Stammers, D. K. Crystal structure of cat-
   muscle pyruvate kinase at a resolution of 2.6 Å. *J. Mol. Biol.* **134**, 109–142 (1979).
6. PyMOL. The PyMOL Molecular Graphics System, Version 1.5.0.4 Schrödinger,
   LLC. (2002).
7. Schuck, P. Size-distribution analysis of macromolecules by sedimentation
   velocity ultracentrifugation and lamm equation modeling. *Biophys. J.* **78**,
   1606–1619 (2000).
8. Lawrence, M. C. & Colman, P. M. Shape complementarity at protein/protein
   interfaces. *J. Mol. Biol.* **234**, 946–950 (1993).
9. Chapman, M. S. & Rossmann, M. G. Comparison of surface properties of
   picornaviruses: strategies for hiding the receptor site from immune surveillance.
   *Virology* **195**, 745–756 (1993).

©2014 Macmillan Publishers Limited. All rights reserved

Extended Data Figure 1 | HAV purification and characterization.

a, Zonal ultracentrifugation of a 15% to 45% (w/v) sucrose density gradient at 103,614 g for 3.5 h was used to purify HAV, as described in Methods. Two predominant particle types were separated; one was located at ~27% sucrose, the other at ~32% sucrose. The absorbance ratios (λ260/λ280) were 0.76 for the top band and 1.66 for the bottom band, indicating that the former contained mainly empty particles without the RNA genome and the latter mainly full particles with the RNA genome. The top band was much broader than the bottom one. Three and two fractions were collected around the top and the bottom bands, respectively, for further purification. b, SDS–PAGE for protein composition analysis using A NuPAGE 4–12% Bis-Tris Gel (Invitrogen). Each lane was loaded with 5–10 μg sample (lane 1, full particles; lane 2, empty particles; lane 3, markers). The calculated molecular masses of VP0, VP1, VP2, VP3 and VP4 were 27.26 kDa, 30.73 kDa, 24.80 kDa, 27.86 kDa and 2.50 kDa, respectively. The results appear to show incomplete cleavage of VP0 in the full particles and no VP0 cleavage in the empty particles. c, d, Sedimentation velocity experiments were performed on a Beckman XL-I analytical ultracentrifuge at 20 °C. Samples were loaded into a conventional double-sector quartz cell and mounted in a Beckman four-hole An-60 Ti rotor. Data were collected at 15,000 r.p.m. at a wavelength of 287 nm. Interference sedimentation coefficient distributions were calculated from the sedimentation velocity data using SEDFIT³⁷. The full particle has a sedimentation coefficient of 144S and the empty particle 82S. e, f, Negative stain electron microscopy of HAV particles. Particles from the top band in panel a, shown in panel f, suggest that light sedimentation fractions were mainly composed of empty particles. Note that some particles appear to have external features. Panel e shows the heavy particles, which appear to contain viral RNA.

Extended Data Figure 2 | HAV capsid protein structure. **a–c**, Stereo diagrams showing the structures of the capsid proteins VP1, VP2 and VP3, respectively. The Cα backbone is shown as a thin line, the N and C termini are labelled, every 10th residue is marked with a small sphere and every 20th is numbered. **d**, VP1 is initially produced with an 8-kDa C-terminal extension (known as PX) unique to HAV among picornaviruses. PX is cleaved from the full particle by an unknown host protease. According to the data shown in Extended Data Fig. 1 and the electron density maps, the particles we have analysed do not contain PX, but if it were present we would expect its course to start at the purple positions (C termini of VP1) on the surface of the virus.

RESEARCH LETTER

Full  
Empty  

Extended Data Figure 3 | Antigenicity of full and empty particles. The reactivity of HAV full and empty particles against a panel of six purified HAV neutralizing monoclonal antibodies was measured by ELISA. The bar charts represent the average OD<sub>450</sub> reading for the six monoclonal antibodies at each dilution with the standard deviation shown as an error bar.

©2014 Macmillan Publishers Limited. All rights reserved

Extended Data Figure 4 | PaSTRy assays. To characterize the stability of HAV full and empty particles compared to CrPV across the pH range from 2.0 to 10.0, differential scanning fluorimetry assays were performed with dyes SYTO9 (to detect RNA exposure) and SYPRO red (to detect protein melting)²³.

a, The raw fluorescence traces of HAV full particles incubated with SYTO9.
b, The raw fluorescence traces of HAV full particles incubated with SYPRO red.
c, The raw fluorescence traces of HAV empty particles incubated with SYPRO red.
d, The raw fluorescence traces of CrPV full particles incubated with SYTO9 across the same pH range. The colour scheme is dark red (pH 2.0), red (pH 3.0), orange (pH 4.0), yellow (pH 5.0), green (pH 6.0), sky blue (pH 7.0), blue (pH 8.0), dark blue (pH 9.0) and purple (pH 10.0). Since the SYTO9 dye didn’t function well below pH 4.0 the fluorescence traces for pH 2.0 and pH 3.0 are omitted. These results indicate that HAV full virions are most stable at pH 5.0 and RNA genome release occurs at about 76 °C and protein melting at 77 °C; that is, there is no notable transition between RNA release and particle loss. HAV empty particles show a similar trend but appear to withstand temperatures up to 81 °C in low pH buffer. In contrast CrPV is most stable at pH 4.0 and RNA genome release occurs at about 54 °C.

a

b

Extended Data Figure 5 | Interactions between α1 helices of VP2 at the icosahedral two-fold. a, The close-packing of the two helices is shown, in particular the packing of Tyr 100 against the adjacent helix. The contact area between these helices and the surface complementarity³⁸ suggests a well-fitting interface for HAV (contact area 83 Å², surface complementarity 0.755; EV71, 111/0.550; Polio-1, 124/0.785; CrPV, 72/0.707; FMDV, 50/0.593). The interface between protomers forming the pentameric units is similar for all of these viruses (HAV, 4,267 Å²; Polio-1, 4369 Å²; CrPV, 4,647 Å²; EV71, 4,131 Å²; FMDV-A22, 3,432 Å²). b, Helical wheel diagrams for helix α1 of several picornaviruses showing the unusually small side chains at the helix interface in HAV.

Extended Data Figure 6 | Antigenicity and YPX₃L ALIX-interacting motifs.

a, Previously indicated antigenic sites of HAV are mapped onto the structure (pink spheres). Late domain YPX₃L motifs are shown as orange spheres.

b–d, Surface maps of HAV generated using RIVEM³⁹ such that area for each residue as drawn corresponds to its accessible area and the coloration is according to the radius from the virus centre (blue, deepest; red, most exposed). On these the antigenic sites of HAV are depicted in purple. b, Previously reported antigenic sites; c, predicted sites⁵; d, new sites determined by peptide mapping (indigo) together with previously identified sites (purple).

a

| Antibodies | Neutralization filters |
|------------|------------------------|
| #1         | <1:8                  |
| #2         | <1:8                  |
| #3         | 1:64                  |
| #4         | <1:8                  |
| #5         | 1:1024                |
| #6         | 1:1024                |
| #7         | 1:4096                |
| #8         | <1:8                  |
| #9         | 1:1024                |
| #10        | 1:512                 |
| #11        | 1:1024                |
| #12        | <1:8                  |
| A9(EV71mAb)| <1:8                  |

b

| Peptide name | Residues     | Sequence                     |
|--------------|--------------|------------------------------|
| PEP1         | VP1: 260-272 | APLNSNAMLSTES                |
| PEP2         | VP1: 111-125 | TLSSTSNNPHGLPST              |
| PEP3         | VP1: 159-178 | TPVGLAVDTPWVEKESALSI         |
| PEP4         | VP1: 232-245 | QIANYNHSDEYLSF               |
| PEP5         | VP1: 146-155 | IITGATDVVDG                  |
| PEP6         | VP1: 97-108  | TTFNSNNKEYT                  |
| PEP7         | VP1: 206-225 | SYLYAVSGALDGLGDKTDST         |
| PEP8         | VP2: 130-146 | PGDQSYGSIASLTVYPH            |
| PEP9         | VP2: 162-179 | PFIYTRGAYHFKDPQYPV           |
| PEP10        | VP2: 188-200 | SELNIGTGTSAYT                |
| PEP11        | VP2: 69-78   | LTTHALFHEV                   |
| PEP12        | VP3: 67-77   | NASDSVGQQIK                  |
| PEP13        | VP3: 81-99   | VDPYFFQMTNTNPDQKCIT          |
| PEP14        | VP3: 137-153 | PGNELIDVSGITLKQAT            |
| PEP15        | VP3: 177-198 | SDTPYRVNRYTKEAHQKGEYTA       |
| PEP16        | VP3: 233-246 | CFAPLYHAMDVTTQ               |
| PEP17        | VP3: 208-219 | NRLTSPSNVAHH                 |
| PEP18        | VP3: 110-121 | FWRGDLVFDFQV                 |
| EV71_P       | VP1: 211-225 | FGEHKQEKDLEYGAC              |

c

Extended Data Figure 7 | Monoclonal antibody neutralization assays and peptide epitope mapping.  
a, In vitro neutralization assays of monoclonal antibodies against HAV (TZ84).  
b, Peptides used for epitope mapping.  
c, Reactivity of the neutralizing mAb 11 against synthetic peptides measured by peptide-ELISA.

Extended Data Table 1 | Data collection, phasing and refinement statistics

| Data collection | Empty particle | Full particle |
|-----------------|----------------|--------------|
| Beam line       | Diamond I03 and I24 |             |
| particle        | empty          | full         |
| Temperature (K) | 100            | 100          |
| Space group     | P2₁2₁2         | P2₁          |
| No. of crystals (positions) | 17(29) | 32(36) |
| Cell dimensions |                |              |
| $a$, $b$, $c$ (Å) | $a=366.1$, $b=442.9$, $c=289.0$ | $a=291.5$, $b=423.3$, $c=314.8$, $\beta=100.2^\circ$ |
| Resolution (Å)  | 50.0–3.50 (3.56–3.50) | 50.0–3.00 (3.05–3.00) |
| Unique reflections | 404893(2925) | 654559(6711) |
| $R_{\text{merge}}$ | 0.363(---) | 0.410(0.933) |
| $I/\sigma I$ | 3.0(0.3) | 1.4(0.5) |
| Completeness (%) | 68.0(9.9) | 44.4(9.1) |
| Redundancy | 2.5(1.1) | 1.9(1.1) |

| Refinement |               |              |
|------------|---------------|--------------|
| Resolution (Å) | 50.0–3.50 | 50.0–3.00 |
| No. reflections | 392844/3880 | 643384/6633 |
| $R_{\text{work}}$/$R_{\text{free}}$* | 0.264/0.263 | 0.318/0.321 |
| No. atoms | 5358 | 5742 |
| Mean B-factors (Å²) | 179 | 37 |
| r.m.s. deviations |           |              |
| Bond lengths (Å) | 0.004 | 0.008 |
| Bond angles (°) | 1.0 | 1.4 |

Note that the $R_{\text{free}}$ is of limited significance owing to the considerable non-crystallographic symmetry, 30-fold and 60-fold for the empty and full particles, respectively.
