
Antibiotics 2014, 3, 461-492; doi:10.3390/antibiotics3040461

OPEN ACCESS

**antibiotics**

ISSN 2079-6382  
www.mdpi.com/journal/antibiotics

---

**Review**

**Antimicrobial Peptide Resistance Mechanisms of Gram-Positive Bacteria**

Kathryn L. Nawrocki †, Emily K. Crispell † and Shonna M. McBride *

Department of Microbiology and Immunology, Emory University School of Medicine, 1510 Clifton Rd, Atlanta, GA 30322, USA; E-Mails: knawroc@emory.edu (K.L.N.); ecrispell@emory.edu (E.K.C.)

† These authors contributed equally to this work.

* Author to whom correspondence should be addressed; E-Mail: shonna.mcbride@emory.edu; Tel.: +1-404-727-6192.

External Editor: Dr. William M. Shafer

Received: 28 August 2014; in revised form: 25 September 2014 / Accepted: 28 September 2014 / Published: 13 October 2014

---

**Abstract:** Antimicrobial peptides, or AMPs, play a significant role in many environments as a tool to remove competing organisms. In response, many bacteria have evolved mechanisms to resist these peptides and prevent AMP-mediated killing. The development of AMP resistance mechanisms is driven by direct competition between bacterial species, as well as host and pathogen interactions. Akin to the number of different AMPs found in nature, resistance mechanisms that have evolved are just as varied and may confer broad-range resistance or specific resistance to AMPs. Specific mechanisms of AMP resistance prevent AMP-mediated killing against a single type of AMP, while broad resistance mechanisms often lead to a global change in the bacterial cell surface and protect the bacterium from a large group of AMPs that have similar characteristics. AMP resistance mechanisms can be found in many species of bacteria and can provide a competitive edge against other bacterial species or a host immune response. Gram-positive bacteria are one of the largest AMP producing groups, but characterization of Gram-positive AMP resistance mechanisms lags behind that of Gram-negative species. In this review we present a summary of the AMP resistance mechanisms that have been identified and characterized in Gram-positive bacteria. Understanding the mechanisms of AMP resistance in Gram-positive species can provide guidelines in developing and applying AMPs as therapeutics, and offer insight into the role of resistance in bacterial pathogenesis.

Antibiotics 2014, 3

Keywords: Clostridium difficile; antimicrobial; antimicrobial peptide; AMP; resistance

---

## 1. Introduction

Antimicrobial peptides (AMPs) and the bacterial resistance mechanisms against them have been co-evolving for eons. A diverse array of life forms can produce AMPs, which can be used to promote immune defenses, nutrient acquisition or elimination of rival organisms from the environment. As a result, AMPs are found in a multitude of environments, ranging from mammalian tissues to soil and aquatic environments. This ubiquitous presence of AMPs in the environment provides strong selective pressure to drive the development of bacterial resistance against these peptides.

AMPs are typically small, charged, amphipathic molecules that can be produced in a variety of structures. Though structurally diverse, most AMPs work by interacting with the bacterial cell surface, followed by disruption of cellular integrity. Accordingly, the majority of bacterial resistance mechanisms function by limiting the interaction of AMPs with the bacterial cell surface. Mechanisms of AMP resistance include trapping or sequestering of peptides, outright destruction of AMPs by proteolysis, removal of AMPs from the cell via active transport, and structural modification of the cell surface to avoid interaction with AMPs. Many of these resistance mechanisms are upregulated in response to AMPs, allowing the bacteria to adaptively counter the effects of AMPs. Loss of these resistance mechanisms can impair the ability of bacteria to colonize plant or animal hosts and can attenuate virulence for many pathogens. Mechanisms of resistance may evolve specifically within a bacterial lineage or be genetically transferred from other AMP-resistant organisms.

In this review, we evaluate the available literature on Gram-positive bacterial resistance mechanisms to antimicrobial peptides. This review highlights methods of AMP resistance based on mode of action and location within the Gram-positive bacterial cell. We begin with an overview of resistance mechanisms that act on AMPs extracellularly, and then discuss bacterial cell surface alterations. Finally, we consider removal of AMPs from the bacterial cell via transport.

---

## 2. Extracellular Mechanisms of Resistance: Enzymatic Degradation and AMP Blocking

The initial site of AMP interaction is at the bacterial cell surface. As a result, extracellular mechanisms of AMP inactivation have evolved as a first line of defense to minimize damage to the bacterial cell. Extracellular AMP resistance mechanisms have arisen in two main forms: enzymatic inactivation and sequestration (see Table 1 and Figure 1). The majority of these direct targeting mechanisms have evolved to recognize cationic AMPs. Cationic AMPs are positively charged peptides that may differentially target negatively charged moieties on the outer cell envelope, including teichoic acids, lipid II, and phosphatidylglycerol [1–3].

Antibiotics 2014, 3

### 2.1. Extracellular Proteases

The degradation of AMPs by proteases is a mechanism of resistance found in many Gram-positive species, including *Enterococcus faecalis*, *Staphylococcus aureus*, and *Staphylococcus epidermidis* [4–6]. AMP-degrading proteases generally have broad substrate specificity, are typically found in mammalian pathogens, and include both metalloproteidases and cysteine proteases [7,8]. This section will present several examples of AMP-degrading proteases produced by Gram-positive bacteria and detail their effects on resistance.

AMP-degrading proteases are often secreted by bacteria into their surrounding extracellular environments. Gelatinase, an extracellular metallopeptidase produced by some strains of the opportunistic pathogen *E. faecalis*, cleaves the human cathelicidin, LL-37, resulting in the loss of antimicrobial activity *in vitro* [4]. The production of gelatinase by *E. faecalis* is associated with bacterial dissemination in animal models of disease and with increased incidence of dental caries in humans [9,10]. One example of a secreted protease made by *S. aureus* that confers AMP resistance is the aureolysin enzyme [5]. Aureolysin can hydrolyze the C-terminal bactericidal domain of LL-37, rendering the AMP inactive [11]. An infection model using human macrophages revealed that aureolysin contributes to Staphylococcal persistence within the phagosomal compartment [12], an environment that contains high levels of the antimicrobial peptide, LL-37 [13]. Additionally, some species of *Staphylococci* possess proteases that combat anionic AMPs such as dermcidin, a negatively charged peptide secreted by human sweat glands [14]. SepA (or SepP1) made by *S. epidermidis*, is a secreted metalloprotease that can cleave and inactivate dermcidin [6,15]. The SepA protease appears to specifically target dermcidin *in vitro* [6,16].

Gram-positive proteases are also capable of targeting AMPs at the bacterial surface. SpeB is a cysteine proteinase secreted by the pathogenic bacterium *Streptococcus pyogenes* [17]. SpeB has broad substrate specificity and cleaves AMPs, such as LL-37, and other host proteins such as fibrin, immunoglobulins, and other immune modulators [4,18–21]. In an example of adaptive resistance, SpeB was found to complex with the host α₂-macroglobulin (α₂M) proteinase inhibitor during infection [22]. The catalytically active SpeB-α₂M complexes are retained on the bacterial cell surface by association with the *S. pyogenes* G-related α₂M-binding protein (GRAB) [22,23]. The SpeB-α₂M complex has higher proteinase activity against LL-37, relative to free SpeB, and reduces killing of *S. pyogenes in vitro* [22].

| Name                     | Mechanism of Action                  | Antimicrobial Resistance                          | Organisms                                      | Reference     |
|--------------------------|-------------------------------------|-------------------------------------------------|-----------------------------------------------|---------------|
| **AMP Degradation**      |                                     |                                                 |                                               |               |
| Aureolysin              | Protease                           | LL-37                                           | S. aureus                                     | [5,11]        |
| Gelatinase              | Protease                           | LL-37                                           | E. faecalis                                   | [4,10]        |
| SepA                    | Protease                           | dermcidin                                       | S. epidermidis                               | [6,16]        |
| SpeB                    | Protease                           | LL-37                                           | S. pyogenes                                   | [4,21,22]     |
| **Sequestration/Competition for AMP target** |                                                   |                                                 |                                               |               |
| M Protein               | Binding at surface                 | LL-37                                           | S. pyogenes                                   | [24]          |
| PilB                    | Binding at surface                 | cathelicidins                                   | S. agalactiae                                | [25]          |
| SIC                     | Extracellular binding              | α-defensins, LL-37, lysozyme                    | S. pyogenes                                   | [26,27]       |
| Staphylokinase          | Extracellular binding              | Cathelicidin, defensins                         | S. aureus                                     | [28,29]       |
| LciA                    | Binding at surface                 | Lactococcin A                                   | L. lactis                                     | [30,31]       |
| Capsule                 | Binding/shielding                  | Polymyxin B, HNP-1                              | S. pneumoniae                                | [32]          |
| Exopolysaccharide       | Shielding/Sequestration            | LL-37, hBD-3, dermcidin                         | S. epidermidis                               | [33–35]       |
| LanI lipoproteins       | Binding or competition             | lantibiotics                                    | L. lactis, B. subtilis, other lantibiotic producers | [36–38]       |
| **Cell Surface Modifications** |                                 |                                                 |                                               |               |
| DltABCD                 | D-alanylation of teichoic acids    | daptomycin, vancomycin, nisin, defensins, protegrins | S. aureus, L. monocytogenes, B. cereus, C. difficile, S. pyogenes, S. agalactiae, B. anthracis, S. suis | [2,39–45]     |
| MprF                    | Lysylation of phoshatidylglycerol  | defensins, thrombin-induced platelet microbial protein | S. aureus, L. monocytogenes, B. anthracis, M. tuberculosis | [46–50]       |
| OatA                    | Peptidoglycan O-acetylase         | lysozyme                                        | S. aureus, S. epidermidis, S. lugdunensis, E. faecalis, L. monocytogenes | [51–54]       |
| PdgA                    | Peptidoglycan N-acetylgucosamine deacetylase A | lysozyme                                        | S. pneumoniae, E. faecalis, S. suis, L. monocytogenes, B. anthracis | [55–58]       |
| NamH                    | N-acetylmuramic acid hydroxylase  | lysozyme                                        | M. smegmatis                                  | [59]          |

| Name           | Mechanism of Action       | Antimicrobial Resistance                                                                 | Organisms                     | Reference |
|----------------|--------------------------|---------------------------------------------------------------------------------------|--------------------------------|-----------|
| **AMP Efflux** |                          |                                                                                       |                                |           |
| One-component transporter |                  |                                                                                       |                                |           |
| LmrB          | ABC transporter          | LsbA/LsbB                                                                             | *L. lactis*                   | [60]      |
| QacA          | ABC transporter/         | thrombin-induced platelet                                                              | *S. aureus*                   | [61]      |
|                | alteration of membrane  | micrombicidal protein (tPMP)                                                           |                                |           |
|                | structure               |                                                                                       |                                |           |
| BceAB type    |                          |                                                                                       |                                |           |
| AnrAB         | ABC transporter          | nisin, gallidermin, bacitracin, β-lactams                                              | *L. monocytogenes*            | [62, 63]  |
| BceAB         | ABC transporter          | Bacitracin<sup>a</sup>, actagardine, mersacidin, plectasin                              | *B. subtilis<sup>a</sup>*     | [64–68]   |
| BraAB         | ABC transporter          | nisin, nukacin ISK-1, bacitracin                                                        | *S. aureus*                   | [69]      |
| PsdAB         | ABC transporter          | nisin, enduracidin, gallidermin, subtilin                                              | *B. subtilis*                 | [66]      |
| MbrAB         | ABC transporter          | bacitracin                                                                            | *S. mutans*                   | [35]      |
| SP0812-SP0813 | ABC transporter          | bacitracin, vancoresmycin                                                              | *S. pneumoniae*               | [70]      |
| SP0912-SP0913 | ABC transporter          | bacitracin, lincomycin, nisin                                                          | *S. pneumoniae*               | [71]      |
| VraDE         | ABC transporter          | bacitracin, nisin, nukacin ISK-1                                                       | *S. aureus*                   | [69, 72–76] |
| VraFG         | ABC transporter          | nisin, colistin, bacitracin, vancomycin, indolicidin, LL-37, hBD 3                      | *S. aureus*, *S. epidermidis* | [69, 72, 75, 77–79] |
| YsaCB         | ABC transporter          | nisin                                                                                 | *L. lactis*                   | [80]      |
| BcrAB type    |                          |                                                                                       |                                |           |
| BcrAB(C)      | ABC transporter          | bacitracin                                                                            | *B. licheniformis*            | [81]      |
| BcrAB(D)      | ABC transporter          | bacitracin                                                                            | *E. faecalis*                 | [82, 83]  |
| LanFEG type   |                          |                                                                                       |                                |           |
| As-48EFG(H)   | ABC transporter          | AS-48                                                                                | *E. faecalis*                 | [84]      |
| CprABC        | ABC transporter          | nisin, gallidermin, other lantibiotics                                                  | *C. difficile*                | [85, 86]  |
| EpiFEG(H)     | ABC transporter          | epidermin, gallidermin                                                                | *S. epidermidis*              | [87]      |
| LtnFE(I)      | ABC transporter          | lactacin 3147                                                                         | *L. lactis*                   | [88, 89]  |
| McdFEG        | ABC transporter          | macedocin                                                                             | *S. macedonicus*             | [90]      |
| MrsFGE        | ABC transporter          | mersacidin                                                                            | *Bacillus sp.* HIL           | [91, 92]  |
| MutFEG        | ABC transporter          | mutacin II                                                                            | *S. mutans*                   | [93]      |
| NisFEG(I)     | ABC transporter          | nisin                                                                                 | *L. lactis*                   | [37, 94]  |
| NukFEG(H)     | ABC transporter          | nukacin                                                                               | *S. warneri*                  | [95, 96]  |
| SboFEG        | ABC transporter          | salivaricin B                                                                         | *S. salivarius*               | [97]      |
| ScnFEG        | ABC transporter          | streptococcin A-FF 22                                                                  | *S. pyogenes*                 | [98]      |
| SmbFT         | ABC transporter          | Smb, haloduracin                                                                      | *S. mutans*                   | [99]      |
| SpaFEG        | ABC transporter          | subtilin                                                                              | *B. subtilis*                 | [36, 100] |

<sup>a</sup> Confers only bacitracin resistance in *B. subtilis*.

Figure 1. Overview of Antimicrobial Peptide Resistance Mechanisms in Gram-Positive Bacteria. (A) Extracellular mechanisms of AMP resistance include peptide degradation by secreted proteases, AMP sequestration by secreted or membrane-associated protein (e.g., pili, immunity proteins, M proteins), or blocking by capsule polysaccharides; (B) Cell wall and membrane modifications include: Alteration of charge by lysination of the phospholipid head groups or D-alanylation of the lipoteichoic backbone, modification of the cell wall by deacetylation of N-acetylglucosamine or O-acetylation of N-acetylmuramyl residues, and alterations in membrane fluidity by phospholipid tail saturation or carotenoid additions; (C) Transport mechanisms of antimicrobial efflux from the cell include: ATP-driven ABC transporters composed of a single, double, or triple protein pump and involve a supplementary immunity protein, or single protein transporters driven by proton motive force.

2.2. Protein-Mediated Sequestration

Sequestration is another extracellular mechanism of AMP resistance [24–29, 101]. Some Gram-positive bacteria produce extracellular or surface-linked proteins that directly bind to AMPs and block access to

the cell membrane. Mechanisms of protein-mediated AMP sequestration vary between species and strains. We have highlighted specific examples of AMP sequestration mechanisms identified amongst strains of *S. pyogenes*, *S. aureus*, *Streptococcus agalactiae*, and *Lactococcus lactis*.

Proteins that inhibit AMP activity through binding can be secreted into the extracellular environment to inhibit contact of bactericidal peptides with the cellular surface. For example, the Streptococcal inhibitor of complement (SIC) produced by *S. pyogenes* is a hydrophilic, secreted protein that sequesters many AMPs, thereby preventing them from reaching cell-surface targets [102]. SIC binds to α-defensins, LL-37, and lysozyme, neutralizing the AMPs and inhibiting their bactericidal activity against *S. pyogenes* [27, 102, 103]. SIC production promotes bacterial survival *in vitro* and increases the virulence of *S. pyogenes* in animal models of disease [26, 104]. Staphylokinase secretion by *S. aureus* is another example of an extracellular AMP resistance mechanism. Production of the staphylokinase protein by *S. aureus* occurs through the lysogenic conversion of the hlb β-hemolysin toxin gene by a bacteriophage harboring the *sak* gene [105–107]. Staphylokinase binds the murine cathelicidin mCRAMP *in vivo* and also complexes with human defensins HNP-1 and HNP-2 to reduce their bactericidal effects [28, 29]. Studies of staphylokinase binding suggest that the staphylokinase-cathelicidin complex promotes host tissue invasion by activating the conversion of plasminogen to the host extracellular matrix-degrading enzyme, plasmin, although the role this conversion plays in Staphylococcal virulence remains unclear [29, 101, 108].

Proteins attached to the cellular surface can also bind AMPs to prevent contact with cell-associated targets. Examples of such proteins include the M1 protein of *S. pyogenes* and the pilus subunit, PilB of *S. agalactiae*. M1 of *S. pyogenes* can be found on the surface of most clinical isolates and has been linked to both host tissue adherence and invasive disease [109]. A hyper-variable extracellular portion of the M1 protein was shown to bind LL-37 and prevent the AMP from reaching the cell membrane [24]. The sequestration of LL-37 by M1 also promotes Streptococcal survival in neutrophil extracellular traps (NETs) by reducing LL-37 activity [24]. Like the M proteins of *S. pyogenes*, pili are also associated with invasive disease and promotion of host cell adherence by *S. agalactiae* [110, 111]. Pili are large, filamentous, multimeric protein complexes expressed on the cell surface of *S. agalactiae* and other bacteria. Expression of the Streptococcal pilin subunit, PilB, promotes association of LL-37 with the bacterial cell surface and correlates with increased resistance to the murine cathelicidin mCRAMP *in vitro* [25]. In addition, pilB mutants of *S. agalactiae* (GBS) exhibit reduced fitness relative to wild-type strains in murine infection models [25]. These data suggest that in addition to the adhesin properties of pili, pilus-mediated binding of AMPs also contributes to *S. agalactiae* virulence within the host.

Another family of membrane-associated AMP resistance proteins encompasses the LanI immunity proteins of some bacteriocin producer strains. LanI proteins are typically encoded near a bacteriocin biosynthetic operon and provide protection against the bacteriocin made by the producer bacterium [112, 113]. LanI-type immunity proteins are lipoproteins that anchor to the bacterial cell surface and confer resistance by either binding directly to AMPs or outcompeting AMPs by binding directly to the cellular target [114–117]. The LanI lipoproteins often work in concert with LanFEG transporters, possibly acting as substrate-binding partners for specific lantibiotics. The best characterized of the transporter-associated LanI proteins are the NisI and SpaI lipoproteins found in strains of *L. lactis* and *Bacillus subtilis*, respectively [36, 37, 118] (described in transporter section). But, several lantibiotic producers encode only a LanI immunity protein and do not encode an apparent LanFEG transporter

(e.g., Pepl of *S. epidermidis* [119], lactocin S [120] of *L. sakei* and epicidin 280 of *S. epidermidis* [121]). In these systems, the LanI lipoprotein confers full immunity to the associated lantibiotic. Though some LanI structures have been characterized, LanI lipoproteins generally have low, if any, homology to one another [116, 122]. Thus, it is unclear if mechanism of action for LanI-mediated immunity is conserved between different LanI lipoproteins.

## 2.3. Inhibition of AMP Activity by Surface-Associated Polysaccharides

Extracellular polysaccharide production has long been recognized as a factor that promotes both virulence and host colonization by many bacteria [123–125]. Extracellular polysaccharides are composed of structurally diverse polymers that are enzymatically produced by some Gram-positive species [126, 127]. Extracellular polysaccharides that are attached to the cellular surface through covalent linkages with the cell wall are known as capsules (capsular polysaccharide, or CPS), while loosely attached polymers are referred to as exopolysaccharides, or EPS [128–130]. Polysaccharide-mediated AMP resistance is thought to occur by shielding the bacterial membrane via binding or electrostatic repulsion of AMPs [34, 131].

The production of capsular polysaccharides provides resistance to a variety of AMPs and other antimicrobials and can allow some bacteria to evade host detection. Capsule-AMP binding can be mediated by the electrostatic interaction of positively charged AMPs with the negatively charged polysaccharide capsule [32]. For example, free capsular extracts from *Streptococcus pneumoniae* bind both polymyxin B and the α-defensin HNP-1, preventing these AMPs from reaching the cell membrane and promoting bacterial survival *in vitro*. Additionally, both polymyxin B and HNP-1 promote release of the capsule from *S. pneumoniae* without loss of cell viability, suggesting that capsule release may be a mechanism of AMP resistance by sequestering AMPs away from the bacterial cell surface [32]. In another example, production of the exopolysaccharide intercellular adhesion, PIA, by *S. epidermidis* reduces killing by human defensin hBD-3, cathelicidin (LL-37), and the anionic AMP dermcidin. PIA is hypothesized to shield the bacterial membrane from the effects of AMPs [33, 34]. Predictably, PIA production is associated with *S. epidermidis* virulence in multiple animal infection models [132, 133]. However, while many exopolysaccharide capsules can provide resistance to AMPs, this protection is not universal to all capsule-producing Gram-positive bacteria [134–136].

## 3. Membrane and Cell Wall Modifications

The bacterial cell wall and membrane comprise a major target for the bactericidal activity of AMPs [137–139]. Bacteria frequently modify cell surface components to counter the effects of AMPs by reducing the net negative charge of the cell, altering membrane fluidity, or directly modifying AMP targets [140–142].

### 3.1. Repulsion of AMPs

Many AMPs target bacterial cells through electrostatic interactions with the cell surface [137–139]. The net charge of the bacterial cell surface is generated by anionic components of the cell membrane and cell wall, such as phospholipids and teichoic acids [143–145]. In turn, positively charged AMPs are

attracted to the negatively charged bacterial cell surface [144, 145]. Hence, a broad strategy of resistance to positively charged AMPs is to alter the components on the cell surface to decrease the net negative charge of the cell, thereby limiting the electrostatic interaction of AMPs with the bacterial cell surface.

One component of the bacterial cell membrane that carries a negative charge is phosphatidylglycerol [144, 145]. But in many Gram-positive bacteria, the negative charge on phosphatidylglycerol can be masked via the addition of a positively charged amino acid by the multipeptide resistance factor protein, MprF [146, 147]. MprF is an integral lysyl-phosphatidylglycerol synthetase that synthesizes and translocates aminoacylated-phosphatidylglycerol to the external membrane layer of the bacterial cell. MprF syntheses were initially found to incorporate a positively charged lysine into phosphatidylglycerol (Lys-PG), decreasing the net negative charge on the bacterial membrane. In *S. aureus*, *Listeria monocytogenes*, *E. faecalis*, *Enterococcus faecium*, *B. subtilis*, and *Bacillus anthracis*, the aminoacylation of phosphatidylglycerol by MprF confers resistance to positively charged AMPs [47–49, 148–150]. Additionally, an MprF homolog is present in *Mycobacterium tuberculosis*, which also confers resistance to positively charged AMPs. This MprF homolog, LysX, carries out the same functions as MprF, with the addition of a lysyl-tRNA synthetase activity [46, 151]. Lysinylation of phosphatidylglycerol confers resistance to a broad spectrum of AMPs, including human defensins, gallidermin, nisin, lysozyme, daptomycin, polymyxin B, and vancomycin (Table 1) [46, 150–159]. In addition to lysine modifications, some MprF orthologs can modify membrane phosphatidylglycerol with multiple amino acids, including alanine and arginine [149, 160]. The enhanced antimicrobial resistance provided by aminoacylation of phosphatidylglycerol is also associated with increased virulence for several Gram-positive pathogens [46, 48, 49, 161, 162].

The Dlt pathway is another enzymatic mediator of AMP resistance that has been identified and studied in many Gram-positive genera including *Staphylococcus*, *Listeria*, *Enterococcus*, *Bacillus*, *Clostridium*, *Streptococcus*, and *Lactobacillus* [2, 40–45, 163–168]. The enzymatic functions of the DltABCD proteins lead to the D-alanylation of teichoic acids and lipoteichoic acids of the cell wall [169]. The addition of D-alanine to the backbone of teichoic acids can mask the negative charge present along these glycopolymers, thereby leading to increased surface charge and lower attraction of positively charged antimicrobials [169]. Similar to MprF, D-alanylation of teichoic acids by the Dlt pathway leads to a global change in charge distribution across the cell surface, allowing resistance to a broad range of cationic AMPs including vancomycin, nisin, gallidermin daptomycin, polymyxin B, lysozyme, and cathelicidins [2, 39, 141, 163, 166, 170–172].

In addition to charge modification of teichoic acids, high-resolution microscopy of Group B Streptococcus revealed that D-alanylation could increase cell wall density, leading to increased surface rigidity [173]. Accordingly, D-alanylation may confer resistance to AMPs both by reducing the electrostatic interactions between AMPs and the cell surface and by decreasing the permeability of the cell wall [173]. As AMPs are ubiquitous within animals, D-alanylation of the cell wall can affect host colonization for pathogens and non-pathogenic species [41, 152, 164, 174, 175].

## 3.2. Target Modification

The cell wall is a common antimicrobial target for Gram-positive organisms. As a result, bacteria have evolved multiple modifications that limit antimicrobial targeting of the cell wall. Lysozyme,
or N-acetylmuramide glycanhydrolase, an antimicrobial enzyme, is an important component of the host innate immune defense. Lysozyme is cationic at physiological pH, which facilitates its interaction with negatively charged bacterial surfaces. The cationic and muramidase activities of lysozyme directly target the bacterial peptidoglycan, the primary constituent of the cell wall [176]. The muramidase domain of lysozyme hydrolyzes the β-1,4 linkages between N-acetylglucosamine and N-acetylmuramic acid of peptidoglycan, leading to the breakdown of the peptidoglycan macromolecular structure and eventual lysis of the cell [177–179]. As a result, bacterial resistance mechanisms have evolved to counter both the muramidase and cationic activities of lysozyme. In this section, we detail the mechanisms by which peptidoglycan is modified to limit lysozyme activity.

Two peptidoglycan modifiers that contribute to AMP resistance in some Gram-positive bacteria are the enzymes PgdA and OatA. It is proposed that the modifications made by both of these enzymes lead to steric hindrance between AMPs and the cell surface, thereby limiting the contact between lysozyme and its target [180]. PgdA deacetylates N-acetylglucosamine residues of peptidoglycan, generating a less favorable substrate for lysozyme [181–184]. PgdA was first implicated as a peptidoglycan deacetylase in the respiratory pathogen *S. pneumoniae*. PgdA and other peptidoglycan deacetylase orthologs have been shown to contribute to AMP resistance in many bacteria, including *Enterococcus*, *Streptococcus*, *Listeria* and *Bacillus* species [55–58, 180, 183, 185]. Moreover, deacetylation of peptidoglycan enhances colonization and virulence in several pathogens, including *E. faecalis*, *L. monocytogenes* and *S. pneumoniae* [185–187]. As N-acetylglucosamine deacetylases are encoded within the genomes of most Gram-positive bacteria, these enzymes likely contribute to lysozyme and host colonization in many more species.

OatA (also known as Adr in *S. pneumoniae*) is another type of peptidoglycan modifying enzyme found in Gram-positive bacteria that confers resistance to lysozyme [188–190]. OatA performs O-acetylation at the C6-OH group of N-acetylmuramyl residues in peptidoglycan [188–190]. O-acetylation of N-acetylmuramyl residues is thought to prevent lysozyme from interacting with the β-1,4 linkages of peptidoglycan by steric hindrance [180]. OatA and orthologous proteins have been characterized in *Staphylococcus*, *Enterococcus*, *Lactococcus*, *Bacillus*, *Streptococcus* and *Listeria* species [51, 52, 54, 58, 180, 187, 191]. Like deacetylation mechanisms, O-acetylation of peptidoglycan is likely to be widespread among Firmicutes and has been noted to contribute to virulence in animal models of infection [52, 54, 187, 190, 192].

A peptidoglycan modifier unique to *Mycobacterium* is the enzyme NamH (*N*-acetylmuramic acid hydroxylase). NamH hydroxylates *N*-acetylmuramic acid residues leading to the production *N*-glycolylmuramic acid. The modification of peptidoglycan by NamH was determined to confer lysozyme resistance in *Mycobacterium smegmatis* [59]. It is likely that NamH confers lysozyme resistance to Mycobacterial species through the generation of *N*-glycolylmuramic acid, as NamH is well conserved in Mycobacterial genomes. It is hypothesized that *N*-glycolylmuramic acid residues may stabilize the cell wall; however, the mechanism of resistance is not fully understood [193]. However, recent work suggests that the presence of an *N*-glycolyl group blocks lysozyme from accessing the β-1,4 peptidoglycan bonds, preventing the muramidase activity of lysozyme and leaving the cell wall intact [59].
Antibiotics 2014, 3

### 3.3. Alterations to Membrane Order

Apart from AMP repulsion and AMP target modifications as mechanisms of resistance, other changes in membrane composition can also reduce the susceptibility of bacteria to AMP-mediated killing. Alterations in Gram-positive membrane composition appear to contribute to AMP resistance by affecting the peptide interactions with the cell membrane. In particular, the degree of membrane fluidity appears to be an important determinant of AMP susceptibility.

One example of a membrane alteration that confers AMP resistance is the saturation of membrane fatty acids. Investigations into the cell membrane components of nisin-resistant *L. monocytogenes* showed that some resistant strains contained a higher proportion of saturated (straight chain) fatty acids versus unsaturated (branched chain) fatty acids [194,195]. Additionally, a nisin-resistant strain of *L. monocytogenes* produced lower concentrations of the lipid head group phosphatidylglycerol and less diphosphatidylglycerol than a nisin-susceptible strain [194–196]. This nisin-resistant strain also contained higher concentrations of the lipid head group, phosphatidylethanolamine, while the anionic membrane component, cardiolipin, was decreased [197]. These studies suggest that higher concentrations of saturated fatty acids, a decrease in phosphatidylglycerol, and an increase in phosphatidylethanolamine head groups in the *Listeria* membrane lead to a decrease in cell membrane fluidity [194–197]. It is proposed that the decrease in membrane fluidity increases nisin resistance by hindering nisin insertion into the membrane [197].

The addition of other membrane components can also increase rigidity and lead to resistance to host AMPs and daptomycin in *S. aureus* [198]. Increased membrane rigidity in some Gram-positive organisms can result from carotenoid overproduction [199,200]. Carotenoids are organic pigments made of repeating isoprene units that are produced by plants, bacteria, and fungi [201]. Carotenoids, such as staphyloxanthin made by *S. aureus*, can stabilize the leaflets of the cell membrane by increasing order in the fatty acid tails of membrane lipids and lead to decreased susceptibility to AMPs [199,202,203]. This stabilization of fatty acid tails leads to an increase in cell membrane rigidity, which is suggested to limit insertion of AMPs into the membrane [204,205].

Though a higher concentration of saturated fatty acids in the membrane confers AMP resistance in some bacteria, other bacteria increase unsaturated fatty acid concentrations to increase resistance. In *S. aureus*, increased levels of unsaturated membrane lipids increase the resistance to the host AMP, tPMP (thrombin-induced platelet microbicidal proteins) [206]. Unsaturated fatty acids contain double bonds along the length of their carbon chain, which causes lipid disorder, thereby increasing membrane fluidity and impacting resistance to antimicrobials [206,207]. Other studies in AMP resistance found that methicillin-resistant *S. aureus* isolates that developed resistance to daptomycin also had increased resistance to host tPMPs and the human neutrophil peptide, hNP-1. These co-resistant strains have a phenotype defined by increased cell wall thickness and increased membrane fluidity [198]. It is hypothesized that these altered membrane arrangements may prevent efficient AMP insertion into the membrane [198,206,207].

At present, there is no clear explanation as to how alterations in membrane fluidity or rigidity lead to AMPs resistance. From the examples discussed above, it could be argued that the degree of fluidity required for resistance to a particular AMP may be as varied as the structures of the AMPs themselves, or perhaps is constrained to groups with similar mechanisms of action.

Antibiotics 2014, 3

4. AMP Efflux Mechanisms

Transport, or efflux, is a common mechanism used by Gram-positive bacteria for the removal of toxic compounds and antimicrobials from cells. The majority of antimicrobial peptide efflux mechanisms consist of multi-protein ABC (ATP-binding cassette) transporter systems, which use ATP to drive the transport of substrates across or out of the cell membrane [208]. There are three primary types of ABC transporter systems implicated in Gram-positive AMP resistance: three-component ABC-transporters, two-component ABC-transporters, and single protein multi-drug resistance transporters, or MDR pumps [209]. All ABC-transporters are composed of two distinct domains: the transmembrane domain (permease) and the nucleotide-binding domain (NBD), which facilitates ATP-binding [209]. A less common efflux mechanism that has been identified is the Major Facilitator (MFS) Transporter module, which facilitates small solute transport via a chemiosmotic ion gradient [210]. This section will present the key types of AMP transporters found in Gram-positive bacteria and highlight the AMP resistance characteristics of these systems.

4.1. Three-Component (LanFEG) Transporter Systems

Three-component ABC transporters, or LanFEG systems, are best characterized in AMP-producing bacteria. LanFEG systems are members of the ABC-type 2 sub-family of transporters, and consist of one protein with a nucleotide-binding domain (LanF) and two distinct transmembrane permeases (LanE and LanG) [211]. The majority of the characterized LanFEG systems are self-immunity mechanisms that provide protection against bacteriocins (typically lantibiotics) made by bacteriocin producer strains [38, 112] (Table 1). The LanFEG transporters are often found in conjunction with LanI membrane-associated lipoproteins that can function in tandem with the transporter to provide greater resistance to AMPs [112, 212, 213].

The best-characterized LanFEG transporters are the NisFEG and SpaFEG systems found in strains of *L. lactis* and *B. subtilis* that produce the lantibiotic AMPs nisin and subtilin, respectively. Both NisFEG and SpaFEG provide resistance to their cognate substrates, but full resistance is achieved in concert with their associated substrate-binding lipoproteins, NisI and SpaI [100, 213–215]. Immunity to the lantibiotic nukacin ISK from *Streptococcus warneri* does not involve a LanI protein, but instead contains a distinct membrane-associated protein termed NukH [96, 216]. In contrast to the LanI proteins, NukH is not a lipoprotein; however, NukH does appear to function as a substrate-binding partner to the NukFEG transporter. Similar to LanI, NukH confers partial immunity to nukacin ISK, but full immunity requires the complete NukFE GH system [216, 217].

Most characterized LanFEG systems confer resistance only to the AMP made by a producer strain, although examples have been identified that provide resistance to multiple AMP substrates in non-producer bacteria. In *Clostridium difficile*, the CprABC transporter (a LanFEG ortholog) confers resistance to nisin, gallidermin, and likely other structurally dissimilar lantibiotic peptides [85, 86]. The regulation of immunity and AMP biosynthetic genes are typically coupled in bacteriocin producer strains [112]. The ability of the CprABC system to confer resistance to multiple unrelated peptides may result from the uncoupling of the immunity mechanism from bacteriocin synthesis. But non-producers that have immunity genes in the absence of AMP biosynthetic operons can have relaxed substrate

specificity that allows for recognition of multiple bacteriocins. Thus, Lan transporter cross-immunity to multiple AMPs could provide a significant competitive advantage to non-producer bacteria. Indeed, a homology search for LanFEG proteins reveals that the genomes of many other Firmicutes encode predicted bacteriocin transporters that are not coupled with apparent bacteriocin synthesis genes. Hence, like other antibacterial resistance mechanisms, the LanFEG systems have found their way into non-producing species [85, 86].

### 4.2. Two-Component ABC-Transporter Systems

Two-component ABC-transporters make up the majority of transporter-mediated AMP resistance characterized in non-AMP producing bacteria. The canonical two-component ABC-transporter consists of one nucleotide-binding protein and a separate membrane-spanning permease [218, 219]. Unlike most LanFEG systems, two-component transporters often provide resistance to multiple AMPs and are common among Gram-positive bacteria. As outlined in Table 1, numerous examples of these transporters have been identified that can provide resistance to AMPs produced by humans and bacteria, including cyclic peptides and some non-peptide antibiotics [218, 220].

There are two main types of two-component ABC-transporter systems that confer resistance to AMPs among Gram-positive bacteria. The first and most common type is often referred to as the BceAB group [218, 221]. BceAB transporter systems contain an archetypal ATP-binding protein of about 225–300 amino acids and a larger permease component that ranges in size from 620–670 amino acids. The prototype of this transporter group, BceAB, was first identified as a bacitracin resistance mechanism in *B. subtilis* [67, 68]. Since the identification of BceAB, dozens of similar transporters have been discovered in pathogenic and non-pathogenic Gram-positive species, including *S. aureus*, *L. monocytogenes*, *S. pneumoniae*, and *L. lactis* (see Table 1 for examples) [62, 71, 77, 80]. Members of the BceAB group have demonstrated resistance to a wide-range of bacteriocins, mammalian and fungal defensins, peptide antibiotics, and other antimicrobial compounds (Table 1). Although many of the BceAB transporters confer resistance to AMPs *in vitro*, the roles of these transporters in the virulence of pathogenic species are not known.

Another common type of a Gram-positive ABC-transporter that confers AMP resistance is the BcrAB(C) system. The BcrAB(C) transporter confers resistance to bacitracin and was originally identified in a bacitracin producer strain of *Bacillus licheniformis* [81]. BcrAB transporters can be distinguished from the BceAB systems by size and topology: BcrA is an ATP-binding cassette that ranges from about 280–320 amino acids, while the BcrB permease modules are smaller, at approximately 200–250 amino acids. BcrAB is often encoded with a third protein, BcrC (or BcrD), which allows for higher resistance to bacitracin than the BcrAB transporter alone [81, 222, 223]. Initially it was hypothesized that BcrC functioned as part of the BcrAB ABC-transporter, however it was later demonstrated that BcrC acts as an undecaprenyl pyrophosphate (UPP) phosphatase that competes with bacitracin for UPP [222]. The BcrAB transporters are predicted to be structurally similar to the LanFEG transporters, though the Lan systems function through two dissimilar permease components, while Bcr systems operate with only one permease subunit (BcrB) [38, 82, 218]. Aside from the bacitracin producer strains, BcrAB and orthologous transporters have been shown to confer resistance to bacitracin in many strains of *E. faecalis*, as well as some *Streptococcus* and *Clostridium* species [35, 82, 83, 224].

Antibiotics 2014, 3

### 4.3. Single Membrane Protein Antimicrobial Transporters

Multi-drug resistance (MDR) ABC-transporters are common bacterial mechanisms of resistance to peptide and non-peptide antibiotics [225]. Though these transporters are most common among characterized mechanisms for non-peptide antimicrobial resistance in Gram-positive bacteria, there are examples of MDR transporters that confer resistance to AMPs. One notable MDR AMP resistance mechanism consists of the LmrA/B proteins encoded by some *L. lactis* strains [60, 226]. A LmrA MDR efflux pump was first described in a non-producer strain of *L. lactis* [226]. LmrB is an ortholog of LmrA found in *L. lactis* strains that produce the bacteriocins LsbA and LsbB [60]. LmrA/LmrB are membrane proteins with six predicted transmembrane segments and a C-terminal, nucleotide-binding domain [60]. LmrA provides broad resistance against a long list of peptide antibiotics and cytotoxic compounds, while LmrB confers resistance to the two bacteriocins LsbA and LsbB [60, 226]. A BLASTp homology search revealed the presence of additional orthologs of LmrA/B encoded within the genomes of hundreds of Gram-positive Firmicutes, though the function and significance of these remains unknown.

A less common type of single-protein transporter involved in antimicrobial peptide resistance is exemplified by the QacA transporter of *S. aureus* [61]. QacA is a member of the major facilitator superfamily (MFS) of membrane transport proteins, which use proton motive force, rather than ATP, to drive the efflux of substrates [227]. QacA confers resistance to a variety of toxic dyes, antiseptics and disinfectants [228, 229]. In addition to cationic toxins, QacA provides resistance to thrombin-induced platelet microbicidal protein (tPMP), a host-derived antimicrobial peptide [61]. QacA-dependent tPMP resistance was found to confer a survival advantage in an animal model of infection, and increased resistance to tPMP in *S. aureus* also correlates with endocarditis in humans [61, 230]. QacA orthologs have also been identified in other staphylococci, as well as in *Enterococcus* and *Bacillus* species, though the ability of these orthologs to transport AMPs is not understood [231, 232].

### 5. Conclusions

Antimicrobial peptides are diverse in both structure and function and are produced by all forms of life. As such, AMPs are an ancient defense mechanism, and resistance mechanisms to AMPs have been selected for as long as AMPs have existed. Gram-positive bacteria are ancient producers of AMPs and as a consequence, these organisms likely developed some of the first AMP resistance mechanisms.

Herein we have detailed a wide variety of AMP resistance mechanisms found in Gram-positive bacteria (summarized in Figure 1). AMPs resistance mechanisms can be broad spectrum, such as MprF and the Dlt pathway which function by decreasing the net negative charge of the bacterial cell surface, thereby reducing the attraction for positively charged AMPs from the cell. Conversely, AMP resistance mechanisms can be highly specific and only confer resistance to a single peptide. AMP resistance mechanisms can be confined to a particular species or genus, such as NamH in *Mycobacterium*, or can be distributed among multiple species, such as the LanFEG systems. AMPs resistance mechanisms are dynamic; they can be passed from species to species via bacteriophages or horizontal gene transfer, and can change specificity and function over time through evolution [85, 86, 105, 233]. Under selective pressure, AMP resistance mechanisms can evolve to suit the needs of a particular species in its own niche [234].

At present, many AMPs are being investigated as potential antimicrobial therapies [235–240]. AMP drug development should be carefully vetted because like any naturally-produced antimicrobial, cognate resistance mechanisms for AMPs are already present in the producer bacterium. While these resistance mechanisms may be found more frequently in producer strains, each has the propensity to be passed on to other genera or species within a shared environmental niche. Because the presence of AMPs provides high selective pressure for the acquisition of resistance, it is important to consider the potential for resistance mechanism transfer between bacteria when developing AMPs for clinical use [241, 242]. Additionally, depending on the AMP resistance mechanism that is selected for, a multitude of issues may arise if the mechanism of resistance is broad-spectrum. A broad-spectrum AMP resistance mechanism could restrict the already limited clinical treatment options for use against some Gram-positive pathogens and may undermine our own immune response by conferring resistance to our own innate immune system peptides [243].

Antimicrobial peptide resistance is not as well characterized for Gram-positive bacteria as it is for Gram-negative bacteria. Thus, it is likely that many more mechanisms of antimicrobial resistance remain to be discovered in Gram-positive species. As more AMPs are found, new Gram-positive AMP resistance mechanisms will undoubtedly be revealed.

## Acknowledgments

We give special thanks to Rita Tamayo and Adrianne Edwards for helpful criticism of this manuscript. We sincerely apologize to any colleagues whose work was not cited due to the large volume of manuscripts on this topic. This work was supported by the U.S. National Institutes of Health through research grants DK087763 and DK101870 to SMM and training grant AI106699 to KLN. The content of this manuscript is solely the responsibility of the authors and does not necessarily reflect the official views of the National Institutes of Health.

## Author Contributions

K.L.N., E.K.C. and S.M.M. wrote the manuscript.

## Conflicts of Interest

The authors declare no conflict of interest.

## References

1. Koprivnjak, T.; Peschel, A. Bacterial resistance mechanisms against host defense peptides. *Cell. Mol. Life Sci.* **2011**, *68*, 2243–2254.
2. Peschel, A.; Otto, M.; Jack, R.W.; Kalbacher, H.; Jung, G.; Gotz, F. Inactivation of the dlt operon in *Staphylococcus aureus* confers sensitivity to defensins, protegrins, and other antimicrobial peptides. *J. Biol. Chem.* **1999**, *274*, 8405–8410.
3. Staubitz, P.; Neumann, H.; Schneider, T.; Wiedemann, I.; Peschel, A. MprF-mediated biosynthesis of lysylphosphatidylglycerol, an important determinant in staphylococcal defensin resistance. *FEMS Microbiol. Lett.* **2004**, *231*, 67–71.

4. Schmidtchen, A.; Frick, I.M.; Andersson, E.; Tapper, H.; Bjorck, L. Proteinases of common pathogenic bacteria degrade and inactivate the antibacterial peptide LL-37. Mol. Microbiol. 2002, 46, 157–168.
5. Sabat, A.; Kosowska, K.; Poulsen, K.; Kasprowicz, A.; Sekowska, A.; van Den Burg, B.; Travis, J.; Potempa, J. Two allelic forms of the aureolysin gene (aur) within Staphylococcus aureus. Infect. Immun. 2000, 68, 973–976.
6. Lai, Y.; Villaruz, A.E.; Li, M.; Cha, D.J.; Sturdevant, D.E.; Otto, M. The human anionic antimicrobial peptide dermcidin induces proteolytic defence mechanisms in staphylococci. Mol. Microbiol. 2007, 63, 497–506.
7. Hase, C.C.; Finkelstein, R.A. Bacterial extracellular zinc-containing metalloproteases. Microbiol. Rev. 1993, 57, 823–837.
8. Del Papa, M.F.; Hancock, L.E.; Thomas, V.C.; Peregoy, M. Full activation of Enterococcus faecalis gelatinase by a C-terminal proteolytic cleavage. J. Bacteriol. 2007, 189, 8835–8843.
9. Engelbert, M.; Mylonakis, E.; Ausubel, F.M.; Calderwood, S.B.; Gilmore, M.S. Contribution of gelatinase, serine protease, and fsr to the pathogenesis of Enterococcus faecalis endophthalmitis. Infect. Immun. 2004, 72, 3628–3633.
10. Thurlow, L.R.; Thomas, V.C.; Narayanan, S.; Olson, S.; Fleming, S.D.; Hancock, L.E. Gelatinase contributes to the pathogenesis of endocarditis caused by Enterococcus faecalis. Infect. Immun. 2010, 78, 4936–4943.
11. Sieprawska-Lupa, M.; Mydel, P.; Krawczyk, K.; Wojcik, K.; Puklo, M.; Lupa, B.; Suder, P.; Silberring, J.; Reed, M.; Pohl, J.; et al. Degradation of human antimicrobial peptide LL-37 by Staphylococcus aureus-derived proteinases. Antimicrob. Agents Chemother. 2004, 48, 4673–4679.
12. Kubica, M.; Guzik, K.; Koziel, J.; Zarebski, M.; Richter, W.; Gajkowska, B.; Golda, A.; Maciag-Gudowska, A.; Brix, K.; Shaw, L. A potential new pathway for Staphylococcus aureus dissemination: The silent survival of S. aureus phagocytosed by human monocyte-derived macrophages. PLoS One 2008, 3, e1409.
13. Rivas-Santiago, B.; Hernandez-Pando, R.; Carranza, C.; Juarez, E.; Contreras, J.L.; Aguilar-Leon, D.; Torres, M.; Sada, E. Expression of cathelicidin LL-37 during Mycobacterium tuberculosis infection in human alveolar macrophages, monocytes, neutrophils, and epithelial cells. Infect. Immun. 2008, 76, 935–941.
14. Schittek, B.; Hipfel, R.; Sauer, B.; Bauer, J.; Kalbacher, H.; Stevanovic, S.; Schirle, M.; Schroeder, K.; Blin, N.; Meier, F.; et al. Dermcidin: A novel human antibiotic peptide secreted by sweat glands. Nat. Immunol. 2001, 2, 1133–1137.
15. Teufel, P.; Gotz, F. Characterization of an extracellular metalloprotease with elastase activity from Staphylococcus epidermidis. J. Bacteriol. 1993, 175, 4218–4224.
16. Cheung, G.Y.; Rigby, K.; Wang, R.; Queck, S.Y.; Braughton, K.R.; Whitney, A.R.; Teintze, M.; DeLeo, F.R.; Otto, M. Staphylococcus epidermidis strategies to avoid killing by human neutrophils. PLoS Pathog. 2010, 6, e1001133.
17. Hauser, A.R.; Stevens, D.L.; Kaplan, E.L.; Schlievert, P.M. Molecular analysis of pyrogenic exotoxins from Streptococcus pyogenes isolates associated with toxic shock-like syndrome. J. Clin. Microbiol. 1991, 29, 1562–1567.

18. Elliott, S.D. A proteolytic enzyme produced by group A Streptococci with special reference to its effect on the type-specific M antigen. *J. Exp. Med.* 1945, 81, 573–592.
19. Kapur, V.; Majesky, M.W.; Li, L.L.; Black, R.A.; Musser, J.M. Cleavage of interleukin 1 beta (IL-1 beta) precursor to produce active IL-1 beta by a conserved extracellular cysteine protease from *Streptococcus pyogenes*. *Proc. Natl. Acad. Sci. USA* 1993, 90, 7676–7680.
20. Kapur, V.; Topouzis, S.; Majesky, M.W.; Li, L.L.; Hamrick, M.R.; Hamill, R.J.; Patti, J.M.; Musser, J.M. A conserved *Streptococcus pyogenes* extracellular cysteine protease cleaves human fibronectin and degrades vitronectin. *Microb. Pathog.* 1993, 15, 327–346.
21. Rasmussen, M.; Bjorck, L. Proteolysis and its regulation at the surface of *Streptococcus pyogenes*. *Mol. Microbiol.* 2002, 43, 537–544.
22. Nyberg, P.; Rasmussen, M.; Bjorck, L. Alpha2-Macroglobulin-proteinase complexes protect *Streptococcus pyogenes* from killing by the antimicrobial peptide LL-37. *J. Biol. Chem.* 2004, 279, 52820–52823.
23. Rasmussen, M.; Muller, H.P.; Bjorck, L. Protein GRAB of *Streptococcus pyogenes* regulates proteolysis at the bacterial surface by binding alpha2-macroglobulin. *J. Biol. Chem.* 1999, 274, 15336–15344.
24. Lauth, X.; von Kockritz-Blickwede, M.; McNamara, C.W.; Myskowski, S.; Zinkernagel, A.S.; Beall, B.; Ghosh, P.; Gallo, R.L.; Nizet, V. M1 protein allows Group A streptococcal survival in phagocyte extracellular traps through cathelicidin inhibition. *J. Innate. Immun.* 2009, 1, 202–214.
25. Maisey, H.C.; Quach, D.; Hensler, M.E.; Liu, G.Y.; Gallo, R.L.; Nizet, V.; Doran, K.S. A group B streptococcal pilus protein promotes phagocyte resistance and systemic virulence. *FASEB J.* 2008, 22, 1715–1724.
26. Frick, I.M.; Akesson, P.; Rasmussen, M.; Schmidtchen, A.; Bjorck, L. SIC, a secreted protein of *Streptococcus pyogenes* that inactivates antibacterial peptides. *J. Biol. Chem.* 2003, 278, 16561–16566.
27. Fernie-King, B.A.; Seilly, D.J.; Davies, A.; Lachmann, P.J. Streptococcal inhibitor of complement inhibits two additional components of the mucosal innate immune system: Secretory leukocyte proteinase inhibitor and lysozyme. *Infect. Immun.* 2002, 70, 4908–4916.
28. Jin, T.; Bokarewa, M.; Foster, T.; Mitchell, J.; Higgins, J.; Tarkowski, A. *Staphylococcus aureus* resists human defensins by production of staphylokinase, a novel bacterial evasion mechanism. *J. Immunol.* 2004, 172, 1169–1176.
29. Braff, M.H.; Jones, A.L.; Skerrett, S.J.; Rubens, C.E. *Staphylococcus aureus* exploits cathelicidin antimicrobial peptides produced during early pneumonia to promote staphylokinase-dependent fibrinolysis. *J. Infect. Dis.* 2007, 195, 1365–1372.
30. Diep, D.B.; Havarstein, L.S.; Nes, I.F. Characterization of the locus responsible for the bacteriocin production in *Lactobacillus plantarum* C11. *J. Bacteriol.* 1996, 178, 4472–4483.
31. Diep, D.B.; Skaugen, M.; Salehian, Z.; Holo, H.; Nes, I.F. Common mechanisms of target cell recognition and immunity for class II bacteriocins. *Proc. Natl. Acad. Sci. USA* 2007, 104, 2384–2389.
32. Llobet, E.; Tomas, J.M.; Bengoechea, J.A. Capsule polysaccharide is a bacterial decoy for antimicrobial peptides. *Microbiology* 2008, 154, 3877–3886.

33. Vuong, C.; Kocianova, S.; Voyich, J.M.; Yao, Y.; Fischer, E.R.; DeLeo, F.R.; Otto, M. A crucial role for exopolysaccharide modification in bacterial biofilm formation, immune evasion, and virulence. *J. Biol. Chem.* 2004, **279**, 54881–54886.

34. Vuong, C.; Voyich, J.M.; Fischer, E.R.; Braughton, K.R.; Whitney, A.R.; DeLeo, F.R.; Otto, M. Polysaccharide intercellular adhesin (PIA) protects *Staphylococcus epidermidis* against major components of the human innate immune system. *Cell. Microbiol.* 2004, **6**, 269–275.

35. Tsuda, H.; Yamashita, Y.; Shibata, Y.; Nakano, Y.; Koga, T. Genes involved in bacitracin resistance in *Streptococcus mutans*. *Antimicrob. Agents Chemother.* 2002, **46**, 3756–3764.

36. Klein, C.; Entian, K.D. Genes involved in self-protection against the lantibiotic subtilin produced by *Bacillus subtilis* ATCC 6633. *Appl. Environ. Microbiol.* 1994, **60**, 2793–2801.

37. Kuipers, O.P.; Beerthuyzen, M.M.; Siezen, R.J.; de Vos, W.M. Characterization of the nisin gene cluster nisABTCIPR of *Lactococcus lactis*. Requirement of expression of the *nisA* and *nisI* genes for development of immunity. *Eur. J. Biochem.* 1993, **216**, 281–291.

38. Saris, P.E.; Immonen, T.; Reis, M.; Sahl, H.G. Immunity to lantibiotics. *Antonie Van Leeuwenhoek* 1996, **69**, 151–159.

39. Peschel, A.; Vuong, C.; Otto, M.; Gotz, F. The D-alanine residues of *Staphylococcus aureus* teichoic acids alter the susceptibility to vancomycin and the activity of autolytic enzymes. *Antimicrob. Agents Chemother.* 2000, **44**, 2845–2847.

40. Abachin, E.; Poyart, C.; Pellegrini, E.; Milohanic, E.; Fiedler, F.; Berche, P.; Trieu-Cuot, P. Formation of D-alanyl-lipoteichoic acid is required for adhesion and virulence of *Listeria monocytogenes*. *Mol. Microbiol.* 2002, **43**, 1–14.

41. Abi Khattar, Z.; Rejasse, A.; Destoumieux-Garzon, D.; Escoubas, J.M.; Sanchis, V.; Lereclus, D.; Givaudan, A.; Kallasy, M.; Nielsen-Leroux, C.; Gaudriault, S. The dlt operon of *Bacillus cereus* is required for resistance to cationic antimicrobial peptides and for virulence in insects. *J. Bacteriol.* 2009, **191**, 7063–7073.

42. Cox, K.H.; Ruiz-Bustos, E.; Courtney, H.S.; Dale, J.B.; Pence, M.A.; Nizet, V.; Aziz, R.K.; Gerling, I.; Price, S.M.; Hasty, D.L. Inactivation of DltA modulates virulence factor expression in *Streptococcus pyogenes*. *PLoS One* 2009, **4**, e5366.

43. Fisher, N.; Shetron-Rama, L.; Herring-Palmer, A.; Heffernan, B.; Bergman, N.; Hanna, P. The dltABCD operon of *Bacillus anthracis* sterne is required for virulence and resistance to peptide, enzymatic, and cellular mediators of innate immunity. *J. Bacteriol.* 2006, **188**, 1301–1309.

44. Fittipaldi, N.; Sekizaki, T.; Takamatsu, D.; Harel, J.; Dominguez-Punaro Mde, L.; von Aulock, S.; Draing, C.; Marois, C.; Kobisch, M.; Gottschalk, M. D-Alanylation of lipoteichoic acid contributes to the virulence of *Streptococcus suis*. *Infect. Immun.* 2008, **76**, 3587–3594.

45. Poyart, C.; Pellegrini, E.; Marceau, M.; Baptista, M.; Jaubert, F.; Lamy, M.C.; Trieu-Cuot, P. Attenuated virulence of *Streptococcus agalactiae* deficient in D-alanyl-lipoteichoic acid is due to an increased susceptibility to defensins and phagocytic cells. *Mol. Microbiol.* 2003, **49**, 1615–1625.

46. Maloney, E.; Stankowska, D.; Zhang, J.; Fol, M.; Cheng, Q.J.; Lun, S.; Bishai, W.R.; Rajagopalan, M.; Chatterjee, D.; Madiraju, M.V. The two-domain LysX protein of *Mycobacterium tuberculosis* is required for production of lysinylated phosphatidylglycerol and resistance to cationic antimicrobial peptides. *PLoS Pathog.* 2009, **5**, e1000534.

47. Samant, S.; Hsu, F.F.; Neyfakh, A.A.; Lee, H. The *Bacillus anthracis* protein MprF is required for synthesis of lysylphosphatidylglycerols and for resistance to cationic antimicrobial peptides. *J. Bacteriol.* 2009, **191**, 1311–1319.

48. Thedieck, K.; Hain, T.; Mohamed, W.; Tindall, B.J.; Nimtz, M.; Chakraborty, T.; Wehland, J.; Jansch, L. The MprF protein is required for lysinylation of phospholipids in listerial membranes and confers resistance to cationic antimicrobial peptides (CAMPs) on *Listeria monocytogenes*. *Mol. Microbiol.* 2006, **62**, 1325–1339.

49. Peschel, A.; Jack, R.W.; Otto, M.; Collins, L.V.; Staubitz, P.; Nicholson, G.; Kalbacher, H.; Nieuwenhuizen, W.F.; Jung, G.; Tarkowski, A.; *et al.* *Staphylococcus aureus* resistance to human defensins and evasion of neutrophil killing via the novel virulence factor MprF is based on modification of membrane lipids with l-lysine. *J. Exp. Med.* 2001, **193**, 1067–1076.

50. Oku, Y.; Kurokawa, K.; Ichihashi, N.; Sekimizu, K. Characterization of the *Staphylococcus aureus mprF* gene, involved in lysinylation of phosphatidylglycerol. *Microbiology* 2004, **150**, 45–51.

51. Bera, A.; Herbert, S.; Jakob, A.; Vollmer, W.; Gotz, F. Why are pathogenic staphylococci so lysozyme resistant? The peptidoglycan O-acetyltransferase OatA is the major determinant for lysozyme resistance of *Staphylococcus aureus*. *Mol. Microbiol.* 2005, **55**, 778–787.

52. Bera, A.; Biswas, R.; Herbert, S.; Gotz, F. The presence of peptidoglycan O-acetyltransferase in various staphylococcal species correlates with lysozyme resistance and pathogenicity. *Infect. Immun.* 2006, **74**, 4598–4604.

53. Herbert, S.; Bera, A.; Nerz, C.; Kraus, D.; Peschel, A.; Goerke, C.; Meehl, M.; Cheung, A.; Gotz, F. Molecular basis of resistance to muramidase and cationic antimicrobial peptide activity of lysozyme in staphylococci. *PLoS Pathog.* 2007, **3**, e102.

54. Aubry, C.; Goulard, C.; Nahori, M.A.; Cayet, N.; Decalf, J.; Sachse, M.; Boneca, I.G.; Cossart, P.; Dussurget, O. OatA, a peptidoglycan O-acetyltransferase involved in *Listeria monocytogenes* immune escape, is critical for virulence. *J. Infect. Dis.* 2011, **204**, 731–740.

55. Vollmer, W.; Tomasz, A. The *pgdA* gene encodes for a peptidoglycan N-acetylglycosamine deacetylase in *Streptococcus pneumoniae*. *J. Biol. Chem.* 2000, **275**, 20496–20501.

56. Fittipaldi, N.; Sekizaki, T.; Takamatsu, D.; de la Cruz Dominguez-Punaro, M.; Harel, J.; Bui, N.K.; Vollmer, W.; Gottschalk, M. Significant contribution of the *pgdA* gene to the virulence of *Streptococcus suis*. *Mol. Microbiol.* 2008, **70**, 1120–1135.

57. Boneca, I.G.; Dussurget, O.; Cabanes, D.; Nahori, M.A.; Sousa, S.; Lecuit, M.; Psyllinakis, E.; Bouriotis, V.; Hugot, J.P.; Giovannini, M.; *et al.* A critical role for peptidoglycan N-deacetylation in *Listeria* evasion from the host innate immune system. *Proc. Natl. Acad. Sci. USA* 2007, **104**, 997–1002.

58. Laaberki, M.H.; Pfeffer, J.; Clarke, A.J.; Dworkin, J. O-Acetylation of peptidoglycan is required for proper cell separation and S-layer anchoring in *Bacillus anthracis*. *J. Biol. Chem.* 2011, **286**, 5278–5288.

59. Raymond, J.B.; Mahapatra, S.; Crick, D.C.; Pavelka, M.S., Jr. Identification of the *namH* gene, encoding the hydroxylase responsible for the N-glycolylation of the mycobacterial peptidoglycan. *J. Biol. Chem.* 2005, **280**, 326–333.

60. Gajic, O.; Buist, G.; Kojic, M.; Topisirovic, L.; Kuipers, O.P.; Kok, J. Novel mechanism of bacteriocin secretion and immunity carried out by lactococcal multidrug resistance proteins. *J. Biol. Chem.* 2003, 278, 34291–34298.

61. Kupferwasser, L.I.; Skurray, R.A.; Brown, M.H.; Firth, N.; Yeaman, M.R.; Bayer, A.S. Plasmid-mediated resistance to thrombin-induced platelet microbicidal protein in staphylococci: Role of the qacA locus. *Antimicrob. Agents Chemother.* 1999, 43, 2395–2399.

62. Mandin, P.; Fsihi, H.; Dussurget, O.; Vergassola, M.; Milohanic, E.; Toledo-Arana, A.; Lasa, I.; Johansson, J.; Cossart, P. VirR, a response regulator critical for *Listeria monocytogenes* virulence. *Mol. Microbiol.* 2005, 57, 1367–1380.

63. Collins, B.; Curtis, N.; Cotter, P.D.; Hill, C.; Ross, R.P. The ABC transporter AnrAB contributes to the innate resistance of *Listeria monocytogenes* to nisin, bacitracin, and various beta-lactam antibiotics. *Antimicrob. Agents Chemother.* 2010, 54, 4416–4423.

64. Rietkotter, E.; Hoyer, D.; Mascher, T. Bacitracin sensing in *Bacillus subtilis*. *Mol. Microbiol.* 2008, 68, 768–785.

65. Schneider, T.; Kruse, T.; Wimmer, R.; Wiedemann, I.; Sass, V.; Pag, U.; Jansen, A.; Nielsen, A.K.; Mygind, P.H.; Raventos, D.S.; et al. Plectasin, a fungal defensin, targets the bacterial cell wall precursor Lipid II. *Science* 2010, 328, 1168–1172.

66. Staron, A.; Finkeisen, D.E.; Mascher, T. Peptide antibiotic sensing and detoxification modules of *Bacillus subtilis*. *Antimicrob. Agents Chemother.* 2011, 55, 515–525.

67. Mascher, T.; Margulis, N.G.; Wang, T.; Ye, R.W.; Helmann, J.D. Cell wall stress responses in *Bacillus subtilis*: The regulatory network of the bacitracin stimulon. *Mol. Microbiol.* 2003, 50, 1591–604.

68. Ohki, R.; Giyanto; Tateno, K.; Masuyama, W.; Moriya, S.; Kobayashi, K.; Ogasawara, N. The BceRS two-component regulatory system induces expression of the bacitracin transporter, BceAB, in *Bacillus subtilis*. *Mol. Microbiol.* 2003, 49, 1135–1144.

69. Kawada-Matsuo, M.; Yoshida, Y.; Zendo, T.; Nagao, J.; Oogai, Y.; Nakamura, Y.; Sonomoto, K.; Nakamura, N.; Komatsuzawa, H. Three distinct two-component systems are involved in resistance to the class I bacteriocins, Nukacin ISK-1 and nisin A, in *Staphylococcus aureus*. *PLoS One* 2013, 8, e69455.

70. Becker, P.; Hakenbeck, R.; Henrich, B. An ABC transporter of *Streptococcus pneumoniae* involved in susceptibility to vancomycin and bacitracin. *Antimicrob. Agents Chemother.* 2009, 53, 2034–2041.

71. Majchrzykiewicz, J.A.; Kuipers, O.P.; Bijlsma, J.J. Generic and specific adaptive responses of *Streptococcus pneumoniae* to challenge with three distinct antimicrobial peptides, bacitracin, LL-37, and nisin. *Antimicrob. Agents Chemother.* 2010, 54, 440–451.

72. Li, M.; Cha, D.J.; Lai, Y.; Villaruz, A.E.; Sturdevant, D.E.; Otto, M. The antimicrobial peptide-sensing system aps of *Staphylococcus aureus*. *Mol. Microbiol.* 2007, 66, 136–147.

73. Sass, P.; Jansen, A.; Szekat, C.; Sass, V.; Sahl, H.G.; Bierbaum, G. The lantibiotic mersacidin is a strong inducer of the cell wall stress response of *Staphylococcus aureus*. *BMC Microbiol.* 2008, 8, e186.

74. Yoshida, Y.; Matsuo, M.; Oogai, Y.; Kato, F.; Nakamura, N.; Sugai, M.; Komatsuzawa, H. Bacitracin sensing and resistance in *Staphylococcus aureus*. *FEMS Microbiol. Lett.* 2011, 320, 33–39.

75. Hiron, A.; Falord, M.; Valle, J.; Debarbouille, M.; Msadek, T. Bacitracin and nisin resistance in *Staphylococcus aureus*: a novel pathway involving the BraS/BraR two-component system (SA2417/SA2418) and both the BraD/BraE and VraD/VraE ABC transporters. *Mol. Microbiol.* 2011, 81, 602–622.

76. Pietiainen, M.; Francois, P.; Hyyrylainen, H.L.; Tangomo, M.; Sass, V.; Sahl, H.G.; Schrenzel, J.; Kontinen, V.P. Transcriptome analysis of the responses of *Staphylococcus aureus* to antimicrobial peptides and characterization of the roles of vraDE and vraSR in antimicrobial resistance. *BMC Genomics* 2009, 10, e429.

77. Meehl, M.; Herbert, S.; Gotz, F.; Cheung, A. Interaction of the GraRS two-component system with the VraFG ABC transporter to support vancomycin-intermediate resistance in *Staphylococcus aureus*. *Antimicrob. Agents Chemother.* 2007, 51, 2679–2689.

78. Falord, M.; Karimova, G.; Hiron, A.; Msadek, T. GraXSR proteins interact with the VraFG ABC transporter to form a five-component system required for cationic antimicrobial peptide sensing and resistance in *Staphylococcus aureus*. *Antimicrob. Agents Chemother.* 2012, 56, 1047–1058.

79. Li, M.; Lai, Y.; Villaruz, A.E.; Cha, D.J.; Sturdevant, D.E.; Otto, M. Gram-positive three-component antimicrobial peptide-sensing system. *Proc. Natl. Acad. Sci. USA* 2007, 104, 9469–9474.

80. Kramer, N.E.; van Hijum, S.A.; Knol, J.; Kok, J.; Kuipers, O.P. Transcriptome analysis reveals mechanisms by which *Lactococcus lactis* acquires nisin resistance. *Antimicrob. Agents Chemother.* 2006, 50, 1753–1761.

81. Podlesek, Z.; Comino, A.; Herzog-Velikonja, B.; Zgur-Bertok, D.; Komel, R.; Grabnar, M. *Bacillus licheniformis* bacitracin-resistance ABC transporter: Relationship to mammalian multidrug resistance. *Mol. Microbiol.* 1995, 16, 969–976.

82. Manson, J.M.; Keis, S.; Smith, J.M.; Cook, G.M. Acquired bacitracin resistance in *Enterococcus faecalis* is mediated by an ABC transporter and a novel regulatory protein, BcrR. *Antimicrob. Agents Chemother.* 2004, 48, 3743–3748.

83. Matos, R.; Pinto, V.V.; Ruivo, M.; Lopes Mde, F. Study on the dissemination of the bcrABDR cluster in *Enterococcus* spp. reveals that the BcrAB transporter is sufficient to confer high-level bacitracin resistance. *Int. J. Antimicrob. Agents* 2009, 34, 142–147.

84. Diaz, M.; Valdivia, E.; Martinez-Bueno, M.; Fernandez, M.; Soler-Gonzalez, A.S.; Ramirez-Rodrigo, H.; Maqueda, M. Characterization of a new operon, as-48EFGH, from the as-48 gene cluster involved in immunity to enterocin AS-48. *Appl. Environ. Microbiol.* 2003, 69, 1229–1236.

85. McBride, S.M.; Sonenshein, A.L. Identification of a genetic locus responsible for antimicrobial peptide resistance in *Clostridium difficile*. *Infect. Immun.* 2011, 79, 167–176.

86. Suarez, J.M.; Edwards, A.N.; McBride, S.M. The *Clostridium difficile* cpr locus is regulated by a non-contiguous two-component system in response to type A and B lantibiotics. *J. Bacteriol.* 2013, 195, 2621–2631.

87. Otto, M.; Peschel, A.; Gotz, F. Producer self-protection against the lantibiotic epidermin by the ABC transporter EpiFEG of *Staphylococcus epidermidis* Tu3298. *FEMS Microbiol. Lett.* 1998, 166, 203–211.

88. Draper, L.A.; Grainger, K.; Deegan, L.H.; Cotter, P.D.; Hill, C.; Ross, R.P. Cross-immunity and immune mimicry as mechanisms of resistance to the lantibiotic lacticin 3147. *Mol. Microbiol.* 2009, **71**, 1043–1054.

89. McAuliffe, O.; O'Keeffe, T.; Hill, C.; Ross, R.P. Regulation of immunity to the two-component lantibiotic, lacticin 3147, by the transcriptional repressor LtnR. *Mol. Microbiol.* 2001, **39**, 982–993.

90. Papadelli, M.; Karsiotti, A.; Anastasiou, R.; Georgalaki, M.; Tsakalidou, E. Characterization of the gene cluster involved in the biosynthesis of macedocin, the lantibiotic produced by *Streptococcus macedonicus*. *FEMS Microbiol. Lett.* 2007, **272**, 75–82.

91. Altena, K.; Guder, A.; Cramer, C.; Bierbaum, G. Biosynthesis of the lantibiotic mersacidin: Organization of a type B lantibiotic gene cluster. *Appl. Environ. Microbiol.* 2000, **66**, 2565–2571.

92. Guder, A.; Schmitter, T.; Wiedemann, I.; Sahl, H.G.; Bierbaum, G. Role of the single regulator MrsR1 and the two-component system MrsR2/K2 in the regulation of mersacidin production and immunity. *Appl. Environ. Microbiol.* 2002, **68**, 106–113.

93. Chen, P.; Qi, F.; Novak, J.; Caufield, P.W. The specific genes for lantibiotic mutacin II biosynthesis in *Streptococcus mutans* T8 are clustered and can be transferred en bloc. *Appl. Environ. Microbiol.* 1999, **65**, 1356–1360.

94. Siegers, K.; Entian, K.D. Genes involved in immunity to the lantibiotic nisin produced by *Lactococcus lactis* 6F3. *Appl. Environ. Microbiol.* 1995, **61**, 1082–1089.

95. Aso, Y.; Nagao, J.; Koga, H.; Okuda, K.; Kanemasa, Y.; Sashihara, T.; Nakayama, J.; Sonomoto, K. Heterologous expression and functional analysis of the gene cluster for the biosynthesis of and immunity to the lantibiotic, nukacin ISK-1. *J. Biosci. Bioeng.* 2004, **98**, 429–436.

96. Aso, Y.; Sashihara, T.; Nagao, J.; Kanemasa, Y.; Koga, H.; Hashimoto, T.; Higuchi, T.; Adachi, A.; Nomiyama, H.; Ishizaki, A.; *et al.* Characterization of a gene cluster of *Staphylococcus warneri* ISK-1 encoding the biosynthesis of and immunity to the lantibiotic, nukacin ISK-1. *Biosci. Biotechnol. Biochem.* 2004, **68**, 1663–1671.

97. Hyink, O.; Wescombe, P.A.; Upton, M.; Ragland, N.; Burton, J.P.; Tagg, J.R. Salivaricin A2 and the novel lantibiotic salivaricin B are encoded at adjacent loci on a 190-kilobase transmissible megaplasmid in the oral probiotic strain *Streptococcus salivarius* K12. *Appl. Environ. Microbiol.* 2007, **73**, 1107–1113.

98. McLaughlin, R.E.; Ferretti, J.J.; Hynes, W.L. Nucleotide sequence of the streptococcin A-FF22 lantibiotic regulon: model for production of the lantibiotic SA-FF22 by strains of *Streptococcus pyogenes*. *FEMS Microbiol. Lett.* 1999, **175**, 171–177.

99. Biswas, S.; Biswas, I. SmbFT, a putative ABC transporter complex, confers protection against the lantibiotic Smb in *Streptococci*. *J. Bacteriol.* 2013, **195**, 5592–5601.

100. Stein, T.; Heinzmann, S.; Dusterhus, S.; Borchert, S.; Entian, K.D. Expression and functional analysis of the subtilin immunity genes spaIFEG in the subtilin-sensitive host *Bacillus subtilis* MO1099. *J. Bacteriol.* 2005, **187**, 822–828.

101. Rabijns, A.; de Bondt, H.L.; de Ranter, C. Three-dimensional structure of staphylokinase, a plasminogen activator with therapeutic potential. *Nat. Struct. Biol.* 1997, **4**, 357–360.

102. Akesson, P.; Sjoholm, A.G.; Bjorck, L. Protein SIC, a novel extracellular protein of *Streptococcus pyogenes* interfering with complement function. *J. Biol. Chem.* 1996, **271**, 1081–1088.

103. Pence, M.A.; Rooijakkers, S.H.; Cogen, A.L.; Cole, J.N.; Hollands, A.; Gallo, R.L.; Nizet, V. Streptococcal inhibitor of complement promotes innate immune resistance phenotypes of invasive M1T1 group A *Streptococcus*. *J. Innate Immun.* 2010, 2, 587–595.

104. Buckley, A.M.; Spencer, J.; Candlish, D.; Irvine, J.J.; Douce, G.R. Infection of hamsters with the UK *Clostridium difficile* ribotype 027 outbreak strain R20291. *J. Med. Microbiol.* 2011, 60, 1174–1180.

105. Xia, G.; Wolz, C. Phages of *Staphylococcus aureus* and their impact on host evolution. *Infect. Genet. Evol.* 2014, 21, 593–601.

106. Van Wamel, W.J.; Rooijakkers, S.H.; Ruyken, M.; van Kessel, K.P.; van Strijp, J.A. The innate immune modulators staphylococcal complement inhibitor and chemotaxis inhibitory protein of *Staphylococcus aureus* are located on beta-hemolysin-converting bacteriophages. *J. Bacteriol.* 2006, 188, 1310–1315.

107. Coleman, D.C.; Sullivan, D.J.; Russell, R.J.; Arbuthnott, J.P.; Carey, B.F.; Pomeroy, H.M. *Staphylococcus aureus* bacteriophages mediating the simultaneous lysogenic conversion of beta-lysins, staphylokinase and enterotoxin A: Molecular mechanism of triple conversion. *J. Gen. Microbiol.* 1989, 135, 1679–1697.

108. Jin, T.; Bokarewa, M.; McIntyre, L.; Tarkowski, A.; Corey, G.R.; Reller, L.B.; Fowler, V.G., Jr. Fatal outcome of bacteremic patients caused by infection with staphylokinase-deficient *Staphylococcus aureus* strains. *J. Med. Microbiol.* 2003, 52, 919–923.

109. Bisno, A.L.; Brito, M.O.; Collins, C.M. Molecular basis of group A streptococcal virulence. *Lancet Infect. Dis.* 2003, 3, 191–200.

110. Madzivhandila, M.; Adrian, P.V.; Cutland, C.L.; Kuwanda, L.; Madhi, S.A. Distribution of pilus islands of group B *streptococcus* associated with maternal colonization and invasive disease in South Africa. *J. Med. Microbiol.* 2013, 62, 249–253.

111. Maisey, H.C.; Hensler, M.; Nizet, V.; Doran, K.S. Group B streptococcal pilus proteins contribute to adherence to and invasion of brain microvascular endothelial cells. *J. Bacteriol.* 2007, 189, 1464–1467.

112. Chatterjee, C.; Paul, M.; Xie, L.; van der Donk, W.A. Biosynthesis and mode of action of lantibiotics. *Chem. Rev.* 2005, 105, 633–684.

113. Alkhatib, Z.; Abts, A.; Mavaro, A.; Schmitt, L.; Smits, S.H. Lantibiotics: How do producers become self-protected? *J. Biotechnol.* 2012, 159, 145–154.

114. Halami, P.M.; Stein, T.; Chandrashekar, A.; Entian, K.D. Maturation and processing of Spal, the lipoprotein involved in subtilin immunity in *Bacillus subtilis* ATCC 6633. *Microbiol. Res.* 2010, 165, 183–189.

115. Hoffmann, A.; Schneider, T.; Pag, U.; Sahl, H.G. Localization and functional analysis of PepI, the immunity peptide of Pep5-producing *Staphylococcus epidermidis* strain 5. *Appl. Environ. Microbiol.* 2004, 70, 3263–3271.

116. Christ, N.A.; Bochmann, S.; Gottstein, D.; Duchardt-Ferner, E.; Hellmich, U.A.; Dusterhus, S.; Kotter, P.; Guntert, P.; Entian, K.D.; Wohnert, J. The First structure of a lantibiotic immunity protein, Spal from *Bacillus subtilis*, reveals a novel fold. *J. Biol. Chem.* 2012, 287, 35286–35298.

117. Qiao, M.; Immonen, T.; Koponen, O.; Saris, P.E. The cellular location and effect on nisin immunity of the NisI protein from *Lactococcus lactis* N8 expressed in *Escherichia coli* and *L. lactis*. *FEMS Microbiol. Lett.* 1995, 131, 75–80.

118. Takala, T.M.; Saris, P.E. C terminus of NisI provides specificity to nisin. *Microbiology* 2006, 152, 3543–3549.

119. Reis, M.; Eschbach-Bludau, M.; Iglesias-Wind, M.I.; Kupke, T.; Sahl, H.G. Producer immunity towards the lantibiotic Pep5: Identification of the immunity gene pepI and localization and functional analysis of its gene product. *Appl. Environ. Microbiol.* 1994, 60, 2876–2883.

120. Skaugen, M.; Abildgaard, C.I.; Nes, I.F. Organization and expression of a gene cluster involved in the biosynthesis of the lantibiotic lactocin S. *Mol. Gen. Genet.* 1997, 253, 674–686.

121. Heidrich, C.; Pag, U.; Josten, M.; Metzger, J.; Jack, R.W.; Bierbaum, G.; Jung, G.; Sahl, H.G. Isolation, characterization, and heterologous expression of the novel lantibiotic episcidin 280 and analysis of its biosynthetic gene cluster. *Appl. Environ. Microbiol.* 1998, 64, 3140–3146.

122. Twomey, D.; Ross, R.P.; Ryan, M.; Meaney, B.; Hill, C. Lantibiotics produced by lactic acid bacteria: structure, function and applications. *Antonie Van Leeuwenhoek* 2002, 82, 165–185.

123. Peterson, P.K.; Wilkinson, B.J.; Kim, Y.; Schmeling, D.; Quie, P.G. Influence of encapsulation on staphylococcal opsonization and phagocytosis by human polymorphonuclear leukocytes. *Infect. Immun.* 1978, 19, 943–949.

124. Nelson, A.L.; Roche, A.M.; Gould, J.M.; Chim, K.; Ratner, A.J.; Weiser, J.N. Capsule enhances pneumococcal colonization by limiting mucus-mediated clearance. *Infect. Immun.* 2007, 75, 83–90.

125. Ashbaugh, C.D.; Warren, H.B.; Carey, V.J.; Wessels, M.R. Molecular analysis of the role of the group A streptococcal cysteine protease, hyaluronic acid capsule, and M protein in a murine model of human invasive soft-tissue infection. *J. Clin. Invest.* 1998, 102, 550–560.

126. Kogan, G.; Uhrin, D.; Brisson, J.R.; Paoletti, L.C.; Blodgett, A.E.; Kasper, D.L.; Jennings, H.J. Structural and immunochemical characterization of the type VIII group B *Streptococcus* capsular polysaccharide. *J. Biol. Chem.* 1996, 271, 8786–8790.

127. Bentley, S.D.; Aanensen, D.M.; Mavroidi, A.; Saunders, D.; Rabbinowitsch, E.; Collins, M.; Donohoe, K.; Harris, D.; Murphy, L.; Quail, M.A.; et al. Genetic analysis of the capsular biosynthetic locus from all 90 pneumococcal serotypes. *PLoS Genet.* 2006, 2, e31.

128. Candela, T.; Fouet, A. *Bacillus anthracis* CapD, belonging to the gamma-glutamyltranspeptidase family, is required for the covalent anchoring of capsule to peptidoglycan. *Mol. Microbiol.* 2005, 57, 717–726.

129. Deng, L.; Kasper, D.L.; Krick, T.P.; Wessels, M.R. Characterization of the linkage between the type III capsular polysaccharide and the bacterial cell wall of group B *Streptococcus*. *J. Biol. Chem.* 2000, 275, 7497–7504.

130. Mack, D.; Fischer, W.; Krokotsch, A.; Leopold, K.; Hartmann, R.; Egge, H.; Laufs, R. The intercellular adhesin involved in biofilm accumulation of *Staphylococcus epidermidis* is a linear beta-1,6-linked glucosaminoglycan: Purification and structural analysis. *J. Bacteriol.* 1996, 178, 175–183.

131. Campos, M.A.; Vargas, M.A.; Regueiro, V.; Llompart, C.M.; Alberti, S.; Bengoechea, J.A. Capsule polysaccharide mediates bacterial resistance to antimicrobial peptides. *Infect. Immun.* 2004, 72, 7107–7114.

132. Rupp, M.E.; Fey, P.D.; Heilmann, C.; Gotz, F. Characterization of the importance of *Staphylococcus epidermidis* autolysin and polysaccharide intercellular adhesin in the pathogenesis of intravascular catheter-associated infection in a rat model. *J. Infect. Dis.* 2001, 183, 1038–1042.

133. Rupp, M.E.; Ulphani, J.S.; Fey, P.D.; Bartscht, K.; Mack, D. Characterization of the importance of polysaccharide intercellular adhesin/hemagglutinin of *Staphylococcus epidermidis* in the pathogenesis of biomaterial-based infection in a mouse foreign body infection model. *Infect. Immun.* 1999, 67, 2627–2632.

134. Beiter, K.; Wartha, F.; Hurwitz, R.; Normark, S.; Zychlinsky, A.; Henriques-Normark, B. The capsule sensitizes *Streptococcus pneumoniae* to alpha-defensins human neutrophil proteins 1 to 3. *Infect. Immun.* 2008, 76, 3710–3716.

135. Wartha, F.; Beiter, K.; Albiger, B.; Fernebro, J.; Zychlinsky, A.; Normark, S.; Henriques-Normark, B. Capsule and D-alanylated lipoteichoic acids protect *Streptococcus pneumoniae* against neutrophil extracellular traps. *Cell. Microbiol.* 2007, 9, 1162–1171.

136. Jansen, A.; Szekat, C.; Schroder, W.; Wolz, C.; Goerke, C.; Lee, J.C.; Turck, M.; Bierbaum, G. Production of capsular polysaccharide does not influence *Staphylococcus aureus* vancomycin susceptibility. *BMC Microbiol.* 2013, 13, e65.

137. Boman, H.G. Peptide antibiotics and their role in innate immunity. *Annu. Rev. Immunol.* 1995, 13, 61–92.

138. Powers, J.P.; Hancock, R.E. The relationship between peptide structure and antibacterial activity. *Peptides* 2003, 24, 1681–1691.

139. Zasloff, M. Antimicrobial peptides of multicellular organisms. *Nature* 2002, 415, 389–395.

140. Nizet, V. Antimicrobial peptide resistance mechanisms of human bacterial pathogens. *Curr. Issues Mol. Biol.* 2006, 8, 11–26.

141. Peschel, A. How do bacteria resist human antimicrobial peptides? *Trends Microbiol.* 2002, 10, 179–186.

142. Hancock, R.E.; Rozek, A. Role of membranes in the activities of antimicrobial cationic peptides. *FEMS Microbiol. Lett.* 2002, 206, 143–149.

143. Weidenmaier, C.; Peschel, A. Teichoic acids and related cell-wall glycopolymers in Gram-positive physiology and host interactions. *Nat. Rev. Microbiol.* 2008, 6, 276–287.

144. Goldfine, H. Bacterial membranes and lipid packing theory. *J. Lipid. Res.* 1984, 25, 1501–1507.

145. Wiese, A.; Munstermann, M.; Gutsmann, T.; Lindner, B.; Kawahara, K.; Zahringer, U.; Seydel, U. Molecular mechanisms of polymyxin B-membrane interactions: Direct correlation between surface charge density and self-promoted transport. *J. Membr. Biol.* 1998, 162, 127–138.

146. Ernst, C.M.; Peschel, A. Broad-spectrum antimicrobial peptide resistance by MprF-mediated aminoacylation and flipping of phospholipids. *Mol. Microbiol.* 2011, 80, 290–299.

147. Ernst, C.M.; Staubitz, P.; Mishra, N.N.; Yang, S.J.; Hornig, G.; Kalbacher, H.; Bayer, A.S.; Kraus, D.; Peschel, A. The bacterial defensin resistance protein MprF consists of separable domains for lipid lysinylation and antimicrobial peptide repulsion. *PLoS Pathog.* 2009, 5, e1000660.

148. Kristian, S.A.; Durr, M.; van Strijp, J.A.; Neumeister, B.; Peschel, A. MprF-mediated lysinylation of phospholipids in *Staphylococcus aureus* leads to protection against oxygen-independent neutrophil killing. *Infect. Immun.* 2003, 71, 546–549.

149. Bao, Y.; Sakinc, T.; Laverde, D.; Wobser, D.; Benachour, A.; Theilacker, C.; Hartke, A.; Huebner, J. Role of mprF1 and mprF2 in the pathogenicity of *Enterococcus faecalis*. *PLoS One* 2012, 7, e38458.

150. Hachmann, A.B.; Angert, E.R.; Helmann, J.D. Genetic analysis of factors affecting susceptibility of *Bacillus subtilis* to daptomycin. *Antimicrob. Agents Chemother.* 2009, 53, 1598–1609.

151. Maloney, E.; Lun, S.; Stankowska, D.; Guo, H.; Rajagoapalan, M.; Bishai, W.R.; Madiraju, M.V. Alterations in phospholipid catabolism in *Mycobacterium tuberculosis* lysX mutant. *Front. Microbiol.* 2011, 2, e19.

152. Weidenmaier, C.; Peschel, A.; Kempf, V.A.; Lucindo, N.; Yeaman, M.R.; Bayer, A.S. DltABCD- and MprF-mediated cell envelope modifications of *Staphylococcus aureus* confer resistance to platelet microbicidal proteins and contribute to virulence in a rabbit endocarditis model. *Infect. Immun.* 2005, 73, 80338038.

153. Mukhopadhyay, K.; Whitmire, W.; Xiong, Y.Q.; Molden, J.; Jones, T.; Peschel, A.; Staubitz, P.; Adler-Moore, J.; McNamara, P.J.; Proctor, R.A.; *et al.* In vitro susceptibility of *Staphylococcus aureus* to thrombin-induced platelet microbicidal protein-1 (tPMP-1) is influenced by cell membrane phospholipid composition and asymmetry. *Microbiology* 2007, 153, 1187–1197.

154. Ruzin, A.; Severin, A.; Moghazeh, S.L.; Etienne, J.; Bradford, P.A.; Projan, S.J.; Shlaes, D.M. Inactivation of mprF affects vancomycin susceptibility in *Staphylococcus aureus*. *Biochim. Biophys. Acta* 2003, 1621, 117–121.

155. Nishi, H.; Komatsuzawa, H.; Fujiwara, T.; McCallum, N.; Sugai, M. Reduced content of lysyl-phosphatidylglycerol in the cytoplasmic membrane affects susceptibility to moenomycin, as well as vancomycin, gentamicin, and antimicrobial peptides, in *Staphylococcus aureus*. *Antimicrob. Agents Chemother.* 2004, 48, 4800–4807.

156. Jones, T.; Yeaman, M.R.; Sakoulas, G.; Yang, S.J.; Proctor, R.A.; Sahl, H.G.; Schrenzel, J.; Xiong, Y.Q.; Bayer, A.S. Failures in clinical treatment of *Staphylococcus aureus* infection with daptomycin are associated with alterations in surface charge, membrane phospholipid asymmetry, and drug binding. *Antimicrob. Agents Chemother.* 2008, 52, 269–278.

157. Friedman, L.; Alder, J.D.; Silverman, J.A. Genetic changes that correlate with reduced susceptibility to daptomycin in *Staphylococcus aureus*. *Antimicrob. Agents Chemother.* 2006, 50, 2137–2145.

158. Yang, S.J.; Mishra, N.N.; Rubio, A.; Bayer, A.S. Causal role of single nucleotide polymorphisms within the mprF gene of *Staphylococcus aureus* in daptomycin resistance. *Antimicrob. Agents Chemother.* 2013, 57, 5658–5664.

159. Salzberg, L.I.; Helmann, J.D. Phenotypic and transcriptomic characterization of *Bacillus subtilis* mutants with grossly altered membrane composition. *J. Bacteriol.* 2008, 190, 7797–7807.

160. Roy, H.; Ibba, M. Broad range amino acid specificity of RNA-dependent lipid remodeling by multiple peptide resistance factors. *J. Biol. Chem.* 2009, 284, 29677–29683.

161. Mishra, N.N.; Yang, S.J.; Chen, L.; Muller, C.; Saleh-Mghir, A.; Kuhn, S.; Peschel, A.; Yeaman, M.R.; Nast, C.C.; Kreiswirth, B.N.; *et al.* Emergence of daptomycin resistance in daptomycin-naive rabbits with methicillin-resistant *Staphylococcus aureus* prosthetic joint infection is associated with resistance to host defense cationic peptides and mprF polymorphisms. *PLoS One* 2013, 8, e71151.

162. Slavetinsky, C.J.; Peschel, A.; Ernst, C.M. Alanyl-phosphatidylglycerol and lysyl-phosphatidylglycerol are translocated by the same MprF flippases and have similar capacities to protect against the antibiotic daptomycin in *Staphylococcus aureus*. Antimicrob. Agents Chemother. 2012, 56, 3492–3497.

163. McBride, S.M.; Sonenshein, A.L. The dlt operon confers resistance to cationic antimicrobial peptides in *Clostridium difficile*. Microbiology 2011, 157, 1457–1465.

164. Walter, J.; Loach, D.M.; Alqumber, M.; Rockel, C.; Hermann, C.; Pfitzenmaier, M.; Tannock, G.W. D-alanyl ester depletion of teichoic acids in *Lactobacillus reuteri* 100-23 results in impaired colonization of the mouse gastrointestinal tract. Environ. Microbiol. 2007, 9, 1750–1760.

165. Koprivnjak, T.; Mlakar, V.; Swanson, L.; Fournier, B.; Peschel, A.; Weiss, J.P. Cation-induced transcriptional regulation of the dlt operon of *Staphylococcus aureus*. J. Bacteriol. 2006, 188, 3622–3630.

166. Le Jeune, A.; Torelli, R.; Sanguinetti, M.; Giard, J.C.; Hartke, A.; Auffray, Y.; Benachour, A. The extracytoplasmic function sigma factor SigV plays a key role in the original model of lysozyme resistance and virulence of *Enterococcus faecalis*. PLoS One 2010, 5, e9658.

167. Neuhaus, F.C.; Heaton, M.P.; Debabov, D.V.; Zhang, Q. The dlt operon in the biosynthesis of D-alanyl-lipoteichoic acid in *Lactobacillus casei*. Microb. Drug Resist. 1996, 2, 77–84.

168. Perego, M.; Glaser, P.; Minutello, A.; Strauch, M.A.; Leopold, K.; Fischer, W. Incorporation of D-alanine into lipoteichoic acid and wall teichoic acid in *Bacillus subtilis*. Identification of genes and regulation. J. Biol. Chem. 1995, 270, 15598–15606.

169. Neuhaus, F.C.; Baddiley, J. A continuum of anionic charge: Structures and functions of D-alanyl-teichoic acids in gram-positive bacteria. Microbiol. Mol. Biol. Rev. 2003, 67, 686–723.

170. Yang, S.J.; Kreiswirth, B.N.; Sakoulas, G.; Yeaman, M.R.; Xiong, Y.Q.; Sawa, A.; Bayer, A.S. Enhanced expression of dltABCD is associated with the development of daptomycin nonsusceptibility in a clinical endocarditis isolate of *Staphylococcus aureus*. J. Infect. Dis. 2009, 200, 1916–1920.

171. Guariglia-Oropeza, V.; Helmann, J.D. *Bacillus subtilis* sigma(V) confers lysozyme resistance by activation of two cell wall modification pathways, peptidoglycan O-acetylation and D-alanylation of teichoic acids. J. Bacteriol. 2011, 193, 6223–6232.

172. Jann, N.J.; Schmaler, M.; Kristian, S.A.; Radek, K.A.; Gallo, R.L.; Nizet, V.; Peschelm, A.; Landmann, R. Neutrophil antimicrobial defense against *Staphylococcus aureus* is mediated by phagolysosomal but not extracellular trap-associated cathelicidin. J. Leukoc. Biol. 2009, 86, 1159–1169.

173. Saar-Dover, R.; Bitler, A.; Nezer, R.; Shmuel-Galia, L.; Firon, A.; Shimoni, E.; Trieu-Cuot, P.; Shai, Y. D-Alanylation of lipoteichoic acids confers resistance to cationic peptides in group B streptococcus by increasing the cell wall density. PLoS Pathog. 2012, 8, e1002891.

174. Kristian, S.A.; Lauth, X.; Nizet, V.; Goetz, F.; Neumeister, B.; Peschel, A.; Landmann, R. Alanylation of teichoic acids protects *Staphylococcus aureus* against Toll-like receptor 2-dependent host defense in a mouse tissue cage infection model. J. Infect. Dis. 2003, 188, 414–423.

175. Collins, L.V.; Kristian, S.A.; Weidenmaier, C.; Faigle, M.; van Kessel, K.P.; van Strijp, J.A.; Gotz, F.; Neumeister, B.; Peschel, A. *Staphylococcus aureus* strains lacking D-alanine modifications of teichoic acids are highly susceptible to human neutrophil killing and are virulence attenuated in mice. J. Infect. Dis. 2002, 186, 214–219.

176. Meyer, K.; Thompson, R.; Palmer, J.W.; Khorazo, D. The nature of lysozyme action. *Science* 1934, 79, 61.
177. Meyer, K.; Palmer, J.W.; Thompson, R.; Khorazo, D. On the mechanism of lysozyme action. *J. Biol. Chem.* 1936, 113, 479–486.
178. Chipman, D.M.; Sharon, N. Mechanism of lysozyme action. *Science* 1969, 165, 454–465.
179. Nash, J.A.; Ballard, T.N.; Weaver, T.E.; Akinbi, H.T. The peptidoglycan-degrading property of lysozyme is not required for bactericidal activity in vivo. *J. Immunol.* 2006, 177, 519–526.
180. Hebert, L.; Courtin, P.; Torelli, R.; Sanguinetti, M.; Chapot-Chartier, M.P.; Auffray, Y.; Benachour, A. *Enterococcus faecalis* constitutes an unusual bacterial model in lysozyme resistance. *Infect. Immun.* 2007, 75, 5390–5398.
181. Amano, K.; Araki, Y.; Ito, E. Effect of N-acyl substitution at glucosamine residues on lysozyme-catalyzed hydrolysis of cell-wall peptidoglycan and its oligosaccharides. *Eur. J. Biochem.* 1980, 107, 547–553.
182. Amano, K.; Hayashi, H.; Araki, Y.; Ito, E. The action of lysozyme on peptidoglycan with N-unsubstituted glucosamine residues. Isolation of glycan fragments and their susceptibility to lysozyme. *Eur. J. Biochem.* 1977, 76, 299–307.
183. Psylinaakis, E.; Boneca, I.G.; Mavromatis, K.; Deli, A.; Hayhurst, E.; Foster, S.J.; Varum, K.M.; Bouriotis, V. Peptidoglycan N-acetylglycosamine deacetylases from *Bacillus cereus*, highly conserved proteins in *Bacillus anthracis*. *J. Biol. Chem.* 2005, 280, 30856–30863.
184. Blair, D.E.; Schuttelkopf, A.W.; MacRae, J.I.; van Aalten, D.M. Structure and metal-dependent mechanism of peptidoglycan deacetylase, a streptococcal virulence factor. *Proc. Natl. Acad. Sci. USA* 2005, 102, 15429–15434.
185. Vollmer, W.; Tomasz, A. Peptidoglycan N-acetylglycosamine deacetylase, a putative virulence factor in *Streptococcus pneumoniae*. *Infect. Immun.* 2002, 70, 7176–7178.
186. Benachour, A.; Ladjouzi, R.; le Jeune, A.; Hebert, L.; Thorpe, S.; Courtin, P.; Chapot-Chartier, M.P.; Prajsnar, T.K.; Foster, S.J.; Mesnage, S. The lysozyme-induced peptidoglycan N-acetylglycosamine deacetylase PgdA (EF1843) is required for *Enterococcus faecalis* virulence. *J. Bacteriol.* 2012, 194, 6066–6073.
187. Rae, C.S.; Geissler, A.; Adamson, P.C.; Portnoy, D.A. Mutations of the *Listeria monocytogenes* peptidoglycan N-deacetylase and O-acetylase result in enhanced lysozyme sensitivity, bacteriolysis, and hyperinduction of innate immune pathways. *Infect. Immun.* 2011, 79, 3596–3606.
188. Crisostomo, M.I.; Vollmer, W.; Kharat, A.S.; Inhulsen, S.; Gehre, F.; Buckenmaier, S.; Tomasz, A. Attenuation of penicillin resistance in a peptidoglycan O-acetyl transferase mutant of *Streptococcus pneumoniae*. *Mol. Microbiol.* 2006, 61, 1497–1509.
189. Bera, A.; Biswas, R.; Herbert, S.; Kulauzovic, E.; Weidenmaier, C.; Peschel, A.; Gotz, F. Influence of wall teichoic acid on lysozyme resistance in *Staphylococcus aureus*. *J. Bacteriol.* 2007, 189, 280–283.
190. Davis, K.M.; Akinbi, H.T.; Standish, A.J.; Weiser, J.N. Resistance to mucosal lysozyme compensates for the fitness deficit of peptidoglycan modifications by *Streptococcus pneumoniae*. *PLoS Pathog.* 2008, 4, e1000241.

191. Veiga, P.; Bulbarela-Sampieri, C.; Furlan, S.; Maisons, A.; Chapot-Chartier, M.P.; Erkelenz, M.; Mervelet, P.; Noirot, P.; Frees, D.; Kuipers, O.P.; et al. SpxB regulates O-acetylation-dependent resistance of *Lactococcus lactis* peptidoglycan to hydrolysis. *J. Biol. Chem.* 2007, 282, 19342–19354.

192. Shimada, T.; Park, B.G.; Wolf, A.J.; Brikos, C.; Goodridge, H.S.; Becker, C.A.; Reyes, C.N.; Miao, E.A.; Aderem, A.; Gotz, F.; et al. *Staphylococcus aureus* evades lysozyme-based peptidoglycan digestion that links phagocytosis, inflammasome activation, and IL-1beta secretion. *Cell Host Microbe* 2010, 7, 38–49.

193. Brennan, P.J.; Nikaido, H. The envelope of mycobacteria. *Annu. Rev. Biochem.* 1995, 64, 29–63.

194. Mazzotta, A.S.; Montville, T.J. Nisin induces changes in membrane fatty acid composition of *Listeria monocytogenes* nisin-resistant strains at 10 degrees C and 30 degrees C. *J. Appl. Microbiol.* 1997, 82, 32–38.

195. Verheul, A.; Russell, N.J.; van't Hof, R.; Rombouts, F.M.; Abee, T. Modifications of membrane phospholipid composition in nisin-resistant *Listeria monocytogenes* Scott A. *Appl. Environ. Microbiol.* 1997, 63, 3451–3457.

196. Ming, X.T.; Daeschel, M.A. Nisin resistance of foodborne bacteria and the specific resistance responses of *Listeria monocytogenes* Scott A. *J. Food Protect.* 1993, 56, 944–948

197. Crandall, A.D.; Montville, T.J. Nisin resistance in *Listeria monocytogenes* ATCC 700302 is a complex phenotype. *Appl. Environ. Microbiol.* 1998, 64, 231–237.

198. Mishra, N.N.; McKinnell, J.; Yeaman, M.R.; Rubio, A.; Nast, C.C.; Chen, L.; Kreiswirth, B.N.; Bayer, A.S. *In vitro* cross-resistance to daptomycin and host defense cationic antimicrobial peptides in clinical methicillin-resistant *Staphylococcus aureus* isolates. *Antimicrob. Agents Chemother.* 2011, 55, 4012–4018.

199. Mishra, N.N.; Liu, G.Y.; Yeaman, M.R.; Nast, C.C.; Proctor, R.A.; McKinnell, J.; Bayer, A.S. Carotenoid-related alteration of cell membrane fluidity impacts *Staphylococcus aureus* susceptibility to host defense peptides. *Antimicrob. Agents Chemother.* 2011, 55, 526–531.

200. Mishra, N.N.; Rubio, A.; Nast, C.C.; Bayer, A.S. Differential adaptations of methicillin-resistant *Staphylococcus aureus* to serial *in vitro* passage in daptomycin: Evolution of daptomycin resistance and role of membrane carotenoid content and fluidity. *Int. J. Microbiol.* 2012, 2012, e683450.

201. Britton, G. Structure and properties of carotenoids in relation to function. *FASEB J.* 1995, 9, 1551–1558.

202. Pelz, A.; Wieland, K.P.; Putzbach, K.; Hentschel, P.; Albert, K.; Gotz, F. Structure and biosynthesis of staphyloxanthin from *Staphylococcus aureus*. *J. Biol. Chem.* 2005, 280, 32493–32498.

203. Katzif, S.; Lee, E.H.; Law, A.B.; Tzeng, Y.L.; Shafer, W.M. CspA regulates pigment production in *Staphylococcus aureus* through a SigB-dependent mechanism. *J. Bacteriol.* 2005, 187, 8181–8184.

204. Subczynski, W.K.; Wisniewska, A. Physical properties of lipid bilayer membranes: Relevance to membrane biological functions. *Acta Biochim. Pol.* 2000, 47, 613–625.

205. Wisniewska, A.; Subczynski, W.K. Effects of polar carotenoids on the shape of the hydrophobic barrier of phospholipid bilayers. *Biochim. Biophys. Acta* 1998, 1368, 235–246.

206. Bayer, A.S.; Prasad, R.; Chandra, J.; Koul, A.; Smriti, M.; Varma, A.; Skurray, R.A.; Firth, N.; Brown, M.H.; Koo, S.P.; et al. *In vitro* resistance of *Staphylococcus aureus* to thrombin-induced platelet microbicidal protein is associated with alterations in cytoplasmic membrane fluidity. *Infect. Immun.* 2000, 68, 3548–3553.

207. Van Blitterswijk, W.J.; van der Meer, B.W.; Hilkmann, H. Quantitative contributions of cholesterol and the individual classes of phospholipids and their degree of fatty acyl (un)saturation to membrane fluidity measured by fluorescence polarization. *Biochemistry* 1987, 26, 1746–1756.

208. Davidson, A.L.; Chen, J. ATP-binding cassette transporters in bacteria. *Annu. Rev. Biochem.* 2004, 73, 241–268.

209. Davidson, A.L.; Dassa, E.; Orelle, C.; Chen, J. Structure, function, and evolution of bacterial ATP-binding cassette systems. *Microbiol. Mol. Biol. Rev.* 2008, 72, 317–364.

210. Pao, S.S.; Paulsen, I.T.; Saier, M.H., Jr. Major facilitator superfamily. *Microbiol. Mol. Biol. Rev.* 1998, 62, 1–34.

211. Reizer, J.; Reizer, A.; Saier, M.H., Jr. A new subfamily of bacterial ABC-type transport systems catalyzing export of drugs and carbohydrates. *Protein Sci.* 1992, 1, 1326–1332.

212. Stein, T.; Heinzmann, S.; Kiesau, P.; Himmel, B.; Entian, K.D. The spa-box for transcriptional activation of subtilin biosynthesis and immunity in *Bacillus subtilis*. *Mol. Microbiol.* 2003, 47, 1627–1636.

213. Stein, T.; Heinzmann, S.; Solovieva, I.; Entian, K.D. Function of *Lactococcus lactis* nisin immunity genes *nisI* and *nisFEG* after coordinated expression in the surrogate host *Bacillus subtilis*. *J. Biol. Chem.* 2003, 278, 89–94.

214. Immonen, T.; Saris, P.E. Characterization of the *nisFEG* operon of the nisin Z producing *Lactococcus lactis* subsp. *lactis* N8 strain. *DNA Seq.* 1998, 9, 263–274.

215. Ra, S.R.; Qiao, M.; Immonen, T.; Pujana, I.; Saris, E.J. Genes responsible for nisin synthesis, regulation and immunity form a regulon of two operons and are induced by nisin in *Lactococcus lactis* N8. *Microbiology* 1996, 142, 1281–1288.

216. Aso, Y.; Okuda, K.; Nagao, J.; Kanemasa, Y.; Thi Bich Phuong, N.; Koga, H.; Shioya, K.; Sashihara, T.; Nakayama, J.; Sonomoto, K. A novel type of immunity protein, NukH, for the lantibiotic nukacin ISK-1 produced by *Staphylococcus warneri* ISK-1. *Biosci. Biotechnol. Biochem.* 2005, 69, 1403–1410.

217. Okuda, K.; Yanagihara, S.; Shioya, K.; Harada, Y.; Nagao, J.; Aso, Y.; Zendo, T.; Nakayama, J.; Sonomoto, K. Binding specificity of the lantibiotic-binding immunity protein NukH. *Appl. Environ. Microbiol.* 2008, 74, 7613–7619.

218. Gebhard, S. ABC transporters of antimicrobial peptides in Firmicutes bacteria—Phylogeny, function and regulation. *Mol. Microbiol.* 2012, 86, 1295–1317.

219. Higgins, C.F. ABC transporters: Physiology, structure and mechanism—An overview. *Res. Microbiol.* 2001, 152, 205–210.

220. Dintner, S.; Staron, A.; Berchtold, E.; Petri, T.; Mascher, T.; Gebhard, S. Coevolution of ABC transporters and two-component regulatory systems as resistance modules against antimicrobial peptides in Firmicutes bacteria. *J. Bacteriol.* 2011, 193, 3851–3862.

221. Revilla-Guarinos, A.; Gebhard, S.; Mascher, T.; Zuniga, M. Defence against antimicrobial peptides: Different strategies in Firmicutes. *Environ. Microbiol.* 2014, 16, 1225–1237.

222. Bernard, R.; El Ghachi, M.; Mengin-Lecreulx, D.; Chippaux, M.; Denizot, F. BcrC from *Bacillus subtilis* acts as an undecaprenyl pyrophosphate phosphatase in bacitracin resistance. *J. Biol. Chem.* 2005, 280, 28852–28857.

223. Shaaly, A.; Kalamorz, F.; Gebhard, S.; Cook, G.M. Undecaprenyl pyrophosphate phosphatase confers low-level resistance to bacitracin in *Enterococcus faecalis*. *J. Antimicrob. Chemother.* 2013, 68, 1583–1593.

224. Charlebois, A.; Jalbert, L.A.; Harel, J.; Masson, L.; Archambault, M. Characterization of genes encoding for acquired bacitracin resistance in *Clostridium perfringens*. *PLoS One* 2012, 7, e44449.

225. Butaye, P.; Cloeckaert, A.; Schwarz, S. Mobile genes coding for efflux-mediated antimicrobial resistance in Gram-positive and Gram-negative bacteria. *Int. J. Antimicrob. Agents* 2003, 22, 205–210.

226. Van Veen, H.W.; Venema, K.; Bolhuis, H.; Oussenko, I.; Kok, J.; Poolman, B.; Driessen, A.J.; Konings, W.N. Multidrug resistance mediated by a bacterial homolog of the human multidrug transporter MDR1. *Proc. Natl. Acad. Sci. USA* 1996, 93, 10668–10672.

227. Saidijam, M.; Benedetti, G.; Ren, Q.; Xu, Z.; Hoyle, C.J.; Palmer, S.L.; Ward, A.; Bettaney, K.E.; Szakonyi, G.; Meuller, J.; *et al.* Microbial drug efflux proteins of the major facilitator superfamily. *Curr. Drug Targets* 2006, 7, 793–811.

228. Littlejohn, T.G.; Paulsen, I.T.; Gillespie, M.T.; Tennent, J.M.; Midgley, M.; Jones, I.G.; Purewal, A.S.; Skurray, R.A. Substrate specificity and energetics of antiseptic and disinfectant resistance in *Staphylococcus aureus*. *FEMS Microbiol. Lett.* 1992, 74, 259–265.

229. Leelaporn, A.; Paulsen, I.T.; Tennent, J.M.; Littlejohn, T.G.; Skurray, R.A. Multidrug resistance to antiseptics and disinfectants in coagulase-negative staphylococci. *J. Med. Microbiol.* 1994, 40, 214–220.

230. Bayer, A.S.; Cheng, D.; Yeaman, M.R.; Corey, G.R.; McClelland, R.S.; Harrel, L.J.; Fowler, V.G., Jr. *In vitro* resistance to thrombin-induced platelet microbicidal protein among clinical bacteremic isolates of *Staphylococcus aureus* correlates with an endovascular infectious source. *Antimicrob. Agents Chemother.* 1998, 42, 3169–3172.

231. Solheim, M.; Aakra, A.; Vebo, H.; Snipen, L.; Nes, I.F. Transcriptional responses of *Enterococcus faecalis* V583 to bovine bile and sodium dodecyl sulfate. *Appl. Environ. Microbiol.* 2007, 73, 5767–5774.

232. Fernandez-Fuentes, M.A.; Abriouel, H.; Ortega Morente, E.; Perez Pulido, R.; Galvez, A. Genetic determinants of antimicrobial resistance in Gram positive bacteria from organic foods. *Int. J. Food Microbiol.* 2014, 172, 49–56.

233. Gay, K.; Stephens, D.S. Structure and dissemination of a chromosomal insertion element encoding macrolide efflux in *Streptococcus pneumoniae*. *J. Infect. Dis.* 2001, 184, 56–65.

234. Peschel, A.; Sahl, H.G. The co-evolution of host cationic antimicrobial peptides and microbial resistance. *Nat. Rev. Microbiol.* 2006, 4, 529–536.

235. Eckert, R. Road to clinical efficacy: Challenges and novel strategies for antimicrobial peptide development. *Future Microbiol.* 2011, 6, 635–651.

236. Marr, A.K.; Gooderham, W.J.; Hancock, R.E. Antibacterial peptides for therapeutic use: Obstacles and realistic outlook. *Curr. Opin. Pharmacol.* 2006, 6, 468–472.

237. Hancock, R.E.; Sahl, H.G. Antimicrobial and host-defense peptides as new anti-infective therapeutic strategies. *Nat. Biotechnol.* 2006, 24, 1551–1557.

238. Van Heel, A.J.; Mu, D.; Montalban-Lopez, M.; Hendriks, D.; Kuipers, O.P. Designing and producing modified, new-to-nature peptides with antimicrobial activity by use of a combination of various lantibiotic modification enzymes. *ACS Synth. Biol.* 2013, 2, 397–404.

239. Jung, W.J.; Mabood, F.; Souleimanov, A.; Zhou, X.; Jaoua, S.; Kamoun, F.; Smith, D.L. Stability and antibacterial activity of bacteriocins produced by *Bacillus thuringiensis* and *Bacillus thuringiensis* ssp. *kurstaki*. *J. Microbiol. Biotechnol.* 2008, 18, 1836–1840.

240. Chehimi, S.; Delalande, F.; Sable, S.; Hajlaoui, M.R.; van Dorsselaer, A.; Limam, F.; Pons, A.M. Purification and partial amino acid sequence of thuricin S, a new anti-Listeria bacteriocin from *Bacillus thuringiensis*. *Can. J. Microbiol.* 2007, 53, 284–290.

241. Weigel, L.M.; Clewell, D.B.; Gill, S.R.; Clark, N.C.; McDougal, L.K.; Flannagan, S.E.; Kolonay, J.F.; Shetty, J.; Killgore, G.E.; Tenover, F.C. Genetic analysis of a high-level vancomycin-resistant isolate of *Staphylococcus aureus*. *Science* 2003, 302, 1569–1571.

242. Huddleston, J.R. Horizontal gene transfer in the human gastrointestinal tract: Potential spread of antibiotic resistance genes. *Infect. Drug. Resist.* 2014, 7, 167–176.

243. Napier, B.A.; Band, V.; Burd, E.M.; Weiss, D.S. Colistin heteroresistance in *Enterobacter cloacae* is associated with cross-resistance to the host antimicrobial lysozyme. *Antimicrob. Agents Chemother.* 2014, 58, 5594–5597.

© 2014 by the authors; licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution license (http://creativecommons.org/licenses/by/4.0/).
