
LETTER

**Structure of human cytoplasmic dynein-2 primed for its power stroke**

Helgo Schmidt${}^{1*}$, Ruta Zalyte${}^{1*}$, Linas Urnavicius${}^{1}$ & Andrew P. Carter${}^{1}$

Members of the dynein family, consisting of cytoplasmic and axonemal isoforms, are motors that move towards the minus ends of microtubules. Cytoplasmic dynein-1 (dynein-1) plays roles in mitosis and cellular cargo transport${}^{1}$, and is implicated in viral infections${}^{2}$ and neurodegenerative diseases${}^{3}$. Cytoplasmic dynein-2 (dynein-2) performs intraflagellar transport${}^{4}$ and is associated with human skeletal ciliopathies${}^{5}$. Dyneins share a conserved motor domain that couples cycles of ATP hydrolysis with conformational changes to produce movement${}^{6-9}$. Here we present the crystal structure of the human cytoplasmic dynein-2 motor bound to the ATP-hydrolysis transition state analogue ADP.vanadate${}^{10}$. The structure reveals a closure of the motor’s ring of six AAA+ domains (ATPases associated with various cellular activites: AAA1–AAA6). This induces a steric clash with the linker, the key element for the generation of movement, driving it into a conformation that is primed to produce force. Ring closure also changes the interface between the stalk and buttress coiled-coil extensions of the motor domain. This drives helix sliding in the stalk which causes the microtubule binding domain at its tip to release from the microtubule. Our structure answers the key questions of how ATP hydrolysis leads to linker remodelling and microtubule affinity regulation.

There are four nucleotide-binding sites in the dynein motor, but movement only depends on ATP hydrolysis in the first site (AAA1)${}^{7,11,12}$. When this site is nucleotide free or bound to ADP, the microtubule binding domain (MTBD) binds to the microtubule and the linker adopts the straight post-power-stroke conformation${}^{6-8,12-14}$. Upon ATP binding and hydrolysis, the MTBD detaches from the microtubule and the linker is primed into the pre-power-stroke conformation${}^{6,12,14,15}$ (Fig. 1a). MTBD rebinding leads to a force producing swing of the linker (power stroke) back to the post-power-stroke position and the release of ATP hydrolysis products to reset the cycle${}^{6,14-16}$.

To address how the linker is primed and dynein released from microtubules, we co-crystallized the human dynein-2 motor domain with ADP.vanadate (ADP.Vi) to trap it in a pre-power-stroke state${}^{6}$ (Extended Data Fig. 1 and Extended Data Table 1). The linker in this dynein-2:ADP.Vi structure has a $90^\circ$ bend (Fig. 1b) consistent with low-resolution studies of pre-power-stroke dynein${}^{6,8,9,17}$. Dynein’s AAA+ domains are each divided into an $\alpha/\beta$ ‘large’ subdomain (AAAL, helices H0–H4 and beta strands S1–S5) and an $\alpha$ ‘small’ subdomain (AAAS, helices H5–H9)${}^{16}$. The individual subdomains of dynein-2:ADP.Vi are highly similar to those in post-power-stroke crystal structures of dynein-1 from *Saccharomyces cerevisiae*${}^{13}$ (dynein-1:APO; Protein Data Bank (PDB) accession number 4AKI) and *Dictyostelium discoideum*${}^{18}$ (dynein-1:ADP; PDB accession number 3VKG) (Extended Data Fig. 2 and Supplementary Data 1). This suggests conformational changes between these structures (Supplementary Discussion and Extended Data Fig. 3a) are not related to sequence differences but are caused by the different nucleotide states.

In dynein-2:ADP.Vi, all four nucleotide-binding sites are occupied (Fig. 1c). The AAA1 site, found between AAA+ domains AAA1 and AAA2, binds ADP.Vi (Fig. 2a and Extended Data Fig. 4a–d) via conserved motifs${}^{19}$ (Fig. 2b). The trigonal-bipyramidal vanadate group mimics the ATP $\gamma$-phosphate during hydrolysis${}^{10}$. It is surrounded by three important catalytic residues${}^{19}$: the Walker B glutamate (W-B: E1742), the sensor-I asparagine (S-I: N1792) and the AAA2L arginine finger (RF: R2109), suggesting the structure is in the ATP hydrolysis-competent conformation.

In the dynein-1 structures there is a gap between AAA1 and AAA2. The closure of this gap in dynein-2:ADP.Vi (Supplementary Video 1) is driven by the arginine finger-ADP.Vi contact. It is reinforced by additional interactions between AAA1L and AAA2L (Fig. 2c). A pair of conserved inserts in AAA2L${}^{20}$ (the ‘H2 insert’ and the ‘pre-sensor-I’ (PS-I) insert) contact the H2 helix in AAA1L (Fig. 2a, c) and displace H2 and H3 relative to the rest of AAA1L (Extended Data Fig. 5a). The AAA1L sensor-I loop, which varies in position depending on dynein’s nucleotide state (Extended Data Fig. 5b), swings in to contact AAA2L (Fig. 2c).

The other nucleotide-binding sites contain tightly bound nucleotides that co-purify with the motor domain (Extended Data Fig. 4e–j and Supplementary Discussion). The density in AAA2 is consistent with an ATP, as observed in all the dynein structures${}^{13,18}$. As in dynein-1:ADP, the densities in AAA3 and AAA4 suggest the presence of ADP. In the dynein-1:APO structure these sites are empty. In all dynein structures

![Figure 1](attachment://figure_1.png)

**Figure 1 | Crystal structure of dynein-2:ADP.Vi.** a, Schematic representation of a dynein motor domain in post- and pre-power-stroke states. Structural elements are labelled and colour-coded. AAA+ domains (1–6) consist of large (AAAL) and small (AAAS) subdomains. The coiled-coil stalk is supported by the coiled-coil buttress and harbours the MTBD. A carboxy (C)-terminal domain runs underneath the AAA+ ring. b, Overview of dynein-2:ADP.Vi in cartoon/surface representation. The linker features a $90^\circ$ bend. c, Nucleotides (NT1–NT4, sphere representations) are mainly bound between AAA+ large domains (colour-coded). AAA1L and AAA2L form the important AAA1 nucleotide-binding site.

---

${}^{1}$Medical Research Council Laboratory of Molecular Biology, Division of Structural Studies, Francis Crick Avenue, Cambridge CB2 0QH, UK.  
*These authors contributed equally to this work.*

©2014 Macmillan Publishers Limited. All rights reserved

Figure 2 | ADP.Vi binding to AAA1 nucleotide-binding site induces closure of AAA1/AAA2 interface. a, AAA1L, AAA1S and AAA2L enclose ADP.Vi. The AAA2L H2 and PS-I inserts (red) contact AAA1L H2. b, Upper panel: the Mg²⁺.ADP contacts the Walker A (W-A: K1695), Walker B (W-B: D1741) and the sensor-II (S-II: R1867) residues. The trigonal-bipyramidal Vi-group mimics the ATP-hydrolysis transition state and is surrounded by sensor-I

(S-I: N1792) and Walker B (W-B: E1742) residues and the AAA2L arginine finger (RF). Lower panel: schematic diagram showing the distances from ADP.Vi to the catalytic residues (in Å). c, AAA1/AAA2 interface closure is reinforced by the AAA1L sensor-I loop contacting AAA2L. Purple spheres represent contacts.

We had anticipated that the movement of the mobile part of the linker would be driven by its interaction with the highly conserved inserts in AAA2L¹³,¹⁴,¹⁸. It was therefore surprising to find that these inserts contact only the static part of the linker. How then could AAA1 site closure induce linker bending? To address this question we asked what would happen if the AAA+ ring adopted the ADP.Vi state but the linker remained in the straight, post-power-stroke conformation. The rigid-body behaviour of AAA2–AAA4 means that closure of the AAA1 site would lead to a steric clash between the mobile Link1–2 region and the AAA4L PS-I insert (Fig. 3e). This is demonstrated by the overlap between these regions observed in an alignment of the straight linker from dynein-1:ADP onto the dynein-2:ADP.Vi structure (Fig. 3f). The additional contacts between the AAA+ ring and the static Link3–4 (Fig. 3b) prevent it moving and mean that the clash can only be relieved by the mobile Link1–2 adopting its pre-power-stroke position (Fig. 3e, g and Supplementary Video 3).

To test this model we used negative-stain electron microscopy to assay linker movement (Fig. 3h, i and Extended Data Fig. 8a, b). In the presence of ADP all dynein-2 motors had an angle between the stalk and linker of 54 ± 13° (mean ± s.d.) (Fig. 3h). In the presence of ADP.Vi the majority of motors showed a pre-power-stroke conformation with an angle of 145 ± 20°. We then tested the ability of dynein-2 mutants to adopt the pre-power-stroke state in the presence of ADP.Vi (Fig. 3i). Removal of the AAA2L inserts abolished the linker movement, consistent with previous data¹⁸. It also completely prohibited microtubule gliding activity (Extended Data Fig. 8c). When the AAA4L PS-I insert was deleted only a small percentage of motors attained the pre-power-stroke

Figure 3 | Linker bending upon closure of AAA1 nucleotide-binding site.

a, A $90^\circ$ bend between linker subdomains 1 and 2 (Link1–2) and 3 and 4 (Link3–4) forces the hinge helix (H10) to curve.

b, The AAA2L PS-I insert contacts Link3 and AAA1L R1726 forms a salt bridge with E1420 on the hinge helix.

c, Hydrophobic residues (yellow) stabilize the Link2/Link3 bend.

d, Link1–2 contacts the AAA2:H2 and AAA3L:H2–S3 inserts.

e, AAA1 site (blue/cyan) closure causes a rigid-body movement of AAA2–AAA3–AAA4 (cyan-green-yellow) leading to a clash (black star) with Link1–2 (light pink). To relieve the clash, the linker adopts the pre-power-stroke conformation.

f, The straight post-power-stroke linker (grey), aligned via Link3–4 onto dynein-2:ADP.Vi, would clash with the AAA4L PS-I insert (yellow spheres).

g, In dynein-2:ADP.Vi the linker moved to avoid the clash.

h, In a

conformation (Fig. 3i), supporting our model that the AAA4L PS-I insert plays a major role in linker bending. In agreement with this interpretation, the microtubule gliding velocity of this mutant was only 10% of wild type (Extended Data Fig. 8c). In contrast, removal of the Link1–2 contacts with AAA2L and AAA3L had a minimal effect on linker movement (Fig. 3i).

negative-stain electron microscopy assay, the angle between the stalk (yellow) and green fluorescent protein (GFP)–linker (green/purple) of the dynein-2:ADP motor is $54 \pm 13^\circ$ (mean ± s.d.). With ADP.Vi most dynein-2 motors are in a pre-power-stroke state with an angle of $145 \pm 20^\circ$ (mean ± s.d.).

i, Deletion of the AAA2L ($\Delta$AAA2L:H2 + PS-I) or AAA4L inserts ($\Delta$AAA4L:PS-I) hinders the linker adopting the pre-power-stroke conformation with ADP.Vi. Mutation of either AAA3 ($\Delta$AAA3L:H2–S3) or AAA2 (R1413A + E2028A) linker-ring contacts has no effect. Dashed half-circles in h and i mark 20% dynein-2 particles. All negative-stain electron microscopy experiments were done in triplicate. The number of particles used in each experiment are provided in Methods.

In addition to triggering movement of the linker, ADP.Vi binding to dynein reduces the affinity of its MTBD for microtubules ${}^{21}$. Biochemical ${}^{22,23}$ and structural ${}^{14,18,24-26}$ evidence suggests this involves the helices in the stalk, coiled-coil helix 1 (CC1) and coiled-coil helix 2 (CC2), sliding past each other by one turn of $\alpha$-helix. The dynein-2:ADP.Vi structure, where the MTBD has low microtubule affinity (Extended Data Fig. 9

RESEARCH LETTER

![](image.png)

Figure 4 | Buttress movement triggers helix sliding in the stalk.  
a, In dynein-2:ADP.Vi the stalk CC2 is kinked. Compared with the dynein-1:ADP stalk, CC2 is displaced by one turn of α-helix relative to CC1. Blue and red spheres represent equivalent amino-acid residues in the two structures.  
b, The two stalk conformations are stabilized by movement of the buttress, which slips relative to CC1, but moves with CC2 when comparing the two structures. The buttress movement is the result of a rotation of the AAA6L/AAA5S unit (shown by the position of the dashed black line).

and Supplementary Discussion), answers the key question of how the sliding is initiated.

In dynein-2:ADP.Vi the base of the stalk deviates from the symmetrical, regular coiled coil observed in dynein-1:ADP (Fig. 4a). The stalk CC2 helix contains a kink, located near the stalk/buttress interface, which causes it to slip relative to CC1. The resulting asymmetry between the two helices is similar to that observed in the parallel coiled-coil homodimer Bicaudal-D²⁷. A comparison of the dynein-1:ADP and dynein-2:ADP.Vi structures (Fig. 4b) suggests how the movement of the buttress, relative to the stalk, is coupled to the movement of CC2. In dynein-2:ADP.Vi the buttress slides relative to CC1 but moves together with CC2.

The stalk and buttress emerge from AAA4S and AAA5S respectively. Their relative movement is coupled to rearrangements in the AAA+ ring. Closure of the AAA1 site and the rigid body movement of AAA2–AAA4 force the AAA4/AAA5 interface to close and the AAA6L subdomain to rotate towards the ring centre (Fig. 4b and Supplementary Discussion). The AAA5S subdomain rotates as a unit together with AAA6L, and this movement pulls the buttress relative to the stalk (Supplementary Videos 4 and 5).

Unlike myosin and kinesin motors, dynein shares mechanistic similarities with AAA+ proteins that remodel their substrates²⁸. In dynein, one substrate is the linker which is bent by a clash with the AAA+ ring. This bent conformation is stabilized by contacts at the Link2/Link3 interface, the importance of which is highlighted by the fact that a mutation there (G1442D) can cause the human ciliopathy Jeune syndrome⁵ (Supplementary Discussion). When the AAA1 site reopens, the bent linker reverts to its preferred straight conformation⁸ and generates force. In addition to the linker, the dynein AAA+ ring also remodels the stalk. Here the motions of AAA+ domains are directly coupled to sliding of helices in the coiled coil (Supplementary Video 6).

**Online Content** Methods, along with any additional Extended Data display items and Source Data, are available in the online version of the paper; references unique to these sections appear only in the online paper.

Received 11 August; accepted 29 October 2014.  
Published online 1 December 2014.

1. Roberts, A. J., Kon, T., Knight, P. J., Sutoh, K. & Burgess, S. A. Functions and mechanics of dynein motor proteins. *Nature Rev. Mol. Cell Biol.* **14**, 713–726 (2013).
2. Dodding, M. P. & Way, M. Coupling viruses to dynein and kinesin-1. *EMBO J.* **30**, 3527–3539 (2011).
3. Schiavo, G., Greensmith, L., Hafezparast, M. & Fisher, E. M. Cytoplasmic dynein heavy chain: the servant of many masters. *Trends Neurosci.* **36**, 641–651 (2013).
4. Ishikawa, H. & Marshall, W. F. Ciliogenesis: building the cell’s antenna. *Nature Rev. Mol. Cell Biol.* **12**, 222–234 (2011).
5. Schmidts, M. *et al.* Exome sequencing identifies DYNC2H1 mutations as a common cause of asphyxiating thoracic dystrophy (Jeune syndrome) without major polydactyly, renal or retinal involvement. *J. Med. Genet.* **50**, 309–323 (2013).
6. Burgess, S. A., Walker, M. L., Sakakibara, H., Knight, P. J. & Oiwa, K. Dynein structure and power stroke. *Nature* **421**, 715–718 (2003).
7. Kon, T., Mogami, T., Ohkura, R., Nishiura, M. & Sutoh, K. ATP hydrolysis cycle-dependent tail motions in cytoplasmic dynein. *Nature Struct. Mol. Biol.* **12**, 513–519 (2005).
8. Roberts, A. J. *et al.* ATP-driven remodeling of the linker domain in the dynein motor. *Structure* **20**, 1670–1680 (2012).
9. Roberts, A. J. *et al.* AAA+ ring and linker swing mechanism in the dynein motor. *Cell* **136**, 485–495 (2009).
10. Davies, D. R. & Hol, W. G. The power of vanadate in crystallographic investigations of phosphoryl transfer enzymes. *FEBS Lett.* **577**, 315–321 (2004).
11. Gibbons, I. R., Gibbons, B. H., Mocz, G. & Asai, D. J. Multiple nucleotide-binding sites in the sequence of dynein β heavy chain. *Nature* **352**, 640–643 (1991).
12. Kon, T., Nishiura, M., Ohkura, R., Toyoshima, Y. Y. & Sutoh, K. Distinct functions of nucleotide-binding/hydrolysis sites in the four AAA modules of cytoplasmic dynein. *Biochemistry* **43**, 11266–11274 (2004).
13. Schmidt, H., Gleave, E. S. & Carter, A. P. Insights into dynein motor domain function from a 3.3-Å crystal structure. *Nature Struct. Mol. Biol.* **19**, 492–497 (2012).
14. Carter, A. P. Crystal clear insights into how the dynein motor moves. *J. Cell Sci.* **126**, 705–713 (2013).
15. Imamula, K., Kon, T., Ohkura, R. & Sutoh, K. The coordination of cyclic microtubule association/dissociation and tail swing of cytoplasmic dynein. *Proc. Natl Acad. Sci. USA* **104**, 16134–16139 (2007).
16. Carter, A. P., Cho, C., Jin, L. & Vale, R. D. Crystal structure of the dynein motor domain. *Science* **331**, 1159–1165 (2011).
17. Lin, J., Okada, K., Raytchev, M., Smith, M. C. & Nicastro, D. Structural mechanism of the dynein power stroke. *Nature Cell Biol.* **16**, 479–485 (2014).
18. Kon, T. *et al.* The 2.8 Å crystal structure of the dynein motor domain. *Nature* **484**, 345–350 (2012).
19. Wendler, P., Ciniawsky, S., Kock, M. & Kube, S. Structure and function of the AAA+ nucleotide binding pocket. *Biochim. Biophys. Acta* **1823**, 2–14 (2012).
20. Gleave, E. S., Schmidt, H. & Carter, A. P. A structural analysis of the AAA+ domains in *Saccharomyces cerevisiae* cytoplasmic dynein. *J. Struct. Biol.* **186**, 367–375 (2014).
21. Vale, R. D., Soll, D. R. & Gibbons, I. R. One-dimensional diffusion of microtubules bound to flagellar dynein. *Cell* **59**, 915–925 (1989).
22. Gibbons, I. R. *et al.* The affinity of the dynein microtubule-binding domain is modulated by the conformation of its coiled-coil stalk. *J. Biol. Chem.* **280**, 23960–23965 (2005).
23. Kon, T. *et al.* Helix sliding in the stalk coiled coil of dynein couples ATPase and microtubule binding. *Nature Struct. Mol. Biol.* **16**, 325–333 (2009).
24. Carter, A. P. *et al.* Structure and functional role of dynein’s microtubule-binding domain. *Science* **322**, 1691–1695 (2008).
25. Nishikawa, Y. *et al.* Structure of the entire stalk region of the dynein motor domain. *J. Mol. Biol.* **426**, 3232–3245 (2014).
26. Redwine, W. B. *et al.* Structural basis for microtubule binding and release by dynein. *Science* **337**, 1532–1536 (2012).
27. Liu, Y. *et al.* Bicaudal-D uses a parallel, homodimeric coiled coil with heterotypic registry to coordinate recruitment of cargos to dynein. *Genes Dev.* **27**, 1233–1246 (2013).
28. Erzberger, J. P. & Berger, J. M. Evolutionary relationships and structural mechanisms of AAA+ proteins. *Annu. Rev. Biophys. Biomol. Struct.* **35**, 93–114 (2006).

**Supplementary Information** is available in the online version of the paper.

**Acknowledgements** This work was funded by the Medical Research Council, UK (MC_UP_A025_1011), a Wellcome Trust New Investigator Award (WT100387) and an EMBO Young Investigator Award. We thank M. Yu for in-house support with X-ray data collection and Diamond Light Source for access to beamline I02 (MX8547-70). We also thank G. Dornan and M. Barczyk for assistance with insect cell culture. We also thank S. Bullock and D. Barford for their advice and comments on the manuscript.

**Author Contributions** R.Z. and H.S. screened many dynein species for expression and crystallization. R.Z. expressed human dynein-2 in insect cells, obtained crystals in the presence of vanadate and collected data. H.S. phased the structure and built an initial model. A.P.C. built and refined the structure. R.Z. and H.S. made mutants and performed biochemical assays. L.U. performed negative-stain electron microscopy. H.S., R.Z. and A.P.C. prepared the manuscript.

**Author Information** Coordinates and structure factors have been deposited in the Protein Data Bank under accession number 4RH7. Reprints and permissions information is available at www.nature.com/reprints. The authors declare no competing financial interests. Readers are welcome to comment on the online version of the paper. Correspondence and requests for materials should be addressed to A.P.C. (cartera@mrc-lmb.cam.ac.uk).

©2014 Macmillan Publishers Limited. All rights reserved.

METHODS

Cloning of constructs. DNA sequence coding for a variant of human cytoplasmic dynein-2 isoform 1 (GenBank reference number BAG06721), codon-optimized for expression in *Spodoptera frugiperda* (Sf9) cells, was amplified (coding region D1091–Q4307) using Phusion High-Fidelity DNA Polymerase (New England Biolabs). The primers used for construct amplification contained sites homologous to a pFastBac vector (Invitrogen Life Science Technologies) that had been modified to contain a tobacco etch virus protease cleavable tandem Protein A-tag for purification followed by a GFP. InFusion (Clontech Laboratories) was used to insert the *dynein-2<sub>D1091-Q4307</sub>* gene into the pFastBac vector. The final construct used for crystallization, electron microscopy and microtubule gliding assays had an amino (N)-terminal GFP, followed by a glycine (G) serine (S) spacer and dynein-2<sub>D1091-Q4307</sub> (GFP-dynein-2<sub>D1091-Q4307</sub>). All mutants were prepared by standard cloning techniques using GFP-dynein-2<sub>D1091-Q4307</sub> in the pFastBac vector background as a template. ΔAAA2L H2 + PS-I had regions 2022–2030 and 2074–2085 replaced by GG, ΔAAA4L PS-I had region 2734–2774 replaced by GSGSG, ΔAAA3L H2-S3 had region 2339–2344 replaced by GG and R1413A + E2028A had R1413 and E2028 substituted by alanines. All constructs were sequence verified.

Protein expression in Sf9 cells. The modified pFastBac plasmids were transformed into a DH10 EMBacY *Escherichia coli* strain which carried a bacmid harbouring the baculovirus genome. Clones containing bacmids in which the pFastBac vector had been successfully integrated were selected by blue white screening. Recombinant bacmids were prepared according to standard procedures, transfected into 2 ml Sf9 cells (0.5 × 10<sup>6</sup> cells per millilitre) using FuGENE HD Transfection Reagent (Promega) and incubated at 27 °C for 72 h (P1 virus). Half a millilitre of P1 virus was subsequently used to infect 50 ml of Sf9 cells (2 × 10<sup>6</sup> cells per millilitre) followed by incubation at 27 °C and 127 r.p.m. for 72 h (P2 virus). Five millilitres of P2 virus were used to infect 500 ml of Sf9 cells (2 × 10<sup>6</sup> cells per millilitre) followed by the incubation procedure described before. Cells were harvested by centrifugation at 4 °C and 2,500g for 30 min. The pellet was washed in ice-cold PBS, snap-frozen in liquid nitrogen and stored at −80 °C.

Protein purification. Frozen pellets were resuspended in lysis buffer (30 mM HEPES pH 7.4, 50 mM KOAc, 2 mM MgOAc, 0.2 mM EGTA, 10% v/v glycerol, 300 mM KCl, 0.2 mM Mg.ATP, 1 mM DTT and 2 mM PMSF). Resuspended cells were lysed manually in a dounce homogenizer. Cell debris and insoluble proteins were removed by ultracentrifugation at 4 °C and 60,000g for 30 min. Dynein constructs were pulled out from the lysate using IgG sepharose beads (GE Healthcare, 5 ml of beads per litre of Sf9 culture). IgG sepharose beads were washed with 15 bead volumes of lysis and tobacco etch virus protease buffer (50 mM Tris HCl pH 8, 150 mM KOAc, 2 mM MgOAc, 1 mM EGTA, 10% v/v glycerol, 1 mM DTT and 0.2 mM Mg.ATP). Protein was released from the beads during overnight cleavage with tobacco etch virus protease. Size-exclusion chromatography was performed on a Superose 6 column (GE Healthcare) in size-exclusion chromatography buffer (20 mM Tris HCl pH 8.0, 100 mM KOAc, 2 mM MgOAc, 1 mM EGTA, 10% v/v glycerol, 1 mM DTT).

Protein crystallization. Peak fractions of GFP-dynein-2<sub>D1091-Q4307</sub> after size-exclusion chromatography were pooled and concentrated to 8 mg ml<sup>−1</sup>. To lock dynein in its pre-power-stroke state, Mg.ATP (Sigma Aldrich) and Na<sub>3</sub>VO<sub>4</sub> (New England Biolabs) were added to a final concentration of 3 mM each. Crystals were obtained by hanging-drop vapour diffusion at 19 °C, mixing equal volumes of protein with reservoir solution (4–6% PEG 6000 and 0.1 M Tris pH 8.0). Crystallization strictly depended on the presence of both Mg.ATP and Na<sub>3</sub>VO<sub>4</sub>. Crystals did not form under apo, Mg.ATP or Na<sub>3</sub>VO<sub>4</sub> conditions. The crystal quality was markedly improved by microseeding. Seeds were prepared by harvesting GFP-dynein-2<sub>D1091-Q4307</sub> crystals into 100 μl of reservoir solution followed by vortexing with a seed bead (Jena Bioscience) for 30 s. After diluting the seed stock 1:10,000, crystallization was performed by mixing equal volumes of protein and seeds in reservoir solution, followed by equilibration against reservoir as described above.

Data collection and structure determination. Owing to the very fragile nature of the crystals, cryoprotection was performed in the drop by adding 1 drop volume of reservoir solution supplemented with 60% PEG 400. After incubation for 10–30 s, crystals were harvested using MicroMeshes (MiteGen) and flash cooled in liquid nitrogen. Heavy atom derivatization was performed by adding solid Na<sub>3</sub>[PW<sub>12</sub>O<sub>40</sub>] × H<sub>2</sub>O (Tri-Sodium phosphotungstate, Jena Bioscience) directly to the drop. After incubation for 2 h, crystals were collected as described above. Diffraction data were collected at 100 K on beamline I02 at the Diamond Light Source. The data were integrated using MOSFLM<sup>29</sup> and scaled using AIMLESS<sup>30</sup>. In the case of the anisotropic Native-1 data set, a first round of integration and scaling was performed with a resolution limit of 3 Å. The data were then subsequently analysed with the University of California, Los Angeles Molecular Biology Diffraction Anisotropy Server<sup>31</sup> (http://services.mbi.ucla.edu/anisoscale/), which suggested resolution cut-offs of *a* = 4.2 Å, *b* = 4.4 Å and *c* = 3.4 Å. The second round of data integration and scaling was done with the resolution cut-offs mentioned above (Extended Data Table 1). Phasing was performed with AUTOSHARP<sup>32</sup> using the multiple isomorphous replacement with anomalous scattering (MIRAS) approach with a low-resolution Native-2 data set and the Na<sub>3</sub>[PW<sub>12</sub>O<sub>40</sub>] peak and inflection data sets as heavy atom derivatives (Extended Data Table 1). The final electron-density map after density modification was of sufficient quality to identify the location of all dynein-2 subdomains in the asymmetric unit. Homology models for the individual subdomains were obtained by combining PDB accession number 3VKG with the amino-acid sequence of human cytoplasmic dynein-2 isoform 1 using the PHYRE server<sup>33</sup>. The homology models were placed in the asymmetric unit followed by iterative rounds of refinement in REFMAC<sup>34</sup> against the Native-1 data set, employing the ‘Jelly-Body’ and ‘secondary structure restraints’ refinement options, and manual rebuilding in COOT<sup>35</sup>. Refining the model against the anisotropy-corrected data obtained from the University of California, Los Angeles Molecular Biology Diffraction Anisotropy Server<sup>31</sup> significantly improved the quality of the resulting electron-density maps. The final model was evaluated by calculating a simulated annealing composite omit map in CNS<sup>36</sup> and had 99.9% of the residues in the allowed regions of the Ramachandran plot. All figures were prepared using PYMOL (http://www.pymol.org), LIGPLOT<sup>37</sup> and Jalview<sup>38</sup>.

Vanadate-mediated ultraviolet-photo cleavage of dynein-2 crystals. Crystals obtained under the conditions described above were harvested and washed three times in 10 μl reservoir solution followed by exposure to ultraviolet light (254 nm) for 1 h. Crystals were subsequently dissolved in sample buffer, boiled for 10 min at 95 °C and analysed by SDS-polyacrylamide gel electrophoresis.

Nucleotide content analysis. Nucleotides were extracted from dynein-2 samples that had been purified as described above. Nucleotide extraction was performed essentially as described previously<sup>39</sup>. Briefly, concentrated protein was precipitated by adding HClO<sub>4</sub> to a final concentration of 0.5 M. The sample was vortexed and centrifuged for 10 min at 4 °C and 14,000g. The supernatant was mixed with 1 M K<sub>2</sub>HPO<sub>4</sub>, 3 M KOH and 100% acetic acid (final concentrations were 125 mM, 375 mM and 0.5 M respectively). Total nucleotide content and protein concentration before extraction were measured using a NanoDrop ND-1000 spectrophotometer.

Negative-stain electron microscopy. Dynein motor domain constructs were purified as described above and diluted into electron microscopy assay buffer (50 mM Tris HCl pH 8, 150 mM KOAc, 2 mM MgOAc, 1 mM EGTA and 0.1 mM DTT) to a final concentration of 30 nM. Negative-stain electron microscopy was performed either in the presence of 3 mM Mg.ADP (+ADP) or 3 mM Mg.ATP and 3 mM Na<sub>3</sub>VO<sub>4</sub> (+ADP.Vi) on plasma-cleaned carbon film on 400-square-mesh copper grids (Electron Microscopy Sciences). The samples were stained with 2% (w/v) uranyl acetate. Electron micrographs (Extended Data Fig. 8a) were recorded on a Gatan Orius SC200B CCD (charge-coupled device) fitted to a FEI Tecnai G2 Spirit transmission electron microscope operating at 120 kV. Data were collected at ~1 μm underfocus, with a pixel size of 3.29 Å and an estimated dose of 20 electrons per square ångström during 1 s exposures. Automated particle picking was done in EMAN2.10a<sup>40</sup> using the Swarm boxing tool. Subsequent particle analysis used RELION<sup>41</sup>. Autopicked particles were subjected to two-dimensional classification to identify incorrectly picked particles, which were manually checked and removed from the data set. The remaining particles were classified into ten classes, which was sufficient to represent all observed views. Each class was then subclassified into 50 subclasses. Noisy subclasses were discarded and those remaining contained sufficient signal to noise to identify the stalk and linker-GFP clearly (Extended Data Fig. 8b). The ImageJ azimuthal average plugin (http://rsb.info.nih.gov/ij/plugins/azimuthal-average.html) was used to integrate the intensity values surrounding the outside of the motor domain with 1 ° bin width. This generated a plot with two peaks corresponding to the intensity for the stalk and GFP. Fitting using the sum of two Gaussian functions (Igor Pro 6.3, http://www.wavemetrics.com/products/products.htm) was used to measure the angle between them. All experiments were done in triplicate. The angle distribution, using a 10 ° bin width, was visualized by either histogram or rose plot. The number of particles used were as follows: WT + ADP: 3151, 9914, 7534; WT + ADP.Vi: 8710, 5793, 5284; ΔAAA2L H2 + PS-I 8664, 3408, 9220; ΔAAA4L PS-I 19835, 10799, 9955; R1413A + E2028A: 7994, 2445, 11581; ΔAAA3L H2-S3: 6535, 12185, 6399.

Microtubule gliding assays. A microtubule gliding assay was adapted from previously published work<sup>42</sup>. Briefly, anti-GFP antibody (Roche) was non-specifically bound to the glass surface of a flow chamber. The free surface was blocked with assay buffer (30 mM HEPES pH 7.2, 2 mM MgOAc, 1 mM EGTA, 10% (v/v) glycerol, 1 mg ml<sup>−1</sup> casein and 20 μM paclitaxel). GFP-tagged dynein-2 was then applied and after 30 s incubation washed with assay buffer. Finally, motility buffer that contained microtubules, oxygen scavenging system and 1 mM ATP as an energy source was applied and gliding was observed by total internal reflection fluorescence microscopy. All data analysis used ImageJ<sup>43</sup>.

29. Battye, T. G., Kontogiannis, L., Johnson, O., Powell, H. R. & Leslie, A. G. iMOSFLM: a new graphical interface for diffraction-image processing with MOSFLM. *Acta Crystallogr. D* **67**, 271–281 (2011).

30. Evans, P. Scaling and assessment of data quality. *Acta Crystallogr. D* **62**, 72–82 (2006).

31. Strong, M. *et al.* Toward the structural genomics of complexes: crystal structure of a PE/PPE protein complex from *Mycobacterium tuberculosis*. *Proc. Natl Acad. Sci. USA* **103**, 8060–8065 (2006).

32. Vonrhein, C., Blanc, E., Roversi, P. & Bricogne, G. Automated structure solution with autoSHARP. *Methods Mol. Biol.* **364**, 215–230 (2007).

33. Kelley, L.A. & Sternberg, M.J. Protein structure prediction on the Web: a case study using the Phyre server. *Nature Protoc.* **4**, 363–371 (2009).

34. Murshudov, G.N. *et al.* REFMAC5 for the refinement of macromolecular crystal structures. *Acta Crystallogr. D* **67**, 355–367 (2011).

35. Emsley, P. & Cowtan, K. Coot: model-building tools for molecular graphics. *Acta Crystallogr. D* **60**, 2126–2132 (2004).

36. Brünger, A.T. *et al.* Crystallography & NMR system: a new software suite for macromolecular structure determination. *Acta Crystallogr. D* **54**, 905–921 (1998).

37. Wallace, A.C., Laskowski, R.A. & Thornton, J.M. LIGPLOT: a program to generate schematic diagrams of protein-ligand interactions. *Protein Eng.* **8**, 127–134 (1995).

38. Waterhouse, A.M., Procter, J.B., Martin, D.M., Clamp, M. & Barton, G.J. Jalview version 2—a multiple sequence alignment editor and analysis workbench. *Bioinformatics* **25**, 1189–1191 (2009).

39. Huecas, S. & Andreu, J.M. Energetics of the cooperative assembly of cell division protein FtsZ and the nucleotide hydrolysis switch. *J. Biol. Chem.* **278**, 46146–46154 (2003).

40. Tang, G. *et al.* EMAN2: an extensible image processing suite for electron microscopy. *J. Struct. Biol.* **157**, 38–46 (2007).

41. Scheres, S.H. RELION: implementation of a Bayesian approach to cryo-EM structure determination. *J. Struct. Biol.* **180**, 519–530 (2012).

42. Reck-Peterson, S.L. *et al.* Single-molecule analysis of dynein processivity and stepping behavior. *Cell* **126**, 335–348 (2006).

43. Schneider, C.A., Rasband, W.S. & Eliceiri, K.W. NIH Image to ImageJ: 25 years of image analysis. *Nature Methods* **9**, 671–675 (2012).

Extended Data Figure 1 | Examples of the electron density quality in dynein-2:ADP.Vi. $2F_{o} - F_{c}$ electron density in different parts of dynein-2:ADP.Vi. Amino-acid side-chains are clearly resolved in the linker (a),

AAA1 (b), AAA4 (c) and AAA6 (d). Only the main-chain could be traced in the stalk (e) and the buttress (f). The electron density in **a–d** was map-sharpened. The contour level is $1\sigma$, except for **e** which was contoured at $0.75\sigma$.

RESEARCH LETTER

a
RMSD, Å
Link1-2
1.8
AAA1L
3.3
AAA2L
1.5
AAA3L
1.8

RMSD, Å
Link3-4
2.7
AAA4L
2.1
AAA5L
1.5
AAA6L
1.5

b
RMSD, Å
AAA1S
4.2
AAA2S
2.2
AAA3S
2.1
AAA4S
2.9

RMSD, Å
AAA5S
2.1
AAA6S
1.6
C-term domain
4.2

Extended Data Figure 2 | Structural similarity between individual subdomains of dynein-1 and dynein-2. Alignment of individual subdomains from dynein-2:ADP.Vi and dynein-1:ADP (PDB accession number 3VKG).  
a, Alignment of AAA+ large (AAA1L–AAA6L) subdomains and the linker subdomains (Link1–2, Link3–4).  
b, Alignment of individual AAA+ small subdomains (AAA1S–AAA6S) and the C-terminal domain. Dynein-2 subdomains are coloured according to the scheme used in the main text, and shown in the inset cartoons. Dynein-1 subdomains are shown in grey.

Calculated root mean squared deviation (r.m.s.d.) values are shown above each alignment and demonstrate that the subdomains of dynein-2 are structurally highly similar to dynein-1. The AAA+ ring subdomains with the largest r.m.s.d. differences are AAA1L and AAA1S. These subdomains are the most strongly conserved part of the dynein structure and the differences are probably due to the ADP.Vi binding. The distortion of AAA1L, by its interaction with the AAA2L inserts, is described in the main text.

Extended Data Figure 3 | Closed interfaces between AAA+ domains of the AAA+ ring in dynein-2:ADP.Vi. a, Gaps in the AAA+ rings of different dynein motor domain crystal structures. In dynein-1:APO (PDB accession number 4AKI) and dynein-1:ADP (PDB accession number 3VKG) there are gaps between AAA1L/AAA2L and AAA5L/AAA6L or AAA4L/AAA5L. In dynein-2:ADP.Vi a smaller gap exists between AAA5L/AAA6L. Gaps are indicated by black arrows. b, Calculated buried surface areas indicate that the interfaces between AAA1/AAA2, AAA2/AAA3, AAA3/AAA4, AAA4/AAA5 and AAA6/AAA1 are tightly closed in dynein-2:ADP.Vi (buried surface areas 1,059–1,706 Å²). The AAA5/AAA6 interface is more open (buried surface area 837 Å²). Nucleotides are shown in stick representation. AAAL, AAA+ large subdomain; AAAS, AAA+ small subdomain.

RESEARCH LETTER

Extended Data Figure 4 | The four nucleotide binding sites of dynein-2:ADP.Vi. a–c, The AAA1 site contains electron density consistent with an Mg.ADP.Vi molecule. All catalytic amino-acid residues have the correct conformation to support catalysis. d, Photo cleavage¹¹ of washed dynein-2 crystals upon exposure to ultraviolet light (+UV) produces two bands of 300 and 90 kDa (arrowheads). This suggests crystals contain an ADP.Vi group in AAA1. e, f, The AAA2 site contains density consistent with a Mg.ATP molecule. g, h, The AAA3 and i, j, AAA4 sites contain electron density that is best modelled as ADP. In contrast to AAA1, AAA2–AAA4 have lost the catalytic residues necessary for ATP hydrolysis (the Walker B glutamate, the arginine finger, sensor-I and sensor-II motifs). The $F_{o} - F_{c}$ electron density (a, e, g, i) is contoured at 3σ. The $2F_{o} - F_{c}$ electron density (c) is contoured at 1σ. W-A, Walker A motif; W-B, Walker B motif; S-I, sensor-I; S-II, sensor-II; RF, arginine finger. Magnesium ions ($\mathrm{Mg}^{2+}$) are shown as green spheres. The vanadium ion of the vanadate molecule (Van) is shown as a pink sphere.

Extended Data Figure 5 | Changes in conformation within dynein AAA+ ring. a, Superimposition of the AAA1L domains of dynein-2:ADP.Vi (blue) and dynein-1:ADP (grey) shows that helices H2 and H3 of AAA1L are displaced when the H2-β hairpin insert of AAA2L (red) comes into contact with H2 of AAA1L. b, An alignment of the AAA1L domains of dynein-2:ADP.Vi (blue), dynein-1:APO (PDB accession number 4AKI) (pale yellow) and dynein-1:ADP (PDB accession number 3VKG) (grey) shows that the loop containing the sensor-I residue is highly variable between the structures. In the presence of ADP.Vi the loop makes contacts (purple spheres) with AAA2L. c, d, Superimposition of AAA2-AAA4 domains between dynein-2:ADP.Vi and dynein-1:ADP (c) or dynein-1:APO (d) shows that AAA2–AAA4 move as a rigid body.

RESEARCH LETTER

a dynein-2:ADP.Vi  
b dynein-1:ADP  
c dynein-1:APO  

H10  
AAA1L  
H2  
H11  
Link3-4  
H1  
H0  
S8  
Linker connector peptide  
Link3-4  
H10  
H11  
H2  
H0  
S8  
Linker connector peptide  
H10  
H2  
AAA1L  
H11  
H0  
Link3-4  
H1  
S7  
Linker connector peptide  

d  
e  
f  

H10  
Link1-2  
AAA2L  
H2 insert  
H3  
AAA3L  
H2-S3 insert  
S3  
Link1-2  
H10  
AAA2L  
H2 insert  
H7  
H5  
H4  
PS-I insert  
H2  
H1  
AAA5L  

Extended Data Figure 6 | Linker interaction with the AAA+ ring in dynein-2:ADP.Vi, dynein-1:ADP and dynein-1:APO. a–c, Link3–4 interacts with AAA1L in all structures similar. Mainly hydrophobic contacts exist between the linker H11 helix and the H2 helix as well as the S2 β-sheet of AAA1L. In addition the long peptide that connects the linker with AAA1 (yellow) mediates contacts between Link3–4 and AAA1L. d, Link1–2 is stabilized by contacts with AAA2 and AAA3 in dynein-2:ADP.Vi Link1–2, e by contacts with the AAA2 H2 insert in dynein-1:ADP and f by contacts with AAA5 in dynein-1:APO. Red spheres represent contacts.

©2014 Macmillan Publishers Limited. All rights reserved

Link1-2
H5
H7
H10
H11
H12

| Hs_Cyt-2 | 1308 | LLQSLKDS | 1315 | WVYLEPIFG | 1349 | RSLNEFLEEKRSAFPRFYFIGDDDLLEI | 1413 |
| Cr_Cyt-2 | 1371 | LVASLKQS | 1378 | WVYLEPIFG | 1412 | RALADFLEEKRSQFPRFYFLGDDDLLEI | 1476 |
| Tt_Cyt-2 | 1300 | LLASMKES | 1307 | WVYLEPIFG | 1341 | KALNDFLEEKRSEFPRFYFLGDDDLLEI | 1405 |
| Hs_Cyt-1 | 1503 | SVSAMKLS | 1510 | WVYLEGIFT | 1544 | KALGEYLERESSFPRFYFVGDEDLLEI | 1613 |
| Dd_Cyt-1 | 1571 | SISAMKMS | 1578 | WVYLEGIFS | 1620 | KALGEYLERSAFARFYFVGDEDLLEI | 1681 |
| Sc_Cyt-1 | 1411 | ELVSMKAS | 1418 | WLDLYGILG | 1460 | SSLSTFLERQRQFPRFYFLGNDDLLKI | 1521 |
| Dm_Cyt-1 | 1493 | SVAAMKLS | 1500 | WVYLEGIFS | 1534 | KALGEYLERERTSFPREFYVGDEDLLEI | 1603 |
| En_Cyt-1 | 1538 | SLQAMRHS | 1545 | WVYLEGVFT | 1579 | KALGEYLERRVSFPRFYFVGDEDLLEI | 1648 |
| Ca_Cyt-1 | 1433 | ALTSMKNS | 1440 | WLYLEGVFG | 1474 | KSLTDYLEKQRELFPRFYFIGNELLEI | 1544 |
| Hs_IDA4_1 | 1203 | MTQNMSFS | 1210 | WLYLEPIFS | 1244 | KGLSEYLETKRSAFPRFYFLSDDELLEI | 1312 |
| Hs_IDA3_3 | 1049 | KTQTMCGS | 1056 | WLYLEPIFS | 1090 | KGLNDYLEKKRLFPRFFLSNDELLEI | 1158 |
| Hs_ODAg_5 | 1591 | LLGSILLSN | 1598 | WIYLEAVFV | 1632 | KSLTGYLEKKRLCFPRFFFVSDPALLET | 1701 |
| Hs_IDA5_6 | 1067 | NVATLASS | 1074 | WLYLESIFN | 1108 | KCLEAYLESKRVIFPRFYFLSNDELLEI | 1176 |
| Hs_IDA3_7 | 946 | KTQTMRGS | 953 | WLYLEPIFS | 995 | KGLNEYLEKKRLFFPRFFLSNDELLEI | 1055 |
| Hs_ODAg_8 | 1457 | VLGSILLSN | 1464 | WVYLEAVFV | 1498 | KSLTGYLEKKRLLFPRFFVSDEPVILLEI | 1567 |
| Hs_ODAab_9 | 1478 | QLQNLVMS | 1485 | WTHLESIFT | 1527 | KALAELYLDTKRLAFPRFYFLSSSDLLLDI | 1588 |
| Hs_ODAab_17 | 1451 | QLQNLMMS | 1458 | WSHLESIFI | 1500 | KALAELYETKRLAFPRFYEVSSADLLDI | 1668 |

AAA2L H2 insert
AAA3L H2-S3 insert

| Hs_Cyt-2 | 2018 | LLGHIDMDTREWSDGVL | 2034 | CMV-ISTNTGRVYRPK-DCER | 2335 |
| Cr_Cyt-2 | 2073 | LLGSNMNDTREWSDGVL | 2089 | CGKPVTTTSGKALRPP-DNTR | 2378 |
| Tt_Cyt-2 | 1999 | LLGLMNNDTREFTEGVL | 2015 | CMK-GTFSQGRILKPK-DASR | 2317 |
| Hs_Cyt-1 | 2264 | LYGTLPNTRWETDGLF | 2280 | CEYRRTPNGV-VLAPVQLGKW | 2639 |
| Dd_Cyt-1 | 2316 | LFGSLDLTTRWETDGLF | 2332 | CEYKRTPSGTVLRPTQLGKW | 2718 |
| Sc_Cyt-1 | 2114 | LYGSMLKATLEWRDGLF | 2130 | TNYTTTSKGL-TLLPKSDIKN | 2462 |
| Dm_Cyt-1 | 2250 | LYGVLPNTRWETDGLF | 2266 | CEYRKTPNGV-VLSFVQIGKW | 2624 |
| En_Cyt-1 | 2269 | LYGSLDSTTRWETDGLF | 2285 | CEYKKTLSGV-VMSPNQIGRW | 2637 |
| Ca_Cyt-1 | 2152 | IYGKLDLVTRDWTDGLF | 2168 | CEYRTNRGI-QLAPRINGKW | 2511 |
| Hs_IDA4_1 | 1911 | LYGEFDLLTHEWTDGIF | 1927 | IDSKLDKRRKGVFGPP-LGRN | 2269 |
| Hs_IDA3_3 | 1756 | LYGCFDQVSHEWMDGVL | 1772 | IMSKLDRRRKGLFGPP-IGKK | 2116 |
| Hs_ODAg_5 | 2301 | MFGRLDVATNDWTDGIF | 2317 | IESYVDKRMGTTYGPP-AGKK | 2632 |
| Hs_IDA5_6 | 1801 | LYGEVNNLTLEWKDGLM | 1817 | IESKERKRKNILGAP-GNKR | 2139 |
| Hs_IDA3_7 | 1655 | LYGQFDSVSHEWSDGVL | 1671 | VMSKLDKRRKGVFGPP-LGKR | 2019 |
| Hs_ODAg_8 | 2167 | MFGRLDTATNDWTDGIF | 2183 | IESYVDKRIGSTYGPP-GGRK | 2498 |
| Hs_ODAab_9 | 2190 | LFGIINPATGEWKDGLF | 2206 | LEKPLEKKAGRNYGPP-GNKK | 2520 |
| Hs_ODAab_17 | 2153 | LFGIINPVTREWKDGLF | 2169 | LEKPLEKKSGRNYGPP-GTKK | 2483 |

Extended Data Figure 7 | Conservation of contact sites between linker and dynein ring. Multiple alignment of cytoplasmic dynein-1 (Cyt-1), dynein-2 (Cyt-2), axonemal inner arm dyneins (IDA) and outer arm γ (ODAg) and αβ (ODAab) dyneins. Dyneins are from human (Hs), Chlamydomonas reinhardtii (Cr), Tetrahymena thermophila (Tt), D. discoideum (Dd), S. cerevisiae (Sc), Drosophila melanogaster (Dm), Emericella nidulans (En) and Candida albicans (Ca). Residues are shaded by conservation, with dark blue being the most conserved. Red asterisks mark hydrophobic contacts that stabilize the bent linker conformation, black asterisks mark the contact site between AAA2L H2 insert (E2028) and the linker (R1413) and green asterisks mark poorly conserved contacts between the linker and the AAA3 H2-S3 insert.

RESEARCH LETTER

a

WT + ADP
![Image](image1.png)

WT + ADP.Vi
![Image](image2.png)

ΔAAA2L H2 + PS-I
![Image](image3.png)

ΔAAA4L PS-I
![Image](image4.png)

R1413A + E2028A
![Image](image5.png)

ΔAAA3L H2-S3
![Image](image6.png)

b

Percentage of particles
![Histogram](histogram1.png)
Angle (degrees)

WT + ADP
![Image](image7.png)

Percentage of particles
![Histogram](histogram2.png)
Angle (degrees)

WT + ADP.Vi
![Image](image8.png)

Percentage of particles
![Histogram](histogram3.png)
Angle (degrees)

ΔAAA2L H2 + PS-I
![Image](image9.png)

Percentage of particles
![Histogram](histogram4.png)
Angle (degrees)

ΔAAA4L PS-I
![Image](image10.png)

Percentage of particles
![Histogram](histogram5.png)
Angle (degrees)

R1413A + E2028A
![Image](image11.png)

Percentage of particles
![Histogram](histogram6.png)
Angle (degrees)

ΔAAA3L H2-S3
![Image](image12.png)

c

Mean velocity, nm/s
![Bar Chart](bar_chart.png)

- WT
- ΔAAA2L H2 + PS-I
- ΔAAA4L PS-I
- R1413A + E2028A
- ΔAAA3L H2-S3

Extended Data Figure 8 | Characterization of dynein-2 mutants by negative electron microscopy and microtubule gliding assays.
a, Representative micrographs showing the quality of the raw electron microscopy data. Scale bar, 20 nm.
b, Left, histograms showing distribution of angles between the linker and the stalk in three replicate negative-stain electron microscopy experiments (10° bin width); right, representative subclasses used for angle measurement.
c, Mean velocities of dynein-2 mutants in microtubule gliding assays.

GFP-dynein-2<sub>D1091-Q4307</sub> (wild type: WT) glides microtubules at 134 ± 8 nm s<sup>-1</sup> (N = 99). The microtubule gliding velocities for the other constructs are ΔAAA3L H2-S3, 59 ± 4 nm s<sup>-1</sup> (N = 79); K1413A + E2028A, 49 ± 2 nm s<sup>-1</sup> (N = 31); and ΔAAA4L PS-I, 14 ± 1 nm s<sup>-1</sup> (N = 121). Microtubule gliding was not observed in case of ΔAAA2L H2 + PS-I. Error bars, s.e.m.

©2014 Macmillan Publishers Limited. All rights reserved

a Low affinity
b Low affinity
c High affinity

CC1
H1

dynein-2:ADP.Vi
dynein-1 MTBD (PDB ID: 3ERR)

dynein-2:ADP.Vi
dynein-1 MTBD (PDB ID: 3WUQ)

dynein-2:ADP.Vi
dynein-1 MTBD (PDB ID: 3J1T)

Extended Data Figure 9 | The MTBD in dynein-2 ADP.Vi is in the low microtubule affinity conformation. a, b, Alignment of dynein-2 ADP.Vi MTBD (pale yellow) with dynein-1 MTBDs (grey) in the low microtubule affinity conformation (PDB accession numbers 3ERR and 3WUQ respectively), and c with a dynein-1 MTBD in the high microtubule affinity conformation (PDB accession number 3J1T). The stalk CC1 and the MTBD H1 undergo conformational changes depending on the microtubule affinity of the MTBD. In dynein-2:ADP.Vi the arrangement of these structural elements suggests the MTBD is in the low microtubule affinity conformation. Stalk CC1 and MTBD H1 are coloured blue in low-affinity structures and red in high-affinity structures.

©2014 Macmillan Publishers Limited. All rights reserved

Extended Data Table 1 | Data collection, phasing and refinement statistics

| Dataset          | Native-1       | Native-2       | $\mathrm{Na}_{3}\left[\mathrm{PW}_{12} \mathrm{O}_{40}\right]$ |
|-------------------|----------------|----------------|----------------------------------------------------------|
| Space group      | $C 222_{1}$    | $C 222_{1}$    | $C 222_{1}$                                             |
| Cell dimensions  |                |                |                                                        |
| $a, b, c (\AA)$  | 136.0, 487.2, 276.5 | 136.2, 487.7, 276.9 | 135.7, 481.9, 276.5                                  |
| Wavelength ($\AA$) | 0.97949        | 0.97949        | Peak: 1.21416<br>Infection: 1.21476                     |
| Resolution ($\AA$) | 56.5-3.40      | 56.1-6.0       | 65.9-6.0<br>69.5-6.0                                   |
| $R_{\text{sym}}$ or $R_{\text{merge}}$ | 10.1 (69.2)*   | 6.3 (17.7)     | 24.0 (111.6)<br>(170.6)                             |
| $I / \sigma I$   | 7.6 (1.1)      | 207.2 (28.4)   | 8.0 (2.4)<br>5.9 (1.7)                                 |
| Completeness (%) | 62.2 (1.9)$^{\S}$ | 94.9 (99.6)   | 99.8 (99.9)<br>(100.0)                                 |
| Redundancy       | 4.1            | 3.6            | 11.7<br>10.0                                           |

**Refinement**

| Resolution ($\AA$) | 56.6-3.41 |
|--------------------|------------|
| No. reflections    | 74060      |
| $R_{\text{work}} / R_{\text{free}}$ | 23.7/28.5 |
| No. atoms         | 22816      |
| Protein           | 22697      |
| Ligand/ion        | 119        |
| Water             | -          |

**B-factors**

| Protein | 122.0 |
|---------|-------|
| Ligand/ion | 69.8 |
| Water | -     |

**R.m.s deviations**

| Bond lengths ($\AA$) | 0.012 |
|----------------------|-------|
| Bond angles ($^{\circ}$) | 1.55 |

*Highest resolution shell is shown in parenthesis.

$\S$ The low completeness to 3.4 Å is due to the anisotropic diffraction limits ($a = 4.2 \AA$, $b = 4.4 \AA$, $c = 3.4 \AA$). When the data is scaled to a resolution limit of 4.2 Å the overall completeness is 97.9% and the completeness in the highest resolution shell is 78.4%.
