
ARTICLE IN PRESS

YMCNE-02945; No of Pages 9

**Molecular and Cellular Neuroscience xxx (2015) xxx–xxx**

---

Contents lists available at **ScienceDirect**

**Molecular and Cellular Neuroscience**

journal homepage: **www.elsevier.com/locate/ymcne**

---

# Prion degradation pathways: Potential for therapeutic intervention

Rob Goold, Chris McKinnon, Sarah J. Tabrizi \*

* ^a^ Department of Neurodegenerative Disease, UCL Institute of Neurology, University College London, United Kingdom

---

## ARTICLE INFO

Article history:

- Received 13 October 2014
- Accepted 16 December 2014
- Available online xxxx

Keywords:
- Prion disease
- PrPSc
- Autophagy
- Proteasome
- Lysosomal degradation
- Therapeutics

---

## ABSTRACT

Prion diseases are fatal neurodegenerative disorders. Pathology is closely linked to the misfolding of native cellular PrPC into the disease-associated form PrPSc that accumulates in the brain as disease progresses. Although treatments have yet to be developed, strategies aimed at stimulating the degradation of PrPSc have shown efficacy in experimental models of prion disease. Here, we describe the cellular pathways that mediate PrPSc degradation and review possible targets for therapeutic intervention. This article is part of a Special Issue entitled ‘Neuronal Protein’.

© 2015 The Authors. Published by Elsevier Inc. This is an open access article under the CC BY license (http://creativecommons.org/licenses/by/4.0/).

---

## Contents

1. Introduction . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
known to control protein synthesis (Clemens, 2004; Deng et al., 2002; Harding et al., 1999). Phosphorylation of eIF2α inhibits its activity and suppresses global protein synthesis (Walter and Ron, 2011). This pathway forms a key arm of the unfolded protein response (UPR), which is activated during conditions of cellular stress. The UPR has been shown to be particularly significant in prion pathology (Hetz and Mollereau, 2014; Moreno et al., 2012).

Once translated, proteins are scrutinised for correct folding by multiple quality control pathways. In the cytosol, the hsp70/hsp40 chaperone system (Kim et al., 2013) surveys proteins for exposed hydrophobic regions found in misfolded proteins. If attempts at refolding fail, misfolded proteins are targeted for degradation. For secretory or membrane proteins which are translocated directly into the endoplasmic reticulum (ER) during synthesis (cotranslational translocation), specialised quality control systems operate within the ER lumen (ERQC). Here, the situation is more complex than in the cytosol due to the additional need to monitor signal peptide removal, N-linked glycosylation, and disulphide bond formation (Braakman and Hebert, 2013). Since the ER lumen lacks degradation machinery, misfolded proteins must be retro-translocated to the cytosol for degradation as part of the ER-associated degradation (ERAD) pathway. Irreversibly aggregated ER proteins are subject to ERQC and targeted for lysosomal degradation via autophagic pathways (Araki and Nagata, 2011). In addition to ERAD and ERQC pathways, it is likely that protein quality control systems in other cellular compartments also contribute to the clearance of misfolded proteins. An important example is the Golgi quality control (Golgi QC) pathway which directs misfolded proteins from the Golgi directly to lysosomes for degradation (Anelli and Sitia, 2008; Arvan et al., 2002).

Misfolded, damaged or aggregated mature proteins are subject to similar quality control mechanisms as those synthesised de novo (Hipp et al., 2014). Protein aggregates accumulate in cells when levels of misfolded proteins overwhelm the quality control systems. This can arise in conditions of cell stress, mutant protein expression or prion infection. Different classes of protein inclusions have been described depending on their cellular location, stability and protein content. They are thought to play a protective role by sequestering potentially harmful misfolded proteins from the cellular milieu (Sontag et al., 2014). Various systems have evolved to deal with these deposits. Hsp70, Hsp40 and Hsp100 chaperones act in concert to solubilise aggregates, allowing refolding or degradation (Kim et al., 2013). Insoluble aggregates are directly targeted for degradation by binding to adaptor proteins, such as p62 and NBR1 (Bjorkoy et al., 2005; Kirkin et al., 2009; Pankiv et al., 2007). The eventual fate of terminally misfolded or aggregated proteins is degradation. There are two main degradation pathways: the ubiquitin-proteasome system (UPS) and lysosomal proteolysis (including autophagic pathways). These systems are particularly important in neurons whose complex architecture, long lifespan and inability to divide (and thereby dilute the load of damaged proteins), make them particularly vulnerable to proteotoxic stress.

## 2. Ubiquitin–proteasome system

As the principal route of protein degradation in mammalian cells, the UPS represents a major protection against misfolded proteins. Proteins are marked for proteasomal degradation by covalent conjugation of ubiquitin (Ub) in a sequential reaction involving three enzymes: ubiquitin activating enzymes (E1), ubiquitin conjugating enzymes (E2) and ubiquitin ligases (E3) that recognise and transfer ubiquitin to an internal lysine residue on substrate proteins. In humans, there are two E1 molecules, a greater diversity of E2s, and several hundred E3s (Lee et al., 2011). Thus, E3 ubiquitin ligases provide the mechanisms of substrate specificity in proteasomal degradation. Following initial substrate ubiquitination further Ub molecules are added sequentially to the first via one of seven internal lysine residues. In addition to canonical lysine 48 linkages, lysine 11 and 29 linkages have been shown to target proteins for proteasomal degradation, with a chain of four molecules considered the minimum efficient signal (degron) for recognition by the 26S proteasome (Dantuma and Bott, 2014; McKinnon and Tabrizi, 2014). This large (2.5 MDa) multi-subunit complex consists of a barrel-shaped 20S catalytic core responsible for proteolytic activity (Groll et al., 2000) and the 19S regulatory particle, which is important for the recognition, unfolding, and translocation of ubiquitinated substrates into the 20S core particle (Bedford et al., 2010). Mutations in different components of the UPS have been identified in clinical cases of HD, AD and PD (Kitada et al., 1998; van Leeuwen et al., 2006). Furthermore, experimental knockout of proteasome subunits in mice has been shown to result in progressive neurodegeneration, clearly demonstrating the importance of proteasome catalytic activity to neuronal proteostasis and survival (Bedford et al., 2008; Tashiro et al., 2012). Ageing has also been linked with a reduction in UPS activity, a factor that may contribute to the late onset of many neurodegenerative diseases (Gamerdinger et al., 2009; Tydlacka et al., 2008; Zhou et al., 2003).

Although implicated in the clearance of many disease-associated proteins (Bhat et al., 2014; Goold et al., 2013; Li et al., 2010), proteasomal degradation may be restricted to soluble misfolded proteins or smaller oligomeric forms that can be unfolded to allow entry into the 20S catalytic chamber. For larger, more insoluble aggregates, the catalytic chamber may remain inaccessible, preventing their effective degradation (Qin et al., 2003; Scotter et al., 2014). Indeed, many oligomeric and aggregated forms of disease-associated proteins have been shown to inhibit proteasome activity, both in reconstituted systems using purified components, as well as in cultured cells and in vivo models (Andre and Tabrizi, 2012; Deriziotis et al., 2011; Hong et al., 2014; Kristiansen et al., 2007). In the context of UPS impairment, an upregulation of autophagy has been described, which may facilitate the clearance of larger aggregates (Korolchuk et al., 2010). This is a good example of the cross-talk and close interplay thought to exist between the two degradatory systems (Hao et al., 2013; Nedelsky et al., 2008).

## 3. Lysosomal degradation/autophagy

Lysosomes represent the major catabolic compartment in eukaryotic cells. A wide range of enzymatic activities are confined within the lysosomal limiting membrane. These include many classes of proteolytic enzymes (Appelqvist et al., 2013). Several routes deliver cell constituents to lysosomes including endolysosomal pathways mediated by the ESCORT machinery, as well as ERQC and Golgi QC pathways and autophagic pathways (Saftig and Klumperman, 2009). These systems are interlinked and crosstalk between them ensures the efficient removal of obsolete cellular components (Nixon, 2013).

Autophagy is a highly conserved system for the degradation of cytosolic macromolecules and organelles. Several pathways have been described with the most important for neuronal proteostasis being macroautophagy (Jimenez-Sanchez et al., 2012; Nixon, 2013; Yao et al., 2013). This is a process whereby cytosolic contents are engulfed in a double membrane-bound structure, called an autophagosome, which later fuses with lysosomes to enable degradation to take place. The process begins with formation of a crescent shaped isolation membrane (phagophore). The isolation membrane then extends around a region of cytoplasm or selected substrate. Subsequent closure of the inner and outer bilayers of the isolation membrane forms the autophagosome, which later fuses with a lysosome to yield an autolysosome (Rubinsztein et al., 2012). The mammalian target for rapamycin complex (mTORC) is an important negative regulator of autophagy whose activity is influenced by multiple signalling pathways (Rubinsztein et al., 2012). However, mTORC-independent pathways have also been described that involve Beclin 1 and the PI3K vps34 (Sarkar et al., 2005; Williams et al., 2008).

The importance of autophagy to neuronal proteostasis was shown by a mouse conditional knockout of atg5, a key autophagy intermediate, in the CNS. On atg5 deletion, mice developed behavioural deficits and neurodegeneration (Hara et al., 2006). Interestingly, affected mice also accumulated abnormal ubiquitinated proteins which led to the formation of aggregates in neurons (Hara et al., 2006). Induction of autophagy has been shown to be beneficial in many models of neurodegenerative disease through the degradation of aggregation-prone mutant proteins including Huntingtin (Ravikumar et al., 2004), α-Synuclein (Webb et al., 2003), APP (Spilman et al., 2010), Tau (Ozcelik et al., 2013) and TDP-43 (Wang et al., 2012).

### 4. Prion disease and proteostasis

To date, many studies have identified evidence of proteostasis dysregulation in prion disease. Early reports demonstrated the presence of abnormal levels of ubiquitin and ubiquitinated proteins in diseased mouse brain tissue (Kenward et al., 1994; Lowe et al., 1992). More recent studies have confirmed the presence of ubiquitin-positive staining in the form of intracellular inclusions or prominent extracellular puncta in the brains of diseased animals (Kristiansen et al., 2007). The abnormal levels of ubiquitinated protein indicate a failure of protein degradation pathways. Accumulation of proteasomal substrates at later stages of disease correlate with a decrease in proteasome catalytic activity in brain extracts from diseased animals (Deriziotis et al., 2011; Kristiansen et al., 2007). Transgenic mice expressing the UPS reporter UbG76V-GFP showed strong reporter accumulation in the brain regions worst affected by prion disease, supporting a role for proteasomal impairment in disease pathogenesis (Kristiansen et al., 2007).

Abnormalities in the lysosomal system have also been observed in prion diseases. Increases in the number and size of autophagic vacuoles were reported in the brains of patients affected by prion disease, as well as in mouse models, suggesting that autophagy may be up-regulated in prion disease (Boellaard et al., 1991; Liberski et al., 2010; Sikorska et al., 2004). Consistent with this hypothesis, an increase in p62 expression in diseased brain was recently reported and may reflect attempts to increase the clearance of aggregated proteins by autophagy (Homma et al., 2014).

In addition to impairments in degradation systems, many studies have indicated that ER stress is a feature of prion disease in both human patients and animal models (Hetz et al., 2003), with many UPR markers upregulated relatively early in disease pathogenesis (Hetz and Soto, 2006; Moreno et al., 2012; Rane et al., 2008). Moreover, mechanistic studies have shown that prion infection induces a global down-regulation of protein translation through chronic eIF2α phosphorylation (Moreno et al., 2012) and ER protein translocation impairment (Rane et al., 2008). Thus, signs of ER stress appear pre-symptomatically and have been suggested as important mediators of prion toxicity (Hetz and Soto, 2006). However, the causal relationship between these observations and disease pathogenesis is currently unknown. Misfolded PrP in the ER could induce ER stress (Hetz and Mollereau, 2014). Alternatively, the accumulation of PrP in aggresomes may sequester cytosolic components leading to proteostatic impairment (Chakrabarti and Hegde, 2009; Kristiansen et al., 2007). Complicating the picture further is the close relationship that exists between the UPS, autophagy and ER function (Dantuma and Bott, 2014; Hetz and Mollereau, 2014). For example, there is a reciprocal relationship between ER stress and proteasome activity, such that proteasome inhibition has been shown to induce ER stress and vice versa (Lee et al., 2003; Menendez-Benito et al., 2005). Hence, deciphering which, if any of these factors, is causal to disease pathogenesis presents a significant challenge. Despite this, it is clear that disease pathogenesis is intimately linked to ongoing PrPSc propagation (Aguzzì and Falsig, 2012) and that lowering PrPSc load increases the lifespan of prion-infected mice (Mallucci et al., 2003; Mallucci et al., 2007). As a result, prion degradation pathways may represent a viable therapeutic target for the treatment of prion diseases.

### 5. Prion degradation pathways

In vivo observations support a role for both the lysosomal system and the UPS in prion degradation. Several studies have reported that the majority of intracellular PrPSc is found in the endolysosomal system (Jeffrey et al., 2010 and therein). In prion-infected brain tissue, increased numbers and sizes of late endosomes, lysosomes and autophagic vesicles have been described (Boellaard et al., 1991; Liberski et al., 2010; Sikorska et al., 2004). Dual-labelling experiments also confirmed the colocalisation of PrPSc with lysosomal markers (DeArmond and Bajsarowicz, 2010). Interestingly, reports of N-terminal truncation of PrP suggest that lysosomes play an active role in PrPSc degradation (Jeffrey et al., 2003). In addition to the lysosomal system, we have previously reported a biochemical association between PrPSc, 20S proteasome subunits and other cytosolic aggresome markers (Hsp70 and vimentin) in prion-infected mouse brain (Kristiansen et al., 2005). This is of particular interest since aggresomes are thought to sequester misfolded proteins and target them for degradation by both the UPS and autophagy (Dantuma and Bott, 2014; Sontag et al., 2014). Thus, the two major protein clearance pathways appear to be involved in prion degradation (Fig. 1).

These in vivo findings were largely confirmed by in vitro experiments in various neuroblastoma and other cultured cell lines which stably propagate prions. The potential for genetic and pharmacological manipulation of cultured cells has facilitated a more detailed analysis of PrPSc intracellular trafficking and degradation pathways. PrPSc is found on the plasma membrane, in the endolysosomal system, the endosomal recycling compartment, the trans Golgi network and Golgi (via retromer mediated retrograde transport), in the autophagic pathway and in the cytosol (Beranger et al., 2002; Borchelt et al., 1992; Magalhaes et al., 2005; Marianovic et al., 2009; Rouvinski et al., 2014; Veith et al., 2009; Yamasaki et al., 2014). Much of the work on PrPSc intracellular distribution was directed at finding the site of prion conversion (i.e., the templated misfolding of native PrPC into PrPSc). Despite useful information provided by these studies, they rarely examined prion degradation directly. This is important because the PrPSc content of a cell at any instant reflects the fluctuating balance between synthesis (i.e., new prion conversion) and degradation. The wide variety of compounds known to down-regulate PrPSc levels in cultured cells with no apparent commonality in their mode of action gives an indication of the complexity of prion metabolism (Trevitt and Collinge, 2006). Hence, the overall PrPSc content of a cell is not solely a reflection of its degradation rate and should not be interpreted as such. The situation is further complicated by the observation that treatments which block PrPSc degradation often lead to an increase in PrPC levels (Nunziante et al., 2011). Higher cellular levels of PrPC are likely to promote prion conversion and increase PrPSc levels independent of any block in its degradation (Nishida et al., 2000). The converse is also likely to be true, whereby agents which reduce levels of PrPSc also deplete PrPC (Goold et al., 2013; Heiseke et al., 2009). It is therefore necessary to interpret data regarding total PrPSc levels with caution when considering possible mechanisms of degradation.

Recent work looking directly at the degradation of surface-labelled PrPSc has demonstrated an important role of the lysosome in prion degradation (Goold et al., 2013). Autophagy appears to be the major route of PrPSc delivery to lysosomes, at least in chronically-infected cells (Heiseke et al., 2010; Yao et al., 2013). Genetic ablation of key autophagic components and pharmacological blockade both increase PrPSc levels (Goold et al., 2013; Heiseke et al., 2009; Heiseke et al., 2010). Conversely, stimulating autophagy has been shown to decrease PrPSc load (Aguib et al., 2009; Heiseke et al., 2010; Homma et al., 2014). Other non-autophagy dependent routes of lysosomal delivery have also been proposed. Yamasaki and colleagues suggested that upon prion exposure, N2a cells channel a significant proportion of newly endocytosed PrPSc through the endolysosomal pathway for rapid degradation (Yamasaki et al., 2014). Similar findings have previously been reported

ARTICLE IN PRESS

R. Goold et al. / Molecular and Cellular Neuroscience xxx (2015) xxx–xxx

![Diagram](attachment:diagram.png)

Fig. 1. PrP<sup>Sc</sup> formation, trafficking and degradation. Schematic illustrating PrP<sup>Sc</sup> metabolism. PrP<sup>Sc</sup> forms at the plasma membrane or shortly after endocytosis in endosomes, the ERC or lysosomes. Recycling of PrP<sup>Sc</sup> to the plasma membrane allows prion propagation. Newly formed PrP<sup>Sc</sup> undergoes retrograde transport to the trans Golgi network (TGN) and Golgi where it is subject to Golgi quality control and trafficked to lysosomes for degradation. More mature forms of PrP<sup>Sc</sup> are trafficked to lysosomes via the endolysosomal and autophagic pathways. PrP<sup>Sc</sup> may reach the cytosol through lysosomal rupture or ERAD, and accumulates in aggresomes under conditions of proteasome impairment. Unfolding and ubiquitination precede proteasomal degradation (UPS pathways shown in red). Aggresomal PrP<sup>Sc</sup> and smaller insoluble forms are engulfed by phagophores and degraded by autophagic pathways (shown in orange).

in primary dorsal root ganglion neurons ([Jen et al., 2010](https://doi.org/10.1016/j.neurobiolaging.2009.08.004)). The Golgi QC pathway has been shown to be important for the clearance of some PrP genetic mutants and newly synthesised PrP<sup>Sc</sup> ([Ashok and Hegde, 2009](https://doi.org/10.1073/pnas.0902850106); [Goold et al., 2013](https://doi.org/10.1016/j.jmb.2013.03.010)). Taken together, the complexity of prion degradation likely reflects differences in the cell types used and forms of misfolded PrP being studied (e.g., mutant PrP isoforms, newly-formed PrP<sup>Sc</sup> and mature PrP<sup>Sc</sup>).

In addition to lysosomal and autophagic degradation pathways, our recent work suggests that the UPS also plays an important role in PrP<sup>Sc</sup> degradation. In chronically-infected cultured cells, we found that application of proteasome inhibitors precipitated a rapid rise in PrP<sup>Sc</sup> levels, with detectable increases as early as three hours post-application ([Goold et al., 2013](https://doi.org/10.1016/j.jmb.2013.03.010)). Importantly, elevated PrP<sup>Sc</sup> levels were not accompanied by increased PrP<sup>C</sup> expression, suggesting that PrP<sup>Sc</sup> degradation itself was the treatment target. Interestingly, proteasomal inhibition has been shown to lead to aggresome formation in many cell types ([Kawaguchi et al., 2003](https://doi.org/10.1038/nm893)). In prion-infected cells these perinuclear inclusions contain PrP<sup>Sc</sup> and other typical aggresome markers including Hsp70, proteasome subunits and vimentin, ([Kristiansen et al., 2005](https://doi.org/10.1016/j.jmb.2005.03.025)). These pharmacologically-induced aggresomes suggest the presence of cytosolic PrP<sup>Sc</sup> in cultured cells ([Ben Gedalya et al., 2011](https://doi.org/10.1016/j.jmb.2011.04.027); [Dron et al., 2009](https://doi.org/10.1016/j.jmb.2009.03.010); [Kristiansen et al., 2005](https://doi.org/10.1016/j.jmb.2005.03.025)). This is an important observation since proteasomal activity is restricted to the cytosol and nucleus ([McKinnon and Tabrizi, 2014](https://doi.org/10.1016/j.jmb.2014.02.010)) and PrP<sup>Sc</sup> must therefore access one of these compartments to be considered a direct proteasomal substrate.

As an outer leaflet membrane protein, mature PrP would not normally gain access the cytosol. Since prion conversion occurs after PrP maturation ([Borchelt et al., 1990](https://doi.org/10.1016/0092-8674(90)90641-X); [Caughey and Raymond, 1991](https://doi.org/10.1016/0092-8674(91)90641-X)) at the plasma membrane ([Goold et al., 2011](https://doi.org/10.1016/j.jmb.2011.04.027)), and/or following endocytosis ([Beranger et al., 2002](https://doi.org/10.1016/S0092-8674(02)00672-9); [Borchelt et al., 1992](https://doi.org/10.1016/0092-8674(92)90641-X); [Caughey et al., 1991](https://doi.org/10.1016/0092-8674(91)90641-X); [Marijanovic et al., 2009](https://doi.org/10.1016/j.jmb.2009.03.010); [Yamasaki et al., 2014](https://doi.org/10.1016/j.jmb.2014.02.010)), PrP<sup>Sc</sup> must traverse the plasma membrane or an intracellular membrane to gain access to the cytosol. How and where this process takes place remains unclear, yet various mechanisms can be envisaged. The build-up of aggregated PrP<sup>Sc</sup> in lysosomes may de-stabilise the membrane, causing membrane leakage of the lumen contents into the cytosol, an event which has previously been described for other disease related proteins ([Micsenyi et al., 2013](https://doi.org/10.1016/j.jmb.2013.03.010)). Alternatively, PrP<sup>Sc</sup> may act as an ERAD substrate, as has been described for certain PrP mutant forms ([Jin et al., 2000](https://doi.org/10.1016/S0092-8674(00)80004-0); [Zanusso et al., 1999](https://doi.org/10.1016/S0092-8674(99)80004-0)).

Once in the cytosol, PrP<sup>Sc</sup> ubiquitination and unfolding are likely prerequisites for proteasomal degradation. Although ubiquitin-independent pathways to proteasomal degradation have been described ([Finley, 2009](https://doi.org/10.1016/j.jmb.2009.03.010)), most substrates require ubiquitination for efficient recognition ([Bhattacharyya et al., 2014](https://doi.org/10.1016/j.jmb.2014.02.010)). Evidence that PrP can be ubiquitinated has been hard to come by. In vivo, highly sensitive methods were required to detect ubiquitinated PrP, which was restricted to larger PrP<sup>Sc</sup> aggregates present at late stages of disease ([Kang et al., 2004](https://doi.org/10.1016/j.jmb.2004.03.010); [Kovacs et al., 2005](https://doi.org/10.1016/j.jmb.2005.03.025)). Although ubiquitin antibodies stain PrP-enriched aggresomes that form following proteasome inhibition in prion-infected

Please cite this article as: Goold, R., et al., Prion degradation pathways: Potential for therapeutic intervention, Mol. Cell. Neurosci. (2015), http://dx.doi.org/10.1016/j.mcn.2014.12.009

cells (Kristiansen et al., 2005), only a low level of colocalisation between PrP<sup>Sc</sup> and ubiquitin immunostaining in vivo have been reported (Cammarata and Tabaton, 1992). Hence, PrP<sup>Sc</sup> does not seem to be ubiquitinated to a significant degree and its status as a genuine proteasome substrate remains open to debate. It is possible that most PrP<sup>Sc</sup> remains non-ubiquitinated and becomes sequestered in Q-bodies — small, dynamic protein quality control compartments shown to form under basal conditions in cultured cells (Escusa-Toret et al., 2013; Sontag et al., 2014). Q-bodies may coalesce to form aggresomes under conditions of greater cell stress such as those prevalent during pharmacological proteasome inhibition or in the later stages of prion disease (Grenier et al., 2006; Kristiansen et al., 2005). Unfolding and ubiquitination of the Q-body PrP<sup>Sc</sup> population could instigate rapid degradation. Ubiquitinated PrP<sup>Sc</sup> would thus represent only a small proportion of total PrP<sup>Sc</sup> at steady state and could remain below detection thresholds. In addition to direct degradation, the proteasome may also regulate PrP<sup>Sc</sup> levels indirectly through clearance of PrP<sup>C</sup> thereby reducing the substrate levels for prion synthesis (Yedidia et al., 2001). Decreasing the rate of PrP<sup>Sc</sup> synthesis may allow alternative degradative systems to reduce the levels of pre-existing PrP<sup>Sc</sup> independent of, or in conjunction with, direct UPS activity.

Recent studies have highlighted the dynamic nature of PrP<sup>Sc</sup> metabolism. Significant increases in PrP<sup>Sc</sup> levels can be induced by a blockade of degradative activity which lasts only a few hours (Goold et al., 2013). Similarly, stimulation of these pathways clears prions from cells rapidly (Ertmer et al., 2004). Early metabolic labelling experiments suggested that much of the total cellular PrP<sup>Sc</sup> content is relatively stable (Boellaard et al., 1991; Caughey and Raymond, 1991). However, surface-labelling experiments revealed that PrP<sup>Sc</sup> on the plasma membrane is highly labile (Caughey and Raymond, 1991; Goold et al., 2013). This suggests that there are two populations of PrP<sup>Sc</sup> within the infected cell: a plasma membrane population (including newly formed PrP<sup>Sc</sup>) which is metabolised rapidly, and a more stable, and possibly more aggregated, internalised population which comprises the majority of total cellular PrP<sup>Sc</sup>. We found that newly formed PrP<sup>Sc</sup> is a substrate for non-autophagy dependent lysosomal degradation (i.e., the Golgi QC pathway) (Goold et al., 2013). In contrast, PrP<sup>Sc</sup> from chronically-infected cells is also subject to UPS and autophagy-dependent lysosomal degradation (Goold et al., 2013; Heiseke et al., 2009; Heiseke et al., 2010; Yao et al., 2013). This difference in metabolic fates may be due to differential trafficking of PrP<sup>Sc</sup> in cells with established prion propagation (Yamasaki et al., 2014). Alternatively, it could be explained by maturation of PrP<sup>Sc</sup> into a more aggregated state or its de novo appearance in the cytosolic compartment which, as previously discussed, is a necessary prerequisite for UPS-mediated degradation.

Significantly, induction of autophagy has been shown to reduce total cellular PrP<sup>Sc</sup> levels rapidly (Ertmer et al., 2004). This indicates that stimulation of cellular degradation systems is sufficient to overcome the apparent stability of PrP<sup>Sc</sup> levels under steady state conditions (Ertmer et al., 2004; Goold et al., 2013). It is also interesting to note that some treatments which have been shown to reduce PrP<sup>Sc</sup> load in cultured cells were also shown to be effective in vivo, both in terms of a reduction in PrP<sup>Sc</sup> load and clinical outcome (Yao et al., 2013).

### 6. Therapeutics

Prion diseases are fatal neurodegenerative disorders that include Creutzfeldt–Jakob disease (CJD), Gerstmann–Straussler–Scheinker syndrome, kuru and fatal familial insomnia. To date, no therapeutic or prophylactic regimens exist for these disorders. A variety of therapeutic strategies have been proposed, with most directed at preventing prion conversion. One approach is to reduce PrP<sup>C</sup> expression or trafficking to the plasma membrane, reducing its availability for prion conversion (Gilch et al., 2001; Tilly et al., 2003). Alternatively, chemical chaperones which stabilise PrP<sup>C</sup> structure (Cortez and Sim, 2013) or compounds which prevent interaction of PrP<sup>C</sup> with PrP<sup>Sc</sup> could be used to prevent

further protein misfolding (Caughey and Race, 1992; Caughey and Raymond, 1993; Priola et al., 2000). A novel approach targeting the UPR has reported clinical improvements in prion-infected mice (Moreno et al., 2013). This study used GSK2606414, a potent PERK inhibitor, to reduce the chronic phosphorylation of eIF2α and reverse the depression of protein translation that contributes to prion toxicity (Moreno et al., 2012). Interestingly, clinical improvements were evident despite little effect on the level of PrP<sup>Sc</sup>. Despite these encouraging findings, prion pathogenesis is likely to be multi-factorial, with many elements contributing to toxicity (Aguuzzi and Falsig, 2012). Hence, treatments aimed at the primary toxic insult (i.e., prion conversion and PrP<sup>Sc</sup> accumulation) should be effective in treating all aspects of toxicity. Reducing PrP<sup>Sc</sup> load by stimulating cellular degradation pathways (Fig. 2) could therefore, represent an effective therapeutic strategy.

Consistent with this hypothesis, several studies have reported that upregulating PrP<sup>Sc</sup> degradation can lead to significant clinical benefit. A series of reports have shown that autophagy induction leads to both PrP<sup>Sc</sup> clearance in cell models and more importantly, increased lifespan in prion-infected mice (reviewed in Yao et al., 2013). Treatment with rapamycin was shown to activate autophagy in vitro and delay disease onset in mice with prion disease (Cortes et al., 2012; Heiseke et al., 2009). Similar effects were reported using compounds which activate autophagy through mTORC-independent pathways. In prion-infected mice, trehalose was shown to delay the appearance of PrP<sup>Sc</sup> in the spleen (Aguib et al., 2009) and lithium was found to increase lifespan (Heiseke et al., 2009). The relatively modest improvements reported may reflect the difficulty in achieving the necessary drug concentrations in vivo due to poor blood brain barrier penetration, or simply because the effective concentrations of these drugs are particularly high. It should also be noted that the correlation between PrP<sup>Sc</sup> clearance and the stimulation of autophagy was based primarily on preliminary in vitro experiments. It is therefore possible, that the above compounds achieved beneficial effects through modulation of non-autophagic pathways (Aghdam and Barger, 2007; Maiese et al., 2013).

Several drugs originally used to target unrelated pathways have also been found to stimulate autophagy and reduce prion disease severity in mice. Treatment of prion-infected mice at 20 days post-inoculation with FK506, a well-known immunosuppressant drug, resulted in an upregulation of autophagic markers, a reduction in PrP<sup>Sc</sup> levels and an extension in lifespan (Nakagaki et al., 2013). Resveratrol, a phytoalexin enriched in grapes was shown to activate Sirt1, induce autophagy and protect against prion-mediated toxicity, both in cell culture (Jeong et al., 2012; Seo et al., 2012) and in an in vivo C. elegans model (Bizat et al., 2010). The plant extract sulforaphane was originally found to act through the Nrf2 pathway to protect against oxidative stress (Chapple et al., 2012). Recent reports have demonstrated that sulforaphane treatment prevents against prion neurotoxicity in cell culture models (Lee et al., 2014) and induces autophagy in vivo (Liu et al., 2014). Interestingly, this drug was also shown to activate the UPS (Gan et al., 2010; Kwak et al., 2007; Liu et al., 2014), making it an attractive anti-prion agent.

Increased lysosomal breakdown of PrP<sup>Sc</sup> through autophagy-independent pathways could represent an alternative therapeutic avenue. Branched polyamines are a class of compounds with well-established anti-prion activity in cell culture models (Supattapone et al., 1999; Supattapone et al., 2001). On administration to prion-infected mice, they were shown to slow the accumulation of splenic PrP<sup>Sc</sup> following intraperitoneal prion inoculation (Solassol et al., 2004). These compounds bind PrP directly and are thought to facilitate lysosomal degradation of PrP<sup>Sc</sup>, possibly by breaking up aggregates in the acidic lysosomal environment (Supattapone et al., 1999). The tyrosine kinase inhibitor STI571, originally developed to treat chronic myeloid leukaemia (Capdeville et al., 2002) has also been shown to have anti-prion activity. This is likely to be through the inhibition of c-Abl which in turn induces lysosomal degradation of PrP<sup>Sc</sup> through an as yet poorly

Please cite this article as: Goold, R., et al., Prion degradation pathways: Potential for therapeutic intervention, Mol. Cell. Neurosci. (2015), http://dx.doi.org/10.1016/j.mcn.2014.12.009

ARTICLE IN PRESS

R. Goold et al. / Molecular and Cellular Neuroscience xxx (2015) xxx–xxx

Lysosomal degradation | UPS degradation
--- | ---
ATG dependent | ATG independent

| Lysosomal proteases |
| --- |
| PrP<sup>Sc</sup> |
| Ubiquitin |
| USP14 |

![Diagram](attachment:diagram.png)

Fig. 2. Therapeutic targets in PrP<sup>Sc</sup> degradation pathways. Membrane associated PrP<sup>Sc</sup> is trafficked to lysosomes for degradation through endolysosomal, Golgi quality control or autophagic pathways. Cytosolic PrP<sup>Sc</sup> degradation is mediated by autophagy (orange arrows) and the UPS (red arrows). Reagents known to enhance the activity of these pathways are shown in blue highlights. Identified target proteins are indicated (details in the text).

characterised pathway (Ertmer et al., 2004). Importantly, STI571 treatment at an early phase of peripheral scrapie infection delayed the appearance of PrP<sup>Sc</sup> in the brain stem and spinal cord and slowed the onset of clinical disease in mice (Yun et al., 2007). Although untested in vivo, tamoxifen is another widely available pharmaceutical that may have therapeutic applications in prion disease. Tamoxifen and its metabolite 4-hydroxytamoxifen were shown to induce the lysosomal degradation of PrP<sup>Sc</sup> in prion-infected cells, possibly by diverting the trafficking of both PrP and cholesterol to lysosomes (Marzo et al., 2013). A novel approach to upregulate protein clearance is the use of lysosomal modulators (Bahr et al., 2012). Whilst untested in prion disease, these have been shown to increase lysosomal protease expression and activity, and were found to have protective effects in mouse models of AD (Butler et al., 2011; Viswanathan et al., 2012). Their development has come from the surprising observation that mild lysosomal protease inhibition induces the expression of not only the specific enzyme target, but also other unrelated proteases (Bahr et al., 2012). This leads to a global increase in lysosomal enzyme activity and alleviates protein accumulation and toxicity in disease models (Viswanathan et al., 2012).

Although potentially an attractive target for anti-prion therapies, the UPS has so far proved intractable as a drug target. To date, only one bone fide activator has become available. This drug, IU1, is a specific inhibitor of the 19S proteasome-associated ubiquitin chain trimming enzyme, Usp14. Inhibition of this enzyme blocks substrate deubiquitination

and enhances its degradation. Increased degradation of disease associated forms of tau, TDP-43 and ataxin-3 in cell culture models have been reported (Lee et al., 2010). Although untested in vivo, IU1 highlights the potential for therapies targeting UPS activity. Manipulations aimed at increasing the catalytic activity of the 20S proteasome through genetic upregulation of various subunits or small molecule enhancers have been reported but their significance in vivo may be limited (Dantuma and Bott, 2014; McKinnon and Tabrizi, 2014). One exception is sulforaphane, which has been shown to stimulate all three proteasome peptidase activities in brain extracts from drug treated mice (Liu et al., 2014). In addition, the levels of ubiquitinated proteins and a UPS reporter construct were reduced in the brains of these animals. In vitro, sulforaphane increased mtHtt degradation and protected cells against mtHtt toxicity; an effect which was abrogated by proteasome inhibition (Liu et al., 2014). To date, the efficacy of sulforaphane against prion disease remains untested. Its ability to stimulate both the UPS and autophagy (Liu et al., 2014) make it an attractive anti-prion agent.

An alternative approach is to augment UPS activity by stimulating the action or expression of chaperone proteins with small molecule compounds (Dantuma and Bott, 2014). Chaperones counteract aggregation, unfold potential UPS substrates and present them in a form readily degraded by the proteasome. Protective effects of such molecules have been reported in animal models of spinal-bulbar muscular atrophy (SBMA) and amyotrophic lateral sclerosis (ALS) (Kalmar et al., 2012;

Please cite this article as: Goold, R., et al., Prion degradation pathways: Potential for therapeutic intervention, Mol. Cell. Neurosci. (2015), http://dx.doi.org/10.1016/j.mcn.2014.12.009

ARTICLE IN PRESS

R. Goold et al. / Molecular and Cellular Neuroscience xxx (2015) xxx–xxx 7

Malik et al., 2013). Once again, these compounds are yet to be tested in prion disease models.

# 7. Perspectives

Although no effective treatment exists for prion diseases, many pathways have been identified that could be targeted for therapeutic intervention. Prion degradation pathways can be included in this group. There is good experimental evidence from in vivo and in vitro studies that pharmacological induction of lysosomal activity clears PrP<sup>Sc</sup> from neuronal cells and has a protective effect against prion disease pathogenesis. In particular, the benefits of compounds that induce autophagy are well documented. It seems likely that reagents stimulating the UPS could play a similar role. However, small molecules capable of doing this in vivo have yet to be fully characterised and their efficacy in prion disease models remains largely untested. Although still at the experimental level, approaches targeting PrP<sup>Sc</sup> degradation, in combination with other promising methods, may provide effective therapeutic and/or prophylactic treatments against prion diseases.

## Acknowledgements

This work was funded by the Medical Research Council [grant number G0700877 to S.J.T.]. We thank Ray Young for help with graphics.

## References

Aghdam, S.Y., Barger, S.W., 2007. Glycogen synthase kinase-3 in neurodegeneration and neuroprotection: lessons from lithium. Curr. Alzheimer Res. 4, 21–31.

Aguib, Y., Heiseke, A., Gilch, S., Riemer, C., Baier, M., Schatzl, H.M., Ertmer, A., 2009. Autophagy induction by trehalose counteracts cellular prion infection. Autophagy 5, 361–369.

Aguzzì, A., Falsig, J., 2012. PERSPECTIVE prion propagation, toxicity and degradation. Nat. Neurosci. 15, 936–939.

Andre, R., Tabrizi, S.J., 2012. Misfolded PrP and a novel mechanism of proteasome inhibition. Prion 6, 32–36.

Anelli, T., Sitia, R., 2008. Protein quality control in the early secretory pathway. EMBO J. 27, 315–327.

Appelqvist, H., Waster, P., Kagedal, K., Ollinger, K., 2013. The lysosome: from waste bag to potential therapeutic target. J. Mol. Cell Biol. 5, 214–226.

Araki, K., Nagata, K., 2011. Protein folding and quality control in the ER. Cold Spring Harb. Perspect. Biol. 3.

Arvan, P., Zhao, X., Ramos-Castaneda, J., Chang, A., 2002. Secretory pathway quality control operating in Golgi, plasmalemmal, and endosomal systems. Traffic 3, 771–780.

Ashok, A., Hegde, R.S., 2009. Selective processing and metabolism of disease-causing mutant prion proteins. PLoS Pathog. 5.

Bahr, B.A., Wisniewski, M.L., Butler, D., 2012. Positive lysosomal modulation as a unique strategy to treat age-related protein accumulation diseases. Rejuvenation Res. 15, 189–197.

Bedford, L., Hay, D., Devoy, A., Paine, S., Powe, D.G., Seth, R., Gray, T., Topham, I., Fone, K., Rezvani, N., et al., 2008. Depletion of 26S proteasomes in mouse brain neurons causes neurodegeneration and Lewy-like inclusions resembling human pale bodies. J. Neurosci. 28, 8189–8198.

Bedford, L., Paine, S., Sheppard, P.W., Mayer, R.J., Roelofs, J., 2010. Assembly, structure, and function of the 26S proteasome. Trends Cell Biol. 20, 391–401.

Ben Gedalya, T., Lyakhovetsky, R., Yedidia, Y., Bejerano-Sagie, M., Kogan, N.M., Karpuj, M.V., Kaganovich, D., Cohen, E., 2011. Cyclosporin-A-induced prion protein aggresomes are dynamic quality-control cellular compartments. J. Cell Sci. 124, 1891–1902.

Beranger, F., Mange, A., Goud, B., Lehmann, S., 2002. Stimulation of PrPC retrograde transport toward the endoplasmic reticulum increases accumulation of PrPSc in prion-infected cells. J. Biol. Chem. 277, 38972–38977.

Bhat, K.P., Yan, S., Wang, C.E., Li, S.H., Li, X.J., 2014. Differential ubiquitination and degradation of huntingtin fragments modulated by ubiquitin-protein ligase E3A. Proc. Natl. Acad. Sci. U. S. A. 111, 5706–5711.

Bhattacharyya, S., Yu, H.Q., Mim, C., Matouschek, A., 2014. Regulated protein turnover: snapshots of the proteasome in action. Nat. Rev. Mol. Cell Biol. 15, 122–133.

Bizat, N., Peyrin, J.M., Haik, S., Cochois, V., Beaudry, P., Laplanche, J.L., Neri, C., 2010. Neuron dysfunction is induced by prion protein with an insertional mutation via a fyn kinase and reversed by sirtuin activation in *Caenorhabditis elegans*. J. Neurosci. 30, 5394–5403.

Bjorkoy, G., Lamark, T., Brech, A., Outzen, H., Perander, M., Overvatn, A., Stenmark, H., Johansen, T., 2005. p62/SQSTM1 forms protein aggregates degraded by autophagy and has a protective effect on huntingtin-induced cell death. J. Cell Biol. 171, 603–614.

Boellaard, J.W., Kao, M., Schlote, W., Diringer, H., 1991. Neuronal autophagy in experimental scrapie. Acta Neuropathol. 82, 225–228.

Borchelt, D.R., Scott, M., Taraboulos, A., Stahl, N., Prusiner, S.B., 1990. Scrapie and cellular prion proteins differ in their kinetics of synthesis and topology in cultured-cells. J. Cell Biol. 110, 743–752.

Borchelt, D.R., Taraboulos, A., Prusiner, S.B., 1992. Evidence for synthesis of scrapie prion proteins in the endocytic pathway. J. Biol. Chem. 267, 16188–16199.

Brakman, I., Hebert, D.N., 2013. Protein folding in the endoplasmic reticulum. Cold Spring Harb. Perspect. Biol. 5.

Butler, D., Hwang, J., Estick, C., Nishiyama, A., Kumar, S.S., Baveghems, C., Young-Oxendine, H.B., Wisniewski, M.L., Charalambides, A., Bahr, B.A., 2011. Protective effects of positive lysosomal modulation in Alzheimer's disease transgenic mouse models. PLoS One 6.

Cammarata, S., Tabaton, M., 1992. Ubiquitin-reactive axons have a widespread distribution and are unrelated to prion protein plaques in Creutzfeldt–Jakob disease. J. Neurol. Sci. 110, 32–36.

Capdeville, R., Buchdunger, E., Zimmermann, J., Matter, A., 2002. Glivec (ST1571, Imatinib), a rationally developed, targeted anticancer drug. Nat. Rev. Drug Discov. 1, 493–502.

Caughey, B., Race, R.E., 1992. Potent inhibition of scrapie-associated PrP accumulation by congo red. J. Neurochem. 59, 768–771.

Caughey, B., Raymond, G.J., 1991. The scrapie-associated form of PrP is made from a cell-surface precursor that is both protease-sensitive and phospholipase-sensitive. J. Biol. Chem. 266, 18217–18223.

Caughey, B., Raymond, G.J., 1993. Sulfated polyanion inhibition of scrapie-associated PrP accumulation in cultured-cells. J. Virol. 67, 643–650.

Caughey, B., Raymond, G.J., Ernst, D., Race, R.E., 1991. N-terminal truncation of the scrapie-associated form of PrP by lysosomal protease(s) — implications regarding the site of conversion of PrP to the protease-resistant state. J. Virol. 65, 6597–6603.

Chakrabarti, O., Hegde, R.S., 2009. Functional depletion of mahogunin by cytosolically exposed prion protein contributes to neurodegeneration. Cell 137, 1136–1147.

Chapple, S.J., Siow, R.C.M., Mann, G.E., 2012. Crosstalk between Nrf2 and the proteasome: therapeutic potential of Nrf2 inducers in vascular disease and aging. Int. J. Biochem. Cell Biol. 44, 1315–1320.

Clemens, M.J., 2004. Targets and mechanisms for the regulation of translation in malignant transformation. Oncogene 23, 3180–3188.

Cortes, C.J., Qin, K.F., Cook, J., Solanki, A., Mastrianni, J.A., 2012. Rapamycin delays disease onset and prevents PrP plaque deposition in a mouse model of Gerstmann–Straussler–Scheinker disease. J. Neurosci. 32, 12396–12405.

Cortez, L.M., Sim, V.L., 2013. Antiprion properties of orally active chemical chaperones. Prion 7, 84.

Dantuma, N., Bott, L., 2014. The ubiquitin–proteasome system in neurodegenerative diseases: precipitating factor, yet part of the solution. Front. Mol. Neurosci. 7, 70.

DeArmond, S.J., Bajsarowicz, K., 2010. PrPSc accumulation in neuronal plasma membranes links Notch-1 activation to dendritic degeneration in prion diseases. Mol. Neurodegener. 5.

Deng, J., Harding, H.P., Raught, B., Gingras, A.C., Berlanga, J.J., Scheuner, D., Kaufman, R.J., Ron, D., Sonenberg, N., 2002. Activation of GCN2 in UV-irradiated cells inhibits translation. Curr. Biol. 12, 1279–1286.

Deriziotis, P., Andre, R., Smith, D.M., Goud, R., Kinghorn, K.J., Kristiansen, M., Nathan, J.A., Rosenzweig, R., Krutauz, D., Glickman, M.H., et al., 2011. Misfolded PrP impairs the UPS by interaction with the 20S proteasome and inhibition of substrate entry. EMBO J. 30, 3065–3077.

Dron, M., Dandoy-Dron, F., Salamat, M.K.F., Laude, H., 2009. Proteasome inhibitors promote the sequestration of PrPSc into aggresomes within the cytosol of prion-infected CAD neuronal cells. J. Gen. Virol. 90, 2050–2060.

Ertmer, A., Gilch, S., Yun, S.W., Flechsig, E., Klebl, B., Stein-Gerlach, M., Klein, M.A., Schatzl, H.M., 2004. The tyrosine kinase inhibitor STI571 induces cellular clearance of PrPSc in prion-infected cells. J. Biol. Chem. 279, 41918–41927.

Escusa-Toret, S., Vonk, W.I.M., Frydman, J., 2013. Spatial sequestration of misfolded proteins by a dynamic chaperone pathway enhances cellular fitness during stress. Nat. Cell Biol. 15, 1231–1243.

Finley, D., 2009. Recognition and processing of ubiquitin–protein conjugates by the proteasome. Annu. Rev. Biochem. 78, 477–513.

Gamerding, M., Hajieva, P., Kaya, A.M., Wolfrum, U., Hartl, F.U., Behl, C., 2009. Protein quality control during aging involves recruitment of the macroautophagy pathway by BAG3. EMBO J. 28, 889–901.

Gan, N.Q., Wu, Y.C., Brunet, M., Garrido, C., Chung, F.L., Dai, C.K., Mi, L.X., 2010. Sulforaphane activates heat shock response and enhances proteasome activity through up-regulation of Hsp27. J. Biol. Chem. 285, 35528–35536.

Gilch, S., Winklhofer, K.F., Groschup, M.H., Nunziante, M., Lucassen, R., Spielhaupter, C., Muranyi, W., Riesner, D., Tatzelt, J., Schatzl, H.M., 2001. Intracellular re-routing of prion protein prevents propagation of PrPSc and delays onset of prion disease. EMBO J. 20, 3957–3966.

Goold, R., Rabbanian, S., Sutton, L., Andre, R., Arora, P., Moonga, J., Clarke, A.R., Schiavo, G., Jat, P., Collinge, J., et al., 2011. Rapid cell-surface prion protein conversion revealed using a novel cell system. Nat. Commun. 2.

Goold, R., McKinnon, C., Rabbanian, S., Collinge, J., Schiavo, G., Tabrizi, S.J., 2013. Alternative fates of newly formed PrPSc upon prion conversion on the plasma membrane. J. Cell Sci. 126, 3552–3562.

Grenier, C., Bissonnette, C., Volkov, L., Roucou, X., 2006. Molecular morphology and toxicity of cytoplasmic prion protein aggregates in neuronal and non-neuronal cells. J. Neurochem. 97, 1456–1466.

Groll, M., Bajorek, M., Kohler, A., Moroder, L., Rubin, D.M., Huber, R., Glickman, M.H., Finley, D., 2000. A gated channel into the proteasome core particle. Nat. Struct. Biol. 7, 1062–1067.

---

Please cite this article as: Goold, R., et al., Prion degradation pathways: Potential for therapeutic intervention, Mol. Cell. Neurosci. (2015), http://dx.doi.org/10.1016/j.mcn.2014.12.009

ARTICLE IN PRESS

R. Goold et al. / Molecular and Cellular Neuroscience xxx (2015) xxx–xxx

Hao, R., Nanduri, P., Rao, Y.H., Panichelli, R.S., Ito, A., Yoshida, M., Yao, T.P., 2013. Proteasomes activate aggresome disassembly and clearance by producing unanchored ubiquitin chains. Mol. Cell 51, 819–828.

Hara, T., Nakamura, K., Matsui, M., Yamamoto, A., Nakahara, Y., Suzuki-Migishima, R., Yokoyama, M., Mishima, K., Saito, I., Okano, H., et al., 2006. Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice. Nature 441, 885–889.

Harding, H.P., Zhang, Y.H., Ron, D., 1999. Protein translation and folding are coupled by an endoplasmic-reticulum-resident kinase. Nature 397, 271–274.

Heiseke, A., Aguib, Y., Riemer, C., Baier, M., Schatzl, H.M., 2009. Lithium induces clearance of protease resistant prion protein in prion-infected cells by induction of autophagy. J. Neurochem. 109, 25–34.

Heiseke, A., Aguib, Y., Schatzl, H.M., 2010. Autophagy, prion infection and their mutual interactions. Curr. Mol. Biol. 12, 87–97.

Hetz, C., Mollereau, B., 2014. Disturbance of endoplasmic reticulum proteostasis in neurodegenerative diseases. Nat. Rev. Neurosci. 15, 233–249.

Hetz, C.A., Soto, C., 2006. Stressing out the EIR: a role of the unfolded protein response in prion-related disorders. Curr. Mol. Med. 6, 37–43.

Hetz, C., Russelakis-Carneiro, M., Maundrell, K., Castilla, J., Soto, C., 2003. Caspase-12 and endoplasmic reticulum stress mediate neurotoxicity of pathological prion protein. EMBO J. 22, 5435–5445.

Hipp, M., Park, S., Hartl, F.U., 2014. Proteostasis impairment in protein-misfolding and -aggregation diseases. Trends Cell Biol. 24, 506–514.

Homma, T., Ishibashi, D., Nakagaki, T., Satoh, K., Sano, K., Atarashi, R., Nishida, N., 2014. Increased expression of p62/SQSTM1 in prion diseases and its association with pathogenic prion protein. Sci. Rep. 4.

Hong, L., Huang, H.C., Jiang, Z.F., 2014. Relationship between amyloid-beta and the ubiquitin-proteasome system in Alzheimer's disease. Neurol. Res. 36, 276–282.

Jeffrey, M., Martin, S., Gonzalez, L., 2003. Cell-associated variants of disease-specific prion protein immunolabelling are found in different sources of sheep transmissible spongiform encephalopathy. J. Gen. Virol. 84, 1033–1045.

Jeffrey, M., McGovern, G., Martin, S., Siso, S., Gonzalez, L., 2010. Neuropathology of animal prion diseases toxic effects of PrPd, relationships with strain and clinical disease. Prion 4, 113.

Jen, A., Parkyn, C.J., Mootoosamy, R.C., Ford, M.J., Warley, A., Liu, Q., Bu, G.J., Baskakov, I.V., Moestrup, S., McGuinness, L., et al., 2010. Neuronal low-density lipoprotein receptor-related protein 1 binds and endocytoses prion fibrils via receptor cluster 4. J. Cell Sci. 123, 246–255.

Jeong, J.K., Moon, M.H., Bae, B.C., Lee, Y.J., Seol, J.W., Kang, H.S., Kim, J.S., Kang, S.J., Park, S.Y., 2012. Autophagy induced by resveratrol prevents human prion protein-mediated neurotoxicity. Neurosci. Res. 73, 99–105.

Jimenez-Sanchez, M., Thomson, F., Zavodszky, E., Rubinsztein, D.C., 2012. Autophagy and polyglutamine diseases. Prog. Neurobiol. 97, 67–82.

Jin, T.C., Gu, Y.P., Zanusso, G., Sy, M.S., Kumar, A., Cohen, M., Gambetti, P., Singh, N., 2000. The chaperone protein BiP binds to a mutant prion protein and mediates its degradation by the proteasome. J. Biol. Chem. 275, 38699–38704.

Kalmar, B., Edet-Amana, E., Greensmith, L., 2012. Treatment with a coinducer of the heat shock response delays muscle denervation in the SOD1-G93A mouse model of amyotrophic lateral sclerosis. Amyotroph. Lateral Scler. 13, 378–392.

Kang, S.C., Brown, D.R., Whiteman, M., Li, R.L., Pan, T., Perry, G., Wisniewski, T., Sy, M.S., Wong, B.S., 2004. Prion protein is ubiquitinated after developing protease resistance in the brains of scrapie-infected mice. J. Pathol. 203, 603–608.

Kawaguchi, Y., Kovacs, J.J., McLaurin, A., Vance, J.M., Ito, A., Yao, T.P., 2003. The deacetylase HDAC6 regulates aggresome formation and cell viability in response to misfolded protein stress. Cell 115, 727–738.

Kenward, N., Hope, J., Landon, M., Mayer, R.J., 1994. Expression of polyubiquitin and heat-shock protein-70 genes increases in the later stages of disease progression in scrapie-infected mouse-brain. J. Neurochem. 62, 1870–1877.

Kim, Y.E., Hipp, M.S., Bracher, A., Hayer-Hartl, M., Hartl, F.U., 2013. Molecular chaperone functions in protein folding and proteostasis. Annu. Rev. Biochem. 82 (82), 323–355.

Kirkin, V., Lamark, T., Sou, Y.S., Bjorkoy, G., Nunn, J.L., Bruun, J.A., Shvets, E., Mcewan, D.G., Clausen, T.H., Wild, P., et al., 2009. A role for NBR1 in autophagosomal degradation of ubiquitinated substrates. Mol. Cell 33, 505–516.

Kitada, T., Asakawa, S., Hattori, N., Matsumine, H., Yamamura, Y., Minoshima, S., Yokochi, M., Mizuno, Y., Shimizu, N., 1998. Mutations in the parkin gene cause autosomal recessive juvenile parkinsonism. Nature 392, 605–608.

Korolchuk, V.I., Menzies, F.M., Rubinsztein, D.C., 2010. Mechanisms of cross-talk between the ubiquitin-proteasome and autophagy-lysosome systems. FEBS Lett. 584, 1393–1398.

Kovacs, G.G., Preusser, M., Strohschneider, M., Budka, H., 2005. Subcellular localization of disease-associated prion protein in the human brain. Am. J. Pathol. 166, 287–294.

Kristiansen, M., Messenger, M.J., Klohn, P.C., Brandner, S., Wadsworth, J.D.F., Collinge, J., Tabrizi, S.J., 2005. Disease-related prion protein forms aggresomes in neuronal cells leading to caspase activation and apoptosis. J. Biol. Chem. 280, 38851–38861.

Kristiansen, M., Deriziotis, P., Dimcheff, D.E., Jackson, G.S., Ovaa, H., Naumann, H., Clarke, A.R., van Leeuwen, F.W.B., Menendez-Benito, V., Dantuma, N.P., et al., 2007. Disease-associated prion protein oligomers inhibit the 26S proteasome. Mol. Cell 26, 175–188.

Kwak, M.K., Cho, J.M., Huang, B., Kensler, T.W., 2007. Role of induction of the 26S proteasome in protective effects of sulforaphane against hydrogen peroxide-mediated cytotoxicity in murine neuroblastoma cells. FASEB J. 21, A1175–A1176.

Lee, A.H., Iwakoshi, N.N., Anderson, K.C., Glimcher, L.H., 2003. Proteasome inhibitors disrupt the unfolded protein response in myeloma cells. Proc. Natl. Acad. Sci. U. S. A. 100, 9946–9951.

Lee, B.H., Lee, M.J., Park, S., Oh, D.C., Elsasser, S., Chen, P.C., Gartner, C., Dimova, N., Hanna, J., Gygi, S.P., et al., 2010. Enhancement of proteasome activity by a small-molecule inhibitor of USP14. Nature 467, 179–184.

Lee, P.C.W., Sowa, M.E., Gygi, S.P., Harper, J.W., 2011. Alternative ubiquitin activation/conjugation cascades interact with N-end rule ubiquitin ligases to control degradation of RGS proteins. Mol. Cell 43, 392–405.

Lee, J.H., Jeong, J.K., Park, S.Y., 2014. Sulforaphane-induced autophagy flux prevents prion protein-mediated neurotoxicity through AMPK. Neuroscience 278, 31–39.

Li, X., Wang, C.E., Huang, S., Xu, X.S., Li, X.J., Li, H., Li, S.H., 2010. Inhibiting the ubiquitin-proteasome system leads to preferential accumulation of toxic N-terminal mutant huntingtin fragments. Hum. Mol. Genet. 19, 2445–2455.

Liberski, P.P., Sikorska, B., Hauw, J.J., Kopp, N., Streichenberger, N., Giraud, P., Boellaard, J., Budka, H., Kovacs, G.G., Ironside, J., et al., 2010. Ultrastructural characteristics (or evaluation) of Creutzfeldt-Jakob disease and other human transmissible spongiform encephalopathies or prion diseases. Ultrastruct. Pathol. 34, 351–361.

Lindquist, S.L., Kelly, J.W., 2011. Chemical and biological approaches for adapting proteostasis to ameliorate protein misfolding and aggregation diseases — progress and prognosis. Cold Spring Harb. Perspect. Biol. 3.

Liu, Y.Y., Hettinger, C.L., Zhang, D., Rezvani, K., Wang, X.J., Wang, H.M., 2014. Sulforaphane enhances proteasomal and autophagic activities in mice and is a potential therapeutic reagent for Huntington's disease. J. Neurochem. 129, 539–547.

Lowe, J., Fergusson, J., Kenward, N., Laszlo, L., Landon, M., Farquhar, C., Brown, J., Hope, J., Mayer, R.J., 1992. Immunoreactivity to ubiquitin protein conjugates is present early in the disease process in the brains of scrapie-infected mice. J. Pathol. 168, 169–177.

Magalhaes, A.C., Baron, G.S., Lee, K.S., Steele-Mortimer, O., Dorward, D., Prado, M.A.M., Caughey, B., 2005. Uptake and neuritic transport of scrapie prion protein coincident with infection of neuronal cells. J. Neurosci. 25, 5207–5216.

Maiese, K., Chong, Z.Z., Shang, Y.C., Wang, S.H., 2013. mTOR: on target for novel therapeutic strategies in the nervous system. Trends Mol. Med. 19, 51–60.

Malik, B., Nirmalananthan, N., Gray, A.L., La Spada, A.R., Hanna, M.G., Greensmith, L., 2013. Co-induction of the heat shock response ameliorates disease progression in a mouse model of human spinal and bulbar muscular atrophy: implications for therapy. Brain 136, 926–943.

Mallucci, G., Dickinson, A., Linehan, J., Klohn, P.C., Brandner, S., Collinge, J., 2003. Depleting neuronal PrP in prion infection prevents disease and reverses spongiosis. Science 302, 871–874.

Mallucci, G.R., White, M.D., Farmer, M., Dickinson, A., Khatun, H., Powell, A.D., Brandner, S., Jefferys, J.G.R., Collinge, J., 2007. Targeting cellular prion protein reverses early cognitive deficits and neuropathological dysfunction in prion-infected mice. Neuron 53, 325–335.

Marijanovic, Z., Caputo, A., Campana, V., Zurzolo, C., 2009. Identification of an intracellular site of prion conversion. PLoS Pathog. 5.

Marzo, L., Marijanovic, Z., Bowman, D., Chamoun, Z., Caputo, A., Zurzolo, C., 2013. 4-Hydroxytamoxifen leads to PrPSc clearance by conveying both PrPC and PrPSc to lysosomes independently of autophagy. J. Cell Sci. 126, 1345–1354.

McKinnon, C., Tabrizi, S.J., 2014. The ubiquitin-proteasome system in neurodegeneration. Antioxid. Redox Signal. 21, 2302–2321.

Menendez-Benito, V., Verhoef, L.G.G.C., Masucci, M.G., Dantuma, N.P., 2005. Endoplasmic reticulum stress compromises the ubiquitin-proteasome system. Hum. Mol. Genet. 14, 2787–2799.

Micsenyi, M.C., Sikora, J., Stephney, G., Dobrenis, K., Walkley, S.U., 2013. Lysosomal membrane permeability stimulates protein aggregate formation in neurons of a lysosomal disease. J. Neurosci. 33, 10815–10827.

Moreno, J.A., Radford, H., Peretti, D., Steinert, J.R., Verity, N., Martin, M.G., Halliday, M., Morgan, J., Dinsdale, D., Ortori, C.A., et al., 2012. Sustained translational repression by eIF2 alpha-P mediates prion neurodegeneration. Nature 485, 507–511.

Moreno, J.A., Halliday, M., Molloy, C., Radford, H., Verity, N., Axten, J.M., Ortori, C.A., Willis, A.E., Fischer, P.M., Barrett, D.A., et al., 2013. Oral treatment targeting the unfolded protein response prevents neurodegeneration and clinical disease in prion-infected mice. Sci. Transl. Med. 5.

Nakagaki, T., Satoh, K., Ishibashi, D., Fuse, T., Sano, K., Kamatari, Y.O., Kuwata, K., Shigematsu, K., Iwamaru, Y., Takenouchi, T., et al., 2013. FK506 reduces abnormal prion protein through the activation of autolysosomal degradation and prolongs survival in prion-infected mice. Autophagy 9, 1386–1394.

Nedelsky, N.B., Todd, P.K., Taylor, J.P., 2008. Autophagy and the ubiquitin-proteasome system: collaborators in neuroprotection. Biochim. Biophys. Acta Mol. Basis Dis. 1782, 691–699.

Nixon, R.A., 2013. The role of autophagy in neurodegenerative disease. Nat. Med. 19, 983–997.

Nunziante, M., Ackermann, K., Dietrich, K., Wolf, H., Gadkte, L., Gilch, S., Vorberg, I., Groschup, M., Schatzl, H.M., 2011. Proteasomal dysfunction and endoplasmic reticulum stress enhance trafficking of prion protein aggregates through the secretory pathway and increase accumulation of pathologic prion protein. J. Biol. Chem. 286, 33942–33953.

Ozcnelik, S., Fraser, G., Castets, P., Schaeffer, V., Skachokova, Z., Breu, K., Clavaguera, F., Sinnreich, M., Kappos, L., Goedert, M., et al., 2013. Rapamycin attenuates the progression of tau pathology in P301S tau transgenic mice. Plos One 8.

Pankiv, S., Clausen, T.H., Lamark, T., Brech, A., Bruun, J.A., Outzen, H., Overvatn, A., Bjorkoy, G., Johansen, T., 2007. p62/SQSTM1 binds directly to Atg8/LC3 to facilitate degradation of ubiquitinated protein aggregates by autophagy. J. Biol. Chem. 282, 24131–24145.

---

Please cite this article as: Goold, R., et al., Prion degradation pathways: Potential for therapeutic intervention, Mol. Cell. Neurosci. (2015), http://dx.doi.org/10.1016/j.mcn.2014.12.009

Powers, E.T., Morimoto, R.I., Dillin, A., Kelly, J.W., Balch, W.E., 2009. Biological and chemical approaches to diseases of proteostasis deficiency. Annu. Rev. Biochem. 78, 959–991.

Priola, S.A., Raines, A., Caughey, W.S., 2000. Porphyrin and phthalocyanine antiscrapie compounds. Science 287, 1503–1506.

Prusiner, S.B., 1998. Prions. Proc. Natl. Acad. Sci. U. S. A. 95, 13363–13383.

Qin, Z.H., Wang, Y.M., Kegel, K.B., Kazantsev, A., Apostol, B.L., Thompson, L.M., Yoder, J., Aronin, N., DiFiglia, M., 2003. Autophagy regulates the processing of amino terminal huntingtin fragments. Hum. Mol. Genet. 12, 3231–3244.

Rane, N.S., Kang, S.W., Chakrabarti, O., Feigenbaum, L., Hegde, R.S., 2008. Reduced translocation of nascent prion protein during ER stress contributes to neurodegeneration. Dev. Cell 15, 359–370.

Ravikumar, B., Vacher, C., Berger, Z., Davies, J.E., Luo, S.Q., Oroz, L.G., Scaravilli, F., Easton, D.F., Duden, R., O'Kane, C.J., et al., 2004. Inhibition of mTOR induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of Huntington disease. Nat. Genet. 36, 585–595.

Rouvinski, A., Karniely, S., Kounin, M., Moussa, S., Goldberg, M.D., Warburg, G., Lyakhovetsky, R., Papy-Garcia, D., Kutzsche, J., Korth, C., et al., 2014. Live imaging of prions reveals nascent PrPSc in cell-surface, raft-associated amyloid strings and webs. J. Cell Biol. 204, 423–441.

Rubinsztein, D.C., Shpilka, T., Elazar, Z., 2012. Mechanisms of autophagosome biogenesis. Curr. Biol. 22, R29–R34.

Saftig, P., Klumperman, J., 2009. Lysosome biogenesis and lysosomal membrane proteins: trafficking meets function. Nat. Rev. Mol. Cell Biol. 10, 623–635.

Sarkar, S., Floto, R.A., Berger, Z., Imarisio, S., Cordenier, A., Pasco, M., Cook, L.J., Rubinsztein, D.C., 2005. Lithium induces autophagy by inhibiting inositol monophosphatase. J. Cell Biol. 170, 1101–1111.

Scotter, E.L., Vance, C., Nishimura, A.L., Lee, Y.B., Chen, H.J., Urwin, H., Sardone, V., Mitchell, J.C., Rogelj, B., Rubinsztein, D.C., et al., 2014. Differential roles of the ubiquitin proteasome and autophagy in the clearance of soluble and aggregated TDP-43 species. J. Cell Sci. 127, 1263–1278.

Seo, J.S., Moon, M.H., Jeong, J.K., Seol, J.W., Lee, Y.J., Park, B.H., Park, S.Y., 2012. SIRT1, a histone deacetylase, regulates prion protein-induced neuronal cell death. Neurobiol. Aging 33, 1110–1120.

Sikorska, B., Liberski, P.P., Giraud, P., Kopp, N., Brown, P., 2004. Autophagy is a part of ultrastructural synaptic pathology in Creutzfeldt–Jakob disease : a brain biopsy study. Int. J. Biochem. Cell Biol. 36, 2563–2573.

Solassol, J., Crozet, C., Perrier, V., Leclaire, J., Beranger, F., Caminade, A.M., Meunier, B., Dormont, D., Majoral, J.P., Lehmann, S., 2004. Cationic phosphorus-containing dendrimers reduce prion replication both in cell culture and in mice infected with scrapie. J. Gen. Virol. 85, 1791–1799.

Sontag, E.M., Vonk, W.I.M., Frydman, J., 2014. Sorting out the trash: the spatial nature of eukaryotic protein quality control. Curr. Opin. Cell Biol. 26, 139–146.

Soto, C., 2003. Unfolding the role of protein misfolding in neurodegenerative diseases. Nat. Rev. Neurosci. 4, 49–60.

Spilman, P., Podlutskaya, N., Hart, M.J., Debnath, J., Gorostiza, O., Bredesen, D., Richardson, A., Strong, R., Galvan, V., 2010. Inhibition of mTOR by rapamycin abolishes cognitive deficits and reduces amyloid-beta levels in a mouse model of Alzheimer's disease. PLoS One 5.

Supattapone, S., Nguyen, H.O.B., Cohen, F.E., Prusiner, S.B., Scott, M.R., 1999. Elimination of prions by branched polyamines and implications for therapeutics. Proc. Natl. Acad. Sci. U. S. A. 96, 14529–14534.

Supattapone, S., Wille, H., Uyechi, L., Safar, J., Tremblay, P., Szoka, F.C., Cohen, F.E., Prusiner, S.B., Scott, M.R., 2001. Branched polyamines cure prion-infected neuroblastoma cells. J. Virol. 75, 3453–3461.


Tashiro, Y., Urushitani, M., Inoue, H., Koike, M., Uchiyama, Y., Komatsu, M., Tanaka, K., Yamazaki, M., Abe, M., Misawa, H., et al., 2012. Motor neuron-specific disruption of proteasomes, but not autophagy, replicates amyotrophic lateral sclerosis. J. Biol. Chem. 287, 42984–42994.

Tilly, G., Chapuis, J., Vilette, D., Laude, H., Vilotte, J.L., 2003. Efficient and specific down-regulation of prion protein expression by RNAi. Biochem. Biophys. Res. Commun. 305, 548–551.

Trevitt, C.R., Collinge, J., 2006. A systematic review of prion therapeutics in experimental models. Brain 129, 2241–2265.

Tydlacka, S., Wang, C.E., Wang, X.J., Li, S.H., Li, X.J., 2008. Differential activities of the ubiquitin-proteasome system in neurons versus glia may account for the preferential accumulation of misfolded proteins in neurons. J. Neurosci. 28, 13285–13295.

van Leeuwen, F.W., van Tijn, P., Sonnemans, M.A.F., Hobo, B., Mann, D.M.A., Van Broeckhoven, C., Kumar-Singh, S., Cras, P., Leuba, G., Savioz, A., et al., 2006. Frameshift proteins in autosomal dominant forms of Alzheimer disease and other tauopathies. Neurology 66, S86–S92.

Veith, N.M., Plattner, H., Stuermer, C.A.O., Schulz-Schaeffer, W.J., Burkle, A., 2009. Immunolocalisation of PrPSc in scrapie-infected N2a mouse neuroblastoma cells by light and electron microscopy. Eur. J. Cell Biol. 88, 45–63.

Viswanathan, K., Hoover, D.J., Hwang, J., Wisniewski, M.L., Ikonne, U.S., Bahr, B.A., Wright, D.L., 2012. Nonpeptidic lysosomal modulators derived from Z-Phe-Ala-diazomethylketone for treating protein accumulation diseases. ACS Med. Chem. Lett. 3, 920–924.

Walter, P., Ron, D., 2011. The unfolded protein response: from stress pathway to homeostatic regulation. Science 334, 1081–1086.

Wang, I.F., Guo, B.S., Liu, Y.C., Wu, C.C., Yang, C.H., Tsai, K.J., Shen, C.K.J., 2012. Autophagy activators rescue and alleviate pathogenesis of a mouse model with proteinopathies of the TAR DNA-binding protein 43. Proc. Natl. Acad. Sci. U. S. A. 109, 15024–15029.

Webb, J.L., Ravikumar, B., Atkins, J., Skepper, J.N., Rubinsztein, D.C., 2003. Alpha-synuclein is degraded by both autophagy and the proteasome. J. Biol. Chem. 278, 25009–25013.

Williams, A., Sarkar, S., Cuddon, P., Ttofi, E.K., Saiki, S., Siddiqi, F.H., Jahreiss, L., Fleming, A., Pask, D., Goldsmith, P., et al., 2008. Novel targets for Huntington's disease in an mTOR-independent autophagy pathway. Nat. Chem. Biol. 4, 295–305.

Yamasaki, T., Baron, G.S., Suzuki, A., Hasebe, R., Horiuchi, M., 2014. Characterization of intracellular dynamics of inoculated PrP-res and newly generated PrPSc during early stage prion infection in Neuro2a cells. Virology 450, 324–335.

Yao, H., Zhao, D.M., Khan, S.H., Yang, L.F., 2013. Role of autophagy in prion protein-induced neurodegenerative diseases. Acta Biochim. Biophys. Sin. 45, 494–502.

Yedidia, Y., Horonchik, L., Tzaban, S., Yanai, A., Taraboulos, A., 2001. Proteasomes and ubiquitin are involved in the turnover of the wild-type prion protein. EMBO J. 20, 5383–5391.

Yun, S.W., Ertmer, A., Flechsig, E., Gilch, S., Riederer, P., Gerlach, M., Schatzl, H.M., Klein, M.A., 2007. The tyrosine kinase inhibitor imatinib mesylate delays prion neuroinvasion by inhibiting prion propagation in the periphery. J. Neurovirol. 13, 328–337.

Zanusso, G., Petersen, R.B., Jin, T.C., Jing, Y., Kanoush, R., Ferrari, S., Gambetti, P., Singh, N., 1999. Proteasomal degradation and N-terminal protease resistance of the codon 145 mutant prion protein. J. Biol. Chem. 274, 23396–23404.

Zhou, H., Cao, F.L., Wang, Z.S., Yu, Z.X., Nguyen, H.P., Evans, J., Li, S.H., Li, X.J., 2003. Huntingtin forms toxic NH2-terminal fragment complexes that are promoted by the age-dependent decrease in proteasome activity. J. Cell Biol. 163, 109–118.
