
ARTICLE IN PRESS

BBAMCR-17477; No. of pages: 10; 4C: 2, 4, 5, 6, 7

Biochimica et Biophysica Acta xxx (2015) xxx–xxx

---

Contents lists available at ScienceDirect

**Biochimica et Biophysica Acta**

journal homepage: www.elsevier.com/locate/bbamcr

---

Review

**Mitophagy in yeast: Molecular mechanisms and physiological role** ✩

Tomotake Kanki ✩, Kentaro Furukawa, Shun-ichi Yamashita

Laboratory of Biosignaling, Niigata University Graduate School of Medical and Dental Sciences, 1-757 Asahimachi-dori, Chuo-ku, Niigata 951-8510, Japan

---

### ARTICLE INFO

Article history:

- Received 31 October 2014
- Received in revised form 24 December 2014
- Accepted 8 January 2015
- Available online xxxx

Keywords:
- Autophagy
- Mitophagy
- Mitochondrion
- Atg32
- ERMES
- Yeast

---

### ABSTRACT

Mitochondria autophagy (mitophagy) is a process that selectively degrades mitochondria via autophagy. Recently, there has been significant progress in the understanding of mitophagy in yeast. Atg32, a mitochondrial outer membrane receptor, is indispensable for mitophagy. Phosphorylation of Atg32 is an initial cue for selective mitochondrial degradation. Atg32 expression and phosphorylation regulate the induction and efficiency of mitophagy. In addition to Atg32-related processes, recent studies have revealed that mitochondrial fission and the mitochondria–endoplasmic reticulum (ER) contact site may play important roles in mitophagy. Mitochondrial fission is required to regulate mitochondrial size. Mitochondria–ER contact is mediated by the ER–mitochondria encounter structure and is important to supply lipids from the ER for autophagosome biogenesis for mitophagy. Mitophagy is physiologically important for regulating the number of mitochondria, diminishing mitochondrial production of reactive oxygen species, and extending chronological lifespan under caloric restriction. These findings suggest that mitophagy contributes to maintain mitochondrial homeostasis. However, whether mitophagy selectively degrades damaged or dysfunctional mitochondria in yeast is unknown.

This article is part of a Special Issue entitled: Mitophagy.

© 2015 Elsevier B.V. All rights reserved.

---

### 1. Introduction

Autophagy is a catabolic process that delivers cytoplasmic proteins or organelles to lysosomes in mammalian cells or vacuoles in yeast, and is conserved in almost all eukaryotes. Autophagy is initiated in response to cellular stresses, such as nutrient starvation, oxidative stress, infection, and inflammatory stimuli. Upon induction of autophagy, a cup-shaped double-membrane structure, called an isolation membrane (or phagophore), emerges in the cytoplasm. The isolation membrane then elongates with curvature and finally becomes enclosed, forming an autophagosome containing cytoplasmic components. Subsequently, autophagosomes fuse with lysosomes/vacuoles, and lysosomal hydrolases degrade the sequestered material [1–5]. As well as a response to cellular stress, autophagy plays diverse roles in cellular development, tumor suppression, immune responses, aging, and in the prevention of many diseases in mammals, such as infection, neurodegenerative disease, cardiomyopathy, and diabetes [6–9].

Although autophagy was initially thought as a nonselective degradative pathway, recent evidence has been accumulating that autophagy can selectively degrade specific organelles or proteins [10]. Mitochondria, peroxisomes, ribosomes, endoplasmic reticulum (ER), lipid droplets, protein aggregates, and invasive microbes are recognized as selective autophagic cargo, and these selective autophagic processes are termed mitophagy [11], pexophagy [12], ribophagy [13], ERphagy [14], lipophagy

---

✩ This article is part of a Special Issue entitled: Mitophagy.

* Corresponding author. Tel.: +81 25 227 2156; fax: +81 25 227 0769.

E-mail address: kanki@med.niigata-u.ac.jp (T. Kanki).

http://dx.doi.org/10.1016/j.bbamcr.2015.01.005

0167-4889/© 2015 Elsevier B.V. All rights reserved.

---

Please cite this article as: T. Kanki, et al., Mitophagy in yeast: Molecular mechanisms and physiological role, Biochim. Biophys. Acta (2015), http://dx.doi.org/10.1016/j.bbamcr.2015.01.005

degradation system [27–29], DNA repair enzymes [30,31], and phospholipid hydroperoxide glutathione peroxidase [32]. In addition, researchers have recently suggested that mitophagy selectively degrades damaged or excess mitochondria, and as a result, maintains mitochondrial quality and quantity [33–36].

In recent years, there has been significant progress in the study of mitophagy. In mammalian cells, several mitophagy-related factors, such as Nix/BNIP3L and BNIP3, FUNDC1, Parkin, PTEN-induced putative kinase 1 (PINK1), and optineurin, have been identified [34,37–43]. Nix/BNIP3L and BNIP3 are mitochondrial outer membrane proteins that are required for autophagic elimination of mitochondria during erythroid cell maturation [38,39]. FUNDC1 is also a mitochondrial outer membrane protein and binds with LC3 (the ortholog of yeast Atg8) and ULK1 (the ortholog of yeast Atg1) to induce mitophagy upon hypoxia or carbonylcyanide p-trifluoromethoxyphenylhydrazone treatment [40,42]. Parkin and PINK1 are encoded by the PARK2 and PARK6 genes, respectively; both are responsible for familial Parkinson's disease [44, 45]. PINK1 targets the mitochondrial inner membrane in energized mitochondria, is cleaved by MPP or PARL, and then is released into the cytosol or degraded within mitochondria [46,47]. When mitochondria are depolarized, PINK1 cannot translocate across the mitochondrial outer membrane and accumulates on the outer membrane of depolarized mitochondria [36,48]. Parkin, an E3 ubiquitin ligase, then translocates to the mitochondria in a PINK1-dependent manner and ubiquitates many mitochondrial outer membrane proteins. The ubiquitinated mitochondria are eventually degraded by mitophagy [34,36,41,48–53]. Optineurin accumulates on ubiquitinated mitochondria and recruits DFCP1 and LC3 to lead to autophagosome formation for mitophagy [43]. Although mitophagy in mammalian cells has been intensively studied, many open questions remain, such as whether Nix, FUNDC1, and Parkin–PINK1 cooperate to degrade mitochondria. In contrast, the molecular mechanisms of mitophagy in yeast are relatively well understood because of the identification and characterization of the indispensable mitophagy factor Atg32 (Fig. 1). In this

---

**ARTICLE IN PRESS**

T. Kanki et al. / Biochimica et Biophysica Acta xxx (2015) xxx–xxx

review, we summarize the molecular mechanisms and physiological role of mitophagy, including recent progress in yeast.

### 2. Molecular mechanisms of mitophagy

#### 2.1. Mitophagy, a type of autophagy

There are three morphologically different types of autophagy: macroautophagy, microautophagy, and chaperone-mediated autophagy (CMA). Macroautophagy is a well-known type of autophagy that involves autophagosomes containing cytoplasmic components that deliver their cargo to lysosomes/vacuoles for degradation. In contrast to macroautophagy, microautophagy sequesters cytoplasmic components by direct invagination or protrusion/septation of the lysosomal or vacuolar membrane [54]. CMA is a chaperone-dependent process that relies on cytosolic and lysosomal hsc70 to move substrates into the lysosome lumen. CMA has been identified in higher eukaryotes, but not in yeast [55]. Although mitophagy is a type of autophagy, whether mitophagy belongs to macroautophagy or microautophagy is still unclear. Kissova et al. and Nowikovsky et al. found direct invagination of mitochondria within vacuolar membranes by electron microscopy, suggesting that mitophagy belongs to microautophagy [56,57]. In contrast, Okamoto et al. and Kanki et al. found autophagosomes containing mitochondria by electron microscopy, suggesting that mitophagy belongs to macroautophagy [58,59]. The size of autophagic bodies containing mitochondria is similar to the size of those containing cytosolic components, suggesting that the mechanism of autophagosome formation in mitophagy is similar or the same as that of nonselective autophagy [58,59]. In addition, most of the ATG genes that are essential for nonselective autophagy are also required for mitophagy. In 2004, Kissova et al. first reported that ATG5 is essential for mitophagy [60]. Since then, the role of most of the ATG genes in mitophagy has been examined [56,59,61–64] (Table 1). In addition to the ATG genes, CCZ1 and MON1 (which are required for autophagosome fusion with vacuoles),

![Diagram](attachment://image.png)

**Fig. 1.** Molecular mechanisms of mitophagy in yeast. The cell surface stress sensors, Sln1 and Wsc1, detect mitophagy-inducing stimuli and activate the Hog1 and Slt2 signaling pathways. Downstream of Hog1, CK2 phosphorylates Ser-114 and Ser-119 on Atg32. Atg11 interacts with Atg32 depending on phosphorylation of Ser-114 on Atg32. Atg11 recruits mitochondria to the pre-autophagosomal structure/phagophore assembly site (PAS) where the autophagosome is generated to enclose the mitochondria. At the PAS, the Atg8-family interacting motif (AIM) on Atg32 interacts with Atg8, which anchors on the isolation membrane, and the Atg32–Atg8 interaction facilitates the formation of the autophagosome surrounding the mitochondria. Autophagosomes carrying mitochondria eventually fuse with vacuoles for mitochondrial degradation. The factors downstream of the Slt2 signaling pathway that are required for mitophagy have not been identified yet.

---

Please cite this article as: T. Kanki, et al., Mitophagy in yeast: Molecular mechanisms and physiological role, Biochim. Biophys. Acta (2015), http://dx.doi.org/10.1016/j.bbamcr.2015.01.005

ARTICLE IN PRESS

T. Kanki et al. / Biochimica et Biophysica Acta xxx (2015) xxx–xxx

Table 1  
Requirement for ATG genes in macroautophagy, pexophagy, and mitophagy in S. cerevisiae.

| ATG genes | Macroautophagy | Pexophagy | Mitophagy | References |
|-----------|----------------|-----------|-----------|------------|
| ATG1      | ++             | ++        | ++        | [19,20,62,63] |
| ATG2      | ++             | ++        | ++        | [20,59,117] |
| ATG3      | ++             | NA        | ++        | [20,59] |
| ATG4      | ++             | NA        | ++        | [20,59] |
| ATG5      | ++             | ++        | ++        | [19,20,60] |
| ATG6      | ++             | NA        | ++        | [20,63] |
| ATG7      | ++             | ++        | ++        | [19,20,59,118] |
| ATG8      | ++             | ++        | ++        | [19,20,63] |
| ATG9      | ++             | ++        | ++        | [20,56,61,119] |
| ATG10     | ++             | ++        | ++        | [19,59] |
| ATG11     | –              | ++        | ++        | [61,65] |
| ATG12     | ++             | NA        | ++        | [20,63] |
| ATG13     | ++             | NA        | +         | [20,59,64] |
| ATG14     | ++             | ++        | ++        | [19,20,59] |
| ATG15*    | (++)           | (++)      | (++)      | [59,120–122] |
| ATG16     | ++             | NA        | ++        | [20,59] |
| ATG17     | ++             | +         | +         | [59,61,64,123–125] |
| ATG18     | ++             | ++        | ++        | [59,126,127] |
| ATG19     | –              | –         | –         | [61,128] |
| ATG20     | –              | +         | +         | [59,61,129] |
| ATG21     | –              | NA        | +/-       | [59,64,130] |
| ATG22     | –              | NA        | –         | [59,131] |
| ATG23     | –              | –         | +         | [59,119] |
| ATG24     | –              | +         | +         | [59,61,64,129] |
| ATG26     | –              | –         | –         | [59,132] |
| ATG27     | +              | +         | +         | [59,133,134] |
| ATG29     | ++             | +         | +/-       | [59,61,135] |
| ATG31     | ++             | ++        | +/-       | [59,61,136] |
| ATG32     | –              | –         | ++        | [58,59] |
| ATG33     | –              | –         | +         | [64] |
| ATG34     | –              | NA        | NA        | [76] |
| ATG36     | –              | ++        | –         | [68] |
| ATG38     | ++             | –         | –         | [137] |

Phenotypes of indicated gene knockout strain: ++, severe defect; +, partial defect; –, no defect.  
* This gene encodes a lipase; the cargo can be delivered into the vacuole, but cannot be degraded.

to peroxisomes and interacts with Atg11 to select peroxisomes as a cargo [68].

### 2.3. Atg32, a receptor for mitophagy

Two research groups performed a genome-wide screen for mutants that are defective in mitophagy; one group identified 36 mutants and the other identified 32 mutants [59,64]. From these screens, ATG32 was identified as indispensable for mitophagy. Deletion of ATG32 completely inhibits mitophagy, but does not affect bulk autophagy, the Cvt pathway, or pexophagy [58,59], indicating that ATG32 is a mitophagy-specific gene. To date, the ortholog of Atg32 has not been identified in mammals.

Atg32, the protein encoded by ATG32, is composed of 529 amino acids and is predicted to have a single transmembrane domain. Atg32 is located in the mitochondrial outer membrane with its N- and C-terminal regions oriented towards the cytosol and the mitochondrial intermembrane space (IMS), respectively [58,59] (Fig. 2). Because Atg32 is a mitophagy-specific protein and localizes in the mitochondrial outer membrane, Atg32 was speculated to be a receptor protein for mitophagy. In fact, yeast two-hybrid and immunoprecipitation experiments have shown that Atg32 interacts with Atg11 [58,59]. Notably, the Atg32–Atg11 interaction is dramatically increased under mitophagy-inducing conditions.

### 2.4. Phosphorylation mediates the Atg32–Atg11 interaction

The role of the C-terminus IMS region of Atg32 is unclear. Deletion of the IMS region of Atg32 marginally affects the efficiency of mitophagy, suggesting that the C-terminal IMS region of Atg32 is dispensable for mitophagy [69,70]. In contrast, Wang et al. reported that processing of the IMS of Atg32 by Yme1 is an important regulatory mechanism of mitophagy [71]. The transmembrane domain of Atg32 is required for its mitochondrial outer membrane targeting because Atg32 lacking this domain cannot localize on mitochondria and cannot induce mitophagy [69,70]. The N-terminal cytosolic region of Atg32 is essential for mitophagy. Interestingly, when this region is artificially anchored on peroxisomes, the peroxisomes are degraded by autophagy [70], suggesting that the cytosolic region of Atg32 is sufficient to function as a receptor of selective autophagy.

Atg32 is phosphorylated when mitophagy is induced. While more than two residues on Atg32 appear to be phosphorylated, Ser-114 and Ser-119 are the dominant residues for phosphorylation [69]. Yeast two-hybrid analysis has shown that the N-terminal 150 amino acids of Atg32 are essential for the Atg32–Atg11 interaction, and Ser-114 and Ser-119 are within the Atg11-interacting residues. Therefore, phosphorylation of Ser-114 and/or Ser-119 might be important for the Atg32–Atg11 interaction. As expected, a previous study showed that a Ser-114-to-Ala mutant of Atg32 did not interact with Atg11, and as a result could not induce mitophagy, while a Ser-119-to-Ala mutant of Atg32 only marginally affected mitophagy [69]. Similarly, phosphorylation of Ser-159 on PpAtg32, a residue corresponding to Ser-114 on Atg32 (Fig. 3), is essential for mitophagy in *P. pastoris* [72,73]. These findings suggest that phosphorylation of Ser-114 on Atg32 (serine 159 on PpAtg32) is an initial molecular switch for selective mitochondrial degradation.

Recently, phosphorylation-dependent receptor–adaptor interactions, such as the Atg32–Atg11 interaction, have been identified in another selective autophagy process. Ser-391 and Ser-394 of Atg19 are phosphorylated by the protein kinase Hrr25 and the phosphorylated Atg19 interacts with Atg11 [74,75]. Similarly, Ser-383 on Atg34, the α-mannosidase Ams1-selective autophagy receptor [76], and Ser-97 on Atg36, the pexophagy receptor in *S. cerevisiae* [68], are also phosphorylated by Hrr25. These phosphorylated receptors interact with Atg11 [74, 75,77]. In *P. pastoris*, Ser-112 on PpAtg30 is phosphorylated, and phosphorylated PpAtg30 interacts with PpAtg11 [73]. Interestingly, the phosphorylation sites of PpAtg30, Sc/PpAtg32, and ScAtg36 include the I/V-L-S<sup>phos</sup>-S/I-S common sequence (Fig. 3A). This sequence might be the consensus sequence for Atg11 interaction. Atg19 and Atg34

membrane trafficking-related genes, such as VAM7, TRS85, YPT7, and VPS41 (which are also required for nonselective autophagy), and PEP4 (a vacuolar protease) have been shown to be required for mitophagy [59,64]. These findings suggest that most of the molecular processes of mitophagy—from forming autophagosomes to delivering cargo into vacuoles—are the same as those of nonselective autophagy. The salient difference between mitophagy and nonselective autophagy is that mitophagy selects mitochondria as a cargo.

#### 2.2. Atg11, a protein for selective autophagy

Atg11 is an adaptor protein for selective autophagy that is localized in the cytosol. ATG11, the gene encoding the Atg11 protein, was first reported as essential for the Cvt pathway in *S. cerevisiae* [19]. This gene is essential for pexophagy in *Pichia pastoris*. Interestingly, Atg11 is not required for nonselective autophagy [65]. Because the Cvt pathway and pexophagy are types of selective autophagy, Atg11 was thought to work at the point of cargo selection [65]. In fact, Atg11 works as an adaptor protein, interacts with cargo-specific receptor proteins, and recruits the cargo to the pre-autophagosomal structure/phagophore assembly site (PAS), where the initial sequestering membrane structure is generated. In the Cvt pathway, Atg19 is a receptor protein that binds the Ape1 complex, which is composed of prApe1 and Ams1, to form the Cvt complex. Atg11 interacts with Atg19 to select and recruit the Cvt complex to the PAS [66]. Similarly, during pexophagy in *P. pastoris*, PpAtg30 is a receptor protein of pexophagy that localizes to peroxisomes, where it subsequently binds PpAtg11, allowing recruitment of peroxisomes to the PAS [67]. Recently, a receptor protein for pexophagy, Atg36, was identified in *S. cerevisiae*. Atg36 also localizes

Please cite this article as: T. Kanki, et al., Mitophagy in yeast: Molecular mechanisms and physiological role, Biochim. Biophys. Acta (2015), http://dx.doi.org/10.1016/j.bbamcr.2015.01.005

ARTICLE IN PRESS

T. Kanki et al. / Biochimica et Biophysica Acta xxx (2015) xxx–xxx

![Diagram](https://i.imgur.com/yourimage.png)

Fig. 2. Structure and domains of Atg32. Atg32 consists of 529 amino acids with a single transmembrane domain (TMD). Atg32 is located in the mitochondrial outer membrane (OM) with the N- and C-terminal domains oriented towards the cytosol and the mitochondrial intermembrane space (IMS), respectively. The N-terminal cytosolic region of Atg32 contains the Atg8-family interacting motif (AIM), which interacts with Atg8 and an Atg11-interacting region.

have a D-D/E-S-S<sup>phos</sup>-I-I/M-S-T-S/P sequence (Fig. 3A). This sequence is also important for Atg11 interaction. However, there is only some similarity between these two consensus sequences, such as the I/V-L/I/M-S motif. Further studies are required, especially of molecular structure, to understand the phosphorylation-mediated receptor–Atg11 interaction.

### 2.5. Atg32–Atg8 interaction

Atg8 is a ubiquitin-like protein that is conjugated to the lipid phosphatidylethanolamine (PE), which localizes to the isolation membrane [78]. The Atg8-family interacting motif (AIM) or the LC3-interacting region (LIR) is a WXXL-like sequence (W/Y-X-X-L/I/V) and is found in most of the adaptor proteins for selective autophagy from yeast to mammals [79]. As shown in Fig. 3B, the receptor proteins Atg32, Atg34, Atg36, PpAtg30, and PpAtg32 have an AIM/LIR. Similarly, in mammals, the mitophagy receptor, Nix, and the autophagic receptors for ubiquitinated proteins, p62 and NBR1, have this motif (Fig. 3B). Atg32 possesses an AIM/LIR in its cytosolic region and interacts with Atg8 (Fig. 2) [59,70]. An AIM/LIR is not essential for mitophagy because mutation of the AIM/LIR in Atg32 (from WQAI to AQAA) only partially affects efficiency of mitophagy [70]. Interestingly, recent reports have shown that isolation membrane-integrated, PE-conjugated Atg8 (Atg8–PE) recruits Atg12–Atg5 to the membrane, and then Atg16 crosslinks Atg8–PE–Atg12–Atg5 complexes to form a continuous two-dimensional membrane scaffold with meshwork-like architecture [80,81]. The Atg32–Atg8 interaction may facilitate formation of autophagosomes surrounding mitochondria by connecting mitochondrial Atg32 to the autophagosome concave face, where Atg8–PE–Atg12–Atg5–Atg16 form a meshwork-like architecture. In addition, a recent report showed that phosphorylation of serine or threonine residues located immediately upstream of the AIM/LIR of Atg32 or PpAtg32 is required for the Atg32–Atg8 interaction [73]. This finding suggests that the Atg32–Atg8 interaction is also regulated by phosphorylation.

### 2.6. Mitochondrial fission and the mitochondria–ER contact site are related to mitophagy

Mitochondrial fusion and fission frequently occur, and the shape and size of mitochondria depend on the balance between them [82]. Because the size of mitochondria degraded by mitophagy should be smaller than that of autophagosomes, mitochondrial fission machinery may be essential for mitophagy to make mitochondria smaller than autophagosomes. In fact, several studies have suggested that mitochondrial fission is related to efficiency of mitophagy in mammalian cells [35, 51,83–86]. However, the role of mitochondrial fission in mitophagy is still controversial in yeast. Dnm1, Fis1, Mdv1, and Caf4 are known factors for mitochondrial fission in yeast [87–91]. Mao et al. and Abeliovich et al. showed that *dnm1*-, *fis1*- , and *mdv1/caf4*-deleted strains significantly suppressed mitophagy [92,93]. Mao et al. further analyzed the role for Dnm1 in mitophagy. When mitophagy is induced, Atg11 recruits the mitochondrial fission machinery by the Atg11–Dnm1 interaction, and the recruited fission machinery divides mitochondria to facilitate mitophagy. The C-terminal GTPase effector domain of Dnm1 is required for the Atg11–Dnm1 interaction. Dnm1 with a mutation in this domain does not affect mitochondrial fission, but loses its interaction with Atg11. The Dnm1 mutant cannot induce mitophagy efficiently, suggesting that Atg11 recruits the fission machinery to the mitophagy site, and the fission machinery divides the mitochondria for mitophagic cargo (Fig. 4) [92]. However, a genome-wide screen for mitophagy-deficient mutants did not identify mitochondrial fission machinery as essential factors for mitophagy, although only the *dnm1* mutant was identified to slightly suppress mitophagy [59,64]. In addition, Mendl et al. reported that all of the mitochondrial fission machinery (Dnm1, Fis1, Mdv1, and Caf4) is not required for rapamycin-induced mitophagy [94]. If mitochondrial fission machinery is not required for mitophagy, there may be another system that makes mitochondria smaller than autophagosomes, or mitophagy occurs by microautophagy. Further study is required to clarify the relationship between mitochondrial fission and mitophagy.

---

**A**

| S. cerevisiae Atg32 | 107: TTNG-SILS SSDTSEEE |
| P. pastoris Atg32    | 151: TGNGESVLSSSEEY-EET |
| P. angusta Atg32     | 134: PNIG-SILS SSDN-EER |
| P. pastoris Atg30    | 104: NYNEDDIILS SSRRSSED |
| S. cerevisiae Atg36  | 89: PSDEIAILS ISSDSNKN |
| S. cerevisiae Atg19  | 386: ELDDSS IISTSISLSYD |
| S. cerevisiae Atg34  | 378: SPDESS IMSTPFLGETD |

**B**

| S. cerevisiae Atg32 | 83: SGSWQATQPL |
| P. pastoris Atg32   | 118: VTAWQMVHRD |
| P. pastoris Atg30   | 70: NSEWILFSPE |
| S. cerevisiae Atg36 | 279: WFGWTLISR |
| S. cerevisiae Atg19 | 410: ALTWEEL |
| S. cerevisiae Atg34 | 406: PFTWEEI |
| Human Nix           | 33: NSSWVELPMN |
| Human NBR1          | 729: SEDYIIILPE |
| Human p62           | 335: DDDWTHLSSK |

Fig. 3. Sequence alignment of the Atg11-interacting region and the Atg8-family interacting motif (AIM). (A) Sequence alignment of the Atg11-interacting region of the indicated selective autophagy receptor proteins. (B) Sequence alignment of the AIM of the indicated proteins.

---

Please cite this article as: T. Kanki, et al., Mitophagy in yeast: Molecular mechanisms and physiological role, Biochim. Biophys. Acta (2015), http://dx.doi.org/10.1016/j.bbamcr.2015.01.005

ARTICLE IN PRESS

T. Kanki et al. / Biochimica et Biophysica Acta xxx (2015) xxx–xxx

![Diagram](attachment:diagram.png)

Fig. 4. Mitochondrial fission and the mitochondria–endoplasmic reticulum contact site play a role in mitophagy. When mitophagy is induced, the Atg32–Atg11 interaction marks mitochondria destined for degradation, and the fission complex (Dnm1 and Fis1) is recruited through an interaction between Dnm1 and Atg11. The ER–mitochondria encounter structure (ERMES) complex also localizes on degrading mitochondria and facilitates mitophagy, presumably supplying lipids from the endoplasmic reticulum (ER) for extension of the isolation membrane.

ER–mitochondria contact is mediated by the ER–mitochondria encounter structure (ERMES), which is composed of Mmm1, Mdm10, Mdm12, and Mdm34 [95]. Bockler and Westermann recently reported that deletion of one of the ERMES components severely suppressed mitophagy. Because ERMES colocalizes with Atg5, Atg8, Atg9, Atg14, and Atg32 when mitophagy is induced, this research group speculated that the ER–mitochondria contact site is important to supply lipids from the ER for autophagosome biogenesis for mitophagy [96]. Mitochondrial fission might occur at the ER–mitochondria contact site [97]. Mao et al. found that the ERMES components Mdm12 and Mdm34 colocalize with Atg11 and Atg32 when mitophagy is induced [92]. This finding suggests that ERMES plays a role in mitochondrial fission to facilitate mitophagy. The importance of ER–mitochondria contact in mitophagy has recently been suggested in mammalian cells. Yang and Yang reported that impaired mitochondria by photo-inactivation are sequestered by isolation membranes or omegasomes, which form from ER–mitochondria contact sites in HeLa cells [98]. The relationships among mitochondrial fission, ERMES, and mitophagy in yeast are summarized in Fig. 4.

### 2.7. Other mitophagy-related factors

Aup1, Whi2, and Atg33 have been reported as related to mitophagy. Whi2 functions in the general stress response and the Ras-protein kinase A signaling pathway [99]. Mendl et al. showed that deletion of WHI2 affects mitophagy and proposed that mitophagy is linked to the general stress response and the Ras-protein kinase A signaling pathway [94]. However, Mao et al. reported that deletion of WHI2 did not affect mitophagy that was induced by nitrogen starvation [92]. Therefore, the role of Whi2 in mitophagy is still controversial. Aup1, which was identified by a screen for protein phosphatase homologs that interact with the serine/threonine kinase Atg1, is thought to be required for efficient mitophagy to survive in prolonged stationary phase culture in a medium containing lactate as the sole carbon source [82]. Interestingly, deletion of RTG3, a transcription factor that mediates the retrograde signaling pathway, causes a defect in stationary phase mitophagy, and deletion of AUP1 leads to alterations in the patterns of Rtg3 phosphorylation under these conditions [100]. These findings suggest that the function of Aup1 in mitophagy may be regulation of Rtg3-dependent transcription. However, how the mitochondrial intermembrane space protein Aup1 affects cytosolic Rtg3 phosphorylation is still unclear. Atg33 is a mitochondrial outer membrane protein that was identified by a genome-wide screen for mitophagy [64]. Deletion of Atg33 partially suppresses mitophagy induced by rapamycin or at the post-log phase [64,101]. However, the role of Atg33 in mitophagy has not yet been identified.

## 3. Regulation of mitophagy

### 3.1. Induction of mitophagy in yeast

Mitophagy was originally thought to be rarely induced in *S. cerevisiae* under common culture conditions used by most laboratories. To induce mitophagy, cells were pre-cultured in a nonfermentable medium that facilitates the proliferation of mitochondria (e.g., where lactate or glycerol is the sole carbon source) and then exposed to nitrogen starvation or the Tor inhibitor rapamycin [70,62]. Mitophagy is also induced during the stationary phase when yeast cells are cultured in a nonfermentable medium [60–62]. However, mitophagy was recently reported to be induced when cells are pre-cultured in a fermentable medium and then exposed to nitrogen starvation [96,102], although fewer mitochondria are degraded by mitophagy than when the cells are cultured in a nonfermentable medium. Interestingly, mitophagy is easily induced in *P. pastoris* compared with in *S. cerevisiae*. *S. cerevisiae* is a fermenting yeast and has fewer mitochondria available for degradation during fermentation. *P. pastoris* is a low-fermenting yeast that always has a certain number of mitochondria, independent of the type of carbon source available in the medium. Therefore, mitophagy can be efficiently induced when cells are pre-cultured, even in a fermentable medium, and then exposed to nitrogen starvation in *P. pastoris* [72].

In mammalian cells, there is accumulating evidence that dysfunctional (or depolarized) mitochondria are degraded by mitophagy [34, 35]. An example of this situation is that an uncoupler, such as carbonyl cyanide m-chlorophenylhydrazone (CCCP) or the herbicide paraquat,

Please cite this article as: T. Kanki, et al., Mitophagy in yeast: Molecular mechanisms and physiological role, Biochim. Biophys. Acta (2015), http://dx.doi.org/10.1016/j.bbamcr.2015.01.005

ARTICLE IN PRESS

T. Kanki et al. / Biochimica et Biophysica Acta xxx (2015) xxx–xxx

efficiently induces mitophagy [34,36,51]. In contrast, in yeast, CCCP or paraquat only slightly induces mitophagy [58,60].

### 3.2. Kinases related to mitophagy

Two mitogen-activated protein kinases (MAPKs), Hog1 and Slt2, are related to mitophagy in yeast (Fig. 1) [69,103]. A genome-wide yeast mutant screen for mitophagy showed that the MAPK kinase kinase Bck1 is required for mitophagy [64]. Bck1 is involved in the cell wall integrity signaling pathway, which includes Pkc1, Bck1, redundant Mkk1/2, and Slt2 kinases [104]. Mao et al. examined the relationship between mitophagy and the Slt2 signaling pathway and found that the Pkc1–Bck1–Mkk1/2–Slit2 signaling pathway is required for mitophagy [103]. Although there are six plasma membrane or ER membrane sensors upstream of the Slit2 signaling pathway—Wsc1, Wsc2, Wsc3, Wsc4, Mid2, and Mtl1 [104]—Wsc1 is predominantly involved in induction of mitophagy [103].

Hog1 is a kinase integral to the osmoregulatory signal transduction cascade (HOG signaling pathway) [105]. Mao et al. and Aoki et al. reported that Hog1 is required for mitophagy [69,103]. Mao et al. analyzed the upstream components of the HOG signaling pathway to determine if it was related to mitophagy. They found that Sln1 (a sensor in the plasma membrane), Ssk1 (a cytoplasmic response regulator downstream of Sln1), and Pbs2 (a MAPK kinase just upstream of Hog1) were related to mitophagy [103]. Aoki et al. identified Hog1 as a phosphorylator of Atg32-related kinase. Phosphorylation of Atg32 is an essential step that mediates the Atg32–Atg11 interaction and mitophagy. A *HOG1*-deleted strain suppressed Atg32 phosphorylation and could not induce mitophagy efficiently [69]. Although Hog1 is required for Atg32 phosphorylation, this kinase cannot phosphorylate Atg32 in vitro, suggesting that Hog1 is not a kinase that directly phosphorylates Atg32 [69].

Casein kinase 2 (CK2) was recently reported to be essential for Atg32 phosphorylation [106]. In a CK2 temperature-sensitive mutant strain, or a CK2 specific inhibitor (4,5,6,7-tetrabromo-benzotriazole)-treated strain, phosphorylation of Atg32 is suppressed and, as a result, the Atg32–Atg11 interaction and mitophagy are suppressed. In vitro, CK2 can phosphorylate recombinant wild-type Atg32, but not a Ser-114/119-to-Ala mutant [106]. These findings suggest that CK2 directly phosphorylates Atg32 to induce mitophagy. Because Hog1 is also related indirectly to Atg32 phosphorylation, the Hog1 signaling pathway should be upstream of CK2 and regulate Atg32 phosphorylation. However, how Hog1 regulates Atg32 phosphorylation by CK2 has not been clarified.

### 3.3. Regulation of Atg32 expression and processing

A recent study reported that Atg32 expression was suppressed by Tor and the Ume6–Sin3–Rpd3 complex (Fig. 5) [72]. In *P. pastoris*, PpAtg32 is barely expressed under normal culture conditions but is expressed after nitrogen starvation. In PpSin3- or PpRpd3-deleted cells, PpAtg32 is efficiently expressed without nitrogen starvation. Chromatin immunoprecipitation assays show an interaction of PpSin3 with the promoter of *PpATG32*. Deletion of 301–350 bp upstream of the *PpATG32* open reading frame induces PpAtg32 expression without nitrogen starvation. This suggests that the deleted promoter region of *PpATG32* contains the upstream repression sequence where PpSin3 and PpRpd3 interact to inhibit *PpATG32* transcription. Inhibition of Tor by rapamycin induces PpAtg32 expression, but results in neither PpAtg32 phosphorylation nor induction of mitophagy [72]. These findings suggest that PpAtg32 expression is suppressed by Tor and the downstream PpSin3–PpRpd3 complex, and that PpAtg32 expression and phosphorylation are regulated independently. Similarly, in *S. cerevisiae*, Atg32 expression is suppressed by Tor and the downstream Ume6–Sin3–Rpd3 complex [72]. Cooperation of Atg32 expression and phosphorylation is important for mitophagy, because when Atg32 expression is suppressed, mitophagy cannot be efficiently induced, even if Atg32 is phosphorylated [72].

Cellular oxidative stress is thought to positively affect mitophagy (Fig. 5). Deffieu et al. observed that N-acetylcysteine and cysteine, which increase cellular levels of reduced glutathione, prevented mitophagy [107]. Okamoto et al. reported that N-acetylcysteine suppressed the expression of Atg32 and as a result, inhibited mitophagy [59]. These findings suggest that Atg32 expression and mitophagy are affected by cellular oxidative conditions. Currently, the relationships among cellular oxidative conditions and Tor and the downstream Ume6–Sin3–Rpd3 complex are unclear.

Yme1 is a catalytic subunit of i-AAA protease, which is localized in the mitochondrial inner membrane. Yme1 processes the C-terminal IMS domain of Atg32. The *yme1Δ* or protease inactive *yme1*<sup>E541Q</sup> strain significantly blocks mitophagy and the Atg32–Atg11 interaction, suggesting that the processing of Atg32 by Yme1 positively affects efficiency of mitophagy (Fig. 5) [71]. However, Welter et al. reported that the *yme1Δ* strain shows up-regulation of mitophagy [101]. In addition, deletion of the C-terminal IMS region of Atg32 marginally affects efficiency of mitophagy [69,70]. Therefore, the role of Yme1 in mitophagy is still controversial.

### 4. Physiological relevance of mitophagy in yeast

The current belief is that mitophagy eliminates dysfunctional mitochondria and contributes to mitochondrial quality control. However, there is limited evidence to support this idea in yeast. Fmc1 is a mitochondrial matrix protein that is required for assembly or stability of the F<sub>1</sub> sector of mitochondrial F<sub>o</sub>F<sub>1</sub> ATP synthase [108]. In the temperature sensitive *fmc1* mutant, the F<sub>o</sub>F<sub>1</sub> ATPase subunits aggregate and the mitochondrial membrane potential is impaired at 37 °C. Autophagy is

![Figure 5](attachment:fig5.png)

**Fig. 5.** Regulation of Atg32 expression. Transcription of *ATG32* is suppressed by the Ume6–Sin3–Rpd3 complex, which interacts with the upstream repression sequence (URS) of the *ATG32* promoter region. Inhibition of Tor releases the Ume6–Sin3–Rpd3 complex and *ATG32* can be transcribed. Oxidative stress can also induce expression of Atg32. Expressed Atg32 is targeted to the mitochondrial outer membrane. When mitophagy is induced, the C-terminal intermembrane space (IMS) domain of Atg32 is processed by Yme1 to facilitate the Atg32–Atg11 interaction and mitophagy.

---

Please cite this article as: T. Kanki, et al., Mitophagy in yeast: Molecular mechanisms and physiological role, Biochim. Biophys. Acta (2015), http://dx.doi.org/10.1016/j.bbamcr.2015.01.005
induced in this mutant and mitochondria are eliminated at 37 °C [109]. Mdm38 is a mitochondrial inner membrane protein that is involved in membrane insertion of mitochondrially synthesized proteins [110, 111]. Mdm38 is also thought to be required for K⁺/H⁺ and/or Ca2⁺/ H⁺ exchange activity of the mitochondrial inner membrane [112,113]. In the *mdm38Δ* mutant, mitochondria lose their membrane potential, appear swollen, and are eventually eliminated by autophagy [57]. Although these findings suggest that dysfunctional mitochondria are eliminated by autophagy, they cannot provide direct evidence that mitophagy contributes to mitochondrial quality control. In addition, the mitophagy-deficient *atg32Δ* strain does not show any phenotype regarding cell growth, viability, or mitochondrial function under normal culture conditions [58,59].

Kurihara et al. found that mitophagy is important for the regulation of mitochondrial volume to minimize ROS production from mitochondria, and as a result, maintains the quality of mitochondria in *S. cerevisiae* [33]. The number of mitochondria changes depending on metabolism in *S. cerevisiae*. When cells that contain a substantial number of mitochondria are exposed to cellular stresses, such as nitrogen starvation, they eliminate mitochondria, which produce unwanted ROS. In the mitophagy-deficient *atg32Δ* or *atg11Δ* strains, cells cannot adequately eliminate mitochondria and the undegraded mitochondria produce excess ROS. More specifically, when wild-type cells are precultured in nonfermentable medium and encounter nitrogen starvation, they induce mitophagy and quickly eliminate mitochondria that have proliferated during respiratory growth. As a result, cellular ROS production, which is mainly caused by mitochondria, is suppressed. However, in mitophagy-deficient *atg32Δ* or *atg11Δ* cells, undegraded mitochondria produce excess ROS. ROS damage mitochondria, and damaged mitochondria produce further ROS, finally leading to mitochondrial DNA deletion. By this vicious cycle, severe mtDNA damage occurs and cells show a “petite” phenotype (Fig. 6) [33]. A similar finding was observed

in the proteasome-inactivated quiescent state of the fission yeast Schizosaccharomyces pombe [114]. Inactivation of the proteasome results in an accumulation of cellular ROS. Mitophagy contributes to preventing accumulation of ROS by eliminating mitochondria [114]. These findings suggest that mitophagy is required to regulate the number of mitochondria to minimize ROS production, and as a result, maintains the quality of mitochondria and cellular homeostasis.

A recent report showed that some mitochondrial matrix proteins undergo mitophagy with differential kinetics in *S. cerevisiae* [93]. This finding suggests that the physiological role of mitophagy is not limited to regulating mitochondrial volume en masse and reducing ROS production, but it also maintains mitochondrial quality by preferentially degrading specific components of the mitochondrial matrix.

Richard et al. found that mitophagy plays a pivotal role in longevity extension by a caloric restriction diet in chronologically aging yeast [115]. They found that under caloric restriction conditions in mitophagy-deficient cells, mitochondrial function is decreased, as shown by abnormal mitochondrial morphology, decreased cellular O₂ consumption, decreased mitochondrial inner membrane potential, increased cytosolic cytochrome C release from mitochondria, and reduced cellular ATP levels. This mitochondrial dysfunction is in part due to disability of oxidative phosphorylation complexes and increased mitochondrial ROS production. They also found that mitophagy is required for maintaining the homeostasis of membrane lipids in mitochondria, the endoplasmic reticulum, and the plasma membrane [115]. These findings suggest that mitophagy is important for maintaining mitochondrial quality and cellular lipid composition under caloric restriction.

### 5. Perspectives

In this review, we describe recent progress in studies on mitophagy in yeast. Identification of the mitophagy receptor protein, Atg32, has

![Diagram](attachment:diagram.png)

**Fig. 6.** Physiological importance of mitophagy in yeast. When cells encounter cellular stress, such as nitrogen starvation, mitophagy eliminates mitochondria and reduces reactive oxygen species (ROS) production in wild-type cells. In mitophagy-deficient cells, the remaining mitochondria produce excess ROS, which in turn can damage mitochondria. Damaged mitochondria produce more ROS. Eventually, ROS-damaged mitochondrial DNA (mtDNA) and cells with damaged mtDNA show the petite phenotype. Green mitochondria, intact; red mitochondria, damaged.

---

Please cite this article as: T. Kanki, et al., Mitophagy in yeast: Molecular mechanisms and physiological role, Biochim. Biophys. Acta (2015), http://dx.doi.org/10.1016/j.bbamcr.2015.01.005
ARTICLE IN PRESS

T. Kanki et al. / Biochimica et Biophysica Acta xxx (2015) xxx–xxx

revealed the molecular mechanisms and regulatory processes of mitophagy. However, the following issues remain unknown. 1) Phosphorylation of Atg32 is mediated by CK2. However, it is unclear how CK2, which has constitutive kinase activity, phosphorylates Atg32 only when mitophagy is induced. Because the Hog1 signaling pathway is required for Atg32 phosphorylation, phosphorylation of Atg32 by CK2 might be regulated downstream of CK2. Similarly, phosphorylation of the receptor protein for the Cvt pathway, Atg19, and pexophagy, Atg36, are phosphorylated by Hrr25, which is also constitutively active. How the receptor proteins of selective autophagy are regulated needs to be understood. The signaling pathway for the MAP kinase Slt2 is also important for mitophagy. However, the mitophagy-related molecular processes downstream of Slt2 are unknown. Recently, MAPK1 (ERK2) and MAPK14 (p38α) signaling pathways were reported to be required for mitophagy in mammalian cells [116]. The relationship between MAP kinases and mitophagy needs to be studied to understand the regulation of mitophagy in yeast and mammals. 2) Atg32 expression is suppressed by Tor and the downstream Ume6–Sin3–Rpd3 complex. However, Ume6-, Sin3-, or Rpd3-deleted cells do not express Atg32 at a comparable level to starved cells [72]. Presumably, the expression of Atg32 is also regulated by levels of transcription factors or rates of translation.

The largest open question about mitophagy in yeast may be whether mitophagy can select damaged or dysfunctional mitochondria as a cargo. This question is directly related to the physiological importance of mitophagy in yeast. In mammalian cells, there is accumulating evidence that dysfunctional (or depolarized) mitochondria are degraded by mitophagy [34,35]. In yeast, there is no doubt that mitophagy selectively degrades mitochondria. However, to the best of our knowledge, there is no experimental evidence that mitophagy selects damaged, but not intact, mitochondria for degradation. Identification of how mitophagy could distinguish damaged from intact mitochondria is important, if such a system exists in yeast.

Because mitophagy is evolutionarily conserved, we speculate that there are similar molecular processes, regulatory mechanisms, and physiological roles in yeast and mammals. The interplay of yeast and mammalian mitophagy studies will consolidate our understanding of this cellular process.

### Transparency document

The **Transparency document** associated with this article can be found, in the online version.

### Acknowledgements

This work was supported in part by the Japanese Ministry of Education, Culture, Sports, Science and Technology (MEXT) program “Promotion of Environmental Improvement for Independence of Young Researchers” under the Special Coordination Funds for Promoting Science and Technology; JSPS KAKENHI (grant nos. 25117714, 25560414, and 26291039); and a Grant for Basic Science Research Projects from The Sumitomo Foundation.

### References

[1] N. Mizushima, A. Yamamoto, M. Hatano, Y. Kobayashi, Y. Kabeya, K. Suzuki, T. Tokuhisa, Y. Ohsumi, T. Yoshimori, Dissection of autophagosome formation using Apg5-deficient mouse embryonic stem cells, J. Cell Biol. 152 (2001) 657–668.

[2] Z. Yang, D.J. Klionsky, Eaten alive: a history of macroautophagy, Nat. Cell Biol. 12 (2010) 814–822.

[3] T.O. Berg, M. Fengsrud, P.E. Stromhaug, T. Berg, P.O. Seglen, Isolation and characterization of rat liver amphisomes. Evidence for fusion of autophagosomes with both early and late endosomes, J. Biol. Chem. 273 (1998) 21883–21892.

[4] W.A. Dunn Jr., Studies on the mechanisms of autophagy: formation of the autophagic vacuole, J. Cell Biol. 110 (1990) 1923–1933.

[5] H. Nakatogawa, K. Suzuki, Y. Kamada, Y. Ohsumi, Dynamics and diversity in autophagy mechanisms: lessons from yeast, Nat. Rev. Mol. Cell Biol. 10 (2009) 458–467.

[6] B. Levine, N. Mizushima, H.W. Virgin, Autophagy in immunity and inflammation, Nature 469 (2011) 323–335.

[7] N. Mizushima, B. Levine, A.M. Cuervo, D.J. Klionsky, Autophagy fights disease through cellular self-digestion, Nature 451 (2008) 1069–1075.

[8] A.M. Choi, S.W. Ryter, B. Levine, Autophagy in human health and disease, N. Engl. J. Med. 368 (2013) 651–662.

[9] D.C. Rubinsztein, The roles of intracellular protein-degradation pathways in neurodegeneration, Nature 443 (2006) 780–786.

[10] K. Suzuki, Selective autophagy in budding yeast, Cell Death Differ. 20 (2013) 43–48.

[11] J.J. Lemasters, Selective mitochondrial autophagy, or mitophagy, as a targeted defense against oxidative stress, mitochondrial dysfunction, and aging, Rejuvenation Res. 8 (2005) 3–5.

[12] S.V. Scott, D.J. Klionsky, Delivery of proteins and organelles to the vacuole from the cytoplasm, Curr. Opin. Cell Biol. 10 (1998) 523–529.

[13] C. Kraft, A. Deplazes, M. Sohrmann, M. Peter, Mature ribosomes are selectively degraded upon starvation by an autophagy pathway requiring the Ubp3p/Bre5p ubiquitin protease, Nat. Cell Biol. 10 (2008) 602–610.

[14] S. Bernales, S. Schuck, P. Walter, ER-phagy: selective autophagy of the endoplasmic reticulum, Autophagy 3 (2007) 285–287.

[15] R. Singh, S. Kaushik, Y. Wang, Y. Xiang, I. Novak, M. Komatsu, K. Tanaka, A.M. Cuervo, M.J. Czaja, Autophagy regulates lipid metabolism, Nature 458 (2009) 1131–1135.

[16] W.X. Ding, H.M. Ni, W. Gao, T. Yoshimori, D.B. Stolz, D. Ron, X.M. Yin, Linking of autophagy to ubiquitin–proteasome system is important for the regulation of endoplasmic reticulum stress and cell viability, Am. J. Pathol. 171 (2007) 513–524.

[17] M. Komatsu, S. Waguri, M. Koike, Y.S. Sou, T. Ueno, T. Hara, N. Mizushima, J. Iwata, J. Ezaki, S. Murata, J. Hamazaki, Y. Nishito, S. Iemura, T. Natsume, T. Yanagawa, J. Uwayama, E. Warabi, H. Yoshida, T. Ishii, A. Kobayashi, M. Yamamoto, Z. Yue, Y. Uchiyama, E. Kominami, K. Tanaka, Homeostatic levels of p62 control cytoplasmic inclusion body formation in autophagy-deficient mice, Cell 131 (2007) 1149–1163.

[18] B. Levine, Eating oneself and uninvited guests: autophagy-related pathways in cellular defense, Cell 120 (2005) 159–162.

[19] T.M. Harding, K.A. Morano, S.V. Scott, D.J. Klionsky, Isolation and characterization of yeast mutants in the cytoplasm to vacuole protein targeting pathway, J. Cell Biol. 131 (1995) 591–602.

[20] M. Tsukada, Y. Ohsumi, Isolation and characterization of autophagy-defective mutants of *Saccharomyces cerevisiae*, FEBS Lett. 333 (1993) 169–174.

[21] M. Thumm, R. Egner, B. Koch, M. Schlumpberger, M. Straub, M. Veenhuis, D.H. Wolf, Isolation of autophagocytosis mutants of *Saccharomyces cerevisiae*, FEBS Lett. 349 (1994) 275–280.

[22] T.M. Harding, A. Hefner-Gravink, M. Thumm, D.J. Klionsky, Genetic and phenotypic overlap between autophagy and the cytoplasm to vacuole protein targeting pathway, J. Biol. Chem. 271 (1996) 17621–17624.

[23] W. Yuan, D.L. Tuttle, Y.J. Shi, G.S. Ralph, W.A. Dunn Jr., Glucose-induced microautophagy in *Pichia pastoris* requires the alpha-subunit of phosphofructokinase, J. Cell Sci. 110 (Pt 16) (1997) 1935–1945.

[24] Y. Sakai, A. Koller, L.K. Rangell, G.A. Keller, S. Subramani, Peroxisome degradation by microautophagy in *Pichia pastoris*: identification of specific steps and morphological intermediates, J. Cell Biol. 141 (1998) 625–636.

[25] D.J. Klionsky, J.M. Cregg, W.A. Dunn Jr., S.D. Emr, Y. Sakai, I.V. Sandoval, A. Sibirny, S. Subramani, M. Thumm, M. Veenhuis, Y. Ohsumi, A unified nomenclature for yeast autophagy-related genes, Dev. Cell 5 (2003) 539–545.

[26] D.C. Wallace, A mitochondrial paradigm of metabolic and degenerative diseases, aging, and cancer: a dawn for evolutionary medicine, Annu. Rev. Genet. 39 (2005) 359–407.

[27] D.A. Bota, K.J. Davies, Lon protease preferentially degrades oxidized mitochondrial aconitase by an ATP-stimulated mechanism, Nat. Cell Biol. 4 (2002) 674–680.

[28] K. Luce, H.D. Osiewacz, Increasing organismal healthspan by enhancing mitochondrial protein quality control, Nat. Cell Biol. 11 (2009) 852–858.

[29] C.M. Haynes, Y. Yang, S.P. Blais, T.A. Neubert, D. Ron, The matrix peptide exporter HAF-1 signals a mitochondrial UPR by activating the transcription factor ZC376.7 in *C. elegans*, Mol. Cell 37 (2010) 529–540.

[30] M. Muftuoglu, M.P. Mori, N.C. de Souza-Pinto, Formation and repair of oxidative damage in the mitochondrial DNA, Mitochondrion 17 (2014) 164–181.

[31] M. Alexeyev, I. Shokolenko, G. Wilson, S. LeDoux, The maintenance of mitochondrial DNA integrity—critical analysis and update, Cold Spring Harb. Perspect. Biol. 5 (2013) a012641.

[32] R. Brigelius-Flohe, M. Maiorino, Glutathione peroxidases, Biochim. Biophys. Acta 1830 (2013) 3289–3303.

[33] Y. Kurihara, T. Kanki, Y. Aoki, Y. Hirota, T. Saigusa, T. Uchiumi, D. Kang, Mitophagy plays an essential role in reducing mitochondrial production of reactive oxygen species and mutation of mitochondrial DNA by maintaining mitochondrial quantity and quality in yeast, J. Biol. Chem. 287 (2012) 3265–3272.

[34] D. Narendra, A. Tanaka, D.F. Suen, R.J. Youle, Parkin is recruited selectively to impaired mitochondria and promotes their autophagy, J. Cell Biol. 183 (2008) 795–803.

[35] G. Twig, A. Elorza, A.J. Molina, H. Mohamed, J.D. Wikstrom, G. Walzer, L. Stiles, S.E. Haigh, S. Katz, G. Las, J. Alroy, M. Wu, B.F. Py, J. Yuan, J.T. Deeney, B.E. Corkey, O.S. Shirihai, Fission and selective fusion govern mitochondrial segregation and elimination by autophagy, EMBO J. 27 (2008) 433–446.

[36] D.P. Narendra, S.M. Jin, A. Tanaka, D.F. Suen, C.A. Gautier, J. Shen, M.R. Cookson, R.J. Youle, PINK1 is selectively stabilized on impaired mitochondria to activate Parkin, PLoS Biol. 8 (2010) e1000298.

---

Please cite this article as: T. Kanki, et al., Mitophagy in yeast: Molecular mechanisms and physiological role, Biochim. Biophys. Acta (2015), http://dx.doi.org/10.1016/j.bbamcr.2015.01.005

[37] I. Novak, V. Kirkin, D.G. McEwan, J. Zhang, P. Wild, A. Rozenknop, V. Rogov, F. Lohr, D. Popovic, A. Occhipinti, A.S. Reichert, J. Terzic, V. Dotsch, P.A. Ney, I. Dikic, Nix is a selective autophagy receptor for mitochondrial clearance, EMBO Rep. 11 (2010) 45–51.

[38] H. Sandoval, P. Thiagarajan, S.K. Dasgupta, A. Schumacher, J.T. Prchal, M. Chen, J. Wang, Essential role for Nix in autophagic maturation of erythroid cells, Nature 454 (2008) 232–235.

[39] R.L. Schweers, J. Zhang, M.S. Randall, M.R. Loyd, W. Li, F.C. Dorsey, M. Kundu, J.T. Opferman, J.L. Cleveland, J.L. Miller, P.A. Ney, NIX is required for programmed mitochondrial clearance during reticulocyte maturation, Proc. Natl. Acad. Sci. U. S. A. 104 (2007) 19500–19505.

[40] W. Wu, W. Tian, Z. Hu, G. Chen, L. Huang, W. Li, X. Zhang, P. Xue, C. Zhou, L. Liu, Y. Zhu, X. Zhang, L. Li, L. Zhang, S. Sui, B. Zhao, D. Feng, ULK1 translocates to mitochondria and phosphorylates FUNDC1 to regulate mitophagy, EMBO Rep. 15 (2014) 566–575.

[41] C. Vives-Bauza, C. Zhou, Y. Huang, M. Cui, R.L. de Vries, J. Kim, J. May, M.A. Tocilescu, W. Liu, H.S. Ko, J. Magrane, D.J. Moore, V.L. Dawson, R. Grailhe, T.M. Dawson, C. Li, K. Tieu, S. Przedborski, PINK1-dependent recruitment of Parkin to mitochondria in mitophagy, Proc. Natl. Acad. Sci. U. S. A. 107 (2010) 378–383.

[42] L. Liu, D. Feng, G. Chen, M. Chen, Q. Zheng, P. Song, Q. Ma, C. Zhu, R. Wang, W. Qi, L. Huang, P. Xue, B. Li, X. Wang, H. Jin, J. Wang, F. Yang, P. Liu, Y. Zhu, S. Sui, Q. Chen, Mitochondrial outer-membrane protein FUNDC1 mediates hypoxia-induced mitophagy in mammalian cells, Nat. Cell Biol. 14 (2012) 177–185.

[43] Y.C. Wong, E.L. Holzbaur, Optineurin is an autophagy receptor for damaged mitochondria in parkin-mediated mitophagy that is disrupted by an ALS-linked mutation, Proc. Natl. Acad. Sci. U. S. A. 111 (2014) E4439–E4448.

[44] T. Kitada, S. Asakawa, N. Hattori, H. Matsumine, Y. Yamamura, S. Minoshima, M. Yokochi, Y. Mizuno, N. Shimizu, Mutations in the parkin gene cause autosomal recessive juvenile parkinsonism, Nature 392 (1998) 605–608.

[45] E.M. Valente, S. Salvi, T. Ialongo, R. Marongiu, A.E. Elia, V. Caputo, L. Romito, A. Albanese, B. Dallapiccola, A.R. Bentivoglio, PINK1 mutations are associated with sporadic early-onset parkinsonism, Ann. Neurol. 56 (2004) 336–341.

[46] R.E. Thomas, L.A. Andrews, J.L. Burman, W.Y. Lin, L.J. Pallanck, PINK1-Parkin pathway activity is regulated by degradation of PINK1 in the mitochondrial matrix, Plos Genet. 10 (2014) e1004279.

[47] K. Yamano, R.J. Youle, PINK1 is degraded through the N-end rule pathway, Autophagy 9 (2013) 1758–1769.

[48] N. Matsuda, S. Sato, K. Shiba, K. Okatsu, K. Saisho, C.A. Gautier, Y.S. Sou, S. Saiki, S. Kawajiri, F. Sato, M. Kimura, M. Komatsu, N. Hattori, K. Tanaka, PINK1 stabilized by mitochondrial depolarization recruits Parkin to damaged mitochondria and activates latent Parkin for mitophagy, J. Cell Biol. 189 (2010) 211–221.

[49] E. Ziviani, R.N. Tao, A.J. Whitworth, Drosophila parkin requires PINK1 for mitochondrial translocation and ubiquitinates mitofusin, Proc. Natl. Acad. Sci. U. S. A. 107 (2010) 5018–5023.

[50] A.C. Poole, R.E. Thomas, S. Yu, E.S. Vincow, L. Pallanck, The mitochondrial fusion-promoting factor mitofusin is a substrate of the PINK1/Parkin pathway, Plos ONE 5 (2010).

[51] A. Tanaka, M.M. Cleland, S. Xu, D.P. Narendra, D.F. Suen, M. Karbowski, R.J. Youle, Proteasome and p97 mediate mitophagy and degradation of mitofusins induced by Parkin, J. Cell Biol. 191 (2010) 1367–1380.

[52] M.E. Gegg, J.M. Cooper, K.Y. Chau, M. Rojo, A.H.V. Schapira, J.W. Taanman, Mitofusin 1 and mitofusin 2 are ubiquitinated in a PINK1/parkin-dependent manner upon induction of mitophagy, Hum. Mol. Genet. 19 (2010) 4861–4870.

[53] A. Rakovic, A. Grunewald, J. Kottwitz, N. Bruggemann, P.P. Pramstaller, K. Lohmann, C. Klein, Mutations in PINK1 and Parkin impair ubiquitination of mitofusins in human fibroblasts, Plos ONE 6 (2011) e16746.

[54] D. Mijaljica, M. Prescott, R.J. Devenish, Microautophagy in mammalian cells: revisiting a 40-year-old conundrum, Autophagy 7 (2011) 673–682.

[55] S. Kaushik, A.M. Cuervo, Chaperone-mediated autophagy: a unique way to enter the lysosome world, Trends Cell Biol. 22 (2012) 407–417.

[56] I. Kissova, B. Salin, J. Schaeffer, S. Bhatia, S. Manon, N. Camougrand, Selective and non-selective autophagic degradation of mitochondria in yeast, Autophagy 3 (2007) 329–336.

[57] K. Nowikovsky, S. Reipert, R.J. Devenish, R.J. Schweyen, Mdm38 protein depletion causes loss of mitochondrial K+/H+ exchange activity, osmotic swelling and mitophagy, Cell Death Differ. 14 (2007) 1647–1656.

[58] T. Kanki, K. Wang, Y. Cao, M. Baba, D.J. Klionsky, Atg32 is a mitochondrial protein that confers selectivity during mitophagy, Dev. Cell 17 (2009) 98–109.

[59] K. Okamoto, N. Kondo-Okamoto, Y. Ohsumi, Mitochondria-anchored receptor Atg32 mediates degradation of mitochondria via selective autophagy, Dev. Cell 17 (2009) 87–97.

[60] I. Kissova, M. Deffieu, S. Manon, N. Camougrand, Uth1p is involved in the autophagic degradation of mitochondria, J. Biol. Chem. 279 (2004) 39068–39074.

[61] T. Kanki, D.J. Klionsky, Mitophagy in yeast occurs through a selective mechanism, J. Biol. Chem. 283 (2008) 32386–32393.

[62] R. Tal, G. Winter, N. Ecker, D.J. Klionsky, H. Abeliovich, Aup1p, a yeast mitochondrial protein phosphatase homolog, is required for efficient stationary phase mitophagy and cell survival, J. Biol. Chem. 282 (2007) 5617–5624.

[63] Y. Zhang, H. Qi, R. Taylor, W. Xu, L.F. Liu, S. Jin, The role of autophagy in mitochondria maintenance: characterization of mitochondrial functions in autophagy-deficient S. cerevisiae strains, Autophagy 3 (2007) 337–346.

[64] T. Kanki, K. Wang, M. Baba, C.R. Bartholomew, M.A. Lynch-Day, Z. Du, J. Geng, K. Mao, Z. Yang, W.L. Yen, D.J. Klionsky, A genomic screen for yeast mutants defective in selective mitochondria autophagy, Mol. Biol. Cell 20 (2009) 4730–4738.

[65] J. Kim, Y. Kamada, P.E. Stromhaug, J. Guan, A. Hefner-Gravink, M. Baba, S.V. Scott, Y. Ohsumi, W.A. Dunn Jr., D.J. Klionsky, Cvt9/Gsa9 functions in sequestering selective cytosolic cargo destined for the vacuole, J. Cell Biol. 153 (2001) 381–396.

[66] T. Shintani, W.P. Huang, P.E. Stromhaug, D.J. Klionsky, Mechanism of cargo selection in the cytoplasm to vacuole targeting pathway, Dev. Cell 3 (2002) 825–837.

[67] J.C. Farre, R. Manjithaya, R.D. Mathewson, S. Subramani, PpAtg30 tags peroxisomes for turnover by selective autophagy, Dev. Cell 14 (2008) 365–376.

[68] A.M. Motley, J.M. Nuttall, E.H. Hettema, Pex3-anchored Atg36 tags peroxisomes for degradation in Saccharomyces cerevisiae, EMBO J. 31 (2012) 2852–2868.

[69] Y. Aoki, T. Kanki, Y. Hirota, Y. Kurihara, T. Saigusa, T. Uchiumi, D. Kang, Phosphorylation of Serine 114 on Atg32 mediates mitophagy, Mol. Biol. Cell 22 (2011) 3206–3217.

[70] N. Kondo-Okamoto, N.N. Noda, S.W. Suzuki, H. Nakatogawa, I. Takahashi, M. Matsunami, A. Hashimoto, F. Inagaki, Y. Ohsumi, K. Okamoto, Autophagy-related protein 32 acts as autophagic degron and directly initiates mitophagy, J. Biol. Chem. 287 (2012) 10631–10638.

[71] K. Wang, M. Jin, X. Liu, D.J. Klionsky, Proteolytic processing of Atg32 by the mitochondrial i-AAA protease Yme1 regulates mitophagy, Autophagy 9 (2013) 1828–1836.

[72] M. Aihara, X. Jin, Y. Kurihara, Y. Yoshida, Y. Matsushima, M. Oku, Y. Hirota, T. Saigusa, Y. Aoki, T. Uchiumi, T. Yamamoto, Y. Sakai, D. Kang, T. Kanki, Tor and the Sin3-Rpd3 complex regulate expression of the mitophagy receptor protein Atg32 in yeast, J. Cell Sci. 127 (2014) 3184–3196.

[73] J.C. Farre, A. Burkenroad, S.F. Burnett, S. Subramani, Phosphorylation of mitophagy and pexophagy receptors coordinates their interaction with Atg8 and Atg11, EMBO Rep. 14 (2013) 441–449.

[74] T. Pfaffenwimmer, W. Reiter, T. Brach, V. Nogellova, D. Papinski, M. Schuschnig, C. Abert, G. Ammerer, S. Martens, C. Kraft, Hrr25 kinase promotes selective autophagy by phosphorylating the cargo receptor Atg19, EMBO Rep. 15 (2014) 862–870.

[75] C. Tanaka, L.J. Tan, K. Mochida, H. Kirisako, M. Koizumi, E. Asai, M. Sakoh-Nakatogawa, Y. Ohsumi, H. Nakatogawa, Hrr25 triggers selective autophagy-related pathways by phosphorylating receptor proteins, J. Cell Biol. 207 (2014) 91–105.

[76] K. Suzuki, C. Kondo, M. Morimoto, Y. Ohsumi, Selective transport of alpha-mannosidase by autophagic pathways: identification of a novel receptor, Atg34p, J. Biol. Chem. 285 (2010) 30019–30025.

[77] K. Mochida, Y. Ohsumi, H. Nakatogawa, Hrr25 phosphorylates the autophagic receptor Atg34 to promote vacuolar transport of alpha-mannosidase under nitrogen starvation conditions, FEBS Lett. 588 (2014) 3862–3869.

[78] Y. Ichimura, T. Kirisako, T. Takao, Y. Satomi, Y. Shimonishi, N. Ishihara, N. Mizushima, I. Tanida, E. Kominami, M. Ohsumi, T. Noda, Y. Ohsumi, A ubiquitin-like system mediates protein lipidation, Nature 408 (2000) 488–492.

[79] N.N. Noda, Y. Ohsumi, F. Inagaki, Atg8-family interacting motif crucial for selective autophagy, FEBS Lett. 584 (2010) 1379–1385.

[80] A. Kaufmann, T. Wollert, Scaffolding the expansion of autophagosomes, Autophagy 10 (2014) 1343–1345.

[81] A. Kaufmann, V. Beier, H.G. Franquelim, T. Wollert, Molecular mechanism of autophagic membrane-scaffold assembly and disassembly, Cell 156 (2014) 469–481.

[82] K. Okamoto, J.M. Shaw, Mitochondrial morphology and dynamics in yeast and multicellular eukaryotes, Annu. Rev. Genet. 39 (2005) 503–536.

[83] L.C. Gomes, G. Di Benedetto, L. Scorrano, During autophagy mitochondria elongate, are spared from degradation and sustain cell viability, Nat. Cell Biol. 13 (2011) 589–598.

[84] A.S. Rambold, B. Kostelecky, N. Elia, J. Lippincott-Schwartz, Tubular network formation protects mitochondria from autophagosomal degradation during nutrient starvation, Proc. Natl. Acad. Sci. U. S. A. 108 (2011) 10190–10195.

[85] Y. Ikeda, A. Shirakabe, Y. Maejima, P. Zhai, S. Sciarretta, J. Toli, M. Nomura, K. Mihara, K. Egashira, M. Ohishi, M. Abdellatif, J. Sadoshima, Endogenous Drp1 mediates mitochondrial autophagy and protects the heart against energy stress, Circ. Res. 116 (2015) 264–278.

[86] Y. Kageyama, M. Hoshijima, K. Seo, D. Bedja, P. Sysa-Shah, S.A. Andrabi, W. Chen, A. Hoke, V.L. Dawson, T.M. Dawson, K. Gabrielson, D.A. Kass, M. Iijima, H. Sesaki, Parkin-independent mitophagy requires Drp1 and maintains the integrity of mammalian heart and brain, EMBO J. 33 (2014) 2798–2813.

[87] D. Otsuga, B.R. Keegan, E. Brisch, J.W. Thatcher, G.J. Hermann, W. Bleazard, J.M. Shaw, The dynamin-related GTPase Dnm1p controls mitochondrial morphology in yeast, J. Cell Biol. 143 (1998) 333–349.

[88] W. Bleazard, J.M. McCaffery, E.J. King, S. Bale, A. Mozdy, Q. Tieu, J. Nunnari, J.M. Shaw, The dynamin-related GTPase Dnm1 regulates mitochondrial fission in yeast, Nat. Cell Biol. 1 (1999) 298–304.

[89] A.D. Mozdy, J.M. McCaffery, J.M. Shaw, Dnm1p GTPase-mediated mitochondrial fission is a multi-step process requiring the novel integral membrane component Fis1p, J. Cell Biol. 151 (2000) 367–380.

[90] Q. Tieu, J. Nunnari, Mdv1p is a WD repeat protein that interacts with the dynamin-related GTPase, Dnm1p, to trigger mitochondrial division, J. Cell Biol. 151 (2000) 353–366.

[91] E.E. Griffin, J. Graumann, D.C. Chan, The WD40 protein Caf4p is a component of the mitochondrial fission machinery and recruits Dnm1p to mitochondria, J. Cell Biol. 170 (2005) 237–248.

[92] K. Mao, K. Wang, X. Liu, D.J. Klionsky, The scaffold protein Atg11 recruits fission machinery to drive selective mitochondria degradation by autophagy, Dev. Cell 26 (2013) 9–18.

[93] H. Abeliovich, M. Zarei, K.T. Rigbolt, R.J. Youle, J. Dengjel, Involvement of mitochondrial dynamics in the segregation of mitochondrial matrix proteins during stationary phase mitophagy, Nat. Commun. 4 (2013) 2789.

[94] N. Mendl, A. Occhipinti, M. Muller, P. Wild, I. Dikic, A.S. Reichert, Mitophagy in yeast is independent of mitochondrial fission and requires the stress response gene WHI2, J. Cell Sci. 124 (2011) 1339–1350.

[95] B. Kornmann, E. Currie, S.R. Collins, M. Schuldiner, J. Nunnari, J.S. Weissman, P. Walter, An ER-mitochondria tethering complex revealed by a synthetic biology screen, Science 325 (2009) 477–481.

[96] S. Bockler, B. Westermann, Mitochondrial ER contacts are crucial for mitophagy in yeast, Dev. Cell 28 (2014) 450–458.

[97] J.R. Friedman, L.L. Lackner, M. West, J.R. DiBenedetto, J. Nunnari, G.K. Voeltz, ER tubules mark sites of mitochondrial division, Science 334 (2011) 358–362.

[98] J.Y. Yang, W.Y. Yang, Bit-by-bit autophagic removal of parkin-labelled mitochondria, Nat. Commun. 4 (2013) 2428.

[99] J.E. Leadsham, K. Miller, K.R. Ayscough, S. Colombo, E. Martegani, P. Sudbery, C.W. Gourlay, Whi2p links nutritional sensing to actin-dependent Ras-cAMP-PKA regulation and apoptosis in yeast, J. Cell Sci. 122 (2009) 706–715.

[100] D. Journo, A. Mor, H. Abeliovich, Aup1-mediated regulation of Rtg3 during mitophagy, J. Biol. Chem. 284 (2009) 35885–35895.

[101] E. Welter, M. Montino, R. Reinhold, P. Schlotterhose, R. Krick, J. Dudek, P. Rehling, M. Thumm, Uth1 is a mitochondrial inner membrane protein dispensable for post-log-phase and rapamycin-induced mitophagy, FEBS J. 280 (2013) 4970–4982.

[102] A. Eiyama, N. Kondo-Okamoto, K. Okamoto, Mitochondrial degradation during starvation is selective and temporally distinct from bulk autophagy in yeast, FEBS Lett. 587 (2013) 1787–1792.

[103] K. Mao, K. Wang, M. Zhao, T. Xu, D.J. Klionsky, Two MAPK-signaling pathways are required for mitophagy in Saccharomyces cerevisiae, J. Cell Biol. 193 (2011) 755–767.

[104] D.E. Levin, Cell wall integrity signaling in Saccharomyces cerevisiae, Microbiol. Mol. Biol. Rev. 69 (2005) 262–291.

[105] B.T. Seet, T. Pawson, MAPK signaling: Sho business, Curr. Biol. 14 (2004) R708–R710.

[106] T. Kanki, Y. Kurihara, X. Jin, T. Goda, Y. Ono, M. Aihara, Y. Hirota, T. Saigusa, Y. Aoki, T. Uchiumi, D. Kang, Casein kinase 2 is essential for mitophagy, EMBO Rep. 14 (2013) 788–794.

[107] M. Deffieu, I. Bhatia-Kissova, B. Salin, A. Galinier, S. Manon, N. Camougrand, Glutathione participates in the regulation of mitophagy in yeast, J. Biol. Chem. 284 (2009) 14828–14837.

[108] L. Lefebvre-Legendre, J. Vaillier, H. Benabdelhak, J. Velours, P.P. Slonimski, J.P. di Rago, Identification of a nuclear gene (FMC1) required for the assembly/stability of yeast mitochondrial F(1)-ATPase in heat stress conditions, J. Biol. Chem. 276 (2001) 6789–6796.

[109] M. Priault, B. Salin, J. Schaeffer, F.M. Vallette, J.P. di Rago, J.C. Martinou, Impairing the bioenergetic status and the biogenesis of mitochondria triggers mitophagy in yeast, Cell Death Differ. 12 (2005) 1613–1621.

[110] D. Lupo, C. Vollmer, M. Deckers, D.U. Mick, I. Tews, I. Sinning, P. Rehling, Mdm38 is a 14-3-3-like receptor and associates with the protein synthesis machinery at the inner mitochondrial membrane, Traffic 12 (2011) 1457–1466.

[111] A.E. Frazier, R.D. Taylor, D.U. Mick, B. Warscheid, N. Stoepel, H.E. Meyer, M.T. Ryan, B. Guiard, P. Rehling, Mdm38 interacts with ribosomes and is a component of the mitochondrial protein export machinery, J. Cell Biol. 172 (2006) 553–564.

[112] E. Froschauer, K. Nowikovsky, R.J. Schwewen, Electroneutral K⁺/H⁺ exchange in mitochondrial membrane vesicles involves Yol027/Letm1 proteins, Biochim. Biophys. Acta 1711 (2005) 41–48.

[113] D.W. Jiang, L.L. Zhao, D.E. Clapham, Genome-wide RNAi screen identifies Letm1 as a mitochondrial Ca²⁺/H⁺ antiporter, Science 326 (2009) 144–147.

[114] K. Takeda, T. Yoshida, S. Kikuchi, K. Nagao, A. Kokubu, T. Pluskal, A. Villar-Briones, T. Nakamura, M. Yanagida, Synergistic roles of the proteasome and autophagy for mitochondrial maintenance and chronological lifespan in fission yeast, Proc. Natl. Acad. Sci. U. S. A. 107 (2010) 3540–3545.

[115] V.R. Richard, A. Leonov, A. Beach, M.T. Burstein, O. Koupaki, A. Gomez-Perez, S. Levy, L. Pluska, S. Mattie, R. Rafesh, T. Iouk, S. Sheibani, M. Greenwood, H. Vali, V.I. Titorenko, Macromitophagy is a longevity assurance process that in chronologically aging yeast limited in calorie supply sustains functional mitochondria and maintains cellular lipid homeostasis, Aging 5 (2013) 234–269.

[116] Y. Hirota, S. Yamashita, Y. Kurihara, X. Jin, M. Aihara, T. Saigusa, D. Kang, T. Kanki, Mitophagy is primarily due to alternative autophagy and requires the MAPK1 and MAPK14 signaling pathways, Autophagy (2015) (in press).

[117] C.W. Wang, J. Kim, W.P. Huang, H. Abeliovich, P.E. Stromhaug, W.A. Dunn Jr., D.J. Klionsky, Apg2 is a novel protein required for the cytoplasm to vacuole targeting, autophagy, and pexophagy pathways, J. Biol. Chem. 276 (2001) 30442–30451.

[118] J. Kim, V.M. Dalton, K.P. Eggerton, S.V. Scott, D.J. Klionsky, Apg7p/Cvt2p is required for the cytoplasm-to-vacuole targeting, macroautophagy, and peroxisome degradation pathways, Mol. Biol. Cell 10 (1999) 1337–1351.

[119] K.A. Tucker, F. Reggiori, W.A. Dunn Jr., D.J. Klionsky, Atg23 is essential for the cytoplasm to vacuole targeting pathway and efficient autophagy but not pexophagy, J. Biol. Chem. 278 (2003) 48445–48452.

[120] U.D. Epple, E.L. Eskelinen, M. Thumm, Intravacuolar membrane lysis in Saccharomyces cerevisiae. Does vacuolar targeting of Cvt17/Aut5p affect its function? J. Biol. Chem. 278 (2003) 7810–7821.

[121] U.D. Epple, I. Suriapranata, E.L. Eskelinen, M. Thumm, Aut5/Cvt17p, a putative lipase essential for disintegration of autophagic bodies inside the vacuole, J. Bacteriol. 183 (2001) 5942–5955.

[122] S.A. Teter, K.P. Eggerton, S.V. Scott, J. Kim, A.M. Fischer, D.J. Klionsky, Degradation of lipid vesicles in the yeast vacuole requires function of Cvt17, a putative lipase, J. Biol. Chem. 276 (2001) 2083–2087.

[123] H. Cheong, T. Yorimitsu, F. Reggiori, J.E. Legakis, C.W. Wang, D.J. Klionsky, Atg17 regulates the magnitude of the autophagic response, Mol. Biol. Cell 16 (2005) 3438–3453.

[124] Y. Kabeya, Y. Kamada, M. Baba, H. Takikawa, M. Sasaki, Y. Ohsumi, Atg17 functions in cooperation with Atg1 and Atg13 in yeast autophagy, Mol. Biol. Cell 16 (2005) 2544–2553.

[125] Y. Kamada, T. Funakoshi, T. Shintani, K. Nagano, M. Ohsumi, Y. Ohsumi, Tor-mediated induction of autophagy via an Apg1 protein kinase complex, J. Cell Biol. 150 (2000) 1507–1513.

[126] H. Barth, K. Meiling-Wesse, U.D. Epple, M. Thumm, Autophagy and the cytoplasm to vacuole targeting pathway both require Aut10p, FEBS Lett. 508 (2001) 23–28.

[127] J. Guan, P.E. Stromhaug, M.D. George, P. Habibzadegh-Tari, A. Bevan, W.A. Dunn Jr., D.J. Klionsky, Cvt18/Gsa12 is required for cytoplasm-to-vacuole transport, pexophagy, and autophagy in Saccharomyces cerevisiae and Pichia pastoris, Mol. Biol. Cell 12 (2001) 3821–3838.

[128] S.V. Scott, J. Guan, M.U. Hutchins, J. Kim, D.J. Klionsky, Cvt19 is a receptor for the cytoplasm-to-vacuole targeting pathway, Mol. Cell 7 (2001) 1131–1141.

[129] D.C. Nice, T.K. Sato, P.E. Stromhaug, S.D. Emr, D.J. Klionsky, Cooperative binding of the cytoplasm to vacuole targeting pathway proteins, Cvt13 and Cvt20, to phosphatidylinositol 3-phosphate at the pre-autophagosomal structure is required for selective autophagy, J. Biol. Chem. 277 (2002) 30198–30207.

[130] H. Barth, K. Meiling-Wesse, U.D. Epple, M. Thumm, Ma1p is essential for maturation of proaminopeptidase I but not for autophagy, FEBS Lett. 512 (2002) 173–179.

[131] Z. Yang, J. Huang, J. Geng, U. Nair, D.J. Klionsky, Atg22 recycles amino acids to link the degradative and recycling functions of autophagy, Mol. Biol. Cell 17 (2006) 5094–5104.

[132] Y. Cao, D.J. Klionsky, Atg26 is not involved in autophagy-related pathways in Saccharomyces cerevisiae, Autophagy 3 (2007) 17–20.

[133] A.E. Wurmser, S.D. Emr, Novel PtdIns(3)P-binding protein Etf1 functions as an effector of the Vps34 PtdIns 3-kinase in autophagy, J. Cell Biol. 158 (2002) 761–772.

[134] W.L. Yen, J.E. Legakis, U. Nair, D.J. Klionsky, Atg27 is required for autophagy-dependent cycling of Atg9, Mol. Biol. Cell 18 (2007) 581–593.

[135] T. Kawamata, Y. Kamada, K. Suzuki, N. Kuboshima, H. Akimatsu, S. Ota, M. Ohsumi, Y. Ohsumi, Characterization of a novel autophagy-specific gene, ATG29, Biochem. Biophys. Res. Commun. 338 (2005) 1884–1889.

[136] Y. Kabeya, T. Kawamata, K. Suzuki, Y. Ohsumi, Cis1/Atg31 is required for autophagosome formation in Saccharomyces cerevisiae, Biochem. Biophys. Res. Commun. 356 (2007) 405–410.

[137] Y. Araki, W.C. Ku, M. Akioka, A.I. May, Y. Hayashi, F. Arisaka, Y. Ishihama, Y. Ohsumi, Atg38 is required for autophagy-specific phosphatidylinositol 3-kinase complex integrity, J. Cell Biol. 203 (2013) 299–313.
