
Neuron
Review

The Roles of PINK1, Parkin, and Mitochondrial Fidelity in Parkinson’s Disease

Alicia M. Pickrell¹ and Richard J. Youle¹,*  
¹Biochemistry Section, Surgical Neurology Branch, National Institute of Neurological Disorders and Stroke (NINDS), NIH, Bethesda, MD 20892, USA  
*Correspondence: youler@ninds.nih.gov  
http://dx.doi.org/10.1016/j.neuron.2014.12.007

Understanding the function of genes mutated in hereditary forms of Parkinson’s disease yields insight into disease etiology and reveals new pathways in cell biology. Although mutations or variants in many genes increase the susceptibility to Parkinson’s disease, only a handful of monogenic causes of parkinsonism have been identified. Biochemical and genetic studies reveal that the products of two genes that are mutated in autosomal recessive parkinsonism, PINK1 and Parkin, normally work together in the same pathway to govern mitochondrial quality control, bolstering previous evidence that mitochondrial damage is involved in Parkinson’s disease. PINK1 accumulates on the outer membrane of damaged mitochondria, activates Parkin’s E3 ubiquitin ligase activity, and recruits Parkin to the dysfunctional mitochondrion. Then, Parkin ubiquitinates outer mitochondrial membrane proteins to trigger selective autophagy. This review covers the normal functions that PINK1 and Parkin play within cells, their molecular mechanisms of action, and the pathophysiological consequences of their loss.

Parkinson’s disease (PD) affects 1%–2% of the population and is the second most common neurodegenerative disease (de Rijk et al., 1995). Four cardinal signs characterize PD, rigidity, bradykinesia, postural instability, and tremor (Lang and Lozano, 1998a, 1998b), that stem from the progressive loss of dopaminergic neurons in the substantia nigra pars compacta (SN). Motor symptoms appear when approximately 50%–60% of these neurons degenerate, causing a 70%–80% depletion of dopamine (DA) levels in the dorsal striatum (Lang and Lozano, 1998a, 1998b). The remaining SN neurons typically contain inclusions in the cytoplasm called Lewy bodies that immunostain for α-synuclein and other proteins (Spillantini et al., 1998). There is no cure, and the best treatment options only provide symptomatic relief with no abatement or reversal of disease progression. Although a large majority of diagnosed PD cases are idiopathic, autosomal dominant and recessive familial forms have been identified. By understanding how monogenic forms of PD lead to cell dysfunction and neuron death, researchers are identifying causes of parkinsonism and new pathways in cell biology.

**PARK2 (Parkin) and PARK6 (PINK1) Identification**

In 1997, a genetic linkage analysis showed that chromosome 6q25.2-27 harbored an unidentified gene responsible for autosomal recessive juvenile parkinsonism (AR-JP) in 13 Japanese families (Matsumine et al., 1997). One year later, the Shimizu group cloned the AR-JP gene, PARK2, and identified more Japanese AR-JP patients with either an exon 4 or a large-scale deletion between exons 3 and 7 (Kitada et al., 1998). Other patients of various ethnicities with early-onset PD were soon reported to also harbor PARK2 mutations with varying deletions or point mutations that cause PARK2 protein loss of function (Hattori et al., 1998a, 1998b; Leroy et al., 1998; Lücking et al.,

PARK2 contains 12 exons that encode the 465 amino acid protein, Parkin (Kitada et al., 1998). Parkin is an E3 ubiquitin ligase with an amino-terminal ubiquitin-like (Ubl) domain and a carboxyl-terminal ubiquitin ligase domain (Hristova et al., 2009; Shimura et al., 2000).

The second gene to be identified in early-onset recessive PD cases was found in 2001 within a large Italian pedigree on chromosome 1 at the PARK6 locus (Valente et al., 2001, 2002b). These PARK6 mutation patients had symptoms clinically identical to those of patients with sporadic forms of PD (Bentivoglio et al., 2001; Valente et al., 2002a). The 8 exon PARK6 gene encodes the 581 amino acid protein phosphatase and tensin homolog (PTEN)-induced kinase 1 (PINK1) (Valente et al., 2004), a name stemming from its prior identification in a screen for proteins transcriptionally upregulated by exogenous PTEN overexpression (Unoki and Nakamura, 2001). The protein sequence reveals a predicted C-terminal kinase domain and a mitochondrial targeting sequence at the N terminus, suggesting that it is imported into the mitochondria, which is consistent with its mitochondrial localization in cells (Valente et al., 2004). This mitochondrial localization supported prior evidence of an involvement of mitochondrial dysfunction in the pathophysiology of PD.

**Mitochondrial Dysfunction in PD**

Evidence linking mitochondrial dysfunction to PD arose in the late 1970s when accidental exposure to 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine (MPTP), a contaminant from the synthesis of 1-methyl-4-phenyl-4-propionoxy-piperidine (MPPP) (a drug used for illicit purposes), was found to cause parkinsonism and DA neurodegeneration (Langston et al., 1983). Follow-up studies found that MPTP oxidized to MPP+, which is selectively taken up into DA neurons causing an inhibition of I,

Model of Parkin-induced mitophagy

and possibly in man (Betarbet et al., 2000; Liou et al., 1997; Tanner et al., 2011). These early observations indicated that DA neurons appear particularly sensitive to mitochondrial dysfunction.

Pathogenic mtDNA mutations also are associated with PD. MtDNA is a 16,569-base-pair-length genome that encodes 13 genes for subunit components of OXPHOS and its own tRNAs and rRNAs (Anderson et al., 1981). As hundreds to thousands of copies of mtDNA reside in virtually each mammalian cell, a state of heteroplasmy arises when different mtDNA genotypes, such as wild-type and mutant forms, coexist within the same cell. SN neurons from autopsies of normal-aged people and PD patients harbor high levels of mutated mtDNA with large-scale deletions that cause mitochondrial dysfunction (Bender et al., 2006; Kraytsberg et al., 2006). Furthermore, mitochondrial disease patients with mutations in polymerase γ, the polymerase responsible for mtDNA replication, excessively accumulate mtDNA mutations and also have an increased risk of developing PD (Luoma et al., 2004; Reeve et al., 2013). The many links between mitochondrial dysfunction and the pathogenesis of PD stimulated interest in the role PINK1 plays in that organelle.

### PINK1 and Parkin Mediate Mitophagy

Although PINK1 localizes to mitochondria and Parkin resides in the cytosol, genetic epistasis analyses in *Drosophila* revealed that both proteins work in the same pathway (Clark et al., 2006; Park et al., 2006) and maintain mitochondrial fidelity (Greene et al., 2003). Cell biology studies then revealed that Parkin is recruited from the cytosol to depolarized mitochondria to mediate the selective autophagic removal of the damaged organelle (mitophagy) (Narendra et al., 2008). PINK1 accumulates on dysfunctional mitochondria, and its kinase activity is required for Parkin translocation to mitochondria and mitophagy, linking the function of these autosomal recessive PARK gene products to the same biochemical pathway (Geisler et al., 2010a; Matsuda et al., 2010; Narendra et al., 2010b; Vives-Bauza et al., 2010). These findings have led to the unraveling of an intriguing process, by which PINK1 detects mitochondrial dysfunction and then signals Parkin to ubiquitinate specifically the damaged mitochondria to instigate their removal by autophagy. This suggests that, acting together, PINK1 and Parkin constitute a mitochondrial quality control function (Figure 1). Further supporting the model that mitochondrial quality control is the essential function of PINK1 and Parkin contributing to the prevention of parkinsonism in man, patient mutations in both PINK1 (Geisler et al., 2010b; Narendra et al., 2010b) and Parkin (Geisler et al., 2010a; Lee et al., 2010; Matsuda et al., 2010; Narendra et al., 2010b) prevent PINK1 recruitment of Parkin to mitochondria (Figure 2 and Table 1). These results also offer further evidence that mitochondrial dysfunction may play a role in sporadic PD (Figure 3). Recent advances have revealed the molecular mechanisms of PINK1 sensing mitochondrial damage and the activation of Parkin.

### PINK1 Detects Mitochondrial Damage via Selective Proteolysis

PINK1 accumulates specifically on damaged mitochondria flagging them for elimination. The damage-sensing mechanism

Figure 1. Model of Parkin-Induced Mitophagy

Dysfunctional mitochondria (yellow mitochondrion) fail to import and degrade PINK1 stabilizing it on the outer mitochondrial membrane (OMM). After PINK1 accumulation, PINK1 phosphorylates ubiquitin and Parkin to activate Parkin’s E3 ligase activity. Parkin ubiquitinates substrates on the outer mitochondria for two divergent processes: autophagosome recruitment and ubiquitin proteasome degradation of ubiquitinated mitochondrial substrates. Fis1 is a receptor on the outer membrane that binds two proteins, TBC1D15 and TBC1D17, to govern the developing LC3 isolation membrane to generate the autophagosome around the damaged mitochondria. The autophagosome is then delivered to the lysosome for degradation.

a mitochondrial respiratory chain component of the oxidative phosphorylation (OXPHOS) machinery (Javitch et al., 1985; Nicklas et al., 1985; Ramsay and Singer, 1986). Complex I deficiency also was found in the brain, skeletal muscle, and platelets of sporadic PD patients (Schapira et al., 1989, 1990). Pesticides and herbicides that selectively inhibit complex I, such as rotenone and paraquat, also cause parkinsonism in animal models

258 Neuron 85, January 21, 2015 ©2015 Elsevier Inc.

PINK1
MTS
Protein Kinase Domain

Parkin
UBL
RING 0
RING 1
IBR
RING 2

V3I/W54R
A31D
R33X
Q33Q
Q34fsX39/A38VfsX44
Q34RfsX39/A38VfsX44
Q34RfsX39/A38VfsX44
N52MfsX29
V56E/C212Y
W74CfsX8
G96C/C1305T
P113TfsX51/Exon3del40bp
K211N/R275W
K211N/Exon2,3dup
C212fsX224/R275N/
C212Y
C238AfsX21/Exon3del
C238W/Exon6,7del
T240R
T240M/255Adel
T240M
C253Y/Exon2,4del
C253Y
C268X/202-204AGdel
R275W/Exon3,4del
R275W/Exon3,4del
R275W/G430D
R275W/G437L
D280N/R334C
C284R/Exon1del
C289G
A291LfsX7
Q311X
C323_V324del Exon2del
V324AfsX111
R334C/G444Q
R334C
Q392SfsX3
E395X/Exon9dup
N428KfsX141/Exon6del
C441R/Exon3del
C441R/Exon4del
C441R/Exon4del
P437L/Exon12del
D460GfsX109/Exon7del

* M1L
* S10N
* G12R
* V15M
* D18N
* G32R
* P52L
* R67F
* R68P
* A78V
* R98W
* K186N
* G193R
* P196L
* P209A
* A124V
* T145M
* R152W
* V170G
* G227R
* E231G
* N235I
* M237V
* Q239X/R492X
* E240K
* A246X
* R246X
* H271Q
* Y258X
* E240KL489P
* A244GV317V
* R263G
* L268V
* R246N
* T257I
* A280T
* P296L
* R276Q
* R278V
* R279H
* G309D
* T313M
* D297MfsX22
* R256C
* V258M
* H265R
* Y267H
* N273S
* R275W
* L347P
* N367S/K520RfsX3
* L369PR492X
* G386A
* C388R
* V317L
* M318L
* P322L
* A339T
* M341L
* M342V
* D362H
* A383T
* F385L
* G395V
* P399L
* R407Q
* G409V
* P416R
* E417G
* S419P
* T415N
* G411S
* R334H
* R334C
* R349SfsX20
* R351P
* R359D
* R366W
* R366Q
* W437X
* G440E
* Q456X
* E476K
* R396G
* R398T
* C418R
* G429E
* G430D
* C431F
* P432C
* P433L
* P437L
* P439L
* P442T
* N451S
* V482M
* P498L
* D525N
* A537T
* N542S
* C575R

Figure 2. Cartoon Depiction of the Location of Mutations in Monogenic PINK1 and Parkin PD Patients
*, heterozygous mutation; ins, insertion; /, compound heterozygous; fs, frameshift; dup, duplication; TM, transmembrane domain; MTS, mitochondrial targeting sequence; UBL, ubiquitin-like domain; RING, really interesting new gene domain; IBR, in between RING domain.

Table 1. Summary of PARK2- and PARK6-KO Models in Comparison to Monogenic PD Patients

| Gene Inactivated | Species     | Age of Onset                | DA Neurotransmission Defect                  | Motor Phenotype                          | SN Neuron Loss   | Lewy Body Formation | Reference                     |
|------------------|-------------|----------------------------|----------------------------------------------|------------------------------------------|------------------|---------------------|------------------------------|
| PARK2            | human       | juvenile<br>< 20 years;<br>early onset<br>20–40 years | DA depletion in striatum                 | cardinal signs                           | progressive      | absent or present   | Matsumine et al., 1997;<br>Kitada et al., 1998;<br>Pramstaller et al., 2005 |
| PARK6            | human       | juvenile<br>< 20 years;<br>early onset<br>20–40 years | DA depletion in striatum                 | cardinal signs                           | progressive      | absent or present   | Valente et al., 2001;<br>Samaranch et al., 2010 |
| PARK2            | Drosophila  | NA                         | absent                                      | flight wing degeneration,<br>motor coordination deficits | absent           | unknown             | Greene et al., 2003         |
| PARK2            | Drosophila  | NA                         | absent                                      | flight wing degeneration,<br>motor coordination deficits | absent           | unknown             | Pesah et al., 2004          |
| PARK2            | Drosophila  | 1 day (DA neuron loss<br>progressive) | unknown                                    | flight wing degeneration,<br>motor coordination deficits | DL region        | unknown             | Whitworth et al., 2005      |
| PARK2            | Drosophila  | NA                         | DA depletion                               | flight wing degeneration,<br>motor coordination deficits | absent           | unknown             | Cha et al., 2005            |
| PARK6            | Drosophila  | NA                         | absent                                      | flight wing degeneration,<br>motor coordination deficits | absent           | unknown             | Clark et al., 2006          |
| PARK6            | Drosophila  | 30 days                    | decreased TH<sup>+</sup> expression          | flight wing degeneration,<br>motor coordination deficits | DM and DL regions | unknown             | Park et al., 2006           |
| PARK6            | Drosophila  | 25 days                    | DA depletion                               | flight wing degeneration,<br>motor coordination deficits | DM and DL regions | unknown             | Yang et al., 2006           |
| PARK2            | mouse       | NA                         | increased extracellular DA release         | beam slip motor deficit                 | absent           | unknown             | Goldberg et al., 2003       |
| PARK2            | mouse       | NA                         | decreased DA uptake                        | motor coordination deficits             | absent           | unknown             | Itier et al., 2003          |
| PARK2            | mouse       | NA                         | absent                                      | startle response deficit                | absent           | unknown             | von Coelln et al., 2004     |
| PARK2            | mouse       | NA                         | absent                                      | absent                                   | absent           | unknown             | Perez and Palminter, 2005   |
| PARK2            | mouse       | conditional Cre-loxP       | unknown                                    | unknown                                  | yes, lentiviral-Cre infection | unknown             | Shin et al., 2011          |
| PARK6            | mouse       | NA                         | evoked DA release deficit                  | unknown                                  | absent           | unknown             | Kitada et al., 2007         |
| PARK6            | mouse       | NA                         | DA depletion                               | unknown                                  | absent           | unknown             | Akundi et al., 2011         |

DL, dorsolateral; DM, dorsomedial; NA, not applicable.

unfolded protein expressed in the mitochondrial matrix was found to stabilize PINK1 and activate Parkin without depolarizing the inner mitochondrial membrane (Jin and Youle, 2013). If expression of the LON protease was diminished, the levels of PINK1 increased even further (Jin and Youle, 2013). The authors concluded that loss of LON, which is known to help degrade misfolded mitochondrial matrix proteins, increased misfolded protein accumulation and thereby increased PINK1 accumulation. How misfolded proteins cause PINK1 accumulation without mitochondrial depolarization is yet unresolved, although it is possible that misfolded proteins somehow inhibit TIM complex import of PINK1 (Figure 4). Another study showed that LON-mutant flies accumulate PINK1, although a different interpretation of the mechanism was reached (Thomas et al., 2014). The

Drosophila phenotype suggested that LON directly degraded PINK1 in the matrix compartment and that loss of LON prevented this degradation. However, this model is hard to reconcile with the finding that PARL-processed PINK1 is degraded mainly in the cytosol by the N-end rule (Yamano and Youle, 2013). Nevertheless, in both flies and mammals it appears that, upon mitochondrial damage, PINK1 becomes stabilized on the OMM, and from there it recruits Parkin to mitochondria and activates Parkin’s E3 ubiquitin ligase activity.

**PINK1 Is a Ubiquitin Kinase**

The substrate of PINK1 kinase activity mediating Parkin activation and recruitment to mitochondria was predicted to be in the cytosol and not a mitochondrial protein, because ectopic

Neuron

Review

---

**Healthy**

![Image Description: Healthy mitochondria on the left, damaged mitochondria on the right with arrows indicating "Mitochondrial damage" and "Removal of damaged mitochondria".]

**Parkinson’s disease**

![Image Description: Damaged mitochondria on the left, healthy mitochondria on the right with arrows indicating "Mitochondrial damage" and "Removal of damaged mitochondria".]

**Parkinson’s disease**

![Image Description: Healthy mitochondria on the left, damaged mitochondria on the right with arrows indicating "Mitochondrial damage" and "Removal of damaged mitochondria".]

---

**Figure 3. The Balance between Mitochondrial Damage and Its Removal Contributes to PD Pathogenesis**

Healthy neurons efficiently remove damaged mitochondria by mitophagy as a quality control mechanism to ensure cell survival. Excess mitochondrial damage (from triggers such as MPTP exposure, paraquat, and aging-induced dysfunction) causes neuronal demise, contributing to PD pathogenesis. Likewise, reducing the cell’s ability to remove damaged mitochondria (from the loss of Parkin or PINK1) may cause an accumulation of these dysfunctional organelles, leading to early-onset PD.

The expression of PINK1 on peroxisomes recruits Parkin to induce pexophagy (Lazarou et al., 2012). Consistent with the cytosolic substrate model, PINK1 directly phosphorylates Parkin at Ser65 within the Ubl domain of Parkin, and Ser65 phosphorylation stimulates Parkin’s E3 ligase activity and recruitment to mitochondria (Kondapalli et al., 2012; Shiba-Fukushima et al., 2012). However, although Parkin mutated at Ser65 prevents phosphorylation by PINK1, Ser65Ala Parkin as well as Parkin lacking the entire Ubl domain still translocates to mitochondria in a PINK1 kinase-dependent process, indicating that another cytosolic PINK1 substrate is involved in Parkin activation (Kane et al., 2014). This was initially confusing because cell-free systems displayed Parkin activation simply by mixing recombinant PINK1 and recombinant Parkin, and critically, but not realized at the time, E2 enzymes and ubiquitin for assessing Parkin enzyme activity (Lazarou et al., 2013).

Three groups independently discovered that PINK1 phosphorylates ubiquitin at Ser65 and that phospho-ubiquitin activates Parkin E3 ligase activity (Kane et al., 2014; Kazlauskaite et al., 2014b; Koyano et al., 2014). This explained all the prior in-cell and cell-free results and fit seamlessly with the previous finding that PINK1 phosphorylates Parkin in its Ubl domain at Ser65, homologous to ubiquitin Ser65 (Kondapalli et al., 2012). Kane et al. (2014) used mass spectrometry to compare PINK1-knockout (KO) cells to wild-type cells to reveal that endogenous PINK1 stabilized by carbonyl cyanide m-chlorophenyl hydrazone (CCCP)-induced depolarization of mitochondria produces phos-

pho-Ser65 ubiquitin. It was found that phospho-ubiquitin activates Parkin’s E3 ubiquitin ligase activity in a purified cell-free system and that Parkin binds to phosphomimetic ubiquitin Ser65Asp. In living cells, the inhibition of ubiquitin phosphorylation by the expression of ubiquitin Ser65Ala inhibits Parkin translocation to depolarized mitochondria. The fact that phospho-Ub peptides were isolated from purified mitochondrial OMM peptides suggests that PINK1 phosphorylates ubiquitin chains linked to OMM proteins that recruit Parkin to mitochondria and activate it (Kane et al., 2014). Kazlauskaite et al. (2014b) overexpressed PINK1 in cells treated with CCCP to identify ubiquitin phosphorylation at Ser65 by mass spectrometry. Purified recombinant PINK1 was shown to phosphorylate ubiquitin in vitro at Ser65. It also was found that Parkin mutated in the Ubl at Ser65 to mimic phosphorylation (Ser65Asp) displayed greater ubiquitin ligase activity upon incubation with phospho-ubiquitin than did wild-type Parkin, indicating that PINK1 phosphorylation of both Parkin and ubiquitin may be important in activation. However, both Kane et al. (2014) and Kazlauskaite et al. (2014b) found that phosphomimetic mutants of ubiquitin Ser65 are sufficient for Parkin activation, whereas phosphomimetic mutants of Parkin Ser65, at least in cells, are not.

Koyano et al. (2014) also found ubiquitin is phosphorylated by PINK1. Like Kazlauskaite et al. (2014b), Koyano et al. (2014) reported that Parkin Ser65 phosphorylation in combination with ubiquitin Ser65 phosphorylation yields maximal Parkin activity. They also showed that ubiquitin phosphomimetic Ser65Asp further mutated at the C terminus to prevent ubiquitin chain formation activates Parkin, indicating that phospho-ubiquitin is allosteric and unrelated to thioester loading (see the following section) on Parkin. Parkin also can conjugate both phosphomimetic Ser65Asp and phospho-null Ser65Ala ubiquitin mutants to mitochondrial proteins, supporting an allosteric role as phospho-ubiquitin is neither required nor excluded as a Parkin substrate. By immunoprecipitating ubiquitin chains, Ordureau et al. (2014) found PINK1 increases phospho-Ser65-ubiquitin on mitochondria, further indicating that chains of ubiquitin attached to proteins on the OMM become phosphorylated by PINK1 and may serve as Parkin receptors on mitochondria (Ordureau et al., 2014). Parkin binds not only to phospho-ubiquitin, but also to phosphorylated ubiquitin chains. Ordureau et al. (2014) further showed that phospho-ubiquitin binds to Ser65-phosphorylated Parkin with a 21 times higher affinity than to Parkin, suggesting a feedforward model of Parkin activation: first Parkin is phosphorylated by PINK1 and then further activated by phospho-Ser65 ubiquitin. Interestingly, the active site Cys431 of Parkin becomes exposed to Ubiquitin-vinyl sulfone when Parkin is activated by its Ubl phosphorylation, but not by activation by phospho-ubiquitin, suggesting that phospho-Ser65 ubiquitin and phospho-Ser65 Parkin have different modes of activating Parkin (Ordureau et al., 2014).

Although biochemical studies indicate that Parkin phosphorylation at Ser65 is required for Parkin activation, PINK1-null and Parkin-null *Drosophila* can be partially rescued with Ser65Ala Parkin, indicating that Parkin displays Ser65 phosphorylation-independent functions in vivo (Shiba-Fukushima et al., 2014). Whether the Ser65Ala Parkin activity in vivo is due to

---

Neuron 85, January 21, 2015 ©2015 Elsevier Inc. 261

Neuron
Review

Figure 4. PINK1 Is Turned over by Sequential Proteolysis

(A) PINK1 is continuously imported into healthy mitochondria then degraded. Mitochondrial membrane potential drives mitochondrial import through the TIM complex of the mitochondria where proteases MPP and PARL cleave PINK1's mitochondrial targeting sequence and trans-membrane domain. PARL's cleavage between Ala103 and Phe104 creates a free N-terminal phenylalanine that is identified by N-degron type 2 E3 ubiquitin ligases targeting PINK1 to the ubiquitin proteasome.

(B) Depolarization of mitochondria or blocking mitochondrial import causes PINK1 to accumulate on the OMM. Tomm7 is an accessory subunit of the TOM complex that shunts and retains PINK1 on the OMM.

phospho-Ser65 ubiquitin remains to be explored. A model is presented showing that a positive feedback amplification mechanism drives mitophagy to completion when the product of Parkin (ubiquitinated OMM proteins) yields substrates for PINK1 phosphorylation that in turn further activate Parkin, leading to an amplification loop (Figure 5). The recruitment of Parkin to mitochondria is likely initiated by PINK1 phosphorylating ubiquitin already attached to mitochondrial outer membrane proteins, such as mitofusin 1 (Mfn1), prior to Parkin activation.

In conclusion, phosphorylation of ubiquitin at Ser65 activates, or derepresses, the ubiquitin ligase activity of Parkin. Ubiquitin-phosphate is an allosteric effector of Parkin ubiquitin ligase activity. As ubiquitin is the substrate of Parkin, this resembles homotropic allosteric regulation in which an enzyme substrate binds to an allosteric site on a protein to activate enzyme activity, although, in this case, the allosteric site binds phospho-ubiquitin generated by PINK1 and not unmodified ubiquitin. Where phospho-ubiquitin binds to Parkin and the conformation of activated Parkin remain unknown. This interplay of ubiquitin with Parkin as both activator and substrate may relate to why a full genome small interfering RNA screen for modifiers of Parkin translocation yielded a rich interactome of ubiquitin pathway genes that both accelerated and inhibited Parkin translocation after CCCP treatment (Hasson et al., 2013).

Other substrates of PINK1 have been reported to facilitate mitophagy by altering mitochondrial trafficking or dynamics. PINK1 was reported to phosphorylate Miro1 at Ser156 to activate Parkin-mediated proteasomal degradation of Miro1 to arrest mitochondrial motility (Wang et al., 2011). However, while in agreement that Miro1 and Miro2 stability is genetically linked and dependent upon the PINK1/Parkin pathway, others were unable to replicate the result that PINK1 phosphorylates Ser156 on Miro1 (Kazlauskaite et al., 2014a; Liu et al., 2012). Mitofusin2 (Mfn2), a protein involved in OMM fusion, was identified to be phosphorylated at Thr111 and Ser442 by PINK1 to induce Mfn2 binding to Parkin and initiate Mfn2 ubiquitination and degradation (Chen and Dorn, 2013). However, Parkin translocates constitutively to mitochondria in Mfn1/Mfn2-KO cells, arguing that Mfn2 is not involved in Parkin translocation (Narendra et al., 2008). Alternatively, PINK1 may phosphorylate monoubiquitin and/or polyubiquitin chains covalently linked to

Mfn2 to recruit Parkin to mitochondria. Mitofusins are ubiquitinated even in yeast that lack PINK1 and Parkin (Neutzner et al., 2008), supporting the idea that ubiquitinated mitofusins, among other ubiquitinated OMM proteins, may represent a starting point for the PINK1/Parkin positive feedback ubiquitin phosphorylation cycle (Figure 5).

PINK1 may phosphorylate other mitochondrial targets, but they appear unrelated to the PINK1/Parkin mitophagic pathway. Omi/HtrA2, a serine protease, and tumor necrosis factor receptor-associated protein 1 (TRAP-1), a chaperone protein, both have been reported to be targets of PINK1 (Plun-Favreau et al., 2007; Pridgeon et al., 2007). In vitro data, however, failed to validate Omi/HtrA2 and TRAP-1 as PINK1 targets (Kondapalli et al., 2012). Furthermore, Omi/HtrA2 is not downstream of the PINK1/Parkin pathway, and TRAP-1 completely rescues PINK1-deficient, but not Parkin-mutant flies (Costa et al., 2013; Yun et al., 2008; Zhang et al., 2013).

Positive feedback loop drives mitophagy to completion

The product of Parkin E3 ligase activity becomes the substrate for PINK1

Mitochondrial damage
↓
Ubiquitin Amide-linked OMM protein
↓
PINK1 stabilization
↓
Phospho-ubiquitin Amide-linked OMM protein
↓
Inactive parkin
↓
Activated parkin

Figure 5. The Activation of Parkin Drives a Positive Feedback Loop Providing Ubiquitin as a Substrate for PINK1 to Complete Mitophagy

Mitochondrial dysfunction stabilizes PINK1, enabling it to phosphorylate ubiquitin and Parkin. This phosphorylation event activates Parkin, which ubiquitinates substrates on the outer membrane. These ubiquitin chains on the proteins of the OMM provide additional substrates for PINK1’s kinase activity, causing a strong positive feedback cycle assuring mitochondrial clearance.

262 Neuron 85, January 21, 2015 ©2015 Elsevier Inc.

Neuron

Review

A
IBR
RING1
RING0
B

![Image Description](image_url)

Figure 6. Structural Representation of Parkin
(A) Ribbon diagram of the various domains in individual colors: cyan, Ubl; gray, RING0; green, RING1; violet, IBR; blue, Rep; and bronze, RING2. The residues Ser65 (red), Arg275 (blue), and Cys431 (yellow) are highlighted in stick representation.
(B) Juxtaposition of the ribbon diagram over a surface presentation of Parkin (Trempe et al., 2013).

Activating PINK1 kinase activity could have therapeutic applications. The ATP analog, kinetin triphosphate, rescues kinase activity of a patient mutant form of PINK1 and increases wild-type PINK1 activity, revealing a new way to drug the PINK1/Parkin pathway (Hertz et al., 2013).

### Parkin E3 Ubiquitin Ligase Structure and Function
Parkin is an E3 ubiquitin ligase of an unusual class called ring-between-ring (RBR) domain proteins that share three tandem zinc coordination domains (Marín, 2009). In contrast to single RING E3 ligases that juxtapose an E2 enzyme (linked via a high-energy thioester bond to ubiquitin) with a protein substrate to facilitate ubiquitin transfer from the cysteine of the E2 to a lysine on the substrate, RBR proteins, Parkin (Lazarou et al., 2013; Zheng and Hunter, 2013), HOIP (Smit et al., 2012), and HHARI (Wenzel et al., 2011), all transfer ubiquitin from the E2 directly onto the RBR itself, yielding another high-energy thioester intermediate, similar to HECT domain ubiquitin ligases (Wenzel et al., 2011). Parkin has four zinc-coordinating domains, RING0, RING1, IBR, and RING2 (Figures 2 and 6; Hristova et al., 2009). However, only RING1 adopts the canonical ubiquitin ligase RING domain structure that binds to E2 ubiquitin ligases and facilitates the discharge of thioester-bound ubiquitin from the E2 to the substrate. Like other RBR proteins, Parkin requires activation to allow the ubiquitin-charged E2 to bind to Parkin RING1 and transfer ubiquitin to Parkin Cys431 in RING2 to form the HECT-like thioester intermediate (Figure 6A; Riley et al., 2013; Trempe et al., 2013; Wauer and Komander, 2013). Several E2s can transfer ubiquitin to PINK1-activated Parkin in cell-free systems, including UBE2K, UBE2D1, UBE2D2, UBE2E1, UBE2L3, and UBE2C (Kazlauskaite et al., 2014a; Lazarou et al., 2013).

The structure of Parkin shows that the RING0, RING1, and the IBR domains interact and a linker domain wraps around the interface of RING0 and RING1 to reach the RING2 (Riley et al., 2013; Trempe et al., 2013; Wauer and Komander, 2013; Figure 6B). The structure of Parkin represents the inactive or autoinhibited state because the Cys431 site of thioester formation in RING2 (yellow

in Figure 6) is occluded by the RING0 domain, and the E2-binding site on RING1 is blocked by the linker region called a repressor element (blue helix in Figure 6; Riley et al., 2013; Trempe et al., 2013; Wauer and Komander, 2013). A substantial conformational change induced by phospho-ubiquitin and probably contributed to by phospho-S65 Parkin must shift the ubiquitin-E2-binding site on RING1 proximal to the Cys431 acceptor site on RING2. Loading of ubiquitin onto Parkin Cys431 can be activated by PINK1 (Lazarou et al., 2013; Zheng and Hunter, 2013), the PINK1 substrate phosphomimetic ubiquitin Ser65Asp (Kane et al., 2014), or the Parkin Ubl phosphomimetic Ser65Glu (Iguchi et al., 2013). Determining the structure of activated Parkin bound to phospho-ubiquitin will be illuminating.

The ubiquitin-like domain at the N terminus of Parkin folds back and binds to the RING1 domain (Spratt et al., 2013; Trempe et al., 2013). Although cell-free experiments indicated that this Ubl domain interaction inhibits Parkin E3 ligase activity (Chaugule et al., 2011), these experiments did not take PINK1 activation of Parkin into account. Parkin lacking the Ubl domain is still activated by PINK1, indicating that phospho-ubiquitin binding to Parkin does more than displace the Parkin Ubl domain (Kazlauskaite et al., 2014b). However, phosphorylation of the Parkin Ubl domain Ser65 and mutations in the linker region between the IBR and RING2 domains near the Ubl domain may destabilize the Ubl domain binding to RING0 and increase Parkin sensitivity to activation (Koyano et al., 2014; Trempe et al., 2013). Parkin conformational instability may also be linked to age-related or PD-related decreases in Parkin solubility that could connect the PINK1/Parkin pathway to sporadic forms of PD (Meng et al., 2011; Pawlyk et al., 2003; Vandiver et al., 2013).

### Parkin Ubiquitination: Targets and Consequences
Once activated, Parkin modifies many cytosolic and OMM proteins with K48- and K63-linked ubiquitin chains (Chan et al., 2011; Narendra et al., 2012; Sarraf et al., 2013). Quantification of all different chain linkages reveals that Parkin primarily forms noncanonical K6- and K11-linked chains and K48- and K63-linked chains. Mitochondrial proteins from cells displayed more

Neuron 85, January 21, 2015 ©2015 Elsevier Inc. 263

Neuron
Review

Figure 7. Electron Micrographs of Parkin-Induced Mitophagy and Autophagosomal Engulfment of Mitochondria
Mouse embryonic fibroblasts overexpressing Parkin and treated with 20 mM CCCP for 12 hr. White arrowhead points to autophagosome forming around mitochondria. Scale bar = 500 nm. Images from [Yoshii et al., 2011](https://doi.org/10.1038/nsmb.2049).

Mitophagy Mechanisms
Inhibiting autophagic machinery genetically or pharmacologically does not disrupt PINK1/Parkin recruitment to the mitochondria, but it does prevent or delay the elimination of Parkin-ubiquitinated mitochondria ([Fogel et al., 2013](https://doi.org/10.1016/j.molcel.2013.02.015); [Narendra et al., 2008](https://doi.org/10.1016/j.cell.2008.08.025)). Different assays used to assess mitophagy have distinct advantages and drawbacks. Loss of immunostaining of mitochondrial outer membrane proteins such as Tom20 is convenient and robust, but may also reflect OMMAD, the proteasomal degradation of OMM proteins, rather than mitophagy ([Yoshii et al., 2011](https://doi.org/10.1038/nsmb.2049)). Thus, staining for inner mitochondrial membrane or matrix proteins or labeling with mitotracker green is a better measure of mitophagy than staining of outer membrane proteins. However, inner membrane proteins are turned over at different rates and are not necessarily precise measures of mitophagy. The fluorescent protein Keima imported into the mitochondrial matrix changes its excitation wavelength from 440 to 586 when exposed to the acidic lysosomal pH following autophagy and yields a convenient measure of mitophagy ([Katayama et al., 2011](https://doi.org/10.1038/nmeth.1688)). Loss of mitochondrial DNA in nucleoids is another clear way to measure loss of mitochondria. If nucleoid loss is prevented in Atg5-KO cells or by lysosomal inhibitors, such as bafilomycin, one may be confident that the process is mitophagy.

Presumably, the ubiquitin chains that Parkin attaches to mitochondrial proteins initiate mitophagy, as is observed for other autophagy substrates ([Kirkin et al., 2009](https://doi.org/10.1038/nature08070)). Mitophagy ([Itakura et al., 2012](https://doi.org/10.1016/j.cell.2012.06.021)), like starvation-induced autophagy ([Itakura and Mizushima, 2011](https://doi.org/10.1016/j.cell.2011.08.034)), involves the independent recruitment adjacent to the mitochondria of three components of the autophagosome-targeting and -expansion machinery: the serine/threonine kinase UNC-51-like kinase 1, the membrane-spanning Atg9 protein, and LC3-associated isolation membranes. Thus, ubiquitin chains attached to mitochondria seem to independently recruit different complexes to growing isolation membranes that expand alongside mitochondria. How these autophagy components are recruited to isolation membranes growing around mitochondria remains unclear, except that adaptor proteins such as p62, NBR1, NDP52, Tax1BP1, and optineurin, proteins that bind both to ubiquitin chains and LC3/GABARAP family members, may be involved. On the other hand, it is becoming clear how LC3 autophagosomal membrane generation is restricted to accurately surround the mitochondria, as seen in electron microscopy images of Parkin-mediated mitophagy (Figure 7). Two mitochondrial localized RabGAPs, TBC1D15 and TBC1D17, bound to the OMM protein Fis1 govern isolation membrane engulfment of mitochondria. KO of either Fis1 or

K48- and K63-linked chains than K6- and K-11, whereas purified Parkin produced more noncanonical chain linkages than K48- and K63-linked chains ([Ordureau et al., 2014](https://doi.org/10.1038/nsmb.2826)).

Although the role of K6-linked chains is unknown, the K63-linked chains appear to be involved in p62, an autophagy receptor, recruitment ([Geisler et al., 2010a](https://doi.org/10.1038/emboj.2010.16); [Narendra et al., 2010a](https://doi.org/10.1038/emboj.2010.16); [Okatsu et al., 2010](https://doi.org/10.1038/emboj.2010.16); [Sims et al., 2012](https://doi.org/10.1038/emboj.2010.16); [van Wijk et al., 2012](https://doi.org/10.1038/emboj.2010.16)). Although an initial report indicated that p62 is required for Parkin-mediated mitophagy ([Geisler et al., 2010](https://doi.org/10.1038/emboj.2010.16)), two groups using p62-KO mouse embryonic fibroblasts did not find that it is essential for mitophagy ([Narendra et al., 2010a](https://doi.org/10.1038/emboj.2010.16); [Okatsu et al., 2010](https://doi.org/10.1038/emboj.2010.16)). However, the robust clumping of mitochondria that occurs following Parkin activation is prevented in p62-KO cells. The NBR1 ubiquitin receptor also accumulates on mitochondria after Parkin translocation ([Chan et al., 2011](https://doi.org/10.1038/emboj.2010.16)). However, the significance of autophagy receptors in mediating mitophagy remains unclear.

Parkin-generated K11- and K48-linked chains on mitochondria likely lead to an endoplasmic-reticulum-associated-degradation-like extraction of proteins from the OMM (OMMAD) by p97/VCP and subsequent degradation by the proteasome ([Chan et al., 2011](https://doi.org/10.1038/emboj.2010.16); [Kim et al., 2013](https://doi.org/10.1038/emboj.2010.16); [Narendra et al., 2012](https://doi.org/10.1038/emboj.2010.16); [Yoshii et al., 2011](https://doi.org/10.1038/nsmb.2049)). This removal and proteolysis of OMM proteins appears to be necessary for mitophagy ([Chan et al., 2011](https://doi.org/10.1038/emboj.2010.16); [Tanaka et al., 2010](https://doi.org/10.1038/emboj.2010.16)). Modulating the levels of OMM proteins also controls mitochondrial motility and dynamics during the mitophagy process. Mfn1 and Mfn2 are ubiquitinated and degraded by the proteasome in a Parkin-dependent manner ([Gegg et al., 2010](https://doi.org/10.1038/emboj.2010.16); [Poole et al., 2010](https://doi.org/10.1038/emboj.2010.16); [Rakovíc et al., 2011](https://doi.org/10.1038/emboj.2010.16); [Tanaka et al., 2010](https://doi.org/10.1038/emboj.2010.16); [Ziviani et al., 2010](https://doi.org/10.1038/emboj.2010.16)). Miro1, a mitochondrial Rho GTPase involved in trafficking, was identified and confirmed in vitro as a Parkin substrate ([Kazlauskaite et al., 2014a](https://doi.org/10.1038/emboj.2010.16); [Sarraf et al., 2013](https://doi.org/10.1038/emboj.2010.16); [Wang et al., 2011](https://doi.org/10.1038/emboj.2010.16)). One group reported the ability of Parkin to indirectly generate linear ubiquitin chains on mitochondria via NF-κB essential modulator (NEMO) and to elicit protection from apoptosis by stimulating the NF-κB pathway ([Müller-Rischart et al., 2013](https://doi.org/10.1038/emboj.2010.16)). Thirty-six OMM substrates of Parkin have been identified with high confidence ([Sarraf et al., 2013](https://doi.org/10.1038/emboj.2010.16)), suggesting that no specific substrate is required for ubiquitin signaling of mitophagy and that the chain-linkage type and density on mitochondria may be the trigger.
TBC1D15/TBC1D17 leads to excessive expansion of LC3-labeled autophagosomal membranes during Parkin-mediated mitophagy ([Yamano et al., 2014](https://doi.org/10.1038/nsmb.2769)). Parkin activation also induces dysregulated trafficking of LC3-bound tubules in the absence of TBC1D15 that appears to be caused by excessive activation of Rab7 ([Yamano et al., 2014](https://doi.org/10.1038/nsmb.2769)). Fis1 also regulates autophagosomal membrane engulfment of mitochondria in vivo in *C. elegans* ([Shen et al., 2014](https://doi.org/10.1038/nature13015)).

There are tantalizing links between the mitochondrial fission machinery and mitophagy. In yeast, the mitochondrial fission protein, Dnm1, homologous to mammalian Drp1, is required for certain forms of mitophagy ([Abeliovich et al., 2013](https://doi.org/10.1038/ng.2515); [Frank et al., 2012](https://doi.org/10.1038/cdd.2011.175); [Mao et al., 2013](https://doi.org/10.1038/cdd.2012.169)). This leads to the model that the fragmentation of mitochondria may facilitate their engulfment by autophagosomes ([Buhlman et al., 2014](https://doi.org/10.1038/emboj.2014.100); [Tanaka et al., 2010](https://doi.org/10.1038/emboj.2010.16); [Twig et al., 2008](https://doi.org/10.1038/onc.2008.104)) and fits with phenotypes of PINK1- and Parkin-mutant flies that are reversed by promoting mitochondrial fission ([Deng et al., 2008](https://doi.org/10.1038/ng.216); [Poole et al., 2008](https://doi.org/10.1038/ng.215); [Yang et al., 2008](https://doi.org/10.1038/ng.217)).

PINK1 and Parkin can remove small pieces of vesicles from mitochondria ([McLelland et al., 2014](https://doi.org/10.1038/cdd.2013.169); [Yang and Yang, 2013](https://doi.org/10.1038/cdd.2013.169)). This may be related to the piecemeal disposal of the nuclear envelope by autophagy ([Krick et al., 2008](https://doi.org/10.1038/cdd.2008.104)), during which membrane can be pinched off from larger cargo for degradation, and may explain the selective disposal of OXPHOS complex subunits by mitophagy ([Vincow et al., 2013](https://doi.org/10.1038/cdd.2013.169)).

### Modifiers of the PINK1/Parkin Pathway

RNAi screens have been used to uncover a number of genes that modify Parkin translocation and/or mitophagy. Knockdown of genes, such as HK1 and HK2 ([McCoy et al., 2014](https://doi.org/10.1038/cdd.2013.169)), BAG4 and HSPA1L ([Hasson et al., 2013](https://doi.org/10.1038/cdd.2013.169)), IF1 ([Lefebvre et al., 2013](https://doi.org/10.1038/cdd.2013.169)), SREBF1 and FBXW7 ([Ivatt et al., 2014](https://doi.org/10.1038/cdd.2013.169)), and SMURF1 ([Orvedahl et al., 2011](https://doi.org/10.1038/cdd.2013.169)), inhibits Parkin translocation, indicating that those gene products normally participate in Parkin activation. Knockdown of other genes, such as SIAH3 ([Hasson et al., 2013](https://doi.org/10.1038/cdd.2013.169)), accelerates Parkin translocation, indicating the gene product normally inhibits Parkin activation. Deubiquitinases (DUBs), such as USP30 ([Bingol et al., 2014](https://doi.org/10.1038/cdd.2013.169)) and USP15 ([Cornelissen et al., 2014](https://doi.org/10.1038/cdd.2013.169)), antagonize Parkin ubiquitin ligase activity by cleaving ubiquitin chains on mitochondria. Another DUB, USP8, has been identified to remove K6 chains from Parkin itself to positively regulate Parkin recruitment ([Durcan et al., 2014](https://doi.org/10.1038/cdd.2013.169)). Drugging the DUBs may be a strategy to activate the PINK/Parkin pathway and promote mitochondrial quality control. Targeted screens of E2 ubiquitin ligases that may load Parkin with ubiquitin ([Fiesel et al., 2014](https://doi.org/10.1038/cdd.2013.169); [Geisler et al., 2014](https://doi.org/10.1038/cdd.2013.169)) show that UBE2D, UBE2L3, and UBE2N participate in Parkin activity.

Affinity-binding studies indicate that AMBRA1, a protein involved in autophagy, binds to Parkin and promotes mitophagy ([Van Humbeeck et al., 2011](https://doi.org/10.1038/cdd.2013.169)). Overexpression of an OMM-targeted AMBRA1 also stimulates mitophagy in the presence or absence of Parkin ([Strappazzon et al., 2014](https://doi.org/10.1038/cdd.2013.169)). Similarly, PINK1 has been reported to bind another autophagy protein, Beclin1, to promote autophagy ([Michiorri et al., 2010](https://doi.org/10.1038/cdd.2013.169)).

Parkin-mediated mitophagy is also linked to apoptosis ([Winklhofer, 2014](https://doi.org/10.1038/cdd.2013.169); [Zhang et al., 2014](https://doi.org/10.1038/cdd.2013.169)). Interestingly, antiapoptotic Bcl-2 family proteins that localize to the OMM, Bcl-xL and Mcl-1, inhibit Parkin translocation and, therefore, suppress

mitophagy ([Hollville et al., 2014](https://doi.org/10.1038/cdd.2013.169)). Furthermore, translocation of the proapoptotic Bcl-2 family protein Bax to mitochondria, which is a key decision step in apoptosis ([Wolter et al., 1997](https://doi.org/10.1038/cdd.2013.169)), is inhibited by Parkin, perhaps contributing to Parkin inhibition of apoptosis ([Johnson et al., 2012](https://doi.org/10.1038/cdd.2013.169)). However, in some experimental systems, Parkin promotes apoptosis by ubiquitinating and inducing proteasomal degradation of Mcl-1 ([Zhang et al., 2014](https://doi.org/10.1038/cdd.2013.169)).

### Parkin Activity beyond Mitochondria

Numerous Parkin substrates were identified before Parkin’s link to mitochondria was identified ([Scarffe et al., 2014](https://doi.org/10.1038/cdd.2013.169)); and, recently, Parkin also was found to ubiquitinate the transcriptional repressor named PARIS, causing it to be degraded by the proteasome and thereby inducing mitochondrial biogenesis ([Shin et al., 2011](https://doi.org/10.1038/cdd.2013.169)). In addition to mitochondria, Parkin has been found to accumulate on intracellular *Mycobacterium*, likely to induce their autophagy ([Manzanillo et al., 2013](https://doi.org/10.1038/cdd.2013.169)) as a host defense mechanism and consistent with genetic studies linking Parkin mutations to increased susceptibility to leprosy ([Mira et al., 2004](https://doi.org/10.1038/cdd.2013.169)). Parkin, like other RBR proteins, needs to be activated to display ubiquitin ligase activity. A mystery for these nonmitochondrial substrates is how Parkin’s autoinhibition is relieved. For mitochondrial substrates, phospho-ubiquitin produced by PINK1 kinase activity is sufficient to bind and induce Parkin activation. The kinase(s) that may activate Parkin for these proposed extramitochondrial substrates remain(s) to be identified.

### Physiological Triggers of Parkin Translocation beyond CCCP

It is experimentally convenient in cultured cell lines, in primary neurons ([Ashrafi et al., 2014](https://doi.org/10.1038/cdd.2013.169)) and in induced pluripotent stem cells (iPSCs) differentiated into neurons ([Rakovic et al., 2013](https://doi.org/10.1038/cdd.2013.169); [Seibler et al., 2011](https://doi.org/10.1038/cdd.2013.169)), to trigger PINK1/Parkin-mediated mitophagy using pharmacological reagents to depolarize ΔΨm, such as CCCP, oligomycin combined with antimycin A, valinomycin, or the photosensitizer KillerRed ([Wang et al., 2012](https://doi.org/10.1038/cdd.2013.169); [Yang and Yang, 2011](https://doi.org/10.1038/cdd.2013.169)). However, the pathway also proceeds constitutively in neurons without chemical perturbation. Using mitochondrial-targeted Keima to assess mitophagy, Bingol and colleagues (2014) showed that mitophagy occurs over a period of days (in mouse primary hippocampal neurons) without the addition of depolarizing drugs, and that this constitutive mitophagy requires endogenous PINK1 and Parkin ([Bingol et al., 2014](https://doi.org/10.1038/cdd.2013.169)).

Genetic models of mitochondrial dysfunction that mimic pathophysiological processes also show that Parkin recruitment is possible without pharmacological manipulation. Genetic loss of Mfn1 and Mfn2 leads to a subset of mitochondria lacking membrane potential ([Chen et al., 2005](https://doi.org/10.1038/cdd.2013.169)) that selectively recruit Parkin ([Narendra et al., 2008](https://doi.org/10.1038/cdd.2013.169)). Parkin is also recruited to dysfunctional mitochondrial complex I components in neurons differentiated from iPSCs derived from mitochondrial encephalomyopathy, lactic acidosis, and stroke-like episodes (MELAS) patients harboring the mtDNA mutation m.3243A > G ([Hämäläinen et al., 2013](https://doi.org/10.1038/cdd.2013.169)). Cybrid cell lines harboring mtDNA mutations display Parkin-mediated mitophagy in the presence of overexpressed Parkin or upon inhibition of mammalian target of rapamycin complex 1, mTORC1, thereby activating autophagy ([Gilkerson et al., 2012](https://doi.org/10.1038/cdd.2013.169); [Suen et al., 2010](https://doi.org/10.1038/cdd.2013.169)).A type of mitochondrial dysfunction without loss of ΔΨm also may trigger Parkin-mediated mitophagy. Overexpression of a mutant form of ornithine transcarbamylase (ΔOTC), an enzyme involved in the urea cycle, abnormally aggregates inside the mitochondrial matrix and triggers the mitochondrial unfolded protein response ([Zhao et al., 2002](https://doi.org/10.1074/jbc.M205689200)). The aggregated ΔOTC causes stabilization of PINK1, recruitment of Parkin to mitochondria, and subsequent removal of ΔOTC without depolarizing mitochondria ([Jin and Youle, 2013](https://doi.org/10.1016/j.cell.2013.05.028)). The ΔOTC expressed in *Drosophila* phenocopies the muscle and flight wing mitochondrial defects found in PINK1- and Parkin-mutant flies, and this phenotype can be rescued by the overexpression of Parkin in ΔOTC flies ([Pimenta de Castro et al., 2012](https://doi.org/10.1016/j.cub.2012.03.039)).

### Mouse Models Manipulating Parkin or PINK1 Expression

After the discovery that autosomal recessive mutations in Parkin lead to a loss of function, many laboratories independently generated Parkin-KO mice to study Parkin’s function in vivo. The first two Parkin-KO mouse models lacked exon 3 and exhibited only mild phenotypes, such as the disruption of fine motor skills and slight abnormalities in dopamine metabolism and release, but, disappointingly, no dopaminergic neuron loss ([Goldberg et al., 2003](https://doi.org/10.1073/pnas.2534710100); [Itier et al., 2003](https://doi.org/10.1016/S0896-6273(03)00677-6)). Another PARK2 mouse model with a deletion of exon 7 also did not cause motor behavioral phenotypes or DA neurodegeneration ([von Coelln et al., 2004](https://doi.org/10.1016/j.nbd.2004.03.004)). An exhaustive study of a Parkin-null mouse lacking exon 2 revealed no substantial progressive motor phenotype, no DA neurochemical defects in the striatum, and no DA neurodegeneration ([Perez and Palmiter, 2005](https://doi.org/10.1523/JNEUROSCI.4338-04.2005)). A conditional Cre-loxP exon 7 Parkin-KO mouse was generated that showed DA neurodegeneration after lentiviral delivery of a GFP-tagged Cre recombinase was delivered stereotactically to the midbrain of adult mice ([Shin et al., 2011](https://doi.org/10.1016/j.neuron.2011.05.025)). This mouse model showed a loss of Parkin protein and upregulation of PARIS, which the authors concluded was responsible for DA neuron death ([Shin et al., 2011](https://doi.org/10.1016/j.neuron.2011.05.025)). However, it is unclear if the DA loss in this inducible model led to locomotor deficits. It was also untested if the conditional loss of Parkin in adults causes neurodegeneration of other neuronal populations. Most Parkin-KO mouse models fail to model the pathophysiology of PD or display the selective DA neuron loss with the ubiquitous loss of Parkin seen in man.

The lack of DA neurodegeneration and phenotypes that poorly recapitulate human PD also occurred when research groups generated PINK1-KO mouse models. The deletion in exons 4–7 of PINK1 resulted in mice displaying no signs of DA neurodegeneration or altered DA metabolism ([Kitada et al., 2007](https://doi.org/10.1038/ng2023)). A PINK1-KO mouse with exons 4 and 5 deleted displayed a progressive reduction of DA in the striatum, but these results were puzzling because there was no evidence of DA neuron loss ([Akundi et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.09.014)).

Mitochondrial impairments have been reported in the central nervous system of Parkin- and PINK1-KO mice. Aged Parkin-KO brains have altered expression of mitochondrial proteins and a reduction in respiratory capacity ([Palacino et al., 2004](https://doi.org/10.1073/pnas.0402217101); [Periquet et al., 2005](https://doi.org/10.1016/j.expneurol.2005.04.015); [Stichel et al., 2007](https://doi.org/10.1016/j.neurobiolaging.2006.08.005)). Primary PINK1-KO/knockdown (KD) fibroblasts and neurons display reduced ΔΨm, impairments in respiration, Ca²⁺ overload in mitochondria, and heightened reactive oxygen species (ROS) production ([Gandhi et al., 2009](https://doi.org/10.1016/j.neurobiolaging.2008.08.006); [Gautier et al., 2012](https://doi.org/10.1016/j.neurobiolaging.2011.09.010); [Heeman et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.11.014)). ATP levels are not changed in the striatum of PINK1-KO mice ([Gautier et al., 2008](https://doi.org/10.1016/j.neurobiolaging.2007.08.003)), whereas PINK1-KO/KD fibroblasts and neurons in culture display impaired ATP synthesis ([Heeman et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.11.014)). Mitochondria isolated from the striatum or whole brain of aged PINK1-KO mice display defects in complex I ([Gautier et al., 2008](https://doi.org/10.1016/j.neurobiolaging.2007.08.003); [Morais et al., 2009](https://doi.org/10.1016/j.neurobiolaging.2008.08.005)), and mitochondria derived from the brains of PINK1-KO mice have reduced calcium-buffering capacity ([Akundi et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.09.014)). Despite these mitochondrial dysfunctions, Parkin- and PINK1-KO DA neurons remain viable and functional. Even two-year-old Parkin/PINK1/DJ-1 triple-KO mouse lacked DA neurodegeneration ([Kitada et al., 2009](https://doi.org/10.1016/j.neurobiolaging.2008.08.006)). These results suggest that mice compensate for the loss of PINK1 and Parkin in DA neurons or that central nervous system neurons in aged mice do not reach a threshold of mitochondrial dysfunction necessary to cause detrimental phenotypes. The murine resistance to PINK1-KO phenotypes does not extend to rats, as PINK1-KO Long Evans rats present progressive DA neurodegeneration and motor phenotypes ([Dave et al., 2014](https://doi.org/10.1016/j.neurobiolaging.2013.11.006)). Although neuronal health appears unaffected, PINK1-KO mice develop a progressive cardiomyopathy caused by mitochondrial dysfunction and oxidative stress ([Billia et al., 2011](https://doi.org/10.1016/j.yjmcc.2011.05.002)). Consistent with this cardiac defect, Parkin-KO mice display a more severe phenotype than wild-type mice following experimental myocardial infarction that was attributed to an accumulation of damaged mitochondria; however, they did not display the cardiac hypertrophy reported for PINK1-KO mice ([Kubli et al., 2013](https://doi.org/10.1016/j.yjmcc.2013.04.007)). Cardiac ischemic preconditioning also required Parkin activity ([Huang et al., 2011](https://doi.org/10.1016/j.yjmcc.2011.03.004)).

As unchallenged Parkin-KO mice lack obvious phenotypes, several groups crossed mouse models of mitochondrial dysfunction with the Parkin-KO mouse to assess potential roles for Parkin in vivo in mitochondrial quality control. The phenotype caused by deletion of mitochondrial transcription factor A (TFAM) in DA neurons was no worse in a Parkin-KO background ([Sterky et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.11.015)). This lack of synthetic phenotype indicates that endogenous Parkin does not mitigate the effects on mitochondria caused by loss of TFAM. This could be due to the essential role of TFAM in mtDNA replication that leaves no viable pool of mitochondria for a quality control pathway to enrich. However, one might anticipate Parkin to be recruited to the defective mitochondria in TFAM-null cells, which was not detected ([Sterky et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.11.015)). In addition, there are reports that the loss of Parkin did not worsen motor phenotypes or translocate, respectively, in two mouse models affecting mitochondrial fission and fusion via Purkinje cell-specific Drp-1-KO mice ([Kageyama et al., 2012](https://doi.org/10.1016/j.neurobiolaging.2011.11.010)) and dopaminergic neuron-specific Mfn2-KO mice ([Lee et al., 2012](https://doi.org/10.1016/j.neurobiolaging.2011.11.011)). On the other hand, knocking out choline kinase beta in the skeletal muscle of mice leads to mitochondrial dysfunction, Parkin translocation to mitochondria, and mitophagy ([Mitsuhashi et al., 2011](https://doi.org/10.1016/j.neurobiolaging.2010.11.016)). In a model of muscle atrophy following denervation, Parkin translocation to mitochondria is more readily detected in vivo in ATG7-KO mice than in wild-type mice ([Furuya et al., 2014](https://doi.org/10.1016/j.neurobiolaging.2013.11.007)). Parkin translocation appears to be more apparent in vivo when mitophagy is blocked downstream, thereby delaying the elimination of the mitochondrial Parkin. Thus, Parkin translocation to mitochondria can occur

266 Neuron 85, January 21, 2015 ©2015 Elsevier Inc.
in vivo, at least in skeletal muscle. However, there is no experimental evidence to date in mammals that Parkin rescues mitochondrial damage in dopaminergic neurons.

**Drosophila Models Manipulating Parkin or PINK1 Expression**

The absence of adequate mammalian models led to the generation of other mutant organisms to elucidate the functions of these genes. The generation of Parkin- and PINK1-mutant flies was the cornerstone in understanding the genetic link between these two proteins and their relationship to mitochondrial integrity.

Parkin-mutant *Drosophila* display a severe flight muscle defect leading to locomotive behavioral problems, male sterility, and reduced lifespan (Greene et al., 2003; Pesah et al., 2004). Parkin-mutant flies are more susceptible to oxidative stress, and, although not all DA neuron clusters are affected, some DA neurons display abnormal shrinkage and morphology, and one group reported neuron loss (Cha et al., 2005; Greene et al., 2003, 2005; Pesah et al., 2004; Whitworth et al., 2005). Mitochondria are abnormal and swollen in muscle cells lacking Parkin activity, and this observation became important in connecting PINK1 to Parkin mechanistically (Greene et al., 2003).

PINK1-mutant or -KD *Drosophila* phenocopy Parkin-mutant flies and display the same flight muscle degeneration, sensitivity to environmental stressors, and reduced lifespan (Clark et al., 2006; Park et al., 2006; Yang et al., 2006). PINK1-KD and -mutant flies also display selective DA neuron degeneration and the same abnormal mitochondrial morphology present in Parkin-mutant flies (Clark et al., 2006; Park et al., 2006; Yang et al., 2006). Such strong similarities between the two models indicated that PINK1/Parkin work in the same pathway. This was corroborated when researchers showed that PINK1/Parkin double-mutant flies were no worse than single-mutant flies. Overexpression of Parkin rescued the PINK1-deficient phenotype, but not vice versa, indicating that PINK1 acts upstream of Parkin in the same pathway (Clark et al., 2006; Park et al., 2006; Yang et al., 2006).

The swollen mitochondrial phenotype in both PINK1- and Parkin-mutant flies led to the investigation of proteins that control mitochondrial morphology. Genetic loss of dynamin-related protein 1 (Drp1), a mitochondrial fission protein, caused synthetic lethality in PINK1 and Parkin mutants (Deng et al., 2008; Poole et al., 2008). The relationship with mitochondrial morphogenesis was extended to Marf (a homolog of human Mfn) and optical atrophy 1 (OPA1), two proteins required for mitochondrial fusion (Deng et al., 2008; Poole et al., 2008). The silencing of Marf or OPA1 or the overexpression of Drp1 reverted the mitochondrial morphology and biochemical deficits exhibited in PINK1- and Parkin-mutant flies (Deng et al., 2008; Liu et al., 2011; Park et al., 2009; Poole et al., 2008; Yang et al., 2008). The link between mitochondrial morphology and PINK1/Parkin is not so clear in cultured mammalian cells as mitochondria appear grossly normal in Parkin- and PINK1-null cells. However, as mitophagy has been linked to mitochondrial fission in yeast and mammals, perhaps the fission activity of PINK1 and Parkin in flies reflects mitochondrial division to segregate debris for autophagic elimination (Twig and Shirihai, 2011; Youle and van der Bliek, 2012).

Ubiquitous or neuronal overexpression of Parkin boosts mitochondrial number and increases the lifespan of flies (Rana et al., 2013). PINK1-deficient flies have deficits in complex I, decreased ΔΨm, and reduced ATP levels (Liu et al., 2011; Morais et al., 2009). Cardiomyocyte-specific Parkin-KO flies accumulate depolarized mitochondria that produce excess ROS (Bhandari et al., 2014). Increasing mitochondrial function by facilitating electron transport or overexpressing the yeast equivalent of complex I rescues the PINK1-KO phenotypes, but not Parkin-KO phenotypes in *Drosophila* (Vilain et al., 2012; Vos et al., 2012). Constitutively active Ret, the receptor for glia-cell-lined-derived neurotrophic factor, also rescues only PINK1-KO flies, restoring the mitochondrial deficits seen in this model (Klein et al., 2014). These data suggest PINK1 has functions independent of Parkin that may stem from PINK1 phosphorylation of ubiquitin. Interestingly, an E3 ubiquitin ligase located on the OMM named MULAN can compensate for Parkin activity and appears to work in a parallel pathway to ubiquitinate mitofusin (Yun et al., 2014).

Loss-of-function mutations in Parkin and PINK1 in *Drosophila* lead to the accumulation of dysfunctional mitochondria in DA neurons due to defective mitophagy (Burman et al., 2012). Mass spectrometry analysis of fly neurons shows that the turnover rates of different subunits of mitochondrial OXPHOS complexes differ despite occurring via mitophagy (Vincow et al., 2013). This indicates that segregation of mitochondrial proteins occurs, possibly of properly folded and functional proteins versus misfolded or aggregated proteins. The molecular basis of such protein segregation mechanisms leading to mitophagy remains to be found.

**PINK1 and Parkin Patients Compared to Sporadic PD Patients**

Patients with PINK1 and Parkin recessive mutations have PD that is clinically indistinguishable from that of sporadic cases, although they have an earlier onset of the disease (Houlden and Singleton, 2012; Matsumine et al., 1997; Valente et al., 2001). However, it is controversial whether or not Parkin and PINK1 patients have a different neuropathological presentation of Lewy body formation as compared to sporadic cases (Houlden and Singleton, 2012; Kitada et al., 1998; Pramstaller et al., 2005; Samaranch et al., 2010). As α-synuclein mutations are linked to PD, and α-synuclein is a major component of Lewy bodies, its aggregation is a leading candidate for the etiology of PD. Are mutations in α-synuclein and PINK1/Parkin unrelated causes of PD or do they share common downstream targets? Mouse models and cell culture models have attempted to elucidate whether there is an interaction between the PINK1/Parkin pathway and mutant α-synuclein expression; however, to date, it is unclear if this is the case (Choubey et al., 2011; Oliveras-Salvá et al., 2014; Stichel et al., 2007; von Coelln et al., 2006). The α-synuclein aggregates can localize to mitochondria and may cause mitochondrial dysfunction (Devi et al., 2008; Dryanovski et al., 2013). Macroautophagy is impaired by α-synuclein (Winslow et al., 2010), and blocking autophagy causes the accumulation of damaged mitochondria in vivo (Takamura et al., 2011). Whether mitochondrial dysfunction can lead to α-synuclein aggregation is unknown. Analyses of potential mechanistic

Neuron 85, January 21, 2015 ©2015 Elsevier Inc. 227
links between these and other PD disease genes are active areas of investigation.

### Concluding Remarks

Elucidating the functions of genes mutated in monogenic forms of PD has helped researchers better grasp an understanding of mitochondrial cell biology pathways and how these may go awry in sporadic cases and contribute to disease (Figure 3). Augmenting mitophagy by activating the PINK1/Parkin pathway is an attractive target for therapeutic intervention for PD and a variety of maternally inherited mitochondrial diseases. The new understanding of mechanism and the importance of mitochondrial quality control may also extend to other common neurodegenerative diseases associated with mitochondrial dysfunction, such as Alzheimer’s disease and amyotrophic lateral sclerosis.

### ACKNOWLEDGMENTS

The NIH NINDS intramural program (R.J.Y. and A.M.P.) and the Postdoctoral Research Associate Program at the National Institute of General Medical Sciences (A.M.P.) supported this work. We thank Soojay Banerjee for assistance with artwork. We also thank Katherine Roche for thoughtful reading of the manuscript and members of the Youle lab for comments.

### REFERENCES

Abeliovich, H., Zarei, M., Rigbolt, K.T., Youle, R.J., and Dengjel, J. (2013). Involvement of mitochondrial dynamics in the segregation of mitochondrial matrix proteins during stationary phase mitophagy. Nat. Commun. 4, 2789.

Akundi, R.S., Huang, Z., Eason, J., Pandya, J.D., Zhi, L., Cass, W.A., Sullivan, P.G., and Büeler, H. (2011). Increased mitochondrial calcium sensitivity and abnormal expression of innate immunity genes precede dopaminergic defects in Pink1-deficient mice. PLoS ONE 6, e16038.

Anderson, S., Bankier, A.T., Barrell, B.G., de Bruijn, M.H., Coulson, A.R., Drouin, J., Eperon, I.C., Nierlich, D.P., Roe, B.A., Sanger, F., et al. (1981). Sequence and organization of the human mitochondrial genome. Nature 290, 457–465.

Ashrafi, G., Schlehe, J.S., LaVoie, M.J., and Schwarz, T.L. (2014). Mitophagy of damaged mitochondria occurs locally in distal neuronal axons and requires PINK1 and Parkin. J. Cell Biol. 206, 655–670.

Bender, A., Krishnan, K.J., Morris, C.M., Taylor, G.A., Reeve, A.K., Perry, R.H., Jaros, E., Hersheson, J.S., Betts, J., Klostock, T., et al. (2006). High levels of mitochondrial DNA deletions in substantia nigra neurons in aging and Parkinson disease. Nat. Genet. 38, 515–517.

Bentivoglio, A.R., Cortelli, P., Valente, E.M., Ialongo, T., Ferraris, A., Elia, A., Montagna, P., and Albanese, A. (2001). Phenotypic characterisation of autosomal recessive PARK6-linked parkinsonism in three unrelated Italian families. Mov. Disord. 16, 999–1006.

Betarbet, R., Sherer, T.B., MacKenzie, G., Garcia-Osuna, M., Panov, A.V., and Greenamyre, J.T. (2000). Chronic systemic pesticide exposure reproduces features of Parkinson’s disease. Nat. Neurosci. 3, 1301–1306.

Bhandari, P., Song, M., Chen, Y., Burelle, Y., and Dorn, G.W., 2nd. (2014). Mitochondrial contagion induced by Parkin deficiency in Drosophila hearts and its containment by suppressing mitofusin. Circ. Res. 114, 257–265.

Billia, F., Hauck, L., Konecny, F., Rao, V., Shen, J., and Mak, T.W. (2011). PTEN-inducible kinase 1 (PINK1)/Park6 is indispensable for normal heart function. Proc. Natl. Acad. Sci. USA 108, 9572–9577.

Bingol, B., Tea, J.S., Phu, L., Reichelt, M., Bakalarski, C.E., Song, Q., Foreman, O., Kirkpatrick, D.S., and Sheng, M. (2014). The mitochondrial deubiquitinase USP30 opposes parkin-mediated mitophagy. Nature 510, 370–375.

Buhlman, L., Damiano, M., Bertolin, G., Ferrando-Miguel, R., Lombès, A., Brice, A., and Corti, O. (2014). Functional interplay between Parkin and Drp1

in mitochondrial fission and clearance. Biochim. Biophys. Acta 1843, 2012–2026.

Burman, J.L., Yu, S., Poole, A.C., Decal, R.B., and Pallanck, L. (2012). Analysis of neural subtypes reveals selective mitochondrial dysfunction in dopaminergic neurons from parkin mutants. Proc. Natl. Acad. Sci. USA 109, 10438–10443.

Cha, G.H., Kim, S., Park, J., Lee, E., Kim, M., Lee, S.B., Kim, J.M., Chung, J., and Cho, K.S. (2005). Parkin negatively regulates JNK pathway in the dopaminergic neurons of Drosophila. Proc. Natl. Acad. Sci. USA 102, 10345–10350.

Chan, N.C., Salazar, A.M., Pham, A.H., Sweredoski, M.J., Kolawa, N.J., Graham, R.L., Hess, S., and Chan, D.C. (2011). Broad activation of the ubiquitin-proteasome system by Parkin is critical for mitophagy. Hum. Mol. Genet. 20, 1726–1737.

Chaugule, V.K., Burchell, L., Barber, K.R., Sidhu, A., Leslie, S.J., Shaw, G.S., and Walden, H. (2011). Autoregulation of Parkin activity through its ubiquitin-like domain. EMBO J. 30, 2853–2867.

Chen, Y., and Dorn, G.W., 2nd. (2013). PINK1-phosphorylated mitofusin 2 is a Parkin receptor for culling damaged mitochondria. Science 340, 471–475.

Chen, H., Chomyn, A., and Chan, D.C. (2005). Disruption of fusion results in mitochondrial heterogeneity and dysfunction. J. Biol. Chem. 280, 26185–26192.

Choubey, V., Safiulina, D., Vaarmann, A., Cagelinec, M., Waeski, P., Kuum, M., Zharkovsky, A., and Kaasik, A. (2011). Mutant A53T alpha-synuclein induces neuronal death by increasing mitochondrial autophagy. J. Biol. Chem. 286, 10814–10824.

Clark, I.E., Dodson, M.W., Jiang, C., Cao, J.H., Huh, J.R., Seol, J.H., Yoo, S.J., Hay, B.A., and Guo, M. (2006). Drosophila pink1 is required for mitochondrial function and interacts genetically with parkin. Nature 441, 1162–1166.

Cornelissen, T., Haddad, D., Wauters, F., Van Humbeeck, C., Mandemakers, W., Kootjorio, B., Sue, C., Gevaert, K., De Strooper, B., Verstreken, P., and Vandenberghe, W. (2014). The deubiquitinase USP15 antagonizes Parkin-mediated mitochondrial ubiquitination and mitophagy. Hum. Mol. Genet. 23, 5227–5242.

Costa, A.C., Loh, S.H., and Martins, L.M. (2013). Drosophila Trap1 protects against mitochondrial dysfunction in a PINK1/parkin model of Parkinson’s disease. Cell Death Dis. 4, e467.

Dave, K.D., De Silva, S., Sheth, N.P., Ramboz, S., Beck, M.J., Quang, C., Switzer, R.C., 3rd, Ahmad, S.O., Sunkin, S.M., Walker, D., et al. (2014). Phenotypic characterization of recessive gene knockout rat models of Parkinson’s disease. Neurobiol. Dis. 70, 190–203.

de Rijk, M.C., Breteler, M.M., Graveland, G.A., Ott, A., Grobbee, D.E., van der Meché, F.G., and Hofman, A. (1995). Prevalence of Parkinson’s disease in the elderly: the Rotterdam Study. Neurology 45, 2143–2146.

Deas, E., Plun-Favreau, H., Gandhi, S., Desmond, H., Kjaer, S., Loh, S.H., Renton, A.E., Harvey, R.J., Whitworth, A.J., Martins, L.M., et al. (2011). PINK1 cleavage at position A103 by the mitochondrial protease PARL. Hum. Mol. Genet. 20, 867–879.

Deng, H., Dodson, M.W., Huang, H., and Guo, M. (2008). The Parkinson’s disease genes pink1 and parkin promote mitochondrial fission and/or inhibit fusion in Drosophila. Proc. Natl. Acad. Sci. USA 105, 14503–14508.

Devi, L., Raghavendran, V., Prabhu, B.M., Avadhani, N.G., and Anandatheerthavarada, H.K. (2008). Mitochondrial import and accumulation of alpha-synuclein impair complex I in human dopaminergic neuronal cultures and Parkinson disease brain. J. Biol. Chem. 283, 9089–9100.

Dryanovski, D.I., Guzman, J.N., Xie, Z., Galteri, D.J., Volpicelli-Daley, L.A., Lee, V.M., Miller, R.J., Schumacker, P.T., and Surmeier, D.J. (2013). Calcium entry and α-synuclein inclusions elevate dendritic mitochondrial oxidant stress in dopaminergic neurons. J. Neurosci. 33, 10154–10164.

Durcan, T.M., Tang, M.Y., Pérusse, J.R., Dashti, E.A., Aguileta, M.A., McLeland, G.L., Gros, P., Shaler, T.A., Faubert, D., Coulombe, B., and Fon, E.A. (2014). USP8 regulates mitophagy by removing K6-linked ubiquitin conjugates from parkin. EMBO J. 33, 2473–2491. [http://dx.doi.org/10.15252/embj.201489729](http://dx.doi.org/10.15252/embj.201489729).

268 Neuron 85, January 21, 2015 ©2015 Elsevier Inc.
Fiesel, F.C., Moussaud-Lamodière, E.L., Ando, M., and Springer, W. (2014). A specific subset of E2 ubiquitin-conjugating enzymes regulate Parkin activation and mitophagy differently. J. Cell Sci. 127, 3488–3504.

Fogel, A.I., Dlouhy, B.J., Wang, C., Ryu, S.W., Neutzner, A., Hasson, S.A., Sideris, D.P., Abeliovich, H., and Youle, R.J. (2013). Role of membrane association and Atg14-dependent phosphorylation in beclin-1-mediated autophagy. Mol. Cell. Biol. 33, 3675–3688.

Frank, M., Duvezin-Caubet, S., Koob, S., Occhipinti, A., Jagasia, R., Petcherski, A., Ruonala, M.O., Priault, M., Salin, B., and Reichert, A.S. (2012). Mitophagy is triggered by mild oxidative stress in a mitochondrial fission dependent manner. Biochim. Biophys. Acta 1823, 2297–2310.

Furuya, N., Ikeda, S., Sato, S., Soma, S., Ezaki, J., Oliva Trejo, J.A., Takeda-Ezaki, M., Fujimura, T., Arikawa-Hirasawa, E., Tada, N., et al. (2014). PARK2/Parkin-mediated mitochondrial clearance contributes to proteasome activation during slow-twitch muscle atrophy via NFE2L1 nuclear translocation. Autophagy 10, 631–641.

Gandhi, S., Wood-Kaczmar, A., Yao, Z., Plun-Favreau, H., Deas, E., Klupsch, K., Downward, J., Latchman, D.S., Tabrizi, S.J., Wood, N.W., et al. (2009). PINK1-associated Parkinson’s disease is caused by neuronal vulnerability to calcium-induced cell death. Mol. Cell 33, 627–638.

Gautier, C.A., Kitada, T., and Shen, J. (2008). Loss of PINK1 causes mitochondrial functional defects and increased sensitivity to oxidative stress. Proc. Natl. Acad. Sci. USA 105, 11364–11369.

Gautier, C.A., Giaime, E., Caballero, E., Núñez, L., Song, Z., Chan, D., Villalobos, C., and Shen, J. (2012). Regulation of mitochondrial permeability transition pore by PINK1. Mol. Neurodegener. 7, 22.

Gegg, M.E., Cooper, J.M., Chau, K.Y., Rojo, M., Schapira, A.H., and Taanman, J.W. (2010). Mitofusin 1 and mitofusin 2 are ubiquitinated in a PINK1/parkin-dependent manner upon induction of mitophagy. Hum. Mol. Genet. 19, 4861–4870.

Geisler, S., Holmström, K.M., Skujat, D., Fiesel, F.C., Rothfuss, O.C., Kahle, P.J., and Springer, W. (2010a). PINK1/Parkin-mediated mitophagy is dependent on VDAC1 and p62/SQSTM1. Nat. Cell Biol. 12, 119–131.

Geisler, S., Holmström, K.M., Treis, A., Skujat, D., Weber, S.S., Fiesel, F.C., Kahle, P.J., and Springer, W. (2010b). The PINK1/Parkin-mediated mitophagy is compromised by PD-associated mutations. Autophagy 6, 871–878.

Geisler, S., Vollmer, S., Golombek, S., and Kahle, P.J. (2014). The ubiquitin-conjugating enzymes UBE2N, UBE2L3 and UBE2D2/3 are essential for Parkin-dependent mitophagy. J. Cell Sci. 127, 3280–3293.

Gilkerson, R.W., De Vries, R.L., Lebot, P., Wikstrom, J.D., Torgyekes, E., Shirihai, O.S., Przedborski, S., and Schon, E.A. (2012). Mitochondrial autophagy in cells with mtDNA mutations results from synergistic loss of transmembrane potential and mTORC1 inhibition. Hum. Mol. Genet. 21, 978–990.

Goldberg, M.S., Fleming, S.M., Palacino, J.J., Cepeda, C., Lam, H.A., Bhatnagar, A., Meloni, E.G., Wu, N., Ackerson, L.C., Klapstein, G.J., et al. (2003). Parkin-deficient mice exhibit nigrostriatal deficits but not loss of dopaminergic neurons. J. Biol. Chem. 278, 43628–43635.

Greene, J.C., Whitworth, A.J., Kuo, I., Andrews, L.A., Feany, M.B., and Pallanck, L.J. (2003). Mitochondrial pathology and apoptotic muscle degeneration in Drosophila parkin mutants. Proc. Natl. Acad. Sci. USA 100, 4078–4083.

Greene, J.C., Whitworth, A.J., Andrews, L.A., Parker, T.J., and Pallanck, L.J. (2005). Genetic and genomic studies of Drosophila parkin mutants implicate oxidative stress and innate immune responses in pathogenesis. Hum. Mol. Genet. 14, 799–811.

Greene, A.W., Grenier, K., Aguileta, M.A., Muise, S., Farazifard, R., Haque, M.E., McBride, H.M., Park, D.S., and Fon, E.A. (2012). Mitochondrial processing peptidase regulates PINK1 processing, import and Parkin recruitment. EMBO Rep. 13, 378–385.

Hämäläinen, R.H., Manninen, T., Koivumäki, H., Kislin, M., Otonkoski, T., and Suomalainen, A. (2013). Tissue- and cell-type-specific manifestations of heteroplasmic mtDNA 3243A>G mutation in human induced pluripotent stem cell-derived disease model. Proc. Natl. Acad. Sci. USA 110, E3622–E3630.

Hasson, S.A., Kane, L.A., Yamano, K., Huang, C.H., Sliter, D.A., Buehler, E., Wang, C., Heman-Ackah, S.M., Hessa, T., Guha, R., et al. (2013). High-content genome-wide RNAi screens identify regulators of parkin upstream of mitophagy. Nature 504, 291–295.

Hattori, N., Kitada, T., Matsumine, H., Asakawa, S., Yamamura, Y., Yoshino, H., Kobayashi, T., Yokochi, M., Wang, M., Yoritaka, A., et al. (1998a). Molecular genetic analysis of a novel Parkin gene in Japanese families with autosomal recessive juvenile parkinsonism: evidence for variable homozygous deletions in the Parkin gene in affected individuals. Ann. Neurol. 44, 935–941.

Hattori, N., Matsumine, H., Asakawa, S., Kitada, T., Yoshino, H., Elibol, B., Brookes, A.J., Yamamura, Y., Kobayashi, T., Wang, M., et al. (1998b). Point mutations (Thr240Arg and Glu311Stop) in the Parkin gene. Biochem. Biophys. Res. Commun. 249, 754–758.

Heeman, B., Van den Haute, C., Aelvoet, S.A., Valsecchi, F., Rodenburg, R.J., Reumers, V., Debyser, Z., Callewaert, G., Koopman, W.J., Willems, P.H., and Baekelandt, V. (2011). Depletion of PINK1 affects mitochondrial metabolism, calcium homeostasis and energy maintenance. J. Cell Sci. 124, 1115–1125.

Hertz, N.T., Berthet, A., Sos, M.L., Thorn, K.S., Burlingame, A.L., Nakamura, K., and Shokat, K.M. (2013). A neo-substrate that amplifies catalytic activity of parkinson’s-disease-related kinase PINK1. Cell 154, 737–747.

Hollville, E., Carroll, R.G., Cullen, S.P., and Martin, S.J. (2014). Bcl-2 family proteins participate in mitochondrial quality control by regulating Parkin/PINK1-dependent mitophagy. Mol. Cell 55, 451–466. [http://dx.doi.org/10.1016/j.molcel.2014.06.001](http://dx.doi.org/10.1016/j.molcel.2014.06.001).

Houlden, H., and Singleton, A.B. (2012). The genetics and neuropathology of Parkinson’s disease. Acta Neuropathol. 124, 325–338.

Hristova, V.A., Beasley, S.A., Rylett, R.J., and Shaw, G.S. (2009). Identification of a novel Zn2+-binding domain in the autosomal recessive juvenile Parkinson-related E3 ligase parkin. J. Biol. Chem. 284, 14978–14986.

Huang, C., Andres, A.M., Ratliff, E.P., Hernandez, G., Lee, P., and Gottlieb, R.A. (2011). Preconditioning involves selective mitophagy mediated by Parkin and p62/SQSTM1. PLoS ONE 6, e20975.

Iguchi, M., Kujuro, Y., Okatsu, K., Koyano, F., Kosako, H., Kimura, M., Suzuki, N., Uchiyama, S., Tanaka, K., and Matsuda, N. (2013). Parkin-catalyzed ubiquitin-ester transfer is triggered by PINK1-dependent phosphorylation. J. Biol. Chem. 288, 22019–22032.

Itakura, E., and Mizushima, N. (2011). p62 Targeting to the autophagosome formation site requires self-oligomerization but not LC3 binding. J. Cell Biol. 192, 17–27.

Itakura, E., Kishi-Itakura, C., Koyama-Honda, I., and Mizushima, N. (2012). Structures containing Atg9A and the ULK1 complex independently target depolarized mitochondria at initial stages of Parkin-mediated mitophagy. J. Cell Sci. 125, 1488–1499.

Itier, J.M., Ibanez, P., Mena, M.A., Abbas, N., Cohen-Salmon, C., Bohme, G.A., Laville, M., Pratt, J., Corti, O., Pradier, L., et al. (2003). Parkin gene inactivation alters behaviour and dopamine neurotransmission in the mouse. Hum. Mol. Genet. 12, 2277–2291.

Ivatt, R.M., Sanchez-Martinez, A., Godena, V.K., Brown, S., Ziviani, E., and Whitworth, A.J. (2014). Genome-wide RNAi screen identifies the Parkinson disease GWAS risk locus SREBF1 as a regulator of mitophagy. Proc. Natl. Acad. Sci. USA 111, 8494–8499.

Javitch, J.A., D’Amato, R.J., Strittmatter, S.M., and Snyder, S.H. (1985). Parkinsonism-inducing neurotoxin, N-methyl-4-phenyl-1,2,3,6 -tetrahydropyridine: uptake of the metabolite N-methyl-4-phenylpyridine by dopamine neurons explains selective toxicity. Proc. Natl. Acad. Sci. USA 82, 2173–2177.

Jin, S.M., and Youle, R.J. (2013). The accumulation of misfolded proteins in the mitochondrial matrix is sensed by PINK1 to induce PARK2/Parkin-mediated mitophagy of polarized mitochondria. Autophagy 9, 1750–1757.

Jin, S.M., Lazarou, M., Wang, C., Kane, L.A., Narendra, D.P., and Youle, R.J. (2010). Mitochondrial membrane potential regulates PINK1 import and proteolytic destabilization by PARL. J. Cell Biol. 191, 933–942.

Johnson, B.N., Berger, A.K., Cortese, G.P., and Lavoie, M.J. (2012). The ubiquitin E3 ligase parkin regulates the proapoptotic function of Bax. Proc. Natl. Acad. Sci. USA 109, 6283–6288.

Kageyama, Y., Zhang, Z., Roda, R., Fukaya, M., Wakabayashi, J., Wakabayashi, N., Kensler, T.W., Reddy, P.H., Iijima, M., and Sesaki, H. (2012). Mitochondrial division ensures the survival of postmitotic neurons by suppressing oxidative damage. J. Cell Biol. 197, 535–551.

Kane, L.A., Lazarou, M., Fogel, A.I., Li, Y., Yamano, K., Sarraf, S.A., Banerjee, S., and Youle, R.J. (2014). PINK1 phosphorylates ubiquitin to activate Parkin E3 ubiquitin ligase activity. J. Cell Biol. 205, 143–153.

Katayama, H., Kogure, T., Mizushima, N., Yoshimori, T., and Miyawaki, A. (2011). A sensitive and quantitative technique for detecting autophagic events based on lysosomal delivery. Chem. Biol. 18, 1042–1052.

Kazlauskaite, A., Kelly, V., Johnson, C., Baillie, C., Hastie, C.J., Peggie, M., Macartney, T., Woodroof, H.I., Alessi, D.R., Pedrioli, P.G., and Muqit, M.M. (2014a). Phosphorylation of Parkin at Serine65 is essential for activation: elaboration of a Miro1 substrate-based assay of Parkin E3 ligase activity. Open Biol. 4, 130213.

Kazlauskaite, A., Kondapalli, C., Gourlay, R., Campbell, D.G., Ritorto, M.S., Hofmann, K., Alessi, D.R., Knebel, A., Trost, M., and Muqit, M.M. (2014b). Parkin is activated by PINK1-dependent phosphorylation of ubiquitin at Ser65. Biochem. J. 460, 127–139.

Kim, N.C., Tresse, E., Kolaitis, R.M., Molliex, A., Thomas, R.E., Alami, N.H., Wang, B., Joshi, A., Smith, R.B., Ritson, G.P., et al. (2013). VCP is essential for mitochondrial quality control by PINK1/Parkin and this function is impaired by VCP mutations. Neuron 78, 65–80.

Kirkin, V., McEwan, D.G., Novak, I., and Dikic, I. (2009). A role for ubiquitin in selective autophagy. Mol. Cell 34, 259–269.

Kitada, T., Asakawa, S., Hattori, N., Matsumine, H., Yamamura, Y., Minoshima, S., Yokochi, M., Mizuno, Y., and Shimizu, N. (1998). Mutations in the parkin gene cause autosomal recessive juvenile parkinsonism. Nature 392, 605–608.

Kitada, T., Pisani, A., Porter, D.R., Yamaguchi, H., Tscherter, A., Martella, G., Bonsi, P., Zhang, C., Pothos, E.N., and Shen, J. (2007). Impaired dopamine release and synaptic plasticity in the striatum of PINK1-deficient mice. Proc. Natl. Acad. Sci. USA 104, 11441–11446.

Kitada, T., Tong, Y., Gautier, C.A., and Shen, J. (2009). Absence of nigral degeneration in aged parkin/DJ-1/PINK1 triple knockout mice. J. Neurochem. 111, 696–702.

Klein, P., Müller-Rischart, A.K., Motori, E., Schön Bauer, C., Schnorrer, F., Winklhof er, K.F., and Klein, R. (2014). Ret rescues mitochondrial morphology and muscle degeneration of Drosophila Pink1 mutants. EMBO J. 33, 341–355.

Kondapalli, C., Kazlauskaite, A., Zhang, N., Woodroof, H.I., Campbell, D.G., Gourlay, R., Burchell, L., Walden, H., Macartney, T.J., Deak, M., et al. (2012). PINK1 is activated by mitochondrial membrane potential depolarization and stimulates Parkin E3 ligase activity by phosphorylating Serine 65. Open Biol. 2, 120080.

Koyano, F., Okatsu, K., Kosako, H., Tamura, Y., Go, E., Kimura, M., Kimura, Y., Tsuchiya, H., Yoshihara, H., Hirokawa, T., et al. (2014). Ubiquitin is phosphorylated by PINK1 to activate parkin. Nature 510, 162–166.

Kraytsberg, Y., Kudryavtseva, E., McKee, A.C., Geula, C., Kowall, N.W., and Khrapko, K. (2006). Mitochondrial DNA deletions are abundant and cause functional impairment in aged human substantia nigra neurons. Nat. Genet. 38, 518–520.

Krick, R., Muehe, Y., Prick, T., Bremer, S., Schlotterhose, P., Eskelinen, E.L., Millen, J., Goldfarb, D.S., and Thumm, M. (2008). Piecemeal microautophagy of the nucleus requires the core macroautophagy genes. Mol. Biol. Cell 19, 4492–4505.

Kubli, D.A., Zhang, X., Lee, Y., Hanna, R.A., Quinsay, M.N., Nguyen, C.K., Jimenez, R., Petrosyan, S., Murphy, A.N., and Gustafsson, A.B. (2013). Parkin protein deficiency exacerbates cardiac injury and reduces survival following myocardial infarction. J. Biol. Chem. 288, 915–926.

Lang, A.E., and Lozano, A.M. (1998a). Parkinson’s disease. First of two parts. N. Engl. J. Med. 339, 1044–1053.

Lang, A.E., and Lozano, A.M. (1998b). Parkinson’s disease. Second of two parts. N. Engl. J. Med. 339, 1130–1143.

Langston, J.W., Ballard, P., Tetrud, J.W., and Irwin, I. (1983). Chronic Parkinsonism in humans due to a product of meperidine-analog synthesis. Science 219, 979–980.

Lazarou, M., Jin, S.M., Kane, L.A., and Youle, R.J. (2012). Role of PINK1 binding to the TOM complex and alternate intracellular membranes in recruitment and activation of the E3 ligase Parkin. Dev. Cell 22, 320–333.

Lazarou, M., Narendra, D.P., Jin, S.M., Tekle, E., Banerjee, S., and Youle, R.J. (2013). PINK1 drives Parkin self-association and HECT-like E3 activity upstream of mitochondrial binding. J. Cell Biol. 200, 163–172.

Lee, J.Y., Nagano, Y., Taylor, J.P., Lim, K.L., and Yao, T.P. (2010). Disease-causing mutations in parkin impair mitochondrial ubiquitination, aggregation, and HDAC6-dependent mitophagy. J. Cell Biol. 189, 671–679.

Lee, S., Sterky, F.H., Mourier, A., Terzioglu, M., Cullheim, S., Olson, L., and Larsson, N.G. (2012). Mitofusin 2 is necessary for striatal axonal projections of midbrain dopamine neurons. Hum. Mol. Genet. 21, 4827–4835.

Lefebvre, V., Du, Q., Baird, S., Ng, A.C., Nascimento, M., Campanella, M., McBride, H.M., and Screaton, R.A. (2013). Genome-wide RNAi screen identifies ATPase inhibitory factor 1 (ATPIF1) as essential for PARK2 recruitment and mitophagy. Autophagy 9, 1770–1779.

Leroy, E., Anastasopoulos, D., Konitsiotis, S., Lavedan, C., and Polymeropoulos, M.H. (1998). Deletions in the Parkin gene and genetic heterogeneity in a Greek family with early onset Parkinson’s disease. Hum. Genet. 103, 424–427.

Lin, W., and Kang, U.J. (2008). Characterization of PINK1 processing, stability, and subcellular localization. J. Neurochem. 106, 464–474.

Liu, H.H., Tsai, M.C., Chen, C.J., Jeng, J.S., Chang, Y.C., Chen, S.Y., and Chen, R.C. (1997). Environmental risk factors and Parkinson’s disease: a case-control study in Taiwan. Neurology 48, 1583–1588.

Liu, W., Acín-Peréz, R., Geghman, K.D., Manfredi, G., Lu, B., and Li, C. (2011). Pink1 regulates the oxidative phosphorylation machinery via mitochondrial fission. Proc. Natl. Acad. Sci. USA 108, 12920–12924.

Liu, S., Sawada, T., Lee, S., Yu, W., Silverio, G., Alapatt, P., Millan, I., Shen, A., Saxton, W., Kanao, T., et al. (2012). Parkinson’s disease-associated kinase PINK1 regulates Miro protein level and axonal transport of mitochondria. PLoS Genet. 8, e1002537.

Lücking, C.B., Abbas, N., Dürr, A., Bonifati, V., Bonnet, A.M., de Broucker, T., De Michele, G., Wood, N.W., Agid, Y., and Brice, A.; The European Consortium on Genetic Susceptibility in Parkinson’s Disease and the French Parkinson’s Disease Genetics Study Group (1998). Homozygous deletions in parkin gene in European and North African families with autosomal recessive juvenile parkinsonism. Lancet 352, 1355–1356.

Luoma, P., Melberg, A., Rinne, J.O., Kaukonen, J.A., Nupponen, N.N., Chalmers, R.M., Oldfors, A., Rautakorpi, I., Peltonen, L., Majamaa, K., et al. (2004). Parkinsonism, premature menopause, and mitochondrial DNA polymerase gamma mutations: clinical and molecular genetic study. Lancet 364, 875–882.

Manzanillo, P.S., Ayres, J.S., Watson, R.O., Collins, A.C., Souza, G., Rae, C.S., Schneider, D.S., Nakamura, K., Shiloh, M.U., and Cox, J.S. (2013). The ubiquitin ligase parkin mediates resistance to intracellular pathogens. Nature 501, 512–516.

Mao, K., Wang, K., Liu, X., and Klionsky, D.J. (2013). The scaffold protein Atg11 recruits fission machinery to drive selective mitochondria degradation by autophagy. Dev. Cell 26, 9–18.

Marín, I. (2009). RBR ubiquitin ligases: diversification and streamlining in animal lineages. J. Mol. Evol. 69, 54–64.

Matsuda, N., Sato, S., Shiba, K., Okatsu, K., Saisho, K., Gautier, C.A., Sou, Y.S., Saiki, S., Kawajiri, S., Sato, F., et al. (2010). PINK1 stabilized by mitochondrial depolarization recruits Parkin to damaged mitochondria and activates latent Parkin for mitophagy. J. Cell Biol. 189, 211–221.

Matsumine, H., Saito, M., Shimoda-Matsubayashi, S., Tanaka, H., Ishikawa, A., Nakagawa-Hattori, Y., Yokochi, M., Kobayashi, T., Igarashi, S., Takano, H., et al. (1997). Localization of a gene for an autosomal recessive form of

270 Neuron 85, January 21, 2015 ©2015 Elsevier Inc.

juvenile Parkinsonism to chromosome 6q25.2-27. Am. J. Hum. Genet. 60, 588–596.

McCoy, M.K., Kaganovich, A., Rudenko, I.N., Ding, J., and Cookson, M.R. (2014). Hexokinase activity is required for recruitment of parkin to depolarized mitochondria. Hum. Mol. Genet. 23, 145–156.

McLelland, G.L., Soubannier, V., Chen, C.X., McBride, H.M., and Fon, E.A. (2014). Parkin and PINK1 function in a vesicular trafficking pathway regulating mitochondrial quality control. EMBO J. 33, 282–295.

Meissner, C., Lorenz, H., Weihofen, A., Selkoe, D.J., and Lemberg, M.K. (2011). The mitochondrial intramembrane protease PARL cleaves human Pink1 to regulate Pink1 trafficking. J. Neurochem. 117, 856–867.

Meng, F., Yao, D., Shi, Y., Kabakoff, J., Wu, W., Reicher, J., Ma, Y., Moosmann, B., Masliah, E., Lipton, S.A., and Gu, Z. (2011). Oxidation of the cysteine-rich regions of parkin perturbs its E3 ligase activity and contributes to protein aggregation. Mol. Neurodegener. 6, 34.

Michiorri, S., Gelmetti, V., Giarda, E., Lombardi, F., Romano, F., Marongiu, R., Nerini-Molteni, S., Sale, P., Vago, R., Arena, G., et al. (2010). The Parkinson-associated protein PINK1 interacts with Beclin1 and promotes autophagy. Cell Death Differ. 17, 962–974.

Mira, M.T., Alcais, A., Nguyen, V.T., Moraes, M.O., Di Flumeri, C., Vu, H.T., Mai, C.P., Nguyen, T.H., Nguyen, N.B., Pham, X.K., et al. (2004). Susceptibility to leprosy is associated with PARK2 and PACRG. Nature 427, 636–640.

Mitsuhashi, S., Hatakeyama, H., Karahashi, M., Koumura, T., Nonaka, I., Hayashi, Y.K., Noguchi, S., Sher, R.B., Nakagawa, Y., Manfredi, G., et al. (2011). Muscle choline kinase beta defect causes mitochondrial dysfunction and increased mitophagy. Hum. Mol. Genet. 20, 3841–3851.

Morais, V.A., Verstreken, P., Roethig, A., Smet, J., Snellinx, A., Vanbrabant, M., Haddad, D., Frezza, C., Mandemakers, W., Vogt-Weisenhorn, D., et al. (2009). Parkinson’s disease mutations in PINK1 result in decreased Complex I activity and deficient synaptic function. EMBO Mol. Med. 1, 99–111.

Müller-Rischart, A.K., Pils, A., Beaudette, P., Patra, M., Hadian, K., Funke, M., Peis, R., Deinlein, A., Schweimer, C., Kuhn, P.H., et al. (2013). The E3 ligase parkin maintains mitochondrial integrity by increasing linear ubiquitination of NEMO. Mol. Cell 49, 908–921.

Narendra, D., Tanaka, A., Suen, D.F., and Youle, R.J. (2008). Parkin is recruited selectively to impaired mitochondria and promotes their autophagy. J. Cell Biol. 183, 795–803.

Narendra, D., Kane, L.A., Hauser, D.N., Fearnley, I.M., and Youle, R.J. (2010a). p62/SQSTM1 is required for Parkin-induced mitochondrial clustering but not mitophagy; VDAC1 is dispensable for both. Autophagy 6, 1090–1106.

Narendra, D.P., Jin, S.M., Tanaka, A., Suen, D.F., Gautier, C.A., Shen, J., Cookson, M.R., and Youle, R.J. (2010b). PINK1 is selectively stabilized on impaired mitochondria to activate Parkin. PLoS Biol. 8, e1000298.

Narendra, D., Walker, J.E., and Youle, R. (2012). Mitochondrial quality control mediated by PINK1 and Parkin: links to parkinsonism. Cold Spring Harb. Perspect. Biol. 4, a011338.

Neutzner, A., Benard, G., Youle, R.J., and Karbowski, M. (2008). Role of the ubiquitin conjugation system in the maintenance of mitochondrial homeostasis. Ann. N Y Acad. Sci. 1147, 242–253.

Nicklas, W.J., Vyas, I., and Heikkila, R.E. (1985). Inhibition of NADH-linked oxidation in brain mitochondria by 1-methyl-4-phenyl-pyridine, a metabolite of the neurotoxin, 1-methyl-4-phenyl-1,2,5,6-tetrahydropyridine. Life Sci. 36, 2503–2508.

Okatsu, K., Saisho, K., Shimanuki, M., Nakada, K., Shitara, H., Sou, Y.S., Kimura, M., Sato, S., Hattori, N., Komatsu, M., et al. (2010). p62/SQSTM1 cooperates with Parkin for perinuclear clustering of depolarized mitochondria. Genes Cells 15, 887–900.

Okatsu, K., Uno, M., Koyano, F., Go, E., Kimura, M., Oka, T., Tanaka, K., and Matsuda, N. (2013). A dimeric PINK1-containing complex on depolarized mitochondria stimulates Parkin recruitment. J. Biol. Chem. 288, 36372–36384.

Oliveras-Salvá, M., Macchi, F., Coessens, V., Deleersnijder, A., Gérard, M., Van der Perren, A., Van den Haute, C., and Baekelandt, V. (2014). Alpha-synuclein-induced neurodegeneration is exacerbated in PINK1 knockout mice. Neurobiol. Aging 35, 2625–2636.

Ordureau, A., Sarraf, S.A., Duda, D.M., Heo, J.M., Jedrychowski, M.P., Sviderskiy, V.O., Olszewski, J.L., Koerber, J.T., Xie, T., Beausoleil, S.A., et al. (2014). Quantitative proteomics reveal a feedforward mechanism for mitochondrial PARKIN translocation and ubiquitin chain synthesis. Mol. Cell 56, 360–375. http://dx.doi.org/10.1016/j.molcel.2014.09.007

Orvedahl, A., Sumpter, R., Jr., Xiao, G., Ng, A., Zou, Z., Tang, Y., Narimatsu, M., Gilpin, C., Sun, Q., Roth, M., et al. (2011). Image-based genome-wide siRNA screen identifies selective autophagy factors. Nature 480, 113–117.

Palacino, J.J., Sagi, D., Goldberg, M.S., Krauss, S., Motz, C., Wacker, M., Klose, J., and Shen, J. (2004). Mitochondrial dysfunction and oxidative damage in parkin-deficient mice. J. Biol. Chem. 279, 18614–18622.

Park, J., Lee, S.B., Lee, S., Kim, Y., Song, S., Kim, S., Bae, E., Kim, J., Shong, M., Kim, J.M., and Chung, J. (2006). Mitochondrial dysfunction in Drosophila PINK1 mutants is complemented by parkin. Nature 441, 1157–1161.

Park, J., Lee, G., and Chung, J. (2009). The PINK1-Parkin pathway is involved in the regulation of mitochondrial remodeling process. Biochem. Biophys. Res. Commun. 378, 518–523.

Pawlyk, A.C., Giasson, B.I., Sampathu, D.M., Perez, F.A., Lim, K.L., Dawson, V.L., Dawson, T.M., Palminter, R.D., Trojanowski, J.Q., and Lee, V.M. (2003). Novel monoclonal antibodies demonstrate biochemical variation of brain parkin with age. J. Biol. Chem. 278, 48120–48128.

Perez, F.A., and Palminter, R.D. (2005). Parkin-deficient mice are not a robust model of parkinsonism. Proc. Natl. Acad. Sci. USA 102, 2174–2179.

Periquet, M., Corti, O., Jacquier, S., and Brice, A. (2005). Proteomic analysis of parkin knockout mice: alterations in energy metabolism, protein handling and synaptic function. J. Neurochem. 95, 1259–1276.

Pesah, Y., Pham, T., Burgess, H., Middlebrooks, B., Verstreken, P., Zhou, Y., Harding, M., Bellen, H., and Mardon, G. (2004). Drosophila parkin mutants have decreased mass and cell size and increased sensitivity to oxygen radical stress. Development 131, 2183–2194.

Pimenta de Castro, I., Costa, A.C., Lam, D., Tufi, R., Fedele, V., Moisoi, N., Dinsdale, D., Deas, E., Loh, S.H., and Martins, L.M. (2012). Genetic analysis of mitochondrial protein misfolding in Drosophila melanogaster. Cell Death Differ. 19, 1308–1316.

Plun-Favreau, H., Klupsch, K., Moisoi, N., Gandhi, S., Kjaer, S., Frith, D., Harvey, K., Deas, E., Harvey, R.J., McDonald, N., et al. (2007). The mitochondrial protease HtrA2 is regulated by Parkinson’s disease-associated kinase PINK1. Nat. Cell Biol. 9, 1243–1252.

Poole, A.C., Thomas, R.E., Andrews, L.A., McBride, H.M., Whitworth, A.J., and Pallanck, L.J. (2008). The PINK1/Parkin pathway regulates mitochondrial morphology. Proc. Natl. Acad. Sci. USA 105, 1638–1643.

Poole, A.C., Thomas, R.E., Yu, S., Vincow, E.S., and Pallanck, L. (2010). The mitochondrial fusion-promoting factor mitofusin is a substrate of the PINK1/parkin pathway. PLoS ONE 5, e10054.

Pramstaller, P.P., Schlossmacher, M.G., Jacques, T.S., Scaravilli, F., Eskelson, C., Pepivani, I., Hedrich, K., Adel, S., Gonzales-McNeal, M., Hilker, R., et al. (2005). Lewy body Parkinson’s disease in a large pedigree with 77 Parkin mutation carriers. Ann. Neurol. 58, 411–422.

Pridgeon, J.W., Olzmann, J.A., Chin, L.S., and Li, L. (2007). PINK1 protects against oxidative stress by phosphorylating mitochondrial chaperone TRAP1. PLoS Biol. 5, e172.

Rakovic, A., Grünewald, A., Kottwitz, J., Brüggemann, N., Pramstaller, P.P., Lohmann, K., and Klein, C. (2011). Mutations in PINK1 and Parkin impair ubiquitination of Mitofusins in human fibroblasts. PLoS ONE 6, e16746.

Rakovic, A., Shurkewitsch, K., Seibler, P., Grünewald, A., Zanon, A., Hagenah, J., Krainc, D., and Klein, C. (2013). Phosphatase and tensin homolog (PTEN)-induced putative kinase 1 (PINK1)-dependent ubiquitination of endogenous Parkin attenuates mitophagy: study in human primary fibroblasts and induced pluripotent stem cell-derived neurons. J. Biol. Chem. 288, 2223–2237.

Ramsay, R.R., and Singer, T.P. (1986). Energy-dependent uptake of N-methyl-4-phenylpyridinium, the neurotoxic metabolite of 1-methyl-4-phenyl-1,2,3,6-tetrahydropyridine, by mitochondria. J. Biol. Chem. 261, 7585–7587.

Rana, A., Rera, M., and Walker, D.W. (2013). Parkin overexpression during aging reduces proteotoxicity, alters mitochondrial dynamics, and extends lifespan. Proc. Natl. Acad. Sci. USA 110, 8638–8643.

Reeve, A., Meagher, M., Lax, N., Simcox, E., Hepplewhite, P., Jaros, E., and Turnbull, D. (2013). The impact of pathogenic mitochondrial DNA mutations on substantia nigra neurons. J. Neurosci. 33, 10790–10801.

Riley, B.E., Lougheed, J.C., Callaway, K., Velasquez, M., Brecht, E., Nguyen, L., Shaler, T., Walker, D., Yang, Y., Regnstrom, K., et al. (2013). Structure and function of Parkin E3 ubiquitin ligase reveals aspects of RING and HECT ligases. Nat. Commun. 4, 1982.

Samaranch, L., Lorenzo-Betancor, O., Arbelo, J.M., Ferrer, I., Lorenzo, E., Irigoyen, J., Pastor, M.A., Marrero, C., Isla, C., Herrera-Henriquez, J., and Pastor, P. (2010). PINK1-linked parkinsonism is associated with Lewy body pathology. Brain 133, 1128–1142.

Sarraf, S.A., Raman, M., Guarani-Pereira, V., Sowa, M.E., Huttlin, E.L., Gygi, S.P., and Harper, J.W. (2013). Landscape of the PARKIN-dependent ubiquitylome in response to mitochondrial depolarization. Nature 496, 372–376.

Scarffe, L.A., Stevens, D.A., Dawson, V.L., and Dawson, T.M. (2014). Parkin and PINK1: much more than mitophagy. Trends Neurosci. 37, 315–324.

Schapira, A.H., Cooper, J.M., Dexter, D., Jenner, P., Clark, J.B., and Marsden, C.D. (1989). Mitochondrial complex I deficiency in Parkinson’s disease. Lancet 1, 1269.

Schapira, A.H., Cooper, J.M., Dexter, D., Clark, J.B., Jenner, P., and Marsden, C.D. (1990). Mitochondrial complex I deficiency in Parkinson’s disease. J. Neurochem. 54, 823–827.

Seibler, P., Graziotto, J., Jeong, H., Simunovic, F., Klein, C., and Krainc, D. (2011). Mitochondrial Parkin recruitment is impaired in neurons derived from mutant PINK1 induced pluripotent stem cells. J. Neurosci. 31, 5970–5976.

Shen, Q., Yamano, K., Head, B.P., Kawajiri, S., Cheung, J.T., Wang, C., Cho, J.H., Hattori, N., Youle, R.J., and van der Bliek, A.M. (2014). Mutations in Fis1 disrupt orderly disposal of defective mitochondria. Mol. Biol. Cell 25, 145–159.

Shiba-Fukushima, K., Imai, Y., Yoshida, S., Ishihama, Y., Kanao, T., Sato, S., and Hattori, N. (2012). PINK1-mediated phosphorylation of the Parkin ubiquitin-like domain primes mitochondrial translocation of Parkin and regulates mitophagy. Sci. Rep. 2, 1002.

Shiba-Fukushima, K., Inoshita, T., Hattori, N., and Imai, Y. (2014). PINK1-mediated phosphorylation of Parkin boosts Parkin activity in Drosophila. PLoS Genet. 10, e1004391.

Shimura, H., Hattori, N., Kubo, Si., Mizuno, Y., Asakawa, S., Minoshima, S., Shimizu, N., Iwai, K., Chiba, T., Tanaka, K., and Suzuki, T. (2000). Familial Parkinson disease gene product, parkin, is a ubiquitin-protein ligase. Nat. Genet. 25, 302–305.

Shin, J.H., Ko, H.S., Kang, H., Lee, Y., Lee, Y.I., Pletinkova, O., Troconso, J.C., Dawson, V.L., and Dawson, T.M. (2011). PARIS (ZNF746) repression of PGC-1α contributes to neurodegeneration in Parkinson’s disease. Cell 144, 689–702.

Sims, J.J., Scavone, F., Cooper, E.M., Kane, L.A., Youle, R.J., Boeke, J.D., and Cohen, R.E. (2012). Polyubiquitin-sensor proteins reveal localization and linkage-type dependence of cellular ubiquitin signaling. Nat. Methods 9, 303–309.

Smit, J.J., Monteferrario, D., Noordermeer, S.M., van Dijk, W.J., van der Reijden, B.A., and Sixma, T.K. (2012). The E3 ligase HOIP specifies linear ubiquitin chain assembly through its RING-IBR-RING domain and the unique LDD extension. EMBO J. 31, 3833–3844.

Spillantini, M.G., Crowther, R.A., Jakes, R., Hasegawa, M., and Goedert, M. (1998). alpha-Synuclein in filamentous inclusions of Lewy bodies from Parkinson’s disease and dementia with lewy bodies. Proc. Natl. Acad. Sci. USA 95, 6469–6473.

Spratt, D.E., Martinez-Torres, R.J., Noh, Y.J., Mercier, P., Manczyk, N., Barber, K.R., Aguirre, J.D., Burchell, L., Purkiss, A., Walden, H., and Shaw,

G.S. (2013). A molecular explanation for the recessive nature of parkin-linked Parkinson’s disease. Nat. Commun. 4, 1983.

Sterky, F.H., Lee, S., Wibom, R., Olson, L., and Larsson, N.G. (2011). Impaired mitochondrial transport and Parkin-independent degeneration of respiratory chain-deficient dopamine neurons in vivo. Proc. Natl. Acad. Sci. USA 108, 12937–12942.

Stichel, C.C., Zhu, X.R., Bader, V., Linnartz, B., Schmidt, S., and Lübbert, H. (2007). Mono- and double-mutant mouse models of Parkinson’s disease display severe mitochondrial damage. Hum. Mol. Genet. 16, 2377–2393.

Strappazzon, F., Nazio, F., Corrado, M., Cianfanelli, V., Romagnoli, A., Fimia, G.M., Campello, S., Nardacci, R., Piacentini, M., Campanella, M., and Cecconi, F. (2014). AMBRA1 is able to induce mitophagy via LC3 binding, regardless of PARKIN and p62/SQSTM1. Cell Death Differ.. http://dx.doi.org/10.1038/cdd.2014.139.

Suen, D.F., Narendra, D.P., Tanaka, A., Manfredi, G., and Youle, R.J. (2010). Parkin overexpression selects against a deleterious mtDNA mutation in heteroplasmic cybrid cells. Proc. Natl. Acad. Sci. USA 107, 11835–11840.

Takamura, A., Komatsu, M., Hara, T., Sakamoto, A., Kishi, C., Waguri, S., Eishi, Y., Hino, O., Tanaka, K., and Mizushima, N. (2011). Autophagy-deficient mice develop multiple liver tumors. Genes Dev. 25, 795–800.

Tanaka, A., Cleland, M.M., Xu, S., Narendra, D.P., Suen, D.F., Karbowski, M., and Youle, R.J. (2010). Proteasome and p97 mediate mitophagy and degradation of mitofusins induced by Parkin. J. Cell Biol. 191, 1367–1380.

Tanner, C.M., Kamel, F., Ross, G.W., Hoppin, J.A., Goldman, S.M., Korell, M., Marras, C., Bhudhikanok, G.S., Kasten, M., Chade, A.R., et al. (2011). Rotenone, paraquat, and Parkinson’s disease. Environ. Health Perspect. 119, 866–872.

Thomas, R.E., Andrews, L.A., Burman, J.L., Lin, W.Y., and Pallanck, L.J. (2014). PINK1-Parkin pathway activity is regulated by degradation of PINK1 in the mitochondrial matrix. PLoS Genet. 10, e1004279.

Trempe, J.F., Sauvé, V., Grenier, K., Seirafi, M., Tang, M.Y., Ménade, M., Al-Abdul-Wahid, S., Krett, J., Wong, K., Kozlov, G., et al. (2013). Structure of parkin reveals mechanisms for ubiquitin ligase activation. Science 340, 1451–1455.

Twig, G., and Shirihai, O.S. (2011). The interplay between mitochondrial dynamics and mitophagy. Antioxid. Redox Signal. 14, 1939–1951.

Twig, G., Elorza, A., Molina, A.J., Mohamed, H., Wikstrom, J.D., Walzer, G., Stiles, L., Haigh, S.E., Katz, S., Las, G., et al. (2008). Fission and selective fusion govern mitochondrial segregation and elimination by autophagy. EMBO J. 27, 433–446.

Unoki, M., and Nakamura, Y. (2001). Growth-suppressive effects of BPOZ and EGR2, two genes involved in the PTEN signaling pathway. Oncogene 20, 4457–4465.

Urban, S., and Freeman, M. (2003). Substrate specificity of rhomboid intramembrane proteases is governed by helix-breaking residues in the substrate transmembrane domain. Mol. Cell 11, 1425–1434.

Valente, E.M., Bentivoglio, A.R., Dixon, P.H., Ferraris, A., lalongo, T., Frontali, M., Albanese, A., and Wood, N.W. (2001). Localization of a novel locus for autosomal recessive early-onset parkinsonism, PARK6, on human chromosome 1p35-p36. Am. J. Hum. Genet. 68, 895–900.

Valente, E.M., Brancati, F., Caputo, V., Graham, E.A., Davis, M.B., Ferraris, A., Breteler, M.M.B., Gasser, T., Bonifati, V., Bentivoglio, A.R., et al.; European Consortium on Genetic Susceptibility in Parkinson’s Disease (2002a). PARK6 is a common cause of familial parkinsonism. Neurol. Sci. 23 (Suppl 2), S117–S118.

Valente, E.M., Brancati, F., Ferraris, A., Graham, E.A., Davis, M.B., Breteler, M.M.B., Gasser, T., Bonifati, V., Bentivoglio, A.R., De Michele, G., et al.; European Consortium on Genetic Susceptibility in Parkinson’s Disease (2002b). PARK6-linked parkinsonism occurs in several European families. Ann. Neurol. 51, 14–18.

Valente, E.M., Abou-Sleiman, P.M., Caputo, V., Muqit, M.M., Harvey, K., Gispert, S., Ali, Z., Del Turco, D., Bentivoglio, A.R., Healy, D.G., et al. (2004). Hereditary early-onset Parkinson’s disease caused by mutations in PINK1. Science 304, 1158–1160.

Van Humbeeck, C., Cornelissen, T., Hofkens, H., Mandemakers, W., Gevaert, K., De Strooper, B., and Vandenberghe, W. (2011). Parkin interacts with Ambra1 to induce mitophagy. J. Neurosci. 31, 10249–10261.

van Wijk, S.J., Fiskin, E., Putyrski, M., Pampaloni, F., Hou, J., Wild, P., Kensche, T., Grecco, H.E., Bastiaens, P., and Dikic, I. (2012). Fluorescence-based sensors to monitor localization and functions of linear and K63-linked ubiquitin chains in cells. Mol. Cell 47, 797–809.

Vandiver, M.S., Paul, B.D., Xu, R., Karuppagounder, S., Rao, F., Snowman, A.M., Ko, H.S., Lee, Y.I., Dawson, V.L., Dawson, T.M., et al. (2013). Sulfhydration mediates neuroprotective actions of parkin. Nat. Commun. 4, 1626.

Vilain, S., Esposito, G., Haddad, D., Schaap, O., Dobreva, M.P., Vos, M., Van Meensel, S., Morais, V.A., De Strooper, B., and Verstreken, P. (2012). The yeast complex I equivalent NADH dehydrogenase rescues pink1 mutants. PLoS Genet. 8, e1002456.

Vincow, E.S., Merrihew, G., Thomas, R.E., Shulman, N.J., Beyer, R.P., MacCoss, M.J., and Pallanck, L.J. (2013). The PINK1-Parkin pathway promotes both mitophagy and selective respiratory chain turnover in vivo. Proc. Natl. Acad. Sci. USA 110, 6400–6405.

Vives-Bauza, C., Zhou, C., Huang, Y., Cui, M., de Vries, R.L., Kim, J., May, J., Tocilescu, M.A., Liu, W., Ko, H.S., et al. (2010). PINK1-dependent recruitment of Parkin to mitochondria in mitophagy. Proc. Natl. Acad. Sci. USA 107, 378–383.

von Coelln, R., Thomas, B., Savitt, J.M., Lim, K.L., Sasaki, M., Hess, E.J., Dawson, V.L., and Dawson, T.M. (2004). Loss of locus coeruleus neurons and reduced startle in parkin null mice. Proc. Natl. Acad. Sci. USA 101, 10744–10749.

von Coelln, R., Thomas, B., Andrabi, S.A., Lim, K.L., Savitt, J.M., Saffary, R., Stirling, W., Bruno, K., Hess, E.J., Lee, M.K., et al. (2006). Inclusion body formation and neurodegeneration are parkin independent in a mouse model of alpha-synucleinopathy. J. Neurosci. 26, 3685–3696.

Vos, M., Esposito, G., Edirisinghe, J.N., Vilain, S., Haddad, D.M., Slabbaert, J.R., Van Meensel, S., Schaap, O., De Strooper, B., Meganathan, R., et al. (2012). Vitamin K2 is a mitochondrial electron carrier that rescues pink1 deficiency. Science 336, 1306–1310.

Wang, X., Winter, D., Ashrafi, G., Schlehe, J., Wong, Y.L., Selkoe, D., Rice, S., Steen, J., LaVoie, M.J., and Schwarz, T.L. (2011). PINK1 and Parkin target Miro for phosphorylation and degradation to arrest mitochondrial motility. Cell 147, 893–906.

Wang, Y., Nartiss, Y., Steipe, B., McQuibban, G.A., and Kim, P.K. (2012). ROS-induced mitochondrial depolarization initiates PARK2/PARKIN-dependent mitochondrial degradation by autophagy. Autophagy 8, 1462–1476.

Wauer, T., and Komander, D. (2013). Structure of the human Parkin ligase domain in an autoinhibited state. EMBO J. 32, 2099–2112.

Wenzel, D.M., Lissounov, A., Brzovic, P.S., and Klevit, R.E. (2011). UBCH7 reactivity profile reveals parkin and HHARI to be RING/HECT hybrids. Nature 474, 105–108.

Whitworth, A.J., Theodore, D.A., Greene, J.C., Benes, H., Wes, P.D., and Pallanck, L.J. (2005). Increased glutathione S-transferase activity rescues dopaminergic neuron loss in a Drosophila model of Parkinson’s disease. Proc. Natl. Acad. Sci. USA 102, 8024–8029.

Winklhofer, K.F. (2014). Parkin and mitochondrial quality control: toward assembling the puzzle. Trends Cell Biol. 24, 332–341.

Winslow, A.R., Chen, C.W., Corrochano, S., Acevedo-Arozena, A., Gordon, D.E., Peden, A.A., Lichtenberg, M., Menzies, F.M., Ravikumar, B., Imarisio, S., et al. (2010). α-Synuclein impairs macroautophagy: implications for Parkinson’s disease. J. Cell Biol. 190, 1023–1037.

Wolter, K.G., Hsu, Y.T., Smith, C.L., Nechushtan, A., Xi, X.G., and Youle, R.J. (1997). Movement of Bax from the cytosol to mitochondria during apoptosis. J. Cell Biol. 139, 1281–1292.

Yamano, K., and Youle, R.J. (2013). PINK1 is degraded through the N-end rule pathway. Autophagy 9, 1758–1769.

Yamano, K., Fogel, A.I., Wang, C., van der Bliek, A.M., and Youle, R.J. (2014). Mitochondrial Rab GAPs govern autophagosome biogenesis during mitophagy. eLife 3, e01612.

Yang, J.Y., and Yang, W.Y. (2011). Spatiotemporally controlled initiation of Parkin-mediated mitophagy within single cells. Autophagy 7, 1230–1238.

Yang, J.Y., and Yang, W.Y. (2013). Bit-by-bit autophagic removal of parkin-labelled mitochondria. Nat. Commun. 4, 2428.

Yang, Y., Gehrke, S., Imai, Y., Huang, Z., Ouyang, Y., Wang, J.W., Yang, L., Beal, M.F., Vogel, H., and Lu, B. (2006). Mitochondrial pathology and muscle and dopaminergic neuron degeneration caused by inactivation of Drosophila Pink1 is rescued by Parkin. Proc. Natl. Acad. Sci. USA 103, 10793–10798.

Yang, Y., Ouyang, Y., Yang, L., Beal, M.F., McQuibban, A., Vogel, H., and Lu, B. (2008). Pink1 regulates mitochondrial dynamics through interaction with the fission/fusion machinery. Proc. Natl. Acad. Sci. USA 105, 7070–7075.

Yoshii, S.R., Kishi, C., Ishihara, N., and Mizushima, N. (2011). Parkin mediates proteasome-dependent protein degradation and rupture of the outer mitochondrial membrane. J. Biol. Chem. 286, 19630–19640.

Youle, R.J., and Narendra, D.P. (2011). Mechanisms of mitophagy. Nat. Rev. Mol. Cell Biol. 12, 9–14.

Youle, R.J., and van der Bliek, A.M. (2012). Mitochondrial fission, fusion, and stress. Science 337, 1062–1065.

Yun, J., Cao, J.H., Dodson, M.W., Clark, I.E., Kapahi, P., Chowdhury, R.B., and Guo, M. (2008). Loss-of-function analysis suggests that Omi/HtrA2 is not an essential component of the PINK1/PARKIN pathway in vivo. J. Neurosci. 28, 14500–14510.

Yun, J., Puri, R., Yang, H., Lizzio, M.A., Wu, C., Sheng, Z.H., and Guo, M. (2014). MUL1 acts in parallel to the PINK1/parkin pathway in regulating mitofusin and compensates for loss of PINK1/parkin. eLife 3, e01958.

Zhang, L., Karsten, P., Hamm, S., Pogson, J.H., Müller-Rischart, A.K., Exner, N., Haass, C., Whitworth, A.J., Winklhofer, K.F., Schulz, J.B., and Voigt, A. (2013). TRAP1 rescues PINK1 loss-of-function phenotypes. Hum. Mol. Genet. 22, 2829–2841.

Zhang, C., Lee, S., Peng, Y., Bunker, E., Giaime, E., Shen, J., Zhou, Z., and Liu, X. (2014). PINK1 triggers autocatalytic activation of Parkin to specify cell fate decisions. Curr. Biol. 24, 1854–1865.

Zhao, Q., Wang, J., Levichkin, I.V., Stasinopoulos, S., Ryan, M.T., and Hoogenraad, N.J. (2002). A mitochondrial specific stress response in mammalian cells. EMBO J. 21, 4411–4419.

Zheng, X., and Hunter, T. (2013). Parkin mitochondrial translocation is achieved through a novel catalytic activity coupled mechanism. Cell Res. 23, 886–897.

Ziviani, E., Tao, R.N., and Whitworth, A.J. (2010). Drosophila parkin requires PINK1 for mitochondrial translocation and ubiquitinates mitofusin. Proc. Natl. Acad. Sci. USA 107, 5018–5023.
