
New Biotechnology • Volume 00, Number 00 • February 2015

**RESEARCH PAPER**

ELSEVIER

---

**Biochemistry, genetics and regulation of bacilysin biosynthesis and its significance more than an antibiotic**

Gülay Özcengiz${}^{1}$ and İsmail Öğülür${}^{2}$

${}^{1}$ Department of Biological Sciences and Molecular Biology and Genetics, Middle East Technical University, 06800 Ankara, Turkey  
${}^{2}$ Marmara University, Research and Training Hospital, Division of Pediatric Allergy and Immunology, 34899 Istanbul, Turkey

**Bacillus subtilis** has the capacity to produce more than two dozen bioactive compounds with an amazing variety of chemical structures. Among them, bacilysin is a non-ribosomally synthesized dipeptide antibiotic consisting of L-alanine residue at the N terminus and a non-proteinogenic amino acid, L-anticapsin, at the C terminus. In spite of its simple structure, it is active against a wide range of bacteria and fungi. As a potent antimicrobial agent, we briefly review the biochemistry and genetics as well as the regulation of bacilysin biosynthesis within the frame of peptide pheromones-based control of secondary activities. Biological functions of bacilysin in the producer *B. subtilis* beyond its antimicrobial activity as well as potential biotechnological use of the biosynthetic enzyme L-amino acid ligase (Lal) are also discussed.

---

### General

The potential of *Bacillus subtilis* to produce more than two dozen bioactive compounds has been recognized for over 50 years with low molecular weight, bioactive peptides constituting the majority. Such peptides have rigid, hydrophobic and/or cyclic components and unusual molecules, like D-amino acids [1]. In principle, there are two distinct pathways leading to the combination of such non-proteinaceous constituents [2]: (i) the non-ribosomal peptides are synthesized by non-ribosomal peptide synthetases (NRPSs), which are large multienzymes catalyzing a series of reactions explained by ‘multicarrier thiotemplate model’, and (ii) the linear precursor peptides synthesized ribosomally are modified post-translationally and processed proteolytically. The non-ribosomal synthesis of peptide antibiotics is commonly seen in bacteria and fungi [3–5]. The NRPSs with modular catalytic domains do catalyze all steps in peptide biosynthesis involving the selection and serial condensation of amino acid residues.

Bacilysin, the simplest peptide antibiotic known, was discovered in as early as 1946 as an antibiotic synthesized by a strain of *B. subtilis* that causes partial lysis of growing cultures of *Staphylococcus aureus* [6]. In the same year, it was independently reported by Foster and Woodruff with the name bacillin, as an antibiotic formed by *B. subtilis* in synthetic medium with an action antagonized by complex media, blood and crude materials [7]. In late 1940s, bacilysin from *B. subtilis* A14 was shown to be a very heat stable bactericidal agent unextractable with organic solvents [8]. The production of bacilysin and its isolation in low yields from culture filtrates of A14 strain was first reported in 1965 [9]. The molecule was described as C₁₂H₁₈N₂O₅ with a molecular weight of 270 Da. Its hydrolysis by means of acid treatment yielded L-alanine and an uncharacterized L-amino acid like a tyrosine derivative [10]. Walker and Abraham isolated bacilysin and its terminal epoxy amino acid in improved yields from the culture fluids and by the application of physicochemical methods, assigned its exact chemical structure as L-alanyl-β-(2,3-epoxyclohexanonyl)-L-alanine [11,12]. The C-terminal amino acid was identical to anticapsin produced by *Streptomyces griseoplanus* which inhibits the formation of hyaluronic acid capsule of group A streptococci [13,14], and thus named anticapsin (Fig. 1b). On the other hand, the dipeptide antibiotic tetaine produced by *Bacillus pumilus* B-180 [15] was reported as chemically and physically identical to bacilysin [16]. Atsumi et al. found bacillin,

---

Corresponding author: Özcengiz, G. (ozcengiz@metu.edu.tr)

http://dx.doi.org/10.1016/j.nbt.2015.01.006

1871-6784/© 2015 Published by Elsevier B.V.

Please cite this article in press as: Özcengiz, G., and Öğülür, I., Biochemistry, genetics and regulation of bacilysin biosynthesis and its significance more than an antibiotic, New Biotechnol. (2015), http://dx.doi.org/10.1016/j.nbt.2015.01.006

NBT 748 1–7

ARTICLE IN PRESS

RESEARCH PAPER

New Biotechnology • Volume 00, Number 00 • February 2015

![Diagram](attachment:diagram.png)

FIGURE 1  
(a) Organization of the bacilysin biosynthetic gene cluster ywfABCDEFG renamed as bacABCDEFG in *Bacillus subtilis* 168. (b) Latest proposed scheme of the biosynthetic pathway from prephenate to bacilysin  
Adapted from Parker and Walsh [44].

bacilysin and tetaine, all synonymous [17]. By monitoring the appearance of antibiotic activity with the same analytical techniques in the culture fluids of *B. subtilis* A14 [18] and Marburg [19], the researchers verified that both strains produce bacilysin after the stage of exponential growth.

Bacilysin is active against a broad range of bacteria and some fungi like *Candida albicans* [20]. Its antimicrobial action was reversed by glucosamine and N-acetyl glucosamine [20,21], demonstrating that it interferes with the synthesis of glucosamine and consequently with cell wall formation. Antibacterial action of bacilysin mainly relies on its transport into host cells, its hydrolysis by intracellular peptidases to anticapsin and the inhibition by the latter of glucosamine 6-phosphate (GlcN6P) synthase [22]. In *S. aureus*, several di- and tripeptides compete with each other and with bacilysin for uptake [23] since there is only a single peptide transport system for all these peptides in this bacterium [24]. This finding explained the former observations that *S. aureus* mutants resistant to bacilysin arise readily, while this is not the case in *Escherichia coli* having both di- and oligopeptide transport systems for bacilysin uptake. Anticapsin is a more powerful inhibitor of GlcN6P synthase in *S. aureus* cell extracts as compared to bacilysin [25,26]. It was clarified by kinetic studies that anticapsin inhibits GlcN6P synthase activity non-competitively with respect to fructose 6-phosphate, but partly competitively with respect to L-glutamine. When the enzyme GlcN6P synthase was incubated with anticapsin, this resulted in a time-dependent and irreversible inhibition. It was then demonstrated that anticapsin is an analogue of glutamine and covalently binds to the enzyme via cross-linking of the epoxy group of anticapsin and a thiol group of the enzyme [27].

### Biochemistry and genetics of bacilysin biosynthesis

Like other peptide antibiotics of the genus *Bacillus*, bacilysin had long been thought to be synthesized by a multifunctional nonribosomal peptide synthetase (NRPS). The first cell-free synthesis of bacilysin was obtained when L-alanine, L-anticapsin, ATP and Mg²⁺ were provided in reaction mixture [28]. L-Alanyl-L-alanine dipeptide was also synthesized in the same reaction mixture. Our group reported partial purification of a biosynthetic enzyme fraction (ca. 125 kDa) by fast flow gel permeation followed by anion exchange FPLC [29]. Enzyme characterization studies, overall, showed that bacilysin biosynthesis does not fit to the general mechanisms of NRPSs. The ATP-PPi exchange reaction of amino acid activation was found to occur with L-alanine, but not with anticapsin. The latter was not activated as an amino acid phosphate, either. In another study, *in vitro* reaction mixtures were

---

www.elsevier.com/locate/nbt

Please cite this article in press as: Özcengiz, G., and Öğülür, I., Biochemistry, genetics and regulation of bacilysin biosynthesis and its significance more than an antibiotic, New Biotechnol. (2015), http://dx.doi.org/10.1016/j.nbt.2015.01.006

prepared to replace anticapsin with furfurylamine, norephedrine, glycine, and benzylamine, respectively. The formation of novel molecules in the reaction mixtures was analyzed by TLC and GC-MS analysis [30]. Among the substrates tested to replace anticapsin, the enzyme could only link furfurylamine to L-alanine to form 2-amino-N-(2-furylmethyl)propanamide. The mechanisms of bacilysin synthesis remained unclear for years.

The analysis of previously characterized auxotrophic mutants of *B. subtilis* with respect to their ability to synthesize bacilysin demonstrated that L-anticapsin moiety of bacilysin is formed from prephenate, an intermediate in the aromatic amino acid biosynthetic pathway [31]. In subsequent studies, an NTG-induced bacilysin nonproducer mutant, namely strain NG79, was isolated and through PBS-1 transduction mapping experiments, the *bac*-lesion was located [32]. Inaoka et al. have performed a very pioneering study by cloning a polycistronic operon, *ywfBCDEFG*, and a monocistronic gene, *ywfH* and showing them to encode bacilysin biosynthetic machinery [33]. Subsequently, the *ywfBCDEF* genes were confirmed to have the core functions of bacilysin biosynthesis and accordingly renamed as *bacABCDE* [34]. The *ywfH* gene is divergently located and transcribed on the opposite DNA strand from *bacABCDE*. Later on, for convenience, *ywfG* and *ywfH* genes were renamed as *bacF* and *bacG*, respectively [35] (Fig. 1a). When *bacD* gene was mutated or *bacABC* genes were transformed into a *B. subtilis Δ(ywfA-bacABCDE)* deletion mutant, anticapsin was found to be accumulated [34]. The organization of *bac* cluster was the same in different species of *Bacillus* with a sequence identity of 72.6–88.6% [34]. Indeed, *bacABCDEFG* are needed for the flux from prephenate to mature bacilysin [44]. *bacE* gene within the same cluster encodes a protein with no biosynthetic function, but this protein is involved in resistance of the producer by pumping bacilysin out of the cell [34].

The details of the action of multiple enzymes in the *bac* operon for the synthesis of bacilysin were elucidated through extensive biochemical and biophysical studies. The most crucial reports resolved the sequential action of BacABGF (four-enzyme pathway) to convert prephenate to 2S, 4S, 7R- and 2S,4R,7R-tetrahydrotyrosine (H₄Tyr) diastereomers on the way to anticapsin [35–37]. To our general knowledge, BacA is responsible for decarboxylation of prephenate and specific isomerization of the pro-R double bond yields the endocyclic dienyl product 7R-endocyclic-dihydroxyphenylpyruvate (H₂HPP) [38]. BacB carries out an isomerization moving the double bond to conjugate with the 2-keto moiety to generate 7R-exocyclic-H₂HPP, [36,39], indeed giving both the 3E- and 3Z-geometric isomers of it [35,37]. Analysis of multiple crystal forms of this enzyme was reported [40]. BacG (YwfH) is a NADPH-dependent reductase for stereoselective reduction of 3E- and 3Z-ex-H₂HPP to tetrahydrohydroxyphenylpyruvate (H₄HPP) 4R- and 4S-diastereomers which are then acted on by BacF and converted to corresponding H₄Tyr diastereomers [35]. BacG structure was described at three conformational steps, providing a rationale for the ping-pong reaction mechanism [41]. BacF catalyzes reductive amination of the 2-keto group of the anticapsin candidate by using L-phenylalanine as an amino group donor [35]. BacC protein, as an NAD⁺-dependent alcohol dehydrogenase oxidizes the C7-hydroxyl by which anticapsin takes its final form. BacD, as a member of the ATP-grasp superfamily of enzymes, is an L-amino acid ligase (Lal) responsible for the dipeptide ligation in

which L-Ala is added to the amino group of anticapsin as an ultimate step [34,42,43].

Regarding the catalysis in between BacB and BacF, the above-explained biosynthetic model was recently subjected to some revisions with an attempt of filling in missing steps and correcting the others [44]. A series of genetic and biochemical experiments revealed that H₄Tyr diastereomers were not actually on bacilysin biosynthesis pathway. Dihydroanticapsin and dihydrobacilysin accumulation in a Δ*bacC* strain and their conversion to anticapsin and bacilysin when pure BacC and BacD proteins were added to the cell extracts were noteworthy. The purified BacC acted on only 4S stereoisomers, as expected from the previously reported stereochemistry [45]. Accordingly, installation of C₅–C₆ double bond epoxide should be the function of an earlier step before C₇ hydroxyl oxidation to the ketone in the latest proposed pathway [44], addressing yet unproven, but the most likely step of epoxidation (Fig. 2b).

### Regulation of bacilysin biosynthesis

It has been well established that culturing the producer in a chemically defined medium leads to bacilysin biosynthesis in a growth-associated manner. Repression and/or inhibition of bacilysin production was observed with certain major nutrients, most typically with nitrogen sources like ammonium and casamino acids [46,47]. Glucose was not inhibitory and carbon sources other than glucose and sucrose yielded lower specific titers of bacilysin. Feedback regulation by bacilysin of its own biosynthesis was also demonstrated [48].

As shown for many different bacterial genera, quorum sensing is the principal mechanism by which bacterial populations respond to their high cell density [49]. When small signaling peptides externalized come to a crucial concentration outside the cell, they start to elicit a regulatory response in Gram-positive bacteria [50]. In *B. subtilis*, the complex physiological processes like sporulation, biofilm formation, competence development, proteolytic enzyme synthesis and peptide antibiotic production are all controlled by quorum sensing pathway, so-called ComQXPA pathway [51]. The onset of the pathway requires the accumulation of cell-derived extracellular peptide pheromones, namely ComX and Competence and Sporulation Factor (CSF) [52–56]. *comX* gene encodes a 55-amino acid protein which is the inactive form of the relevant pheromone. This protein is then subjected to extensive processing and excreted outside as an active decapeptide pheromone [56]. ComP, which is a membrane-bound receptor protein kinase of ComX pheromone is activated by autophosphorylation and controls the response regulator ComA by donating a phosphate to form active ComA–P which is a master player stimulating the transcription of the target genes. ComA is also subjected to negative regulation of RapC, an aspartyl-phosphate phosphatase that removes its phosphate, thereby inactivating it [53,57]. The transport of the other pheromone consisting of only five carboxy terminal amino acids of the 40-amino acid protein encoded by *phrC* gene product, namely CSF, as well as other related peptide pheromones like PhrF and PhrK back to the cell, by the oligopeptide permease Opp (Spo0K). This transporter plays a very crucial role in quorum sensing. Besides its action in stimulating ComA-dependent gene expression, the antagonistic actions of Rap proteins are to be noted and constitute another feature of the circuit as previously discussed [58–60].

NBT 748 1–7

ARTICLE IN PRESS

RESEARCH PAPER

New Biotechnology • Volume 00, Number 00 • February 2015

![Diagram](#)

FIGURE 2

Major controls over bacilysin biosynthetic operon and their interactions with sporulation initiation, competence development and surfactin production, as schematized using the references cited in the text. PhrC (CSF) and ComX pheromones are bold boxed. Positive and negative regulations are shown by arrows and T-bars, respectively, bold lines referring to direct interaction with P_bac*: basal-level transcription and transient-state induction, **: transient-state induction, ***: both bind to P_bac simultaneously. The dependence of bac operon expression on Spo0H (σH), Phr peptides C, F and K, srfA operon and LutR is not shown on this picture, but explained in the text.

In *B. subtilis*, the complex regulatory circuits controlling sporulation and competence highlight two important regulators, Spo0A and AbrB, the former being the master transcriptional regulator positively controlling initiation of sporulation when activated under nutritional stress and high cell density conditions by a phosphorelay including KinA, KinB, and KinC, a phosphorylation pathway starting from Spo0F phosphorylation and ends when the phosphate group is received by Spo0A [61–63]. Spo0A-P binds to promoters of Spo0A regulon genes (more than 120 genes) and directly controls their transcription, altering the expression of up to 500 genes. Spo0A-P has two opposite effects: in its activation effect, it acts on the transcription of sporulation genes, including degQ, rapA, rapC and srfA (mentioned later), but negatively regulates the transcription of abrB gene. When the transition state regulator, AbrB, is depleted from the cell due to Spo0A-mediated repression, genes under this negative control protein are activated [64]. spo0H gene (sporulation regulatory gene) coding for σH which is involved in expression of vegetative and early stationary stage genes including mature PhrC peptide production and the stationary phase overtranscription of phr genes [56,65] is one of the targets for negative regulation by AbrB. More recently, new modulators, positively controlled by Spo0A as well as those inhibiting Spo0A activation were reported [66].

Our group demonstrated that quorum sensing global regulation operates in bacilysin biosynthesis through the action of ComQ/ComX, PhrC (CSF), ComP/ComA in *B. subtilis* [67,68]. In a transposon library screened for bacilysin-negative phenotype, this kind of involvement was significant, suggesting that bacilysin biosynthesis is under quorum sensing pathway [69,70].

phrA, phrC and comA deletion mutants were next constructed to verify that the bacilysin biosynthesis is related to the pheromones ComA and PhrC(CSF) in a manner dependent on Opp (Spo0K) [67]. The surfactin biosynthetic operon, comprising srfAABCD encodes for nonribosomal synthesis of another *B. subtilis* antibiotic, the lipopeptide surfactin [71,72]. Within the srfA gene, the competence regulatory gene (comS) resides [73,74]. Extracellular ComX and the resulting ComQXPA system control the expression of srfA and comS through a complex network [75]. Bacilysin and surfactin biosynthesis being regulated by common regulatory factors raised the question of whether srfA operon plays a role in bacilysin biosynthesis [72,76]. To test this, Karataş et al. suppressed srfA operon in *B. subtilis* PY79 and showed that the operon has a direct role in the bacilysin production [68]. To gain further insight, srfA gene was disrupted to discard ca. 26 kb part while keeping the comS gene intact in bacA-lacZ fusion bearing strain. As a result, bacA-lacZ expression was observed to be abolished in srfA mutant cells, suggesting that intact srfA operon is required for the transcription of bac operon (unpublished data). Karataş et al. also demonstrated that no bacilysin biosynthesis occurs in spo0H and/or spo0A-blocked mutants of *B. subtilis* while the biosynthetic activity increases in abrB-disrupted ones. In addition, bacilysin-negative phenotype was suppressed by an abrB mutation in spo0A-blocked mutants. Taken together, these data suggested that the transcription of the genes involved in bacilysin biosynthesis is negatively controlled by AbrB and this effect can be relieved by Spo0A [68].

CodY family consists of several bacterial GTP-sensing transcriptional pleiotropic repressors. The codY encoded protein was

initially shown to be involved in repression of *B. subtilis* dipeptide permease operon within the exponential growth phase in nutrient-rich medium [77]. CodY was proposed to be necessary for the initiation of sporulation and genetic competence by sensing intracellular GTP level and repressing early stationary genes. Since a *codY*-disrupted mutant could sporulate well under rich nutritional conditions, *codY* appeared as a crucial factor in preventing sporulation unless the nutrients are limiting [78, 79]. ppGpp (guanosine 5′-diphosphate 3′-diphosphate) has an essential role in transcription of the *bac* operon which mainly depends on the GTP level in the cell [33]. The level of intracellular GTP is regulated through a CodY-mediated repression system. In wild-type (*rel+*) cells, decreasing intracellular GTP level improves *bac* operon expression and also the deletion of *codY* enhances the transcription of these genes. On the other hand, decoyinine (a GMP synthetase inhibitor) supplementation or *codY* deletion did not activate the expression in *relA* cells, referred to as ‘relaxed cells’ which are unable to initiate stringent response properly [33]. The study performed by Köroğlu et al. indicated that activation and induction of *bacABCDEF* expression is tightly regulated and balanced by four proteins: ComA, Spo0A, AbrB and CodY. The two proteins AbrB and CodY are negative transcriptional regulators, capable of simultaneously binding to the promoter of *bac* operon (*P*<sub>*bac*</sub>), while ComA and Spo0A are positive transcriptional regulators. The basal level of transcription of the *bac* operon is ComA-dependent, whereas both ComA and Spo0A activity are required for transition state induction of *bac* operon. Both proteins compete for binding to *P*<sub>*bac*</sub> and are capable of displacing each other from *P*<sub>*bac*</sub> region in a concentration-dependent manner [80]. The expression of *bacA-lacZ* fusion was examined in three null mutant backgrounds, *phrC*, *phrF* and *phrK* [80]. The mutations in *phrC*, *phrF* and *phrK* genes affect *bac* operon expression at different extents, but the impact was the most significant with the *phrC* mutation, which resulted in a complete loss of *bac* expression. At this point, it is important to note that ComX is strain-specific signaling peptide while PhrC (CSF) is species-specific one conserved among *B. subtilis* and its close relatives. PhrC was shown to partially compensate when ComX pheromone-based signaling lacks among *B. subtilis* strains [81].

The response regulator DegU and its cognate kinase DegS represent a new type of cross communication between two phosphorylation systems in *B. subtilis* [82]. They constitute an important two component signal transduction system that controls numerous cellular processes, including motility, exoprotease production, biofilm formation, and genetic competence. When DegU protein is phosphorylated (DegU-P), it controls its own promoter, inducing many genes including exoproteases, and is degraded by a ClpCP protease [83]. CcpA (catabolite control protein A) also positively regulates *degU* through binding directly to the *degU* upstream region [84]. DegU was shown to positively regulate *bac* operon as well as the monocistronic *ywfH* (*bacG*) at the transcriptional level in *Bacillus amyloliquefaciens* [85]. Another protein, the transition state regulator ScoC (Hpr), is known to negatively regulate protease production and sporulation. Bacilysin biosynthesis is also negatively regulated by ScoC which is directly bound to the promoter of *bacABCDEF* [86]. Bacilysin operon promoter has two putative binding sites for ScoC, located between positions −50 and −42 (ScoC box1) and between positions −12 and −4 (ScoC box2). The CodY binding site is in the promoter region between positions −7 and +8 (CodY box) [87]. ScoC box2 and CodY box overlap in *bacABCDEF* operon upstream [86]. As recently reported, the putative binding sites of both DegU and ScoC are found in the *bacA* promoter region in *B. amyloliquefaciens* [85].

It was reported that *lutR* gene (formerly *yvfI*) encoding a GntR type transcription factor is required for bacilysin biosynthesis in *B. subtilis* PY79 [88]. Such kinds of transcription factors respond to stimuli by acting on distinct regulons [89]. The regulatory roles of LutR, encompassing a broad range of physiological activities in *B. subtilis* were recently documented [90].

As a summary, Fig. 2 schematically shows how bacilysin biosynthesis is regulated by many factors as shared with other physiological processes.

### Functions of bacilysin in its producer

Beyond their antimicrobial activities, the functions of peptide antibiotics in their producers have been the subject of speculation and discussions long before the common master players of these processes became identified [1, 2, 76, 91–94]. Function in sporulation, germination and outgrowth have been suggested. Main observations which led the researchers to establish such a link included (i) peptide antibiotic production usually begins at the late log phase of growth and both processes are triggered by starvation stress, (ii) random mutants no longer producing peptide antibiotics are usually defective in sporulation or germination/outgrowth, (iii) both sporulation and peptide antibiotic production are initiated or suppressed by the same nutritional regulations, and (iv) asporogenous/oligosporogenous mutants blocked at certain stages of sporulation are found as peptide antibiotic-nonproducers. For instance, bacilysin-negative derivative of *B. subtilis* PY79 obtained by NTG mutagenesis [32] was oligosporogenous and much more sensitive to heat, chemicals and lysozyme than the parental strain [95]. Besides, dipicolinic acid content of its spores was much lower. The improvement of the spore quality upon bacilysin supplementation to the cultures till mid-log phase as well as the restoration of the sporulation in *Bac*<sup>+</sup> transductants of this strain led to the suggestion that bacilysin might somehow affect sporulation before stage 1. Moreover, comparison of a set of asporogenous/oligosporogenous mutants of BGSC origin revealed that *spoOA*, *spoOF*, *spoOH*, *spoOK* mutants and some of those blocked at stage V were bacilysin-negative [96]. Currently, regulons involved in complex responses of *Bacillus* cells to environmental stimuli and coordination of gene expression with overlapping circuits were well understood (previous section), but still there are many peptide antibiotics produced by *Bacillus* and many other species with unknown bioactivities and specific functions in the producer [97]. One of the best studied examples is matrix-producing cannibal cell-type of *B. subtilis*, producing the extracellular matrix components required for biofilm formation along with cannibalism toxins, antibiotic-like sporulation killing factor (*skf*) and sporulation delay protein (*sdp*) along with Spo0A activating their operons [98]. These peptides to which the producing cells are immune can kill their *B. subtilis* siblings as well as other bacterial cells and delay entrance of *B. subtilis* into sporulation.

We recently analyzed the differential expression of proteins in *B. subtilis* and its bacilysin-deleted derivative OGU1 by employing

both gel-based and gel-free proteomics and observed a wide range
of the differences in protein expression profile in the absence of
bacilysin biosynthesis. Our proteome data apparently showed that
bacilysin, either directly or indirectly, acts like a small pleiotropic
signal molecule affecting diverse cellular functions including
spore quality (manuscript submitted).

**Biotechnological significance of L-amino acid ligase (Lal)**

Current (chemical synthesis and proteases/peptidases) versus
emerging (ribosome system; designed hybrid NRPSs, Lal method
in the form of either resting cell system or direct fermentation)
technologies for dipeptide manufacturing with their inherent
advantages and drawbacks have already been reviewed [99].
Among these, *B. thermoproteolyticus* neutral proteinase enzyme
thermolysin has found a much wider use for peptide bond formation
by a reverse reaction of hydrolysis. For instance, the low-
calorie sweetener aspartame (L-Asp-L-Phe-OMe) is currently manufactured by the use of thermolysin. Soluble dipeptides like L-Gly-
L-Tyr and L-Ala-L-Gln have been recognized as ingredients of
patient infusions, but could not find a broad use because of the
current unavailability of a cost-effective manufacturing. Lal, the
enzyme L-amino acid ligase catalyzing the final step of bacilysin

biosynthesis, should become the newest popular target for the
production of useful dipeptides with unique physical and chemical structure and function. Tabata and Hashimoto reported the
first successful production of L-Ala-L-Gln dipeptide by direct
fermentation using recombinant *E. coli* harboring *B. subtilis* Lal
expressed under a stationary-phase-specific promoter. Metabolic
manipulations to the host also included decreasing dipeptide-
degrading function by deleting certain genes, increasing the pool
of substrate amino acids by manipulating glutamine synthesis,
and providing high levels of another key enzyme, alanine dehydrogenase [100]. Though the current research on the possibilities
of the use of Lal is at a very early stage, the study reviewed herein is
a promising and encouraging one.

It was demonstrated very recently that Trp332 of Lal is crucial
for N-terminal substrate specificity while Arg328 is significant for
enantioselective recognition of L-amino acids [101]. The work
might provide guidelines to obtain specifically designed Lal
enzymes that can successfully synthesize unique dipeptides of
industrial importance.

**Acknowledgement**

We greatly acknowledge Mustafa Çiçek and Volkan Yıldırım of
METU for their technical assistance with the artwork.

---

**References**

[1] Katz E, Demain A. The peptide antibiotics of *Bacillus*: chemistry, biogenesis and possible functions. Bacteriol Rev 1977;41:449–74.
[2] Stein T. *Bacillus subtilis* antibiotics: structures, syntheses, and specific functions. Mol Microbiol 2005;56:845–57.
[3] Sieber SA, Marahiel MA. Learning from nature's drug factories: nonribosomal synthesis of cyclic peptides. J Bacteriol 2003;185:7036–43.
[4] Finking R, Marahiel MA. Biosynthesis of nonribosomal peptides 1. Annu Rev Microbiol 2004;58:453–88.
[5] Walsh CT. Polyketide and nonribosomal peptide antibiotics: modularity and versatility. Science 2004;303:1805–10.
[6] Abraham EP, Callow D, Gilliver K. Adaptation of *Staphylococcus aureus* to growth in the presence of certain antibiotics. Nature 1946;158:818–21.
[7] Foster JW, Woodruff HB. Bacillin, a new antibiotic substance from a soil isolate of *Bacillus subtilis*. J Bacteriol 1946;51:363–9.
[8] Newton GGF. Antibiotics from a strain of *B. subtilis*: bacilipin A and bacilipin-B and bacilysin. Br J Exp Pathol 1949;30:306–19.
[9] Rogers HJ, Newton GGF, Abraham EP. Production and purification of bacilysin. Biochem J 1965;97:573–8.
[10] Rogers HJ, Lomakina N, Abraham EP. Observations on the structure of bacilysin. Biochem J 1965;97:579–86.
[11] Walker JE, Abraham EP. Isolation of bacilysin and a new amino acid from culture filtrates of *Bacillus subtilis*. Biochem J 1970;118:557–61.
[12] Walker JE, Abraham EP. The structure of bacilysin and other products of *Bacillus subtilis*. Biochem J 1970;118:563–70.
[13] Neuss N, Molloy BB, Shah R, DeLaHiguera N. The structure of anticapsin, a new biologically active metabolite of *Streptomyces griseoplanus*. Biochem J 1970;118:571–5.
[14] Shah R, Neuss N, Gorman M, Boeck LD. Isolation, purification, and characterization of anticapsin. J Antibiot 1970;23:613–9.
[15] Borowski E. An improved method of preparing the antibiotic tetaine. II. Preparation of antibiotic in pure state. Acta Biochim Pol 1957;4:231–40.
[16] Kaminsky K, Sokokowsky T. The probable identity of bacilysin and tetaine. J Antibiot 1973;26:184–5.
[17] Atsumi K, Oiwa R, Omura S. Production of bacillin by *Bacillus* sp. strain no. KM-208 and its identity with tetaine (bacilysin). J Antibiot 1975;28:77–8.
[18] Roscoe J, Abraham EP. Experiments relating to the biosynthesis of bacilysin. Biochem J 1966;99:793–800.
[19] Walker JE. Antibiotic production and sporulation in *Bacillus subtilis*. Biochem J 1971;121:571–3.
[20] Kenig M, Abraham EP. Antimicrobial activities and antagonists of bacilysin and anticapsin. J Gen Microbiol 1976;94:37–45.
[21] Walton RB, Ricker EL. Reversal of the antibiotic bacillin by N-acetyl glucosamine. J Bacteriol 1962;84:1148–51.
[22] Kenig M, Vandamme E, Abraham EP. The mode of action of bacilysin and anticapsin and biochemical properties of bacilysin-resistant mutants. J Gen Microbiol 1976;94:46–54.
[23] Perry D, Abraham EP. Transport and metabolism of bacilysin and other peptides by suspensions of *Staphylococcus aureus*. J Gen Microbiol 1979;115:213–21.
[24] Perry D. Peptide transport in *Staphylococcus aureus*. J Gen Microbiol 1981;124:425–8.
[25] Chmara H, Milewski S, Dzieduszcka M, Smulkowski M, Sawlewicz P, Borowski E. Epoxypeptides – a novel group of metabolic-inhibitors in prokaryotic and eukaryotic microorganisms. Drugs Exp Clin Res 1982;8:11–20.
[26] Chmara H. Inhibition of glucosamine synthase by bacilysin and anticapsin. J Gen Microbiol 1985;131:265–71.
[27] Milewski S, Chmara H, Borowski E. Anticapsin – an active-site directed inhibitor of glucosamine-6-phosphate synthetase from *Candida albicans*. Drugs Exp Clin Res 1986;12:577–83.
[28] Sakajoh M, Solomon NA, Demain AL. Cell-free synthesis of the dipeptide antibiotic bacilysin. J Ind Microbiol 1987;2:201–8.
[29] Yazgan A, Özçengiz G, Özçengiz E, Kilinc K, Marahiel MA, Alaeddinoğlu NG. Bacilysin biosynthesis by a partially-purified enzyme fraction from *Bacillus subtilis*. Enzyme Microb Technol 2003;29:400–6.
[30] Tüzün I, Karataş AY, esenoglu Ö, Demir AS, Özçengiz G. 2-Amino-N-(2-furylmethyl) propanamide as a novel alanylglycine equivalent synthesized by bacilysin synthetase. Enzyme Microb Technol 2003;33:725–8.
[31] Hilton MD, Alaeddinoğlu NG, Demain AL. Synthesis of bacilysin by *Bacillus subtilis* branches from prephenate of the aromatic amino acid pathway. J Bacteriol 1988;170:482–4.
[32] Hilton MD, Alaeddinoğlu NG, Demain AL. *Bacillus subtilis* mutant deficient in the ability to produce dipeptide antibiotic bacilysin: isolation and mapping of mutation. J Bacteriol 1988;170:1018–20.
[33] Inaoka T, Takahashi K, Ohnishi-Kameyama M, Yoshida M, Ochi K. Guanine nucleotides guanosine 5′-diphosphate 3′-diphosphate and GTP co-operatively regulate the production of an antibiotic bacilysin in *Bacillus subtilis*. J Biol Chem 2003;278:2169–76.
[34] Steinborn G, Hajirezaei MR, Hofemeister J. *bac* genes for recombinant bacilysin and anticapsin production in *Bacillus* host strains. Arch Microbiol 2005;183:71–9.
[35] Parker JB, Walsh CT. Stereochemical outcome at four stereogenic centers during conversion of prephenate to tetrahydrotyrosine by BacABGF in the bacilysin pathway. Biochemistry 2012;51:5622–32.
[36] Mahlstedt SA, Walsh CT. Investigation of anticapsin biosynthesis reveals a four-enzyme pathway to tetrahydrotyrosine in *Bacillus subtilis*. Biochemistry 2010;49:912–23.
[37] Parker JB, Walsh CT. Olefin isomerization regiochemistries during tandem action of BacA and BacB on prephenate in bacilysin biosynthesis. Biochemistry 2012;51:3241–51.
[38] Mahlstedt SA, Fielding EN, Moore BS, Walsh CT. Prephenate decarboxylases: a new prephenate-utilizing enzyme family that performs nonaromatizing decarboxylation en route to diverse secondary metabolites. Biochemistry 2010;49:9021–3.

---

6 www.elsevier.com/locate/nbt

Please cite this article in press as: Özçengiz, G., and Öğülür, I., Biochemistry, genetics and regulation of bacilysin biosynthesis and its significance more than an antibiotic, New Biotechnol. (2015), http://dx.doi.org/10.1016/j.nbt.2015.01.006

[39] Rajavel M, Mitra A, Gopal B. Role of *Bacillus subtilis* BacB in the synthesis of bacilysin. J Biol Chem 2009;284:31882–92.

[40] Rajavel M, Gopal B. Analysis of multiple crystal forms of *Bacillus subtilis* BacB suggests a role for a metal ion as a nucleant for crystallization. Acta Crystallogr D 2010;66:635–9.

[41] Rajavel M, Perinbam K, Gopal B. Structural insights into the role of *Bacillus subtilis* YwfH (BacG) in tetrahydrotyrosine synthesis. Acta Crystallogr D 2013;69:324–32.

[42] Tabata K, Ikeda H, Hashimoto S. *ywfE* in *Bacillus subtilis* codes for a novel enzyme, L-amino acid ligase. J Bacteriol 2005;187:5195–202.

[43] Shomura Y, Hinokuchi E, Ikeda H, Senoo A, Takahashi Y, Saito J, et al. Structural and enzymatic characterization of BacD, an L-amino acid dipeptide ligase from *Bacillus subtilis*. Protein Sci 2012;21:707–16.

[44] Parker JB, Walsh CT. Action and timing of BacC and BacD in the late stages of biosynthesis of the dipeptide antibiotic bacilysin. Biochemistry (NY) 2013;52:889–901.

[45] Wild H. Enantioselective total synthesis of the antifungal natural-products chlorotetaine, bacilysin, and anticapsin and of related-compounds – revision of the relative configuration. J Org Chem 1994;59:2748–61.

[46] Özcengiz G, Alaeddinoğlu NG, Demain AL. Regulation of biosynthesis of bacilysin by *Bacillus subtilis*. J Ind Microbiol 1990;6:91–100.

[47] Başalp A, Özcengiz G, Alaeddinoğlu NG. Changes in patterns of alkaline serine protease and bacilysin formation caused by common effectors of sporulation in *Bacillus subtilis* 168. Curr Microbiol 1992;24:129–35.

[48] Özcengiz G, Alaeddinoğlu NG. Bacilysin production by *Bacillus subtilis*: effects of bacilysin, pH and temperature. Folia Microbiol 1991;36:522–6.

[49] Redfield RJ. Is quorum sensing a side effect of diffusion sensing? Trends Microbiol 2002;10:365–70.

[50] Thoendel M, Horswill AR. Biosynthesis of peptide signals in gram-positive bacteria. Adv Appl Microbiol 2010;71:91–112.

[51] Grossman AD. Genetic networks controlling the initiation of sporulation and the development of genetic competence in *Bacillus subtilis*. Annu Rev Genet 1995;29:477–508.

[52] Comella N, Grossman AD. Conservation of genes and processes controlled by the quorum response in bacteria: characterization of genes controlled by the quorum-sensing transcription factor ComA in *Bacillus subtilis*. Mol Microbiol 2005;57:1159–74.

[53] Perego M, Glaser P, Hoch JA. Aspartyl-phosphate phosphatases deactivate the response regulator components of the sporulation signal transduction system. Mol Microbiol 1996;19:1151–7.

[54] Weinrauch Y, Penchev Y, Dubnau R. A *Bacillus subtilis* regulatory gene product for genetic competence and sporulation resembles sensor protein members of the bacterial two-component signal-transduction systems. Genes Dev 1990;4:860–72.

[55] Solomon JM, Magnuson R, Srivastava A, Grossman AD. Convergent sensing pathways mediate response to two extracellular competence factors in *Bacillus subtilis*. Genes Dev 1995;9:547–58.

[56] Lazazzera BA, Kurtser IG, McQuade RS, Grossman AD. An autoregulatory circuit affecting peptide signaling in *Bacillus subtilis*. J Bacteriol 1999;181:5193–200.

[57] Magnuson R, Solomon J, Grossman AD. Biochemical and genetic characterization of a competence pheromone from *B. subtilis*. Cell 1994;77:207–16.

[58] Auchtung JM, Lee CA, Grossman AD. Modulation of the ComA-dependent quorum response in *Bacillus subtilis* by multiple Rap proteins and Phr peptides. J Bacteriol 2006;188:5273–85.

[59] Bongiorni C, Ishikawa S, Stephenson N, Ogasawara N, Perego M. Synergistic regulation of competence development in *Bacillus subtilis* by two Rap-Phr systems. J Bacteriol 2005;187:b4353–61.

[60] Smits WK, Bongiorni C, Veengin JW, Hamoen LW, Kuipers OP, Perego M. Temporal separation of distinct differentiation pathways by a dual specificity Rap-Phr system in *Bacillus subtilis*. Mol Microbiol 2007;65:103–20.

[61] Burbulis D, Trach KA, Hoch JA. Initiation of sporulation in *B. subtilis* is controlled by a multicomponent phosphorelay. Cell 1991;64:545–52.

[62] Hoch JA. The phosphorelay signal transduction pathway in the initiation of *Bacillus subtilis* sporulation. J Cell Biochem 1993;51:55–61.

[63] Sonenshein AL. Control of sporulation initiation in *Bacillus subtilis*. Curr Opin Microbiol 2000;3:561–6.

[64] Strauch MA, Webb V, Spiegelman B, Hoch JA. The Spo0A protein of *Bacillus subtilis* is a repressor of the abrB gene. Proc Natl Acad Sci USA 1990;87:1801–5.

[65] Jaacks KJ, Healy J, Losick R, Grossman AD. Identification and characterization of genes controlled by the sporulation regulatory gene *spoOH* in *Bacillus subtilis*. J Bacteriol 2001;171:4121–9.

[66] Garti-Levi S, Eswara A, Smith Y, Fujita M, Ben-Yehuda S. Novel modulators controlling entry into sporulation in *Bacillus subtilis*. J Bacteriol 2013;195:1475–83.

[67] Yazgan A, Özcengiz G, Marahiel MH. Tn10 insertional mutations of *Bacillus subtilis* that block the biosynthesis of bacilysin. Biochim Biophys Acta 2001;1518:87–94.

[68] Karataş AY, Cetin S, Özcengiz G. The effects of insertional mutations in comQ, comP, srfA, spo0H, spo0A and abrB genes on bacilysin biosynthesis in *Bacillus subtilis*. Biochim Biophys Acta 2003;1626:51–6.

[69] Perego M. A peptide export-import control circuit modulating bacterial development regulates protein phosphatases of the phosphorelay. Proc Natl Acad Sci USA 1997;94:8612–7.

[70] Lazazzera BA. The intracellular function of extracellular signaling peptides. Peptides 2001;22:1519–27.

[71] Cosby WM, Vollenbroich D, Lee OH, Zuber P. Altered *srf* expression in *Bacillus subtilis* resulting from changes in culture pH is dependent on the Spo0K oligopeptide permease and the ComQX system of extracellular control. J Bacteriol 1998;180:1438–45.

[72] Nakano MM, Magnuson R, Myers A, Curry J, Grossman AD, Zuber P. *srfA* is an operon required for surfactin production, competence development, and efficient sporulation in *Bacillus subtilis*. J Bacteriol 1991;173:1770–8.

[73] Cosmino P, Rodriguez F, De Ferra F, Grandi G, Perego M, Venema G, et al. Sequence and genetic analysis of the genetic locus responsible for surfactin synthesis in *Bacillus subtilis*. Mol Microbiol 1993;8:821–31.

[74] Hamoen LW, Eshuis H, Jongbloed J, Venema G, van Sinderen D. A small gene, designated *comS*, located within the coding region of the fourth amino acid-activating domain of *srfA* is required for competence development in *Bacillus subtilis*. Mol Microbiol 1995;15:55–63.

[75] Hamoen LW, Venema G, Kuipers OP. Controlling competence in *Bacillus subtilis*: shared use of regulators. Microbiology 2003;149:9–17.

[76] Marahiel MA, Nakano MM, Zuber P. Regulation of peptide antibiotic production in *Bacillus*. Mol Microbiol 1993;7:631–6.

[77] Slack FJ, Serror P, Joyce E, Sonenshein AL. A gene required for nutritional repression of the *Bacillus subtilis* dipeptide permease operon. Mol Microbiol 1995;15:689–702.

[78] Ratnayake-Lecamwasam M, Serror P, Wong KW, Sonenshein AL. *Bacillus subtilis* CodY represses early-stationary-phase genes by sensing GTP levels. Genes Dev 2001;15:1093–103.

[79] Inaoka T, Ochi K. RelA protein is involved in induction of genetic competence in certain *Bacillus subtilis* strains by moderating the level of intracellular GTP. J Bacteriol 2002;184:3923–30.

[80] Köroğlu TE, Öğülür I, Mutlu S, Yazgan-Karataş A, Özcengiz G. Global regulatory systems operating in bacilysin biosynthesis in *Bacillus subtilis*. J Mol Microbiol Biotechnol 2011;20:144–55.

[81] Pottathil M, Jung A, Lazazzera BA. CSF, a species-specific extracellular signaling peptide for communication among strains of *Bacillus subtilis* and *Bacillus mojavensis*. J Bacteriol 2008;190:4095–9.

[82] Macek B, Mijakovic I, Olsen JV, Gnad F, Kumar C, Jensen PR, et al. The serine/threonine/tyrosine phosphoproteome of the model bacterium *Bacillus subtilis*. Mol Cell Proteomics 2007;6:697–707.

[83] Ogura M, Tsukahara K. Autoregulation of the *Bacillus subtilis* response regulator gene *degU* is coupled with the proteolysis of DegU-P by ClpCP. Mol Microbiol 2010;75:1244–59.

[84] Ishii H, Tanaka T, Ogura M. The *Bacillus subtilis* response regulator gene *degU* is positively regulated by CcpA and by catabolite-repressed synthesis of ClpC. J Bacteriol 2013;195:193–201.

[85] Mariappan A, Makarewicz O, Chen XH, Borris R. Two-component response regulator DegU controls the expression of bacilysin in plant-growth-promoting bacterium *Bacillus amyloliquefaciens* FZB42. J Mol Microbiol Biotechnol 2012;22:114–25.

[86] Inaoka T, Wang G, Ochi K. ScoC regulates bacilysin production at the transcription level in *Bacillus subtilis*. J Bacteriol 2009;191:7367–71.

[87] Belitsky BR, Sonenshein AL. Genetic and biochemical analysis of CodY-binding sites in *Bacillus subtilis*. J Bacteriol 2008;190:1224–36.

[88] Köroğlu TE, Kurt-Gür G, Ünlü EC, Yazgan-Karataş A. The novel gene *yvfI* in *Bacillus subtilis* is essential for bacilysin biosynthesis. Antonie Van Leeuwenhoek 2008;94:471–9.

[89] Kunst F, Ogasawara N, Moszer I, Albertini AM, Alloni G, Azevedo V, et al. The complete genome sequence of the gram positive bacterium *Bacillus subtilis*. Nature 1997;390:249–56.

[90] Irigül-Sönmez Ö, Köroğlu TE, Öztürk B, Ákos T, Kovács AT, Kuipers OP, et al. In *Bacillus subtilis* LutR is part of the global complex regulatory network governing the adaptation to the transition from exponential growth to stationary phase. Microbiology SGM 2014;160:243–60.

[91] Frangou-Lazaridis M, Seddon B. Effect of gramicidin S on the transcription system of the producer *Bacillus brevis* Nagano. J Gen Microbiol 1985;131:437–49.

[92] Hodgson B. Possible roles for antibiotics and other biologically active peptides at specific stages during sporulation of *Bacillaceae*. J Theor Biol 1970;30:111–9.

[93] Sarkar N, Paulus H. Function of peptide antibiotics in sporulation. Nat New Biol 1972;239:228–30.

[94] Ristow H, Pschorn W, Hansen J, Winkel U. Induction of sporulation in *Bacillus brevis* by peptide antibiotics. Nature (Lond) 1979;280:165–6.

[95] Özcengiz G, Alaeddinoğlu NG. Bacilysin production and sporulation in *Bacillus subtilis*. Curr Microbiol 1991;23:61–4.

[96] Özcengiz G, Alaeddinoğlu NG. Bacilysin production in relation to sporulation in *Bacillus subtilis*. Turk J Biol 1991;15:85–93.

[97] Shank AE, Kolter R. Extracellular signaling and multicellularity in *Bacillus subtilis*. Curr Opin Microbiol 2011;14:741–7.

[98] Gonzalez-Pastor JE. Cannibalism: a social behavior in sporulating *Bacillus subtilis*. FEMS Microbiol Rev 2011;35:415–24.

[99] Hashimoto S. Occurrence, biosynthesis and biotechnological production of dipeptides. In: Wendisch VF, editor. Amino acid biosynthesis-pathways, regulation and metabolic engineering. Berlin, Heidelberg: Springer-Verlag; 2007, ISBN: 978-3-540-48595-7.

[100] Tabata K, Hashimoto S. Fermentative production of L-alanyl-L-glutamine by a metabolically engineered *Escherichia coli* strain expressing L-amino acid á-ligase. Appl Environ Microbiol 2007;73:6378–85.

[101] Tsuda T, Asami M, Koguchi Y, Kojima S. Single mutation alters the substrate specificity of L-amino acid ligase. Biochemistry 2014;53:2650–60.
