
NPR

---

**REVIEW**

The expanding roles of c-di-GMP in the biosynthesis of exopolysaccharides and secondary metabolites

Zhao-Xun Liang*

Covering: up to the end of 2014

The cyclic dinucleotide c-di-GMP has emerged in the last decade as a prevalent intracellular messenger that orchestrates the transition between the motile and sessile lifestyles of many bacterial species. The motile-to-sessile transition is often associated with the formation of extracellular matrix-encased biofilm, an organized community of bacterial cells that often contributes to antibiotic resistance and host-pathogen interaction. It is increasingly clear that c-di-GMP controls motility, biofilm formation and bacterial pathogenicity partially through regulating the production of exopolysaccharides (EPS) and small-molecule secondary metabolites. This review summarizes our current understanding of the regulation of EPS biosynthesis by c-di-GMP in a diversity of bacterial species and highlights the emerging role of c-di-GMP in the biosynthesis of small-molecule secondary metabolites.

### 1. C-di-GMP as a bacterial second messenger

C-di-GMP or bis(3′-5′)-cyclic dimeric GMP was first discovered in 1987 as an allosteric regulator of cellulose synthase in the bacterium *Gluconacetobacter xylinum.* C-di-GMP is a cyclic dinucleotide that consists of two guanosine monophosphate moieties connected by O₃′-P phosphodiester bonds (Fig. 1A and 1B). The widespread distribution of c-di-GMP in the bacterial kingdom was not appreciated till the beginning of the 21st century. Our current knowledge of the biological roles of c-di-GMP was mainly gained from the studies of Gram-negative bacteria such as *Pseudomonas aeruginosa*, *Escherichia coli*, *Vibrio cholerae*, *Salmonella typhimurium* and *Caulobacter crescentus*. More recent studies on Gram-positive bacteria such as *Bacillus subtilis*, *Listeria monocytogenes* and *Streptomyces coelicolor* also began to reveal the role of c-di-GMP in firmicutes and actinobacteria. C-di-GMP mediates a myriad of cellular functions and many aspects of bacterial physiology. Many of the cellular functions controlled by c-di-GMP are crucial for the transition between the phenotypically distinct motile and sessile lifestyles. The sessile lifestyle is best known to be adopted by the bacterial cells that form surface-associated biofilm. In mature biofilm, cells are encased in an extracellular polymeric matrix composed of various amount of extracellular DNA, proteinaceous components and EPSs. The complex biofilm matrix protects bacteria from antimicrobials, physical stresses and predation by host

---

Division of Structural Biology & Biochemistry, School of Biological Sciences, Nanyang Technological University, Singapore 637551. E-mail: zxliang@ntu.edu.sg

This journal is © The Royal Society of Chemistry 2015

NPR

A

B

C

D

Fig. 1 Overview of c-di-GMP signalling. (A) Chemical structure of c-di-GMP. (B) Intercalating structure of the biologically relevant dimeric form of c-di-GMP. (C) Synthesis of c-di-GMP by GGDEF domain-based diguanylate cyclase (DGC) and degradation of c-di-GMP by EAL or HD-GYP domain-based phosphodiesterase (PDE). (D) Number of the c-di-GMP-metabolizing GGDEF, EAL and HD-GYP domains encoded by the genomes of representative species from five major bacterial phyla (data are taken from http://www.ncbi.nlm.nih.gov/Complete_Genomes/SignalCensus.html).

immune systems. ${ }^{5-7}$ As a result of biofilm formation, bacterial infection can be difficult to eradicate because of the increased resistance to antibiotics, phagocytosis and other components of immune system. ${ }^{5,8}$ Because of the vital role played by c-di-GMP in integrating environmental inputs and controlling the motile-sessile transition in both directions, *i.e.*, biofilm formation and biofilm dispersal, it is hoped that anti-biofilm strategies can be developed by intervening c-di-GMP signalling pathways.

The molecular mechanism of c-di-GMP signalling has been under intensive investigation. Many c-di-GMP signalling proteins and pathways from a diversity of bacterial species have been unveiled. Most importantly, diguanylate cyclase (DGC) and phosphodiesterase (PDE) family proteins responsible for the synthesis and degradation of the cyclic dinucleotide were uncovered (Fig. 1C). ${ }^{9}$ DGC proteins contain a GGDEF domain that synthesizes one c-di-GMP molecule from two molecules of GTP. ${ }^{10-13}$ PDE proteins contain an EAL or HD-GYP domain that breaks down c-di-GMP into $5^{\prime}$-pGpG or GMP. ${ }^{11,14-17}$ The cellular concentration of c-di-GMP is controlled by the opposing activities of DGC and PDE proteins, which are often regulated by associated sensory domains. The crystal structures of several GGDEF, EAL and HD-GYP proteins have been determined to reveal the key residues involved in catalysis, substrate-binding and product inhibition. ${ }^{18-26}$ Identification of the catalytic residues by structural and biochemical studies also revealed the existence of a significant number of enzymatically inactive GGDEF, EAL and HD domains. ${ }^{14,27-31}$ The number of genes encoding the GGDEF, EAL and HD-GYP proteins varies greatly among bacterial species (Fig. 1D). While some bacterial genomes such as those of Staphylococcus aureus and Haemophilus influenzae do not encode any c-di-GMP signalling proteins, a large number of GGDEF, EAL and HD-GYP protein-encoding genes can be found in many bacterial genomes. ${ }^{3,32,33}$ For

Zhao-Xun Liang studied Organic Chemistry at Lanzhou University in China, where he received B.S. & M.Sc. degrees (1994 & 1997) under the guidance of Prof. Qing-Xiang Guo. He obtained his Ph.D. degree in Chemistry from Northwestern University under the supervision of Brian Hoffman (1997–2002). After working with Judith Klinman on dehydrogenases for three years as a postdoctoral fellow at the University of California at Berkeley, he joined the School of Biological Sciences of Nanyang Technological University (NTU) in Singapore as an assistant professor in the fall of 2005. He is currently an associated professor focusing on the discovery of novel enzymes/proteins involved in bacterial biofilm formation and secondary metabolite biosynthesis.

example, the genomes of the Gram-negative *E. coli* K12 and *P. aeruginosa* PAO1 contain 29 and 39 copies of GGDEF/EAL/HD-GYP genes, respectively, and the genomes of the Gram-positive *S. coelicolor* and *Clostridium difficile* contain 7 and 37 copies, respectively. Remarkably, the genomes of the environmental strains *Kineococcus radiotolerans* and *Shewanella amazonensis* contain 83 and 73 GGDEF, EAL and HD-GYP domain-encoding genes, respectively.

For c-di-GMP to elicit cellular response, it must bind to its cellular targets or effectors. A surprisingly diverse array of structurally distinct c-di-GMP binding effectors has been discovered so far. The c-di-GMP effectors include PilZ domains, non-enzymatic GGDEF and EAL domains and several distinct c-di-GMP binding domains found in transcriptional regulators and enzymes. It is not uncommon for a bacterial genome to encode multiple c-di-GMP effectors of different families.³⁴ The diversity and multiplicity of the effectors would enable c-di-GMP to regulate different cellular processes. PilZ domains are the first discovered c-di-GMP receptors and putatively the most prevalent ones.³⁵⁻³⁹ C-di-GMP binding PilZ domain can be a stand-alone protein, or the regulatory domain of a large multifunctional protein.³⁵ Enzymatically inactive GGDEF domains such as the ones found in PopA from *C. crescentus* and PelD from *P. aeruginosa* also function as c-di-GMP effectors.²⁷,³⁸,⁴⁰⁻⁴² The enzymatically inactive EAL domains of FimX from *P. aeruginosa* and LapD from *P. fluorescens* also act as c-di-GMP binding domains.⁴³⁻⁴⁸ A polynucleotide phosphorylase RNA processing enzyme in *E. coli* was reported to contain a c-di-GMP binding site.⁴⁹ Transcriptional regulators such as FleQ from *P. aeruginosa* and VpsT from *Vibrio cholerae* bind c-di-GMP with novel c-di-GMP binding motifs.⁵⁰,⁵¹ In addition to the protein effectors, c-di-GMP can also control gene transcription by binding to riboswitches, which are the regulatory segments of messenger RNA.⁵²⁻⁵⁵ In the opportunistic pathogen *P. aeruginosa* PAO1, 12 c-di-GMP-binding proteins have been discovered as of today. The 12 effectors include seven PilZ domain proteins, three transcription regulators (FleQ, BrlR, PA4395), two proteins (PelD, FimX) that contain enzymatically inactive GGDEF or EAL domain.³⁸,⁴³,⁵⁰,⁵⁶⁻⁵⁸ With so many c-di-GMP signaling proteins in the cell, cross-talk among the pathways must be minimized to achieve high signalling specificity. It has been suggested that signalling specificity can be achieved either by using effectors with different binding affinities for c-di-GMP or through temporal or spatial sequestering the signalling proteins.⁵⁹⁻⁶³

The field of c-di-GMP signalling is still expanding rapidly with new components, pathways and mechanisms being revealed constantly. A number of excellent reviews and commentary articles have been published over the years.⁶⁴⁻⁷¹ The readers are referred to those reviews for a more comprehensive view of c-di-GMP signalling. From the perspective of someone who is interested in secondary metabolism, it is interesting that c-di-GMP controls the production of an increasing number of bacterial polysaccharides and small-molecule metabolites. It is now firmly established that c-di-GMP regulates the biosynthesis of several classes of EPSs important for bacterial survival, infection and biofilm formation. Recent studies also suggest that c-di-GMP mediates the production of

small-molecule secondary metabolites such as actinorhodin. This review highlights recent advances in our understanding of the regulatory role of c-di-GMP in the biosynthesis of EPSs and small-molecule secondary metabolites.

## 2. Regulation of EPS biosynthesis by c-di-GMP

Bacteria synthesize a wide variety of polysaccharides that include intracellular polysaccharides, structural polysaccharides and extracellular polysaccharides or exopolysaccharides (EPSs). EPSs are high-molecular-weight biopolymers secreted by microbes into their surrounding environment. EPSs confer the microbe additional survival advantage by protecting against nonspecific and specific host immunity and enhancing resistance to desiccation, oxidizing stress and host defensive factors.⁴,⁵,⁷² EPSs are also the major structural components of the extracellular polymeric matrix involved in surface adhesion, cell-cell interaction and formation of bacterial biofilm.⁷³⁻⁷⁵ In many bacteria, including the human pathogens *E. coli*, *V. cholerae*, and *P. aeruginosa*, EPSs are indispensable for biofilm formation. The mutants that are deficient in EPS production are severely compromised in forming mature biofilms.⁷⁶⁻⁷⁸

Bacterial EPSs are composed of monosaccharides, with the hexoses such as glucose, galactose, mannose and the pentoses such as fucose and rhamnose as the most common building blocks.⁷⁹,⁸⁰ Incorporation of modified monosaccharides such as methyl, acetyl, amino and sulphated sugar building blocks generates EPSs with unique physical and chemical properties. The structural diversity of EPSs can be further increased by incorporating non-sugar units such as peptidic moieties. The chemical structures of several best studied bacterial EPSs are shown in Fig. 2. Because of structural heterogeneity and complexity, the composition and structure of many bacterial EPSs remain to be fully characterized today.

Bacterial EPSs are either synthesized extracellularly by cell wall-anchored enzymes, or assembled in the cytoplasm and transported through inner and outer membranes to form a cell-surface anchored capsule or slime encasing the bacterial cells. Production and secretion of bacterial EPSs are highly regulated processes controlled by cellular and environmental signals through a web of complex signalling networks. Regulation of EPS biosynthesis by c-di-GMP, which may occur at both transcriptional or post-transcriptional levels, has emerged as a highly conserved regulatory mechanism. As detailed below, the regulatory mechanisms of c-di-GMP in EPS production are rather diverse, as evidenced by the involvement of different c-di-GMP binding effectors and signalling pathways.

### 2.1. Cellulose

As the most abundant biopolymer on earth, cellulose consists of glucose units that are connected through C1–C4 acetal linkages. Although cellulose is predominantly produced by plants and algae, bacteria such as *Salmonella enteritidis*, *E. coli*, *G. xylinum*, *Thermosynechococcus vulcanus*, *Rhodobacter sphaeroides* and a

NPR

cellulose

alginate

PNAG

Psl

curdlan

Fig. 2 Chemical structures of cellulose, alginate, Psl and 2,3-poly-β-1,6-N-acetylglucosamine (PNAG) and curdlan polysaccharides.

number of *Pseudomonas* strains also produce extracellular cellulose.¹⁸¹⁻⁸⁵ Extracellular cellulose promotes the attachment of bacteria to a variety of biotic and abiotic surfaces.⁸⁶⁻⁸⁸ Cellulose also serves as a structural component of the extracellular polymeric matrix required to maintain the robustness of biofilm for several bacterial species.⁷⁵ Bacterial cellulose production correlates with biofilm formation and survival of some bacteria in host environment.⁸³,⁸⁹ Cellulose-deficient bacterial mutants usually form defective biofilm and are more susceptible to antibiotic and oxidative stresses. For instance, the cellulose-deficient mutants of *S. enteritidis* were more sensitive to chlorine treatment, which suggests that cellulose production and biofilm formation contribute to the survival of *S. enteritidis* on surface environments.⁸³

Prokaryotic cellulose is synthesized by membrane-embedded cellulose-synthesizing complexes. The essential and highly conserved glycosyltransferase subunits use uridine diphosphate-activated glucose (UDP-Glc) as substrate to assemble the nascent polysaccharide. α-Proteobacteria such as *G. xylinum* and *R. sphaeroides* synthesize and secrete cellulose via a protein complex consisting of three basic subunits (Fig. 3A). For *G. xylinum*, the inner-membrane-embedded

Nat. Prod. Rep.

Review

From the pioneering work of Ross *et al.*, the cellulose-synthesizing activity of AcsA of *G. xylinum* was found to be positively regulated by an intracellular metabolite, which turned out to be c-di-GMP, an unprecedented cyclic dinucleotide at that time. It was later found that AcsA contains a C-terminal PilZ domain that exhibits c-di-GMP binding capacity. The crystal structure of the catalytically active *R. sphaeroides* BcsA–BcsB (homologs of AcsA and AcsB) complex in an intermediate state during cellulose synthesis and translocation was determined recently (Fig. 4). From the crystal structure, it can be seen that the c-di-GMP-binding PilZ domain is localized within the C-terminal region of BcsA, right next to the catalytic glycosyltransferase domain. The C-terminus of

BcsA extends from the last transmembrane helix and folds into a six-stranded β-barrel to form the PilZ<sub>BcsA</sub> domain. The PilZ<sub>BcsA</sub> domain is connected to an amphipathic α-helix located next to the putative active site of the glycosyltransferase domain. The amphipathic helix extends past the substrate-binding pocket and interacts with the so-called gating loop. Comparison of the crystal structures of c-di-GMP-free and c-di-GMP-bound BcsA revealed that c-di-GMP binding to the PilZ<sub>BcsA</sub> domain induces a conformational change in the gating loop to allow the access of UDP-Glc to the active site (Fig. 4). The structures further revealed that c-di-GMP binding breaks a salt bridge between R580 and the gating loop to promote the opening of the gating loop.

High cellular levels of c-di-GMP correlate with increased enzymatic activity of AcsA and cellulose biosynthesis. The c-di-GMP levels in *G. xylinum* are controlled by the GGDEF and EAL domains of the six paralogous proteins AxDGC1, 2, 3 and AxPDE1, 2, 3 (Fig. 3A). The proteins AxDGC1, 2 and 3 contain a Per-ARNT-Sim (PAS) domain and an enzymatically active GGDEF domain. The PAS domain binds a flavin (FAD) cofactor and the redox state of the cofactor modulates the DGC activity of AxDGC1-3. AxPDE1, 2 and 3 contain a heme-bound PAS domain and an active EAL domain. The regulatory PAS domain suppresses the activity of the EAL domain when the heme iron is coordinated with O₂. Collectively, the six proteins control the cellular concentration of c-di-GMP and enable the obligate aerobe *G. xylinum* to suppress cellulose synthesis when oxygen availability is low.

In contrast to *G. xylinum* and *R. sphaeroides*, γ-proteobacteria such as *E. coli*, *S. enteritidis* and *S. typhimurium* contain two divergent operons (*bcsABZD* and *bcsEFG*) that code for the cellulose-synthesizing complex (Fig. 3B). The genes *bcsA*, *bcsB* and *bcsD* encode proteins for cellulose synthesis, assembly and export. BcsZ is a cellulose hydrolase that cleaves intra-strand β-1,4 linkages in the cellulose chain and is involved in releasing the growing polymer in *E. coli*. While the functions of BcsF and BcsG are not clear, it was recently found that c-di-GMP binds to the GGDEF I-site like (GIL) domain of BcsE. BcsE is required for maximal cellulose production in *S. typhimurium* probably by affecting the temporal initiation and abundance of secreted cellulose fibres. The findings suggest that cellulose production in these bacteria is controlled by c-di-GMP through a two-tiered system involving PilZ<sub>BcsA</sub> and GIL<sub>BcsE</sub> domains. In *E. coli* and *S. typhimurium*, several key components from the c-di-GMP-dependent regulatory cascades that control cellulose production have been identified. Cellulose synthesis in Salmonella could be induced when one of the DGC proteins of Salmonella, specifically STM1987, YegE, YfiN, and AdrA, or a heterologous DGC protein (HmsT), was individually expressed under activating environmental conditions. A master regulator, CsgD (or AgfD), controls the expression of the *adrA* (or *yaiC*) gene that codes for the DGC protein AdrA (Fig. 3B). AdrA raises the concentration of a local pool of c-di-GMP to stimulate cellulose synthesis. In the commensal *E. coli* strain 1094, cellulose synthesis is regulated by a CsgD-independent pathway that involves the DGC protein YdeQ, but not AdrA. Identification of the CsgD-independent mechanism highlights the

Fig. 4 Crystal structure of the *R. sphaeroides* BcsA/BcsB complex with c-di-GMP bound. C-di-GMP and the nascent polysaccharide chain are shown as sticks. The c-di-GMP-free structure of BcsA/BcsB complex is superimposed on the structure to show c-di-GMP-induced conformational change in PilZ<sub>BcsA</sub> and the gating loop. The c-di-GMP-free PilZ<sub>BcsA</sub> and the gating loop are in magenta. The R580 residue is shown as sticks to highlight the conformational change upon c-di-GMP binding. The thick red arrows indicate the conformational changes of the gating loop and R580 induced by c-di-GMP binding to enlarge the active site for substrate binding.

diversity of regulatory mechanisms used by *E. coli* and likely other bacteria to control cellulose production. This view is further reinforced by the findings that the binding of Sal4, a monoclonal polymeric IgA antibody, to the O-antigen of *S. Typhimurium* activates yet another DGC protein (YeaJ) to increase the cellular levels of c-di-GMP to stimulate EPS production.¹⁰⁵

The marine bacterium *Vibrio fischeri* encodes 48 proteins with GGDEF and/or EAL domains. *V. fischeri* produces cellulose and another EPS named symbiosis polysaccharide (Syp). Bassia *et al.* showed that disruption of the gene encoding the c-di-GMP degrading PDE protein BinA caused increases in cellular c-di-GMP level, cellulose production and biofilm formation.¹⁰⁶ The phenotypes of the ΔbinA mutant strain could be reversed by a single transposon insertion in the cellulose-synthesizing *bcs* genes. These results indicate that the production of cellulose is regulated by c-di-GMP in *V. fischeri*, though it is not clear whether Syp synthesis is also controlled by c-di-GMP.

*Agrobacterium tumefaciens* is a pathogen that causes crown gall disease in plants. The interaction of *A. tumefaciens* with its host requires the production of attachment matrix. One component of the attachment matrix produced by *A. tumefaciens* is cellulose. The cellulose fibrils facilitate the attachment of the bacterium to the plant cells.⁸⁸ Mutants deficient in cellulose production bind weakly to plant cell surfaces and do not efficiently establish biofilms. The production of cellulose by *A. tumefaciens* strain C58 is controlled by two closely linked operons, *celABC* and *celDE*.¹⁰⁷ The genes *celA* and *celB* encode the two essential cellulose synthase subunits; whereas *celD* and *celE* codes for proteins responsible for the synthesis of UDP-glucose. The gene *celC* encodes a potential exporter that transports the cellulose fibrils into extracellular milieu. Synthesis of cellulose in *A. tumefaciens* is dependent on the DGC protein CelR. Overexpression of *celR* resulted in increased production of cellulose; whereas deletion of *celR* greatly reduced the amount of cellulose extractable from the cells.¹⁰⁸ CelR increases cellulose production likely by raising c-di-GMP levels to stimulate the activity of the PilZ domain-containing CelA glycosyltransferase subunit. Cellulose production in *A. tumefaciens* is also regulated by DivK, a response regulator that exerts its effect through CelR.¹⁰⁹ The involvement of a specific DGC or PDE protein such as CelR suggests that cellulose synthesis is regulated by a local pool of c-di-GMP. The presence of local c-di-GMP pools is not limited to the regulation of cellulose biosynthesis. It has in fact been suggested as one of the common phenomena of c-di-GMP signalling.⁶⁰,⁶¹,¹¹⁰–¹¹²

The wrinkly spreader (WS) genotype of *Pseudomonas fluorescens* SBW25 colonizes the air-liquid interface of spatially structured microcosms to form a thick biofilm.⁸⁵,¹¹³ Its ability to colonize this niche is largely due to overproduction of cellulosic polymer from the *wss* operon.¹¹⁴ Chemical analysis of the biofilm matrix showed that the cellulosic polymer is partially acetylated cellulose,¹¹⁵ which is also used by *P. fluorescens* SBW25 for colonizing plant surfaces. The *wss* operon is composed of 10 genes that encode the cellulose synthase subunits WssBCDE, acetylating proteins WssFGHI and two other proteins (WssA and WssJ) that are probably involved in positioning the enzyme complex.¹¹⁵ The involvement of WssGHI in acetylation is supported by genetic deletion and the observation that WssGHI are homologues of the alginate-acetylating proteins AlgFIJ from *P. aeruginosa*. Although the precise function of WssF remains to be fully determined, it was postulated that WssF is involved in delivering or presenting the acyl group to WssGHI.¹¹⁵ The production of acetylated cellulose by *P. fluorescens* is found to be stimulated by constitutive activation of WspR, a DGC that is functionally interchangeable with its ortholog from *P. aeruginosa*.⁸⁵,¹¹⁶,¹¹⁷ These observations suggest that cellulose production and biofilm formation by *P. fluorescens* SBS25 is regulated by a pool of c-di-GMP controlled by the Wsp chemosensory system.

In addition to post-translational regulation, c-di-GMP-dependent regulation of bacterial cellulose synthesis can also take place at the transcriptional level. For instance, c-di-GMP controls the binding of the transcriptional regulator Bcam1349 in *Burkholderia cenocepacia* to the promoter region of type 3 fimbriae and cellulose synthase operons. It was demonstrated that Bcam1349 binds to the promoter region of the cellulose synthase genes, and that the protein-DNA binding interaction is significantly enhanced at high c-di-GMP concentration.¹¹⁸,¹¹⁹

### 2.2. Alginate

Alginate are a family of linear polysaccharides composed of 1,4-linked β-d-mannuronic acid and α-l-guluronic acid. Unlike cellulose, some of the hydroxyl groups of the d-mannuronic acid units are O-acetylated at the C2′ and C3′ positions in a random manner. Alginate are commercially valuable polysaccharides that are widely used as thickening, stabilizing, and gelling agent in food, paper, textile and pharmaceutical industries.¹²⁰ Although alginate are mainly isolated from brown seaweeds, alginate are also known to be produced by two bacterial genera: *Pseudomonas* and *Azotobacter*. In addition to its role in providing additional structural strength to the cells, alginate is known to be crucial for the formation of *P. aeruginosa* biofilm in the airway of cystic fibrosis patients.¹²¹–¹²³ The mucoid phenotype of *P. aeruginosa* found in the bacterial biofilm of cystic fibrosis patients is characterized by the production of a large amount of alginate.¹²³,¹²⁴

Most of the information about bacterial alginate biosynthesis was gathered from the studies on *P. aeruginosa*. The *P. aeruginosa* alginate biosynthetic gene cluster (*alg* operon) comprises 12 genes that encode biosynthetic enzymes, transcriptional regulator, scaffold protein and transporters.¹²⁵ One additional gene (*algC*) that encodes the phosphomannomutase needed for precursor synthesis is located elsewhere in the genome. The biosynthetic proteins are assembled into a membrane-spanning multi-protein complex, with the biosynthetic enzymes located in both cytoplasmic and periplasmic spaces¹²⁵ (Fig. 5). The pathway can be divided into four stages that include: (i) production of the activated sugar-nucleotide precursor GDP-mannuronic acid from fructose-6-phosphate via the concerted actions of AlgA, AlgB, AlgC and AlgD. (ii) Polymerization catalyzed by Alg8 and Alg44. The inner-membrane-anchored Alg8 contains a cytoplasm-located family-

the correct assembly of the biosynthetic enzymes in the periplasmic space to initiate the polymerization process.

$P$. aeruginosa genomes encode a large number of GGDEL, EAL and HD-GYP proteins. Disruption of a single gene ($mucR$) that encodes a GGDEF-EAL didomain-containing protein in the mucoid alginate-producing $P$. aeruginosa strain PDO 300 resulted in a non-mucoid phenotype and a 38-fold decrease in alginate production. ${ }^{129}$ It was found that the c-di-GMP synthesizing and degrading activities of MucR are growth mode-dependent, with MucR exhibiting DGC activity during biofilm growth but PDE activity during planktonic growth modes. ${ }^{130}$ The observations indicate that MucR regulates alginate production by generating a localized c-di-GMP pool in the vicinity of Alg44 (Fig. 5).

Because the quantity and quality of alginate isolated from brown seaweeds vary greatly, it is hoped that alginate can be produced in the future by fermentation using a genetically tractable *Pseudomonas* or *Azotobacter* strain. Given the regulatory role of c-di-GMP in alginate biosynthesis, the production yield could be improved by manipulating cellular c-di-GMP level or engineering Alg8 or Alg44 homologs to create a constitutively active strain.

### 2.3. Poly-$\beta$-1,6-N-acetylglucosamine (PNAG)

The intercellular polysaccharide adhesin poly-$\beta$-1,6-N-acetylglucosamine (PNAG) polymer is essential for adherence and biofilm formation for several bacterial species. PNAG is positively charged linear homoglycan composed of $\beta$-1,6-N-acetylglucosamine residues with approximately 20% deacetylated residues. The PNAG polysaccharide family was first described in *Staphylococcus* species, and further in *E. coli*, *S. epidermidis* and other bacteria. ${ }^{131-134}$ Pathogenic *E. coli* produces PNAG to colonize surfaces and form biofilm for surviving in animal hosts. ${ }^{131,135}$ PNAG is being considered as a virulence factor in various staphylococcal infections and is a target for vaccination development.

The Pga machinery of *E. coli* for the synthesis and exporting of PNAG is encoded by the $pgaABCD$ operon. ${ }^{135,136}$ The human pathogen *Acinetobacter baumannii* also contains a $pgaABCD$ locus that encodes proteins for synthesizing cell-associated PNAG. ${ }^{137}$ The outer membrane-anchored PgaA and PgaB are required for PNAG export (Fig. 6A). PgaA transports the growing PNAG chain to the cell surface and PgaB deacetylates about 3% of the residues during PNAG export. PgaC is an inner-membrane-embedded glycosyltransferase with the catalytic domain exposed to the cytoplasm. PgaC assembles PNAG from the monomer uridine diphosphate N-acetylglucosamine (UDP-GlcNAc) using a processive mechanism. ${ }^{136}$ PgaD is a small protein with two transmembrane helices and is essential for PNAG synthesis. Its function remained unknown till the finding that c-di-GMP binds at the interface of PgaC and PgaD to stimulate the glycosyltransferase activity. ${ }^{138}$ Although the structural details are still missing, it was proposed that c-di-GMP binds to a pocket located at the PgaC-PgaD interface to promote the formation of an active glycosyltransferase complex. It was further suggested that PgaD is rapidly degraded to switch

This journal is © The Royal Society of Chemistry 2015

regulated by CsrA, a protein that binds to mRNA and inhibits YdeH expression. Together, these observations suggest that the regulation of PNAG biosynthesis by c-di-GMP occurs on multiple levels, and that PNAG production is controlled by local pools of c-di-GMP and specific DGC proteins.

As the etiologic agent of bubonic plague, *Yersinia pestis* causes transmissible infections between flea and mammals by growing as an attached biofilm in the foregut of the flea vector. Biofilm formation by *Y. pestis* within the flea enhances the regurgitation of bacteria into the dermis of the mammalian host and plays a significant role in the long-term maintenance of the plague bacillus in the wild. Formation of *Y. pestis* biofilm is dependent on the EPS synthesized by the proteins encoded by the *hmsHFRS* gene cluster. The *hms* genes are homologs of the *pga* (*E. coli*) and *ica* (*S. epidermidis*) genes that encode PNAG biosynthetic genes. In addition to gene homology, several lines of indirect evidence also support that *Y. pestis* produces a surface EPS composed of PNAG. It was proposed that PNAG may confer an advantage on *Y. pestis* by protecting the pathogen from antibiotics and host defenses after the establishment of the infection.

The production of PNAG and associated biofilm formation in *Y. pestis* is controlled by c-di-GMP, likely through a regulatory mechanism similar to that of *E. coli*. The genome of *Y. pestis* encodes six GGDEF proteins and six EAL proteins, but only two of the GGDEF proteins (HmsT and Y3730) and one EAL protein (HmsP) are predicted to be catalytically active. HmsT and HmsP were proposed to control PNAG biosynthesis via their opposing c-di-GMP synthesizing and degrading activities (Fig. 6B). In line with this view, the expression level of *hmsT* was found to correlate with c-di-GMP levels and biofilm thickness in *Y. pestis*. Deletion of *hmsT* or disruption of its GGDEF-encoding region resulted in poor biofilm formation; whereas elimination of HmsP or its EAL domain resulted in thicker biofilm. Perry and coworkers found evidence for cellular compartmentalization of soluble segments of Hms inner membrane proteins, including the glycosyltransferase domain of HmsR, the GGDEF domain of HmsT and the EAL domain of HmsP. The observations indicate that synthesis and regulation of the *Y. pestis* PNAG take place within a large enzymatic complex resided near the cytoplasmic membrane. Moreover, the expression of *hmsT* and *hmsP* seems to be under the control of several regulatory systems. The small RNA chaperone Hfq contributes to the repression of c-di-GMP production and biofilm formation through both positive transcriptional regulation of *hmsP* and negative post-transcriptional control of *hmsT*. The ferric uptake regulator (Fur), an iron-regulating system, acts as a repressor of biofilm formation by directly repressing the transcription of *hmsT* to decrease cellular c-di-GMP levels. *Y. pestis* biofilm is also negatively regulated by the Rcs phosphorelay system. The Rcs system is likely to mediate biofilm formation through modulating c-di-GMP levels and PNAG production since it controls *hmsT* transcription. Together, the current data support a model whereby PNAG production in *Y. pestis* is modulated by local c-di-GMP pools, the concentrations of which are once again controlled at multiple levels by different signaling systems.

Pectobacterium atrosepticum is a phytopathogenic enterobacterium that causes soft rot of potato tubers and blackleg disease in potato stems. *P*. atrosepticum could be induced to flocculate through c-di-GMP-dependent production of PNAG.¹³² The genome of *P*. atrosepticum encodes 23 c-di-GMP metabolizing proteins. A systematic study by overexpressing each of the 23 proteins revealed that the c-di-GMP metabolizing proteins are capable of modulating a wide range of c-di-GMP concentrations from 0.34 pmol (mg wet weight of bacteria)⁻¹ to 6936 pmol (mg wet weight of bacteria)⁻¹.¹⁵⁵ Overexpression of a few *P*. atrosepticum DGCs induced PNAG production and associated flocculation, supporting the regulatory role of c-di-GMP. In particular, induction of ECA3374 caused a significant increase in c-di-GMP levels and increased flocculation. PNAG expression could also be activated upon induction of the heterologous DGC protein PleD.

The plant pathogen Xanthomonas oryzae pv. oryzicola (Xoc) causes bacterial leaf streak in rice through the stomata or wounds on rice leaves. The Xoc genome encodes a number of putative glycosyltransferases from the *gumBCDK*, *pgaABCD* and *xagABCD* operons. The three operons are likely to be responsible for the production of xanthan gum, PNAG and another structurally undefined EPS, respectively. A total of 32 genes encoding HD-GYP, GGDEF and/or EAL domains were identified in the genome of Xoc BLS256.¹⁵⁶ One of the HD-GYP domain-containing proteins, RpfG, is a c-di-GMP degrading response regulator associated with the histidine kinase RpfC.¹⁷,¹⁵⁶ Mutation of *rpfG* led to altered expression of EPS genes, with the expression of *xag* and *pga* genes significantly up-regulated in the Δ*rpfG* mutant. The results indicate that high cellular levels of c-di-GMP are likely to stimulate the production of PNAG and the uncharacterized EPS in Xoc.

### 2.4. Pel polysaccharide

The Pel polysaccharide produced by *P*. aeruginosa not only serves as an intercellular adhesin for the formation and maintenance of biofilms, but has been implicated in enhanced tolerance to aminoglycoside antibiotics.⁷⁸,¹⁵⁷–¹⁵⁹ Pel polysaccharide is known to be a glucose-rich polysaccharide that is distinct from cellulose, though the chemical composition and structure of Pel polysaccharide have not been fully established. A locus (*pel*) that encompasses seven genes from the *P*. aeruginosa genome encodes the Pel biosynthetic and exporting machinery.¹⁶⁰ The *pelA–G* genes are highly conserved in different *P*. aeruginosa strains as well as the Gram-negative plant pathogen Ralstonia solanacearum. When mutation was introduced in the *pelG* gene of *R*. solanacearum, a biofilm-defective phenotype similar to that observed in *P*. aeruginosa *pel* mutants was observed.¹⁶⁰ Unlike alginate biosynthesis, which requires enzymes for the synthesis of the precursor GDP-mannuronate, the precursors for Pel originate from primary metabolic pathways. The sole glycosyltransferase encoded by the *pel* locus is the cytoplasm-resided PelF (Fig. 7). The polysaccharide chain is likely to be assembled by PelF and translocated across the cytoplasmic membranes by PelD, PelE, and PelG. The deacetylase activity of PelA is also important for the production of the final Pel polysaccharide.¹⁶¹

Pel biosynthesis in *P*. aeruginosa is positively regulated by c-di-GMP at post-translational level. The PelD protein contains a GAF domain and a non-enzymatic GGDEF domain that functions as a c-di-GMP binding domain³⁸ (Fig. 7). Analysis of PelD orthologs identified a number of conserved residues critical for c-di-GMP binding as well as the biosynthesis of Pel polysaccharide. The X-ray crystal structures of the cytoplasmic portion of PelD in complex with c-di-GMP have been determined.²⁷,⁴² The GGDEF domain functions as a c-di-GMP effector by using a binding site that normally serves as an allosteric inhibition site (I-site) in DGCs. PelD binds intercalating dimeric c-di-GMP with low μM affinity and replacement of the residues involved in binding not only reduced the affinity for c-di-GMP but also abolished Pel production.⁴²

While the *psl* genes in *P*. aeruginosa are constitutively expressed in planktonic cells,¹⁶² the *pel* genes were found to be induced upon attachment and biofilm formation.¹⁵⁸ In addition to the posttranslational regulation through PelD, Pel biosynthesis in *P*. aeruginosa is also regulated at transcriptional level by c-di-GMP. The transcriptional regulator FleQ represses *pel* transcription and the repression is relieved when cellular c-di-GMP levels are high.⁵⁰,⁵¹ This is because the binding of c-di-GMP to FleQ can induce the dissociation of FleQ from the *pelA* promoter (Fig. 7). FleN, an anti-activator of FleQ, also participates in the control of *pel* expression.

Pel biosynthesis is likely to be controlled by local pools of c-di-GMP produced by several DGC and PDE proteins. WspR is a GGDEF domain-containing response regulator that is phosphorylated by the Wsp chemotaxis-like regulatory system. Activated WspR produces c-di-GMP to bolster Pel biosynthesis and biofilm formation.¹⁶³–¹⁶⁵ In *P*. aeruginosa PA14, the DGC proteins RoeA and SadC positively regulate the production of Pel and contribute to biofilm formation.¹⁶⁶ The DGC protein YfiN or TpbB was also found to stimulate Pel production.¹⁶⁷,¹⁶⁸ YfiN forms a tripartite signalling system with YfiR and YfiB to sense and respond to an unknown extracellular signal. BifA contains an active EAL domain for regulating Pel production, as supported by the observations that the Δ*bifA* mutant exhibited increased c-di-GMP level relative to the wild type and increased synthesis of Pel.¹⁶⁹,¹⁷⁰ Fluorescent protein-tagging experiments suggested that WspR, RoeA and SadC are differentially located in the cells, suggesting that the DGC proteins may produce spatially separate c-di-GMP pools for the transcriptional and post-translational regulation of Pel production.¹⁶⁵,¹⁶⁶

### 2.5. Psl polysaccharide

Psl polysaccharide is composed of repeating pentamer consisting of D-mannose, L-rhamnose and D-glucose (Fig. 2).¹⁷¹ Psl polysaccharide was first identified from the *P*. aeruginosa strains that were not able to produce alginate but were still able to form biofilm on abiotic surfaces.¹⁷² Psl is believed to be important for biofilm formation based on the observation that deletion of *psl* deficient mutants lack the characteristic mushroom-like biofilm structure.¹⁵⁹,¹⁷³,¹⁷⁴ It has been suggested that Pel and Psl can

NPR

OM
IM
Wsp system
YfiB
YfiR
YfiN
Sad C
RoeA
BifA
PelD
GTP
GTP
GTP
5′-pGpG
Pel
PelB
PelC
PelA
PelE
PelG
PelF
substrates
GTP
WspR
c-di-GMP
FleQ
c-di-GMP
PelD
pelA pelB pelC pelD pelE pelF pelG

Fig. 7 Pel polysaccharide biosynthesis and regulation by c-di-GMP in *P. aeruginosa*. A structural model of the cytoplasmic portion of the c-di-GMP binding PelD based on the crystal structure of the PelD subunit (PDB: 4EUV) is also shown.

serve redundant functions as structural scaffolds in mature biofilms and that the redundancy could help preserve the capacity to produce a biofilm when EPS genes are subjected to mutation.¹⁵⁹ Psl polysaccharide may be distributed in a helical pattern surrounding the cell surface of *P. aeruginosa* to facilitate cell–cell interaction during biofilm formation.¹⁷⁵

An operon that contains 12 genes from the *P. aeruginosa* genome encodes the Psl biosynthetic and export machinery.¹⁷³ The *psl* operon seems to be temporally and spatially regulated during biofilm formation. The *psl* genes are constitutively expressed in planktonic *P. aeruginosa* cells and suppressed after surface attachment.¹⁶² During the development of biofilm structure, the expression levels of *psl* in the cells located at the centers of microcolonies increase.¹⁶² In contrast to the EPS biosynthetic pathways discussed above, the biosynthesis of Psl involves four glycosyltransferases (PslC, F, H and I) (Fig. 8). Also distinct from the biosynthesis of cellulose, alginate, Pel and PNAG, c-di-GMP does not seem to directly bind to any of the biosynthetic proteins for controlling Psl production. Instead, Psl production is positively regulated by c-di-GMP at the transcriptional level. The c-di-GMP responsive transcriptional regulator FleQ represses the transcription of *pel* and *psl* operons. This repression is relieved in the presence of c-di-GMP.⁵⁰,⁵¹ It is also found that the protein RsmA represses the translation of Psl proteins through binding to the ribosome binding site of *psl* mRNA to prevent ribosome access.¹⁷⁶ Considering that the RsmA is regulated by the cellular levels of the small RNAs *rsmZ* and *rsmZ*, and that the two small RNAs are intimately linked to cellular c-di-GMP levels,¹⁷⁷ it is feasible that

c-di-GMP also regulates Psl biosynthesis at the translational level.

Although biofilm matrix has long been assumed to play a passive structural and protective role for biofilm cells, a recent

OM
IM
SadC
SiaD
GTP
GTP
Psl
PsID
PsIG
PsIE
PslH
PslJ
PslK
PslL
PslI
PslF
PslC
PslB
Substrates
c-di-GMP
RsmA
mRNA
FleQ
pslA pslB pslC psID psIE pslF pslG pslH pslI pslJ pslK pslL

Fig. 8 Schematic illustration of the Psl polysaccharide synthesizing machinery and transcriptional regulation of *psl* genes by c-di-GMP in *P. aeruginosa*.

study suggested that the matrix may also play an active role in stimulating its own synthesis. It was suggested that Psl acts as the extracellular signal to stimulate two active DGCs, SiaD and SadC, to produce c-di-GMP in *P. aeruginosa*. Elevated intracellular concentrations of c-di-GMP further stimulate the production of Psl and other components of the biofilm. Such mechanism represents a positive feedback regulatory mechanism that is analogous to autocrine signalling in eukaryotes.

### 2.6. Curdlan

Agrobacterium species produces the EPS curdlan under nitrogen-limiting condition. Curdlan is a neutral water-insoluble EPS composed primarily of linear β-(1,3)-glycosidic linkages (Fig. 2). Curdlan is of special interest due to its gelation property and unique solubility in alkaline media and organic solvents. The genome of *Agrobacterium* sp. ATCC31749 contains 31 genes that encode GGDEF-domain containing proteins. The expression levels for three of the GGDEF-domain genes (*AGRO_0033*, *AGRO_0636*, *AGRO_3967*) were found to be up-regulated for more than two-fold under nitrogen-limiting conditions. Meanwhile, the *AGRO_3967* mutant showed a 57% decrease in curdlan production, whereas the Δ0033 and Δ0636 mutants produced curdlan at levels similar to that of the wild-type. These observations suggest that c-di-GMP in *Agrobacterium* sp. ATCC31749 is likely to regulate curdlan biosynthesis, and that the c-di-GMP pool controlled by *AGRO_3967* is the most important for controlling curdlan production.

### 2.7. Other exopolysaccharides

In addition to the six families of EPSs discussed above, the production of several structurally undefined EPSs produced by various bacterial species is also regulated by c-di-GMP as discussed below.

The rugose variants of the human pathogen *Vibrio cholerae* produce elevated levels of extracellular matrix by expressing *Vibrio* polysaccharide (VPS) genes and genes encoding matrix proteins. The genome of *V. cholerae* encodes 31 proteins with GGDEF domains, 12 proteins with EAL domains, and 10 that contain both GGDEF and EAL domains. Expression of *vps* genes is under the control of two positive transcriptional regulators, VpsT and VpsR. Biochemical and genetic studies have established VpsT as a transcriptional regulator that inversely regulates biofilm formation and motility by directly binding to c-di-GMP. C-di-GMP controls *Vibrio* polysaccharide synthesis by inducing the dimerization of VpsT to enhance the binding of VpsT to the promoter region of *vpsL*, the first gene of the *vps* operon. The crystal structure of VpsT in complex with dimeric c-di-GMP has been determined by Sondermann and coworkers. A model of the VpsT/c-di-GMP/DNA complex is shown in Fig. 9 to highlight the binding modes of dimeric c-di-GMP and DNA.

CdgC is a c-di-GMP-degrading PDE that regulates the expression of genes involved in VPS production, extracellular protein secretion, flagellar biosynthesis, and virulence factor production. CdgC, which represses the expression of *vps* genes during both exponential and stationary phases, is

---

**Fig. 9** Binding of DNA by the transcriptional regulator VpsT in the presence of c-di-GMP. The model of VpsT/c-di-GMP/DNA ternary complex is based on the crystal structure of the VspT/c-di-GMP binary complex. C-di-GMP is shown as sticks and two subunits of VpsT are shown in surface mode and the two other subunits in cartoon mode.

NPR

homologous to VpsR of *V. cholerae* and CpsR of *V. parahaemolyticus*, while BrpT was homologous to VpsT of *V. cholerae*. Disruption of the glycosyltransferase or regulator gene within the *brp* locus abated the inducing effect of c-di-GMP on biofilm formation, rugosity, and stress resistance. Depletion of cellular c-di-GMP abrogated these phenotypes in the rugose variant. The parental and *brp* mutant strains formed only scant monolayers on glass surfaces and oyster shells; whereas the rugose variant formed expansive biofilms with limited depth. The regulation of the *brp* genes in *V. vulnificus* represents yet another example of transcriptional regulation of EPS production by c-di-GMP.

The Gram-negative soil bacteria rhizobia are able to interact with compatible legumes through a complex process to form root or stem nodules, in which differentiated rhizobial cells convert atmospheric nitrogen into ammonia in exchange for carbon sources from host plants. *Sinorhizobium meliloti*, a model rhizobium species, produces at least three types of EPSs that include succinoglycan (EPSI), galactoglycan (EPSII) and capsule polysaccharide (KPS).<sup>195</sup> The EPSs are required for the successful establishment of efficient symbiosis between *S. meliloti* and alfalfa. The genome of *Sinorhizobium meliloti* encodes a total of 19 GGDEF and EAL domain-containing proteins. Disruption of 11 of these genes resulted in mutants that produced more EPSs and displayed less competitive nodulation on the host plant.<sup>196</sup>

The plant pathogen *Pseudomonas syringae* causes disease in a wide range of plants such as beans, tomato and rice. *P. syringae* uses a type III secretion system (T3SS) to deliver virulence factors into the plant to promote survival of the bacterium. The *P. syringae* T3SS is a product of the hypersensitive response and pathogenicity (*hrp*) and hypersensitive response and conserved (*hrc*) gene clusters, which are under the control of HrpR and HrpS. The *Chp8* gene embedded in the Hrp regulon of *P. syringae* pv. tomato DC3000 is expressed in response to plant signals and HrpRS.<sup>197</sup> Expression of *Chp8*, a bifunctional protein that contains enzymatically active GGDEF and EAL domains, increased the levels of cellular c-di-GMP. The high c-di-GMP level correlates with increased EPS production, decreased expression of the major pathogen-associated molecular pattern flagellin and enhanced pathogenicity.

In *Pseudomonas putida* KT2440, *rup4959* encodes a response regulator that contains both GGDEF and EAL domains. Overexpression of *rup4959* led to an increase in the global level of c-di-GMP, production of a Calcofluor stainable EPS, formation of biofilm and development of wrinkly colony morphology. A screen for the targets of c-di-GMP led to the identification of a gene cluster (*PP3133-PP3141*) that is responsible for the production of the surface polysaccharide.<sup>198</sup> The c-di-GMP binding effector involved in the regulation of the production of the strain-specific polysaccharide remains to be determined.

*Erwinia amylovora* is the causal agent of fire blight and a phytopathogen that infects plant species of the family Rosaceae, such as apple and pear trees. *E. amylovora* produces two distinct EPSs, amylovoran and levan, both of which contribute to plant infection.<sup>199</sup> Amylovoran is an acidic EPS composed of repeating units of galactose and glucuronic acid, with the

gene operon (*ams*) encoding amylovoran biosynthesis identified.<sup>200,201</sup> Biofilm formation by *E. amylovora* is crucial for effective colonization of host tissues and spread of the pathogen in trees. *E. amylovora* contains three DGC proteins encoded by *edcA*, *edcC*, and *edcE*. Phenotypic analyses demonstrated that c-di-GMP positively regulates the biosynthesis of amylovoran and biofilm formation while represses motility.<sup>202</sup> Disease assays on immature pears and apple tree shoots demonstrated that c-di-GMP negatively regulates virulence in these infection models.

The foodborne pathogen *Listeria monocytogenes* contains three DGCs (DgcA, DgcB and DgcC) and three PDEs (PdeB, PdeC and PdeD). Deletion of all three *pde* genes or overexpression of heterologous *dgc* genes stimulated the production of a structurally unknown EPS.<sup>203</sup> The EPS biosynthetic gene operon (*pssA-E*) contains an enzymatically inactive GGDEF domain protein (PssE) that functions as a c-di-GMP effector for regulating EPS synthesis. The c-di-GMP-inducible EPS causes cell aggregation in minimal medium and impairs bacterial migration in semi-solid agar, but does not promote biofilm formation on abiotic surfaces.<sup>203</sup> The EPS also greatly enhances bacterial tolerance to commonly used disinfectants as well as desiccation, which may contribute to survival of *L. monocytogenes* on contaminated food products and in food-processing facilities.

In *Burkholderia cenocepacia*, an opportunistic human pathogen that causes chronic infections in immunocompromised individuals and particularly in patients with cystic fibrosis, high c-di-GMP levels correlate with biofilm formation. Bcam1349, a member of the CRP/FNR family of transcriptional regulators, is a c-di-GMP responsive protein that regulates biofilm formation in *B. cenocepacia* H111.<sup>118</sup> Fazli *et al.* found that the structural stability of *B. cenocepacia* biofilm is enhanced by a major EPS produced by the gene cluster *Bcam1330-Bcam1341*.<sup>119</sup> Bcam1349 binds to the promoter region of the EPS gene cluster and that the protein-DNA binding is enhanced in the presence of c-di-GMP. Overproduction of both c-di-GMP and Bcam1349 leads to increased transcription of the EPS genes, indicating that c-di-GMP and Bcam1349 act together in regulating EPS production in *B. cenocepacia*.

In summary, bacterial EPS production is a carbon and energy-intensive process that is often tightly regulated at both transcriptional and post-transcriptional levels. The studies described above firmly establish c-di-GMP as an evolutionarily conserved regulator for bacterial EPS synthesis. It is remarkable to observe the diversity of c-di-GMP binding effectors employed by the microbes for regulating EPS production, with c-di-GMP binding to the PilZ domains in cellulose and alginate synthesis, the GGDEF domain of PelD, the transcriptional regulators VpsT and FleQ and the PgaC-PgaD complex. It is equally remarkable that the EPS biosynthesis seems to be regulated by c-di-GMP at different levels involving multiple signalling pathways. The regulation at transcriptional, translational and post-translational levels presumably allows the bacterial cells to respond to cellular, metabolic and environmental changes with different response dynamics.

Review

3. The emerging role of c-di-GMP in the regulation of secondary metabolite biosynthesis

The genomes of some well-known secondary metabolite-producing actinobacteria such as *Saccharopolyspora erythraea*, *S. coelicolor* and *Salinispora tropica* encode multiple c-di-GMP signalling proteins (Fig. 1D), suggesting the use of c-di-GMP by these bacteria as second messenger. Actinobacteria such as *S. coelicolor* often produce secondary metabolites at the transition between different developmental stages, such as the transition from vegetative mycelium to the building of aerial mycelium and spore.²⁰⁴ The production of secondary metabolite is considered to be a stage-dependent cellular response elicited by metabolic, nutritional, oxidative or other environmental signals. Given the role of the second messenger c-di-GMP in sensing environmental input and eliciting cellular response, it is logical to expect c-di-GMP to be involved in regulating secondary metabolism. Recent studies indeed provided evidence for the regulation of secondary metabolism by c-di-GMP in several bacteria species as discussed below.

### 3.1. Antibiotic production in *Streptomyces coelicolor*

Studies from Buttner and coworkers provided strong evidence for the regulation of secondary metabolism by c-di-GMP in *S. coelicolor*. In *S. coelicolor*, BldD is a transcriptional regulator essential for morphological development and antibiotic production.²⁰⁵,²⁰⁶ The highly pleiotropic BldD controls the transcription of more than 160 genes, including a number of genes that are involved in antibiotic production and cellulose synthesis. Three of the genes (*cdgA*, *cdgB* and *SCO5511*) that encode GGDEF or EAL domain-containing proteins were found to be controlled by BldD. Overexpression of *cdgA* in *S. coelicolor* M600 resulted in a strain that displayed an aerial hyphae-free bald phenotype and reduced production of the blue-pigmented antibiotic actinorhodin (1).²⁰⁶ The gene *cdgA* encodes a protein with a PAS-GGDEF-EAL domain arrangement, with the PAS domain as the putative sensor domain for modulating the activities of the C-terminal GGDEF and EAL domains.²⁰⁷,²⁰⁸ Mutations that abolish the DGC activity eliminated the bald phenotype and restored the production of actinorhodin, suggesting that cellular c-di-GMP concentration affects aerial hyphae formation and actinorhodin biosynthesis. The gene *cdgB* encodes a protein with a PAS-GGDEF domain arrangement, with the PAS or/and GAF domain as the potential sensory domain(s). CdgB contains an active GGDEF domain and exhibits DGC activity.²⁰⁵ Deletion or overexpression of *cdgB* inhibited aerial-mycelium formation on R2 and R5 media; whereas overexpression of *cdgB* suppresses the production of actinorhodin. The third gene *SCO5511* encodes a membrane-anchored protein that contains PAS, GGDEF and EAL domains. *SCO5511* is likely to play a role controlling global or local cellular c-di-GMP concentration, though whether it regulates secondary metabolism remains unknown. In addition to the regulation

of actinorhodin production, c-di-GMP is likely to regulate the activity of the cellulose synthase CslA located on the tips of growing hyphae for controlling mycelium formation.²⁰⁵

![Chemical Structure](https://i.imgur.com/actinorhodin.png)

**actinorhodin (1)**

The structures of the C-terminal domain of *S. venezuelae* BldD in complex with c-di-GMP as well as a 4.5 Å structure of the BldD-(c-di-GMP)-DNA complex were determined recently. The structures revealed that the C-terminal domain of BldD interacts with c-di-GMP using a unique c-di-GMP-binding signature sequence (RXD-X₈-RXXD).²⁰⁹ Binding of a tetrameric form, rather than the more common dimeric form of c-di-GMP, drives the dimerization of BldD to promote DNA binding. Hence, c-di-GMP acts as a small-molecule dimerizing regulator to control the DNA-binding activity of the transcriptional regulator BldD, which is similar to the regulatory mechanism of VpsT in *V. cholerae*.¹⁸²

### 3.2. Secondary metabolite production in *Pseudomonas aeruginosa*

The opportunistic pathogen *P. aeruginosa* is one of model microorganisms extensively used for the study of the role of c-di-GMP in biofilm formation and bacterial pathogenicity. *P. aeruginosa* contains a highly complex c-di-GMP signalling network consisting of over 50 c-di-GMP metabolizing proteins and effectors. During long-term lung colonization, *P. aeruginosa* undergoes phenotypic and genetic adaptation resulting in the progressive loss of virulence and the development of increased persistence. Distinct adaptive colony morphotypes such as mucoid colonies and small colony variants (SCVs) were found in the cystic fibrosis sputum samples. In the SCV form, *P. aeruginosa* adopts a hyper-adherent lifestyle that is implicated in immune evasion and long-term persistence of infection. Studies suggest that the SCV morphology is strongly correlated with cellular c-di-GMP levels.¹⁶⁴,²¹⁰,²¹¹ A genetic screen for SCV-related loci identified the *yfiBNR* operon, which encodes a tripartite signalling system that regulates c-di-GMP levels and EPS biosynthesis (Section 2.4, Fig. 7) in *P. aeruginosa*.¹⁶⁷ YfiN is a membrane-anchored c-di-GMP synthesizing DGC protein whose activity is tightly controlled by the periplasmic protein YfiR. Increased levels of the siderophores pyochelin (2) and pyoverdin (3) were observed for the Δ*yfiR* strain.¹⁶⁷,¹⁶⁸ The Δ*yfiR* strain also showed increased production of the phenazine pyocyanin (4). Given the roles of 2, 3 and 4 in bacterial infection, the observations indicate c-di-GMP is likely to contribute to bacterial pathogenesis through controlling the production of secondary metabolites.

NPR

pyocyanin (2)

Pyochelin (3)

pyoverdine (4)

tropodithietic acid (5)

### 3.3. Tropodithietic acid production in Ruegeria

The Roseobacter clade genera Ruegeria and Phaeobacter produce the potent antibiotic tropodithietic acid (TDA) (5). TDA has been studied for their antibacterial activity and as probiotics for marine aquaculture. For the Phaeobacter strains isolated from biofilms associated with fish and invertebrate larvae cultures, TDA production is greatly influenced by culture conditions, as evidenced by the observation that TDA production by Phaeobacter sp. 27-4 in marine broth only occurred under static growth conditions but not in shaken broth cultures.²¹² TDA production coincided with formation of a thick layer of biofilm at the air-liquid interface consisting of multicellular, star-shaped aggregates. A total of nine GGDEF and EAL domain-encoding genes were identified in *R. mobilis* F1926 genome. Overexpression of a *dgC* gene increased cellular c-di-GMP concentration and production of TDA; whereas overexpression of a *pde* gene decreased c-di-GMP concentration and TDA production.²¹³ Overexpression of the *dgC* and *pde* genes also led to changes in biofilm formation, with high c-di-GMP correlated with enhanced biofilm formation. The results suggest that c-di-GMP plays a central role in coordinating the production of TDA in *Ruegeria mobilis* to mediate motility and biofilm formation.

### 3.4. DKxanthenes and lichenicidin production in *Myxococcus xanthus*

*M. xanthus* is a Gram-negative bacterium that adopts two distinct life forms depending on the nutritional status of cells. In the shortage of nutrients, *M. xanthus* cells organize into spreading colonies and in the absence of nutrients cells aggregate to form fruiting bodies. The *M. xanthus* genome encodes at least 24 proteins involved in c-di-GMP metabolism and regulation. One of the c-di-GMP signalling proteins (SgmT) is a unique hybrid histidine kinase that contains a C-terminal GGDEF domain. Similar to the non-enzymatic GGDEF domain of PelD found in the biosynthesis of Pel polysaccharide, the GGDEF domain of SgmT functions as a c-di-GMP binding domain. The GGDEF domain is indispensable for the sequestration of SgmT in clusters localized along the cell length.²¹⁴

DKxanthene-534 (6)

Bliα (7)

Bliβ (8)

The c-di-GMP-binding SgmT and its downstream response regulator DigR are essential for the formation of extracellular matrix and type IV pili-dependent motility. It was found that the SgmT-associated signalling pathway modulates the transcription of over 100 genes, including five genes that code for biosynthetic enzymes. Deletion of *SgmT* or *DigR* resulted in a down-regulation of the expression of the five biosynthetic genes.²¹⁴ Three of the five genes are involved in the biosynthesis of DKxanthene-534 (6), a yellow polyene pigment that give *M. xanthus* colonies the characteristic yellow color;²¹⁵,²¹⁶ whereas the other two are predicted to be involved in the biosynthesis of the lantibiotic lichenicidin consisting of the Bliα (7) and Bliβ (8) subunits.²¹⁷–²¹⁹ However, despite the effect of gene deletion on the expression of the biosynthetic genes, the direct involvement

of c-di-GMP in the production of 6, 7 and 8 has not been demonstrated. Measuring the production of 6, 7 or 8 for the mutants in the future with deleted or overexpressed *dgc* or *pde* genes will be essential for establishing the role of c-di-GMP in secondary metabolite biosynthesis in *M. xanthus*.

### 4. C-di-GMP signalling – from input signal to EPS and secondary metabolite

The production of bacterial EPS and secondary metabolites are tightly regulated processes that can be up- or down-regulated by environmental signals. C-di-GMP is likely to play an important role in sensing and responding to environmental signals, as evidenced by the observation that many GGDEF, EAL and HD-GYP domains involved in the biosynthesis of EPS and secondary metabolite are associated with sensory or phospho-receiver (REC) domains. Judging by the diversity of the sensory domains found in c-di-GMP signaling proteins, the production of EPS and secondary metabolite is likely to be regulated by a wide range of environmental signals that reflect the diverse ecological niches. With a small number of exceptions as discussed below, the majority of the environmental signals controlling the production of EPS and secondary metabolite through c-di-GMP-dependent pathways remain to be determined.

Most c-di-GMP-dependent pathways involved in sensing input signals contain the so-called one-component or two-component system. The one-component systems involve a single protein that contains a sensory domain and one or two c-di-GMP metabolizing domains. One of the earliest one-component systems known to regulate EPS biosynthesis is the O₂-sensing AxPDE1 from *G. xylinum*. By using a heme-binding PAS domain, AxPDE1 responds to rising O₂ level by suppressing the activity of the PDE domain to increase c-di-GMP concentration. Another example is the *G. xylinum* AxDGC2 protein, which uses a redox-responsive flavin-binding PAS domain to modulate the activity of the DGC domain in synthesizing c-di-GMP. The use of AxPDE1, AxDGC2 and their paralogs allows *G. xylinum* to control cellulose production in response to O₂ levels (Fig. 3A). A two-gene operon that encodes DosC (previously EcDos or YddV) and DocP (or YddU) regulates the transcription of *pga* operon for PNAG biosynthesis in *E. coli* (Fig. 6A). DosC is a DGC protein with a globin-coupled O₂ sensor; whereas DosP is a PDE Protein with a heme-binding PAS domain that also senses O₂. DosP and DosC interact with each other to form a functional complex for controlling c-di-GMP concentration and PNAG production according to O₂ availability. In addition to the DosC/P system, PNAG biosynthesis is also regulated by the GGDEF domain-containing YdeH (or DgcZ) protein (Fig. 6B). The sensory domain of YdeH contains a 3His/1Cys site for binding Zn²⁺ ion. The DGC activity of YdeH is inhibited at high Zn²⁺ concentration, likely because the binding of Zn²⁺ ion impedes the formation of the functional dimer. In addition to small molecules such as O₂ and metal ion, environmental signals can be extracellular protein, EPS and other polymeric substrates. As discussed in Section 2.5, the DGC

proteins SadC and SiaD from *P. aeruginosa* have been suggested to be involved in the sensing of extracellular Psl. Moreover, the protein YeaJ from *S. Typhimurium* contains a periplasmic Cache domain that functions as a sensory domain for sensing Sal4 antibody. The interaction between the Cache domain and Sal4 antibody controls EPS and biofilm formation through modulating the activity of the C-terminal GGDEF domain. Studies have also uncovered atypical one-component systems that use discrete sensory domains to modulate the activity of DGC or PDE domain through protein-protein interaction. For example, the transmembrane protein MbaA from *V. cholerae* is a GGDEF-EAL protein that regulates EPS production and biofilm formation. The PDE activity of MbaA is modulated by the stand-alone sensory protein NspS, which is located in the periplasmic space for binding the signal norspermidine.

Input signals have also been identified for a number of c-di-GMP-dependent one-component systems that are not related to the biosynthesis of EPS or secondary metabolite. The examples include the O₂-responsive RbdA from *P. aeruginosa* and HmsP from *Yersinia pestis*, cGMP-sensing XC-0249 from *Xanthomonas campestris* and several light responsive proteins from cyanobacteria and proteobacteria. Bacteria have also evolved sensory domains for sensing autoinducers or quorum-sensing (QS) signals. The membrane permeable QS signal cis-2-dodecenoic acid negatively controls c-di-GMP level through the receptor protein RpfR in *B. cenocepacia*. The signal cis-2-dodecenoic acid binds directly to the PAS domain of RpfR to stimulates the PDE activity of RpfR through an allosteric conformational change. The heme-bound H-NOX domain is a nitric oxide (NO) sensory domain found in *P. aeruginosa*, *Shewanella* and other bacterial species. By interacting with GGDEF, EAL, HD-GYP or histidine kinase proteins, the H-NOX domain proteins are able to regulate cellular c-di-GMP levels and biofilm formation in response to changes in NO concentration.

In addition to the one-component systems, bacteria also use two-component systems or chemotaxis-like systems for sensing environmental signals. An archetypical two-component system consists of a histidine kinase or a chemoreceptor and the cognate response regulators or CheY-like proteins. The histidine kinases usually contain an N-terminal sensory domain located in the membrane or periplasmic space for sensing environmental signals. By binding to the sensory domain, the input signal controls phospho-relay between the histidine kinase and its downstream response regulator. The majority of signals perceived by the sensory domains of two-component systems remain speculative at this moment. A large number of response regulators that contain a GGDEF, EAL or HD-GYP domain are encoded by bacterial genomes. Modulation of DGC activity as a result of phosphorylation of N-terminal REC domains has been demonstrated for a few GGDEF/EAL proteins. By using the c-di-GMP-synthesizing and degrading response regulators, bacteria are able to translate environmental inputs into higher or lower c-di-GMP levels through the phosphorelay systems.

By integrating the one-component and two-component systems into multi-tiered signalling cascades, bacteria are able to sense and respond to environmental signals and control the

NPR

production of EPS or secondary metabolites by modulating cellular c-di-GMP levels. The use of different sensory domains in co-existing c-di-GMP signaling systems enables the bacterium to respond to different environmental signals. The environmental signals will trigger a change in global or local c-di-GMP concentration to elicit specific cellular responses at transcriptional or post-transcriptional level through specific c-di-GMP-binding receptors.

### 5. Summary and outlook

Studies have firmly established the role of c-di-GMP in the regulation of bacterial EPS biosynthesis, with high c-di-GMP levels stimulating the production of EPSs. The regulatory role of c-di-GMP in EPS biosynthesis seems to be evolutionarily conserved across different bacterial species, with structurally distinct c-di-GMP binding effectors involved in the regulatory mechanisms. So far, c-di-GMP has been found to bind to PilZ and the I-site of non-enzymatic GGDEF domain, as well as the PgaC/PgaD complex and several transcriptional regulators to regulate EPS biosynthesis. Another important view emerged from recent studies is that it is common for EPS biosynthesis to be regulated by multi-tiered c-di-GMP signalling networks, with the regulation often occurring at the transcriptional and post-translational levels. The studies also suggest that EPS biosynthesis is often controlled by local c-di-GMP pools and highly specific c-di-GMP signalling pathways, rather than by the global c-di-GMP pool.

Some of the biodegradable bacterial EPSs such as alginate, cellulose and curdlan are already considered to be commercially valuable biopolymers. Besides the applications in the health and biotechnology industries, polysaccharides are also used as thickeners, bioadhesives, stabilizers, probiotic, and gelling agents in food and cosmetic industries and as emulsifier, biosorbent, and bioflocculant in the environmental sector. A better understanding of the role of c-di-GMP in EPS biosynthesis may eventually enable large-scale production of the bacterial polysaccharides by metabolic engineering approaches. The knowledge can also be instrumental for the engineering of biofilm-forming strains. For example, the conductive biofilms of *Geobacter sulfurreducens* have potential applications in renewable energy, bioremediation, and bioelectronics. To promote biofilm formation on electrode, a strain was constructed by deleting a PilZ protein-encoding gene. The mutant strain formed biofilm more effectively on electrode, likely due to the increased production of EPS and pili.

Although the full scope of c-di-GMP regulation in secondary metabolism remains to be seen, the issue of whether c-di-GMP regulates secondary metabolism is beyond dispute. The current number of secondary metabolic pathways known to be regulated by c-di-GMP is still relatively small. This is probably due to the fact that the bacterial systems employed to study c-di-GMP signalling such as *E. coli* and *P. aeruginosa* only contain a small number of secondary biosynthetic pathways. Hence, the current examples from the literature are likely to represent the tip of the iceberg. The number of secondary biosynthetic pathways regulated by c-di-GMP is likely to increase substantially with the focus shifting towards some of the slow-growing Gram-positive bacteria such as actinomycetes. Secondary metabolite profiling in conjunction with systematic deletion or overexpression of DGC and PDE proteins will potentially uncover c-di-GMP-dependent biosynthetic pathways in genetically tractable actinomycetes.

Bacterial secondary metabolites have long been an important source of bioactive compounds or drug leads. Many secondary metabolic pathways are inactive or silent under the culturing conditions in research laboratories due to the absence of environmental cues. Activating such cryptic pathways represents an attractive strategy for discovering novel microbial natural products. Some actinobacteria that are known to contain cryptic secondary biosynthetic pathways such as *S. erythraea* and *S. tropica* harbor extensive c-di-GMP signaling networks, as indicated by the number of GGDEF, EAL and HD-GYP proteins encoded by the bacterial genomes (Fig. 1D). By overexpressing endogenous or exogenous DGC or PDE proteins, we may be able to activate the cryptic biosynthetic pathways that are directly regulated by c-di-GMP. This strategy has recently been demonstrated by the activation of a silent EPS pathway in *P. atrosepticum* by expressing an exogenous DGC. In addition, the artificially adjusted levels of c-di-GMP can trigger shifts in gene expression, mimicking the shifts observed during the transition between different developmental stages or life styles to cope with environmental changes. If the secondary metabolite is essential for the new lifestyle or developmental stage, the perturbation of cellular c-di-GMP levels may “trick” the microbe into activating the biosynthetic pathway even if the biosynthetic pathway is not directly regulated by c-di-GMP.

### 6. Acknowledgements

The research work on the role of c-di-GMP in bacterial biofilm and pathogenicity in our lab is supported by a generous academic research Tier II grant (Z.-X.L.) from the Ministry of Education of Singapore.

### 7. Notes and references

1. P. Ross, H. Weinhouse, Y. Aloni, D. Michaeli, P. Weinberger-Ohana, R. Mayer, S. Braun, E. de Vroom, G. A. van der Marel, J. H. van Boom and M. Benziman, *Nature*, 1987, **325**, 279–281.
2. P. Ross, R. Mayer, H. Weinhouse, D. Amikam, Y. Huggirat, M. Benziman, E. de Vroom, A. Fidder, P. de Paus and L. A. Sliedregt, *J. Biol. Chem.*, 1990, **265**, 18933–18943.
3. M. Y. Galperin, A. N. Nikolskaya and E. V. Koonin, *FEMS Microbiol. Lett.*, 2001, **203**, 11–21.
4. H.-C. Flemming and J. Wingender, *Nat. Rev. Microbiol.*, 2010, **8**, 623–633.
5. N. Hoiby, T. Bjarnsholt, M. Givskov, S. Molin and O. Ciofu, *Int. J. Antimicrob. Agents*, 2010, **35**, 322–332.
6. C. A. Fux, J. W. Costerton, P. S. Stewart and P. Stoodley, *Trends Microbiol.*, 2005, **13**, 34–40.
7. L. Hall-Stoodley, J. W. Costerton and P. Stoodley, *Nat. Rev. Microbiol.*, 2004, **2**, 95–108.

Review

8 P. S. Stewart and J. W. Costerton, *Lancet*, 2001, 358, 135–138.

9 R. Tal, H. C. Wong, R. Calhoon, D. Gelfand, A. L. Fear, G. Volman, R. Mayer, P. Ross, D. Amikam, H. Weinhouse, A. Cohen, S. Sapir, P. Ohana and M. Benziman, *J. Bacteriol.*, 1998, 180, 4416–4425.

10 N. Ausmees, R. Mayer, H. Weinhouse, G. Volman, D. Amikam, M. Benziman and M. Lindberg, *FEMS Microbiol. Lett.*, 2001, 204, 163–167.

11 R. Simm, M. Morr, A. Kader, M. Nimtz and U. Romling, *Mol. Microbiol.*, 2004, 53, 1123–1134.

12 D. A. Ryjenkov, M. Tarutina, O. V. Moskvin and M. Gomelsky, *J. Bacteriol.*, 2005, 187, 1792–1798.

13 R. Paul, *Genes Dev.*, 2004, 18, 715–727.

14 A. J. Schmidt, D. A. Ryjenkov and M. Gomelsky, *J. Bacteriol.*, 2005, 187, 4774–4781.

15 R. Tamayo, A. D. Tischler and A. Camilli, *J. Biol. Chem.*, 2005, 280, 33324–33330.

16 R. P. Ryan, Y. Fouhy, J. F. Lucey, L. C. Crossman, S. Spiro, Y.-W. He, L.-H. Zhang, S. Heeb, M. Camara, P. Williams and J. M. Dow, *Proc. Natl. Acad. Sci. U. S. A.*, 2006, 103, 6712–6717.

17 M. O. Andrade, *Mol. Microbiol.*, 2006, 62, 537–551.

18 P. Wassmann, C. Chan, R. Paul, A. Beck, H. Heerklotz, U. Jenal and T. Schirmer, *Structure*, 2007, 15, 915–927.

19 N. De, *PLoS Biol.*, 2008, 6, e67.

20 C. Y. Yang, K. H. Chin, M. L. Chuah, Z. X. Liang, A. H. Wang and S. H. Chou, *Acta Crystallogr.*, Sect. D: *Biol. Crystallogr.*, 2011, 67, 997–1008.

21 F. Zahringer, E. Lacanna, U. Jenal, T. Schirmer and A. Boehm, *Structure*, 2013, 21, 1149–1157.

22 A. Deepthi, C. W. Liew, Z. X. Liang, K. Swaminathan and J. Lescar, *PLoS One*, 2014, 9, e110912.

23 T. R. M. Barends, E. Hartmann, J. J. Griese, T. Beitlich, N. V. Kirienko, D. A. Ryjenkov, D. A. Reinstein, R. I. Shoeman, M. Gomelsky and I. Schlichting, *Nature*, 2009, 459, 1015–1018.

24 M. W. Chen, M. Kotaka, C. Vonrhein, G. Bricogne, F. Rao, M. L. Chuah, D. Svergun, G. Schneider, Z. X. Liang and J. Lescar, *J. Bacteriol.*, 2012, 194, 4837–4846.

25 D. Bellini, D. L. Caly, Y. McCarthy, M. Bumann, S. Q. An, J. M. Dow, R. P. Ryan and M. A. Walsh, *Mol. Microbiol.*, 2014, 91, 26–38.

26 E. Wigren, Z. X. Liang and U. Romling, *Mol. Microbiol.*, 2014, 91, 1–5.

27 Z. Li, J. H. Chen, Y. Hao and S. K. Nair, *J. Biol. Chem.*, 2012, 287, 30191–30204.

28 C. Chan, R. Paul, D. Samoray, N. C. Amiot, B. Giese, U. Jenal and T. Schirmer, *Proc. Natl. Acad. Sci. U. S. A.*, 2004, 101, 17084–17089.

29 F. Rao, Y. Yang, Y. Qi and Z. X. Liang, *J. Bacteriol.*, 2008, 190, 3622–3631.

30 A. L. Lovering, M. J. Capeness, C. Lambert, L. Hobley and R. E. Sockett, *mBio*, 2011, 2, e00163-11.

31 G. Minasov, S. Padavattan, L. Shuvalova, J. S. Brunzelle, D. J. Miller, A. Basle, C. Massa, F. R. Collart, T. Schirmer and W. F. Anderson, *J. Biol. Chem.*, 2009, 284, 13174–13184.

32 M. Y. Galperin, *BMC Microbiol.*, 2005, 5, 35.

33 A. S. Seshasayee, G. M. Fraser and N. M. Luscombe, *Nucleic Acids Res.*, 2010, 38, 5970–5981.

34 R. P. Ryan, T. Tolker-Nielsen and J. M. Dow, *Trends Microbiol.*, 2012, 20, 235–242.

35 D. Amikam and M. Y. Galperin, *Bioinformatics*, 2006, 22, 3–6.

36 D. A. Ryjenkov, R. Simm, U. Romling and M. Gomelsky, *J. Biol. Chem.*, 2006, 281, 30310–30314.

37 M. Christen, *Proc. Natl. Acad. Sci. U. S. A.*, 2007, 104, 4112–4117.

38 V. T. Lee, J. M. Matewish, J. L. Kessler, M. Hyodo, Y. Hayakawa and S. Lory, *Mol. Microbiol.*, 2007, 65, 1474–1484.

39 J. Benach, *EMBO J.*, 2007, 26, 5153–5166.

40 A. Duerig, S. r. Abel, M. Folcher, M. Nicoller, T. Schwede, N. Amiot, B. Giese and U. Jenal, *Genes Dev.*, 2009, 23, 93–104.

41 S. Abel, P. Chien, P. Wassmann, T. Schirmer, V. Kaever, M. T. Laub, T. A. Baker and U. Jenal, *Mol. Cell*, 2011, 43, 550–560.

42 J. C. Whitney, K. M. Colvin, L. S. Marmont, H. Robinson, M. R. Parsek and P. L. Howell, *J. Biol. Chem.*, 2012, 287, 23582–23593.

43 M. V. A. S. Navarro, N. De, N. Bae, Q. Wang and H. Sondermann, *Structure*, 2009, 17, 1104–1116.

44 Y. Qi, M. L. C. Chuah, X. Dong, K. Xie, Z. Luo, K. Tang and Z.-X. Liang, *J. Biol. Chem.*, 2011, 286, 2910–2917.

45 Y. Qi, L. Xu, X. Dong, Y. H. Yau, C. L. Ho, S. L. Koh, S. G. Shochat, S. H. Chou, K. Tang and Z. X. Liang, *J. Bacteriol.*, 2012, 194, 5922–5931.

46 P. D. Newell, R. D. Monds and G. A. O'Toole, *Proc. Natl. Acad. Sci. U. S. A.*, 2009, 106, 3461–3466.

47 M. V. Navarro, P. D. Newell, P. V. Krasteva, D. Chatterjee, D. R. Madden, G. A. O'Toole and H. Sondermann, *PLoS Biol.*, 2011, 9, 1000588.

48 P. D. Newell, C. D. Boyd, H. Sondermann and G. A. O'Toole, *PLoS Biol.*, 2011, 9, 1000587.

49 J. R. Tuckerman, G. Gonzalez and M. A. Gilles-Gonzalez, *J. Mol. Biol.*, 2011, 407, 633–639.

50 J. W. Hickman and C. S. Harwood, *Mol. Microbiol.*, 2008, 69, 376–389.

51 C. Baraquet, K. Murakami, M. R. Parsek and C. S. Harwood, *Nucleic Acids Res.*, 2012, 40, 7207–7218.

52 N. Sudarsan, E. R. Lee, Z. Weinberg, R. H. Moy, J. N. Kim, K. H. Link and R. R. Breaker, *Science*, 2008, 321, 411–413.

53 A. G. Chen, N. Sudarsan and R. R. Breaker, *RNA*, 2011, 17, 1967–1972.

54 K. D. Smith and S. A. Strobel, *Biochem. Soc. Trans.*, 2011, 39, 647–651.

55 K. D. Smith, S. V. Lipchock, T. D. Ames, J. M. Wang, R. R. Breaker and S. A. Strobel, *Nature Struct. Mol. Biol.*, 2009, 16, 1218–U1227.

56 M. Merighi, V. T. Lee, M. Hyodo, Y. Hayakawa and S. Lory, *Mol. Microbiol.*, 2007, 65, 876–895.

57 X. Fang, I. Ahmad, A. Blanka, M. Schottkowski, A. Cimdins, M. Y. Galperin, U. Römling and M. Gomelsky, *Mol. Microbiol.*, 2014, 93, 439–452.

NPR

58 J. R. Chambers, J. Liao, M. J. Schurr and K. Sauer, Mol. Microbiol., 2014, 92, 471–487.

59 I. S. Pultz, M. Christen, H. D. Kulasekara, A. Kennard, B. Kulasekara and S. I. Miller, Mol. Microbiol., 2012, 86, 1424–1440.

60 J. P. Massie, E. L. Reynolds, B. J. Koestler, J.-P. Cong, M. Agostoni and C. M. Waters, Proc. Natl. Acad. Sci. U. S. A., 2012, 109, 12746–12751.

61 P. D. Newell, S. Yoshioka, K. L. Hvorecny, R. D. Monds and G. A. O'Toole, J. Bacteriol., 2011, 193, 4685–4698.

62 C. Solano, B. a. Garc, C. Latasa, A. Toledo-Arana, V. Zorraquino, J. Valle, J. Casals, E. Pedroso and I. i. Lasa, Proc. Natl. Acad. Sci. U. S. A., 2009, 106, 7997–8002.

63 S. Ozaki, A. Schalch-Moser, L. Zumthor, P. Manfredi, A. Ebbensgaard, T. Schirmer and U. Jenal, Mol. Microbiol., 2014, 94, 580–594.

64 U. Römling, M. Y. Galperin and M. Gomelsky, Microbiol. Mol. Biol. Rev., 2013, 77, 1–52.

65 H. Sondermann, N. J. Shikuma and F. H. Yildiz, Curr. Opin. Microbiol., 2012, 15, 140–146.

66 R. Hengge, Nat. Rev. Microbiol., 2009, 7, 263–273.

67 T. Schirmer and U. Jenal, Nat. Rev. Microbiol., 2009, 7, 724–735.

68 D. A. D'Argenio and S. I. Miller, Microbiology, 2004, 150, 2497–2502.

69 E. Mills, I. S. Pultz, H. D. Kulasekara and S. I. Miller, Cell. Microbiol., 2011, 13, 1122–1129.

70 U. Romling, M. Gomelsky and M. Y. Galperin, Mol. Microbiol., 2005, 57, 629–639.

71 P. A. Cotter and S. Stibitz, Curr. Opin. Microbiol., 2007, 10, 17–23.

72 H. Koo, M. L. Falsetta and M. I. Klein, J. Dent. Res., 2013, 92, 1065–1073.

73 J. A. Leigh and D. L. Coplin, Annu. Rev. Microbiol., 1992, 46, 307–346.

74 I. W. Sutherland, Microbiology, 2001, 147, 3–9.

75 D. O. Serra, A. M. Richter and R. Hengge, J. Bacteriol., 2013, 195, 5540–5554.

76 P. I. Watnick and R. Kolter, Mol. Microbiol., 1999, 34, 586–595.

77 P. N. Danese, L. A. Pratt and R. Kolter, J. Bacteriol., 2000, 182, 3593–3596.

78 L. Friedman and R. Kolter, Mol. Microbiol., 2004, 51, 675–690.

79 U. U. Nwodo, E. Green and A. I. Okoh, Int. J. Mol. Sci., 2012, 13, 14002–14015.

80 S. Pereira, A. Zille, E. Micheletti, P. Moradas-Ferreira, R. De Philippis and P. Tamagnini, FEMS Microbiol. Rev., 2009, 33, 917–941.

81 U. Römling, Res. Microbiol., 2002, 153, 205–212.

82 Y. Kawano, T. Saotome, Y. Ochiai, M. Katayama, R. Narikawa and M. Ikeuchi, Plant Cell Physiol., 2011, 52, 957–966.

83 C. Solano, B. García, J. Valle, C. Berasain, J.-M. Ghigo, C. Gamazo and I. Lasa, Mol. Microbiol., 2002, 43, 793–808.

84 X. Zogaj, M. Nimtz, M. Rohde, W. Bokranz and U. Römling, Mol. Microbiol., 2001, 39, 1452–1463.

85 S. Ude, D. L. Arnold, C. D. Moon, T. Timms-Wilson and A. J. Spiers, Environ. Microbiol., 2006, 8, 1997–2011.

86 A. G. Matthysse, R. Deora, M. Mishra and A. G. Torres, Appl. Environ. Microbiol., 2008, 74, 2384–2390.

87 G. Smit, S. Swart, B. J. J. Lugtenberg and J. W. Kijne, Mol. Microbiol., 1992, 6, 2897–2903.

88 A. G. Matthysse, Front. Plant Sci., 2014, 5, 252.

89 W. Bokranz, X. Wang, H. Tschape and U. Romling, J. Med. Microbiol., 2005, 54, 1171–1182.

90 J. L. W. Morgan, J. Strumillo and J. Zimmer, Nature, 2013, 493, 181–186.

91 J. L. W. Morgan, J. T. McNamara and J. Zimmer, Nat. Struct. Mol. Biol., 2014, 21, 489–496.

92 O. Omadjela, A. Narahari, J. Strumillo, H. Melida, O. Mazur, V. Bulone and J. Zimmer, Proc. Natl. Acad. Sci. U. S. A., 2013, 110, 17856–17861.

93 N. Sunagawa, T. Fujiwara, T. Yoda, S. Kawano, Y. Satoh, M. Yao, K. Tajima and T. Dairi, J. Biosci. Bioeng., 2013, 115, 607–612.

94 S.-Q. Hu, Y.-G. Gao, K. Tajima, N. Sunagawa, Y. Zhou, S. Kawano, T. Fujiwara, T. Yoda, D. Shimura, Y. Satoh, M. Munekata, I. Tanaka and M. Yao, Proc. Natl. Acad. Sci. U. S. A., 2010, 107, 17957–17961.

95 H. Weinhouse, S. Sapir, D. Amikam, Y. Shilo, G. Volman, P. Ohana and M. Benziman, FEBS Lett., 1997, 416, 207–211.

96 T. Fujiwara, K. Komoda, N. Sakurai, K. Tajima, I. Tanaka and M. Yao, Biochem. Biophys. Res. Commun., 2013, 4, 02463–02461.

97 Y. Qi, F. Rao and Z.-X. Liang, Biochemistry, 2009, 48, 10275–10285.

98 A. L. Chang, J. R. Tuckerman, G. Gonzalez, R. Mayer, H. Weihouse, G. Volman, D. Amikam, M. Benziman and M. A. Gilles-Gonzalez, Biochemistry, 2001, 40, 3420–3426.

99 O. Mazur and J. Zimmer, J. Biol. Chem., 2011, 286, 17601–17606.

100 E. Brombacher, A. Baratto, C. Dorel and P. Landini, J. Bacteriol., 2006, 188, 2027–2037.

101 H. Weber, C. Pesavento, A. Possling, G. Tischendorf and R. Hengge, Mol. Microbiol., 2006, 62, 1014–1034.

102 U. Gerstel, C. Park and U. Romling, Mol. Microbiol., 2003, 49, 639–654.

103 U. Romling, W. D. Sierralta, K. Eriksson and S. Normark, Mol. Microbiol., 1998, 28, 249–264.

104 S. Da Re and J.-M. Ghigo, J. Bacteriol., 2006, 188, 3073–3087.

105 J. J. Amarasinghe, R. E. D'Hondt, C. M. Waters and N. J. Mantis, Infect. Immun., 2013, 81, 653–664.

106 C. M. Bassiss and K. L. Visick, J. Bacteriol., 2010, 192, 1269–1278.

107 A. G. Matthysse, D. L. Thomas and A. R. White, J. Bacteriol., 1995, 177, 1076–1081.

108 D. M. Barnhart, S. Su, B. E. Baccaro, L. M. Banta and S. K. Farrand, Appl. Environ. Microbiol., 2013, 79, 7188–7202.

109 D. M. Barnhart, S. Su and S. K. Farrand, J. Bacteriol., 2014, 196, 1257–1274.

110 R. Simm, A. Lusch, A. Kader, M. Andersson and U. Romling, J. Bacteriol., 2007, 189, 3613–3623.

Review

111 A. Boehm, M. Kaiser, H. Li, C. Spangler, C. A. Kasper, M. Ackermann, V. Kaever, V. Sourjik, V. Roth and U. Jenal, *Cell*, 2010, 141, 107–116.

112 S. Abel, T. Bucher, M. Nicoller, I. Hug, V. Kaever, P. Abel zur Wiesch and U. Jenal, *PLoS Genet.*, 2013, 9, e1003744.

113 A. Koza, P. D. Hallett, C. D. Moon and A. J. Spiers, *Microbiology*, 2009, 155, 1397–1406.

114 A. J. Spiers and P. B. Rainey, *Microbiology*, 2005, 151, 2829–2839.

115 A. J. Spiers, J. Bohannon, S. M. Gehrig and P. B. Rainey, *Mol. Microbiol.*, 2003, 50, 15–27.

116 P. Goymen, S. G. Kahn, J. G. Malone, S. M. Gehrig, A. J. Spiers and P. B. Rainey, *Genetics*, 2006, 173, 515–526.

117 J. G. Malone, R. Williams, M. Christen, U. Jenal, A. J. Spiers and P. B. Rainey, *Microbiology*, 2007, 153, 980–994.

118 M. Fazli, A. O'Connell, M. Nilsson, K. Niehaus, J. M. Dow, M. Givskov, R. P. Ryan and T. Tolker-Nielsen, *Mol. Microbiol.*, 2011, 82, 327–341.

119 M. Fazli, Y. McCarthy, M. Givskov, R. P. Ryan and T. Tolker-Nielsen, *MicrobiologyOpen*, 2012, 25, 61.

120 B. H. Rehm and S. Valla, *Appl. Microbiol. Biotechnol.*, 1997, 48, 281–288.

121 Z. J. Song, H. Wu, O. Ciofu, K. F. Kong, N. Hoiby, J. Rygaard, A. Kharazmi and K. Mathee, *J. Med. Microbiol.*, 2003, 52, 731–740.

122 J. W. Costerton, *Trends Microbiol.*, 2001, 9, 50–52.

123 A. Bragonzi, D. Worlitzsch, G. B. Pier, P. Timpert, M. Ulrich, M. Hentzer, J. B. Andersen, M. Givskov, M. Conese and G. Doring, *J. Infect. Dis.*, 2005, 192, 410–419.

124 I. D. Hay, K. Gatland, A. Campisano, J. Z. Jordens and B. H. A. Rehm, *Appl. Environ. Microbiol.*, 2009, 75, 6022–6025.

125 Z. U. Rehman, Y. Wang, M. F. Moradali, I. D. Hay and B. H. A. Rehm, *Appl. Environ. Microbiol.*, 2013, 79, 3264–3272.

126 L. Oglesby, S. Jain and D. E. Ohman, *Microbiology*, 2008, 154, 1605–1615.

127 Z. U. Rehman and B. H. Rehm, *Appl. Environ. Microbiol.*, 2013, 18, 18.

128 J. C. Whitney, I. D. Hay, C. Li, P. D. W. Eckford, H. Robinson, M. F. Amaya, L. F. Wood, D. E. Ohman, C. E. Bear, B. H. Rehm and P. Lynne Howell, *Proc. Natl. Acad. Sci. U. S. A.*, 2011, 108, 13083–13088.

129 I. D. Hay, U. Remminghorst and B. H. A. Rehm, *Appl. Environ. Microbiol.*, 2009, 75, 1110–1120.

130 Y. Li, S. Heine, M. Entian, K. Sauer and N. Frankenberg-Dinkel, *J. Bacteriol.*, 2013, 195, 3531–3542.

131 A. Kropec, T. Maira-Litran, K. K. Jefferson, M. Grout, S. E. Cramton, F. Gotz, D. A. Goldmann and G. B. Pier, *Infect. Immun.*, 2005, 73, 6868–6876.

132 D. Pérez-Mendoza, S. J. Coulthurst, J. Sanjuán and G. P. C. Salmond, *Microbiology*, 2011, 157, 3340–3348.

133 L. Ye, X. Zheng and H. Zheng, *Glycobiology*, 2014, 24, 351–358.

134 M. E. Rupp, P. D. Fey, C. Heilmann and F. Gotz, *J. Infect. Dis.*, 2001, 183, 1038–1042.

135 X. Wang, J. F. Preston, 3rd and T. Romeo, *J. Bacteriol.*, 2004, 186, 2724–2734.

136 Y. Itoh, J. D. Rice, C. Goller, A. Pannuri, J. Taylor, J. Meisner, T. J. Beveridge, J. F. Preston and T. Romeo, *J. Bacteriol.*, 2008, 190, 3670–3680.

137 A. H. K. Choi, L. Slamti, F. Y. Avci, G. B. Pier and T. Maira-Litrán, *J. Bacteriol.*, 2009, 191, 5953–5963.

138 S. Steiner, C. Lori, A. Boehm and U. Jenal, *EMBO J.*, 2012, 30, 315.

139 L. Tagliabue, D. Antoniani, A. Maciag, P. Bocci, N. Raffaelli and P. Landini, *Microbiology*, 2010, 156, 2901–2911.

140 J. R. Tuckerman, G. Gonzalez, E. H. S. Sousa, X. Wan, J. A. Saito, M. Alam and M. A. Gilles-Gonzalez, *Biochemistry*, 2009, 48, 9764–9774.

141 A. Boehm, S. Steiner, F. Zaehringer, A. Casanova, F. Hamburger, D. Ritz, W. Keck, M. Ackermann, T. Schirmer and U. Jenal, *Mol. Microbiol.*, 2009, 72, 1500–1516.

142 A. G. Bobrov, V. Kirillina and R. D. Perry, *Adv. Exp. Med. Biol.*, 2007, 603, 201–210.

143 C. Darby, *Trends Microbiol.*, 2008, 16, 158–164.

144 R. D. Perry, M. L. Pendrak and P. Schuetze, *J. Bacteriol.*, 1990, 172, 5929–5937.

145 H. Li, L. Xu, J. Wang, Y. Wen, C. Vuong, M. Otto and Q. Gao, *Infect. Immun.*, 2005, 73, 3188–3191.

146 P. Yoong, C. Cywes-Bentley and G. B. Pier, *mBio*, 2012, 3, e00217-12.

147 O. Kirillina, J. D. Fetherstone, A. G. Bobrov, J. Abney and R. D. Perry, *Mol. Microbiol.*, 2004, 54, 75–88.

148 A. G. Bobrov, O. BKirillina and R. D. Perry, *FEMS Microbiol. Lett.*, 2005, 247, 123–130.

149 R. Simm, J. D. Fetherston, A. Kader, U. Romling and R. D. Perry, *J. Bacteriol.*, 2005, 187, 6816–6823.

150 Y. C. Sun, A. Koumoutsis, C. Jarrett, K. Lawrence, F. C. Gherardini, C. Darby and B. J. Hinnebusch, *PLoS One*, 2011, 6, 0019267.

151 A. G. Bobrov, O. Kirillina, S. Forman, D. Mack and R. D. Perry, *Environ. Microbiol.*, 2008, 10, 1419–1432.

152 L. E. Bellows, B. J. Koestler, S. M. Karaba, C. M. Waters and W. W. Lathem, *Mol. Microbiol.*, 2012, 86, 661–674.

153 F. Sun, H. Gao, Y. Zhang, L. Wang, N. Fang, Y. Tan, Z. Guo, P. Xia, D. Zhou and R. Yang, *PLoS One*, 2012, 7, 21.

154 Y.-C. Sun, X.-P. Guo, B. J. Hinnebusch and C. Darby, *J. Bacteriol.*, 2012, 194, 2020–2026.

155 H. Tan, J. A. West, J. P. Ramsay, R. E. Monson, J. L. Griffin, I. K. Toth and G. P. C. Salmond, *Microbiology*, 2014, 160, 1427–1439.

156 Y. Zhang, C. Wei, W. Jiang, L. Wang, C. Li, Y. Wang, J. M. Dow and W. Sun, *PLoS One*, 2013, 8, e59428.

157 L. Friedman and R. Kolter, *J. Bacteriol.*, 2004, 186, 4457–4465.

158 K. M. Colvin, V. D. Gordon, K. Murakami, B. R. Borlee, D. J. Wozniak, G. C. Wong and M. R. Parsek, *PLoS Pathog.*, 2011, 7, e1001264.

159 K. M. Colvin, Y. Irie, C. S. Tart, R. Urbano, J. C. Whitney, C. Ryder, P. L. Howell, D. J. Wozniak and M. R. Parsek, *Environ. Microbiol.*, 2012, 14, 1913–1928.

160 P. Vasseur, I. Vallet-Gely, C. Soscia, S. Genin and A. Filloux, Microbiology, 2005, 151, 985–997.
161 K. M. Colvin, N. Alnabelseya, P. Baker, J. C. Whitney, P. L. Howell and M. R. Parsek, *J. Bacteriol.*, 2013, 195, 2329–2339.
162 J. Overhage, M. Schemionek, J. S. Webb and B. H. A. Rehm, Appl. Environ. Microbiol., 2005, 71, 4407–4413.
163 Z. T. Guvener and C. S. Harwood, Mol. Microbiol., 2007, 66, 1459–1473.
164 J. W. Hickman, D. F. Tifrea and C. S. Harwood, Proc. Natl. Acad. Sci. U. S. A., 2005, 102, 14422–14427.
165 V. Huangyutitham, Z. T. Güvener and C. S. Harwood, *mBio*, 2013, 4, e00242-13.
166 J. H. Merritt, D. G. Ha, K. N. Cowles, W. Lu, D. K. Morales, J. Rabinowitz, Z. Gitai and G. A. O'Toole, *mBio*, 2010, 1, e00183-10.
167 J. G. Malone, T. Jaeger, P. Manfredi, A. Dotsch, A. Blanka, R. Bos, G. R. Cornelis, S. Haussler and U. Jenal, *PLoS Pathog.*, 2012, 8, 14.
168 J. G. Malone, T. Jaeger, C. Spangler, D. Ritz, A. Spang, C. Arrieumerlou, V. Kaever, R. Landmann and U. Jenal, *PLoS Pathog.*, 2010, 6, e1000804.
169 S. L. Kuchma, K. M. Brothers, J. H. Merritt, N. T. Liberati, F. M. Ausubel and G. A. O'Toole, *J. Bacteriol.*, 2007, 189, 8165–8178.
170 A. Jiménez-Fernández, A. López-Sánchez, P. Calero and F. Govantes, Environ. Microbiol. Rep., 2015, DOI: 10.111/1758-2229.12153.
171 L. Ma, H. Lu, A. Sprinkle, M. R. Parsek and D. J. Wozniak, *J. Bacteriol.*, 2007, 189, 8353–8356.
172 D. J. Wozniak, T. J. O. Wyckoff, M. Starkey, R. Keyser, P. Azadi, G. A. O'Toole and M. R. Parsek, Proc. Natl. Acad. Sci. U. S. A., 2003, 100, 7907–7912.
173 K. D. Jackson, M. Starkey, S. Kremer, M. R. Parsek and D. J. Wozniak, *J. Bacteriol.*, 2004, 186, 4466–4475.
174 L. Ma, K. D. Jackson, R. M. Landry, M. R. Parsek and D. J. Wozniak, *J. Bacteriol.*, 2006, 188, 8213–8221.
175 L. Ma, M. Conover, H. Lu, M. R. Parsek, K. Bayles and D. J. Wozniak, *PLoS Pathog.*, 2009, 5, e1000354.
176 Y. Irie, M. Starkey, A. N. Edwards, D. J. Wozniak, T. Romeo and M. R. Parsek, Mol. Microbiol., 2010, 78, 158–172.
177 J. A. Moscoso, H. Mikkelsen, S. Heeb, P. Williams and A. Filloux, Environ. Microbiol., 2011, 13, 3128–3138.
178 Y. Irie, B. R. Borlee, J. R. O'Connor, P. J. Hill, C. S. Harwood, D. J. Wozniak and M. R. Parsek, Proc. Natl. Acad. Sci. U. S. A., 2012, 109, 20632–20636.
179 X. B. Zhan, C. C. Lin and H. T. Zhang, Appl. Microbiol. Biotechnol., 2012, 93, 525–531.
180 R. Zhang and K. J. Edgar, Biomacromolecules, 2014, 15, 1079–1096.
181 A. M. Ruffing and R. R. Chen, Microb. Cell Fact., 2012, 11, 17.
182 P. V. Krasteva, J. C. N. Fong, N. J. Shikuma, S. Beyhan, M. V. A. S. Navarro, F. H. Yildiz and H. Sondermann, Science, 2010, 327, 866–868.
183 B. Lim, S. Beyhan, J. Meir and F. H. Yildiz, Mol. Microbiol., 2006, 60, 331–348.
184 D. Srivastava, R. C. Harris and C. M. Waters, *J. Bacteriol.*, 2011, 193, 6331–6341.
185 C. Casper-Lindley and F. H. Yildiz, *J. Bacteriol.*, 2004, 186, 1574–1578.
186 B. Lim, S. Beyhan and F. H. Yildiz, *J. Bacteriol.*, 2007, 189, 717–729.
187 N. J. Shikuma, J. C. Fong and F. H. Yildiz, *PLoS Pathog.*, 2012, 8, 24.
188 C. M. Waters, W. Lu, J. D. Rabinowitz and B. L. Bassler, *J. Bacteriol.*, 2008, 190, 2527–2536.
189 N. Bomchil, P. Watnick and R. Kolter, *J. Bacteriol.*, 2003, 185, 1384–1390.
190 E. Karatan, T. R. Duncan and P. I. Watnick, *J. Bacteriol.*, 2005, 187, 7434–7443.
191 D. A. Linkous and J. D. Oliver, *FEMS Microbiol. Lett.*, 1999, 174, 207–214.
192 A. Nakhamchik, C. Wilde and D. A. Rowe-Magnus, Appl. Environ. Microbiol., 2008, 74, 4199–4209.
193 Y. Guo and D. A. Rowe-Magnus, Infect. Immun., 2010, 78, 1390–1402.
194 K. L. Garrison-Schilling, Z. M. Kaluskar, B. Lambert and G. S. Pettis, *PLoS One*, 2014, 9, e100890.
195 N. Fraysse, F. Couderc and V. Poinsot, Eur. J. Biochem., 2003, 270, 1365–1380.
196 Y. Wang, J. Xu, A. Chen, Y. Wang, J. Zhu, G. Yu, L. Xu and L. Luo, Acta Biochim. Biophys. Sin., 2010, 42, 410–417.
197 C. Engl, C. J. Waite, J. F. McKenna, M. H. Bennett, T. Hamann and M. Buck, *mBio*, 2014, 5, e00168-14.
198 M. A. Matilla, M. L. Travieso, J. L. Ramos and M. I. Ramos-Gonzalez, Environ. Microbiol., 2011, 13, 1745–1766.
199 J. M. Koczan, M. J. McGrath, Y. Zhao and G. W. Sundin, Phytopathology, 2009, 99, 1237–1244.
200 M. Nimtz, A. Mort, T. Domke, V. Wray, Y. Zhang, F. Qiu, D. Coplin and K. Geider, Carbohydr. Res., 1996, 287, 59–76.
201 P. Bugert and K. Geider, Mol. Microbiol., 1995, 15, 917–933.
202 A. C. Edmunds, L. F. Castiblanco, G. W. Sundin and C. M. Waters, *J. Bacteriol.*, 2013, 195, 2155–2165.
203 L.-H. Chen, V. K. Köseoğlu, Z. T. Giuvener, T. Myers-Morales, J. M. Reed, S. E. F. D'Orazio, K. W. Miller and M. Gomelsky, *PLoS Pathog.*, 2014, 10, e1004301.
204 A. Manteca, R. Alvarez, N. Salazar, P. Yagüe and J. Sanchez, Appl. Environ. Microbiol., 2008, 74, 3877–3886.
205 N. T. Tran, C. D. Den Hengst, J. P. Gomez-Escribano and M. J. Buttner, *J. Bacteriol.*, 2011, 193, 3100–3108.
206 C. D. den Hengst, N. T. Tran, M. J. Bibb, G. Chandra, B. K. Leskiw and M. J. Buttner, Mol. Microbiol., 2010, 78, 361–379.
207 M. H. Hefti, K. J. Francoijs, S. C. de Vries, R. Dixon and J. Vervoort, Eur. J. Biochem., 2004, 271, 1198–1208.
208 B. L. Taylor and I. B. Zhulin, Microbiol. Mol. Biol. Rev., 1999, 63, 479.
209 N. Tschowri, M. A. Schumacher, S. Schlimpert, N. b. Chinnam, K. C. Findlay, R. G. Brennan and M. J. Buttner, *Cell*, 2014, 158, 1136–1147.
210 A. Meissner, V. Wild, R. Simm, M. Rohde, C. Erck, F. Bredenbruch, M. Morr, U. Romling and S. Haussler, Environ. Microbiol., 2007, 9, 2475–2485.

Review

211 M. Starkey, J. H. Hickman, L. Ma, N. Zhang, S. De Long, A. Hinz, S. Palacios, C. Manoil, M. J. Kirisits, T. D. Starner, D. J. Wozniak, C. S. Harwood and M. R. Parsek, *J. Bacteriol.*, 2009, 191, 3492–3503.

212 J. B. Bruhn, K. F. Nielsen, M. Hjelm, M. Hansen, J. Bresciani, S. Schulz and L. Gram, *Appl. Environ. Microbiol.*, 2005, 71, 7263–7270.

213 P. W. D'Alvise, O. Magdenoska, J. Melchiorsen, K. F. Nielsen and L. Gram, *Environ. Microbiol.*, 2014, 16, 1252–1266.

214 T. Petters, X. Zhang, J. Nesper, A. Treuner-Lange, N. Gomez-Santos, M. Hoppert, U. Jenal and L. Sogaard-Andersen, *Mol. Microbiol.*, 2012, 84, 147–165.

215 P. Meiser, H. B. Bode and R. Müller, *Proc. Natl. Acad. Sci. U. S. A.*, 2006, 103, 19128–19133.

216 P. Meiser, K. J. Weissman, H. B. Bode, D. Krug, J. S. Dickschat, A. Sandmann and R. Muller, *Chem. Biol.*, 2008, 15, 771–781.

217 M. Begley, P. D. Cotter, C. Hill and R. P. Ross, *Appl. Environ. Microbiol.*, 2009, 75, 5451–5460.

218 M. Singh and D. Sareen, *PLoS One*, 2014, 9, e91352.

219 T. Caetano, J. M. Krawczyk, E. Mösker, R. D. Süssmuth and S. Mendo, *Chem. Biol.*, 2011, 18, 90–100.

220 L. E. Ulrich, E. V. Koonin and I. B. Zhulin, *Trends Microbiol.*, 2009, 13, 52–56.

221 M. M. Mendez-Ortiz, M. Hyodo, Y. Hayakawa and J. Membrillo-Hernandez, *J. Biol. Chem.*, 2006, 281, 8090–8099.

222 K. Kitanishi, K. Kobayashi, Y. Kawamura, I. Ishigami, T. Ogura, K. Nakajima, J. Igarashi, A. Tanaka and T. Shimizu, *Biochemistry*, 2010, 49, 10381–10393.

223 V. Anantharaman and L. Aravind, *Trends Biochem. Sci.*, 2000, 25, 535–537.

224 S. Q. An, K. H. Chin, M. Febrer, Y. McCarthy, J. G. Yang, C. L. Liu, D. Swarbreck, J. Rogers, J. Maxwell Dow, S. H. Chou and R. P. Ryan, *EMBO J.*, 2013, 32, 2430–2438.

225 M. Tarutina, D. A. Ryjenkov and M. Gomelsky, *J. Biol. Chem.*, 2006, 281, 34751–34758.

226 Z. Cao, E. Livoti, A. Losi and W. Gärtner, *Photochem. Photobiol.*, 2010, 86, 606–611.

227 P. Savakis, S. De Causmaecker, V. Angerer, U. Ruppert, K. Anders, L. O. Essen and A. Wilde, *Mol. Microbiol.*, 2012, 85, 239–251.

228 M. Agostoni, B. J. Koestler, C. M. Waters, B. L. Williams and B. L. Montgomery, *mBio*, 2013, 4.

229 Y. Deng, N. Schmid, C. Wang, J. Wang, G. Pessi, D. Wu, J. Lee, C. Aguilar, C. H. Ahrens, C. Chang, H. Song,

L. Eberl and L.-H. Zhang, *Proc. Natl. Acad. Sci. U. S. A.*, 2012, 109, 15479–15484.

230 L. Plate and M. A. Marletta, *Mol. Cell*, 2012, 46, 449–460.

231 N. Liu, Y. Xu, S. Hossain, N. Huang, D. Coursolle, J. A. Gralnick and E. M. Boon, *Biochemistry*, 2012, 51, 2087–2099.

232 A. Rodrigue, Y. Quentin, A. Lazdunski, V. Mejean and M. Foglino, *Trends Microbiol.*, 2000, 8, 498–504.

233 A. H. West and A. M. Stock, *Trends Biochem. Sci.*, 2001, 26, 369–376.

234 M. Y. Galperin, *J. Bacteriol.*, 2006, 188, 4169–4182.

235 R. Paul, S. Weiser, N. C. Amiot, C. Chan, T. Schirmer, B. Giese and U. Jenal, *Genes Dev.*, 2004, 18, 715–727.

236 M. Levet-Paulo, J. C. Lazzaroni, C. Gilbert, D. Atlan, P. Doublet and A. Vianney, *J. Biol. Chem.*, 2011, 286, 31136–31144.

237 B. H. A. Rehm, *Nat. Rev. Microbiol.*, 2010, 8, 578–592.

238 C. Alvarez-Lorenzo, B. Blanco-Fernandez, A. M. Puga and A. Concheiro, *Adv. Drug Delivery Rev.*, 2013, 65, 1148–1171.

239 C. Leang, N. S. Malvankar, A. E. Franks, K. P. Nevin and D. R. Lovley, *Energy Environ. Sci.*, 2013, 6, 1901–1908.

240 K. Ochi and T. Hosaka, *Appl. Microbiol. Biotechnol.*, 2013, 97, 87–98.

241 L. Laureti, L. Song, S. Huang, C. Corre, P. Leblond, G. L. Challis and B. Aigle, *Proc. Natl. Acad. Sci. U. S. A.*, 2011, 108, 6258–6263.

242 C. Olano, I. García, A. González, M. Rodriguez, D. Rozas, J. Rubio, M. Sánchez-Hidalgo, A. F. Braña, C. Méndez and J. A. Salas, *Microb. Biotechnol.*, 2014, 7, 242–256.

243 Y. M. Chiang, S. L. Chang, B. R. Oakley and C. C. C. Wang, *Curr. Opin. Chem. Biol.*, 2011, 15, 137–143.

244 G. L. Challis, *Microbiology*, 2008, 154, 1555–1569.

245 E. Zazopoulos, K. X. Huang, A. Staffa, W. Liu, B. O. Bachmann, K. Nonaka, J. Ahlert, J. S. Thorson, B. Shen and C. M. Farnet, *Nat. Biotechnol.*, 2003, 21, 187–190.

246 S. Beyhan, A. D. Tischler, A. Camilli and F. H. Yildiz, *J. Bacteriol.*, 2006, 188, 3600–3613.

247 D. Perez-Mendoza, I. M. Aragon, H. A. Prada-Ramirez, L. Romero-Jimenez, C. Ramos, M. T. Gallegos and J. Sanjuan, *PLoS One*, 2014, 9, e91645.

248 E. B. Purcell, R. W. McKee, S. M. McBride, C. M. Waters and R. Tamayo, *J. Bacteriol.*, 2012, 194, 3307–3316.

249 M. He, Z. Ouyang, B. Troxell, H. Xu, A. Moh, J. Piesman, M. V. Norgard, M. Gomelsky and X. F. Yang, *PLoS Pathog.*, 2011, 7, 30.

250 C. L. Ho, K. S. Chong, J. A. Oppong, M. L. Chuah, S. M. Tan and Z. X. Liang, *J. Am. Chem. Soc.*, 2013, 135, 566–569.
