
Cell Reports

Synthetic Quantitative Array Technology Identifies the Ubp3-Bre5 Deubiquitinase Complex as a Negative Regulator of Mitophagy

### Graphical Abstract

![Graphical Abstract Image](image_url)

### Authors
Matthias Müller, Peter Kötter, ..., Karl-Dieter Entian, Andreas S. Reichert

### Correspondence
reichert@hhu.de

#### In Brief
Müller et al. developed a method, termed synthetic quantitative array technology, to screen for mitophagy modulators in yeast. Using this method, they identified the Ubp3-Bre5 deubiquitination complex, which inhibits mitophagy but promotes other types of autophagy. Deubiquitination thus appears to regulate a switch between distinct autophagy pathways.

### Highlights
- Development of an enzyme-based high-throughput screening technique in *S. cerevisiae*
- Genome-wide quantitative screen for modulators of rapamycin-induced mitophagy
- Identification of 86 positive and 10 negative regulators of mitophagy
- Reciprocal regulation of mitophagy and other autophagy pathways by deubiquitination

---

Müller et al., 2015, Cell Reports 10, 1215–1225  
February 24, 2015 ©2015 The Authors  
http://dx.doi.org/10.1016/j.celrep.2015.01.044

Cell Reports
Resource

---

**Synthetic Quantitative Array Technology Identifies the Ubp3-Bre5 Deubiquitinase Complex as a Negative Regulator of Mitophagy**

Matthias Müller,$^{1}$ Peter Kötter,$^{2}$ Christina Behrendt,$^{1,3}$ Elena Walter,$^{1,4}$ Christian Q. Scheckhuber,$^{1,4}$ Karl-Dieter Entian,$^{2}$ and Andreas S. Reichert$^{1,3,*}$

$^{1}$Mitochondrial Biology, Buchmann Institute for Molecular Life Sciences and Zentrum für Molekulare Medizin, Goethe University, Max-von-Laue-Straße 15, 60438 Frankfurt am Main, Germany  
$^{2}$Institut für Molekulare Biowissenschaften, Goethe University, Max-von-Laue-Straße 9, 60438 Frankfurt am Main, Germany  
$^{3}$Institute of Biochemistry and Molecular Biology I, Medical Faculty, Heinrich Heine University, Universitätsstraße 1, 40225 Düsseldorf, Germany  
$^{4}$Present address: Integrative Fungal Research Cluster, Georg-Voigt-Straße 14-16, 60325 Frankfurt am Main, Germany  

*Correspondence:* reichert@hhu.de  
http://dx.doi.org/10.1016/j.celrep.2015.01.044  

This is an open access article under the CC BY-NC-ND license (http://creativecommons.org/licenses/by-nc-nd/3.0/).

---

### SUMMARY

Mitophagy is crucial to ensuring mitochondrial quality control. However, the molecular mechanism and regulation of mitophagy are still not fully understood. Here, we developed a quantitative methodology termed synthetic quantitative array (SQA) technology, which allowed us to perform a genome-wide screen for modulators of rapamycin-induced mitophagy in *S. cerevisiae*. SQA technology can be easily employed for other enzyme-based reporter systems and widely applied in yeast research. We identified 86 positive and 10 negative regulators of mitophagy. Moreover, SQA-based analysis of non-selective autophagy revealed that 63 of these regulators are specific for mitophagy and 33 regulate autophagy in general. The Ubp3-Bre5 deubiquitination complex was found to inhibit mitophagy but, conversely, to promote other types of autophagy, including ribophagy. This complex translocates dynamically to mitochondria upon induction of mitophagy. These findings point to a role of ubiquitination in mitophagy in yeast and suggest a reciprocal regulation of distinct autophagy pathways.

---

### INTRODUCTION

Mitochondria are double-membrane enclosed organelles that serve central cellular functions such as production of ATP via oxidative phosphorylation, iron-sulfur cluster biogenesis, and regulation of apoptosis. They are the main source of reactive oxygen species (ROS), which potentially can cause severe oxidative damage to proteins, lipids, and DNA. Mitochondrial dysfunction has been associated with aging and an increasing number of diseases in humans, such as neurodegeneration, diabetes, and cancer ([Wallace, 2005](https://doi.org/10.1016/j.cell.2005.02.029)). One important pathway for mitochondrial

quality control is the selective removal of mitochondria by an autophagy-related process termed mitophagy ([Lemasters, 2005](https://doi.org/10.1016/j.jmb.2005.07.001)). However, the detailed molecular mechanism and in particular the regulation of this process are still not well understood. Autophagy is a ubiquitous catabolic process that is highly conserved among eukaryotes ([Klionsky, 2007](https://doi.org/10.1016/j.cell.2007.09.019); [Todde et al., 2009](https://doi.org/10.1016/j.cell.2009.01.038)). This “self-eating” process enables cells to adapt to changes in nutrient availability and a variety of stress conditions. Autophagy has long been assumed to be a non-selective process that randomly degrades bulk portions of the cytoplasm, but considerable progress has been made in identifying a variety of selective types of autophagy ([Suzuki, 2013](https://doi.org/10.1016/j.cell.2013.01.028)). One well-known example of this is the cytoplasm-to-vacuole targeting (Cvt) pathway in yeast, which allows the selective delivery of at least two resident hydrolases, aminopeptidase 1 (Ape1) and α-mannosidase 1 (Ams1), into the vacuole ([Lynch-Day and Klionsky, 2010](https://doi.org/10.1016/j.cell.2010.02.027)). Furthermore, the selective degradation of whole organelles (e.g., mitochondria [mitophagy], peroxisomes [pexophagy], ribosomes [ribophagy], and ER [ER-phagy]) and the removal of aggregated proteins (aggrephagy) and invading bacteria (xenophagy) have been investigated ([Kraft et al., 2008](https://doi.org/10.1016/j.cell.2008.02.037); [Lemasters, 2005](https://doi.org/10.1016/j.jmb.2005.07.001); [Suzuki, 2013](https://doi.org/10.1016/j.cell.2013.01.028); [Youle and Narendra, 2011](https://doi.org/10.1016/j.cell.2011.02.019)). Studies in yeast and other fungi have revealed the identity of more than 30 autophagy-related (ATG) genes so far ([Klionsky, 2007](https://doi.org/10.1016/j.cell.2007.09.019)). Many of these genes are required for both non-selective and selective autophagy, and thus form the core machinery for autophagy induction and autophagosome biogenesis. However, some ATG factors are only needed for distinct, selective types of autophagy. In particular, the identification of Atg32 as a specific receptor protein led to detailed information about the selective nature of mitophagy ([Kanki et al., 2009b](https://doi.org/10.1016/j.cell.2009.05.027); [Okamoto et al., 2009](https://doi.org/10.1016/j.cell.2009.05.028)). Atg32 is localized to the mitochondrial outer membrane and interacts with Atg11, a common adaptor protein for several selective types of autophagy, upon induction of mitophagy, leading to recruitment of mitochondria to the autophagy machinery. Atg32 was reported to interact with Atg8, a ubiquitin (Ub)-like component of the autophagosome, through the conserved consensus sequence W/YXXI/L/V ([Kondo-Okamoto et al., 2012](https://doi.org/10.1016/j.cell.2012.02.037)). This sequence is called the

---

Cell Reports 10, 1215–1225, February 24, 2015 ©2015 The Authors 1215

Mitophagy has been analyzed in yeast cells after application of the target of rapamycin (TOR) kinase inhibitor rapamycin (Kissová et al., 2004; Mendl et al., 2011), upon nitrogen starvation (Kanki and Klionsky, 2008), via post-log phase growth (Kanki et al., 2009a; Okamoto et al., 2009), and upon induction of mitochondrial dysfunction (Nowikovsky et al., 2007; Priault et al., 2005). Studies of mitophagy using yeast as a model organism have strongly promoted our understanding of this complex process in mammalian cells and its link to diverse pathologies. To date, two well-established mechanisms have been described for mitophagy in mammals. The first mechanism depends on NIX, a selective autophagy receptor that is located at the outer membrane of mitochondria and is required for mitophagy during erythrocyte differentiation (Novak et al., 2010; Sandoval et al., 2008). The second one plays a role in mitochondrial quality control by eliminating dysfunctional mitochondria and depends on the mitochondrial kinase PINK1 and the E3 Ub ligase PARKIN, two factors that are causally linked to Parkinson’s disease (Matsuda et al., 2010; Narendra et al., 2008; Vives-Bauza et al., 2010). Although the role of ubiquitination in mitophagy in mammals is well supported, up to now, there has been no evidence that ubiquitination plays a role in regulating mitophagy in yeast cells.

In two previous studies, investigators performed large-scale genomic screens for yeast mutants defective in mitophagy induced by stationary-phase growth (Kanki et al., 2009a; Okamoto et al., 2009). To monitor mitophagy, they analyzed delivery of a mitochondria-targeted fluorescent protein into the vacuole. In each screen, 23 mutants that exhibited a specific mitophagy defect without affecting other autophagy-related pathways were identified. However, only a small overlap between the two studies was found, which may be due to the different experimental conditions employed. Results obtained from image-based assays are commonly limited by the lack of quantitative data, and consequently it has not been possible to reliably determine increased or partially inhibited mitophagy by such methods.

Here, we report on the development of a powerful quantitative genome-wide screening methodology to overcome these limitations. Mitophagy was analyzed upon induction with rapamycin, a condition that has never been investigated on a genome-wide scale in yeast before. In principle, this method, which we term synthetic quantitative array (SQA) technology, combines the synthetic genetic array (SGA)-based generation of an enzymatic tester mutant library with a subsequently performed quantitative high-throughput assay. Using this technology, we discovered numerous mitophagy- and autophagy-modulating factors, which included both positive and negative regulators of these pathways. Detailed analysis of candidates revealed an unexpected role of the Ubp3-Bre5 deubiquitination complex in mitophagy regulation.

## RESULTS

### Strategy for the Genome-wide Identification of Mitophagy Modulators by SQA Technology

To obtain genome-wide knowledge about the molecular factors that modulate mitophagy in yeast cells, we decided to combine the strengths of the SGA methodology with a biochemical high-throughput assay that would allow enzymatic quantification of mitophagy (Figures 1A–1C). The quantitative

Figure 1. **SQA Technology**

(A) Flow chart of the SQA technology applied to a genome-wide screen for modulators of rapamycin-induced mitophagy.

(B) Outline of the modified SGA procedure for generating a yeast mutant mitophagy tester library.

(C) Schematic overview of the high-throughput screening procedure for analysis of rapamycin-induced mitophagy conducted in 96-well plates.

Atg8 family interacting motif (AIM) in yeast and the LC3 interacting region (LIR) in mammalian cells, and is found ubiquitously in various autophagy-receptor proteins (Noda et al., 2008).
aspect of this SGA-based procedure led us to coin the term “synthetic quantitative array” (SQA) technology. In order to analyze mitophagy in a precise and quantitative manner, we used the previously described alkaline phosphatase (ALP)-based assay for mitophagy (Mendl et al., 2011) with some modifications. This assay relies on expression of an inactive proenzyme of ALP targeted to the mitochondrial matrix (mtALP). Upon mitophagy induction, a part of the mtALP is delivered into the vacuole and activated by the vacuolar proteinase A. For a genome-wide application of this assay, we used the SGA methodology to cross a yeast knockout collection of non-essential genes with ~5,000 mutant strains with a mitophagy tester strain lacking endogenous PHO8 and harboring an mtALP expression vector (Figure 1A). The resulting SQA mutant strains served as a mitophagy tester library, which was subjected to a quantitative, high-throughput screening procedure for rapamycin-induced mitophagy. We applied a three-step ALP-based selection procedure (Figure 1A): (1) we miniaturized the ALP-based assay technique and performed the first round of screening entirely in 96-well plates in a high-throughput manner (genome-wide screen), (2) we selected the top candidates and then re-evaluated them in a second round of screening in 96-well plates (confirmation screen), and (3) we validated the remaining candidates by employing the well-established ALP-based assay procedure, using flasks for cultivation (final validation).

### Generation of a Mitophagy Tester Mutant Library by a Modified SGA Approach

To obtain the mitophagy tester mutant library, we used an adapted SGA procedure (Figure 1B). The SGA technique was originally developed for high-throughput genetic interaction studies in *S. cerevisiae* and can be used for the systematic generation of haploid double-mutant arrays (Tong et al., 2001). Here, we mated the *MATα* yeast knockout collection of non-essential genes with a *MATa* mitophagy tester strain that contained all of the genetic characteristics required for the ALP-based assay procedure (Figure 1B). After selection for heterozygous diploid cells, the strains were sporulated, leading to meiotic formation of haploid tetrads. The mitophagy tester strain was additionally deleted for *CAN1*, which encodes an arginine permease, conferring sensitivity to the toxin canavanine. This locus was replaced by *STE2pr-his5* (*S. pombe*), a *MATa*-specific reporter. This double selection strategy ensured efficient selection for *MATa* haploid cells upon sporulation. Finally, we were able to obtain an arranged array of haploid double mutants deleted for *PHO8* and a gene X of interest that contained the mtALP plasmid. Among the original 4,922 deletion strains, 196 mutants failed to grow on YPD medium from the beginning, which is not uncommon for large yeast libraries that have been stored for long periods of time. Only 57 strains did not survive the SGA procedure, possibly due to technical limitations or synthetic lethal interactions. In summary, this procedure allowed us to generate a library of 4,669 SQA mutant strains that could be quantitatively tested for mitophagy.

### High-Throughput Quantification of the SQA Mutant Library Revealed 324 Candidate Genes that Modulate Mitophagy

To analyze mitophagy of all SQA mutants in a quantitative, high-throughput manner, we had to miniaturize the standard ALP-based assay protocol. We optimized the procedure so that cultivation of SQA strains, induction of mitophagy, preparation of cell extracts, and determination of ALP activity and protein concentration could all be performed in 96-well plates (Figure 1C). Strains were grown on selective medium containing glycerol as a non-fermentable carbon source. Mitophagy was induced by addition of rapamycin and cells were harvested after 24 hr of incubation. We established a mechanical cell lysis method using glass beads that allowed us to prepare total protein extracts from low amounts of cells. We then used these extracts to efficiently determine the specific mtALP activity of each mutant. We analyzed the complete SQA library distributed on 53 plates by this screening procedure. The resulting specific mtALP activity of each strain was normalized to the median of all strains measured on the same plate to allow a better plate-to-plate comparison. Arrangement of all SQA mutant strains according to the obtained level of mitophagy (Figure 2A) shows that, depending on the set thresholds, we were able to identify a number of mitophagy-defective and -enhanced mutants. Using thresholds of <50.0% and >200.0% of normalized mitophagy, we identified 295 deletion strains that exhibited a defect in mitophagy (6.3% of all analyzed strains) and 29 deletion strains that showed an increased mitophagy (0.6% of all analyzed strains). Thus, we were able to perform a quantitative, genome-wide analysis of mitophagy in yeast (Table S1), allowing the reliable identification of both positive and negative regulators of this process.

### A Two-Step Validation Procedure Identified 86 Genes that Promote and 10 Genes that Inhibit Mitophagy

Next, we spotted the 324 identified potential mitophagy modulators on new plates and subjected them to a second round of the 96-well-based screening procedure. The specific mtALP activity of each strain was normalized to the averaged specific activity of a positive control measured on the same plate. In order to minimize falsely excluded mitophagy modulators, we set thresholds (less stringently) to <60.0% and >150.0% of normalized mitophagy. We further excluded ten mutants that have been annotated by the manufacturer of the yeast deletion library (Open Biosystems) as being incorrect. We could confirm 230 mitophagy-defective and 19 mitophagy-enhanced SQA mutant strains (Table S2). We subsequently excluded one mutant deleted for *PEP4*, encoding vacuolar proteinase A, or 17 already known ATG genes. To finally validate our remaining candidates, we quantified the rapamycin-induced mitophagy level of each SQA mutant strain by the standard ALP-based assay procedure. Using flasks for cultivation and a cell homogenizer for generation of cell extracts, we were able to obtain quantitative data for each strain in a robust and well-established manner. Furthermore, this procedure allowed us to follow the growth behavior and to check for mtALP expression and processing by western blot analysis. We identified 197 strains with reduced rapamycin-induced mitophagy levels and 12 strains with increased levels (Table S2).
Figure 2. Identification of 86 Mitophagy-Defective and 10 Mitophagy-Enhanced Mutant Strains

(A) SQA strains were analyzed for rapamycin-induced mitophagy in 96-well plates. The specific mtALP activity of each strain was measured once and normalized to the median specific activity of all strains on the same plate. Strains were ranked according to the extent of mitophagy induction. Thresholds were set to <50.0% and >200.0% to define mitophagy-defective and -enhanced mutants, respectively. Normalized mitophagy levels are depicted on a logarithmic scale. See also Table S1.

(B) Schematic flow chart summarizing the results of the SQA-based identification of modulators of rapamycin-induced mitophagy. See also Figures S2 and S3 and Tables S2 and S4.

(C) GO analysis of identified mitophagy modulators compared with the entire yeast genome. See also Table S5.

After performing three steps of screening and validation, and applying various conservative selection criteria, we identified a total of 96 mitophagy modulators, 86 of which promote mitophagy and 10 of which inhibit mitophagy (Figure 2B; Tables S1 and S2). During the final validation, we also compared some of the identified candidate mutants with well-established positive and negative controls, including strains lacking atg32, atg11, and atg1, which were excluded from our list of final candidates (Figure S3). Notably, a total of 100 additional deletion mutant strains, which exhibited strongly reduced cell growth under the applied respiratory conditions, were found to be impaired in mitophagy (Tables S2 and S4).

Next, in order to classify the 96 mitophagy modulators, we performed a Gene Ontology (GO) analysis by using annotation data for specific GO terms provided by the Gene Ontology Consortium (Ashburner et al., 2000) (Figure 2C; Table S5). Most of the gene products (70%) were reported to show a cytoplasmic localization, which is statistically expected considering the distribution of annotation data for the entire yeast genome (63%). Yet, we noted a high proportion of mitophagy modulators being localized to mitochondria (29%). This seems to be clearly overrepresented compared with the yeast genome (18%), indicating that certain steps of mitophagy are spatially controlled at the organelle (Figure 2C). Also, vacuolar proteins appear to be relatively enriched in our data set. The fact that the absolute number of such genes is rather low can be attributed to our exclusion of ATG genes (Table S3, bottom part), some of which represent essential vacuole-associated proteins. Next, we analyzed the biological processes in which the potential mitophagy modulators are

We used thresholds of <75.0% and >150.0%, respectively, compared with a positive control. Nine strains that did not show mtALP expression were removed (Table S2). We also excluded 93 strains that exhibited a growth defect under the applied respiratory conditions, to minimize the probability that our final list would contain mutants that are only indirectly affected in mitophagy (Table S2). In 15 cases, the SQA strains still expressed endogenous PHO8 (data not shown), presumably due to a diploid contamination during mutant generation. To analyze these genes, we regenerated the strains manually by tetrad dissection and determined the levels of rapamycin-induced mitophagy. Subsequently, 11 of these mutants (four based on the set thresholds and seven based on their growth behavior) could be excluded from the final list of mitophagy modulators (Table S2).

- 96 Mitophagy Modulators -
| LEU3 | PMR1 | VPS1 |
| --- | --- | --- |
| NSR1 | RMR1 |  |
| ATP15 | LAS21 | RRM3 | UBC8 |
| CCZ1 | MRF1 | RSM19 | VAM3 |
| COG5 | MRPL37 | SGF73 | VPS41 |
| EAF1 | MRPL38 | SPT5 | YLL007C |
| FPR1 | MST1 | SSZ1 | YNL184C |
| HMI1 | NAM2 | TPS1 | YPR071W |
| IRC19 | RRG9 | TRS85 | ZUO1 |

| AGE2 | LAP2 | PRY2 | YBR141C |
| --- | --- | --- | --- |
| ALO1 | LAP3 | RAM1 | YDR336W |
| ARL1 | MDM30 | SAL1 | YIL141W |
| ARO7 | MDM31 | SPR6 | YIL168W |
| BUB3 | MGR2 | SQS1 | YJR084W |
| BXI1 | MOT2 | STB2 | YMR135W-A |
| CNN1 | MPA43 | SUR7 | YNL013C |
| CTS1 | MPD2 | TOF1 | YNR029C |
| EGD1 | MRPL31 | TRF5 | YOL013W-A |
| EXO1 | NOT5 | UGA2 | YOL019W |
| FKH1 | NTH2 | VTC4 |  |
| FMP41 | OPY1 | WHI2 |  |
| GPA2 | PDR12 | WSC2 |  |
| HRB1 | PFA3 | YBR053C |  |
| IST1 | POG1 | YBR063C |  |

| BUD27 | YGR160W |  |
| --- | --- | --- |
| HAP3 | SSN3 | YNL170W |
| BRE5 | DSS1 | UBP3 |

Figure 3. Classification of Modulators of Rapamycin-Induced Mitophagy

Levels of rapamycin-induced mitophagy and non-selective autophagy of 96 mitophagy-affected mutants were used to classify them into six categories. Arrows indicate a decrease (red) or an increase (blue) of the respective process in the corresponding deletion strains. See also Table S3.

Identification of 63 Specific Mitophagy Modulators by SQA-Based Autophagy Analysis

In order to determine which of the identified mutants are specifically affected in mitophagy, we in addition analyzed the level of non-selective autophagy induced by rapamycin in these strains. For this purpose, we employed the ALP-based assay for general autophagy using a cytosol-targeted proenzyme of ALP (cytALP) (Mendl et al., 2011). To determine autophagy within the 96 mitophagy-modulating mutants, we applied the SQA technology once more by crossing the respective single gene deletion strains with an autophagy tester strain that met the genetic requirements for the cytALP-based assay. We quantified the level of rapamycin-induced non-selective autophagy for each strain using the standard ALP-based assay procedure (Table S3). By comparing the levels of rapamycin-induced mitophagy and autophagy for all of the analyzed mutant strains, we were able to classify the strains into six categories (Figure 3; Table S3). Most of the strains (55) were specifically defective in mitophagy, as general autophagy was largely unaffected. Two strains (∆bud27 and ∆ygr160w) were specifically enhanced in mitophagy without showing an effect on autophagy induction. Importantly, we also identified six strains in which both processes were inversely regulated: three mutants (∆hap3, ∆ssn3, and ∆ynl170w) were specifically defective for mitophagy and enhanced in autophagy, whereas the other three mutants (∆bre5, ∆ubp3, and ∆dss1) were affected in the opposite manner. This strongly argues for the existence of regulatory pathways that can modulate selective mitophagy and non-selective autophagy in a reciprocal manner. Twenty-eight deletion mutants exhibited defects in the induction of rapamycin-induced mitophagy as well as autophagy, whereas five deletion mutants showed a strong increase in both levels. Thus, we conclude that the SQA technology as applied here also identified a total of 33 genes that modulate autophagy in a general manner. As those genes do not affect mitophagy specifically, we excluded them from our final candidate list. In summary, we identified 63 specific modulators for rapamycin-induced mitophagy in yeast (Figure 3; Table S3).

Ubp3 and Bre5 Regulate Mitophagy and Autophagy in a Reciprocal Manner

Next, we focused on two of the three mutant strains that on the one hand showed enhanced mitophagy but on the other hand reduced non-selective autophagy upon rapamycin treatment, namely, ∆ubp3 and ∆bre5. We did this for three reasons: First, the factors Ubp3 and Bre5 were previously reported to be part of the same protein complex (Li et al., 2005). Second, both factors are required for the deubiquitinating activity of this complex (Cohen et al., 2003), indicating that ubiquitination might play a role during mitophagy in *S. cerevisiae*. Third, such a reciprocal behavior has not been observed before, pointing to an independent means of regulating distinct autophagic processes. To corroborate the results from the SQA-based experiments, we manually crossed the corresponding single gene deletion strains, ∆ubp3 and ∆bre5, with a ∆pho8 mutant by performing tetrad dissection. After transformation with the respective ALP expression plasmids, we quantified the rapamycin-induced levels for mitophagy and autophagy using the standard ALP-based assay procedure. Indeed, the mitophagy levels were elevated in both mutants to ~160%, whereas autophagy levels were decreased to ~40% compared with the wild-type level, confirming our SQA-based results (Figures 4A and 4B, top). The extent of ALP processing as revealed by western blot analysis was consistent with these results (Figures 4A and 4B, bottom). Expression of Ubp3 and Bre5 under control of the endogenous promoter efficiently reversed the effects observed for the corresponding deletion strains (Figures 4C–4F), revealing the specificity of the mitophagy enhancement and autophagy reduction in both mutants.

We next tested the effect of the Ubp3-Bre5 deubiquitination complex on mitophagy and autophagy under different induction conditions. After shifting the cells from lactate-containing medium to glucose-containing medium lacking nitrogen (nitrogen

Figure 4. UBP3 and BRE5 Affect Mitophagy and Non-selective Autophagy in a Reciprocal Manner

(A and B) Strains deleted for the indicated genes and for endogenous PHO8 expressing mtALP (A) or cytALP (B) were analyzed by the standard ALP-based assay (top). Cells were grown in SG medium and treated with rapamycin for 24 hr or left untreated. Specific ALP activities are given as means ± SD (n ≥ 3) normalized to the rapamycin-treated wild-type control. Proteolytic processing of mtALP (A) or cytALP (B) was determined by western blot analysis of total cell extracts (bottom). Bmh2 served as the loading control.

(C–F) Strains deleted for the indicated genes and for endogenous PHO8 expressing mtALP (C and E) or cytALP (D and F) and endogenously expressing Ubp3 (C and D), Bre5 (E and F), or empty vector were analyzed as described above. Mean ± SD of at least three independent experiments normalized to control is shown. Significance levels p < 0.05 (*) or p < 0.01 (**) using a Student’s t test are indicated. See also Figure S1.

and Δbre5 strains in a medium containing a non-fermentable carbon source (glycerol or lactate) to the stationary phase resulted in a significantly higher induction of mitophagy and lower induction of autophagy when compared with the wild-type control after 24 hr of growth (Figure S1B). This is fully in line with the data we obtained using rapamycin for mitophagy induction. In summary, we conclude that Ubp3/Bre5-dependent deubiquitination inhibits mitophagy and at the same time promotes non-selective autophagy. Thus, both processes appear to be regulated in an opposing manner.

Ubp3 and Bre5 Translocate Dynamically from the Cytosol to Mitochondria when Mitophagy Is Induced by Rapamycin

In mammals, it is well established that PARKIN, an E3 Ub ligase, translocates to mitochondria upon induction of mitophagy (Narendra et al., 2008; Vives-Bauza et al., 2010). Here, we sought to determine where Ubp3 and Bre5 are located in the cell, and whether their subcellular localization is altered when mitophagy is induced. We used strains expressing C-terminal GFP fusion proteins of Ubp3 and Bre5: Ubp3-GFP and Bre5-GFP. These strains were treated with rapamycin for 1, 5, or 24 hr, or remained untreated as a control. Under normal growth conditions, both fusion proteins showed a dual localization, namely, cytosolic and mitochondrial (Figures 5A and 5B, control images). After induction of mitophagy with starvation), we found that autophagy induction was extensively impaired in both the Δubp3 and Δbre5 strains, whereas mitophagy was not significantly altered even after 24 hr of starvation (Figure S1A). Although mitophagy was not increased in both mutants, it was still clearly regulated differently compared with non-selective autophagy under these conditions. Cultivating Δubp3

1220 Cell Reports 10, 1215–1225, February 24, 2015 ©2015 The Authors

Figure 5. Ubp3 and Bre5 Translocate Dynamically from the Cytosol to Mitochondria upon Induction of Mitophagy

(A–D) Strains expressing mitoDsRED and either C-terminal GFP fusion proteins of Ubp3 (A and C) or Bre5 (B and D) were analyzed by fluorescence microscopy (A and B) or western blot (C and D).

(A and B) Strains were treated with rapamycin for 1, 5, or 24 hr, or were left untreated as a control. More than 50 cells were counted in three independent experiments and the percentage of cells with GFP signal in the mitochondria (Mito) only or mitochondria and cytosol (Mito + Cytos.) was determined (right in A and B). Representative images are shown on the left.

(C and D) Mitochondria were isolated after 0 or 24 hr treatment with rapamycin and the subcellular localization of Ubp3-GFP (C) and Bre5-GFP (D) was determined by western blot analysis using an anti-GFP antibody. Tim44 served as the loading control.

Mitophagy Is Regulated Differently from Other Selective Types of Autophagy by the Ubp3-Bre5 Deubiquitination Complex

To address how other types of selective autophagy are affected in Δubp3 and Δbre5 mutants, we first analyzed the selective degradation of ribosomes, termed ribophagy, which was previously reported to be reduced in these strains during starvation (Kraft et al., 2008). For this purpose, we applied the previously described Rpl25-GFP-processing assay (Kraft et al., 2008) in wild-type, Δubp3, Δbre5, and Δatg1 cells that were cultivated under respiratory conditions and treated or not treated with rapamycin (Figure 6A). Free GFP clearly accumulated in wild-type cells compared with the untreated control. Rpl25-GFP degradation was completely blocked in the Δatg1 mutant, which is defective in all autophagy-related pathways and served as a negative control. In Δubp3 and Δbre5 cells, processing of Rpl25-GFP was moderately reduced upon rapamycin treatment when compared with the wild-type, consistent with the results of Kraft et al. (2008).

Next, we analyzed whether the Cvt pathway is affected in Δubp3 and Δbre5 strains. We examined Ape1 processing in wild-type, Δubp3, Δbre5, and Δatg1 cells under the same conditions as described above (Figure 6B). The precursor form pr-Ape1 was clearly processed, generating the mature form m-Ape1 in untreated wild-type cells. Processing was drastically increased after treatment with rapamycin. In contrast, it was completely blocked in Δatg1 cells under both conditions, indicating a complete block of the Cvt pathway. Both mutant strains, Δubp3 and Δbre5, exhibited a moderate reduction in Ape1

Cell Reports 10, 1215–1225, February 24, 2015 ©2015 The Authors 1221

duction. In this study, we were able to demonstrate the suitability of this technique for the quantitative analysis of mitophagy as well as non-selective autophagy in yeast.

One general advantage of the SQA technology compared with currently used cell- or colony-based reporter assays using fluorescent proteins or colorimetric staining techniques is that even weak modulators can be identified and ranked based on detailed quantitative data. Moreover, even moderate increases in enzyme activity can be monitored, which is difficult to do with image-based approaches and allows for identification of negative regulators of mitophagy.

Highly reproducible data were obtained, as >75% of all candidates identified in the initial genome-wide screen were corroborated in the second confirmation round. More than 90% of the remaining candidates were verified using the standard ALP-based assay, demonstrating the robustness of the method even when performed on a 96-well scale (Figure 2B). Deletions of 27 ATG genes have been associated with defects in mitophagy (Kanki et al., 2011). Notably, 25 of the respective deletion mutants are included in our generated SQA library (Table S1). Using thresholds of <50.0% and <75.0% in the first genome-wide screening round led to the identification of 16 (64%) and 23 (92%) of these mutants, respectively. When ATG mutants reported to be only partially defective in mitophagy (ATG21, ATG23, ATG27, ATG29, ATG31, and ATG33) are excluded from this reference set, 19 severely affected mutant strains remain. Of these strains 15 (79%) or 17 (89%) could be confirmed using thresholds of <50.0% or <75.0%, respectively. We therefore conclude that our genome-wide screen has a high sensitivity of ~80%–90%.

Our general goal in developing the SQA-based screen was to obtain a detailed mechanistic insight into the regulation of mitophagy in baker’s yeast. This is highly relevant because impaired removal of dysfunctional mitochondria by mitophagy has been implicated in the pathogenesis of numerous diseases in humans, such as Parkinson’s disease (Narendra et al., 2008; Youle and Narendra, 2011). Although some genomic screens for mitophagy have been performed in higher and lower eukaryotes (Kanki et al., 2009a; Lefebvre et al., 2013; Okamoto et al., 2009; Orvedahl et al., 2011), our approach is unique because it is quantitative and uses a specific TOR kinase inhibitor, rapamycin, as the inducer of mitophagy. In summary, using stringent criteria, we identified 86 mitophagy-defective and 10 mitophagy-enhanced mutant strains, which therefore represent positive and negative modulators of mitophagy, respectively (Figure 2A).

We also identified 100 other deletion mutants that were strongly affected in mitophagy and in addition showed a severe respiratory growth defect (Table S4). We excluded these mutants from our final list of candidates, as during ALP-based analysis, strains that are found to be impaired in respiratory growth often tend to exhibit a reduction in both mitophagy and non-selective autophagy, which is likely to have resulted from secondary effects. However, we should emphasize that it is possible that specific regulators of mitophagy are represented within this group of candidates. This is actually supported by the fact that not all strains that were previously reported to be respiratory deficient exhibited a reduced mitophagy response. Furthermore, a recent screen of respiratory-deficient mutant strains identified several candidate genes for mitophagy modulators (Böckler

Figure 6. Other Types of Selective Autophagy Are Impaired in Δubp3 and Δbre5 Cells

(A) Analysis of rapamycin-induced ribophagy. The indicated strains expressing Rpl25-GFP were grown in SG medium and treated with rapamycin for 24 hr or left untreated. Vacuolar cleavage of Rpl25-GFP and generation of free GFP was determined by western blot analysis of total cell extracts.

(B) Analysis of the Cvt pathway. Strains deleted for the indicated genes were analyzed as in (A). Proteolytic processing of Ape1 was determined by western blot analysis of total cell extracts. Bmh2 served as the loading control.

processing in untreated cells as well as after rapamycin treatment compared with the wild-type.

In summary, mitophagy is apparently the only autophagy-related pathway that is upregulated under the applied conditions when components of the Ubp3-Bre5 complex are absent. Non-selective autophagy as well as two selective types of autophagy, ribophagy and the Cvt pathway, were impaired. These findings strongly point to a unique and specific regulation of mitophagy by the Ubp3-Bre5 deubiquitination complex.

DISCUSSION

Obtaining quantitative biochemical data from gene deletion libraries on a genome-wide scale is highly valuable for identifying molecular factors involved in complex biological processes. Moreover, such data form the basis for generating theoretical models in systems biology. Here, we developed a technique, termed SQA technology, that allowed us to identify a number of previously unknown modulators of mitophagy in yeast and can be easily applied to other enzyme-based reporter assays (Figure 1A). The first step exploits the power of the well-established SGA technique and is required to generate a mutant tester library that harbors the gene deletions of interest and meets the genetic requirements for the respective enzyme-based reporter system. The next step is to obtain quantitative data from this tester library in a high-throughput manner. For this purpose, we had to develop a large-scale protocol that would allow us to collect total protein extracts from several thousands of yeast strains after applying a standardized protocol for mitophagy in-

(and Westermann, 2014). That study also provided evidence that respiratory deficiency does not impair mitophagy per se.

By applying the SQA technology to determine rapamycin-induced, non-selective autophagy, we were able to define six categories of mutant strains giving rise to 33 general autophagy modulators and 63 specific mitophagy modulators (Figure 3). Although many of these strains have not been linked to these processes before, we could confirm ATG32, EGD1, WHI2, MDM31, and HAP3 as mitophagy modulators, consistent with earlier reports (Böckler and Westermann, 2014; Kanki et al., 2009b; Mendl et al., 2011; Okamoto et al., 2009). It is remarkable that about two-thirds of all identified modulators seem to be specifically required for mitophagy. In our view, this further demonstrates how little we know about the regulation of mitophagy, in particular when induced by rapamycin.

Regarding the final list of 63 specific mitophagy modulators, we noted that only two genes (ATG32 and EGD1) were commonly identified in the three genome-wide mitophagy screens that have been performed in *S. cerevisiae* to date (Figure S2). This is most likely due to the different experimental conditions applied for induction of mitophagy. Apparently, selective degradation of mitochondria is a complex process that is differentially regulated depending on specific mitophagy-inducing signals. Furthermore, all of these studies differed in the method used to monitor mitophagy, emphasizing the necessity to retrieve detailed quantitative data from large-scale experiments. However, at least ATG32 and EGD1 seem to regulate mitophagy independently of the induction mode, pointing to a universal core machinery that apparently is modulated by various upstream factors depending on the environmental conditions applied.

Obtaining distinct quantitative data for mitophagy and autophagy by using SQA technology allowed us to get a handle on regulatory mechanisms that affect both processes differentially. Notably, this depends on reliable detection of increased levels of mitophagy and autophagy, which has not been systematically addressed before. Earlier fluorescence-based screens were clearly designed to identify mutant strains with defective mitophagy (Kanki et al., 2009a; Okamoto et al., 2009). In our study, a small subset of ten mutant strains showed an increase in rapamycin-induced mitophagy (Figure 3). Five of them (∆leu3, ∆pmr1, ∆vps1, ∆nsr1, and ∆rmr1) affected non-selective autophagy in a similar manner and thus appear to be general inhibitors of rapamycin-induced autophagy in yeast. The five remaining mitophagy-enhanced deletion strains exhibited no significant effect on autophagy (∆bud27 and ∆ygr160w) or were inversely affected (∆bre5, ∆ubp3, and ∆dss1). These genes encode negative regulators of mitophagy, representing a class of mitophagy modulators that have not been characterized to date.

We focused on two factors, Ubp3 and Bre5, because they both showed the same reciprocal response regarding mitophagy and autophagy (Figure 4) and were reported to exist in a single protein complex (Li et al., 2005). Ubp3 is one of 16 known Ub-specific proteases in yeast that cleave Ub from Ub-conjugated proteins. Ubp3 activity requires the positive regulator Bre5 as a cofactor (Cohen et al., 2003). This complex has been reported to play a regulatory role in diverse cellular pathways, including anterograde and retrograde transport, the Ras/PKA signaling pathway, transcriptional activation, and protein quality control

(Brew and Huffaker, 2002; Cohen et al., 2003; Li and Wang, 2013; Solé et al., 2011). The Ubp3-Bre5 complex is also required for starvation-induced ribophagy via deubiquitination of ribosomal proteins in the cytoplasm (Kraft et al., 2008), which is corroborated by our study (Figure 6A). Furthermore, this complex was associated with regulation of the Cvt pathway by modulating the ubiquitination status of Atg19 (Baxter et al., 2005). Our finding that both Ubp3 and Bre5 inhibit mitophagy and at the same time promote bulk autophagy, ribophagy, and the Cvt pathway (Figures 4, 5, and 6) has several important implications. First, regulation of rapamycin-induced mitophagy in yeast depends on the ubiquitination/deubiquitination machinery. This mode of regulating mitophagy is already well established in mammals, but to date has not been described in baker’s yeast. It indicates the existence of conserved Ub-dependent regulatory mechanisms of mitophagy in eukaryotes, supporting the view that baker’s yeast is a useful model organism for mitophagy research. Ubiquitination of mitochondria could be a signal for their degradation, and therefore deleting components of the Ubp3-Bre5 deubiquitination machinery would result in increased mitophagy rates. It further suggests that rapamycin-induced mitophagy in yeast and mitophagy of dysfunctional mitochondria in mammals at least share ubiquitination as a common regulatory mechanism. Second, it strongly displays a reciprocal and interdependent regulation of different autophagy pathways. In contrast to its effect on mitophagy, deubiquitination seems to be essential for the progression of non-selective autophagy (Figure 4B). Interestingly, a potential role of Ub in non-selective autophagy pathways has not yet been described in yeast and mammals. Even different types of selective autophagy seem to be differentially regulated by the Ubp3-Bre5 complex (Figure 6). This implies that mitophagy and other types of autophagy, including ribophagy, are mechanistically linked but appear to represent competing processes. What could be the physiological advantage of such a reciprocal regulation? In the case of degradation of cytoplasmic ribosomes, one could imagine that under conditions of increased stress during translation, e.g., by stalled ribosomes or protein aggregation of nascent polypeptides, it is a high priority to restore functional protein synthesis, which can be promoted by ribophagy of non-functional ribosomes. Simultaneous induction of mitophagy could limit components of the autophagy machinery required for ribophagy. Moreover, when cytoplasmic protein translation is inhibited, mitochondrial biogenesis (which largely relies on cytosolic protein synthesis and import into mitochondria) becomes impaired. This may result in the accumulation of temporarily dysfunctional mitochondria due to a lack of import of newly synthesized components. In order to prevent unwanted and excessive degradation of those temporarily impaired organelles, it is well conceivable that limiting mitophagy at the same time would be beneficial. Conversely, under conditions in which primarily mitochondria are dysfunctional, it would be beneficial to promote mitophagy and repress other types of autophagy. This would ensure that components of the general autophagy machinery would not be limiting. Based on our results, we propose that such a reciprocal regulatory switch could ensure that, depending on the kind of stress inflicted, the type of autophagy pathway that copes best with this stress is specifically enhanced, albeit at the

expense of other types of autophagy. For mitophagy, we suggest that Ubp3-Bre5 is a critical part of this regulatory switch. Moreover, the observed dynamic relocation of Ubp3-Bre5 from the cytosol to mitochondria (Figure 5) could mechanistically explain this reciprocal behavior. In this scenario, for example, Ubp3-Bre5 would be less available for ribophagy in the cytosol and at the same time would inhibit mitophagy by deubiquitinating mitochondrial proteins at the outer membrane.

In summary, we have developed a quantitative technology that can be applied to a wide range of biological problems. Using this technology, we were able to identify a large collection of factors that modulate mitophagy in both positive and negative ways. Finally, we discovered that the Ubp3-Bre5 deubiquitination complex is involved in reciprocal regulation of mitophagy in relation to other types of autophagy. An important implication of this is that our concept of how autophagic processes affect each other has to be revised. The molecular details of how these regulatory switches work and how mitochondrial quality control is linked to other cellular stress response pathways are still unclear. Still, we believe that large-scale genetic studies can form the basis to address these challenging issues.

### EXPERIMENTAL PROCEDURES

#### Yeast Strains, Plasmids, and Growth Conditions
Yeast strains, plasmids, oligonucleotides, experimental details for plasmid construction, and growth conditions are provided in the Supplemental Experimental Procedures.

#### Generation of a Mitophagy Tester Mutant Library
We adapted the SGA procedure described previously ([Tong et al., 2001](https://doi.org/10.1038/ng701)) as follows: Tester strain design included deletion of *PHO8* by a *natMX* cassette and deletion of *CAN1* replaced by the *STE2pr-his5* (*S. pombe*) reporter. The mtALP and cytALP plasmids were introduced for the mitophagy- and autophagy-based SQA approach, respectively. Tester strains are listed in Table S4. Single gene deletion strains from the *MATα* (BY4742; *his3Δ1*; *leu2Δ0*; *lys2Δ0*; *ura3Δ0*) yeast knockout collection obtained from Open Biosystems (GE Healthcare) were crossed with the respective tester strain (*MATa*), and several pinning steps on different selective media gave rise to an arranged array of haploid double mutants that could be analyzed with the ALP-based assay (Figure 1B). It should be noted that a few of the strains in the original deletion library are known to be incorrect and that we did not confirm the deletion for all candidate genes. However, we confirmed proper gene deletion for at least some mutant strains, including *Δubp3* and *Δbre5*. The media used for the SGA procedure are listed in the Supplemental Experimental Procedures. After cells were mated on YPD medium at 30°C for 1 day, strains were selected for heterozygous diploids on SED-Leu + G418/clonNat medium at 30°C for at least 2 days. Diploid cells were then subjected to sporulation on KOAc medium and incubated at 22°C for 7 days. Next the spore-containing cell material was selected for *MATa* haploid cells on SCD-Leu/His/Arg + canavanine/3-amino-triazole medium at 30°C for 4 days. The latter procedure was repeated once. To finally isolate specific haploid double-deletion mutants, cells were first subjected to SED-Leu/His/Arg + canavanine/G418/3-amino-triazole medium at 30°C for at least 3 days, and then for a final selection were spotted on SED-Leu/His/Arg + canavanine/G418/clonNat/3-amino-triazole medium at 30°C for at least 3 days. Strains were analyzed for growth defects and stored in 20% glycerol-containing medium in 96-well plates at –80°C.

#### High-Throughput Screening Method
SQA strains were grown in sterile 96-well plates under respiratory conditions. Plates were cultivated at 30°C with shaking at 200 rpm. Humidity was conferred by wet tissues. The strains were inoculated in 200 μl SG-Leu medium and subsequently diluted several times with fresh medium to maintain exponential growth (Figure 1C). Before each dilution step, mutant strains were resuspended and optical density (OD~600nm~) was measured. Mitophagy was induced by addition of 1 μM rapamycin and cells were harvested after 24 hr of incubation. Before harvesting, OD~600nm~ was measured and plates were centrifuged at 4,000 × g for 15 min. Pellets were washed with 100 μl ice-cold 2 mM phenylmethanesulfonyl fluoride (PMSF), frozen in liquid nitrogen, and stored at –80°C. For preparation of total cell extracts, pellets were resuspended in 100 μl ice-cold lysis buffer (10 mM MgCl₂, 0.1 M KOAc, 50 mM KCl, 10 μM ZnSO₄, 20 mM PIPES [pH 7.0], 0.5% Triton X-100, 1 mM PMSF). After addition of ~150 mg glass beads (Ø 0.5 mm; Carl Roth), the plates were sealed with a plastic film and shaken using a vortex mixer (VXR basic Vibrax; IKA) at 2,000 rpm for 4 min. After addition of 50 μl lysis buffer, the cell lysates were centrifuged at 4,000 × g for 10 min at 4°C. The supernatant was transferred to a new plate and stored at –20°C. ALP activity was determined by incubating 20 μl of the cell lysate with 105 μl of reaction buffer (40 mM MgCl₂, 0.5% Triton X-100, 0.25 M Tris-HCl [pH 8.5], 4 mM p-nitrophenylphosphate) at 37°C for 15 min. The reaction was stopped by adding 125 μl 2 M glycine-NaOH (pH 11.0), and absorption was determined at 405 nm. Protein concentration was determined by using the BCA Protein Assay (Thermo Fisher) according to the manufacturer’s instructions and used to calculate the specific ALP activity for each cell lysate. The specific ALP activities were then normalized to the median of all strains on the same plate (genome-wide screen) or to the averaged specific activity of four positive control strains (confirmation screen) measured on the same plate.

### ALP Assay for Mitophagy and Autophagy
For quantification of mitophagy and autophagy levels, individual yeast strains deleted for *PHO8* and containing mtALP- or cytALP-expressing plasmids, respectively, were analyzed as described previously ([Mendl et al., 2011](https://doi.org/10.1016/j.celrep.2011.01.001)). Specific activities were normalized to the specific activity of a control strain. Cell lysates were subjected to SDS-PAGE and western blotting.

### SUPPLEMENTAL INFORMATION
Supplemental Information includes Supplemental Experimental Procedures, three figures, and five tables and can be found with this article online at http://dx.doi.org/10.1016/j.celrep.2015.01.044.

### ACKNOWLEDGMENTS
We are grateful to Christian Bach, Jakob Maciejko, and Stefanie Lamberth for excellent experimental assistance; Nadine Mendl and Sebastian Koob for inspiring scientific discussions; and Drs. Matthias Peter and Claudine Kraft for kindly providing plasmids. This work was supported by Deutsche Forschungsgemeinschaft project RE-1575/1-2 (M.M., C.B., E.W., and A.S.R.) and the Cluster of Excellence “Macromolecular Complexes” at the Goethe University Frankfurt, DFG project EXC 115 (A.S.R.).

Received: May 28, 2014  
Revised: December 16, 2014  
Accepted: January 16, 2015  
Published: February 19, 2015  

### REFERENCES
Ashburner, M., Ball, C.A., Blake, J.A., Botstein, D., Butler, H., Cherry, J.M., Davis, A.P., Dolinski, K., Dwight, S.S., Eppig, J.T., et al.; The Gene Ontology Consortium (2000). Gene ontology: tool for the unification of biology. Nat. Genet. 25, 25–29.  
Baxter, B.K., Abeliiovich, H., Zhang, X., Stirling, A.G., Burlingame, A.L., and Goldfarb, D.S. (2005). Atg19p ubiquitination and the cytoplasm to vacuole trafficking pathway in yeast. J. Biol. Chem. 280, 39067–39076.  
Böckler, S., and Westermann, B. (2014). Mitochondrial ER contacts are crucial for mitophagy in yeast. Dev. Cell 28, 450–458.  
Brew, C.T., and Huffaker, T.C. (2002). The yeast ubiquitin protease, Ubp3p, promotes protein stability. Genetics 162, 1079–1089.

Cohen, M., Stutz, F., Belgareh, N., Haguenauer-Tsapis, R., and Dargemont, C. (2003). Ubp3 requires a cofactor, Bre5, to specifically de-ubiquitinate the COPII protein, Sec23. Nat. Cell Biol. 5, 661–667.

Kanki, T., and Klionsky, D.J. (2008). Mitophagy in yeast occurs through a selective mechanism. J. Biol. Chem. 283, 32386–32393.

Kanki, T., Wang, K., Baba, M., Bartholomew, C.R., Lynch-Day, M.A., Du, Z., Geng, J., Mao, K., Yang, Z., Yen, W.L., and Klionsky, D.J. (2009a). A genomic screen for yeast mutants defective in selective mitochondria autophagy. Mol. Biol. Cell 20, 4730–4738.

Kanki, T., Wang, K., Cao, Y., Baba, M., and Klionsky, D.J. (2009b). Atg32 is a mitochondrial protein that confers selectivity during mitophagy. Dev. Cell 17, 98–109.

Kanki, T., Klionsky, D.J., and Okamoto, K. (2011). Mitochondria autophagy in yeast. Antioxid. Redox Signal. 14, 1989–2001.

Kissová, I., Deffieu, M., Manon, S., and Camougrand, N. (2004). Uth1p is involved in the autophagic degradation of mitochondria. J. Biol. Chem. 279, 39068–39074.

Klionsky, D.J. (2007). Autophagy: from phenomenology to molecular understanding in less than a decade. Nat. Rev. Mol. Cell Biol. 8, 931–937.

Kondo-Okamoto, N., Noda, N.N., Suzuki, S.W., Nakatogawa, H., Takahashi, I., Matsunami, M., Hashimoto, A., Inagaki, F., Ohsumi, Y., and Okamoto, K. (2012). Autophagy-related protein 32 acts as autophagic degron and directly initiates mitophagy. J. Biol. Chem. 287, 10631–10638.

Kraft, C., Deplazes, A., Sohrmann, M., and Peter, M. (2008). Mature ribosomes are selectively degraded upon starvation by an autophagy pathway requiring the Ubp3p/Bre5p ubiquitin protease. Nat. Cell Biol. 10, 602–610.

Lefebvre, V., Du, Q., Baird, S., Ng, A.C., Nascimento, M., Campanella, M., McBride, H.M., and Screaton, R.A. (2013). Genome-wide RNAi screen identifies ATPase inhibitory factor 1 (ATPIF1) as essential for PARK2 recruitment and mitophagy. Autophagy 9, 1770–1779.

Lemasters, J.J. (2005). Selective mitochondrial autophagy, or mitophagy, as a targeted defense against oxidative stress, mitochondrial dysfunction, and aging. Rejuvenation Res. 8, 3–5.

Li, Y., and Wang, Y. (2013). Ras protein/cAMP-dependent protein kinase signaling is negatively regulated by a deubiquitinating enzyme, Ubp3, in yeast. J. Biol. Chem. 288, 11358–11365.

Li, K., Zhao, K., Ossareh-Nazari, B., Da, G., Dargemont, C., and Marmorstein, R. (2005). Structural basis for interaction between the Ubp3 deubiquitinating enzyme and its Bre5 cofactor. J. Biol. Chem. 280, 29176–29185.

Lynch-Day, M.A., and Klionsky, D.J. (2010). The Cvt pathway as a model for selective autophagy. FEBS Lett. 584, 1359–1366.

Matsuda, N., Sato, S., Shiba, K., Okatsu, K., Saisho, K., Gautier, C.A., Sou, Y.S., Saiki, S., Kawajiri, S., Sato, F., et al. (2010). PINK1 stabilized by mitochondrial depolarization recruits Parkin to damaged mitochondria and activates latent Parkin for mitophagy. J. Cell Biol. 189, 211–221.

Mendl, N., Occhipinti, A., Müller, M., Wild, P., Dikic, I., and Reichert, A.S. (2011). Mitophagy in yeast is independent of mitochondrial fission and requires the stress response gene WHI2. J. Cell Sci. 124, 1339–1350.

Narendra, D., Tanaka, A., Suen, D.F., and Youle, R.J. (2008). Parkin is recruited selectively to impaired mitochondria and promotes their autophagy. J. Cell Biol. 183, 795–803.

Noda, N.N., Kumeta, H., Nakatogawa, H., Satoo, K., Adachi, W., Ishii, J., Fujioka, Y., Ohsumi, Y., and Inagaki, F. (2008). Structural basis of target recognition by Atg8/LC3 during selective autophagy. Genes Cells 13, 1211–1218.

Novak, I., Kirkin, V., McEwan, D.G., Zhang, J., Wild, P., Rozenknop, A., Rogov, V., Lühr, F., Popovic, D., Occhipinti, A., et al. (2010). Nix is a selective autophagy receptor for mitochondrial clearance. EMBO Rep. 11, 45–51.

Nowikovsky, K., Reipert, S., Devenish, R.J., and Schuwenen, R.J. (2007). Mdm38 protein depletion causes loss of mitochondrial K+/H+ exchange activity, osmotic swelling and mitophagy. Cell Death Differ. 14, 1647–1656.

Okamoto, K., Kondo-Okamoto, N., and Ohsumi, Y. (2009). Mitochondria-anchored receptor Atg32 mediates degradation of mitochondria via selective autophagy. Dev. Cell 17, 87–97.

Orvedahl, A., Sumpter, R., Jr., Xiao, G., Ng, A., Zou, Z., Tang, Y., Narimatsu, M., Gilpin, C., Sun, Q., Roth, M., et al. (2011). Image-based genome-wide siRNA screen identifies selective autophagy factors. Nature 480, 113–117.

Priault, M., Salin, B., Schaeffer, J., Vallette, F.M., di Rago, J.P., and Martinou, J.C. (2005). Impairing the bioenergetic status and the biogenesis of mitochondria triggers mitophagy in yeast. Cell Death Differ. 12, 1613–1621.

Sandoval, H., Thiagarajan, P., Dasgupta, S.K., Schumacher, A., Prchal, J.T., Chen, M., and Wang, J. (2008). Essential role for Nix in autophagic maturation of erythroid cells. Nature 454, 232–235.

Solé, C., Nadal-Ribelles, M., Kraft, C., Peter, M., Posas, F., and de Nadal, E. (2011). Control of Ubp3 ubiquitin protease activity by the Hog1 SAPK modulates transcription upon osmostress. EMBO J. 30, 3274–3284.

Suzuki, K. (2013). Selective autophagy in budding yeast. Cell Death Differ. 20, 43–48.

Todde, V., Veenhuis, M., and van der Klei, I.J. (2009). Autophagy: principles and significance in health and disease. Biochim. Biophys. Acta 1792, 3–13.

Tong, A.H., Evangelista, M., Parsons, A.B., Xu, H., Bader, G.D., Pagé, N., Robinson, M., Raghibizadeh, S., Hogue, C.W., Bussey, H., et al. (2001). Systematic genetic analysis with ordered arrays of yeast deletion mutants. Science 294, 2364–2368.

Vives-Bauza, C., Zhou, C., Huang, Y., Cui, M., de Vries, R.L., Kim, J., May, J., Tocilescu, M.A., Liu, W., Ko, H.S., et al. (2010). PINK1-dependent recruitment of Parkin to mitochondria in mitophagy. Proc. Natl. Acad. Sci. USA 107, 378–383.

Wallace, D.C. (2005). A mitochondrial paradigm of metabolic and degenerative diseases, aging, and cancer: a dawn for evolutionary medicine. Annu. Rev. Genet. 39, 359–407.

Youle, R.J., and Narendra, D.P. (2011). Mechanisms of mitophagy. Nat. Rev. Mol. Cell Biol. 12, 9–14.
