
RESEARCH ARTICLE

Toxin Synthesis by *Clostridium difficile* Is Regulated through Quorum Signaling

**Charles Darkoh**,<sup>a</sup> **Herbert L. DuPont**,<sup>a,d,e</sup> **Steven J. Norris**,<sup>b,c</sup> **Heidi B. Kaplan**<sup>b</sup>

The University of Texas Health Science Center, School of Public Health, Division of Epidemiology, Human Genetics, and Environmental Sciences, Center For Infectious Diseases, Houston, Texas, USA<sup>a</sup>; The University of Texas Medical School, Department of Microbiology and Molecular Genetics, Houston, Texas, USA<sup>b</sup>; The University of Texas Medical School, Department of Pathology And Laboratory Medicine, Houston, Texas, USA<sup>c</sup>; St. Luke’s Medical Center, Internal Medicine, Houston, Texas, USA<sup>d</sup>; Baylor College of Medicine, Department of Medicine, Houston, Texas, USA<sup>e</sup>

**ABSTRACT** *Clostridium difficile* infection (CDI) is dramatically increasing as a cause of antibiotic- and hospital-associated diarrhea worldwide. *C. difficile*, a multidrug-resistant pathogen, flourishes in the colon after the gut microbiota has been altered by antibiotic therapy. Consequently, it produces toxins A and B that directly cause disease. Despite the enormous public health problem posed by this pathogen, the molecular mechanisms that regulate production of the toxins, which are directly responsible for disease, remained largely unknown until now. Here, we show that *C. difficile* toxin synthesis is regulated by an accessory gene regulator quorum-signaling system, which is mediated through a small (<1,000-Da) thiolactone that can be detected directly in stools of CDI patients. These findings provide direct evidence of the mechanism of regulation of *C. difficile* toxin synthesis and offer exciting new avenues both for rapid detection of *C. difficile* infection and development of quorum-signaling-based non-antibiotic therapies to combat this life-threatening emerging pathogen.

**IMPORTANCE** *Clostridium difficile* infection (CDI) is the most common definable cause of hospital-acquired and antibiotic-associated diarrhea in the United States, with the total cost of treatment estimated between 1 and 4.8 billion U.S. dollars annually. *C. difficile*, a Gram-positive, spore-forming anaerobe, flourishes in the colon after the gut microbiota has been altered by antibiotic therapy. As a result, there is an urgent need for non-antibiotic CDI treatments that preserve the colonic microbiota. *C. difficile* produces toxins A and B, which are directly responsible for disease. Here, we report that *C. difficile* regulates its toxin synthesis by quorum signaling, in which a novel signaling peptide activates transcription of the disease-causing toxin genes. This finding provides new therapeutic targets to be harnessed for novel nonantibiotic therapy for *C. difficile* infections.

---

Received 25 December 2014 Accepted 12 January 2015 Published 24 February 2015

Citation Darkoh C, DuPont HL, Norris SJ, Kaplan HB. 2015. Toxin synthesis by *Clostridium difficile* is regulated through quorum signaling. mBio 6(2):e02569-14. doi:10.1128/mBio.02569-14

Editor Eric J. Rubin, Harvard School of Public Health

Copyright © 2015 Darkoh et al. This is an open-access article distributed under the terms of the [Creative Commons Attribution-Noncommercial-ShareAlike 3.0 Unported license](https://creativecommons.org/licenses/by-nc-sa/3.0/), which permits unrestricted noncommercial use, distribution, and reproduction in any medium, provided the original author and source are credited.

Address correspondence to Charles Darkoh, Charles.darkoh@uth.tmc.edu.

This article is a direct contribution from a Fellow of the American Academy of Microbiology.

---

Treatment of *Clostridium difficile* infections (CDI), with an estimated annual cost of 1 to 4.8 billion U.S. dollars (1–5), has been hampered by resistance to multiple antibiotics, increased virulence of the infecting strains, recurrence of the infection, and lack of drugs that preserve the colonic microbiota. Morbidity and mortality resulting from CDI-associated diseases have also increased significantly over the past 10 years, making *C. difficile* the number one emerging pathogen in the United States (4, 6–8). The highest risk factor for CDI is previous antibiotic therapy (9). Furthermore, treatment with antimicrobials is, in as many as 25% of cases, ineffective, resulting in recurrence of CDI (10, 11). As a result, there is an urgent need for non-antibiotic CDI treatments that preserve the colonic microbiota, either as stand-alone therapies or as adjunctive therapies designed to augment the efficacy of the current treatment options.

Pathogenic strains of *C. difficile* possess a 19.6-kb pathogenicity locus, which is composed of *tcdR*, *tcdB*, *tcdE*, *tcdA*, and *tcdC* (see Fig. S1 in the supplemental material). This locus is responsible for the production of toxins A and B, encoded by *tcdA* and *tcdB*, respectively (12–14). The toxins are essential in *C. difficile* pathogenesis, because strains that do not produce either of these toxins are not associated with disease (12–15). Two regulators of transcription encoded by the pathogenicity locus, TcdR and TcdC, have been proposed to control the expression at the *tcdA* and *tcdB* promoters. The *tcdR* gene encodes a positively acting sigma factor that controls transcription of the toxin promoters and its own promoter (16, 17). In contrast, *tcdC* is proposed to encode a negative regulator of toxin synthesis (18–20). Evidence to support the suggested TcdC activity includes the emergence of epidemic strains (NAP1/027 strains) with deletions or frameshift mutations in *tcdC* that produce high toxin levels (7, 18–23). However, neither the deletion of the entire *tcdC* gene nor restoration of the *tcdC* frameshift mutations alters toxin synthesis (24, 25), suggesting that the mechanism of *C. difficile* toxin regulation may involve other key regulatory elements yet to be discovered.

Many virulence factors produced by bacterial pathogens are more effective at high cell density and as a result are controlled by quorum signaling (26–30). This is a mechanism of cell-cell com-
communication whereby bacteria coordinate information about their cell density and consequently regulate gene expression through the production, detection, and response to extracellular signaling molecules. The quorum-signaling mechanism enables bacteria to harness energetically expensive processes at the population level to potentiate the impact of those processes on their environment or host. Quorum signaling molecules accumulate in the environment as the bacterial population density increases. Bacteria that quorum sense monitor the concentration of the signaling molecule as a measure of their cell density and coordinate the expression of genes that modulate activities that are advantageous when performed by groups of bacteria acting in synchrony (31, 32).

We demonstrate here that *C. difficile* toxin production is predominantly controlled by a novel thiolactone quorum-signaling peptide, which is independent of *tcdC*-mediated regulation. This finding has profound implications for clinical treatment, because synthesis of the toxins, which are directly responsible for disease and colonic injury, can be targeted by blocking this newly identified quorum-signaling regulatory pathway as a novel non-antibiotic treatment for CDI.

### RESULTS

#### The *C. difficile* stationary-phase culture supernatant induces early toxin production.
*C. difficile* cells synthesize toxins A and B during stationary growth phase (Fig. 1A). To examine the regulation of toxin production, we first developed a rapid and sensitive assay (the CdiffTox activity assay) to detect toxin A and B activity (33). This activity assay enabled the detection of toxin synthesis by *C. difficile* strain 630 at 16 h of growth in culture, compared to >32 h for analysis by enzyme-linked immunosorbent assay (ELISA) (Fig. 1A) or Western blotting. Because toxin production occurred only at high cell density, we questioned if quorum-signaling regulation could be involved. We hypothesized that a small signaling molecule accumulates extracellularly during growth and induces toxin production once a threshold concentration is reached.

To investigate this hypothesis, we developed a *C. difficile* quorum-signaling bioassay. Low-density, logarithmic-phase *C. difficile* strain 630 cells normally produce toxin after 16 h of growth. In the bioassay, strain 630 “tester” cells were exposed to samples potentially containing the *C. difficile* toxin-inducing (TI) signaling activity and were assayed for premature toxin synthesis. We first tested whether boiled, filtered, cell-free stationary-phase culture supernatants collected at various times would stimulate the tester cells to prematurely produce toxin. TI activity was detected in the culture supernatant fluids collected after 8 h of growth from both the non-hypervirulent strain 630 and hypervirulent strain R20291 (Fig. 1B). No TI activity was detected using fresh medium or 4-h culture supernatant from strain 630. Interestingly, the R20291 strain produced detectable TI activity after as little as 4 h of growth; in addition, the R20291 culture supernatants induced higher levels of toxin production in the tester cells than strain 630 supernatants (Fig. 1B). Overall, these results indicate that toxin-inducing activity accumulates externally during *C. difficile* cell growth and suggest that it functions as a cell density-dependent signal.

To establish the size range of the TI molecule(s), the culture supernatant fluid was dialyzed through membranes of different molecular mass cutoffs. This was performed by suspending the dialysis membranes containing the supernatant fluid in fresh brain heart infusion (BHI) medium overnight at 4°C and testing for either retention or loss of the toxin-inducing activity. Induction of toxin synthesis was lost when the culture supernatant fluid was dialyzed through dialysis membranes with a molecular mass cutoff of 1 kDa or greater (membranes with molecular mass cutoffs of 1, 3.5, and 12 kDa were tested). However, the TI activity was retained when the supernatant was dialyzed using a membrane with a molecular mass cutoff of 500 Da. These data suggest that a small molecule that is less than 1 kDa is responsible for the toxin-inducing activity in the supernatant (data not shown). The characteristics of the TI activity—small size (<1 kDa) and accumulation in high-cell-density culture medium—match the hallmarks of most quorum signals identified to date (34–36), further supporting the hypothesis that *C. difficile* toxin production is likely mediated by a quorum-signaling system.

The TI activity (hereafter referred to as the TI signal) was purified from stationary-phase culture supernatant fluid after enrichment of the activity. First, the culture supernatant fluid was boiled to inactivate the toxins and other high-molecular-mass proteins. This was followed by acetone precipitation, anion exchange chromatography, and then high-performance liquid chromatography (HPLC). The TI activity eluted from the C<sub>18</sub> column as a single sharp peak in an acetonitrile gradient (0 to 95% over 15 min at 0.5 ml/min) at 5.5 min and 20% solvent concentration (Fig. 2A and B). Interestingly, the growth kinetics of cells cultured in the presence and absence of the purified TI signal were not significantly different, indicating that the TI signal has no detectable effect on growth (data not shown).

#### The TI signal stimulates elevated toxin synthesis in hypervirulent clinical *C. difficile* strains.
To determine if the addition of exogenous TI signal affects toxin production by clinical isolates of hypervirulent (NAP1 strains) and non-hypervirulent *C. difficile* strains, various clinical strains were used as tester strains in the bioassay (Fig. 3). It is important to note that, in fresh medium without added TI signal, the hypervirulent strains expressed significantly greater (*P* = 0.004) toxin levels at 4 h of culture than did the non-hypervirulent strains. Many hypervirulent *C. difficile* strains, including strains R20291 and CD196, contain a mutated *tcdC* gene (21, 23), which has a frameshift mutation and thus would produce only a truncated product, whereas the non-hypervirulent strains encode a full-length, 232-amino-acid (aa) TcdC product. Previous studies indicate an association between a deletion or frameshift mutation in the *tcdC* gene and increased transcription of *tcdA* and *tcdB* (7, 18–23). Remarkably, when incubated with the TI signal from strain 630, all the strains produced elevated levels of toxin in the bioassay with no significant differences (*P* = 0.318). Hence, the TI signal activated toxin production at similar levels in both hypervirulent and non-hypervirulent strains. These results suggest the presence of a conserved regulatory pathway controlling *C. difficile* toxin synthesis in response to the production and extracellular accumulation of the TI signal. Furthermore, the purified TI signal from the hypervirulent strain R20291 induced elevated toxin production in strain 630 (see Fig. S2 in the supplemental material), implying that these strains produce the same TI signal and stimulate toxin synthesis via a common pathway.

#### The purified TI signal induces early transcription of the *C. difficile* toxin genes.
To test whether the TI signal induces transcription of the toxin genes, *tcdA* and *tcdB* transcript levels
Clostridium difficile Toxin Synthesis Regulation

**A**
- **Toxin Activity**
- **ELISA**
- **Growth**

| Incubation time (h) | 0 | 4 | 8 | 12 | 16 | 20 | 24 | 28 | 32 | 36 | 48 |
|---------------------|---|---|---|----|----|----|----|----|----|----|----|
| Toxin production (Specific activity [μMoles of nitrophenol/μg protein]) & ELISA (A 410nm)] | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 |
| Growth (O.D. 600nm) | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 |

**B**
- **Source of culture supernatant fluid**
  - **630**
  - **R20291**

| Time of culture supernatant collection | Medium | 4 | 8 | 16 | 24 | 28 | 32 |
|----------------------------------------|--------|---|---|----|----|----|----|
| Toxin Activity (μMoles of nitrophenol/μg of protein) | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 | 0.0 |

FIG 1  
(A) Toxin production in C. difficile is regulated and occurs during the stationary phase. Comparison of toxin detection by the more sensitive Cdifftox activity assay (33) with ELISA indicated that toxin production occurs during the early stationary phase of growth. An overnight culture of strain 630 cells was diluted 1:100 in brain heart infusion medium and incubated anaerobically at 37°C. Aliquots were collected every 4 h for OD<sub>600</sub> measurement and toxin detection. The ELISA assay was performed using the Wampole C. difficile TOX A/B II assay (Technologies Lab, Blacksburg, VA).  
(B) Toxin production in C. difficile is regulated in a cell density-dependent manner. The C. difficile quorum-signaling bioassay was performed using cell-free boiled and filtered stationary-phase culture supernatants from non-hypervirulent C. difficile strain 630, and the hypervirulent R20291 strain induced early toxin production in low-density log-phase strain 630 tester cells. Low-density log-phase cells were incubated for 4 h anaerobically at 37°C in a medium containing 75% reduced BHI broth and 25% supernatants collected at the times indicated. The resulting culture supernatant was tested for toxin activity. The means ± standard deviations (SD) from three independent experiments are shown.

were measured by reverse transcriptase PCR in early-logarithmic-phase cultures incubated with and without the TI signal. Strain 630 cells grown in the presence of the TI signal accumulated tcdA and tcdB mRNA within 4 h of incubation (Fig. 4). However, in the absence of the TI signal, no tcdA or tcdB transcripts were detected during the 12-h growth period (Fig. 4). As expected, toxin activity followed a temporal pattern similar to gene transcription (see Fig. S3 in the supplemental material). In contrast, the transcript profile of TcdC, which has been proposed to play a negative role in the transcriptional regulation of tcdA and tcdB, was unchanged in the TI signal-treated and untreated cells (Fig. 4). These findings indicate that tcdC transcription is not regulated by the TI signal.

Furthermore, it is noteworthy that toxin accumulates to high levels in TI signal-treated cells and that these cells also express high levels of tcdC. Thus, the positive regulatory effect of the TI signal on toxin production appears to override the proposed negative regulatory properties of TcdC (18). The high levels of toxin production and high levels of tcdA, tcdB, and tcdC transcripts in the TI signal-treated cells (Fig. 4) add to reports (24, 25) that contradict the proposed negative toxin-regulatory role of TcdC. Under our experimental conditions, the induction of toxin synthesis by the TI signal is not related to tcdC expression. These results indicate that the TI signaling pathway is the dominant regulatory system controlling C. difficile toxin synthesis.

Darkoh et al.

A

![Image Description](#)

**Retention Time (min)**

B

![Image Description](#)

**TI Signal injected**

**Toxin Activity**
(μMoles of nitrophenol/μg protein)

**HPLC Fractions**

FIG 2 HPLC analysis of the toxin-inducing (TI) activity. (A) Chromatogram from the HPLC purification showing the normalized peaks. Q-Sepharose-purified TI activity (100 μl) was injected onto a Phenomenex Jupiter 4μ Proteo 90A (250-mm by 4.6-mm) C₁₈ column (Phenomenex, Torrance, CA). The purification was performed with water as buffer A and acetonitrile as buffer B. A gradient of acetonitrile from 0 to 95% over 15 min at a flow rate of 0.5 ml/min was used. Fractions (0.5 ml) were collected and dried using an SPD111 SpeedVac (Thermo Scientific, Waltham, MA). Each fraction was resuspended in 200 μl of water and tested for TI activity using the quorum-signaling bioassay. (B) Toxin-inducing activities of the fractions collected from HPLC purification. Only fraction no. 6 (corresponding to the peak indicated with the yellow arrow in panel A) was active. Toxin activity was determined using the Cdiff tox activity assay. The means ± SD from three independent experiments are shown.

Role of the accessory gene regulator (Agr) system in TI signal-mediated regulation. To identify genes involved in TI signaling, we created a Himar-based transposon vector (37, 38) specific for random mutagenesis in *C. difficile*. We isolated isogenic mutants in strain R20291 that are defective in toxin production (Tox⁻) as determined by a tan colony phenotype compared to the blue parent strain on the Cdiff tox agar plates, an assay developed in our laboratory that differentiates toxin-producing colonies from nontoxin producers (39). One of these Tox⁻ mutants contained an insertion in the *C. difficile agrA* gene that encodes the response regulator component of the Agr quorum-signaling system (see Fig. S4 in the supplemental material). This R20291 *agrA* mutant generates the TI signal but does not respond to the TI signal and is deficient in toxin synthesis (Fig. 5A and B). Toxin synthesis and response to the TI signal were restored when the *agrA* mutant was complemented with the functional wild-type

*C. difficile agrA* (Fig. 5A). Martin et al. (40) also reported decreased *C. difficile tcdA* expression in the R20291 strain containing an insertion in the *agrA* gene.

The Agr system, thus far identified in Gram-positive bacteria, has been shown to regulate virulence in several Gram-positive bacteria (11, 41, 42). The Agr system is best characterized in *Staphylococcus aureus*, and it is transcribed as a four-gene operon carrying *agrA, agrB, agrC,* and *agrD* (35). The *agrD* gene encodes an autoinducer prepeptide, which is processed by the transmembrane protein AgrB, leading to release of an autoinducer peptide (AIP) into the extracellular milieu. The AgrC sensor histidine kinase protein senses and binds extracellular AIP, which activates its ATPase activity, leading to phosphorylation of the AgrA response regulator. Phosphorylated AgrA either activates transcription at the promoter region of the *agr* operon, leading to production of more of the Agr system components, or at the RNAIII promoter,

two-component systems present in its genome, two of which are likely candidates. For example, the VirR-like response regulator (CAJ67933.1) with a linked cognate VirS-like histidine sensor kinase in strain 630 is 47% identical to the AgrA of R20291.

Alignment of the AgrD1 and AgrD2 autoinducer prepeptides from the nine sequenced *C.* difficile strains reveals that every strain contains an AgrD1. These AgrD1 prepeptides are identical in amino acid sequence and have a Cys-28 residue, which is conserved among all active AgrD prepeptides (see Fig. S5 in the supplemental material). Only five of these sequenced strains (all hypervirulent) contain AgrD2, which all share 100% sequence identity and are 34% identical in sequence to AgrD1. The AgrD2 prepeptides do not contain any cysteine residues and have Ser-28 instead of Cys-28. In the *Staphylococcus aureus* AIPs, the conserved Cys-28 residues (as in AgrD1) form a thioester bond with amino acid residue 32 to create a 5-amino-acid cyclic peptide with a “tail” consisting of a combination of residues 24 to 27, depending on the subspecies. Synthesized *C.* difficile AIP modeled after the *S.* aureus AIP, which had the structures NSTCPWII, TCPWII, and acetyl-CPWII containing a thioester linkage between Cys-28 and Ile-32, were not active in the bioassay. However, hydroxylamine treatment, which is expected to disrupt the thioester bond (43–45), caused loss of the purified *C.* difficile TI signal activity (see Fig. S6 in the supplemental material), suggesting that a thioester bond is present in and necessary for the active TI signal. These data provide support for the cysteine-containing AgrD1 prepeptide as the source of the TI signal.

A second look at the R20291 strain bearing the transposon insertion in what we now term agrA2 presents an interesting finding. This insertion is expected to be polar and prevent expression of all the agr2 genes: agrA2, agrC2, agrD2, agrB2. As indicated above, this mutant is defective in TI signal response (Fig. 5A), presumably due to the absence of the signal response pathway elements AgrA2 and AgrC2, yet retains the ability to synthesize the TI signal (Fig. 5B). The ability of the agrA2 mutant to generate the TI signal and the presence of a thiolactone in the TI signal suggest that the cysteine-containing AgrD1 prepeptide is the source of the TI signal. The implication of this finding is that the TI signal is

FIG 3 Purified toxin-inducing (TI) signal stimulated early toxin production in both hypervirulent and non-hypervirulent *C.* difficile strains. The *C.* difficile cells were washed three times in fresh brain heart infusion medium and incubated with and without purified TI signal for 4 h at 37°C anaerobically. The culture supernatant was tested for toxin activity. Blue = strains incubated with only BHI medium; red = strains incubated with purified TI signal. The data represent the means ± SD from three independent experiments.

to increase the levels of the small RNA RNAIII, which in turn regulates the transcription of target genes.

Many of the Clostridia have two sets of agr genes (designated agr1 and agr2) that are located at different loci. The agr1 locus contains only the quorum signal generation pathway genes (agrB1 and agrD1), whereas the agr2 locus contains both the quorum signal generation and response pathway genes (agrB2D2 and agrC2A2, respectively). It is unclear how these two loci interrelate in a given strain. In *C.* difficile, the hypervirulent strains with sequenced genomes, including R20291, encode both agr1 and agr2 loci. However, surprisingly, the sequenced genomes of the non-hypervirulent strains, including 630, reveal that these strains encode only agr1. The quorum-signaling response pathway elements of 630 have not been identified, but there are eight orthologous

FIG 4 Purified toxin-inducing (TI) signal induces early *tcdA* and *tcdB* transcription but does not alter *tcdC* transcription. Strain 630 cells were cultured in the presence (+) or absence (−) of the purified TI signal, and aliquots were collected for total RNA isolation followed by cDNA synthesis by reverse transcription. PCR was performed using primers specific for *tcdA*, *tcdB*, *tcdC*, and 16S rRNA (control). Representative results from one of three separate experiments are shown. No PCR product was observed if reverse transcriptase was omitted (not shown).

Darkoh et al.

**FIG 5** The two-component response regulator AgrA is required for toxin expression. (A) A mutant of hypervirulent *C. difficile* strain R20291 with a transposon insertion in *agrA* (*agrA::Tn*) neither responds to the TI signal nor produces detectable toxins. Complementation of the *agrA* mutant with a plasmid expressing the functional *C. difficile agrA* gene (*agrA::Tn+Comp*) restores both TI signaling and toxin production. The means ± SD from three independent experiments are shown. (B) The *C. difficile agrA* mutant (R20291 *agrA::Tn*) synthesizes the TI signal but does not respond to the TI signal. The TI signal was purified from the *agrA* mutant (R20291 *agrA::Tn*) and tested in the bioassay using the R20291 wild type, R20291 *agrA* mutant (R20291 *agrA::Tn*), and strain 630 as tester cells. Toxin activity was measured using the Cdifftox activity assay. The means ± SD from three independent experiments are shown.

synthesized by a signal-generating pathway encoded by one locus (*agrB1D1*), which appears to be present in all *C. difficile* strains, and sensed and transduced by a signal response pathway encoded by another locus (*agrA2C2* in the case of strain R20291). However, in strain 630, which lacks *agrA2C2*, the TI signal may be sensed and transduced by one of the candidate two-component systems present in its genome. This genetic arrangement is unusual, because the component genes of most of the Agr quorum-signaling systems identified to date are located at the same locus (35, 36, 46).

The TI signal is present in CDI patient stools. To determine if quorum signal-dependent regulation of *C. difficile* toxin synthesis has clinical relevance, we analyzed TI signal levels in the stools from CDI patients. Stool samples from five *C. difficile* toxin-positive and five *C. difficile* toxin-negative patients were subjected to the TI signal purification protocol. Each of the five toxin-positive clinical stool preparations produced a single HPLC peak that eluted at the same elution time as the previously purified TI signal activity (data not shown). The material eluted from all the

toxin-positive stool samples induced early toxin synthesis in the bioassay (Fig. 6). Importantly, none of the toxin-negative clinical stool samples produced an HPLC peak similar to that of the TI signal HPLC peak. Moreover, none of the fractions from the toxin-negative clinical samples induced toxin synthesis in the bioassay. These results highlight the clinical relevance of the TI signal in *C. difficile* pathogenesis in the human host and provide further support for our hypothesis that *C. difficile* toxin production is controlled by the TI quorum signal that accumulates in the native colonic environment during infection.

**DISCUSSION**

A new paradigm for *C. difficile* regulation of toxin synthesis and pathogenesis. Several factors, such as nutrients, amino acids, and antibiotics, have been suggested to play a role in *C. difficile* toxin synthesis, but no specific regulatory mechanism has been proposed until now. This study provides a new understanding of CDI pathogenesis that explains the natural history of the infection and

by different loci (see Fig. S7 in the supplemental material). Another level of complexity revealed here is the ability of *C. difficile* 630 to respond to the TI signal using a non-Agr-designated two-component regulatory system that must sense and transduce the TI signal in this strain lacking the *agr2* locus. Identification and characterization of this new two-component regulatory system will likely provide evolutionary insights into this unique “hybrid” *C. difficile* Agr system. Furthermore, a clear understanding of this TcdC-independent regulation of toxin synthesis, including the role of the *agrB2D2* locus in the hypervirulent R20291 strain, will be forthcoming with our ongoing investigation of this system. Overall, these findings present an exciting opportunity to develop the first quorum-signaling-based non-antibiotic therapies, which can target many elements of the pathway, including the TI signal generation, sensing, and response pathway components, to combat *C. difficile*, a multidrug-resistant pathogen of significant public health importance.

## MATERIALS AND METHODS

**Bacterial strains and growth conditions.** Toxigenic *C. difficile* strain ATCC BAA-1382 (*tcdA*⁺ *tcdAB*⁺; strain 630) was purchased from the American Type Culture Collection (ATCC, Manassas, VA). *C. difficile* clinical isolates were kindly provided by Kevin Garey (University of Houston, Houston, TX); these isolates are R20291 (NAP1), CD196 (NAP1), TL176 (NAP4), CF5 (NAP9), and Liv022 (NAP11). The bacterial cultures were grown in BBL brain heart infusion (BHI) medium (Becton, Dickinson, Cockeysville, MD), or single colonies were isolated on the Cdifftox agar (39) plates. Cultures were incubated anaerobically in an atmosphere of 10% H₂, 5% CO₂, and 85% N₂ at 37°C in a controlled atmosphere anaerobic chamber (Plas-Labs, Lansing, MI). The BHI medium used in all the experiments was reduced overnight in the anaerobic chamber prior to use. The substrate for the Cdifftox activity assay (33), *p*-nitrophenyl-β-*d*-glucopyranoside, was purchased from Biosynth International (Itasca, IL).

**Sample storage conditions for the bacterial stocks.** Bacterial stocks were stored short term in chopped meat broth (BD Diagnostics, Franklin Lakes, NJ) at room temperature or long term in either 10% dimethyl sulfoxide or 15% glycerol stocks at −80°C.

**Toxin assays.**

(i) **Cdifftox activity assay.** The combined activities of the *C. difficile* toxins A and B were detected in the culture supernatant fluid using the Cdifftox activity assay (33). Briefly, the culture was centrifuged for 15 min at 10,000 × *g* at 4°C, and 250 μl of the supernatant fluid was incubated with 30 μl of 30 mM *p*-nitrophenyl-β-*d*-glucopyranoside and incubated either at 37°C for 4 h or overnight at room temperature. The assay was quantitated spectrophotometrically at an absorbance of 410 nm. A molar extinction coefficient for *p*-nitrophenol of ε = 17,700 M⁻¹ cm⁻¹ (47) was used for the calculation of the micromoles of *p*-nitrophenol produced per microgram of protein.

(ii) **Enzyme-linked immunosorbent assay.** For comparison, the presence of toxins A and B in samples was also confirmed with the Wampole *C. difficile* TOX A/B II assay (Technologies Lab, Blacksburg, VA). This assay was done using the protocol provided by the manufacturer.

(iii) **C. difficile quorum-signaling bioassay.** The *C. difficile* quorum-signaling bioassay was performed by incubating low-density logarithmic-phase cells (tester cells) with culture supernatant fluid collected from stationary-phase cells. This assay is based on the regulatory mechanism common to quorum (cell-density)-signaling systems (36). A small, soluble signaling molecule is produced continually by growing cells and accumulates extracellularly over time, such that it is at a low extracellular concentration in low-density cells and at a high extracellular concentration in high-density cells. As a result, low-density cells do not exhibit the quorum-signaling-dependent behavior, whereas the high-density cells exhibit the behavior. Thus, the introduction of the high-density supernatant containing a level of the signal that surpasses the critical threshold concentration to low-density cells stimulates these cells to exhibit the
quorum-dependent behavior prematurely. In the current assay, the presence of toxin-inducing activity in the stationary-phase culture supernatant induced low-density logarithmic-phase cells to produce toxin within 4 h. To prepare the tester cells, a single colony was selected from a Cdifftox agar plate (39) and cultured in 5 ml of fresh BHI medium overnight (16 to 18 h). The overnight culture (optical density at 600 nm \([OD_{600}]\) = 1.4 to 1.5) was diluted 1:100 with fresh BHI medium and incubated for 3 h (early log phase). For the quorum-sensing assay, 50 \(\mu\)l of washed low-density early-log-phase tester cells was added to a fresh medium containing 1 ml of boiled 0.2-\(\mu\)m-filtered stationary-phase supernatant and 2.95 ml of fresh BHI medium. The culture was incubated for 4 h anaerobically at 37°C. As a control, the tester cells were added to 3.95 ml of fresh reduced BHI medium and incubated under the same conditions as the treated cells. At the end of the incubation period, the supernatant was tested for toxin activity using the Cdifftox activity assay (33) and toxin production by ELISA.

Dialysis of the stationary-phase culture supernatant fluid. Boiled and 0.2-\(\mu\)m-filtered 48-h culture supernatant fluid was dialyzed in fresh BHI medium for 16 h. The culture supernatant fluid was placed into a seamless cellulose dialysis membrane (Sigma-Aldrich, St. Louis, MO) with different molecular mass cutoff values (0.1 to 0.5, 1, 3.5, and 12 kDa). The dialysis membranes containing the supernatant fluid (5 ml) were placed in a flask containing 500 ml of fresh BHI medium and incubated overnight at 4°C on a magnetic stirrer. The dialysis medium was changed four times during the dialysis period. The fluids in the dialysis membranes were tested for toxin-inducing activity using the quorum-signaling bioassay.

Purification of the *C. difficile* toxin-inducing activity from stationary-phase culture supernatant fluid. The toxin-inducing activity was purified from boiled stationary-phase supernatant as described below. Briefly, a 48-h culture supernatant (2 liters) from *C. difficile* strain 630 was boiled for 10 min to inactivate the toxins. The sample was cooled and centrifuged for 10 min at 10,000 × g to remove the denatured toxins and other high-molecular-weight proteins. The clear supernatant fluid was precipitated in 60% ice-cold acetone and incubated overnight at \(-20^{\circ}\mathrm{C}\). The precipitate was centrifuged at 10,000 × g for 20 min, and the pellet was resuspended in 40 ml of Milli-Q water, sterilized with a 0.2-\(\mu\)m surfactant-free cellulose acetate (SFCA) filter (Corning Incorporated, Corning, NY) and stored at \(4^{\circ}\mathrm{C}\) for the subsequent purification steps. Following testing for toxin-inducing activity, the filtered precipitate was further purified by Q-Sepharose anion exchange chromatography (GE Healthcare Bio-Sciences Corp., Piscataway, NJ). The anion exchange purification was performed with 50 mM Tris-HCl (pH 7.5) as buffer A and a step-gradient elution, with buffer B containing 50 mM Tris-HCl (pH 7.5) and 1 M NaCl. The fractions were dialyzed using a membrane with molecular mass cutoff values of 100 to 500 Da to remove the NaCl. Active toxin-inducing fractions were pooled and further purified by high-performance liquid chromatography (HPLC) using a Shimadzu Prominence HPLC system (Shimadzu Scientific Instruments, Columbia, MD) with a Phenomenex Jupiter \(4\mu\) Proteo 90A (250-mm by 4.6-mm) \(C_{18}\) column (Phenomenex, Torrance, CA). The HPLC purification was performed with HPLC-grade water as buffer A and acetonitrile as buffer B. A gradient of acetonitrile from 0 to 95% in 15 min at a flow rate of 0.5 ml/min was used. Fractions (0.5 ml) were collected and dried using an SPD111 SpeedVac (Thermo Scientific). To identify fractions that contained the toxin-inducing activity, the dried fractions were resuspended in 200 \(\mu\)l of Milli-Q water, and 100 \(\mu\)l was added to 3 ml of reduced BHI medium containing 50 \(\mu\)l of the tester *C. difficile* strain 630 cells. The culture was incubated for 8 h anaerobically at \(37^{\circ}\mathrm{C}\), and the supernatant fluid was tested for toxin activity using the Cdifftox activity assay (33). The active fraction was named toxin-inducing (TI) signal.

Effect of the purified toxin-inducing signal on hypervirulent strains of *C. difficile*. The effect of the purified TI signal on toxin synthesis in clinical epidemic hypervirulent and nonhypervirulent strains of *C. difficile* was evaluated. The following strains were tested: ATCC BAA-1832 (non-

epidemic historical strain 630), R20291 (NAP1), CD196 (NAP1), CF5 (NAP9), Liv022 (NAP11), and TL176 (NAP4). Washed, low-density, early-log-phase cells (50 \(\mu\)l) of these strains were added to reduced BHI medium (5 ml) containing 1 \(\mu\)g/ml of the purified TI signal and incubated anaerobically for 4 h at \(37^{\circ}\mathrm{C}\). For a negative control, the tester cells were incubated in only reduced BHI. The culture supernatant fluid was tested for the toxins at the end of the 4-h incubation period.

Analysis of *tcdA*, *tcdB*, and *tcdC* transcription. Tester *C. difficile* strain 630 cells at an \(OD_{600}\) of 0.6 were diluted 1:100 in fresh reduced BHI medium (35 ml) containing 1 \(\mu\)g/ml of the purified toxin-inducing activity and incubated anaerobically at \(37^{\circ}\mathrm{C}\) for 12 h. As a negative control, the tester cells were cultured in only fresh BHI medium. Aliquots of the culture were taken every 4 h for a measurement of growth at \(OD_{600}\), toxin testing, and mRNA analysis. Total RNA was isolated using the RNeasy kit (Qiagen) according to the manufacturer’s directions. The mRNA was converted to cDNA by reverse transcription using the ProtoScript AMV first-strand cDNA synthesis kit (New England Biolabs, Ipswich, MA) according to the manufacturer’s instructions. For reverse transcription, 1 \(\mu\)g of total RNA was used. The primers used for the PCR were 5′TGCC AGAAGCTCGCTCCACA3′ (forward) and 5′TGCACTTTGCTTTGATCA AAGCTCCA3′ (reverse) for *tcdA*, 5′GAGCACAAAAGGGTATTGCT ACTGGGC3′ (forward) and 5′CCAGACACAGCTAATCTTATTTGC CT3′ (reverse) for *tcdC*, 5′GTGTAGCAATGAAAGTCCAAGTTTAC GC3′ (forward) and 5′CACTTAGCTCTTTTGATTGCTGCACCT3′ (reverse) for *tcdB*, and 5′ACACGGTCCAAACTCCTACG3′ (forward) and 5′AGGCGAGTTTCAGCCTACAA3′ (reverse) for 16S rRNA. The PCR was performed using OneTaq Quick-Load 2× master mix (New England Biolabs) with the following conditions: initial denaturation of \(94^{\circ}\mathrm{C}\) for 30 s and 36 cycles of \(94^{\circ}\mathrm{C}\) for 30 s, \(56^{\circ}\mathrm{C}\) for 30 s, and \(68^{\circ}\mathrm{C}\) for 30 s. The PCR products were separated by 1.5% agarose gel electrophoresis and stained with ethidium bromide. Controls included samples of the RNA preparation processed without the reverse transcription step, which uniformly yielded no detectable PCR product.

Random mutagenesis. Random *C. difficile* mutants were generated using a transposon vector specially designed in our laboratory based on Himar (37, 38) for use with *C. difficile*. To generate mutants, *C. difficile* R20291 cells were incubated with conjugative *Escherichia coli* HB101 RK24 cells bearing the transposon vector. The conjugants were plated on Cdifftox agar plates (39) containing 250 \(\mu\)g/ml D-cycloserine, 8 \(\mu\)g/ml cefoxitin, and 15 \(\mu\)g/ml thiamphenicol. The conjugative *E. coli* cannot grow in the presence of D-cycloserine and cefoxitin and so was eliminated. Mutants (transformants) unable to produce toxins remained tan, whereas transformants able to produce active toxins turned blue on the Cdifftox agar plates. The presence of the transposon in the transformants was also confirmed by using PCR with primers specific for the plasmid. The transposon insertion sites were identified by an *E. coli* plasmid rescue procedure. Briefly, genomic DNA from the toxin-deficient mutants (tan colonies) was digested with MluI. The digest was ligated and transformed into competent Top10 *E. coli* cells. The transformed *E. coli* cells were grown on LB agar plates containing 25 \(\mu\)g/ml chloramphenicol to select for the colonies containing the “rescued” transposon insertion site-containing vector. The resulting plasmids were isolated and sequenced at the insertion site. To identify the genes containing the insertions, the *C. difficile* sequences adjacent to the transposon were subjected to BLAST comparison ([http://ncbi.nlm.nih.gov/blast](http://ncbi.nlm.nih.gov/blast)) to the *C. difficile* R20291 genomic sequence.

For complementation studies, the *agrA* coding sequence and the sequences 300 bp upstream of the start codon were PCR amplified from the wild-type R20291 template using oligonucleotides containing SalI and SphI restriction sites. The digested PCR product was inserted into pUTE657 (48) modified for conjugation in *C. difficile*, creating *agrA::Tn+Comp*. The plasmid was confirmed by DNA sequencing and conjugated into the *C. difficile* R20291 *agrA::Tn* mutant, as described above.

Multiple sequence alignment of *agrD* genes. To compare the similarities of the *agrD* gene in different *C. difficile* strains, the amino acid se-
quences from all the nine strains whose genomes have been sequenced were analyzed by multiple sequence alignment in comparison with the AgrD sequences of *Staphylococcus aureus*. This analysis was performed using BioEdit (49). Sequences with the following accession numbers were used for *C. difficile* strains: AM180355.1 (630), FN545816.1 (R20291), FN538970.1 (CD196), FN668941.1 (BI1), FN668375.1 (M68), FN665653.1 (M120), FN665652.1 (CF5), FN665654.1 (2007855), and FN668944.1 (BI9). For *S. aureus* Newman agr types, the following accession numbers were used: AY580334.1 (*agr* I), JX398934.1 (*agr* II), EF029035.1 (*agr* III), and JX398937.1 (*agr* IV).

Detection of the TI signal CDI patient stools. We utilized the HPLC-based TI signal purification method detailed above to test CDI patient stools for the presence of the TI signal. Five *C. difficile* toxin-positive and five *C. difficile* toxin-negative stool samples from patients diagnosed by the St. Luke’s Episcopal Hospital Diagnostic Laboratory at Texas Medical Center, Houston, were examined. Briefly, 1 g of stool was suspended in 5 ml of double-distilled water (ddH₂O) and centrifuged at 10,000 × g for 20 min. The supernatant was treated as described above for the purification of the TI signal. The fractions were tested for toxin activity in the bioassay detailed above.

**SUPPLEMENTARY MATERIAL**

Supplemental material for this article may be found at [http://mbio.asm.org/lookup/suppl/doi:10.1128/mBio.02569-14/-/DCSupplemental](http://mbio.asm.org/lookup/suppl/doi:10.1128/mBio.02569-14/-/DCSupplemental).

- Figure S1, TIF file, 0.8 MB.
- Figure S2, TIF file, 1.5 MB.
- Figure S3, TIF file, 1.3 MB.
- Figure S4, TIF file, 1.9 MB.
- Figure S5, TIF file, 2.9 MB.
- Figure S6, TIF file, 0.9 MB.
- Figure S7, TIF file, 2 MB.

**ACKNOWLEDGMENTS**

This work was supported in part by discretionary funds from the University of Texas School of Public Health, Texas Medical Center Digestive Diseases Center (Public Health Service grant DK56338), the Molecular Basis of Infectious Diseases Training Grant from the NIH Institute of Allergy and Infectious Diseases (T32AI055449), UNCF-MERCK Science Initiative (UMSI), and the Gillson-Longenbaugh Foundation.

We thank Alex Therien (Merck & Co.) and John Putkey (UTHSC) for discussions and comments, Paul Williams (University of Nottingham) for kindly providing the *C. difficile* AIP (NSTCPWII), and Hung Ton-That (UTHSC) for the generous use of his HPLC system.

**REFERENCES**

1. Dubberke ER, Olsen MA. 2012. Burden of *Clostridium difficile* on the healthcare system. Clin Infect Dis 55(Suppl 2):S88–S92. [http://dx.doi.org/10.1093/cid/cis335](http://dx.doi.org/10.1093/cid/cis335).
2. DuPont HL. 2011. The search for effective treatment of *Clostridium difficile* infection. N Engl J Med 364:473–475. [http://dx.doi.org/10.1056/NEJMe1013236](http://dx.doi.org/10.1056/NEJMe1013236).
3. Kyne L, Hamel MB, Polavaram R, Kelly CP. 2002. Health care costs and mortality associated with nosocomial diarrhea due to *Clostridium difficile*. Clin Infect Dis 34:346–353. [http://dx.doi.org/10.1086/338260](http://dx.doi.org/10.1086/338260).
4. O’Brien JA, LaHue BJ, Caro JJ, Davidson DM. 2007. The emerging infectious challenge of *Clostridium difficile*-associated disease in Massachusetts hospitals: clinical and economic consequences. Infect Control Hosp Epidemiol 28:1219–1227. [http://dx.doi.org/10.1086/522676](http://dx.doi.org/10.1086/522676).
5. Wilkins TD, Lyerly DM. 2003. *Clostridium difficile* testing: after 20 years, still challenging. J Clin Microbiol 41:531–534. [http://dx.doi.org/10.1128/JCM.41.2.531-534.2003](http://dx.doi.org/10.1128/JCM.41.2.531-534.2003).
6. Ananthakrishnan AN. 2011. *Clostridium difficile* infection: epidemiology, risk factors and management. Nat Rev Gastroenterol Hepatol 8:17–26. [http://dx.doi.org/10.1038/nrgastro.2010.190](http://dx.doi.org/10.1038/nrgastro.2010.190).
7. McDonald LC, Killgore GE, Thompson A, Owens RC, Jr, Kazakova SV, Sambol SP, Johnson S, Gerding DN. 2005. An epidemic, toxin gene-variant strain of *Clostridium difficile*. N Engl J Med 353:2433–2441. [http://dx.doi.org/10.1056/NEJMoa051590](http://dx.doi.org/10.1056/NEJMoa051590).
8. Redelings MD, Sorvillo F, Mascola L. 2007. Increase in *Clostridium difficile*-related mortality rates, United States, 1999–2004. Emerg Infect Dis 13:1417–1419. [http://dx.doi.org/10.3201/eid1309.061116](http://dx.doi.org/10.3201/eid1309.061116).
9. Bartlett JG, Perl TM. 2005. The new *Clostridium difficile*—what does it mean? N Engl J Med 353:2503–2505. [http://dx.doi.org/10.1056/NEJMe058221](http://dx.doi.org/10.1056/NEJMe058221).
10. Burke KE, Lamont JT. 2013. Fecal transplantation for recurrent *Clostridium difficile* infection in older adults: a review. J Am Geriatr Soc 61:1394–1398. [http://dx.doi.org/10.1111/jgs.12378](http://dx.doi.org/10.1111/jgs.12378).
11. Hansen D, Pollan LD, Fernando H. 2013. Fulminant *Clostridium difficile* colitis: a complication of perioperative antibiotic prophylaxis. J Oral Maxillofac Surg 71:1880–1885. [http://dx.doi.org/10.1016/j.joms.2013.04.035](http://dx.doi.org/10.1016/j.joms.2013.04.035).
12. Geric B, Rupnik M, Gerding DN, Grabnar M, Johnson S. 2004. Distribution of *Clostridium difficile* variant toxotypes and strains with binary toxin genes among clinical isolates in an American Hospital. J Med Microbiol 53:887–894. [http://dx.doi.org/10.1099/jmm.0.45610-0](http://dx.doi.org/10.1099/jmm.0.45610-0).
13. Rupnik M, Brazier JS, Duerden BI, Grabnar M, Stubbs SL. 2001. Comparison of toxotyping and PCR ribotyping of *Clostridium difficile* strains and description of novel toxotypes. Microbiology 147:439–447.
14. Voth DE, Ballard JD. 2005. *Clostridium difficile* toxins: mechanism of action and role in disease. Clin Microbiol Rev 18:247–263. [http://dx.doi.org/10.1128/CMR.18.2.247-263.2005](http://dx.doi.org/10.1128/CMR.18.2.247-263.2005).
15. Kuehne SA, Cartman ST, Heap JT, Kelly ML, Cockayne A, Minton NP. 2010. The role of toxin A and toxin B in *Clostridium difficile* infection. Nature 467:711–713. [http://dx.doi.org/10.1038/nature09397](http://dx.doi.org/10.1038/nature09397).
16. Mani N, Dupuy B. 2001. Regulation of toxin synthesis in *Clostridium difficile* by an alternative RNA polymerase sigma factor. Proc Natl Acad Sci USA 98:5844–5849. [http://dx.doi.org/10.1073/pnas.101126598](http://dx.doi.org/10.1073/pnas.101126598).
17. Mani N, Lyras D, Barroso L, Howarth P, Wilkins T, Rood JI, Sonenshein AL, Dupuy B. 2002. Environmental response and autoregulation of *Clostridium difficile* TxeR, a sigma factor for toxin gene expression. J Bacteriol 184:5971–5978. [http://dx.doi.org/10.1128/JB.184.21.5971-5978.2002](http://dx.doi.org/10.1128/JB.184.21.5971-5978.2002).
18. Carter GP, Douce GR, Govind R, Howarth PM, Mackin KE, Spencer J, Buckley AM, Antunes A, Kotsanas D, Jenkin GA, Dupuy B, Rood JI, Lyras D. 2011. The anti-sigma factor TcdC modulates hypervirulence in an epidemic BI/NAP1/027 clinical isolate of *Clostridium difficile*. PLoS Pathog. 7:e1002317. [http://dx.doi.org/10.1371/journal.ppat.1002317](http://dx.doi.org/10.1371/journal.ppat.1002317).
19. Hundsberger T, Braun V, Weidmann M, Leukel P, Sauerborn M, von Eichel-Streiber C. 1997. Transcription analysis of the genes *tcdA-E* of the pathogenicity locus of *Clostridium difficile*. Eur J Biochem 244:735–742. [http://dx.doi.org/10.1111/j.1432-1033.1997.t01-1-00735.x](http://dx.doi.org/10.1111/j.1432-1033.1997.t01-1-00735.x).
20. Matamouros S, England P, Dupuy B. 2007. *Clostridium difficile* toxin expression is inhibited by the novel regulator TcdC. Mol Microbiol 64:1274–1288. [http://dx.doi.org/10.1111/j.1365-2958.2007.05739.x](http://dx.doi.org/10.1111/j.1365-2958.2007.05739.x).
21. Curry SR, Marsh JW, Muto CA, O’Leary MM, Pasculle AW, Harrison LH. 2007. *tcdC* genotypes associated with severe TcdC truncation in an epidemic clone and other strains of *Clostridium difficile*. J Clin Microbiol 45:215–221. [http://dx.doi.org/10.1128/JCM.01599-06](http://dx.doi.org/10.1128/JCM.01599-06).
22. Jarraud S, Lyon GJ, Figueiredo AM, Lina G, Vandenesch F, Etienne J, Muir TW, Novick RP. 2000. Exfoliation-producing strains define a fourth *agr* specificity group in *Staphylococcus aureus*. J Bacteriol 182:6517–6522. [http://dx.doi.org/10.1128/JB.182.22.6517-6522.2000](http://dx.doi.org/10.1128/JB.182.22.6517-6522.2000).
23. Warny M, Pepin J, Fang A, Killgore G, Thompson A, Brazier J, Frost E, McDonald LC. 2005. Toxin production by an emerging strain of *Clostridium difficile* associated with outbreaks of severe disease in North America and Europe. Lancet 366:1079–1084. [http://dx.doi.org/10.1016/S0140-6736(05)67420-X](http://dx.doi.org/10.1016/S0140-6736(05)67420-X).
24. Bakker D, Smits WK, Kuijper EJ, Corver J. 2012. TcdC does not significantly repress toxin expression in *Clostridium difficile* 630ΔErm. PLoS One 7:e43247. [http://dx.doi.org/10.1371/journal.pone.0043247](http://dx.doi.org/10.1371/journal.pone.0043247).
25. Cartman ST, Kelly ML, Heeg D, Heap JT, Minton NP. 2012. Precise manipulation of the *Clostridium difficile* chromosome reveals a lack of association between the *tcdC* genotype and toxin production. Appl Environ Microbiol 78:4683–4690. [http://dx.doi.org/10.1128/AEM.00249-12](http://dx.doi.org/10.1128/AEM.00249-12).
26. Chung WO, Park Y, Lamont RJ, McNab R, Barbieri B, Demuth DR. 2001. Signaling system in *Porphyromonas gingivalis* based on a LuxS protein. J Bacteriol 183:3903–3909. [http://dx.doi.org/10.1128/JB.183.13.3903-3909.2001](http://dx.doi.org/10.1128/JB.183.13.3903-3909.2001).
27. Fong KP, Chung WO, Lamont RJ, Demuth DR. 2001. Intra- and inter-species regulation of gene expression by *Actinobacillus actinomycetemcomitans* LuxS. Infect Immun 69:7625–7634. [http://dx.doi.org/10.1128/IAI.69.12.7625-7634.2001](http://dx.doi.org/10.1128/IAI.69.12.7625-7634.2001).

Darkoh et al.

28. Ohtani K, Hayashi H, Shimizu T. 2002. The *luxS* gene is involved in cell-cell signalling for toxin production in *Clostridium perfringens*. Mol Microbiol 44:171–179. [http://dx.doi.org/10.1046/j.1365-2958.2002.02863.x](http://dx.doi.org/10.1046/j.1365-2958.2002.02863.x).

29. Sperandio V, Mellies JL, Nguyen W, Shin S, Kaper JB. 1999. Quorum sensing controls expression of the type III secretion gene transcription and protein secretion in enterohemorrhagic and enteropathogenic *Escherichia coli*. Proc Natl Acad Sci USA 96:15196–15201. [http://dx.doi.org/10.1073/pnas.96.26.15196](http://dx.doi.org/10.1073/pnas.96.26.15196).

30. Zhu J, Miller MB, Vance RE, Dziejman M, Bassler BL, Mekalanos JJ. 2002. Quorum-sensing regulators control virulence gene expression in *Vibrio cholerae*. Proc Natl Acad Sci USA. 99:3129–3134. [http://dx.doi.org/10.1073/pnas.052694299](http://dx.doi.org/10.1073/pnas.052694299).

31. Miller MB, Bassler BL. 2001. Quorum sensing in bacteria. Annu Rev Microbiol 55:165–199. [http://dx.doi.org/10.1146/annurev.micro.55.1.165](http://dx.doi.org/10.1146/annurev.micro.55.1.165).

32. Schauder S, Bassler BL. 2001. The languages of bacteria. Genes Dev 15:1468–1480. [http://dx.doi.org/10.1101/gad.899601](http://dx.doi.org/10.1101/gad.899601).

33. Darkoh C, Kaplan HB, DuPont HL. 2011. Harnessing the glucosyltransferase activities of *Clostridium difficile* for functional studies of toxins A and B. J Clin Microbiol 49:2933–2941. [http://dx.doi.org/10.1128/JCM.00037-11](http://dx.doi.org/10.1128/JCM.00037-11).

34. Ng WL, Bassler BL. 2009. Bacterial quorum-sensing network architectures. Annu Rev Genet 43:197–222. [http://dx.doi.org/10.1146/annurev-genet-102108-134304](http://dx.doi.org/10.1146/annurev-genet-102108-134304).

35. Novick RP, Geisinger E. 2008. Quorum sensing in staphylococci. Annu Rev Genet 42:541–564. [http://dx.doi.org/10.1146/annurev.genet.42.110807.091640](http://dx.doi.org/10.1146/annurev.genet.42.110807.091640).

36. Rutherford ST, Bassler BL. 2012. Bacterial quorum sensing: its role in virulence and possibilities for its control. Cold Spring Harb Perspect Med 2:pii=a012427. [http://dx.doi.org/10.1101/cshperspect.a012427](http://dx.doi.org/10.1101/cshperspect.a012427).

37. Bouhenni R, Gehrke A, Saffarini D. 2005. Identification of genes involved in cytochrome *c* biogenesis in *Shewanella oneidensis*, using a modified mariner transposon. Appl Environ Microbiol 71:4935–4937. [http://dx.doi.org/10.1128/AEM.71.8.4935-4937.2005](http://dx.doi.org/10.1128/AEM.71.8.4935-4937.2005).

38. Liu ZM, Tucker AM, Driskell LO, Wood DO. 2007. Mariner-based transposon mutagenesis of *Rickettsia prowazekii*. Appl Environ Microbiol 73:6644–6649. [http://dx.doi.org/10.1128/AEM.01727-07](http://dx.doi.org/10.1128/AEM.01727-07).

39. Darkoh C, DuPont HL, Kaplan HB. 2011. Novel one-step method for detection and isolation of active-toxin-producing *Clostridium difficile* strains directly from stool samples. J Clin Microbiol 49:4219–4224. [http://dx.doi.org/10.1128/JCM.01033-11](http://dx.doi.org/10.1128/JCM.01033-11).

40. Martin MJ, Clare S, Goulding D, Faulds-Pain A, Barquist L, Browne HP, Pettit L, Dougan G, Lawley TD, Wren BW. 2013. The *agr* locus regulates virulence and colonization genes in *Clostridium difficile*. J Bacteriol 195:3672–3681. [http://dx.doi.org/10.1128/JB.00473-13](http://dx.doi.org/10.1128/JB.00473-13).

41. Chen J, McClane BA. 2012. Role of the Agr-like quorum-sensing system in regulating toxin production by *Clostridium perfringens* type B strains CN1793 and CN1795. Infect Immun 80:3008–3017. [http://dx.doi.org/10.1128/IAI.00438-12](http://dx.doi.org/10.1128/IAI.00438-12).

42. Li J, Chen J, Vidal JE, McClane BA. 2011. The Agr-like quorum-sensing system regulates sporulation and production of enterotoxin and beta2 toxin by *Clostridium perfringens* type A non-food-borne human gastrointestinal disease strain F5603. Infect Immun 79:2451–2459. [http://dx.doi.org/10.1128/IAI.00169-11](http://dx.doi.org/10.1128/IAI.00169-11).

43. Buddelmeijer N, Young R. 2010. The essential *Escherichia coli* apolipoprotein N-acyltransferase (Lnt) exists as an extracytoplasmic thioester acyl-enzyme intermediate. Biochemistry 49:341–346. [http://dx.doi.org/10.1021/bi9020346](http://dx.doi.org/10.1021/bi9020346).

44. List F, Bocola M, Haeger MC, Sterner R. 2012. Constitutively active glutaminase variants provide insights into the activation mechanism of anthranilate synthase. Biochemistry 51:2812–2818. [http://dx.doi.org/10.1021/bi201618v](http://dx.doi.org/10.1021/bi201618v).

45. Skaff DA, Ramyar KX, McWhorter WJ, Barta ML, Geisbrecht BV, Miziorko HM. 2012. Biochemical and structural basis for inhibition of *Enterococcus faecalis* hydroxymethylglutaryl-CoA synthase, *mvaS*, by hymeglusin. Biochemistry 51:4713–4722. [http://dx.doi.org/10.1021/bi300037k](http://dx.doi.org/10.1021/bi300037k).

46. Geisinger E, George EA, Chen J, Muir TW, Novick RP. 2008. Identification of ligand specificity determinants in AgrC, the *Staphylococcus aureus* quorum-sensing receptor. J Biol Chem 283:8930–8938. [http://dx.doi.org/10.1074/jbc.M710227200](http://dx.doi.org/10.1074/jbc.M710227200).

47. Shikita M, Fahey JW, Golden TR, Holtzclaw WD, Talalay P. 1999. An unusual case of “uncompetitive activation” by ascorbic acid: purification and kinetic properties of a myrosinase from *Raphanus sativus* seedlings. Biochem J 341:725–732. [http://dx.doi.org/10.1042/0264-6021:3410725](http://dx.doi.org/10.1042/0264-6021:3410725).

48. Wemmer DE, Kern D. 2005. Beryllium fluoride binding mimics phosphorylation of aspartate in response regulators. J Bacteriol 187:8229–8230. [http://dx.doi.org/10.1128/JB.187.24.8229-8230.2005](http://dx.doi.org/10.1128/JB.187.24.8229-8230.2005).

49. Hall TA. 1999. BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT. Nucleic Acids Symp Ser 41:95–98.

50. Lyerly DM, Saum KE, MacDonald DK, Wilkins TD. 1985. Effects of *Clostridium difficile* toxins given intragastrically to animals. Infect Immun 47:349–352.
