
IAI Accepted Manuscript Posted Online 30 March 2015  
Infect. Immun. doi:10.1128/IAI.00240-15  
Copyright © 2015, American Society for Microbiology. All Rights Reserved.

The CpAL quorum sensing system regulates production of hemolysins CPA and  
PFO to build C. perfringens biofilms  

Jorge E. Vidal${}^{1}$, Joshua R. Shak${}^{1}$ and Adrian Canizalez-Roman${}^{2}$  

${}^{1}$Hubert Department of Global Health, Rollins School of Public Health, Emory University, Atlanta GA  
30322, ${}^{2}$University of Sinaloa School of Medicine, Culiacan Sinaloa, Mexico  

Running title: CpAL regulation of C. perfringens biofilms  

*Address correspondence to:*  
Jorge E. Vidal, PhD  
1518 Clifton Rd NE Room 6007, Atlanta GA, 30322.  
Phone 404-712-8675, Fax: 404-712-8969.  
Email [jvidalg@emory.edu](mailto:jvidalg@emory.edu),

Abstract

Clostridium perfringens strains produce severe disease in humans and animals including myonecrosis and enteritis necroticans. Diseases are mediated by production of potent toxins that often damage the site of infection, e.g., skin epithelium during myonecrosis. In planktonic cultures, regulation of important toxins, such as CPA, CPB, and PFO, is controlled by the C. perfringens Agr-like (CpAL) quorum sensing (QS) system. Strains also encode a functional LuxS/AI-2 system. While C. perfringens strains form a biofilm-like structures, regulation of biofilm formation is poorly understood. Therefore, our studies investigated the role of CpAL and LuxS/AI-2 QS systems, and QS-regulated factors, in controlling formation of biofilms. We first demonstrate that biofilm production by reference strains differs depending on the culture medium. Increased biomass correlated with the presence of extracellular DNA in the supernatant, which was released by lysis of a fraction of the biofilm population and planktonic cells. Whereas mutants in ΔagrB were not able to produce biofilms, a ΔluxS mutant made wild-type levels. Transcript levels of CpAL-regulated cpa and pfoA genes, but not cpb, were upregulated in biofilms in comparison to planktonic cultures. Accordingly, Δcpa and ΔpfoA mutants, in type A (S13) or type C (CN3685) backgrounds, were unable to produce biofilms whereas CN3685Δcpb made wild-type levels. Biofilm formation was restored in complemented strains Δcpa/cpa and ΔpfoA/pfoA. Confocal microscopy studies further detected CPA partially colocalizing with eDNA on the biofilm structure. Thus, CpAL regulates biofilm formation in C. perfringens by increasing levels of certain toxins required to build biofilms.

Word count: 246

The Gram positive, spore-forming bacteria *Clostridium perfringens* is the most widely distributed pathogen in nature (1-3). This anaerobe affects humans and animals producing severe diseases including food poisoning (4), gastrointestinal syndromes, and enterotoxemias (3, 5, 6), gas gangrene, and numerous histotoxic infections (7-9). With the exception of food poisoning caused by *C. perfringens* enterotoxin (CPE), a common characteristic in most toxigenic *C. perfringens* vegetative infections is the rapid progression of the disease to a final, often fatal, outcome. The virulence of *C. perfringens* is directly related to its prolific production of more than 17 different potent toxins. Current schemes, however, utilize differential production of four lethal typing toxins [alpha (CPA), beta (CPB), epsilon (Etx) and/or iota (Itx)], to classify *C. perfringens* isolates into 5 pathogenic types [A-E] (2, 10-13). *C. perfringens* type A, for example, must produce CPA whereas type C strains produce CPA and CPB. All toxinotypes make other biomedically important toxins such as perfringolysin O (PFO) or CPB2 (10).

*C. perfringens* type A and type C strains have been isolated from human cases of severe disease (2). *C. perfringens* type A produces near ~90% of all gas gangrene cases (3, 9, 13). Gas gangrene, or myonecrosis, is considered one of the most fulminate Gram-positive infections in humans and animals (8, 9). Infection starts from the site of a recent surgical wound or trauma where *C. perfringens* type A strains must first attach to the disrupted epidermal epithelium and proliferate while at the same time producing toxins that necrotize the tissue (8, 14-16). Tissue destruction associated with *C. perfringens* infection progresses rapidly to involve an entire extremity (8, 14, 17-19). Amputation remains the single best life-saving treatment, though mortality still remains high (8, 9, 15, 16).

Production of *C. perfringens* alpha toxin (CPA), and in some extent PFO, have largely been implicated in clostridial myonecrosis (3, 9, 13, 20, 21). For example, a *C. perfringens* type A strain

[cpa⁺ and pfoA⁺] was highly virulent in a mouse model of clostridial myonecrosis, while isogenic cpa or pfoA mutants showed reduced virulence (including reduced tissue destruction) (17, 19, 22, 23). Elimination of both CPA and PFO production (a double cpa-pfoA toxin gene mutant) removed most of the histopathological features typical of clostridial myonecrosis (24).

C. perfringens type C is the etiologic agent of human enteritis necroticans (also called Pigbel, Darmbrand or “gangrene of the bowel”) which originates in the intestine (2). Enteritis necroticans (EN), is particularly aggressive in diabetics and immunocompromised patients from developed countries (they survive less than 48 h after the first appearance of symptoms) (25-30). EN is characterized by its sudden onset with abdominal cramps, shock, bloody diarrhea, acute inflammation and pronounced necrosis of intestinal mucosa (27, 31). Necrosis of the intestinal epithelium always coincides with the presence of Gram positive rods characteristic of C. perfringens attached to the surface of the necrotic mucosa that, when visualized by microscopy, appears to form biofilm-like structures (28, 29). CPB, the most important toxin for developing EN (32-34), has been detected on the apical side of intestinal cells underneath those attached bacteria indicating that in situ production of CPB is important for pathogenesis (29). A similar EN disease produced by type A and type C strains affects chickens and other domestic birds. Chicken EN globally cost to the poultry industry 2 billion USD per year (35).

We recently demonstrate that C. perfringens strains encodes a functional agr-like (accessory gene regulator) quorum sensing system (QS) that controls production of some C. perfringens toxins (36, 37). The C. perfringens Agr-like (CpAL) system is related to the Agr system from S. aureus and other Gram positives (36, 38-41). Signaling through CpAL requires a secreted pheromone encoded by the agrD gene which is processed to a functional cyclic peptide by a transmembrane protein encoded by the agrB gene (36, 37). The CpAL system regulates in vitro production of CPA, and PFO in all studied toxin types including type A and type C strains (36, 42-44). CpAL also regulates production of C.

perfringens beta toxin (CPB) in type B and type C strains (42, 43). Moreover, a functional CpAL system

was required *in vivo* to produce necrotizing enteritis in rabbit ileal loops by regulating intestinal levels of

CPB (42).

Another QS system encoded by genome-sequenced *C.* perfringens strains, the LuxS/AI-2

system, was implicated in transcriptional regulation of *pfoA*, but not *cpa* or other toxin genes, in mid-

exponential planktonic cultures of type A strain 13 (45). While *in vivo* and *in vitro* studied, LuxS-AI-2

regulation has not been identified in *pfoA*-encoded *C.* perfringens type C strain CN385 (42, 46).

As mentioned earlier, *C.* perfringens strains may form biofilm-like structures on host tissues or

epithelia. Biofilms are bacterial communities surrounded by a matrix providing the mechanical stability,

mediate their adhesion to surfaces and form a cohesive, three-dimensional polymer network that protect

them from the host response and antibiotics (47). Biofilm production in *C.* perfringens has been poorly

studied. Initial attempts to characterize *C.* perfringens biofilms demonstrate that biofilms are produced

*in vitro* by some strains (48, 49). The matrix has been recently studied and found to be made of some

proteins, β-1,4 linked polysaccharides and extracellular DNA (eDNA) (49, 50). A type IV pilus (TFP) is

also required for biofilm formation, these pili were located on the biofilm matrix (48).

More recently, Obana *et al* (2014) reported that biofilms formed at 37°C have an increased

biomass and a phenotypically different biofilm matrix in comparison to those formed at 25°C.

Furthermore, incubation at 37°C, but not at 25°C, upregulated levels of *pilA2* mRNA which encodes

the main pilin subunit of the matrix-associated *C.* perfringens TFP (50). These data suggest that *C.*

perfringens biofilms are differentially regulated. In this study, we investigated quorum sensing

regulation of *C.* perfringens biofilms and demonstrate that the CpAL, but not the LuxS/AI-2 system, is

required for proper formation of the biofilm structure. Moreover, CpAL-regulated CPA and PFO toxins,

but not CPB, were required for biofilm formation and transcription of encoded genes *cpa* and *pfoA*, but

not *cpb*, were up-regulated in biofilms when compared to planktonic cultures.

Material and Methods

Strains, culture media and chemicals. Wild-type strains and mutant derivatives utilized in this study are listed in Table 1. Media for culturing *C. perfringens* included FTG (fluid thioglycolate medium; Difco Laboratories), TGY (3% tryptic soy broth [Becton-Dickinson]; 2% glucose [Fisher scientific]; 1% yeast extract [Becton-Dickinson], 0.1% sodium thioglycolate [Sigma Aldrich]), Brain Heart Infusion (BHI) agar (Becton-Dickinson). Sodium thioglycolate, a reducing agent, present in TGY maintains a low oxygen tension in the medium. Blood agar plates were prepared with BHI and 5% of defibrinated sheep blood (Reme, Lenexa KS); egg yolk agar plates were prepared with BHI containing 4% (v/v) egg yolk. For culturing *E. coli*, Luria-Bertani (LB) broth (1% tryptone [Becton-Dickinson], 0.5% yeast extract, 1% NaCl) and LB agar (1.5% agar [Becton-Dickinson]) medium were used. All antibiotics used in this study were purchased from the Sigma-Aldrich Chemical Company.

Production of *C. perfringens* biofilms. Strains were inoculated into FTG and grown overnight at 37°C. To produce biofilms, these FTG cultures were inoculated (diluted 1:10) in triplicate into either 8-well glass slides (Lab-Tek), polystyrene 96-well plates (Corning), CellBIND® surface polystyrene 24-well plates (Corning) or tissue culture-treated polystyrene 6-well plate (Corning) containing FTG or TGY with no antibiotics and incubated at 37°C for the indicated time.

Quantification of biofilm biomass. Biofilm development was evaluated using the classic crystal violet staining method as previously reported (51). Alternatively, biofilms were stained by fluorescence with membrane-permeant fluorescent reagents SYTO 9® (Molecular probes) or DAPI. These molecules stain nucleic acids. Briefly, biofilms were washed three times with sterile PBS (pH=7.4) and incubated with 5 mM of SYTO 9® for 20 min at room temperature or DAPI (30 nM) and incubated for 5 min. After two

137 washes with PBS, fluorescence (arbitrary units) was quantified using a VICTOR™ X 3 Multilabel Plate
138 Reader (Perkin-Elmer). Biofilms were photographed using an inverted Evos fluorescence microscope
139 (Advanced Microscopy Group). Fluorescence arbitrary units of a chosen experimental condition (e.g.,
140 wt biofilms 8h post-inoculation) were set to 100% of biofilm biomass and used to calculate the
141 percentage of biofilm biomass of all other tested conditions.
142 Quantification of extracellular DNA (eDNA). Strains were inoculated into 24-well plates and
143 incubated for the indicated time at 37°C. The supernatant (~1 ml) was removed, centrifuged for 5 min at
144 12,000×g in a refrigerated centrifuge (Eppendorf) and filter sterilized using a 0.45 μM syringe filter
145 (Fisher Scientific). This supernatant was mixed with a 0.5 volume of ethanol, vortexed for 10 s and
146 eDNA was purified using the QIAamp eDNA minikit (QIAGEN) following the manufacturer's
147 instructions. To quantify amounts of DNA, a quantitative PCR (qPCR) assay targeting the cpa gene was
148 utilized. Reactions were performed with IQ™ SYBR green supermix (BioRad), 300 nM of each primer
149 [JEV193L and JEV194R (Table 2)] and 2 μl of DNA template. Reactions were run in duplicate using a
150 CFX 96 Real-Time PCR Detection System (Bio-Rad) with the following conditions: 1 cycle at 95°C for
151 3 min and 40 cycles of 95°C for 15 s and 55°C for 1 min and 72°C for 1 min. Melting curves were
152 generated by a cycle of 95°C for 1 min, 65°C for 1 min and 80 cycles starting at 65°C with 0.5°C
153 increments. For quantification purposes, standards (1, 10, 100 and 1000 pg of purified chromosomal
154 DNA per reaction) were run in parallel to generate a standard curve. Amounts of eDNA in pg/ml were
155 calculated using the software Bio-Rad CFX manager.
156 Viability of biofilms and planktonic cells using the LIVE/DEAD assay. *C. perfringens* type A strain
157 13, or type C strain CN 3685, was inoculated in 96-well plates containing TGY and incubated for 4 or 8
158 h at 37°C. Biofilms made 4 or 8 h post-inoculation were washed three times with sterile PBS and
159 immediately stained utilizing the LIVE/DEAD BacLight bacterial viability kit L7012 (Invitrogen, Grand

Island, NY). The assay has successfully been used to investigate biofilms made by *C. difficile* and *C.*

perfringens* (48, 52). Planktonic cells were removed at the indicated time and stained as above. The

concentration of dyes was utilized as per manufacturer's recommendations. Stained biofilms, or

planktonic cells, were washed additionally two times with PBS and fluorescence (arbitrary units) were

obtained with a VICTOR™ X3 Multilabel Plate Reader (Perkin-Elmer). Fluorescent dyes included in

the kit incorporate, or not, into bacterial cells as a function of membrane integrity, and therefore

viability. Preparations were also observed and photographed utilizing an inverted Evos fluorescence

microscope (Advanced Microscopy Group).

Construction of CN3685 and Strain 13 null mutants using the TargeTron® technology. The

following genes were insertional-inactivated using the Clostridium TargeTron gene knock-out system

(Sigma-Aldrich): *cpb* (34), *pfoA* (34), *plc/cpa* (52), *luxS* (46) and *agrB* (53). The *plc/cpa* mutant was

prepared with plasmid pJIR750ai included in the TargeTron gene knock-out system. Derivative

plasmids to insertional-inactivate all others were prepared by cloning target-specific sequences as

follows: a set of primers (IBS, EBS1d and EBS2) that had been previously published (Table 2), where

utilized to generate a target-specific ~350-bp PCR product, this target-specific fragment replaced the

*plc/cpa*-specific fragment on pJIR750ai. Intron plasmid, that integrated a ~900 bp intron fragment into

the target gene, was then electroporated into either strain 13 or CN3685. Except for the mutation into the

*agrB* gene, in which the intron was integrated in the sense orientation, the intron fragment was inserted

in the antisense orientation in all other target genes. Transformants were selected on BHI agar plates

containing 15 μg/ml of chloramphenicol (Cm) and then PCR-screened for an intron-disrupted gene

using the following target-specific primers: *cpb* (*cpb*-Fwd and *cpb*-Rev), *plc/cpa* (JEV-193L and

JEV194R), *pfoA* (JEV181L and JEV182R), *agrB* (JEV135L and 126R), and *luxS* ((luxS-L and luxS-R).

Our PCR screening was designed as to amplify a ~200-300 bp product from wild-type genes, but

183 amplified a larger ~1.1-1.2 kb product from those mutants. Mutants carrying an intron insertion were
184 grown in FTG medium without Cm for ~10 days, with daily subculturing, to cure the intron-carrying
185 donor plasmid. Curing was shown by lack of growth on Cm-containing BHI plates.
186 Preparation of cpa and pfoA complemented strains. Plasmids pJIR750 (ATCC® 87015™) or
187 pJIR751 (ATCC® 87016™) (54) were purified and double-digested with restriction enzymes, EcoRI and
188 BamHI or EcoRI and XbaI, respectively. Digested plasmids were again purified using the Qiaquick gel
189 extraction kit (Qiagen). Simultaneously, the pfoA or plc gene, including promoter regions, were
190 amplified by PCR using primers JEV165L and JEV166R (pfoA) or JEV167L and JEV168R (plc),
191 purified and double-digested with enzymes EcoRI and BamHI or EcoRI and XbaI, respectively.
192 Digested fragments were ligated to pJIR750-pfoA (pCP13pfoA) or pJIR751-plc (pCP13plc) using T4
193 DNA ligase (Promega) and transformed into competent cells of S13ΔpfoA (pJIR750-pfoA) or S13Δplc
194 (pJIR751-plc) by electroporation as previously described (36). Transformants were plated onto BHI
195 plates containing Cm or erythromycin, respectively and isolated colonies were screened for the presence
196 of both the intron-disrupted mutant gene and plasmid-encoded wt genes as mentioned above.
197 Preparation of an agrB complemented strain. Taking advantage that the insertion of the intron within
198 agrB created a conditional mutation (i.e., the intron fragment was integrated in the sense strand),
199 complementation of the ΔagrB null mutant CPJV21 was achieved at the mRNA level by removing the
200 intron from disrupted agrB mRNA, thus restoring a functional agrB transcript (55). Removal of this
201 intron was achieved by reintroducing pCPJVagrB, which encodes the LtrA protein required to splice out
202 the intron insertion, into CPJV21. Transformants were selected on BHI agar containing 15 μg/ml of
203 chloramphenicol. Strain CPJV32 was grown at 30°C to allow maximum expression of the LtrA protein.
204 RNA extraction and analysis of RNA preparations. Planktonic or biofilm cells were added with 1
205 volume of RNeasy Protect Bacteria® (Qiagen) and immediately centrifuged for 15 min at 15000 x g in a

206 refrigerated centrifuge (Eppendorf). Total RNA was then extracted from the pellet using the RNeasy
207 Mini Kit (Qiagen) as outlined by the manufacturer and additionally treated with 2 U of DNase I
208 (Promega) essentially as previously described (46, 51). Integrity of our RNA preparations [RNA
209 integrity number (RIN)], and RNA concentration of samples, were obtained by using the RNA 6000
210 Nano kit and the 2100 Bioanalyzer (Agilent technologies).
211 RT-PCR reactions. Reactions were performed with 100 ng of DNase I-treated RNA as template. Those
212 reactions contained 500 nM of each primer (Table 2), 1x PCR master mix (New England Biolabs),
213 molecular grade water and 10 units of AMV reverse transcriptase (Promega). Control RT-PCR reactions
214 were similarly performed, except for the omission of reverse transcriptase (negative) or the addition of
215 100 ng of genomic DNA (positive). Reactions conditions were the following: initial incubation at 42°C
216 for 30 min and denaturation at 95°C for 5 min, followed by 35 cycles of 95°C for 15 s, 55°C for 30 s and
217 68°C for 1 min and a final extension at 68°C for 10 min. RT-PCR products were run in 2% agarose gels
218 and stained with SYBR® Safe DNA Gel Stain (Invitrogen).
219 Quantitative RT-PCR (qRT-PCR). Reactions were performed with iScript One-Step RT-PCR kit with
220 SYBR Green (Bio-Rad), 300 nM of the indicated primers (Table 2), and 10 ng of DNase I-treated RNA
221 template. Reactions were run in duplicate using a CFX 96 Real-Time PCR Detection System (Bio-Rad)
222 and the following conditions; 1 cycle at 50°C for 30 min, 1 cycle at 95°C for 10 min and 40 cycles of
223 95°C for 15 and 55°C for 1 min. Melting curves were generated as above. The relative quantitation of
224 mRNA expression was normalized to the constitutive expression of the housekeeping polC gene and
225 calculated by the comparative CT (2^-ΔΔCT) method (56).
226 Hemoglobin (Hb) release assay. *C. perfringens* strains were grown in 24-well plates containing TGY
227 for 4 h; maximal expression of PFO, and therefore Hb release, has been observed at this time point (36).
228 Culture supernatant was obtained by centrifugation at 20,000 × g for 10 min and then filter sterilized

229 using a 0.45 μm filter (Fisher Scientific). The Hb release assay was then performed essentially as
230 previously described (46). Photographs of erythrocytes pellets in ΔpfoA mutants were also obtained.

231 Detection of CPA by dot blot. Strains were grown in 24-well plates containing TGY for 8h at 37°C
232 after which the culture was centrifuged at 20,000×g for 10 min. An aliquot (50 μl) of the supernatant
233 was spotted onto a nitrocellulose membrane, air dried and then blocked with PBS-Tween 20 (0.05% v/v)
234 and nonfat dry milk (5% wt/v) for 1 h at room temperature. Membranes were probed with a polyclonal
235 anti-CPA antibody (Bioss, Antibodies, Woburn MA) followed by incubation with horseradish
236 peroxidase-conjugated secondary anti-rabbit antibodies (Jackson ImmunoResearch) and addition of
237 SuperSignal West Pico Chemiluminescent Substrate (Pierce).

238 Confocal microscopy studies. C. perfringens strains were inoculated in 8-well glass slides (Lab-Tek)
239 containing TGY and incubated for 24 h at 37°C. Biofilms produced were washed twice with PBS and
240 fixed with 2% paraformaldehyde for 15 min at room temperature. Fixed biofilms were then
241 permeabilized with 0.5% Triton X-100 for 5 min, washed again with PBS and blocked with 1% BSA for
242 30 min at 37°C. Nucleic acids were stained with SYTO 9 as above. CPA was stained using a rabbit
243 polyclonal anti-CPA antibody followed by a goat anti-rabbit secondary antibody conjugated to Alexa
244 Fluor 555 (Molecular Probes). Stained biofilms were finally washed three times with PBS, mounted
245 with vectashield mounting medium (Vector laboratories) and analyzed with an Olympus FV 1000
246 confocal microscope. Confocal images were analyzed with ImageJ version 1.49k (National Institutes of
247 Health, USA).

248 Statistical analysis. All statistical analysis presented in this study were performed by the Mann Whitney
249 U test and using the software SigmaPlot Version 12.0 (Systat Software, Inc.).

250

251 Results

252 Production of *C. perfringens* biofilms is regulated by environmental signals.

253 Environmental signals, such as those present in bacterial culture media, have been associated

254 with biofilm production in some Gram positive bacteria (51, 57, 58). We therefore investigated whether

255 the culture medium would impact formation of *C. perfringens* biofilms. To assess this, we used two

256 classic *C. perfringens* broth media, FTG and TGY (59). As shown in Fig. 1A, we found that the biofilm

257 mass produced in TGY by two different *C. perfringens* type C reference strains, CN3685 (33, 34) and

258 JGS1495 (60), was significantly greater than that produced on FTG. Biofilms produced on TGY were

259 observed in the wells as a thick white pellicle (Fig. 1B, bottom panels). In comparison with biofilms

260 made on FTG that covered ~70% of the substrate, biofilms produced on TGY covered the entire surface

261 (Fig. 1B). Similar results were obtained when a *C. perfringens* type A strain 13 was inoculated in FTG

262 and TGY (not shown). We next investigated the biofilm phenotype of other *C. perfringens* type C strains

263 isolated from porcine enteritis (Table 1). All tested strains produced similar biofilm biomass on a

264 microtiter plate, 24 h post inoculation (Fig. 1C). Together, our results suggest that *C. perfringens*

265 biofilms are regulated by environmental signals.

266

267 Development of a fluorescent assay to study *C. perfringens* biofilms.

268 To improve current methods to study *C. perfringens* biofilms, we developed a fluorescence-

269 based method using DAPI, or SYTO9. These fluorophores have different excitation/emission spectra

270 and are readily available. *C. perfringens* biofilms stained with these fluorescence reporters were

271 compared with the biofilm mass obtained using the classic crystal violet procedure. Fluorescence

272 arbitrary units of biofilms, produced 8 h post-inoculation, were set to 100% and biomass of all other

273 time points was calculated. The classic protocol, utilizing crystal violet, revealed similar biomass 4 and

274 6 h post-inoculation and a statistically different mass 8 h post-inoculation (Fig. 2A). The fluorescence

method revealed a statistically different biomass which increased at each time point (Fig. 2B). The latter

correlated with microscopic observations of *C. perfringens* biofilms (Fig. 2C). Similar results were

obtained when biofilms were stained with DAPI (not shown).

*C. perfringens* releases extracellular DNA (eDNA) into the culture medium. The biofilm matrix of

other pathogens, such as *S. pneumoniae* (61, 62) or *Enterococcus faecalis* (63), is partially composed of

DNA that has been excreted into the medium (eDNA). It has been recently described that eDNA is

present in the *C. perfringens* biofilm matrix (49, 50). To assess if *C. perfringens* cells release DNA to

the extracellular milieu, DNA was purified from supernatants of cultures of *C. perfringens* type C strain

CN3685. Released eDNA was quantified using qPCR reactions targeting the chromosomally-encoded

*cpa* gene (13). Presence of eDNA was detected as early as 2 h post-inoculation (5.5x10⁴ pg/ml),

reaching a maximum peak 6 h post-inoculation (1.1x10⁵ pg/ml) with a non-statistically significant drop

of eDNA in the supernatant after 8 h of incubation (Fig. 3A). Increasing presence of eDNA in the

supernatant is consistent with the increased biomass formed through time (see Fig. 2). As release of

eDNA in other species has been linked in part to autolysis, we sought to investigate the viability of *C.*

*perfringens* biofilms and planktonic cells. Our experiments showed that ~25% of the biofilm bacteria,

appeared to be dead, as scored by the live/dead stain, at 4 h post-inoculation or CN3685 (Fig. 3B) or

strains 13 (not shown). Likewise, ~16% of the biofilm population at 8 h post-inoculation consisted of

dead bacteria (Fig. 3C). Similar populations of dead planktonic cells were observed in 8 h cultures of

strain 13 or CN3685 (Fig. 3C and 3D). Our data suggest that eDNA is in part provided by lysis of both

planktonic and biofilm bacteria.

The CpAL quorum sensing system, but not the LuxS/AI-2, regulates *C. perfringens* biofilms.

C. perfringens strains produce two different quorum sensing systems, an Agr-like QS system and

the LuxS/AI-2 system (36, 37, 45). To investigate whether biofilms are regulated by quorum sensing, we

prepared CN3685 derivatives CN3685ΔagrB and CN3685ΔluxS strains using the targetron technology.

Whereas CN3685ΔluxS produced biofilm levels similar to those of the wt strain, the mutant strain

CN3685ΔagrB made significantly less biofilm mass 24h (Fig. 4A) or 48h post inoculation (not shown).

A time-course study revealed C. perfringens wt bacteria had already attached to the substrate 3h post-

inoculation (early biofilms), and covered the entire surface 6h later (Fig. 4B). Similarly, the luxS mutant

produced wt biofilm levels 6h post-inoculation whereas biofilms of the ΔagrB, at the same time-point,

were not observed (Fig. 4B). Similar results were obtained when the agrB was insertionally-inactivated

in gangrene-producer strain 13 (not shown).

To confirm the role of the CpAL system in controlling C. perfringens biofilm formation, the

mobile group II intron that had been inserted in the sense orientation (i.e., relative to the direction of

transcription) was removed by trans-splicing from the intron-disrupted agrB gene. To remove the intron

the plasmid pCPJVagrB was reintroduced into CN3685ΔagrB to create CPJV 32. This strain, CPJV 32,

was then grown at 30°C to produce maximal expression of the temperature-sensitive LtrA protein which

is required for the splicing-induced intron removal (34, 55). Incubation at 30°C allowed strain CPJV 32

to produce a significant increase on the biomass, as quantified by fluorescence (Fig. 4C). CPJV 32

Biofilms were observed as a thick biofilm biomass on the bottom of the wells which cover most of the

substrate (Fig. 4D). These experiments confirmed the CpAL system regulates biofilm formation of C.

perfringens type C strain CN 3685.

Up-regulated expression of cpa and pfoA in C. perfringens biofilm cells.
The CpAL system controls mRNA levels of genes encoding important toxins such as *cpb*, *cpa* and *pfoA* in planktonic cultures of type A, type B, and type C strains (36, 42, 43). To evaluate biofilm levels of these transcripts, total RNA was extracted from CN 3685 biofilms and planktonic cultures and mRNA expression was assessed by qRT-PCR. Of evaluated genes, the *cbp* gene, encoding *C. perfringens* beta toxin (61), is maximally expressed in planktonic cultures during the early-mid log phase of growth (60). In comparison to planktonic cultures, the population of biofilm bacteria 4 h post-inoculation had down-regulated transcription of the *cbp* gene (~4 fold decrease) but mRNA levels of *cpa* and *pfoA* remained similarly expressed (Fig. 5A). Expression the *cpa* or *pfoA* transcript was ~9-fold or ~4-fold up-regulated in 24 h biofilms, respectively, relative to planktonic cells grown for 4 h (Fig. 5B) whereas *cpb* expression did not change. Moreover, mRNA levels of *cpa* or *pfoA* in biofilm cells increased ~17- or ~6-fold in comparison to planktonic cells 24 h post-inoculation as quantified by qRT-PCR (Fig. 5C) or detected by conventional RT-PCR (Fig. 5D).

Relative to 4 h biofilms, however, levels of *cpa* mRNA gradually increased 8, 16 or 24 h post-inoculation whereas *pfoA* mRNA levels were down-regulated in 8 h biofilms after which its expression remained the same (Fig. 6A). Levels of the *cpb* message did not change in biofilms incubated for 8, 16 or 24 h, in comparison to 4 h biofilms (Fig. 6A). These results indicate that mRNA levels of both *cpa* and *pfoA* are increased in biofilms compared to planktonic cells, at time points tested in this study, but only mRNA levels of *cpa* are increasingly being produced as the biofilm structure matures.

To further confirm the up-regulated transcription of *cpa* and *pfoA* in biofilm cells, mRNA levels of those genes were evaluated in total RNA extracted from the gangrene producer, type A, strain 13. As shown in Fig. 5D, mRNA expression for *cpa* and *pfoA* increased ~9 and ~6 fold in 24 h biofilm cells relative to planktonic cells, respectively. Similar expression of *cpa* and *pfoA* transcripts as that presented
342 above for strain CN 3685, were observed when 8, 16 or 24 h biofilms were compared to the expression in
343 biofilms produced 4 h post-inoculation (Fig. 6B).
344
345 Clostridium perfringens alpha toxin (CPA) and perfringolysin O (PFO) are required to produce
346 biofilms. The up-regulated production of toxin genes in biofilms and recent findings demonstrating that
347 the CpAL system regulates production of CPA, CPB and PFO (36, 42, 43) prompted us to directly
348 evaluate the role of these toxins in biofilm formation. To assess this, we inserted an intron separately
349 into genes *cpa*, *cpb*, or *pfoA*, encoding CPA, CPB or PFO, respectively (34, 52). Expected phenotypes
350 for the mutation were evaluated in each mutant strain; for example, a strain with an insertional-
351 inactivated *cpa* gene (CN3685Δ*cpa* or S13Δ*cpa*) was not able to hydrolyze phospholipids in egg yolk
352 agar palates (not shown) and did not produce CPA in the culture supernatant (Fig. 7A). A Hb release
353 assay confirmed strains CN3685Δ*pfoA* or S13Δ*pfoA* did not lyse red blood cells (not shown) whereby
354 a compact pellet of erythrocytes was observed after 2 h of incubation (Fig. 7A).

355 While the gangrene-causing, genome-sequenced, strain 13 made robust biofilms 24 h post-
356 inoculation (Fig. 7B and 7C), the Δ*plc* and Δ*pfoA* mutants produced significantly less biofilms than that
357 of their parents. Similar results were obtained for CN 3685 when biofilms made by mutant derivatives
358 Δ*plc* and Δ*pfoA* were compared to those produced by the parental strain. In contrast, CN3685Δ*cpb*
359 produced wt biofilm levels at 24 h post-inoculation (not shown).

360 Mutant S13Δ*plc* and S13Δ*pfoA* were each complemented with the *cpa* or *pfoA* wt gene,
361 respectively. As expected, the complemented strains regained the ability to produce CPA and hydrolyze
362 phospholipids or lyses red blood cells (Fig. 7A). Biofilms produced by complemented strains,

S13Δplc/plc or S13ΔpfoA/pfoA were restored to wt levels (Fig. 7B and 7C), confirming the role of these hemolysins in the formation of *C. perfringens* biofilms.

CPA is detected on *Clostridium perfringens* biofilms.

Given the increasing upregulated expression of *cpa* in biofilms, the presence of CPA in biofilms was investigated by confocal microscopy. Signal of CPA was detected in optical middle and top sections of biofilms with made by S13 and S13Δplc/plc. 24 h post-inoculation whereas CPA signal was absence in the few S13Δplc bacteria that remained attached to the substrate (Fig. 8). In S13 and S13Δplc/plc biofilms, CPA colocalized in specific areas with DNA staining suggesting that CPA may form a complex with eDNA in the biofilm matrix.

### Discussion

*Clostridium perfringens* infections have been associated to the production of potent and often lethal toxins (11, 64) yet the microenvironment where strains must produce toxins are different. We have demonstrated in this work that CPA, a toxin produced by all toxigenic types, and PFO, are necessary for *C. perfringens* type A strain 13 and type C strain CN 3685 to produce biofilms as their absence rendered strains unable to form wt levels of biofilms. This absence did not have an effect on growing rates of planktonic cells (not shown). The need of CPA and PFO production for biofilm formation correlated well with the up-regulated transcription of *cpa* and *pfoA* toxin genes observed in biofilms compared to planktonic cultures. The specific role that these toxins play in the building of the *C. perfringens* biofilm structure needs, however, further efforts. Perhaps, CPA help to stabilize the structure by forming complexes with other molecules within the biofilm matrix, e.g., eDNA.

CPA and PFO produce lysis of red blood cells of different species, whereas CPA has phospholipase activity and produces alpha hemolysis on blood agar plates. PFO is a potent pore-forming toxin that induces beta hemolysis of red blood cells (65). Another hemolysin produced by the Gram-positive *S. pneumoniae*, the pneumolysin Ply, was recently found to be essential for proper assemblage of early pneumococcal biofilms *in vitro* and in a life-like environment (66). PFO and Ply belong to the cholesterol-dependent cytolysins (CDCs) family sharing ~70% homology and ~60% identities. Ply is located on the pneumococcal cell wall (67) and appears to link pneumococcal cells within a growing biofilm structure (66). In contrast, PFO has a signal peptide for its secretion into the supernatant. It is unclear at this point why, or how, secreted toxins such as PFO and CPA may help in the building of *C. perfringens* biofilms. CPA may be incorporated into the matrix where it was seen partially colocalizing with eDNA in our confocal studies. Although not structurally related to CPA or PFO, recent discoveries showed that a hemolysin produced by *Staphylococcus aureus* forms a nucleoprotein complex with released DNA leading to a solid biofilm matrix required to build staphylococcal biofilms both *in vitro* and *in vivo* (68). Other exotoxins, such as alpha-toxin (also known as HLA) secreted by *S. aureus* strains have also been implicated in the formation of staphylococcal biofilms on mucosal tissues and virulence (69).

The composition of the *C. perfringens* biofilm matrix is under active investigation. Two different research groups recently showed that treatment of preformed biofilms with proteinase K, cellulose or DNase I dispersed *C. perfringens* biofilms suggesting the presence of proteins, β-1,4 linked polysaccharides and eDNA (49, 50). Our studies further demonstrated the presence of eDNA in the supernatant of biofilm cultures acting as a source of DNA to be incorporated into the matrix. Although *C. perfringens* strain CN3685, utilized in these studies, carries plasmids ranging in size from ~65 to ~110 kb (70), extracellular DNA identified corresponded to chromosomal DNA since our quantitative

408 assays targeted a chromosomally-encoded *cpa* gene (13). Chromosomal eDNA was, in part, provided by

409 lysis of a subpopulation of biofilm bacteria and planktonic cells as quantified and visualized by using the

410 live/dead cell assay. Since most *C. perfringens* strains encode virulence plasmids (70, 71), the possibility

411 exist that plasmids are also released as a source of eDNA for the matrix.

412 Regulation of *C. perfringens* biofilms appears to be a complex mechanism which was

413 demonstrate in this work to be mediated by the CpAL quorum sensing system but not by the LuxS/AI-2.

414 Whereas a mutation in the *luxS* gene did not have an effect on the biomass, our experiments showed that

415 CpAL was required for proper formation of the structure as the absence of the *agrB*, [encoding a

416 transmembrane protein required for processing the AgrD pheromone (36)], rendered strain CN 3685, or

417 strain 13, unable to produce mature biofilm structures. Instead, Δ*agrB* mutants grew mostly as

418 planktonic bacteria. CpAL regulation of biofilms might involve only a subset of CpAL-regulated factors

419 since the CPB toxin, which is regulated by this system, was not implicated in the formation of biofilms

420 by CN 3685. Although not addressed in this work, the CpAL system could activate biofilm-specific

421 downstream regulators. For example, it has been demonstrated that the two component regulatory

422 system VirR/S transcriptionally regulates toxin levels of CPA, PFO, and other virulence factors, in

423 planktonic cultures (72, 73). Whereas the VirR/S system is the logical downstream candidate as the VirS

424 transmembrane protein appear to be the receptor for the AgrD pheromone (37), other transcriptional

425 regulator called CtrAB have been recently described to indirectly (i.e., via an unknown factor) control

426 transcript levels of *cpa* and *pfoA* (74). A *ctrAB* null mutant, however, produced wt biofilm biomass

427 when incubated at 37°C but presented some effects on biofilms formed at 25°C (50). Our data showing

428 up-regulated transcription, in biofilm cells, of CpAL-regulated *cpa* and *pfoA* genes, but not *cbp*, further

429 reinforce our hypothesis that biofilms activates specific downstream signals in response to the CpAL-

430 released AgrD pheromone. Homologs quorum sensing systems have also been described in most

toxigenic clostridia (41), with most producing biofilms to survive in different hostile environments.

Whether these biofilms are controlled by Agr-like systems remains to be investigated.

An additional contribution on this work was the protocol we developed and validated to fluorescently-stain C. perfringens biofilms. The fluorescence-based protocol was compared against the classic crystal violet procedure that, although helpful, was not as sensitive as fluorescence staining of biofilms which allowed us to detect statistically significant differences of the biofilm biomass on each time point tested. The fluorescence protocol has the additional advantage that biofilms can be stained and quantified within minutes after which fluorescence micrographs can be obtained from the same preparations. As growing evidences that other clostridia may produce biofilms in different environments, having a simplified and more sensitive protocol will help to improve our knowledge about biofilms produced by other anaerobes.

In summary, we hypothesize that proliferation of C. perfringens strains in the site of the infection may result in activation of the CpAL system which regulates production of components of the biofilm matrix. The biofilm structure may confer resistance to innate immune molecules and antibiotics while at parallel may induce degradation of epithelial tissue by production of toxins. In gangrene cases produced by type A strains, for example, the border between healthy and necrotic tissue often advances several inches per hour, despite appropriate antibiotic therapy. Therefore, studies of the C. perfringens biofilm structure seem necessary to further understand diseases and ultimately help improve prognosis.

Acknowledgements

The authors are grateful to Dr. Tohru Shimizu and Dr. Kaori Othani from Kanazawa University, Japan, for their gift of strain 13, Dr. Julian I. Rood from Monash University, Victoria Australia, for providing strain CN3685 and Dr. Glenn Songer from Iowa State University for providing C. perfringens strains

JGS 1075, JGS 1076, JGS 1495, and JGS 1659. We acknowledge contributions of Shazaib Jiwani from Oglethorpe University on some laboratory procedures. We also thank Dr. June R. Scott from Emory University for her thoughtful suggestions on our manuscript.

### References

1. Fosse J, Laroche M, Oudot N, Seegers H, Magras C. On-farm multi-contamination of pigs by food-borne bacterial zoonotic hazards: An exploratory study. Vet Microbiol.
2. McClane BA, F.A. Uzal, M. Fernandez-Miyakawa, D. Lyerly and T.D. Wilkins. 2004. The Enterotoxigenic Clostridia, p. 698-752. In M. Dworkin SF, E. Rosenberg, K.F. Schleifer, and E. Stackebrandt (ed.), The Prokaryotes, vol. 4. Springer-Verlag, New York, NY.
3. Rood JI, Cole ST. 1991. Molecular genetics and pathogenesis of *Clostridium perfringens*. Microbiol Rev 55:621-648.
4. McClane BA. 2001. The complex interactions between *Clostridium perfringens* enterotoxin and epithelial tight junctions. Toxicon 39:1781-1791.
5. Songer JG. 1996. Clostridial enteric diseases of domestic animals. Clin Microbiol Rev 9:216-234.
6. Songer JG. Clostridia as agents of zoonotic disease. Vet Microbiol 140:399-404.
7. Hirn M, Niinikoski J, Lehtonen OP. 1993. Effect of hyperbaric oxygen and surgery on experimental multimicrobial gas gangrene. European surgical research. Europaische chirurgische Forschung 25:265-269.
8. Finsterer J, Hess B. 2007. Neuromuscular and central nervous system manifestations of *Clostridium perfringens* infections. Infection 35:396-405.

9. Bryant AE. 2003. Biology and pathogenesis of thrombosis and procoagulant activity in invasive infections caused by group A streptococci and Clostridium perfringens. Clin Microbiol Rev 16:451-462.

10. Uzal FA, J. E. Vidal, B. A. McClane and A. A. Gurjar. 2010. Clostridium perfringens Toxins Involved in Mammalian Veterinary Diseases. The open toxinology journal 3 DOI: 10.2174/1875414701003010024:24-42.

11. Smedley JG, 3rd, Fisher DJ, Sayeed S, Chakrabarti G, McClane BA. 2004. The enteric toxins of Clostridium perfringens. Rev Physiol Biochem Pharmacol 152:183-204.

12. Popoff MR, Bouvet P. 2009. Clostridial toxins. Future microbiology 4:1021-1064.

13. Rood JI. 1998. Virulence genes of Clostridium perfringens. Annu Rev Microbiol 52:333-360.

14. Lanting B, Athwal GS, Naudie DD. 2007. Spontaneous Clostridium perfringens myonecrosis of the shoulder: a case report. Clinical orthopaedics and related research 461:20-24.

15. Schropfer E, Rauthe S, Meyer T. 2008. Diagnosis and misdiagnosis of necrotizing soft tissue infections: three case reports. Cases journal 1:252.

16. Kuroda S, Okada Y, Mita M, Okamoto Y, Kato H, Ueyama S, Fujii I, Morita S, Yoshida Y. 2005. Fulminant massive gas gangrene caused by Clostridium perfringens. Internal medicine (Tokyo, Japan) 44:499-502.

17. Stevens DL, Tweten RK, Awad MM, Rood JI, Bryant AE. 1997. Clostridial gas gangrene: evidence that alpha and theta toxins differentially modulate the immune response and induce acute tissue necrosis. J Infect Dis 176:189-195.

18. Bryant AE, Bergstrom R, Zimmerman GA, Salyer JL, Hill HR, Tweten RK, Sato H, Stevens DL. 1993. Clostridium perfringens invasiveness is enhanced by effects of theta toxin

497 upon PMNL structure and function: the roles of leukocytotoxicity and expression of CD11/CD18
498 adherence glycoprotein. FEMS Immunol Med Microbiol 7:321-336.
499 19. Elle mor DM, Baird RN, Awad MM, Boyd RL, Rood JI, Emmins JJ. 1999. Use of genetically
500 manipulated strains of Clostridium perfringens reveals that both alpha-toxin and theta-toxin are
501 required for vascular leukostasis to occur in experimental gas gangrene. Infect Immun 67:4902-
502 4907.
503 20. Kennedy CL, Krejany EO, Young LF, O'Connor JR, Awad MM, Boyd RL, Emmins JJ,
504 Lyras D, Rood JI. 2005. The alpha-toxin of Clostridium septicum is essential for virulence. Mol
505 Microbiol 57:1357-1366.
506 21. Kennedy CL, Lyras D, Cheung JK, Hiscox TJ, Emmins JJ, Rood JI. 2009. Cross-
507 complementation of Clostridium perfringens PLC and Clostridium septicum alpha-toxin mutants
508 reveals PLC is sufficient to mediate gas gangrene. Microbes and infection / Institut Pasteur
509 11:413-418.
510 22. Hickey MJ, Kwan RY, Awad MM, Kennedy CL, Young LF, Hall P, Cordner LM, Lyras D,
511 Emmins JJ, Rood JI. 2008. Molecular and cellular basis of microvascular perfusion deficits
512 induced by Clostridium perfringens and Clostridium septicum. PLoS Pathog 4:e1000045.
513 23. Awad MM, Bryant AE, Stevens DL, Rood JI. 1995. Virulence studies on chromosomal alpha-
514 toxin and theta-toxin mutants constructed by allelic exchange provide genetic evidence for the
515 essential role of alpha-toxin in Clostridium perfringens-mediated gas gangrene. Mol Microbiol
516 15:191-202.
517 24. Awad MM, Elle mor DM, Boyd RL, Emmins JJ, Rood JI. 2001. Synergistic effects of alpha-
518 toxin and perfringolysin O in Clostridium perfringens-mediated gas gangrene. Infect Immun
519 69:7904-7910.

520 25. Gui L, Subramony C, Fratkin J, Hughson MD. 2002. Fatal enteritis necroticans (pigbel) in a
diabetic adult. Mod Pathol 15:66-70.

521 26. Tonnellier M, Maury E, Guglielminotti J, Offenstadt G. 2001. A fatal sandwich. Lancet

522 Infect Dis 1:202.

523 27. Severin WP, de la Fuente AA, Stringer MF. 1984. Clostridium perfringens type C causing

524 necrotising enteritis. J Clin Pathol 37:942-944.

525 28. Petrillo TM, Beck-Sague CM, Songer JG, Abramowsky C, Fortenberry JD, Meacham L,

526 Dean AG, Lee H, Bueschel DM, Nesheim SR. 2000. Enteritis necroticans (pigbel) in a diabetic

527 child. N Engl J Med 342:1250-1253.

528 29. Matsuda T, Okada Y, Inagi E, Tanabe Y, Shimizu Y, Nagashima K, Sakurai J, Nagahama

529 M, Tanaka S. 2007. Enteritis necroticans 'pigbel' in a Japanese diabetic adult. Pathol Int 57:622-

530 626.

531 30. Farrant JM, Traill Z, Conlon C, Warren B, Mortensen N, Gleeson FV, Jewell DP. 1996.

532 Pigbel-like syndrome in a vegetarian in Oxford. Gut 39:336-337.

533 31. Clarke LE, Diekmann-Guiroy B, McNamee W, Java DJ, Jr., Weiss SM. 1994. Enteritis

534 necroticans with midgut necrosis caused by Clostridium perfringens. Arch Surg 129:557-560.

535 32. Sakurai J, Duncan CL. 1978. Some properties of beta-toxin produced by Clostridium

536 perfringens type C. Infect Immun 21:678-680.

537 33. Vidal JE, McClane BA, Saputo J, Parker J, Uzal FA. 2008. Effects of Clostridium

538 perfringens beta-toxin on the rabbit small intestine and colon. Infect Immun 76:4396-4404.

539 34. Sayeed S, Uzal FA, Fisher DJ, Saputo J, Vidal JE, Chen Y, Gupta P, Rood JI, McClane

540 BA. 2008. Beta toxin is essential for the intestinal virulence of Clostridium perfringens type C

541 disease isolate CN 3685 in a rabbit ileal loop model. Mol Microbiol 67:15-30.

543 35. Shojadoost B, Vince AR, Prescott JF. 2012. The successful experimental induction of necrotic
enteritis in chickens by Clostridium perfringens: a critical review. Veterinary research 43:74.
544
545 36. Vidal JE, Chen J, Li J, McClane BA. 2009. Use of an EZ-Tn5-based random mutagenesis
546 system to identify a novel toxin regulatory locus in Clostridium perfringens strain 13. PLoS One
547 4:e6232.
548 37. Ohtani K, Yuan Y, Hassan S, Wang R, Wang Y, Shimizu T. 2009. Virulence gene regulation
549 by the agr system in Clostridium perfringens. J Bacteriol 191:3919-3927.
550 38. Novick RP, Projan SJ, Kornblum J, Ross HF, Ji G, Kreiswirth B, Vandenesch F,
551 Moghazeh S. 1995. The agrP2 operon: an autocatalytic sensory transduction system in
552 Staphylococcus aureus. Mol Gen Genet 248:446-458.
553 39. Ji G, Beavis RC, Novick RP. 1995. Cell density control of staphylococcal virulence mediated
554 by an octapeptide pheromone. Proc Natl Acad Sci USA 92:12055-12059.
555 40. Wuster A, Babu MM. 2008. Conservation and evolutionary dynamics of the agr cell-to-cell
556 communication system across firmicutes. J Bacteriol 190:743-746.
557 41. Cooksley CM, Davis IJ, Winzer K, Chan WC, Peck MW, Minton NP. 2010. Regulation of
558 neurotoxin production and sporulation by a putative agrBD signaling system in proteolytic
559 Clostridium botulinum. Appl Environ Microbiol 76:4448-4460.
560 42. Vidal JE, Ma M, Saputo J, Garcia J, Uzal FA, McClane BA. 2012. Evidence that the Agr-like
561 quorum sensing system regulates the toxin production, cytotoxicity and pathogenicity of
562 Clostridium perfringens type C isolate CN 3685. Molecular microbiology 83:179-194.
563 43. Chen J, McClane BA. 2012. Role of the Agr-like quorum-sensing system in regulating toxin
564 production by Clostridium perfringens type B strains CN 1793 and CN 1795. Infect Immun
565 80:3008-3017.

566 44. Chen J, Rood JI, McClane BA. 2011. Epsilon-toxin production by Clostridium perfringens  
567 type D strain CN3718 is dependent upon the agr operon but not the VirS/VirR two-component  
568 regulatory system. mBio 2.  
569 45. Ohtani K, Hayashi H, Shimizu T. 2002. The luxS gene is involved in cell-cell signalling for  
570 toxin production in Clostridium perfringens. Mol Microbiol 44:171-179.  
571 46. Vidal JE, Ohtani K, Shimizu T, McClane BA. 2009. Contact with enterocyte-like Caco-2 cells  
572 induces rapid upregulation of toxin production by Clostridium perfringens type C isolates. Cell  
573 Microbiol 11:1306-1328.  
574 47. Flemming HC, Wingender J. 2010. The biofilm matrix. Nature reviews 8:623-633.  
575 48. Varga JJ, Therit B, Melville SB. 2008. Type IV pili and the CcpA protein are needed for  
576 maximal biofilm formation by the gram-positive anaerobic pathogen Clostridium perfringens.  
577 Infect Immun 76:4944-4951.  
578 49. Charlebois A, Jacques M, Archambault M. 2014. Biofilm formation of Clostridium  
579 perfringens and its exposure to low-dose antimicrobials. Frontiers in microbiology 5:183.  
580 50. Obana N, Nakamura K, Nomura N. 2014. A sporulation factor is involved in the  
581 morphological change of Clostridium perfringens biofilms in response to temperature. J  
582 Bacteriol 196:1540-1550.  
583 51. Vidal JE, Ludewick HP, Kunkel RM, Zahner D, Klugman KP. 2011. The LuxS-dependent  
584 quorum-sensing system regulates early biofilm formation by Streptococcus pneumoniae strain  
585 D39. Infect Immun 79:4050-4060.  
586 52. Chen Y, McClane BA, Fisher DJ, Rood JI, Gupta P. 2005. Construction of an alpha toxin  
587 gene knockout mutant of Clostridium perfringens type A by use of a mobile group II intron.  
588 Appl Environ Microbiol 71:7542-7547.

589 53. Li J, Chen J, Vidal JE, McClane BA. 2011. The Agr-like quorum-sensing system regulates sporulation and production of enterotoxin and beta 2 toxin by Clostridium perfringens type A

590 non-food-borne human gastrointestinal disease strain F 5603. Infect Immun 79:2451-2459.

591 Bannam TL, Rood JI. 1993. Clostridium perfringens-Escherichia coli shuttle vectors that carry single antibiotic resistance determinants. Plasmid 29:233-235.

592 54. Yao J, Zhong J, Fang Y, Geisinger E, Novick RP, Lambowitz AM. 2006. Use of targetrons to disrupt essential and nonessential genes in Staphylococcus aureus reveals temperature sensitivity

593 of Ll.LtrB group II intron splicing. RNA 12:1271-1281.

594 55. Livak KJ, Schmittgen TD. 2001. Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method. Methods 25:402-408.

595 56. Ethapa T, Leuzzi R, Ng YK, Baban ST, Adamo R, Kuehne SA, Scarselli M, Minton NP, Serruto D, Unnikrishnan M. 2013. Multiple factors modulate biofilm formation by the anaerobic pathogen Clostridium difficile. J Bacteriol 195:545-555.

596 57. Yarwood JM, Bartels DJ, Volper EM, Greenberg EP. 2004. Quorum sensing in Staphylococcus aureus biofilms. J Bacteriol 186:1838-1850.

597 58. Fernandez-Miyakawa ME, Marcellino R, Uzal FA. 2007. Clostridium perfringens type A toxin production in 3 commonly used culture media. J Vet Diagn Invest 19:184-186.

598 59. Fisher DJ, Fernandez-Miyakawa ME, Sayeed S, Poon R, Adams V, Rood JI, Uzal FA, McClane BA. 2006. Dissecting the contributions of Clostridium perfringens type C toxins to lethality in the mouse intravenous injection model. Infect Immun 74:5200-5210.

599 60. Vidal JE, Howery KE, Ludewick HP, Nava P, Klugman KP. 2013. Quorum-sensing systems LuxS/autoinducer 2 and Com regulate Streptococcus pneumoniae biofilms in a bioreactor with living cultures of human respiratory cells. Infect Immun 81:1341-1353.

612 62. Moscoso M, Garcia E, Lopez R. 2006. Biofilm formation by Streptococcus pneumoniae: role
of choline, extracellular DNA, and capsular polysaccharide in microbial accretion. Journal of
bacteriology 188:7785-7795.
613
614
615 63. Thomas VC, Thurlow LR, Boyle D, Hancock LE. 2008. Regulation of autolysis-dependent
extracellular DNA release by Enterococcus faecalis extracellular proteases influences biofilm
development. Journal of bacteriology 190:5690-5698.
616
617
618 64. Petit L, Gibert M, Popoff MR. 1999. Clostridium perfringens: toxin type and genotype.
Trends Microbiol 7:104-110.
619
620 65. Uzal FA, Freedman JC, Shrestha A, Theoret JR, Garcia J, Awad MM, Adams V, Moore
RJ, Rood JI, McClane BA. 2014. Towards an understanding of the role of Clostridium
perfringens toxins in human and animal disease. Future microbiology 9:361-377.
621
622
623 66. Shak JR, Ludewick HP, Howery KE, Sakai F, Yi H, Harvey RM, Paton JC, Klugman KP,
Vidal JE. 2013. Novel role for the Streptococcus pneumoniae toxin pneumolysin in the
assembly of biofilms. mBio 4:e00655-00613.
624
625
626 67. Price KE, Camilli A. 2009. Pneumolysin localizes to the cell wall of Streptococcus
pneumoniae. J Bacteriol 191:2163-2168.
627
628 68. Huseby MJ, Kruse AC, Digre J, Kohler PL, Vocke JA, Mann EE, Bayles KW, Bohach GA,
629 Schlievert PM, Ohlendorf DH, Earhart CA. 2010. Beta toxin catalyzes formation of
630 nucleoprotein matrix in staphylococcal biofilms. Proc Natl Acad Sci USA 107:14407-14412.
631 69. Anderson MJ, Lin YC, Gillman AN, Parks PJ, Schlievert PM, Peterson ML. 2012. Alpha-
632 toxin promotes Staphylococcus aureus mucosal biofilm formation. Frontiers in cellular and
633 infection microbiology 2:64.

634 70. Gurjar A, Li J, McClane BA. 2010. Characterization of toxin plasmids in Clostridium
perfringens type C isolates. Infect Immun 78:4860-4869.
635 71. Sayeed S, Li J, McClane BA. 2007. Virulence plasmid diversity in Clostridium perfringens
type D isolates. Infect Immun 75:2391-2398.
636 72. Ba-Thein W, Lyristis M, Ohtani K, Nisbet IT, Hayashi H, Rood JI, Shimizu T. 1996. The
virR/virS locus regulates the transcription of genes encoding extracellular toxin production in
Clostridium perfringens. J Bacteriol 178:2514-2520.
637 73. Lyristis M, Bryant AE, Sloan J, Awad MM, Nisbet IT, Stevens DL, Rood JI. 1994.
Identification and molecular analysis of a locus that regulates extracellular toxin production in
Clostridium perfringens. Mol Microbiol 12:761-777.
638 74. Obana N, Nakamura K. 2011. A novel toxin regulator, the CPE 1446-CPE 1447 protein
heteromeric complex, controls toxin genes in Clostridium perfringens. J Bacteriol 193:4417-
4424.
639 75. Shimizu T, Ohtani K, Hirakawa H, Ohshima K, Yamashita A, Shiba T, Ogasawara N,
Hattori M, Kuhara S, Hayashi H. 2002. Complete genome sequence of Clostridium
perfringens, an anaerobic flesh-eater. Proc Natl Acad Sci USA 99:996-1001.
640 76. Sawires YS, Songer JG. 2006. Clostridium perfringens: insight into virulence evolution and
population structure. Anaerobe 12:23-43.
641 77. Meer RR, Songer JG. 1997. Multiplex polymerase chain reaction assay for genotyping
Clostridium perfringens. American journal of veterinary research 58:702-705.
642 
643 Figure legends

Fig. 1. *C. perfringens* biofilms are differentially produced depending on the bacterial culture medium. (A) Type C stain CN3685, or JGS1495, was inoculated into either FTG or TGY and biofilms were incubated for 24 h. A set of wells containing biofilms were washed, photographed (bottom panel) and then crystal violet absorbance was obtained. Asterisks indicate statistically significance, \( p < 0.05 \), evaluated by the Mann Whitney U test. Another set of wells was stained with DAPI and (B) fluorescence micrographs were obtained. Experiments were repeated at least three times. (C) *C. perfringens* strains were inoculated in TGY medium and incubated for 24 h. Biofilms were stained with crystal violet and A630 obtained. Arbitrary units obtained from CN3685 were set to 100% biofilms and those made by the other strains were calculated. Error bars represent the standard error of the mean calculated using data from, at least, three independent experiments.

Fig. 2. Fluorescence-based method to quantify *C. perfringens* biofilms. Strain CN3685 was inoculated in 96-well plates containing THY and incubated at \( 37^\circ \mathrm{C} \) for the indicated time. Biofilms were washed with PBS and then (A) stained with crystal violet which absorbance (A630) was obtained or (B) fluorescently stained with SYTO 9 and fluorescence arbitrary units were obtained with a fluorometer. Biofilm biomass (%) was calculated in A and B by setting 100% biomass units obtained for the 8 h time point to calculate biomass (%) of the other two. (*) indicates statistically significance, \( p < 0.05 \), evaluated by the Mann Whitney U test. Error bars represent the standard error of the mean calculated using data from three independent experiments. Biofilms stained in B were photographed (C) using an inverted microscope and either the EVOS® lightcube GFP 470/510 (SYTO 9) or phase contrast. Bar at bottom right is valid for all panels.

Fig. 3. Extracellular DNA is released, by bacterial lysis, into the supernatant of growing C. perfringens biofilms. Strain CN3685 was inoculated in 24-well plates containing TGY and incubated at 37°C for the indicated time after which 1 ml of supernatant was removed. (A) Extracellular DNA was purified and quantified utilizing qPCR reactions, eDNA (pg/ml). In another set of experiments, 4 or 8 h biofilms (B), or planktonic cells (C) were stained using the LIVE/DEAD BacLight kit and fluorescence arbitrary units were obtained using a fluorometer. (*) Statistically significance, p<0.05 evaluated by the Mann Whitney U test, in comparison with the Live bacteria. Biofilms (D) or 8 h planktonic cultures (E) were also photographed with an Evos inverted microscope using the light cube GFP 470/510 (Live) or RFP 531/593 (Dead). Bar at upper right is valid for all panels.

Fig. 4. CpAL control of C. perfringens biofilms. Strain CN3685, or mutant derivatives, were inoculated in 96-well plates containing THY and biofilms were incubated for 3, 6, or 24 h at 37°C. (A) Biofilms (24 h) were stained with SYTO 9 and their biomass was obtained using a fluorometer. (B) Micrographs of fluorescence-stained biofilms 3 or 6 h post-inoculation were obtained using an Evos inverted microscope. In another set of experiments, CN3685 or CPJV21 (ΔagrB) was incubated at 37°C, whereas CN3685 and CPJV32 (ΔagrB/agrB) was incubated at 30°C. (C) SYTO 9-stained biofilms were quantified by fluorescence. (*) indicates statistically significance, p<0.05, evaluated by the Mann Whitney U test. In (D) their macroscopic aspect was photographed and micrographs were also obtained with a fluorescence microscope. Where shown error bars represent the standard error of the mean calculated using data from three independent experiments.

Fig. 5. Upregulated transcription of *cpa* and *pfoA*, but not *cpb*, in C. perfringens biofilms vs planktonic cells. C. perfringens type C CN3685 or type A strain 13, was inoculated in TGY and incubated at 37°C for 4 or 24 h. Planktonic cells and biofilms were harvested, RNA extracted and used

as template in qRT-PCR reactions targeting the *cpb*, *cpa* or *pfoA* gene. Average $C_T$ values were normalized to the *polC* gene and the fold differences were calculated using the comparative $C_T$ method ($2^{-\Delta\Delta C_T}$). Graphs show fold changes in mRNA expression of (A) 4 h biofilms relative to planktonic cells, (B) 24 h biofilms relative to 4 h planktonic cells, (C and E) 24 h biofilms relative to 24 h planktonic cells. (D) RNA from 24 h biofilms (Bio) or planktonic (Plank) was utilized in conventional RT-PCR reactions targeting the indicated gene. Error bars represent the standard error of the mean calculated using data from, at least, three independent experiments.

Fig. 6. Upregulated transcription of *cpa* in *C. perfringens* biofilms. (A) *C. perfringens* type C CN3685 or (B) type A strain 13, was inoculated in TGY and incubated at 37°C for 4 or 24 h. Biofilms were harvested, RNA extracted and used as template in qRT-PCR reactions targeting the *cpb*, *cpa* or *pfoA* gene. Average $C_T$ values were normalized to the *polC* gene and the fold differences were calculated using the comparative $C_T$ method ($2^{-\Delta\Delta C_T}$). Panels show fold changes in mRNA expression relative to biofilms 4 h post-inoculation compared to 8, 16 or 24 h biofilms. Error bars represent the standard error of the mean calculated using data from, at least, four independent experiments.

Fig. 7. CPA and PFO required for *C. perfringens* biofilms. (A) Phenotype of strain 13 (WT), S13Δ*cpa* or S13Δ*pfoA* mutants and corresponding complemented strain. Left panel shows lysis of erythrocytes (note an intact erythrocytes pellet when supernatants from Δ*pfoA* were used) or right panel CPA production assessed by dot blot using anti-CPA antibodies. (B and C) Strain 13 (WT), or derivatives, was inoculated in 96-well plates containing TGY and biofilms were incubated for 24 h at 37°C. Biofilms were stained with SYTO 9 and (B) micrographs of fluorescence-stained biofilms were obtained using an Evos inverted microscope or (C) their biomass (arbitrary units) was obtained using a

fluorometer. Asterisks indicate statistically significance, p<0.05, in comparison to WT as evaluated by  
the Mann Whitney U test. Error bars represent the standard error of the mean calculated using data from,  
at least, four independent experiments.  

Fig. 8. Detection of CPA on *C. perfringens* biofilms. Strain 13 (S13) S13Δplc, or S13Δplc/plc was  
inoculated in a 4-well chamber slide containing TGY and incubated for 24 h at 37°C. Bacteria were  
stained with SYTO 9 and CPA detected using rabbit polyclonal anti-*Clostridium perfringens* alpha toxin  
antibodies followed by goat anti-rabbit secondary antibodies conjugated to Alexa Fluor 555. Optical  
middle or top sections were obtained with a confocal microscope. Arrows point out areas of  
colocalization.

| Strain       | Description                                                                 | Reference or source |
|--------------|---------------------------------------------------------------------------|---------------------|
| CN3685       | type C strain isolated from peritoneal fluid of a sheep with struck           | (33, 34)            |
| JGS 1495     | Type C strain isolated from a Piglet with necrotic enteritis                 | (60)                |
| Strain 13    | Type A, gangrene strain                                                   | (75)                |
| JGS 1075     | Type C strain isolated from porcine enteritis                              | (76)                |
| JGS 1076     | Type C strain isolated from porcine enteritis                              | (76)                |
| JGS 1659     | Type C strain isolated from porcine enteritis                              | (76)                |
| CPJV 20      | CN3685ΔluxS                                                               | This study          |
| CPJV 21      | CN3685ΔagrB                                                               | This study          |
| CPJV 22      | CN3685Δplc                                                                | This study          |
| CPJV 23      | CN3685ΔpfoA                                                               | This study          |
| CPJV 25      | CN3685Δcpb                                                                | This study          |
| CPJV 26      | Strain13ΔagrB                                                             | This study          |
| CPJV 27      | Strain13Δplc                                                              | This study          |
| CPJV 28      | Strain13ΔpfoA                                                             | This study          |
| CPJV 30      | CPJV 27 encoding pCP13plc                                                  | This study          |
| CPJV 31      | CPJV 28 encoding pCP13pfoA                                                 | This study          |
| CPJV 32      | CPJV 21 encoding pCPJVagrB                                                 | This study          |
| Plasmids     |                                                                           |                     |
| pJIR 750     | E.coli-C.perfringens shuttle vector encoding a chloramphenicol resistant gene, catP | (54)                |
| pJIR 751     | E.coli-C.perfringens shuttle vector encoding an erythromycin resistant gene, ermB | (54)                |
| pJIR750ai    | pJIR 750 with plc targeted intron in antisense orientation                  | (52)                |
| pCPJVlux     | pJIR750ai with luxS targeted intron in antisense orientation               | This study          |
| pCPJVagrB    | pJIR750ai with agrB targeted intron in sense orientation                   | This study          |
| pCPJVpfoA    | pJIR750ai with pfoA targeted intron in antisense orientation               | This study          |
| pCPJVcpb     | pJIR750ai with cpb targeted intron in antisense orientation                | This study          |
| pCP13plc     | pJIR 751 encoding the plc gene from strain 13                              | This study          |
| pCP13pfoA    | pJIR 750 encoding the pfoA gene from strain 13                            | This study          |

| Primer name | Sequence (5' to 3') | Source or reference |
|-------------|---------------------|---------------------|
| cpb-Fwd     | GCGAATATGCTGAATCATCTA | (77)               |
| cpb-Rev     | GCAGGAACATTAGTATATCTTC |                   |
| JEV135L     | TAAAAAACGAAAAGAAGAAAAA | This study         |
| JEV126R     | TATGTAGGTTAGAGTCATACA |                   |
| JEV165L     | TTGAATCCCCCAGTTATTCACGATTAAG | This study       |
| JEV166R     | TTGGATCCTTAATTGTAAGTAATACTAGATCCA |             |
| JEV167L     | TTGAATTCAGGTTAAAACCTGTTTTTGATGA | This study      |
| JEV168R     | TTTCTAGATTTTGTAATACCACCAAAACCAAT |               |
| JEV181L     | ACCTCCACTTATGGTTTCAAATGT | This study        |
| JEV182R     | TTGTTATGTTCTTGTGCATCTCCT |                   |
| JEV193L     | CTAGCATGAGTCATAGTTGGGATG | This study        |
| JEV194R     | ACATGTAGTCATCTGTTCCAGCAT |                   |
| pol CJVL    | AATATATGATACTGAAGAGAGAGTAA | (46)             |
| pol CJVR    | TCTAAATTATCTAAATCTATGTCTACT |                 |
| luxS-L      | CCTAAAGGAGATATAGTTTCAAAG | (46)             |
| luxS-R      | TCACTAGAGTTAGCTAAGAGTCAT |                   |
| KO-IBS*     | AAAAAAAGCTTATAATTATCCTTAATTGCCATACTTG<br>TGCGCCCAGATAGGGTG | (34)           |
| KO-EBS1d    | CAGATTGTACAAATGTGGTGATAACAGATAAGTCA<br>TACTTGCTAACTTACCTTTCTTTGT |               |
| KO-EBS2     | TGAAACGCAAGTTTCTAATTTCGATTGCAATT CGATA<br>GAGGAAAGTG TCT |                   |
| IBS-pfoA**  | AAAAAAAGCTTATAATTATCCTTAATTGCCATACTTG<br>TGCGCCCAGATAGGGTG | (52)           |
| EBS1d-pfoA  | CAGATTGTACAAATGTGGTGATAACAGATAAGTCA<br>TACTTGCTAACTTACCTTTCTTTGT |               |

| EBS 2-pfoA | TGAACGCAAGTTTCTAATTT CGATTGCAATTTCGATAGAGGAAAGTG TCT |
| --- | --- |
| 566/567s-IBS# | AAAAAAGCTTATAATTATCCTTATTCCACACTG AAGTGCGCC-CAGATAGGGGTG | (53) |
| 566/567s-EBS1d | CAGATTGTACAAAATGTGGTGATAACAGATAA AGTCACTGAATATAACTTACCTTTCTTTGT |
| 566/567s-EBS 2 | TGAACGCAAGTTTCTAATTT CGGGTTTGGAAATCGATAGAGGAAAGTG TCT |
| luxS-IBS### | AAAAAAGCTTATAATTATCCTTATACTCCAAAG CTGTGCGCCCAGATAGGGGTG | (46) |
| luxS-EBS1d | CAGATTGTACAAAATGTGGTGATAACAGATAA AGTCAAAGCTTCTAACTTACCTTTCTTTGT |
| luxS-EBS 2 | TGAACGCAAGTTTCTAATTT CGGGTTTGAGTATCGATAGAGGAAAGTG TCT |

Primers to:*prepare acpb mutant, **prepare apfoA mutant, #prepare anagrB mutant, ##prepare aluxS
mutant.

Fig. 1. Vidal, et al

A
CV A630
| FTG | TGY |
| --- | --- |
| * | H |

B
FTG
TG Y

C
% Biofilm formation
| CN3685 | JGS1075 | JGS1076 | JGS1659 |
| --- | --- | --- | --- |
|  |  |  |  |

CN3685
JGS1495

Fig. 2. Vidal et al

A

| % Biofilm biomass |
|-------------------|
| 100              |
| 80               |
| 60               |
| 40               |
| 20               |
| 0                |

Time (h)
4   6   8

B

| % Biofilm biomass |
|-------------------|
| 100              |
| 80               |
| 60               |
| 40               |
| 20               |
| 0                |

Time (h)
4   6   8

C

SYTO9          Phase contrast
4 h
![Image](image1.png) ![Image](image2.png)

6 h
![Image](image3.png) ![Image](image4.png)

8 h
![Image](image5.png) ![Image](image6.png)
50 μm

Fig. 3, Vidal et al.

A

| eDNA (pg/mL) |
| --- |
| 1.2x10⁵ |
| 10⁵ |
| 8.0x10⁴ |
| 6.0x10⁴ |
| 4.0x10⁴ |
| 2.0x10⁴ |

| Time (h) |
| --- |
| 2 |
| 4 |
| 6 |
| 8 |

B

| RFU (arbitrary units) |
| --- |
| 2500 |
| 2000 |
| 1500 |
| 1000 |
| 500 |

| Live | Dead |
| --- | --- |
| 4 |  |
| 8 |  |

C

| RFU (arbitrary units) |
| --- |
| 3000 |
| 2500 |
| 2000 |
| 1500 |
| 1000 |
| 500 |

| Live | Dead |
| --- | --- |
| S13 |  |
| CN3685 |  |

D

| Live | Dead |
| --- | --- |
| 50 µm |  |
| 4 h |  |
| 8 h |  |

Biofilms

E

| CN3586 |
| --- |
| 50 µm |  |

| S13 |  |
| --- | --- |

Planktonic cells

Fig. 4. Vidal et al,

A

| % Biofilm formation |
| --- |
| 100 |
| 80 |
| 60 |
| 40 |
| 20 |
| CN3685 |
| ΔagrB |
| ΔluxS |

B

| WT 3 h | ΔluxS 6 h |
| --- | --- |
| WT 6 h | ΔagrB 6 h |

C

| RFU (arbitrary units) |
| --- |
| 40000 |
| 30000 |
| 20000 |
| 10000 |
| CN3685 |
| ΔagrB |
| ΔagrB/ΔagrB |

D

| WT | ΔagrB | WT | ΔagrB |
| --- | --- | --- | --- |
| 37°C |  |  |  |
| 30°C | ΔagrB/ΔagrB | WT | ΔagrB/ΔagrB |

Fig. 5. Vidal et al,

A

Fold change in mRNA expression  
relative to planktonic  

B

Fold change in mRNA expression  
relative to planktonic cells  

C

Fold change in mRNA expression  
relative to planktonic  

D

Bio    Plank   Bio    Plank   Bio    Plank   Bio    Plank  
polC   pfoA    cpa    cpb  

E

Fold change in mRNA expression  
relative to planktonic  

S13  

CN3685

Fig. 6. Vidal et al,

A

| Relative fold change in |
| mRNA expression         |

| cpa | 25 |
| --- | --- |
|     | 20 |
|     | 15 |
|     | 10 |
|     | 5  |
|     |    |
|     | 8  | 16 | 24 |
| Time (h)               |

B

| Relative fold change in |
| cpa mRNA                |

| 25 |
| --- |
| 20 |
| 15 |
| 10 |
| 5  |
|    |
| 8  | 16 | 24 |
| Time (h)               |

| Relative fold change in |
| mRNA expression         |

| pfoA | 5  |
| ---- | --- |
|      | 0  |
|      | -5 |
|      | -10|
|      |    |
|      | 8  | 16 | 24 |
| Time (h)               |

| Relative fold change in |
| pfoA mRNA              |

| 2  |
| --- |
| 0  |
| -2 |
| -4 |
| -6 |
| -8 |
|    |
| 8  | 16 | 24 |
| Time (h)               |

| Relative fold change in |
| mRNA expression         |

| cpb | 6  |
| ---- | --- |
|      | 4  |
|      | 2  |
|      | 0  |
|      | -2 |
|      |    |
|      | 8  | 16 | 24 |
| Time (h)               |

Fig. 7. Vidal et al.,

A  
WT | ΔpfoA | WT | Δplc/plc | Δplc  

B  
WT | Δplc | ΔpfoA | Δplc/plc | ΔpfoA/pfoA  

C  
% Biofilm formation  
WT | Δplc | Δplc/plc | ΔpfoA | ΔpfoA/pfoA  

Fig. 8. Vidal et al.,

| SYTO9 | CPA | Merge | Merge |
| --- | --- | --- | --- |
| ![Image](https://example.com/image1) | ![Image](https://example.com/image2) | ![Image](https://example.com/image3) | ![Image](https://example.com/image4) |
| Optical middle section |  |  | Optical top section |

| S13Δplc | S13Δplc/plc |
| --- | --- |
| ![Image](https://example.com/image5) | ![Image](https://example.com/image6) |
| ![Image](https://example.com/image7) | ![Image](https://example.com/image8) |
