
Hindawi Publishing Corporation
BioMed Research International
Volume 2015, Article ID 759348, 17 pages
http://dx.doi.org/10.1155/2015/759348

---

**Review Article**

**The Formation of Biofilms by Pseudomonas aeruginosa: A Review of the Natural and Synthetic Compounds Interfering with Control Mechanisms**

Tsiry Rasamiravaka,$^{1}$ Quentin Labtani,$^{1}$ Pierre Duez,$^{2}$ and Mondher El Jaziri$^{1}$

$^{1}$ Laboratory of Plant Biotechnology, Université Libre de Bruxelles, rue des Professeurs Jeener et Brachet 12, 6041 Gosselies, Belgium  
$^{2}$ Department of Therapeutical Chemistry and Pharmacognosy, Université de Mons, Avenue Maistriau 19, Bâtiment Mendeleiev, 7000 Mons, Belgium  

Correspondence should be addressed to Mondher El Jaziri; jaziri@ulb.ac.be  

Received 29 June 2014; Revised 3 September 2014; Accepted 7 September 2014  

Academic Editor: Nikos Chorianopoulos  

Copyright © 2015 Tsiry Rasamiravaka et al. This is an open access article distributed under the Creative Commons Attribution License, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.

P. aeruginosa is an opportunistic pathogenic bacterium responsible for both acute and chronic infections. Beyond its natural resistance to many drugs, its ability to form biofilm, a complex biological system, renders ineffective the clearance by immune defense systems and antibiotherapy. The objective of this report is to provide an overview (i) on P. aeruginosa biofilm lifestyle cycle, (ii) on the main key actors relevant in the regulation of biofilm formation by P. aeruginosa including QS systems, GacS/GacA and RetS/LadS two-component systems and C-di-GMP-dependent polysaccharides biosynthesis, and (iii) finally on reported natural and synthetic products that interfere with control mechanisms of biofilm formation by P. aeruginosa without affecting directly bacterial viability. Concluding remarks focus on perspectives to consider biofilm lifestyle as a target for eradication of resistant infections caused by P. aeruginosa.

---

### 1. Introduction

The misuse and abuse of antibiotics are recognized to create selective pressure, resulting in the widespread development of resistant bacterial strains [1, 2]. Antibiotics are also known to kill “good/beneficial” indigenous bacteria, which may have protective role against pathogenic bacteria [3, 4]. Another important point to consider is that antibiotics have been found to be less effective in biofilm-growing bacteria [5].

Facing these limitations of antibiotics, there is an increasing need for the discovery and the development of antimicrobial agents that present novel or unexplored properties to efficiently control and manage bacterial infectious diseases [6]. Inhibition of bacterial virulence and/or biofilm formation by targeting nonmicrobicidal mechanisms are examples of increasingly explored antipathogenic approaches [7–9]. Among opportunistic pathogenic bacteria, P. aeruginosa, which produces several virulence factors, is known to be an important human and plant pathogen, responsible for various infections, particularly in immunocompromised persons [10]. Besides this, the remarkable ability of P. aeruginosa to form biofilms in many environments renders antibiotic treatments inefficient and therefore promotes chronic infectious diseases [5, 11].

Three global nonmicrobicidal strategies have been proposed to struggle against pathogenic bacteria with biofilm formation ability by (i) avoiding microbial attachment to a surface; (ii) disrupting biofilm development and/or affecting biofilm architecture in order to enhance the penetration of antimicrobials; and (iii) affecting biofilm maturation and/or inducing its dispersion and degradation [8, 12, 13].

The present review covers the scope of natural compounds from both prokaryote and eukaryote organisms that have been identified to disrupt biofilm lifestyle cycle in P. aeruginosa without affecting directly bacterial viability. As a prerequisite and for a better understanding of the proposed mechanisms of action of some of the identified compounds, relevant key molecular actors in P. aeruginosa biofilm formation and its regulation, such as the chemical signalization machinery involved in bacteria-environment

BioMed Research International

Stage I | Stage II | Stage III | Stage IV | Stage V
--- | --- | --- | --- | ---
2h | 8h | 14h | 1–4 days | 5 days

(a) | (b) | (c) | (d) | (e)

25 μm

1 day | 2 days | 3 days | 4 days

EPS
Bacteria

FIGURE 1: Biofilm lifestyle cycle of *P. aeruginosa* PAO1 grown in glucose minimal media. In stage I, planktonic bacteria initiate attachment to an abiotic surface, which becomes irreversible in stage II. Stage III corresponds to microcolony formation. Stage IV corresponds to biofilm maturation and growth of the three-dimensional community. Dispersion occurs in stage V and planktonic bacteria that are released from the biofilm to colonize other sites. The biofilm formation by *P. aeruginosa* PAO1 was revealed with Syto9 and visualized in Leica DM IRE2 inverted fluorescence microscope with 400x magnification at 2 h (Stage I), 8 h (Stage II), 14 h (Stage III), 1 to 4 days (Stage IV), and 5 days (Stage V). Images represent a 250 × 250-μm field.

interaction, including quorum sensing (QS) pathways, will be summarized.

a highly hydrated polar mixture that contributes to the overall structural scaffold and architecture of the biofilm [22–24].

Depending on *P. aeruginosa* strains and/or nutritional conditions, different biofilm phenotypes can be developed [25]. For instance, in glucose minimal media, biofilm lifestyle cycle of *P. aeruginosa* PAO1 can be subdivided into five major phenotypic steps (Figure 1). The process begins by the reversible adhesion of planktonic bacteria onto a surface suitable for growth (Figure 1(a), Stage I), followed by irreversible attachment of bacteria, which thereafter form microcolonies in EPS matrix (Figure 1(b), Stage II). Progressively, bacterial microcolonies expand and their confluences lead to a more structured phenotype with noncolonized space (Figure 1(c), Stage III). Then, noncolonized spaces are filled with bacteria, which finally cover the entire surface (Figure 1(d), Stage IV).

## 2. Biofilm Lifestyle Cycle of *P. aeruginosa*

Biofilm formation is an endless cycle, in which organized communities of bacteria are encased in a matrix of extracellular polymeric substances (EPS) that hold microbial cells together to a surface [14, 15]; these are thought to be determinant in 65–80% of all microbial infections [16–18]. In this microscopic world, biofilms are metaphorically called a “city of microbes” [19, 20] with EPS, which represents 85% of total biofilm biomass, as “house of the biofilm cells” [21]. EPS is composed mainly of biomolecules, exopolysaccharides, extracellular DNA (eDNA), and polypeptides that form

Meanwhile, the growth of three-dimensional communities is observed (Figure 1, Stages III and IV). Finally, bacteria disperse from the sessile structure and reenter in planktonic state to spread and colonize other surfaces [15, 26] (Figure 1(e), Stage V).

P. aeruginosa produces at least three polysaccharides (alginate, Pel, and Psl) that are determinant for the stability of the biofilm structure [27, 28]. Mucoid and nonmucoid P. aeruginosa strains differ by the qualitative composition of their polysaccharides in the biofilm matrix, predominantly alginate or Psl/Pel, respectively [29–31]. Alginate, a linear unbranched polymer composed of D-mannuronic acid and L-guluronic acid [32], contributes to the structural stability and protection of biofilms as well as to the retention of water and nutrients [33]. The Pel polysaccharide is mainly a glucose-rich matrix material, with still unclarified composition [34, 35], while Psl comprises a repeating pentasaccharide consisting of D-mannose, L-rhamnose, and D-glucose [36]. Pel and Psl can serve as a primary structure scaffold for biofilm development and are involved at early stages of biofilm formation [30, 37, 38].

eDNA constitutes an important functional component of P. aeruginosa biofilm matrix; indeed (i) P. aeruginosa biofilm formation is prevented by exposition to DNase I [39]; (ii) biofilms that are deficient in eDNA have been shown to be more sensitive to the detergent sodium dodecyl sulfate [40]; (iii) eDNA facilitates the twitching motility-mediated biofilm expansion by maintaining coherent cell alignments [41]; (iv) eDNA has been proposed to play an important role in the initial and early development of P. aeruginosa biofilms as a cell-to-cell interconnecting compound [24, 42, 43]; and (v) finally, eDNA constitutes a nutrient source for bacteria during starvation [44, 45].

Beyond their role in bacterial motilities [46–48], P. aeruginosa extracellular appendages flagella, type IV pili and cup fimbriae, are also considered to be matrix components that play adhesive roles in the cell-to-surface interactions (irreversible attachment) as well as in microcolony formation in biofilms. Mutants defective in flagellar-mediated motility and mutants defective in biogenesis of the polar-localized type IV pili do not develop microcolonies compared to the wild type strains [49–51].

### 3. Overview of Global Regulating Systems Involved in P. aeruginosa Biofilm Formation

The complex regulation of biofilm formation involves multiple bacterial machineries, including the QS systems and the two-component regulatory systems that both interact mainly with EPS production [52]. Deficiency in the network regulation required for biofilm matrix formation effectively results in the alteration of the biofilm structure and architecture and, therefore, of its protective role. The main key actors relevant in the regulation of biofilm formation by P. aeruginosa are summarized in Figure 2.

#### 3.1. QS Mechanisms and Biofilm Formation

QS is a cell-to-cell communication used by many bacteria to detect their population density by producing and perceiving diffusible signal molecules that coordinate virulence factors production, motility, and biofilm formation [53, 54]. P. aeruginosa possesses two main QS systems (las and rhl) which drive the production (throughout synthases LasI and RhlI) and the perception (by the transcription factors LasR and RhlR) of the autoinducer signaling molecules N-(3-oxododecanoyl)-L-homoserine lactone (3-oxo-C12-HSL) and N-butanoyl-L-homoserine lactone (C4-HSL) (Figure 3(a)), respectively [54]. A third QS system, based on quinolone signals (PQS system), interacts with the acyl homoserine lactones (AHLs) systems in an intricate way [54].

Davies et al. [55] have evidenced the role of the las system for biofilm formation and maturation; compared to wild type biofilm, the biofilm of lasI mutant appears flat, undifferentiated, and quickly dispersed from the surface upon exposure to sodium dodecyl sulfate. The precise implication of las system in biofilm formation is not yet clear. However, Gilbert et al. [56] reported that the QS regulator LasR can bind to the promoter region of the psl operon, suggesting that QS can regulate psl expression. The rhl system has been reported to intervene in P. aeruginosa biofilm formation [57] by enhancing Pel polysaccharide biosynthesis; transcription of the pel operon is actually reduced in rhlI mutant. The PQS system, for its part, is linked to eDNA release during biofilm development; biofilm formed by pqsA mutant contains less eDNA than biofilm formed by the wild type [40, 42]. All together these data indicate that the three QS systems known in P. aeruginosa play roles in biofilm lifestyle cycle.

Importantly, an indirect link between biofilm formation and QS has been reported, through the control of swarming and twitching motilities, as well as rhamnolipids and lectins production. The swarming motility, a form of organized surface translocation, depends on extensive flagellation and cell-to-cell contact [58, 59]; regulated by the rhl system [60], swarming motility is implicated in early stages of P. aeruginosa biofilm establishment. Strains grown under conditions that promote swarming motility (growth medium with glutamate or succinate as carbon source) form flat and uniform biofilm while strains with limited swarming motility result in biofilm containing nonconfluent cell aggregates [25]. Twitching motility, a flagella-independent form of bacterial translocation, occurs by successive extension and retraction of polar type IV pili [47]. Known to be regulated by the rhl system on Fe-limited minimal medium [61], twitching motilities are necessary for the assembly of a monolayer of P. aeruginosa cells into microcolonies [49].

Beyond their biosurfactant and virulence factor roles [62], rhamnolipids, whose production is under the rhl system control [63], present multiple roles in biofilm formation by P. aeruginosa. Indeed, they are believed to be involved in (i) forming microcolonies [64]; (ii) maintaining open channel structures that prevent bacterial colonization by disrupting both cell-to-cell and cell-to-surface interactions [26]; (iii) facilitating three-dimensional mushroom-shaped structures formation in P. aeruginosa biofilms [64]; and (iv) facilitating the cell dispersion from the biofilm as P. aeruginosa

BioMed Research International

Input signal(s)

RetS
GacS
LadS

GacA
GacA
P

2

RsmZ/Y

RsmA

pslA-L
lasR
lasI
las system
rhlR
rhlI
rhl system
pelA-G
algA-44-X

LasR
3-oxo-C12-HSL
1
RhI
C4-HSL
RhlR

PQS system

Psl
eDNA
Rhamnolipids
Lectins (LecA, LecB)
Pel
Alginate

Swarming
Twitching

Components and mechanisms involved in biofilm formation

FIGURE 2: Relevant bacterial systems and factors implicated in the regulation of *P. aeruginosa* biofilm formation. (1) Quorum sensing system; (2) Two-component regulatory system GacS/GacA and RetS/LadS (RR: response regulator domain receiver; P: phosphorylation) pathway; (3) Exopolysaccharides production and c-di-GMP pool regulation. See text for explanation.

variants which produce more rhamnolipids than wild-type *P. aeruginosa* exhibit hyper-detaching properties [65, 66]. Finally, the cytotoxic virulence factor, galactophilic lectins LecA and LecB, has been proposed to contribute to biofilm development in *P. aeruginosa*, since LecA and LecB mutants form thin biofilms as compared to the wild type bacteria [67, 68]. Both LecA and LecB expressions are regulated by the *rhl* QS system [69].

### 3.2. Biofilm Regulation by GacS/GacA and RetS/LadS Two-Component Systems

Among the 60 two-components systems found in the genome of *P. aeruginosa* [70], the GacS/GacA system acts as a super-regulator of the QS system and is involved in the production of multiple virulence factors as well as in biofilm formation [71]. The Gac system consists of a transmembrane sensor kinase (GacS) that, upon autophosphorylation, transfers a phosphate group to its cognate regulator (GacA) which in turn upregulates the expression of the small regulatory RNAs (*RsmZ* and *RsmY*). *RsmZ* and *RsmY* capture the small RNA-binding regulatory protein RsmA (encoded by *rsmA* gene), a repressor that posttranscriptionally regulates the *psl* locus (*pslA-L*) [72–74]. The GacS/GacA system also has a control on the AHL system as it inactivates free RsmA which negatively controls the synthesis of C4-HSL and 3-oxo-C12-HSL and therefore the extracellular virulence factors controlled by the *las* and *rhl* systems [75–77].

The hybrid sensor histidine kinase RetS is known to repress biofilm formation [78, 79] whereas the histidine kinase LadS antagonizes the effect of RetS [80]. Indeed, Δ*retS* mutant form more structured biofilms as compared to wild type *P. aeruginosa* PAO1 [78]; the PA14 strain (naturally deficient in *ladS* gene) displays attenuated biofilm formation compared to PA14 *LadS*⁺ strain [81]. It is reported that RetS

BioMed Research International

(a) Native AHLs

C4-HSL

3-Oxo-C12-HSL

(b) AHLs analogues

3-Oxo-C12-cyclohexanone

N-Acylcyclopentylamines

N-(indole-3-butanoyl)-L-HSL

N-(4-bromo-phenylacetanoyl)-L-HSL

Penicillic acid

Patulin

Manoalide

(c) Compounds with lactone ring analogues

FIGURE 3: Structure of natural and synthetic AHL-based compounds which inhibit biofilm formation by *P. aeruginosa*. (a) Native N-acyl-l-homoserine lactone, signal molecules of *P. aeruginosa* (C4-HSL and 3-oxo-C12-HSL), (b) synthetic analogue of AHLs with side aromatics and synthetic analogues of AHLs with modified lactone rings, and (c) natural (manoalide, penicillic acid, and patulin), and synthetic (furanones) compounds with lactone ring analogues.

and LadS interact with the GacS/GacA system by modulating the phosphorylation state of GacS, which consequently inhibits and promotes, respectively, the phosphorylation of GacA [82, 83].

It is interesting to note that GacS/GacA and RetS/LadS systems are proposed to be involved in mediating the transition of the *P. aeruginosa* phenotype from an acute to chronic phase infection [78].

3.3. C-di-GMP-Dependent Polysaccharides Biosynthesis and Biofilm Formation. Polysaccharides production is dependent on the intracellular pool of bis-(3′-5′)-cyclic dimeric guanosine monophosphate (c-di-GMP) [84, 85], a ubiquitous intracellular second messenger widely distributed in bacteria [86]. In bacterial cells, c-di-GMP is generated from two molecules of guanosine triphosphate by diguanylate cyclases and broken down into 2-GMP by specific phosphodiesterases [86].

High levels of c-di-GMP promote the biosynthesis of polysaccharides (alginate and Pel). Indeed, a binding process of c-di-GMP to PelD and Alg44 proteins is required for Pel and alginate polymer formation, respectively [85, 87]. However, the exact molecular mechanism by which this interaction regulates the polymerization of sugar precursors is not known.

Conversely, low levels of c-di-GMP promote bacterial motilities by enhancing flagellar formation and bacterial dispersion [85].

## 4. Natural and Synthetic Products That Affect *P. aeruginosa* Biofilm Formation

Plants and animals are naturally exposed to bacterial infections and they respond to bacterial components and signal molecules in different manners, including the activation of defense mechanisms and/or the expression of stress management genes [88–93]. Therefore, it is obvious to expect that eukaryotes have developed chemical mechanisms to combat pathogens by killing them or silencing virulence mechanisms such as QS system and/or biofilm formation. Tables 1 and 2 summarize the reported natural and synthetic products that affect *P. aeruginosa* biofilm formation.

### 4.1. Antibiofilm Compounds with Anti-QS Activity

Several classes of molecules have been reported to present both antibiofilm formation and anti-QS properties in *P. aeruginosa* [94–96].

Some AHL analogues (Figure 3(b)) have been shown to exhibit this double inhibitory activity. Geske et al. [97] have reported that synthetic analogues of AHLs with additional aromatic moieties [N-(indole-3-butanoyl)-L-HSL and N-(4-bromo-phenylacetanoyl)-L-HSL] display inhibitory activity on LasR-based QS system as well as biofilm formation in *P. aeruginosa* PAO1. Synthetic AHLs analogues, where the homoserine lactone ring is replaced by a cyclohexanone ring, downregulate expression of the *LasI* AHL synthase, resulting in a reduced expression of the virulence factors pyocyanin and elastase and in an alteration of biofilm morphology/phenotype [98]. Nonhydrolysable cyclopentyl analogues of AHLs (*N*-acyle cyclopentylamides) inhibit the *lasI* and *rhlA* expression, the production of virulence factors, including elastase, pyocyanin, and rhamnolipids, and the biofilm formation, without affecting bacterial growth [99].

Halogenated furanones (particularly furanones C-30 and C-56), inspired from natural compounds produced by the marine macroalga *Delisea pulchra*, exhibit biofilm reduction and target the *las* and *rhl* systems in *P. aeruginosa* [55, 100, 101]. Besides, in mouse lungs infected with *P. aeruginosa*, they were found to inhibit bacterial colonization to improve the clearance of bacteria from the host and to reduce the tissue damage [102].

Among the macrolide antibiotics, azithromycin, derived from *Saccharopolyspora erythraea*, has been the most investigated anti-QS antibiotic that presents a strong QS and biofilm inhibitory effect in *P. aeruginosa* [103–105]. Indeed, at subinhibitory azithromycin concentration (2 μg/mL), *P. aeruginosa* produces lower AHL signal molecules and virulence factors [106, 107] suggesting that the observed biofilm inhibition is at least partially due to the reduction of both C4-HSL and 3-oxo-C12-HSL production [108]. Interestingly, azithromycin has been reported to diminish the expression of *GacA* but also *RsmA* at translational level [109], to inhibit the synthesis of alginate [103] and to reduce the three types of motility (swimming, swarming, and twitching) [110].

Penicillic acid and patulin, two secondary fungal metabolites from *Penicillium* species, were shown to effect QS-controlled gene expression in *P. aeruginosa*, most likely by affecting the RhlR and LasR regulatory proteins at posttranscriptional level. In vitro studies showed that *P. aeruginosa* PAO1 biofilms treated with patulin and tobramycin were considerably more susceptible to the antibiotic as compared to control biofilms exposed to either tobramycin or patulin alone [111]. However, treatment with patulin alone did not affect development of the biofilm and no hypothesis of mechanisms of action was proposed by authors. The genotoxicity of patulin certainly limits its potential usefulness [112].

Manoalide, a sesterterpenoid from the marine organism *Luffariella variabilis*, exhibits antibiofilm and anti-QS activities (*las* system) in *P. aeruginosa* without bactericidal effects [113], although presenting antibiotic activity against gram-positive bacteria [114].

Solenopsin A alkaloid, isolated from the ant *Solenopsis invicta*, inhibits *P. aeruginosa* pyocyanin production, probably through disruption of the *rhl* signaling system and reduces biofilm production in a dose-dependent manner [115].

Mammalian cells release enzymes called paraoxonases 1 (extracted from human and murine sera) that have lactonase activity; degrading *P. aeruginosa* AHLs, they prevent, in an indirect way, QS and biofilm formation [116, 117].

The phenolic compound curcumin, a major constituent of turmeric roots (*Curcuma longa* L.), downregulates virulence factors (pyocyanin, elastase, and protease) in *P. aeruginosa* PAO1 and inhibits adherence of the bacteria to polypropylene surfaces. This was correlated with a decrease in 3-oxo-C12-HSL production [118]. Rosmarinic acid, a natural phenolic compound produced by the root of *Ocimum basilicum* L. upon *P. aeruginosa* infection, prevents biofilm formation but fails to penetrate mature biofilm under in vivo and in vitro conditions [89]. Structure-based virtual screenings against LasR and RhlR receptor proteins effectively indicate that rosmarinic acid is a potential QS inhibitor [119]. Ellagic acid derivatives, from *Terminalia chebula* Retz., have been shown to downregulate *lasIR* and *rhlIR* genes expression with a concomitant AHLs decrease, resulting in the attenuation of virulence factor production and in an enhanced sensitivity of biofilm towards tobramycin [120]. Girennavar et al. [121] demonstrated that the furocoumarins from grapefruit juice, bergamottin and dihydroxybergamottin, inhibit the activities of the autoinducers AI-1 (*N*-3 hydroxybutanoyl-homoserine lactone) and AI-2 (furanosyl borate diester) in a *V. harveyi* bioassay. Besides, these authors showed that AI-1 and AI-2 inhibit biofilm formation in *E. coli* O157:H7, *Salmonella typhimurium*, and *P. aeruginosa* without affecting bacterial growth. However, the mechanisms of action remain unclear.

| Natural products compounds | Origin | Class | Activities | Biofilm inhibition (a) | Dispersion promotion | Synergy with antibiotic | References |
|---------------------------|--------|-------|------------|----------------------|---------------------|-------------------------|------------|
| Alginate lyase            | P. aeruginosa | Enzyme | - | + | + | [134] |
| Ursolic acid              | Diospyros dendo Welw. | Triterpenoid | - | + (24h) | motility | NC | [141] |
| p-Coumaroyl-hydroxy-ursolic acid | Diospyros dendo Welw. | Coumarate ester of triterpene | NC | + (24h) | NC | [142] |
| Zingerone                 | Zingiber officinale Rosc. | Phenolic compound | NC | + (168h) | c-di-GMP adherence | NC | [140] |
| Casbane diterpene        | Croton nepetaefolius Bail. | Diterpenoid | NC | + (24h) | NC | + (2) | [137] |
| DNase I                   | Bovine pancreas | Enzyme | NC | + (18–24h) | + | [136] |
| DNase-II 2                | Human stratum corneum | Enzyme (lactonase) | + | + (24h) | NC | NC | [116, 117] |
| Paraoxonases 1            | Human and murine sera | - | - | - | - | - | - |
| Luffariella variabilis    | (Polejaeff, 1884) (marine organism) | Sesterterpenoid | + (las system) | + (24h) | NC | NC | [113, 114] |
| Solenopsin A              | Solenopsis invicta (insect; ant) | Alkaloid | + (rhl system) | + (24h) | NC | NC | [115] |
| Catechin                  | Combretum albiflorum (Tul.) Jong kind | Flavonoid | + (las rhl systems) | + (24h) | NC | NC | [124] |
| Naringenin                | Commercial | Flavonoid | + (las rhl systems) | + (48h) | NC | NC | [125] |
| Coumarate ester           | D. trichocarpa Baker. | Phenolic compound | + (las rhl systems) | + (48h) | + | + (1) | [126] |
| Ajoe ne                   | Allium sativum L. | Organosulfur | + (las rhl systems) | + (96h) | NC | + (1) | [130] |
| Ellagic acid derivatives  | Terminalia chebula Retz. | Phenolic compound | + (las rhl systems) | + (72h) | NC | + (1) | [120] |
| Rosmarinic acid          | Ocimum basilicum L. | Phenolic compound | + (las rhl systems) | + (18h) | - | NC | [89, 119] |
| Eugenol                   | Syzygium aromaticum (L.) Merr. Et Perry | Phenylpropanoid | + (las pqs systems) | + (24h) | NC | NC | [128] |
| Curcumin                  | Curcuma longa L. | Phenolic compound | + (AHLs) | + (48h) | NC | NC | [118] |
| Bergamottin and dihydroxybergamottin | Citrus paradisi Macfad. (Rio Red and Marsh White grapefruits) | Furocoumarins | + (AI-1 and AI-2) | + (24h) | NC | NC | [121] |
| Penicillin acid           | Penicillium species | Furaneone | + (LasR, RhlR) | NC | NC | NC | [111] |
| Patulin                   | Euro pyranone | + (LasR RhlR) | - | + (1) | - | - | - |
| Emodin                    | Rheum palmatum L. | Anthraquinone | + (docking trar) | + (72h) | NC | + (3) | [123] |
| Baicalein                 | Scutellaria baicalensis Georgi. | Flavonoid | + (docking trar) | + (72h) | NC | + | [122] |

+ : yes; – : no; NC: not communicated.

(a) Experiment duration.
(1) Aminoglycosides, (2) ciprofloxacin, (3) ampicillin.

Synthetic compounds (Natural compound origin) | Activities | Biofilm inhibition (a) | Dispersion promotion | Synergistic antibiotic and/or immune defense effect | References
--- | --- | --- | --- | --- | ---
TAGE and CAGE (Bromoageliferin) | NC | + (24h) | + | NC | [132]
Dihydrosventrin (Sventrin) | NC | + (24h) | + | NC | [133]
N-(4-bromo-phenylacetanoyl)-L-HSL; N-(indole-3-butanoyl)-L-HSL (AHLs) | + AHLs antagonist (las system) | + (24h) | NC | NC | [97]
3-oxo-C12-cyclohexanone (AHLs) | + AHLs antagonist (las system) | + (24h) | NC | NC | [98]
Furanone C-30 and C-56 (Furanone) | + (lasI and rhlA) | + (24h) | NC | NC | [99]
S-phenyl-L-cysteine sulfoxide (Cysteine sulfoxide alliin) | + (las, rhl systems) | + (24h) | NC | NC | [55, 100, 101]
Diphenyl disulfide (Disulfide derivatives of the alliinase mediated reactions) | + las system | + (24h) | NC | NC | [148]
Azithromycin (Erythromycin) | + (gacA, las and rhl systems) | + (72h) | + | + (1) | [103, 108, 109]

+ : yes; - : no; NC: not communicated.
(a) Experiment duration.
(1) Aminoglycosides.

BioMed Research International

![Figure 4: Phenolic compounds and derivatives with antibiofilm and anti-QS properties.](image.png)

Docking screening for QS inhibitors predicted that the flavone baicalein, obtained from the roots of *Scutellaria baicalensis* Georgi, could interact with *A. tumefaciens* QS transcription activator protein TraR. Effectively, at 20 μM, baicalein promotes the proteolysis of the signal receptor TraR protein in *Escherichia coli* biosensor, significantly inhibiting the biofilm formation by *P. aeruginosa* [122]. Similarly, the screening of traditional Chinese medicinal plants identified the anthraquinone emodin, extracted from rhubarb (*Rheum palmatum* L.); emodin actually inhibits the *P. aeruginosa* biofilm formation at 20 μM, increasing the activity of ampicillin [123].

The flavan-3-ol catechin, isolated from the bark of *Combretum albiflorum* (Tul.) Jongkind, as well as the flavanone naringenin, both at 4 mM final concentration, do interfere with QS mechanism in *P. aeruginosa* PAO1 by affecting autoinducers perception and biofilm formation [124–126]. A coumarate ester isolated from the bark extract of Malagasy endemic *Dalbergia trichocarpa* Baker interferes with *P. aeruginosa* QS systems (*las* and *rhl*), inhibits the biofilm formation and increases the effectiveness of the antibiotic tobramycin in killing biofilm-encapsulated *P. aeruginosa* [126] (Figures 4 and 5).

Recently, *Meliaceae*, *Melastomataceae*, *Lepidobotryaceae*, and *Sapindaceae*, collected from neotropical rainforests in Costa Rica, presented significant anti-QS activities in a *Chromobacterium violaceum* bioassay and/or inhibition of biofilm formation by *P. aeruginosa* PA14 [127]. Although the exact natures of the active constituents are not yet elucidated, the authors suggest that they could belong to polar polyphenols similar to tannic acid.

A recent screening of various herbal extracts revealed that clove extract (*Syzygium aromaticum* (L.) Merr. Et Perry) inhibits QS-controlled gene expression (*las* and *pqs* systems) in *P. aeruginosa* with eugenol as major active constituent [128]. Eugenol, at subinhibitory concentrations (400 μM) inhibited virulence factors production including elastase, pyocyanin and biofilm formation. In agreement with this finding, subinhibitory concentrations of the clove essential oil significantly reduces *las*- and *rhl*-regulated virulence factors, exopolysaccharide production, and biofilm formation by *P. aeruginosa* PAO1 [129].

Ajoene, an allyl sulfide isolated from garlic (*Allium sativum* L.), has been reported to affect QS-regulated genes in *P. aeruginosa*, including the production of rhamnolipids. Additionally, ajoene synergizes with the antibiotic tobramycin in killing biofilm-encapsulated *P. aeruginosa*, improving the clearance of *P. aeruginosa* from lungs in a mouse model of pulmonary infection [130]. A naturally-inspired organosulfur compound (S-phenyl-L-cysteine sulfoxide) and its derivative (diphenyl disulfide) have been reported to significantly reduce the amount of biofilm formation by *P. aeruginosa* [131]. The S-phenyl-L-cysteine sulfoxide antagonizes both the *las* and *rhl* QS systems whereas the diphenyl disulfide only interferes with the *las* system.

FIGURE 5: *P. aeruginosa* biofilm phenotypes and effectiveness of tobramycin treatment in presence of DMSO 1% or coumarate ester (CE) at 300 μg/mL. (a) After 1 day of incubation, *P. aeruginosa* fails to form structured confluent aggregate in presence of CE as compared to DMSO treatment. (b) CE considerably increases the susceptibility of *P. aeruginosa* to tobramycin (100 μg/mL), as shown by the increased proportion of dead cells compared with DMSO. The bacterial viability was assessed by staining the cells with SYTO-9 (green areas—live bacteria) and propidium iodide (red areas—dead bacteria) furnished in the LIVE/DEAD BacLight kit. Cells were visualized using a Leica DM IRE2 inverted fluorescence microscope using a 40x objective lens and colored images were assembled using Adobe Photoshop.

### 4.2. Antibiofilm Compounds without or with Unspecified Anti-QS Activity

Various organisms, including prokaryotes and eukaryotes (marine organisms, animals, and plants) have been reported to produce secondary metabolites which exert antibiofilm activity. Some of those natural compounds have been used as models to build synthetic antibiofilm compounds against *P. aeruginosa*.

Bromoageliferin, pyrrole-imidazole alkaloids from marine sponges (*Agelas conifer*, Agelaceae), has been the scaffolding for the development of two derivatives, trans-bromoageliferin analogue 1 (TAGE) and cis-bromoageliferin analogue 2 (CAGE). Both synthetic derivatives inhibit biofilm formation and furthermore are able to disperse preexisting *P. aeruginosa* PAO1 biofilms without demonstrating a bactericidal or growth-inhibiting effect [132]. Analogues based upon the oroidin template, parent molecules of bromoageliferin, have been synthesized and screened in *P. aeruginosa* for their antibiofilm ability [133]. The authors found that the most potent analogue turned out to be dihydrosventrin, a variant of the pyrrole-imidazole alkaloids sventrin (from *Agelas sventre*) which exhibits biofilm inhibition and biofilm dispersion for different strains of *P. aeruginosa* without any microbiocidal activity.

Alginate lyase, produced by *P. aeruginosa* itself, promotes biofilm dispersion and acts synergically with antibiotics for successful elimination of mucoid strains of *P. aeruginosa* established in the respiratory tracts of cystic fibrosis patients [134]. However, a recent study demonstrated that this effect cannot be attributed to the catalytic activity of the enzyme. Indeed, bovine serum albumin or simple amino acids lead to the same results. The authors postulate that alginate lyase acts simply as a nutrient source, modulating cellular metabolism and thus inducing cellular detachment and enhancing tobramycin efficacy [135].

Bovine pancreatic Dnase I and Dnase-1L2, extracted from human stratum corneum, exhibited strong antibiofilm activity in *P. aeruginosa* [136]. Indeed, the degradation of extracellular DNA leads to an altered biofilm that permits increased antibiotics penetration [137].

Extracts of Ginger (*Zingiber officinale* Rosc.), long used by Indians, Asians, and Arabs to treat numerous ailments [137], inhibit *P. aeruginosa* PA14 biofilm formation through the reduction of c-di-GMP production and consequent reduction of total polysaccharides production [138]. The ginger extract revealed no AHL-based QS inhibition in the *Chromobacterium violaceum* CV026 and *Agrobacterium tumefaciens* NT1 reporter biosensor systems. The major component of dry ginger root, zingerone (vanillyl acetone), has been shown to inhibit biofilm formation, to increase the susceptibility of *P. aeruginosa* PAO1 to ciprofloxacin [139] and to inhibit swimming, swarming, and twitching motilities. However, authors did not propose any mechanism of action.

The casbane diterpene, isolated from the ethanol extract of *Croton nepetaefolius* Baill., a plant native from northeastern Brazil, inhibits biofilm formation in several clinical relevant species, including *P. aeruginosa* (at 250 μg/mL) without

affecting the planktonic growth. Authors suggest that this inhibition of biofilm formation may be related to an interaction between casbane diterpene and lipopolysaccharides present on the cell surface, which might affect their adherence properties [140].

Ursolic acid (3β-hydroxy-urs-12-en-28-oic acid) from *Diospyros dendo* Welw. is identified to inhibit biofilm formation without interfering with QS systems in *E. coli*, *P. aeruginosa*, and *V. harveyi*; ursolic acid, at 10 μg/mL, has been found to reduce 72% of *E. coli* JM109 biofilm. Transcriptomic analyses led to the conclusion that ursolic acid inhibits biofilm formation by inducing motility [141]. The 3β-O-cis-p-coumaroyl-20β-hydroxy-12-ursen-28-oic acid, isolated from the same plant, strongly inhibits biofilm formation by *P. aeruginosa* PAO1 [142]. However, the mechanism of activity was not investigated.

and in human body which could lead to unexpected biofilm development. Thus, before progressing in clinical trial of relevant bioactive compounds, effort on the improvement of experimental *in vitro* and *in vivo* conditions should be addressed and clinical trial protocols should be discussed.

Potent antibiofilm agents are considered interesting if they exert a sustainable bioactivity; this can be indicated by an activity that resists accumulating bacterial toxins, enzymes, and metabolites for more than 48 h in culture media. As less than half of bioactive products have been tested up to 48 hours, further investigations are warranted to select those compounds with sustained activities, which would have more chances to be active in clinical conditions. Halogenated furanones have been widely studied for their powerful anti-QS and antibiofilm activities (<10 μM) [100]. However, their toxic and carcinogenic properties relegate them so far to the role of positive QS inhibitory controls in laboratory experiments [150, 151]. In this regard, herbal phenolic compounds and their derivatives, frequent in food components, and more particularly those already present in popular and approved herbal drugs (i.e., rosmarinic acid in *Melissa officinalis* L.), are promising candidates to develop antibiofilm agents; however, structure-activity studies are still required to better assign essential structural features responsible for antibiofilm activity. In the same perspective, searching for compounds active at nanomolar levels should be privileged as these could presumably present lower toxicity risks. The QS system is an obvious target for biofilm-associated infections as QS interacts, directly and/or indirectly, in different steps of biofilm formation. Intriguingly, even if QS inhibition is the most extensively studied approach against *P. aeruginosa*, several anti-QS natural compounds have not been yet investigated for their antibiofilm activity (e.g., human sexual hormones and some antibiotics at subinhibitory concentration, notably ceftazidime and ciprofloxacin) [103, 152]. Attractive therapeutic agents are those which modulate QS system(s) with an extending or particular impact on biofilm lifestyle; they could then be helpful as a preventive or curative approach and at every step of infectious diseases (acute and chronic). However, finding universal antibiofilm compounds represents a challenge as biofilm lifestyle, composition, and phenotype strongly depend on several parameters, such as nutritional conditions. In this regard, we support the hypothesis that compounds which target GacS/GacA pathway are worthy of interest with respect to the pathway hierarchically upstream position that controls positively both QS system and exopolysaccharides biosynthesis (Psl) (Figure 2). Such compounds could possibly impair almost all the biofilm lifestyle cycle of *P. aeruginosa*, from irreversible attachment to dispersion stages (Table 3) and could be powerful allies for conventional antibiotics in the struggle against bacterial biofilm-mediated infections [8, 12, 95].

### 5. Concluding Remarks and Perspectives

There is increasing evidence that biofilm-mediated infection facilitates the development of chronic infectious diseases and recurrent infections [143–145]. Relevance in using antibiofilm compounds is based on the restoration of antibiotic effectiveness by facilitating their penetration through compromised biofilm structure. Moreover, a degradation of the biofilm matrix could render infectious bacteria reachable to immune defenses (e.g., polymorphonuclear leukocytes, innate, and specific antibodies) [146, 147]. Thus, antibiofilm compounds could be interesting antibiotic adjuvants to prevent or treat chronic infections. Similarly, relevance in using anti-QS compounds is based on the concomitant drastic reduction of virulence factors expression, which gives the necessary time for immune defense systems to elaborate appropriate responses by the recruitment of immune cells and production of specific antibodies. Unlike antibiofilm compounds, anti-QS compounds are interesting to prevent or jugulate acute infection. However, it should also be noted that (i) anti-QS and antibiofilm compounds may lose their appeal in immune compromised patients who often harbor bacteria that are still alive but present in a disorganized and less virulent stage; (ii) QS systems do not control the totality of virulence factors expression; and (iii) the development of anti-QS bacterial resistance cannot be excluded [148]. These facts partly explain why the discovery of QS modulators has not yet led to major therapeutic breakthroughs. In our opinion, such bioactive compounds will probably not substitute antibiotics but rather optimize the effectiveness of infectious diseases treatment, notably through biofilm disruption and antibiotic dose reduction; their use is also appealing to optimize the use of microbiocidal products by reducing biofilm encroachment on biomaterials and medical devices.

In the perspective of therapeutic application, very few studies have been progressed to clinical trial. To the best of our knowledge, garlic is the only extract with anti-QS and antibiofilm to have been tested in a clinical trial with non-significant results, contrary to its drastic *in vitro* bioactivity effect [149]. One reason of this fact is that behavior of clinical isolates may be different when grown in laboratory condition

### Conflict of Interests

The authors declare that there is no conflict of interests regarding the publication of this paper.

| Biofilm lifestyle cycle | Stage I | Stage II | Stage III | Stage IV | Stage V |
|-------------------------|--------|----------|-----------|----------|---------|
| Reversible attachment   |        | Irreversible attachment | Maturation-1 (microcolony development) | Maturation-2 (maintenance) | Dispersion |
| Implicated factors      | c-di-GMP |           |           |          |         |
| General target          | (i) Promoting planktonic lifestyle<br>(ii) Blocking switch from planktonic to biofilm lifestyle | Reducing initial adhesion and interaction | Interfering with QS communication | Promoting dispersion and degradation |          |
| Examples of extracts or compounds | (i) Ginger extract<br>(ii) Ursolic acid | (i) Casbane diterpene<br>(ii) Coumarate ester | (i) Cell-to-cell communication<br>lasI<br>(ii) Flagella and type IV pili<br>(iii) Adhesins<br>(iv) Psl, Pel | (i) Cell-to-cell communication<br>(ii) Type IV pili<br>(iii) Matrix components: Psl, Pel, eDNA<br>(iv) Lectin A, B<br>(v) rhlA | (i) Reactivating metabolic activity for antibiotic efficiency.<br>(ii) Interfering with QS communication | (i) Solenopsin A<br>(ii) Naringenin<br>(iii) Furaneone C-30 and -56<br>(iv) Flavan-3-ol Catechin<br>(v) Ajoeone<br>(vi) Coumarate ester | DNase I, IL2 |

BioMed Research International

Acknowledgment
The authors would like to thank C.U.D. (Commission Universitaire pour le Développement, Belgium) for financial support throughout the project PIC-Madagascar 2009.

References
[1] L. C. Epps and P. D. Walker, “Fluoroquinolone consumption and emerging resistance,” US Pharmacology, vol. 10, pp. 47–54, 2006.
[2] B. K. English and A. H. Gaur, “The use and abuse of antibiotics and the development of antibiotic resistance,” Advances in Experimental Medicine and Biology, vol. 659, pp. 73–82, 2010.
[3] F. Guarner and J.-R. Malagelada, “Gut flora in health and disease,” The Lancet, vol. 361, no. 9356, pp. 512–519, 2003.
[4] L. Beaugerie and J.-C. Petit, “Antibiotic-associated diarrhoea,” Best Practice & Research: Clinical Gastroenterology, vol. 18, no. 2, pp. 337–352, 2004.
[5] N. Høiby, O. Ciofu, and T. Bjarnsholt, “Pseudomonas aeruginosa biofilms in cystic fibrosis,” Future Microbiology, vol. 5, no. 11, pp. 1663–1674, 2010.
[6] L. Cegelski, G. R. Marshall, G. R. Eldridge, and S. J. Hultgren, “The biology and future prospects of antivirulence therapies,” Nature Reviews Microbiology, vol. 6, no. 1, pp. 17–27, 2008.
[7] V. C. Kalia and H. J. Purohit, “Quenching the quorum sensing system: potential antibacterial drug targets,” Critical Reviews in Microbiology, vol. 37, no. 2, pp. 121–140, 2011.
[8] M. S. Blackledge, R. J. Worthington, and C. Melander, “Biologically inspired strategies for combating bacterial biofilms,” Current Opinion in Pharmacology, vol. 13, no. 5, pp. 699–706, 2013.
[9] G. Sharma, S. Rao, A. Bansal, S. Dang, S. Gupta, and R. Gabrani, “Pseudomonas aeruginosa biofilm: potential therapeutic targets,” Biologicals, vol. 42, no. 1, pp. 1–7, 2014.
[10] A. Deep, U. Chaudhary, and V. Gupta, “Quorum sensing and bacterial pathogenicity: from molecules to disease,” Journal of Laboratory Physicians, vol. 3, no. 1, pp. 4–11, 2011.
[11] T. Bjarnsholt, “The role of bacterial biofilms in chronic infections,” APMIS Supplementum, no. 136, pp. 1–51, 2013.
[12] L. Yang, Y. Liu, H. Wu et al., “Combating biofilms,” FEMS Immunology & Medical Microbiology, vol. 65, no. 2, pp. 146–157, 2012.
[13] J. Masak, A. Cejkova, O. Schreiberova, and T. Rezanka, “Pseudomonas biofilms: possibilities of their control,” FEMS Microbiology and Ecology, vol. 89, no. 1, pp. 1–14, 2014.
[14] R. M. Donlan, “Biofilms: microbial life on surfaces,” Emerging Infectious Diseases, vol. 8, no. 9, pp. 881–890, 2002.
[15] E. Karatan and P. Watnick, “Signals, regulatory networks, and materials that build and break bacterial biofilms,” Microbiology and Molecular Biology Reviews, vol. 73, no. 2, pp. 310–347, 2009.
[16] J. W. Costerton, P. S. Stewart, and E. P. Greenberg, “Bacterial biofilms: a common cause of persistent infections,” Science, vol. 284, no. 5418, pp. 1318–1322, 1999.
[17] M. Chicurel, “Bacterial biofilms and infections. Slimebusters,” Nature, vol. 408, no. 6810, pp. 284–286, 2000.
[18] M. Hentzer, L. Eberl, and M. Givskov, “Transcriptome analysis of Pseudomonas aeruginosa biofilm development: anaerobic respiration and iron limitation,” Biofilms, vol. 2, no. 1, pp. 37–61, 2005.
[19] P. Watnick and R. Kolter, “Biofilm, city of microbes,” Journal of Bacteriology, vol. 182, no. 10, pp. 2675–2679, 2000.

[20] I. A. Nikolaev and V. K. Plakunov, “Biofilm: ‘City of microbes’ or an analogue of multicellular organisms?” Mikrobiologiia, vol. 76, no. 2, pp. 149–163, 2007.
[21] H.-C. Flemming, T. R. Neu, and D. J. Wozniak, “The EPS matrix: the ‘House of Biofilm Cells,’” Journal of Bacteriology, vol. 189, no. 22, pp. 7945–7947, 2007.
[22] I. W. Sutherland, “The biofilm matrix—an immobilized but dynamic microbial environment,” Trends in Microbiology, vol. 9, no. 5, pp. 222–227, 2001.
[23] S. S. Branda, S. Vik, L. Friedman, and R. Kolter, “Biofilms: the matrix revisited,” Trends in Microbiology, vol. 13, no. 1, pp. 20–26, 2005.
[24] H.-C. Flemming and J. Wingender, “The biofilm matrix,” Nature Reviews Microbiology, vol. 8, no. 9, pp. 623–633, 2010.
[25] J. D. Shrout, D. L. Chopp, C. L. Just, M. Hentzer, M. Givskov, and M. R. Parsek, “The impact of quorum sensing and swarming motility on Pseudomonas aeruginosa biofilm formation is nutritionally conditional,” Molecular Microbiology, vol. 62, no. 5, pp. 1264–1277, 2006.
[26] M. E. Davey, N. C. Caiazza, and G. A. O'Toole, “Rhamnolipid surfactant production affects biofilm architecture in Pseudomonas aeruginosa PAO1,” Journal of Bacteriology, vol. 185, no. 3, pp. 1027–1036, 2003.
[27] C. Ryder, M. Byrd, and D. J. Wozniak, “Role of polysaccharides in Pseudomonas aeruginosa biofilm development,” Current Opinion in Microbiology, vol. 10, no. 6, pp. 644–648, 2007.
[28] A. Ghafoor, I. D. Hay, and B. H. A. Rehm, “Role of exopolysaccharides in Pseudomonas aeruginosa biofilm formation and architecture,” Applied and Environmental Microbiology, vol. 77, no. 15, pp. 5238–5246, 2011.
[29] L. Ma, M. Conover, H. Lu, M. R. Parsek, K. Bayles, and D. J. Wozniak, “Assembly and development of the Pseudomonas aeruginosa biofilm matrix,” PLoS Pathogens, vol. 5, no. 3, Article ID e1000354, 2009.
[30] L. Ma, J. Wang, S. Wang et al., “Synthesis of multiple Pseudomonas aeruginosa biofilm matrix exopolysaccharides is post-transcriptionally regulated,” Environmental Microbiology, vol. 14, no. 8, pp. 1995–2005, 2012.
[31] E. E. Mann and D. J. Wozniak, “Pseudomonas biofilm matrix composition and niche biology,” FEMS Microbiology Reviews, vol. 36, no. 4, pp. 893–916, 2012.
[32] H. Ertesvåg and S. Valla, “Biosynthesis and applications of alginate,” Polymer Degradation and Stability, vol. 59, no. 1–3, pp. 85–91, 1998.
[33] I. W. Sutherland, “Biofilm exopolysaccharides: a strong and sticky framework,” Microbiology, vol. 147, no. 1, pp. 3–9, 2001.
[34] L. Friedman and R. Kolter, “Genes involved in matrix formation in Pseudomonas aeruginosa PA14 biofilms,” Molecular Microbiology, vol. 51, no. 3, pp. 675–690, 2004.
[35] M. J. Franklin, D. E. Nivens, J. T. Wadge, and P. Lynne Howell, “Biosynthesis of the Pseudomonas aeruginosa extracellular polysaccharides, alginate, Pel, and Psl,” Frontiers in Microbiology, vol. 167, no. 2, pp. 1–16, 2011.
[36] M. S. Byrd, I. Sadovskaya, E. Vinogradov et al., “Genetic and biochemical analyses of the Pseudomonas aeruginosa Psl exopolysaccharide reveal overlapping roles for polysaccharide synthesis enzymes in Psl and LPS production,” Molecular Microbiology, vol. 73, no. 4, pp. 622–638, 2009.
[37] K. M. Colvin, V. D. Gordon, K. Murakami et al., “The pel polysaccharide can serve a structural and protective role in the biofilm matrix of Pseudomonas aeruginosa,” PLoS Pathogens, vol. 7, no. 1, Article ID e1001264, 2011.

[38] P. Vasseur, I. Vallet-Gely, C. Soscia, S. Genin, and A. Filloux, “The pel genes of the *Pseudomonas aeruginosa* PAK strain are involved at early and late stages of biofilm formation,” *Microbiology*, vol. 151, no. 3, pp. 985–997, 2005.

[39] J. J. T. M. Swartjes, T. Das, S. Sharifi et al., “A functional DNase i coating to prevent adhesion of bacteria and the formation of biofilm,” *Advanced Functional Materials*, vol. 23, no. 22, pp. 2843–2849, 2013.

[40] L. Yang, K. B. Barken, M. E. Skindersoe, A. B. Christensen, M. Givskov, and T. Tolker-Nielsen, “Effects of iron on DNA release and biofilm development by *Pseudomonas aeruginosa*,” *Microbiology*, vol. 153, no. 5, pp. 1318–1328, 2007.

[41] E. S. Gloag, L. Turnbull, A. Huang et al., “Self-organization of bacterial biofilms is facilitated by extracellular DNA,” *Proceedings of the National Academy of Sciences of the United States of America*, vol. 110, no. 28, pp. 11541–11546, 2013.

[42] M. Allesen-Holm, K. B. Barken, L. Yang et al., “A characterization of DNA release in *Pseudomonas aeruginosa* cultures and biofilms,” *Molecular Microbiology*, vol. 59, no. 4, pp. 1114–1128, 2006.

[43] C. B. Whitchurch, T. Tolker-Nielsen, P. C. Ragas, and J. S. Mattick, “Extracellular DNA required for bacterial biofilm formation,” *Science*, vol. 295, no. 5559, p. 1487, 2002.

[44] S. E. Finkel and R. Kolter, “DNA as a nutrient: novel role for bacterial competence gene homologs,” *Journal of Bacteriology*, vol. 183, no. 21, pp. 6288–6293, 2001.

[45] H. Mulcahy, L. Charron-Mazenod, and S. Lewenza, “*Pseudomonas aeruginosa* produces an extracellular deoxyribonuclease that is required for utilization of DNA as a nutrient source,” *Environmental Microbiology*, vol. 12, no. 6, pp. 1621–1629, 2010.

[46] T. Köhler, L. K. Curty, F. Barja, C. van Delden, and J.-C. Pechere, “Swarming of *Pseudomonas aeruginosa* is dependent on cell-to-cell signaling and requires flagella and pili,” *Journal of Bacteriology*, vol. 182, no. 21, pp. 5990–5996, 2000.

[47] J. S. Mattick, “Type IV pili and twitching motility,” *Annual Review of Microbiology*, vol. 56, pp. 289–314, 2002.

[48] E. Déziel, F. Lépine, S. Milot, and R. Villemur, “*rhlA* is required for the production of a novel biosurfactant promoting swarming motility in *Pseudomonas aeruginosa*: 3-(3-hydroxyalkanoyloxy)alkanoic acids (HAAs), the precursors of rhamnolipids,” *Microbiology*, vol. 149, no. 8, pp. 2005–2013, 2003.

[49] G. A. O’Toole and R. Kolter, “Flagellar and twitching motility are necessary for *Pseudomonas aeruginosa* biofilm development,” *Molecular Microbiology*, vol. 30, no. 2, pp. 295–304, 1998.

[50] S. Ruer, S. Stender, A. Filloux, and S. De Bentzmann, “Assembly of fimbrial structures in *Pseudomonas aeruginosa*: functionality and specificity of chaperone-usher machineries,” *Journal of Bacteriology*, vol. 189, no. 9, pp. 3547–3555, 2007.

[51] K. B. Barken, S. J. Pamp, L. Yang et al., “Roles of type IV pili, flagellum-mediated motility and extracellular DNA in the formation of mature multicellular structures in *Pseudomonas aeruginosa* biofilms,” *Environmental Microbiology*, vol. 10, no. 9, pp. 2331–2343, 2008.

[52] T. R. De Kievit, “Quorum sensing in *Pseudomonas aeruginosa* biofilms,” *Environmental Microbiology*, vol. 11, no. 2, pp. 279–288, 2009.

[53] M. R. Parsek and E. P. Greenberg, “Sociomicrobiology: the connections between quorum sensing and biofilms,” *Trends in Microbiology*, vol. 13, no. 1, pp. 27–33, 2005.

[54] P. N. Jimenez, G. Koch, J. A. Thompson, K. B. Xavier, R. H. Cool, and W. J. Quax, “The multiple signaling systems

[55] D. G. Davies, M. R. Parsek, J. P. Pearson, B. H. Iglewski, J. W. Costerton, and E. P. Greenberg, “The involvement of cell-to-cell signals in the development of a bacterial biofilm,” *Science*, vol. 280, no. 5361, pp. 295–298, 1998.

[56] K. B. Gilbert, T. H. Kim, R. Gupta, E. P. Greenberg, and M. Schuster, “Global position analysis of the *Pseudomonas aeruginosa* quorum-sensing transcription factor LasR,” *Molecular Microbiology*, vol. 73, no. 6, pp. 1072–1085, 2009.

[57] Y. Sakuragi and R. Kolter, “Quorum-sensing regulation of the biofilm matrix genes (*pel*) of *Pseudomonas aeruginosa*,” *Journal of Bacteriology*, vol. 189, no. 14, pp. 5383–5386, 2007.

[58] G. M. Fraser and C. Hughes, “Swarming motility,” *Current Opinion in Microbiology*, vol. 2, no. 6, pp. 630–635, 1999.

[59] R. M. Harshey, “Bees aren’t the only ones: swarming in gram-negative bacteria,” *Molecular Microbiology*, vol. 13, no. 3, pp. 389–394, 1994.

[60] R. Daniels, J. Vanderleyden, and J. Michiels, “Quorum sensing and swarming migration in bacteria,” *FEMS Microbiology Reviews*, vol. 28, no. 3, pp. 261–289, 2004.

[61] G. M. Patriquin, E. Banin, C. Gilmour, R. Tuchman, E. P. Greenberg, and K. Poole, “Influence of quorum sensing and iron on twitching motility and biofilm formation in *Pseudomonas aeruginosa*,” *Journal of Bacteriology*, vol. 190, no. 2, pp. 662–671, 2008.

[62] M. Van Gennip, L. D. Christensen, M. Alhede et al., “Inactivation of the *rhlA* gene in *Pseudomonas aeruginosa* prevents rhamnolipid production, disabling the protection against polymorphonuclear leukocytes,” *Acta Pathologica, Microbiologica, et Immunologica Scandinavica*, vol. 117, no. 7, pp. 537–546, 2009.

[63] D. H. Dusane, S. S. Zinjarde, V. P. Venugopalan, R. J. C. Mclean, M. M. Weber, and P. K. S. M. Rahman, “Quorum sensing: implications on Rhamnolipid biosurfactant production,” *Biotechnology and Genetic Engineering Reviews*, vol. 27, pp. 159–184, 2010.

[64] S. J. Pamp and T. Tolker-Nielsen, “Multiple roles of biosurfactants in structural biofilm development by *Pseudomonas aeruginosa*,” *Journal of Bacteriology*, vol. 189, no. 6, pp. 2531–2539, 2007.

[65] B. R. Boles, M. Thoendel, and P. K. Singh, “Rhamnolipids mediate detachment of *Pseudomonas aeruginosa* from biofilms,” *Molecular Microbiology*, vol. 57, no. 5, pp. 1210–1223, 2005.

[66] S. R. Schooling, U. K. Charaf, D. G. Allison, and P. Gilbert, “A role for rhamnolipid in biofilm dispersion,” *Biofilms*, vol. 1, pp. 91–99, 2004.

[67] S. P. Diggle, R. E. Stacey, C. Dodd, M. Cámara, P. Williams, and K. Winzer, “The galactophilic lectin, LecA, contributes to biofilm development in *Pseudomonas aeruginosa*,” *Environmental Microbiology*, vol. 8, no. 6, pp. 1095–1104, 2006.

[68] D. Tieler, S. Hacker, R. Loris et al., “*Pseudomonas aeruginosa* lectin LecB is located in the outer membrane and is involved in biofilm formation,” *Microbiology*, vol. 151, no. 5, pp. 1313–1323, 2005.

[69] K. Winzer, C. Falconer, N. C. Garber, S. P. Diggle, M. Camara, and P. Williams, “The *Pseudomonas aeruginosa* lectins PA-IL and PA-IIL are controlled by quorum sensing and by RpoS,” *Journal of Bacteriology*, vol. 182, no. 22, pp. 6401–6411, 2000.

[70] A. Rodriguez, Y. Quentin, A. Lazdunski, V. Méjean, and M. Foglino, “Two-component systems in *Pseudomonas aeruginosa*: why so many?” *Trends in Microbiology*, vol. 8, no. 11, pp. 498–504, 2000.

[71] M. D. Parkins, H. Ceri, and D. G. Storey, “Pseudomonas aeruginosa GacA, a factor in multihost virulence, is also essential for biofilm formation,” *Molecular Microbiology*, vol. 40, no. 5, pp. 1215–1226, 2001.

[72] A. Brencic, K. A. McFarland, H. R. McManus et al., “The GacS/GacA signal transduction system of Pseudomonas aeruginosa acts exclusively through its control over the transcription of the RsmY and RsmZ regulatory small RNAs,” *Molecular Microbiology*, vol. 73, no. 3, pp. 434–445, 2009.

[73] A. Brencic and S. Lory, “Determination of the regulon and identification of novel mRNA targets of Pseudomonas aeruginosa RsmA,” *Molecular Microbiology*, vol. 72, no. 3, pp. 612–632, 2009.

[74] Y. Irie, M. Starkey, A. N. Edwards, D. J. Wozniak, T. Romeo, and M. R. Parsek, “Pseudomonas aeruginosa biofilm matrix polysaccharide Psl is regulated transcriptionally by RpoS and post-transcriptionally by RsmA,” *Molecular Microbiology*, vol. 78, no. 1, pp. 158–172, 2010.

[75] E. Kay, B. Humair, V. Dénervaud et al., “Two GacA-dependent small RNAs modulate the quorum-sensing response in Pseudomonas aeruginosa,” *Journal of Bacteriology*, vol. 188, no. 16, pp. 6026–6033, 2006.

[76] G. Pessi, F. Williams, Z. Hindle et al., “The global posttranscriptional regulator RsmA modulates production of virulence determinants and N-acylhomoserine lactones in Pseudomonas aeruginosa,” *Journal of Bacteriology*, vol. 183, no. 22, pp. 6676–6683, 2002.

[77] C. Reimmann, M. Beyeler, A. Latifi et al., “The global activator GacA of Pseudomonas aeruginosa PAO positively controls the production of the autoinducer N-butyryl-homoserine lactone and the formation of the virulence factors pyocyanin, cyanide, and lipase,” *Molecular Microbiology*, vol. 24, no. 2, pp. 309–319, 1997.

[78] A. L. Goodman, B. Kulasekara, A. Rietsch, D. Boyd, R. S. Smith, and S. Lory, “A signaling network reciprocally regulates genes associated with acute infection and chronic persistence in Pseudomonas aeruginosa,” *Developmental Cell*, vol. 7, no. 5, pp. 745–754, 2004.

[79] W. Kong, L. Chen, J. Zhao et al., “Hybrid sensor kinase PA1611 in Pseudomonas aeruginosa regulates transitions between acute and chronic infection through direct interaction with RetS,” *Molecular Microbiology*, vol. 88, no. 4, pp. 784–797, 2013.

[80] I. Ventre, A. L. Goodman, I. Vallet-Gely et al., “Multiple sensors control reciprocal expression of Pseudomonas aeruginosa regulatory RNA and virulence genes,” *Proceedings of the National Academy of Sciences of the United States of America*, vol. 103, no. 1, pp. 171–176, 2006.

[81] H. Mikkelsen, R. McMullan, and A. Filloux, “The Pseudomonas aeruginosa reference strain PA14 displays increased virulence due to a mutation in ladS,” *PLoS ONE*, vol. 6, no. 12, Article ID e29113, 2011.

[82] A. L. Goodman, M. Merighi, M. Hyodo, I. Ventre, A. Filloux, and S. Lory, “Direct interaction between sensor kinase proteins mediates acute and chronic disease phenotypes in a bacterial pathogen,” *Genes and Development*, vol. 23, no. 2, pp. 249–259, 2009.

[83] A. R. Records and D. C. Gross, “Sensor kinases RetS and LadS regulate Pseudomonas syringae type VI secretion and virulence factors,” *Journal of Bacteriology*, vol. 192, no. 14, pp. 3584–3596, 2010.

[84] J. W. Hickman, D. F. Tifrea, and C. S. Harwood, “A chemosensory system that regulates biofilm formation through modulation of cyclic diguanylate levels,” *Proceedings of the National

[85] M. Merighi, V. T. Lee, M. Hyodo, Y. Hayakawa, and S. Lory, “The second messenger bis-(3′-5′)-cyclic-GMP and its PilZ domain-containing receptor Alg44 are required for alginate biosynthesis in Pseudomonas aeruginosa,” *Molecular Microbiology*, vol. 65, no. 4, pp. 876–895, 2007.

[86] R. Hengge, “Principles of c-di-GMP signalling in bacteria,” *Nature Reviews Microbiology*, vol. 7, no. 4, pp. 263–273, 2009.

[87] V. T. Lee, J. M. Matewish, J. L. Kessler, M. Hyodo, Y. Hayakawa, and S. Lory, “A cyclic-di-GMP receptor required for bacterial exopolysaccharide production,” *Molecular Microbiology*, vol. 65, no. 6, pp. 1474–1484, 2007.

[88] C. A. Janeway Jr. and R. Medzhitov, “Innate immune recognition,” *Annual Review of Immunology*, vol. 20, pp. 197–216, 2002.

[89] T. S. Walker, H. P. Bais, E. Déziel et al., “Pseudomonas aeruginosa-plant root interactions. Pathogenicity, biofilm formation, and root exudation,” *Plant Physiology*, vol. 134, no. 1, pp. 320–331, 2004.

[90] J. D. G. Jones and J. L. Dangl, “The plant immune system,” *Nature*, vol. 444, no. 7117, pp. 323–329, 2006.

[91] U. Mathesius, S. Mulders, M. Gao et al., “Extensive and specific responses of a eukaryote to bacterial quorum-sensing signals,” *Proceedings of the National Academy of Sciences of the United States of America*, vol. 100, no. 3, pp. 1444–1449, 2003.

[92] A. Hartmann and A. Schikora, “Quorum sensing of bacteria and trans-kingdom interactions of N-acyl homoserine lactones with eukaryotes,” *Journal of Chemical Ecology*, vol. 38, no. 6, pp. 704–713, 2012.

[93] A. Hartmann, M. Rothballer, B. A. Hense, and P. Schröder, “Bacterial quorum sensing compounds are important modulators of microbe-plant interactions,” *Frontiers in Plant Science*, vol. 5, article 131, 2014.

[94] M. G. Kocíolek, “Quorum-sensing inhibitors and biofilms,” *Anti-Infective Agents in Medicinal Chemistry*, vol. 8, no. 4, pp. 315–326, 2009.

[95] W. R. J. D. Galloway, J. T. Hodgkinson, S. Bowden, M. Welch, and D. R. Spring, “Applications of small molecule activators and inhibitors of quorum sensing in Gram-negative bacteria,” *Trends in Microbiology*, vol. 20, no. 9, pp. 449–458, 2012.

[96] B. LaSarre and M. J. Federle, “Exploiting quorum sensing to confuse bacterial pathogens,” *Microbiology and Molecular Biology Reviews*, vol. 77, no. 1, pp. 73–111, 2013.

[97] G. D. Geske, R. J. Wezeman, A. P. Siegel, and H. E. Blackwell, “Small molecule inhibitors of bacterial quorum sensing and biofilm formation,” *Journal of the American Chemical Society*, vol. 127, no. 37, pp. 12762–12763, 2005.

[98] K. M. Smith, Y. Bu, and H. Suga, “Library screening for synthetic agonists and antagonists of a Pseudomonas aeruginosa autoinducer,” *Chemistry and Biology*, vol. 10, no. 6, pp. 563–571, 2003.

[99] T. Ishida, T. Ikeda, N. Takiguchi, A. Kuroda, H. Ohtake, and J. Kato, “Inhibition of quorum sensing in Pseudomonas aeruginosa by N-acyl cyclopentylamides,” *Applied and Environmental Microbiology*, vol. 73, no. 10, pp. 3183–3188, 2007.

[100] M. Hentzer, K. Riedel, T. B. Rasmussen et al., “Inhibition of quorum sensing in Pseudomonas aeruginosa biofilm bacteria by a halogenated furanone compound,” *Microbiology*, vol. 148, no. 1, pp. 87–102, 2002.

[101] M. Hentzer, H. Wu, J. B. Andersen et al., “Attenuation of Pseudomonas aeruginosa virulence by quorum sensing inhibitors,” *EMBO Journal*, vol. 22, no. 15, pp. 3803–3815, 2003.

[102] H. Wu, Z. Song, M. Givskov, and N. Høiby, “Effects of quorum-sensing on immunoglobulin G responses in a rat model of chronic lung infection with *Pseudomonas aeruginosa*,” *Microbes and Infection*, vol. 6, no. 1, pp. 34–37, 2004.

[103] M. E. Skindersoe, M. Alhede, R. Phipps et al., “Effects of antibiotics on quorum sensing in *Pseudomonas aeruginosa*,” *Antimicrobial Agents and Chemotherapy*, vol. 52, no. 10, pp. 3648–3663, 2008.

[104] K. Tateda, R. Comte, J.-C. Pechere, T. Köhler, K. Yamaguchi, and C. van Delden, “Azithromycin inhibits quorum sensing in *Pseudomonas aeruginosa*,” *Antimicrobial Agents and Chemotherapy*, vol. 45, no. 6, pp. 1930–1933, 2001.

[105] T. Ichimiya, K. Takeoka, K. Hiramatsu, K. Hirai, T. Yamasaki, and M. Nasu, “The influence of azithromycin on the biofilm formation of *Pseudomonas aeruginosa* in vitro,” *Chemotherapy*, vol. 42, no. 3, pp. 186–191, 1996.

[106] J. C. Pechère, “Azithromycin reduces the production of virulence factors in *Pseudomonas aeruginosa* by inhibiting quorum sensing,” *Japanese Journal of Antibiotics*, vol. 54, pp. 87–89, 2001.

[107] D. Sofer, N. Gilboa-Garber, A. Belz, and N. C. Garber, “Sub-inhibitory erythromycin represses production of *Pseudomonas aeruginosa* lectins, autoinducer and virulence factors,” *Chemotherapy*, vol. 45, no. 5, pp. 335–341, 1999.

[108] S. Favre-Bonté, T. Köhler, and C. van Delden, “Biofilm formation by *Pseudomonas aeruginosa*: role of the C4-HSL cell-to-cell signal and inhibition by azithromycin,” *Journal of Antimicrobial Chemotherapy*, vol. 52, no. 4, pp. 598–604, 2003.

[109] I. Pérez-Martínez and D. Haas, “Azithromycin inhibits expression of the GacA-dependent small RNAs RsmY and RsmZ in *Pseudomonas aeruginosa*,” *Antimicrobial Agents and Chemotherapy*, vol. 55, no. 7, pp. 3399–3405, 2011.

[110] A. Bala, R. Kumar, and K. Harjai, “Inhibition of quorum sensing in *Pseudomonas aeruginosa* by azithromycin and its effectiveness in urinary tract infections,” *Journal of Medical Microbiology*, vol. 60, no. 3, pp. 300–306, 2011.

[111] T. B. Rasmussen, M. E. Skindersoe, T. Bjarnsholt et al., “Identity and effects of quorum-sensing inhibitors produced by *Penicillium* species,” *Microbiology*, vol. 151, no. 5, pp. 1325–1340, 2005.

[112] N. Glaser and H. Stopper, “Patulin: mechanism of genotoxicity,” *Food and Chemical Toxicology*, vol. 50, no. 5, pp. 1796–1801, 2012.

[113] M. E. Skindersoe, P. Ettinger-Epstein, T. B. Rasmussen, T. Bjarnsholt, R. de Nys, and M. Givskov, “Quorum sensing antagonism from marine organisms,” *Marine Biotechnology*, vol. 10, no. 1, pp. 56–63, 2008.

[114] S. S. Ebada, W. Lin, and P. Proksch, “Bioactive sesterterpenes and triterpenes from marine sponges: occurrence and pharmacological significance,” *Marine Drugs*, vol. 8, no. 2, pp. 313–346, 2010.

[115] J. Park, G. F. Kaufmann, J. P. Bowen, J. L. Arbiser, and K. D. Janda, “Solenopsin A, a venom alkaloid from the fire ant *Solenopsis invicta*, inhibits quorum-sensing signaling in *Pseudomonas aeruginosa*,” *Journal of Infectious Diseases*, vol. 198, no. 8, pp. 1198–1201, 2008.

[116] E. A. Ozer, A. Pezzulo, D. M. Shih et al., “Human and murine paraoxonase 1 are host modulators of *Pseudomonas aeruginosa* quorum-sensing,” *FEMS Microbiology Letters*, vol. 253, no. 1, pp. 29–37, 2005.

[117] F. Yang, L.-H. Wang, J. Wang, Y.-H. Dong, Y. H. Jiang, and L.-H. Zhang, “Quorum quenching enzyme activity is widely conserved in the sera of mammalian species,” *FEBS Letters*, vol. 579, no. 17, pp. 3713–3717, 2005.

[118] T. Rudrappa and H. P. Bais, “Curcumin, a known phenolic from *Curcuma longa*, attenuates the virulence of *Pseudomonas aeruginosa* PAO1 in whole plant and animal pathogenicity models,” *Journal of Agricultural and Food Chemistry*, vol. 56, no. 6, pp. 1955–1962, 2008.

[119] A. Annapoorani, V. Umamageswaran, R. Parameswari, S. K. Pandian, and A. V. Ravi, “Computational discovery of putative quorum sensing inhibitors against LasR and RhlR receptor proteins of *Pseudomonas aeruginosa*,” *Journal of Computer-Aided Molecular Design*, vol. 26, no. 9, pp. 1067–1077, 2012.

[120] S. Sarabhai, P. Sharma, and N. Capalash, “Ellagic acid derivatives from *Terminalia chebula* Retz. Downregulate the expression of quorum sensing genes to attenuate *Pseudomonas aeruginosa* PAO1 virulence,” *PLoS ONE*, vol. 8, no. 1, Article ID e53441, 2013.

[121] B. Girennavar, M. L. Cepeda, K. A. Soni et al., “Grapefruit juice and its furocoumarins inhibits autoinducer signaling and biofilm formation in bacteria,” *International Journal of Food Microbiology*, vol. 125, no. 2, pp. 204–208, 2008.

[122] Z. Zeng, L. Qian, L. Cao et al., “Virtual screening for novel quorum sensing inhibitors to eradicate biofilm formation of *Pseudomonas aeruginosa*,” *Applied Microbiology and Biotechnology*, vol. 79, no. 1, pp. 119–126, 2008.

[123] X. Ding, B. Yin, L. Qian et al., “Screening for novel quorum-sensing inhibitors to interfere with the formation of *Pseudomonas aeruginosa* biofilm,” *Journal of Medical Microbiology*, vol. 60, part 12, pp. 1827–1834, 2011.

[124] O. M. Vandeputte, M. Kiendrebeogo, S. Rajaonson et al., “Identification of catechin as one of the flavonoids from *Combretum albiflorum* bark extract that reduces the production of quorum-sensing-controlled virulence factors in *Pseudomonas aeruginosa* PAQ1,” *Applied and Environmental Microbiology*, vol. 76, no. 1, pp. 243–253, 2010.

[125] O. M. Vandeputte, M. Kiendrebeogo, T. Rasamiravaka et al., “The flavanone naringenin reduces the production of quorum sensing-controlled virulence factors in *pseudomonas aeruginosa* PAO1,” *Microbiology*, vol. 157, no. 7, pp. 2120–2132, 2011.

[126] T. Rasamiravaka, A. Jedrzejowski, M. Kiendrebeogo et al., “Endemic Malagasy *Dalbergia* species inhibit quorum sensing in *Pseudomonas aeruginosa* PAO1,” *Microbiology*, vol. 159, no. 5, pp. 924–938, 2013.

[127] C. A. Ta, M. Freundorfer, T. F. Mah et al., “Inhibition of bacterial quorum sensing and biofilm formation by extracts of neotropical rainforest plants,” *Planta Medica*, vol. 80, no. 4, pp. 343–350, 2014.

[128] L. Zhou, H. Zheng, Y. Tang, W. Yu, and Q. Gong, “Eugenol inhibits quorum sensing at sub-inhibitory concentrations,” *Biotechnology Letters*, vol. 35, no. 4, pp. 631–637, 2013.

[129] F. M. Husain, I. Ahmad, M. Asif, and Q. Tahseen, “Influence of clove oil on certain quorum-sensing-regulated functions and biofilm of *Pseudomonas aeruginosa* and *Aeromonas hydrophila*,” *Journal of Biosciences*, vol. 38, no. 5, pp. 835–844, 2013.

[130] T. H. Jakobsen, M. van Gennip, R. K. Phipps et al., “Ajoene, a sulfur-rich molecule from garlic, inhibits genes controlled by quorum sensing,” *Antimicrobial Agents and Chemotherapy*, vol. 56, no. 5, pp. 2314–2325, 2012.

[131] N. C. Cady, K. A. McKean, J. Behnke et al., “Inhibition of biofilm formation, quorum sensing and infection in *Pseudomonas aeruginosa* by natural products-inspired organosulfur compounds,” *PLoS ONE*, vol. 7, no. 6, Article ID e38492, 2012.

[132] R. W. Huigens III, J. J. Richards, G. Parise et al., “Inhibition of *Pseudomonas aeruginosa* biofilm formation with bromoageliferin analogues,” *Journal of the American Chemical Society*, vol. 129, no. 22, pp. 6966–6967, 2007.

[133] J. J. Richards, T. E. Ballard, R. W. Huigens III, and C. Melander, “Synthesis and screening of an oroidin library against *Pseudomonas aeruginosa* biofilms,” *ChemBioChem*, vol. 9, no. 8, pp. 1267–1279, 2008.

[134] M. A. Alkawash, J. S. Soothill, and N. L. Schiller, “Alginate lyase enhances antibiotic killing of mucoid *Pseudomonas aeruginosa* in biofilms,” *Acta Pathologica, Microbiologica, et Immunologica Scandinavica*, vol. 114, no. 2, pp. 131–138, 2006.

[135] J. W. Lampp and K. E. Griswold, “Alginate lyase exhibits catalysis-independent biofilm dispersion and antibiotic synergy,” *Antimicrobial Agents and Chemotherapy*, vol. 57, no. 1, pp. 137–145, 2013.

[136] L. Eckhart, H. Fischer, K. B. Barken, T. Tolker-Nielsen, and E. Tschachler, “DnaseL2 suppresses biofilm formation by *Pseudomonas aeruginosa* and *Staphylococcus aureus,*” *British Journal of Dermatology*, vol. 156, no. 6, pp. 1342–1345, 2007.

[137] G. V. Tetz, N. K. Artemenko, and V. V. Tetz, “Effect of DNase and antibiotics on biofilm characteristics,” *Antimicrobial Agents and Chemotherapy*, vol. 53, no. 3, pp. 1204–1209, 2009.

[138] H.-S. Kim and H.-D. Park, “Ginger extract inhibits biofilm formation by *Pseudomonas aeruginosa* PA14,” *PLoS ONE*, vol. 8, no. 9, Article ID e76106, 2013.

[139] L. Kumar, S. Chhibber, and K. Harjai, “Zingerone inhibit biofilm formation and improve antibiofilm efficacy of ciprofloxacin against *Pseudomonas aeruginosa* PAO1,” *Fitoterapia*, vol. 90, pp. 73–78, 2013.

[140] V. A. Carneiro, H. S. Dos Santos, F. V. S. Arruda et al., “Casbane diterpene as a promising natural antimicrobial agent against biofilm-associated infections,” *Molecules*, vol. 16, no. 1, pp. 190–201, 2011.

[141] D. Ren, R. Zuo, A. F. González-Barrios et al., “Differential gene expression for investigation of *Escherichia coli* biofilm inhibition by plant extract ursolic acid,” *Applied and Environmental Microbiology*, vol. 71, no. 7, pp. 4022–4034, 2005.

[142] J.-F. Hu, E. Garo, M. G. Goering et al., “Bacterial biofilm inhibitors from *Diospyros dendo,*” *Journal of Natural Products*, vol. 69, no. 1, pp. 118–120, 2006.

[143] M. I. Gómez and A. Prince, “Opportunistic infections in lung disease: *Pseudomonas* infections in cystic fibrosis,” *Current Opinion in Pharmacology*, vol. 7, no. 3, pp. 244–251, 2007.

[144] T. Bjarnsholt, K. Kirketerp-Møller, P. Ø. Jensen et al., “Why chronic wounds will not heal: a novel hypothesis,” *Wound Repair and Regeneration*, vol. 16, no. 1, pp. 2–10, 2008.

[145] T. Høgsberg, T. Bjarnsholt, J. S. Thomsen, and K. Kirketerp-Møller, “Success rate of split-thickness skin grafting of chronic venous leg ulcers depends on the presence of *Pseudomonas aeruginosa*: a retrospective study,” *PLoS ONE*, vol. 6, no. 5, Article ID e20492, 2011.

[146] T. S. Walker, K. L. Tomlin, G. S. Worthen et al., “Enhanced *Pseudomonas aeruginosa* biofilm development mediated by human neutrophils,” *Infection and Immunity*, vol. 73, no. 6, pp. 3693–3701, 2005.

[147] W.-C. Chiang, M. Nilsson, P. Ø. Jensen et al., “Extracellular DNA shields against aminoglycosides in *Pseudomonas aeruginosa* biofilms,” *Antimicrobial Agents and Chemotherapy*, vol. 57, no. 5, pp. 2352–2361, 2013.

[148] B. Mellbye and M. Schuster, “The sociomicrobiology of antivirulence drug resistance: a proof of concept,” *mBio*, vol. 2, no. 5, 2011.

[149] A. R. Smyth, P. M. Cifelli, C. A. Ortori et al., “Garlic as an inhibitor of *Pseudomonas aeruginosa* quorum sensing in cystic fibrosis—a pilot randomized controlled trial,” *Pediatric Pulmonology*, vol. 45, no. 4, pp. 356–362, 2010.

[150] R. T. LaLonde, L. Bu, A. Henwood, J. Fiumano, and L. Zhang, “Bromine-, chlorine-, and mixed halogen-substituted 4-methyl-2(5H)-furanones: synthesis and mutagenic effects of halogen and hydroxyl group replacements,” *Chemical Research in Toxicology*, vol. 10, no. 12, pp. 1427–1436, 1997.

[151] Y. Kim, N.-H. Nam, Y.-J. You, and B.-Z. Ahn, “Synthesis and cytotoxicity of 3,4-diaryl-2(5H)-furanones,” *Bioorganic and Medicinal Chemistry Letters*, vol. 12, no. 4, pp. 719–722, 2002.

[152] A. Beury-Cirou, M. Tannières, C. Minard et al., “At a supra-physiological concentration, human sexual hormones act as quorum-sensing inhibitors,” *PLoS ONE*, vol. 8, no. 12, Article ID e83564, 2013.

BioMed
Research International
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Stem Cells
International
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

International Journal of
Peptides
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Advances in
Virology
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

International Journal of
Genomics
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Journal of
Nucleic Acids
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

International Journal of
Zoology
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Journal of
Signal Transduction
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

The Scientific
World Journal
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Genetics
Research International
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Anatomy
Research International
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

International Journal of
Microbiology
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Advances in
Bioinformatics
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Archaea
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Enzyme
Research
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

International Journal of
Evolutionary Biology
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Molecular Biology
International
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014

Journal of
Marine Biology
Hindawi Publishing Corporation
http://www.hindawi.com
Volume 2014
