
Viruses 2015, 7, 2668-2682; doi:10.3390/v7052668

OPEN ACCESS

viruses

ISSN 1999-4915

www.mdpi.com/journal/viruses

Review

# Nucleocytoplasmic Shuttling of Influenza A Virus Proteins

Jing Li¹,†, Meng Yu¹,²,†, Weinan Zheng¹ and Wenjun Liu¹,²,*

1 Key Laboratory of Pathogenic Microbiology and Immunology, Institute of Microbiology, Chinese Academy of Sciences, Beijing 100101, China; E-Mails: lj418@163.com (J.L.); zhengdayumeng@163.com (M.Y.); zwn@foxmail.com (W.Z.)

2 University of Chinese Academy of Sciences, Beijing 100049, China

† These authors contributed equally to this work.

* Author to whom correspondence should be addressed; E-Mail: liuwj@im.ac.cn; Tel.: +86-10-6480-7497; Fax: +86-10-6480-7503.

Academic Editor: Alexander Ploss

Received: 24 December 2014 / Accepted: 20 May 2015 / Published: 22 May 2015

---

Abstract: Influenza viruses transcribe and replicate their genomes in the nuclei of infected host cells. The viral ribonucleoprotein (vRNP) complex of influenza virus is the essential genetic unit of the virus. The viral proteins play important roles in multiple processes, including virus structural maintenance, mediating nucleocytoplasmic shuttling of the vRNP complex, virus particle assembly, and budding. Nucleocytoplasmic shuttling of viral proteins occurs throughout the entire virus lifecycle. This review mainly focuses on matrix protein (M1), nucleoprotein (NP), nonstructural protein (NS1), and nuclear export protein (NEP), summarizing the mechanisms of their nucleocytoplasmic shuttling and the regulation of virus replication through their phosphorylation to further understand the regulation of nucleocytoplasmic shuttling in host adaptation of the viruses.

Keywords: influenza A virus; nucleocytoplasmic shuttling; virus replication

---

## 1. Introduction

Influenza A virus (IAV), which contains an eight-segment, single-stranded, negative-sense RNA genome, is a member of the Orthomyxovirus family. Further subtyping is based on the antigenicity of the hemagglutinin (HA) and neuraminidase (NA) surface glycoproteins. Wild waterfowl are the natural

reservoir of IAV, which can infect many kinds of mammals, including humans [1]. Currently, two major subtype (H1N1 and H3N2) viruses are circulating among humans. Although still an area of active research, the prevention and control of IAV infection mainly rely on vaccines, and the trivalent inactivated vaccine, which is recommended by the World Health Organization (WHO), is widely used.

## 2. Influenza A Virus Proteins

Influenza virions are roughly spherical, made up of a viral envelope, a matrix layer, and a central core. The envelope consists of HA and NA, usually in a 4:1 ratio [2], and the M2 protein acts as a proton channel. The oligomerization of M1 forms the matrix layer to stabilize the structure of the virion [3,4]. The trimeric polymerase complex (composed of PB1, PB2, and PA), nucleoprotein (NP), and viral RNA constitute the basic unit of the IAV genome for transcription and replication, which is called the viral RNA ribonucleoprotein (vRNP) [5]. In addition, IAV also encodes several other proteins, such as nuclear export protein (NEP), which participates in export of newly made vRNPs; nonstructural protein 1 (NS1), which mainly inhibits the type I interferon (IFN-I) system of host cells; and PB1-F2, PB1-N40, and PA-X, PA-N155, PA-N182, M42, and NS3 [6–11], which all play various roles in the virus lifecycle.

Unlike most RNA viruses, IAV replicates in the host nucleus [12]. In the early stage of infection, vRNPs must enter the nucleus for replication and transcription. Then, the newly synthesized proteins translocate back into the nucleus to help in the assembly and nuclear export of progeny vRNPs. Finally, the formation of mature virions and budding occurs. Thus, nucleocytoplasmic shuttling is important throughout the entire IAV lifecycle. The transport of proteins, as well as cellular and viral RNPs, from the cytoplasm to the nucleus (and vice versa) is an active, energy-dependent, signal-mediated process. Taking into account that virus RNAs transport as RNPs rather than naked RNAs, RNA-binding proteins play a key role in RNA transport. Most of these proteins both bind RNA and contain sequences that allow them to be exported/imported between the cytoplasm and nucleus. In this review, we discuss key findings that have advanced our understanding of the nucleocytoplasmic shuttling of influenza virus proteins (especially for the M1, NP, and NS protein) and their influence on virus replication.

## 3. Nucleocytoplasmic Shuttling Mechanism of IAV Proteins

In eukaryotic cells, the transport of proteins and RNAs into and out of the nucleus through nuclear pore complexes (NPCs)—large macromolecular structures embedded in the double membrane of the nuclear envelope—requires the help of transporter molecules and is an active process. There are two types of transporter molecules, named importin and exportin based on their functions. The importins, α and β, both belong to the importin β super-family. The former is able to recognize and bind the nuclear location signals (NLSs) in the target protein, and the complex is recognized and bound by importin β, which subsequently interacts with the NPC to accomplish the translocation of target proteins. Like the importins, exportins are responsible for ferrying cargo out of the nucleus via the same working principle. Chromosome region maintenance protein 1 (CRM1) is an exportin that recognizes and binds to target proteins carrying a nuclear export signal (NES) and mediates their export from the nucleus [13]. The classic NLSs recognized by importin α consist of two classes. The first is represented by the NLS of the Simian virus 40 (SV40) Large T-antigen (LT) protein and is composed of several contiguous basic amino acids ($^{126}$PKKKRKV$^{132}$, the basic amino acids are shown in bold). The second category is

represented by the NLS of nucleoplasmin, which is characterized by continuous basic amino acid sequences at both ends separated by >10 random amino acid residues (¹⁵⁶KRPAATKKAGQAKKKK¹⁷¹, the basic amino acids are shown in bold) [14–16]. There are many subtypes in the importin α family, with different cargo specificities for different NLSs. Additionally, the conformation and phosphorylation of target proteins' NLSs are also associated with importin α [17]. The NES recognized by CRM1 is composed of hydrophobic amino acids with larger side chains that appear at regular intervals. Its characteristic sequence is Φ-X(2-3)-Φ-X-Φ, where Φ stands for Leu, Val, Ile, Met, or Phe, and X can be any amino acid [18].

After the first NLS of SV40 LT was identified in 1984, functional NLSs and NESs were subsequently found in influenza virus proteins, including PB1, PB2, PA, NP, M1, NS1, and NEP (Table 1). The proteins with NLSs can be recognized and bound by importin α for nuclear entry. Another non-classical nuclear import factor, Ran binding protein 5 (RanBP5), interacts with PB1-PA dimer to transport into the nucleus [19, 20]. After the completion of replication, vRNPs require the help of two viral proteins, M1 and NEP, which contain the NESs for recognition by CRM1 to move out of the nucleus for assembly and budding of progeny viruses [21–24]. The following section describes the nucleocytoplasmic shuttling mechanism of four IAV proteins (M1, NP, NS1, and NEP) in detail.

### 3.1. Nucleocytoplasmic Shuttling of M1

M1 is the most abundant protein in the virion and forms the matrix layer inside the virus envelope. The exterior matrix layer combines with HA, NA, and M2 located on the envelope, and the interior matrix layer combines with vRNP. Both layers form the virion structure. Initially, it was thought that M1 interacted with HA and NA inside the cytoplasm. Subsequently, the M1 protein was found to enter the nucleus and be indispensable for vRNP nuclear export [25]. Aside from the well-characterized basic amino acid-rich NLS [26], there is another basic amino acid stretch at position 76–78 that is highly conserved among influenza A and B virus M1 proteins and plays a critical role in virus replication. Indeed, there is evidence showing that R77A or R78A substitutions in M1 result in aberrant M1 intracellular localization and defects in virus assembly and budding [27]. The viral polymerase proteins (PA, PB1, and PB2) are imported into the nucleus to assemble vRNPs, together with NP and viral RNA. It is well established that vRNP nuclear export is mediated by the nuclear presence of M1 and NEP. In the absence of a functional NES, M1 is not able to interact with CRM1 itself. NEP likely bridges the M1 and CRM1 interaction by binding to M1 with its C-terminus, while the two N-terminal NESs interact with CRM1 and allow access to the CRM1-dependent export pathway (Figure 1). Our previous experiments show that intracellular distribution of M1 is not sensitive to the CRM1 inhibitor leptomycin B (LMB), further confirming the indispensable role of NEP for vRNP nuclear export. Furthermore, the M1 individual nuclear export was specially dependent on its NES and critical for influenza A virus replication [24].

| Protein | NLS | NES | References |
|---------|-----|-----|------------|
| NS1     | NLS 1: ${}^{34}$RLRR${}^{38}$, highly conserved<br>NLS 2: ${}^{216}$PKQKRK${}^{221}$, two NLSs act independently | ${}^{138}$FDRLETLILL${}^{147}$ | [28]<br>[29] |
| PB1     | ${}^{187}$RKRRVRDNMTKKMVTQRTIGKRKQR${}^{211}$, bipartite NLS |  | [30]<br>[31] |
| PB2     | NLS 1: ${}^{449}$GIESIDNVVMGMIGILPDMTPSTEMSMRGVRIKMGVD ETSSAEKIV${}^{495}$, required for efficient import<br>NLS 2: ${}^{736}$KRKR${}^{739}$, bipartite NLS K736 required for efficient import |  | [32]<br>[33] |
| PA      | NLS 1: ${}^{124}$RREVHIYYLEKANKIK${}^{139}$, bipartite NLS<br>NLS 2: 186–247<br>E154 required for efficient import |  | [34] |
| M1      | ${}^{101}$RKLKR${}^{105}$ | ${}^{59}$ILGFVFTLTV${}^{68}$L66A,<br>V68A mutation impairs<br>M1 nuclear export | [26]<br>[24] |
| NP      | NLS 1: ${}^{3}$TKGTKRSYEQM${}^{13}$, unconventional NLS, 3Ser crucial for N-terminal phosphorylation<br>NLS 2: ${}^{198}$KGINDRNFWRGENGRRTR${}^{216}$, bipartite NLS | NES 1: ${}^{24}$EIRASVGKMIDGIGRFYIQMCTELKL${}^{49}$<br>NES 2: ${}^{183}$VKGVGTMVMELIRMI${}^{197}$<br>NES 3: ${}^{248}$PGNAEFEDLIFLARSALILRGSVAHKS${}^{274}$ | [37]<br>[38]<br>[39]<br>[40]<br>[41]<br>[42] |
| NEP     | Passive diffusion, no need of NLS | NES 1: ${}^{12}$ILMRMSKMQL${}^{21}$<br>NES 2: ${}^{31}$IITQFESLKI${}^{40}$ | [21]<br>[43]<br>[44] |

Cell
membrane

Cytoplasm

Nuclear membrane

Nucleus

Early

RanBP5

NPC

RanBP5

RanBP5

Ran

Late

HA

NA

M2

PA

PB1

NP

NEP

M1

NLS

NES

Figure 1. Model of the nucleocytoplasmic shuttling of influenza virus proteins and vRNPs.

The importin/exportin molecules recognize and bind the NLSs/NESs in the target proteins and then are bound by the importin/exportin molecule, which subsequently interacts with the NPC to accomplish the nucleocytoplasmic shuttling. The importins α-β and RanBP5 pathway separately involved into the vRNPs import. CRM1 is a commonly utilized exportin for influenza virus vRNPs. The export of IAV RNPs seems to depend on a “RanGTP-CRM1-NEP-M1-RNP” model for transport out of the nucleus as a complex.

3.2. Nucleocytoplasmic Shuttling of NP

There are at least two NLSs in NP, including the unconventional NLS1 (3–13), which interacts with importin α [36], and NLS2 (198–216) [39], which together with NLS1 was originally hypothesized to be a traditional bipartite NLS. However, the crystal structure of NP shows that the two clusters are too close to be recognized by importins as a bipartite NLS [45]. Although it is hard to judge which of the two NLSs contributes most to the import of vRNP, import is almost completely inhibited when both NLSs are blocked using antibodies [46,47]. In addition, *in vitro* experiments and a reverse genetics approach reveal that NLS1 plays a pivotal role in mediating vRNP nuclear import [48,49].

Three NESs have been identified in NP that have the familiar sequence characteristics of common export signals. Two of them do not rely on CRM1 for export, though the third does. Further, all three

must be simultaneously mutated to inhibit NP nuclear export, proving their functional independence [42]. The three NESs are closely related to the nuclear export of vRNP, and mutation of the NP NESs inhibits the replication of the virus in the post-transcriptional stage. Through function comparison via mutation of all seven NESs in IAV proteins, it was found that NP NES 3 played a key role in the replication of the virus, but the mechanism for this phenomenon has not been elucidated [50].

### 3.3. Nucleocytoplasmic Shuttling of NS1

NS1 is mainly located inside the nucleus and transports to the cytoplasm in the late stage of infection. Two NLSs and one NES have been identified in NS1 [28]. S221 in NLS2 is very important for the nuclear localization of NS1. Further, the nuclear export of NS1 does not (or only partially) rely on CRM1 [51]. By comparison among different strains, it was found that the NS1 NES is highly conserved, and mutations of conserved residues, such as L141A and A149V, lead to the retention of NS1 inside the nucleus during the late stage of infection [52].

### 3.4. Nucleocytoplasmic Shuttling of NEP

NEP is known as a nuclear export protein. It may not need a NLS as it is only 14 kDa in size, and at present, no NLS has been found in NEP. Through the use of an inhibitor of NLS-dependent active nuclear transport and an energy depletion assay, it was demonstrated that NEP enters the nucleus by passive diffusion [44].

NEP is involved in the nuclear export of vRNPs [53]. NEP is able to interact with CRM1 in mammalian two-hybrid systems, suggesting that the nuclear export of NEP depends on binding to CRM1 [54]. Further, it has been demonstrated that the NES1 and NES2 of NEP [43] are CRM1 dependent and jointly participate in the nuclear export of NEP and vRNP. Interestingly, the NEPs of different strains result in distinct cellular distribution patterns, forming unique nuclear aggregates, with the weaker binding capacity to CRM1 due to one amino acid change in the NES, perhaps facilitating the export of the vRNP complex [44].

## 4. Biological Significance of Nucleocytoplasmic Shuttling on IAV Replication

### 4.1. Interactions among Members of the vRNP Nucleocytoplasmic Shuttling Complex

The vRNP is the basic unit for the replication and transcription of IAV. The interaction between members of the vRNP complex is of great significance for the structural stability and function of the complex. Several functional domains of related proteins involved in the interaction with the members of the polymerase complex [55, 56]. For example, several binding sites on NP have been identified, including two NP-NP interaction regions (aa 189–358 and 371–469), two RNA interaction regions (aa 1–77 and 79–180), two PB2 interaction regions (aa 1–166 and 255–498), and three polymerase interaction regions (R204, W207, and R208) [57, 58]. As with the relevant interaction regions in the trimeric polymerase, these regions support the formation of the vRNP and are required for its function. It has been found that disturbing the self-polymerization of NP and destroying the interaction between NP and PB2 or NP and PB1 regulate the replication of influenza virus in a negative way [59].

C-terminus of NEP also interacts with PB2 and PB1 in the viral polymerase complex to regulate the activity of the polymerase, resulting in alteration of the synthesis of vRNA [60].

In the process of infection, IAV enters the host cytoplasm, and at the same time, the M2 ion channel opens, internally acidifying the virus. Subsequently, the matrix layer depolymerizes, and as a result, M1 and vRNP dissociate from each other [55]. This conformational change results in the exposure of the NLS, which leads to the entry of the vRNP into the nucleus for replication and transcription. Each polymerase subunit and NP contain their own NLSs, and it is possible that the monomers are transported into the nucleus (for NP and PB2) or that PB1-PA dimers or trimers form in the cytoplasm before nuclear import, perhaps as part of a mechanism regulating different phases of the infection [61]. It has been intriguing trying to identify which NLSs play a decisive role in transport into the nucleus. Indeed, structure-function experiments demonstrate that the role of NLS1 is very important in mediating vRNP nuclear localization [62].

For vRNP nuclear export, a “CRM1-NEP-M1-vRNP” complex forms, where CRM1 recognizes and binds to the NES on NEP, accompanied by GTP hydrolysis (Figure 1). Thus, the nuclear export of vRNP depends on CRM1 [63]. Although NP contains an NES and can interact with CRM1, the NP/CRM1 interaction itself fails to induce the hydrolysis of RanGTP, suggesting that the NESs on NP cannot directly mediate vRNP nuclear export [23]. Indeed, M1 and NEP are necessary for this process [21, 25]. M1 is exported from the nucleus via a CRM1-independent pathway, while all of the NESs of NEP are CRM1 dependent, suggesting that the NES on NEP directly mediates the nuclear export of vRNP. However, NES mutation on M1 leads to the nuclear retention of M1 and the accumulation of NEP, in addition to significantly reducing the virus titer, indicating that M1 plays an important role in vRNP nuclear export [24]. When the vRNP translocates out of the nucleus, the C-terminus of NEP binds to the N-terminus of M1, covering the NLS on M1 and inhibiting vRNP re-entry into the nucleus; mutation of NEP W78 destroys the interaction between NEP and M1 [23]. Experiments also suggest that the ability of NEP to support the nuclear export of vRNPs by stabilizing the interaction between M1 and vRNPs requires the NES-containing N-terminus, as well as the last three amino acids of NEP, and is functionally linked to the polymerase activity-enhancing function of NEP during viral genome replication [64].

### 4.2. Phosphorylation of IAV Proteins to Regulate Nucleocytoplasmic Shuttling

The phosphorylation of IAV proteins is an important regulatory mechanism that influences viral replication by affecting the nucleocytoplasmic shuttling of both the individual viral proteins themselves and the vRNP complex. Nuclear transport of M1 depends on phosphorylation mediated by protein kinase C (PKC), as PKC inhibition effectively decreases virus replication [65]. Mutation of the M1 Y132 phosphorylation site in A/WSN/1933 (H1N1) eliminates the interaction between M1 and importin α1, inhibiting M1 nuclear import, and thus having a negative influence on virus replication. JAK kinase inhibition can block this phosphorylation and impair WSN M1 entry into the nucleus but not H9N2 M1, suggesting that differential regulation of the interaction between M1 and importins exists in different subtypes of IAVs [66] (Figure 2). There are additional potential phosphorylation sites in the IAV M1 protein, such as T108, that are close to the NLS but whose functions have not been determined [67].

Viruses 2015, 7

![Diagram](attachment:diagram.png)

Figure 2. Model of Y132 phosphorylation participating in nuclear import of IAV M1. The conserved tyrosine 132 of M1 is a phosphorylation site. The NLS-neighboring phosphorylated Y132 residue is crucial for the cytoplasmic-nuclear translocation of M1 by increasing the binding capacity of M1 and nuclear import factor importin-α1, subsequently affecting the replication of IAV.

NP is rich in serine residues, which is crucial for its phosphorylation, especially on the N-terminus of NLS1 [37]. Inhibiting phosphorylation of infected cells stabilizes the localization of NP in the nucleus and inhibits its nuclear export [38,68]. Several highly conserved phosphorylation sites have been identified in NP, such as S9/Y10 and S165. Mutation of the former significantly decreases viral titers [67], and the S165D mutation (which mimics phosphorylation) may maintain NP in a monomeric state, reducing its affinity for RNA [69]. In our lab, we demonstrated that the phosphorylation of S9, Y10, and Y296 was essential for virus growth in cell culture and modulation of activity of the viral polymerase in a mouse model and the nuclear-cytoplasmic shuttling of NP. The phosphorylation and de-phosphorylation of S9 and Y10 controlled nuclear import of NP by affecting the binding affinity between NP and importin-α. In addition, the phosphorylation of Y296 caused nuclear retention of NP by reducing the interaction between NP and CRM1 [70].

The phosphorylation of T215 in NS1 is mediated by the CDK/ERK pathway, and the T215A mutation attenuates virus replication [71]. The functional analysis of several phosphorylation sites, including T215, S48, and S42, shows that the S42D mutation eliminates dsRNA binding by NS1, also attenuating virus replication [72].

# 5. Conclusions

To adapt to new host species, influenza viruses must overcome significant barriers to cross-species transmission and replication. The changes in the receptor-binding properties of the viral HA to adapt to new host species are well characterized, but the nuclear import machinery to determine the host tropism requires a better understanding. Previous studies show that the preference of PB2 for different importin-α isoforms (α1, α3, α4, α5, α6, and α7) differs between hosts. For efficient virus replication in different hosts, all viruses depend on importin-α1. Meanwhile, the polymerase subunit PB2 of avian viruses requires importin-α3, PB2 of mammalian virus shows importin-α7 specificity, and H1N1v replication depends on both, suggesting ongoing adaptation of this virus [73]. PB2 specificity for given importin-α isoform is mediated by both NLS and protein context. Researchers have found out PB2 627-NLS domain can induce selective importin binding in host-specific manner and areas where more research is required, including other transport associated protein in IAV. The specific interaction of the polymerase subunit PB2 and the NP of mammalian viruses with importin-α3/7 regulates the efficacy of viral replication and determine host range underlining the importance of the nuclear envelope in interspecies transmission [73–75]. Work in our laboratory also demonstrates differences in the nucleocytoplasmic shuttling of NPs between human-origin WSN and avian-origin IAVs, which impact the replication, adaptation, and pathogenicity of the viruses (unpublished). In addition, we have also found that the CA04 strain displays different vRNP distribution patterns, forming unique nuclear aggregates, with respect to WSN by comparing different strains of H1N1 NEP for vRNP nuclear translocation. In the further study, we confirmed that CA04 NEP interacts less efficiently with CRM1 and site T48 is responsible for the nuclear aggregation [44]. In view of NEP playing an important role in viral polymerase activity and vRNP nuclear transport efficiency, these differences may be one reason for divergence of the pathogenicity and host adaptation range between these virus strains. Future work will dissect the finely tuned interactions induced by natural/pressure selection to acquire cross-species activity.

Phosphorylation is one of the most common and efficient post-translational modification methods in organisms, playing an important role in regulating many metabolic pathways. Nucleocytoplasmic shuttling of proteins and phosphorylation are closely related. Phosphorylation and dephosphorylation can strengthen or weaken the nucleocytoplasmic shuttling of a protein through a variety of mechanisms. For instance, phosphorylation of key NLS or NES residues on viral proteins can strengthen the affinity between the proteins and their nuclear transport receptors [76]. Alternatively, phosphorylation can strengthen the ability of a protein to separate itself from the NPC. In addition, phosphorylation can lead to a conformational change in the protein, selectively exposing its NLS or NES. In contrast, phosphorylation of the NPC or virus proteins can also inhibit nucleocytoplasmic shuttling. Regardless, research into the mechanism of how phosphorylation regulates nucleocytoplasmic shuttling is still at an early stage. Development of better quantitative methods to determine the influence of phosphorylation on the dynamic processes of protein transport will lead to a more comprehensive understanding of the

role of phosphorylation in the IAV life cycle, which is beneficial for influenza prevention and control. In addition, sumoylation is critical for protein-protein interactions in many biological systems. For example, the sumoylation of matrix protein M1 modulate the assembly and morphogenesis of influenza A virus [77, 78], or sumoylation of NP is indispensable for virus growth and intracellular trafficking [79]. Further research found out that the sumoylation of M1 promotes M1-mediated vRNP nuclear export to increase viral replication [80]. Thereby, the modification of virus/cellular protein to further regulate the virus life cycle needs to be given more attention for further research.

# Acknowledgments

This work was supported by grants from the National Natural Science Foundation of China (NSFC) (Grant No. 31402216), the National Key Technologies Research and Development Program of China (2013ZX10004-610), China Ministry of Science and Technology (MOST) Project 973 (Grant Nos. 2012CB955501 and 2011CB504705), the Key Research Program of the Chinese Academy of Sciences (KSZD-EW-Z-005-001), and IDRC-APEIR Project Funding. W.J.L. is a principal investigator of the NSFC Innovative Research Group (Grant No. 81321063).

# Author Contributions

J.L., M.Y., W.N.Z., and W.J.L. contributed to the content and editing of this article.

# Conflicts of Interest

The authors declare no conflict of interest.

# References

1. Nelson, M.I.; Holmes, E.C. The evolution of epidemic influenza. *Nat. Rev. Genet.* **2007**, *8*, 196–205.
2. Kilbourne, E.D.; Johansson, B.E.; Grajower, B. Independent and disparate evolution in nature of influenza A virus hemagglutinin and neuraminidase glycoproteins. *Proc. Natl. Acad. Sci. USA* **1990**, *87*, 786–790.
3. Nayak, D.P.; Balogun, R.A.; Yamada, H.; Zhou, Z.H.; Barman, S. Influenza virus morphogenesis and budding. *Virus Res.* **2009**, *143*, 147–161.
4. Coloma, R.; Valpuesta, J.M.; Arranz, R.; Carrascosa, J.L.; Ortin, J.; Benito, J.M. The structure of a biologically active influenza virus ribonucleoprotein complex. *PLoS Pathog.* **2009**, *5*, 1–10.
5. Klumpp, K.; Ruigrok, R.W.H.; Baudin, F. Roles of the influenza virus polymerase and nucleoprotein in forming a functional RNP structure. *EMBO J.* **1997**, *16*, 1248–1257.
6. Selman, M.; Dankar, S.K.; Forbes, N.E.; Jia, J.-J.; Brown, E.G. Adaptive mutation in influenza A virus non-structural gene is linked to host switching and induces a novel protein by alternative splicing. *Emerg. Microbes Infect.* **2012**, *1*, e42.

7. Wise, H.M.; Hutchinson, E.C.; Jagger, B.W.; Stuart, A.D.; Kang, Z.H.; Robb, N.; Schwartzman, L.M.; Kash, J.C.; Fodor, E.; Firth, A.E.; *et al.* Identification of a novel splice variant form of the influenza A virus M2 ion channel with an antigenically distinct ectodomain. *PLoS Pathog.* 2012, 8, e1002998.

8. Jagger, B.W.; Wise, H.M.; Kash, J.C.; Walters, K.A.; Wills, N.M.; Xiao, Y.L.; Dunfee, R.L.; Schwartzman, L.M.; Ozinsky, A.; Bell, G.L.; *et al.* An overlapping protein-coding region in influenza A virus segment 3 modulates the host response. *Science* 2012, 337, 199–204.

9. Wise, H.M.; Foeglein, A.; Sun, J.; Dalton, R.M.; Patel, S.; Howard, W.; Anderson, E.C.; Barclay, W.S.; Digard, P. A complicated message: Identification of a novel PB1-related protein translated from influenza A virus segment 2 mRNA. *J. Virol.* 2009, 83, 8021–8031.

10. Muramoto, Y.; Noda, T.; Kawakami, E.; Akkina, R.; Kawaoka, Y. Identification of novel influenza A virus proteins translated from PA mRNA. *J. Virol.* 2013, 87, 2455–2462.

11. Chen, W.; Calvo, P.A.; Amalide, D.; Gibbs, J.; Schubert, U.; Bacik, I.; Basta, S.; O'Neill, R.; Schickli, J.; Palese, P.; *et al.* A novel influenza A virus mitochondrial protein that induces cell death. *Nat. Med.* 2001, 7, 1306–1312.

12. Stegmann, T.; M.Whitel, J.; Helenius, A. Intermediates in influenza induced membrane fusion. *EMBO J.* 1990, 9, 4231–4241.

13. Pemberton, L.F.; Blobelt, G.; Rosenblum, J.S. Transport routes through the nuclear pore complex. *Curr. Opin. Cell Biol.* 1998, 10, 392–399.

14. Makkerh, J.P.S.; Dingwall, C.; Laskey, R.A. Comparative mutagenesis of nuclear localization signals reveals the importance of neutral and acidic amino acids. *Curr. Biol.* 1996, 6, 1025–1027.

15. Dingwall, C.; Laskey, R.A. Nuclear import: A tale of two sites. *Curr. Biol.* 1998, 8, 922–924.

16. Wang, Y.E.; Park, A.; Lake, M.; Pentecost, M.; Torres, B.; Yun, T.E.; Wolf, M.C.; Holbrook, M.R.; Freiberg, A.N.; Lee, B. Ubiquitin-regulated nuclear-cytoplasmic trafficking of the nipah virus matrix protein is important for viral budding. *PLoS Pathog.* 2010, 6, e1001186.

17. Jans, D.A.; Hubner, S. Regulation of protein transport to the nucleus: Central role of phosphorylation. *Physiol. Rev.* 1996, 76, 651–685.

18. Henderson, B.R.; Eleftheriou, A. A comparison of the activity, sequence specificity, and CRM1-dependence of different nuclear export signals. *Exp. Cell Res.* 2000, 256, 213–224.

19. Fodor, E.; Smith, M. The PA subunit is required for efficient nuclear accumulation of the PB1 subunit of the influenza A virus RNA polymerase complex. *J. Virol.* 2004, 78, 9144–9153.

20. Deng, T.; Engelhardt, O.G.; Thomas, B.; Akoulitchev, A.V.; Brownlee, G.G.; Fodor, E. Role of ran binding protein 5 in nuclear import and assembly of the influenza virus RNA polymerase complex. *J. Virol.* 2006, 80, 11911–11919.

21. E.O'Neill, R.; Talon, J.; Palese, P. The influenza virus NEP (NS2 protein) mediates the nuclear export of viral ribonucleoproteins. *EMBO J.* 1998, 17, 288–296.

22. Bancroft, C.T.; Parslow, T.G. Evidence for segment-nonspecific packaging of the influenza A virus genome. *J. Virol.* 2002, 76, 7133–7139.

23. Akarsu, H.; Burmeister, W.P.; Petosa, C.; Petit, I. Crystal structure of the M1 protein-binding domain of the influenza A virus nuclear export protein (NEP/NS2). *EMBO J.* 2003, 22, 4646–4655.

24. Cao, S.; Liu, X.; Yu, M.; Li, J.; Jia, X.; Bi, Y.; Sun, L.; Gao, G.F.; Liu, W. A nuclear export signal in the matrix protein of influenza A virus is required for efficient virus replication. *J. Virol.* 2012, 86, 4883–4891.

25. Sakaguchi, A.; Hirayama, E.; Hiraki, A.; Ishida, Y.-I.; Kim, J. Nuclear export of influenza viral ribonucleoprotein is temperature-dependently inhibited by dissociation of viral matrix protein. *Virology* 2003, 306, 244–253.

26. Ye, Z.; Robinson, D.; Wagner, R.R. Nucleus-targeting domain of the matrix protein (M1) of influenza virus. *J. Virol.* 1995, 69, 1964–1970.

27. Das, S.C.; Watanabe, S.; Hatta, M.; Noda, T.; Neumann, G.; Ozawa, M.; Kawaoka, Y. The highly conserved arginine residues at positions 76 through 78 of influenza A virus matrix protein M1 play an important role in viral replication by affecting the intracellular localization of M1. *J. Virol.* 2012, 86, 1522–1530.

28. Greenspan, D.; Palese, P.; Krystal, M. Two nuclear location signals in the influenza virus NS1 nonstructural protein. *J. Virol.* 1988, 62, 3020–3026.

29. Li, Y.; Yamakita, Y.; Krug, R.M. Regulation of a nuclear export signal by an adjacent inhibitory sequence: The effector domain of the influenza virus NS1 protein. *Proc. Natl. Acad. Sci. USA* 1998, 95, 4864–4869.

30. Nath, S.T.; Nayak, D.P. Function of two discrete regions is required for nuclear localization of polymerase basic protein 1 of A/WSN/33 influenza virus (H1N1). *Mol. Cell. Biol.* 1990, 10, 4139–4145.

31. Hutchinson, E.C.; Orr, O.E.; Man Liu, S.; Engelhardt, O.G.; Fodor, E. Characterization of the interaction between the influenza A virus polymerase subunit PB1 and the host nuclear import factor Ran-binding protein 5. *J. Gen. Virol.* 2011, 92, 1859–1869.

32. Mukaigawa, J.; Nayak, D.P. Two signals mediate nuclear localization of influenza virus (A/WSN/33) polymerase basic protein 2. *J. Virol.* 1991, 65, 245–253.

33. Tarendeau, F.; Boudet, J.; Guilligay, D.; Mas, P.J.; Bougault, C.M.; Boulo, S.; Baudin, F.; Ruigrok, R.W.; Daigle, N.; Ellenberg, J.; *et al.* Structure and nuclear import function of the C-terminal domain of influenza virus polymerase PB2 subunit. *Nat. Struct. Mol. Biol.* 2007, 14, 229–233.

34. Nieto, A.; Luna, S.D.L.; Bfircena, J.; Portela, A.; Ortin, J. Complex structure of the nuclear translocation signal of influenza virus polymerase PA subunit. *J. Gen. Virol.* 1994, 75, 29–36.

35. Fau, D.J.; Hurtley, S.M.; Warren, G. Reconstitution of an endocytic fusion event in a cell-free system. *Cell* 1985, 43, 643–652.

36. Wang, P.; Palese, P.; O’Neill, R.E. The NPI-1/NPI-3 (karyopherin alpha) binding site on the influenza A virus nucleoprotein NP is a nonconventional nuclear localization signal. *J. Virol.* 1997, 71, 1850–1856.

37. Arrese, M.; Portela, A. Serine 3 is critical for phosphorylation at the N-terminal end of the nucleoprotein of influenza virus A/Victoria/3/75. *J. Virol.* 1996, 70, 3385–3391.

38. Neumann, G.; Castrucci, M.R.; Kawaoka, Y. Nuclear import and export of influenza virus nucleoprotein. *J. Virol.* 1997, 71, 9690–9700.

39. Weber, F.; Kochs, G.; Gruber, S.; Haller, O. A classical bipartite nuclear localization signal on Thogoto and influenza A virus nucleoproteins. *Virology* 1998, 250, 9–18.

Viruses 2015, 7

40. Bullido, R.; Gomez-Puertas, P.; Albo, C.; Portela, A. Several protein regions contribute to determine the nuclear and cytoplasmic localization of the influenza A virus nucleoprotein. J. Gen. Virol. 2000, 81, 135–142.

41. Wu, W.W.; Pante, N. The directionality of the nuclear transport of the influenza A genome is driven by selective exposure of nuclear localization sequences on nucleoprotein. Virol. J. 2009, 6, e68.

42. Yu, M.; Liu, X.; Cao, S.; Zhao, Z.; Zhang, K.; Xie, Q.; Chen, C.; Gao, S.; Bi, Y.; Sun, L.; et al. Identification and characterization of three novel nuclear export signals in the influenza A virus nucleoprotein. J. Virol. 2012, 86, 4970–4980.

43. Huang, S.; Chen, J.; Chen, Q.; Wang, H.; Yao, Y.; Chen, J.; Chena, Z. A second CRM1-dependent nuclear export signal in the influenza A virus NS2 protein contributes to the nuclear export of viral ribonucleoproteins. J. Virol. 2013, 87, 767–778.

44. Gao, S.; Wang, S.; Cao, S.; Sun, L.; Li, J.; Bi, Y.; Gao, G.F.; Liu, W. Characteristics of nucleocytoplasmic transport of H1N1 influenza A virus nuclear export protein. J. Virol. 2014, 88, 7455–7463.

45. Ye, Q.; Krug, R.M.; Tao, Y.J. The mechanism by which influenza A virus nucleoprotein forms oligomers and binds RNA. Nature 2006, 444, 1078–1082.

46. Zhang, H.; Hale, B.G.; Xu, K.; Sun, B. Viral and host factors required for avian H5N1 influenza A virus replication in mammalian cells. Viruses 2013, 5, 1431–1446.

47. Wu, W.W.; Sun, Y.H.; Pante, N. Nuclear import of influenza a viral ribonucleoprotein complexes is mediated by two nuclear localization sequences on viral nucleoprotein. Virol. J. 2007, 4, e49.

48. Cros, J.F.; Garcia-Sastre, A.; Palese, P. An unconventional NLS is critical for the nuclear import of the influenza A virus nucleoprotein and ribonucleoprotein. Traffic 2005, 6, 205–213.

49. Ozawa, M.; Fujii, K.; Muramoto, Y.; Yamada, S.; Yamayoshi, S.; Takada, A.; Goto, H.; Horimoto, T.; Kawaoka, Y. Contributions of two nuclear localization signals of influenza A virus nucleoprotein to viral replication. J. Virol. 2007, 81, 30–41.

50. Chutiwitoonchai, N.; Kakisaka, M.; Yamada, K.; Aida, Y. Comparative analysis of seven viral nuclear export signals (NESs) reveals the crucial role of nuclear export mediated by the third NES consensus sequence of nucleoprotein (NP) in influenza A virus replication. PLoS ONE 2014, 9, e105081.

51. Han, H.; Cui, Z.Q.; Wang, W.; Zhang, Z.P.; Wei, H.P.; Zhou, Y.F.; Zhang, X.E. New regulatory mechanisms for the intracellular localization and trafficking of influenza A virus NS1 protein revealed by comparative analysis of A/PR/8/34 and A/Sydney/5/97. J. Gen. Virol. 2010, 91, 2907–2917.

52. Li, Z.; Jiang, Y.; Jiao, P.; Wang, A.; Zhao, F.; Tian, G.; Wang, X.; Yu, K.; Bu, Z.; Chen, H. The NS1 gene contributes to the virulence of H5N1 avian influenza viruses. J. Virol. 2006, 80, 11115–11123.

53. Neumann, G.; T.Hughes, M.; Kawaoka, Y. Influenza A virus NS2 protein mediates vRNP nuclear export through NES-independent interaction with hCRM1. EMBO J. 2000, 19, 6751–6758.

54. Iwatsuki Horimoto, K.; Horimoto, T.; Fujii, Y.; Kawaoka, Y. Generation of influenza A virus NS2 (NEP) mutants with an altered nuclear export signal sequence. J. Virol. 2004, 78, 10149–10155.

55. Eisfeld, A.J.; Neumann, G.; Kawaoka, Y. At the centre: Influenza A virus ribonucleoproteins. Nat. Rev. Microbiol. 2015, 13, 28–41.
56. Hutchinson, E.C.; Fodor, E. Nuclear import of the influenza A virus transcriptional machinery. Vaccine 2012, 30, 7353–7358.
57. Elton, D.; Medcalf, L.; Bishop, K.; Harrison, D.; Digard, P. Identification of amino acid residues of influenza virus nucleoprotein essential for RNA binding. J. Virol. 1999, 73, 7357–7367.
58. Marklund, J.K.; Ye, Q.; Dong, J.; Tao, Y.J.; Krug, R.M. Sequence in the influenza A virus nucleoprotein required for viral polymerase binding and RNA synthesis. J. Virol. 2012, 86, 7292–7297.
59. Wang, Z.; Liu, X.; Zhao, Z.; Xu, C.; Zhang, K.; Chen, C.; Sun, L.; Gao, G.F.; Ye, X.; Liu, W. Cyclophilin E functions as a negative regulator to influenza virus replication by impairing the formation of the viral ribonucleoprotein complex. PLoS ONE 2011, 6, e22625.
60. Robb, N.C.; Smith, M.; Vreede, F.T.; Fodor, E. NS2/NEP protein regulates transcription and replication of the influenza virus RNA genome. J. Gen. Virol. 2009, 90, 1398–1407.
61. Deng, T.; Sharps, J.; Fodor, E.; Brownlee, G.G. In vitro assembly of PB2 with a PB1-PA dimer supports a new model of assembly of influenza A virus polymerase subunits into a functional trimeric complex. J. Virol. 2005, 79, 8669–8674.
62. Wu, W.W.; Weaver, L.L.; Pante, N. Ultrastructural analysis of the nuclear localization sequences on influenza A ribonucleoprotein complexes. J. Mol. Biol. 2007, 374, 910–916.
63. Elton, D.; Simpson-Holley, M.; Archer, K.; Medcalf, L.; Hallam, R.; McCauley, J.; Digard, P. Interaction of the influenza virus nucleoprotein with the cellular CRM1-mediated nuclear export pathway. J. Virol. 2001, 75, 408–419.
64. Brunotte, L.; Flies, J.; Bolte, H.; Reuther, P.; Vreede, F.; Schwemmle, M. The nuclear export protein of H5N1 influenza A viruses recruits matrix 1 (M1) protein to the viral ribonucleoprotein to mediate nuclear export. J. Biol. Chem. 2014, 289, 20067–20077.
65. Halder, U.C.; Bhowmick, R.; Roy Mukherjee, T.; Nayak, M.K.; Chawla-Sarkar, M. Phosphorylation drives an apoptotic protein to activate antiapoptotic genes: Paradigm of influenza a matrix 1 protein function. J. Biol. Chem. 2013, 288, 14554–14568.
66. Wang, S.; Zhao, Z.; Bi, Y.; Sun, L.; Liu, X.; Liu, W. Tyrosine 132 phosphorylation of influenza A virus M1 protein is crucial for virus replication by controlling the nuclear import of M1. J. Virol. 2013, 87, 6182–6191.
67. Hutchinson, E.C.; Denham, E.M.; Thomas, B.; Trudgian, D.C.; Hester, S.S.; Ridlova, G.; York, A.; Turrell, L.; Fodor, E. Mapping the phosphoproteome of influenza A and B viruses by mass spectrometry. PLoS Pathog. 2012, 8, e1002993.
68. Bui, M.; Myers, J.E.; Whittaker, G.R. Nucleo-cytoplasmic localization of influenza virus nucleoprotein depends on cell density and phosphorylation. Virus Res. 2003, 84, 37–44.
69. Chenavas, S.; Estrozi, L.F.; Slama-Schwok, A.; Delmas, B.; di Primo, C.; Baudin, F.; Li, X.; Crepin, T.; Ruigrok, R.W. Monomeric nucleoprotein of influenza A virus. PLoS Pathog. 2013, 9, 1–10.
70. Zheng, W.; Li, J.; Wang, S.; Cao, S.; Jiang, J.; Chen, C.; Ding, C.; Qin, C.; Ye, X.; Gao, G.F.; Liu, W. Phosphorylation controls the nuclear-cytoplasmic shuttling of influenza A virus nucleoprotein. J. Virol. 2015, doi:10.1128/JVI.00015-15.

71. Hale, B.G.; Knebel, A.; Botting, C.H.; Galloway, C.S.; Precious, B.L.; Jackson, D.; Elliott, R.M.; Randall, R.E. CDK/ERK-mediated phosphorylation of the human influenza A virus NS1 protein at threonine-215. *Virology* 2009, **383**, 6–11.

72. Hsiang, T.Y.; Zhou, L.; Krug, R.M. Roles of the phosphorylation of specific serines and threonines in the NS1 protein of human influenza A viruses. *J. Virol.* 2012, **86**, 10370–10376.

73. Gabriel, G.; Klingel, K.; Otte, A.; Thiele, S.; Hudjetz, B.; Arman-Kalcek, G.; Sauter, M.; Shmidt, T.; Rother, F.; Baumgarte, S.; *et al.* Differential use of importin-alpha isoforms governs cell tropism and host adaptation of influenza virus. *Nat. Commun.* 2011, **2**, e156.

74. Boivin, S.; Hart, D.J. Interaction of the influenza A virus polymerase PB2 C-terminal region with importin alpha isoforms provides insights into host adaptation and polymerase assembly. *J. Biol. Chem.* 2011, **286**, 10439–10448.

75. Hudjetz, B.; Gabriel, G. Human-like PB2 627K influenza virus polymerase activity is regulated by importin-alpha1 and-alpha7. *PLoS Pathog.* 2012, **8**, e1002488.

76. Nardozzi, J.D.; Lott, K.; Cingolani, G. Phosphorylation meets nuclear import: A review. *Cell Commun. Signal.* 2010, **8**, e32.

77. Pal, S.; Santos, A.; Rosas, J.M.; Ortiz-Guzman, J.; Rosas-Acosta, G. Influenza A virus interacts extensively with the cellular SUMOylation system during infection. *Virus Res.* 2011, **158**, 12–27.

78. Wu, C.Y.; Jeng, K.S.; Lai, M.M. The SUMOylation of matrix protein M1 modulates the assembly and morphogenesis of influenza A virus. *J. Virol.* 2011, **85**, 6618–6628.

79. Han, Q.L.; Chang, C.; Li, L.; Klenk, C.; Cheng, J.K.; Chen, Y.X.; Xia, N.S.; Shu, Y.L.; Chen, Z.; Gabriel, G.; *et al.* Sumoylation of Influenza A Virus Nucleoprotein Is Essential for Intracellular Trafficking and Virus Growth. *J. Virol.* 2014, **88**, 9379–9390.

80. Gao, S.; Wu, J.; Liu, R.Y.; Li, J.; Song, L.; Teng, Y.; Sheng, C.; Liu, D.; Yao, C.; Chen, H.; *et al.* Interaction of NS2 with AIMP2 facilitates the switch from ubiquitination to SUMOylation of M1 in influenza A virus-infected cells. *J. Virol.* 2015, **89**, 300–311.

© 2015 by the authors; licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution license (http://creativecommons.org/licenses/by/4.0/).

Copyright of Viruses (1999-4915) is the property of MDPI Publishing and its content may not be copied or emailed to multiple sites or posted to a listserv without the copyright holder's express written permission. However, users may print, download, or email articles for individual use.
