
SCIENTIFIC REPORTS

OPEN

Stability of HIB-Cul3 E3 ligase adaptor HIB Is Regulated by Self-degradation and Availability of Its Substrates

Received: 14 January 2015  
Accepted: 06 July 2015  
Published: 12 August 2015  

Zizhang Zhou, Congyu Xu, Ping Chen, Chen Liu, Shu Pang, Xia Yao & Qing Zhang

The HIB-Cul3 complex E3 ligase regulates physiological homeostasis through regulating its substrate stability and its activity can be modulated by changing HIB abundance. However, regulation of HIB remains elusive. Here we provide evidence that HIB is degraded through the proteasome by Cul3-mediated polyubiquitination in K48 manner in *Drosophila*. Strikingly, HIB is targeted for degradation by itself. We further identify that three degrons (52LKSS56T, 76LDEE80S and 117MESQ121R) and K185 and K198 of HIB are essential for its auto-degradation. Finally, we demonstrate that HIB-Cul3 substrates, Ci and Puc, can effectively protect HIB from HIB-Cul3-mediated degradation. Taken together, our study indicates that there is an exquisite equilibrium between the adaptor and targets to achieve the tight control of the HIB, which is essential for maintaining suitable Hh and JNK signaling. And the mechanism of adaptor self-degradation and reciprocal control of the abundance between adaptor and its substrates is also applied to BTB-Cul3 E3 ligase adaptor dKeap1, dDiablo and dKLHL18.

### Highlights

1. HIB is unstable and degraded through ubiquitin/proteasome pathway
2. HIB is ubiquitinated by HIB-Cul3 E3 ligase
3. HIB N-terminal degrons and C- terminal K185 and K198 are essential for its degradation
4. Substrates of HIB-Cul3 ligase protect HIB from degradation

Most cellular processes require a strictly controlled coordination of regulatory proteins. Precise protein turnover allows cells to adapt rapidly to various internal and external stimuli. Ubiquitin-mediated protein degradation by the 26S proteasome has emerged as a key mechanism to control protein turnover. This process of delivering ubiquitin to substrates requires the coordinated action of three classes of enzymes: the E1 ubiquitin-activating enzyme, the E2 ubiquitin-conjugating enzyme, and the E3 ubiquitin-protein ligase. While the E1 and E2 enzymes are primarily involved in activating and transferring ubiquitin through high-energy thioester bonds, E3 ligases play crucial roles in providing the specificity of substrate recognition.

Ubiquitin ligases (E3) that confer the substrate specificity have been grouped into two families: the single-subunit forms including single RING-finger, U-box and HECT-domain E3s, and the multi-subunit RING-finger ubiquitin ligases, including the Cullin-Rbx type and the APC E3s. The SCF complex (Skp1, Cul1, F-box) is an example of a multi-subunit RING-finger ubiquitin ligase. In the SCF ubiquitin ligase complex, Cul1 serves as a scaffold protein to assemble Skp1/F-box proteins with the small ring finger Rbx1. F-box proteins directly contact their substrates and determine the specificity of the SCF complex.

State Key Laboratory of Pharmaceutical Biotechnology and MOE Key Laboratory of Model Animals for Disease Study, Model Animal Research Center of Nanjing University, Nanjing, 210061, China. Correspondence and requests for materials should be addressed to Q.Z. (email: zhangqing@nju.edu.cn)

SCIENTIFIC REPORTS | 5:12709 | DOI: 10.1038/srep12709

In addition to Cul1, genomes encode six additional Cullins (Cul2, Cul3, Cul4A, Cul4B, Cul5 and Cul7), most of which also form ubiquitin ligase complexes$^{10}$. Notably, Cul3 interacts with some BTB-containing proteins and functions as a typical BTB-Cul3 E3 ligase complex$^{11,12}$.

As a BTB protein, HIB (Hedgehog-induced MATH and BTB domain containing protein) bridges its C-terminal BTB domain to Cul3 to form an E3 ligase complex and specifically recruits substrates for ubiquitination and degradation with its N-terminal MATH domain$^{13}$. HIB was first identified for its involvement in Hh pathway$^{13,14}$. In *Drosophila*, *hib* is one of the target genes in Hh pathway; the expression of *hib* is directly activated by Ci, a transcription factor unique to the Hh pathway. HIB directly binds to Ci and then mediates the ubiquitination and degradation of Ci. Overexpression of HIB-Cul3 E3 ligase adaptor HIB downregulates Ci, and loss of HIB causes Ci accumulation in *Drosophila*$^{13,14}$. These results indicate that one approach to modulate HIB-Cul3 E3 ligase activity is to regulate the level of its adaptor HIB.

Although HIB-Cul3 E3 ligase plays vital roles in many cellular processes, its regulation remains to be determined. Here, we show that HIB is degraded in a ubiquitin-dependent manner. Furthermore, we find that the destabilization of HIB requires an intact HIB-Cul3 complex; the adaptor HIB ubiquitinates itself for degradation. We further present that the S/T/D/E/Q-rich degrons and the lysine residues at 185 and 198 in HIB are responsible for its ubiquitination. Moreover, we illustrate that HIB-Cul3 E3 ligase substrates, such as Ci and Puc, can competitively bind HIB-Cul3 to protect HIB from degradation, suggesting that there is a fine-tuning balance of the E3 ligase and its substrates to maintain physiological homeostasis.

## Results

### HIB protein is unstable and degraded through Cul3

In BTB-Cul3 E3 ligase-mediated protein degradation, BTB-Cul3 recognizes substrates through different adaptor BTB proteins$^{12}$. In *Drosophila*, the BTB protein HIB associates with Cul3 to form an E3 ligase and recruits substrates including Ci for ubiquitination and degradation$^{13,15}$. Compared with the control disc (Fig. 1A), overexpression of *hib* resulted in a decrease of Ci (Fig. 1B), whereas knockdown of *hib* increased Ci level (Fig. 1C), indicating that the regulation of HIB-Cul3 E3 ligase activity can be achieved through modulating HIB level. To test whether the level of HIB is important for maintaining physiological homeostasis, we overexpressed and knocked down *hib* in wing disc and found that either *hib* overexpression or *hib* knockdown causes abnormal adult wing phenotypes (Fig. 1D–F). To examine HIB protein stability, we used the translation inhibitor cycloheximide (CHX) to block translation and determine the stability of HIB in S2 cells. Strikingly, HIB was unstable, with a half-life of less than 4 hrs (Fig. 1G). Since HIB and Cul3 form an E3 ligase, first, we tested whether the degradation of HIB was regulated by Cul3. As shown in Fig. 1G,H, Cul3 promoted HIB destabilization in a dose-related manner. Conjugation of the ubiquitin-like molecule Nedd8 to a conserved lysine residue (K717) on Cul3 is essential for its E3 ligase activity$^{16}$. To test whether the E3 ligase activity of Cul3 is essential for HIB destabilization, we employed a dominant-negative form of Cul3 named Cul3KR which K717 is replaced by R. Compared with Cul3, Cul3KR could stabilize HIB in S2 cells possibly by neutralizing endogenous Cul3 (Fig. 1I). We further examine the endogenous HIB protein level in S2 cells when Cul3 or Cul3KR was expressed. Cul3 expression caused HIB decreased, while Cul3KR expression resulted in an increase of HIB (Fig. 1J).

To confirm the result, we made *cul3* mutant clones and simultaneously expressed Fg-tagged HIB under the control of *GMR-gal4* in the posterior region of eye discs or *MS1096-gal4* in the pouch region of wing discs. Fg-tagged HIB accumulated in the *cul3*<sup>gtf2</sup> mutant clones both in eye discs (Fig. 1K–K") and in wing discs (Fig. 1L–L"). Given that HIB protein mainly localized in nuclei$^{13}$, the degradation of HIB occurred in nuclei (punctate staining in the nuclei). In addition, HA-tagged HIB was degraded when Cul3 was co-expressed in wing discs (Fig. 1M–M"). Contrarily, Cul3KR expression did not result in HIB decreased (Fig. 1N–N"). All the above data verify that the degradation of HIB protein is through Cul3 and Cul3 ligase activity is essential for its function on HIB degradation.

### HIB protein degradation is via polyubiquitination and the proteasome

Cul3-mediated substrate turnover is through substrate ubiquitination and ultimately 26S proteasome degradation$^{17}$. Likewise, treatment with the proteasome inhibitor MG132 effectively increased HIB protein level (Fig. 2A, lane 3), whereas in the absence of MG132, HIB protein abundance was apparently reduced when Cul3 was co-expressed (Fig. 2A, lane 4), suggesting that the degradation of HIB is likely through ubiquitination and the proteasome. To further test this possibility, we carried out a ubiquitination assay. As shown in Fig. 2B, Cul3 promoted HIB ubiquitination (lane 2), whereas Cul3KR attenuated HIB ubiquitination (lane 3). Without Cul3 co-transfection, the weak ubiquitination of HIB is likely through endogenous Cul3 (Fig. 2B, lane 1). Taken together, these data support the assertion that Cul3 promotes the ubiquitin-dependent degradation of HIB.

Ubiquitin-mediated protein degradation can occur through monoubiquitination or polyubiquitination$^{18}$. To test which type of ubiquitination contributes to HIB degradation, we employed Ub-K0, a mutant form of Ub, in which all seven lysine residues (K6, K11, K27, K29, K33, K48 and K63) are replaced by Arginines (Rs), preventing polyubiquitin chain formation$^{19}$. The ubiquitination level of HIB apparently decreased when co-expressed with Ub-K0 compared with wild-type Ub (Fig. 2C), suggesting that HIB is subjected to polyubiquitination. To further characterize the Cul3-mediated Ub modification

www.nature.com/scientificreports/

![](https://i.imgur.com/yourimage.png)

Figure 1. HIB protein is unstable, and its degradation is Cul3-dependent. All the wing discs shown in this study are oriented with anterior to the left and ventral at the top. GMR-gal4 and MS1096-gal4 lines drive the target gene expression in the posterior region of eye discs and in the pouch region of wing discs, respectively. The target gene expression with MS1096-gal4 is typically stronger in the dorsal region of the wing pouch. cul3<sup>gt2</sup> mutant clones in eye and wing discs were marked by the lack of GFP expression.

(A–C) Wing discs of MS1096 (A) hib overexpression by MS1096 (B) and hib knockdown by MS1096 (C) were stained to show Ci expression. Overexpression of hib decreased Ci (B, arrow), whereas knockdown of hib upregulated Ci (C, arrow). (D–F) Comparison of adult wing phenotypes from control flies (D) hib overexpression (E) and hib knockdown (F). Of note, both knockdown and overexpression of hib resulted in serious wing defect. (G) Western blots of lysates from S2 cells expressing indicated proteins and treated with CHX for the indicated time intervals. Quantification analyses of the western blots were shown on the right. Notably, Cul3 promoted HIB degradation. (H–I) Cul3 dose-dependently promoted HIB degradation (E) whereas Cul3KR stabilized HIB (F). S2 cells were transfected with 5 μg Myc-HIB and 0 μg, 0.5 μg, 1.5 μg, 4.5 μg Fg-Cul3 or Fg-Cul3KR plasmids respectively. The relative intensities of Myc-HIB bands were measured by Image J. Actin is shown as a loading control. (J) Cul3 decreased the endogenous HIB protein in S2 cells, while Cul3KR stabilized endogenous HIB. The relative intensities of HIB bands were measured by Image J. The arrowhead marks the band of HIB protein. Actin acts as a loading control. (K–K") Ectopically expressed Fg-HIB with GMR accumulated in cul3<sup>gt2</sup> mutant clones in the posterior region of eye discs. (L–L") Ectopic-expressed Fg-HIB with MS1096 was accumulated in cul3<sup>gt2</sup> mutant clones in wing discs. (M–M") Co-overexpression HIB and Cul3 with MS1096 in wing discs induced HIB degradation. (N–N") HIB was not degraded when co-overexpressed with Cul3KR. Cul3KR is a dominant negative mutant form of Cul3 with K717 mutated to R. Scale bars: 50 μm for all images.

www.nature.com/scientificreports/

A

MG132
DMSO

Myc-HIB
- + + +
HA-Cul3
- - + +

1 2 3 4

IP: Myc
IB: Myc

IP: HA
IB: HA

B

MG132

Myc-HIB
+ + +
Fg-Cul3
- + -
Fg-Cul3KR
- - +

1 2 3

170-
130-
100-

IP: Myc
IB: Ub

IP: Myc
IB: Myc

IP: Fg
IB: Fg

C

MG132

Myc-HIB
+ + + +
Fg-Cul3
- + + +
HA-Ub
- - + -
HA-Ub-K0
- - - +

1 2 3 4

170-
130-
100-

IP: Myc
IB: HA

IP: Myc
IB: Myc

IP: Fg
IB: HA

D

Myc-HIB + Fg-Cul3 + MG132

1 2 3 4 5 6 7 8 9 10
11 12 13 14

IP: Myc
IB: HA

IP: Myc
IB: Myc

IP: Myc
IB: Fg

IP: Myc
IB: HA

E

Fg
MS1096
Fg-cul3
Myc-hib
Ub-K6R

E'
Myc

F
Fg
MS1096
Fg-cul3
Myc-hib
Ub-K11R

F'
Myc

H
Fg
MS1096
Fg-cul3
Myc-hib
Ub-K29R

H'
Myc

I
Fg
MS1096
Fg-cul3
Myc-hib
Ub-K33R

I'
Myc

J
Fg
MS1096
Fg-cul3
Myc-hib
Ub-K63R

J'
Myc

K
Fg
MS1096
Fg-cul3
Myc-hib
Ub-K48R

K'
Myc

K''
Myc

L
Myc
MS1096
Fg-cul3
Myc-hib
Ub-K0

L'
Fg

L''
Myc

Figure 2. Cul3 promotes HIB K48-linked polyubiquitination. Ub-K0 is a mutant form of Ub, in which all Ks are replaced with Rs. Ub-K6R, K11R, K27R, K29R, K33R, K48R, and K63R are mutant forms of Ub with K on residue 6, 11, 27, 29, 33, 48, and 63, respectively, replaced with R. (A) HIB was degraded through the proteasome. In the presence of the proteasome inhibitor MG132, the level of HIB had no detectable difference when Cul3 was co-expressed. In the absence of MG132, HIB protein abundance was remarkably reduced. (B) Cul3 promoted HIB ubiquitination. The ubiquitination of HIB was apparently increased when Cul3 was co-expressed, whereas, the ubiquitination level was decreased when Cul3KR was co-expressed. (C) Cul3 induced the polyubiquitination of HIB. The ubiquitination level had a striking difference when HA-Ub or HA-Ub-K0 was co-expressed. Ub-K0 cannot form polyubiquitin chains. (D) Cul3 promoted K48 linkage polyubiquitination of HIB. Ubiquitination assays were carried out using different Ub mutants. (E–J') Ub-K6R (E'), Ub-K11R (F'), Ub-K27R (G'), Ub-K29R (H'), Ub-K33R (I') and Ub-K63R (J') could not prevent HIB degradation induced by Cul3. (K–L") Ub-K48R (K') and Ub-K0 (L') apparently attenuated Cul3-mediated HIB degradation. Scale bars: 50µm for all images.

on HIB, we began with identifying the type of Ub chain linkages, as the distinct linkage of polyubiquitin chains has been implicated in determining diverse fates of substrates^{20-25}. An ubiquitination assay was performed via several Ub mutants, in whom one of the seven Ks was replaced by R: Ub-K6R, Ub-K11R, Ub-K27R, Ub-K29R, Ub-K33R, Ub-K48R and Ub-K63R. As shown in Fig. 2D, the ubiquitination level of HIB was eliminated when Ub-K48R and Ub-K0 were used (lanes 8 and 10), whereas other mutants

www.nature.com/scientificreports/

A

| Myc-HIB-C | + | - | - | + | + |
| --- | --- | --- | --- | --- | --- |
| HA-Cul3 | - | + | - | + | - |
| HA-Cul3-ΔN41 | - | - | + | - | + |

1 2 3 4 5

B

| HA-Cul3 | - | - | - | + | - | + | - | - |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| HA-Cul3-ΔN41 | - | - | - | - | + | - | + | - |
| Fg-Roc1a | - | + | - | - | - | + | + | - |
| Fg-Roc1b | - | - | + | - | - | - | + | + |

1 2 3 4 5 6 7 8 9

IP: HA  
IB: Myc  

IP: Myc  
IB: Myc  

IP: HA  
IB: HA  

IP: Fg  
IB: HA  

IP: HA  
IB: HA  

IP: Fg  
IB: Fg  

D

MG132

| Myc-HIB | - | + | + | + |
| --- | --- | --- | --- | --- |
| HA-Cul3 | - | - | + | - |
| HA-Cul3-ΔN41 | - | - | - | + |
| Fg-Ub | - | + | + | + |

1 2 3 4

IP: Myc  
IB: Fg  

IP: Myc  
IB: Myc  

IP: Myc  
IB: HA  

Figure 3. The degradation of HIB is conducted by a BTB-containing protein. (A) The HIB-C associated with Cul3 but not Cul3-ΔN41. HIB-C contains aa180–374 of HIB. Cul3-ΔN41 is a form of Cul3 with a deletion of amino acids from 35 to 75. (B) Both Cul3 and Cul3-ΔN41 interacted with Roc1a and Roc1b. (C–C") HIB was not degraded when co-expressed with Cul3-ΔN41 (compared with Fig. 1M–M"). (D) Compared with Cul3, Cul3-ΔN41 did not promote HIB ubiquitination. Scale bars: 50 μm for all images.

did not affect the ubiquitination of HIB. The Ub-K48, which harbors only one K on 48, had the similar effect on Cul3-mediated HIB ubiquitination (compare Fig. 2D lane13 with lane 12). The findings suggest that Cul3 catalyzes HIB K48-linked polyubiquitination. Furthermore, we generated different Ub lysine mutant transgenic flies and examined which of these mutants could attenuate HIB degradation. Compared with other Ub mutants, only Ub-K0 and Ub-K48R could effectively inhibit Cul3-mediated HIB destabilization (Fig. 2E–L"). Taken together, Cul3 promotes K48-linkages polyubiquitin modification of HIB and subsequent degradation.

A BTB domain-containing protein is responsible for HIB degradation. Increasing evidence suggests that BTB proteins function as substrate-specific adaptors of Cul3 ligases [12,26]. To test whether Cul3-mediated HIB ubiquitination occurs through a BTB-containing protein, we constructed a truncated form of Cul3 (Cul3-ΔN41) with a deletion of 41 amino acids in the N-terminus (from Trp³⁵ to Tyr⁷⁵). These 41 residues include a hydrophobic helix, H2, which is involved in binding to the BTB domain [27]. As expected, wild-type Cul3 could bind to the C-terminal fragment (HIB-C) (Fig. 3A, lane 4), whereas deletion of the N-terminal 41 residues apparently disrupted this interaction (Fig. 3A, lane 5). We further found that Cul3-ΔN41 could bind the ring box proteins, such as Roc1a and Roc1b (Fig. 3B), indicating that the deletion of 41 amino acids did not affect Cul3 interaction with an E2 ubiquitin-conjugating enzyme. If Cul3-dependent HIB degradation is through a BTB-containing protein, Cul3-ΔN41 will not induce HIB ubiquitination and degradation. Consistent with this hypothesis, HIB was not degraded when Cul3-ΔN41 was co-expressed in the wing discs (Fig. 3C–C"). The ubiquitination assay also showed
that Cul3, but not Cul3-ΔN41, promoted HIB ubiquitination (compared Fig. 3D lane3 with lane4). These data indicate that a BTB-containing protein is account for HIB ubiquitination and degradation.

**HIB ubiquitination is mediated by HIB-Cul3 E3 ligase.** The above results suggest that a BTB-containing protein conjugates with Cul3 to promote HIB degradation. Given that HIB is a BTB protein and forms a complex with Cul3 as an E3 ligase${}^{13,15}$, it was prudent to first test whether HIB degradation is mediated by itself. We employed a mutant form of HIB, in which the 3-box domain (Cul3-associated domain; from Val${}^{299}$ to Thr${}^{330}$) was deleted${}^{28}$. The 3-box domain sits adjacent to the BTB domain and plays a decisive role for HIB association with Cul3 to allow for HIB-Cul3 E3 ligase activity${}^{29,30}$. Deletion of the 3-box domain abolishes the association between BTB and Cul3${}^{29}$ and destroys the E3 ligase function of HIB, as monitored by the hindered capacity to degrade the well-known target, Ci (compare Fig. 4B,B” with Fig. 4A–A”).

We found that HIB-Δ3box failed to be degraded when Cul3 was co-expressed in wing discs (Fig. 4C–C”). However, co-expression of exogenous wild-type HIB and Cul3 could induce an apparent decrease of HIB-Δ3box in wing discs (Fig. 4D–D”). Furthermore, HIB-Δ3box protein was mildly decreased both near A/P boundary in wing discs (Fig. 4B,C’) and the posterior region in eye discs (Fig. 4F–F”), where the endogenous *hib* is expressed. Consistent with this observation, co-expression of the dominant-negative Cul3KR apparently upregulated the HIB-Δ3box protein level in the posterior of eye discs (compare Fig. 4G–G” with 4F–F”). Furthermore, we found that HIB promoted HIB-Δ3box degradation in a dose-dependent manner in S2 cells (Fig. 4H). We also test the ubiquitination level of HIB-Δ3box protein, and found that only co-expression of HIB-Cul3 increased the ubiquitination of HIB-Δ3box (Fig. 4I). These data demonstrate that HIB is targeted for ubiquitination through HIB-Cul3 E3 ligase.

The N-terminal degrons and the C-terminal Lys${}^{185}$ or Lys${}^{198}$ are essential for HIB degradation. HIB contains two functional domains: the MATH domain at the N-terminus for substrate recognition and binding, and the BTB domain for association with Cul3 (Fig. 5A)${}^{13,15}$. To study the roles of the MATH domain and the BTB domain for HIB degradation, we employed two truncated constructs: Myc-HIB-N, comprising the MATH domain and its flanking sequence (aa 1 to 179); and Myc-HIB-C, including BTB domain and its flanking sequence (aa 180 to 374) (Fig. 5A)${}^{13,15}$. As shown in Fig. 5B,C, Myc-HIB-N and Myc-HIB-C failed to be degraded by HIB-Cul3 E3 ligase. We also analyzed the stability of HIB-N and HIB-C in wing discs when co-expressed with the HIB-Cul3 complex and found that these two truncated HIB proteins were not degraded by HIB-Cul3 E3 ligase (Fig. 5D–G). To further confirm these results, we generated *hib* mutant clones in eye discs and simultaneously expressed Fg-tagged HIB-N or HIB-C by GMR-gal4 in the posterior region of eye discs. Consistently, there were no any detectable protein accumulations *hib* mutant clones (Fig. 5H–I’). These results support that the intact structure of HIB is essential for its degradation.

Neither the N-terminal nor the C-terminal fragments alone is sufficient for HIB degradation, suggesting that the HIB-Cul3 recognition degron(s) and the polyubiquitin chain attachment site(s) located in different fragments. HIB, acting as an E3 ligase, recognizes the substrates by its N-terminal fragment, and the C-terminus was account for association with Cul3${}^{13}$. The Co-IP experiments determined that HIB-N interacted with HIB-N, not HIB-C (Fig. 5J), indicating that the HIB-Cul3 recognition sites sit in HIB-N. Given HIB binds substrates through some S/T/D/E/Q-rich degrons${}^{15,29}$, we found three potential S/T/D/E/Q-rich motifs in the MATH domain of HIB: ${}^{52}$LKSS${}^{56}$T, ${}^{76}$LDEE${}^{80}$S and ${}^{117}$MESQ${}^{121}$R. When mutated these S/T/D/E/Q-rich motifs to generate HIB-N-3m that all the amino acids were replaced by alanines, we found that HIB could bind HIB-N, but not HIB-N-3m (Fig. 5K). To further check whether these motifs are important for HIB destabilization, we generated a Fg-*hib*-3m transgenic fly, and tested the stability of this mutant form in wing discs. HIB-Cul3 E3 ligase failed to promote Fg-HIB-3m degradation (Fig. 5L,M). These data indicate that the substrate HIB is recognized by HIB-Cul3 E3 ligase via its S/T/D/E/Q-rich degrons in MATH domain.

The previous data showed that the MATH domain of HIB did not self-interact, but HIB-N could form a dimer/oligomer${}^{15}$, suggesting the flanking sequences of MATH play important roles for HIB-N self-association. To test this possibility, various truncated constructs were generated (Figure S1A). Consistently, HA-tagged HIB-MATH did not pull down Fg-tagged HIB-MATH (Figure S1B). However, both HIB-N-1 (contains MATH domain and upstream sequence) and HIB-N-2 (contains MATH domain and downstream sequence) could form homodimer (Figures S1C,D), suggesting that the upstream and downstream sequences of MATH domain are essential for HIB-N self-interaction.

According to the above information, the polyubiquitin chains likely attached to the C-terminus of HIB. To determine this possibility, we fused one of the degrons (LKSST) to the N-terminus of HIB-C to construct LKSST-HIB-C chimeric protein. When expressed in S2 cells, HIB-N pulled down LKSST-HIB-C, but not HIB-C (Figure S1E), indicating that LKSST-HIB-C may be a substrate of HIB-Cul3 ligase. Consistently, HIB induced LKSST-HIB-C degradation in a dose-dependent manner (Figure S1F). Furthermore, we found that the lysine residues at 185 and 198 had important role for HIB ubiquitination. The mutation of K185 or K198 to R partially attenuated HIB destabilization induced by HIB-Cul3 ligase (Fig. 5N–Q). Furthermore, the degradation was almost totally blocked when both K185 and K198 were replaced by Rs (Fig. 5R,S). Through ubiquitination assay, the ubiquitination level of HIB decreased when either K185
www.nature.com/scientificreports/

Figure 4. HIB is degraded by itself. (A-A") A wing disc expressing HA-hib by MS1096 was stained to show HA (green), Ci (red) and ptc-lacZ (white) expression. Overexpression of hib decreased Ci and ptc-lacZ levels. (B-B") 3-box domain (aa299–330) deletion destroyed HIB E3 ligase function (compared with Fig. 4A–A"). (C-C") HIB-Δ3box failed to be degraded by Cul3 in wing discs. The level of HIB-Δ3box had a subtle decrease near the A/P boundary, where the endogenous hib is expressed. (D-D") HIB-Δ3box was apparently degraded when co-expressed with HIB and Cul3. (E-E") GFP marked the expression region of DE-gal4 in eye discs. (F-F") HIB-Δ3box was destabilized in the posterior region of eye discs, where the endogenous hib is ubiquitously expressed. (G-G") Co-expression of Cul3KR could effectively stabilize HIB-Δ3box protein in the posterior of eye discs. (H) HIB-Cul3 E3 ligase promoted HIB-Δ3box degradation. The relative intensities of HA-HIB-Δ3box bands were measured by Image J. Actin acts as a loading control. (I) Co-expression of HIB and Cul3 promoted the ubiquitination of HIB-Δ3box in S2 cells. Scale bars: 50 μm for all images.

www.nature.com/scientificreports/

Figure 5. The recognition degrons and K185 or K198 on HIB are essential for it ubiquitination and degradation. (A) Schematic drawings show the domains in HIB and its truncated fragments used in subsequent assays. Green and red bars denote the MATH and BTB domains of HIB. (B–C) HIB-Cul3 E3 ligase did not promote the degradation of HIB-N (B) and HIB-C (C) in S2 cells. Actin acts a loading control. (D–E) Wing discs expressing HA-hib-N plus Myc-hib and Fg-cul3 were stained to show Fg (green), HA (red) and Myc (white). HIB-Cul3 failed to degrade HIB-N. (F–G) Wing discs expressing HA-hib-C plus hib-cul3 were stained to show Fg (green), HA (red) and Myc (white). HIB-Cul3 failed to degrade HIB-C. (H–H′) Fg-HIB-N protein levels did not increase in hib mutant clones marked by the lack of GFP. Fg-HIB-N overexpression was driven by GMR-gal4 in the posterior region of eye discs. (I–I′) HIB-C protein did not accumulate in hib mutant clones in eye discs. (J) HIB-N pulled down HIB-N, not HIB-C. (K) HIB interacted with HIB-N, but not HIB-N-3m. HIB-N-3m is a mutant form of HIB-N that three potential S/T/D/E/Q-rich motifs are replaced by As. (L–M) HIB-Cul3 did not degrade HIB-3m. (N–O) Wing discs expressing HA-hib-K185R plus Myc-hib and Fg-cul3 were stained to show Fg (green), HA (red) and Myc (white). K185 mutation to R (HA-HIB-K185R) partially blocked HIB destabilization caused by HIB-Cul3 E3 ligase. (P–Q) K198 mutation to R (HA-HIB-K198R) partially prevented HIB from degradation induced by HIB-Cul3. (R–S) The degradation of HIB was totally blocked when both K185 and K198 were replaced by Rs (HA-HIB-K185R-K198R). (T) Western blots of immunoprecipitates (top) or lysates (bottom three panels) from S2 cells expressing indicated proteins and treated with MG132. HIB ubiquitination levels induced by HIB-Cul3 E3 ligase were decreased when K185 and K198 were replaced by Rs on HIB. Scale bars: 50 μm for all images.
or K198 was replaced by R (Fig. 5T). These data suggest that the polyubiquitin chains are attached to K195 and K198, and that the ubiquitin-tagged HIB is transported into the proteasome for degradation.

**Self-ubiquitination is applied for some other BTB-containing E3 ligases.** BTB-domain–containing proteins commonly form homodimers using BTB domain ${ }^{15,29,31,32}$. Consistently, the HIB-C indeed formed a homodimer/oligomer (Figure S1G). However, except for that, we found HIB-N also formed a dimer (Fig. 5J and Figures S1C,D). To address whether this BTB/BTB-domain-independent self-interaction was unique to HIB, we tested other BTB-containing E3 ligases including dKeap1, dDiablo and dKLHL18 (Fig. 6A). Through Co-IP experiments, dKeap1-$\Delta$BTB could pull down dKeap1; dDiablo-$\Delta$BTB pulled down dDiablo; dKLHL18-$\Delta$BTB pulled down dKLHL18 (Fig. 6B–D). In addition, dKeap1-$\Delta$BTB, dDiablo-$\Delta$BTB and dKLHL18-$\Delta$BTB could form homodimers, respectively (Figures S2B–D). These results indicate that BTB/BTB-independent self-association is likely to be applied for these adaptors. Except for BTB domain, dKeap1 harbors Back and six repeated Kelch domains (Fig. 6A and Figure S2A). We found that dKeap1-Back did not form a dimer (Figure S2E), and dKeap1-Back also did not bind dKeap1-Kelch (Figure S2F). However, Co-IP experiments showed that dKeap1-Kelch formed dimer (Figure S2G). These results suggest that the BTB/BTB-independent self-association of dKeap1 is mediated by the Kelch domain. We further found that this case was applied for dDiablo and dKLHL18 (Figures S2H–M). Although dKeap1, dDiablo and dKLHL18 harbor Kelch domain, the amino acid composition in Kelch domain differs significantly. To test whether the Kelch domains form heterodimer, we expressed different Kelch domains in S2 cells. Through Co-IP assay, Kelch domain exclusively formed homodimer (Figure S2N). The previous data revealed that dKeap1, dDiablo and dKLHL18 associated with Cul3 using BTB domains to form E3 ligases ${ }^{33-36}$. We examined the ubiquitination of dKeap1-$\Delta$BTB and found that dKeap1-Cul3 overexpression promoted its ubiquitination (Fig. 6E), suggesting that dKeap1 was likely to be a substrate of dKeap1-Cul3 E3 ligase. In addition, we uncovered that this self-ubiquitination was also applied for dDiablo and dKLHL18 (Fig. 6F,G).

We further investigated whether the degradation of dKeap1 was mediated by dKeap1 itself and found that dKeap1 promoted dKeap1-$\Delta$BTB destabilization in a dose-dependent manner (Fig. 6H, lane1–3 and Figure S2, lane1–3). The degradation of dKeap1-$\Delta$BTB was inhibited when treated with lysosome inhibitor $NH_{4}Cl$ (Fig. 6H, lane5 and lane 6), not proteasome inhibitor MG132 (Fig. 6H, lane 4), suggesting that the destabilization of dKeap1-$\Delta$BTB was through lysosome. In fact, the previous data have revealed that mammalian Keap1 is degraded by autophagy ${ }^{37-39}$. We further identified that dKeap1 occurred K63 linkage, not K48 linkage polyubiquitination (Fig. 6I). The K63-linked polyubiquitin chain modifications commonly targeted the substrates for vesicle-mediated trafficking or degradation ${ }^{40-42}$. Taken together, these results suggest that the ubiquitination and degradation of dKeap1 is mediated, at least in part, by dKeap1-Cul3 E3 ligase.

**Substrates efficiently prevent HIB from degradation through competitive association with HIB-Cul3 ligase.** Both HIB and Ci are ubiquitinated and subsequently destabilized by HIB-Cul3 E3 ligase. We sought to determine whether they competitively bind to the HIB-Cul3 complex. Since Ci N-terminus (CiN) can bind HIB to result in Ci degradation through HIB-Cul3 E3 ligase ${ }^{13,15}$. We performed assays in S2 cells and found that CiN substantially stabilized HIB in a dose-dependent manner (Fig. 7A). The full-length Ci (Ci ${ }^{\mathrm{FL}}$ ) also attenuated Cul3-mediated HIB degradation (Fig. 7B). To further verify this result in wing discs, we employed CiN because Ci ${ }^{\mathrm{FL}}$ overexpression stimulated endogenous $hib$ transcription ${ }^{13}$. Co-expressed CiN could efficiently stabilize HIB (Fig. 7C,D). In contrast, mutating the HIB binding sites in CiN led to the loss of its protection of HIB degradation (Fig. 7E,F). Next, it was essential to investigate whether this protection is broadly applicable to other substrates. As expected, another well-known substrate, JNK phosphatase puckered (Puc), also prevented HIB from destabilization (Fig. 7G). In addition, we found the dKeap1 substrate, dNrf2, also attenuated dKeap1-$\Delta$BTB degradation mediated by dKeap1 (Figure S3A, lane4-6). The corresponding substrates (dDsh and CG10324) ${ }^{33,35}$ also protected dDiablo and dKLHL18 from self-degradation (Figures S3B,C). We further found that the substrates of Keap1-, dDiablo- and dKLHL18-Cul3 E3 ligases could block their cognate Kelch-Kelch self-association (Figures S3D–F). These results suggest that the reciprocal control of the abundance between adaptor and its substrates is applied for dKeap1, dDiablo and dKLHL18.

**Discussion**

HIB-Cul3 E3 ligase recognizes and ubiquitinates substrates, culminating in their degradation by the 26S proteasome ${ }^{13,15}$. In *Drosophila*, HIB influences many cellular processes through targeting its substrates, including Hedgehog pathway transcription factor Ci and JNK pathway Puc. Both knockdown and overexpression of $hib$ in wing cause serious wing abnormalities (Fig. 1D–F), suggesting that the strict monitoring of HIB level is crucial. Self-degradation of the adaptor HIB will change its abundance, limiting the portion of Cul3 in Cul3 pool are motivated to couple with HIB to form HIB-Cul3 E3 ligase and thus limiting HIB-Cul3 E3 ligase activity. This idea is supported by data showing that either the overexpression or knockdown of HIB alone is enough to modulate HIB-Cul3 E3 ligase activity as determined by alterations in the level of its substrate Ci.

The well-defined substrates Ci and Puc effectively protect HIB from degradation through competitive interacting with HIB-Cul3 ligase, raising the possibility that some other substrates may have similar
www.nature.com/scientificreports/

A
dKeap1
BTB Back (Kelch)₆

dDiablo
BTB Back (Kelch)₆

dKLHL18
BTB Back (Kelch)₆

B
Myc-dKeap1-ΔBTB - +
Fg-dKeap1 + +

IP: Myc
IB: Fg
IB: Fg
IB: Myc

C
Myc-dDiablo-ΔBTB - +
Fg-dDiablo + +

IP: Myc
IB: Fg
IB: Fg
IB: Myc

D
Myc-dKLHL18-ΔBTB - +
Fg-dKLHL18 + +

IP: Myc
IB: Fg
IB: Myc

E
MG132/NH₄Cl

Fg-dKeap1 - - +
Fg-Cul3 - + +
HA-Ub + + +
Myc-dKeap1-ΔBTB + + +

IP: Myc
IB: HA
IB: Myc
IB: Fg-Cul3
IB: Fg-dKeap1
IB: HA-Ub

F
MG132

Fg-dDiablo - - +
Fg-Cul3 - + +
HA-Ub + + +
Myc-dDiablo-ΔBTB + + +

IP: Myc
IB: HA
IB: Myc
IB: Fg-Cul3
IB: Fg-dDiablo
IB: HA-Ub

G
MG132

Fg-dKLHL18 - - +
Fg-Cul3 - + +
HA-Ub + + +
Myc-dKLHL18-ΔBTB + + +

IP: Myc
IB: HA
IB: Myc
IB: Fg-Cul3
IB: Fg-dKLHL18
IB: HA-Ub

I
Fg-dKeap1/MG132/NH₄Cl

HA-Ub
HA-Ub-K48R
HA-Ub-K63R
HA-Ub-K48
HA-Ub-K63

IP: Fg
IB: HA
IB: Fg
IB: HA

H
Fg-dKeap1 - 1 3 3 3 3
Myc-dKeap1-ΔBTB + + + + + +
MG132 - - + - +
NH₄Cl - - - + +

IB: Myc
IB: Fg
IB: Actin

Figure 6. Some BTB-containing adaptors are regulated by auto-ubiquitination. (A) Schematic drawings show the domains in dKeap1, dDiablo and dKLHL18. Yellow and black bars denote the BTB and tandem Kelch domains. (B) dKeap1-ΔBTB pulled down dKeap1. (C) dDiablo-ΔBTB binds dDiablo. (D) dKLHL18-ΔBTB pulled down dKLHL18. (E) dKeap1-Cul3 promoted the ubiquitination of dKeap1-ΔBTB. The transfected cells were treated with NH₄Cl for 16hrs and MG132 for 4hrs prior cell harvesting. (F) dDiablo-Cul3 promoted the ubiquitination of dDiablo-ΔBTB. (G) dKLHL18-Cul3 promoted the ubiquitination of dKLHL18-ΔBTB. From F to G, the transfected S2 cells were treated with MG132 for 4hrs before cell harvesting. (H) dKeap1 promoted dKeap1-ΔBTB degradation in a dose-dependent manner. The indicated cells were treated with NH₄Cl for 16hrs or/and MG132 for 4hrs prior cell harvesting. (I) dKeap1 was modified by K63-linked polyubiquitin chains. The transfected cells were treated with NH₄Cl for 16hrs and MG132 for 4hrs prior cell harvesting.

www.nature.com/scientificreports/

A
HA-CiN
Myc-HIB + + + +
Fg-Cul3 + + + +

B
Myc-Ci
Myc-HIB + + + +
Fg-Cul3 + + + +

IP: Myc
IB: Myc
IB: HA
IB: Fg
IB: Actin

IP: Myc
IB: Myc
IB: Myc-Ci
IB: Fg
IB: Actin

C
Fg
C'
HA
C''
M1096
Fg-cul3
HA-hib
Myc-ciN
—

E
Fg
E'
HA
E''
MS1096
Fg-cul3
HA-hib
Myc-ciN-2m
—

D
Myc
M1096
Fg-cul3
HA-hib
Myc-ciN
—

F
Myc
MS1096
Fg-cul3
HA-hib
Myc_ciN-2m
—

G
Fg-Puc
HA-Cul3 + + + +
Myc-HIB + + + +
Myc-ciN
—

IP: Myc
IB: Myc
IB: Fg
IB: HA
IB: Actin

H
N41
Cul3
Roc1
BTB
HIBs act as
E2
E3-ligases
transfer
MATH
MATH
BTB
HIBs act as
substrates

I
Cytosol
Nucleus
Hh pathway → CiFL → Ciact → HIB →
feedback

Figure 7. Substrates prevent HIB from degradation. CiN (aa1–440) is a truncated form of Ci that contains two HIB binding motifs (^368^PEQPSSTS^376^G and ^379^AQVEADSASSQLS^392^D). CiN-2m is a mutant form of CiN, in which both binding motifs are mutated. The expression of CiN and CiN-2m were shown in (D) and (F) respectively. (A) CiN prevented HIB from degradation in a dose-dependent manner. (B) Ci inhibited Cul3-induced HIB degradation in a dose-dependent manner. (C–D) HIB degradation through Cul3 was blocked when overexpressed with CiN in wing discs. (E–F) The inhibitory effect of CiN disappeared when both HIB binding motifs on CiN were mutated. (G) Puc could efficiently protect HIB from destabilization. (H) HIB protein associates with Cul3 through its BTB domain. HIB acting as an adaptor of HIB-Cul3 E3 ligase recruits dissociative HIB protein through the MATH domain, thus HIB-Cul3 promotes dissociative HIB ubiquitination and degradation. Substrates competitively bind the adaptor HIB, thus preventing dissociative HIB degradation. (I) The physiological significance of HIB auto-regulated degradation. In the presence of Hedgehog protein, Ci is transported to the nucleus and acts as a transcription activator. Ci promotes *hib* expression; conversely, HIB degrades Ci through ubiquitinating Ci. When Ci level is low, excess HIB is degraded in an auto-regulated ubiquitination. In contrast, when the HIB level is low, Ci will protect HIB from degradation. This delicate feedback loop allows appropriate Ci and HIB levels to remain in the nucleus. Scale bars: 50 μm for all images.

functions. This protection could play an important role for keeping the equilibria between the adapter and targets. Ultimately, these equilibria tightly monitor the homeostasis of corresponding signaling pathways. In Hedgehog pathway, for example, in the presence of Hh, Ci accumulates in the nucleus and promotes HIB synthesis. In turn, HIB degrades Ci through HIB-Cul3 E3 ligase. When the Ci protein
level is low, excess HIB is degraded by itself through an auto-catalytic mechanism. However, when HIB protein reaches a very low level, Ci competitively associates with HIB-Cul3 and prevents HIB from further destabilization. Through this elegant regulation, Ci and HIB form a fine-tuning feedback loop that strictly monitors Hh pathway activity (Fig. 7H,I).

Although the previous data have been demonstrated that F-box adaptors in SCF E3 ligase are degraded by an auto-catalytic mechanism ${ }^{43,44}$, the regulation of other adaptors in Cul-containing E3s, including Cul2 and Cul3, has to be determined. Our findings uncovered that a BTB-containing adaptor HIB in HIB-Cul3 E3 ligase was regulated via auto-ubiquitination. Except for HIB, we also found the some other BTB-containing adaptors, including dKeap1, dDiablo and dKLHL18 were regulated by auto-ubiquitination. Therefore, it is interesting to investigate more BTB-containing adaptors to test whether they are modulated by self-ubiquitination. Here we should emphasize that even above BTB-containing adaptors are modulated by self-ubiquitination, but maybe their degradation goes through distinct ways. It is possible that they are destabilized by either proteasome or lysosome affected by their ubiquitination manners. Given that most Cullin proteins (Cul1, Cul2, Cul3, Cul4, Cul5 and Cul7) together with adaptors to form E3 ligases, it is fruitful to determine whether other adaptors are regulated by auto-catalytic mechanism.

Except for HIB, some Kelch domain-containing proteins are also conjugated with Cul3 to form various Kelch-Cul3 E3 ligases. For Kelch domain-containing adaptors, albeit their primary sequences of amino acids are different, they form beta-propellers which consist of several Kelch repeats ${ }^{45}$. The previous findings revealed that beta-propeller subunits could form a dimer through the aromatic residues on the dimer interface and charged side chains ${ }^{46}$. Our data showed that the Kelch domains from dKeapl, dDiablo and dKLHL18 exclusively formed homo-dimers, suggesting that beta-propeller may favor of forming homo-dimer, however, distinct primary sequences may endue their specificity for homo-dimerization and substrate recognizing. The advantage of adaptor self-degradation likely provides a fine-tuning mechanism to efficiently switch adaptors and achieve different BTB-Cul3 E3 ligase activities according to stimuli.

SPOP is an evolutionary homolog of HIB in vertebrates, and it shares extensive amino acid sequence identity with HIB ${ }^{13}$. As reported, SPOP recruits several targets to the SPOP-Cul3 E3 ligase for ubiquitination and subsequent degradation, including polycomb group BMI ${ }^{47}$, anti-apoptotic protein Daxx ${ }^{48}$ and breast cancer metastasis suppressorl (BRMS1) ${ }^{49}$. Recently, it has been reported that the mutation or deficiency of SPOP closely links to many tumors, including glioma ${ }^{50}$, prostate cancers ${ }^{51}$, breast cancers ${ }^{49}$ and endometrial tumors ${ }^{52}$, suggesting that the regulation of SPOP is important for homeostasis. A previous study also demonstrated that SPOP could undergo ubiquitination via Cul3 ${ }^{53}$. It is fruitful to investigate whether SPOP is ubiquitinated by SPOP-Cul3 E3 ligase.

## Methods

### Constructs, Mutants, and Transgenes.
The constructs for expression in S2 cells were generated as follows: the Fg-HIB, Fg-Cul3, Fg-Cul3KR, Myc-HIB, HA-Cul3, Myc-HIB-C, Myc-HIB-N, HA-CiN and Myc-Ci constructs have been previously described ${ }^{13,15}$. To construct Fg-Rocla, HA-Rocl b, Fg-Ub, HA-Ub, Fg-dKeapl, Fg-dDiablo, Fg-dKLHL18 HA-dNrf2 and Fg-Puc plasmids, we amplified the corresponding cDNA fragments, and then cloned them into pUAST-3×Fg or pUAST-3×HA vectors. The point mutant constructs used in this study were generated using PCR-based site-directed mutagenesis at the background of Fg-Ub or Fg-HIB. The truncated constructs were generated by inserting the corresponding coding sequences into the pUAST-3×HA or pUAST-6×Myc vectors. $cul3^{gf2}$, DE-gal4, GMR-gal4, MS1096-gal4, eyflp, hs-flp and UAS-GFP were from Bloomington Stock Center (Flybase). UAS-Fg-cul3, UAS-Fg-cul3KR, UAS-HA-hib, UAS-Fg-hib, UAS-Fg-hib-N, UAS-Fg-hib-C, UAS-Myc-ciN, UAS-Myc-ciN-2m, ptc-lacZ and $hib^{\Delta 6}$ have been described ${ }^{13,15}$. UAS-HA-ub-K6R, UAS-HA-ub-K11R, UAS-HA-ub-K27R, UAS-HA-ub-K29R, UAS-HA-ub-K33R, UAS-HA-ub-K48R, UAS-HA-ub-K63R, UAS-HA-ub-KO, UAS-HA-cul3, UAS-HA-cul3-$\Delta N 41$, UAS-Myc-hib, UAS-HA-hib-N, UAS-HA-hib-C, UAS-HA-hib-$\Delta 3$box, UAS-HA-hib-3m, UAS-HA-hib-K185R, UAS-HA-hib-K198R and UAS-HA-hib-K185R-K198R, were generated by injection of corresponding vectors into Drosophila embryos according to the methods described previously ${ }^{54}$. The parental strain for all germline transformations was $w^{1118}$. All stocks were maintained and raised under standard conditions.

### Immunostaining and microscopy.
For immunostaining of imaginal discs, third-instar larvae were dissected in PBS and fixed in freshly made 4% formaldehyde in PBS buffer at room temperature for 20 min, then washed three times with buffer PBT (PBS, 0.1% Triton X-100). To avoid non-specific interaction, larvae were blocked with PBTA (PBT, 10% BSA) for 30 min, next, were incubated overnight with needed primary antibodies in PBTA at 4°C, then washed with PBT for three times and incubated with appropriate fluorophore-conjugated secondary antibody dilutions in PBTA for 2 hr at room temperature. After washed for three times in PBT, imaginal discs were dissected and mounted in 80% glycerol. Images were captured with FV10-ASW 2.0 Olympus confocal microscope. Primary antibodies were used at the following dilutions: mouse anti-HA (F7) (1:200; Santa Cruz); rabbit anti-HA (Y11) (1:200; Santa Cruz); mouse anti-Myc (9E10) (1:200; Santa Cruz); rabbit anti-Fg (PA1-984B) (1:200; Thermo Scientific); mouse anti-$\beta$-Gal (G8021) (1:500; Sigma) and Rat anti-Ci (2A1) (1:50; DSHB).
www.nature.com/scientificreports/

Generating mutant clones. Clones of mutant cells were generated by the FLP-FRT system as described⁵⁵. The genotypes of the generated clones were as follows: *cul3* clones in wing discs expressing UAS-*Fg-hib*: *hs-flp*, MS1096; *cul3*<sup>gf2</sup> FRT40/*hs-Myc-GFP* FRT40; UAS-*Fg-hib*/+; *cul3* clones in eye discs expressing UAS-*Fg-hib*: *eyflp*; *cul3*<sup>gf2</sup> FRT40/*hs-Myc-GFP* FRT40; GMR-*gal4*/UAS-*Fg-hib*; *hib* clones in eye discs expressing UAS-*Fg-hib-N* or UAS-*Fg-hib-C*: *eyflp*; GMR-*gal4*/UAS-*Fg-hib-N* or UAS-*Fg-hib-C*; *hib*<sup>Δ6</sup> FRT82/*hs-Myc-GFP* FRT82. For GFP expression, the 3<sup>rd</sup> larvae were heat shocked at 37°C for 1.5 hrs and then kept at room temperature for another 1.5 hrs and subjected to dissection according to standard protocols.

Cell culture, Transfection, Immunoprecipitation and Western blot. S2 cells were maintained at 25°C in Schneider’s *Drosophila* Medium (S9895, Sigma) supplemented with 10% FBS (Gibco) and 1% penicillin/streptomycin (P0781, Sigma). Transfection was performed using calcium phosphate according to the manufacturer’s instructions (Invitrogen). An *ub-gal4* plasmid was co-transfected with pUAST expression vectors for all experiments. 48 hrs after transfection, cells were harvested for immunoprecipitation and western blot analysis with standard protocols (described in *Molecular Cloning*). The primary antibodies used were mouse anti-HA (F7) (1:2500; Santa Cruz); rabbit anti-HA (Y11) (1:2500; Santa Cruz), mouse anti-Myc (9E10) (1:2500; Santa Cruz); mouse anti-Fg (1:2500; Sigma); mouse anti-Ub (P4D1) (1:1000; Santa Cruz); rabbit anti-HIB (1:1000; ABclonal Technology) and mouse anti-Actin (A00702) (1:5000: Genscript). Rabbit anti-HIB antibody was generated in rabbit with full length HIB protein as antigen (from ABclonal Technology). After incubation with HRP-coupled secondary antibodies (goat anti-mouse diluted 1:10000, Abmax; goat anti-rabbit diluted 1:10000, Jackson ImmunoResearch), the blots were visualized using a chemiluminescent detection kit (GE healthcare).

Protein Stability Assays. S2 cells were plated in 10-cm dishes and transfected with plasmids after 18–24 hrs. After another 24 hrs, the cells were transferred into 6-well cell culture plates at equivalent densities. Cells were treated with 20 μg/ml CHX (Calbiochem) for the indicated times before harvesting. After western blots, the band intensity was measured by Image J.

Ubiquitination Assays. S2 cells were transiently transfected with the indicated combinations of expression vectors. Four hours before cells harvesting, MG132 (Calbiochem) was added to the media at a final concentration of 10 μM. The ubiquitination assays were then carried out based on the previously described protocol¹³. Briefly, cells were lysed with denaturing buffer (1% SDS, 50 mM Tris-base, pH 7.5, 0.5 mM EDTA, and 1 mM DTT) and incubated at 100°C for 5 min. The lysates were then diluted 10-fold with regular lysis buffer and subjected to immunoprecipitation and Western blot analysis.

References

1. Hochstrasser, M. Ubiquitin proteasomes, and the regulation of intracellular protein degradation. *Current opinion in cell biology* **7**, 215–223 (1995).
2. Wilkinson, K. D. Ubiquitination and deubiquitination: targeting of proteins for degradation by the proteasome. *Seminars in cell & developmental biology* **11**, 141–148, doi: 10.1006/scdb.2000.0164 (2000).
3. Hochstrasser, M. Ubiquitin and intracellular protein degradation. *Current opinion in cell biology* **4**, 1024–1031 (1992).
4. Hochstrasser, M. Ubiquitin-dependent protein degradation. *Annual review of genetics* **30**, 405–439, doi: 10.1146/annurev.genet.30.1.405 (1996).
5. Qiu, L. *et al.* Recognition and ubiquitination of Notch by Itch, a hect-type E3 ubiquitin ligase. *The Journal of biological chemistry* **275**, 35734–35737, doi: 10.1074/jbc.M007300200 (2000).
6. Liu, J. & Nussinov, R. Flexible cullins in cullin-RING E3 ligases allosterically regulate ubiquitination. *The Journal of biological chemistry* **286**, 40934–40942, doi: 10.1074/jbc.M111.277236 (2011).
7. Huang, J. *et al.* High-throughput screening for inhibitors of the e3 ubiquitin ligase APC. *Methods in enzymology* **399**, 740–754, doi: 10.1016/S0076-6879(05)99049-6 (2005).
8. Zhou, P. & Howley, P. M. Ubiquitination and degradation of the substrate recognition subunits of SCF ubiquitin-protein ligases. *Molecular cell* **2**, 571–580 (1998).
9. Passmore, L. A. & Barford, D. Getting into position: the catalytic mechanisms of protein ubiquitylation. *The Biochemical journal* **379**, 513–525, doi: 10.1042/BJ20040198 (2004).
10. Sarikas, A., Hartmann, T. & Pan, Z. Q. The cullin protein family. *Genome biology* **12**, 220, doi: 10.1186/gb-2011-12-4-220 (2011).
11. Ohta, A. *et al.* The CUL3-KLHL3 E3 ligase complex mutated in Gordon’s hypertension syndrome interacts with and ubiquitylates WNK isoforms: disease-causing mutations in KLHL3 and WNK4 disrupt interaction. *The Biochemical journal* **451**, 111–122, doi: 10.1042/BJ20121903 (2013).
12. Pintard, L., Willems, A. & Peter, M. Cullin-based ubiquitin ligases: Cul3-BTB complexes join the family. *The EMBO journal* **23**, 1681–1687, doi: 10.1038/sj.emboj.7600186 (2004).
13. Zhang, Q. *et al.* A hedgehog-induced BTB protein modulates hedgehog signaling by degrading Ci/Gli transcription factor. *Developmental cell* **10**, 719–729, doi: 10.1016/j.devcel.2006.05.004 (2006).
14. Kent, D., Bush, E. W. & Hooper, J. E. Roadkill attenuates Hedgehog responses through degradation of Cubitus interruptus. *Development* **133**, 2001–2010, doi: 10.1242/dev.02370 (2006).
15. Zhang, Q. *et al.* Multiple Ser/Thr-rich degrons mediate the degradation of Ci/Gli by the Cul3-HIB/SPOP E3 ubiquitin ligase. *Proceedings of the National Academy of Sciences of the United States of America* **106**, 21191–21196, doi: 10.1073/pnas.0912008106 (2009).
16. Duda, D. M. *et al.* Structural insights into NEDD8 activation of cullin-RING ligases: conformational control of conjugation. *Cell* **134**, 995–1006, doi: 10.1016/j.cell.2008.07.022 (2008).
17. Luke-Glaser, S., Pintard, L., Lu, C., Mains, P. E. & Peter, M. The BTB protein MEL-26 promotes cytokinesis in *C. elegans* by a CUL-3-independent mechanism. *Current biology: CB* **15**, 1605–1615, doi: 10.1016/j.cub.2005.07.068 (2005).
18. Shabek, N. *et al.* The size of the proteasomal substrate determines whether its degradation will be mediated by mono- or polyubiquitylation. *Molecular cell* **48**, 87–97, doi: 10.1016/j.molcel.2012.07.011 (2012).

19. Kulathu, Y. & Komander, D. Atypical ubiquitylation - the unexplored world of polyubiquitin beyond Lys48 and Lys63 linkages. *Nature reviews. Molecular cell biology* 13, 508–523, doi: 10.1038/nrm3394 (2012).

20. Ciechanover, A. The ubiquitin-proteasome proteolytic pathway. *Cell* 79, 13–21 (1994).

21. Geetha, T., Jiang, J. & Wooten, M. W. Lysine 63 polyubiquitination of the nerve growth factor receptor TrkA directs internalization and signaling. *Molecular cell* 20, 301–312, doi: 10.1016/j.molcel.2005.09.014 (2005).

22. Hershko, A. The ubiquitin system for protein degradation and some of its roles in the control of the cell division cycle. *Cell death and differentiation* 12, 1191–1197, doi: 10.1038/sj.cdd.4401702 (2005).

23. Liao, B. & Jin, Y. Wwp2 mediates Oct4 ubiquitination and its own auto-ubiquitination in a dosage-dependent manner. *Cell research* 20, 332–344, doi: 10.1038/cr.2009.136 (2010).

24. Wu, C. J., Conze, D. B., Li, T., Srinivasula, S. M. & Ashwell, J. D. Sensing of Lys 63-linked polyubiquitination by NEMO is a key event in NF-kappaB activation [corrected]. *Nature cell biology* 8, 398–406, doi: 10.1038/ncb1384 (2006).

25. Zhang, Z. *et al.* Ter94 ATPase complex targets k11-linked ubiquitinated ci to proteasomes for partial degradation. *Developmental cell* 25, 636–644, doi: 10.1016/j.devcel.2013.05.006 (2013).

26. Xu, L. *et al.* BTB proteins are substrate-specific adaptors in an SCF-like modular ubiquitin ligase containing CUL-3. *Nature* 425, 316–321, doi: 10.1038/nature01985 (2003).

27. Furukawa, M. & Xiong, Y. BTB protein Keap1 targets antioxidant transcription factor Nrf2 for ubiquitination by the Cullin 3-Roc1 ligase. *Molecular and cellular biology* 25, 162–171, doi: 10.1128/MCB.25.1.162-171.2005 (2005).

28. Liu, C. *et al.* Hedgehog signaling downregulates suppressor of fused through the HIB/SPOP-Crn axis in Drosophila. *Cell research* 24, 595–609, doi: 10.1038/cr.2014.29 (2014).

29. Zhuang, M. *et al.* Structures of SPOP-substrate complexes: insights into molecular architectures of BTB-Cul3 ubiquitin ligases. *Molecular cell* 36, 39–50, doi: 10.1016/j.molcel.2009.09.022 (2009).

30. Errington, W. J. *et al.* Adaptor protein self-assembly drives the control of a cullin-RING ubiquitin ligase. *Structure* 20, 1141–1153, doi: 10.1016/j.str.2012.04.009 (2012).

31. Espinas, M. L. *et al.* The N-terminal P O Z domain of GAGA mediates the formation of oligomers that bind DNA with high affinity and specificity. *The Journal of biological chemistry* 274, 16461–16469 (1999).

32. Katsani, K. R., Hajibagheri, M. A. & Verrijzer, C. P. Co-operative DNA binding by GAGA transcription factor requires the conserved BTB/POZ domain and reorganizes promoter topology. *The EMBO journal* 18, 698–708, doi: 10.1093/emboj/18.3.698 (1999).

33. Fujiyama-Nakamura, S. *et al.* BTB protein, dKLHL18/CG3571, serves as an adaptor subunit for a dCul3 ubiquitin ligase complex. *Genes to cells : devoted to molecular & cellular mechanisms* 14, 965–973, doi: 10.1111/j.1365-2443.2009.01323.x (2009).

34. Hochmuth, C. E., Biteau, B., Bohmann, D. & Jasper, H. Redox regulation by Keap1 and Nrf2 controls intestinal stem cell proliferation in Drosophila. *Cell stem cell* 8, 188–199, doi: 10.1016/j.stem.2010.12.006 (2011).

35. Strutt, H., Searle, E., Thomas-MacArthur, V., Brookfield, R. & Strutt, D. A Cul-3-BTB ubiquitylation pathway regulates junctional levels and asymmetry of core planar polarity proteins. *Development* 140, 1693–1702, doi: 10.1242/dev.089656 (2013).

36. Sykiotis, G. P. & Bohmann, D. Keap1/Nrf2 signaling regulates oxidative stress tolerance and lifespan in Drosophila. *Developmental cell* 14, 76–85, doi: 10.1016/j.devcel.2007.12.002 (2008).

37. Bae, S. H. *et al.* Sestrins activate Nrf2 by promoting p62-dependent autophagic degradation of Keap1 and prevent oxidative liver damage. *Cell metabolism* 17, 73–84, doi: 10.1016/j.cmet.2012.12.002 (2013).

38. Taguchi, K. *et al.* Keap1 degradation by autophagy for the maintenance of redox homeostasis. *Proceedings of the National Academy of Sciences of the United States of America* 109, 13561–13566, doi: 10.1073/pnas.1121572109 (2012).

39. Zhang, D. D. *et al.* Ubiquitination of Keap1, a BTB-Kelch substrate adaptor protein for Cul3, targets Keap1 for degradation by a proteasome-independent pathway. *The Journal of biological chemistry* 280, 30091–30099, doi: 10.1074/jbc.M501279200 (2005).

40. Lauwers, E., Jacob, C. & Andre, B. K63-linked ubiquitin chains as a specific signal for protein sorting into the multivesicular body pathway. *The Journal of cell biology* 185, 493–502, doi: 10.1083/jcb.200810114 (2009).

41. Lim, K. L. & Lim, G. G. K63-linked ubiquitination and neurodegeneration. *Neurobiology of disease* 43, 9–16, doi: 10.1016/j.nbd.2010.08.001 (2011).

42. Olzmann, J. A. & Chin, L. S. Parkin-mediated K63-linked polyubiquitination: a signal for targeting misfolded proteins to the aggresome-autophagy pathway. *Autophagy* 4, 85–87 (2008).

43. Galan, J. M. & Peter, M. Ubiquitin-dependent degradation of multiple F-box proteins by an autocatalytic mechanism. *Proceedings of the National Academy of Sciences of the United States of America* 96, 9124–9129 (1999).

44. Li, Y., Gazdoiu, S., Pan, Z. Q. & Fuchs, S. Y. Stability of homologue of Slimb F-box protein is regulated by availability of its substrate. *The Journal of biological chemistry* 279, 11074–11080, doi: 10.1074/jbc.M312301200 (2004).

45. Li, X., Zhang, D., Hannink, M. & Beamer, L. J. Crystal structure of the Kelch domain of human Keap1. *The Journal of biological chemistry* 279, 54750–54758, doi: 10.1074/jbc.M410073200 (2004).

46. Christie, J. M. *et al.* Plant UVR8 photoreceptor senses UV-B by tryptophan-mediated disruption of cross-dimer salt bridges. *Science* 335, 1492–1496, doi: 10.1126/science.1218091 (2012).

47. Hernandez-Munoz, I. *et al.* Stable X chromosome inactivation involves the PRC1 Polycomb complex and requires histone MACROH2A1 and the CULLIN3/SPOP ubiquitin E3 ligase. *Proceedings of the National Academy of Sciences of the United States of America* 102, 7635–7640, doi: 10.1073/pnas.0408918102 (2005).

48. Kwon, J. E. *et al.* BTB domain-containing speckle-type POZ protein (SPOP) serves as an adaptor of Daxx for ubiquitination by Cul3-based ubiquitin ligase. *The Journal of biological chemistry* 281, 12664–12672, doi: 10.1074/jbc.M600204200 (2006).

49. Kim, B. *et al.* Breast cancer metastasis suppressor 1 (BRMS1) is destabilized by the Cul3-SPOP E3 ubiquitin ligase complex. *Biochemical and biophysical research communications* 415, 720–726, doi: 10.1016/j.bbrc.2011.10.154 (2011).

50. Liu, Y. *et al.* Insight in glioma susceptibility through an analysis of 6p22.3, 12p13.33-12.1, 17q22-23.2 and 18q23 SNP genotypes in familial and non-familial glioma. *Human genetics* 131, 1507–1517, doi: 10.1007/s00439-012-1187-x (2012).

51. Barbieri, C. E. *et al.* Exome sequencing identifies recurrent SPOP, FOXA1 and MED12 mutations in prostate cancer. *Nature genetics* 44, 685–689, doi: 10.1038/ng.2279 (2012).

52. Le Gallo, M. *et al.* Exome sequencing of serous endometrial tumors identifies recurrent somatic mutations in chromatin-remodeling and ubiquitin ligase complex genes. *Nature genetics* 44, 1310–1315, doi: 10.1038/ng.2455 (2012).

53. Bunce, M. W., Boronenkov, I. V. & Anderson, R. A. Coordinated activation of the nuclear ubiquitin ligase Cul3-SPOP by the generation of phosphatidylinositol 5-phosphate. *The Journal of biological chemistry* 283, 8678–8686, doi: 10.1074/jbc.M7102222200 (2008).

54. Rubin, G. M. & Spradling, A. C. Genetic transformation of Drosophila with transposable element vectors. *Science* 218, 348–353 (1982).

55. Theodosiou, N. A. & Xu, T. Use of FLP/FRT system to study Drosophila development. *Methods* 14, 355–365, doi: 10.1006/meth.1998.0591 (1998).

Acknowledgements
We apologize to colleagues whose works were not cited owing to space limitations. We thank Dr. Jin Jiang (University of Texas Southwestern Medical Center at Dallas, USA), The Bloomington Stock Center and Developmental Studies Hybridoma Bank at the University of Iowa for providing fly stocks and reagents. This work was supported by grants from the National Key Scientific Program of China (2011CB943902) and the National Natural Science Foundation of China (30971679, 31071264 and 31271531).

Author Contributions
The authors have made the following declarations about their contributions: Conceived and designed the experiments: Q.Z. and Z.Z. Performed the experiments: Z.Z., C.X., P.C., C.L., S.P. and X.Y. Analyzed the data: Q.Z. and Z.Z. Wrote the paper: Q.Z. and Z.Z.

Additional Information
Supplementary information accompanies this paper at http://www.nature.com/srep

Competing financial interests: The authors declare no competing financial interests.

How to cite this article: Zhou, Z. *et al.* Stability of HIB-Cul3 E3 ligase adaptor HIB Is Regulated by Self-degradation and Availability of Its Substrates. *Sci. Rep.* **5**, 12709; doi: 10.1038/srep12709 (2015).

![Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/) This work is licensed under a Creative Commons Attribution 4.0 International License. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in the credit line; if the material is not included under the Creative Commons license, users will need to obtain permission from the license holder to reproduce the material. To view a copy of this license, visit http://creativecommons.org/licenses/by/4.0/
