
Accepted Manuscript

Role of autophagy in the pathogenesis of amyotrophic lateral sclerosis

Jae Keun Lee, Jin Hee Shin, Ji Eun Lee, Eui-Ju Choi

PII: S0925-4439(15)00226-4  
DOI: doi: 10.1016/j.bbadis.2015.08.005  
Reference: BBADIS 64284  

To appear in: BBA - Molecular Basis of Disease  

Received date: 2 March 2015  
Revised date: 6 August 2015  
Accepted date: 7 August 2015  

Please cite this article as: Jae Keun Lee, Jin Hee Shin, Ji Eun Lee, Eui-Ju Choi, Role of autophagy in the pathogenesis of amyotrophic lateral sclerosis, BBA - Molecular Basis of Disease (2015), doi: 10.1016/j.bbadis.2015.08.005  

This is a PDF file of an unedited manuscript that has been accepted for publication. As a service to our customers we are providing this early version of the manuscript. The manuscript will undergo copyediting, typesetting, and review of the resulting proof before it is published in its final form. Please note that during the production process errors may be discovered which could affect the content, and all legal disclaimers that apply to the journal pertain.

# Role of autophagy in the pathogenesis of amyotrophic lateral sclerosis

Jae Keun Lee${}^{1}$, Jin Hee Shin${}^{2}$, Ji Eun Lee${}^{3,4,*}$, and Eui-Ju Choi${}^{1,*}$

${}^{1}$Laboratory of Cell Death and Human Diseases, Department of Life Sciences, Korea University, Seoul, Korea; ${}^{2}$Department of Microbiology, Ajou University School of Medicine, 206 World cup-ro, Yeongtong-gu, Suwon 443-749, Korea; ${}^{3}$Department of Health Sciences and Technology, Samsung Advanced Institute for Health Sciences & Technology (SAIHST), Sungkyunkwan University; ${}^{4}$Samsung Genome Institute (SGI), Samsung Medical Center, #81 Irwon-Dong Gangnam-Gu, Seoul 135-710, Korea

*Corresponding authors: Eui-Ju Choi (ejchoi@korea.ac.kr) and Ji Eun Lee (jieun.lee@skku.edu)

Abstract

Amyotrophic lateral sclerosis (ALS) is a late-onset neurodegenerative disease characterized by the selective degeneration of upper and lower motor neurons associated with the abnormal aggregation of ubiquitinated proteins. The molecular mechanisms underlying the pathogenesis of ALS remain unclear, however. Autophagy is a major pathway for the elimination of protein aggregates and damaged organelles and therefore contributes to cellular homeostasis. This catabolic process begins with the formation of the double membrane-bound autophagosome that engulfs portions of the cytoplasm and subsequently fuses with a lysosome to form an autolysosome, in which lysosomal enzymes digest autophagic substrates. Defects at various stages of autophagy have been associated with pathological mutations of several ALS-linked genes including SOD1, p62, TDP-43, and optineurin, suggesting that such defects may play a causative role in the pathogenesis of this condition. In this review, we summarize the dysregulation of autophagy associated with ALS as well as potential therapeutic strategies based on modulation of the autophagic process.

Key words: amyotrophic lateral sclerosis; autophagosome; autophagy; lysosome; neurodegeneration

# 1. Introduction

Amyotrophic lateral sclerosis (ALS) is an adult-onset neurological disorder that is characterized by muscle weakness and atrophy, paralysis, and eventual death by respiratory failure and which results from the selective degeneration of upper and lower motor neurons that project from the spinal cord, brain stem, and motor cortex [1]. Most cases of ALS are sporadic, with ~10% of patients experiencing a familial form of the disease [1]. Both sporadic and familial cases share similar clinical characteristics.

A pathological hallmark of ALS is the presence of cytoplasmic inclusions or protein aggregates in affected motor neurons [2], suggesting that impairment of protein degradation plays a role in the disease process. Unlike mitotic cells, which are able to clear aggregates of intracellular proteins through their dilution or asymmetric distribution during cell division [3], postmitotic neurons rely on two major pathways of protein degradation for aggregate removal: the ubiquitin-proteasome system (UPS) and autophagy-associated lysosomal degradation. Whereas the UPS mostly mediates the degradation of short-lived proteins conjugated with ubiquitin, the autophagy-based system preferentially targets long-lived proteins and damaged organelles [4]. Dysfunction of these two pathways has been implicated in the pathogenesis of various neurodegenerative diseases including ALS [4].

Although the precise role of autophagy in the pathogenesis of ALS is still under controversy, emerging evidence supports the notion that defects in autophagic flux may contribute to the demise of motor neurons and disease progression in ALS [5]. Various stages of the autophagy-dependent lysosomal degradation of protein aggregates may be impaired in individuals with this disease. In this review, we summarize the case for a potential role of autophagic dysfunction in the pathogenesis of ALS.

# 2. Autophagic processing

Autophagy is a catabolic process that is highly conserved from yeast and fungi to plants and mammals and which is responsible for the lysosomal degradation of proteins and damaged organelles [6]. Autophagy has been classified into three subtypes on the basis of how cargo is delivered to lysosomes: chaperone-mediated autophagy, microautophagy, and macroautophagy [7]. In chaperone-mediated autophagy, substrate proteins interact with
lysosome-associated membrane protein type 2 (LAMP2) and are subsequently transported into the lysosomal lumen for degradation [8]. Such substrates contain a consensus pentapeptide motif (KFERQ) that is recognized by a cytosolic chaperone, heat shock cognate 70 (HSC70) [9]. Microautophagy refers to the direct engulfment of cytoplasmic cargo by an autophagic tube, which forms by invagination of the lysosomal membrane and undergoes vesicle scission into the lysosomal lumen [10]. Macroautophagy (hereafter referred to as autophagy) is mediated by a rearrangement of subcellular membranes that results in the envelopment of cytosol or organelles for delivery into lysosomes, where the enveloped cargo is digested and recycled [6].

Autophagy is initiated by the formation of a phagophore (or isolation membrane) in a manner dependent on multiple signaling molecules (Fig. 1). The phagophore undergoes elongation and is transformed into a double membrane-bound autophagosome [7]. The autophagosome is transported along a microtubule track toward the microtubule-organizing center, around which lysosomes are enriched [11]. During this trafficking process, the autophagosome undergoes fusion first with endosomal vesicles or multivesicular bodies (also known as amphisomes) and then with a lysosome, resulting in formation of an autolysosome. Finally, the contents of the autophagosome are digested by lysosomal acidic hydrolases, yielding basic metabolites that can be used for new synthetic processes or as an energy source. Many key regulators of autophagy, termed autophagy-related (Atg) proteins, were first identified in yeast but are evolutionarily conserved from yeast to mammals [12].

The autophagic process may be divided into three main stages: initiation, maturation, and degradation. Defects at any of these stages may give rise to neurodegeneration including that associated with ALS [5, 13, 14]. We will now focus on each stage to shed light on the potential role of autophagic defects in the pathogenesis of ALS.

## 3. ALS-associated defects at different stages of autophagy

### 3.1. Initiation stage: substrate recognition and autophagosome formation

Initiation of autophagy involves the rearrangement of subcellular membranes to allow the sequestration of cargo as well as the consecutive formation of a protein-kinase autophagy regulatory complex and a lipid-kinase signaling complex. The rearrangement of subcellularmembranes requires recruitment of Atg proteins to the phagophore and leads to autophagosome formation. The initial event, referred to as vesicle nucleation, is induced by phosphorylation (activation) of the Unc51-like kinase 1 (ULK1)-Atg13-FIP200 complex under the control of mammalian target of rapamycin (mTOR) complex 1 (mTORC1) or AMP-activated protein kinase (AMPK) [15, 16]. The activated ULK1-Atg13-FIP200 complex promotes movement of a multiprotein complex containing Beclin-1 (mammalian ortholog of yeast Atg6) and class III phosphoinositide 3-kinase (PI3KCIII, also known as Vps34) to the pre-autophagosomal structure (PAS), thereby triggering autophagosome formation [17] (Fig. 1A). Elongation of the PAS and its encapsulation of a cytoplasmic cargo proceed under the control of two ubiquitin-like conjugation systems: the Atg5-Atg12 conjugation system and the microtubule-associated protein 1A/1B-light chain 3 (LC3) conjugation system [18]. The Atg5-Atg12 covalent conjugate, whose formation is mediated by sequential action of the E1-like enzyme Atg7 and the E2-like enzyme Atg10, possesses an E3-like ligase activity that promotes the conjugation of LC3 to phosphatidylethanolamine to produce lipidated LC3-II, which localizes to the autophagosomal membrane [19, 20] (Fig. 1A). The lipidated LC3-II remains membrane-anchored during subsequent autophagosome maturation and serves as a key marker for this structure [21].

Genetic ablation of Atg5 or Atg7—both of which are required for autophagosome formation—in the central nervous system of mice gives rise to motor deficits that are associated with the spontaneous accumulation of ubiquitin-positive protein aggregates in motor neurons, suggesting that defective autophagosome formation may contribute to the pathogenesis of ALS [22, 23]. Furthermore, an ALS-linked mutant of superoxide dismutase 1 (SOD1) interacts with Beclin-1 and thereby destabilizes its association with Bcl-xL and impedes the initiation of autophagy, leading to a defective autophagy flux. This effect of the mutant SOD1 protein may be associated with neurotoxicity in a mouse model of ALS [24] (Fig. 2A, Table 1).

Whereas autophagy contributes to the nonselective digestion of cytosolic proteins, it also mediates the selective degradation of various ubiquitin-conjugated substrates including protein aggregates, intracellular organelles, and microorganisms [25, 26]. Autophagy receptors bind the ubiquitinated cargo and link it to autophagosome-associated ubiquitin-like proteins such as LC3 and GABARAP proteins [27, 28]. These receptors are thus thought to

5
play a key role in recognition of ubiquitinated cargo for engulfment, a process that appears to be prone to disruption in neurodegenerative diseases [13, 29]. Missense mutations in the autophagy receptor p62 (SQSTM1) have been identified in both familial and sporadic cases of ALS [30]. Genetic ablation of p62 also results in an autophagy defect as well as impaired locomotor activity such as swimming deficits associated with shorter motor axons in zebrafish [31] (Fig. 2B, Table 1). Moreover, p62 promotes the autophagy-dependent degradation of an ALS-linked SOD1 mutant as well as aggregates of TAR DNA-binding protein-43 (TDP-43), a major component of ALS-associated cytoplasmic inclusions [32, 33]. Rare ALS-associated mutations in the gene for optineurin (OPTN), another autophagy receptor, interfere with autophagy-mediated degradation of protein aggregates and damaged mitochondria [34-36] (Fig. 2B, Table 1). TANK-binding kinase 1 (TBK1), which phosphorylates two autophagy receptors p62 and OPTN [39, 40], has been recently identified as another ALS-causing gene [37, 38]. The TBK1-mediated phosphorylation of the autophagy receptors may modulate the recognition of autophagic cargoes by the receptors, leading to a defect in autophagic flux. A more detailed mechanism by which TBK1 plays a role in the pathogenesis of ALS needs to be further studied.

### 3.2. Maturation stage: fusion of the autophagosome with a lysosome

Maturation of the autophagosome into an autolysosome is a dynamic process that includes serial fusion of the autophagosome first with late endosomes or amphisomes and then with a lysosome [41]. The maturation process requires vesicle trafficking, which promotes productive encounters of autophagosomes with lysosomes in a microtubule- and dynein-dependent manner [42, 43] (Fig. 1B). This movement of the autophagosome in neurons is highly susceptible to interruption, given that most autophagosomes are formed in axons and that axon terminals are far from lysosomes in the cell body. Movement of autophagosomes to the cell body thus requires retrograde transport in neurons [44]. Impaired trafficking gives rise to the abnormal accumulation of autophagosomes and p62- or ubiquitin-positive substrates in neurons and is implicated as a common pathology in many neurodegenerative diseases [45, 46].

Dynactin serves as an adaptor to link dynein with cargo along the tracks of microtubules

and thereby regulates the motor function of dynein [47]. Point mutations in the gene for the p150 subunit of dynactin have been found to give rise to motor neuron diseases including familial ALS as a consequence of defects in the transport of autophagosomes to lysosomes [48, 49]. The accumulation of p62-or ubiquitin-positive aggregates has been associated with reduced activity of the dynactin-dynein complex in both clinical and experimental studies [50, 51] (Fig. 2C, Table 1). Defective dynein may also disrupt retrograde transport of autophagosomes to lysosomes and thereby give rise to abnormal accumulation of aggregates. Thus, mutations in the gene for dynein may be associated with the demise of motor neurons in a mouse model of familial ALS [52, 53] (Fig. 2C, Table 1). Furthermore, mutant SOD1 interacts physically with the dynein complex and perturbs retrograde axonal transport [54] (Fig. 2C, Table 1). Thus, dysfunction of the dynactin-dynein complex may be implicated in impaired maturation of autophagosomes in ALS. However, a role of dynein in the pathogenesis of ALS appears to be more complicated than expected. In this regard, it is noteworthy to mention that Legs at odd angles (Loa) mutation in cytoplasmic dynein heavy chain (Dync1h1) delays disease onset and extends the lifespan in SOD1(G93A) mice partly due to improvement of mitochondrial abnormality shown in the mice [55, 56].

The fusion of autophagosomes with lysosomes requires various non-Atg proteins including ATPases, endosomal sorting complexes required for transport (ESCRT), Rab proteins, and soluble N-ethylmaleimide-sensitive factor-activating protein receptor (SNARE) proteins [57]. The fusion process may be also affected by ALS-linked mutations in the genes for valosin-containing protein (VCP, also known as p97) and charged multivesicular body protein 2B (CHMP2B) [58, 59]. VCP is a member of the AAA-type ATPase family and interacts with various ubiquitin-conjugated proteins, thereby facilitating recycling or degradation of proteins either by the UPS or by the autophagy-dependent lysosome system [60]. Loss of VCP activity results in the accumulation of immature autophagosomes containing ubiquitin-positive substrates [61, 62] (Fig. 2D, Table 1). Similar to those in the VCP gene, mutations in the gene for CHMP2B (ESCRT-III subunit) lead to ALS and frontotemporal dementia presumably through impairment of the autophagosome-lysosome fusion process [59, 63] (Fig. 2D, Table 1). Ectopic expression of disease-linked mutants of either VCP or CHMP2B in cultured neurons results in an increase in the number of autophagosomes with p62-positive substrates as well as cytoplasmic accumulation of TDP-43
[61, 64, 65], suggesting that fusion-related ATPase and ESCRT proteins play an important role in the regulation of autophagosomal maturation and intracellular dynamics of TDP-43.

Ectopic P granules protein 5 homolog (Epg5), which is associated with autophagosome maturation and endocytic trafficking [66], is also implicated in ALS pathogenesis. Genetic ablation of Epg5 in mice thus gives rise to ALS-like features such as functional impairment of the endosomal sorting complex and accumulation of nondegradative autophagic vacuoles in association with selective degeneration both of pyramidal neurons in cortical layer 5 and of spinal cord motor neurons [67].

### 3.3. Degradation stage: lysosomal clearance of autophagic substrates

A distinctive feature of lysosomes is their acidic lumen (pH 4.5–5.0), which supports the efficient catabolic function of hydrolytic enzymes such as proteases, lipases, and nucleases [68]. Lysosomal acidification is maintained by a proton pump—the vacuolar ATPase (v-ATPase)—as well as by lysosomal membrane proteins such as LAMP1 and LAMP2 that protect the lysosomal membrane from self-digestion [69]. Such components thus contribute to the digestion of autophagic substrates in the autolysosome (Fig. 1C). The integrity of the lysosomal membrane is compromised under various neuropathologic conditions, suggesting that lysosomal membrane proteins may be potential therapeutic targets [69-71].

In addition to the direct role of lysosomes in the proteolytic degradation of autophagic substrates, lysosome biogenesis is linked to the initiation stage of autophagy in a manner dependent on the basic helix-loop-helix leucine-zipper transcription factor TFEB. TFEB thus regulates the expression of genes related to lysosome biogenesis and autophagosome formation [72]. mTORC1 interacts with and phosphorylates TFEB under basal conditions and thereby prevents its translocation to the nucleus [73-75]. Such phosphorylation is also mediated by extracellular signal-regulated kinase 2 (ERK2) and by the β isoform of protein kinase C (PKCβ) [76, 77]. In comparison, de-phosphorylation of TFEB by calcineurin promotes its activation and nuclear translocation [78]. Inhibition of mTORC1 activity triggers the activation of TFEB and its translocation to the nucleus, resulting in trans-activation of its target genes [73-76]. Manipulations that increase the trans-activation activity of TFEB, including ectopic expression of TFEB or depletion of its repressor ZKSCAN3, promote
autophagic flux by enhancing both autophagosome formation and lysosome biogenesis [76, 79]. Impairment of lysosomal function also appears to be associated with the pathogenesis of Alzheimer's disease and Parkinson's disease [80-82], while a causative role of lysosomal dysfunction is relatively less studied in ALS.

4. Therapeutic modulation of autophagy in ALS

The evidence implicating defective autophagy in the pathogenesis of neurodegenerative diseases supports enhancement of autophagy as a potential therapeutic strategy. Such an approach might thus be expected to eliminate toxic protein aggregates in neurons by maintaining lysosomal integrity or promoting autophagic flux [83-85]. The potential therapeutic effects of various inducers of autophagy in ALS are listed in Table 2. Chronic administration of autophagy-inducing drugs such as the mTOR inhibitor rapamycin protected motor neurons and improved both motor performance and clearance of toxic TDP-43 aggregates in a mouse model of TDP-43 proteinopathy [86, 87], although its mechanism is not fully understood. Beneficial effects of rapamycin on lifespan and locomotor activity were also apparent in a Drosophila model of ALS with transgenic expression of dTDP, the Drosophila ortholog of TDP-43 [88]. Several mutations in fused in sarcoma (FUS), a multifunctional DNA/RNA binding protein, are associated with familial cases of ALS [89] and cytosolic inclusions (aggregates) of these mutants are highly localized in stress granules [90]. Rapamycin-induced autophagy not only reduced the accumulation of FUS-positive stress granules, but also prevented fragmentation of neurites and neurotoxicity in cultured cortical neurons ectopically expressing ALS-linked FUS(R521C) [91]. In comparison, rapamycin failed to exhibit a beneficial effect on motor neuron survival in other ALS model mice expressing pathological mutants of either SOD1 or VCP [92-95]. Moreover, chronic treatment of rapamycin results in more severe impairment in mitochondrial integrity, higher levels of pro-apoptotic Bax, and caspase-3 activation in spinal motor neurons in SOD1(G93A) mice, all of which may contribute to demise of motor neurons [92]. Intriguingly, deficiency of mature lymphocytes potentiated a therapeutic effect of rapamycin in ALS mice expressing a human SOD1(G93A) transgene [96]. It remains unclear how rapamycin has shown either protective or destructive effects in different mouse models of ALS.

Activation of AMPK by mood stabilizers may be another route to autophagy induction with potential beneficial effects on several neurodegenerative diseases including ALS [97]. For example, several studies have reported that lithium ameliorates ALS-like features in mice transgenic for an ALS-linked SOD1 mutant, presumably due to modulation of various cellular processes including autophagy [98, 99]. However, the therapeutic effect of lithium on ALS should be interpreted with caution. The beneficial effect of lithium in mouse models [100, 101] and in clinical trials [102, 103] is still controversial. In detail, while the neuroprotective effect of lithium was observed at 0.4-0.8 mEq/l in SOD(G93A) mice [98, 99], the adverse effect was at over 1.0 mEq/l in the same animals [100, 101]. The disaccharide trehalose acts as a chemical chaperone through AMPK activation, and it was found to promote clearance of misfolded proteins and to confer neuroprotection in models of proteinopathy induced by mutant forms of huntingtin, α-synuclein, or tau [104-106]. Administration of trehalose also facilitated induction of autophagy and autophagy-mediated degradation of mutant SOD1, thereby alleviating the severity of disease [107-109]. Besides of stimulating AMPK, trehalose can induce heat shock protein B8 (HspB8), which also facilitates autophagy with clearing autophagic substrates including mutant SOD1 [110, 111]. Collectively, these observations suggest that autophagy is a promising therapeutic target in neurodegenerative diseases [112, 113]. However, caution is warranted in such targeting of autophagy in the development of therapeutics for human diseases, given that most modulators of autophagy also have diverse autophagy-independent effects. Potential side effects of such drugs should therefore be thoroughly investigated.

5. Conclusion

Autophagy is a multistep degradative process that begins with formation of the autophagosome in a manner dependent on sequential regulation by the ULK1 complex (ULK1-Atg13-FIP200) and the PI3K CIII complex (Vps34–Vps15–Atg14–Beclin-1). The autophagosome loaded with target substrates fuses with late endosomes or amphisomes and then with a lysosome to form the autolysosome, in which acidic hydrolases degrade the autophagosome contents. Impairment of each of these steps in the autophagy process has been associated with a variety of human diseases including neurodegenerative disorders, reflecting the importance of autophagic flux for neuronal homeostasis. The vesicle trafficking

responsible for transport of the autophagosome to a lysosome is thought to be particularly susceptible to impairment in neurons given the relatively long distances that separate these two structures. Whereas malfunction of lysosomal hydrolases has been associated with neurodegeneration, its contribution to ALS pathology remains unclear.

A growing body of evidence implicates autophagy defects as a potential cause of ALS pathology and therefore provides a basis for the development of autophagy-targeted therapeutics. Impairment of autophagy may occur at multiple steps in individuals with ALS. Given that autophagy is required for a wide range of biological activities, not only for adaptive responses to pathological stresses but also for maintenance of intracellular homeostasis under physiological conditions, drugs that target this process must be evaluated carefully for potential side effects.

The nature of autophagy defects and their contribution to pathogenesis may differ among neurodegenerative diseases. It will be important that future studies address these core defects, the cellular responses to them, and their role in disease propagation. Such studies may provide a sounder basis for the development of autophagy-modulating drugs.

Acknowledgements

The work in the authors' laboratories was supported by National Research Foundation grants (2006-0093855, 2009-0080985 to E.-J.C., 2013R1A1A1059056 to J.E.L., 2014R1A1A1002076 to J.K.L., and 2014R1A1A3052540 to J.H.S.) funded by the Ministry of Science, ICT, and Future Planning of Korea, by Korea University grants (to E.-J.C. and to J.K.L), and by a grant from the Samsung Medical Center (GFO1150121 to J.E.L.).

Footnote

¹ Abbreviations: ALS, amyotrophic lateral sclerosis; LAMP2, lysosome-associated membrane protein type 2; PAS, pre-autophagosomal structure; UPS, ubiquitin-proteasome system;

References

[1] A. Al-Chalabi, O. Hardiman, The epidemiology of ALS: a conspiracy of genes, environment and time, Nat Rev Neurol, 9 (2013) 617-628.

[2] A.M. Blokhuis, E.J. Groen, M. Koppers, L.H. van den Berg, R.J. Pasterkamp, Protein aggregation in amyotrophic lateral sclerosis, Acta Neuropathol, 125 (2013) 777-794.

[3] M.A. Rujano, F. Bosveld, F.A. Salomons, F. Dijk, M.A. van Waarde, J.J. van der Want, R.A. de Vos, E.R. Brunt, O.C. Sibon, H.H. Kampinga, Polarised asymmetric inheritance of accumulated protein damage in higher eukaryotes, PLoS Biol, 4 (2006) e417.

[4] D.C. Rubinsztein, The roles of intracellular protein-degradation pathways in neurodegeneration, Nature, 443 (2006) 780-786.

[5] C.Y. Song, J.F. Guo, Y. Liu, B.S. Tang, Autophagy and Its Comprehensive Impact on ALS, Int J Neurosci, 122 (2012) 695-703.

[6] D.J. Klionsky, Autophagy: from phenomenology to molecular understanding in less than a decade, Nat Rev Mol Cell Biol, 8 (2007) 931-937.

[7] N. Mizushima, Autophagy: process and function, Genes Dev, 21 (2007) 2861-2873.

[8] J.F. Dice, Chaperone-mediated autophagy, Autophagy, 3 (2007) 295-299.

[9] A.M. Cuervo, J.F. Dice, E. Knecht, A population of rat liver lysosomes responsible for the selective uptake and degradation of cytosolic proteins, J Biol Chem, 272 (1997) 5606-5615.

[10] W.W. Li, J. Li, J.K. Bao, Microautophagy: lesser-known self-eating, Cell Mol Life Sci, 69 (2012) 1125-1136.

[11] P. Marambio, B. Toro, C. Sanhueza, R. Troncoso, V. Parra, H. Verdejo, L. Garcia, C. Quiroga, D. Munafo, J. Diaz-Elizondo, R. Bravo, M.J. Gonzalez, G. Diaz-Araya, Z. Pedrozo, M. Chiong, M.I. Colombo, S. Lavandero, Glucose deprivation causes oxidative stress and stimulates aggresome formation and autophagy in cultured cardiac myocytes, Biochim Biophys Acta, 1802 (2010) 509-518.

[12] N. Mizushima, T. Yoshimori, Y. Ohsumi, The role of Atg proteins in autophagosome formation, Annu Rev Cell Dev Biol, 27 (2011) 107-132.

[13] R.A. Nixon, The role of autophagy in neurodegenerative disease, Nat Med, 19 (2013) 983-997.

[14] S. Chen, X. Zhang, L. Song, W. Le, Autophagy dysregulation in amyotrophic lateral sclerosis, Brain Pathol, 22 (2012) 110-116.

[15] N. Mizushima, The role of the Atg1/ULK1 complex in autophagy regulation, Curr Opin Cell Biol, 22 (2010) 132-139.

[16] L. Shang, X. Wang, AMPK and mTOR coordinate the regulation of Ulk1 and mammalian autophagy initiation, Autophagy, 7 (2011) 924-926.

[17] A. Simonsen, S.A. Tooze, Coordination of membrane events during autophagy by multiple class III PI3-kinase complexes, J Cell Biol, 186 (2009) 773-782.

[18] Y. Ohsumi, N. Mizushima, Two ubiquitin-like conjugation systems essential for autophagy, Semin Cell Dev Biol, 15 (2004) 231-236.

[19] T. Hanada, N.N. Noda, Y. Satomi, Y. Ichimura, Y. Fujioka, T. Takao, F. Inagaki, Y. Ohsumi, The Atg12-Atg5 conjugate has a novel E3-like activity for protein lipidation in autophagy, J Biol Chem, 282 (2007) 37298-37302.

[20] H. Nakatogawa, Y. Ichimura, Y. Ohsumi, Atg8, a ubiquitin-like protein required for autophagosome formation, mediates membrane tethering and hemifusion, Cell, 130 (2007) 165-178.

[21] Y. Kabeya, N. Mizushima, T. Ueno, A. Yamamoto, T. Kirisako, T. Noda, E. Kominami, Y. Ohsumi, T. Yoshimori, LC3, a mammalian homologue of yeast Apg8p, is localized in autophagosome membranes after processing, EMBO J, 19 (2000) 5720-5728.

[22] T. Hara, K. Nakamura, M. Matsui, A. Yamamoto, Y. Nakahara, R. Suzuki-Migishima, M. Yokoyama, K. Mishima, I. Saito, H. Okano, N. Mizushima, Suppression of basal autophagy in neural cells causes neurodegenerative disease in mice, Nature, 441 (2006) 885-889.

[23] M. Komatsu, S. Waguri, T. Chiba, S. Murata, J. Iwata, I. Tanida, T. Ueno, M. Koike, Y. Uchiyama, E. Kominami, K. Tanaka, Loss of autophagy in the central nervous system causes neurodegeneration in mice, Nature, 441 (2006) 880-884.

[24] M. Nassif, V. Valenzuela, D. Rojas-Rivera, R. Vidal, S. Matus, K. Castillo, Y. Fuentealba, G. Kroemer, B. Levine, C. Hetz, Pathogenic role of BECN1/Beclin 1 in the development of amyotrophic lateral sclerosis, Autophagy, 10 (2014) 1256-1271.

[25] Z. Xie, D.J. Klionsky, Autophagosome formation: core machinery and adaptations, Nat Cell Biol, 9 (2007) 1102-1109.

[26] V. Kirkin, D.G. McEwan, I. Novak, I. Dikic, A role for ubiquitin in selective autophagy, Mol Cell, 34 (2009) 259-269.

[27] M. Filimonenko, P. Isakson, K.D. Finley, M. Anderson, H. Jeong, T.J. Melia, B.J. Bartlett, K.M. Myers, H.C. Birkeland, T. Lamark, D. Krainc, A. Brech, H. Stenmark, A. Simonsen, A. Yamamoto, The selective macroautophagic degradation of aggregated proteins requires the PI3P-binding protein Alfy, Mol Cell, 38 (2010) 265-279.

[28] V. Rogov, V. Dotsch, T. Johansen, V. Kirkin, Interactions between autophagy receptors and ubiquitin-like proteins form the molecular basis for selective autophagy, Mol Cell, 53 (2014) 167-178.

[29] A. Stolz, A. Ernst, I. Dikic, Cargo recognition and trafficking in selective autophagy, Nat Cell Biol, 16 (2014) 495-501.

[30] F. Fecto, J. Yan, S.P. Vemula, E. Liu, Y. Yang, W. Chen, J.G. Zheng, Y. Shi, N. Siddique, H. Arrat, S. Donkervoort, S. Ajroud-Driss, R.L. Sufit, S.L. Heller, H.X. Deng, T. Siddique, SQSTM1 mutations in familial and sporadic amyotrophic lateral sclerosis, Arch Neurol, 68 (2011) 1440-1446.

[31] S. Lattante, H. de Calbiac, I. Le Ber, A. Brice, S. Ciura, E. Kabashi, Sqstm1 knock-

down causes a locomotor phenotype ameliorated by rapamycin in a zebrafish model of ALS/FTLD, Hum Mol Genet, (2014).

[32] J. Gal, A.L. Strom, D.M. Kwinter, R. Kilty, J. Zhang, P. Shi, W. Fu, M.W. Wooten, H. Zhu, Sequestosome 1/p62 links familial ALS mutant SOD1 to LC3 via an ubiquitin-independent mechanism, J Neurochem, 111 (2009) 1062-1073.

[33] O.A. Brady, P. Meng, Y. Zheng, Y. Mao, F. Hu, Regulation of TDP-43 aggregation by phosphorylation and p62/SQSTM1, J Neurochem, 116 (2011) 248-259.

[34] H. Maruyama, H. Morino, H. Ito, Y. Izumi, H. Kato, Y. Watanabe, Y. Kinoshita, M. Kamada, H. Nodera, H. Suzuki, O. Komure, S. Matsuura, K. Kobatake, N. Morimoto, K. Abe, N. Suzuki, M. Aoki, A. Kawata, T. Hirai, T. Kato, K. Ogasawara, A. Hirano, T. Takumi, H. Kusaka, K. Hagiwara, R. Kaji, H. Kawakami, Mutations of optineurin in amyotrophic lateral sclerosis, Nature, 465 (2010) 223-226.

[35] W.C. Shen, H.Y. Li, G.C. Chen, Y. Chern, P.H. Tu, Mutations in the ubiquitin-binding domain of OPTN/optineurin interfere with autophagy-mediated degradation of misfolded proteins by a dominant-negative mechanism, Autophagy, (2014) e36098.

[36] Y.C. Wong, E.L. Holzbaur, Optineurin is an autophagy receptor for damaged mitochondria in parkin-mediated mitophagy that is disrupted by an ALS-linked mutation, Proc Natl Acad Sci USA, 111 (2014) E4439-4448.

[37] E.T. Cirulli, B.N. Lasseigne, S. Petrovski, P.C. Sapp, P.A. Dion, C.S. Leblond, J. Couthouis, Y.F. Lu, Q. Wang, B.J. Krueger, Z. Ren, J. Keebler, Y. Han, S.E. Levy, B.E. Boone, J.R. Wimbish, L.L. Waite, A.L. Jones, J.P. Carulli, A.G. Day-Williams, J.F. Staropoli, W.W. Xin, A. Chesi, A.R. Raphael, D. McKenna-Yasek, J. Cady, J.M. Vianney de Jong, K.P. Kenna, B.N. Smith, S. Topp, J. Miller, A. Gkazi, A. Al-Chalabi, L.H. van den Berg, J. Veldink, V. Silani, N. Ticozzi, C.E. Shaw, R.H. Baloh, S. Appel, E. Simpson, C. Lagier-Tourenne, S.M. Pulst, S. Gibson, J.Q. Trojanowski, L. Elman, L. McCluskey, M. Grossman, N.A. Shneider, W.K. Chung, J.M. Ravits, J.D. Glass, K.B. Sims, V.M. Van Deerlin, T. Maniatis, S.D. Hayes, A. Ordureau, S. Swarup, J. Landers, F. Baas, A.S. Allen, R.S. Bedlack, J.W. Harper, A.D. Gitler, G.A. Rouleau, R. Brown, M.B. Harms, G.M. Cooper, T. Harris, R.M. Myers, D.B. Goldstein, Exome sequencing in amyotrophic lateral sclerosis identifies risk genes and pathways, Science, 347 (2015) 1436-1441.

[38] A. Freischmidt, T. Wieland, B. Richter, W. Ruf, V. Schaeffer, K. Muller, N. Marroquin, F. Nordin, A. Hubers, P. Weydt, S. Pinto, R. Press, S. Millecamps, N. Molko, E. Bernard, C. Desnuelle, M.H. Soriani, J. Dorst, E. Graf, U. Nordstrom, M.S. Feiler, S. Putz, T.M. Boeckers, T. Meyer, A.S. Winkler, J. Winkelmann, M. de Carvalho, D.R. Thal, M. Otto, T. Brannstrom, A.E. Volk, P. Kursula, K.M. Danzer, P. Lichtner, I. Dikic, T. Meitinger, A.C. Ludolph, T.M. Strom, P.M. Andersen, J.H. Weishaupt, Haploinsufficiency of TBK1 causes familial ALS and fronto-temporal dementia, Nat Neurosci, 18 (2015) 631-636.

[39] M. Pilli, J. Arko-Mensah, M. Ponpuak, E. Roberts, S. Master, M.A. Mandell, N. Dupont, W. Ornatowski, S. Jiang, S.B. Bradfute, J.A. Bruun, T.E. Hansen, T. Johansen, V. Deretic, TBK-1 promotes autophagy-mediated antimicrobial defense by controlling autophagosome maturation, Immunity, 37 (2012) 223-234.

[40] P. Wild, H. Farhan, D.G. McEwan, S. Wagner, V.V. Rogov, N.R. Brady, B. Richter, J. Korac, O. Waidmann, C. Choudhary, V. Dotsch, D. Bumann, I. Dikic, Phosphorylation of the autophagy receptor optineurin restricts Salmonella growth, Science, 333 (2011) 228-233.

[41] A. Longatti, S.A. Tooze, Vesicular trafficking and autophagosome formation, Cell Death Differ, 16 (2009) 956-965.

[42] E. Fass, E. Shvets, I. Degani, K. Hirschberg, Z. Elazar, Microtubules support production of starvation-induced autophagosomes but not their targeting and fusion with lysosomes, J Biol Chem, 281 (2006) 36303-36316.

[43] S. Kimura, T. Noda, T. Yoshimori, Dynein-dependent movement of autophagosomes mediates efficient encounters with lysosomes, Cell Struct Funct, 33 (2008) 109-122.

[44] Q.J. Wang, Y. Ding, D.S. Kohtz, N. Mizushima, I.M. Cristea, M.P. Rout, B.T. Chait, Y. Zhong, N. Heintz, Z. Yue, Induction of autophagy in axonal dystrophy and degeneration, J Neurosci, 26 (2006) 8057-8068.

[45] B. Boland, R.A. Nixon, Neuronal macroautophagy: from development to degeneration, Mol Aspects Med, 27 (2006) 503-519.

[46] Z. Yue, L. Friedman, M. Komatsu, K. Tanaka, The cellular pathways of neuronal autophagy and their implication in neurodegenerative diseases, Biochim Biophys Acta, 1793 (2009) 1496-1507.

[47] J. Eschbach, L. Dupuis, Cytoplasmic dynein in neurodegeneration, Pharmacol Ther, 130 (2011) 348-363.

[48] I. Puls I, C. Jonnakuty, B.H. LaMonte, E.L. Holzbaur, M. Tokito, E. Mann, M.K. Floeter, K. Bidus, D. Drayna, S.J. Oh, R.H. Brown, Jr., C.L. Ludlow, K.H. Fischbeck, Mutant dynactin in motor neuron disease, Nat Genet, 33 (2003) 455-456.

[49] C. Munch, R. Sedlmeier, T. Meyer, V. Homberg, A.D. Sperfeld, A. Kurt, J. Prudlo, G. Peraus, C.O. Hanemann, G. Stumm, A.C. Ludolph, Point mutations of the p150 subunit of dynactin (DCTN1) gene in ALS, Neurology, 63 (2004) 724-726.

[50] S. Sasaki, Autophagy in spinal cord motor neurons in sporadic amyotrophic lateral sclerosis, J Neuropathol Exp Neurol, 70 (2011) 349-359.

[51] B. Ravikumar, A. Acevedo-Arozena, S. Imarisio, Z. Berger, C. Vacher, C.J. O'Kane, S.D. Brown, D.C. Rubinsztein, Dynein mutations impair autophagic clearance of aggregate-prone proteins, Nat Genet, 37 (2005) 771-776.

[52] M. Hafezparast, R. Klocke, C. Ruhrberg, A. Marquardt, A. Ahmad-Annuar, S. Bowen, G. Lalli, A.S. Witherden, H. Hummerich, S. Nicholson, P.J. Morgan, R. Oozageer, J.V. Priestley, S. Averill, V.R. King, S. Ball, J. Peters, T. Toda, A. Yamamoto, Y. Hiraoka, M. Augustin, D. Korthaus, S. Wattler, P. Wabnitz, C. Dickneite, S. Lampel, F. Boehme, G. Peraus, A. Popp, M. Rudelius, J. Schlegel, H. Fuchs, M. Hrabe de Angelis, G. Schiavo, D.T. Shima, A.P. Russ, G. Stumm, J.E. Martin, E.M. Fisher, Mutations in dynein link motor neuron degeneration to defects in retrograde transport, Science, 300 (2003) 808-812.

[53] H.S. Ilieva, K. Yamanaka, S. Malkmus, O. Kakinohana, T. Yaksh, M. Marsala, D.W.

Cleveland, Mutant dynein (Loa) triggers proprioceptive axon loss that extends survival only in the SOD1 ALS model with highest motor neuron death, Proc Natl Acad Sci USA, 105 (2008) 12599-12604.

[54] F. Zhang, A.L. Strom, K. Fukada, S. Lee, L.J. Hayward, H. Zhu, Interaction between familial amyotrophic lateral sclerosis (ALS)-linked SOD1 mutants and the dynein complex, J Biol Chem, 282 (2007) 16691-16699.

[55] D. Kieran, M. Hafezparast, S. Bohnert, J.R. Dick, J. Martin, G. Schiavo, E.M. Fisher, L. Greensmith, A mutation in dynein rescues axonal transport defects and extends the life span of ALS mice, J Cell Biol, 169 (2005) 561-567.

[56] A.M. El-Kadi, V. Bros-Facer, W. Deng, A. Philpott, E. Stoddart, G. Banks, G.S. Jackson, E.M. Fisher, M.R. Duchen, L. Greensmith, A.L. Moore, M. Hafezparast, The legs at odd angles (Loa) mutation in cytoplasmic dynein ameliorates mitochondrial function in SOD1G93A mouse model for motor neuron disease, J Biol Chem, 285 (2010) 18627-18639.

[57] D. Ritz, M. Vuk, P. Kirchner, M. Bug, S. Schutz, A. Hayer, S. Bremer, C. Lusk, R.H. Baloh, H. Lee, T. Glatter, M. Gstaiger, R. Aebersold, C.C. Weihl, H. Meyer, Endolysosomal sorting of ubiquitylated caveolin-1 is regulated by VCP and UBXD1 and impaired by VCP disease mutations, Nat Cell Biol, 13 (2011) 1116-1123.

[58] J.O. Johnson, J. Mandrioli, M. Benatar, Y. Abramzon, V.M. Van Deerlin, J.Q. Trojanowski, J.R. Gibbs, M. Brunetti, S. Gronka, J. Wuu, J. Ding, L. McCluskey, M. Martinez-Lage, D. Falcone, D.G. Hernandez, S. Arepalli, S. Chong, J.C. Schymick, J. Rothstein, F. Landi, Y.D. Wang, A. Calvo, G. Mora, M. Sabatelli, M.R. Monsurro, S. Battistini, F. Salvi, R. Spataro, P. Sola, G. Borghero, G. Galassi, S.W. Scholz, J.P. Taylor, G. Restagno, A. Chio, B.J. Traynor, Exome sequencing reveals VCP mutations as a cause of familial ALS, Neuron, 68 (2010) 857-864.

[59] N. Parkinson, P.G. Ince, M.O. Smith, R. Highley, G. Skibinski, P.M. Andersen, K.E. Morrison, H.S. Pall, O. Hardiman, J. Collinge, P.J. Shaw, E.M. Fisher, ALS phenotypes with mutations in CHMP2B (charged multivesicular body protein 2B), Neurology, 67 (2006) 1074-1077.

[60] H. Meyer, M. Bug, S. Bremer, Emerging functions of the VCP/p97 AAA-ATPase in the ubiquitin system, Nat Cell Biol, 14 (2012) 117-123.

[61] J.S. Ju, R.A. Fuentealba, S.E. Miller, E. Jackson, D. Piwnica-Worms, R.H. Baloh, C.C. Weihl, Valosin-containing protein (VCP) is required for autophagy and is disrupted in VCP disease, J Cell Biol, 187 (2009) 875-888.

[62] E. Tresse, F.A. Salomons, J. Vesa, L.C. Bott, V. Kimonis, T.P. Yao, N.P. Dantuma, J.P. Taylor, VCP/p97 is essential for maturation of ubiquitin-containing autophagosomes and this function is impaired by mutations that cause IBMPFD, Autophagy, 6 (2010) 217-227.

[63] G. Skibinski, N.J. Parkinson, J.M. Brown, L. Chakrabarti, S.L. Lloyd, H. Hummerich, J.E. Nielsen, J.R. Hodges, M.G. Spillantini, T. Thusgaard, S. Brandner, A. Brun, M.N. Rossor, A. Gade, P. Johannsen, S.A. Sorensen, S. Gydesen, E.M. Fisher, J. Collinge, Mutations in the endosomal ESCRTIII-complex subunit CHMP2B in frontotemporal dementia, Nat Genet, 37 (2005) 806-808.

[64] M. Filimonenko, S. Stuffers, C. Raiborg, A. Yamamoto, L. Malerod, E.M. Fisher, A. Isaacs, A. Brech, H. Stenmark, A. Simonsen, Functional multivesicular bodies are required for autophagic clearance of protein aggregates associated with neurodegenerative disease, J Cell Biol, 179 (2007) 485-500.

[65] M. Neumann, D.M. Sampathu, L.K. Kwong, A.C. Truax, M.C. Micsenyi, T.T. Chou, J. Bruce, T. Schuck, M. Grossman, C.M. Clark, L.F. McCluskey, B.L. Miller, E. Masliah, I.R. Mackenzie, H. Feldman, W. Feiden, H.A. Kretzschmar, J.Q. Trojanowski, V.M. Lee, Ubiquitinated TDP-43 in frontotemporal lobar degeneration and amyotrophic lateral sclerosis, Science, 314 (2006) 130-133.

[66] Y. Tian, H. Ren, Y. Zhao, Q. Lu, X. Huang, P. Yang, H. Zhang, Four metazoan autophagy genes regulate cargo recognition, autophagosome formation and autolysosomal degradation, Autophagy, 6 (2010) 984-985.

[67] Zhao, Y.G. Zhao, X. Wang, L. Xu, L. Miao, D. Feng, Q. Chen, A.L. Kovacs, D. Fan, H. Zhang, Mice deficient in Epg5 exhibit selective neuronal vulnerability to degeneration, J Cell Biol, 200 (2013) 731-741.

[68] J.A. Mindell, Lysosomal acidification mechanisms, Annu Rev Physiol, 74 (2012) 69-86.

[69] P. Saftig, J. Klumperman, Lysosome biogenesis and lysosomal membrane proteins: trafficking meets function, Nat Rev Mol Cell Biol, 10 (2009) 623-635.

[70] H. Appelqvist, L. Sandin, K. Bjornstrom, P. Saftig, B. Garner, K. Ollinger, K. Kagedal, Sensitivity to lysosome-dependent cell death is directly regulated by lysosomal cholesterol content, PLoS One, 7 (2012) e50262.

[71] A. Armstrong, N. Mattsson, H. Appelqvist, C. Janefjord, L. Sandin, L. Agholme, B. Olsson, S. Svensson, K. Blennow, H. Zetterberg, K. Kagedal, Lysosomal network proteins as potential novel CSF biomarkers for Alzheimer's disease, Neuromolecular Med, 16 (2014) 150-160.

[72] H.M. Shen, N. Mizushima, At the end of the autophagic road: an emerging understanding of lysosomal functions in autophagy, Trends Biochem Sci, 39 (2014) 61-71.

[73] A. Roczeniak-Ferguson, C.S. Petit, F. Froehlich, S. Qian, J. Ky, B. Angarola, T.C. Walther, S.M. Ferguson, The transcription factor TFEB links mTORC1 signaling to transcriptional control of lysosome homeostasis, Sci Signal, 5 (2012) ra42.

[74] J.A. Martina, Y. Chen, M. Gucek, R. Puertollano, MTORC1 functions as a transcriptional regulator of autophagy by preventing nuclear transport of TFEB, Autophagy, 8 (2012) 903-914.

[75] C. Settembre, R. Zoncu, D.L. Medina, F. Vetrini, S. Erdin, T. Huynh, M. Ferron, G. Karsenty, M.C. Vellard, V. Facchinetti, D.M. Sabatini, A. Ballabio, A lysosome-to-nucleus signalling mechanism senses and regulates the lysosome via mTOR and TFEB, EMBO J, 31 (2012) 1095-1108.

[76] C. Settembre, C. Di Malta, V.A. Polito, M. Garcia Arencibia, F. Vetrini, S. Erdin, S.U. Erdin, T. Huynh, D. Medina, P. Colella, M. Sardiello, D.C. Rubinsztein, A. Ballabio, TFEB links autophagy to lysosomal biogenesis, Science, 332 (2011) 1429-1433.

[77] M. Ferron, C. Settembre, J. Shimazu, J. Lacombe, S. Kato, D.J. Rawlings, A. Ballabio, G. Karsenty, A RANKL-PKCb-TFEB signaling cascade is necessary for lysosomal biogenesis in osteoclasts, Genes Dev, 27 (2013) 955-969.

[78] D.L. Medina, S. Di Paola, I. Peluso, A. Armani, D. De Stefani, R. Venditti, S. Montefusco, A. Scotto-Rosato, C. Prezioso, A. Forrester, C. Settembre, W. Wang, Q. Gao, H. Xu, M. Sandri, R. Rizzuto, M.A. De Matteis, A. Ballabio, Lysosomal calcium signalling regulates autophagy through calcineurin and TFEB, Nat Cell Biol, 17 (2015) 288-299.

[79] S. Chauhan, J.G. Goodwin, G. Manyam, J. Wang, A.M. Kamat, D.D. Boyd, ZKSCAN3 is a master transcriptional repressor of autophagy, Mol Cell, 50 (2013) 16-28.

[80] C. Settembre, A. Fraldi, D.L. Medina, A. Ballabio, Signals from the lysosome: a control centre for cellular clearance and energy metabolism, Nat Rev Mol Cell Biol, 14 (2013) 283-296.

[81] M. McBrayer, R.A. Nixon, Lysosome and calcium dysregulation in Alzheimer's disease: partners in crime, Biochem Soc Trans, 41 (2013) 1495-1502.

[82] B. Dehay, M. Martinez-Vicente, G.A. Caldwell, K.A. Caldwell, Z. Yue, M.R. Cookson, C. Klein, M. Vila, E. Bezard, Lysosomal impairment in Parkinson's disease, Mov Disord, 28 (2013) 725-732.

[83] B. Ravikumar, C. Vacher, Z. Berger, J.E. Davies, S. Luo, L.G. Oroz, F. Scaravilli, D.F. Easton, R. Duden, C.J. O'Kane, D.C. Rubinsztein, Inhibition of mTOR induces autophagy and reduces toxicity of polyglutamine expansions in fly and mouse models of Huntington disease, Nat Genet, 36 (2004) 585-595.

[84] P. Boya, G. Kroemer, Lysosomal membrane permeabilization in cell death, Oncogene, 27 (2008) 6434-6451.

[85] Y.T. Cheung, N.Q. Zhang, C.H. Hung, C.S. Lai, M.S. Yu, K.F. So, R.C. Chang, Temporal relationship of autophagy and apoptosis in neurons challenged by low molecular weight beta-amyloid peptide, J Cell Mol Med, 15 (2011) 244-257.

[86] I.F. Wang, B.S. Guo, Y.C. Liu, C.C. Wu, C.H. Yang, K.J. Tsai, C.K. Shen, Autophagy activators rescue and alleviate pathogenesis of a mouse model with proteinopathies of the TAR DNA-binding protein 43, Proc Natl Acad Sci USA, 109 (2012) 15024-15029.

[87] I.F. Wang, K.J. Tsai, C.K. Shen, Autophagy activation ameliorates neuronal pathogenesis of FTLD-U mice: a new light for treatment of TARDBP/TDP-43 proteinopathies, Autophagy, 9 (2013) 239-240.

[88] C.W. Cheng, M.J. Lin, C.J. Shen, Rapamycin Alleviates Pathogenesis of a New Drosophila Model of ALS-TDP, J Neurogenet, (2015) 1-47.

[89] H. Deng, K. Gao, J. Jankovic, The role of FUS gene variants in neurodegenerative diseases, Nat Rev Neurol, 10 (2014) 337-348.

[90] Y.R. Li, O.D. King, J. Shorter, A.D. Gitler, Stress granules as crucibles of ALS pathogenesis, J Cell Biol, 201 (2013) 361-372.

[91] H.H. Ryu, M.H. Jun, K.J. Min, D.J. Jang, Y.S. Lee, H.K. Kim, J.A. Lee, Autophagy regulates amyotrophic lateral sclerosis-linked fused in sarcoma-positive stress granules in neurons, Neurobiol Aging, 35 (2014) 2822-2831.

[92] X. Zhang, L. Li, S. Chen, D. Yang, Y. Wang, Z. Wang, W. Le, Rapamycin treatment augments motor neuron degeneration in SOD1(G93A) mouse model of amyotrophic lateral sclerosis, Autophagy, 7 (2011) 412-425.

[93] S. Saxena, F. Roselli, K. Singh, K. Leptien, J.P. Julien, F. Gros-Louis, P. Caroni, Neuroprotection through excitability and mTOR required in ALS motoneurons to delay disease and extend survival, Neuron, 80 (2013) 80-96.

[94] A. Bhattacharya, A. Bokov, F.L. Muller, A.L. Jernigan, K. Maslin, V. Diaz, A. Richardson, H. Van Remmen, Dietary restriction but not rapamycin extends disease onset and survival of the H46R/H48Q mouse model of ALS, Neurobiol Aging, 33 (2012) 1829-1832.

[95] J.K. Ching, C.C. Weihl, Rapamycin-induced autophagy aggravates pathology and weakness in a mouse model of VCP-associated myopathy, Autophagy, 9 (2013) 799-800.

[96] K.A. Staats, S. Hernandez, S. Schonefeldt, A. Bento-Abreu, J. Dooley, P. Van Damme, A. Liston, W. Robberecht, L. Van Den Bosch, Rapamycin increases survival in ALS mice lacking mature lymphocytes, Mol Neurodegener, 8 (2013) 31.

[97] S. Sarkar, Chemical screening platforms for autophagy drug discovery to identify therapeutic candidates for Huntington's disease and other neurodegenerative disorders, Drug Discov Today Technol, 10 (2013) e137-144.

[98] J.H. Shin, S.I. Cho, H.R. Lim, J.K. Lee, Y.A. Lee, J.S. Noh, I.S. Joo, K.W. Kim, B.J. Gwag, Concurrent administration of Neu2000 and lithium produces marked improvement of motor neuron survival, motor function, and mortality in a mouse model of amyotrophic lateral sclerosis, Mol Pharmacol, 71 (2007) 965-975.

[99] F. Fornai, P. Longone, L. Cafaro, O. Kastsiuchenka, M. Ferrucci, M.L. Manca, G. Lazzeri, A. Spalloni, N. Bellio, P. Lenzi, N. Modugno, G. Siciliano, C. Isidoro, L. Murri, S. Ruggieri, A. Paparelli, Lithium delays progression of amyotrophic lateral sclerosis, Proc Natl Acad Sci USA, 105 (2008) 2052-2057.

[100] A. Gill, J. Kidd, F. Vieira, K. Thompson, S. Perrin, No benefit from chronic lithium dosing in a sibling-matched, gender balanced, investigator-blinded trial using a standard mouse model of familial ALS, PLoS One, 4 (2009) e6489.

[101] A. Pizzasegola, I. Caron, C. Daleno, A. Ronchi, C. Minoia, M.T. Carri, C. Bendotti, Treatment with lithium carbonate does not improve disease progression in two different strains of SOD1 mutant mice, Amyotroph Lateral Scler, 10 (2009) 221-228.

[102] K.E. Morrison, S. Dhariwal, R. Hornabrook, L. Savage, D.J. Burn, T.K. Khoo, J. Kelly, C.L. Murphy, A. Al-Chalabi, A. Dougherty, P.N. Leigh, L. Wijesekera, M. Thornhill, C.M. Ellis, K. O'Hanlon, J. Panicker, L. Pate, P. Ray, L. Wyatt, C.A. Young, L. Copeland, J. Ealing, H. Hamdalla, I. Leroi, C. Murphy, F. O'Keeffe, E. Oughton, L. Partington, P. Paterson, D. Rog, A. Sathish, D. Sexton, J. Smith, H. Vanek, S. Dodds, T.L. Williams, I.N. Steen, J. Clarke, C. Eziefula, R. Howard, R.

Orrell, K. Sidle, R. Sylvester, W. Barrett, C. Merritt, K. Talbot, M.R. Turner, C. Whatley, C. Williams, J. Williams, C. Cosby, C.O. Hanemann, I. Iman, C. Philips, L. Timings, S.E. Crawford, C. Hewamadduma, R. Hibberd, H. Hollinger, C. McDermott, G. Mils, M. Rafiq, P.J. Shaw, A. Taylor, E. Waines, T. Walsh, R. Addison-Jones, J. Birt, M. Hare, T. Majid, Lithium in patients with amyotrophic lateral sclerosis (LiCALS): a phase 3 multicentre, randomised, double-blind, placebo-controlled trial, Lancet Neurol, 12 (2013) 339-345.

[103] A. Chio, G. Borghero, A. Calvo, M. Capasso, C. Caponnetto, M. Corbo, F. Giannini, G. Logroscino, J. Mandrioli, N. Marcello, L. Mazzini, C. Moglia, M.R. Monsurro, G. Mora, F. Patti, M. Perini, V. Pietrini, F. Pisano, E. Pupillo, M. Sabatelli, F. Salvi, V. Silani, I.L. Simone, G. Soraru, M.R. Tola, P. Volanti, E. Beghi, Lithium carbonate in amyotrophic lateral sclerosis: lack of efficacy in a dose-finding trial, Neurology, 75 (2010) 619-625.

[104] M. Tanaka, Y. Machida, S. Niu, T. Ikeda, N.R. Jana, H. Doi, M. Kurosawa, M. Nekooki, N. Nukina, Trehalose alleviates polyglutamine-mediated pathology in a mouse model of Huntington disease, Nat Med, 10 (2004) 148-154.

[105] S. Sarkar, J.E. Davies, Z. Huang, A. Tunacliffe, D.C. Rubinsztein, Trehalose, a novel mTOR-independent autophagy enhancer, accelerates the clearance of mutant huntingtin and alpha-synuclein, J Biol Chem, 282 (2007) 5641-5652.

[106] J.A. Rodriguez-Navarro, L. Rodriguez, M.J. Casarejos, R.M. Solano, A. Gomez, J. Peruch, A.M. Cuervo, J. Garcia de Yebenes, M.A. Mena, Trehalose ameliorates dopaminergic and tau pathology in parkin deleted/tau overexpressing mice through autophagy activation, Neurobiol Dis, 39 (2010) 423-438.

[107] K. Castillo, M. Nassif, V. Valenzuela, F. Rojas, S. Matus, G. Mercado, F.A. Court, B. van Zundert, C. Hetz, Trehalose delays the progression of amyotrophic lateral sclerosis by enhancing autophagy in motoneurons, Autophagy, 9 (2013) 1308-1320.

[108] X. Zhang, S. Chen, L. Song, Y. Tang, Y. Shen, L. Jia, W. Le, MTOR-independent, autophagic enhancer trehalose prolongs motor neuron survival and ameliorates the autophagic flux defect in a mouse model of amyotrophic lateral sclerosis, Autophagy, 10 (2014) 588-602.

[109] Y. Li, Y. Guo, X. Wang, X. Yu, W. Duan, K. Hong, J. Wang, H. Han, C. Li, Trehalose decreases mutant SOD1 expression and alleviates motor deficiency in early but not end-stage amyotrophic lateral sclerosis in a SOD1-G93A mouse model, Neuroscience, (2015).

[110] V. Crippa, D. Sau, P. Rusmini, A. Boncoraglio, E. Onesto, E. Bolzoni, M. Galbiati, E. Fontana, M. Marino, S. Carra, C. Bendotti, S. De Biasi, A. Poletti, The small heat shock protein B8 (HspB8) promotes autophagic removal of misfolded proteins involved in amyotrophic lateral sclerosis (ALS), Hum Mol Genet, 19 (2010) 3440-3456.

[111] P. Rusmini, V. Crippa, E. Giorgetti, A. Boncoraglio, R. Cristofani, S. Carra, A. Poletti, Clearance of the mutant androgen receptor in motoneuronal models of spinal and bulbar muscular atrophy, Neurobiol Aging, 34 (2013) 2585-2603.

[112] D.C. Rubinsztein, P. Codogno, B. Levine, Autophagy modulation as a potential

therapeutic target for diverse diseases, Nat Rev Drug Discov, 11 (2012) 709-730.

[113] R.L. Vidal, S. Matus, L. Bargsted, C. Hetz, Targeting autophagy in neurodegenerative diseases, Trends Pharmacol Sci, 35 (2014) 583-591.

| Autophagy step                | ALS-associated protein     | Autophagy defect                                                                                   | References |
|------------------------------|---------------------------|--------------------------------------------------------------------------------------------------|------------|
| Vesicle nucleation           | SOD1                      | Mutant SOD1 impairs vesicle nucleation by destabilizing the Beclin-1-Bcl-xL complex                     | [24]       |
| Substrate recognition        | p62(SQSTM1)               | Genetic ablation of p62 results in dysregulation of autophagy in zebrafish                             | [31]       |
|                              | OPTN                      | OPTN mutation impedes autophagy-mediated protein degradation as well as mitophagy-mediated clearance of damaged mitochondria | [34-36]    |
| Autophagosome trafficking    | Dynactin                  | Dynactin mutation impedes autophagosome transport                                                  | [48,49]    |
|                              | Dynein                    | Dynein mutation impairs axonal transport of the autophagosome                                        | [52,53]    |
|                              | SOD1                      | Mutant SOD1 perturbs retrograde axonal transport by interacting with the dynactin-dynein complex         | [56]       |
| Autophagosome-lysosome fusion | VCP(p97)                  | Mutation of VCP impairs autophagosome-lysosome fusion                                               | [58]       |
|                              | CHMP2B                    | CHMP2B mutation impedes autophagosome-lysosome fusion                                                | [59,63]    |

| Mode of action           | Autophagy inducer | Therapeutic effects                                                                 | References   |
|-------------------------|-------------------|----------------------------------------------------------------------------------|--------------|
| mTORC1 inhibition       | Rapamycin        | attenuates ALS-like features and TDP-43 aggregation in TDP-43 transgenic animals      | [86-88]      |
|                         |                   | reduces accumulation of FUS-positive stress granules and neurodegeneration induced by ALS-linked FUS mutation | [91]         |
|                         |                   | has no beneficial effect in mutant SOD1 transgenic mice                              | [92-94]      |
|                         |                   | has beneficial effects in mutant SOD1 transgenic mice lacking mature lymphocytes     | [96]         |
|                         |                   | augments disease propagation in mutant VCP transgenic mice                           | [95]         |
|                         | Tamoxifen        | attenuates ALS-like features and TDP-43 aggregation in TDP-43 transgenic mice          | [87]         |
| AMPK activation         | Lithium          | has beneficial effects in mutant SOD1 transgenic mice                                | [99]         |
|                         |                   | has no effect in mutant SOD1 transgenic mice                                        | [100, 101]   |
|                         | Trehalose        | ameliorates ALS-like features in mutant SOD1 transgenic mice                          | [107-110]    |
| Unknown                 | Spermidine       | attenuate ALS-like features and TDP-43 aggregation in TDP-43 transgenic mice          | [87]         |
|                         | Carbamazepine    |                                                                                   |              |

Figure Legends

Fig. 1. Three key stages in the process of autophagy.

A. The initiation stage of autophagy includes vesicle nucleation for a specific membrane structure known as the phagophore, elongation and closure of this structure to form a double membrane-bound vacuole, and formation of the autophagosome. Inhibition of mTOR or activation of AMPK triggers vesicle nucleation in a manner dependent on regulation by the ULK1 complex (ULK1, Atg13, FIP200) and the PI3K CIII complex (Vps34, Vps15, Atg14, Beclin-1), both of which come together at the phagophore assembly site. Elongation of the phagophore is regulated by two types of ubiquitination-like reaction. In the first reaction, Atg12 is covalently conjugated with Atg5 by the sequential actions of Atg7, Atg10, and Atg16. Docking of the conjugate on the phagophore promotes the second ubiquitination-like reaction, the covalent conjugation of phosphatidylethanolamine (PE) to LC3. This conjugation facilitates closure of the inner and outer bilayers of the phagophore. B. The maturation stage of autophagy involves the consecutive fusion of the autophagosome with endocytic compartments (late endosomes and amphisomes) and a lysosome to form the autolysosome. This stage begins with movement of the cargo-loaded autophagosome along a microtubule track toward the microtubule-organizing center in a manner dependent on the dynein-dynactin complex. C. The degradation stage includes the digestion of engulfed material in the autolysosome by acidic hydrolases originally present in the lysosome. Acidification of the lysosome is highly dependent on v-ATPase, a multimeric proton-pumping enzyme.

Fig. 2. Autophagy defects due to ALS-associated gene mutations.

A. Autophagy receptors such as p62 and OPTN recognize diverse ubiquitin-conjugated autophagic substrates. ALS-linked pathological mutations in p62 or OPTN disrupt such substrate recognition, resulting in a failure to load the ubiquitinated substrates into the autophagosome and consequent accumulation of protein aggregates and damaged mitochondria. B. Mutant SOD1 impedes the vesicle nucleation step of autophagy through abnormal interaction with Beclin-1 and consequent destabilization of the Beclin-1–Bcl-xL complex. C. Movement of the autophagosome along a microtubule track to a lysosome is

dependent on the dynactin-dynein complex. Mutations of dynactin or dynein impair this trafficking of the autophagosome. Furthermore, ALS-associated SOD1 mutants may also impair this process by targeting the dynactin-dynein complex. D. Fusion between the autophagosome and a lysosome may be disrupted by rare ALS-associated mutations in the genes for VCP and CHMP2B.* ALS-linked mutations have been found in the genes of the indicated proteins.

Initiation

A
- ULK1 complex
  - Atg13
  - FIP200 ULK1

B
- Atg12-Atg5 conjugation
  - Atg12 → Atg7 → Atg10 → Atg5
  - Atg5, Atg12, Atg16

C
- PI3K CIII complex
  - Beclin-1
  - Vps34
  - Vps15

D
- LC3 conjugation
  - LC3 → lipidation → LC3-PE
  - LC3

E
- mitochondria
- protein aggregates

F
- Phagophore
- Autophagosome

Maturation

G
- Fusion step
  - Late endosome → Amphisome → Lysosome

Degradation

H
- Vesicle acidification
  - v-ATPase
  - H+

I
- Lysosome

J
- Autolysosome

Fig. 2

A Substrate recognition
- protein aggregates
- mitochondria
- Ubiquitin
- : Autophagy receptors (p62*, OPTN*)

B Vesicle nucleation
- Atg13
- FIP200 ULK1
- Vps34
- Vps15
- Beclin-1
- Bcl-X
- SOD1*
- Phagophore

C Autophagosome trafficking
- LC3
- SOD1*
- dynein*
- dynactin*
- microtubule
- Autophagosome

D Autophagosome/lysosome fusion
- LC3
- Lysosome
- VCP* CHMP2B* Autolysosome

Highlights

- Impaired autophagy could contribute to ALS pathogenesis.
- ALS-linked gene mutations lead to autophagy failures.
- Autophagy is a potential therapeutic target for ALS and other neurodegenerative diseases.
