
Personal Account

THE CHEMICAL RECORD

Aminoglycoside Antibiotics: New Insights into the Biosynthetic Machinery of Old Drugs

Fumitaka Kudo$^{[a]}$ and Tadashi Eguchi$^{*[b]}$

Chem. Rec. 2016, 16, 4–18 © 2015 The Chemical Society of Japan & Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim Wiley Online Library 4

ABSTRACT: 2-Deoxystreptamine (2DOS) is the unique chemically stable aminocyclitol scaffold of clinically important aminoglycoside antibiotics such as neomycin, kanamycin, and gentamicin, which are produced by *Actinomycetes*. The 2DOS core can be decorated with various deoxyaminosugars to make structurally diverse pseudo-oligosaccharides. After the discovery of biosynthetic gene clusters for 2DOS-containing aminoglycoside antibiotics, the function of each biosynthetic enzyme has been extensively elucidated. The common biosynthetic intermediates 2DOS, paromamine and ribostamycin are constructed by conserved enzymes encoded in the gene clusters. The biosynthetic intermediates are then converted to characteristic architectures by unique enzymes encoded in each biosynthetic gene cluster. In this Personal Account, we summarize both common biosynthetic pathways and the pathways used for structural diversification.

Keywords: antibiotics, biosynthesis, enzymes, glycosides, neomycin

1. Introduction

Aminoglycoside antibiotics are a historically important class of antibacterial agents including streptomycin, which was the first effective antituberculosis agent, and the widely effective antibacterial agent kanamycin A (Figure 1). Streptomycin was the first aminoglycoside antibiotic, discovered from *Streptomyces griseus* by the Waksman lab in 1943.[1] In 1948, neomycin was isolated from *Streptomyces fradiae.*[2] After that, a variety of aminoglycoside antibiotics including kanamycin A (1957)[3] and the gentamicins (1963)[4] were discovered from the culture of microorganisms such as *Streptomyces* and *Micromonospora*, respectively. Most contain an aminocyclitol as an aglycone, connected with deoxyaminosugars to make a structurally diverse family of antibiotics (Figure 1). A majority of 2-deoxy-scyllo-inosamine (2DOIA)-derived aminocyclitol is 2-deoxystreptamine (2DOS) and, thus, the majority of aminoglycoside antibiotics contain 2DOS as an aglycone (Figure 1A). Another family of aminocyclitols is derived from *myo*-inositol 1-phosphate (MIP), which is biosynthesized from D-glucose 6-phosphate (G6P) by *myo*-inositol-1-phosphate synthase. MIP is converted to aminocyclitols by a combination of phosphatases, dehydrogenases, and aminotransferases (Figure 1B).[5]

The mode of action of aminoglycoside antibiotics has been extensively studied and is well understood.[6] The target biomolecules in bacteria are ribosomal RNAs to inhibit the translation process leading to the death of enemy bacteria.[7] Crystal structure analysis of rRNA complexes with aminoglycoside antibiotics has revealed the specific interaction of the drugs with the target rRNAs.[8,9] In general, aminoglycoside antibiotics have several amino groups and are positively charged to interact with negatively charged nucleotides such as RNAs. Thus, the introduction of amino groups into the biosynthetic intermediates is critical to gain the ability to interact with negatively charged molecules. In addition, the characteristic conformation of aminoglycosides consisting of several deoxyaminosugar units with a particular aminocyclitol would be feasible to interact with the relatively rigid conformation of RNA. The aminoglycoside producer strains selectively construct such molecules for specific purposes, presumably to combat enemy bacteria.

On the other hand, resistant strains have emerged as a result of heavy usage of aminoglycoside antibiotics.[6] Such resistant bacteria inactivate aminoglycosides by *N*-acetylation, *O*-phosphorylation, and *O*-adenylation, and significantly reduce the interaction with rRNAs.[10] In addition, methylation of own rRNA and changing the efflux system of drugs to reduce the amount of antibiotics are known to be common resistance mechanisms. Some aminoglycoside antibiotics have natural defensive structural features against such resistant bacteria, such as the deoxygenated moieties in gentamicin and tobramycin and the *N*-aminoacyl group in butirosin (Figure 1). These naturally occurring aminoglycoside modifications have thus been applied to create semisynthetic molecules such as amikacin, dibekacin and arbekacin, which can be effectively used against aminoglycoside-resistant bacteria.[11] Therefore, combinatorial biosynthesis of aminoglycoside antibiotics by pathway engineering could be an alternative and efficient

[a] F. Kudo  
Department of Chemistry  
Tokyo Institute of Technology  
2-12-1 Ookayama  
Meguro-ku Tokyo  
152-8551 (Japan)  
E-mail: fkudo@chem.titech.ac.jp  

[b] T. Eguchi  
Department of Chemistry and Materials Science  
Tokyo Institute of Technology  
2-12-1 Ookayama  
Meguro-ku Tokyo  
152-8551 (Japan)  
E-mail: eguchi@chem.titech.ac.jp

P e r s o n a l A c c o u n t

THE CHEMICAL RECORD

A) 2DOIA derived family

![Chemical Structure Image]

B) MIP derived family

Fig. 1. Structures of aminoglycoside antibiotics. (A) 2-Deoxy-scyllo-inosamine (2DOIA)-derived aminoglycosides, and (B) myo-inositol 1-phosphate (MIP)-derived aminoglycosides. The red color indicates the 2DOIA-derived aminocyclitol and most of these are 2-deoxystreptamine (2DOS). The blue color indicates MIP-derived aminocyclitol or cyclitol.

synthetic methodology toward the production of potentially useful aminoglycoside antibiotics.

The ability of aminoglycoside antibiotics to interact with negatively charged nucleotides has also been examined with other molecules beyond rRNAs to expand their usefulness in human healthcare. [12] The readthrough of nonsense mutations of DNA by several natural and unnatural aminoglycoside derivatives has been reported. [13] Antiplasmid activity of apramycin

Fumitaka Kudo received his B.Sc. in 1994, his M.Sc. in 1996, and his Ph.D. in 1999 from the Tokyo Institute of Technology under the direction of Katsumi Kakinuma. He was a postdoctoral fellow at Brown University working with David E. Cane from 1999 to 2001 and at Johns Hopkins University working with Craig A. Townsend from 2001 to 2003. In 2003, he joined the Department of Chemistry at the Tokyo Institute of Technology as an assistant professor and was promoted to associate professor in 2010. His research focuses on bioorganic chemistry in microbial natural product biosynthesis, including aminoglycoside and aminocyclitol antibiotics and polyketide antibiotics.

to remove a plasmid that codes resistance enzymes has also been reported. [14] HIV treatment with aminoglycosides to interact with the fatal nucleotide molecules has been investigated. [15,16] The conjugation of aminoglycosides with lipophilic alkyl chains to create amphiphilic molecules was successful for modification of their antimicrobial activity by adding antifungal activity. [17,18] Furthermore, such amphiphilic lipid–aminoglycoside conjugates were found to interact with siRNA and deliver these to the

Tadashi Eguchi received his B.Sc. from Yokohama City University in 1980, his M.Sc. in 1982, and his Ph.D. in 1990 from the Tokyo Institute of Technology. After working in the pharmaceutical industry, he joined Iwaki Meisei University as an assistant professor in 1987. He moved to the Tokyo Institute of Technology as an assistant professor in 1990 and was appointed to associate professor in 1995 and full professor in 2005. His research group is currently pursuing several areas of research with a focus on both the elucidation of the biosynthetic pathways of biologically active natural products and the mechanisms of novel enzymatic reactions.

target gene as vector.[19] Conjugation of peptide nucleic acids (PNAs) to aminoglycosides is also recognized to have potential for generating delivery molecules to target characteristic RNA molecules.[20] Aminoglycosides have considerable ability to interact with DNA and RNA molecules as characteristic scaffolds and their combination with other types of functional groups has greatly expanded their utility.[21]

Major side effects of aminoglycosides are nephrotoxicity and ototoxicity.[22,23] Since aminoglycosides with more amino groups tend to show higher toxicity, the positively charged chemical properties of aminoglycosides are also involved in the toxicity mechanisms.[12,24,25] A strong binding of aminoglycosides with phospholipids in kidney and cochlea is proposed as one of the mechanisms. A significant accumulation of aminoglycosides in the kidneys causes efficient absorption of drugs leading to nephrotoxicity. Regarding the ototoxicity, various toxic mechanisms including chelation of metal ions to generate reactive oxygen species and inhibition of mitochondrial protein synthesis have been reported.[22,26] It is remarkable that apramycin shows less ototoxicity with potent antibacterial activity.[27] This fact indicates that the architectures of aminoglycosides rather than cationic properties appear to be one of the significant factors for ototoxicity. Thus, modification of the structures of aminoglycosides can be developed to create less toxic molecules that retain potent antibacterial activity.

Functional analysis of the biosynthetic genes and enzymes for aminoglycoside antibiotics has been extensively performed in many laboratories for biochemical interest as well as for biotechnological application toward making novel aminoglycoside structures. Supplementation of unnatural biosynthetic intermediates into blocked mutants of aminoglycoside producer strains resulted in the first mutasynthesis to produce unnatural aminoglycoside antibiotics,[30] leading to current synthetic biology for engineering biosynthetic pathways to produce unnatural natural products.[31] Feeding experiments with radioisotope- and stable-isotope-labeled glucoses and glucosamines indicated that most of the components of aminoglycosides are derived from sugars, as predicted from their chemical structures.[32] However, the timing of the modifications of the sugar and aminocyclitol moieties was not understood until recently.

The discovery of 2-deoxy-scyllo-inosose (2DOI) synthase from the butirosin producer strain *Bacillus circulans*[33] accelerated the identification of biosynthetic gene clusters for a variety of deoxycyclitol-containing aminoglycoside antibiotics.[34–38] To date, more than ten gene clusters have been identified and deposited in the public gene data bank. Comparison of the gene clusters suggested that conserved genes are responsible for the biosynthesis of the common biosynthetic intermediates.[36] Additionally, unique enzymes encoded in each biosynthetic gene cluster appear to be involved in the modifications of the common biosynthetic intermediates. The biosynthetic enzymes for 2DOS, paromamine, neamine,

ribostamycin, and neomycin have been established. Herein, we summarize the common biosynthetic pathways and then describe how the common intermediates are converted to the characteristic structures of aminoglycoside antibiotics.

## 2. Biosynthetic Pathways of Common Biosynthetic Intermediates

Compared with polymerized sugars, 2DOS, also known as 1,3-diamino-4,5,6-trihydroxycyclohexane, is a chemically stable scaffold because of its lack of an acetal moiety. This stability could be one of the reasons that a majority of aminoglycoside antibiotics have 2DOS as the central aglycone. An incorporation study suggested that the six-membered carbacyclitol is constructed from D-glucose or G6P.[39] However, the amino acid sequence of the enzyme was not discovered until 1999.[33]

### 2.1. 2-Deoxystreptamine

An incorporation study of deuterium-labeled glucose in the ribostamycin producer *Streptomyces ribosidificus* suggested that a 3-dehydroquinate (DHQ) synthase-like reaction mechanism from the shikimate pathway could be operational in the construction of the cyclitol, because two methylene hydrogen atoms at the 6-position of glucose remain in the 2-deoxy moiety of 2DOS.[39,40] In fact, 2DOI synthase (Scheme 1A) activity was identified from the neomycin producer *S. fradiae*, although the enzyme was not isolated from the cultured cells.[41–43] The same enzymatic activity was also identified from the butirosin producer *B. circulans*;[44] the enzyme was successfully purified,[45] and the corresponding gene *btrC* was cloned.[33] The amino acid sequence of the enzymes showing moderate similarity (26–34%) to DHQ synthase suggested that 2DOI synthase catalyzes a similar reaction mechanism to DHQ synthase. The crystal structure of BtrC revealed that the active site of BtrC is similar to that of DHQ synthase.[46] However, two conserved amino acid residues at positions 243 (Glu in DOIS and Asn in DHQS) and 239 (Gly in DOIS and Arg in DHQS) in BtrC are significantly different between these two enzyme families (Scheme 1). Additionally, the location of the conserved glutamate residue at position 235 in BtrC is different. Therefore, we proposed that Glu243 in BtrC abstracts the proton at C-5 of G6P to prompt the dephosphorylation to give an α,β-unsaturated ketone intermediate (Scheme 1A, step 2).[47] After reduction of the carbonyl group at C-4 (step 3) and ring opening of the hemiacetal moiety (step 4), the aldol-type condensation occurs to give 2DOI (step 5).[48] The stereoselective C–C bond formation is controlled by Glu235 as the acid/base catalyst acting on the anomeric hydroxy group (steps 4 and 5).[46] In the DHQS reaction, the substrate phosphate group abstracts a proton to give an α,β-unsaturated ketone intermediate (Scheme 1B, step 2) and Asn268 is not directly involved in this event.

A) 2-deoxy-scyllo-inosose (2DOI) synthase

B) 3-dehydroquinate (DHQ) synthase

Scheme 1. 2-Deoxy-scyllo-inosose (2DOI) synthase and 3-dehydroquinate (DHQ) synthase. The amino acid position numbers derived from BtrC for 2DOI synthase and from *Aspergillus nidulans* DHQ synthase are indicated.

Arg264 of DHQS plays a crucial role in recognizing the carboxylate of DAHP and stereoselective aldol-type condensation (steps 4 and 5). Thus, only a few amino acid residue differences between DOIS and DHQS appear to determine the substrate and reaction specificities.

In the next step, 2DOI is converted to 2DOIA by an L-glutamine:2DOI aminotransferase (2DOI-amt), namely, BtrS in butirosin biosynthesis and NeoS in neomycin biosynthesis (Scheme 2).[50,51] To prevent confusion, the Piepersberg group labeled the biosynthetic enzymes using the same alphabetical codes as the conserved genes based on the butirosin biosynthetic genes.[37] We have adopted their nomenclature, as shown in Table 1.[36] Thus, the “S” series of enzymes is responsible for the transamination in the 2DOIA formation. The C-3 position of 2DOIA is then oxidized by the NADP⁺-dependent dehydrogenase NeoE in neomycin biosynthesis,[52] while a quite unique radical S-adenosylmethionine (SAM) dehydrogenase BtrN is responsible for the dehydrogenation to give 3-amino-2-deoxy-scyllo-inosose (amino DOI) in the butirosin biosynthesis.[53] This is the first example of the anaerobic

Scheme 2. Biosynthetic pathway for neomycin B. The enzymatic functions of the Neo enzymes are listed in Table 1.*The homologous BtrL and BtrP from butirosin biosynthesis have been functionally characterized, but not NeoL and NeoP.

Table 1. Biosynthetic enzymes for neomycin B (Neo).

| Neomycin biosynthetic enzyme<sup>[a]</sup> | Size of protein (AA) | Enzymatic functions |
|------------------------------------------|----------------------|---------------------|
| NeoC<sup>[b]</sup>                       | 430                  | 2-deoxy-scyllo-inosose (2DOI) synthase |
| NeoS<sup>[b][c]</sup>                    | 424                  | Gln:2DOI aminotransferase<br>Gln:amino DOI aminotransferase |
| NeoE<sup>[d]</sup>                       | 340                  | 2-deoxy-scyllo-inosamine dehydrogenase<sup>[e]</sup> |
| NeoM<sup>[b]</sup>                       | 421                  | UDP-GlcNAc:2DOS GlcNAc transferase |
| NeoD<sup>[b][f]</sup>                    | 279                  | 2′-N-acetylparomamine deacetylase<br>2‴-N-acetylneomycin Y₂ deacetylase |
| NeoQ<sup>[b][f]</sup>                    | 541                  | Paromamine 6′-dehydrogenase<br>Neomycin Y₂ 6‴-dehydrogenase |
| NeoB<sup>[b][f]</sup>                    | 416                  | Gln:6′-oxoparomamine aminotransferase<br>Gln:6‴-oxoneomycin Y₂ aminotransferase |
| NeoL<sup>[b]</sup>                       | 660                  | PRPP:neamine<br>5-phosphoribosyltransferase |
| NeoP<sup>[b]</sup>                       | 233                  | 5″-phosphoribostamycin phosphatase |
| NeoF<sup>[f]</sup>                       | 366                  | UDP-GlcNAc:ribostamycin GlcNAc transferase |
| NeoN<sup>[f]</sup>                       | 299                  | Neomycin C 5‴-epimerase<sup>[g]</sup> |
| NeoA<sup>[b][i]</sup>                    | 1321                 | Unknown |
| NeoV<sup>[b][i]</sup>                    | 83                   | Unknown |

<sup>[a]</sup> The same alphabetical enzymes are shown as the conserved genes based on the butirosin biosynthetic enzymes in *Bacillus circulans*.  
<sup>[b]</sup> These proteins are conserved in neomycin and butirosin biosynthetic gene clusters.  
<sup>[c]</sup> This enzyme catalyzes two reactions in 2DOS biosynthesis.  
<sup>[d]</sup> In butirosin biosynthesis, radical SAM dehydrogenase BtrN is utilized instead of BtrE.  
<sup>[e]</sup> NADP⁺-dependent enzyme.  
<sup>[f]</sup> These enzymes catalyze two reactions in neomycin biosynthesis.  
<sup>[g]</sup> These proteins are conserved only in neomycin-type pseudotetrasaccharide biosynthetic gene clusters.  
<sup>[h]</sup> Radical SAM enzyme.  
<sup>[i]</sup> These two proteins are hypothesized to be involved in the ribosylation event.

clusters. In fact, the neomycin biosynthetic gene cluster was identified from the producer strain *S. fradiae* by using 2DOI-amt as a probe. The Spencer group,<sup>[56]</sup> the Piepersberg group,<sup>[37]</sup> and our group<sup>[52]</sup> deposited the same neomycin biosynthetic gene cluster during a similar period in 2005. Comparison of the neomycin and butirosin biosynthetic gene clusters indicated that the conserved genes are presumably involved in the formation of a common biosynthetic intermediate, pseudo-trisaccharide ribostamycin.<sup>[36]</sup> Thus, NeoA, B, C, D, E, L, M, P, Q, S, and V in the *neo* gene cluster are supposed to be involved in the formation of ribostamycin from G6P.

### 2.2. Paromamine

A unique aminocyclitol 2DOS is glycosylated by the glycosyltransferase NeoM, an “M” series enzyme, which uses UDP-N-acetylglucosamine (UDP-GlcNAc) as the glycosyl donor to yield 2′-N-acetylparomamine in neomycin biosynthesis (Scheme 2).<sup>[57]</sup> The acetyl group at C-2′ of 2′-N-acetylparomamine is removed by the deacetylase NeoD, a “D” series enzyme, to give paromamine.<sup>[57]</sup> The function of the deacetylase was established for the first time by the Spencer group with BtrD in butirosin biosynthesis.<sup>[58]</sup> Paromamine seems to be a key branching biosynthetic intermediate for this class of antibiotics. In neomycin/butirosin biosynthesis, the 5-hydroxy group of the 2DOS moiety is ribosylated, while the 6-hydroxy group of the 2DOS moiety of paromamine is glycosylated in kanamycin biosynthesis. In fact, five types of genes, C, S, E, M, and D, are highly conserved in the biosynthetic gene clusters, which are responsible for the biosynthesis of paromamine from G6P and UDP-GlcNAc.

### 2.3. Neamine

The 6′-hydroxy group of paromamine is converted to an amino group by the FAD-dependent dehydrogenase NeoQ, a “Q” series enzyme, and the aminotransferase NeoB, a “B” series enzyme, to give neamine in neomycin biosynthesis (Scheme 2).<sup>[59]</sup> Thus, three post-glycosylation enzymes, D, Q, and B, are responsible for elaborating the unique 2,6-diamino-2,6-dideoxysugar (neosamine C) moiety of this class of amino glycoside antibiotics. It should be noted that this biosynthetic order is different from the biosynthesis of the unusual NDP-deoxyaminosugars,<sup>[60]</sup> which are constructed prior to the glycosylation events in glycosylated polyketide and glycopeptide biosynthesis. Consequently, it was found that the biosynthetic pathway for the pseudodisaccharide neamine is linear, rather than a strategically efficient convergent biosynthetic pathway.

### 2.4. Ribostamycin

Neamine is then ribosylated by an “L” series phosphoribosyltransferase enzyme to give 5‴-phosphoribostamycin, which is subsequently hydrolyzed to ribostamycin by the “P” series

omycin.[57] NeoF uses UDP-GlcNAc to attach GlcNAc at the 3′′-hydroxy group of ribostamycin to give 2′′-N-acetylneomycin Y₂ (Scheme 2). The acetyl group of 2′′-N-acetylneomycin Y₂ is removed by the deacetylase NeoD to give neomycin Y₂.[57] Thus, NeoD functions as a deacetylase in neomycin and paromamine biosynthesis. The 6′′-hydroxy group of neomycin Y₂ is converted to an amino group by the dehydrogenase NeoQ and the aminotransferase NeoB.[56,62] Thus, the combination of the deacetylase NeoD, the dehydrogenase NeoQ, and the aminotransferase NeoB is involved twice in the elaboration of the two 2,6-diamino-2,6-dideoxysugar moieties in neomycin biosynthesis.

Paromomycin produced by *S. rimosus* subsp. *paromomycinus* and lividomycin produced by *S. lividus* have only one neosamine C moiety at the 3′′-hydroxy group of the pseudotrisaccharide, 6′-deamino-6′-hydroxyribostamycin (Figure 1). Actually, it was proven that the “Q” and “B” type enzymes work only once to modify the bottom part of the GlcN moiety in lividomycin biosynthesis (Scheme 3).[63] Therefore, presumably the dehydrogenases LivQ and ParQ selectively recognize the 6′-deamino-6′-hydroxyneomycin Y₂ and catalyze the dehydrogenation, while these enzymes do not oxidize paromamine. NeoQ may only recognize the glucosamine moiety of both paromamine and neomycin Y₂ in neomycin biosynthesis. Thus, the substrate specificity of “Q” type enzymes plays an important role in determining the final structure of aminoglycoside antibiotics. The substrate specificity of the “Q” and “B” enzymes is crucial to control the antibacterial activity and toxicity, since the number of amino groups controls the affinity against the negatively charged target biomolecules.

### Scheme 3. Biosynthetic pathway for paromomycin and lividomycin B. Only the C-6′′ position is converted to the amino group and the C-6′ position is unchanged.

phosphatase enzyme (Scheme 2).[61] We could detect the enzymatic activity of BtrL and BtrP in the butirosin biosynthetic pathway. The chemical structure of 5′′′-phosphoribostamycin was not confirmed by NMR, but LC-ESI-MS analysis indicated that 5′′′-phosphoribostamycin was generated by BtrL from 5-phosphoribose-1-diphosphate (PRPP) and neamine. Addition of BtrP into the BtrL reaction afforded ribostamycin. Thus, in this glycosylation event, the primary metabolite, PRPP, is directly utilized as the sugar donor before the modification of the sugar moiety.

Among the conserved biosynthetic genes, the “A” and “V” series of enzymes, which were speculated to be involved in the ribosylation, have not yet been characterized. The functions of these two genes remain unknown.

#### 2.5. Neomycin C

The last glycosylation in neomycin biosynthesis is catalyzed by the unique glycosyltransferase NeoF, which is conserved in the biosynthetic gene clusters of the neomycin-class pseudotetrasaccharide aminoglycoside antibiotics paromomycin and lividomycin.

#### 2.6. Neomycin B

The C-5′′′ position of the neosamine C moiety in the neomycin-type pseudotetrasaccharides including paromomycin and lividomycin is usually epimerized to the L form of the sugar, neosamine B. The enzyme responsible for catalyzing this reaction was recently identified from the study of the biosynthesis of neomycin B.[64] A unique radical SAM enzyme NeoN was found to catalyze the epimerization at the C-5′′′ position of neomycin C to give neomycin B. The entire biosynthetic pathway for neomycin B from G6P with UDP-GlcNAc and PRPP was thus elucidated.

Putative transporter and resistance genes for neomycin B biosynthesis are also encoded in the *neo* gene cluster. Consequently, the majority of all biosynthetic genes in the *neo* cluster are utilized for the biosynthesis of neomycin B. It is generally true that most of the proteins encoded in biosynthetic gene clusters are somehow involved in the construction of dead-end natural products. Therefore, to understand the biosynthetic pathways active in the cells of microorganisms, characterization of all biosynthetic genes in the cluster is necessary.

Scheme 4. Biosynthetic pathway for butirosin A. The enzymatic function of BtrE is different from the other type of “E” dehydrogenase. BtrF is sequentially different from the other type of “F” enzyme in neomycin biosynthesis (see Table 1). BtrG, H, I, J, K, O, and U are unique enzymes for butirosin biosynthesis.

## 3. Modification of Common Biosynthetic Intermediates toward Structural Diversification

The elucidation of the neomycin biosynthetic pathway revealed the presence of linear biosynthetic machinery responsible for constructing a dead-end natural product in the biosynthesis of aminoglycoside antibiotics (Scheme 2). To expand the structural diversity of aminoglycoside antibiotics, modifications of the common biosynthetic intermediates in the neomycin B biosynthetic pathway must be used. Presumably, pathway-specific enzymes that are characteristically encoded in each biosynthetic gene cluster should be involved. In fact, some unique enzymes have recently been identified as being responsible for catalyzing some unique modification reactions.

### 3.1. Toward Butirosin A

The most striking structural feature of butirosin is the (2S)-4-amino-2-hydroxybutyrate (AHBA) moiety at the C-1 position of 2DOS. Indeed, unique biosynthetic enzymes are encoded in the butirosin biosynthetic gene cluster. The Spencer group has elegantly characterized the functions of the enzymes BtrG, BtrH, BtrI, BtrJ, BtrK, BtrO, and BtrU (Scheme 4). This pathway was already described in three review articles.

Another structural feature of butirosin A is the stereochemistry at C-3′′ of the pentofuranose moiety. An epimerization reaction at C-3 of ribose in ribostamycin or butirosin B should be involved to give xylostasin or butirosin A, respectively (Scheme 4). The candidate enzymes are two types of putative dehydrogenases, BtrE and BtrF, which are not involved in ribostamycin biosynthesis or installation of the

AHBA moiety. BtrE is a homologue of the “E” series of enzymes that are responsible for the dehydrogenation of 2DOIA to give amino DOI. However, since BtrE lacks the catalytically important zinc-binding motif necessary for this dehydrogenase activity, it is likely involved in a different biosynthetic step. In the presence of both BtrE and BtrF as well as NAD⁺ and NADPH, ribostamycin was converted to xylostasin. BtrE oxidizes the C-3′′ of ribostamycin in the presence of NAD⁺ to give 3′′-oxoribostamycin, which is then reduced by BtrF to xylostasin. Thus, these two enzymes are required for the epimerization. The whole biosynthetic pathway for butirosin A has been completely elucidated. Putative transporter genes for butirosin are also encoded in the *btr* gene cluster. Therefore, it has been verified that the majority of the biosynthetic genes in the cluster are utilized for the biosynthesis of butirosin A.

### 3.2. Toward Kanamycin A

Apart from the ribostamycin/neomycin family of aminoglycoside antibiotics, the kanamycin family has a 3-aminosugar moiety at the 6-hydroxy group of 2DOS to give a pseudotrisaccharide (Figure 1). Paromamine is believed to be the most likely branching biosynthetic intermediate, which is glycosylated by another “M” type of glycosyltransferase, M2 (Scheme 5). We elucidated that the glycosyltransferase KanM2 catalyzes the glycosylation of paromamine in the presence of UDP-glucose to give 3′′-deamino-3′′-hydroxykanamycin C. Although it is still unclear when the 3-amino group of the third sugar moiety is introduced, it is possible that it occurs before the glycosylation of paromamine. NDP-glucose is presumably converted to NDP-

Scheme 5. Biosynthetic pathways for kanamycin A, tobramycin, and apramycin. The protein sequences for Q, B, M2, D2, and S2 are similar to the analogous alphabetical Neo enzymes listed in Table 1 and these proteins are supposed to have similar enzymatic functions. KanJ and KanK are unique enzymes in kanamycin biosynthesis. AprD3 and AprD4 are unique enzymes in apramycin/tobramycin biosynthesis.

kanosamine (3-amino-3-deoxyglucose), which becomes the glycosyl donor to give kanamycin C from paromamine in the KanM2 reaction. In the rifamycin biosynthetic pathway, UDP-kanosamine is supposed to be the biosynthetic intermediate for the 5-amino-3-hydroxybenzoic acid moiety.[72] The presence of enzymatic activity responsible for the production of UDP-kanosamine in rifamycin biosynthesis was partially reported.[72] However, the in vitro analysis of the corresponding enzymes is still unclear. It is presumed that the combination of a “D2” dehydrogenase enzyme and an “S2” aminotransferase enzyme is responsible for the introduction of the 3-amino group in kanamycin biosynthesis. However, the expected enzymatic activities of KanD2 and KanS2 have never been observed (unpublished). Detailed enzymatic analysis of the corresponding enzymes is necessary to understand the installation mechanism of the 3″-amino group in the biosynthesis of kanamycins. The pseudodisaccharide neamine is also likely a glycosyl acceptor to give kanamycin B. Although KanQ and KanB are predicted to function in the installation of the amino group at the C-6′ position, the timing of the amination reaction in kanamycin biosynthesis is still unclear. From the biosynthetic viewpoint of neomycin B, these amination reactions could occur after the glycosylation. Although a parallel pathway for kanamycin A in the producer strain *S. kanamyceticus* is proposed,[73] our in vitro analysis of NeoM in neomycin biosynthesis suggests that UDP-Glc is not utilized in the glycosylation of 2DOS to give the pseudodisaccharide.[57] Unfortunately, in vitro analysis of KanM1 expressed in *E. coli* failed to confirm its substrate specificity.

Two unique enzymes, a putative α-ketoglutarate-dependent non-heme iron dioxygenase, KanJ, and a putative oxidoreductase, KanK, are encoded in the kanamycin biosynthetic gene cluster, suggesting these unique enzymatic functions in the biosynthesis. Kanamycin A lacks the 2′-amino group of the neosamine C moiety among related aminoglycoside antibiotics and thus a deamination reaction seems to be involved. In fact, KanJ and KanK have been shown to be responsible for the deamination reaction.[74] KanJ catalyzes the hydroxylation at C-2′ of kanamycin B to give a hemiaminal intermediate that releases ammonia to give 2′-oxokanamycin B. The existence of 2′-oxokanamycin B was confirmed after reduction with NaBD₄. The carbonyl group of 2′-oxokanamycin B is then reduced by an NADH-dependent reductase KanK to give kanamycin A. This is a unique completion step in kanamycin A biosynthesis.

### 3.3. Toward Gentamicins

Gentamicin, produced by *Micromonospora echinospora*, is one of the most modified 2DOS-containing aminoglycoside antibiotics and various biosynthetically related compounds have been isolated from the producer strains. The *gen* biosynthetic gene cluster is unique and GenC, GenS, GenE, GenM1, and GenD are encoded to be responsible for the biosynthesis of paromamine.[37,75] Additionally, GenM2, GenD2, and GenS2 are encoded for the installation of the 3-aminosugar, 3-aminoxylose in this case. GenQ and GenB1 (B2, B3, B4 as homologues) are also encoded for the amination at C-6′.[36,37] The Yoon group confirmed that GenC, GenS, GenE, GenM1, GenD and GenM2 are the minimal set required for gentamicin A₂ formation (Scheme 6).[76] In addition, the Sun and Leadlay groups jointly established that the installation of two amino groups in gentamicin biosynthesis occurs after two glycosylation steps.

Scheme 6. Biosynthetic pathway for gentamicins. The protein sequences for M2, D2, S2, Q, B1, and B2 are similar to the analogous alphabetical Neo enzymes listed in Table 1 and these proteins are supposed to have similar enzymatic functions. GenD1 and GenK are unique methylcobalamin-dependent radical SAM C-methyltransferases in gentamicin biosynthesis. GenN is a unique $N$-methyltransferase.

catalyzed by GenD2, GenS2, GenQ, and GenB1 (and B2, B3, B4). [77,78] Thus, post-glycosylation modification of sugar moieties appears to be a common feature in the biosynthesis of aminoglycoside antibiotics. The substrate specificity of the “Q” series of dehydrogenases to give ketone intermediates could be important to determine the final structure of aminoglycoside antibiotics. Recently, GenB2 was found to be involved in the epimerization at C-6′ of gentamicins. [79] It is interesting how GenB2 recognizes substrates and catalyzes the epimerization (Scheme 6).

Two C-methylation reactions in gentamicin biosynthesis have also been elucidated recently. Methylcobalamin-dependent radical SAM enzymes GenK [80] and GenD1 [77] are responsible for the C-methylation at C-6′ and C-4″ of pseudotrisaccharide intermediates, respectively. As typical radical SAM enzymes, GenK and GenD1 are hypothesized to abstract a hydrogen atom from the corresponding sites of their substrates to give a radical intermediate using a 5′-deoxyadenosyl radical derived from SAM. [80] The radical intermediate makes a C–C bond with the methyl group on the methylcobalamin to give the C-methylated products. It is still unclear how methylcobalamin and the oxidized [4Fe–4S] cluster are returned to their initial states.

should be noted that methylcobalamin-dependent radical SAM C-methyltransferases are also involved in the biosynthesis of various natural products such as thienamycin and fosfomycin, judging from the sequence analysis of the biosynthetic enzymes in the related secondary metabolite biosynthetic gene clusters. [81–83] Thus, the putative radical-mediated C-methylation mechanism appears to be a common strategy to modify secondary metabolites. GenD1 was reported to have strict substrate specificity, and the 3″-amino group of the substrate is to be methylated by a methyltransferase GenN. GenK is relatively more tolerant of substrates, catalyzing the C-methylation at C-6′ of a pseudotrisaccharide. [77]

Deoxygenations at both C-3′ and C-4′ are also characteristic transformations in gentamicin biosynthesis. Uncharacterized enzymes in the gen cluster remain to be solved.

### 3.4. Other Aminoglycoside Antibiotics

As shown in Figure 1, various 2DOS-containing aminoglycoside antibiotics have been isolated. As described for the neomycin biosynthetic pathway, it is evident that conserved

biosynthetic enzymes are responsible for the construction of the common biosynthetic intermediates, 2DOS, paromamine, and ribostamycin, in most cases. Clearly, unique enzymes in each biosynthetic gene cluster are generally responsible for the structural diversification of the respective aminoglycosides.

Tobramycin is 3′-deoxykanamycin B and, thus, deoxygenation of paromamine or kanamycin C is involved in the biosynthesis (Scheme 5). Two similar biosynthetic gene clusters for tobramycin have been identified from *Streptoalloteichus hindustanus* and *Streptomyces tenebrarius* so far.[37] Interestingly, both microorganisms produce apramycin in addition to tobramycin. Thus, the biosynthesis of both tobramycin and apramycin are presumed to share some biosynthetic genes. Comparison of the biosynthetic gene clusters involved in kanamycin, tobramycin/apramycin, neomycin, and lividomycin biosynthesis suggested that the putative radical SAM enzyme AprD4 and the oxidoreductase AprD3 located in the apramycin biosynthesis gene cluster are candidate enzymes for catalyzing the deoxygenation in tobramycin/apramycin biosynthesis.[36,37] The involvement of AprD4 in the deoxygenation was confirmed by gene inactivation.[84] Conversion of neamine to nebramine (3′-deoxyneamine) with the cell-free extract of *Streptomyces venezuelae* harboring the *aprD3* and *aprD4* genes has been reported, although the reaction conditions were not described.[73] Thus, the catalytic reaction mechanism for the deoxygenation is still unclear. A radical-mediated reaction mechanism involving radical SAM chemistry and similar to the DesII-catalyzed deoxygenation in TDP-desosamine biosynthesis is proposed.[85]

Apramycin is a highly modified aminoglycoside containing an octose unit and a 4-amino-4-deoxyglucose moiety, but only one hydroxy group at C-4 of 2DOS is glycosylated (Scheme 5). Paromamine is supposed to be a common biosynthetic intermediate, given that the genes C, E, S, M, and D are conserved in the apramycin biosynthetic gene cluster.[37] Many characteristic enzymes are encoded in the *apr* gene cluster and are likely involved in the construction of the unique structure of apramycin.

Hygromycin B, produced by *Streptomyces hygroscopicus* subsp. *hygroscopicus*, is also a unique aminoglycoside where only the 5-hydroxy group of 2DOS is glycosylated (Figure 1). The sugar moiety is further glycosylated with a unique aminoheptose. Except for the 2DOS biosynthetic enzymes C, S, and E, the other biosynthetic enzymes remain to be characterized.[36,37]

Istamycin, produced by *Streptomyces tejimariensis*, is a pseudodisaccharide aminoglycoside antibiotic containing the unique deoxyaminocyclitol, 2-deoxyfortamine (2DOF). The aminated position in the aminocyclitol is different from 2DOS and also two positions of stereochemistry are epimerized (Figure 1). Furthermore, a unique deoxyaminosugar is attached at the 4-hydroxy group of 2DOF. Since the C, S, E, M, and D genes are conserved in the istamycin biosynthetic

gene cluster, a biosynthetic pathway similar to that for paramamine except for the oxidation/transamination of 2DOIA catalyzed by “E” and “S” enzymes is presumed.[36,37] It is also interesting to identify the substrate specificity of the “M” series of enzymes involved in istamycin biosynthesis. Modification of the aminocyclitol and sugar moieties could be similar to the biosynthesis of fortimicin, whose aminocyclitol is derived from *myo*-inositol 1-phosphate. Extensive biosynthetic studies with various blocked mutants of the producer strain *Micromonospora olivasterospora* revealed a possible biosynthetic pathway for fortimicin,[86–89] although only a few enzymatic analyses have been reported.[90] Thus, to understand this unique biosynthetic pathway for istamycin and fortimicin, detailed enzymatic analysis is necessary.

In addition, some of the biosynthetic enzymes for MIP-derived aminocyclitol-containing aminoglycoside antibiotics have been characterized. Actually, extensive biosynthetic studies with cell-free systems of the producer strains and genetic analysis, especially for streptomycin and bluensomycin producers, were carried out to reveal the biosynthetic order for the unique streptidine moiety (Scheme 7).[37,91,92] However, only some of the biosynthetic genes for streptomycin have been characterized so far. StsC is an aminotransferase that catalyzes the amination of scyllo-inosose to afford scyllo-inosamine (Scheme 7).[93] StsC shows similarity to the “S” aminotransferases in 2DOS biosynthesis and in fact the functional analysis of BtrS was inspired by the enzymatic function of StsC. Introduction of a guanidino moiety on the amino group of scyllo-inosamine is catalyzed by StrB1, an amidinotransferase, after phosphorylation at the 4-hydroxy group of scyllo-inosamine.[94] The other biosynthetic order for streptidine-6-phosphate has been proposed based on cell-free enzymatic reactions and genetic analysis, and the corresponding biosynthetic enzymes remain to be determined. Regarding the TDP-dihydrotreptose biosynthesis, TDP-glucose synthase StrD, TDP-glucose-4,6-dehydratase StrE, and TDP-4-dehydrorhamnose-3,5-epimerase StrM have been functionally characterized.[91] Although TDP-L-rhamnose synthase homologue StrL was predicted to be responsible for the TDP-dihydrotreptose formation, StrL showed only the TDP-L-rhamnose synthase activity.[91,95] A CDP-glucose synthase StrQ has also been characterized and would be involved in the CDP-L-glucosamine biosynthesis.[96] Furthermore, an excreted phosphatase StrK is known to be involved in the completion of streptomycin biosynthesis by hydrolyzing the 6-phosphate group of streptomycin-6-phosphate.[91,97]

### 4. Summary and Outlook

Herein, we have summarized details of the common biosynthetic pathway involved in the production of 2DOS-containing aminoglycoside antibiotics with butirosin and neomycin

a) streptidine-6-phosphate biosynthesis

![Chemical Structure](https://i.imgur.com/1234567.png)

b) TDP-dihydrostreptose biosynthesis

![Chemical Structure](https://i.imgur.com/8901234.png)

c) CDP-L-glucosamine biosynthesis

![Chemical Structure](https://i.imgur.com/5678901.png)

d) glycosylations

![Chemical Structure](https://i.imgur.com/2345678.png)

Scheme 7. Biosynthetic pathway for streptomycin.

biosynthetic machineries. It was thus revealed that 2DOS, paromamine, neamine, ribostamycin, and neomycin are biosynthesized by the conserved biosynthetic enzymes contained in the biosynthetic gene clusters as shown in Table 1 and Scheme 2. In addition, the recently characterized post-glycosylation modification enzymes involved in the biosynthesis of butirosin A, kanamycin A, and gentamicins have been described.

So far, the crystal structures of BtrC, ${}^{[46]}$ BtrS, ${}^{[98]}$ and BtrN ${}^{[99]}$ have been reported among various 2DOS-containing aminoglycoside antibiotic biosynthetic enzymes and these give some clues to understanding the reaction mechanisms. For instance, the structure of BtrC with a G6P analogue, carbaglucose-6-phosphate, revealed that the 2-hydroxy group of G6P is mainly recognized by Lys186 and the 3-hydroxy group of G6P by Asp135. ${}^{[46]}$ This fact agrees well with the observation that 2-fluoro-G6P and 3-amino-G6P are converted to the corresponding 2DOI derivatives, but not 3-fluoro-G6P and 2-amino-G6P, because the fluorine atom behaves as a proton acceptor and the amino group as a proton donor in hydrogen bonding. ${}^{[100]}$ Generally, it is anticipated that detailed structural information of biosynthetic enzymes could permit efficient protein engineering. ${}^{[101]}$ Therefore, the structural analysis of the enzymes in the biosynthesis of aminoglycoside antibiotics would help to improve the properties of enzymes to create designed molecules that might have greater biological activity. It is also interesting to understand how these enzymes recognize their unique but similar biosynthetic intermediates and convert these to dead-end natural products. Because certain combinations of enzymes give unique natural products (although we cannot ignore the relaxed substrate specificity of secondary metabolite enzymes), we might be able to create designed aminoglycosides by engineering biosynthetic pathways. Furthermore, a genome-mining approach with detailed structural information of the enzymes could be applicable to identify novel aminoglycoside antibiotic biosynthetic gene clusters and permit the identification of new aminoglycosides from natural sources.

It is also evident that several unique enzymes encoded in each biosynthetic gene cluster remain to be characterized. These enzymes appear to be involved in the structural diversification of aminoglycoside antibiotics from common biosynthetic intermediates. One important class of uncharacterized enzymes in the aminoglycoside antibiotic biosynthetic

pathways is the radical SAM superfamily enzymes, which contain a [4Fe–4S] cluster responsible for cleaving SAM to give a 5′-deoxyadenosyl radical, triggering various radical-mediated enzymatic reactions.[102,103] Various putative radical SAM enzymes are encoded in the gene clusters for aminoglycoside antibiotics. Thus, this family of enzymes plays an important role in the expansion of structural diversity among aminoglycoside antibiotics.

It is unclear why such a broad structural diversity of aminoglycoside antibiotics is required by microorganisms. In general, aminoglycoside antibiotics bind to ribosomal RNAs and inhibit protein synthesis in the enemy microorganisms. For self-defense, the producer strains modify their own aminoglycosides to be less active. Combinations of unique enzymes for structural diversification of aminoglycosides and resistance enzymes could be important for maintaining their living environment. To understand the complex relationship between growth and self-defense, we first need to identify the entire set of possible biosynthetic reaction mechanisms involved in aminoglycoside biosynthesis.

### Acknowledgements

We appreciate the many undergraduate and graduate students who were involved in the aminoglycoside antibiotic biosynthetic research projects. The research projects described in this manuscript were supported in part by Grants-in-Aid for Scientific Research from the Ministry of Education, Culture, Sports, Science and Technology of Japan, the Uehara Memorial Foundation, the Noda Institute for Scientific Research, Amano Enzyme, and the Naito Foundation.

### REFERENCES

[1] S. A. Waksman, *Science* **1953**, *118*, 259–266.

[2] S. A. Waksman, H. A. Lechevalier, *Science* **1949**, *109*, 305–307.

[3] H. Umezawa, M. Ueda, K. Maeda, K. Yagishita, Y. Okami, R. Utahara, Y. Osato, K. Nitta, T. Takeuchi, *J. Antibiot.* **1957**, *10*, 181–188.

[4] M. J. Weinstein, G. M. Luedemann, E. M. Oden, G. H. Wagman, J. P. Rosselet, J. A. Marquez, C. T. Coniglio, W. Charney, H. L. Herzog, J. Black, *J. Med. Chem.* **1963**, *6*, 463–464.

[5] J. Distler, K. Mansouri, G. Mayer, M. Stockmann, W. Piepersberg, *Gene* **1992**, *115*, 105–111.

[6] B. Becker, M. A. Cooper, *ACS Chem. Biol.* **2013**, *8*, 105–115.

[7] J. Poehlsgaard, S. Douthwaite, *Nat. Rev. Microbiol.* **2005**, *3*, 870–881.

[8] A. P. Carter, W. M. Clemons, D. E. Brodersen, R. J. Morgan-Warren, B. T. Wimberly, V. Ramakrishnan, *Nature* **2000**, *407*, 340–348.

[9] P. Pfister, S. Hobbie, Q. Vicens, E. C. Bottger, E. Westhof, *ChemBioChem* **2003**, *4*, 1078–1088.

[10] M. S. Ramirez, M. E. Tolmasky, *Drug Resist. Updates* **2010**, *13*, 151–171.

[11] T. Matsumoto, *Clin. Pharmacol.* **2014**, *6*, 139–148.

[12] J. L. Houghton, K. D. Green, W. Chen, S. Garneau-Tsodikova, *ChemBioChem* **2010**, *11*, 880–902.

[13] G. Ho, J. Reichardt, J. Christodoulou, *J. Inherited Metab. Dis.* **2013**, *36*, 955–959.

[14] J. C. Denap, J. R. Thomas, D. J. Musk, P. J. Hergenrother, *J. Am. Chem. Soc.* **2004**, *126*, 15402–15404.

[15] A. Lapidot, A. Berchanski, G. Borkow, *FEBS J.* **2008**, *275*, 5236–5257.

[16] E. Ennifar, M. W. Aslam, P. Strasser, G. Hoffmann, P. Dumas, F. L. van Delft, *ACS Chem. Biol.* **2013**, *8*, 2509–2517.

[17] C. W. Chang, J. Y. Takemoto, *MedChemComm* **2014**, *5*, 1048–1057.

[18] M. Fosso, M. N. AlFindee, Q. Zhang, P. Nziko Vde, Y. Kawasaki, S. K. Shrestha, J. Bearss, R. Gregory, J. Y. Takemoto, C. W. Chang, *J. Org. Chem.* **2015**, *80*, 4398–4411.

[19] L. Desigaux, M. Sainlos, O. Lambert, R. Chevre, E. Letrou-Bonneval, J. P. Vigneron, P. Lehn, J. M. Lehn, B. Pitard, *Proc. Natl. Acad. Sci. U.S.A.* **2007**, *104*, 16534–16539.

[20] I. Das, J. Desire, D. Manvar, I. Baussanne, V. N. Pandey, J. L. Decout, *J. Med. Chem.* **2012**, *55*, 6021–6032.

[21] J. Zhou, G. Wang, L. H. Zhang, X. S. Ye, *Med. Res. Rev.* **2007**, *27*, 279–316.

[22] J. Xie, A. E. Talaska, J. Schacht, *Hear. Res.* **2011**, *281*, 28–37.

[23] M. P. Mingeot-Leclercq, P. M. Tulkens, *Antimicrob. Agents Chemother.* **1999**, *43*, 1003–1012.

[24] M. Hainrichson, I. Nudelman, T. Baasov, *Org. Biomol. Chem.* **2008**, *6*, 227–239.

[25] L. Chen, M. Hainrichson, D. Bourdetsky, A. Mor, S. Yaron, T. Baasov, *Bioorg. Med. Chem.* **2008**, *16*, 8940–8951.

[26] S. N. Hobbie, S. Akshay, S. K. Kalapala, C. M. Bruell, D. Shcherbakov, E. C. Bottger, *Proc. Natl. Acad. Sci. U.S.A.* **2008**, *105*, 20888–20893.

[27] T. Matt, C. L. Ng, K. Lang, S. H. Sha, R. Akbergenov, D. Shcherbakov, M. Meyer, S. Duscha, J. Xie, S. R. Dubbaka, D. Perez-Fernandez, A. Vasella, V. Ramakrishnan, J. Schacht, E. C. Bottger, *Proc. Natl. Acad. Sci. U.S.A.* **2012**, *109*, 10984–10989.

[28] E. Shulman, V. Belakhov, G. Wei, A. Kendall, E. G. Meyron-Holtz, D. Ben-Shachar, J. Schacht, T. Baasov, *J. Biol. Chem.* **2014**, *289*, 2318–2330.

[29] M. E. Huth, K. H. Han, K. Sotoudeh, Y. J. Hsieh, T. Effertz, A. A. Vu, S. Verhoeven, M. H. Hsieh, R. Greenhouse, A. G. Cheng, A. J. Ricci, *J. Clin. Invest.* **2015**, *125*, 583–592.

[30] K. L. Rinehart, Jr., *Jpn. J. Antibiot.* **1979**, *32*, S32–S46.

[31] M. R. Seyedsayamdost, J. Clardy, *ACS Synth. Biol.* **2014**, *3*, 745–747.

[32] K. L. Rinehart, Jr., R. M. Stroshane, *J. Antibiot.* **1976**, *29*, 319–353.

[33] F. Kudo, H. Tamegai, T. Fujiwara, U. Tagami, K. Hirayama, K. Kakinuma, *J. Antibiot.* **1999**, *52*, 559–571.

[34] Y. Ota, H. Tamegai, F. Kudo, H. Kuriki, A. Koike-Takeshita, T. Eguchi, K. Kakinuma, *J. Antibiot.* 2000, 53, 1158–1167.

[35] F. Kudo, M. Numakura, H. Tamegai, H. Yamamoto, T. Eguchi, K. Kakinuma, *J. Antibiot.* 2005, 58, 373–379.

[36] F. Kudo, T. Eguchi, *J. Antibiot.* 2009, 62, 471–481.

[37] W. Piepersberg, K. M. Aboshanab, H. Schmidt-Beissner, U. F. Wehmeier, in *Aminoglycoside Antibiotics* (Eds.: D. P. Arya), Wiley-VCH, Weinheim, 2007, pp. 15–118.

[38] P. M. Flatt, T. Mahmud, *Nat. Prod. Rep.* 2007, 24, 358–392.

[39] K. Kakinuma, Y. Ogawa, T. Sasaki, H. Seto, N. Otake, *J. Am. Chem. Soc.* 1981, 103, 5614–5616.

[40] K. Kakinuma, Y. Ogawa, T. Sasaki, H. Seto, N. Otake, *J. Antibiot.* 1989, 42, 926–933.

[41] N. Yamauchi, K. Kakinuma, *J. Antibiot.* 1992, 45, 774–780.

[42] N. Yamauchi, K. Kakinuma, *J. Antibiot.* 1993, 46, 1916–1918.

[43] N. Yamauchi, K. Kakinuma, *J. Org. Chem.* 1995, 60, 5614–5619.

[44] F. Kudo, N. Yamauchi, R. Suzuki, K. Kakinuma, *J. Antibiot.* 1997, 50, 424–428.

[45] F. Kudo, Y. Hosomi, H. Tamegai, K. Kakinuma, *J. Antibiot.* 1999, 52, 81–88.

[46] E. Nango, T. Kumasaka, T. Hirayama, N. Tanaka, T. Eguchi, *Proteins* 2008, 70, 517–527.

[47] T. Hirayama, F. Kudo, Z. Huang, T. Eguchi, *Bioorg. Med. Chem.* 2007, 15, 418–423.

[48] E. Nango, F. Kudo, T. Eguchi, K. Kakinuma, *Chem. Lett.* 2003, 32, 438–439.

[49] E. P. Carpenter, A. R. Hawkins, J. W. Frost, K. A. Brown, *Nature* 1998, 394, 299–302.

[50] H. Tamegai, E. Nango, M. Kuwahara, H. Yamamoto, Y. Ota, H. Kuriki, T. Eguchi, K. Kakinuma, *J. Antibiot.* 2002, 55, 707–714.

[51] F. Huang, Y. Li, J. Yu, J. B. Spencer, *Chem. Commun.* 2002, 2860–2861.

[52] F. Kudo, Y. Yamamoto, K. Yokoyama, T. Eguchi, K. Kakinuma, *J. Antibiot.* 2005, 58, 766–774.

[53] K. Yokoyama, M. Numakura, F. Kudo, D. Ohmori, T. Eguchi, *J. Am. Chem. Soc.* 2007, 129, 15147–15155.

[54] K. Yokoyama, D. Ohmori, F. Kudo, T. Eguchi, *Biochemistry* 2008, 47, 8950–8960.

[55] K. Yokoyama, F. Kudo, M. Kuwahara, K. Inomata, H. Tamegai, T. Eguchi, K. Kakinuma, *J. Am. Chem. Soc.* 2005, 127, 5869–5874.

[56] F. Huang, S. F. Haydock, T. Mironenko, D. Spiteller, Y. Li, J. B. Spencer, *Org. Biomol. Chem.* 2005, 3, 1410–1418.

[57] K. Yokoyama, Y. Yamamoto, F. Kudo, T. Eguchi, *ChemBioChem* 2008, 9, 865–869.

[58] A. W. Truman, F. Huang, N. M. Llewellyn, J. B. Spencer, *Angew. Chem. Int. Ed.* 2007, 46, 1462–1464.

[59] F. Huang, D. Spiteller, N. A. Koorbanally, Y. Li, N. M. Llewellyn, J. B. Spencer, *ChemBioChem* 2007, 8, 283–288.

[60] X. M. He, H. W. Liu, *Annu. Rev. Biochem.* 2002, 71, 701–754.

[61] F. Kudo, T. Fujii, S. Kinoshita, T. Eguchi, *Bioorg. Med. Chem.* 2007, 15, 4360–4368.

[62] F. Kudo, T. Kawashima, K. Yokoyama, T. Eguchi, *J. Antibiot.* 2009, 62, 643–646.

[63] D. Clausnitzer, W. Piepersberg, U. F. Wehmeier, *J. Appl. Microbiol.* 2011, 111, 642–651.

[64] F. Kudo, S. Hoshi, T. Kawashima, T. Kamachi, T. Eguchi, *J. Am. Chem. Soc.* 2014, 136, 13909–13915.

[65] Y. Li, N. M. Llewellyn, R. Giri, F. Huang, J. B. Spencer, *Chem. Biol.* 2005, 12, 665–675.

[66] N. M. Llewellyn, Y. Li, J. B. Spencer, *Chem. Biol.* 2007, 14, 379–386.

[67] N. M. Llewellyn, J. B. Spencer, *Chem. Commun.* 2008, 3786–3788.

[68] N. M. Llewellyn, J. B. Spencer, *Nat. Prod. Rep.* 2006, 23, 864–874.

[69] F. Kudo, T. Eguchi, *Methods Enzymol.* 2009, 459, 493–519.

[70] R. Takeishi, F. Kudo, M. Numakura, T. Eguchi, *ChemBioChem* 2015, 16, 487–495.

[71] F. Kudo, H. Sucipto, T. Eguchi, *J. Antibiot.* 2009, 62, 707–710.

[72] H. G. Floss, T. W. Yu, K. Arakawa, *J. Antibiot.* 2011, 64, 35–44.

[73] J. W. Park, S. R. Park, K. K. Nepal, A. R. Han, Y. H. Ban, Y. J. Yoo, E. J. Kim, E. M. Kim, D. Kim, J. K. Sohng, Y. J. Yoon, *Nat. Chem. Biol.* 2011, 7, 843–852.

[74] H. Sucipto, F. Kudo, T. Eguchi, *Angew. Chem. Int. Ed.* 2012, 51, 3428–3431.

[75] J. Unwin, S. Standage, D. Alexander, T. Hosted, Jr., A. C. Horan, E. M. Wellington, *J. Antibiot.* 2004, 57, 436–445.

[76] J. W. Park, J. S. Hong, N. Parajuli, W. S. Jung, S. R. Park, S. K. Lim, J. K. Sohng, Y. J. Yoon, *Proc. Natl. Acad. Sci. U.S.A.* 2008, 105, 8399–8404.

[77] C. Huang, F. Huang, E. Moison, J. Guo, X. Jian, X. Duan, Z. Deng, P. F. Leadlay, Y. Sun, *Chem. Biol.* 2015, 22, 251–261.

[78] J. Guo, F. Huang, C. Huang, X. Duan, X. Jian, F. Leeper, Z. Deng, P. F. Leadlay, Y. Sun, *Chem. Biol.* 2014, 21, 608–618.

[79] Y. Gu, X. Ni, J. Ren, H. Gao, D. Wang, H. Xia, *ChemBioChem* 2015, 16, 1933–1942.

[80] H. J. Kim, R. M. McCarty, Y. Ogasawara, Y. N. Liu, S. O. Mansoorabadi, J. LeVieux, H. W. Liu, *J. Am. Chem. Soc.* 2013, 135, 8093–8096.

[81] M. R. Bauerle, E. L. Schwalm, S. J. Booker, *J. Biol. Chem.* 2015, 290, 3995–4002.

[82] Q. Zhang, W. A. van der Donk, W. Liu, *Acc. Chem. Res.* 2012, 45, 555–564.

[83] D. R. Marous, E. P. Lloyd, A. R. Buller, K. A. Moshos, T. L. Grove, A. J. Blaszczyk, S. J. Booker, C. A. Townsend, *Proc. Natl. Acad. Sci. U.S.A.* 2015, 112, 10354–10358.

[84] X. Ni, D. Li, L. Yang, T. Huang, H. Li, H. Xia, *Appl. Microbiol. Biotechnol.* 2011, 89, 723–731.

[85] P. H. Szu, M. W. Ruszczycky, S. H. Choi, F. Yan, H. W. Liu, *J. Am. Chem. Soc.* 2009, 131, 14030–14042.

[86] S. Itoh, Y. Odakura, H. Kase, S. Satoh, K. Takahashi, T. Iida, K. Shirahata, K. Nakayama, *J. Antibiot.* 1984, 37, 1664–1669.

[87] Y. Odakura, H. Kase, S. Itoh, S. Satoh, S. Takasawa, K. Takahashi, K. Shirahata, K. Nakayama, *J. Antibiot.* 1984, 37, 1670–1680.

[88] T. Dairi, T. Ohta, E. Hashimoto, M. Hasegawa, *Mol. Gen. Genet.* 1992, 236, 39–48.

[89] T. Dairi, T. Ohta, E. Hashimoto, M. Hasegawa, *Mol. Gen. Genet.* 1992, 232, 262–270.

[90] T. Dairi, K. Yamaguchi, M. Hasegawa, *Mol. Gen. Genet.* 1992, 236, 49–59.

[91] U. F. Wehmeier, W. Piepersberg, *Methods Enzymol.* 2009, 459, 459–491.

[92] J. B. Walker, *J. Bacteriol.* 1990, 172, 5844–5851.

[93] J. Ahlert, J. Distler, K. Mansouri, W. Piepersberg, *Arch. Microbiol.* 1997, 168, 102–113.

[94] E. Fritsche, A. Bergner, A. Humm, W. Piepersberg, R. Huber, *Biochemistry* 1998, 37, 17664–17672.

[95] H. Yamase, L. Zhao, H.-w. Liu, *J. Am. Chem. Soc.* 2000, 122, 12397–12398.

[96] S. Beyer, G. Mayer, W. Piepersberg, *Eur. J. Biochem.* 1998, 258, 1059–1067.

[97] K. Mansouri, W. Piepersberg, *Mol. Gen. Genet.* 1991, 228, 459–469.

[98] B. Popovic, X. Tang, D. Y. Chirgadze, F. Huang, T. L. Blundell, J. B. Spencer, *Proteins* 2006, 65, 220–230.

[99] P. J. Goldman, T. L. Grove, S. J. Booker, C. L. Drennan, *Proc. Natl. Acad. Sci. U.S.A.* 2013, 110, 15949–15954.

[100] T. Eguchi, S. Sasaki, Z. Huang, K. Kakinuma, *J. Org. Chem.* 2002, 67, 3979–3984.

[101] S. Lutz, *Curr. Opin. Biotechnol.* 2010, 21, 734–743.

[102] P. A. Frey, A. D. Hegeman, F. J. Ruzicka, *Crit. Rev. Biochem. Mol. Biol.* 2008, 43, 63–88.

[103] J. B. Broderick, B. R. Duffus, K. S. Duschene, E. M. Shepard, *Chem. Rev.* 2014, 114, 4229–4317.

Received: July 4, 2015  
Published online: October 12, 2015
