
CHAPTER ONE

# New Insight into the Mechanism and Function of Autophagy in Plant Cells

Xiaochen Yang${}^{1,2}$ and Diane C. Bassham${}^{1,2,3,*}$

${}^{1}$Department of Genetics, Development and Cell Biology, Iowa State University, Ames, IA, USA  
${}^{2}$Interdepartmental Plant Biology Program, Iowa State University, Ames, IA, USA  
${}^{3}$Plant Sciences Institute, Iowa State University, Ames, IA, USA  

*Corresponding author: E-mail: bassham@iastate.edu

## Contents

1. Introduction                                                                                                   2
2. Milestones in Plant Autophagy Study                                                                          3
   2.1 Morphological Studies of Plant Autophagy                                                                3
   2.2 Identification of ATG Genes and Their Regulation and Function                                             5
   2.3 Selective Autophagy                                                                                       8
3. New Discoveries in Autophagy Mechanism                                                                       9
   3.1 Recent Updates on Function of Autophagy Genes                                                            9
   3.2 Evidence of Relationship between Autophagy and Cellular Trafficking                                      11
4. New Discoveries in Autophagy Functions                                                                      13
   4.1 Organelle-Specific Degradation by Autophagy                                                              13
       4.1.1 Mitophagy                                                                                          14
       4.1.2 Degradation of ER by autophagy                                                                    15
       4.1.3 Degradation of chloroplasts by autophagy                                                          16
       4.1.4 Pexophagy                                                                                         18
   4.2 Role of Autophagy in Immune Defense                                                                     20
   4.3 Autophagy and Cell Death                                                                                 22
   4.4 Autophagy Effect on Plant Metabolism                                                                    26
5. Conclusions                                                                                                  28
6. Future Perspectives                                                                                          29
Acknowledgments                                                                                                30
References                                                                                                     30

## Abstract

Autophagy is a degradation pathway that is conserved throughout eukaryotic organisms and plays important roles in the tolerance of abiotic and biotic stresses. It functions as a housekeeping process to remove unwanted cell components under normal conditions, and is induced during stress and senescence to break down damaged cellular

International Review of Cell and Molecular Biology, Volume 320  
ISSN 1937-6448  
http://dx.doi.org/10.1016/bs.ircmb.2015.07.005  
© 2015 Elsevier Inc. All rights reserved.

contents and to recycle materials. The target components are engulfed into specialized transport structures termed autophagosomes and are subsequently delivered to the vacuole for degradation. Here, we review milestones in the study of autophagy in plants, discuss recent advances in our understanding of the mechanism and physiological roles of plant autophagy, and highlight potential future directions of research.

---

### 1. INTRODUCTION

Autophagy (meaning “self-eating”) is a universal mechanism in eukaryotic cells for degrading cell constituents to dispose of damaged or toxic components, and for recycling cellular materials during development or upon encountering stress conditions ([Li and Vierstra, 2012](https://doi.org/10.1093/jxb/ers158); [Liu and Bassham, 2012](https://doi.org/10.1093/jxb/ers157); [Mehrpour et al., 2010](https://doi.org/10.1016/j.cell.2010.04.015); [Yang and Klionsky, 2009](https://doi.org/10.1016/j.cell.2009.05.002)). Usually, cells have a basal level of autophagy under normal conditions to maintain homeostasis. Under adverse growth conditions, autophagy can be induced to remove damaged or unwanted materials or to recycle materials to provide anabolic substrates and metabolites for cells ([Liu and Bassham, 2012](https://doi.org/10.1093/jxb/ers157)).

Three distinct types of autophagy have been identified: microautophagy ([Mijaljica et al., 2011](https://doi.org/10.1016/j.tcb.2011.02.005)), macroautophagy ([Yang and Klionsky, 2009](https://doi.org/10.1016/j.cell.2009.05.002)), and chaperone-mediated autophagy ([Orenstein and Cuervo, 2010](https://doi.org/10.1016/j.cell.2010.04.016)). In plants, only microautophagy and macroautophagy have been reported ([Basham et al., 2006](https://doi.org/10.1093/jxb/erj018); [van Doorn and Papini, 2013](https://doi.org/10.1016/j.plantsci.2013.01.001)). Microautophagy is not well understood in plants and has mainly been studied during leaf senescence and seed development and germination ([Basham et al., 2006](https://doi.org/10.1093/jxb/erj018); [Inada et al., 1998](https://doi.org/10.1104/pp.117.3.1053); [Levanony et al., 1992](https://doi.org/10.1007/BF00033136); [Toyooka et al., 2001](https://doi.org/10.1007/s004250100540); [van Doorn and Papini, 2013](https://doi.org/10.1016/j.plantsci.2013.01.001)). During microautophagy, cytoplasmic components associate with the surface of vacuoles and are directly engulfed by invagination of the tonoplast and pinching off of the membrane to release vesicles named autophagic bodies ([Basham et al., 2006](https://doi.org/10.1093/jxb/erj018)). On the other hand, macroautophagy in plants has been more extensively studied and the core machinery has been described ([Floyd et al., 2012](https://doi.org/10.1093/jxb/ers159); [Li and Vierstra, 2012](https://doi.org/10.1093/jxb/ers158); [Liu and Bassham, 2012](https://doi.org/10.1093/jxb/ers157); [Lv et al., 2014](https://doi.org/10.1093/jxb/ert450)). In plants, as in other organisms, macroautophagy involves the formation of autophagosomes, which are double-membrane vesicles that enwrap cytoplasmic components ([Bassham, 2007](https://doi.org/10.1016/j.plantsci.2007.04.008)). Upon induction of macroautophagy, the phagophore, a cup-shaped double membrane, is formed and the membranes elongate to form complete double-membrane autophagosomes. These are then transported to the vacuole, followed by fusion of the autophagosomal outer membrane with the vacuole membrane.
The resulting autophagic bodies, composed of the inner membrane and its contents, are digested by hydrolytic enzymes within the vacuole (Liu and Basnham, 2012). Presumably, the breakdown materials are exported from the vacuole to the cytoplasm for reuse, but little is known about this process in plants (Lv et al., 2014; Mizushima, 2007). In this review, we will focus on macroautophagy (hereafter autophagy) in plants (Figure 1).

The core mechanism of autophagy is mediated by $AuTophaGy$ ($ATG$) genes. Originally, these $ATG$ genes were identified in yeast cells in screens for mutants with defects in protein turnover, peroxisome degradation, and delivery of a resident vacuolar hydrolase, and more than 30 $ATG$ genes have been reported (Harding et al., 1995; He and Klionsky, 2009; Thumm et al., 1994; Tsukada and Ohsumi, 1993; Yang and Klionsky, 2009). By a comparison of yeast and plant genome sequences, many plant $ATG$ genes have also been identified and shown to have similar functions as in yeast cells (Avin-Wittenberg et al., 2012; Thompson and Vierstra, 2005). In addition, the development of markers in plants for analysis of autophagy enables the rapid detection of autophagy activation. For instance, the most commonly used autophagosome labels are a green fluorescent protein (GFP) fusion of the autophagosome-localized protein $ATG8$ (GFP-$ATG8$), and fluorescent dyes that stain acidic vesicles such as monodansylcadaverine and LysoTracker Red (Bassham, 2014). The autophagy inhibitors used in mammals, such as 3-methyladenine (3-MA), and the phosphatidylinositol 3-kinase inhibitors wortmannin and LY294002, can also inhibit autophagy in plant cells (Takatsuka et al., 2004). Studies of plant autophagy have been greatly facilitated in the past few years by the use of these tools, along with autophagy-defective Arabidopsis mutants.

Here, we will primarily review recent advances in our understanding of plant autophagy, in particular the identification of organelle-specific autophagy pathways and physiological functions of autophagy, and also discuss potential future directions in plant autophagy research.

### 2. MILESTONES IN PLANT AUTOPHAGY STUDY

#### 2.1 Morphological Studies of Plant Autophagy

Early plant autophagy studies were primarily focused on the morphological analysis of autophagy, its role in vacuole biogenesis, and its relationship to nutrient starvation (Aubert et al., 1996; Chen et al., 1994; Marty, 1978; Moriyasu and Ohsumi, 1996; Vanderwilden et al., 1980). Plant
Induction  
ATG1, 11, **13**, 101  

Vesicle  
nucleation  
ATG5, ATG6, VPS34  

Elongation  
and closure  
ATG3, 4, 5, 7, 8, 10,  
11, **12**, 16  

Fusion  
VTI12  

Digestion  

Wortmannin,  
3-MA  

atg9, atg18a  

Membrane  
delivery  
ATG2, 9, **18**  

atg5, atg7  

Transport  
EXO70B1, ESCRT-III  

ConA, E64d  

Recycling  

Figure 1 Schematic representation of autophagy in *Arabidopsis*. The autophagy pathway can be dissected into a series of steps, including induction, vesicle nucleation, phagophore elongation and autophagosome closure, vesicle transport to and fusion with the vacuole, and digestion and recycling of cargo. Upon induction of autophagy, a cup-shaped double-membrane structure, the phagophore, forms around the cargo. The membrane elongates and closes to form a double-membrane vesicle, the autophagosome. During this step, two ubiquitin-like protein conjugation pathways are required. ATG8 is conjugated to phosphatidylethanolamine and recruited to the autophagosome membrane. Lipids for membrane formation and elongation are delivered by the ATG9 complex. The outer membrane of the autophagosome fuses with the vacuole and a single-membrane autophagic body is then broken down by hydrolases in the vacuole. For each step, required genes are summarized and genes in red (dark gray in print versions) are present in gene families. Available tools to block each step are listed; those underlined in black are commonly used drugs and those underlined in red (dark gray in print versions) are commonly used mutants. 3-MA, 3-methyladenine; ConA, concanamycin A.

autophagy is probably initiated at phagophore assembly sites (PAS) and proceeds via elongation of a cup-shaped phagophore, closure to form double-membrane autophagosomes, and fusion of autophagosomes and vacuoles ([Thompson and Vierstra, 2005](https://doi.org/10.1093/jxb/eri068)). Autophagy functions to deliver target components to the vacuole and recycle materials to provide sources of metabolites and energy; therefore, much plant autophagy research was initially focused on responses to starvation ([Aubert et al., 1996](https://doi.org/10.1074/jbc.271.17.10258); [Chen et al., 1994](https://doi.org/10.1074/jbc.269.20.14064); [Moriyasu and Ohsumi, 1996](https://doi.org/10.1074/jbc.271.17.10258)). Upon nutrient starvation, autophagy is activated, as indicated by the presence of an increased number of autophagosomes. Analogous to the fusion of autophagosomes with lysosomes in mammalian cells, autophagosomes are transported to and fuse with vacuoles in plants ([Marty, 1978, 1997](https://doi.org/10.1016/S0014-4827(78)80001-6)). The resulting single-membrane-bound autophagic bodies, along with their cargo, are then digested ([Marty, 1978, 1997](https://doi.org/10.1016/S0014-4827(78)80001-6)). These descriptions of plant autophagy by electron and light microscopy built a foundation of knowledge for future studies of autophagy in many plant species.

### 2.2 Identification of ATG Genes and Their Regulation and Function

Generally, based on their functions in autophagy, the ATG genes can be divided into three major groups: the phosphatidylinositol 3-kinase (PI3K) complex, ATG9 and its cycling system, and two ubiquitination-like conjugation pathways ([Yang and Klionsky, 2009](https://doi.org/10.1016/j.cell.2009.05.005)). Since the genome sequence of *Arabidopsis* became available ([Arabidopsis, 2000](https://doi.org/10.1038/35096500)), the study of autophagy also significantly expanded from morphological studies to analyses of gene function and regulation. The core autophagy machinery, including most of the ATG genes, is conserved between yeast and plants ([Doelling et al., 2002](https://doi.org/10.1074/jbc.M205232200)). By comparison of yeast and plant genome sequences, many ATG genes have been discovered and characterized in plants ([Avin-Wittenberg et al., 2012](https://doi.org/10.1016/j.plantsci.2012.01.001); [Thompson and Vierstra, 2005](https://doi.org/10.1093/jxb/eri068)). Most of the major groups of ATG genes have been found in plants, and their functions are similar to those in yeast cells ([Liu and Bassham, 2012](https://doi.org/10.1016/j.plantsci.2012.01.001); [Lv et al., 2014](https://doi.org/10.1016/j.plantsci.2014.01.001)) (Table 1).

The two ubiquitination-like conjugation pathways are highly conserved between yeast and plants. The conjugation process involves two parallel pathways that generate an ATG8-phosphatidylethanolamine (PE) conjugate, which is the form functional in autophagy ([Fujioka et al., 2008](https://doi.org/10.1016/j.molcel.2008.01.017); [Liu and Bassham, 2012](https://doi.org/10.1016/j.plantsci.2012.01.001); [Ohsumi, 2001](https://doi.org/10.1016/S0092-8674(01)00200-5)). ATG8 is typically synthesized as a longer precursor that is processed by the protease ATG4, generating a C-terminal conserved glycine residue. The cleaved ATG8 product is activated

Table 1 Comparison of yeast and Arabidopsis autophagy core complexes

| Complex                     | Yeast                          | Arabidopsis                  | Properties in yeast                                                                                   | Characterized Arabidopsis mutants               |
|-----------------------------|---------------------------------|-------------------------------|-------------------------------------------------------------------------------------------------------|-----------------------------------------------|
| PI3K complex ATG6           | AtATG6                         |                               | Formation of autophagosomes, vacuolar protein sorting                                                  | atg6-1, atg6 antisense                        |
|                             | ATG14 Not identified            |                               | Connects VPS34 and ATG6                                                                              | None                                          |
|                             | VPS15                          | AtVPS15                      | Serine/threonine kinase, modulates PI3K complex formation                                              | VPS15/vps15 heterozygous                       |
| Initiation and membrane     | VPS34                          | AtVPS34                      | Phosphatidylinositol 3-kinase                                                                        | VPS34/vps34 heterozygous                       |
| delivery                    | ATG1                           | AtATG1a, 1b, 1c, 1t          | Serine/threonine protein kinase, required for autophagy initiation                                     | atg1a-1, atg1a-2                              |
|                             | ATG2                           | AtATG2                       | Interacts with ATG18, required for retrieval of ATG9 to the PAS                                      | atg2-1, atg2-2                                |
|                             | ATG9                           | AtATG9                       | Integral membrane protein, delivery of lipids to expanding phagophore                                 | atg9-1, atg9-2                                |
|                             | ATG13                          | AtATG13a, 13b                | Forms complex with ATG1, required for autophagy initiation                                             | atg13a13b double mutant                       |
|                             | ATG18                          | AtATG18a-h                   | PIP3-interacting peripheral membrane protein                                                          | atg18a, RNAi-ATG18a                           |
|                             | ATG27                          | Not identified               | Type I transmembrane protein, required for the anterograde movement of ATG9                            | None                                          |
| Ubiquitin-like protein      | ATG3                           | AtATG3                       | E2-like enzyme                                                                                       | atg3                                          |
| conjugation systems         | ATG4                           | AtATG4a, 4b                  | Cysteine protease, cleaves ATG8 to expose a terminal glycine                                           | atg4a4b double mutant                         |
|                             | ATG5                           | AtATG5                       | Covalently attached to ATG12                                                                          | atg5-1                                        |
|                             | ATG7                           | AtATG7                       | E1-like enzyme                                                                                       | atg7-1, atg7-2                                |
|                             | ATG8                           | AtATG8a-i                    | Ubiquitin-like protein, phosphatidylethanolamine conjugated form localizes to autophagosomes             | None                                          |
|                             | ATG10                          | AtATG10                      | E2-like enzyme                                                                                       | atg10-1                                       |
|                             | ATG12                          | AtATG12a, 12b                | Conjugated to ATG5                                                                                  | atg12a12b double mutant                       |
|                             | ATG16                          | Not characterized             | Noncovalently bound to ATG5                                                                         | None                                          |
and bound by the E1-like activating enzyme ATG7. ATG8 is then transferred from ATG7 to the E2-like enzyme ATG3. Finally, the ATG8 is covalently ligated to PE, generating the autophagosome-localized lipidated form (Lv et al., 2014; Reggiori and Klionsky, 2013; Yang and Klionsky, 2009). The second conjugation process initiates with activation of ATG12 by ATG7. ATG5 is then conjugated to ATG12 with the aid of a distinct E2-like enzyme ATG10. The ATG12-ATG5 complex interacts with the phagophore via ATG16 to form a tetrameric complex through ATG16 oligomerization (Lv et al., 2014; Reggiori and Klionsky, 2013; Yang and Klionsky, 2009). The ATG12-ATG5/ATG16 complex functions as an E3 ligase which can transfer PE to ATG8 in vitro (Chung et al., 2010; Hanada et al., 2007). Both conjugation processes are required for phagophore expansion and autophagosome closure (Li and Vierstra, 2012; Liu and Bassham, 2012; Lv et al., 2014).

In contrast to the yeast *ATG* genes, which are mostly single copy, many plant *ATG* genes exist as gene families, including *ATG4*, *ATG8*, and *ATG12*. Members of these gene families have some functional overlap, but also some distinct functions (Chung et al., 2010; Slavikova et al., 2005; Yoshimoto et al., 2004). For instance, the *ATG8* family in Arabidopsis has nine members with a high degree of sequence similarity. These *ATG8* genes have similar expression patterns under normal conditions but can be differentially induced in different organs of plants by starvation (Slavikova et al., 2005). *ATG4* has two isoforms with different expression patterns and different functions (Woo et al., 2014). The *ATG12* family has two members, *ATG12a* and *ATG12b*, which have overlapping functions (Chung et al., 2010). Single mutants of *ATG12a* or *ATG12b* do not have detectable phenotypes, but the double mutant has early senescence and is hypersensitive to starvation stresses. However, *ATG12a* and *ATG12b* have distinct expression patterns during different developmental stages or in response to stress conditions (Chung et al., 2010). *ATG12b* transcript levels are much higher than *ATG12a* in young seedlings, but much lower in older plants. Moreover, *ATG12a* expression is greatly increased during leaf senescence (Chung et al., 2010). Thus, while plant autophagy involves highly conserved *ATG* genes and basic molecular mechanisms, plants have many *ATG* gene isoforms which are differently regulated in response to different conditions.

The understanding of autophagy function has also greatly expanded in last few years. Besides its important role in starvation responses, autophagy is also required in plants for tolerance of many abiotic stress conditions,
including oxidative and salt stress, and also in responses to pathogen infection (Hofius et al., 2009; Liu et al., 2005, 2009; Xiong et al., 2007). Moreover, autophagy is involved in plant development, for example, in programmed cell death (PCD) and pollen formation (Kurusu et al., 2014; Kwon et al., 2010). Therefore, autophagy is implicated in almost every aspect of plant growth, from embryogenesis to senescence, and in numerous stress responses.

## 2.3 Selective Autophagy

Previously, autophagy was considered to be a bulk degradation pathway with no selectivity for specific cargos. However, in mammalian and yeast cells, selective autophagy pathways have now been reported that require adapters for cargo selection (Johansen and Lamark, 2011; Yang and Klionsky, 2009), often via tagging of the cargo with ubiquitin. Protein ubiquitination is catalyzed by a cascade of reactions which involve activation, conjugation, and ligation steps. Usually, ubiquitin-tagged proteins are targeted for degradation by 26S proteasomes (Gamerding et al., 2011; Nandi et al., 2006). However, proteasomes can only degrade individual polypeptides (Gamerding et al., 2011; Nandi et al., 2006). Under certain stress conditions, proteins to be degraded may form large ubiquitinated complexes by cross-linking of adjacent amino acids (AAs) and internal hydrogen bonds (Nandi et al., 2006). These large complexes cannot be degraded by 26S proteasomes but can be selectively degraded by autophagy (Johansen and Lamark, 2011; Nandi et al., 2006). Two of the best characterized adapters in mammalian cells that mediate clearance of ubiquitinated protein aggregates are neighbor of BRCA1 gene 1 (NBR1) and p62. NBR1 and p62 have a ubiquitin-associated domain that can bind to ubiquitinated proteins and an LC3-interacting region that interacts with the mammalian ATG8 homolog LC3 (MAP1 light chain 3) (Johansen and Lamark, 2011). In plants, autophagy also is involved in the degradation of ubiquitin-tagged proteins (Zhou et al., 2013). Recently, in *Arabidopsis*, AtNBR1 (and its homolog Joka2 in tobacco) was reported to have a domain structure similar to NBR1 and p62 and may be involved in the selective degradation of ubiquitin-tagged proteins via autophagy (Svenning et al., 2011; Zhou et al., 2013; Zientara-Rytter et al., 2011; Zientara-Rytter and Sirko, 2014b). Another possible adapter for selective autophagy is ATG8-interacting protein 1 (ATI1), which interacts with ATG8, defines a compartment associated with the endoplasmic reticulum (ER) network, and is delivered to the vacuole (Honig et al., 2012). In addition to a role at the ER, ATI1 is also

involved in delivery of plastid proteins to the vacuole by autophagy (Michaeli et al., 2014), implicating this protein in multiple transport pathways.

---

### 3. NEW DISCOVERIES IN AUTOPHAGY MECHANISM

#### 3.1 Recent Updates on Function of Autophagy Genes

The functions of many of the core autophagy genes have now been studied in plants, including their roles in ATG8 lipidation, phagophore initiation, and autophagosome formation (Liu and Bassham, 2012; Shibutani and Yoshimori, 2014). However, many details of these processes are not fully understood even in yeast (Shibutani and Yoshimori, 2014). Recently, some of the known ATG genes have been shown to have additional roles in the autophagy pathway (Kurusu et al., 2014; Le Bars et al., 2014; Woo et al., 2014).

ATG5 is one of the best-studied ATG genes in all species due to its important role in the PE conjugation of ATG8. In addition to this role, ATG5 functions early in the formation of the phagophore. ATG5 is present at the phagophore formation site and is crucial for the recruitment of ATG proteins to the membrane during phagophore elongation (Itakura and Mizushima, 2010; Koyama-Honda et al., 2013; Mizushima et al., 2001). Recently, Le Bars et al. (2014) showed that ATG5 functions during phagophore formation and elongation in Arabidopsis. Using real-time and three-dimensional imaging, ATG5 was found to define the initiation site of the phagophore on a subregion of the ER membrane, followed by recruitment of ATG8 for phagophore elongation. Immediately prior to closure of the cup-shaped phagophore, ATG5 leaves the phagophore, and the newly formed autophagosome then leaves the ER. However, one question arising is why ATG5-mediated phagophore initiation only occurs at the ER. The extent to which autophagosomes form at other sites in addition to the ER is not clear, and other pathways may also participate in phagophore initiation.

ATG7 functions as an ubiquitin E1-like enzyme and is required for PE conjugation to ATG8 during autophagosome formation. An *atg7* knockout mutant therefore lacks autophagy even under stress conditions (Doelling et al., 2002). Nonetheless, *Arabidopsis atg7* mutants can finish their life cycle and produce seeds, although *atg7* mutants are slightly smaller than wild-type (WT) plants (Doelling et al., 2002). Recently, ATG7 was reported to be

required for pollen development by mediating lipid metabolism in rice anthers (Kurusu et al., 2014). A rice *atg7* mutant is defective in starvation-induced autophagy (Kurusu et al., 2014) and, unlike in *Arabidopsis*, has a sterile phenotype. This can be rescued by fertilization with WT pollen, indicating defective pollen (Kurusu et al., 2014), shown to be due to loss of autophagy in tapetum cells during microspore development (Kurusu et al., 2014). The tapetum cells are in the innermost layer of the anther where lipids are degraded and act as nutrient and metabolite suppliers for microspore development and pollen maturation (Ariizumi and Toriyama, 2011). Therefore, the sterility of *atg7* is caused by defective lipid degradation by autophagy during pollen development (Kurusu et al., 2014).

This study indicates the variability in pollen formation and its requirement for autophagy between plant species. In contrast to rice, most *Arabidopsis* autophagy mutants can finish their life cycles and do not have pollen formation or germination defects. One reason may be that lipidic tapetosomes exist in the tapetum cells of *Arabidopsis* but not in rice. These lipidic tapetosomes have ER-derived vesicles and lipid droplets that can be delivered to the pollen to provide nutrients and metabolites, and thus autophagy is not required (Hsieh and Huang, 2007). In *Arabidopsis*, the only autophagy mutant with reproductive defects is *atg6* (Fujiki et al., 2007; Harrison-Lowe and Olsen, 2008; Qin et al., 2007). ATG6 is a conserved component of the PI3K complex and recruits other ATG proteins to the PAS (Liang et al., 1999; Liu et al., 2005; Melendez et al., 2003). However, the sterile phenotype of the *Arabidopsis atg6* mutant is distinct from that of *osatg7*. Pollen development and maturation are both normal in the *atg6* mutant, but *atg6* mutant pollen grains cannot germinate (Fujiki et al., 2007; Harrison-Lowe and Olsen, 2008; Qin et al., 2007). ATG6 is a multifunctional protein that is also required for other vesicle trafficking pathways, including transport to the vacuole. The pollen germination defect therefore is probably not due to the autophagy defects but rather to loss of other ATG6 functions, such as in vesicular trafficking (Fujiki et al., 2007; Kametaka et al., 1998; Lobstein et al., 2004; Monteiro et al., 2005; Zouhar et al., 2009).

During autophagosome formation, ATG8 is conjugated to PE by two ubiquitination-like conjugation pathways. ATG4 is a cysteine protease that cleaves the C-terminus of ATG8 to expose a conserved glycine residue (Kirisako et al., 2000). The exposed glycine residue is required for the binding of the E2-like enzyme ATG7 to ATG8 (Kirisako et al., 2000; Woo et al., 2014; Yoshimoto et al., 2004). In *Arabidopsis*, there are nine ATG8 and two ATG4 proteins. The specificity of the ATG4 isoforms for different isoforms
of ATG8 was recently addressed (Woo et al., 2014). Synthetic ATG4 substrates were generated by fusion of citrine fluorescent protein to the N-terminus of ATG8 and Renilla luciferase superhRLUC (ShR) to the C-terminus. Upon cleavage by ATG4, the ShR is separated from ATG8 and the cleavage efficiency can be quantified based on the amount of free ShR activity. In addition, efficiency can be monitored by bioluminescence resonance energy transfer due to the close excitation and emission wavelengths of citrine and ShR. With this method, ATG4a was shown to be the predominant enzyme for processing the ATG8 family. ATG4a cleaves ATG8a, ATG8c, ATG8d, and ATG8i more efficiently than does ATG4b, whereas ATG4a and ATG4b cleave the remaining ATG8s to a similar degree. This system was also used successfully in vivo in *Arabidopsis* (Woo et al., 2014). The synthetic ATG8 fusions were expressed in plants and were properly processed in vivo in an ATG4-dependent manner (Woo et al., 2014). The differences in specificity between the two ATG4 isoforms suggest that they may have some nonredundant functions. Previous studies showed that distinct ATG8 genes have different expression patterns in response to stress conditions (Slavikova et al., 2005; Tsai et al., 2012; Yoshimoto et al., 2004). The specificity of ATG4 for different ATG8 isoforms may also be correlated with stress responses in different plant tissues and organs. In response to different stresses, the ATG8 isoforms may also be processed by ATG4 with different efficiencies. However, the difference between the ATG8s is still not clear in terms of their roles in distinct biological processes and under different stress conditions (Liu and Bassham, 2012; Lv et al., 2014; Slavikova et al., 2005).

### 3.2 Evidence of Relationship between Autophagy and Cellular Trafficking

Vesicular trafficking transports important materials, including proteins and lipids, to different organelles to support cell functions (Hwang and Robinson, 2009). Typical trafficking steps include vesicle budding, transport, capture by the target organelle, and fusion between the vesicle and its target (Brandizzi and Hawes, 2004; Hwang and Robinson, 2009; Jurgens, 2004). Coat proteins, SNAREs, and tethering factors are important to ensure correct and efficient delivery of cell components to their target (Kim and Brandizzi, 2012). Coat proteins interact with cargo receptors during vesicle budding (Jurgens, 2004) whereas SNARE proteins and tethers are important for specific recognition and fusion between vesicle and target membranes (Kim and Brandizzi, 2012). Similar to the vesiculartrafficking process, autophagy also involves autophagosome formation, transport, and fusion with the vacuole in plants. However, autophagosome formation does not appear to occur via direct vesicle budding from an organelle, as in vesicular trafficking. In addition, autophagosomes are double-membrane vesicles. Although the mechanism of autophagosome formation is still not fully understood, based on current information, the phagophore forms around its cargo, possibly guided by ATG5 (Le Bars et al., 2014). The membrane for phagophore elongation may be delivered via membrane trafficking pathways involving ATG9 and its recycling system, which is proposed to deliver lipids to the expanding phagophore (He and Klionsky, 2007). Therefore, autophagosomes are newly formed upon induction of autophagy.

Despite these differences, recent results have indicated that autophagy shares many key factors with typical vesicle trafficking mechanisms. Firstly, in yeast and mammalian cells, SNARE proteins are involved in autophagosome biogenesis and maturation and are also involved in the fusion between autophagosomes and lysosomes (Darsow et al., 1997; Fader et al., 2009; Furuta et al., 2010; Mayer and Wickner, 1997; Moreau et al., 2011; Nair et al., 2011; Sato et al., 1998). In plants, the SNARE VTI12 may be required for autophagy (Surpin et al., 2003). A *vti12* mutant is sensitive to starvation and shows accelerated senescence, phenotypes similar to those of autophagy-defective mutants (Surpin et al., 2003).

Secondly, studies in mammalian cells showed that exocyst complex components such as SEC5 and EXO84, typically involved in exocytosis, are also involved in autophagy-related transport to the vacuole (Bodemann et al., 2011; Heider and Munson, 2012). Recently, Kulich et al. (2013) found that EXO70B1, a subunit of the exocyst complex, which is also involved in exocytosis in plants (Cvrckova et al., 2012), functions in autophagosome transport to plant vacuoles. EXO70B1 resides on compartments that also contain ATG8f, presumably autophagosomes, which are delivered to vacuoles (Kulich et al., 2013). Moreover, a knockout mutant in EXO70B1 is more susceptible to starvation (Kulich et al., 2013). These results suggest that trafficking components are important for autophagy-related transport.

Thirdly, ESCRT-III (endosomal sorting complex required for transport-III) is an important factor in sorting into multivesicular bodies, a type of endosome, which then fuse with vacuoles or lysosomes, leading to degradation of their contents (Henne et al., 2011). AMSH1 (associated molecule with the SH3 domain of stam1) is a deubiquitinating enzyme which interactswith the endocytosis machinery. The ESCRT-III subunit VPS2.1 interacts with AMSH1 and this interaction is required for the autophagy pathway (Katsiarimpa et al., 2013). In addition, another subunit of ESCRT-III, charged multivesicular body protein 1 (CHMP1), is required for plastid turnover by autophagy (Spitzer et al., 2015). Recently, Gao et al. (2015) identified a plant-specific ESCRT component FREE1 that regulates vacuolar protein transport and also autophagic degradation (Kolb et al., 2015). In a *free1* mutant, lytic vacuole formation is defective and autophagosomes accumulate even under normal conditions (Gao et al., 2015). Moreover, FREE1 interacts with autophagy regulator SH3 domain-containing protein 2 (SH3P2) (Gao et al., 2015). Therefore, FREE1 is also involved in the autophagy pathway.

Finally, it has been suggested that autophagosomes interact with microtubules. In mammalian cells, the autophagosome membrane protein LC3 interacts with microtubules (He et al., 2003). However, in yeast, microtubules are not required for bulk autophagy induction upon stress treatment, although ATG8 interacts with tubulin via ATG4 in vitro (Lang et al., 1998). In plant cells, studies also showed that ATG8 interacts with microtubules in vitro, suggesting that microtubules may be involved in autophagy (Ketelaar et al., 2004). Recently, Zientara-Rytter et al. (2014a) reported that the selective autophagy marker Joka2 colocalizes with microtubules. Therefore, it is possible that microtubules are specifically required for selective autophagy and not for nonselective autophagy.

Together, this evidence indicates that vesicular trafficking and autophagy have a number of features and components in common. However, more studies are needed to investigate the roles of vesicular trafficking proteins during autophagy. For instance, key proteins regulating the steps of recognition and fusion of autophagosomes with the vacuole have not been fully elucidated in plants.

$\rightarrow$ **4. NEW DISCOVERIES IN AUTOPHAGY FUNCTIONS**

**4.1 Organelle-Specific Degradation by Autophagy**

Until recently, reports of selective autophagy in plants have been very limited (Floyd et al., 2012). Potential selective autophagy adapters have now been described in plants, although their function in selective autophagy is still not fully understood (Lamark et al., 2009; Svenning et al., 2011; Zhou et al., 2013). Accumulating data indicate that autophagy can degrade
particular organelles under certain conditions, suggesting specificity, although organelle-specific adapters have yet to be identified. Here, we review recent progress in our understanding of organelle degradation by autophagy (Figure 2).

### 4.1.1 Mitophagy

Mitochondria are found in almost all eukaryotes and provide energy for cells (Dudkina et al., 2010). Most redox and electron transport chain reactions

Senescence, nitrogen  
and fixed-carbon  
starvation, HR-PCD  

Fixed-carbon  
starvation,  
oxidative stress  

Degradation of  
chloroplasts  
chmp1, ati1, atg5,  
atg7, atg4a4b  

RCB  

Pexophagy  
atg2, atg7,  
atg18a, atg3  

Vacuole  

Degradation of ER  
ire1b, RNAi-ATG18a  

TM, DTT, heat,  
pathogens  

Nitrogen and fixed-  
carbon starvation,  
senescence  

Mitophagy  
atg7, atg11  

---

**Figure 2** Schematic representation of organelle-specific autophagy. Four types of organelle-specific autophagy are depicted: degradation of chloroplasts, degradation of peroxisomes (pexophagy), degradation of endoplasmic reticulum (ER), and degradation of mitochondria (mitophagy). Autophagy can degrade chloroplasts either by engulfing the entire chloroplast or by engulfing fragments of the chloroplast, termed Rubisco-containing bodies (RCB). Similarly, for the other organelles, autophagy may engulf the entire organelle or a portion. Stress conditions that trigger each type of organelle-specific autophagy are summarized. Mutants that have been shown to have defects in a specific type of organelle-specific autophagy are listed. HR-PCD, hypersensitive response-programmed cell death; TM, tunicamycin; DTT, dithiothreitol.

occur in the inner membrane of mitochondria (Dudkina et al., 2010), including those which can generate toxic reactive oxygen species (ROS). The mitochondria can be easily damaged by these ROS (Mao et al., 2013), therefore, mitochondrial quality control is critical for cell viability. Extensive studies in yeast and animals have shown that autophagy is involved in the degradation of mitochondria in a process termed mitophagy (Liu et al., 2014). Recently, Li et al. (2014) showed that the autophagy gene ATG11 is involved in mitochondrial quality control in plants. ATG11 was previously studied in yeast and mammalian cells and has important functions in connecting the ATG1/13 complex to the PAS and binding to selective autophagy adapters (Reggiori and Klionsky, 2013; Suzuki et al., 2007; Yorimitsu and Klionsky, 2005). In Arabidopsis, an *atg11* mutant is defective in the degradation of mitochondria by autophagy during senescence (Li et al., 2014). ATG11 colocalized with a mitochondrial marker, suggesting that autophagy delivers mitochondria to the vacuole for degradation (Li et al., 2014). Consistent with this, the degradation of mitochondrial proteins is attenuated in an *atg11* mutant (Li et al., 2014). However, the selectivity of mitophagy in plants, and the requirement for an adapter that recognizes mitochondria and possibly interacts with ATG11, remains to be addressed.

### 4.1.2 Degradation of ER by autophagy

ER stress is defined as the accumulation of unfolded proteins in the ER lumen (Howell, 2013). Under adverse or heavy protein secretion conditions, the demand on protein folding and maturation increases and thus unfolded or misfolded proteins accumulate in the ER due to its limited folding capacity. This accumulation activates the unfolded protein response (UPR), which transmits signals to the nucleus to increase expression of ER stress response genes, such as molecular chaperones. If the UPR is unable to restore homeostasis within the ER, degradation of ER is induced to eliminate malfunctioning ER components (Howell, 2013). Autophagy has been shown to be one of the pathways for ER degradation (Bernales et al., 2006; Ogata et al., 2006; Urano et al., 2000; Yorimitsu et al., 2006), and defects in autophagy have been linked to many ER stress-related diseases, such as Parkinson and Huntington disease (Lynch-Day et al., 2012; Martin et al., 2014). In plant cells, autophagy degrades ER components under ER stress conditions (Liu et al., 2012).

Autophagy is highly induced upon treatment with tunicamycin and dithiothreitol, which are two well-known ER stress inducers (Howell,

2013). An ER marker colocalizes with the autophagosome marker ATG8 under these conditions and is found inside autophagic bodies in the vacuole (Liu et al., 2012). Two main signaling pathways activate the UPR in plant cells, with key components being IRE1b and bZiP28. Interestingly, *IRE1b* is required for the induction of autophagy upon ER stress, while *bZiP28* is not. Moreover, the downstream target of *IRE1b*, *bZiP60*, is also not required for autophagy (Liu et al., 2012), suggesting that an alternative, as yet undescribed, signaling pathway is responsible for its activation. However, the selectivity of degradation of ER by autophagy is still unknown. For instance, it is unknown whether autophagy can generally degrade the ER and its components, or whether certain regions of the ER, for example, containing misfolded protein aggregates, are selected for degradation via selective autophagy adapters. Moreover, the detailed mechanism of this process also needs to be addressed.

### 4.1.3 Degradation of chloroplasts by autophagy

The chloroplast is a unique organelle of photoautotrophs that captures light energy and converts it to chemical energy in the form of sugars. The process of CO₂ fixation in chloroplasts includes many enzymatic and electron transfer reactions (Ishida et al., 2014); similar to mitochondria, chloroplasts are easily damaged by side products of these reactions and quality control of chloroplasts is vital for plant growth. Moreover, chloroplasts contain the majority of leaf protein and therefore nitrogen and carbon content (Ishida et al., 2014). The most abundant chloroplast protein is Rubisco, which catalyzes the fixation of CO₂ into organic molecules. Studies have shown that autophagy is involved in degradation of chloroplasts and chloroplast proteins, such as Rubisco (Ishida et al., 2014; Michaeli and Galili, 2014). Autophagy can degrade pieces of chloroplasts or entire chloroplasts, depending on the conditions (Chiba et al., 2003; Ishida et al., 2008; Ono et al., 2013; Wada et al., 2009). Chloroplast-derived Rubisco-containing bodies (RCBs), which contain stromal components, were observed during leaf senescence in different plant species (Chiba et al., 2003). Subsequently, autophagy was shown to be involved in delivery of RCBs to the vacuole for degradation. Ishida et al. (2008) demonstrated that the stroma-targeted marker DsRed co-localizes with the autophagosome marker ATG8, and that the formation of RCBs was disrupted in an *atg5* mutant. In addition, to investigate the role of autophagy in chloroplast degradation, Wada et al. (2009) used an individually darkened rosette leaf assay, which accelerates leaf senescence. Two pathways were distinguished for chloroplast degradation by
autophagy: via RCBs to degrade fragments of chloroplasts or via degradation of entire chloroplasts (Wada et al., 2009).

Autophagy also plays a role in the degradation of chloroplasts to counteract infection by avirulent *Pst* DC3000 (*AvrRps4*) (Dong and Chen, 2013). Upon pathogen-induced cell death, chlorosis of the infected area is commonly observed, possibly due to disruption and degradation of chloroplasts (Hofius et al., 2009). Chloroplasts are also the primary source for the defense-related hormones salicylic acid (SA) and jasmonic acid (Vlot et al., 2009; Wasternack, 2007). Dong et al. therefore hypothesized that autophagy may be involved in the degradation of chloroplasts upon *Pst* DC3000 (*AvrRps4*) infection. A significant increase in cytoplasmic structures containing stroma-targeted GFP, which may be RCBs, was found upon infection with *Pst* DC3000 (*AvrRps4*), while no such bodies were observed in autophagy-defective mutants. In addition, the growth of *Pst* DC3000 (*AvrRps4*) was significantly increased in autophagy-defective mutants (Dong and Chen, 2013). These results suggest that autophagy degrades components of chloroplasts upon pathogen infection, and that this is important in the infection process.

In addition to RCBs, a novel autophagic structure, decorated by ATI1, has been shown to be involved in the turnover of plastids (Michaeli et al., 2014). ATI1 was associated with chloroplasts after carbon starvation (Michaeli et al., 2014) and ATI1-decorated bodies, containing stromal markers, appeared to bud off from the chloroplasts (Michaeli et al., 2014). The ATI1 on these bodies interacted with ATG8f, indicating a potential involvement of autophagy in their degradation (Michaeli et al., 2014). As discussed in Section 3.2, CHMP1, an ESCRT-III subunit, is involved in the delivery of plastid cargo to the vacuole for degradation (Spitzer et al., 2015). A *chmp1* mutant accumulates stromal and plastid envelope proteins, and cytoplasmic bodies containing plastid proteins (Spitzer et al., 2015). Moreover, the autophagy proteins ATG8 and ATG1a interacted with chloroplasts and were required for the release of stroma-containing bodies. The autophagy pathway is also impaired in the *chmp1* mutant (Spitzer et al., 2015). These results suggest that a novel autophagic structure is involved in the degradation of chloroplast proteins.

It is still unclear if RCBs are specifically recognized for selective autophagy, and the mechanisms for RCB formation are also not clear. Moreover, autophagy-independent pathways have also been reported to be involved in the degradation of chloroplasts, such as degradation by senescence-associated vacuoles (Martinez et al., 2008; Otegui et al., 2005) and
chloroplast vesiculation-related degradation (Wang and Blumwald, 2014). The cross talk between these different chloroplast degradation pathways, which may be specialized for specific conditions, remains to be investigated. For instance, chloroplasts are degraded by autophagy specifically during fixed-carbon starvation but not nitrogen starvation, which may therefore induce other chloroplast degradation pathways (Izumi et al., 2010).

### 4.1.4 Pexophagy

The peroxisome is a single-membrane organelle found in all eukaryotic cells. In plant cells, it is important for seed germination, plant growth, and development (Hu et al., 2012; Kamada et al., 2003). Peroxisomes are usually located between chloroplasts and mitochondria in leaves under normal conditions and their correct location is important for interactions between organelles during photorespiration (Reumann and Weber, 2006). The glycolate generated in chloroplasts diffuses to peroxisomes due to their close interaction. The glycolate is then oxidized in peroxisomes resulting in glycine, which is transported to mitochondria for further processing (Reumann and Weber, 2006). One by-product produced in peroxisomes is hydrogen peroxide, which can cause damage to proteins or other cell components. An important group of proteins within peroxisomes scavenges and degrades hydrogen peroxide to prevent this damage (Hu et al., 2012; Reumann and Weber, 2006). These include catalase (CAT), a suicide enzyme, which breaks down hydrogen peroxide to water and oxygen but in the meantime is inactivated (Anand et al., 2009). During stress conditions, when insufficient CAT is present, accumulating hydrogen peroxide can damage peroxisomes (Yanik and Donaldson, 2005), which therefore need to be degraded (Lingard et al., 2009).

In animal and yeast cells, peroxisomes have been shown to be degraded through a selective autophagy pathway named pexophagy (Till et al., 2012). Recently, pexophagy was also reported by several groups to occur in plant cells (Farmer et al., 2013; Goto-Yamada et al., 2014; Kim et al., 2013; Shibata et al., 2014; Voitsekhovskaja et al., 2014; Yoshimoto et al., 2014). Shibata et al. identified ethyl methanesulfonate-mediated *Arabidopsis* mutants with abnormal cellular locations of peroxisomes, identified by GFP fused to a peroxisome targeting signal (GFP-PTS1). Three mutants with aggregated peroxisomes were identified. After map-based cloning, the mutated genes were found to correspond to the core autophagy genes *ATG2*, *ATG7*, and *ATG18a*. The peroxisome marker and an autophagosome marker were found to be colocalized, suggesting that the peroxisomes

may be degraded by autophagy. As CAT is the most abundant enzyme in peroxisomes, the amount of CAT was compared in WT plants and autophagy mutants. Indeed, CAT accumulated in the autophagy-defective mutants, but these CAT enzymes were less active than those of WT, potentially due to the suicide mechanism of CAT. Autophagy may therefore be involved in the degradation of damaged peroxisomes, which instead form aggregates in autophagy-defective mutants (Shibata et al., 2014).

The LON protease is found in peroxisomes and is required for sustained import of matrix proteins into peroxisomes in mammalian and plant cells (Burkhart et al., 2013; Lingard et al., 2009; Omi et al., 2008). Farmer et al. used a forward genetic screen to identify suppressors of *lon2* in *Arabidopsis*. Mutations in the core autophagy genes *ATG2*, *ATG3*, and *ATG7* were found to suppress the defects in peroxisome size and abundance of the *lon2* mutant and the decreased stability of peroxisomal matrix proteins (Farmer et al., 2013). Similarly, Goto-Yamada et al. (2014) recently reported that disrupting LON2 resulted in enhanced degradation of peroxisomes by autophagy.

Several other reports have reached a similar conclusion by directly analyzing the morphology and degradation of peroxisomes in autophagy-defective mutants or in the presence of autophagy inhibitors (Kim et al., 2013; Voitsekhovskaja et al., 2014; Yoshimoto et al., 2014). Peroxisomal markers accumulated in hypocotyls of an autophagy-defective *atg7* mutant. When degradation in the vacuole was inhibited using ConA, peroxisome markers accumulated in vacuoles of WT plants but not of the *atg7* mutant (Kim et al., 2013). Similarly, accumulation of peroxisome markers in leaves, but interestingly not in roots, was observed in an *atg5* mutant (Yoshimoto et al., 2014). Voitsekhovskaja et al. (2014) reported that during carbohydrate starvation, peroxisomes were degraded by autophagy. The autophagy inhibitor 3-MA reduced the degradation of peroxisomes, increasing their number (Voitsekhovskaja et al., 2014). Taken together, these studies indicate that pexophagy occurs in plants and is involved in maintaining the morphology and number of peroxisomes. Several interesting questions are also raised. As reported by Yoshimoto et al., the degradation of peroxisomes by autophagy is only found in leaves but not roots. Is this related to the function of peroxisomes during photorespiration? In addition, it will be interesting to determine whether autophagy can degrade peroxisomes simply by engulfing the entire organelle, or whether certain areas of damaged peroxisomes, for example, containing inactivated CAT, can be recognized and removed.

4.2 Role of Autophagy in Immune Defense

Pathogen infection causes stress, yield loss, and even entire plant death (Jones and Dangl, 2006). It is therefore vital to understand the mechanisms of pathogen infection and also the mechanisms by which plants defend against infection. Two tiers of pathogen defense mechanisms cooperate to counteract infection, basal defense and gene-mediated defense (Cui et al., 2015; Jones and Dangl, 2006; Zipfel, 2014). Upon pathogen infection, pattern recognition receptors at the cell surface function as the first layer of defense to recognize pathogens via pathogen-associated molecular patterns, such as bacterial cold-shock protein, flagellin, or EF-Tu (Nurnberger et al., 2004; Zipfel and Felix, 2005). Pattern-triggered immunity (PTI) is induced to defend against these pathogens by the activation of mitogen-activated protein kinase pathways (Dangl et al., 2013; Zipfel, 2014). However, pathogens have evolved to adapt to the first layer of defense by generating virulence factors that can disrupt PTI. These effectors can modify the host immune system to enhance their virulence. Plants, in turn, recognize these effectors via nucleotide-binding/leucine-rich-repeat receptors and trigger a series of rapid reactions to restrict infection of the pathogens. This response is termed effector-triggered immunity (ETI) and is usually accompanied by accumulation of ROS and PCD (Dangl et al., 2013; Zipfel, 2014). Rapid PCD, known as the hypersensitive response (HR), is a key ETI strategy to restrict the spread of the pathogen (Jones and Dangl, 2006). Studies have shown that autophagy is involved in pathogen-induced cell death and plays important roles in defense (Hayward and Dinesh-Kumar, 2011). However, the exact role of autophagy in pathogen defense is still not clear. To complicate matters, autophagy plays different roles in response to biotrophic and necrotrophic pathogens (Hayward and Dinesh-Kumar, 2011; Lenz et al., 2011; Zhou et al., 2014).

For necrotrophic pathogens, studies to date have shown that autophagy functions to prevent infection and to restrict the spread of cell death (Kabbage et al., 2013; Lai et al., 2011; Lenz et al., 2011). Lenz et al. systematically compared the roles of autophagy in defense against a necrotrophic fungal pathogen and a virulent biotrophic bacterial pathogen. The autophagy-defective Arabidopsis mutants *atg5*, *atg10*, and *atg18a* were infected with *Alternaria brassicicola* (necrotrophic) or *Pseudomonas syringae pv. tomato* (*Pst*) (biotrophic). Autophagy was found to play prosurvival roles during necrotrophic pathogen infection, but to enhance biotrophic pathogen infection (Lenz et al., 2011). Lai et al. (2011) also reported that
autophagy-defective mutants are more susceptible to the fungal necrotrophic pathogens *Botrytis cinerea* and *A. brassicicola*, and that autophagy is induced upon *B. cinerea* infection. Recently, Kabbage et al. (2013) analyzed another necrotrophic pathogen, *Sclerotinia sclerotiorum*, which can secrete the nonselective phytotoxin oxalic acid (OA) to promote infection. OA was found to be a suppressor of autophagy during pathogen infection. When OA synthesis was blocked, pathogen-induced cell death was limited to a small area, indicating suppression of infection. However, when the autophagy pathway was also blocked, induction of cell death by the OA-deficient pathogen was restored. Moreover, autophagy-defective mutants cannot rapidly accumulate ROS in response to *S. sclerotiorum*, and in WT plants, autophagy genes are induced upon infection (Kabbage et al., 2013). These studies all point to the conclusion that autophagy plays a positive role in protecting plants from necrotrophic pathogen infection.

There is conflicting data regarding the role of autophagy during biotrophic pathogen infection. Liu et al. found that the autophagy factor Beclin1/ATG6 was required for induction of autophagy by tobacco mosaic virus (TMV) infection in *Nicotiana benthamiana* plants. TMV-induced HR cell death was restricted to the infection site in WT but not in *beclin1* knock-down plants (Liu et al., 2005), suggesting that autophagy functions to restrict PCD to the site of infection. Similarly, in *Arabidopsis*, ATG6 was found to play a prosurvival role during pathogen infection (Patel and Dinesh-Kumar, 2008). In contrast, Hofius et al. (2009) showed that autophagy promoted HR cell death during biotrophic pathogen infection. In *Arabidopsis* autophagy-defective mutants, biotrophic pathogen-induced HR cell death was significantly reduced compared to that in WT plants (Hofius et al., 2009).

A possible explanation for this discrepancy may be found in the relationship between autophagy and SA signaling during pathogen infection (Yoshimoto et al., 2009). The induction of autophagy by pathogens was shown to require the SA signaling pathway, and the spread of cell death was age dependent. In younger leaves, no difference was apparent in cell death between WT plants and autophagy mutants. However, in old leaves, cell death induced by the pathogen was restricted to the infection site in WT plants but not in autophagy mutants. This phenotype was correlated with low SA content in young leaves and high SA content in old leaves (Yoshimoto et al., 2009). Based on this study, the previous conflicting results may be caused by differences in the age of the plants under study, which therefore have different levels of SA.
However, recent studies have indicated that this conclusion does not apply in all cases. Autophagy was found to have a negative role in powdery mildew infection, and both SA-dependent and SA-independent pathways contributed to limiting pathogen-induced cell death at the infection site in an *atg2-2* mutant (Wang et al., 2011). In addition, autophagy has been shown to promote degradation of chloroplasts during *Pst* DC3000-induced cell death, but whether autophagy functions in a prosurvival or prodeath mode was not determined in this case (Dong and Chen, 2013). Later, Kwon et al. showed that the infection by *Pst* DC3000 is unrestricted in *atg5* mutants, unlike in WT plants. The small GTP-binding protein RabG3b was found to positively contribute to HR cell death via activation of autophagy during pathogen infection (Kwon et al., 2013). Therefore, to date, the role of autophagy in biotrophic pathogen defense is still unclear. Many different factors may function together in the defense against biotrophic pathogens, and autophagy may be differentially regulated and function differently in distinct situations in response to these infections, depending on factors such as the specific pathogen and age of the plant (Table 2).

### 4.3 Autophagy and Cell Death

PCD is a conserved phenomenon that occurs throughout eukaryotes (Lord and Gunawardena, 2012) and plays important roles during organism development, growth, fertilization, and death (Lam, 2004; Lord and Gunawardena, 2012). The PCD process has been extensively studied in mammalian cells due to its numerous important functions. PCD is required for the proper development of organ shape, immune defense, and embryogenesis (Elmore, 2007; Lord and Gunawardena, 2012). Based primarily on studies in animal cells, PCD has been classified into three types: apoptosis, autophagic cell death, and programmed necrosis (Kroemer et al., 2009). In plant cells, the PCD process has different characteristics compared to that of mammalian cells due to their distinct cellular and subcellular organization. In addition, homologs of most animal PCD-related genes are not present in plant genomes (van Doorn, 2011).

Because of the thick cell wall and lack of professional phagocytic cells, apoptosis, with its characteristic engulfment of cell corpses by macrophages, cannot occur in plants. However, studies have shown that plant PCD has some apoptosis-like features, including cell shrinkage, chromatin condensation, and nuclear segmentation, but not engulfment by neighboring cells (van Doorn and Woltering, 2005). van Doorn and Woltering (2005) divided

Table 2 Summary of effects of pathogens on autophagy mutants

| Pathogens                          | Pathogen effectors | Host plant    | Effects on autophagy mutant                                      | Age         | References                     |
|------------------------------------|--------------------|---------------|------------------------------------------------------------------|-------------|---------------------------------|
| TMV                                | N protein          | Nicotiana     | More spread of cell death than                                  | Unknown      | Liu et al. (2005).             |
|                                    |                    | benthamiana   | wild type                                                       |             |                                 |
| Pst DC 3000                        | AvrRpm1            | Arabidopsis   | No further spread of infection                                  | Unknown      | Hofius et al. (2009)           |
|                                    |                    |               | beyond the initial infection site                              |             |                                 |
| Pst DC 3000                        | AvrRpm1            | Arabidopsis   | More spread of cell death than                                  | Unknown      | Patel and Dinesh-Kumar         |
|                                    |                    |               | wild type                                                       |             | (2008)                         |
| Pto DC 3000                        | AvrRpm1            | Arabidopsis   | More spread of cell death than                                  | 4–5 weeks    | Lenz et al. (2011)             |
|                                    |                    |               | wild type                                                       |             |                                 |
| Pst DC 3000                        | AvrRpm1            | Arabidopsis   | More spread of cell death than                                  | 7–8 weeks    | Yoshimoto et al. (2009)        |
|                                    |                    |               | wild type                                                       |             |                                 |
| Pst DC 3000                        | AvrRps4            | Arabidopsis   | No further spread of infection                                  | Unknown      | Hofius et al. (2009)           |
|                                    |                    |               | beyond the initial infection site                              |             |                                 |
| Pst DC 3000                        | AvrRps4            | Arabidopsis   | Slightly more spread of cell death                              | Young        | Dong and Chen (2013)            |
|                                    |                    |               | than wild type                                                  |             |                                 |
| Pst DC 3000                        | AvrRps4            | Arabidopsis   | More spread of cell death than                                  | Old          |                                 |
|                                    |                    |               | wild type                                                       |             |                                 |
| Pst DC 3000                        | AvrRpt2            | Arabidopsis   | Similar to wild type                                            | 4 weeks      | Wang et al. (2011)              |
| Golovinomyces                      | PMR4               | Arabidopsis   | No further spread of infection                                  | 4 weeks      | Wang et al. (2011)              |
| cichoracearum                      |                    |               | beyond the initial infection site                              |             |                                 |
| Golovinomyces orontii Emwa isolate | EC2                | Arabidopsis   | Similar to wild type                                            | 4 weeks      | Yoshimoto et al. (2009)        |
| of Hyaloperonospora arabidopsidis   | ATR1               | Arabidopsis   | More spread of cell death than                                  | Unknown      | Hofius et al. (2009)           |
|                                    |                    |               | wild type                                                       |             |                                 |
| Noco2 isolate of                   | AvrRpp14           | Arabidopsis   | Similar to wild type                                            | 4–5 weeks    | Lenz et al. (2011)             |
| Hyaloperonospora arabidopsidis     |                    |               |                                                                  |             |                                 |

TMV, tobacco mosaic virus; Pst, *Pseudomonas syringae* pv. tomato; Pto, *Pseudomonas syringae* pv. tomato.
plant PCD into two classes: vacuolar cell death and necrosis. Vacuolar cell death is characterized by removal of cell components by a combination of an autophagy-like process and release of enzymes from the lytic vacuole, whereas necrosis is defined as cell death that does not have vacuolar cell death features but typically has early rupture of the plasma membrane and protoplast shrinkage (van Doorn, 2011). For vacuolar cell death, autophagy can be used as a marker, and it also has important functions during cell death. Recently, autophagy has been shown to be involved in PCD in many different plant developmental processes, including embryogenesis, tracheary element (TE) formation, male reproductive development, and leaf senescence (Liu and Basham, 2012; Minina et al., 2014; van Doorn, 2011; van Doorn and Woltering, 2010).

The xylem is an important structure for the transport of mineral nutrients and water in vascular plant species (Baucher et al., 2007), and during xylem differentiation, TEs undergo secondary cell wall deposition and PCD (Fukuda, 1997). The Rab GTPase RabG3b contributes to autophagic cell death of TEs during xylem formation (Kwon et al., 2010). RabG3b belongs to the RabG small GTPase subclass, and its mammalian homolog regulates fusion between early endosomes and late endosomes or lysosomes (Jager et al., 2004) and is required for the fusion of autophagosomes with lysosomes (Gutierrez et al., 2004). Transgenic plants overexpressing a RabG3b constitutively active mutant, RabG3bCA, grew much taller than WT, whereas transgenic plants overexpressing a dominant negative mutant RabG3bDN were significantly shorter than WT. RabG3b is required for the formation of xylem cells in the middle and basal sections of inflorescence stems and is also involved in secondary cell wall formation, which is indicative of xylem differentiation (Kwon et al., 2010). Similarly, overexpression of RabG3bCA in poplars promoted xylem development (Kwon et al., 2011). Autophagy was shown to be activated during TE differentiation and in RabG3bCA cells, and inhibited in RabG3bDN cells. Autophagic bodies accumulated in Rab3bCA cells, and RabG3b colocalized with ATG8e, suggesting that RabG3b contributes to autophagy during TE formation and cell death (Kwon et al., 2010).

Leaf senescence is a PCD process characterized by a transition from nutrient production to nutrient mobilization. Autophagy is known for its role in nutrient remobilization during leaf senescence (Guiboileau et al., 2012). Although autophagy-defective mutants can finish their life cycle under normal conditions, they have early senescence phenotypes and are sensitive to nutrient starvation (Doelling et al., 2002; Thompson et al.,
2005). Leaf senescence is characterized by chlorosis due to the degradation of chloroplasts, which contain more than 75% of the total leaf nitrogen in C3 plants (Makino and Osmond, 1991). Three pathways have been described for the vacuolar degradation of chloroplasts (see Section 4.1.3); autophagy (Ishida et al., 2014, 2008; Wada et al., 2009), senescence-associated vacuoles (Carrion et al., 2013; Otegui et al., 2005), and chloroplast vesiculation (Wang and Blumwald, 2014). Autophagy delivers either chloroplast fragments or entire chloroplasts to the vacuole during senescence, and chlorophylls and photosynthetic proteins are degraded more slowly in autophagy mutants than in WT plants (Ishida et al., 2014, 2008; Sakuraba et al., 2014; Wada et al., 2009). These studies indicate that autophagy plays important positive roles during chlorosis of leaves during leaf senescence, which can be considered to be a type of autophagic PCD.

ROS accumulation has often been observed during PCD (Doke, 1983; Lamb and Dixon, 1997). Accumulating evidence has shown that the ROS are not simply a by-product of PCD, but can also act as a trigger for cell death (Jabs et al., 1996; Lamb and Dixon, 1997; Van Breusegem and Dat, 2006). However, the detailed mechanism of ROS-mediated PCD is still unclear. Catalase has recently been shown to promote autophagy-dependent cell death (Hackenberg et al., 2013). An *Arabidopsis* mutant, *nca2*, with reduced catalase activity had no change in autophagy during normal or starvation conditions when compared to WT plants (Hackenberg et al., 2013). However, autophagic cell death was suppressed in the *nca2* mutant upon infection with the biotrophic pathogen *Pst* (Hackenberg et al., 2013). The autophagy-defective mutant *atg2* also had reduced cell death in response to *Pst*. In addition, the selective autophagy marker NBR1 and ubiquitinated proteins accumulated in *nca2* mutants upon *Pst* infection (Hackenberg et al., 2013). These results suggest that catalase functions upstream of autophagy-dependent cell death during the HR.

Interestingly, Minina et al. (2013) recently showed that autophagy is regulated by a metacaspase, a class of arginine-/lysine-specific cysteine proteases distantly related to animal caspases, and that this determines the mode of cell death in embryo suspensors (Uren et al., 2000; Vercammen et al., 2004). Suspensors are embryonic regions that connect the embryo to surrounding maternal tissues during early seed development (Kawashima and Goldberg, 2010; Zhao et al., 2013), and are temporary organs that undergo PCD during seed maturation (Kawashima and Goldberg, 2010; Zhao et al., 2013). In the embryo suspensor of Norway spruce, autophagy was found to be required for cell disassembly during vacuolar cell death, and the activation
of autophagy was dependent on the metacaspase mcII-pa. Deficiency in either mcII-pa or autophagy suppressed vacuolar cell death, and led to necrotic cell death as an alternative. To date, although much progress has been made in determining the roles of autophagy during cell death, one remaining question is still whether autophagy has prosurvival or prodeath functions, or, most likely, has different functions in response to different conditions or development processes. The study by Minina et al. (2013) raised the interesting possibility that autophagy is important in determining the transition between vacuolar cell death and necrotic cell death; this also complicates the determination of the role of autophagy in cell death, as the cell death mechanism needs to be addressed.

### 4.4 Autophagy Effect on Plant Metabolism

Autophagy is a recycling process that can provide anabolic substrates and energy sources for the cell, thus playing an important role in survival during starvation (Bassham, 2007). Autophagy has been shown to be important in providing nutrient sources for growth at night, especially when other nutrient sources are compromised (Izumi et al., 2013a). Under short-day conditions, autophagy-defective mutants are much smaller than WT, suggesting that autophagy promotes growth during the night by delivering materials for degradation in the vacuole to provide metabolites for growth. Plants with mutations in phosphoglucomutase or ADP-glucose pyrophosphorylase, termed starchless mutants, are unable to accumulate starch by the end of the day (Caspar et al., 1985; Izumi et al., 2013b; Lin et al., 1988). The free sugars are used up rapidly and thus sugar availability for plant growth during the night is reduced. Therefore, starchless mutants are smaller than WT plants and senesce prematurely (Izumi et al., 2013b). Autophagy defects enhance the phenotypes of starchless mutants when plants are exposed to a prolonged night (Izumi et al., 2013b). As an alternative respiratory substrate, autophagic degradation supplies free AAs for the cells. The free AA pool was significantly increased in a starchless mutant in phosphoglucomutase (*pgm-1*), potentially due to increased autophagic activity. In addition, the increased AA in the *pgm-1* mutant was compromised in *pgm-1 atg5-1* double mutants. This indicates that autophagy is activated and AA catabolism induced to compensate for sugar limitation in starchless mutants (Izumi et al., 2013b).

In addition to changes in sugar and AA metabolism, autophagy deficiency significantly changes the metabolite profile in Arabidopsis (Izumi et al., 2013a). Large-scale analysis of metabolites in an *atg5* mutant and
WT plants indicated that 36 metabolites were significantly increased in the *atg5* mutant (Izumi et al., 2013a). One interesting group of increased metabolites in the *atg5* mutant is phospholipid-related intermediates that can be used as energy sources during starvation (Bargmann and Munnik, 2006; Izumi et al., 2013a; Kunz et al., 2009). Four metabolites, urate, saccharopine, polyamine, and putrescine, were significantly decreased in the *atg5* mutant (Izumi et al., 2013a). These metabolites are possible intermediates of purine or lysine catabolic pathways and are involved in abiotic stresses (Galili et al., 2001; Groppa and Benavides, 2008; Zrenner et al., 2006). These changes most likely reflect the activation of compensatory mechanisms to provide energy and metabolites when autophagy is disrupted.

As nitrogen is a key nutrient for plant growth and development, autophagy positively regulates nitrogen recycling and mobilization under both normal and stress conditions (Li and Vierstra, 2012; Liu and Bassham, 2012; Lv et al., 2014). Nitrogen remobilization from leaves into developing seeds in *Arabidopsis* is controlled by autophagy under both nitrate-rich and nitrate-limited conditions (Guiboileau et al., 2012). ${}^{15}$N-labeled nitrate was fed to plants during vegetative growth and the remobilization of ${}^{15}$N in *atg5* mutant and WT plants during reproductive growth was assessed. The relocation of nitrogen from leaves to seeds under low nitrogen conditions was suppressed in an *atg5* mutant compared with WT plants (Guiboileau et al., 2012). Comparison of metabolite levels, enzyme activities, and protein content between WT and autophagy-defective mutants under low and high nitrogen conditions (Guiboileau et al., 2012) indicated that autophagy mutants accumulate more ammonium, AA, and proteins than WT plants; however, the sugar concentration was much lower in autophagy mutants than WT (Guiboileau et al., 2013).

In order to more comprehensively understand the function of autophagy in metabolism, the multiple impacts of autophagy on plant development and plant response to the environment were examined by metabolomic analyses (Masclaux-Daubresse et al., 2014). Similar to the analysis by Izumi et al. (2013a) discussed above, metabolite profiling of autophagy mutants showed significant changes in various metabolites, for example, AAs, sterols, sugars, and flavonoids. In particular, autophagy mutants had a reduced concentration of different sugars, including hexoses, acids, and alcohol, under normal growth conditions. These sugar forms are important for maintenance of the redox status in plant cells. The concentration of antioxidant molecules, such as $\alpha$-tocopherol and spermidine, was therefore examined and found to be reduced in autophagy-defective mutants. Moreover, as glutathione is also
important for regulation of redox conditions in cells, the glutathione concentration was further examined by enzymatic assays. Both total glutathione and oxidized glutathione were found to be increased in the mutants, but the degree of glutathione oxidation was unchanged (Masclaux-Daubresse et al., 2014). Under low nitrogen conditions, WT leaves accumulate anthocyanin, which can be observed as purple coloration, and anthocyanin can be delivered to the vacuole by autophagy (Pourcel et al., 2010). Autophagy-defective mutant leaves do not become purple under these conditions and anthocyanin concentrations were found to be reduced in an *atg5* mutant (Masclaux-Daubresse et al., 2014). With this comprehensive analysis of autophagy mutants, extensive information is now available on the effect of autophagy on plant metabolic pathways to provide energy for the cell and also to increase stress tolerance.

Overexpression of a soybean *ATG8* (*GmATG8c*) homolog in *Arabidopsis* was shown to lead to higher tolerance to nitrogen limitation and also higher yield under normal conditions (Xia et al., 2012). *GmATG8c* was selected from a transcript analysis of 11 soybean *ATG8* gene family members based on upregulation upon nitrogen starvation, and was found to also increase soybean callus tolerance to nitrogen depletion. This study showed that it is possible to increase nutrient recycling by overexpressing autophagy proteins, thus enhancing starvation tolerance (Xia et al., 2012). Wada et al. (2015) recently showed that *ATG7*, and presumably the autophagy pathway, affects nitrogen use efficiency in rice. As discussed in Section 3.1, a rice *atg7* mutant is sterile; in this study, vegetative growth is examined. The *atg7* mutant is smaller and has less biomass than WT under normal conditions (Wada et al., 2015). In addition, *atg7* was defective in protein degradation for nitrogen remobilization during senescence (Wada et al., 2015). These studies provide a good foundation for future studies on the effect of autophagy on metabolism. The large-scale analyses provide information on autophagy-related metabolism, and further analysis of these data sets should continue to generate testable hypotheses on the function of autophagy in regulating metabolic pathways and their relationship to stress resistance and nutrient recycling.

---

### 5. CONCLUSIONS

The core mechanism of autophagy has been well studied in the past two decades. Autophagy is involved in many abiotic and biotic stress responses, in
which it degrades and recycles cell components to eliminate damaged components and also to provide alternative energy sources and metabolites. Recently, selective autophagy has received substantial attention, and selective autophagy adapters have now been reported in plants. Certain organelles can be degraded by autophagy under specific conditions, although the currently known selective autophagy adapters may not be responsible for organelle-specific autophagy. In addition, large-scale metabolite analyses have shown that autophagy extensively affects plant metabolism during normal and also stress conditions. These new contributions further illustrate the importance of autophagy in plant development and stress tolerance.

### 6. FUTURE PERSPECTIVES

In plants, many ATG genes exist in small gene families, but the reason for this is unknown. Some studies have already shown that genes within a family are differentially induced upon different stress conditions and during different developmental stages, potentially allowing more nuanced regulation of autophagy activity. Recently developed genome editing tools, such as TALEN (transcription activator-like effector nuclease)- and Cas9/guide RNA-mediated editing, are possible methods to efficiently study these gene families by allowing the disruption either of single members of a gene family or multiple members simultaneously. Moreover, a few ATG genes are missing from plant genomes when compared to yeast, and whether functionally equivalent genes that are unrelated in sequence are present in plants is an interesting evolutionary question.

The core mechanism of autophagy seems to be conserved between all eukaryotes, and many ATG genes have been characterized, but many mechanistic details of autophagosome formation and transport are still not fully understood. Recent studies have shed light on the initiation of formation of autophagosomes in plants. In addition, the mechanism of fusion between autophagosomes and vacuoles is still unknown. Although VTI12 has been suggested to be involved in autophagy, it is not clear if it functions in the fusion step. Another interesting topic that remains almost completely unaddressed is the factors involved in the degradation of cargo inside the vacuole and in recycling of the breakdown products from the vacuole to the cytoplasm.

Although it is well known that autophagy is required for tolerance of many stresses and for some developmental processes, its exact role is not
well studied in several areas. Firstly, the role of autophagy in biotrophic pathogen defense is still under debate. In response to different biotrophic pathogens, autophagy plays either a prosurvival or a prodeath role. It would be interesting to examine different biotrophic pathogens classified by, for example, mode of infection, and investigate whether autophagy functions differently in response to the different classes. The interaction of autophagy with other pathogen defense pathways also deserves additional attention. Secondly, although organelle-specific autophagy has been reported in plants, whether any selective autophagy adapters are involved in this process is still unclear. NBR1 has been reported in plants and is involved in the degradation of ubiquitinated proteins, but whether ubiquitination also acts as a signal for organelle-specific autophagy is unknown. Finally, the roles of autophagy in plant metabolism have been discussed, but recent studies have focused on large-scale analyses during nitrogen or carbon starvation. It will be interesting to dissect in more detail the effect of autophagy on different metabolic pathways and under different conditions.

# ACKNOWLEDGMENTS

We thank Yunting Pu and Brice Floyd for helpful discussions. This work was supported by grant no. IOS-1353867 from the National Science Foundation to D.C.B.

# REFERENCES

Anand, P., Kwak, Y., Simha, R., Donaldson, R.P., 2009. Hydrogen peroxide induced oxidation of peroxisomal malate synthase and catalase. Arch. Biochem. Biophys. 491, 25–31.

Arabidopsis Genome Initiative, 2000. Analysis of the genome sequence of the flowering plant *Arabidopsis thaliana*. Nature 408, 796–815.

Ariizumi, T., Toriyama, K., 2011. Genetic regulation of sporopollenin synthesis and pollen exine development. Annu. Rev. Plant Biol. 62, 437–460.

Aubert, S., Gout, E., Bligny, R., Marty, F., Douce, R., 1996. Ultrastructural and biochemical characterization of autophagy in higher plant cells subjected to carbon deprivation: control by the supply of mitochondria with respiratory substrates. J. Cell Biol. 133, 1251–1263.

Avin-Wittenberg, T., Honig, A., Galili, G., 2012. Variations on a theme: plant autophagy in comparison to yeast and mammals. Protoplasma 249, 285–299.

Bargmann, B.O., Munnik, T., 2006. The role of phospholipase D in plant stress responses. Curr. Opin. Plant Biol. 9, 515–522.

Bassham, D.C., 2007. Plant autophagy—more than a starvation response. Curr. Opin. Plant Biol. 10, 587–593.

Bassham, D.C., 2014. Methods for analysis of autophagy in plants. Methods 75, 181–188.

Bassham, D.C., Laporte, M., Marty, F., Moriyasu, Y., Ohsumi, Y., Olsen, L.J., Yoshimoto, K., 2006. Autophagy in development and stress responses of plants. Autophagy 2, 2–11.

Baucher, M., El Jaziri, M., Vandeputte, O., 2007. From primary to secondary growth: origin and development of the vascular system. J. Exp. Bot. 58, 3485–3501.

Bernalés, S., Mcdonald, K.L., Walter, P., 2006. Autophagy counterbalances endoplasmic reticulum expansion during the unfolded protein response. PLoS Biol. 4, e423.

Bodemann, B.O., Orvedahl, A., Cheng, T., Ram, R.R., Ou, Y.H., Formstecher, E., Maiti, M., Hazelett, C.C., Wauson, E.M., Balakireva, M., Camonis, J.H., Yeaman, C., Levine, B., White, M.A., 2011. RalB and the exocyst mediate the cellular starvation response by direct activation of autophagosome assembly. Cell 144, 253–267.

Brandizzi, F., Hawes, C., 2004. A long and winding road: symposium on membrane trafficking in plants. EMBO Rep. 5, 245–249.

Burkhart, S.E., Lingard, M.J., Bartel, B., 2013. Genetic dissection of peroxisome-associated matrix protein degradation in *Arabidopsis thaliana*. Genetics 193, 125–141.

Carrión, C.A., Costa, M.L., Martinez, D.E., Mohr, C., Humbeck, K., Guiamet, J.J., 2013. In vivo inhibition of cysteine proteases provides evidence for the involvement of ‘senescence-associated vacuoles’ in chloroplast protein degradation during dark-induced senescence of tobacco leaves. J. Exp. Bot. 64, 4967–4980.

Caspar, T., Huber, S.C., Somerville, C., 1985. Alterations in growth, photosynthesis, and respiration in a starchless mutant of *Arabidopsis thaliana* (L.) deficient in chloroplast phosphoglucomutase activity. Plant Physiol. 79, 11–17.

Chen, M.H., Liu, L.F., Chen, Y.R., Wu, H.K., Yu, S.M., 1994. Expression of alpha-amylases, carbohydrate metabolism, and autophagy in cultured rice cells is coordinately regulated by sugar nutrient. Plant J. 6, 625–636.

Chiba, A., Ishida, H., Nishizawa, N.K., Makino, A., Mae, T., 2003. Exclusion of ribulose-1,5-bisphosphate carboxylase/oxygenase from chloroplasts by specific bodies in naturally senescing leaves of wheat. Plant Cell Physiol. 44, 914–921.

Chung, T., Phillips, A.R., Vierstra, R.D., 2010. ATG8 lipidation and ATG8-mediated autophagy in *Arabidopsis* require ATG12 expressed from the differentially controlled ATG12A AND ATG12B loci. Plant J. 62, 483–493.

Cui, H., Tsuda, K., Parker, J.E., 2015. Effector-triggered immunity: from pathogen perception to robust defense. Annu. Rev. Plant Biol. 66, 487–511.

Cvrčková, F., Grunt, M., Bezvoda, R., Hala, M., Kulich, I., Rawat, A., Zarsky, V., 2012. Evolution of the land plant exocyst complexes. Front. Plant Sci. 3, 159.

Dangl, J.L., Horvath, D.M., Staskawicz, B.J., 2013. Pivoting the plant immune system from dissection to deployment. Science 341, 746–751.

Darsow, T., Rieder, S.E., Emr, S.D., 1997. A multispecificity syntaxin homologue, Vam3p, essential for autophagic and biosynthetic protein transport to the vacuole. J. Cell Biol. 138, 517–529.

Doelling, J.H., Walker, J.M., Friedman, E.M., Thompson, A.R., Vierstra, R.D., 2002. The APG8/12-activating enzyme APG7 is required for proper nutrient recycling and senescence in *Arabidopsis thaliana*. J. Biol. Chem. 277, 33105–33114.

Doke, N., 1983. Involvement of superoxide anion generation in the hypersensitive response of potato-tuber tissues to infection with an incompatible race of *Phytophthora infestans* and to the hyphal wall components. Physiol. Plant Pathol. 23, 345–357.

Dong, J., Chen, W., 2013. The role of autophagy in chloroplast degradation and chlorophagy in immune defenses during *Pst DC3000 (AvrRps4)* infection. PLoS One 8, e73091.

van Doorn, W.G., 2011. Classes of programmed cell death in plants, compared to those in animals. J. Exp. Bot. 62, 4749–4761.

van Doorn, W.G., Papini, A., 2013. Ultrastructure of autophagy in plant cells: a review. Autophagy 9, 1922–1936.

van Doorn, W.G., Woltering, E.J., 2005. Many ways to exit? Cell death categories in plants. Trends Plant Sci. 10, 117–122.

van Doorn, W.G., Woltering, E.J., 2010. What about the role of autophagy in PCD? Trends Plant Sci. 15, 361–362.

Dudkina, N.V., Kouril, R., Peters, K., Braun, H.P., Boekema, E.J., 2010. Structure and function of mitochondrial supercomplexes. Biochim. Biophys. Acta 1797, 664–670.

Elmore, S., 2007. Apoptosis: a review of programmed cell death. Toxicol. Pathol. 35, 495–516.

Fader, C.M., Sanchez, D.G., Mestre, M.B., Colombo, M.I., 2009. TI-VAMP/VAMP7 and VAMP3/cellubrevin: two v-SNARE proteins involved in specific steps of the autophagy/multivesicular body pathways. Biochim. Biophys. Acta 1793, 1901–1916.

Farmer, L.M., Rinaldi, M.A., Young, P.G., Danan, C.H., Burkart, S.E., Bartel, B., 2013. Disrupting autophagy restores peroxisome function to an *Arabidopsis lon2* mutant and reveals a role for the LON2 protease in peroxisomal matrix protein degradation. Plant Cell 25, 4085–4100.

Floyd, B.E., Morriss, S.C., Macintosh, G.C., Basham, D.C., 2012. What to eat: evidence for selective autophagy in plants. J. Integr. Plant Biol. 54, 907–920.

Fujiki, Y., Yoshimoto, K., Ohsumi, Y., 2007. An *Arabidopsis* homolog of yeast ATG6/VPS30 is essential for pollen germination. Plant Physiol. 143, 1132–1139.

Fujioka, Y., Noda, N.N., Fujii, K., Yoshimoto, K., Ohsumi, Y., Inagaki, F., 2008. In vitro reconstitution of plant Atg8 and Atg12 conjugation systems essential for autophagy. J. Biol. Chem. 283, 1921–1928.

Fukuda, H., 1997. Tracheary element differentiation. Plant Cell 9, 1147–1156.

Furuta, N., Fujita, N., Noda, T., Yoshimori, T., Amano, A., 2010. Combinational soluble N-ethylmaleimide-sensitive factor attachment protein receptor proteins VAMP8 and Vti1b mediate fusion of antimicrobial and canonical autophagosomes with lysosomes. Mol. Biol. Cell 21, 1001–1010.

Galili, G., Tang, G., Zhu, X., Gakiere, B., 2001. Lysine catabolism: a stress and development super-regulated metabolic pathway. Curr. Opin. Plant Biol. 4, 261–266.

Gamerdinge, M., Carra, S., Behl, C., 2011. Emerging roles of molecular chaperones and co-chaperones in selective autophagy: focus on BAG proteins. J. Mol. Med. (Berl) 89, 1175–1182.

Gao, C., Zhuang, X., Cui, Y., Fu, X., He, Y., Zhao, Q., Zeng, Y., Shen, J., Luo, M., Jiang, L., 2015. Dual roles of an *Arabidopsis* ESCRT component FREE1 in regulating vacuolar protein transport and autophagic degradation. Proc. Natl. Acad. Sci. U.S.A. 112, 1886–1891.

Goto-Yamada, S., Mano, S., Nakamori, C., Kondo, M., Yamawaki, R., Kato, A., Nishimura, M., 2014. Chaperone and protease functions of LON protease 2 modulate the peroxisomal transition and degradation with autophagy. Plant Cell Physiol. 55, 482–496.

Groppa, M.D., Benavides, M.P., 2008. Polyamines and abiotic stress: recent advances. Amino Acids 34, 35–45.

Guiboileau, A., Avila-Ospina, L., Yoshimoto, K., Soulay, F., Azzopardi, M., Marmagne, A., Lothier, J., Masclaux-Daubresse, C., 2013. Physiological and metabolic consequences of autophagy deficiency for the management of nitrogen and protein resources in *Arabidopsis* leaves depending on nitrate availability. New Phytol. 199, 683–694.

Guiboileau, A., Yoshimoto, K., Soulay, F., Bataille, M.P., Avice, J.C., Masclaux-Daubresse, C., 2012. Autophagy machinery controls nitrogen remobilization at the whole-plant level under both limiting and ample nitrate conditions in *Arabidopsis*. New Phytol. 194, 732–740.

Gutierrez, M.G., Munafo, D.B., Beron, W., Colombo, M.I., 2004. Rab7 is required for the normal progression of the autophagic pathway in mammalian cells. J. Cell Sci. 117, 2687–2697.

Hackenberg, T., Juul, T., Auzina, A., Gwizdz, S., Malolepszy, A., Van Der Kelen, K., Dam, S., Bressendorff, S., Lorentzen, A., Roepstorff, P., Nielsen, K.L., Jorgensen, J.E., Hofius, D., Van Breusegem, F., Petersen, M., Andersen, S.U., 2013. Catalase and NO

CATALASE ACTIVITY1 promote autophagy-dependent cell death in *Arabidopsis*. Plant Cell 25, 4616–4626.

Hanada, T., Noda, N.N., Satomi, Y., Ichimura, Y., Fujioka, Y., Takao, T., Inagaki, F., Ohsumi, Y., 2007. The Atg12-Atg5 conjugate has a novel E3-like activity for protein lipidation in autophagy. J. Biol. Chem. 282, 37298–37302.

Harding, T.M., Morano, K.A., Scott, S.V., Klionsky, D.J., 1995. Isolation and characterization of yeast mutants in the cytoplasm to vacuole protein targeting pathway. J. Cell Biol. 131, 591–602.

Harrison-Lowe, N.J., Olsen, L.J., 2008. Autophagy protein 6 (ATG6) is required for pollen germination in *Arabidopsis thaliana*. Autophagy 4, 339–348.

Hayward, A.P., Dinesh-Kumar, S.P., 2011. What can plant autophagy do for an innate immune response? Annu. Rev. Phytopathol. 49, 557–576.

He, C.C., Klionsky, D.J., 2007. Atg9 trafficking in autophagy-related pathways. Autophagy 3, 271–274.

He, C.C., Klionsky, D.J., 2009. Regulation mechanisms and signaling pathways of autophagy. Annu. Rev. Genet. 43, 67–93.

He, H., Dang, Y., Dai, F., Guo, Z., Wu, J., She, X., Pei, Y., Chen, Y., Ling, W., Wu, C., Zhao, S., Liu, J.O., Yu, L., 2003. Post-translational modifications of three members of the human MAP1LC3 family and detection of a novel type of modification for MAP1LC3B. J. Biol. Chem. 278, 29278–29287.

Heider, M.R., Munson, M., 2012. Exorcising the exocyst complex. Traffic 13, 898–907.

Henne, W.M., Buchkovich, N.J., Emr, S.D., 2011. The ESCRT pathway. Dev. Cell 21, 77–91.

Hofius, D., Schultz-Larsen, T., Joensen, J., Tsitsigiannis, D.I., Petersen, N.H., Mattsson, O., Jorgensen, L.B., Jones, J.D., Mundy, J., Petersen, M., 2009. Autophagic components contribute to hypersensitive cell death in *Arabidopsis*. Cell 137, 773–783.

Honig, A., Avin-Wittenberg, T., Ufaz, S., Galili, G., 2012. A new type of compartment, defined by plant-specific Atg8-interacting proteins, is induced upon exposure of *Arabidopsis* plants to carbon starvation. Plant Cell 24, 288–303.

Howell, S.H., 2013. Endoplasmic reticulum stress responses in plants. Annu. Rev. Plant Biol. 64, 477–499.

Hsieh, K., Huang, A.H.C., 2007. Tapetosomes in *Brassica tapetum* accumulate endoplasmic reticulum-derived flavonoids and alkanes for delivery to the pollen surface. Plant Cell 19, 582–596.

Hu, J.P., Baker, A., Bartel, B., Linka, N., Mullen, R.T., Reumann, S., Zolman, B.K., 2012. Plant peroxisomes: biogenesis and function. Plant Cell 24, 2279–2303.

Hwang, I., Robinson, D.G., 2009. Transport vesicle formation in plant cells. Curr. Opin. Plant Biol. 12, 660–669.

Inada, N., Sakai, A., Kuroiwa, H., Kuroiwa, T., 1998. Three-dimensional analysis of the senescence program in rice (*Oryza sativa* L.) coleoptiles. Investigations of tissues and cells by fluorescence microscopy. Planta 205, 153–164.

Ishida, H., Izumi, M., Wada, S., Makino, A., 2014. Roles of autophagy in chloroplast recycling. Biochim. Biophys. Acta 1837, 512–521.

Ishida, H., Yoshimoto, K., Izumi, M., Reisen, D., Yano, Y., Makino, A., Ohsumi, Y., Hanson, M.R., Mae, T., 2008. Mobilization of rubisco and stroma-localized fluorescent proteins of chloroplasts to the vacuole by an ATG gene-dependent autophagic process. Plant Physiol. 148, 142–155.

Itakura, E., Mizushima, N., 2010. Characterization of autophagosome formation site by a hierarchical analysis of mammalian Atg proteins. Autophagy 6, 764–776.

Izumi, M., Hidema, J., Ishida, H., 2013a. Deficiency of autophagy leads to significant changes of metabolic profiles in *Arabidopsis*. Plant Signal. Behav. 8, e25023.

Izumi, M., Hidema, J., Makino, A., Ishida, H., 2013b. Autophagy contributes to nighttime energy availability for growth in *Arabidopsis*. Plant Physiol. 161, 1682–1693.

Izumi, M., Wada, S., Makino, A., Ishida, H., 2010. The autophagic degradation of chloroplasts via rubisco-containing bodies is specifically linked to leaf carbon status but not nitrogen status in *Arabidopsis*. Plant Physiol. 154, 1196–1209.

Jabs, T., Dietrich, R.A., Dangl, J.L., 1996. Initiation of runaway cell death in an *Arabidopsis* mutant by extracellular superoxide. Science 273, 1853–1856.

Jäger, S., Bucci, C., Tanida, I., Ueno, T., Kominami, E., Saftig, P., Eskelinen, E.L., 2004. Role for Rab7 in maturation of late autophagic vacuoles. J. Cell Sci. 117, 4837–4848.

Johansen, T., Lamark, T., 2011. Selective autophagy mediated by autophagic adapter proteins. Autophagy 7, 279–296.

Jones, J.D., Dangl, J.L., 2006. The plant immune system. Nature 444, 323–329.

Jürgens, G., 2004. Membrane trafficking in plants. Annu. Rev. Cell Dev. Biol. 20, 481–504.

Kabbage, M., Williams, B., Dickman, M.B., 2013. Cell death control: the interplay of apoptosis and autophagy in the pathogenicity of *Sclerotinia sclerotiorum*. PLoS Pathog. 9, e1003287.

Kamada, T., Nito, K., Hayashi, H., Mano, S., Hayashi, M., Nishimura, M., 2003. Functional differentiation of peroxisomes revealed by expression profiles of peroxisomal genes in *Arabidopsis thaliana*. Plant Cell Physiol. 44, 1275–1289.

Kametaka, S., Okano, T., Ohsumi, M., Ohsumi, Y., 1998. Apg14p and Apg6/Vps30p form a protein complex essential for autophagy in the yeast, *Saccharomyces cerevisiae*. J. Biol. Chem. 273, 22284–22291.

Katsiarimpa, A., Kalinowska, K., Anzenberger, F., Weis, C., Ostertag, M., Tsutsumi, C., Schwechheimer, C., Brunner, F., Huckelhoven, R., Isono, E., 2013. The deubiquitinating enzyme AMSH1 and the ESCRT-III subunit VPS2.1 are required for autophagic degradation in *Arabidopsis*. Plant Cell 25, 2236–2252.

Kawashima, T., Goldberg, R.B., 2010. The suspensor: not just suspending the embryo. Trends Plant Sci. 15, 23–30.

Ketelaar, T., Voss, C., Dimmock, S.A., Thumm, M., Hussey, P.J., 2004. *Arabidopsis* homologues of the autophagy protein Atg8 are a novel family of microtubule binding proteins. FEBS Lett. 567, 302–306.

Kim, J., Lee, H., Lee, H.N., Kim, S.H., Shin, K.D., Chung, T., 2013. Autophagy-related proteins are required for degradation of peroxisomes in *Arabidopsis* hypocotyls during seedling growth. Plant Cell 25, 4956–4966.

Kim, S.J., Brandizzi, F., 2012. News and views into the SNARE complexity in *Arabidopsis*. Front. Plant Sci. 3, 28.

Kirisako, T., Ichimura, Y., Okada, H., Kabeya, Y., Mizushima, N., Yoshimori, T., Ohsumi, M., Takao, T., Noda, T., Ohsumi, Y., 2000. The reversible modification regulates the membrane-binding state of Apg8/Aut7 essential for autophagy and the cytoplasm to vacuole targeting pathway. J. Cell Biol. 151, 263–275.

Kolb, C., Nagel, M.K., Kalinowska, K., Hagmann, J., Ichikawa, M., Anzenberger, F., Alkofer, A., Sato, M.H., Braun, P., Isono, E., 2015. FYVE1 is essential for vacuole biogenesis and intracellular trafficking in *Arabidopsis thaliana*. Plant Physiol. 167, 1361–1373.

Koyama-Honda, I., Itakura, E., Fujiwara, T.K., Mizushima, N., 2013. Temporal analysis of recruitment of mammalian ATG proteins to the autophagosome formation site. Autophagy 9, 1491–1499.

Kroemer, G., Galluzzi, L., Vandenabeele, P., Abrams, J., Alnemri, E.S., Baehrecke, E.H., Blagosklonny, M.V., El-Deiry, W.S., Golstein, P., Green, D.R., Hengartner, M., Knight, R.A., Kumar, S., Lipton, S.A., Malorni, W., Nunez, G., Peter, M.E., Tschopp, J., Yuan, J., Piacentini, M., Zhivotovsky, B., Melino, G., Nomenclature Committee on Cell Death, 2009. Classification of cell death: recommendations of the Nomenclature Committee on Cell Death 2009. Cell Death Differ 16, 3–11.

Kulich, I., Pecenkova, T., Sekeres, J., Smetana, O., Fendrych, M., Foissner, I., Hoftberger, M., Zarsky, V., 2013. *Arabidopsis* exocyst subcomplex containing subunit EXO70B1 is involved in autophagy-related transport to the vacuole. Traffic 14, 1155–1165.

Kunz, H.H., Scharnewski, M., Feussner, K., Feussner, I., Flugge, U.I., Fulda, M., Gierth, M., 2009. The ABC transporter PXA1 and peroxisomal beta-oxidation are vital for metabolism in mature leaves of *Arabidopsis* during extended darkness. Plant Cell 21, 2733–2749.

Kurusu, T., Koyano, T., Hanamata, S., Kubo, T., Noguchi, Y., Yagi, C., Nagata, N., Yamamoto, T., Ohnishi, T., Okazaki, Y., Kitahata, N., Ando, D., Ishikawa, M., Wada, S., Miyao, A., Hirochika, H., Shimada, H., Makino, A., Saito, K., Ishida, H., Kinoshita, T., Kurata, N., Kuchitsu, K., 2014. OsATG7 is required for autophagy-dependent lipid metabolism in rice postmeiotic anther development. Autophagy 10, 878–888.

Kwon, S.I., Cho, H.J., Jung, J.H., Yoshimoto, K., Shirasu, K., Park, O.K., 2010. The Rab GTPase RabG3b functions in autophagy and contributes to tracheary element differentiation in *Arabidopsis*. Plant J. 64, 151–164.

Kwon, S.I., Cho, H.J., Kim, S.R., Park, O.K., 2013. The Rab GTPase RabG3b positively regulates autophagy and immunity-associated hypersensitive cell death in *Arabidopsis*. Plant Physiol. 161, 1722–1736.

Kwon, S.I., Cho, H.J., Lee, J.S., Jin, H., Shin, S.J., Kwon, M., Noh, E.W., Park, O.K., 2011. Overexpression of constitutively active *Arabidopsis* RabG3b promotes xylem development in transgenic poplars. Plant Cell Environ. 34, 2212–2224.

Lai, Z., Wang, F., Zheng, Z., Fan, B., Chen, Z., 2011. A critical role of autophagy in plant resistance to necrotrophic fungal pathogens. Plant J. 66, 953–968.

Lam, E., 2004. Controlled cell death, plant survival and development. Nat. Rev. Mol. Cell Biol. 5, 305–315.

Lamark, T., Kirkin, V., Dikic, I., Johansen, T., 2009. NBR1 and p62 as cargo receptors for selective autophagy of ubiquitinated targets. Cell Cycle 8, 1986–1990.

Lamb, C., Dixon, R.A., 1997. The oxidative burst in plant disease resistance. Annu. Rev. Plant Physiol. Plant Mol. Biol. 48, 251–275.

Lang, T., Schaeffeler, E., Bernreuther, D., Bredschneider, M., Wolf, D.H., Thumm, M., 1998. Aut2p and Aut7p, two novel microtubule-associated proteins are essential for delivery of autophagic vesicles to the vacuole. EMBO J. 17, 3597–3607.

Le Bars, R., Marion, J., Le Borgne, R., Satiat-Jeunemaitre, B., Bianchi, M.W., 2014. ATG5 defines a phagophore domain connected to the endoplasmic reticulum during autophagosome formation in plants. Nat. Commun. 5, 4121.

Lenz, H.D., Haller, E., Melzer, E., Kober, K., Wurster, K., Stahl, M., Bassham, D.C., Vierstra, R.D., Parker, J.E., Bautor, J., Molina, A., Escudero, V., Shindo, T., Van Der Hoorn, R.A., Gust, A.A., Nurnberger, T., 2011. Autophagy differentially controls plant basal immunity to biotrophic and necrotrophic pathogens. Plant J. 66, 818–830.

Levanony, H., Rubin, R., Altschuler, Y., Galili, G., 1992. Evidence for a novel route of wheat storage proteins to vacuoles. J. Cell Biol. 119, 1117–1128.

Li, F., Vierstra, R.D., 2012. Autophagy: a multifaceted intracellular system for bulk and selective recycling. Trends Plant Sci. 17, 526–537.

Li, F.Q., Chung, T., Vierstra, R.D., 2014. AUTOPHAGY-RELATED11 plays a critical role in general autophagy- and senescence-induced mitophagy in *Arabidopsis*. Plant Cell 26, 788–807.

Liang, X.H., Jackson, S., Seaman, M., Brown, K., Kempkes, B., Hibshoosh, H., Levine, B., 1999. Induction of autophagy and inhibition of tumorigenesis by beclin 1. Nature 402, 672–676.

Lin, T.P., Caspar, T., Somerville, C.R., Preiss, J., 1988. A starch deficient mutant of *Arabidopsis thaliana* with low ADP glucose pyrophosphorylase activity lacks one of the two subunits of the enzyme. Plant Physiol. 88, 1175–1181.

Lingard, M.J., Monroe-Augustus, M., Bartel, B., 2009. Peroxisome-associated matrix protein degradation in *Arabidopsis*. Proc. Natl. Acad. Sci. U.S.A. 106, 4561–4566.

Liu, L., Sakakibara, K., Chen, Q., Okamoto, K., 2014. Receptor-mediated mitophagy in yeast and mammalian systems. Cell Res. 24, 787–795.

Liu, Y., Schiff, M., Czymmek, K., Talloczy, Z., Levine, B., Dinesh-Kumar, S.P., 2005. Autophagy regulates programmed cell death during the plant innate immune response. Cell 121, 567–577.

Liu, Y., Basham, D.C., 2012. Autophagy: pathways for self-eating in plant cells. Annu. Rev. Plant Biol. 63, 215–237.

Liu, Y., Burgos, J.S., Deng, Y., Srivastava, R., Howell, S.H., Basham, D.C., 2012. Degradation of the endoplasmic reticulum by autophagy during endoplasmic reticulum stress in *Arabidopsis*. Plant Cell 24, 4635–4651.

Liu, Y., Xiong, Y., Basham, D.C., 2009. Autophagy is required for tolerance of drought and salt stress in plants. Autophagy 5, 954–963.

Lobstein, E., Guyon, A., Ferault, M., Twell, D., Pelletier, G., Bonhomme, S., 2004. The putative *Arabidopsis* homolog of yeast Vps52p is required for pollen tube elongation, localizes to Golgi, and might be involved in vesicle trafficking. Plant Physiol. 135, 1480–1490.

Lord, C.E., Gunawardena, A.H., 2012. Programmed cell death in *C. elegans*, mammals and plants. Eur. J. Cell Biol. 91, 603–613.

Lv, X., Pu, X., Qin, G., Zhu, T., Lin, H., 2014. The roles of autophagy in development and stress responses in *Arabidopsis thaliana*. Apoptosis 19, 905–921.

Lynch-Day, M.A., Mao, K., Wang, K., Zhao, M., Klionsky, D.J., 2012. The role of autophagy in Parkinson’s disease. Cold Spring Harb. Perspect. Med. 2, a009357.

Makino, A., Osmond, B., 1991. Effects of nitrogen nutrition on nitrogen partitioning between chloroplasts and mitochondria in pea and wheat. Plant Physiol. 96, 355–362.

Mao, K., Wang, K., Liu, X., Klionsky, D.J., 2013. The scaffold protein Atg11 recruits fission machinery to drive selective mitochondria degradation by autophagy. Dev. Cell 26, 9–18.

Martin, D.D., Ladha, S., Ehrnhofer, D.E., Hayden, M.R., 2014. Autophagy in Huntington disease and huntingtin in autophagy. Trends Neurosci. 38, 26–35.

Martinez, D.E., Costa, M.L., Gomez, F.M., Otegui, M.S., Guiamet, J.J., 2008. ‘Senescence-associated vacuoles’ are involved in the degradation of chloroplast proteins in tobacco leaves. Plant J. 56, 196–206.

Marty, F., 1978. Cytochemical studies on GERL, provacuoles, and vacuoles in root meristematic cells of *Euphorbia*. Proc. Natl. Acad. Sci. U.S.A. 75, 852–856.

Marty, F., 1997. The biogenesis of vacuoles: insights from microscopy. Adv. Bot. Res. 25, 1–42.

Masclaux-Daubresse, C., Clement, G., Anne, P., Routaboul, J.M., Guiboileau, A., Soulay, F., Shirasu, K., Yoshimoto, K., 2014. Stitching together the multiple dimensions of autophagy using metabolomics and transcriptomics reveals impacts on metabolism, development, and plant responses to the environment in *Arabidopsis*. Plant Cell 26, 1857–1877.

Mayer, A., Wickner, W., 1997. Docking of yeast vacuoles is catalyzed by the Ras-like GTPase Ypt7p after symmetric priming by Sec18p (NSF). J. Cell Biol. 136, 307–317.

Mehrpour, M., Esclatine, A., Beau, I., Codogno, P., 2010. Overview of macroautophagy regulation in mammalian cells. Cell Res. 20, 748–762.

Melendez, A., Talloczy, Z., Seaman, M., Eskelinen, E.L., Hall, D.H., Levine, B., 2003. Autophagy genes are essential for dauer development and life-span extension in *C. elegans*. Science 301, 1387–1391.

Michaeli, S., Galili, G., 2014. Degradation of organelles or specific organelle components via selective autophagy in plant cells. Int. J. Mol. Sci. 15, 7624–7638.

Michaeli, S., Honig, A., Levanony, H., Peled-Zehavi, H., Galili, G., 2014. *Arabidopsis* ATG8-INTERACTING PROTEIN1 is involved in autophagy-dependent vesicular trafficking of plastid proteins to the vacuole. Plant Cell 26, 4084–4101.

Mijaljica, D., Prescott, M., Devenish, R.J., 2011. Microautophagy in mammalian cells revisiting a 40-year-old conundrum. Autophagy 7, 673–682.

Minina, E.A., Bozhkov, P.V., Hofius, D., 2014. Autophagy as initiator or executioner of cell death. Trends Plant Sci. 19, 692–697.

Minina, E.A., Filonova, L.H., Fukada, K., Savenkov, E.I., Gogvadze, V., Clapham, D., Sanchez-Vera, V., Suarez, M.F., Zhivotovsky, B., Daniel, G., Smertenko, A., Bozhkov, P.V., 2013. Autophagy and metacaspase determine the mode of cell death in plants. J. Cell Biol. 203, 917–927.

Mizushima, N., 2007. Autophagy: process and function. Genes Dev. 21, 2861–2873.

Mizushima, N., Yamamoto, A., Hatano, M., Kobayashi, Y., Kabeya, Y., Suzuki, K., Tokuhisa, T., Ohsumi, Y., Yoshimori, T., 2001. Dissection of autophagosome formation using Apg5-deficient mouse embryonic stem cells. J. Cell Biol. 152, 657–667.

Monteiro, D., Coelho, P.C., Rodrigues, C., Camacho, L., Quader, H., Malho, R., 2005. Modulation of endocytosis in pollen tube growth by phosphoinositides and phospholipids. Protoplasma 226, 31–38.

Moreau, K., Ravikumar, B., Renna, M., Puri, C., Rubinsztein, D.C., 2011. Autophagosome precursor maturation requires homotypic fusion. Cell 146, 303–317.

Moriyasu, Y., Ohsumi, Y., 1996. Autophagy in tobacco suspension-cultured cells in response to sucrose starvation. Plant Physiol. 111, 1233–1241.

Nair, U., Jotwani, A., Geng, J., Gammoh, N., Richerson, D., Yen, W.L., Griffith, J., Nag, S., Wang, K., Moss, T., Baba, M., Mcnew, J.A., Jiang, X., Reggiori, F., Melia, T.J., Klionsky, D.J., 2011. SNARE proteins are required for macroautophagy. Cell 146, 290–302.

Nandi, D., Tahliliani, P., Kumar, A., Chandu, D., 2006. The ubiquitin-proteasome system. J. Biosci. 31, 137–155.

Nurnberger, T., Brunner, F., Kemmerling, B., Piater, L., 2004. Innate immunity in plants and animals: striking similarities and obvious differences. Immunol. Rev. 198, 249–266.

Ogata, M., Hino, S., Saito, A., Morikawa, K., Kondo, S., Kanemoto, S., Murakami, T., Taniguchi, M., Tanii, I., Yoshinaga, K., Shiosaka, S., Hammarback, J.A., Urano, F., Imaizumi, K., 2006. Autophagy is activated for cell survival after endoplasmic reticulum stress. Mol. Cell. Biol. 26, 9220–9231.

Ohsumi, Y., 2001. Molecular dissection of autophagy: two ubiquitin-like systems. Nat. Rev. Mol. Cell Biol. 2, 211–216.

Omi, S., Nakata, R., Okamura-Ikeda, K., Konishi, H., Taniguchi, H., 2008. Contribution of peroxisome-specific isoform of Lon protease in sorting PTS1 proteins to peroxisomes. J. Biochem. 143, 649–660.

Ono, Y., Wada, S., Izumi, M., Makino, A., Ishida, H., 2013. Evidence for contribution of autophagy to rubisco degradation during leaf senescence in *Arabidopsis thaliana*. Plant Cell Environ. 36, 1147–1159.

Orenstein, S.J., Cuervo, A.M., 2010. Chaperone-mediated autophagy: molecular mechanisms and physiological relevance. Semin. Cell Dev. Biol. 21, 719–726.

Otegui, M.S., Noh, Y.S., Martinez, D.E., Vila Petroff, M.G., Staehelin, L.A., Amasino, R.M., Guiamet, J.J., 2005. Senescence-associated vacuoles with intense proteolytic activity develop in leaves of *Arabidopsis* and soybean. Plant J. 41, 831–844.

Patel, S., Dinesh-Kumar, S.P., 2008. *Arabidopsis* ATG6 is required to limit the pathogen-associated cell death response. Autophagy 4, 20–27.

Pourcel, L., Irani, N.G., Lu, Y., Riedl, K., Schwartz, S., Grotewold, E., 2010. The formation of Anthocyanic Vacuolar Inclusions in *Arabidopsis thaliana* and implications for the sequestration of anthocyanin pigments. Mol. Plant 3, 78–90.

Qin, G.J., Ma, Z.Q., Zhang, L., Xing, S.F., Hou, X.H., Deng, J., Liu, J.L., Chen, Z.L., Qu, L.J., Gu, H.Y., 2007. *Arabidopsis* AtBECLIN 1/AtAtg6/AtVps30 is essential for pollen germination and plant development. Cell Res. 17, 249–263.

Reggiori, F., Klionsky, D.J., 2013. Autophagic processes in yeast: mechanism, machinery and regulation. Genetics 194, 341–361.

Reumann, S., Weber, A.P., 2006. Plant peroxisomes respire in the light: some gaps of the photorespiratory C2 cycle have become filled—others remain. Biochim. Biophys. Acta 1763, 1496–1510.

Sakuraba, Y., Lee, S.H., Kim, Y.S., Park, O.K., Hortensteiner, S., Paek, N.C., 2014. Delayed degradation of chlorophylls and photosynthetic proteins in *Arabidopsis* autophagy mutants during stress-induced leaf yellowing. J. Exp. Bot. 65, 3915–3925.

Sato, T.K., Darsow, T., Emr, S.D., 1998. Vam7p, a SNAP-25-like molecule, and Vam3p, a syntaxin homolog, function together in yeast vacuolar protein trafficking. Mol. Cell. Biol. 18, 5308–5319.

Shibata, M., Oikawa, K., Yoshimoto, K., Kondo, M., Mano, S., Yamada, K., Hayashi, M., Sakamoto, W., Ohsumi, Y., Nishimura, M., 2014. Highly oxidized peroxisomes are selectively degraded via autophagy in *Arabidopsis*. Plant Cell 26, 1377.

Shibutani, S.T., Yoshimori, T., 2014. A current perspective of autophagosome biogenesis. Cell Res. 24, 58–68.

Slavikova, S., Shy, G., Yao, Y., Glozman, R., Levanony, H., Pietrokovski, S., Elazar, Z., Galili, G., 2005. The autophagy-associated Atg8 gene family operates both under favourable growth conditions and under starvation stresses in *Arabidopsis* plants. J. Exp. Bot. 56, 2839–2849.

Spitzer, C., Li, F., Buono, R., Roschzttardtz, H., Chung, T., Zhang, M., Osteryoung, K.W., Vierstra, R.D., Otegui, M.S., 2015. The endosomal protein CHARGED MULTIVESICULAR BODY PROTEIN1 regulates the autophagic turnover of plastids in *Arabidopsis*. Plant Cell 27, 391–402.

Surpin, M., Zheng, H., Morita, M.T., Saito, C., Avila, E., Blakeslee, J.J., Bandyopadhyay, A., Kovaleva, V., Carter, D., Murphy, A., Tasaka, M., Raikhel, N., 2003. The VTI family of SNARE proteins is necessary for plant viability and mediates different protein transport pathways. Plant Cell 15, 2885–2899.

Suzuki, K., Kubota, Y., Sekito, T., Ohsumi, Y., 2007. Hierarchy of Atg proteins in pre-autophagosomal structure organization. Genes Cells 12, 209–218.

Svenning, S., Lamark, T., Krause, K., Johansen, T., 2011. Plant NBR1 is a selective autophagy substrate and a functional hybrid of the mammalian autophagic adapters NBR1 and p62/SQSTM1. Autophagy 7, 993–1010.

Takatsuka, C., Inoue, Y., Matsuoka, K., Mori Yasu, Y., 2004. 3-methyladenine inhibits autophagy in tobacco culture cells under sucrose starvation conditions. Plant Cell Physiol. 45, 265–274.

Thompson, A.R., Doelling, J.H., Suttangkakul, A., Vierstra, R.D., 2005. Autophagic nutrient recycling in *Arabidopsis* directed by the ATG8 and ATG12 conjugation pathways. Plant Physiol. 138, 2097–2110.

Thompson, A.R., Vierstra, R.D., 2005. Autophagic recycling: lessons from yeast help define the process in plants. Curr. Opin. Plant Biol. 8, 165–173.

Thumm, M., Egner, R., Koch, B., Schlumpberger, M., Straub, M., Veenhuis, M., Wolf, D.H., 1994. Isolation of autophagocytosis mutants of *Saccharomyces cerevisiae*. FEBS Lett. 349, 275–280.

Till, A., Lakhani, R., Burnett, S.F., Subramani, S., 2012. Pexophagy: the selective degradation of peroxisomes. Int. J. Cell Biol. 2012, 512721.

Toyooka, K., Okamoto, T., Minamikawa, T., 2001. Cotyledon cells of *Vigna mungo* seedlings use at least two distinct autophagic machineries for degradation of starch granules and cellular components. J. Cell Biol. 154, 973–982.

Tsai, Y.C., Koo, Y., Delk, N.A., Gehl, B., Braam, J., 2012. Calmodulin-related CML24 interacts with ATG4b and affects autophagy progression in *Arabidopsis*. Plant J. 73, 325–335.

Tsukada, M., Ohsumi, Y., 1993. Isolation and characterization of autophagy-defective mutants of *Saccharomyces cerevisiae*. FEBS Lett. 333, 169–174.

Urano, F., Wang, X., Bertolotti, A., Zhang, Y., Chung, P., Harding, H.P., Ron, D., 2000. Coupling of stress in the ER to activation of JNK protein kinases by transmembrane protein kinase IRE1. Science 287, 664–666.

Uren, A.G., O’ourke, K., Aravind, L., Pisabarro, M.T., Seshagiri, S., Koonin, E.V., Dixit, V.M., 2000. Identification of paracaspases and metacaspases: two ancient families of caspase-like proteins, one of which plays a key role in MALT lymphoma. Mol. Cell 6, 961–967.

Van Breusegem, F., Dat, J.F., 2006. Reactive oxygen species in plant cell death. Plant Physiol. 141, 384–390.

Vanderwilden, W., Herman, E.M., Chrispeels, M.J., 1980. Protein bodies of mung bean cotyledons as autophagic organelles. Proc. Natl. Acad. Sci. U.S.A. 77, 428–432.

Vercammen, D., Van De Cotte, B., De Jaeger, G., Eeckhout, D., Casteels, P., Vandepoele, K., Vandenberghe, I., Van Beeumen, J., Inze, D., Van Breusegem, F., 2004. Type II metacaspases Atmc4 and Atmc9 of *Arabidopsis thaliana* cleave substrates after arginine and lysine. J. Biol. Chem. 279, 45329–45336.

Vlot, A.C., Dempsey, D.A., Klessig, D.F., 2009. Salicylic Acid, a multifaceted hormone to combat disease. Annu. Rev. Phytopathol. 47, 177–206.

Voitsekhovskaja, O.V., Schiermeyer, A., Reumann, S., 2014. Plant peroxisomes are degraded by starvation-induced and constitutive autophagy in tobacco BY-2 suspension-cultured cells. Front. Plant Sci. 5, 629.

Wada, S., Hayashida, Y., Izumi, M., Kurusu, T., Hanamata, S., Kanno, K., Kojima, S., Yamaya, T., Kuchitsu, K., Makino, A., Ishida, H., 2015. Autophagy supports biomass production and nitrogen use efficiency at the vegetative stage in rice. Plant Physiol. 168, 60–73.

Wada, S., Ishida, H., Izumi, M., Yoshimoto, K., Ohsumi, Y., Mae, T., Makino, A., 2009. Autophagy plays a role in chloroplast degradation during senescence in individually darkened leaves. Plant Physiol. 149, 885–893.

Wang, S., Blumwald, E., 2014. Stress-induced chloroplast degradation in *Arabidopsis* is regulated via a process independent of autophagy and senescence-associated vacuoles. Plant Cell 26, 4875–4888.

Wang, Y., Nishimura, M.T., Zhao, T., Tang, D., 2011. ATG2, an autophagy-related protein, negatively affects powdery mildew resistance and mildew-induced cell death in *Arabidopsis*. Plant J. 68, 74–87.

Wasternack, C., 2007. Jasmonates: an update on biosynthesis, signal transduction and action in plant stress response, growth and development. Ann. Bot. 100, 681–697.

Woo, J., Park, E., Dinesh-Kumar, S.P., 2014. Differential processing of *Arabidopsis* ubiquitin-like Atg8 autophagy proteins by Atg4 cysteine proteases. Proc. Natl. Acad. Sci. U.S.A. 111, 863–868.

Xia, T., Xiao, D., Liu, D., Chai, W., Gong, Q., Wang, N.N., 2012. Heterologous expression of ATG8c from soybean confers tolerance to nitrogen deficiency and increases yield in *Arabidopsis*. PLoS One 7, e37217.

Xiong, Y., Contento, A.L., Nguyen, P.Q., Bassham, D.C., 2007. Degradation of oxidized proteins by autophagy during oxidative stress in *Arabidopsis*. Plant Physiol. 143, 291–299.

Yang, Z.F., Klionsky, D.J., 2009. An overview of the molecular mechanism of autophagy. Curr. Top. Microbiol. Immunol. 335, 1–32.

Yanik, T., Donaldson, R.P., 2005. A protective association between catalase and isocitrate lyase in peroxisomes. Arch. Biochem. Biophys. 435, 243–252.

Yorimitsu, T., Klionsky, D.J., 2005. Atg11 links cargo to the vesicle-forming machinery in the cytoplasm to vacuole targeting pathway. Mol. Biol. Cell 16, 1593–1605.

Yorimitsu, T., Nair, U., Yang, Z.F., Klionsky, D.J., 2006. Endoplasmic reticulum stress triggers autophagy. J. Biol. Chem. 281, 30299–30304.

Yoshimoto, K., Hanaoka, H., Sato, S., Kato, T., Tabata, S., Noda, T., Ohsumi, Y., 2004. Processing of ATG8s, ubiquitin-like proteins, and their deconjugation by ATG4s are essential for plant autophagy. Plant Cell 16, 2967–2983.

Yoshimoto, K., Jikumaru, Y., Kamiya, Y., Kusano, M., Consonni, C., Panstruga, R., Ohsumi, Y., Shirasu, K., 2009. Autophagy negatively regulates cell death by controlling NPR1-dependent salicylic acid signaling during senescence and the innate immune response in Arabidopsis. Plant Cell 21, 2914–2927.

Yoshimoto, K., Shibata, M., Kondo, M., Oikawa, K., Sato, M., Toyooka, K., Shirasu, K., Nishimura, M., Ohsumi, Y., 2014. Organ-specific quality control of plant peroxisomes is mediated by autophagy. J. Cell Sci. 127, 1161–1168.

Zhao, P., Zhou, X.M., Zhang, L.Y., Wang, W., Ma, L.G., Yang, L.B., Peng, X.B., Bozhkov, P.V., Sun, M.X., 2013. A bipartite molecular module controls cell death activation in the Basal cell lineage of plant embryos. PLoS Biol. 11, e1001655.

Zhou, J., Wang, J., Cheng, Y., Chi, Y.J., Fan, B.F., Yu, J.Q., Chen, Z.X., 2013. NBR1-mediated selective autophagy targets insoluble ubiquitinated protein aggregates in plant stress responses. PLoS Genet. 9, e1003196.

Zhou, J., Yu, J.Q., Chen, Z., 2014. The perplexing role of autophagy in plant innate immune responses. Mol. Plant Pathol. 15, 637–645.

Zientara-Rytter, K., Lukomska, J., Moniuszko, G., Gwozdecki, R., Surowiecki, P., Lewandowska, M., Liszewska, F., Wawrzynska, A., Sirko, A., 2011. Identification and functional analysis of Joka2, a tobacco member of the family of selective autophagy cargo receptors. Autophagy 7, 1145–1158.

Zientara-Rytter, K., Sirko, A., 2014a. Selective autophagy receptor Joka2 co-localizes with cytoskeleton in plant cells. Plant Signal. Behav. 9, e28523.

Zientara-Rytter, K., Sirko, A., 2014b. Significant role of PB1 and UBA domains in multimerization of Joka2, a selective autophagy cargo receptor from tobacco. Front. Plant Sci. 5, 13.

Zipfel, C., 2014. Plant pattern-recognition receptors. Trends Immunol. 35, 345–351.

Zipfel, C., Felix, G., 2005. Plants and animals: a different taste for microbes? Curr. Opin. Plant Biol. 8, 353–360.

Zouhar, J., Rojo, E., Bassham, D.C., 2009. AtVPS45 is a positive regulator of the SYP41/SYP61/VTI12 SNARE complex involved in trafficking of vacuolar cargo. Plant Physiol. 149, 1668–1678.

Zrenner, R., Stitt, M., Sonnewald, U., Boldt, R., 2006. Pyrimidine and purine biosynthesis and degradation in plants. Annu. Rev. Plant Biol. 57, 805–836.
