
JBC Papers in Press. Published on November 30, 2015 as Manuscript M115.698233

The latest version is at http://www.jbc.org/cgi/doi/10.1074/jbc.M115.698233

Binding affinities in T7 DNA replisome

Binding Affinities among DNA Helicase-Primase, DNA Polymerase and Replication Intermediates in the Replisome of Bacteriophage T7

Huidong Zhang${}^{1,2}$, Yong Tang${}^{1}$, Seung-Joo Lee${}^{2}$, Zeliang Wei${}^{1}$, Jia Cao${}^{1}$, and Charles C. Richardson${}^{2}$

${}^{1}$Institute of Toxicology, College of Preventive Medicine, Third Military Medical University, Chongqing 400038, PR China; ${}^{2}$Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, Boston, Massachusetts 02115.

Running title: Binding affinities in T7 DNA replisome

To whom correspondence should be addressed at: Huidong Zhang, Institute of Toxicology, College of Preventive Medicine, Third Military Medical University, Chongqing 400038, PR China, Tel: +86 23 68752292; Fax: +86 23-68752296; E-mail: huidongzhang@tmmu.edu.cn; Charles C. Richardson, Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, 240 Longwood Avenue, Boston, MA 02115. Tel.: 617-432-1864; Fax: 617-432-3362; E-mail: ccr@hms.harvard.edu

Keywords: bacteriophage; DNA replication; DNA-protein interaction; enzyme kinetics; pre-steady-state kinetics

---

ABSTRACT

The formation of a replication loop on the lagging strand facilitates coordinated synthesis of the leading- and lagging-DNA strands and provides a mechanism for recycling of the lagging-strand DNA polymerase. As an Okazaki fragment is completed, the loop is released and a new loop is formed as the synthesis of a new Okazaki fragment is initiated. Loop release requires the dissociation of the complex formed by the interactions among helicase, DNA polymerase and DNA. The completion of the Okazaki fragment may result in either a nick or a single-stranded DNA region. In the replication system of bacteriophage T7, the dissociation of the polymerase from either DNA region is faster than that observed for the dissociation of the helicase from DNA polymerase, implying that the replication loop is released more likely through the dissociation of the lagging-strand DNA from polymerase, retaining the polymerase at replication fork.

dissociation of DNA polymerase from DNA and that of helicase from a DNA polymerase-DNA complex are much faster at a nick DNA than the release from an ssDNA region. These results suggest that the replication loop is released as a result of the nick formed when the lagging strand DNA polymerase encounters the previously synthesized Okazaki fragment, releasing lagging-strand DNA and retaining DNA polymerase at the replication fork for the synthesis of next Okazaki fragment.

Bacteriophage T7 has a simple but efficient DNA replication system (1) consisting of gene 5 DNA polymerase (gp5), the processivity factor Escherichia coli thioredoxin (trx), gene 4 helicase-primase (gp4) and gene 2.5 ssDNA binding protein (gp2.5) (Figure 1). Gp5 forms a high affinity complex with trx (gp5/trx) to increase the processivity of nucleotide polymerization (2,3). The helicase domain of the hexameric gp4 encircles the lagging-strand DNA and unwinds

1

Copyright 2015 by The American Society for Biochemistry and Molecular Biology, Inc.

dsDNA in a reaction fueled by the hydrolysis of dTTP (4,5). The primase domain in the N-terminal half of gp4 recognizes specific sequences on the lagging-strand template at which it catalyzes the synthesis of tetraribonucleotides that are used as primer for the initiation of the synthesis of Okazaki fragments. Primase maintains contact with the priming sequence during ongoing DNA synthesis, organizing a priming loop that keeps the primer in physical proximity to the replication complex (6). Because of the polarity of DNA synthesis, the leading-strand is synthesized continuously whereas the lagging-strand is synthesized discontinuously, in the form of Okazaki fragments. Interactions between the lagging-strand DNA polymerase and the helicase domain of gp4 are important in the formation and maintenance of a replication loop (Figure 1). Gp2.5 coats the ssDNA to remove secondary structures and also physically interacts with both gp4 and gp5, interactions essential for coordination of leading- and lagging-strand DNA synthesis (7).

The formation of the replication loops not only allows both leading- and lagging-strand DNA polymerases to synthesize DNA in the same direction and but also facilitates recycling of the lagging-strand DNA polymerase to the newly synthesized RNA primers at the fork. The lagging-strand polymerase must transiently release from DNA upon termination of each Okazaki fragment. Biochemical studies have revealed a number of molecular scenarios that help to explain how the formation and release of replication loops are regulated (8-11). The visualization of replication intermediates in the T4 and T7 bacteriophage replication systems by electron microscopy demonstrates the existence of replication loops and allows for determination of length distributions (11,12).

The replication loop consists of single- and double-stranded regions of DNA and a complex of helicase, DNA polymerase and DNA (Figure 1). After the synthesis of an Okazaki fragment, the replication loop releases to enable the synthesis of the next Okazaki fragment (1). The release of the loop requires the disruption of this junction by dissociation of DNA polymerase from either helicase or DNA. Since the lagging-strand DNA synthesis is discontinuous, the lagging DNA polymerase must cycle on and off DNA for

synthesis of each new Okazaki fragment, an event accompanied by loop release and eventually reformation. Two mechanisms have been proposed to explain the cycling of DNA polymerase. In the collision model, the lagging-strand polymerase collides with the 5' terminus of an earlier completed fragment and releases from DNA (13). In the signaling model, the lagging-strand polymerase releases from DNA due to one or more distinct macromolecular interaction events involved in repetitive lagging-strand cycles (10). For example, the association of the DnaG primase with the primosome triggers the lagging polymerase recycling in the E. coli replication system (9). The signaling model does not require the collision of DNA polymerase with the previous Okazaki fragment.

In the T7 replication system, loop release was found to occur via both mechanisms using single-molecular technique (14-16). In addition to loop release via the collision mechanism, the synthesis of pAC also triggers loop release before the nascent Okazaki fragment is completed, leaving an ssDNA gap in the lagging DNA strand (14-16). Notably, DNA bound to the polymerase has different structures in both mechanisms. In the collision model, completion of an Okazaki fragment by DNA polymerase results in DNA containing a nick (13,17). On the other hand, in the signaling model, a ssDNA template for lagging-strand DNA synthesis contains a single-stranded region. We hypothesize that the different intermediate DNA structures formed during DNA synthesis may affect the stability of the loop junction and contribute to loop release via different pathways.

In this study, duplex DNA containing a nick or short and long ssDNA regions (Table 1) were used to examine the binding affinity and dissociation among helicase, DNA polymerase and DNA at a replication fork. Our data show that the binding affinities among helicase, DNA polymerase and DNA are weaker at a nick than are those at single-stranded regions. We also find that the interaction between helicase and DNA polymerase is stronger than that between DNA and DNA polymerase. Taken together, we propose the replication loop is more likely to release via the collision pathway by dissociation of polymerase from the DNA; the polymerase is thus retained at the replication fork.

# EXPERIMENTAL PROCEDURES

**Materials**—Oligonucleotides were purchased from Integrated DNA Technologies. DNA substrates containing a nick or single-stranded regions were prepared by annealing the designated strands (Table 1). The dideoxy primer strands contained a dideoxy-nucleotide at the 3'-end to enable the formation of a stable DNA polymerase-DNA complex in the presence of the next incoming nucleoside 5'-triphosphate (18). Bacteriophage T7 gene 5 DNA polymerase lacking exonuclease activity (gp5exo⁻), gene 4 helicase-primase (gp4) and *E. coli* thioredoxin (trx) were overproduced and purified as described previously (18). For simplicity, we denote gp5 exo⁻ as gene 5 DNA polymerase (gp5) throughout the text.

**Binding of DNA polymerase (gp5/trx) to DNA**—Surface plasmon resonance (SPR) analysis of the binding of gp5/trx to three kinds of DNA was performed using a Biacore 3000 instrument (Uppsala, Sweden) as described previously (18). Three annealed DNA substrates (300 RU) were immobilized to a streptavidin (SA) chip through a biotin at 5' terminus of template strand. In a control flow cell, biotin was used instead of the biotinylated DNA to compensate for background. Binding studies were carried out in buffer A (20 mM HEPES, pH 7.5, 2.5 mM DTT, 200 mM potassium glutamate, 1% glycerol (w/v)) at a flow rate of 10 μl/min at room temperature. A varying concentration of gp5/trx (20-fold molar excess of trx, 0.005–0.6 μM of gp5) was flowed over the chip to study the binding of the polymerase to DNA. The binding signal was fitted to Equation (1) using a steady-state model provided by BIAEVAL 3.0.2 computational software (Biacore). The dissociation constants \( K_d \) were calculated using the steady-state average RU (18).

\[
Y = B \times RU_{max} / (B + K_d)
\]

where \( Y \) is the response signal corresponding to the binding; \( B \) is concentration of protein, μM; \( RU_{max} \) is the maximal binding amount, RU; \( K_d \) is the dissociation constant, μM. All experiments were carried out three times and standard errors were derived using Prism software (The below is the same).

In the presence of 5 mM MgCl₂ and 0.05 mM ddATP, the active site of DNA polymerase is preferentially located at the 3'-end of primer where ddATP is incorporated opposite thymidine

in the template strand (18,19). The binding affinities of the polymerase to DNA were determined by the same methods described above (18). The chip surface was regenerated by injection of 150 μl of 1 M NaCl at a flow rate of 100 μl/min.

**Dissociation Rates of DNA Polymerase from DNA Determined by Trap Assays**—The dissociation rates of DNA polymerase from DNA were determined using a RQF-3 KinTek quench flow apparatus (20,21). One sample syringe containing a pre-incubated solution of 200 nM gp5/trx and 500 nM unlabeled DNA containing a nick or single-stranded region in the absence or presence of 0.5 mM ddATP and 20 mM MgCl₂. This sample was rapidly mixed with 450 nM trap primer/template in the other sample syringe at time intervals ranging from 0.1 to 10 s. The trap DNA consists of a 5' ³²P-radiolabeled 24-mer primer annealed to a 36-mer template. After the sample was mixed at various times, polymerization was initiated by mixing with 500 μM dCTP and 20 mM MgCl₂ in the central drive syringe and incubated for 0.25 s. The ³²P-labeled 24-mer primer was extended to 25-mer after incorporation of dCMP. After the sample was expelled from the rapid-quench apparatus, the reaction was rapidly terminated by mixing with 500 μl of 0.3 M EDTA in 50% formamide (v/v). The amount of ³²P-labeled extended product (25-mer) was quantified by gel analysis and plotted against time. The graph was fit to a single-exponential equation in Prism software using Equation 2.

\[
Y = E_f + E_0(1 - e^{-kt})
\]

where \( E_f \) = free enzyme concentration, nM; \( E_0 \) = concentration of DNA polymerase bound to DNA, nM; \( k \) = dissociation rate of polymerase from DNA, s⁻¹; and \( t \) is time, sec (20).

**Binding Affinity of Helicase to DNA Polymerase-DNA Complex**—The binding of helicase to DNA polymerase-DNA complex was investigated by SPR as previously described (18). DNA substrates (300 RU) were coupled to a streptavidin (SA) chip. Binding studies were carried out in buffer A containing 5 mM MgCl₂ and 0.05 mM ddATP at a flow rate of 10 μl/min at room temperature. A mixture of 0.3 μM gp5 and 6 μM trx (20-fold molar excess) in buffer A was injected into the flow cell. A varying concentration of wild-type gp4 or gp4-ΔC17

lacking the C-terminal 17 amino acid residues
(0.01-0.8 μM in monomeric concentration) was
injected in buffer A containing 0.1 mM ATP and
0.1 mM ddATP. In a control flow cell, the same
DNA was immobilized without the polymerase to
compensate background. The signal was fitted to
Eq. 1 using the steady-state model provided by
BIAEVAL 3.0.2 computational software (Biacore)
(18). The dissociation constants \( K_d \) were
calculated using the steady-state average response
unit (RU). The chip surface was regenerated as
described previously.

RESULTS

Binding Affinity of DNA Polymerase to
Replication DNA Intermediates—During DNA
synthesis the T7 DNA replisome undergoes
changes, not the least of which is the formation,
growth, and release of the lagging strand
replication loop. To analyze interactions among
the components of the replisome, we first
examined binding of DNA polymerase to DNA
containing structures that might arise
as intermediates during DNA replication.
The binding affinity of DNA polymerase to
DNA was studied using surface plasmon
resonance (SPR) (18). In the SPR experiments,
three kinds of DNA (300 response units, RU) were
immobilized to a SA chip. DNA containing a nick
or single-stranded region was prepared by
annealing each ssDNA with specific
oligonucleotides (Table 1). To compare the
binding affinities of DNA polymerase to the
various DNAs, varying concentrations of
polymerase were flowed over the chip and the
response signals were measured. Polymerase
lacking the proof-reading exonuclease activity was
used in these experiments to prevent degradation
of the DNA. The binding affinities (\( K_d \)) were
obtained by fitting the response signal and protein
concentration to Eq. 1 using the steady-state model
(18).
The absence of ddATP and Mg²⁺ in buffer A
led to a random binding of DNA polymerase to
DNA to form a binary complex (19). DNA
polymerase binding to DNA containing a single-
stranded region exhibited higher binding affinities
(\( K_d \) of 0.29 μM and 0.37 μM) than that with a nick
(0.61 μM) (Figure 2), in agreement with previous
results that DNA polymerase can bind non-

specifically to dsDNA and diffuse along the
duplex (22). In the presence of ddATP and Mg²⁺,
DNA polymerase, DNA and ddATP form a
ternary complex, in which the polymerase was
preferentially positioned and locked at the 3'-end
of primer strand (18, 19). The dissociation
constants (\( K_d \)) of the polymerase from DNA
containing single-stranded region were about 0.26
μM, lower than that from a nicked DNA (\( K_d \) of
0.44 μM) (Figure 3), indicating the polymerase
bound better to DNA containing single-stranded
region than to a nick.
Comparison of \( K_d \) values shows DNA
polymerase is more prone to dissociation from
nicked DNA rather than from DNA containing
single-stranded region in both binary and ternary
complexes. Addition of ddATP and Mg²⁺ results
in formation of the ternary complex, enhancing the
binding of DNA polymerase to DNA. The
enhanced binding may arise as a result of the
polymerase being now located at the end of primer,
thus leading to a stronger binding mode than that
of the random binding in the binary complex.
DNA structures obviously affect the binding
affinity of DNA polymerase, giving a higher
binding affinity to DNA containing a single-
stranded region than to the nicked DNA in both
binary and ternary complexes.
Dissociation Rates of Polymerase from DNA
Determined by Trap Assays—The effects of DNA
structures on DNA polymerase binding was
further examined by measuring the dissociation
rates of DNA polymerase in the binary or ternary
complexes by DNA trapping experiment using a
RQF-3 KinTek quench flow apparatus (Figure 4A).
In the absence of ddATP and Mg²⁺, DNA
polymerase binds to DNA to form a binary
complex. Once the polymerase dissociates from
the binary complex, it will bind to the trap DNA
composed of a radiolabeled 24-mer primer and 36-
mer template, and then extend the 24-mer primer
to a 25-mer by incorporating dNMP. Measuring
the amount of extended 25-mer product
determines the dissociation rate of polymerase
from DNA. The dissociation rates determined in
this assay indicate that dissociation from nicked
DNA (1.0 s⁻¹) is faster than that from DNA
containing a short or long single-stranded region
(0.51 s⁻¹ or 0.42 s⁻¹, respectively) (Figure 4). In
the ternary complex formed in the presence of
ddATP and Mg²⁺, dissociation of the polymerase

from all DNA structures was slower than from the binary complex in the absence of ddATP and Mg²⁺ (Figure 5). Dissociation of the polymerase from the ternary complex occurred at a rate of 0.54 s⁻¹ for a nicked DNA and 0.069 or 0.064 s⁻¹ for DNA containing a short or long single-stranded region, respectively.

The dissociation rates of DNA polymerase were slower from DNA containing a single-stranded region than that from nicked DNA in both binary and ternary complexes, suggesting that the binding of the polymerase to DNA is favorable in the presence of a single-stranded region compared with a nick. In the ternary complex, the dissociation rate of the polymerase from DNA containing a single-stranded region is about 8-fold lower than that from a nick. The decreased rate could arise from an inability of the nicked DNA to form a stable ternary complex with polymerase and ddATP, thus leading to a fast dissociation of the polymerase from the nicked DNA. Dissociation of DNA polymerase was significantly slower in the ternary complex than from the binary complex, indicating that the presence of ddATP and Mg²⁺ stabilizes the complex of DNA polymerase and DNA.

Binding Affinity of Helicase to DNA Polymerase-DNA Complex—Helicase (gp4) interacts with DNA polymerase (gp5) that is bound to DNA (18). Whether DNA structures affect the binding affinity of helicase to polymerase bound to DNA is unknown. Therefore, the binding affinity between helicase and DNA polymerase to which DNA containing a nick or a short or long single-stranded region is bound was determined. In the SPR experiments, each of three DNA was immobilized onto the chip; DNA polymerase was flowed over the DNA in the presence of ddATP and Mg²⁺ to form a stable complex.

The C-terminal tail of helicase plays an important role in the interaction with DNA polymerase (18, 23). In order to determine the contribution of the C-terminal tail of helicase to the interaction between helicase and DNA-bound polymerase, both helicase (gp4) and helicase lacking the C-terminal tail (gp4-ΔC17) were examined. Varied concentrations of gp4-ΔC17 were flowed over the chip and the interactions between gp4-ΔC17 and polymerase were detected. In a control experiment where polymerase was omitted, no detectable binding of gp4-ΔC17 to the immobilized DNA was observed (18). The binding affinities of gp4-ΔC17 to DNA polymerase bound to DNA containing a short or long single-stranded region were similar each other (Kd of 0.28 μM) but was 2-fold higher than the binding to a nick (Kd of 0.44 μM) (Figure 6).

Similarly, the binding affinities of helicase to DNA polymerase bound to various DNAs were also investigated. The DNA structures (nick, short or long single-stranded region) obviously affected the binding affinity between helicase and DNA polymerase. The binding affinities between helicase and polymerase bound to DNA containing either short or long single-stranded region were similar (Kd of 0.11 μM) but was 2-fold tighter than that at a nick (Kd of 0.26 μM) (Figure 7). The higher affinity could arise from more exposed surface for the interaction with helicase when polymerase is bound to DNA containing a single-stranded region than that at a nick.

Gp4 or gp4-ΔC17 forms a complex with DNA polymerase bound to DNA containing a single-stranded region better than that at a nick. Comparison of gp4 with gp4-ΔC17 indicates that the presence of the C-terminal tail of helicase significantly increases the binding affinity between helicase and DNA polymerase that was bound to any of the three DNAs, in agreement with previous results (18). Compared with the binding affinity between polymerase and DNA (Table 2), helicase binding to polymerase bound to DNA is much tighter, suggesting that the dissociation of DNA polymerase from DNA is more favorable than the dissociation of DNA polymerase from helicase.

DISCUSSION

Formation of a replication loop during lagging-strand DNA synthesis allows both leading- and lagging-strand DNA synthesis to occur within a single complex of proteins, the replisome. Consequently, the synthesis of both strands proceeds in the same overall direction and the lagging-strand polymerase can be recycled to a newly synthesized RNA primer. After synthesis of an Okazaki fragment, the replication loop is released for the synthesis of a new Okazaki fragment, an event that triggers the reformation of the replication loop. The release of the replication loop requires the dissolution of the junction that

tethers the lagging strand to the replisome. This junction is composed of helicase, DNA polymerase and DNA. Dissociation of the polymerase from DNA or dissociation of polymerase from helicase can disrupt the junction.

Three interactions between helicase and DNA polymerase are known to exist during leading-strand DNA synthesis in the replisome of the replication system of bacteriophage T7 (18). The C-terminal tail of the helicase interacts with a basic patch on the polymerase for its loading onto DNA. A second interaction involves the acidic C-terminal tail of the helicase and a basic patch on the thioredoxin binding domain of the polymerase to enable recycling of the polymerase during DNA synthesis. DNA polymerase that transiently dissociates during leading strand DNA synthesis is captured by one of the acidic C-terminal tails of the hexameric helicase and can then be recycled back to the leading strand to continue DNA synthesis (18,23). After dissociation from DNA, the polymerase is still attached to helicase (18,23). A third interaction that does not involve the acidic C-terminal tail of the helicase occurs during leading strand synthesis.

On the lagging-strand DNA, protein interactions determine the stability of the replication loop. Two mechanisms have been proposed to explain how the replication loop is released (14,15). In the collision mechanism, the encounter of lagging-strand DNA polymerase with the 5' terminus of the previously synthesized Okazaki fragment triggers loop release. In this model the lagging strand DNA polymerase dissociates from the nick that is formed upon closure of the lagging strand gap. In the signaling model, the condensation of the first two nucleotides of the primer to yield pppAC triggers loop release before the nascent Okazaki fragment is completed. This mechanism results in a ssDNA gap in the lagging-strand DNA to which DNA polymerase binds. The use of both mechanisms provides an elegant solution to cope with the stochastic nature of the primase activity (14). The appearance of either nicks or gaps in the DNA in these two models led us to examine the affinities of the replication proteins with similar structures. Differences in affinity of the proteins for these structures could help to identify the trigger that leads to loop release.

Affinities represented by dissociation constants ($K_d$) of DNA polymerase from DNA, as well as helicase from a polymerase-DNA complex, as determined by SPR, are lower for nicked DNA than those for DNA containing a single-stranded region of DNA. The trap assays also show that the dissociation of polymerase from nicked DNA (0.54 s⁻¹) is considerably faster than that from DNA containing a single-stranded region (0.064-0.069 s⁻¹). All the data show that the polymerase-DNA complex and helicase-polymerase-DNA complex are less stable at nicked DNA than at DNA containing a single-stranded region, suggesting that the replication loop is more likely to release at a nick after polymerase encounters the previously synthesized Okazaki fragment via the collision mode (Figure 8) (14).

We also compared the binding affinity of DNA polymerase-helicase with that of DNA polymerase-DNA. The dissociation constant ($K_d$) between helicase and DNA polymerase was 0.089 μM (18). Gp4E, which has the helicase domain but lacks the primase domain, binds to DNA polymerase with a dissociation constant of 0.09 μM (19), indicating that the helicase domain alone contributes to the interaction with the polymerase. The dissociation constants ($K_d$) between helicase and DNA-bound polymerase were 0.11-0.26 μM (Table 2). Thus, the binding of DNA to the polymerase weakens its interaction with the helicase. In addition, binding of the polymerase to the various DNA structures (0.25-0.44 μM) was weaker than that to the helicase (Table 2). Therefore, the loop is most likely released by dissociation of the polymerase from DNA rather than dissociation from the helicase. During loop release, the lagging strand DNA will release from the replication fork while the lagging-strand DNA polymerase, still attached to helicase at the replication fork, can initiate the synthesis of the next Okazaki fragment by extending a new primer.

The retention of DNA polymerase within the replisome has been observed in other replication systems. In the T4 replication system, after loop release the lagging-strand polymerase remains associated with the replisome (10). In the E. coli replication system, pol III core and the τ subunit of the clamp loader are tightly bound (24). The τ subunit interacts with the pol III core and the helicase even after release of the replication loop (25). The β sliding clamp which is reserved on an

Okazaki fragment attracts a new Pol III core in preference over other polymerases from solution after Pol III dissociates (25, 26).

**Acknowledgments—** This work was supported by the Natural Science Foundation of China Grant 31370793 and 81422041. We also thank Steven Moskowitz (Advanced Medical Graphics) for illustrations.

**Conflict of interest—** The authors declare that they have no conflicts of interest with the contents of this article.

**Author contributions—** HZ and YT designed the study and collect data. HZ, SJL, ZW and JC analyzed the data. HZ, SJL and CCR wrote the paper. HZ, SJL and CCR approved the final version of the manuscript.

## REFERENCES

1. Hamdan, S.M., and Richardson, C.C. (2009) Motors, switches, and contacts in the replisome. *Ann. Rev. Biochem.* **78**, 205-243.
2. Tabor, S., Huber, H.E., and Richardson, C.C. (1987) *Escherichia coli* thioredoxin confers processivity on the DNA-polymerase-activity of the gene-5 protein of bacteriophage-T7. *J. Biol. Chem.* **262**, 16212-16223.
3. Wuite, G.J.L., Smith, S.B., Young, M., Keller, D., and Bustamante, C. (2000) Single-molecule studies of the effect of template tension on T7 DNA polymerase activity. *Nature* **404**, 103-106.
4. Ahnert, P., and Patel, S.S. (1997) Asymmetric interactions of hexameric bacteriophage T7 DNA helicase with the 5'- and 3'-tails of the forked DNA substrate. *J. Biol. Chem.* **272**, 32267-32273.
5. Satapathy, A.K., Kochaniak, A.B., Mukherjee, S., Crampton, D.J., van Oijen, A., and Richardson, C.C. (2010) Residues in the central beta-hairpin of the DNA helicase of bacteriophage T7 are important in DNA unwinding. *Proc. Natl. Acad. Sci. U. S. A.* **107**, 6782-6787.
6. Pandey, M., Syed, S., Donmez, I., Patel, G., Ha, T., and Patel, S.S. (2009) Coordinating DNA replication by means of priming loop and differential synthesis rate. *Nature* **462**, 940-943.
7. Marintcheva, B., Hamdan, S.M., Lee, S.J., and Richardson, C.C. (2006) Essential residues in the C terminus of the bacteriophage T7 gene 2.5 single-stranded DNA-binding protein. *J. Biol. Chem.* **281**, 25831-25840.
8. Lopez de Saro, F.J.G., R.E. O'Donnell, M. (2003) A peptide switch regulates DNA polymerase processivity. *Proc. Natl. Acad. Sci. U. S. A.* **100**, 14689-14694.
9. Li, X., and Marians, K.J. (2000) Two Distinct Triggers for Cycling of the Lagging Strand Polymerase at the Replication Fork. *J. Biol. Chem.* **275**, 34757–34765.
10. Yang, J.S., Nelson, S.W., and Benkovic, S.J. (2006) The control mechanism for lagging strand polymerase recycling during bacteriophage T4 DNA replication. *Mol. Cell* **21**, 153-164.
11. Lee, J., Chastain, P.D., Griffith, J.D., and Richardson, C.C. (2002) Lagging strand synthesis in coordinated DNA synthesis by bacteriophage T7 replication proteins. *J. Mol. Biol.* **316**, 19-34.
12. Park, K., Debyser, Z., Tabor, S., Richardson, C.C., and Griffith, J.D. (1998) Formation of a DNA loop at the replication fork generated by bacteriophage T7 replication proteins. *J. Biol. Chem.* **273**, 5260-5270.
13. Georgescu, R.E., Kurth, I., Yao, N.Y., Stewart, J., Yurieva, O., and O'Donnell, M. (2009) Mechanism of polymerase collision release from sliding clamps on the lagging strand. *EMBO J.* **28**, 2981-2991.
14. Hamdan, S.M., Loparo, J.J., Takahashi, M., Richardson, C.C., and van Oijen, A.M. (2009) Dynamics of DNA replication loops reveal temporal control of lagging-strand synthesis. *Nature* **457**, 336-339.
15. Stratmann, S.A., and van Oijen, A.M. (2014) DNA replication at the single-molecule level. *Chem. Soc. Rev.* **43**, 1201-1220.
16. Geertsema, H.J., and van Oijen, A.M. (2013) A single-molecule view of DNA replication: the dynamic nature of multi-protein complexes revealed. *Curr. Opin. Struc. Biol.* **23**, 788-793.

Binding affinities in T7 DNA replisome

17. Benkovic, S.J., Valentine, A.M., and Salinas, F. (2001) Replisome-mediated DNA replication. Ann. Rev. Biochem. 70, 181-208.

18. Zhang, H., Lee, S.J., Zhu, B., Tran, N.Q., Tabor, S., and Richardson, C.C. (2011) Helicase-DNA polymerase interaction is critical to initiate leading-strand DNA synthesis. Proc. Natl. Acad. Sci. U. S. A. 108, 9372-9377.

19. Hamdan, S.M., Marintcheva, B., Cook, T., Lee, S.J., Tabor, S., and Richardson, C.C. (2005) A unique loop in T7 DNA polymerase mediates the binding of helicase-primase, DNA binding protein, and processivity factor. Proc. Natl. Acad. Sci. U. S. A. 102, 5096-5101.

20. Woodside, A.M., and Guengerich, F.P. (2002) Misincorporation and stalling at O-6-methylguanine and O-6-benzylguanine: Evidence for inactive polymerase complexes. Biochemistry 41, 1039-1050.

21. Patel, S.S., Wong, I., and Johnson, K.A. (1991) Pre-Steady-State Kinetic-Analysis of Processive DNA-Replication Including Complete Characterization of an Exonuclease-Deficient Mutant. Biochemistry 30, 511-525.

22. Etson, C.M., Hamdan, S.M., Richardson, C.C., and van Oijen, A.M. (2010) Thioredoxin suppresses microscopic hopping of T7 DNA polymerase on duplex DNA. Proc. Natl. Acad. Sci. U. S. A. 107, 1900-1905.

23. Hamdan, S.M., Johnson, D.E., Tanner, N.A., Lee, J.B., Qimron, U., Tabor, S., van Oijen, A.M., and Richardson, C.C. (2007) Dynamic DNA helicase-DNA polymerase interactions assure processive replication fork movement. Mol. Cell 27, 539-549.

24. Downey, C.D., and McHenry, C.S. (2010) Chaperoning of a replicative polymerase onto a newly assembled DNA-bound sliding clamp by the clamp loader. Mol. Cell 37, 481-491.

25. Kurth, I., and O'Donnell, M. (2013) New insights into replisome fluidity during chromosome replication. Trends in Biochem. Sci. 38, 195-203.

26. Indiani, C., McInerney, P., Georgescu, R., Goodman, M.F., and O'Donnell, M. (2005) A sliding-clamp toolbelt binds high-and low-fidelity DNA polymerases simultaneously. Mol. Cell 19, 805-815.


FOOTNOTES

This work was supported by the Natural Science Foundation of China Grant 31370793 and 81422041 to HZ.

1 Institute of Toxicology, College of Preventive Medicine, Third Military Medical University, Chongqing 400038, PR China. Tel.: +86 (023) 6875 2292; E-mail: huidongzhang@tmmu.edu.cn.

2 Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, 240 Longwood Avenue, C2-219, Boston, MA 02115. Tel.: (617) 432-1864; Fax: (617) 432-3362; E-mail: ccr@hms.harvard.edu.

3 The abbreviations used are: bp, base pair; nt, nucleotide; kb, kilo basepair.


FIGURE LEGENDS

FIGURE 1. Model of the replisome of bacteriophage T7. The T7 replisome consists of DNA polymerase (gp5), the processivity factor *E. coli* thioredoxin (trx), the hexameric helicase-primase (gp4) and the ssDNA-binding protein (gp2.5). The helicase unwinds dsDNA to generate two ssDNA templates for both leading- and lagging-strand DNA synthesis. The primase synthesizes oligoribonucleotides as primers for the lagging-strand DNA polymerase to initiate the synthesis of Okazaki fragments. Gp2.5 coats the lagging-strand DNA template formed by the interaction among helicase, polymerase and DNA. The replication loop consists of ssDNA/dsDNA and the interaction junction of helicase-DNA polymerase-
DNA. The release of the loop is dependent on the dissociation of the loop junction by separation of the DNA from polymerase ((2), inset) or the polymerase from helicase ((1), inset).

FIGURE 2. Binding of DNA polymerase to various DNAs in the absence of Mg²⁺ and ddATP. A, Scheme for measuring the interaction of gp5/trx with DNA containing a nick or a short or long single-stranded region that was prepared and immobilized on to the SA chip (300RU). Gp5/trx randomly binds to DNA without Mg²⁺ and ddATP to form a binary complex. B, Sensograms for the binding of increasing concentrations of gp5/trx (1:20, molar ratio, 0.005-0.6 μM of gp5) to the three immobilized DNA in buffer A (see Experimental Procedure). C, The binding affinities of gp5/trx to DNA were determined using the steady-state average response at each concentration of gp5/trx shown in (B). The solid lines represent the theoretical curve calculated from the steady-state fit model (Biacore). Representative data from multiple experiments are shown.

FIGURE 3. Binding of DNA polymerase to various DNAs in the presence of Mg²⁺ and ddATP. A, Scheme for measuring the interaction of gp5/trx with DNA containing a nick, short or long single-stranded region that was prepared and immobilized on to the SA chip (300RU). Gp5/trx is locked at the 3'-end of primer in the presence of Mg²⁺ and ddATP to form a ternary complex. B, Sensograms for the binding of increasing concentrations of gp5/trx (1:20, molar ratio, 0.005-0.4 μM of gp5) to the three immobilized DNA in the buffer A (see Experimental Procedure). C, The binding affinities of gp5/trx to DNA were determined as described in Figure 2C. Representative data from multiple experiments are shown.

FIGURE 4. Dissociation rates ($k_{\text{off}}$) of DNA polymerase from DNA containing a nick or a short or long single-stranded region in the absence of ddATP and Mg²⁺. A, Scheme for measuring the dissociation of polymerase from DNA containing a nick or a short or long single-stranded region. Dissociated polymerase binds to 24-mer/36-mer, and catalyzes the extension of a 24-mer to a 25-mer product. B, Gp5/trx (200 nM) was pre-incubated with 500 nM unlabeled DNA containing a nick or a short or long single-stranded region in the absence of ddATP and Mg²⁺ to form a binary complex. This solution was mixed with 450 nM $^{32}$P labeled 24-mer/36-mer at time intervals ranging from 0.1 to 10 s. Then polymerization of nucleotides was initiated with 500 μM dCTP and 20 mM Mg²⁺ for a constant time of 0.25 s, in which dCMP was incorporated and $^{32}$P-labeled primer was extended from a 24-mer to a 25-mer. The dissociation rates of polymerase from DNA were obtained by fitting the 25-mer product and time according to Eq. (2). The solid lines represent the theoretical curves calculated from the steady-state fit model. Representative data from multiple experiments are shown.

FIGURE 5. Dissociation rates ($k_{\text{off}}$) of DNA polymerase from DNA containing a nick or a short or long single-stranded region in the presence of ddATP and Mg²⁺. A and B, the same as described in Figure 4 except for 200 nM gp5/trx and 500 nM unlabeled DNA was pre-incubated in the presence of 0.5 mM ddATP and 20 mM Mg²⁺ to form a ternary complex. Representative data from multiple experiments are shown.

FIGURE 6. Binding affinity of helicase lacking the C-terminal tail (gp4-ΔC17) with DNA polymerase bound to DNA containing a nick or a short or long single-stranded region. A, Scheme for measuring the interaction of gp4-ΔC17 with DNA polymerase binding to DNA containing a nick or short or a long single-stranded region in the presence of Mg²⁺ and ddATP. Three kinds of DNA were immobilized on the SA chip (300RU). A mixture of 0.3 μM gp5 and 6 μM trx (1:20, molar ratio) was flowed over the chip to form a stable gp5/trx-DNA-ddATP ternary complex. Gp4-ΔC17 was injected into the flow cell. B, Increasing concentrations of gp4-ΔC17 monomer (0.01-0.8 μM) in the buffer containing 0.1 mM ATP and 0.1 mM ddATP was injected into the flow cell, and the response binding signal was recorded. The arrows mean the beginning of injecting proteins into chip cell. C, The binding affinities of gp4-ΔC17 to gp5/trx were determined using the steady-state average response at each concentration of gp4-ΔC17. The

9
solid lines represent the theoretical curves calculated from the steady-state fit model. Representative data from multiple experiments are shown.

FIGURE 7. Binding affinity of helicase (gp4) with DNA polymerase bound to DNA containing a nick or a short or long single-stranded region. A-C, the same as described in Figure 6 except for the full-length helicase was used instead of the helicase lacking the C-terminal tail. Representative data from multiple experiments are shown.

FIGURE 8. Model for the release of the replication loop. Upon finishing the synthesis of an Okazaki fragment, the loop is released by dissociation of DNA polymerase from DNA rather than from the helicase. The lagging-strand DNA polymerase is retained at the replication fork for the synthesis of the next Okazaki fragment. The dissociation of DNA polymerase from a nick in DNA is easier than that from a single-stranded DNA gap, suggesting loop release is triggered upon DNA synthesis reaching the 5'-terminus of the previously synthesized Okazaki fragment (collision model).

TABLE 1. DNA containing a nick or short and long single-stranded regions used in this study

| Substrate           | Name    | Structure                                                                 |
|---------------------|---------|--------------------------------------------------------------------------|
| Nick                | S1+S2   | 5'—ACG<sub>dd</sub> AAAG——3'                                              |
|                     | T       | 3'—TGC—TTTC——TT-Biotin-5'                                                 |
| short single-       | S1+S3   | 5'—ACG<sub>dd</sub> TAAT——3'                                              |
| stranded region     | T       | 3'—TGC—TTTCTGCATTA——TT-Biotin-5'                                          |
| long single-        | S1      | 5'—ACG<sub>dd</sub>                                                        |
| stranded region     | T       | 3'—TGC—TTTCTGCATTA——TT-Biotin-5'                                          |
| S1                  |         | 5'-CTAGTTACAGAGTTATGGTGACG<sub>dd</sub>-3'                                 |
| S2                  |         | 5'-AAAGACGTAA TCATGGTCATAGCTG TTTCCCT-3'                                   |
| T                   |         | 5'-Biotin-TTAGG AAACAGCTATGACCA TGATTACGTCTTTCG TCACCATAACTCTGT AACTAG-3'    |
| S3                  |         | 5'-TAATCATGGTCATAG CTGTTTCCCT-3'                                           |
| Trap primer         |         | 5'-<sup>32</sup>P-GCCTCGAGCCAGCCGCCGAGACGCAG-3'                            |
| Trap template       |         | 3'-CGGAGCTCGGGTCGGCGTCTGCGTCTGCGTCTGCGGGCT-5'                              |

TABLE 2. The dissociation constants (K<sub>d</sub>) among helicase, DNA polymerase and DNA

| Dissociation constants, μM | Short single-stranded region | Long single-stranded region | No DNA |

Binding affinities in T7 DNA replisome

| gp 5/trx-DNA | 0.44 | 0.27 | 0.25 |
|---------------|------|------|------|
| gp 4-gp 5/trx-DNA | 0.26 | 0.11 | 0.11 |
| gp 4-gp 5/trx |  |  | 0.089* |
| gp4E-gp 5/trx |  |  | 0.09** |

*From a previous work (Ref. 18)  
**From a previous work (Ref. 19)

DNA polymerase  
Thioredoxin  

Leading strand  

helicase  
primase { gp4  
gp2.5  

5' primer  

Lagging strand  

Loop junction  

(1) gp 4-(gp 5/trx-DNA)  
(2) gp 5/trx-DNA  

ssDNA binding protein  

FIGURE 1.

Binding affinities in T7 DNA replisome

A No ddATP and Mg²⁺

3' 5'
23 bp
nick
T
35bp
SA chip

3' 5'
Δ 7 nt
T
T

3' 5'
Δ 33 nt
T
T

B

Nick
Short ssDNA
Long ssDNA

Response (RU)
3000-
2000-
1000-
0
50
100
150
200
Time (sec)

C

Nick
Short ssDNA
Long ssDNA

Response (RU)
3000-
2000-
1000-
0
0.2
0.4
0.6
0.8
K_D = 0.61 ± 0.21 μM
0
0.2
0.4
0.6
0.8
K_D = 0.37 ± 0.07 μM
0
0.2
0.4
0.6
0.8
K_D = 0.29 ± 0.05 μM
gp5/trx (μM)

FIGURE 2.

Binding affinities in T7 DNA replisome

A With ddATP and Mg²⁺

B

| Response (RU) |
|---------------|
| Nick          |
| Short ssDNA   |
| Long ssDNA    |

C

| Response (RU) |
|---------------|
| Nick          |
| Short ssDNA   |
| Long ssDNA    |

FIGURE 3.

Binding affinities in T7 DNA replisome

A
No ddATP and Mg²⁺
24-mer/36-mer
25-mer/36-mer

Nick
Short ssDNA
Long ssDNA
Polymerase dissociation and binding
dCTP
C

B
Nick
Short ssDNA
Long ssDNA
25-mer product (nM)
k_off = 1.0 ± 0.02 s⁻¹
k_off = 0.51 ± 0.01 s⁻¹
k_off = 0.42 ± 0.01 s⁻¹
Time (sec)

FIGURE 4.

Binding affinities in T7 DNA replisome

A With ddATP and Mg²⁺ 24-mer/36-mer 25-mer/36-mer

Nick Short Long ssDNA ssDNA

Polymerase dissociation and binding

dCTP C

B

Nick Short ssDNA Long ssDNA

25-mer product (nM)

150-
100-
50-
0
3
6
9
12

k_off = 0.54 ± 0.05 s⁻¹

Time (sec)

80-
60-
40-
20-
0
3
6
9
12

k_off = 0.069 ± 0.016 s⁻¹

12

k_off = 0.064 ± 0.014 s⁻¹

FIGURE 5.

Binding affinities in T7 DNA replisome

A

B

| Response (RU) |
| --- |
| 4000 |
| 3000 |
| 2000 |
| 1000 |
| 0 |

gp5/trx  
gp4ΔC17  

Time (sec)

Nick  
Short ssDNA  
Long ssDNA  

C

| Response (RU) |
| --- |
| 1500 |
| 1000 |
| 500 |
| 0 |

$ K_{D} = 0.44 \pm 0.05 \, \mu M $

$ K_{D} = 0.27 \pm 0.05 \, \mu M $

$ K_{D} = 0.28 \pm 0.03 \, \mu M $

gp4ΔC17 (μM)

FIGURE 6.

Binding affinities in T7 DNA replisome

A

B
![Image Description](image1.png)

C
![Image Description](image2.png)

FIGURE 7.

Binding affinities in T7 DNA replisome

![Image 1](image1.png)
Collision
![Image 2](image2.png)

Less  
probably  
Loop release by  
release of DNA in  
signal mechanism  

More  
probably  
Loop release by  
release of DNA in  
collision mechanism  

![Image 3](image3.png)  
![Image 4](image4.png)

FIGURE 8.

DNA and Chromosomes:
Binding Affinities among DNA
Helicase-Primase, DNA Polymerase and
Replication Intermediates in the Replisome
of Bacteriophage T7

Huidong Zhang, Yong Tang, Seung-Joo Lee,
Zeliang Wei, Jia Cao and Charles C.
Richardson
*J. Biol. Chem.* published online November 30, 2015

---

Access the most updated version of this article at doi: [10.1074/jbc.M115.698233](http://dx.doi.org/10.1074/jbc.M115.698233)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- When this article is cited
- When a correction for this article is posted

Click [here](https://www.jbc.org/site/alerts) to choose from all of JBC's e-mail alerts

This article cites 0 references, 0 of which can be accessed free at [http://www.jbc.org/content/early/2015/11/30/jbc.M115.698233.full.html#ref-list-1](http://www.jbc.org/content/early/2015/11/30/jbc.M115.698233.full.html#ref-list-1)
