
JOURNAL OF BACTERIOLOGY, Aug. 1989, p. 4436–4441

0021-9193/89/084436-06$02.00/0

Copyright © 1989, American Society for Microbiology

# Genetic Regulation of the Tricarboxylate Transport Operon (tctI)  
of *Salmonella typhimurium*

KATHARINA A. WIDENHORN,† JACQUELINE M. SOMERS, AND WILLIAM W. KAY*

Department of Biochemistry and Microbiology, University of Victoria, Victoria, British Columbia, Canada V8W 2Y2

Received 6 February 1989/Accepted 10 May 1989

Tricarboxylates are transported into *Salmonella typhimurium* by a binding protein-dependent transport system known as TctI. Genetically, it comprises three structural genes, *tctCBA*, as well as a fourth gene of unknown function (*tctD*), which is transcribed divergently from *tctC* (K. A. Widenhorn, J. M. Somers, and W. W. Kay, J. Bacteriol. 170:3223–3227, 1988). Deletions in *tctD* strongly reduced expression of *tctC* or of *tctC-lacZ* transcriptional fusions; however, expression was restored when *tctD* was present in trans. Expression of *tctD-lacZ* transcriptional fusions was strongly repressed in the presence of D-glucose but could be alleviated by the addition of cyclic AMP. Furthermore, transcription of *tctD* was found not to be autogenously regulated. Thus, *tctD* is considered to be regulated by catabolite repression and encodes a transcriptional activator of *tctCBA* expression. From the DNA sequence of *tctD*, the predicted gene product was hydrophilic and shared distinct homologies with other globally regulated transcriptional activators such as OmpR and NtrC.

---

Preparation of Mu d1 lysates. A 10-ml LB culture of a mini-Mu d1-Mu cts lysogen (POI1681Tr [5] transformed with the appropriate plasmid) was incubated at 30°C until the cells reached the early log phase (*A*<sub>578</sub> = 0.3). Mu lytic growth was heat induced at 42°C for 25 min, and the culture was transferred to 37°C until lysis was complete (~2 h). Chloroform (0.1 ml) was added, and cell debris was removed by centrifugation (10 min, 4,500 × g). Mu lysates were kept on ice no longer than 24 h before use.

Transduction with Mu d1. An overnight LB culture of the Mu-immune strain MC4100(RP4-2-Tc::Mu) (22) was harvested and concentrated twofold in 10 mM MgSO<sub>4</sub> containing 5 mM CaCl<sub>2</sub>. Samples (5 to 500 μl) of a freshly prepared Mu lysate were each added to 0.1 ml of recipient cells. Cells and phages were mixed and incubated for 30 min at room temperature. Samples were spread on LB plates containing the appropriate antibiotics.

Biochemical techniques. β-Galactosidase assays based on o-nitrophenyl-β-D-galactopyranoside hydrolysis were performed by the method of Miller (16). β-Galactosidase activity of growing or stationary-phase cells was measured after the cells had been made permeable with chloroform and 0.1% sodium dodecyl sulfate (SDS). Enzyme activity is expressed in units also according to Miller (16).

Whole-cell lysates were subjected to SDS-polyacrylamide gel electrophoresis (PAGE) according to the system of Laemmli as modified by Ames (2). Gels were stained with Coomassie brilliant blue R-250. Electrophoretic transfer of proteins from SDS-polyacrylamide gels to nitrocellulose and development of the blot with affinity-purified polyclonal anti-TctC immunoglobulin G was done by the method of Towbin et al. (26), except that 0.05 M Tris hydrochloride (pH 7.4)-0.005 M EDTA-0.15 M sodium chloride containing 0.5% Tween 20 was used at all steps after the initial blocking.

DNA cloning and sequencing. The entire *tctI* operon was subcloned in both orientations as a 4.5-kilobase (kb) EcoRI-HindIII fragment derived from pKW101 (27) into M13 bacteriophages mWB2349 and mWB2341 (4). A sequential series of overlapping deletions was constructed by the method of Dale et al. (7). Deletions were sized by agarose gel electrophoresis. The DNA of both strands of *tctD* as well as flanking DNA was sequenced by the dideoxynucleotide

Bacterial shock-sensitive transport systems utilize periplasmic binding proteins as essential components of active transport complexes (3, 8). Such transport systems normally comprise a periplasmic binding protein and three inner membrane proteins, one of which has a nucleotide-binding site and is thought to be involved in energy coupling (3, 10). In *Salmonella typhimurium*, TctI is the only one of three resident tricarboxylate transport systems (13) which deploys a periplasmic binding protein known as C-protein (24). From fine structure genetic mapping (23) as well as cloning and expression studies (28), it was found that the *tctI* operon comprises a total of four genes. Two genes (*tctA*, *tctB*) encoding inner membrane proteins and another (*tctC*) encoding the tricarboxylate-specific periplasmic binding protein (C-protein) are transcribed from one DNA strand. A fourth gene (*tctD*) coding for a soluble or weakly membrane-associated protein is transcribed divergently from *tctCBA* (27). Therefore, TctI would at first appear to contain the same number of protein components as other binding protein-dependent bacterial transport systems. However, various mutants in *tctD* elicited properties of regulatory mutations (unpublished observations). We therefore undertook to elucidate the function of TctD and demonstrate here its role as a trans-acting transcriptional activator. The function of TctD, its regulation, and its relationship to other transcriptional regulators are discussed.

## MATERIALS AND METHODS

Bacterial strains, bacteriophages, and plasmids. The bacterial strains and plasmids used are listed in Table 1, and a diagram of constructed plasmids is shown in Fig. 1. These constructions are described in detail in the text. Strains harboring plasmids were grown in the presence of the appropriate antibiotics (micrograms per milliliter): ampicillin, 50; kanamycin, 50; chloramphenicol, 20. For routine growth, cells were grown in Luria-Bertani broth (LB), Davis minimal medium (DMM) (23), or minimal medium containing 0.4% peptone and 10 mM citrate (PCM).

* Corresponding author.
† Present address: Research Institute of Molecular Pathology, A-1030 Vienna, Austria.

VOL. 171, 1989 REGULATION OF TRICARBOXYLATE TRANSPORT IN S. TYPHIMURIUM 4437

TABLE 1. Bacterial strains, plasmids, and phages

| Strain, plasmid, or phage | Relevant genotype or description | Reference or source |
|---------------------------|----------------------------------|---------------------|
| POI1681Tr                 | F⁻ Mu dl-1681 ara::(Mu cts)3 Δ(proAB-argF-lacIPOZYA)XIII rpsL recA56 srl::Tn10 | B.A. Castilho (5) |
| LE392                     | supE supF hsdR galK trpR mutB lacY | CSH" |
| DL291                     | araD Δ(argF-lac) relA rpsL flbB deoC ptsF glpR Δ(glpT-glpA) gyrA recA | D. Ludtke |
| KS1070                    | hisF trpB metA rpsL xyl ΔtctI bio::Tn10 galE | J.M. Somers |
| pGP1-2                    | Kan' | S. Tabor (25) |
| RP4-2-Tc::Mu              | Kan'; conferring Mu immunity | R. Simon (22) |
| pKW123                    | Amp' tctC'D⁺ᵇ (1.4-kb insert in pT7-5) | This study |
| pKW131                    | Amp' tctA⁺B⁺C⁺D⁺ (4.0-kb insert in pT7-6; tctCBA under φ10 control) | This study |
| pKW132                    | Amp' tctA⁺B⁺C⁺ (3.5-kb insert in pT7-6; tctCBA under φ10 control) | This study |
| pKW134                    | Cam' tctC'D⁺ (1.4-kb insert in pACYC184) | This study |
| pKW135                    | Cam' tctC'D⁺ (1.4-kb insert in pACYC184) | This study |
| pKW136                    | Amp' tctC' (0.7-kb insert in pT7-6; tctC' under φ10 control) | This study |
| pKW137                    | Cam' tctC' (0.7-kb insert in pACYC184) | This study |
| Mu d11681                 | Mu cts62::IS121 d(trp'B⁺A'-W209-lac'ZYA) (Kan'), a defective lac transcription fusion phage | B.A. Castilho (5) |
| mWB2341                   | lac pro HindII inserted at nucleotide 5869 of M13. EcoRI site eliminated. | W.M. Barnes (4) |
| mWB2349                   | Polylinker HindIII, XbaI, BglII, PstI, Xhol added to MWB2341 | W.M. Barnes (4) |

"Cold Spring Harbor genetics course.

"The genotype tctA⁺B⁺C⁺D⁺ indicates that all genes are present on the plasmid and functional. A prime superscript indicates a gene with an incomplete C terminal.

chain-terminating method with ³⁵S-ATP as outlined in the standard Pharmacia protocol.

RESULTS

Effect of tctD on gene expression from tctC. In preliminary experiments, we observed the following discrepancies in TctC synthesis. When two different plasmids with Tn5 insertions in tctD but with otherwise intact tctCBA genes were introduced into LE392, both strains produced only traces of TctC as judged from Coomassie brilliant blue-stained or immunostained SDS gels. However, when the same two tctD::Tn5 plasmids were introduced into KS1070(pG1-2) and transcribed from the phage T7 promoter by T7 RNA polymerase, TctC was synthesized at the usual level (data not shown). This suggested that tctD affected tctC expression only when transcribed from its own promoter, whereas when independent of bacterial transcription, tctD had no effect on tctC expression.

To investigate these observations further, we constructed two pT7 recombinant plasmids. In pKW131, a tctA⁺B⁺C⁺D⁺ DNA fragment was inserted into pT7-6 in an orientation such

![Figure 1](#fig1)

FIG. 1. tctI operon and plasmids used in this study. Vectors are highlighted in bold print; in cases of T7 phage 10 promoter-directed transcription, only the genes expressed from the vector promoter are indicated.

that tctCBA was transcribed from the T7 promoter, provided that pGP1-2 was present and phage RNA polymerase was induced (Fig. 1). pKW132 is similar in the orientation of the insert; however, tctD had been previously deleted from the tctI fragment (Fig. 1). Whole-cell lysates were prepared from both strains [LE392(pKW131) and LE392(pKW132)] after growth in LB or PCM and examined for tctC expression. The presence of tctD (pKW131) greatly increased the amount of TctC synthesized as judged from Coomassie brilliant blue-stained (Fig. 2A) or immunostained (Fig. 2B) gels. When the

![Figure 2](#fig2)

FIG. 2. Effect of tctD on the expression of tctC. (A) A Coomassie brilliant blue-stained SDS-12% polyacrylamide gel is shown. LE392 (pKW131) (tctA⁺B⁺C⁺D⁺) and LE392(pKW132) (tctA⁺B⁺C⁺) were each grown to the stationary phase in either LB or PCM. Samples from each strain were harvested, and cells were suspended in SDS sample buffer and analyzed by SDS-PAGE. Lanes: 1, molecular weight standards, including TctC (29,000 M); 2, LE392(pKW131), LB grown; 3, LE392(pKW132), LB grown; 4, LE392(pKW131), PCM grown; 5, LE392(pKW132), PCM grown. (B) Western blot (immunoblot) of whole-cell lysates from LE392(pKW131) and LE392(pKW 132). Proteins from whole-cell lysates were separated by SDS-PAGE, transferred electrophoretically to a nitrocellulose filter, and cross-reacted with anti-TctC immunoglobulin G. Lanes: 1, purified TctC (29,000 M); 2, LE392(pKW131), LB grown; 3, LE392(pKW132), LB grown; 4, LE392(pKW131), PCM grown; 5, LE392(pKW132), PCM grown. The position of TctC is indicated by an arrow.

TABLE 2. Effect of medium composition on *tctC* expression

| Strain or plasmid (relevant genotype) | $\beta$-Galactosidase activity ($10^3$ units)$^{a}$ |
|---------------------------------------|------------------------------------------------|
| DL291(pKW136::Mu d1, pKW135)          | LB: 2.5, PCM: 15.2, DMM-glucose: 0.2            |
| (tctC-lacZ/tctC'D$^{+b}$)             |                                              |
| DL291(pKW137::Mu d1, pKW123)          | LB: 1.6, PCM: 14.5, DMM-glucose: 1.5            |
| (tctC-lacZ/tctC'D$^{+}$)              |                                              |
| DL291(pKW136::Mu d1, pKW6)            | LB: ND$^{c}$, PCM: 7.2, DMM-glucose: ND         |
| (tctC-lacZ/tctA$^{+}$B$^{+}$C$^{+}$D$^{+}$) |                                              |

$a$: $\beta$-Galactosidase activity was measured in cells which had been grown to the stationary phase in LB or DMM containing either 0.4% peptone (wt/vol) and 10 mM citrate (PCM) or 10 mM glucose.

$b$: tctD is present on the plasmid; the C-terminal part of *tctC* is truncated.

$c$: ND, Not determined.

Transcriptional activity of the *tctC* promoter was measured after cells had been grown to the stationary phase in either complex or minimal medium (Table 2). With the exception of cells grown on DMM containing 10 mM glucose, TctD invariably activated transcription from the *tctC* promoter regardless of the replicon or construction. $\beta$-Galactosidase activity was highest when DL291(pKW136::Mu d1, pKW135) or DL291(pKW137::Mu d1, pKW123) had been grown in PCM. DMM containing 10 mM glucose almost completely prevented transcription from the *tctC* promoter, suggesting catabolite repression.

To confirm the trans effect of TctD on *tctC* transcription, we compared $\beta$-galactosidase activities of *tctC*-lacZ fusions in haploid strains [DL291(pKW136::Mu d1) and DL291(pKW137::Mu d1)] and merodiploid strains containing *tctD* in trans as a complete or truncated gene [DL291(pKW136::Mu d1, pKW135); DL291(pKW136::Mu d1, pKW137) and DL291(pKW137::Mu d1, pKW123); Mu d1(pKW136, pKW137)] were compared. Table 3 shows clearly that TctD markedly enhanced transcription from the *tctC* promoter 8- to 10-fold, depending on the fusion replicon used.

Generation of *tctD*-lacZ transcriptional fusions. Mini-Mu dl1681 was also used to fuse the *tctD* promoter region present in pKW125 (Fig. 1) to the *lac* structural genes. Lac$^+$ fusions were generated as already outlined; plasmids were isolated and the position and orientation of various Mu insertions were once again determined by agarose gel electrophoresis after restriction with endonucleases EcoRI and HindIII. Of 30 plasmids isolated from independent Lac$^+$ clones, 5 had *lac* genes inserted into *tctD*. The putative *tctD*-lacZ fusions were also stabilized by excision of the Mu AB genes with the restriction endonuclease PstI; furthermore, the location of the five Mu insertions was confirmed by T7 polymerase-directed expression and radiolabeling of plasmid-encoded proteins in the KS1070(pGP1-2) background (25). TctD (29,000 $M_r$) was absent from whole-cell lysates obtained from each of the five strains when analyzed by SDS-PAGE and autoradiography (data not shown). Plasmid pKW125::Mu d1-5a (*tctD*-lacZ) was chosen for further studies.

Effect of medium composition on *tctD* expression. The expression of *tctD* was investigated by measuring $\beta$-galactosidase activity in a strain harboring a *tctD*-lacZ transcriptional fusion plasmid [DL291(pKW125::Mu d1-5a)]. Cells which had been pregrown in LB were inoculated into fresh medium, and $\beta$-galactosidase activity was observed during growth. Glucose (0.2% final concentration) strongly repressed the expression of $\beta$-galactosidase when added to either LB or DMM containing 0.2% glycerol, indicating that

same DNA inserts were transcribed from the T7 promoter by T7 RNA polymerase and protein gene products were labeled in the presence of [$^3$S]methionine, KS1070(pGP1-2, pKW131) and KS1070(pGP1-2, pKW132) produced TctC as well as TctB and TctA in approximately equal amounts (data not shown).

trans-acting element encoded by *tctD*. To investigate the possibility that TctD trans-activates *tctC* expression, we constructed *tctC*-lacZ transcriptional fusions by insertion mutagenesis with Mu d11681 *lac* mini-Mu (5), which carries a promoter-deficient *lacZYA* operon.

Two plasmids which encompass the promoter region and an N-terminal segment of *tctC* (pKW136, Amp$^r$; pKW137, Cam$^r$; Fig. 1) were each introduced into strain POI1681Tr, which has the mini-Mu-defective prophage Mu d11681 (Kan$^r$) and a complementing Mu cts (temperature sensitive) helper phage integrated into the chromosome (5). Cultures from each of the two transformed strains [POI1681Tr (pKW136), POI1681Tr(pKW137)] were heat induced for Mu-lytic growth, and a Mu-immune recipient strain [MC4100 (RP4-2-Tc::Mu)] was infected with either of the two phage lysates. Transductants were selected for their antibiotic resistance markers, and Lac$^+$ clones were identified as blue colonies on LB-X-Gal(5-bromo-4-chloro-3-indolyl-$\beta$-galactoside) plates. A total of 12 pKW136::Mu d1 and 20 pKW137::Mu d1 plasmids were isolated from independent Lac$^+$ clones. The position and orientation of the mini-Mu insertions in both plasmids were mapped by agarose gel electrophoresis after restriction with endonucleases HindIII and EcoRI. Three pKW136::Mu d1 and two pKW137::Mu d1 plasmids expressed the *lycZYA* genes from the *tctC* promoter.

---

TABLE 3. Effect of TctD protein on *tctC* expression

| Strain and plasmid                     | Relevant plasmid genotype or description                          | $\beta$-Galactosidase activity ($10^3$ units)$^{a}$ |
|----------------------------------------|--------------------------------------------------------------------|---------------------------------------------------|
| Haploid strains                        |                                                                    |                                               |
| DL291(pKW136::Mu d1)                   | *tctC*-lacZ transcriptional fusion; pT7-6 replicon                | 2.2                                             |
| DL291(pKW137::Mu d1)                   | *tctC*-lacZ transcriptional fusion; pACYC184 replicon             | 1.1                                             |
| Merodiploid strains                    |                                                                    |                                               |
| DL291(pKW136::Mu d1, pKW135)           | *tctC*-lacZ fusion/*tctC'*D$^{+b}$                                | 15.2                                            |
| DL291(pKW136::Mu d1, pKW137)           | *tctC*-lacZ fusion/*tctC'*D$^{+c}$                                | 2.7                                             |
| DL291(pKW137::Mu d1, pKW123)           | *tctC*-lacZ fusion/*tctC'*D$^{+}$                                 | 14.5                                            |
| DL291(pKW136::Mu d1, pKW136)           | *tctC*-lacZ fusion/*tctC'*D$^{+}$                                 | 2.5                                             |

$a$: $\beta$-Galactosidase activity was measured in cells which had been grown to the stationary phase in DMM containing 0.4% peptone (wt/vol) and 10 mM citrate.

$b$: tctD is present on the plasmid; the C-terminal part of *tctC* is truncated.

$c$: tctD and *tctC* are both C-terminally truncated.

VOL. 171, 1989 REGULATION OF TRICARBOXYLATE TRANSPORT IN S. TYPHIMURIUM 4439

**FIG. 3. Effect of medium composition on *tctD* expression.** (A) Strain DL291(pKW125::Mu d1-5a) was pregrown in LB, and samples were inoculated into LB (■), LB containing 0.2% glucose (●), or LB containing 0.2% glucose and 5 mM cAMP (□). (B) Cells from the same LB preculture were inoculated into DMM containing 0.2% glycerol (■), 0.2% glucose (□), or 0.2% glycerol and 0.2% glucose (●). β-Galactosidase activity was observed during cell growth.

transcription from the *tctD* promoter was subject to catabolite repression (Fig. 3A and B). Indeed, when 5 mM cyclic AMP (cAMP) was added to LB containing 10 mM glucose, the repressive effect of glucose was largely relieved (Fig. 3A).

**Role of TctD in *tctD* transcription.** In an attempt to determine whether *tctD* is autogenously regulated, we transformed a strain harboring pKW125::Mu d1-5a (*tctD-lacZ*) with a second compatible plasmid encoding TctD (pKW135; Fig. 1). The expression of *tctD* was then observed in both haploid [DL291(pKW125::Mu d1-5a)] and merodiploid [DL291(pKW125::Mu d1-5a, pKW134)] strains. Transcription of *tctD* was not influenced by the presence of TctD whether cells had been grown in either LB or PCM (Table 4). Thus, autogenous regulation of *tctD* was ruled out.

**DNA sequence analysis of *tctD*.** The nucleotide sequence of a 1,293-base-pair DNA fragment encompassing *tctD* was determined by dideoxynucleotide sequencing and is shown with the predicted amino acid sequence in Fig. 4. An open reading frame from nucleotides 155 to 826 encodes a 224-residue protein with a predicted molecular weight of 25,407, which is in close agreement with the molecular weight observed for TctD by SDS-PAGE (27). The initiation code was immediately preceded by a near-perfect Shine-Dalgarno sequence of AGGATGT (-4 to -10). Of six potential -10 promoter consensus sequences found, only one had a possible -35 consensus sequence 13 base pairs upstream. These are shown in boxes in Fig. 4. In addition, the two most probable target sequences (TGTGA) for the cAMP receptor protein (CRP)-cAMP interaction site are shown in boxes. These are directly repeated within 3 base pairs of each other. Six bases 3′ to the proximal repeat is a sequence resembling another part of the CRP-binding consensus sequence (TCA-ATA).

A hydrophobicity plot of the deduced amino acid sequence (not shown) did not indicate that TctD was particularly hydrophobic. Only two 8-residue stretches (amino acids 72 to 80 and 155 to 163) exhibited significant hydrophobic indices. In addition, the protein comprised only 45% hydrophobic residues (Ala, Phe, Ile, Leu, Met, Val, Trp, and Tyr). When the algorithm developed by Lipman and Pearson (15) was used to compare the amino acid sequence of TctD with those in the National Biomedical Research Foundation library, considerable homology between TctD and certain other transcriptional activators such as OmpR and NtrC was immediately apparent (Table 5).

**TABLE 4. Role of TctD protein in *tctD* transcription**

| Strain (plasmid) | Relevant plasmid genotype and description | β-Galactosidase activity (10³ units)<sup>a</sup> |
|-------------------|------------------------------------------|------------------------------------------------|
| Haploid strain [DL291 (pKW125::Mu d1-5a)] | *tctD-lacZ* transcriptional fusion | 0.57 | 1.2 |
| Merodiploid strain [DL291 (pKW125::Mu d1-5a, pKW135)] | *tctD-lacZ* transcriptional fusion/ *tctC'D*<sup>b</sup> | 0.83 | 1.3 |

<sup>a</sup> β-Galactosidase activity was measured in cells which had been grown to the stationary phase in either LB or PCM.  
<sup>b</sup> *tctD* is present on the plasmid; the C-terminal part of *tctC* is truncated.

**DISCUSSION**

Expression of the TctI structural genes *tctCBA* is strongly dependent on the trans-acting *tctD* gene product, which presumably binds to the *tctC* promoter to activate transcription. Transcription of *tctD* is itself positively controlled presumably by the binding of the CRP-cAMP complex to the *tctD* promoter. This presumption is strengthened by the

4440 WIDENHORN ET AL.

TABLE 5. Comparison of amino acid sequence homologies between TctD, OmpR, NtrC, and ArcA

| Proteins compared | Region of homology | % Identity |
|-------------------|--------------------|------------|
| Length            | Location           |            |
| TctD vs OmpR      | 219                | 4–215 vs 8–228 | 33.8 |
| TctD vs NtrC      | 122                | 8–129 vs 10–128 | 34.4 |
| TctD vs ArcA      | 228                | 2–216 vs 5–231 | 27.2 |

specific sensor protein, although anaerobic (U. Gauglitz, M.Sc. thesis, University of Göttingen, Göttingen, Federal Republic of Germany, 1984) and cryptic citrate transport systems have been reported (9; Gauglitz, M.Sc. thesis).

TctD also shows homology with ArcA (previously known as Dye), a pleiotropic regulatory protein which under anaerobic conditions represses the synthesis of a wide variety of enzymes associated with aerobic catabolism (11). Transport systems for substrates of aerobic pathways have been suggested as candidates for control by *arcA* (11). Perhaps TctD is functionally related to globally regulated systems that affect expression of aerobic transport systems for intermediary catabolites.

The TctI system apparently differs in its molecular composition from that of other periplasmic permeases. The histidine, maltose, branched-chain amino acid, oligopeptide, ribose, β-methylgalactoside, and phosphate transport operons encode a single periplasmic binding protein and three inner membrane proteins (3), whereas *tctI* encodes only C-protein (TctC) and two cytoplasmic membrane components (TctA and -B). The L-arabinose transport operon also encodes only two inner membrane proteins (AraG and AraH), but one of these (AraH) shares unmistakable homologies with one of the components of the previously mentioned systems, the ATP-binding component thought to mediate active transport (21). From a homology search through the DNA sequence of TctI (J. M. Somers and W. W. Kay, unpublished data), no similar sequences were apparent. This suggests that TctI differs from other shock-sensitive systems in its mode of active transport, that a hypothetical ATP-binding protein gene is not in the vicinity of the *tctI* operon, and/or that this component is shared with another transport system which must also be present in *E. coli*. The concept of protein components shared by two transport systems is not unprecedented since some amino acid transport systems are known to share binding protein components (14, 17).

The presence of a typical CRP-cAMP target sequence immediately upstream of *tctD*. The putative promoter (-10 and -35) consensus sequences (Fig. 4) differ somewhat from other procaryotic consensus sequences suggestive of very weak promoter activity. The preceding CRP-cAMP interaction site is presumably required to enhance transcription from the weak adjacent promoter.

Tricarboxylate transport in *S.* typhimurium is known to be repressed by the addition of D-glucose and partially reversed by cAMP (12). In addition, *S.* typhimurium strains with mutations in either *crp* (coding for the CRP protein) or *cya* (coding for adenylate cyclase) are unable to grow on citrate (1). Furthermore, *cpd* (coding for cAMP phosphodiesterase) mutants which accumulate cAMP (1) result in higher C-protein (TctC) synthesis (24). The regulation of TctI could be simplistically viewed as governed only by catabolite repression of *tctD*. However, we have not excluded the possibility that *tctCBA* also requires the CRP-cAMP complex to initiate transcription. No strong evidence for a CRP-cAMP consensus sequence in the region of the *tctC* promoter could be found (J. M. Somers and W. W. Kay, unpublished data). As an interesting comparison, transcription of the activator (*malT*) of the maltose regulon requires the CRP-cAMP complex but that of the divergently transcribed structural genes *malP* and *malQ* does not (6, 20).

The further finding that *tctD-lacZ* expression is the same irrespective of the presence or absence of a *tctD⁺* allele indicates that *tctD* expression is not regulated by its own gene product. The expression of *tctD* is therefore similar to that of *malT* (6) in that neither of these genes in autoregulated; they are only sensitive to catabolite repression. The role of citrate in the regulation of TctI is unknown even though we routinely add it to growth media. Citrate is known to induce tricarboxylate transport in *S.* typhimurium (12), but it is unclear at present which of the three tricarboxylate transport systems, TctI, -II, or -III (13), is induced by citrate. Preliminary evidence with *tctC-lacZ* transcriptional fusions suggests that TctI does not require citrate per se to activate TctD and effect transcription of *tctCBA* (K. A. Widenhorn and W. W. Kay, unpublished data), as is the case for D-arabinose (21) and maltotriose (20) induction of their respective periplasmic transport systems.

It is interesting that TctD shows homology with regulatory proteins that are known to respond to exogenous stimuli (Table 5). Such proteins belong to two-component regulatory systems in which a membrane sensor activates a regulatory component to effect transcription from outside the cell (19). The transport of C₄ dicarboxylates in Rhizobium species is also an example of such a system (18). If TctD were indeed part of such a system, it would have to interact with a hypothetical membrane sensor to be activated. In this regard, it has been observed that TctD associates weakly with the cytoplasmic membrane (27). It is also interesting that expression of TctI occurs in *Escherichia coli* (28), which normally lacks a tricarboxylate transport system and therefore would not be expected to express a tricarboxylate-

ACKNOWLEDGMENTS

This work was supported by a grant to W.W.K. from the Natural Sciences and Engineering Research Council of Canada. K.A.W. was the recipient of a University of Victoria research scholarship. We are indebted to K.-H. Müller for helpful suggestions with Mu phage.

LITERATURE CITED

1. Alper, M. D., and B. N. Ames. 1978. Transport of antibiotics and metabolite analogs by systems under cyclic AMP control: positive selection of *Salmonella typhimurium cya* and *crp* mutants. J. Bacteriol. 133:149–157.
2. Ames, G. F.-L. 1974. Resolution of bacterial proteins by polyacrylamide gel electrophoresis on slabs. Membrane, soluble, and periplasmic fraction. J. Biol. Chem. 249:634–644.
3. Ames, G. F.-L. 1986. Bacterial periplasmic transport systems: structure, mechanism, and evolution. Annu. Rev. Biochem. 55:397–425.
4. Barnes, W. M., and M. Bevan. 1983. Kilo-sequencing: an ordered strategy for rapid DNA sequence data acquisition. Nucleic Acids Res. 11:349–367.
5. Castilho, B. A., P. Olfsøn, and M. J. Casabadan. 1984. Plasmid insertion mutagenesis and *lac* gene fusion with mini-Mu bacteriophage transposons. J. Bacteriol. 158:488–495.
6. Chapon, C. 1982. Role of the catabolite activator protein in the maltose regulon of *Escherichia coli*. J. Bacteriol. 150:722–729.
7. Dale, R. M. K., B. A. McClure, and J. P. Houchins. 1985. A rapid single strand cloning strategy for producing a sequential

series of overlapping clones for use in DNA sequencing: application to sequencing the corn mitochondrial 18S rDNA. Plasmid 13:31–40.

8. Furlong, C. E. 1986. Osmotic-shock-sensitive transport systems, p. 768–796. *In* F. C. Neidhardt, J. L. Ingraham, K. B. Low, B. Magasanik, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli and Salmonella typhimurium*: cellular and molecular biology, vol. 1. American Society for Microbiology, Washington, D.C.

9. Hall, B. G. 1982. Chromosomal mutation for citrate utilization by Escherichia coli K-12. J. Bacteriol. 151:269–273.

10. Hobson, A. C., R. Weatherwax, and G. F.-L. Ames. 1984. ATP-binding sites in the membrane components of histidine permease, a periplasmic transport system. Proc. Natl. Acad. Sci. USA 81:7333–7337.

11. Iuchi, S., and E. C. C. Lin. 1988. *arcA* (dye), a global regulatory gene in *Escherichia coli* mediating repression of enzymes in aerobic pathways. Proc. Natl. Acad. Sci. USA 85:1888–1892.

12. Kay, W. W., and M. Cameron. 1978. Citrate transport in *Salmonella typhimurium*. Arch. Biochem. Biophys. 190:270–280.

13. Kay, W. W., G. D. Sweet, K. A. Widenhorn, and J. M. Somers. 1987. Organic acids, p. 269–300. *In* B. P. Rosen and S. Silver (ed.), Ion transport in procaryotes. Academic Press, Inc., Orlando, Fla.

14. Kustu, S. G., and G. F.-L. Ames. 1973. The *hisP* protein, a known histidine transport component in *S.* *typhimurium*, is also an arginine transport component. J. Bacteriol. 116:107–113.

15. Lipman, D. J., and W. R. Pearson. 1985. Rapid and sensitive protein similarity searches. Science 227:1435–1441.

16. Miller, J. H. 1972. Experiments in molecular genetics. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

17. Nazos, P. M., T. Z. Su, R. Landick, and D. L. Oxender. Branched-chain amino acid transport in *Escherichia coli*, p. 24–28. *In* L. Leive and D. Schlessinger (ed.), Microbiology—1984. American Society for Microbiology, Washington, D.C.

18. Ronson, C. W., B. T. Nixon, L. M. Albright, and F. M. Ausubel.

19. Rhizobium meliloti *ntrA* (*rpoN*) gene is required for diverse metabolic functions. J. Bacteriol. 169:2424–2431.

20. Ronson, C. W., B. T. Nixon, and F. M. Ausubel. 1987. Conserved domains in bacterial regulatory proteins that respond to environmental stimuli. Cell 49:579–581.

21. Schwartz, M. 1987. The maltose regulon, p. 1482–1503. *In* F. C. Neidhardt, J. L. Ingraham, K. B. Low, B. Magasanik, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli and Salmonella typhimurium*: cellular and molecular biology, vol. 2. American Society for Microbiology, Washington, D.C.

22. Scripture, J. B., C. Voelker, S. Miller, R. T. O'Donnell, L. Polgar, J. Rade, B. I. Horazdovsky, and R. W. Hogg. 1987. High-affinity *L*-arabinose transport operon. Nucleotide sequence and analysis of gene products. J. Mol. Biol. 197:37–46.

23. Simon, R., U. Priefer, and A. Pühler. 1983. A broad host range mobilization system for *in vivo* genetic engineering-transposon mutagenesis in gram negative bacteria. Bio/Technology 1:784–790.

24. Somers, J. M., and W. W. Kay. 1983. Genetic fine structure of the tricarboxylate transport (*tct*) locus of *Salmonella typhimurium*. Mol. Gen. Genet. 190:20–26.

25. Sweet, G. D., C. M. Kay, and W. W. Kay. 1984. Tricarboxylate-binding proteins of *Salmonella typhimurium*. J. Biol. Chem. 259:1586–1592.

26. Tabor, S., and C. C. Richardson. 1985. A bacteriophage T7 RNA polymerase system for controlled exclusive expression of specific genes. Proc. Natl. Acad. Sci. USA 82:1074–1078.

27. Towbin, H., T. Staehlin, and J. Gordon. 1979. Electrophoretic transfer of proteins from polyacrylamide gels to nitrocellulose sheets: procedure and some applications. Proc. Natl. Acad. Sci. USA 76:4350–4354.

28. Widenhorn, K. A., J. M. Somers, and W. W. Kay. 1988. Expression of the divergent tricarboxylate transport operon (*tctI*) of *Salmonella typhimurium*. J. Bacteriol. 170:3223–3227.

29. Widenhorn, K. A., W. Boos, J. M. Somers, and W. W. Kay. 1988. Cloning and properties of the *Salmonella typhimurium* tricarboxylate transport operon in *Escherichia coli*. J. Bacteriol. 170:883–888.
