
JVI Accepted Manuscript Posted Online 30 December 2015  
J. Virol. doi:10.1128/JVI.02865-15  
Copyright © 2015, American Society for Microbiology. All Rights Reserved.

Crystal Structure of the Measles Virus Nucleoprotein Core in complex with an N-terminal  
Region of Phosphoprotein  

Sergey G. Guryanov, Lassi Liljeroos*, Prasad Kasaragod, Tommi Kajander, Sarah J. Butcher#  

Institute of Biotechnology, University of Helsinki, Helsinki, Finland  

Running Head: Measles Nucleoprotein-Phosphoprotein Structure  

#Address correspondence to Prof. Sarah J. Butcher, sarah.butcher@helsinki.fi.  

*Present address: GlaxoSmithKline Vaccines, Siena, Italy  

Word count for the abstract: 203 words.  
Word count for the text: 4907 words.

Abstract

The enveloped negative-stranded RNA virus, measles virus (MeV) is an important human pathogen. The nucleoprotein (N°) assembles with the viral RNA into helical ribonucleocapsids (NC) which are in turn coated by a helical layer of the matrix protein. The viral polymerase complex uses the NC as its template. The N° assembly onto the NC and the activity of the polymerase are regulated by the viral phosphoprotein (P). Here, we pulled down an N°1-408 fragment lacking most of its C-terminal tail domain by several affinity-tagged, N-terminal, P fragments to map the N°-binding region of P to the first 48 amino acids. We showed biochemically and using P mutants the importance of the hydrophobic interactions for the binding. We fused an N° binding peptide, P1-48, to the C-terminus of an N°21-408 fragment lacking both the N-terminal peptide and the C-terminal tail of N protein to reconstitute and crystallize the N°-P complex. We solved the X-ray structure of the resulting N°-P chimeric protein at 2.7 Å resolution. The structure reveals the molecular details of the conserved N°-P interface and explains how P chaperones N° preventing both self-assembly of N° and its binding to RNA. Finally, we propose a model for a pre-initiation complex for RNA polymerization.

Importance

Measles virus is an important, highly contagious, human pathogen. The nucleoprotein N binds only to viral genomic RNA and forms the helical ribonucleocapsid that serves as a template for viral replication. We address how N is regulated by another protein, the phosphoprotein, P, to prevent newly synthesized N from binding to cellular RNA. We describe the atomic model of an N-P complex and compare it to helical ribonucleocapsid. We thus provide insight into how P chaperones N and helps to start viral RNA synthesis. Our results provide a new insight into mechanisms of paramyxovirus replication. New data on the mechanisms of phosphoprotein

chaperone action allows better understanding of the virus genome replication and nucleocapsid
assembly. We describe a conserved structural interface for the N-P interaction which could be a
target for drug development not only to treat measles but also potentially other paramyxovirus
diseases.

Introduction

Measles virus (MeV) belongs to the *Paramyxoviridae* family that include several other human
pathogens like respiratory syncytial (RSV), mumps, and parainfluenza viruses. It has a helical
ribonucleocapsid (NC) containing a nonsegmented –ssRNA genome wrapped around the outside
of the nucleoprotein (N) helix (1). The helical NC is active in both transcription and replication.
During virus assembly the matrix protein forms an additional helix covering the majority of the
NC potentially inhibiting transcription and promoting packaging into progeny virions (2). There
is still only limited data on the detailed molecular interactions required to go from replication
initiation to packaging of nascent RNA. The availability of N in a chaperoned, assembly-
competent state with the phosphoprotein (P) versus the assembled helical state is thought to be
critical to these processes.

N is composed of an ordered N_CORE region (1-391 aa) and an intrinsically disordered N_TAIL
region (392-525 aa) (Fig. 1A). N_CORE contains two domains (N_NTD and N_CTD) flanked by N-
and C-terminal arms (NT_arm and CT_arm). A recent atomic model of the MeV NC from a cryo-EM
reconstruction revealed the molecular details of N oligomerization mediated by exchange of the
NT_arm and CT_arm between consecutive N monomers, and showed the RNA-binding site on the
groove between the two N_CORE domains (1).

P is a modular protein comprised of an ordered tetramerization domain (MD; amino acids 304-377) forming a parallel four-helix coiled-coil (3) and an extreme C-terminal domain (XD) alternating with disordered regions (Fig. 1A). For transcription and replication, the RNA polymerase (L) in complex with P attaches to the NC via an interaction between the XD domain in P and the molecular recognition element (MoRE; Fig. 1A) in N (4-7). The three helix bundle in XD binds a helix from N's MoRE element to facilitate this interaction (7). P has a second role, it binds N<sub>CORE</sub> through its N-terminal soyuzl motif (8) and serves a chaperone function required to keep newly synthesized N from binding to cellular RNA (9). This RNA-free N<sup>0</sup> is then transferred from the N<sup>0</sup>-P complex to the nascent NC by a currently unknown mechanism.

Whereas the XD and MD domains of P have been well characterized, the interaction between the P N-terminus and the N in the N<sup>0</sup>-P complex is less well described. The dual function for P has been established for many viruses of the Mononegavirales order and a crystal structure of the Vesicular stomatitis virus (VSV), Ebola virus and Nipah virus (NiV) N<sup>0</sup>-P complexes have been solved (10-12). In VSV N<sup>0</sup>-P, the P N-terminal amino acids 17-31 formed an amphipathic α-helix and occupied a hinge region in N adjacent to the RNA-binding site (13). In NiV N<sup>0</sup>-P, P amino acids 1-35 formed two α-helices separated by a kink (11). Interestingly, the NiV P binding site does not overlap with the predicted RNA binding groove, therefore the chaperoning mechanism of P appears to be remarkably different from that in VSV. In the present study, we addressed the MeV N<sup>0</sup>-P complex formation and structure. We expressed and purified MeV N<sup>0</sup>-P complexes from E. coli in a monodisperse form and mapped the location of the N-binding region on P to the first 48 N-terminal amino acids. Then we designed a chimeric N-P protein that was crystallized to reconstitute the N<sup>0</sup>-P complex and solved the structure at 2.7 Å resolution. We also characterized the mode of interaction between the P N-terminus and N<sup>0</sup> and show the

importance of hydrophobic interactions. Based on the structural data we describe conformational
changes upon RNA-binding and propose a model for the pre-initiation complex for RNA
replication and transcription.

Materials and Methods

Cloning and expression. All constructs were derived from reverse transcribed N and P genes of
MeV wild-type isolate (a gift from I. Davidkin, Helsinki, Finland) (2). The N coding sequence
was identical to the GenBank sequence for the Halonen strain (accession number U01996). The
P coding sequence differed from the GenBank sequence AF266288 for the Edmonston strain by
three nucleotides, one was synonymous, the other two resulted in the amino acids G225 and
D492. Truncated P constructs were generated by PCR and cloned into NcoI and XhoI sites in
pET41(a). The P constructs had an N-terminal glutathione-S-transferase followed by
hexahistidine sequence (GST-H<sub>6</sub> tag) for purification. N<sub>1-408</sub> and N<sub>21-408</sub> were constructed
similarly, but cloned into pET22(b) with a stop-codon added to the 3′ end and did not contain
any tags. For the N<sub>21-408</sub>-P<sub>1-48</sub> chimera, the N<sub>21-408</sub> and P<sub>1-48</sub> coding sequences were amplified by
PCR to generate megaprimers with overlapping sequences. Then the megaprimers were annealed
and extended. The product was amplified with primers coding for an N-terminal H<sub>6</sub>-TEV tag,
MGSSHHHHHHENLYFQ|S, where the tobacco etch virus (TEV) protease recognition sequence
is underlined and the cleavage site is shown by a vertical line (14). A stop-codon was introduced
at the 3′ end. The product was cloned into NcoI and XhoI sites in pET22(b). Mutations in the P
constructs were introduced by site-directed mutagenesis.

Proteins were expressed in E.coli Rosetta (DE3) (Merck Millipore). Expression was induced at
OD<sub>600</sub> 0.5 with 0.5 mM IPTG and was allowed to proceed for 16–20 h at 22 °C. Cells were
collected by low speed centrifugation and frozen at -80 °C as pellets until use.

115 Protein purification. N${}_{1-408}^{0}$ and GST-H${}_{6}$-P${}_{1-48}$ were co-expressed in E.coli Rosetta (DE3). The
116 cell pellet was resuspended in buffer A (20 mM Tris-HCl, 500 mM NaCl, 20 mM imidazole, 2
117 mM CaCl${}_{2}$, pH 8.0) supplemented with 200 μg/ml lysozyme and one EDTA-free protease
118 inhibitor tablet/25 ml (Thermo Scientific). Cells were lysed with a French press at 22 000 psi,
119 cell debris spun down by low speed centrifugation (11 000 × g for 15 min at +4°C) and the
120 resulting supernatant was incubated with Ni-loaded IMAC beads (GE Healthcare) for 45 min at
121 room temperature. After washing with buffer A, the beads were exchanged into buffer B (20 mM
122 Tris-HCl, 150 mM NaCl, 10 mM MgCl${}_{2}$, 2 mM ATP, pH 8.0) and incubated at 37°C for 10 min.
123 Next, the beads were exchanged into buffer C (20 mM Tris-HCl, 150 mM NaCl, 2 mM CaCl${}_{2}$,
124 pH 8.0). The N${}_{1-408}^{0}$-P${}_{1-48}$ heterocomplex was then released from the beads by an overnight
125 digestion with enterokinase light chain (New England Biolabs). For 2 ml of Ni-IMAC beads with
126 protein from 1 l cell culture, 0.16 μg of enzyme was used. The released protein was then
127 concentrated with Millipore Amicon Ultra-4 30 kDa cutoff spin concentrators and polished with
128 size exclusion chromatography using a Superdex 200 column (GE Healthcare). Peak fractions
129 were collected and concentrated to the desired concentration with the same concentrator.
130 For the N${}_{21-408}^{0}$-P${}_{1-48}$ chimera purification, a cell pellet containing the tagged chimera was
131 resuspended in buffer D (20 mM Tris-HCl, 150 mM NaCl, 2 mM MgCl${}_{2}$, 20 mM imidazole, pH
132 8.0) supplemented with 10 μg/ml lysozyme, 1 μg/ml DNase I (Sigma-Aldrich), and 0.5 mM
133 Pefabloc (Roche). Cells were lysed, and the lysate was cleared and incubated with Ni-IMAC as
134 above. After washing with buffer D, protein was eluted with buffer D supplemented with 0.2 M
135 imidazole. Eluted protein was incubated with TEV protease (purified in-house) overnight at
136 +4°C. Cleaved protein was purified on a Superdex 200 column in buffer E (20 mM Tris-HCl,

150 mM NaCl, pH 8.0), monomer peak was collected, and uncleaved protein was removed by  
passing through an Ni-IMAC column. Purified protein was concentrated as above.  
N°-P heterocomplex interaction experiments. To find the minimal length of P that stably  
interacted with N₁₋₄₀₈, 5 different P constructs were co-expressed with N₁₋₄₀₈ as above. The cells  
were lysed by sonication in PBS supplemented with 0.17 mg/ml lysozyme. The cleared lysates  
were incubated with glutathione beads in PBS for 30 minutes at room temperature. After 3  
washes with PBS, the proteins were eluted with 10 mM reduced glutathione in 50 mM Tris HCl,  
pH 8.0.  
To test the stability of the interactions, the purified N₁₋₄₀₈°–GST-H₆-P₁₋₄₈ heterocomplex was  
bound in 50 mM Na phosphate pH 7.4, 10 mM imidazole, 300 mM NaCl to Ni-IMAC beads, and  
eluted with 20 mM Tris pH 8.0, supplemented with one of the following: 0 M NaCl, 0.5 M NaCl,  
1 M NaCl, 2 M NaCl, 0.5 M KCl, 1 M KCl, 2 M KCl, 2 M urea, 4 M urea, 8 M urea, 0.1%  
Triton-X 100, or 1% Triton-X 100.  
To probe N₂₁₋₄₀₈° heterocomplex formation with mutated GST-H₆-P constructs, cell lysates of  
individually expressed proteins were mixed and incubated overnight at +4°C. Lysate samples  
were incubated with Ni-IMAC beads and washed with lysis buffer E. The samples were eluted  
with buffer E supplemented with 0.2 M imidazole. Eluates were analyzed by SDS-PAGE.  
Sec-MALLS experiment. To analyze the exact stoichiometry of the P₁₋₄₈–N₁₋₄₀₈ heterodimeric  
complex, 47 μl of the complex (1 mg/ml) released by enterokinase digestion was run on a  
Superdex 200 10/300 GL column coupled into UV, refractive index and multi-angle laser light  
scattering (MALLS) detectors (Wyatt Technology). The molecular weight of the complex was  
then calculated based on the refractive index and MALLS signals using the ASTRA 6 software  
(Wyatt Technology).

160 Electron microscopy of negative stained samples. Samples were pipetted on glow-discharged
161 carbon coated copper grids and stained with 1% (w/v) sodium phosphotungstate (pH 7.0). Grids
162 were imaged with an FEI F 20 transmission electron microscope and images collected with a
163 Gatan Ultrascan 4000 CCD camera.
164 Structure determination. Crystals of the N<sub>21-408</sub><sup>0</sup>-P<sub>1-48</sub> chimeric protein were grown by sitting-
165 drop vapor diffusion (22°C) by mixing 200 nl of protein (8 mg/ml) with 200 nl of reservoir (0.1
166 M sodium citrate, pH 5.2, 3% polyethylene glycol 8000). Crystals were cryoprotected in mother
167 liquor containing 20% glycerol and flash-frozen in liquid nitrogen. Diffraction data were
168 collected at the Diamond Light Source beamline I03. The dataset was processed and scaled using
169 the xia2 package (with XDS and AIMLESS) (15, 16). A summary of the data collection is given
170 in Table 1. The structure was solved by molecular replacement using PHASER (17). The N-
171 C-terminal domains, corresponding to amino acids 31-261 and 262-371 respectively, of the NiV
172 N (PDB ID: 4CO6, (11)) were used as search models separately. The model was rebuilt using
173 several cycles of autobuilding and refinement with PHENIX (18) and manual rebuilding with
174 COOT (19). No density was observed for N regions 21-30, 119-120, 133-139 and P region 39-48
175 and therefore they were left out of the model. The last refinement cycles were done using TLS
176 parameters (two TLS groups). The final refinement statistics are summarized in Table 1. The
177 final R-factors (R<sub>work</sub>/R<sub>free</sub>) of the refined structure are 21.1/26.6 (%) (shown in Table 1). In the
178 Ramachandran plot, 90% of the residues in the structure are in the most favored regions. Final
179 refined coordinates and structure factors have been deposited in the PDB with accession code
180 5E4V.
181 Structure analysis. All the structure illustrations were prepared using UCSF Chimera software
182 (20). Interface surface was estimated using the PDBBePISA server (21). Calculation of the relative

angle between the N domains in N°-P vs. NC structure (PDB ID: 4UFT) was done using

Modeller software (22) as described earlier (23). Structure alignments and RMSD values

calculations were made using UCSF Chimera. Dali multiple structural alignment (24) was used

to generate the corresponding primary sequence alignment followed by phylogenetic tree

generation by PHYLIP in Unipro UGENE software (25).

Results

Mapping the interaction of the MeV Ncore° with the N-terminal region of P and

crystallization of the complex. In order to obtain a well-structured Ncore°-P complex, we

analyzed the protease sensitivity of N by limited trypsin proteolysis (not shown). Based on mass-

spectrometric analysis of the fragments observed we cloned a C-terminally truncated construct

containing the first 408 amino acids, N1-408, thus excluding the disordered C-terminal Ntail

region (7).

To screen for Ncore interaction in the N-terminal region of P, we used co-expression of N1-408

together with GST-hexahistidine (GST-H6) fusions of P1-38, P1-48, P1-58, P1-68 or P1-78 (Fig. 1A)

and analyzed the interactions by GST affinity chromatography. All of the P constructs readily

interacted with N1-408 and could be clearly seen on SDS-PAGE (Fig. 1B). Thus, the PN-terminal

interaction site with Ncore resides within the first 38 amino acids. We analyzed negatively-

stained N1-408°-GST-H6-P1-48 eluate and the flow-through with electron microscopy. In the

eluate we observed a monodisperse solution of a small complex (Fig. 1C, left panel) whereas in

the flow-through NC-like helical particles were readily visible (Fig. 1C, right panel). Probably,

the NC-like particles contained N assembled on nonspecific cellular RNA (26). After GST-H6

tag cleavage, the purified complex eluted from gel-filtration as a single peak corresponding to a

1:1 heterodimer and was verified by SEC-MALLS to be 52 ± 2 kDa in size (Fig. 1D). This

206 complex appeared not to contain nucleic acid as the A₂₆₀/₂₈₀ was 0.55, whereas the expected ratio
207 for pure protein is ~0.6. Despite extensive efforts, the heterodimeric complex failed to
208 crystallize. Hence, we designed a chimeric construct H₆-TEV-N₂₁₋₄₀₈⁰-P₁₋₄₈ where the N-terminal
209 region of P was directly fused to the C terminus of the N_CORE domain lacking its NT_arm region
210 (Fig. 1A). The chimera was readily expressed as a soluble protein and purified. The gel filtration
211 mobility (Fig. 1D) and the A₂₆₀/₂₈₀ ratio of the chimera were similar to those of the heterodimeric
212 complex, with an additional dimer peak. The solution state of the N₂₁₋₄₀₈⁰-P₁₋₄₈ chimera suggests
213 that the P₁₋₄₈ sequence is bound to the P-binding site reconstituting the N⁰-P complex, preventing
214 the formation of helical complexes.
215 Crystal structure of the MeV N_CORE⁰-P complex. The MeV N₂₁₋₄₀₈⁰-P₁₋₄₈ chimera was
216 crystallized in the space group P3₁21 as a dimer with the P₁₋₄₈ sequence swapped between
217 chimera monomers. We determined the structure at 2.7 Å resolution by molecular replacement
218 using the NiV N⁰-P complex structure PDB ID: 4CO6 (11) as a starting model (Fig. 2 and Table
219 1). The amino acid sequence of the N₂₁₋₄₀₈⁰-P₁₋₄₈ chimera could be traced starting from N residue
220 31 to P residue 38 with the exception of N residues 119-120 and 133-139. The buried surface
221 interface in the crystallized dimer was 6520 Å², indicating a stable interaction interface for the
222 dimer as seen in gel filtration.
223 N_CORE⁰ is primarily an α-helical protein with two domains, the NTD (31-265 aa) and the CTD
224 (266-372 aa) (Fig. 2B) separated by a hinge. The NTD is formed by α-helices α_N1 to α_N9, one
225 3/10 helix η_N1, and parallel β-sheet β_N1-β_N2 with the adjacent short β-strand β_N3 (Fig. 2). The
226 CTD is formed by helices α_N10 to α_N15, and four 3/10 helices η_N2 to η_N5. The CT_arm (373-408
227 aa) continues as helices α_N15 to α_N17, with α_N15 and α_N16 adopting a helix-turn-helix
228 conformation. P₁₋₄₈ forms two helices (Fig. 2 and 3). The first helix α_P1 is a continuation of the

229 αN17 helix and binds the partner molecule in the crystallized dimer to the groove formed by
230 helices αN10, αN11. The second helix, αP2, contacts helix ηN2.
231 Interaction of N_CORE^0 with P is mainly hydrophobic. Conservation of the N_CORE^0 binding
232 interface for P (8, 11) and of the P N-terminal region in some paramyxoviruses has been
233 described (8, 11). In the MeV N_CORE^0-P complex, the binding interface is mostly composed of
234 conserved hydrophobic residues (Fig. 3A and 3B, in blue). To biochemically probe the binding
235 of the P N-terminal region to N_CORE^0, we screened for dissociation of N_1-408 from GST-H6-P_1-48
236 bound on Ni-IMAC beads in different conditions and looked for release of N_1-408. The screen was
237 designed to include conditions which would hinder either ionic or hydrophobic interactions
238 between the proteins. NaCl or KCl concentrations ranging between 0 and 2M did not cause
239 significant release of N_1-408, whereas 0.1% and 1% Triton-X 100 caused release of N_1-408 from
240 the complex (Fig. 3C) to similar levels as 4 and 8M urea, respectively.
241 To evaluate the role of P's hydrophobic amino acid residues in N^0 binding, we expressed N_21-408
242 lacking the NT_arm. We found that N_21-408 was insoluble and formed NC-like particles upon
243 expression (not shown) similar to N_1-408 but could bind to P_1-48 in vitro. We pulled-down N_21-408
244 with GST-H6-tagged wild type P_1-48 and its mutants (Fig. 3D). While wild type P_1-48 can
245 efficiently bind N_21-408, substitutions of hydrophobic by negatively-charged amino acids (L13D,
246 I16D, and L19E) severely affected the interaction. Substitution of L13 by the small amino acid
247 Ala also strongly affected the interaction, possibly due to the increased solvent accessibility of
248 the binding interface. In line with this observation, we still observed residual binding of the
249 shorter P_8-48 peptide where most of the interacting hydrophobic amino acids were retained (Fig.
250 3D, lane 9). Thus, hydrophobic interactions make a major contribution to P N-terminal region
251 and N_CORE^0 binding.

Comparison of the MeV RNA-bound helical form with the chaperoned form. Direct comparison of the MeV N<sub>CORE</sub><sup>0</sup>-P and helical NC structures reveals several factors that could contribute to P's chaperone activity, these include conformational changes (Fig. 4) as well as the position of P and RNA binding (Fig. 5 and 6). The largest difference between the two structures is that there is a relative domain movement in the MeV N in the N<sub>CORE</sub><sup>0</sup>-P complex, compared to the helical RNA-bound NC form (Fig. 4A and 4B) (1). By aligning either the CTD or NTD only, we measured an ~40° relative rotation of the two domains, with the hinge occurring between α<sub>N</sub>9 and α<sub>N</sub>10 (Fig. 4). The RMSD for the individual domains in the two different MeV N conformations were calculated (Table 2). This comparison indicated that the 4.3 Å resolution cryoEM structure agrees well with our crystal structure, and the changes seen between the structures could be interpreted reliably (Fig. 4A and 4B). Besides bending of the hinge between helices α<sub>N</sub>9 and α<sub>N</sub>10, helix α<sub>N</sub>6 forming the lower lobe of the RNA binding cleft differs between the two different states. The helix undergoes both a shift and a rotation around its axis by half a turn (Fig. 4D). The helix movement increases the proximity of the two sides of the interdomain cleft in N<sub>CORE</sub><sup>0</sup>-P and therefore collapses the NC RNA binding site. In addition, the surface electrostatic charge distribution changes quite dramatically depending on the α<sub>N</sub>6 conformation. In N<sub>CORE</sub><sup>0</sup>-P a new negatively-charged groove is evident on the NTD surface that could potentially bind RNA (Fig. 5). It has a contribution from the conserved R194 that interacts with the RNA backbone in NC (1). In NC, Y199 stacks with Y260, a key residue that regulates RNA binding pocket size. In N<sub>CORE</sub><sup>0</sup>-P, Y199 faces the exterior and W196 occupies the space instead, thus potentially participating in the local stacking configuration.

Alignment of the N<sub>CTD</sub> domains of N<sub>CORE</sub><sup>0</sup>-P and NC (1) models (Fig. 4A and 4C) shows that the P N-terminus would clash with helix α<sub>N</sub>9 of N in the NC conformation thus the alternative

conformation is favored. When we consider the superposition of N<sub>CTDS</sub> in the context of the  
assembled NC, we can discern a direct effect of P binding (Fig. 6). The α<sub>P1</sub> helix overlaps with  
the NT<sub>arm</sub> of the N<sub>i-1</sub> protomer and the α<sub>P2</sub> helix overlaps with the CT<sub>arm</sub> of the N<sub>i+1</sub> protomer.  
Hence P could specifically inhibit the association of adjacent protomers to the growing NC helix  
by steric hindrance, rather than by competing with the RNA-binding site.  

Comparison to other virus nucleoproteins. Comparison of the N structure from MeV and other  
viruses of the Mononegavirales order reveals their structural similarity (Fig. 7). Each protein is  
composed of two domains with a single interdomain connection. Phylogenetic analysis based on  
Dali multiple structural alignment (24) shows that the two structurally closest to MeV N are NiV  
N and PIV5 N with their amino acid sequence identity of 32% to NiV and 24% to PIV5.  

To our knowledge, MeV and VSV are the only Mononegavirales members with both the N<sup>0</sup>-P  
and N-RNA complexes available. Compared to the 40° rotation transition between the N<sup>0</sup>-P and  
the RNA-bound states in MeV N, in VSV N the RMSD reported between the two states was less  
than 1 Å (10) reflecting the fact that both VSV states were crystallized in a ring form that  
probably constrained the conformation. In contrast to MeV, VSV P blocks the RNA-binding site,  
rather than binding on the opposite side of the molecule. For the other N<sup>0</sup>-P complexes reported,  
NiV N<sup>0</sup>-P is the closest, with a similar binding site to MeV N<sup>0</sup>-P (Fig. 3 and 6), and relative  
domain positioning (referred to as an “open conformation” in NiV (11)). None of the other RNA-  
bound states are from a helical NC structure, rather they are all ring structures, but in PIV5,  
RSV, and RabV the RNA-bound states also indicate similar domain positioning to MeV NC  
(“closed state”) (11, 27). These comparisons emphasize the importance of the flexibility in the  
interdomain region in regulating N’s interactions with other viral components.

297 Discussion

298 Here, for the first time in Paramyxovirus research, our data allow direct comparison of the

299 structure of the nucleoprotein from the same virus in two functional states: a P-bound naive state,

300 and an RNA-bound helical assembly. Our X-ray crystallographic atomic model at higher

301 resolution confirms, complements and improves upon the recently published cryoEM

302 reconstruction (1). The overall fold of MeV N is most similar to those of NiV and Parainfluenza

303 virus 5 (PIV5) N proteins (Fig. 7) (11, 27). In addition, the structure of MeV P<sub>1-48</sub> and its binding

304 site are very similar to those of NiV N<sup>0</sup>-P complex (11). Noting the extensive, conserved

305 hydrophobic interactions of the P protein helix α<sub>P1</sub> and N (8, 11), we showed by mutation of the

306 hydrophobic residues in P and by biochemical analyses that we could indeed impair the binding

307 interaction.

308 Our finding suggests that both N domains mostly preserve their fold upon transition from the N<sup>0</sup>-

309 P to the NC state. Notably, N in both N<sup>0</sup>-P and NC has a flexible region between residues 118-

310 140 composed of a well-defined α<sub>N4</sub> helix (124-130) flanked by unresolved regions. In NiV N,

311 the α<sub>N4</sub> helix is longer, and only one unresolved region was left, whereas in PIV5 there are no

312 gaps here. This region is on the outer surface of the NC. Hence this flexible region could interact

313 with the flexible C-terminus of N, or with the polymerase complex.

314 How does P act as a chaperone? The roles of P are at least two-fold, firstly to act as a

315 chaperone to keep N<sup>0</sup> in its RNA-free, soluble, monomeric form, and secondly to position the

316 polymerase complex for polymerization. In the first role as a chaperone, it has been proposed

317 that in Nipah N<sup>0</sup>-P complex binding of P to N locks the “open” conformation by rigidifying the

318 N<sub>CTD</sub> structure (11). Our model however suggests a significant impact from steric interference

319 between the P N-terminus and N<sub>NTD</sub> (Fig. 6). Alignment of the N<sub>CTD</sub> domains of MeV N<sup>0</sup>-P and

320 NC (1) models (Fig. 4A and 4C) shows that the P N-terminus will clash with helix αN9 of N in
321 the RNA-bound conformation thus favoring the RNA-free conformation in N°-P. The flexibility
322 of the N molecule is therefore an inherently important part of our model, compared to the
323 published NiV model (11). We have additional evidence that in the NC, N can assume different
324 conformations. The pitch of the protease-treated NC used for high resolution structure
325 determination is 5.0 nm (1), and imposes a rigidity on the helix that was important for image
326 processing. However, the recombinant full-length protein, forms flexible helices with pitches
327 ranging from 5.0-6.6 nm (28, 29), and those imaged inside virions have a pitch of 6.4 nm (2). In
328 the latter, the rigidity of the NC helix is enforced by interaction with an outer layer of matrix
329 protein. Where the matrix is lacking, the NC is flexible. From the current work, at least two
330 flexible regions could affect the twist and pitch, the twisting of the two domains (28), induced by
331 the interdomain hinge region described above, and the conformation of the αN6 helix. Confident
332 assignment of amino acids Trp196 and Tyr199 in this helix in both the cryoEM and X-ray
333 electron densities showed rotation and elongation of the αN6 helix (Fig. 4D) reflecting the
334 intrinsic flexibility in this part of the molecule. Noticeably, in both the NiV N°-P and PIV5 N-
335 RNA structures (11, 27) a loop preceding the corresponding helix is unresolved that further
336 supports the inherent mobility of the αN6 helix.
337 What determines whether N binds P or RNA? The RNA binding in the NC state is favored by
338 specific arrangement of the amino acid residues from both the NNTD and NCTD (1). In the N°-P
339 state, the rotation of the N domains forces the overlap of these two binding surfaces, hence we
340 hypothesize that the RNA binding affinity is reduced. This is supported by two observations,
341 firstly, the solubility of the chimera in an E.coli cell lysate – we find a predominance of RNA-
342 free monomers and dimers, rather than helical assemblies even in the presence of E.coli RNA

343 compared to N expressed alone. Hence the N<sup>0</sup>-P interaction hampers NC assembly and binding
344 to RNA in our hands. Secondly, the surface charge distribution of the chimera is altered,
345 changing and shrinking the position of the positively-charged surface in the N<sup>0</sup>-P compared to
346 the NC. This suggests that N's affinity for P in our constructs was higher than for RNA. There is
347 probably a balance in the cell, during infection, dictated by the local concentrations of the
348 relevant components, and the avidity of N for RNA and its neighboring N subunits that together
349 orchestrate the assembly of the NC. The flexibility of N facilitates its exchange between its
350 binding partners, P and RNA. P can further regulate helix assembly through sterically impeding
351 both side-to-side and vertical growth of the helix through occupying the same sites as both the
352 NT<sub>arm</sub> of the N<sub>i-1</sub> protomer (yellow) and CT<sub>arm</sub> of the N<sub>i+1</sub> protomer (Fig. 6).
353 Model for the formation of a pre-initiation complex. According to the current Paramyxovirus
354 models, both transcription and replication are initiated at the 3′ end of the genomic RNA (30).
355 The linear unidirectional organization of the “herringbone” NC means that the 3′ and 5′ ends of
356 the NC do not present the same molecular surface due to this polarity. In addition, the transition
357 between the bulk of the helix to the tip means that there is an extra potential site for P binding on
358 the last molecule of the NC at the 3′ end (Fig. 8). The specific architecture of the pointed 3′ end
359 of the NC could thus facilitate the recognition of the initiation site and assembly of the pre-
360 initiation complex through the interactions of P, L, N, and RNA. We propose a simple model for
361 formation of a pre-initiation complex as shown in Fig. 8. In this model, the first interactions
362 occur between the RNA polymerase complex, L-P, and NC through P’s XD domains (Fig. 1A) in
363 a low affinity interaction with the flexible extended N<sub>TAILS</sub> (5, 31). This transient interaction
364 allows one-dimensional diffusion of the polymerase complex along the NC. The accurate
365 positioning on the tip occurs when the P N-terminal region binds to a vacant NT<sub>arm</sub> binding site

at the NC's 3' end. Binding of P may initiate NC uncoiling as has been observed with mumps
virus (32) to facilitate the release of the genomic RNA 3' end from the RNA-binding groove.
RNA release from the NC 3' end by P is indirectly supported by the ability of the P N-terminal
region to dissociate the N<sub>21-408</sub> assembly where effectively all NT<sub>arm</sub> sites are vacant (Fig. 3D).
Upon RNA 3' end release, it may transiently bind to the exposed positively charged patch on N
by its sugar-phosphate backbone (Fig. 5). The polymerase complex is then positioned for the
entry of the first six nucleotides of the RNA 3' end in to the active site of L (33-35). Bipartite
promoter recognition by the polymerase complex is required for genomic RNA replication (36).
In analogy to mumps virus, this may require further uncoiling of the NC, promoted by P (32).
Elongation will also require NC uncoiling to expose the RNA. The processivity of the
polymerase complex will promote this, and additional P may be injected into the helical
assembly resulting in local NC uncoiling and template RNA exposure according to the cartwheel
model (37). N could be recycled onto the NC once the polymerase complex has passed due to the
transient association with the P CTD. The presence of assembled matrix on the NC during early
stages of infection will necessitate additional disassembly which is as yet not understood.

In conclusion, our MeV N<sup>0</sup>-P structure and its comparison to the previously reported NC state
provides insight into MeV NC assembly and polymerase activity.

Funding Information

This work was supported by the Academy of Finland (grants 139178 and 275199 to S.J.B.), the
Sigrid Juselius Foundation (to S.J.B.), and Biocenter Finland (to S.J.B. and T.K.). The Diamond
Light Source beam line I03 is acknowledged for provision of synchrotron beam time. The
research leading to these results has received funding from the European Community's Seventh
Framework Programme (FP7/2007-2013) under BioStruct-X (grant agreement N°283570). The

funders had no role in study design, data collection and interpretation, or the decision to submit

the work for publication.

Acknowledgments

We thank the following people for their help: Eevakaisa Vesanan for electron microscopy, Juho Kellosalo for SEC-MALLS experiments, Tuula Nyman for mass spectrometry, and the Finnish Instruct National Associate Center, the National Biocenter Finland Cryo-Electron Microscopy Unit, Crystallization Facility, and Proteomics and Metabolomics Unit for the use of their facilities. L.L. was a fellow of the Viikki Doctoral Programme in Molecular Biosciences.

References

1. Gutsche, I., A. Desfosses, G. Effantin, W. L. Ling, M. Haupt, R. W. Ruigrok, C. Sachse, and G. Schoehn. 2015. Near-atomic cryo-EM structure of the helical measles virus nucleocapsid. Science 348:704-707.
2. Liljeroos, L., J. T. Huiskonen, A. Ora, P. Susi, and S. J. Butcher. 2011. Electron cryotomography of measles virus reveals how matrix protein coats the ribonucleocapsid within intact virions. Proc Natl Acad Sci USA 108:18085-18090.
3. Communie, G., T. Crépin, D. Maurin, M. R. Jensen, M. Blackledge, and R. W. Ruigrok. 2013. Structure of the tetramerization domain of measles virus phosphoprotein. J Virol 87:7166-7169.
4. Gely, S., D. F. Lowry, C. Bernard, M. R. Jensen, M. Blackledge, S. Costanzo, J.-M. Bourhis, H. Darbon, G. Daughdrill, and S. Longhi. 2010. Solution structure of the C-terminal X domain of the measles virus phosphoprotein and interaction with the intrinsically disordered C-terminal domain of the nucleoprotein. J Mol Recognit 23:435-447.

5. Jensen, M. R., G. Communie, E. A. Ribeiro, Jr., N. Martinez, A. Desfosses, L.
   Salmon, L. Mollica, F. Gabel, M. Jamin, S. Longhi, R. W. Ruigrok, and M.
   Blackledge. 2011. Intrinsic disorder in measles virus nucleocapsids. Proc Natl Acad Sci
   USA 108:9839-9844.

6. Johansson, K., J.-M. Bourhis, V. Campanacci, C. Cambillau, B. Canard, and S.
   Longhi. 2003. Crystal structure of the measles virus phosphoprotein domain responsible
   for the induced folding of the C-terminal domain of the nucleoprotein. J Biol Chem
   278:44567-44573.

7. Kingston, R. L., D. J. Hamel, L. S. Gay, F. W. Dahlquist, and B. W. Matthews. 2004.
   Structural basis for the attachment of a paramyxoviral polymerase to its template. Proc
   Natl Acad Sci USA 101:8301-8306.

8. Karlin, D., and R. Belshaw. 2012. Detecting remote sequence homology in disordered
   proteins: discovery of conserved motifs in the N-termini of Mononegavirales
   phosphoproteins. PLoS One 7:e31719. doi:31710.31371/journal.pone.0031719.

9. Harty, R. N., and P. Palese. 1995. Measles virus phosphoprotein (P) requires the NH₂-
   and COOH-terminal domains for interactions with the nucleoprotein (N) but only the
   COOH terminus for interactions with itself. J Gen Virol 76:2863-2867.

10. Leyrat, C., F. Yabukarski, N. Tarbouriech, E. A. Ribeiro, Jr., M. R. Jensen, M.
    Blackledge, R. W. Ruigrok, and M. Jamin. 2011. Structure of the vesicular stomatitis
    virus N⁰-P complex. PLoS Pathog 7:e1002248.
    doi:1002210.1001371/journal.ppat.1002248.

11. Yabukarski, F., P. Lawrence, N. Tarbouriech, J. M. Bourhis, E. Delaforge, M. R.
    Jensen, R. W. Ruigrok, M. Blackledge, V. Volchkov, and M. Jamin. 2014. Structure

of Nipah virus unassembled nucleoprotein in complex with its viral chaperone. Nat Struct

Mol Biol 21:754-759.

Kirchdoerfer, R. N., D. M. Abelson, S. Li, M. R. Wood, and E. O. Saphire. 2015. Assembly of the Ebola virus nucleoprotein from a chaperoned VP35 complex. Cell Rep 12:140-149.

Green, T. J., X. Zhang, G. W. Wertz, and M. Luo. 2006. Structure of the vesicular stomatitis virus nucleoprotein-RNA complex. Science 313:357-360.

Carrington, J. C., and W. G. Dougherty. 1988. A viral cleavage site cassette: identification of amino acid sequences required for tobacco etch virus polyprotein processing. Proc Natl Acad Sci USA 85:3391-3395.

Kabsch, W. 2010. Xds. Acta Crystallogr D Biol Crystallogr 66:125-132.

Evans, P. R., and G. N. Murshudov. 2013. How good are my data and what is the resolution? Acta Crystallogr D Biol Crystallogr 69:1204-1214.

McCoy, A. J., R. W. Grosse-Kunstleve, P. D. Adams, M. D. Winn, L. C. Storoni, and R. J. Read. 2007. Phaser crystallographic software. J Appl Crystallogr 40:658-674.

Adams, P. D., P. V. Afonine, G. Bunkóczi, V. B. Chen, I. W. Davis, N. Echols, J. J. Headd, L. W. Hung, G. J. Kapral, R. W. Grosse-Kunstleve, A. J. McCoy, N. W. Moriarty, R. Oeffner, R. J. Read, D. C. Richardson, J. S. Richardson, T. C. Terwilliger, and P. H. Zwart. 2010. PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr D Biol Crystallogr 66:213-221.

Emsley, P., B. Lohkamp, W. G. Scott, and K. Cowtan. 2010. Features and development of Coot. Acta Crystallogr D Biol Crystallogr 66:486-501.

20. Pettersen, E. F., T. D. Goddard, C. C. Huang, G. S. Couch, D. M. Greenblatt, E. C. Meng, and T. E. Ferrin. 2004. UCSF Chimera—a visualization system for exploratory research and analysis. J Comput Chem 25:1605-1612.
21. Krissinel, E., and K. Henrick. 2007. Inference of macromolecular assemblies from crystalline state. J Mol Biol 372:774-797.
22. Sali, A., and T. L. Blundell. 1993. Comparative protein modelling by satisfaction of spatial restraints. J Mol Biol 234:779-815.
23. Pandurangan, A. P., S. Shakeel, S. J. Butcher, and M. Topf. 2014. Combined approaches to flexible fitting and assessment in virus capsids undergoing conformational change. J Struct Biol 185:427-439.
24. Holm, L., and P. Rosenstrom. 2010. Dali server: conservation mapping in 3D. Nucleic Acids Res 38:W545-W549.
25. Okonechnikov, K., O. Golosova, M. Fursov, and the UGENE team. 2012. Unipro UGENE: a unified bioinformatics toolkit. Bioinformatics 28:1166-1167.
26. Spehner, D., R. Drillien, and P. M. Howley. 1997. The assembly of the measles virus nucleoprotein into nucleocapsid-like particles is modulated by the phosphoprotein. Virology 232:260-268.
27. Alayyoubi, M., G. P. Leser, C. A. Kors, and R. A. Lamb. 2015. Structure of the paramyxovirus parainfluenza virus 5 nucleoprotein-RNA complex. Proc Natl Acad Sci USA 112:E1792-1799. doi:1710.1073/pnas.1503941112.
28. Bhella, D., A. Ralph, and R. P. Yeo. 2004. Conformational flexibility in recombinant measles virus nucleocapsids visualised by cryo-negative stain electron microscopy and real-space helical reconstruction. J Mol Biol 340:319-331.

29. Desfosses, A., G. Goret, L. Farias Estrozi, R. W. Ruigrok, and I. Gutsche. 2011. Nucleoprotein-RNA orientation in the measles virus nucleocapsid by three-dimensional electron microscopy. J Virol 85:1391-1395.
30. Noton, S. L., and R. Fearns. 2015. Initiation and regulation of paramyxovirus transcription and replication. Virology 479-480:545-554.
31. Shu, Y., J. Habchi, S. Costanzo, A. Padilla, J. Brunel, D. Gerlier, M. Oglesbee, and S. Longhi. 2012. Plasticity in structural and functional interactions between the phosphoprotein and nucleoprotein of measles virus. J Biol Chem 287:11951-11967.
32. Cox, R., A. Pickar, S. Qiu, J. Tsao, C. Rodenburg, T. Dokland, A. Elson, B. He, and M. Luo. 2014. Structural studies on the authentic mumps virus nucleocapsid showing uncoiling by the phosphoprotein. Proc Natl Acad Sci USA 111:15208-15213.
33. Liang, B., Z. Li, S. Jenni, A. A. Rahmeh, B. M. Morin, T. Grant, N. Grigorieff, S. C. Harrison, and S. P. Whelan. 2015. Structure of the L Protein of Vesicular Stomatitis Virus from Electron Cryomicroscopy. Cell 162:314-327.
34. Butcher, S. J., J. M. Grimes, E. V. Makeyev, D. H. Bamford, and D. I. Stuart. 2001. A mechanism for initiating RNA-dependent RNA polymerization. Nature 410:235-240.
35. Salgado, P. S., E. V. Makeyev, S. J. Butcher, D. H. Bamford, D. I. Stuart, and J. M. Grimes. 2004. The structural basis for RNA specificity and Ca2+ inhibition of an RNA-dependent RNA polymerase. Structure 12:307-316.
36. Tapparel, C., D. Maurice, and L. Roux. 1998. The activity of Sendai virus genomic and antigenomic promoters requires a second element past the leader template regions: a motif (GNNNNN)3 is essential for replication. J Virol 72:3117-3128.

37. Curran, J. 1998. A role for the Sendai virus P protein trimer in RNA synthesis. J Virol 72:4274-4280.

Figure Legends

Figure 1. Protein constructs and N<sup>0</sup>-P complex analysis. (A) Domain structure of measles N and P proteins and protein constructs used in this study. Numbers refer to the amino acid position. (B) Co-expressed N and GST-H<sub>6</sub>-P constructs after elution from glutathione sepharose beads. Lane 1: marker; 2: P<sub>1-38</sub> and N<sub>1-408</sub>; 3: P<sub>1-48</sub> and N<sub>1-408</sub>; 4: P<sub>1-58</sub> and N<sub>1-408</sub>; 5: P<sub>1-68</sub> and N<sub>1-408</sub>; 6: P<sub>1-78</sub> and N<sub>1-408</sub>; 7: GST-H<sub>6</sub> and N<sub>1-408</sub>. (C) Electron microscopy of negatively stained N<sub>1-408</sub><sup>0</sup>—GST-H<sub>6</sub>-P<sub>1-48</sub> heterocomplex (left) and flow-through from the glutathione sepharose beads containing NC-like helical particles (right). Images are at the same scale. Scale bar 100 nm. (D) Data from the SEC-MALLS experiment with the N<sub>1-408</sub><sup>0</sup>—P<sub>1-48</sub> heterocomplex after release from Ni-IMAC by enterokinase. The solid line is the absorbance profile, blue crosses show the molecular mass distribution. For comparison UV<sub>280</sub> profile of N<sub>21-408</sub><sup>0</sup>-P<sub>1-48</sub> chimera is shown with a dashed line.

Figure 2. Crystal structure of MeV N<sup>0</sup>-P complex. Cartoon representation of the chimeric N<sub>21-408</sub><sup>0</sup>-P<sub>1-48</sub> structure. (A) The crystallized dimer. Monomer 1 is sky-blue and orange; monomer 2 is blue-green and orange-red. (B) The interaction of monomer 1N (sky-blue) and monomer 2P (orange-red) fragments composing one N<sup>0</sup>-P heterocomplex. Secondary structure elements are labelled.

Figure 3. Hydrophobic interactions in N<sup>0</sup>-P binding. (A) View of N<sub>21-408</sub>-P<sub>1-48</sub> binding interface in cartoon representation. The P (orange-red) residues interacting with N (sky-blue) are shown in stick representation and labelled. Residues conserved throughout the Paramyxovirinae are

labeled as follows: violet, acidic; green, polar; blue, hydrophobic; orange, glycine (8). (B)

Alignment of P N-termini of MeV, NiV, and PIV5. Asterisks denote residues making contacts

with N⁰. Conserved residues are shaded in color. (C) SDS-PAGE of protein released from N₁₋₄₀₈

— GST-H₆-P₁₋₄₈ heterodimer complex bound to Ni-IMAC beads when subjected to different

conditions. Lane 1: marker; 2: 0M NaCl; 3: 0.5M NaCl; 4: 1M NaCl; 5: 2M NaCl; 6: 0.5M

KCl; 7: 1M KCl; 8: 2M KCl; 9: 2M urea; 10: 4M urea; 11: 8M urea; 12: 0.1% Triton-X 100;

13: 1% Triton-X 100. (D) SDS-PAGE of pull-down of N₂₁₋₄₀₈ by GST-H₆-P₁₋₄₈ and its mutants.

Lane 1: markers; 2: control N₂₁₋₄₀₈ only; 3: control GST-H₆-P₁₋₄₈wt only; 4: N₂₁₋₄₀₈ + GST-H₆-P₁-

48wt; 5: N₂₁₋₄₀₈ + GST-H₆-P₁₋₄₈L13D; 6: N₂₁₋₄₀₈ + GST-H₆-P₁₋₄₈L13A; 7: N₂₁₋₄₀₈ + GST-H₆-P₁₋₄₈I16D;

8: N₂₁₋₄₀₈ + GST-H₆-P₁₋₄₈L19E; 9: N₂₁₋₄₀₈ + GST-H₆-P₈₋₄₈. Note that some of the P mutations

affected the electrophoretic mobility of the tagged constructs due to impaired SDS binding to the

protein molecules with changed net charge (compare lanes 5 and 7 with lane 3). The deletion

construct P₈₋₄₈ migrates faster reflecting its shorter amino acid sequence.

Figure 4. MeV N⁰-P vs. NC structure. Tubes and planks representation of overlays of N⁰-P (N,

sky-blue; P, orange-red) and NC model (light grey) with their CTDs (A) or NTDs (B) aligned.

(C) Cartoon representation close-up view of hinge region from (A). (D) Cartoon representation

close-up view of αN6 helix from (B). Arrow on the right panel shows hypothetical turn direction

of αN6 helix upon RNA binding.

Figure 5. Potential alternative RNA-binding site. (A) Three consecutive N protomers of MeV NC, with the second protomer in a surface representation. The orientation is such that the outer surface of the nucleocapsid is facing the viewer. (B) \(N^0\) surface representation in the same orientation as the second N protomer shown in panel (A). Missing sidechains in \(N^0\) were added manually. Surface models are colored according to the electrostatic surface charge (positive, blue; negative, red). The positively-charged patch in panel (B) is highlighted by a black box. Scale bar shows the electrostatic charge values.

Figure 6. P interferes with NC assembly. The \(P_{1-48}\) fragment overlaps with both the \(NT_{\text{arm}}\) and \(CT_{\text{arm}}\) of NC. Cartoon representation of superposed \(N^0\)-P (\(N^0\), sky-blue, P, orange-red) and NC (grey; PDB ID: 4UFT), \(NT_{\text{arm}}\) of the \(N_{i-1}\) protomer (yellow) and \(CT_{\text{arm}}\) of the \(N_{i+1}\) protomer (pink). Molecules were aligned using the \(N_{\text{CTD}}\) domains.

Figure 7. Phylogenetic analysis of N structures. Phylogenetic comparison of the MeV \(N^0\) structure with the other published Mononegavirales nucleoprotein structures as a non-rooted tree. The scale bar corresponds to the number of expected substitutions per amino acid site between nodes. Atomic models are aligned by their CTDs and shown in ribbon representation. PDB IDs are given above the virus acronym.

Figure 8. Schematic model of RNA synthesis pre-initiation complex formation. (A) Model of non-specific binding of the polymerase complex to NC. The NC is composed of genomic RNA (black ribbon), and the N protomers with the \(N_{\text{CORE}}\) domains as sky-blue peanuts, \(NT_{\text{arm}}\) in yellow, \(N_{\text{TAL}}\) in pink. A complex of polymerase subunit L (grey) with the P tetramer (orange-red) binds the NC through the interaction of P's XD domains with MoRE motifs in \(N_{\text{TAL}}\). There is a vacant P/NT\(_{\text{arm}}\) site on the 3' protomer of the NC (dark blue crescent). (B) Model of the pre-initiation complex formation. One of the P N-termini occupies the P/NT\(_{\text{arm}}\) site on the 3'

566 protomer of the NC. Binding results in release of the RNA 3' end, which is then positioned in the
567 polymerase catalytic site to form a pre-initiation complex.
568
569

Table 1. Data collection and refinement statistics.

| Synchrotron radiation source | Diamond Light Source beamline I03 |
| Wavelength (Å)              | 0.9763                          |
| Resolution range (Å)         | 78.9-2.71 (2.807-2.71)*          |
| Space group                  | P3₁21                           |
| Unit cell                    | a=91.14, b=91.14, c=94.21, α=β=90°, γ=120° |
| Total reflections            | 63741                           |
| Unique reflections           | 12656                           |
| Multiplicity                 | 5.1(4.9)*                       |
| Completeness (%)             | 99.8(99.5)*                     |
| Mean I/sigma(I)              | 7.5(1.6)*                       |
| R-merge                      | 11.2(78.3)*                     |
| Reflections used for R-free  | 700                             |
| R-work (%)                   | 21.1                            |
| R-free (%)                   | 26.6                            |
| Number of non-hydrogen atoms |                                 |
| macromolecules               | 2975                            |
| water                        | 9                               |
| Protein residues             | 407                             |
| RMSD (bonds) (Å)             | 0.005                           |
| RMSD (angles) (°)            | 0.976                           |
| Ramachandran favored (%)     | 90.52                           |
| Ramachandran outliers (%)    | 1.75                            |

Average B-factor (Å²)
protein 68.9
solvent 49.6

*Statistics for the highest-resolution shell are shown in parentheses.

Table 2. RMSD values between N structures of MeV N°-P and MeV NC, NiV N°-PorPIV4N-
RNA complexes.

| MeV N° domain | RMSD MeV N°-P vs. MeV NC | RMSD MeV N°-P vs. NiV N°-P | RMSD MeV N°-P vs PIV5N-RNA |
|---------------|---------------------------|-----------------------------|------------------------------|
| NTD(31-265)    | 164Ca pairs: 1.2 Å        | 149Ca pairs: 1.0 Å          | 130Ca pairs: 1.2 Å           |
|               | 220Ca pairs: 2.7 Å        | 215Ca pairs: 2.6 Å          | 221Ca pairs: 3.2 Å           |
| CTD(266-372)   | 85Ca pairs: 1.1 Å         | 83Ca pairs: 0.8 Å           | 91Ca pairs: 0.9 Å            |
|               | 107Ca pairs: 1.9 Å        | 107Ca pairs: 1.7 Å          | 107Ca pairs: 1.8 Å           |

The first record in each cell shows the number of Ca pairs used for alignment in UCSF Chimera
and the corresponding RMSD value; the second record shows the number of all possible Ca pairs
and their corresponding RMSD value. PDB IDs and resolutions: MeV NC, 4UFT, 4.3 Å; NiV
N°-P, 4CO6, 2.5 Å; PIV5N-RNA, 4XJN, 3.11 Å.

A

N_CORE
1 36 265 372 391 488 499 525
NTD CTD CT_arm N_TAIL
N NT_arm MoRE
N1-408
N21-408
P 1 25 304 376 450 507
soyuz1 MD XD
GST-H6 P1-38
P1-48
P1-58
P1-68
P1-78
N-P H6-TEV N21-408 P1-48
B
1 2 3 4 5 6 7
55-
43- N1-408
C
D
Normalized UV
absorbance at 280 nm
1.0-
0.5-
0.0-
10 15 Elution volume (ml)
Molecular mass (kDa)
100
10
52±2 kDa

A

B
α_P2
n_N2
n_N3
N_CTD
α_N11
α_N13
β_N2 α_N2 α_N3
β_N1
N_NTD
α_N4
β_N3
α_N1
α_N5
α_N6
α_N9
n_N1
α_N7
α_N8
α_N12
α_N14
α_N10
α_P1
n_N5
n_N4
α_N15
α_N16
α_N17
P

A

B
MeV P₁₋₂₂ MA EE QAR H V KNG L ECIR AL KAE
NiV P₁₋₂₀ MD KLE LV N D GLN II DF IQKN
PIV5 P₉₋₃₀ SP DE INK LI ET GL NT VE YF TSQ
* * * * * * * * * * * * *

C
66-
55-
43-
35-
27-
1 2 3 4 5 6 7 8 9 10 11 12 13
-N₁₋₄₀₈

D
50-
40-
30-
1 2 3 4 5 6 7 8 9
-N₂₁₋₄₀₈
GST-P

A
B

C
αP1
Q5
M1
V262
αN9
αN10
E263
E263
V262
αN9

D
Y260
W196
R195
αN6
Y199
90°
Y260
R195
W196
Y199
W196

![](image.png)

$\alpha_{P2}$  
$CT_{\text{arm}}$  
$i+1$  

$\alpha_{P1}$  
$NT_{\text{arm}}$  
$i-1$  

$N_{i+1}$  
$N_{i}$  
$N_{i-1}$

4ZTA
EboV

2GTT
RabV

3PMK
VSV

2WJ8
RSV

1N93
BDV

4CO6
NiV

4XJN
PIV5

MeV N°

A

L
3'
5'
XD
XD
XD
XD

B

L
3'
5'
XD
XD
XD
XD
