
Review

# Molecular Studies of HTLV-1 Replication: An Update

Jessica L. Martin¹, José O. Maldonado², Joachim D. Mueller³, Wei Zhang⁴ and Louis M. Mansky⁵,*  

1. Institute for Molecular Virology, Pharmacoinmunology Training Program & Pharmacology Graduate Program, University of Minnesota, 18-242 Moos Tower, 515 Delaware Street SE, Minneapolis, MN 55455, USA; mart3243@umn.edu  
2. Institute for Molecular Virology & DDS-PhD Dual Degree Program, University of Minnesota, 18-242 Moos Tower, 515 Delaware Street SE, Minneapolis, MN 55455, USA; jmaldo@umn.edu  
3. Institute for Molecular Virology & School of Physics and Astronomy, University of Minnesota, 18-242 Moos Tower, 515 Delaware Street SE, Minneapolis, MN 55455, USA; mueller@physics.umn.edu  
4. Institute for Molecular Virology, School of Dentistry & Characterization Facility, University of Minnesota, 18-242 Moos Tower, 515 Delaware Street SE, Minneapolis, MN 55455, USA; zhangwei@umn.edu  
5. Institute for Molecular Virology, School of Dentistry & Pharmacology Graduate Program, University of Minnesota, 18-242 Moos Tower, 515 Delaware Street SE, Minneapolis, MN 55455, USA  

* Correspondence: mansky@umn.edu; Tel.: +1-612-626-5525; Fax: +1-612-626-5515  

Academic Editor: Eric O. Freed  

Received: 25 November 2015; Accepted: 18 January 2016; Published: 27 January 2016  

---

**Abstract:** Human T-cell leukemia virus type 1 (HTLV-1) was the first human retrovirus discovered. Studies on HTLV-1 have been instrumental for our understanding of the molecular pathology of virus-induced cancers. HTLV-1 is the etiological agent of adult T-cell leukemia (ATL) and can lead to a variety of neurological pathologies, including HTLV-1-associated myelopathy/tropical spastic paraparesis (HAM/TSP). The ability to treat the aggressive ATL subtypes remains inadequate. HTLV-1 replicates by (1) an infectious cycle involving virus budding and infection of new permissive target cells and (2) mitotic division of cells harboring an integrated provirus. Virus replication initiates host antiviral immunity and the checkpoint control of cell proliferation, but HTLV-1 has evolved elegant strategies to counteract these host defense mechanisms to allow for virus persistence. The study of the molecular biology of HTLV-1 replication has provided crucial information for understanding HTLV-1 replication as well as aspects of viral replication that are shared between HTLV-1 and human immunodeficiency virus type 1 (HIV-1). Here in this review, we discuss the various stages of the virus replication cycle—both foundational knowledge as well as current updates of ongoing research that is important for understanding HTLV-1 molecular pathogenesis as well as in developing novel therapeutic strategies.

**Keywords:** deltaretrovirus; antiretroviral; lentivirus

---

## 1. Introduction

Human T-cell leukemia virus type 1 (HTLV-1) was independently discovered in 1980 by two research groups and identified as the etiological agent of adult T-cell leukemia (ATL) [1,2]. As the first human retrovirus discovered, research on HTLV-1 laid the foundational framework for subsequent studies of human immunodeficiency virus type 1 (HIV-1), infectious causes of cancer, and the molecular mechanisms of leukemogenesis [3].

Shortly after the discovery of HTLV-1, another human retrovirus was discovered—human T-cell leukemia virus type 2, HTLV-2—which closely resembled HTLV-1 in genome structure and nucleotide sequence [4]. Unlike HTLV-1, HTLV-2 has not been convincingly associated with human pathology. Nevertheless, both HTLV-1 and HTLV-2 are included in worldwide prevalence estimates. Historically,

Viruses 2016, 8, 31; doi:10.3390/v8020031 www.mdpi.com/journal/viruses

it has been estimated that 15–20 million people are infected worldwide [5,6]. A more recent study has estimated the number closer to 5–10 million, with the majority of these individuals residing in Japan and the Caribbean Basin [7]. A third and fourth type of HTLV, human T-cell leukemia virus type 3 (HTLV-3) and human T-cell leukemia virus type 4 (HTLV-4), have been discovered in central Africa in the past decade; both are closely related to HTLV-1, and likely share similarities in replication, pathogenesis and transmission [8,9].

HTLV-1 is the etiological agent of ATL as well as a variety of neurological pathologies, primarily HTLV-1-associated-myelopathy/tropical spastic paraparesis (HAM/TSP) [10]. Both ATL and HAM/TSP have a low incidence among HTLV-1 carriers. It is thought that approximately 2%–6% of patients infected with HTLV-1 will acquire either pathology [11,12]. ATL generally presents after a long latency in patients infected during childhood. This is in contrast to HAM/TSP, which is associated with infection later in life [13].

ATL is an aggressive malignancy of the peripheral T-cells and can be divided into four subtypes—acute, lymphomatous, chronic, or smoldering. Patients with the acute form of ATL have a prognosis of approximately 6 months—an estimate that has not significantly changed since the discovery of the disease, despite advances in treatments [14]. Current recommended therapies for ATL include chemotherapy, monoclonal antibodies, allogeneic bone marrow transplants, and a combination of interferon-α (IFN-α) and azidothymidine (AZT) [15–18]. Interestingly, the mechanism of action of the combination of IFN-α and AZT appears to correlate with an induction of cell apoptosis by phosphorylation of p53 [19].

HAM/TSP is characterized by spasticity and weakness of the legs along with urinary disturbances [19]. The primary pathology of HAM/TSP is associated with HTLV-1 infection in the spinal cord leading to inflammation. Unlike ATL, which appears to have a complex and multi-faceted pathology, the incidence of HAM/TSP has been shown to correlate with HTLV-1 proviral loads as well as the site of proviral integration [20,21]. Treatment of HAM/TSP is symptom-based and includes antispasmodic and anti-inflammatory medications [22].

Research into the HTLV-1 life cycle to date has been essential in the discovery and development of better therapeutic strategies. Here in this review, we highlight what is currently known as well as recent advances in the study of HTLV-1 replication. The recent advances help to provide further reason for hope in effective therapeutic options for HTLV-1-infected individuals.

## 2. HTLV-1 Infectious Replication Cycle

### 2.1. Attachment and Fusion

HTLV-1 primarily infects CD4⁺ T-cells but has the potential to infect a wide variety of cells, including CD8⁺ T-cells, B-lymphocytes, endothelial cells, myeloid cells, fibroblasts, as well as other mammalian cells [23–27]. This wide variety of target cells is due in part to the ability of the surface subunit (SU) of the HTLV-1 envelope glycoprotein (Env) to interact with three widely distributed cellular surface receptors including the glucose transporter (GLUT1) [28], heparin sulfate proteoglycan (HSPG) [29], and the VEGF-165 receptor neuropilin-1 (NRP-1) [30]. Once HTLV-1 has attached to the cell, the membrane fusion process occurs by a series of proposed sequential events between SU and the target cell receptor proteins (Figure 1A,B) [30,31]. Briefly, the HTLV-1 Env interacts with HSPG first followed by NRP-1, which results in the formation of a complex. Following this event, GLUT1 associates with the HSPG/NRP-1 complex to initiate the fusion process, through interactions with the HTLV-1 Env transmembrane (TM) protein, which allows for the HTLV-1 capsid (CA) core containing the viral genome and viral proteins to be released into the cytoplasm of the permissive target cell (Figure 1B).

Viruses 2016, 8, 31

Figure 1. HTLV-1 life cycle. The major steps in the life cycle of HTLV-1 are shown. A mature, infectious HTLV-1 virion attaches and fuses to the target cell membrane through interaction with the target cell surface receptors GLUT1/HSPG/NRP-1 via the HTLV-1 envelope surface and transmembrane domains of the envelope (Env) protein (A). Following fusion, the viral core containing the viral genomic RNA (gRNA) is delivered into the cytoplasm (B), and during and/or following entry the gRNA genome undergoes reverse transcription to convert the gRNA into double-stranded DNA (dsDNA) (C). The dsDNA is then transported into the nucleus (D), and it is integrated into the host genome; (E,F). The provirus is then transcribed by cellular RNA polymerase II (G), as well as post-transcriptionally modified (H). Both full-length and spliced viral mRNAs are exported from the nucleus to the cytoplasm (I). The viral proteins are then translated by the host cell translation machinery (J), and the Gag, Gag-Pol and Env proteins transported to the plasma membrane (PM) along with two copies of the gRNA genome (K). These viral proteins and gRNA assemble at a virus budding site along the PM to form an immature virus particle (L). The budding particle releases from the cell surface (M), and undergoes a maturation process through the action of the viral protease, which cleaves the viral polyproteins to form an infectious, mature virus particle (N).

It has been demonstrated that GLUT1 plays a key role in both the binding of the SU and the infection of CD4⁺ cells [32]. Paradoxically, other retroviruses have mechanisms that decrease surface expression of their receptors, such as HIV-1 Nef and Vpu [33,34]. The decrease of receptor expression on the cell surface is thought to prevent both superinfection and intracellular Env-receptor interactions, which can inhibit proper proteolytic processing of the Env precursor polyprotein. HTLV-1 does not encode for an accessory protein that reduces surface expression of GLUT1, and it is therefore unclear how HTLV-1 modulates plasma membrane receptor expression. However, it has been recently shown that HTLV-1-based virus-like particles (VLPs) produced in cells with high levels of GLUT1 were better able to fuse with target cells than those produced from cells with low levels of GLUT1 [35]. In 293T cells,

HTLV-1 Env avoids interaction with GLUT1 through the separate intracellular localization of GLUT1 and Env [35]. This recent observation is important because it suggests that separate intracellular localization of GLUT1 and HTLV-1 Env is required for proper fusion activity of the HTLV-1 Env. This study may also have implications for HTLV-1 cellular tropism, as CD4⁺ regulatory T-cells, the primary viral reservoir for HTLV-1-infected individuals, express GLUT1 at low levels as compared to other types of CD4⁺ T-cells [36].

### 2.2. Reverse Transcription, Nuclear Transport and Integration

The HTLV-1 CA core enters the infected cell and contains two copies of the viral genomic RNA (gRNA) along with reverse transcriptase (RT), integrase (IN), and the viral protease (PR). Reverse transcription of HTLV-1 RNA to double-stranded DNA (dsDNA) has not been extensively studied but likely occurs after virus entry (Figure 1C) [37,38]. It is thought that HIV-1 reverse transcription is linked to intracellular uncoating of the CA core [39]. Additionally, HIV-1 RT and IN interactions have been shown to be necessary for production of early reverse transcription products [40]. Complementary studies with HTLV-1 have yet to be done, so it is unclear whether HTLV-1 CA uncoating correlates with reverse transcription or if RT-IN interactions occur during early reverse transcription. Recombination can occur during reverse transcription, and recent evidence from phylogenetic analyses strongly suggests that recombination played a distinct role in emergence of HTLV-1 in the human population approximately 4000 years ago [41].

Unlike HIV-1, which is highly sensitive to the effects of the APOBEC family of cytidine deaminases, HTLV-1 appears less sensitive APOBECs. There is some evidence that APOBEC3G may lead to G-to-A hypermutation in some HTLV-1 sequences *in vivo* [42,43], but the overall effect on HTLV-1 sequence diversity appears to be negligible—perhaps due to the propensity of HTLV-1 to be propagated by clonal expansion of infected cells rather than replication via reverse transcription. HTLV-1 has been previously shown to prevent APOBEC3G packaging through an element at the C-terminal nucleocapsid (NC) region of Gag [44].

The partially disassembled core containing the reverse transcription complex (preintegration complex) is translocated to the nucleus (Figure 1D) where integration into the host cell chromosome occurs to form the provirus (Figure 1E,F). It has been found that HTLV-1 integrates into the genome in the absence of preferred sites [45–50]. Such studies have analyzed hundreds of thousands of HTLV-1 integration sites [51,52] and have not been able to identify HTLV-1 proviral integration site hotspots. Interestingly, in HTLV-1-induced disease states, the integration sites of HTLV-1 become non-random. For example, it was recently demonstrated that the clinical diagnosis of HAM/TSP correlates with proviral integration into transcriptionally active regions [53].

### 2.3. Viral Gene Transcription

The long terminal repeats (LTRs) of the HTLV-1 provirus contain the necessary promoter and enhancer elements to initiate RNA transcription (Figure 1G), with the polyadenylation signal located in the 3′ LTR [1]. Tax, a non-structural protein and the main driver of viral transcription, potently activates viral transcription during the early phase of infection by recruiting multiple cellular transcription factors [54]. Three conserved 21-bp repeat elements, known as the Tax-responsive element 1 (TRE-1), bind the cyclic AMP response element binding protein (CREB) at the TRE-1 site through its N-terminus (NTD) [55–61], while the C-terminal domain (CTD) of Tax is believed to promote the transcriptional initiation and RNA polymerase elongation by directly interacting with the TATA binding protein [5,62]. The Tax-CREB promoter complex recruits the multifunctional cellular coactivators CREB binding protein (CBP), p300, and the p300/CBP-associated factor to the LTR [63–68].

Recently, several host factors that directly interfere with HTLV-1 viral transcription have been identified. TCF1 and LEF1 are transcription factors specifically found in T-cells. They antagonize Tax activity through physical association with Tax, preventing transcription of the viral proteins. In most HTLV-1-infected cell lines, however, TCF1 and LEF1 expression is low due to downregulation via

STAT5a, which is activated by Tax [69]. The host protein SIRT1 deacetylase has also been shown to downregulate HTLV-1 viral transcription by inhibiting Tax. Unlike TCF1 and LEF1, SIRT1 appears to inhibit Tax-CREB interactions. Interestingly, the well-known SIRT1 activator resveratrol significantly decreases the transmission of HTLV-1 produced from MT2 cells [70,71]. This suggests that resveratrol may be a potential therapeutic option for patients infected with HTLV-1 or a prophylactic option to prevent virus transmission. In addition to these cellular host factors, the facilitate chromatin transcription (FACT) proteins SUPT16H and SSRP1 have been shown to inhibit both HTLV-1 and HIV-1 transcription by preventing interaction of HTLV-1 Tax and HIV-1 Tat with their respective viral LTRs [72].

### 2.4. Post-Transcriptional Regulation

Rex is a positive post-transcriptional regulator essential for splicing and transport of HTLV-1 mRNA (Figure 1H,I). Rex specifically interacts with the U3 and R regions of the HTLV-1 gRNA known as the Rex-responsive element (RexRE). During the early stages of viral gene transcription, suboptimal levels of Rex are present [73], which results in the exclusive export of doubly spliced (tax, rex, p30II, p12, p13, and hbz) viral mRNAs to the cytoplasm (Figure 1I) [74]. Once Rex accumulates in the nucleus, Rex reduces splicing of viral mRNA and the singly spliced (env) and unspliced (gag-pro-pol) mRNAs are then exported from the nucleus to the cytoplasm leading to the production of enzymatic and structural proteins (Figure 1J) [74]. Rex binds to the RexRE through a highly basic RNA-binding NTD, while the CTD is important for protein oligomerization [75,76]. Rex also contains an activation domain containing the nuclear export signal, which targets Rex to the nuclear pore complex in order for Rex to move between the nucleus and cytoplasm [77,78].

Despite the presence of host cell mechanisms to export doubly spliced RNA, all HTLV-1 mRNA transcripts, including those that are doubly spliced, have RexREs present. A recent study has shown that Rex may have a CRM1-dependent role in nuclear export of all HTLV-1 mRNAs, even the doubly spliced mRNAs that should be exported via host cell mechanisms [79]. These observations suggest that viral mRNA export is under a more complex regulation than previously thought. Furthermore, another recent study has suggested that there are three alternatively spliced HTLV-1 transcripts that encode for novel Rex isoforms, which may also contribute to the regulation of HTLV-1 protein expression levels [80].

### 2.5. Viral Protein Translation

As soon as HTLV-1 mRNAs are exported to the cytoplasm, the host protein-synthesis machinery translates the viral proteins. Presumably, the full-length viral gRNA is either translated or trafficked to the plasma membrane, where it can dimerize, interact with the Gag polyprotein, and be packaged into assembling particles (Figure 1K,L) [81]. The doubly spliced and unspliced mRNAs are translated by free ribosomes to express the enzymatic and structural proteins, respectively, while the singly spliced mRNA is translated by membrane-bound ribosomes to express Env [45].

Many RNA viruses use a cap-independent mechanism to recruit the 40S ribosomal subunit to an internal ribosome entry segment (IRES) within the 5′UTR of the mRNA, which allows for ribosomal scanning and protein translation to occur [82–84]. It was thought that HTLV-1 mRNA contains an IRES element [85] used for the translation of the Gag protein, but another study suggests that a 5′ proximal post-transcriptional control element modulates post-transcriptional HTLV-1 gene expression by interacting with the host RNA helicase A instead of an IRES element, implying that the translation of the HTLV-1 mRNA is cap-dependent [86]. Interestingly, a recent study has demonstrated that HTLV-1 translation is inhibited by the drug edeine, a cap-independent translation inhibitor, suggesting that an IRES element in the 5′UTR recruits the ribosome to the mRNA [87]. Obviously, more research is needed to firmly establish the mechanism(s) used by HTLV-1 to translate its viral proteins.

Viruses 2016, 8, 31

## 2.6. Gag and Viral RNA Trafficking

Viral particle formation occurs after Gag traffics from the cytoplasm to the plasma membrane (PM) (Figure 1K). How HTLV-1 Gag translocates from the site of translation to the membrane is poorly understood. However, it is known that monomeric forms of HTLV-1 Gag exist in the cytoplasm and are detected at the membrane shortly after the initiation of viral protein translation [88]. This is in contrast to HIV-1 Gag, where low ordered oligomers are observed in the cytoplasm until micromolar concentrations are reached prior to detecting oligomeric Gag at the plasma membrane [88]. HIV-1 Gag interacts with many cellular proteins, including cytoskeleton-associated proteins, though their relationship to HIV-1 Gag trafficking is unclear [89]. HTLV-1-infected cells regulate cytoskeletal polarization [90], though it is unclear if this is related to Gag trafficking to the plasma membrane.

HTLV-1 Gag nucleocapsid (NC) protein binds to HTLV-1 RNA relatively weakly as compared to that of other retroviral NC proteins, due in part to the anionic carboxy-terminal domain (CTD) of the HTLV-1 NC [91]. The HTLV-1 MA has been recently reported to bind RNA, and it was found that HTLV-2 MA binds RNA at much higher affinity than HTLV-2 NC [92]. This is in direct contrast to HIV-1, in which NC binds to RNA more strongly than HIV-1 MA [92]. These recent findings highlight the importance of both the MA and NC domains in viral RNA interactions that are likely critically important for viral gRNA recognition and gRNA packaging. How HTLV-1 RNA traffics through the cytoplasm in order to get to the plasma membrane (and to virus budding sites) is poorly understood, but a recent study with HIV-1 gRNA suggests that the viral gRNA diffuses through the cytoplasm to the membrane [93]. It is formally possible that HTLV-1 gRNA also diffuses through the cytoplasm to reach the membrane, but it could also bind to Gag before reaching the membrane (Figure 1I–K). There is a significant need for future studies in order to better understand these aspects of HTLV-1 replication.

## 2.7. Assembly, Budding and Maturation

Gag-gRNA, Gag-Gag and Gag-membrane interactions are all required for the assembly and budding of virus particles (Figure 1L) [94]. Gag forms higher order oligomers by oligomerizing with other Gag molecules through interactions primarily involving the CA domain and to some extent the NC domain [95–100]. Once at the PM, virus budding sites are identified and are characterized by the interaction of HIV-1 MA with lipid-rich [101] assembly sites known as lipid rafts [102–104]. Membrane binding of HIV-1 Gag is dependent upon interaction of MA with phosphatidylinositol-(4,5)-bisphosphate PI(4,5)P₂ [105]. HTLV-1 Gag has been shown to not have a preference for binding to PI(4,5)P₂, which has implications for how HTLV-1 Gag targets the PM and identifies virus budding sites [105]. Cellular factors are also recruited to the virus budding sites, resulting in budding and subsequent release of immature virus particles (Figure 1L,M) [100,106,107]. The viral protease (PR) cleaves the Gag and Pol polyproteins during and shortly after the release of immature virus particles (Figure 1N) [108]. MA remains closely associated with the PM; CA forms a capsid shell that contains reverse transcriptase, integrase and the NC-coated gRNA. The mature virus particle, if infectious, is capable of infecting a permissive target cell (Figure 1N) [109].

## 3. HTLV-1 Transmission

### 3.1. Inter-Host Transmission

There are generally three modes of inter-host HTLV-1 transmission described: (1) blood and blood products, (2) vertical or (3) sexual transmission [110], but the main mode of transmission is thought to be vertical, *i.e.* from mother-to-child through breastfeeding [111]. Mother-to-child transmission rates vary from 5% to 27% for children nursed by infected mothers and correlate with the duration of breastfeeding [112,113]. While it is not clear precisely how infection occurs through the mucosal and epithelial barriers of the gastrointestinal tract, it is thought that infected lymphocytes in breast milk carry the virus into the gut [114]. Once in the gut, either cell-free virus or cells carrying the virus must pass through the epithelium. A recent study demonstrated *in vitro* that cell-free HTLV-1 may cross

the epithelial barrier via transcytosis before infecting subepithelial dendritic cells [115]. The precise mechanism of transcytosis for HTLV-1 remains unclear. However, studies with HIV-1 have shown that transcytosis across vaginal epithelial cells occurs via the endocytic recycling pathway [116]. It is plausible that other mechanism(s) are involved in HTLV-1 infection across the gut epithelial barrier due to the low infectivity of cell-free virus. While cell-free HIV-1 is generally thought to be much more infectious than cell-free HTLV-1, it has been suggested that HIV-1-infected lymphocytes more efficiently infect target cells in the gut than cell-free virus—possibly through the formation of a viral synapse that induces transcytosis [117]. The role of the virological synapse in these transmission events has not been carefully studied. It is also not known whether HTLV-1 infected lymphocytes can transmigrate as a whole cell across the epithelial barrier and infect subepithelial immune cells.

Zoonotic transmission events of simian T-cell leukemia virus type 1 (STLV-1) to humans after contact with nonhuman primates through bites or bushmeat slaughtering still occur in Africa, establishing the emergence of new HTLV-1 infections in humans. A recent study found that more than 8% of individuals bitten by nonhuman primates in Africa are infected with HTLV-1, and virus transmission cannot be attributed to mother-to-child transmission [118]. The strains of HTLV-1 found in those infected closely resembled the subtypes of STLV-1 commonly found in the primate species from which they were bitten [118,119]. In fact, it is likely that the emergence of HTLV-3 and HTLV-4 may be attributable to recent STLV zoonotic transmission events, as STLV-4 is known to be endemic in African gorillas, and phylogenetic analyses have shown that HTLV-4 is not an ancient human virus but recently emerged in the human population [120]. While these findings highlight the potential ongoing role of nonhuman primates as virus reservoirs, they also highlight interest in the virus-host interactions that facilitate cross-species transmission as well as potential risks in transmission and emergence of more highly pathogenic types of HTLV. While monkeys in Japan also harbor STLV-1 strains [121], those strains are more highly divergent from the HTLV-1 strains in Japanese patients, indicating that zoonotic transmission of HTLV-1 may not be a major public health threat in regions outside of Africa [122].

### 3.2. Cell-to-Cell Transmission

In general, there are two distinct methods of virus transmission between cells: virus infection of cells in the absence of cell-to-cell contacts and virus infection involving cell-to-cell contacts. Most retroviruses can efficiently infect target cells in the absence of cell-to-cell contacts—in which the virus buds from the cell and infects a target cell through diffusion. HTLV-1 is notorious for being poorly infectious in the absence of direct cell-to-cell transmission, and co-cultivation of permissive target cells with virus-producing cells are the most effective means of virus transmission [123].

### 3.3. Virological Synapses

Immunofluorescence and confocal microscopy were used previously to demonstrate that Gag and Env proteins are more evenly distributed in isolated T-cells, but once the cell comes into contact with another cell, cell polarization occurs—impacting the localization of HTLV-1 Gag, Env and the genomic RNA towards the cell-cell junction. This cell-to-cell junction, termed the virological synapse (VS), shares many features with the previously described immunological synapse, which includes features such as ordered talin domains and microtubule organizing center (MTOC) polarization [124]. Cryoelectron tomography studies of HTLV-1 associated VS structures suggest that there is no fusion of the cell membranes [125]. To the contrary, HTLV-1 transmission occurs via rapid budding and fusion of the HTLV-1 virus across the VS from the infected to uninfected cell (Figure 2).

It has been reported that the formation of the VS is triggered by HTLV-1 infection and is not dependent on signaling through the T-cell receptor as is seen in immunological synapses [124]. The VS forms when the surface adhesion molecule intercellular adhesion molecule-1 (ICAM-1) is engaged by its ligand lymphocyte function-associated antigen 1 (LFA-1) [90,126]. ICAM-1 then activates the MEK/ERK pathway, which contributes to MTOC relocation. The HTLV-1 Tax protein, the key virus

transcription accessory protein, works in synergy with ICAM-1 to facilitate MTOC polarization. While it is primarily a nuclear protein, Tax can be found in the cytoplasm near the MTOC as well as in the cell-cell contact region [127]. Tax activates the CREB-signaling pathway during the formation of the HTLV-1 VS [126]. The CREB pathway increases expression of Gem, a small GTP-binding protein in the RAS superfamily, which is involved in cytoskeleton remodeling and cell migration [128]. Tax also appears to upregulate ICAM-1 in HTLV-1-infected cells, indicating that ICAM-1 and Tax appear to have synergistic roles in HTLV-1 VS formation [129].

![Diagram](attachment:diagram.png)

Figure 2. HTLV-1 Cell-to-Cell Transmission. Shown in the diagram is a host cell (bottom) that has anchored itself to a permissive target cell (top) using ICAM-1 and LFA1. The HTLV-1 accessory protein p8 has been shown to increase the expression of the LFA1 receptors as well as increase the number of cell-cell synapses, which p8 then traffics through to increase LFA1 in the target cell. Once ICAM-1 is bound, it triggers a signaling cascade that leads to the relocation of the MTOC. Additionally, HTLV-1 Tax is found bound to Golgi bodies that are attached to the MTOC. This initiates a variety of cell signaling cascades. For example, one established pathway increases expression of Gem, a protein that increases cell motility and possibly MTOC relocation. As HTLV-1 Gag is produced, it is found concentrated at the MTOC. Subsequently, HTLV-1 Gag is found at the sites of cell-cell contacts where the virus particles bud into the viral synapse formed by the cellular adhesion points. Virus particles produced can bind receptors for entry into the permissive target cell. The question marks along the lines with arrows indicate mechanisms of transport or activation that are not well understood.

When comparing HTLV-1 cell-cell transmission to HIV-1 cell-cell transmission, HTLV-1 is more dependent on cell contact for infectivity. Coculturing of virus-producing cells with permissive target cells significantly enhances HTLV-1 transmission by several thousand-fold, but only increases HIV-1 transmission 10–100 hundred-fold [130,131]. Furthermore, disruption of actin and tubulin

polymerization inhibits HTLV-1 spread to a greater extent than that of HIV-1 spread [131]. Nonetheless, HIV-1 cell-cell transmission remains a highly relevant form of virus spread *in vivo*. A recent study helps to highlight this through the observation that granulocytes (e.g., basophils) may actually capture HIV-1 and contribute to virus transmission [132].

### 3.4. Viral Biofilms

In addition to transmission via the VS, HTLV-1 particles have been reported to have the ability to form a biofilm-like, carbohydrate-rich extracellular structure on the surface of cells. These structures are composed of collagen, agrin, tetherin, and galectin-3 and may function as a way to concentrate HTLV-1 particles in a single location to increase the likelihood of infection of a permissive target cell [133]. A recent report described the creation of antiviral biofilm monoclonal antibodies using purified biofilms from MT2 cells. It was found that in addition to the structural proteins previously identified, the antigens CD4, CD150, CD25, CD70, and CD80 were also identified in these viral biofilms. Tax expression was found to modulate the level of these antigens in the viral biofilm [134].

While it has been observed that dendritic cells may be infected by cell-free HTLV-1 *in vitro* [31], a recent study analyzed the infectivity of chronically infected C91PL cell culture supernatant as compared to purified biofilms in primary human monocyte-derived dendritic cells (MDDCs) and T lymphocytes. It was found that while MDDCs were more easily infected than were T lymphocytes, both cell types achieved significantly higher proviral loads when exposed to viral biofilms as opposed to supernatant-derived virus [135].

## 4. Monoclonal Expansion of HTLV-1 Infected Cells and Leukemogenesis

While HTLV-1 cell-cell transmission is likely a critical determinant of virus transmission from an infected individual to a susceptible individual, many studies have established that the primary route of replication for HTLV-1 *in vivo* is through mitotic division of host cells and subsequent propagation of the provirus by clonal expansion. DNA analysis of HTLV-1 sequences from patients in geographically distinct locations shows very little genetic variation among HTLV-1 isolates [136]. This is likely because of a low evolutionary rate of $7.06 \times 10^{-7}$–$1.38 \times 10^{-5}$ substitutions per site per year in the LTR and env regions [137]. While the HTLV-1 reverse transcriptase has a reduced mutation rate as compared to HIV-1 reverse transcriptase ($7 \times 10^{-6}$ mutations per target base pair per replication cycle for HTLV-1 as compared to $3.4 \times 10^{-5}$ for HIV-1), the fourfold difference is likely not sufficient alone to explain the relative difference in genetic diversity between the two viruses [138,139]. Also, administration of reverse transcriptase inhibitors was found to not reduce proviral loads, even when administered shortly after infection [140,141].

The most compelling evidence in favor of a clonal expansion of HTLV-1 proviruses is the clonality of T-cells in infected individuals. HTLV-1 integrates randomly into the genome [142,143]. When the HTLV-1 proviral sites are amplified by PCR, it has been consistently found that the T-cells are clonal in both symptomatic and asymptomatic carriers [144,145]. In some HTLV-1 infected individuals, more than one in every 1500 peripheral blood mononuclear cells (PBMCs) were found to be clonal [146].

Since HTLV-1 persistence depends to some extent on the clonal expansion of infected T-cells, it is not surprising that HTLV-1 encodes for gene products that have been found to increase cell proliferation. The oncogenic potential of HTLV-1 is likely a byproduct of the induction of these cellular proliferative pathways. In fact, expression of the HTLV-1 basic leucine zipper (HBZ) and Tax proteins in double transgenic mice was recently shown to be sufficient for the development of lymphoma in the absence of any other viral genes [147]. Since similar pathways influence both cell immortalization and cellular transformation, these two processes are difficult to uncouple.

### 4.1. Tax

Tax has been established to have many roles in the HTLV-1 proliferative cycle, including essential roles in mediating transcription of HTLV-1 genes as well as the recruitment of the MTOC to sites

of cell-cell contact. Tax can also interfere with host cell cycle regulation, apoptotic pathways, and proliferative pathways through a variety of mechanisms. The most commonly studied pathway through which Tax increases cellular proliferation is via the NF-κB/Rel family of proteins.

### 4.2. Tax and Canonical NF-κB Signaling

The NF-κB/Rel proteins are transcription factors that share a Rel homology domain responsible for DNA binding and dimerization. There are five proteins in this family: RelA (p65), RelB, c-Rel, p105 (processed to p50), and p100 (processed to p52). When the NF-κB pathways are turned off, NF-κB dimers are bound to the inhibitor of κB (IκB) proteins. Activation of NF-κB can occur through a canonical pathway in which nuclear localization of NF-κB is induced by inflammatory stimuli such as tumor necrosis factor (TNF) or through a non-canonical pathway reviewed in [148].

NF-κB is constitutively active in HTLV-1 infected cells, an abnormality due to the tight regulation of NF-κB by its inhibitor, IκB [149]. IκB kinase (IKK) phosphorylates IκB molecules, leading to the eventual ubiquitination and degradation of IκB and freeing NF-κB dimers to traffic to the nucleus. HTLV-1 Tax increases the activity of IKK by regulating a variety of upstream effectors. Tak1 is a kinase that can phosphorylate and activate IKK. Tax binds directly to Tak1 to form an IKK-Tak1-Tax complex that appears to increase the efficiency of IKK [150]. Tax can also increase the activity of Tak1 by binding to Tak1-binding protein 2 (TAB2); the physiological function of this remains unclear [151]. Tax can also bind directly to TRAF6 and can stimulate its function, leading to the activation of NF-κB through IKK phosphorylation [152]. Finally, Tax can also stimulate MEKK1, which is an upstream regulator of TAK1 [153].

In addition to its upstream effects on IKK, Tax also binds directly to an IKK subunit, IKKγ (referred to as NEMO) [154,155]. IKKγ is a regulatory subunit that modulates IKK activity through an unknown mechanism [156]. Protein phosphatase 2A (PP2A) is a positive regulator of IKKγ, and PP2A binding to helix 2 (HLX2) is necessary for proper IKK function [157]. Tax binds IKKγ in a region two heptads downstream of HLX2 termed coiled-coil region 2 (CCR2). When Tax binds this region, it exposes the HLX2 domain for PP2A binding and simultaneously inactivates PP2A, preventing it from dissociating, resulting in IKK being constitutively active [158,159].

Tax also has the ability to form complexes with NF-κB monomers in the cytoplasm. Tax can bind directly to cytoplasmic RelA (p65) and can induce translocation into the nucleus, leading to high transcriptional activity [151,160–162]. This particular protein-protein interaction appears to also rely on the CREB-binding protein (CBP or p300), which facilitates the transcriptional activity of RelA.

A role for Tax that has only recently been described is the recognition and inactivation of the ubiquitin-editing enzyme A20. This enzyme is a negative regulator of the canonical NF-κB pathway. It binds with the regulatory protein TAX1BP1 and E3 ligase Itch to form a complex that inactivates essential NF-κB upstream regulators. Tax binds to TAX1BP1, preventing the interaction of TAX1BP1, A20, and Itch [163,164].

### 4.3. Tax and Non-Canonical NF-κB Signaling

Tax is known to facilitate the non-canonical pathway of NF-κB as well as the canonical pathway traditionally induced by MEKK1 and Tak1. This is an important distinction, as the NF-κB dimers are distinct for each pathway and increase expression of different gene products. For example, when the non-canonical pathway is blocked in the presence of Tax, tumorigenesis is significantly delayed in Tax transgenic mice [165]. IKKα phosphorylates p100, which leads to its ubiquitination and cleavage to p52. Tax facilitates this process by recruiting IKKα to p100 and inducing cleavage [166,167].

### 4.4. Tax and Other Cell Proliferative Pathways

In addition to its interaction with NF-κB, HTLV-1 Tax has roles in a variety of other cell proliferation pathways. In particular, Tax interacts directly or indirectly with cyclin-dependent kinases (CDK) [168,169], phosphoinositide 3-kinase (PI3K) [170], transforming growth factor β-1

(TGFβ-1) [171, 172], and p53 [173–175]. Through a series of complex interactions, Tax ensures that HTLV-1 replication occurs.

### 4.5. Tax Downregulation

Tax plays a key role in the proliferation of T-cells but is known to be a highly immunogenic protein, and Tax-expressing cells are targeted by cytotoxic T-cells [176, 177]. That Tax is expressed only at early time points helps to avoid immune surveillance. In patients with ATL, *tax* mRNA transcripts were only found in 34% of all cases [178], indicating that there are downregulation mechanisms in place to prevent HTLV-1-infected cells from being destroyed by the immune system.

Several mechanisms regulate the downregulation of Tax expression. Many *tax* genes become mutated so that non-functional transcripts are produced [178–180]. Cells containing these *tax* gene sequences are more likely to replicate due to decreased immune surveillance. Additionally, the *tax* gene is often methylated, leading to gene silencing [178, 181]. Finally, the HTLV-1 bZIP factor (HBZ) works to downregulate transcription of many HTLV-1 genes, including Tax [182].

### 4.6. HBZ

The HBZ gene is an important HTLV-1 gene that is consistently associated with ATL. HBZ mRNA transcripts have been found in virtually all ATL cells, and it was shown that HBZ plays a key role in the proliferation of T-cells [183]. Interestingly, the HBZ mRNA appears to play a different role in T-cell proliferation than the HBZ protein [183]. In mouse T-cells, an HBZ start codon mutant that distinguishes between RNA function and protein function was found to increase T-cell proliferation by inhibiting apoptosis and promoting S-phase entry [184]. HBZ RNA attenuates apoptosis by promoting the transcription of Survivin, a caspase inhibitor that prevents apoptosis. While HBZ protein promotes S-phase entry, it can also promote apoptosis through its pro-inflammatory effects [184].

HBZ has been reported to directly or indirectly interact with the following proteins that are essential in CREB-dependent cellular proliferation: CREB, CBP, ATF-1, and ATF-3 [185–188]. Additionally, HBZ interacts with the Jun family of transcription factors, JunB, JunD, and c-Jun [189–191]. The combined activities of Tax and HBZ are essential for cellular proliferation. Taken together, the activities of both HBZ and Tax are essential for the transformation of T-cells.

## 5. Conclusions and Future Directions

As the first human retrovirus discovered in the early 1980s, HTLV-1 has been studied extensively, yet there is still no treatment or vaccine for HTLV-1 infection. Additionally, ATL and HAM/TSP treatments are symptom-based and do not directly treat the viral infection. Continued research on the molecular aspects of HTLV-1 replication will enhance opportunities for the discovery of potential antiretroviral targets that can be exploited for the development of effective therapeutic strategies. For example, a recent candidate peptide (HBZ<sub>157--176</sub>) has been described for use in vaccine development [192]. Such observations help to enhance the likelihood for specific forms of therapeutic intervention for the treatment of HTLV-1 infection.

### Acknowledgments:

This work is supported by National Institutes of Health grant R01 GM098550. Jessica L. Martin has been supported by NIH grant T32 DA007097. José O. Maldonado has been supported from NIH grants T32 AI083196 (Institute for Molecular Virology Training Program) and F30 DE022286.

### Author Contributions:

Jessica L. Martin and José O. Maldonado drafted the manuscript, and Joachim D. Mueller, Wei Zhang, and Louis M. Mansky edited the text to create the final submitted version.

### Conflicts of Interest:

The authors declare no conflict of interest.

References

1. Poiesz, B.J.; RusCetti, F.W.; Gazdar, A.F.; Bunn, P.A.; Minna, J.D.; Gallo, R.C. Detection and isolation of type C retrovirus particles from fresh and cultured lymphocytes of a patient with cutaneous T-cell lymphoma. *Proc. Natl. Acad. Sci. USA* **1980**, *77*, 7415–7419. [CrossRef] [PubMed]

2. Yoshida, M.; Miyoshi, I.; Hinuma, Y. Isolation and characterization of retrovirus from cell lines of human adult t-cell leukemia and its implication in the disease. *Proc. Natl. Acad. Sci. USA* **1982**, *79*, 2031–2035. [CrossRef] [PubMed]

3. Gallo, R.C. History of the discoveries of the first human retroviruses: HTLV-1 and HTLV-2. *Oncogene* **2005**, *24*, 5926–5930. [CrossRef] [PubMed]

4. Kalyanaraman, V.S.; Sarngadharan, M.G.; Robert-Guroff, M.; Miyoshi, I.; Golde, D.; Gallo, R.C. A new subtype of human T-cell leukemia virus (HTLV-II) associated with a T-cell variant of hairy cell leukemia. *Science* **1982**, *218*, 571–573. [CrossRef] [PubMed]

5. Caron, C.; Rousset, R.; Beraud, C.; Moncollin, V.; Egly, J.M.; Jalinot, P. Functional and biochemical interaction of the HTLV-I Tax1 transactivator with tbp. *EMBO J.* **1993**, *12*, 4269–4278. [PubMed]

6. Proietti, F.A.; Carneiro-Proietti, A.B.; Catalan-Soares, B.C.; Murphy, E.L. Global epidemiology of HTLV-I infection and associated diseases. *Oncogene* **2005**, *24*, 6058–6068. [CrossRef] [PubMed]

7. Gessain, A.; Cassar, O. Epidemiological aspects and world distribution of HTLV-1 infection. *Front. Microbiol.* **2012**, *3*. [CrossRef] [PubMed]

8. Calattini, S.; Chevalier, S.A.; Duprez, R.; Bassot, S.; Froment, A.; Mahieux, R.; Gessain, A. Discovery of a new human T-cell lymphotropic virus (HTLV-3) in central africa. *Retrovirology* **2005**, *2*. [CrossRef] [PubMed]

9. Wolfe, N.D.; Heneine, W.; Carr, J.K.; Garcia, A.D.; Shanmugam, V.; Tamoufe, U.; Torimiro, J.N.; Prosser, A.T.; Lebreton, M.; Mpoudi-Ngole, E.; *et al.* Emergence of unique primate T-lymphotropic viruses among central african bushmeat hunters. *Proc. Natl. Acad. Sci. USA* **2005**, *102*, 7994–7999. [CrossRef] [PubMed]

10. Yoshida, M.; Seiki, M.; Yamaguchi, K.; Takatsuki, K. Monoclonal integration of human T-cell leukemia provirus in all primary tumors of adult T-cell leukemia suggests causative role of human T-cell leukemia virus in the disease. *Proc. Natl. Acad. Sci. USA* **1984**, *81*, 2534–2537. [CrossRef] [PubMed]

11. Murphy, E.L.; Figueroa, J.P.; Gibbs, W.N.; Brathwaite, A.; Holding-Cobham, M.; Waters, D.; Cranston, B.; Hanchard, B.; Blattner, W.A. Sexual transmission of human T-lymphotropic virus type I (HTLV-I). *Ann. Int. Med.* **1989**, *111*, 555–560. [CrossRef] [PubMed]

12. Kaplan, J.E.; Osame, M.; Kubota, H.; Igata, A.; Nishitani, H.; Maeda, Y.; Khabbaz, R.F.; Janssen, R.S. The risk of development of HTLV-I-associated myelopathy/tropical spastic paraparesis among persons infected with HTLV-I. *J. Acquir. Immune Defic. Syndr.* **1990**, *3*, 1096–1101. [PubMed]

13. Hisada, M.; Stuver, S.O.; Okayama, A.; Li, H.C.; Sawada, T.; Hanchard, B.; Mueller, N.E. Persistent paradox of natural history of human T lymphotropic virus type I: Parallel analyses of Japanese and Jamaican carriers. *J. Infect. Dis.* **2004**, *190*, 1605–1609. [CrossRef] [PubMed]

14. Tsukasaki, K.; Hermine, O.; Bazarbachi, A.; Ratner, L.; Ramos, J.C.; Harrington, W., Jr.; O'Mahony, D.; Janik, J.E.; Bittencourt, A.L.; Taylor, G.P.; *et al.* Definition, prognostic factors, treatment, and response criteria of adult T-cell leukemia-lymphoma: A proposal from an international consensus meeting. *J. Clin. Oncol.* **2009**, *27*, 453–459. [CrossRef] [PubMed]

15. Tanosaki, R.; Uike, N.; Utsunomiya, A.; Saburi, Y.; Masuda, M.; Tomonaga, M.; Eto, T.; Hidaka, M.; Harada, M.; Choi, I.; *et al.* Allogeneic hematopoietic stem cell transplantation using reduced-intensity conditioning for adult T cell leukemia/lymphoma: Impact of antithymocyte globulin on clinical outcome. *J. Am. Soc. Blood Marrow Transplant.* **2008**, *14*, 702–708. [CrossRef] [PubMed]

16. Tsukasaki, K.; Utsunomiya, A.; Fukuda, H.; Shibata, T.; Fukushima, T.; Takatsuka, Y.; Ikeda, S.; Masuda, M.; Nagoshi, H.; Ueda, R.; *et al.* VCAP-AMP-VECP compared with biweekly CHOP for adult T-cell leukemia-lymphoma: Japan clinical oncology group study JCOG9801. *J. Clin. Oncol.* **2007**, *25*, 5458–5464. [CrossRef] [PubMed]

17. Yamamoto, K.; Utsunomiya, A.; Tobinai, K.; Tsukasaki, K.; Uike, N.; Uozumi, K.; Yamaguchi, K.; Yamada, Y.; Hanada, S.; Tamura, K. Phase I study of KW-0761, a defucosylated humanized anti-CCR4 antibody, in relapsed patients with adult T-cell leukemia-lymphoma and peripheral T-cell lymphoma. *J. Clin. Oncol.* **2010**, *28*, 1591–1598. [CrossRef] [PubMed]

18. Bazabachi, A.; Plumelle, Y.; Carlos Ramos, J.; Tortevoye, P.; Otrock, Z.; Taylor, G.; Gessain, A.; Harrington, W.; Panelatti, G.; Hermine, O. Meta-analysis on the use of zidovudine and interferon-alfa in adult T-cell leukemia/lymphoma showing improved survival in the leukemic subtypes. *J. Clin. Oncol.* 2010, 28, 4177–4183. [CrossRef] [PubMed]

19. Kinpara, S.; Kijiyama, M.; Takamori, A.; Hasegawa, A.; Sasada, A.; Masuda, T.; Tanaka, Y.; Utsunomiya, A.; Kannagi, M. Interferon-α (IFN-α) suppresses HTLV-1 gene expression and cell cycling, while IFN-α combined with zidovudine induces p53 signaling and apoptosis in HTLV-1-infected cells. *Retrovirology* 2013, 10. [CrossRef] [PubMed]

20. Meekings, K.N.; Leipzig, J.; Bushman, F.D.; Taylor, G.P.; Bangham, C.R. HTLV-1 integration into transcriptionally active genomic regions is associated with proviral expression and with HAM/TSP. *PLoS Pathog.* 2008, 4. [CrossRef] [PubMed]

21. Nagai, M.; Usuku, K.; Matsumoto, W.; Kodama, D.; Takenouchi, N.; Moritoyo, T.; Hashiguchi, S.; Ichinose, M.; Bangham, C.R.; Izumo, S.; *et al.* Analysis of HTLV-I proviral load in 202 HAM/TSP patients and 243 asymptomatic HTLV-I carriers: High proviral load strongly predisposes to HAM/TSP. *J. Neurovirol.* 1998, 4, 586–593. [CrossRef] [PubMed]

22. Oh, U.; Jacobson, S. Treatment of HTLV-I-associated myelopathy/tropical spastic paraparesis: Toward rational targeted therapy. *Neurol. Clin.* 2008, 26, 781–797. [CrossRef] [PubMed]

23. Yamamoto, N.; Matsumoto, T.; Koyanagi, Y.; Tanaka, Y.; Hinuma, Y. Unique cell lines harbouring both epstein-barr virus and adult T-cell leukaemia virus, established from leukaemia patients. *Nature* 1982, 299, 367–369. [CrossRef] [PubMed]

24. Ho, D.D.; Rota, T.R.; Hirsch, M.S. Infection of human endothelial cells by human T-lymphotropic virus type I. *Proc. Natl. Acad. Sci. USA* 1984, 81, 7588–7590. [CrossRef] [PubMed]

25. Longo, D.L.; Gelmann, E.P.; Cossman, J.; Young, R.A.; Gallo, R.C.; O’Brien, S.J.; Matis, L.A. Isolation of HTLV-transformed B-lymphocyte clone from a patient with HTLV-associated adult T-cell leukaemia. *Nature* 1984, 310, 505–506. [CrossRef] [PubMed]

26. Yoshikura, H.; Nishida, J.; Yoshida, M.; Kitamura, Y.; Takaku, F.; Ikeda, S. Isolation of HTLV derived from Japanese adult T-cell leukemia patients in human diploid fibroblast strain IMR90 and the biological characters of the infected cells. *Int. J. Cancer* 1984, 33, 745–749. [CrossRef] [PubMed]

27. Koyanagi, Y.; Itoyama, Y.; Nakamura, N.; Takamatsu, K.; Kira, J.; Iwamasa, T.; Goto, I.; Yamamoto, N. *In vivo* infection of human T-cell leukemia virus type I in non-T cells. *Virology* 1993, 196, 25–33. [CrossRef] [PubMed]

28. Manel, N.; Kim, F.J.; Kinet, S.; Taylor, N.; Sitbon, M.; Battini, J.L. The ubiquitous glucose transporter GLUT-1 is a receptor for HTLV. *Cell* 2003, 115, 449–459. [CrossRef]

29. Jones, K.S.; Petrow-Sadowski, C.; Bertolette, D.C.; Huang, Y.; Rus셋ti, F.W. Heparan sulfate proteoglycans mediate attachment and entry of human T-cell leukemia virus type 1 virions into CD4⁺ T cells. *J. Virol.* 2005, 79, 12692–12702. [CrossRef] [PubMed]

30. Ghez, D.; Lepelletier, Y.; Lambert, S.; Fourneau, J.M.; Blot, V.; Janvier, S.; Arnulf, B.; van Endert, P.M.; Heveker, N.; Pique, C.; *et al.* Neuropilin-1 is involved in human T-cell lymphotropic virus type 1 entry. *J. Virol.* 2006, 80, 6844–6854. [CrossRef] [PubMed]

31. Jones, K.S.; Petrow-Sadowski, C.; Huang, Y.K.; Bertolette, D.C.; Rus셋ti, F.W. Cell-free HTLV-1 infects dendritic cells leading to transmission and transformation of CD4⁺ T cells. *Nat. Med.* 2008, 14, 429–436. [CrossRef] [PubMed]

32. Jin, Q.; Agrawal, L.; VanHorn-Ali, Z.; Alkhatib, G. Infection of CD4⁺ T lymphocytes by the human T cell leukemia virus type 1 is mediated by the glucose transporter glut-1: Evidence using antibodies specific to the receptor’s large extracellular domain. *Virology* 2006, 349, 184–196. [CrossRef] [PubMed]

33. Aiken, C.; Konner, J.; Landau, N.R.; Lenburg, M.E.; Trono, D. Nef induces CD4 endocytosis: Requirement for a critical dileucine motif in the membrane-proximal CD4 cytoplasmic domain. *Cell* 1994, 76, 853–864. [CrossRef]

34. Margottin, F.; Bour, S.P.; Durand, H.; Selig, L.; Benichou, S.; Richard, V.; Thomas, D.; Strebel, K.; Benarous, R. A novel human WD protein, H-β TRCP, that interacts with HIV-1 VPU connects CD4 to the ER degradation pathway through an F-box motif. *Mol. Cell* 1998, 1, 565–574. [CrossRef]

35. Maeda, Y.; Terasawa, H.; Tanaka, Y.; Mitsuura, C.; Nakashima, K.; Yusa, K.; Harada, S. Separate cellular localizations of human T-lymphotropic virus 1 (HTLV-1) Env and glucose transporter type 1 (glut1) are required for HTLV-1 Env-mediated fusion and infection. *J. Virol.* 2015, 89, 502–511. [CrossRef] [PubMed]

36. Macintyre, A.N.; Gerriets, V.A.; Nichols, A.G.; Michalek, R.D.; Rudolph, M.C.; Deoliveira, D.; Anderson, S.M.; Abel, E.D.; Chen, B.J.; Hale, L.P.; *et al.* The glucose transporter glut1 is selectively essential for CD4 T cell activation and effector function. *Cell Metab.* 2014, 20, 61–72. [CrossRef] [PubMed]

37. Temin, H.M.; Mizutani, S. RNA-dependent DNA polymerase in virions of Rous sarcoma virus. *Nature* 1970, 226, 1211–1213. [CrossRef] [PubMed]

38. Baltimore, D. RNA-dependent DNA polymerase in virions of RNA tumour viruses. *Nature* 1970, 226, 1209–1211. [CrossRef] [PubMed]

39. Hulme, A.E.; Perez, O.; Hope, T.J. Complementary assays reveal a relationship between HIV-1 uncoating and reverse transcription. *Proc. Natl. Acad. Sci. USA* 2011, 108, 9975–9980. [CrossRef] [PubMed]

40. Tekeste, S.S.; Wilkinson, T.A.; Weiner, E.M.; Xu, X.; Miller, J.T.; Le Grice, S.F.; Clubb, R.T.; Chow, S.A. Interaction between reverse transcriptase and integrase is required for reverse transcription during HIV-1 replication. *J. Virol.* 2015. [CrossRef] [PubMed]

41. Desrames, A.; Cassar, O.; Gout, O.; Hermine, O.; Taylor, G.P.; Afonso, P.V.; Gessain, A. Northern African strains of human T-lymphotropic virus type 1 arose from a recombination event. *J. Virol.* 2014, 88, 9782–9788. [CrossRef] [PubMed]

42. Ooms, M.; Krikoni, A.; Kress, A.K.; Simon, V.; Munk, C. APOBEC3A, APOBEC3B, and APOBEC3H haplotype 2 restrict human T-lymphotropic virus type 1. *J. Virol.* 2012, 86, 6097–6108. [CrossRef] [PubMed]

43. Fan, J.; Ma, G.; Nosaka, K.; Tanabe, J.; Satou, Y.; Koito, A.; Wain-Hobson, S.; Vartanian, J.P.; Matsuoka, M. APOBEC3G generates nonsense mutations in human T-cell leukemia virus type 1 proviral genomes *in vivo*. *J. Virol.* 2010, 84, 7278–7287. [CrossRef] [PubMed]

44. Derse, D.; Hill, S.A.; Princler, G.; Lloyd, P.; Heidecker, G. Resistance of human T cell leukemia virus type 1 to APOBEC3G restriction is mediated by elements in nucleocapsid. *Proc. Natl. Acad. Sci. USA* 2007, 104, 2915–2920. [CrossRef] [PubMed]

45. Coffin, J.M.; Hughes, S.H.; Varmus, H.E. The interactions of retroviruses and their hosts. In *Retroviruses*; Cold Spring Harbor: NY, USA, 1997.

46. Kitamura, Y.; Lee, Y.M.; Coffin, J.M. Nonrandom integration of retroviral DNA *in vitro*: Effect of CPG methylation. *Proc. Natl. Acad. Sci. USA* 1992, 89, 5532–5536. [CrossRef] [PubMed]

47. Doi, K.; Wu, X.; Taniguchi, Y.; Yasunaga, J.; Satou, Y.; Okayama, A.; Nosaka, K.; Matsuoka, M. Preferential selection of human T-cell leukemia virus type I provirus integration sites in leukemic *versus* carrier states. *Blood* 2005, 106, 1048–1053. [CrossRef] [PubMed]

48. Holman, A.G.; Coffin, J.M. Symmetrical base preferences surrounding HIV-1, avian sarcoma/leukosis virus, and murine leukemia virus integration sites. *Proc. Natl. Acad. Sci. USA* 2005, 102, 6103–6107. [CrossRef] [PubMed]

49. Kang, Y.; Moressi, C.J.; Scheetz, T.E.; Xie, L.; Tran, D.T.; Casavant, T.L.; Ak, P.; Benham, C.J.; Davidson, B.L.; McCray, P.B., Jr. Integration site choice of a feline immunodeficiency virus vector. *J. Virol.* 2006, 80, 8820–8823. [CrossRef] [PubMed]

50. Derse, D.; Crise, B.; Li, Y.; Princler, G.; Lum, N.; Stewart, C.; McGrath, C.F.; Hughes, S.H.; Munroe, D.J.; Wu, X. Human T-cell leukemia virus type 1 integration target sites in the human genome: Comparison with those of other retroviruses. *J. Virol.* 2007, 81, 6731–6741. [CrossRef] [PubMed]

51. Gillet, N.A.; Cook, L.; Laydon, D.J.; Hlela, C.; Verdonck, K.; Alvarez, C.; Gotuzzo, E.; Clark, D.; Farre, L.; Bittencourt, A.; *et al.* Strongyloidiasis and infective dermatitis alter human T lymphotropic virus-1 clonality *in vivo*. *PLoS Pathog.* 2013, 9. [CrossRef] [PubMed]

52. Cook, L.B.; Melamed, A.; Niederer, H.; Valganon, M.; Laydon, D.; Foroni, L.; Taylor, G.P.; Matsuoka, M.; Bangham, C.R. The role of HTLV-1 clonality, proviral structure, and genomic integration site in adult T-cell leukemia/lymphoma. *Blood* 2014, 123, 3925–3931. [CrossRef] [PubMed]

53. Niederer, H.A.; Laydon, D.J.; Melamed, A.; Elemans, M.; Asquith, B.; Matsuoka, M.; Bangham, C.R. HTLV-1 proviral integration sites differ between asymptomatic carriers and patients with HAM/TSP. *J. Virol.* 2014, 11. [CrossRef] [PubMed]

54. Kashanchi, F.; Brady, J.N. Transcriptional and post-transcriptional gene regulation of HTLV-1. *Oncogene* 2005, 24, 5938–5951. [CrossRef] [PubMed]

55. Jeang, K.T.; Boros, I.; Brady, J.; Radonovich, M.; Khoury, G. Characterization of cellular factors that interact with the human T-cell leukemia virus type I p40x-responsive 21-base-pair sequence. *J. Virol.* 1988, 62, 4499–4509. [PubMed]

56. Baranger, A.M.; Palmer, C.R.; Hamm, M.K.; Giebler, H.A.; Brauweiler, A.; Nyborg, J.K.; Schepartz, A. Mechanism of DNA-binding enhancement by the human T-cell leukaemia virus transactivator Tax. *Nature* 1995, **376**, 606–608. [CrossRef] [PubMed]

57. Adya, N.; Giam, C.Z. Distinct regions in human T-cell lymphotropic virus type I Tax mediate interactions with activator protein CREB and basal transcription factors. *J. Virol.* 1995, **69**, 1834–1841. [PubMed]

58. Adya, N.; Zhao, L.J.; Huang, W.; Boros, I.; Giam, C.Z. Expansion of CREB'S DNA recognition specificity by Tax results from interaction with Ala-Ala-Arg at positions 282–284 near the conserved DNA-binding domain of CREB. *Proc. Natl. Acad. Sci. USA* 1994, **91**, 5642–5646. [CrossRef] [PubMed]

59. Goren, I.; Semmes, O.J.; Jeang, K.T.; Moelling, K. The amino terminus of Tax is required for interaction with the cyclic AMP response element binding protein. *J. Virol.* 1995, **69**, 5806–5811. [PubMed]

60. Tie, F.; Adya, N.; Greene, W.C.; Giam, C.Z. Interaction of the human T-lymphotropic virus type 1 Tax dimer with CREB and the viral 21-base-pair repeat. *J. Virol.* 1996, **70**, 8368–8374. [PubMed]

61. Zhao, L.J.; Giam, C.Z. Human T-cell lymphotropic virus type I (HTLV-I) transcriptional activator, Tax, enhances CREB binding to HTLV-I 21-base-pair repeats by protein-protein interaction. *Proc. Natl. Acad. Sci. USA* 1992, **89**, 7070–7074. [CrossRef] [PubMed]

62. Ching, Y.P.; Chun, A.C.; Chin, K.T.; Zhang, Z.Q.; Jeang, K.T.; Jin, D.Y. Specific TATAA and bZIP requirements suggest that HTLV-I Tax has transcriptional activity subsequent to the assembly of an initiation complex. *Retrovirology* 2004, **1**. [CrossRef] [PubMed]

63. Seiki, M.; Inoue, J.; Takeda, T.; Yoshida, M. Direct evidence that p40x of human T-cell leukemia virus type I is a trans-acting transcriptional activator. *EMBO J.* 1986, **5**, 561–565. [PubMed]

64. Giebler, H.A.; Loring, J.E.; van Orden, K.; Colgin, M.A.; Garrus, J.E.; Escudero, K.W.; Brauweiler, A.; Nyborg, J.K. Anchoring of CREB binding protein to the human T-cell leukemia virus type 1 promoter: A molecular mechanism of Tax transactivation. *Mol. Cell. Biol.* 1997, **17**, 5156–5164. [CrossRef] [PubMed]

65. Kwok, R.P.; Laurence, M.E.; Lundblad, J.R.; Goldman, P.S.; Shih, H.; Connor, L.M.; Marriott, S.J.; Goodman, R.H. Control of CAMP-regulated enhancers by the viral transactivator Tax through CREB and the co-activator CBP. *Nature* 1996, **380**, 642–646. [CrossRef] [PubMed]

66. Jiang, H.; Lu, H.; Schiltz, R.L.; Pise-Masison, C.A.; Ogryzko, V.V.; Nakatani, Y.; Brady, J.N. PCAF interacts with Tax and stimulates Tax transactivation in a histone acetyltransferase-independent manner. *Mol. Cell. Biol.* 1999, **19**, 8136–8145. [CrossRef]

67. Harrod, R.; Kuo, Y.L.; Tang, Y.; Yao, Y.; Vassilev, A.; Nakatani, Y.; Giam, C.Z. P300 and p300/CAMP-responsive element-binding protein associated factor interact with human T-cell lymphotropic virus type-1 Tax in a multi-histone acetyltransferase/activator-enhancer complex. *J. Biol. Chem.* 2000, **275**, 11852–11857. [CrossRef] [PubMed]

68. Bex, F.; Yin, M.J.; Burny, A.; Gaynor, R.B. Differential transcriptional activation by human T-cell leukemia virus type 1 Tax mutants is mediated by distinct interactions with CREB binding protein and p300. *Mol. Cell. Biol.* 1998, **18**, 2392–2405. [CrossRef] [PubMed]

69. Ma, G.; Yasunaga, J.; Akari, H.; Matsuoka, M. Tcf1 and LEF1 act as T-cell intrinsic HTLV-1 antagonists by targeting Tax. *Proc. Natl. Acad. Sci. USA* 2015, **112**, 2216–2221. [CrossRef] [PubMed]

70. Zou, T.; Yang, Y.; Xia, F.; Huang, A.; Gao, X.; Fang, D.; Xiong, S.; Zhang, J. Resveratrol inhibits CD4⁺ T cell activation by enhancing the expression and activity of Sirt1. *PloS ONE* 2013, **8**, e75139. [CrossRef] [PubMed]

71. Tang, H.M.; Gao, W.W.; Chan, C.P.; Cheng, Y.; Deng, J.J.; Yuen, K.S.; Iha, H.; Jin, D.Y. Sirt1 suppresses human T-cell leukemia virus type 1 transcription. *J. Virol.* 2015, **89**, 8623–8631. [CrossRef] [PubMed]

72. Huang, H.; Santoso, N.; Power, D.; Simpson, S.; Dieringer, M.; Miao, H.; Gurova, K.; Giam, C.Z.; Elledge, S.; Zhu, J. Fact proteins, SUPT16H and SSRP1, are transcriptional suppressors of HIV-1 and HTLV-1 that facilitate viral latency. *J. Biol. Chem.* 2015. [CrossRef] [PubMed]

73. Green, P.L.; Chen, I.S. Regulation of human T cell leukemia virus expression. *FASEB J.* 1990, **4**, 169–175. [PubMed]

74. Hidaka, M.; Inoue, J.; Yoshida, M.; Seiki, M. Post-transcriptional regulator (Rex) of HTLV-1 initiates expression of viral structural proteins but suppresses expression of regulatory proteins. *EMBO J.* 1988, **7**, 519–523. [PubMed]

75. Bogerd, H.P.; Huckaby, G.L.; Ahmed, Y.F.; Hanly, S.M.; Greene, W.C. The type I human T-cell leukemia virus (HTLV-I) Rex trans-activator binds directly to the HTLV-I Rex and the type 1 human immunodeficiency virus Rev RNA response elements. *Proc. Natl. Acad. Sci. USA* 1991, **88**, 5704–5708. [CrossRef] [PubMed]

76. Nosaka, T.; Siomi, H.; Adachi, Y.; Ishibashi, M.; Kubota, S.; Maki, M.; Hatanaka, M. Nuclear targeting signal of human T-cell leukemia virus type I Rex-encoded protein is essential for cytoplasmic accumulation of unspliced viral mRNA. *Proc. Natl. Acad. Sci. USA* **1989**, *86*, 9798–9802. [CrossRef](https://doi.org/10.1073/pnas.86.24.9798) [PubMed](https://pubmed.ncbi.nlm.nih.gov/2557457)

77. Rehberger, S.; Gounari, F.; DucDodon, M.; Chlichlia, K.; Gazzolo, L.; Schirrmacher, V.; Khazaie, K. The activation domain of a hormone inducible HTLV-1 Rex protein determines colocalization with the nuclear pore. *Exp. Cell Res.* **1997**, *233*, 363–371. [CrossRef](https://doi.org/10.1006/excr.1997.3674) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9231865)

78. Palmeri, D.; Malim, M.H. The human T-cell leukemia virus type 1 posttranscriptional trans-activator Rex contains a nuclear export signal. *J. Virol.* **1996**, *70*, 6442–6445. [PubMed](https://pubmed.ncbi.nlm.nih.gov/8764057)

79. Bai, X.T.; Sinha-Datta, U.; Ko, N.L.; Bellon, M.; Nicot, C. Nuclear export and expression of human T-cell leukemia virus type 1 Tax/Rex mRNA are Rxre/Rex dependent. *J. Virol.* **2012**, *86*, 4559–4565. [CrossRef](https://doi.org/10.1128/JVI.06559-11) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22301163)

80. Rende, F.; Cavallari, I.; Andresen, V.; Valeri, V.W.; D'Agostino, D.M.; Franchini, G.; Ciminale, V. Identification of novel monocistronic HTLV-1 mRNAs encoding functional Rex isoforms. *Retrovirology* **2015**, *12*, 58. [CrossRef](https://doi.org/10.1186/s12977-015-0154-7) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25982236)

81. Butsch, M.; Boris-Lawrie, K. Destiny of unspliced retroviral RNA: Ribosome and/or virion? *J. Virol.* **2002**, *76*, 3089–3094. [CrossRef](https://doi.org/10.1128/JVI.76.6.3089-3094.2002) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11861308)

82. Hellen, C.U.; Sarnow, P. Internal ribosome entry sites in eukaryotic mRNA molecules. *Genes Dev.* **2001**, *15*, 1593–1612. [CrossRef](https://doi.org/10.1101/gad.891.x) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11358595)

83. Sarnow, P.; Cevallos, R.C.; Jan, E. Takeover of host ribosomes by divergent IRES elements. *Biochem. Soc. Trans.* **2005**, *33*, 1479–1482. [CrossRef](https://doi.org/10.1042/BST0331479) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16246050)

84. Jackson, R.J. Alternative mechanisms of initiating translation of mammalian mRNAs. *Biochem. Soc. Trans.* **2005**, *33*, 1231–1241. [CrossRef](https://doi.org/10.1042/BST0331231) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16246046)

85. Attal, J.; Theron, M.C.; Taboit, F.; Cajero-Juarez, M.; Kann, G.; Bolifraud, P.; Houdebine, L.M. The RU5 (“R”) region from human leukaemia viruses (HTLV-1) contains an internal ribosome entry site (IRES)-like sequence. *FEBS Lett.* **1996**, *392*, 220–224. [CrossRef](https://doi.org/10.1016/0014-5793(96)00819-8)

86. Bolinger, C.; Yilmaz, A.; Hartman, T.R.; Kovacic, M.B.; Fernandez, S.; Ye, J.; Forget, M.; Green, P.L.; Boris-Lawrie, K. RNA helicase a interacts with divergent lymphotropic retroviruses and promotes translation of human T-cell leukemia virus type 1. *Nucleic Acids Res.* **2007**, *35*, 2629–2642. [CrossRef](https://doi.org/10.1093/nar/gkm156) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17384116)

87. Olivares, E.; Landry, D.M.; Caceres, C.J.; Pino, K.; Rossi, F.; Navarrete, C.; Huidobro-Toro, J.P.; Thompson, S.R.; Lopez-Lastra, M. The 5′ untranslated region of the human T-cell lymphotropic virus type 1 mRNA enables cap-independent translation initiation. *J. Virol.* **2014**, *88*, 5936–5955. [CrossRef](https://doi.org/10.1128/JVI.03485-13) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24789795)

88. Fogarty, K.H.; Chen, Y.; Grigsby, I.F.; Macdonald, P.J.; Smith, E.M.; Johnson, J.L.; Rawson, J.M.; Mansky, L.M.; Mueller, J.D. Characterization of cytoplasmic Gag-gag interactions by dual-color z-scan fluorescence fluctuation spectroscopy. *Biophys. J.* **2011**, *100*, 1587–1595. [CrossRef](https://doi.org/10.1016/j.bpj.2011.01.042) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21397986)

89. Ritchie, C.; Cylinder, I.; Platt, E.J.; Barklis, E. Analysis of HIV-1 Gag protein interactions via biotin ligase tagging. *J. Virol.* **2015**, *89*, 3988–4001. [CrossRef](https://doi.org/10.1128/JVI.03354-14) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25673688)

90. Barnard, A.L.; Igakura, T.; Tanaka, Y.; Taylor, G.P.; Bangham, C.R. Engagement of specific T-cell surface molecules regulates cytoskeletal polarization in HTLV-1-infected lymphocytes. *Blood* **2005**, *106*, 988–995. [CrossRef](https://doi.org/10.1182/blood-2004-10-4038) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15797989)

91. Qualley, D.F.; Stewart-Maynard, K.M.; Wang, F.; Mitra, M.; Gorelick, R.J.; Rouzina, I.; Williams, M.C.; Musier-Forsyth, K. C-terminal domain modulates the nucleic acid chaperone activity of human T-cell leukemia virus type 1 nucleocapsid protein via an electrostatic mechanism. *J. Biol. Chem.* **2010**, *285*, 295–307. [CrossRef](https://doi.org/10.1074/jbc.M109.051868) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19889797)

92. Sun, M.; Grigsby, I.F.; Gorelick, R.J.; Mansky, L.M.; Musier-Forsyth, K. Retrovirus-specific differences in matrix and nucleocapsid protein-nucleic acid interactions: Implications for genomic RNA packaging. *J. Virol.* **2014**, *88*, 1271–1280. [CrossRef](https://doi.org/10.1128/JVI.02275-13) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24259408)

93. Chen, J.; Grunwald, D.; Sardo, L.; Galli, A.; Plisov, S.; Nikolaitchik, O.A.; Chen, D.; Lockett, S.; Larson, D.R.; Pathak, V.K., *et al.* Cytoplasmic HIV-1 RNA is mainly transported by diffusion in the presence or absence of gag protein. *Proc. Natl. Acad. Sci. USA* **2014**, *111*, E5205–E5213. [CrossRef](https://doi.org/10.1073/pnas.1413879111) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25349412)

94. Zhang, W.; Cao, S.; Martin, J.L.; Mueller, J.D.; Mansky, L.M. Morphology and ultrastructure of retrovirus particles. *AIMS Biophys.* **2015**, *2*, 343–369. [CrossRef](https://doi.org/10.3934/aimsb.2015.2.343) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26839768)

95. Gamble, T.R.; Yoo, S.; Vajdos, F.F.; von Schwedler, U.K.; Worthylake, D.K.; Wang, H.; McCutcheon, J.P.; Sundquist, W.I.; Hill, C.P. Structure of the carboxyl-terminal dimerization domain of the HIV-1 capsid protein. *Science* **1997**, *278*, 849–853. [CrossRef](https://doi.org/10.1126/science.278.5344.849) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9342873)

96. Rayne, F.; Bouamr, F.; Lalanne, J.; Mamoun, R.Z. The NH2-terminal domain of the human T-cell leukemia virus type 1 capsid protein is involved in particle formation. *J. Virol.* **2001**, *75*, 5277–5287. [CrossRef](https://doi.org/10.1128/JVI.75.11.5277-5287.2001) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11333910)

97. Ganser-Pornillos, B.K.; von Schwedler, U.K.; Stray, K.M.; Aiken, C.; Sundquist, W.I. Assembly properties of the human immunodeficiency virus type 1 CA protein. *J. Virol.* **2004**, *78*, 2545–2552. [CrossRef](https://doi.org/10.1128/JVI.78.5.2545-2552.2004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14963137)

98. Ako-Adjei, D.; Johnson, M.C.; Vogt, V.M. The retroviral capsid domain dictates virion size, morphology, and coassembly of gag into virus-like particles. *J. Virol.* **2005**, *79*, 13463–13472. [CrossRef](https://doi.org/10.1128/JVI.79.21.13463-13472.2005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16223966)

99. Hogue, I.B.; Hoppe, A.; Ono, A. Quantitative fluorescence resonance energy transfer microscopy analysis of the human immunodeficiency virus type 1 Gag-gag interaction: Relative contributions of the CA and NC domains and membrane binding. *J. Virol.* **2009**, *83*, 7322–7336. [CrossRef](https://doi.org/10.1128/JVI.00418-09) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19494088)

100. Lingappa, J.R.; Reed, J.C.; Tanaka, M.; Chutiraka, K.; Robinson, B.A. How HIV-1 Gag assembles in cells: Putting together pieces of the puzzle. *Virus research* **2014**, *193*, 89–107. [CrossRef](https://doi.org/10.1016/j.virusres.2014.06.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24997086)

101. Maldonado, J.O.; Martin, J.L.; Mueller, J.D.; Zhang, W.; Mansky, L.M. New insights into retroviral Gag-gag and Gag-membrane interactions. *Frontiers Microbiol.* **2014**, *5*. [CrossRef](https://doi.org/10.3389/fmicb.2014.00005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24478754)

102. Lingwood, D.; Kaiser, H.J.; Levental, I.; Simons, K. Lipid rafts as functional heterogeneity in cell membranes. *Biochem. Soc. Trans.* **2009**, *37*, 955–960. [CrossRef](https://doi.org/10.1042/BST0370955) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19514532)

103. Ono, A. HIV-1 assembly at the plasma membrane: Gag trafficking and localization. *Futur. Virol.* **2009**, *4*, 241–257. [CrossRef](https://doi.org/10.2217/fvl.09.10) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21426120)

104. Sonnino, S.; Prinetti, A. Membrane domains and the “lipid raft” concept. *Curr. Med. Chem.* **2013**, *20*, 4–21. [PubMed](https://pubmed.ncbi.nlm.nih.gov/23110624)

105. Inlora, J.; Collins, D.R.; Trubin, M.E.; Chung, J.Y.; Ono, A. Membrane binding and subcellular localization of retroviral Gag proteins are differentially regulated by MA interactions with phosphatidylinositol-(4,5)-bisphosphate and RNA. *mBio* **2014**, *5*, e02202. [CrossRef](https://doi.org/10.1128/mBio.02202-14) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25428807)

106. Demirov, D.G.; Freed, E.O. Retrovirus budding. *Virus Res.* **2004**, *106*, 87–102. [CrossRef](https://doi.org/10.1016/j.virusres.2004.08.014) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15567490)

107. Morita, E.; Sundquist, W.I. Retrovirus budding. *Ann. Rev. Cell Dev. Biol.* **2004**, *20*, 395–425. [CrossRef](https://doi.org/10.1146/annurev.cellbio.20.010403.113931) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15473851)

108. Le Blanc, I.; Grange, M.P.; Delamarre, L.; Rosenberg, A.R.; Blot, V.; Pique, C.; Dokhelar, M.C. HTLV-1 structural proteins. *Virus Res.* **2001**, *78*, 5–16. [CrossRef](https://doi.org/10.1016/S0168-1702(01)00300-3)

109. Konvalinka, J.; Krausslich, H.G.; Muller, B. Retroviral proteases and their roles in virion maturation. *Virology* **2015**, *479–480*, 403–417. [CrossRef](https://doi.org/10.1016/j.virol.2015.03.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25843367)

110. Goncalves, D.U.; Proietti, F.A.; Ribas, J.G.; Araujo, M.G.; Pinheiro, S.R.; Guedes, A.C.; Carneiro-Proietti, A.B. Epidemiology, treatment, and prevention of human T-cell leukemia virus type 1-associated diseases. *Clin. Microbiol. Rev.* **2010**, *23*, 577–589. [CrossRef](https://doi.org/10.1128/CMR.00045-10) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20610826)

111. Wiktor, S.Z.; Pate, E.J.; Murphy, E.L.; Palker, T.J.; Champegnie, E.; Ramlal, A.; Cranston, B.; Hanchard, B.; Blattner, W.A. Mother-to-child transmission of human T-cell lymphotropic virus type I (HTLV-I) in Jamaica: Association with antibodies to envelope glycoprotein (gp46) epitopes. *J. Acquir. Immune Defic. Syndr.* **1993**, *6*, 1162–1167. [PubMed](https://pubmed.ncbi.nlm.nih.gov/8251840)

112. Takahashi, K.; Takezaki, T.; Oki, T.; Kawakami, K.; Yashiki, S.; Fujiyoshi, T.; Usuku, K.; Mueller, N.; Osame, M.; Miyata, K.; *et al.* Inhibitory effect of maternal antibody on mother-to-child transmission of human T-lymphotropic virus type I. The mother-to-child transmission study group. *Int. J. Cancer* **1991**, *49*, 673–677. [CrossRef](https://doi.org/10.1002/ijc.2910490513) [PubMed](https://pubmed.ncbi.nlm.nih.gov/1717988)

113. Nyambi, P.N.; Ville, Y.; Louwagie, J.; Bedjabaga, I.; Glowaczower, E.; Peeters, M.; Kerouedan, D.; Dazza, M.; Larouze, B.; van der Groen, G.; *et al.* Mother-to-child transmission of human T-cell lymphotropic virus types I and II (HTLV-I/II) in Gabon: A prospective follow-up of 4 years. *J. Acquir. Immune Defic. Syndr.* **1996**, *12*, 187–192. [CrossRef](https://doi.org/10.1097/00042560-199607000-00006)

114. Li, H.C.; Biggar, R.J.; Miley, W.J.; Maloney, E.M.; Cranston, B.; Hanchard, B.; Hisada, M. Provirus load in breast milk and risk of mother-to-child transmission of human T lymphotropic virus type I. *J. Infec. Dis.* **2004**, *190*, 1275–1278. [CrossRef](https://doi.org/10.1086/424643) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15472838)

115. Martin-Latil, S.; Gnadig, N.F.; Mallet, A.; Desdouits, M.; Guivel-Benhamine, F.; Jeannin, P.; Prevost, M.C.; Schwartz, O.; Gessain, A.; Ozden, S.; *et al.* Transcytosis of HTLV-1 across a tight human epithelial barrier and infection of subepithelial dendritic cells. *Blood* **2012**, *120*, 572–580. [CrossRef](https://doi.org/10.1182/blood-2011-09-380461) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22262762)

116. Kinlock, B.L.; Wang, Y.; Turner, T.M.; Wang, C.; Liu, B. Transcytosis of HIV-1 through vaginal epithelial cells is dependent on trafficking to the endocytic recycling pathway. *PloS ONE* **2014**, *9*, e96760. [CrossRef](https://doi.org/10.1371/journal.pone.0096760) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24789025)

117. Alfsen, A.; Yu, H.; Magerus-Chatinet, A.; Schmitt, A.; Bomsel, M. HIV-1-infected blood mononuclear cells form an integrin- and agrin-dependent viral synapse to induce efficient HIV-1 transcytosis across epithelial cell monolayer. *Mol. Biol. Cell* **2005**, *16*, 4267–4279. [CrossRef](https://doi.org/10.1091/mbc.E05-04-0338) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15958802)

118. Filippone, C.; Betsem, E.; Tortevoye, P.; Cassar, O.; Bassot, S.; Froment, A.; Fontanet, A.; Gessain, A. A severe bite from a nonhuman primate is a major risk factor for HTLV-1 infection in hunters from central Africa. *Clin. Infect. Dis.* **2015**, *60*, 1667–1676. [CrossRef](https://doi.org/10.1093/cid/ciu1075) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25005971)

119. Kazanji, M.; Mouinga-Ondeme, A.; Lekana-Douki-Etenna, S.; Caron, M.; Makuwa, M.; Mahieux, R.; Gessain, A. Origin of HTLV-1 in hunters of nonhuman primates in central Africa. *J. Infect. Dis.* **2015**, *211*, 361–365. [CrossRef](https://doi.org/10.1093/infdis/jiu458) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25563907)

120. LeBreton, M.; Switzer, W.M.; Djoko, C.F.; Gillis, A.; Jia, H.; Sturgeon, M.M.; Shankar, A.; Zheng, H.; Nkeunen, G.; Tamoufe, U.; *et al.* A gorilla reservoir for human T-lymphotropic virus type 4. *Emerg. Microbes Infect.* **2014**, *3*. [CrossRef](https://doi.org/10.7717/peerj.407) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24834377)

121. Miura, M.; Yasunaga, J.; Tanabe, J.; Sugata, K.; Zhao, T.; Ma, G.; Miyazato, P.; Ohshima, K.; Kaneko, A.; Watanabe, A.; *et al.* Characterization of simian T-cell leukemia virus type 1 in naturally infected Japanese macaques as a model of HTLV-1 infection. *Retrovirology* **2013**, *10*. [CrossRef](https://doi.org/10.1186/1742-4690-10-10) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23414486)

122. Song, K.J.; Nerurkar, V.R.; Saitou, N.; Lazo, A.; Blakeslee, J.R.; Miyoshi, I.; Yanagihara, R. Genetic analysis and molecular phylogeny of simian T-cell lymphotropic virus type I: Evidence for independent virus evolution in Asia and Africa. *Virology* **1994**, *199*, 56–66. [CrossRef](https://doi.org/10.1006/viro.1994.1007) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8303225)

123. Fan, N.; Gavalchin, J.; Paul, B.; Wells, K.H.; Lane, M.J.; Poiesz, B.J. Infection of peripheral blood mononuclear cells and cell lines by cell-free human T-cell lymphoma/leukemia virus type I. *J. Clin. Microbiol.* **1992**, *30*, 905–910. [PubMed](https://pubmed.ncbi.nlm.nih.gov/1373095)

124. Igakura, T.; Stinchcombe, J.C.; Goon, P.K.; Taylor, G.P.; Weber, J.N.; Griffiths, G.M.; Tanaka, Y.; Osame, M.; Bangham, C.R. Spread of HTLV-I between lymphocytes by virus-induced polarization of the cytoskeleton. *Science* **2003**, *299*, 1713–1716. [CrossRef](https://doi.org/10.1126/science.1080115) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12610304)

125. Majorovits, E.; Nejmeddine, M.; Tanaka, Y.; Taylor, G.P.; Fuller, S.D.; Bangham, C.R. Human T-lymphotropic virus-1 visualized at the virological synapse by electron tomography. *PLoS ONE* **2008**, *3*, e2251. [CrossRef](https://doi.org/10.1371/journal.pone.0002251) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18461133)

126. Nejmeddine, M.; Negi, V.S.; Mukherjee, S.; Tanaka, Y.; Orth, K.; Taylor, G.P.; Bangham, C.R. HTLV-1-Tax and ICAM-1 act on T-cellsignal pathways to polarize the microtubule-organizing center at the virological synapse. *Blood* **2009**, *114*, 1016–1025. [CrossRef](https://doi.org/10.1182/blood-2009-03-210938) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19502363)

127. Nejmeddine, M.; Barnard, A.L.; Tanaka, Y.; Taylor, G.P.; Bangham, C.R. Human T-lymphotropic virus, type 1, Tax protein triggers microtubule reorientation in the virological synapse. *J. Biol. Chem.* **2005**, *280*, 29653–29660. [CrossRef](https://doi.org/10.1074/jbc.M502558200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15890863)

128. Chevalier, S.A.; Turpin, J.; Cachat, A.; Afonso, P.V.; Gessain, A.; Brady, J.N.; Pise-Masison, C.A.; Mahieux, R. Gem-induced cytoskeleton remodeling increases cellular migration of HTLV-1-infected cells, formation of infected-to-target T-cellconjugates and viral transmission. *PLoS Pathog.* **2014**, *10*, e1003917. [CrossRef](https://doi.org/10.1371/journal.ppat.1003917) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24675976)

129. Fukudome, K.; Furuse, M.; Fukuhara, N.; Orita, S.; Imai, T.; Takagi, S.; Nagira, M.; Hinuma, Y.; Yoshie, O. Strong induction of ICAM-1 in human T cells transformed by human T-cell-leukemia virus type 1 and depression of ICAM-1 or LFA-1 in adult T-cell-leukemia-derived cell lines. *Int. J. Cancer.* **1992**, *52*, 418–427. [CrossRef](https://doi.org/10.1002/ijc.2910520311) [PubMed](https://pubmed.ncbi.nlm.nih.gov/1383050)

130. Dimitrov, D.S.; Willey, R.L.; Sato, H.; Chang, L.J.; Blumenthal, R.; Martin, M.A. Quantitation of human immunodeficiency virus type 1 infection kinetics. *J. Virol.* **1993**, *67*, 2182–2190. [PubMed](https://pubmed.ncbi.nlm.nih.gov/8455968)

131. Mazurov, D.; Ilinskaya, A.; Heidecker, G.; Lloyd, P.; Derse, D. Quantitative comparison of HTLV-1 and HIV-1 cell-to-cell infection with new replication dependent vectors. *PLoS Pathog.* **2010**, *6*, e1000788. [CrossRef](https://doi.org/10.1371/journal.ppat.1000788) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20195503)

132. Jiang, A.P.; Jiang, J.F.; Guo, M.G.; Jin, Y.M.; Li, Y.Y.; Wang, J.H. Human blood-circulating basophils capture HIV-1 and mediate viral trans-infection of CD4<sup>+</sup> T cells. *J. Virol.* 2015, 89, 8050–8062. [CrossRef](https://doi.org/10.1128/JVI.00706-15) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25972478)

133. Pais-Correia, A.M.; Sachse, M.; Guadagnini, S.; Robbiati, V.; Lasserre, R.; Gessain, A.; Gout, O.; Alcover, A.; Thoulouze, M.I. Biofilm-like extracellular viral assemblies mediate HTLV-1 cell-to-cell transmission at virological synapses. *Nat. Med.* 2010, 16, 83–89. [CrossRef](https://doi.org/10.1038/nm.2071) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20051987)

134. Tarasevich, A.; Filatov, A.; Pichugin, A.; Mazurov, D. Monoclonal antibody profiling of cell surface proteins associated with the viral biofilms on HTLV-1 transformed cells. *Acta Virol.* 2015, 59, 247–256. [CrossRef](https://doi.org/10.4149/av_2015_3_247) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26374588)

135. Alais, S.; Mahieux, R.; Dutartre, H. Viral source-independent high susceptibility of dendritic cells to human T-cell leukemia virus type 1 infection compared to that of T lymphocytes. *J. Virol.* 2015, 89, 10580–10590. [CrossRef](https://doi.org/10.1128/JVI.00706-15) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25972478)

136. Gessain, A.; Gallo, R.C.; Franchini, G. Low degree of human T-cell leukemia/lymphoma virus type I genetic drift *in vivo* as a means of monitoring viral transmission and movement of ancient human populations. *J. Virol.* 1992, 66, 2288–2295. [PubMed](https://pubmed.ncbi.nlm.nih.gov/1315083)

137. Van Dooren, S.; Pybus, O.G.; Salemi, M.; Liu, H.F.; Goubau, P.; Remondegui, C.; Talarmin, A.; Gotuzzo, E.; Alcantara, L.C.; Galvao-Castro, B.; *et al.* The low evolutionary rate of human T-cell lymphotropic virus type-1 confirmed by analysis of vertical transmission chains. *Mol. Biol. Evolut.* 2004, 21, 603–611. [CrossRef](https://doi.org/10.1093/molbev/msh051) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14739241)

138. Mansky, L.M. *In vivo* analysis of human T-cell leukemia virus type 1 reverse transcription accuracy. *J. Virol.* 2000, 74, 9525–9531. [CrossRef](https://doi.org/10.1128/JVI.74.20.9525-9531.2000) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10982371)

139. Mansky, L.M.; Temin, H.M. Lower *in vivo* mutation rate of human immunodeficiency virus type 1 than that predicted from the fidelity of purified reverse transcriptase. *J. Virol.* 1995, 69, 5087–5094. [PubMed](https://pubmed.ncbi.nlm.nih.gov/7635880)

140. Taylor, G.P.; Goon, P.; Furukawa, Y.; Green, H.; Barfield, A.; Mosley, A.; Nose, H.; Babiker, A.; Rudge, P.; Usuku, K.; *et al.* Zidovudine plus lamivudine in human T-lymphotropic virus type-I-associated myelopathy: A randomised trial. *Retrovirology* 2006, 3, 63. [CrossRef](https://doi.org/10.1186/1742-4690-3-63) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16923180)

141. Miyazato, P.; Yasunaga, J.; Taniguchi, Y.; Koyanagi, Y.; Mitsuya, H.; Matsuoka, M. *De novo* human T-cell leukemia virus type 1 infection of human lymphocytes in NOD-SCID, common γ-chain knockout mice. *J. Virol.* 2006, 80, 10683–10691. [CrossRef](https://doi.org/10.1128/JVI.00706-15) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16923180)

142. Seiki, M.; Eddy, R.; Shows, T.B.; Yoshida, M. Nonspecific integration of the HTLV provirus genome into adult T-cell leukaemia cells. *Nature* 1984, 309, 640–642. [CrossRef](https://doi.org/10.1038/309640a0) [PubMed](https://pubmed.ncbi.nlm.nih.gov/6325947)

143. Ohshima, K.; Ohgami, A.; Matsuoka, M.; Etoh, K.; Utsunomiya, A.; Makino, T.; Ishiguro, M.; Suzumiya, J.; Kikuchi, M. Random integration of HTLV-1 provirus: Increasing chromosomal instability. *Cancer Lett.* 1998, 132, 203–212. [CrossRef](https://doi.org/10.1016/S0304-3835(98)00114-5)

144. Wattel, E.; Vartanian, J.P.; Pannetier, C.; Wain-Hobson, S. Clonal expansion of human T-cell leukemia virus type I-infected cells in asymptomatic and symptomatic carriers without malignancy. *J. Virol.* 1995, 69, 2863–2868. [PubMed](https://pubmed.ncbi.nlm.nih.gov/7708058)

145. Etoh, K.; Tamiya, S.; Yamaguchi, K.; Okayama, A.; Tsubouchi, H.; Ideta, T.; Mueller, N.; Takatsuki, K.; Matsuoka, M. Persistent clonal proliferation of human T-lymphotropic virus type I-infected cells *in vivo*. *Cancer Res.* 1997, 57, 4862–4867. [PubMed](https://pubmed.ncbi.nlm.nih.gov/9350285)

146. Cavrois, M.; Gessain, A.; Wain-Hobson, S.; Wattel, E. Proliferation of HTLV-1 infected circulating cells *in vivo* in all asymptomatic carriers and patients with TSP/HAM. *Oncogene* 1996, 12, 2419–2423. [PubMed](https://pubmed.ncbi.nlm.nih.gov/8663886)

147. Zhao, T.; Satou, Y.; Matsuoka, M. Development of t cell lymphoma in HTLV-1 bZIP factor and Tax double transgenic mice. *Archives Virol.* 2014, 159, 1849–1856. [CrossRef](https://doi.org/10.1007/s00705-014-2080-3) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24619378)

148. Oeckinghaus, A.; Hayden, M.S.; Ghosh, S. Crosstalk in NF-κB signaling pathways. *Nat. Immunol.* 2011, 12, 695–708. [CrossRef](https://doi.org/10.1038/ni.2038) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21637289)

149. Peloponese, J.M., Jr.; Jeang, K.T. Role for Akt/protein kinase B and activator protein-1 in cellular proliferation induced by the human T-cell leukemia virus type 1 Tax oncoprotein. *J. Biol. Chem.* 2006, 281, 8927–8938. [CrossRef](https://doi.org/10.1074/jbc.M511833200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16436395)

150. Wu, X.; Sun, S.C. Retroviral oncoprotein Tax deregulates NF-κB by activating Tak1 and mediating the physical association of tak1-IKK. *EMBO Rep.* 2007, 8, 510–515. [CrossRef](https://doi.org/10.1038/sj.embor.7400939) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17369748)

151. Avesani, F.; Romanelli, M.G.; Turci, M.; Di Gennaro, G.; Sampaio, C.; Bidoia, C.; Bertazzoni, U.; Bex, F. Association of HTLV Tax proteins with Tak1-binding protein 2 and RelA in calreticulin-containing cytoplasmic structures participates in Tax-mediated NF-κB activation. *Virology* **2010**, *408*, 39–48. [CrossRef](https://doi.org/10.1016/j.virol.2010.07.010) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20656228)

152. Choi, Y.B.; Harhaj, E.W. HTLV-1 Tax stabilizes mcl-1 via traf6-dependent k63-linked polyubiquitination to promote cell survival and transformation. *PLoS Pathog.* **2014**, *10*, e1004458. [CrossRef](https://doi.org/10.1371/journal.ppat.1004458) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25275426)

153. Yin, M.J.; Christerson, L.B.; Yamamoto, Y.; Kwak, Y.T.; Xu, S.; Mercurio, F.; Barbosa, M.; Cobb, M.H.; Gaynor, R.B. HTLV-I Tax protein binds to mekk1 to stimulate IkB kinase activity and NF-κB activation. *Cell* **1998**, *93*, 875–884. [CrossRef](https://doi.org/10.1016/S0092-8674(00)81166-5)

154. Harhaj, E.W.; Sun, S.C. Ikky serves as a docking subunit of the IkB kinase (Ikk) and mediates interaction of IKK with the human T-cell leukemia virus Tax protein. *J. Biol. Chem.* **1999**, *274*, 22911–22914. [CrossRef](https://doi.org/10.1074/jbc.274.33.22911) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10438533)

155. Shembade, N.; Harhaj, N.S.; Yamamoto, M.; Akira, S.; Harhaj, E.W. The human T-cell leukemia virus type 1 Tax oncoprotein requires the ubiquitin-conjugating enzyme UBC13 for NF-κB activation. *J. Virol.* **2007**, *81*, 13735–13742. [CrossRef](https://doi.org/10.1128/JVI.00486-07) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17652456)

156. Israel, A. The IKK complex, a central regulator of NF-κB activation. *Cold Spring Harb. Perspect. Biol.* **2010**, *2*, a000158. [CrossRef](https://doi.org/10.1101/cshperspect.a000158) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20457561)

157. Kray, A.E.; Carter, R.S.; Pennington, K.N.; Gomez, R.J.; Sanders, L.E.; Llanes, J.M.; Khan, W.N.; Ballard, D.W.; Wadzinski, B.E. Positive regulation of IκB kinase signaling by protein serine/threonine phosphatase 2A. *J. Biol. Chem.* **2005**, *280*, 35974–35982. [CrossRef](https://doi.org/10.1074/jbc.M505869200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16024708)

158. Fu, D.X.; Kuo, Y.L.; Liu, B.Y.; Jeang, K.T.; Giam, C.Z. Human T-lymphotropic virus type I Tax activates I-κB kinase by inhibiting I-κB kinase-associated serine/threonine protein phosphatase 2A. *J. Biol. Chem.* **2003**, *278*, 1487–1493. [CrossRef](https://doi.org/10.1074/jbc.M208864200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12414818)

159. Hong, S.; Wang, L.C.; Gao, X.; Kuo, Y.L.; Liu, B.; Merling, R.; Kung, H.J.; Shih, H.M.; Giam, C.Z. Heptad repeats regulate protein phosphatase 2A recruitment to I-κB kinase γ/NF-κB essential modulator and are targeted by human T-lymphotropic virus type 1 Tax. *J. Biol. Chem.* **2007**, *282*, 12119–12126. [CrossRef](https://doi.org/10.1074/jbc.M700666200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17369380)

160. Azran, I.; Jeang, K.T.; Aboud, M. High levels of cytoplasmic HTLV-1 Tax mutant proteins retain a Tax-NF-κB-CBP ternary complex in the cytoplasm. *Oncogene* **2005**, *24*, 4521–4530. [CrossRef](https://doi.org/10.1038/sj.onc.1208586) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15864413)

161. Lamsoul, I.; Lodewick, J.; Lebrun, S.; Brasseur, R.; Burny, A.; Gaynor, R.B.; Bex, F. Exclusive ubiquitination and sumoylation on overlapping lysine residues mediate NF-κB activation by the human T-cell leukemia virus Tax oncoprotein. *Mol. Cell. Biol.* **2005**, *25*, 10391–10406. [CrossRef](https://doi.org/10.1128/MCB.25.23.10391-10406.2005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16287845)

162. Petropoulos, L.; Lin, R.; Hiscott, J. Human T cell leukemia virus type 1 Tax protein increases NF-κB dimer formation and antagonizes the inhibitory activity of the IkBα regulatory protein. *Virology* **1996**, *225*, 52–64. [CrossRef](https://doi.org/10.1006/viro.1996.0550) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8875150)

163. Shembade, N.; Harhaj, N.S.; Parvatiyar, K.; Copeland, N.G.; Jenkins, N.A.; Matesic, L.E.; Harhaj, E.W. The E3 ligase ITCH negatively regulates inflammatory signaling pathways by controlling the function of the ubiquitin-editing enzyme A20. *Nat. Immunol.* **2008**, *9*, 254–262. [CrossRef](https://doi.org/10.1038/ni1571) [PubMed](https://pubmed.ncbi.nlm.nih.gov/18235448)

164. Pujari, R.; Hunte, R.; Thomas, R.; van der Weyden, L.; Rauch, D.; Ratner, L.; Nyborg, J.K.; Ramos, J.C.; Takai, Y.; Shembade, N. Human T-cell leukemia virus type 1 (HTLV-1) Tax requires CADM1/TSLC1 for inactivation of the NF-κB inhibitor A20 and constitutive NF-κB signaling. *PLoS Pathog.* **2015**, *11*, e1004721. [CrossRef](https://doi.org/10.1371/journal.ppat.1004721) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25849598)

165. Fu, J.; Qu, Z.; Yan, P.; Ishikawa, C.; Aqeilan, R.I.; Rabson, A.B.; Xiao, G. The tumor suppressor gene WWOX links the canonical and noncanonical NF-κB pathways in HTLV-I Tax-mediated tumorigenesis. *Blood* **2011**, *117*, 1652–1661. [CrossRef](https://doi.org/10.1182/blood-2010-07-297884) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21148267)

166. Higuchi, M.; Tsubata, C.; Kondo, R.; Yoshida, S.; Takahashi, M.; Oie, M.; Tanaka, Y.; Mahieux, R.; Matsuoka, M.; Fujii, M. Cooperation of NF-κB2/p100 activation and the PDZ domain binding motif signal in human T-cell leukemia virus type 1 (HTLV-1) Tax1 but not HTLV-2 Tax2 is crucial for interleukin-2-independent growth transformation of a T-cell line. *J. Virol.* **2007**, *81*, 11900–11907. [CrossRef](https://doi.org/10.1128/JVI.00976-07) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17715258)

167. Xiao, G.; Cvijic, M.E.; Fong, A.; Harhaj, E.W.; Uhlik, M.T.; Waterfield, M.; Sun, S.C. Retroviral oncoprotein Tax induces processing of NF-κB2/p100 in T cells: Evidence for the involvement of IKKα. *EMBO J.* **2001**, *20*, 6805–6815. [CrossRef](https://doi.org/10.1093/emboj/20.23.6805) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11714945)

168. Fraedrich, K.; Muller, B.; Grassmann, R. The HTLV-1 Tax protein binding domain of cyclin-dependent kinase 4 (CDK4) includes the regulatory pstaire helix. *Retrovirology* **2005**, *2*, 54. [CrossRef](https://doi.org/10.1186/1742-4690-2-54) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15938758)

169. Suzuki, T.; Yoshida, M. HTLV-1 Tax protein interacts with cyclin-dependent kinase inhibitor p16INK4A and counteracts its inhibitory activity to CDK4. *Leukemia* **1997**, *11* (Suppl. 3), 14–16. [PubMed](https://pubmed.ncbi.nlm.nih.gov/9220377)

170. Liu, Y.; Wang, Y.; Yamakuchi, M.; Masuda, S.; Tokioka, T.; Yamaoka, S.; Maruyama, I.; Kitajima, I. Phosphoinositide-3 kinase-PKB/Akt pathway activation is involved in fibroblast Rat-1 transformation by human T-cell leukemia virus type I Tax. *Oncogene* **2001**, *20*, 2514–2526. [CrossRef](https://doi.org/10.1038/sj.onc.1204358) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11368146)

171. Kim, S.J.; Kehrl, J.H.; Burton, J.; Tendler, C.L.; Jeang, K.T.; Danielpour, D.; Thevenin, C.; Kim, K.Y.; Sporn, M.B.; Roberts, A.B. Transactivation of the transforming growth factor β 1 (TGF-β1) gene by human T lymphotropic virus type 1 Tax: A potential mechanism for the increased production of TGF-β1 in adult T cell leukemia. *J. Exp. Med.* **1990**, *172*, 121–129. [CrossRef](https://doi.org/10.1084/jem.172.1.121) [PubMed](https://pubmed.ncbi.nlm.nih.gov/2165683)

172. Moriuchi, M.; Moriuchi, H. Transforming growth factor-β enhances human T-cell leukemia virus type I infection. *J. Med. Virol.* **2002**, *67*, 427–430. [CrossRef](https://doi.org/10.1002/jmv.10145) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12115880)

173. Ariumi, Y.; Kaida, A.; Lin, J.Y.; Hirota, M.; Masui, O.; Yamaoka, S.; Taya, Y.; Shimotohno, K. HTLV-1 Tax oncoprotein represses the p53-mediated trans-activation function through coactivator CBP sequestration. *Oncogene* **2000**, *19*, 1491–1499. [CrossRef](https://doi.org/10.1038/sj.onc.1203416) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10734493)

174. Suzuki, T.; Uchida-Toita, M.; Yoshida, M. Tax protein of HTLV-1 inhibits CBP/p300-mediated transcription by interfering with recruitment of CBP/p300 onto DNA element of E-box or p53 binding site. *Oncogene* **1999**, *18*, 4137–4143. [CrossRef](https://doi.org/10.1038/sj.onc.1202728) [PubMed](https://pubmed.ncbi.nlm.nih.gov/10376404)

175. Zane, L.; Yasunaga, J.; Mitagami, Y.; Yedavalli, V.; Tang, S.W.; Chen, C.Y.; Ratner, L.; Lu, X.; Jeang, K.T. WIP1 and p53 contribute to HTLV-1 Tax-induced tumorigenesis. *Retrovirology* **2012**, *9*, 114. [CrossRef](https://doi.org/10.1186/1742-4690-9-114) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23216906)

176. Harashima, N.; Kurihara, K.; Utsunomiya, A.; Tanosaki, R.; Hanabuchi, S.; Masuda, M.; Ohashi, T.; Fukui, F.; Hasegawa, A.; Masuda, T.; et al. Graft-versus-Tax response in adult T-cell leukemia patients after hematopoietic stem cell transplantation. *Cancer Res.* **2004**, *64*, 391–399. [CrossRef](https://doi.org/10.1158/0008-5472.CAN-03-2778) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14744776)

177. Rowan, A.G.; Suemori, K.; Fujiwara, H.; Yasukawa, M.; Tanaka, Y.; Taylor, G.P.; Bangham, C.R. Cytotoxic T lymphocyte lysis of HTLV-1 infected cells is limited by weak HBZ protein expression, but non-specifically enhanced on induction of Tax expression. *Retrovirology* **2014**, *11*. [CrossRef](https://doi.org/10.1186/s12977-014-0108-3) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24985386)

178. Takeda, S.; Maeda, M.; Morikawa, S.; Taniguchi, Y.; Yasunaga, J.; Nosaka, K.; Tanaka, Y.; Matsuoka, M. Genetic and epigenetic inactivation of Tax gene in adult T-cell leukemia cells. *Int. J. Cancer* **2004**, *109*, 559–567. [CrossRef](https://doi.org/10.1002/ijc.11712) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14983508)

179. Furukawa, Y.; Kubota, R.; Tara, M.; Izumo, S.; Osame, M. Existence of escape mutant in HTLV-I Tax during the development of adult T-cell leukemia. *Blood* **2001**, *97*, 987–993. [CrossRef](https://doi.org/10.1182/blood.V97.4.987) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11157481)

180. Tamiya, S.; Matsuoka, M.; Etoh, K.; Watanabe, T.; Kamihira, S.; Yamaguchi, K.; Takatsuki, K. Two types of defective human T-lymphotropic virus type I provirus in adult T-cell leukemia. *Blood* **1996**, *88*, 3065–3073. [PubMed](https://pubmed.ncbi.nlm.nih.gov/8879986)

181. Koiwa, T.; Hamano-Usami, A.; Ishida, T.; Okayama, A.; Yamaguchi, K.; Kamihira, S.; Watanabe, T. 5′-long terminal repeat-selective CPG methylation of latent human T-cell leukemia virus type 1 provirus *in vitro* and *in vivo*. *J. Virol.* **2002**, *76*, 9389–9397. [CrossRef](https://doi.org/10.1128/JVI.76.18.9389-9397.2002) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12183599)

182. Gaudray, G.; Gachon, F.; Basbous, J.; Biard-Piechaczyk, M.; Devaux, C.; Mesnard, J.M. The complementary strand of the human T-cell leukemia virus type 1 RNA genome encodes a bZIP transcription factor that down-regulates viral transcription. *J. Virol.* **2002**, *76*, 12813–12822. [CrossRef](https://doi.org/10.1128/JVI.76.24.12813-12822.2002) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12414958)

183. Satou, Y.; Yasunaga, J.; Yoshida, M.; Matsuoka, M. HTLV-I basic leucine zipper factor gene mRNA supports proliferation of adult T cell leukemia cells. *Proc. Natl. Acad. Sci. USA* **2006**, *103*, 720–725. [CrossRef](https://doi.org/10.1073/pnas.0508685103) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16407138)

184. Mitobe, Y.; Yasunaga, J.; Furuta, R.; Matsuoka, M. HTLV-1 bZIP factor rna and protein impart distinct functions on T-cell proliferation and survival. *Cancer Res.* **2015**, *75*, 4143–4152. [CrossRef](https://doi.org/10.1158/0008-5472.CAN-15-0102) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26068497)

185. Lemasson, I.; Lewis, M.R.; Polakowski, N.; Hivin, P.; Cavanagh, M.H.; Thebault, S.; Barbeau, B.; Nyborg, J.K.; Mesnard, J.M. Human T-cell leukemia virus type 1 (HTLV-1) bZIP protein interacts with the cellular transcription factor CREB to inhibit HTLV-1 transcription. *J. Virol.* **2007**, *81*, 1543–1553. [CrossRef](https://doi.org/10.1128/JVI.01949-06) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17101685)

186. Hagiya, K.; Yasunaga, J.; Satou, Y.; Ohshima, K.; Matsuoka, M. ATF3, an HTLV-1 bZip factor binding protein, promotes proliferation of adult T-cell leukemia cells. *Retrovirology* **2011**, *8*, 19. [CrossRef](https://doi.org/10.1186/1742-4690-8-19) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21356087)

187. Ma, Y.; Zheng, S.; Wang, Y.; Zang, W.; Li, M.; Wang, N.; Li, P.; Jin, J.; Dong, Z.; Zhao, G. The HTLV-1 HBZ protein inhibits cyclin D1 expression through interacting with the cellular transcription factor CREB. *Mol. Biol. Rep.* **2013**, *40*, 5967–5975. [CrossRef](https://doi.org/10.1007/s11033-013-2616-2) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23681456)

188. Wurm, T.; Wright, D.G.; Polakowski, N.; Mesnard, J.M.; Lemasson, I. The HTLV-1-encoded protein HBZ directly inhibits the acetyl transferase activity of p300/CBP. *Nucleic Acids Res.* **2012**, *40*, 5910–5925. [CrossRef](https://doi.org/10.1093/nar/gks248) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22438471)

189. Borowiak, M.; Kuhlmann, A.S.; Girard, S.; Gazzolo, L.; Mesnard, J.M.; Jalinot, P.; Dodon, M.D. HTLV-1 bZIP factor impedes the menin tumor suppressor and upregulates JunD-mediated transcription of the *Htert* gene. *Carcinogenesis* **2013**, *34*, 2664–2672. [CrossRef](https://doi.org/10.1093/carcin/bgt156) [PubMed](https://pubmed.ncbi.nlm.nih.gov/23716725)

190. Hivin, P.; Basbous, J.; Raymond, F.; Henaff, D.; Arpin-Andre, C.; Robert-Hebmann, V.; Barbeau, B.; Mesnard, J.M. The HBZ-SP1 isoform of human T-cell leukemia virus type I represses JunB activity by sequestration into nuclear bodies. *Retrovirology* **2007**, *4*, 14. [CrossRef](https://doi.org/10.1186/1742-4690-4-14) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17326820)

191. Thebault, S.; Basbous, J.; Hivin, P.; Devaux, C.; Mesnard, J.M. HBZ interacts with JunD and stimulates its transcriptional activity. *FEBS Lett.* **2004**, *562*, 165–170. [CrossRef](https://doi.org/10.1016/S0014-5793(04)00225-7)

192. Sugata, K.; Yasunaga, J.I.; Mitobe, Y.; Miura, M.; Miyazato, P.; Kohara, M.; Matsuoka, M. Protective effect of cytotoxic T lymphocytes targeting HTLV-1 bZIP factor. *Blood* **2015**, *126*, 1095–1105. [CrossRef](https://doi.org/10.1182/blood-2015-03-634785) [PubMed](https://pubmed.ncbi.nlm.nih.gov/26101142)
