
Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

Research Article

Distinct subtypes of gastric cancer defined by molecular characterization include novel mutational signatures with prognostic capability

Xiangchun Li${}^{1,4}$, William KK Wu${}^{1,2}$, Rui Xing${}^{3}$, Sunny H. Wong${}^{1}$, Yuexin Liu${}^{5}$, Xiaodong Fang${}^{4}$, Yanlin Zhang${}^{7}$, Mengyao Wang${}^{4}$, Jiaqian Wang${}^{4}$, Lin Li${}^{4}$, Yong Zhou${}^{4}$, Senwei Tang${}^{1}$, Shaoliang Peng${}^{8}$, Kunlong Qiu${}^{4}$, Longyun Chen${}^{4}$, Kexin Chen${}^{5}$, Huanming Yang${}^{4}$, Wei Zhang${}^{6}$, Matthew T.V. Chan${}^{2}$, Youyong Lu${}^{3}$, Joseph JY Sung${}^{1}$, Jun Yu${}^{1}$

${}^{1}$Institute of Digestive Disease and Department of Medicine & Therapeutics, State Key Laboratory of Digestive Disease, LKS Institute of Health Sciences, CUHK Shenzhen Research Institute, The Chinese University of Hong Kong, Hong Kong.

${}^{2}$Department of Anaesthesia and Intensive Care, The Chinese University of Hong Kong, Hong Kong.

${}^{3}$Laboratory of Molecular Oncology, Key Laboratory of Carcinogenesis and Translational Research (Ministry of Education), Peking University Cancer Hospital & Institute, Beijing, 100142, China.

${}^{4}$Beijing Genomics Institute-Shenzhen, Shenzhen 518083, Guangdong, People's Republic of China.

${}^{5}$Department of Epidemiology and Biostatistics, Tianjin Medical University Cancer Institute and Hospital, Tianjin 300060, People’s Republic of China.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443

Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

$^{6}$Department of Pathology, University of Texas MD Anderson Cancer Center Informatics Center, Houston, TX 77030.

$^{7}$Department of Computer Science, City University of Hong Kong.

$^{8}$School of Computer Science, National University of Defense Technology, Changsha, Hunan, 410073, PR China.

Running Title: Genomic analysis of gastric cancer

Keywords: gastric cancer; genome; prognostic marker; E-cadherin

Financial support: The work was supported by Cancer Genome Project of the Chinese University of Hong Kong (2009).

Conflict of interest statement: The authors declare that they have no conflict of interest.

Word count: 3,680 words

Number of figures: 5

Number of references: 48

Corresponding authors:

Prof. Jun Yu, Institute of Digestive Disease, Department of Medicine and Therapeutics, The Chinese University of Hong Kong, Shatin, NT, Hong Kong. Tel: (852) 3763 6099; Fax: (852) 2144 5330; Email: junyu@cuhk.edu.hk; or Dr. William KK Wu, Department of Anaesthesia and Intensive Care, The Chinese University of

2

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443

Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

Hong Kong, Shatin, NT, Hong Kong. Tel: (852) 2632 3147; Fax: (852) 2637 2422;

Email: [wukakei@cuhk.edu.hk](http://wukakei@cuhk.edu.hk)

3

Downloaded from [cancerres.aacrjournals.org](http://cancerres.aacrjournals.org) on February 17, 2016. © 2016 American Association for Cancer Research.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

Abstract

Gastric cancer is not a single disease and its subtype classification is still evolving. Next-generation sequencing studies have identified novel genetic drivers of gastric cancer, but their use as molecular classifiers or prognostic markers of disease outcome have yet to be established. In this study, we integrated somatic mutational profiles and clinicopathological information from 544 gastric cancer patients by previous genomics studies to identify significantly mutated genes with prognostic relevance. Gastric cancer patients were classified into regular (86.8%) and hyper-mutated (13.2%) subtypes based on mutation burden. Notably, TpCpW mutations occurred significantly more frequently in regular, but not hyper-mutated, gastric cancers where they were associated with APOBEC expression. In the former group, six previously unreported (XIRP2, NBEA, COL14A1, CNBD1, ITGAV, AKAP6) and 12 recurrent mutated genes exhibited high mutation prevalence (≥ 3.0%) and an unexpectedly higher incidence of nonsynonymous mutations. We also identified two molecular subtypes of regular-mutated gastric cancer that were associated with distinct prognostic outcomes, independently of disease staging, as confirmed in a distinct patient cohort by targeted capture sequencing. Lastly, in diffuse-type gastric cancer, CDH1 mutation was found to be associated with shortened patient survival, independently of disease staging. Overall, our work identified previously unreported significantly mutated genes and a mutation signature predictive of patient survival in newly classified subtypes of gastric cancer, offering opportunities to stratify patients into optimal treatment plans based on molecular subtyping.

Keywords: gastric cancer; genome; prognostic marker; E-cadherin

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443

Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

# Introduction

Despite a decrease in both incidence and mortality owing to progresses in *Helicobacter pylori* eradication and cancer screening, gastric cancer remains the 7th most common cancer and the third leading cause of cancer death worldwide with a 5-year survival rate of 29.6% (1). According to Lauren classification, gastric cancer can be divided into two distinct subtypes (i.e. intestinal- and diffuse-type) with substantial differences in histology and pathogenesis (2). The intestinal type is featured by well-differentiated, glandular neoplastic cells that are structurally analogous to intestinal cells and more likely to occur in area with high risk of gastric cancer (2); whereas the diffuse-type is characterized by poorly differentiated neoplastic cells that morphologically resemble signet ring cells and exhibit deeper invasion and infiltration of the whole stomach wall (3-4).

Cancer is a genetic disease arising from changes in DNA sequences (5-6). Many endogenous and exogenous factors can cause somatic mutations, such as defective DNA repair, infidelity in DNA replication and mutagenic exposures. The landscape of somatic mutations bear the signatures of mutagenic factors (i.e. mutational processes) that have acted upon the genome (7). For example, a vast majority of somatic mutations in skin and lung cancers are associated with ultraviolet and smoking exposure, respectively. Until recently, with the benefit of vast amount of genome data, researchers began to elucidate mutational signatures in human cancers. Alexandrov et al. identified 21 mutational processes of 30 different types of primary cancers, some of which were linked to known factors including DNA mismatch repair deficiency and APOBEC overactivity (8). Several independent studies also reported that APOBEC-mediated mutagenesis is widespread across human cancers and

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443

Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

APOBEC3B is responsible for clustered mutation in cancer genomes, as well as PIK3CA hotspot mutations (9-11).

Next-generation sequencing has become a powerful tool to elucidate potential driver genetic aberrations underlying cancer development. In gastric cancer, Wang et al. and Zang et al. reported frequent somatic mutations in chromatin remodeling and cell adhesion genes (e.g., ARID1A, MLL, MLL3, and FAT4), respectively (12-13). Comprehensive genomic characterizations conducted by The Cancer Genome Atlas (TCGA) (14) and Wang et al. (15) further revealed genetic alterations underlying gastric carcinogenesis with multi-omics profiling. Together with study conducted by Kakiuchi et al. (16), these three studies highlighted a novel driver gene RHOA in diffuse-type gastric cancer with varying mutational prevalence. Through whole-exome deep sequencing, we reported that high-clonality is correlated with poorer prognosis in gastric cancer from the Chinese population (17). These studies also implicated that gastric cancer with mutator phenotype has a higher mutational burden and is more likely to exhibit microsatellite instability. Nevertheless, our knowledge of cancer driver genes, especially those mutated at low or intermediate frequencies, in gastric cancer is far from complete because of limited sample size in these studies (18). The mutational processes underlying gastric cancers with or without mutator phenotype are still obscure and have not been investigated. Moreover, gastric cancer is genetically and clinically heterogeneous. Thus, the lack of systematic clinical correlation analyses impedes translation of these findings into clinical benefits. Therefore, deeper insights into gastric carcinogenesis, particularly related to clinical properties, are needed.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443

Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

The purposes of this study are to characterize mutational processes operative in gastric cancer and to identify previously unreported significantly mutated genes (SMGs) and prognosticators for patients with gastric cancer.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

# Materials and Methods

## Genome data

All somatic mutations including single nucleotide substitution and short insertion/deletion were collected from recent publications (Supplementary Table 1) representing 5 geographically different studied cohorts and annotated by ANNOVAR (19). All somatic mutations were examined in a panel of 442 sequenced normal samples. Mutations present in this panel were removed. Clinical data were also acquired from these publications (Supplementary Dataset 1). However, only TCGA and Tianjin cohorts have patients' follow-up and vital status.

## Clustering based on mutation loads

We applied the optimal k-means clustering via dynamic programming^{20} to the number of somatic mutations identified in each case. It can guarantee that the within-cluster distance for each cluster is always minimum. Given that the exact cluster number is unknown, we ran this algorithm with different cluster numbers ranging from 1 to 9. The optimal cluster number was selected based on Bayesian information criterion. Finally, an exact cluster number of 6 was selected. Through visual inspection, we divided 544 cases into regular-mutated and hyper-mutated groups for follow-up analyses. The optimal k-means clustering was performed with R package Ckmeans.1d.dp (20).

## Identifying genes overrepresented in the hyper-mutated gastric cancer

To identify genes overrepresented in the hyper-mutated group, we need to take into account different background mutation rates between the hyper- and regular-mutated groups, as well as different background mutation rates among mutational categories.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443

Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

Herein, we used goodness-of-fit test with Poisson distribution to address this issue.

The background mutation rate for the $i^{\text{th}}$ category in the regular- and hyper-mutated groups are $\alpha_i$ and $\beta_i$, respectively. For a given gene, the number of nucleotides in the $i^{\text{th}}$ category is $n_i$. The expected number of mutations given $\alpha_i$, $\beta_i$ and $n_i$ across all categories $\phi$ can be estimated by Poisson distribution in regular- and hyper-mutated gastric cancers, i.e. $\lambda_{\alpha} = \sum_{\phi} \alpha_i * n_i$ and $\lambda_{\beta} = \sum_{\phi} \beta_i * n_i$, respectively. The $\alpha_i$ and $\beta_i$ can be obtained after running MutSigCV${}^{21}$ with $\pi_{\alpha}$ and $\pi_{\beta}$ as the numbers of observed mutations in the regular- and hyper-mutated gastric cancers, respectively.

To test whether a gene is significantly overrepresented in the hyper-mutated group, we examined the difference between $\pi_{\beta}/\pi_{\alpha}$ and $\lambda_{\beta}/\lambda_{\alpha}$. If $\pi_{\beta}/\pi_{\alpha}$ is significantly greater than $\lambda_{\beta}/\lambda_{\alpha}$, this gene was considered to be significantly overrepresented in the hyper-mutated group.

### Identification of significantly mutated genes

We identified significantly mutated genes with three algorithms using MutSigCV, MutSigCL and MutSigFN. MutSigCV (21) quantifies significance of non-silent mutations in a gene with background mutation rate estimated by silent mutations with other confounding covariates taken into account. MutSigCL and MutSigFN measure the significance of clustered mutations and functional impact of mutation, respectively (18). In MutSigFN analysis, we separately used CADD and Polyphen2 scores available from dbNSFP database (22) to measure functional impact of somatic mutations. For efficient computation, we carried out a two-step permutation. We performed 999 times in the first step to define candidate SMGs (i.e. $P < 0.05$) followed by extensive permutation with 1,000,000 times. We then combined $P$ values obtained from the $1^{\text{st}}$ and $2^{\text{nd}}$ steps. $P$ values were then false discovery rate (FDR)-

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

corrected ($q$ values) using the method of Benjamini and Hochberg. For the final analysis of SMGs, we applied additional filtering steps to eliminate possible false positives that may result from the batch effect via combining somatic mutations from different studies. In the regular-mutated group, a gene was considered to be a SMG if it satisfied these conditions: (1) statistically significant ($q$-value<0.1) by at least one of MutSig algorithms; (2) expressed in the TCGA pan-cancer dataset (23), human cancer cell lines (24) and/or reported in previous studies (18, 23, 25, 26); (3) mutated in at least 3 out 5 cohorts; (4) mutational prevalence comparable among different cohorts (if mutated). In the hyper-mutated group, we employed similar but more stringent criteria to select SMGs, including statistically significant by at least 2 MutSig algorithms and mutated in at least 2 out of the 3 available cohorts (i.e. Hong Kong, TCGA and Tianjin China). This produced a final list of 66 SMGs in the hyper-mutated group.

## Molecular typing

We applied nonnegative matrix factorization (NMF)(27-29) to perform molecular subtyping. A binary matrix $A$ describing mutations of SMGs (rows) across cancer samples (columns) was constructed. Specifically, $a(i, j)=1$ if gene $i$ was mutated in sample $j$, otherwise $a(i, j)=0$; then $A$ was factorization into 2 nonnegative matrices $W$ and $H$ (i.e. $A \approx WH$). Matrix $H$ was used to group samples into clusters. Optimal number of clusters were selected based on cophenetic coefficient and dispersion value (29-30).

## Targeted Capture Sequencing

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443

Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

Genomic DNA from gastric cancers and lymphocytes was fragmented and hybridized  
to commercially available capture arrays for enrichment. All samples were collected  
from patients diagnosed with primary gastric cancer with long-term survival data (15-  
60 months). Data was analyzed using a bioinformatic pipeline as previously described  
(31). All patients had given written informed consent and the study protocol was  
approved by the clinical ethics committee of the Peking University Cancer Hospital &  
Institute.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

# Results

## Classification of gastric cancer based on mutation loads

Somatic mutational profiles of 544 gastric cancer patients from previous genomics studies were aggregated. These samples exhibited large variation in mutation density, ranging from 0 to 200.2 mutations per megabase pair (Mb). To avoid undue effects of samples with high-burden mutation on subsequent analyses (32-33), unsupervised clustering of these 544 gastric cancer samples was performed according to their number of somatic mutations. Two distinct clusters with varying mutation burdens were identified (Fig. 1A; Supplementary Fig. 1A), which were hereafter referred to as regular- (n = 455; 2.4 mutations per Mb; range: 0-8.3) and hyper-mutated (n = 89; 20.5 mutations per Mb; range: 9.6-200.2) gastric cancer, the latter of which showed marked overrepresentation of samples with microsatellite instability (Fisher’s exact test; odds ratio (OR) = 1012.4; p < 0.001). Although several genes (BRCA2, FANCM, PRKDC, MSH3, etc.) that are involved in maintaining genomic integrity were frequently mutated in the hyper-mutated group, these genes were not significantly enriched when the heterogeneity of background mutation rates was considered (Supplementary Dataset 2).

## Mutation signatures of gastric cancer in relation to clinicopathological features and APOBEC3B expression

To gain further insights into the mutational processes operative in regular- and hyper-mutated gastric cancer, we delineated their mutation signatures using the computational framework proposed by Alexandrov et al. (34). Seven mutation signatures were extracted from all 544 gastric cancer samples, namely Signatures 1-7, each of which contributed to different proportion of mutations in the regular- and

hyper-mutated groups (Supplementary Fig. 1B; Fig. 1B and C). In kernel principle component analysis, we found that mutation signatures corresponding to different studies were intermingled whereas the regular- and hyper-mutated groups were clearly distinguishable (Supplementary Fig. 1C), suggesting a minimal impact of batch effect on mutation signatures. General linear regression was performed to analyze the relationship between mutation signatures and clinicopathological features.

We observed that age at diagnosis was associated with Signature 2 (Fig. 1C), which is concordant with a previous report (8). Of interest, Signature 6, which was dominated by mutations at TpCpW DNA motif (where W=A or T; mutated nucleotide underlined), accounted for 32.5% mutations in regular- but only 3.2% in hyper-mutated gastric cancer. In this regard, regular-mutated gastric cancer harbored 6-times more mutations at TpCpW DNA motif than the hyper-mutated group (Wilcox test; median: 0.06 vs. 0.01; \( p < 0.001 \); Fig. 1D). This mutation pattern is also known as APOBEC signature, which is widespread across multiple human cancer types (10).

The dominance of TpCpW mutations in regular-mutated gastric cancer could be exemplified by the mutation pattern of PIK3CA (Fig. 1E), which is a driver gene in gastric cancer. The RNA-editing enzyme APOBEC3B has been reported to contribute to mutations at TpCpW motif in cancer genome (10). To interrogate the contribution of APOBEC3B to mutations in gastric cancer, we analyzed the relationship between its expression levels and number of TpCpW mutations. We observed that APOBEC3B mRNA levels were positively correlated with the number of TpCpW in regular- (Spearman’s rho = 0.42; \( p < 0.001 \); Fig. 1F) but not hyper-mutated gastric cancer (Spearman’s rho=0.21; \( q=0.67 \); Fig. 1F) despite similar expression levels in both groups (\( q = 0.32 \); Fig. 1F), signifying their difference in mutagenesis. Since

regular- and hyper-mutated gastric cancers were characterized by distinct molecular features, these two subgroups were analyzed separately in subsequent analyses.

# Significantly mutated genes in gastric cancer

A total number of 39,891 and 101,189 non-silent somatic mutations, including missense, nonsense, splice-site and frame-shift mutations, were detected in 455 and 89 cases of regular- and hyper-mutated gastric cancer, respectively. To identify SMGs that are causally linked to tumorigenesis, we used three algorithms, namely MutSigCV (21), MutSigCL and MutSigFN to identify genes whose mutations were positively selected, clustered in hotspots or of functional consequences. In regular-mutated gastric cancer, 31 SMGs were identified (Fig. 2A; Supplementary Dataset 2), among which twelve reported [TP53 (48.4%), ARID1A (13.8%), CDH1 (11.6%), PIK3CA (8.4%), APC (6.8%), RHOA (5.7%), SMAD4 (6.2%), ERBB4 (5.7%), KRAS (4.4%), ERBB2 (3.2%), CTNNB1 (3.1%)] and six previously unreported [XIPR2 (7.3%), NBEA (7.0%) COL14A1 (4.4%), AKAP6 (3.7%), CNBD1 (3.1%), ITGAV (3.1%)] SMGs exhibited moderate-to-high mutation prevalence (≥3.0%). In particular, four genes, namely TP53, CDH1, SMAD4 and CTNNB1 were ranked as significantly mutated by all 3 algorithms. The 6 previously unreported SMGs were frequently mutated across multiple human cancer types (35–36) (Supplementary Dataset 3). ITGAV, upstream regulator of PI3K signaling pathways and located in 2q32.1, was significantly deleted in Hong Kong and TCGA cohorts (14–15). NBEA (located in 13q12.11) and CNBD1 (located in 8q22) were focally deleted in TCGA cohort with the latter significantly mutated in 1% of previous pan-cancer data set but not in individual cancer types (18). AKAP6, protein kinase anchoring factor, was reported to be significantly mutated in esophageal adenocarcinoma (37). COL14A1

were reported to be frequently mutated and down-regulated in other cancer types (38-39). In the hyper-mutated group, we first used 3 MutSig algorithms to define SMGs by evaluating only single-nucleotide substitutions, giving rise to 4 SMGs, including *ARID1A* and *TP53* (Supplementary Dataset 2). By including short insertions/deletions, the final list of SMGs was expanded to 66, including *MLL2*, *RNF43, B2M, ACVR2A* and *RNF43* (Supplementary Dataset 2).

**SMG mutation patterns predictive of survival in regular-mutated gastric cancer**

To determine if SMG mutation status could be used for molecular typing, NMF-based unsupervised clustering on binary mutation matrix of SMGs was performed, which yielded two subgroups (hereafter referred to as C1 and C2; Supplementary Fig. 1D). The first subgroup C1 was enriched with mutations in *TP53* (89.9%; *q* < 0.001), *XIRP2* (13.7%; *q*<0.001), *APC* (11.1%; *q* = 0.01), *ERBB4* (8.4%; *q*=0.047) and *AKAP6* (6.6%; *q*=0.004) whereas the second subgroup C2 was overrepresented by mutations in *ARID1A* (27.5%; *q* < 0.001), *CDH1* (17.5%; *q* < 0.001), *PIK3CA* (14.4%; *q* < 0.001) and *RHOA* (9.2%; *q* = 0.007). C2 was also featured with more gastric body-located (OR = 1.71; 95% CI: 1.0-2.93; *p* = 0.05) and diffuse-type (OR = 2.45; 95% CI: 1.49-4.06; *p*<0.001) gastric cancer. In comparison with TCGA molecular subtypes, we found that C1 was more enriched with chromosome instability (CIN) subtype (Fisher's exact test, *p*<0.001) whereas C2 had even distribution of CIN and genome stable (GS) subtypes (45 CINs vs. 42 GSs).

Importantly, C1 was associated with a significantly better prognostic outcome (*p* = 0.02; Fig. 2B). Univariate analysis indicated that advanced TNM staging (Stage III/IV) and C2 subtype but not other factors, including tumor location (antrum, body or cardia), were significantly associated with poorer survival outcome (Fig. 2C).

Multivariate analysis revealed that the prognostic significance of C1/2 was independent of age, sex, Lauren classification, TNM staging and studied cohorts (Fig. 2D). Nevertheless, subgroup analysis stratified by Lauren's classification indicated that the prognostic significance of C1/2 was more apparent in diffuse-type than intestinal-type gastric cancer (Supplementary Fig. 2A).

### C1/2 signature validation in an independent cohort

We observed that 8 (TP53, ARID1A, CDH1, PIK3CA, XIRP2, APC, ERBB2 and RHOA) out of the abovementioned 31 SMGs exhibited differential mutation frequencies with mutational prevalence >5%. Importantly, we found that these 8 SMGs could achieve comparable prediction accuracy in discriminating C1 from C2 as by the 31 SMGs (Supplementary Fig. 2B). By performing targeted capture sequencing in an independent cohort of gastric cancer patients and classifying these patients into C1 and C2 subgroups based on these 8 SMGs, we found that C1/2 remained a significant independent prognostic marker in gastric cancer (Supplementary Fig. 2C).

### Mutation distribution in cancer signaling pathways

By mapping SMGs and other well-known genes to cell signaling pathways, we observed that several pathways were frequently altered in regular-mutated gastric cancer, including genotoxic/oncogenic stress response (57.6%), histone modification/chromatin remodeling (26.6%) and growth factor receptor signaling (22.4%) and Wnt signaling (24.4%) (Fig. 4).

### Recurrent point mutations in gastric cancer

Recurrent point mutations could help to identify cancer driver genes and druggable targets (40). Herein, we depicted the panorama of recurrent point mutations in gastric cancer (Supplementary Dataset 2). Out of 31 SMGs in regular-mutated gastric cancer, twenty of them were significantly enriched for recurrent point mutations, including TP53, PIK3CA, CDH1, KRAS, RHOA, ERBB2 and ERBB4. Nevertheless, only three SMGs, namely TP53, PIK3CA and KRAS, were identified in hyper-mutated gastric cancer. Notably, two recurrent mutations (p.E542K and p.E545K) in PIK3CA are significantly overrepresented in the regular-mutated gastric cancer (OR=4.26; Fisher's exact test, \( p=0.004 \)), which were exclusively located at TpCpW motif (Fig. 3A). Conversely, another PIK3CA hotspot mutation (p.H1047R) was significantly enriched in the hyper-mutated group (OR=4.46; Fisher's exact test, \( p=0.02 \)), which occurred in ApTpG rather than TpCpW motif (Fig. 3B). For other SMGs, recurrent point mutations of CDH1 and RHOA were only observed in regular-mutated but not hyper-mutated gastric cancers.

### CDH1 mutation as a prognostic factor in diffuse-type gastric cancer

To discover single-gene prognosticators, we analyzed the association between nine SMGs (mutated at >5%) and survival data. We found that CDH1 and SMAD4 mutations were significantly associated with shortened survival in patients with gastric cancer as revealed by Kaplan-Meier analysis. In subgroup analysis stratified by Lauren’s subtype, CDH1 mutation(s) was a significant prognostic factor in diffuse-type but not intestinal-type gastric cancer independent of TNM staging (Supplementary Fig. 3A). In contrast, SMAD4 mutation was associated with shortened survival only in the intestinal-type gastric cancer. We then verified the prognostic significance of CDH1 and SMAD4 in an independent cohort (17), in which

these two genes were sequenced on an orthogonal sequencing platform. We observed that $CDH1$ mutations remained an independent factor for poor survival in the diffuse-type gastric cancer (Supplementary Fig. 3B). However, the significance of $SMAD4$ mutations could not be verified. The prognostic significance of $CDH1$ in the diffuse-type gastric cancer by combined analysis of two cohorts was shown in Fig. 5.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

# Discussion

In this study, we performed systematic analyses on 544 gastric cancers and correlated genetic events with clinicopathological features. Major findings derived from our study include: (1) there are ubiquitous and specific mutational processes underlying the pathogenesis of different subtypes of gastric cancer with varying mutation burdens; (2) several previously unreported SMGs that are mutated at intermediate or low prevalence were identified; (3) regular-mutated gastric cancer can be further stratified into two subtypes (i.e. C1 and C2) with distinct clinical outcomes; and (4) CDH1 mutation is an independent prognostic factor for poorer survival in patients with diffuse-type gastric cancer.

In line with previous observation (8, 10), we revealed that APOBEC-mediated mutation pattern is ubiquitous in gastric cancer. Although hyper-mutated gastric cancer that showed defects in DNA damage repair is more likely to generate single-strand DNA breakage, which is an ideal substrate for APOBEC family of cytidine deaminases (41), this mutational process contributed to the majority of somatic mutations in regular-mutated but not hyper-mutated gastric cancer. Besides, we observed that PIK3CA mutations (p.E542K and p.E545K) of regular-mutated gastric cancer are more likely to occur in TpCpW motif in comparison to the hyper-mutated group. These discrepancies have not been reported in gastric cancer. Concordant with our findings, a recent study on esophageal squamous cell carcinoma reported that these two PIK3CA hotspot mutations were overrepresented in APOBEC-signature tumors (42). Additionally, a supervised random forest clustering (43-44) with mutational exposures as input ranked Signature 6 (APOBEC signature) as the most important predictive feature in distinguishing the regular-mutated from the hyper-
mutated gastric cancer with high prediction accuracy (Supplementary Figs. 1C and E). Taken together, these findings suggest that APOBEC-mediated mutagenic activity are operative to greater extent in regular-mutated gastric cancer and alternative underlying mutagenic factors, presumably defective DNA proof-reading and repair, have substantially greater impact on the hyper-mutated gastric cancer.

According to a previous study, nearly 600 samples per cancer type are required to achieve a complete catalogue of cancer genes (18). Our study encompassed the largest number of gastric cancer samples available (n = 544) by combining data from previous whole-genome and whole-exome sequencing studies on gastric cancer (14–17, 45). Thus, we are able to identify SMGs that are mutated less frequently with a higher statistical power<sup>18</sup>. For instance, we found that FBXW7, widely reported as cancer driver in multiple human malignancies but not gastric cancer (23), was significantly mutated at 2.4% in our gastric cancer study; besides, there are other previously unreported SMGs (e.g. XIRP2, NBEA, COL14A1, CNBD1, AKAP6 and ITGAV) that are reported for the first time in gastric cancer, underscoring the importance of increasing sample size to identify potential cancer genes. However, the statistic power to identify cancer genes mutated at 2% of samples is merely 12% in regular-mutated gastric cancer (18), thus more rare SMGs are awaited to be discovered.

Batch effect is a common phenomenon in meta-analysis. To exclude artifacts in our SMG list, we employed stringent criteria to filter out possible false positives, including genes showing bias towards specific cohorts. However, cancer development is closely related to environmental exposures, which can provide unique selective

20

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.
pressure over specific cancer genes. Due to different cohorts may have their unique exposures, thus SMGs displaying cohort-specific features could be discarded in this study, leading to underestimation of the number of SMG. Another limitation of our study is the lack of functional validation of the identified SMGs. Until such data is available, whether these SMGs are indeed drivers in gastric tumorigenesis remains uncertain.

A major clinically relevant finding of this study is the use of 8 SMGs to classify regular-mutated gastric cancer into two prognostically distinct subgroups, namely C1 and C2 (Fig. 2 and Supplementary Fig. 1C). The prognostic significance of C1/2 was validated in an additional cohort and independent of age, sex, Lauren classification, TNM staging and studied cohorts. Together with other genome-based molecular subtyping studies (46-48), this finding highlighted the importance of molecular subtyping based on genome data in clinical utility. However, the lack of clinical and surgical data (residual tumor, number of removed lymph nodes, more refined clinical tumor staging) may be a potential limitation of the present study.

Another key finding emerged from our study is the identification of CDH1 mutation(s) as an independent prognostic marker for poor prognosis in diffuse-type gastric cancer. Somatic mutations in CDH1 are frequently reported for sporadic gastric cancer with predilection towards the diffuse type(15-16), which is associated with dismal survival (3). Interestingly, we revealed that CDH1 mutation(s) allow further stratification of diffuse-type gastric cancer into distinct subgroups with significantly different survival outcomes. In summary, our findings may be leveraged

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

to speed up interpretation of cancer genome data that may foreshadow clinical
outcomes and thus potentially help to guide gastric cancer intervention.

22

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

References

1. Ferlay J, Soerjomataram I, Ervik M, Dikshit R, Eser S, Mathers C, et al. GLOBOCAN 2012: estimated cancer incidence, mortality and prevalence worldwide in 2012, Section of cancer information. IARC, Lyon. v1.0
2. Lauren P. The two histological main types of gastric carcinoma: diffuse and so-called intestinal-type carcinoma. Acta Pathol Microbiol Scand. 1965;64:31–49.
3. Yamashita K, Sakuramoto S, Katada N, Futawatari N, Moriya H, Hirai K, et al. Diffuse type advanced gastric cancer showing dismal prognosis is characterized by deeper invasion and emerging peritoneal cancer cell: The latest comparative study to intestinal advanced gastric cancer. Hepatogastroenterology. 2009;56:276–81.
4. Marrelli D, Roviello F, De Manzoni G, Morgagni P, Di Leo A, Saragoni L, et al. Different patterns of recurrence in gastric cancer depending on Lauren’s histological type: longitudinal study. World J Surg. 2002;26:1160–5.
5. Stratton MR, Campbell PJ, Futreal PA. The cancer genome. Nature. 2009;458:719–24.
6. Vogelstein B, Kinzler KW. Cancer genes and the pathways they control. Nat Med. 2004;10:789–99.
7. Helleday T, Eshtad S, Nik-Zainal S. Mechanisms underlying mutational signatures in human cancers. Nat Rev Genet. 2014;15:585–98.
8. Alexandrov LB, Nik-Zainal S, Wedge DC, Aparicio SA, Behjati S, Biankin AV, et al. Signatures of mutational processes in human cancer. Nature. 2013;500:415–21.
9. Burns MB, Temiz NA, Harris RS. Evidence for APOBEC3B mutagenesis in multiple human cancers. Nat Genet. 2013;45:977–83.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

10. Roberts SA, Lawrence MS, Klimczak LJ, Grimm SA, Fargo D, Stojanov P, et al. An APOBEC cytidine deaminase mutagenesis pattern is widespread in human cancers. Nat Genet. 2013;45:970–6.
11. Henderson S, Chakravarthy A, Su X, Boshoff C, Fenton TR. APOBEC-mediated cytosine deamination links PIK3CA helical domain mutations to human papillomavirus-driven tumor development. Cell Rep. 2014;7:1833–41.
12. Wang K, Kan J, Yuen ST, Shi ST, Chu KM, Law S, et al. Exome sequencing identifies frequent mutation of ARID1A in molecular subtypes of gastric cancer. Nat Genet. 2011;43:1219–23.
13. Zang ZJ, Cutcutache I, Poon SL, Zhang SL, McPherson JR, Tao J, et al. Exome sequencing of gastric adenocarcinoma identifies recurrent somatic mutations in cell adhesion and chromatin remodeling genes. Nat Genet. 2012;44:570–4.
14. Bass AJ, Thorsson V, Shmulevich I, Reynolds SM, Miller M, Bernard B, et al. Comprehensive molecular characterization of gastric adenocarcinoma. Nature. 2014;513:202–9.
15. Wang K, Yuen ST, Xu J, Lee SP, Yan HH, Shi ST, et al. Whole-genome sequencing and comprehensive molecular profiling identify new driver mutations in gastric cancer. Nat Genet. 2014;46:573–82.
16. Kakiuchi M, Nishizawa T, Ueda H, Gotoh K, Tanaka A, Hayashi A, et al. Recurrent gain-of-function mutations of RHOA in diffuse-type gastric carcinoma. Nat Genet. 2014;46:583–7.
17. Chen K, Yang D, Li X, Sun B, Song F, Cao W, et al. Mutational landscape of gastric adenocarcinoma in Chinese: implications for prognosis and therapy. Proc Natl Acad Sci. 2015;6:1–6.

Downloaded from [cancerres.aacrjournals.org](http://cancerres.aacrjournals.org) on February 17, 2016. © 2016 American Association for Cancer Research.

18. Lawrence MS, Stojanov P, Mermel CH, Robinson JT, Garraway LA, Golub TR, et al. Discovery and saturation analysis of cancer genes across 21 tumour types. Nature. 2014;505:495–501.
19. Wang K, Li M, Hakonarson H. ANNOVAR: Functional annotation of genetic variants from high-throughput sequencing data. Nucleic Acids Res. 2010;38:1–7.
20. Wang H, Song M. Ckmeans.1d.dp: Optimal k-means clustering in one dimension by dynamic programming. R J. 2011;3:29–33.
21. Lawrence MS, Stojanov P, Polak P, Kryukov GV, Cibulskis K, Sivachenko A, et al. Mutational heterogeneity in cancer and the search for new cancer-associated genes. Nature. 2013;499:214–8.
22. Liu X, Jian X, Boerwinkle E. dbNSFP v2.0: A database of human non-synonymous SNVs and their functional predictions and annotations. Hum Mutat 2013;34:2393–402.
23. Kandoth C, McLellan MD, Vandin F, Ye K, Niu B, Lu C, et al. Mutational landscape and significance across 12 major cancer types. Nature. 2013;502:333–9.
24. Klijn C, Durinck S, Stawiski EW, Havery PM, Jiang Z, Liu H, et al. A comprehensive transcriptional portrait of human cancer cell lines. Nat Biotechnol. 2015;33:306-12.
25. Tamborero D, Gonzalez-Perez A, Perez-Llamas C, Deu-Pons J, Kandoth C, Reimand J, et al. Comprehensive identification of mutational cancer driver genes across 12 tumor types. Sci Rep. 2013;3:2650.
26. Futreal PA, Coin L, Marshall M, Down T, Hubbard T, Wooster R, et al. A census of human cancer genes. Nat Rev Cancer. 2004;4:177–83.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

27. Brunet J, Tamayo P, Golub TR, Mesirov JP. Metagenes and molecular pattern discovery using matrix factorization. Proc Natl Acad Sci USA. 2004;101:4164–9.
28. Gao Y, Church G. Improving molecular cancer class discovery through sparse non-negative matrix factorization. Bioinformatics. 2005;21:3970–5.
29. Kim H, Park H. Sparse non-negative matrix factorizations via alternating non-negativity-constrained least squares for microarray data analysis. Bioinformatics. 2007;23:1495–502.
30. Gaujoux R, Seoighe C. A flexible R package for nonnegative matrix factorization. BMC Bioinformatics. 2010;11:367.
31. Yu J, Wu WKK, Li X, He J, Li XX, Ng SS, et al. Novel recurrently mutated genes and a prognostic mutation signature in colorectal cancer. Gut. 2014;1–10.
32. Cancer T, Atlas G. Comprehensive molecular characterization of human colon and rectal cancer. Nature. 2012;487:330–7.
33. Bass AJ, Laird PW, Shmulevich I, Thorsson V, Schultz N, Sheth M, et al. Comprehensive Molecular Characterization of Gastric Adenocarcinoma. Nature. 2014;513:202–9.
34. Alexandrov LB, Nik-Zainal S, Wedge DC, Campbell PJ, Stratton MR. Deciphering signatures of mutational processes operative in human cancer. Cell Rep. 2013;3:246–59.
35. Cerami E, Gao J, Dogrusoz U, Gross BE, Sumer SO, Aksoy BA, et al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov. 2012;2:401–4.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

36. Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, et al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Sci Signal. 2013;6:pl1.
37. Dulak AM, Stojanov P, Peng S, Lawrence MS, Fox C, Stewart C, et al. Exome and whole-genome sequencing of esophageal adenocarcinoma identifies recurrent driver events and mutational complexity. Nat Genet. 2013;45:478–86.
38. Stirzaker C, Zotenko E, Song JZ, Qu W, Nair SS, Locke WJ, et al. Methylome sequencing in triple-negative breast cancer reveals distinct methylation clusters with prognostic value. Nat Commun. 2015;6:1–11.
39. Ni X, Zhuo M, Su Z, Duan J, Gao Y, Wang Z, et al. Reproducible copy number variation patterns among single circulating tumor cells of lung cancer patients. Proc Natl Acad Sci USA. 2013;110:21083–8.
40. Vogelstein B, Papadopoulos N, Velculescu VE, Zhou S, Diaz LA, Kinzler KW. Cancer genome landscapes. Science. 2013;339:1546–58.
41. Smith HC, Bennett RP, Kizilyer A, McDougall WM, Prohaska KM. Functions and regulation of the APOBEC family of proteins. Semin Cell Dev Biol. 2012;23:258–68.
42. Zhang L, Zhou Y, Cheng C, Cui H, Cheng L, Kong P, et al. Genomic analyses reveal mutational signatures and frequently altered genes in esophageal squamous cell carcinoma. Am J Hum Genet. 2015;96:597–611.
43. Breiman L. Random Forests. Mach Learn. 2001;45:5–32.
44. Liaw A, Wiener M. Classification and Regression by randomForest. R News 2002;2:18–22.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

45. Wong SS, Kim KM, Ting JC, Yu K, Fu J, Liu S, et al. Genomic landscape and genetic heterogeneity in gastric adenocarcinoma revealed by whole-genome sequencing. Nat Commun. 2014;5:5477.
46. Hoadley KA, Yau C, Wolf DM, Cherniack AD, Tamborero D, Ng S, et al. Multiplatform analysis of 12 cancer types reveals molecular classification within and across tissues of origin. Cell. 2014;158:1–16.
47. Gross AM, Orosco RK, Shen JP, Egloff AM, Carter H, Choueiri M, et al. Multi-tiered genomic analysis of head and neck cancer ties TP53 mutation to 3p loss. Nat Genet. 2014;46:939–43.
48. Akbani R, Akdemir KC, Aksoy BA, Albert M, Ally A, Amin SB, et al. Genomic Classification of Cutaneous Melanoma. Cell. 2015;161:1681–96.

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

# Figure Legends

Fig. 1 Mutational signatures of human gastric cancer. (A) Mutation burden stratified gastric cancers into the regular- (RGC) and hyper-mutated (HGC) types. RGC could be further classified into two subgroups (i.e. C1 and C2) based on mutation patterns of 40 significantly mutated genes (SMGs). (B) Seven mutational signatures (i.e. Signatures 1-7) indicative of distinct underlying mutational processes were derived from 544 gastric cancer genomes. (C) Mutational exposures (number of mutations) were attributed to each mutation signature. There were significantly more mutations attributable to Signature 6 (i.e. APOBEC signature) in RGC as compared to HGC. Prevalence and proportion of mutations associated with each mutational signature were also shown. A signature is considered to be operative in a tumor if it contributed to more than 100 single nucleotide variants (SNVs) or more than 25% of all SNVs in that sample; (D) Proportion of APOBEC signature mutations and (E) mutational patterns of PIK3CA in RGC and HGC; (F) Relationships between APOBEC3B mRNA levels and number of APOBEC signature mutations in RGC and HGC.

Fig. 2 Mutational landscape and prognostic significance of molecular subtyping in regular-mutated gastric cancer. (A) Mutational landscape of SMGs ordered by overall mutation frequencies in regular-mutated gastric cancer samples. Molecular classification into C1 and C2 based on mutation status of 40 SMGs was performed using nonnegative matrix factorization. Asterisks indicate SMGs with preferential mutations in either molecular subtype. (B) Kaplan-Meier survival curves displaying survival outcomes of C1- and C2-type regular-mutated gastric cancer. (C) Univariate and (D) multivariate Cox regression analyses for age, sex, TNM staging,

Lauren classification and molecular subtypes. Hazards ratio (HR), 95% confidence interval (CI) and p-values are displayed.

Fig. 3 Mutation plots of PIK3CA in the regular- and hyper-mutated gastric cancers. The distribution of different classes of mutations (different shapes) and functional domains of PIK3CA are shown.

Fig. 4 Altered signaling pathways in regular-mutated gastric cancer. Key pathways and inferred functions are summarized. Red and blue colors denote SMGs with activating and inactivating mutations, respectively, whereas genes in white are not identified as SMGs but known to have tumorigenic roles. Potential druggable targets are marked with asterisks.

Fig. 5 Prognostic significance of CDH1 mutation(s) in a combined cohort of diffuse-type gastric cancer. (A) Kaplan-Meier survival curves showing survival outcomes for two subgroups stratified by CDH1 mutation status; (B) Univariate and (C) multivariate Cox regression analyses for CDH1 mutation in relation to age, sex and TNM staging.

A Overal Gastric Cancer

544

Regular-mutated
2.4 (0~8.3)
mutations/Mb

Hyper-mutated
30.5 (9.6~200.2)
mutations/Mb

Molecular
subtyping

C1 C2

Deciphering
of mutational
processes

Identification
of SMGs

Altered signaling
pathways

Identification of
biomarkers

B

C>A C>G C>T T>A T>C T>G C>A C>G C>T T>A T>C T>G C>A C>G C>T T>A T>C T>G

Signature 1

CTT (11.3%)
GTT (6.5%)

ACG (26.7%)
CCG (26.4%)
GCG (16.1%)
TCG (13.4%)

Signature 2

Signature 3

TCT (20.1%)
TTT (8%)

Signature 4

Signature 5

TCT (4.9%)
TCA (6.1%)
TCC (4.6%)
TCA (3%)

Signature 6

C>A C>G C>T T>A T>C T>G C>A C>G C>T T>A T>C T>G

Signature 7

C Regular-mutated Hyper-mutated

D

Mutational
signature

Operativ in
RGC

Operativ in
HGC

Mutation in
RGC

Mutation in
HGC

Association
in RGC

Association
in HGC

Signature 1
11.9%
10.1%
15.5%
5.7%

Signature 2
59.3%
82.0%
26.6%
26.3%
Age

Signature 3
0.7%
3.4%
5.9%
6.1%

Signature 4
2.0%
80.9%
8.6%
22.2%

Signature 5
1.1%
51.7%
4.8%
20.2%

Signature 6
70.8%
10.1%
32.5%
3.2%
APOBEC
APOBEC

Signature 7
0.4%
80.9%
6.2%
16.2%

E

Median:
0.06 vs. 0.01
p<0.001

F

Regular-mutated
Hyper-mutated
q=0.01

G

Regular-mutated Gastric Cancer
Spearman rho=0.42; q<0.001

H

Hyper-mutated Gastric Cancer
Spearman rho=0.21; q=0.67

I

Median:
3.79 vs. 3.43
q=0.32

Fraction of APOBEC signature mutations

Number of PIK3CA mutations

Number of APOBEC signature mutations

Number of APOBEC signature mutations

Regular-
mutated
Hyper-
mutated

Regular-
mutated
Hyper-
mutated

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

A

| Lauren's subtype | Mol. subtype |
|------------------|--------------|
| C1              | C2           |
| NOS             | Diffuse      |
| Intestinal      | Mixed        |

*TP53 48.4%
*ARID1A 13.8%
*CDH1 11.6%
*PIK3CA 8.4%
*XIRP2 7.3%
NBEA 7.0%
*APC 6.8%
SMAD4 6.2%
*ERBB4 5.7%
*RHOA 5.7%
KRAS 4.4%
COL14A1 4.4%
*AKAP6 3.7%
CDH11 3.7%
CTNNB1 3.3%
BNC2 3.3%
ERBB2 3.1%
ITGAV 3.1%
TGFBR2 3.1%
CNBD1 3.1%
RNF43 2.4%
FBXW7 2.4%
*MAP2K7 2.2%
CDKN2A 2.0%
SAMSNI 2.0%
PIK3R1 1.8%
ELF3 1.8%
PIGR 1.8%
THEMIS 1.3%
AKAP2 1.1%
ZHX3 1.1%

Nonsense
Splice site
Frame shift
Inframe indel
Missense
Syn.

B

| C1 (n=98) | C2 (n=88) |
|-----------|-----------|

Log-rank test, p-value = 0.02

Overall survival (%)

Time since operation (months)
Mutation frequency

C

| Variable               | HR (95% CI)         | P-value |
|------------------------|---------------------|---------|
| Age                    | 0.99 (0.96 ~ 1.02)  | 0.45    |
| Sex (Male vs. Female)  | 1.32 (0.65 ~ 2.67)  | 0.45    |
| Stage (III, IV vs. I, II) | 3.53 (1.47 ~ 8.5) | 0.005   |
| Subtype (Diffuse vs. Intestinal) | 1.85 (0.97 ~ 3.51) | 0.06    |
| Cohort (TCGA vs. Asian) | 1.02 (0.48 ~ 2.16) | 0.96    |
| Mol. Subtype (C2 vs. C1) | 2.19 (1.12 ~ 4.3) | 0.02    |

Hazard Ratio (95% CI)

D

| Variable               | HR (95% CI)         | P-value |
|------------------------|---------------------|---------|
| Age                    | 1.0 (0.97 ~ 1.03)   | 0.94    |
| Sex (Male vs. Female)  | 1.61 (0.77 ~ 3.39)  | 0.21    |
| Stage (III, IV vs. I, II) | 3.9 (1.59 ~ 9.56) | 0.003   |
| Subtype (Diffuse vs. Intestinal) | 1.27 (0.63 ~ 2.56) | 0.51    |
| Cohort (TCGA vs. Asian) | 1.21 (0.53 ~ 2.76) | 0.66    |
| Mol. Subtype (C2 vs. C1) | 2.28 (1.1 ~ 4.73) | 0.026   |

Hazard Ratio (95% CI)

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

A. PIK3CA mutations in regular-mutated gastric cancer

PIK3CA
1                                                                                             1068
p.E9G     p.R38H    p.R88Q    p.R93W    p.103-105del   p.K111E    p.G118D    p.Y182H    p.N345K    p.E365K    p.N380S    p.A400V    p.C420R    p.E453K    p.E542K    p.E545K    p.Q546K    p.Y1021C   p.M1043I   p.H1047L/p.H1047R

PI3K_p85B          PI3K_rbd           PI3K_C2            PI3Ka              PI3_PI4_kinase


B. PIK3CA mutations in hyper-mutated gastric cancer

PIK3CA
1                                                                                             1068
p.I13S    p.R38H    p.R88Q    p.R93W    p.R93Q    p.R108H    p.K111E    p.K179E    p.C215G    p.V243A    p.N345K    p.C378R    p.R412Q    p.C420R    p.G512X    p.E542K    p.E545K    p.E545G    p.E546H    p.F744I    p.L766I    p.Y879R    p.K948E    p.Y1021H   p.H1047Y   p.H1047R   p.H1047Q

PI3K_p85B          PI3K_rbd           PI3K_C2            PI3Ka              PI3_PI4_kinase


Nonsense      Missense       Splice site      Insertion       Deletion


This markdown representation includes the text and structure of the image, preserving all labels and annotations as they appear. The figure legend is also included at the bottom.
Genetic Alterations of Signaling Pathways in Regular-mutated Gastric Cancer

### Genotoxic/oncogenic stress (57.6%)
- **DNA damage**
  - ATM: 4.0%
  - PARP1: 1.1%
  - TP53BP1: 2.2%
- **Oncogenic stress**
  - CDKN2A: 2.0%
  - MAP2K7: 2.2%
  - JNK1/2: 1.1%
  - CGREF1: 1.1%
  - TP53: 48.4%
    - CDKs → Cell cycle
    - CDKN1A
    - FBXW7: 2.4%
    - SYNCRIP: 1.1%
    - MYC: 1.1%
  - BRCA1: 0.7%
  - BAP1: 1.1%
  - Apoptosis
    - FANCD2: 0.4%

### Histone modification/chromatin remodeling (26.6%)
- ARID1A: 13.8%
- MLL: 1.8%
- MLL2: 3.7%
- MLL3: 3.5%
- MLL4: 3.1%
- SMARCA4: 1.5%
- MECP2: 0.7%
- HDAC1: 2.0%
- SAMSNI

### Growth Factor Receptor Signaling (22.4%)
- ERBB2: 3.1%
- ERBB4: 5.7%
- S1PR4: 1.3%
- EGFR: 1.8%
- KRAS: 4.4%
- PIK3CA: 8.4%
- PIK3R1: 1.8%
- MTOR: 1.1%
- Cell survival

### Zinc Finger TFs (4.4%)
- BNC2: 3.3%
- ZHX3: 1.1%

### Stress Granule Formation (1.8%)
- G3BP1: 0.9%
- CASC3: 0.9%

### Wnt Signaling (24.4%)
- APC: 6.8%
- CTNNB1: 3.3%
- RNF43: 2.4%
- HNF1A: 2.0%
- CK2A: 0.4%
- CDH1: 11.6%
- FZD: 3.3%
- Cell Motility / Invasiveness

### TGF-β Signaling (18.5%)
- TGFB1: 3.1%
- ITGAV: 3.1%
- TGFBR2: 1.8%
- ELF3: 0.7%
- RHOA: 5.7%
- SMAD4: 6.2%

### ECM (14.3%)
- FLG: 10.5%
- COL14A1: 4.4%

### Predicted activating
- 0.1–0.9%
- 1–2.9%
- 3–5.9%
- 6–8.9%
- > 9%

### Predicted inactivating
- 0.1–0.9%
- 1–2.9%
- 3–5.9%
- 6–8.9%
- > 9%

### cAMP Signaling (4.2%)
- CNBD1: 3.1%
- AKAP2: 1.1%

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

A

| Wild-type (n=156) |
| Mutant (n=32)     |
| Log-rank test, p-value=0.01 |

B

| Variable                     | HR (95% CI)               | P-value |
|------------------------------|---------------------------|---------|
| Age                          | 1.0 (0.98 ~ 1.02)        | 0.68    |
| Sex (Male vs. Female)        | 0.98 (0.57 ~ 1.7)        | 0.95    |
| Stage (III, IV vs. I, II)    | 2.48 (1.24 ~ 4.97)       | 0.01    |
| Cohort (TCGA vs. Asian)      | 1.09 (0.43 ~ 2.76)       | 0.85    |
| CDH1 (mutant vs. wildtype)   | 2.27 (1.19 ~ 4.32)       | 0.01    |

C

| Variable                     | HR (95% CI)               | P-value |
|------------------------------|---------------------------|---------|
| Age                          | 0.99 (0.97 ~ 1.01)        | 0.43    |
| Sex (Male vs. Female)        | 0.96 (0.55 ~ 1.68)        | 0.89    |
| Stage (III, IV vs. I, II)    | 2.61 (1.3 ~ 5.26)         | 0.007   |
| Cohort (TCGA vs. Asian)      | 1.23 (0.47 ~ 3.19)        | 0.67    |
| CDH1 (mutant vs. wildtype)   | 2.46 (1.28 ~ 4.71)        | 0.007   |

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.

Author Manuscript Published OnlineFirst on February 8, 2016; DOI: 10.1158/0008-5472.CAN-15-2443  
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

Cancer Research  
The Journal of Cancer Research (1916–1930) | The American Journal of Cancer (1931–1940)

Distinct subtypes of gastric cancer defined by molecular characterization include novel mutational signatures with prognostic capability

Xiangchun Li, William KK Wu, Rui Xing, et al.  

Cancer Res Published OnlineFirst February 8, 2016.

---

**Updated version**  
Access the most recent version of this article at:  
doi:10.1158/0008-5472.CAN-15-2443  

**Supplementary Material**  
Access the most recent supplemental material at:  
http://cancerres.aacrjournals.org/content/suppl/2016/02/06/0008-5472.CAN-15-2443.DC1.html  

**Author Manuscript**  
Author manuscripts have been peer reviewed and accepted for publication but have not yet been edited.

---

**E-mail alerts**  
Sign up to receive free email-alerts related to this article or journal.

**Reprints and Subscriptions**  
To order reprints of this article or to subscribe to the journal, contact the AACR Publications Department at pubs@aacr.org.

**Permissions**  
To request permission to re-use all or part of this article, contact the AACR Publications Department at permissions@aacr.org.

---

Downloaded from cancerres.aacrjournals.org on February 17, 2016. © 2016 American Association for Cancer Research.
