
Article
HTLV-1 Rex Tunes the Cellular Environment Favorable for Viral Replication

Kazumi Nakano * and Toshiki Watanabe

Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, 4-6-1, Shirokanedai, Minatoku, Tokyo 108-8639, Japan; tnabe@ims.u-tokyo.ac.jp
* Correspondence: nakanokz@ims.u-tokyo.ac.jp; Tel.: +81-3-5449-5295

Academic Editor: Louis M. Mansky

Received: 9 November 2015; Accepted: 9 February 2016; Published: 24 February 2016

Abstract: Human T-cell leukemia virus type-1 (HTLV-1) Rex is a viral RNA binding protein. The most important and well-known function of Rex is stabilizing and exporting viral mRNAs from the nucleus, particularly for unspliced/partially-spliced mRNAs encoding the structural proteins essential for viral replication. Without Rex, these unspliced viral mRNAs would otherwise be completely spliced. Therefore, Rex is vital for the translation of structural proteins and the stabilization of viral genomic RNA and, thus, for viral replication. Rex schedules the period of extensive viral replication and suppression to enter latency. Although the importance of Rex in the viral life-cycle is well understood, the underlying molecular mechanism of how Rex achieves its function has not been clarified. For example, how does Rex protect unspliced/partially-spliced viral mRNAs from the host cellular splicing machinery? How does Rex protect viral mRNAs, antigenic to eukaryotic cells, from cellular mRNA surveillance mechanisms? Here we will discuss these mechanisms, which explain the function of Rex as an organizer of HTLV-1 expression based on previously and recently discovered aspects of Rex. We also focus on the potential influence of Rex on the homeostasis of the infected cell and how it can exert its function.

Keywords: HTLV-1 Rex; pro-viral expression; unspliced RNA; NMD; alternative splicing; cell cycle regulation

---

1. Molecular Events in the Host Cell Caused by HTLV-1 Infection

Infection of T-cells with human T-cell leukemia virus type 1 (HTLV-1) causes adult T-cell leukemia (ATL), HTLV-I associated myelopathy/tropical spastic paraparesis (HAM/TSP) and HTLV-1 uveitis (HU) [1,2], although the molecular basis of such variations in the pathogenesis of HTLV-1 has not been fully elucidated. The structure of the genomic HTLV-1 RNA and the molecular events triggered by HTLV-1 infection have been thoroughly investigated [3–7]. Briefly, the genomic RNA of HTLV-1 is composed of 8685 nucleotides with two long terminal repeats (LTRs), which function as the viral promoter, at the both 5′ and 3′ ends. Although the genomic RNA is compact, HTLV-1 has various RNA signals to obtain the most out of its coding potential. By utilizing (1) three overlapped reading frames with two -1 programmed ribosomal frameshift signal (-1PRF), (2) two alternative splicing sites, (3) and multiple start and stop codons, HTLV-1 genomic RNA encodes more than 10 viral proteins [8]. HTLV-1 has three alternatively-spliced forms of viral mRNAs, which are unspliced, singly (partially)-spliced and doubly (fully)-spliced. The unspliced HTLV-1 mRNA encodes Gag, Pro, and Pol proteins, while singly-spliced RNA encodes Env. The doubly-spliced HTLV-1 mRNA encodes functional accessory proteins, such as Tax, Rex, P30II, p12, p13 in sense open reading frames (ORFs) and HBZ (HTLV-1 basic leucine zipper factor protein) in an anti-sense ORF.

After integration to the human genome, transcription and translation from the HTLV-1 provirus rely entirely on the host cell machinery. The translated viral accessory proteins then function in a precise schedule for effective viral replication [9,10] (Figure 1). The viral mRNA from the provirus for the first round of transcription is completely spliced to tax/rex mRNA by the cellular splicing machinery. Tax is more effectively translated from tax/rex mRNA because of its stronger Kozak sequence compared with that of Rex [11]. Then, Tax stimulates the transactivation of LTRs for enhanced tax/rex mRNA transcription. Such feed-forward activation of the HTLV-1 provirus results in the gradual accumulation of Rex in the infected cell. Subsequently, accumulation of sufficient Rex permits Rex-mediated nuclear export of unspliced and partially spliced viral RNA. The active export of these viral mRNAs to the cytoplasm by Rex results in enhanced translation of the viral structural proteins, Gag, Pro, Pol, and Env and, thereby, enhances viral replication. Inversely with the active nuclear-export of unspliced and partially-spliced viral mRNA by Rex, that of tax/rex mRNA is reduced; thus, cellular concentrations of Tax and Rex proteins are also decreased. Moreover, p30II from the minor doubly-spliced viral mRNA binds and retains tax/rex mRNA in the nucleoli by its strong nucleolar localization signal (NoLS). In combination, the cellular levels and activities of Tax and Rex proteins are gradually reduced, and both viral expression and replication are diminished to enter the latency. Rende et al. [12] mathematically analyzed the molecular events in early-phase HTLV-1 infection and confirmed that viral expression was indeed divided into two phases. The first phase was Tax/Rex expression, and the second phase was structural protein expression, which were both controlled by the functions of Tax and Rex. Furthermore, they concluded that the two-phase kinetics of HTLV-1 expression was strictly regulated by Rex, indicating that Rex is the major conductor of HTLV-1 expression.

![Diagram](attachment:diagram.png)

Figure 1. After HTLV-1 entry, the viral genomic RNA is reverse-transcribed and integrated into the host human genome (1). The viral mRNA from the provirus for the first-round of transcription is completely spliced to tax/rex mRNA by the cellular splicing machinery (2). Tax stimulates the transactivation of LTRs for further viral transcription, resulting in the gradual accumulation of Rex in the infected cell (3). Rex then starts exporting the unspliced and partially spliced viral mRNAs, encoding Gag, Pro, Pol, and Env, to the cytoplasm by binding to RxRE of viral mRNA (4), resulting in active viral replication (5). Due to active nuclear export of unspliced and partially spliced viral mRNA by Rex, that of tax/rex mRNA is eventually reduced to enter the latency.

2. Canonical Rex Function as a Post-Transcriptional Regulator of Viral Expression

2.1. Rex-Dependent Nuclear Export of Viral mRNAs

Rex binds to the Rex Responsive Element (RxRE) of the HTLV-1 mRNAs to form Rex-viral mRNA complex for selective nuclear-export. Unlike the Rev Responsive Element (RRE) in human immunodeficiency virus type-1 (HIV-1) mRNAs, RxRE is in all HTLV-1 mRNAs [5,13]. The RxRE of HTLV-1 mRNA maps to the region of 255 nucleotides (nt) from the U3 to the R region of the 3′-LTR and forms a stable secondary structure with four stem loops. Such a unique structure of RxRE is considered to function as the landmark for Rex to selectively bind to the viral mRNAs [14,15]. Although, all HTLV-1 derived mRNAs have RxRE, the nuclear export efficiency by Rex is different among HTLV-1 mRNAs. It has been widely accepted that cytoplasmic accumulations of unspliced and partially-spliced HTLV-1 mRNAs are Rex dependent, while that of fully spliced tax/rex mRNA is suppressed by Rex [12,16]. Subsequently, Bai et al. demonstrated that Rex also stimulated the nuclear-export of tax/rex mRNA and Tax expression, at least partially [17]. Most recently, Cavallari et al. elegantly demonstrated that not only unspliced gag/pol mRNA and singly spliced env mRNA, but also some of singly- and fully-spliced mRNAs encoding viral accessory proteins were also nuclear-exported in Rex-dependent manner. They showed that p30II, p12/p8, and p13 mRNAs were Rex-dependent, while tax/rex and p21rex mRNAs were Rex-independent. Interestingly, all Rex-dependent viral mRNAs contain 75 nt intronic regions, which control Rex-dependency as a cis-acting sequence [18]. Another study group demonstrated that HBZ, the antisense protein of HTLV-1, inhibited the nuclear-export of intron-containing mRNA by Rex, thus inhibited active viral replication and induced latency [19]. These reports suggest that Rex-mediated nuclear exports of HTLV-1 mRNAs are finely-tuned by RxRE, inherent cis-acting viral sequence, and viral proteins.

Rex binds to Chromosomal Maintenance 1 (CRM1), also known as Exportin 1 (XPO1), via its nuclear export signal (NES) for nuclear export. CRM1 is a cellular nuclear export protein which is responsible for the translocation of various cellular proteins with NES. Thus, the export of HTLV-1 mRNAs to the cytoplasm is dependent on CRM1, which is separated from the bulk cellular mRNAs exported in an Aly/Ref export-factor-dependent manner. The molecular mechanism of the RxRE-Rex-CRM1 complex formation has been extensively studied by Hakata et al. [20,21]. The authors revealed that the Rex has to be multimerized to bind to RxRE. They also propose a possibility that CRM1 is involved not only in the translocation of Rex but also in its multimerization. Therefore, one may speculate that Rex initially forms a complex with CRM1, which assists oligomerization of Rex on CRM1 before binding to RxRE. Nevertheless, the detailed order of the complex formation has never been investigated.

2.2. Primary Structure of Rex and Its Function

HTLV-1 Rex protein consists of 189 amino acids with its molecular weight of approximately 27 kDa. The HTLV-1 Rex protein contains several functional domains essential for its function. The primary structure of Rex has been well-described [22,23]. The N-terminal arginine-rich RNA-binding domain (aa 1–19) is required for binding to RxRE. Additionally, this domain overlaps with the nuclear localization signal (NLS), which is essential for Rex to shuttle-back to nucleus with importin-β, and with the p30II-binding domain. Rex interacts with CRM1 through the NES (aa 66–118) for nuclear export. Rex has two multimerization domains (aa 57–66 and 106–124) with NES in between and both of them are considered to be necessary for stable oligomerization of Rex. Most recently, a stability domain was identified at the C-terminal region of Rex (aa 170–189) [24–26]. The authors demonstrated that deletion of the stability domain destabilized Rex significantly but did not influence the function of Rex.

Viruses 2016, 8, 58

---

### 2.3. Rex Activity and Phosphorylation

It has been described that the activity Rex is finely regulated through its phosphorylation [27] at several serine(Ser)/threonine(Thr) residues [24]. The treatment of HUT102 (an HTLV-1-infected cell line) with a protein kinase C inhibitor, H-7 [1-(5-isoquinolinyl-sulfonyl)-2-methylpiperazine] destabilized unspliced viral mRNA and reduced the expression level of Gag-p19 protein [27]. To date, seven phosphorylation sites of Rex have been identified at Thr-22, Ser-36, Thr-37, Ser-70, Ser-97, Ser-106, and Thr-174 [24,28]. Kesic *et al.* [24] evaluated the importance of phosphorylation sites and demonstrated that Rex phosphorylation at Ser-97 and Thr-174 was the most critical for the efficiency of RxRE-dependent nuclear export by Rex.

---

### 2.4. Regulation of Rex by Other HTLV-1 Viral Proteins

As described above, Rex plays a central role in selective expression of HTLV-1 viral structural proteins and is, thus, in active viral reproduction. The Rex activity is critical to switch from the early productive period to late latent period. Therefore Rex activity has to be finely tuned during HTLV-1 infection. HTLV-1 has an elegant auto-regulatory mechanism to regulate the activity of Rex, i.e., by two-phased HTLV-1 expression kinetics, which is described above (see Section 1), and by the function of other viral proteins.

The suppressive function of p30II for the Rex has been well investigated, following extensive reproduction of HTLV-1 virus by Tax and Rex, p30II is expressed from the minor doubly-spliced HTLV-1 mRNA. P30II selectively binds to tax/rex mRNA. Then, p30II, with a strong nucleolar localization signal (NoLS), localizes and retained tax/rex mRNA in nucleoli, thus preventing their expression and functions. Such time-lagged operations of the positive (Tax and Rex) and negative (p30II) regulators of HTLV-1 promotes the early infectious phase followed by the late infectious phase with a rapid shutdown to escape from the host immune surveillance against pathogens [29–32]. On the other hand, Rex binds to p30II and rescues tax/rex mRNA [32]. The timing of Rex-p30II interaction is considered to regulate switching from the early active-viral-reproduction phase to the late rapid-shutdown phase to escape from the host immune system. Most recently, HBZ, the antisense protein of HTLV-1, was demonstrated to inhibit the nuclear-export of intron-containing mRNA by Rex, thus inhibiting active viral replication and induced latency [19].

Since p21Rex is constitutively expressed in primary peripheral blood mononuclear cells from HTLV-1 carriers and ATL patients [33–35], it has been expected that p21Rex plays a role in the HTLV-1 life cycle, such as p27Rex suppressor as a dominant-negative isoform, although, a clear biological function of p21Rex has not been elucidated, yet. P21Rex is expressed from a defective HTLV-1 mRNA without the exon 2, and lacks the N-terminus 78 amino acids of p27Rex, ranging from NLS to the N′-multimerization domain [36,37]. Without NLS, p21Rex localizes to the cytoplasm; thus, the functional importance of this isoform has not yet been elucidated. More recently, Bai *et al.* demonstrated that p21Rex neither nuclear-exported the viral mRNA, nor influenced the p27Rex function [17]. Therefore, p21Rex seems not to be involved in HTLV-1 lifecycle as an isoform of Rex, although possible roles of this short Rex isoform in the cellular biological pathways are required to be elucidated in the future.

---

### 3. Non-Canonical Functions of Rex: Exploring New Aspects of Rex

#### 3.1. NMD Inhibition by Rex

##### 3.1.1. Rex Stabilizes HTLV-1 Genomic RNA by Inhibition of NMD

For viruses, stabilization of viral mRNAs in the host cells is a major issue to overcome for self-replication [38]. Recently, a lot of attention has been directed towards one of the host mRNA decay mechanisms in the host-pathogen interaction, Nonsense-mediated mRNA decay (NMD), and has revealed how viruses evade NMD and protect viral mRNAs [39–42]. These reports have shown
that each virus has its own strategy to stabilize viral mRNAs; for example, by an inherent viral RNA stabilization mechanism, utilizing host RNA stability factors, inhibition of the host mRNA decay machinery, or hijacking the host cell RNA metabolism with viral nucleases \[38,43\].

NMD is an essential and evolutionarily-conserved cellular mRNA quality control mechanism. The principal function of NMD is to prevent the expression of harmful truncated proteins by selective elimination of aberrant mRNAs containing premature termination codons (PTCs) (see reviews \[43,44\]) (Figure 2A). As indicated above, the major Rex function is the stabilization and export of the viral unspliced and partially-spliced mRNAs to cytoplasm. However, unspliced HTLV-1 mRNA (*i.e.*, viral genomic RNA) contains various RNA signals, such as multiple start and stop codons, overlapping ORFs, programmed ribosomal frameshift signals, and a long 3′-untranslated region (>1000 nt). These RNA signals are unusual for eukaryotic cells and have the potential to initiate NMD (Figure 2B). However, it is not clear how HTLV-1 evades NMD to protect its genomic RNA. Our laboratory has demonstrated that full-length HTLV-1 transcripts exhibit enhanced turnover in NMD-activated cells that overexpress UPF1, while knockdown of UPF1 by small interfering (si) RNA promotes enhanced stability of HTLV-1 genomic mRNA \[45\]. By confirming that the genomic and full-length mRNAs of HTLV-1 are sensitive to NMD, we further demonstrated that Rex inhibited NMD. We suggest that through the inhibition of NMD, Rex stabilizes viral transcripts in the cytoplasm to secure translation of viral structural proteins. In contrast, it is highly probable that Rex also perturbs cellular mRNA metabolism and host cell homeostasis by inhibition of the global NMD activity. It is noteworthy that Rex-mediated inhibition of NMD is not RNA- or sequence-specific, but Rex establishes a general blockage of NMD. Thus, not only the viral transcripts, but also natural host-encoded NMD substrates are stabilized in the presence of Rex. We demonstrated that Rex stabilized well-known NMD target mRNAs, such as *IL-6*, *MAP3K14*, and *FYN* mRNAs.

It has been reported that *IL-2Ra* mRNA was stabilized up to a five-fold level in Rex-overexpressing cells compared with the control cells without Rex \[46,47\], although the underlying mechanism has not been clarified. Since *IL-2Ra* mRNA can be a NMD target because of its upstream (u)-ORF structure, we speculate that this mRNA is stabilized by Rex through NMD inhibition. Indeed, UPF1 knockdown by siRNA in HeLa cells resulted in a significant increase in the *IL-2Ra* mRNA expression (Nakano unpublished data).

### 3.1.2. How Does Rex Protect Viral mRNAs from NMD in the Cytoplasm?

NMD is a complex mechanism coupled with splicing and translation. Briefly, the core components of NMD are UPF1, UPF2, and UPF3, which detect the PTC-containing mRNA to be degraded via NMD, and SMG1, SMG5, SMG6, and SMG7, which phosphorylate/dephosphorylate UPF1 (Figure 2B). This results in the regulation of the activity of UPF1, the key molecule of NMD. UPF1 is a component of the termination complex, assembled when the ribosome reaches the termination codon of mRNA, while UPF2 and UPF3 are components of the exon junction complex (EJC) formed at the exon-exon boundary and are removed by the ribosome while it moves through. In normal mRNA, the termination codon is in the last exon, thus EJC does not remain at the end of translation. In contrast, PTC is located upstream of EJC, thus UPF1 on PTC comes into contact with UPF2 and UPF3, which triggers the phosphorylation of UPF1 by SMG1, and the onset of NMD. Phosphorylated UPF1 is dephosphorylated by the SMG5/SMG7 complex and recycled, while SMG7 completes the NMD process in mRNA processing bodies (p-bodies) of the cytoplasm. Recently, it was reported that SMG6 functioned as an endonuclease in the degradation of PTC-containing mRNA (see review \[43\]).

Since NMD machinery is coupled with splicing and translation apparatus, we speculate that Rex may influence the overall NMD activity directly via interaction with NMD core-components, and indirectly via interaction with splicing and translational machinery. Comprehensive protein-protein interactome analysis between Rex and the host-cellular proteins by a high-resolution mass spectrometry (MS) will be fruitful to understand the overall molecular landscape of NMD inhibition by Rex.
Viruses 2016, 8, 58

A  
NMD-inducing RNA signals  

(1) Upstream Open Reading Frame (uORF)  
(4) -1 Ribosomal Frameshift Signal (-1RFS)  

(2) PTC in Alternatively spliced transcripts  
(5) Long 3'-UTR  

(3) Introns in 3'-UTR  

B  

HTLV-1 genomic RNA  

NMD  

Figure 2. (A) NMD is an essential and evolutionarily-conserved cellular mRNA quality control mechanism. The principal function of NMD is to prevent the expression of harmful truncated proteins by selective elimination of aberrant mRNAs containing premature termination codons (PTCs). It has been reported that NMD also regulates the expression levels of normal mRNAs, which inherently contain RNA signals to generate PTC: 1. Upstream (U)-ORF, 2. alternative splicing producing PTC, 3. intron in 3'-UTR, 4. -1 programmed ribosomal frameshift signal (-1PRF) and 5. long 3'-UTR more than 1000 nt; (B) The genomic RNA of HTLV-1 contains various RNA signals potentially initiate NMD, i.e., 1. multiple start and stop codons, 2. two alternative splicing sites, 3. intron in the 3'-UTR region of gag/pro/pol mRNA, 4. overlapping ORFs, and -1 PRF and 5. a long 3'-UTR (>1000 nt). We reported that HTLV-1 genomic RNA was indeed destabilized by NMD, and HTLV-1 Rex had a new function to inhibit NMD [32].

3.2. Regulation of mRNA Splicing Machinery by Rex  

It is well known that mRNA splicing is coupled with transcription, and virtually all primary (unspliced) mRNAs are spliced at the site of transcription (see review [48]). Thus, HTLV-1 unspliced and partially-spliced mRNAs cannot evade splicing only through selective nuclear export by Rex. Consequently, we may speculate that Rex has a function to inhibit cellular splicing activity against the viral mRNAs, which is independent of the well-known CRM1-dependent nuclear export mechanism of Rex. Gröne et al. [49] demonstrated that Rex increased the nuclear quantity of unspliced viral mRNAs
and reduced the number of spliced viral mRNAs. Since Rex did not influence the total quantity of viral transcripts, the authors conclude that Rex has a function by which it reduces splicing activity.

SF2/ASF regulates splicing activity and plays an important role in the splice-site selection \[48,50\]. Splicing patterns of HTLV-1 mRNA are governed by SF2/ASF *i.e.*, the differential pX splice site utilization of HTLV-1 mRNA is dependent on the expression level of SF2/ASF \[51\], although the viral mechanism to regulate the splicing activity through SF2/ASF has not been fully investigated. Interestingly, Powell *et al.* \[52\] demonstrated that HIV-1 Rev suppressed cellular splicing activity by recruiting SF2/ASF to the Rev-RRE RNP complex. Tange *et al.* \[53\] also showed the interaction between Rev and p32, the ASF/SF2-associated protein. The authors speculated that the interaction might function as a bridge between Rev and the host cellular splicing machinery. Considering the homologous function and molecular mechanism of HTLV-1 Rex and HIV-Rev, it is possible that Rex has a similar mechanism to suppress splicing machinery by binding and inhibiting the function of SF2/ASF.

Heterogeneous Nuclear Ribonucleoprotein A1 (hnRNPA1) is another cellular protein which is known to interact with Rex. hnRNPA1 associates with mRNA as a component of the RNP complex in the nucleus and influences the transcription, maturation and transport of mRNA \[54\]. HnRNPA1 also plays a crucial role in the regulation of alternative splicing, mainly as a splicing suppressor \[48\]. A recent study clearly showed that the expression level of hnRNPA1 had strong implications for the determination of exon-inclusion/skipping \[55\]. Hamaia *et al.* \[56\] first demonstrated that Rex function was impaired in a T cell line not infected by HTLV-1, Jurkat, and speculated that Rex was unable to bind to RxRE in the cell line. Later, the same study group found that hnRNPA1 bound to RxRE in competition with Rex, thus influencing the function of Rex \[57\]. Subsequently, Kress *et al.* \[58\] demonstrated that hnRNPA1 suppressed the Rex activity in a dose-dependent manner, while the suppression of hnRNPA1 in C91/PL, a HTLV-1-infected cell line, increased the Rex-dependent nuclear export of unspliced and partially-spliced mRNA. The authors proposed the possibility that hnRNPA1 enhances the splicing processes of viral mRNA. Indeed, hnRNPA1 caused enhanced exon 2 skipping in HTLV-1 mRNA \[51\]. On the other hand, the basal hnRNPA1 level was lower in HTLV-1-infected T cell lines (C91/PL, MT2, and HUT102) compared with other T cell lines without HTLV-1 infection (CBL and Jurkat) \[57\]. The authors concluded that HTLV-1 may have a mechanism to downregulate hnRNPA1, which is not advantageous for viral replication. Glutathione S-transferase (GST)-Rex pulldown assays conducted in our laboratory showed that Rex physically interacted with hnRNPA1 (Nakano, unpublished data). The underlying mechanism of how Rex is involved in the downregulation/inhibition of hnRNPA1 requires further investigation.

It has been shown that Rex changes the preference for exon usage during FYN mRNA splicing/maturation from exon7B to exon7A, resulting in enhanced production of the brain-type Fyn-B instead of the T cell-type Fyn-T \[59\]. Fyn is a proto-oncogene, belonging to the membrane-associated tyrosine kinase family. Its overexpression/disorder has been implicated to the tumorigenesis of several malignancies. Fyn has two major isoforms of distinct functions, Fyn-B expressed in the brain and Fyn-T expressed exclusively in hematopoietic cells, which are derived from exon7A and exon7B, respectively. Picard *et al.* \[60\] reported that the expression level of FYN-B mRNA was significantly increased in acute lymphoblastic leukemia or chronic lymphocytic leukemia. As indicated above, hnRNPA1, the regulator of exon usage, is downregulated in HTLV-1-infected cells. Moreover, we found that Rex interacts with hnRNPA1. If Rex itself is involved in the downregulation and/or suppression of hnRNPA1, such deregulation of hnRNPA1 function by Rex may have implications to alterations in the exon usage during mRNA maturation, such as observed in FYN mRNA. Aberrant overexpression of various splicing variants caused by genetic lesions in the splicing machinery may have implication to the HTLV-1 pathogenesis.

Taken together, the biological significance of the molecular interactions between Rex and the splicing-regulatory proteins in the regulation of splicing activity and splicing patterns requires elucidation in the future.
Viruses 2016, 8, 58

3.3. Cell-Cycle Regulation: Does Rex Interfere the Host Cell-Cycle Regulation?

A wealth of evidence has indicated that a number of viruses have mechanisms to modify cellular cell-cycle regulation for the promotion of viral replication. It has been well documented and reviewed that HIV-1 Vpr induces G2 arrest of the host cell-cycle [61–63]. The G2/M check point or DNA damage checkpoint is regulated by the activity of the Cdc2 (Cdk1) and CyclinB complexes, which are finely tuned by various kinases and phosphatases. Cdc2 undergoes inhibitory phosphorylation by Wee1 and Myt1, or Chk1/2, which are activated by the ATM/ATR DNA damage response pathway. Cdc25s are phosphatases and activate Cdc2 by dephosphorylation. When the cell senses DNA damage, Cdc2 is inhibited by the ATM/ATR pathway, and the cell cycle is arrested at G2. At the G2/M transition, PLK1 phosphorylates Cdc25s and Wee1 for activation and inhibition, respectively. Thus, Cdc2 is activated to enter the M phase (see review [64]). Furthermore, PLK1 is phosphorylated and activated by Aurora kinase A (AURKA) and its co-factor, Bora [65]. For the molecular mechanism of G2 arrest by Vpr, Zhao and Elder [61] indicated the importance of the interaction between Vpr and IkB kinase-associated serine/threonine protein phosphatase 2A (PP2A) in the induction of G2 arrest, although the detailed mechanism has yet to be clarified. Goh et al. [66] demonstrated that Vpr interacts with and inhibits Cdc25C. However, because Vpr is also known to activate ATR and Chk1 [67], it has not been fully elucidated whether Vpr directly inhibits Cdc25C or does so through the ATR pathway. Noronha et al. [68] investigated the influence of Vpr using a different methodology and showed that Vpr altered the subcellular localization of CyclinB1, Wee1, and Cdc25C. These authors also found that Vpr-induced herniations of the nuclear envelope and speculated that such disrupted nuclear architecture might interrupt normal cell-cycle progression.

Why does HIV-1 Vpr induce G2 arrest? For viral replication, transcription from the provirus and translation of viral proteins are dependent on the host machinery. It is thought that G2 arrest by Vpr is beneficial for the selective translation of viral proteins (Figure 3). The m⁷G-Cap structure of transcribed mRNA is first recognized by the Cap binding complex (CBC) and subjected to the pioneer round of translation for the quality check of mRNA. Then, CBC is replaced by the eIF4F complex for the steady-state translational procedure, which is regulated by eIF4E within the eIF4F complex. It is thought that the translation of a viral protein from HIV-1 mRNA relies on CBC-dependent pioneer-round translation, which is cell-cycle independent. In contrast, the major eIF4E-dependent translation is inhibited during G2 phase. Therefore, G2-arrest by Vpr can enhance the translation of viral proteins (see review [69]). Furthermore, Sharma et al. [70] elegantly demonstrated that Vpr abrogates activated (phosphorylated) eIF4E levels. They also showed that CBC was retained at the Cap structure of unspliced and partially-spliced HIV-1 mRNAs in the cytoplasm. Taken together, these findings indicate that Vpr suppresses eIF4E activity by the reduction of its active form, as well as by the induction of G2-arrest. Thus, only CBC-bound HIV-1 mRNAs can be effectively subjected to cellular translational machinery (Figure 3). Most recently, it has been demonstrated that Vpr interacts with and activates the SLX4 endonuclease complex, which activates the DNA damage/repair response through the ATR/Chk1 pathway, resulting in G2 arrest [71].

HTLV-1, with a similar life-cycle to HIV-1, may have a similar strategy to enhance self-reproduction. However, the HIV-1 Vpr homologue has not been identified among the HTLV-1-encoded proteins. In the review by Zhao and Elder [61], they mention that HTLV-1 Tax showed similar characteristics to HIV-1 Vpr, such as binding to PP2A and the induction G2 arrest. Haoudi et al. [72] first indicated that Tax bound to and activated Chk2 in the DNA damage response, resulting in G2 arrest. Moreover, the interaction between Tax and Chk2 was further investigated by the same group, and they subsequently concluded that Tax inhibits the Chk2-induced DNA damage response through its retention in chromatin in order to evade the cellular DNA damage response to Tax-induced DNA instability [73]. Another study group also demonstrated that Tax bound to and inhibited the activity of Chk1, which is also involved in the ATM/ATR-mediated DNA damage response [74]. Fu et al. [75] showed that Tax interacted with PP2A to activate I kappa B kinase (IKK), thus influencing the nuclear factor (NF)-κB pathway. Together, these previous reports indicate that Tax

inhibits the ATM/ATR-dependent DNA damage response and, thus, is not likely to induce G2 arrest. Anupam et al. [76] conducted a protein-interactome analysis for p30II and demonstrated that p30II interacted with ATM and modulated the activity of the G2/M checkpoint. There have been no reports implicating Rex to the cell-cycle regulation. Yet, we observed G2 arrest in CEM (ALL patient-derived human T cell line) overexpressing Rex (Nakano, unpublished data). Rex has arginine-rich NLS at the N-terminus similar to Vpr-NLS2, which is essential for G2 arrest. It has been demonstrated that eIF4E specifically binds to the mRNAs of cell-cycle promoting proteins in nucleus and is exported by CRM1. This mechanism is separated from TAP/NXF1 and REF/Aly-dependent export of bulk mRNAs [77–80]. Since Rex is also nuclear-exported by CRM1, we speculate that Rex may compete for CRM1 with eIF4E. Consequently, Rex may suppress the eIF4E-CRM1-dependent nuclear export of mRNAs encoding cell-cycle promoting proteins and, therefore, may induce cell-cycle arrest. The interaction between Rex and eIF4E and other cell-cycle regulating proteins should be investigated in the future.

![Diagram](attachment:diagram.png)

Figure 3. The G2-arrest by Vpr is beneficial for the selective translation of viral proteins. In the regular mRNA translation, the m⁷G-Cap structure of transcribed mRNA is first recognized by the Cap binding complex (CBC) and subjected to the pioneer round of translation for the quality check of mRNA. Then, CBC is replaced by the eIF4F complex for the steady-state translational procedure, which is regulated by eIF4E within the eIF4F complex. The majority of eIF4E-dependent translation is inhibited during G2 phase. On the other hand, CBC-dependent pioneer-round translation is cell-cycle independent. Since the translation of a viral protein from HIV-1 mRNA relies on CBC-dependent pioneer-round translation, G2-arrest by Vpr can enhance the translation of viral proteins by suppressing the eIF4E-dependent translation. Additionally, Vpr reduces the activated (phosphorylated) eIF4E level. Taken together, Vpr assists selective translation of HIV-1 mRNAs by induction of G2-arrest, as well as by suppression of eIF4E activity.

Viruses 2016, 8, 58

4. Function of Rex and the Viral Pathogenesis

4.1. Do Rex-1/Rex-2 Functions Relate to the Pathogenesities of HTLV-1/HTLV-2?

Comparative analysis between Rex-1 (Rex) from HTLV-1 and Rex-2 from HTLV-2 can be helpful to understand the relationship between the function of Rex and the viral pathogenicity [13]. Both HTLV-1 and HTLV-2 belong to the same genus [81] and infect human T cells. Both viruses encode a similar set of viral proteins, including Tax and Rex and, thus, reproduce through a similar pathway. Yet, only HTLV-1 causes ATL and HAM/TSP in infected T cells, but not HTLV-2. The primary structures of Rex-1 and Rex-2 show 60% homology with common functional domains, such as RNA binding domain (RBD)/NLS, two multimerization domains, nuclear export signal (NES), and stability domain (SD) [24–26] and, thus, function as the viral RNA binding/transporting proteins through the common cellular pathways. On the other hand, the position of RxRE in the viral mRNA is different between HTLV-1 and HTLV-2, which may modulate impacts of Rex-1 and Rex-2 functions in their respective viral life cycles [13]. It has been clarified that all HTLV-1 mRNAs have RxRE, which is located in the U3/R region, while only unspliced HTLV-2 mRNA has RxRE, which is located in the R/U5 region [82]. Thus, it can be speculated that Rex-1 nuclear-exports all HTLV-1 mRNAs, including tax/rex mRNA, which enhances Tax/Rex expression and, thus, viral reproduction, whereas Rex-2 does not, resulting in a low viral production. Indeed, Bai et al. demonstrated that the nuclear export of the doubly spliced tax/rex mRNA of HTLV-1 was also enhanced by Rex-1 in a RxRE-1/CRM1-dependent manner [17]. Differences in nuclear export efficiencies of viral mRNA/Rex/CRM1 complex between HTLV-1 and HTLV-2 may influence viral replications and activities and, thus on pathogenesities of these viruses in infected T cells.

4.2. HTLV-1 Rex and HIV-1 Rev: Are They Similar or Different?

4.2.1. HIV-1 Rev, the Molecular Counterpart of HTLV-1 Rex

Rev protein of HIV-1 (Human Immunodeficiency Virus type-I) is the molecular counterpart of HTLV-1 Rex. HTLV-1 and HIV-1 both belong to the family of Retroviridae, and are further specified to the genuses of Deltaretrovirus and Lentivirus, respectively. In addition, the major tropism of both viruses is human CD4⁺ T cells. HTLV-1 and HIV-1 have genomic RNA of a similar size, i.e., about 8.5 knt and 9.75 knt, respectively, which encodes viral proteins with considerably homologous functions. HTLV-1 Rex and HIV-1 Rev bind viral mRNAs and shuttle between the nucleus and cytoplasm for nuclear export of viral transcripts, through quite similar mechanisms yet, interestingly, the homology between their amino acid sequences is very low [22,83]. Messenger-RNAs of HTLV-1 and HIV-1 have regions to form complex secondary structures called RxRE and RRE. Rex and Rev bind to viral mRNAs thorough RxRE and RRE as highly-specific landmarks, respectively. In terms of primary functional domains, both Rex and Rev have arginine-rich RNA binding domains for selective binding to their respective responsive elements (Figure 4A). It has been well documented that Rex and Rev stabilize unspliced and partially-spliced viral mRNAs, encoding viral structural proteins, and actively transport them to the cytoplasm for selective translation (Figure 4B). For shuttling between nucleus and cytoplasm, both proteins have NLSs for binding to importin-β and NESs for binding to CRM1 [23,83]. Furthermore, they bind to B-23 via NLSs to be translocated to the nucleolus [84–87] (Figure 4B). While the uniform role and mechanism of Rex and Rev are extensively discussed, some differences in the detailed molecular mechanism between Rex and Rev have been also described. It is now accepted that multimerization is essential for Rex to interact with RxRE, yet the monomer Rex is still able to bind to CRM1 for translocation to the cytoplasm [20–22]. Quite the opposite, the monomer Rev is known to bind to RRE, however multimerization of Rev up to 12 molecules is necessary for stable binding to CRM1 and for effective cytoplasmic-translocation [88,89]. These differences between these two viral RNA binding proteins may be closely related to the nuclear export efficiency of viral mRNAs,

thus, viral replication and, consequently, to different disease associations, *i.e.*, ATL and HAM/TSP with HTLV-1, and AIDS with HIV-1.

**A**

|  | Multimerlization |
| --- | --- |
| **HTLV-1 Rex** | ARM NES Stability |
| 1 | 189 |

|  | Multimerlization |
| --- | --- |
| **HIV-1 Rev** | ARM NES |
| 1 | 116 |

**B**

![Diagram](#)

Figure 4. (A) The primary structures of HTLV-1 Rex and HIV-1 Rev. The homology in the primary sequences of HTLV-1 Rex and HIV-1 Rev is low, but they share most of the functional domains critical for their function, *i.e.*, arginine-rich RNA binding domain, NLS, NES, and two multimerization domains; (B) HTLV-1 Rex and HIV-1 Rev play similar functions through similar mechanisms. In the nucleolus, Rex and Rev specifically bind to the respective viral mRNAs through Rex responsive element (RxRE) for Rex and the Rev responsive element (RRE) for Rev. They stabilize unspliced or partially-spliced viral mRNA and actively transport these to the cytoplasm for selective translation of viral structural proteins by CRM1 binding through their NES. Rex and Rev return to the nucleus by binding to Importin-β, and further translocated to the nucleolus (No) by binding to B-23 via NLS.

### 4.2.2. The Structural Biology of HTLV-1 Rex; Learning from that of HIV-1 Rev

Along with accumulation of knowledge in the molecular characteristics and functions, both Rex and Rev propose a common question; how these small viral proteins bind to RxRE/RRE of viral mRNAs and CRM1 simultaneously, and how they form a large, but stable, RNP complex for the nuclear export of viral mRNAs. To answer these questions, information from the structural biology

may contribute significantly. In terms of structural biology, investigations into Rev have progressed far more than those of Rex. In contrast, our knowledge of the structure of Rex has not been updated from that of the N-terminal arginine-rich domain (aa 1–16) of the Rex peptide solved by nuclear magnetic resonance [90]. On the contrary, great efforts have been made to obtain the structural information on the Rev–Rev dimer interface or the Rev–RRE interaction since the early 2000's. Daugherty *et al.* [91] clarified for the first time that Rev formed a homo-oligomer on the RRE via an oligomerization domain. In 2010, two different groups reported the partial structure of Rev in a dimer form [92,93]. Daugherty *et al.* [93] proposed the “jellyfish model” in which a rigid Rev dimer(s) forms an oligomeric structure, similar to the head of a jelly fish, while an unstructured NES region extended from each Rev molecule to form a tentacle-like alignment in binding to CRM1. They also mentioned that at the RNA-binding surfaces, “the two ARMs are arranged to reach out from the body of the Rev dimer to grasp RNA, much as two human arms are positioned to grip objects”.

Based on the Rev structure, the detailed molecular mechanism of HIV-1 viral mRNA nuclear export by Rev has since been clarified greatly. Fang *et al.* [94] investigated the structure of RRE thoroughly and demonstrated that RRE functioned as the topological landmark for Rev by forming an unusual structure, to which only Rev was able to specifically bind with high affinity. Finally, it was demonstrated that the RRE-Rev-CRM1 “HIV-1 export complex” was assembled co-transcriptionally at the transcription site, thus unspliced HIV-1 mRNAs were stably exported from the nucleus [95]. Moreover, it has been demonstrated that RRE-Rev assembly starts with changes in the RRE structure so that binding to the first two dimer complexes of Rev leads to further conformational changes of RRE, triggering oligomerization of Rev [96]. This enables stable binding to the CRM1 dimer [97]. Structural analysis of the Rev-dimer and RRE complex revealed that the Rev-dimer architecture can be flexibly altered depending on the structure of RRE [98]. The authors speculated that such changes in RRE structure and in the Rev-dimer architecture might alter the whole architecture of the “jellyfish complex” *i.e.*, the RRE-Rev-CRM1 complex. Overall, the structural biology of Rev indeed provided tremendous information to clarify the regulatory mechanism of the nuclear export efficiency of the HIV-1 export complex, thus resulting in pathogenesis of HIV-1. At present, the same questions still remain for HTLV-1 Rex. The structural biology of Rex is promising to provide significant information to answer these questions in the future.

## 5. Conclusions

Accumulating data on the analysis of the Rex interactome shows that Rex has a significantly high potential to interact with a wide variety of cellular proteins. These cellular proteins are crucial for the maintenance of the cellular homeostasis by playing essential roles in mRNA surveillance and metabolism, nucleo-cytoplasmic shuttling, tumor growth regulation and in post-translational modification of proteins, such as SUMOylation [13,99,100]. These data strongly suggest that Rex modifies a wide range of cellular pathways in order to organize the host cellular environment suitable for the stabilization and translocation of viral mRNAs, as well as for selective translation of viral proteins for effective self-replication (Figure 5A). Such Rex-oriented tuning of the host cell environment can alter cellular homeostasis, and thus may provide a basis for the pathogenesis of HTLV-1 (Figure 5B).

Uninfected Cell

Splicing  
Correct splicing  

NMD  
Degradation of abnormal mRNA  

Cell cycle  

eIF4E-dependent  
translation  

Maintenance of cellular homeostasis  

B  

HTLV-1 Infected Cell  

Splicing  
Abnormal splicing  

Rex  

NMD  
Stabilization of abnormal mRNA  

Cell cycle  

CBC-dependent  
translation  

Cellular environment favorable for viral replication  

Figure 5. In this review, we focused on three cellular pathways, NMD, splicing machinery, and cell-cycle regulation, since we may expect that Rex interacts with these pathways to adjust the cellular environment suitable for the viral replication. (A) In the normal cells, the activities of NMD, splicing, and cell-cycle regulation are optimized to maintain the cellular homeostasis by eliminating PTC-containing harmful mRNAs, production of correctly spliced mRNAs encoding functional proteins, and adjusting the cell-cycle for n optimal cell proliferation rate and for effective eIF4E-dependent RNA translations, respectively; (B) In HTLV-1 infected cells, Rex inhibits NMD for stabilization of the viral genomic mRNA [32]. Additionally, based on previous reports and newly discovered aspects of Rex in our laboratory, we assume that Rex may suppress the activity of splicing machinery and may induce G2 arrest. These adjustments of host-cell mechanisms are favorable for the stabilization and translocation of viral mRNAs, as well as for selective translation of viral proteins for effective self-replication. On the other hand, suppression/alteration in these pathways may cause accumulation of abnormal PTC-containing mRNAs, thus harmful proteins; abnormal splicing patterns; deregulated cell proliferation; and suppression of eIF4E-dependent translation. Therefore, Rex-oriented tuning of the host cell environment may alter cellular homeostasis, and provide a basis for the pathogenesis of HTLV-1.

Acknowledgments: This work was supported by Grants-in-Aid for Scientific Research from the Ministry of Education, Culture, Sports, Science, and Technology of Japan, to TW (No. 19659241) and to KN (No. 22700863, No. 24501304 and No. 15K06827).

Author Contributions: K.N. and T.W. conceived and designed the experiments; K.N. performed the experiments and analyzed the data; T.W. contributed reagents/materials/analysis tools; K.N. and T.W. wrote the paper.

Conflicts of Interest: The authors declare no conflicts of interest.

References

1. Gallo, R.C. The discovery of the first human retrovirus: HTLV-1 and HTLV-2. *Retrovirology* 2005, 7, 1–7.
2. Takatsuki, K. Discovery of adult T-cell leukemia. *Retrovirology* 2005, 3, 1–3.
3. Seiki, M.; Hattori, S.; Hirayama, Y.; Yoshida, M. Human adult T-cell leukemia virus: Complete nucleotide sequence of the provirus genome integrated in leukemia cell DNA. *Proc. Natl. Acad. Sci. USA* 1983, 80, 3618–3622. [CrossRef] [PubMed]
4. Johnson, J.M.; Harrod, R.; Franchini, G. Molecular biology and pathogenesis of the human T-cell leukaemia/lymphotropic virus Type-1 (HTLV-1). *Int. J. Exp. Pathol.* 2001, 82, 135–147. [CrossRef] [PubMed]
5. Franchini, G.; Fukumoto, R.; Fullen, J.R. T-cell control by human T-cell leukemia/lymphoma virus type 1. *Int. J. Hematol.* 2003, 78, 280–296. [CrossRef] [PubMed]
6. Kashanchi, F.; Brady, J.N. Transcriptional and post-transcriptional gene regulation of HTLV-1. *Oncogene* 2005, 24, 5938–5951. [CrossRef] [PubMed]
7. Kannian, P.; Green, P.L. Human T Lymphotropic Virus Type 1 (HTLV-1): Molecular Biology and Oncogenesis. *Viruses* 2010, 2, 2037–2077. [CrossRef] [PubMed]
8. Balvay, L.; Lastra, M.L.; Sargueil, B.; Darlix, J.-L.; Ohlmann, T. Translational control of retroviruses. *Nat. Rev. Microbiol.* 2007, 5, 128–140. [CrossRef] [PubMed]
9. Corradin, A.; DI Camillo, B.; Rende, F.; Ciminale, V.; Toffolo, G.M.; Cobelli, C. Retrovirus HTLV-1 gene circuit: A potential oscillator for eukaryotes. *Pac. Symp. Biocomput.* 2010, 432, 421–432.
10. Li, M.; Kesic, M.; Yin, H.; Yu, L.; Green, P.L. Kinetic analysis of human T-cell leukemia virus type 1 gene expression in cell culture and infected animals. *J. Virol.* 2009, 83, 3788–3797. [CrossRef] [PubMed]
11. Green, P.L.; Chen, I.S.Y. Regulation of human T cell leukemia virus expression. *FASEB J.* 1990, 4, 169–174. [PubMed]
12. Rende, F.; Cavallari, I.; Corradin, A.; Silic-Benussi, M.; Toulza, F.; Toffolo, G.M.; Tanaka, Y.; Jacobson, S.; Taylor, G.P.; D’Agostino, D.M.; *et al.* Kinetics and intracellular compartmentalization of HTLV-1 gene expression: Nuclear retention of HBZ mRNAs. *Blood* 2011, 117, 4855–4859. [CrossRef] [PubMed]
13. Nakano, K.; Watanabe, T. HTLV-1 Rex: The courier of viral messages making use of the host vehicle. *Front. Microbiol.* 2012, 3, 330. [CrossRef] [PubMed]
14. Ahmed, Y.F.; Gilmartin, G.M.; Hanly, S.M.; Nevins, J.R.; Greene, W.C. The HTLV-I Rex response element mediates a novel form of mRNA polyadenylation. *Cell* 1991, 64, 727–737. [CrossRef]
15. Ahmed, Y.F.; Hanly, S.M.; Malim, M.H.; Cullen, B.R.; Greene, W.C. Structure-function analyses of the HTLV-I Rex and HIV-1 Rev RNA response elements: Insights into the mechanism of Rex and Rev action. *Genes Dev.* 1990, 4, 1014–1022. [CrossRef] [PubMed]
16. Hidaka, M.; Inoue, J.; Yoshida, M.; Seiki, M. Post-transcriptional regulator (rex) of HTLV-1 initiates expression of viral structural proteins but suppresses expression of regulatory proteins. *EMBO J.* 1988, 7, 519–523. [PubMed]
17. Bai, X.T.; Sinha-Datta, U.; Ko, N.L.; Bellon, M.; Nicot, C. Nuclear export and expression of HTLV-I tax/rex mRNA is RxRE/Rex-dependent. *J. Virol.* 2012, 86, 4559–4565. [CrossRef] [PubMed]
18. Cavallari, I.; Rende, F.; Bona, M.K.; Sztuba-solinska, J.; Silic-benussi, M.; Tognon, M.; Legrice, S.F.J.; Franchini, G.; Ciminale, V. Expression of alternatively spliced human T-cell leukemia virus type 1 mRNAs is influenced by mitosis and by a novel *cis*-acting regulatory sequence. *J. Virol.* 2016, 90, 1486–1498.
19. Philip, S.; Zahoor, M.A.; Zhi, H.; Ho, Y.-K.; Giam, C.-Z. Regulation of human T-lymphotropic virus type I latency and reactivation by HBZ and Rex. *PLoS Pathog.* 2014, 10, e1004040. [CrossRef] [PubMed]
20. Hakata, Y.; Umemoto, T.; Matsushita, S.; Shida, H. Involvement of human CRM1 (exportin 1) in the export and multimerization of the Rex protein of human T-cell leukemia virus type 1. *J. Virol.* 1998, 72, 6602–6607. [PubMed]
21. Hakata, Y.; Yamada, M.; Shida, H. Rat CRM1 is responsible for the poor activity of human T-cell leukemia virus type 1 Rex protein in rat cells. *J. Virol.* 2001, 75, 11515–11525. [CrossRef] [PubMed]
22. Baydoun, H.H.; Bellon, M.; Nicot, C. HTLV-1 Yin and Yang: Rex and p30 master regulators of viral mRNA trafficking. *AIDS Rev.* 2008, 10, 195–204. [PubMed]
23. Younis, I.; Green, P.L. The human T-cell leukemia virus Rex protein. *Front. Biosci.* 2005, 1, 431–445. [CrossRef]
24. Kesic, M.; Doueiri, R.; Ward, M.; Semmes, O.J.; Green, P.L. Phosphorylation regulates human T-cell leukemia virus type 1 Rex function. *Retrovirology* 2009, 6, 105. [CrossRef] [PubMed]

25. Kesic, M.; Ward, M.; Semmes, O.J.; Green, P.L. Site-specific phosphorylation regulates human T-cell leukemia virus type 2 Rex function *in vivo*. *J. Virol.* **2009**, *83*, 8859–8868. [CrossRef](https://doi.org/10.1128/JVI.01745-08) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19608737)

26. Xie, L.; Kesic, M.; Yamamoto, B.; Li, M.; Younis, I.; Lairmore, M.D.; Green, P.L. Human T-cell leukemia virus type 2 Rex carboxy terminus is an inhibitory/stability domain that regulates Rex functional activity and viral replication. *J. Virol.* **2009**, *83*, 5232–5243. [CrossRef](https://doi.org/10.1128/JVI.02436-08) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19307200)

27. Adachi, Y.; Nosaki, T.; Hatanaka, M. Protein kinase inhibitor H-7 blocks accumulation of unspliced mRNA of human T-cell leukemia virus type I (HTLV-I). *Biochem. Biophys. Res. Commun.* **1990**, *169*, 469–475. [CrossRef](https://doi.org/10.1016/0006-291X(90)91582-8)

28. Adachi, Y.; Copeland, T.D.; Takahashi, C.; Nosaka, T.; Ahmed, A.; Oroszlan, S.; Hatanaka, M. Phosphorylation of the Rex protein of human T-cell leukemia virus type I. *J. Biol. Chem.* **1992**, *267*, 21977–21981. [PubMed](https://pubmed.ncbi.nlm.nih.gov/1383008)

29. Bai, X.T.; Baydoun, H.H.; Nicot, C. HTLV-I p30: A versatile protein modulating virus replication and pathogenesis. *Mol. Aspects Med.* **2010**, *31*, 344–349. [CrossRef](https://doi.org/10.1016/j.mam.2010.04.003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/20417680)

30. Ghorbel, S.; Sinha-Datta, U.; Dundr, M.; Brown, M.; Franchini, G.; Nicot, C. Human T-cell leukemia virus type I p30 nuclear/nucleolar retention is mediated through interactions with RNA and a constituent of the 60S ribosomal subunit. *J. Biol. Chem.* **2006**, *281*, 37150–37158. [CrossRef](https://doi.org/10.1074/jbc.M605883200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16920710)

31. Nicot, C.; Dundr, M.; Johnson, J.M.; Fullen, J.R.; Alonzo, N.; Fukumoto, R.; Princler, G.L.; Derse, D.; Misteli, T.; Franchini, G. HTLV-1-encoded p30II is a post-transcriptional negative regulator of viral replication. *Nat. Med.* **2004**, *10*, 197–201. [CrossRef](https://doi.org/10.1038/nm986) [PubMed](https://pubmed.ncbi.nlm.nih.gov/14702087)

32. Sinha-Datta, U.; Datta, A.; Ghorbel, S.; Dodon, M.D.; Nicot, C. Human T-cell lymphotropic virus type I rex and p30 interactions govern the switch between virus latency and replication. *J. Biol. Chem.* **2007**, *282*, 14608–14615. [CrossRef](https://doi.org/10.1074/jbc.M611558200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17322320)

33. Berneman, Z.N.; Gartenhaus, R.B.; Reitz, M.S.; Blattner, W.A.; Manns, A.; Hanchard, B.; Ikehara, O.; Gallo, R.C.; Klotman, M.E. Expression of alternatively spliced human T-lymphotropic virus type I pX mRNA in infected cell lines and in primary uncultured cells from patients with adult T-cell leukemia/lymphoma and healthy carriers. *Proc. Natl. Acad. Sci. USA* **1992**, *89*, 3005–3009. [CrossRef](https://doi.org/10.1073/pnas.89.7.3005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/1542685)

34. Orita, S.; Takagi, S.; Saiga, A.; Minoura, N.; Araki, K.; Kinoshita, K.; Kondo, T.; Hinuma, Y.; Igarashi, H. Human T cell leukaemia virus type 1 p21X mRNA: Constitutive expression in peripheral blood mononuclear cells of patients with adult T cell leukaemia. *J. Gen. Virol.* **1992**, *73 Pt 9*, 2283–2289. [CrossRef](https://doi.org/10.1099/0022-1317-73-9-2283) [PubMed](https://pubmed.ncbi.nlm.nih.gov/1383008)

35. Saiga, A.; Aono, Y.; Imai, J.; Kinoshita, K.; Orita, S.; Igarashi, H. Presence of antibodies to p21X and/or p27rex proteins in sera from human T-cell leukemia virus type I-infected individuals. *J. Virol. Methods* **1996**, *57*, 157–168. [CrossRef](https://doi.org/10.1016/0166-0934(96)02020-8)

36. Kiyokawa, T.; Seiki, M.; Iwashita, S.; Imagawa, K.; Shimizu, F.; Yoshida, M. T-cell leukemia virus type I p27X-III and p21x-III, proteins encoded by the pX sequence of human T-cell leukemia virus type I. *Proc. Natl. Acad. Sci. USA* **1985**, *82*, 8359–8363. [CrossRef](https://doi.org/10.1073/pnas.82.23.8359) [PubMed](https://pubmed.ncbi.nlm.nih.gov/3000736)

37. Orita, S.; Kobayashi, H.; Aono, Y.; Saiga, A.; Maeda, M.; Igarashi, H. p21X mRNA is expressed as a singly spliced pX transcript from defective provirus genomes having a partial deletion of the pol-env region in human T-cell leukemia virus type 1-infected cells. *Nucleic Acids Res.* **1993**, *21*, 3799–3807. [CrossRef](https://doi.org/10.1093/nar/21.15.3799) [PubMed](https://pubmed.ncbi.nlm.nih.gov/8395080)

38. Dickson, A.M.; Wilusz, J. Strategies for viral RNA stability: Live long and prosper. *Trends Genet.* **2011**, *27*, 286–293. [CrossRef](https://doi.org/10.1016/j.tig.2011.04.003) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21601986)

39. Garcia, D.; Garcia, S.; Voinnet, O. Nonsense-Mediated Decay Serves as a General Viral Restriction Mechanism in Plants. *Cell Host Microbe* **2014**, *16*, 391–402. [CrossRef](https://doi.org/10.1016/j.chom.2014.08.008) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25215259)

40. Balistreri, G.; Horvath, P.; Schweingruber, C.; Zünd, D.; McInerney, G.; Merits, A.; Mühlemann, O.; Azzalin, C.; Helenius, A. The host nonsense-mediated mRNA decay pathway restricts mammalian RNA virus replication. *Cell Host Microbe* **2014**, *16*, 403–411. [CrossRef](https://doi.org/10.1016/j.chom.2014.08.009) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25215260)

41. Quek, B.L.; Beemon, K. Retroviral strategy to stabilize viral RNA. *Curr. Opin. Microbiol.* **2014**, *18*, 78–82. [CrossRef](https://doi.org/10.1016/j.mib.2014.02.004) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24613350)

42. Ramage, H.R.; Kumar, G.R.; Verschueren, E.; Johnson, J.R.; Von Dollen, J.; Johnson, T.; Newton, B.; Shah, P.; Horner, J.; Krogan, N.J.; *et al.* A combined proteomics/genomics approach links hepatitis C virus infection with nonsense-mediated mRNA decay. *Mol. Cell* **2015**, *57*, 329–340. [CrossRef](https://doi.org/10.1016/j.molcel.2014.11.022) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25557588)

43. Popp, M.; Maquat, L. Organizing principles of mammalian nonsense-mediated mRNA decay. *Annu. Rev. Genet.* **2013**, *47*, 139–165. [CrossRef](https://doi.org/10.1146/annurev-genet-111212-133451) [PubMed](https://pubmed.ncbi.nlm.nih.gov/24279969)

44. Kervestin, S.; Jacobson, A. NMD: A multifaceted response to premature translational termination. *Nat. Rev. Mol. Cell Biol.* **2012**, *13*, 700–712. [CrossRef](https://doi.org/10.1038/nrm3427) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22863805)

45. Nakano, K.; Ando, T.; Yamagishi, M.; Yokoyama, K.; Ishida, T.; Ohsugi, T.; Tanaka, Y.; Brighty, D.W.; Watanabe, T. Viral interference with host mRNA surveillance, the nonsense-mediated mRNA decay (NMD) pathway, through a new function of HTLV-1 Rex: Implications for retroviral replication. *Microbes Infect.* 2013, **15**, 491–505. [CrossRef] [PubMed]

46. Kanamori, H.; Kodama, T.; Matsumoto, A.; Itakura, H.; Yazaki, Y. Stabilization od interleukin-2 receptor α chain mRNA by HTLV-1 Rex in mouse L cells: Lower amounts of Rex do not stabilize the mRNA. *Biochem. Biophys. Res. Commun.* 1994, **198**, 243–250.

47. White, K.N.; Nosaka, T.; Kanamori, H.; Hatanaka, M.; Honjo, T. The nucleolar localisation signal of the HTLV-1 protein p27Rex is important for stabilisation of IL-2 receptor α subunit mRNA by p27Rex. *Biochem. Biophys. Res. Commun.* 1991, **175**, 98–103. [CrossRef]

48. Kornblihtt, A.R.; Schor, I.E.; Alló, M.; Dujardin, G.; Petrillo, E.; Muñoz, M.J. Alternative splicing: A pivotal step between eukaryotic transcription and translation. *Nat. Rev. Mol. Cell Biol.* 2013, **14**, 153–165. [CrossRef] [PubMed]

49. Gröne, M.; Koch, C.; Grassmann, R. The HTLV-1 Rex protein induces nuclear accumulation of unspliced viral RNA by avoiding intron excision and degradation. *Virology* 1996, **218**, 316–325. [CrossRef] [PubMed]

50. Karni, R.; de Stanchina, E.; Lowe, S.W.; Sinha, R.; Mu, D.; Krainer, A.R. The gene encoding the splicing factor SF2/ASF is a proto-oncogene. *Nat. Struct. Mol. Biol.* 2007, **14**, 185–193. [CrossRef] [PubMed]

51. Princler, G.; Julias, J.G.; Hughes, S.H.; Derse, D. Roles of viral and cellular proteins in the expression of alternatively spliced HTLV-1 pX mRNAs. *Virology* 2003, **317**, 136–145. [CrossRef] [PubMed]

52. Powell, D.M.; Amaral, M.C.; Wu, J.Y.; Maniatis, T.; Greene, W.C. HIV Rev-dependent binding of SF2/ASF to the Rev response element: Possible role in Rev-mediated inhibition of HIV RNA splicing. *Proc. Natl. Acad. Sci. USA* 1997, **94**, 973–978. [CrossRef] [PubMed]

53. Tange, T.O.; Jensen, T.H.; Kjems, J. In vitro interaction between human immunodeficiency virus type 1 Rev protein and splicing factor ASF/SF2-associated protein, p32. *J. Biol. Chem.* 1996, **271**, 10066–10072. [PubMed]

54. Jean-Philippe, J.; Paz, S.; Caputi, M. hnRNP A1: The Swiss army knife of gene expression. *Int. J. Mol. Sci.* 2013, **14**, 18999–19024. [CrossRef] [PubMed]

55. Huelga, S.C.; Vu, A.Q.; Arnold, J.D.; Liang, T.Y.; Liu, P.P.; Yan, B.Y.; Donohue, J.P.; Shiue, L.; Hoon, S.; Brenner, S.; *et al.* Integrative genome-wide analysis reveals cooperative regulation of alternative splicing by hnRNP proteins. *Cell Rep.* 2012, **1**, 167–178. [CrossRef] [PubMed]

56. Hamaia, S.; Casse, H.; Gazzolo, L.; Duc Dodon, M. The human T-cell leukemia virus type 1 Rex regulatory protein exhibits an impaired functionality in human lymphoblastoid Jurkat T cells. *J. Virol.* 1997, **71**, 8514–8521. [PubMed]

57. Dodon, M.D.; Hamaia, S.; Martin, J.; Gazzolo, L. Heterogeneous nuclear ribonucleoprotein A1 interferes with the binding of the human T cell leukemia virus type 1 Rex regulatory protein to its response element. *J. Biol. Chem.* 2002, **277**, 18744–18752. [CrossRef] [PubMed]

58. Kress, E.; Baydoun, H.H.; Bex, F.; Gazzolo, L.; Dodon, M.D. Critical role of hnRNP A1 in HTLV-1 replication in human transformed T lymphocytes. *Retrovirology* 2005, **2**, 8. [CrossRef] [PubMed]

59. Weil, R.; Levraud, J.P.; Dodon, M.D.; Bessia, C.; Hazan, U.; Kourilsky, P.; Israël, A. Altered expression of tyrosine kinases of the Src and Syk families in human T-cell leukemia virus type 1-infected T-cell lines. *J. Virol.* 1999, **73**, 3709–3717. [PubMed]

60. Picard, C.; Gabert, J.; Olive, D.; Collette, Y. Altered splicing in hematological malignancies reveals a tissue-specific translational block of the Src-family tyrosine kinase fyn brain isoform expression. *Leukemia* 2004, **18**, 1737–1739. [CrossRef] [PubMed]

61. Zhao, R.Y.; Elder, R.T. Viral infections and cell cycle G2/M regulation. *Cell Res.* 2005, **15**, 143–149. [CrossRef] [PubMed]

62. Andersen, J.L.; Le Rouzic, E.; Planelles, V. HIV-1 Vpr: Mechanisms of G2 arrest and apoptosis. *Exp. Mol. Pathol.* 2008, **85**, 2–10. [CrossRef] [PubMed]

63. Guenzel, C.A.; Hérâte, C.; Benichou, S. HIV-1 Vpr-a still “enigmatic multitasker”. *Front. Microbiol.* 2014, **5**, 127. [CrossRef] [PubMed]

64. Boutros, R.; Lobjois, V.; Ducommun, B. CDC25 phosphatases in cancer cells: Key players? Good targets? *Nat. Rev. Cancer* 2007, **7**, 495–507. [CrossRef] [PubMed]

65. Bruinsma, W.; Macurek, L.; Freire, R.; Lindqvist, A.; Medema, R.H. Bora and Aurora-A continue to activate Plk1 in mitosis. *J. Cell Sci.* 2014, **127**, 801–811. [CrossRef] [PubMed]

66. Goh, W.C.; Manel, N.; Emerman, M. The human immunodeficiency virus Vpr protein binds Cdc25C: Implications for G2 arrest. *Virology* **2004**, *318*, 337–349. [CrossRef](https://doi.org/10.1016/j.virol.2003.11.021) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15033513)

67. Lai, M.; Zimmerman, E.S.; Planelles, V.; Chen, J.; Irol, J.V. Activation of the ATR pathway by human immunodeficiency virus type 1 Vpr involves its direct binding to chromatin *in vivo*. *J. Virol.* **2005**, *79*, 15443–15451. [CrossRef](https://doi.org/10.1128/JVI.79.24.15443-15451.2005) [PubMed](https://pubmed.ncbi.nlm.nih.gov/16286368)

68. De Noronha, C.M.C.; Sherman, M.P.; Lin, H.W.; Cavrois, M.V.; Moir, R.D.; Goldman, R.D.; Greene, W.C. Dynamic disruptions in nuclear envelope architecture and integrity induced by HIV-1 Vpr. *Science* **2001**, *294*, 1105–1109. [CrossRef](https://doi.org/10.1126/science.1064604) [PubMed](https://pubmed.ncbi.nlm.nih.gov/11679670)

69. Guerrero, S.; Batisse, J.; Libre, C.; Bernacchi, S.; Marquet, R.; Paillart, J.-C. HIV-1 replication and the cellular eukaryotic translation apparatus. *Viruses* **2015**, *7*, 199–218. [CrossRef](https://doi.org/10.3390/v7010199) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25569583)

70. Sharma, A.; Yilmaz, A.; Marsh, K.; Cochrane, A.; Boris-Lawrie, K. Thriving under stress: Selective translation of HIV-1 structural protein mRNA during Vpr-mediated impairment of eIF4E translation activity. *PLoS Pathog.* **2012**, *8*, e1002612. [CrossRef](https://doi.org/10.1371/journal.ppat.1002612) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22589710)

71. Brégnard, C.; Benkirane, M.; Laguette, N. DNA damage repair machinery and HIV escape from innate immune sensing. *Front. Microbiol.* **2014**, *5*, 176. [PubMed](https://pubmed.ncbi.nlm.nih.gov/24744733)

72. Haoudi, A.; Daniels, R.C.; Wong, E.; Kupfer, G.; Semmes, O.J. Human T-cell leukemia virus-I tax oncoprotein functionally targets a subnuclear complex involved in cellular DNA damage-response. *J. Biol. Chem.* **2003**, *278*, 37736–37744. [CrossRef](https://doi.org/10.1074/jbc.M304536200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12813072)

73. Gupta, S.K.; Guo, X.; Durkin, S.S.; Fryrear, K.F.; Ward, M.D.; Semmes, O.J. Human T-cell leukemia virus type 1 Tax oncoprotein prevents DNA damage-induced chromatin egress of hyperphosphorylated Chk2. *J. Biol. Chem.* **2007**, *282*, 29431–29440. [CrossRef](https://doi.org/10.1074/jbc.M703438200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17581775)

74. Park, H.U.; Jeong, J.-H.; Chung, J.H.; Brady, J.N. Human T-cell leukemia virus type 1 Tax interacts with Chk1 and attenuates DNA-damage induced G2 arrest mediated by Chk1. *Oncogene* **2004**, *23*, 4966–4974. [CrossRef](https://doi.org/10.1038/sj.onc.1207609) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15122335)

75. Fu, D.-X.; Kuo, Y.-L.; Liu, B.-Y.; Jeang, K.-T.; Giam, C.-Z. Human T-lymphotropic virus type I tax activates I-kappa B kinase by inhibiting I-kappa B kinase-associated serine/threonine protein phosphatase 2A. *J. Biol. Chem.* **2003**, *278*, 1487–1493. [CrossRef](https://doi.org/10.1074/jbc.M209297200) [PubMed](https://pubmed.ncbi.nlm.nih.gov/12435736)

76. Anupam, R.; Datta, A.; Kesic, M.; Green-Church, K.; Shkriabai, N.; Kvaratskhelia, M.; Lairmore, M.D. Human T-lymphotropic virus type 1 p30 interacts with REGgamma and modulates ATM (ataxia telangiectasia mutated) to promote cell survival. *J. Biol. Chem.* **2011**, *286*, 7661–7668. [CrossRef](https://doi.org/10.1074/jbc.M110.185397) [PubMed](https://pubmed.ncbi.nlm.nih.gov/21183698)

77. Wickramasinghe, V.O.; Laskey, R.a Control of mammalian gene expression by selective mRNA export. *Nat. Rev. Mol. Cell Biol.* **2015**, *16*, 431–442. [CrossRef](https://doi.org/10.1038/nrm3995) [PubMed](https://pubmed.ncbi.nlm.nih.gov/25915047)

78. Topisirovic, I.; Siddiqui, N.; Lapointe, V.L.; Trost, M.; Thibault, P.; Bangeranye, C.; Piñol-Roma, S.; Borden, K.L.B. Molecular dissection of the eukaryotic initiation factor 4E (eIF4E) export-competent RNP. *EMBO J.* **2009**, *28*, 1087–1098. [CrossRef](https://doi.org/10.1038/emboj.2009.43) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19240745)

79. Culjkovic, B.; Topisirovic, I.; Skrabanek, L.; Ruiz-Gutierrez, M.; Borden, K.L.B. eIF4E promotes nuclear export of cyclin D1 mRNAs via an element in the 3′UTR. *J. Cell Biol.* **2005**, *169*, 245–256. [CrossRef](https://doi.org/10.1083/jcb.200411082) [PubMed](https://pubmed.ncbi.nlm.nih.gov/15863613)

80. Culjkovic, B.; Topisirovic, I.; Skrabanek, L.; Ruiz-Gutierrez, M.; Borden, K.L.B. eIF4E is a central node of an RNA regulon that governs cellular proliferation. *J. Cell Biol.* **2006**, *175*, 415–426. [CrossRef](https://doi.org/10.1083/jcb.200606084) [PubMed](https://pubmed.ncbi.nlm.nih.gov/17060636)

81. Vandamme, A.-M.; Salemi, M.; Desmyter, J. The simian origins of the pathogenic human T-cell lymphotropic virus type I. *Trends Microbiol.* **1998**, *6*, 477–483. [CrossRef](https://doi.org/10.1016/S0966-842X(98)01359-9)

82. Rende, F.; Cavallari, I.; Romanelli, M.G.; Diani, E.; Bertazzoni, U.; Ciminale, V. Comparison of the genetic organization, expression strategies and oncogenic potential of HTLV-1 and HTLV-2. *Leuk. Res. Treat.* **2012**, *2012*, 1–14. [CrossRef](https://doi.org/10.1155/2012/875486) [PubMed](https://pubmed.ncbi.nlm.nih.gov/22649488)

83. Suhasini, M.; Reddy, T.R. Cellular proteins and HIV-1 Rev function. *Curr. HIV Res.* **2009**, *7*, 91–100. [CrossRef](https://doi.org/10.2174/157016209787581594) [PubMed](https://pubmed.ncbi.nlm.nih.gov/19275588)

84. Adachi, Y.; Copeland, T.D.; Hatanakall, M.; Oroszlansii, S. Nuclear targeting signal of Rex protein of human T-cell leukemia virus type I specifically binds to nucleolar shuttle protein B-23. *J. Biol. Chem.* **1993**, *268*, 13930–13934. [PubMed](https://pubmed.ncbi.nlm.nih.gov/8349617)

85. Palmeri, D.; Malim, M.H. Importin β can mediate the nuclear import of an arginine-rich nuclear localization signal in the absence of importin α. *Mol. Cell. Biol.* **1999**, *19*, 1218–1225. [CrossRef](https://doi.org/10.1128/MCB.19.3.1218) [PubMed](https://pubmed.ncbi.nlm.nih.gov/9916098)

86. Truant, R.; Cullen, B.R. The Arginine-rich domains present in human immunodeficiency virus type 1 Tat and Rev function as direct importin β-dependent nuclear localization signals. *Mol. Cell. Biol.* **1999**, *19*, 1210–1217. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

87. Yoneda, Y. Nucleocytoplasmic protein traffic and its significance to cell function. *Genes Cells* **2000**, *5*, 777–787. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

88. Zapp, M.L.; Hope, T.J.; Parslow, T.G.; Green, M.R. Oligomerization and RNA binding domains of the type 1 human immunodeficiency virus Rev protein: A dual function for an arginine-rich binding motif. *Proc. Natl. Acad. Sci. USA* **1991**, *88*, 7734–7738. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

89. Zemmel, R.W.; Kelley, A.C.; Karn, J.; Butler, P.J. Flexible regions of RNA structure facilitate co-operative Rev assembly on the Rev-response element. *J. Mol. Biol.* **1996**, *258*, 763–777. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

90. Jiang, F.; Gorin, A.; Hu, W.; Majumdar, A.; Baskerville, S.; Xu, W.; Ellington, A.; Patel, D.J. Anchoring an extended HTLV-1 Rex peptide within an RNA major groove containing junctional base triples. *Structure* **1999**, *7*, 1461–1472. [CrossRef](https://doi.org/)

91. Daugherty, M.D.; D'Orso, I.; Frankel, A.D. A Solution to limited genomic capacity: Using adaptable binding surfaces to assemble the functional HIV Rev oligomer on RNA. *Mol. Cell* **2008**, *31*, 824–834. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

92. DiMattiaa, M.A.; Wattsc, N.R.; Stahlc, S.J.; Raderd, C.; Wingfieldc, P.T.; Stuart, D.I.; Stevenb, A.C.; Grimes, J.M. Implications of the HIV-1 Rev dimer structure at 3.2 Å resolution for multimeric binding to the Rev response element. *Proc. Natl. Acad. Sci. USA* **2010**, *107*, 5810–5814. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

93. Daugherty, M.; Liu, B.; Frankel, A. Structural basis for cooperative RNA binding and export complex assembly by HIV Rev. *Nat. Struct. Mol. Biol.* **2010**, *17*, 1337–1343. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

94. Fang, X.; Wang, J.; O’Carroll, I.P.; Mitchell, M.; Zuo, X.; Wang, Y.; Yu, P.; Liu, Y.; Rausch, J.W.; Dyba, M.A.; *et al.* An unusual topological structure of the HIV-1 rev response element. *Cell* **2013**, *155*, 594–605. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

95. Nawroth, I.; Mueller, F.; Basyuk, E.; Beerens, N.; Rahbek, U.L.; Darzacq, X.; Bertrand, E.; Kjems, J.; Schmidt, U. Stable assembly of HIV-1 export complexes occurs cotranscriptionally. *RNA* **2014**, *20*, 1–8. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

96. Bai, Y.; Tambe, A.; Zhou, K.; Doudna, J.A.; States, U.; Division, P.B.; Berkeley, L. RNA-guided assembly of Rev-RRE nuclear export complexes. *Elife* **2014**, *3*, e03656. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

97. Booth, D.S.; Cheng, Y.; Frankel, A.D. The export receptor Crm1 forms a dimer to promote nuclear export of HIV RNA. *Elife* **2014**, *3*, e04121. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

98. Jayaraman, B.; Crosby, D.C.; Homer, C.; Ribeiro, I.; Mavor, D.; Frankel, A.D. RNA-directed remodeling of the HIV-1 protein Rev orchestrates assembly of the Rev-Rev response element complex. *Elife* **2014**, *3*, e04120. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

99. Simonis, N.; Rual, J.-F.; Lemmens, I.; Boxus, M.; Hirozane-Kishikawa, T.; Gatot, J.-S.; Dricot, A.; Hao, T.; Vertommen, D.; Legros, S.; *et al.* Host-pathogen interactome mapping for HTLV-1 and 2 retroviruses. *Retrovirology* **2012**, *9*, 26. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)

100. Abe, M.; Suzuki, H.; Nishitsuji, H.; Shida, H.; Takaku, H. Interaction of human T-cell lymphotropic virus type I Rex protein with Dicer suppresses RNAi silencing. *FEBS Lett.* **2010**, *584*, 4313–4318. [CrossRef](https://doi.org/) [PubMed](https://pubmed.ncbi.nlm.nih.gov/)
